<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Methanol_Single_Expression skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Methanol Single Expression</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmSingleExpression" href="/Team:Aachen/Lab/Methanol/Characterization#Single_Expression"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Single Expression</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH> construct description </TH><TH> sequencing confirmed plasmid   </TH><TH> confirmed cryo
</TH></TR><TR><TD> RBS.mdh in pSB1A30 </TD><TD> #BZBQ# </TD><TD> #IGEM#
</TD></TR><TR><TD> RBS.hps in pSB1A30</TD><TD> #4PAB# </TD><TD> #4P9H#
</TD></TR><TR><TD> RBS.phi in pSB1A30 </TD><TD> #PEA6# </TD><TD> #TFE1#
</TD></TR><TR><TD> RBS.xpk in pSB1A30</TD><TD> #1NFT# </TD><TD> #ONK4#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-16">15-06-16</SPAN></H2><P><B>restrictions:</B></P><OL><LI> B0015 in pSB1C3 #TK6C# with XbaI and PstI</LI><LI> hps in pSB1C3 #H4M9# with EcoRI and SpeI</LI><LI> phi in pSB1C3 #ANNA# with EcoRI and SpeI</LI></OL><P><I>calculation:</I></P><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> Volumes for (1.) B0015 [µl]  </TH><TH> Volumes for (2.) hps [µl] </TH><TH> Volumes for (3.) phi [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 14.7 </TD><TD> 15.2 </TD><TD> 15.5
</TD></TR><TR><TD> 10x NEBuffer 2.1 </TD><TD> 2.0 </TD><TD> 2.0 </TD><TD> 2.0
</TD></TR><TR><TD> Enzyme I </TD><TD> 0.4 (XbaI) </TD><TD> 0.4 (EcoRI) </TD><TD> 0.4 (EcoRI)
</TD></TR><TR><TD> Enzyme II </TD><TD> 0.4 (PstI) </TD><TD> 0.4 (SpeI) </TD><TD> 0.4 (SpeI)
</TD></TR><TR><TD> DNA </TD><TD> 2.5 (#TK6C#) </TD><TD> 2.0 (#H4M9#)</TD><TD> 1.7 (#ANNA#)
</TD></TR></TBODY></TABLE><UL><LI> restriction products stored in Falcon #TUSS#</LI></UL><P><B>ligation:</B></P><UL><LI><B>(A)</B> (1.) B0015 digest and (2.) hps digest into pSB1K30 digest (#SZKD#)</LI><LI><B>(B)</B> (1.) B0015 digest and (3.) phi digest into pSB1K30 digest (#SZKD#)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> Volumes for <B>(A)</B> [µl] </TH><TH> Volumes for <B>(B)</B> [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 7 </TD><TD> 7
</TD></TR><TR><TD> 10x T4 Ligase Buffer </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> upstream Part Digest </TD><TD> 4 (B0015 digest) </TD><TD> 4 (B0015 digest)
</TD></TR><TR><TD> downstream Part Digest </TD><TD> 4 (Hps digest) </TD><TD> 4 (Phi digest)
</TD></TR><TR><TD> pSB1K30 digest #SZKD# </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 1 </TD><TD> 1
</TD></TR></TBODY></TABLE><UL><LI> ligation products stored in falcon #TUSS#</LI><LI> heatshock transformation in DH5α, plate on LB + K</LI></UL><H2><SPAN class="mw-headline" id="15-06-17">15-06-17</SPAN></H2><UL><LI>because backbone is probably not well digested (and because we have all four genes right now!!), we make the restriction and ligation again</LI></UL><P><B>restriction</B></P><UL><LI>xpk in pSB1C3 (#KFEY#) with EcoRI and SpeI (X)</LI><LI>mdh in pSB1C3 (#9PBD#) with EcoRI and SpeI (M) </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> Volumes for Xpk (X) [µl]  </TH><TH> Volumes for mdh (M) [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 16.5 </TD><TD> 15.4
</TD></TR><TR><TD> 10x NEBuffer 2.1 </TD><TD> 2.0 </TD><TD> 2.0
</TD></TR><TR><TD> Enzyme I (EcoRI) </TD><TD> 0.4 </TD><TD> 0.4
</TD></TR><TR><TD> Enzyme II (SpeI) </TD><TD> 0.4  </TD><TD> 0.4
</TD></TR><TR><TD> DNA </TD><TD> 0.7 (#KFEY#) </TD><TD> 1.8 (#9PBD#)
</TD></TR></TBODY></TABLE><UL><LI>xpk restictions: tube X</LI><LI>mdh restrictions: tube M</LI></UL><P><B>ligation</B></P><UL><LI><B>(A)</B>  B0015 digest from #OHOZ# and (2) hps digest from #TUSS# into (K30) pSB1K30 digest from #SZKD#</LI><LI><B>(B)</B>  B0015 digest from #OHOZ# and (3) phi digest from #TUSS# into (K30) pSB1K30 digest from #SZKD#</LI><LI><B>(C)</B>  B0015 digest from #OHOZ# and (X) xpk digest from #OHOZ# into (K30) pSB1K30 digest from #SZKD#</LI><LI><B>(D)</B>  B0015 digest from #OHOZ# and (M) mdh digest from #OHOZ# into (K30) pSB1K30 digest from #SZKD#</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> Volumes for <B>(A)</B> [µl] </TH><TH> Volumes for <B>(B)</B> [µl] </TH><TH> Volumes for <B>(C)</B> [µl] </TH><TH> Volumes for <B>(D)</B> [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 7 </TD><TD> 7 </TD><TD> 7 </TD><TD> 7
</TD></TR><TR><TD> 10x T4 Ligase Buffer </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> upstream Part Digest </TD><TD> 4 (B0015 digest B0015 from #OHOZ#) </TD><TD> 4 (B0015 digest B0015 from #OHOZ#) </TD><TD> 4 (B0015 digest B0015 from #OHOZ#) </TD><TD> 4 (B0015 digest B0015 from #OHOZ#)
</TD></TR><TR><TD> downstream Part Digest  </TD><TD> 4 (hps digest 2 from #TUSS#) </TD><TD> 4 ((phi digest 3 from #TUSS#) </TD><TD> 4 (xpk digest X from #OHOZ#) </TD><TD> 4 (mdh digest M from #OHOZ#)
</TD></TR><TR><TD> pSB1K30 digest (plasmid from #SZKD#) </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> T4 DNA Ligase </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR></TBODY></TABLE><P><B>Trafo</B></P><UL><LI> heatshock transformation in BL21</LI></UL><H2><SPAN class="mw-headline" id="15-06-18">15-06-18</SPAN></H2><UL><LI> make overnight cultures and masterplates
</LI><LI> 12 clones of each construct were plated on master plates. 6 overnights were made.</LI></UL><H2><SPAN class="mw-headline" id="15-06-19">15-06-19</SPAN></H2><P>Colony PCR
</P><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> mdh.B0015 in pSB1K30 clones 1-12 </TD><TD> 37-48 </TD><TD> 1682 </TD><TD> 2 </TD><TD> 1'50&quot; </TD><TD> Negative for all clones. Observed bands at 1227 bp (RFP)
</TD></TR><TR><TD> hps.B0015 in pSB1K30 clones 1-12 </TD><TD> 13-24 </TD><TD> 1160 </TD><TD> 1 </TD><TD> 1'13&quot; </TD><TD> Negative for all clones. Observed bands at 1227 bp (RFP)
</TD></TR><TR><TD> phi.B0015 in pSB1K30 clones 1-12 </TD><TD> 1-12 </TD><TD> 1079 </TD><TD> 1 </TD><TD> 1'13&quot; </TD><TD> Negative for all clones. Observed bands at 1227 bp (RFP)
</TD></TR><TR><TD> xpk.B0015 in pSB1K30 clones 1-12 </TD><TD> 73-84 </TD><TD> 3002 </TD><TD> 3 </TD><TD> 3'05&quot; </TD><TD> Negative for all clones. Observed bands at 1227 bp (RFP)
</TD></TR></TBODY></TABLE><UL><LI> We picked only &quot;red&quot; clones, because we forgot to induce with IPTG.</LI><LI> with green light the red flourescence could be observed very well</LI><LI> we picked the few white clones and put them on a masterplate LB+K</LI><LI> next step will be a new colony PCR on monday.</LI></UL><H2><SPAN class="mw-headline" id="15-06-22">15-06-22</SPAN></H2><P>colony PCR
</P><UL><LI>tube numbers on masterplate</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> mdh.B0015 in pSB1K30 clones 1-3 </TD><TD> 6-8 </TD><TD> 1682 </TD><TD> 2 </TD><TD> 1'50&quot; </TD><TD> bands are too short
</TD></TR><TR><TD> hps.B0015 in pSB1K30 clones 1-3 </TD><TD> 1-3 </TD><TD> 1160 </TD><TD> 1 </TD><TD> 1'13&quot; </TD><TD> bands are too short
</TD></TR><TR><TD> phi.B0015 in pSB1K30 clones 1-2 </TD><TD> 4-5 </TD><TD> 1079 </TD><TD> 1 </TD><TD> 1'13&quot; </TD><TD> bands are too short
</TD></TR><TR><TD> xpk.B0015 in pSB1K30 clones 1-3 </TD><TD> 9-11 </TD><TD> 3002 </TD><TD> 3 </TD><TD> 4'00&quot; </TD><TD> bands are too short
</TD></TR></TBODY></TABLE><UL><LI> restriction/ligation didn't work out as we expected.</LI><LI> problem: pSB1K30 digest never works properly</LI></UL><UL><LI> next step: plate cryo #8T49# to get more plasmid of pSB1K30</LI></UL><H2><SPAN class="mw-headline" id="15-06-23">15-06-23</SPAN></H2><UL><LI> cut #LYZH# (pSB1A30) with EcoRI and PstI</LI><LI> cut #TK6C# (B0015 in pSB1C3) with XbaI and PstI
</LI><LI> reaction volumes: 50 µl</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> volumes [µl] for pSB1A30 digest </TH><TH> volumes [µl] for B0015 digest
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 38.5 </TD><TD> 36.8
</TD></TR><TR><TD> NEBuffer 2.1 </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> Enzyme I </TD><TD> 1 (EcoRI) </TD><TD> 1 (XbaI)
</TD></TR><TR><TD> Enzyme II </TD><TD> 1 (PstI) </TD><TD> 1 (PstI)
</TD></TR><TR><TD> DNA </TD><TD> 4.5 (#LYZH#) </TD><TD> 6.3 (#TK6C#)
</TD></TR></TBODY></TABLE><UL><LI> restrictions stored in Falcon #OHOZ# </LI><LI> gel check of  digests:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> digest </TH><TH> expected bands </TH><TH> slot on gel </TH><TH> result
</TH></TR><TR><TD> #TUSS#:2 </TD><TD> 684/2051 </TD><TD> 5 </TD><TD> corresponding bands
</TD></TR><TR><TD> #TUSS#:3 </TD><TD> 558/2051 </TD><TD> 6 </TD><TD> corresponding bands
</TD></TR><TR><TD> #OHOZ#:X </TD><TD> 2526/2051 </TD><TD> 2 </TD><TD> corresponding bands
</TD></TR><TR><TD> #OHOZ#:M </TD><TD> 1206/2051 </TD><TD> 1 </TD><TD> corresponding bands
</TD></TR><TR><TD> #OHOZ#:1 </TD><TD> 902/2173 </TD><TD> 3 </TD><TD> corresponding bands
</TD></TR><TR><TD> #OHOZ#:2 </TD><TD> 155/2052 </TD><TD> 4 </TD><TD> bands of this length not visible in general on gel
</TD></TR></TBODY></TABLE><UL><LI> ligate each RBS.CDS digest with the B0015 out of pSB1C3 into pSB1A30 digest </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> (A) RBS.mdh.B0015 in pSB1A30 volumes [µl] </TH><TH> (B) RBS.hps.B0015 in pSB1A30 volumes [µl] </TH><TH> (C) RBS.phi.B0015 in pSB1A30 volumes [µl] </TH><TH> (D) RBS.xpk.B0015 in pSB1A30 volumes [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 7 </TD><TD> 7 </TD><TD> 7 </TD><TD> 7
</TD></TR><TR><TD> T4 Ligase Buffer </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> upstream part digest </TD><TD> 4 (#OHOZ#:M) </TD><TD> 4 (#TUSS#:2) </TD><TD> 4 (#TUSS#:3) </TD><TD> 4 (#OHOZ#:X)
</TD></TR><TR><TD> downstream Part digest (#OHOZ#:2) </TD><TD> 4 </TD><TD> 4 </TD><TD> 4 </TD><TD> 4
</TD></TR><TR><TD> backbone digest (#OHOZ#:1) </TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD><TD> 2
</TD></TR><TR><TD> T4 Ligase </TD><TD> 1 </TD><TD> 1 </TD><TD> 1 </TD><TD> 1
</TD></TR></TBODY></TABLE><P>ligation products stored in falcon #CAJO#
</P><UL><LI> heatshock transformation into BL21. Don't forget induction with IPTG for red/white screening!</LI></UL><H2><SPAN class="mw-headline" id="15-06-24">15-06-24</SPAN></H2><UL><LI> make masterplates</LI><LI> again do heat shock transformation in BL21 cells with the ligation products (#CAJO#)</LI><LI> purify plasmid of pSB1K30. plasmid stored in: #1K30#</LI></UL><H2><SPAN class="mw-headline" id="15-06-25">15-06-25</SPAN></H2><UL><LI> do colony PCR and make overnights of successful clones</LI><LI> perhaps new masterplates of transformants from 15-06-24</LI></UL><P><B>Colony PCR</B></P><UL><LI> with KAPA 2G Fast Ready MasterMix</LI></UL><P>Programm: 
</P><TABLE class="wikitable"><TBODY><TR><TH>! Step </TH><TH> Time  </TH><TH> Temp
</TH></TR><TR><TD> denature </TD><TD> 3'00</TD><TD> 95
</TD></TR><TR><TD> denature </TD><TD> 0'15</TD><TD> 95
</TD></TR><TR><TD> anneal </TD><TD> 0'15</TD><TD> 58
</TD></TR><TR><TD> elongate </TD><TD> 0'25<I> (Machine1) / 0'45</I> (Machine2)</TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 2'00</TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>30 cycles!
</P><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> RBS.mdh.B0015 in pSB1A30 clones 1 </TD><TD> 1 </TD><TD> 1682 </TD><TD> 2 </TD><TD> 0'25</TD><TD> no bands
</TD></TR><TR><TD> RBS.hps.B0015 in pSB1A30 clones 1-3 </TD><TD> 2, 3, 4 </TD><TD> 1160 </TD><TD> 2 </TD><TD> 0'25</TD><TD> no bonds
</TD></TR><TR><TD> RBS.phi.B0015 in pSB1A30 clones 1-6 </TD><TD> 5, 6, 7, 8, 9, 10 </TD><TD> 1079 </TD><TD> 2 </TD><TD> 0'25</TD><TD> very bad bands
</TD></TR><TR><TD> RBS.xpk.B0015 in pSB1A30 clones 1-3 </TD><TD> 11, 12, 13  </TD><TD> 3002 </TD><TD> 5 </TD><TD> 0'45</TD><TD> very bad bands
</TD></TR></TBODY></TABLE><P><B>new transformants</B></P><UL><LI>make masterplates of all transformants WITH IPTG!
</LI><LI>Plate 1: 26 g --&gt; 26 µl 1000x IPTG</LI><LI>Plate 2: 30 g --&gt; 30 µl 1000x IPTG</LI></UL><LI>make overnight cultures of  35 clones</LI><H2><SPAN class="mw-headline" id="15-06-26">15-06-26</SPAN></H2><P><B>Colony PCR</B></P><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> RBS.mdh.B0015 in pSB1A30 clones 6, 8, 10, 11 </TD><TD> 1-4 </TD><TD> 1682 </TD><TD> 2 </TD><TD> 0'25</TD><TD> no bands
</TD></TR><TR><TD> RBS.hps.B0015 in pSB1A30 clones 3 </TD><TD> 5 </TD><TD> 1160 </TD><TD> 2 </TD><TD> 0'25</TD><TD> no bands
</TD></TR><TR><TD> RBS.phi.B0015 in pSB1A30 clones 2, 4, 6, 7, 8, 13, 17-21 </TD><TD> 6-16</TD><TD> 1079 </TD><TD> 2 </TD><TD> 0'25</TD><TD> no bands
</TD></TR><TR><TD> RBS.xpk.B0015 in pSB1A30 clones 1, 7 </TD><TD> 17, 18  </TD><TD> 3002 </TD><TD> 5 </TD><TD> 0'45</TD><TD> no bands
</TD></TR></TBODY></TABLE><UL><LI>control with red clones: mdh (tube MR), hps (tube HR), phi (tube PR), xpk (XR)</LI></UL><P><B>We have to think about a new strategy, as the Biobrick assembly seems not to work</B></P><H2><SPAN class="mw-headline" id="15-06-29">15-06-29</SPAN></H2><P>A new beginning! Work carefully! Our new approach is to ligate the RBS.CDS constructs into a prefix cut (with EcoRI and XbaI) 
pSB1A3 vector containing the B0015. 
</P><P>In a next step we will swap the RBS.CDS.B0015 constructs into pSB1K30 backbones.
</P><P>Monday and Wednesday are the days with the most workload, while tuesday and thursday give space to plan further steps 
</P><UL><LI> cut #FW6L# with EcoRI and XbaI; reaction volume: 50 µl; water: 36.3 µl; DNA: 6.7 µl</LI><LI> cut #1K30# with EcoRI and PstI; reaction volume: 50 µl; water: 37 µl; DNA: 6 µl</LI><LI> control of digests on gel
</LI><LI> expected lengths:
</LI><LI> BEX: 19 bp / 2273 bp (bands at expected length)</LI><LI> K30: 902 bp / 2222 bp (very weak bands)</LI></UL><P>As digest of pSB1K30 never works, we will now use pSB1A30
</P><UL><LI> digest RBS.CDS constructs with EcoRI and PstI
</LI><LI> RBS.mdh from #YSVQ#</LI><LI> RBS.hps from #H4M9#</LI><LI> RBS.phi from #ANNA#</LI><LI> RBS.mdh from #KFEY#</LI></UL><H2><SPAN class="mw-headline" id="15-06-30">15-06-30</SPAN></H2><H4><SPAN class="mw-headline" id="RBS.CDS.B0015_in_pSB1A3">RBS.CDS.B0015 in pSB1A3</SPAN></H4><UL><LI> transform ligation products into DH5alpha again</LI></UL><H4><SPAN class="mw-headline" id="RBS.CDS_in_pSB1A30">RBS.CDS in pSB1A30</SPAN></H4><UL><LI> control digests on gel</LI><LI>results
</LI><LI> #TUSS#:mdh (expected: 1220 bp / 2037 bp): bands at expected length</LI><LI> #TUSS#:hps (expected: 698 bp / 2037 bp): bands at expected length</LI><LI> #TUSS#:phi (expected: 617 bp / 2037 bp): bands at expected length</LI><LI> #TUSS#:xpk (expected: 2540 bp / 2037 bp): bands at expected length</LI><LI> #TUSS#:K30 (expected: 902 bp / 2222 bp): no bands at all</LI></UL><LI> ligation of fragments
</LI><LI> Ligation of #OHOZ#:1 with ...
</LI><LI> ...#TUSS#:mdh (ML)</LI><LI> ...#TUSS#:hps (HL)</LI><LI> ...#TUSS#:phi (PL)</LI><LI> ...#TUSS#:xpk (XL)</LI><P>Pipeting volumes for ligation
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [µl]
</TH></TR><TR><TD> water </TD><TD> 10
</TD></TR><TR><TD> T4 Ligase Buffer </TD><TD> 2
</TD></TR><TR><TD> Backbone digest (#OHOZ#:1) </TD><TD> 3
</TD></TR><TR><TD> part digest (from #TUSS#) </TD><TD> 4
</TD></TR><TR><TD> T4 Ligase </TD><TD> 1
</TD></TR></TBODY></TABLE><UL><LI> ligation products stored in falcon #LIGJ#</LI><LI> transformation into BL21 Gold (DE3)</LI></UL><H2><SPAN class="mw-headline" id="15-07-01">15-07-01</SPAN></H2><H4><SPAN class="mw-headline" id="Colony_PCR_of_RBS.CDS.B0015_in_pSB1A30">Colony PCR of RBS.CDS.B0015 in pSB1A30</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number / row in gel </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> RBS.mdh.B0015 in pSB1A30 clones 6, 8, 10, 11 </TD><TD> 1-4 </TD><TD> 1682 </TD><TD>2 </TD><TD> 0'25</TD><TD> no positive clones
</TD></TR><TR><TD> RBS.hps.B0015 in pSB1A30 clones 3 </TD><TD> 5 </TD><TD> 1160 </TD><TD> 2 </TD><TD> 0'25</TD><TD> no positive clones
</TD></TR><TR><TD> RBS.phi.B0015 in pSB1A30 clones 2, 4, 6, 7, 8, 13, 17-21 </TD><TD> 6-16</TD><TD> 1079 </TD><TD> 2 </TD><TD> 0'25</TD><TD> clones 13, 19, 20 and 21 positive
</TD></TR><TR><TD> RBS.xpk.B0015 in pSB1A30 clones 1, 7 </TD><TD> 17, 18  </TD><TD> 3002 </TD><TD> 5 </TD><TD> 0'45</TD><TD> clone 7 positive
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="RBS.CDS.B0015_in_pSB1A3_.28E.2C_F.2C_G.2C_H.29">RBS.CDS.B0015 in pSB1A3 (E, F, G, H)</SPAN></H4><UL><LI> make masterplates and overnight cultures</LI></UL><H4><SPAN class="mw-headline" id="RBS.CDS_in_pSB1A30_.28ML.2C_HL.2C_PL.2C_XL.29">RBS.CDS in pSB1A30 (ML, HL, PL, XL)</SPAN></H4><UL><LI> make masterplates and overnights</LI></UL><H2><SPAN class="mw-headline" id="15-07-02">15-07-02</SPAN></H2><H4><SPAN class="mw-headline" id="RBS.CDS.B0015_in_pSB1A30">RBS.CDS.B0015 in pSB1A30</SPAN></H4><UL><LI> purify plasmid of:
</LI><LI> xpk clone 7</LI><LI> phi clones 13, 19, 20, 21</LI></UL><H4><SPAN class="mw-headline" id="RBS.CDS.B0015_in_pSB1A3_.28E.2C_F.2C_G.2C_H.29_2">RBS.CDS.B0015 in pSB1A3 (E, F, G, H)</SPAN></H4><UL><LI> colony PCR </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct ID </TH><TH> construct description </TH><TH> clone numbers </TH><TH> expected length [bp] </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> E </TD><TD> RBS.mdh.B0015 in pSB1A3 </TD><TD> 1-8 </TD><TD> 1631 </TD><TD> 0'25</TD><TD> nothing
</TD></TR><TR><TD> F </TD><TD> RBS.hps.B0015 in pSB1A3 </TD><TD> 9-16 </TD><TD> 1109 </TD><TD> 0'25</TD><TD> nothing
</TD></TR><TR><TD> G </TD><TD> RBS.phi.B0015 in pSB1A3 </TD><TD> 17-24 </TD><TD> 1028 </TD><TD> 0'25</TD><TD> nothing
</TD></TR><TR><TD> H </TD><TD> RBS.xpk.B0015 in pSB1A3  </TD><TD> 25-32 </TD><TD> 2951 </TD><TD> 0'45</TD><TD> clone 26 positive
</TD></TR></TBODY></TABLE><UL><LI> purify plasmid and make cryos of positive clones</LI><LI> prepare positive clones for sequencing </LI></UL><UL><LI> because almost nothing worked: new transformation of ligation products #CAJO#:E/F/G/H
</LI><LI> 50 µl competent DH5alpha</LI><LI> 2 µl ligation product</LI><LI> 200 µl SOC media</LI></UL><H4><SPAN class="mw-headline" id="RBS.CDS_in_pSB1A30_.28ML.2C_HL.2C_PL.2C_XL.29_2">RBS.CDS in pSB1A30 (ML, HL, PL, XL)</SPAN></H4><UL><LI> colony PCR </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct ID </TH><TH> construct description </TH><TH> clone numbers </TH><TH> expected length [bp] </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> ML </TD><TD> RBS.mdh in pSB1A30 </TD><TD> 33-40 </TD><TD> 1553 </TD><TD> 0'25</TD><TD> Clone: 34, 35, 37, 39
</TD></TR><TR><TD> HL </TD><TD> RBS.hps in pSB1A30 </TD><TD> 41-48 </TD><TD> 1031 </TD><TD> 0'25</TD><TD> Clone: 41 - 48
</TD></TR><TR><TD> PL </TD><TD> RBS.phi in pSB1A30 </TD><TD> 49-56 </TD><TD> 950 </TD><TD> 0'25</TD><TD> Clone: 49 - 56
</TD></TR><TR><TD> XL </TD><TD> RBS.xpk in pSB1A30  </TD><TD> 57-64 </TD><TD> 2873 </TD><TD> 0'45</TD><TD> clones 57 and 62 positive
</TD></TR></TBODY></TABLE><UL><LI> purify plasmid and make cryos of positive clones</LI><LI> prepare positive clones for sequencing</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Construct </TH><TH> Clone </TH><TH> Plasmid ID </TH><TH> Cryo ID </TH><TH> Plasmid konzentration </TH><TH> Sequencing result
</TH></TR><TR><TD> H: RBS.xpk.B0015 in pSB1A3 </TD><TD> 26 </TD><TD> #8ZRY#</TD><TD> #84BM# </TD><TD> 234.7 </TD><TD> VALID
</TD></TR><TR><TD> ML: RBS.mdh in pSB1A30 </TD><TD> 34 </TD><TD> #1RZF# </TD><TD> #8AZD#</TD><TD> 136.5 </TD><TD> discard
</TD></TR><TR><TD> 35 </TD><TD> #PZRX# </TD><TD>#MB3A# </TD><TD> 160.3 </TD><TD> VALID
</TD></TR><TR><TD> 37 </TD><TD> #BZBQ# </TD><TD> #IGEM# </TD><TD> 187.3 </TD><TD> VALID
</TD></TR><TR><TD> 39 </TD><TD> #6WTB# </TD><TD> #HEXE# </TD><TD> 133.8 </TD><TD> discard
</TD></TR><TR><TD> HL: RBS.hps in pSB1A30  </TD><TD> 41 </TD><TD> #YSAE# </TD><TD> #MKCP# </TD><TD> 165.4 </TD><TD> VALID
</TD></TR><TR><TD> 42 </TD><TD> #T8KB# </TD><TD> #DR63# </TD><TD> 133.4 </TD><TD> discard
</TD></TR><TR><TD> 43 </TD><TD> #H8KF# </TD><TD> #F4KN# </TD><TD> 158.1 </TD><TD> discard
</TD></TR><TR><TD> 44 </TD><TD> #4PAB# </TD><TD> #4P9H# </TD><TD> 188.0 </TD><TD> VALID
</TD></TR><TR><TD> PL: RBS.phi in pSB1A30 </TD><TD> 49 </TD><TD> #PEA6# </TD><TD> #TFE1# </TD><TD> 168.4 </TD><TD> VALID
</TD></TR><TR><TD> 50 </TD><TD> #814A# </TD><TD> #QHB3# </TD><TD> 139.7 </TD><TD> discard
</TD></TR><TR><TD> 51 </TD><TD> #AQQ8# </TD><TD> #1XAB# </TD><TD> 151.2 </TD><TD> VALID
</TD></TR><TR><TD> 52 </TD><TD> #VBEZ# </TD><TD> #B9XO# </TD><TD> 135.2 </TD><TD> discard
</TD></TR><TR><TD> XL: RBS.xpk in pSB1A30 </TD><TD> 57 </TD><TD> #4AE3# </TD><TD> #NF66# </TD><TD> 110.5 </TD><TD> VALID
</TD></TR><TR><TD> 62 </TD><TD> #1NFT# </TD><TD> #ONK4# </TD><TD> 268.2 </TD><TD> VALID
</TD></TR><TR><TD> RBS.xpk.B0015 in pSB1A30 </TD><TD> 7 </TD><TD> #KANH# </TD><TD> #CNEK# </TD><TD> 110.1 </TD><TD> VALID
</TD></TR><TR><TD> RBS.phi.B0015 in pSB1A30 </TD><TD> 13 </TD><TD> #RX39# </TD><TD> #D8S8# </TD><TD> 85.0 </TD><TD> VALID
</TD></TR><TR><TD> 19 </TD><TD> #3MNR# </TD><TD> #PZ6F# </TD><TD> 45.5 </TD><TD> discard
</TD></TR><TR><TD> 20 </TD><TD> #XTW8# </TD><TD> #E1KA# </TD><TD> 58.9 </TD><TD> VALID
</TD></TR><TR><TD> 21 </TD><TD> #O3FR# </TD><TD> #N9EE# </TD><TD> 53.7 </TD><TD> discard
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-03">15-07-03</SPAN></H2><UL><LI> plasmid purification of all positive clones from yesterday</LI><LI> prepare sequencing</LI><LI> make LB+A plates</LI><LI> masterplates and overnights of unsuccessful RBS.CDS.B0015 in pSB1A3 constructs</LI><LI> plan workflow for saturday
</LI><LI> list of items to take to iAMB:
</LI><LI> KAPA Mastermix</LI><LI> Primers</LI><LI> Master plates</LI><LI> PCR strips</LI><LI> 2-log Ladder</LI></UL><UL><LI> edit gel pictures of yesterday (iJR, at home)</LI></UL><H2><SPAN class="mw-headline" id="15-07-06">15-07-06</SPAN></H2><P><B>as our colony PCRs are very unreliable und the last time, so we will check all the factors that could influence the results (product lengths)</B></P><P>Test with KAPA 2G FAST READY mastermix, each with the same programm:
</P><TABLE class="wikitable"><TBODY><TR><TH> Step </TH><TH> Time  </TH><TH> Temp
</TH></TR><TR><TD> denature </TD><TD> 3'00</TD><TD> 95
</TD></TR><TR><TD> cyc. denature </TD><TD> 0'15</TD><TD> 95
</TD></TR><TR><TD> cyc. anneal </TD><TD> 0'15</TD><TD> 58
</TD></TR><TR><TD> cyc. elongate </TD><TD> variable </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 2'00</TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>30 cycles!
</P><UL><LI>Test with 1 µl of purified plasmids:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct ID </TH><TH> construct description </TH><TH> tube number/row </TH><TH> expected length [bp] </TH><TH> elongation time </TH><TH> cycler </TH><TH> result
</TH></TR><TR><TD> #CD8B# </TD><TD> J04450 in pSB1C3 plasmid </TD><TD> 1 </TD><TD> 1348 </TD><TD> 0'21</TD><TD> a </TD><TD> very strong band with expected length
</TD></TR><TR><TD> #KFEY# </TD><TD> Xpk in pSB1C3 plasmid </TD><TD> 2 </TD><TD> 2814 </TD><TD> 0'42</TD><TD> b </TD><TD> weak band with expected length + grease
</TD></TR><TR><TD> #CT8H# </TD><TD> glgAB in pSB1A3 plasmid </TD><TD> 3 </TD><TD> 3986 </TD><TD> 1'00</TD><TD> c </TD><TD> no bands
</TD></TR><TR><TD> - </TD><TD> Water  </TD><TD>  4 </TD><TD> --- </TD><TD> 1'00</TD><TD> c </TD><TD> no bands
</TD></TR></TBODY></TABLE><UL><LI>test with 1µl of cryos: </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct ID </TH><TH> construct description </TH><TH> tube number/row </TH><TH> expected length [bp] </TH><TH> elongation time </TH><TH> cycler </TH><TH> result
</TH></TR><TR><TD> #W39E# </TD><TD> J04450 in pSB1C3 plasmid </TD><TD> 5 </TD><TD> 1348 </TD><TD> 0'21</TD><TD> a </TD><TD> strong band with expected length
</TD></TR><TR><TD> #8WY9# </TD><TD> Xpk in pSB1C3 plasmid </TD><TD> 6 </TD><TD> 2814 </TD><TD> 0'42</TD><TD> b </TD><TD> no bands
</TD></TR><TR><TD> #1Q88# </TD><TD> glgAB in pSB1A3 plasmid </TD><TD> 7 </TD><TD> 3986 </TD><TD> 1'00</TD><TD> c </TD><TD> very weak band with expected length
</TD></TR><TR><TD> - </TD><TD> Water  </TD><TD>  8 </TD><TD> --- </TD><TD> 1'00</TD><TD> c </TD><TD> weak band at ca. 4000 bp because of pipetting mistake (iCG)
</TD></TR></TBODY></TABLE><P><B>It seems like the mastermix has problems with longer product lengths</B></P><UL><LI> spread a little bit of the cryos on the corresponding plates, to test colony PCR with colonys</LI></UL><H2><SPAN class="mw-headline" id="15-07-07">15-07-07</SPAN></H2><H4><SPAN class="mw-headline" id="Expression_test">Expression test</SPAN></H4><UL><LI> first expression test of mdh</LI><LI> plating of mdh in A30 in BL21 clones:
</LI><LI> #IGEM#</LI><LI> #MB3A#</LI><LI> #8ADZ#</LI><LI> #HEXE#</LI></UL><LI> plating of rfp control:
</LI><LI> #6DA3#</LI><H4><SPAN class="mw-headline" id="Colony_PCR_of_template_plasmids_E.2C_F.2C_G_and_H">Colony PCR of template plasmids E, F, G and H</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> construct ID </TH><TH> construct description </TH><TH> numbers of clones </TH><TH> expected length [bp] </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> E </TD><TD> RBS.mdh.B0015 in pSB1A3 </TD><TD> 27 </TD><TD> 1631 </TD><TD> 1'38</TD><TD> clone 6, 7, 8, 10, 17, 25, 27 positive
</TD></TR><TR><TD> F </TD><TD> RBS.hps.B0015 in pSB1A3 </TD><TD> 23 </TD><TD> 1109 </TD><TD> 1'07</TD><TD> clone 6, 9, 21, 22, 23 positive
</TD></TR><TR><TD> G </TD><TD> RBS.phi.B0015 in pSB1A3 </TD><TD> 24 </TD><TD> 1028 </TD><TD> 1'07</TD><TD> clone 4, 12, 13, 15, 17, 18, 19 positive
</TD></TR><TR><TD> H </TD><TD> RBS.xpk.B0015 in pSB1A3  </TD><TD> 22 </TD><TD> 2951 </TD><TD> 2'57</TD><TD> clone 2, 6, 17, 22
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-08">15-07-08</SPAN></H2><UL><LI> purify plasmid of constructs E, F, G, H</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct ID </TH><TH> construct description </TH><TH> clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing results
</TH></TR><TR><TD> E </TD><TD> RBS.mdh.B0015 in pSB1A3 </TD><TD> 6 </TD><TD> #CHWW# </TD><TD> #H39R# </TD><TD> VALID
</TD></TR><TR><TD> 25 </TD><TD> #84BB# </TD><TD> #RRN3# </TD><TD> VALID
</TD></TR><TR><TD> 27 </TD><TD> #WBCK# </TD><TD> #L4OY# </TD><TD> VALID, but discard to safe space
</TD></TR><TR><TD> F </TD><TD> RBS.hps.B0015 in pSB1A3 </TD><TD> 6 </TD><TD> #HBDX# </TD><TD> #EZWV# </TD><TD> VALID
</TD></TR><TR><TD> 21 </TD><TD> #KW16# </TD><TD> #8181# </TD><TD> VALID
</TD></TR><TR><TD> 22 </TD><TD> #RSWY# </TD><TD> #HVXO# </TD><TD> VALID, but discard to safe space
</TD></TR><TR><TD> G </TD><TD> RBS.phi.B0015 in pSB1A3 </TD><TD> 4 </TD><TD> #E89D# </TD><TD> #X9WT# </TD><TD> discard before sequencing
</TD></TR><TR><TD> 12 </TD><TD> #HWPO# </TD><TD> #H9BV# </TD><TD> discard before sequencing
</TD></TR><TR><TD> 18 </TD><TD> #F6V1# </TD><TD> #DK8F# </TD><TD> discard before sequencing
</TD></TR><TR><TD> H </TD><TD> RBS.xpk.B0015 in pSB1A3  </TD><TD> 2 </TD><TD> #3A1V# </TD><TD> #DDT3# </TD><TD> discard before sequencing
</TD></TR><TR><TD> 6 </TD><TD> #E63S# </TD><TD> #Q9B3# </TD><TD> discard before sequencing
</TD></TR><TR><TD> 22 </TD><TD> #S9KK# </TD><TD> #3F69# </TD><TD> discard before sequencing
</TD></TR></TBODY></TABLE><UL><LI> check sequencing results of sequencing from 15-07-02</LI><LI> prepare overnights for expression test of platet cryos:
</LI><LI> 2x #IGEM# (mdh in pSB1A30)</LI><LI> #6DA3# (RFP control)</LI></UL><LI> check out results table on top of this page!</LI><H2><SPAN class="mw-headline" id="15-07-09">15-07-09</SPAN></H2><P><B>Expression Tests</B></P><UL><LI>we test Mdh (+) induced, Mdh (-) not induced, and Rfp control</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> time (h)\ Sample   </TH><TH> (1) Mdh (-) </TH><TH> (2) Mdh (+) </TH><TH> (0) RFP
</TH></TR><TR><TD> 0 </TD><TD> 0.11 </TD><TD> 0.11 </TD><TD> 0.12
</TD></TR><TR><TD> 1</TD><TD> 0.16 </TD><TD> 0.2 </TD><TD> 0.29
</TD></TR><TR><TD> 2 </TD><TD> 0.27 </TD><TD> 0.47 </TD><TD> 0.91
</TD></TR><TR><TD> 2.3 </TD><TD> 0.37 </TD><TD> 0.72 </TD><TD> 1.212
</TD></TR><TR><TD> ...</TD></TR></TBODY></TABLE><UL><LI> at 12:20 induced with IPTG, 50 µl (Mdh +) respectively 51 µl (RFP)</LI></UL><H2><SPAN class="mw-headline" id="15-07-10">15-07-10</SPAN></H2><UL><LI> samples after 5h and after 20 h after induction with IPTG.</LI><LI> SDS gel</LI><LI> samples were set to OD 10</LI><LI> 7 µl of samples were loaded to gel</LI><LI> SDS gel showed no significant bands</LI><LI> repeat expression test of Mdh with other three genes next week </LI></UL><H2><SPAN class="mw-headline" id="15-07-13">15-07-13</SPAN></H2><UL><LI> plate cryos on LB+A plates for precultures:
</LI><LI> mdh: #IGEM#</LI><LI> hps: #4P9H#</LI><LI> phi: #TFE1#</LI><LI> xpk: #ONK4#</LI><LI> RFP: #6DA3#</LI></UL><H2><SPAN class="mw-headline" id="15-07-15">15-07-15</SPAN></H2><H4><SPAN class="mw-headline" id="Expression_Tests">Expression Tests</SPAN></H4><UL><LI> main culture preparation</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> ID </TH><TH> OD Preculture </TH><TH> Volume added to 50 ml LB+A [ml]
</TH></TR><TR><TD> mdh 1 </TD><TD> 5.031 </TD><TD> 1.5
</TD></TR><TR><TD> mdh 2 </TD><TD> 4.872 </TD><TD> 1.5
</TD></TR><TR><TD> hps 1  </TD><TD> 5.382 </TD><TD> 1.4
</TD></TR><TR><TD> hps 2 </TD><TD> 5.421 </TD><TD> 1.4
</TD></TR><TR><TD> phi 1 </TD><TD> 5.826 </TD><TD> 1.2
</TD></TR><TR><TD> phi 2 </TD><TD> 6.264 </TD><TD> 1.2
</TD></TR><TR><TD> xpk 1  </TD><TD> 5.442 </TD><TD> 1.4
</TD></TR><TR><TD> xpk 2 </TD><TD> 5.361 </TD><TD> 1.4
</TD></TR><TR><TD> RFP Control </TD><TD> 4.542 </TD><TD> 1.7
</TD></TR></TBODY></TABLE><UL><LI> main culture OD values</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> sample    </TH><TH> OD (1 h)  </TH><TH> OD (1,5 h) </TH><TH> OD (2 h) </TH><TH> induced
</TH></TR><TR><TD> mdh 1 </TD><TD> 0.234 </TD><TD> 0.408 </TD><TD> 0.568 </TD><TD> no
</TD></TR><TR><TD> mdh 2 </TD><TD> 0.254 </TD><TD> 0.469 </TD><TD> 0.655 </TD><TD> yes
</TD></TR><TR><TD> hps 1 </TD><TD> 0.344 </TD><TD> 0.624 </TD><TD> 0.814 </TD><TD> no
</TD></TR><TR><TD> hps 2 </TD><TD> 0.305 </TD><TD> 0.577 </TD><TD> 0.761 </TD><TD> yes
</TD></TR><TR><TD> phi 1 </TD><TD> 0.282 </TD><TD> 0.526 </TD><TD> 0.721 </TD><TD> yes
</TD></TR><TR><TD> phi 2 </TD><TD> 0.295 </TD><TD> 0.581 </TD><TD> 0.776 </TD><TD> no
</TD></TR><TR><TD> xpk 1 </TD><TD> 0.270 </TD><TD> 0.476 </TD><TD> 0.648 </TD><TD> no
</TD></TR><TR><TD> xpk 2  </TD><TD> 0.275 </TD><TD> 0.494 </TD><TD> 0.650 </TD><TD> yes
</TD></TR><TR><TD> Rfp </TD><TD> 0.348 </TD><TD> 0.640 </TD><TD> 0.816 </TD><TD> yes
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-16">15-07-16</SPAN></H2><H4><SPAN class="mw-headline" id="Expression_tests_2">Expression tests</SPAN></H4><UL><LI> samples taken at 6 h and 21 h after induction with 50 µl IPTG</LI><LI> cell pellets were resuspended in 150 µl water </LI><LI> suspensions were diluted to OD 10</LI><LI> 10 µl SDS added to 50 µl cell suspension (OD 10)</LI><LI> incubate at 95 °C for 15 min</LI><LI> finished SDS preparations stored in Falcons:
</LI><LI> mdh: #MDQL# </LI><LI> hps: #PBEA#</LI><LI> phi: #BHBB#</LI><LI> xpk: #ZR8V#  </LI></UL><H2><SPAN class="mw-headline" id="15-07-17">15-07-17</SPAN></H2><H4><SPAN class="mw-headline" id="Expression_tests_3">Expression tests</SPAN></H4><P><B>Mdh - SDS gel</B></P><UL><LI> negative expression test</LI></UL><P><B>Hps - SDS gel</B></P><UL><LI> no positive expression test so far</LI></UL><P><B>Phi - SDS gel</B></P><UL><LI> no positive expression test so far</LI></UL><P><B>Xpk - SDS gel</B></P><UL><LI> positive expression test</LI></UL><H2><SPAN class="mw-headline" id="15-07-21">15-07-21</SPAN></H2><H4><SPAN class="mw-headline" id="Expression_tests_4">Expression tests</SPAN></H4><P><B>Hps - SDS gel</B></P><UL><LI> no reliable results</LI></UL><P><B>Phi - SDS gel</B></P><UL><LI> no reliable results</LI></UL><H4><SPAN class="mw-headline" id="New_expression_tests_of_Hps_and_Phi">New expression tests of Hps and Phi</SPAN></H4><UL><LI> precultures from cryos #4P9H# (hps) and #TFE1# (phi) </LI><LI> preculture from cryo #IGEM# (mdh) as control</LI></UL><H2><SPAN class="mw-headline" id="15-07-22">15-07-22</SPAN></H2><P><B>New expression tests of Hps and Phi</B></P><UL><LI> make maincultures of Hps and Phi</LI><LI> 2 ml samples taken 6 h after induction with IPTG</LI></UL><H2><SPAN class="mw-headline" id="15-07-23">15-07-23</SPAN></H2><H4><SPAN class="mw-headline" id="New_expression_tests_of_Hps_and_Phi_2">New expression tests of Hps and Phi</SPAN></H4><UL><LI> prepare samples and load them on the gel</LI><LI> run gel at 130 V for 110 min</LI><LI> results: gel has bad quality, no clear bands visible</LI></UL><H2><SPAN class="mw-headline" id="15-07-27">15-07-27</SPAN></H2><P>clone mdh, hps and phi into J61002 with constitutive Anderson Promoter J23119 for <SPAN style="color:red">expression tests with Anderson Promoter 19</SPAN></P><P><B>restriction</B></P><P>templates: 
</P><P>Cut with XbaI/PstI
</P><UL><LI> RBS.mdh in A30 #BZBQ#</LI><LI> RBS.hps in A30 #4PAB#</LI><LI> RBS.phi in A30 #PEA6#</LI></UL><P>Products are stored in Falcon #FPTS#
</P><P><B>ligation</B></P><P>Backbone J61002.J23119 (cut with SpeI/PstI) in #TF4M# Tube1 (derived from #AHZ6#) is already digested.
</P><P>Ligate the backbone with each construct, products are stored in #YMVY#
</P><P><B>Transformation</B></P><P>Heatshock transformation in BL21; plate on LB+Amp plates.
</P><H2><SPAN class="mw-headline" id="15-07-28">15-07-28</SPAN></H2><UL><LI>Masterplates of many transformants and overnights of 6 clones each.</LI></UL><H2><SPAN class="mw-headline" id="15-07-29">15-07-29</SPAN></H2><P>Do colony PCR with masterplates with #A9W9# &amp; #XE3D# (chosen annealing temp. 58 °C). (negative with RFP 1143bp, positive Hps 939 bp, positive Phi 858 bp, positive MDH 1461 bp). 
</P><UL><LI> results of cPCR and red/white screening disagree. In both screenings positive: phi #3, hps #1,2,3,4,5</LI><LI> do NEW masterplates and overnights.</LI></UL><H2><SPAN class="mw-headline" id="15-07-30">15-07-30</SPAN></H2><P><B>Colony PCR on new masterplates</B></P><UL><LI> this time with one primer that binds on the backbone (#A9W9#) and an insert-specific reverse primer (for mdh: #MSP1#, hps:#HSP2#, phi:#PSP2#)</LI></UL><UL><LI>based on the colony PCR, overnights of mdh #4&amp;5, phi #3&amp;5 and hps #1,2,3 were used for cryos and plasmid purification. </LI></UL><UL><LI>cryo cultures: #4EMV# #3PVM#; #1MDN# #38L9#; #KL96# #B39O# #BEZO#</LI></UL><UL><LI> purified plasmids: #PS1M# #ONMP#; #LSHO# #KQ1W#; #MXYY# #1DPL# #KTZ8# </LI></UL><UL><LI>test digest of plasmids with EcoRI. Expected length of hps.J61002: 2731bp; phi.J61002: 2650; mdh.J61002: 3253bp; negative rfp.J61002: 2987 bp</LI></UL><H2><SPAN class="mw-headline" id="15-08-03">15-08-03</SPAN></H2><UL><LI>run test digest from 30th July on gel: mdh#4, Mdh#5, phi#3, phi#5, hps#1, hps#2, hps#3; all bands at expected lenght</LI></UL><UL><LI>repeat colony PCR for hps with #A9W9# and #HSP2#; expected lenght is 621 bp, chosen annealing Temp 58 °C</LI></UL><UL><LI>overnight cultures of successful clones</LI></UL><H2><SPAN class="mw-headline" id="15-08-05">15-08-05</SPAN></H2><UL><LI>Prepare three main cultures (hps #KL96#, mdh #3PVM#, phi #1MDN#). RFP control #13YN# did not grow. We have to use #BYS8#</LI></UL><UL><LI>Prepare 2 SDS (12 % for Mdh; 15 % for Hps, Phi) gels for tomorrow.</LI></UL><UL><LI>Preculture of #BYS8#</LI></UL><P>Tomorrow: Prepare cell pellets according to the protocol. Run one gel with Mdh and Rfp as control and a second gel with Hps, Phi and Mdh as control. The SDS-PAGE with Mdh and RFP as control has to be done on Friday.
</P><H2><SPAN class="mw-headline" id="15-08-06">15-08-06</SPAN></H2><UL><LI>Main culture of the Rfp control (BYS8). OD at 10.45 is 0.3. OD at 11:30 is 0.788. Take 2 2 ml samples at 17:15 and freeze the pellet.</LI></UL><UL><LI>Load SDS gel (15 %) with the 6 h-samples of Mdh (#3PVM#), Phi (#1MDN#) and Hps (#KL96#). Gel runs at 120 V for 90 minutes. Expected Mdh 40 kDa; Phi 20 kDa &amp; Hps 22.5 kDa. All samples are negative.</LI></UL><UL><LI>Take 22h samples for back-ups and freeze them.</LI></UL><UL><LI>Prepare two more SDS gels (12 % for Mdh and Rfp tomorrow and one 15 % as backup for Hps,Phi &amp; Mdh)</LI></UL><H2><SPAN class="mw-headline" id="15-08-07">15-08-07</SPAN></H2><P>Prepare suspensions with OD 0.13 of the samples (6 and 22 hours)
</P><UL><LI>Do SDS-PAGE Mdh and Rfp with 12 % acrylamid. </LI><LI>Hps, Phi &amp; Mdh with 15 % acrylamid. Results can be found in SDS-Gels 15-08-07. </LI><LI>Gel with Hps, Phi &amp; Mdh is completely negative. </LI><LI>Gel with Mdh &amp; Rfp has a very strong band in the Rfp 6 h Sample slightly abouve 26 kDa but Rfp has 25 kDa.</LI></UL><P>Prepare for sequencing: 15-08-07 Sequencing Sheet
</P><P>sequencing of #PS1M# #ONMP#; #LSHO# #KQ1W#; #MXYY# #1DPL# #KTZ8#
</P><P>Plated all four genes + RFP in pSB1A30 in BL21 Gold (DE3) cells on LB+A (#IGEM#, #4P9H#, #TFE1#, #ONK4#, #6DA3#)
</P><H3><SPAN class="mw-headline" id="sequencing_results_of_mdh.2C_phi_.26_hps_in_J61002">sequencing results of mdh, phi &amp; hps in J61002</SPAN></H3><UL><LI> #PS1M#: VALID</LI><LI> #ONMP#: VALID</LI><LI> #LSHO#: VALID</LI><LI> #KQ1W#: VALID</LI><LI> #MXYY#: VALID</LI><LI> #1DPL#: VALID</LI><LI> #KTZ8#: VALID</LI></UL><H2><SPAN class="mw-headline" id="15-08-08">15-08-08</SPAN></H2><P>Do overnight cultures from all genes in pSB1A30 (#IGEM#, #4P9H#, #TFE1#, #ONK4#, #6DA3#), cultivation startet 16:30.
</P><H2><SPAN class="mw-headline" id="15-08-09">15-08-09</SPAN></H2><P>Do 50 ml mainculture of A30 constructs (injected with 2 ml overnight culture) in LB media with ampicilin. Incubation started at 10:07. Induce with 60 µl 1000x stock IPTG at 12:15 (OD600 of samples: #IGEM# 0.79; #4P9H# 0.758; #TFE1# 0.871; #ONK4# 0.640; #6DA3# 0.310). 
</P><P>After 6 h (18:15) take 2x2 ml samples of each cultivation and again next morning.
</P><H2><SPAN class="mw-headline" id="15-08-10">15-08-10</SPAN></H2><UL><LI>Take 2x2 ml (21 h) samples of mainculture.</LI></UL><UL><LI>make two 15 % SDS-Gels.</LI></UL><P><B>Result:</B></P><P>Xpk, Hps, Phi &amp; Mdh are successfully expressed (Mdh is a bit harder to see)
</P><H2><SPAN class="mw-headline" id="expression_of_polycistronic_construct">expression of polycistronic construct</SPAN></H2><P>The expression of the polycistronic construct is <A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid" title="Team:Aachen/Lab/Methanol/Polycistronic Expression Plasmid">here</A>.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>