Team:Aachen/Notebook/Documentation/Glycogen BioBricks
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
BioBricks
Laboratory Notebook
All glycogen synthesis genes were successfully cloned into BioBricks.
BioBrick
gene
backbone
confirmed cryo
sequence-confirmed plasmid
submitted to registry
BBa_K118016
glgC
pSB1C3
#EK3S#
#4D6H#
BBa_K1585310
B0034.glgA
pSB1C3
#KMST#
#M1M6#
✓
BBa_K1585311
B0034.glgB
pSB1C3
#F4T6#
#YHT3#
✓
BBa_K1585312
B0034.glgC
pSB1C3
#DFKW#
#OP1N#
✓
BBa_K1585312
B0034.glgC
pSB1A2
#ZC8K#
#F8Y8#
15-05-11
order enzymes
Transformed BBa_K118016 from Kit Plate 2014 #3 well 20G into competent NEB10beta cells by heat shock (plated on LB+C)
gradient PCR using J04450 in pSB1C3 (#ZWDH#) as the template (diluted 1:10) primers are: #TSTV# and #FYTO# - the 10 products were put into #ES4X# and frozen
15-05-12
made master plates and liquid cultures (5 ml LB+C) of two BBa_K118016 clones
run the agarose gel of yesterdays gradient PCR products (#ES4X#) (expected length: 2070 bp)
PCR did not work - Discussion of potential reasons: tubes were heated to long before cycler program was started -> primers could have been digested
pSB1C3 gradientPCR repeated. Products stored in #18FF#
15-05-13
PCR products from #18FF# were digested with DpnI
made cryos of BBa_K118016 in pSB1C3 in DH5α (#RKD1# and #EK3S#)
pPrep BBa_K118016 in pSB1C3 (#F3HE# and #4D6H#)
purify the good pSB1C3 backbone PCR products into #6MZP# and #WHYV#
diluted glgA (#OB8B#), glgB 1+2 (#3XZX# and #AV9S#), Phi (#8D18#) gBlocks to a concentration of 10 ng/µl using 20 µl of ddH2O
CPEC assembly of #6MZP# or #WHYV# with synthesized linear DNA:
component
BBa_K1585310 (glgA)
BBa_K1585311 (glgB)
water [µl]
4
0.2
5x HF buffer [µl]
5
5
DMSO [µl]
0.75
0.75
dNTPs [µl]
5
5
Phusion polymerase
0.5
0.5
pSB1C3 backbone (#WHYV#)
2.3
2.3
fragment 1
7.4 µl #OB8B#
5.1 µl #3XZX#
fragment 2
none
6.1 µl #AV9S#
length [bp]
3525
4278
elongation time
53
1'04
The CPEC products were stored in #XY9Y#, electroporated (1.5 µl + 25 µl) into NEB10beta cells and plated on LB+C
Note: one week later we found out that there was an error in the Excel sheet - the elongation time should have been twice as long!
15-05-14
send BBa_K118016 in pSB1C3 (#F3HE# and #4D6H#) into sequencing using #A9W9# and #XE3D# at 65° annealing
CPEC results:
construct
clones
next steps
BBa_K1585310 (glgA)
0
run the CPEC product on an Agarose gel
BBa_K1585311 (glgB)
5
master plates and overnight cultures
15-05-15
all K1585311 in pSB1C3 constructs failed - the cultures and the master plates are dark red.
make cryo cultures and plasmid preps of 4xK1585311 in pSB1C3
check PCR the prepped plasmids of K1585311 in pSB1C3 using #A9W9# and #XE3D# at 65 °C annealing and 2523 bp expected product length
We can then try the CPEC again and for glgA do restriction with E+P and ligation into digested pSB1C3 in parallel - one of them will succeed.
For that amplified the gBlocks by Phusion PCR. (20 µl reactions, 2 ng template DNA, 0'30" initial denaturation, 0'30" denaturation, 0'30" annealing) The products are in #9CRS#.
gBlock amplification
template
Primer_F
Primer_R
Annealing Temperature [°C]
Product length [bp]
Elongation time
Result
glgA gBlock #OB8B#
#B8WH#
#3N6Z#
55
1535
0'46"
correct band + 3 unspecific products
glgB part 1 gBlock #3XZX#
#B8WH#
#BCAA#
55
1063
0'32"
perfect
glgB part 2 gBlock #AV9S#
#WWL1#
#3N6Z#
62
1265
0'38"
correct band + 1 unspecific product
prepare 4xK1585311 in pSB1C3 for sequencing with #A9W9# and #XE3D#
measured the concentrations of #F3HE# and #4D6H#
prepare K118016 in pSB1C3 (#F3HE# and #4D6H#) for sequencing with #A9W9# and #XE3D# at 65 °C
15-05-18
printed the K118016 sequencing sheet and send it in
Amplification of gBlocks by Phusion PCR:
conditions for 25 cycles:
step
temperature [°C]
duration
initial denaturation
98
0'30"
denaturation
98
0'30"
gradient annealing
55-67
0'30"
elongation
72
0'46"
final elongation
72
5'00"
column
1
2
3
4
5
6
7
8
9
10
11
12
temperature [°C]
54.9
55.1
55.9
57.0
58.4
59.9
61.5
63.1
64.6
65.8
66.6
67.0
3x 20 µl reactions, 20 ng template DNA
component
µl for mastermix
water
33.6
forward primer
1.5
reverse primer
1.5
HF buffer
12
dNTPs
4.8
Phusion polymerase
0.6
gBlock template
6
template
Primer_F
Primer_R
Annealing Temperature [°C]
Product length [bp]
Elongation time
Result
glgA gBlock #OB8B#
#B8WH#
#3N6Z#
55.1 (2) + 57.0 (4) + 59.9 (6)
1535
0'46"
weak band at 55.1 °C, medium bands at 57.0 and 59.9 °C, unspecific products in all
glgB part 2 gBlock #AV9S#
#WWL1#
#3N6Z#
59.9 (6) + 61.5 (7) + 63.1 (8)
1265
0'38"
medium bands at all temperatures, unspecific products of 700 bp in all
The PCR products are stored in #8K9Q#.
B0034 whole plasmid SDM of K118016 (#4D6H# and #F3HE#) to make K1585312 in pSB1C3. Q5 High-Fidelity Master Mix instead of Phusion. Products stored in #VVRL#.
15-05-19
annotate the gel pictures and fill in the results in the table of yesterdays documentation
transformation of potential K1585310 in pSB1C3 (#VVRL#) into DH5α
repeat yesterdays PCR with glgA and glgB part 2 gBlocks. This time with PfuS polymerase and higher annealing temperatures.
conditions for 25 cycles:
step
temperature [°C]
duration
initial denaturation
98
0'30"
denaturation
98
0'30"
gradient annealing
55-67
0'30"
elongation
72
0'46"
final elongation
72
5'00"
column
1
2
3
4
5
6
7
8
9
10
11
12
temperature [°C]
54.9
55.1
55.9
57.0
58.4
59.9
61.5
63.1
64.6
65.8
66.6
67.0
3x 50 µl reactions with PfuS polymerase according to the cheat sheet. We use 1 µl of gBlock template (10 ng/µl) per reaction. That's 10 ng per 50 µl reaction and completely sufficient! A slight delay was caused by using the wrong dNTP mix at first. Somehow we had more master mix volume than anticipated.
template
Primer_F
Primer_R
Annealing Temperature [°C]
Product length [bp]
Elongation time
Result
glgA gBlock #OB8B#
#B8WH#
#3N6Z#
59.9 (6) + 61.5 (7) + 63.1 (8)
1535
0'46"
two weak bands
glgB part 2 gBlock #AV9S#
#WWL1#
#3N6Z#
63.1 (8) + 64.6 (9) + 65.8 (10)
1265
0'38"
nothing
The PCR products are stored in #18BT#.
Purification of the PCR products:
fragment
from
tubes
prepped into
glgA
#18BT#
A1+A2
#BPTC#
glgB part 1
#9CRS#
B1
#KONM#
glgB part 2
#9CRS#
B2
#LO4N#
glgB part 2
#8K9Q#
B1+B2+B3
#TX1M#
15-05-20
measured the DNA concentrations in yesterdays purified PCR products (#BPTC#, #KONM#, #LO4N#, #TX1M#)
gradient PCR using #TSTV# and #FYTO# on 45 ng J04450 in pSB1C3 (#ZDWH#) per 50 µl PfuS reaction mix. Annealing temperatures 65-68 °C, 1'15" elongation time. Products stored in #YDTH# - on the gel it looks like 68 °C is really good, 67.3 has multiple unspecific products, but for an unknown reason the lower temperatures did not work.
pSB1C3 PCR product purification (#6WCN#)
Restrictions (2x the concentration of T4 Ligase due to human error.)
digestion of amplified glgA (#BPTC#) with EcoRI and PstI -> product is I in #CQ1V#
digestion of prepped pSB1C3 (#CD8B#) with EcoRI and PstI -> product is V in #CQ1V#
Ligation of 2 µl digested glgA and 2 µl digested pSB1C3
Q5-CPEC of glgB part 1 (#KONM#), glgB part 2 (#LO4N#) and linearized pSB1C3 (#6WCN#) -> product is B in #4A4P#
98:0'30", 15x(98:0'10", 55:0'30", 72:2'20"), 72:10'00"
transformation of 5 µl glgA ligation product into 50 µl home made chemically competent NEB10b + 200 µl SOC
transformation of 5 µl glgB CPEC product into 50 µl commercial chemically competent NEB5a + 200 µl SOC
after 60' of incubation at 37 °C the total volume of each transformation was plated on LB+C
master plates (four clones, one clone from each transformation plate) and overnight cultures (LB+C) of K1585312 in pSB1C3 (sequencing results: #F3HE# and #4D6H# are 100 % correct)
15-05-21
made cryos and pPrep of K1585312 in pSB1C3
clone number
cryo
purified plasmid
1
#A11C#
#OO3E#
2
#HNOX#
#DV8N#
3
#ENOA#
#P4DP#
4
#PM4P#
#X8TM#
sent K1585310 in pSB1C3 into sequencing with #A9W9# at 63 °C
master plates and overnight cultures LB+C of glgA and glgB clones
15-05-22
ordered sequencing primers
made LB+C plates and new LB medium
cryos of white glgA and glgB clones
plasmid prep of white glgA and glgB clones
Name / Clone Number
cryo
purified plasmid
sequencing result (27.05.)
glgA #1
#FYX1#
#6D83#
mRFP insert
glgB #1
#DVLR#
#4L6C#
ugly/contaminated
glgB #2
#F4T6#
#YHT3#
perfect
glgB #3
#CX98#
#3BQH#
non-silent C>T point mutation (P>S)
glgB #4
#Q8D6#
#AOMO#
non-silent G>T point mutation (G>C)
glgB #5
#4YEX#
not yet prepped
(not yet sequenced, but colony PCR looks good)
15-05-26
make new master plates of all remaining white colonies of glgA and some of glgB (for glgA also a few overnights?)
15-05-27
moved master plates of glgA and B into the fridge
cryos of glgA clones, freeze remaining culture for plasmid prep -> all glgA clones were negative on the colony PCR
analyze sequencing results (glgA is without the insert, glgB looks good for #YHT3#, but the sequencing did not reach the center - as expected)
prepared #YHT3# for sequencing with multiple new primers
added 1 µl to both PCR products in #VVRL# and put it into the 37 °C room for overnight digest
15-05-28
trash all existing glgA cryos and clones, because all of them are negative (sequencing/colony PCR)
the glgC DpnI digest (#VVRL#) is back in the freezer
re-do the amplification of glgA from existing gBlock DNA (#OB8B#) with the primers #B8WH# and #3N6Z# (iJR)
5x 50 µl PfuS PCR reaction
expected: 1535 bp
products in #64LF#
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
98
0'30"
denaturation
98
0'30"
gradient annealing
61-63.5
0'30"
elongation
72
0'50"
final elongation
72
5'00"
purify good glgA products into #F1VM# -> concentration is awful (19.5 ng/µl)
digest purified glgA product with EcoRI and PstI (product: #????#)
ligate #CQ1V#:I with pSB1C3 (#CQ1V#:V) (6 µl insert, 2 µl backbone) => product is #QZFH#:A
CPEC assembly of glgA
component
volume [µl]
water
10.7
Q5 buffer
5
DMSO
0.75
dNTPs (2 mM)
5
Q5 polymerase
0.5
pSB1C3 fragment (#6WCN#)
0.8
#BPTC#
2.3
elongation time
1'46"
product stored in
#PXR4#
Heat shock transformation of CPEC product (#PXR4#:A), ligation product (#QZFH#) and glgC digested ligation (#VVRL#) into DH5α cells
trafo program in the cycler: 4 °C for 30', 42 °C for 60", 4 °C for 5'
number
description
id
1
glgA ligation
#OZFH#:A
2
glgA CPEC
#PXR4#:A
3
Hps ligation
#SEXY#:JR
4
glgC re-digested PCR product
#VVRL#:1
15-05-29
Testing last weeks clones of glgA and glgB
colony PCR of re-plated glgA clones (done by iJR and iCG) showed all negative
colony PCR of re-plated glgB clones (done by iJR and iCG) showed #15, #17, #18, #24 positive
overnight cultures of glgB clones #15, #17, #18, #24
Colony PCR glgB
colony PCR of re-plated glgB clones showed #15, #17, #18, #24 positive
Colony PCR glgB
colony PCR of re-plated glgB clones showed #15, #17, #18, #24 positive
Working with new clones
construct
colonies
red
glgA Ligation
11
45 %
glgA CPEC
1140
50 %
glgC
6
0 %
master plates of glgA Ligation/CPEC clones
15-05-30
cryos of glgB clones
gene
clone number
cryo-id
plasmid-id
sequencing result
glgB
15
#AWCC#
#ONXK#
S>Y mutation at position 2879
glgB
17
#BOO9#
#AC85#
part of the sequence looks contaminated
glgB
18
#ZKEA#
not prepped yet
glgB
24
#RVHD#
not prepped yet
colony PCR
template
clone numbers
expected
good clones
glgA ligated
1, 2, 3, 6, 7, 8
1770
2, 3, 6, 7
glgA CPEC
1-12
1770
4 (?)
glgC
1-6
1632
1, 2, 3, 5, 6
glgB
5
2523
yes
15-05-31
overnight cultures
construct
clone number
cryo-id
plasmid-id
sequencing result
glgA ligated
2
#KMST#
#M1M6#
perfect
glgA ligated
3
#8PFH#
#B64M#
perfect
glgA ligated
6
#FHPK#
glgA ligated
7
#4HKH#
glgA CPEC
4
#VWLO#
#6SHZ#
contaminated with two versions
glgC
1
#SOD9#
#PZNS#
negative
glgC
2
#YAPZ#
#6QC3#
negative
glgC
3
#XBNP#
#YWRR#
negative
glgC
5
#SDP4#
#AET8#
negative
glgC
6
#X8Q1#
#LQCS#
negative
15-06-03
sequencing results analyzed:
glgA: #B64M# and #M1M6# are perfect
glgB: #YHT3# is perfect
glgC: did not work yet
heat shock of B0034 in pSB1A2 from KitPlate 2015.4 1N into NEB10b and plating on LB+A
15-06-04
4 overnight and master plate of B0034 in pSB1A2
15-06-05
cryos von B0034 in pSB1A2 in NEB10b
By accident the cultures were inoculated with two different clones.
freeze 4 cultures for later pPrep
colony PCR clones 1, 2, 3 from the master plate using #A9W9# and #XE3D#. Expected product length: 251 bp
gel picture looks perfect
plasmid prep of one good clone
pipet the plasmid for sequencing
15-06-07
inoculation of one LB+A overnight culture
new overnights from master plates (clone 1, 2 and 3)
15-06-08
cryos
plasmid prep from overnight #1
freeze #2 and #3
cut B0034 and glgC biobrick (BBa_K118016)
glgC from #4D6H# mit Xba I + Pst
B0034 from #4EX9# mit EcoRI + Spe
pSB1K3 from #AQY3# mit EcoRI + Pst
15-06-09
ligate B0034 and glgC in pSB1K3, product in #SY86#
trafo of B0034.glgC in pSB1K3 in DH5α per heat shock (#SY86#)
15-06-10
overnights of B0034.glgC in pSB1K3
15-06-11
cryos and plasmid prep of B0034.glgC in pSB1K3
clone number
cryo ID
plasmid ID
sequencing result
B0034.glgC in pSB1K3 #1
#66ZZ#
#WMVZ#
no RBS
B0034.glgC in pSB1K3 #2
#333V#
#YSVR#
no RBS
B0034.glgC in pSB1K3 #3
#C9TN#
was red
-
B0034.glgC in pSB1K3 #4
#SSOL#
#TDS1#
no RBS
B0034.glgC in pSB1K3 #5
#Z8BB#
#HNEA#
no RBS
pipette B0034.glgC in pSB1K3 for sequencing
15-06-17
Digestions with 40' digest and 20' inactivation
gene
enzymes
expectation
observation
products
glgC (#4D6H#)
XbaI + PstI
...
not run
#OHOZ#:C
B0034 (#4EX9#)
SpeI + PstI
...
not run
#OHOZ#:R
Ligations and transformations
construct
part I
part II
backbone
product
host
B0034.glgC in pSB1A2
#OHOZ#:R
#OHOZ#:C
-
#OKNR#:G
DH5α
15-06-18
make master plates and liquid cultures of transformation (B0034.glgC in pSB1A2; 6 liquid, 12 plate)
15-06-19
Colony PCR on glgC construct
gene / clones
tube number
expected length
machine
elongation time
result
glgC in pSB1A2 clones 1-12
25-36
1556
2
1'50"
all positive
Cryos
gene / clone
cryo ID
plasmid ID
sequencing result
glgC in pSB1A2 #1
#9F3M#
#SMTS#
perfect
glgC in pSB1A2 #2
#ZC8K#
#F8Y8#
perfect
glgC in pSB1A2 #3
#YV1C#
don't prep
-
glgC in pSB1A2 #4
#D3E6#
don't prep
-
glgC in pSB1A2 #5
#RTWO#
don't prep
-
glgC in pSB1A2 #6
#TN6K#
don't prep
-
15-06-22
prep good plasmids of glgA and glgC (table above)
pipette glgA in pSB1K30 and B0034.glgC for sequencing (#A9W9# and #XE3D#)
15-06-24
analysis of sequencing results
15-06-25
discard excess B0034.glgC in pSB1A2 cryos and plasmids
15-06-26
Digestions
BioBrick in Backbone
gene
template ID
enzymes
product
purpose
K1585312 in pSB1A2
glgC
#F8Y8#
EcoRI, PstI
#LVVK#:1
subcloning into pSB1C3, pSB1K30
15-06-29
Ligation and transformation
construct
gene
Part I
Part II
Backbone
products
host
purpose
K1585312 in pSB1C3
glgC
#LVVK#:1
-
#CQ1V#:V
#RRCV#:1
DH5α
for part submission
15-06-30
we have clones of K1585312 into pSB1C3 (red-white screening)
made master plate (6 clones) and overnight cultures (3 cultures)
15-07-01
cryos and freezings of the three K1585312 in pSB1C3 cultures
colony PCR on all 6 clones > clone 1 and 2 look good
gene / clone
cryo ID
plasmid ID
sequencing result
B0034.glgC in pSB1C3 #1
#4XPZ#
#DRPT#
glgA instead of glgC
B0034.glgC in pSB1C3 #2
#1NZB#
#EZOB#
glgA instead of glgC
B0034.glgC in pSB1C3 #3
#FCSS#
no prep
-
pipetted for sequencing
15-07-03
overnight culture of glgC in pSB1A2 from #ZC8K# in LB+A
15-07-06
PCR 1 to subclone glgC into pSB1C3
Backbone amplification:
component
template
forward primer
reverse primer
product length
purified  pcr product ID
product ID
tube #
J04450 in pSB1C3
#CD8B#
#TSTV#
#FYTO#
2070
#ZV3C#
#MRVW#:II
1+2
PfuS PCR
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
98
2'00"
denaturation
98
0'30"
annealing
64
0'30"
elongation
72
1'10"
final elongation
72
5'00"
15-07-08
PCR 2 to subclone glgC into pSB1C3
Insert amplification:
component
template
forward primer
reverse primer
product length
product ID
glgC in pSB1A2
#F8Y8#
#O3M6#
#NBZZ#
1360
#MRVW#: I
15-07-09
PCR product purification
PCR product
purified PCR product
#MRVW#:I
#XNHT#
#MRVW#:II
#ZV3C#
CPEC assembly of glgC in pSB1C3
component
volume [µl]
water
5.9
Q5 buffer
5
DMSO
0.75
dNTPs (2 mM)
5
Q5 polymerase
0.5
pSB1C3 fragment (#ZV3C#)
2.7
glgC with prefix/suffix #XNHT#
5.2
elongation time
1'42" (1'47")
product stored in
#SWMB#:I
5 µl #SWMB#:I were heat shocked into DH5α and plated on LB+C
15-07-10
make masterplates (LB+C) and overnights of glgC in pSB1C3
15-07-11
cryo cultures
freeze culture
gene / clone
cryo ID
plasmid ID
sequencing result
B0034.glgC in pSB1C3 #1
#ESAY#
#QXBD#
good, but not perfect
B0034.glgC in pSB1C3 #2
#DFKW#
#OP1N#
perfect
B0034.glgC in pSB1C3 #3
#VW9S#
-
-
B0034.glgC in pSB1C3 #4
#L81A#
#MCM6#
-
B0034.glgC in pSB1C3 #5
#SOVA#
#NPCT#
contaminated chromatogram
B0034.glgC in pSB1C3 #6
#Q9NL#
-
-
15-07-13
plasmid prep of glgC in pSB1C3 (see table above)
15-07-15
analysing sequencing results (see table above)
References
▲
Contact
CC-BY-SA iGEM Aachen 2015
