<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Glycogen_BioBricks skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Glycogen BioBricks</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmGlycogenSynthesisI" href="/Team:Aachen/Lab/Glycogen/Synthesis/I._Biobricks"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">BioBricks</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><P>All glycogen synthesis genes were successfully cloned into BioBricks.
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick </TH><TH> gene </TH><TH> backbone </TH><TH> confirmed cryo </TH><TH> sequence-confirmed plasmid </TH><TH> submitted to registry
</TH></TR><TR><TD> BBa_K118016 </TD><TD> glgC </TD><TD> pSB1C3 </TD><TD> #EK3S# </TD><TD> #4D6H# </TD></TR><TR><TD> BBa_K1585310 </TD><TD> B0034.glgA </TD><TD> pSB1C3 </TD><TD> #KMST# </TD><TD> #M1M6# </TD><TD>  ✓
</TD></TR><TR><TD> BBa_K1585311 </TD><TD> B0034.glgB </TD><TD> pSB1C3 </TD><TD> #F4T6# </TD><TD> #YHT3# </TD><TD>  ✓
</TD></TR><TR><TD> BBa_K1585312 </TD><TD> B0034.glgC </TD><TD> pSB1C3 </TD><TD> #DFKW# </TD><TD> #OP1N# </TD><TD>  ✓
</TD></TR><TR><TD> BBa_K1585312 </TD><TD> B0034.glgC </TD><TD> pSB1A2 </TD><TD> #ZC8K# </TD><TD> #F8Y8# </TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-11">15-05-11</SPAN></H2><UL><LI><B>order enzymes</B></LI><LI> Transformed BBa_K118016 from <B>Kit Plate 2014 #3 well 20G</B> into competent NEB10beta cells by heat shock (plated on LB+C)</LI><LI> gradient PCR using J04450 in pSB1C3 (#ZWDH#) as the template (diluted 1:10) primers are: #TSTV# and #FYTO# - the 10 products were put into #ES4X# and frozen</LI></UL><H2><SPAN class="mw-headline" id="15-05-12">15-05-12</SPAN></H2><UL><LI> made master plates and liquid cultures (5 ml LB+C) of two BBa_K118016 clones</LI><LI> run the agarose gel of yesterdays gradient PCR products (#ES4X#) (expected length: 2070 bp)
</LI><LI><B> PCR did not work</B> - Discussion of potential reasons: tubes were heated to long before cycler program was started -&gt; primers could have been digested</LI></UL><LI> pSB1C3 gradientPCR repeated. Products stored in #18FF#</LI><H2><SPAN class="mw-headline" id="15-05-13">15-05-13</SPAN></H2><UL><LI> PCR products from #18FF# were digested with DpnI</LI><LI> made cryos of BBa_K118016 in pSB1C3 in DH5α (#RKD1# and #EK3S#)</LI><LI> pPrep BBa_K118016 in pSB1C3 (#F3HE# and #4D6H#)</LI><LI> purify the good pSB1C3 backbone PCR products into #6MZP# and #WHYV#</LI><LI> diluted glgA (#OB8B#), glgB 1+2 (#3XZX# and #AV9S#), Phi (#8D18#) gBlocks to a concentration of 10 ng/µl using 20 µl of ddH<SUB>2</SUB>O</LI><LI><A href="/Team:Aachen/Notebook/Protocols#CPEC" title="Team:Aachen/Notebook/Protocols">CPEC</A> assembly of #6MZP# or #WHYV# with synthesized linear DNA:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> BBa_K1585310 (glgA) </TH><TH> BBa_K1585311 (glgB)
</TH></TR><TR><TD> water [µl] </TD><TD> 4 </TD><TD><S>0.2</S></TD></TR><TR><TD> 5x HF buffer [µl] </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> DMSO [µl] </TD><TD> 0.75 </TD><TD> 0.75
</TD></TR><TR><TD> dNTPs [µl] </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> Phusion polymerase </TD><TD> 0.5 </TD><TD> 0.5
</TD></TR><TR><TD> pSB1C3 backbone (#WHYV#) </TD><TD> 2.3 </TD><TD> 2.3
</TD></TR><TR><TD> fragment 1 </TD><TD> 7.4 µl #OB8B# </TD><TD> 5.1 µl #3XZX#
</TD></TR><TR><TD> fragment 2 </TD><TD> none </TD><TD> 6.1 µl #AV9S#
</TD></TR><TR><TD> length [bp] </TD><TD> 3525 </TD><TD> 4278
</TD></TR><TR><TD> elongation time </TD><TD> 53</TD><TD> 1'04
</TD></TR></TBODY></TABLE><P>The CPEC products were stored in #XY9Y#, electroporated (1.5 µl + 25 µl) into NEB10beta cells and plated on LB+C
</P><P><B>Note:</B> one week later we found out that there was an error in the Excel sheet - the elongation time should have been twice as long!
</P><H2><SPAN class="mw-headline" id="15-05-14">15-05-14</SPAN></H2><UL><LI> send BBa_K118016 in pSB1C3 (#F3HE# and #4D6H#) into sequencing using #A9W9# and #XE3D# at 65° annealing</LI><LI> CPEC results:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> clones </TH><TH> next steps
</TH></TR><TR><TD> BBa_K1585310 (glgA) </TD><TD> 0 </TD><TD> run the CPEC product on an Agarose gel
</TD></TR><TR><TD> BBa_K1585311 (glgB) </TD><TD> 5 </TD><TD> master plates and overnight cultures
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-15">15-05-15</SPAN></H2><UL><LI> all K1585311 in pSB1C3 constructs failed - the cultures and the master plates are dark red.</LI><LI><S>make cryo cultures and plasmid preps of 4xK1585311 in pSB1C3</S></LI><LI><S>check PCR the prepped plasmids of K1585311 in pSB1C3 using #A9W9# and #XE3D# at 65 °C annealing and 2523 bp expected product length</S></LI></UL><P>We can then try the CPEC again and for glgA do restriction with E+P and ligation into digested pSB1C3 in parallel - one of them will succeed.
</P><P>For that amplified the gBlocks by Phusion PCR. (20 µl reactions, 2 ng template DNA, 0'30&quot; initial denaturation, 0'30&quot; denaturation, 0'30&quot; annealing) The products are in #9CRS#.
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>gBlock amplification</B></TD></TR></TBODY></TABLE></DIV><TABLE class="wikitable"><TBODY><TR><TH> template </TH><TH> Primer_F </TH><TH> Primer_R </TH><TH> Annealing Temperature [°C] </TH><TH> Product length [bp] </TH><TH> Elongation time </TH><TH> Result
</TH></TR><TR><TD> glgA gBlock #OB8B# </TD><TD> #B8WH# </TD><TD> #3N6Z# </TD><TD> 55 </TD><TD> 1535 </TD><TD> 0'46&quot; </TD><TD> correct band + 3 unspecific products
</TD></TR><TR><TD> glgB part 1 gBlock #3XZX# </TD><TD> #B8WH# </TD><TD> #BCAA# </TD><TD> 55 </TD><TD> 1063 </TD><TD> 0'32&quot; </TD><TD> perfect
</TD></TR><TR><TD> glgB part 2 gBlock #AV9S# </TD><TD> #WWL1# </TD><TD> #3N6Z# </TD><TD> 62 </TD><TD> 1265 </TD><TD> 0'38&quot; </TD><TD> correct band + 1 unspecific product
</TD></TR></TBODY></TABLE><UL><LI><S>prepare 4xK1585311 in pSB1C3 for sequencing with #A9W9# and #XE3D#</S></LI><LI> measured the concentrations of #F3HE# and #4D6H#</LI><LI> prepare K118016 in pSB1C3 (#F3HE# and #4D6H#) for sequencing with #A9W9# and #XE3D# at 65 °C</LI></UL><H2><SPAN class="mw-headline" id="15-05-18">15-05-18</SPAN></H2><UL><LI> printed the K118016 sequencing sheet and send it in</LI></UL><P>Amplification of gBlocks by Phusion PCR:
</P><P>conditions for 25 cycles:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 55-67 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'46&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> column </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> 7 </TH><TH> 8 </TH><TH> 9 </TH><TH> 10 </TH><TH> 11 </TH><TH> 12
</TH></TR><TR><TD> temperature [°C] </TD><TD> 54.9 </TD><TD> 55.1 </TD><TD> 55.9 </TD><TD> 57.0 </TD><TD> 58.4 </TD><TD> 59.9 </TD><TD> 61.5 </TD><TD> 63.1 </TD><TD> 64.6 </TD><TD> 65.8 </TD><TD> 66.6 </TD><TD> 67.0
</TD></TR></TBODY></TABLE><P>3x 20 µl reactions, 20 ng template DNA
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> µl for mastermix
</TH></TR><TR><TD> water </TD><TD> 33.6
</TD></TR><TR><TD> forward primer </TD><TD> 1.5
</TD></TR><TR><TD> reverse primer </TD><TD> 1.5
</TD></TR><TR><TD> HF buffer </TD><TD> 12
</TD></TR><TR><TD> dNTPs </TD><TD> 4.8
</TD></TR><TR><TD> Phusion polymerase </TD><TD> 0.6
</TD></TR><TR><TD> gBlock template </TD><TD> 6
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> template </TH><TH> Primer_F </TH><TH> Primer_R </TH><TH> Annealing Temperature [°C] </TH><TH> Product length [bp] </TH><TH> Elongation time </TH><TH> Result
</TH></TR><TR><TD> glgA gBlock #OB8B# </TD><TD> #B8WH# </TD><TD> #3N6Z# </TD><TD> 55.1 (2) + 57.0 (4) + 59.9 (6) </TD><TD> 1535 </TD><TD> 0'46&quot; </TD><TD> weak band at 55.1 °C, medium bands at 57.0 and 59.9 °C, unspecific products in all
</TD></TR><TR><TD> glgB part 2 gBlock #AV9S# </TD><TD> #WWL1# </TD><TD> #3N6Z# </TD><TD> 59.9 (6) + 61.5 (7) + 63.1 (8) </TD><TD> 1265 </TD><TD> 0'38&quot; </TD><TD> medium bands at all temperatures, unspecific products of 700 bp in all
</TD></TR></TBODY></TABLE><P>The PCR products are stored in #8K9Q#.
</P><UL><LI><A href="/wiki/index.php?title=Manual/B0034_Insertion-Mutagenesis&amp;action=edit&amp;redlink=1" class="new" title="Manual/B0034 Insertion-Mutagenesis (page does not exist)">B0034 whole plasmid SDM</A> of K118016 (#4D6H# and #F3HE#) to make K1585312 in pSB1C3. Q5 High-Fidelity Master Mix instead of Phusion. Products stored in #VVRL#.</LI></UL><H2><SPAN class="mw-headline" id="15-05-19">15-05-19</SPAN></H2><UL><LI> annotate the gel pictures and fill in the results in the table of yesterdays documentation</LI><LI> transformation of potential K1585310 in pSB1C3 (#VVRL#) into DH5α</LI><LI> repeat yesterdays PCR with glgA and glgB part 2 gBlocks. This time with PfuS polymerase and higher annealing temperatures.</LI></UL><P>conditions for <B>25 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 55-67 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'46&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH> column </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> 7 </TH><TH> 8 </TH><TH> 9 </TH><TH> 10 </TH><TH> 11 </TH><TH> 12
</TH></TR><TR><TD> temperature [°C] </TD><TD> 54.9 </TD><TD> 55.1 </TD><TD> 55.9 </TD><TD> 57.0 </TD><TD> 58.4 </TD><TD> 59.9 </TD><TD> 61.5 </TD><TD> 63.1 </TD><TD> 64.6 </TD><TD> 65.8 </TD><TD> 66.6 </TD><TD> 67.0
</TD></TR></TBODY></TABLE><P>3x 50 µl reactions with PfuS polymerase according to the cheat sheet. We use 1 µl of gBlock template (10 ng/µl) per reaction. That's 10 ng per 50 µl reaction and completely sufficient! A slight delay was caused by using the wrong dNTP mix at first. Somehow we had more master mix volume than anticipated.
</P><TABLE class="wikitable"><TBODY><TR><TH> template </TH><TH> Primer_F </TH><TH> Primer_R </TH><TH> Annealing Temperature [°C] </TH><TH> Product length [bp] </TH><TH> Elongation time </TH><TH> Result
</TH></TR><TR><TD> glgA gBlock #OB8B# </TD><TD> #B8WH# </TD><TD> #3N6Z# </TD><TD> 59.9 (6) + 61.5 (7) + 63.1 (8) </TD><TD> 1535 </TD><TD> 0'46&quot; </TD><TD> two weak bands
</TD></TR><TR><TD> glgB part 2 gBlock #AV9S# </TD><TD> #WWL1# </TD><TD> #3N6Z# </TD><TD> 63.1 (8) + 64.6 (9) + 65.8 (10) </TD><TD> 1265 </TD><TD> 0'38&quot; </TD><TD> nothing
</TD></TR></TBODY></TABLE><P>The PCR products are stored in #18BT#.
</P><P>Purification of the PCR products:
</P><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> from </TH><TH> tubes </TH><TH> prepped into
</TH></TR><TR><TD> glgA </TD><TD> #18BT# </TD><TD> A1+A2 </TD><TD> #BPTC#
</TD></TR><TR><TD> glgB part 1 </TD><TD> #9CRS# </TD><TD> B1 </TD><TD> #KONM#
</TD></TR><TR><TD> glgB part 2 </TD><TD> #9CRS# </TD><TD> B2 </TD><TD> #LO4N#
</TD></TR><TR><TD> glgB part 2 </TD><TD> #8K9Q# </TD><TD> B1+B2+B3 </TD><TD> #TX1M#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-20">15-05-20</SPAN></H2><UL><LI> measured the DNA concentrations in yesterdays purified PCR products (#BPTC#, #KONM#, #LO4N#, #TX1M#)</LI><LI> gradient PCR using #TSTV# and #FYTO# on 45 ng J04450 in pSB1C3 (#ZDWH#) per 50 µl PfuS reaction mix. Annealing temperatures 65-68 °C, 1'15&quot; elongation time. Products stored in #YDTH# - on the gel it looks like 68 °C is really good, 67.3 has multiple unspecific products, but for an unknown reason the lower temperatures did not work.</LI><LI> pSB1C3 PCR product purification (#6WCN#)</LI><LI> Restrictions (2x the concentration of T4 Ligase due to human error.)
</LI><LI> digestion of amplified glgA (#BPTC#) with EcoRI and PstI -&gt; product is I in #CQ1V#</LI><LI> digestion of prepped pSB1C3 (#CD8B#) with EcoRI and PstI -&gt; product is V in #CQ1V#</LI></UL><LI> Ligation of 2 µl digested glgA and 2 µl digested pSB1C3</LI><LI> Q5-CPEC of glgB part 1 (#KONM#), glgB part 2 (#LO4N#) and linearized pSB1C3 (#6WCN#) -&gt; product is B in #4A4P#
</LI><LI> 98:0'30&quot;, 15x(98:0'10&quot;, 55:0'30&quot;, 72:2'20&quot;), 72:10'00&quot;</LI><LI> transformation of 5 µl glgA ligation product into 50 µl home made chemically competent NEB10b + 200 µl SOC</LI><LI> transformation of 5 µl glgB CPEC product into 50 µl commercial chemically competent NEB5a + 200 µl SOC</LI><LI> after 60' of incubation at 37 °C the total volume of each transformation was plated on LB+C</LI><UL><LI> master plates (four clones, one clone from each transformation plate) and overnight cultures (LB+C) of K1585312 in pSB1C3 (sequencing results: #F3HE# and #4D6H# are 100 % correct)</LI></UL><H2><SPAN class="mw-headline" id="15-05-21">15-05-21</SPAN></H2><UL><LI> made cryos and pPrep of K1585312 in pSB1C3  </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone number </TH><TH> cryo </TH><TH> purified plasmid
</TH></TR><TR><TD> 1 </TD><TD> #A11C# </TD><TD> #OO3E#
</TD></TR><TR><TD> 2 </TD><TD> #HNOX# </TD><TD> #DV8N#
</TD></TR><TR><TD> 3 </TD><TD> #ENOA# </TD><TD> #P4DP#
</TD></TR><TR><TD> 4 </TD><TD> #PM4P# </TD><TD> #X8TM#
</TD></TR></TBODY></TABLE><UL><LI> sent K1585310 in pSB1C3 into sequencing with #A9W9# at 63 °C</LI><LI> master plates and overnight cultures LB+C of glgA and glgB clones</LI></UL><H2><SPAN class="mw-headline" id="15-05-22">15-05-22</SPAN></H2><UL><LI> ordered sequencing primers</LI><LI> made LB+C plates and new LB medium</LI><LI> cryos of white glgA and glgB clones</LI><LI> plasmid prep of white glgA and glgB clones</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Name / Clone Number </TH><TH> cryo </TH><TH> purified plasmid </TH><TH> sequencing result (27.05.)
</TH></TR><TR><TD> glgA #1 </TD><TD> #FYX1# </TD><TD> #6D83# </TD><TD> mRFP insert
</TD></TR><TR><TD> glgB #1 </TD><TD> #DVLR# </TD><TD> #4L6C# </TD><TD> ugly/contaminated
</TD></TR><TR><TD> glgB #2 </TD><TD> #F4T6# </TD><TD> #YHT3# </TD><TD> perfect
</TD></TR><TR><TD> glgB #3 </TD><TD> #CX98# </TD><TD> #3BQH# </TD><TD> non-silent C&gt;T point mutation (P&gt;S)
</TD></TR><TR><TD> glgB #4 </TD><TD> #Q8D6# </TD><TD> #AOMO# </TD><TD> non-silent G&gt;T point mutation (G&gt;C)
</TD></TR><TR><TD> glgB #5 </TD><TD> #4YEX# </TD><TD> not yet prepped </TD><TD> (not yet sequenced, but colony PCR looks good)
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-26">15-05-26</SPAN></H2><UL><LI> make new master plates of all remaining white colonies of glgA and some of glgB (for glgA also a few overnights?)</LI></UL><H2><SPAN class="mw-headline" id="15-05-27">15-05-27</SPAN></H2><UL><LI> moved master plates of glgA and B into the fridge</LI><LI> cryos of glgA clones, freeze remaining culture for plasmid prep -&gt; all glgA clones were negative on the colony PCR</LI><LI> analyze sequencing results (glgA is without the insert, glgB looks good for #YHT3#, but the sequencing did not reach the center - as expected)</LI><LI> prepared #YHT3# for sequencing with multiple new primers</LI><LI> added 1 µl to both PCR products in #VVRL# and put it into the 37 °C room for overnight digest</LI></UL><H2><SPAN class="mw-headline" id="15-05-28">15-05-28</SPAN></H2><UL><LI> trash all existing glgA cryos and clones, because all of them are negative (sequencing/colony PCR)</LI><LI> the glgC DpnI digest (#VVRL#) is back in the freezer</LI><LI> re-do the amplification of glgA from existing gBlock DNA (#OB8B#) with the primers #B8WH# and #3N6Z# (iJR)
</LI><LI> 5x 50 µl PfuS PCR reaction</LI><LI> expected: 1535 bp</LI><LI> products in #64LF#</LI></UL><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 61-63.5 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'50&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> purify good glgA products into #F1VM# -&gt; concentration is awful (19.5 ng/µl)</LI><LI><S>digest purified glgA product with EcoRI and PstI (product: #????#)</S></LI><LI> ligate #CQ1V#:I with pSB1C3 (#CQ1V#:V) (6 µl insert, 2 µl backbone) =&gt; product is #QZFH#:A</LI></UL><P>CPEC assembly of glgA
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [µl]
</TH></TR><TR><TD> water </TD><TD> 10.7
</TD></TR><TR><TD> Q5 buffer </TD><TD> 5
</TD></TR><TR><TD> DMSO </TD><TD> 0.75
</TD></TR><TR><TD> dNTPs (2 mM) </TD><TD> 5
</TD></TR><TR><TD> Q5 polymerase </TD><TD> 0.5
</TD></TR><TR><TD> pSB1C3 fragment (#6WCN#) </TD><TD> 0.8
</TD></TR><TR><TD> #BPTC# </TD><TD> 2.3
</TD></TR><TR><TD> elongation time </TD><TD> 1'46&quot;
</TD></TR><TR><TD> product stored in </TD><TD> #PXR4#
</TD></TR></TBODY></TABLE><UL><LI> Heat shock transformation of CPEC product (#PXR4#:A), ligation product (#QZFH#) and glgC digested ligation (#VVRL#) into DH5α cells
</LI><LI> trafo program in the cycler: 4 °C for 30', 42 °C for 60&quot;, 4 °C for 5'</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> number </TH><TH> description </TH><TH> id
</TH></TR><TR><TD> 1 </TD><TD> glgA ligation </TD><TD> #OZFH#:A
</TD></TR><TR><TD> 2 </TD><TD> glgA CPEC </TD><TD> #PXR4#:A
</TD></TR><TR><TD> 3 </TD><TD> Hps ligation </TD><TD> #SEXY#:JR
</TD></TR><TR><TD> 4 </TD><TD> glgC re-digested PCR product </TD><TD> #VVRL#:1
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-29">15-05-29</SPAN></H2><P>Testing last weeks clones of glgA and glgB
</P><UL><LI> colony PCR of re-plated glgA clones (done by iJR and iCG) showed all negative</LI><LI> colony PCR of re-plated glgB clones (done by iJR and iCG) showed #15, #17, #18, #24 positive</LI><LI> overnight cultures of glgB clones #15, #17, #18, #24</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Colony PCR glgB</B>colony PCR of re-plated glgB clones showed #15, #17, #18, #24 positive
</TD></TR></TBODY></TABLE></DIV><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Colony PCR glgB</B>colony PCR of re-plated glgB clones showed #15, #17, #18, #24 positive
</TD></TR></TBODY></TABLE></DIV><P>Working with new clones
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> colonies </TH><TH> red
</TH></TR><TR><TD> glgA Ligation </TD><TD> 11 </TD><TD> 45 %
</TD></TR><TR><TD> glgA CPEC </TD><TD> 1140 </TD><TD> 50 %
</TD></TR><TR><TD> glgC </TD><TD> 6 </TD><TD> 0 %
</TD></TR></TBODY></TABLE><UL><LI> master plates of glgA Ligation/CPEC clones</LI></UL><H2><SPAN class="mw-headline" id="15-05-30">15-05-30</SPAN></H2><UL><LI> cryos of glgB clones</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene </TH><TH> clone number </TH><TH> cryo-id </TH><TH> plasmid-id </TH><TH> sequencing result
</TH></TR><TR><TD> glgB </TD><TD> 15 </TD><TD> #AWCC# </TD><TD> #ONXK# </TD><TD> S&gt;Y mutation at position 2879
</TD></TR><TR><TD> glgB </TD><TD> 17 </TD><TD> #BOO9# </TD><TD> #AC85# </TD><TD> part of the sequence looks contaminated
</TD></TR><TR><TD> glgB </TD><TD> 18 </TD><TD> #ZKEA# </TD><TD> not prepped yet </TD></TR><TR><TD> glgB </TD><TD> 24 </TD><TD> #RVHD# </TD><TD> not prepped yet </TD></TR></TBODY></TABLE><UL><LI> colony PCR</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> template </TH><TH> clone numbers </TH><TH> expected </TH><TH> good clones
</TH></TR><TR><TD> glgA ligated </TD><TD> 1, 2, 3, 6, 7, 8 </TD><TD> 1770 </TD><TD> 2, 3, 6, 7
</TD></TR><TR><TD> glgA CPEC </TD><TD> 1-12 </TD><TD> 1770 </TD><TD> 4 (?)
</TD></TR><TR><TD> glgC </TD><TD> 1-6 </TD><TD> 1632 </TD><TD> 1, 2, 3, 5, 6
</TD></TR><TR><TD> glgB </TD><TD> 5 </TD><TD> 2523 </TD><TD> yes
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-31">15-05-31</SPAN></H2><UL><LI> overnight cultures</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> clone number </TH><TH> cryo-id </TH><TH> plasmid-id </TH><TH> sequencing result
</TH></TR><TR><TD> glgA ligated </TD><TD> 2 </TD><TD> #KMST# </TD><TD> #M1M6# </TD><TD> perfect
</TD></TR><TR><TD> glgA ligated </TD><TD> 3 </TD><TD> #8PFH# </TD><TD> #B64M# </TD><TD> perfect
</TD></TR><TR><TD> glgA ligated </TD><TD> 6 </TD><TD> #FHPK# </TD></TR><TR><TD> glgA ligated </TD><TD> 7 </TD><TD> #4HKH# </TD></TR><TR><TD> glgA CPEC </TD><TD> 4 </TD><TD> #VWLO# </TD><TD> #6SHZ# </TD><TD> contaminated with two versions
</TD></TR><TR><TD> glgC </TD><TD> 1 </TD><TD> #SOD9# </TD><TD> #PZNS# </TD><TD> negative
</TD></TR><TR><TD> glgC </TD><TD> 2 </TD><TD> #YAPZ# </TD><TD> #6QC3# </TD><TD> negative
</TD></TR><TR><TD> glgC </TD><TD> 3 </TD><TD> #XBNP# </TD><TD> #YWRR# </TD><TD> negative
</TD></TR><TR><TD> glgC </TD><TD> 5 </TD><TD> #SDP4# </TD><TD> #AET8# </TD><TD> negative
</TD></TR><TR><TD> glgC </TD><TD> 6 </TD><TD> #X8Q1# </TD><TD> #LQCS# </TD><TD> negative
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-03">15-06-03</SPAN></H2><UL><LI> sequencing results analyzed:
</LI><LI> glgA: #B64M# and #M1M6# are perfect</LI><LI> glgB: #YHT3# is perfect</LI><LI> glgC: did not work yet</LI></UL><LI> heat shock of B0034 in pSB1A2 from KitPlate 2015.4 1N into NEB10b and plating on LB+A</LI><H2><SPAN class="mw-headline" id="15-06-04">15-06-04</SPAN></H2><UL><LI> 4 overnight and master plate of B0034 in pSB1A2</LI></UL><H2><SPAN class="mw-headline" id="15-06-05">15-06-05</SPAN></H2><UL><LI><S>cryos von B0034 in pSB1A2 in NEB10b</S><I>By accident the cultures were inoculated with two different clones.</I></LI><LI><S>freeze 4 cultures for later pPrep</S></LI><LI> colony PCR clones 1, 2, 3 from the master plate using #A9W9# and #XE3D#. Expected product length: 251 bp</LI><LI> gel picture looks perfect</LI><LI><S>plasmid prep of one good clone</S></LI><LI><S>pipet the plasmid for sequencing</S></LI></UL><H2><SPAN class="mw-headline" id="15-06-07">15-06-07</SPAN></H2><UL><LI><S>inoculation of one LB+A overnight culture</S></LI><LI> new overnights from master plates (clone 1, 2 and 3)</LI></UL><H2><SPAN class="mw-headline" id="15-06-08">15-06-08</SPAN></H2><UL><LI> cryos</LI><LI> plasmid prep from overnight #1</LI><LI> freeze #2 and #3</LI></UL><UL><LI> cut B0034 and glgC biobrick (BBa_K118016)
</LI><LI> glgC from #4D6H# mit Xba I + Pst</LI><LI> B0034 from #4EX9# mit EcoRI + Spe</LI></UL><LI> pSB1K3 from #AQY3# mit EcoRI + Pst</LI><H2><SPAN class="mw-headline" id="15-06-09">15-06-09</SPAN></H2><UL><LI> ligate B0034 and glgC in pSB1K3, product in #SY86#</LI><LI> trafo of B0034.glgC in pSB1K3 in DH5α per heat shock (#SY86#)</LI></UL><H2><SPAN class="mw-headline" id="15-06-10">15-06-10</SPAN></H2><UL><LI> overnights of B0034.glgC in pSB1K3</LI></UL><H2><SPAN class="mw-headline" id="15-06-11">15-06-11</SPAN></H2><UL><LI> cryos and plasmid prep of B0034.glgC in pSB1K3</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> B0034.glgC in pSB1K3 #1 </TD><TD> #66ZZ# </TD><TD> #WMVZ# </TD><TD> no RBS
</TD></TR><TR><TD> B0034.glgC in pSB1K3 #2 </TD><TD> #333V# </TD><TD> #YSVR# </TD><TD>no RBS
</TD></TR><TR><TD> B0034.glgC in pSB1K3 #3 </TD><TD> #C9TN# </TD><TD> was red </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1K3 #4 </TD><TD> #SSOL# </TD><TD> #TDS1# </TD><TD> no RBS
</TD></TR><TR><TD> B0034.glgC in pSB1K3 #5 </TD><TD> #Z8BB# </TD><TD> #HNEA# </TD><TD> no RBS
</TD></TR></TBODY></TABLE><UL><LI> pipette B0034.glgC in pSB1K3 for sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-06-17">15-06-17</SPAN></H2><P>Digestions with 40' digest and 20' inactivation
</P><TABLE class="wikitable"><TBODY><TR><TH> gene </TH><TH> enzymes </TH><TH> expectation </TH><TH> observation </TH><TH> products
</TH></TR><TR><TD> glgC (#4D6H#) </TD><TD> XbaI + PstI </TD><TD> ... </TD><TD> not run </TD><TD> #OHOZ#:C
</TD></TR><TR><TD> B0034 (#4EX9#) </TD><TD> SpeI + PstI </TD><TD> ... </TD><TD> not run </TD><TD> #OHOZ#:R
</TD></TR></TBODY></TABLE><P>Ligations and transformations
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> part I </TH><TH> part II </TH><TH> backbone </TH><TH> product </TH><TH> host
</TH></TR><TR><TD> B0034.glgC in pSB1A2 </TD><TD> #OHOZ#:R </TD><TD> #OHOZ#:C </TD><TD> - </TD><TD> #OKNR#:G </TD><TD> DH5α
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-18">15-06-18</SPAN></H2><UL><LI> make master plates and liquid cultures of transformation (B0034.glgC in pSB1A2; <B>6 liquid, 12 plate</B>)</LI></UL><H2><SPAN class="mw-headline" id="15-06-19">15-06-19</SPAN></H2><P>Colony PCR on glgC construct
</P><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> glgC in pSB1A2 clones 1-12 </TD><TD> 25-36 </TD><TD> 1556 </TD><TD> 2 </TD><TD> 1'50&quot; </TD><TD> all positive
</TD></TR></TBODY></TABLE><P>Cryos
</P><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> glgC in pSB1A2 #1 </TD><TD> #9F3M# </TD><TD> #SMTS# </TD><TD> perfect
</TD></TR><TR><TD> glgC in pSB1A2 #2 </TD><TD> #ZC8K# </TD><TD> #F8Y8# </TD><TD> perfect
</TD></TR><TR><TD> glgC in pSB1A2 #3 </TD><TD> #YV1C# </TD><TD> don't prep </TD><TD> -
</TD></TR><TR><TD> glgC in pSB1A2 #4 </TD><TD> #D3E6# </TD><TD> don't prep </TD><TD> -
</TD></TR><TR><TD> glgC in pSB1A2 #5 </TD><TD> #RTWO# </TD><TD> don't prep </TD><TD> -
</TD></TR><TR><TD> glgC in pSB1A2 #6 </TD><TD> #TN6K# </TD><TD> don't prep </TD><TD> -
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-22">15-06-22</SPAN></H2><UL><LI> prep good plasmids of glgA and glgC (table above)</LI><LI> pipette glgA in pSB1K30 and B0034.glgC for sequencing (#A9W9# and #XE3D#)</LI></UL><H2><SPAN class="mw-headline" id="15-06-24">15-06-24</SPAN></H2><UL><LI> analysis of sequencing results</LI></UL><H2><SPAN class="mw-headline" id="15-06-25">15-06-25</SPAN></H2><UL><LI> discard excess B0034.glgC in pSB1A2 cryos and plasmids</LI></UL><H2><SPAN class="mw-headline" id="15-06-26">15-06-26</SPAN></H2><P>Digestions
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1A2 </TD><TD> glgC </TD><TD> #F8Y8# </TD><TD> EcoRI, PstI </TD><TD> #LVVK#:1 </TD><TD> subcloning into pSB1C3, pSB1K30
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-29">15-06-29</SPAN></H2><P>Ligation and transformation
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1C3 </TD><TD> glgC </TD><TD> #LVVK#:1 </TD><TD> - </TD><TD> #CQ1V#:V </TD><TD> #RRCV#:1 </TD><TD> DH5α </TD><TD> for part submission
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-30">15-06-30</SPAN></H2><UL><LI> we have clones of K1585312 into pSB1C3 (red-white screening)</LI><LI> made master plate (6 clones) and overnight cultures (3 cultures)</LI></UL><H2><SPAN class="mw-headline" id="15-07-01">15-07-01</SPAN></H2><UL><LI> cryos and freezings of the three K1585312 in pSB1C3 cultures</LI><LI> colony PCR on all 6 clones &gt; clone 1 and 2 look good</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> B0034.glgC in pSB1C3 #1 </TD><TD> #4XPZ# </TD><TD> #DRPT# </TD><TD> glgA instead of glgC
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #2 </TD><TD> #1NZB# </TD><TD> #EZOB# </TD><TD> glgA instead of glgC
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #3 </TD><TD> #FCSS# </TD><TD> no prep </TD><TD> -
</TD></TR></TBODY></TABLE><UL><LI> pipetted for sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-07-03">15-07-03</SPAN></H2><UL><LI> overnight culture of glgC in pSB1A2 from #ZC8K# in LB+A</LI></UL><H2><SPAN class="mw-headline" id="15-07-06">15-07-06</SPAN></H2><P>PCR 1 to subclone glgC into pSB1C3
</P><P>Backbone amplification:
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> purified  pcr product ID </TH><TH> product ID </TH><TH> tube #
</TH></TR><TR><TD> J04450 in pSB1C3 </TD><TD> #CD8B# </TD><TD> #TSTV# </TD><TD> #FYTO# </TD><TD> 2070 </TD><TD> #ZV3C# </TD><TD> #MRVW#:II </TD><TD> 1+2
</TD></TR></TBODY></TABLE><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 2'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 64 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'10&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-08">15-07-08</SPAN></H2><P>PCR 2 to subclone glgC into pSB1C3
</P><P>Insert amplification:
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> product ID
</TH></TR><TR><TD> glgC in pSB1A2 </TD><TD> #F8Y8# </TD><TD> #O3M6# </TD><TD> #NBZZ# </TD><TD> 1360 </TD><TD> #MRVW#: I
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-09">15-07-09</SPAN></H2><P>PCR product purification
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR product </TH><TH> purified PCR product
</TH></TR><TR><TD> #MRVW#:I </TD><TD> #XNHT#
</TD></TR><TR><TD> #MRVW#:II </TD><TD> #ZV3C#
</TD></TR></TBODY></TABLE><P>CPEC assembly of glgC in pSB1C3
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [µl]
</TH></TR><TR><TD> water </TD><TD> 5.9
</TD></TR><TR><TD> Q5 buffer </TD><TD> 5
</TD></TR><TR><TD> DMSO </TD><TD> 0.75
</TD></TR><TR><TD> dNTPs (2 mM) </TD><TD> 5
</TD></TR><TR><TD> Q5 polymerase </TD><TD> 0.5
</TD></TR><TR><TD> pSB1C3 fragment (#ZV3C#) </TD><TD> 2.7
</TD></TR><TR><TD> glgC with prefix/suffix #XNHT# </TD><TD> 5.2
</TD></TR><TR><TD> elongation time </TD><TD> 1'42&quot; (1'47&quot;)
</TD></TR><TR><TD> product stored in </TD><TD> #SWMB#:I
</TD></TR></TBODY></TABLE><P>5 µl #SWMB#:I were heat shocked into DH5α and plated on LB+C
</P><H2><SPAN class="mw-headline" id="15-07-10">15-07-10</SPAN></H2><UL><LI> make masterplates (LB+C) and overnights of glgC in pSB1C3</LI></UL><H2><SPAN class="mw-headline" id="15-07-11">15-07-11</SPAN></H2><UL><LI> cryo cultures </LI><LI> freeze culture</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> B0034.glgC in pSB1C3 #1 </TD><TD> #ESAY# </TD><TD> #QXBD# </TD><TD> good, but not perfect
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #2 </TD><TD> #DFKW# </TD><TD> #OP1N# </TD><TD> perfect
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #3 </TD><TD> #VW9S# </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #4 </TD><TD> #L81A# </TD><TD> #MCM6# </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #5 </TD><TD> #SOVA# </TD><TD> #NPCT# </TD><TD> contaminated chromatogram
</TD></TR><TR><TD> B0034.glgC in pSB1C3 #6 </TD><TD> #Q9NL# </TD><TD> - </TD><TD> -
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-13">15-07-13</SPAN></H2><UL><LI> plasmid prep of glgC in pSB1C3 (see table above)</LI></UL><H2><SPAN class="mw-headline" id="15-07-15">15-07-15</SPAN></H2><UL><LI> analysing sequencing results (see table above)</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>