<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Glycogen_Polycistronic_Expression skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Glycogen Polycistronic Expression</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmGlycogenSynthesisIII" href="/Team:Aachen/Lab/Glycogen/Synthesis/III._Polycistronic_Expression"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Combined Expression</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><H2><SPAN class="mw-headline" id="15-06-09">15-06-09</SPAN></H2><P>digest glgA/B plasmids (<B>stored in #SZKD#</B>)
</P><TABLE class="wikitable"><TBODY><TR><TH> name </TH><TH> plasmid ID </TH><TH> enzyme 1 </TH><TH> enzyme 2
</TH></TR><TR><TD> A1 (glgA)  </TD><TD> #M1M6# </TD><TD> EcoRI </TD><TD> SpeI
</TD></TR><TR><TD> B1 (glgB) </TD><TD> #YHT3# </TD><TD> XbaI </TD><TD> PstI
</TD></TR><TR><TD> pSB1A3    </TD><TD> #TRTN# </TD><TD> EcoRI </TD><TD> PstI
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-10">15-06-10</SPAN></H2><UL><LI> after digest glg A and B:
</LI><LI> assemble A1 with B1 in pSB1A3 (ligation tube: A+B)</LI><LI><B>stored in #QHHR#</B></LI></UL><LI> transformation into DH5Î± </LI><H2><SPAN class="mw-headline" id="15-06-11">15-06-11</SPAN></H2><UL><LI> master plates of glgA+B in pSB1A3 (LB+A)</LI></UL><H2><SPAN class="mw-headline" id="15-06-12">15-06-12</SPAN></H2><UL><LI> colony PCR on glgAB construct to identify positive clones</LI><LI> make overnight cultures of glgAB construct</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> glgA.glgB in pSB1A3 #5 </TD><TD> #Y4K4# </TD><TD> #P6QM# </TD><TD> glgB is missing
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-13">15-06-13</SPAN></H2><UL><LI> make cryos of glgAB construct (table above)</LI><LI> plasmid prep of glgAB (table above)</LI></UL><H2><SPAN class="mw-headline" id="15-06-15">15-06-15</SPAN></H2><UL><LI> measured concentrations of purified glgAB constructs</LI></UL><H2><SPAN class="mw-headline" id="15-06-16">15-06-16</SPAN></H2><UL><LI> digested glgAB plasmid (#P6QM#) with XbaI and PstI (product in #TZTD#) gel had three bands! -&gt; new digest and ligation</LI></UL><H2><SPAN class="mw-headline" id="15-06-17">15-06-17</SPAN></H2><P>Digestions with 40' digest and 20' inactivation
</P><P>(previous digests A1, B1 were discarded from #SZKD# because agarose gels were really ugly)
</P><TABLE class="wikitable"><TBODY><TR><TH> gene </TH><TH> enzymes </TH><TH> expectation </TH><TH> observation </TH><TH> products
</TH></TR><TR><TD> glgA (#M1M6#) </TD><TD> EcoRI + SpeI </TD><TD> 3533, <B>2051</B>, <B>1482</B></TD><TD> confirmed </TD><TD> #SZKD#:A
</TD></TR><TR><TD> glgB (#YHT3#) </TD><TD> XbaI + PstI </TD><TD> 4286, <B>2234</B>, <B>2052</B></TD><TD> confirmed </TD><TD> #SZKD#:B
</TD></TR></TBODY></TABLE><P>Ligations and transformations
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> part I </TH><TH> part II </TH><TH> backbone </TH><TH> product </TH><TH> host
</TH></TR><TR><TD> glgAB in pSB1A3 </TD><TD> #SZKD#:A </TD><TD> #SZKD#:B </TD><TD> #SZKD#:pSB1A3 </TD><TD> #99MC#:LAB </TD><TD> BL21
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-18">15-06-18</SPAN></H2><UL><LI> transformation of glgAB in pSB1A3 in BL21 did not work!</LI><LI> ligation of glgAB on a gel shows 3 independent bands
</LI><LI> therefore, we cut pSB1A3 again with EcoRI and PstI</LI></UL><P>Digestions with 40' digest and 20' inactivation
</P><P>(previous digests pSB1A3 was discarded from #SZKD# because we think it was cut with EcoRI and SpeI)
</P><TABLE class="wikitable"><TBODY><TR><TH> gene </TH><TH> enzymes </TH><TH> expectation </TH><TH> observation </TH><TH> products
</TH></TR><TR><TD> pSB1A3 (#TRTN#) </TD><TD> EcoRI + PstI </TD><TD> ... </TD><TD> ... </TD><TD> #SZKD#:1A3
</TD></TR></TBODY></TABLE><P>Ligations and transformations
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> part I </TH><TH> part II </TH><TH> backbone </TH><TH> product </TH><TH> host
</TH></TR><TR><TD> glgAB in pSB1A3 </TD><TD> #SZKD#:A </TD><TD> #SZKD#:B </TD><TD> #SZKD#:1A3</TD><TD> #99MC#:LAB </TD><TD> DH5Î±
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-19">15-06-19</SPAN></H2><P>Master plates of glgAB clones that were white under green light.
</P><H2><SPAN class="mw-headline" id="15-06-22">15-06-22</SPAN></H2><UL><LI> colony PCR on glgAB plate from Saturday (#A9W9# and #XE3D#)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> glgAB in pSB1A2 clones 1-7 </TD><TD> 18-24 </TD><TD> 3986 </TD><TD> 3 </TD><TD> 3'59&quot; </TD><TD> clones 1-6 are good
</TD></TR></TBODY></TABLE><UL><LI> overnight cultures of good clones from the colony PCR glgAB (LB+A)</LI></UL><H2><SPAN class="mw-headline" id="15-06-23">15-06-23</SPAN></H2><UL><LI> cryos and plasmid prep from overnight culture of glgAB</LI><LI> the plasmid prep went wrong and resulted in a concentration of &lt;Â 10Â ng/Âµl</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> glgAB in pSB1A3 #5 </TD><TD> #1Q88# </TD><TD> #CT8H# </TD><TD> confirmed ligation sites
</TD></TR><TR><TD> glgAB in pSB1A3 #6 </TD><TD> #YV86# </TD><TD> #O3TQ# </TD><TD> confirmed ligation sites
</TD></TR></TBODY></TABLE><UL><LI> overnight cultures of the two clones above from the master plates</LI></UL><H2><SPAN class="mw-headline" id="15-06-24">15-06-24</SPAN></H2><UL><LI> in the morning: freeze overnight cultures of glgB and glgAB</LI><LI> in the afternoon: plasmid prep of glgB and glgAB (re-use the existing IDs from the table of Monday)</LI><LI> pipette glgAB for sequencing #CT8H#  and #O3TQ# were emptied during sequencing!</LI></UL><H2><SPAN class="mw-headline" id="15-06-26">15-06-26</SPAN></H2><P>Digestions
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1A2 </TD><TD> glgC </TD><TD> #F8Y8# </TD><TD> EcoRI, SpeI </TD><TD> #LVVK#:2 </TD><TD> construction of glgCAB
</TD></TR><TR><TD> K1585320 in pSB1A2 </TD><TD> glgAB </TD><TD> #CT8H# </TD><TD> XbaI, PstI </TD><TD> #LVVK#:3 </TD><TD> construction of glgCAB
</TD></TR></TBODY></TABLE><UL><LI> overnight culture</LI></UL><H2><SPAN class="mw-headline" id="15-06-29">15-06-29</SPAN></H2><UL><LI> Plasmid prep of glgAB in pSB1A3 because #CT8H# was empty (same ID)</LI></UL><P>Digestions
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585320 in pSB1A2 </TD><TD> glgAB </TD><TD> #CT8H# </TD><TD> XbaI, PstI </TD><TD> #LVVK#:3 </TD><TD> construction of glgCAB
</TD></TR></TBODY></TABLE><UL><LI> run an agarose gel with the digestion products
</LI><LI> gel showed that digest of glgAB and pSB1K30 has not worked!</LI><LI> we will use old pSB1K30 stored in #SZKD#</LI></UL><P>Ligation and Transformation
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585321 in pSB1K30 </TD><TD> glgCAB </TD><TD> #LVVK#:2 </TD><TD> #LVVK#:3 </TD><TD> #LVVK#:4 </TD><TD> #RRCV#:3 </TD><TD> BL21 Gold (DE3) </TD><TD> for polycistronic expression
</TD></TR></TBODY></TABLE><UL><LI> new glgAB digestion because wrong concentration was used</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585320 in pSB1A2 </TD><TD> glgAB </TD><TD> #CT8H# </TD><TD> XbaI, PstI </TD><TD> #LVVK#:3 </TD><TD> construction of glgCAB
</TD></TR></TBODY></TABLE><UL><LI> repeat ligation and freeze product for transformation</LI></UL><P>Ligation and Transformation
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585321 in pSB1K30 </TD><TD> glgCAB </TD><TD> #LVVK#:2 </TD><TD> #LVVK#:3neu </TD><TD> #SZKD#:K30 </TD><TD> #RRCV#:3 </TD><TD> BL21 Gold (DE3) </TD><TD> for polycistronic expression
</TD></TR></TBODY></TABLE><UL><LI> glgAB digest did not work again! </LI></UL><P>PCR amplification of glgAB
</P><UL><LI> 50Â Âµl Q5 reaction with #A9W9# and #XE3D#</LI><LI> 2'00&quot; elongation time, 58Â Â°C annealing</LI><LI> products stored in #XMVF#</LI></UL><H2><SPAN class="mw-headline" id="15-06-30">15-06-30</SPAN></H2><UL><LI> agarose gel of glgAB PCR product shows a very strong band at the expected length</LI><LI> the PCR product was purified into #PZ9B#</LI></UL><P>Digestion of glgAB PCR product
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585320 in pSB1A2 </TD><TD> glgAB </TD><TD> #PZ9B# </TD><TD> XbaI, PstI </TD><TD> #LVVK#:3 </TD><TD> construction of glgCAB
</TD></TR></TBODY></TABLE><P>Ligation and Transformation
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585321 in pSB1C3 </TD><TD> glgCAB </TD><TD> #LVVK#:2 </TD><TD> #LVVK#:3 </TD><TD> #CQ1V#:V </TD><TD> 5Â Âµl #RRCV#:3 </TD><TD> 40Â Âµl BL21 Gold (DE3) </TD><TD> for polycistronic expression
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-01">15-07-01</SPAN></H2><UL><LI> made a master plate of glgCAB in pSB1C3 with all available clones</LI></UL><H2><SPAN class="mw-headline" id="15-07-02">15-07-02</SPAN></H2><UL><LI> colony PCR</LI><LI> 4 overnight cultures per clone (to have enough for sequencing)</LI></UL><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 95 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> annealing #A9W9# and #XE3D# </TD><TD> 58 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'20&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 2'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> repeat colony PCR
</LI><LI> tubes are numbered 1-18 for colonies</LI><LI> colonies picked: 1,2,5,6,7,11,12,13,16-21,23-26 </LI><LI> 19: water</LI></UL><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 95 </TD><TD> 10'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> annealing #A9W9# and #XE3D# </TD><TD> 58 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'25&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 2'00&quot;
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-03">15-07-03</SPAN></H2><UL><LI> plasmid prep of many different glgCAB clones</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> digest result
</TH></TR><TR><TD> 1 </TD><TD> #LMCZ# </TD><TD> #ZNMX# </TD><TD> negative
</TD></TR><TR><TD> 2 </TD><TD> #K49Y# </TD><TD> #KLRW# </TD><TD> negative
</TD></TR><TR><TD> 5 </TD><TD> #FYOF# </TD><TD> #ZKDX# </TD><TD> negative
</TD></TR><TR><TD> 6 </TD><TD> #K8MM# </TD><TD> #AYWZ# </TD><TD> negative
</TD></TR><TR><TD> 7 </TD><TD> #LBRK# </TD><TD> #M9BH# </TD><TD> negative
</TD></TR><TR><TD> 11 </TD><TD> #YBH1# </TD><TD> #EAFT# </TD><TD> negative
</TD></TR><TR><TD> 12 </TD><TD> #99ER# </TD><TD> #V6DM# </TD><TD> negative
</TD></TR><TR><TD> 13 </TD><TD> #L39B# </TD><TD> #Z9BA# </TD><TD> negative
</TD></TR><TR><TD> 16 </TD><TD> #S3BW# </TD><TD> #A4Y6# </TD><TD> negative
</TD></TR><TR><TD> 17 </TD><TD> #V3SS# </TD><TD> #49S6# </TD><TD> negative
</TD></TR><TR><TD> 18 </TD><TD> #WOXE# </TD><TD> #XO6W# </TD><TD> negative
</TD></TR><TR><TD> 20 </TD><TD> #KEMW# </TD><TD> #4HTD# </TD><TD> negative
</TD></TR></TBODY></TABLE><UL><LI> (test) digestion of plasmids with EcoRI+PstI (products in #PCSW#)</LI><LI> agarose gel</LI><LI><S>overnight cultures of good clones from test digest</S></LI></UL><H2><SPAN class="mw-headline" id="15-07-06">15-07-06</SPAN></H2><P>PCRs for Gibson/CPEC assembly
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> product ID </TH><TH> tube #
</TH></TR><TR><TD> J04450 in pSB1C3 </TD><TD> #CD8B# </TD><TD> #TSTV# </TD><TD> #FYTO# </TD><TD> 2070 </TD><TD> #MRVW#:II </TD><TD> 1+2
</TD></TR></TBODY></TABLE><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 2'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 64 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'10&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-08">15-07-08</SPAN></H2><P>PCRs for Gibson/CPEC assembly
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> product ID
</TH></TR><TR><TD> glgC in pSB1A2 </TD><TD> #F8Y8# </TD><TD> #O3M6# </TD><TD> #C3ZA# </TD><TD> 1360 </TD><TD> #MRVW#: IV
</TD></TR><TR><TD> glgAB in pSB1A3 </TD><TD> #CT8H# </TD><TD> #93FZ# </TD><TD> #NBZZ# </TD><TD> 3713 </TD><TD> #MRVW#:V
</TD></TR></TBODY></TABLE><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>: 
</P><P>glgC
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 59 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'45&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><P>glgAB
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 57 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'58&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><P>The design of primer #93FZ# was flawed: it bound to the RBS.ATG section of glgB (within glgAB) with three mismatches and amplified a product of 2235 bp.
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> product ID
</TH></TR><TR><TD> glgA' </TD><TD> #M1M6# </TD><TD> #93FZ# </TD><TD> #AVY6# </TD><TD> 1157 </TD><TD> #MRVW#: VI
</TD></TR><TR><TD> glg'AB </TD><TD> #CT8H# </TD><TD> #EP9Z# </TD><TD> #NBZZ# </TD><TD> 2907 </TD><TD> #MRVW#:VII
</TD></TR></TBODY></TABLE><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>: 
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> glgA' </TH><TH> glg'AB
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot; </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot; </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 58 </TD><TD> 0'30&quot; </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'35&quot; </TD><TD> 1'27&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot; </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-09">15-07-09</SPAN></H2><UL><LI> 90'00&quot; DpnI digest of #MRVW#:VI and #MRVW#:VII</LI><LI> the gel of #MRVW#:VII did not show any band - the old VII was discarded and the PCR was repeated with longer elongation time</LI></UL><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>: 
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> glg'AB
</TH></TR><TR><TD>initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 56.5 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'35&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72</TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> expected length: 2907 bp</LI></UL><P>
PCR product purification
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR product </TH><TH> purified PCR product
</TH></TR><TR><TD> #MRVW#:II </TD><TD> #ZV3C#
</TD></TR><TR><TD> #MRVW#:III </TD><TD> #T6X9#
</TD></TR><TR><TD> #MRVW#:IV </TD><TD> #QSWQ#
</TD></TR><TR><TD> #MRVW#:V </TD><TD> failed/obsolete
</TD></TR><TR><TD> #MRVW#:VI </TD><TD> #4TVW#
</TD></TR><TR><TD> #MRVW#:VII </TD><TD> #E9AQ#
</TD></TR></TBODY></TABLE><P>Gibson Assembly of glgCAB in pSB1C3 (0.17 pmol total fragment amount, 30'00&quot; incubation)
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [Âµl]
</TH></TR><TR><TD> water </TD><TD> 0
</TD></TR><TR><TD> Gibson Master Mix 2x </TD><TD> 12.5
</TD></TR><TR><TD> linear pSB1C3 #ZV3C# </TD><TD> 1.4
</TD></TR><TR><TD> glgC for glgCAB #QSWQ# </TD><TD> 5.3
</TD></TR><TR><TD> glgA' for glgCAB #4TVW# </TD><TD> 1.6
</TD></TR><TR><TD> glg'AB for glgCAB #E9AQ# </TD><TD> 4.3
</TD></TR><TR><TD> product stored in </TD><TD> #SWMB#:III
</TD></TR></TBODY></TABLE><P>5Â Âµl #SWMB#:III were heat shocked into DH5Î± and plated on LB+C
</P><H2><SPAN class="mw-headline" id="15-07-10">15-07-10</SPAN></H2><UL><LI> make masterplate (LB+C)</LI></UL><H2><SPAN class="mw-headline" id="15-07-14">15-07-14</SPAN></H2><UL><LI> analyze master plate of additional glgCAB clones under green light, all clones were red!</LI><LI> repeat Gibson Assembly with now DpnI digested pSB1C3 backbone</LI></UL><P>Gibson Assembly of glgCAB in pSB1C3 (0.17 pmol total fragment amount, 60'00&quot; incubation)
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [Âµl]
</TH></TR><TR><TD> water </TD><TD> 0</TD></TR><TR><TD> Gibson Master Mix 2x </TD><TD> 12.5
</TD></TR><TR><TD> linear pSB1C3 #ZV3C# </TD><TD> 1.4
</TD></TR><TR><TD> glgC for glgCAB #QSWQ# </TD><TD> 5.3
</TD></TR><TR><TD> glgA' for glgCAB #4TVW# </TD><TD> 1.6
</TD></TR><TR><TD> glg'AB for glgCAB #E9AQ# </TD><TD> 4.3
</TD></TR><TR><TD> product stored in </TD><TD> #SWMB#:III
</TD></TR></TBODY></TABLE><P>5Â Âµl #SWMB#:III were heat shocked into DH5Î± and plated on LB+C
</P><H2><SPAN class="mw-headline" id="15-07-15">15-07-15</SPAN></H2><UL><LI> transformation was successful: under red-light 8 white clones were visible</LI><LI> overnights and masterplate of these good clones</LI></UL><H2><SPAN class="mw-headline" id="15-07-16">15-07-16</SPAN></H2><UL><LI>cryos</LI><LI> plasmid prep</LI><LI> test digest with Xba and Pst (products in #NMTE#)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone </TH><TH> cryo </TH><TH> plasmid ID </TH><TH> test digest result
</TH></TR><TR><TD> glgCAB in pSB1A30 #1 </TD><TD> #9HZM# </TD><TD> #QWNV# </TD><TD> negative
</TD></TR><TR><TD> glgCAB in pSB1A30 #2 </TD><TD> #SR8W# </TD><TD> #LTYR# </TD><TD> negative
</TD></TR><TR><TD> glgCAB in pSB1A30 #3 </TD><TD> #VPTH# </TD><TD>#QWBZ# </TD><TD> negative
</TD></TR><TR><TD> glgCAB in pSB1A30 #4 </TD><TD> #NHYB# </TD><TD> #6CEE# </TD><TD> negative
</TD></TR><TR><TD> glgCAB in pSB1A30 #5</TD><TD> #NHBT#</TD><TD> #6KF4# </TD><TD> could be something
</TD></TR><TR><TD> glgCAB in pSB1A30 #6 </TD><TD> #H443# </TD><TD> #QOX1# </TD><TD> negative
</TD></TR><TR><TD> glgCAB in pSB1A30 #7 </TD><TD> #YF88# </TD><TD> #CHT1# </TD><TD> could be something
</TD></TR><TR><TD> glgCAB in pSB1A30 #8 </TD><TD> #W4VO# </TD><TD> #OQO3# </TD><TD> negative
</TD></TR></TBODY></TABLE><UL><LI> new overnights of clone#5 and #7 to have enough template for sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-07-17">15-07-17</SPAN></H2><P>CPEC assembly of glgCAB in pSB1C3
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [Âµl]
</TH></TR><TR><TD> water </TD><TD> 0
</TD></TR><TR><TD> Q5 buffer </TD><TD> 5
</TD></TR><TR><TD> DMSO </TD><TD> 0.75
</TD></TR><TR><TD> dNTPs (O5) </TD><TD> 1
</TD></TR><TR><TD> Q5 polymerase </TD><TD> 0.5
</TD></TR><TR><TD> pSB1C3 fragment (#ZV3C#) </TD><TD> 2.7
</TD></TR><TR><TD> glgC for glgCAB #QSWQ# </TD><TD> 9.8
</TD></TR><TR><TD> glgA' for glgCAB #4TVW# </TD><TD> 3.0
</TD></TR><TR><TD> glg'AB for glgCAB #E9AQ# </TD><TD> 8.0
</TD></TR><TR><TD> elongation time </TD><TD> 3'32&quot;
</TD></TR><TR><TD> product stored in </TD><TD> #SWMB#:IV
</TD></TR></TBODY></TABLE><UL><LI> agarose gel with glgA', Gibson product and CPEC product</LI><LI> heat shock transformation of #SWMB#Â :III and IV into DH5Î±</LI><LI> plasmid prep of potentially good glgCAB clones #5 in #FYH1# and #7 in #4RDQ#</LI><LI> test digest of #FYH1# and #4RDQ# with EcoRI (products in #4SK6#:1-2)</LI><LI> agarose gel comparing with undigested #FYH1# -&gt; #FYH1# looks promising, but #4RDQ# is too short</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Test digest of glgCAB in pSB1C3</B>#FYH1# and #4RDQ# were test digested with EcoRI. #FYH1# looks very promising.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="15-07-18">15-07-18</SPAN></H2><UL><LI> master plate of many white glgCAB clones on LB+C</LI></UL><H2><SPAN class="mw-headline" id="15-07-19">15-07-19</SPAN></H2><UL><LI> 3x 5Â ml LB+C overnight cultures of glgCAB clones #1 (CPEC) and #23 (Gibson)</LI></UL><H2><SPAN class="mw-headline" id="15-07-20">15-07-20</SPAN></H2><UL><LI> Cryos and freezings of glgCAB clones #1 and #23</LI><LI> plasmid prep of glgCAB clones #1 and #23</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> strain </TH><TH> cryo </TH><TH> plasmid
</TH></TR><TR><TD> glgCAB in pSB1C3 by CPEC in DH5Î± #1 </TD><TD> #VAPH# </TD><TD> #EY9B#
</TD></TR><TR><TD> glgCAB in pSB1C3 by Gibson in DH5Î± clone #23 </TD><TD> #PAZS# </TD><TD> #6BZC#
</TD></TR></TBODY></TABLE><UL><LI> pipette glgCAB in pSB1C3 #FYH1# for sequencing with all primers</LI><LI> test digest of glgCAB #1 and #23 with EcoRI
</LI><LI> tube 1: #6BZC# (#23)</LI><LI> tube 2: #EY9B# (#1)</LI><LI> products are stored in #1W9S#</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Test digest of glgCAB in pSB1C3</B>#6BZC# and #EY9B# were test digested with EcoRI. #EY9B# looks very promising.
</TD></TR></TBODY></TABLE></DIV><UL><LI><B>clone 1 looks good, but clone 23 does not!</B></LI><LI> prepare moving glgCAB over into a vector with promoter</LI></UL><H2><SPAN class="mw-headline" id="15-07-21">15-07-21</SPAN></H2><UL><LI> digest of glgCAB clones, J23119 and K1585119 to combine CAB with the Anderson Promoter and pSB1A30</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> template </TH><TH> ID </TH><TH> enzymes </TH><TH>tube number </TH><TH> purpose
</TH></TR><TR><TD> Anderson Promoter 19 </TD><TD> #AHZ6# </TD><TD> SpeI, PstI </TD><TD> 1 </TD><TD> to combine CAB and constitutive AP 19
</TD></TR><TR><TD> K1585119 </TD><TD> #OEVV# </TD><TD> SpeI, PstI </TD><TD> 2 </TD><TD> to combine CAB and inducible AP 19
</TD></TR><TR><TD> glgCAB #1 </TD><TD> #EY9B# </TD><TD> XbaI, PstI </TD><TD> 3 </TD><TD> to combine with both AP 19 versions
</TD></TR><TR><TD> glgCAB #5 </TD><TD> #FYH1# </TD><TD> XbaI, PstI </TD><TD> 4 </TD><TD> to combine with both AP 19 versions
</TD></TR><TR><TD> glgCAB #1 </TD><TD> #EY9B# </TD><TD> EcoRI, PstI </TD><TD> 5 </TD><TD> to combine with pSB1A30
</TD></TR><TR><TD> glgCAB #5 </TD><TD> #FYH1# </TD><TD> EcoRI, PstI </TD><TD> 6 </TD><TD> to combine with pSB1A30
</TD></TR></TBODY></TABLE><UL><LI> products stored in #TF4M#</LI></UL><UL><LI> ligation</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> name </TH><TH> template 1 </TH><TH> template 2 </TH><TH> purpose
</TH></TR><TR><TD> L1 </TD><TD> tube 1 (const. AP 19) </TD><TD> tube 3 (glgCAB #1 with X and P)</TD><TD> glgCAB with constitutive AP 19
</TD></TR><TR><TD> L2 </TD><TD> tube 2 (induc. AP 19) </TD><TD> tube 3 (glgCAB #1 with X and P) </TD><TD> glgCAB with inducible AP 19
</TD></TR><TR><TD> L3 </TD><TD> tube 1 (const. AP 19) </TD><TD> tube 4 (glgCAB #5 with X and P) </TD><TD> glgCAB with constitutive AP 19
</TD></TR><TR><TD> L4 </TD><TD> tube 2 (induc. AP 19) </TD><TD> tube 4 (glgCAB #5 with X and P) </TD><TD>  glgCAB with inducible AP 19
</TD></TR></TBODY></TABLE><UL><LI> products are also stored in #TF4M#</LI><LI> all ligation products were transformed in NEB 10Î²</LI></UL><H2><SPAN class="mw-headline" id="15-07-22">15-07-22</SPAN></H2><UL><LI> make a gel with all digests</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Digest and ligation of glgCAB and AP 19</B>For #AHZ6# there are two bands of the right length, one uncut and one cut. For #OEVV# there is only one weak band but it has the correct length. Both CAB digest products of #EY9B# have the correct length. There are no bands for the ligations.
</TD></TR></TBODY></TABLE></DIV><TABLE class="wikitable"><TBODY><TR><TH> strain </TH><TH> cryo </TH><TH> plasmid </TH><TH> sequencing result
</TH></TR><TR><TD> glgCAB in pSB1C3 by CPEC in DH5Î± #1 </TD><TD> #VAPH# </TD><TD> #EY9B# </TD><TD> A&gt;V mutation in glgB, glgC and A look perfect
</TD></TR><TR><TD> glgCAB in pSB1C3 by Gibson in DH5Î± clone #23 </TD><TD> #PAZS# </TD><TD> #6BZC# </TD><TD> sequencing did not work
</TD></TR><TR><TD> glgCAB in pSB1C3 by Gibson in DH5Î± clone #5 </TD><TD> #NHBT# </TD><TD> #FYH1# </TD><TD> glgA and B look perfect but glgC is missing
</TD></TR></TBODY></TABLE><UL><LI> overnight cultures and master plates of transformations of ligations 1+2 (LB+A)</LI><LI> new overnights of clones #3-5 glgCAB in pSB1C3 by CPEC from masterplate</LI></UL><H2><SPAN class="mw-headline" id="15-07-23">15-07-23</SPAN></H2><UL><LI> cryos and plasmid prep of glgCAB, AP19.glgCAB and AP19.lacIsite.glgCAB candidates</LI><LI> test digest with EcoRI (stored in #AMX6#)
</LI><LI> 1: #4HB9#</LI><LI> 2: #6THO#</LI><LI> 3: #BB9Q#</LI><LI> C2: #HOEL#</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> cryo </TH><TH> plasmid </TH><TH> digest result </TH><TH> sequencing result
</TH></TR><TR><TD> K1585321 in pSB1C3 #2 </TD><TD> #BA36# </TD><TD> #NXOK# </TD><TD> ... </TD><TD> no chromatogram
</TD></TR><TR><TD> K1585321 in pSB1C3 #3 </TD><TD> #KRWH# </TD><TD> #BB9Q# </TD><TD> ... </TD><TD> no chromatogram
</TD></TR><TR><TD> K1585321 in pSB1C3 #4 </TD><TD> #RHMM# </TD><TD> #6THO# </TD><TD> ... </TD><TD> no chromatogram
</TD></TR><TR><TD> J23119.K1585321 in J61002 #1 </TD><TD> #K3K4# </TD><TD> #4HB9# </TD><TD> cloning did not work
</TD></TR><TR><TD> J23119.K1585321 in J61002 #2 </TD><TD> #W3QO# </TD><TD> not prepped </TD><TD> cloning did not work
</TD></TR><TR><TD> K1585119.K1585321 in J61002 #1 </TD><TD> #Y4N4# </TD><TD> was red </TD></TR><TR><TD> K1585119.K1585321 in J61002 #2 </TD><TD> #XFF9# </TD><TD> was red </TD></TR><TR><TD> K1585119.K1585321 in J61002 #3 </TD><TD> #KATB# </TD><TD> was red </TD></TR><TR><TD> K1585119.K1585321 in J61002 #4 </TD><TD> #MWVT# </TD><TD> was red </TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-24">15-07-24</SPAN></H2><UL><LI> sequencing results were bad, so we use parts from #FYH1# (just glgC missing) and #EY9B# (mutation in glgB) to construct the full plasmid again</LI><LI> these parts will be assembled in a Gibson Assembly</LI></UL><P>Q5 PCR with 27 cycles
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR </TH><TH> template </TH><TH> primer 1 </TH><TH> primer 2 </TH><TH> annealing temperature </TH><TH> elongation time </TH><TH> product length
</TH></TR><TR><TD> 1 </TD><TD> #FYH1# </TD><TD> #EP9Z# </TD><TD> #6ZOP# </TD><TD> 71 Â°C </TD><TD> 1'23&quot; </TD><TD> 2640 bp (parts of glgA and B)
</TD></TR><TR><TD> 2 </TD><TD> #EY9B# </TD><TD>#89VV# </TD><TD> #AVY6# </TD><TD> 71 Â°C </TD><TD> 2'40&quot; </TD><TD> 5178 bp (all the rest)
</TD></TR></TBODY></TABLE><P>PCR programm
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation (cycle) </TD><TD> 98 Â°C </TD><TD> 0'10&quot;
</TD></TR><TR><TD> annealing (cycle) </TD><TD> 71 Â°C </TD><TD> 0'25&quot;
</TD></TR><TR><TD> elongation (cycle) </TD><TD> 72 Â°C </TD><TD> see above
</TD></TR><TR><TD> final elongation </TD><TD> 72 Â°C </TD><TD> 2'00&quot;
</TD></TR></TBODY></TABLE><P>The products are temporarily stored in #94DS#.
</P><P>Both products are digested with DpnI overnight at 37Â Â°C and put back into #94DS#.
</P><H2><SPAN class="mw-headline" id="15-04-25">15-04-25</SPAN></H2><UL><LI> put DpnI digests back into #94DS# and freeze it</LI></UL><H2><SPAN class="mw-headline" id="15-07-27">15-07-27</SPAN></H2><UL><LI> agarose gel of DpnI digests</LI><LI> PCR product purification
</LI><LI> products stored in #MCWO# and #963L#</LI></UL><LI> Gibson Assembly of #MCWO# and #963L# at 50 Â°C for 50 minutes</LI><LI> heat shock into DH5Î±</LI><H2><SPAN class="mw-headline" id="15-07-28">15-07-28</SPAN></H2><UL><LI> transformation was successful</LI><LI> masterplate of 30 clones from transformation plate of glgCAB in pSB1C3</LI><LI> overnight of 10 clones for test digest</LI></UL><H2><SPAN class="mw-headline" id="15-07-29">15-07-29</SPAN></H2><UL><LI> plasmid prep and cryos of all overnights</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> cryo </TH><TH> plasmid </TH><TH> tube # in digest </TH><TH> digest result </TH><TH> sequencing result
</TH></TR><TR><TD> K1585321 in pSB1C3 #1 </TD><TD> #L8CN# </TD><TD> #96EB#, #4RCD# </TD><TD> 1 </TD><TD> good </TD><TD> point mutation
</TD></TR><TR><TD> K1585321 in pSB1C3 #2 </TD><TD> #NMTT# </TD><TD> #MFQS#, #L6OH#</TD><TD>2 </TD><TD> good </TD><TD> point mutation
</TD></TR><TR><TD> K1585321 in pSB1C3 #3 </TD><TD> #MXKQ# </TD><TD> #QAB9#, #8WE4# </TD><TD>3 </TD><TD> good </TD><TD> point mutation
</TD></TR><TR><TD> K1585321 in pSB1C3 #4 </TD><TD> #CKSK# </TD><TD> #B316# </TD><TD> 4 </TD><TD> good </TD><TD> -
</TD></TR><TR><TD> K1585321 in pSB1C3 #5 </TD><TD> #1TN1# </TD><TD> #HOX9#</TD><TD> 5 </TD><TD> two bands instead of one</TD><TD> -
</TD></TR><TR><TD> K1585321 in pSB1C3 #6 </TD><TD> #MKC3# </TD><TD> #FNTZ# </TD><TD> 6 </TD><TD> good </TD><TD> -
</TD></TR><TR><TD> K1585321 in pSB1C3 #7 </TD><TD> #99NB# </TD><TD> #6DMQ# </TD><TD> 7 </TD><TD> too long (?)</TD><TD> -
</TD></TR><TR><TD> K1585321 in pSB1C3 #8 </TD><TD> #P664# </TD><TD> #6OT6# </TD><TD> 8 </TD><TD> good </TD><TD> -
</TD></TR><TR><TD> K1585321 in pSB1C3 #9 </TD><TD> #P8NZ# </TD><TD> #HF8O# </TD><TD> 9</TD><TD> good </TD><TD> -
</TD></TR><TR><TD> K1585321 in pSB1C3 #10 </TD><TD> #TO3R# </TD><TD> #A1FM# </TD><TD> 10 </TD><TD> good </TD><TD> -
</TD></TR></TBODY></TABLE><UL><LI> test digest with EcoRI + agarose gel</LI><LI> new overnights of good glgCAB in C3 clones no.1-3 (3 per clone to have enough for seqencing)</LI></UL><H2><SPAN class="mw-headline" id="15-07-30">15-07-30</SPAN></H2><UL><LI> plasmid prep (see table above)</LI><LI> pipette for sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-07-31">15-07-31</SPAN></H2><UL><LI> Sequencing showed that there still is a point mutation in glgB </LI><LI> decided to do a QuickChange with #EY9B# </LI><LI> QuickChange  primers were designed and ordered</LI></UL><H2><SPAN class="mw-headline" id="15-08-03">15-08-03</SPAN></H2><UL><LI> QuickChange with PfuS (E2)</LI><LI> tube 1:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> plasmid template (1:100 dilution) </TD><TD> 3.3 Âµl
</TD></TR><TR><TD> deionized H<SUB>2</SUB>O </TD><TD> 38.2 Âµl
</TD></TR><TR><TD> 10x PfuS buffer (B2) </TD><TD> 5 Âµl
</TD></TR><TR><TD> #BABZ# fwd_primer (10 ÂµM) </TD><TD> 2 Âµl
</TD></TR><TR><TD> dNTP mix (10 mM) </TD><TD> 1 Âµl
</TD></TR><TR><TD> PfuS </TD><TD> 0.5 Âµl
</TD></TR></TBODY></TABLE><UL><LI> tube 2:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> plasmid template (1:100 dilution) </TD><TD> 3.3 Âµl
</TD></TR><TR><TD> deionized H<SUB>2</SUB>O </TD><TD> 38.2 Âµl
</TD></TR><TR><TD> 10x PfuS buffer (B2) </TD><TD> 5 Âµl
</TD></TR><TR><TD> #BOXR# rev_primer (10 ÂµM) </TD><TD> 2 Âµl
</TD></TR><TR><TD> dNTP mix (10 mM) </TD><TD> 1 Âµl
</TD></TR><TR><TD> PfuS </TD><TD> 0.5 Âµl
</TD></TR></TBODY></TABLE><UL><LI> PCR program 1 (105 Â°C heated lid)</LI></UL><P><B>3 cyles</B></P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation (cycle) </TD><TD> 98Â°C </TD><TD> 0'10&quot;
</TD></TR><TR><TD> annealing (cycle) </TD><TD> 59Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> extension (cycle) </TD><TD> 72Â°C </TD><TD> 4'15&quot;
</TD></TR></TBODY></TABLE><UL><LI>control</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> plasmid template (1:100 dilution) </TD><TD> 3.3 Âµl
</TD></TR><TR><TD> deionized H<SUB>2</SUB>O </TD><TD> 40.2 Âµl
</TD></TR><TR><TD> 10x PfuS buffer (B2) </TD><TD> 5 Âµl
</TD></TR><TR><TD> dNTP mix (10 mM) </TD><TD> 1Âµl
</TD></TR><TR><TD> PfuS </TD><TD> 0.5 Âµl</TD></TR></TBODY></TABLE><UL><LI> mix 25 Âµl of tube 1 and 25 Âµl of tube 2 and <B>add 0.5 Âµl PfuS</B></LI></UL><UL><LI> PCR program 2 (105Â°C heated lid)</LI></UL><P><B>15 cycles</B></P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation (cycle) </TD><TD> 98 Â°C </TD><TD> 0'10&quot;
</TD></TR><TR><TD> annealing (cycle) </TD><TD> 59 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> extension (cycle) </TD><TD> 72 Â°C </TD><TD> 4'15&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 Â°C </TD><TD> 10'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> products are in #189R#</LI><LI> make an agarose gel</LI><LI> PCR clean-up</LI><LI> DpnI digest over night</LI></UL><H2><SPAN class="mw-headline" id="15-08-04">15-08-04</SPAN></H2><UL><LI> 20 minutes inactivation</LI><LI> PCR clean-up</LI><LI> transformation of PCR product</LI><LI>repeat the QuikChange with higher template and enzyme concentration because no bands were visible on the gel</LI></UL><UL><LI> QuickChange with PfuS (E2)</LI><LI> tube 1:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> plasmid template #EY9B# (1:100 dilution) </TD><TD> 6.6 Âµl
</TD></TR><TR><TD> deionized H<SUB>2</SUB>O </TD><TD> 34.9 Âµl
</TD></TR><TR><TD> 10x PfuS buffer (B2) </TD><TD> 5 Âµl
</TD></TR><TR><TD> #BABZ# fwd_primer (10 ÂµM) </TD><TD> 2 Âµl
</TD></TR><TR><TD> dNTP mix (10 mM) </TD><TD> 1 Âµl
</TD></TR><TR><TD> PfuS </TD><TD> 0.5 Âµl
</TD></TR></TBODY></TABLE><UL><LI> tube 2:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> plasmid template #EY9B# (1:100 dilution) </TD><TD> 6.6 Âµl
</TD></TR><TR><TD> deionized H<SUB>2</SUB>O </TD><TD> 34.9 Âµl
</TD></TR><TR><TD> 10x PfuS buffer (B2) </TD><TD> 5 Âµl
</TD></TR><TR><TD> #BOXR# rev_primer (10 ÂµM) </TD><TD> 2 Âµl
</TD></TR><TR><TD> dNTP mix (10 mM) </TD><TD> 1 Âµl
</TD></TR><TR><TD> PfuS </TD><TD> 0.5 Âµl
</TD></TR></TBODY></TABLE><UL><LI> PCR program 1 (105 Â°C heated lid)</LI></UL><P><B>3 cyles</B></P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation (cycle) </TD><TD> 98 Â°C </TD><TD> 0'15&quot;
</TD></TR><TR><TD> annealing (cycle) </TD><TD> 59 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> extension (cycle) </TD><TD> 72 Â°C </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><UL><LI>control</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> plasmid template (1:100 dilution) </TD><TD> 6.6 Âµl
</TD></TR><TR><TD> deionized H<SUB>2</SUB>O </TD><TD> 36.9 Âµl
</TD></TR><TR><TD> 10x PfuS buffer (B2) </TD><TD> 5 Âµl
</TD></TR><TR><TD> dNTP mix (10 mM) </TD><TD> 1 Âµl
</TD></TR><TR><TD> PfuS </TD><TD> 0.5 Âµl
</TD></TR></TBODY></TABLE><UL><LI> mix 25 Âµl of tube 1 and 25 Âµl of tube 2 and <B>add 1 Âµl PfuS</B></LI></UL><UL><LI> PCR program 2 (105 Â°C heated lid)</LI></UL><P><B>15 cycles</B></P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation (cycle) </TD><TD> 98 Â°C </TD><TD> 0'15&quot;
</TD></TR><TR><TD> annealing (cycle) </TD><TD> 59 Â°C </TD><TD> 0'30&quot;
</TD></TR><TR><TD> extension (cycle) </TD><TD> 72 Â°C </TD><TD> 5'00&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 Â°C </TD><TD> 10'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> products are in #189R#</LI><LI> make an agarose gel</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDM of glgCAB</B>For tubes 1 and 2 there are faint bands at the correct length, but there are also strong undefined bands on top. For the control (#3) the 7066 bp band is faint as expected
</TD></TR></TBODY></TABLE></DIV><UL><LI> correct bands were visible</LI></UL><LI> DpnI digest over night</LI><H2><SPAN class="mw-headline" id="15-08-05">15-08-05</SPAN></H2><UL><LI> 20 minutes inactivation</LI><LI> PCR clean-up</LI><LI> agarose gel of purified PCR products</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> sample </TH><TH> purified digest product ID </TH><TH> gel result </TH><TH> transformation result
</TH></TR><TR><TD> number 1 </TD><TD> #AZYS# </TD><TD> correct bands visible </TD><TD> ...
</TD></TR><TR><TD> number 2 </TD><TD> #ATM8# </TD><TD> correct bands visible </TD><TD> ...
</TD></TR><TR><TD> control </TD><TD> #SWEV# </TD><TD> no 7000 bp bands </TD><TD> ...
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-08-06">15-08-06</SPAN></H2><UL><LI> repeat DpnI digest of Gibson and SDM products</LI><LI> transformation into NEB10Î²</LI></UL><H2><SPAN class="mw-headline" id="15-08-07">15-08-07</SPAN></H2><UL><LI> masterplate of clones from trafo plate</LI></UL><H2><SPAN class="mw-headline" id="15-08-08">15-08-08</SPAN></H2><UL><LI> take out masterplates from 37 Â°C and put them in the fridge</LI></UL><H2><SPAN class="mw-headline" id="15-08-09">15-08-09</SPAN></H2><UL><LI> overnights of 10 clones for test digest on monday</LI></UL><H2><SPAN class="mw-headline" id="15-08-10">15-08-10</SPAN></H2><UL><LI> plasmid prep and cryos of clones</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> test digest result </TH><TH> sequencing result
</TH></TR><TR><TD> glgCAB in pSB1C3 #1 </TD><TD> #XWFL# </TD><TD> #A919#  </TD><TD> good </TD><TD> bad
</TD></TR><TR><TD> glgCAB in pSB1C3 #2 </TD><TD> #ZNNL# </TD><TD> #QOZV# </TD><TD> good </TD><TD> good
</TD></TR><TR><TD> glgCAB in pSB1C3 #3 </TD><TD> #41ZT# </TD><TD> #RW33# </TD><TD> good </TD><TD> good
</TD></TR><TR><TD> glgCAB in pSB1C3 #4 </TD><TD> #6THN# </TD><TD> #DTBM# </TD><TD> good </TD><TD> good
</TD></TR><TR><TD> glgCAB in pSB1C3 #5 </TD><TD> #VMV3# </TD><TD> #COOX# </TD><TD> 2 bands </TD></TR><TR><TD> glgCAB in pSB1C3 #6 </TD><TD> #LRAT# </TD><TD> #AO86# </TD><TD> length not clear </TD></TR><TR><TD> glgCAB in pSB1C3 #7 </TD><TD> #3W3Y# </TD><TD> #TMXD# </TD><TD> length not clear </TD></TR><TR><TD> glgCAB in pSB1C3 #8 </TD><TD> #SHHS# </TD><TD> #XB19# </TD><TD> good </TD><TD> point mutation
</TD></TR><TR><TD> glgCAB in pSB1C3 #9 </TD><TD> #VOBO# </TD><TD> #ATH8# </TD><TD> good </TD><TD> point mutation
</TD></TR><TR><TD> glgCAB in pSB1C3 #10 </TD><TD> #33C1# </TD><TD> #LY4Z# </TD><TD> good </TD><TD>point mutation
</TD></TR></TBODY></TABLE><UL><LI> test digest</LI></UL><H2><SPAN class="mw-headline" id="15-08-11">15-08-11</SPAN></H2><UL><LI> pipette for sequencing with one primer that covers point mutation (#RYZX#)</LI><LI> bring the samples to the Fraunhofer institute</LI></UL><H2><SPAN class="mw-headline" id="15-08-14">15-08-14</SPAN></H2><UL><LI> overnights of #6THN#, #ZNNL# and #41ZT#for further sequencing of glgCAB clones</LI></UL><H2><SPAN class="mw-headline" id="15-08-15">15-08-15</SPAN></H2><UL><LI> plasmid prep of CAB clones
</LI><LI> products #QOZV# and #DTBM#</LI></UL><H2><SPAN class="mw-headline" id="15-08-17">15-08-17</SPAN></H2><UL><LI> digest #EY9B# (tube1), #QOZV# (tube 2) and #DTBM# (tube 3) with EcoRI and PstI to transfer into pSB1A30
</LI><LI> digest of #EY9B# did not work, so we use an digestion product from #TF4M# (tube 5)</LI></UL><P>ligation
</P><TABLE class="wikitable"><TBODY><TR><TH> tube no </TH><TH> insert </TH><TH> backbone </TH><TH> purpose
</TH></TR><TR><TD> 2 </TD><TD> #QOZV# </TD><TD> #OHOZ# tube # 1 </TD><TD> new glgCAB #2 in pSB1A30
</TD></TR><TR><TD> 3 </TD><TD> #DTBM# </TD><TD> #OHOZ# tube # 1 </TD><TD> new glgCAB #4 in pSB1A30
</TD></TR></TBODY></TABLE><UL><LI> products are stored in #3A9X#</LI></UL><UL><LI> transform in DH5Î±</LI></UL><H2><SPAN class="mw-headline" id="15-08-18">15-08-18</SPAN></H2><UL><LI> sequencing results show that #QOZV# and #DTBM# are both missing glgC</LI><LI> all transformations were thrown away</LI><LI> we will test glgCAB with one point mutation</LI></UL><H2><SPAN class="mw-headline" id="15-08-19">15-08-19</SPAN></H2><UL><LI> digest #8WE4# and #EY9B# with EcoRI and PstI to ligate it in pSB1A30 for expression</LI><LI> both look good on gel</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>digest of glgCAB in pSB1C3</B>For tubes 1 and 2 there are strong bands at the correct length (2000 and 5000bp)
</TD></TR></TBODY></TABLE></DIV><UL><LI> products stored in #WV93#</LI></UL><H2><SPAN class="mw-headline" id="15-08-20">15-08-20</SPAN></H2><UL><LI>ligation of cut #8WE4# and #EY9B# in pSB1A30</LI><LI> heat shock in DH5Î±</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> tube no </TH><TH> insert </TH><TH> backbone </TH><TH> purpose
</TH></TR><TR><TD> 2 </TD><TD> #8WE4# </TD><TD> #OHOZ# tube # 1 </TD><TD> glgCAB with point mutation in pSB1A30
</TD></TR><TR><TD> 3 </TD><TD> #EY9B# </TD><TD> #OHOZ# tube # 1 </TD><TD> glgCAB with point mutation in pSB1A30
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-08-21">15-08-21</SPAN></H2><UL><LI> overnight cultures and master plate of DH5Î± glgCAB clones</LI></UL><H2><SPAN class="mw-headline" id="15-08-22">15-08-22</SPAN></H2><UL><LI> make cryo cultures of glgCAB in pSB1A30 DH5Î± overnight cultures
</LI><LI> plasmid prep afterwards</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> no </TH><TH> What? </TH><TH> strain </TH><TH> Cryo </TH><TH> plasmid
</TH></TR><TR><TD> 1 </TD><TD> glgCAB in A30 (cut from #EY9B#) </TD><TD> DH5Î± </TD><TD>  #ADWX# </TD><TD> #XTEN#
</TD></TR><TR><TD> 2 </TD><TD> glgCAB in A30 (cut from #EY9B#) </TD><TD> DH5Î± </TD><TD> #ALXK# </TD><TD> #CTMV#
</TD></TR><TR><TD> 3</TD><TD> glgCAB in A30 (cut from #EY9B#) </TD><TD> DH5Î± </TD><TD> #AR46# </TD><TD> #Q13F#
</TD></TR><TR><TD> 4 </TD><TD> glgCAB in A30 (cut from #EY9B#) </TD><TD> DH5Î± </TD><TD> #3MKZ# </TD><TD> #YFXC#
</TD></TR><TR><TD> 2 </TD><TD> glgCAB in A30 (cut from #8WE4#) </TD><TD> DH5Î± </TD><TD> #TNR8# </TD><TD> #KSZQ#
</TD></TR><TR><TD> 3</TD><TD> glgCAB in A30 (cut from #8WE4#) </TD><TD> DH5Î± </TD><TD> #R4CZ# </TD><TD> #KRYH#
</TD></TR><TR><TD> 4 </TD><TD> glgCAB in A30 (cut from #8WE4#) </TD><TD> DH5Î± </TD><TD> #HMSM# </TD><TD> #NPHP#
</TD></TR></TBODY></TABLE><UL><LI> 3A assembly of glgAB (#CT8H#) and glgC (#4D6H#) in pSB1C3 (already cut: #CQ1V# tube V)
</LI><LI> digest #4D6H# with EcoRI and SpeI, product in #KFA1#</LI><LI> digest #CT8H# with XbaI and PstI, looked bad ont the gel: concentration was too low</LI></UL><UL><LI> make an overnight culture of #1Q88# to produce new #CT8H#</LI></UL><H2><SPAN class="mw-headline" id="15-08-23">15-08-23</SPAN></H2><UL><LI> plasmid prep of new #CT8H# , concentration measurement will be done tomorrow</LI></UL><H2><SPAN class="mw-headline" id="15-08-24">15-08-24</SPAN></H2><UL><LI> test digest of glgCAB constructs with EcoRI</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> tube no </TH><TH> plasmid </TH><TH> result
</TH></TR><TR><TD> 1 </TD><TD> #Q13F# </TD><TD> bad
</TD></TR><TR><TD> 2 </TD><TD> #YFXC# </TD><TD> bad
</TD></TR><TR><TD> 3 </TD><TD> #NPHP# </TD><TD> bad
</TD></TR><TR><TD> 4 </TD><TD> #KRYH# </TD><TD> bad
</TD></TR><TR><TD> 5 </TD><TD> #KSZQ# </TD><TD> good
</TD></TR><TR><TD> 6 </TD><TD> #XTEN# </TD><TD> bad
</TD></TR><TR><TD> 7 </TD><TD> #CTMV# </TD><TD> good
</TD></TR></TBODY></TABLE><UL><LI> transformation of #KSZQ# in BL21 Gold</LI><LI> digest of #CT8H# with XbaI and PstI (stored in #3649#)
</LI><LI> gel looks good</LI></UL><LI> ligation of glgAB (#3649#), glgC (#KFA1#) in pSB1C3 (#CQ1V# tube5)
</LI><LI> tube L1 (also stored in #3649#)</LI><LI>transformation of new glgCAB into DH5Î±</LI><LI> electroporation of #KSZQ# in BL21 ÎX and BL21 ÎP electrocompetent</LI><H2><SPAN class="mw-headline" id="15-08-25">15-08-25</SPAN></H2><UL><LI> masterplate and overnight cultures of #KSZQ# in BL21 Gold, BL21 ÎX and BL21 ÎP in LB+A</LI><LI> masterplate and overnight of new glgCAB in DH5Î± in LB+C</LI><LI> new transformation of #KSZQ# in BL21 Gold</LI></UL><H2><SPAN class="mw-headline" id="15-08-26">15-08-26</SPAN></H2><UL><LI> colony pcr of mutated GlgCAB (CAB')(in pSB1A30)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone </TH><TH> tube #
</TH></TR><TR><TD> CAB' in pSB1A30 in BL21 ÎX clone #1-12 </TD><TD> 1-12
</TD></TR><TR><TD> CAB' in pSB1A30 in BL21 ÎP clone #1-4 </TD><TD> 13-16
</TD></TR><TR><TD> CAB' in pSB1A30 in BL21 ÎP clone #7-12 </TD><TD> 17-22
</TD></TR><TR><TD> CAB' in pSB1A30 in BL21 ÎP clone #5+6 </TD><TD> 22-24
</TD></TR></TBODY></TABLE><UL><LI> none of the clones had the expected length of 5362 bp on the gel
</LI><LI> probably the fragment was too long</LI><LI> do 10 overnights of glgCAB in delta P each for test digest tomorrow, all clones in delta X were red</LI><LI> then new overnight of good one, main culture on friday, SDS on friday</LI></UL><UL><LI> new trafo of CAB' (mutated) in BL21 Gold wild type</LI></UL><UL><LI> plasmid prep and cryos of NEW GlgCAB in psB1C3</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone no </TH><TH> cryo </TH><TH> plasmid </TH><TH> digestion tube </TH><TH> result of digestion
</TH></TR><TR><TD> 1 </TD><TD> #OR9C# </TD><TD> #VCWE# </TD><TD> 1 </TD><TD> bad
</TD></TR><TR><TD> 3 </TD><TD> #DYWY# </TD><TD> #S9H8# </TD><TD> 2 </TD><TD> bad
</TD></TR><TR><TD> 5 </TD><TD> #FQ69# </TD><TD> #9CDT# </TD><TD> 3 </TD><TD> bad
</TD></TR><TR><TD> 7 </TD><TD> #R4MF# </TD><TD> #MDSB# </TD><TD> 4 </TD><TD> many bands, but also the correct one
</TD></TR><TR><TD> 8 </TD><TD> #8MKC# </TD><TD> #KZRZ# </TD><TD> 5 </TD><TD> many bands, but also the correct one
</TD></TR><TR><TD> 9 </TD><TD> #NRSN# </TD><TD> #PTRE# </TD><TD> 6 (PTRE)</TD><TD> bad
</TD></TR><TR><TD> 10 </TD><TD> #3EHE# </TD><TD> accidently discarded </TD><TD> bad
</TD></TR><TR><TD> A </TD><TD> #KLH8# </TD><TD> #RRPP# </TD><TD> 7 </TD><TD> bad
</TD></TR><TR><TD> B </TD><TD> #OC1P# </TD><TD> accidently also #PTRE#  </TD><TD> 8 (PTRE*) </TD><TD> bad
</TD></TR><TR><TD> C </TD><TD> #4EHF# </TD><TD> #94RY# </TD><TD> 9 </TD><TD> bad
</TD></TR></TBODY></TABLE><UL><LI>tubes 4+5 look good, products stored in#ZY4A#</LI></UL><UL><LI> digest of NEW GlgCAB
</LI><LI> gel and inactivation </LI></UL><LI> then ligate tube 4+5 (#MDSB# and #KZRZ#) into pSB1A30 (#OHOZ# tube 1), ligations also in #ZY4A# </LI><LI> transformation in DH5Î±</LI><LI> for sequencing: New GlgCAB in pSB1C3, clones that looked good in test digest</LI><LI><B>all clones from new CAB looked bad in test digest with EcoRI. We will not continue working with these ones..</B></LI><H2><SPAN class="mw-headline" id="15-08-27">15-08-27</SPAN></H2><UL><LI> plasmid prep of glgCAB' in delta P (clones # 1-10) </LI><LI> test digest with eco
</LI><LI> all clones had the wrong length (ca. 3200 bp) </LI></UL><LI> trafo of glgCAB in pSB1A30 in BL21 WT did not work again</LI><P><B>We decided to sequence #KSZQ#, #MDSB# and #KZRZ# and and to digest pSB1A30 again (tube #T6X9#) and ligate CAB' into new pSB1A30</B></P><UL><LI> made a gel to test several constructs</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Cloning glgCAB'(mutated) in pSB1A30</B>All constructs that were used for glgCAB' assembly in A30 had the correct length. Also a fragment of the correct length (complete CAB in A30) can be seen in the coloumn for the ligation tube (7202bp).
</TD></TR></TBODY></TABLE></DIV><UL><LI> CAB' (#8WE4#) was ligated with new digested pSB1A30 (#T6X9#) and cloned into Bl21 Gold DE3</LI></UL><H2><SPAN class="mw-headline" id="15-08-28">15-08-28</SPAN></H2><UL><LI> a few clones were visible on the transformation plate of CAB' in pSB1A30
</LI><LI> make overnight cultures and a master plate</LI></UL><H2><SPAN class="mw-headline" id="15-08-29">15-08-29</SPAN></H2><UL><LI> plasmid prep and test digest of CAB' in pSB1A30 clone #2 (plasmid: #4XWH# cryo: #3OVK#) 
</LI><LI> gel looks bad, no correct bands visible for double digest</LI></UL><LI> new overnight of CAB' in pSB1A30 clone #1 in order to repeat the above (it did not grow)</LI><H2><SPAN class="mw-headline" id="15-08-30">15-08-30</SPAN></H2><UL><LI> try to assemble glgCAB' in pSB1A30 via CPEC
</LI><LI> use pSB1A30 from #T6X9#</LI></UL><P>PCR to amplify CAB'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> amount
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 23.6 Âµl
</TD></TR><TR><TD> #O3M6# </TD><TD> 0.6 Âµl
</TD></TR><TR><TD> #NBZZ# </TD><TD> 0.6 Âµl
</TD></TR><TR><TD> B2 Buffer </TD><TD> 3 Âµl
</TD></TR><TR><TD> dNTPs </TD><TD> 0.6 Âµl
</TD></TR><TR><TD> PfuS (E2) </TD><TD> 0.3 Âµl
</TD></TR></TBODY></TABLE><P>PCR program
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [Â°C] </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 63 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 2'37&quot; (5039 bp)
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> heat shock of both BBa_C0012 from kit plate and ligation product &quot;Lig iJR&quot; (#WV93#) into BL21 Gold DE3</LI><LI> make three overnight cultures of #TNR8# to have enough plasmid for sequencing</LI><LI> do DpnI digest overnight (of PCR product for CPEC)</LI></UL><H2><SPAN class="mw-headline" id="15-08-31">15-08-31</SPAN></H2><UL><LI> heat inactivate DpnI digest, purify and measure concentration (product in #N13D#), do CPEC with #T6X9#, transformation into BL21 Gold</LI></UL><P>CPEC program
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature </TH><TH> time
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation (cycle) </TD><TD> 98 </TD><TD> 0'10&quot;
</TD></TR><TR><TD> annealing (cycle) </TD><TD> 55 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation (cycle) </TD><TD> 72 </TD><TD> 1'52&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 10'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> prep overnight cultures and prepare sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-09-01">15-09-01</SPAN></H2><UL><LI> do masterplates and overnights from Trafo plate (CAB' in A30 via CPEC) </LI><LI> digest glgAB (#CT8H#) and pSB1C30 (#MFNP#) with EcoRI and PstI</LI><LI> ligate glg AB into pSB1C3 (#OLAE# tube A) and pSB1C30 (#OQ81#)
</LI><LI> ligation products stored in #OQ81#</LI><LI> heat shock into DH5Î± (pSB1C3) and BL21 (pSB1C30)</LI></UL><H2><SPAN class="mw-headline" id="15-09-02">15-09-02</SPAN></H2><UL><LI> plasmid prep of overnights </LI><LI> test digest
</LI><LI> gel showed that RFP is in all constructs of CAB </LI></UL><LI> the backbone might be not digested correctly with DpnI after PCR </LI><LI> DpnI digest of #T6X9#</LI><LI> overnights and masterplates of glgAB in pSB1C3 and pSB1C30</LI><H2><SPAN class="mw-headline" id="15-09-03">15-09-03</SPAN></H2><UL><LI> sequencing shows that just glgC was in glgCAB ligation attempts via Biobrick assembly</LI><LI><B>sequencing showed that #KSZQ# (glgCAB' in pSB1A30) is PERFECT </B></LI><LI> cryo and plasmid prep of overnights of glgAB in C3 and C30</LI><LI> only clones 1,2,3,6,7 and glgAB in C30 were prepped becuase the other ones were red
</LI><LI> test digest with EcoRI and PstI</LI><LI> only clones #3+6 of glgAB in pSB1C3 look good</LI><LI> subcloning into pSB1C30 did not work</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone </TH><TH> cryo ID </TH><TH> test digest result </TH><TH> plasmid ID
</TH></TR><TR><TD> glgAB in C3 clone #1 </TD><TD> #XM94# </TD><TD> bad </TD><TD> -
</TD></TR><TR><TD> glgAB in C3 clone #2 </TD><TD> #SZW1# </TD><TD> bad </TD><TD> -
</TD></TR><TR><TD> glgAB in C3 clone #3 </TD><TD> #MQQ4# </TD><TD> perfect </TD><TD> #HAMP#
</TD></TR><TR><TD> glgAB in C3 clone #6 </TD><TD> #1KA4# </TD><TD> perfect </TD><TD> #YC8M#
</TD></TR><TR><TD> glgAB in C3 clone #7 </TD><TD> #B9ZD#</TD><TD> bad </TD><TD> -
</TD></TR><TR><TD> glgAB in C30 clone #1 </TD><TD> #AWWP# </TD><TD> bad </TD><TD> -
</TD></TR></TBODY></TABLE><UL><LI> overnights of good glgAB clones #3+6 from masterplate</LI></UL><UL><LI> PCR purification of DpnI digested #T6X9#</LI></UL><UL><LI> transformation of <B>5 Âµl</B> #KSZQ# into BL21 Gold DE3</LI><LI> M9 overnights of BL21 WT, BL21 glgA, BL21 glgB and BL21 ÎP for iodine staining</LI><LI> LB overnights cultures of good glgAB clones for growth experiments and SDS gels</LI></UL><H2><SPAN class="mw-headline" id="15-09-04">15-09-04</SPAN></H2><UL><LI> master plates and overnights of #KSZQ# transformation clones</LI></UL><UL><LI> pipette sequencing of glgAB clones</LI></UL><UL><LI>for AB in C30: heat shock ligation product (in #OQ81#) into DH5Î±
</LI><LI>for new C30 backbone: amplify #MFNP# with #TSTV# and #FYTO#,digest with 2ÂµL DpnI (for 4 hours), heatinactivate, digest with E+P, ligate, transform into DH5Î±</LI></UL><H2><SPAN class="mw-headline" id="15-09-05">15-09-05</SPAN></H2><UL><LI>cryos and plasmid prep of BL21 CAB' in A30, then test digest with EcoRI (Sciebo: 15-09-05 Test digest of CAB in A30)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> test digest result </TH><TH> sequencing result
</TH></TR><TR><TD> glgCAB' in pSB1A30 #1 </TD><TD> #WCQY# </TD><TD> #YC3V#  </TD><TD> good
</TD></TR><TR><TD> glgCAB' in pSB1A30 #2 </TD><TD> #V4YZ# </TD><TD> #3KZZ# </TD><TD> bad
</TD></TR><TR><TD> glgCAB' in pSB1A30 #3 </TD><TD> #6AME# </TD><TD> #XQ4R# </TD><TD> bad
</TD></TR><TR><TD> glgCAB' in pSB1A30 #4 </TD><TD> #8HCP# </TD><TD> #MKHV# </TD><TD> bad
</TD></TR><TR><TD> glgCAB' in pSB1A30 #5 </TD><TD> #LFRD# </TD><TD> #S3RW# </TD><TD> good
</TD></TR><TR><TD> glgCAB' in pSB1A30 #6 </TD><TD> #B11R# </TD><TD> #BBBF# </TD><TD> bad
</TD></TR><TR><TD> glgCAB' in pSB1A30 #7 </TD><TD> #FNVQ# </TD><TD> #QM4Y# </TD><TD> bad
</TD></TR></TBODY></TABLE><UL><LI>do new overnights of positive clones (#1+5(for SDS))</LI></UL><UL><LI>screen Trafo plates of AB in C30 in DH5Î±
</LI><LI> all clones were white</LI><LI> do 5 overnights of each new and old ligation</LI><LI> parallel masterplates</LI></UL><H2><SPAN class="mw-headline" id="15-09-06">15-09-06</SPAN></H2><UL><LI>do plasmid prep (see Manuals/Plasmid Prep!!!) and test digest with EcoRI of DH5Î± glgAB in pSB1C30</LI><LI>SDS-PAGE (Ladder-GlgCAB clone #1, GlgCAB #5,GlgA,GlgB,Wildtype for control)
</LI><LI>glgC did not grow </LI></UL><LI>HPLC samples</LI><LI>new 10 ml overnights (LB+20mM Glucose) of WT,A,B,deltaX,deltaP</LI><LI>overnights of GlgAB in pSB1C30 for CRYOS AND PREP </LI><LI>overnights of BL21 GlgCAB in pSB1A30 #1, #5 (LB+A+IPTG+20mM Glucose) and BL21 WT</LI><H2><SPAN class="mw-headline" id="15-09-07">15-09-07</SPAN></H2><UL><LI>cryos and prep DH5Î± glgAB in pSB1C30</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> Cryo </TH><TH> purified plasmid
</TH></TR><TR><TD> Old glgAB #2 </TD><TD>#8ZZ4# </TD><TD> #ZBVK#
</TD></TR><TR><TD> Old glgAB #3 </TD><TD> #8MCD# </TD><TD> #14XB#
</TD></TR><TR><TD> New glgAB #1 </TD><TD> #N96D# </TD><TD> #C9ZY#
</TD></TR><TR><TD> New glgAB #2 </TD><TD> #NDEF# </TD><TD> #K8XD#
</TD></TR></TBODY></TABLE><UL><LI>pipette DH5Î± glgAB in pSB1C30 and BL21 glgCAB' in pSB1A30 for sequencing, prepare sequencing sheet</LI><LI>transform four glgAB in pSB1C30 into BL21 Gold (see table above)</LI><LI>overnights of GlgCAB in pSB1A30 #1 and #5, GlgC in A30 (#OHES#) and WT Bl21 Gold for repeated iodine staining (incubate since 9 p.m.!)</LI></UL><H2><SPAN class="mw-headline" id="15-09-08">15-09-08</SPAN></H2><UL><LI>Prep glgCAB in pSB1A30 #1 for remaining sequencing tubes</LI><LI>screen trafo of GlgAB in pSB1C30 BL21 Gold
</LI><LI>do masterplates</LI></UL><H2><SPAN class="mw-headline" id="15-09-09">15-09-09</SPAN></H2><UL><LI> sequencing did not work</LI><LI> make two overnights of both CAB clones #1 and #5 for new sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-09-10">15-09-10</SPAN></H2><UL><LI> pipette #S3RW# and #YC3V# again for sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-09-11">15-09-11</SPAN></H2><UL><LI> both #S3RW# and #YC3V# look perfect in sequencing!
</LI><LI> confirmed glgCAB' in pSB1A30</LI></UL><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH> BioBrick </TH><TH> gene </TH><TH> backbone </TH><TH> confirmed cryo </TH><TH> confirmed plasmid
</TH></TR><TR><TD> BBa_K1585320 </TD><TD> glgAB </TD><TD> pSB1A3 </TD><TD> #1Q88# </TD><TD> #CT8H# (ligation sites confirmed)
</TD></TR><TR><TD> BBa_K1585321 </TD><TD> glgCAB </TD><TD> pSB1C3 </TD><TD> #MXKQ# </TD><TD> #8WE4#
</TD></TR><TR><TD> BBa_K1585321 </TD><TD> glgCAB' </TD><TD> pSB1A30 </TD><TD> #WCQY# </TD><TD> #KSZQ#
</TD></TR></TBODY></TABLE></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">â²</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>