<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Lab_Glycogen_Knockouts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Lab/Glycogen/Knockouts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmGlycogenKnockoutGeneration" href="/Team:Aachen/Lab/Glycogen/Knockouts#Knockout_Generation"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Knockout Generation</P></DIV></DIV></DIV></DIV></A></DIV><DIV><A class="square rmbg pull-right rmPlasmidconstruction" href="/Team:Aachen/Lab/Glycogen/Knockouts#Plasmid_Construction"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Plasmid Construction</P></DIV></DIV></DIV></DIV></A></DIV><H2><SPAN class="mw-headline" id="How_Glycogen_Degradation_Works">How Glycogen Degradation Works</SPAN></H2><P>The glycogen degradation in <I>E. coli</I> is performed by two enzymes:
</P><UL><LI> glycogen phosphorylase (encoded by <I>glgP</I>)</LI><LI> debranching enzyme (encoded by <I>glgX</I>)</LI></UL><P>GlgP removes glucose units from the end of linear glycogen chains while the debranching enzyme GlgX cleaves α-1,6-glycosidic bonds, therefore degrading branches. 
</P><H2><SPAN class="mw-headline" id="Genome_Editing_using_Cas9">Genome Editing using Cas9</SPAN></H2><P>When it became clear that we had to knock out <I>glgX</I> and <I>glgP</I>, we began looking for an efficient and reliable method to generate the knockout strains. Only a few weeks ago, at the beginning of April, a new method for genome editing in <I>Escherichia coli</I> was published<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>.
</P><P>
The method is based on a two-plasmid system, in which the first plasmid, the pCas, contains the cas9 gene and a temperature-sensitive repA101ts ori. To increase recombination efficiency in <I>E. coli</I>, it also expresses lambda-Red genes behind an inducible arabinose-promoter. Upon induction with IPTG, a sgRNA is transcribed to cut the pMB1 ori of the second plasmid. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Plasmid map of pCas</B>pCas contains a repA101ts ori for self-curing, arabinose-induced lambda-Red genes, lacI and araC repressor genes, the Cas9 gene, IPTG-induced sgRNA targeting pMB1 and a kanamycin resistance gene.
</TD></TR></TBODY></TABLE></DIV><P>
The other essential plasmid for this method is the pTargetT, which leads the Cas9 protein to the DNA part to be cut. It provides the sgRNA for targeting the genomic sequence as well as the repair template with a high-copy pMB1 ori. The repair templates consist of two 400 bp homology arms for homology-directed repair. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>General plasmid map of a targeting plasmid</B>A sgRNA with the N20-sequence targeting a protospacer in the genome is expressed behind a J23119 strong constitutive promoter. Next to the ampR gene and the pMB1 ori, the plasmid carries the repair template that is introducing the desired edit.
</TD></TR></TBODY></TABLE></DIV><P><SPAN style="color:transparent">a</SPAN></P><H1><SPAN class="mw-headline" id="Plasmid_Construction">Plasmid Construction</SPAN></H1><P>The CRISPR/Cas9 system offers a variety of genome editing applications such as disruption, insertion and deletion. It works with a single guide RNA (sgRNA) which includes 20 base pairs that are complementary to a specific DNA region that has to be cut; the N20 sequence. It can be any 20-30 base pair DNA sequence behind a PAM site, the Protospacer-Adjacent-Motif. Those can be found almost everywhere in the genome. 
</P><P>
In our project we tested two strategies:  for the debranching enzyme we cleaved the gene at the beginning of the coding sequence, using one sgRNA. In addition, we designed a short DNA sequence (ATAGGCGAAATGTAAGCGGCGTGCCATGAAAAC) that contains a stop codon in each frame and translates to IGEM*AACHEN in the first frame, called iGEM Aachen MultiSTOP. This was inserted between the two homology arms in the repair template. For the glycogen phosphorylase we used two sgRNAs to cut <I>glgP</I> on both sides to completely eliminate this gene. 
</P><DIV class="figure-double col-md-12"><DIV class="figure figure-doubleM col-md-6"><TABLE><TBODY><TR><TD><B>Targeting plasmid glgX</B>To target <I>glgX</I>, we selected a high-scoring CRISPR-site with the sequence 5'-GTAATGCGCGCCGAGGGGAGCGG ca. 25 bp downstream of the glgX start codon. In addition to the two homology arms in the repair template, we designed a short DNA sequence (ATAGGCGAAATGTAAGCGGCGTGCCATGAAAAC) that contains a stop codon in each frame and translates to IGEM*AACHEN in the first frame. This is flanked by the two homology arms.
</TD></TR></TBODY></TABLE></DIV><DIV class="figure figure-doubleM col-md-6"><TABLE><TBODY><TR><TD><B>Targeting plasmid glgP</B>For <I>glgP</I> we used two high scoring CRISPR-sites to eliminate the whole gene.
</TD></TR></TBODY></TABLE></DIV></DIV><P><SPAN style="color:transparent">a</SPAN></P><P>After initial problems with the spectinomycin resistance for pTargetT, we decided to build the targeting plasmid in a pSB1A3 BioBrick backbone. The targeting plasmids for <I>glgX</I> and <I>glgP</I> were assembled via CPEC from synthesized gBlocks, PCR-amplified pSB1A3 backbone and fragments amplified from the <I>E. coli</I> genome. Our results showed that the single cut strategy has a remarkably higher efficiency. For the <I>glgP</I> knockout, additional arabinose induction of the lambda-red genes was necessary for high efficiency. Ultimately, both strategies led to a successful gene knockout.
</P><P><SPAN style="color:transparent">a</SPAN></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> resistance </TH><TH> plasmid ID </TH><TH> cryo ID
</TH></TR><TR><TD> pCas </TD><TD> Kan </TD><TD> #EANB# </TD><TD> #YPKO#
</TD></TR><TR><TD> anti glgX in pSB1A3 </TD><TD> Amp </TD><TD> #1DEN# </TD><TD> #BAB3#
</TD></TR><TR><TD> anti glgP in pSB1A3 </TD><TD> Amp </TD><TD> #C1QW# </TD><TD> #M69X#
</TD></TR></TBODY></TABLE><P><SPAN style="color:transparent">a</SPAN></P><H2><SPAN class="mw-headline" id="Laboratory_Notebook"><A href="/Team:Aachen/Notebook/Documentation/Glycogen_Targeting_Plasmids" title="Team:Aachen/Notebook/Documentation/Glycogen Targeting Plasmids"> Laboratory Notebook</A></SPAN></H2><P><SPAN style="color:transparent">a</SPAN></P><H1><SPAN class="mw-headline" id="Knockout_Generation">Knockout Generation</SPAN></H1><P>After the construction of the targeting plasmids for <I>glgP</I> and <I>glgX</I> was completed, the process of generating the knockouts began. At first, the pCas was heatshocked into <I>E.coli</I> BL21 Gold (DE3) and NEB10β cells. To enable the introduction of targeting plasmids, we made these cells electrocompetent<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP>.
Following the electroporation of pTargetT for <I>glgX</I> and <I>glgP</I>, many clones were screened by genomic PCRs and sent into sequencing for verification. Succeeding the curing of both plasmids through IPTG induction and temperature change, we continued working with our new  BL21 Gold (DE3) knockout strains combined with the methanol- and the glycogen polycistronic construct.
</P><P>
We worked on the generation of <I>glgX</I> and <I>glgP</I> single knockouts in both strains as well as on a double knockout. However, we stopped working with NEB10β cells early on in the project, because our polycistronic <I>glgCAB</I> construct was built behind a T7 promoter, so it could only be expressed in BL21 Gold (DE3) cells. 
</P><H2><SPAN class="mw-headline" id="Results_2">Results</SPAN></H2><P>Both, the <I>glgP</I> and <I>glgX</I> knockouts in BL21 Gold (DE3) were successfully generated and the iGEM Aachen MultiSTOP was inserted in the genome.  We observed that the single cut knockout for <I>glgX</I> had a higher efficiency when clones were screened. However, after additional induction with arabinose the <I>glgP</I> knockout was successfully completed as well. In NEB10β we not only did the single knockouts, but also performed a double knockout of both glycogen degrading enzymes. 
</P><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>Edited genomic region of <I>glgP</I> with iGEM Aachen MultiSTOP</B>A DNA sequence coding for IGEM*AACHEN in amino acids was inserted in the repair template. As shown in the chromatograms, it is now written in the <I>E. coli</I> genome.
</TD></TR></TBODY></TABLE></DIV><TABLE class="wikitable"><TBODY><TR><TH> strain </TH><TH> cryo with pCas </TH><TH> cryo without plasmids (LB) </TH><TH> cryo without plasmids (M9)
</TH></TR><TR><TD> NEB10β ΔglgX (1 mutation in the Multi Stop) </TD><TD> #PH3R# </TD><TD> #CL6W# </TD></TR><TR><TD> NEB10β ΔglgP </TD><TD> #XHEK# </TD><TD> #BXR1# </TD></TR><TR><TD> NEB10β ΔglgX ΔglgP </TD><TD> #LNL4# </TD><TD> #WHPX# </TD></TR><TR><TD> BL21 Gold (DE3) ΔglgX </TD><TD> #ZPKP# </TD><TD> #C81O# </TD><TD> #XFWX# (confirmed again by sequencing)
</TD></TR><TR><TD> BL21 Gold (DE3) ΔglgP </TD><TD> #3RZ1# </TD><TD> #FVNV# </TD><TD> #4BW3#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Laboratory_Notebook_2"><A href="/Team:Aachen/Notebook/Documentation/Glycogen_Knockout_Generation" title="Team:Aachen/Notebook/Documentation/Glycogen Knockout Generation"> Laboratory Notebook</A></SPAN></H2><P><SPAN style="color:transparent">a</SPAN></P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Jiang Y, Chen B, Duan C, Sun B, Yang J, Yang S. Multigene editing in the Escherichia coli genome via the CRISPR-Cas9 system. Appl Environ Microbiol. 2015 Apr;81(7):2506-14. doi: 10.1128/AEM.04023-14. Epub 2015 Jan 30. PubMed PMID:25636838; PubMed Central PMCID: PMC4357945.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="http://openwetware.org/wiki/Electrocompetent_Cells#Materials">http://openwetware.org/wiki/Electrocompetent_Cells#Materials</A></SPAN></LI></OL></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>