Team:Aachen/Notebook/Documentation/Glycogen Polycistronic Expression
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Combined Expression
Laboratory Notebook
15-06-09
digest glgA/B plasmids (stored in #SZKD#)
name
plasmid ID
enzyme 1
enzyme 2
A1 (glgA)
#M1M6#
EcoRI
SpeI
B1 (glgB)
#YHT3#
XbaI
PstI
pSB1A3
#TRTN#
EcoRI
PstI
15-06-10
after digest glg A and B:
assemble A1 with B1 in pSB1A3 (ligation tube: A+B)
stored in #QHHR#
transformation into DH5α
15-06-11
master plates of glgA+B in pSB1A3 (LB+A)
15-06-12
colony PCR on glgAB construct to identify positive clones
make overnight cultures of glgAB construct
clone number
cryo ID
plasmid ID
sequencing result
glgA.glgB in pSB1A3 #5
#Y4K4#
#P6QM#
glgB is missing
15-06-13
make cryos of glgAB construct (table above)
plasmid prep of glgAB (table above)
15-06-15
measured concentrations of purified glgAB constructs
15-06-16
digested glgAB plasmid (#P6QM#) with XbaI and PstI (product in #TZTD#) gel had three bands! -> new digest and ligation
15-06-17
Digestions with 40' digest and 20' inactivation
(previous digests A1, B1 were discarded from #SZKD# because agarose gels were really ugly)
gene
enzymes
expectation
observation
products
glgA (#M1M6#)
EcoRI + SpeI
3533, 2051, 1482
confirmed
#SZKD#:A
glgB (#YHT3#)
XbaI + PstI
4286, 2234, 2052
confirmed
#SZKD#:B
Ligations and transformations
construct
part I
part II
backbone
product
host
glgAB in pSB1A3
#SZKD#:A
#SZKD#:B
#SZKD#:pSB1A3
#99MC#:LAB
BL21
15-06-18
transformation of glgAB in pSB1A3 in BL21 did not work!
ligation of glgAB on a gel shows 3 independent bands
therefore, we cut pSB1A3 again with EcoRI and PstI
Digestions with 40' digest and 20' inactivation
(previous digests pSB1A3 was discarded from #SZKD# because we think it was cut with EcoRI and SpeI)
gene
enzymes
expectation
observation
products
pSB1A3 (#TRTN#)
EcoRI + PstI
...
...
#SZKD#:1A3
Ligations and transformations
construct
part I
part II
backbone
product
host
glgAB in pSB1A3
#SZKD#:A
#SZKD#:B
#SZKD#:1A3
#99MC#:LAB
DH5α
15-06-19
Master plates of glgAB clones that were white under green light.
15-06-22
colony PCR on glgAB plate from Saturday (#A9W9# and #XE3D#)
gene / clones
tube number
expected length
machine
elongation time
result
glgAB in pSB1A2 clones 1-7
18-24
3986
3
3'59"
clones 1-6 are good
overnight cultures of good clones from the colony PCR glgAB (LB+A)
15-06-23
cryos and plasmid prep from overnight culture of glgAB
the plasmid prep went wrong and resulted in a concentration of < 10 ng/µl
gene / clone
cryo ID
plasmid ID
sequencing result
glgAB in pSB1A3 #5
#1Q88#
#CT8H#
confirmed ligation sites
glgAB in pSB1A3 #6
#YV86#
#O3TQ#
confirmed ligation sites
overnight cultures of the two clones above from the master plates
15-06-24
in the morning: freeze overnight cultures of glgB and glgAB
in the afternoon: plasmid prep of glgB and glgAB (re-use the existing IDs from the table of Monday)
pipette glgAB for sequencing #CT8H#  and #O3TQ# were emptied during sequencing!
15-06-26
Digestions
BioBrick in Backbone
gene
template ID
enzymes
product
purpose
K1585312 in pSB1A2
glgC
#F8Y8#
EcoRI, SpeI
#LVVK#:2
construction of glgCAB
K1585320 in pSB1A2
glgAB
#CT8H#
XbaI, PstI
#LVVK#:3
construction of glgCAB
overnight culture
15-06-29
Plasmid prep of glgAB in pSB1A3 because #CT8H# was empty (same ID)
Digestions
BioBrick in Backbone
gene
template ID
enzymes
product
purpose
K1585320 in pSB1A2
glgAB
#CT8H#
XbaI, PstI
#LVVK#:3
construction of glgCAB
run an agarose gel with the digestion products
gel showed that digest of glgAB and pSB1K30 has not worked!
we will use old pSB1K30 stored in #SZKD#
Ligation and Transformation
construct
gene
Part I
Part II
Backbone
products
host
purpose
K1585321 in pSB1K30
glgCAB
#LVVK#:2
#LVVK#:3
#LVVK#:4
#RRCV#:3
BL21 Gold (DE3)
for polycistronic expression
new glgAB digestion because wrong concentration was used
BioBrick in Backbone
gene
template ID
enzymes
product
purpose
K1585320 in pSB1A2
glgAB
#CT8H#
XbaI, PstI
#LVVK#:3
construction of glgCAB
repeat ligation and freeze product for transformation
Ligation and Transformation
construct
gene
Part I
Part II
Backbone
products
host
purpose
K1585321 in pSB1K30
glgCAB
#LVVK#:2
#LVVK#:3neu
#SZKD#:K30
#RRCV#:3
BL21 Gold (DE3)
for polycistronic expression
glgAB digest did not work again!
PCR amplification of glgAB
50 µl Q5 reaction with #A9W9# and #XE3D#
2'00" elongation time, 58 °C annealing
products stored in #XMVF#
15-06-30
agarose gel of glgAB PCR product shows a very strong band at the expected length
the PCR product was purified into #PZ9B#
Digestion of glgAB PCR product
BioBrick in Backbone
gene
template ID
enzymes
product
purpose
K1585320 in pSB1A2
glgAB
#PZ9B#
XbaI, PstI
#LVVK#:3
construction of glgCAB
Ligation and Transformation
construct
gene
Part I
Part II
Backbone
products
host
purpose
K1585321 in pSB1C3
glgCAB
#LVVK#:2
#LVVK#:3
#CQ1V#:V
5 µl #RRCV#:3
40 µl BL21 Gold (DE3)
for polycistronic expression
15-07-01
made a master plate of glgCAB in pSB1C3 with all available clones
15-07-02
colony PCR
4 overnight cultures per clone (to have enough for sequencing)
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
95
3'00"
denaturation
95
0'15"
annealing #A9W9# and #XE3D#
58
0'15"
elongation
72
1'20"
final elongation
72
2'00"
repeat colony PCR
tubes are numbered 1-18 for colonies
colonies picked: 1,2,5,6,7,11,12,13,16-21,23-26
19: water
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
95
10'00"
denaturation
95
0'15"
annealing #A9W9# and #XE3D#
58
0'15"
elongation
72
1'25"
final elongation
72
2'00"
15-07-03
plasmid prep of many different glgCAB clones
clone number
cryo ID
plasmid ID
digest result
1
#LMCZ#
#ZNMX#
negative
2
#K49Y#
#KLRW#
negative
5
#FYOF#
#ZKDX#
negative
6
#K8MM#
#AYWZ#
negative
7
#LBRK#
#M9BH#
negative
11
#YBH1#
#EAFT#
negative
12
#99ER#
#V6DM#
negative
13
#L39B#
#Z9BA#
negative
16
#S3BW#
#A4Y6#
negative
17
#V3SS#
#49S6#
negative
18
#WOXE#
#XO6W#
negative
20
#KEMW#
#4HTD#
negative
(test) digestion of plasmids with EcoRI+PstI (products in #PCSW#)
agarose gel
overnight cultures of good clones from test digest
15-07-06
PCRs for Gibson/CPEC assembly
component
template
forward primer
reverse primer
product length
product ID
tube #
J04450 in pSB1C3
#CD8B#
#TSTV#
#FYTO#
2070
#MRVW#:II
1+2
PfuS PCR
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
98
2'00"
denaturation
98
0'30"
annealing
64
0'30"
elongation
72
1'10"
final elongation
72
5'00"
15-07-08
PCRs for Gibson/CPEC assembly
component
template
forward primer
reverse primer
product length
product ID
glgC in pSB1A2
#F8Y8#
#O3M6#
#C3ZA#
1360
#MRVW#: IV
glgAB in pSB1A3
#CT8H#
#93FZ#
#NBZZ#
3713
#MRVW#:V
PfuS PCR
conditions for 30 cycles:
glgC
step
temperature [°C]
duration
initial denaturation
98
3'00"
denaturation
98
0'30"
annealing
59
0'30"
elongation
72
0'45"
final elongation
72
5'00"
glgAB
step
temperature [°C]
duration
initial denaturation
98
3'00"
denaturation
98
0'30"
annealing
57
0'30"
elongation
72
1'58"
final elongation
72
5'00"
The design of primer #93FZ# was flawed: it bound to the RBS.ATG section of glgB (within glgAB) with three mismatches and amplified a product of 2235 bp.
component
template
forward primer
reverse primer
product length
product ID
glgA'
#M1M6#
#93FZ#
#AVY6#
1157
#MRVW#: VI
glg'AB
#CT8H#
#EP9Z#
#NBZZ#
2907
#MRVW#:VII
PfuS PCR
conditions for 30 cycles:
step
temperature [°C]
glgA'
glg'AB
initial denaturation
98
3'00"
3'00"
denaturation
98
0'30"
0'30"
annealing
58
0'30"
0'30"
elongation
72
0'35"
1'27"
final elongation
72
5'00"
5'00"
15-07-09
90'00" DpnI digest of #MRVW#:VI and #MRVW#:VII
the gel of #MRVW#:VII did not show any band - the old VII was discarded and the PCR was repeated with longer elongation time
PfuS PCR
conditions for 30 cycles:
step
temperature [°C]
glg'AB
initial denaturation
98
3'00"
denaturation
98
0'30"
annealing
56.5
0'30"
elongation
72
1'35"
final elongation
72
5'00"
expected length: 2907 bp
PCR product purification
PCR product
purified PCR product
#MRVW#:II
#ZV3C#
#MRVW#:III
#T6X9#
#MRVW#:IV
#QSWQ#
#MRVW#:V
failed/obsolete
#MRVW#:VI
#4TVW#
#MRVW#:VII
#E9AQ#
Gibson Assembly of glgCAB in pSB1C3 (0.17 pmol total fragment amount, 30'00" incubation)
component
volume [µl]
water
0
Gibson Master Mix 2x
12.5
linear pSB1C3 #ZV3C#
1.4
glgC for glgCAB #QSWQ#
5.3
glgA' for glgCAB #4TVW#
1.6
glg'AB for glgCAB #E9AQ#
4.3
product stored in
#SWMB#:III
5 µl #SWMB#:III were heat shocked into DH5α and plated on LB+C
15-07-10
make masterplate (LB+C)
15-07-14
analyze master plate of additional glgCAB clones under green light, all clones were red!
repeat Gibson Assembly with now DpnI digested pSB1C3 backbone
Gibson Assembly of glgCAB in pSB1C3 (0.17 pmol total fragment amount, 60'00" incubation)
component
volume [µl]
water
0
Gibson Master Mix 2x
12.5
linear pSB1C3 #ZV3C#
1.4
glgC for glgCAB #QSWQ#
5.3
glgA' for glgCAB #4TVW#
1.6
glg'AB for glgCAB #E9AQ#
4.3
product stored in
#SWMB#:III
5 µl #SWMB#:III were heat shocked into DH5α and plated on LB+C
15-07-15
transformation was successful: under red-light 8 white clones were visible
overnights and masterplate of these good clones
15-07-16
cryos
plasmid prep
test digest with Xba and Pst (products in #NMTE#)
clone
cryo
plasmid ID
test digest result
glgCAB in pSB1A30 #1
#9HZM#
#QWNV#
negative
glgCAB in pSB1A30 #2
#SR8W#
#LTYR#
negative
glgCAB in pSB1A30 #3
#VPTH#
#QWBZ#
negative
glgCAB in pSB1A30 #4
#NHYB#
#6CEE#
negative
glgCAB in pSB1A30 #5
#NHBT#
#6KF4#
could be something
glgCAB in pSB1A30 #6
#H443#
#QOX1#
negative
glgCAB in pSB1A30 #7
#YF88#
#CHT1#
could be something
glgCAB in pSB1A30 #8
#W4VO#
#OQO3#
negative
new overnights of clone#5 and #7 to have enough template for sequencing
15-07-17
CPEC assembly of glgCAB in pSB1C3
component
volume [µl]
water
0
Q5 buffer
5
DMSO
0.75
dNTPs (O5)
1
Q5 polymerase
0.5
pSB1C3 fragment (#ZV3C#)
2.7
glgC for glgCAB #QSWQ#
9.8
glgA' for glgCAB #4TVW#
3.0
glg'AB for glgCAB #E9AQ#
8.0
elongation time
3'32"
product stored in
#SWMB#:IV
agarose gel with glgA', Gibson product and CPEC product
heat shock transformation of #SWMB# :III and IV into DH5α
plasmid prep of potentially good glgCAB clones #5 in #FYH1# and #7 in #4RDQ#
test digest of #FYH1# and #4RDQ# with EcoRI (products in #4SK6#:1-2)
agarose gel comparing with undigested #FYH1# -> #FYH1# looks promising, but #4RDQ# is too short
Test digest of glgCAB in pSB1C3
#FYH1# and #4RDQ# were test digested with EcoRI. #FYH1# looks very promising.
15-07-18
master plate of many white glgCAB clones on LB+C
15-07-19
3x 5 ml LB+C overnight cultures of glgCAB clones #1 (CPEC) and #23 (Gibson)
15-07-20
Cryos and freezings of glgCAB clones #1 and #23
plasmid prep of glgCAB clones #1 and #23
strain
cryo
plasmid
glgCAB in pSB1C3 by CPEC in DH5α #1
#VAPH#
#EY9B#
glgCAB in pSB1C3 by Gibson in DH5α clone #23
#PAZS#
#6BZC#
pipette glgCAB in pSB1C3 #FYH1# for sequencing with all primers
test digest of glgCAB #1 and #23 with EcoRI
tube 1: #6BZC# (#23)
tube 2: #EY9B# (#1)
products are stored in #1W9S#
Test digest of glgCAB in pSB1C3
#6BZC# and #EY9B# were test digested with EcoRI. #EY9B# looks very promising.
clone 1 looks good, but clone 23 does not!
prepare moving glgCAB over into a vector with promoter
15-07-21
digest of glgCAB clones, J23119 and K1585119 to combine CAB with the Anderson Promoter and pSB1A30
template
ID
enzymes
tube number
purpose
Anderson Promoter 19
#AHZ6#
SpeI, PstI
1
to combine CAB and constitutive AP 19
K1585119
#OEVV#
SpeI, PstI
2
to combine CAB and inducible AP 19
glgCAB #1
#EY9B#
XbaI, PstI
3
to combine with both AP 19 versions
glgCAB #5
#FYH1#
XbaI, PstI
4
to combine with both AP 19 versions
glgCAB #1
#EY9B#
EcoRI, PstI
5
to combine with pSB1A30
glgCAB #5
#FYH1#
EcoRI, PstI
6
to combine with pSB1A30
products stored in #TF4M#
ligation
name
template 1
template 2
purpose
L1
tube 1 (const. AP 19)
tube 3 (glgCAB #1 with X and P)
glgCAB with constitutive AP 19
L2
tube 2 (induc. AP 19)
tube 3 (glgCAB #1 with X and P)
glgCAB with inducible AP 19
L3
tube 1 (const. AP 19)
tube 4 (glgCAB #5 with X and P)
glgCAB with constitutive AP 19
L4
tube 2 (induc. AP 19)
tube 4 (glgCAB #5 with X and P)
glgCAB with inducible AP 19
products are also stored in #TF4M#
all ligation products were transformed in NEB 10β
15-07-22
make a gel with all digests
Digest and ligation of glgCAB and AP 19
For #AHZ6# there are two bands of the right length, one uncut and one cut. For #OEVV# there is only one weak band but it has the correct length. Both CAB digest products of #EY9B# have the correct length. There are no bands for the ligations.
strain
cryo
plasmid
sequencing result
glgCAB in pSB1C3 by CPEC in DH5α #1
#VAPH#
#EY9B#
A>V mutation in glgB, glgC and A look perfect
glgCAB in pSB1C3 by Gibson in DH5α clone #23
#PAZS#
#6BZC#
sequencing did not work
glgCAB in pSB1C3 by Gibson in DH5α clone #5
#NHBT#
#FYH1#
glgA and B look perfect but glgC is missing
overnight cultures and master plates of transformations of ligations 1+2 (LB+A)
new overnights of clones #3-5 glgCAB in pSB1C3 by CPEC from masterplate
15-07-23
cryos and plasmid prep of glgCAB, AP19.glgCAB and AP19.lacIsite.glgCAB candidates
test digest with EcoRI (stored in #AMX6#)
1: #4HB9#
2: #6THO#
3: #BB9Q#
C2: #HOEL#
construct
cryo
plasmid
digest result
sequencing result
K1585321 in pSB1C3 #2
#BA36#
#NXOK#
...
no chromatogram
K1585321 in pSB1C3 #3
#KRWH#
#BB9Q#
...
no chromatogram
K1585321 in pSB1C3 #4
#RHMM#
#6THO#
...
no chromatogram
J23119.K1585321 in J61002 #1
#K3K4#
#4HB9#
cloning did not work
J23119.K1585321 in J61002 #2
#W3QO#
not prepped
cloning did not work
K1585119.K1585321 in J61002 #1
#Y4N4#
was red
K1585119.K1585321 in J61002 #2
#XFF9#
was red
K1585119.K1585321 in J61002 #3
#KATB#
was red
K1585119.K1585321 in J61002 #4
#MWVT#
was red
15-07-24
sequencing results were bad, so we use parts from #FYH1# (just glgC missing) and #EY9B# (mutation in glgB) to construct the full plasmid again
these parts will be assembled in a Gibson Assembly
Q5 PCR with 27 cycles
PCR
template
primer 1
primer 2
annealing temperature
elongation time
product length
1
#FYH1#
#EP9Z#
#6ZOP#
71 °C
1'23"
2640 bp (parts of glgA and B)
2
#EY9B#
#89VV#
#AVY6#
71 °C
2'40"
5178 bp (all the rest)
PCR programm
step
temperature
time
initial denaturation
98 °C
0'30"
denaturation (cycle)
98 °C
0'10"
annealing (cycle)
71 °C
0'25"
elongation (cycle)
72 °C
see above
final elongation
72 °C
2'00"
The products are temporarily stored in #94DS#.
Both products are digested with DpnI overnight at 37 °C and put back into #94DS#.
15-04-25
put DpnI digests back into #94DS# and freeze it
15-07-27
agarose gel of DpnI digests
PCR product purification
products stored in #MCWO# and #963L#
Gibson Assembly of #MCWO# and #963L# at 50 °C for 50 minutes
heat shock into DH5α
15-07-28
transformation was successful
masterplate of 30 clones from transformation plate of glgCAB in pSB1C3
overnight of 10 clones for test digest
15-07-29
plasmid prep and cryos of all overnights
construct
cryo
plasmid
tube # in digest
digest result
sequencing result
K1585321 in pSB1C3 #1
#L8CN#
#96EB#, #4RCD#
1
good
point mutation
K1585321 in pSB1C3 #2
#NMTT#
#MFQS#, #L6OH#
2
good
point mutation
K1585321 in pSB1C3 #3
#MXKQ#
#QAB9#, #8WE4#
3
good
point mutation
K1585321 in pSB1C3 #4
#CKSK#
#B316#
4
good
-
K1585321 in pSB1C3 #5
#1TN1#
#HOX9#
5
two bands instead of one
-
K1585321 in pSB1C3 #6
#MKC3#
#FNTZ#
6
good
-
K1585321 in pSB1C3 #7
#99NB#
#6DMQ#
7
too long (?)
-
K1585321 in pSB1C3 #8
#P664#
#6OT6#
8
good
-
K1585321 in pSB1C3 #9
#P8NZ#
#HF8O#
9
good
-
K1585321 in pSB1C3 #10
#TO3R#
#A1FM#
10
good
-
test digest with EcoRI + agarose gel
new overnights of good glgCAB in C3 clones no.1-3 (3 per clone to have enough for seqencing)
15-07-30
plasmid prep (see table above)
pipette for sequencing
15-07-31
Sequencing showed that there still is a point mutation in glgB
decided to do a QuickChange with #EY9B#
QuickChange  primers were designed and ordered
15-08-03
QuickChange with PfuS (E2)
tube 1:
component
amount
plasmid template (1:100 dilution)
3.3 µl
deionized H2O
38.2 µl
10x PfuS buffer (B2)
5 µl
#BABZ# fwd_primer (10 µM)
2 µl
dNTP mix (10 mM)
1 µl
PfuS
0.5 µl
tube 2:
component
amount
plasmid template (1:100 dilution)
3.3 µl
deionized H2O
38.2 µl
10x PfuS buffer (B2)
5 µl
#BOXR# rev_primer (10 µM)
2 µl
dNTP mix (10 mM)
1 µl
PfuS
0.5 µl
PCR program 1 (105 °C heated lid)
3 cyles
step
temperature
time
initial denaturation
98°C
0'30"
denaturation (cycle)
98°C
0'10"
annealing (cycle)
59°C
0'30"
extension (cycle)
72°C
4'15"
control
component
amount
plasmid template (1:100 dilution)
3.3 µl
deionized H2O
40.2 µl
10x PfuS buffer (B2)
5 µl
dNTP mix (10 mM)
1µl
PfuS
0.5 µl
mix 25 µl of tube 1 and 25 µl of tube 2 and add 0.5 µl PfuS
PCR program 2 (105°C heated lid)
15 cycles
step
temperature
time
initial denaturation
98 °C
0'30"
denaturation (cycle)
98 °C
0'10"
annealing (cycle)
59 °C
0'30"
extension (cycle)
72 °C
4'15"
final elongation
72 °C
10'00"
products are in #189R#
make an agarose gel
PCR clean-up
DpnI digest over night
15-08-04
20 minutes inactivation
PCR clean-up
transformation of PCR product
repeat the QuikChange with higher template and enzyme concentration because no bands were visible on the gel
QuickChange with PfuS (E2)
tube 1:
component
amount
plasmid template #EY9B# (1:100 dilution)
6.6 µl
deionized H2O
34.9 µl
10x PfuS buffer (B2)
5 µl
#BABZ# fwd_primer (10 µM)
2 µl
dNTP mix (10 mM)
1 µl
PfuS
0.5 µl
tube 2:
component
amount
plasmid template #EY9B# (1:100 dilution)
6.6 µl
deionized H2O
34.9 µl
10x PfuS buffer (B2)
5 µl
#BOXR# rev_primer (10 µM)
2 µl
dNTP mix (10 mM)
1 µl
PfuS
0.5 µl
PCR program 1 (105 °C heated lid)
3 cyles
step
temperature
time
initial denaturation
98 °C
0'30"
denaturation (cycle)
98 °C
0'15"
annealing (cycle)
59 °C
0'30"
extension (cycle)
72 °C
5'00"
control
component
amount
plasmid template (1:100 dilution)
6.6 µl
deionized H2O
36.9 µl
10x PfuS buffer (B2)
5 µl
dNTP mix (10 mM)
1 µl
PfuS
0.5 µl
mix 25 µl of tube 1 and 25 µl of tube 2 and add 1 µl PfuS
PCR program 2 (105 °C heated lid)
15 cycles
step
temperature
time
initial denaturation
98 °C
0'30"
denaturation (cycle)
98 °C
0'15"
annealing (cycle)
59 °C
0'30"
extension (cycle)
72 °C
5'00"
final elongation
72 °C
10'00"
products are in #189R#
make an agarose gel
SDM of glgCAB
For tubes 1 and 2 there are faint bands at the correct length, but there are also strong undefined bands on top. For the control (#3) the 7066 bp band is faint as expected
correct bands were visible
DpnI digest over night
15-08-05
20 minutes inactivation
PCR clean-up
agarose gel of purified PCR products
sample
purified digest product ID
gel result
transformation result
number 1
#AZYS#
correct bands visible
...
number 2
#ATM8#
correct bands visible
...
control
#SWEV#
no 7000 bp bands
...
15-08-06
repeat DpnI digest of Gibson and SDM products
transformation into NEB10β
15-08-07
masterplate of clones from trafo plate
15-08-08
take out masterplates from 37 °C and put them in the fridge
15-08-09
overnights of 10 clones for test digest on monday
15-08-10
plasmid prep and cryos of clones
Clone
cryo ID
plasmid ID
test digest result
sequencing result
glgCAB in pSB1C3 #1
#XWFL#
#A919#
good
bad
glgCAB in pSB1C3 #2
#ZNNL#
#QOZV#
good
good
glgCAB in pSB1C3 #3
#41ZT#
#RW33#
good
good
glgCAB in pSB1C3 #4
#6THN#
#DTBM#
good
good
glgCAB in pSB1C3 #5
#VMV3#
#COOX#
2 bands
glgCAB in pSB1C3 #6
#LRAT#
#AO86#
length not clear
glgCAB in pSB1C3 #7
#3W3Y#
#TMXD#
length not clear
glgCAB in pSB1C3 #8
#SHHS#
#XB19#
good
point mutation
glgCAB in pSB1C3 #9
#VOBO#
#ATH8#
good
point mutation
glgCAB in pSB1C3 #10
#33C1#
#LY4Z#
good
point mutation
test digest
15-08-11
pipette for sequencing with one primer that covers point mutation (#RYZX#)
bring the samples to the Fraunhofer institute
15-08-14
overnights of #6THN#, #ZNNL# and #41ZT#for further sequencing of glgCAB clones
15-08-15
plasmid prep of CAB clones
products #QOZV# and #DTBM#
15-08-17
digest #EY9B# (tube1), #QOZV# (tube 2) and #DTBM# (tube 3) with EcoRI and PstI to transfer into pSB1A30
digest of #EY9B# did not work, so we use an digestion product from #TF4M# (tube 5)
ligation
tube no
insert
backbone
purpose
2
#QOZV#
#OHOZ# tube # 1
new glgCAB #2 in pSB1A30
3
#DTBM#
#OHOZ# tube # 1
new glgCAB #4 in pSB1A30
products are stored in #3A9X#
transform in DH5α
15-08-18
sequencing results show that #QOZV# and #DTBM# are both missing glgC
all transformations were thrown away
we will test glgCAB with one point mutation
15-08-19
digest #8WE4# and #EY9B# with EcoRI and PstI to ligate it in pSB1A30 for expression
both look good on gel
digest of glgCAB in pSB1C3
For tubes 1 and 2 there are strong bands at the correct length (2000 and 5000bp)
products stored in #WV93#
15-08-20
ligation of cut #8WE4# and #EY9B# in pSB1A30
heat shock in DH5α
tube no
insert
backbone
purpose
2
#8WE4#
#OHOZ# tube # 1
glgCAB with point mutation in pSB1A30
3
#EY9B#
#OHOZ# tube # 1
glgCAB with point mutation in pSB1A30
15-08-21
overnight cultures and master plate of DH5α glgCAB clones
15-08-22
make cryo cultures of glgCAB in pSB1A30 DH5α overnight cultures
plasmid prep afterwards
no
What?
strain
Cryo
plasmid
1
glgCAB in A30 (cut from #EY9B#)
DH5α
#ADWX#
#XTEN#
2
glgCAB in A30 (cut from #EY9B#)
DH5α
#ALXK#
#CTMV#
3
glgCAB in A30 (cut from #EY9B#)
DH5α
#AR46#
#Q13F#
4
glgCAB in A30 (cut from #EY9B#)
DH5α
#3MKZ#
#YFXC#
2
glgCAB in A30 (cut from #8WE4#)
DH5α
#TNR8#
#KSZQ#
3
glgCAB in A30 (cut from #8WE4#)
DH5α
#R4CZ#
#KRYH#
4
glgCAB in A30 (cut from #8WE4#)
DH5α
#HMSM#
#NPHP#
3A assembly of glgAB (#CT8H#) and glgC (#4D6H#) in pSB1C3 (already cut: #CQ1V# tube V)
digest #4D6H# with EcoRI and SpeI, product in #KFA1#
digest #CT8H# with XbaI and PstI, looked bad ont the gel: concentration was too low
make an overnight culture of #1Q88# to produce new #CT8H#
15-08-23
plasmid prep of new #CT8H# , concentration measurement will be done tomorrow
15-08-24
test digest of glgCAB constructs with EcoRI
tube no
plasmid
result
1
#Q13F#
bad
2
#YFXC#
bad
3
#NPHP#
bad
4
#KRYH#
bad
5
#KSZQ#
good
6
#XTEN#
bad
7
#CTMV#
good
transformation of #KSZQ# in BL21 Gold
digest of #CT8H# with XbaI and PstI (stored in #3649#)
gel looks good
ligation of glgAB (#3649#), glgC (#KFA1#) in pSB1C3 (#CQ1V# tube5)
tube L1 (also stored in #3649#)
transformation of new glgCAB into DH5α
electroporation of #KSZQ# in BL21 ΔX and BL21 ΔP electrocompetent
15-08-25
masterplate and overnight cultures of #KSZQ# in BL21 Gold, BL21 ΔX and BL21 ΔP in LB+A
masterplate and overnight of new glgCAB in DH5α in LB+C
new transformation of #KSZQ# in BL21 Gold
15-08-26
colony pcr of mutated GlgCAB (CAB')(in pSB1A30)
clone
tube #
CAB' in pSB1A30 in BL21 ΔX clone #1-12
1-12
CAB' in pSB1A30 in BL21 ΔP clone #1-4
13-16
CAB' in pSB1A30 in BL21 ΔP clone #7-12
17-22
CAB' in pSB1A30 in BL21 ΔP clone #5+6
22-24
none of the clones had the expected length of 5362 bp on the gel
probably the fragment was too long
do 10 overnights of glgCAB in delta P each for test digest tomorrow, all clones in delta X were red
then new overnight of good one, main culture on friday, SDS on friday
new trafo of CAB' (mutated) in BL21 Gold wild type
plasmid prep and cryos of NEW GlgCAB in psB1C3
clone no
cryo
plasmid
digestion tube
result of digestion
1
#OR9C#
#VCWE#
1
bad
3
#DYWY#
#S9H8#
2
bad
5
#FQ69#
#9CDT#
3
bad
7
#R4MF#
#MDSB#
4
many bands, but also the correct one
8
#8MKC#
#KZRZ#
5
many bands, but also the correct one
9
#NRSN#
#PTRE#
6 (PTRE)
bad
10
#3EHE#
accidently discarded
bad
A
#KLH8#
#RRPP#
7
bad
B
#OC1P#
accidently also #PTRE#
8 (PTRE*)
bad
C
#4EHF#
#94RY#
9
bad
tubes 4+5 look good, products stored in#ZY4A#
digest of NEW GlgCAB
gel and inactivation
then ligate tube 4+5 (#MDSB# and #KZRZ#) into pSB1A30 (#OHOZ# tube 1), ligations also in #ZY4A#
transformation in DH5α
for sequencing: New GlgCAB in pSB1C3, clones that looked good in test digest
all clones from new CAB looked bad in test digest with EcoRI. We will not continue working with these ones..
15-08-27
plasmid prep of glgCAB' in delta P (clones # 1-10)
test digest with eco
all clones had the wrong length (ca. 3200 bp)
trafo of glgCAB in pSB1A30 in BL21 WT did not work again
We decided to sequence #KSZQ#, #MDSB# and #KZRZ# and and to digest pSB1A30 again (tube #T6X9#) and ligate CAB' into new pSB1A30
made a gel to test several constructs
Cloning glgCAB'(mutated) in pSB1A30
All constructs that were used for glgCAB' assembly in A30 had the correct length. Also a fragment of the correct length (complete CAB in A30) can be seen in the coloumn for the ligation tube (7202bp).
CAB' (#8WE4#) was ligated with new digested pSB1A30 (#T6X9#) and cloned into Bl21 Gold DE3
15-08-28
a few clones were visible on the transformation plate of CAB' in pSB1A30
make overnight cultures and a master plate
15-08-29
plasmid prep and test digest of CAB' in pSB1A30 clone #2 (plasmid: #4XWH# cryo: #3OVK#)
gel looks bad, no correct bands visible for double digest
new overnight of CAB' in pSB1A30 clone #1 in order to repeat the above (it did not grow)
15-08-30
try to assemble glgCAB' in pSB1A30 via CPEC
use pSB1A30 from #T6X9#
PCR to amplify CAB'
component
amount
ddH2O
23.6 µl
#O3M6#
0.6 µl
#NBZZ#
0.6 µl
B2 Buffer
3 µl
dNTPs
0.6 µl
PfuS (E2)
0.3 µl
PCR program
step
temperature [°C]
time
initial denaturation
98
3'00"
denaturation
98
0'30"
annealing
63
0'30"
elongation
72
2'37" (5039 bp)
final elongation
72
5'00"
heat shock of both BBa_C0012 from kit plate and ligation product "Lig iJR" (#WV93#) into BL21 Gold DE3
make three overnight cultures of #TNR8# to have enough plasmid for sequencing
do DpnI digest overnight (of PCR product for CPEC)
15-08-31
heat inactivate DpnI digest, purify and measure concentration (product in #N13D#), do CPEC with #T6X9#, transformation into BL21 Gold
CPEC program
step
temperature
time
initial denaturation
98
0'30"
denaturation (cycle)
98
0'10"
annealing (cycle)
55
0'30"
elongation (cycle)
72
1'52"
final elongation
72
10'00"
prep overnight cultures and prepare sequencing
15-09-01
do masterplates and overnights from Trafo plate (CAB' in A30 via CPEC)
digest glgAB (#CT8H#) and pSB1C30 (#MFNP#) with EcoRI and PstI
ligate glg AB into pSB1C3 (#OLAE# tube A) and pSB1C30 (#OQ81#)
ligation products stored in #OQ81#
heat shock into DH5α (pSB1C3) and BL21 (pSB1C30)
15-09-02
plasmid prep of overnights
test digest
gel showed that RFP is in all constructs of CAB
the backbone might be not digested correctly with DpnI after PCR
DpnI digest of #T6X9#
overnights and masterplates of glgAB in pSB1C3 and pSB1C30
15-09-03
sequencing shows that just glgC was in glgCAB ligation attempts via Biobrick assembly
sequencing showed that #KSZQ# (glgCAB' in pSB1A30) is PERFECT
cryo and plasmid prep of overnights of glgAB in C3 and C30
only clones 1,2,3,6,7 and glgAB in C30 were prepped becuase the other ones were red
test digest with EcoRI and PstI
only clones #3+6 of glgAB in pSB1C3 look good
subcloning into pSB1C30 did not work
clone
cryo ID
test digest result
plasmid ID
glgAB in C3 clone #1
#XM94#
bad
-
glgAB in C3 clone #2
#SZW1#
bad
-
glgAB in C3 clone #3
#MQQ4#
perfect
#HAMP#
glgAB in C3 clone #6
#1KA4#
perfect
#YC8M#
glgAB in C3 clone #7
#B9ZD#
bad
-
glgAB in C30 clone #1
#AWWP#
bad
-
overnights of good glgAB clones #3+6 from masterplate
PCR purification of DpnI digested #T6X9#
transformation of 5 µl #KSZQ# into BL21 Gold DE3
M9 overnights of BL21 WT, BL21 glgA, BL21 glgB and BL21 ΔP for iodine staining
LB overnights cultures of good glgAB clones for growth experiments and SDS gels
15-09-04
master plates and overnights of #KSZQ# transformation clones
pipette sequencing of glgAB clones
for AB in C30: heat shock ligation product (in #OQ81#) into DH5α
for new C30 backbone: amplify #MFNP# with #TSTV# and #FYTO#,digest with 2µL DpnI (for 4 hours), heatinactivate, digest with E+P, ligate, transform into DH5α
15-09-05
cryos and plasmid prep of BL21 CAB' in A30, then test digest with EcoRI (Sciebo: 15-09-05 Test digest of CAB in A30)
Clone
cryo ID
plasmid ID
test digest result
sequencing result
glgCAB' in pSB1A30 #1
#WCQY#
#YC3V#
good
glgCAB' in pSB1A30 #2
#V4YZ#
#3KZZ#
bad
glgCAB' in pSB1A30 #3
#6AME#
#XQ4R#
bad
glgCAB' in pSB1A30 #4
#8HCP#
#MKHV#
bad
glgCAB' in pSB1A30 #5
#LFRD#
#S3RW#
good
glgCAB' in pSB1A30 #6
#B11R#
#BBBF#
bad
glgCAB' in pSB1A30 #7
#FNVQ#
#QM4Y#
bad
do new overnights of positive clones (#1+5(for SDS))
screen Trafo plates of AB in C30 in DH5α
all clones were white
do 5 overnights of each new and old ligation
parallel masterplates
15-09-06
do plasmid prep (see Manuals/Plasmid Prep!!!) and test digest with EcoRI of DH5α glgAB in pSB1C30
SDS-PAGE (Ladder-GlgCAB clone #1, GlgCAB #5,GlgA,GlgB,Wildtype for control)
glgC did not grow
HPLC samples
new 10 ml overnights (LB+20mM Glucose) of WT,A,B,deltaX,deltaP
overnights of GlgAB in pSB1C30 for CRYOS AND PREP
overnights of BL21 GlgCAB in pSB1A30 #1, #5 (LB+A+IPTG+20mM Glucose) and BL21 WT
15-09-07
cryos and prep DH5α glgAB in pSB1C30
construct
Cryo
purified plasmid
Old glgAB #2
#8ZZ4#
#ZBVK#
Old glgAB #3
#8MCD#
#14XB#
New glgAB #1
#N96D#
#C9ZY#
New glgAB #2
#NDEF#
#K8XD#
pipette DH5α glgAB in pSB1C30 and BL21 glgCAB' in pSB1A30 for sequencing, prepare sequencing sheet
transform four glgAB in pSB1C30 into BL21 Gold (see table above)
overnights of GlgCAB in pSB1A30 #1 and #5, GlgC in A30 (#OHES#) and WT Bl21 Gold for repeated iodine staining (incubate since 9 p.m.!)
15-09-08
Prep glgCAB in pSB1A30 #1 for remaining sequencing tubes
screen trafo of GlgAB in pSB1C30 BL21 Gold
do masterplates
15-09-09
sequencing did not work
make two overnights of both CAB clones #1 and #5 for new sequencing
15-09-10
pipette #S3RW# and #YC3V# again for sequencing
15-09-11
both #S3RW# and #YC3V# look perfect in sequencing!
confirmed glgCAB' in pSB1A30
Results
BioBrick
gene
backbone
confirmed cryo
confirmed plasmid
BBa_K1585320
glgAB
pSB1A3
#1Q88#
#CT8H# (ligation sites confirmed)
BBa_K1585321
glgCAB
pSB1C3
#MXKQ#
#8WE4#
BBa_K1585321
glgCAB'
pSB1A30
#WCQY#
#KSZQ#
▲
Contact
CC-BY-SA iGEM Aachen 2015
