Team:Aachen/Lab/Glycogen/Synthesis
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Combined Expression
Single Expression
BioBricks
How Glycogen Synthesis Works
The glycogen formation in E. coli is performed by three enzymes:
glycogen synthase (encoded by glgA)
branching eyzme (encoded by glgB)
ADP-glucose pyrophophorylase (encoded by glgC)
GlgC uses glucose-1-phosphate to build ADP-glucose that is needed by the GlgA to elongate glycogen chains.  Through autophosphorylation, the glycogen synthase GlgA also acts as the starting particle, just like glycogenin in eukaryotes.  The branching enzyme GlgB forms alpha-1,6-branches using existing chains.
In order to increase the glycogen formation, we built expression plasmids of the genes as well as a polycistronic version . Our constructs were later characterized using a variety of methods.
BioBricks
Summary
In order to share our constructs for glycogen synthesis with the iGEM community, all three synthesis genes were cloned into pSB1C3, sequenced and submitted as new BioBricks.
BBa_K1585310 (B0034.glgA) was synthesized with prefix, suffix and overhangs to pSB1C3. After a PCR amplification, both the gBlock and the backbone were cut, ligated and transformed into E.coli DH5α. The purified plasmid was then sequenced and submitted to the registry.
BBa_K1585311 (B0034.glgB) was synthesized with a similar design. Due to DNA synthesis limitation, the gene was divided into two gBlocks with overhangs to each other. The pSB1C3 backbone was amplified with a forward primer that binds on the suffix and a reverse primer that binds the RFC10 prefix. Therefore, all parts were cloned with overhangs to the next part and were assembled by CPEC assembly.
The third BioBrick BBa_K1585312 (B0034.glgC) is based on a construct designed by team Edinburgh in 2008 (BBa_K118016). The glgC has a point mutation which increases activity in the absence of fructose 1,6-bisphosphate and lowers the affinity to the inhibitor AMP. [1] We wanted to combine glgC with B0034 because previously it was only available with a strong but uncharacterized RBS (BBa_J15001). This task should be realized by B0034 Insertion Mutagenesis , a method to combine B0034 with every RFC10 BioBrick. This approach, however, failed and we finally moved glgC into the suffix of BBa_B0034 (B0034 in pSB1A2) using restriction enzymes. Afterwards, we transferred BBa_K1585310 (B0034.glgC) from pSB1A2 to the shipping plasmid pSB1C3 using CPEC.
Results
All glycogen synthesis genes were successfully cloned into BioBricks.
BioBrick
gene
backbone
confirmed cryo
sequence-confirmed plasmid
submitted to registry
BBa_K1585310
B0034.glgA
pSB1C3
#KMST#
#M1M6#
✓
BBa_K1585311
B0034.glgB
pSB1C3
#F4T6#
#YHT3#
✓
BBa_K1585312
B0034.glgC
pSB1C3
#DFKW#
#OP1N#
✓
a
Laboratory Notebook
Single Expression
Summary
After all three genes, glgA, glgB and glgC, were cloned into the RFC10 BioBrick standard, the next step was to test the expression of each glycogen synthesis enzyme. For all expression plasmids, we used the vector library “pSB1X30” from team Heidelberg [2] . These backbones have a lacI repressible T7 promoter inserted directly upstream to the BioBrick prefix. Therefore we used E. coli BL21 Gold (DE3) for our expression test, which contains the T7 RNA polymerase.
To express GlgA in pSB1K30 we subcloned its coding sequence from pSB1C3 and transformed into Bl21 Gold (DE3). The expression plasmid for GlgB, the branching enzyme, was also built via 3A assembly, following the same procedure as for GlgA. The coding sequence for the ADP-glucose pyrophosphorylase, GlgC, was subcloned into pSB1A30. This was done via CPEC assembly. Here, pSB1A30 was used because after expression and analysis via SDS-PAGE for GlgA and GlgB, no difference between induced and not induced was seen.
The three expression plasmids were transformed into E. coli BL21 Gold (DE3). All proteins were detected on SDS gels when compared to a RFP control, thus, the expression of all glycogen synthesis genes was confirmed.
a
Results
BioBrick
gene
backbone
confirmed cryo
confirmed plasmid
BBa_K1585310
B0034.glgA
pSB1K30
#W8VB#
#K8DM#
BBa_K1585311
B0034.glgB
pSB1K30
#Q6Z1#
#KK8K# (confirmed ligation sites)
BBa_K1585312
B0034.glgC
pSB1A30
#OHES#
#VMN3#
SDS-PAGE of GlgA expression
GlgA was expressed in pSB1K30 and compared to mRFP expression. The small arrows point to the GlgA bands of 52.4 kDa. No difference between induced and uninduced was observed. After overnight expression the strong bands were still present.
SDS-PAGE of GlgB expression
GlgB was expressed in pSB1K30 and compared to mRFP expression. The small arrows point to the glgB bands of 84 kDa. No difference between induced and uninduced was observed.
SDS-PAGE of GlgC expression
GlgC was expressed in pSB1A30 and compared to mRFP expression. The small arrows point to the glgC bands of 48 kDa. No difference between induced and uninduced was observed. After overnight expression the strong bands were no longer present.
a
Laboratory Notebook
Combined Expression
Summary
Following the construction of the BioBricks K15853110 , K1585311 , K1585312 and their confirmed single expression, we aimed to build an operon that combines all three genes. We opted for an inducible expression to avoid that the constitutive expression of three relatively big proteins hinders cell growth. Therefore, we used plasmids containing the IPTG-inducible T7 promoter (Heidelberg pSBXXX collection) in E. coli BL21 Gold (DE3).
First, glgA and glgB were subcloned into pSB1A3 by BioBrick Assembly forming a 3986 bp long translational unit. This construct was later subcloned into two more plasmid backbones: pSB1C3 to provide it to the iGEM registry and pSB1C30 to characterize the complementary function of these enzymes without the overexpression of GlgC and the associated surplus of ADP-glucose.
In the next step, glgC and glgAB were fused in pSB1C3 via Gibson Assembly and CPEC. We decided to put glgC at the first position within the reading frame. The amount of ADP-glucose created by GlgC determines the activity of the following two enzymes and can therefore be considered the bottleneck of glycogen synthesis. We obtained glgCAB with a point mutation in the sequence of glgB, changing one alanine to a similiarly aliphatic aminoacid, valine. We tried to correct the substition via QuikChange and new Gibson Assembly/CPEC. Despite multiple attempts, we always received the same point mutation again. When we analysed the tertiary structure of GlgB, we assessed that the mutation was not positioned in the active site and should therefore not impede its function a lot. Due to time reasons we decided to proceed with the mutated construct which was subsequently cloned into pSB1A30 in order to confirm gene expression by SDS-PAGE .
All enzymes were detected on SDS gels with E. coli Bl21 Gold (DE3) wild type as negative control. Both polycistronic constructs were submitted in RFC10 standard to registry.
Results
BioBrick
gene
backbone
confirmed cryo
confirmed plasmid
BBa_K1585320
glgAB
pSB1A3
#1Q88#
#CT8H# (ligation sites confirmed)
BBa_K1585321
glgCAB
pSB1C3
#MXKQ#
#8WE4#
BBa_K1585321
glgCAB'
pSB1A30
#WCQY#
#KSZQ#
BBa_K1585320
glgAB
pSB1C3
#MQQ4#
#HAMP# (ligation sites confirmed)
BBa_K1585320
glgAB
pSB1C30
#8ZZ4#
#ZBVK# (ligation sites confirmed)
SDS-PAGE of GlgCAB and GlgAB expression
GlgCAB was expressed in pSB1A30 and GlgAB in pSB1C30. The gel shows gene expression after IPTG induction compared to GlgA in pSB1K30 and BL21 Gold (DE3).
SDS-PAGE of GlgCAB expression
GlgCAB was expressed in pSB1A30 after IPTG induction and compared to the BL21 Gold (DE3). The small arrows indicate the bands of expected protein sizes.
a
Laboratory Notebook
References
↑ Meyer CR, Ghosh P, Nadler S, Preiss J. Cloning, expression, and sequence of an allosteric mutant ADP-glucose pyrophosphorylase from Escherichia coli B. ArchBiochem Biophys. 1993 Apr;302(1):64-71. PubMed PMID: 8385906.
↑ https://2014.igem.org/Team:Heidelberg/Parts#Backbones
green
▲
Contact
CC-BY-SA iGEM Aachen 2015
