Team:Aachen/Lab/Methanol/Biobricks
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Summary
Before assembling our methanol condensation genes in one genetic construct, we wanted to obtain the required genes separately in a pSB1C3 standard BioBrick vector.
BioBrick
enzyme
gene
sequence length [bp]
source organism
BBa_K1585210
methanoldehydrogenase 2
mdh
1219
Bacillus methanolicus MGA3
BBa_K1585211
3-hexulose-6-phosphate
hps
697
Bacillus methanolicus MGA3
BBa_K1585212
6-phospho-3-hexuloisomerase
phi
616
Bacillus methanolicus MGA3
BBa_K1585213
phosphoketolase
xpk
2539
Bifidobacterium adolescentis
As the coding sequences that are required for methanol condensation come from different organisms, they first had to be codon optimized. As the next step, extensions at both ends were designed to make the genes consistent with our cloning strategy. After amplification, the BioBrick vectors were assembled via restriction and ligation as well as CPEC , respectively.
in silico work
Bacillus methanolicus is an organism that can naturally metabolize methanol and carries the coding sequences of mdh, hps and phi in its genome [1] . We decided to take the DNA sequences of these genes from the B. methanolicus MGA3 genome, which we found in the NCBI database. According to the publication by Bogorad et al. [2] the xpk coding sequence for the MCC pathway is derived from the Bifidobacterium adolescentis genome.
To avoid illegal restriction sites and to adapt the coding sequences of mdh, hps, phi and xpk to the codon usage of E. coli, the coding sequences were optimized with the "Optimize Codons" tool of the software "Geneious".
Subsequently, we designed and ordered linear gBlocks with additional extensions up- and downstream the coding sequence. Each of the 58 bp upstream extensions contain an overlap to pSB1C3 , the BioBrick prefix and the BBa_B0034 ribosome binding site plus the BioBrick scar TAGTAC. Downstream the CDS, we added an 41 bp extension with the suffix and another overlapping region to pSB1C3 .
Since the CDS of the xpk is too long for gBlock synthesis, we devided its sequence into two half and designed extensions for both fragments to assemble them together with a linearized pSB1C3 backbone via CPEC .
Results
mdh, hps and phi were successfully cloned into the pSB1C3 backbone via restriction and ligation. The two xpk fragments and a linear pSB1C3 fragment were assembled in a CPEC reaction. All four resulting BioBricks were sequenced and submitted to the BioBrick registry.
BioBrick
construct
sequencing confirmed plasmid
confirmed  cryo
BBa_K1585210
RBS.mdh in pSB1C3
#9PBD#
#WNFY#
BBa_K1585211
RBS.hps in pSB1C3
#H4M9#
#863A#
BBa_K1585212
RBS.phi in pSB1C3
#ENKK#
#AE8V#
BBa_K1585213
RBS.xpk in pSB1C3
#KFEY#
#8WY9#
Laboratory Notebook
References
↑ Witthoff S, Schmitz K, Niedenführ S, Nöh K, Noack S, Bott M, Marienhagen J. Metabolic engineering of Corynebacterium glutamicum for methanol metabolism. Appl Environ Microbiol. 2015 Mar;81(6):2215-25. doi: 10.1128/AEM.03110-14. Epub 2015 Jan 16. PubMed PMID: 25595770; PubMed Central PMCID: PMC4345391.
↑ Igor W. Bogorada, Chang-Ting Chena, Matthew K. Theisena, Tung-Yun Wua, Alicia R. Schlenza, Albert T. Lama and James C. Liaoa: Building carbon–carbon bonds using a biocatalytic methanol condensation cycle
▲
Contact
CC-BY-SA iGEM Aachen 2015
