Team:Aachen/Notebook/Protocols
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
The most important recipe for effective research and reproducible results are reliable protocols.
On this page we are providing several protocols that we have used over the course of our project.
Culturing
Lysogeny-Broth (LB Medium)[+]
spacer
weight components
10 g/L NaCl
10 g/L Trypton/Pepton
5 g/L Yeast extract
(15 g/L Agar for plates)
fill to 1 L with VE/deionized water
mix well by shaking
for plates, let it cool to 60 °C
add antibiotics and mix thoroughly
pour plates and pile
M9 Medium[+]
spacerM9 is a minimal Medium with a low autofluorescence. It is suited for standardized reproducible culturing and allows to investigate the effect of additives.
Endconcentrations
MgSO4 (1 mM)
Traceelement Solution
Glucose (80 mM)
Thiamine (0.3 µM)
antibiotics from 1000x stock
10x Salt solution
Ingredient
Concetration
KH2PO4
227 mM
Na2HPO4
472 mM
NaCl
85 mM
NH4Cl (standard/low)
187 mM / 22.4 mM
fill up to 1 L with deionized water and autoclave
1000x Trace elements
Component
Final Concetration
Concetration in stock solution
Iron(III) chloride
50 mM
13,515 mg/L
Calcium chloride
20 mM
2,220 mg/L
Manganese(II) chloride
10 mM
1,258 mg/L
Zinc sulfate
10 mM
1,615 mg/L
Cobalt(II) chloride
2 mM
260 mg/L
Copper (II) chloride
2 mM
269 mg/L
Nickel(II) chloride
2 mM
259 mg/L
Sodium molybdate
2 mM
412 mg/L
Sodium selenite
2 mM
346 mg/L
Boric acid
2 mM
124 mg/L
Hydrochloric acid
1 mM
20 ml
fill up to 1 L with deionized water and autoclave
All Solutions were prepared and autoclaved separately. The M9 Medium was then mixed under sterile conditions.
Overnight Cultures[+]
spacer# prepare sterile glass tubes of 5 ml of LB+antibiotics
use a sterile pipet tip to pick bacteria from plates
throw the tip into the tubes
incubate in a rotary shaker
SOC Medium[+]
spacerSOC is a complex medium especially suited to increase transformation efficiency in Escherichia coli cells.
components
0.5 % (w/v) yeast extract
2 % (w/v) tryptone
10 mM NaCl
2.5 mM KCl
20 mM MgSO4
fill up with deionized water
adjust to pH 7.5 with NaOH
after autoclaving, add sterile glucose solution to a final concentration of 20 mM
Cloning
B0034_Insertion-Mutagenesis[+]
spacerWith this method B0034 can be inserted into any RFC10 part that begins with prefix-CDS.
The protocol is taken from Bryksin et al. 2014 [1] .
Requirements
30 ng of purified template BioBrick plasmid
chemically competent cells
#CEXL#
#CNNY#
DpnI
Procedure
pipet a Phusion polymerase PCR of 50 µl total volume
water
HF-buffer
30 ng plasmid template
30 pmol #CEXL#
30 pmol #CNNY#
dNTPs
Phusion polymerase
PCR
98 °C for 30'
55 °C for 30'
appropriate extension temperature and time
18 cycles
DpnI digestion
add 10 U of DpnI to the PCR reaction mixture
incubate for 60' at 37 °C
Transformation
transform 1 µl of digestion product into chemically competent cells (might require more for less competent cells)
CPEC[+]
spacerThe CPEC (circular polymerase extension cloning) method [2] was used by iGEM Aachen 2015 to assemble e.g. the BioBricks of glgB and xpk . A big benefit of this approach is that it needs just one enzyme and is scarless. [3]
Linearize the destination vector by amplification with suitable primers. Insert(s) must be amplificated with primers that contain overhangs to create overlapping regions with the backbone (and additional parts).
Mix 100 ng of each fragment together with calculated amounts of H2O, DMSO, dNTPs and DNA polymerase
Repeat 15 cycles of denaturation (98 °C), annealing and elongation in a thermocycler
Transform the construct in a strain of your choice
The overlapping regions have to be designed carefully to avoid secondary structures and to adjust the annealing temperature.
steps of CPEC assembly
after backbone linearisation with appropriate primers, the target parts can be inserted.
Gibson Assembly[+]
spacerWith the Gibson assembly method fast assembly of multiple DNA fragments is possible under isothermal conditions, regardless of fragment length. Therefore fragments with overlapping ends of 15-30 bp are needed and can be created via PCR. The Gibson Assembly was performed using the Gibson Assembly® Master Mix according to the Protocol by New England Biolabs [4] , unless stated otherwise. To design the primers for the respective PCRs the Software Geneious [5] was used.
Set up the reaction according to the table below on ice (2-3 fragment assembly).
Incubate samples in a thermocycler at 50 °C for 15 minutes when 2 or 3 fragments are being assembled or 60 minutes when 4-6 fragments are being assembled. Following incubation, store samples on ice or at –20 °C for subsequent transformation.
Transform NEB 5-alpha Competent E. coli cells with 2 μl of the assembly reaction, following the transformation protocol.
Total Amount of Fragments
0.02-0.5 pmols
Gibson Assembly Master Mix (2X)
10 µl
Deionized H2O
10-X µl
Total Volume
20 µl
Genomic Amplification[+]
spacer# prepare culture
collect cells from the agar plate and resuspend them in 50 µl of ddH2O
centrifuge 1'00" at full speed
measure the DNA concentration
resuspend cells
98 °C for 5'00"
centrifuge 1'00" at full speed
measure the DNA concentration
dilute DNA to 500 ng/µl
(use 1 µg DNA template per 50 µl PCR reaction volume)
use 3'00" denaturing time at the beginning of the PCR program
Transformation[+]
spacer Heat Shock
thaw cells on ice
pipet 50 µl of cells into a pre-chilled PCR tube
pipet DNA:
5 µl of Ligation/CPEC product
1 µl of kit plate concentrate
1 µl of purified plasmid
use the thermocycler for the following program:
task
time
temperature
incubate
30'00
4
shock
1'00
42
incubate
5'00
4
pipet all cells into a 1.5 ml tube with 200 µl of pre-warmed SOC medium
incubate at 37 °C for 60'00"
plate all 251-255 µl onto LB+antibiotic plates
Reference: iGEM Transformation Protocol [6]
Electroporation
add electrocompetent to DNA on ice
move the mixture to the cuvette
dry and shock the cells
add 1ml of SOC medium
incubate at 37 °C for 1-2 h
Plasmid Preparation[+]
spacerPlasmids were prepared as explained in the following manuals
QuickLyse Miniprep Handbook
GE Healthcare illustra plasmidPrep Mini Spin Kit
RDP Assembly[+]
spacerThis year our iGEM team tested and applied the RDP assembly method by Synbiota. We used it to assemble the polycistronic methanol conversion plasmid and to design a monocistronic diversity library to screen for a more efficient methanol uptake plasmid.
RDP parts
RDP parts can be assembled successively to build new devices or whole new circuits with unique functions. The sequence of every RDP part begins and ends with a 4 bp long 5' overhang.
There are four differnet kinds overhangs:
X: 5'- GATG -3'
X': 5'- CTAC -3'
Z: 5'- CGGC -3'
Z': 5'- GCCG -3'
As one can easily notice, X and X' as well as Z and Z' are homologous. A RDP part can have one of two possible orientations:
X - Z' or Z - X'
Usually, you can convert a certain sequence into a RDP part by PCR amplification. You have to use forward and reverse primers with extension containing BsaI restriction sites prior to the overhang sequence you want to introduce.
In the end, the 4 bp overhangs arise from a BsaI digest of the amplification product.
RDP parts with a length of <100 bp can be created by annealing synthesized oligos (HPLC purification recommended).
Assembling the parts
But how are RDP parts joined together to form whole new circuits? Synbiota's RDP assembly kit provides so called anchors. Anchors are linear DNA fragments, contain an antibiotics resistance gene and are bound to magnetic beads. At their downstream ends, anchors have either X' or Z' overhangs.
You can add any desired RDP part with the appropriate overhangs to the anchors that were diluted in an eppi. Matching 4 bp overhangs will be joined by T4 ligase in the reaction mix.
By applying a magnet to the eppi, you can collect all magnetic beads with the bound DNA and simply discard the used reaction mix. After washing and diluting the magnetic beads  again, you can add your next RDP part. After assembling the RDP parts, you'll have to add a so called "cap". It is also provided in Synbiota's assembly kit and contributes an origin of replication.
RDP assembly
bla
You can add your first RDP part to the anchor. T4 ligase in the reaction mix will ligate the homologous overhangs
Before you can add the next RDP part, you have to collect the magnetic beads, discard the used reaction mix and then wash and resuspend the beads again.
In the end, the cap is added to terminate the assembly
Finally, anchor and cap can be joined to form a circular plasmid. By choosing  a certain anchor and cap for the assembly you determine the resistance marker and copy number of the resulting plasmid.
Circularizing the assembly product
After releasing the finished RDP circuit from the magnetic bead, anchor and cap join to form a circular plasmid.
RDP plasmids can again be digested with BsaI and NotI to extract the assembled RDP parts as one composite RDP part.
Extracting a composite RDP part
To extract the assembled RDP parts as one composite RDP part, the plasmid is digested with BsaI to introduce the 4 bp overhangs and NotI to prevent the interference of the backbone fragment in future assembly steps.
Experiences
We experienced the RDP assembly method to be really easy to handle. Both the in silico  work to design new RDP circuits and the efficiency of the method itself convinced us. Thus, RDP assembly can be a suitable alternative to other assembly methods like Gibson Assembly  or CPEC.
Analytics
Acid Hydrolysis[+]
spacerOne way to break down polysaccharides into monomers is acid hydrolysis. This short instruction is based on another protocol [7]
Procedure
Resuspend cell pellet (or dried crudely extracted glycogen) in 500 µL 5 M HCl in an appropriately sized glass vial
Cook sample with tightly closed lid for at least 6 hours at 105 °C
Dry sample until black/brown residue is left. This is done at 85 °C with no lid under a hood with constant airflow!
Store samples at room temperature
Cell Lysis[+]
spacerTo measure non secreted protein cell lysis is essential.
take a 1 ml culture sample into a 2 ml tube and spin down
discard the supernatant and resuspend in 1 ml of 50 mM KPi buffer
add 0.5 g of 0.1 mm glass beads
cool down the sample on ice for 5 minutes
bead mill 3 times for 0'30" and cool the sample on ice inbetween for 1'00"
centrifuge for 2 minutes at 10,000 x g at 4 °C
transfer the supernatant into a new tube
measure the total protein concentration
dilute the sample with 50 mM KPi buffer to a concentration of 6 µg/µl
Electrophoresis[+]
spacerAgarose-Electrophoresis is used to seperate DNA by the number of base pairs.
marker was the GeneRuler 1 kb DNA Ladder
fill pockets with 5 µl DNA ladder or 10 µl sample volume with 6x loading dye
running conditions: 110 V for 30-40 minutes
Glycogen Kit[+]
spacerGlycogen Assay Protocol [8]
1. Sample preparation
Homogenize 106 (animal) cells with 200 µL dH2O on ice.
approximately 109 E.coli cells (1 ml culture with OD 1.25) [9] [10]
Sonicate cells
Boil homegenates for 10 minutes.
Spin/centrifuge at 18 000 g for 10 minutes. The supernatant is now ready for assay!
Add 2-50 µL samples to a 96-well pate.
Adjust volume to 50 µL/well with Hydrolysis Buffer
Notes:
For samples having glucose background, prepare well(s) containing same amont of sample as in the test well as background control.
(Accurate determination of Glycogen in the test samples is best determined via spiking samples with a known amount of standard (example given 0.8 µg)
2. Standard Curve preparation
Dilute the glycogen standard to 0.02 mg/mL by adding 10 µL of the Standard to 990 µL of distilled water, mix well.
Add 0, 2, 4, 8, 10 µL to a series of wells.
Adjust volume to 50 µL/well with Hydrolis Buffer to generate 0, 0.04, 0.08, 0.12, 0.16, and 0.2 µg per well
3. Add 1 µL Hydrolysis Enzyme mix to Standards and samples and mix well.
for dilution samples, dilute with hydrolysis buffer
Incubate for 30 minutes at room temperature.
Note:
If glucose is present in your sample, you may do a glucose control without addition of hydrolysis enzyme to determine the level of glucose background.
4. Reaction Mix
Mix enough reagents for the number of samples and standards to be performed
For each well prepare a total 50 µL Reaction Mix.
Dev.  Buffer: 48.7 µL
Dev. Enzyme Mix: 1 µL
OxiRed Probe: 0.3 µL
Add 50 µL of the Reaction Mix to each well containing the Glycogen Standard or samples, mix well.
5. Measurement
Incubate the reaction for 30 minutes at room temperature protected from light.
Fluorescence (Ex/Em = 535/587 nm)
6. Calculation
Correct background by subtracting the 0 Glycogen Standard sample value (see 2.) from all readings.
Apply sample readings to the standard curve to get B µg of glycogen.
Sample c_(Glycogen) = B / V * D µg/µL
B is the amount of Glycogen from Standard curve (µg)
V is the sample volume added into the reaction well (µL)
D is the sample dilution factor
For spiked samples, correct for any sample interference by subtracting the sample reading from spiked sample.
For Spiked samples:
B = [OD_(sample corrected) / OD_(sample+glycogen Standard (corrected)) - OD_(sample corrected)] * Glycogen Spike (µg)
Notes about Glycogen:
Glycogen  molecular size: 60 000 molecules
Molecular Weight (MW) ~ 106-107 daltons
Glucose MW: 180.16 g/mol
Glycogen Assay Protocol short version
1. Sample preparation
Homogenize 106 cells with 200 µL dH2O on ice.
Boil homegenates for 10 minutes.
Spin/centrifuge at 18 000 g for 10 minutes. The supernatant is now ready for assay!
Add 2-50 µL samples to a 96-well pate.
Adjust volume to 50 µL/well with Hydrolysis Buffer
2. Standard Curve preparation
Dilute the glycogen standard to 0.02 mg/mL by adding 10 µL of the Standard to 990 µL of distilled water, mix well.
Add 0, 2, 4, 8, 10 µL to a series of wells.
Adjust volume to 50 µL/well with Hydrolis Buffer to generate 0, 0.04, 0.08, 0.12, 0.16, and 0.2 µg per well
3. Add 1 µL Hydrolysis Enzyme mix to Standards and samples and mix well.
Incubate for 30 minutes at room temperature.
4. Reaction Mix
Mix enough reagents for the number of samples and standards to be performed
For each well prepare a total 50 µL Reaction Mix.
Dev.  Buffer: 48.7 µL
Dev. Enzyme Mix: 1 µL
OxiRed Probe: 0.3 µL
Add 50 µL of the Reaction Mix to each well containing the Glycogen Standard or samples, mix well.
5. Measurement
Incubate the reaction for 30 minutes at room temperature protected from light.
Fluorescence (Ex/Em = 535/587 nm)
Pre-extraction of glycogen
Another way to subtract the glucose background, is the pre-extraction of glycogen
This procedure removes the majority of glucose background with minimal effect on glycogen
Take tissue or cells to a final content of 30-50 % in 30 % KOH
Heat to 100 °C for 2 h
Cool and add 2 volumes of 95% ethanol to precipitate crude glycogen
Centrifuge (16000 g for 10 min) and collect the precipitate
Suspend the pellet in a minimal amount of distilled water (25 ml original cell culture -> 5 ml were enough)
Acidify sample to pH 3 with 5 M HCl
Add 1 volume of ethanol to reprecipitate
Repeat steps 4-7 two more times, then let sample dry over night (pre-weigh container for drying)
The dried material can be weighed and dissolved for analysis
Iodine Staining[+]
spacerIodine is capable of penetrating the cells and interacting with long polysaccharides as starch or glycogen. Therefore, it is a suitable chemical for qualitative and semi-quantitative glycogen measurement.
Procedure
prepare overnight cultures of strains (additional 20 mM glucose also possible)
to compare tubes with each other:
adjust all cultures to the lowest OD
centrifuge 2 ml and discard supernatant
resuspend pellet in 200 µl iodine solution
to do a glycogen standard for semi-quantitative analysis:
prepare glycogen solutions, each 50 µl, with concentrations from 8 g/L down to 0.125 g/l
add 300 µl iodine solution (recipe see below)
transfer mixture to a well plate (triplicates, 100 µl in each well)
scan the well plate
to compare with glycogen standard:
adjust culture to the OD 1 and transfer 1 ml into tubes
centrifuge and discard supernatant
resuspend pellets in 50 µl water, then add 300 µl staining solution
transfer 100 µl triplicates of mixture to a well plate
to compare with glycogen standard, scan the well plate and analyse the picture
Iodine staining solution (modified version of Lugol's Iodine)
Ingredient
Concentration
I2
5 mM
KI
5 mM
Nash Assay[+]
spacer
Principle of detection
The colorimetric and fluorometric assay first described by T. Nash in 1953 [11] detects the presence of formaldehyde. Two molecules of acetylacetone react with formaldehyde and ammonia to form the strong yellow and fluorescing diacetyl-dihydro lutidine which can be detected via absorption at a wavelength of 412 nm and via fluorescence at an excitation wavelength of 410 nm and an emission wavelength of 510 nm.
Components [12]
Nash reagent (components for 1L)
2 ml acetylaceton
300 g ammonium acetate
6 ml acetic acid
fill up to 1 L with deionized water
A few ml for some samples should be enough depending on the scale of measurement (MTP/cuvette).
Reaction Buffer
50 mM KPi Buffer
40 mM K2HPO4 - 6.97 g/L
10 mM KH2PO4 - 1.36 g/L
5 mM MgSO4
500 µM NAD+
1 M methanol
Procedure
Sample
prepare solutions of whole cell samples as well as supernatant and fragments of lysed cells
adjust to same OD/same protein concentration for comparability
Protocol
mix Nash reagent : reaction buffer : sample in the ratio of 20 : 19 : 1 in a MTP/cuvette
Measurement
measure absorbance at 412 nm
measure fluorescence with an excitation wavelength of 410 nm and an emission wavelength of 510 nm
measure hourly starting directly after mixing for at least 5 hours
incubate at 37 °C at 100 rpm
Keep your samples closed at all times!
SDS-PAGE[+]
spacerIn the SDS-PAGE proteins get denatured and their electrical charge gets masked by the anionic detergent sodium dodecyl sulfate. As a result the proteins can be separated by size through a polacrylamid gel and an electrical current.
Culture
inoculate overnight culture
inoculate main culture (50 ml) with a start OD of 0.15
induce with IPTG at OD 0.6
Take samples (2 ml)
sample: 5-6 hours after induction
sample: ca 18 hours after induction
centrifuge and freeze pellet
Gel
Ingredient
Stacking Gel (1 mL)
Running Gel (5 mL; 15%)
H2O
0.72 mL
1.73 mL
1.5 M Tris
0.13 mL (pH 6.8)
1.3 mL (pH 8.8)
10 % Ammoniumperoxodisulfat
0.01 mL
0.05 mL
10 % N,N′-Methylenbisacrylamid
0.13 mL
0.05 mL
30 % Acrylamid
0.13 mL
1.87 mL
TEMED
0.001 mL
0.002 mL
Mix the first 5 reagents. Then add TEMED, mix well by pipetting up and down and cast the running gel. Use isopropanol to even it out. Let the running gel dry for half an hour, remove excess isopropanol and add the stacking gel with an appropriate comb on top.
sample preparation
resuspend pellet in 150 µl water
measure and dilute to an OD of 10-15
mix 50 µl cell suspension with 10 µl SDS loading dye (if 6x dye)
heat at 95 °C for 15 minutes
load the gel with 7-10 µl
use PageRuler Protein Ladder
Staining & Destaining
stain with Coomassie Blue staining solution for half an hour
destain first with destaining solution, then proceed destaining with destilled water
HPLC[+]
spacer
for methanol
Retention time
Lower detection limit
16,76 +/- 0,02 min
~20 mM
for sugars
HPLC
Beckman System Gold
Column
Organic Acid Resin (PS-DVB), 300 x 8.0 mm, CS-Chromatographie
Detector
RI 166, Beckman
Column Temperature
75°C
Solvent
5 mM H2SO4
Flow Rate
0.8 ml/min
Duration
20 min
Injection Volume
10 µl
Tray Cooling
5°C
Samples have to be centrifuged in order to remove solids (cells/precipitates). No further sample preparation is required.
Dinitrosalicylic Acid Staining[+]
The dinitrosalicylic acid assay is used to analyse the distribution of branches in Glycogen. Dinitrosalicylic acid reacts with the reducing ends of glycogen (or sugars). [13]
Procedure
Take the purified samples (purification via Glycogen Kit section Pre-extraction of glycogen) and add one volume of dinitrosalicylic acid solution (solve 30 g dinitrosalicylic acid in 20 mL 2 M NaOH and add 80 mL water)
Heat stained samples at 90 °C for 15 minutes at 800 rpm
Add one third of the total volume of potassium sodium tartarte solution (40 % wt/vol) to samples, to stabilyze the color
Cool samples down to 25 °C
Analyze samples at 540 nm absorbance and substract the absorbance value of color blank each time
References
↑ Bryksin AV, Bachman HN, Cooper SW, Balavijayan T, Blackstone RM, Du H, Jenkins JP, Haynes CL, Siemer JL, Fiore VF, Barker TH. One primer to rule them all: universal primer that adds BBa_B0034 ribosomal binding site to any coding standard 10 BioBrick. ACS Synth Biol. 2014 Dec 19;3(12):956-9. doi:10.1021/sb500047r. PubMed PMID: 25524097; PubMed Central PMCID: PMC4277749.
↑ Quan, Jiayuan, and Jingdong Tian. “Circular Polymerase Extension Cloning of Complex Gene Libraries and Pathways.” Ed. Paulo Lee Ho. PLoS ONE 4.7 (2009): e6441. PMC. Web. 16 Sept. 2015.
↑ https://j5.jbei.org/j5manual/pages/22.html
↑ https://www.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly
↑ http://www.geneious.com
↑ http://parts.igem.org/Help:Transformation_Protocol Protocol of the iGEM HQ
↑ Blank Lars, Protocol for 13C Tracer Experiments
↑ Glycogen Assay Kit Manual
↑ https://www.thermofisher.com/de/de/home/references/ambion-tech-support/rna-tools-and-calculators/macromolecular-components-of-e.html
↑ http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp
↑ Nash. 1953. The colorimetric estimation of formaldehyde by means of the Hantzsch reaction. Biochemical Journal, 55(3), 416-421.
↑ Kleeberg & Klinger. 1982. Sensitive formaldehyde determination with Nash's reagent and a tryptophan reaction. Journal of Pharmacological Methods, 8(1), 19-31.
↑ S. K. Meur, V. Sitakara Rao, and K. B. De. Spectrophotometric Estimation of Reducing Sugars by Variation of pH. Z. Anal. Chem. 283, 195-197 (1977).
▲
Contact
CC-BY-SA iGEM Aachen 2015
