Team:Aachen/Lab/Methanol/Monocistronic Diversity Library
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Within our polycistronic methanol conversion construct the expression of the four genes mdh, hps, phi and xpk is controlled by the constitutive promoter BBa_J23119 . To tune the expression levels of the genes for a more efficient expression system it is neccessary to control them seperately.
We decided to adjust the expression levels of the four genes by using well-characterized constitutive promoters with different strengths from the Anderson Promoter Library.
Since we do not know which promoter combination is best for the efficient expression of the methanol uptake pathway, we developed a straightforward cloning strategy to build a monocistronic diversity library using the RDP Assembly standard.
Monocistronic methanol conversion plasmid
The general design of a monocistronic methanol uptake plasmid. Later on we will introduce the design of a monocistronic diversity library, that varies in the promoter sequences prior to hps, phi and xpk.
(Mdh = methanol dehydrogenase, Hps = 3-hexulose-6-phosphate, Phi = 6-phospho-3-hexuloisomerase, Xpk = phosphoketolase)
Design
The design of our monocistronic diversity library is mainly based on the RDP Assembly method by Synbiota. To better understand our design, we recommend to first read our detailed description about how the RDP assembly method works.
Because we expected the promoter BBa_J23119 to be the strongest within the Anderson Promoter Library, we chose this one to always control the bottleneck enzyme Mdh in our circuit design. Diversity is introduced by varying the promoters for the three remaining genes hps, phi and xpk. We decided on a set of four promoters from the Anderson Library:
BBa_J23100
BBa_J23104
BBa_J23110
BBa_J23119
These promoters cover a range from medium to very high in strength.
To minimize the genetic instability of our circuit [1] , we designed it to have four different terminator sequences downstream to every CDS.
With three sites to introduce diversity in our circuit and (only) four different promoters, our resulting diversity library contains 43 = 64 variants. Obviously, it is nearly impossible for an iGEM team to clone every single variant seperately during one summer.
Our first step towards an efficient cloning strategy was to split up the circuit into RDP part fragments. To introduce diversity we first planed to add equimolar promoter RDP part mixes at the black labeled sites in the circuit.
Monocistronic diversity library RDP circuit
The circuit is composed by RDP parts. The black labled sites demonstrate, where we planed to introduce diversity by adding equimolar promoter part mixes during the assembly.
(#FO4B#: Kanamycin anchor, #XYD9#: BBa_J23119 RDP part, #PRDW#: BBa_K1585210 RDP part, #PLTB#: trp terminator RDP part, #VO4C#: BBa_K1585211 RDP part, #8AMS#: BBa_B1006 RDP part, #ZALV#: BBa_K1585212 RDP part, #HTSR#: BBa_B1002 RDP part, #ZR1Q#: BBa_K1585213 . BBa_B0015 RDP part, #OZD1#: high copy cap)
But to reduce the assembly steps and increase the assembly efficiency we decided to respectively combine three RDP parts from the circuit to form three different precursor sets.
Precursor sets
One precursor set always consists of:
BBa_B0034 + CDS RDP part
the appropriate terminator RDP part
promoter RDP part
Because we're using four different promoters, each of the three precursor sets will exist of four different precursor constructs, that only differ in their promoter part.
The picture below illustrates how we designed the assembly of our precursor sets.
Assembly Strategy of the precursor plasmids
First, the RDP part carrying the RBS and CDS is assembled to the cap. After the terminator part has been added, the reaction mix is split up into four. Respectively one of the four different promoter parts is added to one of the four reaction mix fractions. After finishing the assembly with an appropriate cap, you will receive four different precursor plasmids from the four different reaction mixes.
( BBa_J23100 , BBa_J23104 , BBa_J23110 and BBa_J23119 represent the four different Anderson promoters)
In total, this strategy results in 12 different precursor plasmids:
Precursor
description
BBa_K1585210 .Trp*. BBa_J23100
mdh.AP00 precursor
BBa_K1585210.Trp*. BBa_J23104
mdh.AP04 precursor
BBa_K1585210.Trp*. BBa_J23110
mdh.AP10 precursor
BBa_K1585210.Trp*. BBa_J23119
mdh.AP19 precursor
BBa_K1585211 . BBa_B1006 .BBa_J23100
hps.AP00 precursor
BBa_K1585211.BBa_B1006.BBa_J23104
hps.AP04 precursor
BBa_K1585211.BBa_B1006.BBa_J23110
hps.AP10 precursor
BBa_K1585211.BBa_B1006.BBa_J23119
hps.AP19 precursor
BBa_K1585212 . BBa_B1002 .BBa_J23100
phi.AP00 precursor
BBa_K1585212.BBa_B1002.BBa_J23104
phi.AP04 precursor
BBa_K1585212.BBa_B1002.BBa_J23110
phi.AP10 precursor
BBa_K1585212.BBa_B1002.BBa_J23119
phi.AP19 precursor
The resulting precursor plasmids have to be purified and cut with BsaI and NotI to convert them into new RDP parts. To prevent complications, we chose chloramphenicol as resistance marker for the precursor plasmids. Since we planed to use kanamycin as resistance marker in the monocistronic diversity library, the cut backbone fragments of the precursor plasmids would not lead to viable clones.
In the end, this is how the design of the final assembly of the monocistronic diversity library looks like: We introduce diversity by adding the precursor RDP parts of one type in equimolar amounts to the reaction.
Final assembly design for the monocistronic diversity library
bla
mdh precursor mix: equimolar mix of the four different BBa_K1585210.Trp.BBa_J231XX precursor RDP parts
hps precursor mix: equimolar mix of the four different BBa_K1585211.BBa_B1006.BBa_J231XX precursor RDP parts
phi precursor mix: equimolar mix of the four different BBa_K1585212.BBa_B1002.BBa_J231XX precursor RDP parts
(#FO4B#: kanamycin cap, #XYD9#: BBa_J23119 RDP part, #ZR1Q#: BBa_K1585213 . BBa_B0015 RDP part, #OZD1#: high copy cap)
The assembly product is supposed to be transformed into an appropriate expression strain (e.g. E. coli BL21 Gold DE3). By testing the growth performance, the resulting transformants can be screened. Strains harboring more efficient methanol conversion plasmids are expected to perform better in presence of methanol than others.
Results
The following table gives an overview on our achieved progress in creating precursor plasmids:
Failure = no valid plasmid sequenced
Success = a valid plasmid without mutations was sequenced
Precursor
Results
Plasmid ID
BBa_K1585210 .Trp*. BBa_J23100
Failure
-
BBa_K1585210.Trp*. BBa_J23104
Success
#OX44#
BBa_K1585210.Trp*. BBa_J23110
Failure
-
BBa_K1585210.Trp*. BBa_J23119
Success
#DNRY#
BBa_K1585211 . BBa_B1006 .BBa_J23100
Success
#ERZK#
BBa_K1585211.BBa_B1006.BBa_J23104
Failure
-
BBa_K1585211.BBa_B1006.BBa_J23110
Failure
-
BBa_K1585211.BBa_B1006.BBa_J23119
Failure
-
BBa_K1585212 . BBa_B1002 .BBa_J23100
Success
#931O#
BBa_K1585212.BBa_B1002.BBa_J23104
Success
#EZYZ#
BBa_K1585212.BBa_B1002.BBa_J23110
Success
#XW1Y#
BBa_K1585212.BBa_B1002.BBa_J23119
Success
#RABS#
*Trp = trp terminator, RDP part provided by Synbiota
Although we successfully created all required basic RDP parts, we were not able to create all the precursor RDP parts to continue with the assembly of the diversity library. In fact, we did not have problems to assemble the precursors. The major problem we had to face were mutations within the promoter sequences.
Synbiota recommends to use HPLC purified oligos for creating your own RDP part below 100 bp in length, but since we had to create lots of RDP parts and a limited budget, we decided to order mostly standard desalting oligos. This might be a reason, why many of our promoter sequences showed deletions and/or point mutations.
A more detailed documentation about creating the RDP parts and sequencing results can be found under:
Laboratory Notebook
Outlook
After characterizing the Mdh activity in our polycistronic strain and considering it's reduced growth rate, we think that the monocistronic expression strategy is worth exploring. Creating a plasmid with atuned expression levels of the genes will provide more knowledge about the interactions between the enzymes and can help searching for an optimal way to make bacteria take up methanol efficiently.
References
↑ Jack BR, Leonard SP, Mishler DM, Renda BA, Leon D, Suárez GA, Barrick JE. Predicting the Genetic Stability of Engineered DNA Sequences with the EFM Calculator. ACS Synth Biol. 2015 Aug 21;4(8):939-43. doi: 10.1021/acssynbio.5b00068. Epub 2015 Jul 1. PubMed PMID: 26096262.
▲
Contact
CC-BY-SA iGEM Aachen 2015
