<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><H1><SPAN class="mw-headline" id="Favorite_Parts">Favorite Parts</SPAN></H1><UL><LI> targeting plasmids <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585350">BBa_K1585350</A><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585351">BBa_K1585351</A></LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585321">K1585321</A> polycistronic <I>GlgCAB</I></LI><LI> polycistronic methanol assimilation pathway <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585241">BBa_K1585241</A></LI></UL><H1><SPAN class="mw-headline" id="Parts_Table">Parts Table</SPAN></H1><P>The following table lists all parts we created and  submitted to the registry.
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick </TH><TH> Description </TH><TH> Submitted to the Registry
</TH></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585200">K1585200</A></TD><TD> Mdh2 (Methanoldehydrogenase 2) from <I>Bacillus methanolicus</I> MGA3, codon optimized for <I>E. coli</I> (<A href="/Team:Aachen/Basic_Part" title="Team:Aachen/Basic Part">Basic Part</A>) </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585201">K1585201</A></TD><TD> Hps (3-hexulose-6-phosphate synthase) from <I>Bacillus methanolicus</I>  MGA3, codon optimized for <I>E. coli</I></TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585202">K1585202</A></TD><TD> Phi (6-Phospho-3-hexuloisomerase) from <I>Bacillus methanolicus</I> MGA3, codon optimized for <I>E. coli</I></TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585203">K1585203</A></TD><TD> Xpk (D-Xylulose-5-phosphate-phosphoketolase) from <I>Bifidobacterium adolescentis</I>, codon optimized for <I>E. coli</I></TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585210">K1585210</A></TD><TD> B0034 with Mdh2 from <I>Bacillus methanolicus</I> MGA3, codon optimized for <I>E. coli</I></TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585211">K1585211</A></TD><TD> B0034 with Hps from <I>Bacillus methanolicus</I> MGA3, codon optimized for <I>E. coli</I></TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585212">K1585212</A></TD><TD> B0034 with Phi from <I>Bacillus methanolicus</I> MGA3, codon optimized for <I>E. coli</I></TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585213">K1585213</A></TD><TD> B0034 with Xpk from <I>Bifidobacterium adolescentis</I>, codon optimized for <I>E. coli</I></TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585300">K1585300</A></TD><TD> GlgA from <I>E. coli</I></TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585301">K1585301</A></TD><TD> GlgB from <I>E. coli</I></TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585310">K1585310</A></TD><TD> B0034 with glgA from <I>E. coli</I></TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585311">K1585311</A></TD><TD> B0034 with glgB from <I>E. coli</I></TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585312">K1585312</A></TD><TD> B0034 with glgC deltaG336D </TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585320">K1585320</A></TD><TD> Combined translational unit of  of glgA and glgB  </TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585321">K1585321</A></TD><TD> Combined translational unit of glgC, glgA and glgB (<A href="/Team:Aachen/Composite_Part" title="Team:Aachen/Composite Part">Composite Part</A>) </TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585241">K1585241</A></TD><TD> Polycistronic expression plasmid of Mdh, Hps, Phi and Xpk controlled by Anderson Promoter 19 (J23119), Ribosome binding site B0034 and Terminator B0015 </TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585350:Design">K1585350</A></TD><TD> anti glgX targeting plasmid to generate gene knockout together with pCas </TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585351">K1585351</A></TD><TD> anti glgP targeting plasmid to generate gene knockout together with pCas </TD><TD> ✓
</TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585100">K1585100</A></TD><TD> Anderson promoter J23100 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585101">K1585101</A></TD><TD> Anderson promoter J23101 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585102">K1585102</A></TD><TD> Anderson promoter J23102 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585103">K1585103</A></TD><TD> Anderson promoter J23103 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585104">K1585104</A></TD><TD> Anderson promoter J23104 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585105">K1585105</A></TD><TD> Anderson promoter J23105 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585106">K1585106</A></TD><TD> Anderson promoter J23106 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585110">K1585110</A></TD><TD> Anderson promoter J23110 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585113">K1585113</A></TD><TD> Anderson promoter J23113 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585115">K1585115</A></TD><TD> Anderson promoter J23115 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585116">K1585116</A></TD><TD> Anderson promoter J23116 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585117">K1585117</A></TD><TD> Anderson promoter J23117 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585118">K1585118</A></TD><TD> Anderson promoter J23118 with lac I binding site </TD></TR><TR><TD><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585119">K1585119</A></TD><TD> Anderson promoter J23119 with lac I binding site </TD></TR></TBODY></TABLE><P>&lt;groupparts&gt;iGEM015 Aachen&lt;/groupparts&gt;</P><H2><SPAN class="mw-headline" id="K1585210">K1585210</SPAN></H2><P>This is the translational unit of methanol dehydrogenase 2 from <I>Bacillus methanolicus</I> MGA3. It is codon optimized for <I>E. coli</I>. The mdh catalyzes the conversion of methanol to formaldehyde.
</P><CENTER><DIV class="figure" style="float:center; margin: 0px 10px 10px 0px; border:0px solid #aaa;width:600px;padding:10px 10px 0px 0px;"><TABLE><TBODY><TR><TD><B>Catalysed reaction by Mdh</B>Methanol + NAD → Formaldehyde + NADH + H<SUP>+</SUP>.
</TD></TR></TBODY></TABLE></DIV></CENTER><P><B>Expression verification &amp; localization of the Mdh</B></P><P>The expression of the Mdh behind a T7 promoter was verified by doing a SDS-PAGE. To localize the Mdh not only whole cells were used but also the fragments of lysed cells and the respective supernatant. The expected bands were clearly visible in all samples proving the expression of the Mdh. However, the Mdh specific band had the highest intensity in the sample of the cell fragments whereas the same band was only of slight intensity in the supernatant. This indicates that the Mdh is incorporated into inclusion bodies. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Expression test of #IGEM#</B>A SDS-PAGE of the whole cell, the cell fragments and the lysate supernatant of the Mdh expressing BL21 Gold (#IGEM#) and the GlgC expressing BL21 Gold (#OHES#) as a negative control was run. The expected band at the weigth of the Mdh can be clearly seen in all #IGEM# samples. The according band with the strongest intensity occurs in the lane with #IGEM# fragments indicating the formation of inclusion bodies containg the Mdh.
</TD></TR></TBODY></TABLE></DIV><P>
To avoid the incorporation of the Mdh into inclusion bodies, the alternative <I>E. coli</I> strains SHuffle T7 Express and C43 were used. SHuffle T7 Express allows more efficient protein folding in the cytoplasm and lacks proteases whereas C43 allows the expression of toxic proteins. Additionally, cultivation at a lower temperature of 30 °C was tested. It was shown that all strains were able to grow on M9 medium, however, another test for the expression and localization of the Mdh did not reveal major differences between the strains. In every case the Mdh was still incorporated into inclusion bodies.
</P><P><B>Functionality of the expressed Mdh</B></P><P>To test if the expressed Mdh is functional, we modified a colorimetric and fluorescent formaldehyde assay described by <A href="/wiki/index.php?title=2015.igem.org/Team:Aachen/Notebook/Protocols&amp;action=edit&amp;redlink=1" class="new" title="2015.igem.org/Team:Aachen/Notebook/Protocols (page does not exist)">T. Nash</A>. In the in the presence of formaldehyde the yellow and fluorescing diacetyl-dihydro lutidine is formed. In the reaction catalyzed by the Mdh methanol is converted to formaldehyde which can be detected by the mentioned assay.
</P><P>First, the different cells as well as their respective fragments and lysate supernatants were screened for formaldehyde production. The samples were taken 6 h after induction from shake flask cultures. Several cells, fragments and supernatants showed a production of formaldehyde but the Mdh activity in the intact cells was higher for each strain.
</P><P>The formaldehyde assay was repeated only with whole cells and additional samples taken 20 h after induction. Moreover, the assay was conducted not only at 37 °C but also at 30 °C. The GlgC expressing BL21 Gold was used as a negative control. Again, in several strains functional Mdh could be detected and some general conclusions could be drawn:
</P><UL><LI> More formaldehyde was produced in the samples taken 6 h after induction</LI><LI> Strains cultivated at 37 °C show a stronger response than the same ones cultivated at 30 °C</LI><LI> The assay works better and faster at an incubation temperature of 37 °C</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Comparison of different strains at varying cultivation conditions.</B>The highest activity could be shown in BL21 Gold (DE3) at a cultivation temperature of 37 °C in M9 in a sample taken 6 h after induction.
</TD></TR></TBODY></TABLE></DIV><P>By far the highest formaldehyde production was observed in the BL21 Gold (DE3) cells 6 h after induction cultivated at 37 °C despite its formation of inclusion bodies.
</P><P>To show the significance of the production of formaldehyde the assay was conducted with the Mdh expressing BL21 Gold (DE3) cultivated in the <A href="/wiki/index.php?title=2015.igem.org/Team:Aachen/Lab/Methanol/Labeling_Experiment&amp;action=edit&amp;redlink=1" class="new" title="2015.igem.org/Team:Aachen/Lab/Methanol/Labeling Experiment (page does not exist)">labled methanol experiment</A> along with the BL21 Gold (DE3) expressing GlgC as a negative control, both in multiple replicates (n&gt;35).
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Formaldehyde formation during in vivo assay</B>#IGEM# expresses Mdh and #OHES# expresses glgC as a negative control (4.5 h after induction). Both constructs were incorporated in vector pSB1A30 and the strains were cultivated at 37 °C. Each value is made from at least 35 replicates of each construct.
</TD></TR></TBODY></TABLE></DIV><P>The Mdh expressing strain showed significantly more formaldehyde production indicating a functional expression of the Mdh.
</P><H2><SPAN class="mw-headline" id="K1585211">K1585211</SPAN></H2><P>This is the translational unit of 3-hexulose-6-phosphate synthase (Hps) from <I>Bacillus methanolicus</I> MGA3. It is codon optimized for <I>E. coli</I>.
</P><CENTER><DIV class="figure" style="float:center; margin: 0px 10px 10px 0px; border:0px solid #aaa;width:600px;padding:10px 10px 0px 0px;"><TABLE><TBODY><TR><TD><B>Catalysed reaction by Hps</B>Formaldehyde + Ribulose-5-phosphat  →  D-arabino-hex-3-ulose 6-phosphate.
</TD></TR></TBODY></TABLE></DIV></CENTER><H2><SPAN class="mw-headline" id="K1585212">K1585212</SPAN></H2><P>This is the translational unit of the 6-phospho-3-hexuloisomerase (PHI) from <I>Bacillus methanolicus</I> MGA3. It is codon optimized for <I>E. coli</I>.
</P><CENTER><DIV class="figure" style="float:center; margin: 0px 10px 10px 0px; border:0px solid #aaa;width:600px;padding:10px 10px 0px 0px;"><TABLE><TBODY><TR><TD><B>Catalysed reaction by Phi</B>D-arabino-hex-3-ulose 6-phosphate → D-fructose 6-phosphate.
</TD></TR></TBODY></TABLE></DIV></CENTER><H2><SPAN class="mw-headline" id="K1585213">K1585213</SPAN></H2><P>This is the translational unit for the D-Xylulose-5-phosphate-phosphoketolase (Xpk). It derives from Bifidobacterium adolescentis and is codon optimized for <I>E. coli</I>.
</P><CENTER><DIV class="figure" style="float:center; margin: 0px 10px 10px 0px; border:0px solid #aaa;width:600px;padding:10px 10px 0px 0px;"><TABLE><TBODY><TR><TD><B>Catalysed reaction by Xpk</B>D-Xylulose-5-phosphate + phosphate → Acetyl-phosphate + D-Glyceraldehyde-3-phosphate.
</TD></TR></TBODY></TABLE></DIV></CENTER><H2><SPAN class="mw-headline" id="K1585310">K1585310</SPAN></H2><P>This is the translational unit of GlgA (Glycogen synthase), codon-optimized for <I>E.coli</I>. The enzyme elongates linear glycogen chains by using ADP-glucose units and forming α-1,4-glycosidic bonds. Through autophosphorylation, GlgA is also the starting point of glycogen synthesis in bacteria.
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology">Usage and Biology</SPAN></H3><P>The construct was confirmed by sequencing. Moreover, the expression of <I>glgA</I> was tested by a SDS-PAGE. In the picture below, you can see that <I>glgA</I> is expressed in strains containing BBa_K1585310 in a <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1362093">pSB1K30</A> expression vector. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS-PAGE of GlgA in comparison to mRFP</B>After induction with IPTG at OD=0.6, <I>glgA</I> was expressed. Samples were taken 6 and 18.5 hours after induction. The small arrows indicates the expected bands for GlgA. mRFP in pSB1K30 (T7 promoter) was used as the negative control.
</TD></TR></TBODY></TABLE></DIV><P>On top of that, the function of the construct was proven by an iodine staining with BL21 Gold (DE3) and BL21 Gold (DE3) expressing <I>glgA</I> in pSB1K30. The iodine staining is performed with Lugol's iodine which dyes glycogen resulting in a brown color. If more glycogen is present, the color of stainend cultures is darker. In the picture below, the brown color of BL21 Gold (DE3) expressing <I>glgA</I> indicates that the enzyme has the expected activity.
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Iodine staining of BL21 Gold (DE3) with <I>glgA</I> in pSB1K30 compared to BL21 Gold (DE3) wild type</B>Iodine staining was performed with overnight cultures which were adjusted to the same OD. The darker color of the strain expressing <I>glgA</I> shows that more glycogen is present. Therefore, the functionality of the glycogen synthase is shown.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="K1585311">K1585311</SPAN></H2><P>This is the translational unit of GlgB (glycogen branching enzyme) from <I>E. coli</I>. The enzyme catalyzes the formation of α-1,6-branches in glycogen synthesis.
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology_2">Usage and Biology</SPAN></H3><P>The construct was confirmed by sequencing. The expression was tested in Bl21 Gold (DE3) strains containing BBa_K1585311 in a <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1362093">pSB1K30</A> expression vector after IPTG induction. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS-PAGE of GlgB in comparison to mRFP</B>After induction with IPTG at OD=0.6, <I>glgB</I> was expressed. Samples were taken 6 and 19.5 hours after induction. The small arrows indicates the expected bands for GlgB. mRFP in pSB1K30 (T7 promoter) was used as negative control.
</TD></TR></TBODY></TABLE></DIV><P>We wanted to investigate the functionality of the strains expression <I>glgB</I>. Therefore, we used <A href="/Team:Aachen/Notebook/Protocols#Dinitrosalicylic_Acid_Staining" title="Team:Aachen/Notebook/Protocols"> dinitrosalicylic acid staining (DNS)</A> for detection of reducing ends <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> which should correspond to the branching frequency. All samples of  <I>glgB</I> strains were compared to wild type samples. For best comparison the samples were grown to stationary phase and adjusted to the same OD before staining. Since every sugar or alkyl would have reacted with 3,5-dinitrosalicylic acid, we purified our samples before the staining. In order to identify the branching frequency, we analyzed the absorbance values of the purified samples compared to the absorbance values of hydrolyzed samples. By calculating the absorbance ratio of the non-hydrolyzed divided by the hydrolyzed samples, we aimed for information about the branches per glycogen unit. 
The reaction principle can be described as follows.
</P><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>reaction principle</B></TD></TR></TBODY></TABLE></DIV><P><SPAN style="color:transparent">a</SPAN></P><P>3,5-Dinitrosalicylic acid reacts with the reducing ends to 3-Amino-5-Nitrosalicylic acid, resulting in a changed π-system and therefore a change in absorbance. During the reaction, the reducing ends are oxidized whereas one nitro group of 3,5-Dinitrosalicylic acid is reduced to an amino group of 3-Amino-5-Nitrosalicylic acid. The more free reducing ends are present the more 3-Amino-5-Nitrosalicylic acid will be formed. Thus, the absorbance will increase.
</P><H2><SPAN class="mw-headline" id="Results_from_Dinitrosalicylic_Acid_Staining">Results from Dinitrosalicylic Acid Staining</SPAN></H2><P>The absorbance of non-hydrolyzed samples of the <I>glgB</I> strain is higher than the absorbance of the wild type (see figure <I>DNS staining of hydrolyzed samples of the </I>glgB<I> strain and wild type</I>). To identify the amount of glycogen in the samples, we hydrolyzed the samples and applied our staining. Our results show that the number of branches in glycogen is higher in the <I>glgB</I> strain compared to the wild type (<I>DNS staining of non-hydrolyzed samples of the </I>glgB<I> strain and wild type</I>).
</P><DIV class="figure-double col-md-12"><DIV class="figure figure-doubleM col-md-6"><TABLE><TBODY><TR><TD><B>DNS staining of non-hydrolyzed samples of the <I>glgB</I> strain and wild type</B>The non-hydrolyzed samples of the <I>glgB</I> strain and wild type show that the overexpression of <I>glgB</I> leads to glycogen molecules with a higher number of branches compared to the wild type. Error bars show propagation of uncertainty.
</TD></TR></TBODY></TABLE></DIV><DIV class="figure figure-doubleM col-md-6"><TABLE><TBODY><TR><TD><B>DNS staining of hydrolyzed samples of the <I>glgB</I> strain and wild type</B>The hydrolyzed samples indicate that the glycogen amount is higher in the wild type compared to the <I>glgB</I> overexpressing strain. Error bars show propagation of uncertainty. Since cell fragments are still left in the samples, the blank also shows an absorbance. Therefore, the line does not go though the origin.
</TD></TR></TBODY></TABLE></DIV></DIV><P><SPAN style="color:transparent">a</SPAN></P><P>The non-hydrolyzed values divided by the hydrolyzed values generate a ratio. Based on this ratio we can identify the number of branches per glycogen unit. It indicates that the number of branches per unit is higher in the <I>glgB</I> strain compared to the wild type.
</P><DIV class="figure-double col-md-12"><DIV class="figure figure-doubleM col-md-6"><TABLE><TBODY><TR><TD><B>ratio of non-hydrolyzed and hydrolyzed DNS staining values of the <I>glgB</I> strain and wild type</B>Although the amount of glycogen is higher in the wild type (see figure above), the number of branches per glycogen unit is higher in the <I>glgB</I> strain. Therefore the overexpression of <I>glgB</I> successfully increased the number of branches.  Error bars show propagation of uncertainty.
</TD></TR></TBODY></TABLE></DIV><DIV class="figure figure-doubleM col-md-6"><TABLE><TBODY><TR><TD><B>glycogen calibration curve</B>With the calibration curve we are able to determine the glycogen concentration of our samples. Error bars show propagation of uncertainty.
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="K1585312">K1585312</SPAN></H2><P>This is the translational unit of GlgC, the ADP-glucose pyrophophorylase. It creates ADP-glucose from glucose-1-phosphate for glycogen synthesis. 
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology_3">Usage and Biology</SPAN></H3><P>The BioBrick is based on <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K118016">BBa_K118016</A> by Team Edinburg 2008, but was fused with the well-characterized RBS B0034.
For expression tests, BBa_K1585312 was cloned into a <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1362091">pSB1A30</A> expression vector and transformed in BL21 Gold (DE3).
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS-PAGE of GlgC in comparison to mRFP</B><I>glgC</I> was expressed in pSB1A30 and compared to mRFP in pSB1A30 (T7 promoter) expression. After induction with IPTG at OD=0.6, <I>glgC</I> was expressed, indicated by the small arrows pointing to the GlgC bands of 48 kDa. No difference between induced and uninduced was observed. After overnight expression the strong bands were no longer present.
</TD></TR></TBODY></TABLE></DIV><P>On top of that, the function of the construct was proven by an iodine staining with BL21 Gold (DE3) and BL21 Gold (DE3) expressing <I>glgC</I> in pSB1K30. The iodine staining is performed with Lugol's iodine which dyes glycogen resulting in a brown color. If more glycogen is present, the color of stainend cultures is darker. In the picture below, the brown color of BL21 Gold (DE3) expressing <I>glgC</I> indicates that the enzyme has the expected activity.
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Iodine staining of BL21 Gold (DE3) with <I>glgC</I> in pSB1K30 compared to BL21 Gold (DE3) wild type</B>Iodine staining was performed with LB and LB + 40 mM glucose overnight cultures which were adjusted to the same OD. The cultures grown in medium with extra glucose were darker than those grown on just LB medium. The darker color of the strain expressing <I>glgC</I> in both samples shows that more glycogen was present. Therefore, the functionality of the ADP-glucose pyrophosphorylase is proved.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="K1585320">K1585320</SPAN></H2><P>This composite part combines <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1585320">glgA and glgB</A> for glycogen synthesis. GlgA uses ADP-glucose for chain elongation (α-1,4 linked) whereas GlgB uses existing chains to form α-1,6-linked branches.
</P><H2><SPAN class="mw-headline" id="K1585321">K1585321</SPAN></H2><P>This composite part combines all 3 enzymes involved in glycogen synthesis. The ADP-glucose pyrophophorylase (GlgC) forms ADP-glucose from ATP and glucose-1-phosphate, the glycogen synthase (GlgA) elongates α-1,4-linked chains and the branching enzyme (GlgB) catalyzes the formation of α-1,6-linked branches. This construct is an extension and improvement of <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K118016">BBa_K118016</A>.
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology_4">Usage and Biology</SPAN></H3><P>In order to upregulate the whole glycogen synthesis pathway, the polycistronic plasmid was built. The <I>glgC</I> coding sequence is based on the part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K118016">BBa_K118016</A> from Team Edinburgh 2008, but he RBS B0034 was added to the existing Biobrick.
The construct was confirmed by sequencing. The expression of all three enzymes GlgC, GlgA and GlgB was tested in Bl21 Gold (DE3) strains containing BBa_K1585321 in a pSB1A30 expression vector. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS-PAGE of <I>glgCAB</I> in pSB1A30</B>glgCAB in pSB1A30 was expressed in Bl21 Gold (DE3) strains and IPTG induced. The small arrows indicates the expected bands for all three enzymes. The Bl21 Gold (DE3) wild type was used as negative control.
</TD></TR></TBODY></TABLE></DIV><P>The combined functionality was characterized by iodine staining (see picture below). It was performed with Lugol's iodine which dyes glycogen in a brownish color. If more glycogen is present, the color of stainend cultures is darker. The darker staining of Bl21 Gold (DE3) transformants of BBa_K1585321 picture indicates considerably more glycogen accumulations compared to the wild type.
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Iodine stained samples of Bl21 Gold (DE3) with glgCAB compared to the wild type</B>The samples were taken from overnight cultures in LB + 20 mM glucose and were adjusted to the same OD before staining with 200 µl Lugol's iodine solution.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="K1585241">K1585241</SPAN></H2><P>Polycistronic expression plasmid of Mdh, Hps, Phi and Xpk controlled by Anderson Promoter 19 (J23119), Ribosome binding site B0034 and Terminator B0015
</P><H2><SPAN class="mw-headline" id="K1585350">K1585350</SPAN></H2><P>This plasmid is targeting <I>glgX</I> in the <I>E. coli</I> genome by expressing a single guide RNA (sgRNA) for CRISPR/Cas9 genome editing. It contains a repair template that disrupts the beginning of the <I>glgX</I> coding sequence and introduces a sequence with multiple stop codons.
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology_5">Usage and Biology</SPAN></H3><P>The construct is part of a two-plasmid knockout system published by <A rel="nofollow" class="external text" href="http://aem.asm.org/content/81/7/2506.long%7C">Jiang et al</A>. This part, the pTargetT, contains the sgRNA to target the <I>glgX</I> coding sequence. Additionally, the construct consists of a repair template with two 400 bp homology arms for homology-directed repair. To succesfully knock out the <I>glgX</I>, you need a second plasmid, pCas. It contains the pCas gene as well as a temperature-sensitive repA101ts ori. To increase recombination efficiency in <I>E. coli</I>, it also expresses lambda-Red genes behind an inducible arabinose-promoter.  Upon induction with IPTG, a sgRNA is transcribed to cut the pMB1 ori of the pTargetT.
</P><P>After cloning both the targeting plasmid and the pCas plasmid in BL21 Gold (DE3) and inducing with arabinose, the Cas9 gene is expressed. Guided by the sgRNA, it cleaves the double strand at the beginning of the <I>glgX</I> gene. The succesful genome editing could be verified by sequencing.
</P><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>Genomic region and sequencing chromatograms of BL21 Gold (DE3) Δ<I>glgX</I></B>The sequencing results show that the &quot;IGEM AACHEN MULTISTOP&quot; sequence disrupts the gene. This insert contains a stop codon in each possible frame. The picture demonstrates the functionality of BBa_K1585350.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="K1585351">K1585351</SPAN></H2><P>This plasmid is targeting <I>glgP</I> in the <I>E. coli</I> genome by expressing two sgRNAs for CRISPR/Cas9 genome editing. The complete <I>glgP</I> coding sequence is cut out and multiple stop codons are introduced by the repair templates. 
</P><H3><SPAN class="mw-headline" id="Usage_and_Biology_6">Usage and Biology</SPAN></H3><P>The construct is part of a two-plasmid knockout system published by <A rel="nofollow" class="external text" href="http://aem.asm.org/content/81/7/2506.long%7C">Jiang et al</A>. This part, the pTargetT, contains the sgRNA to target the <I>glgP</I> coding sequence. Additionally, the construct consists of a repair template with two 400 bp homology arms for homology-directed repair. To succesfully knock out the <I>glgP</I>, you need a second plasmid, pCas. It contains the pCas gene as well as a temperature-sensitive repA101ts ori. To increase recombination efficiency in <I>E. coli</I>, it also expresses lambda-Red genes behind an inducible arabinose-promoter. Upon induction with IPTG, a sgRNA is transcribed to cut the pMB1 ori of the pTargetT.
After cloning both the targeting plasmid and the pCas plasmid in BL21 Gold (DE3) and inducing with arabinose, the Cas9 gene is expressed. Guided by the sgRNA, it cleaves the double strand at the beginning of the <I>glgX</I> gene. The succesful genome editing could be verified by sequencing.
</P><P>Sequencings of the respective genomic region have shown that <I>glgP</I> was successfully cut out. 
In further experiments, the effect of this gene knockout was characterized. The expected consequence was an accumulation of glycogen due to the absense of a glycogen phosphorylase degrading α-1,4-linked glucose. We confirmed this by iodine staining with Lugol's iodine, which dyes glycogen in a brown color. If more glycogen is present, the color of stainend cultures is darker. In the picture below, the BL21 Gold (DE3) ∆<I>glgP</I> strain is much darker than the wild type. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B><I>glgP</I> gene knockout</B>The knockout was achieved cutting out most of the coding sequence of <I>glgP</I>, thereby inserting a short coding sequence that contains multiple stop codons in both frames.
</TD></TR></TBODY></TABLE></DIV><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>Iodine staining of BL21 Gold (DE3) <I>glgP</I> knockout compared to BL21 Gold (DE3) wild type</B>Iodine staining was performed with overnight cultures in M9 medium which were adjusted to the same OD. The darker color of the <I>glgP</I> knockout strain shows that more glycogen is present. Thus, the effectivity of the targeting plasmid is shown.
</TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">S. K. Meur, V. Sitakara Rao, and K. B. De. Spectrophotometric Estimation of Reducing Sugars by Variation of pH. Z. Anal. Chem. <B>283</B>, 195-197 (1977)</SPAN></LI></OL></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>