<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Glycogen_Targeting_Plasmids skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Glycogen Targeting Plasmids</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmPlasmidconstruction" href="/Team:Aachen/Lab/Glycogen/Knockouts#Plasmid_Construction"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Plasmid Construction</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><H1><SPAN class="mw-headline" id="In_silico_work">In silico work</SPAN></H1><H2><SPAN class="mw-headline" id="15-05-05">15-05-05</SPAN></H2><UL><LI> designed pTargetT for glgX knock-out</LI><LI> Gibson Assembly with 3 fragments:</LI><LI> gRNA synthesized (no amplification required for CPEC)</LI><LI> backbone with kanamycin resistance :
</LI><LI> to be amplified by PCR from pTargetF</LI></UL><LI> editing template with homology arms and Insert (IGEM*AACHEN)
</LI><LI> to be assembled from genomic PCR of left and right homology arm and primers with overhangs (IGEM AACHEN)</LI><H2><SPAN class="mw-headline" id="15-05-06">15-05-06</SPAN></H2><UL><LI> decided to use CPEC assembly method
</LI><LI> 4 (5) fragements: backbone, left homology arm with IGEM AACHEN overhang, right homology arm with IGEM AACHEN overhang, sgRNA (+sgRNA right for two double-strand breaks)</LI><LI> designed overhangs and primers for genomic PCR and backbone PCR</LI></UL><P><B>glgP</B></P><TABLE class="wikitable"><TBODY><TR><TH> PCR </TH><TH> forward primer </TH><TH> Melting temperature fwd </TH><TH> reverse primer </TH><TH> Melting temperature</TH><TH> expected length of PCR product  [bp]
</TH></TR><TR><TD> genomic PCR left arm </TD><TD> #WQRS# </TD><TD> 66.2 °C </TD><TD> #PF1Y# </TD><TD> 66.9°C </TD><TD> 422
</TD></TR><TR><TD> genomic PCR right arm </TD><TD> #KQCX# </TD><TD>65.3°C </TD><TD> #COT4#  </TD><TD> 65.3°C </TD><TD> 461
</TD></TR><TR><TD><S>pTargetF backbone</S></TD><TD> #Z9T8# </TD><TD> 63.2 °C </TD><TD> #BBDC# </TD><TD> 59.1°C </TD><TD> 1950
</TD></TR><TR><TD> pSB1A3 backbone #93QQ# </TD><TD> #8E3Q# </TD><TD> 73.3 °C </TD><TD> #WVZQ# </TD><TD> 67.5°C </TD><TD> 2205
</TD></TR><TR><TD> amplification of sgRNA left (synthesized with overhang)</TD><TD> NOT REQUIRED #PXMQ# </TD><TD> 65.8°C </TD><TD> NOT REQUIRED #YESR# </TD><TD> 65.6°C </TD><TD> 205
</TD></TR><TR><TD> amplification of sgRNA right (synthesized with overhang)</TD><TD> NOT REQUIRED #A4DX# </TD><TD> 68.2°C </TD><TD> NOT REQUIRED #VRBM# </TD><TD> 69.3°C </TD><TD> 252
</TD></TR></TBODY></TABLE><P><B>glgX</B></P><TABLE class="wikitable"><TBODY><TR><TH> PCR </TH><TH> forward primer </TH><TH> Melting temperature fwd </TH><TH> reverse primer </TH><TH> Melting temperature</TH><TH> expected length of PCR product
</TH></TR><TR><TD> genomic PCR left arm </TD><TD> #BQBK# </TD><TD> 65.5 °C </TD><TD> #HPO4# </TD><TD> 66.5°C </TD><TD> 433
</TD></TR><TR><TD> genomic PCR right arm </TD><TD> #SOZQ# </TD><TD>66.0°C </TD><TD> #K4TH#  </TD><TD> 66.1°C </TD><TD> 446
</TD></TR><TR><TD><S>pTargetF backbone</S></TD><TD> #Z9T8# </TD><TD> 63.2 °C </TD><TD> #BBDC# </TD><TD> 59.1°C </TD><TD> 1950
</TD></TR><TR><TD> pSB1A3 backbone #93QQ# </TD><TD> #8E3Q# </TD><TD> 73.3 °C </TD><TD> #WVZQ# </TD><TD> 67.5°C </TD><TD> 2205
</TD></TR><TR><TD> amplification of sgRNA (synthesized with overhang)</TD><TD> NOT REQUIRED #PXMQ# </TD><TD> 65.8°C </TD><TD> NOT REQUIRED #QEV4# </TD><TD> 65.5°C </TD><TD> 208
</TD></TR></TBODY></TABLE><P><B>glgX and glgP</B></P><TABLE class="wikitable"><TBODY><TR><TH> PCR </TH><TH> forward primer </TH><TH> Melting temperature fwd </TH><TH> reverse primer </TH><TH> Melting temperature</TH><TH> expected length of PCR product
</TH></TR><TR><TD> glgP and backbone fragment </TD><TD> #8E3Q# </TD><TD> ... </TD><TD> #4HNZ# </TD><TD> ... </TD><TD> 3437
</TD></TR><TR><TD> glgX fragment </TD><TD> #RTT6# </TD><TD> ... </TD><TD> #K4TH#  </TD><TD> ... </TD><TD> 1008
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-07">15-05-07</SPAN></H2><UL><LI> checked primers for mismatches in the genome</LI><LI> ordered primers for genomic PCR and backbone amplification (primer for snythesized gRNA not required, enough template for CPEC is synthesized)</LI><LI> made a calculation table for CPEC (required volumes/concentrations...)
</LI><LI> useful tool to calculate molecular masses of DNA molecules <A rel="nofollow" class="external text" href="http://www.bioinformatics.org/sms2/dna_mw.html">1</A></LI><LI> remember to choose double-stranded under &quot;treat sequences&quot;</LI></UL><H1><SPAN class="mw-headline" id="Wetlab_Documentation">Wetlab Documentation</SPAN></H1><H2><SPAN class="mw-headline" id="15-05-05_2">15-05-05</SPAN></H2><UL><LI> the plasmids pCas and pTargetF were <A href="/wiki/index.php?title=Manual/Plasmid_Extraction_from_Filter_Paper&amp;action=edit&amp;redlink=1" class="new" title="Manual/Plasmid Extraction from Filter Paper (page does not exist)">extracted from half of the filter paper</A> into #WATT# and #VMLL#</LI><LI> The remaining half of the filter paper is in #NCKV# (pCas) and #9OLA# (pTargetF)</LI><LI> produced kanamycin stocks</LI><LI> produced LB-Medium for agar plates (LB+K)</LI><LI> make agar plates with kanamycin from 1L LB-Medium </LI></UL><UL><LI> electroporation of pCas (#WATT#) and pTargetF (#VMLL#) into electrocompetent NEB10beta - we plated both on LB+K which is why pTargetF clones did not survive. Incubation at 30 °C</LI></UL><P><SPAN style="color: orange;"><B>NEVER HAVE pCas OVER 30 °C!!</B></SPAN></P><H2><SPAN class="mw-headline" id="15-05-06_2">15-05-06</SPAN></H2><UL><LI> Transformation of pTargetF did not work, no colonies</LI><LI> (maybe cultivate next time not at 30 °C, but at 37 °C)</LI><LI> make Master Plate and overnight cultures of pCas (three colonies: 1, 2, 3)</LI></UL><H2><SPAN class="mw-headline" id="15-05-07_2">15-05-07</SPAN></H2><UL><LI> make new overnight cultures (Start 9:30) from master plate, three times (1, 2, 3) --&gt; did not grow fast enough to make plasmid prep in the evening</LI><LI> plasmid prep of non shaked overnight cultures, low concentrations (1: 45.5 ug/ul   2: 22.2 ug/ul   3: 34.0 ug/ul) </LI><LI> cryos of <B>non shaked</B> overnight cultures</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> pCas clone (non-shaked) # </TH><TH> purified plasmid ID</TH><TH> cryo ID
</TH></TR><TR><TD> 1 </TD><TD>#F4VY# </TD><TD> #RF9Q#
</TD></TR><TR><TD> 2 </TD><TD> #KP48# </TD><TD> #R9WX#
</TD></TR><TR><TD> 3 </TD><TD> #HXDA# </TD><TD> #HFQM#
</TD></TR></TBODY></TABLE><UL><LI> make new overnight cultures for tomorrow (from master plate)</LI></UL><H2><SPAN class="mw-headline" id="15-05-08">15-05-08</SPAN></H2><UL><LI> cryo cultures from new pCas cultures</LI><LI> plasmid preps from pCas cultures </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> pCas clone (new) # </TH><TH> purified plasmid ID</TH><TH> cryo ID
</TH></TR><TR><TD> 1 </TD><TD> #EANB# </TD><TD> #YPKO#
</TD></TR><TR><TD> 2 </TD><TD> #ORLH# </TD><TD> #SD9P#
</TD></TR><TR><TD> 3 </TD><TD> #PFOK# </TD><TD> #FSRQ#
</TD></TR></TBODY></TABLE><UL><LI> test digestion with EcoRI - expecting three bands three strong bands (11044, 8452, <B>6939</B>, 5618, <B>4105</B>, <B>1513</B>)</LI><LI> the gel (1kb ladder, #ORLH#, #PFOK#, #EANB#) was really ugly, but it seems like #ORLH# and #EANB# look good. The digests are frozen in <B>#1XZ1#</B> and should be put on a gel again</LI></UL><H2><SPAN class="mw-headline" id="15-05-11">15-05-11</SPAN></H2><UL><LI> put the pCas digests (#1XZ1#) on the gel for a second time - the 1 % gel in the bio6 was really ugly (100 V, 35 min)</LI><LI> iLM and iSL ran another gel in the bio7 - 1.2 %, 100 V, 43 min and the gel looked much better:</LI></UL><P>(#EANB# and #ORLH# are okay, #PFOK# is bad)
</P><UL><LI><A href="/wiki/index.php?title=Manual/Transformation&amp;action=edit&amp;redlink=1" class="new" title="Manual/Transformation (page does not exist)">electroporation</A> of 2 µl pTargetF into 20 µl NEB10b - plate on LB+S</LI></UL><H2><SPAN class="mw-headline" id="15-05-12">15-05-12</SPAN></H2><UL><LI> spectinomycin is not working -&gt; prepare plates with higher concentrations tomorrow</LI><LI> plated <I>E.coli</I> DH5α from cryo stock onto LB<SUP>-</SUP> an incubate at 37 °C</LI></UL><H2><SPAN class="mw-headline" id="15-05-13">15-05-13</SPAN></H2><UL><LI> making plates with our spectinomycin 1000x stock as well as the 1000x stock from the bio7 (to compare both spectomycin 'sources')</LI><LI> plate DH5α on the new LB+S plates to check wether they work now</LI><LI> designed 32 sequencing primers for pCas </LI><LI> gradient PCRs for the amplification of pTargetF backbone using the primers #Z9T8# + #BBDC# (Tm 63.2/59.1) and the template #VMLL# </LI><LI> products stored in #L4X3#</LI><LI> expected length: 1950 bp (ladder 1 kb ruler, gel looks very good!)</LI></UL><H2><SPAN class="mw-headline" id="15-05-14">15-05-14</SPAN></H2><UL><LI> the LB+S plates from our own spectinomycin stock inhibited DH5α growth, but did not stop it. The spectinomycin stock from the bio7 was better, but still not 100 %.</LI><LI> new LB+S plates were prepared from both S-stocks but this time with twice the concentrations (LB+S new/old x2)</LI><LI> DH5α plated on the both LB+S x2 plates</LI><LI> Prep of the pTargetF gradient PCR backbone product (#L4X3#) into #CFWR# and #LLFO#</LI><LI><A href="/wiki/index.php?title=Manual/genomic_PCR&amp;action=edit&amp;redlink=1" class="new" title="Manual/genomic PCR (page does not exist)">colony-gradient PCRs</A> colony-gradient PCRs for the amplification of homology arms</LI></UL><P>conditions for 25 cycles:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'10&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 60-71 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> glgX left homology arm using #BQBK# + #HPO4# (Tm 65.5/65.5) - products stored in #P11R# - not a single band on the gel </LI><LI> glgX right homology arm using #SOZQ# + #K4TH# (Tm 66.1/66.0) - products stored in #EBMW# - not a single band on the gel either =(</LI><LI> discarded the PCR products</LI><LI><B>possible problems</B>: elongation time too short? Annealing time too short? more likely: no accessible template DNA (because we didn't have any unspecific bands either) - sequence mismatch between DH5α and K-12 is unlikely, because <A rel="nofollow" class="external text" href="http://en.wikipedia.org/wiki/Escherichia_coli_(molecular_biology)#K-12">DH5α is a derivative of K-12</A>.</LI></UL><H2><SPAN class="mw-headline" id="15-05-15">15-05-15</SPAN></H2><UL><LI> DH5α survived the double concentration on the LB+S x2 plates. A bit less than 1x but still there's growth. We're tired of making even higher concentrations. Instead we will make control transformations in parallel when transforming pTargetF. For the pTargetT constructs we decided to build them in a pSB1C3 backbone instead. (the ori-targeting works there too!)</LI></UL><P>modifying the <A href="/wiki/index.php?title=PCR_High_Fidelity&amp;action=edit&amp;redlink=1" class="new" title="PCR High Fidelity (page does not exist)">genomic PCRs</A> manual, because it was unsuccessful:
</P><OL><LI> centrifuge cells for 1 min at 11 000 </LI><LI> resuspend cells in 50 µl of water</LI><LI> centrifuge 1'00&quot; at full speed</LI><LI> measure the DNA concentration (135.8 ng/µl)</LI><LI> resuspend cells</LI><LI> 98 °C for 5'00&quot;</LI><LI> centrifuge 1'00&quot; at full speed</LI><LI> measure the DNA concentration (3246.8 ng/µl)</LI></OL><P>conditions for 30 cycles:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 55-71 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'20&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><P>Template: #ZDPA# (genomic DNA) for glgX left homology arm using #BQBK# + #HPO4# (Tm 65.5/65.5) -&gt; products stored in #BE1R# -&gt; the gel shows one band of the correct length in the 57.6 °C annealing temperature tube
</P><H2><SPAN class="mw-headline" id="15-05-18">15-05-18</SPAN></H2><UL><LI> Primer #KQCX# was not in the shipment last week. Placing a new order at Eurofins</LI></UL><UL><LI> with the results from Friday, do <A href="/wiki/index.php?title=Manual/genomic_PCR&amp;action=edit&amp;redlink=1" class="new" title="Manual/genomic PCR (page does not exist)">colony-gradient PCRs</A> for the amplification of homology arms
</LI><LI> glgX left homology arm using #BQBK# + #HPO4# (Tm 65.5/65.5) 
</LI><LI> PfuS PCR with 5 samples (gradient 56.6-58.6) </LI><LI> 2x template concentration #ZDPA# (10 µL instead of 5 µL)</LI><LI> Products in #BVNW#</LI></UL><LI> glgX right homology arm using #SOZQ# + #K4TH# (Tm 66.1/66.0) 
</LI><LI> PfuS PCR with 10 samples (gradient 55-71)</LI><LI> 2x template concentration #ZDPA#</LI><LI> products in #FQCM#</LI><H2><SPAN class="mw-headline" id="15-05-19">15-05-19</SPAN></H2><UL><LI> annotated lots of gel pictures</LI><LI> PCR-prep the glgX and glgP (left) homology arms </LI><LI> PCR to amplify the left glgP homology arm using #WQRS# + #PF1Y# (Tm 66.2/66.9)
</LI><LI> PfuS PCR with 9 samples (gradient 55-71)</LI><LI> 2x template concentration #ZDPA#</LI><LI> products in #KE38#</LI><LI> strong band at 56.1 and medium at 63.8 °C</LI></UL><P>PCR product purification
</P><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> from </TH><TH> tubes </TH><TH> prepped into
</TH></TR><TR><TD> glgX left homology arm </TD><TD> #BVNW# </TD><TD> 2 </TD><TD> #11BX#
</TD></TR><TR><TD> glgX right homology arm </TD><TD> #FQCM# </TD><TD> 4+5+6 </TD><TD> #WXRD#
</TD></TR><TR><TD> glgP left homology arm </TD><TD> #KE38# </TD><TD> 3+6 </TD><TD> #EEO6#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-20">15-05-20</SPAN></H2><UL><LI> measured concentrations of yesterdays PCR products</LI><LI> still waiting for the primers</LI></UL><H2><SPAN class="mw-headline" id="15-05-21">15-05-21</SPAN></H2><UL><LI> still waiting for primers</LI></UL><H2><SPAN class="mw-headline" id="15-05-22">15-05-22</SPAN></H2><UL><LI> genomic PCR to amplify the only remaining homology arm
</LI><LI> glgP using #KQCX# + #COT4# (Tm 65.3/65.3)</LI></UL><LI> gradient PCR to amplify the pSB1A3 backbone</LI><LI> PCR-prep the backbone and homology arms</LI><LI> elute the sgRNA gBlocks according to the protocol (20 µl water, 50 °C for 20')</LI><H2><SPAN class="mw-headline" id="15-05-26">15-05-26</SPAN></H2><P>genomic PCR to amplify the only remaining homology arm
</P><UL><LI> glgP using #KQCX# + #COT4# (Tm 65.3/65.3)</LI><LI> PfuS polymerase</LI><LI> 10x 25 µl reactions</LI><LI> double template concentration (#ZDPA#)</LI><LI> products in #HFLQ#</LI></UL><P>conditions for <B>25 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 55-71 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><P>gradient PCR to amplify the pSB1A3 backbone
</P><UL><LI> diluted #93QQ# template </LI><LI> primers #WVZQ# and #8E3Q#</LI><LI> PfuS polymerase</LI><LI> 5x 50 µl reactions</LI><LI> products in #HODO#</LI></UL><P>conditions for <B>25 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 62-70 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'15&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><UL><LI> PCR-prep the backbone and homology arms</LI><LI> elute the sgRNA gBlocks according to the protocol (20 µl water, 50 °C for 20')</LI></UL><H2><SPAN class="mw-headline" id="15-05-27">15-05-27</SPAN></H2><UL><LI> repeat both PCRs with narrower gradients</LI></UL><P>gradient PCR to amplify the pSB1A3 backbone
</P><UL><LI> diluted #93QQ# template </LI><LI> primers #WVZQ# and #8E3Q#</LI><LI> PfuS polymerase</LI><LI> 5x 50 µl reactions</LI><LI> products in #PF8P#</LI></UL><P>conditions for <B>25 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 67.2-69.2 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'30&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><P>genomic PCR to amplify the only remaining homology arm
</P><UL><LI> glgP using #KQCX# + #COT4# (Tm 65.3/65.3)</LI><LI> PfuS polymerase</LI><LI> 10x 25 µl reactions</LI><LI> double template concentration (#ZDPA#)</LI><LI> products in #4ADM#</LI></UL><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> gradient annealing </TD><TD> 60-66 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'20&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><P>PCR product purification (from #PF8P#)
</P><TABLE class="wikitable"><TBODY><TR><TH> fragment </TH><TH> from </TH><TH> tubes </TH><TH> prepped into
</TH></TR><TR><TD> glgP right homology arm </TD><TD> #4ADM# </TD><TD> 3+6+8+9 </TD><TD> #MWH3#
</TD></TR><TR><TD> pSB1A3 backbone </TD><TD> #PF8P# </TD><TD> 1+2</TD><TD> #RW11#
</TD></TR><TR><TD>  pSB1A3 backbone </TD><TD> #PF8P# </TD><TD> 3+4 </TD><TD> #TRTN#
</TD></TR></TBODY></TABLE><UL><LI><S>make a pCas preculture from the plate</S> other stuff is higher priority - we'll do this next week</LI><LI><S>prepare buffers for electrocompetent cells</S></LI></UL><H2><SPAN class="mw-headline" id="15-05-28">15-05-28</SPAN></H2><UL><LI> Diluted the sgRNA gBlocks in 20 µl AE buffer (#RFZK#, #6XLF#, #ZVAC#).</LI></UL><P>CPEC for anti glgX/P targeting constructs in pSB1A3
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> µl for anti glgX </TH><TH> µl for anti glgP
</TH></TR><TR><TD> water </TD><TD> 11 </TD><TD> 9.8
</TD></TR><TR><TD> Q5 buffer </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> DMSO </TD><TD> 0.75 </TD><TD> 0.75
</TD></TR><TR><TD> dNTPs (2 mM) </TD><TD> 5 </TD><TD> 5
</TD></TR><TR><TD> Q5 polymerase </TD><TD> 0.5 </TD><TD> 0.5
</TD></TR><TR><TD> pSB1A3 backbone fragment (#TRTN#) </TD><TD> 1.2 </TD><TD> 1.2
</TD></TR><TR><TD> sgRNA 1 </TD><TD> 0.9 #RFZK# </TD><TD> 0.9 #6XLF#
</TD></TR><TR><TD> sgRNA 2 </TD><TD> none </TD><TD> 1.1 #ZVAC#
</TD></TR><TR><TD> left homology arm </TD><TD> 0.3 #11BX# </TD><TD> 0.5 #EEO6#
</TD></TR><TR><TD> right homology arm </TD><TD> 0.3 #WXRD# </TD><TD> 0.2 #MWH3#
</TD></TR><TR><TD> elongation time </TD><TD> 1'36&quot; (1'46&quot;) </TD><TD> 1'42&quot; (1'46&quot;)
</TD></TR><TR><TD> products stored in </TD><TD> #PXR4# </TD><TD> #PXR4#
</TD></TR></TBODY></TABLE><UL><LI> Heat shock transformation of CPEC product (#PXR4#:A), ligation product (#QZFH#) into DH5α cells
</LI><LI> trafo program in the cycler: 4 °C for 30', 42 °C for 60&quot;, 4 °C for 5'</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> number </TH><TH> description </TH><TH> id
</TH></TR><TR><TD> 5 </TD><TD> anti glgX in pSB1A3 CPEC </TD><TD> #PXR4#:X
</TD></TR><TR><TD> 6 </TD><TD> anti glgP in pSB1A3 CPEC </TD><TD> #PXR4#:P
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-29">15-05-29</SPAN></H2><P>Working with new clones
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> colonies </TH><TH> red [%]
</TH></TR><TR><TD> anti glgX </TD><TD> 1448 </TD><TD> 50 %
</TD></TR><TR><TD> anti glgP </TD><TD> 1240 </TD><TD> 92 %
</TD></TR></TBODY></TABLE><UL><LI> master plates of anti glgX/P CPEC clones</LI></UL><H2><SPAN class="mw-headline" id="15-05-30">15-05-30</SPAN></H2><UL><LI> colony PCR</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> template </TH><TH> clone numbers </TH><TH> expected </TH><TH> good clones
</TH></TR><TR><TD> anti glgX </TD><TD> 1-12 </TD><TD> 1351 </TD><TD> 10, 11, 2, 3, 4
</TD></TR><TR><TD> anti glgP </TD><TD> 1-10 </TD><TD> 1574 </TD><TD> 1, 4, 5, 6, 7, 8
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-05-31">15-05-31</SPAN></H2><UL><LI> overnight cultures</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> clone number </TH><TH> cryo-id </TH><TH> plasmid-id </TH><TH> sequencing result
</TH></TR><TR><TD> anti glgX </TD><TD> 2 </TD><TD> #RKS1# </TD></TR><TR><TD> anti glgX </TD><TD> 3 </TD><TD> #1NT8# </TD><TD> #3OSZ# </TD><TD> a copy of the right end was inserted before the terminator
</TD></TR><TR><TD> anti glgX </TD><TD> 4 </TD><TD> #BAB3# </TD><TD> #1DEN# </TD><TD> perfect! not a single mutation
</TD></TR><TR><TD> anti glgX </TD><TD> 10 </TD><TD> #DNMN# </TD><TD> #MANY# </TD><TD> two point mutations in the N20 sequence
</TD></TR><TR><TD> anti glgX </TD><TD> 11 </TD><TD> #WCNN# </TD></TR><TR><TD> anti glgP </TD><TD> 1 </TD><TD> #XQM8# </TD></TR><TR><TD> anti glgP </TD><TD> 4 </TD><TD> #M69X# </TD><TD> #C1QW# </TD><TD> sequencing was difficult, looks good
</TD></TR><TR><TD> anti glgP </TD><TD> 5 </TD><TD> #SXAX# </TD><TD> #DC6X# </TD><TD> sequencing was difficult, point mutation at pos. 499
</TD></TR><TR><TD> anti glgP </TD><TD> 6 </TD><TD> #D6CE# </TD><TD> #HCTL# </TD><TD> sequencing was difficult, looks good
</TD></TR><TR><TD> anti glgP </TD><TD> 7 </TD><TD> #R4LZ# </TD><TD> #4MCW# </TD><TD> sequencing was difficult, looks good
</TD></TR><TR><TD> anti glgP </TD><TD> 8 </TD><TD> #CS39# </TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-01">15-06-01</SPAN></H2><UL><LI> plasmid preps of anti glgX and anti glgP plasmids (see the table above)</LI><LI> pipeted the plasmids for sequencing with the priemrs #A9W9#, #XE3D#, #WQRS#</LI></UL><H2><SPAN class="mw-headline" id="15-06-05">15-06-05</SPAN></H2><UL><LI> pipetting anti glgP for sequencing with 225 ng/kb plasmid template and 2 µl of 20 µl #PF1Y# primer</LI></UL><H2><SPAN class="mw-headline" id="15-06-10">15-06-10</SPAN></H2><UL><LI> analysis of sequencing results: see table above!</LI></UL><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> resistance </TH><TH> plasmid ID </TH><TH> cryo ID
</TH></TR><TR><TD> pCas </TD><TD> Kan </TD><TD> #EANB# </TD><TD> #YPKO#
</TD></TR><TR><TD> anti glgX in pSB1A3 </TD><TD> Amp </TD><TD> #1DEN# </TD><TD> #BAB3#
</TD></TR><TR><TD> anti glgP in pSB1A3 </TD><TD> Amp </TD><TD> #C1QW# </TD><TD> #M69X#
</TD></TR></TBODY></TABLE><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>Plasmid map of glgX-targeting plasmid</B>A sgRNA with the N20-sequence 5'-GTAATGCGCGCCGAGGGGAG targeting a protospacer in the beginning of the glgX CDS is expressed behind a J23119 strong constitutive promoter. Next to the ampR gene and the pMB1 ori, the plasmid carries a 818 bp repair template consisting of an iGEM Aachen Multi Stop sequence flanked by two 400 bp homology regions.
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>