<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><P>The most important recipe for effective research and reproducible results are reliable protocols.
</P><P>On this page we are providing several protocols that we have used over the course of our project.
</P><DIV class="col-md-12"><H1><SPAN class="mw-headline" id="Culturing">Culturing</SPAN></H1></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#Lysogeny-Broth_(LB_Medium)"><H4>Lysogeny-Broth (LB Medium)<SPAN class="Lysogeny-Broth_(LB_Medium)" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Lysogeny-Broth_(LB_Medium)" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN><OL><LI>weight components
</LI><LI><B>10 g/L NaCl</B></LI><LI><B>10 g/L Trypton/Pepton</B></LI><LI><B>5 g/L Yeast extract</B></LI><LI><B>(15 g/L Agar for plates)</B></LI></OL><LI>  fill to 1 L with VE/deionized water</LI><LI>  mix well by shaking</LI><OL><LI>  for plates, let it cool to <B>60 °C</B></LI><LI>  add antibiotics and mix thoroughly</LI><LI>  pour plates and pile
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#M9_Medium"><H4>M9 Medium<SPAN class="M9_Medium" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual M9_Medium" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>M9 is a minimal Medium with a low autofluorescence. It is suited for standardized reproducible culturing and allows to investigate the effect of additives.
<P><B>Endconcentrations</B></P><UL><LI>MgSO<SUB>4</SUB> (1 mM)</LI><LI>Traceelement Solution</LI><LI>Glucose (80 mM)</LI><LI>Thiamine (0.3 µM)</LI><LI>antibiotics from 1000x stock</LI><LI>10x Salt solution </LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Ingredient</TH><TH>Concetration</TH></TR><TR><TD>KH2PO4</TD><TD>227 mM </TD></TR><TR><TD>Na2HPO4</TD><TD>472 mM </TD></TR><TR><TD>NaCl</TD><TD>85 mM </TD></TR><TR><TD>NH4Cl (standard/low)</TD><TD>187 mM / 22.4 mM </TD></TR></TBODY></TABLE><P>
fill up to <B>1 L</B> with deionized water and autoclave
</P><UL><LI>1000x Trace elements</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Component</TH><TH>Final Concetration</TH><TH>Concetration in stock solution</TH></TR><TR><TD>Iron(III) chloride</TD><TD>50 mM</TD><TD>13,515 mg/L</TD></TR><TR><TD>Calcium chloride</TD><TD>20 mM </TD><TD>2,220 mg/L</TD></TR><TR><TD>Manganese(II) chloride</TD><TD>10 mM </TD><TD>1,258 mg/L</TD></TR><TR><TD>Zinc sulfate</TD><TD>10 mM </TD><TD>1,615 mg/L </TD></TR><TR><TD>Cobalt(II) chloride</TD><TD>2 mM </TD><TD>260 mg/L</TD></TR><TR><TD>Copper (II) chloride</TD><TD>2 mM </TD><TD>269 mg/L </TD></TR><TR><TD>Nickel(II) chloride</TD><TD>2 mM </TD><TD>259 mg/L </TD></TR><TR><TD>Sodium molybdate</TD><TD>2 mM </TD><TD>412 mg/L </TD></TR><TR><TD>Sodium selenite</TD><TD>2 mM </TD><TD>346 mg/L </TD></TR><TR><TD>Boric acid</TD><TD>2 mM</TD><TD>124 mg/L </TD></TR><TR><TD>Hydrochloric acid</TD><TD>1 mM</TD><TD>20 ml</TD></TR></TBODY></TABLE><P>
fill up to <B>1 L</B> with deionized water and autoclave
</P><P>
All Solutions were prepared and autoclaved separately. The M9 Medium was then mixed under sterile conditions.
</P></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#Overnight_Cultures"><H4>Overnight Cultures<SPAN class="Overnight_Cultures" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Overnight_Cultures" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN># prepare sterile glass tubes of 5 ml of LB+antibiotics
<OL><LI> use a sterile pipet tip to pick bacteria from plates</LI><LI> throw the tip into the tubes</LI><LI> incubate in a rotary shaker
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#SOC_Medium"><H4>SOC Medium<SPAN class="SOC_Medium" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual SOC_Medium" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>SOC is a complex medium especially suited to increase transformation efficiency in <I>Escherichia coli</I> cells.
<OL><LI> components
<DL><DD><B>0.5 % (w/v) yeast extract</B></DD><DD><B>2 % (w/v) tryptone</B></DD><DD><B>10 mM NaCl</B></DD><DD><B>2.5 mM KCl</B></DD><DD><B>20 mM MgSO<SUB>4</SUB></B></DD></DL></LI><LI> fill up with deionized water</LI><LI> adjust to pH 7.5 with NaOH</LI><LI> after autoclaving, add sterile glucose solution to a final concentration of 20 mM
</LI></OL><DIV class="col-md-12"><H1><SPAN class="mw-headline" id="Cloning">Cloning</SPAN></H1></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#B0034_Insertion-Mutagenesis"><H4>B0034_Insertion-Mutagenesis<SPAN class="B0034_Insertion-Mutagenesis" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual B0034_Insertion-Mutagenesis" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>With this method B0034 can be inserted into any RFC10 part that begins with prefix-CDS.
<P>The protocol is taken from Bryksin et al. 2014<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>.
</P><P><B>Requirements</B></P><UL><LI> 30 ng of purified template BioBrick plasmid</LI><LI> chemically competent cells</LI><LI> #CEXL#</LI><LI> #CNNY#</LI><LI> DpnI</LI></UL><P><B>Procedure</B></P><OL><LI> pipet a Phusion polymerase PCR of 50 µl total volume
</LI><LI> water</LI><LI> HF-buffer</LI><LI> 30 ng plasmid template</LI><LI> 30 pmol #CEXL#</LI><LI> 30 pmol #CNNY#</LI><LI> dNTPs</LI><LI> Phusion polymerase</LI></OL><LI> PCR
</LI><LI> 98 °C for 30'</LI><LI> 55 °C for 30'</LI><LI> appropriate extension temperature and time</LI><LI> 18 cycles</LI><LI> DpnI digestion
</LI><LI> add 10 U of DpnI to the PCR reaction mixture</LI><LI> incubate for 60' at 37 °C</LI><LI> Transformation
</LI><LI> transform 1 µl of digestion product into chemically competent cells (might require more for less competent cells)
</LI></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#CPEC"><H4>CPEC<SPAN class="CPEC" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual CPEC" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>The CPEC (circular polymerase extension cloning) method <SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> was used by iGEM Aachen 2015 to assemble e.g. the BioBricks of <A href="/Team:Aachen/Lab/Glycogen/Synthesis" title="Team:Aachen/Lab/Glycogen/Synthesis"><I>glgB</I></A> and <A href="/Team:Aachen/Lab/Methanol/Biobricks" title="Team:Aachen/Lab/Methanol/Biobricks"><I>xpk</I></A>.
A big benefit of this approach is that it needs just one enzyme and is scarless. <SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP><OL><LI> Linearize the destination vector by amplification with suitable primers. Insert(s) must be amplificated with primers that contain overhangs to create overlapping regions with the backbone (and additional parts).</LI><LI> Mix 100 ng of each fragment together with calculated amounts of H<SUB>2</SUB>O, DMSO, dNTPs and DNA polymerase</LI><LI> Repeat 15 cycles of denaturation (98 °C), annealing and elongation in a thermocycler</LI><LI> Transform the construct in a strain of your choice</LI></OL><P>
The overlapping regions have to be designed carefully to avoid secondary structures and to adjust the annealing temperature. 
</P><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>steps of CPEC assembly</B>after backbone linearisation with appropriate primers, the target parts can be inserted.
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#Gibson_Assembly"><H4>Gibson Assembly<SPAN class="Gibson_Assembly" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Gibson_Assembly" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>With the Gibson assembly method fast assembly of multiple DNA fragments is possible under isothermal conditions, regardless of fragment length. Therefore fragments with overlapping ends of 15-30 bp are needed and can be created via PCR. The Gibson Assembly was performed using the Gibson Assembly® Master Mix according to the Protocol by New England Biolabs <SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP>, unless stated otherwise. To design the primers for the respective PCRs the Software Geneious<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP> was used.
<UL><LI>Set up the reaction according to the table below on ice (2-3 fragment assembly).</LI><LI>Incubate samples in a thermocycler at 50 °C for 15 minutes when 2 or 3 fragments are being assembled or 60 minutes when 4-6 fragments are being assembled. Following incubation, store samples on ice or at –20 °C for subsequent transformation.</LI><LI>Transform NEB 5-alpha Competent E. coli cells with 2 μl of the assembly reaction, following the transformation protocol. </LI></UL><TABLE class="wikitable"><TBODY><TR><TD>Total Amount of Fragments</TD><TD>0.02-0.5 pmols</TD></TR><TR><TD>Gibson Assembly Master Mix (2X)</TD><TD>10 µl</TD></TR><TR><TD>Deionized H<SUB>2</SUB>O</TD><TD>10-X µl</TD></TR><TR><TD>Total Volume</TD><TD>20 µl</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#{{{id}}}"><H4>Genomic Amplification<SPAN class="{{{id}}}" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual {{{id}}}" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN># prepare culture
<OL><LI> collect cells from the agar plate and resuspend them in <B>50 µl of ddH<SUB>2</SUB>O</B></LI><LI> centrifuge <B>1'00&quot; at full speed</B></LI><LI> measure the DNA concentration</LI><LI> resuspend cells</LI><LI><B>98 °C</B> for <B>5'00&quot;</B></LI><LI> centrifuge 1'00&quot; at full speed</LI><LI> measure the DNA concentration</LI><LI> dilute DNA to <B>500 ng/µl</B></LI><LI> (use 1 µg DNA template per 50 µl PCR reaction volume)</LI><LI> use <B>3'00&quot; denaturing time</B> at the beginning of the PCR program
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#{{{id}}}"><H4>Transformation<SPAN class="{{{id}}}" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual {{{id}}}" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN><B>Heat Shock</B><OL><LI> thaw cells on ice</LI><LI> pipet 50 µl of cells into a pre-chilled PCR tube</LI><LI> pipet DNA:
</LI><LI> 5 µl of Ligation/CPEC product</LI><LI> 1 µl of kit plate concentrate</LI><LI> 1 µl of purified plasmid</LI></OL><LI> use the thermocycler for the following program:</LI><TABLE class="wikitable"><TBODY><TR><TH>task</TH><TH>time</TH><TH>temperature</TH></TR><TR><TD>incubate</TD><TD>30'00</TD><TD>4 </TD></TR><TR><TD>shock</TD><TD>1'00</TD><TD>42</TD></TR><TR><TD>incubate</TD><TD>5'00</TD><TD>4</TD></TR></TBODY></TABLE><OL><LI> pipet all cells into a 1.5 ml tube with 200 µl of pre-warmed SOC medium</LI><LI> incubate at 37 °C for 60'00&quot;</LI><LI> plate all 251-255 µl onto LB+antibiotic plates</LI></OL><P>Reference: iGEM Transformation Protocol<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP></P><P><B>Electroporation</B></P><OL><LI>add electrocompetent to DNA on ice</LI><LI>move the mixture to the cuvette</LI><LI>dry and shock the cells</LI><LI>add 1ml of SOC medium</LI><LI>incubate at 37 °C for 1-2 h
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#Plasmid_Preparation"><H4>Plasmid Preparation<SPAN class="Plasmid_Preparation" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Plasmid_Preparation" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>Plasmids were prepared as explained in the following manuals
<UL><LI><A rel="nofollow" class="external text" href="https://www.qiagen.com/de/resources/resourcedetail?id=022dbe29-4066-4886-8f99-09852b0050e1&amp;lang=en">QuickLyse Miniprep Handbook</A></LI><LI><A rel="nofollow" class="external text" href="https://www.gelifesciences.com/gehcls_images/GELS/Related%20Content/Files/1314774443672/litdoc28951561AF_20110831095457.pdf">GE Healthcare illustra plasmidPrep Mini Spin Kit</A></LI></UL><DIV class="col-md-12 protocol"><A class="protocol" href="#RDP_Assembly"><H4>RDP Assembly<SPAN class="RDP_Assembly" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual RDP_Assembly" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>This year our iGEM team tested and applied the RDP assembly method by Synbiota. We used it to assemble the <A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid" title="Team:Aachen/Lab/Methanol/Polycistronic Expression Plasmid">polycistronic methanol conversion plasmid</A> and to design a <A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library" title="Team:Aachen/Lab/Methanol/Monocistronic Diversity Library">monocistronic diversity library</A> to screen for a more efficient methanol uptake plasmid. 
<P><B>RDP parts</B></P><P>RDP parts can be assembled successively to build new devices or whole new circuits with unique functions. The sequence of every RDP part begins and ends with a 4 bp long 5' overhang.
</P><P>
There are four differnet kinds overhangs:
</P><P>
X: 5'- GATG -3'
</P><P>X': 5'- CTAC -3'
</P><P>Z: 5'- CGGC -3'
</P><P>Z': 5'- GCCG -3'
</P><P>
As one can easily notice, X and X' as well as Z and Z' are homologous.
A RDP part can have one of two possible orientations:
</P><P>
X - Z' or Z - X'
</P><P>
Usually, you can convert a certain sequence into a RDP part by PCR amplification. You have to use forward and reverse primers with extension containing BsaI restriction sites prior to the overhang sequence you want to introduce.
</P><P>
In the end, the 4 bp overhangs arise from a BsaI digest of the amplification product. 
</P><P>
RDP parts with a length of &lt;100 bp can be created by annealing synthesized oligos (HPLC purification recommended).
</P><P><B>Assembling the parts</B></P><P>But how are RDP parts joined together to form whole new circuits? Synbiota's RDP assembly kit provides so called anchors. Anchors are linear DNA fragments, contain an antibiotics resistance gene and are bound to magnetic beads. At their downstream ends, anchors have either X' or Z' overhangs.
</P><P>
You can add any desired RDP part with the appropriate overhangs to the anchors that were diluted in an eppi. Matching 4 bp overhangs will be joined by T4 ligase in the reaction mix.
</P><P>
By applying a magnet to the eppi, you can collect all magnetic beads with the bound DNA and simply discard the used reaction mix. After washing and diluting the magnetic beads  again, you can add your next RDP part. After assembling the RDP parts, you'll have to add a so called &quot;cap&quot;. It is also provided in Synbiota's assembly kit and contributes an origin of replication.
</P><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>RDP assembly</B><SPAN style="color:transparent">bla</SPAN><OL><LI> You can add your first RDP part to the anchor. T4 ligase in the reaction mix will ligate the homologous overhangs</LI><LI> Before you can add the next RDP part, you have to collect the magnetic beads, discard the used reaction mix and then wash and resuspend the beads again.</LI><LI> In the end, the cap is added to terminate the assembly</LI></OL></TD></TR></TBODY></TABLE></DIV><P>
Finally, anchor and cap can be joined to form a circular plasmid. By choosing  a certain anchor and cap for the assembly you determine the resistance marker and copy number of the resulting plasmid.
</P><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>Circularizing the assembly product</B>After releasing the finished RDP circuit from the magnetic bead, anchor and cap join to form a circular plasmid.
</TD></TR></TBODY></TABLE></DIV><P>
RDP plasmids can again be digested with BsaI and NotI to extract the assembled RDP parts as one composite RDP part.
</P><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>Extracting a composite RDP part</B>To extract the assembled RDP parts as one composite RDP part, the plasmid is digested with BsaI to introduce the 4 bp overhangs and NotI to prevent the interference of the backbone fragment in future assembly steps.
</TD></TR></TBODY></TABLE></DIV><P><B>Experiences</B></P><P>We experienced the RDP assembly method to be really easy to handle. Both the in silico 
work to design new RDP circuits and the efficiency of the method itself convinced us.
Thus, RDP assembly can be a suitable alternative to other assembly methods like Gibson Assembly 
or CPEC.
</P></DIV></DIV><DIV class="col-md-12"><H1><SPAN class="mw-headline" id="Analytics">Analytics</SPAN></H1></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#Acid_Hydrolysis"><H4>Acid Hydrolysis<SPAN class="Acid_Hydrolysis" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Acid_Hydrolysis" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>One way to break down polysaccharides into monomers is acid hydrolysis. This short instruction is based on another protocol <SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP><P><B>Procedure</B></P><OL><LI> Resuspend cell pellet (or dried crudely extracted glycogen) in 500 µL 5 M HCl in an appropriately sized glass vial</LI><LI> Cook sample with tightly closed lid for at least 6 hours at 105 °C</LI><LI> Dry sample until black/brown residue is left. This is done at 85 °C with no lid under a hood with constant airflow!</LI><LI> Store samples at room temperature
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#Cell_Lysis"><H4>Cell Lysis<SPAN class="Cell_Lysis" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Cell_Lysis" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>To measure non secreted protein cell lysis is essential.
<UL><LI> take a 1 ml culture sample into a 2 ml tube and spin down</LI><LI> discard the supernatant and resuspend in 1 ml of 50 mM KPi buffer</LI><LI> add 0.5 g of 0.1 mm glass beads</LI><LI> cool down the sample on ice for 5 minutes</LI><LI> bead mill 3 times for 0'30&quot; and cool the sample on ice inbetween for 1'00&quot;</LI><LI> centrifuge for 2 minutes at 10,000 x g at 4 °C</LI><LI> transfer the supernatant into a new tube</LI><LI> measure the total protein concentration</LI><LI> dilute the sample with 50 mM KPi buffer to a concentration of 6 µg/µl
</LI></UL><DIV class="col-md-12 protocol"><A class="protocol" href="#Electrophoresis"><H4>Electrophoresis<SPAN class="Electrophoresis" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Electrophoresis" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>Agarose-Electrophoresis is used to seperate DNA by the number of base pairs.
<UL><LI>marker was the <A rel="nofollow" class="external text" href="https://tools.thermofisher.com/content/sfs/manuals/MAN0013007_GeneRuler_1kb_DNALadder_RTU_50ug_UG.pdf">GeneRuler 1 kb DNA Ladder</A></LI></UL><UL><LI>fill pockets with 5 µl DNA ladder or 10 µl sample volume with 6x loading dye</LI></UL><UL><LI>running conditions: 110 V for 30-40 minutes
</LI></UL><DIV class="col-md-12 protocol"><A class="protocol" href="#Glycogen_Kit"><H4>Glycogen Kit<SPAN class="Glycogen_Kit" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Glycogen_Kit" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN><B>Glycogen Assay Protocol<SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP></B><P>
1. Sample preparation
</P><UL><LI> Homogenize 10<SUP>6</SUP> (animal) cells with 200 µL dH<SUB>2</SUB>O on ice.
</LI><LI> approximately 10<SUP>9</SUP><I>E.coli</I> cells (1 ml culture with OD 1.25) <SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP><SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP></LI></UL><LI> Sonicate cells</LI><LI> Boil homegenates for 10 minutes.</LI><LI> Spin/centrifuge at 18 000 g for 10 minutes. The supernatant is <B>now ready</B> for assay!</LI><LI> Add 2-50 µL samples to a 96-well pate.</LI><LI> Adjust volume to 50 µL/well with <I>Hydrolysis Buffer</I></LI><P>Notes:
</P><UL><LI> For samples having glucose background, prepare well(s) containing same amont of sample as in the test well as background control.</LI><LI> (Accurate determination of Glycogen in the test samples is best determined via spiking samples with a known amount of standard (example given 0.8 µg)</LI></UL><P>
2. Standard Curve preparation
</P><UL><LI> Dilute the glycogen standard to 0.02 mg/mL by adding 10 µL of the <I>Standard</I> to 990 µL of distilled water, mix well.</LI><LI> Add 0, 2, 4, 8, 10 µL to a series of wells.</LI><LI> Adjust volume to 50 µL/well with <I>Hydrolis Buffer</I> to generate 0, 0.04, 0.08, 0.12, 0.16, and 0.2 µg per well</LI></UL><P>
3. Add 1 µL Hydrolysis Enzyme mix to Standards and samples and mix well.
</P><UL><LI> for dilution samples, dilute with hydrolysis buffer</LI></UL><P>Incubate for 30 minutes at room temperature.
</P><P>Note:
</P><P>If glucose is present in your sample, you may do a glucose control without addition of hydrolysis enzyme to determine the level of glucose background.
</P><P>
4. Reaction Mix
</P><UL><LI> Mix enough reagents for the number of samples and standards to be performed</LI><LI> For each well prepare a total 50 µL <I>Reaction Mix</I>.
</LI><LI> Dev.  Buffer: 48.7 µL</LI><LI> Dev. Enzyme Mix: 1 µL</LI><LI> OxiRed Probe: 0.3 µL</LI></UL><LI> Add 50 µL of the <I>Reaction Mix</I> to each well containing the <I>Glycogen Standard</I> or samples, mix well.</LI><P>
5. Measurement
</P><UL><LI> Incubate the reaction for 30 minutes at room temperature protected from light.
</LI><LI> Fluorescence (Ex/Em = 535/587 nm)</LI></UL><P>
6. Calculation
</P><UL><LI> Correct background by subtracting the 0 <I>Glycogen Standard</I> sample value (see 2.) from all readings.</LI><LI> Apply sample readings to the standard curve to get B µg of glycogen.</LI></UL><P>Sample c_(Glycogen) = B / V * D µg/µL
</P><P>B is the amount of Glycogen from Standard curve (µg)
</P><P>V is the sample volume added into the reaction well (µL)
</P><P>D is the sample dilution factor
</P><UL><LI> For spiked samples, correct for any sample interference by subtracting the sample reading from spiked sample.</LI></UL><P>For Spiked samples:
</P><P>B = [OD_(sample corrected) / OD_(sample+glycogen Standard (corrected)) - OD_(sample corrected)] * Glycogen Spike (µg)
</P><P>
Notes about Glycogen:
</P><UL><LI> Glycogen  molecular size: 60 000 molecules
</LI><LI> Molecular Weight (MW) ~ 10<SUP>6</SUP>-10<SUP>7</SUP> daltons</LI><LI> Glucose MW: 180.16 g/mol</LI></UL><P><B>Glycogen Assay Protocol short version</B></P><P>
1. Sample preparation
</P><UL><LI> Homogenize 10<SUP>6</SUP> cells with 200 µL dH<SUB>2</SUB>O on ice.</LI><LI> Boil homegenates for 10 minutes.</LI><LI> Spin/centrifuge at 18 000 g for 10 minutes. The supernatant is <B>now ready</B> for assay!</LI><LI> Add 2-50 µL samples to a 96-well pate.</LI><LI> Adjust volume to 50 µL/well with <I>Hydrolysis Buffer</I></LI></UL><P>
2. Standard Curve preparation
</P><UL><LI> Dilute the glycogen standard to 0.02 mg/mL by adding 10 µL of the <I>Standard</I> to 990 µL of distilled water, mix well.</LI><LI> Add 0, 2, 4, 8, 10 µL to a series of wells.</LI><LI> Adjust volume to 50 µL/well with <I>Hydrolis Buffer</I> to generate 0, 0.04, 0.08, 0.12, 0.16, and 0.2 µg per well</LI></UL><P>
3. Add 1 µL Hydrolysis Enzyme mix to Standards and samples and mix well.
</P><UL><LI> Incubate for 30 minutes at room temperature.</LI></UL><P>
4. Reaction Mix
</P><UL><LI> Mix enough reagents for the number of samples and standards to be performed</LI><LI> For each well prepare a total 50 µL <I>Reaction Mix</I>.
</LI><LI> Dev.  Buffer: 48.7 µL</LI><LI> Dev. Enzyme Mix: 1 µL</LI><LI> OxiRed Probe: 0.3 µL</LI></UL><LI> Add 50 µL of the <I>Reaction Mix</I> to each well containing the <I>Glycogen Standard</I> or samples, mix well.</LI><P>5. Measurement
</P><UL><LI> Incubate the reaction for 30 minutes at room temperature protected from light.
</LI><LI> Fluorescence (Ex/Em = 535/587 nm)</LI></UL><P><B>Pre-extraction of glycogen</B></P><UL><LI> Another way to subtract the glucose background, is the pre-extraction of glycogen</LI><LI> This procedure removes the majority of glucose background with minimal effect on glycogen</LI></UL><OL><LI> Take tissue or cells to a final content of 30-50 % in 30 % KOH</LI><LI> Heat to 100 °C for 2 h</LI><LI> Cool and add 2 volumes of 95% ethanol to precipitate crude glycogen</LI><LI> Centrifuge (16000 g for 10 min) and collect the precipitate</LI><LI> Suspend the pellet in a minimal amount of distilled water (25 ml original cell culture -&gt; 5 ml were enough)</LI><LI> Acidify sample to pH 3 with 5 M HCl</LI><LI> Add 1 volume of ethanol to reprecipitate</LI><LI> Repeat steps 4-7 two more times, then let sample dry over night (pre-weigh container for drying)</LI><LI> The dried material can be weighed and dissolved for analysis
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#Iodine_Staining"><H4>Iodine Staining<SPAN class="Iodine_Staining" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Iodine_Staining" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>Iodine is capable of penetrating the cells and interacting with long polysaccharides as starch or glycogen. Therefore, it is a suitable chemical for qualitative and semi-quantitative glycogen measurement.
<P><B>Procedure</B></P><UL><LI> prepare overnight cultures of strains (additional <B>20 mM</B> glucose also possible)</LI><LI> to compare tubes <B>with each other</B>:
</LI><LI> adjust all cultures to the lowest OD</LI><LI> centrifuge 2 ml and discard supernatant</LI><LI> resuspend pellet in <B>200 µl iodine solution</B></LI></UL><UL><LI> to do a glycogen standard for <B>semi-quantitative analysis</B>:
</LI><LI> prepare glycogen solutions, each 50 µl, with concentrations <B>from 8 g/L down to 0.125 g/l</B></LI><LI> add 300 µl iodine solution (recipe see below)</LI><LI> transfer mixture to a well plate (triplicates, 100 µl in each well)</LI><LI> scan the well plate</LI></UL><UL><LI> to compare with <B>glycogen standard</B>:
</LI><LI> adjust culture to the <B>OD 1</B> and transfer 1 ml into tubes</LI><LI> centrifuge and discard supernatant</LI><LI> resuspend pellets in 50 µl water, then add <B>300 µl staining solution</B></LI><LI> transfer 100 µl triplicates of mixture to a well plate</LI><LI> to compare with glycogen standard, scan the well plate and analyse the picture</LI></UL><P>Iodine staining solution (modified version of <B>Lugol's Iodine</B>)
</P><TABLE class="wikitable"><TBODY><TR><TH>Ingredient</TH><TH>Concentration </TH></TR><TR><TD>I2</TD><TD>5 mM</TD></TR><TR><TD>KI</TD><TD>5 mM</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#Nash_Assay"><H4>Nash Assay<SPAN class="Nash_Assay" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Nash_Assay" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN><H3><SPAN class="mw-headline" id="Principle_of_detection">Principle of detection</SPAN></H3><P>The colorimetric and fluorometric assay first described by T. Nash in 1953<SUP id="cite_ref-11" class="reference"><A href="#cite_note-11">[11]</A></SUP> detects the presence of formaldehyde. Two molecules of acetylacetone react with formaldehyde and ammonia to form the strong yellow and fluorescing diacetyl-dihydro lutidine which can be detected via absorption at a wavelength of 412 nm and via fluorescence at an excitation wavelength of 410 nm and an emission wavelength of 510 nm.
</P><H3><SPAN class="mw-headline" id="Components_.5B12.5D">Components <SUP id="cite_ref-12" class="reference"><A href="#cite_note-12">[12]</A></SUP></SPAN></H3><P><B>Nash reagent (components for 1L)</B></P><UL><LI> 2 ml acetylaceton</LI><LI> 300 g ammonium acetate</LI><LI> 6 ml acetic acid</LI><LI> fill up to 1 L with deionized water</LI></UL><P>A few ml for some samples should be enough depending on the scale of measurement (MTP/cuvette).
</P><P><B>Reaction Buffer</B></P><UL><LI> 50 mM KPi Buffer
</LI><LI> 40 mM K<SUB>2</SUB>HPO<SUB>4</SUB> - 6.97 g/L</LI><LI> 10 mM KH<SUB>2</SUB>PO<SUB>4</SUB> - 1.36 g/L</LI></UL><LI> 5 mM MgSO<SUB>4</SUB></LI><LI> 500 µM NAD<SUP>+</SUP></LI><LI> 1 M methanol</LI><H3><SPAN class="mw-headline" id="Procedure">Procedure</SPAN></H3><P><B>Sample</B></P><UL><LI> prepare solutions of whole cell samples as well as supernatant and fragments of lysed cells</LI><LI> adjust to same OD/same protein concentration for comparability</LI></UL><P><B>Protocol</B></P><UL><LI> mix Nash reagent : reaction buffer : sample in the ratio of 20 : 19 : 1 in a MTP/cuvette</LI><LI> Measurement
</LI><LI> measure absorbance at 412 nm</LI><LI> measure fluorescence with an excitation wavelength of 410 nm and an emission wavelength of 510 nm</LI><LI> measure hourly starting directly after mixing for at least 5 hours</LI></UL><LI> incubate at 37 °C at 100 rpm</LI><P><B>Keep your samples closed at all times!</B></P></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#SDS-PAGE"><H4>SDS-PAGE<SPAN class="SDS-PAGE" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual SDS-PAGE" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN>In the SDS-PAGE proteins get denatured and their electrical charge gets masked by the anionic detergent sodium dodecyl sulfate. As a result the proteins can be separated by size through a polacrylamid gel and an electrical current.
<P><B>Culture</B></P><OL><LI> inoculate overnight culture</LI><LI> inoculate main culture (50 ml) with a start OD of 0.15</LI><LI> induce with IPTG at OD 0.6</LI><LI> Take samples (2 ml)
</LI><LI> sample: 5-6 hours after induction</LI><LI> sample: ca 18 hours after induction</LI></OL><LI> centrifuge and freeze pellet</LI><P><B>Gel</B></P><TABLE class="wikitable"><TBODY><TR><TH>Ingredient</TH><TH>Stacking Gel (1 mL) </TH><TH>Running Gel (5 mL; 15%)</TH></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>0.72 mL</TD><TD>1.73 mL </TD></TR><TR><TD>1.5 M Tris</TD><TD>0.13 mL (pH 6.8)</TD><TD>1.3 mL (pH 8.8)</TD></TR><TR><TD>10 % Ammoniumperoxodisulfat</TD><TD>0.01 mL</TD><TD>0.05 mL</TD></TR><TR><TD>10 % N,N′-Methylenbisacrylamid</TD><TD>0.13 mL</TD><TD>0.05 mL</TD></TR><TR><TD>30 % Acrylamid</TD><TD>0.13 mL</TD><TD>1.87 mL</TD></TR><TR><TD>TEMED</TD><TD>0.001 mL</TD><TD>0.002 mL</TD></TR></TBODY></TABLE><P>
Mix the first 5 reagents. Then add TEMED, mix well by pipetting up and down and cast the running gel. Use isopropanol to even it out. Let the running gel dry for half an hour, remove excess isopropanol and add the stacking gel with an appropriate comb on top.
</P><P><B>sample preparation</B></P><OL><LI> resuspend pellet in 150 µl water</LI><LI> measure and dilute to an OD of 10-15</LI><LI> mix 50 µl cell suspension with 10 µl SDS loading dye (if 6x dye)</LI><LI> heat at 95 °C for 15 minutes</LI><LI> load the gel with 7-10 µl</LI><LI> use <A rel="nofollow" class="external text" href="https://tools.thermofisher.com/content/sfs/gallery/high/26619-ladder-002.jpg">PageRuler Protein Ladder</A></LI></OL><P><B>Staining &amp; Destaining</B></P><OL><LI> stain with Coomassie Blue staining solution for half an hour</LI><LI> destain first with destaining solution, then proceed destaining with destilled water
</LI></OL><DIV class="col-md-12 protocol"><A class="protocol" href="#HPLC"><H4>HPLC<SPAN class="HPLC" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual HPLC" style="display:none; padding-bottom:1.5em;"><SPAN style="display:none;">spacer</SPAN><UL><LI>for methanol</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Retention time</TH><TH>Lower detection limit</TH></TR><TR><TD>16,76 +/- 0,02 min</TD><TD>~20 mM</TD></TR></TBODY></TABLE><UL><LI>for sugars</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>HPLC</TD><TD>Beckman System Gold</TD></TR><TR><TD>Column</TD><TD>Organic Acid Resin (PS-DVB), 300 x 8.0 mm, CS-Chromatographie</TD></TR><TR><TD>Detector</TD><TD>RI 166, Beckman</TD></TR><TR><TD>Column Temperature</TD><TD>75°C</TD></TR><TR><TD>Solvent</TD><TD>5 mM H2SO4</TD></TR><TR><TD>Flow Rate</TD><TD>0.8 ml/min</TD></TR><TR><TD>Duration</TD><TD>20 min</TD></TR><TR><TD>Injection Volume</TD><TD>10 µl</TD></TR><TR><TD>Tray Cooling</TD><TD>5°C</TD></TR></TBODY></TABLE><P>Samples have to be centrifuged in order to remove solids (cells/precipitates). No further sample preparation is required.
</P></DIV></DIV><DIV class="col-md-12 protocol"><A class="protocol" href="#Dinitrosalicylic_Acid_Staining"><H4>Dinitrosalicylic Acid Staining<SPAN class="Dinitrosalicylic_Acid_Staining" style="padding-left:1em;color:#bf1521">[+]</SPAN></H4></A><DIV class="col-md-10 col-md-offset-1 manual Dinitrosalicylic_Acid_Staining" style="display:none; padding-bottom:1.5em;">
The dinitrosalicylic acid assay is used to analyse the distribution of branches in Glycogen. Dinitrosalicylic acid reacts with the reducing ends of glycogen (or sugars). <SUP id="cite_ref-13" class="reference"><A href="#cite_note-13">[13]</A></SUP><P><B>Procedure</B></P><OL><LI> Take the purified samples (purification via <A href="/Team:Aachen/Notebook/Protocols#Glycogen_Kit" title="Team:Aachen/Notebook/Protocols">Glycogen Kit</A> section <B>Pre-extraction of glycogen</B>) and add one volume of dinitrosalicylic acid solution (solve 30 g dinitrosalicylic acid in 20 mL 2 M NaOH and add 80 mL water)</LI><LI> Heat stained samples at 90 °C for 15 minutes at 800 rpm</LI><LI> Add one third of the total volume of potassium sodium tartarte solution (40 % wt/vol) to samples, to stabilyze the color</LI><LI> Cool samples down to 25 °C</LI><LI> Analyze samples at 540 nm absorbance and substract the absorbance value of color blank each time
</LI></OL><DIV class="col-md-12"><H1><SPAN class="mw-headline" id="References">References</SPAN></H1></DIV><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Bryksin AV, Bachman HN, Cooper SW, Balavijayan T, Blackstone RM, Du H, Jenkins JP, Haynes CL, Siemer JL, Fiore VF, Barker TH. One primer to rule them all: universal primer that adds BBa_B0034 ribosomal binding site to any coding standard 10 BioBrick. ACS Synth Biol. 2014 Dec 19;<B>3</B>(12):956-9. doi:10.1021/sb500047r. PubMed PMID: 25524097; PubMed Central PMCID: PMC4277749.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Quan, Jiayuan, and Jingdong Tian. “Circular Polymerase Extension Cloning of Complex Gene Libraries and Pathways.” Ed. Paulo Lee Ho. PLoS ONE 4.7 (2009): e6441. PMC. Web. 16 Sept. 2015.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="https://j5.jbei.org/j5manual/pages/22.html">https://j5.jbei.org/j5manual/pages/22.html</A></SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="https://www.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly">https://www.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly</A></SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="http://www.geneious.com">http://www.geneious.com</A></SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="http://parts.igem.org/Help:Transformation_Protocol">http://parts.igem.org/Help:Transformation_Protocol</A> Protocol of the iGEM HQ</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Blank Lars, Protocol for 13C Tracer Experiments</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external text" href="http://www.biovision.com/manuals/K646-100.pdf">Glycogen Assay Kit Manual</A></SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="https://www.thermofisher.com/de/de/home/references/ambion-tech-support/rna-tools-and-calculators/macromolecular-components-of-e.html">https://www.thermofisher.com/de/de/home/references/ambion-tech-support/rna-tools-and-calculators/macromolecular-components-of-e.html</A></SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp">http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp</A></SPAN></LI><LI id="cite_note-11"><SPAN class="mw-cite-backlink"><A href="#cite_ref-11">↑</A></SPAN><SPAN class="reference-text">Nash. 1953. The colorimetric estimation of formaldehyde by means of the Hantzsch reaction. <I>Biochemical Journal</I>, <B>55</B>(3), 416-421.</SPAN></LI><LI id="cite_note-12"><SPAN class="mw-cite-backlink"><A href="#cite_ref-12">↑</A></SPAN><SPAN class="reference-text">Kleeberg &amp; Klinger. 1982. Sensitive formaldehyde determination with Nash's reagent and a tryptophan reaction. <I>Journal of Pharmacological Methods</I>, <B>8</B>(1), 19-31.</SPAN></LI><LI id="cite_note-13"><SPAN class="mw-cite-backlink"><A href="#cite_ref-13">↑</A></SPAN><SPAN class="reference-text">S. K. Meur, V. Sitakara Rao, and K. B. De. Spectrophotometric Estimation of Reducing Sugars by Variation of pH. Z. Anal. Chem. <B>283</B>, 195-197 (1977).</SPAN></LI></OL></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>