<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Methanol_Polycistronic_Expression_Plasmid skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Methanol Polycistronic Expression Plasmid</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
		</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmRDP" href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Polycistronic Plasmid</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><P><B><A href="/Team:Aachen/Notebook/Documentation/Methanol_Polycistronic_Expression_Plasmid#Building_the_Polycistronic_Plasmid_via_RDP_Assembly" title="Team:Aachen/Notebook/Documentation/Methanol Polycistronic Expression Plasmid">Building the Polycistronic Plasmid via RDP Assembly</A></B></P><P><B><A href="/Team:Aachen/Notebook/Documentation/Methanol_Polycistronic_Expression_Plasmid#Building_BioBricks_out_of_polycistronic_plasmid" title="Team:Aachen/Notebook/Documentation/Methanol Polycistronic Expression Plasmid">Building BioBricks out of polycistronic plasmid</A></B></P><H1><SPAN class="mw-headline" id="Building_the_Polycistronic_Plasmid_via_RDP_Assembly">Building the Polycistronic Plasmid via RDP Assembly</SPAN></H1><H2><SPAN class="mw-headline" id="15-07-14">15-07-14</SPAN></H2><H4><SPAN class="mw-headline" id="gradient_PCR_for_following_RDP_part:_X-mdh-Z.27">gradient PCR for following RDP part: X-mdh-Z'</SPAN></H4><UL><LI> gradient PCR products formerly stored in falcon #SY3M# (but now discarded)</LI><LI> check on agarose gel after amplification</LI><LI> PCR program:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 00:30 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:30 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:30 </TD><TD> 60-67
</TD></TR><TR><TD> elongate </TD><TD> 00:40 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 05:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><UL><LI> pipeting volumes:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 15.55
</TD></TR><TR><TD> F-Primer #KLF1# </TD><TD> 0.8
</TD></TR><TR><TD> R-Primer #DD84# </TD><TD> 0.8
</TD></TR><TR><TD> 10x PCR buffer B2 </TD><TD> 2
</TD></TR><TR><TD> dNTPs (10mM) O5 </TD><TD> 0.4
</TD></TR><TR><TD> PfuS </TD><TD> 0.2
</TD></TR><TR><TD> Template #YSVQ# </TD><TD> 0.25
</TD></TR></TBODY></TABLE><UL><LI> results: 63.7 °C is the best temperature for this primer combination</LI></UL><H4><SPAN class="mw-headline" id="Q5_amplification_of_RDP_part_X-mdh-Z.27">Q5 amplification of RDP part X-mdh-Z'</SPAN></H4><UL><LI> amplification product stored in falcon #K913# (previosly in falcon #SY3M#)</LI><LI> PCR program:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 03:00 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 63.7
</TD></TR><TR><TD> elongate </TD><TD> 00:40 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><UL><LI> pipeting volumes:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 20.75
</TD></TR><TR><TD> FPrimer #KLF1# </TD><TD> 2
</TD></TR><TR><TD> RPrimer #DD84# </TD><TD> 2
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> Template #YSVQ# </TD><TD> 0.25
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-28">15-07-28</SPAN></H2><H3><SPAN class="mw-headline" id="Gradient_PCRs_for_RDP_parts">Gradient PCRs for RDP parts</SPAN></H3><UL><LI> Gradient PCR products stored in: #964L# and #LPPT# (now discarded) </LI><LI>  Z-xpk.B0015-X' gradient PCR products stored in #K1MB#</LI></UL><H4><SPAN class="mw-headline" id="Z-hps-X.27">Z-hps-X'</SPAN></H4><UL><LI> Polymerase: PfuS</LI><LI> Reaction Volume: 25 µl</LI><LI> FW Primer: #ODYN#</LI><LI> RV Primer: #PWVZ#</LI><LI> Template: #H4M9#</LI><LI> result: 57.8 °C</LI></UL><H4><SPAN class="mw-headline" id="X-phi-Z.27">X-phi-Z'</SPAN></H4><UL><LI> Polymerase: PfuS</LI><LI> Reaction Volume: 25 µl</LI><LI> FW Primer: #S6WV#</LI><LI> RV Primer: #1XBB#</LI><LI> Template: #ANNA#</LI><LI> result: 61.9 °C</LI></UL><H4><SPAN class="mw-headline" id="Z-xpk.B0015-X.27">Z-xpk.B0015-X'</SPAN></H4><UL><LI> Polymerase: PfuS</LI><LI> Reaction Volume: 25 µl</LI><LI> FW Primer: #RKW3#</LI><LI> RV Primer: #CNAT#</LI><LI> Template: #8ZRY#</LI><LI> result: 58.0 °C</LI></UL><H2><SPAN class="mw-headline" id="15-07-29">15-07-29</SPAN></H2><P><B>RDP part Q5 amplification</B></P><UL><LI> Q5 amplification products stored in falcon #K913#</LI></UL><H4><SPAN class="mw-headline" id="Z-hps-X.27_2">Z-hps-X'</SPAN></H4><UL><LI> FW Primer: #ODYN#</LI><LI> RV Primer: #PWVZ#</LI><LI> Template: #H4M9# (0.5 µl)</LI><LI> Elongation: 0'23</LI><LI> Annealing: 57.8 °C</LI></UL><H4><SPAN class="mw-headline" id="X-phi-Z.27_2">X-phi-Z'</SPAN></H4><UL><LI> FW Primer: #S6WV#</LI><LI> RV Primer: #1XBB#</LI><LI> Template: #ANNA# (0.5 µl)</LI><LI> Elongation: 0'27</LI><LI> Annealing: 61.9 °C</LI></UL><H4><SPAN class="mw-headline" id="Z-xpk.B0015-X.27_2">Z-xpk.B0015-X'</SPAN></H4><UL><LI> FW Primer: #RKW3#</LI><LI> RV Primer: #CNAT#</LI><LI> Template: #8ZRY# (0.2 µl)</LI><LI> Elongation: 1'23</LI><LI> Annealing: 58.0 °C</LI></UL><H2><SPAN class="mw-headline" id="15-07-30">15-07-30</SPAN></H2><H3><SPAN class="mw-headline" id="Summary">Summary</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> construct name </TH><TH> Q5 PCR clean-up ID </TH><TH> minimal required ammount [µg]
</TH></TR><TR><TD>  Z-hps-X' </TD><TD> #ZDFQ# </TD><TD> 15
</TD></TR><TR><TD> X-phi-Z' </TD><TD> #XL1B# </TD><TD> 15
</TD></TR><TR><TD>  X-mdh-Z' </TD><TD> #KLV4# </TD><TD> 15
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Further_Q5_amplifications_for_higher_DNA_amount">Further Q5 amplifications for higher DNA amount</SPAN></H3><UL><LI> stored in falcon #64M1#</LI></UL><H4><SPAN class="mw-headline" id="1:_X-mdh-Z.27">1: X-mdh-Z'</SPAN></H4><UL><LI> 6*50 µl reactions</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 20.75
</TD></TR><TR><TD> FPrimer #KLF1# </TD><TD> 2
</TD></TR><TR><TD> RPrimer #DD84# </TD><TD> 2
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> Template #YSVQ# </TD><TD> 0.25
</TD></TR></TBODY></TABLE><UL><LI>PCR program:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 03:00 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 63.7
</TD></TR><TR><TD> elongate </TD><TD> 00:40 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>Stored in #BOS6#; DNA concentration: 153,8 [ng/µl]
</P><H4><SPAN class="mw-headline" id="2:_Z-hps-X.27">2: Z-hps-X'</SPAN></H4><UL><LI> 6*50 µl reactions</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 20.5
</TD></TR><TR><TD> FPrimer #ODYN# </TD><TD> 2
</TD></TR><TR><TD> RPrimer #PWVZ# </TD><TD> 2
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> Template #H4M9# </TD><TD> 0.5
</TD></TR></TBODY></TABLE><UL><LI>PCR program:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 03:00 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 57.8
</TD></TR><TR><TD> elongate </TD><TD> 00:23 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>Stored in #NXRA#; DNA concentration 105,6 [ng/µl]
</P><H4><SPAN class="mw-headline" id="3:_X-phi-Z.27">3: X-phi-Z'</SPAN></H4><UL><LI> 6*50 µl reactions</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 20.5
</TD></TR><TR><TD> FPrimer #S6WV# </TD><TD> 2
</TD></TR><TR><TD> RPrimer #1XBB# </TD><TD> 2
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> Template #ANNA# </TD><TD> 0.5
</TD></TR></TBODY></TABLE><UL><LI>PCR program:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 03:00 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 61.9
</TD></TR><TR><TD> elongate </TD><TD> 00:27 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>Stored in #PRTC#;
DNA concentration: 138,6 [ng/µl]
</P><H4><SPAN class="mw-headline" id="4:_Z-xpk.B0015-X.27">4: Z-xpk.B0015-X'</SPAN></H4><UL><LI> 4*50 µl reactions</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 20.8
</TD></TR><TR><TD> FPrimer #RKW3# </TD><TD> 2
</TD></TR><TR><TD> RPrimer #CNAT# </TD><TD> 2
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> Template #8ZRY# </TD><TD> 0.2
</TD></TR></TBODY></TABLE><UL><LI>PCR program:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 03:00 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 61.9
</TD></TR><TR><TD> elongate </TD><TD> 01:23 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>Stored in #XMO9#
DNA concentration: 186,9 [ng/µl]
</P><H2><SPAN class="mw-headline" id="15-08-06">15-08-06</SPAN></H2><H3><SPAN class="mw-headline" id="BsaI_digest_of_RDP_Parts_10x_overdigest">BsaI digest of RDP Parts 10x overdigest</SPAN></H3><P><B> X-mdh-Z'</B></P><P>Tube 'W'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 75
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #BOS6# </TD><TD> 13
</TD></TR></TBODY></TABLE><P>Concentration after clean-up: 18.7 [ng/µl]
</P><P><B> Z-hps-X' </B></P><P>Tube 'X'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 69.1
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #NXRA# </TD><TD> 18.9
</TD></TR></TBODY></TABLE><P>Concentration after clean-up: 12.7 [ng/µl]
</P><P><B> X-phi-Z' </B></P><P>Tube 'Y'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 73.6
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #PRTC# </TD><TD> 14.4
</TD></TR></TBODY></TABLE><P>Concentration after clean-up: 11.8 [ng/µl]
</P><P><B> Z-xpk.B0015X' </B></P><P>Tube 'Z'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 77.3
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #XMO9# </TD><TD> 10.7
</TD></TR></TBODY></TABLE><P>Concentration after clean-up: 22.0 [ng/µl]
</P><P><B>Store digestions in Falcon #TRYE#</B></P><H2><SPAN class="mw-headline" id="15-08-07">15-08-07</SPAN></H2><P>We did the digestion twice because of non sufficient concentrations
</P><DL><DD><B>First Digest:</B></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Construct </TH><TH> conc. [ng/µl]
</TH></TR><TR><TD> &quot;W&quot; / X-mdh-Z' </TD><TD>  18,7
</TD></TR><TR><TD> &quot;X&quot; / Z-hps-X' </TD><TD> 12,7
</TD></TR><TR><TD> &quot;Y&quot; / X-phi-Z' </TD><TD>  11,8
</TD></TR><TR><TD> &quot;Z&quot; /Z-xpk.B0015-X' </TD><TD> 22,0
</TD></TR></TBODY></TABLE><DL><DD><B>Second digest:</B></DD></DL><H3><SPAN class="mw-headline" id="BsaI_digestion_of_RDP_Parts_5x_overdigest">BsaI digestion of RDP Parts 5x overdigest</SPAN></H3><P><B> X-mdh-Z</B></P><P>Tube 'I'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 62
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #BOS6# </TD><TD> 26
</TD></TR></TBODY></TABLE><P>Concentration after clean-up: 49.0 [ng/µl]
</P><P><B> Z-hps-X' </B></P><P>Tube 'II'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 50.1
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #NXRA# </TD><TD> 37.9
</TD></TR></TBODY></TABLE><P>Concentration after clean-up: 66.5 [ng/µl]
</P><P><B> X-phi-Z' </B></P><P>Tube 'III'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 59.1
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #PRTC# </TD><TD> 28.9
</TD></TR></TBODY></TABLE><P><B>Z-xpk.B0015-X' </B></P><P>Concentration after clean-up: 22.4 [ng/µl]
</P><P>Tube 'IV'
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume per tube [µl]
</TH></TR><TR><TD> ddH<SUB>2</SUB>O </TD><TD> 66.6
</TD></TR><TR><TD> BsaI </TD><TD> 2
</TD></TR><TR><TD> NEB 10x CutSmart Buffer </TD><TD> 10
</TD></TR><TR><TD> #XMO9# </TD><TD> 21.4
</TD></TR></TBODY></TABLE><DL><DD><B>Results of second digest</B></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Construct </TH><TH> conc. [ng/µl]
</TH></TR><TR><TD> &quot;I&quot; / X-mdh-Z' </TD><TD>  49.0
</TD></TR><TR><TD> &quot;II&quot; /Z-hps-X' </TD><TD> 66.5
</TD></TR><TR><TD> &quot;III&quot; /X-phi-Z' </TD><TD>  22.4
</TD></TR><TR><TD> &quot;IV&quot; / Z-xpk.B0015-X' </TD><TD> 25.3
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="RDP_parts_summary">RDP parts summary</SPAN></H3><P>Assembly-ready RDP parts:
</P><TABLE class="wikitable"><TBODY><TR><TH> Part ID </TH><TH> Part description </TH><TH> c [pmol/µl] </TH><TH> volume [µl]
</TH></TR><TR><TD> #PRDW# </TD><TD> X-mdh-Z'  </TD><TD> 0.04 </TD><TD> 44.7
</TD></TR><TR><TD> #VO4C# </TD><TD> Z-hps-X'  </TD><TD> 0.03 </TD><TD> 49
</TD></TR><TR><TD> #ZALV# </TD><TD> X-phi-Z'  </TD><TD> 0.04 </TD><TD> 78
</TD></TR><TR><TD> #ZR1Q# </TD><TD> Z-xpk.B0015.X'  </TD><TD> 0.04 </TD><TD> 22
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Resuspending_RDP_parts_from_kitplate">Resuspending RDP parts from kitplate</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> Part ID </TH><TH> Part description </TH><TH> volume [µl]
</TH></TR><TR><TD> #PLTB# </TD><TD> Z-Ter-X' </TD><TD> 110
</TD></TR><TR><TD> #RYPQ# </TD><TD> dA20-ChlR-Z' </TD><TD> 110
</TD></TR><TR><TD> #XOXM# </TD><TD> dA20-ChlR-X' </TD><TD> 110
</TD></TR><TR><TD> #HNXW# </TD><TD> dA20-AmpR-Z' </TD><TD> 110
</TD></TR><TR><TD> #FO4B# </TD><TD> dA20-KanR-Z' </TD><TD> 110
</TD></TR><TR><TD> #PVFL# </TD><TD> Z-CAP-Ori.3-dT20 </TD><TD> 110
</TD></TR><TR><TD> #OZD1# </TD><TD> X-CAP-Ori.3-dT20 </TD><TD> 110
</TD></TR><TR><TD> #RAXA# </TD><TD> X-Blk </TD><TD> 220
</TD></TR><TR><TD> #BWD6# </TD><TD> Z-Blk </TD><TD> 220
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Assembling_oligos">Assembling oligos</SPAN></H3><UL><LI> oligos had to be lyophilized to be resuspended in TE buffer</LI><LI> Gel control of the 43 bp oligos: looked good</LI><LI><B>oligo assembly</B>: BBa_B1002-RDP (#HTSR#), BBa_B1006-RDP (#8AMS#), AP19-RDP (#MBWS#)
</LI><LI>heated on 90 °C, then linear cool down in 45 min to room temperature</LI></UL><H2><SPAN class="mw-headline" id="15-08-12">15-08-12</SPAN></H2><H3><SPAN class="mw-headline" id="Assembly_of_polycistronic_versions">Assembly of polycistronic versions</SPAN></H3><DL><DD>T7 polycistronic version assembly product: #DSHM#</DD><DD>AP19 polycistronic version assembly product: #MCLC#</DD></DL><UL><LI>each 5 µl transformed in DH5 alpha and additionally, T7 version in BL21</LI></UL><H2><SPAN class="mw-headline" id="15-08-13">15-08-13</SPAN></H2><H3><SPAN class="mw-headline" id="tasks:">tasks:</SPAN></H3><UL><LI> masterplates of T7.poly</LI><LI> masterplates of AP19.poly</LI></UL><H2><SPAN class="mw-headline" id="15-08-14">15-08-14</SPAN></H2><UL><LI>Colony PCR on AP19.Poly and T7.Poly</LI></UL><P><B>Results table:</B></P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> Primers </TH><TH> elongation time </TH><TH> expected length [bp] </TH><TH> tested clones </TH><TH> positive clones
</TH></TR><TR><TD> AP19 polycistronic construct </TD><TD> varying </TD><TD> 2'15</TD><TD> varying </TD><TD> 0, 1, 3, 4 </TD><TD> 1
</TD></TR><TR><TD> #W6WP#, #HSP2# </TD><TD> 1878 </TD><TD> 0, 1, 3, 4 </TD></TR><TR><TD> #HSP1#, #XSP3# </TD><TD> 1939 </TD><TD> 0, 1, 3, 4 </TD></TR><TR><TD> #XSP2#, #94LB# </TD><TD> 2182 </TD><TD> 0, 1, 3, 4 </TD></TR><TR><TD> T7 polycistronic construct </TD><TD> varying </TD><TD> 2'15</TD><TD> varying </TD><TD> 1, 2, 7, 8 </TD><TD> -
</TD></TR><TR><TD> #W6WP#, #HSP2# </TD><TD> 1927 </TD><TD> 1, 2, 7, 8 </TD></TR><TR><TD> #HSP1#, #XSP3# </TD><TD> 1939 </TD><TD> 1, 2, 7, 8 </TD></TR><TR><TD> #XSP2#, #94LB# </TD><TD> 2182 </TD><TD> 1, 2, 7, 8 </TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-08-16">15-08-16</SPAN></H2><H3><SPAN class="mw-headline" id="Tasks:_2">Tasks:</SPAN></H3><UL><LI> purify plasmid of AP19 polycistronic clone #1
</LI><LI>Cryo #Q1Q3#</LI><LI>purified plasmid #POLY#</LI></UL><LI> make masterplates of yesterdays trafos</LI><LI> do/make colony PCR/overnight cultures of yesterdays masterplates</LI><LI> test KAPA 2G Mix</LI><LI> do test PCR with purified plasmid of AP19 poly
</LI><LI>RESULT: POSITIVE!!</LI><H2><SPAN class="mw-headline" id="15-08-17">15-08-17</SPAN></H2><P>Again, do:
</P><UL><LI> AP19 Poly plasmid purification +  cryo:
</LI><LI> cryo: #LHZV#</LI><LI> pasmid: #WVDR#</LI></UL><H2><SPAN class="mw-headline" id="15-08-18">15-08-18</SPAN></H2><H3><SPAN class="mw-headline" id="Morning">Morning</SPAN></H3><P>Plasmid purifications and sequencing
</P><UL><LI> elute all 6 AP19.Poly overnight cultures (all from #Q1Q3#) into one target container named #C3NS# for sufficient DNA concentration</LI></UL><P>Pipetting calculation table is ready at: <B>\WetLab\Documentation\Methanol\15-08-18 Sequencings\</B></P><TABLE class="wikitable"><TBODY><TR><TH> construct &amp; clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing sample numbers </TH><TH> sequencing result </TH><TH> further procedure
</TH></TR><TR><TD> AP19.POLY in pSB1KRDP #1 </TD><TD> #W3NB# </TD><TD><B>#C3NS#</B></TD><TD> 1-8 </TD><TD> NO CONTIGS FOUND </TD><TD> Repeat Sequencing
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Noon">Noon</SPAN></H3><P>Purify plasmids of T7.POLY in BL21 (DE3)
</P><TABLE class="wikitable"><TBODY><TR><TH> construct &amp; clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> check PCR result
</TH></TR><TR><TD> T7.lacO.POLY in pSB1KRDP #1 </TD><TD> #LHBN# </TD><TD> #VYCO# </TD><TD> discard
</TD></TR><TR><TD> T7.lacO.POLY in pSB1KRDP #5 </TD><TD> #HB93# </TD><TD> #ANPV# </TD><TD> discard
</TD></TR><TR><TD> T7.lacO.POLY in pSB1KRDP #6 </TD><TD> #6LOZ# </TD><TD> #96TS# </TD><TD> discard
</TD></TR><TR><TD> T7.lacO.POLY in pSB1KRDP #7 </TD><TD> #KYOQ# </TD><TD> #XVE3# </TD><TD> discard
</TD></TR><TR><TD> T7.lacO.POLY in pSB1KRDP #8 </TD><TD> #MCMR# </TD><TD> #BSLA# </TD><TD> discard
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Check-PCR_of_purified_T7.POLY_plasmids">Check-PCR of purified T7.POLY plasmids</SPAN></H4><UL><LI>with three primer pairs, to find a complete construct</LI><LI>elongation time 0'33</LI><LI> #POLY# as positive control (fitting bands already 15-08-16)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Part ID </TH><TH> Sample </TH><TH> Primers </TH><TH> expected length [bp] </TH><TH> result
</TH></TR><TR><TD> #VYCO# </TD><TD> 1 </TD><TD> #W6WP# + #HSP2# </TD><TD> 1927 </TD><TD> negative
</TD></TR><TR><TD> 2 </TD><TD> #HSP1# + #XSP3#</TD><TD> 1939 </TD><TD> negative
</TD></TR><TR><TD> 3 </TD><TD> #XSP2# + #94LB#</TD><TD> 2182 </TD><TD> negative
</TD></TR><TR><TD> #ANPV# </TD><TD> 4 </TD><TD> #W6WP# + #HSP2#</TD><TD>1927 </TD><TD> negative
</TD></TR><TR><TD> 5 </TD><TD> #HSP1# + #XSP3#</TD><TD>1939 </TD><TD> negative
</TD></TR><TR><TD> 6 </TD><TD> #XSP2# + #94LB#</TD><TD>2182 </TD><TD> negative
</TD></TR><TR><TD> #96TS# </TD><TD> 7 </TD><TD> #W6WP# + #HSP2#</TD><TD>1927 </TD><TD> negative
</TD></TR><TR><TD> 8 </TD><TD> #HSP1# + #XSP3#</TD><TD>1939 </TD><TD> negative
</TD></TR><TR><TD> 9 </TD><TD> #XSP2# + #94LB#</TD><TD>2182 </TD><TD> negative
</TD></TR><TR><TD> #XVE3# </TD><TD> 10 </TD><TD> #W6WP# + #HSP2#</TD><TD>1927 </TD><TD> negative
</TD></TR><TR><TD> 11 </TD><TD> #HSP1# + #XSP3#</TD><TD>1939 </TD><TD> negative
</TD></TR><TR><TD> 12 </TD><TD> #XSP2# + #94LB#</TD><TD>2182 </TD><TD> negative
</TD></TR><TR><TD> #BSLA# </TD><TD> 13 </TD><TD> #W6WP# + #HSP2#</TD><TD>1927 </TD><TD> negative
</TD></TR><TR><TD> 14 </TD><TD> #HSP1# + #XSP3#</TD><TD>1939 </TD><TD> negative
</TD></TR><TR><TD> 15 </TD><TD> #XSP2# + #94LB#</TD><TD>2182 </TD><TD> negative
</TD></TR><TR><TD> #POLY# (AP19.Poly, positive control)</TD><TD> 16 </TD><TD> #W6WP# + #HSP2#</TD><TD>1927 </TD><TD> positive
</TD></TR><TR><TD> 17 </TD><TD> #HSP1# + #XSP3#</TD><TD>1939 </TD><TD> positive
</TD></TR><TR><TD> 18 </TD><TD> #XSP2# + #94LB#</TD><TD>2182 </TD><TD> positive
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="RDP_Assembly">RDP Assembly</SPAN></H4><UL><LI> Assembly of #FO4B# and #PVFL# (Kanamycin anchor with high copy cap) the product is #QWRC# -&gt; we need a strain with this plasmid as a negative control that carries the plasmid but does not express anything for SDS-PAGE</LI></UL><P>Transformation of plasmids in BL21 Gold (DE3) and plate on M9 + K + 40mM Glucose as well as LB+K
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> transformed plasmid </TH><TH> BL21 Gold (DE3) cryo
</TH></TR><TR><TD> AP19.POLY in pSB1KRDP </TD><TD> #POLY# </TD><TD> #AW9K#
</TD></TR><TR><TD> pSB1KRDP </TD><TD> #QWRC#  </TD><TD> #....#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-08-19">15-08-19</SPAN></H2><UL><LI> make master plates and overnights of BL21 Gold (DE3) transformants (M9+K+40mM Glucose)
</LI><LI> pSB1KDP from #QWRC# on/in LB+K</LI><LI> pSB1KDP from #QWRC# on/in M9+K</LI><LI> AP19.Poly from #POLY# on/in LB+K</LI><LI> AP19.Poly from #POLY# on/in M9+K</LI></UL><UL><LI> analyze sequencing results of polycistronic construct
</LI><LI> Sequencing of #C3NS# did not work, so check with PCR and test digest
</LI><LI> Test PCR: three primer pairs (#W6WP# + #HSP2#; #HSP1# + #XSP3#; #XSP2# + #94LB#)</LI><LI> #POLY# as positive control</LI></UL><LI> if positive: send in for sequencing</LI><H3><SPAN class="mw-headline" id="Sequencing_for_tomorrow:">Sequencing for tomorrow:</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> construct &amp; clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing sample numbers </TH><TH> sequencing result
</TH></TR><TR><TD> AP19.POLY in pSB1KRDP #1 </TD><TD>#Q1Q3# #LHZV# #W3NB#</TD><TD> #XULU# </TD><TD> 1-8 </TD><TD>positive (result after several sequencings because of problems with sequencing service)
</TD></TR></TBODY></TABLE><P><B>sequencing prouven polycistronic construct!!!!</B></P><H1><SPAN class="mw-headline" id="Sequencing_Result_of_AP19.Poly_construct">Sequencing Result of AP19.Poly construct</SPAN></H1><DIV class="figraw figure-large col-md-6"><TABLE><TBODY><TR><TD><B>Successful Sequencing of Polycistronic Construct</B></TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="Building_BioBricks_out_of_polycistronic_plasmid">Building BioBricks out of polycistronic plasmid</SPAN></H1><H2><SPAN class="mw-headline" id="15-08-19_2">15-08-19</SPAN></H2><P><B>Next steps</B></P><UL><LI>build RuMP pathway (AP19.mdh.hps.phi, like MCC but without xpk, for comparison)</LI><LI>bring the polycistronic construct into pSB1C3 (to provide it as a BioBrick) and into pSB1K3 (to characterize it further) METHODS:
</LI><LI>CPEC</LI><LI>restriction with XbaI and SpeI and ligation </LI><LI>restiction with EcoRI and PstI and ligations</LI><LI>Gibson Assembly</LI><LI>digestion with XbaI and SpeI, then digestion with alkaline phosphatase, then ligation</LI></UL><H2><SPAN class="mw-headline" id="15-08-17_polycistronic_expression_test">15-08-17 polycistronic expression test</SPAN></H2><P>SDS-PAGE of AP19 Polycistronic construct derived from cryo #Q1Q3#
</P><H3><SPAN class="mw-headline" id="Results:">Results:</SPAN></H3><UL><LI> mdh: (expected 40,7 kDa) is visible as a big band in Q1Q3</LI><LI> phi+hps: (expected 20 &amp; 22,6 kDa) there is a big band between 17 &amp; 26 kDa</LI><LI> xpk: (expected 92 kDa) is visible as a big band in #Q1Q3#</LI></UL><H2><SPAN class="mw-headline" id="15-08-25">15-08-25</SPAN></H2><H3><SPAN class="mw-headline" id="RuMP_pathway">RuMP pathway</SPAN></H3><UL><LI> for comparison of MCC pathway, we built the RuMP pathway in a polycistronic plasmid</LI><LI> #FO4B# + #XYD9# + #PRDW# + #VO4C# + #ZALV# + #PLTB# + #OZ1D#</LI><LI> AP19.MDH.HPS.PHI.Ter in pSB1KRDP, assembly product stored in #BMDS#</LI><LI> Hps RDP part is empty now </LI></UL><H3><SPAN class="mw-headline" id="Cloning_AP19_Poly_into_Biobrick_backbone_via_CPEC">Cloning AP19 Poly into Biobrick backbone via CPEC</SPAN></H3><UL><LI> fragment amplification products in:
</LI><LI> #OX3R#</LI><LI> #PQMO# </LI></UL><LI> CPEC products stored in Falcon #ZTYD#</LI><H4><SPAN class="mw-headline" id="Strategy:_Polycistronic_construct_into_Biobrick_backbone">Strategy: Polycistronic construct into Biobrick backbone</SPAN></H4><H5><SPAN class="mw-headline" id="Fragment_amlifications">Fragment amlifications</SPAN></H5><UL><LI> amplify #XULU# with primers #HSTL# and #8XOZ#</LI></UL><P>Programme:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 00:30 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 66
</TD></TR><TR><TD> elongate </TD><TD> 02:40 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>PCR mix:
</P><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> Volume [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 19
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> #HSTL# </TD><TD> 2.5
</TD></TR><TR><TD> #8XOZ# </TD><TD> 2.5
</TD></TR><TR><TD> #XULU# </TD><TD> 1
</TD></TR></TBODY></TABLE><UL><LI> amplification product stored in #OX3R#</LI><LI> Amplify Biocbrick backbones with #FYTO# and #TSTV#
</LI><LI> pSB1C3 amplification product stored in #6WCN#</LI><LI> use #QLE4# as template for pSB1K3 backbone amplification</LI></UL><P>Programme:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 00:30 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:15 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:15 </TD><TD> 68
</TD></TR><TR><TD> elongate </TD><TD> 01:10 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 02:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 16
</TD></TR></TBODY></TABLE><P>PCR mix:
</P><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> Volume [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 21.5
</TD></TR><TR><TD> 2x Q5 MM </TD><TD> 25
</TD></TR><TR><TD> #TSTV# </TD><TD> 1.25
</TD></TR><TR><TD> #FYTO# </TD><TD> 1.25
</TD></TR><TR><TD> #QLE4# </TD><TD> 1
</TD></TR></TBODY></TABLE><UL><LI> pSB1K3 amplification product stored in #PQMO# </LI></UL><P><B>CPEC</B></P><UL><LI> use #OX3R# and [#6WCN# / #PQMO#] in a CPEC to build the AP19 Poly construct in pSB1[C/K]3 backbone</LI></UL><UL><LI> Components (total reaction volume=25 µl)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> components </TH><TH> amount
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 11.5 µl
</TD></TR><TR><TD> Q5 Buffer </TD><TD> 5 µl
</TD></TR><TR><TD> DMSO </TD><TD> 0.75 µl
</TD></TR><TR><TD> dNTPs </TD><TD> 5 µl
</TD></TR><TR><TD> Q5 Polymerase </TD><TD> 0.5 µl
</TD></TR><TR><TD> Backbone (#6WCN# / #PQMO#) </TD><TD> 100 ng
</TD></TR><TR><TD> fragment (#OX3R#) </TD><TD> 0.073 pmol
</TD></TR></TBODY></TABLE><UL><LI> CPEC Programme:</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> time </TH><TH> T [C°]
</TH></TR><TR><TD> denature </TD><TD> 00:30 </TD><TD> 98
</TD></TR><TR><TD> denature </TD><TD> 00:10 </TD><TD> 98
</TD></TR><TR><TD> anneal </TD><TD> 00:30 </TD><TD> 55
</TD></TR><TR><TD> elongate </TD><TD> 03:45 </TD><TD> 72
</TD></TR><TR><TD> elongate </TD><TD> 10:00 </TD><TD> 72
</TD></TR><TR><TD> store </TD><TD> forever </TD><TD> 8
</TD></TR></TBODY></TABLE><UL><LI> products stored in Falcon #ZTYD#</LI></UL><H2><SPAN class="mw-headline" id="15-08-26">15-08-26</SPAN></H2><H4><SPAN class="mw-headline" id="RuMP_Pathway_2">RuMP Pathway</SPAN></H4><UL><LI>not all of the 10 overnights grew</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> conc. [ng/µl] </TH><TH> result
</TH></TR><TR><TD> #1 </TD><TD> #QM6C# </TD><TD> #VK8F# </TD><TD> 19,1 </TD><TD> neg
</TD></TR><TR><TD> #2 </TD><TD> #KYAV# </TD><TD> #HYNT# </TD><TD> 21,3 </TD><TD> nge
</TD></TR><TR><TD> #4 </TD><TD> #YROE# </TD><TD> #VLVZ# </TD><TD> 51,7 </TD><TD>neg
</TD></TR><TR><TD> #6 </TD><TD> #F3KA# </TD><TD> #OD9H#</TD><TD> 47,2 </TD><TD>neg
</TD></TR><TR><TD> #7 </TD><TD> #KFYB# </TD><TD> #YDBL#</TD><TD> 34,8 </TD><TD>neg
</TD></TR><TR><TD> #8</TD><TD> #8OCO# </TD><TD> #861V#</TD><TD> 30,8 </TD><TD>neg
</TD></TR><TR><TD> #9</TD><TD> #1H9P# </TD><TD> #HH6E#</TD><TD> 51,7 </TD><TD>neg
</TD></TR></TBODY></TABLE><UL><LI>SDS page will show which one expresses the genes</LI><LI>staining seems to be difficult (used DNA loading dye instead of SDS loading dye)</LI></UL><P><B>Test PCR</B></P><UL><LI>Primers W6WP and 94LB, expected lenght 2926 bp, 0'44<I> elongation time</I></LI></UL><P><B>Sequencing</B></P><P>Plasmid #YDBL# was send to sequencing with #W6WP# &amp; #94LB#
</P><H2><SPAN class="mw-headline" id="15-08-28">15-08-28</SPAN></H2><UL><LI>screening for good constructs: ALL CLONES NEGATIVE!!</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct  </TH><TH> clone </TH><TH> Cryo ID </TH><TH> Plasmid ID </TH><TH> Plasmid conc. [ng/µl] </TH><TH> result
</TH></TR><TR><TD> AP19.Poly in pSB1K3 in DH5alpha </TD><TD>  #1 </TD><TD> #NBBB#  </TD><TD> #34ZR#  </TD><TD> neg
</TD></TR><TR><TD>  #2  </TD><TD> #NLCA# </TD><TD> #DHLZ#  </TD><TD> neg
</TD></TR><TR><TD> #3  </TD><TD> #DAAW#  </TD><TD> #VSHE# </TD><TD> neg
</TD></TR><TR><TD> #4 </TD><TD> #XTDS#  </TD><TD> #Z9YB#  </TD><TD> neg
</TD></TR><TR><TD> AP19.Poly in pSB1K3 in BL21 </TD><TD> #1 </TD><TD> #K9V1# </TD><TD> #F3MF#  </TD><TD> neg
</TD></TR><TR><TD> #2 </TD><TD> #L4DS#  </TD><TD> #BLPR#  </TD><TD> neg
</TD></TR><TR><TD> #3 </TD><TD>  #MH8O# </TD><TD>#RLRA#  </TD><TD> neg
</TD></TR><TR><TD>  #4</TD><TD> #FWZW#  </TD><TD> #O31X#  </TD><TD> neg
</TD></TR><TR><TD> AP19.Poly in pSB1C3 in BL21 </TD><TD> #1 </TD><TD> #H1YN#  </TD><TD> #3M83#  </TD><TD> neg
</TD></TR><TR><TD> AP19.Poly in pSB1C3 in DH5alpha </TD><TD> #1 </TD><TD> #B1WF#  </TD><TD> #DZLQ#  </TD><TD> neg
</TD></TR><TR><TD> #2 </TD><TD> #ZEFM#  </TD><TD> #4YSX#  </TD><TD> neg
</TD></TR><TR><TD> #3 </TD><TD> #ERWH#  </TD><TD> #D1ZS#  </TD><TD> neg
</TD></TR><TR><TD> #4 </TD><TD> #NLZR#  </TD><TD> #ADHH#  </TD><TD> neg
</TD></TR><TR><TD> #5 </TD><TD> #D4H3#  </TD><TD> #NRZ9#  </TD><TD> neg
</TD></TR><TR><TD> #6 </TD><TD> #4A4L#  </TD><TD> #A4BS#  </TD><TD> neg
</TD></TR><TR><TD> #7 </TD><TD> #3VMY#  </TD><TD> #6XKM#  </TD><TD> neg
</TD></TR><TR><TD> #8 </TD><TD> #VFB3#  </TD><TD> #MM4R#  </TD><TD> neg
</TD></TR><TR><TD> #9 </TD><TD> #F9DQ#  </TD><TD> #LZWV#  </TD><TD> neg
</TD></TR><TR><TD> #10 </TD><TD> #OL63#  </TD><TD> #CMME#  </TD><TD> neg
</TD></TR><TR><TD> #11 </TD><TD> #AA4O#  </TD><TD> #6RFK#  </TD><TD> neg
</TD></TR><TR><TD> #12 </TD><TD> #8ZX6#  </TD><TD> #QLS1#  </TD><TD> neg
</TD></TR><TR><TD> #13 </TD><TD> #L3PR#  </TD><TD> #E6XR#  </TD><TD> neg
</TD></TR><TR><TD> #14 </TD><TD> #FS19#  </TD><TD> #HLVZ#  </TD><TD> neg
</TD></TR><TR><TD> #15 </TD><TD> #CVEX#  </TD><TD> #QY8T#  </TD><TD> neg
</TD></TR><TR><TD> #16 </TD><TD> #DDO6#  </TD><TD> #WXRA#  </TD><TD> neg
</TD></TR><TR><TD> #17 </TD><TD> #LRPO#  </TD><TD> #YQZ8#  </TD><TD> neg
</TD></TR><TR><TD> #18 </TD><TD> #DDQQ#  </TD><TD> #BLOA#  </TD><TD> neg
</TD></TR></TBODY></TABLE><UL><LI>cultures of clones with kanamycin resistance in BL21 were prepared for SDS-PAGEs</LI></UL><H2><SPAN class="mw-headline" id="15-08-29">15-08-29</SPAN></H2><UL><LI> results of check PCR (AP19 Poly in BioBrick backbones): NO PLASMID WITH CONSTRUCT</LI><LI>again do transformation of:
</LI><LI> AP19 Poly in pSB1C3 in DH5alpha (Ligation and Trafo ID: 3)</LI><LI> AP19 Poly in pSB1K3 in BL21 Gold DE3 (Ligation and Trafo ID: 4)</LI></UL><LI> digests and ligation products stored in Falcon #OLAE#</LI><P><B>digest</B>:
</P><UL><LI> 30 µl reaction volume</LI><LI> enzyme and buffer volumes acording to cheat sheet</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> plasmid </TH><TH> Enzymes </TH><TH> Water added [µl]
</TH></TR><TR><TD> #OLAE#:A </TD><TD> #CD8B# </TD><TD> E + P </TD><TD> 21.6
</TD></TR><TR><TD> #OLAE#:D </TD><TD> #CD8B# </TD><TD> X + S </TD><TD> 21.6
</TD></TR><TR><TD> #OLAE#:E </TD><TD> #AQY3# </TD><TD> X + S </TD><TD> 21.4
</TD></TR><TR><TD> #OLAE#:F </TD><TD> #OX3R# </TD><TD> X + S </TD><TD> 24.0
</TD></TR></TBODY></TABLE><P><B>ligation</B>:
</P><UL><LI> 20 µl reaction volume</LI><LI> 4 µl part digest</LI><LI> 2 µl backbone digest</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> part digest </TH><TH> backbone digest
</TH></TR><TR><TD> #OLAE#:3 </TD><TD> #OLAE#:F </TD><TD> #OLAE#:D
</TD></TR><TR><TD> #OLAE#:4 </TD><TD> #OLAE#:F </TD><TD> #OLAE#:E
</TD></TR></TBODY></TABLE><P><B>transformation</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH>  transformed plasmid or ligation product ID </TH><TH> strain </TH><TH> description
</TH></TR><TR><TD> 3 </TD><TD> #OLAE#:3 </TD><TD> DH5alpha </TD><TD> AP19 Poly (from #OX3R#) in pSB1C3
</TD></TR><TR><TD> 4 </TD><TD> #OLAE#:4 </TD><TD> BL21 Gold DE3 </TD><TD> AP19 Poly (from #OX3R#) in pSB1K3
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Tasks_for_following_days">Tasks for following days</SPAN></H3><UL><LI> masterplates and overnight cultures of transformations !!!red-white screening!!! (see above)  (sunday, 15-08-30) <B>done</B></LI><LI> plasmid purification and cryos of overnights cultures  (monday, 15-08-31)</LI><LI> test digest for screening (monday, 15-08-31)</LI><LI> strategic decission on what to send in for sequencing (monday, 15-08-31)</LI><LI> is it possible to have all required BioBrick and expression plasmids sequenced and validated on wednesday? If yes, then we' ll have to concentrate on generating relieable data. If not, a small group has to focus on cloning, until it is finished. (monday, 15-08-31)</LI></UL><H2><SPAN class="mw-headline" id="15-08-31">15-08-31</SPAN></H2><H3><SPAN class="mw-headline" id="restriction_ligation_of_AP19.Poly_Biobrick_parts">restriction ligation of AP19.Poly Biobrick parts</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> construct  </TH><TH> clone </TH><TH> Cryo ID </TH><TH> Plasmid ID </TH><TH> Plasmid conc. [ng/µl] </TH><TH> result
</TH></TR><TR><TD> AP19.Poly in pSB1C3; DH5alpha </TD><TD>  #1 </TD><TD> #BL11#  </TD><TD> #ATYH#  </TD><TD> 155,5   </TD><TD> neg
</TD></TR><TR><TD>  #2  </TD><TD> #AR3W# </TD><TD> #PEQT#  </TD><TD>  114,0 </TD><TD> neg
</TD></TR><TR><TD> #3  </TD><TD> #C8QO#  </TD><TD> #ZBZT# </TD><TD>  103,0 </TD><TD> neg
</TD></TR><TR><TD> #4 </TD><TD> #AKYK#  </TD><TD> #11AZ#  </TD><TD>  200,5 </TD><TD> neg
</TD></TR><TR><TD> #5 </TD><TD> #KAKW# </TD><TD> #OWF9#  </TD><TD> 192,5  </TD><TD> neg
</TD></TR><TR><TD> #6 </TD><TD> #AWSE#  </TD><TD> #M1DW#  </TD><TD> 144,5  </TD><TD> neg
</TD></TR><TR><TD> #7 </TD><TD>  #XS6A# </TD><TD> #ZCZN#  </TD><TD>  194,5 </TD><TD> neg
</TD></TR><TR><TD>  #8</TD><TD> #ZW99# </TD><TD> #4CCM#  </TD><TD>  154,0 </TD><TD> neg
</TD></TR><TR><TD> AP19.Poly in pSB1K3; BL21 Gold </TD><TD> #1 </TD><TD> #8PL8#  </TD><TD> #81KE#  </TD><TD> 286,5  </TD><TD> neg
</TD></TR><TR><TD> #2 </TD><TD> #L6PR#  </TD><TD> #WARS# </TD><TD>  201,0 </TD><TD> neg
</TD></TR><TR><TD> #3 </TD><TD> #LM4Y#  </TD><TD> #ZW1Y#  </TD><TD> 207,0  </TD><TD> neg
</TD></TR><TR><TD> #4 </TD><TD> #KOKW# </TD><TD> #DLNH#  </TD><TD> 237,0 </TD><TD> neg
</TD></TR><TR><TD> #5 </TD><TD> #Y8NN# </TD><TD> #S98X# </TD><TD> 182,0 </TD><TD> neg
</TD></TR><TR><TD> #6 </TD><TD> #9VS1#  </TD><TD> #Q49D# </TD><TD> 231,0  </TD><TD> neg
</TD></TR><TR><TD> #7 </TD><TD> #FYOP#  </TD><TD> #FXHO#  </TD><TD>  221,5 </TD><TD> neg
</TD></TR><TR><TD> #8 </TD><TD> #FM6X# </TD><TD> #RF3B#  </TD><TD>  207,5 </TD><TD> neg
</TD></TR></TBODY></TABLE><UL><LI>for screening of construct, do test digest with SpeI and XbaI</LI><LI>just negative clones of Poly and Mdh construct</LI></UL><P><B>Just negative results in test digest</B> never just cut X,S for cloning 
</P><H4><SPAN class="mw-headline" id="new_restriction_ligation">new restriction ligation</SPAN></H4><P><B>digest</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> plasmid </TH><TH> Enzymes </TH><TH> Water added [µl]
</TH></TR><TR><TD> #FCBA#:8 </TD><TD> #OX3R# </TD><TD> E + P </TD><TD> 24.0
</TD></TR><TR><TD> #FCBA#:7 </TD><TD> #AQY3# </TD><TD> E + P </TD><TD> 21.4
</TD></TR></TBODY></TABLE><UL><LI>also use digest #OLAE#:A (J04450 in pSB1C3 #CD8B# digest with E + P)</LI><LI>check digest on gel</LI></UL><P><B>Ligation</B>:
</P><UL><LI> 20 µl reaction volume</LI><LI> 6 µl part digest</LI><LI> 2 µl backbone digest</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> # </TH><TH> part digest </TH><TH> backbone digest
</TH></TR><TR><TD> Y </TD><TD> #FCBA#:8 </TD><TD> #OLAE#:A
</TD></TR><TR><TD> Z </TD><TD> #FCBA#:8 </TD><TD> #FCBA#:7
</TD></TR></TBODY></TABLE><UL><LI> ligation products in #FCBA#</LI></UL><H2><SPAN class="mw-headline" id="15-09-01">15-09-01</SPAN></H2><H3><SPAN class="mw-headline" id="Summary_2">Summary</SPAN></H3><P>Our aim is to get AP19.Poly in the BioBrick backbones pSB1C3 and pSB1K3. Later this day we can make masterplates and overnight cultures of the AP19 clones in both backbones. Tomorrow plasmid purification and test digest will follow.
</P><P>In the evening of wednesday we will prepare sequencing. We need positive clones of:
</P><UL><LI> AP19.Poly in pSB1C3 </LI><LI> AP19.Poly in pSB1K3</LI></UL><P><B>TODO for today</B></P><UL><LI>Night: AP19.Poly in K3 and AP19.Poly in C3 Masterplates and overnights</LI></UL><H2><SPAN class="mw-headline" id="15-09-02">15-09-02</SPAN></H2><P><B>TODO</B></P><UL><LI>Plasmid purification of all constructs</LI><LI>screening of all constructs with test digest </LI><LI>prepare sequencing for thursday morning</LI></UL><H4><SPAN class="mw-headline" id="Constructs_for_screening">Constructs for screening</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> construct  </TH><TH> clone </TH><TH> Cryo ID </TH><TH> Plasmid ID </TH><TH> Plasmid conc. [ng/µl] </TH><TH> result
</TH></TR><TR><TD> AP19.Poly in pSB1C3; DH5alpha </TD><TD>  #1 </TD><TD> ##  </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD>  #2  </TD><TD> ## </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> #3  </TD><TD> ##  </TD><TD> ## </TD><TD> neg
</TD></TR><TR><TD> #4 </TD><TD> ##  </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> #5 </TD><TD> ## </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD>  #6 </TD><TD> ##  </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD>  #7  </TD><TD> ## </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> #8  </TD><TD> ##  </TD><TD> ## </TD><TD> neg
</TD></TR><TR><TD> #9 </TD><TD> ##  </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> #10 </TD><TD> ## </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD>  #11 </TD><TD> ##  </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD>  #12  </TD><TD> ## </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> #13  </TD><TD> ##  </TD><TD> ## </TD><TD> neg
</TD></TR><TR><TD> #14 </TD><TD> ##  </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> #15 </TD><TD> ## </TD><TD> ##  </TD><TD> neg
</TD></TR><TR><TD> AP19.Poly in pSB1K3; BL21 </TD><TD>  #1 </TD><TD> #SCDM#  </TD><TD> #LX9X#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD>  #2  </TD><TD> #4EW8# </TD><TD> #PAFZ#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #3  </TD><TD> #8XQ9#  </TD><TD> #C6NZ# </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #4 </TD><TD> #XH9V#  </TD><TD> #16DW#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #5 </TD><TD> #8WRB# </TD><TD> #4P1T#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD>  #6 </TD><TD> #QMKT#  </TD><TD> #SBOC#  </TD><TD> test via SDS
</TD></TR><TR><TD>  #7  </TD><TD> #8NWN# </TD><TD> #VLZ1#  </TD><TD> test via SDS
</TD></TR><TR><TD> #8  </TD><TD> #VA9S#  </TD><TD> #SFKE# </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #9 </TD><TD> #48BC#  </TD><TD> #C3OX#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #10 </TD><TD> #BOSP# </TD><TD> #VC9Y#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD>  #11 </TD><TD> #B6ML#  </TD><TD> #VQBD#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD>  #12  </TD><TD> #CKFD# </TD><TD> #EQBA#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #13  </TD><TD> #YHQT#  </TD><TD> #QD9R# </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #14 </TD><TD> #QAMS#  </TD><TD> #43DT#  </TD><TD> no succesfull overnight
</TD></TR><TR><TD> #15 </TD><TD> #R9D1# </TD><TD> #KZ6P#  </TD><TD> no succesfull overnight
</TD></TR></TBODY></TABLE><UL><LI>overnights of AP19.Poly in pSB1K3 didn't grow the night, so they were made again at 15:30</LI><LI>all purified plasmids of AP19.Poly in K3 are sceened by sequencing with one primer</LI><LI>ONE succesfull overnight of AP19.Poly in K3</LI></UL><H2><SPAN class="mw-headline" id="15-09-03">15-09-03</SPAN></H2><UL><LI>from the AP19.Poly in pSB1K3 constructs just the overnight of #6 grew, these will be prepared for plasmid purification and tests (test digest/PCR)</LI><LI>as restriction ligation strategy for biobrick constructs seems to be not reliable, we try Gibson Assembly for AP19.Poly in C3/K3</LI></UL><H4><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> Reaction Tube </TH><TH> Description </TH><TH> H2O </TH><TH> Backbone [µl]  </TH><TH> Insert [µl] </TH><TH> Gibson MM </TH><TH> transformed in </TH><TH> result
</TH></TR><TR><TD> A </TD><TD>Poly in K3 neg control </TD><TD> 12,4 </TD><TD> 0,8 (#PQMO#) </TD><TD> 1,9 (#OX3R#) </TD><TD> 0 </TD><TD> BL21 Gold DE3 </TD></TR><TR><TD> B </TD><TD>Poly in K3 </TD><TD> 4,9 </TD><TD> 0,8 (#PQMO#) </TD><TD> 1,9 (#OX3R#)  </TD><TD> 7,5 </TD><TD> BL21 Gold DE3 </TD></TR><TR><TD> C </TD><TD>Poly in C3 neg control </TD><TD> 12,1 </TD><TD> 1,0 (#6WCN#) </TD><TD> 1,9 (#OX3R#) </TD><TD> 0 </TD><TD> DH5alpha </TD></TR><TR><TD> D </TD><TD>Poly in C3 </TD><TD> 4,6 </TD><TD> 1,0 (#6WCN#) </TD><TD> 1,9 (#OX3R#) </TD><TD> 7,5 </TD><TD> DH5alpha </TD></TR></TBODY></TABLE><UL><LI>products stored in #D6LK#</LI></UL><H2><SPAN class="mw-headline" id="15-09-04">15-09-04</SPAN></H2><H4><SPAN class="mw-headline" id="Gibson_Assembly_transformation">Gibson Assembly transformation</SPAN></H4><UL><LI>The Gibson Assembly masterplates show also clones when no Gibson MM was used (just transformation of linar [?] DNA parts), so we can't trust the colonies on the plates with mastermix
</LI><LI>DpnI digest seems not to work, so the ligation product was digested with DpnI again (1 µl of Schwaneberg DpnI, 1 µl of S2 DpnI, all afternoon at 37 °C)</LI><LI>after 9 h of incubation, constructs were transformed in BL21 (AP19.Poly in pSB1K3-constructs) and DH5alpha (AP19.Poly in pSB1C3-constructs)</LI></UL><H2><SPAN class="mw-headline" id="15-09-05">15-09-05</SPAN></H2><H3><SPAN class="mw-headline" id="AP19.POLY_in_pSB1K3_check_PCR">AP19.POLY in pSB1K3 check PCR</SPAN></H3><UL><LI> Templates: 
</LI><LI> 1: #SBOC#</LI><LI> 2: #VLZ1#</LI></UL><LI> KAPA 2G Mastermix</LI><TABLE class="wikitable"><TBODY><TR><TH> Mastermix </TH><TH> Primerpair </TH><TH> expected length [bp]
</TH></TR><TR><TD> A </TD><TD> #A9W9# and #HSP2# </TD><TD> 1804
</TD></TR><TR><TD> B </TD><TD> #HSP1# and #XSP3# </TD><TD> 1939
</TD></TR><TR><TD> C </TD><TD> #XSP2# and #XE3D# </TD><TD> 2157
</TD></TR></TBODY></TABLE><P>PCR program:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> T [°C] </TH><TH> time
</TH></TR><TR><TD> denature </TD><TD> 95 </TD><TD> 0:30
</TD></TR><TR><TD> denature </TD><TD> 95 </TD><TD> 0:15
</TD></TR><TR><TD> anneal </TD><TD> 61 </TD><TD> 0:15
</TD></TR><TR><TD> elongate </TD><TD> 72 </TD><TD> 0:35
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 2:00
</TD></TR><TR><TD> store </TD><TD> 16 </TD><TD> forever
</TD></TR></TBODY></TABLE><P>Summary and results:
</P><TABLE class="wikitable"><TBODY><TR><TH> ID </TH><TH> Primerpair </TH><TH> template </TH><TH> expected length [bp] </TH><TH> result
</TH></TR><TR><TD> 1A </TD><TD> #A9W9# and #HSP2# </TD><TD> #SBOC# </TD><TD> 1804 </TD><TD>  wrong band
</TD></TR><TR><TD> 1B </TD><TD> #HSP1# and #XSP3# </TD><TD> #SBOC# </TD><TD> 1939 </TD><TD>  wrong band
</TD></TR><TR><TD> 1C </TD><TD> #XSP2# and #XE3D# </TD><TD> #SBOC# </TD><TD> 2157 </TD><TD>  wrong band
</TD></TR><TR><TD> 2A </TD><TD> #A9W9# and #HSP2# </TD><TD> #VLZ1# </TD><TD> 1804 </TD><TD>   no bands
</TD></TR><TR><TD> 2B </TD><TD> #HSP1# and #XSP3# </TD><TD> #VLZ1# </TD><TD> 1939 </TD><TD> no bands
</TD></TR><TR><TD> 2C </TD><TD> #XSP2# and #XE3D# </TD><TD> #VLZ1# </TD><TD> 2157 </TD><TD>  no bands
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Gibson_Assembly_2">Gibson Assembly</SPAN></H4><UL><LI>trafo plates looks better after long incubation with DpnI, only very few red colonies</LI><LI>15 overnights of each construct and more clones on trafo plates </LI></UL><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> clones  </TH><TH> result
</TH></TR><TR><TD> AP19.Poly in pSB1K3 </TD><TD> #1 </TD><TD> -
</TD></TR><TR><TD> #2 </TD><TD> -
</TD></TR><TR><TD> #3 </TD><TD> -
</TD></TR><TR><TD> #4 </TD><TD> will be sequenced
</TD></TR><TR><TD> #5</TD><TD> will be sequenced
</TD></TR><TR><TD> #6 </TD><TD> will be sequenced
</TD></TR><TR><TD> #7</TD><TD> will be sequenced
</TD></TR><TR><TD> #8 </TD><TD> -
</TD></TR><TR><TD> #9 </TD><TD> -
</TD></TR><TR><TD> #10 </TD><TD> will be sequenced
</TD></TR><TR><TD> #11  </TD><TD> -
</TD></TR><TR><TD> #12 </TD><TD> -
</TD></TR><TR><TD> #13 </TD><TD> -
</TD></TR><TR><TD> #14 </TD><TD> -
</TD></TR><TR><TD> #15 </TD><TD> -
</TD></TR><TR><TD> AP19.Poly in pSB1C3 </TD><TD> #1 </TD><TD> -
</TD></TR><TR><TD> #2 </TD><TD> will be sequenced
</TD></TR><TR><TD> #3 </TD><TD> -
</TD></TR><TR><TD> #4 </TD><TD> will be sequenced
</TD></TR><TR><TD> #5</TD><TD> -
</TD></TR><TR><TD> #6 </TD><TD> will be sequenced
</TD></TR><TR><TD> #7</TD><TD> -
</TD></TR><TR><TD> #8 </TD><TD> -
</TD></TR><TR><TD> #9 </TD><TD> -
</TD></TR><TR><TD> #10 </TD><TD> -
</TD></TR><TR><TD> #11  </TD><TD> -
</TD></TR><TR><TD> #12 </TD><TD> -
</TD></TR><TR><TD> #13 </TD><TD> will be sequenced
</TD></TR><TR><TD> #14 </TD><TD> -
</TD></TR><TR><TD> #15 </TD><TD> will be sequenced
</TD></TR></TBODY></TABLE><UL><LI>in general: many clones that look good on gel, 5 of each construct will be sequenced (two 5 ml overnights)</LI></UL><H2><SPAN class="mw-headline" id="15-09-06">15-09-06</SPAN></H2><H3><SPAN class="mw-headline" id="plasmid_purification_and_test_digests:">plasmid purification and test digests:</SPAN></H3><UL><LI> only use 1.5 ml of culture
</LI><LI> ID distribution of temporary prepped plasmids for analytic purposes:
</LI><LI> AP19 Poly in K3: PK 1-15</LI><LI> AP19 Poly in C3: PC 1-15</LI></UL><LI> store rest of the cultures in 4 °C room</LI><LI> do test digest with EcoRI and PstI of:
</LI><LI> AP19 Poly in K3 &quot;minipreps&quot; of overnigths from Gibson Assembly</LI><LI> AP19 Poly in C3 &quot;minipreps&quot; of overnights from Gibson Assembly</LI><LI> wait for results, then purify remaining culture volume (this time with plasmid purification kit) of positive clones and prepare for sequencing.</LI><LI> use masterplates to inoculate new liquid cultures
</LI><LI> maybe we need more volume to purifiy</LI><H4><SPAN class="mw-headline" id="test_digest">test digest</SPAN></H4><UL><LI>Master mix for 34 reactions </LI><LI>calculated DNA conc.: 25 ng/µl (new plasmid purification protocol, can't measure concentrations)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Component </TH><TH> amount [µl]
</TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> 170,2
</TD></TR><TR><TD> EcoRI</TD><TD> 7,4
</TD></TR><TR><TD> PstI </TD><TD> 7,4
</TD></TR><TR><TD>10x NEB 2.1 </TD><TD>  37
</TD></TR></TBODY></TABLE><UL><LI>to each tube, 6µl mastermix and 4 µl of DNA were added</LI></UL><H4><SPAN class="mw-headline" id="plasmid_purification_with_qiagen_kit">plasmid purification with qiagen kit</SPAN></H4><UL><LI> AP19 Poly in K3: 4, 5, 6, 7, 10</LI><LI> AP19 Poly in C3: 2, 6, 12, 13, 15</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> ID </TH><TH> construct </TH><TH> plasmid ID </TH><TH> cryo ID
</TH></TR><TR><TD> PK 4 </TD><TD> AP19 Poly in pSB1K3 from Gibson Assembly </TD><TD> #FRNW# </TD><TD> #MEPN#
</TD></TR><TR><TD> PK 5 </TD><TD> #QVDN# </TD><TD> #BCCB#
</TD></TR><TR><TD> PK 6 </TD><TD> #Q3CN# </TD><TD> #OHMA#
</TD></TR><TR><TD> PK 7 </TD><TD> #9V18# </TD><TD> #O9HS#
</TD></TR><TR><TD> PK 10 </TD><TD> #FO14# </TD><TD> #B4KL#
</TD></TR><TR><TD> PC 2 </TD><TD> AP19 Poly in pSB1C3 from Gibson Assembly </TD><TD> #N9H9# </TD><TD> #6PEY#
</TD></TR><TR><TD> PC 6 </TD><TD> #91RC# </TD><TD> #HT1M#
</TD></TR><TR><TD> PC 12 </TD><TD> #P6AM# </TD><TD> #XX6C#
</TD></TR><TR><TD> PC 13 </TD><TD> #1DWH# </TD><TD> #TC4A#
</TD></TR><TR><TD> PC 15 </TD><TD> #PZHC# </TD><TD> #RMNC#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-09-07">15-09-07</SPAN></H2><UL><LI>all plasmids of polycistronic in K3 and C3 purified into 60 µl elution buffer (10 ml cell culture) </LI><LI>send in sequencing of #QVDN#, #9V18#; #91RC#, #1DWH# ALL NEGATIVE</LI><LI>again, overnights of Poly in K3 4, 5, 6, 7, and Poly in C3 2, 6, 12, 13, and 15 were made to put them on SDS (check if all genes are is expressed)</LI></UL><H2><SPAN class="mw-headline" id="15-09-08">15-09-08</SPAN></H2><P><B>SDS Gel</B>
of AP19.Poly in pSB1K3 and pSB1C3 
</P><TABLE class="wikitable"><TBODY><TR><TH> Construct  </TH><TH> Result on SDS Gel
</TH></TR><TR><TD> Poly in C3 #2 </TD><TD> -
</TD></TR><TR><TD> Poly in C3 #6 </TD><TD> -
</TD></TR><TR><TD> Poly in C3 #12 </TD><TD> -
</TD></TR><TR><TD> Poly in C3 #13</TD><TD>  -
</TD></TR><TR><TD> Poly in K3 #5 </TD><TD> -
</TD></TR><TR><TD> Poly in K3 #6</TD><TD> -
</TD></TR><TR><TD> Poly in K3 #7</TD><TD>  -
</TD></TR></TBODY></TABLE><UL><LI>Gel staining didn't work properly, so no reliable results can be made</LI></UL><H2><SPAN class="mw-headline" id="15-09-09">15-09-09</SPAN></H2><UL><LI> Sequencing of #QVDN#, #9V18#; #91RC#, #1DWH# is all negative)</LI></UL><H4><SPAN class="mw-headline" id="Test_PCR">Test PCR</SPAN></H4><P>for screening of all purified plasmids of AP19.Poly in pSB1C3 and pSB1K3
</P><UL><LI>make 2 mastermixes with two different primer pairs
</LI><LI>Primers A: #A9W9# + #PSP2# ; expected lenght 3164 bp</LI><LI>Primers B: #HSP1# + #XE3D# ; expected lenght 3990 bp</LI></UL><LI>KAPA 2G Mastermix PCR with 0,5 µl of purified plasmid as template</LI><P><B>Programme</B></P><TABLE class="wikitable"><TBODY><TR><TH>  Time </TH><TH> Temp
</TH></TR><TR><TD> 3:00 </TD><TD> 95
</TD></TR><TR><TD> 0:25 </TD><TD> 95
</TD></TR><TR><TD> 0:20 </TD><TD> 61
</TD></TR><TR><TD> 1:00 </TD><TD>  72
</TD></TR><TR><TD> 2:00 </TD><TD> 72
</TD></TR></TBODY></TABLE><P><B>Constructs</B></P><TABLE class="wikitable"><TBODY><TR><TH><B>Template</B></TH><TH> #FRNW#</TH><TH> #CVDN# </TH><TH> #Q3CN# </TH><TH> #9V18# </TH><TH> #FO14# </TH><TH> #N9H9# </TH><TH> #91RC# </TH><TH> #P6AM# </TH><TH> #1DWH# </TH><TH> #PZHC#
</TH></TR><TR><TD><B>PCR tubes</B></TD><TD> 1A, 1B </TD><TD> 2A, 2B </TD><TD> 3A, 3B </TD><TD> 4A, 4B </TD><TD> 5A, 5B  </TD><TD> 6A, 6B </TD><TD> 7A, 7B </TD><TD> 8A, 8B  </TD><TD> 9A, 9B </TD><TD> 10A, 10B
</TD></TR><TR><TD><B>result</B></TD><TD> no bands
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="alkaline_phosphatase_digestion">alkaline phosphatase digestion</SPAN></H4><P><B>Last try (!!!) to get polycistronic construct into a BioBrick vector</B><I>just as much work as absulutely necessary will be invested in this workflow</I></P><P>Added 1 µl of phosphatase to #OLAE#:D and #OLAE#:E. Incubation for 20' at 37 °C.
</P><P>Ligation with #OLAE#:F. Product: #9VCN#: I (in pSB1C3) + II (in pSB1K3)
</P><P>Transformation of 5 µl into BL21 Gold (DE3)
</P><H2><SPAN class="mw-headline" id="15-09-11">15-09-11</SPAN></H2><UL><LI>new purified plasmids of #9PBD# (MDH in pSB1C3), #H4M9# (HPS in pSB1C3) and #KFEY# (XPK in pSB1C3)</LI><LI>overnights of AP19.Poly in pSB1C3 that grew were all red, in these in pSB1K3 did not grow</LI><LI>new overnights were made from master plate at 10:00 am</LI></UL><H2><SPAN class="mw-headline" id="15-09-12">15-09-12</SPAN></H2><UL><LI>even though there were colonies on the LB plate from the transformation, the clones didn't grow in a liquid LB overnight cultures!</LI><LI>all 15 clones didn't grow in overnight culture, even after several trials!</LI></UL><DL><DD>SEEMS LIKE SUCCESSFULL TRANSFORMATIONS OF POLYCISTRONIC CONSTRUCT IN BIOBRICK BACKBONE CAN'T GROW </DD></DL><DL><DD>NO POLYCISTRONIC CONSTRUCT IN BIOBRICK BACKBONE WILL BE SEND TO THE REGISTRY!</DD></DL><H1><SPAN class="mw-headline" id="References">References</SPAN></H1></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>