<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Glycogen_Single_Expression skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Glycogen Single Expression</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmGlycogenSynthesisII" href="/Team:Aachen/Lab/Glycogen/Synthesis/II._Single_Expression"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Single Expression</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><H2><SPAN class="mw-headline" id="15-06-09">15-06-09</SPAN></H2><P>digest glgA/B plasmids (<B>stored in #SZKD#</B>)
</P><TABLE class="wikitable"><TBODY><TR><TH> name </TH><TH> plasmid ID </TH><TH> enzyme 1 </TH><TH> enzyme 2
</TH></TR><TR><TD> A2 (glgA) </TD><TD> #M1M6# </TD><TD> EcoRI </TD><TD> PstI
</TD></TR><TR><TD> B2 (glgB) </TD><TD> #YHT3# </TD><TD> EcoRI </TD><TD> PstI
</TD></TR><TR><TD> pSB1K30   </TD><TD> #PCFX# </TD><TD> EcoRI </TD><TD> PstI
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-10">15-06-10</SPAN></H2><UL><LI> after digest glg A and B:
</LI><LI> A and B will be ligated in pSB1K30 (ligation tubes: A2 and B2)</LI><LI><B>stored in #QHHR#</B></LI></UL><LI> transformation into DH5α </LI><H2><SPAN class="mw-headline" id="15-06-11">15-06-11</SPAN></H2><UL><LI> master plates of glgA, glgB in pSB1K30 (LB+K)</LI></UL><H2><SPAN class="mw-headline" id="15-06-12">15-06-12</SPAN></H2><UL><LI> colony PCR on glgA/B constructs to identify positive clones</LI><LI> make overnight cultures of glgA/B constructs</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> clone number </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> glgA in pSB1K30 #2 </TD><TD> #3DPV# </TD><TD> #D6VB# </TD><TD> mRFP
</TD></TR><TR><TD> glgA in pSB1K30 #3 </TD><TD> #4LCM# </TD><TD> #RLDN# </TD><TD> mRFP
</TD></TR><TR><TD> glgB in pSB1K30 #5 </TD><TD> #6YBE# </TD><TD> #ATE6# </TD><TD> pT7 is missing
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-13">15-06-13</SPAN></H2><UL><LI> make cryos of glgA/B constructs (table above)</LI><LI> plasmid prep of glgA, glgB (table above)</LI></UL><H2><SPAN class="mw-headline" id="15-06-15">15-06-15</SPAN></H2><UL><LI> measured concentrations of purified glgA/glgB constructs</LI><LI> heat shock of glgA (#D6VB#, #RLDN#) and glgB (#ATE6#) in pSB1K30 into BL21(DE3)</LI></UL><H2><SPAN class="mw-headline" id="15-06-16">15-06-16</SPAN></H2><UL><LI> BL21 transformation master plates (glgB transformation was not successful)</LI></UL><H2><SPAN class="mw-headline" id="15-06-17">15-06-17</SPAN></H2><P>Digestions with 40' digest and 20' inactivation
</P><P>(previous digests A2, B2, pSB1K30 were discarded from #SZKD# because agarose gels were really ugly)
</P><TABLE class="wikitable"><TBODY><TR><TH> gene </TH><TH> enzymes </TH><TH> expectation </TH><TH> observation </TH><TH> products
</TH></TR><TR><TD> glgA (#M1M6#) </TD><TD> EcoRI + PstI </TD><TD> ... </TD><TD> confirmed </TD><TD> #SZKD#:A2
</TD></TR><TR><TD> glgB (#YHT3#) </TD><TD> EcoRI + PstI </TD><TD> ... </TD><TD> confirmed </TD><TD> #SZKD#:B2
</TD></TR><TR><TD> pSB1K30 (#PCFX#) </TD><TD> EcoRI + PstI </TD><TD> ... </TD><TD> confirmed but weak </TD><TD> #SZKD#:K30
</TD></TR></TBODY></TABLE><P>
Ligations and transformations
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> part I </TH><TH> part II </TH><TH> backbone </TH><TH> product </TH><TH> host
</TH></TR><TR><TD> glgA in pSB1K30 </TD><TD> #SZKD#:A2 </TD><TD> - </TD><TD> #SZKD#:K30 </TD><TD> #99MC#:LA </TD><TD> BL21
</TD></TR><TR><TD> glgB in pSB1K30 </TD><TD> #SZKD#:B2 </TD><TD> - </TD><TD> #SZKD#:K30 </TD><TD> #99MC#:LB </TD><TD> BL21
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-18">15-06-18</SPAN></H2><UL><LI> make master plates and liquid cultures of transformations (glgA in pSB1K30, glgB in pSB1K30; <B>6 liquid, 12 plate</B>)</LI></UL><H2><SPAN class="mw-headline" id="15-06-19">15-06-19</SPAN></H2><P>Colony PCR on glgA,B,C constructs
</P><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> glgA in pSB1K30 clones 1-12 </TD><TD> 49-60 </TD><TD> 1770 </TD><TD> 2 </TD><TD> 1'50&quot; </TD><TD> three okay
</TD></TR><TR><TD> glgB in pSB1K30 clones 1-12 </TD><TD> 61-72 </TD><TD> 2523 </TD><TD> 3 </TD><TD> 3'05&quot; </TD><TD> all negative
</TD></TR></TBODY></TABLE><P>Cryos
</P><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> glgA in pSB1K30 #2 </TD><TD> #VWSS# </TD><TD> #FHTH# </TD><TD> -
</TD></TR><TR><TD> glgA in pSB1K30 #3 </TD><TD> #W8VB# </TD><TD> #K8DM# </TD><TD> perfect
</TD></TR><TR><TD> glgA in pSB1K30 #6 </TD><TD> #4TPL# </TD><TD> #K3EO# </TD><TD> good, but possibly one (unlikely) insertion
</TD></TR></TBODY></TABLE><P>Master plates of glgA, glgB clones that were white under green light.
</P><H2><SPAN class="mw-headline" id="15-06-22">15-06-22</SPAN></H2><UL><LI> prep good plasmids of glgA (table above)</LI><LI> pipette glgA in pSB1K30 for sequencing (#A9W9# and #XE3D#)</LI><LI> colony PCR on glgA, glgB plates from Saturday (#A9W9# and #XE3D#)</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clones </TH><TH> tube number </TH><TH> expected length </TH><TH> machine </TH><TH> elongation time </TH><TH> result
</TH></TR><TR><TD> glgA in pSB1K30 clones 13-15 </TD><TD> 12-14 </TD><TD> 1821 </TD><TD> 2 </TD><TD> 1'50&quot; </TD><TD> all three are good
</TD></TR><TR><TD> glgB in pSB1K30 clones 13-15 </TD><TD> 15-17 </TD><TD> 2572 </TD><TD> 3 </TD><TD> 3'05&quot; </TD><TD> clone 14 is good
</TD></TR></TBODY></TABLE><UL><LI> overnight cultures of good clones from the glgB colony PCR (LB+K)</LI></UL><H2><SPAN class="mw-headline" id="15-06-23">15-06-23</SPAN></H2><UL><LI> cryos and plasmid prep from overnight cultures of glgB</LI><LI> the plasmid prep went wrong and resulted in concentrations &lt; 10 ng/µl</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> glgB in pSB1K30 #14 </TD><TD> #Q6Z1# </TD><TD> #KK8K# </TD><TD> confirmed ligation sites
</TD></TR></TBODY></TABLE><UL><LI> overnight cultures of glgB in pSB1K30 from the master plate</LI></UL><H2><SPAN class="mw-headline" id="15-06-24">15-06-24</SPAN></H2><UL><LI> in the morning: freeze overnight culture of glgB</LI><LI> in the afternoon: plasmid prep of glgB (re-used the existing ID from the table of Monday)</LI><LI> pipette glgB for sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-06-26">15-06-26</SPAN></H2><P>Digestions
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1A2 </TD><TD> glgC </TD><TD> #F8Y8# </TD><TD> EcoRI, PstI </TD><TD> #LVVK#:1 </TD><TD> subcloning into pSB1C3, pSB1K30
</TD></TR><TR><TD> pSB1K30 </TD><TD> mRFP </TD><TD> #1K30# </TD><TD> EcoRI, PstI </TD><TD> #LVVK#:4 </TD><TD> target backbone for expression
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-29">15-06-29</SPAN></H2><P>Ligation and Transformation (first try)
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1K30 </TD><TD> glgC </TD><TD> #LVVK#:1 </TD><TD> - </TD><TD> #LVVK#:4 </TD><TD> #RRCV#:2 </TD><TD> BL21 Gold (DE3) </TD><TD> for expression tests
</TD></TR></TBODY></TABLE><P>&gt;&gt; #LVVK#:4 (pSB1K30) looked bad on agarose gel! Do the ligation again with pSB1K30 from a different tube!
</P><P>Ligation (second try, only 1 µl of K30)
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1K30 </TD><TD> glgC </TD><TD> #LVVK#:1 </TD><TD> - </TD><TD> #SZKD#:K30 </TD><TD> #RRCV#:2 </TD><TD> BL21 Gold (DE3) </TD><TD> for expression tests
</TD></TR></TBODY></TABLE><UL><LI> transformation is planned for the day after</LI></UL><H2><SPAN class="mw-headline" id="15-06-30">15-06-30</SPAN></H2><UL><LI> heat shock of K1585312 in pSB1K30 ligation product (5 µl #RRCV#:2) into 40 µl BL21 Gold (DE3)</LI></UL><H2><SPAN class="mw-headline" id="15-07-01">15-07-01</SPAN></H2><UL><LI> master plate of K1585312 in pSB1K30 (BL21 Gold DE3)</LI></UL><H2><SPAN class="mw-headline" id="15-07-02">15-07-02</SPAN></H2><UL><LI> colony PCR of K1585312 in pSB1K30 (desired: 1683, negative: 1227)</LI><LI><S>overnights of good clones</S></LI></UL><P>colony PCR with KAPA 2G
</P><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 95 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> annealing </TD><TD> 58 </TD><TD> 0'15&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'25&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 2'00&quot;
</TD></TR></TBODY></TABLE><P>
Digestions
</P><TABLE class="wikitable"><TBODY><TR><TH> BioBrick in Backbone </TH><TH> gene </TH><TH> template ID </TH><TH> enzymes </TH><TH> product </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1A2 </TD><TD> glgC </TD><TD> #F8Y8# </TD><TD> EcoRI, PstI </TD><TD> #LVVK#:1 </TD><TD> subcloning into pSB1K30
</TD></TR><TR><TD> pSB1K30 </TD><TD> mRFP </TD><TD> #4K19# </TD><TD> EcoRI, PstI </TD><TD> #LVVK#:4 </TD><TD> target backbone for expression
</TD></TR></TBODY></TABLE><P>Ligation
</P><TABLE class="wikitable"><TBODY><TR><TH> construct </TH><TH> gene </TH><TH> Part I </TH><TH> Part II </TH><TH> Backbone </TH><TH> products </TH><TH> host </TH><TH> purpose
</TH></TR><TR><TD> K1585312 in pSB1K30 </TD><TD> glgC </TD><TD> #LVVK#:1 </TD><TD> - </TD><TD> #LVVK#:4 </TD><TD> #RRCV#:2B </TD><TD> BL21 Gold (DE3) </TD><TD> for expression tests
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-03">15-07-03</SPAN></H2><UL><LI><S>cryo cultures</S></LI><LI><S>plasmid prep</S></LI><LI><S>pipette for sequencing</S></LI></UL><H2><SPAN class="mw-headline" id="15-07-06">15-07-06</SPAN></H2><UL><LI><S>transformation of #RRCV#:2B into BL21 Gold (DE3)</S></LI><LI> decided to use pSB1A30 because the difference between induced and non-induced pSB1K30 could not be seen on SDS gels</LI></UL><P>PCR 1 to subclone glgC into pSB1A30: backbone amplification
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> purified PCR product ID </TH><TH> product ID </TH><TH> tube #
</TH></TR><TR><TD> I13507 in pSB1A30 </TD><TD> #LYZH# </TD><TD> #TSTV# </TD><TD> #FYTO# </TD><TD> 2206 </TD><TD> #T6X9# </TD><TD> #MRVW#:III </TD><TD>3+4
</TD></TR></TBODY></TABLE><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 2'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 64 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 1'10&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-08">15-07-08</SPAN></H2><UL><LI><S>master plate (LB+K+IPTG) of glgC in pSB1K30 transformations</S></LI></UL><P>PCR2 to subclone glgC into pSB1A30: glgC amplification
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> template </TH><TH> forward primer </TH><TH> reverse primer </TH><TH> product length </TH><TH> product ID
</TH></TR><TR><TD> glgC in pSB1A2 </TD><TD> #F8Y8# </TD><TD> #O3M6# </TD><TD> #NBZZ# </TD><TD> 1360 </TD><TD> #MRVW#: I
</TD></TR></TBODY></TABLE><P>PfuS PCR
</P><P>conditions for <B>30 cycles</B>:
</P><TABLE class="wikitable"><TBODY><TR><TH> step </TH><TH> temperature [°C] </TH><TH> duration
</TH></TR><TR><TD> initial denaturation </TD><TD> 98 </TD><TD> 3'00&quot;
</TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> annealing </TD><TD> 58 </TD><TD> 0'30&quot;
</TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 0'45&quot;
</TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 5'00&quot;
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-09">15-07-09</SPAN></H2><P>PCR product purification
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR product </TH><TH> purified PCR product
</TH></TR><TR><TD> #MRVW#:I </TD><TD> #XNHT#
</TD></TR><TR><TD> #MRVW#:III </TD><TD> #T6X9#
</TD></TR></TBODY></TABLE><P>CPEC assembly of glgC in pSB1A30
</P><TABLE class="wikitable"><TBODY><TR><TH> component </TH><TH> volume [µl]
</TH></TR><TR><TD> water </TD><TD> 5.1
</TD></TR><TR><TD> Q5 buffer </TD><TD> 5
</TD></TR><TR><TD> DMSO </TD><TD> 0.75
</TD></TR><TR><TD> dNTPs (2 mM) </TD><TD> 5
</TD></TR><TR><TD> Q5 polymerase </TD><TD> 0.5
</TD></TR><TR><TD> pSB1A30 fragment (#T6X9#) </TD><TD> 3.9
</TD></TR><TR><TD> glgC with prefix/suffix #XNHT# </TD><TD> 4.8
</TD></TR><TR><TD> elongation time </TD><TD> 1'47&quot;
</TD></TR><TR><TD> product stored in </TD><TD> #SWMB#:II
</TD></TR></TBODY></TABLE><P>5 µl #SWMB#:II were heat shocked into BL21 Gold (DE3) and plated on LB+A
</P><H2><SPAN class="mw-headline" id="15-07-10">15-07-10</SPAN></H2><UL><LI> make masterplates (LB+A) and overnights of glgC in pSB1A30</LI></UL><H2><SPAN class="mw-headline" id="15-07-11">15-07-11</SPAN></H2><UL><LI> cryo cultures </LI><LI> freeze culture</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> gene / clone </TH><TH> cryo ID </TH><TH> plasmid ID </TH><TH> sequencing result
</TH></TR><TR><TD> B0034.glgC in pSB1A30 #1 </TD><TD> #HXXD# </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #2 </TD><TD> #D9TA# </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #3 </TD><TD> #PX1E# </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #4 </TD><TD> #KKYF# </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #5 </TD><TD> #M9EY# </TD><TD> - </TD><TD> -
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #6 </TD><TD> #OHES# </TD><TD> #VMN3# </TD><TD> looks perfect
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #9 </TD><TD> #S6KX# </TD><TD> #YLS8# </TD><TD> no T7 promoter
</TD></TR><TR><TD> B0034.glgC in pSB1A30 #10 </TD><TD> #YNNY# </TD><TD> #8MKM# </TD><TD> no T7 promoter
</TD></TR></TBODY></TABLE><UL><LI> all clones except #6 were red</LI></UL><H2><SPAN class="mw-headline" id="15-07-13">15-07-13</SPAN></H2><UL><LI> plasmid prep of glgC in pSB1A30 clone #6 in #VMN3#</LI><LI> new overnights of white clones on masterplate: clone #9 and #10</LI></UL><H2><SPAN class="mw-headline" id="15-07-14">15-07-14</SPAN></H2><UL><LI> plasmid prep of glgC in pSB1A30 clone #9+10 (table above)</LI><LI> all three glgC in pSB1A30 plasmids went into sequencing</LI></UL><H2><SPAN class="mw-headline" id="15-07-15">15-07-15</SPAN></H2><UL><LI> analysis of sequencing results</LI><LI> 2 precultures of glgC in pSB1A30 from clone #6, one of RFP in pSB1A30 #6DA3#</LI></UL><H1><SPAN class="mw-headline" id="Expression">Expression</SPAN></H1><H2><SPAN class="mw-headline" id="15-06-30_2">15-06-30</SPAN></H2><P><B>glgA</B>
Plated #W8VB# and #KK16# on LB+K
</P><H2><SPAN class="mw-headline" id="15-07-01_2">15-07-01</SPAN></H2><UL><LI> make overnight precultures from the plate (2x #W8VB#, 1x #KK16# -&gt; A,B,C)</LI><LI> reserve 3x 500 ml shake flasks</LI></UL><H2><SPAN class="mw-headline" id="15-07-02_2">15-07-02</SPAN></H2><P><B>08:50</B> inoculate 50 ml main cultures LB+K (labelled A,B,C) with 1 ml of culture
</P><P>monitor the OD and induce cultures A+B with IPTG at OD=0.6
</P><P><B>5 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><P><B>10 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><P><B>24 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><TABLE class="wikitable"><TBODY><TR><TH> flask </TH><TH> combination </TH><TH> inoculated with (time) </TH><TH> IPTG induction (time) </TH><TH> 6 h sample </TH><TH> 18.5 h sample </TH><TH> pellet
</TH></TR><TR><TD> A </TD><TD> K1585310 in pSB1K30 induced </TD><TD> 1 ml #W8VB# preculture (08:50) </TD><TD> 50 µl IPTG (15:10) </TD><TD> A6, 20:45 #9E9N# </TD><TD> #TX14# </TD><TD> #F3Y6#
</TD></TR><TR><TD> B </TD><TD> K1585310 in pSB1K30 uninduced </TD><TD> 1 ml #W8VB# preculture (08:50) </TD><TD> not induced </TD><TD> B6, 20:45 #EL3O# </TD><TD> #X4KT# </TD><TD> #KBPA#
</TD></TR><TR><TD> C </TD><TD> I13507 in pSB1K30 induced </TD><TD> 1 ml #KK16# preculture (08:50) </TD><TD> 50 µl IPTG (15:10) </TD><TD> C6, 20:45 #9KA9# </TD><TD> #81RD# </TD><TD> #ZCWT#
</TD></TR></TBODY></TABLE><P>OD monitoring (numbers show time in hours)
</P><TABLE class="wikitable"><TBODY><TR><TH> flask/ OD </TH><TH> 1  </TH><TH>  2</TH><TH> 3.167 </TH><TH> 5.333</TH><TH> 6.333 </TH><TH> 7.5 </TH><TH> 8.5
</TH></TR><TR><TD> A </TD><TD> 0.052 </TD><TD> 0.069 </TD><TD> 0.084 </TD><TD> 0.261 </TD><TD> 1.07 </TD><TD> 1.728 </TD><TD> 1.689
</TD></TR><TR><TD> B </TD><TD> 0.056 </TD><TD> 0.077 </TD><TD> 0.094 </TD><TD> 0.314 </TD><TD> 1.241 </TD><TD> 1.834 </TD><TD> 1.811
</TD></TR><TR><TD> C </TD><TD> 0.024 </TD><TD> 0.027 </TD><TD> 0.036  </TD><TD> 0.116 </TD><TD> 0.406 </TD><TD> 1.178 </TD><TD> 1.225
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-03_2">15-07-03</SPAN></H2><UL><LI> take the final sample</LI><LI> prepare SDS-PAGE (cooked samples in #B13F#)</LI><LI> run SDS-PAGE</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS-PAGE of GlgA expression</B>GlgA was expressed in pSB1K30 and compared to mRFP expression. The small arrows point to the GlgA bands of 52.4 kDa. No difference between induced and uninduced was observed. After overnight expression the strong bands were still present.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="15-07-06_2">15-07-06</SPAN></H2><P><B>glgB</B> Plated #Q6Z1# on LB+K
</P><H2><SPAN class="mw-headline" id="15-07-07">15-07-07</SPAN></H2><UL><LI> make overnight precultures from the plate (2x Q6Z1, (1x KK16) -&gt; A,B,C)</LI><LI> reserve 3x 500 ml shake flasks</LI></UL><H2><SPAN class="mw-headline" id="15-07-08_2">15-07-08</SPAN></H2><P><B>08:55</B> inoculate 50 ml main cultures LB+K (labelled A,B,C) with preculture to an OD of 0.1
</P><P>monitor the OD and induce cultures A+B with IPTG at OD=0.6
</P><P><B>5 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><P><B>10 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><P><B>24 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><TABLE class="wikitable"><TBODY><TR><TH> flask </TH><TH> combination </TH><TH> inoculated with (time) </TH><TH> IPTG induction (time) </TH><TH> 5 h sample </TH><TH> 19.5 h sample </TH><TH> pellet
</TH></TR><TR><TD> A </TD><TD> K1585311 in pSB1K30 induced </TD><TD> 2.5 ml #Q6Z1# preculture (08:50) </TD><TD> 50 µl IPTG (13:40) </TD><TD> A1, 18:50 #B9CL# </TD><TD> #6W3P# </TD><TD> #WP6P#
</TD></TR><TR><TD> B </TD><TD> K1585311 in pSB1K30 uninduced </TD><TD> 2.5 ml #Q6Z1# preculture (08:50) </TD><TD> not induced </TD><TD> B1, 18:50 #SKMW# </TD><TD> #NM6Q# </TD><TD> #ASLE#
</TD></TR><TR><TD> C </TD><TD> I13507 in pSB1K30 induced </TD><TD> 4.5 ml #KK16# preculture (08:50) </TD><TD> 50 µl IPTG (13:40) </TD><TD> C1, 18:50 #9SZH# </TD><TD> #QEHO# </TD><TD> #1CAP#
</TD></TR></TBODY></TABLE><P>OD monitoring (numbers show time in hours)
</P><TABLE class="wikitable"><TBODY><TR><TH> flask/ OD </TH><TH> 1  </TH><TH>  2.25 </TH><TH> 3.2 </TH><TH> 4.2 </TH><TH> 5 </TH><TH> 8.33 </TH><TH> 9 </TH><TH> 24.2
</TH></TR><TR><TD> A </TD><TD> 0.147 </TD><TD> 0.190 </TD><TD> 0.285 </TD><TD> 0.481 </TD><TD> 0.666 </TD><TD> 2.95 </TD><TD> 2.15 </TD><TD> 2.5
</TD></TR><TR><TD> B </TD><TD> 0.147 </TD><TD> 0.198 </TD><TD> 0.311 </TD><TD> 0.552 </TD><TD> 0.785 </TD><TD> 2.9 </TD><TD> 2.42 </TD><TD> 2.6
</TD></TR><TR><TD> C </TD><TD> 0.100 </TD><TD> 0.108 </TD><TD> 0.109 </TD><TD> 0.115 </TD><TD> 0.111 </TD><TD> 0.64 </TD><TD> 1.41 </TD><TD> 2.0
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-09_2">15-07-09</SPAN></H2><UL><LI> cooked SDS samples in #6SOX#</LI><LI> run SDS page</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS-PAGE of GlgB expression</B>GlgB was expressed in pSB1K30 and compared to mRFP expression. The small arrows point to the glgB bands of 84 kDa. No difference between induced and uninduced was observed.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="15-07-15_2">15-07-15</SPAN></H2><UL><LI><B>glgC</B> overnight of clone # from Masterplate of glgC in pSB1A30</LI></UL><H2><SPAN class="mw-headline" id="15-07-16">15-07-16</SPAN></H2><P><B>09:15</B> inoculate 25 ml main cultures LB+A (labelled A,B,C) with preculture to an OD of 0.1
</P><P>monitor the OD and induce cultures A+B with IPTG at OD=0.6
</P><P><B>5 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><P><B>10 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><P><B>24 h after induction</B> from each culture take 2x 2 ml sample (centrifuge, discard supernatant and freeze)
</P><TABLE class="wikitable"><TBODY><TR><TH> flask </TH><TH> combination </TH><TH> inoculated with (time) </TH><TH> IPTG induction (time) </TH><TH> 5 h sample </TH><TH> 19.5 h sample </TH><TH> pellet
</TH></TR><TR><TD> A </TD><TD> K1585312 in pSB1A30 induced </TD><TD> 0.65 ml #OHES# (from master plate) preculture A (09:15) </TD><TD> 25 µl IPTG (13:05) </TD><TD> A1, 18:10 #NMHL# </TD><TD> A19, 09:00 #ZXSL# </TD><TD> #AQND#
</TD></TR><TR><TD> B </TD><TD> K1585312 in pSB1A30 uninduced </TD><TD> 0.65 ml #OHES# (from master plate) preculture A (09:15) </TD><TD> not induced </TD><TD> B1, 18:10 #WDQZ# </TD><TD> B19, 09:00 #BSVL# </TD><TD> #XROO#
</TD></TR><TR><TD> C </TD><TD> I13507 in pSB1A30 induced </TD><TD> 0.37 ml #6DA3# preculture (09:15) </TD><TD> 25 µl IPTG (11:30) </TD><TD> C1, 18:10 #H4XD# </TD><TD> C19, 09:00 #EQMY# </TD><TD> #LDVR#
</TD></TR></TBODY></TABLE><P>OD monitoring (numbers show time in hours)
</P><TABLE class="wikitable"><TBODY><TR><TH> flask/ OD </TH><TH> 1 </TH><TH> 2.25 </TH><TH> 3 </TH><TH> 4 </TH><TH> 6 </TH><TH> 9 </TH><TH> 24
</TH></TR><TR><TD> A </TD><TD> 0.132 </TD><TD> 0.199 </TD><TD> 0.293 </TD><TD> 0.469 </TD><TD> 2.8 </TD><TD> 4.21, 4.21, 4.21 </TD><TD> 5.72, 6.03, 5.65
</TD></TR><TR><TD> B </TD><TD> 0.126 </TD><TD> 0.202 </TD><TD> 0.291 </TD><TD> 0.463 </TD><TD> 3.6 </TD><TD> 4.34, 4.32, 4.35 </TD><TD> 5.12, 4.91, 5.05
</TD></TR><TR><TD> C </TD><TD> 0.254 </TD><TD> 1.031 </TD><TD> 1.612 </TD><TD> 4.1 </TD><TD> 7.9 </TD><TD> 5.39, 5.41, 5.42 </TD><TD> 5.38, 5.20, 5.12
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-07-17">15-07-17</SPAN></H2><UL><LI> cooked SDS samples of glgC expressio are in #FWCT#</LI><LI> SDS PAGE of glgC expression tests</LI></UL><DIV class="figraw figure-medium col-md-6"><TABLE><TBODY><TR><TD><B>SDS PAGE of glgC expression</B>glgC was expressed in pSB1A30 and compared to mRFP expression. The small arrows point to the GlgC bands of 48 kDa. No difference between induced and uninduced was observed. After overnight expression the strong bands were no longer present.
</TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>