Team:Aachen/Lab/Glycogen/Knockouts
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Knockout Generation
Plasmid Construction
How Glycogen Degradation Works
The glycogen degradation in E. coli is performed by two enzymes:
glycogen phosphorylase (encoded by glgP)
debranching enzyme (encoded by glgX)
GlgP removes glucose units from the end of linear glycogen chains while the debranching enzyme GlgX cleaves α-1,6-glycosidic bonds, therefore degrading branches.
Genome Editing using Cas9
When it became clear that we had to knock out glgX and glgP, we began looking for an efficient and reliable method to generate the knockout strains. Only a few weeks ago, at the beginning of April, a new method for genome editing in Escherichia coli was published [1] .
The method is based on a two-plasmid system, in which the first plasmid, the pCas, contains the cas9 gene and a temperature-sensitive repA101ts ori. To increase recombination efficiency in E. coli, it also expresses lambda-Red genes behind an inducible arabinose-promoter. Upon induction with IPTG, a sgRNA is transcribed to cut the pMB1 ori of the second plasmid.
Plasmid map of pCas
pCas contains a repA101ts ori for self-curing, arabinose-induced lambda-Red genes, lacI and araC repressor genes, the Cas9 gene, IPTG-induced sgRNA targeting pMB1 and a kanamycin resistance gene.
The other essential plasmid for this method is the pTargetT, which leads the Cas9 protein to the DNA part to be cut. It provides the sgRNA for targeting the genomic sequence as well as the repair template with a high-copy pMB1 ori. The repair templates consist of two 400 bp homology arms for homology-directed repair.
General plasmid map of a targeting plasmid
A sgRNA with the N20-sequence targeting a protospacer in the genome is expressed behind a J23119 strong constitutive promoter. Next to the ampR gene and the pMB1 ori, the plasmid carries the repair template that is introducing the desired edit.
a
Plasmid Construction
The CRISPR/Cas9 system offers a variety of genome editing applications such as disruption, insertion and deletion. It works with a single guide RNA (sgRNA) which includes 20 base pairs that are complementary to a specific DNA region that has to be cut; the N20 sequence. It can be any 20-30 base pair DNA sequence behind a PAM site, the Protospacer-Adjacent-Motif. Those can be found almost everywhere in the genome.
In our project we tested two strategies:  for the debranching enzyme we cleaved the gene at the beginning of the coding sequence, using one sgRNA. In addition, we designed a short DNA sequence (ATAGGCGAAATGTAAGCGGCGTGCCATGAAAAC) that contains a stop codon in each frame and translates to IGEM*AACHEN in the first frame, called iGEM Aachen MultiSTOP. This was inserted between the two homology arms in the repair template. For the glycogen phosphorylase we used two sgRNAs to cut glgP on both sides to completely eliminate this gene.
Targeting plasmid glgX
To target glgX, we selected a high-scoring CRISPR-site with the sequence 5'-GTAATGCGCGCCGAGGGGAGCGG ca. 25 bp downstream of the glgX start codon. In addition to the two homology arms in the repair template, we designed a short DNA sequence (ATAGGCGAAATGTAAGCGGCGTGCCATGAAAAC) that contains a stop codon in each frame and translates to IGEM*AACHEN in the first frame. This is flanked by the two homology arms.
Targeting plasmid glgP
For glgP we used two high scoring CRISPR-sites to eliminate the whole gene.
a
After initial problems with the spectinomycin resistance for pTargetT, we decided to build the targeting plasmid in a pSB1A3 BioBrick backbone. The targeting plasmids for glgX and glgP were assembled via CPEC from synthesized gBlocks, PCR-amplified pSB1A3 backbone and fragments amplified from the E. coli genome. Our results showed that the single cut strategy has a remarkably higher efficiency. For the glgP knockout, additional arabinose induction of the lambda-red genes was necessary for high efficiency. Ultimately, both strategies led to a successful gene knockout.
a
Results
construct
resistance
plasmid ID
cryo ID
pCas
Kan
#EANB#
#YPKO#
anti glgX in pSB1A3
Amp
#1DEN#
#BAB3#
anti glgP in pSB1A3
Amp
#C1QW#
#M69X#
a
Laboratory Notebook
a
Knockout Generation
After the construction of the targeting plasmids for glgP and glgX was completed, the process of generating the knockouts began. At first, the pCas was heatshocked into E.coli BL21 Gold (DE3) and NEB10β cells. To enable the introduction of targeting plasmids, we made these cells electrocompetent [2] . Following the electroporation of pTargetT for glgX and glgP, many clones were screened by genomic PCRs and sent into sequencing for verification. Succeeding the curing of both plasmids through IPTG induction and temperature change, we continued working with our new  BL21 Gold (DE3) knockout strains combined with the methanol- and the glycogen polycistronic construct.
We worked on the generation of glgX and glgP single knockouts in both strains as well as on a double knockout. However, we stopped working with NEB10β cells early on in the project, because our polycistronic glgCAB construct was built behind a T7 promoter, so it could only be expressed in BL21 Gold (DE3) cells.
Results
Both, the glgP and glgX knockouts in BL21 Gold (DE3) were successfully generated and the iGEM Aachen MultiSTOP was inserted in the genome.  We observed that the single cut knockout for glgX had a higher efficiency when clones were screened. However, after additional induction with arabinose the glgP knockout was successfully completed as well. In NEB10β we not only did the single knockouts, but also performed a double knockout of both glycogen degrading enzymes.
Edited genomic region of glgP with iGEM Aachen MultiSTOP
A DNA sequence coding for IGEM*AACHEN in amino acids was inserted in the repair template. As shown in the chromatograms, it is now written in the E. coli genome.
strain
cryo with pCas
cryo without plasmids (LB)
cryo without plasmids (M9)
NEB10β ΔglgX (1 mutation in the Multi Stop)
#PH3R#
#CL6W#
NEB10β ΔglgP
#XHEK#
#BXR1#
NEB10β ΔglgX ΔglgP
#LNL4#
#WHPX#
BL21 Gold (DE3) ΔglgX
#ZPKP#
#C81O#
#XFWX# (confirmed again by sequencing)
BL21 Gold (DE3) ΔglgP
#3RZ1#
#FVNV#
#4BW3#
Laboratory Notebook
a
References
↑ Jiang Y, Chen B, Duan C, Sun B, Yang J, Yang S. Multigene editing in the Escherichia coli genome via the CRISPR-Cas9 system. Appl Environ Microbiol. 2015 Apr;81(7):2506-14. doi: 10.1128/AEM.04023-14. Epub 2015 Jan 30. PubMed PMID:25636838; PubMed Central PMCID: PMC4357945.
↑ http://openwetware.org/wiki/Electrocompetent_Cells#Materials
▲
Contact
CC-BY-SA iGEM Aachen 2015
