Team:Aachen/Lab/Methanol/Polycistronic Expression Plasmid
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
In order to create a strain that is able to metabolize methanol, we wanted to build a polycistronic plasmid with all required genes for methanol conversion in E. coli.
To build this multi fragment construct, the assembly method had to be chosen carefully. BioBrick Assembly is ineffective and inappropriate for such complex elements. Using CPEC or related methods, you can hardly control the order of the assembly. Thus, we had to search for an alternative that is suitable for our concept.
All methanol conversion genes in a polycistronic frame
The general design of a polycistronic methanol uptake plasmid.
(Mdh = methanol dehydrogenase, Hps = 3-hexulose-6-phosphate, Phi = 6-phospho-3-hexuloisomerase, Xpk = phosphoketolase)
To face this challenge, we used the RDP cloning standard . First, RDP parts out of mdh, hps, phi and xpk with the necessary extensions were created.  These fragments were joined together with BBa_J23119 in a backbone with kanamycin resistance following the RDP Assembly method. This strong constitutive promotor from the Anderson Promoter Library was chosen because methanol conversion should be possible at every stage of bacterial growth.  After sequence validation of the construct, the polycistronic plasmid was characterized.
In silico work
Suitable primers to create RDP parts out of mdh, hps, phi and xpk coding sequences had to be designed. We used the software "Geneious" to make primers with an annealing region and corresponding overhangs. GC content and annealing temperature were carefully chosen. These overhangs contain BsaI restriction sites. By BsaI digest, the Z- respectively X- ends of a part can be created. They are necessary for the assembly via the RDP Assembly method.
Overview of RDP Assembly of the polycistronic construct
Assembly order of polycistronic construct. pSB1KRDP anchor #FO4B# and cap #OZD1# connect to form a circular plasmid.
Wetlab work
To make the coding sequences compatible with the RDP standard, they were amplified via high fidelity PCR with the primers that we designed previously. The templates were the coding sequences upstream to B0034 in the pSB1C3 backbone ( BBa_1585210 , BBa_K1585211 , BBa_K1585212 , BBa_K1585213 ). The overlapping region for the annealing of the parts were created by BsaI digest. For shorter parts like promoters, we ordered oligos that could be annealed to form short RDP parts. The fragments were mixed in an adequate ratio, and cooled down from 80 °C to room temperature in a linear gradient.
BioBrick
corresponding RDP part
tubefront ID
BBa_J23119
Z-AP19-X'
#XYD9#
BBa_K1585210
X-B0034.mdh-Z'
#PRDW#
BBa_K1585211
Z-B0034.hps-X'
#VO4C#
BBa_K1585212
X-B0034.phi-Z'
#ZALV#
BBa_K1585213 + BBa_B0015
Z-B0034.xpk.B0015-X'
#ZR1Q#
Results
We successfully assembled all required genes in a polycistronic frame behind an AP19 promoter in a backbone with kanamycin resistance. This was confirmed through sequencing results.
A SDS gel showed that all four genes are expressed.  Next, we tested the physiology and the enzyme activity of a strain that carries our construct.
Expression test of polycistronic plasmid on SDS-Page
The cell pellet of E. coli BL21 Gold (DE3) that carries the polycistronic construct downstream of Anderson Promoter 19 was applied on an SDS-PAGE.
Successful sequencing of polycistronic construct
We intensively tried to clone the polycistronic construct into the pSB1C3 BioBrick backbone. After trials via CPEC , restriction and ligation with different sets of enzymes, Gibson Assembly, and ligation after alkaline phosphatase digest, we decided to abandon this project. Our assumption is that a strain that carries the polycistronic expression frame in a BioBrick backbone is just not able to survive in a overnight culture, because clones from the LB plate after transformation never grew in a overnight culture.
Laboratory Notebook: Building the Polycistronic Plasmid via RDP Assembly
Laboratory Notebook: Building BioBricks out of polycistronic plasmid
References
▲
Contact
CC-BY-SA iGEM Aachen 2015
