Team:Aachen/Notebook/Documentation/Glycogen Targeting Plasmids
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Plasmid Construction
Laboratory Notebook
In silico work
15-05-05
designed pTargetT for glgX knock-out
Gibson Assembly with 3 fragments:
gRNA synthesized (no amplification required for CPEC)
backbone with kanamycin resistance :
to be amplified by PCR from pTargetF
editing template with homology arms and Insert (IGEM*AACHEN)
to be assembled from genomic PCR of left and right homology arm and primers with overhangs (IGEM AACHEN)
15-05-06
decided to use CPEC assembly method
4 (5) fragements: backbone, left homology arm with IGEM AACHEN overhang, right homology arm with IGEM AACHEN overhang, sgRNA (+sgRNA right for two double-strand breaks)
designed overhangs and primers for genomic PCR and backbone PCR
glgP
PCR
forward primer
Melting temperature fwd
reverse primer
Melting temperature
expected length of PCR product  [bp]
genomic PCR left arm
#WQRS#
66.2 °C
#PF1Y#
66.9°C
422
genomic PCR right arm
#KQCX#
65.3°C
#COT4#
65.3°C
461
pTargetF backbone
#Z9T8#
63.2 °C
#BBDC#
59.1°C
1950
pSB1A3 backbone #93QQ#
#8E3Q#
73.3 °C
#WVZQ#
67.5°C
2205
amplification of sgRNA left (synthesized with overhang)
NOT REQUIRED #PXMQ#
65.8°C
NOT REQUIRED #YESR#
65.6°C
205
amplification of sgRNA right (synthesized with overhang)
NOT REQUIRED #A4DX#
68.2°C
NOT REQUIRED #VRBM#
69.3°C
252
glgX
PCR
forward primer
Melting temperature fwd
reverse primer
Melting temperature
expected length of PCR product
genomic PCR left arm
#BQBK#
65.5 °C
#HPO4#
66.5°C
433
genomic PCR right arm
#SOZQ#
66.0°C
#K4TH#
66.1°C
446
pTargetF backbone
#Z9T8#
63.2 °C
#BBDC#
59.1°C
1950
pSB1A3 backbone #93QQ#
#8E3Q#
73.3 °C
#WVZQ#
67.5°C
2205
amplification of sgRNA (synthesized with overhang)
NOT REQUIRED #PXMQ#
65.8°C
NOT REQUIRED #QEV4#
65.5°C
208
glgX and glgP
PCR
forward primer
Melting temperature fwd
reverse primer
Melting temperature
expected length of PCR product
glgP and backbone fragment
#8E3Q#
...
#4HNZ#
...
3437
glgX fragment
#RTT6#
...
#K4TH#
...
1008
15-05-07
checked primers for mismatches in the genome
ordered primers for genomic PCR and backbone amplification (primer for snythesized gRNA not required, enough template for CPEC is synthesized)
made a calculation table for CPEC (required volumes/concentrations...)
useful tool to calculate molecular masses of DNA molecules 1
remember to choose double-stranded under "treat sequences"
Wetlab Documentation
15-05-05
the plasmids pCas and pTargetF were extracted from half of the filter paper into #WATT# and #VMLL#
The remaining half of the filter paper is in #NCKV# (pCas) and #9OLA# (pTargetF)
produced kanamycin stocks
produced LB-Medium for agar plates (LB+K)
make agar plates with kanamycin from 1L LB-Medium
electroporation of pCas (#WATT#) and pTargetF (#VMLL#) into electrocompetent NEB10beta - we plated both on LB+K which is why pTargetF clones did not survive. Incubation at 30 °C
NEVER HAVE pCas OVER 30 °C!!
15-05-06
Transformation of pTargetF did not work, no colonies
(maybe cultivate next time not at 30 °C, but at 37 °C)
make Master Plate and overnight cultures of pCas (three colonies: 1, 2, 3)
15-05-07
make new overnight cultures (Start 9:30) from master plate, three times (1, 2, 3) --> did not grow fast enough to make plasmid prep in the evening
plasmid prep of non shaked overnight cultures, low concentrations (1: 45.5 ug/ul   2: 22.2 ug/ul   3: 34.0 ug/ul)
cryos of non shaked overnight cultures
pCas clone (non-shaked) #
purified plasmid ID
cryo ID
1
#F4VY#
#RF9Q#
2
#KP48#
#R9WX#
3
#HXDA#
#HFQM#
make new overnight cultures for tomorrow (from master plate)
15-05-08
cryo cultures from new pCas cultures
plasmid preps from pCas cultures
pCas clone (new) #
purified plasmid ID
cryo ID
1
#EANB#
#YPKO#
2
#ORLH#
#SD9P#
3
#PFOK#
#FSRQ#
test digestion with EcoRI - expecting three bands three strong bands (11044, 8452, 6939, 5618, 4105, 1513)
the gel (1kb ladder, #ORLH#, #PFOK#, #EANB#) was really ugly, but it seems like #ORLH# and #EANB# look good. The digests are frozen in #1XZ1# and should be put on a gel again
15-05-11
put the pCas digests (#1XZ1#) on the gel for a second time - the 1 % gel in the bio6 was really ugly (100 V, 35 min)
iLM and iSL ran another gel in the bio7 - 1.2 %, 100 V, 43 min and the gel looked much better:
(#EANB# and #ORLH# are okay, #PFOK# is bad)
electroporation of 2 µl pTargetF into 20 µl NEB10b - plate on LB+S
15-05-12
spectinomycin is not working -> prepare plates with higher concentrations tomorrow
plated E.coli DH5α from cryo stock onto LB- an incubate at 37 °C
15-05-13
making plates with our spectinomycin 1000x stock as well as the 1000x stock from the bio7 (to compare both spectomycin 'sources')
plate DH5α on the new LB+S plates to check wether they work now
designed 32 sequencing primers for pCas
gradient PCRs for the amplification of pTargetF backbone using the primers #Z9T8# + #BBDC# (Tm 63.2/59.1) and the template #VMLL#
products stored in #L4X3#
expected length: 1950 bp (ladder 1 kb ruler, gel looks very good!)
15-05-14
the LB+S plates from our own spectinomycin stock inhibited DH5α growth, but did not stop it. The spectinomycin stock from the bio7 was better, but still not 100 %.
new LB+S plates were prepared from both S-stocks but this time with twice the concentrations (LB+S new/old x2)
DH5α plated on the both LB+S x2 plates
Prep of the pTargetF gradient PCR backbone product (#L4X3#) into #CFWR# and #LLFO#
colony-gradient PCRs colony-gradient PCRs for the amplification of homology arms
conditions for 25 cycles:
step
temperature [°C]
duration
initial denaturation
98
3'00"
denaturation
98
0'10"
gradient annealing
60-71
0'30"
elongation
72
0'15"
final elongation
72
5'00"
glgX left homology arm using #BQBK# + #HPO4# (Tm 65.5/65.5) - products stored in #P11R# - not a single band on the gel
glgX right homology arm using #SOZQ# + #K4TH# (Tm 66.1/66.0) - products stored in #EBMW# - not a single band on the gel either =(
discarded the PCR products
possible problems: elongation time too short? Annealing time too short? more likely: no accessible template DNA (because we didn't have any unspecific bands either) - sequence mismatch between DH5α and K-12 is unlikely, because DH5α is a derivative of K-12 .
15-05-15
DH5α survived the double concentration on the LB+S x2 plates. A bit less than 1x but still there's growth. We're tired of making even higher concentrations. Instead we will make control transformations in parallel when transforming pTargetF. For the pTargetT constructs we decided to build them in a pSB1C3 backbone instead. (the ori-targeting works there too!)
modifying the genomic PCRs manual, because it was unsuccessful:
centrifuge cells for 1 min at 11 000
resuspend cells in 50 µl of water
centrifuge 1'00" at full speed
measure the DNA concentration (135.8 ng/µl)
resuspend cells
98 °C for 5'00"
centrifuge 1'00" at full speed
measure the DNA concentration (3246.8 ng/µl)
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
98
3'00"
denaturation
98
0'30"
gradient annealing
55-71
0'30"
elongation
72
0'20"
final elongation
72
5'00"
Template: #ZDPA# (genomic DNA) for glgX left homology arm using #BQBK# + #HPO4# (Tm 65.5/65.5) -> products stored in #BE1R# -> the gel shows one band of the correct length in the 57.6 °C annealing temperature tube
15-05-18
Primer #KQCX# was not in the shipment last week. Placing a new order at Eurofins
with the results from Friday, do colony-gradient PCRs for the amplification of homology arms
glgX left homology arm using #BQBK# + #HPO4# (Tm 65.5/65.5)
PfuS PCR with 5 samples (gradient 56.6-58.6)
2x template concentration #ZDPA# (10 µL instead of 5 µL)
Products in #BVNW#
glgX right homology arm using #SOZQ# + #K4TH# (Tm 66.1/66.0)
PfuS PCR with 10 samples (gradient 55-71)
2x template concentration #ZDPA#
products in #FQCM#
15-05-19
annotated lots of gel pictures
PCR-prep the glgX and glgP (left) homology arms
PCR to amplify the left glgP homology arm using #WQRS# + #PF1Y# (Tm 66.2/66.9)
PfuS PCR with 9 samples (gradient 55-71)
2x template concentration #ZDPA#
products in #KE38#
strong band at 56.1 and medium at 63.8 °C
PCR product purification
fragment
from
tubes
prepped into
glgX left homology arm
#BVNW#
2
#11BX#
glgX right homology arm
#FQCM#
4+5+6
#WXRD#
glgP left homology arm
#KE38#
3+6
#EEO6#
15-05-20
measured concentrations of yesterdays PCR products
still waiting for the primers
15-05-21
still waiting for primers
15-05-22
genomic PCR to amplify the only remaining homology arm
glgP using #KQCX# + #COT4# (Tm 65.3/65.3)
gradient PCR to amplify the pSB1A3 backbone
PCR-prep the backbone and homology arms
elute the sgRNA gBlocks according to the protocol (20 µl water, 50 °C for 20')
15-05-26
genomic PCR to amplify the only remaining homology arm
glgP using #KQCX# + #COT4# (Tm 65.3/65.3)
PfuS polymerase
10x 25 µl reactions
double template concentration (#ZDPA#)
products in #HFLQ#
conditions for 25 cycles:
step
temperature [°C]
duration
initial denaturation
98
3'00"
denaturation
98
0'30"
gradient annealing
55-71
0'30"
elongation
72
0'15"
final elongation
72
5'00"
gradient PCR to amplify the pSB1A3 backbone
diluted #93QQ# template
primers #WVZQ# and #8E3Q#
PfuS polymerase
5x 50 µl reactions
products in #HODO#
conditions for 25 cycles:
step
temperature [°C]
duration
initial denaturation
98
0'30"
denaturation
98
0'30"
gradient annealing
62-70
0'30"
elongation
72
1'15"
final elongation
72
5'00"
PCR-prep the backbone and homology arms
elute the sgRNA gBlocks according to the protocol (20 µl water, 50 °C for 20')
15-05-27
repeat both PCRs with narrower gradients
gradient PCR to amplify the pSB1A3 backbone
diluted #93QQ# template
primers #WVZQ# and #8E3Q#
PfuS polymerase
5x 50 µl reactions
products in #PF8P#
conditions for 25 cycles:
step
temperature [°C]
duration
initial denaturation
98
0'30"
denaturation
98
0'30"
gradient annealing
67.2-69.2
0'30"
elongation
72
1'30"
final elongation
72
5'00"
genomic PCR to amplify the only remaining homology arm
glgP using #KQCX# + #COT4# (Tm 65.3/65.3)
PfuS polymerase
10x 25 µl reactions
double template concentration (#ZDPA#)
products in #4ADM#
conditions for 30 cycles:
step
temperature [°C]
duration
initial denaturation
98
3'00"
denaturation
98
0'30"
gradient annealing
60-66
0'30"
elongation
72
0'20"
final elongation
72
5'00"
PCR product purification (from #PF8P#)
fragment
from
tubes
prepped into
glgP right homology arm
#4ADM#
3+6+8+9
#MWH3#
pSB1A3 backbone
#PF8P#
1+2
#RW11#
pSB1A3 backbone
#PF8P#
3+4
#TRTN#
make a pCas preculture from the plate
other stuff is higher priority - we'll do this next week
prepare buffers for electrocompetent cells
15-05-28
Diluted the sgRNA gBlocks in 20 µl AE buffer (#RFZK#, #6XLF#, #ZVAC#).
CPEC for anti glgX/P targeting constructs in pSB1A3
component
µl for anti glgX
µl for anti glgP
water
11
9.8
Q5 buffer
5
5
DMSO
0.75
0.75
dNTPs (2 mM)
5
5
Q5 polymerase
0.5
0.5
pSB1A3 backbone fragment (#TRTN#)
1.2
1.2
sgRNA 1
0.9 #RFZK#
0.9 #6XLF#
sgRNA 2
none
1.1 #ZVAC#
left homology arm
0.3 #11BX#
0.5 #EEO6#
right homology arm
0.3 #WXRD#
0.2 #MWH3#
elongation time
1'36" (1'46")
1'42" (1'46")
products stored in
#PXR4#
#PXR4#
Heat shock transformation of CPEC product (#PXR4#:A), ligation product (#QZFH#) into DH5α cells
trafo program in the cycler: 4 °C for 30', 42 °C for 60", 4 °C for 5'
number
description
id
5
anti glgX in pSB1A3 CPEC
#PXR4#:X
6
anti glgP in pSB1A3 CPEC
#PXR4#:P
15-05-29
Working with new clones
construct
colonies
red [%]
anti glgX
1448
50 %
anti glgP
1240
92 %
master plates of anti glgX/P CPEC clones
15-05-30
colony PCR
template
clone numbers
expected
good clones
anti glgX
1-12
1351
10, 11, 2, 3, 4
anti glgP
1-10
1574
1, 4, 5, 6, 7, 8
15-05-31
overnight cultures
construct
clone number
cryo-id
plasmid-id
sequencing result
anti glgX
2
#RKS1#
anti glgX
3
#1NT8#
#3OSZ#
a copy of the right end was inserted before the terminator
anti glgX
4
#BAB3#
#1DEN#
perfect! not a single mutation
anti glgX
10
#DNMN#
#MANY#
two point mutations in the N20 sequence
anti glgX
11
#WCNN#
anti glgP
1
#XQM8#
anti glgP
4
#M69X#
#C1QW#
sequencing was difficult, looks good
anti glgP
5
#SXAX#
#DC6X#
sequencing was difficult, point mutation at pos. 499
anti glgP
6
#D6CE#
#HCTL#
sequencing was difficult, looks good
anti glgP
7
#R4LZ#
#4MCW#
sequencing was difficult, looks good
anti glgP
8
#CS39#
15-06-01
plasmid preps of anti glgX and anti glgP plasmids (see the table above)
pipeted the plasmids for sequencing with the priemrs #A9W9#, #XE3D#, #WQRS#
15-06-05
pipetting anti glgP for sequencing with 225 ng/kb plasmid template and 2 µl of 20 µl #PF1Y# primer
15-06-10
analysis of sequencing results: see table above!
Results
construct
resistance
plasmid ID
cryo ID
pCas
Kan
#EANB#
#YPKO#
anti glgX in pSB1A3
Amp
#1DEN#
#BAB3#
anti glgP in pSB1A3
Amp
#C1QW#
#M69X#
Plasmid map of glgX-targeting plasmid
A sgRNA with the N20-sequence 5'-GTAATGCGCGCCGAGGGGAG targeting a protospacer in the beginning of the glgX CDS is expressed behind a J23119 strong constitutive promoter. Next to the ampR gene and the pMB1 ori, the plasmid carries a 818 bp repair template consisting of an iGEM Aachen Multi Stop sequence flanked by two 400 bp homology regions.
▲
Contact
CC-BY-SA iGEM Aachen 2015
