Team:Aachen/Lab/Libraries
Project
Interactive Journey
Overview
Background
Outlook
Lab
Overview
Modeling
Methanol
Biobricks
Polycistronic Expression Plasmid
Monocistronic Diversity Library
Characterization
Glycogen
Synthesis
Knockouts
Characterization
Libraries
Bioreactor
Hardware
Software
Characterization
Notebook
Protocols
Documentation
Construction Manuals
Human Practice
Overview
Maker Faire
Community Labs
tubefront
Safety
Collaborations
Achievements
Judging
Parts
Team
Members
Partners
Attributions
The Challenge
The Team
The Strategy
Biological Approach
The Bioreactor
The People
Maker Faire
Community Labs
tubefront
The Solution
Engineering
Biology
The Future
Introduction
The Anderson Promoters are a family of constitutive promoters with a range of different strengths. The J61002 backbone is especially useful as it provides RFP for colour screening.  However, we wanted more diversity, more choice. Therefore we built inducible Anderson Promoters by introducing a LacI binding site.
In this way, we wanted to provide a useful biobrick collection for our project and future teams. The library enables numerous biobrick combinations and allows flexible promoter choices. As the balance of transcription rates is essential for the metabolization of methanol and the accumulation of glycogen, the library ideally facilitates up- and down regulations. Furthermore, IPTG-induced gene expression makes sense when growing bacteria on a difficult substrate such as methanol.
Achievements
We measured successfully the strength variation of 14 different Anderson Promoters.
We showed that our promoters K15851xx show nearly no activity in absence of IPTG.
We discovered that the lacI binding site of part R0010 does not work behind a promoter of the Anderson library because they are not IPTG inducible.
We demonstrated a profound molecular strategy to establish a promoter library, which would be applicable with any other operator.
Molecular Design
The first step
was to transform each promoter from the kitplate and extract the promoters by PCR from purified plasmids. Therefore, we designed primers (A9W9 and XE3D) to extract and amplify the promoters, resulting in a 1143 bp long fragment that contains prefix, RBS, promoter, rfp and suffix. We had to apply a different reverse primer for J23115 due to a mutation. These fragments can then be digested with EcoRI and SpeI.
For our backbone
we used BBa_J23102 as a template. The amplification was done in the way that the forward primer binds to the prefix while the reverse primer binds on the RBS, introducing a lacI binding site as well as a Xbal restriction site as an overhang.
That way the LacI binding site, that derived from BBa_R0010, was added upstream of the RBS. By cutting the fragement with EcoRI and XbaI the backbone can be merged with any promoters, keeping the introduced LacI binding site. Additionally, any other accordingly cut promoter can be cloned into the original J61002 (without LacI binding site) if the backbone is cut with SpeI instead of XbaI. We followed a simple ligation protocol to obtain the final construct.
'
Characterization
We performed the characterization of our verified constructs in DH5alpha with LB+Amp in 96-well microtiter plate, in which we cultivated and carried out hourly measurements of OD600 and fluorescence ( BBa_E1010 is excitated at 584 nm wavelength and the emission is at 607 nm).
Both factors were quantified in order to observe a possible correlation between the growth rate and the product formation.
Aiming for ultimate reproducibility, we inoculated every wellplate via cryo-replicator from a -80 °C well plate. Each sample was measured in independent triplicates. The efficiency of induction was examined by measuring two identical well plates, but only inducing one plate with IPTG.
The tested original promoters are J23100, J23101, J23102, J23103, J23104, J23105, J23106, J23110, J23113, J23115, J23116, J23117, J23118, J23119.
The tested new promoters with lacI binding site are K1575100, K1575101, K1575102, K1575103, K1575104, K1575105, K1575106, K1575110, K1575113, K1575115, K1575116, K1575117, K1575118 & K1575119.
Results
relative strength of 14 different J231xx in J61002
promoter strength is visualized as specific flourescence therefore relative flourescent units (RFU) are devided by OD600. The promoters are arranged dependent to their strength.
The specific relative flourescence of the original Anderson Promoters in backbone J61002 shows strength variations compared to the original strength of Anderson Promoters from the Registry . The strongest promoter of the 14 measured ones is J23101. J23119 is not the strongest promoter as expected but only the fifth strongest.
Induction test of K15851xx
Our IPTG inducible promoters show no significant increase of flourescence after IPTG induction.
Discussion
We analyzed the original Anderson Promoters in DH5alpha cells of E.coli. The fluorescence values in the registry however were evaluated in strain TG1 which may explain the different measurement values.
Our lacI binding region (lacO) is copied from R0010 and is different to J33207 . It is also different to the lacO region behind the inducible T7 promoter in pSBXXX0 ( Heidelberg 2014 Part: BBa_K1362091 - ...97).
More precisely, we used a sequence of 35 bp although the lacO site of part J33207 is only annotated with 28 bp. Additionally our lacI binding site differs from J33207 in one base pair.
This derivation in the sequence might be a reason that our promoters are not inducible.
Laboratory Notebook
▲
Contact
CC-BY-SA iGEM Aachen 2015
