<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook_Documentation_Promoter_Library skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook/Documentation/Promoter Library</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="main" class="clearfix"><NAV class="navbar navbar-default"><DIV class="dropdownbar"><DIV class="fluid-container"><DIV class="collapse navbar-collapse"><UL id="menu-header" class="nav navbar-nav navbar-left"><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Project</A></LI><LI><A class="menu-link" href="/Team:Aachen/InteractiveTour10">Interactive Journey</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Overview">Overview</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Background">Background</A></LI><LI><A class="menu-link" href="/Team:Aachen/Project/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Lab</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Modeling">Modeling</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Methanol">Methanol</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Biobricks">Biobricks</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Polycistronic_Expression_Plasmid">Polycistronic Expression Plasmid</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Monocistronic_Diversity_Library">Monocistronic Diversity Library</A></LI><LI><A href="/Team:Aachen/Lab/Methanol/Characterization">Characterization</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Glycogen">Glycogen</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Synthesis">Synthesis</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Knockouts">Knockouts</A></LI><LI><A href="/Team:Aachen/Lab/Glycogen/Characterization">Characterization</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Lab/Libraries">Libraries</A></LI><LI id="" class="dropdown"><A class="menu-link" href="/Team:Aachen/Lab/Bioreactor">Bioreactor</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Hardware">Hardware</A></LI><LI><A href="/Team:Aachen/Software">Software</A></LI><LI><A href="/Team:Aachen/Lab/Bioreactor/Characterization_of_Bioreactor">Characterization</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Notebook</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI class="dropdown" id=""><A class="menu-link" href="/Team:Aachen/Notebook/Documentation">Documentation</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Notebook/Construction_Manuals">Construction Manuals</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Human Practice</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Overview">Overview</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Maker_Faire">Maker Faire</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Community_Labs">Community Labs</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/tubefront">tubefront</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Practices/Safety">Safety</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Achievements</A></LI><LI><A class="menu-link" href="/Team:Aachen/Achievements#Judging">Judging</A></LI><LI><A class="menu-link" href="/Team:Aachen/Parts">Parts</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown" class="dropdown-toggle">Team</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Members">Members</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Partners">Partners</A></LI><LI id=""><A class="menu-link" href="/Team:Aachen/Attributions#Attributions">Attributions</A></LI></DIV></DIV></DIV></NAV><DIV class="container tour"><DIV class="tourBoundingBox" style="display:none;"><DIV class="tourBoundingBoxInner"><DIV class="parentTourItem "><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour10">
					The Challenge
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour20">
					The Team
				</A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
					The Strategy
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour31">
							Biological Approach
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour32">
							The Bioreactor
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
					The People
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour41">
							Maker Faire
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour42">
							Community Labs
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour43">
							tubefront
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
					The Solution
				</A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour51">
							Engineering
						</A></DIV><DIV class="childTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour52">
							Biology
						</A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" href="/Team:Aachen/InteractiveTour60">
					The Future
				</A></DIV></DIV></DIV></DIV><DIV class="container container-primary"><DIV class="row"><DIV><A class="square rmbg pull-right rmPromotorlibrary" href="/Team:Aachen/Lab/Libraries"><DIV class="rmcontent"><DIV class="rmtable"><DIV class="rmtable-cell"><DIV class="rmtext"><P class="rmheadline">Promoter Library</P></DIV></DIV></DIV></DIV></A></DIV><H1><SPAN class="mw-headline" id="Laboratory_Notebook">Laboratory Notebook</SPAN></H1><P>The promoter library was the first subproject that we began to approach in the lab. The work started in late april.
</P><H2><SPAN class="mw-headline" id="15-04-22">15-04-22</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Sarah, Tobi
</P><P><I><B>consumable material:</B></I></P><P>Made liquide <A rel="nofollow" class="external text" href="http://igem.rwth-aachen.de/2015/index.php/Manual/LB_Medium">LB-Media</A> and LB-Agarplates with AmpR.
</P><P><B>Topic Promotor Library:</B></P><P>Primer #XLMF#, #NNF3#, #BXT8# diluted to stock in Box P.
</P><P>Primer #XLMF#, #NNF3#, #BXT8# diluted to work with in Box P'.
</P><P>Suspend J23100-J23105 in pSB_J61002 from Kit Plate 4 (2014).
</P><P><A rel="nofollow" class="external text" href="http://igem.rwth-aachen.de/2015/index.php/Manual/transformation">Transformation</A> of J23100-J23105 in pSB_J61002 into competend DH5α (from Schwaneberg)
</P><P>The transformants incubate in plates in the 37 °C since 5 pm. 
</P><P><B>Next steps</B> for tomorrow: 
</P><P>Make overnight cultures of J23100-J23105 in pSB_J61002.
</P><P>Make LB-Media with Chloramphenicol (plates and liquids).
</P><P>suspend the other Promotors from Kit4 2014 and transform them.
</P><H2><SPAN class="mw-headline" id="15-04-23">15-04-23</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Sarah, Tobi
</P><P><I><B>consumable material:</B></I></P><P>made 3x 1 ml of Chloramphenicol antibiotic for stock.
</P><P>LB-Media with chloramphenicol for plates. =&gt; plates are in 4 °C fridge.
</P><P><B>Promotor Library</B></P><P>J23106,J23107, J23110, J23113-18 are <A rel="nofollow" class="external text" href="http://igem.rwth-aachen.de/2015/index.php/Manual/transformation">transformed</A> in DH5α. The plates are cultivating at 37 °C since 11:45 am.
</P><P>Transformationefficiency of J23100-J23105 in pSB_J61002. 
</P><TABLE class="wikitable"><TBODY><TR><TH> Template </TH><TH> efficiency [(cells) cfu/ng (DNA)]
</TH></TR><TR><TD> J23105 </TD><TD> 160
</TD></TR><TR><TD> J23104 </TD><TD> 650
</TD></TR><TR><TD> J23103 </TD><TD> 88
</TD></TR><TR><TD> J23102 </TD><TD> 940
</TD></TR><TR><TD> J23101 </TD><TD> 68
</TD></TR><TR><TD> J23100 </TD><TD> 41
</TD></TR></TBODY></TABLE><P>J23100-J23105 in pSB_J61002 are picked from plate. 3 clones of each is in an overnight culture at 37 °C
</P><P><B>Next steps</B></P><P>Plasmidprep from the overnight cultures of J23100-J23105 in pSB_J61002.
</P><H2><SPAN class="mw-headline" id="15-04-24">15-04-24</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sarah, Tobi, Sven
</P><P><I><B>consumable material:</B></I></P><P>sterile Eppis (1.5 ml)
</P><P>Agarosegel (solution in 65 °C)
</P><P><B>Promotor Library</B></P><P>Plates of J23106,J23107, J23110, J23113-18 are in 4 °C fridge.
</P><P>Make Cryo of J23100-J23105 in pSB_J61002 from overnight cultures (always #1). (We choose only one of the clones.) 
</P><P>Make Plasmidprep with rest of overnight culture from the selected clone (always #1). Measure plasmid concentration with Nanodrop.
</P><P><B>Next steps</B> on Monday
</P><P>Look at transformation efficiency of J23106,J23107, J23110, J23113-18.
</P><P>Make overnight cultures of J23106,J23107, J23110, J23113-18. 
</P><P>Solute from Kit and transformation of J23108, -109, -111, -119 (all in pSB1C3).
</P><P>Solute from Kit and transformation of BBa_K118017 in pSB1C3.
</P><P>Gradient PCR with Primer NNF3(Tm 58 °C) and BXT8(Tm 64 °C) (primers for backbone amplification). Control with gel.
</P><P>Do PCR for backbone amplification.
</P><H2><SPAN class="mw-headline" id="15-04-27">15-04-27</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Laura G., Caro, Jonas, Tobi
</P><P><I><B>consumable material:</B></I></P><P><B>Promotor Library</B></P><P>Transformation efficiency of J23106, -07, -10, -13-18 in pSB_J61002 (incubation time was 20h instead of 12h).
</P><TABLE class="wikitable"><TBODY><TR><TH> Template </TH><TH> efficiency [(cells) cfu/ng (DNA)]
</TH></TR><TR><TD> J23106 </TD><TD> 1200
</TD></TR><TR><TD> J23107 </TD><TD> 5200
</TD></TR><TR><TD> J23110 </TD><TD> 5200
</TD></TR><TR><TD> J23113 </TD><TD> ca 1300
</TD></TR><TR><TD> J23114 </TD><TD> 1800
</TD></TR><TR><TD> J23115 </TD><TD> ca 3600
</TD></TR><TR><TD> J23116 </TD><TD> 2000
</TD></TR><TR><TD> J23117 </TD><TD> 3200
</TD></TR><TR><TD> J23118 </TD><TD> 1600
</TD></TR></TBODY></TABLE><P>Transformation of J23108, 09, 11, 19 and BBa_K118018 (glgC+rbs) and cultivation on plates since 13:00 (accidently 1000 µl are on the plate).
</P><P>Make overnight cultures of J23106,J23107, J23110, J23113-18 (We picked 2 clones of each; cultivation started at 16:20).
</P><P>PCR II of J23100-J23105 in pSB_J61002 with A9W9 and XE3D (15-04-27 Gel upper row). Digested by DpnI. PCR Products are in -20 °C freezer in Falkon #1 (EOLX, VCZX, NDS4, H9MQ, T6FB, KQN8).
</P><P>
Gradient PCR with J23102 (MRLK) for primer NNF3 and BXT8 (15-04-27 Gel lower row). Must be done once again, bands did not look good!
</P><P><B>Next steps</B> on Tuesday
</P><P>purification of PCR Products.
</P><P>Gel annotation from PCR 15-04-27.
</P><P>Cryo culture of J23106,J23107, J23110, J23113-18.
</P><P>Plasmidprep of J23106,J23107, J23110, J23113-18.
</P><P>Overnight cultures of J23108, 09, 11, 19 and BBa_K118018 (glgC+rbs).
</P><H2><SPAN class="mw-headline" id="15-04-28">15-04-28</SPAN></H2><P><B>Biotec lab;</B> Attendens: Tobi, Sven, Laura G, Caro
</P><P><I><B>consumable material:</B></I></P><P>sterile Eppis
</P><P><B>Promotor Library</B></P><P>Evaluation: The colour of overnight cultures of J23106, J23107, J23110, J23113 - J23118 fits to their strength.
</P><P>Annotation of PCR gel from promoter amplification J23100-05 (15-04-27 upper row).
</P><P>Purification of the PCR products J23100-105. Measurement of concentration after digestion with DpnI and heat inactivation for 20 minutes at 80 °C
</P><TABLE class="wikitable"><TBODY><TR><TH> Promoter </TH><TH> Concentration (ng/ul) </TH><TH> ID
</TH></TR><TR><TD> J23100 </TD><TD> 89.7 </TD><TD> E0LX
</TD></TR><TR><TD> J23101</TD><TD> 107.9 </TD><TD> VYN9
</TD></TR><TR><TD> J23102 </TD><TD> 94.0 </TD><TD> NDS4
</TD></TR><TR><TD> J23103 </TD><TD> 89.8 </TD><TD> H9MQ
</TD></TR><TR><TD> J23104 </TD><TD> 90.4 </TD><TD> T6FB
</TD></TR><TR><TD> J23105 </TD><TD> 85.0 </TD><TD> KQN8
</TD></TR></TBODY></TABLE><P>Cryo of J23106,J23107, J23110, J23113-18.
</P><P>Plasmidprep of J23106,J23107, J23110, J23113-18. Measurement of concentrations. All in Box &quot;Pink&quot; row F1-9.
</P><P>Transformation efficiency of J23111 in pSB1C3 is 800 cfu/ng. Of BBa_K118018 (GlgC+RBS) 36 nfu/ng (perhaps due to chloramphenicol?). All other plates &quot;many&quot; (keep in mind that 1 ml was plated). None of the colonies are visibly red. 
</P><P>Overnight cultures of J23108, 09, 11, 19 and BBa_K118018 (glgC+rbs). Started at 16:00.
</P><P>Gradient PCR with J23102 (MRLK) for primer NNF3 and BXT8 (63.2-69.7 °C). Tubes are in -20 freezer to make the gel tomorrow.
</P><P><B>Next steps</B></P><P>Run gel of gradient PCR.
</P><P>Cryo of J23108, 09, 11, 19 and BBa_K118018 (glgC+rbs).
</P><P>Plasmidprep of J23108, 09, 11, 19 and BBa_K118018 (glgC+rbs).
</P><P>if time: PCR II of J23106,J23107, J23110, J23113-18 for promotoramplification (Primer A9W9 and XE3D). =&gt;after DpnI digestion and purification with kit.
</P><H2><SPAN class="mw-headline" id="15-04-29">15-04-29</SPAN></H2><P><B>Biotec lab;</B> Attendens: Tobi, Sven, Jonas, Laura G., Caro
</P><P><I><B>consumable material:</B></I></P><P><B>Promotor Library</B></P><P>Overnight cultures of J23108, 09, 11 and 19 are colourless.
</P><P>Cryos and plasmid prep of J23108, 09,11,19 and BBa_K118018 (GlgC+ RBS). See concentrations below.
</P><TABLE class="wikitable"><TBODY><TR><TH> Part </TH><TH> Concentration (ng/ul) </TH><TH> ID
</TH></TR><TR><TD> J23108</TD><TD> 49.3 </TD><TD> W1KW
</TD></TR><TR><TD> J23109</TD><TD> 44.7 </TD><TD> W49P
</TD></TR><TR><TD> J23111 </TD><TD> 49.2 </TD><TD> S8R6
</TD></TR><TR><TD> J23119</TD><TD> 49.2 </TD><TD> RWNO
</TD></TR><TR><TD> BBa_K118018 (GlgC+RBS)  </TD><TD> 250.8 </TD><TD> 3Q8D
</TD></TR><TR><TD>  BBa_K118018 (GlgC+RBS) Nr.2 </TD><TD> 328.7</TD><TD> V3SC
</TD></TR></TBODY></TABLE><P>Gel of gradient PCR of BXT8 and NNF3 (Tip: next time always 100V)
</P><P><B>Next steps</B></P><P>get gel picture from server and do annotation (from gradient PCR 15-04-29)
</P><P>PCR II of J23106,J23107, J23110, J23113-18 for promotor amplification (Primer A9W9 and XE3D). =&gt;afterwards DpnI digestion and purification with kit
</P><P>PCR II of J23108, 09,11,19 for promotor amplification.
</P><P>order primer and gene synthesis (first talk to Ljubica) (=&gt; primer BXT8 might be wrong designed)
</P><H2><SPAN class="mw-headline" id="15-04-30">15-04-30</SPAN></H2><P><B>Biotec lab;</B> Attendens: Jonas, Tobi, Sven, Caro
</P><P><I><B>consumable material:</B></I></P><P><B>Promotor Library</B></P><P>Primers for assamblies are ordered (plus BXT8 is wrong designed).
</P><P>gel picture (from gradient PCR 15-04-29) is absolutely nonsense and will not be annotated.
</P><P>PCR II of J23106,J23107, J23110, J23113-14, 16-18 and J23108, 09,11,19 for promotor amplification (Primer A9W9 and XE3D), each 4 reactions per sample.
</P><UL><LI> Run the gel of PCR II from today (12 samples)</LI></UL><OL><LI>successful: J23106, -07, -08, -09, -11, -13, -17, -18</LI><LI>failed: J23110, -14, -19</LI><LI>insecure: J23116</LI></OL><P><B>Next steps</B> on Monday
</P><P>DpnI digestion and purification of successful PCR-products (J23106, -07, -08, -09, -11, -13, -17, -18).
</P><P>PCR II of J23100-05 in 4 preparations each (Primer A9W9 and XE3D). PCR of BBa_K118018 (GlgC+ RBS) (Templates: 3Q8D, V3SC) for control! (Maybe do PCR with J23116 again.)
</P><P>DpnI digestion of the successful PCR-products.
</P><P><B>Next week:</B></P><P>gradient PCR I with new Primers
</P><P>PCR I for backbone amplification with lacI
</P><H2><SPAN class="mw-headline" id="15-05-04">15-05-04</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sarah, Tobi, Sven, Jonas, Laura 2x, Caro
</P><P><I><B>consumable material:</B></I></P><P>Kanamycin stock 1000x in 10 Eppis (-20 °C freezer)
</P><P><B>Promotor Library</B></P><P>gelcontrol of J23110, -14, -16 and -19 of all 4 preparations.
</P><UL><LI> PCR products with expected length: J23110 #3+4, J23114 #1+4, J23116 #2+3, J23119 #1</LI></UL><P>(estimated mistakes: too little buffer in PCR from 15.04-30)
</P><UL><LI> digestion of J23110, -14, -16, -19 with DpnI</LI></UL><P>
DpnI digestion of successful PCR-products (J23106, -07, -08, -09, -11, -13, -17, -18) (all in separate Eppis).
</P><P>PCR II of J23100-05 in 3 preparations each, -16 in 4 preparations. 
</P><P>=&gt; Gel control (start: 15:30), all preparations loaded.
</P><P><B>Next steps</B> on Tuesday
</P><P>Gel for recontrol of failed PCR products of J23100-05 + -16.
</P><P>Heat inactivation and purification of digested J23106, -07, -08, -09, -11, -13, -17, -18.
</P><P>Heat inactivation and purification of digested J23110 #3+4, J23114 #1+4, J23116 #2+3, J23119 #1. Use all these as template for new promotor-amplifications.
</P><P>PCR II J23115 in 4 preparations and PCR of BBa_K118018 (GlgC+ RBS) (Templates: 3Q8D, V3SC) in 4 preparations each (for control). 
</P><P>=&gt; gel for control 
</P><P>=&gt; DpnI overnight of the successful PCR products.
</P><P><B>if</B> Primers arrived: gradient PCR I with new Primers for backbone amplification with lacI
</P><H2><SPAN class="mw-headline" id="15-05-05">15-05-05</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi, Sven, Caro, Jonas, Michael
</P><P><I><B>consumable material:</B></I></P><P>Kanamycin plates
</P><P><B>Promotor Library</B></P><UL><LI> make new gels of ugly gels from yesterday (J23100-05, -16)</LI></UL><UL><LI> Heat inactivation and purification of digested J23106, -07, -08, -09, -11, -13, -17, -18.</LI></UL><UL><LI> Heat inactivation and purification of digested J23110 #3+4, J23114 #1+4, J23116 #2+3, J23119 #1</LI></UL><P>The only successful concentration measurements: J23106, -07, -09, -17 &amp; -18. All in Pink Box with ID.
</P><P>PCR II of J23108, J23110, J23111, J23113, J23114, J23115, J23116, J23119 with Plasmids from stock. Each in 3 preparations (a, b, c).
</P><P><B>Next steps</B></P><P>Gel of PCR II with J23108, J23110, J23111, J23113, J23114, J23115, J23116, J23119, maybe DpnI digestion afterwards with successful PCR products (overnight).
</P><P>New gradient PCR for Backboneamplification with new Primer (NNF3 &amp; FYTO)
</P><H2><SPAN class="mw-headline" id="15-05-06">15-05-06</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi, Jonas, Laura
</P><P><I><B>consumable material:</B></I></P><P><B>Promotor Library</B></P><P>Gel control of the repeated PCR with J23108, -10, -11, -13, -14, -15, -16, -19.
</P><P>DpnI digestion 
</P><P>Successful: -08,-10,-11,-13,-14,-16,-19b (19 a,c)
</P><P>Uncertain: -15 (expected length 185 bp)
</P><P>expected length of J23115 is 185 bp.
</P><P><B>Next steps</B></P><UL><LI>Purification of DpnI-digested PCR products</LI><LI>Gradient PCR for backbone (PCR mix is in freezer! Has to be done on Thursday 15-05-07. PCR machine was blocked)</LI><LI>Gel control for Gradient PCR</LI><LI>Final PCR for backbone amplification + control</LI><LI>repeat PCR with J23115</LI></UL><H2><SPAN class="mw-headline" id="15-05-07">15-05-07</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi, Jonas, Laura, 
</P><P><I><B>consumable material:</B></I></P><P>2 % Agarose for gel in 65 °C
</P><P><B>Promotor Library</B></P><P>Run new gradient PCR to find out annealing temperatures for backbone amplification (primers: NNF3 and FYTO, template: MRLK)
</P><P>Do the gel control of the new gradient PCR
</P><P>Purification of PCR products of J23108, J23110, J23111, J23113, J23114, J23116 &amp; J23119. All are containered and in Box &quot;Pink&quot;.
</P><P>Do 2 % Agarosegel for loading J23115 with 50 bp ladder.
</P><P>Do backbone amplification with NNF3/FYTO at 60.3 °C in 10 preparations, all with same template. Run all in one slott on a gel and extract backbone from gel (purify and have a look how much is still missing).
</P><P>
The gel of J23115 shows bands at the expected length. Therefore, digestion with DpnI since 16:30 at 37 °C.
</P><P>Made PCR I for backbone amplification in 10 preparations (Is in freezer for the gel tomorrow)
</P><P><B>Next steps</B></P><P>run 2 % gel of backbone (PCR I) and extract backbone (2992 bp).
</P><P>do PCR I for backbone amplification again for more backbones =&gt; run gel and extract again.
</P><P>purification of J23115
</P><H2><SPAN class="mw-headline" id="15-05-08">15-05-08</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Laura G., Tobi
</P><P><I><B>consumable material:</B></I></P><P><B>Promotor Library</B></P><P>purification of PCR product J23115
</P><P>2 % Gel for Backbone isolation =&gt; do backbone purification with kit =&gt; 10 new containers with new &quot;J61002 with lacI&quot;
</P><P>PCR I for more backbone =&gt; 2 % gel loaded and cutted out (and freezed over the weekend (Eppi A-H))
</P><P><B>Next steps</B></P><P>purification of cutted gel with J61002 with LacI
</P><P>perhaps: more amplificates of J23100-J23105
</P><H2><SPAN class="mw-headline" id="15-05-11">15-05-11</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi, Sarah, Sven
</P><P><I><B>consumable material:</B></I>
petri dishes 2x
</P><P><B>Promotor Library</B></P><P>We did another purification of cut agarose gel with J61002+LacI. The concentration measurement was nonsense.
</P><P>It had to be checked if we already have enough backbone template for restriction and ligation.
</P><P>We digested the backbone (different enzymes  for &quot;with LacI-BS&quot; and &quot;normal&quot;) and heat inactivated.
</P><P><B>Next steps (in general)</B></P><P><A rel="nofollow" class="external text" href="http://igem.rwth-aachen.de/2015/index.php/Manual/DigestionBB-Assembly">restriction</A> with Xba and EcoRI of backbone (DRPV) in 10 preparations with 20 µl reaction volume each (this is enough for 100 ligations). Incubation at 37 °C for 40 min and heat inactivation at 80 °C for 20 min. Done for backbone with and without lacI. Freezed at -20 °C.
</P><P>Restriction with Spe and EcoRI of each promotor in 1 preparation with 20 µl volume each (enough for 10 ligation for each promotor). Incubate at 37 °C for 40 min and heat inactivation at 80 °C for 20 min. Freezed at -20 °C.
</P><P>For <A rel="nofollow" class="external text" href="http://igem.rwth-aachen.de/2015/index.php/Manual/LigationBB-Assembly">ligation</A>, 2 µl of each DNA part from restriction is necessary.
</P><H2><SPAN class="mw-headline" id="15-05-12">15-05-12</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi, Sven
</P><P><I><B>consumable material:</B></I></P><UL><LI>make ampicillin petri dishes for transformations next week</LI></UL><P><I><B>Promotor library:</B></I></P><P>Dilution of J23113 and -16 
</P><P>Digestion of Promotors 00-19 and heat inactivation (restriction with Spe and EcoRI of each promotor in 1 preparation with 20 µl volume each (enough for 10 ligation for each promotor). Incubation at 37 °C for 40 min and heatinactivate at 80 °C for 20 min).
</P><P>for <A rel="nofollow" class="external text" href="http://igem.rwth-aachen.de/2015/index.php/Manual/LigationBB-Assembly">ligation</A> is 2 µl of each DNA part from restriction necessary. Every restricted promotor is ligated in new backbone with lacI. Everything is freezed -20 °C in Falkon #2.
</P><P><B>next steps</B></P><P>finish ligation of J23108, J23109, J23111, J23115 and J23119 in original J61002.
</P><P>transformation of ligation products in competent DH5α cells (on monday)
</P><H2><SPAN class="mw-headline" id="15-05-13">15-05-13</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi, Sven
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Ligation of J23108,-09,-11,-15,-19 in the originial backbone J61002 (without LacI), Incubation for 45 minutes
</P><P>Heat inactivation for 20 minutes
</P><P><B>next steps</B></P><H2><SPAN class="mw-headline" id="15-05-18">15-05-18</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi
</P><P><I><B>consumable material:</B></I>
DH5α
</P><P><I><B>Promotor library:</B></I></P><P>Heatshock transformation of ligation products in DH5α , plate them on Amp-plates (clones with new lacI promotor should be colorless, the five clones with original promotors should be as red as their strength). We used 3 µl of ligation product on ca. 50 µl  of competent cells. After 30 minutes ice, 45 seconds water bath, 2 minutes ice we added 200 µl  of preheated SOC medium. Incubation for 1 hour (37 °C /250 rpm)
</P><P>Plates incubate at 37 °C since 16:50.
</P><P><B>next steps:</B></P><P>pick single clones from all transformations (19+4) for overnight cultures.
</P><H2><SPAN class="mw-headline" id="15-05-19">15-05-19</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi, Sven
</P><P><I><B>consumable material:</B></I></P><P>new LB-Amp-plates
</P><P><I><B>Promotor library:</B></I></P><P>In generel poor cell growth but there are colonies on every plate. However, also the 5 promotors (08,09,11,15,19) in the original backbone (without LacI binding site) are white. They should be red. They might turn to red with more time?
</P><P>We did masterplates (with 6 clones of each) of all transformations. J23102 and -09 only had as few colonies as there are spread on the masterplates.
</P><P>We make overnight cultures from J23108 ori, -15 ori, -19 ori, -11 ori, -06, -17, -16, -05. &quot;Ori&quot; stands for the original J61002 backbone without LacI binding site.
</P><P><I><B>Next steps:</B></I></P><P>Do cryos, plasmid preps and prepare sequencing from the overnight cultures (results might come on Friday)
</P><P>Colony PCR from masterplates (on Thursday):Which primers would we use to check the orientation of our insert?
</P><H2><SPAN class="mw-headline" id="15-05-20">15-05-20</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I>
Masterplates:
</P><TABLE class="wikitable"><TBODY><TR><TH> promoter </TH><TH> strength </TH><TH> Clone colours </TH><TH> considerations
</TH></TR><TR><TD> J23100 </TD><TD> 2547 </TD><TD> all red </TD></TR><TR><TD> J23101 </TD><TD> 1791 </TD><TD> 1-4 red </TD></TR><TR><TD> J23102 </TD><TD> 2197 </TD><TD> 4 red </TD></TR><TR><TD> J23103 </TD><TD> 17 </TD><TD> all white </TD><TD> because of LacI or weak promotor?
</TD></TR><TR><TD> J23104 </TD><TD> 1831 </TD><TD> 3-5 red </TD></TR><TR><TD> J23105 </TD><TD> 623 </TD><TD> 2 red </TD></TR><TR><TD> J23106 </TD><TD> 1185 </TD><TD> 2,4 red </TD></TR><TR><TD> J23107 </TD><TD> 908 </TD><TD> 4,5 red </TD></TR><TR><TD> J23108 </TD><TD> 1303 </TD><TD> 3,6 red </TD></TR><TR><TD> J23109 </TD><TD> 106 </TD><TD> only 2 clones, both white </TD></TR><TR><TD> J23110 </TD><TD> 844 </TD><TD> 1 red </TD></TR><TR><TD> J23111 </TD><TD> 1487 </TD><TD> 1 red </TD></TR><TR><TD> J23113 </TD><TD> 21 </TD><TD> 2,5,6 red </TD></TR><TR><TD> J23114 </TD><TD> 256 </TD><TD> all white </TD></TR><TR><TD> J23115 </TD><TD> 387 </TD><TD> 3 red </TD></TR><TR><TD> J23116 </TD><TD> 396 </TD><TD> all white </TD></TR><TR><TD> J23117 </TD><TD> 162 </TD><TD> all white </TD></TR><TR><TD> J23118 </TD><TD> 1429 </TD><TD> 2,3 red </TD></TR><TR><TD> J23119 </TD><TD> strongest </TD><TD> all white </TD></TR><TR><TD> J23119 ori </TD><TD> strongest </TD><TD> all red</TD></TR></TBODY></TABLE><P>Cryo and plasmidprep from 2 clones (each) of 08 ori, 15 ori, 19 ori, 11 ori, 06, 17, 16, 05.
</P><P>for sequencing: 05#2, 11ori#2, 16#2, 17#2, 19ori#2 (#4FYS#, #KFWP#, #1CSA#, #ZXCE#, #9TVM#)
</P><H2><SPAN class="mw-headline" id="15-05-21">15-05-21</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi, Sven
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>DO overnights from all missing constructs (2 clones each) and do cryo of all tomorrow. Do some plasmid preps and freeze the rest of the cultures for plasmid preps next tuesday.
</P><P>Primer 39OW was designed for sequencing from RFP reverse (necessary for vector library).
</P><H2><SPAN class="mw-headline" id="15-05-22">15-05-22</SPAN></H2><P><B>Biotec lab;</B> Attendens: Laura G., Tobi
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Make cryo of all overnights. 
</P><P>16 clones are freezed in eppis (-20 °C) for plasmid prep on tuesday. Clones: 13#4, 19#1, 03#2, 09ori#4, 19#2, 14#2, 13#3, 10#2, 01#4, 15#4, 09#1, 14#1, 18#5, 01#5, 09ori#5, 11#6.
</P><P>16 clones are prepped today and the plasmids are freezed. clones: 08#4, 04#2, 07#2, 09#2, 07#3, 11#5, 15#6, 02#2, 10#3, 00#4, 02#1, 18#4, 00#6, 08#5, 03#1, 04#1.
</P><P><B>next steps:</B> on Tuesday
</P><P>plasmid prep of the last 16 clones.
</P><P>check sequencing results of the 5 first promotorconstucts.
</P><P>on Wednesday: prepare 43 plasmids for sequencing.
</P><H2><SPAN class="mw-headline" id="15-05-26">15-05-26</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>16 clones  plasmidprep. clones: 13#4, 19#1, 03#2, 09ori#4, 19#2, 14#2, 13#3, 10#2, 01#4, 15#4, 09#1, 14#1, 18#5, 01#5, 09ori#5, 11#6.
</P><P>check sequencing results of the 5 first promoter constucts.
</P><UL><LI> #DHAY# expected: J23111 (original), result: between Prefix and Promotor are 18 bp too much (including an unwanted SpeI), between Promotor and RBS are 11 bp too much (including SpeI).</LI></UL><UL><LI> #HMZF# expected: J23119 (original), result: sequence of J23102 original</LI></UL><UL><LI> #9TVM# expected: K1585105 (05+lacI), result: sequence of J23102 original</LI></UL><UL><LI> #1CSA# expected: K1585116 (16+lacI), result: successful!!!</LI></UL><UL><LI> #MELT# expected: K1585117 (17+lacI), result: nonsense chromatogramme</LI></UL><P><B>next steps:</B></P><P>on Wednesday: prepare 43 plasmids (rest of AP+lacI) for sequencing. (see Inventory)
</P><H2><SPAN class="mw-headline" id="15-05-27">15-05-27</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>prepare 43 plasmids (rest of AP+lacI) for sequencing.
</P><P><B>next steps</B></P><H2><SPAN class="mw-headline" id="15-05-28">15-05-28</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>try to understand sequencing result of #DHAY#
</P><P><B>next steps:</B></P><H2><SPAN class="mw-headline" id="15-05-29">15-05-29</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>We checked the sequencing results. 16 of 24 promoters are successfully built.
</P><P><B>next steps:</B></P><P>discard unnecessary tubes with plasmids and cryos after sequencing results.
</P><H2><SPAN class="mw-headline" id="15-06-01">15-06-01</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I> LB + Amp plates, LB
</P><P><I><B>Promotor library:</B></I></P><P>new transformation  of stored ligation products: J23108 (original) J23109 (original) and BBa_K1585109. Plates incubate since 14:30.
Ligated constructs in <A rel="nofollow" class="external text" href="http://tubefront.mosthege.net/iGEM_Aachen/editbox.php?id=FOLKON%20NR.2">Falcon #2</A>. 
</P><P>new overnights from masterplates: 07 (#1, #6), 08 (#1, #2), 11 (#2, #3, #4), 14 (#4, #5, #6), 11ori (#5) (for sequencing) incubate since 15:30.
</P><P>Discard from Box C 7 tubes, Box D 17 tubes, Box A 14 tubes and Box E 10 tubes.
</P><P><B>next steps:</B></P><P>cryos and plasmid prep from overnights for sequencing.
</P><P>overnights from transformations
</P><H2><SPAN class="mw-headline" id="15-06-02">15-06-02</SPAN></H2><P><B>Biotec lab;</B> Attendens: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Cryos (in Box B) and plasmid prep from 07 #1 #6, 08 #1 #2, 11 #2 #3 #4, 14 #4 #5 #6, 11 ori #5
</P><P>Masterplates and overnights from 08 ori, 09 ori and 09 transformations. Incubation since 14:40.
</P><H2><SPAN class="mw-headline" id="15-06-03">15-06-03</SPAN></H2><P><B>Biotec lab;</B> Attendantss: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Cryos (in Box B) and plasmid prep from 08 ori #1, 09 ori #1 (since these were red on the masterplate) and 09 (#1-3)
</P><P>Sequencing of the above
</P><TABLE class="wikitable"><TBODY><TR><TH> Promotor+Clone</TH><TH> Plasmid </TH><TH> Cryo
</TH></TR><TR><TD>09 #1  </TD><TD> #MXEY# </TD><TD> #OW1V#
</TD></TR><TR><TD>09 #2 </TD><TD> #FWRP# </TD><TD> #BS8O#
</TD></TR><TR><TD>09 #3 </TD><TD> #NWV8# </TD><TD> #4AEQ#
</TD></TR><TR><TD>09ori #1 </TD><TD> #ZWAX# </TD><TD> #TYLE#
</TD></TR><TR><TD>08ori #1 </TD><TD> #FM8N# </TD><TD> #BQKN#
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="15-06-11">15-06-11</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Sequencing results were aligned to planned sequences. All samples were negative. K1585114 und K1585107 had a point mutation in every promoter. We assume this mutation was introduced by PCR since it is present in all otherwise positive samples of the respective construct. A detailed overview can be found under sequencing results.
</P><H2><SPAN class="mw-headline" id="15-06-15">15-06-15</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven 
</P><P><I><B>consumable material:</B></I> new LB + A agar plates, new dilutions of A9W9 and XE3D (10µM, can be found in P' as usual)
</P><P><I><B>Promotor library:</B></I></P><P>We did another PCR to amplify J23107 (#MAMH#)and J23114 (#6ORC#)to hopefully get rid of the mutation. 
DpnI digestion and purification: product can be found at #DMLM# and #O34K#.
Restriction and ligation of J23108, 09, 11 and K1585107, 08, 09, 11, 14 were repeated. Digestion was repeated for all inserts. The old already cut backbones were used (#9P6B# and #FD1K#). Digested inserts can be found in Falcon #FAKE#. Ligated Products can be found in Falcon #FUCK#.
Transformation into <I>E. coli</I> DH5α. Incubation at 37 °C over night. 
Plates of K1585119 (#CKYR#) and J23119 (#BYS8#) from cryos for further testing.
</P><H2><SPAN class="mw-headline" id="15-06-16">15-06-16</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven 
</P><P><I><B>consumable material:</B></I> IPTG (50x) aliquots, can be found in the antibiotics box (bio 6)
</P><P><I><B>Promotor library:</B></I></P><P>Master plates and overnight cultures of J23108, 09, 11 and K1585107, 08, 09, 11, 14. 
Overnight cultures of J23119 and K1585119. K1585119 was cultivated with and without IPTG to test the expression in comparison to the constitutive J23119.
Cultures of J23108 and J23111 contructs looked slighty reddish on the transformation plates. This is expected since they are both medium strength promoters.
</P><H2><SPAN class="mw-headline" id="15-06-17">15-06-17</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven 
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>
We discarded cryos and purified plasmids of the last unsuccessful sequencing. 
Plasmid preps of J23108, 09, 11 and K1585107, 08, 09, 11, 14. Three clones each.
Prepared sequencing samples of those constructs
Comparison between J23119, K1585119 and K1585119 + IPTG.
The constitutive promoter J23119 showed a strong rfp expression as expected. Both K1585119 cultures showed only a very weak expression. Repression via the LacI binding site therefore seems to be working as intended whereas the induction with IPTG failed. This may be caused by a too low IPTG concentration. To ensure this the experiment has to be repeated with multiple IPTG concentrations.
</P><UL><LI>Note: It took quite a long time till rfp expression was visible, even the J23119 culture was still not red after 16h incubation.</LI></UL><H2><SPAN class="mw-headline" id="15-06-18">15-06-18</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven 
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P><B>To Do:</B></P><P>Seperate sequencing table for discarded constructs
new overnight cultures of J23119 and K1585119 with varying IPTG concentrations (0.1; 0.5; 1.0; 2.5; 5 and 10 mM)
</P><H2><SPAN class="mw-headline" id="15-06-21">15-06-21</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Sequencing results were all negative
</P><P>IPTG induced cultures did not show any red color for all tested IPTG concentrations. We had to do a reverse sequencing to verify that rfp coding sequence is intact.
</P><H2><SPAN class="mw-headline" id="15-06-22">15-06-22</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>New PCR for J23108 (PCR with W1KW),J23109(PCR with W49P) and J2311 (PCR with S8R6) was not successful.
</P><P>Gel of restricted promotors showed nonsense (reason for nonsense ligation products)
</P><P>backbone: restriction of 3N4M (once with Eco/Xba and once with Eco/Spe). Both tubes are in Falcon &quot;22.06. Back&quot;
</P><P>prepare sequencing of successful promoters with reverse primer (XE3D) since IPTG induction failed
</P><P><B>To Do:</B>
Do we have to amplify backbone once more??
</P><H2><SPAN class="mw-headline" id="15-06-23">15-06-23</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>We did new PCRs of J23107, 08, 09, 11, 14
However, the gel showed hardly any bands. Because the PCR components seem to be fine, we have to use a new, prepped template.
</P><H2><SPAN class="mw-headline" id="15-06-24">15-06-24</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>We did plates of the original kitplate transformants for J23107 (#X4YM#), J23108 (#XXM6#), J23109 (#KS96#), J23111 (#MBKL#) and J23114 (#XBVM#).
<B>note:</B> J23108, J23109 and J23111 in pSB1C3
</P><H2><SPAN class="mw-headline" id="15-06-25">15-06-25</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Preparation for induction tests: plates and overnights of J23119 (#BYS8#) and K1585119 (#CKYR# and #YFV9#). 
</P><P>plates from cryo and overnight cultures of J23107, J23108, J23109, J23111 and J23114.
</P><H2><SPAN class="mw-headline" id="15-06-26">15-06-26</SPAN></H2><P><B>Biotec lab;</B> Attendants: Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Result of induction test: (#YFV9# not ind; #YFV9# ind; #BYS8#; #CKYR# ind; #CKYR# not ind)
</P><P>
Plasmidprep of overnights (#DE6O#, #BRRS#, #CXSS#, #PT93#, #DMCX#)
</P><P>PCR and gel verification.
</P><H2><SPAN class="mw-headline" id="15-06-29">15-06-29</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>PCR of #PT93# and #DMCX# with A9W9/XE3D to amplify J23111 and J23114 inserts.
</P><P>DpnI digestion of J23107, J23108, J23109 (pcr 2015-06-26) and J23111, J23114 (see above).
</P><P>The gel showed only light bands. The purification of digested PCR products was not successful. Almost no DNA measured, only salt peak at 230 nm. 
</P><P>
Repeat full PCR for J23107 (#DE6O#), J23108 (#BRRS#), J23109 (#CXSS#), J23111 (#PT93#) and J23114 (#DMCX#). (see 2015-06-26)
</P><P>Gel of full repeat of PCR promoter amplification
</P><P><B>Characterisation</B></P><P>Overnights of J23119 (constitutive, #BYS8#), K1585119 induced and non-induced (#CKYR# and #YFV9#) and pSB1K30 (positive control) to verify whether or not our IPTG stock is fine. 
</P><P>Induced cultures were prepared in 3 concentrations each: 0.2x (1 µM), 1x (5 µM) and 5x (25 µM).
</P><P>pSB1K30 in NEB Turbo is used as positive control. <B>Note: Plasmid with kanamycin resistance!</B></P><P>Incubation at 37 °C for 18 h.
</P><H2><SPAN class="mw-headline" id="15-06-30">15-06-30</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>another PCR of all 5 promotors: 30 cycles and 2ul template
</P><P>Gel of J23109, -11 &amp; -14 is nonsense.
</P><P>Do DpnI digest of J23107 &amp; J23108 for 1,5h plus heat inactivation.
</P><P><I><B>Result of induction test <SPAN style="color:red;">(only visual with green light)</SPAN>:</B></I></P><P><SPAN style="font-style:italic; font-size:10px">constitutive J23119 has the strongest red colour. Not induced K30 &amp; AP19 are colourless. Strain #CKYR# and #YFV9# have always the same colour under the same conditions. Induced K30 is always a little less colourful as the induced AP19s (that is unexpected). 0,2x IPTG shows no difference to not induced cultures. 1x IPTG is slightly red. 5x IPTG have red colour but not as strong as the constitutive one.</SPAN></P><P>Experiment gives hardly any usable information. J23119 should clearly show a strong pink/red color visible in regular light (see documentation 15-06-17). The negative result of the J23119 implies that there might be some problems with cultivation. Cultures might not get enough oxygen to form the rfp (people at bio6 sometimes forget to switch on the shaker after they removed their samples). 
</P><P>'<B><I>New Induction Test'</I>:</B></P><P>Same setup as before. This time we used shakers at the iAMB. Both IPTG stocks are tested at the same time
</P><P>Overnights of J23119 (constitutive, #BYS8# from 2 different plates), K1585119 induced (3 times) and non-induced from #CKYR# and pSB1K30 in NEB Turbo ## induced 3 times and one not induced <B>(Note: Plasmid with kanamycin resistance!)</B></P><P>For induction we used two different IPTG stocks 50x &amp; 1000x.
</P><P>Induced cultures were prepared in 3 concentrations each: 1x (5 µM), 5x (25 µM) and 10x (50 µM).
</P><P>Incubation at 37 °C starting at 16:00.
</P><P><I><B>Results:</B></I></P><P>J23119 cultures both show a very strong pink color after 24 h of incubation. All other cultures do not show a significant amount of rfp expression. Positive control is negative due to the use of a wrong strain (NEB Turbo is a cloning strain without T7 polymerase)
</P><P>
Next time we use BBa_I13507 in pSB1A30 in BL21 with Cas9 to express RFP (Cas9 will be destroied at 37 °C) (alternatively BBa_I13507 in pSB1A30 in BL21 is in Cryo #6DA3#). And pSB1K30 in BL21 (#KK16#).
</P><H2><SPAN class="mw-headline" id="15-07-01">15-07-01</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Purification of J23107 &amp; -08 PCR products is successful (#3X3H#, #Z6MT#).
</P><P>Restriction of J23107 &amp; -08 with EcoRI and SpeI (stored #WT6F#). Gel control of restriction. 
</P><P>=&gt; ligation with backbone with and without LacI binding site (stored #DWCR#).
</P><P>Transformation of ligation products (AmpR) 07 &amp; 08 mit LacI backbone, 08 in original backbone. 
</P><P><I><B>Induction Test:</B></I></P><P>Repeat of the previous induction experiment. This time we used pSB1K30 and pSB1A30 (both induced 1x and not induced to test). One J23119 culture was used as positive control for general rfp expression.
</P><P><I><B>Results:</B></I></P><P>Positive control was negative even after 40 h. Cultivation in bio6 shakers probably does not provide enough oxygen to enable rfp processing. We will try to do a first real characterization experiment in deep well plates next week. Aerobic conditions should not be a problem there. 
</P><P><I><B>next steps</B></I></P><P>Masterplates of transformants.
</P><P>PCR of J23109, -11 &amp; -14
</P><H2><SPAN class="mw-headline" id="15-07-02">15-07-02</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>PCR of J23109, -11 &amp; -14. Gel control.
</P><P>DpnI digest and heat inactivation #Z1AR#, #1AWV#, #ZBWQ#. Digest everything with EcoRI &amp; SpeI and ligate to backbone with and without LacI binding site. Transform all ligation products.
</P><P>Do overnights and masterplates (pick lots of clones) of successful transformations (only J23108 and K1585107).
</P><H2><SPAN class="mw-headline" id="15-07-03">15-07-03</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>We made cryos and plasmid prep of overnight cutures (K1575107 #1-8 &amp; J23108 #1-8).
</P><P><I><B>next steps</B></I> (on monday)
</P><P>prepare for sequencing:
</P><PRE>#84YY#
#3X4N#
#S3Z3#
#EY1D#
#BOBB#
#HQBY#
#HPP3#
#HWKM#
#CZEK#
#KYYR#
#Y9H4#
#Y6FZ#
#VEEK#
#E6F1#
#6N6W#
#1836#
</PRE><H2><SPAN class="mw-headline" id="15-07-04">15-07-04</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Cryo of overnight cultures and freezing of the cultures for plasmidprep on monday.
</P><H2><SPAN class="mw-headline" id="15-07-06">15-07-06</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Plasmidprep of J23109, -11 &amp; K1575109, -14
</P><P>Prepare sequencing of J23108, J23109, J23111 &amp; K1575107, K1575109, K1575114
</P><H2><SPAN class="mw-headline" id="15-07-07">15-07-07</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>We plated all constitutive and inducible cryos with J61002 backbone on LB+A.
</P><H2><SPAN class="mw-headline" id="15-07-08">15-07-08</SPAN></H2><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>Check last sequencing: no successful construct!!!
</P><P>For characterization: Do overnights in wellplate of J23100, J01, J02, J03, J04, J05, J06, J10, J13, J15, J16, J17, J18 &amp; J19 and K15751xx versions of each. Incubation started 18:30 and 37 °C.
</P><P><I><B>next steps</B></I></P><P>Add glycerol to overnights for freezing.
</P><P>discard plasmid in -20 °C: J23108, J23109, J23111 &amp; K1575107, K1575109, K1575114 from last week.
</P><P>discard cryos in -80 °C: J23108, J23109, J23111 &amp; K1575107, K1575109, K1575114 from last week.
</P><H2><SPAN class="mw-headline" id="15-07-09">15-07-09</SPAN></H2><P><B><SPAN style="color:red">CHARACTERIZATION</SPAN></B></P><P><B>Biotec lab;</B> Attendants: Sven, Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P>We did a cryo wellplate with all verified promoters: 90 µl of 50 % glycerol to the overnight wellplate. Freezed in -80 °C in Bio7. 
</P><P>From this cryo wellplate we inoculated 2 wellplates with LB+Amp, one of these plates with IPTG.
</P><P>We measured fluorescence and OD in the platereader every hour. We measured both OD and fluoresence to understand a possible correlation between both factors.
</P><P>Note: mRFP has its excitation peak at 584nm and its emission peak at 607nm wavelength.
</P><H2><SPAN class="mw-headline" id="15-07-14">15-07-14</SPAN></H2><P><B>Biotec lab;</B> Attendants: Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P><I><B>Characterization:</B></I></P><P>We added 200 µl LB media per well in grey SystemDuetz plates. Ampicilin was added in each well except from 6 C-H &amp; 12 C-H. In one wellplate IPTG (20 µl/20 ml x1000stock). Inoculation via cyroreplicator. Add 3 µl DH5α (as wildtype control) from overnight in 6C+D &amp; 12C+D.
</P><P>We did not do a measurement at the beginning because grey plates do not fit in the platereader. For measurement the cultures get transfered into black wells with glas bottom.
</P><P>Incubation at 30 °C and 300 rpm started at 10:10 a.m.
</P><P>We did an OD/F measurement at 19:00 because there should be some spots with only LB media as reference.
</P><H2><SPAN class="mw-headline" id="15-07-15">15-07-15</SPAN></H2><P><B>Biotec lab;</B> Attendants: Tobi, Laura
</P><P><I><B>consumable material:</B></I></P><P><I><B>Promotor library:</B></I></P><P><I><B>Characterization:</B></I></P><P>We did another OD/F measurement again in the morning at 8:15.
</P><P>Sterilisation of 2 grey and 2 black wellplates with glassbottoms.
</P><H2><SPAN class="mw-headline" id="15-07-17">15-07-17</SPAN></H2><P>OD/F measurment was started. Measurement every 2-3 hours.
</P><H2><SPAN class="mw-headline" id="15-07-18">15-07-18</SPAN></H2><P>Inoculation of one plate in the morning. Measurements every hour.
</P><H2><SPAN class="mw-headline" id="15-07-19">15-07-19</SPAN></H2><P>Last measurement was done in the morning.
</P><H2><SPAN class="mw-headline" id="15-07-20">15-07-20</SPAN></H2><P>Do overnights of J23100, J01, J02, J03, J04, J05, J06, J10, J13, J15, J16, J17, J18 &amp; J19 and corresponding K15751xx versions of each. Incubation started at xxh in iAMB and 37 °C. 
</P><H2><SPAN class="mw-headline" id="15-07-21">15-07-21</SPAN></H2><P>We did a new Cryo wellplate with new layout.
</P><H2><SPAN class="mw-headline" id="15-07-22">15-07-22</SPAN></H2><P>New measurement plates. Incubation at 30 °C and 300 rpm (150 ul in each well) since 14 o'clock.
</P><H2><SPAN class="mw-headline" id="15-07-23">15-07-23</SPAN></H2><P>First measurement at 9:00. Induction of IPTG wellplate. Shaking started at 9:40.
</P><P>Measure each hour. At 10.45; 12.00; 13.15; 14.30; 15.45; 17.00 &amp; 18.15
</P><H2><SPAN class="mw-headline" id="Result">Result</SPAN></H2><P>The characterized promoters are:
</P><P>J23100, J23101, J23102, J23103, J23104, J23105, J23106, J23110, J23113, J23115, J23116, J23117, J23118 &amp; J23119
</P><P>The native Anderson Promoters show the expected relative fluorescence.
</P><P>Plotting of the files:
</P><P>
The characterized promoters with lacI binding site are: 
</P><P>K1575100, K1575101, K1575102, K1575103, K1575104, K1575105, K1575106, K1575110, K1575113, K1575115, K1575116, K1575117, K1575118 &amp; K1575119.
</P><H2><SPAN class="mw-headline" id="Results_Summary"><B>Results Summary</B></SPAN></H2><P>14 different Anderson Promoters are present in backbone J61002 with RFP (E1010) as control between the SpeI &amp; PstI restriction sites. 
</P><P>These 14 Anderson Promoters are also present with an lacI-binding site.
</P><P>The inhibition by lacI works perfectly.
</P><P>The induction with IPTG is not significantly possible.
</P></DIV><DIV class="hexagon-bg"><DIV class="hexagon"><DIV class="colorable fill"><CENTER><A href="#">▲</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="bar"><DIV class="bar-wrap"><DIV class="disclaimer"><A href="mailto:igem@rwth-aachen.de">Contact</A></DIV><DIV class="copyright">CC-BY-SA iGEM Aachen 2015</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>