<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HZAU-China_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HZAU-China/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HZAU-China/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HZAU-China/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HZAU-China/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HZAU-China/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HZAU-China/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P>&lt;!DOCTYPE html&gt;<TITLE>2014HZAU-China</TITLE></P><DIV id="main-wrapper"><NAV id="mainmenu"><UL id="menu"><LI class="dropdown"><A href="https://2014.igem.org/Team:HZAU-China">home</A></LI><LI class="selected dropdown"><A href="https://2014.igem.org/Team:HZAU-China/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Design"><SPAN>-</SPAN>Overview</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Background"><SPAN>-</SPAN>Background</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Input"><SPAN>-</SPAN>Input module</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Processing"><SPAN>-</SPAN>Processing module</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Output"><SPAN>-</SPAN>Output module</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Application"><SPAN>-</SPAN>Application</A></LI></UL><LI class="dropdown"><A href="https://2014.igem.org/Team:HZAU-China/Review">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Overview"><SPAN>-</SPAN>Overview</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Construction"><SPAN>-</SPAN>Construction</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Characterization"><SPAN>-</SPAN>Characterization</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Help"><SPAN>-</SPAN>Help each other</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Protocol"><SPAN>-</SPAN>Protocol</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Labnotes"><SPAN>-</SPAN>Labnotes</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:HZAU-China/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/MOverview"><SPAN>-</SPAN>Overview</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Biological"><SPAN>-</SPAN>Biological processes</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Comparison"><SPAN>-</SPAN>Comparison between different designs</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Analysis"><SPAN>-</SPAN>Simulation and sensitivity analysis</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Circuit"><SPAN>-</SPAN>Design principle of rewirable circuit</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:HZAU-China/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Achievements"><SPAN>-</SPAN>Judgement</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Parts"><SPAN>-</SPAN>Parts</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:HZAU-China/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Team"><SPAN>-</SPAN>Members</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Attributions"><SPAN>-</SPAN>Attributions</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Gallery"><SPAN>-</SPAN>Gallery</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/acknowledgment"><SPAN>-</SPAN>Acknowledgment</A></LI><LI class="dropdown"><A href="https://2014.igem.org/Team:HZAU-China/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Collaboration"><SPAN>-</SPAN>Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Meetup"><SPAN>-</SPAN>Meetup</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Humanities"><SPAN>-</SPAN>Humanities</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Economics"><SPAN>-</SPAN>Social Sciences</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Pubic outreach"><SPAN>-</SPAN>Public outreach</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Safety"><SPAN>-</SPAN>Safety</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Social media"><SPAN>-</SPAN>Social media</A></LI><LI><A href="https://2014.igem.org/Team:HZAU-China/Other"><SPAN>-</SPAN>Other dream</A></LI></NAV></DIV><SECTION id="pagetitle-wrapper"><DIV class="pagetitle-content"><H2>Project</H2></DIV></SECTION><SECTION id="breadcrumb-wrapper"><DIV id="breadcrumb-content"><UL><LI><A href="https://2014.igem.org/Team:HZAU-China">Home</A></LI><LI>Project</LI></UL></DIV></SECTION><SECTION id="content-wrapper"><DIV class="row"><DIV class="eleven columns"><DIV class="offset-by-one columns"><H3 style="text-align:center">1. Overview</H3><P class="highlighttext">
Cells sense the environment, process information, and make response to stimuli. To make cells work well in complex natural environments, lots of processes have to be preset to react to various signals. However, when well-characterized modules are combined to construct higher order systems, unpredictable behaviors often occur because of the interplay between modules. Another significant problem is that complex integrated systems composed of numerous parts may cause cell overload.</P><P class="highlighttext">
We proposed an elegant method to design higher order systems. Instead of merely combining different functional modules, we constructed one integrated processing module with fewer parts by utilizing the common structures between modules. The circuit we designed is a rewirable one and the topological structure of the processing module can be altered to <SPAN style="font-weight:bold;">adapt</SPAN> to environmental change. The basic idea is to rewire the connections between parts and devices to <SPAN style="font-weight:bold;">implement multiple functions</SPAN> with the help of the site-specific recombination systems.</P><P class="highlighttext">
Based on the design principle we put forward, we built two circuits to verify our idea. Each circuit has three modules including an input module, a processing module, and an output module. The input module receives environmental signal and triggers the rewiring of the processing module. The output module monitors real-time processes using fluorescence intensity.</P><P class="highlighttext">
Our design approach may lead to a revolutionary step towards <SPAN style="font-weight:bold;">system integration</SPAN> in synthetic biology. Potential fields of application include organism development, living therapeutics and environment improvement.</P><H3 style="text-align:center">2. Background</H3><P class="highlighttext">Since its inception more than a decade ago, synthetic biology has undergone considerable development and has attained significant achievements with the help of the engineering slant. However, there are still obstacles to build a cell. Engineers try to abstract the DNA sequences into some standard functional parts and assemble them using some principles in electrical engineering. So far, the limited understanding of biological system prevents us to combine parts and modules to create larger scale systems. The complexity of synthetic systems didn’t increase rapidly as the Moore’s law (Purnick and Weiss, 2009). </P><H5>Challenges</H5><P class="highlighttext">There are some common problems that make the circuits we designed not work as our expected. Many failure modes have been collated by Brophy and Voigy in their review (Brophy and Voigt, 2014). In our project, we mainly focus on two modes, <SPAN style="font-weight:bold;">crosstalk and host overload</SPAN>, that emerge especially when we create more sophisticated systems. More specifically, regulators may interact with each other’s targets leading to errors in the desired operation, and the synthetic circuits may compete with natural parts that maintain the normal cellular processes for limited resources.</P><H5>Solution</H5><P class="highlighttext">We designed a time-sharing system that can process information according to the input signal. Cells rewire its synthetic circuit to <SPAN style="font-weight:bold;">alter the topological structure of regulatory pathway</SPAN> when they receive the corresponding stimuli. In this way, we <SPAN style="font-weight:bold;">reuse the existing synthetic module</SPAN> rather than add a new one to implement another function, which reduces the resource cost in running unnecessary function and prevents the interplay between parallel modules. After overcoming these two big problems, our engineered cells are more <SPAN style="font-weight:bold;">versatile</SPAN> and <SPAN style="font-weight:bold;">flexible</SPAN> in information processing. </P><H3 style="text-align:center">3. Input Module</H3><P class="highlighttext">The input module is in charge of the expression of recombinase according to the received signals. When there’s no input signal, there is leaky expression. Such leakage may unexpectedly alter the function of the processing module. So we need <SPAN style="font-weight:bold;">tighter regulation</SPAN>. We cloned the recombinase under the control of the inducible riboregulators (Callura <SPAN style="font-style:italic;">et al.</SPAN>, 2012; Bonnet <SPAN style="font-style:italic;">et al.</SPAN>, 2013). </P><H5>Site-specific recombination</H5><P class="highlighttext">The site-specific recombination system we used in our project is Cre recombinase meditated inversion system. We chose two mutant loxP sites, lox66 and lox71, instead of the wild-type loxP sites. After the first Cre-mediated recombination, one wild-type loxP site and one double mutant loxP site are generated. The double mutant loxP site, lox72, exhibits a very low affinity to Cre recombinase. So it can be regarded as a unidirectional one, or an irreversible one.</P><H5>Riboregulator</H5><P class="highlighttext">In a riboregulator, the cis-repressed RNA is like a lock, and the trans-activating RNA is like a key. When they reach a certain concentration, the trans-activating RNA is designed to target and hybridize to the stem-loop of the crRNA message. The resulting RNA duplex causes a conformational change in the crRNA that unfolds the stem-loop, exposing the RBS and permitting translation. In this way we can achieve a post-transcriptional control.</P><H5>Coherent feedforward loop</H5><P class="highlighttext">We incorporated a coherent feedforward loop into our input module. This motif can provide pulse filtration in which short pulses of signal will not generate a response but persistent signals will generate a response after short delay. So we utilized this property to filter noise.</P><H3 style="text-align:center">4. Processing Module</H3><H5>Motivation</H5><P class="highlighttext">Many engineered cells that worked well in a laboratory environment didn’t work well in complex natural environments. Natural cell can adapt to the environment since they have a hierarchical regulatory network. So we want to integrate multiple processing functions in our engineered cells. Taking the limited resources in cells into account, we construct one integrated processing module with fewer parts by utilizing the common structures between modules, instead of merely combining different functional modules.</P><H5>How we design</H5><P class="highlighttext">We put forward a concept of <SPAN style="font-weight:bold;">rewirable circuit</SPAN>. The topological structure of regulatory pathway can be modified to adapt to environmental change. First, we constructed a transcription regulatory pathway by combining inducible promoters and corresponding regulators. Besides, we added some special sequences into the synthetic circuit. These sites can be recognized by recombinase protein and help to rearrange the parts connection by site-specific recombination. After the rearrangement, the circuit we designed is rewired to achieve another function.</P><P class="highlighttext">We use two examples to demonstrate our idea, and we will describe the general steps about how to design rewirable circuits in <A href="https://2014.igem.org/Team:HZAU-China/Circuit
">the modeling part</A>.</P><P class="highlighttext">In design 1, the processing module is similar to the classic repressilator, which is composed of three NOT gates. cI represses tetR, which represses lacI, which represses cI. What's different is that the transcription directions of these three genes in our design are no longer the same as that in the classic repressilator. This is because we want to put two promoters more closely. And we add a pair of reversed lox sites on either side of the promoters (Fig. 1). </P><P class="figuretext">Figure 1. Circuit in design 1.</P><P class="highlighttext">After rewiring the circuit, the regulatory relationship among these three genes is altered. It is now that cI represses lacI. So cI and lacI inhibit each other. And the <SPAN style="font-weight:bold;">repressilator</SPAN> which is composed of three NOT gates is rewired to be a <SPAN style="font-weight:bold;">toggle switch</SPAN> composed of two NOT gates(Fig. 2).</P><P class="figuretext">Figure 2. Abstract structure in design 1.</P><P class="highlighttext">In design 2, the parts we used are from a quorum sensing system. When the signal molecule AHL binds to luxR proteins, it activates the luxpR promoter and represses the luxpL promoter. LuxI synthesizes more AHL and AiiA contributes to the degradation of AHL. We used luxpR promoter to express luxI and used luxpL promoter to express AiiA (Fig. 3). Then a direct <SPAN style="font-weight:bold;">positive feedback loop</SPAN> and an indirect positive feedback loop formed (Fig. 4). When the inversion happens, it’s now luxpR that will express AiiA and it's luxpL that will express luxI (Fig. 3). So now more AHL will repress its own production, and will activate its own degradation. And this is a <SPAN style="font-weight:bold;">negative feedback loop</SPAN> (Fig. 4).</P><P class="figuretext">Figure 3. Circuit in design 2.</P><P class="figuretext">Figure 4. Abstract structure in design 2.</P><H3 style="text-align:center">5. Output Module</H3><P class="highlighttext">In the first stage, the output module is the fluorescent indicator that can monitor the real-time processes of our system. Once its function is confirmed, we can use other functional parts as output to solve real world problems.</P><H5>RNA level fluorescence</H5><P class="highlighttext">Beside fluorescent proteins, we try to use an RNA-fluorophore complex (Paige <SPAN style="font-style:italic;">et al.</SPAN>, 2011) to monitor the real-time processes. The complex contains RNA aptamers and some corresponding fluorophores. We synthesized the fluorophore 3,5-dimethoxy-4-hydroxybenzylidene imidazolinone (DHMBI), because several aptamers were identified that exhibited markedly different spectral properties when they bound to DHMBI. We also synthesized the 13-2min sequence, one of the aptamers that can interact with DHMBI, with a modified tRNA scaffold, which can stabilize the structure.</P><P class="figuretext">Figure 5. The resulting image under the fluorescent microscope.</P><H3 style="text-align:center">6. Application</H3><H5>Multiple functions integration</H5><P class="highlighttext">Multiple functions integration is the general goal we want to achieve. As the number of system components grows, it becomes increasingly difficult to coordinate component inputs and outputs to produce the overall desired behavior. For this reason, we increase the complexity of the system by reusing the exist parts instead of addition of new parts. Our design allows the cells to run different functions at different time and it will not give extra burden to cells when a function is unnecessary.</P><H5>Organism development</H5><P class="highlighttext">Many researches understand what orchestrates epigenomic changes by Waddington’s model of epigenetic determination of development (Fig. 6) (Mohammad and Baylin, 2010). From a dynamic view, the organism development is like jumping among different attractors. Once the cell falls into a stable steady state, it will be very hard to jump out. Many motifs in developmental network like mutual inhibition and double-positive feedback loop exhibit irreversibility unless the environment has a big change. If the gene circuit that decides the cell fate were rewirable, we could easily reprogram the cell.</P><P class="figuretext">Figure 6. Depiction of potential cell signaling in Waddington's model of epigenetic determination of development.</P><H5>Living therapeutics</H5><P class="highlighttext">The tumor suppressor p53 can induce cell cycle arrest or apoptosis according to degree of DNA damage. It was reported that p53 and its downstream targets applied an oscillation mode to repair DNA damage and chose a bistability mode to trigger apoptosis once the damage cannot be fixed by oscillation mode (Zhang <SPAN style="font-style:italic;">et al.</SPAN>, 2011). These functions were achieved by a very complex systems (Fig. 7).</P><P class="figuretext">Figure 7. A complex mechanism described in previous study.</P><P class="highlighttext">In our design, we can achieve these functions by using only three genes and rewiring their regulatory pathway. We can construct an oscillation into the therapeutic bacterium that colonizes a niche in the human microbiome to maintain homeostasis. And once the the equilibrium is broken, the oscillation will be rewired to be a switch used for next decision.</P><H5>Environment improvement</H5><P class="highlighttext">Some environment projects in synthetic biology utilize an event trigger to keep expressing some special proteins to tackle the environmental problem. These systems often contain a positive feedback loop module that can memorize the received signal and activate the downstream functional protein. After the problem is handled, we don’t need the positive feedback module anymore, but it is difficult to stop this module. In this case, we can rewire the system rather than kill all these meritorious cells. The positive feedback module can be rewired to be a negative feedback module, which is used to maintain the lower steady state or control the population of the engineered cells. Once the environmental problem recurs, it can be rewired into a positive feedback one again.</P><H5>References</H5><P class="highlighttext">Purnick, P. E., &amp; Weiss, R. (2009). The second wave of synthetic biology: from modules to systems. Nature reviews Molecular cell biology, 10(6), 410-422.</P><P class="highlighttext">Brophy, J. A., &amp; Voigt, C. A. (2014). Principles of genetic circuit design. Nature methods, 11(5), 508-520.</P><P class="highlighttext">Siuti, P., Yazbek, J., &amp; Lu, T. K. (2013). Synthetic circuits integrating logic and memory in living cells. Nature biotechnology, 31(5), 448-452.</P><P class="highlighttext">Callura, J. M., Cantor, C. R., &amp; Collins, J. J. (2012). Genetic switchboard for synthetic biology applications. Proceedings of the National Academy of Sciences, 109(15), 5850-5855.</P><P class="highlighttext">Paige, J. S., Wu, K. Y., &amp; Jaffrey, S. R. (2011). RNA mimics of green fluorescent protein. Science, 333(6042), 642-646.</P><P class="highlighttext">Mohammad, H. P., &amp; Baylin, S. B. (2010). Linking cell signaling and the epigenetic machinery. Nature biotechnology, 28(10), 1033-1038.</P><P class="highlighttext">Zhang, X. P., Liu, F., &amp; Wang, W. (2011). Two-phase dynamics of p53 in the DNA damage response. Proceedings of the National Academy of Sciences,108(22), 8990-8995.</P></DIV></DIV></DIV></SECTION><FOOTER><DIV class="row"><DIV class="four columns mobile-one"><P class="copyright-text">© 2014 Huazhong Agricultural University iGEM Team. All rights reserved.</P></DIV><DIV class="eight columns mobile-one"><H5>Contacts</H5><DIV class="six columns mobile-one"><UL><LI class="address-icon">No.1, Shizishan Street, Hongshan DistrictWuhan, Hubei Province430070 P.R.China</LI><LI class="email-icon">Email : hzauigem@gmail.com</LI></UL></DIV><DIV class="six columns mobile-one"><UL><LI class="twitter-icon">Twitter : hzau_igem</LI><LI class="wechat-icon">Wechat : hzauigem</LI><LI class="qq-icon">QQ Group : 313297095</LI><LI class="y-icon">YouTube : hzauigem</LI></UL></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:HZAU-China/Project">http://2014.igem.org/Team:HZAU-China/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HZAU-China/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HZAU-China/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HZAU-China/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HZAU-China/Project&amp;oldid=400267" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>