"
Page
Discussion
View source
History
teams
Log in
 
Team:HZAU-China/Labnotes
From 2014.igem.org
<!DOCTYPE html>
2014HZAU-China
home
Project
-Overview
-Background
-Input module
-Processing module
-Output module
-Application
Wetlab
-Overview
-Construction
-Characterization
-Help each other
-Protocol
-Labnotes
Modeling
-Overview
-Biological processes
-Comparison between different designs
-Simulation and sensitivity analysis
-Design principle of rewirable circuit
Achievements
-Judgement
-Parts
Team
-Members
-Attributions
-Gallery
-Acknowledgment
Outreach
-Collaboration
-Meetup
-Humanities
-Social Sciences
-Public outreach
-Safety
-Social media
-Other dream
Labnotes
Home
Wetlab
Labnotes
Molecule Labnotes
Author: Yihui Zheng
Members: Yihui Zheng, Yajing Zhang, Zhixiang Liu, Xudong Luo, Xiaoxue Guo, Lu Wang, Siqi Li.
Introduction:
We have eight gene circuits to construct (as shown in the Figure 1), and after finishing these circuits, we will integrate some of them into one plasmid to form two repressilators and two detection devices (as shown in the Figure 2a and Figure 2b).
Figure 1. eight gene circuits
Figure 2a. first rewirable circuit and its detection device
Figure 2b. second rewirable circuit and its detection device (quorum sensing)
May
Content: we found the parts we needed in the Plate Kits and did a lot of transformations to get the plasmid for subsequent experiments.
Parts list: BBa_J06602  BBa_E0420  BBa_K886000  BBa_R1051  BBa_P0412  BBa_R0040 BBa_P0451  BBa_R0011  BBa_R0051  BBa_I714075  BBa_J61047  BBa_R0063  BBa_K516022  BBa_K805016  BBa_J37032  BBa_K081019  BBa_K773003  BBa_I13602  BBa_R0061  BBa_R0062  BBa_R0063  BBa_C0060  BBa_B0015  BBa_I718016  BBa_I0462  BBa_P0440  BBa_E0022  BBa_J01010
June
Content: eight gene circuits’ construction
Construction details:
1.Construction of gene circuit one: we used standard method of biobrick connection to put the coding sequence of cI protein behind PLlacI promoter. But we failed to construct it because of some unknown reasons. The sequence results showed that there is either a pretty long and unknown segment which insert into the coding sequence of cI protein or deficiency of the sequence of PLlacI promoter(as shown in the Figure 3). So, we had an assumption that there were little LacI proteins in DH-5α, and it was not much enough to repress PLlacI promoter, resulting in the overexpression of cI protein which may restrain the growth of host.
Figure 3. diagram of sequence results
2.Construction of gene circuit three: Firstly, we used standard method of biobrick connection to put the coding sequence of LacI protein behind lox66 because the length of lox66 was 34bp and it was too small to purify from the enzyme-digested product and then put it in front of LacI protein. Due to the same reason, we put the product of previous step behind PLtet01 promoter to finish this gene circuit. Compared to construction of other gene circuits, it seemed that this gene circuit was the simplest job we’ve made since that we only tried once and took almost eight days to finish this work.
3.Construction of gene circuit two: we constructed this gene circuit by following the method we’ve mentioned above. The only difference from the construction of gene circuit three is that we put the coding sequence of TetR protein behind lox71. By the way, lox66 and lox71 are two recombination sites which are recognized specifically by Cre protein. To our surprise, when we did double-digest to this circuit for verification, we found that the length of product of enzyme-digested was not right and there was an extra stripe that we didn’t know what it was. Results are shown in the Figure 4. According to the results, we had another assumption to explain the phenomenon: there were some recombination systems in DH-5α, resulting in homologous recombination of two strains which had synclastic lox71 site. Definitely, We’ve done a lot of work to verify our assumption which will be explained in the part of characterization.
Figure 4. the stripes which arrows point towards are enzyme-digested product of gene circuit two and the rightest stripe in each figure is DL-5000 marker. There were at least three stripes and the length of extra stripe is wrong.
July
Content: eight gene circuits’ construction
1.Construction of gene circuit four: we used standard method of biobrick connection to put the coding sequence of mCherry protein behind Pλ promoter. And when we planned to construct the part CFP expression controlled by PLlacI promoter, we didn’t find CFP with LVA tag in the Plate Kits, so we decided to get this part through PCR which means that we added a LVA tag behind the coding sequence of CFP. And then, we used standard method of biobrick to connect PLlacI promoter with the coding sequence of CFP to finish the whole circuit. Figure 5 will show the visual effects of mCherry protein expression and CFP expression under the fluorescent microscope.
Figure 5. a was without any excitation light; b and c were mCherry protein expression and CFP expression under the fluorescent microscope
2.Construction of gene circuit five: Because of the complex stem-loop structure of RNA aptamer, it was very hard to get this part by overlapping PCR. So, we found a company to help us to synthetize the sequence of RNA aptamer, and we standardized RNA aptamer and also we connected it with the promoter (lambda cI regulated). To test whether it works or not, we synthetized corresponding fluorophore called DHMBI, and saw the figure under the fluorescent microscope as shown in the output module .
3.Construction of gene circuit six and eight: Construction of gene circuit six and eight: These two gene circuits are familiar in their structure and then, we constructed them simultaneously. Firstly, we divided the circuit into two parts: riboregulator and Cre protein with double terminators. When it came to the synthesis of riboregulator, we designed four specific primers which had almost 20 overlapping sequence and conducted twice PCR to get our riboregulator (more details are shown in our protocol and the input module ). Then, we connected both of them by the standard method of biobrick connection. Figure 6a and 6b are the visual effects of gene circuit six without AHL and gene circuit with AHL under the fluorescent microscope, and Figure 6c is the quantitative result of these two samples and we can find that there is a obvious difference between gene circuit six without AHL and with AHL which is accord to our expectation.
Figure 6. a and b are the visual effects of gene circuit six without AHL and with AHL; c is the quantitative result of these two samples
August
Content: we verified our first assumption that the overexpression of cI protein may restrain the growth of host which cause the failure of construction of gene circuit one, so, we designed another method to construct it and finally made it. Then, we integrated three circuits into one plasmid to form our repressilator.
1.Verification for the first assumption that there were little LacI proteins in DH-5α, and it was not enough to repress PLlacI promoter, resulting in the overexpression of cI protein which may restrain the growth of host:
We designed a primer contained PstI site which could match the middle part of the coding sequence of cI protein, and then, we conducted a PCR by using VF2 primer and the designed primer to get a sequence which can express an inactive cI protein. We put this sequence behind PLlacI promoter through the same way mentioned above and the sequencing results showed that we constructed it and it proved our assumption to some extent. Then, we came up with a different way to construct our first repressilator which will be explained in the following.
2.New way to construct our first repressilator: Due to toxicity of cI protein on host bacteria and the expression of cI protein which was regulated by PLlacI promoter, we came up with a new method that putting the gene circuit three in front of PLlacI promoter because circuit three would express LacI protein to repress the activation of PLlacI promoter and thus, cI protein would not be overexpressed. So, we digested circuit three by EcoRI enzyme and SpeI enzyme and digested PLlacI promoter by EcoRI enzyme and XbaI enzyme, and then do a ligation to get a circuit contained PLlacI promoter controlled by LacI protein. After that, we connected the coding sequence of cI protein with the previous product to get a compound contained circuit one and circuit three. Gel electrophoresis spectrum diagram of plasmid is shown in Figure 7.
Figure 7. plasmid contained circuit one and circuit three (a,b)
September
Content: we constructed the first repressilator and change the vector of detection device from high copy to low copy.
1.The construction of first repressilator: The first repressilator contained three gene circuits: circuit one, two and three, and at that moment, we connected circuit one with circuit three, lacking of circuit two. We tried our best to construct it and due to the special condition that we needed to connect this two part reversely and change the vector of repressilator from high copy to low copy, we designed a new method to connect two parts reversely that we digested circuit one and three by EcoRI enzyme and PstI enzyme, meanwhile, we digested circuit two by EcoRI enzyme and XbaI enzyme and target vector by XbaI enzyme and PstI enzyme, leading to the result that there was a common EcoRI site for ligation between circuit two and the compound contained circuit one and three. We’ve tried at least five times in this construction and we tried to separate this procedure into two steps: connection first and then change the vector, but all tries failed. Surprisingly, in the sixth trial, we may design an optimal system of ligation including the proportion between target DNA and vector, we constructed this repressilator successfully. Gel electrophoresis spectrum diagram of plasmid is shown in Figure 8
Figure 8. plasmid of repressilator
2.Changing the vector of detection device from high copy to low copy: we should change the vectors of two detection devices including the compound contained circuit four and circuit six and the gene circuit eight, and at the same time, we changed the vector of RNA aptamer generator which played the same function as fluorescent protein. The indicator to judge whether it had changed successfully or not was the resistance gene: the vector of high copy had chloramphenicol-resistance gene and the vector of low copy had Ampicillin-resistance gene.
October
Content: we constructed our seventh gene circuit about quorum sensing and tested the intensity of GFP through fluorescent microscope and microplate reader.
Construction of gene circuit seven: We divided this circuit into four parts: AiiA protein generator, LuxI protein generator, LuxR protein generator and GFP generator. Both the parts GFP expression controlled by LuxR promoter and luxR protein generator under  regulation were found in Plate Kits. As for another two parts, we did them the same thing as the construction of gene circuit three. And then, we did the same thing as the construction of our first repressilator to connect three segments to form our seventh gene circuit. To judge whether it was connected correctly and successfully, we co-transformed our gene circuit seven and eight (with Cre protein) and transformed only the gene circuit seven (without Cre protein) as a comparison, the results shown in the Figure 9 and Figure 10 were exactly accord to our expectation which indicated that we constructed it successfully.
Figure 9. a was without any excitation light; b was the phenomenon when we co-transformed the gene circuit seven and eight (with Cre protein); c was the phenomenon when we only transformed the gene circuit seven (without Cre protein) as a comparison.
Figure 10. result of fluorescence intensity
Function test labnotes
1. Input module test
1.1 Cre recombinase function test
Author：Lu Wang
1.1.1 Experimental Background
Our purpose is to confirm the function of Cre recombinase.
1.1.2 Materials and equipment
Materials: Part：mCherry (BBa_J06504) Strain: Escherichia. Coli (DH5α) LB, assorted antibiotics
1.1.3 Method
Figure 1. Our designed device
Co-transform the pSB1A3 and pSB1C3 which we designed as our Confirmation device (as shown in Figure 1) into Escherichia. Coli (DH5α) competent. Culture it overnight from a fresh single colony of Escherichia. Coli (DH5α) strain in LB containing chloramphenicol, ampicillin and IPTG inducer. And wait for about 8 hours to observe the culture’s color.
1.1.4 Result and discussion
Observation on Group Level
Figure 2. Observation on Group Level (the left one is Negative Control contained mCherry without promoter,the middle one is Group One only contained confirmation device,the right one is confirmation device with Cre recombinase)
Figure 3. Observation on Group Level (the left one is Group One only contained confirmation device,the right one is confirmation device with Cre recombinase)
The results indicate that the culture of Negative Control and Group One has become opaque but still keep being pale yellow. The Group Two’s culture has turned red obviously and also opaque.
Therefore, conclusion could be drawn that plasmids which code Cre recombinase in the registry work properly and the confirmation device we assembled is valid.
1.2 Riboregulator function test
Author：Lu Wang
1.2.1 Experimental Background
Confirm if the leakage of lac promoter’s expression will be reduced under the control of riboregulator.
1.2.2 Materials and equipment
Materials: Strain: Escherichia. Coli (DH5α,BL21(DE3)) LB , ampicillin antibiotic Equipment: Multifunctional microplate reader
1.2.3 Method
Figure 4. Designed Device (1)
Figure 5. Designed Device (2)
Firstly, construct these two devices shown in Figure 4 and Figure 5. Respectively transform each of them into Escherichia. Coli strain BL21 (DE3). Pick single colonies and add each of them into 5ml LB liquid with its corresponding antibiotic and incubate it in shaking incubator at 37 ℃,180rpm for 8 hours. Secondly, add 5ul of previous single colony cultural from each tube into 5ml fresh M9 broth. And we divided all the sample tubes into two groups: one was the cells with designed device (1) plasmid and we named it group A. Another was the cells with designed device (2) plasmid and we named it group B. Add 0.1ul IPTG into 12 tubes of both group A and B, while add nothing into the other 12 tubes of both group A and B as control . Incubate them at 37 ℃ for 4 hours. Lastly, measure the fluorescent intensity by the multifunctional microplate reader.
1.2.4 Result and discussion
Figure 6.The data from the enzyme-labeled instrument.
Experimental analysis：
Figure 7.Experimental analysis based on the data from enzyme-labeled instrument
Conclusion:
Riboregulators ensure much lower leakage of the promoters.
1.3 Test of input module's tight regulation
Author：Lu Wang
1.3.1 Experimental Background
Our purpose is to confirm whether the cre will be expressed or not under the control of riboswitch driven by IPTG.
1.3.2 Materials and equipment
Materials: Vector-pSB1A3, pSB1C3 Strain: Escherichia. Coli (DH5α) LB , assorted antibiotics Equipment: Multifunctional microplate reader
1.3.3 Method
Figure 8. input device
Figure 9. verification device
Process:
1. Co-transform our input device and verification device into BL21(DE3) competent cell and plate it onto a selective plate, incubate it at 37 ℃ for 14 hours.
2. Pick 3 different single colonies numbered 1, 2, 3 into 3 tubes of LB broth with its corresponding antibiotic and incubate it respectively at 37℃ for 8 hours.
3. Add 5ul of previous single colony cultural from each tube to two new tubes named A,B respectively with M9 broth in it. Add 0.1ul IPTG to all A tubes (1A, 2A, 3A) while add nothing to all B tubes (1B, 2B, 3B). Incubate them at 37℃ for 4 hours.
4. Measure the fluorescent intensity by the enzyme-labeled instrument.
Prediction:
a) If the fluorescent degree in cultural tubes with IPTG showed much less weaker than in the tubes without it, then it indicated that the device had achieved its function for reducing the expression of Cre protein as expected.
b) If the fluorescent degree in the cultural tubes with IPTG showed almost same or even stronger than in the tubes without it, it meant that the devices had lost its function. However, there may be 3 reasons which are responsible for its failure. The first reason may be the failed function of riboregulator to repress the translation process. Secondly, despite the strong repression dueing to the riboregulator, the leakage expression of Cre protein can still express a little bit Cre protein which may cause the same result of a device without riboregulator. Thirdly, the low production of lacI in BL21(DE3) bacteria is unable to repress the operon efficiently which leads to the high expression of both taRNA and cre mRNA and thus weaken the function of riboregulator.
1.3.4 Result and discussion
Figure 10. Fluorescent degree (OD600)
Figure 11. The rigour detection of output device
As shown above, even though the addition of riboregulator does reduce a little of the expression of Cre protein, the difference between control group and experimental group is still very small which means that the riboregulator can hardly exert some critical influence on the control of the expression of cre gene.
2. Output module test
2.1 The Detection of Aptamer (Qualitative Detection)
Author: Yajing Zhang, Siqi Li
2.1.1 Experimental Background
Our purpose is to confirm whether the aptamers is available or not.
2.1.2 Materials and equipment
Material: DMHBI, M9 broth, Escherichia. Coli（DH5α）
Equipment: Fluorescence microscope
2.1.3 Method
The synthesis of DMHBI
N-Acetylglycine (5.26 g, 0.045mol), anhydrous sodium acetate (3.69 g, 0.045mol), 4-hydroxy-2,3-dimethoxybenzaldehyde (8.19 g, 0.045mol), and acetic anhydride (15 ml) were stirred at 90℃for 2 h. After allowing the reaction to cool to room temperature, cold ethanol (20 ml) was added while still stirring and the reaction was left stirring overnight at 4℃. The resulting crystalline solid was then washed with coldethanol, hot water, hexanesand dried to afford 9.65 g (yield 70%) of 3as a pale yellow solid: H NMR (500 MHz,CDCl_3) δ7.56 (s, 2H), 7.18 (s, 1H), 3.98 (s, 6H), 2.54(s, 3H), 2.39(s, 3H); LC/MS (LC: gradient 20-95% MeCN[0.1% HCO_2 H] over 2.5min, 0.5 ml/min flow rate, MS: ESI^+): retention time, 2.15 min; purity, 95%; 306.36 〖[M+H]〗^+. Then we can get Compound 3.Compound 3 (1.12 g, 0.005mol) was refluxed with 15 ml of ethanol, 1 ml of 40% aqueous methylamine, and 700 mg of potassium carbonate for 4h.  The reaction mixture was removed from heat and upon cooling formed an orange precipitate. The precipitate containing the product was filtered and washed briefly with cold ethanol. The precipitate was then redissolved in a 1:1 mixture of ethyl acetate and 500 mM sodium acetate pH 3.0. The organic layer was separated, dried with anhydrous sodium sulfate and solvent was removed under reduced pressure to yield 717 mg (yield 52%) of DMHBI as an orange solid: 1H NMR (500MHz, DMSO-d_6) δ7.62 (s, 2H), 6.90 (s, 1H), 3.80 (s, 6H), 3.09 (s, 3H), 2.34 (s, 3H).
Figure 12. The process during synthesis
Then the DMHBI should be stored in such conditions: Prevent from light and oxygen Preserve it in low temperature
According to the literature, we will get an orange solid. But in reality, we get something dark brown. After detection, our products are in liquid state and we get 40% in purity of product. In order to get more product in high purity, we invite ShaoQing Zhuang who is a student in Huazhong University of Science and Technology to help us with the synthesis of the DMHBI. And finally we get more than 95% in purity of product.
DH5α cells (TransGen Biotech) were transformed with 10μl of plasmid DNA expressing chimeras of the human tRNALys3 scaffold fused to the Spinach aptamer’s sequence. Cells were plated, grown overnight and single colonies were picked for inoculation overnight in Luria Broth containing chloramphenicol. At =0.8, 150μl culture was removed, pelleted and resuspended in 100μl M9 minimal media and cultivate for 1 h at 37 ℃. Cells were washed twice and incubated with 200μM DFHBI in M9 media for 5 min. We tested the spinach RNA aptamer in the Tanon 1600R Gel Imaging System under the 302nm, the control groups are the DMHBI and DMHBI + control RNA. The result is below (Figure 15) and we also observed the spinach RNA aptamer by using the fluorescence microscope, the result is in the Figure 13.
2.1.4 Result and discussion
Figure 13. The result of detecting E.coli with promoter  and aptamer by blue exciting light is green point. (10*40, blue filter).
Figure 14. The picture of reference.
Figure 15. The picture of test.
Because the brightness of the RNA-fluorophore complex is very low.(According to reference report, it is only decimusly brighter than the GFP ) And the half- life of RNA is just few minutes. So the concentration of RNA-fluorophore complex is at low level in cells. The Figure 15 has shown the difference among three samples.The sample of intermediate is E.coli with Pλ promoter and aptamer,and it is the brightest. The sample of downmost is E.coli with Pλ promoter, and it is brighter than the sample of uppermost . The sample of uppermost is control test sample, only with M9 medium and DMHBI ,and it is the darkest.We conclude that the aparter is available.
2.2 Fluorescent proteins with LVA tag
2.2.1 Experimental Background
For further use in our project, we are going to test whether fluorescent proteins with LVA tag can degrade rapidly or not.
2.2.2 Materials and equipment
Material: DMHBI, M9 broth, E.coli (DH5α)
Equipment: Fluorescence microscope
2.2.3 Method
Figure 16. Output module of plan 2
2.2.4 Result and discussion
Figure 17. a: without any excitation light b: mCherry c: CFP
3. Processing module test
Author :Yajing Zhang, Yihui Zheng
3.1 Experimental Background
To test our processing module.
3.2 Materials and equipment
Material: Parts: BBa_K805016, BBa_R0062, BBa_R0063, BBa_K516022, BBa_I0462, BBa_I718016, BBa_K886000, BBa_I714075, BBa_B0015, BBa_J61047 Strain: Escherichia. Coli (BL21 (DE3)) LB , assorted antibiotics
Equipment: Perkin-Elmer plate reader
3.3 Method
Figure 18. The testing of processing device.
The processing plasmid and input plasmid were co-transformed into the Escherichia. Coli (BL21 (DE3)) strain as experimental group and only the processing plasmid was transformed into the Escherichia. Coli (BL21 (DE3)) as control.
Single colonies were used to inoculate 5 mL LB media supplemented with 50ug/ mL ampicillin , 50ug/mL kanamycin and grown overnight at 37°C.And the control was used to inoculate 5 mL LB media supplemented with 50ug/mL kanamycin.
The next morning, overnight cultures were inoculated with 5 mL fresh LB cultures supplemented with antibiotics at 1:100 dilution. Cultures were grown at the desired temperature in constant temperature shaker following 3 h or 5h of growth (Both the experimental group and control were set two contrats further to test if the inducer, IPTG for experimental group and AHL for control group, would make influnece. ), then cells were plated in triplicate in a 96-well BD-Falcon plate. The optical density at 600 nm and fluorescence (X excitation = 475 nm and X emission = 515 nm) were measured using a Perkin-Elmer plate reader every 15 or 20 minutes.
When cells grow to mid-stage, one of the experiment groups was induced with 100uM IPTG, the optical density at 600 nm and fluorescence (X excitation = 475 nm and X emission = 515 nm) were measured using a Perkin-Elmer plate reader every 15 or 20 minutes after 3 hours ,at the same time cells were grown at 30°C. (Figure 17)
3.4 Result and discussion
Figure 19. The value of fluorescence intensity changes over time, the time begins with all the E.coli growing to mid-log.
The result indicated the processing device without Cre recombinase was in high fluorescence and the processing device with Cre recombinase was in low fluorescence, it works as expected. And whether it was induced by IPTG had no difference, the reason was mentioned in the tight regulation test for input module, because of the leakage of Cre recombinase. We also found the experimental groups grew better than the control with only processing plasmid. (Figure 18) Maybe the positive feedback caused more loading, but the negative feedback didn't. So we chose the data when all groups of the cells grow to mid-log phase.
Figure 20. the growth curve of the E.coli , a: the control. b, the control with 100nM AHL. c: the experimental group without IPTG inducer. d: the experimental group with 100uM IPTG inducer.  e :the experimental group with 100uM IPTG inducer after growing to mid-log. f: the control growing to stable phase.
4. Other work
4.1.Interaction work(pCusC, PpcoA sequencing and promoter test)
Author：Liu Tian
4.1.1 Experimental Background
The iGEM encourages us to communicate and help with each other. So, when we heard HUST-China has something wrong with their promoter, we decided to give a hand. The one problem they met was the failure in sequencing their promoters in pET28a, and another was that they put mRFP under the promoter to do a promoter test but without result. So we got A&B promoters with mRFP from them.
4.1.2 Proposes
pCusC & PpcoA promoters sequencing pCusC & PpcoA promoters test
4.1.3 Materials and equipment
Materials pCusC & PpcoA promoters with mRFP in pET28a plasmid LB medium Kanamycin
Equipment Multifunctional microplate reader Shaking incubator
4.1.4 Method
Sequencing
We designed the 5’-agatcgggctcgccacttcg-3’ as the reverse primer to sequence the promoters. And Tsing Ke Biology Technology Company helped us to synthesize the primer and sequence those plasmid. (You can get the sequencing results of two samples at supplement 1.) After analyzing the sequencing result for sample pCusC , we found that the T7 promoter had't been digested, and no pCusC sequence in the plasmid. But mRFP had been successfully ligated. So we can observed the red fluorescence due to the expression of mRFP driven by T7 promoter. Therefore, in the subsequent promoter test, we abandoned samples of the pCusC. As for the PpcoA, we successfully found PpcoA and mRFP. However, at the end of PpcoA's sequencing result, there is 6 more base-pairs than HUST’s sequence. We took PpcoA promoter to detection.
Promoter test
Add 2mL LB culture medium, 2uL Kanamycin (50mg/L) and 20uL bacterial samples to a 5mL centrifuge tube. Shaken overnight in the 37°C shaking incubator and set the rotational speed at 180 rpm/min.
Add CuSO4 solution to induce in concentration of 0, 0.02, 0.1 and 1mM. Set three repetitions for each concentration. Cultivate in the 37°C shaking incubator at the rotational speed of 180 rpm/min for 4 hours.
Add 200uL bacterial samples from each centrifuge tube to the 96-well plate. Set two copies for each centrifuge tub. Meanwhile, add LB culture medium containing 1.0mM CuSO4 or not as blank controls.
Read the OD600 and fluorescent intensity (with the emission wavelength at 607nM and excitation wavelength at 584nM) using a multifunctional microplate reader.
Divided the fluorescent intensity result using the value of OD600. Record the data for compare and analysis.
4.1.5 Result and discussion
Figure 21. pCoa Promoter Test
When we added the 1.0mM Cu to bacterium solution，we observed the color change of the visible to the naked eye. So we tried to test the 1.0mM copper ion in LB medium. Then we found its fluorescence value and the value of OD600 with only LB medium samples differred greatly. So we decided to use the 1.0mM copper ion in LB medium as the blank samples of the 1.0mM control. The result shown in figure presents that promoter strength increases with the rising of copper ion concentration below 0.1mM, and then drops down once above 0.1mM.
4.2 Promoter test
Author: Yuan Lu
4.2.1 Experimental Background
The three devices used in our project are concerned with plenty of functions, such as repessilator, feedback and feedforward. Since the difference between frequencies of initiation of the promoters has critical impact on the functions of these processing module, it is important to measure the relative frequencies of initiation between these promoters.
Our purpose is to measure the relative initiation frequencies of promoters used in our devices, which are BBa_J23119、BBa_R1051、BBa_R0062、BBa_R0040、BBa_R0063 and BBa_R0011.
4.2.2 Materials and equipment
Materials Parts:  Promoters (BBa_J23119、BBa_R1051、BBa_R0062、BBa_R0040、BBa_R0063、BBa_R0011), mCherry (BBa_J06504), RBS (BBa_B0034), Terminator (BBa_B0015) Strain: Escherichia Coli (DH5α) LB , assorted antibiotics
Equipment Multi-function Enzyme-labeled instrument
4.2.3 Method
Using BioBrick method, we constructed our detection device by ligating each of the six promoters (LuxpR, LuxpL, BBa_J23119, pcI, plac, ptet) to mCherry (RFP). After transformation, we set up overnight culture from single colonies of Escherichia. Coli (DH5α) strain in LB medium. After 8 hours, we measured the fluorescent degree of each of the samples by using enzyme-labeled instrument.
4.2.4 Experimental Results and Discussion
Figure 22. The detection of the relative strength of the promoters
Analysis:
Figure 23. The detection of the relative strength of the promoters
Based on the data from Figure 22, we do the statistical analysis. With analysis diagram shows in Figure 23,  we can conclude that the rank of relative strength of the promoters from strong to weak is: BBa_R0063、BBa_R0011、BBa_J23119、BBa_R1051、BBa_R0040、BBa_R0062.
4.3 Extra characterization of Part
Author：Lu Wang
4.3.1 Experimental Background
Cre recombinase is the type I topoisomerase found in phage P1 that can catalyze site-specific recombination between loxP sites. Lox71 is a mutation of loxP site which can also be specifically recognized by Cre recombinase. Our purpose is to confirm that unexpected recombination of lox71 site exits through sequencing result and observation under fluorescence microscope.
4.3.2 Materials and equipment
Materials Part: Lac Promoter; Lox71 site (BBa_K886000); mCherry (BBa_J06504) Strain: Escherichia. Coli(DH5α, DH10b, BL21(DE3)) Glycerinum, LB , assorted antibiotics
Equipment Fluorescence microscope
4.3.3 Method
Device Assembly
Ligate the Lac promoter with lox71 site and lox71 site with mCherry by using the BioBrick assembly.
Figure 24. Confirmation device
Join the Lac promoter-lox71 and lox71-mCherry fragments with double digestion and ligation. Transform the confirmation device into Escherichia. Coli ( DH5α, DH10B), and set up overnight culture from fresh single colonies of Escherichia. Coli( DH5α) strain in Luria Broth containing chloramphenicol and IPTG inducer at 37℃ ,180 rpm.
Observe the culture’s color and take a small amount under the fluorescence microscope to observe the single bacteria. Extract the plasmids from single colonies and sequence the plasmid with general purpose primer.
Co-transform the confirmation device and plasmids that code Cre recombinase into Escherichia. Coli(DH5α), set up overnight culture from fresh single colonies of Escherichia. Coli( DH5α) strain in Luria Broth containing chloramphenicol and IPTG inducer at 37℃ ,180 rpm.
4.3.4 Experimental Results and Discussion
After cultivate with IPTG in the 37°C shaking incubator at the rotational speed of 180 rpm/min for 24 hours, we used the fluorescence microscope to observe bacteria and you can see result at Figure 25(DH5α) and Figure 26（DH10B）below.
Figure 25. unexpected recombination in DH5α
Figure 26. Unexpected recombination in DH10B
At the same time, the samples were sequenced and anything was OK. In summary, we confirmed the unexpected recombination of lox71 site even if the device is in the stable E.coli strains such as the DH5α and DH10B. But the probability of its occurrence is extremely low. And it doesn’t affect the sequencing result in a short time. The tetR protein might be poisonous to the E.coli, when the promoter is deleted by lox71 site, the bacteria will get the survival advantage and quickly occupy the dominant position in LB medium. Therefore, we recommend that you save the plasmid rather than bacteria when you use the cre/lox system.
Reference：
Paige J S, Wu K Y, Jaffrey S R. RNA mimics of green fluorescent protein[J]. Science, 2011, 333(6042): 642-646.
© 2014 Huazhong Agricultural University iGEM Team. All rights reserved.
Contacts
No.1, Shizishan Street, Hongshan District
Wuhan, Hubei Province
430070 P.R.China
Email : hzauigem@gmail.com
Twitter : hzau_igem
Wechat : hzauigem
QQ Group : 313297095
YouTube : hzauigem
Retrieved from " http://2014.igem.org/Team:HZAU-China/Labnotes "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
