Team:NPU-China/Demonstrate
menu
HOME
Project
Overview
Background
Mini                                     mtDNA Design
Synthesis
Demonstrate
Model
Future work
Human Practices
Intergrated
Public                                     Engagement
Collaborations
Parts
Parts                                     Overview
Basic                                     Parts
Composite                                     Parts
Parts                                     Collection
Improve
Lab
Notebook
Protocols
Safety
InterLab
Team
Team                                     Members
Attributions
Acknowledgements
For Judge
Map
Medal
Aim to Demonstrate MitoCRAFT
After a year of hard work, we have been advancing the cutting edge of MitoCRAFT.
With the aid of Prof. Jiang Huifeng and his research group, we completed the entire design work of the MitoCRAFT    genome based on the wild-type S. cerevisiae mitochondrial genome.
Fig. 1 Complete genome structure of MitoCRAFT
In particular, what we crave is not a simple mitochondrial genome, instead, we yearn to construct a standard-mode    mitochondrial chassis that is widely used like S. cerevisiae for all sorts of functional verification and expansion    related to mitochondria. In other word, we desire a powerful bio toolbox.
Upon finishing the design of MitoCRAFT, we immediately started the work of de novo synthesis of the MitoCRAFT genome    with corresponding primers.
In the first place, a series of condition optimizations have enabled us to successfully complete the de novo    synthesis of 22 primary fragments. The results are as follows:
Fig. 2-1 Gel electrophoresis of first-stage fragments.
M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
L1:2778bp; L2:2265bp; L3:2883bp; L4:1759; L5:2795bp.
Fig. 2-2 Gel electrophoresis of first-stage fragments.
M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
L6:2288bp; L7:1142bp; L8:1368bp; L9:1453bp;L10:1376bp; L11:1904bp.
Fig. 2-3: Gel electrophoresis of first-stage fragments.
M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
L12:2377bp; L13:1376bp; L14:1053bp; L15:1872bp; L16:1579bp; L17:1763bp.
Fig. 2-4 Gel electrophoresis of first-stage fragments.
M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
L18:2370bp; L19:3306bp; L20:2007bp; L21:525bp; L22(GFP):870bp.
Afterwards, the introduction of PTRCCS system, in the light of the current experimental results, initially proved    the verification of the completion of the synthesis of the entire MitoCRAFT genome:
We transformed the fragment with Overlap (including part of the primary fragment and part of the secondary fragment)    and the linearized vector containing the PTRCCS system into the S. cerevisiae BY4743 strain. The positive clone was    picked to extract the S. cerevisiae genome into a specific E. coli EPI300 strains and the positive bacteria of E.    coli were picked for colony verification. We designed interface validation primers for all 22 primary fragments and    their 23 interfaces to the vector. The following are the results of EPI300-MitoCRAFT-A4 colony validation with a    relatively satisfactory performance:
Fig. 4-1 Gel electrophoresis of A4 plasmid gap verification.
M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
L1: pGF-21-1(954bp); L2: 1-2(496bp); L3: 2-3(501bp);
L4: 3-4(686bp);L6: 4-5(529bp);L8:6-7(537bp); L9:7-8(598bp).
L1 stands for the interface verification from the downstream fragment of the pGF Vector across the primary fragment    of the 21st fragment to the 1st fragment of the primary fragment. L1 is the verification results of the two    interfaces, proving the correctness of the bands, which is to say, both interfaces of pGF-21-1 were correctly    connected.
Fig. 4-2 Gel electrophoresis of A4 plasmid gap verification..M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY     BIOTECH); L11:9-10(563bp); L13:11-12(521bp);L14:12-13(414bp);L15:13-14(866bp);     L17:15-16(435bp);L18:16-17(873bp);L19:17-18(643bp);L20:18-19(406bp);
L22:20-gfp(529bp).
Fig. 4-3 Re-verification of some gap in A4 or A17. M is NormalRun™Prestained 250bp-II DNA ladder;L1:A4     5-6(683bp);L3:A4 gfp-pGF(472bp).
Fig. 4-4 Re-verification of some gap in A4.M is NormalRun™Prestained 250bp-II DNA ladder;     L3:A4 8-9(737bp);L7:A4 19-20(600bp);L9:A4 14-15(875bp);
From the results of the above gel electrophoreses, we can derive that for this clone, only the interface of the    10-11 primary fragment has not been effectively detected and that the PCR results of other clones at this position    are also not obvious, rendering us unable to determin whether the problem was caused by PCR verification or improper    connection. Considering that all ports upstream and downstream were detected to be correctly connected, we can    conclude that we have basically completed the assembly of the MitoCRAFT genome.
We are currently expanding the strain incubation. We plan to use the switch of copy number mode function of the    PTRCCS system to obtain a certain concentration of the plasmid, then verify the sequence accuracy of the MitoCRAFT    genome carried by the strain through methods like sequencing.
In view of our current experimental results, we still have a lot of work to do to ultimately implement the actual     application of MitoCRAFT. The difficulty of synthesizing MitoCRAFT genomes has turned out far beyond our     expectation. Our team members have been conducting PCR since they entered the lab until now, but hopefully, we are     really glad that our MitoCRAFT genome is almost ready to be a triumph. We also look forward to continuing to     explore     this cutting-edge subject after Giant Jamboree. Our specific work plans for the future have been presented on the     Future Work page. We sincerely welcome all iGEMers to contact us, sharing with us your unique experience and     jointly     creating a bright future for MitoCRAFT and synthetic biology.
