<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NPU-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NPU-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A href="#" data-activates="slide-out" class="button-collapse hide-on-large-only"><I class="medium material-icons">menu</I></A></DIV><UL id="slide-out" class="side-nav fixed" style="margin-left: 0px;"><LI class="zd"><A href="https://2018.igem.org/Team:NPU-China" class="zd collapsible-header waves-effect">HOME</A></LI><LI class="zd1"><A class="za1 collapsible-header waves-effect">Project</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Overview" id="Overview" class="waves-effect">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Background" id="Background" class="waves-effect">Background</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Mini_mtDNA_Design" id="Mini_mtDNA_Design" class="waves-effect">Mini
                                    mtDNA Design</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Synthesis" id="Synthesis" class="waves-effect">Synthesis</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Demonstrate" id="Demonstrate" class="waves-effect">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Model" id="Model" class="waves-effect">Model</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Future_work" id="Future_work" class="waves-effect">Future
                                    work</A></LI></UL></DIV><LI class="zd2"><A class="za2 collapsible-header waves-effect">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Human_Practices" id="Human_Practices" class="waves-effect">Intergrated</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Public_Engagement" id="Public_Engagement" class="waves-effect">Public
                                    Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Collaborations" id="Collaborations" class="waves-effect">Collaborations</A></LI></DIV><LI class="zd3"><A class="za3 collapsible-header waves-effect">Parts</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Parts" id="Parts" class="waves-effect">Parts
                                    Overview</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Basic_Part" id="Basic_Part" class="waves-effect">Basic
                                    Parts</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Composite_Part" id="Composite_Part" class="waves-effect">Composite
                                    Parts</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Part_Collection" id="Part_Collection" class="waves-effect">Parts
                                    Collection</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Improve" id="Improve" class="waves-effect">Improve</A></LI></DIV><LI class="zd4"><A class="za4 collapsible-header waves-effect">Lab</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Notebook" id="Notebook" class="waves-effect">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Protocols" id="Protocols" class="waves-effect">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Safety" id="Safety" class="waves-effect">Safety</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/InterLab" id="InterLab" class="waves-effect">InterLab</A></LI></DIV><LI class="zd5"><A class="za5 collapsible-header waves-effect">Team</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Team_Members" id="Team_Members" class="waves-effect">Team
                                    Members</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Attributions" id="Attributions" class="waves-effect">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Acknowledgements" id="Acknowledgements" class="waves-effect">Acknowledgements</A></LI></DIV><LI class="zd6"><A class="za6 collapsible-header waves-effect">For Judge</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Map" id="Map" class="waves-effect">Map</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Medal" id="Medal" class="waves-effect">Medal</A></LI><MAIN><DIV class="container" style="padding-top:50px;"><DIV class="center"><H4>
				Aim to Demonstrate MitoCRAFT
			</H4></DIV><H5>
			After a year of hard work, we have been advancing the cutting edge of MitoCRAFT. 
			With the aid of Prof. Jiang Huifeng and his research group, we completed the entire design work of the MitoCRAFT
			genome based on the wild-type S. cerevisiae mitochondrial genome.
		</H5><DIV class="center" style="padding-top:10px;"><H6>Fig. 1 Complete genome structure of MitoCRAFT</H6></DIV><H5>
			In particular, what we crave is not a simple mitochondrial genome, instead, we yearn to construct a standard-mode
			mitochondrial chassis that is widely used like S. cerevisiae for all sorts of functional verification and expansion
			related to mitochondria. In other word, we desire a powerful bio toolbox.
			Upon finishing the design of MitoCRAFT, we immediately started the work of de novo synthesis of the MitoCRAFT genome
			with corresponding primers. 
			In the first place, a series of condition optimizations have enabled us to successfully complete the de novo
			synthesis of 22 primary fragments. The results are as follows:
		</H5><DIV class="center" style="padding-top:10px"><H6>Fig. 2-1 Gel electrophoresis of first-stage fragments. 
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH); 
				L1:2778bp; L2:2265bp; L3:2883bp; L4:1759; L5:2795bp.
			</H6></DIV><DIV class="center" style="padding-top:10px"><H6>Fig. 2-2 Gel electrophoresis of first-stage fragments. 
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH); 
				L6:2288bp; L7:1142bp; L8:1368bp; L9:1453bp;L10:1376bp; L11:1904bp.
			</H6></DIV><DIV class="center" style="padding-top:10px"><H6>Fig. 2-3: Gel electrophoresis of first-stage fragments. 
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH); 
				L12:2377bp; L13:1376bp; L14:1053bp; L15:1872bp; L16:1579bp; L17:1763bp.
			</H6></DIV><DIV class="center" style="padding-top:10px"><H6>Fig. 2-4 Gel electrophoresis of first-stage fragments. 
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH); 
				L18:2370bp; L19:3306bp; L20:2007bp; L21:525bp; L22(GFP):870bp.
			</H6></DIV><H5 style="padding-top:30px">
			Afterwards, the introduction of PTRCCS system, in the light of the current experimental results, initially proved
			the verification of the completion of the synthesis of the entire MitoCRAFT genome: 
			We transformed the fragment with Overlap (including part of the primary fragment and part of the secondary fragment)
			and the linearized vector containing the PTRCCS system into the S. cerevisiae BY4743 strain. The positive clone was
			picked to extract the S. cerevisiae genome into a specific E. coli EPI300 strains and the positive bacteria of E.
			coli were picked for colony verification. We designed interface validation primers for all 22 primary fragments and
			their 23 interfaces to the vector. The following are the results of EPI300-MitoCRAFT-A4 colony validation with a
			relatively satisfactory performance:
		</H5><DIV><H6>
				Fig. 4-1 Gel electrophoresis of A4 plasmid gap verification. 
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH); 
				L1: pGF-21-1(954bp); L2: 1-2(496bp); L3: 2-3(501bp); 
				L4: 3-4(686bp);L6: 4-5(529bp);L8:6-7(537bp); L9:7-8(598bp).

			</H6></DIV><H5 style="padding-top:30px">
			L1 stands for the interface verification from the downstream fragment of the pGF Vector across the primary fragment
			of the 21st fragment to the 1st fragment of the primary fragment. L1 is the verification results of the two
			interfaces, proving the correctness of the bands, which is to say, both interfaces of pGF-21-1 were correctly
			connected.
		</H5><DIV><H6>
				Fig. 4-2 Gel electrophoresis of A4 plasmid gap verification..M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY
				BIOTECH); L11:9-10(563bp); L13:11-12(521bp);L14:12-13(414bp);L15:13-14(866bp);
				L17:15-16(435bp);L18:16-17(873bp);L19:17-18(643bp);L20:18-19(406bp);
				L22:20-gfp(529bp). </H6></DIV><DIV><H6>
				Fig. 4-3 Re-verification of some gap in A4 or A17. M is NormalRun™Prestained 250bp-II DNA ladder;L1:A4
				5-6(683bp);L3:A4 gfp-pGF(472bp).
			</H6></DIV><DIV><H6>
				Fig. 4-4 Re-verification of some gap in A4.M is NormalRun™Prestained 250bp-II DNA ladder;
				L3:A4 8-9(737bp);L7:A4 19-20(600bp);L9:A4 14-15(875bp);

			</H6></DIV><H5 style="padding-top:30px">
			From the results of the above gel electrophoreses, we can derive that for this clone, only the interface of the
			10-11 primary fragment has not been effectively detected and that the PCR results of other clones at this position
			are also not obvious, rendering us unable to determin whether the problem was caused by PCR verification or improper
			connection. Considering that all ports upstream and downstream were detected to be correctly connected, we can
			conclude that we have basically completed the assembly of the MitoCRAFT genome. 
			We are currently expanding the strain incubation. We plan to use the switch of copy number mode function of the
			PTRCCS system to obtain a certain concentration of the plasmid, then verify the sequence accuracy of the MitoCRAFT
			genome carried by the strain through methods like sequencing.
		</H5><H5 style="padding-top:30px"><B>
				In view of our current experimental results, we still have a lot of work to do to ultimately implement the actual
				application of MitoCRAFT. The difficulty of synthesizing MitoCRAFT genomes has turned out far beyond our
				expectation. Our team members have been conducting PCR since they entered the lab until now, but hopefully, we are
				really glad that our MitoCRAFT genome is almost ready to be a triumph. We also look forward to continuing to
				explore
				this cutting-edge subject after Giant Jamboree. Our specific work plans for the future have been presented on the
				Future Work page. We sincerely welcome all iGEMers to contact us, sharing with us your unique experience and
				jointly
				creating a bright future for MitoCRAFT and synthetic biology.</B></H5></DIV></MAIN></BODY></HTML>