Team:NPU-China/Background
menu
HOME
Project
Overview
Background
Mini                                     mtDNA Design
Synthesis
Demonstrate
Model
Future work
Human Practices
Intergrated
Public                                     Engagement
Collaborations
Parts
Parts                                     Overview
Basic                                     Parts
Composite                                     Parts
Parts                                     Collection
Improve
Lab
Notebook
Protocols
Safety
InterLab
Team
Team                                     Members
Attributions
Acknowledgements
For Judge
Map
Medal
Preface
“The simplest life” is the frontier subject of synthetic biology, incentivizing numerous scientists represented by    J. Craig Venter to devote themselves into the exploration of the limits of gene-scale natural life, finding more    mysteries of life during the process. [1]
The following are background materials for mitochondrial research and synthetic genomics research pertaining to our    project:
1. Mitochondrion
Mitochondrion is an organelle with a bilayer membrane structure in eukaryotic cells. It is a cell-supplied workshop    and plays a pivotal role in cell signaling as well as cell differentiation and apoptosis. Mitochondrion is one of    the two organelles with independent genomes, whose genomes are called mtDNA [1]. The mitochondrial genome lengths    vary widely among dissimilar species, ranging from a few K in mammals to hundreds of K in plants.
Figure 1. Mitochondrial genome length of different species
2. S.cerevisiae Mitochondrial genome
That S.cerevisiae can survive the loss of the mitochondrial genome is reliably a property that makes    Saccharomyces    cerevisiae an excellent material for studying mitochondrial-deficient diseases. The mitochondrial genome of the S.    cerevisiae strain S288C we used is 85,779 bp in length and there are approximately 50-200 copies in haploid    Saccharomyces cerevisiae cells. This genome consists of 35 genes encoding the components of cytochrome oxidase    (cox1, cox2 and cox3), cytochrome b (cob), 3 subunits of ATPase (atp6, atp8 and atp9), two ribosomal RNA subunits    (rnl and rns), one ribosomal protein gene (var1), the rpm1 gene for the RNA subunit of RNase P and 24 tRNA genes    [2], chiefly controlling the process of the electron transport chain and the transcription and translation of the    proteins encoded by mitochondrial genomes. The mitochondrial genome scales of the genus Saccharomyces bear a huge    difference. For example, Candida glabrata has a genome of merely about 20K without obvious intergenic regions, which    offers a splendid reference for the simplification of S. cerevisiae mitochondrial genomes.
3. Synthetic genomics
Being a bottom-up research method, synthetic genomics refers to the whole process of the de novo chemical synthesis    of the entire genome or the majority of the genomes through a series of techniques, whose emergence has made it    possible to simultaneously change various aspects of a biological material of a living organism, thus becoming an    important subject of the synthesis when it comes to complex functional systems and new organisms in the field of    synthetic biology. [3]
After decades of development, scientists have achieved de novo synthesis from thousands of bp of viral genomes to    several million bases of S. cerevisiae genomes.
Figure 2. A timeline of major events in synthetic genomics
4. Genome assembling technic
Chemically synthesized oligonucleotide chains are of limited length and can scarcely facilitate the needs of gene    and gene combination, thus needing to be further assembled into longer DNA fragments. Existing in vitro enzymatic    DNA splicing techniques include restriction endonuclease-dependent splicing techniques (BioBrickstTM, BglBricks,    Golden gate, etc.) and homologous sequence-dependent splicing techniques (In-FusionTM, SLIC, Gibson thermostat    assembly and so forth). The homologous recombination system in the host can also be used to assemble large fragments    of DNA. In particular, the homologous recombination system of S. cerevisiae is exceedingly efficient. Being    capable    of assembling multiple DNA fragments with overlapping sequences in the meanwhile, it is currently the best choice    for assembling the genome. Gibson et al. used this method to successfully assemble 25 DNA fragments of 24 kb into    the genomes of the entire Mycoplasma genitalium in yeast. Contemporarily, the maximum load reported of this    method    is the genome Haemophilus influenzaH of up to 1.8Mb, while the maximum limit still remains unclear. [4]
Reference
[1].Service, R. F. (2016). Synthetic biology. synthetic microbe has fewest genes, but many mysteries. Science,    351(6280), 1380.
[2].Ruan, J., Jian, C., Zhang, T., & Jiang, H. (2017). Mitochondrial genome evolution in the saccharomyces sensu    stricto complex. Plos One, 12(8), e0183035.
[3].König, H., Frank, D., Heil, R., & Coenen, C. (2013). Synthetic genomics and synthetic biology applications    between hopes and concerns. Current Genomics, 14(1), -.
[4].Luo, Z., & Dai, J. (2017). [synthetic genomics: the art of design and synthesis]. Chinese Journal of    Biotechnology, 33(3), 331.
