<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NPU-China_Synthesis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NPU-China/Synthesis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A href="#" data-activates="slide-out" class="button-collapse hide-on-large-only"><I class="medium material-icons">menu</I></A></DIV><UL id="slide-out" class="side-nav fixed" style="margin-left: 0px;"><LI class="zd"><A href="https://2018.igem.org/Team:NPU-China" class="zd collapsible-header waves-effect">HOME</A></LI><LI class="zd1"><A class="za1 collapsible-header waves-effect">Project</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Overview" id="Overview" class="waves-effect">Overview</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Background" id="Background" class="waves-effect">Background</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Mini_mtDNA_Design" id="Mini_mtDNA_Design" class="waves-effect">Mini
                                    mtDNA Design</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Synthesis" id="Synthesis" class="waves-effect">Synthesis</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Demonstrate" id="Demonstrate" class="waves-effect">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Model" id="Model" class="waves-effect">Model</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Future_work" id="Future_work" class="waves-effect">Future
                                    work</A></LI></UL></DIV><LI class="zd2"><A class="za2 collapsible-header waves-effect">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Human_Practices" id="Human_Practices" class="waves-effect">Intergrated</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Public_Engagement" id="Public_Engagement" class="waves-effect">Public
                                    Engagement</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Collaborations" id="Collaborations" class="waves-effect">Collaborations</A></LI></DIV><LI class="zd3"><A class="za3 collapsible-header waves-effect">Parts</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Parts" id="Parts" class="waves-effect">Parts
                                    Overview</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Basic_Part" id="Basic_Part" class="waves-effect">Basic
                                    Parts</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Composite_Part" id="Composite_Part" class="waves-effect">Composite
                                    Parts</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Part_Collection" id="Part_Collection" class="waves-effect">Parts
                                    Collection</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Improve" id="Improve" class="waves-effect">Improve</A></LI></DIV><LI class="zd4"><A class="za4 collapsible-header waves-effect">Lab</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Notebook" id="Notebook" class="waves-effect">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Protocols" id="Protocols" class="waves-effect">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Safety" id="Safety" class="waves-effect">Safety</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/InterLab" id="InterLab" class="waves-effect">InterLab</A></LI></DIV><LI class="zd5"><A class="za5 collapsible-header waves-effect">Team</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Team_Members" id="Team_Members" class="waves-effect">Team
                                    Members</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Attributions" id="Attributions" class="waves-effect">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Acknowledgements" id="Acknowledgements" class="waves-effect">Acknowledgements</A></LI></DIV><LI class="zd6"><A class="za6 collapsible-header waves-effect">For Judge</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Map" id="Map" class="waves-effect">Map</A></LI><LI><A href="https://2018.igem.org/Team:NPU-China/Medal" id="Medal" class="waves-effect">Medal</A></LI><MAIN><DIV class="container" style="padding-bottom: 65px;"><DIV class="center" style="padding-top:50px"><H4> Synthesis of MitoCRAFT Genome </H4></DIV><H5><B>Overview：</B></H5><H5>
			The sequence complexity of the MitoCRAFT genome has brought great challenges to our genome synthesis this year. The
			overall GC content of the sequence is extremely low (approximately 17%) and there are many consecutive AT and GC
			cluster repetitive sequences. In response to such contradictory sequence characteristics, we have tried many
			conditions optimization and possible methods. 
			With regard to the synthesis strategy, we first divided the MitoCRAFT genome into 22 primary fragments within 3kbp
			according to the sequence characteristics, the interface locations of which are specifically selected to avoid low
			GC and repetitive regions as much as possible. There is an overlap of about 100 bp between each two adjacent
			segments, facilitating subsequent in vitro splicing or in vivo recombination based on sequence homology.
			Although we have tried to avoid foreseeable difficulties in the design of the synthesis scheme, we have still
			encountered many obstacles in the synthesis of the primary fragments and the subsequent gene-splicing process. Low
			GC content and multiple repetitive sequences have specific requirements for the whole process of the synthesis of
			PCR-based primary fragments. Therefore, we have explored several aspects including primer design, enzyme preparation
			selection and optimization of PCR parameters. 
			After synthesizing all the primary fragments, we started the fragment splicing. First, several adjacent primary
			fragments were spliced into four secondary fragments of about 10k, then the four secondary fragments were spliced
			into a complete MitoCRAFT Genome. In view of the possible impact of sequence complexity on both in vivo and in vitro
			splicing, we tested a variety of splicing methods. As for in vitro splicing, we attempted In-fusion, Gibson, CPEC,
			Goldengate, DETAL and some other methods. Although some of them can aid part of the splicing, yet the generality
			could scarcely meet the requirements of the complete MitoCRAFT Genome splicing.
			We mainly adopted in vivo splicing based on S.cerevisiae homologous recombination (TAR) and E. coli in vivo
			expansion strategy for our in vivo splicing. We began by attempting to perform in vivo fragment recombination using
			the conventional YCplac33 E. coli-S. cerevisae shuttle vector. At the beginning, despite the connection of the
			initial PCR verification interface, there was still a problem in the transformation and amplification of E. coli
			that a positive clone could not be obtained, but sometimes random mutations or fragment deletions occurred even
			after the correct initial verification. The reason may be that the high copy of the YCplac vector in E. coli caused
			the cytotoxicity of common E. coli and could not guarantee the fidelity of amplification or even the survival of the
			cells themselves.
			In order to solve this problem, we introduced the Plasmid Tightly Regulated Copy-Control System (PTRCCS), which can
			strictly switch between single-copy and multi-copy modes and help E. coli strictly allocate amplified plasmids. With
			the stringent single-copy control module of the system, it is possible to reduce the cytotoxicity caused by high
			copying during cloning. The system's switching of tightly controllable multi-copy mode, the copy number can accrue
			shortly when plasmid extraction is required to obtain a higher concentration of plasmid. With the assistance of
			PTRCCS, we completed the transformation of E. coli. In the light of the preliminary colony PCR verification results,
			we have accomplished the synthesis of the complete 40K (plus vector 50Kbp) MitoCRAFT Genome. 【See more detailed
			information at 3. Complete MitoCRAFT splicing strategy based on S. cerevisiae homologous recombination】

		</H5><H5><B>The following are the specific processes and results of the synthesis</B></H5><H5 style="padding-top:20px"><B> 1. Sequence characteristics analysis of MitoCRAFT genome:</B>
			Analyze the GC content of each part of the MitoCRAFT genome, the results of which are as follows: </H5><DIV class="center"><H6>
				Figure 1. GC content distribution of each part of MitoCRAFT genome
			</H6></DIV><H5 style="padding-top:20px">
			As can be seen from the figure, the genetic characteristics of the MitoCRAFT genome are very complicated. In the
			case of overall low GC content, there are also multiple GC clusters, making it impossible to perform general
			synthesis and assembly.
		</H5><H5 style="padding-top:20px"><B> 2. Synthesis of 22 primary fragments:</B>
			The gene synthesis method based on PCR bridge amplification is a widely used Kbp-level gene fragment synthesis
			method, whose schematic diagram is as follows:
		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure 2. Schematic diagram of the synthesis of primary fragments
			</H6></DIV><H5 style="padding-top:20px">
			After a very tedious PCR-PCR-PCR, we obtained a plasmid library of the primary fragments with correct sequences
			constructed on the pUC57 cloning vector. The following are the results of the plasmid library digestion:
		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure 3-1. Gel electrophoresis of first-stage fragments.
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
				L1:2778bp;L2:2265bp;L3:2883bp;L4:1759;L5:2795bp.

			</H6></DIV><DIV class="center" style="padding-top:20px"><H6>
				Figure 3-2. Gel electrophoresis of first-stage fragments.M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY
				BIOTECH);
				L6:2288bp;L7:1142bp;L8:1368bp;L9:1453bp;L10:1376bp;L11:1904bp.

			</H6></DIV><DIV class="center" style="padding-top:20px"><H6>
				Figure 3-3. Gel electrophoresis of first-stage fragments.
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
				L12:2377bp;L13:1376bp;L14:1053bp;L15:1872bp;L16:1579bp;L17:1763bp.

			</H6></DIV><DIV class="center" style="padding-top:20px"><H6>
				Figure 3-4. Gel electrophoresis of first-stage fragments.
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
				L18:2370bp;L19:3306bp;L20:2007bp;L21:525bp;L22(GFP):870bp.

			</H6></DIV><H5><B>3.Complete MitoCRAFT splicing strategy based on S. cerevisiae homologous recombination</B>
			We transformed the fragment with Overlap (including part of the primary fragment and part of the secondary fragment)
			and the linearized vector containing the PTRCCS system into the S. cerevisiae BY4743 strain. The positive clone was
			picked to extract the S. cerevisiae genome into a specific E. coli EPI300 strains and the positive bacteria of E.
			coli were picked for colony verification. We designed interface validation primers for all 22 primary fragments and
			their 23 interfaces to the vector. The following are the results of EPI300-MitoCRAFT-A4 colony validation with a
			relatively satisfactory performance:
		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure4-1:Gel electrophoresis of A4 plasmid gap verification.
				M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY BIOTECH);
				L1: pGF-21-1(954bp); L2: 1-2(496bp); L3: 2-3(501bp);
				L4: 3-4(686bp);L6: 4-5(529bp);L8:6-7(537bp); L9:7-8(598bp).

			</H6></DIV><H5 style="padding-top:20px">
			L1 stands for the interface verification from the downstream fragment of the pGF Vector across the primary fragment
			of the 21st fragment to the 1st fragment of the primary fragment. L1 is the verification results of
			the two interfaces, proving the correctness of the bands, which is to say, both interfaces of pGF-21-1 were
			correctly connected.
		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure4-2:Gel electrophoresis of A4 plasmid gap verification..M is NormalRun™Prestained 250bp-II DNA ladder(GENERAY
				BIOTECH); L11:9-10(563bp); L13:11-12(521bp);L14:12-13(414bp);L15:13-14(866bp);
				L17:15-16(435bp);L18:16-17(873bp);L19:17-18(643bp);L20:18-19(406bp);
				L22:20-gfp(529bp).


			</H6></DIV><DIV class="center" style="padding-top:20px"><H6>
				Figure4-3:Re-verification of some gap in A4 or A17. M is NormalRun™Prestained 250bp-II DNA ladder;L1:A4
				5-6(683bp);L3:A4 gfp-pGF(472bp).
			</H6></DIV><DIV class="center" style="padding-top:20px"><H6>
				Figure4-4:Re-verification of some gap in A4.M is NormalRun™Prestained 250bp-II DNA ladder;
				L3:A4 8-9(737bp);L7:A4 19-20(600bp);L9:A4 14-15(875bp);

			</H6></DIV><H5 style="padding-top:20px">
			From the results of the above gel electrophoreses, we can derive that for this clone, only the interface of the
			10-11 primary fragment has not been effectively detected and that the PCR results of other clones at this position
			are also not obvious, rendering us unable to determin whether the problem was caused by PCR verification or improper
			connection. Considering that all ports upstream and downstream were detected to be correctly connected, we can
			conclude that we have basically completed the assembly of the MitoCRAFT genome. 
			We are currently expanding the strain incubation. We plan to use the switch of copy number mode function of the
			PTRCCS system to obtain a certain concentration of the plasmid, then verify the sequence accuracy of the MitoCRAFT
			genome carried by the strain through methods like sequencing.

		</H5><H5 style="padding-top:20px"><B> 4. Optimization of PCR conditions 
				4.1 Extension Temperature for high- A/T DNA fragments during PCR amplification. </B>
			Due to the high-A/T of the designed DNA fragments, we found that it’s inefficient to amplify the fragments in normal
			conditions. In the duration of exploration, it’s found that the Extension Temperature had a strong effect on the
			efficiency. Here is the Gradient Experiment of Extension Temperature we’ve explored: 
			Under the I-5(I-5™ 2X High Fidelity Mster Mix) PCR system (showed in right part of the figure bellow), only product
			under the extension temperature of 62℃ shows but low efficiency. 
			Under the PrimeSTAR(2X Takara PrimeSTAR MAX) PCR system (showed in left part of the figure bellow), only products
			under the extension temperature of 62℃ and 66℃ showed an ideal efficiency, while 70℃ has low efficiency and no
			efficiency in 72℃ under both PCR syst.

		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure 5:Gradient Experiment of Extension Temperature.
				M is NormalRun™Prestained 1kb-IV DNA ladder;
				L1-L4 are in PrimeStar PCR system(2778bp);
				L5-L8 are in I-5 PCR system(2778bp).

			</H6></DIV><H5 style="padding-top:20px"><B>
				4.2 Different DNA polymerase PCR system for same template </B>
			We have encountered a troublesome situation when amplifying first-stage fragment 2. No matter how we try to explore
			the condition, not even one time did we successfully amplify the fragment. 
			After seeking for help from Takara, we determine to use Gflex (Tks Gflex™ DNA polymerase, Tkara) PCR system claimed
			having a high efficiency for high-A/C fragments to try again. 
			In the figure showed below, only product under Gflex system amplified successfully.


		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure 6:Different polymerase system for PCR.
				M is NormalRun™Prestained 250bp-II DNA ladder;
				L1: Wrong product under primestar system;
				L2: Right product under Gflex system(2265bp);
				L3: Wrong product with primestar under Gflex system PCR conditions.
			</H6></DIV><H5><B> 4.3 Different Polymerase system for large fragment PCR amplification </B>
			We have triumphantly assembled large fragment 1-4(9.7kbp) in vitro by fusion PCR. However, I-5 PCR system is not be
			able to efficiently amplify the fragment. By using PrimeSTAR system, we successfully amplify the large and high-A/T
			fragment.

		</H5><DIV class="center" style="padding-top:20px"><H6>
				Figure 7. Different Polymerase system for large fragment PCR amplification.
				M is NormalRun™Prestained 1kb-IV DNA ladder;
				L1: Successful amplification of 1-4(9685bp) under primestar system;
				L2: Failed amplification of 1-4 under I-5 system.
			</H6></DIV><H5 style="padding-top:20px"><B>6.In vitro secondary fragment splicing</B>
			We specifically optimized some of the secondary fragment processes and parameters that are difficult to splice:
		</H5><DIV class="center" style="padding-top:20px"><H6>
				Fig. 8 Gel electrophoresis of second-stage fragments after fusion PCR.
				M is NormalRun™Prestained 1kb-IV DNA ladder;
				L1:9-10(2829bp);L2:12-13(3753bp);L3:20-gfp(2878bp);L4:16-17(3342bp)
				L5:1-4(9685bp);L6:5-6(5083bp).

			</H6></DIV><H5 style="padding-top:20px">
			In the figure, L1: 9-10 (2829 bp), L2: 12-13 (3753 bp), L3: 20-gfp (2878 bp), L5: 1-4 (9685 bp) and L6: 5-6 (5083
			bp) entailed two-step fusion PCR for splicing employing 2X Takara Prime STAR Mix. The fragment fusion annealing
			stage required a slow annealing from 75 °C to 40 °C, the annealing rate being 0.1 °C/s and the extension
			temperature
			being 62 °C. In the fragment amplification stage, the annealing temperature was 50 °C and the extension temperature
			was 62 °C. Specifically, the extension temperature of L3:20-gfp (2878 bp) was 66 °C and the annealing temperature
			remained constant.
		</H5><DIV class="center" style="padding-top:20px"><H6>
				Fig. 9 Gel electrophoresis of second-stage fragments after fusion PCR.
				M is NormalRun™Prestained 1kb-IV DNA ladder;
				L7:7-8(2510bp);L8:5-8(7593bp);L9:14-15(2925bp);L10:18-20(7683bp)
				L11:17-18(4133bp);L12:17-19(7439bp).

			</H6></DIV><H5 style="padding-top:20px">
			In the figure, L7: 7-8 (2510 bp), L9: 14-15 (2925 bp) and L11: 17-18 (4133 bp) entailed two-step fusion PCR for
			splicing employing 2X Takara Prime STAR Mix. The fragment fusion annealing stage required a slow annealing from 75
			°C to 40 °C, the annealing rate being 0.1 °C/s and the extension temperature being 62 °C. In the fragment
			amplification stage, the annealing temperature was 50 °C and the extension temperature was 62 °C. Specifically, the
			extension temperature of L3:20-gfp (2878 bp) was 66 °C and the annealing temperature remained constant.
		</H5></DIV></MAIN></BODY></HTML>