Team:NPU-China/Mini mtDNA Design
menu
HOME
Project
Overview
Background
Mini                                     mtDNA Design
Synthesis
Demonstrate
Model
Future                                     work
Human Practices
Intergrated
Public                                     Engagement
Collaborations
Parts
Parts                                     Overview
Basic                                     Parts
Composite                                     Parts
Parts                                     Collection
Improve
Lab
Notebook
Protocols
Safety
InterLab
Team
Team                                     Members
Attributions
Acknowledgements
For Judge
Map
Medal
MitoCRAFT is a sequence-simplified Saccharomyces cerevisiae mitochondrial chassis (approximately 39Kbp) consisting    of two parts. One is a pure chassis that provides a basic transcription and translation system and maintains aerobic    respiration, which we call MitoZero; the other is an extension that extends the functionality to create a    diversified toolbox, which we call CRAFT Plus.
1.Design of MitoZero
The gene-scale diversity of the mitochondrial genomes of S. cerevisiae (from a minimum of about 20 kbp to a maximum    of around 130 kbp) and its significant linear positive correlation with genomic scale and intergenic regions have    drawn our attention. We wondered whether they could be “restored” by simplifying redundant sequences of genes and    complicated gene regulatory sequences, creating a relatively primitive S. cerevisiae mitochondrial chassis,    MitoZero, which provides the basis for a further extended and more complex manual mitochondrial design.
The goal of the design of MitoZero: On the basis of guaranteeing the basic mitochondrial function, the sequence    ought to be simplest and has a relatively simple genetic structure, which facilitates the subsequent complex design.
The scheme of the design of MitoZer: Based on the bioinformatics analysis model, the functionality and relative    conservation degree of each part of the genome will be determined (by reference to generalized and narrowly defined    Saccharomyces cerevisiae) and the non-conserved part of the intergenic region which accounts for a large proportion    of the original mitochondrial genome and the intron regulatory system that is not conserved in the genome will be    simplified.
1.1 Simplification of non-conserved intergenic regions and reduction of genome size
In the course of the evolution of species, the intergenic regions also altered with the change in the relative    position order of the gene modules. Generally speaking, intergenic sequences are difficult to survive under such    changes if they need to perform important biological functions. Therefore, in contrast, the intergenic regions    within the syntenic orthologous blocks, the gene regions in line with the relative position order, are more crucial,    while the intergenic regions between the blocks can be simplified to some extent.
Utilizing the Gen Order Conservation (GOC) calculation model and the Branch-specific Gene Order Loss (bsGOL)    calculation model (see Model), we obtained the specific positions of the 7 blocks of the genome. We retained the    sequence of 100 bp upstream and downstream of each block as sequence buffers and deleted the redundant sequences of    the remaining intergenic regions, totaling 29,301 bp.
1.2 Simplification of non-conserved introns as well as construction of a more primitive transcription and    translation regulation system with the reduction of genome size
In the process of conservation analysis of wild-type mitochondrial genome introns, we found that intron sequences    are much less conserved than exons (only one is conserved), thus being able to be simplified to create a relatively    simple functional gene transcription and translational regulation system. We first removed all non-conserved    introns. Considering that only one intron had no corresponding shear control system, we also removed it. The    simplified introns have a total of 17,512 bp.
Through the simplification of these two aspects, we have created the theoretically primary minimal Saccharomyces    cerevisiae mitochondrial chassis - MitoZero, the sequence of which has been reduced from the original 85,779 bp to    38,966 bp, less than half of the original length.
Figure 1.MitoCRTFT gene map without GPF
2. Design of CRAFT Plus
Saccharomyces cerevisiae mitochondria have correspondingly independent functional expansion capability due to their    relatively independent transcription and translation system. Compared with the original genome of up to 86kbp, the    simplified genome scale has been remarkably reduced. In theory, it is easier to perform molecular cloning    operations, elevating the degree of easy “insertion and removal” of its function expansion.
We designed a GFP-based reporter gene module to verify the function expansion capability of MitoZero.
The GFP module is a simple promoter-CDS-terminator structure whose promoter sequence is a strong one from the    original mitochondria, 5'-ATATAAGTA-3', and similarly, the terminator sequence is 5'-AATAATATTCTT- 3'. Since the    mitochondrial codon table differs from the nuclear genome, we have conducted targeted codon optimization.
The codon optimization table:
（Note: This codon is MitoCRAFT Coden Usage Table based on the Saccharomyces cerevisiae mitochondrial codon table.    Copyright belongs to the original author.）
We have provided the simplest example for followers through our design.
Figure 2.Complete MitoCRTFT gene map containing GPF
