Team:NPU-China/Improve
menu
HOME
Project
Overview
Background
Mini                                     mtDNA Design
Synthesis
Demonstrate
Model
Future                                     work
Human Practices
Intergrated
Public                                     Engagement
Collaborations
Parts
Parts                                     Overview
Basic                                     Parts
Composite                                     Parts
Parts                                     Collection
Improve
Lab
Notebook
Protocols
Safety
InterLab
Team
Team                                     Members
Attributions
Acknowledgements
For Judge
Map
Medal
Improve part
BFD (benzoylformate decarboxylase)
Improvement
The part we improved is BFD enzyme, BBa_K2155001 , used and submitted by Team NWPU in 2016, who at that time    excavated    new catalytic function of this enzyme.
The element was enhanced from the following two aspects:
1. For the core parameters Km and Kcat of the enzyme, Team NWPU did not make specific measurements. We measured the    Km and Kcat values of the enzyme for the first time this year, and further improved the key information of the    component in the iGEM KITS component library.
BFD Km[M]=0.092 Kcat=0.01273429 Kcat/Km[M-1/S-1]=0.138416196
2. We modified the gene sequence of the enzyme, obtaining a mutant BFD-F464W with better catalytic performance,    which the ability to perform new catalytic function has been elevated.
BFD Km[M]=0.059 Kcat=0.4331122 Kcat/Km[M-1/S-1]=7.340884746
Note: The core parameters of wild-type BFD enzyme and BFD-F464W were simultaneously performed.
Usage
Team NWPU has explored the new catalytic function of this enzyme in 2016, catalyzing the production of DHA and    hydroxy acetaldehyde from formaldehyde, which could be used to further convert one-carbon compound into DHA and    hydroxy acetaldehyde that could be utilized by cells for energy metabolism consuming NADH. Through electroporation    method, CO2 can be converted into formaldehyde, which is catalyzed by the enzyme to form DHA and hydroxy    acetaldehyde, actualizing a complete artificial carbon dioxide fixation pathway.
Other teams can also avail themselves of this part to construct metabolic pathways related to formaldehyde, DHA and    Glycolaldehyde.
Figure 1. BFD catalyzes the formation of DHA and glycaldehyde from formaldehyde
Biology
BFD（EC 4.1.1.7）is an enzyme in Pseudomonas putida (Arthrobacter siderocapsulatus), which is expressed by the    gene mdlC. Reaction of the catalysis in nature:
benzoylformate + H+ ⇌ benzaldehyde + CO2
The metabolic pathways involved is (R)-mandelate degradation
This protein is involved in step 3 of the sub-pathway to synthesize benzoate from (R)-mandelate.
Proteins known to be involved in the 4 steps of the subpathway in this organism are:
1. Mandelate racemase (mdlA);
2.(S)-mandelate dehydrogenase (mdlB);
3.Benzoylformate decarboxylase (mdlC);
4.NAD(P)-dependent benzaldehyde dehydrogenase (mdlD).
This sub-pathway is part of the pathway (R)-mandelate degradation, which is the part itself of Aromatic compound    metabolism.
This Protein has several cofactor binding sites:
1. Ca2+
Note: Binds 1 Ca2+ ion per subunit.
2. thiamine diphosphate
Note: Binds 1 thiamine pyrophosphate per subunit.
3. Mg2+
Note: Binds 1 Mg2+ ion per dimer.
Figure 3. 3D structure of BFD
Improvement characterization
1. Activity assay and kinetic properties of BFD and mutants    An initial continuous assay included 50 mM potassium phosphate buffer (pH 7.4), 5 mM MgSO4, 0.5 mM thiamine    diphosphate, 50 μg/mL glycerol dehydrogenase, 0.8 mM NADH, and 67 mM formaldehyde. The reaction was initiated by the    addition of purified BFD or mutants (0.05 mg/mL) at 37℃, and then an initial linear decrease in absorbance at 340 nm    was observed. One unit of enzyme activity was deﬁned as the amount of enzyme catalyzing the conversion of 1 μmol    NADH    per minute. Enzyme kinetics with formaldehyde as substrate were determined in assays with formaldehyde    concentrations    of 0.1-1000 mM. Kinetic parameters kcat and Km were estimated by measuring the initial velocities of enzymic    reaction    and curve-ﬁtting according to the Michaelis-Menten equation, using GraphPad Prism 5 software. All experiments were    conducted in triplicate.
Figure 4. NADH concentration standard curve
2. Part modification: obtaining mutant BFD-F464W with better catalytic performance
2.1 Design of the modification
The relationship between amino acids and substrates within the range of 5 angstroms of BFD active center was    analyzed    to infer the possible modification scheme of point mutations. By evaluating the relationship between amino acid No.    464    (phenylalanine, F) and the substrates, we speculate that phenylalanine can be replaced by tryptophan (W) to optimize    the adaptability to the substrates of the enzyme active center, thereby increasing the enzyme performance.
Figure 5. Selection of single-site saturation mutation sites. The residues locating within 8Å distance from     benzene ring of intermediate analogue were colored brown, thiamine diphosphate by green respectively
2.2 Enzyme-modified molecular cloning operation:
PCR-based enzyme gene sequence mutation
a) Design the primers located at the mutation sites according to the mutated DNA sequence.
Forward primer sequence:
GTACCTACGGTGCTCTGCGTTGGTGGGCTGGTGTTCTGGAA
Reverse primer sequence:
CCACCAACGCAGAGCACCGTAGGTACCGTTGTTCATGATAA
Figure 6.Primer design of F464W
b) Conduct PCR reaction
Figure 7.Point mutant method and PCR program
c) Purify the PCR product with a DNA purification kit.
d) Add the appropriate amount of DMT enzyme, hold for one hour at 37 ° C.
e) Transform 5μl digested DNA into competent cells DH5α, incubate on ice for 30min.      42° C heat shock, 45s. Incubate on ice for 2min. add 200μl of LB. incubate at 37 °C for 1 h, 220rpm/min.
f) Pipet 200μl from each tube onto the plate with appropriate resistance, and spread the mixture evenly across the     plate. Incubate at 37℃ overnight. Position the plates with the agar side at the top, and the lid at the bottom.
g) Select single colonies for sequencing.
2.3 Obtaining F464W enzyme
The coding genes of mutant F464W were ligated into the expression vector pET-28a via NdeІ and XhoI restriction     sites.     E. coli BL21(DE3) cells carrying different recombinant plasmids were inoculated into 5 mL LB (Luria Broth) medium     with     Kanamycine (100 μg/mL) and cultured overnight at 37°C, and then scaled up to 800 mL 2YT medium (16 g/L Tryptone, 10     g/L     yeast extract, 5 g/L NaCl) containing Kanamycine (100 μg/mL). Gene expression was induced by adding IPTG     (isopropyl-β-D-thiogalactopyranoside) to a final concentration of 0.5 mM when OD600 reached 0.6. The cell cultures     continued to grow overnight at 16°C before being harvested by centrifugation at 6,000 g and then was resuspended in     50     mL lysis buffer (50 mM potassium phosphate buffer, pH 7.4, 5 mM MgSO4, 0.5 mM thiamine diphosphate ). The bacterial     pellet was lysed by using a high-pressure homogenizer (JNBIO, China), and the cell debris was removed by     centrifugation     at 10,000 g for 60 min at 4°C. The soluble protein sample was loaded onto a nickel affinity column (GE Healthcare),     rinsing with 50 mL wash buffer (50 mM potassium phosphate buffer, pH 7.4, 5 mM MgSO4, 0.5 mM thiamine diphosphate     and     50 mM imidazole) and then eluting with 20 mL elution buffer (50 mM potassium phosphate buffer, pH 7.4, 5 mM MgSO4,     0.5     mM thiamine diphosphate and 200 mM imidazole). The eluted protein was concentrated and dialyzed against lysis     buffer     (50 mM potassium phosphate buffer, pH 7.4, 5 mM MgSO4, 0.5 mM thiamine diphosphate ) by ultrafiltration with an     Amicon     Ultra centrifugal filter device (Millipore, USA) with a 30 kDa molecular-weight cutoff. The protein concentration     was     determined using a BCA Protein Assay Reagent Kit (Pierce, USA) with BSA as the standard.
Figure 8.Expression of 3FZN（BFD F464W）. M, protein marker; 1, precipitation samples in the cell lysates; 2,     supernatant samples in the cell lysates; 3, 50 mM imidazole eluent; 4, 100 mM imidazole eluent; 5, 200 mM     imidazole eluent; 6, 300 mM imidazole eluent.
2.4 Simultaneous measurement and comparison of the Km and Kcat values of both BFD-F464W and wild-type BFD.
Refer to the above part characterization method for measurement.
Figure 9.BFD reaction rate
Figure 10.BFD-F464W reaction rate
BFD(Wild type)
BFD-F464W
protein add amount[umol]
0.003690037
0.000184502
Vmax [umol/s]
0.00004699
7.99E-05
Km[M]
0.092
0.059
Kcat=Vmax/E(S-1)
0.01273429
0.4331122
Kcat/Km [M-1/S-1]
0.138416196
7.340884746
Table 1.Comparison of Enzyme Kinetic Parameters between BFD and BFD-F464W
Reference
[1] http://www.brenda-enzymes.org/
[2] Purification and crystallization of benzoylformate decarboxylase. Hasson, M.S.; Muscate, A.; Henehan, G.T.M.; Guidinger, P.F.; Petsko, G.A.; Ringe, D.; Kenyon, G.L.; Protein Sci. 4, 955-959 (1995)
