"
Page
Discussion
View source
History
teams
Log in
Â 
Team:LMU-Munich/Primer Design
From 2011.igem.org
(Redirected from Team:LMU-Munich/Primer Designer )
Home
Project
Detailed Information
Sequences
Human Practice
Workshops
School Project
Primer Design
Adjacent Regions
Design Primers
Team
Our Team
Cooperation
Official Team Profile
Gallery
Parts
Lab Notebook
Safety
Sponsoring
Introduction
Our aim with these tools is to assist the user with primer design for coding and non coding Biobricks. In order to achieve this we chose two slightly different approaches for designing coding and noncoding Biobricks. While only one input sequence is needed in order to design a coding biobrick we chose to use flanking regions of the sequence of a non doding biobrick. The reason herefor is that using flanking regions of a non coding sequence which is supposed to be amplified enables the possibility to create a broad range of primers which meet the necessary criteria for a working primer.
Hence we created Adjacent Regions and Design Primers.
Adjacent Regions finds upstream and downstream sequences of the desired target sequence which can be used for the design of non coding Biobricks. Besides the source genome of the to be amplified biobrick the user has the option to choose cutoff rates for the downstream region as well as the upstream region.
Click here to use Adjacent Regions.
Using these upstream and downstream sequences the user can design primers for non coding Biobricks with Design Primers.
Click here to use Design Primers.
Design Primers is also able to design primers for coding sequences. Here the only needed input is the coding sequence of the Biobrick. Design Primers also has a number of options:
The length of the annealing region of the primers.
The Biobrick standard.
You can choose between the "Classic" Biobrick standard and the "Freiburg" standard.
Whether to check for enzyme restriction sites in the input sequence or not.
You can choose between all known restriction sites, only those used in the two Biobrick standards or none at all.
For coding sequences Design Primers varies the primer length between 15 und 35 bp and names only those potential primers which and with a G/C on the 3' end.
For non coding sequences Design Primers uses the by the user set primer length and finds all possible primers within the upstream and downstream regions. It checks every primer for selfannealing and uses only those which are predicted to be not selfannealing.
Additionally for all input sequences the Tm (based on the Wallace rule) and GC content of the annealing region are being determined and named. Further on the 3' ends of the primers are being checked for having a G/C at the end and for potential terminal Hairpins. In case a potential Hairpin has been found the script adds HAIRPIN!?-X-Y to the output. Here X is the number of the nucleotides which compose the loop while Y is the number of the nucleotides which are adjacent to the loop forming nucleotides and potentially annealing.
An example Output:
FORWARD
GAATTCGCGGCCGCTTCTAGATGGCCGGCACGTCAGATCGACTACG
52
0.52
HAIRPIN!?-4-2
1
2
3
4
5
Kind of primer.
Primer; in this case the Freiburg standard.
Tm.
GC Content.
Hairpin warning; in this case the very last CG nucleotides might anneal with GC and form a loop of ACTA.
The input format for both tools can be a fasta sequence with or without a header. Sequences including nucleotide counters are also supported.
In case you find any bugs or have any comments, suggestions or questions please feel free to email us at lmuprimer(at)googlemail.com
Retrieved from " http://2011.igem.org/Team:LMU-Munich/Primer_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
