<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Project_Description"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Project/Description">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Project/Description&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Description&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Description&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Project/Description" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Project/Description</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="nav"><UL><LI><A href="/Team:LMU-Munich" title="Team:LMU-Munich">Home</A></LI><LI><A href="/Team:LMU-Munich/Project" title="Team:LMU-Munich/Project">Project</A></LI><LI><STRONG class="selflink">Detailed Information</STRONG></LI><LI><A href="/Team:LMU-Munich/Sequences" title="Team:LMU-Munich/Sequences">Sequences</A></LI></UL><LI><A href="/Team:LMU-Munich/Human_Practice" title="Team:LMU-Munich/Human Practice">Human Practice</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Workshops" title="Team:LMU-Munich/Human Practice/Workshops">Workshops</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/School_Project" title="Team:LMU-Munich/Human Practice/School Project">School Project</A></LI><LI><A href="/Team:LMU-Munich/Primer_Design" title="Team:LMU-Munich/Primer Design">Primer Design</A></LI><LI><A href="/Team:LMU-Munich/Primer_Design/Adjacent" title="Team:LMU-Munich/Primer Design/Adjacent">Adjacent Regions</A></LI><LI><A href="/Team:LMU-Munich/Primer_Design/Design_Primers" title="Team:LMU-Munich/Primer Design/Design Primers">Design Primers</A></LI><LI><A href="/Team:LMU-Munich/Team" title="Team:LMU-Munich/Team">Team</A></LI><LI><A href="/Team:LMU-Munich/Team" title="Team:LMU-Munich/Team">Our Team</A></LI><LI><A href="/Team:LMU-Munich/Porject/Cooperation" title="Team:LMU-Munich/Porject/Cooperation">Cooperation</A></LI><LI><A href="https://igem.org/Team.cgi?id=549" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:LMU-Munich/Gallery" title="Team:LMU-Munich/Gallery">Gallery</A></LI><LI><A href="/Team:LMU-Munich/Parts" title="Team:LMU-Munich/Parts">Parts</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook" title="Team:LMU-Munich/Lab Notebook">Lab Notebook</A></LI><LI><A href="/Team:LMU-Munich/Safety" title="Team:LMU-Munich/Safety">Safety</A></LI><LI><A href="/Team:LMU-Munich/Sponsoring" title="Team:LMU-Munich/Sponsoring">Sponsoring</A></LI></DIV><DIV class="fullbox"><H2><SPAN class="mw-headline" id="Project_Details"> Project Details</SPAN></H2><P>This year’s project by the iGEM-team from Ludwig-Maximilians-University in Munich uses natural biosensors to detect the concentration of different metals. We have the vision to develop a set of bacterial metal sensors for easy qualitative and quantitative measurement of toxic metals just by reading the output after adding the water test sample.
</P><P>We use two different kinds of metal sensors. The ones in the first category work with reporter genes that lay downstream of an inducible promoter. The respective promoter is activated or deactivated by a specific metal-sensitive protein which binds to DNA dependent on the presence of that metal. As a consequence of this statistical event, there is a concentration-dependent transcription of the reporter gene, which is either GFP, luxAB or lacZ´.
</P><P>
The second kind of metal sensors directly uses the characteristics of special proteins to obtain a measurement of the metal, e.g. by analyzing the activity of an enzyme that needs a special metal ion as a cofactor. 
For easy handling and compatibility we use the E. coli strain DH5α and B. subtilis for our two promoters taken from this organsim.
</P><P><B>Reporter genes</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">GFP</DIV></DIV></DIV><B>GFP</B>: The well-known green fluorescent protein is detectable after excitation with UV-light without addition of further molecules. The output is not very sensitive because GFP is directly measured in contrast to luxAB and lacZ’, which use an enzymatic multiplying response.
<DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">luxAB</DIV></DIV></DIV><B>luxAB</B>: These two genes from Vibrio lux-Operon catalyse the light-emitting oxidation of luciferine. Therefore, very expensive luciferin has to be added to the bacteria, or they have to be cotransformed with a plasmid containing luxCDE, an enzyme cascade producing luciferin out of fatty acids.
<DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">lacZ'</DIV></DIV></DIV><B>lacZ´</B>: lacZ‘ codes the β-subunit of β-galactosidase, an enzyme that catalyses the reaction from X-Gal (5-bromo-4-chloro-3-indolyl- beta-D-galactopyranoside) to a blue insoluble indigo-dye (5,5'-dibromo-4,4'-dichloro-indigo). As only the β-subunit is used, the E.coli strain has to contain the α-subunit and must not contain the whole enzyme. These requirements are fulfilled e.g. in DH5α.
<P><B>Promoter-based Sensors</B></P><P>All genes are transcriptionally fused to the reporter gene, which means that approximately 50 nucleotides from the originally transcribed gene still remain with the promoter to ensure correct read-off. Downstream of this short open reading frame, the reporter gene with its own ribosome binding site is added.
</P><B>pnikA</B>: The nik-operon from E. coli codes for a nickel-influx system, which is constantly active, unless the concentration of nickel inside of the bacteria is too high. In this case, the regulator NikR binds Ni(II)-Ions and attaches to the NikR-operator site, which is located in the promoter sequence of nikA, called pnikA. So with rising nickel concentration, the activity of pnikA is reduced. Since nikR originates from E.coli, it’s sufficient to clone the promoter pNikA from E.coli K12 MG1655 gDNA. The linear response is fulfilled for a concentration range from 1-400 nM. This Regulator works under anaerobic conditions.<P>source(s):
- <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=coordinating%20intracellular%20chivers" class="external text" rel="nofollow">Coordinating intracellular nickel-metal-site structure-function relationships and the NikR and RcnR repressors</A> Iwig JS, Chivers PT
- <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1236639/?tool=pubmed" class="external text" rel="nofollow">Complex transcriptional control links NikABCDE-dependent nickel transport with hydrogenase expression in Escherichia coli</A> RoweJS, Starnes GL, Chivers PT
- <A href="http://www.ncbi.nlm.nih.gov/pubmed/10787413" class="external text" rel="nofollow">Regulation of high affinity nickel uptake in bacteria</A> Ni2+-Dependent interaction of NikR with wild-type and mutant operator sites. Chivers PT, Sauer RT 
- <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC93426/?tool=pubmed" class="external text" rel="nofollow">Isolation and characterization of the nikR gene encoding a nickel-responsive regulator in Escherichia coli</A> de Pina K, Desjardin V, Mandrand-Berthelot MA, Giordano G, Wu LF</P><B>prcnA</B>: The rcn-operon from E. coli codes for a nickel- and cobalt-efflux system. If the repressor RcnR has Ni(II)-ions bound, it cannot attach to DNA and the prcnA-promoter is active. In the absence of nickel or cobalt, the rcnR binds to the rcnR operator and blocks the rcnA-promoter prcnA. Since the regulator originates from E. coli and is not coded on the plasmids, E. coli must be used as reporter organism. The promoter was cloned using Phusion polymerase, primer prcnA-E,N,X-for and prcnA-S-rev and 50°C annealing temperature. The linear response is fulfilled for a concentration range from 0.5-60 µM.<P>source(s):
- <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2006.05369.x/pdf" class="external text" rel="nofollow">Nickel homeostasis in Escherichia coli – the rcnR-rcnA efflux pathway and its linkage to NikR function</A> Iwig JS, Rowe JL, Chivers PT 
- <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=coordinating%20intracellular%20chivers" class="external text" rel="nofollow">Coordinating intracellular nickel-metal-site structure-function relationships and the NikR and RcnR repressors</A> Iwig JS, Chivers PT</P><B>pars</B>: The pars-promoter from Escherichia coli is regulated by the repressor arsR, which can bind As(III)- and Sb(III)-ions. Without these ions, the arsR binds to the pars-promoter, deactivating it. In the presence of As(III) or Sb(III), the repressor leaves the DNA, by this means enabling the promoter to work. For less leakiness of the repressed promoter, a second arsR-binding site was used. 
<P>Since the regulator arsR does not exist in E. coli, it is coded on our plasmids. arsR with a constantly active lacI promoter was fused using 3A-assembly.</P><P><B>pnorB</B>: The fur-norB-System from Neisseria meningitidis is able to detect iron. Here fur is aktivating the norB-promoter if it has Fe-ions bound. Therefore the fur gene is used with a constitutive promoter and the inducible norB-promoter is fused with the reporter genes. The cloning details can be found in our <A href="https://2011.igem.org/Team:LMU-Munich/Lab_Notebook" class="external text" rel="nofollow">lab notebook</A>.
source(s):
-<A href="http://www.ncbi.nlm.nih.gov/pubmed/15130126" class="external text" rel="nofollow">Fur functions as an activator and as a repressor of putative virulence genes in Neisseria meningitidis</A> Delany I, Rappuoli R, Scarlato V.
</P><P><B>pAseR</B>:
The paseR-promoter from Bacillus subtilis is regulated by the repressor AseR, which can bind As(II)- and As(III)-ions. In presence of these ions the repressor is releaved from the aseR binding site next to the promoter PaseR.
The promoter region was amplified from genomic DNA of Bacillus subtilis subtilis W168, which is used as a laboratory wildtype of the gram-positive model organism. In our project we fused the promoter regions upstream of a set of reporter genes (lacZ', GFP and LuxAB) in order to measure its activity in the presence of As-ions.
</P><P><B>pCueR</B>:
The promoter pcueR from Bacillus subtilis controlled by the negative regulator CueR. CueR is capable of binding Cu(II)- and Cu(III)-ions by changing its conformation. In the absence of Cu (II/III)- ions cueR blocks the promoter region of pcueR by binding to the cueR binding site. If Cu - ions are available in the media, CueR releases the promoter region and transcription can be activated. For our project we fused the promotor region to a set of reporter genes in order to quantify the response of B. subtilis to high concentrations of copper ions.
</P><P><B>Protein-based Sensors</B></P><P>These sensors are fused to 10xHis-tag or STREP-tag domains to not only enable a detection via antibodies, but also - and more important - to enable an easy purification of the fusion proteins.
</P><P><B>pabA</B>: pabA is coding for the glutamine aminotransferase that is an subunit of the PABA-synthase (4-Aminobenzoic acid). The synthase catalizes the reaction from chorismate and glutamate to PABA. PABA can be easily oxidized with Fe(III) and is so turning to an yellow-reddish color when adding oxygen.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Plastocyanin</DIV></DIV></DIV><B>petE2</B>: petE2 is also known as plastocyanin from Arabidopsis thaliana. Plastocyanin is a small blue copper protein that shuttles electrons as part of the photosynthetic redox chain. Its redox behavior is changed at low pH as a result of protonation of the solvent-exposed copper-coordinating histidine. Protonation and subsequent redox inactivation could have a role in the down regulation of photosynthesis. It only appears blue, if Cu(I)- oder Cu(II)- ions are bound.
</DIV><DIV class="fullbox"><H2><SPAN class="mw-headline" id="The_Experiments"> The Experiments </SPAN></H2><P>The BioBricks containing the nickel-sensitive promoter pRcnA (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K549001" class="external text" rel="nofollow">[http://partsregistry.org/wiki/index.php?title=Part:BBa_K549002 BBa_K549001</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K549003" class="external text" rel="nofollow">BBa_K549002</A> and BBa_K549003])were tested under aerobic  conditions. The overnight culture was diluted 1:1000 into LB-medium and grown until the mid-log phase. Following the culture was split into several samples. The system was induced using various concentrations of NiCl2*6H2O: 0.1, 0.5, 1, 3, 5 and 10 µM. The BioBricks were then tested according to their reporter gene after 30, 60, 90 and 120 minutes.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:LMU-Munich/Project/Description">http://2011.igem.org/Team:LMU-Munich/Project/Description</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Project/Description" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Project/Description" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Description&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Description&amp;oldid=163035" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>