<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_CU-Boulder_Notebook_CC9Phage_Team"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:CU-Boulder/Notebook/CC9Phage_Team">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:CU-Boulder/Notebook/CC9Phage_Team&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9Phage_Team&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9Phage_Team&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:CU-Boulder/Notebook/CC9Phage_Team" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:CU-Boulder/Notebook/CC9Phage Team</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><HEADER id="topNav"><NAV id="navContainer"><UL class="navLinks"><LI class="navSingleLink"><A href="https://2014.igem.org/Team:CU-Boulder">HOME</A></LI><LI><A>PROJECT</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Getting Started</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Motivation">Motivation</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Modeling">Modeling</A></LI></UL></NAV></HEADER></DIV></DIV></DIV><LI><A>ACHIEVEMENTS</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Our Team Achievements</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Biobricks">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Medal">Medal Requirements</A></LI><LI><A>NOTEBOOK</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">At the Bench</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Phage_Team">Phage Delivery</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Induced">Induced CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Constitutive">Constitutive CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9Phage_Team">Phage CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Measurement">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Protocols">Protocols</A></LI><LI><A>HUMAN PERSPECTIVE</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Reaching Out</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Safety">Ethics &amp; Safety</A></LI><LI><A>TEAM</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">MI6</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Acknowledgments">Acknowledgments</A></LI><P><B>Note:</B><B>Litmus28ibb is a biobricked version of the Litmus28i (from NEB) vector which contains the M13ori (packaging signal) which allows for uptake into M13 or Fd phage. While making this part biobrick compatible, we added J04450 as the insert.</B><B>pCas9 (BBa_K1218011) contains and tracrRNA and Cas9 gene under native promoters, and a minimal CRISPR arrays. The guide RNA (referred here as gRNA) is inserted into the CRISPR array.</B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_1"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_3"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week_4"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week_5"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 5</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Week_6"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 6</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week_7"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week 7</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Week_8"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Week 8</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Week_9"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Week 9</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Week_10"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Week 10</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Week_11"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Week 11</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Week_12"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Week 12</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Week_13"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Week 13</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H2><P><B>7/21</B></P><UL><LI>To ligate pCas9 to Litmus28ibb
</LI></UL><DL><DD>-Digestion
</DD><DD>-Litmus28ibb-J04450 with E+P
</DD><DD>-pCas9 with E+P
</DD></DL><DL><DD>1. pCas9
</DD><DD>2. Litmus28ibb
</DD><DD>-Overnight ligations (10hrs at 37C, 10min at 80C)
</DD></DL><P><B>7/22</B></P><UL><LI>Transformed ligation from 7/21 (Litmus28ibb + pCas9) into ER2738 cells
</LI></UL><DL><DD>-Plated on Amp + Tet
</DD><DD>-Later in day, pricked colonies from morning transformation (Litmus28ibb and pCas9 in ER)
</DD></DL><P><B>7/23</B></P><UL><LI>Miniprepped O/N from 7/22 of Litmus28ibb-pCas9
</LI></UL><DL><DD>-Digested
</DD><DD><UL><LI>None were the correct size. Smaller bands looked like pSB1C3. Plated cells onto Chlor and there was significant growth. pSB1C3 from the pCas9?
</LI></UL></DD></DL><P><B>7/24</B></P><UL><LI>To put pCas9 onto Litmus28i
</LI></UL><DL><DD>-Digested
</DD><DD>-Litmus28ibb with EcoRI+PstI
</DD><DD>-pSB1C3-pCas9 with EcoRI+SpeI
</DD><DD>-PSB3C5-J04450 with XbaI + PstI
</DD></DL><DL><DD>1. Litmus28ibb
</DD><DD>2. pCas9
</DD><DD>3. J04450
</DD><DD>-Overnight ligation (10hr @ 16C)
</DD></DL><P><B>7/25</B></P><UL><LI>Transformed ligation from 7/24 (Litmus28ibb+pCas9) into 5alpha cells
</LI></UL><P><B>7/26</B></P><UL><LI>No growth from 7/25 transformation
</LI><LI>Tranform
</LI></UL><DL><DD>-Litmus28ibb + pCas9 + J04450
</DD></DL><UL><LI>Retry making Litmus28i with pCas9
</LI></UL><DL><DD>-Ligation (10hrs at 16C, 10min at 80C)
</DD><DD><UL><LI>Litmus28ibb + pCas9 + J04450
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H2><P><B>7/27</B></P><UL><LI>Transformation results
</LI></UL><DL><DD>-No growth for no DNA controls (on Chlor)
</DD><DD>-4 colonies for Litmus28bb-JO4450-Cas9 (on Amp-&gt; none are red)
</DD></DL><UL><LI>Picked all 4 colonies for O/Ns
</LI></UL><P><B>7/28</B></P><UL><LI>3 new red colonies on Litmus28bb-Cas9-J04450 plate
</LI></UL><DL><DD>-Set up cultures
</DD></DL><UL><LI>O/Ns grew but did not bother
</LI><LI>Transformations
</LI></UL><DL><DD>-Litmus28bb-J04450-Cas9
</DD><DD>-No DNA control
</DD></DL><P><B>7/29</B></P><UL><LI>Litmus28bb-Cas9-J04450 O/Ns
</LI></UL><DL><DD>-Incubator shut off during night. Samplers were at RT by morning
</DD><DD>-Add 1mL to fresh 4mL LB and Amp
</DD><DD>-Mini-prep DNA
</DD><DD>-Digest with EcoRI and PstI to check insert for correct size
</DD></DL><DL><DD>-Lane 2 contains the expected 3000bp band for the Litmus28ibb backbone and the ~7000bp band for the pCas9-J04450 insert. Unfortunately, this sample also contains a 4000bp band.
</DD></DL><UL><LI>Plated the above sample
</LI></UL><P><B>7/30</B></P><UL><LI>Digested Litmus28ibb-Cas9-RFP with EcoRI-HF
</LI></UL><DL><DD>-Extracted large band ~9000
</DD><DD>-Re-ligated piece
</DD></DL><UL><LI>Sent samples for sequencing
</LI></UL><DL><DD>-Litmus28bb with Cas9 and RFP (LCR) gRNA sequence
</DD></DL><P><B>8/1</B></P><UL><LI>Mini-prepped O/Ns of
</LI></UL><DL><DD>-Litmus28ibb-Cas9-J04450 (LCR)
</DD><DD><UL><LI>Digested with EcoRI-HF and PstI-HF =&gt; all samples had band ~3700
</LI></UL></DD></DL><UL><LI>Digestions of Litmus and Cas9 to retry ligation
</LI></UL><DL><DD>1. Litmus28ibb (E+P)
</DD><DD>2. pCas9(E+P)
</DD><DD>3. Litmus28ibb (E+P)
</DD><DD>4. pCas9(E+S)
</DD><DD>5. pSB3C5 (X+P)
</DD><DD>-Gel extractions
</DD><DD>-Ligations
</DD><DD><UL><LI>Litmus28ibb (E+P) to pCas9 (E+P)
</LI><LI>Litmus28ibb(E+P) to pCas9(E+S) and J04450(E+P)
</LI></UL></DD></DL><P><B>8/2</B></P><UL><LI>Transformation into BW cells (5/30)
</LI></UL><DL><DD>-Hundreds of colonies for LCR Miniprep
</DD></DL><UL><LI>Digestion to get a better idea of what this DNA is
</LI></UL><P><B>8/3</B></P><UL><LI>Isolate the LCR from the extra band in the gel
</LI></UL><DL><DD>-Run 8/2 digestions on gel
</DD></DL><DL><DD>1. LCR (PstI)
</DD><DD>2. LCR (EcoRI)
</DD><DD>-Gel extracted 
</DD><DD>-LCR digested with PstI-HF (top band
</DD></DL><DL><DD>-Re-ligate extracted LCR with no insert (should re-ligate to itself)
</DD><DD>-Transformed ligation into ER2738
</DD></DL><UL><LI>Made O/Ns
</LI></UL><DL><DD>-colonies from 8/2 transformation
</DD><DD>-restreak of LCR from freeze down
</DD></DL><H2><SPAN class="mw-headline" id="Week_3">Week 3</SPAN></H2><P><B>8/4</B></P><UL><LI>Check overnights from 8/3 (LCR = Litmus28ibb+pCas9+RFP) for correct inserts
</LI></UL><DL><DD>-Mini-prep samples
</DD><DD>-Digest with EcoRI and PstI
</DD><DD><UL><LI>All had sizes ~3700bp
</LI><LI>No correct inserts
</LI></UL></DD></DL><UL><LI>Attempt to ligate Litmus28ibb and pCas9(gRNA3)
</LI></UL><DL><DD>-Digestions
</DD><DD><UL><LI>Litmus28ibb-RFP    E+P        extracted Lit back bone
</LI><LI>pCas9 gRNA3        E+P        extracted pCas9 insert
</LI></UL></DD></DL><DL><DD>1. Litmus28ibb
</DD><DD>2. pCas9(gRNA3)
</DD><DD>3. pCas9(unmod.)
</DD><DD>-Ligations (10 hrs at 16C, 10 min at 80C)
</DD><DD><UL><LI>Litmus28ibb + pCas9(gRNA3)
</LI></UL></DD></DL><P><B>8/5</B></P><UL><LI>Transformation into ER2738 (7/18), 5ul of ligation products
</LI></UL><DL><DD>1. Litmus28ibb-Cas9gRNA3
</DD></DL><P><B>8/6</B></P><UL><LI>Another attempt at ligating pCas9 and Litmus28ibb
</LI></UL><DL><DD>-Digest unmodified pCas9 with EcoRI-HF and PstI-HF
</DD><DD>-Digest pCas9 containing pRNA1 with EcoRI-HF and PstI-HF
</DD></DL><DL><DD>1. pCas9(unmod.)
</DD><DD>2. pCas9(gRNA1)
</DD><DD>-Gel extract pieces
</DD><DD>-Ligate separately to Litmus28ibb
</DD></DL><UL><LI>Results of 8/5 transformation
</LI></UL><DL><DD>-6 colonies for Litmus28ibb+pCas9(gRNA3) on Tet+reduced Amp
</DD><DD><UL><LI>Picked colonies for overnights
</LI></UL></DD></DL><DL><DD>-No growth for no DNA control
</DD></DL><UL><LI>Transformation of ligations from earlier today
</LI></UL><DL><DD>-Litmus28ibb + pSB1C3-pCas9(unmod.)
</DD><DD>-Litmus28ibb + pSB1C3-pCas9(gRNA1)
</DD></DL><UL><LI>Mini-prep Litmus28ibb+pCas9(gRNA3) O/N samples from earlier in day
</LI></UL><DL><DD>-Digested with EcoRI-HF and PstI-HF
</DD><DD><UL><LI>All samples had bands of 3000 and 2000bp. None were correct
</LI></UL></DD></DL><P><B>8/8</B></P><UL><LI>M13ori into SB Cas9
</LI></UL><DL><DD>-Digestions
</DD><DD>-unmodified pCas9 with EcoRI-HF and XbaI
</DD><DD>-M13ori with EcoRI-HF and SpeI-HF
</DD><DD>-pCas9(gRNA1) with EcoRI-HF and XbaI
</DD><DD>-pCas9(gRNA3) with EcoRI-HF and XbaI
</DD></DL><P><B>8/9</B></P><UL><LI>To clone M13ori onto pSB1C3-pCas9
</LI></UL><DL><DD>-Run 8/8 digestion on a gel
</DD></DL><DL><DD>1. M13ori
</DD><DD>2. pCas9(gRNA3)
</DD><DD>3. pCas9(gRNA1)
</DD><DD>-Gel extracted
</DD><DD>-Ligation Ligated M13ori to….
</DD><DD>-pSB1C3-pCas9(gRNA1)
</DD><DD>-pSB1C3-pCas9(gRNA3)
</DD><DD>-pSB1C3-pCas9(unmod.)
</DD></DL><DL><DD>-Transform and plate onto low Chlor 934 ug/mL)
</DD></DL><H2><SPAN class="mw-headline" id="Week_4">Week 4</SPAN></H2><P><B>8/10</B></P><UL><LI>Digested ligation products from 8/6 and 8/4. Saw many large bands that may have been undigested
</LI><LI>O/Ns of 8/9 transformation
</LI></UL><DL><DD>-1 colony for M13ori-pCas9(gRNA3) (only 1 colony grew)
</DD><DD>-11 colonies for M13ori-pCas9(gRNA1)
</DD><DD>-No colonies grew for M13ori-pCas9(unmodified)
</DD></DL><P><B>8/11</B></P><UL><LI>Test 8/10 O/Ns for correct inserts
</LI></UL><DL><DD>-Digest 8/10 O/Ns with EcoRI-HF and PstI-HF
</DD></DL><DL><DD>1. M13ori-pCas9(gRNA3)
</DD><DD>2-12. M13ori-pCas9(gRNA1)
<UL><LI>Lanes 1, 8, and 9 are about the right size. Sent these samples for sequencing
</LI></UL></DD></DL><UL><LI>To ligate M13ori to pSB1C3-pCas9(unmod.)
</LI></UL><DL><DD>-Digest
</DD><DD><UL><LI>pSB1C3-pCas9(unmod.) (E+P)
</LI><LI>M13 digested 8/8
</LI></UL></DD></DL><DL><DD>-Ligation: See L5 below
</DD></DL><UL><LI>To ligate pCas9 and M13ori to pSB1A3
</LI></UL><DL><DD>-Digest
</DD><DD><UL><LI>pSB1A3 (E+P)
</LI><LI>M13ori (E+S)
</LI><LI>Unmodified pCas9 (X+P)
</LI><LI>pCas9(gRNA1)-M13ori (E+P)
</LI><LI>pCas9(gRNA3)-M13ori (E+P)
</LI></UL></DD></DL><DL><DD>-Ligation: See L2-4 below
</DD></DL><UL><LI>Another try to put pCas9 onto Litmus28ibb
</LI></UL><DL><DD>-Digested
</DD><DD><UL><LI>Unmodified Litmus28i (E+P)
</LI><LI>pCas9(unmod.) (E+P)
</LI><LI>pCas9(gRNA1)-M13ori (E+P)
</LI><LI>pCas9(gRNA3)-M13ori (E+P)
</LI></UL></DD></DL><DL><DD>-Ligation: See L1 and L6 below
</DD></DL><UL><LI>Gel for all
</LI></UL><DL><DD>1. Unmodified Litmus28i
</DD><DD>2. pSB1C3-pCas9(unmodified)
</DD><DD>3. pSB1A3
</DD><DD>4. pSB1C3-M13ori
</DD></DL><DL><DD>5. pSB1C3-pCas9(unmod.)
</DD><DD>6. pSB1C3-M13ori-pCas9(gRNA1)
</DD><DD>7. pSB1C3-M13ori-pCas9(gRNA3)
</DD></DL><UL><LI>Ligations for both projects
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  #
</TH><TH> Backbone
</TH><TH> Insert 1
</TH><TH> Insert 2
</TH></TR><TR><TD> 1
</TD><TD> Unmod. Litmus28i (E+P)
</TD><TD> pCas9(unmod.) (E+P)
</TD><TD> N/A
</TD></TR><TR><TD> 2
</TD><TD> pSB1A3 (E+P)
</TD><TD> pCas9(unmod.) (X+P)
</TD><TD> M13ori (E+S)
</TD></TR><TR><TD> 3
</TD><TD> pSB1A3 (E+P)
</TD><TD> pCas9(gRNA1)-M13ori (E+P)
</TD><TD> N/A
</TD></TR><TR><TD> 4
</TD><TD> pSB1A3 (E+P)
</TD><TD> pCas9(gRNA3)-M13ori (E+P)
</TD><TD> N/A
</TD></TR><TR><TD> 5
</TD><TD> pSB1C3-pCas9(unmod.) (E+X)
</TD><TD> M13ori (E+S)
</TD><TD> N/A
</TD></TR><TR><TD> 6
</TD><TD> Litmus28ibb (E+P)
</TD><TD> pCas9(gRNA3) (E+P)
</TD><TD> N/A
</TD></TR><TR><TD> 7
</TD><TD> Litmus28ibb (E+P)
</TD><TD> pCas9(gRNA1) (E+P)
</TD><TD> N/A
</TD></TR><TR><TD> 8
</TD><TD> Litmus28ibb (E+P)
</TD><TD> pCas9(unmod.) (E+P)
</TD><TD> N/A
</TD></TR></TBODY></TABLE><P><B>8/12</B></P><UL><LI>Transform ligation from 8/11 into ER2738 cells
</LI><LI>Make phage using M13g6A1 as Helper Phage
</LI></UL><DL><DD>-used pCas9(gRNA1)-M13ori or pCas9(gRNA3)-M13ori as phagemid
</DD></DL><UL><LI>Freeze downs
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Top
</TH><TH> Side
</TH></TR><TR><TD> M. gRNA3 Cas9 (8/12)
</TD><TD> M13ori Cas9 gRNA3 pSB1C3
</TD></TR><TR><TD> M. gRNA1 Cas9 (8/12)
</TD><TD> M13ori(8) Cas9 gRNA1 pSB1C3
</TD></TR><TR><TD> M. gRNA1 Cas9 (8/12)
</TD><TD> M13ori (9) Cas9 gRNA1 pSB1C3
</TD></TR></TBODY></TABLE><P><B>8/13</B></P><UL><LI>Finish phage isolation
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Helper phage
</TH><TH> Phage
</TH><TH> A269
</TH><TH> A320
</TH><TH> genome size
</TH><TH> [ ]
</TH></TR><TR><TD> Fd-CAT
</TD><TD> M. gRNA1
</TD><TD> 0.353
</TD><TD> 0.025
</TD><TD> 7687
</TD><TD> 2.56 x10^12
</TD></TR><TR><TD> Fd-CAT
</TD><TD> M. gRNA3
</TD><TD> 0.554
</TD><TD> 0.037
</TD><TD> 7687
</TD><TD> 4.40 x10^12
</TD></TR></TBODY></TABLE><UL><LI>Infection
</LI></UL><DL><DD>-Infecting ER2738 and BWF’
</DD><DD>-Phage
</DD><DD><UL><LI>pSB1C3-pCas9(gRNA1)
</LI><LI>pSB1C3-pCas9(gRNA3)
</LI></UL></DD></DL><UL><LI>Transformation Results from 8/12
</LI></UL><DL><DD>-Growth for the following… (made O/Ns)
</DD><DD><UL><LI>pSB1A3-M13ori-pCas9(gRNA3)
</LI><LI>pSB1C3-M13ori-pCas9(unmod)-M13ori
</LI><LI>Litmus28ibb-pCas9(gRNA3)
</LI><LI>Litmus28ibb-pCas9(gRNA1)
</LI><LI>Litmus28ibb-pCas9(unmod.)
</LI></UL></DD></DL><DL><DD>-Hundreds of colonies for pCas9(unmod.) on Unmod. Litmus28i. Didn’t trust result so didn’t pick coloneis. EcoRI and PstI sites touch so most likely, Litmus28i was only cut once
</DD></DL><P><B>8/15</B></P><UL><LI>Check O/Ns from 8/14 for correct inserts
</LI></UL><DL><DD>-Mini-prepped samples
</DD><DD>-Digested with E and P
</DD><DD>-Results
</DD><DD><UL><LI>No bands of the right size for
</LI></UL></DD><DD>-Litmus28ibb-pCas9(gRNA3)
</DD><DD>-Litmus28ibb-pCas9(gRNA1)
</DD><DD>-Litmus28ibb-pCas9(unmod.)
</DD><DD>-Expected band sizes for… (see below)
</DD></DL><UL><LI>pCas9(gRNA3)-M13ori on pSB1A3
</LI><LI>pCas9(unmod)-M13ori on pSB1C3
</LI></UL><UL><LI>Transform into BWF’ to test for functionality of Cas9 when associated with m13ori, independent of infectivity
</LI></UL><DL><DD>-M13ori-pCas9(gRNA1)
</DD><DD>-M13ori-pCas9(gRNA3)
</DD></DL><P><B>8/16</B></P><UL><LI>PCR on LCR (Litmus28ibb with pCas9 and RFP to maybe get a good product to religate
</LI></UL><DL><DD>-Didn’t work
</DD></DL><P>Results of 8/13 infection of BWF’ with M13ori-gRNA(1 or 3) to test pCas9 killing
</P><DL><DD>No growth on DNA controls on Chlor(34ug/mL) or Kan(50ug/mL)+Chlor(34ug/mL)
</DD><DD>ON Kan(50ug/mL) + Chlor (170ug/mL) in BWF'
</DD><DD>gRNA1--&gt; 22 colonies
</DD><DD>gRNA3--&gt; 2 colonies
</DD></DL><P>On Kan(25ug/mL)+Chlor(34ug/mL) in BWF’
</P><TABLE class="wikitable"><TBODY><TR><TH> 1:100
</TH><TH> 1:1000
</TH></TR><TR><TD> gRNA1
</TD><TD> 617
</TD><TD> 84
</TD></TR><TR><TD> gRNA3
</TD><TD> 469
</TD><TD> 42
</TD></TR></TBODY></TABLE><P>On Chlor (34ug/mL) in BWF’
</P><TABLE class="wikitable"><TBODY><TR><TH> 1:100
</TH><TH> 1:1000
</TH></TR><TR><TD> gRNA1
</TD><TD> 508
</TD><TD> 67
</TD></TR><TR><TD> gRNA3
</TD><TD> 447
</TD><TD> 41
</TD></TR></TBODY></TABLE><P>On Chlor (34ug/mL) in ER
</P><TABLE class="wikitable"><TBODY><TR><TH> 1:100
</TH><TH> 1:1000
</TH></TR><TR><TD> gRNA1
</TD><TD> 702
</TD><TD> 93
</TD></TR><TR><TD> gRNA3
</TD><TD> 518
</TD><TD> 58
</TD></TR></TBODY></TABLE><P>Results of 8/15 transformation of BWF’ with M13ori—gRNA(1 or 3) to test killing
</P><PRE>Chlor (34ug/mL) + Kan (50ug/mL)
</PRE><TABLE class="wikitable"><TBODY><TR><TH> 1:1
</TH><TH> 1:10
</TH></TR><TR><TD> gRNA1
</TD><TD> Lawn
</TD><TD> 1904
</TD></TR><TR><TD> gNRA3
</TD><TD> Slightly thinner lanw
</TD><TD> 1616
</TD></TR></TBODY></TABLE><P>Chlor (34 ug/mL)
</P><TABLE class="wikitable"><TBODY><TR><TH> Trial 1
</TH><TH> Trial 2
</TH></TR><TR><TD> gRNA1
</TD><TD> 1788
</TD><TD> 1548
</TD></TR><TR><TD> gRNA3
</TD><TD> 1564
</TD><TD> 1416
</TD></TR></TBODY></TABLE><P>Samples were diluted 1:10
</P><H2><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H2><P><B>8/17</B>
Make phage
</P><DL><DD>pSB1C3-M13ori-pCas9(unmod)
</DD><DD>pSB1A3-M13ori-pCas9(gRNA3)
</DD></DL><P>Ran uncut LCR on a gel and gel extracted the three bands
</P><P><B>8/18</B>
While making phage…. We threw away the samples….
Remake phage. Same as 8/17
Transform LCR gel extractions into 5alpha cells
</P><P>8/19
Sent samples for sequencing
</P><DL><DD>pSB1C3-M13ori-pCas9(unmodified)
</DD><DD>pSB1A3-M13ori-pCas9(gRNA3)
</DD></DL><P>Finish making phage
</P><TABLE class="wikitable"><TBODY><TR><TH>  Name
</TH><TH> A269
</TH><TH> A320
</TH><TH> Genome Size
</TH><TH> [ ]
</TH></TR><TR><TD> pSB1A3-M13ori-pCas9(gRNA3)
</TD><TD> 0.697
</TD><TD> 0.162
</TD><TD> Note: We later found that this sample did not actually contain the M13ori
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> 0.764
</TD><TD> 0.146
</TD><TD> 7687
</TD><TD> 3.044 x10^13
</TD></TR></TBODY></TABLE><P>Infect cells (ER and BWF’)
</P><DL><DD>Grew a single 75mL sample. When OD ~0.5, divided samples into 3
</DD><DD>Added 20mL of sample to 125mL flask (3 for each strain)
</DD><DD>Added phage to a final concentration of 1 x10^8 phage/mL
</DD><DD>Incubated at 250rpm, 37C, for 30 minutes
</DD><DD>Plated at dilutions (1:10, 1:100, and 1:1k)
</DD><DD>On Chlor(170ug/mL), Kan(50ug/mL)+Chlor(85ug/mL), Kan(50ug/mL)+Chlor(170ug/mL)
</DD></DL><P><B>8/21</B>
Finished phage amplification protocol from 8/20 but did not receive any phage. 
Infect BWF’ cells with the phage having pSB1C3-M13ori-pCas9 containing the scramble, gRNA1, and gRNA3.
</P><P>'<I>8/22</I>
Present project to department
</P><H2><SPAN class="mw-headline" id="Week_6">Week 6</SPAN></H2><P><B>8/25</B>
Results of 8/21 infection experiment
</P><P>Results on Kan and Chlor
</P><TABLE class="wikitable"><TBODY><TR><TH> Scramble
</TH><TH> gRNA1
</TH><TH> gRNA3
</TH></TR><TR><TD> 1:1
</TD><TD> 149
</TD><TD> 143
</TD><TD> 13
</TD></TR><TR><TD> 1:10
</TD><TD> 14
</TD><TD> 16
</TD><TD> 1
</TD></TR></TBODY></TABLE><P>Results on Chlor
</P><TABLE class="wikitable"><TBODY><TR><TH> Scramble
</TH><TH> gRNA1
</TH><TH> gRNA3
</TH></TR><TR><TD> 1:1
</TD><TD> 147
</TD><TD> 117
</TD><TD> 14
</TD></TR><TR><TD> 1:10
</TD><TD> 14
</TD><TD> NA
</TD><TD> 3
</TD></TR></TBODY></TABLE><P><B>8/28</B>
Retry ligation of pCas9 to litmus28ibb
	Digest Litmus28ibb-J04450, pCas9, and amilCP with EcoRI and PstI. Gel extract pieces
	Ligate pCas9 into Litmus28ibb and amilCP into Limus28ibb
</P><P><B>8/29</B>
Transform 8/28 ligation products into 5alpha cells
</P><P><B>8/29</B>
Transform pSB1C3-pCas9(unmodified) into BW and BWF’ to test targeting of f’ episome. Due to poor infection results we wanted to test the guide RNAs for targeting of the F’ episome. We have searched the vast webs for the sequence of the F’ episome but have yet to (and never will) find it.
</P><P><B>8/30</B>
Transformation results of pCas9 into BW and BWF’. Results were unexpected, we  expected more growth for BW sample or at least equal growth. Differences in competence or concentration could explain results in initial count
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Initial count
</TH><TH> After another 24 hours
</TH></TR><TR><TD> No DNA controls (for BW and BWF’)
</TD><TD> No growth
</TD><TD> No growth
</TD></TR><TR><TD> BW cells
</TD><TD> 9 colonies
</TD><TD> 222
</TD></TR><TR><TD> BWF’ cells
</TD><TD> ~100 colonies
</TD><TD> 158
</TD></TR></TBODY></TABLE><P>Transformation results for ligation of pCas9 to Litmus28ibb
</P><TABLE class="wikitable"><TBODY><TR><TH> Amp (50ug/mL)
</TH><TH> Amp (100ug/mL)
</TH><TH> Notes
</TH></TR><TR><TD> No DNA control
</TD><TD> 3
</TD><TD> 10
</TD><TD> All bright white
</TD></TR><TR><TD> Litmus28ibb-pCas9
</TD><TD> ~50 (many are red)
</TD><TD> 25-30
</TD><TD> High proportion are red. Some are shadowy white. A few are bright white
</TD></TR><TR><TD> Litmus28ibb-amilCP
</TD><TD>&gt;300
</TD><TD>&gt;300
</TD><TD> Mostly shadowy, some red, some bright white
</TD></TR></TBODY></TABLE><P>Made overnight cultures of shadowy and bright coloneis from Litmus28ibb-pCas9 samples (from both plates) and of shadowy colonies from Litmus28ibb-amilCP. Since growth on no DAN control was bright white, we assumed that bright colonies on sample plates were more likely to be contamination, but didn’t really know. Since there were fewer colonies on no DNA control plates, we figures we could randomly select colonies from the sample plates that were not a result of contamination.
</P><H2><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H2><P><B>8/31</B>
Mini-prep overnight cultures from 8/30 and digest samples with EcoRI and PstI. When run on a gel, none of the Litmus28ibb-pCas9 samples yielded bands of the correct size. All Litmus28ibb-amilCP samples did yield the expected band sizes. 
</P><P><B>9/2</B>
Sent some of mini-prep samples from 8/31 for sequencing
</P><P><B>9/3</B>
Digestion of pCas9 with EcoRI and speI and the unmodified NEB version of Litmus28i with EcoRI and XbaI. Gel extracted pieces and ligated together (No, this part would not have the BioBrick prefix/suffix)
</P><P><B>9/4</B>
Transform 9/3 Litmus28i-pCas9 ligation into 5alpha cells. Selected on low ampicillin (50ug/mL) and high ampicillin (100ug/mL)
</P><P><B>9/5</B>
Did receive colonies from 9/4 transformation. Selected 24 colonies for colony PCR with a primer pair that would span the junction of pCas9 and Litmus28i. (forward primer bound to CRISPR region of pCas9 while reverse primer bound in the middle of the Litmus28i backbone)
Colony PCR failed to reveal any promising samples. Set up overnight cultures in a hopeless effort to yield different results.
</P><H2><SPAN class="mw-headline" id="Week_8">Week 8</SPAN></H2><P><B>9/7</B>
Mini-prepped overnight cultures from 9/5. Digested each with EcoRI and PstI and ran reults on gel. Results were consistent with colony PCR: failure. A few samples had very feint bands that were approximately the correct size
</P><P><B>9/8</B>
for the samples on 9/7 with light bands of approximately the correct size, we sent samples for sequencing (another hopeless attempt to get different results)
</P><P><B>9/9</B>
Sequencing results showed that Limtus28i religated
</P><H2><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H2><P><B>9/17</B></P><P>New method to clone pCas9 onto Litmus28ibb vector: PCR-&gt;Digestion-&gt;ligation method
</P><DL><DD>PCR amplified pCas9 construct from pSB1C3-pCas9. PCR amplified Litmus28ibb backbone from Litmus28ibb-J04550
</DD><DD>PCR purified both samples. Digested both samples with EcoRI and PstI and gel extracted the pieces
</DD><DD>Ligated pCas9 to Litmus28ibb vector
</DD></DL><P>Sent pSB1C3-M13ori-pCas9 sample for sequencing to sequence the entire plasmid
</P><P><B>9/18</B></P><P>Transform 9/17 ligation (Litmus28ibb-pCas9) into 10beta cells
</P><P><B>9/19</B></P><P>There was growth on no DNA control. Some colonies for sample.
</P><P><B>9/20</B></P><P>Make chemically competent ER and 5alpha cells using DMSO from bottle instead of fancy tubes. We think this is the reason for later transformation failures
</P><P>Digested pCas9 and Litmus28ibb PCR purifications from 9/17 with EcoRI, PstI, and DpnI. PCR purified digestion samples then ligated pieces together.
</P><H2><SPAN class="mw-headline" id="Week_10">Week 10</SPAN></H2><P><B>9/21</B></P><P>New idea: Cloning pCas9 onto Litmus28i has not worked up until this point. What if we constructed a vector that looked similar to litmus28i. pSB1A3 has the same origin of replication and almost the same ampicillin resistance marker (with a few SNPs). Need to add M13ori (which was taken from Litmus28i)  and pCas9. M13ori would not be in the correct position on plasmid but this is as close as we can get with the remaining time
</P><DL><DD>Digestions: M13ori-pCas9(unmod), pCas9(unmod), pSB1A3 with EcoRI and PstI. Gel showed incomplete digestion so did not gel extract
</DD></DL><P>Transformation into 5alpha cells: no DNA control, 9/20 ligation (Litmus28ibb-pCas9), pSB1C3-amilCP, and pSB1C3-M13ori-pCas9
</P><P><B>9/22</B></P><P>Re-digested M13ori-pCas9(unmod) and pSB1A3-J04450 with EcoRI and PstI. pCas9(unmod) was PCR purified (product looked clean on gel so did not bother to gel extract). 
Ligated M13ori-pCas9(unmod) to pSB1A3 and pCas9(unmod) to pSB1A3
Sent pSB1A3 to M13ori-pCas9 for sequencing to cover region not covered by previous sequencing
</P><P><B>9/23</B></P><P>Sequencing results came back
</P><DL><DD>Litmus28ibb-J04450 contains two nucleotides that are flipped. They are located within the ampicillin resistance gene and do change two amino acids; however, sample grows on ampicillin so these are not loss of function mutation. 
</DD><DD>pSB1C3-M13ori-pCas9 was perfect - the above two samples are ready to submit to the registry
</DD></DL><P>Transform ligations from 9/22 (pSB1A3- M13ori-pCas9(unmod) and pCas9(unmod)) into 5 alpha cells
Colony PCR 18 samples from 9/20 Limtus28ibb-pCas9(unmod) plate.
</P><P><B>9/24</B></P><P>Run gel of 9/23 colony PCR to check for desired Litmus28ibb-pCas9(unmod). There was a band of approximately the correct size in each samples (including the no DNA control). Set up overnight cultures of first 5 samples for further testing
Results of 9/23 transformation
</P><DL><DD>No growth for no DNA control on chlor (170ug/mL), Amp (50ug/mL) or Amp (100ug/mL)
</DD><DD>7 colonies for pSB1C3-M13genes on chlor (170ug/mL)
</DD><DD>27 colonies for pSB1A3-pCas9 on Amp (100ug/mL) and 41 colonies on Amp (50ug/mL)
</DD><DD>0 colonies for pSB1A3-M13ori-pCas9 on Amp(50 and 100ug/mL)
</DD></DL><P>Set up overnight cultures of some of the samples.
</P><P><B>9/25</B></P><P>Mini-prep overnights from 9/24
Digest samples to check for insert. When run on a gel, 3 samples had large bands of ~5000bp and one had an additional band at ~6000. We made overnights of these 3 colonies
</P><P><B>9/26</B></P><P>Digested mini-prepped DNA from the three samples that looked promising from 9/25. Each sample was digested with only EcoRI, only PstI, or EcoRI and PstI. When run on a gel, it can be seen that each samples, when cut once, yielded one band 8000bp while when cut twice, yielded 2 bands of ~2000 and 5000 as expected.
Sent parts to iGEM registry: pSB1C3-M13ori-pCas9 and Litmus28ibb-J04450
To add the M13ori to pSB1A3-Cas9
</P><DL><DD>Digested pSB1A3-pCas9 with EcoRI and XbaL
</DD><DD>Digested pSB1C3-M13ori with EcoRI and SpeI
</DD><DD>Gel extracted bands of the correct sizes and combined these into a ligation reaction.
</DD></DL><P><B>9/27</B>
Transform pSB1A3-M13ori-pCas9 into Amp(50ug/mL) and Amp(100ug/mL)
Later in the day…. Set up overnight cultures of colonies
</P><H2><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H2><P><B>9/28</B></P><P>Mini-prepped overnights made 9/27
Digested overnight samples with EcoRI and PstI
Did receive pSB1A3-M13ori-pCas9 samples with bands of the correct size (as compared to pSB1A3-pCas9)
Set up overnight cultures of the samples that looked promising
Transformed pSB1A3-M13ori-pCas9(unmod) and pSB1A3-M13ori-pCas9(gRNA3) into ER2738 cells
</P><P><B>9/29</B></P><P>Did receive colonies from 9/28 transformation. Set up overnight cultures for both.
Also set up overnight cultures of ER2738 and BWF’ for upcoming infection.
Start phage amplification protocol using pSB1A3-M13ori-pCas9(unmod) and pSB1A3-M13ori-pCas9(gRNA3) as the phagemid. Used Fd-Cat as the helper phagemid
</P><P><B>9/30</B></P><P>Finished phage amplification/isolation protocol for Fd phage packaging pSB1A3-M13ori-pCas9(unmod) and pSB1A3-M13ori-pCas9(gRNA3)
</P><P><B>10/1</B></P><P>Measured concentration of phage isolated on 9/30. There were no phage for the gRNA3 sample. The pSB1A3-M13ori-pCas9(unmod) sample contained some phage but had a high level of contamination/impurity.
Send samples pSB1A3-M13ori-pCas9(unmod) and pSB1A3-M13ori-pCas9(gRNA3) samples for sequencing to verify the presence of the M13ori
</P><P><B>10/2</B></P><P>Sequencing results came in
</P><DL><DD>pSB1A3-M13ori-pCas9(unmod) looks good
</DD><DD>pSB1A3-M13ori-pCas9(gRNA3) doesn’t have the M13ori
</DD></DL><P>Plan
</P><DL><DD>1.SDM on pSB-M13ori-pCas9(unmod) to remove BsaI cut site in pSB1A3 backbone
</DD><DD>2.Change spacer from unmodified to the gRNA3
</DD><DD>3.Redo the infection experiment
</DD></DL><P>SDM
</P><DL><DD>PCR to amplify pSB1A3-M13ori-pCas9(unmod)
</DD><DD>Treat with kinase, Ligase, and DpnI
</DD><DD>Transform into NEB 5alpha cells
</DD></DL><P><B>10/3</B></P><P>Set up O/N cultures of 10/2 transformation
</P><P><B>10/4</B></P><P>Mini-prep overnights. Samples are called pSB1A3(delta BasI)-M13ori-pCas9(unmod) to indicate each part of the construct and the removed BsaI site in the pSB1A3 backbone.
Changing the gRNA spacer in pCas9
</P><DL><DD>Vector digest: digested pSB1A3(delta BasI)-M13ori-pCas9(unmod) with BsaI to remove unmodified spacer. Saw only one band on the gel (cut around spacer) which shows that the additional BsaI site in the pSB1A3 backbone was removed. Gel extracted the digested vector
</DD><DD>Ligation: Used the annealed oligos for gRNA3 (our targeting sample) from the last time we changed the spacer. Ligated these oligos to the digested pSB1A3(delta BasI)--M13ori-pCas9
</DD></DL><H2><SPAN class="mw-headline" id="Week_12">Week 12</SPAN></H2><P><B>10/5</B></P><P>Transform pSB1A3(delta BasI)-M13ori-pCas9(unmod) and pSB1A3(delta BasI)-M13ori-pCas9(gRNA3) in ER2738 cells
</P><P><B>10/6</B></P><P>Transformation results from 10/5
</P><DL><DD>No growth for no DNA control
</DD><DD>pSB1A3(delta BasI)-M13ori-pCas9(unmod) = many colonies
</DD><DD>pSB1A3(delta BasI)-M13ori-pCas9(gRNA3) = no colonies
</DD></DL><P>Re-anneal primers then relegate
</P><DL><DD>Phosphorylation of oligos for gRNA3. Incubate for 1 hour at 37C then 20 minutes at 65C
</DD><DD>Add 2.5ul 1M NaCl
</DD><DD>Anneal in thermocycler. 5 minutes at 95C then a slow cool-down for ~2 hours
</DD><DD>Dilute product 1:10
</DD><DD>Ligation of newly annealed primers to digest and gel extracted pSB1A3(delta BasI)-M13ori-pCas9(unmod) from 10/5
</DD></DL><P><B>10/7</B></P><P>Transform 10/6 ligation into ER2738 cells
</P><P><B>10/8</B></P><P>No growth for 10/7 transformation
Transform ligation from 10/6 into 5alpha cells
Last try to make pSB1A3-M13ori-pCas9(gRNA3)
</P><DL><DD>Know from 10/1 sequencing that we have pSB1A3-pCas9(gRNA3). Therefore, only need to add M13ori upstream of pCas9 part
</DD><DD>Digest pSB1A3-pCas9(gRNA3) with EcoRI and XbaI and pSB1C3-M13ori with ExoRI and SpeI
</DD><DD>Gel extract digested pieces
</DD><DD>Ligate gel extracted pieces together
</DD></DL><P><B>10/9</B></P><P>No growth for 10/8 transformation
Transform 10/8 ligation into 5alpha cells
</P><P><B>10/10</B></P><P>Transform 
</P><DL><DD>pSB1C3-M13ori-pCas9(unmod) from 10/9 into ER2738 cells
</DD><DD>pSB1A3 (delta BsaI)-M13ori-pCas9(gRNA3) from 10/6 into 5 alpha cells
</DD><DD>pSB1A3-M13ori-pCas9(gRNA3) from 10/8 into 5 alpha cells
</DD></DL><P><B>10/11</B></P><P>Transformation results of 10/10
</P><DL><DD>1 colony for pSB1C3-M13ori-pCas9(unmod) in ER2738 (does not seem trustworthy)
</DD><DD>No colonies for no DNA control
</DD><DD>No colonies for the 2 other transformations
</DD></DL><P>Transform pSB1C3-M13ori-pCas9(unmod) into ER2738 cells again
Start phage amplification protocol to make the following phage
</P><TABLE class="wikitable"><TBODY><TR><TH>  Phagemid
</TH><TH> Helper Phagemid
</TH></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> M13g6A1
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> M13K07
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(gRNA3)
</TD><TD> M13g6A1
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(gRNA3)
</TD><TD> M13K07
</TD></TR></TBODY></TABLE><P>pSB1C3-M13ori-pCas9(unmod) grew unexpectedly fast so we missed the OD mark. Growth rate compared to pSB1C3-M13ori-pCas9(gRNA3) sample suggests contamination. Did not proceed with making these phage for today.
</P><P><B>10/12</B></P><P>No growth for 10/11 transformation – conclude that ER2738 cells are not competent (these are the cells made 9/20 with DMSO from a room temp bottle instead of the alliquotes from NEB.
Remake phage
</P><TABLE class="wikitable"><TBODY><TR><TH>  Phagemid
</TH><TH> Helper Phagemid
</TH></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> M13g6A1
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> M13K07
</TD></TR></TBODY></TABLE><P><B>10/13</B></P><P>Finish phage isolation protocol and measure
</P><TABLE class="wikitable"><TBODY><TR><TH>  Phagemid
</TH><TH> Helper Phagemid
</TH><TH> Dilution
</TH><TH> A269
</TH><TH> A320
</TH><TH> Genome size
</TH><TH> Diluted [ ]
</TH><TH> Actual [ ]
</TH></TR><TR><TD> pSB1C3-M13ori-pCas9(gRNA3)
</TD><TD> M13g6A1
</TD><TD> 1:40
</TD><TD> 0.090
</TD><TD> 0.015
</TD><TD> 7687
</TD><TD> 5.85 E11
</TD><TD> 2.34 E13
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(gRNA3)
</TD><TD> M13K07
</TD><TD> 1:40
</TD><TD> 0.085
</TD><TD> 0.013
</TD><TD> 7687
</TD><TD> 5.62 E11
</TD><TD> 2.25 E13
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> M13g6A1
</TD><TD> 1:40
</TD><TD> 0.104
</TD><TD> 0.022
</TD><TD> 7687
</TD><TD> 6.40 E11
</TD><TD> 2.56 E13
</TD></TR><TR><TD> pSB1C3-M13ori-pCas9(unmod)
</TD><TD> M13K07
</TD><TD> 1:20
</TD><TD> 0.091
</TD><TD> 0.016
</TD><TD> 7687
</TD><TD> 5.85 E11
</TD><TD> 1.17 E13
</TD></TR></TBODY></TABLE><P>Infection with phage of BW23115F’ cells
</P><DL><DD>Grew a single stock of cells then dived into 4 samples each with 50mL cells. Added the above phage
</DD><DD>Incubated for 30 minutes at 37C, shaking
</DD><DD>Plated onto Chlor
</DD></DL><H2><SPAN class="mw-headline" id="Week_13">Week 13</SPAN></H2><P><B>10/14</B></P><P>Infection results
</P><DL><DD>No growth on Chlor (170ul/mL)
</DD><DD>Very little to no growth on pCas9(unmod) samples. We think this is b/c the colony picked on 10/11 did not actually contain pSB1C3-M13ori-pCas9(unmod) so nothing was packaged into phage
</DD><DD>~100 colonies for pCas9(gRNA3) samples.
</DD><DD>The samples made with M13g6A1 were also plated on ampicillin. These plates show considerable growth. There is more growth on 
</DD></DL><P>pCas9(unmod) than pCas9(gRNA3) sample plate. We believe this further supports our hypothesis that our pCas9(unmod) sample did not contain pSB1C3-M13ori-pCas9(unmod) so packaged M13g6A1 in the absence of a better phagemid. Also, for the pCas9(gRNA3) sample, there was considerably more growth on the ampicillin plate than on the chlor plate. This could be because pSB1C3-M13ori-pCas9(gRNA3) successfully killed the BW23115F’ cells as desired or the phagemid was not effectively packaged
</P><P>Transformed pSB1C3-M13ori-pCas9(unmod) into chemically competent ER2738 cells that we know work from July.
</P><DIV class="newfooter"><DIV id="newfooterbox"><DIV class="footertext"><UL><LI><A href="https://igem.org/About">About</A></LI><LI><A href="https://igem.org/Contact">Contact</A></LI><LI><A href="https://igem.org/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Main_Page">  igem.org</A></LI><LI><A href="http://parts.igem.org/Main_Page"> Registry</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://2014.igem.org/Requirements"> Requirements</A></LI><LI><A href="https://2014.igem.org/Giant_Jamboree"> Giant Jamboree</A></LI><LI><A href="https://2014.igem.org/Meetups"> Meetups</A></LI><LI><A href="https://2014.igem.org/Calendar_of_Events">Calendar </A></LI><LI><A href="https://2014.igem.org/Partner_Offers">Partner Offers</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Login">Login</A></LI><LI><A href="https://2014.igem.org/Special:Upload">Upload Files</A></LI><LI><A href="https://2014.igem.org/Special:RecentChanges"> Recent Changes</A></LI><LI><A href="https://2014.igem.org/Special:SpecialPages"> Special Pages</A></LI><LI><A href="https://2014.igem.org/Special:Search"> Search</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Press_Kit">Press Kit</A></LI><LI><A href="https://igem.org/Newsletters"> Newsletter</A></LI><LI><A href="https://igem.org/Alumni">AlumniGEM</A></LI></UL></DIV><DIV class="footerextra">
powered by <A href="http://www.mediawiki.org/" style="link-style:none;">MediaWiki</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:CU-Boulder/Notebook/CC9Phage_Team">http://2014.igem.org/Team:CU-Boulder/Notebook/CC9Phage_Team</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:CU-Boulder/Notebook/CC9Phage_Team" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:CU-Boulder/Notebook/CC9Phage_Team" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9Phage_Team&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9Phage_Team&amp;oldid=264837" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>