<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_CU-Boulder_Notebook_CC9_Team_Constitutive"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:CU-Boulder/Notebook/CC9_Team/Constitutive">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:CU-Boulder/Notebook/CC9_Team/Constitutive&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Constitutive&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Constitutive&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:CU-Boulder/Notebook/CC9_Team/Constitutive" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:CU-Boulder/Notebook/CC9 Team/Constitutive</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><HEADER id="topNav"><NAV id="navContainer"><UL class="navLinks"><LI class="navSingleLink"><A href="https://2014.igem.org/Team:CU-Boulder">HOME</A></LI><LI><A>PROJECT</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Getting Started</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Motivation">Motivation</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Modeling">Modeling</A></LI></UL></NAV></HEADER></DIV></DIV></DIV><LI><A>ACHIEVEMENTS</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Our Team Achievements</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Biobricks">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Medal">Medal Requirements</A></LI><LI><A>NOTEBOOK</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">At the Bench</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Phage_Team">Phage Delivery</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Induced">Induced CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Constitutive">Constitutive CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9Phage_Team">Phage CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Measurement">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Protocols">Protocols</A></LI><LI><A>HUMAN PERSPECTIVE</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Reaching Out</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Safety">Ethics &amp; Safety</A></LI><LI><A>TEAM</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">MI6</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Acknowledgments">Acknowledgments</A></LI><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_1"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_7"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 7</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week8"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week8</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week_9"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 9</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Week_10"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 10</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week_11"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week 11</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Week_13.2F14"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Week 13/14</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H2><P><B>6/4</B></P><UL><LI>Resuspended BBa_K1218011 (biobricked pCas9) from Distribution Kit (4M)
</LI></UL><DL><DD>-Part biobricked by Stanford-Brown 2013
</DD><DD>-Part contains native tracrRNA, promoter, and Cas9 (the Cas9 was modified to remove an EcoRI digestion site) along with a minimal CRISPR array
</DD></DL><UL><LI>Made chemically competent DH3alpha cells
</LI><LI>Tested new competent cells through transformation
</LI></UL><DL><DD>-DNA
<UL><LI>1ul p11+RBS(2) (4/23)
</LI><LI>2ul 2B
</LI><LI>2ul 2P
</LI><LI>3ul 2P
</LI><LI>No DNA
</LI></UL></DD></DL><DL><DD>-Plated onto Chlor
</DD><DD>-Also, thawed 5a cells for 45 minutes then returned to freezer
<UL><LI>Tomorrow, we will transform and test competency
</LI><LI>Also include non-competent control
</LI></UL></DD></DL><P><B>6/5</B></P><UL><LI>Transformation results from 6/4
</LI></UL><DL><DD>- No growth on negative (no DNA) control: no contamination
</DD><DD>- &gt; 400 colonies for positive (p11+RBS (2), diluted 1:10) control. Lawn when not diluted
</DD><DD>- ~100 colonies for 2B (2ul) not diluted, 18 colonies on 1:10 dilution
</DD><DD>- 3 colonies for 2P (2ul) not diluted, 0 on 1:10 dilution
</DD><DD>- 20 colonies for 2P (3ul) not diluted, 1 on 1:10 dilution
</DD></DL><UL><LI>Made overnights of 2B and 2P to make freeze downs tomorrow
</LI></UL><P><B>6/7</B></P><UL><LI>Made 6mL O/N cultures
</LI></UL><DL><DD>-4 from (3) cas9 plate
</DD></DL><H2><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H2><P><B>6/8</B></P><UL><LI>Check colonies for presence of Cas9 plasmid
</LI></UL><DL><DD>-Mini-prepped O/Ns from 6/7
</DD><DD>-Digest with EcoRI and PstI (10ul reactions)
</DD><DD>-Run results on a gel
</DD><DD><UL><LI>All 4 samples had the expected bands of 2000 and 5000bp showing that we had Cas9 safely inside cells
</LI></UL></DD></DL><DL><DD>Gel of mini-preps containing Cas9
</DD><DD>-2-log ladder
</DD></DL><P><B>6/9</B></P><UL><LI>Freeze downs
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Top Label
</TH><TH> Side label
</TH><TH> What?
</TH></TR><TR><TD> Cas9     6/9
</TD><TD> From dis kit 1
</TD><TD> Cas9 construct from Sanford-Brown team
</TD></TR><TR><TD> -&gt; K1218011 (tracrRNA, Cas9, minimal CRISPR)
</TD></TR><TR><TD> Cas9     6/9
</TD><TD> From dis kit 2
</TD><TD> “ “
</TD></TR></TBODY></TABLE><UL><LI>Overnight cultures of Cas9 (Qi wrCas9 #P44250) and gRNA (Qi gRNA #P44251 cells from Andrew
</LI></UL><H2><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H2><P><B>7/11</B></P><UL><LI>Trying to get pSB1C3-pCas9 (BBa_K1218011) to function
</LI></UL><DL><DD>-Ordered 2 primers pairs so we could make 2 targeting guide RNAs
</DD><DD><UL><LI>These guide RNAs target the Neocin (also Kanamycin) resistance gene in our BW strain.
</LI></UL></DD></DL><UL><LI>Vector Digest
</LI><LI>Digest 1-2ul of pCas9 with BsaI-HF
</LI></UL><DL><DD>-Sample was divided between two lanes
</DD><DD>-Gel extracted digestion but poor yield
</DD></DL><DL><DD>-Redigest with less DNA
</DD><DD>1. pCas9 from above gel
</DD><DD>2. Different sample of pCas9
</DD></DL><DL><DD>-Repeated, doing everything we could to increase gel extraction yield-- no luck
</DD></DL><UL><LI>Phosphorylation
</LI></UL><DL><DD>-phosphorylated primer pairs together
</DD><DD>-1 hr at 37C then 20min at 65C
</DD></DL><UL><LI>Annealing
</LI></UL><DL><DD>-(forgot to add 2.5ul of 1M NaCl)
</DD><DD>-Incubated in Thermocycler for 5 minutes at 95C
</DD><DD>-Tried to decrement temp 0.1C/s until temp reached 21, then jumped to 4C
</DD><DD><UL><LI>Took less than 30 minutes so we think the samples cooled too fast
</LI></UL></DD></DL><P><B>7/12</B></P><UL><LI>Trying to switch the gRNA in the CRISPR
</LI></UL><DL><DD>-Phosphorylation
</DD><DD>-Phosphorylated primer pairs same as 7/11
</DD></DL><DL><DD>-Annealing
</DD><DD>-Remembered to add 2.5ul of 1M NaCl
</DD><DD>-1. Heat block
</DD><DD><UL><LI>5min at 95C-&gt; removed from head to let cool 2 hours
</LI><LI>Problem-&gt; water condenses on lid
</LI></UL></DD></DL><DL><DD>-2. Thermocycler
</DD><DD><UL><LI>Was supposed to cool slowly… not the case
</LI></UL></DD></DL><DL><DD>-Vector Digest
</DD><DD>-Digested pCas9
</DD><DD><UL><LI>1. 60 min at 37, 20 min at 80 (see #4 on gel below)
</LI><LI>2. 90 min at 37, 20 min at 80 (see #5 on gel below)
</LI></UL></DD></DL><DL><DD>-Compared to un cut on a gel…. all bands looked pretty. Also, included gel extractions from 7/11 to test whether DNA was present
</DD></DL><DL><DD>1. Gel extract 1 from 7/11
</DD><DD>2. Gel extrac 2 from 7/11
</DD><DD>3. Uncut pCas9
</DD><DD>4. Digested pCas9 for 60 minutes
</DD><DD>5. Digested pCas9 for 90 minutes
</DD><DD>-Did not gel extract sample
</DD></DL><DL><DD>-Phosphorylation #3
</DD><DD>-Because I did not like my previous annealing attempts, so I decided to start over
</DD></DL><DL><DD>-Annealing
</DD><DD>-Added 2.5ul 1M NaCl
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  95C
</TH><TH> 94.8
</TH><TH> 94.6
</TH><TH> 94.4
</TH><TH> 94.2
</TH><TH> 94.0
</TH><TH> 21
</TH></TR><TR><TD> 5 min
</TD><TD> :20
</TD><TD> :20
</TD><TD> :20
</TD><TD> :20
</TD><TD> :20
</TD><TD> hold
</TD></TR><TR><TD> decrement
</TD><TD> -1
</TD><TD> -1
</TD><TD> -1
</TD><TD> -1
</TD><TD> -1
</TD></TR></TBODY></TABLE><DL><DD>-font cycled 74 times. After each step, the temp would decrement by 1 to produce a consistent cooling rate of 0.2C/20s (about equivalent to letting a heat block cool
</DD><DD>-Diluted 1:10
</DD><DD>-Ligation
</DD><DD>-4 reactions total
</DD><DD>gRNA1- gel extracted pCas9
</DD><DD>gRNA1- non-gel extracted pCas9
</DD><DD>gRNA2- gel extracted pCas9
</DD><DD>gRNA2- non-gel extracted pCas9
</DD></DL><H2><SPAN class="mw-headline" id="Week8">Week8</SPAN></H2><P><B>7/13</B></P><UL><LI>Transformation of 7/12 ligations (attempts to swap gRNAs in the CRISPR)
</LI></UL><DL><DD>-Plated on low Chlor (34ug/mL) and high Chlor (170ug/mL)
</DD></DL><P><B>7/14</B></P><UL><LI>Did get colonies for pCas9 with new gRNAs (some for each primer pair)
</LI></UL><DL><DD>-Colony PCR using VR and Fwd gRNA primer
</DD><DD>-Dream taq
</DD></DL><DL><DD>-Gel had big black spot
</DD><DD>-(Realized this the next day that the imager was set to high exposure so could not see bands)
</DD></DL><DL><DD>-Set up O/N of colonies in despair
</DD></DL><P><B>7/15</B></P><UL><LI>Check colonies for pSB1C3-pCas9 using mini-prepped DNA
</LI></UL><DL><DD>-Mini-prepped O/Ns
</DD><DD>-Repeated PCR using VR and gRNA primer
</DD></DL><DL><DD>-Found some samples with new gRNA
</DD><DD>-Tried to make O/Ns of correct samples….. turns out there was a miscommunication and DNA, not living cells, were added to the liquid cultures…
</DD></DL><UL><LI>First proof of concept: Transformation of pSB1C3-pCas9 containing targeting gRNAs into BW23115 chemically competent cells and 5alphas
</LI></UL><DL><DD>-DNA
</DD><DD>pSB1C3-pCas9(unmodified)--&gt; non-targeting
</DD><DD>pSB1C3-pCas9(gRNA1)--&gt; targeting
</DD><DD>pSB1C3-pCas9(gRNA2) --&gt; targeting
</DD></DL><DL><DD>-Controls
</DD><DD>-5alpha cells. These cells do not contain kanamycin resistance in their genome so should not be targeted
</DD><DD>-pSB1C3-pCas9(unmodified): gRNA does not target anywhere in either E. coli genome so is we expect no targeted killing; hence no death
</DD></DL><DL><DD>-Hypothesis: gRNA1 and gRNA2 are complimentary to the kanamycin resistance gene in the BW23115 strain. Therefore, we expect the samples that contain pCas9 with these targeting gRNAs to target and cleave the resistance gene and cause either direct cell death or a loss of function of the kanamycin resistance gene, resulting in cell death. 
</DD></DL><P><B>7/17</B></P><UL><LI>Results of 7/15 pCas9 transformation experiment
</LI></UL><DL><DD>-5alphas
</DD><DD><UL><LI>No growth for No DNA control on Chlor
</LI><LI>Much growth for non-targeting gRNA and targeting gRNA1
</LI><LI>Half as much growth for targeting gRNA2
</LI></UL></DD></DL><DL><DD>-BW23115
</DD><DD><UL><LI>No growth for No DNA control on Chlor+Kan
</LI><LI>Fewer, but larger colonies for non-targeting sample
</LI><LI>More growth for targeting gRNA1 and gRNA2? large range of sizes
</LI></UL></DD></DL><DL><DD>-Conclusions
</DD><DD><UL><LI>Targeting the Kanamycin resistance gene did not kill the bacteria. 
</LI></UL></DD></DL><DL><DD>-Repeat the transformation with remaining DNA samples
</DD></DL><P><B>7/18</B></P><UL><LI>Made liquid O/Ns of gRNAs to remake pSB1C3-pCas9(gRNA1) and pSB1C3-pCas9(gRNA2)
</LI></UL><P><B>7/19</B></P><UL><LI>Tested overnights from 7/18 pSB1C3-pCas9 with gRNAs 1 and 2
</LI></UL><DL><DD>-Mini-prepped
</DD><DD>-Repeated PCR with DreamTaq
</DD><DD><UL><LI>used VR (reverse primer for iGEM biobricks (amplifies insert from end)) and fwd primer used to make gRNA
</LI></UL></DD><DD><UL><LI>There should only be a product if the fwd primer can anneal to DNA. Product should be about 400bp
</LI></UL></DD></DL><DL><DD>-Found some samples with correct band at 400bp- these were the correct sizes so are likely correct
</DD></DL><H2><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H2><P><B>7/20</B></P><UL><LI>Freeze downs of samples from 7/19 that looked right
</LI></UL><DL><DD>-gRNA1 targets nt 23-53 of KanR gene      
</DD><DD>-gRNA2 targets ny 551-582 of KanR gene
</DD></DL><P><B>7/21</B></P><UL><LI>Sent samples for sequencing
</LI></UL><DL><DD>-pSB1C3-pCas9(gRNA1)
</DD><DD> 5'- cgatagaaggcgatgcgctgcgaatcggga -3'
</DD></DL><DL><DD>-pSB1C3-pCas9(gRNA2)
</DD><DD> 5'- tcgagcacagctgcgcaaggaacgcccgtc -3'
</DD></DL><P><B>7/22</B></P><UL><LI>Pricked colony from pSB1C3-pCas9 plate
</LI></UL><DL><DD>-Mini-prepped DNA to isolate DNA
</DD></DL><P><B>7/23</B></P><UL><LI>Sequencing samples from 7/21 were lost in the mail. Resent samples
</LI><LI>Primers came in for new guide RNAs
</LI></UL><DL><DD>-Note: Upon review of the protocol, we realized that the first round of primers we bought were incorrectly made, resulting in a 1nt shift away from the PAM site. The NGG motif requirement was not met so binding was absent. We bought new primers to correct this problem
</DD><DD>-gRNA3: Targets same place as gRNA1 but is corrected to bind the correct distance from the PAM site
</DD><DD> 5'- gatagaaggcgatgcgctgcgaatcgggag -3'
</DD></DL><DL><DD>-gRNA4: Targets same place as gRNA2 but is corrected to bind the correct distance from the PAM site
</DD><DD> 5'- cgagcacagctgcgcaaggaacgcccgtcg -3'
</DD></DL><DL><DD>-unmodified: spacer with the BsaI sites that allow for replacement of spacer
</DD><DD> 5'- tgagaccagtctcggaagctcaaaggtctc -3'
</DD></DL><UL><LI>To swap gRNAs on pCas9 (gRNA3 and gRNA4)
</LI></UL><DL><DD>-Phosphorylate primers        1hr @ 37C, 20 min @ 65C
</DD><DD>-Annealed primers
</DD><DD><UL><LI>Used the thermocycler like on 7/12
</LI></UL></DD></DL><DL><DD>-Digest vector DNA (pCas9 DNA with BsaI)
</DD><DD>-Did not get complete digestion
</DD></DL><P><B>7/24</B></P><UL><LI>To swap gRNAs on pSB1C3-pCas9 (gRNA3 and gRNA4)
</LI></UL><DL><DD>-Used digested pSB1C3-pCas9 from 7/12. Same one used the last time we swapped gRNAs
</DD><DD>-Ligated the above pSB1C3-pCas9 to annealed oligos
</DD></DL><P><B>7/25</B></P><UL><LI>Transformed ligation from 7/24 (gRNA1/gRNA2 into pSB1C3-pCas9) into 5alpha cells
</LI></UL><P><B>7/26</B></P><UL><LI>No growth for 7/25 transformation
</LI></UL><DL><DD>-Retried transformation
</DD><DD>-pCas9 was digested a week earlier- possible source of problem
</DD></DL><UL><LI>Retried making pCas9 with gRNA3 and gRNA4
</LI></UL><DL><DD>-Digestion (so we have freshly cut samples)
</DD><DD>pSB1C3-pCas9 with BsaI
</DD></DL><DL><DD>-gel extracted linearlized pCas9 (smaller band)
</DD></DL><DL><DD>-Ligation (10hrs at 16C, 10min at 80C)
</DD><DD><UL><LI>pSB1C3-pCas9 + gRNA3
</LI><LI>pSB1C3-pCas9 + gRNA4
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Week_10">Week 10</SPAN></H2><P><B>7/27</B></P><UL><LI>Transformation results
</LI></UL><DL><DD>-No growth for no DNA controls--&gt; on Chlor
</DD><DD>-No growth for gRNA4 --&gt; on Chlor
</DD><DD>-3 colonies for gRNA3 --&gt; on Chlor
</DD></DL><UL><LI>Picked the 3 colonies for gRNA3 for O/Ns
</LI></UL><P><B>7/28</B></P><UL><LI>All O/Ns grew
</LI><LI>To test the overnights for gRNA3
</LI></UL><DL><DD>-Mini-prep samples -&gt; good yields
</DD><DD>-PCR with Dream-Taq
</DD><DD>-Diluted all samples 1:100
</DD></DL><DL><DD>1. pSB1C3-pCas9(unmod.)
</DD><DD>2. Sample 1
</DD><DD>3. Sample 2
</DD><DD>4. Sample 3
</DD><DD>5. No DNA control
</DD><DD>-All samples contained a band of the expected &lt;400bp, including the Stanford-Brown Cas9 unmodified and no DNA control
</DD></DL><UL><LI>Transformations
</LI></UL><DL><DD>-pSB1C3-pCas9(gRNA3)
</DD><DD>-SB1C3-pCas9(gRNA4)
</DD><DD>-Positive control with pSB1C3-pCas9(unmod.)
</DD><DD>-No DNA control
</DD></DL><P><B>7/30</B></P><UL><LI>Sent samples for sequencing
</LI></UL><DL><DD>-pSB1C3(gRNA3) (5)                    VR
</DD><DD>-pSB1C3(gRNA3)  (6)                    VR
</DD><DD>-pSB1C3(gRNA3)  (7)                    VR
</DD></DL><P><B>7/31</B></P><UL><LI>Sequencing results came back positive
</LI></UL><DL><DD>-Had correct gRNA3 insert
</DD></DL><UL><LI>Transformation experiments with gNRAs: Repeat of 7/15 experiment
</LI></UL><DL><DD>-Samples
</DD><DD>-Scramble- unmodified pCas9
</DD><DD>-non-targeting- gRNA1
</DD><DD>-targeting- gRNA3
</DD><DD>-no DNA control
</DD></DL><DL><DD>-Added 578ng each
</DD><DD>-Transformed into 5alpha and BW (F+, +KanR)
</DD></DL><P><B>8/1</B></P><UL><LI>Transformation results from 7/31
</LI></UL><DL><DD>-Contamination for 5alpha samples
</DD><DD>-BW cells on Chlor
</DD><DD>-Hundreds of colonies for all (unmodified, gRNA1, gRNA3)
</DD><DD>-No colonies on no DNA control
</DD></DL><DL><DD>-BW cells on Chlor + Kan
</DD><DD>-Hundreds of colonies for unmodified and gRNA1
</DD><DD>-3-4 colonies for gRNA3
</DD><DD>-No colonies for no DNA control
</DD></DL><DL><DD>-Notes:
</DD><DD><UL><LI>Unexpected that for gRNA3, cells died on Chlor+Kan but survived at comparable amounts on Chlor
</LI></UL></DD><DD>-Suggests that targeting of an essential gene
</DD><DD>-Later, cells on gRNA3 plate looked sickly, maybe delayed killing?
</DD></DL><H2><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H2><P><B>8/4</B></P><UL><LI>Spotted contamination in SOC-&gt; remade SOC
</LI><LI>Got new 5alpha (from NEB) from Mike to make comp cells (C2987 by NEB)
</LI><LI>Transformation experiment (~578ng)
</LI></UL><DL><DD>-(barrier tips, new SOC, BW cells (5/30))
</DD><DD>-unmodified pCas9
</DD><DD>-gRNA1
</DD><DD>-gRNA3
</DD><DD>-No DNA control
</DD></DL><UL><LI>Test pCas9 for functional Cas9 rather than dCas9
</LI></UL><DL><DD>-Digestion to confirm
</DD><DD>-pCas9            EcoRV        no extraction- just image on gel
</DD></DL><UL><LI>To add gRNA4 to pSB1C3-pCas9
</LI></UL><DL><DD>-Digestion
</DD><DD><UL><LI>pCas9            BsaI        extracted entire plasmid
</LI><LI>insert gel
</LI></UL></DD></DL><DL><DD>-Ligation (10 hrs at 16C, 10 min at 80C)
</DD><DD><UL><LI>pSB1C3-pCas9 + gRNA4
</LI><LI>No DNA control to test for contamination in reagents
</LI></UL></DD></DL><P><B>8/5</B></P><UL><LI>Transformation into ER2738 (7/18), 5ul of ligation products
</LI></UL><DL><DD>1. Litmus28ibb-pCas9gRNA3
</DD><DD>2. pSB1C3-Cas9 (gRNA4)
</DD><DD>3. No DNA in ligation mix
</DD><DD>No DNA control
</DD><DD>Variations on protocol
</DD><DD>-Added 200ul SOC to 1.7ul epi-tubes, incubated in small tubes
</DD><DD>-Incubated at ~200rpm
</DD><DD>-Added only 5ul Ligation products
</DD></DL><UL><LI>Make chem comp 5alpha cells
</LI></UL><DL><DD>-Got new 5alpha cells from Mike (chem comp from NEB) on 8/4
</DD><DD>-grew O/N in 5mL LB
</DD><DD>-Made fresh .8M MgCl2 and .2M CaCl2
</DD></DL><UL><LI>Transformation results from 8/4 [not targeted killing experiment]
</LI></UL><DL><DD>-No growth when only SOC was plate
</DD><DD><UL><LI>SOC is not contaminated with cells ChlorR or ChlorR+kanR
</LI></UL></DD></DL><DL><DD>-No growth for No DNA controls on A+T or C+T
</DD><DD><UL><LI>Negative Ligation control
</LI><LI>Negative transformation control
</LI></UL></DD></DL><DL><DD>-No growth for M13g-3C5 or LCR ligations (8/3)
</DD><DD>-Lawn for amilCP on pSB1C3 transformation
</DD></DL><UL><LI>Targeted killing experiment Transformation results
</LI></UL><DL><DD>-Images of plates can be found in the results section
</DD><DD>-Colony count on chlor (34ug/mL) and kan (25ug/mL) - kanR is essential
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH> 1:1
</TH><TH> 1:10
</TH></TR><TR><TD> unmodified
</TD><TD> lawn
</TD><TD> 2760
</TD></TR><TR><TD> gRNA1
</TD><TD> lawn
</TD><TD> 2316
</TD></TR><TR><TD> gRNA3
</TD><TD> 23
</TD><TD> 1
</TD></TR><TR><TD> no DNA control
</TD><TD> 0
</TD><TD> NA
</TD></TR></TBODY></TABLE><UL><LI>Colony count on chlor (170ug/mL) - kanR is non-essential
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH> 1:1
</TH><TH> 1:10
</TH></TR><TR><TD> unmodified
</TD><TD> 1920
</TD><TD> 226
</TD></TR><TR><TD> gRNA1
</TD><TD> 960
</TD><TD> 71
</TD></TR><TR><TD> gRNA3
</TD><TD> 8
</TD><TD> 1
</TD></TR><TR><TD> no DNA control
</TD><TD> 0
</TD><TD> NA
</TD></TR></TBODY></TABLE><DL><DD>-Shows successful killing when KanR is essential and when KanR is nonessential
</DD></DL><P><B>8/6</B></P><UL><LI>Transformation results from 8/5
</LI></UL><DL><DD>-No colonies for pCas9(gRNA4) on Tet+reduced Chlor
</DD><DD>-BLASTed sequence against MG1655. The 10nts at the 3’ end of spacer match a sequence in MG1655 that is adjacet to a PAM site. We think gRNA is targeting the production strain.
</DD><DD><UL><LI>To compare, the spacer of gRNA3 matches 8nts on its 3’ end.
</LI></UL></DD></DL><DL><DD>-No growth for no DNA control of ligation or transformation
</DD><DD>-Ligation reactants are not cause of recent contamination on Amp or Chlor
</DD></DL><H2><SPAN class="mw-headline" id="Week_13.2F14">Week 13/14</SPAN></H2><P><B>8/12</B></P><UL><LI>Targeted Killing Transformation
</LI></UL><DL><DD>-Transform into 3alpha (7/28) and BWF’ (5/30)
</DD><DD>-DNA
</DD><DD><UL><LI>pSB1C3-pCas9(unmod.)
</LI><LI>pSB1C3-pCas9(gRNA1)
</LI><LI>pSB1C3-pCas9(gRNA3)
</LI></UL></DD></DL><P><B>8/16</B>
Results of 8/12 transformation of BWF’ with gRNA(unmod, 1, or 3) targeted killing
</P><P>No growth on Chlor (34ug/mL) or Chlor (34ug/mL)+Kan(25ug/mL)
</P><TABLE class="wikitable"><TBODY><TR><TH> Chlor+Kan
</TH><TH> Chor
</TH></TR><TR><TD> Scramble
</TD><TD> Healthy lawn
</TD><TD> Healthy lawn
</TD></TR><TR><TD> gRNA1
</TD><TD> Healthy lawn (thinner that scramble)
</TD><TD> Healthy lawn (thinner that scramble)
</TD></TR><TR><TD> gRNA3
</TD><TD> Shadowy, flat lawn, 42 healthy colonies
</TD><TD> Shadowy, flat lawn, 42 healthy colonies
</TD></TR></TBODY></TABLE><P><B>8/19</B></P><P>Sequence survivor colonies from transformation experiments. These are the colonies from gRNA3 samples.
</P><DIV class="newfooter"><DIV id="newfooterbox"><DIV class="footertext"><UL><LI><A href="https://igem.org/About">About</A></LI><LI><A href="https://igem.org/Contact">Contact</A></LI><LI><A href="https://igem.org/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Main_Page">  igem.org</A></LI><LI><A href="http://parts.igem.org/Main_Page"> Registry</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://2014.igem.org/Requirements"> Requirements</A></LI><LI><A href="https://2014.igem.org/Giant_Jamboree"> Giant Jamboree</A></LI><LI><A href="https://2014.igem.org/Meetups"> Meetups</A></LI><LI><A href="https://2014.igem.org/Calendar_of_Events">Calendar </A></LI><LI><A href="https://2014.igem.org/Partner_Offers">Partner Offers</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Login">Login</A></LI><LI><A href="https://2014.igem.org/Special:Upload">Upload Files</A></LI><LI><A href="https://2014.igem.org/Special:RecentChanges"> Recent Changes</A></LI><LI><A href="https://2014.igem.org/Special:SpecialPages"> Special Pages</A></LI><LI><A href="https://2014.igem.org/Special:Search"> Search</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Press_Kit">Press Kit</A></LI><LI><A href="https://igem.org/Newsletters"> Newsletter</A></LI><LI><A href="https://igem.org/Alumni">AlumniGEM</A></LI></UL></DIV><DIV class="footerextra">
powered by <A href="http://www.mediawiki.org/" style="link-style:none;">MediaWiki</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Constitutive">http://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Constitutive</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:CU-Boulder/Notebook/CC9_Team/Constitutive" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:CU-Boulder/Notebook/CC9_Team/Constitutive" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Constitutive&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Constitutive&amp;oldid=312958" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>