<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_CU-Boulder_Notebook_CC9_Team_Induced"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:CU-Boulder/Notebook/CC9_Team/Induced">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:CU-Boulder/Notebook/CC9_Team/Induced&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Induced&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Induced&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:CU-Boulder/Notebook/CC9_Team/Induced" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:CU-Boulder/Notebook/CC9 Team/Induced</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><HEADER id="topNav"><NAV id="navContainer"><UL class="navLinks"><LI class="navSingleLink"><A href="https://2014.igem.org/Team:CU-Boulder">HOME</A></LI><LI><A>PROJECT</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Getting Started</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Motivation">Motivation</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Modeling">Modeling</A></LI></UL></NAV></HEADER></DIV></DIV></DIV><LI><A>ACHIEVEMENTS</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Our Team Achievements</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Biobricks">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Medal">Medal Requirements</A></LI><LI><A>NOTEBOOK</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">At the Bench</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Phage_Team">Phage Delivery</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Induced">Induced CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Constitutive">Constitutive CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9Phage_Team">Phage CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Measurement">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Protocols">Protocols</A></LI><LI><A>HUMAN PERSPECTIVE</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Reaching Out</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Safety">Ethics &amp; Safety</A></LI><LI><A>TEAM</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">MI6</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Acknowledgments">Acknowledgments</A></LI><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_1"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_3"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week_4"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week_5"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 5</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Week_6"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 6</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week_7"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week 7</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Week_8"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Week 8</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Week_9"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Week 9</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Week_10"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Week 10</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Week_11"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Week 11</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Week_12"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Week 12</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H2><P><B>5/12</B></P><DL><DD>Made overnight cultures of….
</DD><DD>-Qi gRNA  P44251   (AmpR)  
</DD><DD><UL><LI>NOTE: plate from Andrew is labeled P44231
</LI></UL></DD></DL><DL><DD>-Qi wr cas9 P44250 (AmpR)
</DD><DD>-BW 23115 Kan casA  (KanR)
</DD></DL><P><B>5/13</B></P><P>Glycerol stocks 
-	Stocks made from overnight cultures (3 from each)
-	500 uL bacteria + 500 uL 40% glycerol stock
</P><UL><LI>Glass tube containing #P44250 broke and was discarded – overnight repeated
</LI></UL><P>Mini Prep
Protocol done per manufacturer’s instructions with following exceptions:
-Done with P44250 on 5/13/14
-Done with P44251 on 5/14/14
-Step 1: Cultures split into 3 separate 1.5 mL aliquots. Supernatant was removed from all three, pellets resuspended in a small amount of EB Buffer and moved into single tube. Centrifuged again and supernatant was removed before proceeding to Step 2. 
</P><P><B>5/14</B></P><TABLE class="wikitable"><TBODY><TR><TH>  Nano Drop of Mini Prep
</TH></TR><TR><TD> Sample
</TD><TD> 260/280 Ratio
</TD><TD> 260/230 Ratio
</TD><TD> DNA Concentration
</TD></TR><TR><TD> P44250
</TD><TD> 1.94
</TD><TD> 2.17
</TD><TD> 49.4 ng/uL
</TD></TR><TR><TD> P44251
</TD><TD> 1.94
</TD><TD> 2.47
</TD><TD> 52.7 ng/uL
</TD></TR></TBODY></TABLE><UL><LI>Samples were blanked with EB Buffer
</LI></UL><P>Restriction Digest
-	NEBcutter V2.0 used to check  digestion cut sites (done in silico)
-	Digest reaction performed as shown below (from NEB)
-	Incubated for 1 hour at 37°C
-	No heat inactivation performed
</P><TABLE class="wikitable"><TBODY><TR><TH> P44250
</TH><TH> P44251
</TH></TR><TR><TD> Cutsmart Buffer
</TD><TD> 5.0 uL
</TD><TD> 5.0 uL
</TD></TR><TR><TD> XbaI
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD></TR><TR><TD> XhoI
</TD><TD> 1.0 uL
</TD><TD> --
</TD></TR><TR><TD> SpeI
</TD><TD> 1.0 uL
</TD><TD> --
</TD></TR><TR><TD> EcoRI-HF
</TD><TD> --
</TD><TD> 1.0 uL
</TD></TR><TR><TD> DNA (200 ng)
</TD><TD> 4.0 uL
</TD><TD> 4.0 uL
</TD></TR><TR><TD> MQ H2O
</TD><TD> 38.0 uL
</TD><TD> 39.0 uL
</TD></TR><TR><TD> Total Volume
</TD><TD> 50.0 uL
</TD><TD> 50.0 uL
</TD></TR></TBODY></TABLE><P>Agarose Gel Electrophoresis
-1% agarose gel prepared 
-2 uL loading dye added to 18 uL of each sample for loading into each lane
-1Kb+ DNA Ladder used
-Overnight staining in EtBr done to better visualize faint bands
</P><P>Lane 1: 2-Log DNA Ladder
Lane 2: P44250 digest
Lane 3: P44251 digest
</P><P>P50 plasmid should have 3 segments after digestion: 4221 bp, 1740 bp, 970 bp
P51 plasmid should have 2 segments after digestion: 540 bp, 2045 bp
</P><DL><DD>-This is demonstrated in the gel image shown below.
</DD></DL><P>
Plate Streaking
</P><DL><DD>-Used to check the TetR gene in the P44250 plasmid
</DD><DD>-2 mL LB containing 50 mg/ uL Tet added to plate and allowed to soak in
</DD><DD>-Plates left at room temperature overnight
</DD><DD>-LB only plate – negative control
</DD><DD>-Amp plate
</DD><DD>-Amp + Tet 
</DD><DD>-Tet 
</DD></DL><P><B>5/15</B></P><P>Dry Lab 
</P><DL><DD>-gRNA designed for Kanamycin 
</DD><DD>-Plasmid design – considering Gibson assembly and cPEC 
</DD></DL><DL><DD>Analysis of streak plates from 5/14/15
</DD><DD>Results: (See Below)	
</DD><DD>-LB: Slightly less growth than Amp plate
</DD><DD>-Amp plate: Normal Growth
</DD><DD>-Amp + Tet:  No bacteria present
</DD><DD>-Tet: No bacteria present
</DD><DD>-Results were inconsistent with expectations. There is some concern regarding the potency of the tetracycline used. Further research revealed that the TetR gene is a repressor for TetO which part of the bi-directional promoter for cas9. Thus TetR is inhibited with presence of tetracycline. We will use TetR as a regulator for cas9. 
</DD></DL><P><B>5/16</B></P><P>Dry Lab 
</P><DL><DD>-gRNA designs modified 
</DD><DD>-Primer design for Gibson assembly of our plasmid
</DD></DL><H2><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H2><P><B>5/19</B></P><P>Transformation:
</P><DL><DD>1 uL P44250 (cas9) plasmid added to K12 E. coli strain
</DD><DD>Heat shock done according to protocol (45 sec @ 42°C)
</DD><DD>Cultures incubated at 37°C for 1.5 hours
</DD><DD>4 Plates with K12 and the plasmid were plated by streaking as follows:
</DD><DD>-Amp, Amp (u plate with 1:10 dilution, u plate 1:1), Amp + Kan, Amp + Kan (u plate with 1:10 dilution, u plate 1:1)
</DD></DL><P><B>5/20</B></P><P>Plates from 5/19/14 (shown below).
</P><DL><DD>Two individual colonies were taken and grown overnight from the Amp plate (marked with “x” in upper left plate of above image). There was no growth on the two plates with Amp and Kan (K12 strain does not contain Kan resistance). 
</DD></DL><P><B>5/22</B></P><P>Transformation: (Chemically competent BWF+ cells with P44250 plasmid)
</P><DL><DD>1 uL P44250 (cas9) plasmid added to conjugated BWF+ strain
</DD><DD>Transformation protocol followed with following exceptions:
</DD><DD>-200 uL SOC added to each sample 
</DD><DD>-Incubated for 2 hours
</DD><DD>-1:10 dilution made (10 uL into 90 uL SOC)
</DD><DD>-Cultures plated on Amp plates, one with full concentration and another with the 1:10 dilution
</DD></DL><P><B>5/23</B></P><P>Arrival of primers previously ordered
</P><DL><DD>All were reconstituted in MQ H2O to 100 uM concentration
</DD></DL><P>Results of plates from 5/22/14:
</P><DL><DD>-BFW+ w/ P50 plasmid: growth at full and 1:10 dilution (Top left of image)
</DD><DD>-BFW+ w/ P50 plasmid full strength: No growth (Bottom left of image)
</DD><DD>-BW only: full strength showed minor growth (Bottom right of image)
</DD><DD>-BW only: No growth at full and 1:10 dilution (Top right of image)
</DD><DD>(*This was an unexpected result – it may be that these two plates were mislabeled)
</DD></DL><P>
Single colonies taken from the BFW+ w/ P50 plasmid plate in the 1:10 dilution and placed in 2 uL liquid culture overnight with the following antibiotics:
</P><DL><DD>-AMP
</DD><DD>-Kan
</DD><DD>-AMP + Kan
</DD><DD>-Tet
</DD></DL><P><B>5/24</B></P><P>As shown the image below, overnight cultures from 5/23/14 are all cloudy demonstrating bacterial growth.
</P><P>
PCR Reaction – Used to amplify out cas9 from P44250 plasmid and CMR from PSB1C3 plasmid
</P><DL><DD><UL><LI>DNA samples diluted so each contains 0.5 ng/uL
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Cas9
</TH><TH> CMR
</TH></TR><TR><TD> Q5 Master Mix volume
</TD><TD> 25 uL
</TD><TD> 25 uL
</TD></TR><TR><TD> dcas9 Primer volume
</TD><TD> 2.5 uL
</TD><TD> --
</TD></TR><TR><TD> dCMR Primer volume
</TD><TD> --
</TD><TD> 2.5 uL
</TD></TR><TR><TD> DNA added
</TD><TD> 2.5 uL
</TD><TD> 2.5 uL
</TD></TR><TR><TD> Q5 Polymerase volume
</TD><TD> 0.5 uL
</TD><TD> 0.5 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 20 uL
</TD><TD> 20 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 50.0 uL
</TD><TD> 50.0 uL
</TD></TR></TBODY></TABLE><P>PCR Settings: 30 cycles as follows
</P><DL><DD>-98 °C Denature 30 sec
</DD><DD>-70 °C Anneal 30 sec
</DD><DD>-72 °C Extension 2.5 min
</DD></DL><H2><SPAN class="mw-headline" id="Week_3">Week 3</SPAN></H2><P><B>5/25</B></P><P>Gel electrophoresis performed on PCR reactions from 5/24/14. 1% gel made from 0.514 g agarose with 50 mL 1X TAE and 5 uL EtBr. 
</P><DL><DD>Lane 1: 5 uL 2-Log DNA Ladder
</DD><DD>Lane 2: CMR sample* (~2 Kb size)
</DD><DD>Lane 3: Cas9 sample* (~5 Kb size)
</DD><DD><UL><LI>Each contained 8 uL reaction sample + 2 uL loading dye 
</LI></UL></DD></DL><P>Gel Results: 
</P><DL><DD>Lane 1 – Ladder
</DD><DD>Lane 2 – Although band of correct size visible for CMR sample (~2Kb), there may be contamination
</DD><DD>Lane 3 – No bands visible – This is perhaps due to having the extension time too short in the PCR reaction (Advisors suggest 60 sec per 1000 bases). 
</DD></DL><P><B>5/27</B></P><P>Repeat of PCR using same protocol as listed on 5/24/14 except for different times on the thermocycler as follows:
</P><TABLE class="wikitable"><TBODY><TR><TH> Cas9
</TH><TH> CMR
</TH></TR><TR><TD> Initial Denature
</TD><TD> 98°C/2 min
</TD><TD> 98°C/2 min
</TD></TR><TR><TD> Denature
</TD><TD> 98°C/30 sec
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 69°C/30 sec
</TD><TD> 72°C/30 sec
</TD></TR><TR><TD> Extend
</TD><TD> 69°C/ 2.5 min
</TD><TD> 72°C/2.5 min
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C/6 min
</TD><TD> 72°C/5 min
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Thermocycler containing CMR stopped after 1 cycle and held at 98°C for ~3hours. Machine was reset and run. This sample may no longer be viable.
</LI></UL></DD></DL><P><B>5/28/14</B></P><P>Dry lab
</P><DL><DD>primer designs
</DD><DD>Repeat of PCR for the CMR reaction, same as described on 5/24/14 and shown below.
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> CMR
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 25 uL
</TD></TR><TR><TD> dCMR Primer
</TD><TD> 2.5 uL
</TD></TR><TR><TD> Q5 Polymerase
</TD><TD> 0.5 uL
</TD></TR><TR><TD> CMR DNA
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 20 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 50.0 uL
</TD></TR></TBODY></TABLE><P><B>5/29</B></P><P>Gel electrophoresis: 1% gel ran at 110V for ~40 minutes.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH><TH> Amount
</TH><TH> Loading dye amount
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD><TD> 0.5  uL
</TD><TD> --
</TD></TR><TR><TD> 2
</TD><TD> Cas9
</TD><TD> 5.0  uL
</TD><TD> 1.0  uL
</TD></TR><TR><TD> 3
</TD><TD> CMR 1 (from 5/24)
</TD><TD> 5.0  uL
</TD><TD> 1.0  uL
</TD></TR><TR><TD> 4
</TD><TD> CMR 2 (from 5/27)
</TD><TD> 5.0  uL
</TD><TD> 1.0  uL
</TD></TR><TR><TD> 5
</TD><TD> CMR 3 (from 5/28)
</TD><TD> 5.0  uL
</TD><TD> 1.0  uL
</TD></TR></TBODY></TABLE><P>Results from Gel (shown below): 
</P><DL><DD>Lane 2: Cas9 band visible of appropriate size (approximately 5,000 bp)
</DD><DD>Lane 3: CMR 1- slight smearing
</DD><DD>Lane 4: CMR 2 – smearing present
</DD><DD>Lane 5: CMR 3- clean, no smearing
</DD><DD>-All CMR lanes have band present of correct size (approx. 2,000 bp)</DD></DL><P>Gel extraction performed on cas9, CMR1, and CMR2 lanes using E.Z.N.A. Gel extraction kit following  the protocol. Following amounts:
</P><DL><DD>-Cas9: 110 uL buffer with 0.1103 g sample
</DD><DD>-CMR1: 53 uL buffer with 0.0533 g sample
</DD><DD>-CMR2: 90 uL buffer with 0.0904 g sample
</DD></DL><P>Nanodrops performed to determine DNA concentration and purity with the following results:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH><TH> **ng/ uL
</TH><TH> **260/280
</TH><TH> **260/230
</TH></TR><TR><TD> Gel-Cas9
</TD><TD> 7.3
</TD><TD> 2.20
</TD><TD> 0.57
</TD><TD> 5.2
</TD><TD> 2.06
</TD><TD> 0.52
</TD></TR><TR><TD> Cas9
</TD><TD> 483.2
</TD><TD> 1.78
</TD><TD> 0.72
</TD><TD> 36.3
</TD><TD> 1.85
</TD><TD> 1.72
</TD></TR><TR><TD> Gel-CMR1
</TD><TD> 9.6
</TD><TD> 2.02
</TD><TD> 0.63
</TD><TD> 6.5
</TD><TD> 2.35
</TD><TD> 0.90
</TD></TR><TR><TD> Gel-CMR2
</TD><TD> 8.3
</TD><TD> 2.06
</TD><TD> 0.83
</TD><TD> 5.0
</TD><TD> 2.21
</TD><TD> 0.69
</TD></TR><TR><TD> CMR2
</TD><TD> 618.8
</TD><TD> 1.66
</TD><TD> 0.61
</TD><TD> 44.5
</TD><TD> 1.89
</TD><TD> 1.41
</TD></TR><TR><TD> CMR 3
</TD><TD> 535.8
</TD><TD> 1.73
</TD><TD> 0.61
</TD><TD> 17.8
</TD><TD> 1.89
</TD><TD> 1.31
</TD></TR><TR><TD> CMR1
</TD><TD> 516.6
</TD><TD> 1.69
</TD><TD> 0.61
</TD><TD> 68.5
</TD><TD> 1.84
</TD><TD> 1.74
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Trial 2 completed on 5/30/14 shown in these columns
</LI></UL></DD></DL><DL><DD>Values for the ratios should be between 2.0 and 2.2 for a pure sample. The values for Trial #1 are much too low. In order to increase sample purity, a DNA cleanup is performed following the protocol given in the E.Z.N.A. gel extraction kit in amounts shown below.
</DD><DD>-25 uL samples
</DD><DD>-25 uL binding buffer
</DD><DD>-Washed 2x with 700 uL SPW
</DD><DD>-30 uL elution buffer added, allowed to rest 4 minutes
</DD><DD>-Centrifuged on max for 2 min
</DD></DL><P><B>5/30</B></P><P>After the DNA cleanup protocol – nanodrop performed again with results listed in above table as Trial #2
</P><P>Restriction Digestion performed on Cas9 and CMR1 with DPN1 (to remove any methylation present) using standard digestion protocol with the following amounts:
</P><DL><DD>26 uL sample
</DD><DD>1.0	uL DPN1 (100 fold dilution)
</DD><DD>3.0 uL 10X CutSmart Buffer 
</DD></DL><P>Gibson Assembly performed to join pSB1C3 backbone with cas9 PCR product. Protocol for Gibson was followed for the reaction.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Gibson Master Mix
</TD><TD> 10 uL
</TD></TR><TR><TD> Cas9
</TD><TD> 1.6 uL (50 ng)
</TD></TR><TR><TD> Cmr
</TD><TD> 2.4 uL (143 ng)
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 6 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 20.0 uL
</TD></TR></TBODY></TABLE><P>Product was transformed into competent cells following the Gibson transformation protocol that included a positive control. 
</P><H2><SPAN class="mw-headline" id="Week_4">Week 4</SPAN></H2><P><B>6/2</B></P><P>Grow Gibson colonies in 6 mL LB media + 30 uL chloroamp
</P><P>Transform Gibson colonies following protocol and with amounts as shown:
</P><DL><DD>-BWF+ w/cas9: 15 uL into 135 uL SOC
</DD><DD>-BW w/cas9: 15 uL into 135 uL SOC
</DD><DD>-BWF+ (no cas9): 30 uL into 270 uL SOC
</DD><DD>-BW (no cas9): 30 uL into 270 uL SOC
</DD></DL><P>90 uL of the above four reactions plated as follows and incubated:
</P><DL><DD>-BWF+ w/cas9: Amp + Kan
</DD><DD>-BW w/cas9: Amp + Kan 
</DD><DD>-BWF+ (no cas9): (2 Plates) Amp + Kan and Kan only
</DD><DD>-BW (no cas9): (2 Plates) Amp + Kan and Kan only
</DD><DD>-All four reactions also plated together on Quad plate with Chloramp
</DD><DD>-BWF+ (Not transformed) on Kan + Tet (plated both at full strength and as 1:10 dilution)
</DD></DL><P><B>6/3</B></P><P>Analysis of Plates from 6/2/14: Only one colony was present on Gibson plate (BWF+ w/cas9)
This colony was taken and suspended in O/N culture (LB and Choroamp)
MiniPrep performed on the O/N cultures
</P><DL><DD><UL><LI>Renamed the product GAcasC (Gibson Assembly w/cas and Chloroamp)
</LI></UL></DD></DL><P>NanoPrep on GAcasC
</P><TABLE class="wikitable"><TBODY><TR><TH>  ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 201.0
</TD><TD> 1.88
</TD><TD> 2.24
</TD></TR></TBODY></TABLE><P>Digest performed according to protocol on GAcasC and P50 plasmid using EcoRV and BamHI as follows: 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> GAcasC
</TH><TH> P50
</TH></TR><TR><TD> Buffer 3
</TD><TD> 2.5 uL
</TD><TD> 2.5 uL
</TD></TR><TR><TD> EcoRV
</TD><TD> 0.5 uL
</TD><TD> 0.5 uL
</TD></TR><TR><TD> BamHI
</TD><TD> 0.5 uL
</TD><TD> 0.5 uL
</TD></TR><TR><TD> BSA (1:4 dilution)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD></TR><TR><TD> DNA (250 ng)
</TD><TD> 1.25 uL
</TD><TD> 5.0 uL
</TD></TR><TR><TD> MQ H2O
</TD><TD> 19.25 uL
</TD><TD> 15.5 uL
</TD></TR><TR><TD> Total Volume
</TD><TD> 25.0 uL
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><P>1% Gel made for electrophoresis. Each sample had 1 uL of loading dye added, 5 uL of DNA ladder and 5 uL of each sample was loaded. Gel ran 120V for ~1 hour. 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Well
</TH><TH> Loaded
</TH></TR><TR><TD> 1
</TD><TD> DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> GAcasC digest
</TD></TR><TR><TD> 3
</TD><TD> CmR1 Dpn1 product
</TD></TR><TR><TD> 4
</TD><TD> Cas9 Dpn1 product
</TD></TR><TR><TD> 5
</TD><TD> P50 digestion product
</TD></TR></TBODY></TABLE><P>Gel Results:
</P><DL><DD>Lane 2 should have multiple products which are not present in the gel. The other lanes are as expected. 
</DD></DL><P>
PCR using Q5 is used to amplify the P51 CRISPR backbone using primers CRE1 &amp; CRE4 provided by Andrew. Primers were diluted 1:10 from 100 mM stock
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 25 uL
</TD></TR><TR><TD> CRE1 Primer
</TD><TD> 2.5 uL
</TD></TR><TR><TD> CRE4 Primer
</TD><TD> 2.5 uL
</TD></TR><TR><TD> P51 DNA
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Q5 Polymerase
</TD><TD> 0.5 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 17.5 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 50.0 uL
</TD></TR></TBODY></TABLE><P>Thermocycler set as follows for the reaction:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Initial Denature
</TH><TH> 98°C/30 sec
</TH></TR><TR><TD> Denature
</TD><TD> 98°C/20 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 60°C/20 sec
</TD></TR><TR><TD> Extend
</TD><TD> 72°C/3 min
</TD></TR></TBODY></TABLE><P>Restriction Digestion performed on P51 with DPN1 (to remove any methylation present) using standard digestion protocol with the following amounts:
</P><DL><DD>26 uL sample
</DD><DD>1.0uL DPN1 (100 fold dilution)
</DD><DD>3.0 uL 10X CutSmart Buffer 
</DD></DL><P>PCR performed to dimerize each of the gRNA primers previously designed (729, 666, 756, SC1, and SC2) following the cPEC protocol and using the following amounts:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 25.0 uL
</TD></TR><TR><TD> Primer
</TD><TD> 1.7 uL
</TD></TR><TR><TD> Primer
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Q5 Polymerase
</TD><TD> 0. 5 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 19.5 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 50.0 uL
</TD></TR></TBODY></TABLE><P>Nanodrop performed on some of these products (due to timing) to check for purity and concentration (results below). It is assumed that the other samples have high enough DNA levels to proceed.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 729
</TD><TD> 405.5
</TD><TD> 1.78
</TD><TD> 0.67
</TD></TR><TR><TD> SC2
</TD><TD> 393.2
</TD><TD> 1.80
</TD><TD> 0.67
</TD></TR><TR><TD> CasC
</TD><TD> 455.2
</TD><TD> 1.5
</TD><TD> 0.60
</TD></TR></TBODY></TABLE><P>cPEC performed to combine the P51 amplification and the each of the dimerized gRNAs separately following the cPEC protocol and using the following amounts:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 12.5 uL
</TD></TR><TR><TD> Insert (our primer dimers)
</TD><TD> 1.7 uL
</TD></TR><TR><TD> Template DNA
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Q5 Polymerase
</TD><TD> 0.25 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 10.05 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><P>Repeat of Gibson to combine cas9 and CmR (pSB1C3 backbone) with cas9 PCR product. Gibson protocol was followed for the reaction and included a positive control (same as reaction completed on 5/30/14).
</P><TABLE class="wikitable"><TBODY><TR><TH>Reactants
</TH><TH>Volume
</TH></TR><TR><TD> Gibson Master Mix
</TD><TD> 10 uL
</TD></TR><TR><TD> Cas9
</TD><TD> 1.6 uL (50 ng)
</TD></TR><TR><TD> Cmr
</TD><TD> 2.4 uL (143 ng)
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 6 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 20.0 uL
</TD></TR></TBODY></TABLE><P>Product was transformed into competent cells following the Gibson transformation protocol that included a positive control. 
</P><P>O/N cell cultures made from plates in order to perform a Tet titration tomorrow. 2 tubes of each were made by taking a single colony and mixing with 5 mL of LB and antibiotics as listed below before incubation. 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Antibiotic
</TH></TR><TR><TD> BWF+ w/ cas
</TD><TD> 5 uL Kan + 10 uL Amp
</TD></TR><TR><TD> BW w/ cas
</TD><TD> 5 uL Kan + 10 uL Amp
</TD></TR><TR><TD> BWF+
</TD><TD> 5 uL Kan
</TD></TR><TR><TD> BW
</TD><TD> 5 uL Kan
</TD></TR><TR><TD> BWF+ (not transformed)
</TD><TD> 5 uL Kan + 10 uL Tet
</TD></TR></TBODY></TABLE><P><B>6/4</B></P><P>Tet titration on O/N cultures was performed by using different concentrations (from 0-1000 ng/mL) of anhydrotetracycline plus a chloramp control.
</P><DL><DD><UL><LI>Cancelled as cell growth had already neared plateau values 
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> 0 ng/mL
</TH><TH> 0.5 ng/mL
</TH><TH> 5 ng/mL
</TH><TH> 20 ng/mL
</TH><TH> 50 ng/mL
</TH><TH> 250 ng/mL
</TH><TH> 1000 ng/mL
</TH><TH> Chloro
</TH></TR><TR><TD> BW
</TD><TD> 2.38
</TD><TD> 2.401
</TD><TD> 2.389
</TD><TD> 2.382
</TD><TD> 2.386
</TD><TD> 2.398
</TD><TD> 2.373
</TD><TD> 2.379
</TD></TR><TR><TD> BWF+
</TD><TD> 2.379
</TD><TD> 2.376
</TD><TD> 2.428
</TD><TD> 2.390
</TD><TD> 2.378
</TD><TD> 2.365
</TD><TD> 2.378
</TD><TD> 2.386
</TD></TR><TR><TD> BWcas
</TD><TD> 2.391
</TD><TD> 2.356
</TD><TD> 2.367
</TD><TD> 2.355
</TD><TD> 2.366
</TD><TD> 2.330
</TD><TD> 2.362
</TD><TD> 2.382
</TD></TR><TR><TD> BWF+ cas
</TD><TD> 2.438
</TD><TD> 2.447
</TD><TD> 2.448
</TD><TD> 2.434
</TD><TD> 2.436
</TD><TD> 2.415
</TD><TD> 2.419
</TD><TD> 2.415
</TD></TR><TR><TD> BWF+ Tet
</TD><TD> 2.135
</TD><TD> 2.146
</TD><TD> 2.141
</TD><TD> 2.152
</TD><TD> 2.128
</TD><TD> 2.127
</TD><TD> 2.134
</TD><TD> 2.134
</TD></TR></TBODY></TABLE><P>New O/N cell cultures made to do the titration on 6/5/14. Same amounts as shown above were used except only one of each was made.  No cells added to O/N culture on 6/4/14, that is why procedure was repeated.
</P><P><B>6/5</B></P><P>Since the Gibson Assembly does not seem to be working, cPEC was performed to combine the CmR backbone with Cas9. Three different cycles were run in an attempt to optimize the protocol; the following amounts were used in all three reactions:
</P><TABLE class="wikitable"><TBODY><TR><TH> Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 10.0 uL
</TD></TR><TR><TD> Cas9 DNA
</TD><TD> 5.0 uL
</TD></TR><TR><TD> CmR DNA
</TD><TD> 5.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 0.00 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 20.0 uL
</TD></TR></TBODY></TABLE><P>Thermocycler was varied for each of the three cPEC reactions as listed below:
</P><TABLE class="wikitable"><TBODY><TR><TH> cPEC1
</TH><TH> cPEC2
</TH><TH> cPEC3
</TH></TR><TR><TD> Initial Denature
</TD><TD> 98°C/30 sec
</TD><TD> 98°C/30 sec
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Denature
</TD><TD> 98°C/10 sec
</TD><TD> 98°C/20 sec
</TD><TD> 98°C/10 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 56°C/30 sec
</TD><TD> 60°C/20 sec
</TD><TD> 66°C/30 sec
</TD></TR><TR><TD> Extend
</TD><TD> 72°C/90 sec
</TD><TD> 72°C/30 sec
</TD><TD> 72°C/90 sec
</TD></TR><TR><TD> # of cycles
</TD><TD> 10
</TD><TD> 15
</TD><TD> 10
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C/10 min
</TD><TD> 72°C/10 min
</TD><TD> 72°C/10 min
</TD></TR></TBODY></TABLE><P>Nanodrop done on the three cPEC casC reactions with results listed below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> CasC1
</TD><TD> 489.3
</TD><TD> 1.80
</TD><TD> 0.72
</TD></TR><TR><TD> CasC2
</TD><TD> 577.7
</TD><TD> 1.76
</TD><TD> 0.59
</TD></TR><TR><TD> CasC3
</TD><TD> 542.5
</TD><TD> 1.80
</TD><TD> 0.74
</TD></TR></TBODY></TABLE><P>DNA cleanup done on these three casC cPEC reactions, followed by another nanodrop to examine purity with the results below. 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> CasC1
</TD><TD> 13.4
</TD><TD> 1.99
</TD><TD> 1.61
</TD></TR><TR><TD> CasC2
</TD><TD> 11.1
</TD><TD> 2.02
</TD><TD> 1.15
</TD></TR><TR><TD> CasC3*
</TD><TD> 9.9
</TD><TD> 1.84
</TD><TD> 1.21
</TD></TR></TBODY></TABLE><P>1% Gel made for electrophoresis. Each sample had 1 uL of loading dye added, 5 uL of DNA ladder and 5 uL of each sample was loaded. Gel ran 122V for ~50 min.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Well
</TH><TH> Loaded
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> cPEC1
</TD></TR><TR><TD> 3
</TD><TD> cPEC2
</TD></TR><TR><TD> 4
</TD><TD> cPEC3
</TD></TR><TR><TD> 5
</TD><TD> GA Control
</TD></TR><TR><TD> 6
</TD><TD> GA Positive Control
</TD></TR><TR><TD> 7
</TD><TD> CasC
</TD></TR><TR><TD> 8
</TD><TD> H2O Control
</TD></TR></TBODY></TABLE><P>
Titration experiment was done to determine the optimal concentration of anhydrous tetracycline (ATC) to be used and to determine if the cas9 protein was lethal to the cells. It was decided to use BWF+ w/cas9 and w/o cas9 for this experiment. Starting ODs were taken of a 1:10 dilution from liquid cultures grown overnight. This was used to calculate the amount to be added to 50 mL LB to start the experiment with an OD close to 0.05. ODs were taken periodically as the cells grew as shown below.
</P><TABLE class="wikitable"><TBODY><TR><TH> Time 0
</TH><TH> Time 1
</TH><TH> Time 2
</TH><TH> Time 3
</TH><TH> Time 4
</TH><TH> Time 5*
</TH><TH> Time 6
</TH></TR><TR><TD> BWF+
</TD><TD> 0.041
</TD><TD> 0.047
</TD><TD> 0.114
</TD><TD> 0.283
</TD><TD> 0.597
</TD><TD> 0.065
</TD><TD> 0.171
</TD></TR><TR><TD> BWF+ w/cas9
</TD><TD> 0.035
</TD><TD> 0.046
</TD><TD> 0.110
</TD><TD> 0.262
</TD><TD> 0.530
</TD><TD> 0.059
</TD><TD> 0.154
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>At Time 5, the cultures were again diluted down. The experiment was cancelled after Time 6 due to time constraints and issues with the plates.
</LI></UL></DD></DL><P><B>6/6</B></P><P>Plates with the Gibson Assembly products have shown very slow growth. Seven colonies are visible on the Gibson plate (shown with positive control plate below).
</P><P>Above left: Gibson Assembly plate with 7 colonies. Above Right: Gibson Assembly positive control plate.
</P><H2><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H2><P><B>6/9</B></P><P>Mini-preps were done following manufacturer's protocol except samples were centrifuged 4 minutes at the end (step 9). Ten samples were done that included 7 GA colonies, 1 GA control (labeled GA8), P50, and P51. All were subjected to a nanodrop and a restriction digestion with BamHI and EcoRV was performed on all the samples all except P51.  The nanodrop results are shown below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> GA1
</TD><TD> 80.5
</TD><TD> 1.88
</TD><TD> 2.04
</TD></TR><TR><TD> GA2
</TD><TD> 87.8
</TD><TD> 1.90
</TD><TD> 2.14
</TD></TR><TR><TD> GA3
</TD><TD> 53.4
</TD><TD> 1.89
</TD><TD> 1.72
</TD></TR><TR><TD> GA4
</TD><TD> 67.1
</TD><TD> 1.88
</TD><TD> 2.02
</TD></TR><TR><TD> GA5
</TD><TD> 49.5
</TD><TD> 1.92
</TD><TD> 1.86
</TD></TR><TR><TD> GA6
</TD><TD> 84.2
</TD><TD> 1.90
</TD><TD> 2.07
</TD></TR><TR><TD> GA7
</TD><TD> 54.6
</TD><TD> 1.94
</TD><TD> 2.08
</TD></TR><TR><TD> GA8
</TD><TD> 59.8
</TD><TD> 1.86
</TD><TD> 1.60
</TD></TR><TR><TD> P50
</TD><TD> 91.3
</TD><TD> 1.88
</TD><TD> 2.12
</TD></TR><TR><TD> P51
</TD><TD> 65.8
</TD><TD> 1.88
</TD><TD> 2.03
</TD></TR></TBODY></TABLE><P>Samples from the digestion reaction were run on a 1% agarose gel (shown below). From these results, it appears that none of the Gibson Assembly colonies contain the correct products.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> GA1
</TD></TR><TR><TD> 3
</TD><TD> GA2
</TD></TR><TR><TD> 4
</TD><TD> GA3
</TD></TR><TR><TD> 5
</TD><TD> GA4
</TD></TR><TR><TD> 6
</TD><TD> GA5
</TD></TR><TR><TD> 7
</TD><TD> GA6
</TD></TR><TR><TD> 8
</TD><TD> GA7
</TD></TR><TR><TD> 9
</TD><TD> GA8
</TD></TR><TR><TD> 10
</TD><TD> P50 Digest
</TD></TR><TR><TD> 11
</TD><TD> SB1C3
</TD></TR><TR><TD> 12
</TD><TD> Undigested GA1
</TD></TR><TR><TD> 13
</TD><TD> Undigested GA2
</TD></TR><TR><TD> 14
</TD><TD> Undigested GA3
</TD></TR><TR><TD> 15
</TD><TD> Undigested P50
</TD></TR></TBODY></TABLE><P> 
Working stock of ATC was made and filter sterilized which was used to remake the ATC plates for the titration experiment. Working stock of ATC was made and filter sterilized which was used to remake the ATC plates for the titration experiment. 20 plates were made with u containing Kan and u containing Amp and Kan. The plates were made with increasing amounts of ATC.
</P><P>gRNAs from cPEC were transformed and plated according to protocol with controls.
</P><P> 
AmilCP restriction digestion was performed on the original and PCR amplified samples and the products run on a 1% agarose gel as shown below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> PCR AmilCP
</TH><TH> Original AmilCP
</TH></TR><TR><TD> Buffer 3
</TD><TD> 1.25 uL
</TD><TD> 1.25 uL
</TD></TR><TR><TD> BxsI
</TD><TD> 1.0 uL
</TD><TD> 2.5 uL
</TD></TR><TR><TD> DNA (125 ng)
</TD><TD> 4.0 uL
</TD><TD> 1.25 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 6.25 uL
</TD><TD> 7.5 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 12.5 uL
</TD><TD> 12.5 uL
</TD></TR></TBODY></TABLE><P><B>6/10</B></P><P>cPEC was performed to join the gRNA (as primer dimers) with the p51 plasmid and AmilCP.
</P><P>First, primers had to be dimerized. Master mixes of all gRNA primers were created and aliquots made by adding 5 uL of each to 40 uL nuclease free H2O (N.F. H2O). Two separate primer mixes were made (one for p51 and one for AmilCP) using 5 uL of each forward and reverse primer dimer that were added to 40 uL of N.F. H2O to produce a dilution (1:1:8). 5 uL of each primer mix was added to 25 uL of the Q5 master mix and 20 uL N.F. H2O to make a 50 uL total reaction for each sample.Products were checked by gel electrophoresis (shown below).
</P><P>Next, a gel purification of a PCR P51 product was performed (to select for linear DNA). This product was digested with DpnI. A PCR cleanup was made of other P51 product that was not gel extracted. All of these steps were done following their recommended protocols.
</P><P>Finally, cPEC reaction was done to combine the gel extracted P51 product with each of the gRNAs using the following amounts. The thermocycler set the same as on 6/3/14:
</P><TABLE class="wikitable"><TBODY><TR><TH>   Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 25.0 uL
</TD></TR><TR><TD> Primer Mix
</TD><TD> 2.5 uL
</TD></TR><TR><TD> DNA (1:10 dilution)
</TD><TD> 1.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 25.5 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 54.0 uL
</TD></TR></TBODY></TABLE><P>Following the cPEC reactions, Nanodrops were taken to assess the concentration and purity of the samples:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> P51
</TD><TD> 416.9
</TD><TD> 1.82
</TD><TD> 0.67
</TD></TR><TR><TD> AmilCP
</TD><TD> 430.5
</TD><TD> 1.82
</TD><TD> 0.67
</TD></TR><TR><TD> 158
</TD><TD> 472.2
</TD><TD> 1.80
</TD><TD> 0.73
</TD></TR><TR><TD> 756
</TD><TD> 479.6
</TD><TD> 1.78
</TD><TD> 0.73
</TD></TR><TR><TD> 666
</TD><TD> 475.8
</TD><TD> 1.80
</TD><TD> 0.73
</TD></TR><TR><TD> 729
</TD><TD> 485.4
</TD><TD> 1.79
</TD><TD> 0.75
</TD></TR><TR><TD> SC1
</TD><TD> 464.6
</TD><TD> 1.80
</TD><TD> 0.73
</TD></TR><TR><TD> SC2
</TD><TD> 475.1
</TD><TD> 1.80
</TD><TD> 0.74
</TD></TR></TBODY></TABLE><P><B>Titration Experiment:</B> Done to determine the optimal concentration of anhydrous tetracycline (ATC) to be used and to determine if the cas9 protein was lethal to the cells. The plan is to make 50 mL of BWF+ w/cas9 and BWF+ w/o cas9 with a starting OD600 of 0.05. These will be placed in an incubator and allowed to grow until reaching 0.5. They will then be diluted again to 0.05 (in 50 mL). Serial dilutions will be made and plated on the prepared plates which will be incubated overnight.
</P><P>Starting ODs were taken of a 1:10 dilution from liquid cultures grown overnight. This was used to calculate the amount to be added to 50 mL LB to start the experiment with an OD close to 0.05. ODs were taken periodically as the cells grew as shown below.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> OD
</TH><TH> Amount added to 50 mL
</TH></TR><TR><TD> BWF+
</TD><TD> 0.544
</TD><TD> 460 uL
</TD></TR><TR><TD> BWF+ w/cas9
</TD><TD> 0.556
</TD><TD> 450 uL
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH>  Time
</TH><TH> BWF+
</TH><TH> BWF+ w/cas9
</TH></TR><TR><TD> T0
</TD><TD> 0.059
</TD><TD> 0.058
</TD></TR><TR><TD> T1
</TD><TD> 0.081
</TD><TD> 0.078
</TD></TR><TR><TD> T2
</TD><TD> 0.180
</TD><TD> 0.168
</TD></TR><TR><TD> T3
</TD><TD> 0.291
</TD><TD> 0.272
</TD></TR><TR><TD> T4
</TD><TD> 0.398
</TD><TD> 0.366
</TD></TR><TR><TD> T5
</TD><TD> 0.538
</TD><TD> 0.489
</TD></TR><TR><TD> T6*
</TD><TD> 0.968
</TD><TD> 0.874
</TD></TR><TR><TD> T7
</TD><TD> 0.165
</TD><TD> 0.153
</TD></TR><TR><TD> T8
</TD><TD> 0.414
</TD><TD> 0.396
</TD></TR><TR><TD> T9†
</TD><TD> 0.561
</TD><TD> 0.490
</TD></TR><TR><TD> T10
</TD><TD> 0.057
</TD><TD> 0.056
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>At T6, the cultures were again diluted down (5.17 mL BWF+, 5.72 mL BWF+ w/cas9 into 50 mL).
</LI></UL></DD><DD>†After T9, 4.898 mL BWF+ and 4.278 mL BWF+ w/ cas9 were again diluted into 50 mL LB. A final OD was taken to ensure that the final OD was close to 0.05 before the cells were plated.
</DD></DL><P>Cells were plated on the previously prepared plates containing increasing concentrations of ATC as previously described (6/9/14) and placed in 37°C incubator overnight.
</P><P><B>6/11</B></P><P><B>Titration Experiment: Results</B></P><P>As seen from the images in the table below, the cas9 protein inhibits growth of the cells even prior to removal of repression by the TetR repressor, demonstrating some leaky expression. Increasing ATC concentrations results in increased cas9 expression in the cells as expected.
</P><P>
Nanodrop of gel extracted (GE) and mini-prep (MP) products as shown below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> GE P51
</TD><TD> 13.7
</TD><TD> 1.94
</TD><TD> 1.18
</TD></TR><TR><TD> GE AmilCP PCR
</TD><TD> 27.3
</TD><TD> 1.81
</TD><TD> 1.19
</TD></TR><TR><TD> GE cas9 PCR
</TD><TD> 7.8
</TD><TD> 1.98
</TD><TD> 0.62
</TD></TR><TR><TD> AmilCP DpnI digest
</TD><TD> 644.8
</TD><TD> 1.6
</TD><TD> 0.65
</TD></TR><TR><TD> P50 PCR
</TD><TD> 493.9
</TD><TD> 1.8
</TD><TD> 0.73
</TD></TR><TR><TD> GA1 MP
</TD><TD> 107.9
</TD><TD> 1.87
</TD><TD> 1.99
</TD></TR><TR><TD> GA2 MP
</TD><TD> 144.9
</TD><TD> 1.88
</TD><TD> 2.11
</TD></TR><TR><TD> GA3 MP
</TD><TD> 168.3
</TD><TD> 1.92
</TD><TD> 2.29
</TD></TR><TR><TD> GA4 MP
</TD><TD> 131.1
</TD><TD> 1.91
</TD><TD> 1.93
</TD></TR><TR><TD> GA5 MP
</TD><TD> 310.6
</TD><TD> 1.88
</TD><TD> 2.34
</TD></TR><TR><TD> 729-1
</TD><TD> 125.3
</TD><TD> 1.88
</TD><TD> 2.11
</TD></TR><TR><TD> 729-2
</TD><TD> 178.3
</TD><TD> 1.90
</TD><TD> 2.17
</TD></TR><TR><TD> 729-3
</TD><TD> 172.6
</TD><TD> 1.84
</TD><TD> 1.81
</TD></TR><TR><TD> 756-1
</TD><TD> 420.7
</TD><TD> 1.87
</TD><TD> 2.25
</TD></TR><TR><TD> 756-2
</TD><TD> 106.8
</TD><TD> 1.89
</TD><TD> 1.99
</TD></TR></TBODY></TABLE><P><B>6/12</B></P><P>Transformations done of all 6 gRNA cPEC reactions, P50 (+control), and P51 PCR (-control)
Gel extraction of Cas9 done to protocol followed by a nanodrop (results below).
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> Cas9
</TD><TD> 5.5
</TD><TD> 2.12
</TD><TD> 0.59
</TD></TR></TBODY></TABLE><P>P50 PCR amplification performed using the following amounts:
</P><DL><DD>-25 uL Master Mix
</DD><DD>-2.5 uL of forward primer (10 uM)
</DD><DD>-2.5 uL of reverse primer (10 uM)
</DD><DD>-1.0 uL DNA (1:50 dilution)
</DD><DD>-19.0 uL NF H2O
</DD></DL><P>Two samples prepared with different thermocycler settings as follows:
</P><TABLE class="wikitable"><TBODY><TR><TH> 1st Round
</TH><TH> 2nd Round
</TH></TR><TR><TD> Initial Denature
</TD><TD> 98°C/30 sec
</TD><TD> 98°C/2 min
</TD></TR><TR><TD> Denature
</TD><TD> 98°C/30 sec
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 69°C/30 sec
</TD><TD> 68°C/30 sec
</TD></TR><TR><TD> Extend
</TD><TD> 72°C/3.5 min
</TD><TD> 72°C/5.5 min
</TD></TR><TR><TD> # of cycles
</TD><TD> 34
</TD><TD> 30
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C/2 min
</TD><TD> 72°C/5 min
</TD></TR></TBODY></TABLE><P>Overnight cultures prepared on GA1-GA5, 729-1, 729-2, 729-3, and 759-1, 759-2
Colonies taken from SC1 plate for overnight cultures (SC-1, SC-2) with 5 mL LB, 10 uL Amp (50 ng/mL)
</P><P><B>6/13</B></P><P>Mini-prep done on O/N cultures SC1-1 and SC1-2 followed by a nanodrop. Remainder of culture was spun down into a pellet and frozen.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> SC1-1
</TD><TD> 30.7
</TD><TD> 2.02
</TD><TD> 2.06
</TD></TR><TR><TD> SC1-2
</TD><TD> 32.5
</TD><TD> 1.94
</TD><TD> 1.48
</TD></TR></TBODY></TABLE><P>Restriction Digestion done following manufacturer’s protocols on the samples as listed below. All were incubated at 37°C for 1 hour followed by 20 minute heat inactivation at 80°C.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Buffer 3
</TH><TH> 10X BSA
</TH><TH> Enzyme 1
</TH><TH> Enzyme 2
</TH><TH> Enzyme Mix*
</TH><TH> DNA (200 ng)
</TH><TH> MQ H2O
</TH><TH> Total Volume
</TH></TR><TR><TD> 756-1
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> 0.5 uL EagI
</TD><TD> 0.5 uL XbaI
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 14.0 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> 756-2
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> 0.5 uL EagI
</TD><TD> 0.5 uL XbaI
</TD><TD> --
</TD><TD> 1.87 uL
</TD><TD> 13.13 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> GA1
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.85 uL
</TD><TD> 13.15 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> GA2
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.38 uL
</TD><TD> 13.62 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> GA3
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.19 uL
</TD><TD> 13.81 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> GA4
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.53 uL
</TD><TD> 13.47 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> GA5
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 0.64 uL
</TD><TD> 14.36 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> P51
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 4.0 uL
</TD><TD> 11.0 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> Cas9
</TD><TD> 2.0 uL
</TD><TD> 2.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 13.0 uL
</TD><TD> 20.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Enzyme Mix is a 1:1:23 dilution of EcoRI and BamHI
</LI></UL></DD></DL><P>Digested products are run on a 1% agarose gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> 756-1
</TD></TR><TR><TD> 3
</TD><TD> 756-2
</TD></TR><TR><TD> 4
</TD><TD> P51 Digest
</TD></TR><TR><TD> 5
</TD><TD> Sb1C3
</TD></TR><TR><TD> 6
</TD><TD> GA1
</TD></TR><TR><TD> 7
</TD><TD> GA2
</TD></TR><TR><TD> 8
</TD><TD> GA3
</TD></TR><TR><TD> 9
</TD><TD> GA4
</TD></TR><TR><TD> 10
</TD><TD> GA5
</TD></TR><TR><TD> 11
</TD><TD> Cas9 Digest
</TD></TR></TBODY></TABLE><P>
From the above gel, it does not appear that are products are as expected. A repeat digest is done of GA4 and P50. These new products were run on a 1% agarose gel with reagents as comparisons (5 uL sample plus 1 uL loading dye for each) as shown below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> GA4
</TD></TR><TR><TD> 3
</TD><TD> P50 Digest
</TD></TR><TR><TD> 4
</TD><TD> Sb1C3
</TD></TR><TR><TD> 5
</TD><TD> CmR1
</TD></TR><TR><TD> 6
</TD><TD> Cas9 PCR
</TD></TR><TR><TD> 7
</TD><TD> Cas9 GE
</TD></TR><TR><TD> 8
</TD><TD> GA4 MP
</TD></TR></TBODY></TABLE><P><B>6/14</B></P><P> 
Gibson Assembly performed both with and without 2% DMSO. Each 20 uL reaction contained 10 uL Gibson Master Mix with the following as listed:  
</P><DL><DD>Reaction #1            	
</DD><DD>-6.5 uL GE Cas9
</DD><DD>-3.5 uL GE AmilCP
</DD></DL><DL><DD>Reaction #2
</DD><DD>-1 uL P50 PCR Product (1:10)
</DD><DD>-5 uL GE AmilCP
</DD><DD>-4 uL MQ H2O
</DD></DL><DL><DD>Reaction #3
</DD><DD>-1 uL P50 PCR Product (1:10)
</DD><DD>-5 uL GE AmilCP
</DD><DD>-4 uL 1:10 dilution of 2% DMSO
</DD></DL><P>Bacterial Transformation: Done according to protocol
</P><DL><DD>-GA1-GA3 diluted 1:4 transformed with 2 uL (100 uL each)
</DD><DD>-BWF+ w/Cas9 from S.B. (25 uL full strength and 25 uL 1:10 dilution)
</DD><DD>-BWF+ (25 uL full strength and 25 uL 1:10 dilution)
</DD><DD>-NEB competent cells with P51 (50 uL)
</DD><DD>-Competent cells with P51 (50 uL) for comparison
</DD></DL><H2><SPAN class="mw-headline" id="Week_6">Week 6</SPAN></H2><P><B>6/15</B></P><P>GA colonies picked for overnight cultures from 2 plates (11 total- 3 from Plate 1, 8 from plate 2), placed in 5 mL LB with 170 ng/ uL Amp.
</P><P><B>6/16</B></P><P>Mini-prep was done to protocol on all overnight cultures followed by a nanodrop with results listed below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 1-1
</TD><TD> 17.5
</TD><TD> 2.08
</TD><TD> 1.94
</TD></TR><TR><TD> 1-2
</TD><TD> 54.1
</TD><TD> 1.93
</TD><TD> 1.96
</TD></TR><TR><TD> 1-3
</TD><TD> 30.4
</TD><TD> 2.00
</TD><TD> 1.78
</TD></TR><TR><TD> 2-1
</TD><TD> 45.8
</TD><TD> 1.93
</TD><TD> 1.90
</TD></TR><TR><TD> 2-2
</TD><TD> 45.4
</TD><TD> 1.92
</TD><TD> 1.89
</TD></TR><TR><TD> 2-3
</TD><TD> 46.9
</TD><TD> 1.93
</TD><TD> 1.97
</TD></TR><TR><TD> 2-4
</TD><TD> 53.5
</TD><TD> 1.96
</TD><TD> 2.01
</TD></TR><TR><TD> 2-5
</TD><TD> 53.5
</TD><TD> 1.94
</TD><TD> 2.04
</TD></TR><TR><TD> 2-6
</TD><TD> 41.9
</TD><TD> 1.95
</TD><TD> 1.84
</TD></TR><TR><TD> 2-7
</TD><TD> 59.3
</TD><TD> 1.94
</TD><TD> 1.99
</TD></TR><TR><TD> 2-8
</TD><TD> 46.5
</TD><TD> 1.94
</TD><TD> 1.98
</TD></TR></TBODY></TABLE><P>Next, a restriction digest was performed on all the samples listed above. EcoRV was used in a 1:100 dilution with Buffer D (both from Promega). Samples were incubated for 1 hour at 37°C and heat inactivation performed for 5 minutes at 80°C. The following amounts were used for all the samples except 1-1 (DNA increased to 6.0 uL and MQ H2O was not added).
</P><DL><DD>-1.0 uL Buffer D
</DD><DD>-3.0 uL Enzyme
</DD><DD>-3.0 uL sample DNA
</DD><DD>-3.0 uL MQ H2O
</DD><DD>-Total Reaction Volume = 10 uL
</DD></DL><P>Digested products are run on a 1% agarose gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> 1-1
</TD></TR><TR><TD> 3
</TD><TD> 1-2
</TD></TR><TR><TD> 4
</TD><TD> 1-3
</TD></TR><TR><TD> 5
</TD><TD> 2-1
</TD></TR><TR><TD> 6
</TD><TD> 2-2
</TD></TR><TR><TD> 7
</TD><TD> 2-3
</TD></TR><TR><TD> 8
</TD><TD> 2-4
</TD></TR><TR><TD> 9
</TD><TD> 2-5
</TD></TR><TR><TD> 10
</TD><TD> 2-6
</TD></TR><TR><TD> 11
</TD><TD> 2-7
</TD></TR><TR><TD> 12
</TD><TD> 2-8
</TD></TR></TBODY></TABLE><P>Repeated digestion (protocol above) on select samples to run again for more definition on gel. Samples are run on a 1% agarose gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> 1-1
</TD></TR><TR><TD> 3
</TD><TD> 1-2
</TD></TR><TR><TD> 4
</TD><TD> 2-1
</TD></TR><TR><TD> 5
</TD><TD> sb1c3
</TD></TR><TR><TD> 6
</TD><TD> Digested sb1c3
</TD></TR><TR><TD> 7
</TD><TD> Digested p50
</TD></TR><TR><TD> 8
</TD><TD> 2-Log DNA Ladder
</TD></TR></TBODY></TABLE><P>Bacterial Transformation: Done according to protocol
</P><DL><DD>-7 CPEC products including p51 as the negative control (10 uL  1:4 dilution) with 40 uL  DH5u
</DD><DD>-BWF+ w (40uL)/ S.B. cas9 (1 uL) 
</DD><DD>-BWF+ (1 uL)
</DD></DL><P>Gibson Assembly products 1-1 thru 1-8 and 2-1 thru 2-3 pelleted down and frozen  
</P><P><B>6/17</B></P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Buffer
</TH><TH> Enzyme 1
</TH><TH> EnzymeMix*
</TH><TH> DNA (200 ng)
</TH><TH> MQ H2O
</TH><TH> Total Volume
</TH></TR><TR><TD> GA x 1-1
</TD><TD> 1.0 uL cutsmart
</TD><TD> 3.0 uL XbaI (1:50)
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> GA x 1-2
</TD><TD> 1.0 uL cutsmart
</TD><TD> 3.0 uL XbaI (1:50)
</TD><TD> --
</TD><TD>     3.0  uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> GA x 2-1
</TD><TD> 1.0 uL cutsmart
</TD><TD> 3.0 uL XbaI (1:50)
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> p50 linear
</TD><TD> 1.0 uL cutsmart
</TD><TD> 3.0 uL SepI (1:50)
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> GA H 1-1
</TD><TD> 1.0 uL Fast green digest
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> GA H 1-2
</TD><TD> 1.0 uL Fast green digest
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> GA H 2-1
</TD><TD> 1.0 uL Fast green digest
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> p50
</TD><TD> 1.0  uL Fast green digest
</TD><TD> --
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 3.0 uL
</TD><TD> 10.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Enzyme mix: 1:1:23 dilution of xhoI and HindIII
</LI></UL></DD></DL><P>Digested products are run on a 1% agarose gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> GA x 1-1
</TD></TR><TR><TD> 3
</TD><TD> GA x 1-2
</TD></TR><TR><TD> 4
</TD><TD> GA x 2-1
</TD></TR><TR><TD> 5
</TD><TD> p50
</TD></TR><TR><TD> 6
</TD><TD> sb1c3
</TD></TR><TR><TD> 7
</TD><TD> GA H 1-1
</TD></TR><TR><TD> 8
</TD><TD> GA H 1-2
</TD></TR><TR><TD> 9
</TD><TD> GA H 2-1
</TD></TR><TR><TD> 10
</TD><TD> p50
</TD></TR><TR><TD> 11
</TD><TD> 2-Log DNA Ladder
</TD></TR></TBODY></TABLE><P><B>6/18</B></P><P>Colony PCR is used to rapidly screen colonies from cPEC reactions. A colony is pricked using a pipette tip, avoiding scooping up any LB agar since this inhibits the PCR reaction. There are ~10^9 cells are in each colony - just barely touching the colony and transferring a tiny fraction of it is more than sufficient for this reaction. This tip is rubbed into the PCR tube . Tips  were then ejected into  3ml LB+AMP  for O/N culture.The following was added to each tube to make a 25 uL reaction.
</P><DL><DD>-12.5 uL 2X Master Mix
</DD><DD>-1.25 uL of forward primer (10 uM)
</DD><DD>-1.25 uL of reverse primer (10 uM)
</DD><DD>-10.0 uL MQ H2O
</DD></DL><P>For this reaction, the thermocycler was programmed as follows (note the longer time to initially lyse the cells and denature the DNA).
</P><TABLE class="wikitable"><TBODY><TR><TH>  Initial Denature
</TH><TH> 98°C/10 min
</TH></TR><TR><TD> Denature
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 65°C/30 sec
</TD></TR><TR><TD> Extend
</TD><TD> 72°C/30 sec
</TD></TR><TR><TD> # of cycles
</TD><TD> 35
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C/5 min
</TD></TR></TBODY></TABLE><P><B>6/19</B></P><P>Glycerol stocks done to protocol for two colonies from each gRNA that were indicated positive by colony PCR results. 
</P><H2><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H2><P><B>6/23 </B></P><P>Chemically competent cells of SBcas9 • BWF+. Done according to protocol.
</P><P><B>6/25</B></P><P>Mini prep done to protocol on CPEC products. 
</P><P>Nanodrops were taken to assess the concentration and purity of the samples:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 2H
</TD><TD> 65.2
</TD><TD> 1.88
</TD><TD> 2.21
</TD></TR><TR><TD>      3E
</TD><TD> 56.3
</TD><TD> 1.91
</TD><TD> 2.01
</TD></TR><TR><TD> 4E
</TD><TD> 38.9
</TD><TD> 1.93
</TD><TD> 2.22
</TD></TR><TR><TD> 5B
</TD><TD> 54.1
</TD><TD> 1.86
</TD><TD> 1.54
</TD></TR><TR><TD> 6B
</TD><TD> 69.4
</TD><TD> 1.88
</TD><TD> 2.12
</TD></TR><TR><TD> 7B
</TD><TD> 66.6
</TD><TD> 1.82
</TD><TD> 1.56
</TD></TR><TR><TD> 8B
</TD><TD> 69.7
</TD><TD> 1.83
</TD><TD> 1.75
</TD></TR><TR><TD> 9B
</TD><TD> 61.1
</TD><TD> 1.89
</TD><TD> 1.81
</TD></TR><TR><TD> 10C
</TD><TD> 52.2
</TD><TD> 1.86
</TD><TD> 1.65
</TD></TR><TR><TD> 11B
</TD><TD> 43.9
</TD><TD> 1.86
</TD><TD> 1.33
</TD></TR><TR><TD> 12B
</TD><TD> 57.5
</TD><TD> 1.83
</TD><TD> 1.53
</TD></TR></TBODY></TABLE><P>Sequencing 12 CPEC products: 12.5 uL DNA + 2.5 uL  10um primer (CRE22)
</P><TABLE class="wikitable"><TBODY><TR><TH> CPEC product
</TH></TR><TR><TD> 1
</TD><TD> 1A
</TD></TR><TR><TD> 2
</TD><TD> 1C
</TD></TR><TR><TD> 3
</TD><TD> 2B
</TD></TR><TR><TD> 4
</TD><TD> 2C
</TD></TR><TR><TD> 5
</TD><TD> 3B
</TD></TR><TR><TD> 6
</TD><TD> 3H
</TD></TR><TR><TD> 7
</TD><TD> 4C
</TD></TR><TR><TD> 8
</TD><TD> 4D
</TD></TR><TR><TD> 9
</TD><TD> 5E
</TD></TR><TR><TD> 10
</TD><TD> 5F
</TD></TR><TR><TD> 11
</TD><TD> 6C
</TD></TR><TR><TD> 12
</TD><TD> 6F
</TD></TR></TBODY></TABLE><P><B>6/26</B></P><P>Sequencing: Total volume 0f 15 uL
</P><TABLE class="wikitable"><TBODY><TR><TH> CPEC Product
</TH><TH> DNA uL
</TH><TH> Primer CRE22
</TH><TH> MQ H2O
</TH></TR><TR><TD> 1
</TD><TD> 2B
</TD><TD> 12.5uL
</TD><TD> 2.5 uL
</TD><TD> --
</TD></TR><TR><TD> 2
</TD><TD> 2C
</TD><TD> 12.0uL
</TD><TD> 2.5uL
</TD><TD> 0.5uL
</TD></TR><TR><TD> 3
</TD><TD> 3B
</TD><TD> 10.3uL
</TD><TD> 2.5uL
</TD><TD> 2.2uL
</TD></TR><TR><TD> 4
</TD><TD> 3H
</TD><TD> 11uL
</TD><TD> 2.5uL
</TD><TD> 1.5uL
</TD></TR><TR><TD> 5
</TD><TD> 4E
</TD><TD> 12.5uL
</TD><TD> 2.5uL
</TD><TD> --
</TD></TR><TR><TD> 6
</TD><TD> 5E
</TD><TD> 11.3uL
</TD><TD> 2.5uL
</TD><TD> 1.2uL
</TD></TR><TR><TD> 7
</TD><TD> 11B
</TD><TD> 12.5uL
</TD><TD> 2.5uL
</TD><TD> --
</TD></TR><TR><TD> 8
</TD><TD> 2H
</TD><TD> 8uL
</TD><TD> 2.5uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 9
</TD><TD> 6B
</TD><TD> 8uL
</TD><TD> 2.5uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 10
</TD><TD> 7B
</TD><TD> 8uL
</TD><TD> 2.5uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 11
</TD><TD> 8B
</TD><TD> 8uL
</TD><TD> 2.5uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 12
</TD><TD> 9B
</TD><TD> 8uL
</TD><TD> 2.5uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 13
</TD><TD> 4C
</TD><TD> 8uL
</TD><TD> 2.5uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 14
</TD><TD> 3E
</TD><TD> 10uL
</TD><TD> 2.5uL
</TD><TD> 2.5uL
</TD></TR><TR><TD> 15
</TD><TD> 5B
</TD><TD> 10uL
</TD><TD> 2.5uL
</TD><TD> 2.5uL
</TD></TR><TR><TD> 16
</TD><TD> 10C
</TD><TD> 10uL
</TD><TD> 2.5uL
</TD><TD> 2.5uL
</TD></TR><TR><TD> 17
</TD><TD> 12B
</TD><TD> 10uL
</TD><TD> 2.5uL
</TD><TD> 2.5uL
</TD></TR></TBODY></TABLE><P>Bacterial Transformation of SBcas9• ER: Done according  to protocol.
</P><P><B>6/27</B></P><P>Mini prep done to protocol on CPEC products. 
Nanodrops were taken to assess the concentration and purity of the samples:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 1A
</TD><TD> 66.9
</TD><TD> 1.90
</TD><TD> 2.26
</TD></TR><TR><TD>      1C
</TD><TD> 61.3
</TD><TD> 1.92
</TD><TD> 2.20
</TD></TR><TR><TD> 2B
</TD><TD> 57.8
</TD><TD> 1.91
</TD><TD> 2.27
</TD></TR><TR><TD> 3B
</TD><TD> 69.4
</TD><TD> 1.88
</TD><TD> 1.14
</TD></TR><TR><TD> 3H
</TD><TD> 51.7
</TD><TD> 1.89
</TD><TD> 2.24
</TD></TR><TR><TD> 9B
</TD><TD> 33.6
</TD><TD> 1.99
</TD><TD> 1.62
</TD></TR><TR><TD> 10C
</TD><TD> 49.8
</TD><TD> 1.93
</TD><TD> 2.09
</TD></TR><TR><TD> 11B
</TD><TD> 40.5
</TD><TD> 1.94
</TD><TD> 2.01
</TD></TR><TR><TD> 12B
</TD><TD> 51.9
</TD><TD> 1.89
</TD><TD> 1.17
</TD></TR></TBODY></TABLE><P>Sequenced all of the above  products (total volume of 15 uL: 10 uL DNA+ 2.5 uLprimer (CRE22)+2 .5 uL  MQ H2O
Exceptions: 
</P><DL><DD>-9B received 12.5 uL DNA + 2.5 uL primer (CRE22) 
</DD><DD>-12B received p51 seq1 as the primer  
</DD></DL><P>Glycerol stock of all the above samples  to protocol: 1:1 w/ 40% glycerol for a total volume of 500 uL each. 
</P><P>Positive sequencing results: 1A, 1C, 2B, 3B, 3H, 9B, 10C, 11B, 12B   
</P><P>Bacterial Transformation of SBcas9• ER and PUC19 as the positive control: Done according to protocol.
</P><P>Prepared 5mL O/N cultures in LB+AMP of colonies selected from glycerol stocks
</P><P>Mini preps done to protocol on the below samples
Nanodrops were taken to assess the concentration and purity of the samples:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 1A
</TD><TD> 78.7
</TD><TD> 1.90
</TD><TD> 2.08
</TD></TR><TR><TD>      1C
</TD><TD> 54.3
</TD><TD> 1.88
</TD><TD> 1.76
</TD></TR><TR><TD> 2B
</TD><TD> 335.4
</TD><TD> 1.48
</TD><TD> 0.91
</TD></TR><TR><TD> 3B
</TD><TD> 52.1
</TD><TD> 1.91
</TD><TD> 1.96
</TD></TR><TR><TD> 3H
</TD><TD> 59.7
</TD><TD> 1.89
</TD><TD> 2.03
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_8">Week 8</SPAN></H2><P><B>6/30</B>
Bacterial Transformation of BWF+ w/ 158, 723, SC1, p51: Done according to protocol.
</P><P><B>7/1</B></P><P>Nanodrops were taken to assess the concentration of the samples to use in a cPEC reaction. 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH></TR><TR><TD> p50 Amplicon
</TD><TD> 27.3
</TD></TR><TR><TD>      GE AmilCP
</TD><TD> 493.9
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Gel extracted
</LI></UL></DD></DL><P>cPEC performed to combine the p50 amplicon with AmilCP following the cPEC protocol and using the following amounts:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 12.5 uL
</TD></TR><TR><TD> Insert (AmilCP)
</TD><TD> 5 uL
</TD></TR><TR><TD> Template DNA p50
</TD><TD> 2.0 uL (1:20 dilution)
</TD></TR><TR><TD> 25% DMSO
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 3.5 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><P>Thermocycler conditions:
</P><TABLE class="wikitable"><TBODY><TR><TH> cPEC
</TH></TR><TR><TD> Initial Denature
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Denature
</TD><TD> 98°C/10 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 60°C/30 sec
</TD></TR><TR><TD> Extend
</TD><TD> 72°C/105 sec
</TD></TR><TR><TD> # of cycles
</TD><TD> 10
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C/10 min
</TD></TR></TBODY></TABLE><P>Successful Bacterial Transformation of BWF+ w/ 158, 723, SC1, p51 (from 6/30) . O/N cultures of each were then made. 
</P><P><B>7/4</B></P><P>Electroporation of p50 amplicon • AmilCP with Marcello 
</P><P><B>7/7</B></P><P>Electroporation was unsuccessful. Redone using Bacterial Transformation done to protocol. 
</P><P>Primer design for qPCR 
</P><P>Streaked p50 on plate
</P><H2><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H2><P><B>7/8</B></P><P>mini prepped p50 done to protocol 
</P><P>Colony PCR done to protocol on the p50+AmilCP transformation product . Selected 12 colonies.
</P><DL><DD><UL><LI>PCR tube lids opened in thermocycler for samples: 8, 12, and positive control 
</LI></UL></DD></DL><P>Colony PCR products are run on a 1% agarose gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> 1
</TD></TR><TR><TD> 3
</TD><TD> 2
</TD></TR><TR><TD> 4
</TD><TD> 3
</TD></TR><TR><TD> 5
</TD><TD> 4
</TD></TR><TR><TD> 6
</TD><TD> 5
</TD></TR><TR><TD> 7
</TD><TD> 6
</TD></TR><TR><TD> 8
</TD><TD> 7
</TD></TR><TR><TD> 9
</TD><TD> 8
</TD></TR><TR><TD> 10
</TD><TD> 9
</TD></TR><TR><TD> 11
</TD><TD> 10
</TD></TR><TR><TD> 12
</TD><TD> 11
</TD></TR><TR><TD> 13
</TD><TD> 12
</TD></TR><TR><TD> 14
</TD><TD> (+) control
</TD></TR><TR><TD> 15
</TD><TD> (-) control
</TD></TR></TBODY></TABLE><P><B>7/9</B></P><P>Mini prepped colony PCR samples: 1,7,8,10,11, and p50
</P><P>Followed by nanodrop with results listed below:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> 1
</TD><TD> 89
</TD><TD> 1.90
</TD><TD> 2.22
</TD></TR><TR><TD> 7
</TD><TD> 110.9
</TD><TD> 1.88
</TD><TD> 2.22
</TD></TR><TR><TD> 8
</TD><TD> 134.4
</TD><TD> 1.85
</TD><TD> 1.77
</TD></TR><TR><TD> 10
</TD><TD> 140.9
</TD><TD> 1.87
</TD><TD> 2.10
</TD></TR><TR><TD> 11
</TD><TD> 93.6
</TD><TD> 1.89
</TD><TD> 2.21
</TD></TR><TR><TD> p50
</TD><TD> 191.7
</TD><TD> 1.87
</TD><TD> 2.35
</TD></TR></TBODY></TABLE><P>The above samples (except 8) were sent for sequencing.
Total volume 0f 15 uL
</P><TABLE class="wikitable"><TBODY><TR><TH>  Product
</TH><TH> DNA uL
</TH><TH> Primer CRE22
</TH><TH> MQ H2O
</TH></TR><TR><TD> 1
</TD><TD> 1
</TD><TD> 6.0uL
</TD><TD> 5.0 uL
</TD><TD> 4.0uL
</TD></TR><TR><TD> 2
</TD><TD> 7
</TD><TD> 5.0uL
</TD><TD> 5.0uL
</TD><TD> 5.0uL
</TD></TR><TR><TD> 3
</TD><TD> 10
</TD><TD> 4.0uL
</TD><TD> 5.0uL
</TD><TD> 6.0uL
</TD></TR><TR><TD> 4
</TD><TD> 11
</TD><TD> 6.0uL
</TD><TD> 5.0uL
</TD><TD> 4.0uL
</TD></TR></TBODY></TABLE><P>Transformation done to protocol on all gRNAs into BWF+
</P><P><B>7/10</B></P><P>Site directed Mutagenesis (SDM) performed on original p50, 1, 7, 10, 11 to remove Xba1 from p50 
Primer names: p50TetRXba1F and  p50TetRXba1R 
Primer Mix: 1 uL forward + 1 uL reverse + 18 uL MQ H2O
</P><P>PCR run on the following samples of p50: p50 DNA dilution for original, 1,7,10, and 11 
p50 DNA sample original- 1:38 dilution
p50 DNA sample 1- 1:18 dilution
p50 DNA sample 7- 1:22 dilution
p50 DNA sample 10- 1:28 dilution
p50 DNA sample 11- 1:19 dilution 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 12.5 uL
</TD></TR><TR><TD> 10 uM primer mix
</TD><TD> 2.5 uL
</TD></TR><TR><TD> DNA p50
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 8.0 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><P>Thermocycler conditions:
</P><TABLE class="wikitable"><TBODY><TR><TD> Initial Denature
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Denature
</TD><TD> 98°C/30 sec
</TD></TR><TR><TD> Anneal
</TD><TD> 55°C/30 sec
</TD></TR><TR><TD> Extend
</TD><TD> 72°C/3.5 minutes
</TD></TR><TR><TD> # of cycles
</TD><TD> 15 cycles
</TD></TR><TR><TD> Final Extension
</TD><TD> 72°C/5 minutes
</TD></TR></TBODY></TABLE><P>SDM performed on 6 gRNAs and the original p51 to remove spe1 site
</P><DL><DD>PCR preformed:
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 12.5 uL
</TD></TR><TR><TD> 10 uM primer mix
</TD><TD> 2.5 uL
</TD></TR><TR><TD> DNA p51
</TD><TD> 2.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 8.0 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>primer mix: 1 uL forward primer+ 1 uL reverse primer +18 uL MQH2O. Primers varied for each gRNA
</LI><LI>control consisted of DNA only, no primers
</LI></UL></DD></DL><DL><DD>Thermocycler conditions:
</DD></DL><TABLE class="wikitable"><TBODY><TR><TD> Initial Denature 98
</TD><TD> 95°C/30 sec
</TD></TR><TR><TD> Denature 98
</TD><TD> 95°C/30 sec
</TD></TR><TR><TD> Anneal 55
</TD><TD> 55°C/1min
</TD></TR><TR><TD> Extend 72
</TD><TD> 68°C/3min
</TD></TR><TR><TD> # of cycles
</TD><TD> 15 cycles
</TD></TR><TR><TD> Final Extension 72
</TD><TD> 3 min
</TD></TR></TBODY></TABLE><P>7/11/2014
</P><P>All PCR gRNAs DPN1 digested:
</P><DL><DD>1. Add 1 ul of the Dpn I restriction enzyme (10 U/ul) directly to each
</DD></DL><P>amplification reaction below the mineral oil overlay.
</P><DL><DD>2. Gently and thoroughly mix each reaction mixture by pipetting the
</DD></DL><P>solution up and down several times. Spin down the reaction mixtures in
a microcentrifuge for 1 minute and immediately incubate each reaction
at 37°C for 1 hour to digest the parental (i.e., the nonmutated)
supercoiled dsDNA.
</P><P>Transformation done to protocol for all gRNAs into DH5u. Full volume (250ul) plated onto Amp  plates and  incubated o/n. 
</P><H2><SPAN class="mw-headline" id="Week_10">Week 10</SPAN></H2><P><B>7/16</B></P><P>O/N cultures of SDM gRNA p51 (4 colonies) and SC1 (2 colonies)
</P><P><B>7/17</B></P><P>Mini prepped all p51 and SC1 cultures to protocol and then nano dropped:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> p51 (1)
</TD><TD> 62.5
</TD><TD> 1.90
</TD><TD> 1.65
</TD></TR><TR><TD>     p51(2)
</TD><TD> 87.2
</TD><TD> 1.95
</TD><TD> 2.20
</TD></TR><TR><TD> p51(3)
</TD><TD> 112.7
</TD><TD> 1.81
</TD><TD> 1.34
</TD></TR><TR><TD> p51(4)
</TD><TD> 67.1
</TD><TD> 1.92
</TD><TD> 2.24
</TD></TR><TR><TD> sc1 (1)
</TD><TD> 126.5
</TD><TD> 1.83
</TD><TD> 1.48
</TD></TR><TR><TD> sc1(2)
</TD><TD> 101.9
</TD><TD> 1.88
</TD><TD> 1.84
</TD></TR></TBODY></TABLE><P>Restriction digest done to protocol: 
</P><TABLE class="wikitable"><TBODY><TR><TH> Sample
</TH><TH> Buffer Green Fast Digest
</TH><TH> Enzyme Mix*
</TH><TH> DNA
</TH><TH> MQ H2O
</TH><TH> Total Volume
</TH></TR><TR><TD> p51 (1)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 3.0 uL
</TD><TD> 5.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> p51 (2)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 6.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> p51 (3)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 6.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> p51 (4)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 3.0 uL
</TD><TD> 5.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> SC1 (1)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 6.0 uL
</TD><TD> 10.0 uL
</TD></TR><TR><TD> SC1 (2)
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 6.0 uL
</TD><TD> 10.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Enzyme Mix contains 1.0 uL Spe1 + 1.0 uL BgI1 + 23.0 uL mqH2O
</LI></UL></DD></DL><P>Samples evaporated after digesting to a total volume of around 4 uL
</P><P>Run on a 1% Gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> SDM p51 (1)
</TD></TR><TR><TD> 3
</TD><TD> SDM p51 (3)
</TD></TR><TR><TD> 4
</TD><TD> SDM SC1 (1)
</TD></TR><TR><TD> 5
</TD><TD> SDM SC1 (2)
</TD></TR><TR><TD> 6
</TD><TD> Blank
</TD></TR><TR><TD> 7
</TD><TD> SDM P51 (4)
</TD></TR></TBODY></TABLE><P><B>7/19</B>
PCR amplify p50
</P><TABLE class="wikitable"><TBODY><TR><TH>  Reactants
</TH><TH> Volume
</TH></TR><TR><TD> Q5 Master Mix
</TD><TD> 12.5 uL
</TD></TR><TR><TD> 10 uM forward primer
</TD><TD> 1.5 uL
</TD></TR><TR><TD> 10 uM reverse primer
</TD><TD> 1.5 uL
</TD></TR><TR><TD> DNA p50 (1:50)
</TD><TD> 1.0 uL
</TD></TR><TR><TD> Nuclease Free H2O
</TD><TD> 9.0 uL
</TD></TR><TR><TD> Final Reaction Volume
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><P><B>7/21</B></P><P>Restriction digest same gRNAs again 
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Buffer cutsmart
</TH><TH> Enzyme Xbal
</TH><TH> Enzyme Spe1
</TH><TH> DNA (200 ng)
</TH><TH> MQ H2O
</TH><TH> Total
</TH></TR><TR><TD> p51 (1)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 3.2 uL
</TD><TD> 12.8 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> p51 (2)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD>             2.3 uL
</TD><TD> 13.7 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> p51 (3)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.8 uL
</TD><TD> 14.2 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> p51 (4)
</TD><TD> 2.0 uL
</TD><TD>1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 3.0 uL
</TD><TD> 13.0uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SC1 (1)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.6 uL
</TD><TD> 14.4 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SC1 (2)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 14.0uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> P51
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 4.0 uL
</TD><TD> 12.0uL
</TD></TR><TR><TD> P51
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 4.0 uL
</TD><TD> 12.0uL
</TD></TR><TR><TD> p51
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 4.0 uL
</TD><TD> 12.0uL
</TD></TR></TBODY></TABLE><P>Enzymes are used as a 1:25 dilution
</P><P>Run on a 1% Gel. The results were still unclear, the following day samples were sent for sequencing.
</P><H2><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H2><P><B>7/23</B></P><P>Sent p51 SDM 1, 3, 4 and p51 original as a control for sequencing. Confirmation that SDM was successful and p51 no longer has the restriction site spe1 in the gRNA location. 
</P><P>Sequencing: Total volume 0f 15 uL
</P><TABLE class="wikitable"><TBODY><TR><TH>  SDM Product
</TH><TH> DNA uL
</TH><TH> Primer p51seq5
</TH><TH> MQ H2O
</TH></TR><TR><TD> 1
</TD><TD> SDM1p51
</TD><TD> 8.0uL
</TD><TD> 2.5 uL
</TD><TD> 4.5uL
</TD></TR><TR><TD> 2
</TD><TD> SDM3p51
</TD><TD> 4.4uL
</TD><TD> 2.5uL
</TD><TD> 8.1uL
</TD></TR><TR><TD> 3
</TD><TD> SDM4p51
</TD><TD> 7.5uL
</TD><TD> 2.5uL
</TD><TD> 5.0uL
</TD></TR><TR><TD> 4
</TD><TD> p51 control
</TD><TD> 4.2uL
</TD><TD> 2.5uL
</TD><TD> 8.3uL
</TD></TR></TBODY></TABLE><DL><DD>Mini prepped 10 o/n cultures of SDM of p50. Done to protocol. Glycerol stocks make of all 10 cultures.
</DD><DD>Mini prepped previous gibson colonies 1 and 2.  SDM of p51 for all samples were confirmed 
</DD></DL><DL><DD>Nanodrops were taken to assess the concentration and purity of the samples:
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> ng/ uL
</TH><TH> 260/280
</TH><TH> 260/230
</TH></TR><TR><TD> p50
</TD><TD> 123.7
</TD><TD> 1.84
</TD><TD> 2.15
</TD></TR><TR><TD>      SDM1
</TD><TD> 179.6
</TD><TD> 1.88
</TD><TD> 2.19
</TD></TR><TR><TD> SDM2
</TD><TD> 117.4
</TD><TD> 1.90
</TD><TD> 2.21
</TD></TR><TR><TD> SDM3
</TD><TD> 140.4
</TD><TD> 1.84
</TD><TD> 1.81
</TD></TR><TR><TD>     SDM4
</TD><TD> 162.6
</TD><TD> 1.89
</TD><TD> 2.22
</TD></TR><TR><TD> SDM5
</TD><TD> 117.3
</TD><TD> 1.89
</TD><TD> 2.00
</TD></TR><TR><TD> SDM6
</TD><TD> 176.4
</TD><TD> 1.88
</TD><TD> 2.01
</TD></TR><TR><TD> SDM7
</TD><TD> 177.0
</TD><TD> 1.87
</TD><TD> 2.30
</TD></TR><TR><TD> SDM8
</TD><TD> 250.1
</TD><TD> 1.89
</TD><TD> 2.25
</TD></TR><TR><TD> SDM9
</TD><TD> 298.3
</TD><TD> 1.90
</TD><TD> 2.31
</TD></TR><TR><TD> SDM10
</TD><TD> 136.0
</TD><TD> 1.91
</TD><TD> 2.17
</TD></TR><TR><TD> Gibson1
</TD><TD> 110.8
</TD><TD> 1.89
</TD><TD> 2.14
</TD></TR><TR><TD> Gibson2
</TD><TD> 160.3
</TD><TD> 1.87
</TD><TD> 2.15
</TD></TR></TBODY></TABLE><P>Restriction digest done to protocol: 
</P><TABLE class="wikitable"><TBODY><TR><TH> Sample
</TH><TH> Buffer Cutsmart
</TH><TH> Enzyme Mix*
</TH><TH> DNA (200ng)
</TH><TH> MQ H2O
</TH><TH> Total Volume
</TH></TR><TR><TD> SDM p50 (1)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.2 uL
</TD><TD> 15.8 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (2)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 15.0 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (3)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.4 uL
</TD><TD> 15.6 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50  (4)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.2 uL
</TD><TD> 15.8uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50  (5)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 15.0 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (6)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.2 uL
</TD><TD> 15.8 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (7)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.2 uL
</TD><TD> 15.8uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (8)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 16.0uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (9)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 16.0uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> SDM p50 (10)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.5 uL
</TD><TD> 15.5uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> p50control
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 15.0uL
</TD><TD> 20.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Enzyme Mix contains 1.0 uL Spe1 + 1.0 uL Xba1 + 48.0 uL mqH2O
</LI></UL></DD></DL><P>Restriction Digest for gibson done to protocol:
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Buffer D
</TH><TH> Enzyme EcoRV
</TH><TH> DNA (200 ng)
</TH><TH> MQ H2O
</TH><TH> Total
</TH></TR><TR><TD> Gibson 1
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 15.0 uL
</TD><TD> 20.0 uL
</TD></TR><TR><TD> Gibson 2
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.2 uL
</TD><TD> 15.8 uL
</TD><TD> 20.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>EcoRv dilution (1:25)
</LI></UL></DD></DL><P>
Run on a 1% Gel. The following lists the sample loaded into each lane.
</P><TABLE class="wikitable"><TBODY><TR><TH>  Lane
</TH><TH> Sample
</TH></TR><TR><TD> 1
</TD><TD> 2-Log DNA Ladder
</TD></TR><TR><TD> 2
</TD><TD> SDM p50 (1)
</TD></TR><TR><TD> 3
</TD><TD> SDM p50 (2)
</TD></TR><TR><TD> 4
</TD><TD> SDM p50 (3)
</TD></TR><TR><TD> 5
</TD><TD> SDM p50 (4)
</TD></TR><TR><TD> 6
</TD><TD> SDM p50  (5)
</TD></TR><TR><TD> 7
</TD><TD> SDM p50 (6)
</TD></TR><TR><TD> 8
</TD><TD> SDM p50 (7)
</TD></TR><TR><TD> 9
</TD><TD> SDM p50 (8)
</TD></TR><TR><TD> 10
</TD><TD> SDM p50 (9)
</TD></TR><TR><TD> 11
</TD><TD> SDM p50 (10)
</TD></TR><TR><TD> 12
</TD><TD> p50 control
</TD></TR><TR><TD> 13
</TD><TD> p50 undigested
</TD></TR><TR><TD> 14
</TD><TD> Gibson 1
</TD></TR><TR><TD> 15
</TD><TD> Gibson 2
</TD></TR><TR><TD> 16
</TD><TD> sb1c3
</TD></TR><TR><TD> 17
</TD><TD> blank
</TD></TR></TBODY></TABLE><P><B>7/25</B></P><P>Restriction digest done to protocol: 
</P><TABLE class="wikitable"><TBODY><TR><TH> Sample
</TH><TH> Buffer Cutsmart
</TH><TH> Enzyme Mix*
</TH><TH> DNA (250 ng)
</TH><TH> MQ H2O
</TH><TH> Total
</TH></TR><TR><TD> SDM p50 (1)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.5 uL
</TD><TD> 20.0 uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> SDM p50 (2)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 19.5 uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> SDM p50 (3)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.8 uL
</TD><TD> 19.7 uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> SDM p50  (4)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.5 uL
</TD><TD> 20.0uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> SDM p50  (5)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 19.5 uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> SDM p50 (6)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.5 uL
</TD><TD> 20.0 uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> SDM p50 (10)
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 19.5uL
</TD><TD> 25.0 uL
</TD></TR><TR><TD> p50control
</TD><TD> 2.0 uL
</TD><TD> 1.0 uL
</TD><TD> 2.0 uL
</TD><TD> 19.5uL
</TD><TD> 25.0 uL
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Enzyme mix: 5.5uL Xba1 + 5.5uL Spe1 for a total volume of 11uL
</LI></UL></DD></DL><P>Run on a 1% Gel. Results were not as anticipated. Restriction Ligation was repeated.
</P><P>Restriction Ligation
</P><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Buffer Cutsmart
</TH><TH> EcoR1
</TH><TH> Xba1
</TH><TH> Spe1
</TH><TH> Pst1
</TH><TH> DNA (200 ng)
</TH><TH> MQ H2O
</TH><TH> Total
</TH></TR><TR><TD> SDM p50 (1)
</TD><TD> 5.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 42.0 uL
</TD><TD> 50.0 uL
</TD></TR><TR><TD> SDM p50 (4)
</TD><TD> 5.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD>       1.0 uL
</TD><TD> 42.0 uL
</TD><TD> 50.0 uL
</TD></TR><TR><TD> SDM p50 (6)
</TD><TD> 5.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 42.0 uL
</TD><TD> 50.0 uL
</TD></TR><TR><TD> Litimus28i
</TD><TD> 5.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 1.0 uL
</TD><TD> 1.5 uL
</TD><TD> 41.5uL
</TD><TD> 50.0 uL
</TD></TR><TR><TD> Litimus+Ic3
</TD><TD> 5.0 uL
</TD><TD> 1.0 uL
</TD><TD> --
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> 0.8 uL
</TD><TD> 42.2 uL
</TD><TD> 50.0 uL
</TD></TR><TR><TD> PsBGA1 (RFP)
</TD><TD> 5.0 uL
</TD><TD> 1.0 uL
</TD><TD> --
</TD><TD> 1.0 uL
</TD><TD> --
</TD><TD> 3.8 uL
</TD><TD> 39.2 uL
</TD><TD> 50.0 uL
</TD></TR></TBODY></TABLE><P>Sequenced SDMp50 was not the correct product, the above project did not continue.
</P><H2><SPAN class="mw-headline" id="Week_12">Week 12</SPAN></H2><P><B>8/18</B>  
O/N cultures of MG, Bw+Kan, Bw-kan 
</P><P><B>8/19</B> 
O/N cultures were all made competent according to protocol 
O/D recordings: 
</P><TABLE class="wikitable"><TBODY><TR><TH> Time
</TH><TH> MG
</TH><TH> Bw+kan
</TH><TH> Bw(-)kan
</TH></TR><TR><TD> 11:00
</TD><TD> .042
</TD><TD> .026
</TD><TD> .071
</TD></TR><TR><TD> 11:34
</TD><TD> .086
</TD><TD> .035
</TD><TD> .088
</TD></TR><TR><TD> 12:10
</TD><TD> .239
</TD><TD> .078
</TD><TD> .223
</TD></TR><TR><TD> 12:20
</TD><TD> .328
</TD><TD> .098
</TD><TD> .296
</TD></TR><TR><TD> 12:28
</TD><TD> --
</TD><TD> --
</TD><TD> .325
</TD></TR><TR><TD> 12:47
</TD><TD> --
</TD><TD> .196
</TD><TD> --
</TD></TR><TR><TD> 1:10
</TD><TD> --
</TD><TD> .308
</TD><TD> --
</TD></TR><TR><TD> 1:15
</TD><TD> --
</TD><TD> .320
</TD><TD> --
</TD></TR></TBODY></TABLE><P>The cells were all pulled at the time they were at an O/D  nearest .325. All O/D’s were diluted to .30
Once the cells were made competent they were transformed using a gRNA  target, gRNA scramble, and  a sb1c3  strain. 
</P><DIV class="newfooter"><DIV id="newfooterbox"><DIV class="footertext"><UL><LI><A href="https://igem.org/About">About</A></LI><LI><A href="https://igem.org/Contact">Contact</A></LI><LI><A href="https://igem.org/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Main_Page">  igem.org</A></LI><LI><A href="http://parts.igem.org/Main_Page"> Registry</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://2014.igem.org/Requirements"> Requirements</A></LI><LI><A href="https://2014.igem.org/Giant_Jamboree"> Giant Jamboree</A></LI><LI><A href="https://2014.igem.org/Meetups"> Meetups</A></LI><LI><A href="https://2014.igem.org/Calendar_of_Events">Calendar </A></LI><LI><A href="https://2014.igem.org/Partner_Offers">Partner Offers</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Login">Login</A></LI><LI><A href="https://2014.igem.org/Special:Upload">Upload Files</A></LI><LI><A href="https://2014.igem.org/Special:RecentChanges"> Recent Changes</A></LI><LI><A href="https://2014.igem.org/Special:SpecialPages"> Special Pages</A></LI><LI><A href="https://2014.igem.org/Special:Search"> Search</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Press_Kit">Press Kit</A></LI><LI><A href="https://igem.org/Newsletters"> Newsletter</A></LI><LI><A href="https://igem.org/Alumni">AlumniGEM</A></LI></UL></DIV><DIV class="footerextra">
powered by <A href="http://www.mediawiki.org/" style="link-style:none;">MediaWiki</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Induced">http://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Induced</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:CU-Boulder/Notebook/CC9_Team/Induced" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:CU-Boulder/Notebook/CC9_Team/Induced" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Induced&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/CC9_Team/Induced&amp;oldid=221905" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>