"
Page
Discussion
View source
History
teams
Log in
 
Team:CU-Boulder/Results
From 2014.igem.org
HOME
PROJECT
Getting Started
Motivation
Design
Results
Future Directions
Modeling
ACHIEVEMENTS
Our Team Achievements
Biobricks
Results
Medal Requirements
NOTEBOOK
At the Bench
Phage Delivery
Induced CRISPR-Cas9
Constitutive CRISPR-Cas9
Phage CRISPR-Cas9
Interlab Study
Protocols
HUMAN PERSPECTIVE
Reaching Out
Human Practices
Collaboration
Ethics & Safety
TEAM
MI6
Members
Acknowledgments
Contents
1 Results
1.1 Our System: CRISPR-Cas9 phage
1.2 New BioBricked Parts
1.3 Phage delivery system (M13ori)
1.4 Making phage with M13g6A1
1.5 Program to design a Spacer Sequence
1.6 Targeted killing of BW23115 through Transformation (characterization of BBa_K1218011)
1.7 Cas9 delivery through phage
1.8 Additional Work and Characterization (BBa_K1445002)
1.9 References
Results
Our System: CRISPR-Cas9 phage
The system is comprised of two parts: the delivery of the CRISPR-Cas9 machinery and the targeting of the endonuclease.
Figure 1. Diagram of phage production.
M13 phage were used as the delivery mode in this system. Depicted in Figure 1 is a single bacterium containing a helper phagemid and a CIRSPR-Cas9 phagemid. The phage coat and assembly proteins are expressed from the helper phagemid, who’s packaging signal has been disrupted by the insertion of a selectable marker (kanamycin in M13K07 or ampicillin in M13g6A1, see Figure 2).  Due to the disruption of the helper phagemid’s packaging signal, the probability it will be packaged into a phage capsid has been greatly reduced. The CRISPR-Cas phagemid, BBa_K1445001 contains the endogenous CRISPR-Cas9 sequence from Streptococcus pyogenes and a fully functional packaging signal. M13 phage capsids preferentially package BBa_K1445001 during phage assembly.  The resulting phage phage containing the  BBa_K1445001 phagemid are isolated and can be used to infect a target population of bacteria.
Figure 2. Diagram of CRISPR-Cas9 sequence targeting.
Once the CRISPR-Cas9 phagemid enters the cell, the bacterium will express the Cas9 endonuclease and a guide RNA(gRNA). Guide RNAs are composed of a spacer sequence that binds the DNA, and a handle to which the Cas9 endonuclease binds. The Cas9 protein and the gRNA come together to search DNA within the cell for PAM sites (Protospacer Adjacent Motifs, shown here in orange). The Cas9 will bind to the PAM site allowing the gRNA to anneal to the target sequence (shown here in green). If binding is successful, the Cas9 endonuclease cleaves the DNA resulting in a double stranded break. If the cell is unable to repair the damage itself or does so incorrectly, the cell will die¹.
New BioBricked Parts
BBa_K1445000: The M13 origin of replication (M13ori) is a noncoding sequence that when cloned onto a plasmid, facilitates the uptake of that plasmid into an M13 phage capsid.
BBa_K1445001: M13ori (BBa_K1445000) cloned onto a CRISPR-Cas9 construct (BBa_K1218011). The CRISPR-Cas9 part codes for a Cas9 endonuclease that is guided to a DNA sequence specified by a spacer located within the minimal CRISPR array. Inclusion of the M13ori allows for its uptake into M13 phage.
BBa_K1445002: BioBricked phagemid vector allows for the easy cloning of BioBrick parts into a backbone that can be packaged into a phage without the addition of the M13ori to the insert region.
Additionally, the genes for M13 phage coat proteins were BioBricked and cloned onto a pSB6A1 vector. This allows the phage producing genes to be cloned onto vectors with a different resistance marker if the resistance interferes with the marker on the phagemid. This part was not submitted to the registry.
Phage delivery system (M13ori)
The M13 origin of replication (M13ori) has been documented as the packaging signal for the M13 phage.  When single stranded, the M13ori forms unique hairpins that signal packaging into a phage capsid. To verify its functionality, its packaging efficiency was compared to that of amilCP. The amilCP sequence is roughly equal in length to the M13ori and both parts were cloned onto the pSB1C3 backbone; therefore, the nucleotide sequence is the only distinguishing factor. The helper phagemid M13K07 made phage in host cells containing pSB1C3-M13ori or pSB1C3-amilCP phagemids. ER2738 E. coli were infected by phage from the M13ori or amilCP sample to assess the respective packaging efficiencies.
Figure 3. Conjugated BW23115 E. coli infected with recombinant phage  A) pSB1C3-M13ori or B) pSB1C3-amilCP phagemids on LB agar with 170 ug/mL chloramphenicol.
The growth in the M13ori sample demonstrates that this phagemid was packaged and delivered by the M13 phage. The absence of growth in the amilCP sample proves that without the presence of a packaging signal, a plasmid cannot be taken up by a phage capsid. These results show that the M13ori is necessary and sufficient for phagemid packaging.
Making phage with M13g6A1
In the phage delivery experiment, M13K07 was used as the helper phagemid. This part contains a kanamycin resistance gene, which is similar to the neomycin phosphotransferase gene being targeted by CRISPR-Cas in this experiment.  A new resistance marker was needed to be able to screen future phage production.  The genes that code for the phage coat proteins were amplified from M13K07 and cloned into the pSB6A1 vector.  Phage amplification, using M13g6A1 as the helper phagemid (and no additional phagemid), produced M13 phage packaging M13g6A1. ER2738 cells were infected with the isolated phage and were plated onto ampicillin to verify the viability of phage produced by this helper phagemid.
Figure 4. ER2738 cells infected with M13g6A1 helper phagemid grown on LB with 100ug/mL ampicillin.
The presence of colonies in Figure 2 confirm that M13g6A1 is capable of producing viable phage.
Program to design a Spacer Sequence
For the following Cas9 experiments, a spacer sequence was designed to target the neomycin phosphotransferase gene present in the targeting strain. This program inputs FASTA files in two sets. In one are the sequences that should be targeted for Cas9 binding while the other set consists of sequences that should not be targeted. It then searches for possible targeting sites that are present in the targeting sequences and absent from the sequences that should not be targeted. The best spacer sequences are then presented to the user.
For this project, the sequence for the neomycin phosphotransferase gene was given to the program as the targeting sequence. The E. coli strains K-12, BL21 and MG1655 genomes were given to the program as sequences to avoid targeting. One of the output spacers was chosen for use in the following CRISPR-Cas9 experiments.
Targeted killing of BW23115 through Transformation (characterization of BBa_K1218011)
The spacer designed to target the neomycin phosphotransferase gene was cloned into the CRISPR-Cas9 construct (BBa_K1218011). The unmodified spacer was used as a non-targeting control since it does not compliment any region in the BW23115(::kan) or ER2738 genomes.
The targeting and non-targeting constructs were transformed into BW23115(::kan) E. coli chemical transformation and selected for on chloramphenicol.
Figure 5. Transformation results of neomycin resistant E. coli with BBa_K1218011 having either A) non-targeting or B) targeting spacer sequence on LB agar with 170 ug/mL chloramphenicol.
The decrease in growth between the non-targeting (1920 colonies) and the targeting (8 colonies) samples is credited to the difference in spacer sequence. When the spacer sequence compliments the genome, the Cas9 endonuclease is able to bind and cleave the DNA, resulting in cell death.  Interestingly, some targeted colonies survived as can be seen in Figure 5B. Sequencing showed that all eight surviving colonies had deleted the spacer region and one or both of the adjacent repeats.  None the less, the difference in transformation efficiency demonstrates the potential for CRISPR-Cas systems as antibacterial therapeutics.
Cas9 delivery through phage
The M13 origin of replication, or M13ori (BBa_K1445000) was cloned upstream of the CRISPR-Cas9 construct (BBa_K1218011) in the targeting and the non-targeting varieties to allow for their uptake into M13 phage. This composite part was packaged into phage using the M13g6A1 helper phagemid. Conjugated BW23115(::kan, F’) E. coli were infected with progeny phage and plated onto chloramphenicol.
Figure 6. Infection results of M13 phage delivering A) non-targeting and B) targeting varieties of part BBa_K1445001 on LB agar with 170 ug/mL chloramphenicol.
The presence of growth in both samples demonstrates that the addition of the M13ori does allow for packaging and delivery by the M13 phage. The decrease in growth from the non-targeting (143 colonies) to the targeting (11 colonies) is accredited to the differences in spacer sequence. With a spacer sequence that targets within the genome of the cell, the Cas9 endonuclease is able to bind and cleave the DNA, successfully killing the bacterium.
Unmodified spacer may have off-targeted binding to F’ episome, causing death in phage producing cell, thereby reducing phage yield, or in BW23115(::kan, F’) resulting in fewer colonies than expected.
Additional Work and Characterization (BBa_K1445002)
Part BBa_J04450 and flanking regions to the sequencing primer sites were cloned into the Litmus28i vector to make a biobricked phagemid backbone that can be packaged into M13 phage. To verify functionality, the vector was packaged into phage using the M13K07 helper phagemid. Progeny phage infected ER2738 E. coli and cells were selected for on ampicillin.
Figure 7. Packaging efficiency of recombinant phagemid to helper phagemid packaging. A) Infected cells diluted 1:1000 and grown on 100 ug/mL Ampicillin to select for BBa_K1445002. B) Infected diluted 1:1000 and grown on 50 ug/mL Kanamycin to select for M13K07 helper phagemid.  BBa_K1445002 was packaged over M13K07 at a ratio of 136:1.
The growth on ampicillin (1496 colonies) shows that the biobricked version of Litmus28i retains its ability to be packaged and delivered by M13 phage. Additionally, the same sample was plated on kanamycin to determine the packaging ratio of phagemid to helper phagemid. As shown in Figure 7B, very little growth was present on kanamycin (11 colonies) confirming that BBa_K1445002 was preferentially packaged over the M13K07 helper phagemid at a ratio of 136:1.
References
1.  Gomaa AA, Klumpe HE, Luo ML, Selle K, Barrangou R, Beisel CL. 2014. Programmable removal of bacterial strains by use of genome-targeting CRISPR-Cas systems. mBio 5(1):e00928-13. doi:10.1128/mBio.00928-13.
About
Contact
Sponsors
igem.org
Registry
Requirements
Giant Jamboree
Meetups
Calendar
Partner Offers
Login
Upload Files
Recent Changes
Special Pages
Search
Press Kit
Newsletter
AlumniGEM
powered by MediaWiki
Retrieved from " http://2014.igem.org/Team:CU-Boulder/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
