<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_CU-Boulder_Notebook_Phage_Team"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:CU-Boulder/Notebook/Phage_Team">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:CU-Boulder/Notebook/Phage_Team&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/Phage_Team&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/Phage_Team&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:CU-Boulder/Notebook/Phage_Team" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:CU-Boulder/Notebook/Phage Team</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><HEADER id="topNav"><NAV id="navContainer"><UL class="navLinks"><LI class="navSingleLink"><A href="https://2014.igem.org/Team:CU-Boulder">HOME</A></LI><LI><A>PROJECT</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Getting Started</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Motivation">Motivation</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Project/Modeling">Modeling</A></LI></UL></NAV></HEADER></DIV></DIV></DIV><LI><A>ACHIEVEMENTS</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Our Team Achievements</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Biobricks">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Medal">Medal Requirements</A></LI><LI><A>NOTEBOOK</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">At the Bench</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Phage_Team">Phage Delivery</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Induced">Induced CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9_Team/Constitutive">Constitutive CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/CC9Phage_Team">Phage CRISPR-Cas9</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Measurement">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Protocols">Protocols</A></LI><LI><A>HUMAN PERSPECTIVE</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">Reaching Out</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Human_Practices">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Notebook/Safety">Ethics &amp; Safety</A></LI><LI><A>TEAM</A><DIV class="navDropMenu"><DIV class="container_12"><DIV class="grid_3"><UL><SPAN class="menuHeader">MI6</SPAN></UL></DIV></DIV></DIV></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:CU-Boulder/Acknowledgments">Acknowledgments</A></LI><P><B>Phage Delivery</B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_1"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 1</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 2</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_3"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 3</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week_4"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week_5"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 5</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Week_6"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 6</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Week_7"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Week 7</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Week_8"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Week 8</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Week_9"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Week 9</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Week_10"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Week 10</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Week_11"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Week 11</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Week_12"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Week 12</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Week_13"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Week 13</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Week_14"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Week 14</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Week_15"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Week 15</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Week_16"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Week 16</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Week_17"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Week 17</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Week_18"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Week 18</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Week_19"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Week 19</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Week_20"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Week 20</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H2><P><B>Notes:</B> Unless stated otherwise, all gels contain 2-log ladder
</P><P><B>5/9</B></P><UL><LI>Obtained BW23115 KanR cells- BW23115 cells that had their native CRISPR-Cas system knocked out by the insertion of a Kanamycin resistance gene
</LI></UL><DL><DD>-Will also be called BW23115 or BW
</DD><DD>-Conjugated BW23115 KanR cells with contain F’ notation (ex. BWF’)
</DD></DL><UL><LI>Obtained ER2738 cells that contain the F’ episome (no changes from NEB sample)
</LI></UL><DL><DD>-Will also be called ER. Assume that all ER samples contain the F’ episome
</DD><DD>-Streaked sample onto LB+Tet (20ug/mL) to select for colonies containing F’ episome
</DD></DL><P><B>5/10</B></P><UL><LI>Did receive colonies from 5/9 selection
</LI></UL><H2><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H2><P><B>5/12</B></P><UL><LI>Need to conjugate BW23115 KanR cells with the F’ episome
</LI></UL><DL><DD>-Set up overnight cultures of ER2738 and BW23115 KanR
</DD><DD>-When mixed, ER2738 will donate it’s F’ episome and BW23115 KanR will receive the F’ episome. F’ episome confers Tetracycline resistance
</DD></DL><P><B>5/13</B></P><UL><LI>Started M13 Amplification: Amplify M13 phage using the M13K07 Helper Phage
</LI></UL><DL><DD>-Let precipitated in NaCl/PEG solution overnight
</DD><DD>-Possible sources of error
</DD><DD><UL><LI>Did not sterilize 2.5M NaCl/20% PEG-8000 solution
</LI><LI>Added 4-fold PEG solution
</LI></UL></DD><DD>Compensated by adding more LB
</DD></DL><UL><LI>During precipitation, put sample in -20C for 30 minutes before realizing mistake and moving to it to 4C. Sample partially froze
</LI></UL><UL><LI>Conjugated BW23115 with F’ episome
</LI></UL><DL><DD>-Added 1mL BW23115 to 1mL ER2738 overnight culture
</DD><DD>-Incubated at 37C for 30 minutes, shaking
</DD><DD>-Plated on LB+Kan(50ug/mL)+Tet(20ug/mL)
</DD><DD><UL><LI>To select for BW cells that took the F’ episome (containing Tet resistance)
</LI></UL></DD></DL><P><B>5/14</B></P><UL><LI>Finished the M13 Amplification
</LI></UL><DL><DD>-Visualized product on UV-vis. There was a tall spike at 269nm indicating that DNA was present. Did not test at 320nm.
</DD></DL><UL><LI>Results of BW23115 Conjugation
</LI></UL><DL><DD>-Many colonies indicating successful conjugation of F’ episome into BW23115
</DD><DD>-Set up overnight to make freeze down tomorrow
</DD></DL><UL><LI>Set up overnight of ER2738 to make chemically competent tomorrow
</LI></UL><P><B>5/15</B></P><UL><LI>Made freeze down of BW23115 KanR F’
</LI></UL><DL><DD>-BW23115 E. coli strain with Kanamycin resistance gene inserted into genome and with F’ episome
</DD></DL><UL><LI>Made chemically competent ER2738 cells
</LI><LI>Transformation of Litmus28i (from NEB) into chemically competent ER2738 cells
</LI></UL><DL><DD>-Added 1ul Litmus28i plasmid to 40ul competent cells
</DD><DD>-Plated on LB + Amp(100ug/mL)
</DD><DD>-Purpose: To make M13 phage that package Litmus28i DNA. Need phagemid (Litmus28i) DNA in infectable cells (cells containing F’ episome) to introduce M13K07 Helper Phage and make phage. 
</DD></DL><P><B>5/16</B></P><UL><LI>Results of 5/15 transformation
</LI></UL><DL><DD>-No growth for No DNA control
</DD><DD>-Many colonies for sample
</DD></DL><H2><SPAN class="mw-headline" id="Week_3">Week 3</SPAN></H2><P><B>5/19</B></P><UL><LI>M13 Amplification to isolate M13-Litmus28i phage
</LI></UL><DL><DD>-Cells: ER2738 cells containing Litmus28i phagemid
</DD><DD>-Helper Phage: M13K07
</DD><DD>-Not much phage was precipitated
</DD></DL><UL><LI>Set up overnight culture of ER2738 to infect tomorrow
</LI></UL><P><B>5/20</B></P><UL><LI>Infected ER2738 cells with M13-Litmus28i phage
</LI></UL><DL><DD>-Plated only on Ampicillin(100ug/mL)  (should have also plated on kanamycin)
</DD><DD>-Infected for 4-5 hours-&gt; should have only infected for 30 minutes maximum. This extra time gives the cells that were infected with M13-M13K07 the time to produced M13-M13K07 phage and reinfect
</DD></DL><P><B>5/21</B></P><UL><LI>Results from M13-Litmus28i infection of ER2738
</LI></UL><DL><DD>-Solid lawn of growth for diluted and non-diluted
</DD><DD>-Also sickly looking growth
</DD></DL><UL><LI>Set up overnights
</LI></UL><DL><DD>-ER2738 cells containing Litmus28i for freeze down
</DD><DD>-BW23115 with F’ episome to make chemically competent cells 
</DD><DD>-ER2738 to redo infection
</DD></DL><P><B>5/22</B></P><UL><LI>Tested absorbance of phage produced through M13 amplification on 5/19
</LI></UL><DL><DD>-Low absorbance of 0.018 at 269nm but no detection at wavelength 320nm
</DD><DD>-Decided to redo M13 amplification
</DD></DL><UL><LI>Made chemically competent BW23115 with f-episome
</LI><LI>Made freeze down of ER2738 containing Litmus28i
</LI><LI>Set up overnight of ER2738 containing Litmus28i to redo M13 amplification tomorrow
</LI></UL><P><B>5/23</B></P><UL><LI>Protocol switch to make phage using phagemid
</LI></UL><DL><DD>-“M13 Amplification” protocol should only be used to make more M13-M13K07, not to make M13 phage containing a different phagemid
</DD><DD>-Switched to new protocol (“Use of M13K07 Helper Phage for isolation of single stranded phagemid DNA” by NEB. Made modifications (see our protocols) to isolate phage rather than single-stranded DNA)
</DD><DD>-Making phage….
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Helper phage
</TH><TH> Phagemid
</TH><TH> Cells
</TH><TH> Notes
</TH></TR><TR><TD> M13K07
</TD><TD> None
</TD><TD> ER2738
</TD><TD> Make more M13-M13K07
</TD></TR><TR><TD> M13K07
</TD><TD> Litmus28i
</TD><TD> ER2738
</TD><TD> Test packaging of Litmus28i
</TD></TR></TBODY></TABLE><UL><LI>Made fresh antibiotics
</LI></UL><DL><DD>-Chloramphenicol (34 ng/mL)
</DD><DD><UL><LI>1.44g chloramphenicol into 42mL EtOH
</LI></UL></DD></DL><DL><DD>-Ampicillin (50 ng/mL)
</DD><DD><UL><LI>4g ampicillin into 80mL mili-Q H2O
</LI></UL></DD></DL><P><B>5/24</B></P><UL><LI>Isolated phage using new protocol
</LI></UL><DL><DD>-Resuspended pellet in 200ul TBS and 200ul 30% glycerol
</DD><DD>-Measured absorbance with UV-vis
</DD><DD><UL><LI>concentration (phage/mL) = 6x10^16 x (A269-A320)/ (#of base pairs in the phage genome)
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Abs (269nm)
</TH><TH> Abs (320nm)
</TH><TH> Genome size
</TH><TH> Concentration (phage/mL)
</TH></TR><TR><TD> M13-M13K07
</TD><TD> 0.721
</TD><TD> 0.060
</TD><TD> 4.57 x10^12
</TD></TR><TR><TD> M13-Litmus28i
</TD><TD> 0.250
</TD><TD> 0.028
</TD><TD> 2823
</TD><TD> 4.72 x10^12
</TD></TR></TBODY></TABLE><UL><LI>Infect ER2738 cells with M13-Litmus28i
</LI></UL><DL><DD>-Wanted 1:10 phage:cell ratio. Math….
</DD><DD><UL><LI>At 1 OD (e.coli), cell/mL = 5x10^8
</LI><LI>5x10^7 phage * (1mL/4.72x10^12 phage) = 0.011ul phage
</LI></UL></DD></DL><UL><LI>Set up overnights
</LI></UL><DL><DD>-ER2738 for infection with M13-Litmus28i
</DD><DD>-BW23115 F’ for infection with M13-Litmus28i to test infectivity of conjugated strain
</DD></DL><H2><SPAN class="mw-headline" id="Week_4">Week 4</SPAN></H2><P><B>5/25</B></P><UL><LI>Infect ER and BWF’ cells with M13-Litmus28i
</LI></UL><DL><DD><UL><LI>Made 5mL culture of ER and BW that was at 1 OD
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> OD
</TH><TH> mL sample for 1OD in 5mL
</TH><TH> mL LB to 5mL
</TH></TR><TR><TD> ER2738
</TD><TD> 2.5
</TD><TD> 2 mL
</TD><TD> 3 mL
</TD></TR><TR><TD> BW23115
</TD><TD> 2.0
</TD><TD> 2.5 mL
</TD><TD> 2.5 mL
</TD></TR></TBODY></TABLE><DL><DD>-Based on calculations from 5/24, we needed to add 0.011 ul phage per 1 mL of cells at 1 OD. This equates to 0.055 ul of phage into 5 mL cells; therefore we made a 1:10 dilution so we could add 0.5ul. Unfortunately, the pipet would not take up 0.5ul so we added 0.8ul of M13-Litmus28i phage
</DD><DD>-Grew the cells for 20 minutes at 37C
</DD><DD>-Plated 300ul onto Kanamycin (50ug/mL) and 300ul onto Ampicillin (100ug/mL) for each sample
</DD><DD><UL><LI>Incubated overnight at 37C
</LI></UL></DD></DL><UL><LI><B>Note:</B> During the production of phage, the phagemid SHOULD be packaged preferentially over the Helper Phagemid but some Helper Phagemid will still be packaged. We plated on Amp to select for cells that were infected with phage containing Phagemid. We plated on Kan to select for cells that were infected with phage containing Helper Phagemid. This allows us to compare the packaging efficiency of Helper Phagemid: Phagemid.
</LI></UL><P><B>5/26</B></P><UL><LI>Results from 5/25 infection with M13-Litmus28i
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Result
</TH><TH> Significance
</TH></TR><TR><TD> ER2738 on Amp
</TD><TD> Lawn
</TD><TD> Litmus28i phagemid was successfully packaged into the M13 phage and is infectable
</TD></TR><TR><TD> ER2738 on Kan
</TD><TD> 100-200 colonies
</TD><TD> Some M13 helper phage is packaged into the M13 phage but at a much lower rate than Litmus28i
</TD></TR><TR><TD> BW23115 on Amp
</TD><TD> Lawn
</TD><TD> BW23115 is ‘equally’ infectable by M13 as ER2738
</TD></TR><TR><TD> BW23115 on Kan
</TD><TD> Lawn
</TD><TD> BW23115 contains Kan resistance in its genome so this tells us nothing
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Conclusions:
</LI></UL></DD><DD>-Cells grew on Ampicillin; therefore, Litmus28i phagemid was successfully packaged into M13 phage. 
</DD><DD>-For ER2738 samples, there was significant growth on Ampicillin compared to Kanamycin; therefore, Litmus28i phagemid is packaged preferentially over M13K07 Helper Phagemid
</DD><DD>-M13-Litmus28i retains its infectivity of cells containing the F’ episome
</DD></DL><UL><LI>Because we received lawns, we have to redo the infection and plate less cells so we can calculate the uptake ratio between the phagemid and helper phage based on the number of colonies
</LI></UL><UL><LI>Started 50 mL overnight of K12 ER2738 and BW23115
</LI></UL><P><B>5/27</B></P><UL><LI>Redo the infection done on 5/25
</LI></UL><DL><DD>-Infectable cells: ER2738 and BW23115
</DD><DD><UL><LI>Plated non-infected samples of each (non-diluted) to check for contaminants
</LI></UL></DD></DL><DL><DD>-Diluted M13-Litmus28i (1) phage by a factor of 10. Added 5.5ul to each sample
</DD><DD>-Grew samples for 20 minutes at 37C, 250rpm
</DD><DD>-Plated 100ul of onto an Ampicillin (100ug/mL) plate and onto a Kanamycin (50ug/mL) plate. Incubated overnight at 37C.
</DD><DD><UL><LI>Dilutions = 1:10; 1:100; and 1:1000
</LI></UL></DD></DL><P><B>5/28</B></P><UL><LI>Results from 5/27
</LI></UL><DL><DD>-Controls were as expected
</DD><DD><UL><LI>No growth for ER2738 non-infected grown on Amp, ER2738 non-infected grown on Kan, or BW23115 non-infected grown on Amp
</LI><LI>Growth for BW23115 non-infected grown on Kan (BW23115 has Kan R in genome)
</LI></UL></DD></DL><DL><DD>-Many colonies were received for all dilutions (1:10, 1:100, and 1:1000) of the following
</DD><DD><UL><LI>ER2738 infected and plated on Amp
</LI><LI>BW23115 infected and plated on Amp
</LI><LI>BW23115 infected and plated on Kan
</LI></UL></DD></DL><DL><DD>-Many (100s to 1000?) colonies grew on 1:10 and 1:100 dilutions of ER2738. 50-100 colonies grew on the 1:1000 dilution of ER2738
</DD><DD><UL><LI>Compare this to the 100-200 colonies that grew from 2/25 infection (which was 300ul non-diluted, infected cells)
</LI><LI>Reasons for increased yield
</LI></UL></DD><DD><UL><LI>Added too much phage?
</LI><LI>volume changed between experiment (5mL to 50mL)
</LI><LI>Overnight culture may not have been saturated. If still in log phase, the cells would continue to grow
</LI></UL></DD></DL><UL><LI>Made 50mL O/N cultures of ER2738 and BW23115 so we can repeat the infection tomorrow and plate further dilutions starting at 1:1000
</LI></UL><DL><DD>-Carry out infection in 5mL and 50mL to test volume effect?
</DD></DL><P><B>5/29</B></P><UL><LI>Measured OD of overnights
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> OD
</TH><TH> mL to have .1OD in 50mL
</TH><TH> mL to have 1OD in 5mL
</TH></TR><TR><TD> K12 ER2738
</TD><TD> 3.0
</TD><TD> 1.7
</TD><TD> 1.7
</TD></TR><TR><TD> BW23115
</TD><TD> 2.9
</TD><TD> 1.7
</TD><TD> 1.7
</TD></TR></TBODY></TABLE><UL><LI>Experiment 1: Infect cells using same method as 5/25 (in a 5mL culture)
</LI></UL><DL><DD>-Started with 1OD cells in 5mL
</DD><DD>-Added about 0.7ul (inaccuracies in pipet) of 1:10 diluted M13-Litmus28i phage
</DD><DD>-Incubated (rotating) for 20 minutes
</DD><DD>-Made 1:1,000 and 1:10,000 dilutions
</DD><DD>-Plated 100ul on Ampicillin (100ug/mL) plates and on Kanamycin (50ugmL) plates
</DD><DD><UL><LI>Included non-infected samples diluted by 1:1000
</LI></UL></DD><DD><UL><LI>This negative control can be used for Experiment 2 since the non-infected parent solution is the same
</LI></UL></DD></DL><DL><DD>-Incubate overnight at 37C
</DD></DL><UL><LI>Experiment 2: Infect cells using protocol from “Eliminating helper phage from phage display”
</LI></UL><DL><DD>-Diluted O/Ns to OD of 0.1 in 50 mL culture
</DD><DD>-Grew samples until of OD of ER2728 = 0.59 and the OD of BW23115 = 0.60
</DD><DD><UL><LI>Missed OD of 0.5 mark, but the two samples are close to each other
</LI></UL></DD></DL><DL><DD>-Chilled samples on ice for 30 minutes
</DD><DD>-Warmed in incubator for 35 minutes (should have been 30)
</DD><DD>-Amount of phage. Rather than use 1:1 as mentioned in protocol, we used multiplicity of 1:10 (phage:cell)
</DD><DD><UL><LI>Added 3.3ul of 1:10 diluted M13-Litmus28i (1) phage
</LI></UL></DD><DD><UL><LI>(On 5/27 we added 5.5ul of diluted phage to 50mL of cells at OD of 1. Our cells were at OD of .6; therefore, 5.5*.6 = 3.3ul)
</LI></UL></DD></DL><DL><DD>-Incubated for 30 minutes at 37C, not shaking
</DD><DD><UL><LI>We later change this to shaking
</LI></UL></DD></DL><DL><DD>-Dilutions
</DD><DD><UL><LI>1:1,000; 1:5,000; 1:10,000; 1:50,000; 1:100,000; 1:1,000,000
</LI><LI>Plated ER2738 and BW23115 on Ampicillin (100ug/mL)
</LI><LI>Plated ER2738 on Kanamycin (50ug/mL)
</LI></UL></DD></DL><UL><LI>Experiment 3: Growth Test (for growth curve)
</LI></UL><DL><DD>-We were concerned by the low OD of the Overnights from the last few days. Wanted to be sure that 2.0-3.0 was not still in log phase. Cultures looked saturated but the OD seemed low.
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Time
</TH><TH> Elapsed time (min)
</TH><TH> ER2738 (no antibiotic)
</TH><TH> ER2738 (Tetracycline (20ug/mL))
</TH><TH> BW23115 (with F’ episome) no antibiotic
</TH></TR><TR><TD> 10:08
</TD><TD> 0
</TD><TD> 0.1
</TD><TD> 0.1
</TD><TD> 0.1
</TD></TR><TR><TD> 11:10
</TD><TD> 62
</TD><TD> 0.24
</TD><TD> 0.21
</TD><TD> 0.21
</TD></TR><TR><TD> 12:00
</TD><TD> 112
</TD><TD> 0.49
</TD><TD> 0.45
</TD><TD> 0.44
</TD></TR><TR><TD> 13:00
</TD><TD> 172
</TD><TD> 1.00
</TD><TD> 0.93
</TD><TD> 0.98
</TD></TR><TR><TD> 14:15
</TD><TD> 217
</TD><TD> 1.29
</TD><TD> 1.21
</TD><TD> 1.31
</TD></TR><TR><TD> 15:40
</TD><TD> 302
</TD><TD> 2.1
</TD><TD> 1.8
</TD><TD> 2.3
</TD></TR><TR><TD> 16:47
</TD><TD> 369
</TD><TD> 2.5
</TD><TD> 1.9
</TD><TD> 3.4
</TD></TR><TR><TD> 18:05
</TD><TD> 447
</TD><TD> 2.6
</TD><TD> 2.2
</TD><TD> 2.5
</TD></TR><TR><TD> 19:00
</TD><TD> 502
</TD><TD> 2.9
</TD><TD> 2.3
</TD><TD> 2.5
</TD></TR><TR><TD> 20:00
</TD><TD> 562
</TD><TD> 3.0
</TD><TD> 2.6
</TD><TD> 2.8
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>The time point at 16:47 (369 minutes elapsed) for BW23115 conjugated (without antibiotics) is most likely an error. It has been removed from the growth plot
</LI></UL></DD></DL><UL><LI>Other
</LI></UL><DL><DD>-Made Amp and Kan plates (1 sleeve of each)
</DD><DD>-Made 50mL O/N of ER2738 and BW23115F’ in case we need further dilutions
</DD><DD>-Made 5mL O/N of ER2738, BW23115F’, and BW23115 (without F’ episome) to make chemically competent tomorrow
</DD><DD><UL><LI>Did not have plate of BW23115 (without F’ episome) so used freeze down. Hoping to get O/N of a picked colony from CRISPR Team tomorrow morning
</LI></UL></DD></DL><P><B>5/30</B></P><UL><LI>Made chemically competent cells of…
</LI></UL><DL><DD>-ER2738
</DD><DD>-BW23115F’ (conjugated with F’ episome)
</DD><DD>-BW23115 (not conjugated- without F’ episome)
</DD><DD><UL><LI>Culture started from plate
</LI></UL></DD></DL><DL><DD>-BW23115* not conjugated (without F’ episome)
</DD><DD><UL><LI>Culture started from freeze down
</LI></UL></DD></DL><UL><LI>Results from infections
</LI></UL><DL><DD>-Negative Controls (cells were not infected; cells were diluted 1:1000)
</DD></DL><DL><DD>-Results from Experiment 1 (5/29)
</DD></DL><DL><DD>-Results from Experiment 2 (5/29)
</DD></DL><UL><LI>Math
</LI></UL><DL><DD>-If there are 5.00E+8 cells in 1mL of culture at OD of 1, then in 1mL of culture at OD of 0.59, there are 2.95E+8 cells. In a 50mL culture at OD of 0.59, there are 1.48E+10 cells.
</DD><DD>-We added 3.3ul (0.0033mL) phage at concentration 4.62E+11 phage/mL which amounts to 1.52E+9 total phage
</DD><DD>-Assuming that 1 phage infects 1 bacterium, we can assume that 1.52E+9 bacterial have the potential to be infected in the 50mL culture
</DD><DD>-We plated 100ul of culture at various dilutions. If not diluted, the number of cells that can be potentially infected in 0.1mL equals 1.52E+9/500, or 3.05E+06 cells. We then accounted for the dilutions (for 1:1000 dilution, we divided 3.05E+06 by 1000 to receive 3.05E+03)
</DD><DD>-The following table contains the number of cells with the potential to be infected assuming a 100% infectivity rate by M13 phage and that 1 cell is infected only once.
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Dilution
</TH><TH> Potential infected cells
</TH><TH> Colonies on Amp
</TH><TH> % Potential (Amp)
</TH><TH> Colonies on Kan
</TH><TH> % Potential(Kan)
</TH><TH> Kan:Amp
</TH></TR><TR><TD> 1:1000
</TD><TD> 1.52E+06
</TD><TD> 476
</TD><TD> 15.62%
</TD><TD> 15
</TD><TD> 0.820%
</TD><TD> 1:19.04
</TD></TR><TR><TD> 1:5000
</TD><TD> 3.05E+05
</TD><TD> 131
</TD><TD> 21.49%
</TD><TD> 13
</TD><TD> 2.133%
</TD><TD> 1:10.08
</TD></TR><TR><TD> 1:10000
</TD><TD> 1.52E+05
</TD><TD> 93
</TD><TD> 30.51%
</TD><TD> 5
</TD><TD> 1.640%
</TD><TD> 1:18.60
</TD></TR><TR><TD> 1:50000
</TD><TD> 3.05E+04
</TD><TD> 17
</TD><TD> 44.29%
</TD><TD> 0
</TD><TD> 0.000%
</TD></TR></TBODY></TABLE><UL><LI>Conclusions from infections
</LI></UL><DL><DD>-Results between and within the three trials are inconsistent. For example, the number of colonies received in experiments 1 and 2 from 5/29 differ greatly. Due to the differences in protocol, variation was expected but not to this extent. 
</DD><DD>-Our dilutions did not yield the expected 10 fold (or 5 fold) decrease in growth that was expected.
</DD><DD>-Plates from 5/29 could be plated better to reduce dense areas of growth and growth around the rim.
</DD><DD>-Though the experiment contained many errors we can say that the phagemid (Litmus 28i) is preferentially packaged compared to the helper phage (M13K07) but not to the degree we expected.
</DD><DD>-Could receive increased occurrences of cells containing M13k07 due to infection, phage production, further infection
</DD></DL><H2><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H2><P><B>6/2</B></P><UL><LI>Tested chemically competent cells through transformation
</LI></UL><DL><DD>-Are cells contaminated?
</DD><DD>-Are cells competent?
</DD></DL><UL><LI>The samples for transformation
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  #
</TH><TH> Cells (Tube label)
</TH><TH> DNA (Tube label)
</TH><TH> Resistance before transformation
</TH><TH> Resistance after transformation
</TH></TR><TR><TD> 1
</TD><TD> K12 ER2738 5/20
</TD><TD> p110+RBS (2) 4/23
</TD><TD> Tet
</TD><TD> Tet, Chlor
</TD></TR><TR><TD> 2
</TD><TD> BW (-f) 5/30
</TD><TD> p110+RBS (2) 4/23
</TD><TD> Kan
</TD><TD> Kan, Chlor
</TD></TR><TR><TD> 3
</TD><TD> BW f-ep comp 5/22
</TD><TD> p110+RBS (2) 4/23
</TD><TD> Kan, Tet
</TD><TD> Kan, Tet, Chlor
</TD></TR><TR><TD> 4
</TD><TD> BW (+f) 5/30
</TD><TD> p110+RBS (2) 4/23
</TD><TD> Kan, Tet
</TD><TD> Kan, Tet, Chlor
</TD></TR><TR><TD> 5
</TD><TD> *BW23115 5/30
</TD><TD> p110+RBS (2) 4/23
</TD><TD> Kan
</TD><TD> Kan, Chlor
</TD></TR><TR><TD> 2B
</TD><TD> K12 ER2738
</TD><TD> 2B [from dis. kit]
</TD><TD> Tet
</TD><TD> Tet, Chlor
</TD></TR><TR><TD> 2P
</TD><TD> BW f-ep comp 5/22
</TD><TD> 2P [from dis. kit]
</TD><TD> Kan, Tet
</TD><TD> Kan, Tet, Chlor
</TD></TR></TBODY></TABLE><P><B>6/3</B></P><UL><LI>Results from 6/2 Transformation
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Growth on Chlor
</TH><TH> Growth on Kan
</TH><TH> Growth on Kan+Tet
</TH><TH> Growth on Amp
</TH></TR><TR><TD> 1 N
</TD><TD> X
</TD><TD> X
</TD><TD> X
</TD><TD> X
</TD></TR><TR><TD> 2 N
</TD><TD> X
</TD><TD> +
</TD><TD> X
</TD><TD> X
</TD></TR><TR><TD> 3 N
</TD><TD> X
</TD><TD> +
</TD><TD> +
</TD><TD> X
</TD></TR><TR><TD> 4 N
</TD><TD> X
</TD><TD> +
</TD><TD> +
</TD><TD> X
</TD></TR><TR><TD> 5 N
</TD><TD> X
</TD><TD> +
</TD><TD> X
</TD><TD> X
</TD></TR><TR><TD> 1
</TD><TD> +
</TD><TD> X
</TD><TD> X
</TD></TR><TR><TD> 2
</TD><TD> +
</TD><TD> +
</TD><TD> X
</TD></TR><TR><TD> 3
</TD><TD> +
</TD><TD> +
</TD><TD> +
</TD></TR><TR><TD> 4
</TD><TD> +
</TD><TD> +
</TD><TD> +
</TD></TR><TR><TD> 5
</TD><TD> +
</TD><TD> +
</TD><TD> Many colonies but close to samp. 4
</TD></TR><TR><TD> 2P
</TD><TD> + (~100)
</TD><TD> +
</TD><TD> +
</TD></TR><TR><TD> 1- JW
</TD><TD> +
</TD><TD> X
</TD><TD> X
</TD></TR><TR><TD> 2- JW
</TD><TD> +
</TD><TD> +
</TD><TD> X
</TD></TR><TR><TD> 3- JW
</TD><TD> +
</TD><TD> +
</TD><TD> +
</TD></TR><TR><TD> 4- JW
</TD><TD> +
</TD><TD> +
</TD><TD> +
</TD></TR><TR><TD> 5-JW
</TD><TD> +
</TD><TD> +
</TD><TD> X
</TD></TR><TR><TD> 2B- JW
</TD><TD> + (24)
</TD><TD> X
</TD><TD> X
</TD></TR></TBODY></TABLE><UL><LI>The transformations with DNA from the well (B2 and P2) had lower efficiencies than those with DNA from a mini-prep. Most likely this is due to the differences in DNA concentration (p110+RBS (2) 4/23 was at 254.4ng/ul)
</LI><LI>Conclusions
</LI></UL><DL><DD>-None of the competent cells were contaminated
</DD><DD>-All of the competent cells are in fact, competent
</DD></DL><UL><LI>Set up O/N of DH5-alpha cells to make competent tomorrow
</LI></UL><P><B>6/4</B></P><UL><LI>Isolation of single-stranded phagemid DNA using M13K07
</LI></UL><DL><DD>-Added ER2738 colony to 50mL LB
</DD><DD><UL><LI>Plate was cold. Next time warm plate before pricking
</LI></UL></DD><DD><UL><LI>Best to use freshly grown plate
</LI></UL></DD></DL><DL><DD>-After 4 hours, OD was at 0.02. Waited 45 minutes and OD was at 0.08. Therefore, we infected at OD 0.08
</DD><DD>-Had started another culture when we did not think the first was growing. In incubator for about 1 hour. OD was 0.00. We infected anyway because last time it worked.
</DD><DD>-Let infection proceed for 60 minutes then added 70ul of Kanamycin to be a final concentration of 70ug/mL
</DD></DL><UL><LI>Primers came in to biobrick M13ori (packaging signal on Litmus28i)
</LI></UL><DL><DD>-Resuspended primers and diluted 1:10
</DD></DL><P><B>6/5</B></P><UL><LI>Isolated single-stranded M13K07 DNA
</LI></UL><DL><DD>-Final concentration = 5724 ng/ul (calculated from a 1:10 dilution)
</DD><DD>-For second sample in pair, we resuspended it in TE but did not proceed to DNA extraction teps
</DD><DD>-For the second culture we started 6/4, we resuspended pellet in TBS and glycerol to preserve the M13 phage. Measured absorbances (before glycerol was added)
</DD></DL><DL><DD> #1
</DD><DD>269 =&gt; 1.690A
</DD><DD>320 =&gt; 0.103A
</DD></DL><DL><DD> #2
</DD><DD>269 =&gt; 1.453
</DD><DD>320 =&gt; 0.059
</DD></DL><UL><LI>For our first biobrick, we wanted to isolate the M13 origin, a segment ~500bp that allows for packaging into the M13 phage. We tried to achieve this by biobrick assembly and by Gibson Assembly.
</LI></UL><DL><DD>-To biobrick M13 ori through biobrick assembly (the old-school way)
</DD><DD>PCR on Litmus 28i to amplify/biobrick M13ori
</DD><DD><UL><LI>Used primers Gem003 F &amp; R
</LI><LI>Diluted Litmus 28i DNA 1:10
</LI></UL></DD></DL><DL><DD>Digestion of p11+RBS (1) to digest pSB1C3 bb with EcoRI-HF and PstI-HF
</DD><DD>Ran samples on gel and gel extracted pieces. We recieved very low yields (out of range for nano drop)
</DD><DD><UL><LI>M13 ori: 4.0 ng/ul
</LI><LI>pSB1C3: 1.8 ng/ul
</LI></UL></DD></DL><DL><DD>Digested M13 ori fragment despite poor extraction yield with EcoRI-HF and PstI-HF
</DD><DD><UL><LI>Used 1.5x as much DNA as instructed based on inaccurate concentration
</LI></UL></DD></DL><DL><DD>-Gel extracted red rectangles
</DD><DD>-Ligation
</DD><DD><UL><LI>10hr @ 16C, 10min @ 65C, 4ever @ 4C
</LI></UL></DD></DL><UL><LI>To biobrick M13 ori through Gibson Assembly (the cool-kids way)
</LI></UL><DL><DD>-PCR on Litmus 28i
</DD><DD><UL><LI>Used primers Gem002 F &amp; R
</LI><LI>Diluted Litmus 28i DNA 1:10
</LI></UL></DD></DL><DL><DD>-PCR on pSB1C3 (p11+RBS (1))
</DD><DD><UL><LI>Used primers Gem001 F &amp; R
</LI><LI>Diluted pSB1C3 DNA 1:3
</LI></UL></DD></DL><P><B>6/6</B></P><UL><LI>Ran gel of PCR products from (6/5). Products will be used for Gibson Assembly
</LI></UL><DL><DD>-Recieved bands for pSB1C3 around 2000bp and M13ori around 500bp
</DD><DD>-No contamination in pSB1C3 PCR negative control
</DD><DD>-Band in M13ori negative control that is the same size as sample. Contaminated by sample?
</DD><DD>-Gel of PCR products #1 and #2 from 6/5
</DD></DL><DL><DD>1. pSB1C3 with promoter+RBS as insert. Amplified with Gem001
</DD><DD>2. No DNA control for Gem001
</DD><DD>3. M13ori amplified with Gem002 from Litmus28i
</DD><DD>4. No DNA control for Gem002
</DD></DL><UL><LI>Gibson Assembly
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Total Amount of Frag.
</TH><TH> .02-.5pmol
</TH><TH> 10 ul total
</TH></TR><TR><TD> Gibson Assembly MM (2x)
</TD><TD> 10 ul
</TD><TD> 10
</TD></TR><TR><TD> Dionized H2O
</TD><TD> 10-x
</TD></TR></TBODY></TABLE><DL><DD>-Diluted pSB1C3 and M13ori PCR products 1:10
</DD><DD>-Incubated 60min @ 50C
</DD><DD>-Also used provided pUC16 as positive control
</DD></DL><UL><LI>Transformation
</LI></UL><DL><DD>1. p110+RBS            Positive control
</DD><DD>2. No DNA            Negative control
</DD><DD>3. Cas9 from distribution kit    so we can have more
</DD><DD>4. Thaw and refreeze cells    Test competency of comp cells after thawed
</DD><DD>5. Not chem comp cells    Negative control for the above
</DD><DD>6. Ligation Product
</DD><DD>7. Gibson product
</DD><DD><UL><LI>7.2. Gibson product diluted 1:4
</LI></UL></DD></DL><DL><DD>8. Gibson positive control
</DD><DD><UL><LI>7.2. Gibson positive control diluted 1:4
</LI></UL></DD><DD>-For the Gibson product and the positive control, we transformed 2ul of product and 2ul of 1:4 diluted product. NEB recommends the first if using their competent cells and the second if using cells from other companies. Our cells are from NEB but we made them competent ourselves so we tried both ways
</DD><DD><UL><LI>Plated on Chlor at concentrations of 170, 85, and 33 ug/mL
</LI></UL></DD></DL><UL><LI>Primers came in
</LI></UL><DL><DD>-Resuspended and made 1:10 dilutions
</DD></DL><P><B>6/7</B></P><UL><LI>Results from 6/6 transformation
</LI></UL><DL><DD>1. Positive control
</DD><DD><UL><LI>Lots of growth, ~300-400 on 1:10 dilution
</LI></UL></DD></DL><DL><DD>2. No DNA negative control
</DD><DD><UL><LI>No growth
</LI></UL></DD></DL><DL><DD>3. Cas9 from distribution kit
</DD><DD><UL><LI>7 potential colonies (some are close to edges through) on non-diluted
</LI></UL></DD></DL><DL><DD>4. Thawed then refroze cells
</DD><DD><UL><LI>Looks like (1)
</LI></UL></DD></DL><DL><DD>5. Not chemically competent cells
</DD><DD><UL><LI>No growth
</LI></UL></DD></DL><DL><DD>6. Ligation product
</DD><DD><UL><LI>13 potential colonies (some are close to edge)
</LI></UL></DD></DL><DL><DD>7. Gibson Assembly Product
</DD><DD><UL><LI>170 -&gt; No colonies
</LI><LI>85 -&gt; No colonies
</LI><LI>33 -&gt; 3 specks
</LI></UL></DD></DL><DL><DD>7.2. Gibson Assembly product diluted 1:10
</DD><DD><UL><LI>170 -&gt; 1 speck
</LI><LI>85 -&gt; 3 colonies
</LI><LI>33 -&gt; 13 colonies
</LI></UL></DD></DL><DL><DD>8. Gibson positive control
</DD><DD><UL><LI>No colonies
</LI></UL></DD></DL><DL><DD>8.2. Gibson positive control diluted 1:4
</DD><DD><UL><LI>No colonies
</LI></UL></DD></DL><UL><LI>Made 6mL O/N cultures
</LI></UL><DL><DD>-4 from (3) cas9 plate
</DD><DD>See Constitutive CRISPR notebook for more info on these samples
</DD></DL><DL><DD>-7 from (6) Ligation product
</DD><DD>-5 from (7.2 [85]) Diluted Gibson product on 85 ug/mL Chlor
</DD><DD>-8 from (7.2 [33]) Diluted Gibson product on 33 ug/mL Chlor
</DD></DL><H2><SPAN class="mw-headline" id="Week_6">Week 6</SPAN></H2><P><B>6/8</B></P><UL><LI>Check colonies for correct constructs.
</LI></UL><DL><DD>-Mini-prepped all 24 O/Ns
</DD><DD><UL><LI>Yielded low concentrations for samples 12, 15, 19, and 22
</LI></UL></DD></DL><DL><DD>-Digested all with EcoRI and PstI (10ul reactions)
</DD><DD>-Ran results on gel
</DD><DD><UL><LI>All 4 cas9 samples had the expected bands of 2000 and 5000bp
</LI><LI>All 7 ligation products have expected bands of 2000 and ~570bp
</LI><LI>3 of 5 Gibson assemblies from 85ug/mL Chlor plate had expected bands of 2000 and ~500bp
</LI><LI>3 of 8 Gibson assemblies from 33ug/mL Chlor plate had expected bands of 2000 and ~500bp
</LI></UL></DD></DL><DL><DD>1-4: Cas9 from Stanford-Brown team
</DD><DD>5-11: pSB1C3-M13ori cloned through ligation
</DD><DD>12-24: pSB1C3-M13ori cloned through Gibson Assembly
</DD><DD><UL><LI>12-16: Grown with 85 ug/mL Chlor
</LI><LI>17-24: Grown with 33 ug/mL Chlor
</LI></UL></DD></DL><UL><LI>Conclusions from gel
</LI></UL><DL><DD>-We have cas9 safely in cells
</DD><DD>-Our ligation reactions successfully yielded M13ori on pSB1C3
</DD><DD>-Combined, we had a 46% success rate for the Gibson Assembly in yielding M13ori on pSB1C3
</DD><DD><UL><LI>The 4 samples that had the lowest concentration after being mini-prepped (12,15, 19, and 22) correlate with samples that had the correct band pattern
</LI></UL></DD></DL><UL><LI>We selected 4 samples from each (4 total between the two Gibson reactions) type
</LI></UL><DL><DD>-For non-Gibson Assembled samples
</DD><DD>Plated 25ul on 170ug/mL Chlor
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Our usual method
</TH><TH> Gibson Method
</TH></TR><TR><TD> 1. Thaw on ice
</TD><TD> 1. Thaw on ice
</TD></TR><TR><TD> 2. Transfer 40ul cells to tube
</TD><TD> 2. Transfer 50ul cell to tube
</TD></TR><TR><TD> 3. Add DNA. 1ul for mini-prep OR up to 10ul for ligation
</TD><TD> 3. Add 2ul to NEB cells OR 2ul of 1:4 diluted to other cells
</TD></TR><TR><TD> 4. Mix by pipet Let sit 30min on ice
</TD><TD> 4. Mix by pipet or flicking Let sit 30 min. on ice
</TD></TR><TR><TD> 5. Heat shock: 42C for 45s
</TD><TD> 5. Heat shock: 42C for 30s
</TD></TR><TR><TD> 6. Ice for 5 minutes
</TD><TD> 6. Ice for 2 min.
</TD></TR><TR><TD> 7. Transfer to culture tube; Add 200ul SOC
</TD><TD> 7. Add 950ul SOC to tube
</TD></TR><TR><TD> 8. Shake or rotate for 60-120min at 37C
</TD><TD> 8. Shake (250rpm) or rotate for 60 min. at 37C
</TD></TR><TR><TD> 9.
</TD><TD> 9. Warm plates to 37C
</TD></TR><TR><TD> 10. Plate 100ul onto selection plate
</TD><TD> 10. Plate 100ul onto plate
</TD></TR><TR><TD> 11. Incubate O/N @ 37C
</TD><TD> 11. Incubate O/N @ 37C
</TD></TR></TBODY></TABLE><DL><DD>-Added 6mL LB and Chlor at concentration of 170ug/mL to grow O/N
</DD></DL><UL><LI>For Gibson Assembled samples
</LI></UL><DL><DD>-Plated 25ul onto 170, 85, and 33ug/mL Chlor
</DD><DD>-Samples from 85ng/mL plate
</DD><DD>Transferred 100ul to new tube, added media, added Chlor at 170ng/mL
</DD></DL><DL><DD>-Samples from 33ng/mL plate
</DD><DD>Tranfered 100ul to new tubes, added media, added Chlor at 85ng/mL to one and 33ng/mL to the other
</DD></DL><UL><LI>Tomorrow, may send for sequencing and make freeze downs
</LI></UL><UL><LI>Transformation
</LI></UL><DL><DD>1. Positive control (p110+RBS diluted 1:10)
</DD><DD>2. Non-diluted Gibson product
</DD><DD>3. Gibson product diluted 1:4
</DD><DD>4. Gibson product diluted 1:10
</DD><DD>-Transformed each sample using our usual method and using the protocol given by Gibson
</DD></DL><DL><DD>-Due to not have plates ready before transformation, in step 4, the samples sat for about 50 minutes. Then in step 8, they both recovered for about 150 minutes. Though not specified in our protocol, we did warm the plates to 37C. In step 10 for our protocol, since we only added 200ul SOC and wanted to plate on 3 selection plates (see below), we only plated 50ul (except for the positive control). 
</DD><DD>-Plated on three concentrations of Chloramphenicol (33 ug/mL, 85 ug/mL, and 170 ug/mL) to determine the differences in yield due to differences in concentration.Obvious hypothesis: more colonies will grow on plates that have a lower concentration of chlor.
</DD></DL><P><B>6/9</B></P><UL><LI>Made chemically competent 5alpha cells with Dan and Alex from main campus
</LI></UL><DL><DD>-Waiting to hear results on competency
</DD></DL><UL><LI>Will eventually make phage containing CRISPR-Cas9 that targets Kanamycin resistance. M13K07 has Kanamycin resistance so we need to switch the resistance on the M13 genes.
</LI></UL><DL><DD>-PCR on pwp 2.po (plasmid that Sam gave us that contains the zeoR gene adjacent to ori) to amplify zeoR and ori. Zeo is on EM7 promoter
</DD><DD>-Primers: Gem008 R &amp; R
</DD><DD>-Anneal temp from NEBuilder: 63.7C
</DD><DD>-Extension time: 90s
</DD><DD>-Expected band size on gel: 1300bp
</DD><DD>-Used phusion polymerase
</DD></DL><UL><LI>PCR on M13K07 DNA to amplify M13 phage genes (also removes majority of M13 ori, all of KanR, and all of p15 ori)
</LI></UL><DL><DD>-Primers: Gem007 F &amp; R
</DD><DD>-Anneal temp from NEBuilder: 60.2C
</DD><DD>-Extension time: 4:30
</DD><DD>-Expected band size on gel: about 6000bp
</DD><DD>-Used phusion polymerase
</DD></DL><UL><LI>Freeze downs
</LI></UL><DL><DD>-Note: Phagemid 1C3 was the original name for ‘pSB1C3-M13ori’
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Top Label
</TH><TH> Side label
</TH><TH> What?
</TH></TR><TR><TD> Phagemid 1C3 6/9
</TD><TD> From Lig
</TD><TD> M13 ori inserted into 1C3 (biobricked); Done through ligation; Contains extra bases as spacer between biobrick prefrix/suffix and part for primer design
</TD></TR><TR><TD> Phagemid 1C3 6/9
</TD><TD> From Lig
</TD><TD> “ “
</TD></TR><TR><TD> Phagemid 1C3 6/9
</TD><TD> Gibson
</TD><TD> M13 ori inserted into 1C3 (biobricked); Done through Gibson cloning
</TD></TR><TR><TD> Phagemid 1C3 6/9
</TD><TD> Gibson
</TD><TD> “ “
</TD></TR></TBODY></TABLE><P><B>6/10</B></P><UL><LI>Ran gel of PCRs from 6/9
</LI></UL><DL><DD>1. Amplification of M13 genes from M13K07 (~6000bp)
</DD><DD>2. No DNA control for (1) amplification
</DD><DD>3. Amplification of Zeo resistance gene + plasmid ori (~1300bp)
</DD><DD>4. No DNA control for (3) amplification
</DD></DL><UL><LI>Gibson Assembly of above parts (did not gel extract)
</LI></UL><DL><DD>-Diluted the PCR products 1:10 then added 3ul of M13K07 genes product and 7ul of ZeoR+ori product
</DD><DD>-Incubated at 50C for 60 min.
</DD><DD>-Transformed Gibson Assembly product into 5alpha cells
</DD><DD><UL><LI>Used our usual protocol
</LI><LI>Add 2ul of DNA
</LI></UL></DD><DD>-In one sample, diluted DNA 1:4 and in the other, we diluted DNA 1:10
</DD></DL><UL><LI>Started Phage Amplification Protocol
</LI></UL><DL><DD>-ER2738 transformed with Litmus 28i
</DD><DD><UL><LI>Grew for ~2.5hr before reaching an OD of 0.04
</LI></UL></DD></DL><DL><DD>-ER2738 transformed with pSB1C3-M13ori (M13ori on pSB1C3)
</DD><DD><UL><LI>Grew for ~ 3.5hr before reaching an OD of 0.01, then in the next 1.5 hours, spiked to 0.19
</LI><LI>We gave up and went home, and will restart tomorrow
</LI></UL></DD></DL><UL><LI>Analyzed transformation results from  6/8
</LI></UL><P><B>6/11</B></P><UL><LI>Made chemically competent 5alpha cells
</LI><LI>Restarted Phage Amplification Protocol
</LI></UL><DL><DD>-Forgot to add phagemid antibiotic at start of growth. Added phagemid antibiotic when we added phage. Incubated for 90 minutes before adding Kanamycin (to select for cells that were infected by M13K07)
</DD><DD>-Phage is at concentration 4.57x10^12 phage/mL
</DD><DD>Protocol calls for final concentration of 1 x10^8 phage/mL
</DD><DD><UL><LI>(4.57 x10^12)*V = (1 x10^8)(50mL)
</LI><LI>V = 0.00109mL
</LI></UL></DD></DL><DL><DD>Added 1.1ul of phage
</DD></DL><UL><LI>Transformations
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>DNA
</TH><TH>Plate Selection
</TH></TR><TR><TD>No DNA control
</TD><TD>(all)
</TD></TR><TR><TD>Positive control (p110+RBS)
</TD><TD>(C)
</TD></TR><TR><TD>M13 genes + ZeoR ori
</TD><TD>(Z)
</TD></TR><TR><TD>M13 genes +ZeoR ori diluted 1:4
</TD><TD>(z)
</TD></TR></TBODY></TABLE><P><B>6/12</B></P><UL><LI>Transformation Results from 6/10 [Took ~36 hours to be clearly visible]
</LI></UL><DL><DD>No DNA control
</DD><DD>-Amp: 0        
</DD><DD>-Zeo (50ug/mL): 200 colonies
</DD></DL><DL><DD>M13 genes + ZeoR ori (1:4 dilution)        
</DD><DD>-Zeo (25ug/mL): 300
</DD><DD>-Zeo (50ug/mL): 200
</DD><DD>-Zeo (100ug/mL): 100
</DD></DL><DL><DD>M13 genes + ZeoR ori (1:10 dilution)        
</DD><DD>-Zeo (25ug/mL): 300
</DD><DD>-Zeo (50ug/mL): 150
</DD><DD>-Zeo (100ug/mL): 150
</DD></DL><UL><LI>Transformation Results for 6/11
</LI></UL><DL><DD>-Positive (p110+RBS) on Chloramphenicol: 500 colonies
</DD><DD>-No DNA on Amp: 0 colonies        Zeo (100 ug/mL): specks
</DD><DD>-No apparent growth on any other plate
</DD><DD><UL><LI>Realized later that we grew our samples on the wrong plates. Will repeat transformation today
</LI></UL></DD></DL><UL><LI>Transformation #1
</LI></UL><DL><DD>This morning there were no colonies on positive (p110+RBS) coltrol from 6/11 even though we observed fast growth in the past. Without waiting for colonies to appear, we started a control transformation
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Diluted
</TH><TH> Time at 42C
</TH></TR><TR><TD> p110+RBS
</TD><TD> No
</TD><TD> 45s
</TD></TR><TR><TD> p110+RBS
</TD><TD> 1:10
</TD><TD> 45s
</TD></TR><TR><TD> p110+RBS
</TD><TD> 1:10
</TD><TD> 30s
</TD></TR><TR><TD> p110+RBS
</TD><TD> 1:10
</TD><TD> No timer. ~43s
</TD></TR><TR><TD> No DNA control
</TD><TD> No
</TD><TD> 45
</TD></TR></TBODY></TABLE><UL><LI>Because ‘No DNA control 6/10’ yielded colonies, we researched Zeocin plates
</LI></UL><DL><DD>-According to Life Technologies (Invitrogen), Zeocin requires low salt medium and a pH of 7.5
</DD><DD>-Low Salt LB Medium (1L)
</DD><DD><UL><LI>Ingredients
</LI></UL></DD><DD>10g    Tryptone
</DD><DD>5g    NaCl
</DD><DD>5g    Yeast Extract
</DD></DL><DL><DD>-Mix ingredients
</DD><DD>-Adjust pH to 7.5 using NaOH (If go over, use HCl)
</DD><DD>-Add agar for plates at 15g/L. Autoclave
</DD><DD>-Thaw Zeocin on ice. Vortex
</DD><DD>-Add Zeocin to final concentration of 25ug/mL
</DD></DL><UL><LI>Transformation #2
</LI></UL><DL><DD>-Repeat of transformation on 6/11 but this time we will plate on the correct plates
</DD><DD>-Also remade Zeocin plates
</DD></DL><UL><LI>Finished isolation of M13 Litmus phage and M13 pSB1C3-M13ori phage
</LI></UL><DL><DD><UL><LI>Note: Phagemid 1C3 was the original name for ‘pSB1C3-M13ori’
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Phage Sample
</TH><TH> A269
</TH><TH> A320
</TH><TH> Concentration (phage/mL)
</TH></TR><TR><TD> Litmus phage (1)
</TD><TD> 0.181
</TD><TD> 0.034
</TD><TD> 3.12 x10^12
</TD></TR><TR><TD> Litmus phage (2)
</TD><TD> 0.227
</TD><TD> 0.047
</TD><TD> 3.83 x10^12
</TD></TR><TR><TD> Phagemid 1C3 phage (1)
</TD><TD> 0.101
</TD><TD> 0.020
</TD><TD> 1.87 x10^12
</TD></TR><TR><TD> Phagemid 1C3 phage (2)
</TD><TD> 0.126
</TD><TD> 0.021
</TD><TD> 2.42 x10^12
</TD></TR></TBODY></TABLE><DL><DD>phage/mL = 6x10^16 x (A269-A320)/ (#of base pairs in the phage genome)
</DD></DL><UL><LI>Set up 50mL O/N of K12 ER2738 (containing f-episome) for infection tomorrow with Litmus phage and pSB1C3-M13ori phage
</LI></UL><P><B>6/13</B></P><UL><LI>Transformation Results for 6/11
</LI></UL><DL><DD>-Several hundred colonies on Positive Control (p110+RBS)    on Chlor
</DD><DD>-No colonies for GA positive control                    on Amp
</DD><DD>-No colonies for M13ori + ZeoR mistakenly plated             on Amp
</DD><DD>-No colonies for No DNA control                    on Amp
</DD><DD>-100-ish colonies for No DNA control                    on Zeo
</DD><DD>-100-ish colonies for cas9+AmpR+gRNA mistakenly plated    on Zeo
</DD><DD>-These last 3 points suggest/confirm that Zeo plates are no good
</DD></DL><UL><LI>Transformation Results from 6/12 control test
</LI></UL><DL><DD>-Colonies grew is about equal amounts on all plates, including No DNA control
</DD><DD>Either plates don’t contain Chlor or competent cells are contaminated
</DD></DL><DL><DD>-Streaked competent cells onto new and old Chlor plates
</DD></DL><UL><LI>Infection test of ER2738: Is Litmus preferentially packaged over M13K07 Helper phage? Is pSB1C3-M13ori preferentially packaged?
</LI></UL><DL><DD>-Infect ER2738 with phage produced 6/12
</DD><DD><UL><LI>Phage should have packaged Litmus 28i phagemid or pSB1C3-M13ori
</LI><LI>Cells infected with phage packaging Litmus 28i will grow on Amp
</LI><LI>Cells infected with phage packaging pSB1C3-M13ori will grow on Chlor
</LI><LI>Cells infected with phage packaging M13K07 will grow on Kan
</LI></UL></DD></DL><DL><DD>-After we plate, we can count the colonies and calculate a ratio of Litmus28i: M13K07 or pSB1C3-M13ori:M13K07 packaging
</DD></DL><P><B>6/14</B></P><UL><LI>Results of contamination test (streaked competent cells onto new and old Chlor plates)
</LI></UL><DL><DD>-Colonies grew in low amounts on both plates. most likely the cells are contaminated
</DD></DL><UL><LI>Transformation results for 6/12
</LI></UL><DL><DD>-Many colonies for No DNA control                    on Zeo
</DD><DD>-Some colonies are turning pinkish-red
</DD></DL><DL><DD>-Many colonies for diluted and non-diluted M13 genes+ZeoR    on Zeo
</DD><DD>-Some colonies are turning pinkish-red
</DD></DL><UL><LI>Could the white colonies be the designed colonies and can we kill the red colonies with Zeocin before killing the white colonies (aka. Use high Zeocin concentrations to select for the correct construct)
</LI></UL><DL><DD>-Selected 1 red colony from No DNA Zeo control plate and 1 white colony from M13genes+ZeoR sample plate
</DD><DD>-Added 100ul H2O then divided amongst 5 culture tubes each (with 5mL of low-salt LB, pH 7.5)
</DD><DD>-Then added Zeocin to a final concentration of: 0, 25, 50, 75, 100ug/mL.
</DD></DL><UL><LI>Results of 6/13 infection
</LI></UL><DL><DD>-Litmus 28i infected cells
</DD><DD><UL><LI>on Kan: Individual colonies for 1:10, 1:100, and 1:1000. No colonies on 1:5000
</LI><LI>on Amp: lawn for 1:10, 1:100, and near lawn for 1:1000. Single colonies for 1:5000
</LI></UL></DD></DL><DL><DD>-pSB1C3-M13ori infected cells
</DD><DD><UL><LI>on Kan: Same as Litmus 28i samples on Kan
</LI><LI>on Chlor: Lawn for all dilutions. Must discredit due to recent Chlor contamination
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H2><P><B>6/15</B></P><UL><LI>Results of Zeocin experiment on 6/14
</LI></UL><DL><DD>-Healthy growth for both at 0ug/mL Zeo
</DD><DD>-No growth for white colony with any Zeo
</DD><DD>-Good growth for red colony even at 100ug/mL Zeo
</DD><DD>-This suggests that the red colonies are naturally resistant to Zeocin. Also, our plates must not contain active Neocin. We are sure that we are adding enough. Possibly, we add it while the media is too hot or leave the plates at room temperature (decondensing) for too long and this deactivates the antibiotic? Perhaps we are not attaining the correct pH
</DD></DL><UL><LI>In light of recent contamination problems on both Chlor and Zeo, we made new competent cells
</LI></UL><DL><DD>-5alpha
</DD><DD>-BW23115
</DD><DD>-BW23115 conjugated (contains f-episome)
</DD></DL><P><B>6/16</B></P><UL><LI>Made new Chlor antibiotic. Made to Chlor plates
</LI><LI>Contamination Test: Streaked Zeo and new Chlor plates each with
</LI></UL><DL><DD>-Colony from Zeo contaminated plate
</DD><DD>-Colony from Chlor contaminated plate
</DD><DD>-Non-transformed OLD chemically competent cells
</DD><DD>-Non-transformed NEW chemically competent cells
</DD></DL><UL><LI>Redid bacterial infection
</LI></UL><DL><DD>-For both samples, we used ER2738 (not from the same ‘O/N’ (also, not a real O/N))
</DD><DD>-Infected one sample with isolated ‘Litmus28i phage’ and one with ‘pSB1C3-M13ori’
</DD><DD>-Alterations to protocol
</DD><DD><UL><LI>Did not started from saturated O/N. Started each with a colony, waited several hours until at OD ~1. We then added these cells to fresh 50mL LB to have an OD of 0.1.
</LI><LI>Missed the 30 minute infection mark. Infected for ~45 minutes.
</LI></UL></DD></DL><DL><DD>-Plated on Kan and Amp (Litmus28i sample) or Chlor (pSB1C3-M13ori sample) at dilutions of
</DD><DD><UL><LI>1:100, 1:1k, 1:10k, and 1:100k
</LI></UL></DD></DL><P><B>6/17</B></P><UL><LI>Q5 PCR to replace KanR with ZeoR in M13K07
</LI></UL><DL><DD>-Used recommendations
</DD><DD>-Unfortunately, I (Jo) don’t know the difference between tightening and loosening the thermocycler lid; therefore, our M13K07 sample to amplify the M13K07 genes evaporated. But that’s ok, because we were sick of Zeocin anyway and decided mid-PCR to not waste our time with a Gibson Assembly and transformation. Instead we will be using the Old-School method of digestion and ligation because let’s face it, it’s a classic (and Gibson sucks) =)
</DD></DL><UL><LI>Results from infection Test
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Litmus 28i
</TH><TH> 1:100
</TH><TH> 1:1k
</TH><TH> 1:10k
</TH><TH> 1:100k
</TH></TR><TR><TD> Amp
</TD><TD> ~6000?
</TD><TD> ~2056
</TD><TD> 377
</TD><TD> 36
</TD></TR><TR><TD> Kan
</TD><TD> 130
</TD><TD> 8
</TD><TD> 3
</TD><TD> 0
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH>  pSB1C3-M13ori
</TH><TH> 1:100
</TH><TH> 1:1k
</TH><TH> 1:10k
</TH><TH> 1:100k
</TH></TR><TR><TD> Chlor
</TD><TD> 8
</TD><TD> 1
</TD><TD> 0
</TD><TD> 0
</TD></TR><TR><TD> Kan
</TD><TD> 97
</TD><TD> 9
</TD><TD> 0
</TD><TD> 0
</TD></TR></TBODY></TABLE><DL><DD>-We consider the Litmus 28i to have been a success. The pSB1C3-M13ori …. not so much. Looking back at the Litmus28i and the M13K07, we noticed that the M13ori is facing the opposite direction as the plasmid ori. Ours faces the same direction as the plasmid ori. Because we are working with phagemids that are single stranded, we think that by flipping the M13ori, we may be able to recover functionality. We will also look into other reasons.
</DD></DL><UL><LI>Results from contamination test (6/16)
</LI></UL><DL><DD>-Old Chlor plate (6/12)
</DD><DD>-Non-transformed chemically competent cells
</DD><DD><UL><LI>Old: some colonies
</LI><LI>New: No colonies
</LI></UL></DD></DL><DL><DD>-New Chlor plate (6/16)
</DD><DD>-Colony from
</DD><DD><UL><LI>Zeo contaminated plate: No growth
</LI><LI>Chlor contaminated plate: Much growth
</LI></UL></DD></DL><DL><DD>-Non-transformed chemically competent cells
</DD><DD><UL><LI>Old: some colonies
</LI><LI>New: No growth
</LI></UL></DD></DL><DL><DD>-Zeo (25ug/mL, Low NaCl, pH 7.5) (6/12)
</DD><DD>-Colony from
</DD><DD><UL><LI>Zeo contaminated plate: Much growth
</LI><LI>Chlor contaminated plate: No growth
</LI></UL></DD></DL><DL><DD>-Non-transformed chemically competent cells
</DD><DD><UL><LI>Old: some specks
</LI><LI>New: some specks
</LI></UL></DD></DL><P><B>6/18</B></P><UL><LI>Ordered primers to…
</LI></UL><DL><DD>-biobrick M13ori (in other direction)
</DD><DD>-biobrick M13K07 genes
</DD></DL><P><B>6/19</B></P><UL><LI>Waited for primers
</LI><LI>Set up O/N cultures to test last infection (6/16) for colonies containing pSB1C3-M13ori
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Sample #
</TH><TH> Selection
</TH><TH> Dilution
</TH><TH> Presumably
</TH><TH> Notes
</TH></TR><TR><TD> 1-8
</TD><TD> Chlor
</TD><TD> 1:100
</TD><TD> pSB1C3-M13ori
</TD></TR><TR><TD> 9
</TD><TD> Chlor
</TD><TD> 1:1k
</TD><TD> pSB1C3-M13ori
</TD></TR><TR><TD> 10-14
</TD><TD> Chlor
</TD><TD> 1:1k
</TD><TD> pSB1C3-M13ori
</TD><TD> These colonies were not present on plate during initial counting  on 6/17
</TD></TR><TR><TD> 15-16
</TD><TD> Kan
</TD><TD> 1:1k
</TD><TD> M13K07
</TD></TR></TBODY></TABLE><DL><DD>-Selection and dilution refer to the plate. Cells were then grown under selection.
</DD><DD>-More colonies were seen on all Chlor plates. No new colonies appeared on Kan plates
</DD></DL><UL><LI>Tomorrow, we will mini-prep, digest, and run the samples on a gel to verify gene transfer.
</LI></UL><P><B>6/20</B></P><UL><LI>Primers are in!
</LI><LI>Cloning of pSB1C3-M13ori(New)
</LI></UL><DL><DD>-PCR to amplify M13ori (packaging signal) from Litmus 28i to be in the other direction
</DD><DD>-Primers (Gem011 F&amp;R) add cut sites to make part biobrick compatible
</DD><DD>-Was able to gel extract
</DD><DD><UL><LI>Band at the same size as sample….. contaminated primers?
</LI></UL></DD></DL><DL><DD>-Then digested with E and P
</DD></DL><DL><DD>-Digested pSB1C3 plasmid with E and P to linearize backbone
</DD><DD><UL><LI>Was able to gel extract
</LI></UL></DD></DL><DL><DD>1. Spill over from 2
</DD><DD>2 and 3. pSB1C3 digested with EcoRI and PstI
</DD><DD>-Insert &lt;100bp so cannot be seen
</DD></DL><DL><DD>4. M13ori
</DD></DL><UL><LI>Ligation of M13ori (packaging signal) into pSB1C3
</LI></UL><DL><DD>10h at 16C
</DD><DD>20m at 80C
</DD><DD>Hold at 4C
</DD></DL><UL><LI>PCR to amplify M13K07 genes from M13K07 DNA isolated from phage
</LI></UL><DL><DD>-Primers (Gem012 F&amp;R) add cut sites to make part biobrick compatible
</DD><DD>-Very light bands and not band might correlate with 6000pb but not enough resolution on gel to be certain. Bands were too light so did not extract
</DD><DD>-Set up PCR again using Q5 to run O/N
</DD><DD><UL><LI>Received only a feint, smudgy band that was too large. We did not bother with extracting the DNA
</LI></UL></DD></DL><UL><LI>Test of 6/19 O/Ns (14 that are presumably pSB1C3-M13ori and 2 that are presumably M13K07)
</LI></UL><DL><DD>-Mini-prepped all O/Ns
</DD><DD>-Digested all with Pst-I (common RE between pSB1C3-M13ori and M13K07)
</DD><DD>-Even though these are phagemids, we assumed that because they were still in the cell, the plasmids were still double stranded so could be recognized by RE. Our assumption was valid
</DD><DD>-Results of digestion
</DD><DD><UL><LI>1,2,4-14: all had bands that corresponded to pSB1C3-M13ori cut once
</LI><LI>3: Slight band at expected size but very feint
</LI><LI>15, 16: as expected, they contain bands of ~9000bp, correlating with the M13K07 phagemid. Interestingly, sample 15 also contained a (brighter) band that corresponds with pSB1C3-M13ori. We assume that this was an incident of double infection (chance or did this occur at high frequency?)
</LI></UL></DD></DL><DL><DD>Check table from 6/19 for more details
</DD><DD>-1-14: pSB1C3-M13ori
</DD><DD>-15-16: M13K07 from 1:1k diluted plate
</DD></DL><P><B>6/21</B></P><UL><LI>Needed to transform our pSB1C3-M13ori(New) (6/20) into cells. First, we transformed into 5alpha cells; however, we need to infect these cells later in order to make phage. Our 5alphas are not competent so we repeated the transformation, this time using ER2738 cells which contain the F’ episome, allowing us to infect with the M13 phage.
</LI></UL><DL><DD>-Also transformed sample #1 from 6/20 mini-preps (pSB1C3-M13ori(Old) into ER2738 cells. Do not see any notes about this plasmid being in infectable cells during the initial experiment. Reason for experiment failure???
</DD></DL><UL><LI>Ran 6/20 O/N PCR (to amplify M13K07 genes) on gel (also re-ran the previous sample from the day with more DNA)
</LI></UL><DL><DD>-Still, band looks too big. Brightest band still occurs between 8000 and 9000bp. Lighter band around 7000bp-- might be ~6000pb but too light to tell
</DD></DL><UL><LI>Alternative plan for amplifying M13K07 genes
</LI></UL><DL><DD>-Digest sample with PstI (cuts outside of the target region)
</DD></DL><DL><DD>Used sample #16 from 6/20 b/c it is double stranded so will cut
</DD></DL><DL><DD>-PCR amplified the linearized digestion product
</DD><DD>1. PCR of M13genes
</DD><DD>2. PCR of M13genes + DMSO
</DD><DD>Gel extracted pieces boxed in red
</DD></DL><DL><DD>-Used primers Gem012 F&amp;R
</DD><DD>-Gel extracted piece but received very low yield
</DD></DL><UL><LI>Alternative to alternative plan for amplifying M13K07 genes
</LI></UL><DL><DD>-Digested with PstI and AgeI
</DD></DL><DL><DD>1. Cut with PstI and AgeI
</DD><DD>2. Uncut
</DD><DD>3. Cut with PstI
</DD><DD>4. Just PCR
</DD><DD>-Gel extracted ~6000pb band after double digestion
</DD><DD>-PCR (Phusion)
</DD><DD>-Digestion (A+P) product
</DD><DD>-Gel extraction product
</DD><DD>-No DNA control
</DD></DL><H2><SPAN class="mw-headline" id="Week_8">Week 8</SPAN></H2><P><B>6/22</B></P><UL><LI>Did receive colonies from 6/21 transformations. Selected colonies for overnight
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>number
</TH><TH>colonies
</TH></TR><TR><TD>1-7
</TD><TD>pSB1C3-M13ori(NEW) in 5alpha
</TD></TR><TR><TD>8-14
</TD><TD>pSB1C3-M13ori(NEW) in ER2738
</TD></TR><TR><TD>15-18
</TD><TD>pSB1C3-M13ori(OLD) in ER2738
</TD></TR><TR><TD>19
</TD><TD>p110+RBS in 5alpha as control
</TD></TR><TR><TD>20
</TD><TD>p110+RBS in ER2738 as control
</TD></TR></TBODY></TABLE><UL><LI>Gel of 6/21 O/N PCR
</LI></UL><DL><DD>Digestion (AgeI+PstI) product ? Yielded the three bands that appear with every PCR of M13K07
</DD></DL><DL><DD>1. M13K07-&gt; digested (AgeI + PstI) -&gt; PCR
</DD><DD>2. M13K07-&gt; digested (AgeI+ PstI) -&gt; extraction-&gt; PCR
</DD><DD>3. No DNA control for PCR
</DD><DD>-Gel extraction product ? No bands
</DD><DD>-No DNA control ? No bands
</DD></DL><UL><LI>PCR purified the above PCR (Digestion (AgeI+PstI) product) and PCR from 6/21
</LI><LI>Digested PCR purified sample with EcoRI+PstI and ran gel
</LI></UL><DL><DD>1. PCR-&gt; PCR purified
</DD><DD>2. Sample after DpnI digest
</DD><DD>3. Sample after digestion with EcoRI and PstI
</DD><DD>4. Samples after digestion with DpnI then EcoRI and PstI
</DD></DL><UL><LI>Ligation of above digestion (M13K07 genes) with pSB1C3
</LI></UL><P><B>6/23</B></P><UL><LI>Transformed pSB1C3-M13ori(New) (6/22 ligation)
</LI><LI>Check overnights from 6/22 for the correct insert
</LI></UL><DL><DD>-Mini-prep O/Ns
</DD><DD>-Digested with EcoRI and PstI
</DD></DL><DL><DD>1-7: pSB1C3-M13ori(New) in 5alpha
</DD><DD>8-14: pSB1C3-M13ori(New) in ER2738
</DD><DD>15-18: pSB1C3-M13ori(Old)
</DD></DL><DL><DD>-All were the expected size (though gel wiggled)
</DD><DD>-Sent 4 samples for sequencing (iGEM primers: VF2 and VR)
</DD><DD><OL><LI> 4: Divergent (new) phagemid 1C3 transformed into 5alpha
</LI><LI> 11, 13: Divergent (new) phagemid 1C3 transformed into ER2738
</LI><LI> 16: Convergent (old) phagemid 1C3 transformed into ER2738
</LI></OL></DD></DL><P><B>6/24</B></P><UL><LI>Results of 6/23 transformation (pSB1C3-M13genes) from 6/22 ligation)
</LI></UL><DL><DD>-Received ~100 colonies
</DD><DD>-Set up O/N cultures for 8 of the colonies
</DD><DD>-Time passed…..
</DD><DD>-Mini-prepped the 8 O/N samples mentioned above (yes, it was a long day)
</DD><DD><UL><LI>Digested samples (E+P)
</LI></UL></DD></DL><UL><LI>Started phage amplification protocol
</LI></UL><DL><DD>-Amount of phage added = 10.9ul of 1:10 diluted phage (M13 phage (1) from 5/24 (4.575 x10^12 phage/mL)) to a final concentration of 1x10^8 phage/mL in 50mL
</DD><DD>-Managed to get both samples to the 14-28hr incubation
</DD></DL><P><B>6/25</B></P><UL><LI>Checked mini-prep samples from 6/24 (pSB1C3-M13genes)
</LI></UL><DL><DD>Ran the digestion overnight (6/24 to 6/25)
</DD></DL><DL><DD>-All contained a band at 2000bp. Mostly empty vector or contained small, light band. Sample 1 had prominent band at ~1250bp.
</DD><DD>-None of samples contained M13genes
</DD></DL><UL><LI>M13K07 is on P15A ori (10-12 copy number) whereas the pSB1C3 ori is on pUC19 (500-700 copies). It’s possible that this overexpression is detrimental to cell
</LI></UL><DL><DD>Alternative low copy plasmids found in distribution kit
</DD><DD><UL><LI>2013 (plate 5)
</LI></UL></DD><DD>pSB6A1 (1K)
</DD><DD>pSB3C5 (3C)
</DD><DD>pSB3K3 (5E)
</DD></DL><UL><LI>2014 (plate 4)
</LI></UL><DL><DD>pSB3C5 (4D)
</DD><DD>pSB6A1 (2L)
</DD></DL><DL><DD>-Suspended and transformed the above plasmids into 5alphas
</DD><DD>-Next, we will select colonies, mini-prep, digest, gel extract, ligate with M13genes
</DD></DL><UL><LI>Started phage amplification protocol  
</LI></UL><DL><DD>-Phagemids in ER2738
</DD><DD>pSB1C3-M13ori(New)
</DD><DD>pSB1C3-M13ori(Old) 
</DD></DL><DL><DD>-Set up O/N of ER2738 for infection tomorrow
</DD></DL><P><B>6/26</B></P><UL><LI>Results of transformation of plasmids from the distribution kit
</LI></UL><DL><DD>-Only received colonies pSB6A1 (2L from 2014) and pSB3C5 (4D from 2014)
</DD><DD>-Pricked colonies for O/N
</DD></DL><UL><LI>PCR of M13 genes so we can ligate it into the above backbones tomorrow
</LI></UL><DL><DD>-DNA: M13K07 digested with EcoRI+PstI (6/21)
</DD><DD>-Primers: Gem012 F &amp; R
</DD></DL><UL><LI>Finished phage amplification protocol
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Samples
</TH><TH> A269
</TH><TH> A320
</TH><TH> Concentration
</TH></TR><TR><TD> Old phagemid 1C3 (1)
</TD><TD> 0.832
</TD><TD> 0.492
</TD><TD> 7.828 x10^12
</TD></TR><TR><TD> Old phagemid 1C3 (2)
</TD><TD> 0.324
</TD><TD> 0.083
</TD><TD> 5.549 x10^12
</TD></TR><TR><TD> New phagemid 1C3 (1)
</TD><TD> 0.391
</TD><TD> 0.077
</TD><TD> 7.229 x10^12
</TD></TR><TR><TD> New phagemid 1C3 (2)
</TD><TD> 0.402
</TD><TD> 0.639
</TD><TD> -5.457 x10^12
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>While resuspending “New phagemid 1C3 (2)”, the tip fell off and we lost half of sample. Evidently, we lost most of phage so we tossed sample
</LI></UL></DD></DL><DL><DD>-Prepped and Infected ER2738 with “Old phagemid 1C3 (1)” and “New phagemid 1C3 (2)”
</DD><DD>Added 1.6ul of 1:10 diluted “Old phagemid 1C3 (1) to ER2738 cells
</DD><DD>Added 1.8ul of 1:10 diluted “New phagemid 1C3 (1) to ER2738 cells
</DD></DL><DL><DD>-Plated the infected cells at dilutions
</DD><DD>1:1 onto Chlor+Kan plates
</DD><DD>1:10 onto Chlor+Kan plates
</DD><DD>1:100 onto Chlor and onto Kan plates
</DD><DD>1:1000 onto Chlor and onto Kan plates
</DD><DD>1:10000 onto Chlor and onto Kan plates
</DD><DD>1:100000 onto Chlor and onto Kan plates
</DD></DL><P><B>6/27</B></P><UL><LI>To ligate M13genes onto different backbones
</LI></UL><DL><DD>-Mini-prepped the O/Ns from 6/26 to get backbones with low copy number
</DD><DD>pSB6A1
</DD><DD>pSB3C5 
</DD><DD><UL><LI>did not grow as well
</LI></UL></DD></DL><DL><DD>-PCR purified PCR (to amplify M13 genes with Gem012 F &amp; R) from 6/26
</DD><DD>-Digestions (50ul)
</DD><DD>Digest pSB6A1 with E + P
</DD><DD>Digest pSB3C5 with E + P
</DD><DD>Digest PCR purification product with E + P
</DD></DL><DL><DD>1. pSB6A1
</DD><DD>2. pSB3C5
</DD><DD>3. M13genes
</DD><DD>-Gel extraction of the above digestions
</DD><DD>For each: Added 10ul of 6x loading dye to 50ul digestions and divided the total volume between 2 wells
</DD></DL><DL><DD>-Ligation
</DD><DD>1) pSB6A1-M13genes
</DD><DD>2) pSB3C5-M13genes
</DD></DL><UL><LI>Results from 6/26 infection (after 22hrs in incubator)
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Anti-Sense Phagemid 1C3
</TH><TH> 1:100
</TH><TH> 1:1000
</TH><TH> 1:10000
</TH><TH> 1:100000
</TH></TR><TR><TD> Chlor
</TD><TD> 85
</TD><TD> 6
</TD><TD> 2
</TD><TD> 0
</TD></TR><TR><TD> Kan
</TD><TD> 405
</TD><TD> 13
</TD><TD> 2
</TD><TD> 1
</TD></TR></TBODY></TABLE><TABLE class="wikitable"><TBODY><TR><TH>  Sense Phagemid1C3
</TH><TH> 1:100
</TH><TH> 1:1000
</TH><TH> 1:10000
</TH><TH> 1:100000
</TH></TR><TR><TD> Chlor
</TD><TD> 121
</TD><TD> 24
</TD><TD> 3
</TD><TD> 0
</TD></TR><TR><TD> Kan
</TD><TD> 25
</TD><TD> 1
</TD><TD> 0
</TD><TD> 0
</TD></TR></TBODY></TABLE><P>-We also plated the infected cells on plates containing Chlor and Kan to test for the possibility of double infection
</P><TABLE class="wikitable"><TBODY><TR><TH>  Phagemid 1C3
</TH><TH> 1:1
</TH><TH> 1:10
</TH></TR><TR><TD> Anti-Sense
</TD><TD> 59
</TD><TD> 0
</TD></TR><TR><TD> Sense
</TD><TD> 4
</TD><TD> 0
</TD></TR></TBODY></TABLE><P><B>6/28</B></P><UL><LI>We noticed that there were more colonies on our infection plates from 6/26 than on 6/27; therefore, we recounted colonies
</LI></UL><DL><DD>-No increase of colonies on Kan plates
</DD><DD>-Significant increase of colonies on Chlor plates
</DD><DD>-Sense refers to the first phagemid 1C3 where the M13ori is in the sense direction compared to the plasmid ori
</DD><DD>-Anti-Sense refers to the new phagemid 1C3 where the M13ori is in the anti-sense direction compared to the plasmid ori
</DD></DL><DL><DD>Numbers on the left are after 22 hours. Numbers on the right are after 38.75 hours
</DD></DL><UL><LI>Results of transformation from 6/27 (pSB6A1-M13genes and pSB3C5-M13genes)
</LI></UL><DL><DD>-We have many colonies. Unfortunately, some are red, suggesting that original insert (J04450) was not successfully separated from backbone through gel extraction
</DD><DD>-Selected colonies to grow overnight in 5mL LB
</DD></DL><H2><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H2><P><B>6/29</B></P><UL><LI>Check O/N cultures for correct constructs (pSB6A1-M13genes and pSB3C5-M13genes)
</LI></UL><DL><DD>-Mini-prep samples
</DD><DD>-Digested with EcoRI and PstI to check insert sizes
</DD><DD>-Gel
</DD></DL><DL><DD>lanes….
</DD><DD>1-11: pSB6A1-M13genes
</DD><DD>12-18: pSB3C5-M13genes
</DD></DL><DL><DD>-Epic failure
</DD><DD>-All pSB6A1 backbones are empty
</DD><DD>-Half of pSB3C5 backbones were empty. The others contained random inserts (1700 OR 3500). We don’t know what these inserts are. 1700 band is likely the digestion product that appears when we digest M13genes
</DD></DL><UL><LI>Question: Can we use empty vectors from these mini-preps as ligation vectors?
</LI></UL><DL><DD>-Digest pSB6A1 mini-prep with…. (see if both cut sites were retained during re-ligation)
</DD><DD>no enzyme
</DD><DD>EcoRI
</DD><DD>PstI
</DD><DD>EcoRI + PstI
</DD></DL><DL><DD>lanes....
</DD><DD>1-2: last two samples of pSB3C5-M13genes from above gel
</DD><DD>3. Uncut
</DD><DD>4. Cut with EcoRI
</DD><DD>5. Cut with PstI
</DD><DD>6. Cut with EcoRI and PstI
</DD></DL><DL><DD>-Appears that two backbones are ligated together
</DD></DL><P><B>6/30</B></P><UL><LI>Made chemically competent ER2738 cells that contain Litmus28i DNA
</LI><LI>To amplify M13genes in order to retry ligation
</LI></UL><DL><DD>-PCR of M13K07 DNA (diluted 1:100) to amplify the M13 genes
</DD><DD>-PCR purify PCR product
</DD><DD>-Run on gel: PCR purification, PCR, noDNAcontrol
</DD><DD><UL><LI>PCR and purification showed bands at ~9k; therefore, did not get product
</LI><LI>No DNA control was clean
</LI></UL></DD></DL><UL><LI>Made freeze downs of
</LI></UL><DL><DD>pSB3C5-J04450
</DD><DD>pSB6A1-J04450
</DD><DD>‘empty’ pSB3C5
</DD><DD>‘empty’ pSB6A1
</DD></DL><P><B>7/1</B></P><UL><LI>Digestion of PCR product from 6/30 to figure out where mistake is
</LI></UL><DL><DD>-Not really useful
</DD></DL><DL><DD>Lanes...
</DD><DD>1. PCR pur -&gt; digested with AgeI
</DD><DD>2. PCR pur -&gt; digested with NgoMIV
</DD><DD>3. PCR pur -&gt; digested with PstI
</DD><DD>4. PCR pur -&gt; uncut
</DD><DD>5. PCR pur -&gt; digested with dpnI
</DD><DD>6. Uncut plasmid DNA
</DD></DL><UL><LI>Talked to Mary: She says we were adding to much DNA
</LI></UL><DL><DD>-PCR again. Used 1:100 dilution of 1:100 diluted M13K07 DNA. (aka 1:10000 dilution)
</DD></DL><UL><LI>Received a beautiful band at 6k bp
</LI></UL><DL><DD>lanes
</DD><DD>1. 1:100 dilution
</DD><DD>2. 1:1000 dilution
</DD><DD>3. 1:10 000 dilution
</DD><DD>4. No DNA control
</DD></DL><P><B>7/2</B></P><UL><LI>To biobrick M13 genes (pSB6A1-M13genes and pSB3C5-M13genes)
</LI></UL><DL><DD>-PCR purified 7/1 PCR product (Primers = Gem012)
</DD><DD>-Digested with EcoRI-HF and PstI-HF
</DD><DD>-Ligation to
</DD><DD>pSBA61 (digested and gel extracted)
</DD><DD>pSB3C5 (digested and gel extracted)
</DD></DL><P><B>7/3</B></P><UL><LI>Transformation
</LI></UL><DL><DD>-pSB6A1-M13genes into 5alpha cells and ER2738 with pSB1C3-M13ori
</DD><DD>-pSB3C5-M13genes into 5alpha cells and ER2738 with Litmus28i
</DD><DD>-Transformed into cells containing a phagemid in order to skip some steps
</DD></DL><P><B>7/3-7/7 Vacation!</B></P><H2><SPAN class="mw-headline" id="Week_10">Week 10</SPAN></H2><P><B>7/7</B></P><UL><LI>Made O/N cultures of 7/3 transformation colonies
</LI></UL><DL><DD>-Transformation results were not recorded until 7/8 (see below)
</DD></DL><P><B>7/8</B></P><UL><LI>Transformation results ( CC = Chemically comp cells, ER = ER2738)
</LI></UL><DL><DD>-No growth on No DNA controls
</DD><DD>ER-Litmus28i CC on AMP-Chlor-Tet
</DD><DD>ER-phagemid1C3 CC on AMP-Chlor-Tet              
</DD><DD>5alpha CC on Amp                      
</DD><DD>5alpha CC on Chlor 
</DD></DL><DL><DD>-Lots of red colonies on positive controls, no white colonies
</DD><DD>ER-Litmus28i CC + pSB3C5 on AMP-Chlor-Tet
</DD><DD>ER-phagemid1C3 CC + pSB6A1 on AMP-Chlor-Tet
</DD></DL><DL><DD>-Lots of white colonies on sample plates, some red colonies
</DD><DD>ER-Lit CC + M13genes-pSB3C5 on  AMP-Chlor-Tet
</DD><DD>ER-phagemid1C3 CC + M13genes-pSB6A1 on  AMP-Chlor-Tet
</DD><DD>5alpha CC + M13genes-pSB3C5 on Chlor
</DD><DD>5alpha CC + M13genes-pSB6A1 on Amp
</DD></DL><UL><LI>Overnights from 7/7 look healthy
</LI></UL><DL><DD>-Renamed O/N to have numbers instead of long names
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample #
</TH><TH> Cells
</TH><TH> DNA
</TH></TR><TR><TD> 1-5
</TD><TD> ER-Lit
</TD><TD> M13genes-pSB3C5
</TD></TR><TR><TD> 6-10
</TD><TD> ER-phagemid1C3
</TD><TD> M13genes-pSB6A1
</TD></TR><TR><TD> 11-15
</TD><TD> 5alpha
</TD><TD> M13genes-pSB3C5
</TD></TR><TR><TD> 16-20
</TD><TD> 5alpha
</TD><TD> M13genes-pSB6A1
</TD></TR></TBODY></TABLE><DL><DD>-Mini-prepped DNA
</DD><DD>-Digest the mini-preps with EcoRI-HF and PstI-HF
</DD><DD>-Run digestions on gel to check sizes
</DD></DL><DL><DD>lanes
</DD><DD>top-left: pSB3C5-M13genes in ER2738 with Litmus28i
</DD><DD>top-right: pSB6A1-M13genes in ER2738 with pSB1C3-1C3
</DD><DD>bottom-left: pSB3C5-M13genes in 5alpha
</DD><DD>bottom-right: pSB6A1-M13genes in 5alpha
</DD></DL><DL><DD>None are correct
</DD></DL><UL><LI>Primers Gem013 came in. Resuspend and diluted primers
</LI></UL><DL><DD>-O/N of pSB3C5 to use for PCR tomorrow
</DD></DL><P><B>7/9</B></P><UL><LI>Wanted to make Litmus28i biobrick compatible for use as a phagemid backbone for us and other iGEM teams
</LI></UL><DL><DD>-Mini-prepped O/N of pSB3C5
</DD><DD>-PCR of pSB3C5 to amplify J04450 with Litmus28i compatible cut sites
</DD><DD>Primers: Gem013
</DD><DD><UL><LI>At the 3’ end, these primers are the same as VF2 and VR so will bind the region flanking J04450. This conserves the terminators that exist between the biobrick prefix and VF2 on one side and those between the biobrick suffix and VR on the other side. Therefore, we are amplifying, VF2 priming site, terminators, J04450, terminators, and VR priming site
</LI><LI>At the 5’ end, these primers contain unique restriction sites found in the Litmus28i MCS
</LI></UL></DD></DL><UL><LI>PCR of M13 genes…. again (did 4 samples)
</LI></UL><DL><DD>-Used the 1:10000 dilutions
</DD></DL><P><B>7/10</B></P><UL><LI>Note: The following two projects were done in parallel when possible
</LI><LI>To make Litmus28i biobrick compatible
</LI></UL><DL><DD>-Ran gel of PCR from 7/9 (J04450 amplification with Gem013)
</DD></DL><DL><DD>1-4: PCRs of M13genes
</DD><DD>5. No DNA control for M13genes PCR
</DD><DD>6. PCR of pSB3C5 backbone
</DD><DD>7. No DNA control for pSB3C5 backbone
</DD><DD>-PRC purified the PCR product
</DD><DD>-Digestion #1
</DD><DD>Restriction enzymes had the same buffer conditions but different activation temperatures so we had to do a 2 part digestion
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Restriction enzymes
</TH><TH> Notes
</TH></TR><TR><TD> J04450
</TD><TD> Sac1
</TD><TD> PCR purified
</TD></TR><TR><TD> Litmus 28i
</TD><TD> Sac1
</TD><TD> From NEB tube
</TD></TR></TBODY></TABLE><DL><DD><UL><LI>Incubate 1hr at 37 C
</LI><LI>Heat inactivated 20 minutes at 80C
</LI></UL></DD></DL><DL><DD>-Digestion #2
</DD><DD>-Added 1ul BsmI to both samples
</DD><DD>-Incubated 1hr at 65C
</DD><DD>-Heat inactivated 20 minutes at 80C
</DD></DL><DL><DD>Ran gel
</DD><DD>-See gel below
</DD><DD>-Tried to extract J04450 segment but received very low yield. Since band otherwise looked clean, we decided to redo digestion then skip straight to ligation
</DD></DL><DL><DD>Repeat digestion #1 for J04450
</DD><DD>Repeat digestion #2 for J04450
</DD><DD>Ligation (10hr at 16C, 10 min at 80C)
</DD><DD>3. Litmus28i + J04450
</DD></DL><UL><LI>To retry ligation to biobrick backbone
</LI></UL><DL><DD>-Ran gel of PCR from 7/9 (M13genes amplified with Gem012)
</DD><DD>See gel below
</DD></DL><DL><DD>-PCR purified the PCR product
</DD><DD>-Digestion
</DD></DL><TABLE class="wikitable"><TBODY><TR><TH>  Sample
</TH><TH> Restriction enzymes
</TH><TH> Notes
</TH></TR><TR><TD> M13 genes
</TD><TD> EcoRI-HF + PstI-HF
</TD><TD> PCR purified
</TD></TR><TR><TD> 6A1
</TD><TD> EcoRI-HF + PstI-HF
</TD><TD> ‘empty’ pSB6A1
</TD></TR><TR><TD> 3C5
</TD><TD> EcoRI-HF + PstI-HF
</TD><TD> contains J04450
</TD></TR></TBODY></TABLE><DL><DD>Incubate 1hr at 37C
</DD><DD>Heat inactivated 20 minutes at 80C
</DD><DD>-Ran gel
</DD><DD><UL><LI>See gel below
</LI><LI>Tried to extract M13genes and pSB3C5 segments but received very low yields. Since bands otherwise looked clean, we decided to redo digestion then skip straight to ligation
</LI></UL></DD></DL><DL><DD>-Repeat digestion for M13genes and pSB3C5
</DD><DD>-Ligation (10h at 16C, 10m at 80C)
</DD><DD>1. pSB3C5 + M13genes
</DD><DD>2. pSB6A1 + M13genes
</DD></DL><UL><LI>Gel
</LI></UL><DL><DD>1. pSB1A3
</DD><DD>2. Limtus28i
</DD><DD>3. M13genes
</DD><DD>4. pSB3C5
</DD><DD>5. J04450
</DD></DL><P><B>7/11</B></P><UL><LI>Transformed ligations from 7/10 into 5alpha cells
</LI></UL><DL><DD>1. pSB3C5 + M13genes
</DD><DD>2. pSB6A1 + M13genes
</DD><DD>3. Litmus28i + J04450
</DD></DL><P><B>7/12</B></P><UL><LI>Results of 7/11 transformation
</LI></UL><DL><DD>-No growth on no DNA control (Amp or Chlor)
</DD><DD>-Lots of growth on Lit-J04450 -&gt; some red-&gt; colonies are too close together to prick individual colonies
</DD><DD>-It’s possible that these red colonies are satellites =(
</DD><DD>-Swiped some and plated on new Amp plate (restreak)
</DD></DL><DL><DD>-Lawn of positive control (Litmus 28i) on Amp
</DD><DD>-Many colonies for L1 (pSB3C5-M13genes) and L2 (pSB6A1-M13genes)
</DD><DD>-Due to high number of red colonies on Lit-J04450 plate, we assumed that most of these colonies contain empty vector
</DD><DD>-Did not make O/N
</DD></DL><UL><LI>Ligated M13 genes to pSB6A1
</LI></UL><DL><DD>-Used the remained of our digested M13genes.
</DD><DD>-Both digestions from 7/10
</DD></DL><UL><LI>Infection Experiment
</LI></UL><DL><DD>-had 3 cell stocks (each taken from a different colony the night before)
</DD><DD>-Tested Litmus 28i, Tandem phagemid 1C3, and double infection
</DD><DD><UL><LI>Therefore, there were 9 flasks total.
</LI></UL></DD></DL><DL><DD>-negative control: Streaked parent cells (non-infected) onto Chlor, Amp, and kan
</DD><DD>-Included double infection plates for Litmus 28i and phagemid 1C3
</DD></DL><H2><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H2><P><B>7/13</B></P><UL><LI>Transformed 7/12 ligation (pSB6A1-M13genes)
</LI><LI>Litmus28i-J04450
</LI></UL><DL><DD>-All growth from 7/12 restreak was white
</DD><DD>-Pricked some red colonies for liquid O/N-will hopefully see red tomorrow
</DD><DD>-Pricked a few red colonies and put into 200ul H2O (a few colonies per tube- 2 tubes total).
</DD><DD><UL><LI>Plated 150ul onto Amp plates
</LI></UL></DD></DL><DL><DD>-Put original plate into incubator to hopefully get bigger colonies
</DD></DL><P><B>7/14</B></P><UL><LI>Finished TWIV ppt
</LI><LI>Set up 2 liquid culture of red Litmus 28i colonies (Litmus28i-J04450)-- slow growth 
</LI></UL><DL><DD>-Mini-prepped red Litmus 28i ‘O/N’ from earlier in day
</DD><DD>-Digested samples—to check for insert and correct cut sites
</DD></DL><UL><LI>Made O/Ns of pSB6A1-M13genes colonies from 7/13 transformation
</LI></UL><P><B>7/15</B></P><UL><LI>Ran gel of Litmus28i-J04450 samples
</LI></UL><DL><DD>1. J04450-Litmus28ibb #1 (EcoRI+PstI)
</DD><DD>2. J04450-Litmus28ibb #1 (uncut)
</DD><DD>3. J04450-Litmus28ibb #2 (EcoRI+PstI)
</DD><DD>4. J04450-Litmus28ibb #2 (EcoRI+PstI)
</DD><DD><UL><LI>Accidentally added restriction enzymes
</LI></UL></DD></DL><DL><DD>5. Litmus28i (EcoRI+PstI)
</DD><DD><UL><LI>Only has PstI site
</LI></UL></DD></DL><DL><DD>-Verifies that Litmus28i-J00450 had correct cut sites
</DD><DD>-From this point on, biobricked Litmus28i is called Litmus28ibb
</DD></DL><UL><LI>Check 7/14 O/Ns for pSB6A1-M13genes
</LI></UL><DL><DD>-Mini-prepped liquid cultures
</DD><DD>-Digested with EcoRI and PstI
</DD><DD>Received bold bands of just under 4000bp. Some lanes had a very feint band ~330bp. None were the correct size
</DD></DL><UL><LI>Transformed Litmus28ibb-J04450 into ER2738 cells
</LI><LI>Remake phage packaging
</LI></UL><DL><DD>-pSB1C3-M13ori(New)
</DD><DD>-pSB1C3-M13ori(Old)
</DD></DL><P><B>7/16</B></P><UL><LI>Results of 7/15 transformation of Litmus28i into 6/30 ER2738 cells
</LI></UL><DL><DD>-lawn on no DNA control-- most likely contaminated cells
</DD><DD>Streaked chem comp 5alpha and ER onto Amp and Amp+Tet plates to determine if problem is with cells or plates
</DD></DL><DL><DD>-O/N culture of ER to make new chem comp cells
</DD></DL><P><B>7/17</B></P><UL><LI>Finished phage isolation
</LI></UL><DL><DD>Note:
</DD><DD><UL><LI>pSB1C3-M13ori (New): M13ori and plasmid ori are convergent
</LI><LI>pSB1C3-M13ori (Old): M13ori and plasmid ori are tandem
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> 269nm
</TH><TH> 320nm
</TH><TH> [] phage/mL
</TH></TR><TR><TD> pSB1C3-M13ori (New)
</TD><TD> 0.386
</TD><TD> 0.034
</TD><TD> 8.126 E12
</TD></TR><TR><TD> pSB1C3-M13ori (New)
</TD><TD> 0.419
</TD><TD> 0.036
</TD><TD> 8.842 E12
</TD></TR><TR><TD> pSB1C3-M13ori (Old)
</TD><TD> 0.283
</TD><TD> 0.025
</TD><TD> 5.956 E12
</TD></TR><TR><TD> pSB1C3-M13ori (Old)
</TD><TD> 0.326
</TD><TD> 0.024
</TD><TD> 6.972 E12
</TD></TR></TBODY></TABLE><UL><LI>Infection to compare the new and the old pSB1C3-M13ori
</LI></UL><DL><DD>-Prepped cells for infection
</DD><DD>-Infected cells with pSB1C3-M13ori (New) or pSB1C3-M13ori (New)
</DD><DD>-and plated on dilutions of 1:100, 1:1000, and 1:10000 on Chlor and kan
</DD><DD>-Also plated 1:1000 dilution on lowered Chlor concentration (34ug/mL)
</DD></DL><UL><LI>Sent Litmus28ibb-J04450 for sequencing
</LI></UL><P><B>7/18</B></P><UL><LI>Results from contamination test
</LI></UL><DL><DD>-5alpha on Amp = no growth
</DD><DD>-5alpha on Amp+Tet = colonies
</DD><DD>-ER on Amp = lawn
</DD><DD>-ER on Amp+Tet = lawn
</DD></DL><UL><LI>Made new chemically competent ER2738
</LI><LI>Transformed Litmus28ibb-J04450 into ER2738 made on 3/30 and ER2738 made on 7/18
</LI></UL><DL><DD>-Had two samples of Litmus28ibb-J04450 and two cell stocks, so 4 samples total
</DD></DL><P><B>7/19</B></P><UL><LI>Results of 7/17 infection
</LI></UL><DL><DD>-pSB1C3-M13ori(Old) (tandem-when plasmid ori and M13ori point in the same direction) packages better than pSB1C3-M13ori(New) (when ori and M13ori are convergent), implying that directionality matters.
</DD></DL><H2><SPAN class="mw-headline" id="Week_12">Week 12</SPAN></H2><P><B>7/20</B></P><UL><LI>Freeze downs
</LI></UL><DL><DD>-Litmus28i-J04450        Litmus 28i is now biobrick compatible
</DD></DL><P><B>7/21</B></P><UL><LI>Send samples for sequencing
</LI></UL><DL><DD>-pSB1C3-M13ori(Old)
</DD><DD>-pSB1C3-M13ori(New)
</DD></DL><UL><LI>To ligate M13ori to a kanamycin backbone (pSB1K3)
</LI></UL><DL><DD>-Resuspended pSB1K3-J04450 from distribution kit (6B on plate 4)
</DD><DD><UL><LI>Contains J04450 as insert (full RFP construct)
</LI></UL></DD></DL><DL><DD>-Transform pSB1K3 into 5alpha cells
</DD></DL><P><B>7/22</B></P><UL><LI>Pricked colony from pSB1K3-J04450 transformation
</LI></UL><DL><DD>-Mini-prepped DNA to get a supply of DNA
</DD></DL><UL><LI>To put M13ori (M13 phackaging signal) onto Kanamycin resistance o we can test packaging ratios with M13K07 on the same antibiotic
</LI></UL><DL><DD>-Digested pSB1K3 with EcoRI-HF and XbaI
</DD><DD>-Digested pSB1C3-M13ori (Old) with EcoRI-HF and SpeI-HF
</DD><DD><UL><LI>DNA was not sufficiently cut. Too much DNA? Problem with enzyme (SpeI-HF)?
</LI></UL></DD></DL><DL><DD>1. pSB1K3
</DD><DD>2. M13ori
</DD></DL><P><B>7/23</B></P><UL><LI>Sequencing samples from 7/21 were lost in the mail. Resent samples
</LI><LI>Yet another attempt to biobrick M13genes
</LI></UL><DL><DD>1. using primers that would amplify genes and M13 ori parts from M13K07
</DD><DD>2. using primers that would amplify genes, M13 ori parts, and plasmid ori from M13K07
</DD></DL><DL><DD>1. Amplified only the M13genes and M13ori
</DD><DD>2. No DNA control for 1
</DD><DD>3. Amplified M13genes, M13ori, and plasmid ori
</DD><DD>4. No DNA control for 3
</DD><DD>-When run on a gel, samples were clean with only one band at around 7kb. No contamination in no DNA controls
</DD></DL><UL><LI>Digestions
</LI></UL><DL><DD>-pSB1K3 with EcoRI-HF and PstI-HF to check for correct insert
</DD><DD>-pSB1C3-M13ori (Old and new) with EcoRI-HF and SpeI-HF to test for efficient cutting with different stock of restriction enzyme
</DD></DL><DL><DD>1-3: pSB1K3
</DD><DD>4. pSB1C3-M13ori (Old)
</DD><DD>5. pSB1C3-M13ori (New)
</DD></DL><UL><LI>Still had inefficient cutting. Tested M13ori next to Litmus28i to disern the problem
</LI></UL><DL><DD><UL><LI>Uncut
</LI><LI>Cut once with (E, X, S, or P)
</LI><LI>Cut twice with (E+S or E+P)
</LI></UL></DD></DL><DL><DD>1. M13ori uncut
</DD><DD>2. M13ori Ecori-HF
</DD><DD>3. M13ori SpeI-HF
</DD><DD>4. M13ori XbaI
</DD><DD>5. M13ori PstI-HF
</DD><DD>6. M13ori EcoRI-HF + SpeI-HF
</DD><DD>7. M13ori EcoRI-HF + PstI-HF
</DD><DD>8. Litmus28i uncut
</DD><DD>9. Litmus28i EcoRI-HF
</DD><DD>10. Litmus28i SpeI-HF
</DD><DD>11. Litmus28i XbaI
</DD><DD>12. Litmus28i PstI-HF
</DD><DD>13. Litmus28i EcoRI-HF + SpeI-HF
</DD><DD>14. Litmus28i EcoRI-HF + PstI-HF
</DD></DL><UL><LI>Only partial digest with only SpeI-HF for both. Complete digestion with all others, including E+S
</LI></UL><P><B>7/24</B></P><UL><LI>To Biobrick M13genes using pSB6A1
</LI></UL><DL><DD>-PCR purified M13 genes (did both samples at the same time)
</DD><DD>-Digested PCR purification and pSB6A1 with EcoRI-HF and PstI-HF
</DD></DL><UL><LI>Gel extracted M13genes
</LI></UL><DL><DD>1. M13genes
</DD><DD>2. pSB6A1
</DD><DD>-Bands of pSB6A1 were too light to gel extract
</DD></DL><UL><LI>To swap the kanamycin resistance marker on M13K07 with ampicillin resistance
</LI></UL><DL><DD>-PCR amplified AmpR from pSB6A1
</DD></DL><UL><LI>made O/N culture of pSB6A1 from freeze down to mini-prep
</LI></UL><DL><DD>-Also streaked cells onto plate
</DD></DL><P><B>7/25</B></P><UL><LI>To change resistance marker on M13K07
</LI></UL><DL><DD>1. Biobrick method
</DD><DD><UL><LI>Tried to mini-prep O/N but pellet (after liquid culture was spun) was not red even after 16+ hours. Set up O/N from plate colonies
</LI></UL></DD></DL><DL><DD>2. Swap only resistance method
</DD><DD><UL><LI>PCR purified 7/24 PCR
</LI><LI>Ran product on gel-&gt; band of correct size
</LI></UL></DD></DL><DL><DD>1. PCR product
</DD><DD>2. PCR product-&gt; PCR purified
</DD><DD>3. No DNA control
</DD><DD>-Digested the PCR purification products of M13genes+ori and AmpR with AgeI-HF and NotI-HF
<UL><LI>Bands on previous gel looked clean enough for both so did not gel extract
</LI></UL></DD><DD>-Overnight ligation
</DD></DL><UL><LI>Set up O/Ns of pSB1K3 colonies
</LI></UL><P><B>7/26</B></P><UL><LI>Transform
</LI></UL><DL><DD>-pSB6A1+M13genes
</DD></DL><UL><LI>To biobrick M13genes
</LI></UL><DL><DD>-Digestion
</DD><DD><UL><LI>pSB6A1 with EcoRI and PstI
</LI></UL></DD></DL><DL><DD>Gel extracted backbone (band ~4000bp)
</DD></DL><UL><LI>Ligations (10hrs 16C, 10min at 80C)
</LI></UL><DL><DD>-pSB6A1+M13genes
</DD></DL><H2><SPAN class="mw-headline" id="Week_13">Week 13</SPAN></H2><P><B>7/27</B></P><UL><LI>Transformation results from 7/26
</LI></UL><DL><DD>-No growth for no DNA controls <I>on Amp</I></DD><DD>-No growth for M13 genes onto pSB6A1 <I>on Amp</I></DD></DL><UL><LI>Set up O/Ns of 5alpha and ER2738 to make competent cell tomorrow
</LI><LI>Made freeze down of pSB1K3
</LI><LI>Plated ER competent cells from 5/15 and 6/16 Onto Chlor plates to check for contamination
</LI></UL><P><B>7/28</B></P><UL><LI>No growth of ER2738 on Chlor (either sample)
</LI></UL><DL><DD>-ER is not contaminated with ChlorR
</DD></DL><UL><LI>Make new 5alpha chem comp cells    
</LI><LI>Transformations
</LI></UL><DL><DD>-pSB6A1-M13genes
</DD><DD>-positive control for Amp (pSB6A1)
</DD><DD>-No DNA control
</DD></DL><P><B>7/29</B></P><UL><LI>Set up O/Ns of pSB6A1-M13genes
</LI></UL><DL><DD>-Plated sample on reduced Amp and regular Tet
</DD><DD>-Later in day…
</DD><DD>-Mini-prepped O/Ns
</DD><DD>-Digested and ran on a gel to check for insert sizes
</DD></DL><DL><DD><UL><LI>All but the 4th lane with 3 bands look correct
</LI></UL></DD></DL><P><B>7/30</B></P><UL><LI>Lawn on last night’s plating-&gt; replate
</LI><LI>Digest pSB3C5 and pSB6A1-M13genes with E and P to move M13genes to a Chlor backbone
</LI></UL><DL><DD>-Extracted bands
</DD><DD>-Ligations
</DD><DD><UL><LI>pSB3C5 + M13genes
</LI></UL></DD></DL><UL><LI>Sent samples for sequencing
</LI><LI>M13 genes on pSB6A1            VF2
</LI><LI>M13 genes on pSB6A1            VR
</LI></UL><P><B>Test experiment for high school kids coming to lab (7/29-7/31)</B></P><UL><LI>General idea: Have 2 strands of DNA, one has an EcoRI site while the other contains a SNP, abolishing the cut sight. We pretend that one is pathogenic (the one that is not cut) and tells kids to figure out which one is which 
</LI><LI>PCR with Dream-Taq
</LI></UL><DL><DD>-Did not have special fastdigest enzyme
</DD><DD>-Hypothesized that green dye will interfere with enzyme effectivity
</DD></DL><UL><LI>PCR with Dream-Taq (no green dye)
</LI></UL><DL><DD>-Digest with EcoRI fastdigest-&gt; did not cut
</DD></DL><UL><LI>Redid experiment several times
</LI></UL><DL><DD>-Should digest with XbaI
</DD><DD>-PCR purified PCR products
</DD></DL><P><B>7/31</B></P><UL><LI>Spent the morning with Heritage High School
</LI><LI>Made phage
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Phage
</TH><TH> A269
</TH><TH> A320
</TH><TH> genome size
</TH><TH> [ ]
</TH></TR><TR><TD> Litmus28ibb-J04450 (1)
</TD><TD> 0.823
</TD><TD> 0.197
</TD><TD> 9.35 x10^12
</TD></TR><TR><TD> Litmus28ibb-J04450 (2)
</TD><TD> 0.940
</TD><TD> 0.168
</TD><TD> 1.14 x10^13
</TD></TR></TBODY></TABLE><P><B>8/2</B></P><UL><LI>Digestion
</LI></UL><DL><DD>-pSB6A1-M13genes (E+P)
</DD><DD>-pSB3C5 (E+P)
</DD></DL><H2><SPAN class="mw-headline" id="Week_14">Week 14</SPAN></H2><P><B>8/3</B></P><UL><LI>Another attempt to clone M13genes onto pSB3C5
</LI></UL><DL><DD>-Run 8/2 digestions on gel
</DD></DL><DL><DD>1. M13genes (E+P)
</DD><DD>2. pSB3C5 (E+P)
</DD><DD>-Gel extracted 
</DD><DD><UL><LI>M13genes away from pSB6A1 backbone
</LI><LI>pSB3C5 backbone away from J04450
</LI></UL></DD></DL><DL><DD>-Ligations
</DD><DD><UL><LI>pSB3C5 + M13genes
</LI><LI>pSB3C5 + no insert
</LI></UL></DD></DL><DL><DD>-Transformed ligations into ER2738
</DD></DL><P><B>8/5</B></P><UL><LI>Transformation of pSB3C5+M13genes failed
</LI><LI>Made phage
</LI></UL><DL><DD>-Fd-CAT DNA packaged with Fd-CAT
</DD><DD>-phagemid 1C3 packaged with M13K07
</DD><DD>-amilCP on pSB1C3 packaged with M13K07
</DD><DD>-Litmus28ibb-J04450 packaged with M13K07
</DD></DL><P><B>8/6</B></P><UL><LI>Isolated phage
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Phage
</TH><TH> [ ]
</TH></TR><TR><TD> Fd CAT
</TD><TD> 2.72 x10^12
</TD></TR><TR><TD> Litmus28ibb-J04450
</TD><TD> 8.37 x10^12
</TD></TR><TR><TD> pSB1C3-amilCP
</TD><TD> 1.42 x10^13
</TD></TR><TR><TD> pSB1C3-M13ori
</TD><TD> 4.32 x10^12
</TD></TR></TBODY></TABLE><P><B>8/6</B></P><UL><LI>Started making phage with Fd CAT the should contain Litmus28ibb-RFP
</LI><LI>Infection
</LI></UL><DL><DD>-Infected ER2738 cells with Litmus28ibb-RFP, pSB1C3-M13ori, or pSB1C3-amilCP
</DD><DD>-Plated Litmus samples on 100ug/mL Amp and 50ug/mL Kan
</DD><DD>-Plated other two samples on 34ug/mL Chlor and 25ug/mL Kan
</DD></DL><P><B>8/7</B></P><UL><LI>Isolated phage containing Litmus28ibb-RFP using helper phage
</LI></UL><DL><DD>-M13K07 (as control)
</DD><DD>-Fd-CAT (no phage pellet was observed-worried that there was no phage)
</DD></DL><UL><LI>Went to CSU to have them test M13ori part compared to amilCP. Grew the samples in 5mL O/Ns then diluted to 0.5OD and grew 30 minutes rather than starting from a fresh colony
</LI></UL><P><B>8/8</B></P><UL><LI>At CSU
</LI></UL><DL><DD>-Finished phage protocol (Test packaging of M13ori part)
</DD><DD>-Only used 20mL infection samples
</DD><DD>-grew to 0.55 OD
</DD><DD>-When making phage (after 14 hour incubation), there was very little growth
</DD><DD>-Plated non-diluted and diluted 1:1000 of M13ori and amilCP sample
</DD></DL><UL><LI>Infected cells using phage isolated 8/7. Plated at dilutions of 1:100 and 1:100k
</LI><LI>Want to test progeny Fd-CAT phage (made 8/7) for infectability
</LI></UL><DL><DD>-Start phage isolation prodocol using Fd-CAT phage from 8/7 do deliver the helper phagemid
</DD><DD><UL><LI>Phagemid:
</LI></UL></DD><DD><UL><LI>None: will make Fd-CAT phage packaing Fd-CAT phagemid
</LI><LI>Litmus28ibb-J04450: make Fd-CAT phage packaging Litmus28ibb-J04450 phagemid
</LI></UL></DD></DL><UL><LI>Remake phage (Fd-CAT) using fresh stock from Mike
</LI></UL><P><B>8/9</B></P><UL><LI>Results of 8/8 infection
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Helper Phagemid
</TH><TH> phagemid
</TH><TH> Selection
</TH><TH> 1:1000 dilution
</TH><TH> 1:00k dilution
</TH></TR><TR><TD> M13K07
</TD><TD> Litmus28ibb-RFP
</TD><TD> Kan+Tet
</TD><TD> 400-500 (some red)
</TD><TD> 61 white; 20 red
</TD></TR><TR><TD> same
</TD><TD> same
</TD><TD> Amp+Tet
</TD><TD> lawn
</TD><TD> 2
</TD></TR><TR><TD> Fd-CAT
</TD><TD> Litmus28ibb-RFP
</TD><TD> Chlor+Tet
</TD><TD> 0
</TD><TD> 0
</TD></TR><TR><TD> same
</TD><TD> same
</TD><TD> Amp+Tet
</TD><TD> 10
</TD><TD> 707
</TD></TR></TBODY></TABLE><UL><LI>NOTE: The same sample was plated on two different plates. For example, the sample using M13K07 as the helper phage was plated on Kan+Tet and Amp+Tet
</LI><LI>Mini-prep
</LI></UL><DL><DD>-pSC3C5-J04450 -- very little growth, low DNA yield
</DD><DD>-Fd-CAT infected cells
</DD><DD><UL><LI>Digest and run on a gel to verify presence of a band- there was a very feint band
</LI></UL></DD></DL><UL><LI>Finished isolating phage…. messed up and used 0.8MgCl2, 0.2M NaCl instead of PEG during precipitation step. Also used the wrong phage
</LI></UL><DL><DD>-Infected ER2738 cells anyway…. no growth by 8/12
</DD></DL><H2><SPAN class="mw-headline" id="Week_15">Week 15</SPAN></H2><P><B>8/10</B></P><UL><LI>Are the progeny phage from 8/6 (original progeny from first Fd-Tet application) viable/ able to reproduce?
</LI></UL><DL><DD>-Used these progeny to make phage that should amplify Fd-CAT phage containing either the Fd-CAT or Litmus28ibb-J04450 phagemid
</DD></DL><P><B>8/11</B></P><UL><LI>Finished isolating phage
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Helper Phagemid
</TH><TH> Phagemid
</TH><TH> A269
</TH><TH> A320
</TH><TH> genome size
</TH><TH> [ ]
</TH></TR><TR><TD> Fd-CAT
</TD><TD> Litmus28ibb (1)
</TD><TD> 0.029
</TD><TD> 0.013
</TD><TD> 4080
</TD><TD> 2.35 x10^11
</TD></TR><TR><TD> Fd-CAT
</TD><TD> Ltimus28ibb (2)
</TD><TD> 0.033
</TD><TD> 0.015
</TD><TD> 4080
</TD><TD> 2.65 x10^11
</TD></TR><TR><TD> Fd-CAT
</TD><TD> Fd-CAT (1)
</TD><TD> 0.326
</TD><TD> 0.082
</TD><TD> 7775
</TD><TD> 1.88 x10^12
</TD></TR><TR><TD> Fd-CAT
</TD><TD> Fd-CAT (2)
</TD><TD> 0.320
</TD><TD> 0.079
</TD><TD> 7775
</TD><TD> 1.86 x10^12
</TD></TR><TR><TD> M13g6A1
</TD><TD> M13g6A1 (1)
</TD><TD> 0.322
</TD><TD> 0.148
</TD><TD> 10,029
</TD><TD> 1.04 x10^12
</TD></TR><TR><TD> M13g6A1
</TD><TD> M13g6A1 (2)
</TD><TD> 0.503
</TD><TD> 0.130
</TD><TD> 10,029
</TD><TD> 2.23 x10^12
</TD></TR></TBODY></TABLE><P><B>8/13</B></P><UL><LI>Infected ER2738 with Fd-CAT packaging Litmus28ibb-J04450
</LI></UL><P><B>8/16</B></P><UL><LI>Results from Infection on 8/11
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>  Helper Phagemid
</TH><TH> Phagemid
</TH><TH> 1:1
</TH><TH> 1:10
</TH><TH> 1:100
</TH><TH> 1:1000
</TH></TR><TR><TD> M13g6A1
</TD><TD> M13g6A1
</TD><TD> almost lawn
</TD><TD> 728
</TD><TD> 287
</TD><TD> 55
</TD></TR><TR><TD> Fd-CAT
</TD><TD> Fd-CAT
</TD><TD> lawn
</TD><TD> almost lawn
</TD><TD> 1260
</TD><TD> 640
</TD></TR><TR><TD> Fd-CAT
</TD><TD> Litmus28ibb-J04450
</TD><TD> lawn
</TD><TD> almost lawn
</TD><TD> 2004
</TD><TD> 304
</TD></TR></TBODY></TABLE><P>Results: packaging of Litmus28ibb-J04450 using Fd-CAT helper phagemid from (8/15)
</P><TABLE class="wikitable"><TBODY><TR><TH>Antibiotic
</TH><TH>1:10
</TH><TH>1:100
</TH><TH>1:1k
</TH><TH>1:10k
</TH><TH>1:100k
</TH></TR><TR><TD>Amp
</TD><TD>Lawn
</TD><TD>Almost lawn
</TD><TD>1025
</TD><TD>217
</TD><TD>25
</TD></TR><TR><TD>Kan
</TD><TD>9
</TD><TD>2
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_16">Week 16</SPAN></H2><P><B>8/19</B></P><P>Sent samples for sequencing
</P><P>Litmus28ibb-J04450
</P><H2><SPAN class="mw-headline" id="Week_17">Week 17</SPAN></H2><P><B>8/27</B></P><P>Transform pSB1C3-M13ori into ER2738 cells
</P><P><B>8/28</B></P><P>Make phage with pSB1C3-M13ori as phagemid and M13g6A1 as helper phagemid
</P><P><B>8/29</B></P><P>Finish phage isolation of M13 phage containing pSB1C3-M13ori
</P><P><B>9/3</B></P><P>digest pSB6A1-M13g and pSB3C5 with EcoRI and PstI. Did not receive a band for M13genes so did not proceed with ligation to pSB3C5
</P><H2><SPAN class="mw-headline" id="Week_18">Week 18</SPAN></H2><P><B>9/4</B></P><P>Mini-prepped new pSB6A1-M13genes (bad quality) and pSB3C5-J04550. Digested both plasmids with EcoRI and PstI and ran products on gel. Did receive a band for the M13genes but it was too feint to gel extract
</P><P><B>9/5</B></P><P>Mini-prep pSB6A1-M13genes again. Digest this and pSB3C5 backbone from previous day with EcoRI and PstI. M14g band was barely bright enough to extract so we extracted M13genes and pSB3C5 backbone. Pieces were ligated together overnight at 16C for 10 hours.
</P><P><B>9/7</B></P><P>Transformed 9/5 ligation (pSB3C5-M13genes into 5alpha cells). Plated sample onto low Chlor plate (34ug/mL)
</P><P><B>9/9</B></P><P>Still no growth for 9/7 transformation of pSB3C5-M13genes ligation PCR M13genes to attempt another ligation into pSB3C5. Diluted DNA to ~1.17ng/ul
</P><P><B>9/10</B></P><P>Gel of 9/9 PCR of M13genes looked clean (only ran 3ul of PCR sample on gel). PCR purified the remaining PCR sample. Digested with EcoRI and PstI. Ligated to previously digested pSB3C5.
</P><H2><SPAN class="mw-headline" id="Week_19">Week 19</SPAN></H2><P><B>9/11</B></P><P>Transform 9/10 ligation (pSB3C5-M13genes) into 5alpha cells
</P><P><B>9/13</B></P><P>Colony PCR of colonies that grew for 9/11 transformation. Looking for pSB3C5-M13genes)
</P><P><B>9/15</B></P><P>Gel of colony PCR showed failure. Set up overnight cultures anyway.
</P><P><B>9/17</B></P><P>Submitted Part to iGEM Registry
</P><TABLE class="wikitable"><TBODY><TR><TD>iGEM #
</TD><TD>Description
</TD></TR><TR><TD>BBa_K1445000
</TD><TD>M13ori- the packaging signal for the M13 and fd phage
</TD></TR></TBODY></TABLE><P>Mini-prepped overnights from 9/15 and sent samples for sequencing
</P><P>Sent Litmus28ibb-J04450 sample for sequencing to sequence the remaining section that was not reached before
</P><H2><SPAN class="mw-headline" id="Week_20">Week 20</SPAN></H2><P><B>9/21</B></P><P>Up until this point, very little luck was had with pSB3C5. Retried putting M13genes onto a chloramphenicol backbone, but this time used pSB1C3. Digested M13genes and pSB1C3 with EcoRI and PstI. Gel showed incomplete digestion so did not gel extract.
</P><P><B>9/22</B></P><P>Digested pSB1C3-J04450 with EcoRI and PstI then gel extracted the backbone. PCR purified M13genes(PCR product was clean so did not bother to gel extract). Ligated the two pieces together.
</P><P><B>9/23</B></P><P>Transform ligations from 9/22 (pSB1C3-M13genes) into 5 alpha cells
</P><P><B>9/27</B></P><P>Transform pSB1C3-M13genes into 5alpha cells. Plate onto low Chlor (34ug/mL)
</P><P>Later in day.... Set up overnight cultures of colonies
</P><P><B>9/28</B></P><P>Mini-prepped O/N cultures from 9/27. Digested samples with EcoRI and PstI. There were no bands of the correct size on the gel.
</P><DIV class="newfooter"><DIV id="newfooterbox"><DIV class="footertext"><UL><LI><A href="https://igem.org/About">About</A></LI><LI><A href="https://igem.org/Contact">Contact</A></LI><LI><A href="https://igem.org/Sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Main_Page">  igem.org</A></LI><LI><A href="http://parts.igem.org/Main_Page"> Registry</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://2014.igem.org/Requirements"> Requirements</A></LI><LI><A href="https://2014.igem.org/Giant_Jamboree"> Giant Jamboree</A></LI><LI><A href="https://2014.igem.org/Meetups"> Meetups</A></LI><LI><A href="https://2014.igem.org/Calendar_of_Events">Calendar </A></LI><LI><A href="https://2014.igem.org/Partner_Offers">Partner Offers</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Login">Login</A></LI><LI><A href="https://2014.igem.org/Special:Upload">Upload Files</A></LI><LI><A href="https://2014.igem.org/Special:RecentChanges"> Recent Changes</A></LI><LI><A href="https://2014.igem.org/Special:SpecialPages"> Special Pages</A></LI><LI><A href="https://2014.igem.org/Special:Search"> Search</A></LI></UL></DIV><DIV class="footertext"><UL><LI><A href="https://igem.org/Press_Kit">Press Kit</A></LI><LI><A href="https://igem.org/Newsletters"> Newsletter</A></LI><LI><A href="https://igem.org/Alumni">AlumniGEM</A></LI></UL></DIV><DIV class="footerextra">
powered by <A href="http://www.mediawiki.org/" style="link-style:none;">MediaWiki</A></DIV></DIV><P>the M13 and fd phage
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:CU-Boulder/Notebook/Phage_Team">http://2014.igem.org/Team:CU-Boulder/Notebook/Phage_Team</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:CU-Boulder/Notebook/Phage_Team" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:CU-Boulder/Notebook/Phage_Team" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/Phage_Team&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:CU-Boulder/Notebook/Phage_Team&amp;oldid=306764" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>