<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Goettingen">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Modeling">MODELING</A></LI></DIV><DIV id="contentContainer"><H2> Project Results</H2><A href="" onclick=" $('#menu1').slideToggle(300, function callback() {  }); return false;"><H1>Transformation Efficiency Kit, RFP construct (iGEM)</H1></A><DIV id="menu1"><P>Before using our competent <EM>E. coli</EM> TOP10 cells in the important experiments, we used the Transformation Efficiency Kit to test the efficiency of our competent cells!</P><P>The kit includes five vials of each different DNA concentration: 50 pg/μl, 20 pg/μl, 10 pg/μl, 5 pg/μl, 0.5 pg/μl of purified DNA from BBa_J04450 (RFP construct) in plasmid backbone pSB1C3.</P><P> </P><P>The first test transformation (50 μl of competent cells, 20 μl plated) showed very poor results:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P style="text-align: justify;">First test transformation of own competent <EM>E.coli</EM> TOP10 with the iGEM Transformation Efficiency Kit containing RFP</P></DIV></DIV></DIV></DIV><P align="center">DNA concentration</P><P align="center">0.5 pg/μl</P><P align="center">5 pg/μl</P><P align="center">10 pg/μl</P><P align="center">20 pg/μl</P><P align="center">50 pg/μl</P><P align="center"># of colonies</P><P align="center">0</P><P align="center">0</P><P align="center">2</P><P align="center">0</P><P align="center">13</P><P align="center">efficiency (cfu)</P><P align="center">0</P><P align="center">0</P><P align="center">3.8x 10<SUP>6</SUP></P><P align="center">0</P><P align="center">3.4x 10<SUP>6</SUP></P><P> </P><TABLE style="width: 240px;" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="bottom" nowrap="nowrap" width="172"><P><STRONG>Results</STRONG></P></TD><TD valign="bottom" nowrap="nowrap" width="68"><P><STRONG>efficiency</STRONG></P></TD></TR><TR><TD valign="bottom" nowrap="nowrap" width="172"><P><STRONG>average</STRONG></P></TD><TD valign="bottom" nowrap="nowrap" width="68"><P align="right">1.4x 10<SUP>6</SUP></P></TD></TR><TR><TD valign="bottom" nowrap="nowrap" width="172"><P><STRONG>weighted</STRONG></P></TD><TD valign="bottom" nowrap="nowrap" width="68"><P align="right">3.5x 10<SUP>6</SUP></P></TD></TR></TBODY></TABLE><P>The second test transformation (200 μl of competent cells, 100 μl plated) showed still poor results but we decided to continue working with our cells:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P style="text-align: justify;">Second test transformation of own competent <EM>E.coli</EM> TOP10 with the iGEM Transformation Efficiency Kit containing RFP</P></DIV></DIV></DIV></DIV><P align="center">DNA concentration</P><P align="center">0.5 pg/μl</P><P align="center">5 pg/μl</P><P align="center">10 pg/μl</P><P align="center">20 pg/μl</P><P align="center">50 pg/μl</P><P align="center"># of colonies</P><P align="center">1</P><P align="center">13</P><P align="center">1</P><P align="center">35</P><P align="center">28</P><P align="center">efficiency (cfu)</P><P align="center">8.0x 10<SUP>6</SUP></P><P align="center">1.0x 10<SUP>7</SUP></P><P align="center">4.0x 10<SUP>5</SUP></P><P align="center">7.0x 10<SUP>6</SUP></P><P align="center">2.3x 10<SUP>6</SUP></P><P> </P><TABLE style="width: 240px;" border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="bottom" nowrap="nowrap" width="172"><P><STRONG>Results</STRONG></P></TD><TD valign="bottom" nowrap="nowrap" width="68"><P><STRONG>efficiency</STRONG></P></TD></TR><TR><TD valign="bottom" nowrap="nowrap" width="172"><P><STRONG>average</STRONG></P></TD><TD valign="bottom" nowrap="nowrap" width="68"><P align="right">5.6x 10<SUP>6</SUP></P></TD></TR><TR><TD valign="bottom" nowrap="nowrap" width="172"><P><STRONG>weighted</STRONG></P></TD><TD valign="bottom" nowrap="nowrap" width="68"><P align="right">3.7x 10<SUP>6</SUP></P></TD></TR></TBODY></TABLE></DIV><A href="" onclick=" $('#menu2').slideToggle(300, function callback() {  }); return false;"><H1>RFP</H1></A><DIV id="menu2"><P><STRONG>RFP</STRONG></P><P>RFP (RFP DsRed) was amplified from pHT315_rfp by PCR (Fig.1). Colonies on a plate were given to us by the Applied and Genomic Microbiology department. Primers contained restriction sites for <EM>Kpn</EM>I and <EM>Sac</EM>I in order to make them compatible for insertion into the multiple cloning site of the pBAD/His A vector. The PCR products were examined in a 0,8% agarose gel by electrophoresis.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:476px;"><DIV class="thumbcaption"><P style="text-align: justify;">Fig.1) A gel of the RFP DsRed PCR product showed the bands to be at the correct height.</P></DIV></DIV></DIV></DIV><P>After purification of the PCR product it was ligated into pJET1.2 by subcloning (blunt end ligation). This vector serves to clone the fluorescent protein without triggering its activity, which may interact with the expression vector or the chosen <EM>E.coli </EM>strain.</P><P>Ligation into pJET1.2 was followed according to the protocol in the methods collection. After over-night incubation colonies were picked, plasmids extracted with the QIAGEN QIAprep Spin Miniprep Kit, restricted with <EM>Kpn</EM>I and <EM>Sac</EM>I and examined in a gel (Fig.2).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><P style="text-align: justify;">Fig.2) Once restriction controls showed the correct bands, both pJET_RFP_3 and pJET_RFP_7 were sent for sequencing by the G2L laboratory.</P></DIV></DIV></DIV></DIV><P>Sequencing of both samples was correct and proved that RFP_3 and RFP_7 were properly inserted into the pJET vector. We decided to keep on working with pJET_RFP_3. The plasmid was transformed with <EM>E.coli</EM> TOP10 and <EM>E.coli</EM> BL21. Cryocultures were frozen for a strain collection.</P><P>The next step was a big restriction of the whole plasmid extract with the correspondent pair of restriction enzymes. That allowed us to isolate the RFP inserts carrying the desired restriction sites (<EM>Kpn</EM>I and <EM>Sac</EM>I). The fragments were purified with the PEQLAB peqGOLD Gel Extraction Kit before T4 ligation. The RFP_3 insert was ligated into pBAD/His A by the T4 ligation system (sticky end ligation) according to the protocol in the methods collection and transformed into <EM>E.coli</EM> TOP10. This vector serves to express the fluorescent protein by triggering its activity.</P> After over-night incubation colonies were picked, plasmids extracted with the QIAGEN QIAprep Spin Miniprep Kit and restricted with <EM>Kpn</EM>I and <EM>Sac</EM>I.<P>To build a component for our Flexosome, we decided to fuse pBAD_RFP with the ACEL (<EM>Acetivibrio cellulolyticus</EM>) dockerin. Both components, restricted with <EM>Kpn</EM>I and <EM>Sac</EM>I and purified with the PEQLAB peqGOLD Gel Extraction Kit were ligated by the T4 ligation system. After transformation into <EM>E.coli</EM> TOP10 and over-night incubation colonies were picked, plasmids extracted with the QIAGEN QIAprep Spin Miniprep Kit, restricted with <EM>Eco</EM>RI and examined in a gel (Fig.3).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><P style="text-align: justify;">Fig.3) Once restriction control showed the correct band height, pBAD_RFP_ACEL was sent for sequencing by the G2L laboratory.</P></DIV></DIV></DIV></DIV><P><STRONG> </STRONG></P><P><STRONG>RESULTS</STRONG></P><P>Sanger sequencing showed correct sequences for the Flexosome bit pBAD_RFP_ACEL.</P><P> </P><P><STRONG>MICROSCOPY</STRONG></P><P><SPAN lang="EN-US">To check the activity of RFP, an induction series with 0,2%, 2% and 5% L-Arabinose in the cell culture medium was prepared. Fluorescence microscopy was performed with the RFP DsRed filter (excitation at 536 nm, emission at 582 nm). Unfortunately no fluorescence could be detected throughout the whole project with different constructs. Neither in the pJET_RFP_3, nor in the pBAD constructs (pBAD_RFP_3 and pBAD_RFP_ACEL) induced with 0,2%, 2% and 5% L-Arabinose in the medium. Since the DNA sequences have been confirmed to be correct by Sanger sequencing it might be a problem of expression. 
</SPAN></P><P> </P><P><STRONG>FUTURE PLANS</STRONG></P>In future we want to exchange our vector system (pET100 instead of pBAD) to ensure that pBAD is not somehow interfering with our chosen <EM>E. coli</EM> strain.</DIV><A href="" onclick=" $('#menu3').slideToggle(300, function callback() {  }); return false;"><H1>Esterase and Phosphatase</H1></A><DIV id="menu3"><P><STRONG>Phosphatase and Esterase</STRONG></P><P>
    Esterase was amplified from pET101_E064 and phosphatase from pTOPOXL_PLP07 by PCR for each enzyme (Fig.1). Both plasmids were given to us by the Applied
    and Genomic Microbiology department. Primers contained restriction sites <EM>Kpn</EM>I and <EM>Sac</EM>I for esterase and <EM>Xho</EM>I and <EM>Pst</EM>I for the phosphatase in order to make
    them compatible for insertion into the multiple cloning site of the pBAD vector. The genes for these two enzymes had been found in screenings of
    metagenomic libraries.
</P><P>
    After purification of the PCR products, they were ligated into pJET by blunt end ligation. This vector serves to clone the enzymes without triggering their
    activity, which may interact with the vector or the <EM>E.coli</EM>. In the case of phosphatase, the PCR products needed to be purified by gel extraction,
    to eliminate left over template plasmids, which could interfere with the transformation.
</P><P>
    Ligation into pJET was followed according to the protocol in the methods collection. After over-night incubation colonies were picked, plasmids extracted
    with a Quiaprep spin Miniprep kit and restricted with either <EM>Kpn</EM>I and <EM>Sac</EM>I or <EM>Xho</EM>I and <EM>Pst</EM>I, depending on the Enzyme (Fig.2&amp;3).
</P><P>
    Once restrictions showed the correct bands, both pJET containing esterase and phosphatase were sent for sequencing by the G2L laboratory.
</P><P>
    The sequencing was successful and proved that both enzymes were properly inserted into the vector. The next step was a big restriction of the whole plasmid
    extract with the correspondent pair of restriction enzymes. That allowed us to isolate the enzyme inserts carrying the desired restriction sites. The
    fragments were purified with the peqGOLD Gel extraction kit.
</P><P>
    The esterase was then ligated into the pBAD vector containing the CCEL (<EM>Clostridium cellulolyticum</EM>) dockering. Competent <EM>E. coli</EM> Top10
    cells were transformed and the pBAD_Est_CCEL construct was isolated from the positive clones with a Quiaprep spin Miniprep kit.
</P><P>
    A restriction control with <EM>Sac</EM>I for the esterase constructs was run in a 0,8% agarose gel (Fig. 4). Samples #2, #3 and #9 correspondto the linear plasmid containing the insert.  Phosphatase construct was control restricted with <EM>EcoR</EM>I (Fig. 5), in this case 13 of 14 samples showed successful insertion of the enzyme fragment into the vector.
</P><P><STRONG>Future plans</STRONG></P><P>
    The next steps include changing to a better expression vector in order to test and measure the activity of the enzymes, especially to ensure that the fusion with the dockerins is not interfering with the function of the enzymes. Finally the aim is to combine the purified enzyme-dockerin proteins with the synthetic scaffold and prove the self assemby of the Flexosome tool.
</P></DIV><A href="" onclick=" $('#menu5').slideToggle(300, function callback() {  }); return false;"><H1>Cellulase</H1></A><DIV id="menu5"><P>
    The cellulase gene was amplified using PCR. The primers which were used contained the restriction sites for ligation into the pBAD vector. The cellulase
    PCR products were then checked on 0.8 % agarose gel.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:217px;"><DIV class="thumbcaption">Figure 1: Cellulase PCR product was run on a 0.8 % agarose gel. The desired cellulase sequence is at 2496 bp.</DIV></DIV></DIV></DIV><P>
    The cellulase PCR products were then purified by using the <STRONG>QIAquick® PCR Purification Kit (QIAGEN). </STRONG>After the PCR products were purified,
    the concentration of DNA was measured by Nanodrop 2000c Spectrophotometer.</P><P><STRONG>Restriction control of pBAD_CCEL and cellulase PCR product</STRONG></P><P>
    The purified cellulase PCR product and the pBAD_CCEL vector were restricted by following a double digest protocol with SacI and PvuII enzymes. After
    restriction control products were purified by using the <STRONG>QIAquick® PCR Purification Kit (QIAGEN) </STRONG>the concentration of DNA was measured by
    Nanodrop 2000c Spectrophotometer.</P><P><STRONG>Ligation of Cellulase into pBAD- CCEL</STRONG></P><P>
    The cellulase gene was then ligated into the pBAD vector containing the CCEL (<EM>Clostridium</EM></P><P><EM>cellulolyticum</EM>
    ) dockerin.
</P><P>
    For ligation of the cellulase PCR product into the restricted pBAD_CCEL vector, the Thermo Fisher T4 Ligase Sticky End Ligation protocol was followed.
</P><P><STRONG>Transformation of pBAD_CellulaseCCEL into chemically competent Top10 <EM>E. coli </EM>cells</STRONG></P><P>
    pBAD_CellulaseCCEL was transformed into competent Top10 <EM>E. coli </EM>cells by using the heat shock transformation method.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 2: Transformation of pBAD_CellulaseCCEL into competent Top 10 E. coli cells. The image shows plates with successful transformants.</DIV></DIV></DIV></DIV><P>
    Colonies from transformation plates were inoculated into LB medium containing ampicillin (100 µg/ml).
</P><P><STRONG>Plasmid Extraction: peqGold Plasmid Miniprep Kit</STRONG></P><P>
    Plasmid extraction was done for colonies inoculated in LB medium by using peqGold Plasmid Miniprep kit and plasmid concentration was measured by using
    Nanodrop 2000c Spectrophotometer.
</P><P>
    Restriction control was done with the enzymes SacI and PvuII from Thermo fisher scientific.Restricted plasmids were checked on a 0.8 %
    agarose gel.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 3: Restriction control of the pBAD_CellulaseCCEL construct from TOP10 E. coli positive clones by with SacI and PvuII.  Colony 1 has the desired insert.</DIV></DIV></DIV></DIV><P>
    The correct insert was confirmed through sequencing.
</P></DIV><A href="" onclick=" $('#menu4').slideToggle(300, function callback() {  }); return false;"><H1>BioBricks</H1></A><DIV id="menu4"><P><STRONG>Dockerin Biobricks</STRONG></P><P>
The dockerins ACEL (<EM>Acetivibrio cellulolyticus</EM>), BCEL (<EM>Bacteroides cellulosolvens</EM>), CCEL (<EM>Clostridium cellulolyticum</EM>) and CTHE (<EM>Clostridium thermocellum</EM>) were amplified from pBAD_ACEL, pBAD_BCEL, pBAD_CCEL and pBAD_CTHE with primers to add the desired <EM>Eco</EM>RI and    <EM>Pst</EM>I restriction sites by PCR. Primers contained restriction sites for <EM>Eco</EM>RI and <EM>Pst</EM>I in order to make them compatible for
    insertion into the iGEM shipping vector pSB1C3.
</P><P>
    After purification, the PCR products were restricted with <EM>Eco</EM>RI and <EM>Pst</EM>I, as well as pSB1C3. Afterwards the restricted dockerins were
    ligated into pSB1C3 by T4 ligation (sticky end ligation) and transformed into <EM>E. coli</EM> TOP10.
</P><P>
    Ligation into pSB1C3 was followed according to the protocol in the methods collection. After over-night incubation colonies were picked, plasmids extracted
    with the QIAGEN QIAprep Spin Miniprep Kit and test restricted with <EM>Eco</EM>RI (pSB1C3_CCEL and pSB1C3_ACEL) and <EM>Eco</EM>RI and <EM>Pst</EM>I
    (pSB1C3_BCEL) (Fig.1). We lost CTHE due to failing transformations at this point.
</P><P>
    Once restriction controls showed the correct bands, pSB1C3_CCEL, pSB1C3_ACEL and pSB1C3_BCEL were sent for sequencing.
</P><P> </P><P><STRONG>Colour BioBricks</STRONG></P><P>
    We also wanted to improve already existing BioBricks by fusing our dockerins to the colours eforRed (<A href="http://parts.igem.org/Part:BBa_K592012" style="color: #000066">BBa_K592012</A>) and amilCP (<A href="http://parts.igem.org/Part:BBa_K592009" style="color: #000066">BBa_K592009</A>) that were
    submitted from the University of Uppsala (Sweden) in 2011.
</P><P>
    So we chose to fuse eforRed to BCEL and amilCP to CCEL. Since the plasmids of the two colour proteins were not distributed with the current plate of the
    iGEM Distribution Kit, we ordered those parts including the right restriction sites as gBlocks from IDT.
</P><P>
    As a first step eforRed and amilCP were restricted with <EM>Kpn</EM>I and <EM>Sac</EM>I to make them compatible with our dockerins in pBAD.
</P><P>
    After purification restricted colours were ligated into pBAD_BCEL and pBAD_CCEL by T4 ligation (sticky end ligation) and transformed into <EM>E. coli</EM>
    TOP10.
</P><P>
    Ligation into pBAD was followed according to the protocol in the methods collection. After over-night incubation colonies were picked, plasmids extracted
    with the QIAGEN QIAprep Spin Miniprep Kit and test restricted with <EM>Kpn</EM>I (Fig.2).
</P><P>
    Once restriction controls showed the correct bands, pBAD_amilCP_CCEL #1 and pBAD_eforRed_BCEL were sent for sequencing. Sequencing showed that both
    plasmids were correct.
</P><P>
    To make our constructs compatible with the iGEM shipping vector pSB1C3 the desired <EM>Eco</EM>RI and <EM>Pst</EM>I restrictions sites were added by PCR.
</P><P>
    After purification, the PCR products were restricted with <EM>Eco</EM>RI and <EM>Pst</EM>I, the same with pSB1C3, and afterwards ligated into pSB1C3 by T4
    ligation (sticky end ligation) and transformed into <EM>E. coli</EM> TOP10.
</P><P>
    After over-night incubation colonies were picked, plasmids extracted with the QIAGEN QIAprep Spin Miniprep Kit and test restricted with <EM>Eco</EM>RI
    (Fig.3). Due to the reason that our dockerin CCEL contains an internal <EM>Pst</EM>I restriction site we continued working only with the eforRed_BCEL
    construct.
</P><P>
    Once restriction controls showed the correct band, pSB1C3_eforRed_BCEL was sent for sequencing. Sequencing showed that the plasmids were correct.
</P><P><STRONG>RESULTS</STRONG></P><P>
    Sequencing showed that the dockerins ACEL and CCEL contain an internal <EM>Pst</EM>I restriction site. Therefore all the constructs containing ACEL or CCEL
    showed in the end truncated dockerin sequences and could not be send in as BioBricks.
</P><P>
    We also build a construct where the enzyme esterase was fused to our CCEL dockerin (pBAD_Est_CCEL) following the same strategy. But again, due to the
    reason that our dockerin CCEL contains an internal <EM>Pst</EM>I restriction site we had to stop our work at this point.
</P><P>
    Furthermore we <A href="https://2015.igem.org/Team:Goettingen/Collaborations" style="color: #000066">collaborated</A> with the current iGEM team of Aachen and tried to fuse three of their enzymes to our constructs but could not finish our work here.
</P><P><STRONG>ACHIEVENTS</STRONG></P><P><STRONG>Sequences of pSB1C3_BCEL and pSB1C3_eforRed_BCEL were correct and submitted as BioBricks to iGEM:</STRONG></P><P>
    pSB1C3_BCEL (<A href="http://parts.igem.org/Part:BBa_K1865000" style="color: #000066">BBa_K1865000</A>)
</P><P>
    pSB1C3_eforRed_BCEL (<A href="http://parts.igem.org/Part:BBa_K1865001" style="color: #000066">BBa_K1865001</A>)
</P></DIV><A href="" onclick=" $('#menu6').slideToggle(300, function callback() {  }); return false;"><H1>Scaffoldin Purification</H1></A><DIV id="menu6"><P><STRONG>Scaffoldin purification</STRONG></P><P>
    After induction according to the methods collection, the targeted ScaA scaffoldin was purified by nickel affinity chromatography using Ni-NTA-agarose columns in a Äkta prime system (Fig. 1A). Following the addition of Imidazol two peaks were obtained. The fractions of interest were visualized in a SDS-PAGE gel (Fig. 1B). The gel showed that only the second fraction presents the expected size of 111kDa. However some other bands appeared along this fraction, therefore in order to obtain even more pure protein a Size Exclusion Chromatography (SEC) was done.
</P><P>
    The SEC was done with a HiLoad Superdex 200 16/600 column (GE Healthcare) (Fig. 2A). SDS gel shows that a high concentration of the desired protein was
    obtained and most of the previous contaminating bands were eliminated. Nevertheless there was an unknown band of low weight present in every fraction,
    apparently it could be a second protein from E. coli with some affinity to the Scaffoldin than is then is co-purified with it (Fig. 2B).
</P><P>
  Future plans include to incubate the Scafoldin with the purified enzyme-dockerin construct in order to prove that the Flexosome can assembly by itself.
</P></DIV><A href="" onclick=" $('#menu7').slideToggle(300, function callback() {  }); return false;"><H1>Copurification</H1></A><DIV id="menu7"><P>
    Copurification of Scaffoldin and Enzyme-Dockerin
</P><P>
    The main peaks after gelfiltration from the scaffoldin CTHE (see results scaffoldin) and CellulaseCCEL (Fig.1) were pooled and concentrated.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">B1-B3 were pooled for copurification.</DIV></DIV></DIV></DIV><P>
    They were then mixed and incubated together at 37°C for 15 min with agitation.
</P><P>
    The solution was then used in another gelfiltration (Fig.2). The resulting peaks were run on a SDS-gel (Fig.3) and checked for cellulase activity.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Fig.2) The peak at A5-A7 may be the bound scaffoldin and dockerin, whilst the unbound  proteins are expected between B3 and C9.</DIV></DIV></DIV></DIV><P>
    The first peak from the copurification showed a distinct cellulase activity. As the peak ran considerably higher than expected than either the scaffoldin
    or the CellulaseCCEL in the gelfitration, it can be expected that this peak contains a successfully formed complex between CellulaseCCEL and the CTHE
    scaffoldin.
</P></DIV><A href="" onclick=" $('#menu8').slideToggle(300, function callback() {  }); return false;"><H1>Giant Jamboree: Poster and Presentation</H1></A><DIV id="menu8"><P>
Did you missed our presentation? Do you want to see our poster again?
No problem, here you can find the files:
</P><P><A href="/wiki/images/a/a0/IGEM-presentation_final_team_Goettingen.ppt" class="internal" title="IGEM-presentation final team Goettingen.ppt">Media:IGEM-presentation_final_team_Goettingen.ppt</A></P><P><A href="/wiki/images/7/7e/Poster_teamGoettingen_2015.pdf" class="internal" title="Poster teamGoettingen 2015.pdf">Media:Poster_teamGoettingen_2015.pdf</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>