<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Goettingen">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Modeling">MODELING</A></LI></DIV><DIV id="contentContainer"><H1><B> About our Project </B></H1><H2> Our project </H2>

The <B>iGEM team Goettingen 2015</B> is currently developing the “<B>Flexosome</B>”. 
It is our <B>enzymatic penknife</B>: a customizable complex for more efficient and <B>synergistic multi-enzymatic processes</B>. It consists of a <B>scaffoldin, dockerins</B> and  <B>exchangeable enzymes</B>.
The enzymes of interest can be attached to the scaffoldin via the dockerin stations and once attached complete the reactions.

<H2> The first step</H2>

Right now the team is working on a proof of concept. Our first Flexosome will contain <B>phosphatase, esterase</B> and <B>cellulase</B> found in metagenomic libraries along with RFP, a fluorescent reporter gene. Once our <EM>E. coli</EM> is able to express and produce this functional Flexosome, this will open the door for more complex and specific applications.

<H2> The inspiration </H2>

We studied the natural <B>cellulosome structure</B>, in which different cellulotytic enzymes are assembled into a bigger scaffoldin via dockerin stations, leading to optimized cellulose degradation. The interaction between scaffoldin and dockerins has been proven to be <B>strong and stable</B>, this gave us the ideal platform to design our <B>&quot;Flexosome”</B> device.

Traditionally cellulosome engineering is thought for optimization of cellulose degradation, but we don’t have to stop there! A much broader range of enzymes are actively being discovered every year. That is why we intend to combine the scaffoldin protein from cellulolytic bacterium with varying and exchangeable enzymes. The result: a very <B>versatile, stable and innovative tool</B>!

<H2> The goal </H2>

We want to achieve a protein construct which will be able to ensure a <B>high local concentration</B> of enzymes and <B>catalyse multiple enzymatic processes</B> at one site, making the product of one process the substrate of the next on the scaffoldin. Ultimately, the construct will not only <B>increase efficiency</B> of the reactions but also be <B>fully customisable</B> for any kind of enzymatic process.


<H2>
    BioBricks
</H2><P>
    As our flexosome is built from multiple different domains, we decided that our BioBricks would also consist of these building blocks. Therefore we created
    BioBricks containing the dockerins alone, as well as in conjunction with a protein. The original plan was to add the enzymes we were using for our proof of
    principle.
</P><P>
    In addition, we decided to use BioBricks from previous and current teams and attach them to our construct, as well as creating composite parts, which would
    be compatible with our flexosome or any other construct containing the corresponding cohesin domains. We <A href="https://2015.igem.org/Team:Goettingen/Collaborations">collaborated</A> on this with <A href="https://2015.igem.org/Team:Aachen">Aachen</A> and also utilised
    some previous <A href="https://2011.igem.org/Team:Uppsala-Sweden/Parts">parts from Uppsala</A> (2011).
</P><P>
    In the end we were able to design a basic part containing the <EM>Bacteroides cellulolyticus </EM>dockerin domain, and a composite part containing the same
    domain together with eforRed from Team Uppsala 2011. For further information please refer to the <A href="http://parts.igem.org/Main_Page">registry</A> or our <A href="https://2015.igem.org/Team:Goettingen/TeamParts">Parts</A> section.
</P><H2> References </H2><P><STRONG>General project background</STRONG></P><P><STRONG>Bayer, E. A., Lamed, R., White, B. A., Flint, H. J.</STRONG> (2008) <A href=" http://onlinelibrary.wiley.com/doi/10.1002/tcr.20160/abstract"> From Cellulosomes to Cellulosomics. </A><EM>The Chemical Record.</EM> Volume 8, Pp. 364 – 377</P><P><STRONG>Blumer-Schuette, S. E., Brown, S. D., Sander, K. B., Bayer, E. A., Kataeva, I., Zurawski, J. V., Conway, J.V., Adams, M. W. W., Kelly, R. M.</STRONG> (2013) <A href=" http://onlinelibrary.wiley.com/doi/10.1111/1574-6976.12044/abstract"> Thermophilic lignocellulose deconstruction.</A><EM>FEMS Microbiology Reviews.</EM> Volume 38, Issue 3, Pp. 393–448</P><P><STRONG>Chen R., Chen Q., Kim, H., Siu K.-H., Sun, Q., Tsai, S.-L., Chen W.</STRONG> (2014) <A href="http://www.sciencedirect.com/science/article/pii/S0958166913007088"> Biomolecular scaffolds for enhanced signaling and catalytic efficiency.</A><EM> Current Opinion in Biotechnology.</EM> Volume 28, Pp. 59 – 68</P><P><STRONG>Ding, S.-Y., Bayer E., A, Steiner D., Shoham, Y. Lamed, R. </STRONG>(2000) <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC111372/"> A Scaffoldin of the Bacteroides cellulosolvens Cellulosome That Contains 11 Type II Cohesins.</A> Volume 182, Issue 17, Pp. 4915 – 4925</P><P><STRONG>Doi, R. H., Kosugi, A.</STRONG> (2004) <A href=" http://www.nature.com/nrmicro/journal/v2/n7/abs/nrmicro925.html"> Cellulosomes: Plant-Cell-Wall-Degrading Enzyme Complexes.  </A><EM>Nature Reviews Microbiology. </EM>Volume 2, Pp. 541 – 551</P><P><STRONG>Fontes, C. M.G.A., Gilbert, H. J. </STRONG>(2010) <A href=" http://www.annualreviews.org/doi/abs/10.1146/annurev-biochem-091208-085603"> Cellulosomes: Highly Efficient Nanomachines Designed to Deconstruct Plant Cell Wall Complex Carbohydrates. </A><EM>Annual Review of Biochemistry</EM>. Volume 79, Pp. 655 – 681</P><P><STRONG>Hirakawa, H. Haga, T., Nagamune, T. </STRONG>(2012) <A href=" http://www.springer.com">Artificial Protein Complexes for Biocatalysis.</A><EM>Topics in Catalysis. </EM>Volume 55, Pp. 1124 –1137</P><P><STRONG>Hyeon, J. E., Jeon, S. D., Han, S. O.</STRONG> (2013) <A href="http://www.ncbi.nlm.nih.gov/pubmed/23563098"> Cellulosome-based, Clostridium-derived multi-functional enzyme complexes for advanced biotechnology tool development: advances and applications.</A><EM>Biotechnology Advances.</EM> Volume 31, Pp. 936 – 944</P><P><STRONG>Shoseyov, O., Takagi, M., Goldstein, M., A., Doi R. H.</STRONG> (1992) <A href=" http://www.ncbi.nlm.nih.gov/pmc/articles/PMC48892/"> Primary sequence analysis of Clostridium cellulovorans cellulose binding protein A.</A><EM>Biochemistry.</EM> Volume 89, Pp. 3483 – 3487</P><P><STRONG>Xu, Q., Bayer, E. A., Goldman, M., Kenig, R. Shoham, Y. Lamed, R. </STRONG>(2004) 
<A href="http://jb.asm.org/content/186/4/968.full"> Architecture of the Bacteroides cellulosolvens Cellulosome: Description of a Cell Surface-Anchoring Scaffoldin and a Family 48 Cellulase.</A><EM>Journal of Bacteriology.</EM> Volume 186, Issue 4, Pp. 968 – 977</P><P> </P><P><STRONG>Esterase</STRONG></P><P><STRONG>Montella I. R., Schama R., Valle D.</STRONG> (2012) <A href=" http://www.ncbi.nlm.nih.gov/pubmed/22666852"> The classification of esterases: an important gene family involved in insecticide resistance--a review.</A>Volume 107, Issue 4, Pp. 437-449</P><P><STRONG>Nacke, H., Will, C., Herzog, S., Nowka, B., Engelhaupt, M., Daniel, R.</STRONG> (2011) <A href="http://femsec.oxfordjournals.org/content/78/1/188.abstract"> Identification of novel lipolytic genes and gene families by screening of metagenomic libraries derived from soil samples of the German Biodiversity Exploratories.</A><EM>FEMS Microbiology Ecology.</EM> Volume 78, Issue 1, Pp. 188-201</P><P><STRONG>Pliego, J., Mateos, J. C., Rodriguez, J., Valero, F., Baeza, M., Femat, R., Camacho, R., Sandoval, G., Herrera-López, E. J.</STRONG> (2015) <A href=" http://www.ncbi.nlm.nih.gov/pubmed/25633600 "> Monitoring lipase/esterase activity by stopped flow in a sequential injection analysis system using p-nitrophenyl butyrate.</A><EM>Sensors.</EM> Volume 15, Issue 2, Pp. 2798-2811</P><P> </P><P><STRONG>Phosphatase</STRONG></P><P><STRONG>Castillo Villamizar, G., Nacke, H., Böhning, M., Elisabeth, G., Kaiser, K., Daniel, R.</STRONG> (2014) Novel phosphatases derived from functional metagenomics. 7<SUP>th</SUP> International Congress on Biocatalysis. <EM>Biocat2014 Hamburg. </EM>August 32 –September 4, 2014, P. 87</P><P><STRONG>Greiner, R., Farouk, A.-E. </STRONG>(2007) <A href=" http://link.springer.com/article/10.1007/s10930-007-9086-z"> Purification and Characterization of a Bacterial Phytase Whose Properties Make it Exceptionally Useful as a Feed Supplement.</A><EM>The Protein Journal. </EM>Volume 26, Issue 7, Pp. 467-474</P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>