<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Goettingen">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Goettingen/Modeling">MODELING</A></LI></DIV><DIV id="contentContainer"><H1> Notebook </H1><A href="" onclick=" $('#menu1').slideToggle(300, function callback() {  }); return false;"><H2>15.12.14</H2></A><DIV id="menu1"><P><STRONG>1.) Finding a topic</STRONG></P><P>
    After establishing who is interested in being a member of this year’s Team, everyone went out to find possible supervisors and projects. Prof. Rolf Daniel
    together with Dr. Elzbieta Brzuszkiewicz and Dr. Silja Brady, whom some of us know from the first microbiology lab course, have a project for us:
</P><P><STRONG>2.) Project Proposal</STRONG></P><P>
    We would create a cellulosome like multi enzyme scaffold as well as several linker regions with which
</P><P>
    enzymes of interest could be fused. This way, if for example a company wants to degrade different
</P><P>
    materials in a solution or wants a specific enzyme complex they would then list the needed enzymes
</P><P>
    and order them from us. We would then transform the enzymes before / after the linker regions, the
</P><P>
    enzymes would be assembled in the cell on the scaffold and then exported by the model organism.
</P><P>
    As the scaffold itself could be easily tagged, i.e. by a strep tag, we could then purify the multi enz
</P><P>
    yme complex and deliver it to the company. As there are enzymes for thousands of different reactions, the applicability would be broad. As Dr.
    Brzuszkiewicz and Dr. Brady weren't sure whe
</P><P>
    ther something like this has already been done, we will still have to do some research into the topic itself.
</P><P><STRONG>2.) Deciding which topic to choose</STRONG></P><P>
    As we currently have a grand total of one project, the question is whether we wait for other topics,
</P><P>
    as not all lecturers have answered yet or, if there are no objections, go ahead with the project of
</P><P>
    Prof. Daniel. For this, please sent me a mail with your thoughts until Sunday, as we need to decide fast.
</P><P><STRONG>3.) Distribution of tasks</STRONG></P><P>
    Everybody should look into Prof. Daniels topic, to see whether something similar has been done and
</P><P>
    what the current status on this research area is. In the coming days Verena will invite you to the
</P><P>  iGEM 2015 Dropbox folder, in which you can upload interesting papers.
</P><P><STRONG>Booklet:</STRONG>
    Dennis agreed to do the first draft over the holidays, Avril mentioned that she could proof
</P><P>
    read the english version, Stefani would help with the design and further improvement.
</P><P><STRONG>Fiscal management</STRONG>
    : David and Joshy have agreed to try and set up a preliminary budget plan.
</P><P>
    Logo: Stefani will design our first logo.
</P><P><STRONG>Wikipedia:</STRONG>
    Joshy agreed to handle our wikipedia for the time being.
</P><P><STRONG>Music video:</STRONG>
    Verena has big plans for a laboratory-themed song and corresponding music video.
</P><P><STRONG>Website:</STRONG>
    we still need someone who either knows a bit about website design or wants to learn how
</P><P>
    to set up a web site. We can use the old site as template, but we still need to convert/cannibalise it
</P><P>
    into our own. Please mail Verena or me if you would like to try your hand as a web designer.
</P><P><STRONG>4.) Dismissing members</STRONG></P><P>
    We are a comparatively small group with an ambitious project and a limited number of members. The coming months are very important, since this decides
    pretty much how easy we will have it later in the year. We currently have registered everybody who either mailed Verena or Julian or came to one of our
    meetings and are currently at 10 members total, 12 being the limit. If we get to the limit,we have to put people on the waiting list. Since we are a small
    group with an ambitious project, having someone in the group who doesn't bother to cancel to agreed appointments or, worse, won't do work she/he agreed on
    doing, would not only unnecessary aggravate the rest of the group but would also slow us down.
</P><P>
    Therefore, with the coming meeting, we will introduce a three strike system. If you can't come to a meeting, that's no problem - just tell someone who will
    attend in advance or post it on facebook. If you don't do so three times, you will be excluded from the iGEM group.
</P><P>
    If someone keeps making trouble within the group, the organisation team reserves the right to vote on excluding them.
</P></DIV><A href="" onclick=" $('#menu2').slideToggle(300, function callback() {  }); return false;"><H2>08.01.15</H2></A><DIV id="menu2"><P><STRONG>1.) Topic: </STRONG></P><P>
    We intend to utilise a scaffold protein in order to assemble enzymes with different functions into one complex, which will hopefully increase their
    efficiency as well as being customisable.
</P><P>
    As model organisms <EM>B.subtilis</EM> or <EM>E.coli</EM> seem to be to most viable at this point
</P><P>
    Issues may arise from the connection of linker regions to the enzymes and the subsequent loss of function.
</P><P><STRONG>2.) Booklet: </STRONG></P><P>
    Most parts of the booklet have been written by Dennis. Avril will proofread the sections and then pass them on to Stefani for layout and design.
</P><P>
    Further information needs to be added about the research projec, budget and sponsoring.
</P><P><STRONG>3.) Website/Wiki:</STRONG></P><P>
    Udhaya and Joschy will take care of the Wiki and Webpage together. Bing Yao from a previous Igem Team has offered to help and teach.
</P><P><STRONG>4.) Locations:</STRONG></P><P>
    Julian has asked Prof. Daniel to ask if someone has a space for us to have an office in their lab. Other options are the Schwann-Schleiden Research Centre,
    experimental medicine or the physics department.
</P><P><U>Update:</U>
    Verena has found out, that there is an iGem Computer in the Microbiology library.
</P><P>
    Usage of the practical room as labspace is in an agreement, and should therefore be possible.
</P><P>
    Equipment from the previous team will be assessed by Verena.
</P><P><STRONG>5.) Organisation Team</STRONG>
    was confirmed in its current setup of: Julian, Verena, Udhaya and Avril. If desired or the need should arise, the team can be extended further along the
    line.
</P><P><STRONG>6.) Finances:</STRONG></P><P>
    Budget for 3 different situations can be found in the dropbox (by Daniel). The booking prices are from the 5th of January and are subject to change.
</P><P>
    Visa prices need to be determined and passports updated.
</P><P>
    Julian will take over the Finances including the management of a bank account, bookings and the overview of our spending and financial support.
</P><P><U>Update:</U>
    200€ are left over from the previous Igem Team to be used by this Igem Team.
</P><P><STRONG>7.) Timetable and Deadlines:</STRONG></P><P>
    Julian has made a preliminary timetable (Dropbox). In addition to this, the deadline for the booklet will be the <STRONG>14th of January</STRONG> and the
    deadline for the first round of Sponsor contacting will be the <STRONG>18th of January</STRONG>. Unresponsive sponsors will be contacted weekly. The
    sponsor list will be split up between the team members at the next meeting (13th of January).
</P><P>
    The best weekdays for team meetings are Mondays and Thursdays .
</P><P><STRONG>8.) Roles:</STRONG></P><P>
    Press and Publicity: Steffi and Dennis
</P><P>
    Finances: Julian
</P><P>
    Sponsoring managers: Sindhu and Nida
</P></DIV><A href="" onclick=" $('#menu3').slideToggle(300, function callback() {  }); return false;"><H2>13.01.15</H2></A><DIV id="menu3"><P><STRONG>1.) Logo and Booklet:</STRONG></P><P>
    Stefani presented two ideas for a possible logo and asked for opinions. She will flesh them out a bit and we can decide on our next meeting which one we
    will choose as our main logo.
</P><P>
    Stefani also presented a possible design for our booklet.
</P><P>
    Udhaya had an idea for a possible slogan we could use, fitting one of our possible logos: “Placing enzymes where they should be”.
</P><P><STRONG>2. ) Sponsors:</STRONG></P><P>
    Sindhu will assign the companies we have so far, so everyone will have to contact 4 to 5.
</P><P><STRONG>3.) Equipment:</STRONG></P><P>
    Verena and Udhaya went through the inherited things from last year’s iGEM-Group. They are now listed in our inventory.
</P><P><STRONG>4.) Meetings:</STRONG></P><P>
    We decided to only hold meetings every two weeks, as every week seems a bit excessive, especially when we meet alternative on Mondays and Tuesdays. The
    schedule is in our timetable.
</P><P><STRONG>5.) Task for next week:</STRONG>
    Everybody should think about a fitting project title until our next meeting.
</P></DIV><A href="" onclick=" $('#menu4').slideToggle(300, function callback() {  }); return false;"><H2> 15.01.15 </H2></A><DIV id="menu4"><P><STRONG>Present:</STRONG>
    Verena, Alaa, Udhaya, Julian, Daniel, Avril, Silja, Ela, Prof. Daniel, Heiko Liesegang
</P><P>
    Supervisors will be met for monthly update
</P><P><STRONG>1.) Useful contacts:</STRONG></P><P>
    Supervisors Ela (Sequencing) and Silja (Biotech), Head of Unit Prof. Daniel, Bioinformatics Heiko Liesegang, Secretary Daniela, Lab Technicians: Ute &amp;
    Gabriele, Synthetic Bio Jun.-Prof. Neumann.
</P><P>
    Prof. Daniel will meet with Prof. Braus to discuss grants, which will probably require writing a formal letter
</P><P>
    Petra from Prof. Daniel's lab used to work as a webdesigner and can be asked for support on the webpage
</P><P>
    Supervisors will check the current sponsor list for additional contacts
</P><P>
    Silja will do the <STRONG>lab safety talk</STRONG> before we start the labwork
</P><P>
    Heiko Liesegang is happy to support the team with Bioinformatics and Industrial appliance of the project (he is situated in the cellar).
</P><P>
    Verena will add the supervisors to the dropbox, so they can have access to the list of sponsors, inventory, etc.</P><P><STRONG>2.) Project:</STRONG></P><P>
    The scaffold protein will be taken from <EM>Clostridium thermocellum</EM>, which is very well described and has been fully sequenced, as well as a known
    structure. The host will be codon optimised <EM>E.coli</EM>, which can be used with a vector system and can have the structure expressed intra- or
    extracellularly. -&gt; Proof of principle on <EM>E.coli</EM> before large scale adaptation with <EM>B.subtilis</EM></P><P><STRONG>3.) Research to be done by team:</STRONG></P><P>
    - Which enzymes: ideally well known
</P><P>
    - How many &quot;dockerins&quot; on the scaffold
</P><P>
    - Which biotechnical processes offer themselves -&gt; e.g. waste water, cow udders
</P><P>
    - How can the enzymes be linked to the scaffold: e.g multiple enzymes with the same dockerin for exchangeability
</P><P>
    - Which Tags can be used
</P><P>
    - Which cloning techniques
</P><P>
    - Will the bacteria survive and work continuously or die after their application
</P><P>
    - How will they be removed from the product
</P><P>
    Ideally we would use a vector system to form a toolkit, which can be modified by the customers. Therefore a modular design and simplicity would be of
    advantage.
</P><P><STRONG>4.) Planning:</STRONG>
    Times expected time by 2 or split group into teams, which work on separate goals: e.g. Scaffold group, lipolytic enzymes group, cellulolytic enzymes group,
    etc.
</P><P>
    Each group should work with multiple enzymes, to guarantee that they create at least one functional tool.
</P><P><STRONG>5.) Labwork:</STRONG></P><P>
    Can start at any time as the Daniel-group will provide us with the resources until we have raised enough money. This can then be paid back at a later
    state. Kits and essential chemicals, etc. are also available through the Lab. -&gt; Start on the Scaffold can begin. Julian, Udhaya and Verena will meet
    Silja on Tuesday to create the theoretical basis/Clone manager plans.
</P><P><STRONG>6.) Meetings</STRONG>
    will have to take place at least weekly as soon as the labwork has started. Next monthd need to be planned and scheduled with everyone's availability and
    working ability, exams, etc.
</P><P><STRONG>7.) Logo</STRONG>
    will have added science things, e.g. helix, magnifying glass. The theme should be used with the university colours (dark and light blue). The booklet can
    have a different title layout to the main part, which can also be designed in a modular, toolbox manner.
</P><P>
    Think of a name for the cellulosome &amp; make it cool: e.g.: superosome, flexosome, legosome
</P><P><STRONG>8.) Sponsors:</STRONG></P><P>
    Possible other sponsors: IVA (for kits), Illumina, Roche
</P></DIV><A href="" onclick=" $('#menu5').slideToggle(300, function callback() {  }); return false;"><H2> 19.01.15</H2></A><DIV id="menu5"><P>
    Present: Silja, Ela, Julian, Verena, Avril, Stefani, Joschi, Sindhu, Nida, Alaa, Dennis, Daniel
</P><P>
    Student Card numbers were collected for easier access to the labs.
</P><P><STRONG>1.) Project:</STRONG></P><P>
    We need to decide whether we want the enzymes to degrade or synthesis and focus on one application to start with.
</P><P>
    Possible enzymes/applications: nitrilases, university waste water/water works/ environmental, contaminated soils, TNT degrading, rubber degrading, Vitamin
    production from intermediates (can be easily proven if it works), bacterial indigo synthesis/other colours
</P><P>
    The easiest is to take a lipase, esterase and/or phosphatase and see if it can degrade different substrates.
</P><P>
    Ela emphasized that the labbooks need to be written with particular care in order to keep the overview and find small mistakes.
</P><P>
    A time plan and team division should be formed in the near future.
</P><P><STRONG>The safety lecture will be given on Monday the 26.1.2015 at 10am. Everyone is required to attend this lecture!</STRONG></P><P><STRONG>2.) Booklet:</STRONG></P><P>
    Projectname was decided: <STRONG>Flexosome</STRONG> (with 10/12 votes), runners up were: Custosome and Supersome
</P><P>
    IGEM logo should be within the text to minimise white space
</P><P>
    A group picture should be added to make the booklet more personal.
</P><P>
    Boxes on page 1 need to be filled/removed/changed
</P><P><STRONG>3.) Sponsoring packages:</STRONG></P><P>
    Bronze package: &gt;500€: Logo will be displayed on the website
</P><P>
    Silver: &gt;2500€: Logo on Website and T-Shirt
</P><P>
    Gold: &gt;5000€: Logo will be displayed everywhere
</P><P><STRONG>4.) Sponsors:</STRONG></P><P>
    Contacts exist to Henkel, Bayer &amp; Illumina. Silja and Ela will introduce us to these and then we can send our booklet/proposal for funding.
</P><P>
    Other possible sponsors are: SI clone manager or VWR
</P></DIV><A href="" onclick=" $('#menu6').slideToggle(300, function callback() {  }); return false;"><H2> 02.02.15 </H2></A><DIV id="menu6"><P><STRONG>Attendance </STRONG></P><P>
    Avi, Udhaya, Nida, Verena and Julian
</P><P><STRONG>1.) Booklet:</STRONG></P><P>
    Avi and Stefani went to the printer center, they test-printed a booklet and gave some options and prices. They showed the booklet to Ela and she also
    talked to Prof. Daniel. He wants the logo and booklet to have more biology related design on it and he wants it to be printed on better paper. His working
    group can pay for it. So Stefani is still working on the booklet. The printing can be done on Din A3 sized paper which will reduce costs (30 € for 80
    booklets!?) For a professional binding we have to give it to a Buchbinder, Avi had contacted the Buchbinder and the guy said the costs will be 20 € for
    half an hour and 40 € if it takes a full hour.
</P><P><STRONG>2.) Sponsoring:</STRONG></P><P>
    Sindhu and Nida divided our company list. Each person has to contact 6 companies. The updated list is in the Dropbox.
</P><P><STRONG>3.) iGEM GWDG account and email address:</STRONG></P><P>
    Verena contacted the GWDG and filled in the form to create an iGEM 2015 user account incl. email-address. As soon as it is done, the GWDG will send a
    letter to Prof. Daniels department.
</P><P><STRONG>4.) Bachelor students from other faculties:</STRONG></P><P>
    Debbie asked a friend who is studying B. Sc. Informatics here in Göttingen and he is interested in participating to help with programming and stuff for the
    website. Since he is focussing on a different area but does not seem to be completely uneducated in websites and webdesign and is 24 years old, we decided
    that if we cannot find another skilled (younger) person, we can try it.
</P><P><STRONG>5.) Group photo</STRONG></P><P>
    A preliminary group photo was taken last week and will be included in the booklet as long as we don’t have another one.
</P><P><STRONG>6.) Scaffoldin</STRONG></P><P>
    Udhaya, Julian and Verena met up with Ela and Silja to further design the Scaffoldin. On the first meeting some interesting organisms for putative
    cohesins/dockerins could be found. Now Ela compared a variety of cohesin structures (sequences) from those organisms by alignment and bioinformatical
    reconstruction of phylogenetic trees. Ela gave all organisms a letter code. For example: “CTHE” for <EM>Clostridium thermocellum.</EM> Due to the fact that
    we want to use (only) 4 different cohesins, we need to cut off scaffoldin proteins that naturally offer more than 4 cohesins in the protein structure. To
    ensure that there will be no unexpected interactions with our desired cohesins (and to have a back-up if the first construct doesn’t work), we finally
    decided to go with a naturally long scaffoldin protein and a shorter one which only harbors 4 cohesin places:
</P><P><U>1) Main construct</U>
    : <EM>Clostridium thermocellum</EM></P><P>
    Scaffolding protein <U>CipA</U> (cohesin type I) with dockerin type II. CipA contains in sum <STRONG>9 cohesins.</STRONG> The first two are somehow
    disconnected by a CBM (cellulose binding motif). A deletion could lead to problems, so we decided to cut off cohesion 1-4 (incl. the CBM). The resulting
    protein offers space for 5 cohesins. The first of these 5 will be the natural cohesin of <EM>C. thermocellum</EM> so that we have 4 free cohesin places for
    our desired cohesins.
</P><P>
    Desired Scaffolding protein structure:
</P><P>
    SP EC – linker 6 CTHE – coh6 (CTHE) – cohX – cohY – cohZ – DOC (CTHE)
</P><P><U>2) Back-Up:</U><EM>Acetivibrio cellulolyticus</EM></P><P>
    Scaffolding protein <U>ScaB</U> (cohesin type II) with dockerin type I. ScaB contains in sum <STRONG>4 cohesins.</STRONG> The first of these 4 will be
    (again) the natural cohesin of <EM>A. cellulolyticus</EM> so that we have 3 free cohesin places for our desired cohesins. It is not clear at the moment
    which dockerin will be used at the end of the structure.
</P><P>
    Desired Scaffolding protein structure:
</P><P>
    SP EC – coh type II – cohX – cohY – cohZ – DOC (ACEL) / DOC (CTHE)
</P><P>
    For further information on the scaffoldin please look in the “Scaffoldin” Dropbox-folder: PowerPoint “scaffoldin_overview”
</P><P><U>3)Websites/tools</U>
    : NCBI (BLAST etc.), InterPro Scan for sequences, alignments, selection and branding of domains in Artemis.
</P><P>
    Calculated 3D structures of CipA (<EM>Clostridium thermocellum</EM>) by LOOPP and Phyre<SUP>2</SUP></P><P>
    What has to be done now is to find the sequences for the fitting dockerin-binding proteins. This will be discussed on the next scaffolding-meeting tomorrow
    (03.2.15, 2 p.m.) with Ela, Silja, Udhaya, Julian and Verena.
</P><P>
    Moreover it is time for us to contact companies doing gene synthesis. Possible companies can be found at “Gene synthesis consortium”
</P><P>
    (<A href="http://www.genesynthesisconsortium.org/members.php">http://www.genesynthesisconsortium.org/members.php</A>)
</P><P>
    A template message how to contact these companies was sent by you per email from Julian last week.
</P></DIV><A href="" onclick=" $('#menu7').slideToggle(300, function callback() {  }); return false;"><H2> 16.02.15 </H2></A><DIV id="menu7"><P>
    Ela and Silja have ordered the organisms from which we will extract the dockerins from the DSMZ. To work with those, we need to start preparing media.
    Udhaya has agreed to find a company until the 7<SUP>th</SUP> which will then produce our scaffoldin constructs.
</P></DIV><A href="" onclick=" $('#menu8').slideToggle(300, function callback() {  }); return false;"><H2>  18.02.15</H2></A><DIV id="menu8"><P><STRONG>1.) Labwork:</STRONG></P><P>
    We have formed our first groups for the two scaffoldins as well as the three enzymes. Debby and Alaa will ask around in the Biochemistry Department for
    concrete ideas for a more complex enzyme systems.
</P><P>
    Coming next week, Silja and Elzbieta will show at least one person from each enzyme group, how to extract the dockerines from the organisms. The groups
    will then each extract a dockerin and prepare the enzymes.
</P><P><STRONG>2.) Sponsors: </STRONG></P><P>
    Sindhu will prepare a letter for Prof. Daniel, which will then also be sent out with each booklet.
</P><P>
    The booklets are currently in print; as soon as Avril hears from the printers she will tell us. We will then start sending them out.
</P><P>
    On our next meeting, we will finalise our working groups as well as set up minimum working hours for the lab.
</P></DIV><A href="" onclick=" $('#menu9').slideToggle(300, function callback() {  }); return false;"><H2> 23.02.15</H2></A><DIV id="menu9"><P><STRONG>1.) Labwork:</STRONG></P><P>
    We agreed on a minimum work hour per month set at 30 h. Each participant can divide their own time as they wish, as long as at least two persons are
    present in the lab as well as an additional PhD / Postdoc on the ground floor in case of accidents.
</P><P>
    If the minimum work hours aren't kept, the person may be kicked from the team. The hours may be transferred to another person, if both parties agree.
</P><P>
    The work groups have been put together.
</P><P>
    Each work group will loan their own equipment. The loaning person is responsible for any damages. Any damages will have to be paid by the person who loaned
    the equipment.
</P><DIV><P>
        Avril has also proposed to create a time table of sorts in which everybody can reserve lab space. If we do work all at the same time in the lab it’ll
        get crowded soon as we only have alloted work space for up to 6 people.
    </P><P>
        First ordered organisms have arrived.
    </P><P>
        Julian and Dennis will sort out Media on Tuesday
    </P><P><STRONG>2.) Booklet:</STRONG></P><P>
        Avril could not reach Herr Nolte from printing last week, and will go there on Tuesday to get it all sorted.
    </P></DIV><P><U>Update:</U>
    Booklets are printed and will be cut and put into letters on Wednesday.
</P><P><STRONG>3.) Sponsors:</STRONG></P><P>
    Sindhu/Nida have updated Sponsors, however everyone needs to check the addresses and whether they are correct. Sindhu has also started writing a general
    letter to send to sponsors.
</P><P><STRONG>4.) Office:</STRONG></P><P>
    Julian: Sorted out an <STRONG>office</STRONG>, which is big and most likely free for the entire time of iGem. It is on the    <STRONG>2nd floor of the Geography</STRONG> building. There are two keys, which a system is needed so everyone who needs the office has access.
</P></DIV><A href="" onclick=" $('#menu10').slideToggle(300, function callback() {  }); return false;"><H2> 16.03.15</H2></A><DIV id="menu10"><P><STRONG>Attendance </STRONG></P><P>
    Alaa, Dennis, Julian, Sindhu, Udhaya, Verena and Ela
</P><P><STRONG>1.) Money - GZMB</STRONG></P><P>
    Julian has managed to write a letter for the GZMB to ask for money and let Prof. Dr. Rolf Daniel sign it. Prof. Daniel will send it this afternoon and we
    should have a response within 2-3 days.
</P><P><STRONG>2.) Schedule – overview</STRONG></P><P><STRONG>Scaffoldins:</STRONG></P><P>
    Finances needed: Maybe Prof. Daniel will pay for one of the constructs, ordering (2-3 weeks), vector cloning (4-6 weeks). Labwork could therefore start in
    4-6 weeks
</P><P><STRONG>Dockerins:</STRONG></P><P>
    Strains have to be ground and DNA isolated, PCR and cloning into pBAD, extraction and purification when fused to enzymes. Lab work can start now.
</P><P><STRONG>Enzymes:</STRONG></P><P>
    Chosen enzymes need to be cloned into pJET (no activity) and then into pBAD with Dockerins. Expression and purification follow afterwards. Labwork can
    start now.
</P><P><STRONG>3.) Working groups</STRONG></P><P>
    Due to started lab work, exams and practical courses we had to rethink our groups for the moment.
</P><P>
    Ela gave us the advice that it would be the best to have (at the moment) 2 groups working on the enzymes (5 enzymes plus GFP), 1 group working on the
    dockerins and 1 group pushing the sponsors. Without money we cannot register and go on with the project!
</P><P><U>Enzyme groups:</U></P><P>
    1) Avi, Stefani, Joschy
</P><P>
    2) Debby, Alaa, Udhaya
</P><P><U>Dockerin group:</U></P><P>
    Julian, Dennis
</P><P><U>Sponsor group:</U></P><P>
    Sindhu, Verena (until the 24<SUP>th</SUP> march, afterwards also free for lab work)
</P><P><STRONG>4.) Lab times</STRONG></P><P>
    We agreed on a minimum work hour per month set at 20 h per person. Due to organisation this month does not count.
</P><P><STRONG>5.) Keys</STRONG></P><P>
    All lab keys and our office key can be found in our own locker. Ask Julian for the numbers of the combination lock!
</P><P><STRONG>6.) Time table</STRONG></P><P>
    We agreed on creating an overall time table that will be uploaded into the Dropbox and updated so that everybody knows exactly what has been done so far in
    the lab and has to be done in the next days!
</P><P><STRONG>7.) Lab equipment lending</STRONG></P><P>
    At least one person of each group has to come <STRONG>tomorrow 17<SUP>th</SUP> March at 10:30 p.m.</STRONG> to lend lab equipment the groups will need the
    next weeks.
</P></DIV><A href="" onclick=" $('#menu11').slideToggle(300, function callback() {  }); return false;"><H2> 23.03.15</H2></A><DIV id="menu11"><P>
    Missing: Sindhu
    <STRONG>1.) Sponsors:</STRONG></P><P>
    We need to wait 2 weeks until we hear from the GZMB about funding. This means we will have to pay the late fee for iGEM. Any sponsor bills for sponsors we
    find until then will be held back until we have a thumbs up from the GZMB funding.
</P><P>
    Avi will put a new file with sponsors into the Dropbox, to see if everyone can edit it. The list will be updated with additional contacts
    
    Debbie has received two no answers and no other replies.
    
    Julian: No news from Sparkasse, another contact will be updated once he can reach Prof. Stülke
    
    Add more local sponsors
    
    Joschy will call his own sponsors, who haven't replied
    
    Will ask Verena to contact VW again, but without calling it sponsoring (differences in taxation)
    <STRONG>2.) Lab updates:</STRONG></P><P><STRONG>D:</STRONG>
    growing bacteria is tricky; two of 4 cultures may be growing but are very slow. They were plated on solid medium in order to see if they will grow there.
    The first media and trace elements didn't work. But have been redone now. The two cultures that have grown so far will also be extracted (tomorrow).
    <STRONG>E1:</STRONG>
    designed primers for esterase and phosphatase in order to fuse it with dockerin and plasmid. The plasmid has a His-tag. Tomorrow they will do the first PCR
    with the primers for the phosphatase. Will ask Gabrielle about Cycler.
    <STRONG>E2:</STRONG>
    nothing so far.
    <STRONG>3.) Website:</STRONG></P><P>
    Alex, who studies economical informatics, has joined our team and will work on the website and help with the Wiki. Julian will contact Cedric about login
    for the website. All company logos shall be added to the Dropbox when they are received.
    
    Joschy has been thinking about the webpage: maybe have a picture in the background instead of a plain one.
    </P><P><STRONG>4.) Other stuff:</STRONG></P><P>
    Joschy will have a look at the structure or a possible picture for our flexosome once he has the sequences. Binding simulation on pymol.
    </P></DIV><A href="" onclick=" $('#menu12').slideToggle(300, function callback() {  }); return false;"><H2> 07.04.15</H2></A><DIV id="menu12"><P><STRONG>1.) Lab:</STRONG></P><P>
    Dockerin1: One plate shows some growth but a PCR has shown nothing so far.
</P><P>
    Liquid cultures had a pellet in 3 cases, from which J. and D. will try to extract the DNA.
</P><P>
    Enzyme 1: have got 2 enzymes ready for insertion (phosphatase and esterase)
</P><P>
    Enzyme2: will start working in the evenings during the practical.
</P><P>
    Avril can show people how to work clone manager to design primers.
</P><P>
    Scaffolding order via Ela or get her to tell someone how to do it.
</P><P>
    People who are not in the plant module should see if they can work alongside each other. Who is taking which course next semester?
</P><P><STRONG>2.) Website</STRONG>
    : Alex should be put in touch with Bing Yao from a previous iGEM Team.
</P><P><STRONG>3.) Sponsors:</STRONG></P><P>
    Sparkasse letter needs to be resent to the correct person.
</P><P>
    New list of sponsors from biotech booklet needs to be contacted.
</P><P><STRONG>4. Boston:</STRONG></P><P>
    Prices for flight and accommodation in Boston will be checked by Monday.
</P></DIV><A href="" onclick=" $('#menu23').slideToggle(300, function callback() {  }); return false;"><H2>13.04.15</H2></A><DIV id="menu23"><P><STRONG>1.) Website:</STRONG></P><P>
    Individual skills need to be worked out to find out who can do which part. Joschy is currently learning how to use HTML. He has made a navigation bar. It
    may be helpful to have some external input.
</P><P>
    The colour scheme will be our iGEM colours (blues)
</P><P>
    The content must be updateable by every group, can be taken from the booklet to start with
</P><P>
    Every team member needs to write a max. 150 word paragraph about themselves and submit a photograph to the dropbox.
</P><P><STRONG>2.) Labwork:</STRONG></P><P>
    Dockerins: DNA has been extracted and we may get 3-4 Dockerins. Dockerins need to be finished in order to have the enzyme groups continue.
</P><P>
    Scaffoldin: Transformation into E.coli worked and the plasmids can now be extracted and verified. First expression tests can follow. Alaa and Sindhu will
    continue with the Scaffoldin from tomorrow.
</P><P>
    E2: Primers will be designed on Thursday
</P><P>
    E1: nothing to report due to a Practical, no work can be done atm.
</P><P>
    Clone Manager: Julian will try to see if we can get access to it somehow.
</P><P><STRONG>3.) Sponsors:</STRONG></P><P>
    GZMB is meeting soon and will decide on funding. We have so far got 4 Sponsors.
</P><P><STRONG>4.) Flights:</STRONG></P><P>
    Alaa has looked up flights if we book now:
</P><P>
    700-870€, Hotel: 45-60€ p.P /pN. We need to book fast as most hotels and hostels will be booked by fellow igemers.
</P><P><STRONG>5.) Costs:</STRONG></P><P>
    Anticipated Labcosts: 18000€
</P><P>
    Flights: 800€ pP
</P><P>
    Hotel: 600€ pP
</P><P>
    We need a minimum of 6000€ from the GZMB if we want to continue with the project.
</P></DIV><A href="" onclick=" $('#menu24').slideToggle(300, function callback() {  }); return false;"><H2>21.05.15 </H2></A><DIV id="menu24"><P>
    About the supervisors needs to be written for our Webpage
</P><P><STRONG>Labwork:</STRONG></P><P>
    Dockerins: Attempt to put them in pBAD has failed so far. Retransformation control worked however. ACEL needs to be reinserted into pJET, BCEL needs to be
    restarted from PCR, CTHE ligation needs to be redone. Change elution steps to gain higher concentrations (30µl instead of 50)
</P></DIV><A href="" onclick=" $('#menu13').slideToggle(300, function callback() {  }); return false;"><H2> 19.06.15</H2></A><DIV id="menu13"><P><STRONG>1.) Labwork:</STRONG></P><P>
    Heiko will have an iGEM meeting slot everyday from 9-9.30am in his office.
</P><P>
    E1:
</P><P>
    -skipping BL21 step, because Esterase showed activity in TOP10 activity plates.
</P><P>
    - Another option is to add a T7 Plasmid to induce the gene
</P><P>
    - Phosphatase is expressing nicely (blue)
</P><P>
    Phosphatase will be sent for sequencing next week.
</P><P>
    E3:
</P><P>
    Ready for expression plates with Cellulase in BL21
</P><P>
    Scaffoldin:
</P><P>
    Has not worked yet, probably due to the unspecific binding of the primers (good binding conditions are hard to find). Using a ligase free ligation in
    pet101 which is sensitive to temperature and ions
</P><P>
    -&gt;do a microdialysis to clean ligation from salts and ions before doing transformation.
</P><P>
    The second scaffoldin was ordered and should arrive in the next couple of weeks.
</P><P>
    RFP:
</P><P>
    Cells are not pink and don’t fluoresce under the fluorescence microscope. Sequences are fine but will be done again in order to make sure.
</P></DIV><A href="" onclick=" $('#menu14').slideToggle(300, function callback() {  }); return false;"><H2> 25.06.15</H2></A><DIV id="menu14"><P>
    Present: Heiko, Ela, Julian, Dennis, Steffi, Avi, Verena, Alaa, Debbie, Nida
</P><P><STRONG>1.) Strain documentation: </STRONG></P><P>
    The team has now got a box in the -80 ̊C freezer to start a strain collection. The Box is on the ground floor, in the middle freezer, top compartment, top
    shelf marked: Rack 1 Robert at the back, 2<SUP>nd</SUP> from the bottom (blue box)
</P><P>
    - To register the strain an information sheet needs to be filled in and submitted to Jaqueline
</P><P>
    - Strains should be frozen from overnight culture in 30% glycerine (e.g. 600ul liquid culture + 900ul Glycerine (from 50% stock solution))
</P><P><STRONG>2.) Labwork:</STRONG></P><P>
    Cellulase:
</P><P>
    pJET_cellulase transformation and restriction worked fine in TOP10 and was then transformed into BL21 cells. Restriction showed a fragment of the correct
    size, however sequencing failed. Activity plates showed no activity so far, but will be incubated over a longer period of time and in addition singularised
    to avoid contamination of other colonies.
</P><P>
    Scaffoldin:
</P><P>
    Trafo in Top10 and BL21 did not work, which is why the TOPO cloning may be problematic.
</P><P>
    - A control with an empty vector should be run.
</P><P>
    - If the annealing is successful the bands should change in a reaction, which can be verified to exclude those issues.
</P><P>
    - In addition Primers should be designed from within the sequence to verify the ends of the scaffoldin. Primers should be around 18bp long and 200bp within
    the construct, which also has TOPO overlaps
</P><P>
    - In addition the scaffoldin band of the PCR could be correct, but with an impurity alongside -&gt; transformation should however still work to some extent
    (second problem?)
</P><P>
    - For the covalent binding stick to the protocol an do not reduce salt concentrations
</P><P>
    - For electroporation the TOPO cloning mixture needs to be desalinated with microdialysis, otherwise the trafo won't work
</P><P>
    Recommendation: Check salts, positive control, annealing conditions and sequencing after PCR purification.
</P><P>
    RFP:
</P><P>
    Sequences are 100%match, but colonies show no fluorescence. Therefore, restricted RFP sequences will be ligated into pBAD in order to verify its function
    in the target plasmid.
</P><P>
    GFP sequences for Primer design and corresponding plasmids/colonies can be acquired from Jaqueline.
</P><P>
    E1&amp;2:
</P><P>
    Sequencing and restriction control were ok. Strains are frozen and documentation is being prepared.
</P><P>
    Ask Silja for origin of the enzymes for better documentation. One is from Heiko Nacke, the other from Genis. In addition the enzyme sequences can be
    Blasted for identification.
</P><P>
    Phosphatase contains PstI, which is why currently it is not compatible with biobricks. In the future the restriction sites will have to be changed, if
    submission should occur (PCR modification).
</P><P>
    Dockerins:
</P><P>
    Heat shock transformation did not work for any of the 4 dockerins, but positive control worked fine.
</P><P>
    Vector was cut with EcoRI and KpnI as a test and KpnI was not very efficient. Due to this the restrictions were done separately.
</P><P>
    Next step is a ligation and a gel to verify it immediately after to find reasons for failure:
</P><P>
    - T4 ligase working?
</P><P>
    - Phosphorylation issues -&gt; skip this step
</P><P>
    - Find Plasmid formula to calculate ideal ligation ratio (depends on insert length and is usually 3:1) -&gt; overloading the vector may cause different
    ligation variants
</P><P>
    - Can E.coli express the dockerins?
</P><P>
    Oder dockerins as codon optimised gBlocks from IDT, as a backup.
</P></DIV><A href="" onclick=" $('#menu17').slideToggle(300, function callback() {  }); return false;"><H2>24.07.15</H2></A><DIV id="menu17"><P>
    Present: Debbie, Avi, Julian, DEnnis
</P><P>
    Excused: Verena, Steffi, Nida, Sindhu
</P><P><STRONG>1.) Labwork:</STRONG></P><P>
    Enzyme 3: Transformations didn't work, so the project will be restarted from PCR stage and a new ligation.
</P><P>
    The sequencing is also not working, possibly due to wrong primers or wrong primer concentrations.
</P><P>
    Dockerin/Scaffoldin: both scaffoldins are at the expression stage
</P><P>
    Dockerins in gBlocks have arrived.
</P><P>
    Old Dockerins may not be restrictable due to missing bands around the restriction sites.
</P><P>
    New Plasmid isolation is on the way and work will continue on the dockerins as before.
</P><P><STRONG>2.) 3D Model:</STRONG></P><P>
    Debbie has checked the sequences and has the Amino Acid sequences for all enzymes, but cannot find the scaffoldin sequences. When used they result in short
    fragments. Dockerins do not work either due to too many stopcodons.
</P><P>
    Double check the files and give her new files.
</P><P><STRONG>3.) Human Practices:</STRONG></P><P>
    Julian will contact people at the Xlab.
</P><P>
    A stop motion movie is being made that simply explains our project
</P><P>
    A talk needs to be organised around our project and synthetic biology -&gt; Beer and Pretzels? Julian will contact some people and organise it.
</P><P>
    Once returned from iGEM, the presenters will also present their project to the faculty at a faculty talk on the 27.10.2015
</P><P><STRONG>4.) Meetup:</STRONG></P><P>
    The group that will travel to Marburg for the German Meetup will prepare short presentation
</P><P><STRONG>5.) Poster:</STRONG></P><P>
    Dennis is happy to help with the organisation of the poster
</P></DIV><A href="" onclick=" $('#menu18').slideToggle(300, function callback() {  }); return false;"><H2>18.08.15</H2></A><DIV id="menu18"><P><STRONG>1.) Labwork:</STRONG></P><P>
    Ligation and transformation of enzymes and dockerins with extra bases worked all around there are positive restriction results (but for 2). Plasmids have
    been sent for sequencing.
</P><P>
    - enzymes are being induced in fat medium for next day protein purification. Induction must be done with L-arabinose due to the pBAD vectore.
</P><P>
    Scaffoldin: ACEL purification yielded 1mg/ml of Protein, but it still needs to be verified on an SDS gel.
</P><P>
    Cellulase_CCEL construct is being sequenced.
</P><P><STRONG>2.) BioBricks:</STRONG></P><P>
    Avril will design Primers with the proper restriction sites for the BioBrick shipping vector. EcoRI and PstI will be used.
</P></DIV><A href="" onclick=" $('#menu20').slideToggle(300, function callback() {  }); return false;"><H2>24.08.15</H2></A><DIV id="menu20"><P><STRONG>1.) Labwork:</STRONG></P><P>
    Each section of labwork needs to be reassigned in order to optimize our work
</P><P>
    Dennis &amp; Julian will work on Gelfiltrations and Protein extractions. They may need some help with all the inductions.
</P><P>
    Alaa will make sure all protocols are written for the Wiki in time for the Wiki Freeze. Every group needs to write up their protocols and results.
</P><P>
    Avril will make Primers for BioBricks of Dockerins with and without Enzymes and ask the Aachen Team for some Enzymes (design Primers for these as well)
</P><P>
    Stefi will work on Poster and Presentation
</P><P>
    Enzyme Dockerin constructs and BioBricks need to be continued and brought to work. See if we can find an enzyme cascade that we could attach to the
    Flexosome.
</P><P><STRONG>2.) Finances:</STRONG></P><P>
    An overview over our current finances has to be made to know what we have for Tshirts and labcosts.
</P><P><STRONG>3.) Safety Form:</STRONG></P><P>
    The safety form deadline is this week. Avril will work on it and double check it with Heiko.
</P></DIV><A href="" onclick=" $('#menu21').slideToggle(300, function callback() {  }); return false;"><H2>27.08.15</H2></A><DIV id="menu21"><P><STRONG>1.) Labwork:</STRONG></P><P>
    Enzymes: Phosphatase activity test in different Buffer is working
</P><P>
    Cellulase: Primers have not come yet
</P><P>
    Induction: Pho_CTHE and rfp_ACEL are in fat medium and the spare amount will be used with the FPLC purification.
</P><P>
    Primers have been designed for most constructs
</P><P>
    Verena, Udhaya and Sindhu will continue the work on Dockerins, Est_BCEl and cjblue for BioBrick creation.
</P><P><STRONG>2.) Wiki:</STRONG></P><P>
    Alaa and Udhaya have been working on the Method Collection for the Wiki
</P><P>
    Ask Alex to make it pretty and insert the Aix-Marseille Gold Medal for collaboration
</P><P><STRONG>3.) ETC:</STRONG></P><P>
    Steffi is working on Poster and Presentation. She has designed Tshirts with Verena and both have looked for enzyme pathways within the biobricks.
</P></DIV><A href="" onclick=" $('#menu22').slideToggle(300, function callback() {  }); return false;"><H2>03.09.15</H2></A><DIV id="menu22"><P><STRONG>1.) Labwork:</STRONG></P><P>
    Protein purification of CTHE has been successful. Other constructs are not working. Not sure whether due to pBAD vector, therefore sequences need to be
    checked.
</P><P>
    Aachen Enzymes have arrived and the first BioBricks made (still need to be sequenced). BioBricks with colours amilCP, cjBlue and efforRed are being used as
    backup.
</P><P>
    Purification: Could the enzymes be deactivated by the purification?
</P><P>
    Can we check scaffoldin interactions, even though enzymes may not function? So far it hasnt been possible to purify any of the enzymes or RFP and not to
    show a 100% positive activity test.
</P><P>
    - Restart expression tests
</P><P>
    - start cellulase induction
</P><P>
    - check RFP again with a control
</P><P>
    - could it be possible to use the scaffoldin to purify the enzyme-dockerins?
</P><P>
    - check for pBAD promoter error and ask Robert for other expression vectors.
</P><P>
    - try and get some sort of interaction even if it is crude
</P><P>
    - is there a BioBrick expression vector?
</P><P><STRONG>2.) Presentation:</STRONG></P><P>
    First presentation meeting today
</P></DIV><A href="" onclick=" $('#menu25').slideToggle(300, function callback() {  }); return false;"><H2>10.09.15</H2></A><DIV id="menu25"><P><STRONG>1.) BioBricks:</STRONG></P><P>
    Est-BCEL plasmid needs to be re-inoculated to get more plasmid. ACEL, BCEL and CCEL plasmids in the BioBrick vector look good and just need sequencing.
</P><P><STRONG>2.) Aachen:</STRONG></P><P>
    HPS and PHI have been succesfully inserted into pBAD with sequencing being fine. Primers for expression in pET100 have arrived.
</P><P><STRONG>3.) Colours:</STRONG></P><P>
    amilCP and eforRed have been sequenced but only amilCP is good, cjBlue is still in the BB vectore´
</P><P><STRONG>4.) Cellulase:</STRONG></P><P>
    Restriction control in pBAD is good. Insertion into pET100 can be tried.
</P><P><STRONG>5.) pET100:</STRONG></P><P>
    Esterase and Phosphatase have been inserted into pET100 with the dockerins. Need to wait if ligation and transformation has worked or if it needs to be
    redone.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>