<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OLS_Canmore_AB_CA_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OLS Canmore AB CA/Composite Part</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA">HOME</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:OLS_Canmore_AB_CA/Faith">FAITH</A></LI></DIV><DIV id="contentContainer"><H2><CENTER><B>Composite Parts</B></CENTER></H2><TABLE><TBODY><TR><TD><H3>Part Name: Keratinase A Expression Cassette (BBa_K1717000)</H3><P><EM>Creator:</EM> Our Lady of the Snows Catholic Academy 2015</P></TD></TR><TR><TD><P><EM>Short Description:</EM> Keratinase A under control of LacI promoter./p&gt;</P></TD></TR><TR><TD><P><EM>Long Description:</EM> Keratinases are proteolytic enzymes which break the disulphide bonds in various keratin substrates.  KER A is a basic part parts comprised from a synthesized KERA protein coding sequence, optimized for expression in <I>E.coli</I> and most active in degrading keratin in hair.</P><P>This gene, when expressed in cells, can be tested for activity through looking for zones of clearing on skim milk agar plates where cells are growing, as well as by looking at degradation of keratin-containing substrates (hair, feathers, chemically synthesized keratin, etc.) when grown with cells.</P></TD></TR><TR><TD><P><EM>Design Considerations:</EM> Keratinases are isolated from several bacterial and fungal species, but have been expressed in E. coli with more limited success.  The bacterial chassis that we are using is a strand of E-coli called K-12. Like all E. coli, this strain is gram-negative. After extensive research and looking at the projects of Chicago, Sheffield and Taiga, we deduced that the signal peptides (which were those of a gram-positive source organism) could be causing the poor secretion of the enzyme in those projects.  Previous teams either left the gram-positive signal peptides intact, or removed signal peptide sequences entirely.  Our approach was instead to remove the original signal peptide sequence and replace it with one optimized for secretion in E. coli (pelB, with the sequence taken from part Part BBa_J32015) The part is designed to express the keratinase when the promoter is induced, and secrete it into the periplasm, where some will escape into the media extracellularily. 
DNA sequences were compared used Clustal to ensure protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  One EcoRI site and one PstI site were identified, and a silent mutation was introduced in order to remove them.</P><P>DNA sequences were compared using Clustal to ensure that the protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  One EcoRI site and one PstI site were identified, and a silent mutation was introduced in order to remove them.</P></TD></TR><TR><TD><P><EM>Source:</EM> The sequence for the KERA coding region was taken from the <A href="http://www.uniprot.org/uniprot/Q53521">Uniprot gene/protein sequence bank</A></P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><H3>Part Name: Keratinase US Expression Cassette (BBa_K1717173)</H3><P><EM>Creator:</EM> Our Lady of the Snows Catholic Academy 2015</P></TD></TR><TR><TD><P><EM>Short Description:</EM> Coding sequence for Keratinase US (KERUS)</P></TD></TR><TR><TD><P><EM>Long Description:</EM> Keratinases are proteolytic enzymes which break the disulphide bonds in various keratin substrates.  KER US is a basic part parts comprised from a synthesized KERUS protein coding sequence, optimized for expression in <I>E.coli</I> and most active in degrading keratin in feathers.</P><P>This gene, when expressed in cells, can be tested for activity through looking for zones of clearing on skim milk agar plates where cells are growing, as well as by looking at degradation of keratin-containing substrates (feathers, hair, chemically synthesized keratin, etc.) when grown with cells.</P></TD></TR><TR><TD><P><EM>Design Considerations:</EM> Keratinases are isolated from several bacterial and fungal species, but have been expressed in <I>E. coli</I> with more limited success.  The bacterial chassis that we are using is a strand of <I>E-coli</I> called K-12. Like all <I>E. coli</I>, this strain is gram-negative. After extensive research and looking at the projects of Chicago, Sheffield and Taiga, we deduced that the signal peptides (which were those of a gram-positive source organism) could be causing the poor secretion of the enzyme in those projects.  Previous teams either left the gram-positive signal peptides intact, or removed signal peptide sequences entirely.  Our approach was instead to remove the original signal peptide sequence and replace it with one optimized for secretion in <I>E. coli</I> (pelB, with the sequence taken from part Part BBa_J32015) When paired with a promoter, the part is designed to express the keratinase and secrete it into the periplasm, where some will escape into the media extracellularily. 
DNA sequences were compared used Clustal to ensure protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  No such sites were found.</P><P>DNA sequences were compared using Clustal to ensure that the protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  One EcoRI site and one PstI site were identified, and a silent mutation was introduced in order to remove them.</P></TD></TR><TR><TD><P><EM>Source:</EM> The sequence for the KERUS coding region was taken from the <A href="http://www.uniprot.org/uniprot/A0A059PWB6">Uniprot gene/protein sequence bank</A></P></TD></TR></TBODY></TABLE></DIV></DIV><P font-color:=""><CENTER>Contact us at <STRONG>alinaarvisais@isidore.redeemer.ab.ca</STRONG> or <STRONG>taliadixon@isidore.redeemer.ab.ca</STRONG>, or even send a message to our facebook page: <A href="https://www.facebook.com/OLeSsence">https://www.facebook.com/OLeSsence</A></CENTER></P></DIV></DIV></DIV></DIV></BODY></HTML>