Team:OLS Canmore AB CA/Basic Part
HOME
TEAM
PROJECT
Description
Experiments & Protocols
Results
Design
PARTS
Team Parts
Basic Parts
Composite Parts
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
SAFETY
FAITH
Basic Parts
Part Name: Keratinase A (BBa_K1717171)
Creator: Our Lady of the Snows Catholic Academy 2015
Short Description: Coding sequence for Keratinase A (KERA)
Long Description: Keratinases are proteolytic enzymes which break the disulphide bonds in various keratin substrates.  KER A is a basic part parts comprised from a synthesized KERA protein coding sequence, optimized for expression in E.coli and most active in degrading keratin in hair.
This gene, when expressed in cells, can be tested for activity through looking for zones of clearing on skim milk agar plates where cells are growing, as well as by looking at degradation of keratin-containing substrates (hair, feathers, chemically synthesized keratin, etc.) when grown with cells.
Design Considerations: Keratinases are isolated form several bacterial and fungal species, but have been expressed in E.coli with more limited success.  The bacterial chassis that we are using is a strand of E. coli called K12.  Like all E.coli, this strain is gram-negative.  After extensive research and looking at the projects of Chicago, Sheffield, and Taiga, we deduced that the signal peptides (which were those of a gram-positive source organism) could be causing the poor secretion of the enzyme in those projects.  Previous teams either left the gram-positive signal peptides intact, or removed the signal peptide sequences entirely.  Our approach was instead to remove the original signal peptide and replace it with one optimized for secretion in E.coli (pelB, with sequence taken from part BBa_J32015).  When paired with a promoter, the part is designed to express the Keratinase and secrete it into the periplasm, where some will escape into the media extracellularily.
DNA sequences were compared using Clustal to ensure that the protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  One EcoRI site and one PstI site were identified, and a silent mutation was introduced in order to remove them.
Source: The sequence for the KERA coding region was taken from the Uniprot gene/protein sequence bank
Part Name: Keratinase US (BBa_K1717172)
Creator: Our Lady of the Snows Catholic Academy 2015
Short Description: Coding sequence for Keratinase US (KERUS)
Long Description: Keratinases are proteolytic enzymes which break the disulphide bonds in various keratin substrates.  KER US is a basic part parts comprised from a synthesized KERUS protein coding sequence, optimized for expression in E.coli and most active in degrading keratin in feathers.
This gene, when expressed in cells, can be tested for activity through looking for zones of clearing on skim milk agar plates where cells are growing, as well as by looking at degradation of keratin-containing substrates (feathers, hair, chemically synthesized keratin, etc.) when grown with cells.
Design Considerations: Keratinases are isolated from several bacterial and fungal species, but have been expressed in E. coli with more limited success.  The bacterial chassis that we are using is a strand of E-coli called K-12. Like all E. coli, this strain is gram-negative. After extensive research and looking at the projects of Chicago, Sheffield and Taiga, we deduced that the signal peptides (which were those of a gram-positive source organism) could be causing the poor secretion of the enzyme in those projects.  Previous teams either left the gram-positive signal peptides intact, or removed signal peptide sequences entirely.  Our approach was instead to remove the original signal peptide sequence and replace it with one optimized for secretion in E. coli (pelB, with the sequence taken from part Part BBa_J32015) When paired with a promoter, the part is designed to express the keratinase and secrete it into the periplasm, where some will escape into the media extracellularily.  DNA sequences were compared used Clustal to ensure protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  No such sites were found.
DNA sequences were compared using Clustal to ensure that the protein coding sequences were not altered, and that start and stop codons were correct.  NEBCutter V2.0 was utilized to check for illegal cut sites.  One EcoRI site and one PstI site were identified, and a silent mutation was introduced in order to remove them.
Source: The sequence for the KERUS coding region was taken from the Uniprot gene/protein sequence bank
Contact us at alinaarvisais@isidore.redeemer.ab.ca or taliadixon@isidore.redeemer.ab.ca, or even send a message to our facebook page: https://www.facebook.com/OLeSsence
