<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_Notebook_Lab_Work_September"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/Notebook/Lab_Work/September">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/Notebook/Lab_Work/September&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Lab_Work/September&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Lab_Work/September&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/Notebook/Lab_Work/September" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/Notebook/Lab Work/September</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV style="background-color:#000000; color:white"><P><CENTER><FONT color="white">Feel free to look around our notebook!
</FONT></CENTER></P><P><CENTER><FONT color="white">Here you can check out the work we have done in the lab, click on a month to take a look!
</FONT></CENTER></P><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#September"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">September</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#September_2.2C_2010"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">September 2, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#ADS"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#September_14.2C_2010"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">September 14, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#JF"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">JF</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#JV"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">JV</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#KG"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">KG</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#AS"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">AS</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#September_15.2C_2010"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">September 15, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#JV.2C_ADS"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">JV, ADS</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#September_16.2C_2010"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">September 16, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#JV_2"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">JV</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#September_17.2C_2010"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">September 17, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#JV_3"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">JV</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#AV"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">AV</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#KG_2"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">KG</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#September_18.2C_2010"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">September 18, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#JS.2C_DM"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">JS, DM</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#TF"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">TF</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#September_20.2C_2010"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">September 20, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#ADS_2"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#HB"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">HB</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#September_21.2C_2010"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">September 21, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#ADS_3"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#TF.2C_MC"><SPAN class="tocnumber">1.8.2</SPAN><SPAN class="toctext">TF, MC</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#AV_2"><SPAN class="tocnumber">1.8.3</SPAN><SPAN class="toctext">AV</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#September_21.2C_2010_2"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">September 21, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#ADS_4"><SPAN class="tocnumber">1.9.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#September_23.2C_2010"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">September 23, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#JV_4"><SPAN class="tocnumber">1.10.1</SPAN><SPAN class="toctext">JV</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#ADS_5"><SPAN class="tocnumber">1.10.2</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#TF.2C_MC_2"><SPAN class="tocnumber">1.10.3</SPAN><SPAN class="toctext">TF, MC</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#September_24.2C_2010"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">September 24, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#ADS_6"><SPAN class="tocnumber">1.11.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#September_25.2C_2010"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">September 25, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#JV_5"><SPAN class="tocnumber">1.12.1</SPAN><SPAN class="toctext">JV</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#ADS_7"><SPAN class="tocnumber">1.12.2</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#September_26.2C_2010"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">September 26, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#ADS_8"><SPAN class="tocnumber">1.13.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#September_29.2C_2010"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">September 29, 2010</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#ADS_9"><SPAN class="tocnumber">1.14.1</SPAN><SPAN class="toctext">ADS</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#AV_3"><SPAN class="tocnumber">1.14.2</SPAN><SPAN class="toctext">AV</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="September"><FONT color="white">September</FONT></SPAN></H1><H2><SPAN class="mw-headline" id="September_2.2C_2010"><FONT color="white">September 2, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Assemble lumazine-dT into pSB1C3 using NEB BioBrick Assembly Kit.<B>Method:</B> Have the following parts: dT [28 ng/(µL)]; lumazine [78 ng/(µL)]; psB1C3 [14 ng/(µL)].</P><UL><LI>Require 500 ng DNA each part in a 50(µL) rxn therefore 17.9(µL) dT and 6.4(µL) Lumazine.  Need 250 ng pSB1C3 or 17.9(µL)
</LI></UL><P>Restriction Reactions - 
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD>Volume(µL)</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>36.1</TD></TR><TR><TD>NEBuffer 2 (10x)</TD><TD>5</TD></TR><TR><TD>Upstream Part (Lumazine)</TD><TD>26.4</TD></TR><TR><TD>EcoRI-HF</TD><TD>1</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>100X BSA</TD><TD>0.5</TD></TR></TBODY></TABLE><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD>Volume(µL)</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>24.6</TD></TR><TR><TD>NEBuffer 2 (10x)</TD><TD>5</TD></TR><TR><TD>Downstream Part (dT)</TD><TD>17.9</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>100X BSA</TD><TD>0.5</TD></TR></TBODY></TABLE><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD>Volume(µL)</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>3.35</TD></TR><TR><TD>NEBuffer 2 (10x)</TD><TD>2.5</TD></TR><TR><TD>Destination Plasmid (pSB1C3)</TD><TD>17.9</TD></TR><TR><TD>EcoRI-HF</TD><TD>.5</TD></TR><TR><TD>PstI</TD><TD>.5</TD></TR><TR><TD>100X BSA</TD><TD>0.25</TD></TR></TBODY></TABLE><UL><LI> 2(µL) was removed prior to adding enzymes for analysis on gel.  
</LI><LI> Split digestion reactions in half: dT - 2 @ 24(µL); Lum - 2 @ 24(µL); psB1C3 - 2 @ 11.5(µL)
</LI><LI> Ran 10 minute and 60 minute reactions at 37<SUP>o</SUP>C.  Followed by 20 minute heat shock at 80<SUP>o</SUP>C. 
</LI><LI> 2(µL) was removed from the heat killed reactions for analysis on gel.
</LI></UL><UL><LI> Performed 4 combinations of ligation: 10 min Restriction + 10 min Ligation; 10 min Restriction + Overnight Ligation; 60 min Restriction + 10 min Ligation; 60 min Restriction + Overnight Ligation
</LI></UL><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD><B>1X(µL)</B></TD><TD><B>2X Master Mix(x2.5)(µL)</B></TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>11</TD><TD>77.5
</TD></TR><TR><TD>T4 Ligase Buffer (10x)</TD><TD>2</TD><TD>5
</TD></TR><TR><TD>Upstream Part</TD><TD>2</TD><TD>5
</TD></TR><TR><TD>Downstream Part </TD><TD>2</TD><TD>5
</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD><TD>2.5
</TD></TR><TR><TD>Plasmid Part</TD><TD>2</TD><TD>5
</TD></TR></TBODY></TABLE><P>Incubated 10 minute and overnight ligations at room temperature ( 25<SUP>o</SUP>C).  Heat killed ligase at 80<SUP>o</SUP>C for 20 min. 
</P><UL><LI> 2(µL) of each reaction was taken for analysis on gel. 
</LI></UL><UL><LI> Confirmed ligation with PCR analysis.  Analyzed restriction, ligation and PCR on a 1.5% TAE agarose gel.  
</LI></UL><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD><TD><B>Master Mix(x5.5)(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>36.5</TD><TD>202.4
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>5</TD><TD>27.5
</TD></TR><TR><TD>dNTPs</TD><TD>2</TD><TD>11
</TD></TR><TR><TD>Forward VF2 Primer</TD><TD>2</TD><TD>11
</TD></TR><TR><TD>Reverse VR Primer</TD><TD>2</TD><TD>11
</TD></TR><TR><TD>Template DNA</TD><TD>2</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2</TD><TD>1.1
</TD></TR></TBODY></TABLE><P>Added 48(µL) to each reaction.  </P><P><B>Objective:</B> Characterize xylE in LB Media vs. M9 Minimal Media.<B>Method:</B>  Three experiments with spectra. </P><UL><LI> Experiment 1 - Measure absorption spectra of 1:10 dilution of:  1.  Cells suspended in m9 media +/- catechol.  2.  Cells suspended in LB media +/- catechol.  3.  m9 media from spun down cells +/- catechol.  4.  LB media from spun down cells +/- catechol.  5.  Sterile LB media +/- catechol.  6.  Water +/- catechol.  
</LI></UL><UL><LI> Experiment 2 - Catechol Breakdown.  Follow absorbance of 2-hydroxymucanate semialdehyde at 375 nm.  Measured all the above samples for 10 min at that wavelength.
</LI></UL><H2><SPAN class="mw-headline" id="September_14.2C_2010"><FONT color="white">September 14, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JF"><FONT color="white">JF</FONT></SPAN></H3><P><B>Objective:</B> Miniprep of RFP expression construct (J04450) via Qiaquick/Qiaprep spin column.  Protocol followed as per kit instructions.</P><H3><SPAN class="mw-headline" id="JV"><FONT color="white">JV</FONT></SPAN></H3><P><B>Objective:</B> PCR the pSB1C3 plasmid from part J04450 miniprep.</P><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>12.8
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>2
</TD></TR><TR><TD>dNTPs</TD><TD>1
</TD></TR><TR><TD>SB-prep-26 Primer</TD><TD>1
</TD></TR><TR><TD>SB-prep-3P Primer</TD><TD>1
</TD></TR><TR><TD>Template DNA</TD><TD>2
</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2
</TD></TR></TBODY></TABLE><P>Used iGEM cycle PSB1C3 in thermocycler.
Ran PCR product on 1% TAE agarose gel.  
</P><H3><SPAN class="mw-headline" id="KG"><FONT color="white">KG</FONT></SPAN></H3><P><B>Objective:</B> PCR of xylE (mRBS-xylE K118021) with fusion standard so that an arginine tail can be attached.</P><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD><TD><B>Master Mix(x6)(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>12.8</TD><TD>76.8
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>2</TD><TD>12
</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>xylE Fusion Prefix Forward Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>xylE Fusion Suffix Reverse Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Template DNA</TD><TD>2</TD><TD>12
</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2</TD><TD>1.2
</TD></TR></TBODY></TABLE><P>PCR- Conditions:
1.  95<SUP>o</SUP>C for 180 sec
2.  95<SUP>o</SUP>C for 30 sec
3.  56.6 +/- 5<SUP>o</SUP>C for 30 sec
4.  72<SUP>o</SUP>C for 150 sec
5.  72<SUP>o</SUP>C for 600 sec
(35 cycles)
</P><P>Analyzed PCR on 2% agarose gel which ran at 100 V for 90 minutes.</P><H3><SPAN class="mw-headline" id="AS"><FONT color="white">AS</FONT></SPAN></H3><P><B>Objective:</B> Assemble xylE-dT using 3AB assembly and 3AB assembly.</P><P>Restriction Reactions - 
</P><P>3AB Upstream Part (xylE)
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD>Volume(µL)</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>30.6</TD></TR><TR><TD>NEBuffer 2 (10x)</TD><TD>5</TD></TR><TR><TD>Plasmid DNA </TD><TD>11.9</TD></TR><TR><TD>EcoRI-HF</TD><TD>1</TD></TR><TR><TD>SpeI</TD><TD>1</TD></TR><TR><TD>100X BSA</TD><TD>0.5</TD></TR></TBODY></TABLE><P>3AB Downstream Part (dT)
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD>Volume(µL)</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>24.6</TD></TR><TR><TD>NEBuffer 2 (10x)</TD><TD>5</TD></TR><TR><TD>Plasmid DNA</TD><TD>17.9</TD></TR><TR><TD>XbaI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>100X BSA</TD><TD>0.5</TD></TR></TBODY></TABLE><P>3AB Plasmid (pSB1C3)
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD>Volume(µL)</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD></TR><TR><TD>NEBuffer 2 (10x)</TD><TD>5</TD></TR><TR><TD>Plasmid DNA</TD><TD>42.5</TD></TR><TR><TD>EcoRI-HF</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>100X BSA</TD><TD>0.5</TD></TR></TBODY></TABLE><UL><LI>Ran 10 minute and 60 minute reactions at 37<SUP>o</SUP>C.  Followed by 20 minute heat shock at 80<SUP>o</SUP>C. 
</LI></UL><P>Ligation Reaction -
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Ingredient</B></TD><TD><B>1X(µL)</B></TD></TR><TR><TD>MilliQ H<SUB>2</SUB>0 Water</TD><TD>11
</TD></TR><TR><TD>T4 Ligase Buffer (10x)</TD><TD>2
</TD></TR><TR><TD>Upstream Part</TD><TD>2
</TD></TR><TR><TD>Downstream Part </TD><TD>2
</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1
</TD></TR><TR><TD>Plasmid Part</TD><TD>2
</TD></TR></TBODY></TABLE><P>Incubated 10 minute and overnight ligations at room temperature ( 25<SUP>o</SUP>C).  Heat killed ligase at 80<SUP>o</SUP>C for 20 min. 
</P><P>Transformation - 
</P><P>A) Thawed 200(µL) Library Efficiency DH5alpha Competent Cells on ice.
B)  Gently mixed cells and then aliquoted 100(µL) into chilled polypropylene tubes.
C)  Added 1(µL) of ligation mix to cells. Added 5(µL) of pUC19 DNA to 100(µL) cells to determine efficiency.  
D)  Incubated cells on ice for 30 minutes.
E)  Heat shocked cells for 45 seconds in a 42<SUP>o</SUP>C water bath.
F)  Placed on ice for 2 minutes.
G)  Added 0.9 mL of room temperature SOC medium.
H)  Shook at 225 rpm for 1 hour.
I)    Diluted control cells 1:100 with SOC medium.
J)   Spread 100(µL) of this dilution on LB-Amp agar plates
K) Spread 50 and 250(µL) of experimental cells on LB-Cam agar plates.  
L)  Incubated overnight at 37<SUP>o</SUP>C
</P><H2><SPAN class="mw-headline" id="September_15.2C_2010"><FONT color="white">September 15, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JV.2C_ADS"><FONT color="white"> JV, ADS</FONT></SPAN></H3><P><B>Objective:</B> Determine if dT ligated onto p-rbs-xylE using PCR analysis.</P><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD><TD><B>Master Mix(x4.5)(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>11.8</TD><TD>53.1
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>2</TD><TD>9
</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>4.5
</TD></TR><TR><TD>Forward VF2 Primer</TD><TD>1</TD><TD>4.5
</TD></TR><TR><TD>Reverse VR Primer</TD><TD>1</TD><TD>4.5
</TD></TR><TR><TD>Template DNA</TD><TD>1</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2</TD></TR></TBODY></TABLE><P>Used iGEM program 7 in thermocycler.
</P><P><B>Objective:</B> Determine if minipreps from Sept 15, 2010 worked.  Analyze assembly intermediates.  <B>Method:</B> Run 1% TAE agarose gels at 100 V for 90 minutes. </P><H2><SPAN class="mw-headline" id="September_16.2C_2010"><FONT color="white">September 16, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JV_2"><FONT color="white"> JV</FONT></SPAN></H3><P><B>Objective:</B> Isolate RFP in pSB1C3 from DH5alpha cells.<B>Method:</B> Used Qiagen minipep protocol and spin column.<B>Results:</B> Gel of miniprepped RFP against previously prepared RFP showed a band of the right size.</P><H2><SPAN class="mw-headline" id="September_17.2C_2010"><FONT color="white">September 17, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JV_3"><FONT color="white"> JV</FONT></SPAN></H3><P><B>Objective:</B> Isolate plasmid DNA from DH5alpha cells:  K249024; B0015; K118021/B0015<B>Method:</B> Used Qiagen minipep protocol and spin column. Ran a 1% agarose gel to view DNA</P><P>GEL PICTURE!
</P><H3><SPAN class="mw-headline" id="AV"><FONT color="white">AV</FONT></SPAN></H3><P><B>Objective:</B> PCR out RFP from the complete construct (J04450) with a N-term fusion standard.</P><P>Composition of each PCR tube:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD>Volume(µL)</TD></TR><TR><TD>10x Pfu Buffer w/ MgSO<SUB>4</SUB></TD><TD>2</TD></TR><TR><TD>dNTP (10mM)</TD><TD>1</TD></TR><TR><TD>N-term Fus Prefix</TD><TD>1</TD></TR><TR><TD>Biobrick Suffix</TD><TD>1</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>O</TD><TD>12.8</TD></TR><TR><TD>Template DNA (J04450)</TD><TD>2</TD></TR><TR><TD>Pfu DNA Polymerase</TD><TD>0.2</TD></TR></TBODY></TABLE><P>PCR conditions:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature (<SUP>o</SUP>C)</B></TD><TD><B>Time (mins)</B></TD><TD><B>Number of cycles</B></TD></TR><TR><TD>Initial Denaturation</TD><TD>95</TD><TD>2</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95</TD><TD>0.5</TD><TD>25</TD></TR><TR><TD>Annealing</TD><TD>48.2, 52.4, 56.0 (gradient)</TD><TD>0.5</TD><TD>25</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>2</TD><TD>25</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>10</TD><TD>1</TD></TR></TBODY></TABLE><P>Results: Amplification showing in gel, but fragment amplified was ~1000bp. RFP and dT should be ~800bp. Primers were checked and discovered the YFP/CFP primers were not compatible with the RFP. Concluded the amplification resulted from sloppy annealing of N-term fus prefix to the bio prefix resulting in the full construct being amplified (~1000bp)
</P><H3><SPAN class="mw-headline" id="KG_2"><FONT color="white">KG</FONT></SPAN></H3><P><B>Objective:</B> PCR xylE to attach fusion standard to the N-term and standard suffix to C-term.</P><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD><TD><B>Master Mix(x6)(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>12.8</TD><TD>76.8
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>2</TD><TD>12
</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Fusion Forward Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Standard Reverse Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Template DNA</TD><TD>2</TD><TD>12
</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2</TD><TD>1.2
</TD></TR></TBODY></TABLE><P>Fusion standard prefix has melting temperature of 66.5<SUP>o</SUP>C.  Standard suffix has melting temperature of 70.5<SUP>o</SUP>C.  
Annealing temperature of 61.5<SUP>o</SUP>C with 5 <SUP>o</SUP>C gradient.  
PCR- Conditions:
1.  95<SUP>o</SUP>C for 180 sec
2.  95<SUP>o</SUP>C for 30 sec
3.  61.5 +/- 5<SUP>o</SUP>C for 30 sec
4.  72<SUP>o</SUP>C for 150 sec
5.  72<SUP>o</SUP>C for 600 sec
(35 cycles)
</P><P><B>Objective:</B> PCR xylE to attach fusion standard to the C-term and standard suffix to N-term.</P><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD><TD><B>Master Mix(x6)(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>12.8</TD><TD>76.8
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>2</TD><TD>12
</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Standard Forward Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Fusion Reverse Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Template DNA</TD><TD>2</TD><TD>12
</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2</TD><TD>1.2
</TD></TR></TBODY></TABLE><P>Standard prefix has melting temperature of 68.7<SUP>o</SUP>C.  Standard suffix has melting temperature of 61.6<SUP>o</SUP>C.  
Annealing temperature of 56.6<SUP>o</SUP>C with 5 <SUP>o</SUP>C gradient.  
</P><P>PCR- Conditions:
1.  95<SUP>o</SUP>C for 180 sec
2.  95<SUP>o</SUP>C for 30 sec
3.  56.6 +/- 5<SUP>o</SUP>C for 30 sec
4.  72<SUP>o</SUP>C for 150 sec
5.  72<SUP>o</SUP>C for 600 sec
(35 cycles)
</P><H2><SPAN class="mw-headline" id="September_18.2C_2010"><FONT color="white">September 18, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JS.2C_DM"><FONT color="white">JS, DM</FONT></SPAN></H3><P><B>Objective:</B> Overexpress mms6.
Method: Followed overexpression protocol.  One flask was induced with 250(µL) IPTG while another was induced with 50(µL).
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Time (minutes)</B></TD><TD><B>OD #1 (600λ)</B></TD><TD><B>OD #2 (600λ)</B></TD></TR><TR><TD>30</TD><TD>0.139</TD><TD>0.133</TD></TR><TR><TD>60</TD><TD>0.200</TD><TD>0.188</TD></TR><TR><TD>90</TD><TD>0.372</TD><TD>0.360</TD></TR><TR><TD>120</TD><TD>0.702</TD><TD>0.704</TD></TR><TR><TD>150(T<SUB>0</SUB>)</TD><TD>0.871</TD><TD>0.810</TD></TR><TR><TD>210(T<SUB>1</SUB>)</TD><TD>2.600</TD><TD>2.500</TD></TR><TR><TD>270(T<SUB>2</SUB>)</TD><TD>3.317</TD><TD>3.300</TD></TR><TR><TD>330(T<SUB>3</SUB>)</TD><TD>4.408</TD><TD>3.946</TD></TR></TBODY></TABLE><UL><LI> Plated cells in presence of IPTG, Fe or IPTG and Fe on AMP(+) plates.  Incubated at 37 <SUP>o</SUP>C.  
</LI><LI> Preparation of these samples was achieved by taking a 60(µL) cell sample and inoculating 5 mL LB media.  These solutions were incubated at 37 <SUP>o</SUP>C for 30 min.  To 1 mL of culture, the appropriate amount of Fe2+ [0.938(µL)], Fe3+[1.88(µL)] or IPTG [1(µL)] was added.
</LI></UL><H3><SPAN class="mw-headline" id="TF"><FONT color="white">TF</FONT></SPAN></H3><P><B>Objective:</B> Obtain preparations of B0015 (dT) and J04450 (RFP).
</P><P><B>Method:</B> Used <A href="/Team:Lethbridge/Notebook/Protocols" title="Team:Lethbridge/Notebook/Protocols">Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep</A> protocol.
</P><P><B>Results:</B> Chloroform, DNA and isopropanol were mixed and therefore no products could be recovered.
</P><H2><SPAN class="mw-headline" id="September_20.2C_2010"><FONT color="white">September 20, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS_2"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Analyze PCR of mms6 (HB) and fluorescent proteins (AV).<B>Method:</B> Analyzed on 2% TAE agarose gel.
Load order:</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Lane</B></TD><TD><B>Sample</B></TD><TD><B>VolumeSample (µL)</B></TD><TD><B>Volume LoadingDye (µL)</B></TD></TR><TR><TD>1</TD><TD>HB1</TD><TD>10</TD><TD>2</TD></TR><TR><TD>2</TD><TD>HB2</TD><TD>10</TD><TD>2</TD></TR><TR><TD>3</TD><TD>HB3</TD><TD>10</TD><TD>2</TD></TR><TR><TD>4</TD><TD>HB4</TD><TD>10</TD><TD>2</TD></TR><TR><TD>5</TD><TD>HB5</TD><TD>10</TD><TD>2</TD></TR><TR><TD>6</TD><TD>100bp Ladder (Fermentas)</TD><TD>0.5</TD><TD>2(+10 H<SUB>2</SUB>0)</TD></TR><TR><TD>7</TD><TD>RFP1</TD><TD>10</TD><TD>2</TD></TR><TR><TD>8</TD><TD>RFP2</TD><TD>10</TD><TD>2</TD></TR><TR><TD>9</TD><TD>RFP3</TD><TD>10</TD><TD>2</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI>mms6 was not amplified
</LI><LI>RFP was amplified
</LI><LI>Expected size is ~800bp
</LI><LI>Actual size is &gt;1000bp
</LI><LI>We believe that the suffix segment of the fusion primer annealed rather than the FP segment. This would cause the promoter (R0040) of J04450 to be amplified, adding ~200bp, giving a final size of &gt;1000bp.
</LI></UL><P><B>Objective:</B> Confirm assembly (3 antibiotic) of K118021 and B0015 from Sept. 14, 2010 via PCR.<B>Method:</B> Set up 50uL reaction mixture with 2uL of ligated DNA. Used VF2 and VR primers.
Used PFU setting on Thermocycler<B>Results:</B><B>Conclusion:</B> Assembly did not work as intended.
</P><H3><SPAN class="mw-headline" id="HB"><FONT color="white">HB</FONT></SPAN></H3><P><B>Objective:</B> PCR mms6 to attach fusion standard to the N-term and standard suffix to C-term.</P><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>1X(µL)</B></TD><TD><B>Master Mix(x6)(µL)</B></TD></TR><TR><TD>Milli-Q H<SUB>2</SUB>O</TD><TD>12.8</TD><TD>76.8
</TD></TR><TR><TD>10x Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>2</TD><TD>12
</TD></TR><TR><TD>dNTPs</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Fusion Forward Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Standard Reverse Primer</TD><TD>1</TD><TD>6
</TD></TR><TR><TD>Template DNA</TD><TD>2</TD><TD>12
</TD></TR><TR><TD>Pfu polymerase</TD><TD>0.2</TD><TD>1.2
</TD></TR></TBODY></TABLE><P>Prefix fusion sense primer has melting temperature of 56.1<SUP>o</SUP>C.  Standard suffix primer has melting temperature of 61.3<SUP>o</SUP>C.  
</P><P>PCR- Conditions:
1.  95<SUP>o</SUP>C for 180 sec
2.  95<SUP>o</SUP>C for 30 sec
3.  51.1<SUP>o</SUP>C for 30 sec
4.  72<SUP>o</SUP>C for 150 sec
5.  72<SUP>o</SUP>C for 600 sec
(35 cycles)
</P><P>Program HBPREFU:
1.  49.1<SUP>o</SUP>C  2.  50.2<SUP>o</SUP>C  3.  51.4<SUP>o</SUP>C  4.  52.6<SUP>o</SUP>C  5.  53.5<SUP>o</SUP>C
</P><H2><SPAN class="mw-headline" id="September_21.2C_2010"><FONT color="white">September 21, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS_3"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Insert xylE (with N and C terminal fusion standards, obtained by PCR of K118021 by KG) into pSB1C3 plasmid for submission to registry.</P><P><B>Method:</B></P><UL><LI>Restriction of xylE PCR product and pSB1C3 (containing J04450 biobrick) via BioBrick Method using EcoRI-HF and PstI (Enzymes from NEB)
</LI><LI>Ligation of xylE PCR product and pSB1C3 via BioBrick Method using T4 DNA ligase (Enzyme from NEB)
</LI><LI>Transform into Library Efficiency Compentent DH5α Cells (Invitrogen)
</LI></UL><P><B>Results:</B></P><UL><LI>Obtained too numerous to count (TNTC) colonies. 
</LI><LI>Obtained ~100 white colonies (indicating removal of RFP and insertion of new BioBrick)
</LI></UL><P><B>Note:</B>
Parent plasmid (from PCR) not digested, possibly K118021 moved into pSB1C3 backbone.
</P><P><B>Follow-up:</B>
Screen white colonies by addition of catechol to solution containing white cells.
</P><H3><SPAN class="mw-headline" id="TF.2C_MC"><FONT color="white">TF, MC</FONT></SPAN></H3><P><B>Objective:</B> Obtain a preparation of B0015 (dT).
</P><P><B>Method:</B> Used <A href="/Team:Lethbridge/Notebook/Protocols" title="Team:Lethbridge/Notebook/Protocols">Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep</A> protocol.
</P><H3><SPAN class="mw-headline" id="AV_2"><FONT color="white">AV</FONT></SPAN></H3><P><B>Objective:</B> PCR amplify the signal sequences synthesized by Mr. Gene (K331007, K331008, K331009, K331012).</P><P>Composition of each PCR tube:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>Volume(µL)</B></TD></TR><TR><TD>10x Pfu Buffer w/ MgSO<SUB>4</SUB></TD><TD>2</TD></TR><TR><TD>dNTP (10mM)</TD><TD>1</TD></TR><TR><TD>VF2</TD><TD>1</TD></TR><TR><TD>VR</TD><TD>1</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>O</TD><TD>12.8</TD></TR><TR><TD>Template DNA</TD><TD>2</TD></TR><TR><TD>Pfu DNA Polymerase</TD><TD>0.2</TD></TR></TBODY></TABLE><P>Used iGEM thermocycler setting: PFU.</P><H2><SPAN class="mw-headline" id="September_21.2C_2010_2"><FONT color="white">September 21, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS_4"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Analyze PCR of K118021-B0015 and subsequent PCR (ADS) and PCR of &lt;partinfo&gt;K118021&lt;/partinfo&gt; to add either N or C terminal fusion standard (KG).<B>Method:</B> Analyze on 2% TAE agarose gel.
Load Order:</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Lane</B></TD><TD><B>Sample</B></TD><TD><B>VolumeSample (µL)</B></TD><TD><B>Volume LoadingDye (µL)</B></TD></TR><TR><TD>1</TD><TD>xylE F-S 1</TD><TD>5</TD><TD>1</TD></TR><TR><TD>2</TD><TD>xylE F-S 2</TD><TD>5</TD><TD>1</TD></TR><TR><TD>3</TD><TD>xylE F-S 3</TD><TD>5</TD><TD>1</TD></TR><TR><TD>4</TD><TD>xylE F-S 4</TD><TD>5</TD><TD>1</TD></TR><TR><TD>5</TD><TD>xylE F-S 5</TD><TD>5</TD><TD>1</TD></TR><TR><TD>6</TD><TD>xylE F-S 6</TD><TD>5</TD><TD>1</TD></TR><TR><TD>7</TD><TD>100bp Ladder (NEB)</TD><TD>0.5</TD><TD>1(+5 H<SUB>2</SUB>0)</TD></TR><TR><TD>8</TD><TD>xylE S-F 1</TD><TD>5</TD><TD>1</TD></TR><TR><TD>9</TD><TD>xylE S-F 2</TD><TD>5</TD><TD>1</TD></TR><TR><TD>10</TD><TD>xylE S-F 3</TD><TD>5</TD><TD>1</TD></TR><TR><TD>11</TD><TD>xylE S-F 4</TD><TD>5</TD><TD>1</TD></TR><TR><TD>12</TD><TD>xylE S-F 5</TD><TD>5</TD><TD>1</TD></TR><TR><TD>13</TD><TD>xylE S-F 6</TD><TD>5</TD><TD>1</TD></TR><TR><TD>14</TD><TD>Empty</TD></TR><TR><TD>15</TD><TD>PCR of K118021-B0015 (ADS)</TD><TD>1</TD><TD>1</TD></TR><TR><TD>16</TD><TD>Empty</TD></TR><TR><TD>17</TD><TD>Empty</TD></TR></TBODY></TABLE><P><B>Results:</B></P><UL><LI>Both KG PCR (Standard-Fusion and Fusion-Standard) amplified
</LI><LI>ADS PCR Amplified, but no insert present.
</LI></UL><H2><SPAN class="mw-headline" id="September_23.2C_2010"><FONT color="white">September 23, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JV_4"><FONT color="white">JV</FONT></SPAN></H3><P><B>Objective:</B> Characterized catechol degradation by xylE enzyme</P><P><B>Method:</B> Measured absorbance of catechol (275nm) and 2-hydroxymuconate semialdehyde (380nm).</P><UL><LI><B>Protocol:</B></LI><LI>1) Grow cells in M9 minimal medium
</LI><LI>2) Take 1/10 dilution of cells
</LI><LI>3) Introduce 1µL of 0.05M catechol solution into the cell dilution. (Final concentration of 50&amp;microM;).
</LI><LI>4) Quench the reaction with 5A% w/v trichloroacetate at certain time points. (0,15sec, 30sec, 45sec, 60sec, 2min, 3min, 4min, 5min, 10min).
</LI><LI>5) Spin down cells.
</LI><LI>6) Measure absorbance of supernatant.</LI></UL><P><B>Results: Cuvette used interfered with Spectra. </B></P><H3><SPAN class="mw-headline" id="ADS_5"><FONT color="white">ADS</FONT></SPAN></H3><P><B><FONT size="+1">NOTE:</FONT></B> In all transformations, heat shock step was missed. HOWEVER, all transformations showed significant number of colony forming units.<B>Objective:</B> Move xylE (two biobrick; one with Fusion prefix, one with fusion suffix) into pSB1C3.<B>Method:</B></P><UL><LI>Restriction of xylE PCR product and pSB1C3 (containing J04450 biobrick) via BioBrick Method using EcoRI-HF and PstI (Enzymes from NEB)
</LI><LI>Ligation of xylE PCR product and pSB1C3 via BioBrick Method using T4 DNA ligase (Enzyme from NEB) 
</LI><LI>Incubated 30 min at RT
</LI></UL><LI>Transform into Subcloning Efficiency Compentent DH5α Cells (Invitrogen)
</LI><P><B>Results:</B> TBD <B>Follow-up:</B> TBD</P><P><B>Objective:</B> Create glycerol stocks of &lt;partinfo&gt;J04450&lt;/partinfo&gt; in pSB1A3 and pSB1T3 for use in RFP-BioBrick Assembly.<B>Method:</B> Transform into Subcloning Efficiency Competent DH5α Cells (Invitrogen)
Obtained all plasmid DNA from 2010 Kit Plate 1
</P><UL><LI>J04450 in pSB1A3 - Well 1C
</LI><LI>J04450 in pSB1T3 - Well 7A
</LI></UL><P><B>Results:</B> Obtained TNTC colonies<B>Follow-up:</B></P><UL><LI>Grow overnight cultures
</LI><LI>Generate Glycerol Stocks
</LI><LI>Generate Plasmid DNA via Maxiprep
</LI></UL><P><B>Objective:</B> Create glycerol stocks of received synthesized (Mr. Gene) signal peptides.<B>Method:</B> Transform into Subcloning Efficiency Competent DH5α Cells (Invitrogen) plasmid DNA containing the following BioBricks:
</P><UL><LI>1) &lt;partinfo&gt;K331007&lt;/partinfo&gt; - β-lactamase Bla Signal Sequence
</LI><LI>2) &lt;partinfo&gt;K331008&lt;/partinfo&gt; - Outer Membrane Protein ompA
</LI><LI>3) &lt;partinfo&gt;K331009&lt;/partinfo&gt; - Heat Stable Toxin I
</LI><LI>4) &lt;partinfo&gt;K331012&lt;/partinfo&gt; - Penicillin Binding Protein DacA
</LI><LI>All inserts in pMA-T vector (Standard Mr. Gene vector)
</LI></UL><P><B>Results:</B> Obtained TNTC Cells <B>Follow-up:</B></P><UL><LI>1) Grow overnight cultures
</LI><LI>2) Purify pDNA
</LI><LI>3) Move into pSB1C3 plasmid
</LI><LI>4) Verify sequence 
</LI><LI>5) Submit to registry for sequencing
</LI></UL><H3><SPAN class="mw-headline" id="TF.2C_MC_2"><FONT color="white">TF, MC</FONT></SPAN></H3><P><B>Objective:</B> Obtain a preparation of J04450 (RFP).</P><P><B>ethod:</B> Used <A href="/Team:Lethbridge/Notebook/Protocols" title="Team:Lethbridge/Notebook/Protocols">Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep</A> protocol.
</P><H2><SPAN class="mw-headline" id="September_24.2C_2010"><FONT color="white">September 24, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS_6"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Generate plasmid DNA of &lt;partinfo&gt;E1010&lt;/partinfo&gt; for downstream PCR<B>Method:</B> Transform plasmid DNA into Subcloning Efficiency Competent DH5α Cells (Invitrogen)
DNA obtained from 2010 Kit Plate 1 Well 18F (E1010 in pSB2K3)<B>Results:</B> Obtained <FONT color="red"> TBD</FONT> colonies<B>Follow-up:</B></P><UL><LI>Grow overnight cultures (Generate glycerol stocks)
</LI><LI>Purify plasmid DNA (Generate pDNA stocks)
</LI><LI>PCR to add terminal fusion standards
</LI></UL><P><B>Objective:</B> Create glycerol stocks of J04450 in pSB1K3 for use in RFP-BioBrick Assembly.<B>Method:</B> Transform into Subcloning Efficiency Competent DH5α Cells (Invitrogen)
Obtained plasmid DNA from 2010 Kit Plate 1 well 5A (J04450 in pSB1K3)<B>Results:</B> Obtained TNTC colonies<B>Follow-up:</B></P><UL><LI>Grow overnight cultures
</LI><LI>Generate Glycerol Stocks
</LI><LI>Generate Plasmid DNA via Maxiprep
</LI></UL><H2><SPAN class="mw-headline" id="September_25.2C_2010"><FONT color="white">September 25, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="JV_5"><FONT color="white">JV</FONT></SPAN></H3><P><B>Objective:</B> Extract Plasmid DNA from DH5α cells.</P><P><B>Method:</B>Qiagen spin column protocol.</P><UL><LI>&lt;partinfo&gt;K331007&lt;/partinfo&gt; (in pMA-T vector)
</LI><LI>&lt;partinfo&gt;K331008&lt;/partinfo&gt; (in pMA-T vector)
</LI><LI>&lt;partinfo&gt;K331009&lt;/partinfo&gt; (in pMA-T vector)
</LI><LI>&lt;partinfo&gt;K331012&lt;/partinfo&gt; (in pMA-T vector)</LI></UL><P>Cells containing plasmids were put into glycerol stocks and put into HJ's -80<SUP>o</SUP>C.</P><H3><SPAN class="mw-headline" id="ADS_7"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Move plasmid DNA made by JV (above, Sept 25, 2010) into pSB1C3 for submission to the Standard Registry of Parts.<B>Method:</B></P><UL><LI>1) Digest at 37<SUP>o</SUP> for 10 min with PstI and EcoRI:
</LI><LI>pSB1C3 containing &lt;partinfo&gt;J04450&lt;/partinfo&gt;</LI><LI>pMA-T with &lt;partinfo&gt;K331007&lt;/partinfo&gt;, &lt;partinfo&gt;K331008&lt;/partinfo&gt;, &lt;partinfo&gt;K331009&lt;/partinfo&gt;, and &lt;partinfo&gt;K331012&lt;/partinfo&gt;.
</LI></UL><LI>2) Heat Kill at 80<SUP>o</SUP>C for 20 min
</LI><LI>3) Mix 2µL of pSB1C3 with each biobrick, ligate with T4 DNA ligase for 10 min
</LI><LI>4) Transform into subcloning efficiency DH5α cells, plate on ampicillin plates.</LI><P><B>Results:</B> Obtained <FONT color="red">RESULT!!!</FONT> positive colonies</P><P><B>Objective:</B> Assemble KG's C-terminal fusion xylE (&lt;partinfo&gt;K331021&lt;/partinfo&gt;) (<FONT color="red">UNCONFIRMED</FONT>) and C-terminal Arginine + dT (&lt;partinfo&gt;S04261&lt;/partinfo&gt;) to create &lt;partinfo&gt;K331011&lt;/partinfo&gt;. <B>Method:</B></P><UL><LI>1) Digest at 37<SUP>o</SUP> for 10 min:
</LI><LI>pSB1C3 containing &lt;partinfo&gt;J04450&lt;/partinfo&gt; with PstI and EcoRI
</LI><LI>&lt;partinfo&gt;K331021&lt;/partinfo&gt; with EcoRI, SpeI
</LI><LI>&lt;partinfo&gt;S04261&lt;/partinfo&gt; with XbaI and PstI
</LI></UL><LI>2) Heat Kill at 80<SUP>o</SUP>C for 20 min
</LI><LI>3) Mix 2µL of pSB1C3 with each biobrick, ligate with T4 DNA ligase for 10 min
</LI><LI>4) Transform into subcloning efficiency DH5α cells, plate on ampicillin plates.</LI><P><B>Results:</B> Obtained <FONT color="red">RESULT!!!</FONT> positive colonies</P><H2><SPAN class="mw-headline" id="September_26.2C_2010"><FONT color="white">September 26, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS_8"><FONT color="white">ADS</FONT></SPAN></H3><P><B>Objective:</B> Purify plasmid DNA from 5mL cultures grown from transformations of KG's ligated PCRs containing (<FONT color="red">UNCONFIRMED</FONT>) &lt;partinfo&gt;K331020&lt;/partinfo&gt; and &lt;partinfo&gt;K331021&lt;/partinfo&gt;.<B>Method:</B> Used Qiagen spin column method with BioBasic EZ-10 spin columns.</P><P><B>Objective:</B> Assemble the following:</P><UL><LI>&lt;partinfo&gt;K331020&lt;/partinfo&gt; + &lt;partinfo&gt;B0015&lt;/partinfo&gt; to obtain <FONT color="red"> To be named</FONT> biobrick
</LI><LI>&lt;partinfo&gt;K331021&lt;/partinfo&gt; + &lt;partinfo&gt;S04261&lt;/partinfo&gt; to obtain <FONT color="red"> To be named</FONT> biobrick
</LI></UL><P><B>Method:</B></P><UL><LI>Digest &lt;partinfo&gt;K331020&lt;/partinfo&gt; and &lt;partinfo&gt;K331021&lt;/partinfo&gt; with EcoRI and SpeI
</LI><LI>Digest &lt;partinfo&gt;B0015&lt;/partinfo&gt; and &lt;partinfo&gt;S04261&lt;/partinfo&gt; with XbaI and PstI
</LI><LI>Digest &lt;partinfo&gt;J04450&lt;/partinfo&gt; in pSB1C3 with EcoRI and PstI
</LI></UL><P>Incubate each at 37<SUP>o</SUP>C for 10 min
Heat kill at 80<SUP>o</SUP>C for 20 min 
Ligate with T4 Ligase for 10 min at room temperature
Transform into DH5&amp;alpha subcloning efficiency cells<B>Results:</B> Obtained <FONT color="red">RESULTS!!!!</FONT> positive colonies.
Also analyzed minipreps on 1.5% TAE Agarose Gel</P><H2><SPAN class="mw-headline" id="September_29.2C_2010"><FONT color="white">September 29, 2010</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="ADS_9"><FONT color="white">ADS</FONT></SPAN></H3><P>Received synthesized parts from Mr. GENE (all in pMA vectors)</P><UL><LI>&lt;partinfo&gt;K331022&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;K331023&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;K331024&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;K331025&lt;/partinfo&gt;</LI></UL><P>Each tube contained 5µg pDNA; reconstitute in 50µL of TE buffer to get 100ng/µL concentration
Transform 1µL into DH5α subcloning efficiency cells
Obtained TNTC colonies
Started overnight 5mL cultures for miniprep and insertion into pSB1C3 vector.
</P><H3><SPAN class="mw-headline" id="AV_3"><FONT color="white">AV</FONT></SPAN></H3><P><B>Objective:</B> PCR the N-term fusion tag onto RFP (E1010) using sloppy annealing of the YFP/CFP fusion primers. </P><P>Composition of each PCR tube:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD>Volume(µL)</TD></TR><TR><TD>10x Pfu Buffer w/ MgSO<SUB>4</SUB></TD><TD>2</TD></TR><TR><TD>dNTP (10mM)</TD><TD>1</TD></TR><TR><TD>N-term Fus Prefix</TD><TD>1</TD></TR><TR><TD>Biobrick Suffix</TD><TD>1</TD></TR><TR><TD>MilliQ H<SUB>2</SUB>O</TD><TD>12.8</TD></TR><TR><TD>Template DNA (J04450)</TD><TD>2</TD></TR><TR><TD>Pfu DNA Polymerase</TD><TD>0.2</TD></TR></TBODY></TABLE><P>PCR conditions:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature (<SUP>o</SUP>C)</B></TD><TD><B>Time (mins)</B></TD><TD><B>Number of cycles</B></TD></TR><TR><TD>Initial Denaturation</TD><TD>95</TD><TD>2</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95</TD><TD>0.5</TD><TD>25</TD></TR><TR><TD>Annealing</TD><TD>47.6, 50.5, 53.8 (gradient)</TD><TD>0.5</TD><TD>25</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>2</TD><TD>25</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>10</TD><TD>1</TD></TR></TBODY></TABLE><P>Results: Gel has not been run. Have decided to order RFP specific primers for next year.</P></TABLE></TABLE></TABLE></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Lethbridge/Notebook/Lab_Work/September">http://2010.igem.org/Team:Lethbridge/Notebook/Lab_Work/September</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/Notebook/Lab_Work/September" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/Notebook/Lab_Work/September" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Lab_Work/September&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Lab_Work/September&amp;oldid=193201" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></TABLE></BLOCKQUOTE></DIV></DIV></DIV></DIV></BODY></HTML>