<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV style="background-color:#000000; color:white"><P><CENTER><FONT color="white">Feel free to look around our notebook!
</FONT></CENTER></P><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Common_Protocols:"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Common Protocols:</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Assembly_of_BioBricks_using_the_Red.2FWhite_3-Antibiotic_Assembly_Method"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Assembly of BioBricks using the Red/White 3-Antibiotic Assembly Method</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Step_1_-_Restriction"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Step 1 - Restriction</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Step_2_-_Ligation"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Step 2 - Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Step_3_-_Transformation"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Step 3 - Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Step_4_-_Selection_of_Colonies_and_Colony_PCR"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Step 4 - Selection of Colonies and Colony PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Step_5_-_Plasmid_DNA_Purification"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Step 5 - Plasmid DNA Purification</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Protocols_prior_to_the_development_of_Red.2FWhite_3-Antibiotic_assembly"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Protocols prior to the development of Red/White 3-Antibiotic assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Competent_Cell_Transformation"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Competent Cell Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Plasmid_DNA_Purification_by_Boiling_Lysis_.28Small_Scale_AKA_Miniprep.29"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Plasmid DNA Purification by Boiling Lysis (Small Scale AKA Miniprep)</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Restriction_of_Plasmid_DNA_.28pDNA.29"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Restriction of Plasmid DNA (pDNA)</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Additional_Protocols"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Additional Protocols</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Overexpression"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Overexpression</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Plasmid_DNA_Purification_by_Alkaline_Lysis_.28Large_Scale_AKA_Maxiprep.29"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep)</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Common_Protocols:"><FONT color="white">Common Protocols:</FONT></SPAN></H1><H2><SPAN class="mw-headline" id="Assembly_of_BioBricks_using_the_Red.2FWhite_3-Antibiotic_Assembly_Method"><FONT color="white">Assembly of BioBricks using the <FONT color="red">R</FONT>e<FONT color="red">d</FONT>/<FONT color="red">W</FONT>h<FONT color="red">i</FONT>t<FONT color="red">e</FONT> 3-Antibiotic Assembly Method</FONT></SPAN></H2><P>This method is a variant of the 3-Antibiotic Assembly Method that has been developed by Ginkgo Bioworks and New England Biolabs.</P><UL><LI>In the 3-Antibiotic Assembly Method, the destination backbone (<A href="http://partsregistry.org/wiki/index.php/Part:pSB1A3" target="new"><FONT color="#00DC00"> pSB1A3</FONT></A>, <A href="http://partsregistry.org/wiki/index.php/Part:pSB1C3" target="new"><FONT color="#00DC00"> pSB1C3</FONT></A>, <A href="http://partsregistry.org/wiki/index.php/Part:pSB1K3" target="new"><FONT color="#00DC00"> pSB1K3</FONT></A>, or <A href="http://partsregistry.org/wiki/index.php/Part:pSB1T3" target="new"><FONT color="#00DC00"> pSB1T3</FONT></A>) is amplified via PCR, using DNA received in the 2010 Distribution. </LI></UL><P>We had difficulty generating a large quantity of plasmid backbone in this manner, and as a result were unable to assemble biobricks. 
As an alternative, we exploited the ability of part <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J04450" target="new"><FONT color="#00DC00"> BBa_J04450</FONT></A> (expressing red fluorescent protein - RFP) to create in its host cell a very strong red color following incubation; this has proven a useful and successful variant of the 3-Antibiotic Assembly Method.
We retained the selection advantage of the 3-Antibiotic Assembly Method (associated with having a destination plasmid containing a different antibiotic resistance than the upstream and downstream plasmid) as the expressing RFP biobrick is available in a wide variety of BBF plasmids.  
With this variation, we can not only visually screen our colonies for the presence of our expected assembly product (as indicated by a lack of red fluorescence), we also have a built-in negative control in that a lack of assembly product will result in a re-ligation of the expressing RFP biobrick into the destination backbone, producing red colonies.</P><H3><SPAN class="mw-headline" id="Step_1_-_Restriction"><FONT color="white">Step 1 - Restriction</FONT></SPAN></H3><P>Performed according to Ginkgo Bioworks/NEB BioBrick Assembly Kit, with several modifications.<B>Digestion of upstream part</B></P><UL><LI>5µL Upstream part plasmid<SUP>†</SUP></LI><LI>0.5µL EcoRI-HF
</LI><LI>0.5µL SpeI
</LI><LI>2.5µL 10x NEBuffer 2
</LI><LI>0.25µL 100x BSA
</LI><LI>16.25µL MilliQ H<SUB>2</SUB>O
</LI></UL><P><B>Digestion of downstream part</B></P><UL><LI>5µL Downstream part plasmid<SUP>†</SUP></LI><LI>0.5µL XbaI
</LI><LI>0.5µL PstI
</LI><LI>2.5µL 10x NEBuffer 2
</LI><LI>0.25µL 100x BSA
</LI><LI>16.25µL MilliQ H<SUB>2</SUB>O
</LI></UL><P><B>Digestion of destination plasmid</B></P><UL><LI>5µL Destination plasmid<SUP>†‡</SUP></LI><LI>0.5µL EcoRI-HF
</LI><LI>0.5µL PstI
</LI><LI>2.5µL 10x NEBuffer 2
</LI><LI>0.25µL 100x BSA
</LI><LI>16.25µL MilliQ H<SUB>2</SUB>O
</LI></UL><P>Restriction digests were incubated at 37<SUP>o</SUP>C for 10 minutes.
Following each restriction digest, all samples were subjected to heating at 80<SUP>o</SUP>C for 20 minutes to irreversibly denature the restriction endonucleases.† Typical concentrations obtained from our minipreps are approximately 50ng/µL, therefore adding 5µL of plasmid DNA gives the restriction reaction a concentration of 10ng/µL, as recommended in the NEB BioBrick Assembly Kit Literature. ‡ Destination plasmid is either pSB1A3, pSB1C3, pSB1K3, or pSB1T3, containing part <A href="http://partsregistry.org/wiki/index.php/Part:BBa_J04450" target="new"><FONT color="#00DC00">BBa_J04450</FONT></A>  which is the expressing red fluorescent protein.
</P><H3><SPAN class="mw-headline" id="Step_2_-_Ligation"><FONT color="white">Step 2 - Ligation</FONT></SPAN></H3><P>Performed according to Ginkgo Bioworks/NEB BioBrick Assembly Kit<B>Ligation of Upstream and Downstream parts into Destination Plasmid</B></P><UL><LI>2µL Upstream part digestion
</LI><LI>2µL Downstream part digestion
</LI><LI>2µL Destination Plasmid digestion
</LI><LI>2µL 10x T4 DNA Ligase Buffer
</LI><LI>1µL T4 DNA Ligase
</LI><LI>11µL MilliQ H<SUB>2</SUB>O
</LI></UL><P>Ligation mixes were incubated on the bench top (~20<SUP>o</SUP>C) for 10 minutes, then transformed into competent DH5α cells.
</P><H3><SPAN class="mw-headline" id="Step_3_-_Transformation"><FONT color="white">Step 3 - Transformation</FONT></SPAN></H3><OL><LI>Thaw 50µL of aliquotted competent cells (DH5α) on ice
</LI><LI>Gently pipet 2.0µL (~1ng) DNA (from ligation mix) into competent cells
</LI><LI>Incubate the cells on ice for 30 minutes
</LI><LI>Heat shock the cells <B>in a water bath</B> at <U>42<SUP>o</SUP>C for EXACTLY 45 seconds</U></LI><LI>Incubate cells on ice for 5 minutes
</LI><LI>Add 400µL of sterile SOC media to the cells and incubate at 37<SUP>o</SUP>C for 90 minutes with shaking (250RPM)
</LI><LI>Plate 200µL on LB agar plate containing the appropriate antibiotic (reserve remaining cells and re-plate if no growth)
</LI><LI>Allow cell suspension to be absorbed into agar by leaving agar side down for 10-15 minutes
</LI><LI>Incubate the plates in the 37<SUP>o</SUP>C incubator for approximately 36 hours (agar on top)
</LI></OL><H3><SPAN class="mw-headline" id="Step_4_-_Selection_of_Colonies_and_Colony_PCR"><FONT color="white">Step 4 - Selection of Colonies and Colony PCR</FONT></SPAN></H3><P>Following ~36 hour incubation at 37<SUP>o</SUP>C, plates were inspected, and white colonies were picked and subsequently subjected to Colony PCR.<B>Colony Picking</B></P><OL><LI>Pick a colony from a transformation plate using a sterile toothpick or micropipette tip
</LI><LI>Transfer the cells to a 1.5mL microcentrifuge tube containing 50µL of sterile MilliQ H<SUB>2</SUB>O
</LI><LI>Place microcentrifuge tube in heat block at 99<SUP>o</SUP>C for 5 minutes to lyse the cells and denature DNases.
</LI><LI>Centrifuge at max speed in a table-top microcentrifuge for 1 minute to remove cell debris
</LI><LI>Use 5µL of supernatant as template for PCR
</LI></OL><P><B>PCR to confirm length of newly assembled part</B>
Reaction Conditions:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Component</B></TD><TD><B>Concentration</B></TD><TD><B>Volume (µL)</B></TD></TR><TR><TD>MilliQ H<SUB>2</SUB>O</TD><TD>N/A</TD><TD>33.8</TD></TR><TR><TD>Pfu Buffer with MgSO<SUB>4</SUB></TD><TD>10x</TD><TD>5</TD></TR><TR><TD>dNTP</TD><TD>10mM</TD><TD>2</TD></TR><TR><TD>VF2 Forward Primer</TD><TD>10µL</TD><TD>2</TD></TR><TR><TD>VR Reverse Primer</TD><TD>10µL</TD><TD>2</TD></TR><TR><TD>Template DNA</TD><TD>N/A</TD><TD>5</TD></TR><TR><TD>Pfu DNA Polymerase</TD><TD>2U/µL</TD><TD>0.2</TD></TR></TBODY></TABLE><P>Cycling Conditions:
</P><TABLE><TABLE border="3"><TBODY><TR><TD><B>Step</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1-Initial Denature</TD><TD>98<SUP>o</SUP>C</TD><TD>3 minutes</TD></TR><TR><TD>2-Denature</TD><TD>98<SUP>o</SUP>C</TD><TD>30 seconds</TD></TR><TR><TD>3-Anneal<SUP>†</SUP></TD><TD>58<SUP>o</SUP>C</TD><TD>30 seconds</TD></TR><TR><TD>4-Extend</TD><TD>72<SUP>o</SUP>C</TD><TD>30 seconds<SUP>‡</SUP></TD></TR><TR><TD>5-Final Extend</TD><TD>72<SUP>o</SUP>C</TD><TD>15 minutes</TD></TR></TBODY></TABLE><P>† Annealing temperature for Phusion is MT +3<SUP>o</SUP>C ‡ Extend time is 15-30 seconds per kb; time adjusted accordingly </P><H3><SPAN class="mw-headline" id="Step_5_-_Plasmid_DNA_Purification"><FONT color="white">Step 5 - Plasmid DNA Purification</FONT></SPAN></H3><P>We followed Qiagen's QIAPrep Spin Miniprep protocol, as follows:
</P><OL><LI>Pellet cells from 1-1.5mL of liquid culture in a microcentrifuge tube (remove and discard supernatant)
</LI><LI>Resuspend cells in 250µL buffer P1
</LI><LI>Add 250µL buffer P2 and mix by inverting 4-6 times
</LI><LI>Add 350µL buffer N3 and mix immediately by inverting 4-6 times
</LI><LI>Centrifuge for 10 minutes at 13000RPM at room temperature in a table-top microcentrifuge
</LI><LI>Apply resulting supernatant to spin column
</LI><LI>Centrifuge for 60 seconds (discard flow through)
</LI><LI>Wash column with 750µL of buffer PE (discard flow through)
</LI><LI>Centrifuge for an addition 60 seconds to remove residual PE buffer (discard flow through)
</LI><LI>Transfer column to clean 1.5mL microcentrifuge tube
</LI><LI>Add 50µL buffer EB, let stand for 1-5 minutes, and centrifuge for 1 minute (retain flow through)
</LI></OL><H2><SPAN class="mw-headline" id="Protocols_prior_to_the_development_of_Red.2FWhite_3-Antibiotic_assembly"><FONT color="white">Protocols prior to the development of Red/White 3-Antibiotic assembly</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="Competent_Cell_Transformation"><FONT color="white">Competent Cell Transformation</FONT></SPAN></H3><OL><LI>Thaw 20µL of aliquotted cells (DH5α or BL21(DE3)) on ice.</LI><LI>Gently pipet 2.0µL of DNA into competent cells
ATTENTION: 
Do not perform any additional mixing
Never use more DNA that <U>10% of the volume of the competent cells </U>otherwise the cells get destroyed by osmotic shock</LI><LI>Incubate the cells on ice for 30 minutes.</LI><LI>Heat shock the cells in a water bath at <U>42<SUP>o</SUP>C for EXACTLY 45 seconds.</U></LI><LI>Incubate the cells on ice for 5 minutes.</LI><LI>Add 250µL sterile LB media to the cells and incubate at 37<SUP>o</SUP>C for 1 hour with shaking (200RPM).</LI><LI>Plate 100µL and 50µL on prewarmed LB agar plate containing the appropriate antibiotic.
For ligations, plate all 250µL.</LI><LI>Leave plate for 10-15 minutes to soak the cell suspension into the agar.</LI><LI>Flip plate over (agar on top)</LI><LI>Incubate the plates in the 37<SUP>o</SUP>C incubator overnight</LI></OL><H3><SPAN class="mw-headline" id="Plasmid_DNA_Purification_by_Boiling_Lysis_.28Small_Scale_AKA_Miniprep.29"><FONT color="white">Plasmid DNA Purification by Boiling Lysis (Small Scale AKA Miniprep)</FONT></SPAN></H3><OL><LI>Aseptically transfer 1.5mL of each overnight culture to a 1.5mL microcentrifuge tube (MCT) and pellet the cells by centrifugation in a benchtop microcentrifuge (2min at 13000RPM)</LI><LI>Remove and discard as much of the supernatant as possible by aspiration (e.g with a Pasteur Pipette). Do not suck up the cell pellet!!</LI><LI>Rinse the cell pellet by washing 1.0mL of sterile MilliQ H<SUB>2</SUB>O gently down the inside wall of the MCT. This removes any traces of the supernatant adhering to the MCT wall while minimizing the disturbance to the cell pellet. (/li&gt;</LI><LI>Resuspend the cell pellet in 350µL of STET. </LI><LI>Add 25µL of the Lysozyme solution and mix by inversion.</LI><LI>Place the MCT in the bioling water bath for EXACTLY 35 seconds, remove and incubate on ice for 5 minutes.</LI><LI>Pellet the cellular debris by centrifugation at 13000RPM for 15 minutes. Transfer the supernatant to a fresh MCT and discard the pellet.</LI><LI>Precipitate the plasmid DNA by adding 40µL of 3.0M sodium acetate (pH 5.2) and 420µL isopropanol. Mix by inversion. Mix by inversion and incubate for 5 minutes at room temperature.</LI><LI>Pellet the plasmid DNA by centrifugation at 13000RPM for 10 minutes at 4<SUP>o</SUP>C. A pellet of plasmid DNA should be visible at the base of the MCT when complete.</LI><LI>Being careful not to disturb the pellet, discard the supernatant and rinse the pellet with 500µL of ice cold ethanol.</LI><LI>Repeat above step.</LI><LI>Invert and tap the open MCT several times against a piece of paper towel on your bench to remove as much ethanol as possible.</LI><LI>Store the open MCT at room temperature for approximately 10 minutes to allow all remaining traces of ethanol to evaporate</LI><LI>Add 50µL of TE (pH 8.0) containing RNase A and resuspend the plasmid DNA by flicking the base of the MCT with your finger. The plasmid DNA is ready for use or can be stored long term at -20<SUP>o</SUP>C.</LI></OL><H3><SPAN class="mw-headline" id="Restriction_of_Plasmid_DNA_.28pDNA.29"><FONT color="white">Restriction of Plasmid DNA (pDNA)</FONT></SPAN></H3><OL><LI>In a 1.5mL microcentrifuge tube, add MilliQ H<SUB>2</SUB>O to final volume of 20µL, 2µL of restriction enzyme buffer, 2µL of plasmid DNA, and 0.25µL of each restriction enxyme.</LI><LI>Incubate at 37<SUP>o</SUP>C for 1 hour.</LI><LI>Heat kill restriction enzymes on a heat block at 80<SUP>o</SUP>C for 20 minutes.</LI><LI>Store short-term on ice or long-term at -20<SUP>o</SUP>C.</LI></OL><H2><SPAN class="mw-headline" id="Additional_Protocols"><FONT color="white">Additional Protocols</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="Overexpression"><FONT color="white">Overexpression</FONT></SPAN></H3><OL><LI>Inoculate two 5mL overnight cultures in LB media containing the appropriate antibiotic corresponding to the resistance of the plasmid backbone on which the gene is located; incubated at at 37<SUP>o</SUP>C in shaker.</LI><LI>Transfer the two 5mL cultures into an 2000mL Erlenmeyer flask containing 500mL of LB media containing the same antibiotic.</LI><LI>Measure and record initial read OD<SUB>600</SUB> reading (Time = 0) blanking against the uninoculated LB media.</LI><LI>Begin incubation at 37<SUP>o</SUP>C in shaker.</LI><LI>Measure and record another OD<SUB>600</SUB> reading 1 hour after (Time = 1) and continue record OD<SUB>600</SUB> every 30 minutes (T + 0.5) until OD<SUB>600</SUB> = 0.600 is reached.</LI><LI>When OD<SUB>600</SUB> has reached 0.600, aliquot out a 2mL sample and induce the 500mL culture (ie. if promoter is pLacI, induce with IPTG) and continue incubating in 37<SUP>o</SUP>C shaker.</LI><LI>Aliquot 2mL samples and record the OD<SUB>600</SUB> reading every hour for 3 hours following induction.</LI><LI>To run SDS PAGE, centrifuge the samples at 13,000 rpm for 5 minutes and discard the supernatant. Resuspend the cell pellets with 100µL of 8M urea. Mix 10µL of the urea/cell sample with 5µL of 6X loading dye. 
</LI></OL><H3><SPAN class="mw-headline" id="Plasmid_DNA_Purification_by_Alkaline_Lysis_.28Large_Scale_AKA_Maxiprep.29"><FONT color="white">Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep)</FONT></SPAN></H3><OL><LI>Grow up a 500mL overnight culture in LB media containing the appropriate antibiotic corresponding to the resistance of the plasmid.</LI><LI>Centrifuge the cells at 5000 rpm for 10 minutes, discard the supernatant, transfer pellet to a 50mL falcon tube, record the weight of the pellet, and store at -20<SUP>o</SUP>C. ( 1- 2.5g cell pellets are expected)</LI><LI>Resuspend the cell pellet in 6mL Alkaline Lysis Solution I (ALS1). Vortex carefully and slowly; may use a clean glass stir rod. Add 20µL of 1mg/mL RNase A.</LI><LI>Add 1 mL of 10 mg/mL lysozyme (in 20mM Tris-HCl, pH 8.0)</LI><LI>Incubate at room temperature for 10 minutes.</LI><LI>Add 12mL of <U>fresh</U> ALS2, mix well, do not vortex, and incubate on ice for 10 minutes.</LI><LI>Add 9mL of ice cold ALS3, mix well, and incubate for 10 minutes on ice.</LI><LI>Centrifuge at 4<SUP>o</SUP>C and 5000 rpm for 15 minutes.</LI><LI>Decant supernatant filter within funnel into fresh 50mL centrifuge tube.</LI><LI><B>1:1 phenol:chloroform extraction:</B> In the fume hood, add 4mL of phenol/chloroform (1:1 -- 2mL of phenol + 2mL of chloroform), vortex for 15 seconds, centrifuge at 4000 rpm and 12<SUP>o</SUP>C for 4 minutes, and collect the upper aqueous phase. To the aqueous phase, add 4mL of chloroform, vortex for 15 seconds, centrifuge at 4000 rpm and 12<SUP>o</SUP>C for 4 minutes, and save the upper aqueous layer.</LI><LI>Add 0.6 volumes of isopropanol to the saved aqueous layer and incubate on ice for 10 minutes. (Alternatively, precipitate overnight at -20<SUP>o</SUP>C)</LI><LI>Centrifuge at 4<SUP>o</SUP>C and 5000 rpm for 15 minutes.</LI><LI>Decant supernatant into a fresh falcon tube (can be saved for further plasmid DNA isolation)</LI><LI>Wash DNA pellet with 2mL 70% ethanol; centrifuge at 4<SUP>o</SUP>C and 5000 rpm for 5 minutes.</LI><LI>Air dry the DNA pellet; resuspend in 4mL 20mM Tris-HCl pH 8.0 by vortexing.</LI><LI>Add 200µL 1mg/mL RNase A and incubate at room temperature overnight.</LI><LI><B>Ethanol precipitation:</B> Add 0.1 volumes of 3M Na-acetate pH 5.2, add 2 volumes of cold ethanol (-20<SUP>o</SUP>C), incubate 30 minutes on ice (or longer at -20<SUP>o</SUP>C). </LI><LI>Centrifuge at 4<SUP>o</SUP>C and 4500 rpm for 15 minutes; carefully remove supernatant with 200µL pipette.</LI><LI>Wash pellet with 750µL of 70% ethanol, centrifuge at 4<SUP>o</SUP>C and 4500 rpm for 2 minutes, remove supernatant, and air dry the pellet for 10 minutes.</LI><LI>Resuspend the pellet in 200µL of MilliQ H<SUB>2</SUB>O (or TE).</LI></OL><P><B>Solutions:</B><I>ALS1:</I> 50mM glucose, 25mM Tris-Cl, 10mM EDTA pH 8.0<I>ALS2:</I> 0.2M NaOH, 1% (w/v) SDS<I>ALS3:</I> 60mL of 5M K-acetate, 11.5mL glacial acetic acid, 28.5mL MilliQ H<SUB>2</SUB>O<I>RNase A:</I> 1mg/mL in 20mM Tris-HCl, pH 8.0
<I>Lysozyme:</I> 10mg/mL in 20mM Tris-HCl, pH 8.0
</P></TABLE></TABLE></BLOCKQUOTE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Lethbridge/Notebook/Protocols">http://2010.igem.org/Team:Lethbridge/Notebook/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/Notebook/Protocols&amp;oldid=167630" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>