<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV style="background-color:#000000; color:white"><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Homology_Modeling"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Homology Modeling</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_Method"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Method</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Results_and_Limitations"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Results and Limitations</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#References"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Homology_Modeling"><FONT color="white">Homology Modeling</FONT></SPAN></H1><P>One aspect of our project is working on the localization of catechol-2,3,-dioxygenase (and other proteins) into the interior of <A href="https://2010.igem.org/Team:Lethbridge/Project/Compartamentalization"><FONT color="#00DC00"> microcompartments</FONT></A>.  To target the protein into the microcompartment (Lumazine modified to have an even more charged interior) requires the fusion of a polyarginine tail to either the C or N terminus of the protein. 

As with any fusion protien, the addition of this polyarginine tail to the protein has the very real potential of blocking (at least partially) the active site of a protein.  Thus if one can predict the structure of the protein with the addition of the tail, it is possible to avoid creating inactive/ low efficiency fusion proteins.
</P><P><FONT color="white">Figure 1. Homology model of xylE from <I>Pseudomonas putida</I>. The original structure (1MPY) is shown in teal; the generated homology model is shown in green. An Fe<SUP>2+</SUP> ion near the proposed active site is shown in brown.
</FONT></P><H2><SPAN class="mw-headline" id="The_Method"><FONT color="white">The Method</FONT></SPAN></H2><P>To model the xylE structure, the sequence for xylE from <I>Pseudomonas putida</I> (NCBI accession number NP_542866) was aligned with the primary sequence from the crystal structure of xylE from the same organism (pdb: 1MPY; several differences in amino acid sequence were observed) using CLUSTALW (Higgins <I>et al.</I>, 1996). Based on this sequence alignment, a homology model was generated using the alignment mode in SWISSMODEL (Guex <I>et al.</I>, 1997; Kiefer <I>et al.</I>, 2009; Arnold <I>et al.</I>, 2006). To model the placement of an N-terminal arginine tag, the tag was manually added to the N-terminus of the model. Energy minimization was carried out in SWISS-PDB viewer in vacuo utilizing a GROMOS96 energy minimization (Guex <I>et al.</I>, 1997). The resulting pdb file was visualized and manipulated using PYMOL, images were taken using the same software (DeLano, 2006).
</P><P><FONT color="white">Figure 2.  Energy minimized position of the N-terminal 10X Arg tag (shown in blue). 
</FONT></P><H2><SPAN class="mw-headline" id="Results_and_Limitations"><FONT color="white">Results and Limitations</FONT></SPAN></H2><P>Homology modeling of xylE shows that while the C arginine tag cannot interfere with the active site, the N terminal tag is actually a concern.  As shown in figure 2, the arginine tag is located perilously close to one side of the active site  and has a real potential for at least partially blocking substrate access.
</P><P>However, these results are from only using one force field and by holding the structure of the rest of the protein static.  The use of simulated annealing methods or even molecular dynamics would greatly increase our ability to predict the effect of these tags.
</P><H3><SPAN class="mw-headline" id="References"><FONT color="white">References</FONT></SPAN></H3><P>Arnold, K., Bordoli, L., Kopp, J., and Schwede, T. (2006) The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling., Bioinformatics 22, 195-201.

DeLano, W. L. (2006) PyMOL, DeLano Scientific.

Guex, N., and Peitsch, M. C. (1997) SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modelling, <I>Electrophoresis</I> 18, 2714-2723.

Higgins, D. G., Thompson, J. D., and Gibson, T. J. (1996) Using CLUSTAL for multiple sequence alignments, <I>Methods Enzymol.</I> 266, 383-402.

Kiefer, F., Arnold, K., Künzli, M., Bordoli, L., and Schwede, T. (2009) The SWISS-MODEL Repository and associated resources, <I>Nucleic Acids Res</I> 37, D387-D392.
</P></BLOCKQUOTE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Lethbridge/Modeling">http://2010.igem.org/Team:Lethbridge/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/Modeling&amp;oldid=169188" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>