"
Page
Discussion
View source
History
teams
Log in
 
Team:Lethbridge/Modeling
From 2010.igem.org
Contents
1Homology Modeling
1.1The Method
1.2Results and Limitations
1.2.1References
Homology Modeling
One aspect of our project is working on the localization of catechol-2,3,-dioxygenase (and other proteins) into the interior of microcompartments .  To target the protein into the microcompartment (Lumazine modified to have an even more charged interior) requires the fusion of a polyarginine tail to either the C or N terminus of the protein.
As with any fusion protien, the addition of this polyarginine tail to the protein has the very real potential of blocking (at least partially) the active site of a protein.  Thus if one can predict the structure of the protein with the addition of the tail, it is possible to avoid creating inactive/ low efficiency fusion proteins.
Figure 1. Homology model of xylE from Pseudomonas putida. The original structure (1MPY) is shown in teal; the generated homology model is shown in green. An Fe2+ ion near the proposed active site is shown in brown.
The Method
To model the xylE structure, the sequence for xylE from Pseudomonas putida (NCBI accession number NP_542866) was aligned with the primary sequence from the crystal structure of xylE from the same organism (pdb: 1MPY; several differences in amino acid sequence were observed) using CLUSTALW (Higgins et al., 1996). Based on this sequence alignment, a homology model was generated using the alignment mode in SWISSMODEL (Guex et al., 1997; Kiefer et al., 2009; Arnold et al., 2006). To model the placement of an N-terminal arginine tag, the tag was manually added to the N-terminus of the model. Energy minimization was carried out in SWISS-PDB viewer in vacuo utilizing a GROMOS96 energy minimization (Guex et al., 1997). The resulting pdb file was visualized and manipulated using PYMOL, images were taken using the same software (DeLano, 2006).
Figure 2.  Energy minimized position of the N-terminal 10X Arg tag (shown in blue).
Results and Limitations
Homology modeling of xylE shows that while the C arginine tag cannot interfere with the active site, the N terminal tag is actually a concern.  As shown in figure 2, the arginine tag is located perilously close to one side of the active site  and has a real potential for at least partially blocking substrate access.
However, these results are from only using one force field and by holding the structure of the rest of the protein static.  The use of simulated annealing methods or even molecular dynamics would greatly increase our ability to predict the effect of these tags.
References
Arnold, K., Bordoli, L., Kopp, J., and Schwede, T. (2006) The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling., Bioinformatics 22, 195-201.
DeLano, W. L. (2006) PyMOL, DeLano Scientific.
Guex, N., and Peitsch, M. C. (1997) SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modelling, Electrophoresis 18, 2714-2723.
Higgins, D. G., Thompson, J. D., and Gibson, T. J. (1996) Using CLUSTAL for multiple sequence alignments, Methods Enzymol. 266, 383-402.
Kiefer, F., Arnold, K., Künzli, M., Bordoli, L., and Schwede, T. (2009) The SWISS-MODEL Repository and associated resources, Nucleic Acids Res 37, D387-D392.
Retrieved from " http://2010.igem.org/Team:Lethbridge/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
