<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV style="background-color:#000000; color:white"><CENTER><P>Check out these important project links!
</P></CENTER><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Catechol_Degradation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Catechol Degradation</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Characterization_of_Catechol_2.2C3-Dioxygenase"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Characterization of Catechol 2,3-Dioxygenase</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Characterized_Parts"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Characterized Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Hypothesis"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Hypothesis</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Method"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Results_of_the_Bradford_Assay"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Results of the Bradford Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Results"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Conclusion"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#References"><SPAN class="tocnumber">1.1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-10"><A href="#Compartmentalization_Parts"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Compartmentalization Parts</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Placement_of_Oligoarginine_Tail_on_Proteins"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Placement of Oligoarginine Tail on Proteins</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Characterized_Parts_2"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Characterized Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Hypothesis_2"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Hypothesis</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Introduction"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Method_2"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Results_2"><SPAN class="tocnumber">2.1.5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Conclusion_2"><SPAN class="tocnumber">2.1.6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Reference"><SPAN class="tocnumber">2.1.7</SPAN><SPAN class="toctext">Reference</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Characterization_of_C-terminal_Oligo_Arginine_Cyan_Fluorescent_Protein"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Characterization of C-terminal Oligo Arginine Cyan Fluorescent Protein</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Characterized_Parts_3"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Characterized Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Hypothesis_3"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Hypothesis</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Introduction_2"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Method_3"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Results_3"><SPAN class="tocnumber">2.2.5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Conclusion_3"><SPAN class="tocnumber">2.2.6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Future_Directions"><SPAN class="tocnumber">2.2.7</SPAN><SPAN class="toctext">Future Directions</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Reference_2"><SPAN class="tocnumber">2.2.8</SPAN><SPAN class="toctext">Reference</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Magnetic_Nanoparticles_Parts"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Magnetic Nanoparticles Parts</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Characterization_of_Mms6"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Characterization of Mms6</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Characterized_Parts_4"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Characterized Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Hypothesis_4"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Hypothesis</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Introduction_3"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Method_4"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Results_4"><SPAN class="tocnumber">3.1.5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Conclusion_4"><SPAN class="tocnumber">3.1.6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#References_2"><SPAN class="tocnumber">3.1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Catechol_Degradation"><FONT color="white">Catechol Degradation</FONT></SPAN></H1><P>The focus of our <A href="https://2010.igem.org/Team:Lethbridge/Project"><FONT color="#00DC00">project</FONT></A> is to decrease the toxicity of tailings pond water through bioremediation. We are specifically interested in the <A href="https://2010.igem.org/Team:Lethbridge/Project/Catechol_Degradation"><FONT color="#00DC00">degradation of the toxic molecule catechol</FONT></A> into 2-hydroxymuconate semialdyhyde (2-HMS); a bright yellow substrate that can be metabolized by the cell. This conversion is accomplished by a protein called catechol 2,3-dioxygenase (xylE).
</P><H2><SPAN class="mw-headline" id="Characterization_of_Catechol_2.2C3-Dioxygenase"><FONT color="white">Characterization of Catechol 2,3-Dioxygenase</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="Characterized_Parts"><FONT color="white">Characterized Parts</FONT></SPAN></H3><P><A href="http://partsregistry.org/Part:BBa_K118021" target="new"><FONT color="#00DC00" size="+1">BBa_K118021</FONT></A></P><H3><SPAN class="mw-headline" id="Hypothesis"><FONT color="white">Hypothesis</FONT></SPAN></H3><P>The addition of the catechol to a suspension of cells expressing catechol 2,3-dioxygenase will result in the production of 2-HMS.
</P><H3><SPAN class="mw-headline" id="Method"><FONT color="white">Method</FONT></SPAN></H3><P><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K118021" target="new"><FONT color="#00DC00">BBa_K118021</FONT></A> was transformed into <I>Escherichia coli</I> DH5α (E. coli) cells using our transformation protocol. We confirmed that the cells hosted the plasmid containing the BBa_118021 part and we begun experiments to characterize catechol degredation. 

In our first experiment we grew a 5 mL culture of our engineered <I>E. coli</I> cells in M9 minimal media overnight. The cells were spun down at 14000 rfc for 2 minutes. We raised the catechol concentration of this solution to 100 mM, the solution immediately turned bright yellow.
</P><P><FONT color="white">Figure 1. Left: M9 media containing 100 mM catechol. The solution contains <I>E. coli</I> cells hosting the pUC19 plasmid.  Right: M9 media containing 100 mM catechol. The solution contains <I>E. coli</I> cells hosting part BBa_118021. The yellow colour suggests the production of 2-HMS.
</FONT></P><P>In our second experiment we wanted to measure the absorbance of 2-hydroxymuconate semialdehyde over time. We grew a 5 mL culture of our engineered <I>E. coli</I> cells in M9 media overnight.  1 mL of the cell solution was removed for analysis.  Catechol was added to a final concentration of 100mM in the cell suspension. Immediately the formation of 2-HMS was tracked. The formation of 2-HMS can easily be tracked, as it absorbs light at 375 nm.
</P><P><FONT color="white">Figure 2.  Production of 2-HMS over time.
</FONT></P><P>Catechol 2,3-dioxygenase contains an iron molecule in its active site. Our hypothesis was that the iron molecule is oxidized after converting a single catechol molecule to 2-HMS; rendering the catechol 2,3-dioxygenase inactive. We also want to test how catechol 2,3-dioxygenase would behave <I>in vitro</I> compared to <I>in vivo</I>.

To test this hypothesis we grew <I>E. coli</I> containing BBa_K118021 in 500 mL of LB media with ampicillin. <I>E.coli</I> containing the pUC19 plasmid were also grown in 500 mL of LB media with ampicillin to act as a negative control throughout the course of the experiment. The cells were grown to a final optical density of 4.55 AU (measured at 600 nm). The cells were first spun down at 3800 rcf for 5 minutes. The supernatant was decanted and the cells were re-suspended in 40 mL M9 minimal media and incubated with 10 mg of lysozyme for 10 minutes. This cell extract was spun down at 10000 rcf for 30 minutes. The supernatant was taken off and spun at 30000 rcf (S30) for 1 hour. The supernatant of the S30 samples was divided. Half the samples were flash frozen in liquid nitrogen and stored at -80<SUP>o</SUP>C. The second half was spun at 100000 rcf for 45 minutes (S100). 

In order to measure the 2-HMS production we needed to determine the concentration of protein per volume of cell extract. To determine the total protein concentration of the S30 and S100 extracts a Bradford assay was conducted using a standard curve of BSA. Concentrations of the negative controls (<I>Escherichia coli</I> DH5α hosting the pUC19 plasmid) for each of the S30 and S100 samples were also determined. Absorbance readings were taken at a 595 nm wavelength and concentration reported in µg/mL.
</P><H3><SPAN class="mw-headline" id="Results_of_the_Bradford_Assay"><FONT color="white">Results of the Bradford Assay</FONT></SPAN></H3><TABLE><TABLE border="3"><TBODY><TR><TD>Sample</TD><TD>Concentration
</TD></TR><TR><TD>S30</TD><TD>779µg/mL
</TD></TR><TR><TD>S30 Control</TD><TD>104.5µg/mL
</TD></TR><TR><TD>S100</TD><TD>519.3µg/mL
</TD></TR><TR><TD>S100 Control</TD><TD>100.1µg/mL
</TD></TR></TBODY></TABLE><P>
Protein mass of S30 and S100 (and S30 Control/S100 Control) extracts were normalized to between 2µg and 10µg in 1mL of either 20mM Tris pH 8.0 or double distilled water. Catechol was added to normalized cell extract at a final concentration of 0.05 mM. The subsequent production of 2-HMS was observed as a function of time by recording absorbance at 375 nm over the ten minute period immediately following the addition of catechol. 
</P><H3><SPAN class="mw-headline" id="Results"><FONT color="white">Results</FONT></SPAN></H3><P>Figure 3 displays that catechol 2,3-dioxygenase can degrade catechol <I>in vitro</I>, producing 2-HMS. We also observe that when approximately twice the amount of S30 extract is added to an excess catechol solution, production of 2-HMS is increased approximately twofold. This suggests that the enzyme (rather than substrate) is the limiting component in catechol degradation by catechol 2,3-dioxygenase. This result corresponds with our hypothesis that the catechol 2,3-dioxygenase active site iron is reduced upon catalysis, rendering our enzyme inactive. The samples in Figure 3 were carried out in double distilled water. However, duplicates of the readings were measured in a buffer (20mM Tris pH 8.0). We do not observe a significant difference in 2-HMS production in samples containing buffer compared to samples containing double distilled water. 
</P><P><FONT color="white">Figure 3.  The production of 2-hydroxymuconate semialdehyde by different concentrations of S30 extract. Green and red lines are 2µg and 4µg respectively of S30 cell extract from cells expressing catechol 2,3-dioxygenase. The blue line is 4µg of control S30 cell extract. 
</FONT></P></TABLE><H3><SPAN class="mw-headline" id="Conclusion"><FONT color="white">Conclusion</FONT></SPAN></H3><P>We have successfully show that our <I>E. coli</I> cells carrying the K118021 biobrick is capable of converting catechol into 2-HMS. Moreover, the reduction Figure 3 shows a reduction of 2-HMS concentration in the solution. It is more than likely, considering that 2-HMS can be metabolized by the cell, that the soluble metabolic machinery of <I>E. coli</I> is metabolizing 2-HMS into its breakdown products. This is a feature of the system that we intend to exploit in our remediation project. </P><P>With this in mind, it is inefficient to utilize a single turnover enzyme such as catechol 2,3-dioxygenase in an industrial setting. We have identified a protein that will reduce the iron that is oxidized in the conversion of catechol to 2-HMS. The gene coding for this enzyme (<I>xylT</I>) codes for a ferredoxin protein, and is located on the same operon as <I>xylE</I> in <I>Pseudomonas putida</I><SUP>1</SUP>&lt;/i&gt;. </P><P>We intend to introduce ferredoxin to our bioremediation project next year with the aim of increasing catechol 2,3-dioxygenase efficiency by allowing it to catalyze multiple turnovers of catechol to 2-HMS</P><H3><SPAN class="mw-headline" id="References"><FONT color="white">References</FONT></SPAN></H3><P><SUP>1</SUP>Polissi A., Harayama S. (1993) <B>In vivo reactivation of catechol 2,3-dioxygenase mediated by a chloroplast-type ferredoxin: a bacterial strategy to expand the substrate specificity of aromatic degradative pathways.</B>. <I>EMBO J.</I>, <B>12</B>(8); 3339-3347
</P><H1><SPAN class="mw-headline" id="Compartmentalization_Parts"><FONT color="white">Compartmentalization Parts</FONT></SPAN></H1><P>One of the sub-projects for the bioremediation of the tailings ponds is to create synthetic <A href="https://2010.igem.org/Team:Lethbridge/Project/Compartamentalization"><FONT color="#00DC00"> microcompartments</FONT></A> that we can then use to isolate various pathways within an <I>Escherichia coli</I> cell.  To do this we need to have a microcompartment as well as a means to characterize the compartment so that the system can be optimized.  Here are the experiments we have performed so far towards the characterization of the microcomparments.
</P><H2><SPAN class="mw-headline" id="Placement_of_Oligoarginine_Tail_on_Proteins"><FONT color="white">Placement of Oligoarginine Tail on Proteins</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="Characterized_Parts_2"><FONT color="white">Characterized Parts</FONT></SPAN></H3><P><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K249004" target="new"><FONT color="#00DC00" size="+1">BBa_K249004</FONT></A><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K249005" target="new"><FONT color="#00DC00" size="+1">BBa_K249005</FONT></A><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331030" target="new"><FONT color="#00DC00" size="+1">BBa_K331030</FONT></A><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331031" target="new"><FONT color="#00DC00" size="+1">BBa_K331031</FONT></A></P><H3><SPAN class="mw-headline" id="Hypothesis_2"><FONT color="white">Hypothesis</FONT></SPAN></H3><P>The placement of an oligoarginine sequence at the N-terminus of a protein will destabilize the protein <I>in vivo</I>.
</P><H3><SPAN class="mw-headline" id="Introduction"><FONT color="white">Introduction</FONT></SPAN></H3><P>The long term goal of our team is to utilize an oligoarginine tail to specifically target enzymes into a microcompartment composed of modified lumazine synthase subunits. While conducting background research on the project, we came upon data originally reported by Bachmair <I>et al.</I><SUP>1</SUP> suggesting that the identity of the amino acid at the N-terminus of a protein is related to its half-life, and mostly notably, that arginine residues at the are destabilizing. This data suggests that by placing an arginine at the N-terminus of a protein to be targeted into a <A href="https://2010.igem.org/Team:Lethbridge/Project/Compartamentalization"><FONT color="#00DC00"> lumazine synthase microcompartment</FONT></A> would cause degradation of our protein before it can be moved into the microcompartment. 

We chose to investigate the how the placement of an oligoarginine sequence affects the stability of the protein to which it is fused.
</P><H3><SPAN class="mw-headline" id="Method_2"><FONT color="white">Method</FONT></SPAN></H3><P>In order to further characterize the C-terminal and N-terminal oligoarginine tag (BioBricks <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K249005" target="new"><FONT color="#00DC00">BBa_K249005</FONT></A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K249004" target="new"><FONT color="#00DC00">BBa_K249004</FONT></A> respectively) and investigate the effect their placement on protein stability, yellow fluorescent proteins (YFP) with the oligoarginine fused to either the C-terminus (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331023" target="new"><FONT color="#00DC00">BBa_K331023</FONT></A>) or N-terminus (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331022" target="new"><FONT color="#00DC00">BBa_K331022</FONT></A>) (and preceded by a ribosomal binding site – <A href="http://partsregistry.org/wiki/index.php/Part:BBa_B0034" target="new"><FONT color="#00DC00">BBa_B0034</FONT></A>) were synthesized. We used our <A href="https://2010.igem.org/Team:Lethbridge/Notebook/Protocols#Assembly_of_BioBricks_using_the_Red.2FWhite_3-Antibiotic_Assembly_Method"><FONT color="#00DC00"> Red/White 3-Antibiotic assembly method</FONT></A> to add a tetracycline repressible promoter (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_R0010" target="new"><FONT color="#00DC00">BBa_R0040</FONT></A>) for constitutive expression of the fusion protein. This addition generated BioBricks <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331031" target="new"><FONT color="#00DC00">BBa_K331031</FONT></A> and <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331030" target="new"><FONT color="#00DC00">BBa_K331030</FONT></A> for the C-terminal tagged and N-terminal tagged YFP respectively.

The BioBrick containing plasmid was transformed into <I>Escherichia coli</I> DH5α cells. These cells were grown to an OD<SUB>600</SUB> of approximately 0.7, and diluted 1:10 with MilliQ H2O immediately prior to analysis by fluorescent spectroscopy.

This dilution of cells was excited at 517 nm, and the emission spectra was read from 522 nm to 650 nm. Fluorescence at 524 nm (emission maxima of YFP) of control cells (<I>Escherichia coli</I> DH5α), N-terminal tagged, and C-terminal tagged YFP were compared.
</P><H3><SPAN class="mw-headline" id="Results_2"><FONT color="white">Results</FONT></SPAN></H3><P>N-terminal tagged YFP did not have substantially more fluorescence than control cells. Cells expressing C-terminal tagged YFP had ten times more fluorescence than control cells and cells expressing N-terminal tagged YFP.
</P><H3><SPAN class="mw-headline" id="Conclusion_2"><FONT color="white">Conclusion</FONT></SPAN></H3><P>Our results showing that the placement of arginine residues at the N-terminus of our YFP results in no observable fluorescence over control cells are consistent with the general N-terminal rule reported by Bachmair <I>et al.</I> Assuming that transcription of this K331030 and K331031 are equivalent, these data suggest that the N-terminal oligoarginine is reducing the half-life of the protein to which it is fused, ie YFP.
</P><H3><SPAN class="mw-headline" id="Reference"><FONT color="white">Reference</FONT></SPAN></H3><P><SUP>1</SUP>Bachmair A., Finley D., Varshavsky A. (1986), <B>In Vivo Half-Life of a Protein Is a Function of Its Amino-Terminal Residue.</B><I>Science</I> 234. <B>4773</B> 179-186.
</P><H2><SPAN class="mw-headline" id="Characterization_of_C-terminal_Oligo_Arginine_Cyan_Fluorescent_Protein"><FONT color="white">Characterization of C-terminal Oligo Arginine Cyan Fluorescent Protein</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="Characterized_Parts_3"><FONT color="white">Characterized Parts</FONT></SPAN></H3><P><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331033" target="new"><FONT color="#00DC00" size="+1">BBa_K331033</FONT></A></P><H3><SPAN class="mw-headline" id="Hypothesis_3"><FONT color="white">Hypothesis</FONT></SPAN></H3><P>Light will be observed at 476 nm when cells containing <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331033" target="new"><FONT color="#00DC00">BBa_K331033</FONT></A> are illuminated with light at 439 nm as a result of the excitation and subsequent fluorescence of Cyan Fluorescent Protein (CFP).
</P><H3><SPAN class="mw-headline" id="Introduction_2"><FONT color="white">Introduction</FONT></SPAN></H3><P>Prior to introducing a oligoarginine tail onto the <A href="https://2010.igem.org/Team:Lethbridge/Project/Catechol_Degradation"><FONT color="#00DC00">catechol-2,3-dioxygenase</FONT></A> protein with the intention of directing the fusion protein into a lumazine synthase microcompartment, there must be some confirmation that oligoarginine tail is indeed causing the protein to which it is attached to localize within the microcompartment. We chose to utilize a technique known as fluorescent resonance energy transfer<SUP>1</SUP> (FRET) to verify localization. In FRET, when a dye pair comes into close proximity to one another, one dye (the donor) will transfer some of its energy to the second dye (the acceptor). This will cause the acceptor to fluoresce without direct excitation.  The dye pair we have chose is CFP as the donor and yellow fluorescent protein (YFP) as the acceptor. 

Here we investigate the ability of  <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331033" target="new"><FONT color="#00DC00">BBa_K331033</FONT></A> to produce CFP and emit light at its characteristic emission wavelength.
</P><H3><SPAN class="mw-headline" id="Method_3"><FONT color="white">Method</FONT></SPAN></H3><P>In order to characterize the tetracycline repressible CFP (BioBrick  <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331033" target="new"><FONT color="#00DC00">BBa_K331033</FONT></A>), cyan fluorescent protein (CFP-<A href="http://partsregistry.org/wiki/index.php/Part:BBa_E0020" target="new"><FONT color="#00DC00">BBa_E0020</FONT></A>) with an oligoarginine tag fused to the C-terminus (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331025" target="new"><FONT color="#00DC00">BBa_K331025</FONT></A>) (and preceded by a ribosomal binding site – <A href="http://partsregistry.org/wiki/index.php/Part:BBa_B0034" target="new"><FONT color="#00DC00">BBa_B0034</FONT></A>) was synthesized. We used our Red/White 3-Antibiotic assembly method to add a tetracycline repressible promoter (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_”R0040" target="new"><FONT color="#00DC00">BBa_R0040</FONT></A>) for constitutive expression of the fusion protein. This construction yielded BioBrick  <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331033" target="new"><FONT color="#00DC00">BBa_K331033</FONT></A>.

The BioBrick containing plasmid was transformed into <I>Escherichia coli</I> DH5α cells. These cells were grown to an OD<SUB>600</SUB> of approximately 0.7 in LB media, and diluted 1:10 with MilliQ H<SUB>2</SUB>O immediately prior to analysis by fluorescent spectroscopy.

This dilution of cells was excited at 439 nm, and the emission spectra was read from 444 nm to 650 nm. 
</P><H3><SPAN class="mw-headline" id="Results_3"><FONT color="white">Results</FONT></SPAN></H3><P>Fluorescence at 476 nm was observed. This peak, along with a shoulder occurring at approximately 510 nm is consistent with results obtained by McRae<SUP>1</SUP><I>et al.</I> in their rapid purification of ECFP.</P><H3><SPAN class="mw-headline" id="Conclusion_3"><FONT color="white">Conclusion</FONT></SPAN></H3><P>The BioBrick  <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K331033" target="new"><FONT color="#00DC00">BBa_K331033</FONT></A> that we constructed generates CFP in the absence of tetracycline, as expected.
</P><H3><SPAN class="mw-headline" id="Future_Directions"><FONT color="white">Future Directions</FONT></SPAN></H3><P>We will assembly this into a single biobrick which can differentially express CFP/YFP and lumazine synthase for confirmation (through FRET) that the oligoarginine tail can be utilized to localize proteins within a modified lumazine synthase microcompartment.
</P><H3><SPAN class="mw-headline" id="Reference_2"><FONT color="white">Reference</FONT></SPAN></H3><P><SUP>1</SUP>Förster T. (1948), Zwischenmolekulare Energiewanderung und Fluoreszenz.  <I>Annalen der Physik</I>, 437: 55-75.
<SUP>2</SUP>McRae S.R., Brown C.L., Bushell G.R. (2005), Rapid Purification of EGFP, EYFP, and ECFP with high yield and purity. <I>Protein Expression and Purification</I> 41. 1 121-127.
</P><H1><SPAN class="mw-headline" id="Magnetic_Nanoparticles_Parts"><FONT color="white">Magnetic Nanoparticles Parts</FONT></SPAN></H1><P>One of the sub-projects for the <A href="https://2010.igem.org/Team:Lethbridge/Project" target="new"><FONT color="#00DC00"> bioremediation of the tailings ponds</FONT></A> is to reduce heavy metals to create <A href="https://2010.igem.org/Team:Lethbridge/Project/Magnetic_Nanoparticles" target="new"><FONT color="#00DC00"> magnetic nanoparticles</FONT></A> that can then be removed from the pond.  To do this we need to be able to produce Mms6 (the iron reducing protein) and show that it can successfully produce the nanoparticles within the <I>Escherichia coli</I> cell.  He is the work we have accomplished so far to characterize Mms6.
</P><H2><SPAN class="mw-headline" id="Characterization_of_Mms6"><FONT color="white">Characterization of Mms6</FONT></SPAN></H2><H3><SPAN class="mw-headline" id="Characterized_Parts_4"><FONT color="white">Characterized Parts</FONT></SPAN></H3><P><A href="http://partsregistry.org/wiki/index.php/Part:BBa_K249019" target="new"><FONT color="#00DC00" size="+1">BBa_K249019</FONT></A></P><H3><SPAN class="mw-headline" id="Hypothesis_4"><FONT color="white">Hypothesis</FONT></SPAN></H3><P>Induction of Mms6 protein overexpression in <I>Escherichia coli</I> DH5α cells will cause a change in growth. 
</P><H3><SPAN class="mw-headline" id="Introduction_3"><FONT color="white">Introduction</FONT></SPAN></H3><P>A parallel project we are undertaking in <A href="https://2010.igem.org/Team:Lethbridge/Project"><FONT color="#00DC00">oilsands remediation</FONT></A> is in the clean-up of heavy metal contaminants utilizing the Mms6 protein expressed by <I>Magnetiospirilum magneticum</I><SUP>1</SUP>. The Mms6 protein is capable of reducing aqueous iron into <A href="https://2010.igem.org/Team:Lethbridge/Project/Magnetic_Nanoparticles"><FONT color="#00DC00">magnetite nanoparticles</FONT></A>. The magnetite nanoparticles can be removed with much greater ease than aqueous iron. 

We chose to investigate how the presence of the Mms6 protein affected <I>Escherichia coli</I> DH5α, its bacterial host cell. 
</P><H3><SPAN class="mw-headline" id="Method_4"><FONT color="white">Method</FONT></SPAN></H3><P>In order to characterize the effect of Mms6 production on <I>Escherichia coli</I> DH5α cells, we utilized BioBrick <A href=" http://partsregistry.org/Part:BBa_K249019" target="new"><FONT color="#00DC00">BBa_K249019</FONT></A>, a part submitted by the <A href="https://2009.igem.org/Team:Lethbridge" target="new"><FONT color="#00DC00">2009 Lethbridge iGEM</FONT></A> team. <A href="http://partsregistry.org/Part: BBa_K249019" target="new"><FONT color="#00DC00">BBa_K249019</FONT></A> is the Mms6 coding region (<A href="http://partsregistry.org/Part: BBa_K249016" target="new"><FONT color="#00DC00">BBa_K249016</FONT></A>) with a ribosomal binding side (<A href="http://partsregistry.org/Part: BBa_B0030" target="new"><FONT color="#00DC00">BBa_B0030</FONT></A>) and a double terminator (<A href="http://partsregistry.org/Part: BBa_B0015" target="new"><FONT color="#00DC00">BBa_B0015</FONT></A>) under the control of a lactose inducible promoter (<A href="BBa_R0010" target="new"><FONT color="#00DC00"> BBa_R0010</FONT></A>). 

We inoculated two 500 mL cultures of LB media with <I>Escherichia coli</I> DH5α cells containing <A href="http://partsregistry.org/Part: BBa_K249019" target="new"><FONT color="#00DC00">BBa_K249019</FONT></A> and grew the cells until they reached approximately 0.6 OD<SUB>600</SUB> units. At this point, expression of the Mms6 protein was induced in one of the 500 mL cultures by the addition of 1 mM Isopropyl β-D-1-thiogalactopyranoside (IPTG). One milliliter of cells from each sample was removed at one hour intervals and the optical density at 600 nm was recorded and plotted.
</P><H3><SPAN class="mw-headline" id="Results_4"><FONT color="white">Results</FONT></SPAN></H3><P>The addition of IPTG to cells containing the <A href="http://partsregistry.org/Part: BBa_K249019" target="new"><FONT color="#00DC00">BBa_K249019</FONT></A> BioBrick caused a slowdown of cell growth after 2 hours, and a subsequent plateau at approximately one-half of the cells density that uninduced cells reached after three hours post-induction.
</P><H3><SPAN class="mw-headline" id="Conclusion_4"><FONT color="white">Conclusion</FONT></SPAN></H3><P>The slowdown in cell growth as a result of production of Mms6 protein could be attributed to a number of factors. 

One potential explanation could be a significant diversion of cellular resources to the production of Mms6 protein. This is unlikely, as subsequent analysis of cell fractions with SDS-PAGE showed no increase in banding at the expected molecular weight. 

Another explanation could be that the Mms6 protein is toxic to the cells, and causes them to die. This possibility is being explored in collaboration with the <A href=" https://2010.igem.org/Team:Calgary" target="new"><FONT color="#00DC00">Calgary team</FONT></A> by using their troubleshooting kit.
</P><H3><SPAN class="mw-headline" id="References_2"><FONT color="white">References</FONT></SPAN></H3><P><SUP>1</SUP>Arakaki A., Masuda F., Amemiya Y., Tanaka T., Matsunaga T. (2010). Control of the morphology and size of magnetite particles with peptides mimicking the Mms6 protein from magnetotactic bacteria. <I>Journal of Colloid and Interface Science</I>. 343:65-60.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Lethbridge/Results">http://2010.igem.org/Team:Lethbridge/Results</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;oldid=204666" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BLOCKQUOTE></DIV></DIV></DIV></DIV></BODY></HTML>