"
Page
Discussion
View source
History
teams
Log in
 
Team:Lethbridge/Notebook/Lab Work/September
From 2010.igem.org
Feel free to look around our notebook!
Here you can check out the work we have done in the lab, click on a month to take a look!
Contents
1September
1.1September 2, 2010
1.1.1ADS
1.2September 14, 2010
1.2.1JF
1.2.2JV
1.2.3KG
1.2.4AS
1.3September 15, 2010
1.3.1JV, ADS
1.4September 16, 2010
1.4.1JV
1.5September 17, 2010
1.5.1JV
1.5.2AV
1.5.3KG
1.6September 18, 2010
1.6.1JS, DM
1.6.2TF
1.7September 20, 2010
1.7.1ADS
1.7.2HB
1.8September 21, 2010
1.8.1ADS
1.8.2TF, MC
1.8.3AV
1.9September 21, 2010
1.9.1ADS
1.10September 23, 2010
1.10.1JV
1.10.2ADS
1.10.3TF, MC
1.11September 24, 2010
1.11.1ADS
1.12September 25, 2010
1.12.1JV
1.12.2ADS
1.13September 26, 2010
1.13.1ADS
1.14September 29, 2010
1.14.1ADS
1.14.2AV
September
September 2, 2010
ADS
Objective: Assemble lumazine-dT into pSB1C3 using NEB BioBrick Assembly Kit.
Method: Have the following parts: dT [28 ng/(µL)]; lumazine [78 ng/(µL)]; psB1C3 [14 ng/(µL)].
Require 500 ng DNA each part in a 50(µL) rxn therefore 17.9(µL) dT and 6.4(µL) Lumazine.  Need 250 ng pSB1C3 or 17.9(µL)
Restriction Reactions -
Ingredient
Volume(µL)
MilliQ H20 Water
36.1
NEBuffer 2 (10x)
5
Upstream Part (Lumazine)
26.4
EcoRI-HF
1
SpeI
1
100X BSA
0.5
Ingredient
Volume(µL)
MilliQ H20 Water
24.6
NEBuffer 2 (10x)
5
Downstream Part (dT)
17.9
XbaI
1
PstI
1
100X BSA
0.5
Ingredient
Volume(µL)
MilliQ H20 Water
3.35
NEBuffer 2 (10x)
2.5
Destination Plasmid (pSB1C3)
17.9
EcoRI-HF
.5
PstI
.5
100X BSA
0.25
2(µL) was removed prior to adding enzymes for analysis on gel.
Split digestion reactions in half: dT - 2 @ 24(µL); Lum - 2 @ 24(µL); psB1C3 - 2 @ 11.5(µL)
Ran 10 minute and 60 minute reactions at 37oC.  Followed by 20 minute heat shock at 80oC.
2(µL) was removed from the heat killed reactions for analysis on gel.
Performed 4 combinations of ligation: 10 min Restriction + 10 min Ligation; 10 min Restriction + Overnight Ligation; 60 min Restriction + 10 min Ligation; 60 min Restriction + Overnight Ligation
Ingredient
1X(µL)
2X Master Mix(x2.5)(µL)
MilliQ H20 Water
11
77.5
T4 Ligase Buffer (10x)
2
5
Upstream Part
2
5
Downstream Part
2
5
T4 DNA Ligase
1
2.5
Plasmid Part
2
5
Incubated 10 minute and overnight ligations at room temperature ( 25oC).  Heat killed ligase at 80oC for 20 min.
2(µL) of each reaction was taken for analysis on gel.
Confirmed ligation with PCR analysis.  Analyzed restriction, ligation and PCR on a 1.5% TAE agarose gel.
Component
1X(µL)
Master Mix(x5.5)(µL)
Milli-Q H2O
36.5
202.4
10x Pfu Buffer with MgSO4
5
27.5
dNTPs
2
11
Forward VF2 Primer
2
11
Reverse VR Primer
2
11
Template DNA
2
Pfu polymerase
0.2
1.1
Added 48(µL) to each reaction.
Objective: Characterize xylE in LB Media vs. M9 Minimal Media.
Method: Three experiments with spectra.
Experiment 1 - Measure absorption spectra of 1:10 dilution of:  1.  Cells suspended in m9 media +/- catechol.  2.  Cells suspended in LB media +/- catechol.  3.  m9 media from spun down cells +/- catechol.  4.  LB media from spun down cells +/- catechol.  5.  Sterile LB media +/- catechol.  6.  Water +/- catechol.
Experiment 2 - Catechol Breakdown.  Follow absorbance of 2-hydroxymucanate semialdehyde at 375 nm.  Measured all the above samples for 10 min at that wavelength.
September 14, 2010
JF
Objective: Miniprep of RFP expression construct (J04450) via Qiaquick/Qiaprep spin column.  Protocol followed as per kit instructions.
JV
Objective: PCR the pSB1C3 plasmid from part J04450 miniprep.
Component
1X(µL)
Milli-Q H2O
12.8
10x Pfu Buffer with MgSO4
2
dNTPs
1
SB-prep-26 Primer
1
SB-prep-3P Primer
1
Template DNA
2
Pfu polymerase
0.2
Used iGEM cycle PSB1C3 in thermocycler. Ran PCR product on 1% TAE agarose gel.
KG
Objective: PCR of xylE (mRBS-xylE K118021) with fusion standard so that an arginine tail can be attached.
Component
1X(µL)
Master Mix(x6)(µL)
Milli-Q H2O
12.8
76.8
10x Pfu Buffer with MgSO4
2
12
dNTPs
1
6
xylE Fusion Prefix Forward Primer
1
6
xylE Fusion Suffix Reverse Primer
1
6
Template DNA
2
12
Pfu polymerase
0.2
1.2
PCR- Conditions: 1.  95oC for 180 sec 2.  95oC for 30 sec 3.  56.6 +/- 5oC for 30 sec 4.  72oC for 150 sec 5.  72oC for 600 sec (35 cycles)
Analyzed PCR on 2% agarose gel which ran at 100 V for 90 minutes.
AS
Objective: Assemble xylE-dT using 3AB assembly and 3AB assembly.
Restriction Reactions -
3AB Upstream Part (xylE)
Ingredient
Volume(µL)
MilliQ H20 Water
30.6
NEBuffer 2 (10x)
5
Plasmid DNA
11.9
EcoRI-HF
1
SpeI
1
100X BSA
0.5
3AB Downstream Part (dT)
Ingredient
Volume(µL)
MilliQ H20 Water
24.6
NEBuffer 2 (10x)
5
Plasmid DNA
17.9
XbaI
1
PstI
1
100X BSA
0.5
3AB Plasmid (pSB1C3)
Ingredient
Volume(µL)
MilliQ H20 Water
NEBuffer 2 (10x)
5
Plasmid DNA
42.5
EcoRI-HF
1
PstI
1
100X BSA
0.5
Ran 10 minute and 60 minute reactions at 37oC.  Followed by 20 minute heat shock at 80oC.
Ligation Reaction -
Ingredient
1X(µL)
MilliQ H20 Water
11
T4 Ligase Buffer (10x)
2
Upstream Part
2
Downstream Part
2
T4 DNA Ligase
1
Plasmid Part
2
Incubated 10 minute and overnight ligations at room temperature ( 25oC).  Heat killed ligase at 80oC for 20 min.
Transformation -
A) Thawed 200(µL) Library Efficiency DH5alpha Competent Cells on ice. B)  Gently mixed cells and then aliquoted 100(µL) into chilled polypropylene tubes. C)  Added 1(µL) of ligation mix to cells. Added 5(µL) of pUC19 DNA to 100(µL) cells to determine efficiency.   D)  Incubated cells on ice for 30 minutes. E)  Heat shocked cells for 45 seconds in a 42oC water bath. F)  Placed on ice for 2 minutes. G)  Added 0.9 mL of room temperature SOC medium. H)  Shook at 225 rpm for 1 hour. I)    Diluted control cells 1:100 with SOC medium. J)   Spread 100(µL) of this dilution on LB-Amp agar plates K) Spread 50 and 250(µL) of experimental cells on LB-Cam agar plates.   L)  Incubated overnight at 37oC
September 15, 2010
JV, ADS
Objective: Determine if dT ligated onto p-rbs-xylE using PCR analysis.
Component
1X(µL)
Master Mix(x4.5)(µL)
Milli-Q H2O
11.8
53.1
10x Pfu Buffer with MgSO4
2
9
dNTPs
1
4.5
Forward VF2 Primer
1
4.5
Reverse VR Primer
1
4.5
Template DNA
1
Pfu polymerase
0.2
Used iGEM program 7 in thermocycler.
Objective: Determine if minipreps from Sept 15, 2010 worked.  Analyze assembly intermediates.
Method: Run 1% TAE agarose gels at 100 V for 90 minutes.
September 16, 2010
JV
Objective: Isolate RFP in pSB1C3 from DH5alpha cells.
Method: Used Qiagen minipep protocol and spin column.
Results: Gel of miniprepped RFP against previously prepared RFP showed a band of the right size.
September 17, 2010
JV
Objective: Isolate plasmid DNA from DH5alpha cells:  K249024; B0015; K118021/B0015
Method: Used Qiagen minipep protocol and spin column. Ran a 1% agarose gel to view DNA
GEL PICTURE!
AV
Objective: PCR out RFP from the complete construct (J04450) with a N-term fusion standard.
Composition of each PCR tube:
Component
Volume(µL)
10x Pfu Buffer w/ MgSO4
2
dNTP (10mM)
1
N-term Fus Prefix
1
Biobrick Suffix
1
MilliQ H2O
12.8
Template DNA (J04450)
2
Pfu DNA Polymerase
0.2
PCR conditions:
Step
Temperature (oC)
Time (mins)
Number of cycles
Initial Denaturation
95
2
1
Denaturation
95
0.5
25
Annealing
48.2, 52.4, 56.0 (gradient)
0.5
25
Extension
72
2
25
Final Extension
72
10
1
Results: Amplification showing in gel, but fragment amplified was ~1000bp. RFP and dT should be ~800bp. Primers were checked and discovered the YFP/CFP primers were not compatible with the RFP. Concluded the amplification resulted from sloppy annealing of N-term fus prefix to the bio prefix resulting in the full construct being amplified (~1000bp)
KG
Objective: PCR xylE to attach fusion standard to the N-term and standard suffix to C-term.
Component
1X(µL)
Master Mix(x6)(µL)
Milli-Q H2O
12.8
76.8
10x Pfu Buffer with MgSO4
2
12
dNTPs
1
6
Fusion Forward Primer
1
6
Standard Reverse Primer
1
6
Template DNA
2
12
Pfu polymerase
0.2
1.2
Fusion standard prefix has melting temperature of 66.5oC.  Standard suffix has melting temperature of 70.5oC.   Annealing temperature of 61.5oC with 5 oC gradient.   PCR- Conditions: 1.  95oC for 180 sec 2.  95oC for 30 sec 3.  61.5 +/- 5oC for 30 sec 4.  72oC for 150 sec 5.  72oC for 600 sec (35 cycles)
Objective: PCR xylE to attach fusion standard to the C-term and standard suffix to N-term.
Component
1X(µL)
Master Mix(x6)(µL)
Milli-Q H2O
12.8
76.8
10x Pfu Buffer with MgSO4
2
12
dNTPs
1
6
Standard Forward Primer
1
6
Fusion Reverse Primer
1
6
Template DNA
2
12
Pfu polymerase
0.2
1.2
Standard prefix has melting temperature of 68.7oC.  Standard suffix has melting temperature of 61.6oC.   Annealing temperature of 56.6oC with 5 oC gradient.
PCR- Conditions: 1.  95oC for 180 sec 2.  95oC for 30 sec 3.  56.6 +/- 5oC for 30 sec 4.  72oC for 150 sec 5.  72oC for 600 sec (35 cycles)
September 18, 2010
JS, DM
Objective: Overexpress mms6.
Method: Followed overexpression protocol.  One flask was induced with 250(µL) IPTG while another was induced with 50(µL).
Time (minutes)
OD #1 (600λ)
OD #2 (600λ)
30
0.139
0.133
60
0.200
0.188
90
0.372
0.360
120
0.702
0.704
150(T0)
0.871
0.810
210(T1)
2.600
2.500
270(T2)
3.317
3.300
330(T3)
4.408
3.946
Plated cells in presence of IPTG, Fe or IPTG and Fe on AMP(+) plates.  Incubated at 37 oC.
Preparation of these samples was achieved by taking a 60(µL) cell sample and inoculating 5 mL LB media.  These solutions were incubated at 37 oC for 30 min.  To 1 mL of culture, the appropriate amount of Fe2+ [0.938(µL)], Fe3+[1.88(µL)] or IPTG [1(µL)] was added.
TF
Objective: Obtain preparations of B0015 (dT) and J04450 (RFP).
Method: Used Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep protocol.
Results: Chloroform, DNA and isopropanol were mixed and therefore no products could be recovered.
September 20, 2010
ADS
Objective: Analyze PCR of mms6 (HB) and fluorescent proteins (AV).
Method: Analyzed on 2% TAE agarose gel.
Load order:
Lane
Sample
Volume
Sample (µL)
Volume Loading
Dye (µL)
1
HB1
10
2
2
HB2
10
2
3
HB3
10
2
4
HB4
10
2
5
HB5
10
2
6
100bp Ladder (Fermentas)
0.5
2(+10 H20)
7
RFP1
10
2
8
RFP2
10
2
9
RFP3
10
2
Results:
mms6 was not amplified
RFP was amplified
Expected size is ~800bp
Actual size is >1000bp
We believe that the suffix segment of the fusion primer annealed rather than the FP segment. This would cause the promoter (R0040) of J04450 to be amplified, adding ~200bp, giving a final size of >1000bp.
Objective: Confirm assembly (3 antibiotic) of K118021 and B0015 from Sept. 14, 2010 via PCR.
Method: Set up 50uL reaction mixture with 2uL of ligated DNA. Used VF2 and VR primers.
Used PFU setting on Thermocycler
Results:
Conclusion: Assembly did not work as intended.
HB
Objective: PCR mms6 to attach fusion standard to the N-term and standard suffix to C-term.
Component
1X(µL)
Master Mix(x6)(µL)
Milli-Q H2O
12.8
76.8
10x Pfu Buffer with MgSO4
2
12
dNTPs
1
6
Fusion Forward Primer
1
6
Standard Reverse Primer
1
6
Template DNA
2
12
Pfu polymerase
0.2
1.2
Prefix fusion sense primer has melting temperature of 56.1oC.  Standard suffix primer has melting temperature of 61.3oC.
PCR- Conditions: 1.  95oC for 180 sec 2.  95oC for 30 sec 3.  51.1oC for 30 sec 4.  72oC for 150 sec 5.  72oC for 600 sec (35 cycles)
Program HBPREFU: 1.  49.1oC  2.  50.2oC  3.  51.4oC  4.  52.6oC  5.  53.5oC
September 21, 2010
ADS
Objective: Insert xylE (with N and C terminal fusion standards, obtained by PCR of K118021 by KG) into pSB1C3 plasmid for submission to registry.
Method:
Restriction of xylE PCR product and pSB1C3 (containing J04450 biobrick) via BioBrick Method using EcoRI-HF and PstI (Enzymes from NEB)
Ligation of xylE PCR product and pSB1C3 via BioBrick Method using T4 DNA ligase (Enzyme from NEB)
Transform into Library Efficiency Compentent DH5α Cells (Invitrogen)
Results:
Obtained too numerous to count (TNTC) colonies.
Obtained ~100 white colonies (indicating removal of RFP and insertion of new BioBrick)
Note:
Parent plasmid (from PCR) not digested, possibly K118021 moved into pSB1C3 backbone.
Follow-up:
Screen white colonies by addition of catechol to solution containing white cells.
TF, MC
Objective: Obtain a preparation of B0015 (dT).
Method: Used Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep protocol.
AV
Objective: PCR amplify the signal sequences synthesized by Mr. Gene (K331007, K331008, K331009, K331012).
Composition of each PCR tube:
Component
Volume(µL)
10x Pfu Buffer w/ MgSO4
2
dNTP (10mM)
1
VF2
1
VR
1
MilliQ H2O
12.8
Template DNA
2
Pfu DNA Polymerase
0.2
Used iGEM thermocycler setting: PFU.
September 21, 2010
ADS
Objective: Analyze PCR of K118021-B0015 and subsequent PCR (ADS) and PCR of <partinfo>K118021</partinfo> to add either N or C terminal fusion standard (KG).
Method: Analyze on 2% TAE agarose gel.
Load Order:
Lane
Sample
Volume
Sample (µL)
Volume Loading
Dye (µL)
1
xylE F-S 1
5
1
2
xylE F-S 2
5
1
3
xylE F-S 3
5
1
4
xylE F-S 4
5
1
5
xylE F-S 5
5
1
6
xylE F-S 6
5
1
7
100bp Ladder (NEB)
0.5
1(+5 H20)
8
xylE S-F 1
5
1
9
xylE S-F 2
5
1
10
xylE S-F 3
5
1
11
xylE S-F 4
5
1
12
xylE S-F 5
5
1
13
xylE S-F 6
5
1
14
Empty
15
PCR of K118021-B0015 (ADS)
1
1
16
Empty
17
Empty
Results:
Both KG PCR (Standard-Fusion and Fusion-Standard) amplified
ADS PCR Amplified, but no insert present.
September 23, 2010
JV
Objective: Characterized catechol degradation by xylE enzyme
Method: Measured absorbance of catechol (275nm) and 2-hydroxymuconate semialdehyde (380nm).
Protocol:
1) Grow cells in M9 minimal medium
2) Take 1/10 dilution of cells
3) Introduce 1µL of 0.05M catechol solution into the cell dilution. (Final concentration of 50&microM;).
4) Quench the reaction with 5A% w/v trichloroacetate at certain time points. (0,15sec, 30sec, 45sec, 60sec, 2min, 3min, 4min, 5min, 10min).
5) Spin down cells.
6) Measure absorbance of supernatant.
Results: Cuvette used interfered with Spectra.
ADS
NOTE: In all transformations, heat shock step was missed. HOWEVER, all transformations showed significant number of colony forming units.
Objective: Move xylE (two biobrick; one with Fusion prefix, one with fusion suffix) into pSB1C3.
Method:
Restriction of xylE PCR product and pSB1C3 (containing J04450 biobrick) via BioBrick Method using EcoRI-HF and PstI (Enzymes from NEB)
Ligation of xylE PCR product and pSB1C3 via BioBrick Method using T4 DNA ligase (Enzyme from NEB)
Incubated 30 min at RT
Transform into Subcloning Efficiency Compentent DH5α Cells (Invitrogen)
Results: TBD
Follow-up: TBD
Objective: Create glycerol stocks of <partinfo>J04450</partinfo> in pSB1A3 and pSB1T3 for use in RFP-BioBrick Assembly.
Method: Transform into Subcloning Efficiency Competent DH5α Cells (Invitrogen)
Obtained all plasmid DNA from 2010 Kit Plate 1
J04450 in pSB1A3 - Well 1C
J04450 in pSB1T3 - Well 7A
Results: Obtained TNTC colonies
Follow-up:
Grow overnight cultures
Generate Glycerol Stocks
Generate Plasmid DNA via Maxiprep
Objective: Create glycerol stocks of received synthesized (Mr. Gene) signal peptides.
Method: Transform into Subcloning Efficiency Competent DH5α Cells (Invitrogen) plasmid DNA containing the following BioBricks:
1) <partinfo>K331007</partinfo> - β-lactamase Bla Signal Sequence
2) <partinfo>K331008</partinfo> - Outer Membrane Protein ompA
3) <partinfo>K331009</partinfo> - Heat Stable Toxin I
4) <partinfo>K331012</partinfo> - Penicillin Binding Protein DacA
All inserts in pMA-T vector (Standard Mr. Gene vector)
Results: Obtained TNTC Cells
Follow-up:
1) Grow overnight cultures
2) Purify pDNA
3) Move into pSB1C3 plasmid
4) Verify sequence
5) Submit to registry for sequencing
TF, MC
Objective: Obtain a preparation of J04450 (RFP).
ethod: Used Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep protocol.
September 24, 2010
ADS
Objective: Generate plasmid DNA of <partinfo>E1010</partinfo> for downstream PCR
Method: Transform plasmid DNA into Subcloning Efficiency Competent DH5α Cells (Invitrogen)
DNA obtained from 2010 Kit Plate 1 Well 18F (E1010 in pSB2K3)
Results: Obtained TBD colonies
Follow-up:
Grow overnight cultures (Generate glycerol stocks)
Purify plasmid DNA (Generate pDNA stocks)
PCR to add terminal fusion standards
Objective: Create glycerol stocks of J04450 in pSB1K3 for use in RFP-BioBrick Assembly.
Method: Transform into Subcloning Efficiency Competent DH5α Cells (Invitrogen)
Obtained plasmid DNA from 2010 Kit Plate 1 well 5A (J04450 in pSB1K3)
Results: Obtained TNTC colonies
Follow-up:
Grow overnight cultures
Generate Glycerol Stocks
Generate Plasmid DNA via Maxiprep
September 25, 2010
JV
Objective: Extract Plasmid DNA from DH5α cells.
Method:Qiagen spin column protocol.
<partinfo>K331007</partinfo> (in pMA-T vector)
<partinfo>K331008</partinfo> (in pMA-T vector)
<partinfo>K331009</partinfo> (in pMA-T vector)
<partinfo>K331012</partinfo> (in pMA-T vector)
Cells containing plasmids were put into glycerol stocks and put into HJ's -80oC.
ADS
Objective: Move plasmid DNA made by JV (above, Sept 25, 2010) into pSB1C3 for submission to the Standard Registry of Parts.
Method:
1) Digest at 37o for 10 min with PstI and EcoRI:
pSB1C3 containing <partinfo>J04450</partinfo>
pMA-T with <partinfo>K331007</partinfo>, <partinfo>K331008</partinfo>, <partinfo>K331009</partinfo>, and <partinfo>K331012</partinfo>.
2) Heat Kill at 80oC for 20 min
3) Mix 2µL of pSB1C3 with each biobrick, ligate with T4 DNA ligase for 10 min
4) Transform into subcloning efficiency DH5α cells, plate on ampicillin plates.
Results: Obtained RESULT!!! positive colonies
Objective: Assemble KG's C-terminal fusion xylE (<partinfo>K331021</partinfo>) (UNCONFIRMED) and C-terminal Arginine + dT (<partinfo>S04261</partinfo>) to create <partinfo>K331011</partinfo>.
Method:
1) Digest at 37o for 10 min:
pSB1C3 containing <partinfo>J04450</partinfo> with PstI and EcoRI
<partinfo>K331021</partinfo> with EcoRI, SpeI
<partinfo>S04261</partinfo> with XbaI and PstI
2) Heat Kill at 80oC for 20 min
3) Mix 2µL of pSB1C3 with each biobrick, ligate with T4 DNA ligase for 10 min
4) Transform into subcloning efficiency DH5α cells, plate on ampicillin plates.
Results: Obtained RESULT!!! positive colonies
September 26, 2010
ADS
Objective: Purify plasmid DNA from 5mL cultures grown from transformations of KG's ligated PCRs containing (UNCONFIRMED) <partinfo>K331020</partinfo> and <partinfo>K331021</partinfo>.
Method: Used Qiagen spin column method with BioBasic EZ-10 spin columns.
Objective: Assemble the following:
<partinfo>K331020</partinfo> + <partinfo>B0015</partinfo> to obtain To be named biobrick
<partinfo>K331021</partinfo> + <partinfo>S04261</partinfo> to obtain To be named biobrick
Method:
Digest <partinfo>K331020</partinfo> and <partinfo>K331021</partinfo> with EcoRI and SpeI
Digest <partinfo>B0015</partinfo> and <partinfo>S04261</partinfo> with XbaI and PstI
Digest <partinfo>J04450</partinfo> in pSB1C3 with EcoRI and PstI
Incubate each at 37oC for 10 min
Heat kill at 80oC for 20 min
Ligate with T4 Ligase for 10 min at room temperature
Transform into DH5&alpha subcloning efficiency cells
Results: Obtained RESULTS!!!! positive colonies.
Also analyzed minipreps on 1.5% TAE Agarose Gel
September 29, 2010
ADS
Received synthesized parts from Mr. GENE (all in pMA vectors)
<partinfo>K331022</partinfo>
<partinfo>K331023</partinfo>
<partinfo>K331024</partinfo>
<partinfo>K331025</partinfo>
Each tube contained 5µg pDNA; reconstitute in 50µL of TE buffer to get 100ng/µL concentration
Transform 1µL into DH5α subcloning efficiency cells
Obtained TNTC colonies
Started overnight 5mL cultures for miniprep and insertion into pSB1C3 vector.
AV
Objective: PCR the N-term fusion tag onto RFP (E1010) using sloppy annealing of the YFP/CFP fusion primers.
Composition of each PCR tube:
Component
Volume(µL)
10x Pfu Buffer w/ MgSO4
2
dNTP (10mM)
1
N-term Fus Prefix
1
Biobrick Suffix
1
MilliQ H2O
12.8
Template DNA (J04450)
2
Pfu DNA Polymerase
0.2
PCR conditions:
Step
Temperature (oC)
Time (mins)
Number of cycles
Initial Denaturation
95
2
1
Denaturation
95
0.5
25
Annealing
47.6, 50.5, 53.8 (gradient)
0.5
25
Extension
72
2
25
Final Extension
72
10
1
Results: Gel has not been run. Have decided to order RFP specific primers for next year.
Retrieved from " http://2010.igem.org/Team:Lethbridge/Notebook/Lab_Work/September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
