"
Page
Discussion
View source
History
teams
Log in
 
Team:Lethbridge/Notebook/Protocols
From 2010.igem.org
Feel free to look around our notebook!
Contents
1Common Protocols:
1.1Assembly of BioBricks using the Red/White 3-Antibiotic Assembly Method
1.1.1Step 1 - Restriction
1.1.2Step 2 - Ligation
1.1.3Step 3 - Transformation
1.1.4Step 4 - Selection of Colonies and Colony PCR
1.1.5Step 5 - Plasmid DNA Purification
1.2Protocols prior to the development of Red/White 3-Antibiotic assembly
1.2.1Competent Cell Transformation
1.2.2Plasmid DNA Purification by Boiling Lysis (Small Scale AKA Miniprep)
1.2.3Restriction of Plasmid DNA (pDNA)
1.3Additional Protocols
1.3.1Overexpression
1.3.2Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep)
Common Protocols:
Assembly of BioBricks using the Red/White 3-Antibiotic Assembly Method
This method is a variant of the 3-Antibiotic Assembly Method that has been developed by Ginkgo Bioworks and New England Biolabs.
In the 3-Antibiotic Assembly Method, the destination backbone ( pSB1A3 , pSB1C3 , pSB1K3 , or pSB1T3 ) is amplified via PCR, using DNA received in the 2010 Distribution.
We had difficulty generating a large quantity of plasmid backbone in this manner, and as a result were unable to assemble biobricks.
As an alternative, we exploited the ability of part BBa_J04450 (expressing red fluorescent protein - RFP) to create in its host cell a very strong red color following incubation; this has proven a useful and successful variant of the 3-Antibiotic Assembly Method.
We retained the selection advantage of the 3-Antibiotic Assembly Method (associated with having a destination plasmid containing a different antibiotic resistance than the upstream and downstream plasmid) as the expressing RFP biobrick is available in a wide variety of BBF plasmids.
With this variation, we can not only visually screen our colonies for the presence of our expected assembly product (as indicated by a lack of red fluorescence), we also have a built-in negative control in that a lack of assembly product will result in a re-ligation of the expressing RFP biobrick into the destination backbone, producing red colonies.
Step 1 - Restriction
Performed according to Ginkgo Bioworks/NEB BioBrick Assembly Kit, with several modifications.
Digestion of upstream part
5µL Upstream part plasmid†
0.5µL EcoRI-HF
0.5µL SpeI
2.5µL 10x NEBuffer 2
0.25µL 100x BSA
16.25µL MilliQ H2O
Digestion of downstream part
5µL Downstream part plasmid†
0.5µL XbaI
0.5µL PstI
2.5µL 10x NEBuffer 2
0.25µL 100x BSA
16.25µL MilliQ H2O
Digestion of destination plasmid
5µL Destination plasmid†‡
0.5µL EcoRI-HF
0.5µL PstI
2.5µL 10x NEBuffer 2
0.25µL 100x BSA
16.25µL MilliQ H2O
Restriction digests were incubated at 37oC for 10 minutes.
Following each restriction digest, all samples were subjected to heating at 80oC for 20 minutes to irreversibly denature the restriction endonucleases.
† Typical concentrations obtained from our minipreps are approximately 50ng/µL, therefore adding 5µL of plasmid DNA gives the restriction reaction a concentration of 10ng/µL, as recommended in the NEB BioBrick Assembly Kit Literature.
‡ Destination plasmid is either pSB1A3, pSB1C3, pSB1K3, or pSB1T3, containing part BBa_J04450 which is the expressing red fluorescent protein.
Step 2 - Ligation
Performed according to Ginkgo Bioworks/NEB BioBrick Assembly Kit
Ligation of Upstream and Downstream parts into Destination Plasmid
2µL Upstream part digestion
2µL Downstream part digestion
2µL Destination Plasmid digestion
2µL 10x T4 DNA Ligase Buffer
1µL T4 DNA Ligase
11µL MilliQ H2O
Ligation mixes were incubated on the bench top (~20oC) for 10 minutes, then transformed into competent DH5α cells.
Step 3 - Transformation
Thaw 50µL of aliquotted competent cells (DH5α) on ice
Gently pipet 2.0µL (~1ng) DNA (from ligation mix) into competent cells
Incubate the cells on ice for 30 minutes
Heat shock the cells in a water bath at 42oC for EXACTLY 45 seconds
Incubate cells on ice for 5 minutes
Add 400µL of sterile SOC media to the cells and incubate at 37oC for 90 minutes with shaking (250RPM)
Plate 200µL on LB agar plate containing the appropriate antibiotic (reserve remaining cells and re-plate if no growth)
Allow cell suspension to be absorbed into agar by leaving agar side down for 10-15 minutes
Incubate the plates in the 37oC incubator for approximately 36 hours (agar on top)
Step 4 - Selection of Colonies and Colony PCR
Following ~36 hour incubation at 37oC, plates were inspected, and white colonies were picked and subsequently subjected to Colony PCR.
Colony Picking
Pick a colony from a transformation plate using a sterile toothpick or micropipette tip
Transfer the cells to a 1.5mL microcentrifuge tube containing 50µL of sterile MilliQ H2O
Place microcentrifuge tube in heat block at 99oC for 5 minutes to lyse the cells and denature DNases.
Centrifuge at max speed in a table-top microcentrifuge for 1 minute to remove cell debris
Use 5µL of supernatant as template for PCR
PCR to confirm length of newly assembled part
Reaction Conditions:
Component
Concentration
Volume (µL)
MilliQ H2O
N/A
33.8
Pfu Buffer with MgSO4
10x
5
dNTP
10mM
2
VF2 Forward Primer
10µL
2
VR Reverse Primer
10µL
2
Template DNA
N/A
5
Pfu DNA Polymerase
2U/µL
0.2
Cycling Conditions:
Step
Temperature
Time
1-Initial Denature
98oC
3 minutes
2-Denature
98oC
30 seconds
3-Anneal†
58oC
30 seconds
4-Extend
72oC
30 seconds‡
5-Final Extend
72oC
15 minutes
† Annealing temperature for Phusion is MT +3oC
‡ Extend time is 15-30 seconds per kb; time adjusted accordingly
Step 5 - Plasmid DNA Purification
We followed Qiagen's QIAPrep Spin Miniprep protocol, as follows:
Pellet cells from 1-1.5mL of liquid culture in a microcentrifuge tube (remove and discard supernatant)
Resuspend cells in 250µL buffer P1
Add 250µL buffer P2 and mix by inverting 4-6 times
Add 350µL buffer N3 and mix immediately by inverting 4-6 times
Centrifuge for 10 minutes at 13000RPM at room temperature in a table-top microcentrifuge
Apply resulting supernatant to spin column
Centrifuge for 60 seconds (discard flow through)
Wash column with 750µL of buffer PE (discard flow through)
Centrifuge for an addition 60 seconds to remove residual PE buffer (discard flow through)
Transfer column to clean 1.5mL microcentrifuge tube
Add 50µL buffer EB, let stand for 1-5 minutes, and centrifuge for 1 minute (retain flow through)
Protocols prior to the development of Red/White 3-Antibiotic assembly
Competent Cell Transformation
Thaw 20µL of aliquotted cells (DH5α or BL21(DE3)) on ice.
Gently pipet 2.0µL of DNA into competent cells
ATTENTION:
Do not perform any additional mixing
Never use more DNA that 10% of the volume of the competent cells otherwise the cells get destroyed by osmotic shock
Incubate the cells on ice for 30 minutes.
Heat shock the cells in a water bath at 42oC for EXACTLY 45 seconds.
Incubate the cells on ice for 5 minutes.
Add 250µL sterile LB media to the cells and incubate at 37oC for 1 hour with shaking (200RPM).
Plate 100µL and 50µL on prewarmed LB agar plate containing the appropriate antibiotic.
For ligations, plate all 250µL.
Leave plate for 10-15 minutes to soak the cell suspension into the agar.
Flip plate over (agar on top)
Incubate the plates in the 37oC incubator overnight
Plasmid DNA Purification by Boiling Lysis (Small Scale AKA Miniprep)
Aseptically transfer 1.5mL of each overnight culture to a 1.5mL microcentrifuge tube (MCT) and pellet the cells by centrifugation in a benchtop microcentrifuge (2min at 13000RPM)
Remove and discard as much of the supernatant as possible by aspiration (e.g with a Pasteur Pipette). Do not suck up the cell pellet!!
Rinse the cell pellet by washing 1.0mL of sterile MilliQ H2O gently down the inside wall of the MCT. This removes any traces of the supernatant adhering to the MCT wall while minimizing the disturbance to the cell pellet. (/li>
Resuspend the cell pellet in 350µL of STET.
Add 25µL of the Lysozyme solution and mix by inversion.
Place the MCT in the bioling water bath for EXACTLY 35 seconds, remove and incubate on ice for 5 minutes.
Pellet the cellular debris by centrifugation at 13000RPM for 15 minutes. Transfer the supernatant to a fresh MCT and discard the pellet.
Precipitate the plasmid DNA by adding 40µL of 3.0M sodium acetate (pH 5.2) and 420µL isopropanol. Mix by inversion. Mix by inversion and incubate for 5 minutes at room temperature.
Pellet the plasmid DNA by centrifugation at 13000RPM for 10 minutes at 4oC. A pellet of plasmid DNA should be visible at the base of the MCT when complete.
Being careful not to disturb the pellet, discard the supernatant and rinse the pellet with 500µL of ice cold ethanol.
Repeat above step.
Invert and tap the open MCT several times against a piece of paper towel on your bench to remove as much ethanol as possible.
Store the open MCT at room temperature for approximately 10 minutes to allow all remaining traces of ethanol to evaporate
Add 50µL of TE (pH 8.0) containing RNase A and resuspend the plasmid DNA by flicking the base of the MCT with your finger. The plasmid DNA is ready for use or can be stored long term at -20oC.
Restriction of Plasmid DNA (pDNA)
In a 1.5mL microcentrifuge tube, add MilliQ H2O to final volume of 20µL, 2µL of restriction enzyme buffer, 2µL of plasmid DNA, and 0.25µL of each restriction enxyme.
Incubate at 37oC for 1 hour.
Heat kill restriction enzymes on a heat block at 80oC for 20 minutes.
Store short-term on ice or long-term at -20oC.
Additional Protocols
Overexpression
Inoculate two 5mL overnight cultures in LB media containing the appropriate antibiotic corresponding to the resistance of the plasmid backbone on which the gene is located; incubated at at 37oC in shaker.
Transfer the two 5mL cultures into an 2000mL Erlenmeyer flask containing 500mL of LB media containing the same antibiotic.
Measure and record initial read OD600 reading (Time = 0) blanking against the uninoculated LB media.
Begin incubation at 37oC in shaker.
Measure and record another OD600 reading 1 hour after (Time = 1) and continue record OD600 every 30 minutes (T + 0.5) until OD600 = 0.600 is reached.
When OD600 has reached 0.600, aliquot out a 2mL sample and induce the 500mL culture (ie. if promoter is pLacI, induce with IPTG) and continue incubating in 37oC shaker.
Aliquot 2mL samples and record the OD600 reading every hour for 3 hours following induction.
To run SDS PAGE, centrifuge the samples at 13,000 rpm for 5 minutes and discard the supernatant. Resuspend the cell pellets with 100µL of 8M urea. Mix 10µL of the urea/cell sample with 5µL of 6X loading dye.
Plasmid DNA Purification by Alkaline Lysis (Large Scale AKA Maxiprep)
Grow up a 500mL overnight culture in LB media containing the appropriate antibiotic corresponding to the resistance of the plasmid.
Centrifuge the cells at 5000 rpm for 10 minutes, discard the supernatant, transfer pellet to a 50mL falcon tube, record the weight of the pellet, and store at -20oC. ( 1- 2.5g cell pellets are expected)
Resuspend the cell pellet in 6mL Alkaline Lysis Solution I (ALS1). Vortex carefully and slowly; may use a clean glass stir rod. Add 20µL of 1mg/mL RNase A.
Add 1 mL of 10 mg/mL lysozyme (in 20mM Tris-HCl, pH 8.0)
Incubate at room temperature for 10 minutes.
Add 12mL of fresh ALS2, mix well, do not vortex, and incubate on ice for 10 minutes.
Add 9mL of ice cold ALS3, mix well, and incubate for 10 minutes on ice.
Centrifuge at 4oC and 5000 rpm for 15 minutes.
Decant supernatant filter within funnel into fresh 50mL centrifuge tube.
1:1 phenol:chloroform extraction: In the fume hood, add 4mL of phenol/chloroform (1:1 -- 2mL of phenol + 2mL of chloroform), vortex for 15 seconds, centrifuge at 4000 rpm and 12oC for 4 minutes, and collect the upper aqueous phase. To the aqueous phase, add 4mL of chloroform, vortex for 15 seconds, centrifuge at 4000 rpm and 12oC for 4 minutes, and save the upper aqueous layer.
Add 0.6 volumes of isopropanol to the saved aqueous layer and incubate on ice for 10 minutes. (Alternatively, precipitate overnight at -20oC)
Centrifuge at 4oC and 5000 rpm for 15 minutes.
Decant supernatant into a fresh falcon tube (can be saved for further plasmid DNA isolation)
Wash DNA pellet with 2mL 70% ethanol; centrifuge at 4oC and 5000 rpm for 5 minutes.
Air dry the DNA pellet; resuspend in 4mL 20mM Tris-HCl pH 8.0 by vortexing.
Add 200µL 1mg/mL RNase A and incubate at room temperature overnight.
Ethanol precipitation: Add 0.1 volumes of 3M Na-acetate pH 5.2, add 2 volumes of cold ethanol (-20oC), incubate 30 minutes on ice (or longer at -20oC).
Centrifuge at 4oC and 4500 rpm for 15 minutes; carefully remove supernatant with 200µL pipette.
Wash pellet with 750µL of 70% ethanol, centrifuge at 4oC and 4500 rpm for 2 minutes, remove supernatant, and air dry the pellet for 10 minutes.
Resuspend the pellet in 200µL of MilliQ H2O (or TE).
Solutions:
ALS1: 50mM glucose, 25mM Tris-Cl, 10mM EDTA pH 8.0
ALS2: 0.2M NaOH, 1% (w/v) SDS
ALS3: 60mL of 5M K-acetate, 11.5mL glacial acetic acid, 28.5mL MilliQ H2O
RNase A: 1mg/mL in 20mM Tris-HCl, pH 8.0 Lysozyme: 10mg/mL in 20mM Tris-HCl, pH 8.0
Retrieved from " http://2010.igem.org/Team:Lethbridge/Notebook/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
