"
Page
Discussion
View source
History
teams
Log in
 
Team:Waterloo
From 2010.igem.org
Edit History Move Discussion
Home
THE STAPHISCOPE!
ABSTRACT
Superbugs, or antibiotic resistant microorganisms, are microbes that have become resistant to traditional treatments. These types of infections are difficult to diagnose, treat, and eradicate, making the healing process time consuming and resource intensive. The native quorum-sensing unit from S.aureus (the Agr system), will be introduced into a non-pathogenic strain of E.coli. The E.coli will then effectively have the ability to eavesdrop on the activity of the pathogenic organism and emit an indication of the magnitude of the infection in the form of RFP. Using sensitivity tuners the system can be designed such that the response will occur at an exact level, when the size of the population poses a threat to the host. Upon a positive result from a diagnosis, further tests could be done to specify whether MRSA (methicillin-resistant S. aureus) or MSSA (Methicillin-sensitive S. aureus) are present.
MODELING
Since the project aims to detect S. aureus at low concentrations, the modelling aims to determine how sensitive the system will be so that it can be adjusted to detect S. aureus at clinically-relevant concentrations. To that end, models for the sensitivity tuners from the 2009 Cambridge team needed to be built that can be combined with models for the AIP detection system. Most importantly, the noisiness of the system needs to be determined to know if false positives will be generated.
WE WOULD LIKE TO THANK OUR GENEROUS SPONSORS.
Project
Modeling
Outreach
Human
Practices
1.0 INTRODUCTION
Methicilin resistant Staphylococcus aureus (MRSA) are bacteria whose presence has been quite problematic in terms of human pathogenic infections. Since its discovery in the 1880s, Staphylococcus aureus has been the known cause for several kinds of minor skin, and major post-surgical infections. Prior to 1940, mortality rates in relation to this pathogenic organism reached 80% (Deurenberg and Stobberingh, 2008.) The first wave of resistant S. aureus came two years after the introduction of penicillin for medicinal practice in 1940. In less than twenty years, most of the S. aureus strains known to man were unaffected by the antibiotic. In the late 1950s, a penicillinase-resistant penicillin was brought forth to the medicinal market; this was methicilin (Deurenberg and Stobberingh, 2008.) The bacterium's subsequent resistance to methicilin came two years after its introduction in 1959. The mecA gene responsible for methicillin resistance is also to blame for the organisms’ desensitization to several classes of antibiotics (Deurenberg and Stobberingh, 2008). This has led to the creation of the term: MRSA, now commonly used worldwide.
While an MRSA infection is much like a S. aureus infection, the difference comes in the lack of sensitivity of the MRSA to several classes of antibiotics. This makes MRSA infections a serious threat in both health and economical aspects.
The infection could be easily transferred directly (through regular skin-to-skin contact), as well as indirectly (through contamination of surfaces). The ease with which infection could occur, as well as the high mortality rate involved with the infection, make MRSA a serious threat to the health of patients worldwide (Durai et al., 2010.)
Apart from health issues, the high treatment costs associated with MRSA infections are also of priority. The treatment of MRSA infected patients may result in intravenous antibiotics being taken, as well as a prolonged hospital stay. A recent study found that the costs of such a treatment can range between $8 364 and $13 940 per patient. This means that one patient’s treatment is approximately a fourth of the cost of an annual MRSA screening for the staff of an entire hospital. As the spending associated with treatment will far out-weight that of screening, it is very important that efficient and low-cost screening methods be implemented in hospitals (Durai et al., 2010.)
There are only a few methods, which currently exit for MRSA screening. These are plating, liquid-broth inoculation, and PCR (polymerase chain reaction) assays. While these will be discussed in further detail later on in the report, it is important to note that these methods could be costly and time consuming, and could sometimes present incorrect results (Durai et al., 2010.)
The International Genetically Engineered team at the university of Waterloo has been working on a method which will be used in supplement to the already-existing ones for the detection of MRSA infections. It is a proof of concept for diagnosing the presence of Staphylococcus aureus in an infection site before it has had the chance to create an infection. This synthetic biology–based diagnostic method will take advantage of the quorum sensing mechanism of Staphylococcus aureus, and utilize Escherichia coli as the sensor and reporter
2.0 CURRENT METHODS USED FOR THE DETECTION OF METHICILLIN-RESISTANT STAPHYLOCOCCUS AUREUS
Early detection and treatment of MRSA carriers, and persons with developed infections is of grave importance. A simplified grouping of the methods available to hospital staff for diagnosis of MRSA presence is presented below.
2.1 CONVENTIONAL METHODS
These methods have been around for a long time and involve basic plating and inoculation assays. They are quite labor and resources intensive. This is especially true nowadays, as a greater amount of tests needs to be completed to accommodate the increased number of potential patients involved with MRSA, either as passive carriers or as infection victims (Fang and Hedin, 2003.)
2.1.1 PLATING METHOD
This method usually involves two series of platings. The first one is inoculation onto one or both of Mannitol Salt Agar plates, and Blood Agar plates. The inoculated solid media then need to be incubated at 35°C for 24 - 48hrs and 18 - 24hrs respectively (Warren at al, 2004.) After growth is observed, one may identify S. aureus as a yellow colony on the mannitol plate, and as a cleared out zone on the Blood Agar, further confirming with Gram staining. Specific tests can be completed by streaking the resulting single colonies onto antibiotic plates for MRSA identification. This, however, takes up another 18-24hrs at the least. This method is very time consuming and potentially inappropriate for the time-dependent nature of infections. It is also highly resource intensive in its requirement for the technicians’ time, as well as the lab’s plates, agar, antibiotics, etc. Nonetheless, it presents 92.8% specificity in differentiation between various S. aureus, and 98.5% sensitivity in recognizing positive versus negative results (Warren at al, 2004.)
2.1.2 ENRICHMENT BROTH METHOD
This particular method utilizes liquid broth, which is to support the growth of MRSA or S. aureus in general. Depending on which of the above is chosen, inoculation of the original sample could occur into one of the following broths – MRSA:  Iso-Sensitest broth  2.3% NaCl, 1 g of aztreonam/ml, and 2 g of oxacillin/ml (Fang and Hedin, 2003); or S. aureus: tryptic soy broth and 6.5% sodium chloride (Warren et al., 2004). These are left overnight, and are to be re-plated on the next day, on the same types of pates as described in section 2.1 above. Unfortunately, the enrichment step proves unworthy, as sensitivity is decreased to 92.4% and specificity, to 91.4% (Warren et al., 2004.) This method also proves very tedious, time and resource consuming.
2.2 RT-PCR METHODS
The above-mentioned methods are considered to be, although fairly reliable, extremely slow. Plating and sensitivity testing are time-consuming processes, which put the previously-mentioned techniques at a minimum delay of 20 to 24 hrs (Warren et al., 2004.) The quickest method available for the detection of Methicillin-resistant Staphylococcus aureus is a PCR assay. Unfortunately, PCR assays are very expensive as compared to other techniques. The cost of some PCR assays (including reagents and technicians’ time) may be approximately $9 CAD per test if done in-house, and around $40 CAD if not (Mahony et al., 2004.) PCR involves the use of primers, which will recognize a sequence particular for MRSA and amplify it, producing a visible fluorescence –detectable result during the amplification stage (Elsayed et al., 2003.) While various testing facilities may use different genes limited to S. aureus, the most commonly used sequences for MRSA detection through PCR are the mecA (responsible for methiciilin resistance) and nuc (S. aureus species-specific marker) genes (Elsayed et al., 2003.) The two methods described below reveal differentiating ways of
1)preparing the samples obtained from patients to be analyzed, and
2)the PCR techniques which follow in accordance with the sample preparation.
2.2.1 BROTH-PCR METHOD
This method involves the use of enrichment broth, which will show preference towards nuc-carriers with mecA in the same cell (Milsson, Alexandersson and Ripa, 2005.) Samples from patients would be taken and inoculated into the above-mentioned enrichment broth and incubated overnight. Results could be obtained on the next day using PCR. The nuc-specific primers which have been used in previous trials with the method are (Fang and Hedin, 2003):
a. NUC1 (5-GCG ATT GAT GGT GAT ACG GTT-3)
b. NUC2 (5-AGC CAA GCC TTG ACG AAC TAA AGC-3)
The broth PCR technique has been tested to give 93.3% sensitivity, 89.6% specificity, 31.8% positive predictive value, and 99.6% negative predictive value.  As the percentage reliability of negative results is very high, especially as compared to that of the positive predictive value, the application of the broth-PCR method will be able to report the negative samples on the morning after testing. This will, in the long run, save as much as 84.9% of the cost and labor which is associated with the necessity of further testing (Fang and Hedin, 2003).
2.2.2 IDI-MRSA DETECTION METHOD
The IDI (Infectio Diagnostic Inc) nasal swab method provides an assay, which completely removes the need for inoculation in liquid or solid media. The assay allows for the performance of PCR directly from a nasal swab sample, using reagents and methods provided by the manufacturer (Infectio Diagnostic Inc). While sample preparation differentiates, PCR methodology applied is quite similar to the one described in section 2.2.1 above. There are, however, distinct differences between the results produced by the two methods. For one, the IDI-MRSA detection method can be complete in as little as 1.5hrs post sample collection, versus next day with the above-mentioned method, and at least 24hrs with the plating methods. As well, The IDI-MRSA provides 91.7% sensitivity, 93.5% specificity, 82.5% positive predictive value, and 97.1% negative predictive value when compared to culture-based methods (Warren et al, 2004.)
3.0 STAPHISCOPE – A SYNTHETIC BIOLOGY PERSPECTIVE ON MRSA DETECTION
As could be noted, there are not many methods available for the detecting of this pathogenic organism. StaphiScope, iGEM Waterloo’s 2010 project, has been designed as a supplement to the methods described above. This diagnostic tool has the goal of sensing and reporting for the presence of S. aureus populations in an area of interest (e.g., a wound) prior to virulence. It will do so faster than any of the described plating methods, and at a lower price. The method relies very heavily on the quorum sensing capabilities of Staphylococcus aureus.
3.1 QUORUM SENSING: WHAT IS IT AND WHY IS IT SO IMPORTANT?
Quorum sensing is defined as “the communication mechanism that enables bacteria to make collective decisions” (Boyen et al., 2009.) This mechanism involves the release of signaling molecules to the cell’s exterior. In gram-positive bacteria such as S. aureus, these signaling molecules are referred to as auto-inducing peptides (AIPs). The signaling molecules could then be detected by special receptors on other bacteria (not necessarily of the same strain) and give feedback in regards to the population density at the site. Bacteria rely on quorum sensing in order to thrive as a population. Most importantly, however, at certain critical concentrations of the AIPs, the bacterial populations make a collective decision to, for example, initiate virulence (Boyen et al., 2009.)
This year, the UW iGEM team is attempting to listen in on the quorum sensing “conversations” of S. aureus, by engineering E. coli to receive the AIP signaling molecule mentioned above. Through a signal amplifier, E. coli would be able to detect the presence of S. aureus at very low concentrations, and prior to the bacteria having the chance to become virulent.
3.2 PRACTICAL APPLICATION OF STAPHISCOPE
The purpose of StaphiScope is to provide the option of speeding up some of the methods mentioned above, without compromising their reliability. It will be a particularly useful tool in speeding up the conventional methods of serial plating. In particular, the plating method described above requires that samples first be plated onto either Mannitol  Salt Agar in order to obtain yellow colonies, or Blood Agar to obtain zones of clearance. This will aid in picking out Staphilococcus aureus apart from other species which may have grown. Only after this initial growth can we plate on antibiotic plates containing β-lactam class of antibiotics (methicilin, penicillin and a few others). This would normally take up another, at the very least, 18-24 hours. Using StaphiScope, we could speed up the first step of this process, by picking out the Staphilococcus aureus from all other species without the requirement of growing them up on solid media overnight first. With StaphiScope, the bacterial population from the samples would be grown into liquid culture, and the presence of RFP will be evaluated. The following will be involved in using the StaphiScope method commercially:
1.Obtain sample from patient on cotton swab. This will, for example, be from a wound.
2.Obtain liquid LB culture and inoculate into it a colony (from a plate containing many) of StaphiScope E. coli. These are E. coli species containing BioBrick K359009 – final construct. These plates could be kept in stock for quick access.
3.Inoculate sample from patient into liquid media containing StaphiScope and place at 37°C for 3 – 4 hrs. In our experience this is sufficient to show color of RFP is it has been activated. In addition, it would be useful to spin the cells down through a centrifuge (2 min at 13, 000 rpm). This would  bring the cells to the bottom, allowing us to see the red color expressed more clearly.
4.Upon positive result (red color produced), the cells could now undergo either plating on media containing the methicillin class of antibiotics, or a PCR-detection method using nuc/mecA genes (as described in section 2.2.1)
This method would bring the time consumption demands down to a few hours, as opposed to at the very least, a day. As well, less media would be used in the process. Finally, it is much cheaper than the faster methods such as PCR.
It is possible to initially inoculate the sample from the patient into liquid broth containing samples of the methicillin class of antibiotics. The expression of RFP in this case would mean that MRSA are present. This direct method of detection could shorten the test to 3-4hrs in length. However, this may lead to false positive results, as it is possible for other bacteria to release AIP as well, activating the StaphiScope.
3.3 TECHNICALITY OF THE STAPHISCOPE S. AUREUS DETECTION TOOL
The primary design of the construct consisted of AIP sensor infrastructure BBa_I746101 + BBa_I746104, permeability device BBa_I746201, amplifier, and reporter. Amplifier would have been selected from BBa_K2743xx series based on the investigations by the modeling team. Red fluorescent protein gene BBa_E1010 from BioBrick I13507 will be used as reporter. Please refer to Figure 1 below for more information.
Figure 1: Primary design of StaphiScope project, incorporating sensor, amplifier and reporter of AIP.
An additional design involved the addition of a generator part. This would have allowed E.coli to produce its own AIP, as a means for testing the response without worrying about permeability into the periplasm.  This design was completed but could not  be tested due to time constrains. As well, no amplifier was added in the final construct. Below is a discussion of the major components of StaphiScope.
3.3.1 COMPONENT #1: FEP A PERMEABILITY DEVICE
As S. aureus releases AIP in order to initiate any kind of quorum sensing, it is important that Escherichia coli will be able to receive and properly identify this signal. As E. coli is a gram-negative bacterium (contains an additional outer membrane), an issue arises of AIP not being able to get through this extra layer. This is why the iGEM team was forced to use FepA as part of the plasmid construct. FepA is an outer membrane permeability device, which will allow for AIP (the quorum sensing signaling molecule which will come from S. aureus) to get through the outer membrane of the gram-negative E. coli. The permeability device comes with its own promoter, on which we will rely for optimal expression.
3.3.2 COMPONENT #2: AIP SENSOR
It is important that AIP does indeed make it past the outer membrane of E. coli, as this is crucial to turning on the sensor and reporter functions of the StaphiScope. The AIP sensor function will allow E. coli to “listen in” on the quorum sensing conversations of the S. aureus. The BioBrick is modeled after the agr (accessory gene regulator) quorum sensing system present in S. aureus. AIP (from same cell or from another cell producing the peptide) binds AgrC (a histidine kinase). This, in turn, allows for phosphotransfer to AgrA and subsequently, through activation of transcription, more AIP is produced, amplifying the signal. It is a loop (Atkinson and Williams, 2009.) This is part of the mechanism which allows the bacteria to be aware of the concentration of its own strain in the infected area, subsequently leading to a “collective decision” such as expressing virulence. Please refer to Figure 2 for a visual display of the previously mentioned.
Figure 2: agr quorum sensing system in S. aureus (Cambridge iGEM, 2008)
3.3.3 COMPONENT #3: SIGNAL AMPLIFIER
This BioBrick will serve the purpose of further amplifying the AIP signal obtained by E. coli, and thus, allowing it to “sense” S. aureus, before S. aureus has had the chance to produce a high enough concentration of the quorum sensing signal to initiate a response. In other words, E.coli would be able to, through amplification, “sense” S. aureus before S. aureus has had the chance to sense itself. Unfortunately, there was insufficient time for proper research to be done on this, and so, the final construct lacks an amplifier. The amplifier is not a necessary part, but simply an assurance that the signal will be received through over-amplification.
3.3.4 COMPONENT #4: RFP REPORTER
This RFP BioBrick will produce a red-fluorescence signal proportional to the AIP concentration E. coli senses. This way, not only would one be able to tell that S. aureus is present, they would also be able to quantitatively compare samples to see where the infection has spread further. This would mean being able to decide whom to treat first, in the worst case scenario.
3.3.5 COMPONENT #5: AIP GENERATOR; PERIPLASM TEST.
The purpose of this construct will be to check the sensor-reporter system. The alternative construct contains AIP Generator for self-induction and no permeability device to facilitate accumulation of AIP in the periplasm. The system would essentially product its own AIP, thus showing activity of sensor-reporter system before final tests in presence of S. aureus.
3.4 BIOBRICKS SUBMITTED
As a supplement to the following information, please refer to the construction tree presented in Figure 3 below.
Figure 3: Construction tree of StaphiScope
K359003 - This part, which has been submitted to the registry, contains a P2 promoter and an RFP reporter.
K359006 - This part, which has not been submitted to the registry, contains the AIP sender/generator and sensor/receiver parts.
K359007 - This part, which has been submitted to the registry, contains the FepA permeability device into the sensor/receiver construct.
K359008 - This part, submitted to the registry, is the testing construct. It contains the sender/generator, sensor/receiver and reporter parts.
K359009 - This is the final construct of StaphiScope. It contains the permeability device, as well as the sensor/receiver, and reporter parts.
3.5 THE QUANTIFICATION INITIATIVE
Characterization of certain parts, functioning as signal ampliﬁers, was done by the iGEM Team Cambridge, which designed them, in terms of arabinose as an input signal. The arabinose sensing system consists of promoter pBAD and it respective repressor AraC. The BioBrick part BBa_I0500 contains both of those components in one part, while BBa_I13453 and BBa_I13458 are similar to BBa_I0500 split into two components: pBAD and AraC respectively. In order to retrieve the data about Team Cambridge’s parts, the modeling division of iGEM team Waterloo 2010 requested characterization of pBAD in relative promoter units (RPUs). The article by Kelly et al. (2009, Measuring the activity of BioBrick promoters using in vivo reference standard) describes the issue of great deviations involved with measuring biological systems due to too many factors to account for. The proposed solution is to measure in reference to a standard: the promoter BBa_J23101. The proposed vector for both the measured part and the standard is pSB3K3 (low copy number kanamycin resistant BioBrick plasmid) with green ﬂuorescent protein gene as reporter. Inspired by swappable promoter construct BBa_J61002 similar construct with J23101 in place of swappable promoter element was derived from pSB3K3, the BBa_K359201.
Figure 4
RFP (BBa_E1010) will be used instead of GFP due to consistent failure to express GFP (BBa_E0040) reported by Waterloo iGEM teams from previous years. The ﬂuorimetric assay can be performed on a ﬂuorescent protein with the use of ﬂow cytometer or microplate reader. The two approaches focus on different aspects of the retrievable data. Flow cytometry acquires data on individual cells, however each new concentration of input and time point requires individual assay. Incubating ﬂuorescence-capable plate reader allows to acquire many time and concentration points overnight or even several nights, and is thus preferable. Checking back on the QC sequencing data for I0500 it was discovered that the integrity of the part in the distribution available was not succesfully confirmed by sequencing, which explains failure of RFP expression with this part.  The alternative approach of assembling complete the pBAD sequence in two halves to the forward and reverse PCR primers used to amplify the vector, such that when the vector is religated the correct promoter sequence is formed. The AraC repressor can be provided on another plasmid.
4.0 FUTURE PLANS
The AIP molecule utilized by S. aureus as part of its quorum sensing mechanism is part of a group of 4 classes of signaling molecules, all with the ability to cross inhibit each other’s activity. It could be speculated that, if present at the same time, they would slowly inhibit the signal transmission between members of the same infection site, and thus prevent infection. Thus, it would be recommended that, after the construction of the StaphiScope, an extension be added to the project, where within Escherichia coli is integrated the function to produce a signal molecule of the cross-inhibitory group (Atkinson and Williams, 2009)
5.0 REFERENCES
Atkinson, S.  and Williams, P. (2009). Quorum Sensing and Social Networking in   the Microbial World. Journal of the Royal Society Interface. 6, 959-978.
Boyen et al. (2009.) Quorum sensing in veterinary pathogens: Mechanisms, clinical importance and future perspectives. Journal of Veterinary Microbiology, 135, 187-195.
Deurenberg, R.H. & Stobberingh, E.E. (2008). The evolution of Staphylococcus aureus. Infection, Genetics and Evolution, 8 (6), 747-763.
Durai, R., Ng, P., and Hoque, H. (2010). Methicillin-Resistant Staphylococcus aureus: An Update. Journal of Association of periOperative Registered Nurses.  91 (5). 599-609.
Elsayed et al., (2003) Development and Validation of a Molecular Beacon Probe–Based Real-Time Polymerase Chain Reaction Assay for Rapid Detection of Methicillin Resistance in Staphylococcus aureus. Arch Pathol Lab Med, 127, 845-849.
Fang, H. and Hedin, G. (2003). Rapid Screening and Identification of Methicillin-resistant Staphylococcus aureus from Clinical Samples by Selective-Broth and Real-Time PCR Assay. Journal of Clinical Microbiology, 41 (7), 2894-2899.
Mahony et al. (2004). Performance and Cost Evaluation of One Commercial and Six In-House Conventional and Real-Time Reverse Transcription-PCR Assays for Detection of Severe Acute Respiratory Syndrome Coronavirus. Journal of Clinical Microbiology, 42 (4), 1471-1476
Milsson, P., Alexandersson, H., and Ripa, T. (2005). Use of broth enrichment and real-time PCR to exclude the presence of methicillin-resistant Staphylococcus aureus in clinical samples: a sensitive screening approach. Journal of Clinical Microbiology and Infection, 11 (12), 1027-1034.
University of Cambridge iGEM Webpage. (2007). Modeling Bacterial Quorum Sensing. Retrieved September 1, 2010, from, https://2008.igem.org/Team:Cambridge/Modelling
University of Waterloo iGEM Team Webpage. (2010). BactoHouse: Abstract. Retrieved September 1, 2010, from http://igem.uwaterloo.ca/BactoHouse.
Warren et al. (2004). Detection of Methicillin-Resistant Staphylococcus aureus Directly from Nasal Swab Specimens by a Real-Time PCR Assay. Journal of Clinical Microbiology, 42 (12). 5578-5581.
1.0 INTRODUCTION
Since the Staphiscope project aims to detect S. aureus at low concentrations, it's important to determine how sensitive the system will be so that it can be adjusted to detect S. aureus at clinically-relevant concentrations. A detector that triggers at too low a concentration may display false positives, while one that triggers at too high a concentration may not give a positive when it should. To achieve the best sensitivity, numerical characterization for the Cambridge's 2009 sensitivity tuners needs to be obtained independent of the promoter used, in order to be combined with models for the AIP detection system and yield a predictive numerical model for Staphiscope. Work toward these ends is ongoing.
2.0 BACKGROUND
The sensitivity of Staphiscope may depend on various factors. For now, analysis has been restricted to just one factor: the choice of part used for the amplifier component of the system. The amplifier will be chosen from one of the 15 amplifiers submitted to the parts registry by Cambridge in 2009. Each amplifier responds uniquely to a given input signal, differing from the others with respect to its activation threshold (amoung a few other parameters less crucial to our analysis). Our goal is to determine which amplifier has an activation threshold in the correct range for the detection of S. aureus in relevant concentrations.
Empirical characterization of the response curves of each amplifier was carried out by Cambridge. However, in Cambridge's system the amplifiers were under control of the the pBAD promoter, which is not the case in Staphiscope. Therefore, the data gathered by Cambridge is not directly applicable to our system, since in general the response curve of each amplifier will be different under different promoters.
To obtain a numerical characterization of each amplifier, independent of promoter choice, we are undertaking the task of "reverse engineering" Cambridge's data to extract the parameters describing the amplifiers. A more detailed explanation of our approach first requires a description of the mathematical models relevant to this system.
3.0 SYSTEM MODEL
The parts characterized by Cambridge consist of a detector (the pBAD/AraC promoter) and one of the fifteen amplifiers. To obtain a mathematical description of the entire system, Cambridge used the following equations to describe the input/output response of these individual components.
The Cambridge 2009 modelling page develops these equations in more detail.
When these individual component models are strung together, the resulting model of the entire system has a sigmoidal shape. This means the response curve for the entire system can be fit to a Hill function, which is of the form
.
Note that 4 parameters are required to specify the response curve: increase in rate, basal rate, switch point, and Hill coefficient. These parameters need to be determined empirically, with the exception of the Hill coefficient, which we assume is equal to 2.
The detector and amplifier also each have the form of a Hill function, but of course each has its own set of parameters which will differ from the Hill parameters of the overall system.
4.0 EXTRACTING DATA
Ultimately we seek a numerical characterization of the amplifier part alone. Since we know its response curve has the form of a Hill function, we need only find its four Hill parameters. The following table summarizes the known and desired data.
If all the parameters other than the amplifier parameters are known, the model above together with the response curves of the amplifiers is enough information to extract the desired parameters. Explicitly solving the equations above for the amplifier parameters is difficult; instead, MATLAB's curve fitting toolbox will be used to find the parameters which, when combined with the known parameters and inserted into the equations for Cambridge's overall system, matches the empirical data obtained by Cambridge. This process must be repeated for each of the 15 amplifiers.
Unfortunately, not all the necessary parameters are known. Some were not measured by Cambridge (those in block 1), but can be found in the literature. However, the Hill parameters for the pBAD/AraC promoter are not known, and must be measured. To do this, an experiment is being designed to characterize this promoter in RPU, but has not yet been carried out.
1.1 Outreach and Synthetic Biology
The newness of synthetic biology means that much of the population is not even aware that it exists (see Awareness & Attitudes Study see link at the bottom of this section). Therefore, an important aspect of our outreach efforts is to introduce the topic of synthetic biology and show its potential. We also hope to give people the foundational information that they need in order to understand future scientific developments. This form of outreach will help to improve the scientific literacy of the general population.
Another emerging issue is misconceptions held by the general public. In the development of synthetic biology, as with many new technologies, there is still much to learn and discover. As a result, the information made available to the public is often not a comprehensive, accurate picture of synthetic biology.
The following are the goals that we hope to achieve through educational outreach:
1.Inform the public about synthetic biology
2.Promote an education in science
3.Showcase opportunities in the field of science
4.Create an enriched science experience for students
5.Broaden the influence of iGEM
For more information about outreach and synthetic biology see:
This ground breaking study
by Peter D. Hart Research Associates, Inc. on awareness and attitudes of the public about synthetic biology found that 9 of 10 individuals think the public should know more about developing technologies
2020 Science
aims to provide a complete picture on the development of new sciences
Synthetic Biology Project
examines the development of synthetic biology
1.2 The Events
The method that we have chosen to achieve our goals is science education. This is the avenue most accessible and familiar to us as science students. By participating in events in our community we are able to influence multiple audiences by different means. Our outreach efforts are primarily focused at young students still deciding whether to continue in the field of science, students pursing science education, and the general public. Some of the events facilitated delivering the information on a small scale (in depth discussions with one or two students) while other events necessitated speaking more generally to larger groups. The events fostered synthetic biology awareness and were great learning experiences for our team.
ESQ Partnership
ESQ ( Engineering Science Quest ) is a day camp hosted at the University of Waterloo that brings hundreds of curious young minds to Waterloo each year to learn more about science and engineering. We held two different weekly activities for campers of ages 8-9 and 12-14.
The activity for children of 8 to 9 years of age involved extracting their own DNA from their cheek cells. It was approximately an hour and a half in length, and involved not only following the outlined procedure, but also, a discussion of synthetic biology, DNA, proteins, enzymes and more. At the end of each activity, the children were allowed to keep a sealed cryovial with their extracted DNA.
The protocol for the activity, as well as the PowerPoint presentation and script associated with it, are available here.
The second activity, which involved children 12 to 14 years of age was named “Do We Really Need to Wash Our Hands.” In this activity, the children were asked to swab their own hands (using a sterilized cotton swab) and plate the resulting swab on solid media. They would then wash their hands/use hand sanitizer and swab and plate again. The plates would be left to incubate at 37°C overnight, and the next day, the resulting growth would be presented to the children. As well, they were allowed to swab two other area of their choice, as to see how “dirty” these are. Most chose their own hair, backpack or shoe.
Prior to each activity, a PowerPoint presentation discussion bacteria, pathogenicity, hand-washing, DNA and synthetic biology was given to the students. As well, they were provided with a hand-out to aid them in their understanding of the activity. The handout contained step-by-step activity protocol, as well as spaces for hypothesizing and discussion. The PowerPoint presentation, protocol, script, and handout are available here.
We hope that these initiatives will help engage students outside of the classroom and get them excited about science.
iGEM Ontario (OGEM) Meeting, June 15 2010 at McMaster University
The second annual OGEM Meeting was held at McMaster University in Hamilton on June 15th, 2010. Members of iGEM teams from University of Waterloo, University of Western Ontario and University of Toronto gathered to discuss the future of a regional synthetic biology community, as well as a regional conference. The day turned over to discussions which centered around creating more communication and support between teams. In addition, the gathering was also an opportunity for teams to get to know one another before heading down to MIT. The meeting was a great success and the second of more regional gatherings to come.
As the meeting was held during the annual CSM (Canadian Society for Microbiologists) conference, members were able to attend a series of lectures given by valuable members of the field. As well, iGEM Ontario members were given the opportunity to speak about their projects and promote the idea of iGEM during a poster presentation held along with researchers in the field of microbiology.
In the future we hope to see this organization (independent of an individual iGEM team) become an important resource to Ontario iGEM teams and for educating the general public about synthetic biology. We are currently working on an iGEM Ontario website.
Building Life: The Science of Synthetic Biology
On June 23, Waterloo iGEM adviser, Dr. Trevor Charles, held an open public lecture aiming at discussing synthetic biology, its means and aims. During the lecture, the purpose of synthetic biology, its ethical and safety implications well as many other current topics were discussed. At the end, a facilitated panel discussion featuring Andre Masella (Waterloo iGEM team), Dr. Kathryn Plaisance (bio-ethicist), as well as Dr. Maria Trainer (Council of Canadian Academics) answered some of the public’s pending questions regarding synthetic biology. The lecture was a great success, with a large number of attendees of various scientific background. It was part of a series of lectures organized by the University of Waterloo’s Department of Biology, in attempt to increase public awareness of current scientific issues.
What We Accomplished
Upon reflection we feel that we have achieved the following goals through our outreach efforts:
1.Introduced synthetic biology to individuals who knew nothing about it
2.Educated multiple audiences on the fundamentals of science such that they will be able to better understand future scientific information and developments
3.Excited young minds about science
4.Strengthened our regional synthetic biology community
5.Communicated the work of other iGEM teams and the benefit and goals of the iGEM competition
6.Increased communication between local synthetic biologists
7.Gained insight and experience into what the public believes about modern science
8.Developed activities and displays to be used and improved upon in the future
Future Plans
We plan to continue our outreach efforts in the future. We plan to expand our efforts and continue to educate the public about synthetic biology and share our love of science. In the future our efforts will continue to center around science education. Our profile in our community is increasing as iGEM becomes a familiar group and synthetic biology ceases to sound frightening and gains familiarity. We hope that the Ontario community of iGEM teams will continue to grow and include teams from all across Canada, helping to strengthen the synthetic biology community across the country. As members of the Waterloo iGEM team we are proud to educate multiple audiences and to share our knowledge and passion with the world.
INTRODUCTION
As synthetic biology expands, as new innovations are made, and pressing world problems are solved, the potential impact synthetic biology will have on the world becomes more evident. Although a primary goal of developers of synthetic biology should be to consider the ethical, societal, safety, environmental, and political impact of the science, we believe that interest should also be paid unto the impact that synthetic biology will have in the business world.
As a group we are most interested in the course of development of synthetic biology in industry; the goal of our project is to try and decipher the path that synthetic biology will forge as it expands in the business world.
Our attempt to answer this question has begun with a comprehensive inquiry into important factors affecting the diffusion of synthetic biology. It is our opinion that before we attempt to make any conclusive statements about the future direction of synthetic biology or the economy as a whole we must have a complete picture of the current landscape of synthetic biology and the markets it could potentially impact.
This inquiry is intended for use by multiple audiences; particularly scientists and members of business in relevant industries. As scientists it is important to understand the context in which discoveries are made, understanding where world needs lie and how discoveries will impact the world makes for better-informed scientists. Members of the business world should also strive to have an understanding of where the need and rationale for discoveries come from. Although profit is the ultimate endgame a well-informed approach helps to prevent ethical pitfalls and often greater success.
This analysis looks at both extrinsic and intrinsic factors relating to the development of synthetic biology as a whole. In particular intrinsic factors such as the development of synthetic biology in specific industries (biofuels, pharmaceuticals, and bioremediation) is examined in depth. Important extrinsic factors such as the impact of patenting and open source are also analyzed. With this information we feel we have laid the foundations for a comprehensive inquiry that will allow us to better understand what the expansion of synthetic biology in the business world will resemble.
OPEN SOURCING and Synthetic Biology
What is Open Sourcing?
What differentiates synthetic biology from biotechnology is that it offers the creation of systems or pathways that would not be found naturally in an organism. Synthetic biologists believe in a standardized system of parts similar to that of electrical engineers having standard circuits and components1. This is also similar to how LINUX modules have been combined to create different software. This contrasts the closed-parts strategy where developers use such methods as patent protection and secrecy to gain a competitive advantage over others2.  That is why the Massachusetts Institute of Technology (MIT) has created the initiative for what is known as the “Registry of Standard Biological Parts” where it indexes biological parts that are currently being built. A standard array of modular gene switches or parts that can be found from a common library and can be mixed and matched in various combinations, which is the goal that synthetic biology targets. This is a similar path as what happened twenty years ago when software became standardized and allowed Microsoft to become a monopoly. The issue in the air currently is if this could happen with the synthetic biology industry3.
Advantages
Open-sourcing has been an idea that has been the basic foundation of synthetic biology throughout the years. Supporters strongly believe in a world where companies and academia will be able to develop and share parts freely for the advancement of the whole field, not just a singular firm or university4.
A term that is used commonly in this industry is called network effects. This means that the more a product is used, the more attractive it becomes. Over time as each part is used repeatedly on a specific metabolic pathway, especially when in successive experiments, its cost goes down. With the limited data that is available, it is predicted that total project costs could be cut down by 25% after its first successive use. It is also likely that these costs would be cut down several more times until it flattens as with each subsequent experiment more knowledge on that part is gained and intuitively, less errors are made.
One such example is with Amyris’ artemisinin project that spanned over the length of five years, costing $20 million. It was reported that 95% of the time spent was on finding and fixing unintended interactions between parts.
Due to the fact that one part must be used in conjunction with other sets of parts gives incentive to companies to create whole libraries. This is very similar to how software companies develop several programs that are able to cover multiple applications. Not only that, but there is opportunity for these companies to make profit by patenting some of these parts and making others openly available due to the strong modularity of the open-parts approach.
It is expected that companies will have their own individual parts needs. This means that other companies cannot just sit around and wait for another to develop a part. This also gives a type of competitive advantage as companies that share parts will not be losing their, ‘technological edge’ to other competitors. Since different companies have idiosyncratic needs and hence expertise, it proposes that community-based libraries will outperform individual companies. The industry will probably have a large number of small, distinctive customers, meaning that patent licensing will be less attractive and the open-parts initiative more so5.
Normally, the value of a patent depends on the inventor’s initial R&D investment, but with synthetic biology parts it would depend on that as well as how many researchers have subsequently used that part. According to the law, this allows patent owners to capture both sources of value, which can be unfair to society as they must deal with high prices without getting anything more in return. The open-parts initiative sets the price of parts equal to zero, so it is naturally able to solve that problem. There are two incentives that will draw companies to an open-parts initiative. The first would be the opportunity to share R&D investments among multiple firms and secondly the opportunity to produce parts faster due to shared insights6.
Barriers and Disadvantages
About twenty years ago when software became standardized, it opened the path for Microsoft to come in, take over the industry and monopolize it. Can something similar potentially happen in the synthetic biology industry? At the moment, synthetic biology is what is called a ‘tipping market’. It is unstable and prone to monopoly. Building on this, the tipping dynamic is indifferent to whether or not the dominant parts constellation is open or closed. If for instance a mature industry is able to grab a hold of these shared parts, it can work towards this open-parts initiative. Though on the other hand, companies, out of necessity will also pay for closed parts, both solutions are equally viable.
Some other issues include an agreement on standard nomenclature for the parts, therefore when actually designing a database, controversies might come up. As well when collaboration occurs there must be adequate legal infrastructure. This must include a license specifying the rights and duties of members. One of the main legal problems is that gene data is unlike software, it cannot be copyrighted.
Many pessimists towards the idea say that it is now too late for synthetic biology to use an open-parts collaboration. Now with Amyris’ advancement in the field it seems that this industry might monopolize and follow the path that Microsoft had. The fact that commercial synthetic biology receives so much government support shows a bit of laissez faire attitude. In America’s Department of Energy $350 million biofuel initiative there was no open-parts requirement at all. It is said that at least when Bill Gates cornered the software market he did it with private money7.
1 http://heinonline.org/HOL/LandingPage?collection=journals&handle=hein.journals/tlr85&div=50&id=&page
2,3,4,6,7 http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2726678/
5 http://www.nature.com/msb/journal/v3/n1/full/msb4100161.html
PATENTS and Synthetic Biology
The ideological reasoning behind applying for a patent lies in the fundamental incentive that it would bring forth financial rewards and a sense of monopolistic supremacy in an advancing market. The imposition of patenting technology has been deemed useful in many industries, most notably in electronics. However, it has also brought forth a considerable level of controversial dialogue in determining how historical protocols can be implemented within new technological streams that sometimes question the concept of man-made invention. For example, Craig Venter’s desire in protecting his synthetic cell research methodology gives him a deserving ownership, but his broad patent places a downgrading threat towards the field of synthetic biology. Ironically speaking, patenting is a legal practice that promotes innovation, but imposes a sense of apprehension for those who wish to adapt towards a more open-sourced strategy. This analysis focuses on the implications of introducing patent law to the field of synthetic biology as well as the practicality of gaining rights to a product not developed entirely by man, but with elements  arising from nature.
Monopolizing Synthetic Biology
The accrued financial and proprietary benefit of monopolizing an idea within a competitive market is indeed an accelerated advantage when a patent is granted. However, many have the misconception that monopolistic proprietorship has a standardized value to any industry it is applied to. For years, Professor John Sulston from the University of Manchester, a believer in promoting an open-source environment in the field of synthetic biology has proposed the implications that would arise should Venter receive approval upon his protocol in developing synthetic organisms.  Sulston states,
"I hope very much these patents won't be accepted because they would bring
genetic engineering under the control of the J Craig Venter Institute (JCVI). They would
have a monopoly on a whole range of techniques (BBC UK, 2010)."
The irony in this matter is that the technological industry used the concept of monopolistic competition to drive both innovation and the development of products that would eventually overpower the market leaders. The reason that this concept cannot presently be applied is that synthetic biology is strongly reliant on collaborative construction. Patenting biological parts or processes will not motivate a group of scientists to pursue their research, but cause them to weigh the opportunity costs between their groundbreaking research and the hefty payments they would make to patent holders.
In the case of Venter’s claim, organizations such as the ETC group believe that Venter’s group is eyeing on a profit making opportunity. Hope Shands of the ETC group states, “The fledgling synthetic biology industry keeps talking about how they’re going to fix climate change – but these sweeping patent claims reveal that the companies are much more focused on securing profits than on human needs” (ETC Group, 2007). The multi-purpose use of a biological methodology would not only give Venter ownership to a scientific methodology, but to a range of applications that could be possible in the chemical, medicinal or environmental industries. Venter reported to Business Week, “If we made an organism that produced fuel that could be the first billion- or trillion-dollar organism. We would definitely patent that whole process” (ETC Group,2007). Many would dispute that this financial and market acquisition would only de-motivate other synthetic biology scientists and facilities to halt their research. But from looking at it from Venter’s perspective, wouldn’t anyone want to reap the rewards on a project that took 15 years and 40 million dollars to reach its success?
Patenting Artificial Life: Is synthetic biology a man-made industry?
As previously noted, synthetic biology encapsulates the field of biotechnology, software and computing. The current patent laws that have been put in place have generally been applicable to all industries individually. However, the field of synthetic biology is an amalgamation of two fields that have faced years of controversial debate in terms of granting or approving patents. The concern lied in the fact that patents related to inventions in biotechnology or software could be broad or narrow, but extensive enough to “hold-up” the concept of innovation and invention (Rai and Boyle, 2007 ).
Synthetic biology is being renowned as a stream towards the development of artificial life, which for some raises social, ethical and legal concerns. Patent Act 101 states that any subject matter that is found in nature is not deemed as patentable. However, if the product found in nature is modified or transformed to something that is novel and non-obvious, it holds credibility in attaining a patent. Looking at the synthetic cell created by Venter’s team, it is difficult to denote the “man-made” material in his composition. His cell includes a computer generated “minimal genome sequence” that is encapsulated within a bacterial cell (found in nature) and still hosts cellular machinery required to allow the cell to adapt and function within its environment. The modification is within the genome, but even that raises some debate on whether it can be patentable.
The software industry has always tackled with patenting mathematical and computational algorithms, particularly in concern was their level of broadness. In perspective to the field of synthetic biology, an algorithm is analogous to the creation of novel and modified genetic sequences, which give rise to new metabolic pathways and cellular functionalities. Yet again, the derivation of their new genetic sequence comes from the naturally degenerate genetic code, bringing up the question of whether this is entirely man-made. What is seen here is a disconnect, in that it would be difficult to set  fundamental patenting protocols for the field of synthetic biology, given the legal complications faced in both the field of biology and computing. This is where many may agree that an open-source strategy such as the MIT Registry of Standard Biological Parts would bring forth more progression, versus the time and effort it would take to validate the patentability of a biological part or process.
SWOT Analysis
Below are other indicators to help define the relative benefit of introducing patent law, as well as the long-term threats that may hinder the advancement of research in the field of synthetic biology.
Strengths
1.Patenting assists in stimulating investment, and secured investments bring forth progression in research. Given that research projects can cost up to hundreds of millions of dollars, patenting synthetic biology technologies would provide a more steady approach in financing the development of an invention.
2.Developers are motivated by reward, which is provided by the successful implementation of patent law
Weaknesses
1.The field of synthetic biology is different in the sense that innovation is not promoted through attaining proprietary rights to an entire process or genome, but through the collaborative use of fine, functional and specific biological parts
2.Synthetic biology is multidisciplinary as it incorporates the field of biotechnology and computing, each has their own sets of legal rights, restrictions, and pitfalls
3.In addition, lawyers would require a more extensive level of knowledge and proficiency in both fields
4.Sometimes, hundreds of parts are necessary for the composition of one biological machine, and attaining rights to each part would create what is called to be a "patent thicket" (Rutz, 2009). Attaining rights to so many parts does not only hinder innovation, but is time-consuming and costly
Opportunities
1.Patents add value, reputation and credibility to an invention. Although this legal concept is frowned upon in the R&D field, it brings a source of financial opportunity that would bring forth strategic partnerships and lump some funding to further progress research. At an economic perspective, patenting would be useful in advancing one's research.
2.Patenting requires thorough documentation and characterization of each aspect of the part, streamlined patenting protocols could add standardization to the part development process
Threats
1.European patent law states that "iventions of commercial exploitation to morality are omitted from patentable rights" (Rutz, 2009).  The general public fears that if rights to a biological process or part are give they may be abused.  Examples are biological warfare or the unintentional release of pathogenic organisms.
2."Patent sharks and trolls" (Rutz , 2009) may find it simple to file lawsuits against such patents. •On the other side of the coin, some have claimed that finding a patent application for a biological part is similar to finding a needle in a haystack. Moreover, it is not the job of a scientist to be rummaging around in identifying his legal rights to using a biological part, nor should his time be consumed by understanding the patenting process relevant to his field.
Conclusion
The concept of patenting makes scientists cringe at the thought that their research and advancements would be diminished by the monopolistic power held by the broad patent holders within synthetic biology. There is currently no established set of specific protocols for patenting products in synthetic biology. Moreover, the attempt to blend the patent laws applicable to a range of disciplines to one as intricate as synthetic biology is difficult. Additionally, companies who wish to have a formalized process in examining and submitting processes would have to understand that a large number of additional resources would be needed to accommodate for the labour intensive, time consuming and costly process of patenting such biological parts. So the underlying question is that what is of greater value in today’s market, leading innovation through open-source strategies or banking on profit-making opportunities by securing an idea with a patent?
INDUSTRY ANALYSIS
Before we attempt to understand the impact of synthetic biology on the business world as a whole, we must first understand what kind of an impact it will have on specific sectors. The goal of the industry analysis was to pick specific industries that we felt would be heavily impacted by the emergence of synthetic biology and attempted to understand their current situation as well as opportunities and threats facing each industry.
Biofuel Industry
The oil industry has become an integral part of the society we live in for, transportation, food, healthcare, and communication. However, there is a dire need for alternative sources of energy and the world is beginning to turn towards biofuels for support in this area. Although there is immense promise in this area there is also many challenges to overcome such as reliance on environmental factors and adequate feedstocks. Although emergence of biofuels created through synthetic biology does not have the market potential to shift the whole method of biofuel production, politicians are optimistic that it has the potential to make an impact and have therefore taken measures to support the infrastructure of its’ development. The emergence of a viable synthetic biology biofuel would serve to excel the development of synthetic biology, particularly on the business stage.
Pharmaceutical Industry
Our knowledge of diseases and treatments has advanced to an exciting point as has society’s perception of health problems and issues. However, as fast as our knowledge advances, diseases like H1N1 and HIV provide a pressure for the pharmaceutical industry to advance further and faster. In addition, aging populations and growing urban centres poise pharmaceutical companies for significant and meaningful innovation over the coming decades. The pharmaceutical industry is expected to grow to an $800 billion industry by 2011, expanding as a global force. Thus, synthetic biology, although faced with challenges in terms of legal and social concerns, is poised to have a significant impact on the pharmaceutical industry. Not only is there promise for curative treatments that would change how we view illnesses like HIV, there promises to be significant implications in the social, technological, and political realms. Unanswered questions about the handling of intellectual property issues will challenge the development of synthetic biology in the pharmaceutical industry as will ethical, safety, and political issues.
Bio-remediation Industry
Unlike other industries, bio-remediation is an industry that has incorporated the use of genetic engineering and synthetic biology since the 1970’s. Based on the research conducted, there is a definitive understanding that this field poses an optimistic approach in achieving results that are environmentally friendly and cost-effective. The major setback with the field of bio-remediation is the intricacy of the techniques that are involved. Aside from the general publics qualms about bio-remediation procedures, even the scientific community does not have a holistic understanding of how the processes within bio-remediation work, or how to optimize a microbe’s “oil-eating” activity based on its metabolic characteristics. Most say that bio-remediation is a cost effective technique in treating vast oil spills, but it seems that investors and economists have not incorporated the cost of time and labour required in mastering the techniques involved. Once these techniques have established, there is a definite opportunity for this field to overpower the current invasive chemical processes that are being used as a last desperate resort in treating these environmental disasters.
The project wiki contains a condensed overview of the industry analyses, if you are interested in learning more be sure to check out the complete industry analysis.
Conclusions and Outlook
After conducting this research into the current and future position of synthetic biology particularly in the context of the business world, we are optimistic about the course that it will take. There are significant hurdles to overcome but overall there is a sense that synthetic biology will impart a positive impact not only in terms of specific products but also in terms of micro and macroeconomic impact.
In this inquiry we have attempted to understand some of the important issues facing synthetic biology from a business perspective. In the future we hope to take our work a step further; we hope to take our knowledge, thoughts, and questions into the business world. Through interaction with business owners, scientists, legal specialists, and other stakeholders we hope to disseminate our findings and build on them. In the future we will continue to focus on our goal of deciphering the path that synthetic biology will take as it emerges in the business world.
References
1)Rai A, Boyle J (2007) Synthetic Biology: Caught between Property Rights, the Public Domain, and the Commons. PLoS Biol 5(3): e58. doi:10.1371/journal.pbio.0050058
2)Barrett, Margaret. Intellectual Property. 2nd ed. New York: Aspen Online, 2008. 33-34
3)Dickinson, Boonsri. "Will Patents Give Craig Venter a Monopoly over Synthetic Life? - SmartPlanet." SmartPlanet - We Make You Smarter - People, Business & Technology. 28 May 2010. Web. 26 Oct. 2010.
.
4)"Artificial Life: Patent Pending | The Economist." The Economist - World News, Politics, Economics, Business & Finance. 14 June 2007. Web. 26 Oct. 2010.
.
5)"Postnote: Synthetic Biology." Parliamentary Office of Science and Technology, Jan. 2008. Web.
.
6)Rutz, Berthold. "Synthetic Biology and Patents : A European Perspective." Nature Publishing Group : Science Journals, Jobs, and Information. 2009. Web. 26 Oct. 2010.
.
7)Joachim, Henkel, and Maurer Stephen. "The Economics of Synthetic Biology." Molecular Systems Biology. Nature Publishing Group, 5 June 2007. Web. 26 Oct. 2010.
.
8)Ghosh, Pallab. "BBC News - Synthetic Life Patents 'damaging'" BBC - Homepage. 24 May 2010. Web. 26 Oct. 2010.
.
9)Ball, Philip. "Biology News: The Patent Threat to Designer Biology." BioEd Online. 22 June 2007. Web. 26 Oct. 2010.
.
10)ETC Group. "Extreme Monopoly: Venter's Team Makes Vast Patent Grab on Synthetic Genomes | ETC Group." 8 Dec. 2007. Web. 26 Oct. 2010.
.
11)Hammond, John, and Robert Gunderman. "The Limited Monopoly- Patent Law 101: What Is Patentable?" The Rochester Journal, June-July 2007. Web. 25 Oct. 2010.
.
Members
Advisors
About UW
OUR TEAM!
OUR UNDERGRADUATES!
Anum-ta Arif
Dan Barlow
Ekta Bibra
Diljot Chhina
Arpita Desai
Jon Eubank
Matt Gingerich
Billy Khan
Leah Kocsis
Jordan Lapointe
Peng-Lin Lu
Cveta Manassieva
Dawson Overton
Mariko Ozawa
James Scott
Alice Qi
Asad Sajjad
Brandon Wang
Fiona Whelan
Hilary Yeung
George Zarubin
OUR GRADUATES!
John Heil
Eddie Ma
Andre Masella
Danielle Nash
OUR ADVISORS!
Dr. Brian Ingalls
Dr. Trevor Charles
Dr. Barb Moffatt
Dr. Marc Aucoin
Dr. Josh Neufeld
Dr. Matthew Scott
UNIVERSITY OF WATERLOO
University of Waterloo was founded in 1957 and has grown to accommodate 30,000 undergraduate and graduate students, and has become Canada’s leading university in comprehensive learning. Also, the university has consistently been voted as the most innovative, most likely to produce the leaders of tomorrow, and best overall University in Canada for over 18 years (according to Maclean’s Magazine). Waterloo’s reputation is however based on its excellent and pioneering co-op program which offers students a balance of work and school on a per term basis, making it a unique learning experience. The city of Waterloo has recognized University of Waterloo and its students, by meeting its demands in terms of funding and involvement. The University has also opened up two new campuses; the pharmacy building, and the joint McMaster medical building in Kitchener, as well as the architecture building in Cambridge, contributing to not only the city of waterloo but the whole Grand River area.
WATERLOO - KITCHENER COMMUNITY
City of Waterloo mainly revolves around the two universities: University of Waterloo and Laurier University. Waterloo is surrounded by Kitchener and thus, the two cities are known as the twin cities, also referred to as Kitchener - Waterloo. The population of the city of Waterloo is always fluctuating due to temporary residents at Waterloo’s two universities. Total population in 2009 was recorded to be 121, 700; approximately 20,000 of which were temporary post-secondary students. Due to its small size, people in the past have tried to merge the two cities together but have been unsuccessful. As of today, both cities have their own identity and their own separate city governments.
Parts List
UW's parts for 2010.
BBa_K359002 -    Signalling       - Agr quorum sensing sensor/generator, FepA pore, with P2 + reporter
BBa_K359003 -    Reporter         - Agr P2 with RFP
BBa_K359006 -    Intermediate     - AIP sensor and generator
S. aureus oligopeptide-based quorum sensing system into a BioBrick-compatible signalling mechanism including the AIP sensor and generator of AIP (auto-inducer peptide).
BBa_K359007 -    Intermediate     - AIP sensor with FepA pore
This part includes the S.aureus Agr quorum sensing sensor which recognizes the oligopeptide. The FepA pore allows for the oligopeptide to diffuse through the outer-membrane
BBa_K359008 -    Intermediate     - AIP generator and sensor, with an added RFP reporter
Generates an AIP signal, senses it and produces an RFP signal with an Excitation peak: 584 nm and Emission peak: 607 nm
BBa_K359009 -    Intermediate     - AIP sensor and consequent RFP reporter; contains FepA permeability pore
This part includes the S.aureus Agr quorum sensing sensor which recognizes the oligopeptide. The FepA pore allows for the oligopeptide to diffuse through the outer-membrane. Consequent RFP signal is produced, with an Excitation peak: 584 nm and Emission peak: 607 nm
BBa_K359201 -    Plasmid Backbone - pSB3K3-S-rbsRFP-P
Lab
Notebook
Safety
Lab Notebook 2010
Monday, May 10
We're still working on 2009 project. We just found out that the donor strain sequence lacks att site and oriT. Inoculated liquid broth in order to obtain colonies for conjugation. The tubes were as follows:
1.DH5α λ rifr attB CT from old patch on rif25, amp100, tet10 (aka RAT)
2.DH5α rifr attB CT from old patch on RAT
Also, we inoculated the Landing Pad Strain cells into liquid broth.
Tuesday, May 11
Tubes incubated from yesterday were taken out. All showed no growth. All tubes are to be re-innoculated. Tubes #1 and # 2 were re-done using an old plate and different colonies of the old patch.
Several dilutions were prepared in order to determine which concentration is best for spread-plating. Dilutions prepared were as follows: e-2, e-4, e-5, e-6, e-7, e-8
Prepared liquid and solid media, and learned how to use autoclave.
Made more plates (Rif/Amp/Tet/Sm)
Wednesday, May 12
All tubes incubated yesterday showed growth. Miniprepped the samples.
As relating to the dilutions, results were as follows
e-8 = very little growth e-7 = some growth e-6 = highly populated
Conjugation experiment was attempted (tri-parental mating). This included Donor (DH5alpha), Recipient (MM294A), Helper(MT616)
Inoculated Landing Pad Strain from frozen stock.
Thursday, May 13
We prepared X-alpha-gal plates and streaked MM294A on them. We would be expecting white.rosy colonies. Nanodropped the samples which were minirepped yesterday (incubations of plates listed on Monday, May 10.)
Friday, May 14
The x-alpha-gal plates showed no good results - none of the colonies were rosy.
Learned how to do REs digestions properly (did several examples on paper, and in the lab.)
Monday, May 17
Planned how to do a restriction digest followed by ligation procedure for the donor strain. Discussed the different purposes for restriction digests: diagnostic (to check if you have the plasmid you expect) and for cloning (to isolate desired DNA).
Tuesday, May 18
LPS Analysis W.R.T SacB : LPS was cut with with EcorI & run on 0.8% agarose gel for 45 minutes at 90V to analyze for truncated sacB
Results: Three distinct bands were found. Therefore, the truncated sacB was not present in the LPS. If sacB were present four bands would have appeared after digestion.
Wednesday, May 19
PCR of DS Right Plank was performed: The right flank was inserted into DS right flank=Pst site, attB, oriT, MPH 1103
PFB 9009 was mini prepped(LPS) and nanodropped.
Nanodrop results of PFB 9009 Mini - prep:
SAMPLE------------>Concentration(ng/µL)----->260/280
PFB9009-1--------->21.4.....................>1.75
PFB9009-2--------->24.0.....................>1.91
PFB9009-3--------->12.9.....................>1.58
Thursday, May 20
Performed a double digest of PFB 9009-2 with FSPI & NdeI to excise and extract rouge transposase gene. Gel extracted the LPS and DS fragements LPS: 0.125g DS: 0.0947g
Dissolved in Buffer to obtain 500µL of LPS and 378µL of DS and extracted the DNA following the protocol from "forEZ-10" kit.
Friday, May 21
Performed a ligation reaction of LPS to circularize it Nanodropped the samples. Results recorded below:
LPS Concentration: 2.06ng/µL 260/280: 5.6 DNA: 224ng
DS Concentration: 1.95ng/µL 260/280: 7.1 DNA: 284ng
The solutions were speed-vaced on high for 10 mins to increase concentration.
Monday, May 24
Prepared agar plates and agar bottles
organized everything in the lab and the freezer so that it was easily accessible
Construction tree was revised and updated
Tuesday, May 25
Planned and carried out ligation of DS right flank + DS2(blunt ends) Started transformation of LPS and DS Note: The desktop cooler has been left out for undefined amount of time (approx. 3 hours). However, it should be okay.
Wednesday, May 26
Transformation results: LPS did not grow. Therefore, reverted to double digest.
Started double digest of LPS with PspI and NdeI
Repeated transformation (using more DNA this time)
Thursday, May 27
Transformation results (pFB10, Km + Sm):
negative control: no growth
Positive control: lots of small colonies (approx. 70)
Therefore transformation was sucessful.
Friday, May 28
Performed a diagnostic digest on DS with PstI + NsPI
Results: Failed. NO 274 fragment was present but 120 fragment was present.
This will be attempted again.
Troubleshooting: Ladder will be added in higher concentration and no loading dye will be added for better resolution.
Integrase strain was inoculated into Tc10 LB broth
Monday, May 31
Miniprepped and nanodropped DS
Results:
SAMPLE 1
CONENTRATION: 110.14ng/µL
260/280: 2.02
SAMPLE 2
CONENTRATION:127.85µL
260/280: 1.06
SAMPLE 3
CONENTRATION:105.89ng/µL
260/280: 1.07
SAMPLE 4
CONENTRATION:189.74ng/µL
260/280: 2.03
SAMPLE 5
CONENTRATION:152.01ng/µL
260/280: 1.05
Planned out next day's activities regarding DS
Tuesday, June 1
Retried to do a diagnostic gel of DS with PstI & NstI with a fragment known to have approx. 100bp(old DS with no right flank) FAILED. No alkaline phosphotase was added to DS when digested with restriction enzymes
Started a RE digest of mini-prep DS plasmid to be re-ligated with right flank construct. Nanodropped the digested right flank sample.
Concentration: 7.8ng/µL
260/280: 4.29
The concentration indicates that we can not use this sample because majority is not DNA.
Therefore, must PCR the right flank again using the "spring 2010" PCR program
PCR products were digested with Mph1103 & PstI
DS was digested with right flank with Mph1103(AvaIII) & PstI to prep for insertion into DS plasmid.
Two bands were present for positive controls#2(both samples). These bands were extracted and purified.
TUBE WEIGHTS
Sample 1:0.1633g
Sample 2:1.0477g (did not show bands).
NANODROP RESULTS
Insert
Concentration: 15.9ng/µL
260/280: 1.61
Vector
Concentration: 8.4
260/280: 2.23
Wednesday, June 2
Digested DS vector with PstI so that right flank can be inserted. Added SAP to avoid re-ligation with itself.
Ligated DS vector with Right flank insert
Transformed newly improved DS(hoepfully, with right flank) into component DH5α
Discussed possible ligation results
Thursday, June 3
Innoculated S. aureus into LB media for the purpose of collecting AIP sipernatant
After collecting staph supernatant wanted to test for the effect of adding supernatant to E.coli cultures
Friday, June 4
Innoculated DH5α into AIP supernatant
made 3 x 5mL tubes of AIP supernatant
innoculated with DH5α (strain box #1, #41)
Results from transformation(previous day): No growth of LPS 10 on Km/Sm plates. No clear red color on Rig/Km but good growth observed. Left in the 37˚C incubator over the weekend.
Monday. June 7
Performed another attempt at digestion of diagnostic Nsp/Pst on DS -- FAILED
Researched and wrote protocols for experiments to be carried out within the week
Learned how to use DNA from kit
Tuesday, June 8
A new batch of competent cells were made.
Their competency will be tested by transforming DS in.
Wednesday, June 9
The streaks of putative LPS 10 still showed no red.
Sensetivity was checked for melibrose. The same patches were streaked(#5,8,26) on minimal melibrose.
If anything grows: ALARM! Also, there are suspicious parts on #5 streak, worth restreaking onto another.
Thursday, June 10
Rif/Km plates were made
Rif stock was made by dissolving 0.25g into 10mL DM50
Prepared everything needed for outreach event
pJET-Right Flank for DS (Donor Strain) has been miniprepped, put in -20C freezer. The streak plates of LPS10 from various sources still show no red colour. Could cutting out the fragment (pFB09->pFB10) messed something up?
Friday, June 11
Tried to cut and gel extract the fragment. Failed, probably due to low Gel Red dye. Will be getting more dye soon. Can try scraping the walls and adding all of it to the gel Monday.
Monday, June 14
Tried to do PstI + Mph1103I digestion again. Failed several times.
Wednesday will do 10uL diagnostics with BglII on the remaining pJET minipreps; looking for ~200bp fragment. Also will do spectrophotometry of DH5-alpha pre-DS and DH5-alpha blank hourly at 600 and 640 nm.
Tuesday, June 15
Updated construction tree. Drew one out and many copies were made.
Attended a club meeting and updated all volunteers with what was happening.
Engineers were given a short biology course to help understand the project better
Wednesday, June 16
Checked concept of absorption RFP estimation
inoculated with loop (approximate the same aount) HiRFP = pSB1A2 - BBa_K093012 (+ L. flank, can be neglected)(J23118 driven E1010)
The purpose of this was to check for consistency of A589/A640 on DH5α and useage of that to check consistency of A584-blank / A640
Thursday, June 17
Gel extracted the fresh right flank PCR product (yes, AGAIN). Tomorrow will check on gel for presence along with "control PCR product" from cloneJET kit.
The parts in DH5 have been inoculated into liquid media to be miniprepped tomorrow.
The RFP measurements on cuvete absorption spec Ultrospec 2000 suggest that sensitivity is too low to pick out RFP.
Should try on Bioscreen C plate reader.
Planning the assembly should be started tomorrow.
Friday, June 18
Parts miniprepped. right flank PCR ligated into pJET and transformed. To be inoculated to liquid Monday.
Monday, June 21
created a final list of parts to be transformed
Do not have: K206000, K206001, J23151 and J23150
All the DNA was transformed into DH5α competent cells
Results of transformation: most plates grew but with very few small colonies. The plates were put back in the incubator for a few hours and checked later. All had at least a few colonies.
These were innoculated into liquid LB and incubated overnight at 37˚C
Tuesday, June 22
Was Done:
Tried diagnostic with Mph1103I alone and PstI alone. The results are here. Wells are as follow
1-2 PstI
2-1 PstI
3-1 PstI
Ladder Fermentas 1kb plus
1-2 Mph1103I
2-1 Mph1103I
3-1 Mph1103I
The evidence hints on presence of PstI site on pJET; without first well showing it is quite non-conclusive.
Red clone of LPS10 strain was found in the fridge and confirmed by X-gal to be MM294A as DH3-alpha is lacZ-. Hooray.
Was patched on Kan and Kan+Strept 48-numbered background plates by Corey and Diana.
To Be Done:
Talk to people who use Bioscreen C plate reader.
See if pJET sequence has PstI site(s).
Cut out right flank with PstI and Mph1103I and run gel preparatively, gel extract the fragment of ~150bp.
Ligate that fragment into pDS (donor strain plasmid) cut with PstI+SAP.
Transform that ligation into DH5-alpha.
From LPS10 patch plates pick the colonies that did not grow on strept or grew poorly, streak on Kan and check on sucrose media (just streak) with empty DH5-alpha as control.
Wednesday, June 23
Was done:
Looked at pJET for PstI. Yes there is one, at position 5. This means that in Mph1103I+PstI digestion we expect fragments of of 153bp (the one we want) and 371bp (the one we don't care about).
Spread 70 µL of red LPS10 on X-alpha-gal+rif+kan.
Streaked red LPS10 on sucrose LB side by side with empty DH5-alpha
no streptomycin sensitive patches detected
Mph1103I PstI digestion preparative was done, gel was stained overnight.
pDS was linearized with PstI
Thursday, June 24
LPS10: sucrose sensitivity not detected; X-alpha-gal plate is mostly blue, only few white colonies, without a sign of red. Should leave x plate in the fridge and hope for colour
Two attempts to cut out right flank with Mph1103I and PstI were perfmored. Both attempts are to be quantified using nanodrop and pipette, and concentrated on the Speedvac, requantified with pipette.
Friday, June 25
Second attempt of Mph-Pst cut out has been ligated into its proper place in pDS. incubate and pray. Tschüs.
Monday, June 28
PstI+SpeI digest (wells 4 and 5) suggest ligation failure of right flank into pDS.
Tuesday, June 29
Transformed J23107 (constitutive promoter) into DH5α
Results: colonies grew, some rosie
These were inoculated and miniprepped
All parts were sent out for sequencing
Wednesday, June 30
Digested RF with PstI.
Result: No RF fragment, but PstI- PstI fragment is there
Thursday, July 1
Attended board meeting and updated everyone on what was going on
Made LB plates
Friday, July 2
Autoclaved 0.8% saline and swabs
make 70% ethanol
stocked up pipettes
Monday, July 5
Inoculated J23107 in LB and incubated overnight in 37C
Inoculated received parts( K206001, I746201, I746001, K20600 onto Amp plates to incubate overnight in the 37C. It will be inoculated in LB tommorow
Tuesday, July 6
inoculated the cultures into LB and will mini-prep tommorow
Wednesday, July 7
Mini-prepped the parts that were inoculated yesterday
These were to be sequenced
Made glycerol stock of parts to be sequenced
Thursday, July 8
pIET - rf has been transformed and grew up to nearly lawn coverage o/N. 50uL DH5a comp + 5uL supercoiled plasmid were used. The negative control was clear. The transformants were inoculated into 16 LB tubes + Amp, without purification.
attempted to assemble the measkit.
Friday, July 9
Nanodropped the following samples:
J23107 - COnstitutive promoter
I74601 - AIP sensor
746001 - AIP generator
I746201 - FepA
I7466104 - AgrA P
K206000 - pBAD strong
K2066001 - pBAD weak
I13458 - pBAD
I13453 - AraC
J23102 - constitutive promoter
E1010 - RFP CDS
J23101 - Constitutive promoter
I13507 - RFP + RBS +TT
Monday, July 12
Made frozen stock of all the cultures from friday
Made DMSO stocks : wanted 7% DMSO and 14% LB --> Made it by mixing 1.68mL of DMSO and 10.32LB
Wednesday, July 13
miniprepped all parts from strain box #2 and nanodropped them. Results are listed below:
I746104 : 100.1ng/uL , 2.16
J23107 : 149.3ng/uL , 2.01
I746201 : 139.7ng/uL , 2.00
I746001 : 185.9ng/uL , 1.96
I746101 : 104.3ng/uL , 2.08
J23102 : 319.4ng/uL , 1.94
I13453 : 139.2ng/uL , 2.01
I13507: 61.0ng/uL , 2.10
K0206000 : 155.1ng/uL , 1.60
I13958 : 155.7ng/uL , 2.03
J23101 : 262.8ng/uL , 2.00
E1010 : 99.1ng/uL , 2.05
Thursday, July 14
Added J23107 promoter to I746101 and I746001 (AIP sensor and AIP generator)
Ran gel electrophesis and gel extracted J23107 ( approx. 3kB fragment)
Cut J23107 with SpeI and PstI & I746101 and I746001 with XbaI and PstI and ligated into vector
Transformed and miniprepped.
Friday, July 15
Strain list made public with the link
I20260 on pSB3K3 inocluated into Kanamycin LB
strain list is updated with 2010 parts added yesterday
J23101 on J61002 from frozen stock is inoculated into Ampicilin LB
Transformed I0500 from 2010/plate3/20B into DH5-alpha, plated on Kanamycin LB Agar
inoculated all recent parts on strain list (except E1010) to fill up plasmid -20 stocks
The primary constructs of AIP sender and reciever has been cut, extracted and set ligated (O/N, 16 °C) to J23107.
Monday, July 18
Inocluated pSB2K3-I0500 into Kan-LB
Tuesday, July 19
Performed preparative digestions of J23101 cut with EcoRI + PstI & PSB3K3 also with EcoRI + PstI
These parts were ran on gel and gel extracted:
J23101 : 0.0515g
PSB3K3 : 0.0259g
Results were not that great. There was not enough DNA and bands were too faint. Also, ladder ran a little funny, may have been overloaded. We will need to redigest. This time a diagnostic gel will be run to see what went wrong.
Wednesday, July 20
LPS: Cveta is looking at the pFB10 sequence
Dan is looking at Parts Sequences
DS: hunting for 153bp fragment on larger scale with new batch of Mph1103I enzyme
Quantification: constructing pSB3K3-J23101-RFP by E+P digest
Assembly: Stage 1 is assembled but not checked by diagnostic digest. Broth inoculated for miniprep, growing.
Thursday, July 21
DS ligation: three attempts were made with different tubes of competent cells each. Attempts:
-ve ctrl failed
-ve ctrl ok, many colonies still on the transformant plate. 4 larger colonies were pathced and broth-inoculated.
3rd attempt made today, spread on amp plate, growing
pSB3K3-J23101-RFP construct inoculated from streak plate into broth. two streakplates are in the fridge.
gel was ran. wells:
Ladder (Fermentas 1kb plus, as always)
Preparative digest of J23107+I746101 construct
Same
Undigested control for preprative digest above
Ladder
Diagnostic digest of J23107+I746101 construct
Undigested control for diagnostic above
pSB2K3-I0500 (part from the kit)
wells 5,6,7 and 8 pic is here, the other half of the gel was exposed to UV only to cut out the top band from well 2 and 3 to be gel-extracted.
pSB3K3-J23101-RFP was already inoculated into broth by Leah yesterday. Dan miniprepped that and the remaining culture of I0500. George made frozen stock #106 out of pSB3K3-J23101-RFP
This is the diagnostic gel we ran on the vector (J23107 + I746001). The bands are as follows:
ladder
digest with (EcoRI and XbaI)
Negative control (no restriction enzymes)
Digest again (left over from preparative, since I had to increase the volume and it didnt all fit into the well)
The preparative looked the same. respective band excised+extracted.
Friday, July 22
DS lig 3 failed (abundant growth on both plates
miniprepped DS lig 2 (4 of them) and another culture of 3K3 RFP J23101.
Tuesday, July 27
Ran the /NspI+PstI digestions on gel (2uL + 0.5uL PstI + 0.5uL NspI + 2uL FD buff + 15 uL water)
Ladder
DS clone 1
DS clone 2
DS clone 3
DS clone 4
pSB1A2 (I746104 part)
Prepped parts list has been updated. The sequencing confirmations are still to come from Dan Barlow.
BW27783 recieved from UBC, inocluated into liquid culture.
Wednesday, July 28
Made frozen stock of BW27783 #107
digestion of 3K3-J23101-RFP preparative: (7uL DNA + 1uL EcoRI + 1 uL SpeI + 2uL FD Green Buff. + 9 uL water); diag (3 uL DNA + 0.5 EcoRI + 0.5uL SpeI + 1uL FD Green Buff. + 5uL water) pic of diag
digestion of 2K3-I0500 preparative: (7uL DNA + 1uL EcoRI + 1 uL SpeI + 2uL FD Green Buff. + 9 uL water); diag (3 uL DNA + 0.5 EcoRI + 0.5uL SpeI + 1uL FD Green Buff. + 5uL water) pic of diag the lower, 1200bp band is the target
Thursday, July 29
pFB10 inoculated into broth, Km+Sm
pFB10 streaked on Sm and Sm+Suc10%.
Transformed K359201-I0500 ligation, plated on Km agar.
FepA-Sensor construct was being miniprepped by Leah
Sensor-Generator construct inoculated into LB.
Friday, July 30
Planned for the upcoming week
Worked on the construction tree
Prepared for Outreach lab event
Tuesday, August 3
Sucrose sensitivity of pFB10 confirmed.
inoculated broth with 6 different colonies of K359201-I0500.
Finding source of microplates. According to this, the clear plates are even better than clear-bottom-black ones.
K201-I500 construct miniprepped, digested E+P, ran on gel:
Sample 1-1
Sample 1-2
Sample 2-1
Sample 2-2
Sample 3-1
Sample 3-2
Ladder
linky
matches expected results (from ApE) ideally.
Wednesday, August 4
21 plates for ESQ activity poured
~10mL of 10% L-arabinose stock created from Charles lab supplies (used 1g). Used protocol from OpenWetWare for creating stock.
Thursday, August 5
Made 0.2% arabinose LB agar plates.
Streaked K359201-I0500 construct on an Ara-LB plate.
Friday, August 6
DH5-allpha with K359201-I0500 shows no red phenotype.
Transformed ligation of P2-RFP (I746104+I13507)
Monday, August 9 - Friday, August 13
All attempts to transform that P2-RFP has failed
Autoclaved
4 baffled fasks with 50mL LB (freshly prepared, measured out with graduated cylinder)
two jars of µfuge tubes
100mL 0.1M MgCl2
2 GSA bottles
LPS: Inoculated 5 different cultures of pFB9010 into Km/Sm LB liquid media, as well as into blank LB media. These are needed in order to continue with the Landing Pad Project from 2009. The cultures will be used to make frozen stocks of the plasmid, as well as minipreps for future work. Results will be seen tomorrow. Previous attempts to grow the pFB9010 plasmid containing cells have been unsuccessful – although the inoculation is from Km/Sm patches, no growth has previous been observed in Km/Sm liquid cultures.
Amp plates were tested by using two plates – one Amp and the other Blank. On them were plated MT616 and DS plasmid. DS plasmid is Amp resistant, MT616 is not. Results to be seen tomorrow.
Monday, August 16
Missed the exponential phase for the comp cell prep, deferred till tomorrow.
LPS: There was no growth in the Km/Sm liquid cultures but there was growth in the blank cultures.
LPS: Two plates of pFB9010 have been recovered. In order to solve ambiguity, liquid cultures of Km and Sm were inoculated from plate 1 (most used). Also, plates of Km, Sm, Km/Sm were streaked with each of the 5 patches from each of plates 1 and 2 (where plate 2 is Sm only for some reason…although it should be both). This should solve which of the resistances is not functional. Results tomorrow.
Tuesday, August 17
Assembly: Nonodropping of both J23107 and I13507 stocks was performed. Digestion of J23107 with PstI and SpeI was unsuccessful. The 0.8% Agarose gel showed that the plasmid was not cut, since the uncut control beside it yielded the same bands.
Assembly: Digestion of I13507 with PstI and XbaI was successful, although the bands were slightly off as related to the ladder. Suggestion was made that Gene Ladder Plus should be ran as 2uL of solution + 8ul of water. Also, it has ran fast before, so beware. Bands expected are 883 and 2057 (from I13507 in pSB1A2 plasmid). 883 was taken (appeared at around 1000, however), since it is an insert. The bands were gel purified and stored at -20 C.
August 18
digested and gel extracted I13453/E+X and I13458/E+S
made 30 plates for ESQ activity
performed half of competent cell procedure, let incubate on ice in 4°C O/N
Results from yesterday were as follow:
Growth from both pFB9010 plates on Km and Sm, but not on Km/Sm.
Liquid cultures showed growth in Km, but not in Sm.
Not sure what this means at the moment, so further tests need to be done.
Transform remainders of pFB9010 stored at -20 C (nanodrop, transform)
Try digestion of J23107 again – did not work AGAIN! Will attempt another time but with another lab’s REs, SAP and buffer.
If digestion works, ligate with I13507. – did not work =(
August 19
Digestion of J23107 was re-done using REs, Buffer and SAP from another lab (except for SpeI, since it was not available from anyone else.) This did not work either.
We think that maybe there is something wrong with the J23107 miniprepped stock in the -20C fridge, so we will make a new one. Took J23107 from frozen stock and streaked onto an Amp100 plate, as well as inoculated into Amp100 liquid culture. This will be miniprepped tomorrow.
Set ligation of I13458+I13453
August 20
Miniprepped J23107
Nanodropped and digested – SUCCESSFUL!!!!
Gel Purified and Nanodropped
Concentration according to nanodropping is ~ 1ng/ul = way too low.
More J23107 and I13507 will be inoculated over the weekend, and will be minirepped and digested on Monday.
Regarding LPS, took a Km20 plate which yielded good results, and inoculated it into the following liquid cultures.
Km20, Km10, Km20/Sm100, Km20/Sm50,Km10/Sm100,Km10/Sm50, Sm50, Sm100.
This was done to ensure that the concentration of Sm or Km is not too high in the liquid cultures (mass transfer accounts for erroneously high concentration in comparison to solid agar on plates.)
August 22
Results from inoculations of various Km/Sm concentrations were as follows:
Contents  Results
Km10  +++ (red precipitate on the bottom)
Km20  +++ (red precipitate on the bottom)
Sm50  +++(very little white-ish red precipitate on the bottom)
Sm100  -/+ (looks little bit cloudier than regular LB, but that could be due to re-suspension of the patches, since it was taken from a patch, not from a single colony)
Km10/Sm50  +++ (red precipitate on the bottom)
Km10/Sm100  +++ (no red precipitate on the bottom)
Km20/Sm50  -/+ (looks little bit cloudier than regular LB, but that could be due to re-suspension of the patches, since it was taken from a patch, not from a single colony)
Km20/Sm100  -/+ (looks little bit cloudier than regular LB, but that could be due to re-suspension of the patches, since it was taken from a patch, not from a single colony)
This probably means that while the resistances are present in the genome, they are not expressed very strongly. Will attempt to grow colonies on plates with reduced Sm/Km. After, will attempt triparental mating again.
innoculated J23107 into 5 liquid Amp 100 cultures, ready to be miniprepped tomorrow (Monday) prepared Km10/Sm50, Km10/Sm100, Km20/Sm50, Km20/Sm100 plates and streaked pFB9010 on them. Results to be seen tomorrow. This was done, since liquid versus solid media has been giving varying results with this project.
August 23
There is no growth on any of the pFB9010 plates (with varying conentrations of Km and Sm)
Miniprepped and nanodropped J23107 (concentrations are written on the tubes)
Digested J23107 with PstI and SpeI
Ran diagnostic + preparative gel
Gel extracted
Digested I13507 with PstI and XbaI
Ran diagnostic + preparative gel
August 25
Digestion of I13507 with PstI and XbaI could not be performed yesterday, so will be performed today
Ran diagnostic + preparative gel
gel extracted
August 26
Concentrated samples of digested J23107 (digestion was done with SAP) and I13507.
Performed ligation (added ATP)
August 27
Ligation was ran on gel (big mistake....should have just went along with transformation right away, since results could not be seen on the gel)
August 30
Performed digest of J23107 again, except used FD Buffer instead of Green FD Buffer
Ran on gel
Sample appears as a smudge (potential contamination)
Performed digestion of I13507 again, using FD buffer instead of FD buffer green.
Ran on gel
Could not see digest.
Transformed ligation of “58+53” (quantification work)
Tuesday, August 31 - Friday, September 3
Performed digest of J23107 and I13507 AGAIN! This time, SAP was not added to the vector digest (usually, if it is a double digest, you don’t really need to add SAP. It helps in the case that one of the REs does not cut). WORKED!!!
Transformed the ligation of J23107 and I13507
Using 50µL of competent cells, 20µL of ligation reaction and 15µL of CaCl2(100mM solution)
September 6 - September 10
Orientation week
September 13 - September, 17
First Lab meeting for Fall 2010
created a new iGEM active lab & a new thread of e-mail to keep everyone updated on lab work done daily
Innoculated I746104 and I13507 into liquid broth (3 different ones)
Monday, September 20 - Wednesday, September 22
Planned for Jamboree
Assessed previous work done and decided and what will be done for the competition
Organized a new schedule to get the work done accordingly for the jamboree
Miniprepped I746104 and I13507
September 23
1. Nanodropped samples of I746104 and I13507. Results are as follow:
I13507 - [102 ng/ul] 260/280 = 1.87
I746104 (1) - [168.8 ng/ul] 260/280 = 1.81
I746104 (2) - [158.8 ng/ul] 260/280 = 1.82
Calculations for digestions: need 617.9 ng of insert, 300ng of vector.
1. Digests:
I746104 with SpeI and PstI 3ul I746104 1ul Pst 1ul Spe 2ul FD Green Buffer 12ul Nucl. Free Water 1ul SAP
This was done in duplicate. Control: 1.5 ul I746104 without REs; 10uL reaction.
I13507 with XbaI and PstI 7ul I13057 1uL PstI 1uL XbaI 2uL FD Buffer Green 9uL Nucl. free water
control: 3ul of DNA, no REs, 10ul reaction.
3. Left to incubate for an hour @ 37C
4. Gel purified using protocol for purification from enzymatic reactions (used 60uL of Elution buffer for non-controls, 30ul for controls)
5. Speed vac for 10 minutes on high.
6. Nanodropped
I746104 (1) - 9.7 ng/uL
I746104 (2) - 8.7 ng/uL
I746104 (3) - 5.5 ng/uL
I13507 (1) - 14.1 ng/uL
I13507 (2) - 21.3 ng/uL
I13507 (contaol) - 14.0 ng/uL
7. Ligation
Vector 8uL
Insert 6.5 uL
10X lig buff 2.5uL
ligase 1.5 uL
ATP 1uL
Nucl. free water 6
total: 25.5 uL
The two I13507 were mixed, and 6.5 uL were taken from each into the 3 aliquots of I746104. Why is there 3 aliquots, when there was only 2 to begin with, you ask? Because Cveta lose the label on one of them, and couldn't tell which were the actual digests and which was the control.
To fix this grave mistake:Cveta saved 2uL from each of the three tubes so we can run a gel tomorrow and find out which is which. If you are doing anything with the tubes, please ensure the labelling does not change. Left to ligate at 16C in PCR machine.
September 24
Ligation of K359003 and I13507 was completed, plated on Amp100 and incubated overnight at 37˚C
September 25
Informed the lab volunteers of work done over the summer
Shared the following with them:
The Handbook
In Brochure form (please print with Acrobat Reader in the "short" duplex format, or otherwise I am not responsible for wasted paper)
September 26
checked the I746104 digestions that went into ligation by gel, #1 is apparently a control.
Transformed and plated the ligation (putative 003).
September 27
Incubated Amp100 K359003 plates grew well. Some bright pink colonies were visible, others were white.
Two aliquots of ligation #2 and #3 were prepared
3 pink colonies from each plate were inoculated from each plate into separate liquid broths of Amp50.
September 28
Miniprepped the 6 separate tubes, nanodropped, written concentration on side of the tube and stuck in second green box (the one labeled "2010 Parts Kit 2", disregard the meaning of that name), so it is easier to find.
September 29
6 clones were digested using the following gel layout:
1: Ladder (2uL + 8uL MQ)
2: clone #1 /E+P
3: clone #2 /E+P
4: clone #3 /E+P
5: clone #4 /E+P
6: clone #5 /E+P
7: clone #6 /E+P
8: I13507 /E+P
October 1
Attempted to make K359009, K359008, K359008 (3A), and K359009 (3A)
3A was not sucessful
K359008 : obtained from K359006 and K359003
K359009 : obtained from K359007 and K359003
Samples were digested, ligated and incubated for over 2 hours at 37˚C
All 4 samples were purified directly from the enzymatic reaction
Samples were nanodropped
SAMPLE ---> CONCENTRATION---> 260/280
K359003 --> 13.5ng/µL ---> 1.57
K359006 --> 13.8ng/µL ---> 1.90
K359003 --> 7.4ng/µL ---> 1.21
K359007 --> 45.7ng/µL ---> 1.78
October 2
1. Digests
Sad news of the day: SpeI is not heat inactivatable (sp?). Some NEB heat inactivatable SpeI was found, so we shall try that one later.
a. Digests # 1 to make K359008 003 cut with XbaI and EcoRI 006 cut with SpeI and EcoRI
b. Digest # 2 to make K359009 003 cut with XbaI and EcoRI 007 cut with SpeI and EcoRI
2. Ligations The above digests were set for ligation. Two aliquots were made for each (but there was barely anything for the second one.) A -ve and +ve control were also included, where -ve = no DNA, +ve = Amp resistant plasmid lying around. Left tubes in PCR machine overnight at 16C.
October 3
Transformed and plated the tubes from yesterday on Amp. They were left in the 37C incubator overnight.
Reinnoculated DB 3.1
October 4
Researched what to do if we ran out of linearized plasmid pSB1c3
http://partsregistry.org/Help:Protocols/Linearized_Plasmid_Backbones
http://partsregistry.org/Help:Spring_2010_DNA_distribution
Decided to leave 3A alone for now
http://openwetware.org/wiki/Synthetic_Biology:BioBricks/3A_assembly
October 5
Checked on the plates from Saturday:
-ve ctrl : no growth
+ve ctrl: no growth
009-1: contaminated
009-2: some rosy colonies and white ones. inoculated 5 rosy ones into tubes, but they were so close to each other, I also made one streak plate as backup
008-1: few rosy (and white) colonies
008-2: (white), rosy and bright red. went for bright red, 5 tubes, 2 streak plates
DB3.1 reinoculated (again)
LEFT:
11 tubes on shaker
3 plates in fridge
3 plates in incubator
October 6
Transformed the DB3.1 :
http://partsregistry.org/Help:Spring_2010_DNA_distribution pSB1C3-BBa_P1010 from Spring 2009 Distribution Plate 1 Well 5E
2009 plate 1 was obtained,
10uL MQ was pipetted into Well 5E and all was transfered to a microfuge tube. *labelled: 2009-1-5E, pSB1C3-P1010 1mL of DB3.1 culture was pipetted into microfuge tube. Spun down (~12,000rpm/2 min), decanted and kept on ice. This was repeated twice with 1mL of CaCl2 but spun down in the fridge with the glass door.
50uL of CaCl2 + 2uL of DNA from the 5E well were added and tube was kept on ice for 30 mintues.
Transformation was proceeded as would normally be done using the protocol.
October 7
Transformation Results: FAILED. No growth on positive control or negative control.
Reinnoculated DB3.1 culture. Transformation will be reattempted.
October 8
Attempted transformation again using same protocol as before.
DIGESTION OF 008 & 009 PERFORMED AS INDICATED BELOW:
008-4,5(older prep),7(newer prep) will be digested E+P and ran on gel :
Ladder
008-4 /E+P
008-5 /E+P
008-7 /E+P
006 /E+P
008-4 undigested
008-5 undigested
008-7 undigested
Monday, October 11
Transformation FAILED. This will not be redone. To be discussed in weekly meeting on thursday.
009-5 was speedvaced,digested with E+P and ran on gel
Worked on updating the Wiki and presentation
Tuesday, October 12
Analyzed results from gel on friday:
Ladder: yeah.
008-4 /E+P: ladder contamination?
008-5 /E+P:
008-7 /E+P: the most probable, don't ask me what is in the lowest band
006 /E+P: overkill amount, had to have lower exposure to view
008-5 undigested: looks fine for undigested
009-5 E+P: see above
007 /E+P: overkill
Wednesday, October 13
Worked to complete Human practices section
Individual photos of active team members were gathered with a small blurb about each person for wiki
Worked on updating Quantification for the lab wiki & the SVG tree
Thursday, October 14
Board meeting: updated on work done in all sections, upcoming work plans made
Weekly Lab meeting: shared information from board meeting with the rest of the members and accordindly, created an agenda:
Wiki: all due Saturday morning by 12 pm (Oct 23)
Lab:
Some intro (very general)
Quantification: (by Monday, Oct 18)
Construction tree- SVG (by Monday, Oct 18)
Monday, October 18
Team photos taken
Looked through our fridge and freezer. Put all relevant microfuge tubes into the "2010 Parts Kit 2" green box.
Could not find 006, nor 007 but did find 008 and 009 and so,inoculated 3 tubes for each.
Digested parts 003, 006, 007, 008, 009 and pSB1C3 with E + P. Purified these digests and then left them over night in part + pSB1C3 ligations. Also, made 3 inoculations of pSB1C3 and 2 inoculations each of 009/008 (total of 7). These are all on the shaker.
Tuesday, October 19
Miniprepped pSB1C3/008/009 (9 tubes in total) from the incubator.This is required in the case that the transformations are unsuccessful,so that we can start over.
Nanodropped them and results are as follows:
pSB1C3 - 1
CONCENTRATION: 48.7ng/uL
260/280 1.96
pSB1C3 - 2
CONCENTRATION: 185ng/uL
260/280 1.91
pSB1C3 - 3
CONCENTRATION: 58.4ng/uL
260/280 1.93
008-1
CONCENTRATION: 59.8ng/uL
260/280 1.75
008-2
CONCENTRATION: 131.7ng/uL
260/280 1.86
009-1
CONCENTRATION: 121.7ng/uL
260/280 1.95
009-2
CONCENTRATION: 209.0ng/uL
260/280 1.92
Transformed all 5 ligations, which were in the PCR machine, in DH5alpha on to Cm plates. Stored the remainder of the ligations in Parts Kit #2, green box in freezer.
Made media broth - 200mL, placed into 5ml tubes and autoclaved.
Wednesday, October 20
Ligated 006 into pSB1C3..? RESULTS: TO BE UPDATED
Cut out the RFP which is currently with the pSB1C3 (5 tubes labelled pSB1C3 contain that plasmid plus an RFP biobrick.
We need to cut out the RFP biobrick and leave just the plasmid for ourselves.
Need to do the insertions of the biobricks into the desired plasmid (pSB1C3).
Thursday, October 21
Worked to complete Human Practices section
Continued work on Jamboree Presentation. Work will be continued on the weekend.
streaked and inoculated into liquid media.
Friday, October 22
Miniprepped the inoculations from yesterday and nanodropped them. Results are as follows:
These need to digested and shipped off to Boston.
Still need have lots of work pendning for the presentation and other administrative work that will be worked on the weekend and carried over for next week.
SAFETY
Our project is not expected to raise any safety issues in terms of researcher, public or environmental safety. The health risks associated with the organisms used and the work completed could be classified as very low to moderate. None of the biobricks we submitted raise any safety concerns. In terms of biosafety rules to be followed, we would consider Biosafety Level 2 to be a sufficient indicator of safety requirements. This stems from the fact that the organisms involved are Escherichia coli and Staphilococcus aureus. However, in practical use (e.g., hospital setting), level 3 facilities would be required. This is to account for several factors, including the increased potential for highly pathogenic strains in the vicinity, as well as the very unlikely possibility of a pathogenic strain of Escherichia coli obtaining the AIP sensing characteristics of our StaphiScope model.  In terms of biosafety issues which could be useful for future iGEM competitions, the appearance of a desired biobrick into a pathogenic strain of bacteria would be the biggest concern. The potential for such an occurrence would be limited by Biosafety Level 2 rules, as described below.
Facilities, equipment, and procedures which are required to contain risk group 2 organisms at Level 2 have been met:
1.Laboratory separated from other activities
2.Biohazard sign
3.Room surfaces impervious and readily cleanable.
4.Equipment includes an autoclave
5.There is personal protective equipment which includes laboratory coats and gloves worn only in the laboratory.
All precautions with respect to recombinant DNA were observed:
1.All waste was autoclaved before being thrown away.
2.Researchers practiced aseptic technique, and frequent hand washing.
3.Bench surfaces were disinfected with ethanol.
Upload file Disclaimers Privacy policy My preferences Special pages
Retrieved from " http://2010.igem.org/Team:Waterloo "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
