<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Liliane_Bettencourt_Project_Population_counter_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Liliane Bettencourt/Project/Population counter/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="logo_left"><SPAN class="plainlinks"><A title="igem.org" href="https://2010.igem.org/"> </A></SPAN></DIV></DIV><DIV id="menu"><UL><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt">home</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project">projects</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Parts">parts</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Team">team</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Notebook">notebook</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Safety">safety</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Brainstorming">Brainstorming</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Acknowledgements">acknowledgements</A></LI></UL></DIV><DIV id="content"><DIV id="frame"><P style="display:block;"><FONT size="4">Population counter</FONT></P><DIV class="submenu"><UL id="submenu_1"><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter" target="_self">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Design" target="_self">Design</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Microfluidics" target="_self">Microfluidics</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/model" target="_self">Modelling</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results" target="_self">Results</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Parts" target="_self">Parts</A></LI><LI><A href="https://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/References" target="_self">References</A></LI></UL></DIV><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Construction_of_our_counters"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Construction of our counters</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Proof_that_IntI1_integrase_works_.21"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Proof that IntI1 integrase works !</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Proof_that_the_Tetracyline_Counter_works_and_first_evaluation_of_the_recombination_rate"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Proof that the Tetracyline Counter works and first evaluation of the recombination rate</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Confirmation_of_the_specific_excision_of_the_terminator_by_sequencing"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Confirmation of the specific excision of the terminator by sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Characterization_of_the_recombination_rate"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Characterization of the recombination rate</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Characterisation_of_the_integrase_toxicity"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Characterisation of the integrase toxicity</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Test_of_the_microfluidic_device"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Test of the microfluidic device</SPAN></A></LI></UL><H2><SPAN class="mw-headline" id="Construction_of_our_counters">Construction of our counters</SPAN></H2><P style="display:block;">First of all, we have spent a significant part of the summer to do cloning using the standard biobrick assembly protocol. For this, we have used a lot of biobricks submitted to the parts registry by previous teams, thank you ! Using these parts we have created plenty of new composite parts that are highly modular (different kind of RBS and promoters) and could serve next iGEM teams. We have also created some new biobricks that are at the core of our project. We hope that other iGEM teams will work with integrons and might need these parts. We have constructed both the basic counter (with RFP and Tetracycline resistance gene) and the full counter + timer designs.
Moreover, each of our constructs was verified by sequencing. For more information you can take a look at the parts that we sent to the registry.

</P><H2><SPAN class="mw-headline" id="Proof_that_IntI1_integrase_works_.21">Proof that IntI1 integrase works !</SPAN></H2><P style="display:block;"> Then we wanted to see if the IntI1 integrase actually works. We have done a double transformation &quot;RFP Counter&quot; (on pSB1A2 which is a high copy plasmid) + &quot;pBAD-IntI1&quot; (on pSulib which is a low copy plasmid with kanamycin resistance) to obtain a working counter. The protocol of the counter testing is quite simple : we dilute an overnight culture 1/100, wait until OD 0.2 and add arabinose to begin the test (t=0h). At regular time intervals, we plated on plates containing ampicillin and glucose. We didn't have to put kanamycin into the plates because we don't want recombinations to happen on the plates. So we don't need the cells to keep their pBAD/Int plasmid. Moreover, to make sure that to recombinations happen after the cells have been plated, we have added glucose to the plates because glucose tightly represses the pBAD promoter, so that we can be sure that the integrase isn't expressed.
 This experiment shows us that IntI1 is working! The double terminator flanked by attC sites has been excised and the constitutive promoter upstream permits the expression of RFP. However, we have 2 important notes :

</P><UL><LI> At the beginning of the test, there are already some fluorescent colonies... We can explain this fact by the leaking of pBad that controls the expression of the integrase. Thereby the integrase is expressed at a low level and excises the terminator while the cells are growing in an overnight culture, even before we start our test. To overcome this problem we had to add glucose into the overnight culture media. Thus, glucose blocks the pBAD promoter and avoids recombination events to occur before our test begins. 
</LI><LI>The expression of RFP is inhomogeneous because the counter is on a high copy plasmid. So some cells have 2 recombined plasmids while others have 20 recombined plasmids ! To obtain a more linear response we have integrated our counter in a low copy plasmid (pSB4A5). In this way, there is more chance that only one event will happen in each cell.
</LI></UL><H2><SPAN class="mw-headline" id="Proof_that_the_Tetracyline_Counter_works_and_first_evaluation_of_the_recombination_rate">Proof that the Tetracyline Counter works and first evaluation of the recombination rate</SPAN></H2><P style="display:block;">Following the same protocol, we tested the Tetracycline Counter. Only one copy of TetA(C) expressed is sufficient to induce tetracycline resistance. Therefore the number of resistant colonies is a good approximation of the recombination rate.
</P><H2><SPAN class="mw-headline" id="Confirmation_of_the_specific_excision_of_the_terminator_by_sequencing">Confirmation of the specific excision of the terminator by sequencing</SPAN></H2><P style="display:block;"> We had to be sure that the clones that have become resistant to tetracycline have changed their phenotype because the terminator got excised and not simply due to leakage. That's why we miniprepped our resistant clones and sent these plasmids to sequencing. Here you can see the nucleotide alignments which prove that the IntI1 integrase has specifically excised the terminator flanked by two attC sites.
</P><H2><SPAN class="mw-headline" id="Characterization_of_the_recombination_rate">Characterization of the recombination rate</SPAN></H2><P style="display:block;">At this point, it is important to determine the recombination rate of IntI1. To do this we have done our double transformations again on plates containing 1% glucose to inhibit the expression of IntI1 before the beginning of the test. We did our overnight cultures with glucose for the same reason, then we diluted, waited until OD 0.2 and added arabinose 0,04%. The counters are on a high copy number plasmid, so as we have explained before it is difficult to estimate the number of recombinants per cell. But we found a trick to calculate the recombination rate ! 
In every cell there is a lot of plasmids, some of them are recombined, others not. To have a good idea of this proportion, we did minipreps at regular time intervals in parallel with direct plating of the cultures. Then we transformed the minipreps that contain the whole population of plasmids. Statistically, only one plasmid can be integrated per competent cell. After the plasmid is integrated into the cell, it is multiplied to reach its copy-number. This can be regarded as an <B>amplification of the recombination result</B>. This way, some cells are red (because they were transformed with an excised plasmid and then copied it) while the others are not fluorescent (because they were transformed with the wild-type plasmid). This protocol is more complicated but it permits to significantly increase the sensitivity of the test.
However, the experiment with the Tetracycline Counter failed. The direct plating gives positive results but the transformation of the minipreps gives no colonies on tetracycline plates. After some researches we noted that the <B>TetA(C) gene is toxic for the cells if over-expressed</B><A href="http://partsregistry.org/wiki/index.php/Part:BBa_J31007">(see description of the biobrick)</A>. The principle of this experiment is to obtain only recombined plasmids in the cells, so we assume that it is lethal for the cells when expressed on a high copy number plasmid.  

</P><P>This graph shows the evolution of the recombination rate deduced from the number of recombined plasmids. In order to assure efficient counting, the recombination rate has to be low (under 5%). From these results we can see that arabinose induction triggers too much recombinations. Actually the absence of arabinose is a perfect condition to test the counter (leaking of pBad).
</P><H2><SPAN class="mw-headline" id="Characterisation_of_the_integrase_toxicity">Characterisation of the integrase toxicity</SPAN></H2><P style="display:block;"> When doing the experiments, we noted that the number of colonies on our plates abnormally decreased with time. We deduced that the expression of the integrase is toxic for the cells, as you can see below.
</P><H2><SPAN class="mw-headline" id="Test_of_the_microfluidic_device">Test of the microfluidic device</SPAN></H2><P style="display:block;">
We have replicated the original pattern from the mask and made PDMS chips. We have managed to test the microfluidic chip. It works : the cells can get trapped inside the chambers, and then grow inside the chambers filling them up. The fact that the cells grow to a high density is important because we need the culture to be dense enough for the quorum sensing to work.
Here are the videos of our microfluidic device we have taken : 
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results">http://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Liliane_Bettencourt/Project/Population_counter/Results&amp;oldid=208297" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>