"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Liliane Bettencourt/Project/Memo-cell/Design
From 2010.igem.org
 
home
projects
parts
team
notebook
safety
Brainstorming
acknowledgements
Memo-Cell project
Introduction
Design
Results
Parts
References
Materials
Contents
1 DNA entity integration
2 Integration site regeneration
3 Selection system
4 Sum up
DNA entity integration
To integrate a DNA sequence into the chromosome in a stable manner, the Phage recombination systems seemed to be appropriate, as this can allow the integration of a whole plasmid into the bacterial chromosome, as long as it carries an attP recombination site. By expression the phage integrase, one can easily trigger the plasmid integration into the chromosome via the recombination between the plasmidic attP site and the chromosomal attB site. Click here to see a detailed figure
When integrating into the chromosome a plasmid via phage recombination, the original chromosomal recombination is disrupted and hence blocks any other integration. To achieve a sequential integration into the chromosome, we had to find a way to recreate a functional recombination site after the first integration, so as to be able to add another piece of DNA.
We could not directly excised the whole DNA sequence that had just been integrated, as no DNA sequence of the plasmid would stay on the chromosome to count. Hence, we had to chose another mechanism.
Integration site regeneration
To do this, we have chosen to engineer the Tn916 transposon.
Click here to see a detailed figure
We redesigned heavily mutated flanking sequences ('arms') of the transposon by replacing them with the two halfs of the phage recombination site so that once the transposon is excised, a functional phage recombination would be regenerated on the chromosome. The designed sequences retain the essential nucleotides of the flanking sequences for the  functional excision of the transposon, resulting in a complex hybrid where mutations have been inserted both in the flanking sequences and the phage recombination site.
Moreover, we had to drastically reduce the size of the Tn916 (18kb) in order for it to be experimentally amenable. We therefore reduced its sizebyf 90%, cloning between the left and right arms either a Kan-resistance cassette (~1kb) or a Kan-resistance and a Lac-Z alpha cassette (~1,8kb). The total size of our mutated transposon was either 1,4 kb or 2,2 kb.
The efficiency of excision using the reduced transposon with mutated arms has been assessed (protocol and detailed results in the result page).
We have also assessed the efficiency of recombination using the mutated phage recombination sites (protocol and detailed results in the result page).
Now that we had working systems to integrate a DNA sequence into the chromosome and another one to reform a functional recombination site, we had to integrate them together so that these two could work sequentially.
To have a sequential integration of DNA sequences one after the other on the chromosome, the new combination site after excision had to be adjacent to the DNA sequence that had just been inserted.
First design
One could have thought to place the mutated Tn916 directly in the plasmid that will be integrated in the chromosome. But then, the transposon excision would happen not only on the chromosome but also in the leftover plasmids in the cytoplasm, drastically reducing the chance to have a plasmid still containing the Transposon at the next round of counting.
Second design
However, as excision of the transposon requires not only the presence of both the Left and the Right arm, but also that these two arms are on the same DNA molecule, we constructed our integrated plasmid with only half of the transposon: the left arm only.
The right arm would be located on the chromosome after the phage recombination site
Hence, once the integration of the plasmid has taken place, the left arm of the transposon would be on the chromosome, in line with the right arm, allowing the efficient excision of the transposon.
However, with this design, it appeared that a successive counting would be endangered due to the repetition of fully functional Transposon right arms on the chromosome. Indeed, the transposase is originally used to move huge DNA segments (~18 kb). Basically, once the plasmid integrated in the chromosome, there would be no reason that the excision of the transposon would use the closest right arm rather than one of the other downstream right arms.
Third and final design
Our final design tackles all potential unwanted molecular scenarios previously brought up.
To restraint the transposon excision to the right arm the closest to the left arm integrated in the chromosome, we designed a system in which only a portion of the right arm would be on the chromosome before the integration process. The other portion would be on the plasmid. Neither of these right arm segment would be functional alone.
Such a design required extensive mutation of the left arm so that a functional phage recombination site could be fit within the arm.
This implied the deletion of transposase and excisase binding sites. This design was cloned and tested successfully! (see results section).
Counting from 1 to N with the final design
Now, with our final design, the «counting» process could be repeated N times by alternating cycles of integration and excision, repeatedly regenerating a unique and functional integration site.
As output, the number of cycles can be determined either by PCR amplification of the counter, or by turning on a reporter delivered by the last cycle.
Selection system
Engineering of the microcin C operon
For our project, we needed to have an inducible death gene, as small as possible (<40bp), so that it could fit within a recombination site but also could be triggered only when recombination did not happen, that is to say, when the bacteria did not «count» well.
Hence, microcin C seemed to be the best choice as it is only 24bp long.
However, microcin C is naturally produced by one bacteria to survive in a depleted media by killing surrounding bacteria, which is the opposite of what we needed.
We had then to engineer to operon so that once induced, the microcin C would kill the producing bacteria but not the rest of the population.
To do this, we had to tackle three major keypoints:
1. Knock-out the gene coding for the membrane transporter responsible for the uptake of the microcin.
2. Knock-out genes within the operon which are responsible for self-immunity against the microcin
3. Mutate the operon so as to delete the four biobrick restriction sites (3 PstI and 1 EcoRI).
What is microcin C?
Microcins are a class of small (>10-kDa) antibacterial agents produced by Escherichia coli and its close relatives. Microcins are produced from ribosomally synthesized peptide precursors. The microcin C (McC) is posttranslationally modified by dedicated maturation enzymes encoded by genes in the microcin C operon mccABCDE. McC is a heptapeptide with covalently attached C-terminal modified AMP. The peptide moiety of McC is encoded by the 21-bp mccA gene, the shortest bacterial gene known. McC has a molecular mass of 1,178 Da and contains a formylated N-terminal methionine, a C-terminal aspartate instead of the asparagine encoded by the mccA gene, and an AMP residue attached to the carboxamido group of the modified aspartate through an N-acyl phosphoramidate bond. The phosphoramidate group is additionally esterified by a 3-aminopropyl moiety.
How it works
McC is taken up by E. coli through the action of the Yej- ABEF transporter and is processed once it is inside the cell. Processing involves deformylation of the N-terminal Met residue by protein deformylase, followed by degradation by any one of the three broad-specificity aminopeptidases (peptidases A, B, and N).  Processed McC strongly inhibits translation by preventing the synthesis of amino- acylated tRNAAsp by aspartyl-tRNA synthetase (AspRS).  The tRNA aminoacylation reaction catalyzed by aminoacyl tRNA synthetases includes two steps. First, the enzyme activates a cognate amino acid by coupling it to ATP and forming aminoacyl-AMP (aminoacyl-adenylate). The aminoacyl moiety is then transferred to tRNA. Processed McC is structurally similar to aspartyl-AMP but is not hydrolyzable. Thus, the inhibition of AspRS results from tight binding of processed McC in place of aspartyl-adenylate.
Thus, McC is a Trojan horse inhibitor: the peptide moiety is required for McC delivery into sensitive cells, where it is processed with subsequent release of the inhibitory payload.
Sum up
Algorithmic overview of the counting process
Retrieved from " http://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Memo-cell/Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
