"
Page
Discussion
View source
History
teams
Log in
 
Team:UNAM-Genomics Mexico/Modules/In vivo
From 2010.igem.org
Home
Team
Us
Official Team Profile
Unofficial Team Profile
Them
Project
Overview
Individual Modules
In vivo
In silico
Whole-System View
Parts created
Model created
Official Notebook
Safety & Human Practices
Collaboration
About
iGEM
Synthetic Biology
UNAM
Gallery
Coupling together Biological Chassis
In order to enable the light based communication between bacteria, we have designed a tertiary cycle of different bacteria chassis, assembling each module of reception and emission to construct a light-based feedback loop of red-green-blue light, which will make the proof of concept of  communication over distance and proper signal decoding.
Tertiary cycle
Coupling together Biological Chassis through Light communication
Chassis Description
First Chassis: inRED-outGREEN
First Chassis: Red Light reception module coupled to Green Light emitter module.
Reception module: Red Light Photosensor cph8
OBJECTIVES
The main goal was the construction of a red light receptor able to indirectly trigger a transcriptional response that in our system corresponds to the expression of luxAB genes (Vibrio fischeri Luciferase).
METHODOLOGY
The red light receptor would be the chimeric protein Cph8(Cph1-EnvZ fusion). Exposure to red light inhibits the activity of the EnvZ histidine kinase domain. When active (in the dark) EnvZ phosphorylates endogenous OmpR, a transcription factor which activates transcription from the OmpC promoter and represses transcription from the OmpF promoter. While under light EnvZ would not phosphorilate OmpR, and OmpF promoter would not be repressed. In order to generate the transcriptional response of the luxAB genes in a light dependent manner, they were coupled to OmpF promoter.
This module of red light reception must be used in E.coli deficient in natural EnvZ to avoid cross-talk in the response.
Cph8 is into bio-brick BBa_I15010, however Cph8 needs another two proteins in order to reach its functional conformation and light sensitivity, these proteins are PcyA and Ho1 and they are coded in bio-brick BBa_K098010. There is another bio-brick which is composed of these two bio-bricks, it is called BBa_M30109.
The experimental strategy used consisted in the extraction of the aforementioned biobricks from the iGEM DNA distribution.
RESULTS
Even it was tried to get BBa_M30109 in the first place and then BBa_K098010 and BBa_I15010 from iGEM kit plate, we could not get those bio-bricks due to issues with DNA submission from iGEM kit plate.
Emission module: LuxAB luciferase from vibrio fischeri plus YFP protein
OBJECTIVES
The final aim was the construction of the blue light emission bacterial luciferase (LuxAB)from Vibrio fischeri.
The structure of the device was designed as follows:
Design LuxY biobrick, send to synthesize and transfer the construction from Mr. Gene plasmid to pSB1C3 DNA submission backbone.
To test that the device works as expected.
Luciferase coded by luxA and luxB needs a substrate (an aldehyde synthesized by enzymes into the lux operon), in the project it was decided to add the aldehyde to the reaction in an exogenous way. So the biobrick to synthesize  the aldehyde was not designed.
METHODOLOGY
The lux operon
Vibrio fischeri’s lux operon is composed of several genes, all of them involved in bioluminescence and its regulation. This is the structure of Vibrio fischeri’s lux operon:
As it is shown in the diagram luxA and luxB genes, those coding for the luciferase, are placed together into the operon so the strategy was amplify them by means of PCR and then ligate PCR product with a strong constitutive promoter; the exogenously aldehyde added to the strain containing luxAB genes would be enough to trigger the luminescence reaction.
In the first place, we decided to purify genomic DNA from Vibrio fischeri strain MJ11 and then make a PCR with the following primers designed to extract luxAB region:
Forward primer:Standard iGEM prefix+RBS+Coding Region
CGG AAT TCG CGG CCG CTT CTAG   AGGA   A ACA GCT ATG AAG TTT GGA AAT ATT TGT TTT TCG TAT CAA CC
Reverse primer: Standard iGEM suffix+Coding Region
CTG CAG CGG CCG CTA CTA GTA TTA TTA GGG TAG ATT CTT TTC AAT TTT TTG GTT CAA C
YFP protein (LuxY)
LuxY is a gene coding for YFP, a protein that shifts the light emission wavelength of Vibrio fischeri’s luciferase from 484nm (blue) to 534nm (green - yellow); because our system needed a green or yellow emission module we looked in the literature for a protein capable to generate this phenotype.
The sequence was taken from the article published  by Thomas O, et al., and then the sequence was synthesized by Mr.gene.
This is the composition of the synthesized sequence, “Gene” stands only for luxY coding region.
Preffix+ Promoter + RBS + Gene + Double Terminator + Suffix
See LuxY(BBa_K360100) annotation at the registry for more details.
RESULTS
We tried to get luxAB from V.fischeri's genome and then make a bio-brick with these genes, although we did not have many problems getting luxAB genes from V.fischeri's genome with a PCR approach, ligations and the following transformations never worked although we tried with several plasmids (see Augusto's results).
However our collaborator, Cambridge University sent us a plasmid with luxABCDE under PBAD promoter so E.coli tranformated with this plasmid and grown in arabinose media showed a blue glowing phenotype.
We cotransformated luxABCDE with luxY and the cotransformation was successful although we have not been able to characterize it yet, the experiments are still in process.
REFERENCES
Thomas O. Baldwin, Mary L. Treat, S. Colette Daubner(1990).Cloning and expression of the luxY gene from Vibrio fischeri strain Y-1 in Escherichia coli and complete amino acid sequence of the yellow fluorescent protein. Biochemistry  29 (23), 5509-5515.
Second Chassis:inGREEN-outBLUE
Second Chassis: Green Light reception module coupled to Blue Light emitter module.
Reception module: Green Light cianobacterial Photosensor CcaS/CcaR
OBJECTIVES
Design and construction of the green light receptor.
Expression of accesory genes for PCB creation, Ho1 & pcyA, in order to make the photosensor sensitive to light.
Demonstrate the Green Receptor activity with a GFP reporter.
METHODOLOGY
Green reception is composed of a two-component system. The green light sensing agent CcaS, that is a histidine kinase that phosphorylates the transcription factor CcaR, it shows greater affinity for DNA when is phosphorylated. This two-component system has been recently described in Synechocystis sp. PCC 6803, and due to the homology relationship with the EnvZ-OmpR system, it might possibly be functional in E.coli.
Once  the sequences of the green light receptor (ccaS),transcriptional factor (ccaR) and DNA regulated region from Synechocystis sp. PCC6803, reported in Yuu Hirose were obtained, they were sent to synthezise.
The DNA regulated region by CCaR would be ligated to the GFP protein, to analyze the activity of the photosensor.
RESULTS
Edinburgh team sent us plasmid with accesory genes for PCB creation because we could not get them from the existing biobrick BBa_K098010, which is also necessary in the red reception module.
CcaR and CcaS were ligated to a weak constitutive promoter (J23117). Activity confirmation is still in progress.
Emission module: LuxAB luciferase from vibrio fischeri plus Lumazine protein
OBJECTIVES
Obtain LuxAB genes from Vibrio fischeri.
Design Lumazine biobrick, send to synthesize and transfer the construction from Mr. Gene plasmid to pSB1C3 DNA submission backbone.
To test that the device works as expected. This characterization will test if Lumazine protein can also be functional working with the Vibrio fischeri luciferase, instead of its native interacting luciferase from Photobacterium phosphoreum.
METHODOLOGY
Lux operon
The strategy to obtain LuxAB genes from Vibrio fischeri was the same working with the green light emission module.
Lumazine
Lumazine protein of Photobacterium phosphoreum (lumP) is a small protein which shifts the wavelength of light emitted by the P.phosphoreum luciferase LuxAB from 495 nm to about 475 nm. It was used to tune the blue light emission wavelength towards another one closer to that of LovTAP system reception.
A version of the lumazine biobrick (BBa_K216007) was sent to us by Edinburgh Team, due to this construction hadn't been tested we decided to synthesize it, in order to avoid unexpected results for possible errors in the already assembled biobrick from Edinburgh.
This is the composition of the synthesized sequence, “Gene” stands only for lumazine coding region.
Preffix+ Promoter + RBS + Gene + Double Terminator + Suffix
See Lumazine (BBa_K360122) annotation at the registry for more details.
RESULTS
In order to test if lumazine protein is working correctly, shifting the emission spectrum from Vibrio fischeri luciferase to more blue light output, co-transformation of the plasmid from Mr. gene harboring the Lumazine gene with the plasmid sent by Cambridge team containing the Vibrio Fischeri luciferase with the substrate generating and recycling pathway (luxABCDE) was successfully made, but the blue shifting glowing phenotype was not obtained, it was still green-blue. It is possible that the wavelength shift had done correctly but was not visible to naked eye due to the change is minuscule (495 nm to ~ 475 nm). So, it was required the usage of a sensible measurement device to detect it.
A first measurement was done with a spectrophotometer but it didn’t detect any RLU (relative light emission unit), even the bioluminescence was visible to naked eye in the dark room.
A second attempt will be using a CCD camera that is expected to have the necessary sensitivity to detect bioluminescence and measure the light emission spectrum.
REFERENCES
Yuu Hirose, Takashi Shimada, et al., (2008) Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein. Proceedings of the National Academy of Sciences, 105(28): 9528–9533.
O'Kane, D.J., Woodward, B., Lee, J., and Prasher, D.C. 1990. Borrowed proteins in bacterial bioluminescence. Proc. Natl. Acad. Sci. USA 88, 1100-1104.
Lee, J., Wang, Y., and Gibson, B.G. 1991. Electronic excitation transfer in the complex of lumazine protein with bacterial bioluminescence intermediates. Biochemistry 30, 6825-6835.
Third Chassis:inBLUE-outRED
Third Chassis: Blue Light reception module coupled to Red Light emitter module.
Reception module: Blue Light Photosensor LovTAP and blue light inducible promoter
Working in this chassis, we decided to use as blue light receptors: LovTAP photosensor and the native blue light inducible promoter from E.coli.
LOVTAP
OBJECTIVES
Design and construct an improved version of the LovTAP photosensor under the regulation of  different constitutive promoters and  with a punctual mutation (ILE427PHE), as was proposed by the model results of the EPF-Lausanne 2009 team.
Transfer the plasmid with the LovTAP synthesized gene from Mr. Gene Plasmid to pSB1C3 DNA submission backbone and ligate the construction to different weak constitutive promoters.
Characterize LovTAP reporter systems, both repressor and activator activity.
The structure of the device is designed as follows:
METHODOLOGY
LovTAP design
The blue light sensor LovTAP biobrick (BBa_K360121) was designed and constructed with some modifications in comparison to the existing LovTAP biobrick (BBa_K191006) available at the registry. It is an improved version that contains a punctual mutation ILE427PHE, which was proposed by the model results of the team EPF-Lausanne 2009, with this mutation LovTAP should react faster and the conformational change should be more stable (the protein stays in the active form for longer, under light induction).
As well, LovTAP was designed without the inverting regulator sensitive to LacI and CAP protein because according to the report of Lausanne team, the expression levels of LovTAP under the inverting regulator didn’t show differences to the induction with IPTG. Thus, the inverting regulator was removed and none promoter was included in the design that was sent to synthesize, instead several constitutive promoters would be ligatated to the synthesized construction to choose the one that works better. The RBS (ribosome binding site) was changed from a strong to a medium strength. Finally, the 2 PstI restriction sites were removed from the coding region of LovTAP, which makes the LovTAP construction compatible with the 10 standard protocol of biobrick assembly.
So, the LovTAP construction that was sent to synthesize to Mr. Gene Company, consists of the following basic parts:
Prefix+ medium strength RSB + LovTAP Coding sequence + Transcriptional Double Terminator + Suffix
E.coli Strain Mutant
As the transcriptional response regulated by LovTAP might also be generated by the trpR repressor in E.coli in tryptophan rich conditions, I looked for an Escherichia coli strain mutant in trpR to avoid the cross-talk of the endogenous function of this gene with LovTAP system.  Dr. Charles Yanofsky had the mutant (Identification number: CY15001) and kindly sent it.
According to the sequencing results, the trpR gene from the mutant strain didn’t have a frame shift as is reported at the literature, instead of that it had a non synonymous mutation (Alanine to Proline) corresponding to the aminoacid 80. The hypothesis is that this change might causes a dramatic structural change to the protein thus being non functional.
LovTAP expression Levels
Dr. Devin Strickland the designer of LovTAP, shared some issues to take into account to work with LovTAP. Analyzing some experimental results reported in his dissertation and in his published paper it is observed that at high expression levels of LovTAP, the behavior in the dark versus the light state is the same. So that, the regulation by light becomes not functional, thus I planned to test LovTAP under three different weak constitutive promoters (J23117, J23114 and J23105) to analyze its behavior and under the strong promoter (J23102), expecting to obtain the behavior reported by Dr. Devin.
LovTAP Reporter systems
The output module of LovTAP is the DNA binding domain of the bacterial transcription factor trp repressor (TrpR), with this domain LovTAP activated can bind as homodimer to the DNA operator called trpL, repressing the transcription.
The functional elements of the trpL promoter (-35 and -10 boxes, trpR binding sites and transcription start site) were synthesized in an oligonucleotide to introduce it by PCR into a selected plasmid. At the ends the primer includes the prefix and the NheI site that will be used for ligations of the promoter with the reporter systems described later. The primer designed is the following:
Primer trpL reverse (5'->3'): NheI site + trpL promoter + Prefix (XbaI + EcoRI sites):
TTGCTAGCGTGAACTTGCGTACTAGTTAACTAGTTCGATGATTAATTGTCAACAGCCTCTAGAAGCGGCCGCGAATTC
The primer is the reverse complementary of that sequence so we can use it as a reverse primer, which we used along with a suffix forward to introduce the trpL promoter into pSB1C3 plasmid by PCR.
Both LovTAP activator and repressor activity were of interest to characterize. So, the following constructions were designed.
LovTAP repressor activity: Reporter system
1.trpL promoter fused to GFP protein(BBa_E0240):
2.trpL promoter fused to RFP protein(BBa_E1010):
LovTAP activator activity: Reporter system
trpL promoter fused to lambda Repressor cI (BBa_P0451 + BBa_K098991) regulating GFP protein (BBa_E0240). Whit this system a double repression is generated to have a final activation response.
Characterizing LovTAP
Once LovTAP with the three weak constitutive promoters (J23117,123114 and J23105) and the strong promoter (J23102) was correctly obtained in plasmid pSB3K3, and the reporter system trpL+RFP was also finished  in plasmid pSB1C3 ,  the co-transformation procedure was done,  in order to have the whole system inside E.coli cells, both trpR wildtype and mutant. Besides, trpL+RFP construction in plasmid pSB1A2 from Lausanne team was kindly sent it by Edinburgh team. Both trpL-RFP reporter systems were used, being the Lausanne system the reference, expecting to obtain similar results. The difference between the reporter systems is that the constructed in plasmid pSB1C3 doesn’t have the transcriptional double terminator.
In order to test if LovTAP works correctly two protocols were implemented: the qualitative and the quantitative approach.
Considerations to take into account:
The RFP protein does not include a degradation tag, so the time required to notice a clear difference between light and dark states will be long, because although LovTAP starts to repress, the RFP produced previously by the cells will be still present.
The plasmid in which LovTAP is being express is pSB3K3 with a copy number around 20 to 30, while trpL-RFP construction is inside plasmids pSB1C3 and pSB1A2; both are high copy number plasmids (100 to 300 per cell). So there are many trpL binding sites that should be repressed by LovTAP. The ideal condition for this experiment would be to have LovTAP and trpL+RFP constructions inside the same plasmid.
Using the LovTAP constructions fused to promoters with different strength, it can be tested at what levels of expression, the LovTAP light regulation is better. It is already known that under high expression levels of LovTAP, trpL promoter is repressed even in dark. To test this scenario J23102 promoter was used.
Qualitative experiment
The qualitative approach was designed to observe and compare the RFP production in the cells harboring LovTAP exposed to light versus dark conditions, both in wild type and trpR mutant strains. Under blue light conditions, when LovTAP repressor activity is activated, it is expected to observe a lower level of RFP in comparison to the cells maintained in the dark state.
Experimental procedure
The samples used were: trpL+RFP reporter system in plasmid pSB1C3, Lausanne trpL-RFP reporter system, LovTAP under promoters J23117, J23114, J23105 and J23102.
Only one colony of each co-transformation was used along the experiment and was tested under light and dark conditions.
TrpR mutant and wild type cells were co transformed using 5 µL of each plasmid (trpL-RFP in pSB1C3/ pSB1A2 and each LovTAP construction in pSB3K3). For each co transformation one  tube containing 5 ml of  LB medium with Kanamicyn and chloramphenicol or Kanamicyn and ampicillin, was inoculated from  a single colony of DH5α. Cultures were grown  overnight (~15hrs) at 37°C with spinning at 250rpm in dark conditions. Then, 1 ml of broth was taken and transferred into 5 ml of fresh LB medium with antibiotics. The cultures were grown for approximately 13 hours under the previous conditions but some were exposed to blue light (470nm) while others were maintained in dark. Finally, the cultures were spun down and compared as follows: the RFP pellets obtained under blue-light versus dark condition, and wild type samples versus mutant samples.
Results
In a first approach was observed that wt and trpR mutant cells co-transformed with LovTAP under J23102 promoter grew in dark and light, produced very low levels of RFP protein in comparison with the control sample (wild type cells with only the reporter construction, trpL+RFP). Besides, there was no a visible difference between dark and light conditions, which is the expected behavior under high expression levels of LovTAP. Considering this result we decided to continue the co-transformations procedure without considering the LovTAP construction under J23102 promoter.
Using LovTAP with promoters J23105, J23114 and J23117 (in order of high to lower strength), we obtained the following pellets that are show in the next images:
Testing LovTAP with trpL+RFP reporter system in plasmid pSB1C3(without the double transcriptional terminator). The figure shows the pellets obtained from the cellular cultures. The non aligned tube at the right is a sample of WT cells harboring only trpL+RFP construction.
Testing LovTAP with Lausanne trpL+RFP reporter system in plasmid pSB1A2. The figure shows the pellets obtained from the cellular cultures. The non aligned tube at the left is a sample of WT cells harboring only trpL+RFP from Lausanne team construction.
According to the images trpR mutants seem to have a lower RFP expression levels versus WT both in dark and light conditions. This is a surprising result because it was expected that showed higher levels than WT as they don’t have the possible crass-talk with trpR native E.coli repressor. Maybe there is another process of trpL repression independent to LovTAP and trpR.
Both trpL-RFP reporter systems give similar results. Comparing blue-light versus dark exposed samples there seems to be a small difference visible by naked-eye, with higher levels of RFP protein in dark state samples. Maybe the small difference observed between light and dark conditions is because LovTAP protein levels are very low to considerably repress the trpL promoter. As well, the long half-life of the RFP protein could be masking a significant difference between both states.
Although these results not formally demonstrate that the LovTAP repression is light dependent, it seems that in cells co-transformed with LovTAP versus those transformed only with the trpL-RFP system, there are lower levels of RFP. Another interesting pattern observed in the samples, is that there are lower levels of RFP as the promoter strength that regulates LovTAP increases, thus suggesting that there is a gradient of repression depending on the LovTAP concentration.
These results suggest that possibly LovTAP is working well, however as this experiment is totally qualitative, it must be improved taking into account the optical density of the samples, because the observations could be due to the presence of different number of cells in each sample. As well replicates of the experiment are needed.
Quantitative experiment
With the aim to have a better characterization of LovTAP, we designed a new protocol considering the methodology describe by Jason R Kelly, we include some changes that are detailed below.
We decided to start the protocol under blue light conditions to test the dark state, expecting that in dark exposed samples the RFP levels increase in comparison with those samples that are always under blue light.  We did this in order to face better the issue of the long half life of the RFP due to it doesn’t have a degradation tag.
Experimental procedure
The samples used were: trpL+RFP reporter system in plasmid pSB1C3, Lausanne trpL-RFP reporter system, LovTAP under promoters J23117, J23114, J23105 and RFP under J23102.
Only one colony of each co-transformation was used along the experiment and was tested under light and dark conditions.
TrpR mutant and wild type cells were co transformed using 5 µL of each plasmid (trpL-RFP in pSB1C3/ pSB1A2 and each LovTAP construction in pSB3K3). As controls RFP constitutively expressed under J23102 promoter was used and wild type cells transformed just with trpL-RFP. For each transformation one  tube containing 5 ml of  LB medium with Kanamicyn and chloramphenicol or Kanamicyn and ampicillin, was inoculated from a single colony of DH5α. Cultures were grown  overnight (~14hrs) at 37°C with spinning at 100rpm under blue light (470nm) conditions. Then cultures were diluted 1:1000 into 5ml of fresh LB media with antibiotics. These were grown for approximately 4.5 hours under the previous conditions under blue light (470nm). After this step the OD600 was measured from each culture.  Based on this OD measurement, the cultures were diluted to the same OD (0.15) in 5 ml of LB fresh media. Then three 200 µL aliquots from each culture were transferred into a flat-bottomed 96 well plate. Samples were loaded in the plate as is shown in the next picture:
An initial measurement of OD600 and fluorescence (530/25 nm excitation filter, 590/35 nm emission filter) by triplicate was done. Using the light emission device to irradiate the incubator, the plate was maintained during 17 hrs at 37°C with spinning at 35rpm inside the incubator but with a mask in the samples selected for the dark conditions.  Then, a second measurement of OD600 and fluorescence was done. Initial data with the second measurement were compared. Background absorbance and  fluorescence were determined by measuring wells containing only media, and the values obtained were used to normalize the other samples.
Results
The average ODs of the wells with only media and antibiotics changed drastically from the initial to the second measurement(km/amp from 0.282 to 0.09, and km/cm 0. to 0.093), dificulting the data analysis. So the data obtained were inconsistent and even it was intented to analyse them, there was not the expected difference between light and dark conditions.
Conclusions
According to the aforementioned results, we think that LovTAP could possibly work well. However we have to improve the characterization protocols in order to support with enough evidence this conclusion. As well, more replicates and controls must be included in the experiments.
We also think that using other reporter system to describe the transcriptional activation activity of LovTAP using a cascade of double repression, might generate better results.  So we will test LovTAP behavior with the cI inverter.
YcgF/YcgE BLUE RECEPTION SYSTEM
OBJECTIVES
The main goal for this section of the project was to implement a system for blue light reception using the YcgF/YcgE sytem, naturally present in certain strains of Escherichia coli.
This reception system would be coupled both to a reporter gene (i.e. GFP) and, for the purposes of our project, a luciferase gene.
METHODOLOGY
We used a modified version of the YcgF/YcgE system for blue light reception (previously reported by the K.U. Leuven 2009 Team), the modification consists in the reduction of the promoter region to 50 bp.
Originally, we tried to amplify the complete promoter region BBa_238013 (86 bp) from genomic DNA of E. coli K12 by PCR but we failed in several trials. So we decided to synthesize the promoter in a primer and then insert it into pSB4A5 by PCR. Because of lenght limitations in primer synthesis we reduced the promoter region to 50 bp.
The reduced promoter retains the fundamental parts of the original one, these are the -35 and -10 box, the spacer between them, the inverted repeat 1 and 2 (inverted regions are the binding sites for the YcgE repressor) and the transcription start site. We registered this Minimum Blue Light Receptor Promoter as BBa_K360041.
In order to test the functionality of our Minimum Blue Promoter we succesfully ligated it to our Strong RBS BBa_K360031 and the GFP BBa_E0040.
Characterizing YcgF/YcgE BLUE RECEPTION SYSTEM
Experimental procedure
It is reported that the response of this promoter is weak in comparison to some others standard strong promoters registered in the Registry of Standard Biological Parts. We implemented a protocol for testing the response of our Minimum Blue Light Receptor Promoter (Min-BP) in which we irradiated cells with the construction Min-BP + RBS BBa_K360031 + GFP BBa_E0040 with blue light (470 nm) leds for different times. We also irradiated with green (540 nm) and red (660 nm) light leds to discard any crosstalk of these wavelengths. GFP expression was compared to a reference: J23101 promoter + RBS BBa_K360031 + GFP BBa_E0040.
We measured GFP expression of our constructions, in order to irradiate the cells with Min- BP + RBS BBa_K360031 + GFP BBa_E0040 we used blue leds and irradiated the cells for 300 minutes, we also measured GFP expression of cells that were incubated in the dark and cells with the constuction J23101 promoter + RBS BBa_K360031 + GFP BBa_E0040.
For the experiment, cells were incubated overnight in M9 medium with glycerol (0.4%) as carbon source, in the morning OD600 was measured and cells diluted to 0.07 nm. After dilutions, we started irradiation with blue leds. Temperature is a very important factor in the function of the YcgF/YcgE system; it is reported that at 25ºC the ratio between the YcgE repressor and YcgF activator allows a good repression unless there is blue ligh irradiation, so we incubated cells with this promoter at 25ºC. Cells with the constitutive promoter BBa_J23101 were also incubated overnight at 37ºC.
Results
A graph showing GFP expression of our Minimum Blue Promoter compared with a constitutive promoter is shown here:
Next graph shows GFP expression in presence and absence ob blue light:
According to the graphs we have shown that our Minimum Blue Light Receptor Promoter works as expected but the response is weak compared to a standard, constitutive promoter BBa_J23101. This results are consistent with previously reported data by the K.U. Leuven 2009 Team.
In order to increase GFP expression in response to blue light, an amplifier could be used. This means to ligate a T7 polymerase downstream the Minimum Blue Promoter and the GFP downstream a T7 promoter. Additionally, we can increment irradiation time and observe response of our blue light-induced promoter.
Emission module: Red Firefly Luciferase
The red light emitter system that we decided to use is the firefly luciferase, this luciferase usually emits green light but with a punctual mutation we can shift the emission spectrum from green to red.
OBJECTIVES
The final aim was the construction of the red light emission  device,  using the promoter region regulated by LovTAP with the red light emitting luciferase coupled to the Luciferin Regenerating Enzyme (LRE). The structure of the device is designed as follows:
Design LRE biobrick and transfer the plasmid with the synthesised gene from Mr. Gene Plasmid to pSB1C3 DNA submission backbone and assemble it to a constitutive promoter.
To test that the device works as expected.
METHODOLOGY
Mutated Firefly Luciferase
The original intention was to use the Photinus pyralis luciferase biobrick (BBa_I712019) as a base and to increase its efficiency by using the primer BBa_K360114 (as registered) due to previous design notes as referred in the part design section of this briobrick. After achieving this, a punctual mutation S284T (also known as S851T) was induced to change its emission spectrum from green to red by means of a PCR with special primers (which are also referred in the biobrick part design). The biobrick name is BBa_K360115; unfortunately, we were never able to find out why it was not functional.
Click Beetle Luciferase
By the middle of the summer, we decided to try the luciferase from another organism: Click Beetle, specifically the Promega Chroma-Luc™ Reporter Vectors pCBG99-Basic Vector Restriction Sites and the pCBR-Basic Vector Restriction Sites, where the first one has a green emission and the second one red. The intention was to work with these vectors as a means of having a previously-known-to-be functional luciferase, so that once the LRE biobrick was ready, characterization could be feasible. The first step was to extract the coding sequence of the luciferases from these vectors by means of PCR and also to standardize them by adding the prefix and suffix with the following primers:
Forward Click Beetle luciferase
GAATTCGCGGCCGCTTCTAGAGATTAAAGAGGAGAAAATGGTGAAGCGTGAGAAAAAT
Reverse Click Beetle luciferase
CTGCAGCGGCCGCTACTAGTATTATTAACCGCCGGCCTTCT
The second step was to ligate it into the BBa_J61002 plasmid with a J23101 promoter, and to transform it, having as an output several colonies which were then grown overnight to check whether the luciferases showed any brightness. The protocol followed for the assay is mentioned in BBa_K216015 biobrick, in the Experience part. In this first assay, no luminometer was used; so eventhough we added 1μL of 100mM luciferin and waited in the dark room for ~30min taking pictures with a SLR camera with high ISO (~400), for about 30secs to 1 min exposure, nothing was observed. After the initial disappointment, Mariana talked to Cambridge team and they help us ever since then by sending us some of their constructions (BBa_K325909, BBa_K325109, BBa_K325209, and 2 other biobricks).
LRE design
Luciferin Regenerating Enzyme (LRE) was first reported by Gomi, K. and Kajiyama, N. [(2001) Oxyluciferin, a Luminescence Product of Firefly Luciferase, Is Enzymatically Regenerated into Luciferin. The Journal of Biological Chemistry, Vol. 276, No. 39. This enzyme proved to recycle the luciferase product oxyluciferin.  Our goal here was to couple both, luciferase and LRE, to be able to make the construction autonomous after the first and single input of luciferin. We sent to synthesize the LRE enzyme.
For more details see LRE (BBa_K360113) entry at the Registry page.
REFERENCES
Kelly JR, Rubin AJ, Davis JH, Ajo-Franklin CM, Cumbers J, Czar MJ, de Mora K, Glieberman AL, Monie DD, Endy D: Measuring the activity of BioBrick promoters using an in vivo reference standard. J Biol Eng 2009, 3:4.
Yanofsky, C., R.L. Kelley, V. Horn 1984. Repression is relieved before attenuation in the trp operon of Escherichia coli as tryptophan starvation becomes increasingly severe. J.Bacteriol. 158:1018-1024
Strickland, D., Moffat, K., & Sosnick, T. (2008). Light-activated DNA binding in a designed allosteric protein. Proceedings of the National Academy of Sciences, 105(31), 10709.
Third Div Content
Here goes even more info.
iGEM
iGEM is the International Genetically Engineered Machines Competition, held each year at MIT and organized with support of the Parts Registry. See more here .
Synthetic Biology
This is defined as attempting to manipulate living objects as if they were man-made machines, specifically in terms of genetic engineering. See more here .
Genomics
We are students on the Genomic Sciences program at the Center for Genomic Sciences of the National Autonomous University of Mexico, campus Morelos. See more here .
This site is best viewed with a Webkit based Browser (eg: Google's Chrome, Apple's Safari),
or a Gecko one (eg: Mozilla's Firefox, Netscape). Some of the code requires an up-to-date browser.
Trident based (Microsoft's Internet Explorer) or Presto based (Opera) are not currently supported. Sorry.
Retrieved from " http://2010.igem.org/Team:UNAM-Genomics_Mexico/Modules/In_vivo "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
