"
Page
Discussion
View source
History
teams
Log in
 
Team:UNAM-Genomics Mexico/Notebook
From 2010.igem.org
Home
Team
Us
Official Team Profile
Unofficial Team Profile
Them
Project
Overview
Individual Modules
In vivo
In silico
Whole-System View
Parts created
Model created
Official Notebook
Safety & Human Practices
Collaboration
About
iGEM
Synthetic Biology
UNAM
Gallery
Notebook
Our notebook is organized in different main sections to facilitate the access and exchange of information. We will be updating it, so new sections and notes will be added continuously. Besides, all members´ notes are available at our respective Open Wet Ware profiles. Please check the Team page for more information.
Wetlab Progress
inBLUE-outRED Chassis
E.coli strain mutant
We have to test the mutant strain CY15001 (trpR-) that Dr. Charles Yanofsky kindly provided us.  As the mutation consists of a frame shift, we decided to make a PCR reaction and then sequence the amplified products.
The primers used are:
Forward (5'->3'): CGC ACG TTT ATG ATA TGC TAT CG
Reverse:(5'->3'): AGG CCT ACA AAA TCA ATC G
These primers would amplify trpR coding region (326nt), 87nt upstream from the start codon and 12nt after the stop codon. Thus, obtaining a final PCR product of 426nt long.
Samples: The trpR mutant colonies that were selected to make the colony PCR were: 1,5,8.
Colony control: E.coli k12 wild type.
Experimental Procedure
11-13 August 2010
PCR colony: Protocol
1. Take with a toothpick a sample of the colony.
2. Dissolve the sample in 200μL of Tris-ETA 10/1-NaCl 10mM solution.
3. Heat the sample during 10 min at 95°.
4. Centrifugate at 14 000 rpm during 2 min.
5. Take 10μL as DNA template for PCR reaction.
6.The  PCR products  around the expected lenght - 421 nt - for E.coli k12 wt and trpR mutant colonies use them to sequence and analyze the trpR frameshifth mutation.
24 August 2010
Sequencing Results:
The trpR gene from the mutant strain doesn’t have a frameshift, instead of that it has a non synonymous mutation (Alanine to Proline) corresponding to the aminoacid 80. We hypothesized that  this change might causes a dramatic structural change to the protein thus being non functional.
LovTAP promoters
In order to test the new LovTAP, designed considering Dr. Devin advices and Lausanne team modeling results. We had to choose the proper promoters under which LovTAP expression will be regulated.
After looked up the promoter in the registry of biological parts. We decided to use the following members of the family J23: J23117, j23114, J23105 and J23102.
All these promoters are contained inside J61002 plasmid, and control the expression of RFP protein. Besides, the plasmid harbors an ampicillin resistance.
Experimental Procedure
11 August  -  18 August 2010
1.Get the plasmids harboring each promoter, from the corresponding plates.
2.Transform cells.
3.Grow up the transformed cells in LB medium with ampicillin antibiotic. This is because the plasmids harboring each promoter have a resistance against ampicillin.
4. Once the cells are correctly transformed with the plasmid J61002 harboring each promoter,  start the plasmid extraction procedure, using the High Pure Plasmid Isolation kit from Roche.
5.  After finishing the isolation of plasmid J61002 with each promoter ,  make the restriction enzyme assay  (SpeI/PstI)in order to remove the RBS site, the RFP gene and the double terminator from plasmid J61002 to replace them with LovTAP gene.
21 August 2010.
6. Once the plasmids harboring the  promoters are correctly digested with the enzymes SpeI and PstI,  make the dephosphatation reaction for each one in order to prepare them for ligation.
Mr. Gene plasmid harboring  LovTAP
Once we  received the synthesis shipment, we started to work with it in order to assemble it with the selected constitutive promoters.
Experimental Procedure
12 August – 16 August 2010
1.Transform cells with the Mr. Gene plasmid 2.Grow up the transformed cells in LB medium with kanamycin  antibiotic.  3. Once the cells are correctly transformed with the Mr. Gene plasmid,  start the plasmid extraction procedure, using the High Pure Plasmid Isolation kit from Roche.
4.  After finishing the isolation of the plasmid,  make the restriction enzyme assay (XbaI/PstI) in order to remove the LovTAP designed sequence to join it with the respective promoters.
18 August – 23 August 2010
5. As the plasmid from Mr. Gene included the enzyme restriction  sites used in the biobrick assembly kit, extract the corresponding gel band of LovTAP, using the gel extraction protocol from QIAGEN.
LovTAP Mr. gene + Promoters
Assembly of the constitutive promoters with LovTAP Mr. gene sequence
Experimental Procedure
25-26 August 2010.
1. Prepare the ligation mixture taking into account the quantity of the DNA insert -LovTAP- and the receiver DNA -plasmids harboring the promoters-.
2. Incubate the sample at 16°C overnight.
3. Transform the cells. Click here for the protocol.
4. Culture the cells in the proper selective medium.
5. Incubate the petri dishes at 37°C overnight.
6. Re-culture the resultant colonies in the proper selective medium; incubate them at 37°C overnight.
7. Analyze the colonies with Colony PCR to confirm that they contain the correct ligation
Primers:  Forward (5'->3'): Preffix primer.  Reverse:(5'->3'): Suffix primer.
8.  Use SalI restriction enzyme, in order to confirm LovTAP ligations, because there is a recognition site for that enzyme inside LovTAP coding region, approximately at the middle of the gene. The SalI recognition site is not present in RFP gene nor in plasmid J61002. So that cutting with this enzyme, LovTAP ligations will be confirmed as true positives (the insert is LovTAP and not RFP gene).
LovTAP ligations with promoters were correctly performed.
LovTAP Mr. gene + Promoters in different Backbones
The constructions of  LovTAP plus constitutive promoters were correctly obtained in plasmid J61002 but we need to transfer them to plasmid psb1c3 for the iGEM DNA submission and to plasmid psb3k3 for characterization. We sent the constructions of LovTAP + promoters to Edinburgh team in plasmid J61002.
Experimental Procedure
1 September 2010
1.Once the ligations of LovTAP with promoters were confimed, digest them with EcoRI and PstI in order to fuse them to backbones pSB3K3 and pSB1C3.
2.Isolate and digest the plasmids Psb3k3 and psb1c3 with EcoRI and PstI, in order to remove the RFP gene.
3 September 2010.
3.Once the plasmids are correctly digested with the enzymes EcoRI and PstI,  started the dephosphatation reaction in order to prepare them for ligation with promoters-LovTAP. 4. Prepare the ligation mixture taking into account the quantity of the DNA insert -LovTAP- and the receiver DNA -plasmids harboring the promoters-.  5. Incubate the sample at 16°C overnight.  6. Transform the cells. Click here for the protocol.  7. Culture the cells in the proper selective medium. 8. Incubate the petri dishes at 37°C overnight.  9. Re-culture the resultant colonies in the proper selective medium; incubate them at 37°C overnight.
8 – 11 September 2010.
10. Analyze the colonies with Colony PCR to confirm that they contain the correct ligation  Primers:  Forward (5'->3'): Preffix primer.  Reverse:(5'->3'): Suffix primer.
14-29  September 2010.
11.  Use SalI restriction enzyme, in order to confirm LovTAP ligations, because there is a recognition site for that enzyme inside LovTAP coding region, approximately at the middle of the gene. The SalI recognition site is not present in RFP gene nor in plasmids pSB3K3/pSB1C3. So that cutting with this enzyme, LovTAP ligations will be confirmed as true positives (the insert is LovTAP and not RFP gene).
The constructions of LovTAP with promoters were correctly transfered to pSB1C3 and pSB3K3 backbones.
trpL promoter
In order to construct the reporter system regulated by LovTAP, we have to fuse the promoter trpL with a reporter gene.
Experimental Procedure
First attempt
1.Insert the trpL promoter through a PCR reaction to the plasmid Psb3k3.
trpL promoter sequence  tggcaaatattctgaaatgagctgttgacaattaatcatcgaactagttaactagtacgcAagttcacgtaaaaagggtat
Primer designed with the trpL sequence:
Preffix +Promoter(functional elements only)+EcoRI
gaattcgcggccgcttctagag gctgttgacaattaatcatcgaactagttaactagtacgcaag cggaattccg
Zepeda screw it up twice when doing this, the first one was placing an SpeI site instead of an EcRI to use it to ligate to an XbaI site of the cI inverter, but then when he re synthesize it with the SpeI and had successfully ligated it to the cI inverter and GFP, he realized that the promoter itself had two SpeI sites, ergo the ligation could not be used.
Second attempt
3 -4 October 2010
In order to overcome the previous problem Claudia designed a primer to change the SpeI site to an NheI site, which is also compatible with XbaI, the PCR was done on the pSB1C3 plasmid because backbone Psb3k3 has an NheI restriction site. New Primer designed with the trpL sequence:  -Primer_trpL_reverse (5'->3'): NheI site + trpL promoter + XbaI site + EcoRI site TTGCTAGCGTGAACTTGCGTACTAGTTAACTAGTTCGATGATTAATTGTCAACAGCCTCTAGAAGCGGCCGCGAATTC   -Primer forward (5'->3'): Suffix  -Template: Plasmid pSB1C3
7 October 2010.
We tested several Tm temperatures (55°C, 58°C , 60°C, 63°C and 65°C) and plasmid template dilutions because  we were obtaining two amplified bands around the expected size. We got the best results at 63°C and using a dilution of the purified plasmid PSB1C3 of 1:100. So, finally the trpL promoter fused to plasmid pSB1C3 was correctly obtained.
LovTAP Reporter systems
•LovTAP activator activity: Reporter system  trpL promoter fused to lambda Repressor cI: Part:BBa_P0451 + Part:BBa_K098991 regulating GFP protein:Part:BBa_E0240.
Experimental Procedure
First attempt
Zepeda searched in the Registry of Standard Biological Parts for an appropriate inverter which is a repressor followed by its binding site. We chose an inverter because it is going to be constitutively repressing anything downstream of it, but when it gets transcriptionally repressed (by LovTAP), whatever is downstream of it is going to be activated.
We chose the cI inverter from Lambda phage (BBa_Q04510) and proceeded to extract it from the 2009 kit plate 1, Zepeda transformed it by heat shock, extracted plasmid and digested it with EcoRI and PstI.
When checking the digestions on an agarose gel, the 1kb band expected from the cI inverter was absent or very thin, so He performed a PCR using the prefix and suffix as primers.  The PCR came out very well with the expected size. As the double digestions were good enough to ligate, we hypothesize that not all plasmids had the inverter that is why it looks very weak on the digestions, but comes out very well in the PCR.
Zepeda did more PCRs for the cI inverter using RTTH polymerase (hot start), purified them and digested with EcoRI and PstI in order to ligate them to GFP.
In order to introduce the GFP with the new RBS he digested the GPF with SpeI/PstI and plasmid pSB1T3 with XbaI/PstI then he ligate them.
He tried several times to ligate the cI inverter to the GFP-PCR with new RBS in the plasmid pSB1T3 with no success.  As non of the attempts to ligate the 2009 cI inverter PCR were  successful and the digestions were doubtful, he decided to extract the cI inverter from the 2010 kit plate.
After the transformation, plasmid extraction and double digestion with EcoRI/PstI he noticed there was something wrong with the plasmid because the in the gel there were always more that the two bands expected, so he extracted the inverter by PCR, digested it with EcoRI/PstI and cloned it into the commercial vector pBluescript II KS + hoping it would work for the ligations.
It was successfully cloned in the new vector so he proceeded to ligate it with the GFP (BBa_E0240) into the pSB1T3, then  he extracted plasmid and double digested it with EcoRI/PstI, in the gel we can see the two bands expected one for the ligation product about 1900 bp and the other for about 2500 bp corresponding to the plasmid, ergo it seems that it came out well. The only problem is that it doesn’t seem to glow when exposed to UV light, but it should as the cI repressor protein is not being transcribed.
Second attempt
30 August 2010
1.Transform and isolate the plasmids pSB1AK3 and pSB1A2, harboring the parts BBa_P0451 (RBS+cI repressor) and BBa_K098991(cI regulated promoter+RBS+GFP) respectively.
3  September 2010.
2.Once the plasmids are correctly  isolated digest them with the proper enzymes: BBa_P0451 (EcoRI/SpeI) and BBa_K098991 (XbaI/PstI).
6  September 2010.
3.Prepare the ligation mixture taking into account the quantity of the DNA inserts -BBa_P0451 and BBa_K098991- and the receiver DNA -plasmids PSB1C3, PSB1T3 and PSB3K3 digested with EcoRI/PstI -.  4. Incubate the sample at 16°C overnight.  5. Transform the cells. Click here for the protocol.  6. Culture the cells in the proper selective medium. 7. Incubate the petri dishes at 37°C overnight.  8. Re-culture the resultant colonies in the proper selective medium; incubate them at 37°C overnight.
2 October 2010
After several  unsuccessful attempts using different  ligation mixtures with the three backbones aforementioned, we finally obtained a possible well done ligation with the whole cI inverter + GFP in plasmid PSB1C3.
9. Analyze the colonies with Colony PCR to confirm that they contain the correct ligation  Primers:  Forward (5'->3'): Preffix primer.  Reverse:(5'->3'): Suffix primer.
4 October 2010
10. Verify the colony phenotype, it should be expressing GFP, because the cI repressor doesn’t have a promoter, thus it´s not repressing the GFP production.
Although we successfully got the cI inverter we didn’t have enough time to ligate it to trpL promoter due to both constructions are in the same antibiotic resistance plasmid psb1c3 and the PCR reactions to obtain the whole  cI inverter were not obtained on time.
LovTAP repressor activity: Reporter system
trpL promoter fused to GFP protein:Part:BBa_E0240 .
trpL promoter fused to RFP protein: Part: BBa_E1010.
Experimental procedure
30 August 2010 and 4 October 2010.
1.Transform and isolate the plasmids pSB1A2 and pSB2K3, harboring the parts BBa_E0240 (RBS+ GFP + double terminator) and BBa_E1010 (RBS+RFP) respectively.  2. Once the plasmids are correctly  isolated digest them with the proper enzymes for ligation (XbaI/PstI).
7 October 2010.
3.Digest the PCR product of the trpLp plus pSB1C3 with NheI/ PstI and ligate it to both GFP (BBa_E0240) and RFP(BBa_E1010). 4. Incubate the sample at 16°C overnight.  5. Transform the cells. Click here for the protocol.  6. Culture the cells in the proper selective medium. 7. Incubate the petri dishes at 37°C overnight.  8. Re-culture the resultant colonies in the proper selective medium; incubate them at 37°C overnight.
17 October 2010
The phenotype of the transformed  colonies with the ligation, should be RED or yellow because the trpL promoter is not being repressed by LovTAP thus both RFP and GFP are being produced.
18 October 2010.
4.Isolate the plasmid with the correct phenotype and digest it, to test that the insert is around the expected size.
Only trpL + RFP ligation was correctly done according to the phenotype observed and to the restriction pattern obtained once the plasmid was isolated.
Characterizing LovTAP
18-30  October 2010.
Once LovTAP with the three weak constitutive promoters (J23117,123114 and J23105) was correctly obtained in plasmid Psb3k3, and the reporter system trpL+RFP was also finished by our team , we started the co-transformation procedure in order to have the whole system inside the cells both trpR wild type and trpR mutant. Besides, we also receive  the trpL+RFP construction from Lausanne team that was kindly sent it by Edinburgh team, because we didn’t get any response from the members of  Lausanne team to provide us with their reporter systems. We are using both our trpL-RFP reporter system and the Lausanne system as a reference, expecting to obtain the same results. The difference between the reporter systems is that ours doesn’t have the double terminator.
Experimental procedure
The samples used were: trpL-RFP reporter system in plasmid pSB1C3, Lausanne trpL-RFP reporter system, LovTAP under promoters J23117, J23114, J23105 and RFP under J23102.  Qualitative approach TrpR mutant and wild type cells were co transformed using 5 µL of each plasmid (trpL-RFP in pSB1C3/ pSB1A2 and each LovTAP construction in pSB3K3). For each co transformation one  tube containing 5 ml of  LB medium with Kanamicyn and chloramphenicol or Kanamicyn and ampicillin, was inoculated from  a single colony of DH5α. Cultures were grown  overnight (~15hrs) at 37°C with spinning at 250rpm in dark conditions. Then, 1 ml of broth was taken and transferred into 5 ml of fresh LB medium with antibiotics. The cultures were grown for approximately 13 hours under the previous conditions but some were exposed to blue light (470nm) while others were maintained in dark. Finally, the cultures were spun down and compared as follows: the RFP pellets obtained under blue-light versus dark condition, and wild type samples versus mutant samples.
Quantitative approach
TrpR mutant and wild type cells were co transformed using 5 µL of each plasmid (trpL-RFP in pSB1C3/ pSB1A2 and each LovTAP construction in pSB3K3). As controls RFP constitutively expressed under J23102 promoter was used and wild type cells transformed just with trpL-RFP. For each transformation one  tube containing 5 ml of  LB medium with Kanamicyn and chloramphenicol or Kanamicyn and ampicillin, was inoculated from  a single colony of DH5α. Cultures were grown  overnight (~14hrs) at 37°C with spinning at 100rpm under blue light (470nm) conditions. Then cultures were diluted 1:1000 into 5ml of fresh LB media with antibiotics. These were grown for approximately 4.5 hours under the previous conditions under blue light (470nm). After this step the OD600 was measured from each culture.  Based on this OD measurement, the cultures were diluted to the same OD (0.15) in 5 ml of LB fresh media. Then three 200 µL aliquots from each culture were transferred into a flat-bottomed 96 well plate.
Samples were loaded in the plate as is shown in the next picture:
An initial measurement of OD600 and fluorescence (530/25 nm excitation filter, 590/35 nm emission filter) by triplicate was done. Using the light emission device to irradiate the incubator, the plate was maintained during 17 hrs at 37°C with spinning at 35rpm inside the incubator but with a mask in the samples selected for the dark conditions.  Then, a second measurement of OD600 and fluorescence was done. Initial data with the second measurement were compared.  Background absorbance and fluorescence were determined by measuring wells containing only media, and the values obtained were used to normalize the other samples.
Characterizing the blue light inducible promoter from E.coli
In order to test the functionality of our Minimum Blue Promoter we succesfully ligated it to our Strong RBS BBa_K360031 and the GFP BBa_E0040.
Experimental Procedure
We implemented a protocol for testing the response of our Minimum Blue Light Receptor Promoter (Min-BP) in which we irradiated cells with the construction Min-BP + RBS BBa_K360031 + GFP BBa_E0040 with blue light (470 nm) leds for different times. We also irradiated with green (540 nm) and red (660 nm) light leds to discard any crosstalk of these wavelengths. GFP expression was compared to a reference: J23101 promoter + RBS BBa_K360031 + GFP BBa_E0040.
We measured GFP expression of our constructions, in order to irradiate the cells with Min- BP + RBS BBa_K360031 + GFP BBa_E0040 we used blue leds and irradiated the cells for 300 minutes, we also measured GFP expression of cells that were incubated in the dark and cells with the constuction J23101 promoter + RBS BBa_K360031 + GFP BBa_E0040.
For the experiment, cells were incubated overnight in M9 medium with glycerol (0.4%) as carbon source, in the morning OD600 was measured and cells diluted to 0.07 nm. After dilutions, we started irradiation with blue leds. Temperature is a very important factor in the function of the YcgF/YcgE system; it is reported that at 25ºC the ratio between the YcgE repressor and YcgF activator allows a good repression unless there is blue ligh irradiation, so we incubated cells with this promoter at 25ºC. Cells with the constitutive promoter BBa_J23101 were also incubated overnight at 37ºC.
inRED-outGREEN Chassis
LuxAB genes
In order to get LuxAB genes Augusto purified genomic DNA from Vibrio fischeri strain MJ11 and then he ran a PCR with the following primers:
Experimental Procedure
PCR reaction to extract LuxAB genes from vibrio fischeri.
Forward primer   CGG AAT TCG CGG CCG CTT CTAG   AGGA   A ACA GCT   ATG AAG TTT GGA AAT ATT TGT TTT TCG TAT CAA CC
Reverse primer CTG CAG CGG CCG CTA CTA GTA TTA TTA GGG TAG ATT CTT TTC AAT TTT TTG GTT CAA C
In general he did not have  many problems getting luxAB genes from Vibrio fischeri’s genome by PCR. We sent luxAB from V. fischeri to Edinburgh as a PCR product.
The PCR reaction yielded a DNA band of the expected size of luxAB genes which are supposed to be (~2100bp).Besides there were resulting unspecific amplification products.
Once Augusto got luxAB genes from genomic DNA he purified PCR product with High Pure PCR Product Purification Kit by Roche and then he tried to ligate them with different plasmids, all of them containing constitutive promoters. The general protocol followed is this:
1.Digestion of the plasmid with SpeI and PstI in order to ligate luxAB genes in front of the promoter carried by the plasmid.
2.Digestion of the PCR product with XbaI and PstI.
3.Dephosphatation of plasmids to avoid re-annealing and false positives after bacterial transformation.
4.Overnight ligation
5.Bacterial transformation with the ligation left overnight and culturing on selective media.
The different plasmids that he used as vectors are the following ones: •J61002 carrying constitutive promoter J23101 •J61002 carrying constitutive promoter J23102 •J61002 carrying constitutive promoter J23106
He also used plasmids without constitutive promoters (of course these plasmids were digested with EcoRI and PstI or SpeI) such as:
•pSB1C3  •pSB1T3 •BBa_I51020
Because none of these ligations were successful we thought that it could be due to issues with our plasmids from the iGEM DNA submission so we decided to ligate luxAB genes into a commercial plasmid, Augusto tried the ligation with
•pBluescript II KS (+/-) restricted with EcoRI and PstI
However none of these ligations were successful, whenever they resulted in probable colonies, an analysis revealed they were always false positives, it seemed that some other smaller alternative PCR product was being ligated into these vectors.
Because we thought there was some smaller unspecific PCR product with bigger chances of ligation than actual luxAB genes, we decided to make a PCR and then a gel band purification; the resulting purification would contain only luxAB genes so it should avoid the ligation of unspecific products to the vector, nonetheless these ligations never resulted so at the end we were unable to get luxAB genes from Vibrio fischeri.
(It is worthy to note that these ligations were carried out several times by several people, some of them with a lot of experience and it never was successful).
Our collaborator team from Cambridge University sent us a plasmid carrying the whole operon lux (except the regulatory genes) this is luxA, luxB, luxC, luxD, luxE, which means that an E.coli transformated with this plasmid would glow without the need of any exogenous input because it has the genes coding for the bacterial luciferase as well as the genes coding for the enzymes needed for substrate synthesis. This operon was under control of PBAD promoter depending on arabinose to be active so the bacteria transformated with this plasmid had to be grown on arabinose medium in order to observe the blue glowing phenotype.
LuxY (YFP protein)
In order to transfer LuxY construction from Mr. Gene plasmid to pSB1C3 backbone and test that it works correctly, the next protocol was implemented:
Experimental Procedure
19 August 2010
1. Amplify by PCR reaction the LuxY construction, using Mr. Gene plasmid as template and the  prefix and suffix sequences as primers.
2.Digest the PCR product with EcoRI and PstI and ligate it to pSB1C3 backbone previously digested (EcoRI/PstI) and dephosphatated.
3. Incubate the ligation mixture at 16°C overnight.
4. Transform the cells.
5. Culture the cells in the proper selective medium.
6. Incubate the petri dishes at 37°C overnight.
7. Re-culture the resultant colonies in the proper selective medium; incubate them at 37°C overnight.
This ligation was intented several times due to problems with the pSB1C3 backbone samples used and to the ligase enzyme.
8. Analyze the colonies with Colony PCR to confirm that they contain the correct ligation  Primers:  Forward (5'->3'): Preffix primer.  Reverse:(5'->3'): Suffix primer.
9.Isolate the corresponding plasmid and digest it, to test that the insert is around the expected size.
10. Co-transform the plasmid haboring LuxY with LuxAB genes from Vibrio fischeri.
Augusto cotransformated the plasmid from Mr.gene, harboring the LuxY with the plasmid sent by Cambridge containing luxABCDE genes. The cotransformation was successful, but we have not been able to see the yellow glowing phenotype yet, it is still green-blue. We have to improve the experimental conditions, particulary the temperature because LuxY is not functional above 20°C.  We hope to use a CCD camera for the bioluminescence measurment.
inGREEN-outBLUE Chassis
Lumazine
In order to transfer Lumazine construction from Mr. Gene plasmid to pSB1C3 backbone and test that it works correctly, the next protocol was implemented:
Experimental Procedure
3 September 2010
1. Amplify by PCR reaction the Lumazine construction, using Mr. Gene plasmid as template and the  prefix and suffix sequences as primers.
7 September 2010
2.Digest the PCR product with EcoRI and PstI and ligate it to pSB1C3 backbone previously digested (EcoRI/PstI) and dephosphatated.
3. Incubate the ligation mixture at 16°C overnight.
4. Transform the cells.
5. Culture the cells in the proper selective medium.
6. Incubate the petri dishes at 37°C overnight.
7. Re-culture the resultant colonies in the proper selective medium; incubate them at 37°C overnight.
This ligation was intented several times due to problems with the pSB1C3 backbone samples used and to the ligase enzyme.
30 September 2010
8. Analyze the colonies with Colony PCR to confirm that they contain the correct ligation
Primers:  Forward (5'->3'): Preffix primer.  Reverse:(5'->3'): Suffix primer.
9.Isolate the corresponding plasmid and digest it, to test that the insert is around the expected size.
10. Co-transform the plasmid haboring Lumazine with LuxAB genes from Vibrio fischeri.
We cotransformated the plasmid from Mr.gene, harboring the Lumazine gene with the plasmid sent by Cambridge containing luxABCDE genes. The cotransformation was successful, but we have not been able to see the blue shifting glowing phenotype yet, it is still green-blue. Its possible that the wavelenght shift is being correctly done but with our vision we can't see the change, so that we have to use a sensible measuremente device to detect it. We tried with an spectrophotometer but it doesn't detect any RLU (relative light emmission unit), even the bioluminescence is visible to naked eye in the dark room. We hope to use a CCD camera for this measurment.
Journal
These are our Reports. Each link contains the report of a the given person.
All the design and work has been done mainly by the team!
Advisors help us with the general organization, experimental planning, fundraising, miscellaneous help, advances in the construction of the green ligh receptor, human practices planning and modelling.
Report for Augusto Berrocal , dealing with the lux operon.
Report for Jorge Buendia , dealing with the Blue Promoter.
Report for Jorge Zepeda , dealing with the CI inverter used in the Blue, and Red reception modules.
Report for Mariana Ruiz-Velasco , dealing with the luciferase mutation and LRE.
Report for Daniela García , dealing with the Group Modeling.
Report for Fabricio Lopez , dealing with the sponsors, notes, and kappa.
Report for Héctor Medina , dealing with Modeling, Kappa, Simbiology, Logos, Wiki, Patents, Presentation, Research...
TimeLine
This is a timeline of the project, including the overall progress, collaborations, and Human Practices.
Contents
1 Notebook
1.1 Wetlab Progress
1.2 inBLUE-outRED Chassis
1.2.1 E.coli strain mutant
1.2.1.1 Experimental Procedure
1.2.2 LovTAP promoters
1.2.2.1 Experimental Procedure
1.2.3 Mr. Gene plasmid harboring  LovTAP
1.2.3.1 Experimental Procedure
1.2.4 LovTAP Mr. gene + Promoters
1.2.4.1 Experimental Procedure
1.2.5 LovTAP Mr. gene + Promoters in different Backbones
1.2.5.1 Experimental Procedure
1.2.6 trpL promoter
1.2.6.1 Experimental Procedure
1.2.7 LovTAP Reporter systems
1.2.7.1 Experimental Procedure
1.2.8 LovTAP repressor activity: Reporter system
1.2.8.1 Experimental procedure
1.2.9 Characterizing LovTAP
1.2.9.1 Experimental procedure
1.2.10 Characterizing the blue light inducible promoter from E.coli
1.2.10.1 Experimental Procedure
1.3 inRED-outGREEN Chassis
1.3.1 LuxAB genes
1.3.1.1 Experimental Procedure
1.3.2 LuxY (YFP protein)
1.3.2.1 Experimental Procedure
1.4 inGREEN-outBLUE Chassis
1.4.1 Lumazine
1.4.1.1 Experimental Procedure
1.5 Journal
1.6 TimeLine
1.7 iGEM
1.8 Synthetic Biology
1.9 Genomics
iGEM
iGEM is the International Genetically Engineered Machines Competition, held each year at MIT and organized with support of the Parts Registry. See more here .
Synthetic Biology
This is defined as attempting to manipulate living objects as if they were man-made machines, specifically in terms of genetic engineering. See more here .
Genomics
We are students on the Genomic Sciences program at the Center for Genomic Sciences of the National Autonomous University of Mexico, campus Morelos. See more here .
This site is best viewed with a Webkit based Browser (eg: Google's Chrome, Apple's Safari),
or a Gecko one (eg: Mozilla's Firefox, Netscape). Some of the code requires an up-to-date browser.
Trident based (Microsoft's Internet Explorer) or Presto based (Opera) are not currently supported. Sorry.
Retrieved from " http://2010.igem.org/Team:UNAM-Genomics_Mexico/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
