<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNAM-Genomics_Mexico_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNAM-Genomics_Mexico/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNAM-Genomics_Mexico/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNAM-Genomics_Mexico/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNAM-Genomics_Mexico/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNAM-Genomics_Mexico/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNAM-Genomics Mexico/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="overall"><DIV id="maincontainer"><DIV id="topsection"><DIV id="myslidemenu" class="jqueryslidemenu"><UL><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="#">Us</A></LI><LI><A href="https://igem.org/Team.cgi?id=360">Official Team Profile</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Team">Unofficial Team Profile</A></LI></UL><LI><A href="https://igem.org/Team_Wikis?year=2010">Them</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Project">Overview</A></LI><LI><A href="#">Individual Modules</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Modules/In vivo">In vivo</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Modules/In silico">In silico</A></LI><LI><A href="#">Whole-System View</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Parts">Parts created</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Modeling">Model created</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Notebook">Official Notebook</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Safety">Safety &amp; Human Practices</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/Collaboration">Collaboration</A></LI><LI><A href="#">About</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/About/iGEM">iGEM</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/About/Synthetic Biology">Synthetic Biology</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/About/CCG">UNAM</A></LI><LI><A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico/About/Gallery">Gallery</A></LI></DIV></DIV><DIV id="contentwrapper"><DIV id="contentcolumn"><DIV class="innertube"><DIV><H2><SPAN class="mw-headline" id="Overall_project"><B>Overall project</B></SPAN></H2><H3><SPAN class="mw-headline" id="The_Idea"><B>The Idea</B></SPAN></H3><P>Synthetic Biology has been enabling changes in all bio-domains, one such being
communication. Cellular communication has relied since time immemorial on
chemical messengers to exchange information. As such, these messengers regardless
of their scope, are constrained to a chemical system; even far reaching
messengers such as hormones are bound within the chemical system that is
the human body. But this mode is about to change.
</P><P>In this project, our goal is to render the chemical barrier deprecated by
using a non-chemical messenger: photons. These will transport
information between cells that have been designed to sense and emit light,
thus creating a photon-based inter-cellular communication system.
</P><P>These messengers are produced through bio-luminescent reactions, and
are quite capable of traversing multiple environments. Consecuently, this enables
the propagation of information beyond the chemical, biological and even spatial
restrictions. As the messenger is effectively decoupled from the chemical layer,
it is a natural step in the communications bridge between organic-based and
silicon-based systems, such as computers.
</P><H3><SPAN class="mw-headline" id="The_Name"><B>The Name</B></SPAN></H3><P>As you might imagine, WiFi is a play on the popular IEEE 802.11 communications standard knows as Wi-Fi. Since our systems achieves the transfer of information without wires, it is thus wireless. As for fidelity, we shall see.
</P><P>Before you begin wondering on Copyright issues, let us make two things clear:
</P><UL><LI> Our name is WiFi, not the <A href="http://www.wi-fi.org/" class="external text" rel="nofollow">Wi-Fi Alliance</A>'s Wi-Fi (notice the hyphen).
</LI><LI> Second, a quick search on the US Patents &amp; Trademarks Office <A href="http://tess2.uspto.gov/" class="external text" rel="nofollow">Trademark Electronic Search System (TESS)</A> returned that WiFi was a trademark in 2006, but is now listed as &quot;Dead&quot;. Therefore, we are not infringing copyright issues by using it in our system.
</LI></UL><H2><SPAN class="mw-headline" id="Project_Details"> Project Details</SPAN></H2><P>The process of transferring information from a sender entity to a receiver one through a determined channel is called communication. Biological entities have relied since time immemorial on chemical messengers to relay information; this holds true for multicellular organisms as well as for populations of unicellular organisms. Being chemical based, these messengers are constrained to a chemical system regardless of the scope of said system, eg: even far reaching messengers such as hormones are bound within the chemical system that is the human body.
</P><P>In this project, our goal is to render the chemical barrier deprecated by enabling chemical-free communication. This has been translated to the implementation of a non-chemical messenger, in this case, photons. Our channel is thus light based; packages of photons, or energy quanta, will transport information from senders to receivers, effectively bypassing most chemical barriers in-between. Consequently, our communicating system is no longer contained within a chemical system, but within a physical one, ie: there must remain a physical channel where photons can be transported. This physical channel may range from something as sophisticated as a microcontroler-based electronic relay system, to something as simple as vacuum (or void). However, this physical layer proves very well to be impervious to most chemical signaling. Ergo, the chemical system's signaling would remain unaffected by the physical channel, and vice versa. In consequence, the exchange of information through physical means is sufficiently independent from the information encoded in the system's endogenous chemical pathways. In other words, it is extraordinarily uninvasive. As an added bonus, our receiver entities are easily transformed into emitter entities. Thus, by using our cells as information processing chassis, we can expand the communications layer. We can effectively render our system one where information is:
</P><UL><LI>Encoded and sent by an emitter
</LI><LI>Recieved and decoded by a receiver
</LI><LI>Plus processed, transformed, and relayed forth
</LI></UL><P>Our ambicious implementation is based on well known systems, mainly bioluminescent proteins from Photinus pyralis and Vibrio fischeri, as well as photoactive receptors like Cyanobacteria cyanobacteriochromes and Light-Oxygen-Voltage domain quimeric proteins. We thus exploit the fact that cells already display primitive photo-communication, both within multicellular organisms as well as within populations of unicellular ones. Moreover, in our system the photonic information is transformed to and from chemical information within the chemical system that is an individual cell. Thus, the chemical barrier that is the membrane has ceased being a barrier to communication and is now a noise isolator. By decoupling the messenger from the chemical layer, we enable a brand new host of applications that were previously unavailable, ranging in domains from neurobiology, to cybernetic coupling, and even to biological telecommunications.
</P><H2><SPAN class="mw-headline" id="Module_Logic">Module Logic</SPAN></H2><P>We decided to break down our device into 3 sub-devices: Reception, Emission, and Transmission. The rationale is as follows: the machinery that transforms the red input into chemical information is independent from the machinery that transforms chemical information into green output, and both are quite different from what transmits the information. Therefore, we can work with &amp; model these six sub-devices.
</P><H2><SPAN class="mw-headline" id="Individual_Modules">Individual Modules</SPAN></H2><P>To learn more about the modules, here's a short description on them.
</P><FORM name="dropmsgform"><DIV id="dropmsg0" class="dropcontent"><H3><SPAN class="mw-headline" id="Red_Reception">Red Reception</SPAN></H3><H4><SPAN class="mw-headline" id="Description">Description</SPAN></H4><P>The cyanobacterial phytochrome (Cph1) fused to the EnvZ histidine kinase domain from E.coli, makes up the chimaeric photoreceptor Cph8 constructed by Levskaya and collaborators. This photosensor requires a specific linear tetrapyrrole cofactor, phycocyanobilin (PCB), to detect red light.  This chromophore has two states, and light triggers the passage of state. Thus, under dark conditions Cph8 shows kinase activity; under light conditions it does not.
</P><P>The light responsive domain (Cph1) has maximal response to light near 660nm. Cph8's substrate is OmpR, a well studied Transcription Factor. When phosphorilated it shows greater affinity for DNA. OmpR regulates two promoters in an antagonistic way: in high concentrations of active OmpR, OmpC is active and OmpF is repressed. In low concentrations of active OmpR, OmpC is repressed and OmpF is active.
</P><P>We plan on constructing our reporter genes under the OmpF promoter and starting our system in a &lt;Dark&gt; state (where active OmpR concentration is high). Thus we hope to achieve an &lt;IF Light&gt; logic gate by using Cph8 as a sensing mechanism, and OmpF as a response one.
</P><P>See also, <A href="http://partsregistry.org/Coliroid" class="external text" rel="nofollow">Coliroid</A>.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade">Signaling Cascade</SPAN></H4><P>When our device is struck by red light, the following cascade will ensure:
</P><UL><LI> Photon input
</LI><LI> PCB conformation change
</LI><LI> EnvZ kinase activity abolished
</LI><LI> Phosphorilated OmpR concentration collapse
</LI><LI> Pops output
</LI></UL><H4><SPAN class="mw-headline" id="References">References</SPAN></H4><P>Levskaya, A., Chevalier, A. A., &amp; Tabor, J. J. (2005). Engineering Escherichia coli to see light. Nature, 438(7067), 442. doi: 10.1038/438442a.
</P></DIV><DIV id="dropmsg1" class="dropcontent"><H3><SPAN class="mw-headline" id="Green_Emission">Green Emission</SPAN></H3><H4><SPAN class="mw-headline" id="Description_2">Description</SPAN></H4><P>Green Emission is composed of a series of enzymes that generate light by the oxidation of a substrate. Our sub-device has 6 enzymes (LuxA, LuxB, LuxC, LuxD, LuxE, LuxY), two catalyze the oxidation step (LuxA, LuxB), one adjusts the emission spectrum (YFP), and three generate and recycle the substrate (LuxC, LuxD, LuxE). This complex converts fatty acids to aldehydes which are in turn used as a substrate by bacterial luciferase to emit light.  
</P><P>We plan on having the adjusting enzyme, as well as the 3 regenerating enzymes expressed constitutively. We would then only use the oxidation enzymes as reporters for whatever event we are observing.
</P><P>While the oxidation per se does not generate light, it does generate an intermediate molecule in an electronically exited state. When said molecule returns to a basal energy state, a photon is released.
</P><P>This light emission system has been taken from the bacterium Vibrio fischeri.
The natural emission spectrum of Vibrio fischeri is blue, nevertheless this strain must emit green light, the spectrum shift is achieved by means of an &quot;antenna&quot; protein called YFP (taken from Vibrio fischeri strain Y-1) which receives the light emitted by bacterial luciferase and then emits light of a different wavelength (yellow in this case), YFP uses NADPH as a substrate.
</P><P>
See also the work of <A href="https://2009.igem.org/Team:Edinburgh/biology(biobricks)" class="external text" rel="nofollow">Edinburgh 2009</A>.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade_2">Signaling Cascade</SPAN></H4><P>When our device recieves Pops, the following cascade will ensure:
</P><UL><LI> Pops intput
</LI><LI> Transcription of genes downstream of promoter: LuxA &amp; LuxB
</LI><LI> Oxidation of substrate
</LI><LI> Photon output
</LI></UL></DIV><DIV id="dropmsg2" class="dropcontent"><H3><SPAN class="mw-headline" id="Green_Reception">Green Reception</SPAN></H3><H4><SPAN class="mw-headline" id="Description_3">Description</SPAN></H4><P>Green Reception is composed of a two-component system. Firstly, we have a sensing agent (CcaS). This protein shows two basal states, both with histidine kinase activities but each with an affinity for different substrates: a phenomenon known as photoconversion. We plan on using the Green phase regulator (CcaR) who happens to be a Transcription Factor.
</P><P>When our regulator is in a phosphorilate state, it shows greater affinity for DNA. Thus it is active. The target promoter region has been recently identified. We thus plan on constructing our reporter genes under this promoter. Such a construction would be an &lt;IF Light&gt; logic gate. 
</P><P>This two-component system has been described in Synechocystis sp. PCC 6803, and due to the homology relationship with
the EnvZ-OmpR system, it might possibly be functional in E.coli. 
</P><P>See also <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2474522/" class="external text" rel="nofollow">this paper</A>.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade_3">Signaling Cascade</SPAN></H4><P>When our device is struck by green light, the following cascade will ensure:
</P><UL><LI> Photon input
</LI><LI> CcaS switches to Green conformation
</LI><LI> Kinase activity starts
</LI><LI> Phosphorilated CcaR concentration builds up
</LI><LI> Pops output
</LI></UL></DIV><DIV id="dropmsg3" class="dropcontent"><H3><SPAN class="mw-headline" id="Blue_Emission">Blue Emission</SPAN></H3><H4><SPAN class="mw-headline" id="Description_4">Description</SPAN></H4><P>Blue Emission is composed of a series of enzymes that generate light by the oxidation of a substrate. Our sub-device has 6 enzymes (LuxA, LuxB, LuxC, LuxD, LuxE, LumP), two catalyze the oxidation step (LuxA, LuxB), one adjusts the emission spectrum (LumP), and three generate and recycle the substrate (LuxC, LuxD, LuxE). We plan on having the adjusting enzyme, as well as the 3 regenerating enzymes expressed constitutively. We would then only use the oxidation enzymes as reporters for whatever event we are observing.
</P><P>While the oxidation per se does not generate light, it does generate an intermediate molecule in an electronically exited state. When said molecule returns to a basal energy state, a photon is released. Likewise, LumP does not actually shift the spectrum, but the enzyme generates a substrate that does.
</P><P>As you may imagine, these genes (sauf LumP) constitute an Operon. This is the Lux Operon from Vibrio fischeri.
</P><P>See also the work of <A href="https://2009.igem.org/Team:Edinburgh/biology(biobricks)" class="external text" rel="nofollow">Edinburgh 2009</A>.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade_4">Signaling Cascade</SPAN></H4><P>When our device receives Pops, the following cascade will ensure:
</P><UL><LI> Pops input
</LI><LI> Transcription of genes downstream of target promoter: LuxA &amp; LuxB
</LI><LI> Oxidation of substrate
</LI><LI> Photon output
</LI></UL></DIV><DIV id="dropmsg4" class="dropcontent"><H3><SPAN class="mw-headline" id="Blue_Reception">Blue Reception</SPAN></H3><H3><SPAN class="mw-headline" id="LovTAP">LovTAP</SPAN></H3><H4><SPAN class="mw-headline" id="Description_5">Description</SPAN></H4><P>Blue Reception is composed of a slightly complicated system. Firstly, we have a quimeric sensing protein (LovTAP). This protein is composed of a sensing domain (a Light-Oxygen-Voltage domain) as well as the DNA-binding domain of the TrpR transcriptional repressor from the Triptophan pathway. 
</P><P>LOV domains bind a flavin-mononucleotide (FMN) or flavin-adenine-dinucleotide (FAD) cofactor, which are used in a wide variety of metabolic pathways as cofactors in redox reactions and are available in most organisms. The cofactor has a broad absorption spectrum, with a maximum at 450 nm. 
</P><P>
Under the presence of light, absorption of a photon leads to the formation of a covalent adduct between the flavin mononucleotide (FMN) cofactor and a conserved cysteine residue in the AsLOV2 domain, which results in conformational rearrangements in LovTAP. This change impacts the affinity of the shared helix for the two domains: disrupting the contacts between the shared helix and the LOV domain and enabling the association of the shared helix with the TrpR domain, which establishes  DNA-binding affinity at the trpL promoter and LovTAP can then bind to DNA as an homodimer, repressing the transcription of the genes downstream of the promoter.
</P><P>In the dark, when the shared helix contacts the LOV domain, the TrpR domain's DNA-binding affinity decreases and LovTAP is in an inactive conformation.
</P><P>Our construction is consequently based on an &quot;inhibit the inhibitor&quot; logic. By constructing our reporter genes under a repressed promoter (cI binding site), and having LovTAP repress the inhibitor of said promoter (cI lambda repressor), we establish a direct &lt;IF Light&gt; logic gate.
</P><P>See also <A href="http://partsregistry.org/Part:BBa_K191003" class="external text" rel="nofollow">LovTAP</A>.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade_5">Signaling Cascade</SPAN></H4><P>When our device is struck by blue light, the following cascade will ensure:
</P><UL><LI> Photon input
</LI><LI> Lov-domain conformational change
</LI><LI> LovTAP dimerization
</LI><LI> trpL promoter is repressed
</LI><LI> Concentration of cI repressor collapses
</LI><LI> trpL promoter is free
</LI><LI> Pops output
</LI></UL><H4><SPAN class="mw-headline" id="References_2">References</SPAN></H4><P>Strickland, D., Moffat, K., &amp; Sosnick, T. (2008). Light-activated DNA binding in a designed allosteric protein. Proceedings of the National Academy of Sciences, 105(31), 10709. National Acad Sciences. Retrieved from <A href="http://www.pnas.org/content/105/31/10709.full" class="external free" rel="nofollow">http://www.pnas.org/content/105/31/10709.full</A>.
</P><P>STRICKLAND, D. (2009). NEW APPROACHES TO THE DESIGN OF ALLOSTERIC PROTEINS.
</P><H3><SPAN class="mw-headline" id="YcgF.2FYcgE_blue_reception_system">YcgF/YcgE blue reception system</SPAN></H3><H4><SPAN class="mw-headline" id="Description_6">Description</SPAN></H4><P>The YcgF/YcgE system is based on the action of the repressor YcgE, which is bound to
the promoter region when there is no blue light, thus inhibiting the transcription of any
gene downstream this promoter.
</P><P>In the presence of blue light, YcgF dimerizes and now it has a great affinity for YcgE,
clearing the promoter and allowing transcription to proceed.
</P><P>It is reported that the response of this promoter is weak in comparison to some others
standard strong promoters registered in the Registry of Standard Biological Parts.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade_6">Signaling Cascade</SPAN></H4><UL><LI> Photon input
</LI><LI> YcgF dimerization
</LI><LI> Binding of YcgF dimer with YcgE.
</LI><LI> Blue promoter is free
</LI><LI> Pops output
</LI></UL></DIV><DIV id="dropmsg5" class="dropcontent"><H3><SPAN class="mw-headline" id="Red_Emission">Red Emission</SPAN></H3><H4><SPAN class="mw-headline" id="Description_7">Description</SPAN></H4><P>Red Emission is composed of mainly two enzymes. Our first enzyme (Luciferase) is a mutated form of the wild type enzyme found in Photinus pyralis. Our mutant is expected to glow red instead of the wild type blue-green. This enzyme catalyzes the oxidation reaction that yields light. The substrate for this reaction (Luciferin) is a most complicated molecule, and to our knowledge no one has ever managed to produce it within an E. coli chassis. Therefore, we need to inoculate the medium with luciferin to enable the reactions. However, recently a new enzyme was discovered (LRE) that recycles luciferin. We thus need only an initial inoculation with luciferin and from there on, the system is sufficiently autonomous.
</P><P>We plan on using Luciferase as a reporter gene, while having LRE expressed constitutively.
</P><P>For the Red Emission mutation, see <A href="http://dx.doi.org/10.1016/j.ab.2005.07.015" class="external text" rel="nofollow">this paper</A>.
</P><P>For the Red Emission protein, see <A href="http://partsregistry.org/Part:BBa_I712019:Design" class="external text" rel="nofollow">this BioBrick part</A>.
</P><H4><SPAN class="mw-headline" id="Signaling_Cascade_7">Signaling Cascade</SPAN></H4><UL><LI> Pops input
</LI><LI> Luciferase downstream of promoter is transcribed
</LI><LI> Oxidation of substrate
</LI><LI> Photon output
</LI></UL></DIV></FORM></DIV></DIV></DIV></DIV><DIV id="leftcolumn"><DIV class="innertube"><DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Overall_project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overall project</SPAN></A></LI><LI class="toclevel-2"><A href="#The_Idea"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Idea</SPAN></A></LI><LI class="toclevel-2"><A href="#The_Name"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The Name</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Project_Details"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-1"><A href="#Module_Logic"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Module Logic</SPAN></A></LI><LI class="toclevel-1"><A href="#Individual_Modules"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Individual Modules</SPAN></A></LI><LI class="toclevel-2"><A href="#Red_Reception"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Red Reception</SPAN></A></LI><LI class="toclevel-3"><A href="#Description"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-3"><A href="#References"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2"><A href="#Green_Emission"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Green Emission</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_2"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade_2"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-2"><A href="#Green_Reception"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Green Reception</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_3"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade_3"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-2"><A href="#Blue_Emission"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Blue Emission</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_4"><SPAN class="tocnumber">4.4.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade_4"><SPAN class="tocnumber">4.4.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-2"><A href="#Blue_Reception"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Blue Reception</SPAN></A></LI><LI class="toclevel-2"><A href="#LovTAP"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">LovTAP</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_5"><SPAN class="tocnumber">4.6.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade_5"><SPAN class="tocnumber">4.6.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-3"><A href="#References_2"><SPAN class="tocnumber">4.6.3</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2"><A href="#YcgF.2FYcgE_blue_reception_system"><SPAN class="tocnumber">4.7</SPAN><SPAN class="toctext">YcgF/YcgE blue reception system</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_6"><SPAN class="tocnumber">4.7.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade_6"><SPAN class="tocnumber">4.7.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-2"><A href="#Red_Emission"><SPAN class="tocnumber">4.8</SPAN><SPAN class="toctext">Red Emission</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_7"><SPAN class="tocnumber">4.8.1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-3"><A href="#Signaling_Cascade_7"><SPAN class="tocnumber">4.8.2</SPAN><SPAN class="toctext">Signaling Cascade</SPAN></A></LI><LI class="toclevel-1"><A href="#iGEM"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">iGEM</SPAN></A></LI><LI class="toclevel-1"><A href="#Synthetic_Biology"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Synthetic Biology</SPAN></A></LI><LI class="toclevel-1"><A href="#Genomics"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Genomics</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV id="maincontainer2"><DIV id="contentwrapper2"><DIV id="contentcolumn2"><DIV class="innertube2"><DIV><H2><SPAN class="mw-headline" id="iGEM">iGEM</SPAN></H2>
iGEM is the International Genetically Engineered Machines Competition, held each year at MIT and organized with support of the Parts Registry. See more <A href="/Team:UNAM-Genomics_Mexico/About/iGEM" title="Team:UNAM-Genomics Mexico/About/iGEM">here</A>.</DIV></DIV></DIV></DIV><DIV id="leftcolumn2"><DIV class="innertube2"><DIV><H2><SPAN class="mw-headline" id="Synthetic_Biology">Synthetic Biology</SPAN></H2>
This is defined as attempting to manipulate living objects as if they were man-made machines, specifically in terms of genetic engineering. See more <A href="/Team:UNAM-Genomics_Mexico/About/Synthetic_Biology" title="Team:UNAM-Genomics Mexico/About/Synthetic Biology">here</A>.</DIV></DIV></DIV><DIV id="rightcolumn2"><DIV class="innertube2"><DIV><H2><SPAN class="mw-headline" id="Genomics">Genomics</SPAN></H2>
We are students on the Genomic Sciences program at the Center for Genomic Sciences of the National Autonomous University of Mexico, campus Morelos. See more <A href="/Team:UNAM-Genomics_Mexico/About/CCG" title="Team:UNAM-Genomics Mexico/About/CCG">here</A>.</DIV></DIV></DIV></DIV></DIV></DIV><P class="left">
This site is best viewed with a Webkit based Browser (eg: Google's Chrome, Apple's Safari),</P><DIV class="left">
or a Gecko one (eg: Mozilla's Firefox, Netscape). Some of the code requires an up-to-date browser.
Trident based (Microsoft's Internet Explorer) or Presto based (Opera) are not currently supported. Sorry.
</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UNAM-Genomics_Mexico/Project">http://2010.igem.org/Team:UNAM-Genomics_Mexico/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNAM-Genomics_Mexico/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNAM-Genomics_Mexico/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNAM-Genomics_Mexico/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNAM-Genomics_Mexico/Project&amp;oldid=177612" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>