<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Delft_Peptides"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Delft/Peptides">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Delft/Peptides&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Peptides&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Peptides&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Delft/Peptides" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Delft/Peptides</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><UL id="menu"><LI><A href="https://2013.igem.org/Team:TU-Delft">Home</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/ProjectOverview">Overview</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Sensing">Sensing Device</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Peptides">Peptide Production</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/PeptideCharacterization">Peptide Characterization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Timer">Timer</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Killswitch">Kill switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/FinalApplication">Band Aid Application</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/new_Standard">New Standard Approach</A></LI></UL><LI><A href="https://2013.igem.org/Team:TU-Delft/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Planning">Planning</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Timer_Plus_Sumo">Timer plus SUMO</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Timer-Sumo-KillSwitch">Timer-SUMO-Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/BandAid">Band Aid</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/NovelPeptides">Novel Peptides</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/human practice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/novel_approach">Novel Approach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/BandAidSelection">BandAid</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Zephyr"> Zephyr  </A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Zephyr_How"> Zephyr: How? </A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/SynBioData">Wiki Interviews</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Meet_with_RIVM">Meeting with RIVM</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Summer Festival">Human Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Extras">Extras</A></LI><LI><A href="https://2013.igem.org/Http://2013.igem.org/Team:TU-Delft/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/PR">PR</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/iGemNetherlandsCollaboration">iGEM Netherlands</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/MeetChristopherVoigt">Meet with Christopher Voigt</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Gallery">Photo Gallery</A></LI><LI><A href="https://2013.igem.org/Team:TU-Delft/Brainstorm">Brainstorm</A></LI><DIV style="margin-left:30px;margin-right:30px;margin-top:-30px; width:900px;float:left;"><H2 align="center">Peptide production</H2><P align="justify"> 
The era of antibiotics are now outdated as the microorganisms started gaining new resistance mechanism to evade the effects of antibiotics. Nowadays more and more researches are focused towards new antimicrobial that can be more effective against these pathogens.</P><P align="justify"> 
The antimicrobial peptides (AMP), natural or de-novo are becoming more popular due to their potential to kill or restrict the growth of microorganisms. They form a part of chemical defense against predators. All organisms, including higher order eukaryotes have some antimicrobial peptides that acts against specific strains of microorganisms.They are usually charged and interact with the opposite charges present on the membrane thus boring a hole through the membrane subsequently lysing the  cell <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [1]</A>.Unlike antibiotics, most AMPs do not interfere in the genetic, transcriptional or translational machinery of the cell.
</P><P align="justify"> 
We chose 3 AMP that are produced in the skins of toads and frogs. They are magainin (GIGKFLHSAKKFGKAFVGQIMNS) from <I>Xenopus laevis</I> , signiferin (IIGHLIKTALGMLGL) from <I>Crinia signifera</I> and maximin-H5 (ILGPVLGLVSDTLDDVLGIL) from <I>Bombina maxima</I>. The reasons for choosing these AMP are because of their target specific nature, charge and length. The peptides are described in literature as being specifically toxic to certain species, in this case a low MIC for Staphylococci. The widely accepted theory behind this selectivity is the idea the peptides have a localized charge in either their C- or N-terminal part. This charge is needed to interact with the membrane of the cell before they can bore holes in them. Another important criterion used in selection of the peptides has been the published toxicity towards mammalian cells with human cells in specia, which is important in making them possible candidates as pharmaceuticals<A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [2]</A>. 
</P><P align="justify"> 
The recombinant production of these peptides in <I>E.coli</I> would help large scale production of these peptides and administer them as antimicrobials <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [5]</A>. But, the major hurdle in producing them in vivo is the formation of inclusion bodies as they are charged. This could be overcome by making protein fusion with SUMO (Small Ubiquitin like Modifier), MBP (Maltose Binding Protein), Trx (Thioredoxin) etc. 
</P><P align="justify"> 
In this project, a SUMO-peptide fusion was opted <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [3]</A> as a suitable expression system as they make the fusion proteins more soluble. The  Ulp-1 cleaving the fusion is a special protease that can cleave of the fusion without leaving any scar. This protease is also expressed and controlled by a negative transcriptional cascade. As the fusion is produced it triggers a cascade that ultimately leads to the expression of Ulp-1 and facilitates in vivo cleavage. This fusion increases the solubility of the peptide and reduces inclusion bodies formation.(Figure 1).
</P><CENTER><P>Figure 1: Schematic diagram of the peptide production </P></CENTER></DIV><DIV style="margin-left:30px;margin-right:30px; width:900px;float:left;"><H2 align="center">SUMO-Peptide Production</H2><B>Aim:</B>
Increasing solubility of peptide with Sumo Fusion.
<B>Description</B><P align="justify">
The peptide by itself is not soluble in the cytoplasm but making a fusion of peptide with Small Ubiquitin like Modifiers (SUMO) will increase the solubility of the peptide, thus increasing the cytoplasmic fraction of the peptide. 
</P><P align="justify">
A gene was constructed in such a way that the SUMO-peptide production was driven by the strong T7 phage promoter. This gene containing plasmid was harboured in a BL21(DE3) strain that has lac promoter driven T7 polymerase. Upon induction by IPTG the SUMO peptide fusion is produced as a soluble protein fraction.
</P><CENTER><P>Figure 2: Part BBa_K1022101/2/3 </P></CENTER></DIV><DIV style="margin-left:30px;margin-right:30px; width:900px;float:left;">   
The protocol can be seen <A href="https://2013.igem.org/Team:TU-Delft/Protocol_10#protocol_10" target="blank"> here</A>. 

<H4>Results</H4><P align="justify">
The presence of plasmid with gene inserts encoding the SUMO-peptide (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022101" target="blank">BBa_K1022101</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022102" target="blank">BBa_K1022102</A>,<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022103" target="blank">BBa_K1022103</A>) was confirmed by a colony pcr. The expected size of the insert with the T7 promoter was approximately around 400 bp which was clearly evident from the agarose gel picture Figure 3. An Eurogentec Smartladder MW-1700-10 (<A href="https://secure.eurogentec.com/uploads/TDS-MW-1700-10.pdf">https://secure.eurogentec.com/uploads/TDS-MW-1700-10.pdf</A>) was used to identify the size of the fragments.
</P><CENTER><P>Figure 3:Colony PCR SUMO Peptide</P></CENTER><P align="justify">
Then strains were subjected to protein expression by means of IPTG induction. The cell lysate and the centrifuged cell debris were run on a 16 % tricine gels at 90 V for 90 minutes. The figure <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig4">4</A> clearly shows a over-expression of our fusion protein  at ~15 kDa both in whole cell lysate (CL) and the membrane (M), This was from just 10 µg of total protein for cell lysate (CL) and 30 µg of total protein in the membrane (M).
</P><P align="justify">
Then just the induced CL and M were run with 30 µg of total protein per well as in figure <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig5">5</A> the results were again consistent (Black circle). One of the over-expressed band was excised and subjected for in-gel tryptic digestion and then to MS/MS measurement in Q-trap MS instrument (Thermo Scientific).  To prove the solubility of the peptide the the CL and M from constructs that express peptide without SUMO was run with the CL from SUMO- peptide as in figure <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig6">6</A>.  
</P><CENTER><P>Figure 4: Uninduced and induced SUMO-Peptide (signiferin <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022102" target="blank">BBa_K1022102</A>)  CL: Cell lysate, M: Membrane pellet </P></CENTER><CENTER><P>Figure 5: SUMO-Peptide (signiferin <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022102" target="blank">BBa_K1022102</A>) with IPTG induction  CL: Cell lysate, M: Membrane pellet</P></CENTER><CENTER><P>Figure 6: SUMO-Peptide (signiferin <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022102" target="blank">BBa_K1022102</A>) solubiltyLane 1: Low molecular weight marker, Lane 2: IPTG induced SUMO-peptide cell lysate fraction (CL)Lane 4: Uninduced peptide-his without SUMO in cell lysate (CL), Lane 6:Uninduced peptide-his without SUMO in membrane pellet(M) Lane 7:Induced peptide-his without SUMO in cell lysate (CL), Lane 9:Induced peptide-his without SUMO in membrane pellet (M)  </P></CENTER><H4>Discussion</H4><P align="justify">
The SUMO domain was added to the peptide to improve its solubility <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [3]</A>. This was really evident from the figure (<A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig4">4</A>, <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig5">5</A>). The peptides without SUMOylation did not have any over-expression around ~1.5 kDa (only peptide)  in the cell lysate which is clearly seen in the figure <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig6">6</A> [ lanes 4,6 (uninduced) 7,9 (induced)] . But when a SUMO fusion of the same peptide was expressed they appeared as a thick band in the cell lysate as in the figure <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig6">6</A> [lane 2,black box] . From the figures [<A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig4">4</A>,<A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig5">5</A>] it is evident that there was some fraction of protein left with the cell debris. This clearly explains that SUMO does not increase the solubility to 100 % but just enhances the fraction of the soluble protein, thus facilitating easy purification. The band which was subjected to the MS/MS measurements clearly gave a 95 % sequence coverage to SUMO-peptide (signiferin) as in figure <A href="https://2013.igem.org/Team:TU-Delft/Peptides#fig7">7</A>. Thus, we prove that the SUMO increases the solubility and also they are not toxic to the cell.
</P><CENTER><P>Figure 7: Sequence coverage from Tandem MS [Red letters]</P></CENTER><H4>Highlights</H4><P align="justify">
The SUMO domain that was used in our project was synthesized as a gene from DNA 2.0. The SUMO was designed to be bio-brick compatible without the restriction sites that are present in the prefix and suffix.  The restriction sites where removed without disturbing the codons.
</P><P align="justify">
The Biobrick was designed in such a way that the ease of cloning SUMO fusion  protein to be expressed has been improved. An Age I restriction site was introduced at the end of SUMO which when translated to amino acids Tyrosine Glycine Glycine. This is the recognition sequence of the Ulp-1<A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [4]</A> protease that can cleave the peptides from the SUMO without any scar. So, one can use Age I and any one of the suffix enzyme to make a SUMO fusion.
</P><H2 align="center">SUMO cleavage by Ulp1</H2><DIV style="margin-left:30px;margin-right:30px; width:900px;float:left;"><H4>Cleavage of SUMO from Peptide by Ulp-1 protease.</H4><P align="justify">
The SUMO-peptide helps in increasing the soluble fraction of peptide but peptides are not biologically active in a fusion, they have to be cleaved from the fusion to get a active peptide fraction. This was achieved by in vivo production on SUMO specific Ulp-1 proteases.  
</P><P align="justify"> A gene was constructed in such a way that the SUMO-peptide production was driven by the strong T7 phage promoter and the Ulp-1 production was driven by arabinose inducible promoter pBad. The plasmid was transformed into an BL21(DE3) pLysS strain. This construct was designed to check whether in vivo cleavage is possible. The main idea of the experiment is to first produce large amount of soluble fraction of  SUMO-peptides and the produce the Ulp protease to cleave the sufficiently produced fusion peptides. </P><CENTER><P>Figure 8: Part BBa_K1022116/117/118</P></CENTER></DIV><DIV style="margin-left:30px;margin-right:30px; width:900px;float:left;">   
The protocol can be seen <A href="https://2013.igem.org/Team:TU-Delft/Protocol_11#protocol_11" target="blank"> here</A>. 

<H4>Results</H4><P align="justify"> The presence of plasmid with gene inserts encoding the SUMO-peptide (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022116" target="blank">BBa_K1022116</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022117" target="blank">BBa_K1022117</A>,<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022118" target="blank">BBa_K1022118</A>) was confirmed by a colony pcr. The expected size of the insert  was approximately around ~1500 bp which was clearly evident from the agarose gel picture figure(<A href="https://2013.igem.org/Team:TU-Delft/Peptides#ulp1">9,10,11</A>). Though the actual size of the inserts are ~1300 bp the use of sequencing primers VF2 and VR which bind 100-150 base pair away from suffix and prefix the bands are around 1500 bp. An Eurogentec Smartladder MW-1700-10 [lane 1] (<A href="https://secure.eurogentec.com/uploads/TDS-MW-1700-10.pdf" target="blank">https://secure.eurogentec.com/uploads/TDS-MW-1700-10.pdf</A>) was used to identify the size of the fragments.</P><CENTER><P>Figure 9: pBad Ulp SUMO maximin-H5 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022116" target="blank">BBa_K1022116</A>) Colony PCR </P></CENTER></DIV><CENTER><P>Figure 10:pBad Ulp SUMO magainin (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022118" target="blank">BBa_K1022118</A>) colony PCR </P></CENTER></DIV><CENTER><P>Figure 11:pBad Ulp SUMO signiferin (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1022117" target="blank">BBa_K1022117</A>) colony PCR </P></CENTER></DIV><DIV style="margin-left:30px;margin-right:30px; width:900px;float:left;"><H4>Discussion</H4><P align="justify">
The cleavage of the peptide from the SUMO is more crucial to free the peptide and make it biologically active. But if we look into the structure of the peptides, they are quite hydrophobic to be present as free peptides in the medium. So, a classical SDS page analysis is not suitable for these peptides. This made us to analyse our whole cell lysates with tandem MS approach. The cell lysates devoid of debris was subjected to MS/MS measurements. which gave a 40 % sequence coverage to the SUMO without the peptide as in <A href="https://2013.igem.org/Team:TU-Delft/Peptides#figure12">figure 12</A>. The free peptide was not present intact in the solution. This could be attributed to the hydrophobicity of the free peptide that makes it really difficult as they are not present free in the solution. But we could clearly prove that the Ulp-1 protease cleaved the peptide from the SUMO.
</P><CENTER><P>Figure 12:Coverage for SUMO without Peptide from Tandem MS [Red Letters] </P></CENTER><H4>Future Insights</H4><P align="justify">
Some intense literature studies gave us some insights of how we can improve the design of the peptides. Usually, the frogs and toads have the peptides with some post translational modifications (PTM) like amidation <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [5]</A>,polyglycylation <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references"> [6]</A> ,etc. These make them less vulnerable to carboxypeptidases or amidopeptidases. But, the major hurdle is that the <I> E. coli</I> cannot perform these PTMs that made us choose peptides without PTMs. This makes it clear that there are more possibilities that the peptides could be cleave of by the proteases.
</P><P align="justify">
We also found that some of the antimicrobial peptide (AMP) group called bombinins <A href="https://2013.igem.org/Team:TU-Delft/Peptides#references">[7]</A> was similar with maximins, which by itself is from the <I>Bombina</I> species. But, the bombinins have D-isoleucine instead of L-isoleucine. This makes it difficult for the peptidases to cleave the peptides. So, the maximins could also have some D-aminoacids that makes it much more difficult to perform these PTMs using <I> E. coli</I>. </P><P align="justify">
Thus, in near future these peptides can be made if we can  manage to make create a strain that can perform these PTMs so we can use them as a chassis to produce our peptides
 </P><H2 align="center">References</H2><OL><LI>Michael R. Yeaman &amp; Nannette Y. Yount,..(2003) 'Mechanisms of Antimicrobial Peptide Action   and Resistance' , Pharmacological Reviews,vol 55,p:27-55   </LI><LI>Wang, Z. and Wang, G.* (2004) APD: the Antimicrobial Peptide Database. Nucleic Acids Research 32, D590-D592</LI><LI>Panavas T, Sanders C, Butt TR.. (2009). SUMO fusion technology for enhanced protein production in prokaryotic and eukaryotic expression systems. Methods in Molecular Biology. 10 (8), p4-20.</LI><LI>Lee, C., Sun, H., Hu, S., Chiu, C., Homhuan, A., Liang, S., Leng, C. and Wang, T. 2008. An improved SUMO fusion protein system for effective production of native proteins. Protein Science, 17 (7), pp. 1241--1248.</LI><LI>Li, Recombinant production of antimicrobial peptides in Escherichia coli: A review. Protein Expression and Purification, 80, 2, 260-267, 2011.
</LI><LI>Redeker, V. 2010. Mass spectrometry analysis of C-terminal posttranslational modifications of tubulins.Methods in cell biology, 95 pp. 77--103.</LI><LI>Mangoni, M., Grovale, N., Giorgi, A., Mignogna, G., Simmaco, M. and Barra, D. 2000. Structure-function relationships in bombinins H, antimicrobial peptides from Bombina skin secretions.Peptides, 21 (11), pp. 1673--1679.</LI></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Delft/Peptides">http://2013.igem.org/Team:TU-Delft/Peptides</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Delft/Peptides" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Delft/Peptides" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Delft/Peptides&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Delft/Peptides&amp;oldid=303989" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>