Team:CLSB-UK/Project/CmpA
Project
Fuel Cell
Synechocystis
Future Research
Lab
Strategy
Experiments
Notebook
Parts
Proof of Concept
Results
Demonstrate
Human Practices
Human Practices
Engagement
Safety
Silver Medal
Judging
People
Team
Collaborations
Attributions
Sponsors
Click on different parts on the picture to find out more!
Back
CmpA Gene
Overview
CmpA encodes the substrate-binding protein of the HCO3- transporter and plays a crucial role in the carbon-concentrating machinery of Synechocystis necessary for efficient energy production via photosynthesis. CmpA has never been made into a Biobrick, and the CLSB UK team synthesized and then characterized this part in our model organism. Despite it having the highest affinity for Bicarbonate and Carbonic acid in the Synechocystis genome it is the rate limiting step of CO2 absorption in the CmpAD operon, so in theory to overexpress it would increase the rate of growth of Synechocystis, especially important as the current slow rate of growth is one of the largest issues with using Synechocystis as a model organism.
The Operon:
CmpA forms the first part of the CmpABCD operon, coding for proteins that allow dissolved Carbon Dioxide to be transported into cyanobacteria for use in photosynthesis. CmpA is the substrate binding membrane protein; CmpB is an integral channel protein; whilst CmpC and D are both ATPases that drive the active transport. CmpC also controls the rate of transport.
The operon is part of the CO2 concentrating mechanism (CCM). The CCM concentrates CO2in a cellular organelle called the carboxysome. It transports (HCO3-) into the carboxysome, where it is converted into CO2. The CO2 concentration in the carboxysome is up to 1000 times higher than it is in the cytoplasm. This is necessary because Rubisco has a high affinity for O2 as well as CO2 and so will often bind O2 rather than CO2, reducing the efficiency of photosynthesis. By over-expressing the CmpA protein, we planned to increase the rate of CO2 transport into the carboxysome.  In turn, we expected that this would increase the efficiency of carbon fixation and consequently photosynthesis.
To a certain extent, the CCM is capable of overcoming this problem. However, the CCM is adapted to conditions that existed millions of years ago when the concentration of CO2 in the atmosphere was far higher than it is today. Since then, the partial pressure of CO2 in the air has decreased and that of O2 has increased. As such, the carbon-concentrating machinery of synechocystis operates at sub-optimal efficiency since it is not fully adapted to present atmospheric conditions. This was the problem that we tried to address.
CmpA has the highest affinity to both the bicarbonate ion and carbonic acid of any protein in the cyanobacterial genome. It is used to transport bicarbonate ions (HCO3- ) into the CmpB channel.
Figure 1. Structure of CmpA-D in the membrane
Structure
CmpA is an α-ß protein similar in structure to the nitrate transporter NrtA. It consists of two domains arranged in a C-clamp shape around the ligand-binding site. Both of these domains contain a central complex of five-stranded ß-pleated sheets surrounded by α-helices. They are joined to each other by a sheaf of irregularly coiled elements. The entire protein is anchored to the plasma membrane by a lipid; CmpA is a member of the periplasmic-binding protein superfamily.
Mechanism
Bicarbonate ions bind to the ligand-binding site contained within the cleft formed by the two α-ß domains. These two domains then wrap around the ligand. This involves only minor conformational changes; the ligand-binding site is fairly inflexible. It should be noted that the binding of bicarbonate is coupled to the binding of Ca2+. The two ions bind co-operatively with high affinity. It is thought that Ca2+ may act as a cofactor or that the CmpA gene may double as a co-transporter for Ca2+ ions. Above is a ribbon diagram showing CmpA at pH 5.0 on the left and 8.0 on the right. The bicarbonate ion is shown on the left in the entrance to the cleft as a sphere. The carbonic acid is shown on the right deep inside the cleft as a sphere.  Ca2+ ion is also shown as a sphere on the right.
Genetic Approach
Given the availability of this gene's sequence , the most sensible approach was to design the primers for amplifying this specific DNA directly out of the Synechocystis PCC sp. 6803 genome. So, initially, we set about designing the pair of primers that would allow us to amplify this specific gene. Unfortunately, having looked at different options for this, we encountered a significant problem. The primers that would have given us the most useful stretch of DNA would have, most likely, formed primer dimers. Both primers also had low C-G percentages and low melting temperatures.
As a result, we have opted for having the gene synthesized as a gBlock by IDT. This approach turned out to be much better as we could synthesise the gBlock that contained the RBS and BioBrick prefix and suffix, simplifying our cloning strategy. We chose to use the RBS that was described in Wang et al. 2012. We then amplified this construct using the oligo primers that we also had synthesised by IDT.
Results
The E.coli transformed with the new part grew slower than other transformants and we were also unable to recover as much plasmid by miniprep. We suspect that this might be due to use of the strong consensus promoter J23119 , which slows down expression times of the AmilCP part in E-Coli, too. This is further confirmed by the fact that the part BBa_K207800 still gave normal E.coli colonies when amplified in pSB1C3 plasmid (this part does not have the promoter so cpm is not expressed.
We also inserted the cpm gene driven by a strong consensus promoter into a low-copy pDF Synechocystis PCC6803 plasmid in order to transform the cyanobacteria. We noticed increased growth rate in cyanobacteria in the exponential growth phase, which is in line with our expectations, and could be potentially useful to other groups working with cyanobacteria. On the other hand, we did not observe any real difference in the voltage generated when using cyanobacteria in a BPV cell .
References
Badger MR, Price GD, Long BM, Woodger FJ. 2006. The environmental plasticity and ecological genomics of the cyanobacterial CO2 concentrating mechanism. J. Exp. Bot. 57:249–265
Eisenhut, M., Ruth, W., Haimovich, M., Bauwe, H., Kaplan, A., Hagemann, M., 2008. The photorespiratory glycolate metabolism is essential for cyanobacteria and might have been conveyed endosymbiontically to plants. Proc. Natl. Acad. Sci. 105, 17199–17204.
Koropatkin NM, Koppenaal DW, Pakrasi HB, Smith TJ. 2007. The structure of a cyanobacterial bicarbonate transport protein, CmpA. J. Biol. Chem. 282:2606–2614
Maeda S, Price GD, Badger MR, Enomoto C, Omata T. 2000. Bicarbonate binding activity of the CmpA protein of the cyanobacterium Synechococcus sp. strain PCC 7942 involved in active transport of bicarbonate. J. Biol. Chem. 275:20551–20555
Price GD1, Badger MR, Woodger FJ, Long BM. 2008. Advances in understanding the cyanobacterial CO2-concentrating-mechanism (CCM): functional components, Ci transporters, diversity, genetic regulation and prospects for engineering into plants. J. Exp. Bot. 59:1441–1461
Raven, J.A., Cockell, C.S., De La Rocha, C.L. 2008. The evolution of inorganic carbon 69 concentrating mechanisms in photosynthesis. Philos. Trans. R. Soc. B: Biol. Sci. 363, 2641-2650
