"
Page
Discussion
View source
History
teams
Log in
 
Team:Tokyo-NoKoGen/bmc
From 2011.igem.org
BMC
Tokyo-NokoGen 2011
Department of Biotechnology and Life Science, Tokyo University of Agriculture and Technology
 
Home
Team
Members
Project: EcoLion
BioBricks
Notebook
Protocols
Attribution
Safety
Sponsors
BMC (Bacterial micro compartment) – localizing target proteins into BMC
 
1. Introduction
For our project EcoLion, we came up with an idea to localize and concentrate heavy metals captured by metallothioneins into a “bacterial micro compartment”. Bacterial micro compartments (BMCs) are proteinaceous internal compartments which particular bacterial naturally have and use to optimize metabolic reactions having toxic or volatile intermediates. BMCs are polyhedral structures with size of 100–150 nm in cross-section and built from several thousand polypeptides of 10–20 types. BMCs encapsulate specific metabolic enzymes within protein shells to function as natural bioreactors.
We focused on a propanediol-utilizing BMC (PduBMC) derived from Citrobacter freundii. It naturally works for utilization of 1,2-propanediol by localizing enzymes into the BMC shell. Figure 1 shows the overview of this reaction. It is known that only seven proteins, such as pduA, -B, B’, -J, -K, -N, and -U, are necessarily to form an empty micro compartment [1].
It is also supposed that tag peptides work for localization specific enzymes in to PduBMC. The N-terminus short peptide of PduP (pduP1-18) is known to function for packaging proteins into PduBMC [2]. Fan et al. have reported that GFP fused to PduP1-18 were successfully localized into recombinant PduBMC in E. coli, while GFP without the localization tag PduP1-18 are distributed in cytoplasm.
We expect those BMCs and localization tags have much potential to construct artificial bioreactors (Figure 2). By fusing specific enzymes with localization-tag peptides, it supposed to localize the enzymes into BMCs, and such artificial BMCs are expected to be used as artificial bioreactors. Our team also came up with the idea to localize and concentrate small molecules captured by target-binding peptides into BMCs in order to collect useful or harmful compounds from the environment.
For our project, we attempted to localize and concentrate heavy metals into PduBMC because heavy metals are toxic to the cell itself, and it is required to isolate them from the cytoplasm. Localization of heavy metals also makes the further process for purification of heavy metals much simpler. The overview of localization of heavy metals into PduBMC is shown in Figure 3. In order to achieve this purpose, we also attempted to express empty PduBMC shells in E. coli and characterized PduBMC shell proteins. We used metallothioneins, heavy metal-binding peptide, to capture target heavy metals, and we constructed fusion protein of a metallothioneins and a localization-tag peptide PduP1-18.
 
 
2. Constructs
 
 
3. Method
SDS-PAGE analysis and fluorescence microscopic analysis were performed to confirm the expression of PduBMC shells and PduP1-18-GFP fusion proteins supposed to be localized in the BMC.
3-1. Confirmation of the expression of pET30c-pduABJKNU by SDS-PAGE analysis
Ⅰ. Constructs for characterizing PduBMC expression (1-5)
．DpET30c-pduABJKNU (T7 promoter)
．DpSB1C3-PLlaco-1-pduABJKNU
．DpET30c
．DpSB1K3-pduABJKNU
．DpSB1A3-Ph-pduABJKNU
E. coli transformed one of the constructs shown above were grown in 100 mL of Luria-Bertani medium at 37 °C. When OD600 reached 0.8, 1 mM IPTG was added and further incubated at 16 °C for 18 hours. The cells were harvested by centrifugation at 5,000 x g at 4 °C for 10 minutes, and resuspended in 30 ml 0.85 % NaCl twice. Moreover, they were resuspended by sonication buffer (50 mM Tris-HCl, 2 mM EDTA) and lysed by sonication (5minutes bursts with 5 minutes cooling intervals on ice for three times.) Subsequently, soluble fraction and insoluble fraction were separated by centrifugation at 12,000 x g for 20 min. In order to remove membrane fraction, soluble fraction was ultracentrifuged at 26,000 x g for 30 min. Moreover, the soluble fraction was purified by ultracentrifugation at 30,000 x g for 90 min then we recovered the pellet. After ultracentrifugation, the pellet was dissolved in 100 µl sonication buffer, and the protein concentration was measured by using DC protein assay. Each protein concentration was adjusted to 0.8 mg/ml and 10µl each were applied to the well. These proteins were stained by CBB stain solution.
 
3-2. Confirmation of the expression of pET30c-pduABJKNU-Ph-pduP1-18-GFP by florescence microscopy
Ⅰ. Constructs for characterizing localization of PduP1-18 fused GFP into PduBMC (1-2)
1. pET30c-pduP1-18-GFP (T7 promoter)
2. pET30c-pduABJKNU-Ph-pduP1-18-GFP
E. coli transformants were grown in 100 ml of Luria-Bertani medium at 37 °C. When OD600 reached 0.8, protein expression was induced with 1 mM IPTG at 16 °C for 50 hours. Then we observed fluorescence from the E. coli cells using a fluorescence microscopy (Keyence).
 
 
4. Results and Discussion
4-1. Confirmation of the expression of pET30c-pduABJKNU by SDS-PAGE analysis
We carried out purification of PduBMC shell proteins and SDS-PAGE analysis to confirm expression of the proteins. We tried to purify BMC shells according to a published paper (Figure 6) [1]. We expected to observe six bands as the result of SDS-PAGE that would indicate expression of PduBMC shells. As our results of SDS-PAGE analysis, those bands were not observed at the lanes 2 and 5 where samples purified from E. coli BL21(DE3)/pET30c-pduABJKNU and E. coli BL21(DE3)/PSB1K3-Ph-pduABJKNU (Figure 7). This might be due to the difference in the amount of culture we prepared, thus amounts of BMC proteins might not be enough for purifying BMC shells and analyzing by SDS-PAGE. We cultured transfromants in 100 mL of LB broth, while the referred research article performed those experiments with 2 L of E. coli cultures.
Moreover, we noticed that there was difference between sequences of our constructed device for PduBMC (pduABB’JKNU) and the one reported in the reference [1]. The alignment between DNA sequences of pduB of our constructs and pduB derived from Citrobacter freundii which was used in the article [1] were shown in Figure 8. In pduB gene sequence, two amino acids were different from that of expected sequence. In addition, the start codon of pduK gene was replaced to valine from methionine. This could be a cause of low expression levels of pduK and low amounts of whole PduBMC shell structures. There were also other difference in the sequence of pduK as indicated in Figure 9 with red boxes. Mutation works to match the sequences of our construct to the reported genes are currently undergoing by inverse PCR and Quikchange® methods.
 
4-2. Confirmation of localization pduP1-18-GFP into the PduBMC by florescence microscopy
We carried out fluorescence microscopic analysis to confirm the co-expression of pduABJKNU and pduP1-18-GFP. The fluorescence microscopic image of E. coli cells co-expressing PduP1-18-GFP and PduBMC are shown in Figure 10. From this image PduP1-18-GFP likely localized into BMC, however this image was looked similar to a fluorescence image of E. coli BL21(DE3)/pET30c-pduP1-18-GFP which did not have PduBMC genes as a control (data not shown).
 
5. Summary
In order to localize and package target proteins into BMC, we tried to construct an empty PduBMC and localize GFP fused to a localization-tag peptide PduP1-18. As results of purification of PduBMC shells and SDS-PAGE analysis, we have not confirmed the assembly of PduBMC shells in E. coli BL21(DE3)/pET30c-pduABJKNU nor E. coli BL21(DE3)/pSB1K3-Ph-pduABJKNU, unfortunately. In addition, the localization of PduP1-18-GFP into PduBMC shells have not been confirmed yet by fluorescencce microscopic analysis of E. coli BL21(DE3)/pET30c-pduABJKNU-Ph-pduP1-18-GFP cells. These results indicate that PduBMC shellls were not assembled in the E. coli transformants. Amino acid sequences in pduB and pduK in BBa_K317029 were different from its original sequences, suggesting the differences may result in low expression of those genes and/or no successful assembly of PduBMC shells. Now we keep going on with expression of functional PduBMC shells. The funtional PduBMC would be a key component to construct compartments to localize and concentrate target molecules, such as cadmium, captured by PduP1-18 fused binding peptides, and also makes it possible to develop artificial bioreactors.
 
 
 
6. Reference
[1] Parsons et al. (2010) Synthesis of empty bacterial microcompartments, directed organelle protein incorporation, and evidence of filament-associated organelle movement. Molecular Cell, 38, 305–315.
[2] Fan et al. (2010) Short N-terminal sequences package proteins into bacterial microcompartments PNAS 107, 7509-7514.
[3] Havemann and Bobik. (2003) Protein content of polyhedral organelles involved in coenzyme B12-dependent degradation of 1,2-propanediol in Salmonella enterica serovar Typhimurium LT2. J. Bacteriol., 185, 5086–5095.
Retrieved from " http://2011.igem.org/Team:Tokyo-NoKoGen/bmc "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
