"
Page
Discussion
View source
History
teams
Log in
 
Team:USP-Brazil/Project
From 2013.igem.org
Template:Https://2013.igem.org/Team:USP-Brazil/templateUP
Overview
The device consists of a pen made of waterproof material, divided into three compartments, initially isolated one from each-other. Click on the images below to understand how it works.
The Chassis
Looking for a chassis that would resist the presence of ethanol, and specially of methanol, the simplest solution was to use yeast cells, largely employed in the production of biofuels, such as Saccharomyces cerevisae [6]. Our best choice would be a methylotrophic organism, that is, one that can use methanol as a carbon source [7]. Our chosen yeast, called Pichia pastoris, is a species of methylotrophic yeast, with its genome sequenced [11], that is commonly used in the production of recombinant proteins [8], mainly due to its growth characteristics, such as rapid growth rate and cell density, which make cell suspensions a paste-dense material [9]. It also posses methanol-responsive promoters, that could be part of a genetic circuit that would respond to the presence of methanol. In addition, P.pastoris cultures are able to grow in media with up to 10% of ethanol [10], which makes it a perfect candidate for our chassis.
Figure 1: Pichia pastoris, the methylotrophic yeast.
Molecular detection
In order to detect methanol in alcoholic drinks, we searched for P.pastoris promoters inducible by methanol, and preferably known to have well-established genetic recombination techniques.
Table 1: Best candidate promoters from Pichia pastoris to create our device.
Promoter name
Gene product
Regulation
Expression level
PAOX1
Alcohol oxidase promoter 1
Induced by methanol
Strong (naturally ∼5% of mRNA and ∼30% of total protein)
PAOX2
Alcohol oxidase promoter 2
Induced by methanol
∼5%–10% of PAOX1
PDAS
Dihydroxyacetone synthase promoter
Induced by methanol
Strong (similar to PAOX1)
PFLD1
Formaldehyde dehydrogenase promoter
Induced by methanol and methylamine
Strong (similar to PAOX1)
The three best choices (see table above) were PAOX1, PFLD1 and PDAS. Since we did not find much information on PDAS, while PAOX1 and PFLD1 are well characterized, both of them having commercial plasmids for genomic integration [13][14], we chose the latter two as our molecular detectors. That, in a simple genetic construction, could work as a proof of concept. A priori, our output system would be the monomeric RFP (Red florescent protein) ( BBa_E1010 ). The RFP will be use for fluorescence tests and for efficiently characterizing the relative promoter strength. We also expect to be able to see the production of RFP with the naked eye, once in E.coli it is shown to be possible.
Figure 2: Fluorescent proteins expressed in an E. coli suspension. Respectively, amilCP BBa_K592009 (blue), amilGFP BBa_K592010 (yellow) and RFP BBa_E1010 (red). Image from the Registry of Parts.
Our team chose to focus on the detection system via methanol-responsive promoters, and to characterize them well as BioBricks.
PAOX1
PAOX1 is a strong promoter which can be controlled by simple changes in its carbon source [16], and it is the most common choice for expression of heterologous proteins in P. pastoris, having a naturally elevated expression rate, of around 5% of the RNA and 30% of total protein production [12] [15].
The challenge in using PAOX1 is its regulation. This promoter is prone to a strong catabolic repression by hexoses and ethanol [17] — the main component of alcoholic beverages. Fortunately, ethanol is also involved in the degradation of peroxisomes, cellular compartments where P. pastoris realizes the metabolism of methanol; this aspect is  interesting for our application, since it means methanol will not be degraded as fast as it would in the absence of ethanol. Therefore, the degradation of peroxisomes would enhance the activation of PAOX1, by allowing methanol to stay for longer in the cell.
In order to develop this biosensor, it is necessary to evaluate the rates at which the promoter PAOX1 is activated via methanol or inhibited via ethanol. In addition, we decided to study a modification on Pichia pastoris Mxr1 transcription factor [18] that should alter its interaction with PAOX1 by turning ethanol into an activator of the promoter [19]. If the rate of activation by ethanol stays below the rate of activation by methanol, the latter should be identifiable when the drink is diluted. It would then, be possible to create a color guide that would help one differ pure and contaminated beverages ( For more information, see the Modeling section ).
In addition to testing the PAOX1 that is native in Pichia pastoris, we have synthesized a modified version of this promoter with up to 33% more strength than the wild type promoter [20], according to the hypothesis that a stronger promoter should allow better visualization of the colorimetric output at naked eye ( For more information, see the Parts section ).
PFLD1
This second promoter has a high transcription rate when activated, just like PAOX1, but unlike the previous one is activated by methylamine or methanol [12]. Also, it is not repressed by hexoses, like PAOX1. Since an extensive search in scientific literature did not uncover any data on the regulation of PFLD1 by ethanol, this promoter represents a great alternative to PAOX1. If it is confirmed that they do not share the same repression characteristic related to this dicarbonyl alcohol, we could eliminate an issue in our project.
Mxr1 (methanol expression regulator 1) modified
Mxr1 (methanol expression regulator 1) is a key regulator, at transcriptional level, of methanol metabolism in the methylotrophic yeast Pichia pastoris [13]. It is a transcriptional factor that binds upstream of the MUT (methanol utilizing) pathway and peroxisome biogenesis gene´s promoters using its zinc finger domain. Among the MUT genes is the AOX1 gene that is regulated by the PAOX1 promoter.
The 14-3-3 proteins are ubiquitous, and have important roles in controlling a wide variety of cellular processes, like gene expression, metabolism, cell cycle and apoptosis. These 14-3-3 proteins are involved in the carbon source-dependent regulation of Mxr1 [14], which is inactive in ethanol and glycerol, but is active in methanol.
Our submitted Mxr1 is mutated in order to NOT be repressed by ethanol, as the original Mxr1 is. This is done by substituted the Ser215 of the protein with Ala, inactivating the 14-3-3 protein interaction (phosphorylation) with Mxr1 [14]. It is also smaller than the original Mxr1 because it was already reported that the major activation domain of Mxr1 is located within the first 400 amino acids.
The modified Mxr1 is able to activate the PAOX1promoter in ethanol, methanol or in a ethanol/methanol solution [14]. This is useful for our biosensor design, which aims to detect levels of methanol above 2% in common alcoholic drinks (normally containing 10 to 60% ethanol). This will allow government to make a preliminary high-throughput screening of ethanol drinks tainted will methanol.
Preservation mechanism
In order to allow portability and storing of the device, and to create a robust, resistant and fast-responding methanol detector, some lyophilization tests were performed, aiming to produce results similar to Saccharomyces cerevisae yeast granules. That was made possible by adapting some protocols, originally for different yeast species to Pichia pastoris. Some tests were realized, essentially looking for answers to the questions below:
How many cells would have to be lyophilized?
What is the ideal dilution of the drink?
How long would it take between the contact of yeast and the drink to obtain results?
Freeze-dry
As a strategy to develop a biological system cheap enough for the specific economic issue related to the population involved on the risk of methanol poisoning, our detector should be very cheap. It also should be easy enough to prepare for storage and remain functional through long periods of time. An easy way to do so is the commonly used methodology for long storage of food, biological samples and pharmaceuticals: the lyophilization or freeze-drying process [19]. Maybe one of the most known examples of lyophilized product is the dry yeast, used for baking dough and for other food recipes.
Just to evaluate how cheap would be to produce a biological detector, we could take—as an estimative—the average cost of dry yeast on retail market, which is around 74.54 g per American dollar [20]. So, if the cost to produce and sell the net weight of our detector by lyophilization was the same of the average price in retail market, the user would spend less than five cents by net weight for each unitary detector (approximately 0.04 USD!), considering the use of 3 grams of lyophilized material per product. This is enough to justify the use of a biodetector (and, of course, the lyophilization conservation method) for the economic profile of the population involved in this problem over the world.
Figure 3: 0.04 USD.
Figure 4: Lyophilizator: essentially a vacuum machine.
The lyophilization process is quite simple. It is ased on completely removing water from a sample by sublimation of ice by vacuum [19]. In matter of microbial cultures, the freezing process must be the most instantaneous as possible in order to form little and more dispersed ice crystals, that could lead to lethal or sublethal effects on cells [19][21]. You may check our Pichia lyophilization protocol in our Notebook .
Back to the solution See the product
References
[1] AJA van Maris et al. Alcoholic  fermentation of carbon sources in biomass hydrolysates by Saccharomyces cerevisiae: current status. Antonie Van Leeuwenhoek, vol 90: 391–418 (2006).
[2] CP Hollenberg and G Gellissen. Production of  recombinant proteins by methylotrophic yeasts. Current Opinion in Biotechnology, vol. 8: 554-560 (1997).
[3] LM Damasceno, CJ Huang and CA Batt. Protein secretion in Pichia pastoris and advances in protein production. Appl Microbiol Biotechnol, vol 93:31-39 (2012).
[4] JM Cregg et al. Expression in the Yeast Pichia pastoris. Methods in Enzymology, vol. 463, ch. 13 (2009).
[5] F Ganske and UT Bornscheuer. Growth of Escherichia coli, Pichia pastoris and Bacillus cereus in the presence of the inonic liquids [BMIM][BF4] and [BMIM][PF6] and organic solvents. Biotechnology Letters, vol. 28: 465-469 (2006).
[6] K De Schutter et al. Genome sequence of the recombinant protein production host Pichia pastoris. Nature Biotechnology, vol. 27(6) (2009).
[7] T Vogl and A Glieder. Regulation of Pichia pastoris promoters and its consequences for protein production. New Biotechnology, vol. 30(4): 385-404 (2013).
[8] http://www.lifetechnologies.com/order/catalog/product/V23020?ICID==%253Dsearch-product
[9] http://www.lifetechnologies.com/order/catalog/product/V17520
[10] https://2010.igem.org/Team:Imperial_College_London/Modules/Fast_Response
[11] M Jalic et al. Process Technology for Production and Recovery of Heterologous Proteins with Pichia pastoris. Biotechnol. Prog. vol. 22: 1465-1473 (2006).
[12] M Inan and M Meagher. Non-Repressing Carbon Sources for Alcohol Oxidase Promoter of Pichia pastoris. Journal of Bioscience and Bioengineering, vol 92(6): 585-589 (2001).
[13] GP Lin-Cereghino et al. Mxr1p, a Key Regulator of the Methanol Utilization Pathway and Peroxisomal Genes in Pichia pastoris. Molecular and Cellular Biology, vol. 26(3): 833-897 (2006).
[14] PK Parua et al. Pichia pastoris 14-3-3 regulates transcriptional activity of the methanol inducible transcription factor Mxr1 by direct interaction. Molecular Microbology, vol. 85(2): 282-298 (2012).
[15] FS Hartner et al. Promoter library designed for fine-tuned gene expression in Pichia pastoris. Nucleic Acids Research, vol 36(12) (2008).
[16] S Shen et al. A strong nitrogen source-regulated promoter for controlled expression of  foreign genes in the yeast Pichia pastoris. Gene, vol. 216: 93-102 (1998).
[17] Average price calculated in sep/2013:  https://www.scienceexchange.com/services/gas-chromatography-mass-spectrometry-gc-ms?page=1
[18] http://jenslabs.com/2013/06/06/ketosense-an-arduino-based-ketosis-detector/
[19] JG Day et al. Cryopreservation and Freeze-Drying Protocols. Methods in Molecular Biology, second edition (2007).
[20] Average price calculated using 9 different product's costs on USA market (september of 2013) http://www.amazon.com/s/ref=nb_sb_noss_1/187-4864785-9882854?url=search-alias%3Dgrocery&field-keywords=dry%20yeast&sprefix=dry+ye%2Cgrocery&rh=i%3Agrocery%2Ck%3Adry%20yeast
[21] RJ Heckly and J Quay. A Brief Review of Lyophilization Damage and Repair in Bacterial Preparations. Cryobiology, vol. 18: 592-597 (1981).
[22] X Polomska et al. Freeze-Drying Preservation of Yeast Adjunct Cultures for Cheese Production. Polish Journal of Food and Nutrition Sciences, vol 62(3): 143-150 (2012).
[23] Sreekrishna, Koti, and Keith E. Kropp. "Pichia pastoris." Nonconventional Yeasts in Biotechnology. Springer Berlin Heidelberg, 1996. 203-253.
[24] F Ganske and UT Bornscheuer. Growth of Escherichia coli, Pichia pastoris and Bacillus cereus in the presence of the inonic liquids [BMIM][BF4] and [BMIM][PF6] and organic solvents. Biotechnology Letters, vol. 28: 465-469 (2006).
Template:Https://2013.igem.org/Team:USP-Brazil/templateDOWN
Retrieved from " http://2013.igem.org/Team:USP-Brazil/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
