<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SSHS-Shenzhen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SSHS-Shenzhen/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Title</TITLE></P><DIV class="header"><UL><LI><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen">Home</A></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Team" class="dropbtn">Team</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Team">Team Members</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Attributions">Attributions</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Description" class="dropbtn">Project</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Description">Abstract</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Background">Background</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Mechanism">Mechanism</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Model">Modeling</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Notebook">Notebook</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Experiments" class="dropbtn">Experiments</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Experiments">Summary</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Design">Design</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Results">Results</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/InterLab">InterLab</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Improvement">Improvement</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Human_Practices" class="dropbtn">Human Practice</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Human_Practices">Summary</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Integrated_Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Public_Engagement">Public engagement</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Meetup">Meetup</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Parts">Parts</A></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Achievements">Achievements</A></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Safety" class="dropbtn">Safety</A></DIV></LI></UL></DIV><TITLE>title</TITLE><DIV class="banner2">
Results
</DIV><H1>Objectives
</H1><P id="para">
The objective of our project is to use topical application of exogenous shRNA/siRNAs to trigger the RNAi mechanism in Phyllotreta striolata, leading to the death of the beetle.
</P><H1>Results
</H1><H2>
1. Target mRNA selection
</H2><P id="para">
Based on the Phyllotreta striolata transcriptome sequence data provided by Professor Weichang Yu, we selected our target mRNAs, which are the mRNAs of Arginine Kinase, Glutathione S-Transferase and Aldose Reductase. These three genes encode important enzymes that are involved in metabolic pathways. The mRNA sequences for Arginine kinase (Fig. 1-1), Glutathione S-Transferase(Fig. 1-2), and Aldose Reductase(Fig. 1-3) were obtained
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 1-1
</B>
 Phyllotreta striolata arginine kinase (ARK) mRNA sequence
</P></DIV><DIV class="ibox"><P id="note"><B>
Fig. 1-2
</B>
 Phyllotreta striolata glutathione S-transferase (GLS) mRNA sequence
</P></DIV><DIV class="ibox"><P id="note"><B>
Fig. 1-3
</B>
 Phyllotreta striolata aldose reductase (ALR) mRNA sequence, the start codons of the three mRNAs were highlighted.
</P></DIV></CENTER><H2>
2. siRNA and shRNA design
</H2><P id="para">
Based on the mRNA sequences, we designed 5 double strand siRNAs and 5 corresponding single strand shRNAs. Factors that affect in vitro transcription efficiency, such as the requirement of a ‘GG’ dinucleotide at the start of the transcript; and factors that affect RNAi efficiency, such as distance of target region to transcription start site, nucleotide composition, and the presence of asymmetry and energy valley within the siRNA; were considered during siRNA/shRNA designing. One of shRNAs (ALR- siRNA-1/ALR-shRNA-1) does not meet the design criteria. These siRNA/shRNA are designed as negative controls.

These criteria include: Target site criteria:
Not being in the first 75 bases from the start codon
Not being in the intron. Nucleotide content of siRNA:
GC content of ~50% GC content.
UU overhangs in 3′-end (increase siRNA stability)
Weak base pairing at 5′-end of the antisense strand (presence of A/U)
Strong base pairing at 5′-end of the sense strand (presence of G/C)
5′-end of the antisense strand start with C (Insect agoraute2 prefers 5’ C)
Based on these criteria, siRNAs that may target the Phyllotreta striolata genes were designed (Table 2-1).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Table 2.1
</B>
 siRNAs designed to target mRNAs of Phyllotreta striolata genes
</P></DIV></CENTER><P id="para">
Based on the siRNA sequences, their corresponding shRNA sequences were designed (Table 2-2).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Table 2.2
</B>
 shRNAs corresponding to the above siRNAs 
</P></DIV></CENTER><H2>
3. siRNA synthesis
</H2><P id="para">
We selected one double strand siRNA for each target mRNA (ARK, GLS, ALR) and sent out the siRNA sequences for direct synthesis. The integrity of siRNAs was identified through 15% denaturing polyacrylamide gel eletrophoresis (Fig. 3). A sharp band can be seen on the gel, and the size of the band is around 21 bases; which is the correct size. Results indicated that the siRNAs have been successfully synthesized.
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 3
</B>
 Detection of synthesized siRNA by PAGE. The integrity of siRNA was identified through 15% denaturing polyacrylamide gel eletrophoresis (200V, 30 min).
</P></DIV></CENTER><H2>
4. In vitro transcription of shRNA
</H2><H4>
4.1 DNA Oligo Template Design
</H4><P id="para">
For primer 1, convert the sense strand of the siRNA sequence to the corresponding DNA sequence, add a 17 base T7 promoter sequence (TAATACGACTCACTATA) to the 5’end of the DNA sequence, add a 8 base loop sequence to the 3’-end of the DNA sequence. For primer 2, add the antisense sequence complementary to the loop sequence to the 3’-end of the DNA sequence. add 2 AA’s to the 5’-end of the Primer 2 oligo. 5 pairs of DNA Oligo (Table 4-1) were ordered.
</P><CENTER><DIV class="ibox"><P id="note"><B>
Table 4.1
</B>
 Primers for acquisition of shRNA templates
</P></DIV></CENTER><H4>
4.2 Fill-in reaction to generate transcription templates
</H4><P id="para">
The integrity of shRNA templates was identified through 3% agarose gel eletrophoresis (Fig. 4-1). Agarose gel electrophoresis showed that the 5 shRNA templates
were successfully generated. Sharp bands can be seen on the gel, and the size of the bands is around 69 bases, which is the correct size.
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 4.1
</B>
 Detection of shRNA templates by agarose gel (3%). The integrity of shRNA templates was identified through 3% agarose gel eletrophoresis (200 V, 30 min)
</P></DIV></CENTER><H4>
4.3 In vitro transcription
</H4><P id="para">
1. In vitro transcription reaction was set up using the prepared template.
</P><P id="para">
2. Incubate the reaction mixtures for 2-3 hours at 37oC.
3. Add 1 μl RNase-Free DNase I (1 Unit/ml) to remove the DNA template, 37oC 15 min.
4. Heat the reaction mixtures for 15 minutes at 70oC to inactivate the enzyme.
5. Extract with Phenol/Chloroform.
a. Add 100 μl RNase-Free Water to dilute the reaction.
b. Add 120 μl phenol/chloroform and vortex briefly to mix.
c. Spin in a microfuge for 1 minute at full speed.
d. Carefully pipette off the top aqueous phase and transfer to a clean tube.
6. Precipitate the shRNA.
a. To the recovered aqueous phase, add 1/10 vol. of 3 M Sodium Acetate (pH 5.2).
b. Add 2.5 volumes of 95-100% ethanol.
c. Incubate for 15 minutes on ice.
d. Pellet the shRNA in a microfuge by spinning at full speed for 15 minutes. e. Remove the supernatant.
f. Carefully wash the pellet once with 70% ethanol.
g. Air dry the pellet for only 2-5 minutes.
7. Add 100 μl of the 1 X Annealing Buffer to the shRNA pellet and resuspend the shRNA. The procedure of the shRNA in vitro transcription system is illustrated in Fig. 4-2.
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 4.2
</B>
 Diagram illustrating the procedure of shRNA in vitro transcription
</P></DIV></CENTER><P id="para">
The integrity of shRNA was identified through 3% agarose gel eletrophoresis (Fig. 4-3) Agarose gel electrophoresis showed that the 5 shRNA were successfully transcribed. Sharp bands of around 52 bases in length were detected on the gel, the size of the bands is correct.
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 4.3
</B>
 Detection of in vitro transcribed shRNA by agarose gel (3%). The integrity of shRNA was identified through 3% agarose gel eletrophoresis (200 V, 30 min).
</P></DIV></CENTER><H2>
5. RNAi efficiency test
</H2><P id="para">
Adult P. striolata were obtained from Shenzhen University field station, and kept in glass bottles. The tissue culture seedlings of Chinese cabbage, Brassica chinensis leaves were placed into the above bottles (Fig.5-1).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 5.1
</B>
Adult P. striolata and Brassica chinensis leaves were placed into the glass bottles for RNAi efficiency test. Each siRNA/shRNA sample has two repeats.
</P></DIV></CENTER><P id="para">
The solutions of siRNA/shRNA (10 ng/mL) were separately sprayed onto the leaves of Chinese cabbage every third day, each solution has two repeats. Around twenty adult beetles of P. striolata were tested per siRNA/shRNA sample. The survival rates of adult beetles, were recorded at different days after siRNA/shRNA treatment (Table 5-1).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Table 5.1
</B>
 RNAi efficiencies of siRNA/shRNA
</P></DIV></CENTER><P id="para">
Results show that, except for water control, and the shRNA control sample ALR- siRNA-1, and ALR-shRNA-1, all the other samples tested could trigger RNAi mechanism, which was demonstrated by the significant survival rate decrease after treatment. Different days of shRNA treatment were displayed in X axis, the survival rates of the beetles were displayed in Y axis. It can be seen that after 11 days of treatment, there was a slight decrease of survival rate, in ALR-siRNA-1 treatment, almost no decrease in water and in ALR-shRNA-1 treatment. The survival rate decrease of other treatments are significant (Fig.5-2).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 5.2
</B>
 The survival rate of Phyllotreta striolata at different days after siRNA/ shRNA treatment.
</P></DIV></CENTER><P id="para">
When different siRNA and shRNA were displayed in the X axis, and the survival rate of the beetles after 11 days treatment were displayed in the Y axis, it can be seen that there is not an obvious pattern relevance between siRNA and its corresponding shRNA in terms of RNAi efficiency (Fig. 5-3).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 5.3
</B>
 Comparison of RNAi efficiencies between siRNA and shRNA
</P></DIV></CENTER><P id="para">
We then displayed shRNAs with different GC contents in the X axis, and the survival rate of the beetles after 11 days treatment in the Y axis, you can see that, when the GC content of the antisense strand is too low, the RNAi efficiency is greatly reduced. This could due to the fact that low GC content leads to weak binding of the siRNA with its target mRNA. Based on our data, it seems that relatively higher GC content, did not affect the RNAi efficiency much (Fig. 5-4).
</P><CENTER><DIV class="ibox"><P id="note"><B>
Fig. 5.4
</B>
 Comparison of RNAi efficiencies between different shRNAs
</P></DIV></CENTER><CENTER><DIV class="ibox"><P id="note"><B>
Table 5.2
</B>
 The effect of nucleotide content of shRNA on RNAi efficiency
</P></DIV></CENTER><H1>
Discussions
</H1><P id="para">
Our results show that all the samples tested, except ALR-siRNA-1, and AlR-shRNA- 1, could trigger RNAi mechanism, which was demonstrated by the survival rate decrease after treatment
After 11 days of treatment, there was a slight decrease of survival rate in water treatment (100% to 94%), in ALR-siRNA-1 (100% to 81%) and in ALR-shRNA-1(100% to 100%). The survival rate decrease of other treatments are significant. The differences of RNAi efficiency between siRNA and its corresponding shRNA, which have the same target site, are not significant, but the nucleotide content of siRNA/shRNA seems play a role in RNAi efficiency. When the GC content is too low, such as ALR-siRNA-1 and ALR-shRNA-1, the RNAi efficiency of this siRNA/shRNA is very low. This result may be caused by low GC content leading to unspecific and weak binding of the guide strand with the target mRNA, while high GC content, such as ALR-shRNA-2, also lead to low RNAi efficiency, because high GC content may hinder unwinding the siRNA duplex.

</P><H1>
Conclusions
</H1><P id="para">

Our project shows that the topical spray application of sequence specific siRNA/shRNA to a target insect is an effective, simple, safe and a relatively inexpensive technology for insect control. Our project also shows that the nucleotide content of shRNA seems play a role in RNAi efficiency, thus this study not only provides an environmentally friendly approach for pest control, our results are also important for the design of efficient siRNA in order to silence genes in P. striolata and provide a basis for similar studies in other organisms.
</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>