<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SSHS-Shenzhen_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SSHS-Shenzhen/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Title</TITLE></P><DIV class="header"><UL><LI><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen">Home</A></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Team" class="dropbtn">Team</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Team">Team Members</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Attributions">Attributions</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Description" class="dropbtn">Project</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Description">Abstract</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Background">Background</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Mechanism">Mechanism</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Model">Modeling</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Demonstrate">Demonstrate</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Notebook">Notebook</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Experiments" class="dropbtn">Experiments</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Experiments">Summary</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Design">Design</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Results">Results</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/InterLab">InterLab</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Improvement">Improvement</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Human_Practices" class="dropbtn">Human Practice</A><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Human_Practices">Summary</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Integrated_Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Public_Engagement">Public engagement</A><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Meetup">Meetup</A></DIV></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Parts">Parts</A></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Achievements">Achievements</A></DIV><DIV class="dropdown"><A href="https://2018.igem.org/Team:SSHS-Shenzhen/Safety" class="dropbtn">Safety</A></DIV></LI></UL></DIV><TITLE>title</TITLE><DIV class="banner2">
Design
</DIV><H1>
1. Method design
</H1><P id="para">
Literature reading revealed that RNAi mechanism has recently emerged as a potentially powerful strategy to engineer disease resistance against pathogen infections in plants. Thus; our sights locked on to this RNAi approach. But as we got our hands on, several questions appeared right away:  
First of all, we needed to figure out how to introduce the RNAi effector into the beetle? But we were not sure if we should adapt <B>transgenic approach or topical application approach.</B></P><P id="para">
The second question we need to answer is how to produce the RNAi effector. We do not know if we should use <B>E. coli expression system or in vitro transcription system.</B></P><P id="para">
Thirdly, since RNAi mechanism can be triggered by introducing either dsRNA, siRNA or shRNA, we need to decided what RNAi effector to produce? Should we produce <B>dsRNA? siRNA? or shRNA?</B></P><P id="para">
The design of our experiment method was shaped and improved by communication and collaborations with experts and other iGEM teams. With the help of professors and other iGEM team members, we learned that realize topical application is a better choice. because transgenic approach may raise concerns about environment and food safety, and topical application of dsRNA, siRNA or shRNA can also trigger the RNAi mechanism, this approach is convenient to operate and does not involve transgenic plants, thus is easier to be accepted by people. We also realized that in vitro transcription system is much less complex and easier to operate than E coli expression system. Because if we use E. coli expression system to produce the shRNA, we first need to put the shRNA template into plasmid, and transform the plasmid into E. coli. Also; after shRNA expression inside E colo, we need to break the cell walls of E. coli, then extract and purify the shRNA in order to obtian the shRNA for application. But for in vitro transcription, we just need to prepare the shRNA template, perform in vitro transcription, we get our product. After numerous discussions with professors and iGEM members of SZU, we realized that shRNA is the best choice. Because when long double strand RNA is used, the dsRNA is cleaved into multiple siRNAs by Dicer, each siRNA can be loaded into Agoraute to form an RNA-induced Silencing Complex (RISC), guiding the RISC to its target mRNA based on sequence complementary. Thus, the chance of off-target is greatly increase. On the other hand, when we use shRNA as RNAi effector, the shRNA is cleaved into a single siRNA, the siRNA can be precisely designed to target a specific gene, which is easier to avoid mistargeting. Then why did not we just produce siRNA? That is because for siRNA production, we need to design 2 template and preform 2 transcription reactions, and the two transcription products need overnight annealing to form double strand siRNA. However, for shRNA production, we only need to design one template, conduct one transcription, after transcription, the single strand shRNA will automatically form the hairpin structure. Thus, shRNA production is less complex and less time-consuming than siRNA production. 
Thus, we finally come up with our experiment design:

</P><H1>
shRNA design
</H1><H2>
shRNA design criteria
</H2><P id="para">
Target site criteria: 
· Not being in the ﬁrst 75 bases from the start codon 
· Not being in the intron.
Nucleotide content of siRNA: 
· GC content of ~50% GC content. 
· UU overhangs in 3′-end (increase siRNA stability)
· Weak base pairing at 5′-end of the antisense strand (presence of A/U) 
· Strong base pairing at 5′-end of the sense strand (presence of G/C) 
· 5′-end of the antisense strand start with C (Insect agoraute2 prefers 5’ C)

Based on the mRNA sequences of these Arginine Kinase, Glutathione S-Transferase and Aldose Reductase genes, and the shRNA design criteria, we designed our shRNAs, ARK-shRNA is designed to silence arginine kinase gene, GLS-shRNAs were designed to silence Glutathione S-Transferase, and ALR-shRNAs were designed to silence Aldose Reductase gene.

</P><DIV class="ibox"><P id="note"><B>
Table 2.2
</B>
 shRNAs corresponding to the above siRNAs 
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>