<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XJTLU-CHINA_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XJTLU-CHINA/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Notebook</TITLE></P><DIV class="header"><DIV class="container" style="margin-left: 30px; margin-right: 20px;"><P class="navilogoText">XJTLU-CHINA</P><DIV class="headerBar"><DIV class="navi"><UL><LI class="mainNavi col1"><DIV id="PrimaryNaviBg1"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA">Home</A></DIV></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg2"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Description">Project</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J1">Map description</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J2">RNAT</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J4">The second trial of RNAT testing</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J5">Chromo-protein testing</A></LI></UL><LI class="mainNavi col1"><DIV id="PrimaryNaviBg3"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts">Parts</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J1">Modularity</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J2">Part Detail</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg4"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Team">Team</A></DIV></LI><LI><A class="jump1" href="#J1">Team introduction</A></LI><LI><A class="jump2" href="#J2">Attribution</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J3">Achievements</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg5"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling">Modeling</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling#J1">Modeling</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg6"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J1">Policy</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J2">Q &amp; A</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg7"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices">Human Practice</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J1">Public speech for iGEM</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J2">iGEM teams work shop</A></LI><LI><A class="jump3" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J3">Questionnaire for XJTLU-iGEM</A></LI><LI><A class="jump4" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J4">LEGO</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg8" style="background-color: #3399CC"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook">Notebook</A></DIV></LI><LI><A class="jump1" href="#J1">Protocols</A></LI><LI><A class="jump2" href="#J2">Mar - May</A></LI><LI><A class="jump3" href="#J3">June</A></LI><LI><A class="jump4" href="#J4">July</A></LI><LI><A class="jump5" href="#J5">Aug - Sep</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg9"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations">Collaboration</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J1">with NYU-Shanghai</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#synbio">Synbio-tech</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J2">with BIT-China</A></LI></DIV></DIV></DIV></DIV><DIV class="Zheader"><DIV class="container" style="margin-left: 30px; margin-right: 20px;"><DIV class="ZheaderBar"><P class="ZnavilogoText">XJTLU-CHINA</P><DIV class="Znavi"><UL><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg1"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA">Home</A></DIV></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg2"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Description">Project</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J1">Map description</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J2">RNAT</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J4">Improve experiment of RNAT</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Description#J5">Chromo-protein testing</A></LI></UL><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg3"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts">Parts</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J1">Modularity</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J2">Part Detail</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg4"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Team">Team</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J1">Team introduction</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J2">Attribution</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J3">Achievements</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg5"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling">Modeling</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling#J1">Modeling</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg6"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J1">Policy</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J2">Q &amp; A</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg7"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices">Human Practice</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J1">Public speech for iGEM</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J2">iGEM teams work shop</A></LI><LI><A class="jump3" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J3">Questionnaire for XJTLU-iGEM</A></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J4">LEGO</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg8" style="background-color: #3399CC"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook">Notebook</A></DIV></LI><LI><A class="jump1" href="#J1">Protocols</A></LI><LI><A class="jump2" href="#J2">Mar - May</A></LI><LI><A class="jump3" href="#J3">June</A></LI><LI><A class="jump4" href="#J4">July</A></LI><LI><A class="jump5" href="#J5">Aug - Sep</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg9"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations">Collaboration</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J1">with NYU-Shanghai</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#synbio">Synbio-tech</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J2">with BIT-China</A></LI></DIV></DIV></DIV></DIV><DIV class="firstPage"><DIV class="container"><DIV class="overview"><DIV class="title"><P class="heading">Notebook</P></DIV></DIV></DIV></DIV><DIV class="extraPage" id="J1"><DIV class="extraPageBg"><DIV class="container"><P class="heading2">Protocols</P><DIV class="protocolBlock"><DIV class="line"><DIV class="left1 block"><A class="" href="https://static.igem.org/mediawiki/2015/0/00/Protocol-Detection_gel_electrophoresisXJTLU15.pdf">Protocol-Detection gel electrophoresis.pdf</A></DIV><DIV class="right1 block"><A class="" href="https://static.igem.org/mediawiki/2015/0/07/Protocol-Gel_electrophorersis_and_DNA_purificationXJTLU15.pdf">Protocol-Gel electrophorersis and DNA purification.pdf</A></DIV></DIV><DIV class="line"><DIV class="left2 block"><A class="" href="https://static.igem.org/mediawiki/2015/3/36/Protocol-Gene_synthesis_using_PCR_amplification_of_primersXJTLU15.pdf">Protocol-Gene synthesis using PCR amplification of primers.pdf</A></DIV><DIV class="right2 block"><A class="" href="https://static.igem.org/mediawiki/2015/e/e7/Protocol-Gibson_assemblyXJTLU15.pdf">Protocol-Gibson assembly.pdf</A></DIV></DIV><DIV class="line"><DIV class="left3 block"><A class="" href="https://static.igem.org/mediawiki/2015/7/75/Protocol-PCR_amplification_of_DNA_fragment_from_plasmidXJTLU15.pdf">Protocol-PCR amplification of DNA fragment from plasmid.pdf</A></DIV><DIV class="right3 block"><A class="" href="https://2015.igem.org/File:Protocol-TransformationXJTLU15.pdf">Protocol-Transformation.pdf</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="secondPage" id="J2"><DIV class="secondPageBg"><DIV class="container"><P class="heading2">Mar - May</P><DIV class="notebookDetail1"><DIV><H3>March</H3><DIV class="notebookDetail2"><P>We built up a team and tried our best to negotiate with professors to get academic supports and financial sponsor. Well, the situation was not optimistic. Most professors’ worktable had already been loaded even in summer vacation and they doubted our capability to fulfill our experiment in time. Even though we just managed to have the registration done and were very excited to become an iGEMer for the first time. Also, it was spring.</P></DIV></DIV><DIV><H3>April</H3><DIV class="notebookDetail2"><P>We deeply studied on the two proposals put forward in March. One is ‘bacteria death countdown’, which aims to give a precise limit on the generation of the propagating <I>E. coli</I>, by altering the rate of the expression of the codon modified toxin; Another is the ‘nisin producting yeast’ which could be used in fermentation. However, we gived up them as there were problems in generation counting and expression system in yeast used to making bread. Then, our instructor suggested us to look over more projects in privious years and get inspired. We did that in our spare time, exchanged information, and brainstormed new ideas together.</P></DIV></DIV><DIV><H3>May</H3><DIV class="notebookDetail2"><P>After several rounds of discussion, we decided our topic shall be bonding circular mRNA with ribothermometer. The former is based on the mechanism of T4 intron system, and the later takes the advantage of the temperature dependent-second structure of the RNA. Several experiment protocols were settled with the preliminary experimental design. According to the preliminary experimental planning, the gene sequence was compiled.</P><P>Later in May, we also held an iGEM public speech in our university, both professors and students were involved, to get a better understanding of synthetic biology and iGEM.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="thirdPage" id="J3"><DIV class="thirdPageBg"><DIV class="container"><P class="heading2">June</P><DIV class="notebookDetail1"><DIV><H3>June 22 - June 28</H3><DIV class="notebookDetail2"><P>This was the start of our summer vacation and we started our work in lab, although this week was just for practicing experiment skills. A tough lesson learned----Agar gels in good quality make a nice day…… Besides, we got familiar with gene synthesis, PCR, electrophoresis, plasmid digestion, transformation, colony selection and so on.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4 notebookDetail4-1"><H4>June 22</H4><OL><LI>PCR:
                                    1) Assembly PCR
                                    2) Amplify PCR
                                    3) Recycle the gel
                                </LI><LI>2, Ligation Ribo-1, Ribo-2, Ribo-3 with Plasmid PUC57 and transformation into TOP 10 F’</LI></OL></DIV><DIV class="notebookDetail4 notebookDetail4-2"><H4>June 23</H4><P>Check and obtain the correct plasmid from the desired transformed cells that cultured last night by conducting the PCR</P></DIV><DIV class="notebookDetail4 notebookDetail4-1"><H4>June 24</H4><P>Waiting for the results of sequencing</P></DIV><DIV class="notebookDetail4 notebookDetail4-2"><H4>June 25</H4><OL><LI>PCR the Ribo-1, Ribo-2, Ribo-3, recycles the gel.</LI><LI>Gibson assembly the PCR products of Ribo-1, Ribo-2, Ribo-3 and pET28A</LI><LI>Transformation into TOP 10 F’</LI></OL></DIV><DIV class="notebookDetail4 notebookDetail4-1"><H4>June 26</H4><OL><LI>Check and obtain the correct plasmid from the desired transformed cells that cultured last night by conducting the PCR</LI><LI>PCR ligation of Ribo-1, Ribo-2, Ribo-3</LI><LI>Gibson assembly the PCR products of PCR product from step 2 and plasmid pET28A</LI><LI>Transformation into TOP 10 F’ and DH10B</LI></OL></DIV><DIV class="notebookDetail4 notebookDetail4-2"><H4>June 27</H4><P>Check and obtain the correct plasmid from the desired transformed cells that cultured last night by conducting the PCR</P></DIV><DIV class="notebookDetail4 notebookDetail4-1"><H4>June 28</H4><P>Waiting for the results of sequencing</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="fourthPage"><DIV class="fourthPageBg" id="J4"><DIV class="container"><P class="heading2">July</P><DIV class="notebookDetail1"><DIV><H3>July 1 - July 5</H3><DIV class="notebookDetail2"><P>The construction of plasmid ws finished in <I>E. coli</I> DH10B, but we could not continue our experiment due to the lack of luciferin and RNA extract box.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>July 1</H4><P>Isolate the plasmid X1026 from the DH10B which carries the plasmid and has been shaken over night.</P></DIV><DIV class="notebookDetail4"><H4>July 2</H4><P>Conduct the experiment of the ribothermometers. We set the experiment as follows:</P><TABLE border="1" style="border-collapse: collapse; margin-left: 20px"><TBODY><TR><TD>No.</TD><TD>Incubate</TD><TD>Induce</TD><TD>IPTG</TD></TR><TR><TD>1</TD><TD>37℃</TD><TD>37℃</TD><TD>+</TD></TR><TR><TD>2</TD><TD>37℃</TD><TD>37℃</TD><TD>-</TD></TR><TR><TD>3</TD><TD>30℃</TD><TD>30℃</TD><TD>+</TD></TR><TR><TD>4</TD><TD>30℃</TD><TD>30℃</TD><TD>-</TD></TR><TR><TD>5</TD><TD>30℃</TD><TD>37℃</TD><TD>+</TD></TR><TR><TD>6</TD><TD>30℃</TD><TD>37℃</TD><TD>-</TD></TR></TBODY></TABLE><P>We stored the products at 4℃.</P></DIV><DIV class="notebookDetail4"><H4>July 3</H4><P>Break the cell from the product of yesterday and isolate the protein. Run SDS-PAGE. </P><DIV class="notebookDetail5"><P>Result of SDS-PAGE</P></DIV></DIV><DIV class="notebookDetail4"><H4>July 4</H4><P>We conducted the PCR of MBP-EGFP and recycled the agrose gel. Cut the pET-28A vector with PstI and NcoI. Insert the MBP-EGFP in to the vector through Gibson assembly.</P></DIV><DIV class="notebookDetail4"><H4>July 5</H4><P>Group meeting, members updated each other. We decided to switch our direction of our project.</P></DIV></DIV></DIV><DIV><H3>July 6 - July 12</H3><DIV class="notebookDetail2"><P>While Melody, Sue and Jodie kept working in lab, other members were searching what exactly pathway/proteins our circular mRNA and ribothermometer gonna work with. When James, Polly and Mag were trying to figure out every detail in synthesizing spider silks by circular mRNA, Gandi jumped out the idea of simulate the process of global warming by bacteria. It indeedly attracted William, so William quickly finished the first edition of the circuit design.</P><P>The ribothermometer worked not very well.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>July 6</H4><P>We selected the colonies with toothpicks and amplified them with primers. Run agrose electrophoresis. However, no bands showed up. We still selected 6 clones and sent them to sanger sequencing. Jodie designed the primers of four control groups. </P></DIV><DIV class="notebookDetail4"><H4>July 7</H4><P>Group meeting with our supervisor Yunpeng Zhong. We decided our final project: simulating the process of global warming with bacteria. Also, we make a plan of the following experiments.</P></DIV><DIV class="notebookDetail4"><H4>July 8</H4><P>We synthesized the vector of circular mRNA but without ribothermometer. And we transformed the vector into <I> E. coli</I> DH10B.</P></DIV><DIV class="notebookDetail4"><H4>July 9</H4><P>We conducted a RT-PCR to test if the mRNA had formed a circle.</P></DIV><DIV class="notebookDetail4"><H4>July 10</H4><P>We amplified the cDNA product of yesterday to see if the length is right. Unfortunately, the length was not right. </P></DIV><DIV class="notebookDetail4"><H4>July 11</H4><P>We cannot believe the result of yesterday. So we repeat the experiments of yesterday and the day before yesterday. This is the last shot of circular mRNA, if it fails again; we shall just give it up, literately! And, yep, it failed again….. </P></DIV><DIV class="notebookDetail4"><H4>July 12</H4><P>Group members met. We decided to focus on the ribothermometer and Gandi’s idea. </P></DIV></DIV></DIV><DIV><H3>July 13 - July 19</H3><DIV class="notebookDetail2"><P>After negotiating with our instructor, we had a vote and we finally set our topic----simulate the process of global warming. We began to check the genetic circuit designed, meanwhile, as the ribothermometer U6 chosen from paper showed a severe leak at 30℃, we decided to try more.</P></DIV></DIV><DIV><H3>July 20 - July 26</H3><DIV class="notebookDetail2"><P>This week’s experimental task was making new sets of ribothermometers. Sue, Melody, Victor, Polly and Jodie were responsible for that. Others rechecked the parts like promoters, operators, repressors used in the circuit.</P><P>Ziang Shi showed us the website designed. It was a surprise! We all liked it.</P><P>Another day, an experienced leader in Sichuan University’s iGEM team joined us.</P><P>Da Xu reminded us of the modeling in our project. James began to work on that.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>June 22</H4><P>Design the vector of the experiments of ribothermometer.</P></DIV><DIV class="notebookDetail4"><H4>June 23</H4><P>Design the primers of the fragments and synthesis the primers.</P></DIV><DIV class="notebookDetail4"><H4>June 23</H4><P>Design the primers of the fragments and synthesis the primers.</P><DIV class="notebookDetail5"><P>RibothermometersE</P></DIV><DIV class="notebookDetail5"><P>Plasmid </P></DIV></DIV><DIV class="notebookDetail4"><H4>June 25</H4><P>We selected the colonies of yesterday. U0, U6, and U6-GC didn’t show right bands on agrose gel. We sent the clone with right bands to sequencing and repeat the transformation of U0, U6 and U6-GC.</P></DIV><DIV class="notebookDetail4"><H4>June 26</H4><P>Colony test of U0, U6 and U6-GC was conducted. Each one contains at least one right band on the agrose gel. We sent them to sequencing.</P></DIV></DIV></DIV><DIV><H3>July 27 - August 2</H3><DIV class="notebookDetail2"><P>Sue, Polly, Melody, Victor and Jodie continued constructing ribothermometers. It took longer time than expected. It just seemed sometimes the bacteria BL21 were not very active. We were upset about the delay. Anyway, at this time, Jodie introduced us a good way to record the experiment every day. Polly tried to grow the <I>E. coli</I> on a membrane.</P><P>There were several bugs in our circuit design. We changed the quorum sensing system and promoters to satisfying different requirements.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>June 27</H4><P>The result of sequencing shows that all of our sequence, except U10 and K115107, was wrong. We repeat the experiment starts from the PCR (July 24th). </P><DIV class="notebookDetail5"><P>pET-28a XbaI&amp;XhoI</P></DIV></DIV><DIV class="notebookDetail4"><H4>June 28</H4><P>We isolated the plasmid of U10 and K115017 from Top 10F’ and named them as T1017 and T1018. Plasmid T1017 and T1018 were transformed into BL21. </P></DIV><DIV class="notebookDetail4"><H4>June 29</H4><P>We repeated what we have done in July 24th, transformed A1, A2, A3, U0, U6, U6-GC, U9, K115002 into the Top0F’ and DH10B.</P></DIV><DIV class="notebookDetail4"><H4>June 30</H4><P>Colony test, U6, U6-GC and K115002 still has no right bands. We sent the rest to sequencing and repeat the transformation of U6, U6-GC and K115002.</P><DIV class="notebookDetail5"><P>A1, A2, A3, U0                      </P></DIV><DIV class="notebookDetail5"><P>U6-GC, U6, U9, K115002</P></DIV></DIV><DIV class="notebookDetail4"><H4>June 31</H4><DIV class="notebookDetail5"><P>Enzyme digestion of U0</P></DIV></DIV><DIV class="notebookDetail4"><H4>August 1</H4><P>Colony test of U0 and U633A</P></DIV><DIV class="notebookDetail4"><H4>August 2</H4><P>We transformed plasmid X022t (A1), X024t (A3), X030t (U6-GC), X032 (U9), T1012 (U10) and X033 (K115002) into BL21.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="fifthPage" id="J5"><DIV class="fifthPageBg"><DIV class="container"><P class="heading2">Aug - Sep</P><DIV class="notebookDetail1"><DIV><H3>August 3 - August 9</H3><DIV class="notebookDetail2"><P>Sue, Polly and Mag tested the ribothermometers in different ways, both qualitatively and quantitatively. However, they still didn’t work well. Vincent processed the data and there was only a 1.2 on-off ratio showed by ribosthermometeres. James thought it was because the lac operator stood in front of the ribothermometers disturbed its secondary structure. James and Neil wrote another set of sequence to fix that problem.</P><P>James finished writing the fifth edition of the sequence in snapgene and sent it to our instructor to be analyzed. But the modeling part had no progress.</P><P>Shanghai meet up.</P></DIV></DIV><DIV><H3>August 10 - August 16</H3><DIV class="notebookDetail2"><P>Zhang Haoqian advised us to change the lux promoter and J23119. Plus his suggestion, we just realized there are still too many parts needed to be tested separately. So instructor taught us to modularize our circuit. We separated every construction into two parts. One involves promoter and RBS, the other one belongs to protein and terminator. In such case, we were able to examine each part and later quickly install proper parts together to construct our final circuits. We together finish coding the sequence.</P><P>A bad news came out that the yellow chromo protein turned out to be orange when expressed. That led our land turn to purple instead of green! It was beautiful but……unacceptable. We needed to change it.</P></DIV></DIV><DIV><H3>August 17 - August 23</H3><DIV class="notebookDetail2"><P>According to the separation of parts, we named all the parts and double-cheaked the sequence:</P><P>-PR-1~PR-11: Different promoters and RBS;</P><P>-CT-1~CT-9: Different proteins and terminator.</P><P>James and Neil respectively designed the new thermometer sequence:</P><P>-TW-1~TW-9: Different promoters and different thermometers designed by James;</P><P>-TJ-1~TJ-9: Different promoters and different thermometers designed by Neil.</P><P>Vincent, William and Neil started to construct and transform the parts.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>August 17</H4><P>Conducted PCR and gel recycle of PR-2~PR-9. PR-2/8/9 had wrong length. Cut the vector pSB1C3 with Xbal1 and Spe1.</P><DIV class="notebookDetail5"><P>PR-2~PR-9</P></DIV><DIV class="notebookDetail5" style=""><P>pSB1C3 cut by Xbal1&amp;Spe1</P></DIV></DIV><DIV class="notebookDetail4"><H4>August 18</H4><P>Conducted PCR and gel recycle of CT-2~CT-9, terminator. CT-3/4/5/7 and terminator have wrong length.</P><DIV class="notebookDetail5"><P>CT-2~CT-9, terminator                   </P></DIV><DIV class="notebookDetail5" style=""><P>Recycle of CT-2/6/8/9</P></DIV></DIV><DIV class="notebookDetail4"><H4>August 19</H4><P>Conducted PCR and gel recycle of TW-3~TW-7, CT-3~CT-7 and terminator. Due to the failed-amplification of terminator again, primer was designed renewedly.</P><DIV class="notebookDetail5"><P>TW-3~TW-7</P></DIV><DIV class="notebookDetail5" style=""><P>CT-3~CT-7</P></DIV></DIV><DIV class="notebookDetail4"><H4>August 20</H4><P>We conducted the PCR of pr-1 pr-2, pr-8, ct-4 and Terminator. Pr-2, pr-8 and ct-4 showed the clear and right band on the agarose gel, while pr-1, pr-9 and Terminator didn’t show any band. Then we recycled the gel of pr-2, pr-8 and ct-4. The vector of pSB1C3 was cut by speI. We introduced pr-2, pr-8 and ct-4 into the Top 10 F’ and DH10B. </P><DIV class="notebookDetail5"><P>Pr-1, pr-8, pr-9                  </P></DIV><DIV class="notebookDetail5" style=""><P>pr-2, ct-4                   </P></DIV><DIV class="notebookDetail5"><P>Ter</P></DIV><DIV class="notebookDetail5" style=""><P>pSB1C3 cut by SpeI </P></DIV></DIV><DIV class="notebookDetail4"><H4>August 21</H4><P>We selected the clone that we introduced into the cell, only ct-4 showed the band. The clone of ct-4 was sent to sequencing to see if the sequence is right. PCR of pr-1, pr-9 and Terminator was conducted again as well as tj-9 and this time they all showed the right band on the agarose gel. The pr-1, pr-2, pr-8, pr-9 and Terminator was recycled and introduced into Top 10 F’ and DH10B.</P><DIV class="notebookDetail5"><P>Pr-1, pr-9</P></DIV><DIV class="notebookDetail5" style=""><P>Terminator                  </P></DIV><DIV class="notebookDetail5"><P>tj-9</P></DIV></DIV><DIV class="notebookDetail4"><H4>August 22</H4><P>We repeat what we did yesterday because nothing showed when we select the clones, nothing showed after the PCR, even the vector was not cut correctly. Leave me alone……. </P></DIV><DIV class="notebookDetail4"><H4>August 23</H4><P>Except pr-1, other fragments sowed at least one right clone. We sent them all to sequencing. Also, the sequencing result of Aug 21st came out, nothing was right. Yep, nothing………. Damn </P></DIV></DIV></DIV><DIV><H3>August 24 - August 30</H3><DIV class="notebookDetail2"><P>James found two kinds of new chromoprotein that would be more suitable for our project and named as CT-10~CT-13. Jodie added label for degradation to the sequences. The further experiment of Neil’s thermometers was built and named as NT-1~NT-8, which were connected by TJ-1~TJ-8 and CT-3 (The design of TJ-9 was deleted after several attempts of amplification. What a pity…). At the same time, James’ thermometers were also designed and named as W-1~W-9, which were connected by TW-1~TW-9 and CT-3. Vincent, William, Jodie and Neil still devoted to constructing and transforming the parts.</P></DIV><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>August 24</H4><P>We connected ct-1, 2, 3, 4 and 7 with the Terminator and conducted the PCR of our chromoprotein. Fortunately, all of them showed the right band on agarosse gel. We introduced them into the cells.</P><DIV class="notebookDetail5"><P>Ct-1, 2, 3, 4, 7                   </P></DIV><DIV class="notebookDetail5"><P>chromoprotein</P></DIV></DIV><DIV class="notebookDetail4"><H4>August 25</H4><P>PCR of tw-1 to9 was conducted as well as ct-4, 5, 6and 9. Well…..the operators forgot to name the pictures. I don’t know what these are. All I know is that something came out but something failed. They do label the tubes; they just forgot to name the pictures, so we still have our products. </P></DIV><DIV class="notebookDetail4"><H4>August 26</H4><P>Tw-2 to 9 showed correct clones and were all sent to sequencing.  The clone of blue chromoprotein was also right. The sequencing result of 23rd came out. So far, we have successfully synthesis the pr-1 to 9, ct-1 to 3, 7, 8, 10 to 13 except, tj-1 to 8, still missing pr-10, 11 and tw- 1. Ct-4, 5, 6 and 9, tw-2 to 9 on sequencing.</P></DIV><DIV class="notebookDetail4"><H4>August 27</H4><P>We conducted the PCR of pr-10, 11 and tw-1. And connected the ct-1, 2, 3, 7, 8, 10, 13 with terminator and connected blue chromoprotein with LVA, AAV tail and terminator as well as the J23100 promoter. As always, we introduced them in to the cells, but this time we choose Epi400 and STBL3 to be our competent cell.</P><DIV class="notebookDetail5"><P>Connection product                </P></DIV><DIV class="notebookDetail5"><P>pr-10, 11and tw-1 </P></DIV></DIV><DIV class="notebookDetail4"><H4>August 28</H4><P>Except tw-1, all fregments showed the right clone. We sent these clones to sequencing. The sequencing result of 26th came out, ct- 9 and tw-2 to 9 was right. We introduced ct-4, 6 and tw-1 into Epi400 and STBL3 again. Also, we started to synthesis the two pathways of our “Marine” <I>E. coli</I>.</P></DIV><DIV class="notebookDetail4"><H4>August 29</H4><P>The clones of tw-1 and ct-6 were still not right. We conducted the PCR of them again and introduced them into Top 10 F’ and STBL3.</P><DIV class="notebookDetail5"><P>Clones of tw-1, ct-4 and ct-6 </P></DIV><DIV class="notebookDetail5"><P>PCR product of tw-1 and ct-6  </P></DIV></DIV><DIV class="notebookDetail4"><H4>August 30</H4><P>The sequencing result of 28th showed that tw-2 to 9 was all right, so we connected them with protein and upgraded them into w-2 to 9 and introduced them into Top 10 F’ and DH10B Still, no right clones of tw-1 showed up. We are just about to give him up.</P><DIV class="notebookDetail5"><P>Clones of ct-6 and tw-1          </P></DIV></DIV></DIV></DIV><DIV><H3>August 31 - September 6</H3><DIV class="notebookDetail3"><DIV class="notebookDetail4"><H4>August 31</H4><P>The sequencing result of W-1~W-9 each had a wrong repeat of terminator. Jodie was sad and decided to take a trip to Nanjing, Jiangsu Province for calming down…</P></DIV><DIV class="notebookDetail4"><H4>September 1</H4><P>The sequencing result of TW-9, T7 Blue aav and Marine-2 were correct. Cheers!</P><P>…But wait… Enzyme Spe1 was used up. Our hero advisor, Yacheng Guan, borrowed one tube from another company. We regarded it as treasure…</P></DIV><DIV class="notebookDetail4"><H4>September 2</H4><P>We got the correct sequencing result of W-5, W-8 and re-amplify W-2/3/4/6/7, TW-1 and PR-11. Also we cut the plasmid to be vector. The ‘treasure’ was nearly used up again…</P></DIV><DIV class="notebookDetail4"><H4>September 3</H4><P>Correct sequencing! W-4/5/9, PR-10.</P></DIV><DIV class="notebookDetail4"><H4>September 7 - September 13</H4><P>Vincent devoted to the test of the three different ribothermometers (A1, 4U and U6) with three different promoters (J23119, pBad and T7). The bacteria were incubated and induced respectively at 30℃ and 37℃. After that, the bacteria were disrupted and plate reader was used to measure the fluorescence of report protein: eGFP.( For more details, please refer to Project: Improve experiment of RNA thermometer.)
Judie conducted the test of the four chromoproteins: AeBlue chromoprotein (BBa_K864401), AeBlue chromoprotein with AAV tag, AeBlue chromoprotein with LVA tag, FwYellow chromoprotein (BBa_K1033910), amilGFP (BBa_K592010, yellow) and amajLime (BBa_K1033916, green-yellow). Also blue and yellow chromoprotein were used to yield green colour. (For more details, please refer to Project: Chromoprotein testing.)
</P></DIV><DIV class="notebookDetail4"><H4>September 14 - September 18</H4><P>Zi’ang Shi devoted to the design of wiki.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>