<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XJTLU-CHINA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XJTLU-CHINA/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team</TITLE></P><DIV class="header"><DIV class="container" style="margin-left: 30px; margin-right: 20px;"><P class="navilogoText">XJTLU-CHINA</P><DIV class="headerBar"><DIV class="navi"><UL><LI class="mainNavi col1"><DIV id="PrimaryNaviBg1"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA">Home</A></DIV></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg2" style="background-color: #3399CC"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Description">Project</A></DIV></LI><LI><A class="jump1" href="#J1">Map description</A></LI><LI><A class="jump2" href="#J2">RNAT</A></LI><LI><A class="jump2" href="#J4">The second trial of RNAT testing</A></LI><LI><A class="jump2" href="#J5">Chromo-protein testing</A></LI></UL><LI class="mainNavi col1"><DIV id="PrimaryNaviBg3"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts">Parts</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J1">Modularity</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J2">Part Detail</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg4"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Team">Team</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J1">Team introduction</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J2">Attribution</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J3">Achievements</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg5"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling">Modeling</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling#J1">Modeling</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg6"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J1">Policy</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J2">Q &amp; A</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg7"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices">Human Practice</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J1">Public speech for iGEM</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J2">iGEM teams work shop</A></LI><LI><A class="jump3" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J3">Questionnaire for XJTLU-iGEM</A></LI><LI><A class="jump4" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J4">LEGO</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg8"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook">Notebook</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J1">Protocols</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J2">Mar - May</A></LI><LI><A class="jump3" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J3">June</A></LI><LI><A class="jump4" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J4">July</A></LI><LI><A class="jump5" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J5">Aug - Sep</A></LI><LI class="mainNavi col1"><DIV id="PrimaryNaviBg9"><A class="primaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations">Collaboration</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J1">with NYU-Shanghai</A></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#synbio">with Synbio-tech</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J2">with BIT-China</A></LI></DIV></DIV></DIV></DIV><DIV class="Zheader"><DIV class="container" style="margin-left: 30px; margin-right: 20px;"><DIV class="ZheaderBar"><P class="ZnavilogoText">XJTLU-CHINA</P><DIV class="Znavi"><UL><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg1"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA">Home</A></DIV></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg2" style="background-color: #3399CC"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Description">Project</A></DIV></LI><LI><A class="jump1" href="#J1">Map description</A></LI><LI><A class="jump2" href="#J2">RNAT</A></LI><LI><A class="jump2" href="#J4">The second trial of RNAT testing</A></LI><LI><A class="jump2" href="#J5">Chromo-protein testing</A></LI></UL><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg3"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts">Parts</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J1">Modularity</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Parts#J2">Part Detail</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg4"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Team">Team</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J1">Team introduction</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J2">Attribution</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Team#J2">Achievements</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg5"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling">Modeling</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Modeling#J1">Modeling</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg6"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J1">Policy</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Safety#J2">Q &amp; A</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg7"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices">Human Practice</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J1">Public speech for iGEM</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J2">iGEM teams work shop</A></LI><LI><A class="jump3" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J3">Questionnaire for XJTLU-iGEM</A></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Practices#J4">LEGO</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg8"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook">Notebook</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J1">Protocols</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J2">Mar - May</A></LI><LI><A class="jump3" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J3">June</A></LI><LI><A class="jump4" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J4">July</A></LI><LI><A class="jump5" href="https://2015.igem.org/Team:XJTLU-CHINA/Notebook#J5">Aug - Sep</A></LI><LI class="ZmainNavi col1"><DIV id="ZPrimaryNaviBg9"><A class="ZprimaryNavi" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations">Collaboration</A></DIV></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J1">with NYU-Shanghai</A></LI><LI><A class="jump1" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#synbio">with Synbio-tech</A></LI><LI><A class="jump2" href="https://2015.igem.org/Team:XJTLU-CHINA/Collaborations#J2">with BIT-China</A></LI></DIV></DIV></DIV></DIV><DIV class="firstPage"><DIV class="container"><DIV class="overview"><DIV class="title"><P class="heading">Project</P></DIV></DIV></DIV></DIV><DIV class="secondPage" id="J1"><DIV class="secondPageBg"><DIV class="container"><DIV class="lv1"><H2>1. Map description</H2></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>1.1 Overview</H3><DIV class="lv2"><P>This year, team of XJTLU aims to simulate the process of global warming by introducing ribothermometers and chromoproteins to Escherichia coli. Global warming causes many consequences, including glacial ablation, the rise of sea level and the withering of plants. The appearance of the Earth will change as the average temperature rises: the white glaciers will melt; the green threes on the land turns into yellow and the coastal area will be flooded with sea water. To simulate this process, a genetic circuit made up with a ribothermometer that will switch at 37 degree Celsius and different color of chromoproteins was designed. Four different strains sand for four differen areas on earth are made: “Inland”, “Coastal”, “Marine” and “Polar”, as shown in figure 1.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 1</P></DIV><DIV class="container"><DIV class="lv1"><H3>1.2 Inland </H3><DIV class="lv2"><P>Figure 2 shows the pathway of “Inland” <I>E. coli</I>. Inspired by Paris-Saclay igem team (2014), we combined a yellow chromoprotein called amilGFP (Part ID: BBa_K592010) and a blue chromoprotein called AeBlue (Part ID: BBa_K864401) to get green color in order to simulate the green color of inland area on the earth. Both chromoprotein followed with aav tail to obtain the color change distinctly from green to yellow. We introduced a tet operator to operate the expression of blue chromoprotein. The express of tetR (Part ID: BBa_C0040) is controlled by the 37℃ ribothermometer on the up steam. When the temperature is below 37℃, the tetR will not be translated and both yellow and blue chromoprotein can express, therefore, the green color shows. However, when the temperature reach above 37℃, tetR starts to be translated and bind with tetO, the gene that codes blue chromoprotein cannot be transcribed. Only the yellow chromoprotein can express. As the consequence of that, the color of the colony changed from green into yellow.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P>Figure 2</P></DIV><DIV class="container"><DIV class="lv1"><H3>1.3 Coastal and Marine</H3><DIV class="lv2"><P>The repress and control system between “Coastal” and “Marine” <I>E. coli</I> is showed in figure 3. When the temperature is below 37℃, LuxI (Part ID: BBa_C0061) in “Marine” <I>E. coli</I> will not be translated, 3-oxo-hexanoyl-HSL will not express. The Lux promoter, Plux (Part ID: BBa_R0061), in “Coastal” <I>E. coli</I> is closed in absence of LuxR/HSL (LuxR Part ID: BBa_C0062). Therefore, tetR won’t express and both yellow and blue chromoprotein will be expressed, the color of “Coastal” <I>E. coli</I> is green. When the temperature rise above 37℃, the ribothermometer switches on and 3-oxo-hexanoyl-HSL starts to express, which starts the transcription of Plux and tetR. TetR binds with tet operator and stops the transcription of yellow chromoprotein. The color of “Coastal” <I>E. coli</I> turns into blue.  This pathway simulates the process of the rise of sea level and the coastal area flooded by sea water.  </P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 3</P></DIV><DIV class="container"><DIV class="lv1"><H3>1.4 Marine and Coastal </H3><DIV class="lv2"><P>Just like the correlation between “Marine” and “Coastal”, the color change of “Polar” <I>E. coli</I> is controlled by LuxR, Plux and LuxI system (Figure 4). The color of “Polar” <I>E. coli</I> is white (the origin color of <I>E. coli</I> colony) when the temperature is below 37℃ and will change into blue when the temperature is above 37℃ as the presence of 3-oxo-hexanoyl-HSL. This pathway simulates the process of the glaciers melting.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 4</P></DIV></DIV></DIV><DIV class="thirdPage" id="J3"><DIV class="thirdPageBg"><DIV class="container"><DIV class="lv1"><H2>2. Chromoprotein </H2></DIV></DIV></DIV></DIV><DIV class="fourthPage" id="J2"><DIV class="fourthPageBg"><DIV class="container"><DIV class="lv1"><H2>2. RNA thermometer </H2></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>2.1 Background </H3><DIV class="lv2"><H4>Introduction</H4><DIV class="lv3"><P>RNA thermometers (RNATs) are temperature-induced riboswitches that post-transcriptionally regulate gene expression in response to temperature shifts by a way of undergoing conformation changes in the secondary structure of RNA to exhibit either turn-on state or turn-off state.</P></DIV></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 1: Responsiveness of mRNA structures to environmental cues (TUDelft, 2008)</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><DIV class="lv3"><P>As shown in Figure 1, the hairpin structure harbors the Shine-Dalgarno sequence (SD sequence) and, in this way, make it inaccessible to the 30S unit of the bacterial ribosome, resulting in inactivationof translation . (SD sequence is polypurine sequence located in around 8 nt upstream from the start codon and responsible for anchoring ribosome in the RNA single strand). Once reaching a certain temperature, hairpin structure would vanish and as a result, exposing the SD sequence to trigger the translation process.</P><P>Natural forms of RNATs were usually found in 5’ untranslated region of many eubacteria and were believed to be related to biological temperature-rising response (Nocker, 2001, cited in Neupert and Bock, 2009). Here, we would introduce two sub-families of RNATs and synthetic RNA thermometers and how they works in our project. </P></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><H4>ROSE </H4><DIV class="lv3"><P>A conserved family of RNATs is the ROSE-like elements (ROSE stands for Repression Of heat Shock gene Expression), first discovered in the 5′ untranslated region of rhizobial heat shock genes (Nocker, 2001). All the RNAts presenting in ROSE family we known control expression of small heat shock genes (Nocker, 2001). However, most naturally occurring ROSE elements are relatively large and fold into complex secondary structures that usually contain 2-4 hairpin structures (Fig. 2). Hence, for practical reason, we tested A1 RNAT that derived from ROSE family. Thank BIT-CHINA for provding DNA sequences of this part. The possible secondary structure of A1 was simulated by RNAstructure (Fig.3).</P></DIV></DIV></DIV></DIV><DIV class="hPImgStyle2Text1"><P style="color: #FFFFFF">Figure 2: Proposed RNA secondary structure of the 5’ UTR from the prfA gene of the pathogenic bacterium Listeria monocytogenes, which belongs to ROSE family (Neupert and Bock, 2009). </P></DIV><DIV class="hPImgStyle2Text2"><P style="color: #FFFFFF">Figure 3: The possible secondary structure of A1.</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><H4>FourU</H4><DIV class="lv3"><P>FourU thermometers as naturally occurring RNA thermometers, for the first time, were found in Salmonella, so called as four highly conserved uridine nucleotides are placed right opposite to the SD sequence. FourU elements usually have two hairpin structures, in the case of temperature rising, the one without SD sequence is relatively steady while the other contains hidden SD sequence would be melted. In 2008, TUDelft designed and submitted part K115002 to iGEM registry based on natural forms of FourU elements. The possible secondary structure of FourU was simulated by RNAstructure (Fig.4). We test this part this year to determine the validity of this part.</P></DIV></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 4: The possible secondary structure of FourU.</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><H4>Synthetic RNA thermometer</H4><DIV class="lv3"><P>The possibility of designing synthetic RNA thermometers was explored based on the assumption that the RNATs do function by the proposed simple hairpin-melting mechanism that expose the ribosome-binding site. Therefore, Stem-loop structure with the SD sequence embedded in the stem would be the simplest RNAT. Based on the work of Juliane Neupert, Daniel Karcher and Ralph Bock (2008), Synthetic RNAT U6 was chosen as one of our testing RNATs. The possible secondary structure of U6 was simulated by RNAstructure (Fig.5).</P></DIV></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 5: The possible secondary structure of U6.</P></DIV><DIV class="container"><DIV class="lv1"><H3>2.2 Method  </H3><DIV class="lv2"><P>In the first trail of experiments the RNA thermometers we’ve tested are from part registry of previous iGEM competition (K115002, K115017), support of team BIT-China (A?1, A2, A3), literature (U9, U10) and our own design (U6-GC). We constructed parts on pET-28a as PT7-LacO- pET-28a random sequence-RNAT-eGFP. Plasmids are transformed into BL21(DE3) for expression. After transformation, samples were cultured at 30 degrees celcius, 37 degrees celcius and 42 degrees celcius separately and induced by IPTG at respective temperature. Below are illusion of LacO-random sequence-RNAT secondary structure and their experimental results. For each structure, the 50th base is the starting point of RNAT.  </P></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><H3> 2.3 Result </H3></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>2.4 Reference List</H3><DIV class="lv2"><P>1.	Neupert, J. &amp; Bock, R. (2009) 'Designing and using synthetic RNA thermometers for temperature-controlled gene expression in bacteria', Nature Protocols, 4 (9), August, pp.1262-1273.</P><P>2.	Nocker, A. et al. mRNA-based thermosensor controls expression of rhizobial heat shock genes. Nucleic Acids Res. 29, 4800–4807 (2001).</P><P>3.	TUDelft (2008) RNA thermometer [Online image]. Available from: https://2008.igem.org/File:Rna_thermometer.png (Accessed: 19th August 2015).</P></DIV></DIV></DIV></DIV></DIV><DIV class="fifthPage" id="J4"><DIV class="fifthPageBg"><DIV class="container"><DIV class="container" id="chromoprotein"><DIV class="lv1"><H2>3. The second trial of RNAT testing</H2></DIV><DIV class="lv1"><H3>3.1 Introduction</H3><DIV class="lv2"><P>The team suspects that the poor performance of RNATs in first round of the test was due to the inappropriate coupling of T7-Lac O regulatory part. The transcription of T7 promoter starts at CACTATAG (transcription start site indicated in bold). Hence, lac operators may also be transcribed and therefore interfere with RNA thermometers. To determine whether this assumption was reasonable, the team redesigned temperature induced transcription cascades by introducing three different promoters: pBad, J23119, T7 and three RNA thermometers (A1, FourU, U6) to the project and eliminating any loop forming sequence between transcription initiation points and the start points of RNA thermometers. The team thus has nine constructs indicated as W1~9, which are showed in detailed as following:</P></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>3.2 Methodology</H3><DIV class="lv2"><P>To test the function of ribothermometer, M1 to M9 plasmids were transformed to BL21 (DE3) competent cell, which subsequently incubated at 30°C and 37 °C overnight respectively. The transformed colonies were then inoculated into 4ml LB tubes, which in turn were shacked at 220rpm at 30°C and 37 °C according to the plate where they were incubated. These tubes of strains were then induced using IPTG for W7, W8 and W9, and arabinose for W4, W5, and W6. J23119，which is one of the strongest component promoter, therefore, no need to be induced. These tubes were then shacked for another 8h before harvesting. The cell density were normalized to OD600 = 0.93, such that the centrifuged bacteria dot can be compared under UV irradiation. (Figure 1-Figure 3) The best-performed strains were further tested for fluorescence of EGFP using plate reader. 
In order to avoid the noise of full cell, in which EGFP is covered by cell wall, the supernatant cell processing sonicate broken was used for the plate reader test. The supernatant were then serial diluted on a black 96-well plate where the fluorescents of EGFP were tested. (Figure 4) The emission ray of 480nm from the top the plate reader excited EGFP, which consequently attributed to the absorbance at about 580nm. (Figure 5) 
 
							</P></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>3.3 Results and Discussion</H3><DIV class="lv2"><P>
                            As Figure 1-3 shows, W1 (J23119+A1 RNAT) which cultured at 37℃ is much brighter than at 30℃ and W7/8/9 (T7+A1/4U/U6), which cultured at 37℃ is a little brighter than at 30℃, which needs further research using plate reader. W2/3/4/5/6 doesn’t show conspicuous differences between 30℃ and 37℃.
Due to the better results of A1 RNAT compared with others, W1/4/7 will continuously be disrupted by sonication and measured.</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 1. A quick test of RNA thermometer A1 constructs in J23119 - RNAT – EGFP (Enhanced green fluorescent protein) reporter expression cascades. The validity of J23119 - RNAT – EGFP was determined by treating the constructs containing strains at 30 celsius degrees and 37 celsius degrees.</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 2. A quick test of RNA thermometer A1 constructs in pBAD - RNAT – EGFP (Enhanced green fluorescent protein) reporter expression cascades. The validity of pBAD - RNAT – EGFP was determined by treating the constructs containing strains at 30 celsius degrees and 37 celsius degrees.</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 3. A quick test of RNA thermometer A1 constructs in T7 - RNAT – EGFP (Enhanced green fluorescent protein) reporter expression cascades. The validity of T7 - RNAT – EGFP was determined by treating the constructs containing strains at 30 celsius degrees and 37 celsius degrees.</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 4. This figure contained the raw data XJTLU-CHINA obtained regarding to the performance of promoter - RNAT - EGFP constructs</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 5.</P></DIV><DIV class="lv2"><P> As Figure 5 shows, the fluorescence of the sample at 37℃ was 12 times higher than the fluorescence of the sample at 30℃, which means the amount of protein expressed at 37℃ was 12 times bigger than at 30℃. 
Hence, it can be said that W1 (J23119+A1) is the best RNAT, which has the most efficient function.</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 6</P></DIV><DIV class="lv2"><P>The result of A1 ribothermometer with pBad promoter is showed in Figure 6. As you can see, the result is not optimistic. Except the result of ? dilution ratio, all the fluorescence of the samples at 30℃ is higher than the fluorescence of the samples at 37℃.
The unsatisfactory result of W4 (pBad+A1) may be because of the misleading during the disruption.</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 7</P></DIV><DIV class="lv2"><P>Figure 7 shows the result of A1 ribothermometer with T7 promoter. It seems that A1 ribothermometer also worked well with T7 promoter. When IPTG is added as the inducer, the fluorescence of the samples at 37℃ was 6 times higher than the fluorescence of the samples at 30℃. On the contrary, in the absence of IPTG the fluorescence of each sample was really low, which means the amount of protein express is low.
As a result, J23119+A1 and T7+A1 could be used as effective ribothermometers.</P></DIV></DIV></DIV><DIV class="container" id="J5"><DIV class="lv1"><H2>4. Chromoprotein testing</H2></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>4.1 Background</H3></DIV></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>Chromoprotein is a type of protein with conjugeted structures and when expressed, it is shown in various colors. Two chromoproteins play the major role in our project as they are essential pigments for us to “draw” a colorful map. Meanwhile, chromoproteins served as report gene when other parts in this project were tested.</P><P>Besides white, the original color of <I>E. coli</I> colonies, the global warming map also needs colonies with color of yellow, blue, and green to represent desert, marine and land areas. Moreover, as this project is a dynamic demonstration of global warming, there should be a balance between the chromoproteins’ degradation and accumulation to make sure one color can appear but later disappear quickly on certain parts of the map. To achieve these goals, 3 experiments have been done. The first two experiments confirmed that all the colors can be observed with naked eyes. The third one showed how the chromoproteins degrade with LVA and AAV tails accumulated in <I>E. coli</I>.    </P><P>The chromoproteins that have been tested are: AeBlue chromoprotein (BBa_K864401), FwYellow chromoprotein (BBa_K1033910), amilGFP (BBa_K592010, yellow) and amajLime (BBa_K1033916, green-yellow).</P></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>4.2 Yield blue and yellow</H3></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 1</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>The results indicated that these exogenous proteins were expressed successfully with little toxic effect on growth of the host bacteria. The expressed color shown in Figure 2 and Figure 3 were also desirable.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 2. The four on the left is Aebluechromoprotein, and the four on the right is Fw Yellow chromoprotein</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>There was also an experiment comparing the expression of chromoprotein under different temperature. The Aeblue chromoprotein was tested and in 16 degrees Celsius it gave a deeper color compared to 37 degrees Celsius which may prove that chromoproteins could be better expressed in lower temperature. However, because it is closed to blue when expressed under 37 degrees Celsius, later other experiments were set under this temperature.  </P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 3. These two are both Aeblue chromoprotein. The left one was induced under 37 degrees Celsius; the right one was induced under 16 degrees Celsius.</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>Next, amilGFP yellow chromoprotein (BBa_K592010), and amajLime green-yellow chromoprotein (BBa_K1033916,) were further tested as backup for the yellow pigments. We spent time on yellow protein screening rather than blue ones because there were more yellow proteins available from iGem distribution.</P><P>These two parts went after a constitutive promoter (BBa_J23119) and RBS (BBa_B0034). They were both assembled on pSB1C3 and then transformed into <I>E. coli</I> SE 2. The construction is shown in Figure 4. </P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 4</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>The results indicated that these exogenous proteins were expressed successfully with little toxic effect on growth of the host bacteria. The expressed color shown in Figure 5 and Figure 6 were also desirable.</P></DIV></DIV></DIV><DIV class="hPImgStyle2Text1"><P style="color: #FFFFFF">Figure5. amajLime green-yellow chromoprotein</P></DIV><DIV class="hPImgStyle2Text2"><P style="color: #FFFFFF">Figure 6. amilGFP yellow chromoprotein</P></DIV><DIV class="container"><DIV class="lv1"><H3>4.3 Yield green------combination of two chromoproteins</H3><DIV class="lv2"><P>Green is a indispensable composition of a world map, and the changing process from green to yellow (Inland) as well as from green to blue (Coastal) are also important to mimic the effect of global warming. Inspired by Paris-Saclay igem team (2014), we combined a yellow chromoprotein and a blue chromoprotein to get green color. </P><P>Blue and yellow chromoproteins were designed to express simultaneously. Two different linking methods were illustrated in Figure 7. The reason why an extra promoter was added after the yellow chromoprotein was to increase the expression of blue chromoprotein. Additionally, it enabled us to respectively control to the expression of two chromoproteins. For instance, the green could turn to yellow by repressing the blue chropmoprotein.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 7. The construction of expressed Aeblue and Fwyellowchromoprotein together</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>The colony of BL21 (DE 3) was cultured in 4ml liquid medium for 5 hours, and induced by IPTG for 3 hours under 37 degree Celsius. The result was shown in figure 8. It was unexpected to see the final colors of the combination were blue and violent instead of green. A possible reason could be that the yellow color expressed here was not pure yellow. Without pictures kept, the colonies tended to be orange when grown on agar plate. That was why we decided to replace FwYellow chromoprotein with the backups. </P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 8. The left one is two T7 promoters and one terminator in the plasmid pET 21a. The right one is one T7 promoter and one terminator in the plasmid pET 21a</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>The testing of amilGFP yellow chromoprotein (BBa_K592010) is shown in Figure 9.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 9. The construction of expressed Aeblue and amilGFP yellow chromoproteintogether</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>This time after 5 hours’ culture in 4ml liquid medium, they were induced by IPTG for 12 hours at 16 degrees centigrade. The result is shown as Figure 10. As a result,a desired green color shown. These two chromoproteins would be applied in our map design. </P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 10.Three mono-colony expressed both Aeblue and amilGFP yellow chromoprotein</P></DIV><DIV class="container"><DIV class="lv1"><H3>4.4 Attach lva/aav tag to chromoproteins</H3></DIV></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>To perform a smooth color change from the combined green to the single expressed blue/yellow, we’d better shorten the half life of the chromoproteins so that it can quickly degrade after its expression is repressed. </P></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>At the beginning spy catcher and spy tag were took into consideration to solve the problem. However, it may overload the host. Then we adopted another method, adding specific oligopeptide to the c-terminal end of the chromoprotein to make it more vulnerable to the attack from endogenous tail-specific proteases. Two tails were chosen for our testing, LVA tag and AAV tag with the sequence of RPAANDENYALVA and RPAANDENYAAAV respectively (Andersen et al., 1998). Due to the time limitation, only Aeblue chromoprotein was tested. The construction was shown in </P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 11</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>Three groups were used in experiments: AeBlue chromoprotein, AeBlue chromoprotein with AAV tag and AeBlue chromoprotein with LVA tag. Two mono-colonies were chosen from each group. 300ml LB was used to culture 6 colonies at 37 degrees Celsius for 4 hours. Then 300ul IPTG was added to induce at 16 degrees Celsius for 12 hours. The data was collected from 5 hours to 12 hours, totally 13 points. For each point, 20ml colony liquid was obtained from the 300ml LB culture. 4ml of them was used for OD measurement at 600nm and the rest 16ml was used for protein extraction. </P><P>Firstly, 16ml bacteria were centrifuged at 6000 rpm for 5 minutes. Then the supernatant was discarded and 10ml PBS was added to resuspend the sediment. After using ultrasonic wave to break the cell, 4ml of mixture was centrifuged at 12000 rpm for 5 minutes. Finally, the absorbance of blue chromoprotein was measured at 597nm by spectrophotometer. </P><P>Results are showed as follows.</P></DIV></DIV></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Table 1：OD value measured at 13 time points at 600nm</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Table 2：Absorbance value measured at 13 time points at 597nm</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Table 3：Absorbance value divided by OD value</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 12. The variation of chromoprotein after induced 300 min</P></DIV><DIV class="hPImgStyle3Text"><P style="color: #FFFFFF">Figure 13 The color of non-tag, AAV tail and LVA tail after induced 720 minutes</P></DIV><DIV class="container"><DIV class="lv1"><DIV class="lv2"><P>As shown in the Figure 12, both blue chromoprotein with LVA tail and AAV tail had less accumulation amount than the group of no-tail, which indicated both LVA tail and AAV tail can rise the degradation rate of the chromoproteins. In addition, blue chromoprotein with LVA tail had an even lower accumulation in host cells compared to blue chromoprotein with AAV tail. It was speculated that the LVA tail provided a higher degradation rate. In conclusion, the tails can speed up the degradation process of chromoprotein, and LVA tail is highly effective in protein degradation, whereas in our project the AAV tag was chosen to gain a balance between clear color performance and high speed of degradation. Figure 13 shows the color of non-tag, AAV tail and LVA tail after induced 720 minutes. </P></DIV></DIV></DIV><DIV class="container"><DIV class="lv1"><H3>4.5 Reference</H3><DIV class="lv2"><P>Anderson,J. (1998) <I>‘New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria’</I>, Applied and environmental microbiology, 64(6), June, pp.2240-2246.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>