Team:XJTLU-CHINA/Notebook
Notebook
XJTLU-CHINA
Home
Project
Map description
RNAT
The second trial of RNAT testing
Chromo-protein testing
Parts
Modularity
Part Detail
Team
Team introduction
Attribution
Achievements
Modeling
Modeling
Safety
Policy
Q & A
Human Practice
Public speech for iGEM
iGEM teams work shop
Questionnaire for XJTLU-iGEM
LEGO
Notebook
Protocols
Mar - May
June
July
Aug - Sep
Collaboration
with NYU-Shanghai
Synbio-tech
with BIT-China
XJTLU-CHINA
Home
Project
Map description
RNAT
Improve experiment of RNAT
Chromo-protein testing
Parts
Modularity
Part Detail
Team
Team introduction
Attribution
Achievements
Modeling
Modeling
Safety
Policy
Q & A
Human Practice
Public speech for iGEM
iGEM teams work shop
Questionnaire for XJTLU-iGEM
LEGO
Notebook
Protocols
Mar - May
June
July
Aug - Sep
Collaboration
with NYU-Shanghai
Synbio-tech
with BIT-China
Notebook
Protocols
Protocol-Detection gel electrophoresis.pdf
Protocol-Gel electrophorersis and DNA purification.pdf
Protocol-Gene synthesis using PCR amplification of primers.pdf
Protocol-Gibson assembly.pdf
Protocol-PCR amplification of DNA fragment from plasmid.pdf
Protocol-Transformation.pdf
Mar - May
March
We built up a team and tried our best to negotiate with professors to get academic supports and financial sponsor. Well, the situation was not optimistic. Most professors’ worktable had already been loaded even in summer vacation and they doubted our capability to fulfill our experiment in time. Even though we just managed to have the registration done and were very excited to become an iGEMer for the first time. Also, it was spring.
April
We deeply studied on the two proposals put forward in March. One is ‘bacteria death countdown’, which aims to give a precise limit on the generation of the propagating E. coli, by altering the rate of the expression of the codon modified toxin; Another is the ‘nisin producting yeast’ which could be used in fermentation. However, we gived up them as there were problems in generation counting and expression system in yeast used to making bread. Then, our instructor suggested us to look over more projects in privious years and get inspired. We did that in our spare time, exchanged information, and brainstormed new ideas together.
May
After several rounds of discussion, we decided our topic shall be bonding circular mRNA with ribothermometer. The former is based on the mechanism of T4 intron system, and the later takes the advantage of the temperature dependent-second structure of the RNA. Several experiment protocols were settled with the preliminary experimental design. According to the preliminary experimental planning, the gene sequence was compiled.
Later in May, we also held an iGEM public speech in our university, both professors and students were involved, to get a better understanding of synthetic biology and iGEM.
June
June 22 - June 28
This was the start of our summer vacation and we started our work in lab, although this week was just for practicing experiment skills. A tough lesson learned----Agar gels in good quality make a nice day…… Besides, we got familiar with gene synthesis, PCR, electrophoresis, plasmid digestion, transformation, colony selection and so on.
June 22
PCR:                                     1) Assembly PCR                                     2) Amplify PCR                                     3) Recycle the gel
2, Ligation Ribo-1, Ribo-2, Ribo-3 with Plasmid PUC57 and transformation into TOP 10 F’
June 23
Check and obtain the correct plasmid from the desired transformed cells that cultured last night by conducting the PCR
June 24
Waiting for the results of sequencing
June 25
PCR the Ribo-1, Ribo-2, Ribo-3, recycles the gel.
Gibson assembly the PCR products of Ribo-1, Ribo-2, Ribo-3 and pET28A
Transformation into TOP 10 F’
June 26
Check and obtain the correct plasmid from the desired transformed cells that cultured last night by conducting the PCR
PCR ligation of Ribo-1, Ribo-2, Ribo-3
Gibson assembly the PCR products of PCR product from step 2 and plasmid pET28A
Transformation into TOP 10 F’ and DH10B
June 27
Check and obtain the correct plasmid from the desired transformed cells that cultured last night by conducting the PCR
June 28
Waiting for the results of sequencing
July
July 1 - July 5
The construction of plasmid ws finished in E. coli DH10B, but we could not continue our experiment due to the lack of luciferin and RNA extract box.
July 1
Isolate the plasmid X1026 from the DH10B which carries the plasmid and has been shaken over night.
July 2
Conduct the experiment of the ribothermometers. We set the experiment as follows:
No.
Incubate
Induce
IPTG
1
37℃
37℃
+
2
37℃
37℃
-
3
30℃
30℃
+
4
30℃
30℃
-
5
30℃
37℃
+
6
30℃
37℃
-
We stored the products at 4℃.
July 3
Break the cell from the product of yesterday and isolate the protein. Run SDS-PAGE.
Result of SDS-PAGE
July 4
We conducted the PCR of MBP-EGFP and recycled the agrose gel. Cut the pET-28A vector with PstI and NcoI. Insert the MBP-EGFP in to the vector through Gibson assembly.
July 5
Group meeting, members updated each other. We decided to switch our direction of our project.
July 6 - July 12
While Melody, Sue and Jodie kept working in lab, other members were searching what exactly pathway/proteins our circular mRNA and ribothermometer gonna work with. When James, Polly and Mag were trying to figure out every detail in synthesizing spider silks by circular mRNA, Gandi jumped out the idea of simulate the process of global warming by bacteria. It indeedly attracted William, so William quickly finished the first edition of the circuit design.
The ribothermometer worked not very well.
July 6
We selected the colonies with toothpicks and amplified them with primers. Run agrose electrophoresis. However, no bands showed up. We still selected 6 clones and sent them to sanger sequencing. Jodie designed the primers of four control groups.
July 7
Group meeting with our supervisor Yunpeng Zhong. We decided our final project: simulating the process of global warming with bacteria. Also, we make a plan of the following experiments.
July 8
We synthesized the vector of circular mRNA but without ribothermometer. And we transformed the vector into E. coli DH10B.
July 9
We conducted a RT-PCR to test if the mRNA had formed a circle.
July 10
We amplified the cDNA product of yesterday to see if the length is right. Unfortunately, the length was not right.
July 11
We cannot believe the result of yesterday. So we repeat the experiments of yesterday and the day before yesterday. This is the last shot of circular mRNA, if it fails again; we shall just give it up, literately! And, yep, it failed again…..
July 12
Group members met. We decided to focus on the ribothermometer and Gandi’s idea.
July 13 - July 19
After negotiating with our instructor, we had a vote and we finally set our topic----simulate the process of global warming. We began to check the genetic circuit designed, meanwhile, as the ribothermometer U6 chosen from paper showed a severe leak at 30℃, we decided to try more.
July 20 - July 26
This week’s experimental task was making new sets of ribothermometers. Sue, Melody, Victor, Polly and Jodie were responsible for that. Others rechecked the parts like promoters, operators, repressors used in the circuit.
Ziang Shi showed us the website designed. It was a surprise! We all liked it.
Another day, an experienced leader in Sichuan University’s iGEM team joined us.
Da Xu reminded us of the modeling in our project. James began to work on that.
June 22
Design the vector of the experiments of ribothermometer.
June 23
Design the primers of the fragments and synthesis the primers.
June 23
Design the primers of the fragments and synthesis the primers.
RibothermometersE
Plasmid
June 25
We selected the colonies of yesterday. U0, U6, and U6-GC didn’t show right bands on agrose gel. We sent the clone with right bands to sequencing and repeat the transformation of U0, U6 and U6-GC.
June 26
Colony test of U0, U6 and U6-GC was conducted. Each one contains at least one right band on the agrose gel. We sent them to sequencing.
July 27 - August 2
Sue, Polly, Melody, Victor and Jodie continued constructing ribothermometers. It took longer time than expected. It just seemed sometimes the bacteria BL21 were not very active. We were upset about the delay. Anyway, at this time, Jodie introduced us a good way to record the experiment every day. Polly tried to grow the E. coli on a membrane.
There were several bugs in our circuit design. We changed the quorum sensing system and promoters to satisfying different requirements.
June 27
The result of sequencing shows that all of our sequence, except U10 and K115107, was wrong. We repeat the experiment starts from the PCR (July 24th).
pET-28a XbaI&XhoI
June 28
We isolated the plasmid of U10 and K115017 from Top 10F’ and named them as T1017 and T1018. Plasmid T1017 and T1018 were transformed into BL21.
June 29
We repeated what we have done in July 24th, transformed A1, A2, A3, U0, U6, U6-GC, U9, K115002 into the Top0F’ and DH10B.
June 30
Colony test, U6, U6-GC and K115002 still has no right bands. We sent the rest to sequencing and repeat the transformation of U6, U6-GC and K115002.
A1, A2, A3, U0
U6-GC, U6, U9, K115002
June 31
Enzyme digestion of U0
August 1
Colony test of U0 and U633A
August 2
We transformed plasmid X022t (A1), X024t (A3), X030t (U6-GC), X032 (U9), T1012 (U10) and X033 (K115002) into BL21.
Aug - Sep
August 3 - August 9
Sue, Polly and Mag tested the ribothermometers in different ways, both qualitatively and quantitatively. However, they still didn’t work well. Vincent processed the data and there was only a 1.2 on-off ratio showed by ribosthermometeres. James thought it was because the lac operator stood in front of the ribothermometers disturbed its secondary structure. James and Neil wrote another set of sequence to fix that problem.
James finished writing the fifth edition of the sequence in snapgene and sent it to our instructor to be analyzed. But the modeling part had no progress.
Shanghai meet up.
August 10 - August 16
Zhang Haoqian advised us to change the lux promoter and J23119. Plus his suggestion, we just realized there are still too many parts needed to be tested separately. So instructor taught us to modularize our circuit. We separated every construction into two parts. One involves promoter and RBS, the other one belongs to protein and terminator. In such case, we were able to examine each part and later quickly install proper parts together to construct our final circuits. We together finish coding the sequence.
A bad news came out that the yellow chromo protein turned out to be orange when expressed. That led our land turn to purple instead of green! It was beautiful but……unacceptable. We needed to change it.
August 17 - August 23
According to the separation of parts, we named all the parts and double-cheaked the sequence:
-PR-1~PR-11: Different promoters and RBS;
-CT-1~CT-9: Different proteins and terminator.
James and Neil respectively designed the new thermometer sequence:
-TW-1~TW-9: Different promoters and different thermometers designed by James;
-TJ-1~TJ-9: Different promoters and different thermometers designed by Neil.
Vincent, William and Neil started to construct and transform the parts.
August 17
Conducted PCR and gel recycle of PR-2~PR-9. PR-2/8/9 had wrong length. Cut the vector pSB1C3 with Xbal1 and Spe1.
PR-2~PR-9
pSB1C3 cut by Xbal1&Spe1
August 18
Conducted PCR and gel recycle of CT-2~CT-9, terminator. CT-3/4/5/7 and terminator have wrong length.
CT-2~CT-9, terminator
Recycle of CT-2/6/8/9
August 19
Conducted PCR and gel recycle of TW-3~TW-7, CT-3~CT-7 and terminator. Due to the failed-amplification of terminator again, primer was designed renewedly.
TW-3~TW-7
CT-3~CT-7
August 20
We conducted the PCR of pr-1 pr-2, pr-8, ct-4 and Terminator. Pr-2, pr-8 and ct-4 showed the clear and right band on the agarose gel, while pr-1, pr-9 and Terminator didn’t show any band. Then we recycled the gel of pr-2, pr-8 and ct-4. The vector of pSB1C3 was cut by speI. We introduced pr-2, pr-8 and ct-4 into the Top 10 F’ and DH10B.
Pr-1, pr-8, pr-9
pr-2, ct-4
Ter
pSB1C3 cut by SpeI
August 21
We selected the clone that we introduced into the cell, only ct-4 showed the band. The clone of ct-4 was sent to sequencing to see if the sequence is right. PCR of pr-1, pr-9 and Terminator was conducted again as well as tj-9 and this time they all showed the right band on the agarose gel. The pr-1, pr-2, pr-8, pr-9 and Terminator was recycled and introduced into Top 10 F’ and DH10B.
Pr-1, pr-9
Terminator
tj-9
August 22
We repeat what we did yesterday because nothing showed when we select the clones, nothing showed after the PCR, even the vector was not cut correctly. Leave me alone…….
August 23
Except pr-1, other fragments sowed at least one right clone. We sent them all to sequencing. Also, the sequencing result of Aug 21st came out, nothing was right. Yep, nothing………. Damn
August 24 - August 30
James found two kinds of new chromoprotein that would be more suitable for our project and named as CT-10~CT-13. Jodie added label for degradation to the sequences. The further experiment of Neil’s thermometers was built and named as NT-1~NT-8, which were connected by TJ-1~TJ-8 and CT-3 (The design of TJ-9 was deleted after several attempts of amplification. What a pity…). At the same time, James’ thermometers were also designed and named as W-1~W-9, which were connected by TW-1~TW-9 and CT-3. Vincent, William, Jodie and Neil still devoted to constructing and transforming the parts.
August 24
We connected ct-1, 2, 3, 4 and 7 with the Terminator and conducted the PCR of our chromoprotein. Fortunately, all of them showed the right band on agarosse gel. We introduced them into the cells.
Ct-1, 2, 3, 4, 7
chromoprotein
August 25
PCR of tw-1 to9 was conducted as well as ct-4, 5, 6and 9. Well…..the operators forgot to name the pictures. I don’t know what these are. All I know is that something came out but something failed. They do label the tubes; they just forgot to name the pictures, so we still have our products.
August 26
Tw-2 to 9 showed correct clones and were all sent to sequencing.  The clone of blue chromoprotein was also right. The sequencing result of 23rd came out. So far, we have successfully synthesis the pr-1 to 9, ct-1 to 3, 7, 8, 10 to 13 except, tj-1 to 8, still missing pr-10, 11 and tw- 1. Ct-4, 5, 6 and 9, tw-2 to 9 on sequencing.
August 27
We conducted the PCR of pr-10, 11 and tw-1. And connected the ct-1, 2, 3, 7, 8, 10, 13 with terminator and connected blue chromoprotein with LVA, AAV tail and terminator as well as the J23100 promoter. As always, we introduced them in to the cells, but this time we choose Epi400 and STBL3 to be our competent cell.
Connection product
pr-10, 11and tw-1
August 28
Except tw-1, all fregments showed the right clone. We sent these clones to sequencing. The sequencing result of 26th came out, ct- 9 and tw-2 to 9 was right. We introduced ct-4, 6 and tw-1 into Epi400 and STBL3 again. Also, we started to synthesis the two pathways of our “Marine” E. coli.
August 29
The clones of tw-1 and ct-6 were still not right. We conducted the PCR of them again and introduced them into Top 10 F’ and STBL3.
Clones of tw-1, ct-4 and ct-6
PCR product of tw-1 and ct-6
August 30
The sequencing result of 28th showed that tw-2 to 9 was all right, so we connected them with protein and upgraded them into w-2 to 9 and introduced them into Top 10 F’ and DH10B Still, no right clones of tw-1 showed up. We are just about to give him up.
Clones of ct-6 and tw-1
August 31 - September 6
August 31
The sequencing result of W-1~W-9 each had a wrong repeat of terminator. Jodie was sad and decided to take a trip to Nanjing, Jiangsu Province for calming down…
September 1
The sequencing result of TW-9, T7 Blue aav and Marine-2 were correct. Cheers!
…But wait… Enzyme Spe1 was used up. Our hero advisor, Yacheng Guan, borrowed one tube from another company. We regarded it as treasure…
September 2
We got the correct sequencing result of W-5, W-8 and re-amplify W-2/3/4/6/7, TW-1 and PR-11. Also we cut the plasmid to be vector. The ‘treasure’ was nearly used up again…
September 3
Correct sequencing! W-4/5/9, PR-10.
September 7 - September 13
Vincent devoted to the test of the three different ribothermometers (A1, 4U and U6) with three different promoters (J23119, pBad and T7). The bacteria were incubated and induced respectively at 30℃ and 37℃. After that, the bacteria were disrupted and plate reader was used to measure the fluorescence of report protein: eGFP.( For more details, please refer to Project: Improve experiment of RNA thermometer.)
Judie conducted the test of the four chromoproteins: AeBlue chromoprotein (BBa_K864401), AeBlue chromoprotein with AAV tag, AeBlue chromoprotein with LVA tag, FwYellow chromoprotein (BBa_K1033910), amilGFP (BBa_K592010, yellow) and amajLime (BBa_K1033916, green-yellow). Also blue and yellow chromoprotein were used to yield green colour. (For more details, please refer to Project: Chromoprotein testing.)
September 14 - September 18
Zi’ang Shi devoted to the design of wiki.
