<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_diary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/diary</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A href="https://2017.igem.org/Team:Cardiff_Wales">Home</A><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/meet_the_team">Meet the Team</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_TSH">Team TSH</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_Luc">Team Luciferase</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_PlantP">Team Plant Promoter</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/projectdescription">Project Description</A><A href="https://2017.igem.org/Team:Cardiff_Wales/results">Results</A><A href="https://2017.igem.org/Team:Cardiff_Wales/basicparts">Basic Parts</A><A href="https://2017.igem.org/Team:Cardiff_Wales/compositeparts">Composite Parts</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Human Practice</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/Human_Practices">Integrated Human Practices</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Public_outreach">Public Outreach</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Survey">Survey</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Our_research">Our HP Research</A></DIV></DIV><DIV class="container"><A href="https://2017.igem.org/Team:Cardiff_Wales/Modelling">Modelling</A><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Lab book</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/protocols">Protocols</A><A href="https://2017.igem.org/Team:Cardiff_Wales/diary">Diary</A><A href="https://2017.igem.org/Team:Cardiff_Wales/health_and_safety">Health and Safety</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Medal Criteria</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/goldfulfilment">Gold</A><A href="https://2017.igem.org/Team:Cardiff_Wales/silverfulfilment">Silver</A><A href="https://2017.igem.org/Team:Cardiff_Wales/bronzefulfilment">Bronze</A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1> Project Diary </H1><LEFT><H3> Week one </H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>10/07/2017</H4></SPAN><DIV class="dropdown1-content"><UL><LI>Meet up</LI><LI>Safety and risk assessment forms</LI><LI>Review to do lists</LI><LI>Safety = Helen</LI><LI>Sequencing = Niall</LI><LI>Plants = Jade and Sarah</LI><LI> Stockroom = Emily, Ryan, and Thomas </LI><LI> Arranged lab and workspaces </LI><LI> Sorted pipette tips into boxes and autoclaved. </LI><LI> Made medium using 25 g/l LB broth. Made medium using 10g/L agar granulated (made up to 400ml water). Then autoclaved. Pyrex class will not smash, cools down quickly/till hand-hot.</LI><LI> Chloramphenicol + kanamycin resistance antibiotics: One medium had 100mg/ml kanamycin added (400microlitres to 400ml), the other had 34mg/ml chloramphenicol added (400microlitres to 400ml). Plated 19 plates of each. </LI><LI> Transferred tobacco plants (grown on 19/6) into new pots to prevent overcrowding.</LI><LI> Organised ourselves into 3 'teams':</LI><LI> Team_PlantP = Niall and Ryan </LI><LI> Team_TSH = Helen, Emily, and Sarah </LI><LI> Team_Luc = Jade and Tom </LI></UL></DIV></DIV><DIV class="dropdown1"><SPAN><H4>11/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Re-suspended iGEM primers 45 - 52 at 100uM. Created working stock at 10uM. </LI><LI> Used Primers in a PCR with Arabidopsis thaliana gDNA to amplify 4 plant promoters (with -ve controls)</LI><LI> Ran PCR results on 1% agarose gel with 1x TAE. No success at amplifying desired sequences (empty gel lanes).</LI></UL></TD><TD><UL><LI> Resuspend TSH gBlocks to concentration 0.1pmol/ul (v.clean water). </LI><LI> Digest TSHantag and TSHantag(his) and introduced into pSB1C3 (plasmid) with EcoR1 and Pst1. </LI><LI> Ligation mix preparation, left to ligate overnight. </LI><LI> Re-do LB broth and agar plates as we used broth in the agar plates! </LI><LI> Planted 16 more pots of tobacco plants.</LI><LI> Helen doing safety forms and biosafety poster exercise </LI></UL></TD><TD><UL><LI> Made up agar plates inoculated with either kanamycin, chloramphenicol or chloramphenicol, X-gal and IPTG.</LI><LI> Transform DNA from iGEM registry - Did not work as the cells used were bad.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>12/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Redid failed PCR from yesterday and ran on a gel - slightly more success (Primer dimers (probably) in 2 lanes). </LI><LI> Niall carried out the first part of 2 SOE-PCRs to convert target nucleotides for BsmBI digestion into non-target nucleotides through site directed mutagenesis. SOE-PCRs used to modify the P19 gene and LUCx5 gene.</LI><LI> Ran on gel - gDNA was poor so no bands. </LI><LI> Ryan: Performed plasmid minipreps according to QIAprep® Spin Miniprep Kit. Measured the concentrations of each of the DNA samples.</LI></UL></TD><TD><UL><LI> Digest and ligation using the P5B1C3g plasmid for golden gate cloning, using both TSHantag and TSHantag-his. </LI><LI> Ligation reaction using PCR machine (15 cycles). </LI><LI> Transformation of biobrick and golden gate plasmids (ligated 11/7 and 12/7) into competent cells. Grown on chloramphenicol for biobrick, and chloramphenicol + Xgal + IPTG for golden gate. </LI><LI> Procedure of transformation: ice (10’), heat shock in water bath at 42degrees (1’), ice (5’), add 1 ml LB broth without antibiotic, shake (1hr at 37 degrees), centrifuge for 1 minute at 13k,  plate using resuspension and glass beads. </LI><LI> Incubation of plates at 37 degrees overnight. </LI></UL></TD><TD><UL><LI> Mini Prep of level 0 parts</LI><LI> Transformed DNA from iGEM registry again</LI><LI> Made competent E.coli, however the water used had not been autoclaved.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>13/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Extracted DNA from gel to get p19 and LucX5 DNA. </LI><LI>Quantified DNA and stored in freezer for future use in plasmids.</LI><LI> Extracted DNA from Arabidopsis thaliana.</LI><LI>Quantified DNA. </LI><LI> Took two best samples and ran PCR with plant promoter primers (IGEM_45 to IGEM52). </LI></UL></TD><TD><UL><LI> Re-do transformations as broth was not added to agar. </LI><LI> Steps: ice, heat shock, ice, shake for 1 hour, plates. </LI><LI> Did surveys for iGEM toulouse about cholera. </LI></UL></TD><TD><UL><LI> Transformed DNA from iGEM registry again as well as one of the level 0 parts mini prepped the day prior (as a test) into competent cells.</LI><LI> Made up agar plates inoculated with either kanamycin, chloramphenicol or chloramphenicol, X-gal and IPTG.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>14/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Ran gels with PDF1p, PR2p, GST6p, and WRKY30p, with gDNAs 2 and 3 (highest DNA concentration samples). All lanes were successful with no contamination, except WRKY which had no result except primer dimers.</LI><LI> Cut and digested the gels and purified DNAs - 3 test tubes each with either PDF, PR or GST promoter regions.</LI><LI> Measured concentrations of the promoter DNA.</LI><LI> Set up PCR of WRKY30p at different temperatures (gradient in the PCR machine). Temperatures 50, 53, 57, and 60 degrees.
 </LI><LI> Ran gels with 8 lanes (4 negative controls, and 4 with one of each of the WRKY temperatures)</LI><LI> All 4 samples had primer dimers but no amplified gDNA - conclusion = most likely primers are not specific enough to amplify gDNA</LI></UL></TD><TD><UL><LI> TSH 5G/TSH10g plates grew colonies. </LI><LI> Redo TSH/H 3:1, TSH/H 5g, TSH 5:1 and biobrick controls for transformations.</LI><LI> Incubation over night</LI></UL></TD><TD><UL><LI> Made up agar plates inoculated with either chloramphenicol or chloramphenicol, X-gal and IPTG. Reducing the concentration of chloramphenicol from 34 to 17, and doubling the concentration of X-gal.</LI><LI> Transformed DNA from iGEM registry again into competent cells.</LI><LI> Made a protein gel plate to be used in a future test.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week two </H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>17/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Ran PCRs with WRKY at 52 and 56 degrees for primer annealing. With half and double primer concentration in each, to see if either would prevent primer dimers and allow annealing to DNA.</LI><LI> Ran gel and found only primer dimers in every sample. gDNA is fine because the positive control had its band. </LI><LI> WRKY primer is ineffective, so we must design another.</LI></UL></TD><TD><UL><LI> Plates again did not work. Put them in the incubator at 37 degrees. </LI><LI> Possible problems with chloramphenicol</LI><LI> Running PCR of TSH 5G and TSH10G colonies.  </LI><LI> Made gels, on 120 for 50 minutes and they did not work so re-doing them!</LI><LI> Re-ran gels, have better results</LI><LI> Helen: from incubated plates 1 was successful (TSH 5:1). Took colonies from previous successful plates using 5 ml broth. Left to incubate and shake overnight at 37 degrees. </LI></UL></TD><TD><UL><LI> Transformed the universal promoter (plate 1 - 3O) from the iGEM registry as well as GFP made in the mini prep (12/07/17) and a negative control to test the competent cells.</LI><LI> Inoculated agrobacterium and E.coli cultures.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>18/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Designed new WRKY primer</LI><LI> Can’t do anything else until level0 plasmids are done to perform minipreps.</LI></UL></TD><TD><UL><LI> Performed a mini prep on the one colony that grew successfully (QLAprep Spin Miniprep Kit).</LI><LI> Performed PCR on the mini prep, GFP plasmid, and level 0 colonies </LI><LI> Ran a gel using the PCR products and the hyperladder 4 </LI><LI> Helen: Grew bacterial cultures using chloramphenicol and kanamycin antibiotics. 5ml LB broth and 5 ul of antibiotics. Left overnight on 200 RPM at 37 degrees. </LI><LI> Grew colonies from plate A (4), B (1), C (4), Rob’s colonies. </LI></UL></TD><TD><UL><LI> Re-incubated the agrobacterium overnight as they were not spinning so could not grow.</LI><LI> Made some more competent E.coli using previously made competent E.coli as a starting point.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>19/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Performed minipreps on samples A1 and A2, C1 and C4. Isolated the DNA and measured concentrations of samples A1-A4 and C1-C4 against EB as a blank.</LI><LI> Performed restriction digest on P19, Luc, PDF1, PR2 and GST6, according to long protocol in ligase buffer. Put in PCR machine for the process.</LI><LI> Performed ligation and gel analysis to see whether the plasid cut and incorporated the recombinant DNA.</LI></UL></TD><TD><UL><LI> Minipreps on plates A (level 0) and C (level 1). </LI><LI> Digest process and then used PCR using long protocol.</LI><LI> Tested level 0 plasmids.</LI><LI> Also completed 2 minipreps of GBA2 using QIAprep protocol</LI><LI> Did a digest of the plasmid in the water bath at 37 degrees using BsmB1 </LI><LI> Ran a gel comparing the undigested plasmid and the ones digested with BsmB1 </LI></UL></TD><TD><UL><LI> Ran a transformation to compare the newly made competent cells with the competent cells made on 12/07/17.</LI><LI> The agrobacterium that were grown overnight were used to infiltrate the leaves of tobacco (to be harvested in two days time), by suspending the cells in induction buffer.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>20/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Transformed bacteria with level 0 plasmid parts - P19, LUCx5, GST-6, PR-2, PDF-1.</LI><LI>Performed level 1 restriction digest/ligation with 35S, XVE and NOST.</LI></UL></TD><TD><UL><LI> Sarah and Emily: Digestion of biobrick plasmid </LI><LI> Ligation reaction for this digest left overnight </LI><LI> Helen: Grew colonies from A (Chl)/C (kan)/GBA2 (kan) in the incubator overnight </LI><LI> Sarah and Emily: emailed BBC wales. </LI><LI> Helen: Fiona Wylie contact. Booked dates for the 7th October and 16th September events as part of our human outreach.  </LI><LI> Christian from Cardiff_iGEM16 came to talk to us, said that looking at the medals protocol is essential! </LI></UL></TD><TD><UL><LI>Checked new vs. old cells that were left to grow overnight. Using GFP and a negative control. The new cells were found to be a lot more competent.</LI><LI>Transformed the correct parts from the iGEM registry as well as B1 and B2.</LI><LI>Made up more chloromycin and kanamycin plates.</LI><LI>Practiced grinding leaves in EZ extraction buffer using a pestle and mortar, to prepare for tomorrow.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>21/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Investigated P19, LUCx5, PDF, PR, GST colonies for successful transformants - Only identified one transformant for P19. Extracted colony and cultured in LB broth w/ CHL to increase cell number. Repeated with -ve controls. After incubation, performed a PCR with bacterial colony and primers 57+58 and ran products on gel.</LI><LI>Repeated restriction digest of P19, LUCx5, PDF1, PR, GST. </LI><LI>Transformed E.coli with level 1 plasmid containing 35s, XVE and NOSt and plated cells to grow over the weekend.</LI></UL></TD><TD><UL><LI> Minipreps on colonies grown overnight at 37 degrees and in the shaking machine (200 rpm). </LI><LI> Minipreps carried out on A/C/GBA2 then put in the team_TSH freezer </LI><LI> Sarah and Emily: biobrick transformations into competent cells. The transformed bacteria were then plated out and incubated at room temperature over the weekend. </LI><LI> Golden gate level 0 plasmid digest and ligation. Changed some methods: increased level to 1ul of BsmBI. Reduced level of acceptor plasmid from 200ng at 2:1 to 50ng at 1:5. Calculations viewed in HD book. </LI><LI> Human outreach aspects: producing a survey investigating people’s perspectives on producing a protein using tobacco plants to treat Grave’s Disease. Dewi put on twitter!! Also Skype call with Washington team to discuss collaborations. </LI></UL></TD><TD><UL><LI>Checked the plates grown overnight - B1 and B2 grew very well however the plates from the iGEM registry grew more slowly.</LI><LI>Cultures from the B1 and B2 plates were inoculated with kanamycin and left to grow over the weekend in a shaking incubator at 22 degrees.</LI><LI>The plants that were infiltrated on 12/07/2017 were prepared by grinding them in a pestle and mortar on ice with EZ extraction buffer, before centrifugation at 13000 rpm for 5 minutes. 1/10 the volume of buffer Z was then added to the supernatant liquid in a fume hood before placing them in the freezer.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week three </H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>24/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH></TR><TR><TD><UL><LI>Inspected colonies - all white with contamination, including the negative control - suspect the plates did not have Chloramphenicol or X-Gal.</LI><LI>Re-ran PCR of level 1 plasmid - no bands so must redo transformation.</LI><LI>Transformed more competent cells with the level 1 35S-XVE-NosT plasmid, as these had not grown at all. Incubate at 37 degrees overnight.</LI><LI>Recombine pGEM-EASY plasmid with PDF, PR, and GST. Geraint will try and transform some cells to see if these work, otherwise they will be transformed in the morning after 22 degree overnight incubation.</LI>
Team_LUC
<LI>Mini prepped the B1 and B2 cultures that were grown over the weekend.</LI><LI>Cultures from the iGEM registry (P1-3O, P3-3f, P3-14A, P6-18J) plates were inoculated with chloramphenicol and left to grow overnight in a shaking incubator at 37 degrees.</LI><LI>Set up PCR of the mini prepped B1 and B2.</LI><LI>Poured agarose gel.</LI><LI>https://www.promega.com/-/.../pgem-t-and-pgem-t-easy-vector-systems-protocol.pdf</LI></UL></TD><TD><UL><LI> Addition of adenine to the g blocks by treating with taq. Must add taq to the Gblock before the normal reaction. </LI><LI> Ligation reaction to get the g blocks into the pGEM plasmid. </LI><LI> Transformations and the cells were left to grow overnight. </LI><LI> Biobrick Plates left to grow over the weekend at room temperature showed 1 white colonies on TSH 5:1. </LI><LI> Golden gate plates showed more success, with white colonies on TSH2, TSHH2. </LI><LI> The unsuccessful biobrick plates were incubated at 37 degrees to encourage growth. </LI><LI> Golden gate plates placed in fridge after incubation to help decipher the colouring of the colonies. </LI><LI> Possible reasons why: X-gal problems? No Chloramphenicol. Did it degrade? Negative control was white! Not good! </LI><LI> Still grew the plates in L </LI><LI> Human outreach: emailed ITV and BBC wales regarding project. Also contacted Wales Online. Contacted british association of endocrine and thyroid surgeons about helping us with clinical outreach. Also contacted British Thyroid Association about our project. </LI><LI> Single white colony on biobrick TSH 5:1 plate was grown up using 300 ul LB broth and prepped for PCR. </LI><LI> 10 Golden gates colonies from both TSH and TSHHH were grown in 300 ul LB broth and shaken at 37 degrees for 3 hours. </LI><LI> PCR: 20 ul reaction with 3ul DNA and primers 57 and 58. 35 cycles (94 (20s)/56 (20s)/72 (1 min 30). </LI><LI> Discussing Washington Uni collaboration. </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>25/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>pGEM: Transformed cells with level 0 pGEM plasmids containing PDF1, PR2 and GST6 inserts - left to grow overnight in incubated.</LI><LI>Isolated potential transformant colonies (PDF1, PR2 and GST6) and grew culture before colony PCR.</LI></UL></TD><TD><UL><LI> Gel was run using hyperladder 1KB+, golden gate and biobrick TSH 5:1 colonies from 21/07/17 (TSH 1-10, TSHH 1-10, +ve GFP, -ve water, TSH 5:1) </LI><LI> Investigated TSH, TSH and negative control plates for growth of successful transformations. Transformants identified on TSHH  LB, chl, x-gal and IPTG plates. Colonies small in size so left in incubator at 37 degrees to encourage further growth. </LI><LI> 3 negative controls, 4 TSH and 5 TSHH colonies were isolated and grown in 300 ul LB + AMP and shaken for 3hr at 37degrees. </LI><LI> Colony PCR of controls, TSH, and TSHH transformant colonies carried out at both 52 and 56 degrees. Primers used = 55 + 56. </LI><LI> pGEM: </LI><LI> Second pGEM transformation to confirm growth of transformants from original plates. </LI></UL></TD><TD><UL><LI>Mini prepped iGEM registry parts grown overnight. They were then put into the PCR machine before running a gel on these parts.</LI><LI>Ran gel electrophoresis of the B1 and B2 parts that underwent PCR yesterday. The bands were successful so they were sent off for sequencing.</LI><LI>Ran polyacrylamide gel electrophoresis (PAGE) of agrobacterium infiltrated leaves prepared on 21/07/2017. The gel was then placed in coomassie stain overnight.</LI><LI>Inoculated more P6-18J to grow overnight.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>26/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Level 0: Ran gel of PDF, PR, and GST pGEM colonies to detect the presence of the insert (amplified in PCR using only promoter plasmids). Absolutely nothing on gel - something went wrong.</LI><LI>pGEM: Miniprepped pGEM PDF, GST, and PR plasmids and measured concentrations.</LI><LI>Performed level 0 digest overnight and ‘Big G’ transformed and grew overnight.</LI><LI>Grew them overnight to screen for blue/white colonies in the morning.</LI></UL></TD><TD><UL><LI> Investigated successful plates from second pGEM transformations. Blue and white colonies were observed on TSH, TSHH and negative LB, AMP, IPTG plates. White colonies were extracted and grew at 37 degrees for three hours (using 300 ul LB broth + AMP and colony). </LI><LI> Colony PCR of controls, TSH and TSHH transformant colonies </LI><LI> Organised Skype with Valencia </LI></UL></TD><TD><UL><LI>Transformed agrobacteria with miniprepped B1 and B2 (and negative control).</LI><LI>Protocol: add 5 microlitres of B1 and B2 to the agrobacterium, leave on ice for 10 minutes, add to liquid nitrogen for 5 minutes, 5 minutes at 37 degrees celsius, and then on ice for 5 minutes. Then add 1 ml LB (no antibiotic), shake at 28 degrees for 2 hours, spin down at 13000 rpm for 1 minute, resuspend pellet and plate onto lk-rif plates, grow overnight at 28 degrees.</LI><LI>Mini prepped the P6-18J grown overnight.</LI><LI>Removed coomassie stain from the gel and used the destaining agent.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>27/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Level 0: P19, LucX5, GST, PD, and PDF plates all white so picked 10 colonies at random from each to plate on fresh x-gal plates to perform PCR on potential white colonies and check for insert.</LI><LI>WRKY: New WRKY primers arrived so set up PCR overnight to see if these primers work.</LI></UL></TD><TD><UL><LI>Minipreps of 1x TSH, 1xTSHH pGEM. </LI><LI>Set up 4x level 0 reactions with level0 acceptor (pSB1C3g). Inserts are TSH gblock2, TSH gblock3, TSH miniprep and TSHH miniprep. Digest and ligation process using BSMBI and T4 DNA ligase, changing the temperature from 37 degrees to 55 degrees, as this is the temperature at which the enzyme cuts </LI><LI>Survey stuff. </LI></UL></TD><TD><UL><LI>Made more competent E.coli using batch #2 as the starting point.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>28/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Level 0: All P19, LUCx5, PDF1, PR2, GST6 colonies taken from white colonies from previous day turned out to be blue on new plates. </LI><LI>Carried out another transformation using PDF, PR, GST, P19 and LUCx5 digests. After transformation, plated cells on old agar plates with IPTG, x-gal and Chl to grow over the weekend. </LI><LI> Level 1: Performed another level 1 digest with 35S, XVE and NOSt, transformed cells and grew over the weekend</LI><LI>WRKY: New primers proved successful - ran PCR products on gel and carried out DNA extraction from gel. Have now isolated WRKY promoter sequence.</LI></UL></TD><TD><UL><LI> Transformations using pGEM (TSH/TSHH) and gblock (TSH/TSHH), grew up the colonies over the weekend </LI><LI> More editing of survey - remove the section on Graves' disease </LI><LI> Valencia skype call for collaborations (look at last years valencia Wiki). </LI><LI> Get ready for talk on monday at 2 pm (what have we done so far). </LI></UL></TD><TD><UL><LI>Transformed competent cells in batch #2 and #3 with GFP (and without as a negative control) to test whether the new batch were more effective. These were left to grow on the bench over the weekend.</LI><LI>Spread colonies from the B2-1 plate onto two new plates (kan + rif) in order to grow over the weekend. B1-1 was not used as the colonies grew poorly over the two day period.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week four</H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>31/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Level 0: Every colony white AGAIN - old plates may have had poor X-gal. Re-plating the transformed cells from 28/07/17 on fresh plates with functional x-gal to grow overnight.</LI><LI>Also re-plating 20 white colonies from P19, LUCx5, PDF, PR, GST on fresh plates to check for successfully transformed cells on plates from 28/07/17.</LI><LI>Level 1: No colony growth on plates from 28/07/17… Not sure why, could be that there isn’t enough pGB-a2 plasmid DNA (Ryan vortexed during the mini-prep protocol where he shouldn’t have), maybe kanamycin is killing all the cells?  - going to ask Geraint and then potentially do another digest.</LI><LI>WRKY: Set up a lvl 0 restriction digest, then transformed cells with product and grew overnight. </LI></UL></TD><TD><UL><LI> Prepping for 2pm talk (what is the project? What have we experienced in the practical work? Human outreach aspects?)
Colonies grown over weekend from 28/0717 </LI><LI> Successful transformants showing white colonies from TSHH pGEM, TSHH gblock and TSH gblock. No colonies were visible from TSH pGEM (plates: chl, x-gal, IPTG). </LI><LI> Grown on chl 300 ul LB broth and colonies. </LI><LI> Colony PCR of TSHH gblock (X), TSHH pGEM (Y), TSH gblock (Z). </LI><LI> Ran a gel using the PCR products. </LI></UL></TD><TD><UL><LI>Made more chloramphenicol 34 µg/ml plates.</LI><LI>Re-transformed the competent cells as was done on 28/7/17 as the negative plates had a lot of colonies on them. This could have been due to the plates used being fairly old. These new transformations were left to grow overnight at 37 degrees celsius.</LI><LI>Inoculated B2-1 with kanamycin and rif to grow overnight at 28 degrees celsius. In order to infiltrate tomorrow.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>01/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Inserted WRKY into pGEM plasmid. Transformed and grew cells - none grew.</LI><LI>Set up ligation of pGEM GST, PR, PDF plasmids into level 1 18J plasmid. Transformed cells and Grew cells on plates - some blue colonies, most plates with no growth.</LI><LI>Set up sequencing tubes of pGEM versions of PDF, PR, and GST - 50ng/ul in 15ul.</LI></UL></TD><TD><UL><LI> PCR: </LI><LI> Gel using PCR products from 31/07/17 inconclusive. No negative control used so bands could not be analysed. </LI><LI> Therefore, grew colonies from PCR products from 31/07. Only 2 white colonies (2 TSHH gblocks (9 + 10) transformed. 14 of the other colonies failed to transform, remaining blue in colour. </LI><LI> Following the transformations, a colony PCR was carried out (2 TSHH gblocks, one blue TSH pGEM, one negative (no DNA), positive control (plasmid)). </LI><LI> Ran a gel using the PCR products. No bands were present in the lanes representing the colonies. </LI><LI> Level 0 plasmid: </LI><LI> Using the level 0 plasmid completed a digest and ligation. </LI><LI> Increased BsmB1 volume from 0.5 ul to 1ul. Also increased T4 ligase from 0.5 ul to 1 ul. </LI><LI> Plates: </LI><LI> Made 16 Chl + x-gal + IPTG and 8 AMP + x-gal + IPTG plates
2 chl + x-gal + IPTG plates used to plate cells from the transformations using the digestion/ligation products from earlier in the day. The plates were grown overnight. </LI><LI> pGEM plates: </LI><LI> Original TSH pGEM and TSHH pGEM colonies grown, grown further on fresh AMP + x-gal + IPTG plates. 20 colonies from each plate grown on the 2 new plates. </LI></UL></TD><TD><UL><LI>Checked the transformations that were redone - they were successful with the negative control plates showing no colonies. #2 and #3 are approximately the same level of competency</LI><LI>The agrobacteria containing B2 were infiltrated into the tobacco leaves.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>02/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Performed BsMB1 digest at 55 degrees on all parts PDF, PR, GST, P19 and Luc. Left for two hours with mixture at 55 degrees.</LI><LI>Performed ligation separately at 16 degrees for 1 hour 45mins. Negative control had the ligase buffer but no insert.</LI><LI>Plated these plates on level 0 Chl plates and grew overnight. -ve control shared between no insert and no ligase.</LI></UL></TD><TD><UL><LI> Plates from transformation </LI><LI> Plates from 01/08/17 failed to transform. Current thought is that 55 degrees is too hot for the ligation to occur. </LI><LI> Therefore, digest and ligation carried out separately when investigating TSH and TSHH gblocks. Digest took place at 55 degrees for 2 hours using buffer 3.1 (10x) BSA, insert (TSH/TSHH gblock), BSMBI, water and P6-18J plasmid. A negative control was also included. </LI><LI> Ligation at 16 degrees for 2 hours using T4 ligase buffer, cut vector and insert TSH from original digest and T4 ligase enzyme. T4 ligase enzyme was not incorporated into the negative control. </LI><LI> Products of the digest and ligation then transformed and left to incubate overnight. </LI><LI> pGEM plates: </LI><LI> Colonies grown from original TSH/TSHH pGEM plates grown in broth and left overnight. </LI></UL></TD><TD><UL><LI>Designed the luciferin experiment for tomorrow.</LI><LI>Calculated the relevant concentration of buffer required for tomorrow.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>03/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Analysed plates: colonies on every plate including both sides of the negative control. Much more on TSH plates than promoter plates - I did not resuspend the cells efficiently? </LI><LI>Picked 5 colonies from each plate for colony PCRs. Grown in LChl for 3 hours.</LI><LI>Made another IPTG stock. </LI><LI>Digested WRKY at 55 degrees, then ligated at 16 degrees. </LI><LI>Created LChl plates with IPTG, and 2x X-Gal. </LI><LI>Transformed cells with WRKY digest/ligation product and plated on LChl, IPTG, 2x X-Gal plates.</LI><LI>Ran Gel with PCR product from promoter colonies.</LI></UL></TD><TD><UL><LI> TSH/H gblock transformations from 02/08 displayed both white and blue colonies after being left in fridge to develop colour. 2 negative controls (1 no ligase/1 no insert), both showed white and blue colonies. White transformants especially apparent on TSH no ligase control. The no insert control showed fewer colonies probably due to inadequate re-suspension. </LI><LI> Colonies grown underwent colony PCR in order to extract possible insert DNA. </LI><LI> pGEM minipreps: </LI><LI> Minipreps of TSH/H pGEM grown overnight. </LI><LI> 1st minipreps were at 14 ug/nl and 13 ug/nl. </LI><LI> 2nd miniprep concentrations were 27.55 TSH and 62.650 TSHH </LI><LI> Sent TSH/H pGEM vectors off for sequencing. </LI></UL></TD><TD><UL><LI>Made the buffer for the luciferin experiment - mes 50mM, EDTA 1mM, glucose 0.5%, DMSO 0.5%</LI><LI>Pipetted the buffer into the relevant wells then added the leaf disc cuttings. Luciferin was then added at either 1 or 0.5 mM to the relevant wells. The plate was left in the dark for 2 hours before testing on the imager.</LI><LI>1mM luciferin in sodium citrate solution was infiltrated into a whole leaf (control, concentration A or concentration B) in order to image whole leaf samples.</LI><LI>Data analysis</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>04/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>WRKY plates have colonies on them. Unclear whether they are blue or white so left over the weekend in the fridge to develop - will perform colony PCRs on monday.</LI><LI>Re-performed digest and ligation of PDF, PR, GST, P19, Luc. Digest at 55 degrees, then denatured at 80 degrees, then ligated at 16 degrees. Transformed cells but did not have time to incubate and plate cells, so left them at room temperature to incubate over the weekend. Plating first thing on monday. </LI></UL></TD><TD><UL><LI> 3 white colonies on TSHH gblock plate and 1 on TSH gblocks. Colony PCR on these successful white colonies. Put fridge over the weekend. </LI><LI> Need to run a gel on monday and miniprep those that have the insert DNA </LI><LI> Also grew more colonies from original TSH/H gblock plates. Ready for miniprep on monday. </LI><LI> Digestion and ligation reactions using the biobrick plasmid and the TSH and TSHH gblocks. </LI><LI> Then transformations using the ligation reaction from earlier and plated out onto chloramphenicol plates. </LI><LI> Gel ran using the cut and uncut biobrick plasmid, there were no bands present. </LI></UL></TD><TD><UL><LI>Planned a repeat luciferin experiment for monday.</LI><LI>Performed a ligation digest of level 0 parts into a level 1 plasmid, which was left in the fridge over the weekend to transform on monday morning.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week five</H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>07/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Streaked the transformed cells from separate digest and ligation of PDF, PR, GST, P19 and Luc. Growing overnight at 37 degrees.</LI><LI>Picked 12 colonies from WRKY plate (possible whites) and grew in 1ml LB with 1ul Chl (wrong measurement - got confused with transformations!) - growing for 20hours to make up for the additional volume.</LI><LI>Spoke with Jamie: is it worth forgetting the colony PCRs and doing diagnostic digests instead? Also isolate just the uncut plasmid and use that for transformations by digesting and extracting the now linear plasmid? </LI></UL></TD><TD><UL><LI> None of the biobrick colonies from Friday grew. </LI><LI> Additional colonies grown from original TSH gblock plates miniprepped before PCR. </LI><LI> Previous colony PCR’s unsuccessful due to base insertions. . Alternative strategy to miniprep before PCR procedures no working so hoping minipreps will help the PCR. </LI></UL></TD><TD><UL><LI>Carried out the luciferin experiment again and carried out the data analysis of it.</LI><LI>Transformed the level 1 ligation mix that was left over the weekend in the fridge. The bacteria were spread over 5 agar plates (kan,IPTG,X-gal) to give a better chance of growth. These plates were left overnight in the 37.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>08/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Screened for blue/white colonies on PDF, PR, GST, P19 and Luc plates. 9, 5, 1, 1 and 0 white colonies, respectively. Picked these cells and grown in 5ml of Chl broth instead of colony PCR due to the low results.</LI><LI>Performed colony PCRs of the 12 white WRKY colonies. Used the same colony twice with 58 and 59 primers and also with the WRKY-specific primers.</LI><LI>Transformed WRKY cells and grew on Chl, X-gal, IPTG plates.</LI></UL></TD><TD><UL><LI> Gel produced from miniprep PCR from 7/08 promising. Regrew remaining colonies overnight (7th/8th). </LI><LI> Mini-prepped these colonies for PCR. </LI><LI> PCR performed on the mini preps and then a gel was run using the PCR products. </LI></UL></TD><TD><UL><LI>White colonies were observed over the 5 level 1 plates that were left to grow overnight (approximately 30 individual white colonies).</LI><LI>Set up a colony PCR however the primers used were not correct. So 10 colonies were inoculated to grow overnight.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>09/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Re-performed digestion and ligation for GST, P19, Luc, and WRKY. Transformed cells and grew overnight.</LI><LI>Miniprepped white colonies from PDF, PR, GST and P19 plates. Performed a PCR on these with primers 57 and 58 and analysed them on a gel.</LI></UL></TD><TD><UL><LI> London plans! Sarah and Helen </LI><LI> Presentation work </LI><LI> TSH6 and TSHH2: </LI><LI> Ran the gel from PCR reaction from 08/08. Both TSH6 and TSHH2 showed successful banding. </LI><LI> Set-up sequencing reactions for 10/08. Had to grow up more plasmid. These mini-prepped and sent off for sequencing tomorrow. </LI><LI> Digest: </LI><LI> Level one digest completed using BSA1, the digest included TSH/H, 35S or LexA and NosT. The digest was done at 37 degrees for 3 hours. </LI><LI> The enzymes were then heat inactivated at 80 degrees for 15 minutes. </LI><LI> T4 ligase was then added and the ligation reaction was done at 16 degrees overnight. </LI></UL></TD><TD><UL><LI>Mini-prepped 5 out of the 10 level 1 colonies grown overnight, resuspending them in 30 µl of Elution buffer.</LI><LI>These were then digested using BsmB1 before running them on a gel. This gel was very inconclusive, so this digest/gel will need to be redone.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>10/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Performed digest and ligation (overnight) with some Level 1s: 35S-P19-NosT, PDF-Luc+-NosT, and PR-Luc+-NosT </LI><LI>Screened for blue-white colonies on Level0 Luc, P19, GST, WRKY and -ve plates. Negative plate had blue and white colonies so inconclusive.</LI><LI>Picked more white colonies to miniprep from the plates of GST and P19 that we knew worked.</LI></UL></TD><TD><UL><LI> TSH6/TSHH2/TSHH 5:1 (A)/TSHH 5:1 (B) grew over night. </LI><LI> Minipreps on these. </LI><LI> Sent off sequencing for TSH6 and TSHH2. </LI><LI> Minipreps used for TSHH 5:1 A/B used in PCR reaction. </LI><LI> Additional minipreps of TSH6 and TSHH2 carried out as none left! </LI></UL></TD><TD><UL><LI>Performed digest and ligation (overnight) with some Level 1s: 35S-P19-NosT, PDF-Luc+-NosT, and PR-Luc+-NosT </LI><LI>Screened for blue-white colonies on Level0 Luc, P19, GST, WRKY and -ve plates. Negative plate had blue and white colonies so inconclusive.</LI><LI>Picked more white colonies to miniprep from the plates of GST and P19 that we knew worked.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>11/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Performed colony PCRs and analysed gels. Inconclusive results -  likely worked for P19 but not for GST -  hard to compare band sizes with LacZ as they are similar.</LI><LI>Sequenced PDF, PR, GST and P19 - GST ok, issues with the others.</LI></UL></TD><TD><UL><LI> Only those colonies miniprepped prior to PCR showed bands once the gel had been run. </LI><LI> Performed a digest on the biobrick containing the TSH to see if there were 2 bans on the gel, when we ran the gel there were no bands. </LI><LI> Level 1: </LI><LI> 3LexA/4LexA/135S/35s plates showed both white and blue colonies. These growing in broth and Kan before PCR/miniprep procedures. </LI><LI> Performed colony PCR on white colonies, no bands. </LI><LI> Unable to do the procedures as the bacteria have not grown.</LI></UL></TD><TD><UL><LI>Inoculated broth with chl and IPTG to grow the level 0 NOS-T, 35s and Luc+ over the weekend at 22 degrees in the shaking incubator.</LI><LI>Made more competent cells using #3 as a starting point.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week six</H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>14/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Grew up more of the lvl 0 p19 and GST colonies that were successful.</LI><LI>Set up sequencing reactions for the lvl 0 PDF and PR colonies.</LI></UL></TD><TD><UL><LI> Mini preps on all the colonies that grew over the weekend (level 1 reactions). </LI><LI> PCR on the level 1 mini preps using primers 44 and 56. </LI></UL></TD><TD><UL><LI>Mini prepped the level 0 35s, Luc+ and NOS-T.</LI><LI>Redid the digest of the level 1 (35s,xve,nos-t) and ran a gel of it. Again the resulting gel was not very conclusive, showing no correlation between any of the bands.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>15/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Miniprepped the level 0 p19 and GST colonies and measured DNA concentrations.</LI><LI>Set up sequencing again for lvl 0 p19 (GST was fine previously so no need).</LI><LI>Performed level 1 digest and ligation of GST and PR2. Ligation left overnight. Also performed on level 0 LucX5 and P19.</LI></UL></TD><TD><UL><LI> Miniprep of TSH6 and TSHH2. </LI><LI> BSA1 digest performed on the PGB-A1 plasmid at 37 degress for 3 hours, this was then run on a gel alongside the uncut plasmid. However, no bands showed. </LI><LI> Gel run on yesterday’s PCR products. Bands showed for both the LexA TSH and one of the LexA TSHH. but no bands showed for the 35S. </LI><LI> PCR performed on the biobrick mini preps using primers 57 and 58. </LI><LI> Picked more white colonies from the level1 35S plates and grew them up overnight in the shaking incubator at 37 degrees. </LI><LI> Regrew LexA colonies which had bands on the gel overnight. </LI><LI> Performed another PCR on the 35S mini preps which didn’t work on the previous gel </LI>.  
<LI>Emailed 2 lecturers in School of Journalism, who are members of the Science, Health and the Media Research Group to ask for their insight on how the media portrays GM. </LI><LI>Started emailing plant biotech companies to find out why they chose plants and the issues they face with production and release of plant pharmaceuticals. </LI><LI>Also emailed lecturers from School of Biosciences who have taught about plant pharmaceuticals, to hopefully provide some more information</LI></UL></TD><TD><UL><LI>Transformation to compare #4 vs. #3 of competent cells. Left to grow overnight at 37 degrees. </LI><LI>Transformed 35S-OTMV from the iGEM registry (plate 6-14D) as there have been problems with the 35S that has been used. So this is to act as a backup if the level 1 reactions do not work. Left to grow overnight. </LI><LI>Re-ran the level 1 digest/gel as was done yesterday but we reduced the digest time to 17 minutes. The bands obtained looked more promising but there is still no correlation between all the different samples tested (there should be as they're all supposedly the same level 1 construct).</LI><LI>Inoculated tubes 1,2,3,5 of the old level 1 cultures again (35s,xve,nos-t) to mini prep tomorrow. </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>16/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Performed level 1 Digestion-Ligations with PDF-Luc+-NosT, and GST with both TSH and TSHH.</LI><LI>Transformed cells with GST and PR2 level 1s and left to grow overnight. Transformed level 0 LucX5 and P19 and incubated too.</LI></UL></TD><TD><UL><LI> Ran a gel using the 35S PCR products from yesterday. </LI><LI> Mini preps done on the 35S colonies grown overnight. </LI><LI> Did another PCR using the biobricks, using primers 57 and 58. </LI><LI> Did a PCR using the new 35S colonies which had been mini prepped, using primers 44 and 56. </LI><LI> Ran a gel on the biobricks and 35S colonies. </LI><LI> Transformed and plated out the PGB-A1 plasmid again. </LI></UL></TD><TD><UL><LI>Sent off/ prepared 6 and 8 for sequencing, which are two of the level 1 constructs containing 35s, xve and nos-t.</LI><LI>Redid the #4 vs #3 and 35s transformations as the ones done yesterday did not work. </LI><LI>Also transformed the 35 s from last years iGEM registry plate. </LI><LI>Mini prepped the level 1 (35s,xve and nos-t) colonies 1,2,3 and 5. </LI><LI>Ran a digest/gel of 1,2,3,5. </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>17/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Level 1 cells had not grown enough so left again for another night.</LI><LI>Level0 p19 and LucX5 had white colonies, so they were picked and grown in broth overnight. </LI></UL></TD><TD><UL><LI>Checked plates from previous day: batch #4 of competent cells still grew on Cl plates, meaning they have developed Cl resistance (we’ve now thrown them away), and the 35S registry part plates failed to grow colonies</LI><LI>Re-did the level 1 reaction (cloning 35S, XVE and NosT into A1 plasmid): digested the plasmid, then added ligase - left at 16 degrees for 4 hours, then left at 4 degrees O/N</LI><LI>Started a culture of competent cells (from competent cell batch #3) to make competent cells up tomorrow</LI><LI>Made fresh stocks of everything needed to make competent cells because of contamination from last time (MgCl2, CaCl2, H2O, 100mM CaCl2 15% Glycerol, 800ml LB broth</LI><LI>Did a load of lab tech work - racked tips, washed glassware etc. </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>18/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>GST and PR2 level1s have clear white colonies. Picked and grown in broth to incubate over the weekend.</LI><LI>New WRKY primers so performing PCR on the genomic DNA. Ran on gel and isolated WRKY band (faint). Gel extracted DNA ready for level 0 reactions.</LI><LI>Performed minipreps on LucX5 and P19 level 0 white colony broths. Ran PCR on samples with primers 57 and 58 to identify the insert. Ran on gel.</LI></UL></TD><TD><UL><LI> Minipreps on 35S TSH/TSHH cultures </LI><LI> Sequencing of level 1 plasmids </LI><LI> PCR on minipreps (35S TSH/TSHH) using primers 44 and 56. </LI><LI> Gel on yesterday's PCR </LI><LI> Gel on the PCR using the mini preps from this morning. </LI><LI> Transformation of agrobacterium using the level 1 </LI><LI> Regrew the 35S colonies over the weekend and regrew TSHH 3+4 (2) </LI></UL></TD><TD><UL><LI>Inoculated 800ml LB broth with 5ml O/N culture of competent cells and put at 37 degrees for ~3 hours</LI><LI>Made more competent cells (batch #4.1), using #3 as a starting point again.</LI><LI>Transformed some competent cells (batch #3) with the level 1 ligation from the previous day (5ul of ligation mix into cells 50ul cells) and then plated on Kan (100mg/ml), IPTG (100mg/ml), X-Gal (20mg/ml). Leaving plates on bench (room temperature) to grow over the weekend</LI><LI>Split up some overgrown tobacco plants into more pots, and watered all plants</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week seven</H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>21/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Grew white colonies from lvl 1 GST+TSH+NOST, GST+TSHH+NOST, PDF+LUC+ + NOST, and -ve plate in 5ml of Kan LB.</LI><LI>Redid WRKY PCR in the hope of getting higher DNA conc from gel. </LI><LI>PCR’d blue colony mini prepped DNA from lvl 0 reactions which we forgot to do on friday.</LI><LI>Mini prepped PR2, GST lvl 1 white colonies for DNA</LI></UL></TD><TD><UL><LI> TSHH3/4 35S and TSH/H LEXA show at least two white colonies on each plate from transformation over the weekend </LI><LI> 2 separate PCRs on 35S TSH/TSHH miniprepped plasmids from 18/08 using primers 44/56 and 64/65 </LI><LI> Ran the gels on both of the PCRs </LI><LI> Streaked out the agrobacterium colonies on new plates and regrew them at 28 degrees </LI></UL></TD><TD><UL><LI>Transformation to compare #3 to #4.1 of competent cells. </LI><LI>Transformed more p6-18j from a previously prepared sample, as we were running out of it. </LI><LI>Checked level 1 transformations that were left to grow over the weekend - there was growth on the negative plate that looks very similar to that on the actual plates. There was also no blue colonies, so the plates were put in the fridge to see if they would turn blue. </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>22/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Performed PCRs with 2:30 extension time: 5 level 1 GST6 colonies with primers 64 and 41, and 7 PR2 colonies with primers 64 and 41 also. 7 PR2 level 1 colonies with primers 47 and 65. 5 GST level 1 colonies with primers 49 and 65. The level 0 p19 and LucX5 with primers 57 and 58. Level 1 GST-TSH-NosT, GST-TSHH-NosT, and PDF-Luc+-NosT, all with primers 64 and 65.</LI><LI>Tom carried out digest and ligation of WRKY DNA to get it into the level 0 plasmid. Ligating overnight.</LI><LI>Ran PCRs on gels for 20minutes at 120V, 400A.</LI></UL></TD><TD><UL><LI> Redid PCR using 44 and 56 primers for 35S plasmids as negative control showed a band. </LI><LI> 2 PCRs using primers 64+56 and 65+44 for 35s colonies and LEXA. </LI><LI> Gels from PCR showed differences in banding patterns, and the negative control had a band which could be due to contamination. So re-did this PCR using 64+56 and 65+44 primers again </LI></UL></TD><TD><UL><LI>Carried out level 0 reaction with bsmb1 of the g blocks (PR/PDF, PR/GST, GST/PDF, WRKY/PDF) and Ryan and Nialls WRKY. Left at 4 degrees overnight. </LI><LI>Checked the transformation to test the new cells. Batch #4.1 have no chl resistance. </LI><LI>Grew up liquid cultures of p6-18j to mini prep tomorrow. </LI><LI>Plates from previous day did not turn blue, going to re-do the transformation tomorrow alongside the level 0 reactions from today</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>23/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Sent PDF, PR +Luc+ and GST TSHH samples off for sequencing with 64 primer.</LI><LI>Set up lvl 1 digest + ligation for GST Luc+, PDF TSHH, PR TSHH + -ve. </LI><LI>Transformed agrobacteria with GST TSHH, PR Luc+ and PDF Luc+ and grew plates overnight. </LI></UL></TD><TD><UL><LI> Ran the gel using yesterday's PCR products </LI><LI> Human Outreach Research </LI></UL></TD><TD><UL><LI>Transformed 5 level 0 reactions from yesterday (4 new parts, 1 WRKY), as well as transforming the level 1 reaction (35S, XVE and NOS-T) from Friday and plated on Kan 100mg/ml, X-Gal 20mg/ml, IPTG 100mg/ml (level 0) and Chloramphenicol 34mg/ml, X-Gal 20mg/ml, IPTG 100mg/ml (level 1)</LI><LI>Organised all of the plants: moved some sick ones upstairs to get better light, got rid of some pale ones, grouped ones of similar size and watered everything</LI><LI>Miniprepped 6 lots of 18J cultures from the previous day as it had all been used</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>24/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Transformed competent cells with lvl1 PDF TSHH, PR2 TSHH, GST LUC+ and -ve then plated on Kan plates.</LI><LI>Dig +Lig with WRKY DNA into lvl 0 plasmid (18J) </LI></UL></TD><TD><UL><LI> Grown agrobacteria containing pGB:35s-TSHH-NosT and pGB:LexA-TSHH-NosT and 35S:LUC+ overnight </LI><LI>Phone call with Phillip Cater and Nicholas Holton from Leaf Systems International. Spoke about the issues with GM pharmaceuticals and the issues of regulations.</LI></UL></TD><TD><UL><LI>Looked at the plates from the previous day: the plates grown on Kanamycin had tiny colonies, so they were put back in the incubator, whilst the plates grown on Chloramphenicol had only 2 white colonies on WRKY/PDF and 1 white colony on PR/PDF - these white colonies were inoculated to grow O/N, and also picked into water to be used for colony PCR (primers 57 and 58). The rest of the level 0 plates (on Chloramphenicol) were put in the fridge in the hope that more white colonies might be seen.</LI><LI>Colony PCR: picked colonies into 20ul water, denatured at 95 degrees for 5 minutes, then took 2ul forward into a PCR reaction (94 degrees 20 seconds, 56 degrees 20 seconds, 72 degrees 2 minutes). </LI><LI>Ran a gel of the PCR reaction </LI><LI>Made buffers for tomorrow to be used in Agrobacterium transformation.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>25/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Picked level 1 constructs in Agrobacterium and streaked onto new plates to grow over the weekend at 28 degrees.</LI><LI>Transformed WRKY level 0 cells again and plated them on Chl plates. Growing at 22 degrees over the weekend.</LI></UL></TD><TD><UL><LI> 35S:LUC+, 35S:TSHH3/4 failed to grow sufficiently. LEXA: TSHH was cloudy. </LI><LI> Agrobacteria grown overnight resuspended in infiltration buffer (MES, MgCl2, acetosyringone) to OD600:0.5. These were left to incubate at room temperature for 2hr </LI><LI> Agrobacteria was then infiltrated into tobacco leaves </LI></UL></TD><TD><UL><LI>Watered all plants</LI><LI>3 white colony cultures from yesterday were miniprepped by Sarah</LI><LI>The level 1 Kan plates from yesterday that were put back in the 37 degree cabinet now have 51 white colonies spread across them - 10 of them were picked into cultures and left at 18 degrees to grow over the weekend </LI><LI>10 tobacco plant leaves were infiltrated with LexA:TSHH Agrobacterium (5 leaves each with two cultures that should be identical)</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week eight</H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>29/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Discovered PDF, PR and GST ‘Luc+’ is actually TSHH so every ‘Luc+’ level 1 before the 29/08 is TSHH </LI><LI>Did lvl 1 DIG LIG for PDF luc+ Nost, PR2 luc+ Nost and GST luc+ nost </LI><LI>Performed a transformation of e.coli with LUC+ to increase luc+ plasmid quantity </LI><LI>Grew agrobacterium up in broth with Rif and Kan. Agrobacterium consisted of colonies with lvl1 plasmids for GST TSHH nosT, PDF TSHH nosT and PR TSHH nosT. </LI><LI>Picked the WRKY level 0 colonies and grew in broth overnight at 37 degrees. </LI><LI>Performed PCR diagnostic on Luc+ samples with Luc+ specific primers and another set with TSH specific primers - Bands present with both, so samples are contaminated and cannot be used. </LI></UL></TD><TD><UL><LI> Leaves infiltrated on 25/08 frozen using liquid N2 before grinding of the leaves on ice </LI><LI> Buffer EZ was added to the extracts before centrifugation at 13,000 rpm. The supernatant was removed from each sample, 40ul of this was taken as the total sample and 10ul Z buffer was added </LI><LI> The rest of the supernatant was used in the protein purification process using the nickel beads. </LI></UL></TD><TD><UL><LI>Set up a PCR reaction with primers 57 and 58 on the mini-prepped DNA of WRKY/PDF 1 &amp; 2 and PR/PDF 1</LI><LI>Practised pouring protein gels with water and it worked fine. Then moved on to practising with resolving gel and the gel would appear fine but slowly pour out. Managed to get one of the gels working after a while and added the stacking gel too. Put in the fridge to be used tomorrow.</LI><LI>Ran a gel of the PCR from this morning… each of the constructs have bands of the right size, so they’re going to be sent for sequencing tomorrow.</LI><LI>Inoculated two cultures of B2 in the hope that they’ll grow this time to be used for infiltration.</LI><LI>Mini-prepped 6 of the 10 level 1 cultures from Friday to be digested tomorrow. </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>30/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Resequenced PDF(1)/PR(5)/GST(2) with ‘Luc+’ (which is probably TSHH) and 64 +65 primers.</LI><LI>Picked colonies of Luc+ lvl 0 plasmids grown on spec yesterday and grew in liquid culture containing spec. </LI><LI>Transformed lvl 1 PDF/PR/GST with actual Luc+ and nosT and plated on Kan, IPTG and x-gal until the 1/09/17. </LI><LI>Infiltrated tobacco with agrobacterium containing PDF/PR/GST with ‘luc+’ (TSHH probably) </LI><LI>Miniprepped level 0 WRKY DNA samples and ran a PCR on them. Couldn’t run a gel because the infiltration took too long, so running tomorrow. </LI></UL></TD><TD><UL><LI> Samples from 29/08 run using the PAGE gel. Samples transferred to 0.5ml eppendorf tube and boiled for 10 mins in PCR machine </LI><LI> Supernatant cooled before loading of the protein gel. 25 ul of NI total, NI elute, NI beads, LexA total, LexA Elute1 and LexA Beads was loaded into the wells of the gel and left to run at 170V until buffer line reaches bottom of the gel </LI><LI> The gel was added to 1 x Coomassie Stain and left for 30 minutes. The gel was left to destain overnight </LI><LI> Level 1: </LI><LI> Minipreps on white colony 1 and 2, TSH1, TSH2, TSHH1 (1), TSHH1 (2), TSHH2 (1), TSHH3 (1) and TSHH4 (1) </LI><LI> These minipreps underwent a PCR reaction for 2.5 hours. Water was used as a negative control and LexA was used as a positive control.  </LI></UL></TD><TD><UL><LI>Sent off WRKY/PDF and PR/PDF for sequencing with primers 57 and 58</LI><LI>Watered all plants</LI><LI>Cleared away all autoclave waste and tidied the lab</LI><LI>Made some running buffer and helped Emily set up the protein gel. Wrapped the other protein gel in foil and left in the fridge.</LI><LI>Made up more buffer to resuspend Agrobacterium</LI><LI>Only 1 of Agrobacterium cultures grew, so that one was resuspended in buffer to an OD600 of 0.281</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>31/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Level 1 promoter-luc+ plates not grown properly so left to grow overnight.</LI><LI>Ran gel on miniprepped WRKY level 0 DNA but could not perform a level 1 digest due to lack of Bsa1.</LI><LI>Ran WRKY samples on gel twice, with different water samples and identified one was contaminated (the cause of the luc+ being TSHH). Four samples had level 0 WRKY in them! </LI></UL></TD><TD><UL><LI> Minipreps on TSH6/TSHH2 </LI><LI> Re-ran gels from yesterdays PCR products. No visible bands for any 35S TSH/TSHH level 1 constructs using 44+ 65 Primers </LI><LI> New PCRs using a) 64+65 b)44 + 65 c) 37+64 </LI><LI> Safety forms part 4+5 </LI><LI> Getting together protocol for Wiki Page </LI><LI> The protein gel from yesterday hadn't run properly so when the gel was destained there were no bands </LI></UL></TD><TD><UL><LI>Separated tobacco plants that were overcrowded</LI><LI>Ran PCR on the level 1 (35S, Nos-T, XVE) using primers 64 and 65</LI><LI>Wrky/PDF and PR/PDF were cloned into level 1 reactions with Luc+ and Nos-T</LI><LI>Ran a gel of the PCR, but ladder isn’t very visible so it needs to be run again</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>01/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Picked level 1 colonies from GST, PR and PDF (supposed) level 1 plates, but the negative plate had some white colonies so the water we used was indeed contaminated. Picked and grew all white colonies to test them. If they are TSHH also then we will have to redo luc+ level 1s again!</LI><LI>Infiltrated the leaves with salicylic acid and/or ethanol at 5% each. </LI><LI>Emily set up PCR to isolate luc+ part with primers 22 and 23. This will isolate the luc+ from the spectinomycin plasmid to put into the pSB1C3g. </LI><LI>Ran Luc+ samples on gel and extracted the DNA.</LI><LI>Measured Luc+ DNA concentrations and set up digest for 30 minutes only due to time with 15minutes to denature. Ligated over the weekend (16 degrees for 4 hours and final hold 4 degrees). </LI><LI>Cut off infiltrated leaves and flash froze them. Stored at -80 degrees over the weekend. </LI></UL></TD><TD><UL><LI>Results of gel run on 31/08 </LI><LI> 64+65 = bands at 1800bp, however white colonies show bands at the same positions as the 35s TSH/TSHH minipreps. Also bands at 1000 bp </LI><LI> 64+37 = bands visible even when 37 is specific to LexA promoter </LI><LI> 44+65 = bands visible but same length as 64+65 </LI><LI> Protocols write up </LI><LI> Transform 35S from the registry- grow it over the weekend. </LI></UL></TD><TD><UL><LI>Transformed the level one digest/ligation from yesterday</LI><LI>Re-ran the gel from yesterday, bands were all the same as the previous gel: -ve produced no bands, as did sample 9… but samples 5, 6, 7, 8 and 10 all have bands</LI><LI>Helped with infiltration</LI><LI>Met Jamie to image the LUC assay, but it didn’t show up any results, possibly because the luciferin was left for 3hrs, or because the concentration of Agrobacterium infiltrated into the plants was too low</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week nine</H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>04/08/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Flash froze PR-TSHH and GST-TSHH infiltrated leaves with and without Salicylic acid and ground them up. Prepared them for a protein extraction. Isolated 5 samples each of 40ul: Total, elution 1, elution 2, elution 3, and a sample with beads in. Froze overnight for protein extract tomorrow.</LI><LI>Sequenced GST TSHH (‘Luc+’) with TSHH forward promoter </LI><LI>Mini prepped colonies from lvl 1 PDF/PR/GST: luc+ : NosT (3 from each) and PCR’d - will run on a gel tomorrow morning </LI><LI>DIG/LIG for WRKY luc+, WRKY TSHH and -ve </LI><LI>Took colonies off agro plates with PDF:’luc+’ (TSHH) and grew in liquid broth with rif and kan overnight in 28 degrees shaker </LI></UL></TD><TD><UL><LI>No 35S colonies grew on the plates. </LI><LI> Prepare glycerol stocks for pSB1C3-TSH(H) by growing an overnight culture. </LI><LI>Sequencing reactions for pSB1C3 - TSH (H) using 57 and 58 primers.</LI><LI>Protein purification using the frozen leaf samples from last week, using the nickel beads. </LI></UL></TD><TD><UL><LI>Did a level 0 digest/ligation of the composite promoters (PR/PDF, PR/GST, GST/PDF)</LI><LI>Poured new PAGE gels</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>05/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Prepared PDF-TSHH agrobacterium samples to 0.5 OD and incubated at room temperature for 2 hours. </LI><LI>Ran a protein gel with GST, PR (SA and NSA) and controls using total, elution1 and bead samples.</LI><LI>Stained the protein gel overnight.</LI><LI>Infiltrated plants with PDF1.2-TSHH level 1 constructs.</LI><LI>Ran gel from the lvl 1 PDF/PR/GST:Luc+:Nost PCR (64+65) - presence of luc+ unsure. Redid PCR with 64+41 and 65+42 gel in lab book.</LI><LI>Transformed E.coli with WRKY:Luc+ and WRKY:TSHH + -ve ~and plated on agar with Kan IPTG and X-gal.</LI></UL></TD><TD><UL><LI>Preparation of the glycerol stock for Level 0 pSBIC3 TSH6,  pSBIC3 TSHH2, Level 1 pGBA2:LexA-TSH and pGBA2:LexA-TSHH </LI><LI>Growing up glycerol stock made by Rob using 5 ml LB, 5 ul Chl, 5 ul glycerol stock. Left to incubate overnight. </LI><LI>Miniprep of TSH6 and TSHH2 level 0 plasmid. </LI><LI>Work on our story for the wiki, open day stuff, and UK meet up stuff </LI><LI>Ran a protein gel using the purified samples from yesterday and stained the gel overnight.</LI></UL></TD><TD><UL><LI>Transformed the level 0 parts from yesterday</LI><LI>Poured lots of Cl and Kan plates</LI><LI>Assisted with running the PAGE gels</LI><LI>Designed a banner for the iGEM team</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>06/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Analysed protein gels - no strong bands and RUBISCO band is weak due to dying leaves.</LI><LI>Miniprepped GST4, 5, 6, 7 and 8 Luc+level 1 colonies as they did not work on the gel. </LI><LI>Ran PCR with GST-Luc+level 1 colonies with primers 64 and 65, 41 and 64, and 42 and 65. </LI><LI>Prepared agrobacterium samples with GST samples 1 and 2, and PR samples 1 and 2 with TSHH for infiltration with healthier plants. </LI><LI>Infiltrated plants with GST and PR TSHH constructs ready for salicylic acid infiltration in 2 days.</LI><LI>Reran PCR for PDF/PR:Luc+ with different primers (64+41, 47_41, 45+41, 42+41, 39+65) ran gel.</LI><LI>Transformed agrobacterium with PDF(3)/PR(2):Luc+ and left to grow for 2 days</LI></UL></TD><TD><UL><LI>Preparing work for the wiki </LI><LI>Destained the protein gel- no obvious TSHH bands </LI></UL></TD><TD><UL><LI>Looked at level 0 composite promoter plates and there were some colonies that looked possibly positive… put them in the fridge </LI>for a few hours to develop the colonies and noticed 4 that looked positive
<LI>Set up a colony PCR with primers 57 and 58 on each of the 4 colonies and 2 negative controls </LI><LI>Inoculated each of the 4 colonies to grow O/N</LI><LI>Digested and ligated each of the 3 composite promoters again into a level 0 in case the PCR doesn’t work </LI><LI>Worked on the banner</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>07/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI>Sent PDF/PR:Luc+ off for sequencing</LI><LI>Grew up white WRKY:TSHH/LUC+ in Kan broth overnight</LI></UL></TD><TD><UL><LI>performed a level one digest using the new 35s constructs with TSHH and NosT </LI><LI>Ligation of the digestion mixes for 2 hours at 16C </LI><LI>Transformations using the level 1 digestion/ligation mixtures- then calls plated onto kan, IPTG and x-gal plates </LI><LI>Transformations using the new 35S constructs- then cells plated out onto spec plates </LI><LI>Miniprep of LUC+ colonies grown overnight at 37 degrees in LB and chl. </LI><LI>PCR reaction using primers 57 and 58. Gel run showed inconclusive bands. </LI><LI>Re-did the LUC+ PCR using primers 25 and 26 which are specific to LUC+. Left overnight at 16 degrees. </LI></UL></TD><TD><UL><LI>Ran a gel of the PCR from yesterday, but there were no obvious bands (which we expected because we didn’t denature the colony at 95 degrees for 5 minutes). </LI><LI>Set up another PCR the same as yesterday (used the same DNA but this time we denatured it at 95 degrees for 5 minutes) </LI><LI>Transformed the level 0 composite promoter ligation mixes into competent cells and plated then left at 37 degrees to grow O/N</LI><LI>Mini-prepped the 6 cultures from yesterday</LI><LI>Ran a gel of the second colony PCR</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>08/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> DIG/LIG of lvl 1 PDF/PR/GST:Luc+:nosT and transformed e.coli + plated on Kan + IPTG + x-gal to grow over the weekend. </LI><LI>Sent PDF/PR/GST:TSHH:nost for sequencing with TSHH primer 55</LI><LI>WRKY lvl 1 TSHH/Luc+ white colonies mini prepped and PCR’d with 64+65 - Ran a gel</LI><LI>Infiltrated plants with SA for PR/GST:TSHH:nosT</LI><LI>PDF/PR: luc+ transformed agro did not grow :( </LI><LI>Activated PDF:TSHH (again - waiting for JA) and infiltrated into plants </LI><LI>Made 1ml agro glycerol stock (30%) for PDF:TSHH:nosT </LI></UL></TD><TD><UL><LI>Ran a gel from LUC+ PCR reaction from 07/09. Band at 1.5 kb on LUC+ 2 (1) culture </LI><LI>However, the initial PCR using 22 and 23 primers did not work. New PCR completed using primers 25 and 26 and SPEC luc + level 0 Plasmid to be ready before gel extraction on monday </LI><LI>No colonies grew on the level 1 plates using the new 35S, so transformed cells using yesterday's ligation mixes again.</LI><LI>Protocols and health and safety for the wiki </LI></UL></TD><TD><UL><LI>Checked plates from previous day: PR/PDF plates had 3 white colonies, PR/GST had 2 white colonies. Colonies were picked (along with one blue colony from each as a control) and used for colony PCR with primers 57 and 58</LI><LI>Ran a gel of the colony PCR - PR/PDF (3) and PR/GST (5) looked promising on the gel, so we’re going to miniprep the 5 white colonies on Monday and run a PCR of them again</LI><LI>Made liquid cultures of each of the 5 positive colonies</LI><LI>Made 4 protein gels to be used next week</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week ten </H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>11/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH></TR><TR><TD><UL><LI> Picked white colonies from lvl 1 PDF/PR/GST:Luc+ plates and grew in broth overnight </LI><LI> WRKY PCR with WRKY forward and reverse TSHH/LUC+ primers and ran gel. </LI><LI> prepared leaves from PR/GST infiltrated plants for protein gel extraction - got part way and froze to continue next day </LI></UL></TD><TD><UL><LI> Ran a gel for LUC+ SPEC level 0 from PCR reaction using 25 + 26 primers from 08/09. Bands apparent at 3kb (whole spec level 0 plasmid) </LI><LI> From this gel, use the Gel extraction protocol to extract the LUC+ DNA. </LI><LI> A digest reaction was carried out using BsmbI, BSA, H20, level 0 pSBIC3, LUC+ NEB buffer 3.1. The digest reaction was left for 1.5 hours at 55 degrees. Following this, denaturation of the digest solution took place at 80 degrees for 15 min before inserting 2 ul T4 DNA ligase Buffer and 1 ul T4 DNA ligase for the ligation reaction to proceed at 16 degrees for 2 hours.  </LI><LI> safety forms </LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>12/09/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH></TR><TR><TD><UL><LI> mini prepped PDF/PR/GST Luc+ lvl 1 colonies and PCR’d with forward primers for each promoter (45/47/49 respectively) and Luc+ reverse primer. </LI><LI> finished preparing leaf samples from yesterday. </LI><LI> grew up agro containing PDF:TSHH ready to infiltrate on thursday (hopefully JA will arrive). </LI><LI> Ran another PCR and gel with WRKY Luc+ samples and TSHH samples with primer 66 (WRKY forward) and primers 41 and 56 (Luc+ reverse and TSHH reverse, respectively). The previous one was not clear. </LI></UL></TD><TD><UL><LI> Transformation of LUC+ pSBIC3 </LI><LI> minipreps of 35s constructs </LI><LI> safety information for the wiki </LI><LI> prepared glycerol stocks of pICH51277 short 35s plus TMV spec, pICH51288 35s double promoter spec, pICH41373 35s prom spec, pICSL13001 Pro_5Uf 35s long, pICH51266 35S Long promoter spec. </LI>. 

</UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>Post-10 week lab work</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Infiltrated plants with PDF1.2:TSHH constructs</LI><LI> Induced PDF:TSHH plants with Jasmonic acid</LI><LI> Cut leaves off appropriate plants and carried out a protein extract. Ran the protein gel - no TSHH bands present.</LI><LI> Modelled the theoretical biofactory</LI><LI> Extensive Wiki editting</LI></UL></TD><TD><UL><LI> Carried out the Exeter collaboration</LI></UL></TD><TD><UL><LI>Looked after plants</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>