<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_projectpractice skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/projectpractice</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A href="https://2017.igem.org/Team:Cardiff_Wales">Home</A><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/meet_the_team">Meet the Team</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_TSH">Team TSH</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_Luc">Team Luciferase</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_PlantP">Team Plant Promoter</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/projectdescription">Project Description</A><A href="https://2017.igem.org/Team:Cardiff_Wales/results">Results</A><A href="https://2017.igem.org/Team:Cardiff_Wales/basicparts">Basic Parts</A><A href="https://2017.igem.org/Team:Cardiff_Wales/compositeparts">Composite Parts</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Human Practice</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/Human_Practices">Integrated Human Practices</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Public_outreach">Public Outreach</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Survey">Survey</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Our_research">Our HP Research</A></DIV></DIV><DIV class="container"><A href="https://2017.igem.org/Team:Cardiff_Wales/Modelling">Modelling</A><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Lab book</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/protocols">Protocols</A><A href="https://2017.igem.org/Team:Cardiff_Wales/diary">Diary</A><A href="https://2017.igem.org/Team:Cardiff_Wales/health_and_safety">Health and Safety</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Medal Criteria</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/goldfulfilment">Gold</A><A href="https://2017.igem.org/Team:Cardiff_Wales/silverfulfilment">Silver</A><A href="https://2017.igem.org/Team:Cardiff_Wales/bronzefulfilment">Bronze</A></DIV></DIV><DIV class="igem_2017_content_wrapper"><CENTER><H1> Project Diary </H1></CENTER><LEFT><H3> Week one </H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>10/07/2017</H4></SPAN><DIV class="dropdown1-content"><UL><LI>Meet up</LI><LI>Safety and risk assessment forms</LI><LI>Review to do lists</LI><LI>Helen = safety</LI><LI>Sequencing = Niall</LI><LI>Plants = Jade and Sarah</LI><LI> Stockroom = Emily, Ryan, and Thomas </LI><LI> Arranged lab and workspaces </LI><LI> Sorted pipette tips into boxes and autoclaved. </LI><LI> Made medium using 25 g/l LB broth. Made medium using 10g/L agar granulated (made up to 400ml water). Then autoclaved. Pyrex class will not smash, cools down quickly/till hand-hot.</LI><LI> Chloramphenicol + kanamycin resistance antibiotics: One medium had 100mg/ml kanamycin added (400microlitres to 400ml), the other had 34mg/ml chloramphenicol added (400microlitres to 400ml). Plated 19 plates of each. </LI><LI> Transferred tobacco plants (grown on 19/6) into new pots to prevent overcrowding.</LI><LI> Organised ourselves into 3 'teams':</LI><LI> Team_PlantP = Niall and Ryan </LI><LI> Team_TSH = Helen, Emily, and Sarah </LI><LI> Team_Luc = Jade and Tom </LI></UL></DIV></DIV><DIV class="dropdown1"><SPAN><H4>11/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Re-suspended iGEM primers 45 - 52 at 100uM. Created working stock at 10uM. </LI><LI> Used Primers in a PCR with Arabidopsis thaliana gDNA to amplify 4 plant promoters (with -ve controls)</LI><LI> Ran PCR results on 1% agarose gel with 1x TAE. No success at amplifying desired sequences (empty gel lanes).</LI></UL></TD><TD><UL><LI> Resuspend TSH gBlocks to concentration 0.1pmol/ul (v.clean water). </LI><LI> Digest TSHantag and TSHantag(his) and introduced into pSB1C3 (plasmid) with EcoR1 and Pst1. </LI><LI> Ligation mix preparation, left to ligate overnight. </LI><LI> Re-do LB broth and agar plates as we used broth in the agar plates! </LI><LI> Planted 16 more pots of tobacco plants.</LI><LI> Helen doing safety forms and biosafety poster exercise </LI></UL></TD><TD><UL><LI> Made up agar plates inoculated with either kanamycin, chloramphenicol or chloramphenicol, X-gal and IPTG.</LI><LI> Transform DNA from iGEM registry - Did not work as the cells used were bad.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>12/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Redid failed PCR from yesterday and ran on a gel - slightly more success (Primer dimers (probably) in 2 lanes). </LI><LI> Niall carried out the first part of 2 SOE-PCRs to convert target nucleotides for BsmBI digestion into non-target nucleotides through site directed mutagenesis. SOE-PCRs used to modify the P19 gene and LUCx5 gene.</LI><LI> Ran on gel - gDNA was poor so no bands. </LI><LI> Ryan: Performed plasmid minipreps according to QIAprep® Spin Miniprep Kit. Measured the concentrations of each of the DNA samples.</LI></UL></TD><TD><UL><LI> Digest and ligation using the P5B1C3g plasmid for golden gate cloning, using both TSHantag and TSHantag-his. </LI><LI> Ligation reaction using PCR machine (15 cycles). </LI><LI> Transformation of biobrick and golden gate plasmids (ligated 11/7 and 12/7) into competent cells. Grown on chloramphenicol for biobrick, and chloramphenicol + Xgal + IPTG for golden gate. </LI><LI> Procedure of transformation: ice (10’), heat shock in water bath at 42degrees (1’), ice (5’), add 1 ml LB broth without antibiotic, shake (1hr at 37 degrees), centrifuge for 1 minute at 13k,  plate using resuspension and glass beads. </LI><LI> Incubation of plates at 37 degrees overnight. </LI></UL></TD><TD><UL><LI> Mini Prep of level 0 parts</LI><LI> Transformed DNA from iGEM registry again</LI><LI> Made competent E.coli, however the water used had not been autoclaved.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>13/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Extracted DNA from gel to get p19 and LucX5 DNA. </LI><LI>Quantified DNA and stored in freezer for future use in plasmids.</LI><LI> Extracted DNA from Arabidopsis thaliana.</LI><LI>Quantified DNA. </LI><LI> Took two best samples and ran PCR with plant promoter primers (IGEM_45 to IGEM52). </LI></UL></TD><TD><UL><LI> Re-do transformations as broth was not added to agar. </LI><LI> Steps: ice, heat shock, ice, shake for 1 hour, plates. </LI><LI> Did surveys for iGEM toulouse about cholera. </LI></UL></TD><TD><UL><LI> Transformed DNA from iGEM registry again as well as one of the level 0 parts mini prepped the day prior (as a test) into competent cells.</LI><LI> Made up agar plates inoculated with either kanamycin, chloramphenicol or chloramphenicol, X-gal and IPTG.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>14/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Ran gels with PDF1p, PR2p, GST6p, and WRKY30p, with gDNAs 2 and 3 (highest DNA concentration samples). All lanes were successful with no contamination, except WRKY which had no result except primer dimers.</LI><LI> Cut and digested the gels and purified DNAs - 3 test tubes each with either PDF, PR or GST promoter regions.</LI><LI> Measured concentrations of the promoter DNA.</LI><LI> Set up PCR of WRKY30p at different temperatures (gradient in the PCR machine). Temperatures 50, 53, 57, and 60 degrees.
 </LI><LI> Ran gels with 8 lanes (4 negative controls, and 4 with one of each of the WRKY temperatures)</LI><LI> All 4 samples had primer dimers but no amplified gDNA - conclusion = most likely primers are not specific enough to amplify gDNA</LI></UL></TD><TD><UL><LI> TSH 5G/TSH10g plates grew colonies. </LI><LI> Redo TSH/H 3:1, TSH/H 5g, TSH 5:1 and biobrick controls for transformations.</LI><LI> Incubation over night</LI></UL></TD><TD><UL><LI> Made up agar plates inoculated with either chloramphenicol or chloramphenicol, X-gal and IPTG. Reducing the concentration of chloramphenicol from 34 to 17, and doubling the concentration of X-gal.</LI><LI> Transformed DNA from iGEM registry again into competent cells.</LI><LI> Made a protein gel plate to be used in a future test.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER><LEFT><H3> Week two </H3></LEFT><CENTER><DIV class="dropdown1"><SPAN><H4>17/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Ran PCRs with WRKY at 52 and 56 degrees for primer annealing. With half and double primer concentration in each, to see if either would prevent primer dimers and allow annealing to DNA.</LI><LI> Ran gel and found only primer dimers in every sample. gDNA is fine because the positive control had its band. </LI><LI> WRKY primer is ineffective, so we must design another.</LI></UL></TD><TD><UL><LI> Plates again did not work. Put them in the incubator at 37 degrees. </LI><LI> Possible problems with chloramphenicol</LI><LI> Running PCR of TSH 5G and TSH10G colonies.  </LI><LI> Made gels, on 120 for 50 minutes and they did not work so re-doing them!</LI><LI> Re-ran gels, have better results</LI><LI> Helen: from incubated plates 1 was successful (TSH 5:1). Took colonies from previous successful plates using 5 ml broth. Left to incubate and shake overnight at 37 degrees. </LI></UL></TD><TD><UL><LI> Transformed the universal promoter (plate 1 - 3O) from the iGEM registry as well as GFP made in the mini prep (12/07/17) and a negative control to test the competent cells.</LI><LI> Inoculated agrobacterium and E.coli cultures.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>18/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Designed new WRKY primer</LI><LI> Can’t do anything else until level0 plasmids are done to perform minipreps.</LI></UL></TD><TD><UL><LI> Performed a mini prep on the one colony that grew successfully (QLAprep Spin Miniprep Kit).</LI><LI> Performed PCR on the mini prep, GFP plasmid, and level 0 colonies </LI><LI> Ran a gel using the PCR products and the hyperladder 4 </LI><LI> Helen: Grew bacterial cultures using chloramphenicol and kanamycin antibiotics. 5ml LB broth and 5 ul of antibiotics. Left overnight on 200 RPM at 37 degrees. </LI><LI> Grew colonies from plate A (4), B (1), C (4), Rob’s colonies. </LI></UL></TD><TD><UL><LI> Re-incubated the agrobacterium overnight as they were not spinning so could not grow.</LI><LI> Made some more competent E.coli using previously made competent E.coli as a starting point.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="dropdown1"><SPAN><H4>19/07/2017</H4></SPAN><DIV class="dropdown1-content"><TABLE><TBODY><TR><TH>Team_PlantP</TH><TH>Team_TSH</TH><TH>Team_Luc</TH></TR><TR><TD><UL><LI> Performed minipreps on samples A1 and A2, C1 and C4. Isolated the DNA and measured concentrations of samples A1-A4 and C1-C4 against EB as a blank.</LI><LI> Performed restriction digest on P19, Luc, PDF1, PR2 and GST6, according to long protocol in ligase buffer. Put in PCR machine for the process.</LI><LI> Performed ligation and gel analysis to see whether the plasid cut and incorporated the recombinant DNA.</LI></UL></TD><TD><UL><LI> Minipreps on plates A (level 0) and C (level 1). </LI><LI> Digest process and then used PCR using long protocol.</LI><LI> Tested level 0 plasmids.</LI><LI> Also completed 2 minipreps of GBA2 using QIAprep protocol</LI><LI> Did a digest of the plasmid in the water bath at 37 degrees using BsmB1 </LI><LI> Ran a gel comparing the undigested plasmid and the ones digested with BsmB1 </LI></UL></TD><TD><UL><LI> Ran a transformation to compare the newly made competent cells with the competent cells made on 12/07/17.</LI><LI> The agrobacterium that were grown overnight were used to infiltrate the leaves of tobacco (to be harvested in two days time), by suspending the cells in induction buffer.</LI></UL></TD></TR></TBODY></TABLE></DIV></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>