<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_Modelling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/Modelling</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><A href="https://2017.igem.org/Team:Cardiff_Wales">Home</A><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/meet_the_team">Meet the Team</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_TSH">Team TSH</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_Luc">Team Luciferase</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Team_PlantP">Team Plant Promoter</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/projectdescription">Project Description</A><A href="https://2017.igem.org/Team:Cardiff_Wales/results">Results</A><A href="https://2017.igem.org/Team:Cardiff_Wales/basicparts">Basic Parts</A><A href="https://2017.igem.org/Team:Cardiff_Wales/compositeparts">Composite Parts</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Human Practice</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/Human_Practices">Integrated Human Practices</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Public_outreach">Public Outreach</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Survey">Survey</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Cardiff_Wales/Our_research">Our HP Research</A></DIV></DIV><DIV class="container"><A href="https://2017.igem.org/Team:Cardiff_Wales/Modelling">Modelling</A><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Lab book</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/protocols">Protocols</A><A href="https://2017.igem.org/Team:Cardiff_Wales/diary">Diary</A><A href="https://2017.igem.org/Team:Cardiff_Wales/health_and_safety">Health and Safety</A></DIV></DIV><DIV class="container"><DIV class="dropdown"><BUTTON class="dropbtn">Medal Criteria</BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Cardiff_Wales/goldfulfilment">Gold</A><A href="https://2017.igem.org/Team:Cardiff_Wales/silverfulfilment">Silver</A><A href="https://2017.igem.org/Team:Cardiff_Wales/bronzefulfilment">Bronze</A></DIV></DIV><DIV class="igem_2017_content_wrapper"><H1> Modelling for a biofactory </H1><DIV class="column_full_size" style="background-color:#ffffff;"><P style="background-color:#ffffff;">
As part of our project, we decided that it would be interesting to see how many of plants it would require to give a single treatment to a person with severe Graves disease/ophthalmopathy, and how many to treat a less-severely affected patient. Initially we attempted to model this using data from <I> N. benthamiana </I> from various scientific papers that have used this as an expression system. In doing so, we created a simple equation that theoretically can be applied to any plant biofactory company, that ultimately calculates how many plants would be needed to create a single effective dose of medicine for a person with a disease/condition of interest. One strength of this equation is that any one or all of the variables can be substituted to fit a new purpose, so that future teams/companies may be able to use this model and change or improve it, to estimate how many plants they would need to create a significant amount of product. To demonstrate this flexibility, we changed some of the variables ourselves and plotted some of the results gained from modelling using <A href="https://uk.mathworks.com/products/matlab.html">MatLab.</A>The variables we changed showed the potential production using different plant vectors/systems, specifically using <I> N. tabacum </I> cultivars TI-95, and I-64, and the newer <A href="http://www.pbltechnology.com/molecular-pharming-high-level-plant-protein-expression-cpmv-ht/">'HyperTrans'</A> and <A href="http://onlinelibrary.wiley.com/doi/10.1002/9780471729259.mc16i01s8/abstract">PVX</A> expression systems, which boast up to 30% of total soluble protein (TSP) being recombinant protein!  Finally, we estimated the number of people in the US that suffer from Graves disease, calculated the mean severity of the disease, and then ran the model to estimate how many plants we would need to give every sufferer of Graves disease in the US a single dose of effective (90% reduction in TSI-created cAMP) treatment. This is because our <A href="https://2017.igem.org/Team:Cardiff_Wales/Human_Practices">human practices</A> involved communication with various GM-plant professionals, who recommended us several scientific articles to read.  These papers showed that plants are useful to use for a production platform as scaling up is linear. Thus, if you need 100x the medicine, you simply need 100x the plants!  The simple equation is shown directly below, with the breakdown of how we applied this equation using our own variables below that. At the bottom of this page or using this <A href="https://static.igem.org/mediawiki/2017/7/7a/T--Cardiff_Wales--Modelling_Extra_Material2.pdf">link</A>, you can find a PDF of all the variables and statistics that were used to model our theoretical biofactory, with some tables showing the key information that we turned into graphs.
</P></DIV><DIV class="column_full_size"><P>
Where: <B>N</B> = The number of plants needed to create a single dose of antagonist for a single person. <B>D</B> =  The dose multiplier (How much protein product you need in relation to the molecule you are trying to inhibit (I)). This can be changed depending on studies carried out on a different antagonist/therapeutic. For example you may have an antagonist that has most effect at 2x the concentration of the molecule you are trying to inhibit, and thus 'D' would be 2. <B>I</B> = The concentration of the molecule you are trying to inhibit. In our model, this is the concentration of Thyroid Stimulating Immunoglobulin (TSI) measured in µU/ml. This can be changed for any other molecule you are trying to inhibit, and the units can therefore be changed to. <B>V</B> =  The mean volume of blood serum in a patient, which is approximately 2850ml (between 2700 - 3000). This was used to estimate the total concentration of TSI in a patient, but should only be used when trying to calculate the total number of units of 'I' in a given volume of liquid. <B>P</B> = The amount of the antagonist a single plant can create. Most production platforms measure this in mg, but our antagonist is measured in international units (IU), and needed to be converted to International µU per plant. This value and its units can change depending on the expression system you are using. We have demonstrated this by modelling different plant species, cultivars, and expression systems, changing the value of 'P'.  
These are the factors that make up the simple interchangeable equation. However, for our model specifically, we needed several calculations to calculate the above variables. Thus, a breakdown of the equations that we used to create this model are shown below. These equations are not interchangeable, but are provided to show how we created these final models and graphs. Every made <B>assumption</B> is stated below in the breakdown too, with all the references to the papers/articles that these values were taken from.

</P></DIV><DIV class="column_full_size" style="background-color:#ffffff;"><CENTER><H3> The breakdown </H3></CENTER><P style="background-color:#ffffff;">
This section will show what value was used for each of the variables in the simple model, the equations that were used to calculate the variables, and references to papers whose data we used in the calculations.</P><P style="background-color:#ffffff;">
The amount of antagonist (µU) needed for every µU of TSI was calculated to be 0.2666667x. Fares et al. (2000) reported that the TSH antagonist had a maximal effect (reducing TSI-related cAMP production by 90%) <I> in vitro </I> when there was 200µU/ml of antagonist for every 750µU/ml TSI. Here in our equation, 'An' represents antagonist concentration (200µU/ml), and 'Ag' represents agonist concentration (750µU/ml). Thus, we calculate that we need the antagonist concentration 0.2666667x the amount of TSI in a patients body -  both measured in µU/ml. Here, we make the assumption that the <I> in vitro </I> study by Fares et al. can be directly applied to patients <I> in vivo </I>. </P><P style="background-color:#ffffff;">

Cyclic adenosine monophosphate (cAMP) is the signalling molecule that is produced as a result of TSH-receptor activation. It regulates the gene expression pathway that results in thyroid hormone production. Therefore, it is the most commonly used indicator of TSH-receptor activity, including TSI abundance in patients with Graves' disease.


The amount of TSI in a patient was calculated using two independent papers. The first paper, by Gerding et al. (2000), measured cAMP levels per 40,000 CHO cells, transfected with the hTSH-receptor, grown in IgG fractions (containing TSI) taken from patients with Graves' ophthalmopathy (GO) (a form of Graves' disease that is also caused by elevated TSI levels) , and divided these into two classes of severity (S). The more severe class was determined to have an average of 42pmol of cAMP, and the less severe class to have 35pmol. Thus, we use the left-half of this equation to deduce how much cAMP is present, on average, in patients with either more-severe or less-severe GO. This is measured in cAMP/cell, and was calculated to be 0.000875pmol of cAMP/cell for in a less-severe case, and 0.00105pmol of cAMP/cell. The right-half of the equation is taken from Kraiem et al. (1987), which used <I> in vitro </I> cell cultures of 25,000 cells, and measured 6.5pmol of cAMP produced in the surrounding solution when 250µU/ml of TSI was added. Using this, we can calculate how much cAMP is produced per cell per µU of TSI, which was calculated to be 0.00000104pmol cAMP/cell/µU/ml. Together, assuming that these <I> in vitro </I> studies reflect <I> in vivo </I>, and that the growth medium for these experiments does not influence cAMP production, we can estimate that a patient with less-severe Graves' disease has 841.35 µU of TSI/ml of serum, and a patient with more-severe Graves' disease has 1009.62 µU of TSI/ml of serum. Thus, these are the two units for 'I' that we used in the model. For the average, we took the mean of these values (925.485µU of TSI/ml) which was used to calculate the number of plants needed to treat the everyone in the US who has Graves' disease. 
</P><P style="background-color:#ffffff;">
In our equation, 'V' is a constant which is simply the volume of blood serum, which is roughly the volume of blood plasma. Here we have used a value of 2850ml (the mean between the lower estimate of 2700ml, and the higher estimate of 3000ml), but clinically this would not be a constant, and would vary per patient. Generally, blood plasma makes up ~55% of total blood volume, so this would need to be calculated per patient.
</P><CENTER><P style="background-color:#ffffff;"><B>OR, for HyperTrans/PVX </B></P></CENTER><P style="background-color:#ffffff;">
In this final equation, 'P' represents plant statistics. For our model, this is how many µU of antagonist one plant can produce. This is the variable that we made several substitutes to, to show that the model can be adjusted to suit new uses. Here, we tested several different variables. These were 3 different plants with 2 different expression systems. These were <I> N. benthamiana</I> and <I> N. tabacum</I> (CVs TI-95 and I-64), each with <I> Agrobacterium</I>-mediated transformation, and using HyperTrans or PVX as an expression system. 'Pc' is Total Soluble Protein (TSP) concentration, and was estimated to be 6.2mg TSP per gram of leaf in <I> N. benthamiana</I> (Robert et al., 2013) and 17.5mg/g in an average <I>N. tabacum</I> cultivar such as CV. I-64 (Song et al., 2015). Protein concentration in <I>N. tabacum</I> CV. TI-95 is above average, and about 1.8x higher than CV. I-64 according to Conley et al. (2010), so 'Pc' for this cultivar is assumed to be 31.5mg/g. 'M' is the total mass of leaves on a plant as calculated by Conley et al. (2010). <I>N. benthamiana</I> has a mean leaf mass of 10g/plant, <I>N. tabacum</I> CV. TI-95 has a mean leaf mass of 110g/plant, and CV. I-64 has a mean leaf mass of 410g/plant! The assumption here is that all the leaves are effectively inflitrated. This clearly would need a large scale production facility as at <A href="http://medicago.com/">Medicago.</A> 

'E' represents the efficiency of the production system, of both the plant and the expression system. Conley et al. (2010) reported that <I> N. benthamiana</I> produced 11.4ng of erythropoietin (EPO) per mg of TSP. <I>N. tabacum</I> CV. I-64 produced 22.12ng/mg, and <I>N. tabacum</I> CV. TI-95 produced 36.05ng/mg. Here we made the assumption that the same amount of TSH-antagonist (mass of 28kDA) would be produced as EPO (mass of 30.4kDa). Lastly, <A href="http://pbltechnology.com/">Pbltechnology (2017)</A> state that transformation using CPMV-HyperTrans can produce recombinant protein up to 30% TSP, and Hefferon (2017) state similar numbers using Potato Virus X in <I>N. benthamiana</I>. Consequently, we calculated how much TSH-antagonist might be created using a HyperTrans or PVX system, by multiplying the total protein content of each plant by 0.3. With these calculations we assume that TSH can be purified from leaf tissue in the same amounts as EPO. As this can only be empirically determined we are satisfied in making this assumption.    Finally, we multiply by 12 as TSH has an activity of 12IU per mg, and as our antagonist is TSH with slightly different amino acids at a few sites to remove glycosylation we can assume that the antagonist also has 12IU per mg. Using the values taken from the EPO statistics in  Conley et al. (2010), we use the first of these equations, as the EPO content is measured in ng/mg so we need to divide by 1,000,000 to convert this to mg/mg. Then, as we want the antagonist to be in IµU, we need to multiply by 1,000,000, so these cancel out. Conversely, using the HyperTrans or PVX systems, we still need to convert IU to IµU so multiply by 1,000,000, but as these generate recombinant protein up to 30% TSP, the 'E' variable is already 0.3mg/mg so does not need to be converted. Thus, this uses the second equation.
</P></DIV><DIV class="column_full_size"><CENTER><H3> Display </H3></CENTER><P> Here are some graphs that have data plotted using our model. The first shows how many µU of TSH antagonist might be extracted from in each plant variety. The second shows the result of using the model when finding out how many plants are needed to give a single effective dose of antagonist to a severely affected person, and the third for a less-severely affected person. </P><CENTER><P> These graphs clearly suggest that the use of <I> Nicotiana tabacum </I> cultivars is more beneficial for plant recombinant protein production. Cultivars of this species usually grow larger than those of <I> N. benthamiana</I>, meaning that they require more space in the lab and so you can have fewer plants. More importantly, these plants grow slower than <I> N. benthamiana </I>. These factors make <I> N. benthamiana </I> the organism of choice for most plant expression facilities (Conley et al, 2010)
The results that this model has obtained for us seem reasonable. <I> Nicotiana benthamiana </I> is a small plant and the amount of protein that can be produced from this plant is well characterised. Conversely, these cultivars of <I> N. tabacum </I> are very large and can have extremely high total protein concentrations, so the large difference between plant species also seems reasonable. With regards to the HyperTrans and PVX model results, these expression systems are relatively new, but do claim that this amount of protein is possible. We believe that the statistics generated for these systems are plausible, but whether they are in practice will likely be revealed with subsequent experiments by future researchers using them. </P></CENTER></DIV><DIV class="column_full_size" style="background-color:#ffffff;"><CENTER><H3> Integrating human practices, and scaling up </H3></CENTER><P> Our human practices involved communication with several professionals in the field of plant GM. Consequently, they gave us advice about plant GM, and informed us of some of the pros and cons. For example, we communicated extensively with Dr. Mauritz Venter, co-founder and the CEO of AzarGen Biotechnologies, and he explained some of these benefits. He kindly referred us to read some articles about AzarGen and plants as biofactories in general. Holtz et al. (2015), Nandi et al. (2016), Rybicki (2010), Ibioinc.com (2017), and Engineering News (2017) all report about the scalability of plants, reporting that they are one of (if not the) easiest platforms to scale up, and that scaling up is linear.  In addition we had a phone call with Philip Cater and Nicholas Holton from the company 'Leaf Expression Systems', who informed us about Medicago. Following this, we communicated with Anne Shiraishi, the communications manager at Medicago. She kindly referred us to read a paper from Lomonossoff and DAoust (2016), which reinforced the information provided by Dr. Venter. For more information about this research visit the <A href="https://2017.igem.org/Team:Cardiff_Wales/Our_research"> research </A> section of our human practices. Using this information, we can scale up our model and create a theoretical biofactory, and calculate how many plants we would need to treat every sufferer of Graves' disease in the US, assuming an average severity. Information from the NIH suggests that 1/200 people suffer from Graves disease in the US (Genetics Home Reference, 2017). Assuming the US population is around 323.1 million, it is estimated that around 1,615,500 people suffer from Graves' disease in the US. Seeing as our research showed that scaling up is linear, we can estimate how many plants of each expression system would be needed to give a single effective dose to every sufferer (assuming a mean severity). This is displayed below.
</P><P> Finally, we recognise that a single effective dose for a patient is not enough to treat the disease permanently, but dosage frequency and actual <I> in vivo </I> effectiveness would need to be calculated with clinical trails. We have demonstrated that this model is flexible and can be changed to suit different requirements and objectives, and we hope it may be a useful tool for other plant iGEM teams or even companies in the future. </P></DIV><DIV class="column_full_size"><CENTER><H3> References </H3></CENTER><P>
Conley, A., Zhu, H., Le, L., Jevnikar, A., Lee, B., Brandle, J. and Menassa, R. (2010). Recombinant protein production in a variety of Nicotiana hosts: a comparative analysis. Plant Biotechnology Journal, 9(4), pp.434-444.

Engineering News. (2017). AzarGen’s case for a commercial plant-made pharmaceutical facility in South Africa. [online] Available at: http://www.engineeringnews.co.za/article/azargens-case-for-a-commercial-plant-made-pharmaceutical-facility-in-south-africa-2017-04-20.

Fares, F., Levi, F., Reznick, A. and Kraiem, Z. (2000). Engineering a Potential Antagonist of Human Thyrotropin and Thyroid-stimulating Antibody. Journal of Biological Chemistry, 276(7), pp.4543-4548.

Genetics Home Reference, N.I.H (2017). Graves disease. [online] Genetics Home Reference. Available at: https://ghr.nlm.nih.gov/condition/graves-disease#statistics.

Gerding, M., van der Meer, J., Broenink, M., Bakker, O., Wiersinga, W. and Prummel, M. (2000). Association of thyrotrophin receptor antibodies with the clinical features of Graves' ophthalmopathy. Clinical Endocrinology, 52(3), pp.267-271.

Hefferon, K. (2017). Plant Virus Expression Vectors: A Powerhouse for Global Health. Biomedicines, 5(3), p.44.

Holtz, B., Berquist, B., Bennett, L., Kommineni, V., Munigunti, R., White, E., Wilkerson, D., Wong, K., Ly, L. and Marcel, S. (2015). Commercial-scale biotherapeutics manufacturing facility for plant-made pharmaceuticals. Plant Biotechnology Journal, 13(8), pp.1180-1190.

Ibioinc.com. (2017). iBio Proprietary Technology Advances Neonatal Respiratory Distress Syndrome Drug Development for South African Biotech Company AzarGen Biotechnologies (Pty) Ltd. [online] Available at: https://www.ibioinc.com/news/press-releases/detail/72/ibio-proprietary-technology-advances-neonatal-respiratory.

KRAIEM, Z., LAHAT, N., GLASER, B., BARON, E., SADEH, O. and SHEINFELD, M. (1987). THYROTROPHIN RECEPTOR BLOCKING ANTIBODIES: INCIDENCE, CHARACTERIZATION AND IN-VITRO SYNTHESIS. Clinical Endocrinology, 27(4), pp.409-421.

Lomonossoff, G. and DAoust, M. (2016). Plant-produced biopharmaceuticals: A case of technical developments driving clinical deployment. Science, 353(6305), pp.1237-1240.

Nandi, S., Kwong, A., Holtz, B., Erwin, R., Marcel, S. and McDonald, K. (2016). Techno-economic analysis of a transient plant-based platform for monoclonal antibody production. mAbs, 8(8), pp.1456-1466.

Pbltechnology.com. (2017). Molecular Pharming: High Level Plant Protein Expression (CPMV-HT) | PBL Technology. [online] Available at: http://www.pbltechnology.com/molecular-pharming-high-level-plant-protein-expression-cpmv-ht/.

Robert, S., Khalf, M., Goulet, M., D’Aoust, M., Sainsbury, F. and Michaud, D. (2013). Protection of Recombinant Mammalian Antibodies from Development-Dependent Proteolysis in Leaves of Nicotiana benthamiana. PLoS ONE, 8(7), p.e70203.

Rybicki, E. (2010). Plant-made vaccines for humans and animals. Plant Biotechnology Journal, 8(5), pp.620-637.

Song, I., Kim, D., Kim, M., Jamal, A., Hwang, K. and Ko, K. (2015). Comparison of total soluble protein in various horticultural crops and evaluation of its quantification methods. Horticulture, Environment, and Biotechnology, 56(1), pp.123-129.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>