<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WashU_StLouis_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WashU_StLouis/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WashU_StLouis/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU_StLouis/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU_StLouis/Protocol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WashU_StLouis/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WashU StLouis/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="content"><UL id="dropmenu"><LI><A href="https://2014.igem.org/Team:WashU_StLouis">Home</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Team/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=WashU_StLouis" target="_blank">Team Profile</A></LI></UL><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project/nif">Nitrogenase</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project/light">Light Regulation</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Notebook/September"> Post Summer </A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Protocol">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Web_Series">Video Series</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Safety">Lab safety</A></LI></DIV><DIV class="floating-menu"><H3>Protocols</H3><CENTER><B>Brauer Group</B></CENTER><A href="#1">PCR</A><A href="#2">Colony PCR</A><A href="#3">Gel Electrophoresis</A><A href="#4">DNA Purification</A><A href="#5">Digestion/Ligation</A><A href="#6">Transformation</A><A href="#7">Starting Cultures</A><A href="#8">Miniprep</A><A href="#9">Light Induction</A><CENTER><B>Rebstock Group</B></CENTER><A href="#10">Electroporation</A><A href="#11">Innoculation</A><A href="#12">10x M9 Stock</A><A href="#13">100x Ferric Citrate Soln</A><A href="#14">1x K.pneumoniae Medium</A><A href="#15">Acetylene Reduction Assay</A><A href="#16">Gel Electrophoresis</A><A href="#17">CPEC</A></DIV><TABLE id="menu" width="100%&quot;" cellspacing="0" height="500px"><TBODY><TR><TD><DIV style="text-align: center;"><H1><SPAN style="font-weight: bold;">Brauer Group Protocols</SPAN></H1></DIV><SPAN style="font-weight: bold;">Polymerase Chain Reaction</SPAN>    Phusion HF Polymerase Protocol<DIV style="margin-left: 40px;">Per tube, add in this
order:                             </DIV></TD></TR><TR><TD style="vertical-align: top; text-align: center;"> Cloning
water </TD><TD style="text-align: center;" valign="top"> 21.5 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> 5x HF buffer </TD><TD style="text-align: center;" valign="top"> 10 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> 5M Butane </TD><TD style="text-align: center;" valign="top"> 10 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> DMSO </TD><TD style="text-align: center;" valign="top"> 4 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> 10µM Primers (f+r) </TD><TD style="text-align: center;" valign="top"> 2.5 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> 10µM dNTPs </TD><TD style="text-align: center;" valign="top"> 1.0 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> Template DNA </TD><TD style="text-align: center;" valign="top"> 0.5 µl </TD></TR><TR><TD style="text-align: center;" valign="top"> Phusion Polymerase </TD><TD style="text-align: center;" valign="top"> 0.5 µ </TD></TR></TBODY></TABLE>
PCR Reaction in Thermal Cycler:
<TABLE style="height: 40px; margin-left: 40px;" border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD valign="top"> 98 °C </TD><TD valign="top"> 30 seconds </TD></TR><TR><TD valign="top"> 30 Cycles </TD></TR><TR><TD valign="top"> 98 °C </TD><TD valign="top"> 20 seconds </TD></TR><TR><TD valign="top"> Annealing Temperature (NEB TM Calculator)
</TD><TD valign="top"> 20 seconds </TD></TR><TR><TD valign="top"> 72 °C </TD><TD valign="top"> Extension Time (20 seconds/kb) </TD></TR></TBODY></TABLE> 72 °C  5 minutes  4 °C  Forever     Colony PCR<DIV style="margin-left: 40px;">1. Prepare 10 µl of cloning water per
colony to be picked in PCR tubes.
2. Prepare culture tube filled with 4mL LB, 4µL resistance
3. Pick colony and resuspend in cloning water by pipetting up and down
4. Next, drop the tip inside the culture tube, or dot onto pre-numbered
LB plate
5. Heat water with colonies for 10 minutes in 98°C; use as “template
DNA”
6. Put culture tubes in 37°C shaker to incubate (overnight) while PCR
is running.
7. Go TAQ PCR mix</DIV><TABLE style="height: 246px; width: 255px; margin-left: 40px;" border="1" cellpadding="5" cellspacing="0"><TBODY><TR><TD style="text-align: center;" valign="top"> Cloning Water </TD><TD style="text-align: center;" valign="top"> 35µl </TD></TR><TR><TD style="text-align: center;" valign="top"> Vortexed 5x Green
Taq Buffer </TD><TD style="text-align: center;" valign="top"> 10µl </TD></TR><TR><TD style="text-align: center;" valign="top"> 10µM Sequencing
Primer </TD><TD style="text-align: center;" valign="top"> 2.5µl </TD></TR><TR><TD style="text-align: center;" valign="top"> 10µM dNTPs </TD><TD style="text-align: center;" valign="top"> 1.0µl </TD></TR><TR><TD style="text-align: center;" valign="top"> Template DNA
(heated above) </TD><TD style="text-align: center;" valign="top"> 1.0µl </TD></TR><TR><TD style="text-align: center;" valign="top"> GoTaq Polymerase </TD><TD style="text-align: center;" valign="top"> 0.5µl </TD></TR></TBODY></TABLE><DIV style="margin-left: 40px;">8. TAQ PCR Protocol
</DIV><TABLE style="width: 392px; height: 136px; margin-left: 40px;" border="1" cellpadding="5" cellspacing="0"><TBODY><TR><TD valign="top"> 95 °C </TD><TD valign="top"> 2 minutes </TD></TR><TR><TD valign="top"> 35 Cycles </TD></TR><TR><TD valign="top"> 95 °C </TD><TD valign="top"> 1 minute </TD></TR><TR><TD valign="top"> Annealing Temperature (NEB TM Calculator)
</TD><TD valign="top"> 45 seconds </TD></TR><TR><TD valign="top"> 72 °C </TD><TD valign="top"> Extension Time (20 seconds/kb) </TD></TR></TBODY></TABLE> 72 °C  5 minutes  4 °C  Forever <DIV style="margin-left: 40px;">
9. Run Gel Visualization
10. Add 15µl PCR product (no need 6x loading dye)
11. Freeze &amp; Miniprep cultures that have desired bands</DIV><SPAN style="font-weight: bold;">Gradient PCR</SPAN><UL><LI>Annealing Temperature: 5 °C below to 5 °C above.</LI><LI>Example: 62°C = 57°C -&gt; 67°C</LI><LI>Increase Annealing Time to 30 seconds</LI></UL><SPAN style="font-weight: bold;">Making a Gel for Gel
Electrophoresis</SPAN><OL><LI>0.7g of Agarose</LI><LI>Add 100 ml 1x TAE buffer into an Erlenmeyer flask</LI><LI>Microwave for 2 minutes</LI><LI>Cool down under tap water for 20 seconds</LI><LI>Add 5 µl Sybrsafe</LI><LI>Pour the gel into mold with comb. Let cool for 30 minutes.</LI><LI>Remove the comb.</LI><LI>Transfer tray into an electrophoresis chamber, wells facing black
side (anode) of chamber</LI><LI>Fill chamber with 1x TAE until TAE just covers the gel</LI></OL><SPAN style="font-weight: bold;">
Gel visualization/extraction</SPAN><DIV style="margin-left: 40px;">1. Prepare samples</DIV><TABLE style="width: 292px; height: 40px; text-align: left; margin-left: 40px;" border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD style="vertical-align: top; text-align: left;"> Visualization
</TD><TD style="text-align: left;" valign="top"> Extraction </TD></TR><TR><TD valign="top"> 10µl PCR Product </TD><TD valign="top"> 50µl PCR Product </TD></TR><TR><TD valign="top"> 2µl 6x Loading Dye </TD><TD valign="top"> 10µl 6x Loading Dye </TD></TR></TBODY></TABLE><DIV style="margin-left: 40px;">
2. Add DNA ladder Standard into first well.
3. Pipet prepared samples into rest of wells.
4. Seal electrophoresis chamber with chamber cap and plug cables in VWR
power unit.
5. Set 125V, Run for 30 minutes (or longer) depending on length of
sample expected.
6. Remove Gel tray once finished and visualize using trans-illuminator.
7. Extract if necessary into a preweighed 1.5 ml microcentrifuge tube.</DIV><SPAN style="font-weight: bold;">Gel purification</SPAN><DIV style="margin-left: 40px;">Using Zymoclean Gel DNA Recovery Kit
1. Add 3 volumes (µl) for each volume of agarose (mg) excised from the
gel.
2. Incubate at 55°C for 10 minutes until gel is dissolved
3. Pipette into a Zymo-Spin Column &amp; Collection Tube
4. Centrifuge (at 16000 x g)for 1 minute, and discard the flow-through.
5. Add 200µl DNA wash Buffer and centrifuge for 30 seconds.
6. Add another 200µl DNA wash buffer and centrifuge for a minute.
7. Place the spin column into a new sterile microcentrifuge tube.
8. Add up to 30µl of Cloning Water directly to the column matrix.
9. Let it sit for 4 minutes, and centrifuge for 4 minutes.</DIV><SPAN style="font-weight: bold;">Treating with DPNI</SPAN><DIV style="margin-left: 40px;">1. Add to a PCR rxn mix after PCR.
2. 1µl DPNI per 50 µl rxn mix.
3. Let it run 37°C for 1 hour.</DIV><SPAN style="font-weight: bold;">DNA Purification</SPAN><DIV style="margin-left: 40px;">Using Zymoclean DNA Clean &amp;
Concentrator
1. Add 5 volumes of DNA binding Buffer to PCR product or Short DNA
fragments.
2. Vortex Tube.
3. Load mixture into Zymo-Spin Column and Collection Tube
4. Centrifuge (16000 x g) for 30 seconds. Discard flow-through.
5. Add 200µl of DNA Wash Buffer and centrifuge for 30 seconds.
6. Add another 200µl of DNA Wash Buffer and centrifuge for 2 minutes.
7. Place the spin column into a new sterile microcentrifuge tube.
8. Add up to 30µl of Cloning Water directly to the column matrix.
9. Let it sit for 4 minutes, and centrifuge for 4 minutes.</DIV><SPAN style="font-weight: bold;">Golden Gate
Digestion/Ligation</SPAN><DIV style="margin-left: 40px;">1. Design Primers accordingly, taking
into account Type IIs Restriction
Enzyme site.
2. Add purified pieces you want to ligate based on concentration
results (100ng / x ng/µl) into a PCR tube.
3. Add cloning water to the mix up to 10.5µl total.
4. Add 1.5µl Cutsmart Buffer to the mix.
5. Vortex ligase buffer. Add 1.0µl to the mix.
6. Add 1µl of Type IIs restriction enzyme (we usually use sapI)
7. Add 1µl of T4 Ligase to the mix.
8. Total should be 15µl in the tube. If volume of pieces you want to
ligate together add up to greater than 10.5µl, scale up the mix
proportionally.
Reaction assembly</DIV><TABLE style="width: 148px; height: 232px; margin-left: 40px;" border="1" cellpadding="1" cellspacing="0"><TBODY><TR><TD valign="top">50 x</TD></TR><TR><TD valign="top">37°C</TD><TD valign="top">3 min</TD></TR><TR><TD valign="top">16°C</TD><TD valign="top">4 min</TD></TR></TBODY></TABLE>50°C5 min80°C5 min<SPAN style="font-weight: bold;">Restriction Enzyme Digestion</SPAN><DIV style="margin-left: 40px;">1. Follow NEB protocols.
2. Find appropriate buffer if doing double digests.
3. Heat inactivate at the end.
4. If doing sequential digests, heat inactivate in between each step.</DIV><SPAN style="font-weight: bold;">Blunt End Ligation</SPAN><DIV style="margin-left: 40px;">1. Add 18µl of purified PCR product
into a PCR tube.
2. Add 2.25µl of 10x T4 Ligase Buffer
3. Add 1.125µl of T4 PolyNucleotide Kinase.
4. Put in thermocycler for 37°C for 45 minutes.
5. Add 1.125µl T4 DNA Ligase
6. Leave at room temperature for at least 1 hour.
7. Heat inactivate whatever you are not using at 65°C for 20 minutes.</DIV><SPAN style="font-weight: bold;">Transformation
(Electroporation)</SPAN><DIV style="margin-left: 40px;">1. Put electroporation cuvette on ice
and thaw electro-competent cells
for 5 minutes.
2. Prepare 500µl LB and a labeled culture tube.
3. Add up to 5µl of DNA directly to the competent cells. (usually use
2µl)
4. Transfer mix into cuvette, ensuring no bubbles and mix touching both
sides.
5. Wipe down cuvette to remove moisture.
6. Place in electroporation machine and start.
7. If arc occurs, try with smaller amounts of DNA.
8. If successful, immediately add the LB directly into the cuvette.
9. Transfer LB + transformed cells into the culture tube.
10. Record time and incubate in the 37°C shaker for 1 hour.</DIV><SPAN style="font-weight: bold;">Plating</SPAN><DIV style="margin-left: 40px;">1. Prepare two appropriate antibiotic
resistance “100µl” and “Rest”
2. Add 100µl of transformed culture (after the 1 hour incubation) onto
the 100µl plate.
3. Use a sterile spreader to streak the liquid culture onto the plate
in the order on the left:
4. On the “Rest” Plate, pour the rest of the liquid culture and spread
evenly on the plate.
5. Flip and put in the 37°C incubator overnight (no longer than 20
hours).
6. Wrap your plate after sufficiently grown and store in the 4°C fridge
for up to 45 days.</DIV><SPAN style="font-weight: bold;">Liquid Cultures</SPAN><DIV style="margin-left: 40px;">1. Determine volume of culture to grow
in a culture tube. We usually
use 4ml of LB.
2. In the biosafety hood, add 4ml of LB, and 4µl of each appropriate
resistance(s).
3. Use a tip to pick up a single colony from the plate.
4. Drop the tip into the media.
5. Cap the culture tube to the first stop.
6. Incubate in 37°C shaker at 250rpm for up to 16 hours.</DIV><SPAN style="font-weight: bold;">Freeze Cultures</SPAN><DIV style="margin-left: 40px;">1. Add 500µl of 30% glycerol into a
cryo-freeze tube.
2. Add 500µl of overnight culture.
3. Store in -80°C.</DIV><SPAN style="font-weight: bold;">Plasmid Miniprep</SPAN><DIV style="margin-left: 40px;">Using Zyppy Plasmid Miniprep Kit
1.Centrifuge overnight culture for 10 minutes at 5000 x g
2. Discard supernatant. Resuspend with 550µl LB
3. Transfer culture to 1.5 ml microcentrifuge tube.
4. Add 100 µl of 7X Lysis Buffer (Blue). Shake well.
5. Within 2 minutes, add 350 µl of cold Neutralization Buffer (Yellow).
Shake well.
6. Centrifuge at 16000 x g for 4 minutes.
7. Transfer supernatant into a Zymo-Spin Column and Collection Tube.
8. Centrifuge at 16000 x g for 30 seconds. Discard flow-through.
9. Add 200µl Endo-Wash Buffer. Centrifuge at 16000 x g for 30 seconds.
10. Add 400µl Zyppy Wash Buffer. Centrifuge at 16000 x g for 2 minutes.
11. Transfer column into new labeled microcentrifuge tube.
12. Add 30µl of cloning water directly to column matrix. Let sit for 4
minutes.
13. Centrifuge at 16000 x g for 4 minutes to elute.</DIV><SPAN style="font-weight: bold;">Light Induction
Experiment</SPAN><DIV style="margin-left: 40px;">1. Co-transform chromophore plasmid
with light sensor (2µl of each miniprepped product).
2. Plate and pick colonies.
3. Start liquid cultures with appropriate antibiotic resistance for
light sensor(s) and appropriate controls.
4. Wrap tubes that you want to use in dark with foil carefully.
5. Freeze experimental stock for future use.
6. Measure OD600
7. Dilute to OD 0.1
8. Grow for 2 hours in 37°C shaker.
9. Transfer into deep well plates and induce with No and Max ATC in
both Light and Dark conditions (2 controls, light and ATC).
10. Grow for 6-8 hours in a lighted growth chamber (wrap plates that
will be in the dark).
11. Spin down colonies at 3000x g for 15 minutes.
12. Resuspend with 200µl 1 x PBS.
13. Measure Fluorescence/Absorbtion in 96 well plates. 
14. For EYFP we used
Excitation and Emission values of 485nm and 528nm respectively.</DIV><DIV style="text-align: center;"><H1>Rebstock Group Protocols</H1></DIV><SPAN style="font-weight: bold;">Electroporation
Protocol</SPAN><OL><LI>Take 1 mL from tube culture and transfer it into 50 mL of LB
broth in a flask. Let culture for 2 hrs in the 37°C room.</LI><LI>Pipet 150 μL of each flask into a well in a 96 well plate and
analyze the optical density. May proceed when OD600 is between 0.1 and
0.15</LI><LI>Chill the flasks for 15 min. and shake/swirl every 5 min.</LI><LI>Empty the flasks into centrifuge tubes</LI><LI>Centrifuge for 5 mins at 4000 rpm. </LI><LI>Extract supernatant with vacuum and fill with autoclaved water to
between 15-20 mL--just make sure they are all even. </LI><LI>Repeat step 5</LI><LI>Repeat step 6</LI><LI>Repeat step 5</LI><LI>Add 200 μL of 10% glycerol into each tube</LI><LI>Pipet 50 μL of each into a cuvette</LI><LI>Pipet 4 μL of plasmid suspension into the cuvette and ice for 10
min. </LI><LI>Transfer to electroporation cuvette--be sure not to touch the
metal--and keep on ice</LI><LI>Perform electroporation under the second E. coli setting: 2500 V
with a 0.2 cm cuvette</LI><LI>Transfer 1 mL of LB to each electroporation cuvette.</LI><LI>Transfer all of the contents of the cuvettes into tubes. Grow for
1 hr at 37°C</LI><LI>Transfer the contents of the tubes into centrifugable cuvettes</LI><LI>Plate varying amounts of the cell suspension. **For large volumes
(&gt;500 μL) spin down the desired volume in a small centrifuge, then
remove most of the supernatant so that the volume is ~150 μL. Resuspend
the cells and plate the remaining suspension.</LI><LI>Make sure the plates are dry before storing them at 37°C
overnight or until discernible colonies form.</LI></OL><SPAN style="font-weight: bold;">Inoculation of
Transformed E. coli
from Plates into Tubes Protocol </SPAN><OL><LI>Prepare all labels before hand--be sure to include culture
identifiers, the date, and your initials</LI><LI>Light a bunsen burner and perform all steps using sterile
technique</LI><LI>Pipet 4 mL of LB into each tube using a volumetric pipet</LI><LI>Using a 10 μL pipettman, add 4 μL of the appropriate antibiotic
to each tube</LI><LI>Now put a tip on your 10 μL pipettman and scrape a single colony
from your plate</LI><LI>Eject the tip into the tube and cap the tube</LI><LI>Place all tubes on the shaker table in the 37°C room</LI></OL><SPAN style="font-weight: bold;">Recipe for 10X M9
Stock Solution for
Nitrogenase Activity Assay (100mL):  </SPAN>     Reagents:
<UL><UL><LI>0.026 g CaCl2·H2O, </LI><LI>0.030 g MgSO4,</LI><LI>10.4 g Na2HPO4, </LI><LI>3.4 g KH2PO4, </LI><LI>4 g glucose </LI></UL></UL>    Procedure:
<OL><LI>Fill a clean 100 mL beaker with ~70 mL of water</LI><LI>Add a magnetic stir bar and place on stir plate, stirring
moderately</LI><LI>Add the appropriate amount of reagent listed above IN THE ORDER
listed above</LI><LI>Increase or decrease the speed of the stir plate until all
particles are dissolved</LI><LI>Once the solution is clear and colorless, remove the stir bar and
pour the contents of the    beaker into a 100 mL
graduated cylinder</LI><LI>Fill with water to the 100 mL mark</LI><LI>Pour the solution back into the original beaker</LI><LI>In a hood with a flame, filter the solution with a sterile filter
into an autoclaved bottle</LI><LI>Label and store for future use   </LI></OL>         <SPAN style="font-weight: bold;">Recipe for 1000X M9 Supplemental Stock
Solution (100 mL) </SPAN>    Reagents:
<UL style="margin-left: 40px;"><LI>0.3 g MnSO4,          </LI><LI>7.6 g Na2MoO4*2H2O,       </LI><LI>0.010 g p-aminobenzoic acid, </LI><LI>0.005 g biotin</LI></UL>    Procedure:
<OL><LI>Fill a 100 mL beaker with ~70 mL of water</LI><LI>Add a magnetic stir bar and place on stir plate, stirring
moderately</LI><LI>Add the appropriate amount of reagent listed above</LI><LI>Increase or decrease the speed of the stir plate until all
particles are dissolved</LI><LI>Remove the stir bar and pour the contents of the beaker into a
100 mL graduated cylinder</LI><LI>Fill with water to the 100 mL mark </LI><LI>Pour the solution back into the original beaker</LI><LI>In a hood with a flame, filter the soution with a sterile filter
into an autoclaved bottle</LI><LI>Label and store for future use</LI></OL><SPAN style="font-weight: bold;">Recipe for 100X
Ferric Citrate Stock
Solution (100 mL)</SPAN>    Reagent:
<UL><UL><LI>0.36 g Ferric citrate</LI></UL></UL>    Procedure:
<OL><LI>Fill a 100 mL beaker with ~70 mL of water</LI><LI>Add a magnetic stir bar and place on a stir plate, stirring
moderately</LI><LI>Add the appropriate amount of reagent listed above</LI><LI>Increase or decrease the speed of the stir plate until the
surface of the particles have dissolved--not all of the rusty-colored
particles will dissolve</LI><LI>Transfer the suspension into an autoclaveable bottle</LI><LI>Autoclave</LI><LI>Once removed from the autoclave, the particles should be
dissolved and the solution should be a yellow-brown color. </LI><LI>Label and store for future use</LI></OL><SPAN style="font-weight: bold;">Recipe for Other 1X
K. pneumoniae
Medium (100 mL)</SPAN>    Mix the following to make 100 mL of solution:
<UL><UL><LI>0.01g CaCl22H2O</LI><LI>0.025g MgSO47H2O</LI><LI>2.5 g Na2HPO4</LI><LI>0.3g KH2PO4</LI><LI>0.1g NaCl</LI><LI>0.00029 g FeCl3</LI><LI>0.000025 g Na2MoO42H2O</LI><LI>2.0 g sucrose </LI><LI>0.6 mL (per liter) of 22% (wt/vol) NH4Ac</LI></UL></UL><SPAN style="font-weight: bold;">Preparation for
Acetylene Reduction
Assay </SPAN><OL><LI>Grow engineered E. coli strains at 37°C overnight in 4 mL of LB
broth with antibiotics</LI><LI>Collect cultures by centrifugation and wash with sterilized water
three times </LI><LI>Resuspend cells in 5 mL of nitrogen-deficient medium with 10 mM
glutamate as a nitrogen source in a sealed 25 mL tube and grow to a
final OD600 of 0.2-0.4.</LI><LI>Then evacuate the headspace and flush with argon.</LI></OL><SPAN style="font-weight: bold;">Preparation for Gel
Electrophoresis </SPAN><OL><LI>Select an appropriately sized plate for the number of samples you
would like to run</LI><LI>Place the two ends into the plate to prevent the liquid gel from
running</LI><LI>Prepare the gel as follows: for a 1% gel, pour 100 mL of 1X TAE
and 1g of agarose into a 250 mL flask</LI><LI>Microwave until it begins to bubble.</LI><LI>Remove with a glove or hot hand and swirl until it becomes sort
of cloudy again</LI><LI>Repeat steps 4 and 5 until the liquid is clear upon removal from
microwave</LI><LI>Then add 1.5 μL of ethidium bromide for every 50 mL of gel
solution--may want to allow the flask to cool to the touch before
adding. Swirl to mix. </LI><LI>Set the comb for the appropriate number of lanes into the
plate--be sure that the comb does not touch the bottom of the plate</LI><LI>Pour the gel</LI><LI>Allow to set for about 30 minutes.</LI></OL><SPAN style="font-weight: bold;">CPEC (Circular
Polymerase Extension
Cloning) Protocol – Andrew Ng</SPAN><OL><LI>Design primers that include overhangs for fragments that you want
to assemble by hand or using a program such as Snapgene (these primers
can also be used for Gibson assembly)</LI><LI>Using Phusion polymerase, PCR each insert as well as your plasmid
backbone and verify on a gel.</LI><OL><LI>IMPORTANT- it is highly recommended that you optimize your PCR
protocol (either using temperature gradient or DMSO) so that only one
band can be seen in your PCR</LI></OL><LI>Digest PCR product using DpnI (digests methylated DNA)</LI><OL><LI>If PCR was performed using Phusion polymerase from NEB, you can</LI></OL><LI>Add the DpnI enzyme from NEB directly. For this, I add 1 ul of
enzyme to 50 ul of PCR product and incubate for 3 hours or more 4. PCR
purify your inserts,</LI><OL><LI>NOTE: if you were unable to get a single band in your PCR, you
might consider gel purification if you can get enough yield</LI></OL><LI>Using a 2:1 molar ratio of insert to vector (if you did not PCR
purify, I would just eyeball concentrations based on length of parts
and darkness of band in gel), make your CPEC reaction as follows:</LI><OL><LI>4 ul HF Buffer</LI><LI>0.4 ul dNTPs</LI><LI>.1 ul Phusion Polymerase</LI><LI>Insert</LI><LI>Vector</LI><LI>Fill to 20 ul with dH20</LI></OL><LI>(Note for this step: The more DNA you have available to use here
the better. I would recommend using a maximum of between 8-10 ul of DNA
in total though due to the salt content)</LI><LI>Use the following thermal cycler protocol (or just navigate to
CPEC in the folder labeled Andrew in the BioRad C1000)</LI><OL><LI>98.0C for 30s</LI><LI>98.0C for 10s</LI><LI>Ramp to 55.0C at a rate of 0.1C/s</LI><LI>55.0C for 30s</LI><LI>72.0 C for 30s/kb of construct</LI><LI>Repeat 1-5 for 10-30 cycles (depending on the “complexity” of
your assembly)</LI><LI>g. 72.0 C for 10 min</LI></OL><LI>￼Transform cells with 2-5ul of CPEC mix</LI></OL><DIV class="footer"><CENTER><TABLE style="background-color:#FFF" width="80%"><TBODY><TR><TD style="padding-left: 15px; padding-right: 15px"><CENTER><H1> Special thanks to our 2014 Sponsors: </H1><H3> If you want to sponsor us, please contact us below! </H3></CENTER></TD></TR></TBODY></TABLE><TABLE style="background-color:#FFF" width="80%"><TBODY><TR><TD style="padding-left: 15px; padding-right: 15px"><P><CENTER> © Washington University in St. Louis iGEM 2014 | 
                       <A href="mailto:washu.igem@gmail.com" style="color:green"> Contact Us </A> | Follow Us on Social Media
         </CENTER></P></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:WashU_StLouis/Protocol">http://2014.igem.org/Team:WashU_StLouis/Protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WashU_StLouis/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WashU_StLouis/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WashU_StLouis/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WashU_StLouis/Protocol&amp;oldid=356637" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>