<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_WashU_StLouis_Project_light"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:WashU_StLouis/Project/light">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:WashU_StLouis/Project/light&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU_StLouis/Project/light&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:WashU_StLouis/Project/light&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:WashU_StLouis/Project/light" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:WashU StLouis/Project/light</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="content"><UL id="dropmenu"><LI><A href="https://2014.igem.org/Team:WashU_StLouis">Home</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Team/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=WashU_StLouis" target="_blank">Team Profile</A></LI></UL><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project/nif">Nitrogenase</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project/light">Light Regulation</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Project/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Notebook/September"> Post Summer </A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Protocol">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Web_Series">Video Series</A></LI><LI><A href="https://2014.igem.org/Team:WashU_StLouis/Safety">Lab safety</A></LI></DIV><DIV class="floating-menu"><H3>Home</H3><A href="#1">Introduction</A><A href="#2">Objectives</A><A href="#3">Approach</A><A href="#4">Workflow</A><A href="#5">Data</A><A href="#6">Results</A><A href="#7">Citation</A></DIV><TABLE style="width: 80%; text-align: left; margin-left: auto; margin-right: auto;"><TBODY><TR><TD style="width: 100%;"><DIV style="text-align: center;"><H1>Using an inducible hybrid light-sensitive promoter for
heterologous
regulation of the <SPAN style="font-style: italic;">nif</SPAN> cluster
</H1></DIV><DIV style="text-align: center;">Benjamin Huang, Jeffrey Lee</DIV><DIV style="text-align: center;"><H3><SPAN style="text-decoration: underline;">Introduction</SPAN></H3></DIV>
Traditionally, gene expression has been induced through chemical means.
However, this method requires expensive chemicals, and can have
deleterious side effects on cell health. In Tabor et. al (2010), it was
shown that it is possible to induce expression of a reporter gene with
light in <SPAN style="font-style: italic;">E. coli</SPAN> using light
sensitive proteins (CcaR/CcaS) from the
model cyanobacterium <SPAN style="font-style: italic;">Synechocystis</SPAN>
S. 6803. Light induction has several
advantages over chemical induction, including: long term cost of
chemical inducers, ability to tune expression levels with different
light intensities, and most importantly, can be turned on and off
easily. The ability to control genes of interest has important
applications in engineering nitrogen fixation. The end goal of our
project is for plants to fix nitrogen on their own within their
chloroplasts--the true powerhouse of plant cells, where the most ATP is
present to overcome the high costs associated with breaking nitrogen’s
triple bond.<H3 style="text-align: center;"><SPAN style="text-decoration: underline;">Objectives</SPAN></H3><SPAN style="font-style: italic;">Synechtocystis</SPAN> PCC6803
(hereafter S. 6803) is a strain of cyanobacteria that is of particular
interest to us. S. 6803 is important because it is the first
cyanobacteria to have its complete genome sequenced; and it is easily
manipulated via homologous recombination. S. 6803 has highly
characterized photosynthetic genes that are light regulated, which
would be useful if we want plants to eventually fix nitrogen. Based on
the Endosymbiotic theory, chloroplasts and
mitochondria are thought to have been cyanobacteria that were engulfed
by cells. Thus, S. 6803 is an important platform from which further
manipulation of the nitrogen fixation can occur. 
Recently it has been shown that one can induce via green light, the
expression of a phycobilisome-related gene [2]. Instead of regulating
gene expression via proteins, having to worry about different
concentration levels, potential cross effects, etc., regulation via
light can be easily tested for and manipulated as well. Therefore, we
propose to design a hybrid inducible light-sensitive promoter for
heterologous regulation of the <SPAN style="font-style: italic;">nif</SPAN>
cluster in <SPAN style="font-style: italic;">E. coli</SPAN>. <H3 style="text-align: center;"><SPAN style="text-decoration: underline;">Approach</SPAN></H3>
We created a 4 piece assembly plasmid integrating light regulation
components from pJT122, but swapping out <SPAN style="font-style: italic;">cph8</SPAN> (for EYFP from pSL2264)
and <SPAN style="font-style: italic;">lacZ</SPAN> (for TetR from
pTet-PP*) and combined them into a plasmid
PBJ003 which should repress expression of EYFP when induced by light.
We used these plasmids because they were easily available in the Moon
Lab at Washington University in St. Louis, and we didn't need to get
them from other labs or the registry. PBJ003 contains the basic cpcG2
promoter driving <SPAN style="font-style: italic;">tetR</SPAN>
production. When induced by light, the TetR generated will bind to the
tet promoter which should repress <SPAN style="font-style: italic;">eyfp
</SPAN>production. We also created a hybrid promoter to swap out
for the basic
cpcG2 promoter. Both these parts are on the registry under <A href="http://parts.igem.org/Part:BBa_K1385000">BBa_K1385000</A> and <A href="http://parts.igem.org/Part:BBa_K1385001">BBa_K1385001</A>.</TD></TR><TR><TD style="vertical-align: top; text-align: justify; width: 40%;">Hybrid
promoters that integrate the regulatory elements from several
sources have been shown to be functional with varying levels of success
[3]. Operator sites for the green-light sensitive PcpcG2 promoter are
well known [4], additionally, the hybrid trc1O promoter has been shown
to be highly expressed in <SPAN style="font-style: italic;">E. coli </SPAN>even
without IPTG
[5]. The trc1O promoter has one lac operator site and is thus easier to
induce than the more tightly repressed trc2O that contains two
repressor binding operator sites. 
In order for PcpcG2 to function as a light-sensitive promoter, a
histidine kinase CcaS and response regulator CcaR will be required.
Upon exposure to green light, CcaS is auto-phosphorylated and then
phosphorylates CcaR. This changes the DNA binding affinity to CcaR
allowing CcaR to bind to PcpcG2 and activate transcription. This could
be obtained by creating our own plasmids with these genes (already
done). Our hybrid promoter could be cloned into a vector that contains
a fluorescent reporter protein EYFP driven by pTet. Our eventual goals
would be to regulate the <SPAN style="font-style: italic;">nif </SPAN>cluster
once we properly characterize the system.</TD></TR></TBODY></TABLE><TABLE style="text-align: left; width: 100%; height: 200px; margin-left: auto; margin-right: auto;" border="0" cellpadding="5" cellspacing="5"><TBODY><TR><TD style="vertical-align: middle; text-align: justify; width: 30%;">A few
factors have been taken into consideration for the design of the
hybrid promoters. The consensus sequence known as the “G-box”, a highly
conserved direct-repeat motif located 109 and 108 bases upstream of the
transcription start site in <I>N. Punctiforme</I> and S. 6803, is a region
where CcaR binds to PcpcG2 as an activator [4]. We choose to use Ptrc1O
and keep the whole sequence intact because it is shown to be a
constitutive promoter in <SPAN style="font-style: italic;">E. coli</SPAN>,
and has been reported to
be leaky in S. 6803 [5]. By combining the upstream components (CcaR
binding site) of PcpcG2 with Ptrc1O, while keeping the Ptrc1O -35 and -10 sequences intact, our hybrid PcpcG2/trc1O promoter in <SPAN style="font-style: italic;">E. coli</SPAN>
should be highly expressed in (green) light with basal levels of
transcription during the dark.</TD></TR></TBODY></TABLE><TABLE style="text-align: left; width: 100%; height: 200px; margin-left: auto; margin-right: auto;" border="0" cellpadding="5" cellspacing="5"><TBODY><TR><TD style="vertical-align: middle; text-align: center;"><DIV style="text-align: justify;">As described above, the
hybrid
promoter functions the opposite of how
we eventually want it to; namely, we want nitrogenase to be highly
repressed in light conditions (where a lot of oxygen will be present to
poison nitrogenase), and highly expressed in the dark. Therefore we
need to create a genetic switch that functions essentially as a NOT
gate, where in high light conditions, our product (currently EYFP),
will be highly repressed. 
Different switches based on RNA have been well documented in E. coli;
we have several options. First we are testing the repression mechanism
using a basic pTet/TetR, repression mechanism. By expressing TetR, the
system driven by the hybrid promoter should show strong repression in
light, and strong expression in dark. 
We planned our next step to swap out the pTet/TetR repression with
a system of CRISPR interference to block transcription initiation or
elongation based on binding to the template or non-template strand of a
chosen sequence [9]. In our case, we would tune the sgRNA to bind to
the -35 box of our hybrid promoter in order to block transcription. The
reason we wanted to use CRISPRi is because it has been shown to be
functional in both prokaryotes and eukaryotes, although slightly more
complex, this becomes a much more powerful tool once people transition
to eukaryotic environments. We were unable to get far enough this
summer to transition to a CRISPR system.
Meanwhile, once the other half of our iGem team gets nif cluster
working in <SPAN style="font-style: italic;">E. coli,</SPAN> we should
be able to swap out the fluorescent
protein for the nif cluster in our plasmid to regulate the nif cluster
in <SPAN style="font-style: italic;">E. coli</SPAN> with light. In
dark conditions, the switch will be “off”,
namely there should be fluorescence; and in the light, the system
should be repressed. We expect to see higher levels of repression with
our hybrid promoter and greater dynamic fold change.</DIV></TD></TR></TBODY></TABLE><TABLE style="text-align: left; width: 100%; height: 200px; margin-left: auto; margin-right: auto;" border="0" cellpadding="5" cellspacing="5"><TBODY><TR><TD style="vertical-align: top; text-align: justify; width: 70%;">Our
project
goals are to design a hybrid promoter that incorporate the
regulatory elements of PcpcG2 with those of Ptrc1O. Then we will
transition to the more complex CRISPRi mechanism by creating a plasmid
containing the basic trc1O promoter that reversibly blocks
transcription of EYFP. The overarching plan is to be able to regulate
the nif cluster with light, so nitrogenase is only expressed during the
dark, which mimics <SPAN style="font-style: italic;">Cyanothece</SPAN>
51142. If successful, this project would help immensely when eventually
integrating nitrogen fixation into plants by improving efficiency.
During the day, when photosynthesis occurs (with a lot of oxygen as
byproduct) nitrogenase would not be produced, and therefore not wasted;
while during the night, nitrogen fixation can occur. This creates a
temporal regulatory system complementary to Cyanothece 51142’s native
circadian regulation of controlling nitrogen fixation (through
metabolic pathways), increasing the overall chance of success.</TD></TR><TR><TD style="vertical-align: top; text-align: center; font-weight: bold;">Condition</TD><TD style="vertical-align: top; text-align: center; font-weight: bold;">Product</TD><TD style="vertical-align: top; text-align: center; font-weight: bold;">Fluorescence</TD></TR><TR><TD style="vertical-align: top; text-align: center;">Light</TD><TD style="vertical-align: top; text-align: center;">TetR</TD><TD style="vertical-align: top; text-align: center;">Basal</TD></TR><TR><TD style="vertical-align: top; text-align: center;">Hybrid
Light</TD><TD style="vertical-align: top; text-align: center;">TetR</TD><TD style="vertical-align: top; text-align: center;"><SPAN style="color: red;">None</SPAN></TD></TR><TR><TD style="vertical-align: top; text-align: center;">Dark</TD><TD style="vertical-align: top; text-align: center;">EYFP</TD><TD style="vertical-align: top; text-align: center;">High</TD></TR></TBODY></TABLE>By
integrating the light regulation from S. 6803 into E<SPAN style="font-style: italic;">. coli</SPAN>, we are
transitioning to the next step in our big picture, which would be to
get the system working in the cyanobacteria S. 6803. Eventually we want
our system to be able to work in chloroplasts of plants, so nitrogen
fixation will be able to occur in the same location where the most
energy is produced without having to worry so much about the oxygen
byproduct of photosynthesis.<H3 style="text-align: center;">Summer <SPAN style="font-style: italic; color: rgb(204, 0, 0);">PROPOSED</SPAN>
Work Flow</H3>
Week 1: Familiarize self with standard laboratory methods and
procedures. Design primers for transcriptional units and plasmid
backbone.
Week 2-3: Test light regulation in PBJ003 (driven by basic PcpcG2). Use
positive and negative controls as well to ensure system is working as
intended with EYFP fluorescence as our gene expression.
Week 4-5: Swap out basic PcpcG2 in PBJ003 with our hybrid promoter and
test for increased Fluorescence of EYFP.
Week 6-8: Swap out tetR/ptet with CRISPRi mechanism with dCas9 and
sgRNA plasmids. If things go as planned, swap out EYFP with nif cluster
in hybrid promoter driven plasmid.
Week 9-10: Analyze data, comparing fluorescence in different
intensities of light, wavelengths, time intervals, etc.<DIV style="text-align: center;"><H3><A href="https://2014.igem.org/Team:WashU_StLouis/Protocol">Methods</A></H3></DIV>
Our general workflow for each plasmid cloning is as follows:
a. Primer design for backbone and pieces 
b. PCR pieces for amplification
c. Run a gel extraction
d. Gel DNA Recovery
Treat with DpnI and purify if necessary
e. Digestion/Ligation (Golden Gate or Blunt End) 
f. Transformation (Electroporation)
g. Plate and grow overnight
h. Pick colonies and start liquid culture overnight 
i. Freeze and Miniprep cultures
j. Run sequencing PCR to check junctions
k. Run visualization gel for sequencing PCR
l. Send for sequencing
m. Verify sequence products are as intended
With every step there is a chance that things could go wrong, and we
would troubleshoot in order to determine the best course of action
going forward.<H3 style="text-align: center;">Data</H3><TABLE style="text-align: left; width: 100%; margin-left: auto; margin-right: auto;" border="0" cellpadding="5" cellspacing="5"><TBODY><TR><TD style="vertical-align: middle; width: 50%; text-align: justify;">Our
experimental setup is fairly straight forward:
Co-transform cultures with Chromophore, and plate on appropriate
antibiotic resistance.
Wrap culture tubes in foil.
Start liquid cultures overnight in the dark.
Measure the OD600 and dilute to OD0.1
Grow cultures for 2 hours in light and dark conditions respectively.
Induce with light and aTc in deep well plates.
Let cultures grow for 7 hours.
Spin down cultures in centrifuge.
Resuspend in 1 x Phosphate Buffered Saline.
Measure Fluorescence in Tecan.
Absorbance:600nm
Gain: 90
Excitation:485nm
Emission: 528nm
Normalize to OD by calculating Fluorescence/Absorbance.</TD></TR><TR><TD style="vertical-align: top; width: 50%;"><DIV style="text-align: center;">Figure above: Original
Light
induction experiment. No fluorescence seen.</DIV><DIV style="text-align: justify;">After all our cloning
mishaps
we finally were able to test our light induction system. However,
nothing worked as expected. The data represented above is the
fluorescence normalized to the absorbance, or OD of the strains we
grew. We essentially had the same fluorescence as our negative control,
and even when we induced with 250ng/mL of ATC we saw no fluorescence.
This led us to run a few more experiments to troubleshoot the plasmids.</DIV></TD><TD style="vertical-align: top; width: 50%; text-align: center;">
Figure
above: Troubleshooting reporter mechanism.<DIV style="text-align: justify;">Our original positive
control
was under control of constitutive promoter, J23100, and was the only
thing we got to fluoresce. We were concerned with competing translation
start sites so we cloned a plasmid that modified the RBS to allow for
EYFP to fold properly, but fluorescence only got weaker. We then
decided to swap out the pTet promoter and lo- and behold got
fluorescence. Our next step was to swap out the reporter mechanism of
our experimental plasmids.</DIV></TD></TR><TR><TD style="vertical-align: top;"><DIV style="text-align: center;">Figure above: In
the dark, when promoter not active.
No statistical difference w/ and w/o chromophore. Both Hybrid and
Regular systems leaky.</DIV></TD><TD style="vertical-align: top; text-align: center;">
Figure above: In the light, when repressor isactive.
Some difference with aTc present. Hybrid very leaky.</TD></TR></TBODY></TABLE><H3 style="text-align: center;">Results</H3><TABLE style="text-align: left; width: 100%; margin-left: auto; margin-right: auto;" border="0" cellpadding="5" cellspacing="5"><TBODY><TR><TD style="vertical-align: middle; width: 50%; text-align: center;"><DIV style="text-align: center;">
Figure above: Final Light
experimental data</DIV></TD><TD style="vertical-align: middle; width: 50%; text-align: justify;">
Normalized fluorescence is compared to the positive control after
subtracting the negative fluorescence value. The Hybrid 0 aTc results
in light ended up fluorescing even less than the negative control,
implying the system is completely off, which is what we ideally want. 
Another thing to note is that the system isnt working 100% as expected,
but it is still promising.
When comparing light to dark values, the general trend is there, but
the promoters driving the tetR are very leaky. There is so much tetR in around that it is
dominating the system; with no aTc around, there is so little
fluorescence, but once we add a meaningful amount of aTc, we are able
to see results closer to what we expected. In the dark, we want our
fluorescence to be relatively high, comparritive to the positive
control, and we want the system to be highly repressed in the light
(without having to induce with aTc).
When we compare the hybrid promoter to the normal cpcG2 promoter, it
seems like there is a bigger change between dark and light in the
system. In constant light, aTc is known to degrade, which would bind up
less of tetR and cause greater repression. Hence, in the light we do
not
know how much of the repression is due to the strength of the promoter,
or the reduction of aTc. It is apparent that the hybrid promoter is
leakier than the original- even with a high concentration of aTc we see
very little fluorescence in the light. A comparable amount of aTc would
have degraded in the light of both the hybrid and normal promoter, but
it is hard to say if the stark difference of both the Hybrid and normal
system with aTc would
have been as apparent when taking into account the degradation of aTc..</TD></TR><TR><TD colspan="2" rowspan="1" style="vertical-align: top;">Our
next steps would be to modify the RBS of TetR in order to find a
saturation point where the system is most tightly on and off without
aTc. We ideally do not want to use aTc at all, as chemical induction is
troublesome and won't be very reliable if degraded by constant light.
After we modified the system to be working without aTc around, we can
look into swapping EYFP with the <SPAN style="font-style: italic;">nif</SPAN>
cluster to have the genes expressed only in the dark.</TD></TR></TBODY></TABLE><DIV style="text-align: center;"><H3>Citations</H3></DIV>
1. Rogers, Oldroyd. Journal of Experimental Botany March 2014
doi:10.1093/jxb/eru098
2. Hirose, Shimada, et al. PNAS July 15, 2008 vol. 105 no.28
3. Lutz and Bujard Nucleic Acids Research, 1997, Vol. 25, No. 6
4. Hirose, Narikawa, et al. PNAS May 11, 2010 vol. 107 no. 19
5. Huang et al. Nucleic Acids Research, 2010 Vol. 38 No. 8
6. Tabor, Levskaya, et al. J. Mol. Biol. (2011) 405, 315-324
7. Kim, White, et al. Molecular Systems Biology 2006
doi:10.1038/msb4100099
8. Brantl, Wagner. J. Bacteriol.  2002,
doi:10.1128/JB.184.10.2740-2747.2002.
9. Larson , Gilbert, et al. Nature, Oct. 2013 vol. 8 no. 11
10. Abe, Koichi, et al. Microbial biotechnology (2014).<DIV class="footer"><CENTER><TABLE style="background-color:#FFF" width="80%"><TBODY><TR><TD style="padding-left: 15px; padding-right: 15px"><CENTER><H1> Special thanks to our 2014 Sponsors: </H1><H3> If you want to sponsor us, please contact us below! </H3></CENTER></TD></TR></TBODY></TABLE><TABLE style="background-color:#FFF" width="80%"><TBODY><TR><TD style="padding-left: 15px; padding-right: 15px"><P><CENTER> © Washington University in St. Louis iGEM 2014 | 
                       <A href="mailto:washu.igem@gmail.com" style="color:green"> Contact Us </A> | Follow Us on Social Media
         </CENTER></P></TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:WashU_StLouis/Project/light">http://2014.igem.org/Team:WashU_StLouis/Project/light</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:WashU_StLouis/Project/light" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:WashU_StLouis/Project/light" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:WashU_StLouis/Project/light&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:WashU_StLouis/Project/light&amp;oldid=394865" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>