Team:Genspace/Safety
<!DOCTYPE html>
Gowanus SuperFUNd - Genspace iGEM 2015
Toggle navigation
Home
Accomplishments
PROJECT
Background
Biosensor
Mining the Microbiome
Design
parts
NOTEBOOK
ATTRIBUTIONS
TEAM
collaborations
HUMAN PRACTICES
SAFETY
Safety
What were the safety issues in the Biosensor part of our project?
Our project is divided into two parts. The first part, building a biosensor for sewage that can be used by the community living around the Gowanus Canal to monitor water quality, is very straightforward and presents typical risk for iGEM projects. The bacterial chassis used was E. coli K12 derivative DH5alpha and all the parts used present no risk to humans.  Since the parts and the chassis are from Group 1 organisms there were no unusual safety issues posed by this part of the project. Use of good microbiological practices would be sufficient to minimize risk.
The biosensor would, ideally, be deployed in a device placed along the canal with the engineered bacteria contained by a membrane that allows small molecules to pass through but not micro organisms. We have identified such a filter, used by the Imperial College iGEM team in 2014. The difficulty will be to design a device that shields the filter from mechanical damage yet allows the color change to be visualized. Once the sewage has been detected, the device needs to be discarded safely and replaced.
How did we address the safety concerns in the Biosensor part of our project?
One way to make the device safer would be to add a kill switch that would be activated by salt concentrations found in the ocean, which is the eventual place that bacteria would end up if they were released into the canal accidentally. Unfortunately due to the short timeframe of iGEM we were unable to build this device although we researched it. Another way to minimize the release of genetically modified organisms into the canal would be to place the biosensor bacteria into a kit where the bacteria are in a container that canal water is added to. The kit would contain bleach for decontamination of the device after use. The downside of this approach is that the community expressed a wish for real-time visual indicators along the canal.
What were the safety issues in the Microbiome Mining part of our project?
The second part of our project involves mining the canal microbiome for organisms and biochemical pathways that might prove useful for detoxification of pollutants. This required a bit more thought and strategizing since Genspace is a BSL1 facility and we cannot amplify samples of uncharacterized microbes from the environment.  In addition, the canal water has environmental toxins in it and so it was desirable to avoid direct contact with the water.
After extraction of DNA from the samples, the purified DNA did not pose a risk. However, it was originally our intention to preserve this genome diversity by cloning it into a cosmid library in E. coli K12, then doing functional screening.  After thinking about it, we realized that if the pathways contained promoters and ribosome binding sites that our strain of E. coli recognized, there might be production of products from the library during the process of creating the library and growing up individual clones for testing. Amplifying a single clone might result in production not only of beneficial functions and molecules, but also potentially harmful ones. , we realized that we might inadvertently clone some pathways producing toxins into the K12, since we had no control over which DNA ended up in the cosmid library. This presented a risk that we might multiply a toxin-containing organism during library generation.
How did we address the safety concerns in the Microbiome Mining part of our project?
Previously, Genspace had launched a citizen science project called Enquete Gowanus where we collected samples from the canal bottom sludge, extracted DNA from them using a MoBio soil kit, and sent the DNA for sequencing analysis. We followed this protocol for sample handling, which was reviewed and accepted by our Advisory Board:
Gloves, rubber boots and full body Tyvek suits were worn. Two sets of samples were collected, one set in glass Ball jars and one set in polyprop 50ml tubes for microbiome sequencing analysis. The collected samples were sealed into biohazard bags and transported to Genspace. Data from our previous Canal collection shows no exotic pathogens in the Canal samples (unlike the subways!), but typical sewage bacteria such as E. coli and pseudomonas. There were also some intriguing results that pointed to organisms capable of metabolizing carbon compounds found in coal tar oil, one of the pollutants that was dumped into the canal for many years.
In order to mine the microbiome, there were several strategies considered. We consulted a member of Genspace’s Advisory Board, Dr. Claudia Mickelson of MIT.  Direct growth and testing of the sludge microbes was not permissible in a BSL1 environment. So on Dr. Mickelson’s advice we decided not to do direct functional testing on samples but to extract DNA from samples using a protocol that the Brady lab at Rockefeller University has used to mine soil samples for new antibiotic synthesis pathways. This consisted of collecting samples in Mason jars, using 250g in the protocol that was optimized to extract DNA keeping relatively long pieces intact, and gel-purifying them. At Genspace the samples were sprayed down on the outsides with 70% alcohol. The samples were sealed into smaller biohazard bags and stored at 4C prior to DNA extraction. Under constant supervision by Dr, Jorgensen, adult team members removed aliquots of about 250g each from each of the 14 jars. The members wore PPE (gloves, lab coats and eye protection) and the aliquots were immediately placed in the lysis solution and agitated to disrupt the cells.  As usual, all pipettes and labware used were disposed of in the biohazard bin, which is taken away for incineration by a professional service. Blenders were decontaminated with bleach solution.
Although we plan to eventually use the purified DNA to make a cosmid library that will hopefully contain long stretches of DNA with entire metabolic pathways, we could not let the team do it at Genspace because of the safety concerns above. Again, we have Dr. Mickelson to thank for providing advice. So we had to think of a new strategy. So we decided to use PCR to probe the extracted DNA to see if there were any sequences that seemed homologous to those characteristic of a particular type of metabolic pathway.  With the help of Brady lab publications, we were able make some primers that we could use on the extracted DNA prior to library production to show that some of the samples had DNA that could possibly contain pathways for new antibiotics. Dr. Mickelson saw no unusual safety issues with this protocol. In the future we hope to partner with a BSL2 facility to create the cosmid library and screen it for functionality.
Place sticky footer content here.
