Team:Genspace/Description
<!DOCTYPE html>
Gowanus SuperFUNd - Genspace iGEM 2015
Toggle navigation
Home
Accomplishments
PROJECT
Background
Biosensor
Mining the Microbiome
Design
parts
NOTEBOOK
ATTRIBUTIONS
TEAM
collaborations
HUMAN PRACTICES
SAFETY
Background
What's In Brooklyn's Gowanus Canal? Used with permission from Hakai Magazine
The Gowanus Canal, located in Brooklyn, NY is one of the most heavily contaminated water bodies in the nation. This 1.8 mile long, 100 foot wide, canal was built in the 19th century and historically was home to many industries including manufactured gas plants, cement factories, oil refineries, tanneries, and chemical plants. After nearly 150 years of use, the canal has become heavily contaminated with PCBs, heavy metals, pesticides, volatile organic compounds, sewage solids from combined sewer overflows, and polycyclic aromatic hydrocarbons (PAHs). in 2010 it was designated by the Environmental Protection Agency (EPA) as a SuperFund site. Efforts are now underway to clean up some of the former industrial sites along the canal’s banks, reduce sewage overflows, and improve water quality.
-From the RiverKeeper  Website
The borough of Brooklyn has undergone a renaissance, driving up real estate prices and luring residential developers. The vision of a cleaned-up waterway flowing through the Gowanus neighborhood has developers vying for the permits to build high-rise residential units along the banks of the canal. These proposed developments are controversial. They have been challenged by residents of the neighborhood for many reasons, including  the concern that new tenants would overwhelm schools and subways, and that the buildings themselves — 12 stories in spots — would perniciously transform the low-rise mingling of factories and row houses the current canal community has come                     to love . One of the main fears voiced by by the Gowanus community is that addition of more residential units and the accompanying sewage production would adversely affect the health of the canal, which is already prone to contamination following rainstorms. “The infrastructure is not there,” said Marlene Donnelly, a designer who has lived in the neighborhood for two decades. “My biggest concern is that the project will add 700 apartments with toilets going into the canal. They’re inundating a                         system that’s overtaxed.” ( NY Times )
Considering the videos we have seen on line, these concerns are well-founded and because there is so much money at stake for developers, we felt that they might try to hide the consequences of their construction.  After engaging with several stakeholder groups (see our Human Practices section) we decided to use synthetic biology to 1) help alert residents in real time when sewage enters the canal and 2) find value in the current polluted state of the canal                     by mining the bottom sludge for novel organisms useful for decontamination of industrial waste.
Our project has two parts.
Biosensor for Sewage
Based on input from the residential community surrounding the canal, we attempted to construct three different biosensors for sewage based on quorum sensing or nitrogen sensing coupled to production of a colorful reporter. We tested one of our three biosensors in the lab and it performed as expected. In consultation with the Gowanus community, landscape designers, and homeland security experts, we brainstormed around a biosensor that could be immobilized within containers deployed along the canal, providing the community with a visual indicator of water quality.
Microbiome Mining
Genspace has an ongoing project to assay the microbiome of the canal bottom sludge. We took advantage of the protocols that had been developed for this project and collected bottom mud samples from 14 sites along the canal chosen for their different conditions. We used a protocol published by Dr. Sean Brady’s lab at Rockefeller University to safely generate high molecular weight DNA that can be queried via PCR and also used for eventual production of a cosmid library. The goal was to mine the canal bottom extremophile DNA for new and useful metabolic and detoxification pathways since the organisms thriving in pollution from coal tar, paint factory solvents, etc. might have evolved to metabolize toxic compounds and may be useful for remediation. We completed a proof-of-principle experiment by using degenerate PCR primers that recognize antibiotic synthesis pathways. Further mining via PCR will be guided by pathway analysis of next-gen sequencing data from the ongoing Genspace microbiome study.
Biosensor
Part selection and sensor design
Inputs: After the group decided to build a biosensor for sewage pollution in the Gowanus Canal, the next obvious question followed; How? Our first task was to identify detectable molecules the are characteristic of human waste to serve as inputs for our biosensor. Two inputs seemed to be good candidates:
1. Quorum sensing
We investigated two quorum sensing systems for this sensor: the N-Acyl Homoserine Lactone (AHL) sensitive Lux pathway, and the autoinducer-2 (AI-2) sensitive Lsr pathway.
Lux
The AHL-sensitive Lux pathway was characterized in A. fischeri, but is also present in various bacterial species, including Pseudomonas aeruginosa, commonly found in large numbers in human waste and sewage .  Large populations ofA. fischeri express bioluminescence.  However, the bioluminescence from a single bacterium would be too low to detect, so in low populations their luminescence is downregulated to conserve energy.  The Lux system relies on two parts: the Lux promoter, a very weak constitutive promoter; and the LuxR protein, which, in the presence of AHL, binds the Lux promoter and enhances its affinity for RNA polymerase.  Both the Lux promoter and LuxR protein were available in the registry, which made them suitable for immediate use.
Lsr
The AI-2-sensitive Lsr pathway relies on a transmembrane protein (LsrACDB) to import AI-2, which is then phosphorylated by LsrK, allowing it to bind the repressor protein LsrR, which detaches from the p LsrA and pLsrR promoters, enabling transcription.  This pathway is used in E. coli to regulate biofilm formation[ source ] and is present in many other bacterial species as well.  However, few components of this pathway were available through the Registry, and the ones that were available did not appear to function as intended.  As a result, we had to have many of the relevant parts synthesized, making use of IDT’s generous offer .  Following the delivery of these parts, we used Synbiota’s RDP system to quickly assemble several large constructs ( BBa_K1799020 , BBa_K1799021 , and BBa_K1799022 ) in a matter of hours, which were then used to begin to characterize the Lsr quorum sensing pathway.
2. Nitrate sensing
 After researching common chemicals in feces, we saw it has been reported that  there is a 100% increase of nitrate in water from 20-40 mg/l. A quick search showed that there are two parts listed within the IGEM registry that are able to detect nitrate: Part BBa K216005 and Part BBa_K381001. The first part is a PyeaR promoter, responsive to nitrate, nitrite and nitric oxide. Unfortunately, this part is not available. The second part, BBa_K381001, consists of PyeaR and GFP compound, and was in the 2015 registry. We could use those parts to construct a similar circuit in which NsrR is constituitively produced. PyeaR is normally repressed by NsrR,but when Nitrate or Nitrite enter the cell they are converted to Nitric Oxide which binds to NsrR, halting the repression and allowing the production of any pigment or fluorescence gene or operon downstream of the PyeaR promoter.
Outputs: In discussions with community representatives, we found that a real time indicator of human waste pollution was desirable. A bright color change indicating human waste pollution would act as a visual indicator and serve not only to alert the community but also to serve as a rallying point around remediation efforts. They requested a bright color such as red, which would also psychologically signal danger from an excess of pollution. A colored substance that would remain within the device and not diffuse out would also be desirable. (find out if RFP size is consistent with the pore size of the filter we were looking at).
Combining these three inputs with a colored output would be achieved by building the following three devices:
Biosensor 1. Nitrate from sewage
Biosensor 2. AHL from Pseudomonas aeruginosa
Biosensor 3. AI-2 from Escherichia coli
Build strategy
Our build strategy had a unique component- we decided to use the Rapid DNA Prototyping (RDP) System from Synbiota to assemble and test some of our parts and  prototype biosensors before putting them in BioBrick format. The RDP system has the advantage of speedy assembly of multiple sequential parts, and it is technically easy to use even for amateur synthetic biologists and high school students (the two major contributors to our team).  We feel that our pioneering use of the RDP system represents a step forward for citizen science, and we have expanded on that theme in the Human Practices section of our wiki.
Our first step was to test-drive the RDP system with a simple biosensor build. The RDP: Lux + Rudolph device was constructed using the Synbiota Rapid DNA Prototyping (RDP) protocol in order to produce a prototype biosensor that detects L-Acyl-Homoserine-Lactone (AHL) and produces red fluorescent protein (RFP) in response.  This mechanism is as follows: AHL is a small molecule (213.23 Daltons) produced by some bacteria that have evolved quorum sensing circuits to detect the presence of high concentrations of bacteria.   We started with the naturally-occurring quorum-sensing circuit where the LuxR regulatory protein binds with AHL and stimulates promotion by the pLux promoter.  In the wild, this can trigger behavior such as biofilm formation. Since the community around the canal asked us for a biosensor that would produce a visual indication of high bacterial presence, we realized that placement of an RFP translational unit based on the ‘Rudolph’ RFP submitted by Genspace at iGEM 2014 ( http://parts.igem.org/Part:BBa_K1429001  ) downstream of the pLux would result in a color change in the presence of bacteria producing AHL.
RDP-compatible parts for the upstream Lux-based device (based on BBa_J37019 ) and the downstream Rudolph RFP (based on BBa_K1429001 ) were created using a protocol supplied by Synbiota and template DNA from the Biobrick distribution kit.  For the RFP translational unit BBa_K1429001 we designed the primer to omit the RBS of the original part, and instead replaced it during the build with an RBS supplied in the RDP kit. The three parts assembled were BBa_J37019 with RDP ends, the RBS from the RDP kit, and the ORF portion of BBa_K1429001 with RDP ends. The build went very well, with a large number of colonies containing the proper construction. The resulting biosensor was characterized  using AHL as the input and red fluorescence (553/570). PCR primers were used to give it biobrick ends, and it was cloned into pSB1C3 and submitted as part BBa_ K1799018 ,
A combination of more than one biosensor system that each detect a different component of sewage would make a much more powerful and accurate sensor.
The next step was to design and order DNA for all the parts of the other two biosensor systems that were not present in the registry. The strategy was to order the DNA for each part such that it could either be submitted to the iGEM registry in biobrick format in pSB1C3 or PCR primers could be used to put it into RDP assembly format. With this in mind, we created the following separate parts and submitted them to the registry:
AI-2 system- parts involved in the detection of AI-2:
K1799000        pLsrA-2  
K1799001        LsrK ORF
K1799002        LsrR ORF
Nitrogen detection system:
K1799015        pYeaR (twin to BBa_K216005  which was not available)
K1799016        NsrR, repressor of pYeaR
Modularization of parts involved in the transport of AI-2 into the cell. We theorized that if we could somehow control expression of components of the  transport system  that perhaps the sensitivity of the AI-2 biosensor could be increased or decreased. Furthermore, we submitted parts representing both the transport system components as the occur naturally, with embedded RBS and overlapping reading frames, and also the whole system engineered as modularized components with all internal signals eliminated. The intent was to test both iterations of the system to see which performed better and was easier to titrate.
K1799003        wild-type LsrA ORF
K1799004        wild-type LsrC ORF
K1799005        wild-type LsrD ORF
K1799006        wild-type LsrB ORF
K1799007        modularized LsrA ORF
K1799008        modularized LsrC ORF
K1799009        modularized LsrD ORF
K1799010        modularized LsrB ORF
Additional Biosensor assemblies
During the iGEM competition period, we assembled and tested two additional biosensors circuits (one built by RDP assembly and much simpler one by a modified 3A assembly).
Biosensor 2. BBa_K1799019   (input = AHL outpu t= beta galactosidase, good for quantitative testing)
(3A assembly)
Biosensor 3. BBa_K1799022  RFP under the control of pLsrA
 
(RDP Assembly)
Experimental Results
In order to more accurately quantitate the effectiveness of the input part of the circuit, it was desirable to have an output that could be easily quantified by the equipment available at Genspace. Therefore we decided to put the LacZ gene under the control of Lux. This was the rationale behind the construction of BBa_K1799019  (biosensor 2 above)
Evaluation of BBa_K1799019 (3A-Assembled Lux + LacZ Device
A Beta-Galactosidase Assay (aka “Miller Assay”) was carried out as described by Marian Price-Carter from the Roth Lab at UCLA ( http://rothlab.ucdavis.edu/protocols/beta-galactosidase-3.html ).  Different concentrations of AHL were found to impact both cell growth profiles as well as Beta-Galactosidase activity levels.   We also found that constant exposure to high levels of AHL (1uM) through repeated 1:100 dilutions had                     no discernable impact on cell growth compared to negative controls (0uM), but had the unexpected effect of minimizing Beta-Galactosidase production – even relative to negative controls.  Indeed, we found the unit-step change in AHL concentration                     to be the salient factor in increased expression of Beta-Galactosidase.  By “unit-step” change, we mean making a 1:100 dilution from an overnight with no AHL into an LB solution with the appropriate antibiotics and a greater-than-zero                     AHL concentration.
The AHL concentrations reported here were determined based on initial attempts to find a concentration range where cell growth was neither substantially delayed nor was Beta-Galactosidase production only modestly increased.  Consistent with previous characterizations involving the Lux promoter (for example, see   http://parts.igem.org/Part:BBa_R0062:Experience  ), we found unit-step AHL concentrations of 0.1nM, 1nM and 10nM to span this range of interest.  A negative control with no AHL and a positive control maintained in 1uM AHL were also included for comparative purposes.
Overnights were first prepared in 0uM and 1uM AHL concentrations with appropriate antibiotics in LB media in a rotating incubator at 37C.  The 1uM AHL overnight was diluted 1:100 into fresh LB media with appropriate antibiotics and with 1uM AHL to serve as the positive control.  The overnight with no AHL was diluted 1:100 into fresh LB media with appropriate antibiotics and either 0nM, 0.1nM, 1nM or 10nM AHL.  The 0nM AHL solution served as the negative control.  Each hour after dilution, 1.5 mL samples were taken and put immediately on ice and stored at 4C to create triplicate 0.5mL samples for use in the Beta-Galactosidase assay.
Cell growth curves as measured by OD600 were obtained and are plotted below.
Figure 1: OD600 versus Hour
There is no substantial difference between the positive and negative controls.  The growth curve associated with the 0.1nM solution is also substantially the same as the controls.  In contrast, the growth curves for 1nM and 10nM are considerably delayed compared to the controls.  However, after 14 hours, all solutions under study have approached stationary phase.
The Beta-Galactosidase Activity (measured in Miller Units) is plotted below.
Figure 2: Beta-Galactosidase Activity versus Hour
As observed in prior work (e.g.,   http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3145315/ ) the high levels (>20,000) of Beta-Galactosidase activity associated with AHL concentrations of 1nM and 10nM are correlated with delayed cell growth.   While the negative control has a relatively low level of Beta-Galactosidase activity, as expected, it is remarkable to note that the positive control has a lower level still.  In a quorum sensing context, we might make sense of this as follows: the constant presence of a high-level of AHL does not require the same action as a sudden change in AHL.  In this case, some refractory mechanism would                         appear reasonable.  The unit-step change of AHL concentration to 0.1nM results in increased activity that attenuates to negative control levels after approximately 5 hours, consistent with being on the low-end of the AHL scale.  In                         contrast, the 1nM and 10nM concentrations of AHL result in dramatic increases in activity.  The onset of activity increase for 1nM appears to be slightly delayed compared to 10nM, which could be related to the effective diffusion                         delay of AHL through the outer membrane pores.  However, after 3 hours, the activity for 1nM is commensurate with 10nM and eventually is greater beyond hour 8.  This is due to an intrinsic difference in behavior as a function                         of AHL concentration that is explored next.
The number of Beta-Galactosidase molecules at each hour is estimated by multiplying the Miller Units (a reasonable proxy for the number of Beta-Galactosidase molecules per cell as noted by Garcia et al., 2011) by the OD600 value scaled by 8x10 8.  Using a three-sample moving average to ride out statistical fluctuations in each time series, we find the results shown below.
 
Figure 3: Beta-Gal Molecules versus Hour
This plot shows that the concentration of 1nM AHL results in the highest yield of Beta-Galactosidase, almost a factor of four times greater than the negative control.  These results are of interest in a variety of applications where the total yield of a synthesized product must be either maximized or set to a target value (often subject to a variety of other constraints).  Further consideration of this type of phenomena is considered in the figures below.
Figure 4: Beta-Gal Molecules & Companion OD600 versus Hour
The yield curves in these concentrations under study have a number of features in common.  The yield over the first three hours increases at the lowest rate (r1).  After three hours, the rate often increases (r2>r1) until a plateau is reached (P).  These three parameters (r1, r2 and P) are sufficient to classify the observed yield curves.  In the case of the positive and negative controls – as well as the 0.1nM concentration – the plateau is reached about an hour after entering stationary phase.  This may also be true for the 10nM concentration as 14 hours of data capture was barely sufficient to reach stationary phase.  However it is noteworthy that the 1nM concentration appears to have reached its plateau of terminal yield after only 11 hours (i.e., even while still in mid-log growth).
Evaluation of the Lsr-based Inverter System
In order to evaluate the LsrR regulatory protein and the pLsrA2 promoter, we devised a logical inverter system.  In the absence of the Lsr-related components, a control system expressed a red fluorescent protein (RFP) in response to increasing levels of the inducer molecule IPTG.  This is accomplished through the conformational effect of IPTG on the LacI regulatory protein which reduces binding to the Lac promoter and increases expression of the RFP.  Hence increasing the amount of IPTG increases the amount of RFP.
In the Lsr-based inverter system, transcription of the LsrR regulatory protein is under control of the Lac promoter and transcription of the RFP is under control of the pLsrA2 promoter (which is inhibited by LsrR).  In the absence of IPTG, LacI binds to the Lac promoter and inhibits LsrR transcription.  This facilitates RNA polymerase binding to the LsrA2 promoter and ultimately increases expression of RFP.  When IPTG is present, the binding of LacI to IPTG results in increased expression of LsrR, which inhibits the pLsrA2 promoter and in turn reduces the expression of RFP.  Hence the inversion effect: increasing the amount of IPTG reduces the amount of RFP.
We constructed three plasmids using the Synbiota Rapid DNA Prototyping (RDP) assembly protocol.  The first plasmid, p1, had an Ampicillin resistance and a medium-copy-number origin of replication.  The p1 plasmid contained a single device:  a constitutive promoter followed by a strong Ribosome Binding Site, followed by the coding sequence for the LacI regulatory protein.  The p1 plasmid was used in both the control system as well as the Lsr-based inverter system under study.
The second plasmid, p2, had a Chloramphenicol resistance and a high-copy-number origin of replication.  The p2 plasmid contained a single device: a Lac promoter followed by a medium strength Ribosome Binding Site, in turn followed by the coding sequence for a red fluorescent protein (RFP).  The p2 plasmid was only used in the control system (i.e., p1+p2).
The third plasmid, p3, had a Chloramphenicol resistance and a high-copy-number origin of replication.  The p3 plasmid contained two devices.  The first device was a Lac promoter followed by a medium strength Ribosome Binding Site, in turn followed by the coding sequence for the LsrR regulatory protein and a terminator.  The second device was a pLsrA2 promoter followed by a medium-strength Ribosome Binding Site, in turned followed by the coding sequence for a red fluorescent protein (RFP).  The p3 plasmid was used only in the LSR-based inverter system under study (i.e., p1+p3).
The following steps were carried out for each individual plasmid:  Synbiota RDP assembly, transformation into the DH5-alpha strain of E. coli, streaking onto LB-agar plates with selective antibiotics, colony PCR, overnight incubation using selective antiobiotics, glycerol stock preparation and plasmid min-prep.  The DH5-alpha strain of E. coli was chosen because it was LuxS-negative.  As such, the E. coli in the systems under study would not produce the AI-2 quorum sensing molecule and interfere with the intended experiment.  The E. coli transformed with p1 were rendered competent and transformed with p2 and p3 to obtain p1+p2 (control) and p1+p3 (Lsr-based inverter) two-plasmid systems, respectively.  The doubly-transformed cells were plated, colony-selected, and had glycerol stocks prepared.  Then fresh 5mL overnights of the control and Lsr-test systems were commenced in LB plus Ampicillin and Chloramphenicol and either 0mM or 1mM IPTG (for a total of four overnights).
On the following morning (long after reaching stationary phase), 4mL aliquots of each overnight were spun down and double-washed with PBS and put on ice to serve as overnight specimens for fluorescent analysis.  Each of the four overnights also underwent a 1:100 dilution into 5mL of fresh LB media with identical antibiotics and IPTG concentrations and grown to mid-log phase (OD600 between 0.3 and 0.4).  These were also spun down and double-washed with PBS and put on ice to serve as mid-log specimens.
Fluorimetric analysis was conducted at Columbia University on a BioTek Synergy H1 Hybrid Reader (and the Genspace iGEM team gratefully acknowledges the generous help of the Columbia University iGEM team in this regard).  The excitation and emission wavelengths for the fluorescent analysis were 503nm and 607nm, respectively (see   http://link.springer.com/article/10.3103%2FS0096392508030036#page-1 ).  The eight samples were transferred in triplicate to a 96 well plate in 100uL aliquots per well.  A 100uL aliquot of PBS served as the blank reference against which OD600 and fluorescent measurements were comparatively made (i.e.,                         by subtracting the OD600 and fluorescent readings of the blank from each aliquot under study).  The comparative fluorescent reading of each aliquot was divided by the comparative OD600 reading of the same aliquot in order to obtain                         a measure of “per cell” fluorescence.  The triplet samples were then averaged and the standard deviation calculated to obtain the results shown below for the overnights:
Figure 1:  Fluorescence Measurements (Overnight results)
The average values of the fluorescence measurements shown above by the blue bars demonstrate the intended effect.  As expected, the control system exhibits increased levels of fluorescence as the IPTG is increased from 0mM (“IPTG-“) to 1mM (“IPTG+”). Conversely, the Lsr-based system (“LSR”) exhibits reduced levels of fluorescence as the IPTG is increased.   The error bars denote the range of +/- one standard deviation and is largely consistent across all samples, except for the Lsr-based system with 0mM IPTG (where each of the three fluorescent readings were similar according to the plate reader).
While the absence of overlap in error bars seen above lends credence to the validity of these results, additional steps can be taken in the future to improve the statistical character of this experiment.  These include the use of stronger Ribosome Binding Sites in plasmids p2 and p3 (to increase the degree of fluorescence), re-suspending the spun-down cells with fractional volumes of PBS to increase the effective cell concentration and increase the total fluorescence in a 100uL aliquot, and also to increase the statistical processing gain of the experiment by using more than three samples per test condition (e.g., ten or more samples).
The figure below demonstrates the results for the mid-log growth:
Figure 2:  Fluorescence Measurements (Mid-log growth results)
As before, the average values shown by the blue bars demonstrate the intended effect (i.e., increasing IPTG increases RFP for the control, while the opposite is true for the Lsr-based inverter).  However, the error bars are now seen to overlap, which erodes confidence in this conclusion.   The smaller number of cells in each aliquot when grown to only mid-log phase (compared to stationary phase) limits the ability of the fluorimeter to accurately measure fluorescence as it will be operating closer to its noise floor.  It is important to note that the same steps noted above to improve accurate system measurements for overnights will also apply to mid-log growth (i.e., stronger Ribosome Binding Sites, greater cell concentrations in each aliquot, and increased sample processing gain).
Evaluation of the RDP: Lux+Rudolph Device
The RDP: Lux + Rudolph device was constructed using the Synbiota Rapid DNA Prototyping (RDP) protocol in order to evaluate a prototype for the biosensor that detects L-Acyl-Homoserine-Lactone (AHL) and glows red in response.  This is accomplished as follows: the AHL small molecule (213.23 Daltons) is produced by some bacteria in quorum sensing applications to signal the presence of high concentrations of bacteria.  The LuxR regulatory protein binds with AHL and stimulates promotion by the pLux promoter.  Placement of a red fluorescent protein (RFP), such as the Rudolph RFP submitted by Genspace at iGEM 2014 ( http://parts.igem.org/Part:BBa_K1429001  ), downstream of the pLux promoter (along with a suitable Ribosome Binding Site) results in increased fluorescence in the presence of AHL and completes the device.
RDP-compatible parts for the upstream Lux-based device (based on BBa_J37019) and the downstream Rudolph RFP (based on BBa_K1429001) were created using a protocol supplied by Synbiota and template DNA from the Biobrick distribution kit.  The rest of the RDP assembly made use of parts supplied by Synbiota (antibiotic resistance, origin of replication, Ribosome Binding Sites, etc.).  After assembly, the plasmids were transformed into competent Top10 cells, plated, colony selected, and incubated overnight in LB plus appropriate antibiotics at 37C in a rotator.  Following glycerol stock preparation and plasmid miniprepping, two fresh overnight incubations were commenced in LB plus appropriate antibiotics: one with an AHL concentration of 1uM and the other with no AHL.
The following morning, dilutions into fresh media were made as follows: 1:100 dilution of the 1uM AHL overnight into fresh LB media with appropriate antibiotics and 1uM AHL (positive control), 1:100 dilution of the 0uM AHL overnight into fresh LB media with appropriate antibiotics and 0uM AHL (negative control), and 1:100 dilution of the 0uM AHL overnight into fresh LB media with appropriate antibiotics and 1uM AHL (test system).  The three systems were sampled once an hour for five hours.  Each sample was pelleted and double-washed in phosphate buffered saline (PBS) and put on ice.
Fluorimetric analysis was conducted at Columbia University on a BioTek Synergy H1 Hybrid Reader (and the Genspace iGEM team gratefully acknowledges the generous help of the Columbia University iGEM team in this regard).  The excitation and emission wavelengths for the fluorescent analysis were 532nm and 588nm, respectively.  (Note that these are different than the 550nm/570nm wavelengths called out on   https://www.dna20.com/eCommerce/catalog/datasheet/54  for the Rudolph RFP.  The Synergy H1 Hybrid Reader warned about having excitation and emission wavelengths that were too close together: the employed 532nm/588nm wavelengths were a close compromise).  The fifteen samples (positive,                         negative and test, spanning five hours) were transferred in triplicate to a 96 well plate in 100uL aliquots per well.  A 100uL aliquot of PBS served as the blank reference against which OD600 and fluorescent measurements were comparatively                         made (i.e., by subtracting the OD600 and fluorescent readings of the blank from each aliquot under study).  The comparative fluorescent reading of each aliquot was divided by the comparative OD600 reading of the same aliquot in order                         to obtain a measure of “per cell” fluorescence.  The triplet samples were then averaged and the standard deviation calculated to obtain the results shown below for the overnights:
Figure 1:  Fluorescence Measurements (Hourly results)
As expected, the positive control (in 1uM AHL) shows a clear, three-fold increase in fluorescence compared to the negative control (in 0uM AHL).  In contrast, the test system is seen to exhibit fluorescent behavior similar to that of the negative control over the five hours of study.  The test system is seen to be different than the negative control system when cell growth is considered in the figure below:
Figure 2:  Cell Growth (Hourly results)
The unit-step introduction of AHL in the test system has a delaying effect on cell growth, whereas the constant presence of the same, high concentration of AHL in the positive control does not impede growth relative to the negative control (where AHL is consistently absent).
Over enough time, we expect the fluorescent behavior of the test system to converge to that of the positive control (since the positive control was created in a manner identical to that of the test system).  A concentration of 1uM AHL over five hours is insufficient to demonstrate convergence.  The behavior of the system as a function of lower AHL concentrations is also an important question as it may better reflect the environmental circumstances that our biosensor would naturally encounter.
In summary, these measurements provided a first confirmation of our prototype device regarding fluorescent responsiveness to AHL concentration (i.e., greater fluorescence in the sustained presence of AHL compared to its absence).  These measurements also guided the next steps in our evaluation of the prototype including: 1) use of different reporters, such as LacZ, 2) evaluation of system behavior over timescales longer than four hours, 3) evaluation of system behavior in response to different concentrations of AHL.  
Mining the Microbiome
Project Background
In the summer of 2014, Genspace initiated a project, Enquete Gowanus, to explore the microbiome of the Gowanus Canal. Just as Craig Venter sailed the seven seas to discover new and useful life forms, the citizen scientists of Genspace ventured into the wilds of the Canal to catalog and study the microrganisms hardy enough to survive in a Superfund site.We partnered with the Gowanus Canal Conservancy, the landscape architecture firm Nelson-Byrd-Wolz, and Dr. Chris Mason’s lab at Weill-Cornell Medical College. To date, four separate collections (summer 2014, winter 2014, spring 2015 and summer 2015) of bottom sludge from 14 different sites along the canal have been made. DNA was extracted from the samples by Genspace members and the microbiome explored through whole shotgun sequencing at Weill-Cornell Medical College. The samples were included in the PathoMap project, an ambitious effort by Dr. Mason's lab to map the microbiome of New York City. The data from the first sampling expedition (summer 2014) was published in the journal Cell Systems  and can be explored in an interactive map .
Our iGEM team decided to build upon this work by making a cosmid library of DNA from the Gowanus Canal. Cosmids are extrachromosomal elements similar to plasmids except much larger. They are able to contain inserts of up to 40 or more kb, large enough to contain whole metabolic pathways, Making a cosmid library from high molecular weight DNA fragments from the Gowanus Canal would be useful for several reasons. It would preserve the DNA in a form that could be replicated, it would allow for functional screening of the E. coli containing the cosmids for interesting new properties such as toxic compound degradation or antibiotic synthesis, and it would allow screening of the library for specific types of enzymes via degenerate  PCR primers to conserved sequences.
Our goal was to identify and archive useful DNA sequences from the Gowanus Canal extremophiles by adding them to the registry.
Sampling Methodology
Our iGEM team used the long tube sampling procedure to collect samples of bottom sludge in gallon glass jars. We felt that there was less opportunity for splashing than with the trap-jaws method.  The large amount of sample was necessary since we wanted to not only extract DNA for eventual sequence analysis, but also to use extracted DNA to create a cosmid library. Since the Canal contains some chemical waste and sewage runoff that could potentially contain human pathogens, the collection team wore disposable Tyvek suits, eye protection, and heavy rubber gloves. We were careful to avoid splashing the water unnecessarily, however was not easy to pull bottom sludge samples to the surface and the protective gear was necessary. Since we had sequenced at least one round of samples from the canal already, we knew what human pathogens it contained. None of these were airborne, so proper protective clothing and attention to splashing was deemed sufficient by our advisory board (see Safety page(link)  for further description). Following collection, the samples were stored overnight at 4C. Freezing would have disrupted the microorganisms and potentially diminished the DNA yield.
Extraction of High Molecular Weight DNA
We followed a procedure published by Dr. Sean Brady’s laboratory at Rockefeller University . Dr. Brady’s group is searching for novel antibiotic synthesis pathways in soil samples, and their work gave us the idea to create the library of DNA from the Gowanus samples. Briefly, samples were placed in individual blenders and the extraction                     accomplished through a combination of shear force, surfactant  and high salt (Lysis buffer containing 1.5M NaCl, 2% SDS and 1% cetyl trimethyl ammonium bromide). No living organisms are expected to survive this treatment. Although the                     original protocol calls for a pre-sieving step, this was not necessary for the Gowanus samples and was omitted, minimizing the contact with living organisms. The resulting crude DNA was further purified from the cell lysate by a series of                     centrifugation and precipitation steps.
Querying the Gowanus DNA for presence of specific pathways
Unfortunately we were unable to probe for more than one set of pathways before the end of the iGEM period due to lack of time. As a proof-of-principle demonstrating that such mining was possible, we used degenerate PCR primers to search for sequences bearing close resemblance to biosynthetically or biomedically interesting gene clusters following another published Brady lab protocol .  Secondary metabolites produced in microbes by nonribosomal peptide synthetase (NRPS) and polyketide synthase (PKS) biosynthetic machinery are a source of pharmacologically relevant chemical compounds such as antibiotics. The PCR probes we used were based on conserved regions within both NRPS adenylation (A) domain and PKS ketosynthase (KS) domain sequences. The paper states that “primer sets we used in this study are known to provide robust amplification of a wide variety of A/KS domains, particularly those found in the genomes of GC-rich soil bacteria traditionally associated with the production of medicinally relevant secondary metabolites.”
Results
It appears that at least some of the samples contain sequences that are worth further exploration!
A-domains
KS-domains
Place sticky footer content here.
