Team:Genspace/Notebook
<!DOCTYPE html>
Gowanus SuperFUNd - Genspace iGEM 2015
Toggle navigation
Home
Accomplishments
PROJECT
Background
Biosensor
Mining the Microbiome
Design
parts
NOTEBOOK
ATTRIBUTIONS
TEAM
collaborations
HUMAN PRACTICES
SAFETY
Notebook
Week 1 Week 2 Week 3 Week 4 Week 5 Week 6 Week 7 Week 8 Week 9 Week 10
Week 1: June 30 - July 2
Tuesday (6/30/2015)
Objective: Safety Training
Purpose: Our safety
Summary: Today was entirely dedicated to researching, splitting the team in two to make the most efficient use of our time. The adults and undergraduates focused on quorum sensing. The high school students looked through different promoters that activated in the presence of molecules common to human sewage.
Changes in chemicals after human waste in sewage water (mg/l)
Bicarbonate (HCO3) 50-100
Chloride (Cl) 20-50
Nitrate (NO3) 20-40
Phosphate (PO4) 5-15
Sulfate (SO4) 15-30
Wednesday (7/1/2015)
Objective: 60% Glycerol stock
Purpose: Student training
Summary: We have finally done stuff! Student team members continued their initial safety training, as well as learning how to do basic techniques (e.g. 60% Glycerol stock). Meanwhile, our senior team members came up with a more detailed plan than just "biosensor". Miranda came up with multiple plans for the system. By modifying Lsr Operons, we hope to use Quorum sensing to detect the presence of E.coli bacteria other than those that we use. -80 Competent cells
1.5 ml of culture into fresh 2.0 ml cryo tubes
Add 0.5 ml of 60% Glycerol
Vortex to mix
Flash freeze in Liquid Nitrogen
Transfer cryo tubes to long-term storage in -80^C freezer
The purpose of this was teaching the kids how to make competent cells YAAAAY
We also spent the day researching various E. coli quorum sensing pathways. Many bacterial species, such as E. coli, use quorum sensing to regulate gene expression in response to changes in local population size.  We stumbled upon the AI-2 pathway, which is shared by gram negative and gram positive bacteria. The system is based on an Lsr regulon, and is comprised of multiple interacting parts, including a promoter, LsrR, LsrK, and LsrACDB.
LsrA, C, D, and B encode subunits of a transmembrane transport protein that brings AI-2 into the cell, where it is phosorphorylated by LsrK.  Phospho-AI-2 binds to LsrR, the repressor for the Lsr promoter, allowing transcription of the regulon.
We hope to couple the Lsr regulon with a downstream fluorescent gene to create a quorum sensing-based “biosensor,” that fluoresces in high concentrations of E. coli. In doing so, we provide the Gowanas community with an instantaneous means of evaluating the canal’s potentially dangerous contents.
In order to prevent the biosensor from self-inducing, we must knock out luxS, the gene that produces AI-2.
This is a diagram of the E. coli AI-2 quorum sensing pathway (above). It was drawn by some of the team members who had been researching quorum sensing, in order to explain the results of this research to the rest of the team.
This is a flowchart of our preliminary plan.
A better diagram of the Lsr pathway
Thursday (7/2/2015)
Objective: Waking parts from the registry
Purpose: iGem Parts
Puncture the foil covering with a clean pipette tip.
Using a fresh tip, pipette 10μL from a fresh aliquot of Molecular Biology Grade Water into wells 18I (K117002) and 18K (K117008), then pipette up and down to mix.  The DNA contains a red dye, so the water should turn red as you mix.
Record the well you opened up and continue by transforming the DNA.
Summary: The "Quorum Sensing" group looked up two parts we need for the sensing project (Lsr Promoter: pLsrA {BBa_K117002} pLsrA-YFP {BBa_K117008}), and decided to teach the students how to take DNA out of the registry.
In pursuing further quorum sensing research, we came across the Lux operon. Like Lsr, the Lux system is important in bacterial quorum sensing. The operon consists of a Lux promoter and its regulator, LuxR. The system is controlled by the signal AHL (N-Acyl homoserine lactone), emitted by the gram negative bacterial species V. fischeri. After diffusing through the cell membrane, two molecules of AHL bind with two molecules of LuxR protein. The complex then binds to the promoter, upregulating transcription of downstream genes.
We plan to ligate the LuxR operon with a fluorescent protein gene. If the circuit works properly, the bacteria will fluoresce when surrounding bacterial population sizes increase.
Week 2: July 6 - July 10
Monday (7/6/2015)
Objective: 3A assembly
Purpose: Student training
In order, place following into PCR tube:
33 ul water
5 ul 10x NE (New England) Buffer 2.1
1 ul Enzyme #1
1 ul Enzyme #2
DNA- 500 ng (10 ul) part or 500 ng (3 ul) DNA backbone
Incubate at 37^C for 10 min.
Heat inactivate at 80^C for 20 min.
Hold at 4^C
We bought a container of Top Fin Liquid Saltwater Concentrate to test how E.coli withstand various salt concentrations.
Tuesday (7/7/2015)
Objective: Plasmid Prep
Purpose: Student training
Place TE buffer into incubator at 37^C
Centrifuge 1.5 ml of the overnight LB culture for 30 seconds
Add 250 ul Resuspension Buffer (R3) with RNase to cell pellet by pipetting up and down
Add 250 ul Lysis Buffer (L7). Mix gently by inverting 5x. Incubate at room temp. for exactly 5 min.
Add 350 ul Precipitation Buffer (N4). Invert 5x. Place in centrifuge at 11,000 RPM for 10 min.
Load supernatant into 2 ml wash tube with spin column. Place in centrifuge for 1 min.
Add 500 ul wash buffer (w10) with ethanol. Incubate at room temp. for 1 min. Centrifuge for 1 min.
Add 700 ul wash buffer (wq). Centrifuge for 1 min.
Place spin column in a clean 1.5 ml recovery tube. Add 75 ul of TE to center, incubate for 1 min. at room temp.
Plates have been streaked with E.coli K12 in preparation for the salt concentration experiment.
Wednesday (7/8/2015)
Objective: Pouring Electrophoresis gels
Purpose: Student training
Summary: LB plates were made by students as a review of our knowledge, using three antibiotics: Ampicillin. Chloramphenicol, and Kanamycin.
0.35 g agarose (0.7%)
50ml TAE water
Foil over flask and poke holes
Microwave: Power level 70%; 4 min.; stop when boils
Once lukewarm, 5 ul Phenix dye (10,000 in water)
Plates have been streaked with E.coli K12 in preparation for the salt concentration experiment.
Thursday (7/9/2015)
Objective: Genomikon kit,  Design saline experiment
Purpose: Progress
NOTICE: We are assuming that the LSC (Liquid Salt Concentration) has a salinity of 31.5% because of its claim of mimicking saltwater conditions in a 9x dilution. Saltwater usually as a salinity of 3.5%
Set Seawater concentration
Set E.coli concentration
Samples at timepoints
Math behind dilution concentrations
Big picture experiment design
Genomikon: Iron bead binds to DNA Anchor connected to the bead - Sticky ends X,X’; Z,Z’ - Cap with amp resistance - Sodium Hydroxide disconnects the iron bead to the gene - DNA will circularize
Friday (7/10/2015)
Purpose: Can long can  E. coli survive in seawater, and if so for how long?
Idea:Using +/- controls and 3.5 concentration of salt water. Focus on time and how we can kill of all GM bacteria quickly. Discussed other bacteria that we could detect in the sewage that may flow from canal to ocean.
Protocol for the Minimal Media Recipe
Under the Sea
Set E. coli concentration (1:100 dilution of O/N)
Set salt concentration
Different times
Plate
Individual Colonies
No time restrictions
Brooth
Shaker mimics sea/ocean
Spectrophotometer
Time restrictions
Week 3: July 13 - July 17
Monday (07/13/15)
Purpose:Purpose of Protocol is to mimic the environment of the ocean and observe when E. coli would not grow anymore. Adjustments were made to the protocol to make the experiment more accurate; the volume of the total solution of saltwater and E. coli will be increased so there would be less evaporation of water.
"Under the Sea" Experiment
Materials:
8 Agar Plates (No antibiotics)
100 microliters of E.coli (O/N)
9900 microliters of Salt water
Positive control -0'0" salt + E.coli
Negative control - 0'0" salt
Procedure:
Make a 1:100 dilution of stationary phase E.coli in 9.9 ml of saltwater
Pour 9,900 microliters of saltwater into 500 ml flask
Take 1 mL, put in spectrophotometer, plate as control
Put 100 Microliters of E.coli into flask, mix, put in spec, plate as +control
Place flask on shaking platform, turn to 5 (level)
Check every hour by taking 1 ml of Solution into spec, plate 1 ml
Finish at 6 PM, last culture at 1 PM the next day
O/N plates, look at results
Adjustments made.
Tuesday (07/14/15)
Objective: New Salinity Test
Purpose: Reliable Results
Summary: Yesterday’s test did not go well. When we came in this morning, we found bacteria in the negative control, and a huge margin of error in the
New protocol: (1:10) dilution with salt water and E. coli where 4 mL of bacteria was used with 36 mL of salt water concentration. A new overnight was made with approx. 13mL.
Results of Salinity Test #1
Lsr Team Transformations
Identified potentially useful part (S03968) and transformed into E. coli Top 10.
Rehydrate part using 10μL of molecular biology grade water.
Part was located in 2012 plate 3 well 8D
Thawed competent cells on ice
Pipette 2μL of resuspended DNA onto cells.  Incubate on ice for 30 minutes.
Heat shock for 1 minute
Incubate on ice for 5 minutes
Add 200μL LB without antibiotic.  Incubate in shaker at 37°C for 30min.
Plate cells on LB agar containing ampicillin
Two plates were prepared for S03968; one using 20μL and one using 200μL of transformed cells.
200μL
Wednesday (07/15/15)
Objective: New Salinity Test
Purpose: Reliable Results
Test: Once again the negative control has grown bacteria in it, leading us to believe that our water supply is contaminated. We have plated a sample directly from an aliquot of the saltwater, and if it turns out positive then we will know why the experiment has been contaminated. We now know to autoclave all saltwater supplies in the future.
Results of Salinity Test #2
Lsr Team: Plates prepared on 14-Jul-2015 from E. coli Top 10 transformed with S03968 grew a significant number of colonies.  A single colony was isolated and transferred into ~5mL of LB containing ampicillin and cultures overnight at 37°C in a rotator.
Speakers: https://www.youtube.com/watch?v=geMMjlwcPVQ
Thursday (07/16/15)
Objective: New Salinity Test
The sample of filtered salt water that was plated had colonies on the plate. Most of the colonies were centered around the edge of the plate. This can indicate contamination even after the filtration.
Today we autoclaved the filtered (expiration materials were checked (2019)) salt water and plated the negative control. We are doing 3A Assembly (a backup plan if the Synbiota kit fails).
Synbiota kit arrived.
Friday (07/17/15)
Objective: Prepping for salinity test
The filtered and autoclaved salt water that was plated showed no colonies of bacteria after incubation overnight.
The plates from the assembly and transformation were stored in the incubator.
Week 4: July 20 - July 24
Monday (7/20/15)
Created agar and poured plates due to shortage.
Results Salt concentration experiment results:
Bacteria was added to agar in tubes due to surprising results we got from type a and type b plates with GFP and RFP in them.            At this time, we realized that we cannot trust the A600 spec. readings from previous experiments because we had used 10 ml of LB broth with E. Coli grown in it. We conducted yet another salinity test, but with no LB in the solution.
Procedure:
Prepare an Overnight of Top 10 E. Coli in 10 ml of no antibiotic LB
Centrifuge 1 ml of the O/N at 12,000 rpm for 1 min.
Dump the supernatant into 10% Bleach
Resuspend pellet in 1 ml of seawater
Create a solution with 49 ml of saltwater and the 1 ml of E.Coli resuspension
Plate 500 ul pure seawater as negative control
Plate 500 ul of the inoculated solution as test
Check every hour with an OD 600 and plate
A600 spec Results:
Negative Control: Ref.
0 hour: 0.005
1 hour: 0.007
2 hour: 0.015
72 hour: 0.336
The negative control did not grow colonies, while the test plates each grew an even spread of white specks (Not a lawn, which indicates a low concentration of bacteria since 500 ul was plated). However, we are having trouble determining whether the white specks are colonies or crystallized salt. We will wait 48 hours, and if the specks grow we will know that they are colonies.
Tuesday (7/21/15)
Did inventory of things we need to order.
Wednesday (7/22/15)
Troubleshooting for red and green
Check to see if band differences is whether the parts different sizes, same backbone
Different bands red 700bp (red fluorescent protein but not Rudolph)
We need to take the red and green and digest it and set it into two different wells
490 ml Distilled Water
10 ml TAE (pure)
Nitrate Sensor
Part: BBa_K381001 is in the registry 2015 Plate 1 4B
Backbone: p5BIC3
Part: BBa_J23110 - "Medium Promoter Already Miniprep" Constitutive promoter 2015 Distribution Plate 4 17P
Thursday:  (7/23/15)
Made glycerol Stock of Part: K381001
Flash Frooze: Red Handle - Position 4 Top Left
New Procedure: Transforming w/ Clay
Materials:
1ml of log phase E.coli  (Stock: 4% sepiolite)
3.9 ml of water (MGW) (Working: 1% sepiolite)
Vortex the clay (0.1ml of sepiolite) * Helping Columbia Team
Procedure
Dilute the sepiolite 1:40 with water
Centrifuge at 12,000 rpm for 30 sec. the cells
Dry the agar plates face down in incubator
100 microliter of 1% sepiolite pipette up and down to resuspend
Vortex for 5 min.
Plasmid Prep of NsrR is in IGEM 2015 - 20 Box 1-9
Friday (7/24/15)
Ran a digest on the nitrate parts and ran a gel. No record of gel. Redoing gel on Monday.
Week 5: July 27 - July 31
Monday (7/27/2015)
Objective:Retrospective Analysis
Purpose:Troubleshooting
Gel Electrophoresis
Making the overnight culture.
Using ethidium bromide for gel to illuminate orange.
Electrophoresis box was found to have faulty wiring, which raises the question of wheher the parts were the problem or the equipment used.
Gel analyzed revealing that that ladder on the left side came out almost perfectly, but the rest of the bands came out ‘S’-shaped and the ladder on the right side was barely visible.
Possible Sources of Error within the box
Negatively charged DNA built up on positive wire causing messed up electrofield
Wires are bent, thus causing impaired electrofield
Notes taken durring discussion:
Tuesday (7/28/15)
Spent today making Competent Cells
Wednesday (7/29/15)
We digested R1, “Rudolph”, and the nitrate promoter.
PCR’ed Synbiota part.
The remaining Lsr system parts for synthesis have been ordered
Thursday (7/30/15)
3A assembly of LuxR & LacZ
Bba_J37019: LuxR (Chloramphenicol Resistant)
Bba_K909006: LacZ  (Chloramphenicol Resistant)
Ribosome Binding Site for both is Elowitz RBS (B0034) with efficiency of 1.
Upstream Master Mix
Downstream Master Mix
Backbone Master Mix
5µL Buffer (10x)
5µL Buffer (10x)
5µL Buffer (10x)
2.5µL EcoRI
2.5µL XbeI
2.5µL EcoRI
2.5µL SpeI
2.5µL PstI
2.5µL PstI
Restriction Digest:
Upstream Part
Downstream Part
Backbone Part
5µL J37019 “Lux” Plasmid
5µL K909006 “Lac” Plasmid
5µL Sp1A3 “Amp” Plasmid
3µL MGBW
3µL MGBW
3µL MGBW
2µL Master Mix
2µL Master Mix
2µL Master Mix
PCR:
Set timer prior to adding master mix.
37˚C - 60 min; 80˚C - 20 min; 4˚C - ∞
Friday (7/31/15)
Objective:Pouring Amp. plates, Lux Rudolph Test, Finish 3A Assembly
Purpose:Progress
Pouring Ampicillin plates to have a supply
3A Assembly of LuxR + LacZ
18 Ampicillin resistant plates were made.
LuxRudolph Test
Spectrophotometry Result:
OD 600
OD 588 (Red)
Reference (LB)
N/A
N/A
Overnight +
1.987
2.021
Overnight -
2.000
2.031
Positive +
0.077
0.172
Negative -
0.000
-0.001
Test
0.076
0.167
+1
0.343
0.303
-1
0.001
0.009
T1
0.353
0.310
+2
0.748
0.776
-2
0.097
0.101
T2
0.728
0.749
+3
1.144
1.177
-3
-0.002
-0.001
T3
1.072
1.102
The Lux Rudolph Test failed because the negative control after the third hour was negative, implying there was no bacteria in the stock tube to begin with. The experiment will be redone on Monday.
LuxRudolph Diagram (left) | Lsr Operon (right)
Ligation of LuxR and LacZ:
RI: I0500
RU: J37019
RD: LacZ
RB: Amp. Resistant
Upstream + Downstream + Backbone:
(RI + RA) + RB
(RU + RA) + RB
(RI + RD) + RB
(RU + RD) + RB
Saline Experiment:
We plated the salt solutions that were made from about two weeks ago. The colonies on the plate that weren’t supposed to be fed LB were smaller, the plate had more colonies  than the other one, which has a smaller amount of larger colonies.
Lsr Operon
The Lsr Operon has RBS within its OFR/CDS, unlike normal circuits where the RBS is prior to the OFR.
This means that when a ribosome starts translation, it is makes 4 proteins from a highly conserved sequence.
This might be good for biological systems, but in terms of modular design it is not as efficient. Since the OFR contains a RBS, if you were to attempt to combine the various OFRs with a separate RBS, it takes twice as many RBS’s.
In Open Reading Frames, the start codon is the first three bases (AUG).
A Ribosome Binding Site, is what the ribosome binds and the OFR comes after. A common sequence is the Shine-Dalgarno that is usually 8 bases upstream of the start codon AUG. The six-base sequence is AGGAGG.
In order to move around this problem and prevent using double the amount of needed RBS’s, we will change the sequence to code for the same proteins, but with different bases. Thus utilizing the degenerative nature of DNA.
Week 6: Aug 3 - Aug 7
Monday (8/3/15)
Today, plasmid protocol was done on the LSR operon from last week to see if the assembly worked.
Tuesday (8/4/15)
Transported RDP parts “Lux” and “Rudolf” to Cooper Union to measure DNA concentration using their Nanodrop spectrophotometer:
Sample
A260
A280
[DNA]
Lux
0.850
0.415
42.5ng/μL
Rudolf*
1.095
0.531
54.5ng/μL
*Rudolf had an additional absorbance peak around 220nm.  A220 was approx. 2.8
Synbiota recommends 0.04pM/μL of sample DNA per reaction.
1pM/μL = 670ng/μL
Lux: 42.5ng/μL = 0.0634pM/μL
Rudolf: 54.5ng/μL = 0.0813pM/μL
Lux dilution factor: 1.585
Rudolf dilution factor: 2.033
Wednesday (8/5/2015)
The Miller Assay using Galactosidase and ONPG (yellow fluorescent). We did a digest and then a transformation of the failed 3A Assembly from a couple of weeks ago.
Thursday (8/5/2015)
Objective:Redoing the Miller Assay
First try - failed, used 500 microliters of cells (turned yellow too quickly)
Second attempt - 1 microliter of cells
Week 7: Aug 10 - Aug 14
Tuesday (8/11)
5 Ordered modularized parts were received from IDT: LsrR, LsrD, LsrB, LsrC,NsrR
Plans to lyophilize today.
Resuspension Protocol:
Centrifuge
Add 100 µl TE Buffer
Vortex
Centrifuge
Place in 50C H2O for 20 minutes
‘2A’ Assembly:
Master Mix (6x)
6µl NE buffer 2.1
3µl EcoRI
4µl EcoRI
Digest:
Part
Backbone
8µl DNA
2µl MM
5µl DNA
2.5µL MBGW
1µL NEB 2.1
0.5µL EcoRI
0.5µL PstR
0.5µL BsaL
Ligation:
Combine
6.5µl Part Digest
2.0µl Backbone Digest
1.0µl Ligation Buffer
0.5µl T4 DNA Ligase
Ligate @ 16C for 30 min
Heat kill @ 80C for 20 min
Heat kill @ 4C for ∞
Thursday (8/13)
Worked in: gentle-beta.synbiota.com
Week 8: Aug 17 - Aug 22
Tuesday (8/18)
Rapid DNA Prototyping for Lux + Rudolph circuit, as reviewing reveals that the previous circuit was not properly made.
Bio Brick Structure:
dA-Anchor (Z’) => (Z) ADP (X’) =>(X) Lux (Z’) =>(Z) RBS (X’) =>(X) Rud (Z’) =>(Z)  Cap-dT
Transforming competent cells
Miller Assay Results:
AHL Positive (+) (1 mM)
AHL Negative (-) (0 mM)
OD 600
1.315
1.268
OD 420
0.888
0.972
OD 550
0.012
0.007
AHL Positive Miller Units : 7911.79
AHL Negative Miller Units: 9082.81
Results were not as expected. Full reaction took ⅚ of a minute, at 1:10 dilution (.1 mL). The - is able to stay in cells longer, while in log phase, while the + is quickly dispersed and expressed in log phase.
“Slow and steady” hypothesis:
The - cells are able to become more turgid/filled with galactosidase enzyme than the + cells.
Week 9: Aug 24 - Aug 28
Monday (8/24)
Gel was made to prepare for LR Digest
Findings thus far:
“The truth will set you free” - Caltech
“... But first it will make you sick.” - Joel Franklin
Plan A (L+R):
Plux -> RBS -> LuxR -> RBS -> Rudolph -> Terminator
AHL is 200 Daltons. This is a positive feedback look.
Simple Model:
If [AHL], then Rudolph is produce.
If [AHL], then Rudolph is not produced.
Analogous Circuit (L+L):
Plux -> LuxR -> LacZ
Overnight Experiments (L+L)
 |->
1mM AHL @ 37C (+ Positive control)
Glycerol Stock Scrape
LB + Ampicillin Broth
|->
0mM AHL @ 37C (- Negative control)
Using the Miller Assay it was revealed the + had 8000 MU, and the - had 9000 MU. Future tests revealed that the - had consistently higher amounts of ß-galactosidase.
Concentration Change Test:
The Negative control has no AHL in it’s surrounding, or within in its cells after the experiment.
The Positive control has AHL in it’s surrounding and within in its cells after the experiment.
The Test has AHL in it’s surrounding, but none in it’s cells at the beginning. And at the end the AHL has diffused into its cells.
Negative
Test
Positive
Start with Negative O/N (1:100) dilution
Start with Negative O/N (1:100) dilution
Start with Positive O/N (1:100) dilution
+0mM AHL
+1mM AHL
+1mM AHL
All three of these samples were placed in a rotating incubator, sampled every hour, sample in spectrophotometer at OD 600, and then the Miller Assay was done for each. The OD 600 revealed the growth functions of both the Negative and Positive were normal, but the Test growth was severely retarded. Consequently the Miller Assay revealed that the amount of ß-gal in the Negative and Positive steadily decreased over time, while the Test has a constant large amount present. This implies that the Test cells are focussing more on the production of a protein, than on their own growth.
The Flanagan Hypothesis
The delay factor in growth between the Negative/Positive and Test is 12:1. A test with differing AHL concentrations would prove this.
Sampling Experiments (L+R):
POS O/N
L+R in LB Amp
1mM AHL
Log growth 1:100 Dilution
Add 5µL LB
+1 mM AHL
Incubate @ 37C
NEG  O/N
L+R in LB Amp
0mM AHL
+0 mM AHL
+1 mM AHL
Tuesday (8/25)
In preparation of Columbia Fluorometer:
Gel was prepared for a digest of RBS parts and promoters:
RBS A
RBS B
RBS C
pReaR
pLsrA2
Wednesday (8/26)
OD 600 readings
Thursday (8/27)
OD 600 readings
Friday (8/28)
PCR Cleanup of:
NsrR RDP
LsrR RDP
LsrA RDP
LsrB RDP
LsrC RDP
LsrD RDP
LuxR RDP
pLux RDP
Gel was pre made (courtesy of Will) for the samples for electrophoresis.
Week 10: Aug 31 - Sept 4
Monday (8/31)
Tuesday (9/1)
washes & labeling
Wednesday (9/2)
Minipreps
Made plates
Thursday (9/3)
plasmid prep & digest
Friday (9/4)
Four different plasmids were made using parts from synbiota:
dA18-AmpR-2’  --  2-adp-x’   --   x-pr.3-2’   --   2-Rbs.4.tx’   --   x-LacI-2’   --2-Ori.2-dT18
dA18-ChlrRJx’ -- x-pr.lacO-2’ -- 2-Rbs.2.tx’ -- x-dRFP-2’ -- 2-Ori3-dT18
dA18-ChlrR-x’ -- x-pr.lacO-2’ -- 2-Rbs.2.tx’ -- x-LsrR-2’ -- 2-Ter-x’ -- x-pLsrA-2-2’ -- 2
Place sticky footer content here.
