<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Genspace_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Genspace/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Gowanus SuperFUNd - Genspace iGEM 2015 </TITLE></P><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A href="https://2015.igem.org/Team:Genspace" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Home</A></LI><LI><A href="https://2015.igem.org/Team:Genspace#accomplishment">Accomplishments</A></LI></UL><LI class="dropdown"><A href="https://2015.igem.org/Team:Genspace/Description" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Description#background">Background</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Description#biosensor">Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Description#microbiome">Mining the Microbiome</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Parts">parts</A></LI><LI class="active"><A href="https://2015.igem.org/Team:Genspace/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Collaborations">collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Genspace/Safety">SAFETY</A></LI></DIV></DIV><DIV id="page" class="container"><H1>Notebook</H1><A class="weekNav lead">Week 1</A><A class="weekNav lead">Week 2</A><A class="weekNav lead">Week 3</A><A class="weekNav lead">Week 4</A><A class="weekNav lead">Week 5</A><A class="weekNav lead">Week 6</A><A class="weekNav lead">Week 7</A><A class="weekNav lead">Week 8</A><A class="weekNav lead">Week 9</A><A class="weekNav lead">Week 10</A><DIV class="weeks"><DIV class="week row"><H3 class="header">Week 1: June 30 - July 2</H3><DIV class="section entry"><H2 class="lead">
              Tuesday (6/30/2015)
            </H2><P><B>Objective:</B> Safety Training</P><P><B>Purpose:</B> Our safety</P><P><B>Summary:</B> Today was entirely dedicated to researching, splitting the team in two to make the most efficient use of our time. The adults and undergraduates focused on quorum sensing. The high school students looked through different promoters that activated in the presence of molecules common to human sewage.
            </P><P>Changes in chemicals after human waste in sewage water (mg/l)</P><UL><LI>Bicarbonate (HCO3) 50-100</LI><LI>Chloride (Cl) 20-50</LI><LI>Nitrate (NO3) 20-40 </LI><LI>Phosphate (PO4) 5-15</LI><LI>Sulfate (SO4) 15-30</LI></UL></DIV><DIV class="section entry"><H2 class="lead">
              Wednesday (7/1/2015)
            </H2><P><B>Objective:</B> 60% Glycerol stock</P><P><B>Purpose:</B> Student training</P><P><B>Summary:</B> We have finally done stuff! Student team members continued their initial safety training, as well as learning how to do basic techniques (e.g. 60% Glycerol stock). Meanwhile, our senior team members came up with a more detailed plan than just &quot;biosensor&quot;. Miranda came up with multiple plans for the system. By modifying Lsr Operons, we hope to use Quorum sensing to detect the presence of E.coli bacteria other than those that we use. -80 Competent cells
            </P><OL><LI>1.5 ml of culture into fresh 2.0 ml cryo tubes </LI><LI>Add 0.5 ml of 60% Glycerol</LI><LI>Vortex to mix</LI><LI>Flash freeze in Liquid Nitrogen</LI><LI>Transfer cryo tubes to long-term storage in -80^C freezer</LI></OL><P>The purpose of this was teaching the kids how to make competent cells YAAAAY</P><P>We also spent the day researching various E. coli quorum sensing pathways. Many bacterial species, such as E. coli, use quorum sensing to regulate gene expression in response to changes in local population size.  We stumbled upon the AI-2 pathway, which is shared by gram negative and gram positive bacteria. The system is based on an Lsr regulon, and is comprised of multiple interacting parts, including a promoter, LsrR, LsrK, and LsrACDB.</P><P>LsrA, C, D, and B encode subunits of a transmembrane transport protein that brings AI-2 into the cell, where it is phosorphorylated by LsrK.  Phospho-AI-2 binds to LsrR, the repressor for the Lsr promoter, allowing transcription of the regulon.</P><P>We hope to couple the Lsr regulon with a downstream fluorescent gene to create a quorum sensing-based “biosensor,” that fluoresces in high concentrations of E. coli. In doing so, we provide the Gowanas community with an instantaneous means of evaluating the canal’s potentially dangerous contents. </P><P>In order to prevent the biosensor from self-inducing, we must knock out luxS, the gene that produces AI-2.</P><P>This is a diagram of the E. coli AI-2 quorum sensing pathway (above). It was drawn by some of the team members who had been researching quorum sensing, in order to explain the results of this research to the rest of the team.</P><P>This is a flowchart of our preliminary plan.</P><P>A better diagram of the Lsr pathway</P></DIV><DIV class="section entry"><H2 class="lead">
              Thursday (7/2/2015)
            </H2><P><B>Objective:</B> Waking parts from the registry</P><P><B>Purpose:</B>  iGem Parts</P><OL><LI>Puncture the foil covering with a clean pipette tip.</LI><LI>Using a fresh tip, pipette 10μL from a fresh aliquot of Molecular Biology Grade Water into wells 18I (K117002) and 18K (K117008), then pipette up and down to mix.  The DNA contains a red dye, so the water should turn red as you mix.</LI><LI>Record the well you opened up and continue by transforming the DNA. </LI></OL><P><B>Summary:</B> The &quot;Quorum Sensing&quot; group looked up two parts we need for the sensing project (Lsr Promoter: pLsrA {BBa_K117002} pLsrA-YFP {BBa_K117008}), and decided to teach the students how to take DNA out of the registry.
            </P><P>In pursuing further quorum sensing research, we came across the Lux operon. Like Lsr, the Lux system is important in bacterial quorum sensing. The operon consists of a Lux promoter and its regulator, LuxR. The system is controlled by the signal AHL (N-Acyl homoserine lactone), emitted by the gram negative bacterial species V. fischeri. After diffusing through the cell membrane, two molecules of AHL bind with two molecules of LuxR protein. The complex then binds to the promoter, upregulating transcription of downstream genes.</P><P>We plan to ligate the LuxR operon with a fluorescent protein gene. If the circuit works properly, the bacteria will fluoresce when surrounding bacterial population sizes increase.</P></DIV></DIV><DIV class="week row"><H3 class="header">Week 2: July 6 - July 10</H3><DIV class="section entry"><H2 class="lead">
              Monday (7/6/2015)
            </H2><P><B>Objective:</B> 3A assembly </P><P><B>Purpose:</B> Student training</P><OL><LI>In order, place following into PCR tube: 
              </LI><LI>33 ul water</LI><LI>5 ul 10x NE (New England) Buffer 2.1</LI><LI>1 ul Enzyme #1</LI><LI>1 ul Enzyme #2</LI><LI>DNA- 500 ng (10 ul) part or 500 ng (3 ul) DNA backbone</LI></OL><LI>Incubate at 37^C for 10 min.</LI><LI>Heat inactivate at 80^C for 20 min.</LI><LI>Hold at 4^C</LI><P>We bought a container of Top Fin Liquid Saltwater Concentrate to test how E.coli withstand various salt concentrations.</P></DIV><DIV class="section entry"><H2 class="lead">
              Tuesday (7/7/2015)
            </H2><P><B>Objective:</B> Plasmid Prep </P><P><B>Purpose:</B> Student training</P><OL><LI>Place TE buffer into incubator at 37^C</LI><LI>Centrifuge 1.5 ml of the overnight LB culture for 30 seconds</LI><LI>Add 250 ul Resuspension Buffer (R3) with RNase to cell pellet by pipetting up and down</LI><LI>Add 250 ul Lysis Buffer (L7). Mix gently by inverting 5x. Incubate at room temp. for exactly 5 min.</LI><LI>Add 350 ul Precipitation Buffer (N4). Invert 5x. Place in centrifuge at 11,000 RPM for 10 min.</LI><LI>Load supernatant into 2 ml wash tube with spin column. Place in centrifuge for 1 min.</LI><LI>Add 500 ul wash buffer (w10) with ethanol. Incubate at room temp. for 1 min. Centrifuge for 1 min.</LI><LI>Add 700 ul wash buffer (wq). Centrifuge for 1 min.</LI><LI>Place spin column in a clean 1.5 ml recovery tube. Add 75 ul of TE to center, incubate for 1 min. at room temp.</LI></OL><P>
              Plates have been streaked with E.coli K12 in preparation for the salt concentration experiment.
            </P></DIV><DIV class="section entry"><H2 class="lead">
              Wednesday (7/8/2015)
            </H2><P><B>Objective:</B> Pouring Electrophoresis gels</P><P><B>Purpose:</B> Student training</P><P><B>Summary:</B> LB plates were made by students as a review of our knowledge, using three antibiotics: Ampicillin. Chloramphenicol, and Kanamycin.</P><OL><LI>0.35 g agarose (0.7%)</LI><LI>50ml TAE water</LI><LI>Foil over flask and poke holes</LI><LI>Microwave: Power level 70%; 4 min.; stop when boils</LI><LI>Once lukewarm, 5 ul Phenix dye (10,000 in water)</LI></OL><P>
              Plates have been streaked with E.coli K12 in preparation for the salt concentration experiment.
            </P></DIV><DIV class="section entry"><H2 class="lead">
              Thursday (7/9/2015)
            </H2><P><B>Objective:</B> Genomikon kit,  Design saline experiment</P><P><B>Purpose:</B> Progress</P><P>NOTICE: We are assuming that the LSC (Liquid Salt Concentration) has a salinity of 31.5% because of its claim of mimicking saltwater conditions in a 9x dilution. Saltwater usually as a salinity of 3.5%</P><OL><LI>Set Seawater concentration</LI><LI>Set E.coli concentration</LI><LI>Samples at timepoints</LI></OL><P>
              Math behind dilution concentrations
            </P><P>Big picture experiment design</P><P>Genomikon: Iron bead binds to DNA Anchor connected to the bead - Sticky ends X,X’; Z,Z’ - Cap with amp resistance - Sodium Hydroxide disconnects the iron bead to the gene - DNA will circularize</P></DIV><DIV class="section entry"><H2 class="lead">
              Friday (7/10/2015)
            </H2><P><B>Purpose:</B> Can long can  E. coli survive in seawater, and if so for how long?</P><P><B>Idea:</B>Using +/- controls and 3.5 concentration of salt water. Focus on time and how we can kill of all GM bacteria quickly. Discussed other bacteria that we could detect in the sewage that may flow from canal to ocean. </P><P><B>Protocol for the Minimal Media Recipe </B></P><P><B>Under the Sea</B></P><OL><LI>Set E. coli concentration (1:100 dilution of O/N)</LI><LI>Set salt concentration </LI><LI>Different times</LI></OL><P><B>Plate</B></P><OL><LI>Individual Colonies</LI><LI>No time restrictions</LI></OL><P><B>Brooth</B></P><OL><LI>Shaker mimics sea/ocean</LI><LI>Spectrophotometer</LI><LI>Time restrictions</LI></OL></DIV></DIV><DIV class="week row"><H3 class="header">Week 3: July 13 - July 17</H3><DIV class="section entry"><H2 class="lead">Monday (07/13/15)</H2><P><B>Purpose:</B>Purpose of Protocol is to mimic the environment of the ocean and observe when E. coli would not grow anymore. Adjustments were made to the protocol to make the experiment more accurate; the volume of the total solution of saltwater and E. coli will be increased so there would be less evaporation of water.</P><P><B>&quot;Under the Sea&quot; Experiment</B></P><P><B>Materials:</B></P><OL><LI>8 Agar Plates (No antibiotics) </LI><LI>100 microliters of E.coli (O/N) </LI><LI>9900 microliters of Salt water </LI><LI>Positive control -0'0&quot; salt + E.coli</LI><LI>Negative control - 0'0&quot; salt</LI></OL><P><B>Procedure:</B></P><OL><LI>Make a 1:100 dilution of stationary phase E.coli in 9.9 ml of saltwater</LI><LI>Pour 9,900 microliters of saltwater into 500 ml flask </LI><LI>Take 1 mL, put in spectrophotometer, plate as control </LI><LI>Put 100 Microliters of E.coli into flask, mix, put in spec, plate as +control </LI><LI>Place flask on shaking platform, turn to 5 (level) </LI><LI>Check every hour by taking 1 ml of Solution into spec, plate 1 ml </LI><LI>Finish at 6 PM, last culture at 1 PM the next day</LI><LI>O/N plates, look at results </LI></OL><P>Adjustments made.</P></DIV><DIV class="section entry"><H2 class="lead">
              Tuesday (07/14/15)
            </H2><P><B>Objective:</B> New Salinity Test</P><P><B>Purpose:</B> Reliable Results</P><P><B>Summary:</B> Yesterday’s test did not go well. When we came in this morning, we found bacteria in the negative control, and a huge margin of error in the </P><P><B>New protocol:</B> (1:10) dilution with salt water and E. coli where 4 mL of bacteria was used with 36 mL of salt water concentration. A new overnight was made with approx. 13mL. </P><P><B>Results of Salinity Test #1</B></P><P>Lsr Team Transformations</P><P>Identified potentially useful part (S03968) and transformed into E. coli Top 10.</P><OL><LI>Rehydrate part using 10μL of molecular biology grade water.
                </LI><LI>Part was located in 2012 plate 3 well 8D</LI><LI>Thawed competent cells on ice</LI><LI>Pipette 2μL of resuspended DNA onto cells.  Incubate on ice for 30 minutes.</LI><LI>Heat shock for 1 minute</LI><LI>Incubate on ice for 5 minutes</LI><LI>Add 200μL LB without antibiotic.  Incubate in shaker at 37°C for 30min.</LI><LI>Plate cells on LB agar containing ampicillin
                </LI><LI>Two plates were prepared for S03968; one using 20μL and one using 200μL of transformed cells.</LI><LI>200μL </LI></OL></DIV><DIV class="section entry"><H2 class="lead">Wednesday (07/15/15)</H2><P><B>Objective:</B> New Salinity Test</P><P><B>Purpose:</B> Reliable Results</P><P><B>Test:</B> Once again the negative control has grown bacteria in it, leading us to believe that our water supply is contaminated. We have plated a sample directly from an aliquot of the saltwater, and if it turns out positive then we will know why the experiment has been contaminated. We now know to autoclave all saltwater supplies in the future. </P><P>Results of Salinity Test #2 </P><P><B>Lsr Team:</B> Plates prepared on 14-Jul-2015 from E. coli Top 10 transformed with S03968 grew a significant number of colonies.  A single colony was isolated and transferred into ~5mL of LB containing ampicillin and cultures overnight at 37°C in a rotator.</P><P><B>Speakers:</B><A href="https://www.youtube.com/watch?v=geMMjlwcPVQ">https://www.youtube.com/watch?v=geMMjlwcPVQ</A></P></DIV><DIV class="section entry"><H2 class="lead">Thursday (07/16/15)</H2><P><B>Objective:</B> New Salinity Test</P><P>The sample of filtered salt water that was plated had colonies on the plate. Most of the colonies were centered around the edge of the plate. This can indicate contamination even after the filtration. </P><P>Today we autoclaved the filtered (expiration materials were checked (2019)) salt water and plated the negative control. We are doing 3A Assembly (a backup plan if the Synbiota kit fails).</P><P>Synbiota kit arrived.</P></DIV><DIV class="section entry"><H2 class="lead">Friday (07/17/15)</H2><P><B>Objective:</B> Prepping for salinity test</P><P>The filtered and autoclaved salt water that was plated showed no colonies of bacteria after incubation overnight. </P><P>The plates from the assembly and transformation were stored in the incubator. </P></DIV></DIV><DIV class="week row"><H3 class="header">Week 4: July 20 - July 24</H3><DIV class="section entry"><H2 class="lead">Monday (7/20/15)</H2><P>Created agar and poured plates due to shortage.</P><P><B>Results</B> Salt concentration experiment results:</P><P>Bacteria was added to agar in tubes due to surprising results we got from type a and type b plates with GFP and RFP in them.
           At this time, we realized that we cannot trust the A600 spec. readings from previous experiments because we had used 10 ml of LB broth with E. Coli grown in it. We conducted yet another salinity test, but with no LB in the solution.</P><P><B>Procedure:</B></P><OL><LI>Prepare an Overnight of Top 10 E. Coli in 10 ml of no antibiotic LB</LI><LI>Centrifuge 1 ml of the O/N at 12,000 rpm for 1 min.</LI><LI>Dump the supernatant into 10% Bleach</LI><LI>Resuspend pellet in 1 ml of seawater</LI><LI>Create a solution with 49 ml of saltwater and the 1 ml of E.Coli resuspension</LI><LI>Plate 500 ul pure seawater as negative control</LI><LI>Plate 500 ul of the inoculated solution as test</LI><LI>Check every hour with an OD 600 and plate</LI></OL><P><B>A600 spec Results:</B></P><OL><LI>Negative Control: Ref.</LI><LI>0 hour: 0.005</LI><LI>1 hour: 0.007</LI><LI>2 hour: 0.015</LI><LI>72 hour: 0.336</LI></OL><P>The negative control did not grow colonies, while the test plates each grew an even spread of white specks (Not a lawn, which indicates a low concentration of bacteria since 500 ul was plated). However, we are having trouble determining whether the white specks are colonies or crystallized salt. We will wait 48 hours, and if the specks grow we will know that they are colonies.</P></DIV><DIV class="section entry"><H2 class="lead">Tuesday (7/21/15)</H2><P>Did inventory of things we need to order.</P></DIV><DIV class="section entry"><H2 class="lead">Wednesday (7/22/15)</H2><P><B>Troubleshooting for red and green</B></P><UL><LI>Check to see if band differences is whether the parts different sizes, same backbone</LI><LI>Different bands red 700bp (red fluorescent protein but not Rudolph)</LI><LI>We need to take the red and green and digest it and set it into two different wells </LI></UL><P>490 ml Distilled Water</P><P>10 ml TAE (pure)</P><P><B>Nitrate Sensor</B></P><UL><LI>Part: BBa_K381001 is in the registry 2015 Plate 1 4B</LI><LI>Backbone: p5BIC3 </LI><LI>Part: BBa_J23110 - &quot;Medium Promoter Already Miniprep&quot; Constitutive promoter 2015 Distribution Plate 4 17P</LI></UL></DIV><DIV class="section entry"><H2 class="lead">Thursday:  (7/23/15)</H2><P>Made glycerol Stock of Part: K381001</P><P>Flash Frooze: Red Handle - Position 4 Top Left</P><H4 class="underline"><B>New Procedure:</B> Transforming w/ Clay
            </H4><P><B>Materials:</B></P><UL><LI>1ml of log phase E.coli  (Stock: 4% sepiolite)</LI><LI>3.9 ml of water (MGW) (Working: 1% sepiolite)</LI><LI>Vortex the clay (0.1ml of sepiolite) * Helping Columbia Team  </LI></UL><P><B>Procedure</B></P><OL><LI>Dilute the sepiolite 1:40 with water </LI><LI>Centrifuge at 12,000 rpm for 30 sec. the cells</LI><LI>Dry the agar plates face down in incubator </LI><LI>100 microliter of 1% sepiolite pipette up and down to resuspend </LI><LI>Vortex for 5 min. </LI></OL><P>Plasmid Prep of NsrR is in IGEM 2015 - 20 Box 1-9</P></DIV><DIV class="section entry"><H2 class="lead">Friday (7/24/15)</H2><P>Ran a digest on the nitrate parts and ran a gel. No record of gel. Redoing gel on Monday.</P></DIV></DIV><DIV class="week row"><H3 class="header">Week 5: July 27 - July 31</H3><DIV class="section entry"><H2 class="lead">Monday (7/27/2015)</H2><P><B>Objective:</B>Retrospective Analysis</P><P><B>Purpose:</B>Troubleshooting</P><P>Gel Electrophoresis</P><P>Making the overnight culture.</P><P>Using ethidium bromide for gel to illuminate orange. </P><P>Electrophoresis box was found to have faulty wiring, which raises the question of wheher the parts were the problem or the equipment used. </P><P>Gel analyzed revealing that that ladder on the left side came out almost perfectly, but the rest of the bands came out ‘S’-shaped and the ladder on the right side was barely visible.</P><P><B>Possible Sources of Error within the box</B></P><UL><LI>Negatively charged DNA built up on positive wire causing messed up electrofield</LI><LI>Wires are bent, thus causing impaired electrofield</LI></UL><P>Notes taken durring discussion:</P></DIV><DIV class="section entry"><H2 class="lead">Tuesday (7/28/15)</H2><P>Spent today making Competent Cells</P></DIV><DIV class="section entry"><H2 class="lead">Wednesday (7/29/15)</H2><P>We digested R1, “Rudolph”, and the nitrate promoter.</P><P>PCR’ed Synbiota part. </P><P>The remaining Lsr system parts for synthesis have been ordered</P></DIV><DIV class="section entry"><H2 class="lead">Thursday (7/30/15)</H2><P>3A assembly of LuxR &amp; LacZ</P><P>Bba_J37019: LuxR (Chloramphenicol Resistant)</P><P>Bba_K909006: LacZ  (Chloramphenicol Resistant)</P><P>Ribosome Binding Site for both is Elowitz RBS (B0034) with efficiency of 1.</P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">Upstream Master Mix</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c21 c2 c27">Downstream Master Mix</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">Backbone Master Mix</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">5µL Buff</SPAN><SPAN class="c3 c2">er (10x)</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">5µL Buffer (10x)</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">5µL Buffer (10x)</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.5µL EcoRI</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.5µL XbeI</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.5µL EcoRI</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.5µL SpeI</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.5µL PstI</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.5µL PstI</SPAN></P></TD></TR></TBODY></TABLE><P><B>Restriction Digest:</B></P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">Upstream Part</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">Downstream Part</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">Backbone Part</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">5µL J37019 “Lux” Plasmid</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">5µL K909006 “Lac” Plasmid</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">5µL Sp1A3 “Amp” Plasmid</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">3µL MGBW</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">3µL MGBW</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">3µL MGBW</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">2µL Master Mix</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">2µL Master Mix</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">2µL Master Mix</SPAN></P></TD></TR></TBODY></TABLE><P><B>PCR:</B></P><P>Set timer prior to adding master mix.</P><P>37˚C - 60 min; 80˚C - 20 min; 4˚C - ∞</P></DIV><DIV class="section entry"><H2 class="lead">Friday (7/31/15)</H2><P><B>Objective:</B>Pouring Amp. plates, Lux Rudolph Test, Finish 3A Assembly    </P><P><B>Purpose:</B>Progress</P><P>Pouring Ampicillin plates to have a supply</P><P>3A Assembly of LuxR + LacZ</P><P>18 Ampicillin resistant plates were made.</P><P>LuxRudolph Test</P><P><B>Spectrophotometry Result:</B></P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">OD 600</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">OD 588 (Red)</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">Reference (LB)</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">N/A</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">N/A</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">Overnight +</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">1.987</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.021</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">Overnight -</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.000</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">2.031</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">Positive +</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.077</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.172</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">Negative -</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.000</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">-0.001</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">Test </SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.076</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.167</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">+</SPAN><SPAN class="c10 c3 c2">1</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.343</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.303</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">-</SPAN><SPAN class="c10 c3 c2">1</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.001</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.009</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">T</SPAN><SPAN class="c10 c3 c2">1</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.353</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.310</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">+</SPAN><SPAN class="c10 c3 c2">2</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.748</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.776</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">-</SPAN><SPAN class="c3 c2 c10">2</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.097</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.101</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">T</SPAN><SPAN class="c10 c3 c2">2</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.728</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.749</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">+</SPAN><SPAN class="c10 c3 c2">3</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">1.144</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">1.177</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">-</SPAN><SPAN class="c10 c3 c2">3</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">-0.002</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">-0.001</SPAN></P></TD></TR><TR class="c18"><TD class="c0" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c2">T</SPAN><SPAN class="c10 c3 c2">3</SPAN></P></TD><TD class="c32" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">1.072</SPAN></P></TD><TD class="c42" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">1.102</SPAN></P></TD></TR></TBODY></TABLE><P>The Lux Rudolph Test failed because the negative control after the third hour was negative, implying there was no bacteria in the stock tube to begin with. The experiment will be redone on Monday.</P><P>LuxRudolph Diagram (left) | Lsr Operon (right)</P><P><B>Ligation of LuxR and LacZ:</B></P><P>RI: I0500</P><P>RU: J37019</P><P>RD: LacZ</P><P>RB: Amp. Resistant </P><P><B>Upstream + Downstream + Backbone:</B></P><OL><LI>(RI + RA) + RB</LI><LI>(RU + RA) + RB</LI><LI>(RI + RD) + RB</LI><LI>(RU + RD) + RB</LI></OL><P><B>Saline Experiment:</B></P><P>We plated the salt solutions that were made from about two weeks ago. The colonies on the plate that weren’t supposed to be fed LB were smaller, the plate had more colonies  than the other one, which has a smaller amount of larger colonies.</P><P><B>Lsr Operon</B></P><P>The Lsr Operon has RBS within its OFR/CDS, unlike normal circuits where the RBS is prior to the OFR.</P><P>This means that when a ribosome starts translation, it is makes 4 proteins from a highly conserved sequence.</P><P>This might be good for biological systems, but in terms of modular design it is not as efficient. Since the OFR contains a RBS, if you were to attempt to combine the various OFRs with a separate RBS, it takes twice as many RBS’s.</P><P>In Open Reading Frames, the start codon is the first three bases (AUG).</P><P>A Ribosome Binding Site, is what the ribosome binds and the OFR comes after. A common sequence is the Shine-Dalgarno that is usually 8 bases upstream of the start codon AUG. The six-base sequence is AGGAGG.</P><P>In order to move around this problem and prevent using double the amount of needed RBS’s, we will change the sequence to code for the same proteins, but with different bases. Thus utilizing the degenerative nature of DNA.</P></DIV></DIV><DIV class="week row"><H3 class="header">Week 6: Aug 3 - Aug 7</H3><DIV class="section entry"><H2 class="lead">Monday (8/3/15) </H2><P>Today, plasmid protocol was done on the LSR operon from last week to see if the assembly worked.</P></DIV><DIV class="section entry"><H2 class="lead">Tuesday (8/4/15)</H2><P>Transported RDP parts “Lux” and “Rudolf” to Cooper Union to measure DNA concentration using their Nanodrop spectrophotometer:</P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">Sample</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c21 c2">A</SPAN><SPAN class="c41 c3 c21 c2">260</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c3 c21 c2">A</SPAN><SPAN class="c3 c21 c2 c41">280</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c27 c3 c21 c2">[DNA]</SPAN></P></TD></TR><TR class="c18"><TD class="c34" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">Lux</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.850</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.415</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">42.5ng/μL</SPAN></P></TD></TR><TR class="c18"><TD class="c34" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c3 c2">Rudolf*</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">1.095</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">0.531</SPAN></P></TD><TD class="c34" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c3 c2">54.5ng/μL</SPAN></P></TD></TR></TBODY></TABLE><P>*Rudolf had an additional absorbance peak around 220nm.  A<SUB>220</SUB>   was approx. 2.8</P><P>Synbiota recommends 0.04pM/μL of sample DNA per reaction.</P><UL><LI>1pM/μL = 670ng/μL</LI><LI>Lux: 42.5ng/μL = 0.0634pM/μL</LI><LI>Rudolf: 54.5ng/μL = 0.0813pM/μL</LI></UL><P>Lux dilution factor: 1.585</P><P>Rudolf dilution factor: 2.033</P></DIV><DIV class="section entry"><H2 class="lead">Wednesday (8/5/2015)</H2><P>The Miller Assay using Galactosidase and ONPG (yellow fluorescent). We did a digest and then a transformation of the failed 3A Assembly from a couple of weeks ago.</P></DIV><DIV class="section entry"><H2 class="lead">Thursday (8/5/2015)</H2><P><B>Objective:</B>Redoing the Miller Assay </P><P>First try - failed, used 500 microliters of cells (turned yellow too quickly)</P><P>Second attempt - 1 microliter of cells</P></DIV></DIV><DIV class="week row"><H3 class="header">Week 7: Aug 10 - Aug 14</H3><DIV class="section entry"><H2 class="lead">Tuesday (8/11) </H2><P>5 Ordered modularized parts were received from IDT: LsrR, LsrD, LsrB, LsrC,NsrR</P><P>Plans to lyophilize today. </P><P><B>Resuspension Protocol:</B></P><OL><LI>Centrifuge</LI><LI>Add 100 µl TE Buffer</LI><LI>Vortex</LI><LI>Centrifuge</LI><LI>Place in 50C H2O for 20 minutes</LI></OL><H4 class="underline">
              ‘2A’ Assembly:
            </H4><P><B>Master Mix (6x)</B></P><UL><LI>6µl NE buffer 2.1</LI><LI>3µl EcoRI</LI><LI>4µl EcoRI</LI></UL><P><B>Digest:</B></P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c44" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Part</SPAN></P></TD><TD class="c28" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Backbone</SPAN></P></TD></TR><TR class="c18"><TD class="c44" colspan="1" rowspan="1"><UL class="c39 lst-kix_uaqzvq10zeqc-0 start"><LI class="c6"><SPAN class="c19 c2">8µl DNA</SPAN></LI><LI class="c6"><SPAN class="c19 c2">2µl MM</SPAN></LI></UL></TD><TD class="c28" colspan="1" rowspan="1"><UL class="c39 lst-kix_26ssmkt2zslm-0 start"><LI class="c6"><SPAN class="c19 c2">5µl DNA</SPAN></LI><LI class="c6"><SPAN class="c19 c2">2.5µL MBGW</SPAN></LI><LI class="c6"><SPAN class="c19 c2">1µL NEB 2.1</SPAN></LI><LI class="c6"><SPAN class="c19 c2">0.5µL EcoRI</SPAN></LI><LI class="c6"><SPAN class="c19 c2">0.5µL PstR</SPAN></LI><LI class="c6"><SPAN class="c19 c2">0.5µL BsaL</SPAN></LI></UL></TD></TR></TBODY></TABLE><P><B>Ligation:</B></P><OL><LI>Combine
                </LI><LI>6.5µl Part Digest</LI><LI>2.0µl Backbone Digest</LI><LI>1.0µl Ligation Buffer</LI><LI>0.5µl T4 DNA Ligase</LI><LI>Ligate @ 16C for 30 min</LI><LI>Heat kill @ 80C for 20 min</LI><LI>Heat kill @ 4C for ∞</LI></OL></DIV><DIV class="section entry"><H2 class="lead">Thursday (8/13)</H2><P>Worked in: <A href="http://gentle-beta.synbiota.com">gentle-beta.synbiota.com</A></P></DIV></DIV><DIV class="week row"><H3 class="header">Week 8: Aug 17 - Aug 22</H3><DIV class="section entry"><H2 class="lead">Tuesday (8/18)</H2><P>Rapid DNA Prototyping for Lux + Rudolph circuit, as reviewing reveals that the previous circuit was not properly made.</P><P>Bio Brick Structure:</P><P>dA-Anchor (Z’) =&gt; (Z) ADP (X’) =&gt;(X) Lux (Z’) =&gt;(Z) RBS (X’) =&gt;(X) Rud (Z’) =&gt;(Z)  Cap-dT</P><P>Transforming competent cells </P><P><B>Miller Assay Results:</B></P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">AHL Positive (+) (1 mM)</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">AHL Negative (-) (0 mM)</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">OD 600</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">1.315</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">1.268</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">OD 420</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">0.888</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">0.972</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">OD 550</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">0.012</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">0.007</SPAN></P></TD></TR></TBODY></TABLE><P>AHL Positive Miller Units : 7911.79</P><P>AHL Negative Miller Units: 9082.81</P><P>Results were not as expected. Full reaction took ⅚ of a minute, at 1:10 dilution (.1 mL). The - is able to stay in cells longer, while in log phase, while the + is quickly dispersed and expressed in log phase.</P><P><B>“Slow and steady” hypothesis:</B></P><P>The - cells are able to become more turgid/filled with galactosidase enzyme than the + cells.</P></DIV></DIV><DIV class="week row"><H3 class="header">Week 9: Aug 24 - Aug 28</H3><DIV class="section entry"><H2 class="lead">Monday (8/24)</H2><P>Gel was made to prepare for LR Digest</P><P><B>Findings thus far:</B></P><UL><LI>“The truth will set you free” - Caltech</LI><LI>“... But first it will make you sick.” - Joel Franklin</LI></UL><P><B>Plan A (L+R):</B></P><UL><LI>Plux -&gt; RBS -&gt; LuxR -&gt; RBS -&gt; Rudolph -&gt; Terminator</LI></UL><P>AHL is 200 Daltons. This is a positive feedback look.</P><P><B>Simple Model:</B></P><UL><LI>If [AHL], then Rudolph is produce.</LI><LI>If <SPAN class="strike">[AHL]</SPAN>, then Rudolph is not produced.</LI></UL><P><B>Analogous Circuit (L+L):</B></P><UL><LI>Plux -&gt; LuxR -&gt; LacZ</LI></UL><P><B>Overnight Experiments (L+L)</B></P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c46" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2"> |-&gt;</SPAN></P></TD><TD class="c50" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">1mM AHL @ 37C (+ Positive control)</SPAN></P></TD></TR><TR class="c18"><TD class="c24" colspan="1" rowspan="1"><P class="c11"><SPAN class="c2">Glycerol Stock Scrape</SPAN></P></TD><TD class="c46" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">LB + Ampicillin Broth</SPAN></P></TD></TR><TR class="c18"><TD class="c46" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">|-&gt;</SPAN></P></TD><TD class="c50" colspan="1" rowspan="1"><P class="c11"><SPAN class="c2">0mM AHL @ 37C (- Negative control)</SPAN></P></TD></TR></TBODY></TABLE><P>Using the Miller Assay it was revealed the + had 8000 MU, and the - had 9000 MU. Future tests revealed that the - had consistently higher amounts of ß-galactosidase.</P><P><B>Concentration Change Test:</B></P><P>The Negative control has no AHL in it’s surrounding, or within in its cells after the experiment.</P><P>The Positive control has AHL in it’s surrounding and within in its cells after the experiment.</P><P>The Test has AHL in it’s surrounding, but none in it’s cells at the beginning. And at the end the AHL has diffused into its cells.</P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Negative</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Test</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Positive</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Start with Negative O/N (1:100) dilution</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Start with Negative O/N (1:100) dilution</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">Start with Positive O/N (1:100) dilution</SPAN></P></TD></TR><TR class="c18"><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">+0mM AHL</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">+1mM AHL</SPAN></P></TD><TD class="c14" colspan="1" rowspan="1"><P class="c11"><SPAN class="c19 c2">+1mM AHL</SPAN></P></TD></TR></TBODY></TABLE><P>
              All three of these samples were placed in a rotating incubator, sampled every hour, sample in spectrophotometer at OD 600, and then the Miller Assay was done for each. The OD 600 revealed the growth functions of both the Negative and Positive were normal, but the Test growth was severely retarded. Consequently the Miller Assay revealed that the amount of ß-gal in the Negative and Positive steadily decreased over time, while the Test has a constant large amount present. This implies that the Test cells are focussing more on the production of a protein, than on their own growth.
            </P><P><B>The Flanagan Hypothesis</B></P><P>The delay factor in growth between the Negative/Positive and Test is 12:1. A test with differing AHL concentrations would prove this.</P><P><B>Sampling Experiments (L+R):</B></P><TABLE cellpadding="0" cellspacing="0" class="c33"><TBODY><TR class="c45"><TD class="c40" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">POS O/N </SPAN></P><P class="c8"><SPAN class="c19 c2">L+R in LB Amp</SPAN></P><P class="c8"><SPAN class="c19 c2">1mM AHL</SPAN></P></TD><TD class="c23" colspan="1" rowspan="3"><P class="c8"><SPAN class="c19 c2">Log growth 1:100 Dilution</SPAN></P></TD><TD class="c52" colspan="1" rowspan="3"><P class="c8"><SPAN class="c19 c2">Add 5µL LB</SPAN></P></TD><TD class="c40" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">+1 mM AHL</SPAN></P></TD><TD class="c40" colspan="1" rowspan="3"><P class="c8"><SPAN class="c19 c2">Incubate @ 37C</SPAN></P></TD></TR><TR class="c45"><TD class="c40" colspan="1" rowspan="2"><P class="c8"><SPAN class="c19 c2">NEG  O/N </SPAN></P><P class="c8"><SPAN class="c19 c2">L+R in LB Amp</SPAN></P><P class="c8"><SPAN class="c19 c2">0mM AHL</SPAN></P></TD><TD class="c40" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">+0 mM AHL</SPAN></P></TD></TR><TR class="c45"><TD class="c40" colspan="1" rowspan="1"><P class="c8"><SPAN class="c19 c2">+1 mM AHL</SPAN></P></TD></TR></TBODY></TABLE></DIV><DIV class="section entry"><H2 class="lead">Tuesday (8/25)</H2><P><B>In preparation of Columbia Fluorometer:</B></P><P>Gel was prepared for a digest of RBS parts and promoters:</P><OL><LI>RBS A</LI><LI>RBS B</LI><LI>RBS C</LI><LI>pReaR</LI><LI>pLsrA2</LI></OL></DIV><DIV class="section entry"><H2 class="lead">Wednesday (8/26)</H2><P>OD 600 readings</P></DIV><DIV class="section entry"><H2 class="lead">Thursday (8/27)</H2><P>OD 600 readings</P></DIV><DIV class="section entry"><H2 class="lead">Friday (8/28)</H2><P><B>PCR Cleanup of:</B></P><UL><LI>NsrR RDP</LI><LI>LsrR RDP</LI><LI>LsrA RDP</LI><LI>LsrB RDP</LI><LI>LsrC RDP</LI><LI>LsrD RDP</LI><LI>LuxR RDP</LI><LI>pLux RDP</LI></UL><P>Gel was pre made (courtesy of Will) for the samples for electrophoresis.</P></DIV></DIV><DIV class="week row"><H3 class="header">Week 10: Aug 31 - Sept 4</H3><DIV class="section entry"><H2 class="lead">Monday (8/31)</H2></DIV><DIV class="section entry"><H2 class="lead">Tuesday (9/1)</H2><P>washes &amp; labeling</P></DIV><DIV class="section entry"><H2 class="lead">Wednesday (9/2)</H2><P>Minipreps</P><P>Made plates   </P></DIV><DIV class="section entry"><H2 class="lead">Thursday (9/3)</H2>
            plasmid prep &amp; digest 
          </DIV><DIV class="section entry"><H2 class="lead">Friday (9/4)</H2><P><B>Four different plasmids were made using parts from synbiota:</B></P><P>dA18-AmpR-2’  --  2-adp-x’   --   x-pr.3-2’   --   2-Rbs.4.tx’   --   x-LacI-2’   --2-Ori.2-dT18</P><P>dA18-ChlrRJx’ -- x-pr.lacO-2’ -- 2-Rbs.2.tx’ -- x-dRFP-2’ -- 2-Ori3-dT18</P><P>dA18-ChlrR-x’ -- x-pr.lacO-2’ -- 2-Rbs.2.tx’ -- x-LsrR-2’ -- 2-Ter-x’ -- x-pLsrA-2-2’ -- 2 </P></DIV></DIV></DIV></DIV><FOOTER class="footer"><DIV class="container"><P class="text-muted">Place sticky footer content here.</P></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>