<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SZU-China_Procedure skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SZU-China/Procedure</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="ibm-top" class="ibm-landing-page"><DIV id="ibm-content-wrapper"><HEADER role="banner" aria-labelledby="ibm-pagetitle-h1"><DIV class="ibm-sitenav-menu-container " style="top:7px" data-widgetprocessed="true"><DIV class="ibm-sitenav-menu-list"><UL role="menubar"><LI role="presentation"><A role="menuitem" href="https://2016.igem.org/Team:SZU-China/Achievements" aria-label="Blockchain Basics" onclick="undefined" tabindex="0">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Achievements">Awards</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Medal">Medal</A></LI></UL><LI role="presentation"><A role="menuitem" href="https://2016.igem.org/Team:SZU-China/Description" aria-label="Blockchain for Business" onclick="undefined" tabindex="-1">Project</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Safety">Safety</A></LI><LI role="presentation"><A role="menuitem" href="https://2016.igem.org/Team:SZU-China/Procedure" onclick="undefined" tabindex="-1">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Procedure">Procedure</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Parts">Parts</A></LI><LI role="presentation"><A role="menuitem" href="https://2016.igem.org/Team:SZU-China/Outreach" onclick="undefined" tabindex="-1">Outreach</A></LI><LI role="presentation"><A role="menuitem" href="https://2016.igem.org/Team:SZU-China/HP/Gold" onclick="undefined" tabindex="-1">Practice</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/HP/Gold">Gold HP</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/HP/Silver">Silver HP</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Integrated_Practices">Integrated HP</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Engagement">Engagement</A></LI><LI role="presentation"><A role="menuitem" href="https://2016.igem.org/Team:SZU-China/Team" onclick="undefined" tabindex="-1">Team</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Team">People</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Collaborations">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:SZU-China/Attributions">Attribution</A></LI></DIV></DIV><DIV id="ibm-leadspace-head" class="ibm-alternate-background" style="background: url(https://static.igem.org/mediawiki/2016/b/b5/T--SZU-China--experiment01.jpg) no-repeat 50% 50% /100% auto;"><DIV id="ibm-leadspace-body" style="min-height:210px;padding-top:20px"><DIV class="ibm-columns"><DIV class="ibm-col-6-3 ibm-alternate-background "><H1 class="ibm-h1" style="font-weight:bold">
Procedure</H1></DIV></DIV></DIV></DIV></HEADER><MAIN role="main"><DIV id="ibm-pcon"><DIV id="ibm-content"><DIV id="ibm-content-body"><DIV id="ibm-content-main"><DIV class="ibm-columns"><DIV class="ibm-col-6-5 ibm-center-block ibm-col-medium-1-1 ibm-padding-top-r1"><DIV class="ibm-pull-quote ibm-h2"><H2 class="ibm-h2 ibm-center">For our experiment this year，there are 3 main steps:</H2><CENTER><P>1.Construct two expression vectors </P></CENTER><CENTER><P>2.Transform expression vectors into Chlamydomonas reinhardtii and verify positive transformants</P></CENTER><CENTER><P>3.Detect Hydrogen yield</P></CENTER><H2 class="ibm-h2 ibm-center">1. Construct 2 expression vector</H2><P> We constructed two expression vectors for our project，they are pDb124-BD-CIB1 and pDh124-VP16-CRY2-UAS-miRNA.The vectors contains the following parts：GAL4 BD：The DNA binding domain of yeast transcription activator protein.PCR from plasmid pGBKT7. </P><P><SPAN class="a">VP16 AD:</SPAN> The activation domain of herpes simplex virus,which has transcriptional activation activity.We made codon optimization for the VP16 AD,according to the codon bias of Chlamydomonas reinhardtii.Synthesized by biological company.</P><P><SPAN class="a">UAS:</SPAN> Upstream activating sequence,can bind with GAL4 BD.Synthesized by biological company.</P><P><SPAN class="a">CRY2:</SPAN> Cryptochrome2, the blue light receptor protein.PCR form Arabidopsis thaliana's cDNA</P><P><SPAN class="a">CIB1:</SPAN> Blue light receptor interacting protein,combing with CRY2 under blue light.PCR form Arabidopsis thaliana's cDNA.</P><P><SPAN class="a">amiRNA-D1 and amiRNA-FNR:</SPAN> Designed by ourselves，exchanging some sequence of miRNA1162，can inhibit the expression of D1 and FNR. Synthesized by biological company.
</P><P>The vector backbones we use in this project is the two kind of vectors we usually use in our lab——pDp124 and pDh124.</P><P>Inside the vectors，5'PsaD is the 5' sequence located before the gene coding region of PsaD，it is a constitutive promoter which can promote the two fusion protein——GAL4 DNA binding domain-CIB1 and VP16-CRY2.3' PsaD is the corresponding 3' sequence of PsaD，it plays an important role in constitutive high-expression of exogenous gene.Paro is responsible for the resistance gene of Paromomycin. Hyg is responsible for the resistance gene of Hygromycin B，we can use Paromomycin and Hygromycin B to verify positive transformants.</P><P><SPAN class="a">amiRNA-D1、amiRNA-FNR:</SPAN> designed by ourselves，exchanged some sequence of miRNA1162，can inhibit the expression of D1 and FNR. Synthetized by biological company.</P><P>The vector backbones we use in this project is the two kind of vectors we usually use in our lab——pDp124 and pDh124</P><H2 class="ibm-h2 ibm-center">2. The vectors construction steps are as the followings: </H2><P>1) Construct GAL4 BD-CIB1 sequence. We used PCR to add two restriction sites PmaC I and Pst I on both sides of GAL4-BD and then connected it in the intermediate carrier for cloning. The 3' terminal sequence intrinsic included the restriction sites Nde I and Pst I. Then we used PCR to add restriction site Nde I and Pst I on both sides of CIB 1's CDS sequence. Thus we can use the two same restriction sites to inserted CIB1 into the 3' terminal of GAL4-BD.</P><P>2) Construct VP16-CRY2-UAS sequence. We used the method of total gene synthesis to synthesis VP16-UAS sequence, and designed the Nde I and Cla I restriction sites between VP16 sequence and UAS sequence. Considered that the sequence intrinsic included PmacC and Nhe I restriction sites, we added PmaC I's isocaudarner and Nhe I's isocaudarner restriction sites on both sides of VP16-UAS's sequence. Then the synthesized sequence was inserted into the intermediate carrier for cloning. Through PCR, we added two restriction sites Nde I and Cla I on both sides of CRY2's CDS sequence. Through the enzyme digestion of the two restriction sites and the connection of CRY2 sequence between VP16 sequence and UAS sequence, eventually developed the fusion protein with VP16 sequence.</P><P>3) Construct pDb124-BD-CIB1 vector. Cleavage the PmaC I and Nhe I so that the constructed GAL4 BD-CIB1 sequence was cut from the intermediate carrier and inserted it into pDh124 carrier which is linearized PmaC and Nhe I restriction site, then we obtained pDh124-VP16-CRY2-UAS carrier.</P><P>4) Construct pDh124-VP16-cry2-UAS vector. Cleavage the Sma I and Xba I so that the constructed VP16-CRY2-UAS sequence was cut from the intermediate carrier and inserted it into pDh124 carrier which is linearized PmaC and Nhe I restriction site, then we obtained pDh124-VP16-cry2-UAS vector.</P><P>5) Construct pDh124-VP16-CRY2-UAS-miRNA vector. Selecting the desired natural miRNA or a manually engineered amiRNA sequence specific to the genes that affect hydrogen production, replacing the mature RNA sequence in the naturally abundant miRNA 1162 precursor of the highly abundant expression, a vector for constructing the vector and transforming of the miRNA precursor sequence. VP16-CRY2-UAS-miRNA vector was obtained by synthesizing the miRNA precursor sequence and linking it to the UAS sequence of pDh124-VP16-CRY2-UAS vector.</P><P>The final pDb124-BD-CIB1 vector with Paromomycin resistance and the pDh124-VP16-CRY2-UAS-miRNA vector with Hygromycin B resistance will be used for the transformation of C.reinhardtii.</P><H2 class="ibm-h2 ibm-center">3.Transform expression vectors into Chlamydomonas reinhardtii</H2><P>The chlamydomonas reinhardtii we use in our project is CC-849(bought from Chlamydomonas Genetic Center, Duke University，Durham，NC27708 USA). This kind of Chlamydomonas has some defect in cell wall,so we can have a higher transformation efficiency.</P><P>We transformed the two expression vectors into Chlamydomonas reinhardtii by glass bead method. </P><P>First, the pDp124-BD-CIB1 vector was transferred into C. reinhardtii CC-849 by &quot;glass bead&quot; method. The green clones were obtained by Paromomycin resistance screening. PCR identification of the transformants, genomic DNA and sequencing of the PCR products yielded real Positive clones. The full - length sequences of GAL4 BD and CIB1 were amplified by PCR using the genomic DNA of the transformants as template, and the bands were sequenced.</P><P>Next, the pDp124-BD-CIB1-positive strain was re-transformed into the pDh124-VP16-CRY2-UAS-miRNA vector by &quot;glass bead&quot; method. The green clones were screened by Hygromycin B resistance, and PCR was carried out again by transforming genomic DNA Identification and sequencing of the PCR products yielded positive clones containing both pDb124-BD-CIB1 and pDh124-VP16-CRY2-UAS-miRNA. The full-length sequences of VP16, CRY2 and miRNA precursors were amplified by PCR using the genomic DNA of the transformants as template, and the product bands were sequenced. The Paromomycin and Hygromycin B double-resistant strains were transgenic algae strains with photo-controlled hydrogen production system.</P><H2 class="ibm-h2 ibm-center">4.Detect Hydrogen yield measured by gas chromatography.</H2><P>①We cultivated the gene-modified green algae in blue light ( The wave length is about 450 nm) and induced the expression of relevant miRNA. When they reached logarithm stage, we measured hydrogen production.</P><P>②We also cultivated the gene-modified green algae in the condition lacking in blue light.  </P><P>When the green algae reached logarithm stage, we used blue light, whose wave length is about 450 nm, to induce the expression of the relevant miRNA and increase hydrogen production. First we used blue light to induce the expression of miRNA and measured hydrogen production every 4 hours. After 8 hours, we re-cultivated the green algae in red light. After 16 hours, again we used blue light to induce the expression of miRNA and measured hydrogen production every 4 hours for 12 hours.In order to proof the efficiency for the light-mediated expression system.</P><P>(Red light LED plant growth lamp, peak wave length: 660nm, half width: 17.9nm, dominant wave length: 642.4 nm, purity: 0.989).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></MAIN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>