<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NKU_China_proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NKU China/proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main" id="menu-bar"><NAV class="menu"><A class="menu-item" href="https://2016.igem.org/Team:NKU_China">Home</A><DIV class="menu-item">
                Team<SPAN class="default">☞</SPAN><SPAN class="active">☟</SPAN><DIV class="sub-menu"><A class="sub-item ui-state-default" href="https://2016.igem.org/Team:NKU_China/Team">Team</A><A class="sub-item ui-state-default" href="https://2016.igem.org/Team:NKU_China/Collaborations">Collaborations</A></DIV></DIV><DIV class="menu-item">
                Project<SPAN class="default">☞</SPAN><SPAN class="active">☟</SPAN><DIV class="sub-menu"><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Description">Description</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Experiments">Experiments</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Model">Model</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Proof">Proof of Concept</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="menu-item">
                HP<SPAN class="default">☞</SPAN><SPAN class="active">☟</SPAN><DIV class="sub-menu"><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Human_Practices">HP</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/HP/Silver">Silver</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/HP/Gold">Gold</A></DIV></DIV><A class="menu-item" href="https://2016.igem.org/Team:NKU_China/Safety">Safety</A><A class="menu-item" href="https://2016.igem.org/Team:NKU_China/Attributions">Attributions</A><A class="menu-item" href="https://2016.igem.org/Team:NKU_China/Parts">Parts</A><A class="menu-item" href="https://2016.igem.org/Team:NKU_China/InterLab">InterLab</A><DIV class="menu-item">
                Notebook<SPAN class="default">☞</SPAN><SPAN class="active">☟</SPAN><DIV class="sub-menu"><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Notebook">Lab Notes</A><A class="sub-item" href="https://2016.igem.org/Team:NKU_China/Protocols">Protocols</A></DIV></DIV></NAV><DIV id="menu-bar-placeholder">
            you can not see me!
        </DIV></DIV><DIV class="main" id="proof"><DIV class="h1">Proof of Concept</DIV><SECTION><DIV class="h2">Fluorescent Microscopy</DIV><DIV class="p">The genetic switch systems that are being characterised make use of two different reporter genes, namely green fluorescent protein (GFP) and red fluorescent protein (RFP). These will give a visual indication as to the functioning of the parts being tested. The biobricks, BBa_I13521 and BBa_I13522 were transformed into E. coli and serve as positive controls for fluorescent microscopy. These two parts consist of the reporter genes, GFP and RFP respectively and are under the control of a constitutive promoter.</DIV><FIGURE style="max-width:45rem"><FIGCAPTION>Figure 1. The fluorescent microscopy positive controls GFP (BBa_I13521) and RFP (BBa_I13522).</FIGCAPTION></FIGURE></SECTION><SECTION><DIV class="h2">Cloning of Parts</DIV><DIV class="p">The parts required for the characterisation of an inverter switch as well as a switch based on excision were synthesised by Integrated DNA technologies as part of their promotion to iGEM teams. These gBlocks were received and blunt end cloned into pUC19 plasmids to create high copy number of our parts. The inserts were then retrieved by restriction digests of the flanking EcoRI and PstI sites for sticky end cloning into the pSB1C. The digests were loaded onto a 1% agarose gel for electrophoresis to separate linear vector from inserts. The bands corresponding to the respective parts being characterised were excised and purified using a Zymo Gel purification kit (Inverter 1, BBa_K1768001; Inverter 2 BBa_K176800; Cre generator, BBa_K1768004).</DIV><FIGURE style="max-width:66rem"><FIGCAPTION>Figure 2. Restriction digest of Inverter 1 (Lane 1), Inverter 2 (Lane 2), Cre generator (Lane 3) and their respective undigested plasmids (Lanes4-6). </FIGCAPTION></FIGURE><DIV class="p">The purified inserts were ligated into pSB1C for part submission and the Cre generator BBa_K1768004 ligated into pSB1A for experimentation. These vectors were supplied with the 2015 Distribution Kit and were digested with EcoRI and PstI to create complimentary sticky ends to the inserts. Cultures were grown from the resulting colonies and screened for biobrick parts with the standard VF and VR primer combination. The amplicons were electrophoresed and the size of fragments were referenced to the expected sizes of the fragments for the respective parts.</DIV><FIGURE style="max-width:50rem"><FIGCAPTION>Figure 3. PCR of biobrick parts in pSB plasmids. Lanes 1,2 Cre generator in pSB1A; Lanes 3,4, Cre generator; Lanes 5,6 Inverter 1; Lanes 7,8 GFP Reverse Compliment, Lanes 9,10 Recombinase ON switch. </FIGCAPTION></FIGURE></SECTION><SECTION><DIV class="h2">Sequencing Parts</DIV><DIV class="p">The parts were sequenced in the pSB plasmids to confirm their integrity by Sanger method using the sequencing facility at The University of Pretoria. The reactions were set up using BigDye 3.1, its buffer and 150 ng of the respective plasmids as template. Forward and reverse reactions were run using VF and VR primers. Unfortunately when analysing the sequence data it was found that the Cre generator (BBa_K1768004) construct contained two significant mutations in the coding region of the construct. The seven bp deletion causes frame shirt and would render the recombinase protein deactivated. The other gBlock fragment sequences were correct with no mutations.</DIV><FIGURE style="max-width:60rem"><FIGCAPTION>Figure 4. Chromatogram showing the two deletions in the Cre construct from two different sequenced constructs.</FIGCAPTION></FIGURE><DIV class="p">Due to time constraints the project will regrettably be completed after the jamboree. The original synthesised gBlock for the Cre recombinase will be re-cloned to create a functional plasmid set for the characterisation of the various switches proposed.</DIV></SECTION><SECTION><DIV class="h2">Future</DIV><DIV class="p">In order to assess that the excision switch and inverter constructs function as expected an experiment will be conducted where the Cre protein expression is activated and thus characterise the behaviour of the biobricks.</DIV><DIV class="p">E. coli colonies containing both the Cre protein and the Recombination switch plasmids will be selected. The respective vectors contain Chloramphenicol and Ampicillin resistant genes and will be cultured under these selective agents in LB media. A series of 5 aliquots of the culture during log phase growth will be induced. These will each containing a different concentration of IPTG (0.1-1.0 mM). The presence of IPTG activates Cre expression through pLac promoter, which in turn leads to recombination. The expected result of recombination will be the expression of RFP.</DIV></SECTION></DIV><DIV class="main" id="foot"><DIV><DIV id="sponsors" class="fooer-column"><DIV class="p">sponsors</DIV></DIV><DIV id="contact" class="fooer-column"><DIV class="p">contact us</DIV><DIV id="address"><DIV class="p"><SPAN>Email:</SPAN> iGEM_NKU_China@hotmail.com</DIV><DIV class="p"><SPAN>Website:</SPAN> 2016.igem.org/Team:NKU_China</DIV><DIV class="p"><SPAN>Address:</SPAN> Nankai University,
                        No.94 Weijin Road, Nankai District
                        Tianjin, P.R.China 300071
                    </DIV></DIV></DIV></DIV><DIV class="p" id="copyright">© NKU_China IGEM - All rights reserved. Based on jQuery and bootstrap.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>