Team:UiOslo Norway/Results
HOME
PROJECT
Description
Results
Achievements
TEAM
Team
Attributions
Acknowledgement
NOTEBOOK
Notebook
Experiments & Protocols
PARTS
Team Parts
Basic Parts
HUMAN PRACTICES
Human Practices
Collaborations
SAFETY
Project Results
The results are described according to the three sub-goals. (Click on a goal to directly scroll down).
Breakdown of methane to methanol in E. coli: Using the enzyme complex soluble methane monooxygenase (sMMO) from Methylococcus capsulatus (Bath).
Conversion of methanol to biomass in E. coli (based on Müller et. al 2015) Using the following enzymes from Bacillus methanolicus MGA3; *  Methanol to formaldehyde with the enzyme methanol dehydrogenase (medh2) *  Formaldehyde to hexulose-6-phosphate with the enzyme 3-hexulose-6-phosphate synthase (hxlA) *  Hexulose-6-phosphate to fructose-6-phosphate with the enzyme 6-phospho-3-hexuloisomerase (hxlB)
Air-filter containing the engineered E. coli A closed container to restrict the genetically modified E. coli. The surrounding air will be pumped through the filter so the methane gas can be taken up by E. coli and be converted to biomass.
1. Breakdown of methane into methanol in E. coli.
Back to top
The first goal of our project was to engineer E. coli for sufficient conversion of methane to methanol using the Methylococcus capsulatus (M. capsulatus) soluble methane monoxygenase (sMMO) We received the BioBricks BBa_K1390001 , BBa_K1390002 , BBa_K1390003 , BBa_K1390004 , BBa_K1390005 , BBa_K1390006 containing the subunits of the M. capsulatus sMMO from the iGEM Team Braunschweig 2014 .  In addition we had the intention to add the GroEL-like chaperon mmoG, which is assumed to have an essential role in correct folding of the multi subunit enzyme complex sMMO. In an initial step we wanted to show particular protein expression of each subunit into E. coli. All subunits were cloned into our designated expression vector pET-30 and all subunits, except MMOC, were expressed (Figure 1).
Figure 1: Pilotexpression of MMOX, MMOY, MMOZ, MMOB, MMOC, MMOD in E. coli. Coomassie staining of pilotexpression of MMOX, MMOY, MMOZ, MMOB, MMOC and MMOD. Proteinexpression was performed for 3 hours. Samples were taken in a not induced stage (n.i.) and 3 hours after induction (a.i.)(A) Pilotexpression of MMOX, MMOY, MMOZ. (B) Pilotexpression of MMOB, MMOC, MMOD. Theoretical molecular weight:MMOC-38.48 kDa, MMOG-59.38 kDa, MMOX-60.56 kDa, MMOY-45.06 kDa, MMOB-15.95 kDa, MMOD-11.91 kDa, MMOZ-19.81 kDa. Orange arrows indicate expressed sMMO subunit. As ladder PageRuler™ Plus Prestained Protein Ladder, 10 to 250 kDa (ThermoScientific) was used. As negative control (NC) selfligated pET-30 was used for expression in E. coli BL21. Proteins were seperated by SDS-Page.
Further we tested individually for soluble heterologous expression of each subunit. We were able to show that MMOX, MMOZ, MMOB, MMOD were soluble expressed at 37 °C. MMOY was completely aggregated in inclusion bodies (Figure 2)
Figure 2: Solubility of MMOX, MMOY, MMOZ, MMOB and MMOD. Coomassie staining of solubility assay of MMOX, MMOY, MMOZ, MMOB and MMOD. Proteinexpression was performed for 3 hours. Cell pellet was lysed and separated into soluble (sol) and insoluble (insol) Fraction. As control 1 ml sample was taken in an uninduced stage (n.i.) . Theorteical molecular weight: MMOX-60.56 kDa, MMOY-45.06 kDa, MMOZ-19.81 kDa, MMOB-15.95 kDa, MMOD-11.91 kDa. Proteins were separated by SDS-PAGE. Orange arrows determine the appropiate protein. As ladder PageRuler™ Plus Prestained Protein Ladder, 10 to 250 kDa (ThermoScientific) was used. As negative control (NC) selfligated pET-30 was used for expression in E. coli BL21. Proteins were separated by SDS-PAGE.
For soluble expression lowering of growth temperature or the addition of chaperons (see iGEM Team Braunschweig 2014 )  might be potential approaches.
In addition we wanted to clone the GroEL like chaperon MMOG. We isolated mmoG from Methylococcus capsulatus genomic DNA. Afterwards we cloned mmoG into pET-30. In this construct we found in the middle of our gene a deletion of 8 base pairs.  Due to time concerns we decided to synthesize mmoG, using the IDT 20 kb free offer for iGEM Teams. We were trying to amplify mmoG using our design primers which was not sufficient. Due to time concerns we were not able to finish our mmoG BioBrick, as well as showing successful expression of mmoG in E. coli.
Summary:
We generated expression constructs for all sMMO subunits.
We showed soluble expression of MMOX, MMOZ, MMOB, MMOD.
Generation of mmoG as a BioBrick part needs further improvement.
2. Conversion of methanol to biomass in E. coli
Back to top
Our second goal was to engineer E. coli for the conversion of methanol into biomass, a process that involves three different steps. In the first step methanol is converted into formaldehyde by the NAD+ dependent methanol dehydrogenase MEDH2 from Bacillus methanolicus MGA3.  We genereated a basic BioBrick part (BBa_K1619001) , which contains the medh2 coding sequence.  We further cloned medh2 into pET-28 and showed soluble expression of MEDH2 in E. coli at 18 °C (Figure 3).
Figure 3: Pilotexpression and Solubility assay of heterologous expressed MEDH2 in E. coli. Coomassie staining of heterologously expressed MEDH2 in pET-28. 1 ml samples were taken in an uninduced stage (n.i.) and after expression overnight at 18 °C. Theoretical weights: MEDH2- 40.15 kDA. Cell pellet was lysed by sonication and the soluble fraction of all cultures was pooled and analyzed (sol).Orange arrows determine the appropiate protein. Proteins were separated by SDS-PAGE.
We wanted to characterize MEDH2 using different approaches. Initially we tried to show methanol dehydrogenase activity using an in vitro approach, which is based on the detection of generated NADH. We tried using E. coli raw extract which was not sufficient (see Notebook Week 13). Therefore we performed a Ni-NTA affinity chromatography to isolate and purify N-Terminal 6xHis-tagged MEDH2 (Figure 4).
Figure 4: Ni-NTA purifcation of MEDH2. Samples were analyzed by SDS-Page. Samples of the flowtrough (FT), of the washingsteps (W1-4) and of the eluates (E1-3) were analyzed. Theoretical molecular weight: MEDH2-40.15 kDa. Orange arrows indicate the protein.
Unfortunately using purified MEDH2 in our in vitro approach was not sufficient.
Further we tried to characterize MEDH2 in an in vivo approach. Unfortunately this approach was not sufficient (see Notebook Week 13).
In the second step generated formaldehyde will incorporated into 3-hexulose-6-phosphate by a 3-hexulose-6-phosphate synthase encoded by hps. We made hps compatible to the BioBrick system removing a XbaI and PstI restriction site by in-vitro mutagenesis and generated a basic BioBrick part (BBa_K1619002) .
Figure 5: Sequencing of hps after making it compatible for the BioBrick System. Seq_1 determines the biobrick compatible sequence of hps and Seq_2 determines the original hps
In the last step 3-hexulose-6-phosphate is converted into Fructose 6 phosphate, which will be incorporated in the E. coli carbon metabolism. This reaction is performed by a 6-phospho-3-hexuloseisomerase (phi). We genereated a basic BioBrick part (BBa_K1619003) , which contains the phi coding sequence. In addition we cloned phi into pET-30 and showed expression in E. coli (figure).
Figure 6: Pilotexpression of PHI. Coomassie staining of expressed PHI in E. coli BL21. Theorteical molecular weight: PHI-20.23 kDa. Proteins were separated by SDS-PAGE. Orange arrows determine the appropiate protein. As ladder PageRuler™ Plus Prestained Protein Ladder, 10 to 250 kDa (ThermoScientific) was used. As negative control (NC) selfligated pET-30 was used for expression in E. coli BL21. Proteins were separated by SDS-PAGE.
Summary:
We generated three basic BioBricks parts ( BBa_K1619001 , BBa_K1619002 , BBa_K1619003 )for the conversion of methanol into biomass. These BioBricks contain the original genomic sequences of medh2, hps and phi.
We expressed MEDH2 in E. coli and purified the enzyme. Enzyme assays for showing the functionality need further improvement.
We expressed PHI in E. coli.
3. Establishment of an airfilter system
Back to top
We were able to assemble our airfilter system as we planned it. We made videos of the bubbling culture inside the filter (Video 1), and of the complete assembled filter system running (Video 2).
We added a culture of the methanotroph Methylococcus capsulatus to the filter as a proof our concept. Methylococcus capsulats was grown for 5 days in NMS media supplemented with 25 % (v/v) methanol. Before adding it to the filter, the culture was harvested by centrifugation and resuspended in NMS-media without methanol. We expected a decrease in the methane concentration over time, since it was the only provided carbon source. Unfortunately we detected a leakage in our system by measuring the methane concentration over a period of 24 hours without adding a culture (Figure 7). So far we have not been able to fix the leakage.
Figure 7: Testing the assembled airfilter system: The concentration of methane was measured every second over a period of 24 hours. Test-run was performed without addition of a culture. Methane concentration is measured in ppm.
iGEM UiOslo 2015 is sponsored by:
