Team:UiOslo Norway/Experiments
HOME
PROJECT
Description
Results
Achievements
TEAM
Team
Attributions
Acknowledgement
NOTEBOOK
Notebook
Experiments & Protocols
PARTS
Team Parts
Basic Parts
HUMAN PRACTICES
Human Practices
Collaborations
SAFETY
Experiments
1. Obtaining the genes
The first project part pursues the goal to get or clone all genes that are involved in our project into a plasmid, which allows a rapid and easy amplification of those genes for further experiments. For the soluble methane monooxygenase (sMMO) from Methylococcus capsulatus (Bath) we got the genes for the subunits (mmoXYZBCD) cloned into the BioBrick standard vector pSC1B3 from the iGEM 2014 team from Braunschweig, Germany. Those six parts are registered as BioBrick parts under the names; Bba_K1390001 (mmoB), Bba_K1390002 (mmoC), Bba_K1390003 (mmoD), Bba_K1390004 (mmoX), Bba_K1390005 (mmoY), and Bba_K1390006 (mmoZ). Genes encoding the enzymes for the conversion of methanol into biomass (medh2, hps, and phi) were amplified by PCR from Bacillus methanolicus (MGA3) genomic DNA and TOPO blunt end cloning into the pCR4 vector was performed. Primers were designed in a way that they bind in the 5’ and 3’ untranslated region (UTR) of each gene. TOPO blunt end cloning of mmoG did not succeed. Instead mmoG was synthesized by IDT as a gBlock gene fragment, codon optimized for protein expression in E. coli.
2. Construction of BioBrick parts
The second project part had the intention to create four new basic BioBrick parts. Those basic parts consist of the coding sequences (CDS) of a gene. The codonoptimized mmoG as well as all three genes encoding the enzymes for the methanol to biomass conversion (medh2, hps, and phi) were created as BioBrick parts. The CDS of the hps gene, encoding the 3-hexulose-6-phosphate synthase, contains PstI and XbaI restriction sites making it not compatible with the BioBrick system. In two rounds of in vitro mutagenesis both restriction sites were removed and hps was cloned into pSB1C3 and submitted as a BioBrick part.
3. Generation of expression constructs
Before assembling the final constructs we wanted to show that each individual protein could be expressed in E. coli. The pET system has been chosen as our preferred system for overexpression of each individual protein. The vector backbones pET-28 and pET-30 were chosen as potential expression vector. Both vectors contain a T7 promoter. Protein expression under control of a T7 promoter can be induced by adding Isopropyl-β-D-thiogalactopyranosid (IPTG).
Figure 1: Schematic overview of the pET28a or pET30a vector with a T7 promoter, a ribosomal binding site (RBS) followed by our gene of interest.
With PCR we added restriction enzyme sites at the 5’ and 3’ end of CDS of each gene. Afterwards the gene was cloned either into pET-30 or pET-28 with the use of the listed restriction enzymes.
Table 1: Details about the expression constructs
Gene
5' Restriction Site
3' Restriction Site
Vector
Protein Tag
mmoX
NdeI
EcoRI
pET-30
--
mmoY
NdeI
EcoRI
pET-30
--
mmoZ
NdeI
EcoRI
pET-30
--
mmoB
NdeI
EcoRI
pET-30
--
mmoC
NdeI
EcoRI
pET-30
--
mmoD
NdeI
EcoRI
pET-30
--
mmoG
NdeI
EcoRI
pET-30
--
medh2
EcoRI
XhoI
pET-28
6xHis-tag N-terminal
hps
NcoI
EcoRI
pET-30
--
phi
NdeI
EcoRI
pET-30
--
4. Protein Expression and Solubility
After generating expression constructs for each gene individually we tested if they could be expressed successful in E. coli. Furthermore it is necessary to show if the heterologous expressed proteins are soluble expressed and not aggregated into inclusion bodies. Aggregation in inclusion bodies prevents a proper folding and thereby proper function of the protein. To test the solubility of heterologous expressed proteins, proteinexpression is performed for a defined period of time. Afterwards the cells are harvested and disrupted (e.g. sonication, lysozyme, glass beads). By centrifugation the lysate is separated into an insoluble fraction, which contains cell material and inclusion bodies, and a soluble fraction, containing the cytoplasmic fraction including most proteins. Both fractions can be analysed via SDS-Page.
5. Ni-NTA-affinity chromatography
Ni-NTA affinity chromatography is a standard method to purify His-tagged proteins. Proteins tagged with a polyhistidine tag interact and form chelate complexes with Nickel (Ni2+) ions, which are connected with a Nitrilotriacetic acid (NTA) residue. Addition of imidazol removes the polyhistidine tagged protein from the chelate complexe. Imidazole is an organic compound and replaces the tagged protein in the interaction with the Ni-NTA chelate complex.
6. Characterization of NAD+ dependent methanol dehydrogenase (medh2)
To characterize the functionality of the NAD+ dependent methanol dehydrogenase (medh2) two different approaches were chosen. We tried to characterize heterologous expressed MEDH2 in E. coli in an in vitro as well as in an in vivo assay.
In vitro assay
During the conversion of methanol into formaldehyde NADH is generated. The production of NADH can be measured at a wavelength of 340 nm.
Figure 1:Schematic overview of in vitro assay of NAD+ dependent methanol dehydrogenase. During the conversion of methanol into formaldehyde NAD+ is used as a cofactor, leading to the formation of NADH. The amount of formed NADH can be measured at a wavelength of 340 nm.
The formation of NADH is proportional to the amount of methanol converted into formaldehyde. Thereby the Beer–Lambert law can be used to calculate the concentration of NADH which corresponds to the concentration of produced formaldehyde. The enzyme assay was performed on cell raw extract which contains a lot of enzymes that use NAD+ as a cofactor. To get a hint about the background in our reaction we established two controls:
E. coli strain without expression of MEDH2
E. coli strain expressing MEDH2 but methanol was not added to the assay reaction
In vivo assay
In a second approach we wanted to determine the in vivo activity of heterologous expressed MEDH2 in E. coli. Since MEDH2 catalyses the reaction from methanol to formaldehyde, we want to detect formaldehyde. Formaldehyde can be detected using the Nash reagent. The Nash reagent consists of 2,4 Pentan dion, acetic acid and ammonium acetate. In the presence of formaldehyde it leads to the formation of 3,5-Diacetyl-1,4-Dihydro-2,6-Lutidine (DDL). By detecting the absorption at 412 nm, the formation of DDL can be measured. The concentration of formaldehyde is proportional to the concentration of DDL, which can be calculated using the Beer–Lambert law.
Figure 2:Schematic overview of in vivo assay of NAD+ dependent methanol dehydrogenase. Methanol is converted to formaldehyde. Formaldehyde will react with components in the Nash reagent to form 3,5-Diacetyl-1,4-Dihydro-2,6-Lutidine (DDL. The amount of formed 3,5-Diacetyl-1,4-Dihydro-2,6-Lutidine (DDL) can be measured at a wavelength of 412 nm.
Protocols
The protocols are ordered into different subjects. Please click on the subject to see the individual protocols. When clicking on a protocol, you will be directed to the protocol, from there you can navigate back to this page.
Molecular biology
PCR with Phusion-HF DNA Polymerase (NEB)
PCR with Taq DNA Polymerase (NEB)
Restriction Digest
Shrimp Alkaline Phosphatase Treatment (Fermentas)
Ligation with T4 DNA Polymerase (NEB)
TOPO Zero blunt cloning (Invitrogen)
Wizard® SV Gel and PCR Clean-Up System (Promega)
QIAprep Spin Miniprep Kit (Qiagen)
NucleoSpin® Tissue (Machery-Nagle)
In vitro Mutagenesis
Agarose Gel
List of primers
Protein Biochemistry
Protein Expression
Solubility Assay
SDS-Page
Ni-NTA Affinity Chromatography
Dialysis
In vitro Assay Methanol Dehydrogenase
In vivo Assay Methanol Dehydrogenase
Handling E. coli
E. coli strains
Transformation of chemically competent E. coli cells
Antibiotic Concentrations
LB media
Handling Methylococcus capsulatus (Bath)
Cultivation of Methylococcus capsulatus (Bath)
NMS medium
NMS-medium for growth of Methylococcus capsulatus (Bath)
iGEM UiOslo 2015 is sponsored by:
