<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UiOslo_Norway_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UiOslo Norway/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuOuterContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:UiOslo_Norway">HOME</A></LI><LI><A href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Achievements">Achievements</A></LI></UL><LI><A href="#">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Acknowledgement">Acknowledgement</A></LI><LI><A href="#">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="#">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Basic_Part">Basic Parts</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:UiOslo_Norway/Safety">SAFETY</A></LI></DIV></DIV><DIV id="contentContainer"><H1 style="text-align:center; font-size:400%;">Project overview</H1><P>
Methane is a potent greenhouse gas, and is leaked into the atmosphere at different natural and industrial places. A big part of the industrial methane emission is in the agricultural sector in places like

<A href="" class="tooltip"> 
	barns<SUP>1</SUP>,
	<SPAN>  Bacteria in the rumen of cows and other cattle produce methane. </SPAN></A> 
or

<A href="" class="tooltip"> 
	paddy fields<SUP>2,3</SUP><SPAN>  Bacteria in the soil that produce methane. </SPAN></A> 

(rice fields). Natural methane emission places are for example 

<A href="" class="tooltip"> 
	wetlands<SUP>4</SUP><SPAN>  land areas saturated with water in which methane producing bacteria reside. </SPAN></A> 
or

<A href="" class="tooltip"> 
	gas hydrates<SUP>5</SUP><SPAN>  Gas hydrates are trapped ice-like crystals of gas that are only stable in a specific temperature and pressure range. Found on continental shelves and under permafrost. </SPAN></A> 

To minimize the leakage of methane in these or other places one would want to breakdown methane locally. Or even better, one could convert methane to methanol or biomass so it can be more easily transported and used as a bio-fuel instead of being discarded. The current technology doesn't allow this kind of small scale local breakdown of methane, because this process requires high pressure and very high temperatures to break the strong bonds within one methane molecule.<SUP>6</SUP> An attractive alternative is bio-conversion of methane. 

<A href="" class="tooltip"> 
	Methanotrophs
	<SPAN>  single-cell organisms that metabolise methane. </SPAN></A> 

can naturally breakdown methane and use it as their sole carbon and energy source. Even better, the enzyme 

<A href="https://2015.igem.org/Team:UiOslo_Norway/Description#1"> 
	methane monooxygenase (MMO)
	</A> 

 that these methanotrophs use can breakdown methane at ambient temperatures and pressure.<SUP>6-9</SUP></P><P>
Thus if we could understand and use this enzyme it would be possible to develop tools to minimize the methane leakage at many places like barns and gas hydrates. However our knowledge about methanotrophs is limited and culturing them is relatively difficult and slow. That is why we want to implement MMO into <I>Escherichia coli (E. coli)</I> so that it can breakdown methane. Since methanol, the breakdown product of methane, is poisonous, we will also implement the Ribulose-Monophosphate (RuMP)- pathway from <I>Bacillus methanolicus</I> to convert methanol to biomass in three steps. To hold the bacteria we want to design an air-filter that could be used practically anywhere for this purpose. The first part of our project is based on the 

<A href="https://2014.igem.org/Team:Braunschweig"> 
	iGEM 2014 team from Braunschweig, Germany
	 </A></P><H2>
Project goal
</H2><P> Our goal is to design a filter containing <I>E. coli</I> that will break down methanol to biomass (fructose-6-phosphate) with the following reaction:
To reach our goal, or come as close as possible, we divided our project in three sub-goals. (Click on a goal to directly scroll down).</P><P><A href="https://2015.igem.org/Team:UiOslo_Norway/Description#1" class="tooltip"><B>Breakdown of methane to methanol in <I>E. coli</I>:</B><SPAN> Using the enzyme complex soluble methane monooxygenase (sMMO) from <I>Methylococcus capsulatus</I> (Bath).</SPAN></A></P><LI><P><A href="https://2015.igem.org/Team:UiOslo_Norway/Description#2" class="tooltip"><B>Conversion of methanol to biomass in <I>E. coli</I> (based on Müller et. al 2015)</B><SPAN> Using the following enzymes from <I>Bacillus methanolicus</I> MGA3;
*  Methanol to formaldehyde with the enzyme methanol dehydrogenase (medh2)
*  Formaldehyde to hexulose-6-phosphate with the enzyme 3-hexulose-6-phosphate synthase (hxlA)
*  Hexulose-6-phosphate to fructose-6-phosphate with the enzyme 6-phospho-3-hexuloisomerase (hxlB)</SPAN></A></P></LI><LI><P><A href="https://2015.igem.org/Team:UiOslo_Norway/Description#3" class="tooltip"><B>Air-filter containing the engineered <I>E. coli</I></B><SPAN> A closed container to restrict the genetically modified <I>E. coli</I>. The surrounding air will be pumped through the filter so the methane gas can be taken up by <I>E. coli</I> and be converted to biomass.</SPAN></A></P></LI><H4>
References
</H4><OL><LI><A href="http://www.epa.gov/climatechange/ghgemissions/usinventoryreport.html"> 
	US EPA, C. C. D. U.S. Greenhouse Gas Inventory Report: 1990 - 2013.
	 </A></LI><LI><A href="http://www.epa.gov/climatechange/ghgemissions/sources/agriculture.html"> 
	US EPA, C. C. D. Agriculture.
	 </A></LI><LI>Bodelier, P. L. E. Sustainability: Bypassing the methane cycle. Nature 523, 534–5 (2015).</LI><LI>Liikanen, A., Silvennoinen, H. &amp; Karvo, A. Methane and nitrous oxide fluxes in two coastal wetlands in the northeastern Gulf of Bothnia, Baltic Sea. Boreal Environ. Res. 14, 351–368 (2009).</LI><LI><A href="http://epa.gov/climatechange/ghgemissions/gases/ch4.html"> 
	US EPA, C. C. D. Methane Emissions.
	 </A></LI><LI>Rosenzweig, A. C. Biochemistry: Breaking methane. Nature 518, 309–10 (2015).</LI><LI>Sirajuddin, S. &amp; Rosenzweig, A. C. Enzymatic Oxidation of Methane. Biochemistry 54, 2283–94 (2015).</LI><LI>Sazinsky, M. H. &amp; Lippard, S. J. Methane monooxygenase: functionalizing methane at iron and copper. Met. Ions Life Sci. 15, 205–56 (2015).</LI><LI>Zhang, Y., Xin, J., Chen, L. &amp; Xia, C. The methane monooxygenase intrinsic activity of kinds of methanotrophs. Appl. Biochem. Biotechnol. 157, 431–41 (2009). </LI></OL><H1 style="text-align:center; font-size:400%;">Project Description</H1><H2>1. Methane to methanol</H2><A href="https://2015.igem.org/Team:UiOslo_Norway/Description#goals"> 
	Back to sub-goals
	 </A><P>Since the C-H bond in methane is very strong and requires expensive high tech equipment<SUP>1</SUP> we want to explore the possibilities of bioconversion of methane. Methanotrophs are single-cell organisms that can oxidize methane and use it as their sole carbon and energy source<SUP>2</SUP>. To date there are two enzyme complexes known that can do the task of breaking methane; soluble methane monooxygenase (sMMO), and the membrane bound particulate methane monooxygenase (pMMO)<SUP>1–3</SUP>. Both enzymes break methane with the following reaction: 
</P><P>Other than that they both can convert methane to methanol and require oxygen for the process, are they structurally very different. Most methanotrophs express pMMO, whereas sMMO is less often present. pMMO is expressed at high copper levels, which makes sense as it uses copper in the core of the enzyme to break the strong C-H bond in methane. At low copper levels however, sMMO is expressed which uses iron-ions in the enzyme core for breaking methane.<SUP>2–4</SUP> The methanotroph <I>Methylococcus capsulatus</I> (Bath) (<I>M. capsulatus</I> (Bath)) is one of the most studied methanotrophs that has both pMMO and sMMO. In our project we used the sMMO operon of (<I>M. capsulatus</I> (Bath)), more information about sMMO (insert link to scroll down).

Last years iGEM (2014)
<A href="https://2014.igem.org/Team:Braunschweig"> 
	team Braunschweig, Germany
	 </A> 

 cloned the sMMO genes of the methanotroph <I>M. capsulatus</I> (Bath) for the purpose of expressing them in <I>Escherichia coli</I> (<I>E. coli</I>). We chose to build on to their project and got their 
<A href="https://2015.igem.org/Team:UiOslo_Norway/Parts" class="tooltip"> 
	six cloned sMMO genes
	<SPAN>  Bba_K1390001 (<I>mmoB</I>)  Bba_K1390002 (<I>mmoC</I>) Bba_K1390003 (<I>mmoD</I>) Bba_K1390004 (<I>mmoX</I>) Bba_K1390005 (<I>mmoY</I>) Bba_K1390006 (<I>mmoZ</I>) </SPAN></A> 
, which were not available (yet) via the BioBrick system. In addition will we clone the <I>mmoG</I> gene of the sMMO operon which is thought to encode a chaperone (MMOG) involved in folding of the other MMO proteins<SUP>5,6</SUP>. MMOG might also be involved in regulating the sMMO operon by binding to a regulatory protein called MMOR<SUP>4,5</SUP>. The Braunschweig team used
<A href="https://2014.igem.org/Team:Braunschweig/Results-content#results1"> 
	 a plasmid
	 </A> 
 with the chaperones GroES, GroEL and TF to help fold the different MMO proteins.
</P><DIV class="highlightBox"><H3>Summary:</H3><UL><LI><P>Our team wants to engineer <I>E. coli</I> so that it can break down methane by cloning and expressing the sMMO of the methanotroph <I>M. capsulatus</I> (Bath).</P></LI><LI><P>We got the sMMO genes, <I>mmoX, mmoY, mmoZ, mmoB, mmoC</I>, and, <I>mmoD</I>, from 
<A href="https://2014.igem.org/Team:Braunschweig"> 
	iGEM 2014 team Braunschweig, Germany.
	 </A></P></LI><LI><P>We will clone the gene <I>mmoG</I> from genomic <I>M. capsulatus</I> (Bath) DNA ourselves.</P></LI></UL></DIV><H3>Soluble methane monooxygenase (sMMO)</H3><P>The sMMO operon of <I>M. capsulatus</I> (Bath), figure 1, has ten known protein encoding genes, summarized in <A href="https://2015.igem.org/Team:UiOslo_Norway/Description#Table1"> Table 1.</A> Two genes, <I>mmoQ</I>, and, <I>mmoS</I>, coding for the proteins MMOQ and MMOS can sense copper and play a role in regulating transcription of the sMMO operon<SUP>3–5</SUP>. A third gene, <I>mmoR</I>, encoding the protein 'MMOR', is a transcriptional activator of the whole sMMO operon<SUP>3–5</SUP>. Since we want to clone the genes that form the enzyme complex sMMO and use them to construct our own sMMO operon for expression in <I>E. coli</I>, the genes <I>mmoQ, mmoS,</I> and <I>mmoR</I> are not relevant and thus excluded from our project.
</P><P><B><FONT size="2">Figure 1:</FONT></B><FONT> sMMO operon of Methylococcus capsulatus (Bath)adapted from Scarlan et. al 2009</FONT></P><P>Six of the other seven genes, <I>mmoX, mmoY, mmoZ, mmoB, mmoC</I>, and, <I>mmoD</I>, encode the proteins of the sMMO complex. Three of these proteins come together and form one big protein, called MMO hydroxylase (MMOH). Hydroxylation means the adding of an -OH group, in this case the change of methane to methanol (CH<SUB>4</SUB> to CH<SUB>3</SUB>-OH) which happens inside MMOH. This reaction is assisted by MMOB, encoded by <I>mmoB</I>. MMOC, encoded by <I>mmoC</I>, is the sMMO reductase by providing MMOH with two electrons (by oxidizing NADH). Or to say is more simple, MMOC will reset MMOH so it can break another methane molecule. MMOD, encoded by <I>mmoD</I>, is thought to inhibit the process by blocking the binding of either MMOB or MMOC to MMOH.
</P><P>The last gene, <I>mmoG</I>, encodes for the protein MMOG, is not previously been uses in a iGEM project. MMOG is thought to be a chaperone which will properly fold the sMMO proteins.</P><P>More technical information about the specific functions of each sMMO subproteins after the summary.</P><DIV class="highlightBox"><H3>Summary:</H3><UL><LI><P>sMMO is build out of 5 proteins, whereof MMOX, MMOY, MMOZ form one protein complex called MMOH.</P></LI><LI><P>MMOH, MMOB and MMOC are needed to convert methane to methanol.</P></LI><LI><P>MMOD inhibits the methane to methanol reaction.</P></LI><LI><P>MMOG is a chaperone folding the other sMMO proteins correctly.</P></LI></UL></DIV><H4><FONT color="#1e0515">MMOH, MMOC and MMOB</FONT></H4><P>MMOH, the hydroxylase, consist of three subunits α (60.6 kDa), β (45.1 kDa), and γ (19.8kDa), encoded by <I>mmoX, mmoY</I>, and <I>mmoZ</I><SUP>2</SUP>. These bound subunits form again a dimer, resulting in α<SUB>2</SUB>β<SUB>2</SUB>γ<SUB>2</SUB>, with in the middle a 'canyon'. On each of the αβγ subunits is a binding site for either MMOB, the regulator, or MMOC, the reductase<SUP>2</SUP>. Binding of MMOB to MMOH is needed for efficient hydrocarbon oxidation (the actual breaking of the C-H to C-OH bonding)<SUP>2</SUP>. MMOC transfers the two electrons from NADH to the diiron site in MMOH<SUP>2</SUP>. It is in this diiron center of MMOH where the actual conversion of methane to methanol takes place. The two iron ions, the electrons (which change the iron ions), and the oxygen together with methane in the center of MMOH (bound to MMOB) make the magic happen to break one C-H bond in methane to a C-OH. Afterward helps MMOC to release methanol and resets MMOH by providing new electrons<SUP>1,3</SUP>.
</P><CENTER><P><B><FONT size="2">Figure 2:</FONT></B><FONT> Overview of the sMMO protein with subunits α,β, and γ of MMOH, and MMOB, MMOD, and MMOR.</FONT></P></CENTER><H4><FONT color="#1e0515">MMOD</FONT></H4><P>There is not much known about the function of the 12 kDa protein MMOD, formerly known as <I>orfY</I>. The first group showing that MMOD is expressed in <I>M. capsulatus</I> (Bath), also has evidence that MMOD can bind to the hydroxylase MMOH<SUP>7</SUP>. It seems that MMOD binds to MMOH in competition with the regulatory protein MMOB, which hints to an inhibitory function of MMOD<SUP>7</SUP>. They also showed a possible involvement of MMOD in the assembly of the metal center in MMOH<SUP>7–9</SUP>. But all their data is based on heterogeneously expressed MMO parts and purified from <I>E. coli</I> and used in <I>in vitro</I> studies. Further studies are needed to resolve the functionality of MMOD both <I>in vitro</I> and <I>in vivo</I>.
</P><H4><FONT color="#1e0515">MMOG</FONT></H4><P>The <I>mmoG</I> gene is suggested to encode a GroEL-like chaperone that contributes to the correct folding of the other sMMO proteins<SUP>5,6</SUP>, but this is so far not proven. They did show that MMOG (and 'MMOR') are required for protein binding to the promoter region of the sMMO operon in the methanotroph <I>Methylosinus trichosporium</I> OB3b5. In another iron monooxygenase, Bacterial binuclear iron monooxygenases, a similar protein as MMOG called mimG was needed for proper folding of one of the other monooxygenase proteins<SUP>6</SUP>. 
</P><CENTER><P><B>Table 1:</B> Summary of the sMMO genes and function.</P><TABLE id="t01"><TBODY><TR><TH>Gene</TH><TH>Protein (subunit)</TH><TH>Size (kDa)</TH><TH>Proteincomplex</TH><TH>Function</TH></TR><TR><TD><I>mmoX</I></TD><TD>MMO X/α</TD><TD>60.6</TD><TD rowspan="3">MMOH (part of sMMO)(α<SUB>2</SUB>β<SUB>2</SUB>γ<SUB>2</SUB>)</TD><TD rowspan="3">hydroxylase<SUP>2</SUP></TD></TR><TR><TD><I>mmoY</I></TD><TD>MMO Y/β</TD><TD>45.1</TD></TR><TR><TD><I>mmoZ</I></TD><TD>MMO Z/γ</TD><TD>19.8</TD></TR><TR><TD><I>mmoB</I></TD><TD>MMOB</TD><TD>15.9</TD><TD>sMMO</TD><TD>Regulatory<SUP>2</SUP></TD></TR><TR><TD><I>mmoC</I></TD><TD>MMOC</TD><TD>38.5</TD><TD>sMMO</TD><TD>Reductase<SUP>2</SUP></TD></TR><TR><TD><I>mmoD</I></TD><TD>MMOD</TD><TD>12</TD><TD>sMMO</TD><TD>Inhibitor?<SUP>7</SUP></TD></TR><TR><TD><I>mmoG</I></TD><TD>MMOG</TD><TD>59.5</TD><TD>--</TD><TD>GroEL-likechaperone<SUP>4</SUP></TD></TR><TR><TD><I>mmoQ</I></TD><TD>MMOQ</TD><TD>69.8</TD><TD rowspan="2">two-component signaltransduction system</TD><TD>regulator<SUP>4</SUP></TD></TR><TR><TD><I>mmoS</I></TD><TD>MMOS</TD><TD>128.6</TD><TD>sensor<SUP>4</SUP></TD></TR><TR><TD><I>mmoR</I></TD><TD>MMOR</TD><TD>63.4</TD><TD>--</TD><TD>Transcriptionalactivator<SUP>4</SUP></TD></TR></TBODY></TABLE></CENTER><H4>
References
</H4><OL><LI> Rosenzweig, A. C. Biochemistry: Breaking methane. Nature 518, 309–10 (2015).  </LI><LI> Sirajuddin, S. &amp; Rosenzweig, A. C. Enzymatic Oxidation of Methane. Biochemistry 54, 2283–94 (2015). </LI><LI>Sazinsky, M. H. &amp; Lippard, S. J. Methane monooxygenase: functionalizing methane at iron and copper. Met. Ions Life Sci. 15, 205–56 (2015).</LI><LI>Csáki, R., Bodrossy, L., Klem, J., Murrell, J. C. &amp; Kovács, K. L. Genes involved in the copper-dependent regulation of soluble methane monooxygenase of Methylococcus capsulatus (Bath): cloning, sequencing and mutational analysis. Microbiology 149, 1785–95 (2003).</LI><LI> Scanlan, J., Dumont, M. G. &amp; Murrell, J. C. Involvement of MmoR and MmoG in the transcriptional activation of soluble methane monooxygenase genes in Methylosinus trichosporium OB3b. FEMS Microbiol. Lett. 301, 181–7 (2009). </LI><LI>Furuya, T., Hayashi, M. &amp; Kino, K. Reconstitution of active mycobacterial binuclear iron monooxygenase complex in Escherichia coli. Appl. Environ. Microbiol. 79, 6033–9 (2013).</LI><LI>Merkx, M. &amp; Lippard, S. J. Why OrfY? Characterization of MMOD, a long overlooked component of the soluble methane monooxygenase from Methylococcus capsulatus (Bath). J. Biol. Chem. 277, 5858–65 (2002).</LI><LI>Rudd, D. J. et al. Determination by X-ray absorption spectroscopy of the Fe-Fe separation in the oxidized form of the hydroxylase of methane monooxygenase alone and in the presence of MMOD. Inorg. Chem. 43, 4579–89 (2004).</LI><LI>Sazinsky, M. H., Merkx, M., Cadieux, E., Tang, S. &amp; Lippard, S. J. Preparation and X-ray structures of metal-free, dicobalt and dimanganese forms of soluble methane monooxygenase hydroxylase from Methylococcus capsulatus (Bath). Biochemistry 43, 16263–76 (2004).  </LI></OL><H2>2. Methanol to biomass</H2><A href="https://2015.igem.org/Team:UiOslo_Norway/Description#goals"> 
	Back to sub-goals
	 </A><P>The goal of this final step is to break down methanol to biomass, so <I>E. coli</I> can utilize methanol as an energy source. Methanol is an one carbon structure that is toxic to the cell. There is however a group of microorganisms that utilizes methanol as an energy source. The group of microorganism that has the ability to grow with only a one carbon compounds as the single carbon source are called methylotrophes<SUP>1</SUP>. One such methylotrophe is <I>Bacillus methanolicus</I> (<I>B. methanolicus</I>), which was used in this project. In this step three enzymes have to be added in <I>E. coli</I> in order to go from methanol to fructose-6-phosphate; methanol dehydrogenase (Medh2), Hexolose-6-phosphate synthase (Hps), and 6-phospho-3-hexuloseisomerase (Phi). </P><H4><FONT color="#1e0515">Methanol to formaldehyde</FONT></H4><P>Methanol is oxidized to formaldehyde in all aerobic methylotrophes by the enzyme methanol dehydrogenase (Medh).  However, the oxidation can occur in two different pathways. Methanol metabolism in probacteria occurs by Periplasmic pyrroloquinoline quinone (PPQ) containing methanol dehydrogenase. The same metabolism in gram positive bacteria occurs by a NAD-linked cytoplasmic enzyme. The challenge with using the PPQ containing methanol dehydrogenase is that <I>E.coli</I> is not able to synthesize the cofactor needed fot this enzyme. So that the PPQ biosynthetic pathway has to be cloned into <I>E.coli</I> or PPQ has to be added to the growth medium. An additional challenge is that the Medh electron acceptor is cytochrome c, which is oxidized by a terminal cytochrome c oxidase that is not found in <I>E.coli</I><SUP>2</SUP>. The NAD-linked cytoplasmic enzyme was therefor chosen because only one additional gene is needed and it utilizes the electron acceptor already found in <I>E. coli</I>. One organism that has the NAD-linked cytoplasmic enzyme is <I>B. methanolicus</I>. Previous studies have shown that wild type (wt) strains of <I>B. methanolicus</I> have several genes encoding for NAD-dependent Medhs, which all can be functionally expressed in <I>E.coli</I><SUP>3</SUP>. Three specific NAD-dependent Medhs have been found in <I>B. methanolicus</I>: Medh, Medh2, and Medh34. All three Medhs have been tested under different conditions and it was found that Medh2 showed similar activity in <I>E. coli</I> lysate as <I>B. methanolicus</I> under similar conditions. Medh2 was therefore found to be the best candidate of the three to use in this project<SUP>2</SUP>. </P><H4><FONT color="#1e0515">Formaldehyde to fructose-6-phosphate</FONT></H4><P>Next step is to convert formaldehyde to biomass in order to obtain energy. Two pathways can perform this conversion; non-orthologs pathways and pathways that can be done with or without fixation of CO<SUB>2</SUB>.  Examples of non-orthologs pathway are ribulose monophosphate (RuMP) pathway, the tetrahydromethanopterin-linked pathway, and the gluthione-linked pathway. The second type of pathways are pathways such as RuMP cycle, serine cycle or Calvin cycle. Previous studies have shown that the most relevant pathway for this project are the RuMP pathway and the serine cycle. The RuMP pathway can be classified into three phases: fixation, cleavage, and rearrangement phase. Hexolose-6-phosphate synthase (Hps) and 6-phospho-3-hexuloseisomerase (Phi) are the two enzymes involved in the fixation phase. Hps and Phi are found in most methylotrophes, but different methylotrophes have different genes involved in the cleavage and rearrangement phase. Nerveless the different variance found within the other two phases of the RuMP cycle can all be expressed in <I>E.coli</I> too. Reason for this is that the variants are needed for processes are already present in <I>E.coli</I> such as glycolysis, Entner-Doudoroff pathway or pentose pathway. What this means is that only two additional genes, Hps and Phi, have to be added to <I>E.coli</I> in order for RuMP cycle to perform its function. The other pathway to consider is the serine pathway. Serine pathway differ form the RuMP pathway in many aspects. One aspect is that serine does not utilize formaldehyde directly, but it is added into the cycle in order to form serine. All enzymes needed for the serine cycle, except three are present in <I>E. coli</I>. However, there is no known activation enzyme or sufficient flux for the process of condensation of formaldehyde<SUP>2</SUP>. Compared to the serine cycle the RuMP cycle only needs two additional genes added to E.coli compared to the serine cycle that needs three. An additional factor is that the serine cycle needs multiple CO<SUB>2</SUB> reduction reactions, where RuMP pathway only needs one, making the RuMP cycle energetically more favorable. </P><DIV class="highlightBox"><H3>Summary:</H3><UL><LI><P>Our team wants to engineer <I>E. coli</I> in order for it to break down methanol to biomass by cloning and expressing the genes methanol dehydrogenase (Medh2), Hexolose-6-phosphate synthase (Hps), and 6-phospho-3-hexuloseisomerase (Phi)</P></LI><LI><P>Medh2 is NAD+ dependent and oxidizes methanol to formaldehyde </P></LI><LI><P> Hps and Phi are both part of the fixation phase of the ribulose monophosphate (RuMP) pathway and convert formaldehyde to hexolose-6-phosphate and fructose-6-phosphate respectively .</P></LI></UL></DIV><H4>
References
</H4><OL><LI> Yasueda, H., Kawahara, Y. &amp; Sugimoto, S. Bacillus subtilis yckG and yckF Encode Two Key Enzymes of the Ribulose Monophosphate Pathway Used by Methylotrophs, and yckH Is Required for Their Expression. J. Bacteriol. 181, 7154–7160 (1999).  </LI><LI> Müller, J. E. N. et al. Engineering Escherichia coli for methanol conversion. Metab. Eng. 28, 190–201 (2015). </LI><LI>Krog, A. et al. Methylotrophic Bacillus methanolicus encodes two chromosomal and one plasmid born NAD+ dependent methanol dehydrogenase paralogs with different catalytic and biochemical properties. PLoS One 8, e59188 (2013).</LI><LI>Ochsner, A. M., Müller, J. E. N., Mora, C. A. &amp; Vorholt, J. A. In vitro activation of NAD-dependent alcohol dehydrogenases by Nudix hydrolases is more widespread than assumed. FEBS Lett. 588, 2993–2999 (2014). </LI></OL><H2>3. Filter design</H2><A href="https://2015.igem.org/Team:UiOslo_Norway/Description#goals"> 
	Back to sub-goals
	 </A><P>The aim of this part of the project is to design and assemble a filter that makes it possible to capture methane from the atmosphere. The system includes 	a filter, a methane sensor, a vessel for gas, a pump, and a valve. Click on the different parts in Figure 3 to get directed to the specific part.</P><CENTER><B><FONT size="2">Figure 3:</FONT></B><FONT> Draft of the filter system.</FONT></CENTER><H4><FONT color="#1e0515">Filter</FONT></H4><P>The most important part of the system is the filter. The filter consists of a plastic column, which is shown in the figure below. Porous, plastic pads were introduced to the column to create turbulence and efficient gas-liquid exchange. Its on these plastic pads that the media containing the engineered <I>E. coli</I> bacteria will be applied. The filter was modified by drilling a hole on a side of the column and sealing the top. This was done in order to connect it to the gas output and to create a close system.
</P><P><B><FONT size="2">Figure 4:</FONT></B><FONT> Filter column and bubbles (water instead of a culture for simplicity)</FONT></P><H4><FONT color="#1e0515">Methane sensor</FONT></H4><P>In order to show that our system worked a methane sensor was added. The sensor measured the concentration of methane after the methane had passed through the filter. For the measurement the methane sensor MQ4 was used. Its function is based on SnO2 whose low conductivity rises when natural gases are present. For the MQ4 sensor to work it was connected to Arduino board, which was then assembled to the breadboard. These components were connected together according to the  

<A href="https://2014.igem.org/Team:Aachen/Collaborations/Methane_Sensor"> 
	iGEM 2014 team from Aachen, Germany
	 </A> 
and the

<A href="https://2014.igem.org/Team:Braunschweig/Notebook-content#protocols"> 
	iGEM 2014 team from Braunschweig, Germany.
	 </A></P><P><B><FONT size="2">Figure 5:</FONT></B><FONT> Assembled sensor, MQ4 sensor, Arduino board, Breadboard.</FONT></P><P>The data we downloaded from 

<A href="https://www.arduino.cc/en/Main/Software"> 
	Arduino 1.6.5 software
	 </A>
and we used 

<A href="http://freeware.the-meiers.org/"> 
	Roger Meier’s program CoolTerm
	 </A></P><H4><FONT color="#1e0515">Gas vessel</FONT></H4><P>The vessel is a 1 liter flask with a rubber stopper. This is where the methane will be introduced to the system. The methane is injected with a syringe through the rubber stopper.
</P><P><B><FONT size="2">Figure 6:</FONT></B><FONT> Reservoir and CO<SUB>2</SUB> capturing flask.</FONT></P><H4><FONT color="#1e0515">Pump</FONT></H4><P>Next part of the system is the pump. When deciding on what specific pump to use it was important to consider possible safety hazards. 
<A href="http://www.aliexpress.com/item/Exquisite-Ultra-Silent-High-Energy-Efficient-Aquarium-Fish-Tank-Oxygen-Air-Pump-Suzie/2036881614.html"> 
	A membrane pump
	 </A>
was chosen, because the area that is in contact with the gas does not consists of electrical parts. Electrical components can serve as a potential area for sparks and is therefore important to avoid. Another factor is that membrane pumps are cheap and are easily accessible. We modified pump's air input as shown on the picture below.
</P><P><B><FONT size="2">Figure 7:</FONT></B><FONT> Original and modified input.</FONT></P><H4><FONT color="#1e0515">Valve</FONT></H4><P>The last components are a valve that allows oxygen to enter the system and a flask containing calcium hydroxide. The valve is added because the bacteria requires oxygen to grow and to break down methane. Due to bacteria consumption of methane and oxygen the pressure is lowered in the system. In order to increase the difference in pressure, a carbon dioxide capture flask was introduced. What this does is lowering the pressure in the system compared to the atmospheric pressure. The air flows into the system, carrying the required oxygen for the reaction. 
</P><CENTER><P><B><FONT size="2">Figure 8:</FONT></B><FONT> Valve.</FONT></P></CENTER><CENTER><H1>iGEM UiOslo 2015 is sponsored by:</H1></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>