Team:UPF-CRG Barcelona/Results
Polybiome
Home
Project
People
Parts
Safety
Human Practices
Notebook
RESULTS
It has been a long journey, yet time has gone by faster than expected. Although we have not been able to fulfill the totality of our initial objectives, we have succeeded.  Since the very beginning, the main goal of this project has been the creation of a bacteria capable of degrading polyamines from its environment. With a proof-of-concept in E.coli, the circuit could be translated to other bacteria and potentially become a commercial probiotic for the prevention of colorectal cancer.
Doing the Knock-Outs of the genes speB (Agmatine decarboxilase) and speC (Ornithine Ureolase) has been an important obstacle for us. Our first approach was to use single mutants from the E.coli Keio Collection (Yale) and trying to remove the second gene with CrispR-Cas9. After repeating the whole process several times, the genomic sequencing of the DNA never confirmed a successful extraction. Moreover, we were never able to grow our double mutant cells in liquid medium.
Since we forsaw the previous situation coming, we purchased the MA255 cells from Yale before starting the lab sessions, which have already incorporated the two Knock-Outs that we want. The inconvenient with those, is that they also require 3 aminoacids in their growth media besides the polyamines to grow, which was a struggle at first but we ended up learning the method to cultivate them.
As stated, the main device of this project is the J23119-FMS1-PatA. This genetic machine is the combination of the regular consitutive promoter and two enzymes that degrade the 3 basic types of polyamines; the Polyamine Oxidase (spermine and spermidine) and the Putrescine Aminotrasnferase (putrescine). The combination of three is a powerful polyamine degrading engine, which we have successufully clonned and transformed in E. coli and that we are submitting to the part registry as the full composite, but also in separated parts. The two genes incorporate the standard medium RBS BBa_B0032 upstream to the actual codifying sequence. Sanger sequencing provided by our sponsor GATC biotech has confirmed the integrity of the samples.
To measure the effectiveness of the device, we have run a multiplate reader for 12 hours to compare the growth of Yale cells with and without our device with a gradient of polyamines. From the literature, we decided that the reliable range of polyamine concentration would be between 0mM to 2mM, since these molecules are harmful, even for bacteria, when in high amounts.
Several conclusions can be extracted from the pictures
First, the cells with the device experience a significant overall reduction in growth rate and total optical density. This is certainly produced by the expression of two constitutive genes, which is a metabolic burden for the cell that reduces its reproductive capacities.
We can verify that, indeed, the double mutant cells require polyamines for their reproduction, since the group without polyamines is not able to grow neither with or without the device. We can appreciate a certain growth during the first hours, probably because of the intracelular reservoirs of polyamines, that are exhausted soon.
We can verify that an excess in polyamines (2mM) ends up being lethal for the cells in both cases. Moreover, the degradation of polyamines by the device group produces peroxide, which is toxic for cells when above 1μM and accelerates the death of the cells further.
The device group with low to medium concentration of polyamines experiences a sudden delay in their growth between hours 4 and 5. This can be explained by the fact that they are polyamine auxotroph but still degrade their own vital compound. This leads to a critical point where there are not enough polyamines for the expanding population of cells and their growth is stopped.
The above effect is less significant in the cases with polyamine concentrations 0.5 and 1μM, since the reservoir of these molecules in the medium is large enough for the cells to live even if they are degrading it.
We foresaw the peroxide problem and anticipated possible solutions to it. We designed a system composed by a constitutive OxyR gene, that would bind once translated to reactive Oxygen Species (ROS), which is molecule that participates in a mechanism of bacteria to maintain several homeostasis levels. The 2013 team of NYMU-Taipei characterized this mechanism and included some parts in the registry. The second part of the system is an OxyR-induced promoter upstream of a peroxidase gene (KatG). With this device, when the concentration of peroxide increased in the cell, then peroxidases would be transcribed thus reducing the levels of this compound. Unfortunately, KatG had a very large nucleotide sequence that we had to break down and order in several separate pieces. We tried some DNA assembling methods but, since we were not succeeding, we focused on the main objective of our project.
The second major point was the development of a diagnostic tool for cancer using polyamine samples in urine. For this purpose, we wanted to build a device composed by a polyamine-activated promoter (atoC) and a gene codifying for vioC, one of the 5 sub-units that form the full violacein complex that is visible at daylight. Our plan was to constitutively express the remaining 4 parts, so that as soon as the 5th was transcribed, the full complex would form and “immediately” start producing coloration. Since we were also somewhat short of time, we planned on doing a proof-of-concept using GFP with atoC, which would, at the same time, allow us to characterize the activity of the promoter. Yet again, we didn’t manage to build the atoC-GFP as the sequencing revealed some deletions in the sequence which inhabilitated the device.
Regarding the mentioned characterization of the atoC promoter, we were willing to perform gas HPLC analysis, however we did not have enough time since this was an extra service which we did not have included in our lab facilities.
One of the main reasons why our time was so limited was that the sequencing bar codes did not arrive until the last minute, and we not now for sure if our parts were completely right. So, in the last few weeks, we had to repeat some of the clonings that went wrong, which is why we could not finish some of them.
Despite that we did not manage to do all what we wanted, we are proud of what we have done. Even if biotechnology is not part of the core subjects of our career, (since our team is composed entirely by biomedical engineers) we have been able to design our project and receive the approval of many experienced researchers. We have learned all the protocols and procedures in a matter of days, but we have just lacked some more experience in the lab, which would have marked the difference between almost getting to a result and actually achieving it.
Follow:
Twitter
Facebook
© 2016 iGEM Barcelona. Powered by Jekyll .
