<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPF-CRG_Barcelona_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPF-CRG Barcelona/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Notebook - Polybiome</TITLE></P><DIV class="masthead"><DIV class="masthead__inner-wrap"><DIV class="masthead__menu"><NAV id="site-nav" class="greedy-nav"><UL class="visible-links"><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona">Home</A></LI><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona/Project">Project</A></LI><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona/People">People</A></LI><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona/Parts">Parts</A></LI><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona/Safety">Safety</A></LI><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona/HP/Silver">Human Practices</A></LI><LI class="masthead__menu-item"><A href="https://2016.igem.org/Team:UPF-CRG_Barcelona/Notebook">Notebook</A></LI></UL></NAV></DIV></DIV></DIV><DIV id="main" role="main"><ARTICLE class="page" itemscope="" itemtype="http://schema.org/CreativeWork"><DIV class="page__inner-wrap"><HEADER><H1 class="page__title" itemprop="headline">Notebook
</H1></HEADER><SECTION class="page__content" itemprop="text"><H1 id="protocols-igem-barcelona"><STRONG>Protocols iGEM Barcelona</STRONG></H1><H6 id="august-2016">August 2016</H6><H2 id="cloning-with-biobricks"><STRONG>Cloning with biobricks</STRONG></H2><H5 id="digestion"><STRONG>Digestion</STRONG></H5><TABLE><THEAD><TR><TH>5’ Mix (1 digestion)</TH><TH>3’ Mix (1 digestion)</TH><TH>EP Mix (1 digestion)</TH></TR></THEAD><TBODY><TR><TD>1 μL CutSmart</TD><TD>1 μL CutSmart</TD><TD>1 μL CutSmart</TD></TR><TR><TD>7.6 μL H$_2$O</TD><TD>7.6 μL H$_2$O</TD><TD>7.6 μL H$_2$O</TD></TR><TR><TD>0.2 μL EcoRI</TD><TD>0.2 μL XbaI</TD><TD>0.2 μL EcoRI</TD></TR><TR><TD>0.2 μL SpeI</TD><TD>0.2 μL PstI</TD><TD>0.2 μL PstI</TD></TR></TBODY></TABLE><P>For multiple digestions, we use 0.1n<SUP id="fnref:1"><A href="#fn:1" class="footnote">1</A></SUP> μL of enzymes and 7.6n μL of H$_2$O (all mixes + 10% volume).</P><UL><LI>Add 1 μL of DNA to each digestion $\to$ 9μL Mix + 1 μL = 10 μL total (use PCR eppendorfs. Note that DNA must be in 100 ng/μL so that its final concentration is 10 μg/μL).</LI><LI>1h at 37°C + 20min at 80°C (digestion + inhibition) + $\infty$ at 10°C.</LI></UL><H5 id="ligation"><STRONG>Ligation</STRONG></H5><TABLE><THEAD><TR><TH>Mix</TH></TR></THEAD><TBODY><TR><TD>0.5 μL 5’ strand</TD></TR><TR><TD>0.5 μL 3’ strand</TD></TR><TR><TD>0.5 μL plasmid</TD></TR><TR><TD>0.5 μL buffer</TD></TR><TR><TD>0.2 μL T4 ligase</TD></TR><TR><TD>3 μL H$_2$O</TD></TR></TBODY></TABLE><UL><LI>Leave the ligation mix at room temperature for 1h</LI></UL><H5 id="transformation"><STRONG>Transformation</STRONG></H5><UL><LI>Take 45 μL of competent cells, add the 5 μL of mix.</LI><LI>Cool cells in ice for 20 minutes.</LI><LI>Perform a thermal shock by heating up to 45°C for 45s.</LI><LI>Place the cells in ice again for 10min and add 450 μL of 2YT.</LI><LI>Incubate at 37°C for 1h with shaking.</LI></UL><H5 id="seeding"><STRONG>Seeding</STRONG></H5><UL><LI>Centrifugate samples at 13000 rpm for 1 minute.</LI><LI>Extract approximately 350 μL of supernatant (can use air pump).</LI><LI>Resuspend pellet with the remaining supernatant.</LI><LI>Place the solution in the plate and spread with a sterilized handle, while rotating the plate.</LI><LI>Incubate at 37°C overnight.</LI></UL><H2 id="minipreps"><STRONG>Minipreps</STRONG></H2><H5 id="genelute-plasmid-miniprep-kit-sigma-aldrich-protocol">GenElute Plasmid Miniprep Kit (Sigma-Aldrich Protocol)</H5><P class="item1">
 	Harvest cells:
</P><P>Pellet 1-5 mL of an overnight recombinant <SPAN style="font-style:italic">E.coli
</SPAN> culture by centrifugation. The optimal volume of culture to use depends upon the plasmid and culture density. For best yields, follow the instructions in the note below. Transfer the appropriate volume of the recombinant <SPAN style="font-style:italic">E.coli
</SPAN> culture to a microcentrifuge tube and pellet cells at &amp;#8805 12,000 3g for 1 minute. Discard the supernatant.
</P><P><SPAN style="font-weight:bold"> Note: </SPAN> For best results with recombinant <SPAN style="font-style:italic">E.coli
</SPAN> grown in LB (Luria Broth), use 1-3 mL of culture for high copy plasmids or 1-5 mL of culture for low copy plasmids. With recombinant <SPAN style="font-style:italic">E.coli
</SPAN> grown in rich media such as TB (Terrific Broth)or 2X YT, use only 1 mL of culture. Higher volumes can cause a reducition in yield.
</P><P class="item1">
1) Resuspend cells:
</P><P>❗ <STRONG>Important reminder:</STRONG> Verify that appropriate volume RNAase A Solution was added to the Resuspension Solution.</P><P>Completely resuspend the bacterial pellet with 200 μL of the Resuspension Solution. 
Vortex or pipette up and down to thoroughly resuspend the cells until homogeneous. Incomplete resuspension will result in poor recovery.</P><P class="item1">
2) Lyse cells:
</P><P>Lyse the resuspended cells by adding 200μL of the Lysis Solution. Immediately mix the contents by gentle inversion (6–8 times) until the mixture becomes clear and viscous. 
<SPAN style="font-weight:bold">Do not vortex.</SPAN> Harsh mixing will shear genomic DNA, resulting in chromosomal DNA contamination in the final recovered plasmid DNA. 
<SPAN style="font-weight:bold">Do not allow the lysis reaction to exceed 5 minutes.</SPAN> Prolonged alkaline lysis may permanently denature supercoiled plasmid DNA that may render it unsuitable for most downstream applications.</P><P class="item1">
3) Neutralize:
</P><P>Precipitate the cell debris by adding 350 μL of the Neutralization/Binding Solution. Gently invert the tube 4–6 times. Pellet the cell debris by centrifuging at ≥ 12,000 xg or maximum speed for 10 minutes. Cell debris, proteins, lipids, SDS, and chromosomal DNA should fall out of solution as a cloudy, viscous precipitate. If the supernatant contains a large amount of floating particulates after centrifugation, recentrifuge the supernatant before proceeding to step 6.</P><P class="item1">
4) Prepare column:
</P><P>Insert a GenElute Miniprep Binding Column into a provided microcentrifuge tube, if not already assembled. 
Add 500 μL of the Column Preparation Solution to each miniprep column and centrifuge at ≥ 12,000 xg for 30 seconds to 1 minute. Discard the flow-through liquid.</P><P><SPAN style="font-weight:bold">Note:</SPAN>The Column Preparation Solution maximizes 
binding of DNA to the membrane resulting in more consistent yields.</P><P class="item1">
5) Load cleared lysate:
</P><P>Transfer the cleared lysate from step 3 to the column prepared in step 4 and centrifuge at ≥ 12,000 xg for 30 seconds to 1 minute. Discard the flow-through liquid.</P><P class="item1">
6) Wash column:
</P><P>❗ <STRONG>Important reminder:</STRONG> Verify that ethanol has been added to the bottle of Wash Solution 2.</P><P>Add 750 μL of the diluted Wash Solution to the column.
Centrifuge at ≥ 12,000 xg for 30 seconds to 1 minute. The column wash step removes residual salt and other 
contaminants introduced during the column load. Discard 
the flow-through liquid and centrifuge again at maximum 
speed for 1 to 2 minutes without any additional Wash 
Solution to remove excess ethanol.</P><P class="item1">
6) Elute DNA:
</P><P>Transfer the column to a fresh collection tube. Add 50 μL of Elution Solution or molecular biology reagent water to the column. For DNA sequencing and other enzymatic applications, use water or 5 mM Tris-HCl, pH 8.0, as an eluant. Centrifuge at ≥ 12,000 xg for 1 minute. The DNA is now present in the eluate and is ready for immediate use or storage at –20°C.
<SPAN style="font-weight:bold">Note:</SPAN> If a more concentrated plasmid DNA preparation is required, the elution volume may be reduced to a 
minimum of 50 μl. However, this may result in a reduction in the total plasmid DNA yield.</P><H5 id="electrophoresis-agarose-gel">Electrophoresis agarose gel</H5><UL><LI>Decide the concentration of agarose.</LI><LI>Decide the size of the gel.</LI><LI>Fill a tube with the decided amount of 1x TBE.</LI><LI>Weight in an erlenmayer the adequate amount of agarose (ex: for a 50 mL 0.8% we would use 0.4 g of agarose).</LI><LI><P>Add the 1x TBE volume in the erlenmayer together with the agarose.</P></LI><LI><P>Microwave the erlenmayer in decreasing periods of time until agarose crystals disappear. Shaking the erlenmayer between rounds.
❗ <STRONG>Note</STRONG>: Do not let the solution boil.</P></LI><LI>Cool down the flask by water contact.</LI><LI>Add 2.5 μL of SYBR-safe for 50 mL gel or 5 μL for a 100 mL gel.</LI><LI>Drop the solution into the mold and let it rest until the liquid polymerizes.</LI><LI>Transfer the mold into the bucket (negative pole close to the wells).</LI><LI>Mix 1 μL DNA + 9 μL H$_2$O + 2 μL loading dye to load in the desired wells.</LI><LI>In 1 or 2 wells load the appropriate DNA ladder.</LI><LI>Visualize the DNA with UV at increasing exposure times until a good image is obtained.</LI></UL><TABLE><THEAD><TR><TH>Agarose (%)</TH><TH>Visualized DNA range (bp)</TH></TR></THEAD><TBODY><TR><TD>0.5</TD><TD>1 k - 30 k</TD></TR><TR><TD>0.7</TD><TD>800 - 12 k</TD></TR><TR><TD>1</TD><TD>500 - 10 k</TD></TR><TR><TD>1.2</TD><TD>400 - 7 k</TD></TR><TR><TD>1.5</TD><TD>200 - 3 k</TD></TR><TR><TD>2</TD><TD>50 - 2 k</TD></TR></TBODY></TABLE><P style="text-decoration:underline;color:black;font-size:12px;font-weight:bold">
100 bp ladder preparation (for agarose gel):
</P><UL><LI>4 μL of H$_2$O</LI><LI>1 μL of loading dye</LI><LI>1 μL DNA ladder</LI></UL><H2 id="agar-plate-preparation-30-plates"><STRONG>Agar plate preparation (30 plates)</STRONG></H2><UL><LI>Get 500 mL of LB agar (from the 50$^\circ$C  oven).</LI><LI>Turn on the bunsen burner.</LI><LI>If needed, add antibiotic (0.5 mL of 1000x stock or 1 mL for 500x stock).</LI><LI>Pick an electronic pipette with 25 mL pipette.</LI><LI>With ON-bunsen, pipette 15 mL of the mix and put it into a plate.</LI><LI>Shake the plates once filled to even the surface and remove bubbles.</LI><LI>Let the plates dry.</LI><LI>Mark the edge of the plates according to the labeling convention.</LI></UL><H2 id="chemically-competent-cells-generation"><STRONG>Chemically competent cells generation</STRONG></H2><UL><LI>Use 15 mL Falcons and inoculate 10 mL LB + 100 μL of cells overnight culture + 5 μL antibiotic.</LI><LI>Incubate at 37°C during 2h 40min.</LI><LI>Cool in ice during 10 min.</LI><LI>Spin 3min at 4°C and 4,000rpm.</LI><LI>Discard the supernatant and resuspend the pellet with 1 mL of CaCl$_2$ (0.1M).</LI><LI>Incubate cells 20 minutes in ice.</LI><LI>Spin 3min at 4°C and 4,000rpm.</LI><LI>Discard the supernatant and resuspend the pellet of every Falcon with the following solution to make 50 μL Glycerol stock solution:
    </LI><LI>425 μL CaCl$_2$ (0.1M)</LI><LI>75 μL glycerol (50 %)</LI></UL><H2 id="pcas9-protocol"><STRONG>pCas9 protocol</STRONG></H2><P class="item1">
Digestion:
</P><UL><LI>Digest 1-2μg of pCas9 with BsaI. The digestion mix consists of:
    </LI><LI>26.075 μL pCas9 (76.7 ng/μL)</LI><LI>1 μL BsaI</LI><LI>5 μ CutSmart 10x</LI><LI>0.5 μL BSA 100x</LI><LI>fill up to 50 μL ddH$_2$O</LI></UL><LI>Gel purification of digested pCas9 (∼9kb).</LI><P class="item1">
Oligos design:
</P><P class="item1">
Phosphorilation:
</P><UL><LI>Phosphorilate the designed oligos with the polynucleotide kinase (PNK) mix, which consists of:
    </LI><LI>1 μL oligo 1</LI><LI>1 μL oligo 2</LI><LI>5 μL T4 ligase 10x</LI><LI>1 μL T4 PNK</LI><LI>42 μL ddH$_2$O</LI></UL><P class="item1">
Annealing:
</P><UL><LI>Add 2.5 μL of NaCl 1 M to phosphorilation mix.</LI><LI>Incubate 5 min at 95°C and cool to room temperature (look at oligo hybridization protocol).</LI><LI>Dilute 1:10.</LI></UL><P class="item1">
Ligation:
</P><UL><LI>0.5 μL annealed oligos</LI><LI>0.25 μL pCas9 purified</LI><LI>0.5 μL T4 ligase buffer 10x</LI><LI>0.25 μL T4 ligase</LI><LI>8.5 μL MilliQ</LI></UL><P>Incubate for 2h at room temperature.</P><H2 id="oligo-hybridization"><STRONG>Oligo hybridization</STRONG></H2><UL><LI>Mix preparation (30 μL)
    </LI><LI>Optimum$^2$ oligo concentration (in mix): 10 μM</LI><LI>3 μL of T4 ligase buffer (10x)</LI><LI>The remaining volume of water up to 30 μL</LI></UL><LI>Use PCR ramp down program (verify maximum and minimum melting temperatures of te oligos).
     Ex: 98°C - 40°C ramp.
$^2$: Add the best volume of each oligo (3 μL of general 100μM stock)</LI><H2 id="pcr"><STRONG>PCR</STRONG></H2><TABLE><THEAD><TR><TH>Mix</TH></TR></THEAD><TBODY><TR><TD>0.5 μL Phusion Taq polimerase</TD></TR><TR><TD>1 μL dNTPs</TD></TR><TR><TD>2.5 μL Forward primer (10 μM)</TD></TR><TR><TD>2.5 μL Reverse primer (10 μM)</TD></TR><TR><TD>10 μL HF buffer</TD></TR><TR><TD>1 μL template DNA</TD></TR><TR><TD>32.5 μL H$_2$O</TD></TR></TBODY></TABLE><H4 id="pcr-protocol"><STRONG>PCR Protocol</STRONG></H4><UL><LI>Initial denaturation: 30 s at 98°C</LI><LI>Repeat the cycle 35 times.
    </LI><LI>Denaturation: 10 s at 98°C</LI><LI>Annealing: 30 s at 52°C</LI><LI>Extension: 1.5 min at 72°C</LI></UL><LI>Final extension: 10 min at 72°C</LI><LI>Cooling: 4°C // Freezer</LI><H2 id="colony-pcr"><STRONG>Colony PCR</STRONG></H2><UL><LI><P>Gently vortex and briefly centrifuge Dream Taq Green PCR Mix (2x) after thawing.</P></LI><LI><P>Place a thin-walled PCR tube on ice and add the following components for each 10 μL reaction:</P></LI></UL><TABLE><THEAD><TR><TH>Mix</TH></TR></THEAD><TBODY><TR><TD>5 μL Dream Taq Green (2x)</TD></TR><TR><TD>0.5 μL Forward primer (10 μM, i.e. 1:10 dilution)</TD></TR><TR><TD>0.5 μL Reverse primer (10 μM, i.e. 1:10 dilution)</TD></TR><TR><TD>DNA template (1 colony)</TD></TR><TR><TD>4 μL H$_2$O</TD></TR></TBODY></TABLE><UL><LI><P>Gently vortex the samples and spin down.</P></LI><LI><P>Perform PCR using the recommended thermal cycling conditions outlined below:</P></LI></UL><H4 id="colony-pcr-protocol"><STRONG>Colony PCR Protocol</STRONG></H4><UL><LI>Initial denaturation: 1-3 min at 95°C</LI><LI>Repeat this cycle 25-40 times:
    </LI><LI>Denaturation: 30 s at 95°C</LI><LI>Annealing: 30 s at 52°C</LI><LI>Automated Fluorescent Extension: 1 min/Kb at 72°C</LI></UL><LI><P>Final extension: 5-15min at 72°C</P></LI><LI>Put 5-15 μL of PCR mixture directly on a gel.</LI><H2 id="polyamine-stocks-preparation"><STRONG>Polyamine stocks preparation</STRONG></H2><UL><LI>Polyamines:</LI></UL><TABLE><THEAD><TR><TH>Putrescine (Put)</TH><TH>Spermidine (Spd)</TH><TH>Spermine (Spm)</TH><TH> </TH></TR></THEAD><TBODY><TR><TD>Temp. fusion</TD><TD>27.5 ºC</TD><TD>23 ºC</TD><TD>29 ºC</TD></TR><TR><TD>Temp. Boiling</TD><TD>158.8 ºC</TD><TD>129 ºC</TD><TD>150 ºC</TD></TR><TR><TD>Solubility</TD><TD>H$_2$0 1 kg/L</TD><TD>H$_2$0 1 kg/L</TD><TD>H$_2$0 1 kg/L</TD></TR><TR><TD>Dangers </TD><TD>always &lt; 61 ºC</TD><TD>always &lt; 110 ºC</TD><TD>-</TD></TR></TBODY></TABLE><UL><LI>The polyamine concentration in the LB medium must be:
    </LI><LI>1.13 mM Putrescine</LI><LI>0.68 mM Spermidine</LI><LI>0.68 mM Spermine</LI></UL><LI><P>We will create 1000x stocks of the polyamines (2 mL):</P></LI><LI>Put stock: 0.20g Put</LI><LI>Spd stock: 0.20g Spd</LI><LI>Spm stock: 0.28g Spm</LI><P><SPAN style="font-weight:bold">Note:</SPAN> We tried by weighting the polyamines. It turns out that the solid polyamines at room temperature (Put&amp;Spm) become liquified when in contact with the spatula. The 3rd (Spm) is liquid. We are planning to heat up Put and Spm and to use their density values to measure the required volume:</P><TABLE><THEAD><TR><TH>Polyamines</TH><TH>Density values (at 25ºC)</TH><TH>Required volume (mL)</TH></TR></THEAD><TBODY><TR><TD>Putrescine</TD><TD>0.877 g/mL</TD><TD>0.23 mL Put</TD></TR><TR><TD>Spermidine</TD><TD>0.925 g/mL</TD><TD>0.22 mL Spd</TD></TR><TR><TD>Spermine</TD><TD>0.937 g/mL</TD><TD>0.30 mL Spm</TD></TR></TBODY></TABLE><UL><LI><P>Heat up putrescine, spermidine and the instruments to 40ºC to liquify them.</P></LI><LI><P>Rapidly pipet the calculated volumes and add MilliQ H$_2$0 up to 2 mL to obtain the 1000x stocks.</P></LI></UL><P>❗  All the measurements must be performed inside the fume hut for security purposes.</P><H2 id="mediums"><STRONG>Mediums</STRONG></H2><P class="item1">
 2YT rich growth media:
</P><UL><LI>1 L dH$_2$O</LI><LI>16 g Bacto Tryptone</LI><LI>10 g Bacto yeast extract</LI><LI>5 g NaCl</LI></UL><P class="item1">
 M9 minimal media (with aminoacids and salts) [special medium for MA225 growth]:
</P><P>Put into sterile water:</P><UL><LI>1x M9 salts</LI><LI>0.1 mM CaCl$_2$</LI><LI>1 mM MgSO$_4$</LI><LI>0.5% w/v Glucose</LI><LI>1.7 μM FeS0$_4$</LI><LI>1 μM Thiamine</LI><LI>50 μg/ml DL-Leucine</LI><LI>50 μg/ml Threonine</LI><LI>100 μg/ml Ampiciline</LI><LI>0.5 mM polyamines: just in case you want to grow auxotroph bacteria</LI></UL><P>If you already have M9 standard M9 medium, just add:</P><UL><LI>1.7 μM FeS0$_4$</LI><LI>1 μM Thiamine</LI><LI>50 μg/ml DL-Leucine</LI><LI>50 μg/ml Threonine</LI></UL><P>Reference: Panagiota S Filippou et al. Effect of polyamines and synthetic polyamine-analogues on the expression of antizyme (AtoC) and its regulatory genes. BMC Biochemistry, 8:1 doi: 10.1186/1471-2091-8-1, January 2007</P><P class="item1">
LB low-salt medium:
</P><P>Quantities per liter:</P><UL><LI>10g of Bactotryptone</LI><LI>5 g of yeast extract</LI><LI>0.5g of NaCl</LI></UL><P>Reference: Dimitri A. Kyriakidis, John S. Heller and E.S. Canellakis. Modulation of ornithine decarboxylase activity in Escherichia Coli by positive and negative effectors. Proc. Natl. Acad. Sci. USA. Vol 75, No. 10, pp 4699-4703, October 1978 (Biochemistry)</P><H1 id="lab-notes">Lab Notes</H1><H3 id="july">July</H3><H2 id="week-1">Week 1</H2><UL><LI><P>Polyamine 1000x stocks preparation (<A href="#polyamine-stocks-preparation">see protocol</A>).</P></LI><LI>PCR amplification of knock-in genes and promoters (<A href="#pcr-protocol">see protocol</A>).
    </LI><LI>PatA (gBlock fragment)</LI><LI>FMS1 (gBlock fragment)</LI><LI>J23119 (gBlock fragment)</LI><LI>AhpCp1 (gBlock fragment)</LI></UL><LI>Phusion PCR via overlap sequences of KatG upstream fragment and KatG downstream fragment (<A href="#">see protocol</A>).</LI><LI>Comprovative electrophoresis gel. Results showed the correct amplification of all gBlock fragments except for the Phusion PCR of KatG.</LI><H3 id="august">August</H3><H2 id="week-2">Week 2</H2><H3 id="section">5/08/2016</H3><UL><LI>Hidratation and MilliQ H$_2$O dilution 1:10 of Crispr primers for knockout of the following genes:
    </LI><LI>SpeF</LI><LI>SpeB</LI><LI>SpeC</LI></UL><LI><P>PCR purification of the correctly amplified gBlocks. The Nanodrop instrument revealed unsatisfactory results.</P></LI><LI><P>Glycerol stock preparation of (SpeF^-^/SpeB^-^) bacteria from the MA255 E.Coli strain from the Yale Coli Genetic Stock Center and pCas9 plasmid. Storage at -80°C. We renamed those cells DMPA. We put some of the cells in two O/N culture following the M9 medium formula of openwetware.org. One of the cultures contained polyamines while the other one not.</P></LI><LI>Miniprep of pCas9 plasmid with cloramphenicol resistance was performed.</LI><H3 id="section-1">7/08/2016</H3><UL><LI>DMPA cells in M9 medium didn’t grow in any case (with and without polyamines). We thought that it happened because DMPA has some other mutations appart from the SpeF and SpeB knock-outs and require a more special media to be grown. We checked the literature and found that they required an M9 media with some salts and aminoacids added (<A href="#mediums">see reference</A>). We proceeded in performing it next day.</LI></UL><H2 id="week-3">Week 3</H2><H3 id="section-2">8/08/2016</H3><UL><LI><P>No results were obtained from 5/08/2016 Cas9 minipreps. Cas9 miniprep was repeated with the Danagen kit. No results were obtained again. Carlos Toscano repeated the miniprep using his Sigma/Aldrich kit. He obtained good results. We decided on working with this kit.</P></LI><LI><P>LB low salt medium preparation (<A href="#mediums">see protocol</A>). This medium was prepared to grow the DMPA, as the paper showed that it also offered good conditions. O/N cultures were performed with 5 different growth media:</P></LI><LI>LB</LI><LI>LB + Str</LI><LI>LB + Str + Amp</LI><LI>LB + Str + PolyA</LI><LI>LB + Str + Amp + PolyA</LI></UL><P>No growth was observed.</P><H3 id="section-3">9/08/2016</H3><UL><LI><P>pCas9 protocol (<A href="#pcas9-protocol">see protocol</A>) applied to Speb and SpeC, to build the plasmids to carry out the DH5α knock-out cells with the crispr-cas9 method. 
For the digestion, 26.075 μl of the pCas9 miniprep were used.
The purification of pCas9 plasmid was not done from an electrophoresis gel, in contrast to what was mentioned in the protocol.
1:10 dilutions are performed taking 1 μl of the annealing result and 9 μl of MilliQ water.</P></LI><LI>PCR amplification of knock-in genes and promoters (<A href="#pcr">see protocol</A>):
    </LI><LI>OxyR (gBlock fragment)</LI><LI>PatA (gBlock fragment)</LI><LI>FMS1 (gBlock fragment)</LI><LI>J23119 (gBlock fragment)</LI><LI>AhpCp1 (gBlock fragment)
With respect to the usual PCR protocol, we diminished the PCR extension time from 1.5 minutes to 1 minute. This is because we just had small pieces of DNA, as KatG had already been amplified by our advisors.</LI></UL><LI>Preparation of aminoacid and salt stocks to prepare M9 medium.
    </LI><LI>L-leucine stock 500x</LI><LI>Threonine stock 500x</LI><LI>Thiamine stock ~100x (0.1 mM)</LI><LI>FeS0$_4$ stock ~50x (54.7 μM)
We didn’t manage to filter FeS0$_4$. Therefore, M9 medium was not completely prepared.</LI><H3 id="section-4">10/08/2016</H3><UL><LI><P>Digestions of AhpCp1 and J23119 promoters and FMS1, PatA and OxyR genes. An electrophoresis gel was performed afterwards in order to be sure that parts were digested. For this, we prepared the DNA ladder and the DNA samples as follows:</P><TABLE><THEAD><TR><TH>DNA ladder</TH></TR></THEAD><TBODY><TR><TD>1 μL DNA ladder</TD></TR><TR><TD>1 μL Purple Loading Dye</TD></TR><TR><TD>4 μL H$_2$0 (milliQ)</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>DNA samples</TH></TR></THEAD><TBODY><TR><TD>5 μL DNA</TD></TR><TR><TD>1 μL Purple Loading Dye</TD></TR></TBODY></TABLE><P>The order in which the samples were put into the wells is the next: AhpCp1 - FMS1 - PatA - OxyR - J23119</P></LI></UL><P>The expected sizes for the genes with which we worked are:</P><TABLE><THEAD><TR><TH>Part</TH><TH>size</TH></TR></THEAD><TBODY><TR><TD>J23119</TD><TD>218 pb</TD></TR><TR><TD>AhpCp1</TD><TD>350 pb</TD></TR><TR><TD>OxyR</TD><TD>1026 pb</TD></TR><TR><TD>KatG</TD><TD>2300 pb</TD></TR><TR><TD>FMS1</TD><TD>1635 pb</TD></TR><TR><TD>PatA</TD><TD>1488 pb</TD></TR><TR><TD>AtoC</TD><TD>203 pb</TD></TR></TBODY></TABLE><UL><LI>E. Coli SpeF- strain BW25113 from Keio Collection were made competent following the protocol (<A href="#chemically-competent-cells-generation">see protocol</A>). Strain 2 and 3 were taken. 6 glycerol stocks of each strain were kept at -80ºC.</LI></UL><H3 id="section-5">11/08/2016</H3><UL><LI><P>2YT media was prepared following the protocol (<A href="#mediums">see protocol</A>).</P></LI><LI><P>E-P digestions performed AhpCp1, OxyR and J23119 and receptor plasmids PaC I PaK. Before performing the digestions, DNA concentration of each sample was measured with the Nanodrop instrument:</P><TABLE><THEAD><TR><TH>DNA fragment</TH><TH>Concentration</TH></TR></THEAD><TBODY><TR><TD>PaC</TD><TD>100 ng/μl</TD></TR><TR><TD>PaK</TD><TD>32 ng/μl</TD></TR><TR><TD>AhpCp1</TD><TD>40.5 ng/μl</TD></TR><TR><TD>OxyR</TD><TD>90.4 ng/μl</TD></TR><TR><TD>J23119</TD><TD>49 ng/μl</TD></TR></TBODY></TABLE></LI><LI><P>In order to have a final volume of 10 μl with 100 ng/μl of DNA sample, as the digestion protocol requires, we modified the quantity of water and DNA in each sample, maintaining 0.1 μl of EcoR1, 0.1 μl of Pst1, 1 μl CutSmart. The concentrations are:</P><TABLE><THEAD><TR><TH>_</TH><TH>PaC</TH><TH>AhpCp1</TH><TH>OxyR</TH><TH>J23119</TH></TR></THEAD><TBODY><TR><TD>100 ng DNA</TD><TD>1 μl</TD><TD>2.5 μl</TD><TD>1.1 μl</TD><TD>2 μl</TD></TR><TR><TD>x μl MilliQ autoclavada</TD><TD>9 μl</TD><TD>7.5 μl</TD><TD>8,9 μl</TD><TD>8 μl</TD></TR></TBODY></TABLE></LI><LI><P>An Electrophoresis gel of the previous digestions was performed. The gel ran 25 minutes at 80V and 120 mA. We put the DNA samples on the following order: PaC – OxyR – J23119, AhpCp1.</P></LI><LI><P>J23119, OxyR and AhpCp1 ligation (keeping them 10 minutes at environmental temperature and 20 minutes at 80ºC) and transformation into DH5α competent cells.</P></LI><LI><P>Also, an electrophoresis gel with PatA was run.</P></LI></UL><H3 id="section-6">12/08/2016</H3><UL><LI><P>pCas9 ligation with SpeC and SpeB primers for knock-out obtention (<A href="#pcas9-protocol">see protocol</A>).</P></LI><LI><P>Repetition of FMS1 and PatA PCRs from the IDT DNA samples, as unsatisfactory results were obtained in the electrophoresis gels (<A href="#pcr">see protocol</A>). After PCR, one stock 1:100 of each gene were performed. The result of the PCR was put in an electrophoresis gel. Null Results.</P></LI></UL><H2 id="week-3-1">Week 3</H2><P>### 15/08/2016</P><UL><LI><P>We received DH5&amp;alpa; cells. They were made competent following the protocol (<A href="#chemically-competent-cells-generation">see protocol</A>). 50 glycerol stocks of 100μ of cells were kept at -80ºC.</P></LI><LI><P>PCRs of FMS1 and PatA genes were performed again, increasing the annealing temperature to 58ºC.</P></LI><LI><P>A purification gel was performed for KatG obtention. This task was carried out by Salva Duran.</P></LI></UL><H3 id="section-7">16/08/2016</H3><UL><LI><P>PaC, FMS1, PatA and AtoC E-P digestion (<A href="#cloning-with-biobricks">see protocol</A>). We used the PCR digestion program to perform this task.</P></LI><LI><P>Ligation of each part (FMS1, PatA and AtoC) with the receptor plasmid PaC, following the protocol (<A href="#cloning-with-biobricks">see protocol</A>).</P></LI><LI>Transformation of FMS1, PatA, AtoC and POC following the protocol into DH5α (<A href="#cloning-with-biobricks">see protocol</A>). Two stocks of DH5α cells were used. After transformation, all the cells were seeded in Petri dishes:
    </LI><LI>FMS1, PatA and AtoC were seeded on LB + cloramphenicol.</LI><LI>PUC was seeded on LB + ampiciline.</LI></UL><LI><P>Concurrenlty, seven more E-P digestions were performed for SpeBa, SpeBb, J23119, AhpCp1a, AhpCp1b, OxyRa, OxyRb. A checking electrophoresis gel was performed to assess that digestion were well-done. However, we ligated all the DNA fragments in PaC plasmids. SpeBa and SpeBb needed to be ligated with pCas9 plasmids. Therefore, these digestions needed to be repeated.</P></LI><LI><P>Minipreps of the cultures of 16/08 were performed. Electrophoresis gel showed not valid results after checking digestion. Definitely, we thought that the problem was the kit. We decided on working with Sigma/Aldrich kit.</P></LI><LI><P>We took 20H from plate 3 and 20J from plate 2 of the DNA distribution kit iGEM 2016. We transfoWe transfo</P></LI><LI><P>Minipreps of the cultures of 16/08 were performed. Electrophoresis gel showed not valid results after checking digestion. Definitely, we thought that the problem was the kit. We decided on working with Sigma/Aldrich kit.</P></LI><LI><P>BBa_K274003 and BBa_K274004 (corresponding to violacein) were taken from 20H and 20J wells from plate 3 of the DNA distribution kit iGEM 2010. We transformed them into DH5α and  0/N cultures with LB + karamicine were prepared. Moreover they were seeded on Petri dishes.</P></LI><LI><P>O/N cultures of J23119, OxyR, SpeB and AhpCp1 with LB and cloramphenicol were prepared so as to repeat the minipreps with the Sigma/Alsrich kit.</P></LI><LI><P>M9 medium preparation. We took 35 ml of standard M9 and we added 350 μl of thiamine, 70 μl de DL-leucine, 70 μl of threonine and 70 μl of FeS0$_4$ of each stock.</P></LI><LI>Two O/N cultures of DMPA were prepared:
    </LI><LI>7.5 ml of DMPA + M9 + 7.5 μl of ampiciline</LI><LI>8 ml of DMPA + M9 + 8 μl of ampiciline + 8 μl of putrescine and 8 μl of spermine.</LI><H2 id="week-4">Week 4</H2><P>### 17/08/2016
 + Repetition of the minipreps of pAC-J23, pAC-OxyR, pAC-AhpCp1, pCas-SpeB, pAC-20J. Two minipreps of each part were performed (a and b). Those minipreps were performed using Sigma/Aldrich kit (<A href="#minipreps">see protocol</A>).
 + Next, E-P checking digestions of all the minipreps were performed (except for the pCas9-SpeB)  (<A href="#cloning-with-biobricks">see protocol</A>). A checking electrophoresis gel was also performed. We filled the wells in the next order: DNA ladder, pAC AhpCp1 a, pAC-AhpCp1 b, pAC- J23119a, pAC-j23119b, pAC-OxyRa, DNA ladder, pAC-OxyRb, pAC-20Ja, pAC-20Jb, pCas9-SpeBa, pCas9-SpeBb.</P><UL><LI><P>Cloning of pAC-KatG into DH5α following the protocol(<A href="#cloning-with-biobricks">see protocol</A>).</P></LI><LI><P>Transformation of pCas9-SpeC into DH5&amp;alpha.</P></LI><LI><P>Transformation of pCas9-SpeB into BW25113 (SpeF-) to produce the SpeB knock-out.</P></LI><LI><P>pCas9 protocol (<A href="#cloning-with-biobricks">see protocol</A>) to prepare the pCas9 plasmid for SpeC knock-out generation.</P></LI><LI><P>0/N culture of PatA, FMS1, 20H and AtoC.</P></LI></UL><H3 id="section-8">18/08/2016</H3><UL><LI>PatA, FMS1 and AtoC minipreps of the overnight cultures were performed following the Sigma-Aldrich protocol (without optiional washing and 100 μl of elution buffer).</LI><LI>A checking digestion with an electrophoresis gel of PatA, FMS1, AtoC minipreps was performed. J23119, 20J, OxyR and Ahp checking digesitons were also repeated. The wells of the gel were load in the following order: Ahpb, AtoCa, AtoCb, FMS1a, FMS1b, J23119b, OxyRb, PatAa, PatAb and 20Jb. DNA ladder was put in positions 1 and 7.</LI><LI>After running the gel, we decided that FMS1b, SoeBa, SpeBb, 20Ja, J23119b, AhpCp1b and AtoCb were successful results.</LI></UL><UL><LI>Overnight cultures were performed:
    </LI><LI>K2M A + LB + Karamicine</LI><LI>K2M B + LB + Karamicine</LI><LI>OxyR A + LB + cloramphenicol</LI><LI>OxyR B + LB + cloramphenicol</LI><LI>PatA A + LB + cloramphenicol</LI><LI>PatA B + LB + cloramphenicol
 Note: K2M correspond to BW25113 Keio cells which has already been transformed with the pCas9-SpeB, to produce the SpeB knock-out</LI></UL><H3 id="section-9">19/08/2016</H3><UL><LI>Repetition of pCas9-SpeC transformation into DH5α and seeding. An O/N culture was also performed. No colonies were observed next day.</LI><LI>KatGa, KatGb, OxyRa, OxyRb, PatAa and PatAb minipreps from the overnight cultures were performed following the Sigma-Aldrich protocol.</LI><LI>A checking digestion + an electrophoresis gel were carried out for all the last minipreps. Results were unsatisfacotry as we obtained a gel in blank.</LI><LI><P>We checked DNA quantity in the samples of KatGb, KatGa, OxyRb, OxyRa, PatAb and PatAa with the nanodrop instrument. Very low quantities (0.3 - 0.45 ng/μl )were observed. We thought that must be the cause of the gel in blank. We decided on repeating all the minipreps in the following days. For this, we kept the pellets of the rest of the overnight cultures that we had used that day (to avoid repeating the overnight cultures).</P></LI><LI>6 falcon tubes were taken out from the incubator at 37ºC:
    </LI><LI>LB + Km + 20H</LI><LI>LB + Km + 20H</LI><LI>M9 + GLC + DMPA + PolyA + Amp</LI><LI>M9 + GLC + DMPA + Amp</LI><LI>LB + Km + 20H</LI><LI>LB + Km + 20H
 There was no growth in any of the falcon tubes.</LI></UL><LI>We put new overnight cultures at the incubator:
    </LI><LI>K2M A + LB + Kn (2n overnight, so that they lose the pCas9 plasmid of the SpeB knockout).</LI><LI>K2M B + LB + Kn</LI><LI>BW25113 + M9 incomplete + Kn</LI><LI>DH5α + M9 incomplete + Kn (as growth contol)</LI><LI>K2M A + M9 incomplete + Kn
 The next day, growth in the last 3 samples was observed.</LI><H3 id="section-10">20/08/2016</H3><UL><LI>Repetition of pCas9 protocol to ligate pCas9 with the SpeC oligos, as bad results had been obtained until now. Transformation of the pCas9.SpeC plasmimd into DH5α and seeding onto Petri dishes.</LI><LI>Transformation of pCas9-SpeC into DH5α + seeding into Cm.</LI><LI>Repetition of OxyRa,OxyRb, PatAa, PatAb, KatGa, KatGb minipreps.</LI><LI>Seeding of K2M from the 2 overnight culture onto LB+Cm.</LI></UL><H3 id="section-11">21/08/2016</H3><UL><LI><P>Cell growth was observed in the petri dish which contained K2M after two overnights onto LB + Cm. Colonies were taken from the Petri dish and were put into LB with and without cm, but with Kn in both cases (one conoly in each ependorf). Same was done for the two petri dishes.
 Falcon preparation: 3 ml LB + 1.5 μl Kn 1000x (+ 1.5 μl Cm 1000x)
 We used 1.5μl and not 3μl because we had a low-copy plasmid and it is enough with this concentraiton.
 Teorically, pCas9 is lost and we expect just cell growth into LB + Kn.</P></LI><LI><P>Growth was not observed in the DH5α-Crispr pCas9-SpeC petri dishes. We thought it was either because of oligo hybridations errors, ligation ones or transformation ones. 
 We observed two colonies in one Petri dish. We put this colonies into LB+Cm to make them growth.
 Falcon preparation: 3 mL LB + 15 μl Cm 1000x.</P></LI><LI><P>DH5α transformation with the previously used stocks of pCas9-SpeC in order to  reject cloning problems. We took the purified pCas9 plasmid and the Spec hybridated oligos and we ligated them.
 Seeding was performed onto 3 petri dishes.
 The rests of the yesterday’s ligation obtained with the pCas9 protocol was used to seed a fourth petri dish in the same conditions (LB + Cm).</P></LI></UL><P>Note: is it possible that we have too much Cm in the Petri dishes for a low-copy number plasmid as pCas9? Can it be the cause of having no growth?</P><P>All Petri dishes and falcon tubes were kept at 37ºC.</P><H3 id="section-12">22/08/2016</H3><UL><LI>KatG,PatA, OxyR digestion repetition, from minipreps of 20/08/2016. We put 5 μl of the E-P mix and 5 μl of each miniprep  together and we incubted them during 1 hour.</LI><LI>A checking digestion + electrophoresis gel was performed. The wells were laoded in the following order: PatAa, PatAb, OxyRa, OxyRb, KatGa, KatGb.</LI><LI>Regarding the cultures prepared on 21/08/2016:
    </LI><LI><P>There was observed growth of the overnight liquid cultures of K2M in all the cases. This indicates that cells didn’t loose the pCas9 plasmid. We decided on seeding the surviving K2M cells. We did a 1:500 dilution to reduce the number of cells. We incubated the cells at 37ºC.</P></LI><LI><P>DH5α transformed with pCas9-SpeC overnight culture growth was observed. However, as there is very little growth, we don’t trust much the results. We decided on repeating the oligo hybridation.</P></LI><LI><P>No growth was observed in any of the Petri dishes containing DH5α with pCas9-SpeC. We reject the possibility of errors during the cloning.</P></LI></UL><H3 id="section-13">23/08/2016</H3><UL><LI>8 falcon tube cultures were prepared to check the loose of pCas9 plasmid in K2M cells:
    </LI><LI>4 falcon tubes with LB (3 ml) + Kn (1.5 μl) + Cm (1.5 μl) + 1 colony</LI><LI>4 falcon tubes with LB + Kn + 1 colony</LI></UL><LI><P>2 clonings were performed to start building devices: J23 was digested with E-S and FMS1 with X-P and all was ligated into a PaK plasmid and seeded onto a LB + Kn Petri dish. On the other hand, AhpCp1 was digested with E-S and KatG was digested with X-P, ligated into PaK and seeded onto LB + Kn Petri dish.</P></LI><LI>We repeated PatA and OxyR clonings as we had not obtained good results for these parts until now. E-P digestions of PatA and OxyR were performed, taking the DNA directly from the Gblocks, and ligated into PaC plasmids. Those plasmids were transformed into DH5α cells and seeded on LB + Cm Petri dishes. For the transformation, we used LB instead of 2YT.</LI><LI>DH5α with pCas9-SpeC cultures were refreshed into LB + Cm  (3 ml + 1.5 μl).</LI><H3 id="section-14">24/08/2016</H3><UL><LI><P>Checking digestions and a checking electrophoresis gel of the previous day digestions were performed. The order in which the wells were loaded is the next: 
 pAK - pAC - J23119 - AhpCp1 - DNA ladder - PatA - FMS1 - OxyR - KatG</P></LI><LI><P>Minipreps and checking digestions were performed for the pCas9-SpeC cultures (0/N and 2nd 0/N)). Null results were observed on the electrophoresis gel.</P></LI><LI><P>DH5α transformation with the 20H and 20J parts onto LB + Cm Petri dishes.</P></LI><LI><P>GFP devices were taken from the 2016 iGEM distribution kit. We took one device which included a promoter, the BBa_I13552 from 6I of plate 3, and one device which didn’t, the Bba_E0240 from 24B well of plate 2. Both devices were transformed into DH5α.</P></LI><LI><P>Petri dishes preparation. LB + Kn and LB + Cm with standard antibiotic quantity and reduced one.</P></LI><LI><P>Liquid overnight cultures from the clonings of the previous day were prepared, as growth was observed in all of them.</P></LI></UL><H3 id="section-15">25/08/2016</H3><UL><LI><P>Minipreps of the overnight cultures were performed: PatA a and b, OxyR a and b, J23119-FMS1 a and b, AhpCp1-KatG a and b. Nanodrop instrument showed right quantities of DNA.</P></LI><LI><P>A checking digestion of the minipreps was performed and an electrophoresis gel was prepared. We also did this for the AhpCp1a, J23119b minipreps from 17/08/2016 and the PaK. Samples were loaded in alphabetical order:
 Ladder, AhpCp1-KatGa, AhpCp1-KatGb, AhpCp1a, FMS1-J23119a, FMS1-J23119b, Ladder 2, J23119, PaK, PatAa, PatAb, OxyRa, OxyRb</P></LI></UL><UL><LI><P>We added LB to the rests of the liquid cultures used for the minipreps to get refreshed overnight cultures.</P></LI><LI>Ligations:
    </LI><LI>pAK + J23119 + PatAb</LI><LI>pAK + AhpCp1 + OxyRa</LI></UL><LI>We took pCas9-SpeC, 20H, 20J colonies from the Petri dishes and prepared liquid overnight cultures.</LI><H3 id="section-16">26/08/2016</H3><UL><LI><P>Minipreps of the overnight cultures were performed. Afterwards, checking digestions and a checking electrophoresis gel was performed. The order of loading was the next:
 AhpCp1 - 6I1 - 6Ib, 24Ba - 24Bb - J23119 - PaC - Ladder - SpeB - SpeB - SpeC - SpeC</P></LI><LI>After running the gel, we decided on:
    </LI><LI>Making glycerol stocks to keep 6I1, 24Bb, together with OxyR and PatA minipreps analysed the day before.</LI><LI>Discarding both SpeB due to anything coulb be observed in the gel.</LI><LI>Repeating AhpCp1 and J23119 digestions.</LI><LI>Repeating the SpeC digestion, as wrong enzymes had been used.</LI></UL><LI><P>AtoC, PaK and SpeC digestions were repeated (with the right enzymes), and a checking electrophoresis gel was run. The order of loading is the next:
 Ladder + AtoC + PaK + SpeC</P></LI><LI><P>After the analysis of this gel we decided on discarding SpeC (due to the fact that we prepared a new liquid culture with LB + 1/2 Cm + PolyAs).</P></LI><LI><P>AhpCp1, J23119 and AtoC were ligated with PaC and transformed into competent DH5α cells and seeded on LB + Cm.</P></LI><LI><P>As until now we didn’t manage to make transformed DH5α loose the pCas9-SpeC plasmid, we decided on following a new strategy: prepare an overnight culture of the transformed DH5α cells into LB + Kn at 41ºC.</P></LI><LI>In order to check that DH5α hasn’t got resistance to Cm per se, we prepared two overnight cultures, one with LB and the other with LB +Cm.</LI><H3 id="section-17">27/08/2016</H3><UL><LI><P>Colony PCR of the promoters AtoC, J23119 and AhpCp1 were performed. 4 colonies were taken from each Petri dish and 2 of them were selected from the result of the gel.</P></LI><LI><P>XbaI digestion of the overnight culture of pCas9-SpeC growth in LB + 1/2 Cm + Polyamines was also loaded into the electrophoresis gel.</P></LI><LI><P>KatG and PatA minipreps gone missing. We prepared again overnight cultures taking colonies from the Petri dishes to repeat minipreps the next day.</P></LI><LI><P>K2M were seeded on Petri dishes with Kn after an overnight culture at 41ºC in order to see if they had finally lost the pCas9 plasmid.</P></LI></UL><H3 id="section-18">28/08/2016</H3><UL><LI>Miniprep of the liquid overnight cultures of FMS1, AtoC, Ahp, J23119 and KatG. Further digestion with EcoRI and PstI with 1 μl of DNA (since the concentrations were unknown). The wells of the gel were loaded as follows:
    </LI><LI>Ahp_1 Ahp_2 AtoC_1 AtoC_2 Fms1_b Ladder Fms1_a J23_3 J23_4 KatG_a KatG_b</LI></UL><UL><LI>Weird bands at the 6 wells of the 3 promoters(Ahp, Atoc and J23), drives us to digest again with EcoRI and PstI and run a second gel. The Wells were loaded as follows:
    </LI><LI>Ladder Ahp_1 Ahp_2 AtoC_1 AtoC_2 J23_3 J23_4</LI></UL><UL><LI><P>After contrastation of the results, we make glycerol stocks of the liquid cultures of KatG_b, FMS1_b, Ahp_1, J23_4 and Atoc_1 (the promoters still present some irregularities regarding their size)</P></LI><LI><P>Different colonies of K2M cells were extracted from the Kn dishes labeled #1 to #4. They were distributed in different liquid mediums (using the same colony for each pair, to see if the Cm resistance was lost - and thus the pCas9 plasmid):</P></LI><LI>LB + Kn</LI><LI>LB + 1/2Cm + Kn</LI></UL><H3 id="section-19">29/08/2016</H3><UL><LI>The following DNA digestions were performed:
    </LI><LI>J23_4, AtoC_1, Ahp_1 and Ahp_2 were E-S digested</LI><LI>FMS1_b, KatG, GFP(24B) and OxyR were X-P digested</LI><LI>Lethal pAK plasmid was E-P digested</LI></UL><LI>The ligations of the fragments was carried out as follows:
    </LI><LI>J23_4 - FMS1_b - pAK</LI><LI>Ahp_1 - KatG - pAK</LI><LI>AtoC_1 - 24B - pAK</LI><LI>Ahp_2 - OxyR - pAK</LI><LI><P>The ligations were transformed in competent DH5&amp;alpha coli strain</P></LI><LI>The digestions were in parallel run through an electrophoresys gel. The order of the wells was as follows:
    </LI><LI>Ahp_1 Ahp_2 AtoC_1 FMS1_b J23_4 Ladder KatG_b OxyR_a pAK 24B</LI><LI><P>Apparently eppendorfs of K2M comming from plates #2 and #3 did not show growth in Cm rich LB. The content of the eppendorfs with Kn was transfused into fresh LB + Kn and glycerol stocks were made of these cells</P></LI><LI><P>We extract pCas-SpeC colonies (the colonies showed two different phenotypes, someones were larger than others, we selected a few of each type). We grow them at 37ºC overnight in liquid LB + 1/2 Cm</P></LI><LI><P>We prepare new stocks of LB and LB-agar to be autocleaved tomorrow</P></LI><LI>New colonies of OxyR and KatG were extracted from the previous plates and grown in LB - Cm</LI><H3 id="section-20">30/08/16</H3><UL><LI><P>Miniprep of KatG and OxyR liquid cultures. The minipreps are digested with (separately) E, S, X, P and XP to verify that they have all the restriction sites. We verify that neither of the two minipreps are correct and discard the samples, as well as all the previously gathered minipreps, glycerol stocks and plates containing either of the two. The cloning of both genes is repeated from the original gBlocks and they are plated in Cm plates.</P></LI><LI><P>Some colonies from AtoC-GFP and J23-FMS1 are extracted and grown in LB - Kn</P></LI></UL><UL><LI>New stocks of agar plates were made:
    </LI><LI>LB + 1/2Kn + 1/2Cm + PolyA’s</LI><LI>Lb + 1/2Cm + PolyA’s</LI><LI>Lb + 1/2Cm</LI></UL><H3 id="section-21">31/08/2016</H3><UL><LI><P>KatG and OxyR colonies extracted from the plates made yesterday and grown in Lb + Cm</P></LI><LI><P>Primers for sequencing the genomic knock-outs of SpeC, SpeB and SpeF were designed</P></LI></UL><H3 id="section-22">02/09/2016</H3><UL><LI>The following digestions were made:
    </LI><LI>J23-FMS1 with E-P</LI><LI>AtoC-GFP with E-P</LI><LI>J23 with E-S</LI><LI>KatG with X-P</LI><LI>OxyR with X-P</LI><LI>PatA with X-P</LI><LI>pAK with E-P</LI></UL><LI><P>An electrophoresys test was made with the J23-FMS1 and AtoC-GFP digestions, but nothing was visible in the gel. To be repeated in the future.</P></LI><LI>The following ligations were performed:
    </LI><LI>PatA-pAK (for a resistance swap, from Cm)</LI><LI>J23-PatA-pAK</LI><LI>Ahp-OxyR-pAK</LI><LI>Ahp-KatG-pAK</LI><LI><P>We create a new stock of complete M9 with theonine, L-leucine, Iron and Thiamine</P></LI><LI><P>We extract double mutant cells (speB and speC) from the plate and grow them in 5mL of M9 with Polyamines @ 37ºC overnight</P></LI><LI>We extract 2 colonies from each pCas-speC plate and grow them in Lb + 1/2Cm + polyamines @ 37ºC overnight</LI><H3 id="section-23">05/09/2016</H3><UL><LI><P>DMPA has grown without ampicillin. We refresh the culture with M9 + Ampicillin to test their resistance.</P></LI><LI><P>The ligations from day 02/09 are transformed in DH5&amp;alpha.</P></LI><LI><P>Medium refresh of the pCas-speC grown cells in Lb + 1/2Cm + polyamines. Incubated @ 37ºC overnight</P></LI><LI><P>miniprep of pCas-speC and electrophoresys gel with an XbaI digestion to know if the plasmid has its size correct. Apparently it isn’t</P></LI></UL><H3 id="section-24">06/09/2016</H3><UL><LI><P>Colony PCR of yesterday’s transformations + electrophoresys gels. Apparently all the plasmids are empty, since no band appears at its corresponding position</P></LI><LI><P>Transformation of Keio-SpeC with pCas-speB (from long ago - 17/08/16). We use 2Yeast + polyamines after the thermal shock. Cells incubated at 37ºC overnight in LB + polyamines + 1/2Cm</P></LI><LI><P>E-P digestion of Oxy-R and KatG (01/09/16) and further test in electrophoresys gel. OxyR is kept, and KatG is discarded.</P></LI></UL><H3 id="section-25">07/09/2016</H3><UL><LI><P>Preparation of more M9 medium.</P></LI><LI>Massive colony PCR of several stored plates to verify its content and do some clean-up. We keep:
    </LI><LI>Ahp-OxyR</LI><LI>PatA (two of them)</LI><LI>J23-PatA</LI></UL><LI>We grow these 3 colonies in LB with their corresponding resistances overnight at 37ºC</LI><H3 id="section-26">08/09/2016</H3><UL><LI><P>From the 3 liquid cultures, we make glycerol stocks and store them at -80ºC.</P></LI><LI><P>We phosphorilate and anneal speB oligos, and ligate them in pCas-speB</P></LI></UL><H3 id="section-27">09/09/2016</H3><UL><LI>We perform electrophoresys tests of yesterday’s liquid cultures.
    </LI><LI>Ahp-OxyR does not return a band</LI><LI>PatA (both) and J23-PatA look correct</LI></UL><LI><P>We make more competent Keio-SpeC (10 stocks in total) and store them in glycerol @ -80ºC</P></LI><LI><P>We transform and seed DH5&amp;alpha with pCas-speB in LB + 1/2Cm</P></LI><LI>We extract Keio SpeC- speB- from plates and grow them in:
    </LI><LI>M9 + Cm + Polyamines</LI><LI>M9 + Cm</LI><H3 id="section-28">13/09/2016</H3><UL><LI><P>Cloning of J23-FMS1-PatA in DH5&amp;alpha, and further seeding in LB + Cm</P></LI><LI><P>We refresh the cultures of Keio SpeC- speB- in:</P></LI><LI>M9 + Kn @ 37ºC</LI><LI>M9 + Kn @ 41ºC (for pCas plasmid loss treatment)</LI></UL><H3 id="section-29">14/09/2016</H3><UL><LI><P>We use old miniprep samples to start over another cloning of Ahp (29/08) + OxyR (1/09) in pAK</P></LI><LI><P>We digest the Violacein plasmids obtained with E-P and run them through an agarose gel. No bands found (maybe we used too little DNA)</P></LI><LI><P>We do Colony PCR of yesterday’s J23-FMS1-PatA plates. We keep a couple of colonies, although the exact size does not correspond with the desired one.</P></LI><LI><P>Transformation of pCas9-speB in DH5&amp;alpha to repeat the knock-out, since double mutant Keio cells have grown without polyamines, and they were not supposed to.</P></LI></UL><H3 id="section-30">16/09/2016</H3><UL><LI>Digestion of J23-FMS1 with ES, patA with XP and pAC with EP. Ligation of the 3 parts and transformation in DH5&amp;alpha.</LI></UL><H3 id="section-31">17/09/2016</H3><UL><LI>Preparation of a 96-plate DH5&amp;alpha and Double mutant Keio (PACO) with a gradient of polyamines (max concentration, 10mM, dilutions 1/2 along the rows)</LI></UL><H3 id="section-32">18/09/2016</H3><UL><LI>Creation of glycerol stocks of PACO cells.</LI></UL><H3 id="section-33">19/09/2016</H3><UL><LI><P>Colony PCR of J23-FMS1-PatA</P></LI><LI><P>Miniprep of Ahp-OxyR and pCas9-speB once again</P></LI><LI><P>Digestion and gel of Ahp-OxyR (no bands appreciated)</P></LI><LI><P>Refreshing of Dh5&amp;alpha and PACO in new M9 + polyamines</P></LI></UL><H3 id="section-34">20/09/2016</H3><UL><LI>We make the following digestions:
    </LI><LI>J23-FMS1 with E-S</LI><LI>PatA with X-P</LI><LI>Ahp-OxyR with E-P</LI><LI>pAC with E-P</LI></UL><LI><P>Ligation of J23-FMS1-PatA in pAC.</P></LI><LI><P>Preparation of more PACO competent cells, and glycerol stocks.</P></LI><LI>Electrophoresys gel of Ahp-OxyR, J23-FMS1-PatA (apparently correct)</LI><H3 id="section-35">21/09/2016</H3><UL><LI><P>Transformation of AtoC-GFP (pAK) in yesterday’s PACO cells. We grow them @ 37ºC</P></LI><LI><P>Refreshment of DH5&amp;alpha in new M9 + NALDA</P></LI><LI><P>Refreshment of PACO in new M9 + Kn + Polyamines</P></LI><LI><P>Refreshment of PACO in M9 + Kn</P></LI></UL><H3 id="section-36">23/09/16</H3><UL><LI>Since this day we did not have a multiplate reader available, we did a DIY version with a cuvette spectrophotometer. We studied the growth of basid DH5&amp;alpha cells with respecto to our PACO cells at different polyamine concentrations (2, 0.2, 0.02 and 0 mM). Besides the abosrbance at 660 nm, we also measured 190 nm since we identified that this wavelength could correspond to polyamine absorbance, and we wanted to track the level of polyamines in the sample. We prepared two stocks of cells with M9 so as to start at 0.1 ODE with respect to plain M9:
    </LI><LI>Stock DH5&amp;alpha: 13 ml M9 + 1.5 ml cells</LI><LI>Stock PACO: 12.5 ml M9 + 2 ml cells</LI></UL><LI><P>We concluded that something besides polyamines absorbs at 190 nm, since we obtained increasing responses as time passed, which was not possible since polyamines should be degrading. Also we did not obtain coherent growth results, since PACO without polyamines were growing faster than PACO with polyamines, and these cells should be polyamine-auxotrophs.</P></LI><LI>Cloning of new J23-FMS1-PatA, J23-FMS1-J23-PatA and AtoC-GFP. Plated in Cm plates and grown over the weekend at room temperature to slow down the growth rate.</LI><H3 id="section-37">26/09/2016</H3><UL><LI><P>Colony PCR of J23-FMS1-PatA, J23-FMS-J23-PatA and AtoC-GFP</P></LI><LI>Electrophoresys gel of the previous PCR’s. Well order:
    </LI><LI>J1 J2 J3 J4 J5 JJ1 JJ2 JJ3 JJ4 JJ5 G1 G2</LI></UL><LI><P>We confirmed the sizes of both GFP’s and J5, discarded the rest of the samples and refreshed the 3 right ones.</P></LI><LI>We let the 3 cultures grow for 10h and did a Miniprep afterwards.</LI><H3 id="section-38">28/09/2016</H3><UL><LI><P>Digestions and electrophoresys gel of the minipreps of 2 days ago. Due to our mistake, the gel was left running for too long and the samples were lost.</P></LI><LI><P>We prepared more M9 medium</P></LI></UL><H3 id="section-39">30/09/2016</H3><UL><LI>We did a 96-well study of Paco, DH5&amp;alpha with AtoC-GFP and DH5&amp;alpha with constitutive GFP cells. We used a decaying gradient of polyamines, starting at 2mM.</LI></UL><H3 id="first-half-of-october">First half of October</H3><UL><LI>We have sequenced all our parts through our sponsor GATC biotech. We have confirmed the sequences of J23, FMS1 and PatA.
    </LI><LI>AtoC, J23, FMS1, PatA, AtoC-GFP, J23-FMS1-PatA</LI></UL><LI><P>We have tried to cultivate plasmids for different polyamine degrading enzymes, but apparently the samples (which were kindly donated to us by XXX) were too degraded to grow in bacteria, since they are not genetically optimized for this purpose.</P></LI><LI><P>We have tried to repeat several times the clonnings of AtoC-GFP and J23-FMS1-PatA without success.</P></LI><LI>We have tested our double mutant cells in different mediums to evaluate their survability.
    </LI><LI>M9, M9+PolyA’s, M9+PolyA’s+Aminoacids, LB</LI><LI><P>We have made glycerol stocks of our successful clonnings</P></LI><LI>We made a successful cloning of J23-FMS1-PatA in the MA255 e.coli strain.</LI><DIV class="footnotes"><OL><LI id="fn:1"><P>“n” is the number of digestions <A href="#fnref:1" class="reversefootnote">↩</A></P></LI></OL></DIV></SECTION></DIV></ARTICLE></DIV><DIV class="page__footer"><FOOTER><DIV class="page__footer"><FOOTER><DIV class="page__footer-follow"><UL class="social-icons"><LI><STRONG>Follow:</STRONG></LI><LI><A href="https://twitter.com/igembarcelona"> Twitter</A></LI><LI><A href="https://facebook.com/igemBarcelona"> Facebook</A></LI></UL></DIV><DIV class="page__footer-copyright">© 2016 iGEM Barcelona. Powered by <A href="http://jekyllrb.com" rel="nofollow">Jekyll</A>. </DIV></FOOTER></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>