<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><BUTTON id="top-scroll" class="control-button">⇧</BUTTON></DIV><DIV id="header"><DIV id="header_team"><P>
        Team Göttingen iGEM 2018
      </P></DIV><DIV id="header_middle"><DIV id="header_middle-container"><P>Glyphosate <FONT size="4" color="white">on my</FONT> plate?</P></DIV></DIV></DIV><DIV id="main_wrapper"><DIV id="content_menu_wrapper"><DIV id="navigation" class="sidebar is-nav-hidden"><A href="https://2018.igem.org/Team:Goettingen" class="sidebar-items">Home</A><A href="https://2018.igem.org/Team:Goettingen/News" class="sidebar-items">News</A><BUTTON class="dropdown_button sidebar-items">Team</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Team" class="sidebar-items sidebar-drop_items">Team Members</A><A href="https://2018.igem.org/Team:Goettingen/Collaborations" class="sidebar-items sidebar-drop_items">Collaborations</A></DIV><BUTTON class="dropdown_button sidebar-items">Project</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Background" class="sidebar-items sidebar-drop_items">Background</A><A href="https://2018.igem.org/Team:Goettingen/Description" class="sidebar-items sidebar-drop_items">Description</A><A href="https://2018.igem.org/Team:Goettingen/Design" class="sidebar-items sidebar-drop_items">Design</A><A href="https://2018.igem.org/Team:Goettingen/Experiments" class="sidebar-items sidebar-drop_items">Experiments</A><A href="https://2018.igem.org/Team:Goettingen/Notebook" class="sidebar-items sidebar-drop_items">Notebook</A><A href="https://2018.igem.org/Team:Goettingen/InterLab" class="sidebar-items sidebar-drop_items">InterLab</A><A href="https://2018.igem.org/Team:Goettingen/Model" class="sidebar-items sidebar-drop_items">Model</A><A href="https://2018.igem.org/Team:Goettingen/Results" class="sidebar-items sidebar-drop_items">Results</A><A href="https://2018.igem.org/Team:Goettingen/Demonstrate" class="sidebar-items sidebar-drop_items">Demonstrate</A><A href="https://2018.igem.org/Team:Goettingen/Improve" class="sidebar-items sidebar-drop_items">Improve</A><A href="https://2018.igem.org/Team:Goettingen/Attributions" class="sidebar-items sidebar-drop_items">Attributions</A></DIV><BUTTON class="dropdown_button sidebar-items">Parts</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Parts" class="sidebar-items sidebar-drop_items">Parts Overview</A><A href="https://2018.igem.org/Team:Goettingen/Basic_Part" class="sidebar-items sidebar-drop_items">Basic Parts</A><A href="https://2018.igem.org/Team:Goettingen/Composite_Part" class="sidebar-items sidebar-drop_items">Composite Parts</A><A href="https://2018.igem.org/Team:Goettingen/Part_Collection" class="sidebar-items sidebar-drop_items">Part Collection</A></DIV><A href="https://2018.igem.org/Team:Goettingen/Safety" class="sidebar-items">Safety</A><BUTTON class="dropdown_button sidebar-items">Human Practices</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Human_Practices" class="sidebar-items sidebar-drop_items">Human Practices</A><A href="https://2018.igem.org/Team:Goettingen/Human_Practices/Survey" class="sidebar-items sidebar-drop_items">Survey</A><A href="https://2018.igem.org/Team:Goettingen/Public_Engagement" class="sidebar-items sidebar-drop_items">Education &amp; Engagement</A></DIV><BUTTON class="dropdown_button sidebar-items">Awards</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Applied_Design" class="sidebar-items sidebar-drop_items">Applied Design</A><A href="https://2018.igem.org/Team:Goettingen/Hardware" class="sidebar-items sidebar-drop_items">Hardware</A><A href="https://2018.igem.org/Team:Goettingen/Measurement" class="sidebar-items sidebar-drop_items">Measurement</A><A href="https://2018.igem.org/Team:Goettingen/Model" class="sidebar-items sidebar-drop_items">Model</A></DIV><A href="https://igem.org/2018_Judging_Form?id=2586" class="sidebar-items">Judging Form</A></DIV><DIV id="main-content"><BUTTON id="show_nav" class="control-button">≡</BUTTON></DIV></DIV><DIV id="footer"><DIV class="l-two-col"><DIV class="contact"><SPAN class="footer_heading">Contact:</SPAN><TABLE><TBODY><TR><TH>E-mail:</TH><TD><A href="mailto:igem2018@gwdg.de">igem2018@gwdg.de</A></TD></TR><TR><TH>Adress:</TH><TD>c/o iGEM 2018</TD></TR><TR><TD>Department of General Microbiology</TD></TR><TR><TD>Grisebachstraße 8</TD></TR><TR><TD>37077 Göttingen</TD></TR><TR><TD>Germany</TD></TR></TBODY></TABLE></DIV><DIV class="shout_out"><P>We are especially grateful to our numerous sponsors on <A href="https://de.gofundme.com/glyphosate-on-my-plate">gofundme,</A> who supported our project and made it possible. Every public donor, who helped us to achieve our goals got a special place on our bacterial wall of fame and a handdrawn microbe to show our appreciation. A link to that wall can be found here:</P><P class="shout_out_wall_of"><A href="https://2018.igem.org/Team:Goettingen/WallOfFame">Bacterial Wall of Fame</A></P></DIV></DIV></DIV></DIV><DIV class="site-content"><DIV class="article"><DIV class="article_head-line"><H2>Results</H2></DIV><OL><LI><A href="#r_Uptake">Identification of glyphosate uptake systems</A></LI><LI><A href="#r_Identification">Identification of glyphosate-regulated promoters</A></LI><LI><A href="#r_reporterDetector">A reporter-based glyphosate detection system</A></LI><LI><A href="#r_Competition">Competition assay for glyphosate detection</A></LI><LI><A href="#r_Disarm">Engineering bacteria to disarm glyphosate</A></LI><LI><A href="#r_EPSP">Characterization of EPSP synthases</A></LI><LI><A href="#r_References">References</A></LI></OL><DIV id="r_Uptake"><H2>1. Identification of glyphosate uptake systems</H2></DIV><H4>1.1. Interaction between glyphosate and <EM>Bacillus subtilis</EM></H4><P>To create a proper detection system using the Gram-positive model bacterium <EM>Bacillus subtilis</EM>, <B>we first had to evaluate how this organism behaves in the presence of glyphosate</B>. Previously, it was shown that glyphosate negatively affects growth of <EM>B. subtilis</EM> due to the inhibition of the EPSP synthase AroE (Figure 1A) (1). Moreover, it has been demonstrated that 1.8 mM of glyphosate was required to inhibit the growth rate by 50%. To re-evaluate the effect of glyphosate on growth of our <EM>B.  subtilis</EM> laboratory strains 168 and SP1, we performed growth experiments in CS-Glc minimal medium that was supplemented with increasing amounts of glyphosate (Figure 1B). CS-Glc medium contains glucose and succinate as carbon sources and ammonium as the nitrogen source (see <A href="https://2018.igem.org/Team:Goettingen/Notebook">Notebook</A>). While the <EM>B. subtilis</EM> strain 168 is auxotrophic for tryptophan due to a mutation in the <EM>trpC</EM> gene, the strain SP1 can grow in the absence of exogenous tryptophan (2).

</P><DIV class="article_picture"><P>Figure 1. (A) Glyphosate inhibits the 5-enolpyruvyl-shikimate-3-phosphate (EPSP) synthase, which converts phosphoenolpyruvate (PEP) and 3-P-shikimate into EPSP and P in <I>B. subtilis</I>.  EPSP is a precursor for the aromatic amino acids phenylalanine (Phe), tyrosine (Tyr) and tryptophane (Trp). (B) How does glyphosate affect growth of our <I>B. subtilis</I> wild type strains?</P></DIV><P>


As shown in Figures 2A and 2B, at a glyphosate concentration of about 1 mM the growth rate of both strains was reduced by 50% and the bacteria were not able to grow at glyphosate concentrations higher than 3 mM. In contrast to a previous study (1), we have observed that 44% fewer glyphosate is required to reduce the growth rate of <EM>B. subtilis</EM> by 50%. This discrepancy might be due to differences in the genetic makeup of the <EM>B. subtilis</EM> strains, in the medium composition, in the purity of glyphosate or due to the different cultivation conditions. However, <B>glyphosate negatively affects growth of <EM>B. subtilis</EM></B> in CS-Glc minimal medium.

</P><DIV class="article_picture"><P>Figure 2. (A) Growth of the <I>B. subtilis</I> wild type strain 168  with increasing amounts of glyphosate (GS). (B) Growth of the <I>B. subtilis</I> wild type strain  SP1 with increasing amounts of GS. The figure inlays show the relationship between the growth rate (μ) and the GS concentration. The bacteria were cultivated at 37°C in CS-Glc minimal medium.</P></DIV><H4>1.2. Genomic adaptation of <EM>Bacillus subtilis</EM> to glyphosate</H4><P>Previously, it has been reported that the growth rate of a <I>B. subtilis</I> culture containing 2.5 mM glyphosate strongly increases after 16 h of incubation (1). It has been suggested that this phenomenon might be either due to a physiological adaptation of the bacteria to glyphosate or due to the emergence of glyphosate-resistant mutants. To assess whether <I>B. subtilis</I> is able to adapt to toxic levels of glyphosate at the genome level, we propagated the wild type strain 168 on CS-Glc agar plates supplemented with increasing amounts of the herbicide (0, 10, 20 30 and 40 mM). After incubation for 24 h at 37°C, bacterial growth was only visible on the plates lacking glyphosate. When the plates were further incubated for 48 h, <B>several glyphosate-resistant mutants appeared on the plates supplemented with 10 mM glyphosate </B>(Figure 3A). No mutants appeared on the plates with glyphosate concentrations higher than 10 mM. Next, we repeated the plating experiment with a mutant (designated as iGEM1) that was isolated from the 10 mM glyphosate plate. After 48 h of incubation, the glyphosate-resistant strain iGEM1 formed mutants on plates containing 30 mM and 40 mM glyphosate (Fig. 2A). Again we isolated mutants from the 30 mM and the 40 mM glyphosate plates that were designated as iGEM 7 and iGEM13, respectively (Figure 3B). Our attempts to further increase the glyphosate resistance of the strains iGEM7 and iGEM13 failed. To ensure that the acquired mutations in the strains iGEM1, iGEM7 and iGEM13 are stable, the mutants were passaged three times on SP rich medium plates. The glyphosate resistance of the mutants was confirmed by streaking the bacteria on CS-Glc medium supplemented with increasing amounts of glyphosate. As in Figure 3C, all strains that were adapted to glyphosate grew on plates with 10 mM glyphosate. The strains iGEM7 and iGEM13 also formed single colonies on 20 mM glyphosate plates. Moreover, albeit weak, the strain iGEM13 showed even growth with 40 mM glyphosate. 

</P><DIV class="article_picture"><P>Figure 3. Isolation and characterization of glyphosate (GS)-resistant mutants.
(A) Emergence of GS-resistant <I>B. subtilis</I> mutants on CS-Glc minimal medium agar plates that were incubated for 48 h at 37°C. WT, <I>B. subtilis</I> wild type strain 168; iGEM1, GS-resistant <I>B. subtilis</I> strain that was isolated from plates containing 10 mM GS.
(B) Genealogy of the GS-resistant <I>B. subtilis</I> mutants.
(C) Evaluation of growth of the GS-resistant mutants on CS-Glc minimal medium plates supplemented with increasing amounts of GS. The plates were incubated for 48 h at 37°C.
</P></DIV><P>

Our growth experiments using CS-Glc liquid medium with increasing amounts of glyphosate also confirmed that the strains iGEM7 and iGEM13 tolerate higher amounts of the herbicide than their progenitor strain iGEM1 (Figure 4; see below). To conclude, <B><I>B. subtilis</I> rapidly adapts to toxic glyphosate concentrations at the genome level!</B></P><DIV class="article_picture"><P>Figure 4. Characterization of glyphosate (GS)-resistant <I>B. subtilis</I> mutants. The wild type (WT) strain (A) and its derivatives iGEM1 (B), iGEM7 (C), and iGEM13 (D) were cultivated for 10 h at 37°C in CS-Glc minimal medium supplemented with increasing amounts of GS. 
</P></DIV><H4>1.3. Identification of mutations by genome and Sanger sequencing</H4><P>To identify the underlying genomic alterations causing glyphosate resistance the strains iGEM1, iGEM7 and iGEM13, we performed whole-genome sequencing with the help of the <A href="http://appmibio.uni-goettingen.de/index.php?sec=g2l">Göttingen Genomics Laboratory</A> (Figure 5A). By analysing the genome sequences we found that the bacteria had deleted a 1240 bp-long fragment containing a part of the 5’ ends of the <I>yhfE</I> and <I>gltT</I> genes and the entire <I>yhfF</I> gene  (Figure 5B). 

  </P><DIV class="article_picture"><P>Figure 5. The mutations of the glyphosate-resistant mutants and the link between glutamate and glyphosate transport.
(A) The illumina sequencing reads were mapped onto the reference the <I>B. subtilis</I> 168 reference genome NC_000964 from <A href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000964">GenBank</A> (3) using the Geneious software package (<A href="https://www.geneious.com/about/">Biomatters Ltd.</A>) (4).

(B) The mutations in the glyphosate (GS)-resistant <I>B. subtilis</I> mutants identified by genome sequencing.
(C) Effect of the identified mutations on the glutamate transporters GltT and GltP in the strains iGEM1, 7 and 13. Membrane topologies of the transporters were illustrated using the web-based protein topology tool <A href="http://wlab.ethz.ch/protter/start/">Protter</A> (5). 
(D) Comparison of glutamate and glyphosate.
</P></DIV><P>

The 1240 bp-long deletion is also present in the strains iGEM7 and iGEM13 because they were derived from the strain iGEM1 (Figure 5B). In addition to the deletion affecting the <I>yhfE</I>, <I>yhfF</I> and <I>gltT</I> genes, the strains iGEM7 and iGEM13 had replaced a single nucleotide and deleted 11 nucleotides, respectively, in the <I>gltP</I> gene. The <I>yhfE</I> gene encodes a putative aminopeptidase and the function of the <I>yhfF</I> gene is unknown (<A href="http://www.subtiwiki.uni-goettingen.de/v3/gene/view/A666F376947A675E04550338C3177FF8AFF44DF4">SubtiWiki</A>; 6). The <I>gltT</I> and <I>gltP</I> genes code for the high-affinity sodium-coupled glutamate/aspartate symporter GltT and for the low-affinity proton/glutamate symporter GltP, respectively (7,8). The fact that all strains had acquired mutations in genes encoding transporters that are involved in glutamate and aspartate uptake suggests that the proteins also transport glyphosate (Figure 5B). The 1240 bp- and 11 bp-long deletions in the <I>gltT</I> and <I>gltP</I> genes truncate the encoded proteins after 282 and 214 amino acids, respectively (Figure 5C). The G232A mutation in the <I>gltP</I> gene results in a replacement of a glutamate residue by a lysine residue at position 78, which is adjacent to the second transmembrane helix of the GltP transporter (Figure 5C). While the deletions certainly destroy the amino acid transporters, the E78K replacement probably reduces the transport of glyphosate <I>via</I> GltP. This idea is in line with the observation that the strain iGEM7 carrying the G232A mutation in the <I>gltP</I> gene did not grow as good as the strain iGEM13 with the deletion in <I>gltP</I> on plates with high amounts of glyphosate (Figure 5C). A structural comparison revealed that glutamate and glyphosate resemble each other because the molecules contain negatively charged groups encompassing positively charged groups (Figure 5D). Thus, both, GltT and GltP could be involved in glyphosate uptake. To conclude, <B><I>B. subtilis</I> rapidly adapts to toxic glyphosate concentrations by inactivating the <I>gltT</I> and <I>gltP</I> genes!</B></P><H4>1.4. Glyphosate-resistant mutants preferentially inactivate the <I>gltT</I> gene</H4><P>Previously, it has been shown that bacteria like <I>Salmonella typhimurium</I> can develop glyphosate resistance due to the acquisition of mutations in the promoter region and in the coding region of the <I>aroA</I> gene, elevating the EPSP synthase levels and rendering the encoded enzyme insensitive to glyphosate, respectively (9-11). Our genome sequencing analyses of the isolated iGEM1, iGEM7 and iGEM13 strains did not identify mutations that alter the synthesis or the coding sequence of the EPSP synthase. To assess whether <I>B. subtilis</I> can develop glyphosate resistance by the acquisition of mutations that alter the target of the herbicide, we isolated 12 additional mutants from CS-Glc plates containing 10 mM glyphosate. DNA sequencing analysis revealed that each strain had mutations within the <I>gltT</I> coding sequence that would truncate or lengthen GltT transporter (Figure 6). Thus, <B>the inactivation of the <I>gltT</I> gene is the predominant mutational event allowing <I>B.  subtilis</I> to grow in the presence of the weedkiller glyphosate</B>.
  </P><DIV class="article_picture"><P>Figure 6. The mutations in the <I>gltT</I> gene and the consequences for the encoded transporter GltT in the <I>B. subtilis</I> mutants that were isolated from CS-Glc medium agar plates containing 10 mM glyphosate.
 
</P></DIV><H4>1.5. Characterization of <I>gltT</I>, <I>gltP</I>  and <I>gltT</I><I>gltP</I> mutant strains</H4><P>Next, we tested whether the clean deletion of either the <I>gltT</I> gene or the <I>gltT</I><I>gltP</I> genes is indeed sufficient to confer high-level resistance of <I>B. subtilis</I> to glyphosate. For this purpose, we constructed the mutant strains BP233 (<I>gltT</I>), BP234 (<I>gltP</I>) and BP235 (<I>gltT</I><I>gltP</I>). To assess the glyphosate resistance of the <I>gltT</I>, <I>gltP</I> and <I>gltT</I><I>gltP</I> mutants, we cultivated the bacteria in CS-Glc minimal medium supplemented with increasing amounts of glyphosate. As shown in Figure 7A and 7C, growth of the wild type and the <I>gltP</I> mutant strains, respectively, was inhibited by 5 mM glyphosate. This observation indicates that <B>GltP is a low-affinity glyphosate transporter</B> and that the cell still contains a transport system for the herbicide. <B>The transporter with a high-affinity for glyphosate seems to be in fact GltT</B> because the deletion of the <I>gltT</I> gene conferred high-level resistance to the herbicide (Figure 7B). Moreover, the deletion of the <I>gltP</I> gene in the background of a <I>gltT</I> mutant strain further enhanced glyphosate tolerance and the growth rates of the strains BP233 (<I>gltT</I>) and BP235 (<I>gltT</I><I>gltP</I>) were reduced by 50% at herbicide concentrations of 6.1 mM and 8.0 mM, respectively (Figure 7B and 7D). Thus, in comparison to the wild type strain, 6- to 8-fold higher glyphosate concentrations are needed to reduce the growth rate of the strains BP233 (<I>gltT</I>) and BP235 (<I>gltT</I><I>gltP</I>) by 50% (Figure 8).

</P><DIV class="article_picture"><P>Figure 7. Inactivation of the glutamate transporters confers high-level resistance to glyphosate (GS). 
(A) Growth of the <I>B. subtilis</I> wild type (WT) strain 168 in CS-Glc minimal medium supplemented with increasing amounts of GS.   
(B) Growth of the Δ<I>gltT</I> mutant strain BP233 in CS-Glc minimal medium supplemented with increasing amounts of GS. The figure inlay shows the relationship between the growth rate (µ) and the GS concentration.  
(C) Growth of the Δ<I>gltP</I> mutant strain BP234 in CS-Glc minimal medium supplemented with increasing amounts of GS.   
(D) Growth of the Δ<I>gltT</I> Δ<I>gltP</I> mutant strain BP235 in CS-Glc minimal medium supplemented with increasing amounts of GS. The figure inlay shows the relationship between the growth rate (µ) and the GS concentration.  
</P></DIV><DIV class="article_picture"><P>Figure 8. The relationship between the growth rate (µ) and the glyphosate (GS) concentration for the <I>B. subtilis</I> wild type (WT) strain 168, 
the Δ<I>gltT</I> mutant strain BP233 and the the Δ<I>gltT</I> Δ<I>gltP</I> mutant strain BP235 in CS-Glc minimal medium supplemented with increasing amounts of GS.

</P></DIV>

We also tested whether the glyphosate that is present in commercially available Roundup® is toxic for <I>B. subtilis</I>. Roundup® was ordered from the German company 
<A href="https://www.westfalia.de/shops/garten/pflanzenschutz_und_duenger/unkrautbekaempfung/pflanzenschutzmittel/1228751-unkrautfrei_alphee_1_liter_die_nr_1_gegen_unkraut.htm?">Westfalia</A>. 1 l of Roundup® Alphee contains 9.4 g of the isopropylamine salt of glyphosate (which corresponds to 7.2 g/l pure glyphosate), 5 g surfactant (undefined) and water. To test the effect of Roundup® Alphee on growth of the bacteria, we propagated the <I>B. subtilis</I> strains 168 (wild type), BP233 (Δ<I>gltT</I>), BP234 (Δ<I>gltP</I>) and BP235 (Δ<I>gltT</I> Δ<I>gltP</I>) on CS-Glc minimal medium agar plates that were supplemented with glyphosate (control) and with equimolar amounts of glyphosate present in Roundup (test conditions). As shown in Figures 9A and 9B, pure glyphosate and <B>Roundup® Alphee killed the <I>B. subtilis</I> wild type strain</B> and the strain lacking the low-affinity glutamate transporter GltP already at a concentration of 2 mM. By contrast, all strains lacking the high-affinity transporter GltT grew in the presence of 2 - 6 mM glyphosate that is present in Roundup. Thus, <B>the unknown surfactant that is present in the commercially available Roundup® solution does not seem to affect growth of <I>B. subtilis</I></B>. Moreover, it is worth to mention that <B>the glyphosate present in the Roundup® solution is rather cheap: 7.2 g of glyphosate would make more than 3 k€ @ <A href="https://www.sigmaaldrich.com/catalog/product/sial/89432?lang=de&amp;region=DE">Merck</A></B>!


<DIV class="article_picture"><P>Figure 9. A. Growth of the strains 168 (wild type), BP233 (Δ<I>gltT</I>), BP234 (Δ<I>gltP</I>) and BP235 (Δ<I>gltT</I> Δ<I>gltP</I>) on CS-Glc minimal medium in the presence of different amounts of glyphosate (GS). B. Roundup® Alphee purchased from <A href="https://www.westfalia.de/shops/garten/pflanzenschutz_und_duenger/unkrautbekaempfung/pflanzenschutzmittel/1228751-unkrautfrei_alphee_1_liter_die_nr_1_gegen_unkraut.htm?">Westfalia</A>. C. Growth of the strains 168, BP233, BP234 and BP235 on CS-Glc minimal medium in the presence of different amounts of Roundup® Alphee.



</P></DIV><H4>1.6. Complementation of the <I>gltT</I> mutation</H4><P>
We also performed a complementation experiment to provide further evidence that GltT is the major glyphosate transporter in <I>B. subtilis</I>. For this purpose, we fused the artificial <I>Palf4</I> promoter (12) together with the <I>gapA</I> ribosome-binding site to the <I>gltT</I> gene and integrated the construct into the <I>amyE</I> locus of the <I>gltT</I> mutant (Figure 10A). Next, we propagated the strains BP233 (<I>gltT</I>) and BP237 (<I>gltT</I><I>Palf4-gltT</I>) together with the wild type strain 168 on agar plates without and with glyphosate (Figure 10B). As expected, all strains grew in the absence of glyphosate. By contrast, only the <I>gltT</I> mutant strain BP233 grew with glyphosate. Thus, like the wild type also the complementation strain BP237 take up glyphosate <I>via</I> GltT. To conclude, under the tested growth conditions <B>the high-affinity glutamate transporter GltT is the major entryway of glyphosate into <I>B. subtilis</I>!</B>.
  
</P><DIV class="article_picture"><P>Figure 10. Complementation of the Δ<I>gltT</I> mutant strain.
(A) The artificial <I>Palf4</I> promoter (12) and the ribosome-binding site (RBS) of the <I>B. subtilis gapA</I> gene were fused by PCR to the <I>gltT</I> gene.
(B) Growth of the <I>B. subtilis</I> wild type 168 (WT), the Δ<I>gltT</I> mutant BP233 and the Δ<I>gltT Palf4-gltT</I> complementation strain BP237 on CS-Glc minimal medium agar plates without glyphosate (0 mM GS) and with glyphosate (10 mM GS). The plates were incubated for 24 h at 37°C. 
 
</P></DIV><H4>1.7. GltT is also involved in the uptake of the herbicide glufosinate</H4><P> Our structural comparison of glutamate and glyphosate revealed that the two molecules resemble each other because they contain negatively charged groups encompassing positively charged groups (see Figure 5D). However, the structural similarity of glutamate and glyphosate is not very high. During a scientific discussion with our collaborator <STRONG> Dr. Till Ischebeck</STRONG> from the <A href="https://www.uni-goettingen.de/de/533683.html">Department of Plant Biochemistry</A> at the University of Göttingen, we came up with the idea that GltT may also transport the herbicide glufosinate, which is structurally more similar to glutamate than glyphosate and inhibits the glutamine synthetase (Figure 11A) (13). Glufosinate (phosphinothricin) is a naturally occurring broad-spectrum systemic herbicide produced by several species of <I>Streptomyces</I> soil bacteria. Plants may also metabolize 

<A href="https://en.wikipedia.org/wiki/Bialaphos">bialaphos</A> (L-Alanyl-L-alanyl-phosphinothricin), another naturally occurring herbicide, directly into glufosinate (13). The compound irreversibly inhibits glutamine synthetase, an enzyme necessary for the <I>de novo</I> synthesis of glutamine from ammonia and glutamate, giving it antibacterial, antifungal and herbicidal properties. Application of glufosinate to plants leads to reduced glutamine and elevated ammonia levels in tissues, halting photosynthesis, resulting in plant death. In the 1960s and early 1970s, scientists at University of Tübingen and at the <A href="https://www.meiji.com/global/about-us/corporate-profile/meiji-seika-pharma/">Meiji Seika Kaisha Company</A> independently discovered that species of <I>Streptomyces</I> bacteria produce a tripeptide they called bialaphos that inhibits bacteria; it consists of two alanine residues and a unique amino acid that is an analog of glutamate that they named &quot;phosphinothricin&quot;.




<B>The inactivation of the <I>gltT</I> gene indeed allowed <I>B. subtilis</I> to tolerate glufosinate (Figures 11B and 11C)!</B> Moreover, the observation that the expression of two copies of the <I>gltT</I> gene in strain BP236 (<I>amyE::Palf4-gltT</I>) increased the sensitivity of the bacteria toward glufosinate further supports the idea that GltT is involved in glufosinate uptake by <I>B. subtilis</I> (Figures 11B and 11C). In contrast to glyphosate, however, the simultaneous deletion of the <I>gltT</I> and <I>gltP</I> genes did not increase glufosinate tolerance of <I>B. subtilis</I>  (Figures 11B and 11C). To conclude, under the tested growth conditions <B>the high-affinity glutamate transporter GltT is the major entryway of glyphosate into <I>B. subtilis</I>  and GltT is also involved in Glufosinate uptake!</B></P><DIV class="article_picture"><P>Figure 11. Glufosinate is also taken up by <I>B. subtilis</I><I>via</I> GltT.
(A) Comparison of glutamate and glufosinate.
(B) Disk diffusion assay to test whether glufosinate is transported <I>via</I> GltT and/or GltP. The strains 168 (wild type), BP233 (Δ<I>gltT</I>), BP234 (Δ<I>gltP</I>, BP235 (Δ<I>gltT</I> Δ<I>gltP</I>), BP237 (Δ<I>gltT</I><I>amyE::Palf4-gltT</I>) and BP236 (<I>amyE::Palf4-gltT</I>) were cultivated in LB medium, washed once in 1 X C-salts and propagated (100 µl cell suspension, OD600 ≈ 2) on CS-Glc minimal medium agar plates. A filter disk containing 5 µl of a 0.5 M glufosinate solution was placed in the center of the plates, which were incubated for 2 days at 37°C. (C) Evaluation of the disk diffusion assay shown in (B). (D) Killing two birds with one stone: the Δ<I>gltT</I> mutation confers resistance to glyphosate and glufosinate! 
</P></DIV><H4>1.8. A novel nechanism conferring resistance to glyphosate</H4><P>
In the past years, the underlying molecular mechanisms conferring resistance to glyphosate have been intensively studied in bacteria and plants. <B>In many cases glyphosate resistance is directly linked to the target of the herbicide</B>. For instance, bacteria like <I>S. aureus</I> are <I>a priori</I> resistant to glyphosate because this organism synthesizes an insensitive EPSP synthase (14-17). Moreover, increased cellular levels of the EPSP synthase, which can be achieved either by overexpression of the coding gene or by gene amplification, can confer resistance to glyphosate (Figure 12) (9, 18-22). Due to the increased cellular levels of the EPSP synthase, the glyphosate is probably titrated away and sufficient amounts of the precursor for aromatic amino acid biosynthesis can be produced. As mentioned in the design section, several glyphosate-insensitive EPSP synthase mutant variants have been isolated and engineered (11, 23-36). Often, a single amino acid exchange is sufficient to render the EPSP synthase insensitive to glyphosate. However, although glyphosate resistance is frequently linked to the target of the herbicide, resistance against the herbicide may occur by other means. Several studies have demonstrated that glyphosate can be detoxified by covalent modification. For instance, the Gat glyphosate <I>N</I>-acetyltransferase from <I>Bacillus licheniformis</I>, which was subjected to directed evolution for creating an enzyme with higher efficiency and increased specificity for the herbicide, converts glyphosate to <I>N</I>-acetylglyphosate, which is not herbicidal and is not an effective inhibitor of EPSP synthases (Figure 12) (37-40). Moreover, the hygromycin phosphotransferases Hph and GlpA from <I>E. coli</I> and <I>Pseudomonas pseudomallei</I>, respectively, phosphorylate glyphosate and thus confer tolerance to the herbicide (Figure 12) (41,42). Interestingly, the <I>gat</I> gene has also been used as a selection marker for genetic engineering of bacteria (40). The enzymes that covalently modify glyphosate have been successfully introduced into crops to increase herbicide resistance (43).
	Recently, it has been also demonstrated that enhanced export of glyphosate can reduce toxicity of the herbicide. For instance, the overexpression of the uncharacterized membrane proteins MFS40 and YhhS from <I>Aspergillus oryzae</I> and <I>E. coli</I>, respectively, with similarity to the major facilitator secondary transporter superfamily, enhance glyphosate tolerance of <I>E. coli</I> (Figure 12) (44,45). It will be interesting to test whether these transporters are suitable for engineering crops to enhance glyphosate tolerance. Proteins of unknown function can also increase glyphosate tolerance. For instance, the overexpression of the <I>igrA</I> gene product from the <I>Pseudomonas</I> sp. strain PG2982 increases glyphosate tolerance in transgenic rice (46-48). Finally, many bacteria can survive in the presence of glyphosate because they are able to degrade the herbicide (Figure 12) (49-58). To conclude, <B>several mechanisms of glyphosate resistance have been described over the past years and novel mechanisms allowing survival with the herbicide are certainly identified in the future</B>.
We have observed that the deletion of the <I>gltT</I> gene in <I>B. subtilis</I> confers high-level glyphosate resistance to the bacteria. <B>The <I>B. subtilis</I><I>gltT</I> mutant strain could be a suitable host to identify glyphosate uptake systems from plants by expressing cDNA libraries and by screening for transformants that are unable to grow with glyphosate</B>. 

</P><DIV class="article_picture"><P>Figure 12. Mechanisms conferring resistance to glyphosate. 
GltT and GltP, <I>B. subtilis</I> glutamate transporters; YhhS and MFS40, major facilitator secondary transporters from <I>E. coli</I> and <I>A. oryzae</I>, respectively; GAT, <I>B. licheniformis</I> glyphosate <I>N</I>-acetyltransferase; GlpA, <I>P. pseudomallei</I> hygromycin phosphotransferase; PEP, phosphoenolpyruvate; 3PS, 3-phosphoshikimate; EPSP, enolpyruvylshikimate-3-phosphate; EPSPS, enolpyruvylshikimate-3-phosphate synthase. Membrane topologies of the transporters were illustrated using the web-based protein topology tool <A href="http://wlab.ethz.ch/protter/start/" target="_blank">Protter</A> (5).

 
</P></DIV><DIV id="r_Identification"><H2>2. Identification of glyphosate-regulated promoters</H2></DIV><H4>2.1. Effect of glyphosate on the <EM>Bacillus subtilis</EM> transcriptome</H4><P>The identification of genes that are up- or down-regulated in a glyphosate-dependent manner requires the application of &quot;omics&quot; techniques such as proteomics or transcriptomics. RNA-sequencing (RNA-Seq) analysis is a valid methodology to identify genes that are either induced or repressed by chemical substances such as nutrients and herbicides.


To assess the effect of glyphosate on the <EM>B. subtilis</EM> transcriptome, we performed an RNA-Seq analysis with the help of the <A href="http://appmibio.uni-goettingen.de/index.php?sec=g2l" target="_blank">Göttingen Genomics Laboratory</A>. The effect of glyphosate on the <EM>Escherichia coli</EM> transcriptome has been analyzed in the past years (59). By contrast, the effect of the weedkiller on gene expression in <EM>B. subtilis</EM> has remained elusive. To obtain bacterial cell pellets for RNA isolation and subsequent RNA-Seq analysis, we cultivated the <EM>B. subtilis</EM> wild type strain SP1 in CS-Glc minimal medium without glyphosate and supplemented with 0.75 mM glyphosate. We selected a concentration of 0.75 mM because it was previously shown that at a glyphosate concentration of about 1 mM the growth rate of the strain SP1 was reduced by 50% (see Figure 2). By choosing a lower concentration, we wanted to ensure that the bacteria were under stress without affecting growth too severe.</P><DIV class="article_picture"><P>Figure 13. Effect of glyphosate on the <EM>B. subtilis</EM> transcriptome. (A) The diagram shows the fold change of the <EM>B. subtilis</EM> RNAs during cultivation in presence of glyphosate. The strongest regulated operon is marked with yellow circles. (B) A list of genes whose expression was severely affected by glyphosate. The four most down- and up-regulated genes (marked in (A)).</P></DIV><P>The RNA-Seq analysis allowed us to identify genes whose expression is altered in a glyphosate-dependent manner. As illustrated in Figure 13A, the expression of only a few genes was affected by glyphosate. A 5-fold increase in gene expression was the highest increase we could observe. By contrast, we observed that the expression of some genes was 11-fold reduced. Nearly all up-regulated genes are involved in sporulation and <B>the <EM>yciA-yciB-yczL-yciC</EM> operon whose expression is affected by glyphosate is related to zinc deprivation responses </B>(60). Unexpectedly, we could not detect any effect of glyphosate on the expression of genes encoding proteins involved in shikimate metabolism and aromatic amino acid biosynthesis. This finding could indicate that the inhibition of the EPSP synthase and the changes of the cellular EPSP concentration does not influence the expression of the genes that are involved in aromatic amino acid biosynthesis in <EM>B. subtilis</EM>.


The promoters involved in the regulation of the <EM>yciA-yciB-yczL-yciC</EM> operon were <STRONG>identified as P<SUB>yciAB</SUB> and P<SUB>yciC</SUB> (61).</STRONG></P><DIV id="r_reporterDetector"><H2>3. A reporter-based glyphosate detection system</H2></DIV><H4>3.1 Utilising P<SUB>yciAB</SUB> and P<SUB>yciC</SUB> to build a reporter-based glyphosate detection system</H4><P>Our RNA-Seq analysis revealed that the expression of the operon containing the <EM>yciA-yciB-yczL-yciC</EM>genes was severely affected when <EM>B. subtilis</EM> was cultivated in the presence of glyphosate. The promoters involved in the regulation of these genes were identified as P<SUB>yciAB</SUB> and P<SUB>yciC</SUB>. Based on our RNA-Seq data, we aimed to construct a reporter-based glyphosate detection system. For this purpose, we fused the identified promoters to the <EM>xylR</EM> gene, which is a transcriptional repressor of the <EM>xylA-xylB</EM> and <EM>xynP-xynB</EM> operons in <EM>B. subtilis</EM> (Figure 14A). A <EM>B. subtilis</EM> mutant strain lacking the <EM>xylR</EM> gene was transformed with the plasmid pAC7 containing our promoter-gene constructs. We expect that upon contact of the constructed strains glyphosate, the expression of the <EM>xylR</EM> gene will be lower, which will lead to the expression of the <EM>xylA</EM> reporter gene. The expression of the <EM>xylA</EM> reporter gene can be monitored by performing a β-xylosidase assay using the colourimetric substrate 4-Nitrophenyl-β-D-xylopyranoside (PNPX). The measured β-xylosidase activity can be directly linked to the expression of the <EM>xylR</EM> gene, which controlled by the glyphosate-dependent promotor (Figure 14B). <B>The β-xylosidase activity should be directly proportional to the glyphosate that is present in the growth medium.</B></P><DIV class="article_picture"><P>Figure 14. (A) Sequence of the promotores P<SUB>yciAB</SUB> and P<SUB>yciC</SUB> including the ribosome-binding site (RBS) were fused to the <EM>xylR</EM> gene by PCR. (B) Mode of action of the glyphosate detection system.</P></DIV><P>By performing a β-xylosidase assay we could determine that this system does not react to the presence of glyphosate. This might be due to the fact, that the promotores are not regulated or the regulation is not strong enough.</P><DIV id="r_Competition"><H2>4. Competition assay for glyphosate detection</H2></DIV><H4>4.1. Competition assay with fluorophores</H4><P>
Unfortunately, current analytical methods, which are based on enzyme-linked immunosorbent assays (ELISAs) and mass spectrometry (MS) are very expansive, slow, need sophisticated equipment and well-trained technical staff. Therefore, <B>the development of a rapid and straightforward method to detect glyphosate is very attractive because the intensive use of the herbicide may lead to its accumulation in the soil and food</B>. The basic idea of the competition assay for the detection of glyphosate is depicted in Figure 15. 
</P><DIV class="article_picture"><P><STRONG>Figure 15. </STRONG>The basic principle of the competition assay for glyphosate detection. In the absence of glyphosate (no selective pressure), both strains grow with the same rate. When glyphosate is present, the herbicide-resistant strain (expressing an orange fluorophore gene) will outcompete the wild type strain. Thus, the velocity at which the wild type strain is eliminated from the co-culture is indicative for the glyphosate present in the culture or test sample.</P></DIV><P>Two <I>B. subtilis</I> strains that tolerate different amounts of glyphosate are labelled with different fluorophore genes (Figure 16). The fluorescence labelling of the bacteria allows to count the amount of each strain when co-cultivated in the absence and in the presence of selective pressure that is exerted by glyphosate using fluorescence microscopy. In the absence of selective pressure, we expect that both strains will survive the co-cultivation. By contrast, in the presence of glyphosate, the strain that tolerates glyphosate will outcompete the glyphosate-sensitive <I>B. subtilis</I> strain. Thus, <B>the selective pressure (≈ [glyphosate]) correlates with the amount of glyphosate-resistant cells and glyphosate-sensitive cells.</B></P><DIV class="article_picture"><P><STRONG>Figure 16. </STRONG> Integration of the fluorophore genes whose expression are controlled by an artificial σ<SUP>A</SUP>-dependent promoter (<EM>P<SUB>lij2</SUB></EM>), which is constitutively active. The constructs are be integrated into the <EM>amyE</EM> gene of the <EM>B. subtilis</EM> chromosome in single copy. RBS, ribosome-binding site; ORF, open reading frame.

</P></DIV><H5>Characterization of the mOrange protein in <I>B. subtilis</I></H5><P>The fluorophore genes are derived from the iGEM DNA distribution kit. Previous work using the <A href="http://parts.igem.org/Part:BBa_E2050">BBa_E2050</A> biobrick
        was focused on the function of the mOrange protein (an mRFP (DsRed) derivative) in the yeast <I>S. cerevisiae</I>. The aim of our experiments was
        to integrate the gene encoding the mOrange protein into the genomes of different <EM>B. subtilis</EM> strains. Unfortunately, the plasmid pSB1C3 does not contain an origin of
        replication for <EM>B. subtilis</EM>. Therefore, we introduced the fluorophore genes into the <EM>B. subtilis</EM> plasmid pAC7. For this purpose, we amplified the fluorophore genes by PCR, ligated the digested PCR products with the plasmid pAC7 that was cut with the same enzymes and transformed competent cells of the <EM>E. coli</EM> strain DH5α. During PCR, the fluorophore genes were fused to the artificial (<EM>P<SUB>lij2</SUB></EM> promoter, which should be constitutively active (see Figure 16). The promoter was designed <EM>in silico</EM> and contains binding sites for the housekeeping sigma factor A and a perfect RBS for <EM>B. subtilis</EM>. As shown in Figure 17, all strains synthesizing mOrange formed cell pellets with an orange color.</P><DIV class="article_picture"><P>Figure 17. A cell pellet of the <EM>B. subtilis</EM> wild type strain 168 is not orange. The cell pellets of the strains iGEM24, iGEM28 and iGEM36 synthesizing mOrange are slightly colored in orange.</P></DIV><P>


In the next step, the plasmid containing the mOrange gene was used to transform the <EM>B. subtilis</EM> wild type strain 168 and the glyphosate resistant strains BP233 (Δ<EM>gltT</EM>) and BP235 (Δ<EM>gltT</EM>Δ<EM>gltP</EM>). The activity of the mOrange fluorophore protein in the <EM>B. subtilis</EM> wild type strain 168 was further analyzed by fluorescence microscopy. Therefore, the strain was cultivated overnight at 37°C in the darkness. 0.01 ml of the culture were transferred to a microscope slide that was covered with solid 1% agarose dissolved in H<SUB>2</SUB>O to inhibit the movement of the cells during long-time exposure. As shown in Figure 18, cells synthesizing the mOrange fluorophore protein were highly fluorescent. Thus, <B>the mOrange fluorophore protein, which is part of the <A href="http://parts.igem.org/Part:BBa_E2050">BBa_E2050</A> biobrick can be produced in <EM>B. subtilis</EM></B>. 
    </P><DIV class="article_picture"><P>Figure 18. The left panel shows cells of the strain iGEM24 (wild type strain producing mOrange) that were visualized using bright field microscopy. Magnification = 40 X; exposure time = 0.85 s; no filter. The right panel shows cells of the same strain visualized by fluorescence microscopy. Magnification = 40 X; exposure time 44.33 s; DAPI filter.</P></DIV><P><B>To create a powerful glyphosate detection system, which is based on intraspecies competition, we need to have strains in hands having similar fitness coefficients</B>. Strains having a different fitness will not allow to generate a stable co-culture because the strain with the higher fitness will alway outcompete the other strain, even in the absence of a selective pressure. To characterize our strains and to determine their fitness we performed different growth experiments. First, we integrated fluorophore genes into the <I>amyE</I> gene of the <I>B. subtilis</I> strains 168, (wild type), Δ<I>gltT</I> (BP233) and Δ<I>gltT</I> Δ<I>gltP</I> (BP235). The wild type strain 168 is very sensitive to glyphosate (see Figure 2). By contrast, the Δ<I>gltT</I> single mutant and Δ<I>gltT</I> Δ<I>gltP</I> double mutant lacking the glyphosate transport systems tolerate high amounts of the herbicide (see Figures 7 and 8). We used the <I>gfp</I>, <I>bfp</I> and <I>morange</I> genes encoding the fluorophores GFP, BFP and mOrange, respectively, for strain labeling. To drive the expression of the fluorophore genes, we attached an artificial promoter to the genes during PCR (see Figure 16). The sigma factor A-dependent artificial promoter is constitutively active and contains a ribosome binding site for <I>B. subtilis</I>. Next, we cultivated the constructed strains in a multi-well plate reader, to determine their fitness based on the growth rates that are achieved in the medium. </P><DIV class="article_picture"><P><STRONG>Figure 19.</STRONG> Growth of the strains iGEM24 (<EM>mOrange</EM>), iGEM28 (Δ<EM>gltT mOrange</EM>) and iGEM36 (Δ<EM>gltT</EM>Δ<EM>gltP mOrange</EM>) in CS-Glc minimal medium. All strains synthesizing mOrange have the same growth rate.</P></DIV><P>As shown in Figure 19, all strains had similar growth rates in CS-Glc minimal medium. Thus, <B>the expression of the mOrange gene and the absence of the glyphosate transporters GltT and GltP does not affect the fitness of the strains!</B></P><P>To detect glyphosate by intraspecies competition, we mixed the wild type strain synthesizing the GFP protein with the Δ<EM>gltT</EM>Δ<EM>gltP</EM> double mutant synthesizing the mOrange fluorescence protein in a 1:1 ratio. As shown in Figure 20, the amount of wild type cells decreased in the co-culture with increasing glyphosate concentrations. By contrast, the amount of cells of the strain carrying the mOrange gene increased with increasing glyphosate concentrations since the Δ<EM>gltT</EM>Δ<EM>gltP</EM> mutant tolerates the herbicide glyphosate. Thus, <B>the intraspecies competition assay, which is based on fluorescently labelled <I>B. subtilis</I> strains seems to be suitable to detect glyphosate!</B></P><DIV class="article_picture"><P><STRONG>Figure 20.</STRONG> Green and orange cells represent the wild type strain (<I>gfp</I>) and  the double transporter mutant (Δ<EM>gltT</EM>Δ<EM>gltP mOrange</EM>), respectively. The ratio indicates the relative amount of orange cells in comparison to green cells. (A) Merged orange and green channels from culture with 0 mM glyphosate after 4 h of cultivation. There are nearly equal amounts of green and orange cells. (B) Merged channels orange and green from culture with 2 mM glyphosate after 4 h of cultivation. The number of orange cells is 4.21-fold higher. (C) Merged orange and green channels from culture with 3.5 mM glyphosate after 4 h of cultivation. Nearly no green cells were visible. (D) Calculated ratio between the number of orange and green cells with increasing glyphosate concentrations.</P></DIV><P>To perform a competition assay in a greater scale, we used a platereader for our growth experiments. This device enabled us to measure different fluorophores and the optical density simultaneously. Strains were cultivated in CS-glucose (CS-Glu) medium to an OD<SUB>600</SUB> of 1–1.5. A 96–well plate was prepared containing 100 µl of CS-Glc minimal medium with the desired concentrations of glyphosate. 10 µl of the cell mixture (1:1 ratio) were transferred into the wells to obtain an OD<SUB>600</SUB> of 0.1 in the wells. We used the <STRONG>Synergy MX II</STRONG> multi-well plate reader from the <A href="http://go.biotek.com/readers_de?Product_Interest_Source=PPC%20-%20Adwords&amp;source=PPC%20-%20Adwords&amp;Lead_Campaign_Source=7011O000002tr2g&amp;Campaign_Source=7011O000002tr2g&amp;gclid=Cj0KCQjw6fvdBRCbARIsABGZ-vTKWShl0pOKo23vGKkfvR5HGHyuG-xBZyMQf7Xg6AU56F2M_aH75kIaAnCWEALw_wcB">BioTek</A> company for our measurements. GFP was excited at 488 nm and emission was measured at 509 nm. for mOrange, the excitation wavelength was 529 nm and emission was detected at 562 nm.</P><DIV class="article_picture"><P><STRONG>Figure 21. </STRONG>The fluorescence is normed to the optical density, which equals the cell number. <STRONG>A:</STRONG> Intensity of GFP fluorescence decreases with inceasing glyphosate concentration(0–2 mM glyphosate). <STRONG>B:</STRONG> Intensity of mOrange fluorescence increases with increasing glyphosate concentration 0–2 mM glyphosate).</P></DIV><P>This competition assay enables us to calculate the concentration of glyphosate in samples from people that are curious, how much glyphosate is in their soil.</P><H4>4.2. Competition with β–galactosidase expressing strains</H4><P>Another approach using competition assays is to use enzymatic indication systems. Therefore, we used the strain BP235 (Δ<EM>gltT</EM>Δ<EM>gltP</EM>) and the strain BP193 (<EM>p<SUB>alf4</SUB>::lacZ</EM>) (12). As previously described, BP235 is characterized by high-level glyphosate resistance, while BP193 resembles the wild type despite the <EM>lacZ</EM> gene. The promoter <EM>p<SUB>alf4</SUB></EM> is an artificial one and harbors a perfect consensus sequence for <EM>B. subtilis</EM> and a ribosome binding site. This leads to constitutive expression of the <EM>lacZ</EM> gene and to blue cells if the medium is supplemented with X-Gal. Cultures of both strains in CS-Glc minimal medium without glyphosate were mixed to obtain a 1:1 ratio of both strains and incubated with increasing glyphosate concentrations (0 - 5mM) for 24 h (Figure 22).</P><DIV class="article_picture"><P><STRONG>Figure 22.</STRONG> Cell mixture (1:1) of strains BP193 and BP235 were incubated with increasing glyphosate concentrations (0–5 mM). Strain BP193 harbors the <EM>lacZ</EM> gene behind an artificial promoter, but does not tolerate glyphosate in contrast to strain BP235. The intensity of the blue color decreases with increasing glyphosate concentration. Strains BP193 and BP235 were also incubated without glyphosate as a control.</P></DIV><P>The intensity of the blue color drecreases with increasing glyphosate concentrations (Figure 22). The change and intensity of the blue color correlates with the glyphosate concentration and can therefore be used as detection and quantification system. Furthermore, a β–galactosidase assay can quantify the activity of the promoter <EM>p<SUB>alf4</SUB></EM> and thus, the concentration of glyphosate.</P><P> 
The promoter-reporter gene fusion of <EM>p<SUB>alf4</SUB></EM> and lacZ allows to determine the activity of the promoter. Since <EM>p<SUB>alf4</SUB></EM> is constitutively active this is directly linked to the survival of the cells. Transcription results in synthesis of ß-galactosidase which in turn hydrolyses o-nitrophenyl-ß-D-galactopyranoside (ONPG), resulting in the formation of o-nitrophenol. Formed o-nitrophenol absorbs light with a wavelength of 420 nm, and can therefore be used as an indicator for the amount of cells and subsequently for the concentration of glyphosate. We determined ß-galactosidase activity for the mixture of BP193 and BP235 in presence of increasing amounts of glyphosate. In addition, we used BP193 and BP235 as controls. Cells were grown in a 1:1 ratio. ß-galactosidase activity was high in the BP193 control and at low glyphosate concentration. Increasing glyphosate concentration is directly linked to cell death of BP193 and a corresponding decrease in ß-galactosidase activity (Figure 23). 
</P><DIV class="article_picture"><P><STRONG>Figure 23.</STRONG> Cell mixture (1:1) of strains BP193 and BP235 were incubated with increasing glyphosate concentrations (0 - 5 mM). Strain BP193 harbors the <EM>lacZ</EM> gene behind an artificial promoter, but does not tolerate glyphosate in contrast to strain BP235. ß-galactosidase activity can be measured at a wavelength of 420 nm using the substrate o-nitrophenyl-ß-D-galactopyranoside. Activity decreases with increasing glyphosate concentration. BP235 does not contain the <EM>lacZ</EM> gene, hence no activity can be observed.</P></DIV><P> 
In an additional step, we verified that the change in coloration occurred due to the death of the sensitive strain on the one hand and the increased growth of the resistant BP235 strain on the other hand. Hence, we adopted the experimental set up and analysed growth of the two strains in the platereader. To analyse growth of BP235, we used the strain constitutively expressing GFP, which was constructed similarly to the previously used strains. For the analysis we again used 100 µL CS-Glu medium supplemented with the appropriate glyphosate concentration. The cells were added in a 1:1 ratio. GFP was excited at 488 nm and emission was measured at 509 nm. </P><DIV class="article_picture"><P><STRONG>Figure 24. </STRONG> Cell mixture (1:1) of strains BP193 and BP235 were incubated with increasing glyphosate concentrations (0–5 mM). Strain BP193 harbors the <EM>lacZ</EM> gene behind an artificial promoter, but does not tolerate glyphosate in contrast to strain BP235.
</P></DIV><DIV id="r_Disarm"><H2>5. Engineering bacteria to disarm glyphosate</H2></DIV><H4>5.1. The glyphosate <I>N</I>-acetyltransferase confers resistance to weedkiller</H4><P>Many years ago it has been found that glyphosate can be inactivated by acetylation yielding <I>N</I>-acetylglyphosate (see Figure 25) (37). Because the acetylated form of glyphosate has low affinity for the active site of EPSP synthase, it is nonherbicidal. <B>The glyphosate <I>N</I>-acetyltransferase GAT from the Gram-positive soil bacterium <I>Bacillus licheniformis</I> transfers the acetyl group from acetyl-CoA onto the amino group of the weedkiller</B> (37). Interestingly, <I>B. subtilis</I> possesses a putative glyphosate N-acetyltransferase, which is designated as <A href="http://www.subtiwiki.uni-goettingen.de/v3/gene/view/2751630B6472346B19FB20F57F16013D0FACA9F7">YitI</A> and shares 59% overall amino acid identity with GAT of <I>B. licheniformis</I>. However, the native GAT enzymes are very poor catalysts for <I>N</I>-acetylation of glyphosate (37). 
Moreover, despite extensive screening of biological amines, including amino acids, nucleotides and antibiotics, the physiological substrates for the native enzymes are unknown (39). As an alternative strategy for glyphosate resistance involving enzymatic conversion of glyphosate to <I>N</I>-acetylglyphosate the GAT was also subjected to directed evolution for creating an enzyme with higher efficiency and increased specificity for the herbicide (37). When introduced into plants, optimized <I>gat</I> genes confer robust tolerance to glyphosate (37). Therefore, we expect that bacteria like <I>B. subtilis</I> expressing the optimized GAT should tolerate high amounts of glyphosate even though it is taken up from the environment <I>via</I> the high-affinity glyphosate transporter GltT (see above). To evaluate the potential of the GAT to allow <I>B. subtilis</I> in the presence of glyphosate concentrations that are toxic for the wild type bacteria, we wanted to express the <I>gat</I> gene in <I>B. subtilis</I> (Figure 25). 

</P><DIV class="article_picture"><P>Figure 25. (A) Sequence alignment showing the similarities of the wild type (wt) <I>gat</I> alleles from <I>Bacillus licheniformis</I> and <I>Bacillus subtilis</I> as well as the <I>gat</I> alleles R7 and R11 from <I>B. licheniformis</I> that were obtained by 11 rounds of gene shuffling. The red arrows indicate the 31 amino acid exchanges in the glyphosate <I>N</I>-acetyltransferase (Gat) variant R11. The optimized Gat variants shown an up to a 4,500-fold increase in catalytic efficiency (kcat/Km) relative to the native enzyme. The <A href="https://www.ncbi.nlm.nih.gov/genbank/?">GenBank</A> accession numbers for the sequences: <I>B. licheniformis</I> ST401 GAT, <A href="https://www.ncbi.nlm.nih.gov/nuccore/AX543338">AX543338</A>; R7 GAT, <A href="https://www.ncbi.nlm.nih.gov/nuccore/AY597417">AY597417</A>; R11 GAT, <A href="https://www.ncbi.nlm.nih.gov/nuccore/AY597418">AY597418</A>; <I>B. subtilis</I><A href="http://subtiwiki.uni-goettingen.de/v3/gene/view/2751630B6472346B19FB20F57F16013D0FACA9F7">YitI</A>, <A href="https://www.ncbi.nlm.nih.gov/protein/CAA70664"> CAA70664</A>. (B) Structure of the Gat R7 variant (PDBid: <A href="https://www.rcsb.org/structure/2jdd">2JDD</A>) in ternary complex with acetyl-CoA and the competitive inhibitor 3-phosphoglycerate (3PG). The four active site residues (Arg-21, Arg-73,
Arg-111, and His-138), which are labelled in black color, contribute to a positively charged substrate-binding site that is conserved throughout the GAT subfamily. (C) Top view on the active site of Gat R7. (D) Construction of a plasmid for the expression of the <I>gat</I> R11 variant in <I>B. subtilis</I>. Expression is driven by the constitutively active σA-dependent <I>Palf4</I> promoter  The plasmid pAC7::<I>gat</I> R11 integrates into the <I>amyE</I> locus of the <I>B. subtilis</I> chromosome in single copy. <I>lacZ</I>, beta-galactosidase; <I>amyE-5'</I> and <I>amyE-3'</I>, fragments of the <I>B. subtilis</I><I>amyE</I> for homologous recombination; <I>amp</I>, ampicillin resistance gene for the selection in <I>E. coli</I>; <I>kan</I>, kanamycin resistance gene for the selection in <I>B. subtilis</I>; <I>ori</I> pBR322, origin of replication for <I>E. coli</I>.

</P></DIV><P>



For this purpose, we ordered the <I>gat</I> gene as a g-Block from <A href="https://eu.idtdna.com/pages">Integrated DNA Technologies</A>, which was free of charge because the company supports the iGEM competition. Next, we amplified the <I>gat</I> gene by PCR using the g-Block as template DNA. An artificial promoter and a ribosome binding site was attached during the PCR (Figure 25D). The promoter-gene fusion was cloned and introduced in single copy into the genomes of the <I>B. subtilis</I> wild type strain and the <I>gltT</I> mutant strain to further increase its glyphosate tolerance (Figure 7). To assess the potential of the GAT to disarm glyphosate in the different <I>B. subtilis</I> strains, we propagated the bacteria on CS-Glc minimal medium agar plates that were supplemented with different amounts of glyphosate. As shown in Figure 26, <B>the strains synthesizing the GAT tolerated significantly more glyphosate than the wild type strain</B>. Moreover, the inactivation of the <I>gltT</I> glyphosate transporter gene and the overexpression of the <I>gat</I> gene confers high-level glyphosate tolerance to the bacteria. Thus, when introduced into <I>B. subtilis</I>, the optimized <I>gat</I> gene confers indeed robust tolerance to glyphosate. To conclude, <B>the engineered bacteria possess two interesting properties: (i) <I>B. subtilis</I> promotes plant growth by protecting plants against pathogens and (ii), the bacteria take up glyphosate from the environment and inactivate the weedkiller!</B></P><DIV class="article_picture"><P>Figure 26. (A) Growth of the wild type and Δ<I>gltT</I> mutant and of the isogenic strains overexpressing the <I>gat</I> R11 and <I>aroA*</I> genes on CS-Glc minimal medium in the presence of different amounts of glyphosate. (B) Growth of the strains described in A on CS-Glc minimal medium in the presence of different amounts of Roundup® Alphee.

 
</P></DIV><P> Cultivation of the different strains in liquid media allows more detailed characterization. Therefore, we analyzed growth of the bacteria in CS-Glc minimal medium supplemented with increasing amounts of glyphosate, using a multi-well plate reader. In general, the analysis supports previous results (see Figure 26). The strain containing the <I>aroA*</I> gene confers only a slight increase in resistance in comparison to the wild type strain and hence a significantly lower resistance in comparison the <EM>gltT</EM> mutant strain (Figure 27A, B and D). However, for the strain containing the <EM>aroA*</EM> gene and lacking the glyphosate transporter GltT a 9-fold higher glyphosate concentration is needed to reduce the growth rate by 50% (Figure 27F). The growth experiments confirm our previous results showing that the glyphosate resistance in can be strongly enhanced in <I>B. subtitlis</I> by expressing the <EM>gat</EM> gene or by inactivating the <EM>gltT</EM>. Moreover, a combination of both even further enhances glyphosate resistance of the bacteria.



 (Figure 27C and E, Figure 26). </P><DIV class="article_picture"><P>Figure 27. (A) Growth of the <EM>B. subtilis</EM> wild type strain in presence of 0 - 3.5 mM glyphosate. (B) Growth of the <EM>B. subtilis</EM><EM>gltT</EM> mutant strain with increasing glyphosate concentrations (0-30 mM). (C) Growth of the <EM>B. subtilis</EM> strain producing Gat with increasing glyphosate concentrations (0-30 mM. (D) Growth of the <EM>B. subtilis</EM> strain producing AroA* with increasing glyphosate concentrations (0-30 mM). (E) Growth of the <EM>B. subtilis</EM> strain producing AroA* with increasing glyphosate concentrations (0-30 mM). Growth of the <EM> B. subtilis</EM> strain lacking GltT and producing Gat with increasing glyphosate concentrations (0-30 mM). (F) Growth of the <EM> B. subtilis</EM> strain lacking GltT and producing AroA* with increasing glyphosate concentratiosn (0-30 mM). Bacteria were cultivated in liquid CS-Glc minimal medium. 

 
</P></DIV><DIV class="article_picture"><P>Figure 28. The relationship between the growth rate (µ) and the glyphosate (GS) concentration for the <EM>B. subtilis</EM> wild type (WT) strain SP1, the Δ<EM>gltT</EM> mutant strain BP233, the <EM>B. subtilis</EM> strain producing Gat, the <EM>B. subtilis</EM> strain producing AroA*, the <EM>B. subtilis</EM> strain lacking GltT and producing AroA*, the <EM>B. subtilis</EM> strain lacking GltT and producing Gat in CS-Glc minimal medium supplemented with increasing amounts of GS. </P></DIV><DIV id="r_EPSP"><H2>6. Characterization of EPSP synthases</H2></DIV><H4>6.1. Essentiality of the <EM>B. subtilis</EM><EM>aroE</EM> EPSP synthase gene</H4><P>To our surprise, <B>we did not find glyphosate-resistant mutants of <EM>B. subtilis</EM> that had accumulated mutations in the <EM>aroE</EM> gene rendering the encoded EPSP synthase insensitive to the herbicide</B>. Since AroE from <EM>B. subtilis</EM> belongs to the class II EPSP synthases, which are less sensitive to glyphosate, the potential of enzyme to further evolve herbicidal insensitivity might be limited (Figure 29) (62-64). 

</P><DIV class="article_picture"><P>Figure 29. The EPSP synthase fall into three different groups. While the <EM>E. coli</EM> EPSP synthase glyphosate sensitive (group I), the <EM>B. subtilis</EM> enzyme belongs to the group II of glyphosate-insensitive EPSP synthases. Not much is known about the glyphosate-dependent inhibition of the EPSP synthases that belong to group III. The phylogenetic three has been constructed using the Geneious software package (<A href="https://www.geneious.com/about/">Biomatters Ltd.</A>). Sequences of the EPSP synthases were derived from the <A href="https://www.uniprot.org">UniProt</A> database.
</P></DIV>

Moreover, <B>experimental evidence suggests that the <EM>B. subtilis aroE</EM> gene is essential for growth even under conditions that do not require the enzymatic function of the EPSP synthase</B> (65). As essential bacterial genes appear to be more conserved than nonessential genes, the <EM>aroE</EM> gene probably does not permit the accumulation of mutations, which can be beneficial (66). To confirm that the <EM>aroE</EM> gene is indeed essential, we transformed the <EM>B.%nbsp;subtilis</EM> with a PCR product consisting of DNA fragments flanking the target gene and the intervening <EM>spc</EM> spectinomycin gene (<EM>aroE::spc</EM> deletion cassette) and propagated the bacteria on SP rich medium plates (see Notebook). Several tiny colonies appeared on the transformation plates after 24 h of incubation. However, the potential transformants were not viable and supplementation of the agar plates with casamino acids also did not improve growth of the bacteria. Thus, under the tested conditions <EM>aroE</EM> seems to be essential in <EM>B. subtilis</EM>.
	Next, we tested whether the chromosomal copy of the <EM>aroE</EM> gene becomes dispensable for the bacteria carrying an EPSP synthase gene on a plasmid. For this purpose, we transformed the <EM>B. subtilis</EM> wild type strain 168 with the plasmids pIGEM2 and pBP143 containing the <EM>aroE</EM> and <EM>aroA</EM> EPSP synthase genes from <EM>B. subtilis</EM> and <EM>E. coli</EM>, respectively. The wild type carrying the empty plasmid pBQ200 served as a control. Next, we transformed the three strains with the <EM>aroE::spc</EM> deletion cassette and with chromosomal DNA of strain BP233 (<EM>gltT::spc</EM>), of which the latter served as the positive control. While we did not get transformants without DNA, many transformants appeared with chromosomal DNA of strain BP233 (Figure 30). Moreover, the chromosomal copy of the <EM>aroE</EM> gene was only dispensable in strains carrying an extra EPSP synthase gene on a plasmid. <B>To conclude, the <EM>aroE</EM> EPSP synthase gene is essential in <EM>B. subtilis</EM>. Moreover, the bacteria seem to require the enzymatic function of the EPSP synthase because also the enzyme from <EM>E. coli</EM> permits growth of the <EM>aroE</EM> mutant</B> (Figure 30).<DIV class="article_picture"><P>Figure 30. The EPSP synthase is essential in <EM>B. subtilis</EM>. 
Transformation experiment to evaluate the essentiality of the EPSP synthase in <EM>B. subtilis</EM>. The bacteria were propagated on SP plates supplemented with spectinomycin and incubated for 24 h at 37°C.
</P></DIV><H4>6.2. Purification and characterization of EPSP synthases</H4><P>This year, our team came toghether with the <A href="https://2018.igem.org/Team:Marburg">iGEM team of Marburg</A>, to investigate the interaction of the EPSP synthases AroE and AroA from <EM>B. subtilis</EM> and <EM>E. coli</EM>, respectively, with glyphosate. Glyphosate targets AroE and AroA in <EM>B. subtilis</EM> and <EM>E. coli</EM>, respectively (see Background information). Since the members of the <A href="https://www.bangelab.org">Bange lab</A>, which is hosting the <A href="https://2018.igem.org/Team:Marburg">iGEM team of Marburg</A>, are experts in the field of 
<A href="https://en.wikipedia.org/wiki/Isothermal_titration_calorimetry">Isothermal titration calorimetry</A> (ITC), we did not hesitate to contact them for support. Our team cloned the <EM>aroE</EM> and <EM>aroA</EM> genes of <EM>B. subtilis</EM> and <EM>E. coli</EM>, respectively, and <B>the resulting constructs are suitable for overexpression of the 
Strep-tagged AroE and AroA enzymes in the <EM>E. coli</EM> strain BL21</B>. The N-terminally Strep-tagged proteins can be purified by Streptactin:Strep-tag affinity purification system from the 
<A href="https://www.iba-lifesciences.com/home.html?gclid=CjwKCAjwlejcBRAdEiwAAbj6Kbo80UFPjKO6Yzdy03cr_0LVHUcV7ZbYCE-x-HBVWXZY9-_pbBP4QhoCrfAQAvD_BwE">IBA</A>, Göttingen. The purified proteins were send to the iGEM team in Marburg (Figure 31). The ITC measurments were performed in Marburg. Unfortunately, not interaction between glyphosate and the EPSP synthases could be detected. Unfortunately, also the second ITC measurement using freshly purified AroA and AroE enzymes from <EM>E. coli</EM> and <EM>B. subtilis</EM>, respectively, did not reveal whether glyphosate may interact with the EPSP synthases. Nevertheless, we are grateful to the iGEM Team Marburg for performing the ITC measurements.
        
</P><DIV class="article_picture"><P>Figure 31. (A) Evaluation of the purification of the N-terminally Strep-tagged EPSP synthases from <I>B. subtilis</I> and <I>E. coli</I> Strep-AroE and Strep-AroA, respectively, by 12% SDS PAGE. The proteins were stained with Coomassie Brilliant Blue. M, unstained protein molecular weight marker <A href="https://www.thermofisher.com/de/de/home/brands/thermo-scientific/molecular-biology.html">Thermo Scientific</A>; CE, crude extract; FT, flow through, W, washing steps; E, elution steps. (B) ITC measurements with Strep-AroE and Strep-AroA and glyphosate. (C) Control experiments with water and the calcium chelator EDTA. </P></DIV><DIV id="r_References"><H2>References</H2></DIV><OL><LI>Fischer <EM>et al.</EM> (1986) J. Bacteriol. 168: 1147-1154
    </LI><LI>Zeigler <EM>et al.</EM> (2008) J. Bacteriol. 190: 6983-6995.
</LI><LI>Barbe <EM>et al.</EM> (2009) Microbiology. 155: 1758-1775
</LI><LI>Kearse <EM>et al.</EM> (2012) Bioinformatics. 28: 1647-1649.
</LI><LI>Omasits <EM>et al.</EM> (2014) Bioinformatics. 30: 884-886. 
</LI><LI>Michna <EM>et al.</EM> (2014) Nucleic Acids Res. 42: D692–D698.    
</LI><LI>Zaprasis <EM>et al.</EM> (2015) Appl. Environ. Microbiol. 81: 250-259.
 </LI><LI>Tolner <EM>et al.</EM> (1995) J. Bacteriol. 177: 2863-2869.
 </LI><LI>Amrhein <EM>et al.</EM> (1980) Naturwissenschaften. 67: 356-357.
</LI><LI>Comai <EM>et al.</EM> (1983) Science. 221: 370-371.
</LI><LI>Stalker <EM>et al.</EM> (1985) J. Biol. Chem. 260: 4724-4728.    
</LI><LI>Gundlach <EM>et al.</EM> (2017) Sci. Signal. 10: eaal3011. 
</LI><LI>Gundlach <EM>et al.</EM> (2017) Sci. Signal. 10: eaal3011. 
</LI><LI> Fraser &amp; Ridley (1984) Planta. 161: 470-474.
</LI><LI>Priestmann et al., (2005) FEBS Lett. 579: 728-732.
</LI><LI>Cao <EM>et al.</EM> (2012) PLoS One. 7: e38718.
</LI><LI>Light <EM>et al.</EM> (2016) Biochemistry. 55: 1239-1245.
</LI><LI>Liu &amp; Cao, (2018) Biotechnol. Lett. 40: 855-864.
</LI><LI>Rogers <EM>et al.</EM> (1983) Appl. Envrion. Microbiol. 46: 37-43.
</LI><LI>Gaines <EM>et al.</EM> (2011) J. Agric. Food Chem. 59: 5886-5889.
</LI><LI>Juglam <EM>et al.</EM> (2014) Plant Physiol. 166: 1200-1207.
</LI><LI>Sammons &amp; Gaines (2014) Pest. Manag. Sci. 70: 1367-1377.
</LI><LI>Dillon <EM>et al.</EM> (2017) Plant Physiol. 173: 1226-1234.
</LI><LI>Sost &amp; Amrhein (1990) Arch. Biochem. Biophys. 282: 433-436.
</LI><LI>Padgette <EM>et al.</EM> (1991) J. Biol. Chem.  266: 22364-22369.
</LI><LI>He <EM>et al.</EM> (2001) Biochim. Biophys. Acta 1568: 1-6.
</LI><LI>He <EM>et al.</EM> (2003) Biosci. Biotechnol. Biochem. 67: 1405-1409.
</LI><LI>Bearson <EM>et al.</EM> (2002) Plant Physiol. 129: 1265-1275.
</LI><LI>Eschenburg <EM>et al.</EM> (2002) Planta 216: 129-135.
</LI><LI>Sun <EM>et al.</EM> (2005) Appl. Environ. Microbiol. 71: 4771-4776.
</LI><LI>Zhou <EM>et al.</EM> (2006) Plant Physiol. 140: 184-195.
</LI><LI>Healy-Fried <EM>et al.</EM> (2007) J. Biol. Chem. 282: 32949-32955.
</LI><LI>Vande Berg <EM>et al.</EM> (2008) Pest. Manag. Sci. 64: 340-345.
</LI><LI>Funke <EM>et al.</EM> (2009) J. Biol. Chem. 284: 9854-9860.
</LI><LI>Tian <EM>et al.</EM> (2010) Appl. Environ. Microbiol. 76: 6001-6005.
</LI><LI>Pollegioni <EM>et al.</EM> (2011) FEBS J. 278: 2753-2766.
</LI><LI>Chekan <EM>et al.</EM> (2016) MedChemComm. 7: 28-36.
    </LI><LI>Castle <EM>et al.</EM> (2004) Science. 304: 1151-1154.
 </LI><LI>Siehl <EM>et al.</EM> (2005) Pest. Manag. Sci. 61: 235-240.
    </LI><LI>Siehl <EM>et al.</EM> (2007) J. Biol. Chem. 282: 11446-11455.
</LI><LI>Norris <EM>et al.</EM> (2009) Appl. Environ. Microbiol. 75: 6062-6075.
</LI><LI>Rao <EM>et al.</EM> (1983) Antimicrob. Agents Chemother. 24: 689-695.
</LI><LI>Penaloza-Vazquez <EM>et al.</EM> (1995) Appl. Environ. Microbiol. 61: 538-543.
</LI><LI>Liu <EM>et al.</EM> (2015) Transgenic Res. 24: 753-763.
</LI><LI>Staub <EM>et al.</EM> (2012) J. Ind. Microbiol. Biotechnol. 39: 641-647.
</LI><LI>Tao <EM>et al.</EM> (2017) Pestic. Biochem. Physiol. 140: 65-68.
</LI><LI>Fitzgibbon &amp; Braymer, (1988) Appl. Environ. Microbiol. 54: 1886-1888.
</LI><LI>Fitzgibbon &amp; Braymer, (1990) Appl. Environ. Microbiol. 56: 3382-3388.
</LI><LI>Fartyal <EM>et al.</EM> (2018) Front. Plant. Sci. 13: 144.
</LI><LI>Pipke <EM>et al.</EM> (1980) Appl. Environ. Microbiol. 53: 974-978.
</LI><LI>Shinabarger <EM>et al.</EM> (1986) J. Bacteriol. 168: 702-707.
</LI><LI>Liu <EM>et al.</EM> (1991) Appl. Environ. Microbiol. 57: 1799-1804.
</LI><LI>Dick &amp; Quinn, (1995) Appl. Microbiol. Biotechnol.  43: 545-550.

</LI><LI>Singh &amp; Walker, (2006) FEMS Microbiol. Rev. 30: 428-471.
</LI><LI>Castro <EM>et al.</EM> (2007) J. Environ. Sci. Health B. 42: 883-886.
</LI><LI>Hove-Jensen <EM>et al.</EM> (2014) Microbiol. Mol. Biol. Rev. 78: 176-197.
</LI><LI>Kryuchkova <EM>et al.</EM> (2014) Microbiol. Res. 169: 99-105. 
</LI><LI>Sviridov <EM>et al.</EM>, (2011) Biochemistry (Mosc). 76: 720-725.
</LI><LI>Sviridov <EM>et al.</EM> (2015) Prikl. Biokhim. Mikrobiol. 51: 183-190.
</LI><LI>Lu <EM>et al.</EM> (2013) Mol Biosyst. 9: 522–30. 
</LI><LI>Gaballa &amp; Helmann (1998) J. Bacteriol. 180: 5815–5821.
</LI><LI>Gaballa <EM>et al.</EM> (2002) J. Bacteriol. 184: 6508–6514.
</LI><LI>Barry <EM>et al.</EM> (1992) WO 92/04449.
</LI><LI>Schönbrunn <EM>et al.</EM> (2001) Proc. Natl. Acad. Sci. USA. 98: 1376-1380.
</LI><LI>Light <EM>et al.</EM> (2016) Biochemistry. 66: 1239-1245.
</LI><LI>Koo <EM>et al.</EM> (2017) Cell Systems. 4: 291-305.
</LI><LI>Jordan <EM>et al.</EM> (2002) Genome Res. 12: 962-968.
</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>