<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><BUTTON id="top-scroll" class="control-button">⇧</BUTTON></DIV><DIV id="header"><DIV id="header_team"><P>
        Team Göttingen iGEM 2018
      </P></DIV><DIV id="header_middle"><DIV id="header_middle-container"><P>Glyphosate <FONT size="4" color="white">on my</FONT> plate?</P></DIV></DIV></DIV><DIV id="main_wrapper"><DIV id="content_menu_wrapper"><DIV id="navigation" class="sidebar is-nav-hidden"><A href="https://2018.igem.org/Team:Goettingen" class="sidebar-items">Home</A><A href="https://2018.igem.org/Team:Goettingen/News" class="sidebar-items">News</A><BUTTON class="dropdown_button sidebar-items">Team</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Team" class="sidebar-items sidebar-drop_items">Team Members</A><A href="https://2018.igem.org/Team:Goettingen/Collaborations" class="sidebar-items sidebar-drop_items">Collaborations</A></DIV><BUTTON class="dropdown_button sidebar-items">Project</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Background" class="sidebar-items sidebar-drop_items">Background</A><A href="https://2018.igem.org/Team:Goettingen/Description" class="sidebar-items sidebar-drop_items">Description</A><A href="https://2018.igem.org/Team:Goettingen/Design" class="sidebar-items sidebar-drop_items">Design</A><A href="https://2018.igem.org/Team:Goettingen/Experiments" class="sidebar-items sidebar-drop_items">Experiments</A><A href="https://2018.igem.org/Team:Goettingen/Notebook" class="sidebar-items sidebar-drop_items">Notebook</A><A href="https://2018.igem.org/Team:Goettingen/InterLab" class="sidebar-items sidebar-drop_items">InterLab</A><A href="https://2018.igem.org/Team:Goettingen/Model" class="sidebar-items sidebar-drop_items">Model</A><A href="https://2018.igem.org/Team:Goettingen/Results" class="sidebar-items sidebar-drop_items">Results</A><A href="https://2018.igem.org/Team:Goettingen/Demonstrate" class="sidebar-items sidebar-drop_items">Demonstrate</A><A href="https://2018.igem.org/Team:Goettingen/Improve" class="sidebar-items sidebar-drop_items">Improve</A><A href="https://2018.igem.org/Team:Goettingen/Attributions" class="sidebar-items sidebar-drop_items">Attributions</A></DIV><BUTTON class="dropdown_button sidebar-items">Parts</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Parts" class="sidebar-items sidebar-drop_items">Parts Overview</A><A href="https://2018.igem.org/Team:Goettingen/Basic_Part" class="sidebar-items sidebar-drop_items">Basic Parts</A><A href="https://2018.igem.org/Team:Goettingen/Composite_Part" class="sidebar-items sidebar-drop_items">Composite Parts</A><A href="https://2018.igem.org/Team:Goettingen/Part_Collection" class="sidebar-items sidebar-drop_items">Part Collection</A></DIV><A href="https://2018.igem.org/Team:Goettingen/Safety" class="sidebar-items">Safety</A><BUTTON class="dropdown_button sidebar-items">Human Practices</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Human_Practices" class="sidebar-items sidebar-drop_items">Human Practices</A><A href="https://2018.igem.org/Team:Goettingen/Human_Practices/Survey" class="sidebar-items sidebar-drop_items">Survey</A><A href="https://2018.igem.org/Team:Goettingen/Public_Engagement" class="sidebar-items sidebar-drop_items">Education &amp; Engagement</A></DIV><BUTTON class="dropdown_button sidebar-items">Awards</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Applied_Design" class="sidebar-items sidebar-drop_items">Applied Design</A><A href="https://2018.igem.org/Team:Goettingen/Hardware" class="sidebar-items sidebar-drop_items">Hardware</A><A href="https://2018.igem.org/Team:Goettingen/Measurement" class="sidebar-items sidebar-drop_items">Measurement</A><A href="https://2018.igem.org/Team:Goettingen/Model" class="sidebar-items sidebar-drop_items">Model</A></DIV><A href="https://igem.org/2018_Judging_Form?id=2586" class="sidebar-items">Judging Form</A></DIV><DIV id="main-content"><BUTTON id="show_nav" class="control-button">≡</BUTTON></DIV></DIV><DIV id="footer"><DIV class="l-two-col"><DIV class="contact"><SPAN class="footer_heading">Contact:</SPAN><TABLE><TBODY><TR><TH>E-mail:</TH><TD><A href="mailto:igem2018@gwdg.de">igem2018@gwdg.de</A></TD></TR><TR><TH>Adress:</TH><TD>c/o iGEM 2018</TD></TR><TR><TD>Department of General Microbiology</TD></TR><TR><TD>Grisebachstraße 8</TD></TR><TR><TD>37077 Göttingen</TD></TR><TR><TD>Germany</TD></TR></TBODY></TABLE></DIV><DIV class="shout_out"><P>We are especially grateful to our numerous sponsors on <A href="https://de.gofundme.com/glyphosate-on-my-plate">gofundme,</A> who supported our project and made it possible. Every public donor, who helped us to achieve our goals got a special place on our bacterial wall of fame and a handdrawn microbe to show our appreciation. A link to that wall can be found here:</P><P class="shout_out_wall_of"><A href="https://2018.igem.org/Team:Goettingen/WallOfFame">Bacterial Wall of Fame</A></P></DIV></DIV></DIV></DIV><DIV class="site-content"><DIV class="article"><DIV class="article_head-line"><H2>Modelling of glyphosate-dependent growth inhibition</H2></DIV><H3><I>B. subtilis</I> as a glyphosate detector</H3><P>As described in the <A href="https://2018.igem.org/Team:Goettingen/Background">Background</A> section, glyphosate specifically inhibits enolpyruvyl-shikimate-3-phosphate (EPSP) synthase, an enzyme present in plants, fungi, bacteria and archaea. The EPSP synthase converts shikimate-3-phosphate (S3P) and phosphoenolpyruvate (PEP) to EPSP, an intermediate in the <I>shikimate pathway</I> and required for synthesis of aromatic amino acids. Interaction of glyphosate with EPSP leads to depletion of cellular levels of aromatic amino acids resulting in growth inhibition. We have observed that growth of <I>B. subtilis</I> is strongly inhibited by glyphosate (50% reduction of the growth rate at 1 mM glyphosate, see <A href="https://2018.igem.org/Team:Goettingen/Results">Results</A> section). To generate a glyphosate detection system, we wanted to make use of the glyphosate sensitivity of <I>B. subtilis</I>. However, the wild type allows to detect glyphosate only in the range between 0 mM and 2 mM. Fortunately, we have discovered the first glyphosate transporters of a living organism (see <A href="https://2018.igem.org/Team:Goettingen/Results" target="_blank">Results</A> section). So we could make use of the <I>B. subtilis</I> double mutant strain (BP235, trpC2 gltT::spc gltP::cat) lacking both glyphosate transporters. The growth rate of this strain is reduced by 50% at a glyphosate concentration of about 8 mM! Therefore, the transporter mutant allows us develop a system with an increased detection range between 8 mM and 30 mM. To employ <I>B. subtilis</I> as a detector for glyphosate, we had to quantify the glyphosate-dependent reduction of the growth rate of the different <I>B. subtilis</I> strains. The growth rate seems to be ideal for this application because it is easy to reproduce. The relation between the growth rate and the glyphosate concentration can be obtained by modeling our growth curves with the bacterial growth model.
  </P><DIV class="article_picture article_picture-right" style="width: 50%"><P>Figure 1. Schematic drawing of the growth curve of a cell culture. The growth can be divided in four phases, namely lag-, log-, stationary and deathphase.</P></DIV><H3>The bacterial growth model</H3><P>Under lab conditions, the right environmental parameters and nutrients provided, bacterial growth can be observed in form of growth curves, plotting the bacterial mass as a function of time. A convenient way to observe those growth curves is provided by measuring the <I>optical density</I> also referred to as the <I>turbidity</I>. During the growth of bacteria three distinct phases can be differentiated (Figure 1), all of which can be associated with physiological changes in the cell cultures:</P><OL><LI>During <STRONG>lag phase</STRONG> the bacteria are suggested to adapt to physiological changes in the medium. Adaption can be achieved by e.g. changes in the proteome of the bacteria in order to meet the needs for the new culture requirements. Bacterial growth is strongly reduced</LI><LI>After adaption bacterial growth enters the <STRONG>log phase</STRONG> alternatively named <STRONG>exponential phase</STRONG>. During this phase the bacteria show exponential growth, the most rapid growth possible under given conditions. The actual double-timing of a certain strain is determined by the growth rate, the increase of cell mass per time.</LI><LI>In the <STRONG>stationary phase</STRONG> no net growth can be observed. Nutrients and carbon sources are used up and increase in cell mass is no longer possible. Dead cells can lyse and provide another source of nutrients, explaining a small increase in cell mass that sometimes still occurs.</LI></OL><P>To be scientifically correct a fourth phase can be observed in the bacterial growth curve namely the <STRONG>death phase</STRONG>. As we measured our bacterial mass in terms of <I>optical density</I> this phase couldn't be observed in our experiments and has no relevance for our modeling approaches.</P><H3>Correlation between growth rate and glyphosate concentration</H3><H4>Obtaining the growth rate μ</H4><P>During their growth bacteria reach a phase of exponential growth as described above. In this phase every bacteria in the culture divides at the fastest rate possible, therefore the growth of the colony is directly dependend on the amount of cells already grown. This is also true for the <I>optical density</I> as it's proportional to cell mass. To put this growth into a context that can be calculated, a constant of proportionality is needed. This constant can be described as the growth rate of the colony, it is specific to strains and also dependend on environmental influences like glyphosate. These dependencies can be described by the following formula:</P><P>This short differential equation describes the growth behaviour of the bacteria during the <STRONG>log phase</STRONG>. The equation can be solved by first separating the derivates:</P><P>Next the equation has to be integrated at both sides yielding to the following form:</P><P>Inserting the initial condition t=0 and OD<SUB>t=0</SUB> results in:</P><P>From this solution a very easy and convenient way to obtain the growth rate for every growth curve arrives. As it is probably known by everyone working in the field of microbiology plotting the logarithm of cell mass (or OD) against time leads to a linear section in the affiliated graph. This linear section is exactly what is described by the last formula, the logarithm of the <STRONG>exponential phase</STRONG> of bacterial growth with the growth rate as its slope. To reproducibly calculate the slope/growth rate we wrote a python script, that enables us to select the linear section of the graph by hand. After that a linear fit is calculated and the resulting growth rate is safed. In Figure 2 usage of the program is shown, for further experiments we used two mutant strains expressing fluorescent proteins, namely iGEM23 (<I>trpC2 amyE::(gfp-lacZ aphA3)</I>), resembling the wild type strain and iGEM36 (<I>trpC2 amyE::(mOrange-lacZ aphA3)</I>Δ<I>gltT::spc</I>Δ<I>gltP::cat</I>), resembling the transporter double mutant. Why the fluorescent labeling is important for our experiments is laid out in detail later in this article.</P><DIV class="article_multpicture"><DIV class="article_multpicture-paragraph"><P style="text-align: center">iGEM23</P><P style="text-align: center">iGEM36</P></DIV><P>Figure 2. Usage of the python script to obtain the growth rate from the linear section of growth curves.</P></DIV><H4>Describing the dependency of growth rate from glyphosate concentration</H4><P>After assessing the growth rates by doing fits of the growth curves, we plotted them against the glyphosate concentration in the environment of every growth experiment respectively. The graphs can be observed in Figure 3 and 4:</P><DIV class="article_picture"><P>Figure 3. Dependency of growth rate to glyphosate concentration in iGEM23.</P></DIV><DIV class="article_picture"><P>Figure 4. Dependency of growth rate to glyphosate concentration in iGEM36.</P></DIV><P>As stated in the <A href="https://2018.igem.org/Team:Goettingen/Results">Results</A> section of our project the growth rate is strongly dependend on glyphosate concentration. While low concentrations of glyphosate (relative to the concentration of 50% growth reduction for every strain) do not strongly affect the growth behavior , increasing the concentration yields to very strong reduction. Moreover, higher doses of glyphosate do not change the growth rate very strongly anymore. The graphs strongly resemble the well-known Hill equation (see below) suggesting cooperativity in either the transport of glyphosate or interaction with the EPSP synthase.</P><P>We fitted the Hill equation to the data using a python script and the <A href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.leastsq.html">leastsq method of the scipy library</A>, the fitted data are shown in Figure 5 and 6.</P><DIV class="article_picture"><P>Figure 5. Least square fit of the dependency between growth rate and glyphosate concentration for iGEM23 using the hill equation. In the right corner, the resulting function with determined parameters is shown.</P></DIV><DIV class="article_picture"><P>Figure 6. Least square fit of the dependency between growth rate and glyphosate concentration for iGEM36 using the hill equation. In the right corner, the resulting function with determined parameters is shown.</P></DIV><P>The hill equation can be transposed to show growth rate as dependency for glyphosate concentration:</P><P>With this transposition it is possible to determine the glyphosate concentration by evaluating the growth behaviour of the strains. For every strain these modelling parameters are different, for iGEM23 and the iGEM36 we already determined the fitting parameters (the values for the parameters can be found in the two diagrams above), enabling us to detect glyphosate from 0.5 up to 50 mM.</P><P>To test our glyphosate detection method we carried out several growth experiments with known glyphosate concentrations. We calculated the growth rates using the python script described above and used the transposed hill curve to determine, under which glyphosate concentration the strains might have grown. For example we carried out the growth experiments with BP235 under a concentration of 20 mM glyphosate and inserted the growth rate into the hill equation:</P><P>Giving this example it can be seen, that also for growth experiments on different days and slighlty different conditions (as it is not possible to perfectly control all environmental factors) the equation still predicts the glyphosate concentration with a pretty high accuracy, in this case the error is &lt; 2.3 %.</P><P>But there is one catch, that made us try to further extend our consideration. We can't be exactly sure, that there are no other environmental conditions that influence our growth experiments except for glyphosate. At last correlation must not mean causality. To ensure that glyphosate is the dominating factor in our experiments, we set out a competition assay between the iGEM23 and iGEM36, gradually increasing the concentration of the herbicide in the media.</P><H3>Modeling the competition for different glyphosate concentrations</H3><H4>Modelling the <I>optical density</I></H4><P>During a competition assay both strains should be influenced by glyphosate in the same way they would while growing isolated from each other. The growth rate and it's reduction by glyphosate can be assumed to follow the hill equations we already determined for both strains. The difference to a normal growing experiment can be found in the competition between both strains for nutrients. Like in an isolated growing experiment, the growth curve can be plotted as logistic growth, but the limiting threshhold for the growth is not only influenced by the cell mass of one single strain but by both strains together. On top of this theory two dependencies can be written out in the following form:</P><DIV class="article_direct-footnote"><P>Where [gly] describes the concentration of glyphosate, μ the growth rate of the strains, S the saturation of <I>optical density</I>, WT the wild type strain (iGEM23) and DM the double mutant strain (iGEM36)</P></DIV><P>This set of differential equations can't be solved elegantly in an analytical way, therefore we used numerical integration to model the <I>optical density</I> (representing the cell mass) of both of our strains in competition. This is quite conveniently done by employing the <A href="https://reference.wolfram.com/language/ref/NDSolve.html">NDSolve</A> method of the computing system mathematica.</P><H4>Results of modelling</H4><P>Now we had all the necessary information to model the cell mass of our both strains in competition for different glyphosate concentrations over the course of several hours. The outcome of this simulation is best depicted as ratio of the <I>optical densitiy</I> of iGEM23 and iGEM36 representing the ratio of cell mass between the two (Figure 7):</P><DIV class="article_picture"><P>Figure 7. Ratio of OD<SUB>iGEM23</SUB>(WT)/OD<SUB>iGEM36</SUB>(mutant) against glyphosate concentration and time as it can be modeled from our previous knowledge of the dependencies between glyphosate and growth rate.</P></DIV><P>At low concentrations of glyphosate, the ratio BP235 and iGEM23 is near 1, both strains grow at the same speed and have similar cell mass. But increasing the dose of glyphosate by only as little as 2 mM, the ratio changes in favor of the mutant strain. As it can be seen in the hill plot (Figures 5 and 6), iGEM36 shows a much lower reduction in growth rate, therefore it should grow much faster than the wild type shifting the ratio in cell mass.</P><P>As laid out before, we are only able to measure the OD of both strains together. To evaluate the quality of our modelling we could therefore only compare the sum of the OD calculated with our modelled to the experimental data. The modelled graph and the experimental data are depicted in the following 3D graphs (Figure 8 and 9):</P><DIV class="article_picture"><P>Figure 8. Experimentally measured data of the OD during the competition assays between iGEM23 and iGEM36 for different glyphosate concentrations up to 2 mM</P></DIV><DIV class="article_picture"><P>Figure 9. Calculated OD during the competition assays between iGEM23 and iGEM36 for different glyphosate concentrations up to 2 </P></DIV><P>Comparing Figure 8 and 9 it can be observed, that the model is predicting the sum of ODs right in most areas. While the experimental data are of course a little rougher both model and experimental data show the same sigmoidal increase of OD. With the model the reduced growth of OD for high glyphosate concentrations is predicted right. While the experimental data show a decrease in maximum OD for high glyphosate concentrations, the model doesn't express a dependency of maximum OD and glyphosate concentration. This is expected by the nature of the model, as it doesn't take into account the death of cells, that also could be influenced by glyphosate concentration. Maximum OD in the model is 0.8, while in the competition assay only a maximum OD of around 0.5 is reached, this can be easily adapted by changing the parameter S in the modelled equations (see above).</P><P>Overall our modeling showed, that based on our assumptions of glyphosate as the dominating factor in our experiments, the ratio between the iGEM23 and iGEM36 should pretty drastically change in favor of the double mutant by increasing the glyphosate concentration in the media.</P><H4>Evaluating growth behaviour of the strains by means of fluorescence</H4><P>As the OD of both strains can't be measured separated from each other, no comparison between the strains by means of OD was possible. Therfore, we created fluorescent mutant strains expressing the fluorophores <I>green fluorescent protein</I> (GFP, iGEM23 - wild type) and mOrange (iGEM36 - double mutant) to monitor the fluorescence of both strains over the course of the experiment. We calculated the increase in fluorescence relative to the fluorescence measured at t=0 for different glyphosate concentrations. Doing this we were able to evaluate if our proposed trend of the mutant strain being the stronger competitor at higher glyphosate concentrations is right. Although it has to be mentioned that this test is more of qualitative nature, as modelling the fluorescence signals of the strains needs further research: To properly model the fluorescence trends of the strains a link between optical density and fluorescence has to be found. To achieve this expression, maturation and degradation rates of the proteins are needed as the model of Leveau and Lindow suggests (1).  The calculated relative fluorescence and a ratio between the strands for 0 mM and 2 mM glyphosate is shown in the following Figure (Figure 10):</P><DIV class="article_picture" style="width: 95%"><P>Figure 10: N-fold increases in fluorescence relative to fluorescence at t=0 (OD = 0.1) for both strains in competition. Upper left: n-fold increase in fluorescence of iGEM23 (wild type). Upper right: n-fold increase in fluorescence of the iGEM36 (double mutant). Bottom: Ratio of relative fluorescence increases.</P></DIV><P>Without the influence of glyphosate the fluorescence of both strains increased nearly equally (Figure 10, upper left and right), at the 10 h mark a ratio of 0.8 between the strains can be found. Adding 2 mM glyphosate to the system strongly shifts the ratio of fluorescence. The iGEM36 is strongly expressing fluorescent proteins, nearly doubling the amount of relative fluorescence increase compared to the competition experiment without glyphosate (Figure 10, upper right). In contrast iGEM23 shows no significant increase in fluorescence over the course of the experiment, at maximum a 2-fold increase can be observed (Figure 10, upper left). The ratio of fluorescence decreases to less than 0.1 after ten hours of competition, suggesting a much stronger growing double mutant.</P><P>Even though these results can't be interpreted quantitatively they still show, that our assumptions might be right. While without glyphosate both strains show a rather equal increase in fluorescence intensity, adding glyphosate shifts the ratio strongly in favour of the double mutant strain. As higher fluorescence intensity correlates with a stronger expression of fluorescent proteins, it can be assumed, that the double mutant clearly dominates in growth rate in comparison to the wild type, as our modelled already suggested.</P><H4>Evaluating cell growth with a β-galactosidase assay</H4><P>Another method for distinction of two strains in one culture was an enzymatic solution. The sensitive strain expresses constitutively a β-galactosidase which leads to hydrolyzation of a chromogenic substrate. This assay is performed with the substrate o-nitrophenyll-β-D-galactopyranoside (ONPG). The hydrolyzation of ONPG leads to the formation of o-nitrophenol, which absorbs light at λ=420 nm. The amount of o-nitrophenol that is formed within a certain time is directly proportional to the activity of a promoter. Since the promoter is active in any cell from the sensitive strain, it is proportional to the growth. Measurements of the enzyme activity can be seen in Figure 11:</P><DIV class="article_picture"><P><STRONG>Figure 11.</STRONG> Cell mixture (1:1) of strains BP193 (wild type) and BP235 (double mutant) were incubated with increasing glyphosate concentrations (0 - 5 mM). Strain BP193 harbors the <EM>lacZ</EM> gene behind an artificial promoter, but does not tolerate glyphosate in contrast to strain BP235 (Δ<I>gltT</I> and Δ<I>gltP</I>). ß-galactosidase activity can be measured at a wavelength of 420 nm using the substrate o-nitrophenyl-ß-D-galactopyranoside. Activity decreases with increasing glyphosate concentration. BP235 does not contain the <EM>lacZ</EM> gene, hence no activity can be observed.</P></DIV><P>High concentrations of glyphosate lead to a strongly reduced enzyme activity, suggesting slower or stagnated growth of the wild type strain. These results complement what our model of the cell mass ratio suggested: For glyphosate concentrations higher than 2 mM the wild type strain is strongly inhibited in growth, reassuring that glyphosate is the dominant factor in our competition and single experiments.</P><P>This assay can also be performed with 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside (X-gal) as chromogenic substrate. The hydrolyzation leads to the formation of an insoluble indigo-dye that is located within the cells. With increasing glyphosate concentration, the intensity of the blue color <A href="https://2018.igem.org/Team:Goettingen/Results#r_Competition" target="_blank">decreases</A> and is therefore an indicator for the glyphosate concentration. This method is further prove that the sensitive wild type strain is strongly reduced in growth and therefore eliminated by the glyphosate-resistant double mutant.</P><H2>References</H2><OL><LI>Leveau &amp; Lindow (2001) J. Bacteriol. 183: 6752-6762.
  </LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>