Team:Goettingen/Design
⇧
Team Göttingen
iGEM 2018
Glyphosate on my plate?
Home News
Team
Team Members Collaborations
Project
Background Description Design Experiments Notebook InterLab Model Results Demonstrate Improve Attributions
Parts
Parts Overview Basic Parts Composite Parts Part Collection
Safety
Human Practices
Human Practices Survey Education & Engagement
Awards
Applied Design Hardware Measurement Model
Judging Form
≡
Contact:
E-mail:
igem2018@gwdg.de
Adress:
c/o iGEM 2018
Department of General Microbiology
Grisebachstraße 8
37077 Göttingen
Germany
We are especially grateful to our numerous sponsors on gofundme,  who supported our project and made it possible.
Every public donor, who helped us to achieve our goals got a special place on our bacterial wall of fame and a handdrawn microbe to show our appreciation. A link to that wall can be found here:
Bacterial Wall of Fame
Design
Glyphosate detection system
Why Bacillus subtilis?
The people that are interested in our project may ask the question why we are using the Gram-positive bacterium Bacillus subtilis as a working horse. The answer is pretty simple because there are several reasons why B. subtilis is perfectly suited for our project. The spore-forming bacterium B. subtilis, which has been first described by Christian G. Ehrenberg in 1835, is commonly found in the upper layers of the soil, and evidence exists that the bacterium is a normal gut commensal in humans (Figure 1) (1). Since its discovery, B. subtilis has been intensively studied and there are currently > 150 labs doing mainly basic research with B. subtilis in order to understand the biology of the organism. The sequence B. subtilis genome has been described in 1997 and refined in 2009 (2,3). A plethora of tools for genetic engineering of B. subtilis and for biochemical analyses of its proteins have been developed. Moreover, due to its natural competence it is extremely easy to genetically manipulate B. subtilis. The intensive research that has been conducted with B. subtilis has constantly increased our level of knowledge about the bacterium. All the information about the genes, proteins, metabolic and regulatory pathways of B. subtilis that is currently available is stored in the SubtiWiki database, which has been developed in the Department of General Microbiology at the University of Göttingen . SubtiWiki has become one of the most complete resources of knowledge on a living organism (4). Since the 1960s B. subtilis has had a history as a test species in space microbiology (5). Its endospores can survive up to 6 years in space if coated by dust particles protecting it from solar UV rays. In the past years, B. subtilis has also been subjected to large-scale genome minimization in order to create a so-called Minibacillus (6). The Minibacillus project addresses two questions: (i) the identification of the minimal gene set that is needed to sustain life, and (ii) the rational design and construction of tightly controllable cell factories for sustainable development (6). The final goal of the project is to obtain a Minibacillus strain that is equipped with a minimal core genome (7). A minimal core genome only consisting of essential genes will help to understand how a specific gene contributes to a living cell! Nontoxigenic and nonpathogenic strains of B. subtilis are widely available and have been safely used in a variety of food applications in industry. B. subtilis spores are used to clean toilets of  the German Autobahn ( http://www.noblebio.com )! Moreover, the bacterium has been engineered for the production of fine chemicals and enzymes such as vitamins and proteases, respectively (8). Given the fact that B. subtilis has been intensively studied over the past 183 years, the bacterium belongs to the best-understood organisms that are available on earth! Some people even believe that B. subtilis is endowed with the highest bacterial IQ (9)! Finally, - and this fact is very important for our project - as a soil bacterium, representatives of B. subtilis are increasingly used in agriculture to promote plant growth and to protect against plant pathogens (10-12).
Figure 1. B. subtilis as a model system to study the interaction between glyphosate and a representative of the soil rhizosphere.
Genomic adaptation of Bacillus subtilis to glyphosate
Glyphosate (N-(phosphonomethyl)glycine) specifically inhibits the 5-enolpyruvyl-shikimate-3-phosphate (EPSP) synthase in plants, fungi, bacteria and archaea (13-16). The EPSP synthase converts phospoenolpyruvate (PEP) and shikimic acid-3-phosphate (S3P) into EPSP, which serves as a precursor for the de novo synthesis of aromatic amino acids tryptophan, tyrosine and phenylalanine (17). Therefore, inhibition of the EPSP synthase by glyphosate results in the depletion of the cellular levels of aromatic amino acids unless they are provided by the environment (13,18-20). Few years after the discovery of the mode of action of glyphosate it has been shown that the aroA gene from enteric bacteria such as Salmonella typhimurium can accumulate mutations, which renders the encoded EPSP synthase insensitive to glyphosate (14). In the meantime, several glyphosate-insensitive EPSP synthase mutant variants have been generated by either random or directed mutagenesis or they have been isolated from glyphosate-resistant organisms (21-35). The structural characterization of the EPSP synthases from Agrobacterium sp. strain CP4 and Escherichia coli revealed that glyphosate targets the PEP binding site and acts as a competitive inhibitor of the enzyme (36,37). EPSP synthases that are insensitive to glyphosate have been used to genetically engineer glyphosate-tolerant crops (38-44).
While the interaction between glyphosate and bacteria like E. coli, Enterobacter and Burkholderia thailandensis has been studied at the transcriptome level (45-47), not much is known about the interaction between weedkiller and the Gram-positive soil bacterium B. subtilis (18). Previous studies revealed that growth of B. subtilis is inhibited by glyphosate due to the inactivation of the EPSP synthase (18,20,48). Moreover, in order to enhance glyphosate resistance in crops, the B. subtilis glycine oxidase has been engineered to improve the degradation of glyphosate to aminomethyl-phosphonic acid and glyoxlate (49). Finally, experimental evidence suggests that the B. subtilis strain Bs-15, which was isolated from a pepper plant is capable of degrading glyphosate (50). However, in this B. subtilis isolate, the glyphosate degradation pathway remains to be identified. The first step in developing a powerful glyphosate detection system using B. subtilis is to assess the potential of the bacteria to adapt to the herbicide at the genome level! For this purpose, we will perform adaptive laboratory experiments (ALE) (51) to assess the potential of B. subtilis to adapt to toxic levels of glyphosate at the genome level. The strains generated by ALE will be analyzed by whole-genome sequencing and reversely engineered in the laboratory to reconstitute the genotypes and phenotypes (Figure 2).
Figure 2. Growth of B. subtilis in the presence of glyphosate may lead to the accumulation of mutations that confer resistance to the weedkiller. The bacteria may also adapt to the weedkiller by differential regulation of gene expression.
Identification of Bacillus subtilis promoters responding to glyphosate
Promoters that are regulated by glyphosate in B. subtilis, will be identified by transcriptome analyses that will be done with the help of Dr. Anja Poehlein and Prof. Dr. Rolf Daniel from the Göttingen Genomics Laboratory . For this purpose, we will cultivate B. subtilis in the absence and in the presence of sublethal amounts of the glyphosate and analyze the transcriptome by performing RNAseq. Genes that are strongly up- and down-regulated are of special interest because the promoters of these genes might be useful for the development of a glyphosate detection system (see Figure 2).
A reporter system for the detection of glyphosate
The reporter system to detect glyphosate will be constructed based on the results of the transcriptome analyses using B. subtilis. The principle of a glyphosate detection system is shown in Figure 3. To allow the detection of glyphosate, we want to fuse a glyphosate-dependent promoter to a gene encoding a transcriptional repressor. The repressor gene should be transcribed in the absence of the weedkiller because the glyphosate-dependent promoter is active. Furthermore, the produced repressor prevents transcription of a reporter gene encoding a fluorescent protein. Therefore, lack fluorescence indicates the absence of glyphosate. By contrast, during growth of the bacteria in the presence of glyphosate, the repressor gene is inactive and the flourescent protein is produced. Therefore, the fluorescence level indicates the presence of glyphosate. Once characterized, the reporter system can also be introduced in B. subtilis that tolerate high amounts of glyphosate (see above).
Figure 3. In the absence of glyphosate, the promoter (P) driving the transcription of the repressor gene is active. The encoded repressor (R) binds to the operator (O) and prevents transcription of the reporter gene. In the presence of glyphosate, the repressor (R) is not synthesized and the RNA polymerase-sigma factor complex (RNAP) can transcribe the reporter gene. The bacteria become fluorescent.
Glyphosate detection by monitoring intraspecies competition
We also want to develop a reporter system for the detection of glyphosate by monitoring intraspecies competition of fluorescently labelled B. subtilis strains that tolerate different amounts of the weedkiller (52). The isolated B. subtilis mutants that grow in the presence of different amounts of glyphosate are perfectly suited to develop a reporter system that is based on intraspecies competition. For labelling of our strains, we will use the genes encoding the blue fluorescent protein mTagBFP, the green fluorescent protein GFP, and the orange fluorescent protein mOrange, which we obtained from the iGEM DNA distribution kit. The detection system is based on an intraspecies competition assay using blue, green and orange B. subtilis strains that are mixed in an 1:1:1 ratio (see Figure 4). The blue, green and orange B. subtilis strains tolerate different amounts of glyphosate. Therefore, in the absence of glyphosate, we expect that all strains survive in a liquid culture. By contrast, during growth in the presence of glyphosate only those cells survive that tolerate high amounts of the weedkiller. Thus, the ratio of the fluorescently labelled strains after cultivation tells us about the amount of glyphosate that was present in the culture. The intraspecies competition assay may also be conducted using B. subtilis strains that are labelled with other reporter genes (e.g. lacZ).
Figure 4. The glyphosate detection system the is based on intraspecies competition. A mixture of spores of fluorescently labelled B. subtilis strains that tolerate different amounts of glyphosate are co-cultivated with medium supplemented with different amounts of the weedkiller.  In the absence of glyphosate, all strains grow equally. At low levels of glyphosate, all cell types will survive but more of the cells that are resistant to the weedkiller are present after co-cultivation. In the presence of high amounts of glyphosate, only the resistant cells will survive and outcompete the non-resistant cells.
Engineering Bacillus subtilis to disarm glyphosate
Many years ago it has been observed that some bacteria, which were isolated from glyphosate contaminated soil, are capable of degrading the weedkiller (53). It has also been found that glyphosate can be inactivated by covalent modification such as phosphorylation (54,55) and acetylation (56). For instance, the glyphosate N-acetyltransferase GAT from the Gram-positive soil bacterium Bacillus licheniformis transfers the acetyl group from acetyl-CoA onto the amino group of the weedkiller (Figure 5) (56). The N-acetylated form of glyphosate does not inhibit the EPSP synthase (see above). The GAT was also subjected to directed evolution for creating an enzyme with higher efficiency and increased specificity for the herbicide (57,58). Therefore, bacteria like B. subtilis expressing the GAT should tolerate high amounts of glyphosate even though it is taken up from the environment. Furthermore, it has been observed that bacteria can become resistant to glyphosate by the accumulation of mutations in the EPSP synthase gene (see above). For instance, it has been reported that mutations in the aroA gene of Salmonella typhimurium render the encoded EPSP synthase insensitive to glyphosate (14). It has also been shown that the EPSP synthase from E. coli can become insensitive to glyphosate. For instance, a mutant variant of the E. coli EPSP synthase in which the proline at position 101 is replaced by leucine decreased the inhibitor potency of glyphosate (18). Although the proline residue at position 101 is not directly involved in glyphosate binding, any mutation at this site seem to result in a structural change in the weedkiller-binding site, thereby rendering the EPSP synthase less sensitive to glyphosate (30). The replacements of glycine by alanine and alanine by threonine at the positions 96 and 183, respectively, in the EPSP synthase also confer glyphosate insensitivity to the enzyme (59). The individual overproduction of the Gat and the AroA(G96A A183T) enzymes may significantly increase glyphosate tolerance of B. subtilis. Moreover, co-overproduction of the enzymes may synergistically increase glyphosate tolerance. In this part of the project, we engineer B. subtilis to improve our reporter systems to allow the detection of high amounts of glyphosate. Currently, we are working on the integration of the gat and the aroA(G96A A183T) genes into the genome of B. subtilis.
Figure 5. The glyphosate N-acetyltransferase GAT from B. licheniformis (PDBid: 2BSW )     converts Acetyl-CoA and the toxic herbicide to N-acetylglyphosate, which is not toxic (56,60). The AroA Pro101Leu mutant variant from E. coli (PDBid: 2QFU ) is less sensitive to inhibition by glyphosate (30).
References
Ehrenberg (1835) Akademie der Wissenschaften zu Berlin. pp. 145-336.
Kunst et al. (1997) Nature 390: 249-256.
Barbe et al. (2009) Microbiology 155: 1758-1775.
Zhu & Stülke (2018) Nucleic Acids Res. 46: D743-748.
Horneck et al. (2010) Microbiol. Mol. Biol. Rev. 74: 121-156.
Juhas et al. (2014) Microbiology 160: 2341-2351.
Reuß et al. (2016)  Microbiol. Mol. Biol. Rev. 80: 955-987.
Hohmann et al. (2016) Wiley- VCH Verlag GmbH & Co. KGaA; 2016:221-297.
Pinto & Mascher (2016) Curr. Genet. 62: 487-498.
Qiao et al. (2017) BMC Microbio. 17: 131.
Xie et al. (2014) Mol. Plant Microbe Interact. 27: 655-663.
Tahier et al. (2017) Front. Microbiol. 8:171.
Amrhein et al. (1983) FEBS Lett. 157: 191-196.
Comai et al. (1983) Science 221: 370-371.
Schulz et al. (1984) Arch. Microbiol. 137: 121-123
Steinrücken & Amrhein (1980) Biochem. Biophys. Res. Commun. 94: 1207-1212.
Herrmann & Weaver (1999) Annu. Rev. Plant. Physiol. Plant. Mol. Biol. 50: 473-503.
Fischer et al. (1986) J. Bacteriol. 168: 1147-1154
Gresshoff  (1979) Aust. J. Plant. Physiol. 6: 177-185.
Majumder et al. (1995) Eur. J. Biochem. 229: 99-106.
Stalker et al. (1985) J. Biol. Chem. 260: 4724-4728.
Sost & Amrhein (1990) Arch. Biochem. Biophys. 282: 433-436.
Padgette et al. (1991) J. Biol. Chem.  266: 22364-22369.
He et al. (2001) Biochim. Biophys. Acta 1568: 1-6.
He et al. (2003) Biosci. Biotechnol. Biochem. 67: 1405-1409.
Bearson et al. (2002) Plant Physiol. 129: 1265-1275.
Eschenburg et al. (2002) Planta 216: 129-135.
Sun et al. (2005) Appl. Envrion. Microbiol. 71: 4771-4776.
Zhou et al. (2006) Plant Physiol. 140: 184-195.
Healy-Fried et al. (2007) J. Biol. Chem. 282: 32949-32955.
Vande Berg et al. (2008) Pest. Manag. Sci. 64: 340-345.
Funke et al. (2009) J. Biol. Chem. 284: 9854-9860.
Tian et al. (2010) Appl. Environ. Microbiol. 76: 6001-6005.
Pollegioni et al. (2011) FEBS J. 278: 2753-2766.
Chekan et al. (2016) MedChemComm. 7: 28-36.
Schönbrunn et al. (2001) Proc. Natl. Acad. Sci. USA 98: 1376-1380.
Funke et al. (2006) Proc. Natl. Acad. Sci. USA. 103: 13010-13015.
Della-Cioppa et al. (1987) Nat. Biotechnol. 5: 579-584.
Padgette et al. (1995) Crop Science 35: 1451-1461.
Barry et al. (1997) U.S. Patent 5633435.
Cao et al. (2012) PLoS One. 7: e38718.
Guo et al. (2015) Front. Plant. Sci. 15: 847.
Zhang et al. (2017) J. Integrative Agriculture 16: 551-558.
Fartyal et al. (2018) Front. Plant. Sci. 13: 144.
Kang et al. (2011) Genome Res. 21: 925-935.
Fei et al. (2013) FEMS Microbiol. Lett. 349: 135-143.
Lu et al. (2013) Mol. Biosyst. 9: 522-530.
Selvapandiyan et al. (1995) FEBS Lett. 374: 253-256.
Pedotti et al. (2009) J. Biol. Chem. 284: 36415-36423.
Yu et al. (2015) Genet. Mol. Res. 14: 14717-14730.
Dragosits & Mattanovich (2013) Microb. Cell Fact. 12: 64.
Stannek et al. (2014) J. Vis. Exp. 18: e51196.
Hove-Jensen et al. (2014) Microbiol. Mol. Biol. Rev. 78: 176-197.
Rao et al. (1983) Antimicrob. Agents Chemother. 24: 689-695.
Penaloza-Vazquez et al. (1995) Appl. Environ. Microbiol. 61: 538-543.
Castle et al. (2004) Science 304: 1151-1154.
Siehl et al. (2005) Pest. Manag. Sci. 61: 235-240.
Siehl et al. (2007) J. Biol. Chem. 282: 11446-11455.
Kahrizi et al. (2007) Plant. Cell Rep. 26: 95-104.
Keenan et al. (2005) Proc. Natl. Acad. Sci. USA 26: 95-104.
