Team:Goettingen/Notebook/July
Contents
1 Again transformations
2 Evaluation of transformations
3 Plasmid preparation and preparation of "modified disc assay"
4
4.1 Modified disc assay
4.2 Preparation of β-galactosidase assay
5 β-galactosidase assay
6
7 β–galactosidase assay
7.1 β–galactosidase assay
7.1.1 Strain iGEM3
7.1.2 Strain GP342
8 Again β–galactosidase assay
9 β–galactosidase assay
10 Overexpression of proteins in E. coli
11 Preparation of biobricks
11.1 Combined-chain reaction (CCR) and multiple-mutation reaction (MMR)
11.1.1 The Material
11.1.2 The Procedure
11.1.3 Reaction mixture
12 Cloning of biobricks 2, 9, 10, 12 and overexpression of AroA and AroE
13 Plasmid preparation
14 Amplification of the biobricks 2, 9, and 10
15 New β–galactosidase assay
16 Sequencing of biobricks
Again transformations
02.07.18
Transformation of pIGEM2, pIGEM2, pIGEM3, and pIGEM5 into E. coli DH5α to geht backup plasmids.
Transformation of SP1 with pIGEM5.
Evaluation of transformations
03.07.18
DH5α transformants were separated. SP1-pIGEM2/3 transformants were also separated. The strain SP1 was transformed with pBQ200 to create an isogenic control. For this transformation, we used 2 µL plasmid. The following table shows the results from the transformations.
Strain
Cfu (normal/rest)
DH5α_pIGEM1
137/960
DH5α_pIGEM2
1072/1450
DH5α_pIGEM3
36/204
DH5α_pIGEM5
616/1096
SP1_pIGEM3
43/94
SP1_pIGEM5
–/–
Plasmid preparation and preparation of "modified disc assay"
04.07.18
The plasmid pIGEM5 was isolated from E. coli with the normal protocol. Concentration: 141.4 ng/µL.
A PCR was performed with the primers ML107 and lacZ-rev on the plasmid to check if there is an insert. This PCR was succesful.
The transformations of SP1 with pBQ200 were separated.
CS-Glu agar with and w/o tryptophan were prepared. In the middle of the plate was a circle of the agar extracted with an open falcon tube. The precultures for this experiment (SP1_pIGEM3) were incubated over night in LB medium at 28°C and agitation.
Furthermore was the β-galactosidase assay prepared for the next day with the strains:
Strain
Origin
Genotype
GP342
168 derivative
trpC2 amyE::(PgltA-lacZ aphaA3)
GP650
GP342 derivative
trpC2 amyE::(PgltA-lacZ aphaA3) gltC::Tn10 spc
iGEM20
cDNA GP342→SP1
amyE::(PgltA-lacZ aphaA3)
iGEM21
cDNA GP650→SP1
amyE::(PgltA-lacZ aphaA3) gltC::Tn10 spc
05.07.18
Modified disc assay
Cells from the over night culture were propagated on CS-Glu agar containing X-Gal, kanamycin and NO tryptophan. Into the holes in the plates, we filled 400 µL CS-Glu medium and 400 µL CS-Glu medium with 50 mM glyphosate. The plates were incubated over night at 37°C.
The transformation of SP1 with pIGEM5 was repeated.
Preparation of β-galactosidase assay
The strains iGEM20 and SP1_pIGEM3 were used. Some colonies were picked from the plates and resuspended in 50 µL LB medium. 5 µL were transferred into 4 mL LB medium and the remaining 45 µL were also transferred into 4 mL LB medium. The cells were incubated over daytime and then was CS-Glu medium (5 mL) inoculated with the following pattern.
A
B
C
D
OD600
0.05
0.05
1/10 cell amount of A/B
0.1
Temperature
37°C
28°C
37°C
37°C
β-galactosidase assay
06.07.18
With the overnight cultures were 10 mL CS-Glu medium inoculated to OD600=0.1, one time with 0.75 mM glyphosate and ones without.
Strain
OD600
Time
SP1_pIGEM3 –GS
0.51
6 h
SP1_pIGEM3 +GS
0.496
6 h
iGEM20 +GS
0.235
6 h
The cells were harvested with the following protocol:
1.5 mL SP1_pIGEM3 with and w/o glyphosate were centrifuged for 10 min at 5,000 rpm and 4°C.
4.5 mL iGEM20 with glyphosate was centrifuged for 10 min at 5,000 rpm and 4°C.
09.07.18
Transformation of SP1 with pIGEM5 was performed like in protocol described.
Precultures of strain iGEM21 in 5 mL CS-Glu w/o tryptophan and strain iGEM3 in 5 mL CS-Glu w/o tryptophan were prepared and incubated over night at 37°C with agitation.
β–galactosidase assay
10.07.18
For the β-galactosidase assay, SP1_pIGEM3 and GP342 were transferred into 4 mL LB medium and incubated over night.
For the modified disc assay, 4 mL LB medium were inoculated with SP1_pIGEM3.
Primers for all biobricks were designed.
β–galactosidase assay
Strain iGEM3
GS concentration [mM]
OD600 after 5.5 h
0
0.592
0.5
0.592
1
0.512
1.5
0.366
2
0.286
Strain GP342
GS concentration [mM]
OD600 after 5.5 h
0
0.64
0.5
0.696
1
0.632
1.5
0.392
2
0.346
Plasmid preparation of pBQ200 and pAC7. Furthermore, a chromosomal DNA isolation was performed for Bacillus strains 168 and SP1.
Again β–galactosidase assay
11.07.18
Over night cultures were transferred into CS-Glu medium to an OD600=0.1. The medium contained 0, 0.5, 1, 1.5, 2, mM glyphosate. After 8 h incubation, the following OD600 values were measured.
OD600 of strain
0 mM GS
0.5 mM GS
1 mM GS
1.5 mM GS
2 mM GS
SP1_pIGEM3
0.584
0.584
0.512
0.366
0.286
GP342
0.64
0.696
0.632
0.392
0.346
The modified disc assay was repeated.
β–galactosidase assay
12.07.18
The β–galactosidase assay was performed as described previously. The results are shown in the table below.
Sample
Δt
A420
A595
MU/mg protein
pIGEM3 0 mM
31:30 min
0.075
0.132
36.07
pIGEM3 0.5 mM
32 min
0.071
0.153
29.46
pIGEM3 1 mM
31:30 min
0.047
0.094
31.74
pIGEM3 1.5 mM
31:30 min
0.025
0.049
32.39
pIGEM3 2 mM
31:30 min
0.014
0.027
32.92
GP342 0 mM
7:30 min
0.392
0.134
780
GP342 0.5 mM
7 min
0.316
0.172
524.9
GP342 1 mM
7 min
0.316
0.118
765.13
GP342 1.5 mM
16 min
0.198
0.066
375
GP342 2 mM
26:30 min
0.115
0.03
289.3
pIGEM3 0 mM
29 min
0.043
0.069
42,9
pIGEM3 0.75 mM
29 min
0.04
0.051
54
GP 342 0 mM
29 min
0.045
0.002
1551
Control
29 min
–
–
–
Overexpression of proteins in E. coli
13.07.18
Preculture over night at 37°C with agitation has to be prepared.
Inoculate 4x 0.5 L LB medium with ampicillin in 2 L shake flasks to an OD600=0.05.
At an OD600=0.6 add an inducer, like IPTG.
Incubate the cultures for 3 h at 37°C with agitation.
Harvest the cells with centrifugation for 10 min at 4°C and 5,000 rpm. Discard the supernatant.
Resuspend the pellet in 25 mL buffer W (cold) and transfer the samples into 50 mL falcon tubes.
Centrifuge the samples for 10 min at 4°C and 5,000 rpm. Discard the supernatant and store the pellet at -20°C.
Preparation of biobricks
17.07.18
The primers for the biobrick production arrived and PCRs and one CCR was performed to amplify the right fragments and to change one basepair in gltT to remove an active PstI restriction site! The fragments for the basic parts were digested with the restriction enzymes EcoRI and PstI. The recipes for the PCR was the same as previously shown.
Combined-chain reaction (CCR) and multiple-mutation reaction (MMR)
Several methods for PCR-based site-directed             mutagenesis have been developed. Among these,             the combined-chain reaction method proved to be             very rapid and reliable (Bi & Sambrook, 1997). The             principle of this method is the use of mutagenic             primers that hybridize more strongly to the             template than the external primers. The mutagenic             primers are phosphorylated at their 5’ ends, and             these are ligated to the 3’-OH groups of the             extended upstream primers by the action of a             thermostable DNA ligase. Moreover, the DNA polymerase             employed must not exhibit 5’→3’ exonuclease             activity, to prevent the degradation of the             extended primers. In our view, Pfu and Pwo             polymerases are both well suited. The original             protocol describes the introduction of two             mutations simultaneously. In a previous study, we             used a combined chain reaction to mutagenize four             distant bases in a DNA fragment in a one-step reaction.
The Material
10x CCR buffer
Phusion or Accuzyme™ polymerase
5 U/mL Ampligase
Primers for amplification
12.5 mM dNTPs
Mutagenesis primer
BSA (20 mg/mL)
The Procedure
Before you start your CCR or MMR you should amplify your gene of interest and clone it into a             plasmid of your choice. The introduction of mutations is far more efficient when including this             step. Do not forget to adjust the elongation time according to your polymerase and the length of your template!
Reaction mixture
Compound
Volume in [µL]
20 pmol fwd primer
2
20 pmol rev primer
2
mutagenesis primer
4
Plasmid DNA
1
10x CCR– or HF–buffer
5
Accuzyme™ or Phusion polymerase
1
Ampligase
3
dNTPs
2
BSA (20 mg/mL)
2
sterile H2O
30
Cloning of biobricks
2
,
9
,
10
,
12
and overexpression of AroA and AroE
19.07.18
The digest and ligation reactions were performed as described before. Furthermore, a purification of the overexpressed proteins AroE from B. subtilis and AroA from E. coli was performed as decribed in the following paragraph.
The AroA strain was cultured in 2 L medium, the AroE strain in 1 L as described before. Harvest the cells with centrifugation at 20,000 rpm for 20 min at 4°C.
Resuspend the pellet in 15 mL buffer W (ice-cold!).
Disrupt the cells with the French press. Repeat this step 2 times.
Centrifuge the samples 30 min at 17,500 rpm and 4°C.
Prepare the 10 mL column by BioRad with 1 mL Strep-Tactin Sepharose which equals a column volume of 500 µL.
Load the supernatant from the centrifugation on the column. Save the flowthrough and load it again on the column.
Wash the column with buffer W: 1. run: 5 mL. 2. run: 1.25 mL.
Elute the protein with buffer E. Load ½ of column volume on the column. Save the flowthrough.
Elute the protein with buffer E two times. Load the column volume on the column. Save the flowthrough.
Load the samples on a SDS protein gel. Mix 15 µL sample and 5 µL Laemmli buffer (5×) and boil it for 10 min at 95°C.
Perform a Bradford assay with the elution fractions.
The results of the protein purification are shown in the next figure.
<img src="
">
<img src="
">
The SDS page from the protein AroE from B. subtilis is shown in this figure.
The SDS page from the protein AroA from E. coli is shown in this figure.
The Bradford assay was performed as described in the Experiments section. For AroE, 4 µL and 8 µL were used and the concentration of AroE sample 1 was determined as 2.5 µg/µL and in the second sample as 2.03 µg/µL. The concentration for AroA was determined using 10 µL and 14 µL as 0.15 µg/µL.
Plasmid preparation
20.07.18
The plasmids of biobricks
3
and
4
were preparated using the DNA, RNA and Protein purification kit by Macherey & Nagel.
Amplification of the biobricks
2
,
9
, and
10
26.07.18
The competent E. coli strain DH5α was transformed with the following ligations:
pBS1C3 +
2
pBS1C3 +
9
pBS1C3 +
10
pBS1C3 re-ligation
New β–galactosidase assay
31.07.18
The following table shows the samples for the β–galactosidase assay.
Sample
Tryptophane
Glyphosate
1
100 µL
–
2
100 µL
1 mM
3
50 µL
–
4
50 µL
1 mM
5
25 µL
–
6
25 µL
1 mM
7
–
–
8
–
1 mM
After 5 h, the cells with the following OD600 were harvested (1.5 mL) with 13,000 rpm for 10 min at 4°C.
Sample
OD600 after 5 h
1
0.57
2
0.5
3
0.59
4
0.5
5
0.59
6
0.48
7
0.61
8
0.51
Sequencing of biobricks
31.07.18
Biobricks
2
,
3
,
4
,
6
,
9
,
10
, and
12
were sent for sequencing with primers iGEM2018_57 and iGEM2018_58.
