<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><BUTTON id="top-scroll" class="control-button">⇧</BUTTON></DIV><DIV id="header"><DIV id="header_team"><P>
        Team Göttingen iGEM 2018
      </P></DIV><DIV id="header_middle"><DIV id="header_middle-container"><P>Glyphosate <FONT size="4" color="white">on my</FONT> plate?</P></DIV></DIV></DIV><DIV id="main_wrapper"><DIV id="content_menu_wrapper"><DIV id="navigation" class="sidebar is-nav-hidden"><A href="https://2018.igem.org/Team:Goettingen" class="sidebar-items">Home</A><A href="https://2018.igem.org/Team:Goettingen/News" class="sidebar-items">News</A><BUTTON class="dropdown_button sidebar-items">Team</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Team" class="sidebar-items sidebar-drop_items">Team Members</A><A href="https://2018.igem.org/Team:Goettingen/Collaborations" class="sidebar-items sidebar-drop_items">Collaborations</A></DIV><BUTTON class="dropdown_button sidebar-items">Project</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Background" class="sidebar-items sidebar-drop_items">Background</A><A href="https://2018.igem.org/Team:Goettingen/Description" class="sidebar-items sidebar-drop_items">Description</A><A href="https://2018.igem.org/Team:Goettingen/Design" class="sidebar-items sidebar-drop_items">Design</A><A href="https://2018.igem.org/Team:Goettingen/Experiments" class="sidebar-items sidebar-drop_items">Experiments</A><A href="https://2018.igem.org/Team:Goettingen/Notebook" class="sidebar-items sidebar-drop_items">Notebook</A><A href="https://2018.igem.org/Team:Goettingen/InterLab" class="sidebar-items sidebar-drop_items">InterLab</A><A href="https://2018.igem.org/Team:Goettingen/Model" class="sidebar-items sidebar-drop_items">Model</A><A href="https://2018.igem.org/Team:Goettingen/Results" class="sidebar-items sidebar-drop_items">Results</A><A href="https://2018.igem.org/Team:Goettingen/Demonstrate" class="sidebar-items sidebar-drop_items">Demonstrate</A><A href="https://2018.igem.org/Team:Goettingen/Improve" class="sidebar-items sidebar-drop_items">Improve</A><A href="https://2018.igem.org/Team:Goettingen/Attributions" class="sidebar-items sidebar-drop_items">Attributions</A></DIV><BUTTON class="dropdown_button sidebar-items">Parts</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Parts" class="sidebar-items sidebar-drop_items">Parts Overview</A><A href="https://2018.igem.org/Team:Goettingen/Basic_Part" class="sidebar-items sidebar-drop_items">Basic Parts</A><A href="https://2018.igem.org/Team:Goettingen/Composite_Part" class="sidebar-items sidebar-drop_items">Composite Parts</A><A href="https://2018.igem.org/Team:Goettingen/Part_Collection" class="sidebar-items sidebar-drop_items">Part Collection</A></DIV><A href="https://2018.igem.org/Team:Goettingen/Safety" class="sidebar-items">Safety</A><BUTTON class="dropdown_button sidebar-items">Human Practices</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Human_Practices" class="sidebar-items sidebar-drop_items">Human Practices</A><A href="https://2018.igem.org/Team:Goettingen/Human_Practices/Survey" class="sidebar-items sidebar-drop_items">Survey</A><A href="https://2018.igem.org/Team:Goettingen/Public_Engagement" class="sidebar-items sidebar-drop_items">Education &amp; Engagement</A></DIV><BUTTON class="dropdown_button sidebar-items">Awards</BUTTON><DIV class="sidebar-dropdown is-hidden"><A href="https://2018.igem.org/Team:Goettingen/Applied_Design" class="sidebar-items sidebar-drop_items">Applied Design</A><A href="https://2018.igem.org/Team:Goettingen/Hardware" class="sidebar-items sidebar-drop_items">Hardware</A><A href="https://2018.igem.org/Team:Goettingen/Measurement" class="sidebar-items sidebar-drop_items">Measurement</A><A href="https://2018.igem.org/Team:Goettingen/Model" class="sidebar-items sidebar-drop_items">Model</A></DIV><A href="https://igem.org/2018_Judging_Form?id=2586" class="sidebar-items">Judging Form</A></DIV><DIV id="main-content"><BUTTON id="show_nav" class="control-button">≡</BUTTON></DIV></DIV><DIV id="footer"><DIV class="l-two-col"><DIV class="contact"><SPAN class="footer_heading">Contact:</SPAN><TABLE><TBODY><TR><TH>E-mail:</TH><TD><A href="mailto:igem2018@gwdg.de">igem2018@gwdg.de</A></TD></TR><TR><TH>Adress:</TH><TD>c/o iGEM 2018</TD></TR><TR><TD>Department of General Microbiology</TD></TR><TR><TD>Grisebachstraße 8</TD></TR><TR><TD>37077 Göttingen</TD></TR><TR><TD>Germany</TD></TR></TBODY></TABLE></DIV><DIV class="shout_out"><P>We are especially grateful to our numerous sponsors on <A href="https://de.gofundme.com/glyphosate-on-my-plate">gofundme,</A> who supported our project and made it possible. Every public donor, who helped us to achieve our goals got a special place on our bacterial wall of fame and a handdrawn microbe to show our appreciation. A link to that wall can be found here:</P><P class="shout_out_wall_of"><A href="https://2018.igem.org/Team:Goettingen/WallOfFame">Bacterial Wall of Fame</A></P></DIV></DIV></DIV></DIV><DIV class="site-content"><DIV class="article"><DIV class="article_head-line"><H1>Experiments</H1></DIV><P> All experiments were performed using the Method Collection &quot;Methods in Molecular Biology of Bacteria&quot; provided by the <A href="http://www.uni-goettingen.de/en/577601.html" target="_blank" rel="noopener">Department of General Microbiology</A>, University of Göttingen. The method collection is based on methods that have been collected and optimized during the past years by the students and supervisors of the Department of General Microbiology. The methods are suitable to engineer bacteria like <I>Bacillus subtilis</I>, <I>Escherichia coli</I>, <I>Listeria monocytogenes</I> and <I>Mycoplasma pneumoniae</I>. The book also contains protocols for the purification and characterization of proteins.


If you are interested in purchasing the whole method collection you can contact <A href="http://www.uni-goettingen.de/en/contact/579943.html" target="_blank" rel="noopener">us!</A> Hopefully, it will be soon available at <A href="https://www.amazon.com/" target="_blank" rel="noopener">Amazon</A>. </P><DIV class="article_head-line article_clear-right"><H2> High-throughput cultivation of bacteria</H2></DIV><P>For growth experiments with the Synergy MX II multi-well plate reader (Biotek) prepare pre- cultures and grow them over night at 28°C. Next day dilute these cultures (for instance 1:100, 1:500 and 1:1,000) in 1 - 5 ml of the same medium that will be later on used for recording growth curves and incubate them at 37°C (or other temperatures) and 220 rpm. At an OD<SUB>600</SUB> of about 0.2 – 0.5 use the cultures to inoculate a 96 well microtitre plate that has been supplemented with 100 µl medium per well. The outermost wells of the microtitre plate should contain 150 µl of distilled water to avoid that medium containing cells evaporate. Incubate the plate for a maximum of 48 h at the intermediate shaking mode. The OD<SUB>600</SUB> can be detected every 10 to 15 min. It is recommended to do at least three technical replicates per strain and condition on the same plate. Sometimes the detection of the OD can be disturbed by condensing water under the lid and your growth curve will look really strange.</P><DIV class="article_head-line"><H2> Genetic modification of bacteria </H2></DIV><H3> Preparation of competent <EM>E. coli </EM> cells</H3><H4>Method 1 (low amount of competent cells, RbCl)</H4><H5>The Material </H5><UL><LI>LB liquid medium</LI><LI>RF1 and RF2 buffer</LI><LI>Liquid nitrogen</LI></UL><H5>The Procedure</H5><OL><LI>Inoculate a 4 ml culture either with a single colony or with a cryo culture of the desired <EM> 
E. coli </EM> strain and incubate the culture with agitation over night at 37°C.</LI><LI>Inoculate a 300 ml shake flask containing 100 ml LB medium with the overnight culture to an OD<SUB>600</SUB> of 0.05 and grow the culture at 37°C until the OD<SUB>600</SUB> is about 0.3.</LI><LI>Transfer the cells into two 50 ml Falcon tubes, incubate the cultures for 15 min on ice and harvest the cells by centrifugation for 15 min at 5,000 rpm and 4°C. Discard the supernatants.</LI><LI>Re-suspend the cells in 1/3 of the original volume (~16 ml/50 ml) of buffer RF1, incubate the cells again on ice and harvest the cells by centrifugation for 15 min at 5,000 rpm and 4°C. Discard the supernatants.</LI><LI>Put 0.4 ml of the cell suspension into the Eppendorf reaction tubes and freeze the cells by transferring them immediately to the liquid nitrogen. Store the competent cells at -80°C.</LI></OL><H4>Method 2 (fast method, cells cannot be stored, CaCl<SUB>2</SUB>)</H4><H5>The Material</H5><UL><LI>LB liquid medium 50 mM CaCl<SUB>2</SUB> solution</LI></UL><H5>The Procedure</H5><OL><LI>Inoculate a 4 ml culture either with a single colony or with a cryo culture of the desired <EM>
E. coli </EM> strain and incubate the culture with agitation over night at 37°C.</LI><LI>Inoculate a 100 ml shake flask containing 10 ml LB medium with the overnight culture to an OD<SUB>600</SUB> of 0.05 - 0.1 and grow the culture at 37°C until the OD<SUB>600</SUB> is about 0.3.
          <DIV class="article_picture"><P> Schematic illustration of competent cell preparation. The desired <EM> E. coli </EM> strain is grown to log phase, pelleted and re-suspended in CaCl<SUB>2</SUB> solution. Now the cells are competent and should be kept on ice all the time. To test the competence of the cells, one aliquot is used to test the transformation efficiency. This is done with a standard plasmid, e.g. pUC19 and a defined amount. Afterwards the efficiency can be calculated.</P></DIV></LI><LI>Transfer the cells into 15 ml Falcon tubes and harvest the cells by centrifugation for 6 min at 5,000 rpm and 4°C. Discard the supernatants and re-suspend the cells in 5 ml of ice-cold CaCl<SUB>2</SUB> solution.</LI><LI>Incubate the cells for 30 min on ice and collect them again by centrifugation for 6 min at 5,000 rpm and 4°C. Re-suspend the pellet in 1 ml of the ice-cold CaCl<SUB>2</SUB> solution. Now the cells are ready to be transformed with your re-ligation and ligation samples.</LI></OL><H4>Method 3 (high amount of competent cells, time-consuming)</H4><H5>The Material</H5><UL><LI>TB buffer (ice-cold)</LI><LI>SOB -Mg</LI><LI>1 M MgCl<SUB>2</SUB></LI><LI>1 M MgSO<SUB>4</SUB></LI><LI>DMSO (100 %)</LI></UL><H5>The Procedure</H5><OL><LI>Inoculate a 20 ml culture either with a single colony or with a cryo culture of the desired <EM>
E. coli</EM> strain and incubate the culture with agitation for 20 h at 28°C.</LI><LI>Inoculate* 250 ml SOB medium supplemented with 10 mM MgCl<SUB>2</SUB> and 10 mM MgSO<SUB>4</SUB> in a 2 l shake flask and grow the cells to an OD<SUB>600</SUB> of 0.5 – 0.9 (20 – 24 h) at 18°C and 200 – 250 rpm.
          <DIV class="article_direct-footnote">*) The volume of the pre-culture strongly depends on the <EM> E. coli </EM> strain. Using DH5a it is recommended to use 4 ml pre-culture and inoculate at 8 a.m. Next day around 10 a.m. this strain should have reached an OD<SUB>600</SUB> between 0.5 and 0.6. If you use the strain XL1-Blue, you will need less cells and it is sufficient to inoculate the SOB medium around lunchtime.</DIV></LI><LI>Incubate the whole flask for 10 min on ice. Collect the cells by centrifugation for 10 min at 4°C and 5,000 rpm. Re-suspend the cells in 80 ml of ice-cold TB and incubate them for 10 min on ice. Collect the cells by centrifugation for 5 min at 5,000 rpm.</LI><LI>Re-suspend the cells in 20 ml of ice-cold TB. Add DMSO to a final concentration of 7 % (1.4 ml) by gently shaking the Falcon tube.</LI><LI>Transfer 0.2 ml aliquots into labeled Eppendorf reaction tubes and freeze the cells in liquid nitrogen. Store the cells at -80°C.
        </LI></OL><H3>Transformation of <EM>E. coli</EM></H3><H5>The Material</H5><UL><LI>LB medium</LI><LI>LB agar plates supplemented with the appropriate antibiotics</LI></UL><H5>The Procedure</H5><OL><LI>Put your ligation samples on ice, defreeze 100 ml of your competent <EM> E. coli </EM> cells on top of the ice and add the cells to your ligation samples. Mix it carefully!</LI><LI>Incubate the Eppendorf reaction tubes for 30 min on ice, transfer the tubes for 90 sec to 42°C (heat shock) and put them back on ice for 5 min.</LI><LI>Add 500 ml LB medium to the cells, transfer them to 15 ml Falcon tubes (or in sterile 1.5 ml Eppendorf reaction tubes) and incubate the cell for 1 h at 37°C with agitation.</LI><LI>Propagate 50 ml of the cells on LB medium agar plates supplemented with the appropriate antibiotics. The remaining cells are collected by centrifugation for 1 min at 13,000 rpm and remove 400 ml of the supernatant. Re-suspend the pellet in the remaining 50 ml of the supernatant and propagate the cells on the same LB medium agar plates. It is highly recommended to do a negative control (only cells, no DNA).</LI><LI>Incubate the plates over night at 37°C. The plates should be stored after incubation over night at 4°C to avoid the emergence of satellite colonies.</LI><DIV class="article_picture"><H3>Preparation and transformation of competent <EM>B. subtilis</EM> cells </H3><H5>The Material</H5><UL><LI>LB medium</LI><LI>MN medium</LI><LI>MNGE medium</LI><LI>Expression mix</LI><LI>Glucose (20 %)</LI><LI>1 M MgSO<SUB>4 </SUB></LI><LI>CAA (10 %)</LI></UL><H5>The Procedure</H5><P>Preparation of competent cells</P><OL><LI>Inoculate 4 ml LB liquid medium with a single colony of a <EM>B. subtilis</EM> strain and incubate the culture over night at 28°C with agitation.</LI><LI>Use the overnight culture to inoculate 10 ml MNGE medium supplemented with 0.1 % CAA in a 100 ml shake flask to an approximate OD<SUB>600</SUB> of 0.1. Incubate the culture at 37°C and 220 rpm until an OD<SUB>600</SUB> of about 1.3. This may take up to 5 h, depending on the strain.</LI><LI>Dilute the culture 1:1 with pre-warmed MNGE (w/o CAA) and incubate the culture for another h at 37°C on a shaker. You can continue with the transformation of <EM>B. subtilis </EM>directly after the nutritional starvation step (see Transformation of <EM>B. subtilis</EM>, step 7) or continue for long term storage.</LI><LI>Transfer 15 ml of the culture to 15 ml Falcon tubes and harvest the cells by centrifugation for 5 min at 5,000 rpm. Transfer the supernatant into a sterile Falcon tube.</LI><LI>Re-suspend the cells in 1.8 ml of the supernatant, add 1.2 ml 50 % glycerine, mix the cell suspension and store the competent cell in 300 ml aliquots at -80°C.</LI><LI>Thaw an aliquot of the frozen, competent bacteria and mix 300 μl of them with 1.7 ml 1x MN medium that has been supplemented with 43 μl glucose (20 %) + 34 μl 1 M MgSO<SUB>4</SUB>.</LI><LI>Add 0.1 - 1 μg DNA (2 mg plasmid DNA) to 400 µl of the competent cells and incubate the reaction tube for 30 min at 37°C.</LI><LI>Add 100 μl expression mix and if required an inducer (IPTG, xylose,…).</LI><LI>Incubate the bacteria for 1 h at 37°C with agitation and propagate the cells on SP medium agar plates supplemented with the appropriate antibiotics </LI></OL><P><STRONG>Remarks: Do not forget the negative control!!!</STRONG></P><DIV class="article_head-line"><H2> Working with DNA </H2></DIV><H3>Preparation of chromosomal DNA from <EM>B. subtilis </EM></H3><P>Here are different methods available for the isolation of chromosomal DNA. We will use the peqGOLD Bacterial DNA Kit from PEQLAB for the preparation of chromosomal DNA. The amount of DNA that will be obtained using this kit is much lower than that obtained with the standard phenol/chloroform extraction. However, the kit was designed for rapid purification of phenol- free total DNA, which is sufficient for PCR, cloning and even genome sequencing.</P><H5>The Material</H5><UL><LI>Lysis buffer</LI><LI>RNase A solution</LI><LI>EtOH (100 %, ice-cold)</LI><LI>peqGOLD Bacterial DNA Kit</LI></UL><H5>The Procedure</H5><OL><LI>Inoculation of 10 ml LB medium supplemented in a 100 ml shake flask with a single colony of <EM> B. subtilis</EM>. The overnight culture has to be incubated with agitation (220 rpm) at 37°C.</LI><LI>Collect the cells from 1.5 to 2 ml of the overnight culture in Eppendorf tubes by centrifugation for 2 min at 13,000 rpm. Discard the supernatant and re-suspend the cells in 200 ml lysis buffer.</LI><LI>Incubate the mixture for 30 - 60 min at 30°C in the lysis buffer to destroy the cell wall.</LI><LI>Spin down the cells for 5 min at 4,000 g and discard the supernatant. Re-suspend the pellet in 400 µl DNA Lysis Buffer T and add 20 µl Proteinase K and 15 µl RNase A. Vortex for 10 seconds.</LI><LI>Incubate the sample for 30 min at 70°C on a shaker. Without thermo-shaker, vortex 3 times for 10 seconds in the incubation time.</LI><LI>Add 200 µl of the DNA Binding Buffer and mix gently. Add the whole sample to the column (with all the precipitants). After centrifugation for 1 min at 8,000 – 10,000 rpm, discard the flow through. Put the remaining solution into the column and repeat the centrifugation step.</LI><LI>Wash the DNA on the column with 650 ml with the DNA Wash Buffer complemented with EtOH. Each time, centrifuge for 1 min at 8,000 – 10,000 rpm and discard the flow trough.</LI><LI>Place the empty DNA column into the collection tube and centrifuge for 2 minutes at 8,000 – 10,000 rpm. This step is essential! Do not reduce the time!</LI><LI>Place the column into a new 1.5 ml Eppendorf reaction tube. To elute the DNA, add 50 ml of buffer Elution Buffer (or sterile HPLC-H<SUB>2</SUB>O) that has been pre-warmed to 70°C and incubate the column for 3 min at room temperature. Collect the DNA by centrifugation for 1 min at 8,000 rpm.</LI><LI>To increase the amount of chromosomal DNA, you may add a second time your first elution to the column or add again 50 µl Elution Buffer or water and centrifuge again for 1 min at full speed.</LI></OL><H3>Isolation of plasmid DNA from <EM> B. subtilis </EM></H3><P>The cell wall of the Gram-positive bacterium <EM> B. subtilis </EM> cannot be disrupted as easy as that of a Gram-negative bacterium like <EM> E. coli </EM>. Therefore, lysozyme has to be added to the re-suspension buffer A1. Lysozyme is a glycoside hydrolase that catalyzes the hydrolysis of 1,4-b-linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in the peptidoglycan.</P><H5>The Material</H5><UL><LI>Liquid LB medium</LI><LI>NucleoSpin<SUP>®</SUP> plasmid kit, Macherey &amp; Nagel</LI><LI>Buffer A1 supplemented with 3 mg/ml lysozyme</LI></UL><H5>The Procedure</H5><OL><LI>Use a single colony from your <EM> B. subtilis </EM> plate to inoculate 4 ml LB liquid medium supplemented with the appropriate antibiotic and grow the culture over night at 37°C with agitation.</LI><LI>Collect the cells from 2 ml of the overnight cultures by centrifugation for 1 min at 13,000 rpm. Re-suspend the pellet in 250 µl of buffer A1 with lysozyme und incubate for 15 min at 37°C. Add 250 ml of buffer A2, mix gently 6 – 8 times and incubate the suspension for 5 min at room temperature.</LI><LI>Add 300 ml of buffer A3, mix gently 6 – 8 times and centrifuge the lysate for 10 min at 13,000 rpm.</LI><LI>Transfer the supernatant into a NucleoSpin® plasmid column, centrifuge for 1 min at 13,000 rpm and discard the flow-through.</LI><LI>Add 500 ml buffer AW pre-heated to 50°C to the column, centrifuge for 1 min at 13,000 rpm and discard the flow-through.</LI><LI>Add 600 ml of buffer A4, centrifuge for 1 min at 13,000 rpm and discard the flow- through.</LI><LI>To dry the silica membrane, centrifuge again for 2 min at 13,000 rpm.</LI><LI>Put the column in a sterile Eppendorf reaction tube and add 50 µl of HPLC water pre-heated to 70°C on top of the silica membrane of the column, incubate the column for 5 min at room temperature and elute the DNA by centrifugation for 1 min at 13,000 rpm.</LI><LI>Quantify the plasmid DNA with the Nanodrop spectrophotometer. The DNA can be stored at -20°C.</LI></OL><P>If the concentration of the plasmid DNA is too low for sequencing (less than 50 ng/µl) you can transform <EM>E. coli </EM> with the plasmid and isolate the plasmid again.</P><H3>Isolation of plasmid DNA from <EM>E. coli</EM></H3><H5>The Material</H5><UL><LI>Liquid LB medium</LI><LI>NucleoSpin<SUP>®</SUP> plasmid kit, Macherey &amp; Nagel</LI></UL><H5>The Procedure</H5><OL><LI>Use single colonies from your transformation plates to inoculate 4 ml (15 – 20 ml for low-copy plasmids) LB liquid medium supplemented with the appropriate antibiotic and grow the culture over night at 37°C with agitation.</LI><LI>Collect the cells from 1.5 ml (15 ml for low-copy plasmids) of the overnight cultures by centrifugation (1 min, 13,000 rpm). Re-suspend the pellet in 250 ml of buffer A1, add 250 ml of buffer A2, mix gently 6 – 8 times and incubate the suspension for 5 min at room temperature. For the isolation of low-copy plasmids add 500 ml of buffers A1 and A2 to the cells.</LI><LI>Add 300 ml (0.6 ml for low-copy plasmids) of buffer A3, mix gently 6 – 8 times and centrifuge the lysate for 5 – 10 min at 13,000 rpm.</LI><LI>Transfer the supernatant into NucleoSpin® plasmid column, centrifuge for 1 min at 13,000 rpm and discard the flow-through.</LI><LI>(Optional: for sequencing of low-copy plasmid DNA add 500 ml buffer AW to the column, centrifuge for 1 min at 13000 rpm and discard the flow-through.)</LI><LI>Add 600 ml of buffer A4, centrifuge (1 min, 13,000 rpm) and discard the flow-through.</LI><LI>To dry the silica membrane, centrifuge again for 2 min at 13,000 rpm.</LI><LI>Add 50 ml of sterile and deionized water on top of the silica membrane of the column incubate the column for 1 - 5 min at room temperature and elute the DNA by centrifugation for 1 min at 13,000 rpm.</LI><LI>The plasmid DNA is highly pure and can be stored at –20°C.</LI></OL><H3>Concentration and purity of DNA/RNA samples</H3><H4>Determination of DNA/RNA concentration</H4><P>A reliable method for the quantification of DNA and RNA is essential for our work in the lab. The concentration of nucleic acids can be determined by fluorescence and absorption spectroscopy. The quantification of nucleic acids by fluorescence spectroscopy relies on binding to fluorescent dyes, e.g. ethidium bromide. This method is very sensitive (10 – 100 ng of DNA can be detected) but time-consuming. This is the reason that DNA is mainly quantified by absorption spectroscopy. DNA can be quantified by absorption spectroscopy because it absorbs ultra violet (UV) light with a maximum at the wavelength of 260 nm. However, RNA absorbs UV light also at 260 nm and aromatic amino acids at 280 nm what means that both can contaminate your measurements. Therefore, it is important to determine the purity of your samples.</P><H4>The purity of the DNA preparation</H4><P>The quantification of DNA makes only sense if the sample is pure. The purity of the sample can be evaluated by recording a spectrum of your DNA solution in the range between 220 and 320 nm. Proteins in the sample absorb light at 280 nm and cause a reduction of the A260 nm/A280 nm ratio. If the DNA sample is pure, this ratio is in the range of 1.8 – 2.0. A strong peak at 230 nm can indicate contamination with organic compounds or chaotropic salts.</P><H5>The Material</H5><UL><LI>Nanodrop ND-1000 Spectrophotometer</LI><LI>DNA/RNA sample</LI><LI>Sterile water</LI></UL><H5>The Procedure</H5><OL><LI>Clean the measurement pedestal (it contains the receiving fibre) with deionized water and pipette 1 ml of water on top of it.</LI><LI>Close the lid (sampling arm). Now the upper measurement pedestal (it contains the second fibre optic cable) is brought into contact with the liquid sample causing the liquid to bridge the gap between the two fibre optic ends. The gap is controlled to both 1 and 0.2 mm paths.</LI><LI>Initiate a spectral measurement using the operating software on the connected PC by pressing the “blank” button to blank the Nanodrop. A pulsed xenon flash lamp will provide the light source and the spectrophotometer analysis the light after passing through the sample. After the measurement the data is logged in an archive file on the PC.</LI><LI>When the blanking is done, open the sampling arm and wipe the sample from both the upper and lower pedestals using a soft laboratory wipe. Wiping prevents sample carryover in successive measurements for samples varying by more than 1,000-fold in concentration.</LI><LI>Pipette 1 ml your DNA sample and press the “Measure” button to determine the amount of DNA in your sample.</LI><LI>Remove the DNA solution from the measurement pedestal and clean the lid and the pedestal with deionized water.</LI></OL><H3>Agarose gel electrophoresis</H3><P>Linear and double-stranded DNA molecules can move through a gel matrix with a velocity that is proportional to the logarithm of their molecular weight (MW). Therefore, the MW of unknown DNA species can be determined by comparing their electrophoretic mobility with that of DNA molecules having known MWs. DNA molecules with lengths ranging from 0.5 kbp to 25 kbp can be separated using agarose gels.</P><H5>Voltage</H5><P>The velocity with that DNA molecules move through a gel matrix depends on the applied voltage. It is important to know that the velocity of large DNA molecules increases more rapidly than that of small DNA molecules. As a consequence, large molecules can be separated from each other with a high accuracy at low voltages.</P><H5>The electrophoresis buffer</H5><P>Most gel systems are based on Tris-Acetate-EDTA (TAE) and Tris-Borat-EDTA (TBE) buffers. The electrophoretic mobility of DNA molecules is quite similar in both buffers. However, the buffering capacity of the TBE buffer is significantly higher. Therefore, it is recommended to use TBE buffer for mini-gels that run at a high voltage.</P><H5>The DNA conformation</H5><P>Supercoiled, nicked and linear DNA molecules, having the same molecular mass move with different velocities through the gel matrix. Because supercoiled DNA has the smallest hydrodynamic radius, these DNA species show the highest electrophoretic mobility.</P><H5>The Material</H5><UL><LI>1x TAE buffer or 1x TBE</LI><LI>5x DNA loading dye</LI><LI>Agarose</LI><LI>HDGreen<SUP>®</SUP> Plus Safe DNA Dye or Midori Green</LI></UL><H5>The Procedure</H5><OL><LI>Prepare a 100 ml mixture of 1 % agarose in 1x TAE. Carefully heat the mixture in a microwave to dissolve the agarose. You can either store the agarose solution at 65°C or cool it down to room temperature and heat it again for the next gel.</LI><LI>Now it is important to wear gloves. Add 3 µl of the HDGreen® Plus Safe DNA Dye to 30 ml dissolved agarose and pour the gel in a small custom-made gel chamber.</LI><LI>While the gel is cooling down, prepare your DNA samples by mixing 2 – 5 µl of your DNA solution with 0.4 – 1 µl of loading dye, either in Eppendorf reaction tubes or in a multi- well microtitre plate.</LI><LI>Remove the comb from the cold gel, add 1x TAE/TBE buffer into the chamber and load the slots with your DNA samples. Do not forget the size standard (λ-marker or commercial 1 kb DNA ladder).</LI><LI>Connect the gel chamber with the power supply and switch it on. Adjust the voltage to 100 – 130 V and press the start button.</LI><LI>If the stain bromphenol blue has passed ¾ of the gel, you should switch off the power supply and check your gel under UV light. Safe picture and print result for lab book.</LI><LI>If possible, take a picture for the group seminar and your lab notebook.</LI><LI>You can also stain your gel after the run has finished in a 1 % aqeous HDGreen® Plus Safe DNA Dye or Midori Green solution for 10 - 30 min at room temperature.</LI></OL><H5>Ethidium bromide staining</H5><P>To stain a agarose gel with Ethidium bromide you have to replace step two and stain the gel after running with the following procedure:</P><OL><LI>Put on one glove and transfer the gel into the staining solution (0.5 µg/ml of Ethidium bromide in water). After 10 min you can destain the gel in the water bath.</LI><LI>After 10 – 30 minutes of destaining you may analyze your agarose gel, either on a UV table or with a gel documentation device (imager).</LI></OL><H5>Error sources</H5><H6>The DNA molecules were separated with low accuracy</H6><P>Wrong agarose concentration; low voltage may cause diffusion of DNA molecules due to a long run.</P><H6>Smearing bands</H6><P>Too much DNA has been loaded to the slots; the applied voltage has been too high. The gel looks smeared at all, you might touched the gel directly with your hands.</P><H6>The gel was melting</H6><P>Even old lab veterans sometimes use water instead of 1-fold TAE to dissolve agarose. You should also regularly refresh the TAE buffer inside the running chamber.</P><H5>Further information concerning the HDGreen<SUP>®</SUP> Plus Safe DNA Dye</H5><H6>Security precaution</H6><P>Ethidium bromide is commonly used to stain nucleic acids in molecular biology laboratories. However, this agent is highly mutagen and therefore considered as hazardous. As alternative staining method we use HDGreen® Plus Safe DNA Dye. According to the manufacturer (INTAS) this dye does not contain substances which at their given concentration, are considered to be hazardous to health. Nevertheless, this dye stains DNA and every DNA staining substance should be treated carefully and only with gloves.</P><H6>Disposal considerations</H6><P>The waste must be disposed of in accordance with federal, state and local environmental control. Therefore, the gels are collected in a container next to the imager.</P><H3>Purification of DNA fragments</H3><H4>PCR purification kit</H4><H5>The Material</H5><UL><LI>DNA product(s) or linearized plasmid(s)</LI><LI>peqGOLD Cycle-Pure Kit</LI></UL><H5>The Procedure</H5><OL><LI>Add an equal volume of CP Buffer to your DNA solution. For fragments &lt; 200 bp add 3 volumes of CP Buffer for better results.</LI><LI>Transfer the mixture to a spin column and centrifuge for 1 min at 13,000 rpm. Discard the flow-through and add 750 µl CG Wash Buffer.</LI><LI>Centrifuge for 1 min at 13,000 rpm, discard the flow-through and repeat the steps 2 and 3 once.</LI><LI>Centrifuge again to dry the silica membrane 2 minutes at 13,000 rpm.</LI><LI>Put the column into a labeled Eppendorf reaction tube, add 30 - 50 µl of water (not less than 30 µl) on top of the silica membrane, incubate for 5 min at room temperature and elute the DNA by centrifugation for 1 min at 13,000 rpm. Please keep in mind that you have to add less water if you want to elute the DNA from a restriction analysis! Always check the purified DNA by agarose gel electrophoresis to make sure that you did not lose the DNA. Determine the DNA concentration and the purity.</LI></OL><H5>Concentration of DNA samples</H5><P>You can concentrate your DNA sample if the amount is too low. Therefore add 1 M NaCl to a final concentration of 0.1 M and add two volumes of 100 % EtOH. Vortex your samples and spin down for 15 minutes at 13,000 rpm. The supernatant can be discarded. Add then 700 µl 70 %age EtOH and centrifuge again for 2 min at 13,000 rpm. Let the EtOH evaporate at room temperature. Now you can dissolve your pellet in water.</P><H3>How to design primers</H3><P>One of the major tools in molecular biology is the polymerase chain reaction (PCR) where DNA fragments are amplified in a logarithmic manner. Therefore, you need at least two primers that bind to your template DNA strands and boarder the region that should be amplified. It is obvious that good primer design is essential for successful amplification.</P><H5>The Material</H5><UL><LI>Template sequence + the sequence of the flanking regions </LI><LI>Program for primer design (optional)</LI></UL><H5>The Procedure</H5><P>The optimal length of primers should be around 18 - 30 bp and the melting temperature should be around 52 - 64°C (usually 58 - 62°C). The temperature can be calculated with a program (e.g. Oligo Calc) or manually (for each C or G add 4°C, for each A or T add 2°C). Non-binding sites are not calculated in the melting temperature (like flanks or additional restriction sites).</P><P>Typical set-up for a forward and a reverse primer. The two primers anneal within the PCR reaction to the two complementary strands. The reading direction is different so that they produce a product. For sequencing the primer can also read only in one direction. The primer should bind once so the template (plasmid or chromosomal DNA). The GC content should be around 40 - 60 % to provide a stronger binding to the template. It is necessary to check if the primers can form undesired secondary structures that can lead to the disturbance of the PCR. This and the self- complementarities can also be analyzed by programs, like Oligo Clac.</P><P>For each method the primers need to be designed specifically. It has to be considered if the part of the DNA that will be amplified via PCR needs to be in frame in the further experiments. In this case a few bp should be added to assure the right reading frame. For some experiments it might also be necessary to remove the stop codon of the gene of interest. A crate of beer is demanded for every wrong designed AND ordered prime. </P><H3>The polymerase chain reaction (PCR)</H3><P>Although the basic principle of the polymerase chain reaction (PCR) has been described already in 1985, this simple method is still very important in molecular biology (Sakai et al., 1985). You just need a few things to set up a PCR: template DNA, oligonucleotides that hybridize to the template, a thermostable polymerase, buffer and nucleotides.</P><H5>The Material</H5><UL><LI>PhuS polymerase or <I>Taq</I> polymerase, purified in-house</LI><LI>Template DNA (chromosomal, plasmid DNA)</LI><LI>5x HF or GC buffer</LI><LI>12.5 mM dNTPs</LI><LI>5 pmol forward primer</LI><LI>5 pmol reverse primer</LI></UL><H5>The Procedure</H5><P>Prepare for your sample a reaction mixture according to the scheme below. If you have many reactions prepare a master mix and scale up the reaction volumes. For many applications a 50 µl sample is sufficient. For fragments that should be used for cloning use commercial Phusion polymerase, this is less error prone.</P><DIV class="article_table"><P>Pipetting scheme for a 100 µl <EM>PfuS</EM> sample</P><TABLE><TBODY><TR><TH>Compound</TH><TH class="article-table_right">Volume in µl</TH></TR><TR><TD>5x HF buffer</TD><TD class="article-table_right">20</TD></TR><TR><TD>dNTPs</TD><TD class="article-table_right">4</TD></TR><TR><TD>forward primer</TD><TD class="article-table_right">4</TD></TR><TR><TD>reverse primer</TD><TD class="article-table_right">4</TD></TR><TR><TD>Template DNA (chromosomal or other)</TD><TD class="article-table_right">1–2</TD></TR><TR><TD>DNA polymerase (<EM> Phusion</EM> or <EM>PfuS</EM>)</TD><TD class="article-table_right">1</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">66–67</TD></TR></TBODY></TABLE></DIV><DIV class="article_table"><P>Pipetting scheme for a 100 µl <EM>Taq</EM> sample</P><TABLE><TBODY><TR><TH>Compound</TH><TH class="article-table_right">Volume in µl</TH></TR><TR><TD>10x Taq buffer</TD><TD class="article-table_right">10</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD class="article-table_right">6</TD></TR><TR><TD>dNTPs</TD><TD class="article-table_right">1</TD></TR><TR><TD>forward oligo</TD><TD class="article-table_right">4</TD></TR><TR><TD>reverse oligo</TD><TD class="article-table_right">4</TD></TR><TR><TD>Template DNA (chromosomal or other)</TD><TD class="article-table_right">1–2</TD></TR><TR><TD>Taq polymerase</TD><TD class="article-table_right">1</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">72–73</TD></TR></TBODY></TABLE></DIV><H5>Troubleshooting for PCR</H5><P>Try to do the following steps to solve your PCR problem:</P><UL><LI>Dilute your primers anew, to be sure the primers are the correct ones. This helps in most cases when the PCR does not function at all.</LI><LI>Dilute your template, to lower the template concentration and enhance the PCR efficiency.</LI><LI>Longer elongation time.</LI><LI>Longer denaturation time.</LI></UL><H3>Digestion of DNA</H3><P>Digestion of DNA by restriction endonucleases is an important method in molecular biology. Restriction enzymes recognize 4 – 10 bp-long palindromic DNA sequences in a highly specific manner. For instance, the enzyme <EM> EcoRI </EM> from the <EM>Escherichia coli </EM>strain R recognizes the sequence 5’-GAATTC-3’. In the early days, the reaction samples to digest DNA were incubated for up to 20 h in order to obtain a fully digested DNA. Nowadays, many biotech companies sell hyperactive variants of the restriction enzymes that are commonly used in the lab. We will use FastDigest enzymes from Thermo Scientific. According to the manual from the manufacturer, 1 mg of DNA (for instance l phage DNA) are cleaved by 1 ml of the FastDigest enzyme within 5 – 15 min. With the same enzyme activity, 1/5 of a purified PCR product should be cleaved within 20 min.</P><H5>The Material</H5><UL><LI>DNA sample</LI><LI>Restriction enzyme</LI><LI>10x Buffer</LI></UL><H5>The Procedure</H5><DIV class="article_table"><P>Reaction mixture to digest DNA</P><TABLE><TBODY><TR><TH>Compound</TH><TH class="article-table_right">Volume in µl</TH></TR><TR><TD>10x buffer</TD><TD class="article-table_right">4</TD></TR><TR><TD>Enzyme</TD><TD class="article-table_right">3</TD></TR><TR><TD>Plasmid DNA (125 ng/µl)</TD><TD class="article-table_right">8</TD></TR><TR class="article_table_hline"><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">add 40</TD></TR></TBODY></TABLE></DIV><P>We recommend incubating the reaction mixtures for at least 30 min at 37°C to obtain fully digested DNA fragments. After digestion, you can use the PCR purification kit from Qiagen to purify the digested DNA. Please keep in mind that you have diluted the DNA solution in the reaction mixture. For eluting the DNA from the spin column you should use 28 – 30 ml of sterile water. To dephosphorylate your plasmid DNA, add 1 ml alkaline phosphatase to the reaction mixture and incubate the plasmid for additional 15 min at 37°C. Watch out for star activity of the restriction enzyme in your sample (e.g. <EM>EcoRI</EM>)! Star activity depends on the enzyme amount, the pH value, the presence of organic solvents, and the incubation time (Wei et al., 2008).</P><H5>There are different types of restriction enzymes:</H5><DIV class="article_table"><P>The types of restriction enzymes</P><TABLE><TBODY><TR><TH>Type</TH><TH>Characteristics</TH><TH>Example</TH></TR><TR><TD>I</TD><TD>3 subunits: S recognizes a specific sequenceM methylates and R cuts the DNA unspecifically</TD><TD><EM>EcoKI</EM></TD></TR><TR><TD>II</TD><TD>2 subunits: a methylase and a restriction enzyme; both enzymes recognize the same DNA sequence</TD><TD><EM>BamHI</EM></TD></TR><TR><TD>III</TD><TD>Several subunits: these enzymes cut a specific sequence 20-25 away from the recognition site</TD><TD><EM>EcoPI</EM></TD></TR></TBODY></TABLE></DIV><H3>Ligation of DNA</H3><P>Ligation is the process in which a digested plasmid is covalently connected to an insert (PCR product), which has been digested with the same restriction enzyme(s). If both, the insert and the plasmid, had been digested with only one enzyme, it is strongly recommended to dephosphorylate the plasmid prior to mix the ligation sample. Otherwise it takes your whole Master thesis or Ph. D. thesis until you find a plasmid containing the insert. We are using the T4 DNA ligase, which is a standard enzyme for ligation. In contrast to the NAD+-dependent DNA ligase, which needs NAD<SUP>+</SUP> as a cofactor, the T4 DNA ligase uses ATP. Moreover, while the NAD<SUP>+</SUP>- dependent DNA ligase can connect only DNA fragments with sticky ends, the T4 DNA ligase can ligate sticky and blunt ends.</P><H5>The Material</H5><UL><LI>T4 DNA ligase</LI><LI>10x Ligation buffer (keep aliquots on ice, contains ATP)</LI></UL><H5>The Procedure</H5><OL><LI>
          Defreeze an aliquot of the ligation buffer on ice (it contains ATP) and mix the ligation and re-ligation samples in a 1.5 ml Eppendorf reaction tube:
          <DIV class="article_table"><P>Reaction mixture for ligation and re-ligation</P><TABLE><TBODY><TR><TH>Ligation</TH><TH>Re-ligation</TH><TH>Compound</TH></TR><TR><TH>sample (µl)</TH><TH>sample (µl)</TH></TR><TR><TD>1</TD><TD>1</TD><TD>T4 DNA ligase</TD></TR><TR><TD>2</TD><TD>2</TD><TD>10x Ligation buffer</TD></TR><TR><TD>3</TD><TD>0</TD><TD>150 ng Insert</TD></TR><TR><TD>1</TD><TD>1</TD><TD>50 ng Plasmid </TD></TR><TR><TD>13</TD><TD>16</TD><TD>sterile H<SUB>2</SUB>O</TD></TR></TBODY></TABLE></DIV></LI><LI>Incubate the samples for at least 2 h at room temperature or over night at 16°C.</LI><LI>Use the complete ligation and re-ligation samples for transformation of competent
          <EM>E. coli</EM> cells. You should include a positive control (circular plasmid DNA) and a negative control (no DNA) to ensure that your <EM> E. coli</EM> cells were competent and not contaminated, respectively. Moreover, the ligation/re-ligation ratio will tell you whether it is worth to analyze your clones.</LI></OL><H5>Error sources</H5><H6>No transformants at all on the plates:</H6><P>The ATP in the ligation buffer was hydrolyzed (old buffer). Probably you forget a compound in the ligation mixture. The cells were not competent.</P><H6>Same number of ligands and re-ligands on the plates:</H6><P>Probably your DNA was not completely digested. Therefore, it is very important to check you digested DNAs by agarose gel electrophoresis prior to the ligation. You can at a least see that your plasmid DNA was linearized.</P><H3>3A Assembly</H3><P><OL><LI>Restriction Digests</LI><OL><LI>The left part sample is cut out with <EM>Eco</EM>RI and <EM>Spe</EM>I.</LI><LI>The right part sample is cut out with <EM>Xba</EM>I and <EM>Pst</EM>I.</LI><LI>The linearized plasmid backbone is a linear piece of DNA. It has a few bases beyond the <EM>Eco</EM>RI and 
                <EM>Pst</EM>I restriction sites. It is cut with <EM>Eco</EM>RI and <EM>Pst</EM>I.
        </LI></OL><LI>All 3 restriction digests are heated to heat kill all of the restriction enzymes.</LI><LI>An equimolar quantity of all 3 restriction digest products are combined in a ligation reaction.</LI><LI>The desired result is the left part sample's <EM>Spe</EM>I overhang ligated with the right part sample's 
            <EM>Xba</EM>I overhang resulting in a scar that cannot be cut with any of our enzymes.</LI><LI>The new composite part sample is ligated into the construction plasmid backbone at the <EM>Eco</EM>RI and 
            <EM>Pst</EM>I sites.
        </LI><LI>When the ligation is transformed into cells and grown on plates with antibiotic chloramphenicol, only colonies 
            with the correct construction survive.
 </LI></OL></P><DIV class="article_picture"><P>3A Assembly overview.</P></DIV><H3>DNA Sequencing</H3><P>The correct sequences of PCR products or of newly constructed plasmids having the desired insert can be confirmed by the in-house sequencing facility, the G2L or by sending the DNA to the companies SeqLab (Göttingen). All facilities determine the DNA sequence by the chain- termination method that has been developed by Frederick Sanger in 1977. Sanger received in 1980, together with Paul Berg and Walter Gilbert, the Nobel Prize in chemistry. This became his second Nobel price because Sanger had been already awarded in 1958 for his work on protein sequencing. The Sanger method is based on the amplification of the template by a DNA polymerase, and the incorporation of normal deoxynucleotidetriphosphates (dNTPs) and modified dideoxynucleotidetriphosphates (ddNTPs). The incorporation of modified (fluorescently-labeled) ddNTPs results in termination of strand elongation. Sequencing with fluorescently-labeled ddNTPs permits DNA sequencing in a single reaction and the fast analysis of the DNA fragments by an optical read out system. </P><H5>The Procedure</H5><H6>…for the <A href="http://appmibio.uni-goettingen.de/index.php?sec=g2l" target="_blank" rel="noopener">G2L:</A></H6><P>You need a personal folder on the network drive from the G2L. The contact person at the G2L is Dr. Sonja Voget. Her phone number is 33655. This folder contains a template Excel sheet, which you can fill in (sample labels, amount of samples etc.). Label your tubes starting with “NameSurname_1… NameSurname_n + 1”. Send the Excel sheet to “sequence@g2l.bio.uni-goettingen.de”. The folder also contains a Word document, which you have to fill in (again sample labels, amount of samples etc.). Print out the form and bring it together with your samples to the G2L until 15:00 o’clock on Friday afternoon. You will find the sequencing data in your personal folder usually at the following Wednesday.</P><H6>…for <A href="https://www.microsynth.seqlab.de/home-de.html" target="_blank" rel="noopener">Seqlab:</A></H6><P>Label your tubes with the pre-printed barcodes. Type Login to the SeqLab homepage and select “Economy Run”, type in your mail address and select the number of samples you have prepared for sequencing. Next you have to choose the number that has been used to label your first sample and finally confirm your order. Put all the samples in a plastic bag and the printed sheet (ordering details and number). Put this bag till 6 p.m. into the post box, which is located at the entrance at the back of the building. Sequencing by SeqLab is performed over night. You will get your sequences next day.</P><DIV class="article_table"><P>Overview of sample preparation</P><TABLE><TBODY><TR><TH>
              Sequencing service </TH><TH>G2L </TH><TH> SeqLab </TH></TR><TR><TD> DNA (ng)</TD><TD>200 </TD><TD> 600 (PCR product) to 1,200 (plasmid) </TD></TR><TR><TD>Oligo (µl, pM)</TD><TD> (1, 5) </TD><TD>(1, 20)</TD></TR><TR><TD>
              Final volume (µl) </TD><TD>5</TD><TD> 15</TD></TR><TR><TD>Reaction tube (ml)</TD><TD> 1.5 </TD><TD> 1.5 </TD></TR></TBODY></TABLE></DIV><H5>Evaluation of your sequencing data</H5><P>You can analyze your sequencing data using the Geneous which is installed on a PC in our department. For further information about the amino acids have a look to 105. Amino acids.</P><H3>Combined-chain reaction (CCR) and multiple-mutation reaction (MMR)</H3><P>Several methods for PCR-based site-directed mutagenesis have been developed. Among these, the combined-chain reaction method proved to be very rapid and reliable (Bi &amp; Stambrook, 1997). The principle of this method is the use of mutagenic primers that hybridize more strongly to the template than the external primers. The mutagenic primers are phosphorylated at their 5’ ends, and these are ligated to the 3’-OH groups of the extended upstream primers by the action of a thermostable DNA ligase (see the Figure beside for the principle). Moreover, the DNA polymerase employed must not exhibit 5’ 3’ exonuclease activity, to prevent the degradation of the extended primers. In our view, Pfu and Pwo polymerases are both well suited. The original protocol describes the introduction of two mutations simultaneously. In a previous study, we used a combined chain reaction to mutagenize four distant bases in a DNA fragment in a one-step reaction.</P><P>The MMR is a further development of the CCR and allows the simultaneous introduction of up to nine mutations in a single PCR approach (more have yet to be proven; Hames et al., 2005). The MMR was especially designed for the work with Mycoplasma genes in order to express recombinant genes in<EM> E. coli </EM>. The species Mycoplasma differ in their genetic code and use the opal codon (UGA) to introduce tryptophan instead of terminating the translation. Hence, it is absolutely necessary to exchange the opal codons, if present, with UGG codons (TGG on DNA level). Meanwhile, the in vitro synthesis of oligonucleotides is getting more inexpensive and will replace this method in the future, especially, because the MMR can be very time consuming.</P><H5>The Material</H5><UL><LI>10x CCR/MMR buffer</LI><LI>Phusion or AccuzymeTM polymerase</LI><LI>5 U/ml Ampligase</LI><LI>Primer for amplification</LI><LI>12.5 mM dNTPs</LI><LI>Mutagenesis primer</LI><LI>BSA (20 mg/ml)</LI></UL><H5>The Procedure</H5><P>Before you start your CCR or MMR you should amplify your gene of interest and clone it into a plasmid of your choice. The introduction of mutations is far more efficient when including this step.</P><DIV class="article_table"><P>CCR/MMR reaction setup</P><TABLE><TBODY><TR><TH>Compound</TH><TH class="article-table_right">Volume in µl</TH></TR><TR><TD>20 pmol forward primer</TD><TD class="article-table_right">2</TD></TR><TR><TD>20 pmol reverse primer</TD><TD class="article-table_right">2</TD></TR><TR><TD>mutagenesis primer</TD><TD class="article-table_right">4</TD></TR><TR><TD>Plasmid DNA</TD><TD class="article-table_right">1</TD></TR><TR><TD>10x CCR buffer or HF buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>Accuzyme<SUP>™</SUP> or Phusion polymerase</TD><TD class="article-table_right">1</TD></TR><TR><TD>Ampligase</TD><TD class="article-table_right">3</TD></TR><TR><TD>dNTPs</TD><TD class="article-table_right">2</TD></TR><TR><TD>BSA (20 mg/ml)</TD><TD class="article-table_right">2</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">30</TD></TR></TBODY></TABLE></DIV><P>Do not forget to adjust the elongation time according to your polymerase and the length of the template.</P><H3>Overexpression of recombinant proteins</H3><P>Large-scale purification is necessary to achieve enough protein for further experiments, e.g. crystallization or enzyme activity assays. After the evaluation of the best expression conditions with test expressions you can scale up your samples.</P><H5>The Material</H5><UL><LI>LB liquid medium (e.g. 1 liter per protein,)</LI><LI>1 M Isopropyl-D-thio-galactopyranoside (IPTG) stock</LI><LI>Washing buffer (e.g. buffer W,) </LI><LI>Elution buffer (e.g. buffer E,)</LI></UL><H5>The Procedure</H5><OL><LI>Inoculate an overnight culture in 25 - 100 ml LB medium with the desired strain(s) and antibiotics. Incubate the culture at 37°C and 220 rpm over night.</LI><LI>Inoculate the over day cultures (usually 500 ml - 1 l in a 2 l baffled flask with long neck) to an OD<SUB>600</SUB> of 0.1 with your overnight culture and add the appropriate antibiotic. Grow the cultures at 37°C with agitation to an OD<SUB>600</SUB> of 0.6 - 0.8. Then add IPTG to a final concentration of 1 mM. Now the expression of the heterologous gene is induced.</LI><LI>After induction, incubate the cultures for additional 2 - 3 h at 37°C with shaking (if these are the conditions of choice).</LI><LI>Split the cultures into two 500 ml samples and centrifuge (15 min, 5,000 rpm) at 4°C.</LI><LI>Discard the supernatant (in case your protein is not secreted).</LI><LI>Re-suspend the pellet in 15 ml buffer W, transfer it to a 50 ml falcon tube and centrifuge the tubes at 8,500 rpm and 4°C for 10 - 15 min.</LI><LI>Discard the supernatant and freeze your cell pellet (-20°C) or continue with cell disruption.</LI><LI>Re-suspend each pellet in 25 - 50 ml of washing buffer. Now your samples are ready for cell disruption, e.g. using the French Press method.</LI></OL><H3>Preparation of cell-free crude extracts</H3><P>This method can be applied to detect proteins, which are really strongly produced by the bacteria. The cell-free crude extract that has been obtained by this procedure can also be used for Western blotting. A drawback of this method is that you may lose some protein by proteolysis. Moreover, the amount of protein you will get is sometimes quite low.</P><H3>The Material</H3><UL><LI>1x PBS, pH 7.4</LI><LI>Lysis buffer</LI><LI>(recommended to add protease inhibitor: EDTA, Pefabloc, PMSF)</LI></UL><H3>The Procedure</H3><OL><LI>Harvest 2 ml from a <EM>  B. subtilis </EM> culture that has been grown over night or to an OD<SUB>600</SUB> of approximately 1 - 2 by centrifugation for 2 min at full speed.</LI><LI>Re-suspend the cells in 50 µl lysis buffer. Keep some ml of the lysis buffer for the Bradford assay. Incubate the cells for 30 min at 37°C and 350 rpm on an Eppendorf thermomixer.</LI><LI>Separate the soluble proteins from the cell debris by centrifugation for 10 min at full speed and 4°C. Transfer the supernatants into new Eppendorf reaction tubes. Determine the protein concentration and analyze your samples by SDS PAGE and Western blotting.</LI></OL><H3>Cell disruption by French press</H3><P>Many experiments required the disruption of bacterial cells, e.g. to purify proteins for enzyme activity assays or protein-protein interaction studies. A widely used method in our lab for cell disruption is using a French pressure cell, or simply French press. In principle, a French press disrupts the cell wall and plasma membrane by passing the cells through a narrow valve under high pressure. The name has nothing to do with the machine some people prepare their coffee in, but was named after its inventor Charles Stacy French of the Carnegie Institution of Washington.</P><H5>The Material</H5><UL><LI>French press (small or big)</LI><LI>Disruption cell (bomb) Pre-cooled in a water bath or in the fridge</LI><LI>Re-suspension buffer Pre-cooled in the fridge or on ice</LI><LI>Cell pellets Frozen cells within a 50 ml falcon tube</LI></UL><H5>The Procedure</H5><P><STRONG>All steps must be carried out on ice!</STRONG></P><OL><LI>Re-suspend your cells pellet in re-suspension buffer by vortexing or pipetting up and down. Usually we use a pellet from 500 ml of cells with an OD<SUB>600</SUB> of 0.8 – 1.2 and re-suspend it in 15 ml cold buffer of choice.</LI><LI>Turn on the French Press and select the pressure for cell disruption (18,000 psi for
          <EM> B. subtilis </EM> and <EM> E. coli </EM>).</LI><LI>Fill the re-suspended cells in the disruption cell and close it tightly.</LI><LI>Put the closed disruption cell into the machine so that the piston is on the top. Make sure that the handles of the piston are turned in your direction so they cannot come in contact with the mounting while the cell gets compressed.</LI><LI>Press start and follow the instructions on the display of the machine.</LI><LI>Open the valve slowly while you put a 50 ml falcon under it.</LI><LI>Collect the opened cells that flow out of the valve. Watch out that the cell suspension flows out slowly and the pressure remains almost constant until the whole suspension has flown out.</LI><LI>Reload the disruption cell. For Gram-positive bacteria, open up the cells three times. For Gram-negative cells one run should be enough.</LI><LI>Spin down your disrupted cells at 8,000 rpm for 15 min at 4°C. Proceed with the supernatant.</LI><LI>Fill the supernatant into ultracentrifugation tubes and balance tem. Spin down for 1 hour at 4°C and 35,000 rpm. Tubes must be filled with buffer at least with half of the tube-volume.</LI><LI>In the meantime, clean the disruption cell with ethanol (70 %) and distilled water. Then store it in the fridge until next use.</LI><LI>Fill the supernatant into pre-cooled falcon tube.</LI></OL><H3>Purification of Strep-tagged proteins</H3><P>The Strep-tag II is a short peptide (8 amino acids, WSHPQFEK), which binds with high selectivity to Strep-Tactin®, an engineered streptavidin. The binding affinity of Strep-tag II to Strep-Tactin® (KD = 1 mM) is nearly 100 times higher than to streptavidin. This technology allows one-step purification of almost any recombinant protein under physiological conditions, thus preserving its bioactivity.</P><H5>The Material</H5><UL><LI>Buffer W</LI><LI>Buffer E</LI><LI>Crude extracts of your sample cells harboring the desired protein</LI><LI>Strep-Tactin<SUP>®</SUP> sepharose (50 %; 1 ml per 1 l cell culture)</LI></UL><H5>The Procedure</H5><H6>Equilibration of the column</H6><OL><LI>Remove the bottom cap from the column and apply the appropriate amount of Strep- Tactin® matrix (1 ml of the 50 % suspension per 1 l of cell culture, column bed volume = 0.5). For all steps, allow the solutions on the column to drain off by gravity before adding the next one.</LI><LI>Equilibrate the column by adding 10 CV (CV = column bed volume) of buffer W.</LI></OL><H6>Adsorption of Strep-tag proteins to the matrix and column washing</H6><OL><LI>Load your crude extract on the column and keep 50 - 100 µl as a sample for later analysis (CE). Also, collect the flow through as a sample (FT).</LI><LI>Wash the column by adding 5 times with 2.5 CV of buffer W. Collect every fraction as “W1” - “W5”.</LI></OL><H6>Elution of Strep-tag proteins from the matrix</H6><OL><LI>Add 4 times 0.5 CV of buffer E and collect all fractions as “E1” - “E4”.</LI></OL><H6>Analysis</H6><OL><LI>To analyze the purification result, perform SDS-PAGE. For the gel, choose an appropriate polyacrylamide concentration according to the molecular weight of the purified protein. Load 5 µg of CE and FT fractions (determined by the Bradford assay) onto the gel and 15 µl of wash and elution fractions. Stain the gel with Coomassie Brilliant blue.</LI></OL><H6>Dialysis/buffer exchange</H6><OL><LI>To get rid of the desthiobiotin in the protein solution or to exchange the buffer, perform dialysis with the protein fraction of interest. Dialyze o/n against 1,000x volume of buffer W or a desired buffer.</LI></OL><H6>Regeneration of Strep-Tactin column</H6><OL><LI>Wash the column 3 times with 5 CVs Buffer R. The color should change from yellow to red.</LI><LI>The regeneration process is complete when the whole column got the same intensity in red color. When not, add more Buffer R.</LI><LI>Overlay with 2 ml Buffer W and store the column at 4°C.</LI><LI>Prior to the next use remove Buffer R by washing with 2 times 4 CVs Buffer W.</LI></OL><H3>Bradford assay</H3><P>The Bradford reagent can be used to determine the concentration of proteins in solution (Bradford, 1976). The procedure is based on the formation of a complex between the dye Coomassie brilliant blue G-250 and proteins in solution. The protein-dye complex causes a shift in the absorption maximum of the dye from 465 to 595 nm. The absorption is proportional to the amount of protein present in the sample. The assay is suitable for a protein concentration in the range between 0.1 - 1.4 mg/ml of protein using bovine serum albumin (BSA) as the standard protein. Protein solutions or crude extracts with protein concentrations above 1.4 mg/ml have to be diluted to be in the linear range of the assay. The Bradford assay is advantageous because it is fast, the staining does not depend on the protein and the reagent is commercially available.</P><H5>The Material</H5><UL><LI>Bradford solution, 5-fold concentrate</LI><LI>Buffer from protein elution</LI></UL><H5>The Procedure</H5><OL><LI>
          Prepare for each protein mixture 3 samples with different concentrations. If you have many samples to measure, we suggest to do a master mix and distribute 1 ml of the master mix to labeled Eppendorf reaction tubes. Using a master mix, you can measure more samples at the same time and it helps you to avoid errors.
          <DIV class="article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>Reference Sample</TH></TR><TR><TD class="article-table_right">800 ml H<SUB>2</SUB>O</TD><TD class="article-table_right">800 ml H<SUB>2</SUB>O</TD></TR><TR><TD class="article-table_right">0.5, 1.0 &amp; 2.0 ml crude extract</TD><TD class="article-table_right">no protein but equal volume of buffer</TD></TR><TR><TD class="article-table_right">200 ml Bradford solution</TD><TD class="article-table_right">200 ml Bradford solution</TD></TR></TBODY></TABLE></DIV></LI><LI>After mixing the samples with the protein solutions, the mixtures have to be incubated for 5 minutes at room temperature. Transfer the mixtures and the reference sample to 1.5 ml cuvettes and measure the absorption at a wavelength of 595 nm.</LI><LI>Next you can calculate the amount of protein in your sample using the following formula:
          <DIV class="article_direct-footnote">The slope “a” of the calibration curve has been determined using the protein BSA. After several repetitions, this value turned out to be 0.0536 when 1 ml (with c = 1 µg/µl BSA) has been added to 1 ml of the five-fold diluted Bradford solution. Do not forget to subtract the absorption of your blank from the absorption of your sample!</DIV></LI></OL><P>For the ß-Gal assay, add 20 ml of your crude extract to the Bradford-water solution. Use 20 ml of the Z-buffer/LD-Mix as a reference for the measurement. The determined absorption can be directly used together with the formula for the calculation of the Miller units.</P><H3>SDS PAGE</H3><P>Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) is a commonly used method to separate complex mixtures of proteins. In an electric field the negatively charged proteins migrate towards the anode. Moving through the small pores of the gel matrix, the proteins are separated according to their size. Using a marker, the relative molecular weights of proteins within the sample can be determined. Denaturating gels are prepared as described by Laemmli et al. (1970). The gels consist of a stacking and a running gel and are poured to a thickness of 1 mm. The acrylamide in the mixtures builds long chains that are bridged by N, N′- methylenbisacrylamid (mixture 37.5:1), the result is a gel matrix. The polymerization process is induced by radical starter APS and the catalysator TEMED. 

</P><H5>The Material</H5><UL><LI>6x SDS loading dye</LI><LI>10x Running (PAGE) buffer</LI><LI>1.5 M Tris-HCl, pH 8.8</LI><LI>1.5 M Tris-HCl, pH 6.8</LI><LI>TEMED/TMEDA (Tetramethylethylenediamine)</LI><LI>Acrylamide:Bis-Acrylamide (37.5:1, TOXIC!!!)</LI><LI>APS (Ammonium persulfate, 10 %)</LI><LI>SDS (Sodium dodecyl sulfate, 10 %)</LI><LI>Power supply &amp; SDS PAGE devices</LI><LI>Iso-propanol (100 %)</LI></UL><H5>The Procedure</H5><OL><LI>Clean the glass plates, combs and mats with ethanol and assemble the glass plate sandwich. Be sure that the plates are well aligned. Place the plates in the casting frame and tighten them. Place the casting frames into the casting stands.</LI><LI>
          Wear gloves for making the gels because acrylamide is a neurotoxin in its unpolymerized form!!!
          <DIV class="article_table"><P>Running gel for denaturing SDS PAGE (enough for 2 gels)</P><TABLE><TBODY><TR><TH>Components for gel</TH><TH class="article-table_right">15%</TH><TH class="article-table_right">12%</TH><TH class="article-table_right">10%</TH></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">2.3 ml</TD><TD class="article-table_right">3.3 ml</TD><TD class="article-table_right">4.0 ml</TD></TR><TR><TD>Acryl-Bisacrylamide 30%</TD><TD class="article-table_right">5.0 ml</TD><TD class="article-table_right">4.0 ml</TD><TD class="article-table_right">3.3 ml</TD></TR><TR><TD>1.5 M Tris-HCl, pH 8.8</TD><TD class="article-table_right">2.5 ml</TD><TD class="article-table_right">2.5 ml</TD><TD class="article-table_right">2.5 ml</TD></TR><TR><TD> SDS solution 10% </TD><TD class="article-table_right"> 0.1 ml </TD><TD class="article-table_right">0.1 ml</TD><TD class="article-table_right"> 0.1 ml </TD></TR><TR><TD> APS 10%</TD><TD class="article-table_right"> 0.1 ml </TD><TD class="article-table_right"> 0.1 ml </TD><TD class="article-table_right"> 0.1 ml</TD></TR><TR><TD>TEMED </TD><TD class="article-table_right"> 8 µl</TD><TD class="article-table_right">8 µl</TD><TD class="article-table_right"> 8 µl</TD></TR></TBODY></TABLE></DIV><DIV class="article_table"><P>Stacking gel for denaturing SDS PAGE (5%, enough for two gels)</P><TABLE><TBODY><TR><TH>
                  Components for gel </TH><TH class="article-table_right"> 5 % </TH></TR><TR><TD> H<SUB>2</SUB>O </TD><TD class="article-table_right"> 6.83 ml</TD></TR><TR><TD> Acryl-Bisacrylamide 30% </TD><TD class="article-table_right">1.3 ml</TD></TR><TR><TD> 1.5 M Tris-HCl, pH 6.8 </TD><TD class="article-table_right"> 0.87 ml</TD></TR><TR><TD> SDS solution 10% </TD><TD class="article-table_right">0.1 ml </TD></TR><TR><TD> APS 10% </TD><TD class="article-table_right">0.1 ml </TD></TR><TR><TD> TEMED </TD><TD class="article-table_right">10 µl </TD></TR></TBODY></TABLE></DIV></LI><LI>Mix the ingredients of the running gel and pour the solution quickly (gel starts to polymerize once TEMED has been added) into the gap between the glass plates. Leave enough room for the stacking gel – appr. 1 cm below the bottom of the comb´s teeth. After a minute you can overlay the solution with iso-propanol or ethanol. This excludes the access of oxygen and plane the surface.</LI><LI>After polymerization, pour off the iso-propanol and rinse with distilled water. Mix the reagents for the stacking gel and pour it on top of the running gel till the space is completely full. Then insert the comb. No alcohol overlay is needed due to the comb.</LI><LI>After polymerization clamp the gel into the electrophoresis apparatus and fill it with running buffer. Remove the comb carefully and use a pipette or syringe to wash the wells with running buffer.</LI><LI>The protein samples (up to 15 µl) are mixed with SDS loading buffer (2.5 µl) and incubated for 10 min at 95°C.</LI><LI>After heating briefly centrifuge the samples to spin down the condensed water.</LI><LI>Then you can apply the samples on a SDS PAGE with a pipette or Hamilton microliter syringe. Do not forget to add molecular weight marker in one lane.</LI><LI>Run the gel at 80 V until the front has moved into the running gel. Then you can increase the voltage to 120 V. Run the gel until the running front has reached the bottom of the running gel.</LI><LI>Pry the plates apart by using a gel spacer. The gel can then be stained (Coomassie staining, or Silver staining) to visualize the separated proteins or processed further (Western Blotting).</LI></OL><H3>Coomassie staining</H3><P>After separation of proteins by gel electrophoresis, the proteins can be fixed and visualized by different staining procedures. The Coomassie Brilliant blue-staining procedure is a fast and very easy method that is commonly used to detect proteins. Coomassie Brilliant blue is the name of two similar triphenylmethane dyes that were developed for use in the textile industry but are now commonly used for staining proteins in analytical biochemistry. Coomassie brilliant blue G- 250 differs from Coomassie Brilliant blue R-250 by the addition of two methyl groups. The name &quot;Coomassie &quot; is a registered trademark of Imperial Chemical Industries.</P><H5>The Material</H5><UL><LI>Fixation solution</LI><LI>Staining solution</LI><LI>De-staining solution</LI></UL><H5>The Procedure</H5><OL><LI>For staining, place the gel in a plastic container and cover the gel with fixation solution. Shake the gel for 10 min at room temperature. You can also stain the membrane after doing a Western Blot with this technique.</LI><LI>Remove the fixation solution and cover the gel with staining solution. Incubate the gel for 10 – 20 min at room temperature on a shaker.</LI><LI>Remove the staining solution (it can be re-used). De-stain the gel over night with 10 % acetic acid under rigorous shaking at room temperature. Adding a paper towel to the solution speeds up the de-staining process. Boiling the gel in a microwave speeds up de- staining.</LI></OL><H3>Silver staining</H3><P>Silver staining is a very sensitive method for staining polyacrylamide gels (sensitivity 5 - 30 ng/protein band). It is widely use to check the purity of protein extracts and to identify protein-protein interactions. During staining, silver ions build up complexes with glutamate, aspartate and cysteine amino acid residues of the proteins and thereby get reduced to metallic silver. Thus, the intensity of the silver staining depends on the amino acid sequence of the different proteins and can vary considerably. Moreover, in contrast to Coomassie staining, silver staining is not appropriate for protein quantification (Winkler et al. 2007).</P><H5>The Material</H5><UL><LI>FA-Fixation solution</LI><LI>Developer</LI><LI>Impregnator</LI><LI>Thiosulfate solution</LI><LI>Stop solution</LI><LI>Ethanol (50%)</LI></UL><H5>The Procedure</H5><P>The protein bands are stained according to method of Nesterenko et al. (1994). For that purpose, the polyacrylamide gels were incubated on a shaker with the following reagents and in the stated order.</P><DIV class="article_table"><TABLE><TBODY><TR><TH>Step</TH><TH>Solution</TH><TH class="article-table_right">Time</TH></TR><TR><TD>Fixing</TD><TD>Fixation solution</TD><TD class="article-table_right">1–24 h</TD></TR><TR><TD>Washing</TD><TD>EtOH (50%)</TD><TD class="article-table_right">20 min, 3 times</TD></TR><TR><TD>Reduction</TD><TD>Thiosulfate solution</TD><TD class="article-table_right">1.5 min</TD></TR><TR><TD>Washing</TD><TD>deion. H<SUB>2</SUB>O</TD><TD class="article-table_right">20 s, 3 times</TD></TR><TR><TD>Staining</TD><TD>Impregnator</TD><TD class="article-table_right">15–25 min</TD></TR><TR><TD>Washing</TD><TD>deion. H<SUB>2</SUB>O</TD><TD class="article-table_right">20 s, 3 times</TD></TR><TR><TD>Development</TD><TD>Developer</TD><TD class="article-table_right">until sufficient staining</TD></TR><TR><TD>Washing</TD><TD>deion. H<SUB>2</SUB>O</TD><TD class="article-table_right">20 s, 2 times</TD></TR><TR><TD>Stop</TD><TD>Stop solution</TD><TD class="article-table_right">5 min</TD></TR></TBODY></TABLE></DIV><H3>Determination of β-galactosidase activity</H3><P>The “ß-Gal” assay is a very common method to determine the activity of a promoter in vivo (Miller, 1972). To monitor the promoter activity, the promoter of interest has to be fused to the lacZ gene by a simple cloning step. In the next step, the so-called promoter- reporter gene fusion has to be introduced into the chromosome of the <EM> B. subtilis</EM> strain of interest. Transcription from an active promoter results in the synthesis of the β-galactosidase that hydrolyses lactose or chromogenic substrates such as o-nitrophenyl-b-D-galactopyranoside (ONPG). Hydrolysis of ONPG results in the formation of o-nitrophenol, which absorps light with a wavelength of 420 nm. The amount of o-nitrophenol that is formed within a certain time is directly proportional to the activity of a promoter. Each condition should be independently measured at least two times.</P><P><STRONG>Determination of β-galactosidase activity in crude extracts from <EM>B. subtilis</EM></STRONG></P><H5>The Material</H5><UL><LI>LD-Mix (Lysozyme / DNase I)</LI><LI>Z buffer</LI><LI>ONPG</LI><LI>PNPX</LI><LI>1 M Na<SUB>2</SUB>CO<SUB>3</SUB> solution</LI><LI>β-Mercaptoethanol</LI></UL><H5>The Procedure</H5><OL><LI>Propagate a single colony over a whole SP agar plate and incubate the plate over the day at 37°C. You could also propagate a single colony on an SP agar plate in the evening and use the plate at the evening next day to inoculate a pre-culture.</LI><LI>Take a single colony of the <EM> B. subtilis</EM> strain of interest and inoculate 2 - 5 ml C minimal medium supplemented with appropriate antibiotics and effectors (carbon and nitrogen sources). Incubate the pre-cultures over night at 28°C with agitation (220 rpm).</LI><LI>Inoculate 10 ml C minimal medium (plus effectors, carbon and nitrogen sources) supplemented in a 100 ml shake flask to an OD<EM>600</EM> of about 0.1 and grow the cultures to an O<EM>D600</EM> of 0.5 - 0.8 (3 – 5 h) at the desired temperature (usually at 37°C for <EM>B. subtilis</EM>).</LI><LI>Collect the cells from 1.5 - 2.0 ml of the cultures by centrifugation for 5 min at 13,000 rpm and 4°C. Discard the supernatants and store the cell pellets for further treatment at -20°C.</LI><LI>Re-suspend the cells in 400 ml Z buffer (with β-ME and 20 ml LD-Mix in 4 ml Z buffer). Incubate the suspension for 10 min at 37°C to lyse the cells. Keep a minimum of 20 ml Z buffer/LD-Mix for the Bradford assay.</LI><LI>Remove cell debris from the soluble fraction by centrifugation for 2 min at 13,000 rpm and 4°C. Transfer the supernatants into new Eppendorf reaction tubes.</LI><LI>Add 100 ml of the cell free crude extract to 700 ml Z buffer (with β-Me) and pre-incubate the mixture for 5 min at 28°C. 800 ml Z buffer (with β-ME) will serve as the reference.</LI><LI>Start the enzymatic reaction by the addition of 200 ml ONPG to the reaction tubes and note the time.</LI><LI>As soon as the reaction mixture turns yellow, stop the reaction by adding 500 ml Na<SUB>2</SUB>CO<SUB>3</SUB> and note the time. Now it is time to measure the absorption of the sample at a wavelength of l = 420 nm. The “reference sample” will serve as the reference!</LI><LI>The specific β-galactosidase activity in Miller Units/mg protein (MU/mg) can be calculated using the formula below:
          <DIV class="article_direct-footnote">A420: absorption of o-Nitrophenol, Dt: time difference between start and stop of the enzymatic reaction in min, V: volume of crude extract used for the reaction (usually 0.1 ml, see step 3), A595 ∙ 1.7: absorption at l = 595 nm obtained by the Bradford assay.</DIV></LI></OL><P>The method to detect the β-galactosidase activity can also be applied to detect the enzymatic activity of the b-xylosidase in cell free crude extract of<EM> B. subtilis</EM> (Lindner et al., 1994). To detect b-xylosidase activity, ONPG has to be replaced by PNPX.</P><H3>Determination of alpha-amylase activity</H3><P>The amyE gene, encoding the a-amylase can be used to integrate promoter-reporter gene fusions into the chromosome of <EM> B. subtilis </EM>. Integration occurs by double-homologous recombination via the 5’- and 3’- ends that are flanking the promoter-reporter gene fusion in plasmids pAC5, pAC6 and pAC7 (Martin-Verstraete at al., 1994; Stülke et al., 1997; Weinrauch et al., 1991). The integration of the promoter-reporter gene fusion results in the inactivation of the amyE gene and the lack of a-amylase activity. As the functionality of the amyE gene can easily be determined by monitoring the activity of the a-amylase, the lack of this enzymatic activity is a good indication for the correct integration of the promoter-reporter gene fusion.<EM> B. subtilis </EM> transformants that did not integrate the promoter-reporter gene fusion by double-homologous recombination (unstable integration) or not at all are still capable of producing α-amylase.For detecting a-amylase activity, the transformants and a positive control (wild type strain 168) have to be streaked on agar plates containing starch. amyE-positive strains, which synthesis α-amylase hydrolyse the starch and the bacteria are surrounded by a halo when the starch has been stained with Lugol solution. amyE-negative strains do not hydrolyse starch and show no halo formation. </P><H5>The Material</H5><UL><LI>5x Lugol (Iodine/K Iodide) solution</LI><LI>Starch medium agar plates</LI></UL><H5>The Procedure</H5><OL><LI>Take single colonies of the <EM> B. subtilis </EM>transformants and a positive control, and propagate the cells on a line on top of the starch plate. Incubate the plates over night at 37°C.</LI><LI>Dilute the Lugol solution to 1-fold and add 5 ml of the solution on the plate.</LI><LI>The amyE positive bacteria form a halo while those with a disrupted amyE gene should not form a halo.</LI><LI>Put the plates under the fume hood and let the plates dry.</LI></OL><H3><EM>E. coli </EM>XL1 red-aided mutagenesis of plasmid DNA</H3><P>For the analysis of important residues in a protein of interest (e.g. protein interaction surface, catalytic site) a random mutagenesis approach can be very helpful. One method for randomly inserted mutations in a gene is the error-prone PCR. The <EM>E. coli </EM>XL1-Red-aided DNA mutagenesis is a very good tool for random mutagenesis of genes, which have been already inserted into vector. Nevertheless, for very small genes (less than 100 bp) or if multiple changes are desired the error-prone PCR is recommended. The strain XL1-Red is deficient for three of the primary DNA repair pathways. The mutS (error- prone mismatch repair), mutD (deficient in 3´- to 5´-exonuclease of DNA polymerase III) and mutT (unable to hydrolyze 8-oxodGTP) are affected by mutations introduced into the XL1-Red strain. The random mutation rate in this triple mutant strain was measured to be ~ 5,000-fold higher than that of the wild type (Greener &amp; Callahan, 1994; Greener et al., 1997). The mutated DNA repair pathways are especially important during replication. Therefore, it is very important to achieve a certain generation time to allow the emergence of mutations. For high-copy- number plasmids overnight growth (30 generations) results in approximately one base change per 2,000 nucleotides. For a gene with a size of 2 kb every single clone should contain one point mutation and for a gene that is 1 kb half of the clones will harbor a mutation. This should be considered for the experimental setup. Low-copy vectors or vectors containing very small inserts should be propagated for additional generations. The culture may be diluted for continued growth until stationary phase is reached. Mutations achieved by this method were transitions, transversions, and 1-bp insertions. The <EM> E. coli </EM> XL1-Red strain grows extremely slowly in rich media (e.g. LB medium), having a doubling time of ~ 90 – 120 minutes. Additionally, the XL1-Red mutator strain should not be propagated on plates for prolonged periods of time. The rapid mutation rate affects the chromosome, and after prolonged growth, the subsequent colonies are probably not genetically identical to the original strain. The <EM> E. coli </EM> XL1-Red-aided DNA mutagenesis method has been successfully applied to isolate mutant variants of the glutamate dehydrogenase RocG and the glutamine synthetase GlnA in <EM>B. subtilis </EM> (Wray et al., 2006; Gunka et al., 2010).</P><H5>The Material</H5><UL><LI>Competent <EM>E. coli </EM> XL1-Red cells</LI><LI>Liquid LB medium</LI><LI>LB medium agar plates with the appropriate antibiotics</LI><LI>NucleoSpin<SUP>®</SUP> plasmid kit, Macherey &amp; Nagel</LI></UL><H5>The Procedure</H5><OL><LI>Use some material of a cryo culture of the <EM> E. coli </EM>strain XL1-Red to inoculate 4 ml LB liquid medium and grow the culture overnight at 37°C with agitation.</LI><LI>Competent cells of the strain XL1-Red can be prepared according to CaCl<SUB>2</SUB>-method and transformed with the respective plasmid to the standard protocol.</LI><LI>Plate the cells on an appropriate amount of plates to obtain enough clones. The amount of clones that are needed depends on the size of the gene. We have plated 40 LB plates resulting in approximately 80 colonies per plate for the mutagenesis of an insert with a size of about 1.3 kb (rocG gene) into a vector with a size of ca. 6.7 kb (the plasmid pBQ200). Briefly, we used 3,200 clones for the experiment. For a gene that is about 1.3 kb, half of the clones (1,600) should have a mutation in the insert. Statistically, the resulting plasmid pool should contain a mutant for every base in the gene (Gunka et al., 2010).</LI><LI>Re-suspend the colonies from each plate in 1 ml of LB medium, and 100 μl of each suspension can be used to inoculate 100 ml shake flasks containing 10 ml of LB medium supplemented with the appropriate antibiotic.</LI><LI>Grow the cultures for 48 h at 37°C to allow the emergence of mutations. If you use a low-copy plasmid or if you mutate a very small gene (less than 1 kb) the culture can be diluted for continued growth.</LI><LI>Isolate the plasmid DNA from each culture individually (harvest 3 ml of the culture) by using the NucleoSpin® plasmid kit (Macherey &amp; Nagel low-copy plasmid protocol).</LI><LI>An indicator strain (<EM>E. coli </EM> or <EM> B. subtilis</EM>) can be transformed with the plasmid pools and the transformants can be screened for the desired phenotype. We have transformed a <EM>B. subtilis </EM> indicator strain with 40 plasmid pools. Approximately 40,000 clones were screened for a certain phenotype and we have isolated more than 20 interesting plasmids. In this experiment all plasmids contained a single point mutation in the insert (Gunka et al., 2010).</LI></OL><H3>Fluorescence microscopy</H3><P>Fluorescence microscopy is a versatile method to investigate protein localization and/or the activity of a promoter. Here we describe a method that is suitable to monitor the activity of a promoter-fluorophore gene fusion in single cells of <EM>B. subtilis </EM>.</P><H5>The Material</H5><UL><LI>1x PBS buffer, pH 7.5</LI><LI>Microscope, microscopy slides and cover slides</LI><LI>1% Agarose in water</LI></UL><H5>The Procedure</H5><OL><LI>Use a single colony of <EM>B. subtilis</EM> to inoculate 4 ml of LB medium supplemented with the appropriate antibiotics. Incubate the culture over night at 37°C with agitation.</LI><LI>Next morning inoculate 10 ml LB liquid medium supplemented in a shake flask to an approximate OD<SUB>600</SUB> of 0.05. Incubate the shake flask at 37°C and 200 rpm in the dark and collect the cells at the growth phase of interest.</LI><LI>To harvest the cells, transfer 1 ml into a 1.5 ml lightproof (brown) Eppendorf reaction tube and centrifuge the tube for 1 min at 13,000 rpm. Discard the supernatant and Re- suspend the cells in 1x PBS buffer (in a volume (µ{l) = OD<SUB>600</SUB> x (1/10) (µl)). Keep the cell suspension on ice until microscopy.</LI><LI>Preparation of the agarose slide for microscopy: boil the agarose, put some agarose on the slide and place the cover slide on top of the agarose; remove the cover slide from the rigid agarose and pipet your cells on top of the agarose; finally, cover the cells with the cover slide.</LI><LI>Fluorescence images will be obtained with an Axioskop 40 FL fluorescence microscope, equipped with digital camera AxioCam MRm. The AxioVision Rel 4.8.2 software will be used for image processing (Carl Zeiss, Göttingen, Germany). We will use objective of the Neofluar series is used to produce a 100 X primary magnification. The applied filter sets is the EGFP HC-Filterset (BP: 472/30, FT 495, LP 520/35; AHF Analysentechnik, Tübingen, Germany) for GFP detection. Images can be taken with different exposure times (e.g. 2 sec)</LI></OL><H3>Competition experiment</H3><P>Intraspecies competition experiments are an appropriate tool to monitor how mutations of certain genes can affect the fitness of the respective bacterial population under certain conditions. Co-cultivation of two strains which differ in a single locus on the chromosome and which express the fluorophores YFP and CFP is an illustrative way to visualize the competition between those strains. </P><H5>The Material</H5><UL><LI>LB medium</LI><LI>SP agar plates</LI><LI>Minimal medium (or the desired medium)</LI><LI>Glycerol (50%)</LI><LI>NaCl (0.9%)</LI></UL><H5>The Procedure</H5><OL><LI>Use a fresh colony of the <EM> B. subtilis </EM>strains of interest, one strain should express YFP and the other strain CFP, to inoculate 4 ml LB medium with the appropriate antibiotics. Incubate the cultures overnight at 28°C and 220 rpm.</LI><LI>Prepare cryo cultures with an OD<SUB>600</SUB> of 1.0 of the overnight cultures.</LI><LI>Prepare three pre-cultures of the cfp- and yfp-labeled strains. Inoculate 4 ml LB medium containing the appropriate antibiotics with 1 µl of the cryo culture which have an OD<SUB>600 </SUB> of 1.0. Incubate the cultures overnight at 28°C and 220 rpm.</LI><LI>The next morning, measure the OD<SUB>600</SUB> of the pre-cultures of each strain and use those cultures, which have a similar OD<SUB>600</SUB> between 1.0 and 1.5.</LI><LI>In 100 ml shake flasks inoculate 20 ml of minimal medium (or the medium in which you will perform the competition experiment) to an OD<SUB>600</SUB> of 0.05 with each of the two strains you want to analyze. This step ensures that your strains are mixed in a 1:1 ratio at the beginning of the experiment.</LI><LI>Harvest directly 10 ml of the culture (point of time t0) in a 15 ml falcon tube by centrifugation for 10 min at 4,000 g.</LI><LI>Re-suspend the pellet in 500 µl LB medium and transfer the cells to a sterile 1.5 ml Eppendorf cup. Prepare a cryo culture by adding 500 µl of 50% glycerol. Store the cryo culture at -80°C.</LI><LI>Incubate the rest of the culture at 37°C and 220 rpm for 24 h in the dark.</LI><LI>After 7 h (time t7) and after 24 hrs (time t24) take additional samples from the growing culture. Measure the OD<SUB>600</SUB> and prepare with appropriate amounts of cells cryo cultures of OD<SUB>600</SUB> 1.0. Store them at -80°C.</LI><LI>After all the samples are taken, thaw the cryo cultures on ice and dilute the cells in 0.9% saline up to a dilution of 10-3.</LI><LI>For counting of the survivors, plate 100 µl of the 10-3 dilution on SP plates with a sterile glass pipette and incubate the plates over night at 37°C in the dark.</LI><LI>For illustrative figures, you can additionally spot 10 µl of the 10-4 dilution of the samples from the different time points next to each other on a SP plate, which you incubate over night at 37°C in the dark.</LI><LI>The next day, you can count the grown colonies (from step 11.) under the stereo fluorescence microscope. Using the YFP and CFP filter set, the survivors of the yfp- labeled and cfp-labeled strain, respectively, can be made visible and you can exactly determine the outcome of the competition experiment. (Dividing the plate with a black pen into four parts and labelling the already counted colonies makes counting easier!) With respect to the total number of colonies on the plate, calculate the percentage of „blue“ and „yellow“ colonies.</LI><LI>To make the clonal composition of the cell population visible at a glance, analyze the spotted samples. Use the cold light source of the stereo fluorescence microscope to bring the spot into focus. Switch the filter to the CFP-filter and take a picture. Do the same for the YFP-filter without moving the plate.</LI><LI>With Adobe Photoshop construct merged pictures of the pictures taken from the different spots.</LI></OL><P><STRONG>To exclude that expression of the fluorophore-encoding genes influences the fitness of the respective strain:</STRONG></P><OL type="a"><LI>Perform the experiment with inversely labeled strains (the result should be comparable to the actual experiment)</LI><LI>Perform the experiment with isogenic strains labeled with cfp and yfp (no negative effect should be detectable)</LI></OL><DIV class="article_head-line"><H2>Recipes for media, buffers &amp; solutions</H2></DIV><H3>Complex media</H3><H5>LB medium</H5><UL><LI>10 g Tryptone</LI><LI>5 g Yeast extract</LI><LI>10 g NaCl</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>SP medium</H5><UL><LI>8 g Nutrient broth</LI><LI>0.25 g MgSO<SUB>4</SUB>·7 H<SUB>2</SUB>O</LI><LI>1 g KCl</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI><LI><EM>After autoclaving do not forget to add:</EM></LI><LI>1 mL 0.5 M CaCl<SUB>2</SUB></LI><LI>1 mL 10 mM MnCl<SUB>2</SUB></LI><LI>2 mL Iron Ferric Ammonium Citrate, 2.2 mg/mL</LI></UL><H5>Starch medium</H5><UL><LI>7.5 g Nutrient broth</LI><LI>5 g Starch</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>SOB-Mg medium</H5><UL><LI>20 g Tryptone</LI><LI>5 g Yeast extract</LI><LI>0.58 g NaCl</LI><LI>0.186 g KCl</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>SOB medium</H5><UL><LI>SOB-Mg medium</LI><LI>10 mM MgCl<SUB>2</SUB></LI><LI>10 mM MgSO<SUB>4</SUB></LI></UL><H3>Minimal media</H3><H5>CS-glucose medium</H5><UL><LI>10 mL 5x C salts</LI><UL><LI>20 g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>80 g K<SUB>2</SUB>HPO<SUB>4</SUB>·3 H<SUB>2</SUB>O</LI><LI>16.5 g (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></LI></UL><LI>500 µL Tryptophane</LI><LI>500 µL III' salts</LI><UL><LI>0.232 g MnSO<SUB>4</SUB>·4 H<SUB>2</SUB>O</LI><LI>12.3 g MgSO<SUB>4</SUB>·7 H<SUB>2</SUB>O</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><LI>500 µL Glucose 50%</LI><LI>500 µL Iron Ferric Ammonium Citrate, 2.2 mg/mL</LI><LI>1 mL Na-Succinate</LI><LI>Ad to 50 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>10x MN medium</H5><UL><LI>136 g K<SUB>2</SUB>HPO<SUB>4</SUB>·3 H<SUB>2</SUB>O</LI><LI>60 g KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>10 g sodium citrate · 2 H<SUB>2</SUB>O</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>MNGE medium</H5><UL><LI>1 mL 10x MN medium</LI><LI>400 µL Glucose (50%)</LI><LI>50 µL Potassium glutamate 40%)</LI><LI>50 µL Iron Ferric Ammonium Citrate, 2.2 mg/mL</LI><LI>100 µL Tryptophane</LI><LI>30 µL 1 M MgSO<SUB>4</SUB></LI><LI>8.37 mL deionized H<SUB>2</SUB>O</LI><LI>+/- 100 µL CAA (10%)</LI></UL><H3>Buffers</H3><H5>50x TAE buffer</H5><UL><LI>242 g Tris-base</LI><LI>57.1 mL Acetic acid (100%)</LI><LI>100 mL 0.5 M EDTA, pH 8.0</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>10x CCR/MMR buffer</H5><UL><LI>200 mM Tris-HCl, pH 8.5</LI><LI>30 mM MgCl<SUB>2</SUB></LI><LI>0.5 M KCl</LI><LI>5 mM NAD<SUP>+</SUP></LI><LI>0.4 mg/mL BSA</LI></UL><H5>Buffer W</H5><UL><LI>12.114 g Tris-base</LI><LI>8.77 g NaCl</LI><LI>0.3722 g Na<SUB>2</SUB>EDTA · 2 H<SUB>2</SUB>O</LI><LI><EM>Adjust the pH with HCl to 8.0</EM></LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>Buffer E</H5><UL><LI>0.027 g D-Desthiobiotin (25 mM)</LI><LI>Add 50 mL buffer W</LI></UL><H5>Fixation solution</H5><UL><LI>10% Acetic acid</LI><LI>45% Methanol (toxic!)</LI></UL><H5>Staining solution</H5><UL><LI>0.5% Coomassie Brillant blue</LI><LI>10% Acetic acid</LI><LI>45% Methanol (toxic!)</LI></UL><H5>De-staining solution</H5><UL><LI>5% Acetic acid</LI><LI>20% Ethanol</LI></UL><H5>10x PAGE buffer</H5><UL><LI>144 g L-Glycine</LI><LI>30 g Tris-base</LI><LI>10 g SDS</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H3>Solutions</H3><H5>IPTG</H5><UL><LI>2.38 g IPTG</LI><LI>Ad to 10 mL with deionized H<SUB>2</SUB>O</LI><LI>Sterilization by filtration, store at -20°C in aliquots.</LI></UL><H5>5y DNA loading dye</H5><UL><LI>5 mL Glycerol (100%)</LI><LI>4.5 mL deionized H<SUB>2</SUB>O</LI><LI>0.2 mL 50x TAE</LI><LI>0.01 g Bromphenol blue</LI><LI>0.01 g Xylene cyanol</LI></UL><H5>6x SDS loading dye</H5><UL><LI>2.5 mL Tris-HCl, pH 6.8</LI><LI>600 µL β-Mercaptoethanol</LI><LI>1.2 g SDS</LI><LI>7.5 mL Glycerol</LI><LI>150 µL Bromphenol blue (2% in EtOH)</LI></UL><H5>Expression mix</H5><UL><LI>500 µL Yeast extract (5%)</LI><LI>250 µL CAA (10%)</LI><LI>50 µL Tryptophane (5 mg/mL)</LI><LI>250 µL sterile H<SUB>2</SUB>O</LI><LI><STRONG>Suitable for 10 samples</STRONG></LI></UL><H5>5x Lugol solution</H5><UL><LI>100 g K-iodide</LI><LI>50 g iodine</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5>LD-Mix</H5><UL><LI>100 mg Lysozyme</LI><LI>10 mg DNase I</LI><LI>Ad to 10 mL with deionized H<SUB>2</SUB>O</LI><LI>Store 500 µL aliquots at -20°C</LI></UL><H5>Z buffer</H5><UL><LI>60 mM Na<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI>40 mM NaH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>1 mM KCl</LI><LI>1 mM MgSO<SUB>4</SUB></LI><LI>50 mM β-Mercaptoethanol (toxic!)</LI></UL><H3>Chromogenic substrates</H3><H5>X-Gal</H5><UL><LI>80 mg 5-Brom-4-chor-3-indoxyl-β-D-galactopyranosid</LI><LI>Add 1 mL DMF(N,N-Dimethylformamide) or DMSO</LI></UL><H5>ONPG</H5><UL><LI>4 mg <EM>o</EM>-Nitrophenyl-β-D-Galactopyranoside (ONPG)</LI><LI>Add 1 mL Z buffer without β-Mercaptoethanol</LI></UL></DIV></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>