<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Notebook_May skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Notebook/May</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="notebook-item"><DIV class="notebook-head"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#"><SPAN class="tocnumber">1</SPAN></A></LI><LI class="toclevel-1"><A href="#_2"><SPAN class="tocnumber">2</SPAN></A></LI><LI class="toclevel-1"><A href="#Identification_of_upregulated_proteins_in_presence_of_glyphosate_in_Bacillus_subtilis"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Identification of upregulated proteins in presence of glyphosate in Bacillus subtilis</SPAN></A></LI><LI class="toclevel-1"><A href="#_3"><SPAN class="tocnumber">4</SPAN></A></LI><LI class="toclevel-2"><A href="#Materials_for_the_Bradford_assay:"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Materials for the Bradford assay:</SPAN></A></LI><LI class="toclevel-2"><A href="#Procedure:"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Procedure:</SPAN></A></LI><LI class="toclevel-2"><A href="#The_protein_expression_was_then_further_analysed_by_SDS-PAGE_and_subsequent_silver_staining."><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">The protein expression was then further analysed by SDS-PAGE and subsequent silver staining.</SPAN></A></LI><LI class="toclevel-3"><A href="#The_Material"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">The Material</SPAN></A></LI><LI class="toclevel-3"><A href="#The_procedure"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">The procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Components_for_gel_15.C2.A0.25.2C_12.C2.A0.25.2C_10.C2.A0.25"><SPAN class="tocnumber">4.3.3</SPAN><SPAN class="toctext">Components for gel 15 %, 12 %, 10 %</SPAN></A></LI><LI class="toclevel-3"><A href="#Stacking_gel_for_denaturing_SDS_PAGE_.285.C2.A0.25.3B_enough_for_two_gels.29"><SPAN class="tocnumber">4.3.4</SPAN><SPAN class="toctext">Stacking gel for denaturing SDS PAGE (5 %; enough for two gels)</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Silver_staining"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Silver staining</SPAN></A></LI><LI class="toclevel-2"><A href="#The_material_2"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">The material</SPAN></A></LI><LI class="toclevel-2"><A href="#The_procedure_2"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">The procedure</SPAN></A></LI><LI class="toclevel-2"><A href="#Results"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Identification_of_novel_mutations_in_B..C2.A0subtilis_168"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Identification of novel mutations in B. subtilis 168</SPAN></A></LI><LI class="toclevel-1"><A href="#Amplification_of_the_gltT_gene"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Amplification of the gltT gene</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_amplification_and_cloning_experiments_for_reporter_constructs"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Plasmid amplification and cloning experiments for reporter constructs</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_3"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparing_competent_E..C2.A0coli_cells.2FPtrpE.2FP_amplification"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Preparing competent E. coli cells/PtrpE/P amplification</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_competent_E..C2.A0coli_strains_DH5.CE.B1_and_XL1-Blue"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Preparation of competent E. coli strains DH5α and XL1-Blue</SPAN></A></LI><LI class="toclevel-3"><A href="#The_Material_3"><SPAN class="tocnumber">9.1.1</SPAN><SPAN class="toctext">The Material</SPAN></A></LI><LI class="toclevel-3"><A href="#The_Procedure_3"><SPAN class="tocnumber">9.1.2</SPAN><SPAN class="toctext">The Procedure</SPAN></A></LI><LI class="toclevel-1"><A href="#New_Primers_for_gltT_and_sequencing"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">New Primers for gltT and sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_4"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Test_transformation_of_the_new_competent_E..C2.A0coli_cells"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Test transformation of the new competent E. coli cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Evaluation_of_the_test_transformation"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Evaluation of the test transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Inoculation_of_transformed_strains"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Inoculation of transformed strains</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparation_of_the_plasmids_for_further_work"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Preparation of the plasmids for further work</SPAN></A></LI><LI class="toclevel-2"><A href="#The_Material_4"><SPAN class="tocnumber">14.1</SPAN><SPAN class="toctext">The Material</SPAN></A></LI><LI class="toclevel-2"><A href="#The_Procedure_4"><SPAN class="tocnumber">14.2</SPAN><SPAN class="toctext">The Procedure</SPAN></A></LI><LI class="toclevel-2"><A href="#Reaction_mixtures_to_digest_the_fragments"><SPAN class="tocnumber">14.3</SPAN><SPAN class="toctext">Reaction mixtures to digest the fragments</SPAN></A></LI><LI class="toclevel-2"><A href="#Reaction_mixtures_to_digest_the_plasmids"><SPAN class="tocnumber">14.4</SPAN><SPAN class="toctext">Reaction mixtures to digest the plasmids</SPAN></A></LI><LI class="toclevel-3"><A href="#pGP172"><SPAN class="tocnumber">14.4.1</SPAN><SPAN class="toctext">pGP172</SPAN></A></LI><LI class="toclevel-3"><A href="#pBQ200"><SPAN class="tocnumber">14.4.2</SPAN><SPAN class="toctext">pBQ200</SPAN></A></LI><LI class="toclevel-3"><A href="#pAC7"><SPAN class="tocnumber">14.4.3</SPAN><SPAN class="toctext">pAC7</SPAN></A></LI><LI class="toclevel-1"><A href="#Construction_of_plasmids"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Construction of plasmids</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_PtrpP_into_pAC7"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Ligation of PtrpP into pAC7</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_PtrpE_into_pAC7"><SPAN class="tocnumber">15.2</SPAN><SPAN class="toctext">Ligation of PtrpE into pAC7</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_aroE_1_into_pGP172"><SPAN class="tocnumber">15.3</SPAN><SPAN class="toctext">Ligation of aroE_1 into pGP172</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_aroE_2_into_pBQ200"><SPAN class="tocnumber">15.4</SPAN><SPAN class="toctext">Ligation of aroE_2 into pBQ200</SPAN></A></LI><LI class="toclevel-2"><A href="#Ligation_of_Ptrp_into_pAC7"><SPAN class="tocnumber">15.5</SPAN><SPAN class="toctext">Ligation of Ptrp into pAC7</SPAN></A></LI><LI class="toclevel-2"><A href="#Re-Ligation_of_vector_pAC7"><SPAN class="tocnumber">15.6</SPAN><SPAN class="toctext">Re-Ligation of vector pAC7</SPAN></A></LI><LI class="toclevel-2"><A href="#Re-Ligation_of_vector_pGP172"><SPAN class="tocnumber">15.7</SPAN><SPAN class="toctext">Re-Ligation of vector pGP172</SPAN></A></LI><LI class="toclevel-2"><A href="#Re-Ligation_of_vector_pBQ200"><SPAN class="tocnumber">15.8</SPAN><SPAN class="toctext">Re-Ligation of vector pBQ200</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_of_DH5.CE.B1"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Transformation of DH5α</SPAN></A></LI></DIV><P class="notebook-head_date">02.05.18</P></DIV><DIV class="notebook-content"><P>Cryo cultures of strains 168, BP233, BP234 were prepared with 90% LB medium and 10% DMSO. The cells were frozen and stored at -80°C.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">03.05.18</P></DIV><DIV class="notebook-content"><P>Strains BP233, BP235 and 168 (WT) were incubated in 4 ml LB medium for 4 h at 37°C and 220 rpm. 2 ml of the culture was harvested, and the cells were washed in 1xC-salts and then resuspended in 300 µl 1xC-salts. BP233 and BP235 were plated on CS-glucose with and without 10 mM glyphosate.</P><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><P>As seen in this figure, both strains grow well on CS-glucose without glyphosate. Strain BP233 also forms a lawn on plates with 10mM glyphosate, while the wildtype does not form a lawn, but single colonies, which could be suppressor mutants. This leads to the hypothesis that GltT, which is a glutamate transporter, transports glyphosate into the cell. The strain BP235 was not stable, so it was constructed again!
    </P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Identification_of_upregulated_proteins_in_presence_of_glyphosate_in_Bacillus_subtilis">Identification of upregulated proteins in presence of glyphosate in Bacillus subtilis</SPAN></H3><P class="notebook-head_date">07.05.18</P></DIV><DIV class="notebook-content"><OL><LI><P>The two <EM>B. subtilis</EM> strains 168 and the tryptophan prototroph strain SP1 were grown at different glyphosate concentrations. To identify upregulated protein expression in the presence of glyphosate.</P></LI></OL><P>For cultivation, the two strains were inoculated in 4 mL LB medium and incubated overnight at 37°C (200 rpm) The overnight cultures were transferred into 4 mL fresh LB medium and grown for 2 h at 28°C (200 rpm).
    </P><DIV class="notebook-table article_table"><P>CS-Glucose medium was prepared with and without tryptophan</P><TABLE><TBODY><TR><TH>Components</TH><TH class="article-table_right">Volume (mL)</TH></TR><TR><TD>5 × C salts</TD><TD class="article-table_right">10</TD></TR><TR><TD>Iron Ferric Ammonium Citrate (CAF; 2.2 mg/mL)</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>III´ salts</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>Sodium Succinate (30%)</TD><TD class="article-table_right">1</TD></TR><TR><TD>50% glucose</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>(Tryptophan</TD><TD class="article-table_right">0.5)</TD></TR><TR class="article_table_hline"><TD><STRONG>Ad to 50 mL with sterile water</STRONG></TD><TD class="article-table_right">100</TD></TR></TBODY></TABLE></DIV><P><EM>B. subtilis</EM> 168 and <EM>B. subtilis</EM> SP1 were transferred into CS-Glucose medium containing increasing glyphosate concentrations to an OD<SUB>600</SUB> of 0.1. OD measurement was carried out every 2 hours until the cells reached an OD<SUB>600</SUB> of ∼2. The cells were harvested by centrifugation for 1 min at 13000 rpm and stored at -20°C. </P><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><P>Growth curves of the wild type SP1 and the 168 mutant, auxotrophic for tryptophan</P></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">14.05.18</P></DIV><DIV class="notebook-content"><P>Proteins were isolated from cultures by lysing the cells with 50 µL lysis buffer and a <STRONG>Bradford assay</STRONG> was carried out to determine the protein concentration.</P><H5><SPAN class="mw-headline" id="Materials_for_the_Bradford_assay:">Materials for the Bradford assay:</SPAN></H5><UL><LI>Bradford solution, 5-fold concentration</LI><LI>buffer from protein elution</LI></UL><H5><SPAN class="mw-headline" id="Procedure:">Procedure:</SPAN></H5><OL><LI><P>Prespare for each portein mixture 3 samples with different concentrations.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH class="article-table_right">Reference sample</TH></TR><TR><TD>800 µL H<SUB>2</SUB>O</TD><TD>800 µL H<SUB>2</SUB>O</TD></TR><TR><TD>0.5, 1.0 &amp; 2.0 µL crude extract</TD><TD>no protein but equal volume of buffer</TD></TR><TR><TD>200 µL Bradford Solution</TD><TD>200 µL Bradford Solution</TD></TR><TR><TD>Sodium Succinate (30%)</TD><TD class="article-table_right">1</TD></TR><TR><TD>50% glucose</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>(Tryptophan</TD><TD class="article-table_right">0.5)</TD></TR><TR class="article_table_hline"><TD><STRONG>Ad to 50 mL with sterile water</STRONG></TD><TD class="article-table_right">100</TD></TR></TBODY></TABLE></DIV></LI><LI><P>After mixing the samples with the protein solutions, the mixtures have to be incubated for 5 minutes at room temperature. Transfer the mixtures and the reference sample to 1.5 mL cuvettes and measure the absorption at a wavelength of 595 nm.</P></LI><LI><P>Calculate the amount of protein in your sample using the following formula:</P><DIV class="article_formula"><PRE>&lt;img class=&quot;article_formula&quot;
              src=&quot;&quot;&gt;</PRE></DIV></LI></OL><DIV class="notebook-table article_table"><P>Protein concentration:</P><TABLE><TBODY><TR><TH>Sample</TH><TH class="article-table_right">Protein concentration (µg/mL)</TH></TR><TR><TD>168 in 0 mM Glyphosate</TD><TD class="article-table_right">0.375</TD></TR><TR><TD>168 in 0.5 mM Glyphosate</TD><TD class="article-table_right">1.049</TD></TR><TR><TD>168 in 1.5 mM Glyphosate</TD><TD class="article-table_right">1.049</TD></TR><TR><TD>SP1 in 0 mM</TD><TD class="article-table_right">0.159</TD></TR><TR><TD>SP1 in 0.5 mM</TD><TD class="article-table_right">0.496</TD></TR><TR><TD>SP1 in 1.5 mM</TD><TD class="article-table_right">1.4045</TD></TR></TBODY></TABLE></DIV><H5><SPAN class="mw-headline" id="The_protein_expression_was_then_further_analysed_by_SDS-PAGE_and_subsequent_silver_staining.">The protein expression was then further analysed by SDS-PAGE and subsequent silver staining.</SPAN></H5><H6><SPAN class="mw-headline" id="The_Material">The Material</SPAN></H6><UL><LI>6× SDS loading dye (98 C)</LI><LI>6× SDS loading dye (98 C)</LI><LI>10× Running (PAGE) buffer (98 S)</LI><LI>1.5 M Tris-HCl, pH 8.8 (95 B)</LI><LI>1.56 M Tris-HCl, pH 6.8 (95 B)</LI><LI>TEMED/TMEDA (Tetramethylethylenediamine)</LI><LI>Acrylamide:Bis-Acrylamide (37.5:1, TOXIC!!!)</LI><LI>APS (Ammonium persulfate, 10 %)</LI><LI>SDS (Sodium dodecyl sulfate, 10 %)</LI><LI>Power supply &amp; SDS PAGE devices</LI><LI>Iso-propanol (100 %)</LI></UL><H6><SPAN class="mw-headline" id="The_procedure">The procedure</SPAN></H6><OL><LI>Clean the glass plates, combs and mats with ethanol and assemble the glass plate sandwich. Be sure that the plates are well aligned. Place the plates in the casting frame and tighten them. Place the casting frames into the casting stands.
      </LI><LI>Wear gloves for making the gels because acrylamide is a neurotoxin in its unpolymerized form!!! Running gel for denaturing SDS PAGE (enough for 2 gels).
        <H6><SPAN class="mw-headline" id="Components_for_gel_15.C2.A0.25.2C_12.C2.A0.25.2C_10.C2.A0.25">Components for gel 15 %, 12 %, 10 %</SPAN></H6></LI><LI>H<SUB>2</SUB>O 2.3 mL 3.3 mL 4.0 mL</LI><LI>Acryl-Bisacrylamide 30 % 5.0 mL 4.0 mL 3.3 mL</LI><LI>1.5 M Tris-HCl, pH 8.8 2.5 mL 2.5 mL 2.5 mL</LI><LI>SDS solution 10 % 0.1 mL 0.1 mL 0.1 mL</LI><LI>APS 10 % 0.1 mL 0.1 mL 0.1 mL</LI><LI>TEMED 8 µl 8 µl 8 µl</LI><H6><SPAN class="mw-headline" id="Stacking_gel_for_denaturing_SDS_PAGE_.285.C2.A0.25.3B_enough_for_two_gels.29">Stacking gel for denaturing SDS PAGE (5 %; enough for two gels)</SPAN></H6><UL><LI>H<SUB>2</SUB>O 6.83 mL</LI><LI>Acryl-Bisacrylamide 30 % 1.3 mL</LI><LI>1.5 M Tris-HCl, pH 6.8 0.87 mL</LI><LI>SDS solution 10 % 0.1 mL</LI><LI>APS 10 % 0.1 mL</LI><LI>TEMED 10 µl</LI></UL><LI>Mix the ingredients of the running gel and pour the solution quickly (gel starts to polymerize once TEMED has been added) into the gap between the glass plates. Leave enough room for the stacking gel – appr. 1 cm below the bottom of the comb´s teeth. After a minute you can overlay the solution with iso-propanol or ethanol. This excludes the access of oxygen and plane the surface
      </LI><LI>After polymerization, pour off the iso-propanol and rinse with distilled water. Mix the reagents for the stacking gel and pour it on top of the running gel till the space is completely full. Then insert the comb. No alcohol overlay is needed due to the comb.</LI><LI>After polymerization clamp the gel into the electrophoresis apparatus and fill it with running buffer. Remove the comb carefully and use a pipette or syringe to wash the wells with running buffer.</LI><LI>The protein samples (up to 15 µl) are mixed with SDS loading buffer (2.5 µl) and incubated for 10 min at 95°C.</LI><LI>After heating briefly centrifuge the samples to spin down the condensed water.</LI><LI>Then you can apply the samples on a SDS PAGE with a pipette or Hamilton microliter syringe. Do not forget to add molecular weight marker in one lane.</LI><LI>Run the gel at 80 V until the front has moved into the running gel. Then you can increase the voltage to 120 V. Run the gel until the running front has reached the bottom of the running gel.</LI><LI>Pry the plates apart by using a gel spacer. The gel can then be stained (Coomassie staining, method 56 or Silver staining, method 57) to visualize the separated proteins or processed further (Western Blotting, method 58).</LI></OL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Silver_staining">Silver staining</SPAN></H3><P class="notebook-head_date">15.05.18</P></DIV><DIV class="notebook-content"><H6><SPAN class="mw-headline" id="The_material_2">The material</SPAN></H6><UL><LI>FA-Fixation solution (98 L)</LI><LI>Developer (98 M)</LI><LI>Impregnator (98 N)</LI><LI>Thiosulfate solution (98 O)</LI><LI>Stop solution (98 P)</LI><LI>Ethanol (50 %)</LI></UL><H6><SPAN class="mw-headline" id="The_procedure_2">The procedure</SPAN></H6><P>The protein bands are stained according to method of Nesterenko et al. (1994). For that purpose, the polyacrylamide gels were incubated on a shaker with the following reagents and in the stated order.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Step</TH><TH>Solution</TH><TH>Time</TH></TR><TR><TD>Fixing</TD><TD>Fixation solution</TD><TD class="article-table_right">1–24 h </TD></TR><TR><TD>Washing</TD><TD>EtOH (50 %)</TD><TD class="article-table_right">20 min, 3×</TD></TR><TR><TD>Reduction</TD><TD>Thiosulfate solution</TD><TD class="article-table_right">1.5 min</TD></TR><TR><TD>Washing</TD><TD>deion. H<SUB>2</SUB>O</TD><TD class="article-table_right">20 s, 3×</TD></TR><TR><TD>Staining</TD><TD>Impregnator</TD><TD class="article-table_right">15–25 min</TD></TR><TR><TD>Washing</TD><TD>deion. H<SUB>2</SUB>O</TD><TD class="article-table_right">20 s, 3×</TD></TR><TR><TD>Development</TD><TD>Developer</TD><TD class="article-table_right">until sufficient staining</TD></TR><TR><TD>Washing</TD><TD>deion. H<SUB>2</SUB>O</TD><TD class="article-table_right">20 s, 2×</TD></TR><TR><TD>Stop</TD><TD>solution</TD><TD class="article-table_right">5 min</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Results">Results</SPAN></H6><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><P>&amp;rightarrow; No significant difference in protein expression could be detected.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Identification_of_novel_mutations_in_B..C2.A0subtilis_168">Identification of novel mutations in <EM>B. subtilis</EM> 168</SPAN></H3><P class="notebook-head_date">16.05.18</P></DIV><DIV class="notebook-content"><P>For the identification of novel mutations in the gltT transporter, the wild type strain was grown on 40 mM glyphosate. Emerging mutations were further analysed by isolation of the complete DNA and amplification of the transporter via PCR. The genomic DNA was isolated with the Kit “peqGOLD Bacterial DNA Kit” from peqlab. The resulting sequences were then screened for novel mutations.</P><P>For the identification of novel mutations in the gltT transporter, the wild type strain was grown on 40 mM glyphosate. Emerging mutations were further analysed by isolation of the chromosomal DNA and amplification of the transporter via PCR. The genomic DNA was isolated with the Kit “peqGOLD Bacterial DNA Kit” from peqlab.</P><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Mutant names</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD>iGEM18</TD><TD class="article-table_right">15.2</TD></TR><TR><TD>iGEM19</TD><TD class="article-table_right">44.5</TD></TR><TR><TD>iGEM20</TD><TD class="article-table_right">57.4</TD></TR><TR><TD>iGEM21</TD><TD class="article-table_right">48.3</TD></TR><TR><TD>iGEM22</TD><TD class="article-table_right">45.3</TD></TR><TR><TD>iGEM23</TD><TD class="article-table_right">54.3</TD></TR><TR><TD>iGEM24</TD><TD class="article-table_right">44.7</TD></TR><TR><TD>iGEM25</TD><TD class="article-table_right">31.9</TD></TR><TR><TD>iGEM26</TD><TD class="article-table_right">29.7</TD></TR><TR><TD>iGEM27</TD><TD class="article-table_right">80.7</TD></TR><TR><TD>iGEM28</TD><TD class="article-table_right">58.8</TD></TR><TR><TD>iGEM29</TD><TD class="article-table_right">51.9</TD></TR><TR><TD>iGEM30</TD><TD class="article-table_right">57.7</TD></TR><TR><TD>iGEM31</TD><TD class="article-table_right">/</TD></TR><TR><TD>iGEM32</TD><TD class="article-table_right">56.1</TD></TR><TR><TD>iGEM33</TD><TD class="article-table_right">62.2</TD></TR><TR><TD>iGEM34</TD><TD class="article-table_right">42.5</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Amplification_of_the_gltT_gene">Amplification of the <EM>gltT</EM> gene</SPAN></H3><P class="notebook-head_date">17.05.18</P></DIV><DIV class="notebook-content"><P>We amplified the <EM>gltT</EM> gene from <EM>B. subtilis</EM> with a PCR run with the following recipe.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Component</TH><TH>Volume in [µL]</TH></TR><TR><TD>10x <EM>Taq</EM> buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>dNTPs (12.5 mM)</TD><TD class="article-table_right">2</TD></TR><TR><TD>fwd Primer: iGEM2018_15</TD><TD class="article-table_right">2</TD></TR><TR><TD>rev Primer: iGEM2018_16</TD><TD class="article-table_right">2</TD></TR><TR><TD><EM>B. subtilis</EM> chromosomal DNA</TD><TD class="article-table_right">1</TD></TR><TR><TD>Dream <EM>Taq</EM> polymerase</TD><TD class="article-table_right">0.25</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">37.75</TD></TR><TR class="article_table_hline"><TD><STRONG>Total</STRONG></TD><TD class="article-table_right">50</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Results_2">Results</SPAN></H6><P>In the following figures are the results of the PCR shown.</P><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><DIV class="article_multpicture-paragraph"><P>Amplified <EM>gltT</EM> gene from 168, iGEM19/20.</P><P>Amplified <EM>gltT</EM> gene from iGEM30/32/33/34.</P></DIV></DIV><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Amplified <EM>gltT</EM> gene from 168, iGEM18/21/22/23/24/25/26/27/28/29.</P></DIV><P>&amp;rightarrow; iGEM21 and iGEM26 did not show any PCR products. Therefore, no sequence could be further analyzed.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Plasmid_amplification_and_cloning_experiments_for_reporter_constructs">Plasmid amplification and cloning experiments for reporter constructs</SPAN></H3><P class="notebook-head_date">18.05.18</P></DIV><DIV class="notebook-content"><P>To produce a reporter system, we needed to identify promoters from genes that are induced upon glyphosate treatment. Our hypothesis is that the promoter of <EM>trpE</EM> and <EM>trpP</EM> are induced by glyphosate, as described in the design section (TO DO, link hier einfügen!!!). Therefore, <EM>PtrpE</EM> as well as two variants of the EPSP-synthase (encoded by <EM>aroE</EM> were amplified by PCR using primers iGEM2018_25/26 and iGEM2018_27/28, respectively.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Component</TH><TH>Volume in [µL]</TH></TR><TR><TD>5x HF buffer</TD><TD class="article-table_right">10</TD></TR><TR><TD>dNTPs (12.5 mM)</TD><TD class="article-table_right">2</TD></TR><TR><TD>fwd Primer: iGEM2018_25 or iGEM2018_27</TD><TD class="article-table_right">2</TD></TR><TR><TD>rev Primer: iGEM2018_26 or iGEM2018_28</TD><TD class="article-table_right">2</TD></TR><TR><TD><EM>B. subtilis</EM> chromosomal DNA</TD><TD class="article-table_right">1</TD></TR><TR><TD><EM>Phusion</EM> DNA polymerase</TD><TD class="article-table_right">0.25</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">32.75</TD></TR><TR class="article_table_hline"><TD><STRONG>Total</STRONG></TD><TD class="article-table_right">50</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Results_3">Results</SPAN></H6><P>In the following figure are the results of the PCR shown.</P><DIV class="article_picture" style="width:30%;"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Lane 1: <EM>aroE_1</EM>. Lane 2: <EM>PtrpE</EM>. Lane 3: <EM>aroE_2</EM>.</P></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparing_competent_E..C2.A0coli_cells.2FPtrpE.2FP_amplification">Preparing competent <EM>E. coli</EM> cells/<EM>PtrpE/P</EM> amplification</SPAN></H3><P class="notebook-head_date">22.05.18</P></DIV><DIV class="notebook-content"><P>The PCR amplification was repeated including <EM>PtrpP</EM>. The following primer combinations were used.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Primers</TH><TH>Product</TH></TR><TR><TD>iGEM2018_1 + iGEM2018_2</TD><TD><EM>aroE_1</EM></TD></TR><TR><TD>iGEM2018_23 + iGEM2018_24</TD><TD><EM>PtrpP</EM></TD></TR><TR><TD>iGEM2018_25 + iGEM2018_26</TD><TD><EM>PtrpE</EM></TD></TR><TR><TD>iGEM2018_27 + iGEM2018_28</TD><TD><EM>aroE_2</EM></TD></TR><TR class="article_table_hline"><TD><STRONG>Template</STRONG></TD><TD><STRONG>chromosomal DNA from strain iGEM21</STRONG></TD></TR></TBODY></TABLE></DIV><P>The following figure shows the results of the PCR.</P><DIV class="article_picture" style="width:50%;"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Lane 1: <EM>aroE_1</EM>. Lane 2: <EM>PtrpP</EM>. Lane 3: <EM>PtrpP</EM>. Lane 4: <EM>PtrpE</EM>. Lane 5: <EM>aroE_2</EM>.</P></DIV><P>Samples from today in addition to the samples on 18.05.18 were purified using the QIAquick PCR Purification Kit. The DNA was eluted in 35 µL H<SUB>2</SUB>O.Concentration of the purified DNA was measured at the Nanodrop device.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD><EM>aroE_1</EM></TD><TD class="article-table_right">90.1</TD></TR><TR><TD><EM>aroE_2</EM></TD><TD class="article-table_right">68.1</TD></TR><TR><TD><EM>PtrpE_1</EM></TD><TD class="article-table_right">89.5</TD></TR><TR><TD><EM>PtrpE_2</EM></TD><TD class="article-table_right">67.8</TD></TR><TR><TD><EM>PtrpP_1</EM></TD><TD class="article-table_right">90.2</TD></TR><TR><TD><EM>PtrpP_2</EM></TD><TD class="article-table_right">43.6</TD></TR></TBODY></TABLE></DIV><H4><SPAN class="mw-headline" id="Preparation_of_competent_E..C2.A0coli_strains_DH5.CE.B1_and_XL1-Blue">Preparation of competent <EM>E. coli</EM> strains DH5α and XL1-Blue</SPAN></H4><P>The preparation was carried out over the following days.</P><H5><SPAN class="mw-headline" id="The_Material_3">The Material</SPAN></H5><UL><LI>TB buffer (ice-cold)</LI><LI>liquid LB medium</LI><LI>SOB medium</LI><LI>1 M MgCl<SUB>2</SUB></LI><LI>1 M MgSO<SUB>4</SUB></LI><LI>DMSO (100%)</LI></UL><H5><SPAN class="mw-headline" id="The_Procedure_3">The Procedure</SPAN></H5><OL><LI>Inoculate 20 mL LB medium either with a single colony or from a cryo stock of the desired <EM>E. coli</EM> strain and incubate the culture with agitation for 20 h at 28°C.</LI><LI>Inoculate 250 mL SOB medium supplemented with 10 mM MgCl<SUB>2</SUB> and 10 mM MgSO<SUB>4</SUB> in a 2 L shake flask and grow the cells to an OD<SUB>600</SUB> of 0.5–0.9 (20–24 h) at 18°C and 200–250 rpm.</LI><LI>Incubate the whole flask for 10 min on ice. Collect the cells by centrifugation for 10 min at 4°C and 5,000 rpm. Re-suspend the cell in 80 mL of ice-cold TB buffer and incubate them for 10 min on ice. Collect the cells by centrifugation for 5 min at 5,000 rpm.</LI><LI>Re-suspend the cells in 20 mL of ice-cold TB buffer. Add DMSO to a final concentration of 7% (1.4 mL) by gently shaking the falcon tube.</LI><LI>Transfer 0.2 mL aliquots into labelled Eppendorf reaction tubes and freeze the cells in liquid nitrogen. Store the cells at -80°C. The volume of the pre-culture strongly depends on the <EM>E. coli</EM> strain. Using DH5α, it is recommended to use 4 mL pre-culture and inoculate at 8 a.m. Nex day around 10 a.m. this strain should have reached an OD<SUB>600</SUB> between 0.5 and 0.6. If you use the strain XL1-Blue, you will need less cells and it is sufficient to inoculate the SOB medium around lunchtime.</LI></OL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="New_Primers_for_gltT_and_sequencing">New Primers for <EM>gltT</EM> and sequencing</SPAN></H3><P class="notebook-head_date">23.05.18</P></DIV><DIV class="notebook-content"><P>The PCR from 17.05.18 was repeated with new primers (iGEM2018_19 and iGEM2018_20) that amplify also the ends of the gene <EM>gltT</EM>. The protocol was the same as before.</P><H6><SPAN class="mw-headline" id="Results_4">Results</SPAN></H6><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><DIV class="article_multpicture-paragraph"><P>Amplified <EM>gltT</EM> gene from iGEM18–iGEM29.</P><P>Amplified <EM>gltT</EM> gene from iGEM30–iGEM34.</P></DIV></DIV><P>The amplified <EM>gltT</EM> fragments were purified and sent to sequencing with primers iGEM2018_29 and iGEM2018_30. The following table provides an overview of mutations in <EM>gltT</EM> from glyphosate adapted mutants.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH><EM>gltT</EM> sequence</TH><TH>Mutation</TH><TH>Consequence</TH></TR><TR><TD>Wild type</TD><TD>—</TD><TD>—</TD></TR><TR><TD>iGEM18</TD><TD>Δ669A</TD><TD>Truncation</TD></TR><TR><TD>iGEM19</TD><TD>Δ669A</TD><TD>Truncation</TD></TR><TR><TD>iGEM20</TD><TD>C211T</TD><TD>Truncation</TD></TR><TR><TD>iGEM22</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR><TR><TD>iGEM23</TD><TD>Δ65T</TD><TD>Truncation</TD></TR><TR><TD>iGEM24</TD><TD>deletion 898–923</TD><TD>Truncation</TD></TR><TR><TD>iGEM25</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR><TR><TD>iGEM27</TD><TD>A388T</TD><TD>Truncation</TD></TR><TR><TD>iGEM29</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR><TR><TD>iGEM30</TD><TD>Δ996&amp;nash;1090</TD><TD>Truncation</TD></TR><TR><TD>iGEM32</TD><TD>Δ669A</TD><TD>Truncation</TD></TR><TR><TD>iGEM33</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Test_transformation_of_the_new_competent_E..C2.A0coli_cells">Test transformation of the new competent <EM>E. coli</EM> cells</SPAN></H3><P class="notebook-head_date">24.05.18</P></DIV><DIV class="notebook-content"><P>We performed a test transformation to prove that the cells were actually competent. The following plasmids were used.</P><UL><LI>pBQ200</LI><LI>pGP172</LI><LI>pAC7</LI><LI>pAC5</LI></UL><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Overview how a transformation of competent <EM>E. coli</EM> cells works with plasmids. The agar plates should be made of complex medium (like LB) and contain the appropriate antibiotics.</P></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Evaluation_of_the_test_transformation">Evaluation of the test transformation</SPAN></H3><P class="notebook-head_date">25.05.18</P></DIV><DIV class="notebook-content"><P>The transformed cells were incubated over night at 37°C and the colonies were colony forming units were counted.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH>Colony forming units</TH></TR><TR><TD>DH5α + pBQ200</TD><TD class="article-table_right">1360</TD></TR><TR><TD>DH5α + pGP172</TD><TD class="article-table_right">1520</TD></TR><TR><TD>DH5α + pAC7</TD><TD class="article-table_right">1440</TD></TR><TR><TD>XL1-Blue + pBQ200</TD><TD class="article-table_right">180</TD></TR><TR><TD>XL1-Blue + pGP172</TD><TD class="article-table_right">208</TD></TR><TR><TD>XL1-Blue + pAC7</TD><TD class="article-table_right">80</TD></TR><TR><TD>XL1-Blue + pAC5</TD><TD class="article-table_right">22</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Inoculation_of_transformed_strains">Inoculation of transformed strains</SPAN></H3><P class="notebook-head_date">28.05.18</P></DIV><DIV class="notebook-content"><P>4 mL LB medium were supplemented with the appropriate antibiotics and inoculated with the transformed strains (25.05.18). The cultures were incubated over night with agitation at 37°C.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparation_of_the_plasmids_for_further_work">Preparation of the plasmids for further work</SPAN></H3><P class="notebook-head_date">29.05.18</P></DIV><DIV class="notebook-content"><P>The isolation of the plasmids was performed with the NucleoSpin® kit from Macherey and Nagel.</P><H5><SPAN class="mw-headline" id="The_Material_4">The Material</SPAN></H5><UL><LI>Liquid LB medium</LI><LI>NucleoSpin® plasmid kit</LI></UL><H5><SPAN class="mw-headline" id="The_Procedure_4">The Procedure</SPAN></H5><OL><LI>Use single colonies from your transformation plates to inoculate 4 mL (15–20 mL for low-copy plasmids) liquid LB medium supplemented with the appropriate antibiotic and grow the culture overnight at 37°C with agitation.</LI><LI>Collect the cells from 1.5 mL (15 mL for low-copy plasmids) of the overnight cultures by centrifugation (1 min, 13,000 rpm). Re-suspend the pellet in 250 mL of buffer A1, add 250 mL of buffer A2, mix gently 6–8 times and incubate the suspension for 5 min at room temperature. For the isolation of low-copy plasmids add 500 mL of buffers A1 and A2 to the cells.</LI><LI>Add 300 mL (0.6 mL for low-copy plasmids) of buffer A3, mix gently 6–8 times and centrifuge the lysate for 5–10 min at 13,000 rpm.</LI><LI>Transfer the supernatant into NucleoSpin® plasmid column, centrifuge for 1 min at 13,000 rpm and discard the flow-through.</LI><LI>OPTIONAL: For sequencing of low-copy plasmid DNA add 500 mL buffer AW to the column, centrifuge for 1 min at 13000 rpm and discard the flow-through.</LI><LI>Add 600 mL of buffer A4, centrifuge (1 min, 13,000 rpm) and discard the flow-through.</LI><LI>To dry the silica membrane, centrifuge again for 2 min at 13,000 rpm.</LI><LI>Add 50 mL of sterile and deionized water on top of the silica membrane of the column incubate the column for 1–5 min at room temperature and elute the DNA by centrifugation for 1 min at 13,000 rpm.</LI><LI>The plasmid DNA is highly pure and can be stored at -20°C.</LI></OL><P>The concentration of the isolated plasmids was determined with the Nanodrop device.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Plasmid</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD>pBQ200</TD><TD class="article-table_right">186.6</TD></TR><TR><TD>pGP172</TD><TD class="article-table_right">84.6</TD></TR><TR><TD>pAC7</TD><TD class="article-table_right">114.1</TD></TR><TR><TD>pAC5</TD><TD class="article-table_right">75.9</TD></TR></TBODY></TABLE></DIV><P>For the cloning procedure, a restriction digest was carried out with <EM>aroE_1</EM>, <EM>aroE_2</EM>, <EM>PtrpP</EM>, and <EM>PtrpE</EM> fragments (from 18.05.18 and 22.05.18).</P><H5><SPAN class="mw-headline" id="Reaction_mixtures_to_digest_the_fragments">Reaction mixtures to digest the fragments</SPAN></H5><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>aroE_1</EM></TD><TD>20</TD></TR><TR><TD><EM>Sac</EM>I</TD><TD>2</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>2</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>3</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>aroE_2</EM></TD><TD>20</TD></TR><TR><TD><EM>Sal</EM>I</TD><TD>2</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>2</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>3</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>PtrpP</EM></TD><TD>20</TD></TR><TR><TD><EM>EcoR</EM>I</TD><TD>2</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>2</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>3</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 30 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>PtrpE</EM></TD><TD>20</TD></TR><TR><TD><EM>Sac</EM>I</TD><TD>2</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>2</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>3</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 30 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><P>The samples were then purified with the PCR purification clean up Kit and eluted in 25 µL sterile H<SUB>2</SUB>O. The concentrations of the digested fragments were determined using the Nanodrop device.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Plasmid</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD><EM>aroE_1</EM></TD><TD class="article-table_right">63</TD></TR><TR><TD><EM>aroE_2</EM></TD><TD class="article-table_right">43.8</TD></TR><TR><TD><EM>PtrpP</EM></TD><TD class="article-table_right">45.7</TD></TR><TR><TD><EM>PtrpE</EM></TD><TD class="article-table_right">47.7</TD></TR></TBODY></TABLE></DIV><H5><SPAN class="mw-headline" id="Reaction_mixtures_to_digest_the_plasmids">Reaction mixtures to digest the plasmids</SPAN></H5><H6><SPAN class="mw-headline" id="pGP172">pGP172</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>pGP172</TD><TD>18</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>3</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>6</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD><TD>Inactivation at 80°C for 5 min</TD></TR></TBODY></TABLE></DIV><P>The digest was tested on an agarose gel. Afterwards, the following mixture was prepared.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Digested pGP172</TD><TD>28</TD></TR><TR><TD><EM>Sac</EM>I</TD><TD>3</TD></TR><TR><TD>FD buffer</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>8</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD><TD>Inactivation at 80°C for 5 min</TD></TR></TBODY></TABLE></DIV><P>The digest was finally incubated with 1 µL &quot;Fast AP&quot; for 10 min at 37°C and eluted in 25 µL H<SUB>2</SUB>O.</P><H6><SPAN class="mw-headline" id="pBQ200">pBQ200</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>pBQ200</TD><TD>15</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>3</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>9</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD><TD>Inactivation at 80°C for 5 min</TD></TR></TBODY></TABLE></DIV><P>The digest was tested on an agarose gel. Afterwards, the following mixture was prepared.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Digested pBQ200</TD><TD>28</TD></TR><TR><TD><EM>Sal</EM>I</TD><TD>3</TD></TR><TR><TD>FD buffer</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>8</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD><TD>Inactivation at 80°C for 5 min</TD></TR></TBODY></TABLE></DIV><P>The digest was finally incubated with 1 µL &quot;Fast AP&quot; for 10 min at 37°C and eluted in 25 µL H<SUB>2</SUB>O.</P><H6><SPAN class="mw-headline" id="pAC7">pAC7</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>pAC7</TD><TD>20</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD>3</TD></TR><TR><TD>FD buffer</TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>4</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 45 min at 37°C</STRONG></TD><TD>Inactivation at 80°C for 5 min</TD></TR></TBODY></TABLE></DIV><P>The digest was tested on an agarose gel. Afterwards, the following mixture was prepared.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Digested pAC7</TD><TD>28</TD></TR><TR><TD><EM>Sal</EM>I</TD><TD>3</TD></TR><TR><TD>FD buffer</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>8</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 30 min at 37°C</STRONG></TD><TD>Inactivation at 80°C for 5 min</TD></TR></TBODY></TABLE></DIV><P>The digest was finally incubated with 1 µL &quot;Fast AP&quot; for 10 min at 37°C and eluted in 30 µL H<SUB>2</SUB>O.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Construction_of_plasmids">Construction of plasmids</SPAN></H3><P class="notebook-head_date">30.05.18</P></DIV><DIV class="notebook-content"><P>Concentration of the purified plasmids was analyzed with the Nanodrop device.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Plasmid</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD>pGP172</TD><TD>36.3</TD></TR><TR><TD>pBQ200</TD><TD>71.2</TD></TR><TR><TD>pAC7</TD><TD>41.3</TD></TR></TBODY></TABLE></DIV><P>In the next step, the fragments and vectors were ligated using 100 ng of the vector and calculating the concentration of the insert dependent on its length.</P><UL><LI>pAC7 (10573 bp) → 0.5 µL insert</LI><LI>pGP172 (4600 bp) → 1.5 µL insert</LI><LI>pBQ200 (6700 bp) → 1.5 µL insert</LI></UL><H6><SPAN class="mw-headline" id="Ligation_of_PtrpP_into_pAC7">Ligation of <EM>PtrpP</EM> into pAC7</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pAC7)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>Insert (<EM>PtrpP</EM>)</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">14</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Ligation_of_PtrpE_into_pAC7">Ligation of <EM>PtrpE</EM> into pAC7</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pAC7)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>Insert (<EM>PtrpE</EM>)</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">14</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Ligation_of_aroE_1_into_pGP172">Ligation of <EM>aroE_1</EM> into pGP172</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pGP172)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>Insert (<EM>aroE_1</EM>)</TD><TD class="article-table_right">1.5</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">13</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Ligation_of_aroE_2_into_pBQ200">Ligation of <EM>aroE_2</EM> into pBQ200</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pBQ200)</TD><TD class="article-table_right">1.5</TD></TR><TR><TD>Insert (<EM>aroE_2</EM>)</TD><TD class="article-table_right">1.5</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">14</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Ligation_of_Ptrp_into_pAC7">Ligation of <EM>Ptrp</EM> into pAC7</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pAC7)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>Insert (<EM>PtrpP</EM>)</TD><TD class="article-table_right">0.5</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">14</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Re-Ligation_of_vector_pAC7">Re-Ligation of vector pAC7</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pAC7)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>Insert</TD><TD class="article-table_right">–</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">14.5</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Re-Ligation_of_vector_pGP172">Re-Ligation of vector pGP172</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pGP172)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>Insert</TD><TD class="article-table_right">–</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">14.5</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Re-Ligation_of_vector_pBQ200">Re-Ligation of vector pBQ200</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>Vector (pBQ200)</TD><TD class="article-table_right">1.5</TD></TR><TR><TD>Insert</TD><TD class="article-table_right">–</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 Ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">15.5</TD></TR><TR class="article_table_hline"><TD><STRONG>Total volume</STRONG></TD><TD class="article-table_right"><STRONG>20</STRONG></TD></TR></TBODY></TABLE></DIV><P>All samples were incubated at 16°C over night.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Transformation_of_DH5.CE.B1">Transformation of DH5α</SPAN></H3><P class="notebook-head_date">31.05.18</P></DIV><DIV class="notebook-content"><P>Competent <EM>E. coli</EM> cells from the strain DH5α were transformed with the ligation samples. Also, a negative controll without DNA was prepared, were the cells received only water. The transformed cells were plated on LB medium agar containing ampicillin and X-Gal in the following manner: on one plate 50 µL and on the second palte 100 µL. The cells were incubated over night at 37°C.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>