Team:Goettingen/Notebook/reporterSystems June
Contents
1 Evaluation of transformation and preparation of lacZ measurements
1.1 Preparation of lacZ measurements
1.1.1 LD-Mix
1.1.2 Z buffer
1.1.3 ONPG
2 Preparation and test digest of new constructed plasmids
2.1 Mastermix I
2.2 Mastermix II
2.3 Mastermix III
3 Sequencing of plasmids
3.1 pGP172_aroE_1
3.2 pBQ200_aroE_2
3.3 pAC7_PtrpE
3.4 pAC7_PtrpP
4 Preparation of competent B. subtilis cells
4.1 The Material
4.2 The Procedure – Preparation of competent cells
4.3 Transformation of B. subtilis
4.4 Sequencing results from 05.06.18
5 Evaluation of transformation
5.1 Determination of α-amylase
5.2 Starch plates
5.3 5x Lugol's solution
6 Ligation of PtrpP into pAC7
7 Naming of plasmids and evaluation of the ligation
7.1 Renaming of plasmids
7.2 Results of ligated PtrpPinto pAC7
8 Disc diffusion assay
9 β-galactosidase assay
9.1 General procedure to determine β-galactosidase activity.
10 Preparation of cell pellets for β-galactosidase–assay
11 β-galactosidase–assay
12 Preparation for RNA sequencing
13 Ligation of PtrpP into pAC7
14 β-galactosidase assay
15 Isolation of plasmids (pAC7_PtrpP)
16 Transformation of SP1
17 Isolation of chromosomal DNA
17.1 Isolation of chromosomal DNA from Bacillus subtilis
18 Transformations
Evaluation of transformation and preparation of lacZ measurements
01.06.18
We counted the colony forming units (cfu) from the transformation.
Transformation
Cfu
Cfu (concentrated)
DH5α + pAC7_PtrpP
1 white/4 blue
44 white/3 blue
DH5α + pBQ200_aroE_2
1 white/17 blue
48 white/0 blue
DH5α + pAC7_PtrpE
1 white/35 blue
60 white/32 blue
DH5α + pGP172_aroE_1
61
120
On the Re-Ligation plates were on all plates only 1 or 2 colonies visible. The negative controll plates did not show any colonies.
Preparation of lacZ measurements
LD-Mix
100 mg Lysozyme
10  DNase I
Ad to 10 mL with deionized H2O
Store 500 µL aliquots at -20°C
Z buffer
60 mM Na2HPO4
40 mM NaH2PO4
10 mM KCl
1 mM MgSO4
50 mM β-Mercaptoethanol (toxic!)
(175 mL β-Mercaptoethanol to 50 mL Z buffer.)
ONPG
4 mg o-Ntrophenyl-β-D-Galactopyranoside (ONPG)
Ad to 1 mL with Z buffer without βMercaptoethanol
Preparation and test digest of new constructed plasmids
04.06.18
The plasmids were isolated using NucleoSpin® plasmid kit, Macherey&Nagel as previously described. The concentration of the purified plasmids was analysed with the Nanodrop device.
Plasmid
DNA concentration in [ng/µL]
pGP172_aroE_1 sample 1
138.8
pGP172_aroE_1 sample 2
121.8
pGP172_aroE_1 sample 3
154
pGP172_aroE_1 sample 4
123.1
pPQ200_aroE_2 sample 1
823.8
pPQ200_aroE_2 sample 2
485.7
pPQ200_aroE_2 sample 3
815.2
pPQ200_aroE_2 sample 4
926.2
pAC7_PtrpE sample 1
192
pAC7_PtrpE sample 2
231.9
pAC7_PtrpP sample 1
193.3
pAC7_PtrpP sample 2
202.7
The isolated plasmids were digested to check whether there is an insert.
Mastermix I
Compound
Volume in [µL]
BamHI
2.5
SacI
2.5
FD buffer
5
H2O
30
Incubate for 15 min at 37°C
Mastermix II
Compound
Volume in [µL]
BamHI
2.5
EcoRI
2.5
FD buffer
5
H2O
30
Incubate for 15 min at 37°C
Mastermix III
Compound
Volume in [µL]
BamHI
2.5
SalI
2.5
FD buffer
5
H2O
30
Incubate for 15 min at 37°C
The samples were put on an agarose gel. The following figure shows the results.
<img src="
">
Lane 1–4: pGP_aroE_1. Lane 5–8: pBQ200_aroE_2. Lane 9&10: pAC7_PtrpE. Lane 11&12: pAC7_PtrpP.
The following samples were selected and stored at -20°C.
pGP172_aroE_1 sample 1
pBQ200_aroE_2 sample 3
pBQ200_aroE_2 sample 4
pAC7_PtrpE sample 1
pAC7_PtrpE sample 2
pAC7_PtrpP sample 1
pAC7_PtrpP sample 2
Sequencing of plasmids
05.06.18
The isolated plasmids were prepared for sequencing with the following mixtures.
pGP172_aroE_1
Compound
Volume in [µL]
DNA
7
iGEM2018_1
T7 fwd.
T7 rev.
6
H2O
2
pBQ200_aroE_2
Compound
Volume in [µL]
DNA
1
iGEM2018_27
FX125
M13-pac fwd
6
H2O
8
pAC7_PtrpE
Compound
Volume in [µL]
DNA
5
ML107
lacZ rev.
6
H2O
4
pAC7_PtrpP
Compound
Volume in [µL]
DNA
5
ML107
lacZ rev.
6
H2O
4
Primer were diluted 1:20 (10 µL primer and 190 µL H2O. Every sample contains between 800–1000 ng DNA in a volume of 15 µL.
Preparation of competent B. subtilis cells
06.06.18
The Material
LB medium
MN medium
MNGE medium
Expression mix
Glucose (20%)
1 M MgSO4
CAA (10%)
The Procedure – Preparation of competent cells
Inoculate 4 mL LB liquid medium with a single colony of a B. subtilis strain and incubate the culture over night at 28°C with agitation.
Use the overnight culture to inoculate 10 mL MNGE medium supplemented with 0.1% CAA in a 100 mL shake flask to an approximate OD600 of about 1.3. This may take up to 5 h, depending on the strain.
Dilute the culture 1:1 with pre-warmed MNGE (w/o CAA) and incubate the culture for another hour at 37°C on a shaker.
You can continue with the transformation of B. subtilis directly after nutritional starvation step (see Transformation of B. subtilis, step 2) or continue for long term storage.
Transfer 15 mL of the culture in sterile falcon tubes and harvest the cells by centrifugation for 5 min at 5,000 rpm. Transfer the supernatant into a sterile falcon tube.
Re-suspend the cells in 1.8 mL of the supernatant, add 1.2 mL 50% glycerine, mix the cell suspension and store the competent cells in 300 mL aliquots at -80°C.
Transformation of B. subtilis
Thaw an aliquot of the frozen, competent bacteria and mix 300 µL of them with 1.7 mL 1x MN medium that has been supplemented with 43 µL glucose (20%) + 34 µL 1 M MgSO4.
Add 0.1–1 µg DNA (2 µg plasmid DNA) to 400 µL of the competent cells and incubate the reaction tube for 30 min at 37°C.
Add 100 µL expression mix and if required an inducer (IPTG, xylose, ...)
Incubate the bacteria for 1 h at 37°C with agitation and propagate the cells on SP medium agar plates supplemented with the appropriate antibiotics.
Strains SP1 and 168 were transformed with pAC7_PtrpE (10 μL). 100 and 200 μ of the transformed cells were propagate on SP medium containing kanamycin. A negativ controll was also performed.
Sequencing results from 05.06.18
Plasmid
Sequence result
pAC7_PtrpP
ok
pBQ200_aroE_2
ok
pAC7_PtrpE
ok
pGP172_aroE_1
Mutation in binding region of the reverse primer → repeat!
Evaluation of transformation
07.06.18
The following table shows the results of the transformation results.
Sample
Concentration in [µL]
Cfu
168 + pAC7_PtrpE
200
1680
168 + pAC7_PtrpE
200
1820
168 + pAC7_PtrpE
100
480
SP1 + pAC7_PtrpE
200
112
SP1 + pAC7_PtrpE
200
80
SP1 + pAC7_PtrpE
100
42
No colonies were found on the negative control. Singularization of 3 clones respectively on SP medium containing kanamycin using wildtype SP1 and 168 as control.
Determination of α-amylase
2 single colonies of the B. subtilis transformants were propagated on a line on a starch plate. The plates were incubated over night at 37°C.
Starch plates
7.5 g Nutrient broth
5 g Starch
Ad to 1,000 mL with deionized H2O
5x Lugol's solution
100 g K Iodide
50 g Iodine
Ad to 1,000 mL with deionized H2O
The Lugol's solution was diluted to 1–fold and 5 mL of the solution was added on the plates. The insertion in the amylase encoding gene amyE was checked with a α-amylase assay. The amyE positive bacteria form a halo while those with a disrupted amyE gene should not form a halo.
The following figures show the the results from the α-amylase assays.
<img src="
">
<img src="
">
pAC7_PtrpE→168. Clones 1&3 are correct.
pAC7_PtrpE→SP1. Clone 2 is correct.
Ligation of PtrpP into pAC7
11.06.18
A restriction digest was carried out to digest PtrpP.
Compound
Volume in [μL]
DNA
20
BamHI
2
EcoRI
2
FD buffer
3
H2O
3
30 min at 37°C
The sample was purified with the DNA purification kit and eluted in 25 μL. Also, the DNA concentration was measured with the Nanodrop device: 22.4 ng/µL. A sample for the ligation and one for the Re-ligation was prepared.
Compound
Volume in [μL]
Vector (pAC7)
2.5
Insert (PtrpP
1
T4 buffer
2
T4 ligase
1
H2O
13.5
at least 2 h at RT
Also, a Re-ligation mixture was prepared without an insert. With the ligation and re-ligation mixtures were competent E. coli cells of strain DH5α transformed as described previously. The transformed cells were propagated on LB medium containing X-Gal and kanamycin.
Naming of plasmids and evaluation of the ligation
12.06.18
Renaming of plasmids
Previous name
New name
pGP172_aroE_1
pIGEM1
pBQ200_aroE_2
pIGEM2
pAC7_PtrpE
pIGEM3
The plasmids were stored at -20°C.
Results of ligated PtrpPinto pAC7
Sample
Amount in [µL]
Cfu
pAC7_PtrpP
100
0
pAC7_PtrpP
rest
4 blue/1 white
Re-ligation pAC7
100
0
Re-ligation pAC7
rest
0
Negative control
100
0
Negative control
rest
0
Overnight cultures of 168_PtrpE and SP1_PtrpE were incubated in LB medium at 28°C.
Disc diffusion assay
13.06.18
To analyze the effects of glyphosate on B. subtilis, 100 µL of cultures 168_PtrpE and SP1_PtrpE were plated on CS-Glu medium containing X-Gal. The plates were dried for 5 min.
A disc containing 5 µL of 70 mM glyphosate solution was placed in the middle of the plate. The plates were incubated at 37°C over night. </div> </div>
β-galactosidase assay
<p class="notebook-head_date">14.06.18</p>
General procedure to determine β-galactosidase activity.
Propagate a single colony over a whole SP agar plate and incubate the plate over the day at 37°C. You could also propagate a single colony on an SP agar plate in the evening and use the plate at the evening next day to inoculate a pre-culture.
Take a single colony of the B. subtilis strain of interest and inoculate 2&nadsh;5 mL C minimal medium supplemented with appropriate antibiotics and effectors (carbon and nitrogen sources). Incubate the pre-cultures over night at 28°C with agitation (220 rpm).
Inoculate 10 mL C minimal medium (plus effectors, carbon and nitrogen sources) supplemented in a 100 mL shake flask to an OD600 of about 0.1 and grow the cultures to an OD600 of 0.5–0.8 (3–5 h) at the desired temperature (usually at 37°C for B. subtilis).
Collect the cells from 1.5–2.0 mL of the cultures by centrifugation for 5 min at 13,000 rpm and 4°C. Discard the supernatants and store the cell pellets for further treatment at -20°C.
Re-suspend the cells in 400 mL Z buffer (with β-ME and 20 mL LD-Mix in 4 mL Z buffer). Incubate the suspension for 10 min at 37°C to lyse the cells. Keep a minimum of 20 mL Z buffer/LD-Mix for the Bradford assay.
Remove cell debris from the soluble fraction by centrifugation for 2 min at 13,000 rpm and 4°C. Transfer the supernatants into new Eppendorf reaction tubes.
Add 100 mL of the cell free crude extract to 700 mL Z buffer (with β-ME) and pre-incubate the mixture for 5 min at 28°C. 800 mL Z buffer (with β-ME) will serve as the reference.
Start the enzymatic reaction by the addition of 200 mL ONPG to the reaction tubes and note the time.
As soon as the reaction mixture turns yellow, stop the reaction by adding 500 mL Na2CO3 and note the time. Now it is time to measure the absorption of the sample at a wavelength of λ = 420 nm. The “reference sample” will serve as the reference!
The specific β-galactosidase activity in Miller Units/mg protein (MU/mg) can be calculated using the formula below:
<img src="
"> <p>A420: absorption of o–Nitrophenol; Δt: time difference between start and stop of the enzymatic reaction in [min]; V: volume of crude extract used for the reaction (usually 0.1 mL, see step 3); A595: absorption at λ = 595 nm obtained by the Bradford assay.</p>
The method to detect the β–galactosidase activity can also be applied to detect the enzymatic activity of the β–xylosidase in cell freee crude extract of B. subtilis (Lindner et al., 1994). To detect β–xylosidase activity, ONPG has to be replaced by PNPX.
Preparation of cell pellets for β-galactosidase–assay
<p class="notebook-head_date">15.06.18</p>
<p>The following table shows the samples used for the lacZ–assay.</p>
Sample
OD600
Volume to inoculate [µL]
SP1 + pIGEM3
4.32
231
168 + pIGEM3
3.48
287
<p>Samples were inoculated with and without tryptophane.</p>
sample
3 h
4 h
5 h
SP1 + pIGEM3 –trp –GS
0.3
0.83
–
SP1 + pIGEM3 –trp +GS
0.29
0.45
0.5
SP1 + pIGEM3 +trp –GS
0.24
0.40
0.5
SP1 + pIGEM3 +trp +GS
0.26
0.46
0.61
168 + pIGEM3 –trp –GS
0.18
0.36
0.38
168 + pIGEM3 –trp +GS
0.43
0.64
–
<p>2x 2 mL of each sample was centrifuged at 13,000 rpm at 4°C. Pellets were stored at -20°C.</p>
β-galactosidase–assay
<p class="notebook-head_date">18.06.18</p>
<p>The lacZ–assay was carried out as described previously. There was not change in coloration observeable, therefore the experiment has to be repeated. The Bradford assay was carried out to verify that the cells are lyzed using the LD mix. It came out that all samples contained proteins.</p> <p>Furthermore, B. subtilis strain 168 was transformed with pBQ200 (as an isogenic control) and pIGEM2.</p>
Preparation for RNA sequencing
<p class="notebook-head_date">19.06.18</p>
<p>Cultures were prepared for RNA sequencing. OD600 measurement of the two ON cultures.</p>
Sample
OD600
Volume to inoculate [µL]
SP1, 1
5.38
185
SP1, 2
4.8
208
<p>Culture 2 was used to inoculate 6x CS-Glu –trp media. 150 µL 50 mM glyphosate solution was added to cultures 4–6 to obtain a concentration of 0.75 mM.</p>
Sample
0 h
3 h
7 h
SP1 0 mM GS, 1
0.088
0.18
1.56
SP1 0 mM GS, 1
0.092
0.16
1.76
SP1 0 mM GS, 3
0.09
0.16
1.54
SP1 0.75 mM GS, 1
0.09
0.15
1.12
SP1 0.75 mM GS, 2
0.088
0.13
0.71
SP1 0.75 mM GS, 3
0.087
0.12
0.96
<p>1 mL cells were harvested at 4°C with centrifugation for 2 min at 13,000 rpm. The supernatant was descarded, and the pellets were placed into liquid nitrogen. Storage at -20°C.</p> <p>Furthermore, a PCR was performed to amplify PtrpP with primers iGEM2018_23 and iGEM2018_24. The template was chromosomal DNA from the wildtype 168. The results are shown in the figure below. The samples were purified and eluted in 35 µL H2O and measured with the Nanodrop device.</p>
<img src="
"> <p>In all four lanes are samples of PtrpP.</p>
<p>To measure the β-galactosidase activity, the strains 168_pIGEM3, SP1_pIGEM3, GP342, and GP650 were propagated on LB agar plates and grown over night at 37°C.</p>
Ligation of PtrpP into pAC7
<p class="notebook-head_date">20.06.18</p>
<p>Strains GP342 (200–300 MU/mg protein) and GP350 (10–20 MU/mg protein) serve as a positive control for the β-galactosidase assay. 4 mL LB medium were inoculated from the plates and incubated at 37°C over daytime. In the evening, the cells were transferred into 3 mL CS-Glu medium and incubated over night at 28°C.</p> <p>PtrpP was digested with the restriction enzymes BamHI and EcoRI as described before. The samples were purified and eluted in 30 µL H2O and measured with the Nanodrop device. Furthermore, the fragment was ligated into the vector pAC7 as described before.</p>
β-galactosidase assay
<p class="notebook-head_date">21.06.18</p>
<p>OD600 measurement of the over night cultures.</p>
Sample
OD600
Volume to inoculate [µL]
168_pIGEM3
0.83
1204
SP1_pIGEM3
1.56
641
GP342
0.7
1428
GP350
0.81
1234
<p>The OD600 was measured.</p>
Sample
0 h
3 h
4 h
5 h
168_pIGEM3
0.088
0.35
0.53
0.62
SP1_pIGEM3
0.09
0.15
0.25
0.47
GP342
0.088
0.23
0.47
0.56
GP350
0.086
0.07
0.01
/
<p>Competent E. coli DH5α cells were transformed, a negative control was also performed. The transformed cells were propagated on LB agar containing ampicilin and X-Gal.</p> <p>OD600 measurement for the β-galactosidase assay n=3.</p>
Sample
Start [sec]
End [min]/th>
A420
A595
168_pIGEM3
0
22
0.006
0.198
SP1_pIGEM3
30
22.3
0.037
0.125
GP342
60
7
0.296
0.167
<p>For the strains, the Miller units per mg protein were calculated with the formula given previously.</p>
Strain
Activity
168_pIGEM3
26.91 units/mg protein
SP1_pIGEM3
2.75 units/mg protein
GP342
590.81 units/mg protein
Isolation of plasmids (pAC7_PtrpP)
<p class="notebook-head_date">25.06.18</p>
<p>The plasmids from 21.06.18 were isolated, eluted in 30 µL H2O, and the concentration was measured with the Nanodrop device. A sample was sent for sequencing with the primers lacZ-rev and ML107.</p>
Transformation of SP1
<p class="notebook-head_date">26.06.18</p>
<p>B. subtilis strain SP1 was made competent as described previously. The cells were transformed with chromosomal DNA of GP650. The transformed cells were propagated on SP agar containing the appropriate antibiotics.</p>
Isolation of chromosomal DNA
<p class="notebook-head_date">27.06.18</p>
<p>Transformations from 26.06.18 were separated on SP agar containing the appropriate antibiotics. Sequencing of pAC7_PtrpP was succesful and the plasmid was named "pIGEM5".</p>
Isolation of chromosomal DNA from Bacillus subtilis
<p>The protocol requires the "Bacterial DNA kit" from "peqGOLD".</p>
Inoculate 4 mL LB medium with the cells of interest and incubate them over night at 37°C. Harvest the cells with centrifugation.
Resuspend the pellet in 100 µL TE buffer and add 100 µL lysozyme. Incubate the reaction for 1 h at 30°C and 800 rpm.
Centrifuge the samples for 5 min at 6,500 rpm, discard the supernatant. Resuspend the pellet in 400 µL DNA lysis buffer T.
Add 20 µL proteinkinase K and 15 µL RNase A. Shake the samples vigorously and incubate them for 30 min at 70°C.
Add 200 µL DNA binding, mix the sample and load it on the column tube. Centrifugation at 11,000 rpm. Discard the flowthrough and place the column in a new collection tube.
Add 650 µL DNA wash buffer on the column and centrifuge for 1 min at 11,000 rpm. Discard the flowthrough.
Dry the column for 2  centrifugation at 11,000 rpm.
Place the column in a sterile Eppendorf cup, add 50 µL sterile H2O, incubate the column for 1 min and centrifuge at 7,900 rpm. Repeat this step.
To avoid spores in the DNA, centrifuge the samples 10–30 min at 13,000 rpm at 4°C.
Store the supernatant at -20°C.
<p>Strain SP1 was transformed with pIGEM5 and chromosomal DNA from strain GP342.</p>
Transformations
<p class="notebook-head_date">28.06.18</p>
<p>Transformations from SP1 + DNA from GP650 worked well, so two clones were picked and separated. </p>
