<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Notebook_July skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Notebook/July</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="notebook-item"><DIV class="notebook-head"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Again_transformations"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Again transformations</SPAN></A></LI><LI class="toclevel-1"><A href="#Evaluation_of_transformations"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Evaluation of transformations</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_preparation_and_preparation_of_.22modified_disc_assay.22"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Plasmid preparation and preparation of &quot;modified disc assay&quot;</SPAN></A></LI><LI class="toclevel-1"><A href="#"><SPAN class="tocnumber">4</SPAN></A></LI><LI class="toclevel-2"><A href="#Modified_disc_assay"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Modified disc assay</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_.CE.B2-galactosidase_assay"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Preparation of β-galactosidase assay</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#.CE.B2-galactosidase_assay"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">β-galactosidase assay</SPAN></A></LI><LI class="toclevel-1"><A href="#_2"><SPAN class="tocnumber">6</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2.E2.80.93galactosidase_assay"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">β–galactosidase assay</SPAN></A></LI><LI class="toclevel-2"><A href="#.CE.B2.E2.80.93galactosidase_assay_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">β–galactosidase assay</SPAN></A></LI><LI class="toclevel-3"><A href="#Strain_iGEM3"><SPAN class="tocnumber">7.1.1</SPAN><SPAN class="toctext">Strain iGEM3</SPAN></A></LI><LI class="toclevel-3"><A href="#Strain_GP342"><SPAN class="tocnumber">7.1.2</SPAN><SPAN class="toctext">Strain GP342</SPAN></A></LI><LI class="toclevel-1"><A href="#Again_.CE.B2.E2.80.93galactosidase_assay"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Again β–galactosidase assay</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2.E2.80.93galactosidase_assay_3"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">β–galactosidase assay</SPAN></A></LI><LI class="toclevel-1"><A href="#Overexpression_of_proteins_in_E..C2.A0coli"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Overexpression of proteins in E. coli</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparation_of_biobricks"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Preparation of biobricks</SPAN></A></LI><LI class="toclevel-2"><A href="#Combined-chain_reaction_.28CCR.29_and_multiple-mutation_reaction_.28MMR.29"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Combined-chain reaction (CCR) and multiple-mutation reaction (MMR)</SPAN></A></LI><LI class="toclevel-3"><A href="#The_Material"><SPAN class="tocnumber">11.1.1</SPAN><SPAN class="toctext">The Material</SPAN></A></LI><LI class="toclevel-3"><A href="#The_Procedure"><SPAN class="tocnumber">11.1.2</SPAN><SPAN class="toctext">The Procedure</SPAN></A></LI><LI class="toclevel-3"><A href="#Reaction_mixture"><SPAN class="tocnumber">11.1.3</SPAN><SPAN class="toctext">Reaction mixture</SPAN></A></LI><LI class="toclevel-1"><A href="#Cloning_of_biobricks_2.2C_9.2C_10.2C_12_and_overexpression_of_AroA_and_AroE"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Cloning of biobricks 2, 9, 10, 12 and overexpression of AroA and AroE</SPAN></A></LI><LI class="toclevel-1"><A href="#Plasmid_preparation"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Plasmid preparation</SPAN></A></LI><LI class="toclevel-1"><A href="#Amplification_of_the_biobricks_2.2C_9.2C_and_10"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Amplification of the biobricks 2, 9, and 10</SPAN></A></LI><LI class="toclevel-1"><A href="#New_.CE.B2.E2.80.93galactosidase_assay"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">New β–galactosidase assay</SPAN></A></LI><LI class="toclevel-1"><A href="#Sequencing_of_biobricks"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Sequencing of biobricks</SPAN></A></LI></DIV><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Again_transformations">Again transformations</SPAN></H3><P class="notebook-head_date">02.07.18</P></DIV><DIV class="notebook-content"><UL><LI>Transformation of pIGEM2, pIGEM2, pIGEM3, and pIGEM5 into <EM>E. coli</EM> DH5α to geht backup plasmids.</LI><LI>Transformation of SP1 with pIGEM5.</LI></UL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Evaluation_of_transformations">Evaluation of transformations</SPAN></H3><P class="notebook-head_date">03.07.18</P></DIV><DIV class="notebook-content"><P>DH5α transformants were separated. SP1-pIGEM2/3 transformants were also separated. The strain SP1 was transformed with pBQ200 to create an isogenic control. For this transformation, we used 2 µL plasmid. The following table shows the results from the transformations.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH>Cfu (normal/rest)</TH></TR><TR><TD>DH5α_pIGEM1</TD><TD>137/960</TD></TR><TR><TD>DH5α_pIGEM2</TD><TD>1072/1450</TD></TR><TR><TD>DH5α_pIGEM3</TD><TD>36/204</TD></TR><TR><TD>DH5α_pIGEM5</TD><TD>616/1096</TD></TR><TR><TD>SP1_pIGEM3</TD><TD>43/94</TD></TR><TR><TD>SP1_pIGEM5</TD><TD>–/–</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Plasmid_preparation_and_preparation_of_.22modified_disc_assay.22">Plasmid preparation and preparation of &quot;modified disc assay&quot;</SPAN></H3><P class="ntoebook-head_date">04.07.18</P></DIV><DIV class="notebook-content"><P>The plasmid pIGEM5 was isolated from <EM>E. coli</EM> with the normal protocol. Concentration: 141.4 ng/µL.A PCR was performed with the primers ML107 and lacZ-rev on the plasmid to check if there is an insert. This PCR was succesful.The transformations of SP1 with pBQ200 were separated.CS-Glu agar with and w/o tryptophan were prepared. In the middle of the plate was a circle of the agar extracted with an open falcon tube. The precultures for this experiment (SP1_pIGEM3) were incubated over night in LB medium at 28°C and agitation.</P><P>Furthermore was the β-galactosidase assay prepared for the next day with the strains:</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH>Origin</TH><TH>Genotype</TH></TR><TR><TD>GP342</TD><TD>168 derivative</TD><TD><EM>trpC2 amyE::(P<SUB>gltA</SUB>-lacZ aphaA3)</EM></TD></TR><TR><TD>GP650</TD><TD>GP342 derivative</TD><TD><EM>trpC2 amyE::(P<SUB>gltA</SUB>-lacZ aphaA3) gltC::Tn10 spc</EM></TD></TR><TR><TD>iGEM20</TD><TD>cDNA GP342→SP1</TD><TD><EM>amyE::(P<SUB>gltA</SUB>-lacZ aphaA3)</EM></TD></TR><TR><TD>iGEM21</TD><TD>cDNA GP650→SP1</TD><TD><EM>amyE::(P<SUB>gltA</SUB>-lacZ aphaA3) gltC::Tn10 spc</EM></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">05.07.18</P></DIV><DIV class="notebook-content"><H5><SPAN class="mw-headline" id="Modified_disc_assay">Modified disc assay</SPAN></H5><P>Cells from the over night culture were propagated on CS-Glu agar containing X-Gal, kanamycin and NO tryptophan. Into the holes in the plates, we filled 400 µL CS-Glu medium and 400 µL CS-Glu medium with 50 mM glyphosate. The plates were incubated over night at 37°C.</P><P>The transformation of SP1 with pIGEM5 was repeated.</P><H5><SPAN class="mw-headline" id="Preparation_of_.CE.B2-galactosidase_assay">Preparation of β-galactosidase assay</SPAN></H5><P>The strains iGEM20 and SP1_pIGEM3 were used. Some colonies were picked from the plates and resuspended in 50 µL LB medium. 5 µL were transferred into 4 mL LB medium and the remaining 45 µL were also transferred into 4 mL LB medium. The cells were incubated over daytime and then was CS-Glu medium (5 mL) inoculated with the following pattern.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>A</TH><TH>B</TH><TH>C</TH><TH>D</TH></TR><TR><TD>OD<SUB>600</SUB></TD><TD>0.05</TD><TD>0.05</TD><TD>1/10 cell amount of A/B</TD><TD>0.1</TD></TR><TR><TD>Temperature</TD><TD>37°C</TD><TD>28°C</TD><TD>37°C</TD><TD>37°C</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id=".CE.B2-galactosidase_assay">β-galactosidase assay</SPAN></H3><P class="notebook-head_date">06.07.18</P></DIV><DIV class="notebook-content"><P>With the overnight cultures were 10 mL CS-Glu medium inoculated to OD<SUB>600</SUB>=0.1, one time with 0.75 mM glyphosate and ones without.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH>OD<SUB>600</SUB></TH><TH>Time</TH></TR><TR><TD>SP1_pIGEM3 –GS</TD><TD>0.51</TD><TD>6 h</TD></TR><TR><TD>SP1_pIGEM3 +GS</TD><TD>0.496</TD><TD>6 h</TD></TR><TR><TD>iGEM20 +GS</TD><TD>0.235</TD><TD>6 h</TD></TR></TBODY></TABLE></DIV><P>The cells were harvested with the following protocol:</P><UL><LI>1.5 mL SP1_pIGEM3 with and w/o glyphosate were centrifuged for 10 min at 5,000 rpm and 4°C.</LI><LI>4.5 mL iGEM20 with glyphosate was centrifuged for 10 min at 5,000 rpm and 4°C.</LI></UL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">09.07.18</P></DIV><DIV class="notebook-content"><P>Transformation of SP1 with pIGEM5 was performed like in protocol described.</P><P>Precultures of strain iGEM21 in 5 mL CS-Glu w/o tryptophan and strain iGEM3 in 5 mL CS-Glu w/o tryptophan were prepared and incubated over night at 37°C with agitation.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id=".CE.B2.E2.80.93galactosidase_assay">β–galactosidase assay</SPAN></H3><P class="notebook-head_date">10.07.18</P></DIV><DIV class="notebook-content"><P>For the β-galactosidase assay, SP1_pIGEM3 and GP342 were transferred into 4 mL LB medium and incubated over night.For the modified disc assay, 4 mL LB medium were inoculated with SP1_pIGEM3.Primers for all biobricks were designed.</P><H5><SPAN class="mw-headline" id=".CE.B2.E2.80.93galactosidase_assay_2">β–galactosidase assay</SPAN></H5><H6><SPAN class="mw-headline" id="Strain_iGEM3">Strain iGEM3</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>GS concentration [mM]</TH><TH>OD<SUB>600</SUB> after 5.5 h</TH></TR><TR><TD>0</TD><TD>0.592</TD></TR><TR><TD>0.5</TD><TD>0.592</TD></TR><TR><TD>1</TD><TD>0.512</TD></TR><TR><TD>1.5</TD><TD>0.366</TD></TR><TR><TD>2</TD><TD>0.286</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Strain_GP342">Strain GP342</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>GS concentration [mM]</TH><TH>OD<SUB>600</SUB> after 5.5 h</TH></TR><TR><TD>0</TD><TD>0.64</TD></TR><TR><TD>0.5</TD><TD>0.696</TD></TR><TR><TD>1</TD><TD>0.632</TD></TR><TR><TD>1.5</TD><TD>0.392</TD></TR><TR><TD>2</TD><TD>0.346</TD></TR></TBODY></TABLE></DIV><P>Plasmid preparation of pBQ200 and pAC7. Furthermore, a chromosomal DNA isolation was performed for <EM>Bacillus</EM> strains 168 and SP1.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Again_.CE.B2.E2.80.93galactosidase_assay">Again β–galactosidase assay</SPAN></H3><P class="notebook-head_date">11.07.18</P></DIV><DIV class="notebook-content"><P>Over night cultures were transferred into CS-Glu medium to an OD<SUB>600</SUB>=0.1. The medium contained 0, 0.5, 1, 1.5, 2, mM glyphosate. After 8 h incubation, the following OD<SUB>600</SUB> values were measured.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>OD<SUB>600</SUB> of strain</TH><TH>0 mM GS</TH><TH>0.5 mM GS</TH><TH>1 mM GS</TH><TH>1.5 mM GS</TH><TH>2 mM GS</TH></TR><TR><TD>SP1_pIGEM3</TD><TD>0.584</TD><TD>0.584</TD><TD>0.512</TD><TD>0.366</TD><TD>0.286</TD></TR><TR><TD>GP342</TD><TD>0.64</TD><TD>0.696</TD><TD>0.632</TD><TD>0.392</TD><TD>0.346</TD></TR></TBODY></TABLE></DIV><P>The modified disc assay was repeated.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id=".CE.B2.E2.80.93galactosidase_assay_3">β–galactosidase assay</SPAN></H3><P class="notebook-head_date">12.07.18</P></DIV><DIV class="notebook-content"><P>The β–galactosidase assay was performed as described previously. The results are shown in the table below.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>Δt</TH><TH>A<SUB>420</SUB></TH><TH>A<SUB>595</SUB></TH><TH>MU/mg protein</TH></TR><TR><TD>pIGEM3 0 mM</TD><TD>31:30 min</TD><TD>0.075</TD><TD>0.132</TD><TD>36.07</TD></TR><TR><TD>pIGEM3 0.5 mM</TD><TD>32 min</TD><TD>0.071</TD><TD>0.153</TD><TD>29.46</TD></TR><TR><TD>pIGEM3 1 mM</TD><TD>31:30 min</TD><TD>0.047</TD><TD>0.094</TD><TD>31.74</TD></TR><TR><TD>pIGEM3 1.5 mM</TD><TD>31:30 min</TD><TD>0.025</TD><TD>0.049</TD><TD>32.39</TD></TR><TR><TD>pIGEM3 2 mM</TD><TD>31:30 min</TD><TD>0.014</TD><TD>0.027</TD><TD>32.92</TD></TR><TR><TD>GP342 0 mM</TD><TD>7:30 min</TD><TD>0.392</TD><TD>0.134</TD><TD>780</TD></TR><TR><TD>GP342 0.5 mM</TD><TD>7 min</TD><TD>0.316</TD><TD>0.172</TD><TD>524.9</TD></TR><TR><TD>GP342 1 mM</TD><TD>7 min</TD><TD>0.316</TD><TD>0.118</TD><TD>765.13</TD></TR><TR><TD>GP342 1.5 mM</TD><TD>16 min</TD><TD>0.198</TD><TD>0.066</TD><TD>375</TD></TR><TR><TD>GP342 2 mM</TD><TD>26:30 min</TD><TD>0.115</TD><TD>0.03</TD><TD>289.3</TD></TR><TR><TD>pIGEM3 0 mM</TD><TD>29 min</TD><TD>0.043</TD><TD>0.069</TD><TD>42,9</TD></TR><TR><TD>pIGEM3 0.75 mM</TD><TD>29 min</TD><TD>0.04</TD><TD>0.051</TD><TD>54</TD></TR><TR><TD>GP 342 0 mM</TD><TD>29 min</TD><TD>0.045</TD><TD>0.002</TD><TD>1551</TD></TR><TR><TD>Control</TD><TD>29 min</TD><TD>–</TD><TD>–</TD><TD>–</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Overexpression_of_proteins_in_E..C2.A0coli">Overexpression of proteins in <EM>E. coli</EM></SPAN></H3><P class="notebook-head_date">13.07.18</P></DIV><DIV class="notebook-content"><OL><LI>Preculture over night at 37°C with agitation has to be prepared.</LI><LI>Inoculate 4x 0.5 L LB medium with ampicillin in 2 L shake flasks to an OD<SUB>600</SUB>=0.05.</LI><LI>At an OD<SUB>600</SUB>=0.6 add an inducer, like IPTG.</LI><LI>Incubate the cultures for 3 h at 37°C with agitation.</LI><LI>Harvest the cells with centrifugation for 10 min at 4°C and 5,000 rpm. Discard the supernatant.</LI><LI>Resuspend the pellet in 25 mL buffer W (cold) and transfer the samples into 50 mL falcon tubes.</LI><LI>Centrifuge the samples for 10 min at 4°C and 5,000 rpm. Discard the supernatant and store the pellet at -20°C.</LI></OL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparation_of_biobricks">Preparation of biobricks</SPAN></H3><P class="notebook-head_date">17.07.18</P></DIV><DIV class="notebook-content"><P>The primers for the biobrick production arrived and PCRs and one CCR was performed to amplify the right fragments and to change one basepair in <EM>gltT</EM> to remove an active <EM>Pst</EM>I restriction site! The fragments for the basic parts were digested with the restriction enzymes <EM>EcoR</EM>I and <EM>Pst</EM>I. The recipes for the PCR was the same as previously shown.</P><H5><SPAN class="mw-headline" id="Combined-chain_reaction_.28CCR.29_and_multiple-mutation_reaction_.28MMR.29">Combined-chain reaction (CCR) and multiple-mutation reaction (MMR)</SPAN></H5><P>Several methods for PCR-based site-directed
            mutagenesis have been developed. Among these,
            the combined-chain reaction method proved to be
            very rapid and reliable (Bi &amp; Sambrook, 1997). The
            principle of this method is the use of mutagenic
            primers that hybridize more strongly to the
            template than the external primers. The mutagenic
            primers are phosphorylated at their 5’ ends, and
            these are ligated to the 3’-OH groups of the
            extended upstream primers by the action of a
            thermostable DNA ligase. Moreover, the DNA polymerase
            employed must not exhibit 5’→3’ exonuclease
            activity, to prevent the degradation of the
            extended primers. In our view, Pfu and Pwo
            polymerases are both well suited. The original
            protocol describes the introduction of two
            mutations simultaneously. In a previous study, we
            used a combined chain reaction to mutagenize four
            distant bases in a DNA fragment in a one-step reaction.
        </P><H6><SPAN class="mw-headline" id="The_Material">The Material</SPAN></H6><UL><LI>10x CCR buffer</LI><LI>Phusion or Accuzyme™ polymerase</LI><LI>5 U/mL Ampligase</LI><LI>Primers for amplification</LI><LI>12.5 mM dNTPs</LI><LI>Mutagenesis primer</LI><LI>BSA (20 mg/mL)</LI></UL><H6><SPAN class="mw-headline" id="The_Procedure">The Procedure</SPAN></H6><P>Before you start your CCR or MMR you should amplify your gene of interest and clone it into a
            plasmid of your choice. The introduction of mutations is far more efficient when including this
            step. Do not forget to adjust the elongation time according to your polymerase and the length of your template!
        </P><H6><SPAN class="mw-headline" id="Reaction_mixture">Reaction mixture</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>20 pmol fwd primer</TD><TD class="article-table_right">2</TD></TR><TR><TD>20 pmol rev primer</TD><TD class="article-table_right">2</TD></TR><TR><TD>mutagenesis primer</TD><TD class="article-table_right">4</TD></TR><TR><TD>Plasmid DNA</TD><TD class="article-table_right">1</TD></TR><TR><TD>10x CCR– or HF–buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>Accuzyme™ or Phusion polymerase</TD><TD class="article-table_right">1</TD></TR><TR><TD>Ampligase</TD><TD class="article-table_right">3</TD></TR><TR><TD>dNTPs</TD><TD class="article-table_right">2</TD></TR><TR><TD>BSA (20 mg/mL)</TD><TD class="article-table_right">2</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">30</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Cloning_of_biobricks_2.2C_9.2C_10.2C_12_and_overexpression_of_AroA_and_AroE">Cloning of biobricks <MARK>2</MARK>, <MARK>9</MARK>, <MARK>10</MARK>, <MARK>12</MARK> and overexpression of AroA and AroE</SPAN></H3><P class="notebook-head_date">19.07.18</P></DIV><DIV class="notebook-content"><P>The digest and ligation reactions were performed as described before. Furthermore, a purification of the overexpressed proteins AroE from <EM>B. subtilis</EM> and AroA from <EM>E. coli</EM> was performed as decribed in the following paragraph.</P><OL><LI>The AroA strain was cultured in 2 L medium, the AroE strain in 1 L as described before. Harvest the cells with centrifugation at 20,000 rpm for 20 min at 4°C. </LI><LI>Resuspend the pellet in 15 mL buffer W (ice-cold!).</LI><LI>Disrupt the cells with the French press. Repeat this step 2 times.</LI><LI>Centrifuge the samples 30 min at 17,500 rpm and 4°C.</LI><LI>Prepare the 10 mL column by BioRad with 1 mL Strep-Tactin Sepharose which equals a column volume of 500 µL.</LI><LI>Load the supernatant from the centrifugation on the column. Save the flowthrough and load it again on the column.</LI><LI>Wash the column with buffer W: 1. run: 5 mL. 2. run: 1.25 mL.</LI><LI>Elute the protein with buffer E. Load ½ of column volume on the column. Save the flowthrough.</LI><LI>Elute the protein with buffer E two times. Load the column volume on the column. Save the flowthrough.</LI><LI>Load the samples on a SDS protein gel. Mix 15 µL sample and 5 µL Laemmli buffer (5×) and boil it for 10 min at 95°C.</LI><LI>Perform a Bradford assay with the elution fractions.</LI></OL><P>The results of the protein purification are shown in the next figure.</P><DIV class="article_multpicture"><DIV><DIV>&lt;img src=&quot;&quot;&gt;</DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><DIV class="article_multpicture-paragraph"><P>The SDS page from the protein AroE from <EM>B. subtilis</EM> is shown in this figure.</P><P>The SDS page from the protein AroA from <EM>E. coli</EM> is shown in this figure.</P></DIV></DIV><P>The Bradford assay was performed as described in the Experiments section. For AroE, 4 µL and 8 µL were used and the concentration of AroE sample 1 was determined as 2.5 µg/µL and in the second sample as 2.03 µg/µL. The concentration for AroA was determined using 10 µL and 14 µL as 0.15 µg/µL.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Plasmid_preparation">Plasmid preparation</SPAN></H3><P class="notebook-head_date">20.07.18</P></DIV><DIV class="notebook-content"><P>The plasmids of biobricks <MARK>3</MARK> and <MARK>4</MARK> were preparated using the DNA, RNA and Protein purification kit by Macherey &amp; Nagel.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Amplification_of_the_biobricks_2.2C_9.2C_and_10">Amplification of the biobricks <MARK>2</MARK>, <MARK>9</MARK>, and <MARK>10</MARK></SPAN></H3><P class="notebook-head_date">26.07.18</P></DIV><DIV class="notebook-content"><P>The competent <EM>E. coli</EM> strain DH5α was transformed with the following ligations:
            <UL><LI>pBS1C3 + <MARK>2</MARK></LI><LI>pBS1C3 + <MARK>9</MARK></LI><LI>pBS1C3 + <MARK>10</MARK></LI><LI>pBS1C3 re-ligation</LI></UL></P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="New_.CE.B2.E2.80.93galactosidase_assay">New β–galactosidase assay</SPAN></H3><P class="notebook-head_date">31.07.18</P></DIV><DIV class="notebook-content"><P>The following table shows the samples for the β–galactosidase assay.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>Tryptophane</TH><TH>Glyphosate</TH></TR><TR><TD><STRONG>1</STRONG></TD><TD>100 µL</TD><TD>–</TD></TR><TR><TD><STRONG>2</STRONG></TD><TD>100 µL</TD><TD>1 mM</TD></TR><TR><TD><STRONG>3</STRONG></TD><TD>50 µL</TD><TD>–</TD></TR><TR><TD><STRONG>4</STRONG></TD><TD>50 µL</TD><TD>1 mM</TD></TR><TR><TD><STRONG>5</STRONG></TD><TD>25 µL</TD><TD>–</TD></TR><TR><TD><STRONG>6</STRONG></TD><TD>25 µL</TD><TD>1 mM</TD></TR><TR><TD><STRONG>7</STRONG></TD><TD>–</TD><TD>–</TD></TR><TR><TD><STRONG>8</STRONG></TD><TD>–</TD><TD>1 mM</TD></TR></TBODY></TABLE></DIV><P>After 5 h, the cells with the following OD<SUB>600</SUB> were harvested (1.5 mL) with 13,000 rpm for 10 min at 4°C.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB> after 5 h</TH></TR><TR><TD><STRONG>1</STRONG></TD><TD class="article-table_right">0.57</TD></TR><TR><TD><STRONG>2</STRONG></TD><TD class="article-table_right">0.5</TD></TR><TR><TD><STRONG>3</STRONG></TD><TD class="article-table_right">0.59</TD></TR><TR><TD><STRONG>4</STRONG></TD><TD class="article-table_right">0.5</TD></TR><TR><TD><STRONG>5</STRONG></TD><TD class="article-table_right">0.59</TD></TR><TR><TD><STRONG>6</STRONG></TD><TD class="article-table_right">0.48</TD></TR><TR><TD><STRONG>7</STRONG></TD><TD class="article-table_right">0.61</TD></TR><TR><TD><STRONG>8</STRONG></TD><TD class="article-table_right">0.51</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Sequencing_of_biobricks">Sequencing of biobricks</SPAN></H3><P class="notebook-head_date">31.07.18</P></DIV><DIV class="notebook-content"><P>Biobricks <MARK>2</MARK>, <MARK>3</MARK>, <MARK>4</MARK>, <MARK>6</MARK>, <MARK>9</MARK>, <MARK>10</MARK>, and <MARK>12</MARK> were sent for sequencing with primers iGEM2018_57 and iGEM2018_58.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>