Team:Goettingen/Notebook/May
Contents
1
2
3 Identification of upregulated proteins in presence of glyphosate in Bacillus subtilis
4
4.1 Materials for the Bradford assay:
4.2 Procedure:
4.3 The protein expression was then further analysed by SDS-PAGE and subsequent silver staining.
4.3.1 The Material
4.3.2 The procedure
4.3.3 Components for gel 15 %, 12 %, 10 %
4.3.4 Stacking gel for denaturing SDS PAGE (5 %; enough for two gels)
5 Silver staining
5.1 The material
5.2 The procedure
5.3 Results
6 Identification of novel mutations in B. subtilis 168
7 Amplification of the gltT gene
7.1 Results
8 Plasmid amplification and cloning experiments for reporter constructs
8.1 Results
9 Preparing competent E. coli cells/PtrpE/P amplification
9.1 Preparation of competent E. coli strains DH5α and XL1-Blue
9.1.1 The Material
9.1.2 The Procedure
10 New Primers for gltT and sequencing
10.1 Results
11 Test transformation of the new competent E. coli cells
12 Evaluation of the test transformation
13 Inoculation of transformed strains
14 Preparation of the plasmids for further work
14.1 The Material
14.2 The Procedure
14.3 Reaction mixtures to digest the fragments
14.4 Reaction mixtures to digest the plasmids
14.4.1 pGP172
14.4.2 pBQ200
14.4.3 pAC7
15 Construction of plasmids
15.1 Ligation of PtrpP into pAC7
15.2 Ligation of PtrpE into pAC7
15.3 Ligation of aroE_1 into pGP172
15.4 Ligation of aroE_2 into pBQ200
15.5 Ligation of Ptrp into pAC7
15.6 Re-Ligation of vector pAC7
15.7 Re-Ligation of vector pGP172
15.8 Re-Ligation of vector pBQ200
16 Transformation of DH5α
02.05.18
Cryo cultures of strains 168, BP233, BP234 were prepared with 90% LB medium and 10% DMSO. The cells were frozen and stored at -80°C.
03.05.18
Strains BP233, BP235 and 168 (WT) were incubated in 4 ml LB medium for 4 h at 37°C and 220 rpm. 2 ml of the culture was harvested, and the cells were washed in 1xC-salts and then resuspended in 300 µl 1xC-salts. BP233 and BP235 were plated on CS-glucose with and without 10 mM glyphosate.
<img src="
">
As seen in this figure, both strains grow well on CS-glucose without glyphosate. Strain BP233 also forms a lawn on plates with 10mM glyphosate, while the wildtype does not form a lawn, but single colonies, which could be suppressor mutants. This leads to the hypothesis that GltT, which is a glutamate transporter, transports glyphosate into the cell. The strain BP235 was not stable, so it was constructed again!
Identification of upregulated proteins in presence of glyphosate in Bacillus subtilis
07.05.18
The two B. subtilis strains 168 and the tryptophan prototroph strain SP1 were grown at different glyphosate concentrations. To identify upregulated protein expression in the presence of glyphosate.
For cultivation, the two strains were inoculated in 4 mL LB medium and incubated overnight at 37°C (200 rpm) The overnight cultures were transferred into 4 mL fresh LB medium and grown for 2 h at 28°C (200 rpm).
CS-Glucose medium was prepared with and without tryptophan
Components
Volume (mL)
5 × C salts
10
Iron Ferric Ammonium Citrate (CAF; 2.2 mg/mL)
0.5
III´ salts
0.5
Sodium Succinate (30%)
1
50% glucose
0.5
(Tryptophan
0.5)
Ad to 50 mL with sterile water
100
B. subtilis 168 and B. subtilis SP1 were transferred into CS-Glucose medium containing increasing glyphosate concentrations to an OD600 of 0.1. OD measurement was carried out every 2 hours until the cells reached an OD600 of ∼2. The cells were harvested by centrifugation for 1 min at 13000 rpm and stored at -20°C.
<img src="
">
<img src="
">
Growth curves of the wild type SP1 and the 168 mutant, auxotrophic for tryptophan
14.05.18
Proteins were isolated from cultures by lysing the cells with 50 µL lysis buffer and a Bradford assay was carried out to determine the protein concentration.
Materials for the Bradford assay:
Bradford solution, 5-fold concentration
buffer from protein elution
Procedure:
Prespare for each portein mixture 3 samples with different concentrations.
Sample
Reference sample
800 µL H2O
800 µL H2O
0.5, 1.0 & 2.0 µL crude extract
no protein but equal volume of buffer
200 µL Bradford Solution
200 µL Bradford Solution
Sodium Succinate (30%)
1
50% glucose
0.5
(Tryptophan
0.5)
Ad to 50 mL with sterile water
100
After mixing the samples with the protein solutions, the mixtures have to be incubated for 5 minutes at room temperature. Transfer the mixtures and the reference sample to 1.5 mL cuvettes and measure the absorption at a wavelength of 595 nm.
Calculate the amount of protein in your sample using the following formula:
<img class="article_formula"               src="
">
Protein concentration:
Sample
Protein concentration (µg/mL)
168 in 0 mM Glyphosate
0.375
168 in 0.5 mM Glyphosate
1.049
168 in 1.5 mM Glyphosate
1.049
SP1 in 0 mM
0.159
SP1 in 0.5 mM
0.496
SP1 in 1.5 mM
1.4045
The protein expression was then further analysed by SDS-PAGE and subsequent silver staining.
The Material
6× SDS loading dye (98 C)
6× SDS loading dye (98 C)
10× Running (PAGE) buffer (98 S)
1.5 M Tris-HCl, pH 8.8 (95 B)
1.56 M Tris-HCl, pH 6.8 (95 B)
TEMED/TMEDA (Tetramethylethylenediamine)
Acrylamide:Bis-Acrylamide (37.5:1, TOXIC!!!)
APS (Ammonium persulfate, 10 %)
SDS (Sodium dodecyl sulfate, 10 %)
Power supply & SDS PAGE devices
Iso-propanol (100 %)
The procedure
Clean the glass plates, combs and mats with ethanol and assemble the glass plate sandwich. Be sure that the plates are well aligned. Place the plates in the casting frame and tighten them. Place the casting frames into the casting stands.
Wear gloves for making the gels because acrylamide is a neurotoxin in its unpolymerized form!!! Running gel for denaturing SDS PAGE (enough for 2 gels).
Components for gel 15 %, 12 %, 10 %
H2O 2.3 mL 3.3 mL 4.0 mL
Acryl-Bisacrylamide 30 % 5.0 mL 4.0 mL 3.3 mL
1.5 M Tris-HCl, pH 8.8 2.5 mL 2.5 mL 2.5 mL
SDS solution 10 % 0.1 mL 0.1 mL 0.1 mL
APS 10 % 0.1 mL 0.1 mL 0.1 mL
TEMED 8 µl 8 µl 8 µl
Stacking gel for denaturing SDS PAGE (5 %; enough for two gels)
H2O 6.83 mL
Acryl-Bisacrylamide 30 % 1.3 mL
1.5 M Tris-HCl, pH 6.8 0.87 mL
SDS solution 10 % 0.1 mL
APS 10 % 0.1 mL
TEMED 10 µl
Mix the ingredients of the running gel and pour the solution quickly (gel starts to polymerize once TEMED has been added) into the gap between the glass plates. Leave enough room for the stacking gel – appr. 1 cm below the bottom of the comb´s teeth. After a minute you can overlay the solution with iso-propanol or ethanol. This excludes the access of oxygen and plane the surface
After polymerization, pour off the iso-propanol and rinse with distilled water. Mix the reagents for the stacking gel and pour it on top of the running gel till the space is completely full. Then insert the comb. No alcohol overlay is needed due to the comb.
After polymerization clamp the gel into the electrophoresis apparatus and fill it with running buffer. Remove the comb carefully and use a pipette or syringe to wash the wells with running buffer.
The protein samples (up to 15 µl) are mixed with SDS loading buffer (2.5 µl) and incubated for 10 min at 95°C.
After heating briefly centrifuge the samples to spin down the condensed water.
Then you can apply the samples on a SDS PAGE with a pipette or Hamilton microliter syringe. Do not forget to add molecular weight marker in one lane.
Run the gel at 80 V until the front has moved into the running gel. Then you can increase the voltage to 120 V. Run the gel until the running front has reached the bottom of the running gel.
Pry the plates apart by using a gel spacer. The gel can then be stained (Coomassie staining, method 56 or Silver staining, method 57) to visualize the separated proteins or processed further (Western Blotting, method 58).
Silver staining
15.05.18
The material
FA-Fixation solution (98 L)
Developer (98 M)
Impregnator (98 N)
Thiosulfate solution (98 O)
Stop solution (98 P)
Ethanol (50 %)
The procedure
The protein bands are stained according to method of Nesterenko et al. (1994). For that purpose, the polyacrylamide gels were incubated on a shaker with the following reagents and in the stated order.
Step
Solution
Time
Fixing
Fixation solution
1–24 h
Washing
EtOH (50 %)
20 min, 3×
Reduction
Thiosulfate solution
1.5 min
Washing
deion. H2O
20 s, 3×
Staining
Impregnator
15–25 min
Washing
deion. H2O
20 s, 3×
Development
Developer
until sufficient staining
Washing
deion. H2O
20 s, 2×
Stop
solution
5 min
Results
<img src="
">
&rightarrow; No significant difference in protein expression could be detected.
Identification of novel mutations in B. subtilis 168
16.05.18
For the identification of novel mutations in the gltT transporter, the wild type strain was grown on 40 mM glyphosate. Emerging mutations were further analysed by isolation of the complete DNA and amplification of the transporter via PCR. The genomic DNA was isolated with the Kit “peqGOLD Bacterial DNA Kit” from peqlab. The resulting sequences were then screened for novel mutations.
For the identification of novel mutations in the gltT transporter, the wild type strain was grown on 40 mM glyphosate. Emerging mutations were further analysed by isolation of the chromosomal DNA and amplification of the transporter via PCR. The genomic DNA was isolated with the Kit “peqGOLD Bacterial DNA Kit” from peqlab.
<img src="
">
Mutant names
DNA concentration in [ng/µL]
iGEM18
15.2
iGEM19
44.5
iGEM20
57.4
iGEM21
48.3
iGEM22
45.3
iGEM23
54.3
iGEM24
44.7
iGEM25
31.9
iGEM26
29.7
iGEM27
80.7
iGEM28
58.8
iGEM29
51.9
iGEM30
57.7
iGEM31
/
iGEM32
56.1
iGEM33
62.2
iGEM34
42.5
Amplification of the gltT gene
17.05.18
We amplified the gltT gene from B. subtilis with a PCR run with the following recipe.
Component
Volume in [µL]
10x Taq buffer
5
dNTPs (12.5 mM)
2
fwd Primer: iGEM2018_15
2
rev Primer: iGEM2018_16
2
B. subtilis chromosomal DNA
1
Dream Taq polymerase
0.25
sterile H2O
37.75
Total
50
Results
In the following figures are the results of the PCR shown.
<img src="
">
<img src="
">
Amplified gltT gene from 168, iGEM19/20.
Amplified gltT gene from iGEM30/32/33/34.
<img src="
">
Amplified gltT gene from 168, iGEM18/21/22/23/24/25/26/27/28/29.
&rightarrow; iGEM21 and iGEM26 did not show any PCR products. Therefore, no sequence could be further analyzed.
Plasmid amplification and cloning experiments for reporter constructs
18.05.18
To produce a reporter system, we needed to identify promoters from genes that are induced upon glyphosate treatment. Our hypothesis is that the promoter of trpE and trpP are induced by glyphosate, as described in the design section (TO DO, link hier einfügen!!!). Therefore, PtrpE as well as two variants of the EPSP-synthase (encoded by aroE were amplified by PCR using primers iGEM2018_25/26 and iGEM2018_27/28, respectively.
Component
Volume in [µL]
5x HF buffer
10
dNTPs (12.5 mM)
2
fwd Primer: iGEM2018_25 or iGEM2018_27
2
rev Primer: iGEM2018_26 or iGEM2018_28
2
B. subtilis chromosomal DNA
1
Phusion DNA polymerase
0.25
sterile H2O
32.75
Total
50
Results
In the following figure are the results of the PCR shown.
<img src="
">
Lane 1: aroE_1. Lane 2: PtrpE. Lane 3: aroE_2.
Preparing competent E. coli cells/PtrpE/P amplification
22.05.18
The PCR amplification was repeated including PtrpP. The following primer combinations were used.
Primers
Product
iGEM2018_1 + iGEM2018_2
aroE_1
iGEM2018_23 + iGEM2018_24
PtrpP
iGEM2018_25 + iGEM2018_26
PtrpE
iGEM2018_27 + iGEM2018_28
aroE_2
Template
chromosomal DNA from strain iGEM21
The following figure shows the results of the PCR.
<img src="
">
Lane 1: aroE_1. Lane 2: PtrpP. Lane 3: PtrpP. Lane 4: PtrpE. Lane 5: aroE_2.
Samples from today in addition to the samples on 18.05.18 were purified using the QIAquick PCR Purification Kit. The DNA was eluted in 35 µL H2O.
Concentration of the purified DNA was measured at the Nanodrop device.
Sample
DNA concentration in [ng/µL]
aroE_1
90.1
aroE_2
68.1
PtrpE_1
89.5
PtrpE_2
67.8
PtrpP_1
90.2
PtrpP_2
43.6
Preparation of competent E. coli strains DH5α and XL1-Blue
The preparation was carried out over the following days.
The Material
TB buffer (ice-cold)
liquid LB medium
SOB medium
1 M MgCl2
1 M MgSO4
DMSO (100%)
The Procedure
Inoculate 20 mL LB medium either with a single colony or from a cryo stock of the desired E. coli strain and incubate the culture with agitation for 20 h at 28°C.
Inoculate 250 mL SOB medium supplemented with 10 mM MgCl2 and 10 mM MgSO4 in a 2 L shake flask and grow the cells to an OD600 of 0.5–0.9 (20–24 h) at 18°C and 200–250 rpm.
Incubate the whole flask for 10 min on ice. Collect the cells by centrifugation for 10 min at 4°C and 5,000 rpm. Re-suspend the cell in 80 mL of ice-cold TB buffer and incubate them for 10 min on ice. Collect the cells by centrifugation for 5 min at 5,000 rpm.
Re-suspend the cells in 20 mL of ice-cold TB buffer. Add DMSO to a final concentration of 7% (1.4 mL) by gently shaking the falcon tube.
Transfer 0.2 mL aliquots into labelled Eppendorf reaction tubes and freeze the cells in liquid nitrogen. Store the cells at -80°C. The volume of the pre-culture strongly depends on the E. coli strain. Using DH5α, it is recommended to use 4 mL pre-culture and inoculate at 8 a.m. Nex day around 10 a.m. this strain should have reached an OD600 between 0.5 and 0.6. If you use the strain XL1-Blue, you will need less cells and it is sufficient to inoculate the SOB medium around lunchtime.
New Primers for gltT and sequencing
23.05.18
The PCR from 17.05.18 was repeated with new primers (iGEM2018_19 and iGEM2018_20) that amplify also the ends of the gene gltT. The protocol was the same as before.
Results
<img src="
">
<img src="
">
Amplified gltT gene from iGEM18–iGEM29.
Amplified gltT gene from iGEM30–iGEM34.
The amplified gltT fragments were purified and sent to sequencing with primers iGEM2018_29 and iGEM2018_30. The following table provides an overview of mutations in gltT from glyphosate adapted mutants.
gltT sequence
Mutation
Consequence
Wild type
—
—
iGEM18
Δ669A
Truncation
iGEM19
Δ669A
Truncation
iGEM20
C211T
Truncation
iGEM22
duplication 839–847
Insertion
iGEM23
Δ65T
Truncation
iGEM24
deletion 898–923
Truncation
iGEM25
duplication 839–847
Insertion
iGEM27
A388T
Truncation
iGEM29
duplication 839–847
Insertion
iGEM30
Δ996&nash;1090
Truncation
iGEM32
Δ669A
Truncation
iGEM33
duplication 839–847
Insertion
Test transformation of the new competent E. coli cells
24.05.18
We performed a test transformation to prove that the cells were actually competent. The following plasmids were used.
pBQ200
pGP172
pAC7
pAC5
<img src="
">
Overview how a transformation of competent E. coli cells works with plasmids. The agar plates should be made of complex medium (like LB) and contain the appropriate antibiotics.
Evaluation of the test transformation
25.05.18
The transformed cells were incubated over night at 37°C and the colonies were colony forming units were counted.
Strain
Colony forming units
DH5α + pBQ200
1360
DH5α + pGP172
1520
DH5α + pAC7
1440
XL1-Blue + pBQ200
180
XL1-Blue + pGP172
208
XL1-Blue + pAC7
80
XL1-Blue + pAC5
22
Inoculation of transformed strains
28.05.18
4 mL LB medium were supplemented with the appropriate antibiotics and inoculated with the transformed strains (25.05.18). The cultures were incubated over night with agitation at 37°C.
Preparation of the plasmids for further work
29.05.18
The isolation of the plasmids was performed with the NucleoSpin® kit from Macherey and Nagel.
The Material
Liquid LB medium
NucleoSpin® plasmid kit
The Procedure
Use single colonies from your transformation plates to inoculate 4 mL (15–20 mL for low-copy plasmids) liquid LB medium supplemented with the appropriate antibiotic and grow the culture overnight at 37°C with agitation.
Collect the cells from 1.5 mL (15 mL for low-copy plasmids) of the overnight cultures by centrifugation (1 min, 13,000 rpm). Re-suspend the pellet in 250 mL of buffer A1, add 250 mL of buffer A2, mix gently 6–8 times and incubate the suspension for 5 min at room temperature. For the isolation of low-copy plasmids add 500 mL of buffers A1 and A2 to the cells.
Add 300 mL (0.6 mL for low-copy plasmids) of buffer A3, mix gently 6–8 times and centrifuge the lysate for 5–10 min at 13,000 rpm.
Transfer the supernatant into NucleoSpin® plasmid column, centrifuge for 1 min at 13,000 rpm and discard the flow-through.
OPTIONAL: For sequencing of low-copy plasmid DNA add 500 mL buffer AW to the column, centrifuge for 1 min at 13000 rpm and discard the flow-through.
Add 600 mL of buffer A4, centrifuge (1 min, 13,000 rpm) and discard the flow-through.
To dry the silica membrane, centrifuge again for 2 min at 13,000 rpm.
Add 50 mL of sterile and deionized water on top of the silica membrane of the column incubate the column for 1–5 min at room temperature and elute the DNA by centrifugation for 1 min at 13,000 rpm.
The plasmid DNA is highly pure and can be stored at -20°C.
The concentration of the isolated plasmids was determined with the Nanodrop device.
Plasmid
DNA concentration in [ng/µL]
pBQ200
186.6
pGP172
84.6
pAC7
114.1
pAC5
75.9
For the cloning procedure, a restriction digest was carried out with aroE_1, aroE_2, PtrpP, and PtrpE fragments (from 18.05.18 and 22.05.18).
Reaction mixtures to digest the fragments
Compound
Volume in [µL]
aroE_1
20
SacI
2
BamHI
2
FD buffer
3
H2O
3
Incubate for 45 min at 37°C
Compound
Volume in [µL]
aroE_2
20
SalI
2
BamHI
2
FD buffer
3
H2O
3
Incubate for 45 min at 37°C
Compound
Volume in [µL]
PtrpP
20
EcoRI
2
BamHI
2
FD buffer
3
H2O
3
Incubate for 30 min at 37°C
Compound
Volume in [µL]
PtrpE
20
SacI
2
BamHI
2
FD buffer
3
H2O
3
Incubate for 30 min at 37°C
The samples were then purified with the PCR purification clean up Kit and eluted in 25 µL sterile H2O. The concentrations of the digested fragments were determined using the Nanodrop device.
Plasmid
DNA concentration in [ng/µL]
aroE_1
63
aroE_2
43.8
PtrpP
45.7
PtrpE
47.7
Reaction mixtures to digest the plasmids
pGP172
Compound
Volume in [µL]
pGP172
18
BamHI
3
FD buffer
3
H2O
6
Incubate for 45 min at 37°C
Inactivation at 80°C for 5 min
The digest was tested on an agarose gel.
Afterwards, the following mixture was prepared.
Compound
Volume in [µL]
Digested pGP172
28
SacI
3
FD buffer
1
H2O
8
Incubate for 45 min at 37°C
Inactivation at 80°C for 5 min
The digest was finally incubated with 1 µL "Fast AP" for 10 min at 37°C and eluted in 25 µL H2O.
pBQ200
Compound
Volume in [µL]
pBQ200
15
BamHI
3
FD buffer
3
H2O
9
Incubate for 45 min at 37°C
Inactivation at 80°C for 5 min
The digest was tested on an agarose gel.
Afterwards, the following mixture was prepared.
Compound
Volume in [µL]
Digested pBQ200
28
SalI
3
FD buffer
1
H2O
8
Incubate for 45 min at 37°C
Inactivation at 80°C for 5 min
The digest was finally incubated with 1 µL "Fast AP" for 10 min at 37°C and eluted in 25 µL H2O.
pAC7
Compound
Volume in [µL]
pAC7
20
BamHI
3
FD buffer
3
H2O
4
Incubate for 45 min at 37°C
Inactivation at 80°C for 5 min
The digest was tested on an agarose gel.
Afterwards, the following mixture was prepared.
Compound
Volume in [µL]
Digested pAC7
28
SalI
3
FD buffer
1
H2O
8
Incubate for 30 min at 37°C
Inactivation at 80°C for 5 min
The digest was finally incubated with 1 µL "Fast AP" for 10 min at 37°C and eluted in 30 µL H2O.
Construction of plasmids
30.05.18
Concentration of the purified plasmids was analyzed with the Nanodrop device.
Plasmid
DNA concentration in [ng/µL]
pGP172
36.3
pBQ200
71.2
pAC7
41.3
In the next step, the fragments and vectors were ligated using 100 ng of the vector and calculating the concentration of the insert dependent on its length.
pAC7 (10573 bp) → 0.5 µL insert
pGP172 (4600 bp) → 1.5 µL insert
pBQ200 (6700 bp) → 1.5 µL insert
Ligation of PtrpP into pAC7
Compound
Volume in [µL]
Vector (pAC7)
2.5
Insert (PtrpP)
0.5
T4 buffer
2
T4 Ligase
1
H2O
14
Total volume
20
Ligation of PtrpE into pAC7
Compound
Volume in [µL]
Vector (pAC7)
2.5
Insert (PtrpE)
0.5
T4 buffer
2
T4 Ligase
1
H2O
14
Total volume
20
Ligation of aroE_1 into pGP172
Compound
Volume in [µL]
Vector (pGP172)
2.5
Insert (aroE_1)
1.5
T4 buffer
2
T4 Ligase
1
H2O
13
Total volume
20
Ligation of aroE_2 into pBQ200
Compound
Volume in [µL]
Vector (pBQ200)
1.5
Insert (aroE_2)
1.5
T4 buffer
2
T4 Ligase
1
H2O
14
Total volume
20
Ligation of Ptrp into pAC7
Compound
Volume in [µL]
Vector (pAC7)
2.5
Insert (PtrpP)
0.5
T4 buffer
2
T4 Ligase
1
H2O
14
Total volume
20
Re-Ligation of vector pAC7
Compound
Volume in [µL]
Vector (pAC7)
2.5
Insert
–
T4 buffer
2
T4 Ligase
1
H2O
14.5
Total volume
20
Re-Ligation of vector pGP172
Compound
Volume in [µL]
Vector (pGP172)
2.5
Insert
–
T4 buffer
2
T4 Ligase
1
H2O
14.5
Total volume
20
Re-Ligation of vector pBQ200
Compound
Volume in [µL]
Vector (pBQ200)
1.5
Insert
–
T4 buffer
2
T4 Ligase
1
H2O
15.5
Total volume
20
All samples were incubated at 16°C over night.
Transformation of DH5α
31.05.18
Competent E. coli cells from the strain DH5α were transformed with the ligation samples. Also, a negative controll without DNA was prepared, were the cells received only water. The transformed cells were plated on LB medium agar containing ampicillin and X-Gal in the following manner: on one plate 50 µL and on the second palte 100 µL. The cells were incubated over night at 37°C.
