<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Notebook_reporterSystems_June skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Notebook/reporterSystems June</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="notebook-item"><DIV class="notebook-head"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Evaluation_of_transformation_and_preparation_of_lacZ_measurements"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Evaluation of transformation and preparation of lacZ measurements</SPAN></A></LI><LI class="toclevel-2"><A href="#Preparation_of_lacZ_measurements"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Preparation of lacZ measurements</SPAN></A></LI><LI class="toclevel-3"><A href="#LD-Mix"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">LD-Mix</SPAN></A></LI><LI class="toclevel-3"><A href="#Z_buffer"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Z buffer</SPAN></A></LI><LI class="toclevel-3"><A href="#ONPG"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">ONPG</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Preparation_and_test_digest_of_new_constructed_plasmids"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Preparation and test digest of new constructed plasmids</SPAN></A></LI><LI class="toclevel-2"><A href="#Mastermix_I"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Mastermix I</SPAN></A></LI><LI class="toclevel-2"><A href="#Mastermix_II"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Mastermix II</SPAN></A></LI><LI class="toclevel-2"><A href="#Mastermix_III"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Mastermix III</SPAN></A></LI><LI class="toclevel-1"><A href="#Sequencing_of_plasmids"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Sequencing of plasmids</SPAN></A></LI><LI class="toclevel-2"><A href="#pGP172_aroE_1"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">pGP172_aroE_1</SPAN></A></LI><LI class="toclevel-2"><A href="#pBQ200_aroE_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">pBQ200_aroE_2</SPAN></A></LI><LI class="toclevel-2"><A href="#pAC7_PtrpE"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">pAC7_PtrpE</SPAN></A></LI><LI class="toclevel-2"><A href="#pAC7_PtrpP"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">pAC7_PtrpP</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparation_of_competent_B..C2.A0subtilis_cells"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Preparation of competent B. subtilis cells</SPAN></A></LI><LI class="toclevel-2"><A href="#The_Material"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">The Material</SPAN></A></LI><LI class="toclevel-2"><A href="#The_Procedure_.E2.80.93_Preparation_of_competent_cells"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">The Procedure – Preparation of competent cells</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_B..C2.A0subtilis"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Transformation of B. subtilis</SPAN></A></LI><LI class="toclevel-2"><A href="#Sequencing_results_from_05.06.18"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Sequencing results from 05.06.18</SPAN></A></LI><LI class="toclevel-1"><A href="#Evaluation_of_transformation"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Evaluation of transformation</SPAN></A></LI><LI class="toclevel-2"><A href="#Determination_of_.CE.B1-amylase"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Determination of α-amylase</SPAN></A></LI><LI class="toclevel-2"><A href="#Starch_plates"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Starch plates</SPAN></A></LI><LI class="toclevel-2"><A href="#5x_Lugol.27s_solution"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">5x Lugol's solution</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_of_PtrpP_into_pAC7"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Ligation of PtrpP into pAC7</SPAN></A></LI><LI class="toclevel-1"><A href="#Naming_of_plasmids_and_evaluation_of_the_ligation"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Naming of plasmids and evaluation of the ligation</SPAN></A></LI><LI class="toclevel-2"><A href="#Renaming_of_plasmids"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Renaming of plasmids</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_of_ligated_PtrpPinto_pAC7"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Results of ligated PtrpPinto pAC7</SPAN></A></LI><LI class="toclevel-1"><A href="#Disc_diffusion_assay"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Disc diffusion assay</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-galactosidase_assay"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">β-galactosidase assay</SPAN></A></LI><LI class="toclevel-2"><A href="#General_procedure_to_determine_.CE.B2-galactosidase_activity."><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">General procedure to determine β-galactosidase activity.</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparation_of_cell_pellets_for_.CE.B2-galactosidase.E2.80.93assay"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Preparation of cell pellets for β-galactosidase–assay</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-galactosidase.E2.80.93assay"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">β-galactosidase–assay</SPAN></A></LI><LI class="toclevel-1"><A href="#Preparation_for_RNA_sequencing"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Preparation for RNA sequencing</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation_of_PtrpP_into_pAC7_2"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Ligation of PtrpP into pAC7</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-galactosidase_assay_2"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">β-galactosidase assay</SPAN></A></LI><LI class="toclevel-1"><A href="#Isolation_of_plasmids_.28pAC7_PtrpP.29"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Isolation of plasmids (pAC7_PtrpP)</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation_of_SP1"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Transformation of SP1</SPAN></A></LI><LI class="toclevel-1"><A href="#Isolation_of_chromosomal_DNA"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Isolation of chromosomal DNA</SPAN></A></LI><LI class="toclevel-2"><A href="#Isolation_of_chromosomal_DNA_from_Bacillus.C2.A0subtilis"><SPAN class="tocnumber">17.1</SPAN><SPAN class="toctext">Isolation of chromosomal DNA from Bacillus subtilis</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformations"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Transformations</SPAN></A></LI></DIV><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Evaluation_of_transformation_and_preparation_of_lacZ_measurements">Evaluation of transformation and preparation of <EM>lacZ</EM> measurements</SPAN></H3><P class="notebook-head_date">01.06.18</P></DIV><DIV class="notebook-content"><P>We counted the colony forming units (cfu) from the transformation.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Transformation</TH><TH>Cfu</TH><TH>Cfu (concentrated)</TH></TR><TR><TD>DH5α + pAC7_<EM>PtrpP</EM></TD><TD>1 white/4 blue</TD><TD>44 white/3 blue</TD></TR><TR><TD>DH5α + pBQ200_<EM>aroE_2</EM></TD><TD>1 white/17 blue</TD><TD>48 white/0 blue</TD></TR><TR><TD>DH5α + pAC7_<EM>PtrpE</EM></TD><TD>1 white/35 blue</TD><TD>60 white/32 blue</TD></TR><TR><TD>DH5α + pGP172_<EM>aroE_1</EM></TD><TD>61</TD><TD>120</TD></TR></TBODY></TABLE></DIV><P>On the Re-Ligation plates were on all plates only 1 or 2 colonies visible. The negative controll plates did not show any colonies.</P><H5><SPAN class="mw-headline" id="Preparation_of_lacZ_measurements">Preparation of <EM>lacZ</EM> measurements</SPAN></H5><H6><SPAN class="mw-headline" id="LD-Mix">LD-Mix</SPAN></H6><UL><LI>100 mg Lysozyme</LI><LI>10  DNase I</LI><LI>Ad to 10 mL with deionized H<SUB>2</SUB>O</LI><LI>Store 500 µL aliquots at -20°C</LI></UL><H6><SPAN class="mw-headline" id="Z_buffer">Z buffer</SPAN></H6><UL><LI>60 mM Na<SUB>2</SUB>HPO<SUB>4</SUB></LI><LI>40 mM NaH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>10 mM KCl</LI><LI>1 mM MgSO<SUB>4</SUB></LI><LI>50 mM β-Mercaptoethanol (toxic!)(175 mL β-Mercaptoethanol to 50 mL Z buffer.)</LI></UL><H6><SPAN class="mw-headline" id="ONPG">ONPG</SPAN></H6><UL><LI>4 mg o-Ntrophenyl-β-D-Galactopyranoside (ONPG)</LI><LI>Ad to 1 mL with Z buffer without βMercaptoethanol</LI></UL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparation_and_test_digest_of_new_constructed_plasmids">Preparation and test digest of new constructed plasmids</SPAN></H3><P class="notebook-head_date">04.06.18</P></DIV><DIV class="notebook-content"><P>The plasmids were isolated using NucleoSpin® plasmid kit, Macherey&amp;Nagel as previously described. The concentration of the purified plasmids was analysed with the Nanodrop device.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Plasmid</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD>pGP172_<EM>aroE_1</EM> sample 1</TD><TD class="article-table_right">138.8</TD></TR><TR><TD>pGP172_<EM>aroE_1</EM> sample 2</TD><TD class="article-table_right">121.8</TD></TR><TR><TD>pGP172_<EM>aroE_1</EM> sample 3</TD><TD class="article-table_right">154</TD></TR><TR><TD>pGP172_<EM>aroE_1</EM> sample 4</TD><TD class="article-table_right">123.1</TD></TR><TR><TD>pPQ200_<EM>aroE_2</EM> sample 1</TD><TD class="article-table_right">823.8</TD></TR><TR><TD>pPQ200_<EM>aroE_2</EM> sample 2</TD><TD class="article-table_right">485.7</TD></TR><TR><TD>pPQ200_<EM>aroE_2</EM> sample 3</TD><TD class="article-table_right">815.2</TD></TR><TR><TD>pPQ200_<EM>aroE_2</EM> sample 4</TD><TD class="article-table_right">926.2</TD></TR><TR><TD>pAC7_<EM>PtrpE</EM> sample 1</TD><TD class="article-table_right">192</TD></TR><TR><TD>pAC7_<EM>PtrpE</EM> sample 2</TD><TD class="article-table_right">231.9</TD></TR><TR><TD>pAC7_<EM>PtrpP</EM> sample 1</TD><TD class="article-table_right">193.3</TD></TR><TR><TD>pAC7_<EM>PtrpP</EM> sample 2</TD><TD class="article-table_right">202.7</TD></TR></TBODY></TABLE></DIV><P>The isolated plasmids were digested to check whether there is an insert.</P><H6><SPAN class="mw-headline" id="Mastermix_I">Mastermix I</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>BamH</EM>I</TD><TD class="article-table_right">2.5</TD></TR><TR><TD><EM>Sac</EM>I</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>FD buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">30</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 15 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Mastermix_II">Mastermix II</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>BamH</EM>I</TD><TD class="article-table_right">2.5</TD></TR><TR><TD><EM>EcoR</EM>I</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>FD buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">30</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 15 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Mastermix_III">Mastermix III</SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD><EM>BamH</EM>I</TD><TD class="article-table_right">2.5</TD></TR><TR><TD><EM>Sal</EM>I</TD><TD class="article-table_right">2.5</TD></TR><TR><TD>FD buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">30</TD></TR><TR class="article_table_hline"><TD><STRONG>Incubate for 15 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><P>The samples were put on an agarose gel. The following figure shows the results.</P><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Lane 1–4: pGP_<EM>aroE_1</EM>. Lane 5–8: pBQ200_<EM>aroE_2</EM>. Lane 9&amp;10: pAC7_<EM>PtrpE</EM>. Lane 11&amp;12: pAC7_<EM>PtrpP</EM>.</P></DIV><P>The following samples were selected and stored at -20°C.</P><UL><LI>pGP172_<EM>aroE_1</EM> sample 1</LI><LI>pBQ200_<EM>aroE_2</EM> sample 3</LI><LI>pBQ200_<EM>aroE_2</EM> sample 4</LI><LI>pAC7_<EM>PtrpE</EM> sample 1</LI><LI>pAC7_<EM>PtrpE</EM> sample 2</LI><LI>pAC7_<EM>PtrpP</EM> sample 1</LI><LI>pAC7_<EM>PtrpP</EM> sample 2</LI></UL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Sequencing_of_plasmids">Sequencing of plasmids</SPAN></H3><P class="notebook-head_date">05.06.18</P></DIV><DIV class="notebook-content"><P>The isolated plasmids were prepared for sequencing with the following mixtures.</P><H6><SPAN class="mw-headline" id="pGP172_aroE_1">pGP172_<EM>aroE_1</EM></SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>DNA</TD><TD class="article-table_right">7</TD></TR><TR class="article_table_hline"><TD><OL><LI>iGEM2018_1</LI><LI>T7 fwd.</LI><LI>T7 rev.</LI></OL></TD><TD class="article-table_right">6</TD></TR><TR class="article_table_hline"><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">2</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="pBQ200_aroE_2">pBQ200_<EM>aroE_2</EM></SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>DNA</TD><TD class="article-table_right">1</TD></TR><TR class="article_table_hline"><TD><OL><LI>iGEM2018_27</LI><LI>FX125</LI><LI>M13-pac fwd</LI></OL></TD><TD class="article-table_right">6</TD></TR><TR class="article_table_hline"><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">8</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="pAC7_PtrpE">pAC7_<EM>PtrpE</EM></SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>DNA</TD><TD class="article-table_right">5</TD></TR><TR class="article_table_hline"><TD><OL><LI>ML107</LI><LI>lacZ rev.</LI></OL></TD><TD class="article-table_right">6</TD></TR><TR class="article_table_hline"><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">4</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="pAC7_PtrpP">pAC7_<EM>PtrpP</EM></SPAN></H6><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [µL]</TH></TR><TR><TD>DNA</TD><TD class="article-table_right">5</TD></TR><TR class="article_table_hline"><TD><OL><LI>ML107</LI><LI>lacZ rev.</LI></OL></TD><TD class="article-table_right">6</TD></TR><TR class="article_table_hline"><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">4</TD></TR></TBODY></TABLE></DIV><P>Primer were diluted 1:20 (10 µL primer and 190 µL H<SUB>2</SUB>O. Every sample contains between 800–1000 ng DNA in a volume of 15 µL.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparation_of_competent_B..C2.A0subtilis_cells">Preparation of competent <EM>B. subtilis</EM> cells</SPAN></H3><P class="notebook-head_date">06.06.18</P></DIV><DIV class="notebook-content"><H5><SPAN class="mw-headline" id="The_Material">The Material</SPAN></H5><UL><LI>LB medium</LI><LI>MN medium</LI><LI>MNGE medium</LI><LI>Expression mix</LI><LI>Glucose (20%)</LI><LI>1 M MgSO<SUB>4</SUB></LI><LI>CAA (10%)</LI></UL><H5><SPAN class="mw-headline" id="The_Procedure_.E2.80.93_Preparation_of_competent_cells">The Procedure – Preparation of competent cells</SPAN></H5><OL><LI>Inoculate 4 mL LB liquid medium with a single colony of a <EM>B. subtilis</EM> strain and incubate the culture over night at 28°C with agitation.</LI><LI>Use the overnight culture to inoculate 10 mL MNGE medium supplemented with 0.1% CAA in a 100 mL shake flask to an approximate OD<SUB>600</SUB> of about 1.3. This may take up to 5 h, depending on the strain.</LI><LI>Dilute the culture 1:1 with pre-warmed MNGE (w/o CAA) and incubate the culture for another hour at 37°C on a shaker.You can continue with the transformation of <EM>B. subtilis</EM> directly after nutritional starvation step (see Transformation of <EM>B. subtilis</EM>, step 2) or continue for long term storage.</LI><LI>Transfer 15 mL of the culture in sterile falcon tubes and harvest the cells by centrifugation for 5 min at 5,000 rpm. Transfer the supernatant into a sterile falcon tube.</LI><LI>Re-suspend the cells in 1.8 mL of the supernatant, add 1.2 mL 50% glycerine, mix the cell suspension and store the competent cells in 300 mL aliquots at -80°C.</LI></OL><H5><SPAN class="mw-headline" id="Transformation_of_B..C2.A0subtilis">Transformation of <EM>B. subtilis</EM></SPAN></H5><OL><LI>Thaw an aliquot of the frozen, competent bacteria and mix 300 µL of them with 1.7 mL 1x MN medium that has been supplemented with 43 µL glucose (20%) + 34 µL 1 M MgSO<SUB>4</SUB>.</LI><LI>Add 0.1–1 µg DNA (2 µg plasmid DNA) to 400 µL of the competent cells and incubate the reaction tube for 30 min at 37°C.</LI><LI>Add 100 µL expression mix and if required an inducer (IPTG, xylose, ...)</LI><LI>Incubate the bacteria for 1 h at 37°C with agitation and propagate the cells on SP medium agar plates supplemented with the appropriate antibiotics.</LI></OL><P>Strains SP1 and 168 were transformed with pAC7_<EM>PtrpE</EM> (10 μL). 100 and 200 μ of the transformed cells were propagate on SP medium containing kanamycin. A negativ controll was also performed.</P><H5><SPAN class="mw-headline" id="Sequencing_results_from_05.06.18">Sequencing results from 05.06.18</SPAN></H5><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Plasmid</TH><TH>Sequence result</TH></TR><TR><TD>pAC7_<EM>PtrpP</EM></TD><TD>ok</TD></TR><TR><TD>pBQ200_<EM>aroE_2</EM></TD><TD>ok</TD></TR><TR><TD>pAC7_<EM>PtrpE</EM></TD><TD>ok</TD></TR><TR><TD>pGP172_<EM>aroE_1</EM></TD><TD><STRONG>Mutation in binding region of the reverse primer → repeat!</STRONG></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Evaluation_of_transformation">Evaluation of transformation</SPAN></H3><P class="notebook-head_date">07.06.18</P></DIV><DIV class="notebook-content"><P>The following table shows the results of the transformation results.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>Concentration in [µL]</TH><TH>Cfu</TH></TR><TR><TD>168 + pAC7_<EM>PtrpE</EM></TD><TD>200</TD><TD>1680</TD></TR><TR><TD>168 + pAC7_<EM>PtrpE</EM></TD><TD>200</TD><TD>1820</TD></TR><TR><TD>168 + pAC7_<EM>PtrpE</EM></TD><TD>100</TD><TD>480</TD></TR><TR><TD>SP1 + pAC7_<EM>PtrpE</EM></TD><TD>200</TD><TD>112</TD></TR><TR><TD>SP1 + pAC7_<EM>PtrpE</EM></TD><TD>200</TD><TD>80</TD></TR><TR><TD>SP1 + pAC7_<EM>PtrpE</EM></TD><TD>100</TD><TD>42</TD></TR></TBODY></TABLE></DIV><P>No colonies were found on the negative control. Singularization of 3 clones respectively on SP medium containing kanamycin using wildtype SP1 and 168 as control.</P><H5><SPAN class="mw-headline" id="Determination_of_.CE.B1-amylase">Determination of α-amylase</SPAN></H5><P>2 single colonies of the <EM>B. subtilis</EM> transformants were propagated on a line on a starch plate. The plates were incubated over night at 37°C.</P><H5><SPAN class="mw-headline" id="Starch_plates">Starch plates</SPAN></H5><UL><LI>7.5 g Nutrient broth</LI><LI>5 g Starch</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><H5><SPAN class="mw-headline" id="5x_Lugol.27s_solution">5x Lugol's solution</SPAN></H5><UL><LI>100 g K Iodide</LI><LI>50 g Iodine</LI><LI>Ad to 1,000 mL with deionized H<SUB>2</SUB>O</LI></UL><P>The Lugol's solution was diluted to 1–fold and 5 mL of the solution was added on the plates. The insertion in the amylase encoding gene <EM>amyE</EM> was checked with a α-amylase assay. The <EM>amyE</EM> positive bacteria form a halo while those with a disrupted <EM>amyE</EM> gene should not form a halo.</P><P>The following figures show the the results from the α-amylase assays.</P><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><DIV class="article_multpicture-paragraph"><P>pAC7_<EM>PtrpE</EM>→168. Clones 1&amp;3 are correct.</P><P>pAC7_<EM>PtrpE</EM>→SP1. Clone 2 is correct.</P></DIV></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Ligation_of_PtrpP_into_pAC7">Ligation of <EM>PtrpP</EM> into pAC7</SPAN></H3><P class="notebook-head_date">11.06.18</P></DIV><DIV class="notebook-content"><P>A restriction digest was carried out to digest <EM>PtrpP</EM>.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [μL]</TH></TR><TR><TD>DNA</TD><TD class="article-table_right">20</TD></TR><TR><TD><EM>BamH</EM>I</TD><TD class="article-table_right">2</TD></TR><TR><TD><EM>EcoR</EM>I</TD><TD class="article-table_right">2</TD></TR><TR><TD>FD buffer</TD><TD class="article-table_right">3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">3</TD></TR><TR class="article_table_hline"><TD><STRONG>30 min at 37°C</STRONG></TD></TR></TBODY></TABLE></DIV><P>The sample was purified with the DNA purification kit and eluted in 25 μL. Also, the DNA concentration was measured with the Nanodrop device: 22.4 ng/µL. A sample for the ligation and one for the Re-ligation was prepared.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Compound</TH><TH>Volume in [μL]</TH></TR><TR><TD>Vector (pAC7)</TD><TD class="article-table_right">2.5</TD></TR><TR><TD><EM>Insert (<EM>PtrpP</EM></EM></TD><TD class="article-table_right">1</TD></TR><TR><TD>T4 buffer</TD><TD class="article-table_right">2</TD></TR><TR><TD>T4 ligase</TD><TD class="article-table_right">1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD class="article-table_right">13.5</TD></TR><TR class="article_table_hline"><TD><STRONG>at least 2 h at RT</STRONG></TD></TR></TBODY></TABLE></DIV><P>Also, a Re-ligation mixture was prepared without an insert. With the ligation and re-ligation mixtures were competent <EM>E. coli</EM> cells of strain DH5α transformed as described previously. The transformed cells were propagated on LB medium containing X-Gal and kanamycin.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Naming_of_plasmids_and_evaluation_of_the_ligation">Naming of plasmids and evaluation of the ligation</SPAN></H3><P class="notebook-head_date">12.06.18</P></DIV><DIV class="notebook-content"><H5><SPAN class="mw-headline" id="Renaming_of_plasmids">Renaming of plasmids</SPAN></H5><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Previous name</TH><TH>New name</TH></TR><TR><TD>pGP172_<EM>aroE_1</EM></TD><TD>pIGEM1</TD></TR><TR><TD>pBQ200_<EM>aroE_2</EM></TD><TD>pIGEM2</TD></TR><TR><TD>pAC7_<EM>PtrpE</EM></TD><TD>pIGEM3</TD></TR></TBODY></TABLE></DIV><P>The plasmids were stored at -20°C.</P><H5><SPAN class="mw-headline" id="Results_of_ligated_PtrpPinto_pAC7">Results of ligated <EM>PtrpP</EM>into pAC7</SPAN></H5><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>Amount in [µL]</TH><TH>Cfu</TH></TR><TR><TD>pAC7_<EM>PtrpP</EM></TD><TD>100</TD><TD>0</TD></TR><TR><TD>pAC7_<EM>PtrpP</EM></TD><TD>rest</TD><TD>4 blue/1 white</TD></TR><TR><TD>Re-ligation pAC7</TD><TD>100</TD><TD>0</TD></TR><TR><TD>Re-ligation pAC7</TD><TD>rest</TD><TD>0</TD></TR><TR><TD>Negative control</TD><TD>100</TD><TD>0</TD></TR><TR><TD>Negative control</TD><TD>rest</TD><TD>0</TD></TR></TBODY></TABLE></DIV><P>Overnight cultures of 168_<EM>PtrpE</EM> and SP1_<EM>PtrpE</EM> were incubated in LB medium at 28°C.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Disc_diffusion_assay">Disc diffusion assay</SPAN></H3><P class="notebook-head_date">13.06.18</P></DIV><DIV class="notebook-content"><P>To analyze the effects of glyphosate on <EM>B. subtilis</EM>, 100 µL of cultures 168_<EM>PtrpE</EM> and SP1_<EM>PtrpE</EM> were plated on CS-Glu medium containing X-Gal. The plates were dried for 5 min.A disc containing 5 µL of 70 mM glyphosate solution was placed in the middle of the plate. The plates were incubated at 37°C over night.
    &lt;/div&gt;&lt;/div&gt;</P><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id=".CE.B2-galactosidase_assay">β-galactosidase assay</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;14.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><H5><SPAN class="mw-headline" id="General_procedure_to_determine_.CE.B2-galactosidase_activity.">General procedure to determine β-galactosidase activity.</SPAN></H5><OL><LI>Propagate a single colony over a whole SP agar plate and incubate the plate over the day at 37°C. You could also propagate a single colony on an SP agar plate in the evening and use the plate at the evening next day to inoculate a pre-culture.</LI><LI>Take a single colony of the <EM>B. subtilis</EM> strain of interest and inoculate 2&amp;nadsh;5 mL C minimal medium supplemented with appropriate antibiotics and effectors (carbon and nitrogen sources). Incubate the pre-cultures over night at 28°C with agitation (220 rpm).</LI><LI>Inoculate 10 mL C minimal medium (plus effectors, carbon and nitrogen sources) supplemented in a 100 mL shake flask to an OD<SUB>600</SUB> of about 0.1 and grow the cultures to an OD<SUB>600</SUB> of 0.5–0.8 (3–5 h) at the desired temperature (usually at 37°C for <EM>B. subtilis</EM>).</LI><LI>Collect the cells from 1.5–2.0 mL of the cultures by centrifugation for 5 min at 13,000 rpm and 4°C. Discard the supernatants and store the cell pellets for further treatment at -20°C.</LI><LI>Re-suspend the cells in 400 mL Z buffer (with β-ME and 20 mL LD-Mix in 4 mL Z buffer). Incubate the suspension for 10 min at 37°C to lyse the cells. Keep a minimum of 20 mL Z buffer/LD-Mix for the Bradford assay.</LI><LI>Remove cell debris from the soluble fraction by centrifugation for 2 min at 13,000 rpm and 4°C. Transfer the supernatants into new Eppendorf reaction tubes.</LI><LI>Add 100 mL of the cell free crude extract to 700 mL Z buffer (with β-ME) and pre-incubate the mixture for 5 min at 28°C. 800 mL Z buffer (with β-ME) will serve as the reference. </LI><LI>Start the enzymatic reaction by the addition of 200 mL ONPG to the reaction tubes and note the time. </LI><LI>As soon as the reaction mixture turns yellow, stop the reaction by adding 500 mL Na<SUB>2</SUB>CO<SUB>3</SUB> and note the time. Now it is time to measure the absorption of the sample at a wavelength of λ = 420 nm. The “reference sample” will serve as the reference!</LI><LI>The specific β-galactosidase activity in Miller Units/mg protein (MU/mg) can be calculated using the formula below:
                <DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;&lt;p&gt;A<SUB>420</SUB>: absorption of o–Nitrophenol; Δt: time difference between start and stop of the enzymatic reaction in [min]; V: volume of crude extract used for the reaction (usually 0.1 mL, see step 3); A<SUB>595</SUB>: absorption at λ = 595 nm obtained by the Bradford assay.&lt;/p&gt;</PRE></DIV><PRE>               The method to detect the β–galactosidase activity can also be applied to detect the enzymatic activity of the β–xylosidase in cell freee crude extract of <EM>B. subtilis</EM> (Lindner <EM>et al</EM>., 1994). To detect β–xylosidase activity, ONPG has to be replaced by PNPX.
</PRE></LI></OL></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparation_of_cell_pellets_for_.CE.B2-galactosidase.E2.80.93assay">Preparation of cell pellets for β-galactosidase–assay</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;15.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;The following table shows the samples used for the lacZ–assay.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH><TH>Volume to inoculate [µL]</TH></TR><TR><TD>SP1 + pIGEM3</TD><TD>4.32</TD><TD>231</TD></TR><TR><TD>168 + pIGEM3</TD><TD>3.48</TD><TD>287</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;Samples were inoculated with and without tryptophane.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>sample</TH><TH>3 h</TH><TH>4 h</TH><TH>5 h</TH></TR><TR><TD>SP1 + pIGEM3 –trp –GS</TD><TD>0.3</TD><TD><STRONG>0.83</STRONG></TD><TD>–</TD></TR><TR><TD>SP1 + pIGEM3 –trp +GS</TD><TD>0.29</TD><TD>0.45</TD><TD>0.5</TD></TR><TR><TD>SP1 + pIGEM3 +trp –GS</TD><TD>0.24</TD><TD>0.40</TD><TD>0.5</TD></TR><TR><TD>SP1 + pIGEM3 +trp +GS</TD><TD>0.26</TD><TD>0.46</TD><TD>0.61</TD></TR><TR><TD>168 + pIGEM3 –trp –GS</TD><TD>0.18</TD><TD>0.36</TD><TD><STRONG>0.38</STRONG></TD></TR><TR><TD>168 + pIGEM3 –trp +GS</TD><TD>0.43</TD><TD><STRONG>0.64</STRONG></TD><TD>–</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;2x 2 mL of each sample was centrifuged at 13,000 rpm at 4°C. Pellets were stored at -20°C.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id=".CE.B2-galactosidase.E2.80.93assay">β-galactosidase–assay</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;18.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;The lacZ–assay was carried out as described previously. There was not change in coloration observeable, therefore the experiment has to be repeated. The Bradford assay was carried out to verify that the cells are lyzed using the LD mix. It came out that all samples contained proteins.&lt;/p&gt;&lt;p&gt;Furthermore, <EM>B. subtilis</EM> strain 168 was transformed with pBQ200 (as an isogenic control) and pIGEM2.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Preparation_for_RNA_sequencing">Preparation for RNA sequencing</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;19.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;Cultures were prepared for RNA sequencing. OD<SUB>600</SUB> measurement of the two ON cultures.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH><TH>Volume to inoculate [µL]</TH></TR><TR><TD>SP1, 1</TD><TD>5.38</TD><TD>185</TD></TR><TR><TD>SP1, 2</TD><TD>4.8</TD><TD>208</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;Culture 2 was used to inoculate 6x CS-Glu –trp media. 150 µL 50 mM glyphosate solution was added to cultures 4–6 to obtain a concentration of 0.75 mM.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>0 h</TH><TH>3 h</TH><TH>7 h</TH></TR><TR><TD>SP1 0 mM GS, 1</TD><TD>0.088</TD><TD>0.18</TD><TD>1.56</TD></TR><TR><TD>SP1 0 mM GS, 1</TD><TD>0.092</TD><TD>0.16</TD><TD>1.76</TD></TR><TR><TD>SP1 0 mM GS, 3</TD><TD>0.09</TD><TD>0.16</TD><TD>1.54</TD></TR><TR><TD>SP1 0.75 mM GS, 1</TD><TD>0.09</TD><TD>0.15</TD><TD>1.12</TD></TR><TR><TD>SP1 0.75 mM GS, 2</TD><TD>0.088</TD><TD>0.13</TD><TD>0.71</TD></TR><TR><TD>SP1 0.75 mM GS, 3</TD><TD>0.087</TD><TD>0.12</TD><TD>0.96</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;1 mL cells were harvested at 4°C with centrifugation for 2 min at 13,000 rpm. The supernatant was descarded, and the pellets were placed into liquid nitrogen. Storage at -20°C.&lt;/p&gt;&lt;p&gt;Furthermore, a PCR was performed to amplify <EM>PtrpP</EM> with primers iGEM2018_23 and iGEM2018_24. The template was chromosomal DNA from the wildtype 168. The results are shown in the figure below. The samples were purified and eluted in 35 µL H<SUB>2</SUB>O and measured with the Nanodrop device.&lt;/p&gt;</PRE><DIV class="article_picture" style="width:40%;"><PRE>&lt;img src=&quot;&quot;&gt;&lt;p&gt;In all four lanes are samples of <EM>PtrpP</EM>.&lt;/p&gt;</PRE></DIV><PRE>&lt;p&gt;To measure the β-galactosidase activity, the strains 168_pIGEM3, SP1_pIGEM3, GP342, and GP650 were propagated on LB agar plates and grown over night at 37°C.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Ligation_of_PtrpP_into_pAC7_2">Ligation of <EM>PtrpP</EM> into pAC7</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;20.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;Strains GP342 (200–300 MU/mg protein) and GP350 (10–20 MU/mg protein) serve as a positive control for the β-galactosidase assay. 4 mL LB medium were inoculated from the plates and incubated at 37°C over daytime. In the evening, the cells were transferred into 3 mL CS-Glu medium and incubated over night at 28°C.&lt;/p&gt;&lt;p&gt;<EM>PtrpP</EM> was digested with the restriction enzymes <EM>BamH</EM>I and <EM>EcoR</EM>I as described before. The samples were purified and eluted in 30 µL H<SUB>2</SUB>O and measured with the Nanodrop device. Furthermore, the fragment was ligated into the vector pAC7 as described before.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id=".CE.B2-galactosidase_assay_2">β-galactosidase assay</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;21.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;OD<SUB>600</SUB> measurement of the over night cultures.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH><TH>Volume to inoculate [µL]</TH></TR><TR><TD>168_pIGEM3</TD><TD>0.83</TD><TD>1204</TD></TR><TR><TD>SP1_pIGEM3</TD><TD>1.56</TD><TD>641</TD></TR><TR><TD>GP342</TD><TD>0.7</TD><TD>1428</TD></TR><TR><TD>GP350</TD><TD>0.81</TD><TD>1234</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;The OD<SUB>600</SUB> was measured.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>0 h</TH><TH>3 h</TH><TH>4 h</TH><TH>5 h</TH></TR><TR><TD>168_pIGEM3</TD><TD>0.088</TD><TD>0.35</TD><TD>0.53</TD><TD>0.62</TD></TR><TR><TD>SP1_pIGEM3</TD><TD>0.09</TD><TD>0.15</TD><TD>0.25</TD><TD>0.47</TD></TR><TR><TD>GP342</TD><TD>0.088</TD><TD>0.23</TD><TD>0.47</TD><TD>0.56</TD></TR><TR><TD>GP350</TD><TD>0.086</TD><TD>0.07</TD><TD>0.01</TD><TD>/</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;Competent <EM>E. coli</EM> DH5α cells were transformed, a negative control was also performed. The transformed cells were propagated on LB agar containing ampicilin and X-Gal.&lt;/p&gt;&lt;p&gt;OD<SUB>600</SUB> measurement for the β-galactosidase assay n=3.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Sample</TH><TH>Start [sec]</TH><TH>End [min]/th&gt;</TH><TH>A<SUB>420</SUB></TH><TH>A<SUB>595</SUB></TH></TR><TR><TD>168_pIGEM3</TD><TD>0</TD><TD>22</TD><TD>0.006</TD><TD>0.198</TD></TR><TR><TD>SP1_pIGEM3</TD><TD>30</TD><TD>22.3</TD><TD>0.037</TD><TD>0.125</TD></TR><TR><TD>GP342</TD><TD>60</TD><TD>7</TD><TD>0.296</TD><TD>0.167</TD></TR></TBODY></TABLE></DIV><PRE>&lt;p&gt;For the strains, the Miller units per mg protein were calculated with the formula given previously.&lt;/p&gt;</PRE><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH>Activity</TH></TR><TR><TD>168_pIGEM3</TD><TD>26.91 units/mg protein</TD></TR><TR><TD>SP1_pIGEM3</TD><TD>2.75 units/mg protein</TD></TR><TR><TD>GP342</TD><TD>590.81 units/mg protein</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Isolation_of_plasmids_.28pAC7_PtrpP.29">Isolation of plasmids (pAC7_<EM>PtrpP</EM>)</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;25.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;The plasmids from 21.06.18 were isolated, eluted in 30 µL H<SUB>2</SUB>O, and the concentration was measured with the Nanodrop device. A sample was sent for sequencing with the primers <EM>lacZ-rev</EM> and <EM>ML107</EM>.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Transformation_of_SP1">Transformation of SP1</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;26.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;<EM>B. subtilis</EM> strain SP1 was made competent as described previously. The cells were transformed with chromosomal DNA of GP650. The transformed cells were propagated on SP agar containing the appropriate antibiotics.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Isolation_of_chromosomal_DNA">Isolation of chromosomal DNA</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;27.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;Transformations from 26.06.18 were separated on SP agar containing the appropriate antibiotics. Sequencing of pAC7_<EM>PtrpP</EM> was succesful and the plasmid was named &quot;pIGEM5&quot;.&lt;/p&gt;</PRE><H5><SPAN class="mw-headline" id="Isolation_of_chromosomal_DNA_from_Bacillus.C2.A0subtilis">Isolation of chromosomal DNA from <EM>Bacillus subtilis</EM></SPAN></H5><PRE>&lt;p&gt;The protocol requires the &quot;Bacterial DNA kit&quot; from &quot;peqGOLD&quot;.&lt;/p&gt;</PRE><OL><LI>Inoculate 4 mL LB medium with the cells of interest and incubate them over night at 37°C. Harvest the cells with centrifugation.</LI><LI>Resuspend the pellet in 100 µL TE buffer and add 100 µL lysozyme. Incubate the reaction for 1 h at 30°C and 800 rpm.</LI><LI>Centrifuge the samples for 5 min at 6,500 rpm, discard the supernatant. Resuspend the pellet in 400 µL DNA lysis buffer T.</LI><LI>Add 20 µL proteinkinase K and 15 µL RNase A. Shake the samples vigorously and incubate them for 30 min at 70°C.</LI><LI>Add 200 µL DNA binding, mix the sample and load it on the column tube. Centrifugation at 11,000 rpm. Discard the flowthrough and place the column in a new collection tube.</LI><LI>Add 650 µL DNA wash buffer on the column and centrifuge for 1 min at 11,000 rpm. Discard the flowthrough.</LI><LI>Dry the column for 2  centrifugation at 11,000 rpm.</LI><LI>Place the column in a sterile Eppendorf cup, add 50 µL sterile H<SUB>2</SUB>O, incubate the column for 1 min and centrifuge at 7,900 rpm. Repeat this step.</LI><LI>To avoid spores in the DNA, centrifuge the samples 10–30 min at 13,000 rpm at 4°C.</LI><LI>Store the supernatant at -20°C.</LI></OL><PRE>&lt;p&gt;Strain SP1 was transformed with pIGEM5 and chromosomal DNA from strain GP342.&lt;/p&gt;</PRE></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H3><PRE>&lt;p class=&quot;notebook-head_date&quot;&gt;28.06.18&lt;/p&gt;</PRE></DIV><DIV class="notebook-content"><PRE>&lt;p&gt;Transformations from SP1 + DNA from GP650 worked well, so two clones were picked and separated. &lt;/p&gt;</PRE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>