<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Goettingen_Notebook_evolution_May skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Goettingen/Notebook/evolution May</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="notebook-item"><DIV class="notebook-head"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#"><SPAN class="tocnumber">1</SPAN></A></LI><LI class="toclevel-1"><A href="#_2"><SPAN class="tocnumber">2</SPAN></A></LI><LI class="toclevel-1"><A href="#_3"><SPAN class="tocnumber">3</SPAN></A></LI><LI class="toclevel-1"><A href="#Growth_experiments"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Growth experiments</SPAN></A></LI><LI class="toclevel-1"><A href="#_4"><SPAN class="tocnumber">5</SPAN></A></LI><LI class="toclevel-1"><A href="#_5"><SPAN class="tocnumber">6</SPAN></A></LI><LI class="toclevel-1"><A href="#_6"><SPAN class="tocnumber">7</SPAN></A></LI><LI class="toclevel-1"><A href="#Identification_of_novel_mutations_in_B..C2.A0subtilis_168"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Identification of novel mutations in B. subtilis 168</SPAN></A></LI><LI class="toclevel-1"><A href="#Amplification_of_the_gltT_gene"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Amplification of the gltT gene</SPAN></A></LI><LI class="toclevel-2"><A href="#Results"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#New_Primers_for_gltT_and_sequencing"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">New Primers for gltT and sequencing</SPAN></A></LI><LI class="toclevel-2"><A href="#Results_2"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#_7"><SPAN class="tocnumber">11</SPAN></A></LI><LI class="toclevel-1"><A href="#Agar_plates_from_the_25.05.18"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Agar plates from the 25.05.18</SPAN></A></LI><LI class="toclevel-1"><A href="#_8"><SPAN class="tocnumber">13</SPAN></A></LI></DIV><P class="notebook-head_date">02.05.18</P></DIV><DIV class="notebook-content"><P>The strains BP233, BP235 and the WT strain 168 were propagated to 3 ml LB medium and inoculated for 4 h at 37°C and 220 rpm. They were harvested in a 2 ml reaction tube by centrifugation at 13000 rpm for one minute. The supernatant was discared and the cells were washed in 1x C-Salts. Afterwards, they were resuspended in 300 µl 1x C-Salts. The OD<SUB>600</SUB> was determined.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH class="article-table_right">OD<SUB>600</SUB></TH></TR><TR><TD>168</TD><TD class="article-table_right">9.8</TD></TR><TR><TD>BP233</TD><TD class="article-table_right">6.1</TD></TR><TR><TD>BP235</TD><TD class="article-table_right">7.6</TD></TR></TBODY></TABLE></DIV><P>The cells were then struck out on cs-glucose agar plates with variating glyphosate concentrations. 168 was propagated on plates containing 0 mM  and 10 mM glyphosate, while BP233 was additionally propagated on plates with the glyphosate concentrations of 30 mM and 40 mM.BP235 was additonally propagated on plates containing glyphosate concentrations of 50 mM and 60 mM. All agar plates were then inoculated at 37 °C for 2 days.   </P><P>The strains 168, BP233 and BP236 were propagated to 3 ml LB medium and inoculated for 4 h at 37°C and 220 rpm. They were harvested in a 2 ml reaction tube by centrifugation at 13000 rpm for one minute. The supernatant was discared and the cells were washed in 1x C-Salts. Afterwards, they were resuspended in 300 µl 1x C-Salts. The OD<SUB>600</SUB> was determined.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Strain</TH><TH class="article-table_right">OD<SUB>600</SUB></TH></TR><TR><TD>168</TD><TD class="article-table_right">10.74</TD></TR><TR><TD>BP233</TD><TD class="article-table_right">8.37</TD></TR><TR><TD>BP236</TD><TD class="article-table_right">7.71</TD></TR></TBODY></TABLE></DIV><P>100 µl of solutions were propagated on CS-Glucose agar plates containing 0 mM glyphosate as well as plates containing 10 mM glyphosate. These plates were inoculated at 37 °C for 2 days.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">03.05.18</P></DIV><DIV class="notebook-content"><P>Strains BP233, BP235 and 168 (WT) were incubated in 4 ml LB medium for 4 h at 37°C and 220 rpm. 2 ml of the culture was harvested, and the cells were washed in 1xC-salts and then resuspended in 300 µl 1xC-salts. BP233 and BP235 were plated on CS-glucose with and without 10 mM glyphosate.</P><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Growth of the strains 168 and BP233 in absence of glyphosate and in presence of 10 mM glyphosate.</P></DIV><P>As seen in this figure, both strains grow well on CS-glucose without glyphosate. Strain BP233 also forms a lawn on plates with 10mM glyphosate, while the wildtype does not form a lawn, but single colonies, which could be suppressor mutants. This leads to the hypothesis that GltT, which is a glutamate transporter, transports glyphosate into the cell. The strain BP235 was not stable, so it was constructed again!
        </P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">04.05.18</P></DIV><DIV class="notebook-content"><P>The growth of the suppressor mutants of the strains 168, BP233 and BP235 on 10 mM and 0 mM glyphosate CS-glucose agar plates was documented.</P><DIV class="article_picture">&lt;img src=&quot;&quot;&gt;</DIV><P>168 colonies were picked from the 10 mM glyphosate plate and steaked out on CS-glucose 10 mM glyphosate plates.</P><P>A growth test of 168, BP233 and BP237 on CS-Glucose agar plates containing 0 mM and 10 mM glyphosate.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Growth_experiments">Growth experiments</SPAN></H3><P class="notebook-head_date">07.05.18</P></DIV><DIV class="notebook-content"><P><EM>B. subtilis</EM> 168 and <EM>B. subtilis</EM> SP1 were transferred into CS-Glucose medium containing increasing glyphosate concentrations to an OD<SUB>600</SUB> of 0.1. OD measurement was carried out every 2 hours until the cells reached an OD<SUB>600</SUB> of ∼2. The cells were harvested by centrifugation for 1 min at 13000 rpm and stored at -20°C. </P><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><P>Growth curves of the wild type SP1 and the 168 mutant, auxotrophic for tryptophan.</P></DIV><DIV class="notebook-content"><P>The mutants of 168 grown on CS-glucose 10 mM glyphosate plates were inoculated at 37°C in liquid LB medium and used for cryo cultures (1-17).</P></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">08.05.18</P></DIV><DIV class="notebook-content"><P>BP233 colonies grown on the 40 mM glyphosate medium were inoculated on 40 mM glyphosate CS-glucose medium.</P><P>BP235 colonies from 50 mM glyphosate medium were inoculated on 40 mM CS-glucose medium</P><P>Cryo cultures of mutant strains were made named igem18-34 and cultures 6-8 were inoculated again.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">14.05.18</P></DIV><DIV class="notebook-content"><P>Over night cultures were prepared for cryo cultures.</P><P>Preparing of over night cultures for cDNA extraction.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">15.05.18</P></DIV><DIV class="notebook-content"><P>Cryo cultures of mutant strains were made named igem35-44.</P><P>The cells of the over night culture were harvested and the cDNA was isolated according to the &lt;a href=&quot;<A rel="nofollow" class="external free" href="https://2018.igem.org/Team:Goettingen/Experiments">https://2018.igem.org/Team:Goettingen/Experiments</A>&quot; target=&quot;_blank&quot; &gt;script&lt;/a&gt;</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Identification_of_novel_mutations_in_B..C2.A0subtilis_168">Identification of novel mutations in <EM>B. subtilis</EM> 168</SPAN></H3><P class="notebook-head_date">16.05.18</P></DIV><DIV class="notebook-content"><P>For the identification of novel mutations in the gltT transporter, the wild type strain was grown on 40 mM glyphosate. Emerging mutations were further analysed by isolation of the complete DNA and amplification of the transporter via PCR. The genomic DNA was isolated with the Kit “peqGOLD Bacterial DNA Kit” from peqlab. The resulting sequences were then screened for novel mutations.</P><P>For the identification of novel mutations in the gltT transporter, the wild type strain was grown on 40 mM glyphosate. Emerging mutations were further analysed by isolation of the chromosomal DNA and amplification of the transporter via PCR. The genomic DNA was isolated with the Kit “peqGOLD Bacterial DNA Kit” from peqlab.</P><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Mutant names</TH><TH>DNA concentration in [ng/µL]</TH></TR><TR><TD>iGEM18</TD><TD class="article-table_right">15.2</TD></TR><TR><TD>iGEM19</TD><TD class="article-table_right">44.5</TD></TR><TR><TD>iGEM20</TD><TD class="article-table_right">57.4</TD></TR><TR><TD>iGEM21</TD><TD class="article-table_right">48.3</TD></TR><TR><TD>iGEM22</TD><TD class="article-table_right">45.3</TD></TR><TR><TD>iGEM23</TD><TD class="article-table_right">54.3</TD></TR><TR><TD>iGEM24</TD><TD class="article-table_right">44.7</TD></TR><TR><TD>iGEM25</TD><TD class="article-table_right">31.9</TD></TR><TR><TD>iGEM26</TD><TD class="article-table_right">29.7</TD></TR><TR><TD>iGEM27</TD><TD class="article-table_right">80.7</TD></TR><TR><TD>iGEM28</TD><TD class="article-table_right">58.8</TD></TR><TR><TD>iGEM29</TD><TD class="article-table_right">51.9</TD></TR><TR><TD>iGEM30</TD><TD class="article-table_right">57.7</TD></TR><TR><TD>iGEM31</TD><TD class="article-table_right">/</TD></TR><TR><TD>iGEM32</TD><TD class="article-table_right">56.1</TD></TR><TR><TD>iGEM33</TD><TD class="article-table_right">62.2</TD></TR><TR><TD>iGEM34</TD><TD class="article-table_right">42.5</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Amplification_of_the_gltT_gene">Amplification of the <EM>gltT</EM> gene</SPAN></H3><P class="notebook-head_date">17.05.18</P></DIV><DIV class="notebook-content"><P>We amplified the <EM>gltT</EM> gene from <EM>B. subtilis</EM> with a PCR run with the following recipe.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH>Component</TH><TH>Volume in [µL]</TH></TR><TR><TD>10x <EM>Taq</EM> buffer</TD><TD class="article-table_right">5</TD></TR><TR><TD>dNTPs (12.5 mM)</TD><TD class="article-table_right">2</TD></TR><TR><TD>fwd Primer: iGEM2018_15</TD><TD class="article-table_right">2</TD></TR><TR><TD>rev Primer: iGEM2018_16</TD><TD class="article-table_right">2</TD></TR><TR><TD><EM>B. subtilis</EM> chromosomal DNA</TD><TD class="article-table_right">1</TD></TR><TR><TD>Dream <EM>Taq</EM> polymerase</TD><TD class="article-table_right">0.25</TD></TR><TR><TD>sterile H<SUB>2</SUB>O</TD><TD class="article-table_right">37.75</TD></TR><TR class="article_table_hline"><TD><STRONG>Total</STRONG></TD><TD class="article-table_right">50</TD></TR></TBODY></TABLE></DIV><H6><SPAN class="mw-headline" id="Results">Results</SPAN></H6><P>In the following figures are the results of the PCR shown.</P><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><DIV class="article_multpicture-paragraph"><P>Amplified <EM>gltT</EM> gene from 168, iGEM19/20.</P><P>Amplified <EM>gltT</EM> gene from iGEM30/32/33/34.</P></DIV></DIV><DIV class="article_picture"><PRE>&lt;img src=&quot;&quot;&gt;</PRE><P>Amplified <EM>gltT</EM> gene from 168, iGEM18/21/22/23/24/25/26/27/28/29.</P></DIV><P>&amp;rightarrow; iGEM21 and iGEM26 did not show any PCR products. Therefore, no sequence could be further analyzed.</P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="New_Primers_for_gltT_and_sequencing">New Primers for <EM>gltT</EM> and sequencing</SPAN></H3><P class="notebook-head_date">23.05.18</P></DIV><DIV class="notebook-content"><P>The PCR from 17.05.18 was repeated with new primers (iGEM2018_19 and iGEM2018_20) that amplify also the ends of the gene <EM>gltT</EM>. The protocol was the same as before.</P><H6><SPAN class="mw-headline" id="Results_2">Results</SPAN></H6><DIV class="article_multpicture"><DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV><DIV><PRE>&lt;img src=&quot;&quot;&gt;</PRE></DIV></DIV><DIV class="article_multpicture-paragraph"><P>Amplified <EM>gltT</EM> gene from iGEM18–iGEM29.</P><P>Amplified <EM>gltT</EM> gene from iGEM30–iGEM34.</P></DIV></DIV><P>The amplified <EM>gltT</EM> fragments were purified and sent to sequencing with primers iGEM2018_29 and iGEM2018_30. The following table provides an overview of mutations in <EM>gltT</EM> from glyphosate adapted mutants.</P><DIV class="notebook-table article_table"><TABLE><TBODY><TR><TH><EM>gltT</EM> sequence</TH><TH>Mutation</TH><TH>Consequence</TH></TR><TR><TD>Wild type</TD><TD>—</TD><TD>—</TD></TR><TR><TD>iGEM18</TD><TD>Δ669A</TD><TD>Truncation</TD></TR><TR><TD>iGEM19</TD><TD>Δ669A</TD><TD>Truncation</TD></TR><TR><TD>iGEM20</TD><TD>C211T</TD><TD>Truncation</TD></TR><TR><TD>iGEM22</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR><TR><TD>iGEM23</TD><TD>Δ65T</TD><TD>Truncation</TD></TR><TR><TD>iGEM24</TD><TD>deletion 898–923</TD><TD>Truncation</TD></TR><TR><TD>iGEM25</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR><TR><TD>iGEM27</TD><TD>A388T</TD><TD>Truncation</TD></TR><TR><TD>iGEM29</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR><TR><TD>iGEM30</TD><TD>Δ996&amp;nash;1090</TD><TD>Truncation</TD></TR><TR><TD>iGEM32</TD><TD>Δ669A</TD><TD>Truncation</TD></TR><TR><TD>iGEM33</TD><TD>duplication 839–847</TD><TD>Insertion</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">25.05.18</P></DIV><DIV class="notebook-content"><P>The wildtype strain 168 as well as the strains BP233 and BP236 were inoculated in 3 ml LB medium for 2-4h. 1.5 ml of each culture were harvested via centrifugation for 1 min at 13000 rpm. The supernatent was discarded and the cells were resuspended in 1 ml 1x C-salts. The were centrifuged again and the supernatent was discarded. The cells were resuspended in 400 µl 1x C-salts. 100 µl of each strain were plated on CS-glucose agar plates containing no glyphosate and plates containing 10 mM glyphosate.  </P></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><H3 class="notebook-head_title"><SPAN class="mw-headline" id="Agar_plates_from_the_25.05.18">Agar plates from the 25.05.18</SPAN></H3><P class="notebook-head_date">28.05.18</P></DIV><DIV class="notebook-content"><DIV class="article_picture">&lt;img src=&quot;&quot;&gt;</DIV></DIV></DIV><DIV class="notebook-item"><DIV class="notebook-head"><P class="notebook-head_date">31.05.18</P></DIV><DIV class="notebook-content"><P>Preparation of CS-glucose (without trp) and CS-glucose containing 50 mM glyphosate. </P><P>SP1 was inoculated in LB medium over night. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>