Team:NPU-China/protocols
Toggle navigation
NPU-China
Home
Team
About us
Attribution
Project
Description
Experiments
Model
Proof of concept
Demonstrate
Hardware
HP
Silver
Gold
Collaboration
Achievements
Interlab
Notebook
Labnotes
Protocols
DNA gel extraction
Electrophoresis
Gibson assenbly
HPLC
Knock out genes of Ecoli
Crispr-cas9
Plasmid preparation
Plasmid transformation
Point mutation
Reagents
References
the LiAc SS carrier DNA PEG method
Whole cell catalysis
DNA gel extraction
1. Excise the agarose gel slice, transfer the gel slice into a 1.5ml microfuge tube.
2. Add a 3X sample             volume of Buffer DE-A. the weight of gel is equivalent to a 100ul volume.
3. Resuspend the gel in Buffer             DE-A by vortexing. Heat at 75℃ until the gel is completely dissolved(no more than 10 min)
4. Add 0.5X Buffer             DE-A volume of Buffer DE-B, mix.
5. Place a miniprep colume into a 2ml microfuge tube. Transfer the solubilized             agarose from step 4 into the column. Centrifuge at 12,000xg for 1 min.
6. Discard the filtrate. Add 500ul             of Buffer W1. Centrifuge at 12,000xg for 30s.
7. Discard the filtrate. Add 700ul of Buffer W2. Centrifuge             at 12,000xg for 30s.
8. Repeat wash with W2. Centrifuge at 12,000xg for 1min.
9. Discard the filtrate.             Centrifuge at 12,000xg for 1min.
10. Transfer the miniprep column into a clean 1.5ml microfuge tube. Add             25-30ul of Eluent or deionized water to the center of the membrane. Let it stand for 1min at room temperature.             Centrifuge at 12,000xg for 1min. (pre-warming the Eluent at 65℃)
Electrophoresis
Agarose-Electrophoresis is used in order to see if the PCR product is correct and seperate DNA by the number of             base pairs.
●marker was the 2 kb Plus DNA Ladder
●fill pockets with 5 µl DNA ladder or 10 µl sample volume with             6x loading buffer
●running conditions: 130 V for 30-40 minutes
Gibson assembly
1. Set up the reaction.
V(ul)=(0.02*bp)/concentration(ng/ul)
2. Add the fragments into Gibson system.
3. Incubate samples in a thermocycler at 50 °C for 1h.
4. Purify the product using DNA purification kit.
5. Transform the product into the competent cells             of E.coli BL21 , following the transformation protocol.
Colony pcr system：20µl
Component
sterile water
Concentration
4µl
Forward primer
1µl
Reversed primer
1µl
Template
4µl
Mix
10µl
 
Method:
 
1
STEP
Initial
TEM
98℃
TIME
5min
CYCLES
1
2
denaturation
 
98℃
20s
 
3
Annealing
58℃
20s
 
4
extension
72℃
1min
 
5
circulation
 
Repeat the process2-4
 
34
6
extension
72℃
5min
1
7
Hold
4℃
Hold
 
 
 
HPLC
for acrylic acid
Retention time
Lower detection limit
HPLC
Waters
Column
HSS
Detector
Column Temperature
Solvent
1 mM H2SO4
Flow Rate
0.6 ml/min
Duration
30 min
Injection Volume
10 µl
Samples have to be centrifuged at 12,000xg for 5 min in order to remove solids (cells/precipitates).
Knock out genes of E ▪ coli (MG1655)
1. Pre-chill 1.5ml and 2ml microcentrifuge tubes, deionized water, 10% glycerol. Add 1ml LB to 2ml microcentrifuge             tubes.
2. When OD600=0.6, incubate competent cells on ice for 20 min.
3. Transfer the competent cells to             50 mL pre-chilled centrifuge tube. Centrifuge at 5,500r/min, 4 ℃ for 5 min.
4. Discard the filtrate. Add             30ml of pre-chilled deionized water, resuspend the cells gently.
5. Centrifuge at 5,500r/min, 4 ℃ for 5             min. Discard the filtrate. Repeat wash with deionized water.
6. Discard the filtrate. Add 30ml of pre-chilled             10% glycerol, resuspend the cells gently.
7. Centrifuge at 6,500r/min, 4 ℃ for 5 min. Discard the filtrate.             Repeat wash with 10% glycerol.
8. Discard the filtrate and leave over 1ml 10% glycerol to resuspend the             competent cells, pipet 80ul into each 1.5ml EP tube, add 5ul of DNA and carefully mix with the competent cells.             Let it stand for 2min.
9. Add electrocompetent to DNA on ice. Move the mixture to the cuvette. Dry and             shock the cells(2500V). Add 1ml of LB medium. Incubate at 30℃ for 4 hours shaking at 220rpm, pipet 100ul from             each tube onto the appropriate plate, and spread the mixture evenly across the plate. Incubate at 30℃ overnight.             Position the plates with the agar side at the top, and the lid at the bottom.
Knock out the genes of Saccharomyces cerevisiae with Crispr-Cas9
1. pcr system：200µl
Component
sterile water
Concentration
129µl
5×Fast pfu Buffer
40µl
dNTPs
16µl
Forward primer
5µl
Reversed primer
5µl
pCAS_Phe-URA3
2µl
Fastpfu
3µl
 
Method:
 
1
STEP
Initial
TEM
95℃
TIME
2min
CYCLES
1
2
denaturation
 
95℃
20s
 
3
Annealing
58℃
30s
 
4
extension
72℃
1min40s
 
5
circulation
 
Repeat the process2-4
 
35
6
extension
72℃
5min
1
7
Hold
4℃
Hold
 
2. Purify the PCR product with a DNA purification kit.
3. Add the appropriate amount of DMT enzyme,             hold for one hour at 37 ° C.
4. transform the DNA into competent cells.
50ul competent cell + 15ul             purified DNA，incubate on ice for 30min，heat shock 45s，incubate on ice for 2min，add LB medium and incubate for             1h.
5. Pipet 100ul from each tube onto the plate with resistance, and spread the mixture evenly across             the plate. Incubate for 12h. Position the plates with the agar side at the top, and the lid at the bottom.
6. use a sterile pipet tip to pick Saccharomyces cerevisiae from plates，throw the tip into the tubes of 5             ml of LB + antibiotics，incubate in a rotary shaker. Prepare plasmid with kit for sequencing.
7. Transfer             plasmid and fragment into Saccharomyces cerevisiae using the LiAc SS carrier DNA PEG method.
plasmid
 
CRISPR-GPD1_5_0
fragment(with 1000bp from forward and reverse)
GPD1
 
 
CRISPR-GPD1_16_0
GPD1
 
 
 
 
CRISPR-DLD1_11_0
DLD1
 
CRISPR-DLD1_18_0
DLD1
 
CRISPR-PDC1_6_0
CRISPR-PDC1_45_0
CRISPR-PDC5_13_0
CRISPR-PDC5_29_0
PDC1
PDC1
PDC5
PDC5
8. Prepare the template: use a sterile toothpick to pick Saccharomyces cerevisiae from plates，put the toothpick             into 100ul 20mMNaOH and mix，99° C boiling for 30min. Colony pcr system：20µl
Component
sterile water
Concentration
4µl
Forward primer
1µl
Reversed primer
1µl
Template
4µl
Mix
10µl
 
Method:
 
1
STEP
Initial
TEM
98℃
TIME
5min
CYCLES
1
2
denaturation
 
98℃
20s
 
3
Annealing
58℃
20s
 
4
extension
72℃
1min
 
5
circulation
 
Repeat the process2-4
 
34
6
extension
72℃
5min
1
7
Hold
4℃
Hold
 
 
plasmid preparation
Tiangen mini plasmid kit
Plasmid transformation
1. Pipette 50µl of competent cells and 2µl of plasmid into 1.5ml tube
2. Heat shock tubes at 42°C for 30s
3. Incubate on ice for 2min
4. Pipette 250µl LB media to each transformation
5. Incubate at 37°C             for 1h
6. Plating
7. Pick single colonies
Reference: http://parts.igem.org/Help:Protocols/Transformation
Point mutation
1. pcr system： 50µl
Component
sterile water
Concentration
35µl
5*TransStart FastPfu buffer
10µl
10mM dNTPs
1µl
10μM Forward primer
1µl
10μM Reversed primer
1µl
template
1µl
TransStart FastPfu DNA polymerase
1µl
 
Method:
 
1
STEP
Initial
TEM
94℃
TIME
5min
CYCLES
1
2
denaturation
 
94℃
30s
 
3
Annealing
58℃
30s
 
4
extension
72℃
3min40s
 
5
circulation
 
Repeat the process2-4
 
30
6
extension
72℃
10min
 1
7
Hold
4℃
Hold
 
2. Purify the PCR product with a DNA purification kit.
3. Add the appropriate amount of DMT enzyme,             hold for one hour at 37 ° C.
4. Transform 5μl digested DNA into competent cells DH5α, incubate on ice for             30min.
42° C heat shock, 45s. Incubate on ice for 2min.
add 200μl of LB. incubate at 37 °C for 1 h, 220rpm/min.
5. pipet 200μl from each tube onto the plate with appropriate resistance, and spread the mixture evenly             across the plate. Incubate at 37℃ overnight. Position the plates with the agar side at the top, and the lid at             the bottom.
6. Select single colonies for sequencing.
Reagents
1. LB medium（lysogeny broth）
The recipe for 1l LB media is as follows:
Tryptone 10g/L
Yeast             extract 5g/L
NaCl 10g/L
2. 0.1mM Kanamycin
MW of Kanamycin：582.58
Store at -20℃
3. LB plate
The recipe for 1l LB plate is as follows:
Tryptone 10g/L
Yeast extract 5g/L
NaCl 10g/L
15g Agar
Add appropriate amount of resistance.
4. 2YT medium
The recipe             for 1l LB plate is as follows:
Tryptone 16g/L
Yeast extract 10g/L
NaCl 5g/L
5. 0.5mM             IPTG
MW of IPTG：238.30
Store at -20℃
6. 50mM PBS buffer,PH8.0
A：0.05mol/L Na2HPO4溶液
B：0.05mol/L KH2P04溶液
137mMNaCl，2.7mMKCl，10mMNa2HPO4,2mMKH2PO4 for 1L.
References
http://parts.igem.org/Help:Protocols/Transformation
https://2015.igem.org/Team:Aachen/Project/Overview
http://www.zymoresearch.com/category/all-products
http://www.corning.com/worldwide/en/products/life-sciences/resources/brands/axygen-brand-products.html
http://www.tsingke.net/shop/
http://www.cwbiotech.com/
the LiAc SS carrier DNA PEG method
1.use a sterile pipet tip to pick Saccharomyces cerevisiae from plates，throw the tip into the tubes of appropriate medium，incubate             in a rotary shaker for 12h.
2.measure OD600, transfer x(x=(50×0.2)/(OD600×dilution ratio))             ml Saccharomyces cerevisiae into 50ml YPAD.
3.incubate for 4-5h to make OD600 reaches 0.8-0.9.
4.boil             ssDNA.
5.Centrifuge at 3000g for 5 min. Discard the filtrate. Repeat washes with 25ml deionized water twice.
6.Transfer the cells to 1.5 mL centrifuge tube. Add 1ml of deionized water, resuspend the cells gently.
7.Centrifuge at 13000rpm for 30s. Discard the filtrate.
8.Add 1ml of deionized water, resuspend the             cells. Pipet 100ul into each 1.5 mL centrifuge tube.
9.Centrifuge using a Mini Centrifuge. Discard the             filtrate.
10.System for transformation:
PEG3350(50%)
240ul
LiAc 1.0M
36 ul
SSDNA（2.0mg/ml）
50 ul
DNA and H2O
34 ul
Total
360ul
11.Incubate at 30℃ for 20min.
12.42℃ heat shock for 40min. pipet 100ul from each tube onto the appropriate             plate, and spread the mixture evenly across the plate. Incubate at 30℃ for 2-3 days. Position the plates with             the agar side at the top, and the lid at the bottom.
13.Prepare plasmid for sequencing.
Whole-cell catalysis
1. Prepare sterile tubes of 5 ml of 2YT+antibiotics. Use a sterile pipet tip to pick bacteria from plates. Throw the tip             into the tubes. Incubate in a rotary shaker at37℃ for 3-4h.
2. Transfer 120µl of bacteria from             a slant culture into an Erlenmeyer flask containing 60 mL LB medium with appropriate resistance, incubate at             37 °C.
3. When the OD600 reaches 0.6-0.8, the induction of IPTG (0.5 mM) should be carried out. Incubate at 30             °C on a rotary shaker incubator at 220 rpm for 14 h.
4. Harvest the bacteria(6000rpm/min 7min). Wash with             30ml PBS.
5. The biocatalytic reaction mixture contained 10% glycerol, E▪ coli and 50mM PBS. Reaction time             gradient: 8h, 16h, 32h.
6. Use HPLC for further analysis.
Reference: Li N, He Y, Chen Y, et al. Production of cyclic adenosine-3′,5′-monophosphate by whole cell             catalysis using recombinant Escherichia coli, overexpressing adenylate cyclase[J]. Korean Journal of Chemical             Engineering, 2013, 30(4):913-917.
Copyright © Your Website 2014
