<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NPU-China_protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NPU-China/protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2017.igem.org/Team:NPU-China">NPU-China</A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2017.igem.org/Team:NPU-China">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="aboutus.html">About us</A></LI><LI><A href="portfolio-2-col.html">Attribution</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="blog-home-1.html">Description</A></LI><LI><A href="blog-home-2.html">Experiments</A></LI><LI><A href="blog-post.html">Model</A></LI><LI><A href="blog-home-1.html">Proof of concept</A></LI><LI><A href="blog-home-2.html">Demonstrate</A></LI><LI><A href="about.html">Hardware</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">HP</A></LI><LI><A href="https://2017.igem.org/Template:NPU-China/silver">Silver</A></LI><LI><A href="sidebar.html">Gold</A></LI><LI><A href="about.html">Collaboration</A></LI><LI><A href="about.html">Achievements</A></LI><LI><A href="about.html">Interlab</A></LI><LI class="dropdown active"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="portfolio-1-col.html">Labnotes</A></LI><LI><A href="https://2017.igem.org/Team:NPU-China/protocols">Protocols</A></LI></DIV></DIV><DIV class="batu" style="background: url('https://static.igem.org/mediawiki/2017/f/fe/Npu-background.png') no-repeat fixed;"><DIV class="container"><DIV class="row"><DIV class="col-md-3" id="myScrollspy" style="font-size:12px;line-height:10px;padding-top: 100px; margin-top: -50px;"><UL class="nav nav-pills nav-stacked" data-spy="affix" style="width:250px; position:fixed;"><LI class="active"><A href="#section-1">DNA gel extraction</A></LI><LI><A href="#section-2">Electrophoresis</A></LI><LI><A href="#section-3">Gibson assenbly</A></LI><LI><A href="#section-4">HPLC</A></LI><LI><A href="#section-5">Knock out genes of Ecoli</A></LI><LI><A href="#section-6">Crispr-cas9</A></LI><LI><A href="#section-7">Plasmid preparation</A></LI><LI><A href="#section-8">Plasmid transformation</A></LI><LI><A href="#section-9">Point mutation</A></LI><LI><A href="#section-10">Reagents</A></LI><LI><A href="#section-11">References</A></LI><LI><A href="#section-12">the LiAc SS carrier DNA PEG method</A></LI><LI><A href="#section-13">Whole cell catalysis</A></LI></UL></DIV><DIV class="col-md-9"><H2 id="section-1" style="padding-top: 100px; margin-top: -50px;">DNA gel extraction</H2><P>
            1. Excise the agarose gel slice, transfer the gel slice into a 1.5ml microfuge tube. 2. Add a 3X sample
            volume of Buffer DE-A. the weight of gel is equivalent to a 100ul volume. 3. Resuspend the gel in Buffer
            DE-A by vortexing. Heat at 75℃ until the gel is completely dissolved(no more than 10 min) 4. Add 0.5X Buffer
            DE-A volume of Buffer DE-B, mix.  5. Place a miniprep colume into a 2ml microfuge tube. Transfer the solubilized
            agarose from step 4 into the column. Centrifuge at 12,000xg for 1 min. 6. Discard the filtrate. Add 500ul
            of Buffer W1. Centrifuge at 12,000xg for 30s. 7. Discard the filtrate. Add 700ul of Buffer W2. Centrifuge
            at 12,000xg for 30s. 8. Repeat wash with W2. Centrifuge at 12,000xg for 1min. 9. Discard the filtrate.
            Centrifuge at 12,000xg for 1min. 10. Transfer the miniprep column into a clean 1.5ml microfuge tube. Add
            25-30ul of Eluent or deionized water to the center of the membrane. Let it stand for 1min at room temperature.
            Centrifuge at 12,000xg for 1min. (pre-warming the Eluent at 65℃)</P><H2 id="section-2" style="padding-top: 100px; margin-top: -50px;">Electrophoresis</H2><P>Agarose-Electrophoresis is used in order to see if the PCR product is correct and seperate DNA by the number of
            base pairs.
             ●marker was the 2 kb Plus DNA Ladder ●fill pockets with 5 µl DNA ladder or 10 µl sample volume with
            6x loading buffer ●running conditions: 130 V for 30-40 minutes</P><H2 id="section-3" style="padding-top: 100px; margin-top: -50px;">Gibson assembly</H2><P>
            1. Set up the reaction. V(ul)=(0.02*bp)/concentration(ng/ul)
             2. Add the fragments into Gibson system. 3. Incubate samples in a thermocycler at 50 °C for 1h.
             4. Purify the product using DNA purification kit. 5. Transform the product into the competent cells
            of E.coli BL21 , following the transformation protocol. Colony pcr system：20µl</P><TABLE border="1" cellspacing="0" cellpadding="0" width="308"><TBODY><TR><TD width="157" valign="top"><P>Component sterile water</P></TD><TD width="151" valign="top"><P>Concentration 4µl
                </P></TD></TR><TR><TD width="157" valign="top"><P>Forward primer</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>Reversed primer</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>Template</P></TD><TD width="151" valign="top"><P>4µl</P></TD></TR><TR><TD width="157" valign="top"><P>Mix</P></TD><TD width="151" valign="top"><P>10µl</P></TD></TR></TBODY></TABLE><P> </P><P>Method:</P><TABLE border="1" cellspacing="0" cellpadding="0" width="484"><TBODY><TR><TD width="25" valign="top"><P> </P><P>1 </P></TD><TD width="113" valign="top"><P>STEP Initial </P></TD><TD width="103" valign="top"><P>TEM 98℃
                </P></TD><TD width="102" valign="top"><P>TIME 5min
                </P></TD><TD width="142" valign="top"><P>CYCLES 1
                </P></TD></TR><TR><TD width="25" valign="top"><P>2 </P></TD><TD width="113" valign="top"><UL><LI>denaturation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>98℃</P></TD><TD width="102" valign="top"><P>20s</P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>3 </P></TD><TD width="113" valign="top"><P>Annealing</P></TD><TD width="103" valign="top"><P>58℃</P></TD><TD width="102" valign="top"><P>20s </P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>4 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>1min</P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>5 </P></TD><TD width="113" valign="top"><UL><LI>circulation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>Repeat the process2-4 </P></TD><TD width="102" valign="top"><P> </P></TD><TD width="142" valign="top"><P>34 </P></TD></TR><TR><TD width="25" valign="top"><P>6 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>5min </P></TD><TD width="142" valign="top"><P>1</P></TD></TR><TR><TD width="25" valign="top"><P>7 </P></TD><TD width="113" valign="top"><P>Hold</P></TD><TD width="103" valign="top"><P>4℃</P></TD><TD width="102" valign="top"><P>Hold</P></TD><TD width="142" valign="top"><P> </P></TD></TR></TBODY></TABLE><P> </P><P> </P><H2 id="section-4" style="padding-top: 100px; margin-top: -50px;">HPLC</H2><P>
            for acrylic acid</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P align="center"><STRONG>Retention time</STRONG></P></TD><TD width="265"><P align="center"><STRONG>Lower detection limit</STRONG></P></TD></TR><TR><TD><P>HPLC</P></TD><TD width="265"><P>Waters</P></TD></TR><TR><TD><P>Column</P></TD><TD width="265"><P>HSS</P></TD></TR><TR><TD><P>Detector</P></TD></TR><TR><TD><P>Column Temperature</P></TD></TR><TR><TD><P>Solvent</P></TD><TD width="265"><P>1 mM H2SO4</P></TD></TR><TR><TD><P>Flow Rate</P></TD><TD width="265"><P>0.6 ml/min</P></TD></TR><TR><TD><P>Duration</P></TD><TD width="265"><P>30 min</P></TD></TR><TR><TD><P>Injection Volume</P></TD><TD width="265"><P>10 µl</P></TD></TR></TBODY></TABLE><P> Samples have to be centrifuged at 12,000xg for 5 min in order to remove solids (cells/precipitates).</P><H2 id="section-5" style="padding-top: 100px; margin-top: -50px;">Knock out genes of E ▪ coli (MG1655)</H2><P>1. Pre-chill 1.5ml and 2ml microcentrifuge tubes, deionized water, 10% glycerol. Add 1ml LB to 2ml microcentrifuge
            tubes.
             2. When OD600=0.6, incubate competent cells on ice for 20 min. 3. Transfer the competent cells to
            50 mL pre-chilled centrifuge tube. Centrifuge at 5,500r/min, 4 ℃ for 5 min. 4. Discard the filtrate. Add
            30ml of pre-chilled deionized water, resuspend the cells gently. 5. Centrifuge at 5,500r/min, 4 ℃ for 5
            min. Discard the filtrate. Repeat wash with deionized water. 6. Discard the filtrate. Add 30ml of pre-chilled
            10% glycerol, resuspend the cells gently.  7. Centrifuge at 6,500r/min, 4 ℃ for 5 min. Discard the filtrate.
            Repeat wash with 10% glycerol. 8. Discard the filtrate and leave over 1ml 10% glycerol to resuspend the
            competent cells, pipet 80ul into each 1.5ml EP tube, add 5ul of DNA and carefully mix with the competent cells.
            Let it stand for 2min. 9. Add electrocompetent to DNA on ice. Move the mixture to the cuvette. Dry and
            shock the cells(2500V). Add 1ml of LB medium. Incubate at 30℃ for 4 hours shaking at 220rpm, pipet 100ul from
            each tube onto the appropriate plate, and spread the mixture evenly across the plate. Incubate at 30℃ overnight.
            Position the plates with the agar side at the top, and the lid at the bottom.</P><H2 id="section-6" style="padding-top: 100px; margin-top: -50px;">Knock out the genes of Saccharomyces cerevisiae with Crispr-Cas9</H2><P>
            1. pcr system：200µl</P><TABLE border="1" cellspacing="0" cellpadding="0" width="308"><TBODY><TR><TD width="157" valign="top"><P>Component sterile water</P></TD><TD width="151" valign="top"><P>Concentration 129µl
                </P></TD></TR><TR><TD width="157" valign="top"><P>5×Fast pfu Buffer </P></TD><TD width="151" valign="top"><P>40µl</P></TD></TR><TR><TD width="157" valign="top"><P>dNTPs</P></TD><TD width="151" valign="top"><P>16µl</P></TD></TR><TR><TD width="157" valign="top"><P>Forward primer</P></TD><TD width="151" valign="top"><P>5µl</P></TD></TR><TR><TD width="157" valign="top"><P>Reversed primer</P></TD><TD width="151" valign="top"><P>5µl</P></TD></TR><TR><TD width="157" valign="top"><P>pCAS_Phe-URA3</P></TD><TD width="151" valign="top"><P>2µl</P></TD></TR><TR><TD width="157" valign="top"><P>Fastpfu</P></TD><TD width="151" valign="top"><P>3µl</P></TD></TR></TBODY></TABLE><P> </P><P>Method:</P><TABLE border="1" cellspacing="0" cellpadding="0" width="484"><TBODY><TR><TD width="25" valign="top"><P> </P><P>1 </P></TD><TD width="113" valign="top"><P>STEP Initial </P></TD><TD width="103" valign="top"><P>TEM 95℃
               </P></TD><TD width="102" valign="top"><P>TIME 2min
                </P></TD><TD width="142" valign="top"><P>CYCLES 1
                </P></TD></TR><TR><TD width="25" valign="top"><P>2 </P></TD><TD width="113" valign="top"><UL><LI>denaturation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>95℃</P></TD><TD width="102" valign="top"><P>20s</P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>3 </P></TD><TD width="113" valign="top"><P>Annealing</P></TD><TD width="103" valign="top"><P>58℃</P></TD><TD width="102" valign="top"><P>30s </P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>4 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>1min40s </P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>5 </P></TD><TD width="113" valign="top"><UL><LI>circulation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>Repeat the process2-4 </P></TD><TD width="102" valign="top"><P> </P></TD><TD width="142" valign="top"><P>35 </P></TD></TR><TR><TD width="25" valign="top"><P>6 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>5min </P></TD><TD width="142" valign="top"><P>1</P></TD></TR><TR><TD width="25" valign="top"><P>7 </P></TD><TD width="113" valign="top"><P>Hold</P></TD><TD width="103" valign="top"><P>4℃</P></TD><TD width="102" valign="top"><P>Hold</P></TD><TD width="142" valign="top"><P> </P></TD></TR></TBODY></TABLE><P> 2. Purify the PCR product with a DNA purification kit. 3. Add the appropriate amount of DMT enzyme,
            hold for one hour at 37 ° C. 4. transform the DNA into competent cells. 50ul competent cell + 15ul
            purified DNA，incubate on ice for 30min，heat shock 45s，incubate on ice for 2min，add LB medium and incubate for
            1h. 5. Pipet 100ul from each tube onto the plate with resistance, and spread the mixture evenly across
            the plate. Incubate for 12h. Position the plates with the agar side at the top, and the lid at the bottom. 6. use a sterile pipet tip to pick Saccharomyces cerevisiae from plates，throw the tip into the tubes of 5
            ml of LB + antibiotics，incubate in a rotary shaker. Prepare plasmid with kit for sequencing. 7. Transfer
            plasmid and fragment into Saccharomyces cerevisiae using the LiAc SS carrier DNA PEG method.</P><TABLE border="1" cellspacing="0" cellpadding="0" width="313"><TBODY><TR><TD width="162" colspan="2" valign="top"><P>plasmid</P><P> </P><P>CRISPR-GPD1_5_0</P></TD><TD width="150" valign="top"><P>fragment(with 1000bp from forward and reverse) GPD1 </P></TD><TD width="1"><P> </P></TD></TR><TR><TD width="5"><P> </P></TD><TD width="157" valign="top"><P>CRISPR-GPD1_16_0</P></TD><TD width="151" colspan="2" valign="top"><P>GPD1 </P></TD></TR><TR><TD width="5"><P> </P></TD><TD width="157" valign="top"><P> </P></TD><TD width="151" colspan="2" valign="top"><P> </P></TD></TR><TR><TD width="5"><P> </P></TD><TD width="157" valign="top"><P>CRISPR-DLD1_11_0</P></TD><TD width="151" colspan="2" valign="top"><P>DLD1 </P></TD></TR><TR><TD width="5"><P> </P></TD><TD width="157" valign="top"><P>CRISPR-DLD1_18_0</P></TD><TD width="151" colspan="2" valign="top"><P>DLD1 </P></TD></TR><TR><TD width="5"><P> </P></TD><TD width="157" valign="top"><P>CRISPR-PDC1_6_0 CRISPR-PDC1_45_0
                   CRISPR-PDC5_13_0
                   CRISPR-PDC5_29_0
                </P></TD><TD width="151" colspan="2" valign="top"><P>PDC1 PDC1
                   PDC5
                   PDC5
                </P></TD></TR></TBODY></TABLE><P> 8. Prepare the template: use a sterile toothpick to pick Saccharomyces cerevisiae from plates，put the toothpick
            into 100ul 20mMNaOH and mix，99° C boiling for 30min. Colony pcr system：20µl</P><TABLE border="1" cellspacing="0" cellpadding="0" width="308"><TBODY><TR><TD width="157" valign="top"><P>Component sterile water</P></TD><TD width="151" valign="top"><P>Concentration 4µl
                </P></TD></TR><TR><TD width="157" valign="top"><P>Forward primer</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>Reversed primer</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>Template</P></TD><TD width="151" valign="top"><P>4µl</P></TD></TR><TR><TD width="157" valign="top"><P>Mix</P></TD><TD width="151" valign="top"><P>10µl</P></TD></TR></TBODY></TABLE><P> </P><P>Method:</P><TABLE border="1" cellspacing="0" cellpadding="0" width="484"><TBODY><TR><TD width="25" valign="top"><P> </P><P>1 </P></TD><TD width="113" valign="top"><P>STEP Initial </P></TD><TD width="103" valign="top"><P>TEM 98℃
                </P></TD><TD width="102" valign="top"><P>TIME 5min
                </P></TD><TD width="142" valign="top"><P>CYCLES 1
                </P></TD></TR><TR><TD width="25" valign="top"><P>2 </P></TD><TD width="113" valign="top"><UL><LI>denaturation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>98℃</P></TD><TD width="102" valign="top"><P>20s</P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>3 </P></TD><TD width="113" valign="top"><P>Annealing</P></TD><TD width="103" valign="top"><P>58℃</P></TD><TD width="102" valign="top"><P>20s </P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>4 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>1min</P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>5 </P></TD><TD width="113" valign="top"><UL><LI>circulation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>Repeat the process2-4 </P></TD><TD width="102" valign="top"><P> </P></TD><TD width="142" valign="top"><P>34</P></TD></TR><TR><TD width="25" valign="top"><P>6 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>5min </P></TD><TD width="142" valign="top"><P>1</P></TD></TR><TR><TD width="25" valign="top"><P>7 </P></TD><TD width="113" valign="top"><P>Hold</P></TD><TD width="103" valign="top"><P>4℃</P></TD><TD width="102" valign="top"><P>Hold</P></TD><TD width="142" valign="top"><P> </P></TD></TR></TBODY></TABLE><P> </P><H2 id="section-7" style="padding-top: 100px; margin-top: -50px;">plasmid preparation</H2><P>Tiangen mini plasmid kit</P><H2 id="section-8" style="padding-top: 100px; margin-top: -50px;">Plasmid transformation</H2><P>1. Pipette 50µl of competent cells and 2µl of plasmid into 1.5ml tube 2. Heat shock tubes at 42°C for 30s 3. Incubate on ice for 2min 4. Pipette 250µl LB media to each transformation 5. Incubate at 37°C
            for 1h 6. Plating 7. Pick single colonies Reference: http://parts.igem.org/Help:Protocols/Transformation</P><H2 id="section-9" style="padding-top: 100px; margin-top: -50px;">Point mutation</H2><P>
            1. pcr system： 50µl</P><TABLE border="1" cellspacing="0" cellpadding="0" width="308"><TBODY><TR><TD width="157" valign="top"><P>Component sterile water</P></TD><TD width="151" valign="top"><P>Concentration 35µl
                </P></TD></TR><TR><TD width="157" valign="top"><P>5*TransStart FastPfu buffer </P></TD><TD width="151" valign="top"><P>10µl</P></TD></TR><TR><TD width="157" valign="top"><P>10mM dNTPs</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>10μM Forward primer</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>10μM Reversed primer</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>template</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR><TR><TD width="157" valign="top"><P>TransStart FastPfu DNA polymerase</P></TD><TD width="151" valign="top"><P>1µl</P></TD></TR></TBODY></TABLE><P> </P><P>Method:</P><TABLE border="1" cellspacing="0" cellpadding="0" width="484"><TBODY><TR><TD width="25" valign="top"><P> </P><P>1 </P></TD><TD width="113" valign="top"><P>STEP Initial </P></TD><TD width="103" valign="top"><P>TEM 94℃
                </P></TD><TD width="102" valign="top"><P>TIME 5min
                </P></TD><TD width="142" valign="top"><P>CYCLES 1
                </P></TD></TR><TR><TD width="25" valign="top"><P>2 </P></TD><TD width="113" valign="top"><UL><LI>denaturation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>94℃</P></TD><TD width="102" valign="top"><P>30s</P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>3 </P></TD><TD width="113" valign="top"><P>Annealing</P></TD><TD width="103" valign="top"><P>58℃</P></TD><TD width="102" valign="top"><P>30s </P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>4 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>3min40s </P></TD><TD width="142" valign="top"><P> </P></TD></TR><TR><TD width="25" valign="top"><P>5 </P></TD><TD width="113" valign="top"><UL><LI>circulation</LI></UL><P> </P></TD><TD width="103" valign="top"><P>Repeat the process2-4 </P></TD><TD width="102" valign="top"><P> </P></TD><TD width="142" valign="top"><P>30 </P></TD></TR><TR><TD width="25" valign="top"><P>6 </P></TD><TD width="113" valign="top"><P>extension</P></TD><TD width="103" valign="top"><P>72℃</P></TD><TD width="102" valign="top"><P>10min </P></TD><TD width="142" valign="top"><P> 1</P></TD></TR><TR><TD width="25" valign="top"><P>7 </P></TD><TD width="113" valign="top"><P>Hold</P></TD><TD width="103" valign="top"><P>4℃</P></TD><TD width="102" valign="top"><P>Hold</P></TD><TD width="142" valign="top"><P> </P></TD></TR></TBODY></TABLE><P> 2. Purify the PCR product with a DNA purification kit. 3. Add the appropriate amount of DMT enzyme,
            hold for one hour at 37 ° C. 4. Transform 5μl digested DNA into competent cells DH5α, incubate on ice for
            30min.
             42° C heat shock, 45s. Incubate on ice for 2min. add 200μl of LB. incubate at 37 °C for 1 h, 220rpm/min.
             5. pipet 200μl from each tube onto the plate with appropriate resistance, and spread the mixture evenly
            across the plate. Incubate at 37℃ overnight. Position the plates with the agar side at the top, and the lid at
            the bottom. 6. Select single colonies for sequencing.</P><H2 id="section-10" style="padding-top: 100px; margin-top: -50px;">Reagents</H2><P>1. LB medium（lysogeny broth） The recipe for 1l LB media is as follows: Tryptone 10g/L  Yeast
            extract 5g/L  NaCl 10g/L 2. 0.1mM Kanamycin MW of Kanamycin：582.58 Store at -20℃            3. LB plate The recipe for 1l LB plate is as follows: Tryptone 10g/L  Yeast extract 5g/L  NaCl 10g/L 15g Agar Add appropriate amount of resistance. 4. 2YT medium The recipe
            for 1l LB plate is as follows: Tryptone 16g/L  Yeast extract 10g/L  NaCl 5g/L 5. 0.5mM
            IPTG  MW of IPTG：238.30 Store at -20℃ 6. 50mM PBS buffer,PH8.0 A：0.05mol/L Na2HPO4溶液 B：0.05mol/L KH2P04溶液 137mMNaCl，2.7mMKCl，10mMNa2HPO4,2mMKH2PO4 for 1L.</P><H2 id="section-11" style="padding-top: 100px; margin-top: -50px;">References</H2><P>http://parts.igem.org/Help:Protocols/Transformation https://2015.igem.org/Team:Aachen/Project/Overview
             http://www.zymoresearch.com/category/all-products
             http://www.corning.com/worldwide/en/products/life-sciences/resources/brands/axygen-brand-products.html
             http://www.tsingke.net/shop/
             http://www.cwbiotech.com/
            </P><H2 id="section-12" style="padding-top: 100px; margin-top: -50px;">the LiAc SS carrier DNA PEG method</H2><P>
            1.use a sterile pipet tip to pick Saccharomyces cerevisiae from plates，throw the tip into the tubes of appropriate medium，incubate
            in a rotary shaker for 12h. 2.measure OD600, transfer x(x=(50×0.2)/(OD600×dilution ratio))
            ml Saccharomyces cerevisiae into 50ml YPAD. 3.incubate for 4-5h to make OD600 reaches 0.8-0.9. 4.boil
            ssDNA. 5.Centrifuge at 3000g for 5 min. Discard the filtrate. Repeat washes with 25ml deionized water twice. 6.Transfer the cells to 1.5 mL centrifuge tube. Add 1ml of deionized water, resuspend the cells gently. 7.Centrifuge at 13000rpm for 30s. Discard the filtrate. 8.Add 1ml of deionized water, resuspend the
            cells. Pipet 100ul into each 1.5 mL centrifuge tube. 9.Centrifuge using a Mini Centrifuge. Discard the
            filtrate. 10.System for transformation:</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="274" valign="top"><P>PEG3350(50%)</P></TD><TD width="271" valign="top"><P>240ul</P></TD></TR><TR><TD width="274" valign="top"><P>LiAc 1.0M</P></TD><TD width="271" valign="top"><P>36 ul</P></TD></TR><TR><TD width="274" valign="top"><P>SSDNA（2.0mg/ml）</P></TD><TD width="271" valign="top"><P>50 ul</P></TD></TR><TR><TD width="274" valign="top"><P>DNA and H2O</P></TD><TD width="271" valign="top"><P>34 ul</P></TD></TR><TR><TD width="274" valign="top"><P>Total</P></TD><TD width="271" valign="top"><P>360ul</P></TD></TR></TBODY></TABLE><P> 11.Incubate at 30℃ for 20min. 12.42℃ heat shock for 40min. pipet 100ul from each tube onto the appropriate
            plate, and spread the mixture evenly across the plate. Incubate at 30℃ for 2-3 days. Position the plates with
            the agar side at the top, and the lid at the bottom. 13.Prepare plasmid for sequencing.</P><H2 id="section-13" style="padding-top: 100px; margin-top: -50px;">Whole-cell catalysis</H2><P>
            1. Prepare sterile tubes of 5 ml of 2YT+antibiotics. Use a sterile pipet tip to pick bacteria from plates. Throw the tip
            into the tubes. Incubate in a rotary shaker at37℃ for 3-4h. 2. Transfer 120µl of bacteria from
            a slant culture into an Erlenmeyer flask containing 60 mL LB medium with appropriate resistance, incubate at
            37 °C.
             3. When the OD600 reaches 0.6-0.8, the induction of IPTG (0.5 mM) should be carried out. Incubate at 30
            °C on a rotary shaker incubator at 220 rpm for 14 h. 4. Harvest the bacteria(6000rpm/min 7min). Wash with
            30ml PBS. 5. The biocatalytic reaction mixture contained 10% glycerol, E▪ coli and 50mM PBS. Reaction time
            gradient: 8h, 16h, 32h. 6. Use HPLC for further analysis. Reference: Li N, He Y, Chen Y, et al. Production of cyclic adenosine-3′,5′-monophosphate by whole cell
            catalysis using recombinant Escherichia coli, overexpressing adenylate cyclase[J]. Korean Journal of Chemical
            Engineering, 2013, 30(4):913-917.</P></DIV></DIV><FOOTER><DIV class="row"><DIV class="col-lg-12"><P>Copyright © Your Website 2014</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>