"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Brainstorm/Rainbow Bacteria
From 2009.igem.org
(Redirected from Team:Groningen/Rainbow Bacteria )
The Project
The Project
WholeSystem
Applications
Transport
Accumulation
Metal-sensitive Promoters
Gas Vesicles
Modelling
Modelling
Detailed model
Characterization
Downloads
HumanPractice
HumanPractice
safety and security
Ethics
Vision
Survey
Biobricks
Biobricks
Our registry
Submitted Biobricks
Used Biobricks
Project Plan
UPEDU
Project Plan
Brainstorm
Future
Risklist
The Team
The Team
Pictures
Videos
contact
Acknowledgements
Notebook
April
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
November
M
T
W
T
F
S
S
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
Glossary
Literature
[1]
Protocols
Use or mention of the Cre/Lox System
Tianjin University 2008 (China) – Made use of the CRE/LoxP combination (together with other strategies) to combine DNA and make “Foolproof Plasmid Self-Assembly Systems”.
MIT 2006 (USA) – Mention of mutant LoxP sites to prevent inversion of sequences (Lox66 and Lox71)
Imperial College 2006 (London) – Biological oscillator by using the CRE/Lox system as a switch by incorporating DNA to prevent downstream transcription. In the end used Quorum Sensing to create a predator – pray system.
Paris 2007 (France) – (Synthetic Multicellular Bacterium) Use of CRE/Lox system to influence differentiation in germ line, because it was readily available, largely used and well described. Low concentrations of DAP triggered the transcription of Cre-recombinase (under the influence of DAP-sensitive promoter), which caused the removal of a gene required for bacterial reproduction by site-specific recombination (SSR), and resulted in the activation of a gene for the production of DAP by them. The SSR sites might become a burden.
Berkeley 2007 (England) – Registration of recombinase parts (Cre), Bacteriophage P1-derived Cre-Lox.
USTC 2008 (China) – made use of Cre to cut out “sender” genes upon receiving a signal from a sender cell and thereby creating a receiver cell.
Tsinghua 2008 – use of the Cre/Lox system in their second project, but is unclear why they used it.
Caltech 2008 - Wilfred Not Cre/LoxP but it does induce variation in the population
Hong kong 2008 Wilfred Look at the overlapping T7 promoters for variation in response
Use or mention of XFP’s
Utah State University – use of GFP as a reporter gene to indicate the maximum production level of PHB.
University of Sheffield – use of GFP as a reporter gene upon detection of a pathogen by a receiver protein.
UNIPV-Pavia – use of GFP and RFP as reporter genes in electronically based circuit (on/off).
UCSF – use of GFP as a reporter gene in DNA silencing by Chromatin.
TUDelft – use of different method to produce colors, but results in this area were not very promising (also due to ordered parts from iGEM).
Tokyo Tech – use of GFP as a reporter gene in their “coli touch” display experiment.
Paris – use of (E)CFP, YFP, GFP and mRFP in the construction of a oscillating time clock project.
Newcastle University – use of a XFP as a reporter gene to indicate the amount of increase in POP’s.
Missouri University – use of RFP with special introduced promoter.
Minnesota – use of GFP and RFP as reporter genes.
Bologna – use of GFP and RFP as reporter genes.
Parts in the Registry of Standard Parts :
Fluorescent protein sequences
Workable are the wild-type GFP (and a few mutants), RFP (and cherry version)
Not-workable (no info) are YFP and CFP (yellow and cyan), OFP (orange), SBFP2 (blue) and a few mutant proteins.
Recombinase
Workable are Cre DNA recombinase, and Hin invertase
No info (not-workable) are mutant (altered stop/start codon) Cre versions, and a few other enzymes like integrase.
Recombination sites
Workable is the Lox site for recombination
No info (not-workable) are lox66 and lox71
Membrane proteins
Only a few mentioned in the lists of receptors, transporters, channels, pumps and “other proteins”, maybe 40 in total, but I am not familiar with these names ( TLR , Omp).
All of the workable protein sequences are listed as “1 star” in the availability list!!
Literature
J. Livet et al., Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system, 2007, Nature, Vol. 450, 56-63.
J. W. Lichtman, J. Livet & J. R. Sanes, A technicolour approach to the connectome, 2008, Nature Reviews neuroscience, Vol. 9, 417-422.
Retrieved from " http://2009.igem.org/Team:Groningen/Brainstorm/Rainbow_Bacteria "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
