"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Protocols
From 2009.igem.org
The Project
The Project
WholeSystem
Applications
Transport
Accumulation
Metal-sensitive Promoters
Gas Vesicles
Modelling
Modelling
Detailed model
Characterization
Downloads
HumanPractice
HumanPractice
safety and security
Ethics
Vision
Survey
Biobricks
Biobricks
Our registry
Submitted Biobricks
Used Biobricks
Project Plan
UPEDU
Project Plan
Brainstorm
Future
Risklist
The Team
The Team
Pictures
Videos
contact
Acknowledgements
Notebook
April
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
May
M
T
W
T
F
S
S
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
June
M
T
W
T
F
S
S
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
July
M
T
W
T
F
S
S
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
August
M
T
W
T
F
S
S
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
September
M
T
W
T
F
S
S
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
October
M
T
W
T
F
S
S
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
November
M
T
W
T
F
S
S
 
 
 
 
 
 
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
Glossary
Literature
[1]
Protocols
Protocols
Cloning
Quality control
Measurements
List of solutions
PCR
Colony PCR
Fermentation
Media
Plasmid Isolation
Restriction analysis
Buoyancy test
Antibiotics
Restriction
Membrane protein isolation
Metal uptake assay for E. coli
Chemicals
Annealing synthetic oligo's
Fluorescence measurement
Ligation
Death assay
Making competent cells
Fluorescence of resting cells with J61002-pArsR
Transformation
Basic Cloning Strategy:
Transform E. coli TOP10 (genotype DH 10B) with GVP (), a metal ion transporter ( HmtA and GlpF ) and accumulation proteins.
PCR the restriction sites out and add BioBrick pre- and suffix → Use BBFRCF10 .
Primers should be ordered for the different genes.
Add a RBS () in the primer for the BioBrick prefix.
Add a terminator () via cloning.
For GVP the RBS is included in the construct, and biobrick suffix is included in the construct. The prefix is missing because of the EcoRI site in the middle of the plasmid!! This may give problems!!
PCR restriction sites out. !!Both PCR reactions for pre/suffix and restriction sites can possibly be done in 3 seperate PCR reactions !!
Test expression / phenotype of separate proteins (if possible in the vectors they are supplied in).
Put both systems (GVP and metal import) on a high and low copy number (supplied by "vector group"). This is needed to prevent plasmid / expression incompatibility when both systems are used in one strain.
The metal transporter and accumulation protein should be cloned behind each other. If possible on a synthetic operon.
Clone the different systems for Cu, Zn, As, (Hg) in parallel .
(If needed and not already supplied by "vector group") In parallel clone metal sensitive promoters in front of a fluorescent protein (GFP) and in front of the gvp cluster.
(If needed and not already supplied by "vector group") In parallel clone different promoters in front of the two systems.
Inducible like pBad or pLac
Constitutive with expected high and low expression yield
Metal sentitive promoter (only for gvp system)
Then try to get both systems in one E. coli strain, test different possibilities with the high + low copy nr. vectors
Cloning
PCR
12.5 μL Phusion mastermix*
1 μL forward primer
1 μL reverse primer
0.5 μL template
10 μL demi water
*Phusion master mix contains:
200 μL 5x Phusion HF buffer
8 μL 25 mM dNTP's
282 μL MilliQ water
10 μL Phusion Polymerase
PCR Reaction
Hotstart
95 °C, 2 min.
25 cycles
95 °C, 30 sec.
61 °C, 20 sec.
72 °C, 1.5 min.
End
72 °C, 10 min.
4 °C, ∞
Plasmid isolation
Usually performed using Miniprep kits like NucleoSpin® Plasmid, (Machery nagel) or GeneElute™ Plasmid Miniprep Kit (Sigma-Aldrich). This is a consensus protocol.
Spin down ON Culture in table top centrifuge, 1 min. 13.000 RPM
Resuspend pellet, RNAse is added to degrade RNA (200 μL)
Add lysis buffer (200 μL), to lyse the cells and to release their contents
Add neutralization buffer (350 μL), proteins will denaturate
Centrifuge in table top centrifuge 10 min. 13.000 RPM
Add clear lysate in column provided in kit
Spin down 1 min. 13.000 RPM
Add wash buffer (usually needs EtOH to be added!) (500 - 750 μL)
Remove flow-through and spin again to remove residual wash buffer
Put column in clean 1.5 mL cup and add 15 - 50 μL MilliQ water or Tris buffer (pH=8.0)*
Incubate for 1 - 2 min.
Spin down 1 min. 13.000 RPM
*Less volume gives higher concentrations, supplied Tris buffer is claimed to give higher yields
Restriction
Mix
1 μL 10x fast digest buffer ( Fermentas ) or correct conventional buffer
0.5 μL Enzyme A*
0.5 μL Enzyme B*
8 μL DNA to be digested**
Incubate 0.5 - 1 h @ 37 °C (Fast digest do not require long incubations, when using conventional enzymes 1 h. should be maintained)
Purify cut plasmid using PCR clean up kit
* a combination of two of the following (when using biobrick standard) SpeI , EcoRI , PstI and/or XbaI
** When digesting vectors, bring the digested vectors to 1% agarose gel and cut out with scalpel, purify using gel purification kit ( NucleoSpin® Extract II, Machery nagel , Zymoclean™ Gel DNA Recovery Kit or similar) to an end volume indicated by the kit (End volume determines concentration, variations are possible) (alternatively, Phosphatase treatment of linearized vector )
Annealing synthetic oligo’s
Phosphorylation of 5' ends & hybridization [1 ]
Mix:
3 μL 100 µM (anti-)sense oligo
1 μL 10 x PNK (polynucleotide kinase) buffer ( Fermentas Buffer A )**
2 μL 10mM ATP **
1 μL T4 polynucleotide kinase (PNK)
3 μL MilliQ
(for selfcloser control, do not add oligo's. Instead 6 μL MilliQ in total)
Incubate @ 37 °C for 1.5 hours.
Mix
10 μL Sense mixture
10 μL Anti-sense mixture
3 μL 0.5 M NaCl
Place in boiling water for 3 min., and allow the reaction to cool to room temperature.
Upon reaching room temperature add restricted vector (see for ratio Ligation
If kept at low temperature before ligation heat up the annealing mixture up to 65 °C for 1 min. to prevent the formation of multimers
**Alternatively T4 DNA Ligase buffer can be used, already containing ATP
Ligation
Mix*
1 μL T4 ligase buffer
7.5 μL vector (purified from gel)
1 μL Insert
0.5 μL T4 ligase
Incubate
1h RT
or
ON @ 4 °C
*This is a consensus, calculations should be performed to have the ligations be done in a 5:1 - 10:1 (Insert:Vector) mass ratio.
Making competent cells
Competent cells: TOP10 & DB3.1
10 mL ON culture is used to inoculate LB , 100 μL ON culture per 20 mL*
Cultures are grown @ 37 °C until an OD600 of 0.2 ~ 0.3 is reached.
Cultures are spinned down 5 min. @ 4000 rpm, 4 °C
Supernatant is removed and pellet (per 20 mL culture) is resuspended in 5 mL chilled 0.1 M CaCl2
Suspension is incubated on ice for 10 min.
Suspensions are spinned down 5 min. @ 4000 rpm, 4 °C
Supernatant is removed and pellet is resuspended in 1770 μL chilled 0.1 M CaCl2 and supplemented with 230 μL 87% glycerol prior to making aliquots.
Cells are divided in 50 μL aliquots
Cells are snapfrozen in liquid nitrogen and stored @ -80 °C
* Cultures should be grown in the ratio 1:5 (medium:air), so 10 mL culture in a 50 mL greiner tube.
Transformation
Add 10 uL of ligation mixture or 1 uL  isolated plasmid to competent cell aliquot
+ control: 1 μL  or  plasmid (no death gene!), - control: 1 μL MilliQ*
Alternatively a single cut plasmid can be taken as a ligation control
*Alternative - control: 1 μL pSB1AC3 or pSB3K3 carrying ccdB deathgene
Incubate on ice for 15 - 30 min.
Heatshock 45 sec. @ 42 °C or 5 min. 37 °C
Let cells relax on ice for 1 - 2 min.
Add LB 200 μL (or 800 μL when spinning cells down, see below)
Incubate 37 °C, 250 RPM for 1 h
Plate out on LB-agar + Kanamycin (30 μg/ml for pSB3K3 ) or Ampicillin (100 μg/mL for pSB1AC3 )
Plate out 50 μL & 200 μL (or 100 μL after spinning down and resuspending cells) of cell suspension
Grow ON @ 37 °C
Checking transformations
See if - control is empty for functioning antibiotics and death gene
See how many colonies on + control for functioning competent cells
See how many selfclosers and compare to samples (>10x on sample vs. selfcloser)
If enough transformants, inoculate 3 - 5 colonies in an ON culture
Alternatively perform colony PCR
Quality control
Colony PCR
Put colony in 1 μL MilliQ water
Put colony suspension in microwave for 1 min. 1000 W
Use this as DNA template
PCR reaction
21 μL Taq mastermix*
1 μL forward primer
1 μL reverse primer
1 μL template
1 μL Taq polymerase
*Taq master mix contains:
100 μL Taq NH4
8 μL dNTP's
80 μL MgCl
652 μL MilliQ water
PCR Reaction**
Hotstart
95 °C, 2 min.
25 cycles
95 °C, 30 sec.
61 °C, 20 sec.
72 °C, 1.5 min.
End
72 °C, 10 min.
4 °C, ∞
Put PCR product on agarose gel
** Indication, actual reaction program depends on primer set (Temperature of annealing) and the length of the template (Duration of elongation)
Restriction analysis
See also Restriction , however in checking the presence of a multitude of bricks more diverse enzymes can be used. Also the incubation time can be shortened (Pour an agarose gel, wait for it to solidify and put reaction on gel) because it is not required that everything is cut.
Membrane protein isolation
Use 20 mL of ON culture to start main culture in 1L LB medium containing  50 μg/mL ampicillin
Incubate 37 °C, 250 RPM until OD600 = 0.6, (approximately 2.5h of incubation, check the OD600 every hour)
Add inducer (e.g. [Team:Groningen/Protocols#IPTG|IPTG]])
Incubate 1 h, 37 °C, 250 RPM
Culture Wash.
Cool culture on ice
Spin down culture @ 8000 rpm, 10 min, 4°C
Wash pellet with 40 ml ice-cold 50 mM KPi, pH 7.0
Spin down culture @ 8000 rpm, 10 min, 4°C
Resuspend pellet in 12 ml 50 mM KPi pH 7.0
Homogenization
Sonication on ice
9 cycles of 15 sec. sonication, 45 sec. rest
Separation fractions
Spin down @ 8000 rpm, 10 min, 4 °C
Collect supernatant
Spin down at 90 000 rpm, 25 min, 4 °C
Resuspend pellet in 1 ml of 50 mM KPi pH 7.0 + 1M NaCl
Spin down @ 80 000 rpm, 25 min, 4 °C
Solubilization
Resuspend in 950 μl of solubilization buffer (50 mM KPi pH 8.0 + 400 mM NaCl + 20% glycerol) and  50 μl of 10% DDM
Incubate in 4°C with shaking for 30 min.
Spin down @ 80 000 rpm, 25 min in 4°C
Collect the supernatant
(Supernatant can be stored at this stage)
Add Ni-NTA resin (30 μL buffer A , 0.1% DDM (Bis(4-chlorophenyl)methane))
Incubate ON @ 4 °C
Spin down, 4 min. 3500 RPM
Remove supernatant
was resin with 1 mL buffer B , 0.1% DDM
Spin down, 4 min. 3500 RPM
Remove supernatant
Elute protein by buffer C , 0.1% DDM (50 μL)
Add protein loading buffer (with dithiothreitol (DTT))
Run 12% SDS-PAGE
Continue to Coomassie staining
Staining of SDS-PAGE gels with Coomassie Brilliant Blue
Heat gel in staining solution and shake for 10 min.
Poor off staining solution and add destain.
Heat gel in destaining solution and shake.
Replace destaining solution after 10 min and repeat until ready.
Measurements
Fermentation
Performed in 2L autoclavable fermentor with dished bottom vessel stirred fermentor
Autoclave closed fermentor system
Inoculate 1.3 L LB (+100 µL Y30 antifoam ) with 20 mL ON culture was used to
Airflow rate of 1 vvm
pH @ 7 (by addition of 4 M NaOH or 1 M HCl)
Temperature @ 37 °C
Agitation 400 to 800 RPM*
Oxygen concentration >50%
Take samples every 0.5 to 1 h. to determine optical density at 600 nm
50 mL samples in every growth phase (pre-exponential, early exponential, exponential, late exponential, steady state)
Spin samples down 35 min., 1000 RPM and remove supernatant
Continue Buoyancy test
*6-bladed flat disc turbine (Rushton type) impeller (60 mm diameter) at the bottom to disperse the bubbles coming from the sparger underneath and a 3-bladed marine impeller, vortex (60 mm diameter) halfway the broth volume to create an axial flow.
Buoyancy test
Continued from cultures (flask or fermentor) after centrifugation
Resuspend pellet in 1 - 5 mL saline solution
Determine OD600
Dilute suspension to OD600 1.5 with saline solution
Put homogeneous suspension in tubes, take care of descent lighting from behind (day light is best)
Record decrease of buoyancy (matter of hours in fermentation cultures, days in shakeflask cultures)
Metal uptake assay for E. coli Kostal 2004
Grow ON culture of E. coli @ 30 °C
Use E. coli + control vector, E. coli + pArsR - RFP , E. coli + pLac-fMT
Inoculate day culture 1:50, grow in 1L TB - Amp (100ml per time/[As(III)] sample)
Take OD600 samples every 1 - 1.5 h of E. coli + pLac-fMT
Induce E. coli + pLac-fMT at OD600 ~0.6 with 0.5 mM IPTG.
Harvest the cells @ stationary phase (after ~30 h) by spinning down @ 4000 RPM for 20 min. in Sorval centrifuge.
Wash 2 times with TB74S buffer
Resuspend in prewarmed (30 °C) TB74S buffer up to a OD600 of ~25
Take a 1 mL sample in small aluminum boxes and dry @ 104 °C for >4 h
Afterwards measure the dry weight of the sample and calculate the weight/volume of the entire sample.
For the concentration range:
Incubate 5 samples (of same time point) for 1h @ 30 °C with 0μM, 10 μM, 20 μM, 50 μM and 100 μM As(III).
For the concentration range:
Incubate 5 samples (of same concentration) @ 30 °C with 10 or 100μM As(III) for 0, 10, 20, 40, 60 min.
Harvest cells by spinning down.
Wash the cells with TB74S buffer
Resuspend in 10ml demi water.
Dry sample @ 65 °C for 2 days.
Store @ 4 °C or -80 °C
Determine the amount of As(III) in the cell at different stages and at different uptake concentrations using ICP-MS
Analysis of arsenic concentration of ICP-MS
Weigh 0.1g dried E. coli cells.
Add 5 ml  65% nitric acid.
For destruction the following microwave program was used:
Stage 1
Stage 2
Power(max)
1200
1200
Power(%)
100
100
Ramp(min)
15
15
Hold(min)
0
30
Temp(°C)
140
210
Let the samples cool down.
Dilute the samples by adding demi water up to 50 mL
If needed, spin down 15 min. @ 4000rpm in a Sorvall centrifuge.
Measure the arsenic concentration by ICP-MS using both the standard mode (shows interference peak from multi-atomic molecule argon-chloride with the arsenic peak) and the collusion cell technology mode (doesn’t show the interference peak but has a 10x lower resolution than standard mode).
Use a standard curve between 0 - 10 µg As/L and 0 - 100 µg As/L using a certified 1000 ppm (mg/L) stock
Fluorescence measurement
Dilute ON culture 1:20 in LB + Ampicillin in 50 mL greiner tube
Incubate 37 °C, 250 RPM until an OD600 ~0.5
Spin down 10 min. 4000 RPM, 4 °C
Resuspend pellet in LB + Ampicillin (half the volume used before)
Incubate @ 4 °C, 30 min.
Load samples onto a 96-wells plate, 250 μL
Induce by 1.25 μL of the following stock solutions
Metal
CuSO4
1 M
100 mM
10 mM
1 mM
0 mM
ZnSO4
1 M
100 mM
10 mM
1 mM
0 mM
NaAsO2
1 M
100 mM
10 mM
1 mM
0 mM
Final con.
5000 μM
500 μM
50 μM
5 μM
0 μM
Measure the fluorescence and OD600 every hour.
RFP is excited @  580 nm and emission is measured at 609 nm.
Store the plates between the measurements in a shaking incubator @ 37 °C.
Fluorescence of resting cells with - pArsR
This protocol was used to be able to correlate the determined fluorescence values with the arsenic concentrations measured by ICP-MS (after an arsenic uptake assay).
Cells were used as prepared for the ( ) arsenic uptake assay.
Cultures with an OD600 of ~25 were induced with 100 µM NaAsO2 for 60 to 160 min.
The fluorescence and OD600 were measured by a plate reader (Tecan, infinite 200, Tecan group, Switzerland).
Relative promoter units were calculated according to formula 9 from Kelly 2009 .
Death assay
Metal sensitivity assay Lewinson 2009
Measurement:
Grow selected strains ON in LB medium with or without antibiotic
Induce in culture with inducer (in our case 0.5 mM IPTG )
Strains used in our tests
Test 1:
Test 2:
WT (+pSB1AC3)
WT (+pSB1AC3)
pLac-HmtA
pLac-HmtA
pLac-GlpF
pLac-GlpF
pLac-GlpF-fMT
pLac-GlpF-fMT
plow-GlpF-fMT
pLac-GlpF
Measure OD600 of ON culture and dilute to an OD600 of 0.05 in LB + antibiotic & inducer ( IPTG in our case). Inducer should be right concentration for use in microtiterplate (because you then dilute culture 150/200=1.33 times)
Add 150 ul of culture to 96 well microtiter plate (in triplo/quadruplo)
Add desired concentration of selected metal in 50 μL LB + antibiotic
Metals used in our test
Metal Concentration
NaAsO2
0 μM
1 μM
10 μM
50 μM
CuSO4
0 μM
50 μM
250 μM
500 μM
Measure in Tecan Infinite 200 microplate reader (Tecan Group Ltd., Männedorf, Switzerland) or Tecan microplate *reader. Protocol:
Measure OD at 600 nm,
Every 15 minutes for 16-20hrs
Linear shaking, 6mm
37°C
Analysis:
Plot for different strains OD600 against time
Plot for different strains, the different metal concentrations against OD600 at 12 hours
List of solutions
Media
LB(Agar)
10 g (Bacto)Trypton
10 g NaCl
5 g Yeast extract
(1.5% Agar, 15 g)
Dissolve in 1 L demi water
Autoclave
Store @ RT (LB) or 60 °C (LB-Agar)
TB medium
12 g Bacto-Tryptone
24 g Bacto-Yeast Extract
4 mL Glycerol [87%]
Dissolve in 900ml demi water
Separetely prepare 100 mL Kpi
0.17 M KH2PO4  (mw=136.09g/mol) (6.94g/300ml)
0.72 M K2HPO4  (mw=174.18g/mol) (7.62g/300ml)
dissolve in demi water
Autoclave and mix
Antibiotics
Ampicillin
100 mg/ml Ampicillin (1000x) Stock
1 g of Ampicillin sodium salt in 10 mL of demiwater (or 50% EtOH)
Add NaOH or KOH to allow the Ampicillin to dissolve
Filter sterilize 0.2 μm filter and aliquot
Store -20 °C
Chloramphenicol
35 mg/ml Chloramphenicol (1000x) Stock
0.35 g in 10 mL 100% EtOH
Filter sterilize 0.2 μm filter and aliquot
Store -20 °C
Kanamycin
50 mg/ml Kanamycin (1000x) Stock
500 mg in 10 mL demi water
Filter sterilize 0.2 μm filter and aliquot
Store -20 °C
Chemicals
Buffer A
10 mM Imidazole
600 mM NaCl
50 mM KPi pH 8.0
10% Glycerol
0.1% DDM
Demi water
Buffer B
20 mM Imidazole
600 mM NaCl
50 mM KPi pH 8.0
10% Glycerol
0.1% DDM
Demi water
Buffer C
500 mM Imidazole
600 mM NaCl
50 mM KPi pH 8.0
10% Glycerol
0.1% DDM
Demi water
0.1 M CaCl2
0.3319 g CaCl2
Dissolve in 30 mL demi water
Destaining solution
16 % methanol
10 % acetic acid
74 % water
0.15 M NaCl (Saline solution, 0.9% NaCl)
9 g NaCl
Dissolve in 1 L demi water
4 M NaOH
160 g NaOH
Dissolve in 1 L demi water
~1 M HCl
500 mL demi water
500 mL HCl (37%, 11 M)
1 M IPTG
2.38 g Isopropyl-beta-D-thiogalactopyranoside (IPTG) in 10 mL demi water.
Filter sterilize with a 0.22 μm syringe filter.
Store in 1 mL aliquots at -20 °C.
Sodium Arsenite (III)
100mM Na-As solution
filter sterilize
Staining solution
0.25 % Coomassie Brilliant Blue R-250
50 % methanol
10 % acetic acid
40 % water
TB74S Buffer
0.605 g Tris (5mM)
8.76 g NaCl (150mM)
Dissolve in 1 L Demi water
Set pH with HCl to 7.4
10x TBE buffer
108 g Tris
55 g Boric acid
8.3 g EDTA
Dissolve in 1 L demi water
Adjust pH to 8.3
Retrieved from " http://2009.igem.org/Team:Groningen/Protocols "
Categories : Team:Groningen | Protocol | Escherichia coli
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
