<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Toronto_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Toronto/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Toronto/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Toronto/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Toronto/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Toronto/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Toronto/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="splash-wrapper"><NAV><DIV class="container" style="position:relative;"><UL id="navbar"><DIV id="tabs" style="display:inline-block; width:80%; position:absolute; bottom:0;"><LI id="home"><A href="https://2014.igem.org/Team:Toronto#navbar" scroll-on-click="">Home</A></LI><LI id="team"><A href="https://2014.igem.org/Team:Toronto/Team#navbar" scroll-on-click=""> Team</A></LI><LI class="active" id="project"><A href="https://2014.igem.org/Team:Toronto/Project#navbar" scroll-on-click=""> Project</A></LI><LI id="parts"><A href="https://2014.igem.org/Team:Toronto/Parts#navbar" scroll-on-click=""> Parts</A></LI><LI id="modeling"><A href="https://2014.igem.org/Team:Toronto/Modeling#navbar" scroll-on-click=""> Modeling</A></LI><LI id="notebook"><A href="https://2014.igem.org/Team:Toronto/Notebook#navbar" scroll-on-click=""> Notebook</A></LI><LI id="humanpractices"><A href="https://2014.igem.org/Team:Toronto/HumanPractices#navbar" scroll-on-click=""> Human Practices</A></LI><LI id="safety"><A href="https://2014.igem.org/Team:Toronto/Safety#navbar" scroll-on-click=""> Safety</A></LI><LI id="attributions"><A href="https://2014.igem.org/Team:Toronto/Attributions#navbar" scroll-on-click=""> Attributions</A></LI></DIV></UL></DIV></NAV></DIV><SECTION id="main"><DIV id="main-inner"><DIV id="view" ng-view="" data-autoscroll="true"><DIV class="first container-wrapper"><DIV class="container"><H1 id="background"><SPAN>Background</SPAN></H1><H3><SPAN>The need for biocontainment</SPAN></H3><P>
Synthetic biology has the potential for a variety of applications outside the lab. For example, many biosensor and bioremediation systems developed using synthetic biology entail the release of genetically modified organisms into an outdoor environment.
</P><P>
The potential impact of synthetic organisms on the natural environment is difficult to predict. However, as demonstrated by the devastating impact of invasive species on ecosystems around the world today, the introduction of new organisms into the environment poses significant risks. Thus, the development of reliable means of biocontainment will be essential for synthetic biology to be considered safe for routine use.
</P><H3><SPAN>Conventional genetic safeguards for biocontainment</SPAN></H3><P>
Many of the biocontainment systems being developed today are genetic safeguards that work by conditionally expressing a gene that leads to the host cell's death (Moe-Behrens et al. 2013). Such systems are called &quot;kill switch&quot; systems. </P><H3><SPAN>The problem with kill switches</SPAN></H3><P>
At first glance, the use of kill switches seems to be a simple and effective solution to the problem of biocontainment. However, when considered from an evolutionary perspective, kill switches may not be as effective in practice as they are conceptually. There are two related issues with kill switches: mutation and selection

</P><H4><SPAN>Mutation</SPAN></H4><P>
Unfortunately organisms aren't as well behaved as machines. You may have a kill switch set up to contain your synthetic organisms, but chances are, there are going to be some cells that have mutations that break your kill switch. There has been evidence that in many implementations of kill switch systems, a small (but not insignificant) proportion of individuals in a population are able to survive the activation of the kill switch due to spontaneous mutations that render the kill switch ineffective (Knudsen and Karlstrom, 1991). These cells are not going to get killed when you turn on the toxic gene, so they could escape and have negative consequences.
</P><H4><SPAN>Selection</SPAN></H4><P>
The other problem with kill-switches is that they actually <I>encourage</I> mutations that inactivate kill switches. Since kill-switches kill off cells that have working kill switches, they impose selective pressure on population to evolve resistance to the kill switch. You're essentially saying to the cells: &quot;mutate or die&quot;, and the cells that end up mutating are going to be the ones that eventually dominate the population.
</P><H3><SPAN>A new approach: Plasmid loss genetic safeguard system</SPAN></H3><P>
The way we've approached this issue is to make synthetic organisms lose their synthetic plasmids, instead of killing them outright. When we turn on a plasmid-loss system, most individuals in a population of synthetic organisms would lose the genes responsible for the engineered trait and revert to a wild-type genotype and phenotype. As in the kill-switch systems, a small proportion of individuals could still retain the engineered trait due to mutation inactivation of the genetic safeguard system. However, while we may not be able to get rid of mutation, we <I>can</I> remove the selective pressure on the non-mutants so that the mutants can't dominate the population.
</P><P><B>Figure 1:</B> The effect of (Top) kill-switch and (Bottom) plasmid loss genetic safeguard systems on a hypothetical population of synthetic organisms, containing a small proportion of mutant individuals resistant to the effect of the genetic safeguard. Not shown: the effect of genetic drift on the mutant sub-population over time.
</P><P>
Although resistant mutants are present in both the kill switch and plasmid loss systems, their evolutionary outcomes differ:
<UL style="width:600px; margin: 0 auto;"><LI>
	Kill-switch system: Once the system is turned on, all individuals with an intact genetic safeguard die, leaving a new population consisting entirely of mutant cells resistant to the kill switch. In evolutionary terms, the resistance allele has gone from a very low allele frequency to fixation (100% allele frequency). Since the non-mutants in the population are all dead, the mutant survivors have unimpeded access to all the resources available in the environment and are able to quickly grow and replicate.
	</LI><LI>
	Plasmid loss system: The allele frequency of the inactivated genetic safeguard does not change. This is because the individuals with an intact genetic safeguard do not die, they simply revert back to the wild-type genotype and phenotype. In the new population, the proportion of resistant individuals remains small due to competition from the majority of wild-type individuals.
	</LI></UL></P><H3><SPAN>The effect of genetic drift</SPAN></H3><P>
Genetic drift is the effect of random chance on proportions of alleles in a population. In a population containing both mutant and wild type cells, the proportions of the two will fluctuate slightly every generation simply due to random variability. In most cases the effect of genetic drift is small. However, in a situation where there are very few mutants and many wild-type cells, the &quot;luck&quot; of these few mutant cells will have a disproportional effect on the overall proportion of mutants in the population.
</P><P>
In the kill switch scenario, the population of mutant cells increases quickly, so they are no longer susceptible to extinction by genetic drift. On the other hand, in the plasmid loss scenario, the number of mutants is kept low, so there is a greater chance for genetic drift to lead to extinction of the mutant allele.
</P><H3><SPAN>The big-picture view of biocontainment</SPAN></H3><P>
Ultimately, as noted by Paris Bettencourt's <A href="http://parts.igem.org/Biosafety#Recommendation_from_Human_Practice_Teams">recommendations on biosafety</A>, no containment system can be perfect.
</P><P>
In the case of our plasmid loss system, It is still possible that the mutant cells, carrying the engineered plasmid, could have enough of a fitness advantage to out-compete the wild-type individuals, who have lost the engineered plasmid. However, this outcome is no worse than the worst-case scenario of kill-switch systems. We expect that this worst-case scenario will be unlikely to occur, since plasmids impose a metabolic burden on their host cells. In addition. Even if the engineered trait do confer a slight fitness advantage to the resistant individuals, it may not be enough of a selective advantage to overcome the effect of genetic drift.
</P><P>
From the perspective of the fault-tree analysis proposed by Paris Bettencourt, our plasmid-loss system is more robust than the previous kill-switch approaches. While kill switches are only one mutation away from failure, our plasmid loss system tolerates mutations and reduces the chance of failure. Because of this, even if our plasmid loss system does not work 100% of the time, it is still an improvement to the overall safety of synthetic biology.
</P><P><B>See also:</B><A href="https://2014.igem.org/Team:Toronto/Parts#navbar">Our Parts and Design Page</A></P><H3><SPAN>References</SPAN></H3><OL style="width: 600px; margin: 0 auto;"><LI>
	Moe-Behrens, G. H., Davis, R., &amp; Haynes, K. A. (2013, 12). Preparing synthetic biology for the world. Frontiers in Microbiology, 4. doi: 10.3389/fmicb.2013.00005
	</LI><LI>
	Knudsen, S. M., and Karlstrom, O. H. (1991). Development of efficient suicide mechanisms for biological containment of bacteria. Appl. Environ. Microbiol. 57, 85:92.
	</LI><OL><FOOTER id="footer"><DIV class="container"><P class="pull-right"><A href="#">Back to top</A></P><P style="text-align:left; color: white;">
							2014 University of Toronto iGEM
                                                        <A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Toronto"> Official Team Profile </A></P></DIV></FOOTER></OL></OL></DIV></DIV></DIV></DIV></SECTION><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Toronto/Project">http://2014.igem.org/Team:Toronto/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Toronto/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Toronto/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Toronto/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Toronto/Project&amp;oldid=403753" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>