<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_project_2c"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/project/2c">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/project/2c&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/project/2c&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/project/2c&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/project/2c" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/project/2c</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="menu"><UL><LI><A href="/Team:NYMU-Taipei">Home</A></LI><LI><A href="/Team:NYMU-Taipei/project/overview">Project</A></LI><LI><A href="/Team:NYMU-Taipei/project/overview">Overview</A></LI><LI>Control</LI><LI><A href="/Team:NYMU-Taipei/project/1c">Target</A></LI><LI><A href="/Team:NYMU-Taipei/project/1c1">Inhibitor</A></LI></UL><LI>Cleanse</LI><LI><A href="/Team:NYMU-Taipei/project/2c">Antibiofilm</A></LI><LI><A href="/Team:NYMU-Taipei/project/2c1">Attachment</A></LI><LI><A href="/Team:NYMU-Taipei/project/3c">Care</A></LI><LI><A href="/Team:NYMU-Taipei/project/lcasei">L. casei</A></LI><LI><A href="/Team:NYMU-Taipei/project/hoperfusion">HOPErfusion</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/overview">Modeling</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/overview">Overview</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m1">Growth&amp;pH</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m2">Competition</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m3">Threshold</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m4">Stephen curve</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m5">Effectiveness</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m6">new part 1</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m7">new part 2</A></LI><LI><A href="/Team:NYMU-Taipei/safety">Safety</A></LI><LI><A href="/Team:NYMU-Taipei/notebook/labnotes">Notebook</A></LI><LI><A href="/Team:NYMU-Taipei/notebook/labnotes">Lab notes</A></LI><LI><A href="/Team:NYMU-Taipei/notebook/protocol">Protocol</A></LI><LI><A href="/Team:NYMU-Taipei/hp/overview">Human Practice</A></LI><LI><A href="/Team:NYMU-Taipei/team/member">Team</A></LI><LI><A href="/Team:NYMU-Taipei/team/member">Members</A></LI><LI><A href="/Team:NYMU-Taipei/team/gallery">Gallery</A></LI><LI><A href="/Team:NYMU-Taipei/team/nymu">NYMU</A></LI><LI><A href="/Team:NYMU-Taipei/team/cooperate">Cooperation</A></LI></DIV><DIV id="toothy"><DIV id="convblank"><DIV id="qnaClose">X</DIV><DIV id="qnaNextBt">Next ⇒</DIV><DIV id="qnaPrevBt">⇐ Prev</DIV></DIV><DIV id="convopen">...</DIV></DIV><DIV id="main-content"><H>Cleanse-<SPAN style="color:rgb(249, 103, 236);">Antibiofilm</SPAN></H><DIV class="abstract"><UL><LI>Enzymatically degrade biofilm from within when biofilm conditions are detected.</LI></UL></DIV><DIV class="cont-panel"><DIV href="#2c-1"><P>Get started.</P></DIV><DIV href="#2c-2"><P>How to do it?</P></DIV><DIV href="#2c-3"><P>Test it!</P></DIV><DIV href="#2c-4"><P>Our result~</P></DIV><DIV href="#2c-5"><P>Reference</P></DIV></DIV><DIV class="article indent"><H1 id="2c-1">Before we get started:</H1><P><B>Biofilm</B> formation is part of the reason <I>S. mutans</I> is so devastating to oral health (see <A href="/Team:NYMU-Taipei/project/overview">[Project Overview]</A> for more information). For adequate oral protection, biofilm formation is a problem we must solve, and we can do so internally, by modifying the <I>S. mutans</I> genome, (see <A href="/Team:NYMU-Taipei/project/1c1">[Control: Inhibitor]</A>), or <B>externally</B>, via a modified probiotic. This means we need to know about <B>conditions unique to biofilms</B> in order to design a sensor.</P><P>What happens to our teeth if it is covered in biofilm? Metabolic acids which are waste products of <I>S. mutans</I> naturally begin to accumulate to very high concentrations in the local space, and cannot disperse out into the rest of the mouth. This results in a <B>low pH</B> in areas covered by biofilms, about <B>4.0~5.0</B>. Since our probiotic is physically close to <I>S. mutans</I> (see <A href="/Team:NYMU-Taipei/project/2c1">[Cleanse: Attachment]</A>), if our probiotic detects a low pH, there’s a good chance it is within a biofilm. Figure 1 depicts this in a simplified diagram.</P><DIV><P style="font-size: 14px;display: block; margin: 0 auto; text-align: center;">Figure 1. </P></DIV><P>Now that we know the target pH range, we need to find out what biofilms are made of, and how to get rid of them. Biofilms are actually conglomerated bacteria colonies enmeshed in a self-synthesized layer of extracellular <B>polysaccharide matrix</B>. Of these, <B>PGA</B>, or poly-gamma glutamate, has been demonstrated to be a main component across the biofilm of several phylogenetically diverse bacterial species, and <B>polyglycine bridges</B> can cross-link glycopeptides that make up the cell walls of biofilm-producing bacteria.</P><P>The ideal biofilm degrader must be able to target some, if not all, of these elements.</P><H1 id="2c-2">So how did we do it?</H1><P class="noindent">After careful literature search, we found two promising enzymes:</P><P><B>Lysostaphin</B> is a zinc-containing enzyme 27kDa in size, taken from the genome of <I>Staphylococcus simulans</I>, and was used previously by the <A target="_blank" href="https://2012.igem.org/Team:HIT-Harbin">[2012 HIT-Harbin iGEM team]</A>. It is particularly powerful because it <B>combines the actions</B> of three different peptidoglycan degradation enzymes, and as such can target a wide variety of the glycopeptides that form cell walls, including features such as the polyglycine bridge. Also, unlike other enzymes of this type, it can <B>target both actively growing and non-dividing cells</B>. To add to its merits, lysostaphin <B>causes no side effects in humans</B>.</P><P><B>Dispersin B</B> comes from <I>Aggregatibacter actinomycetemcomitans</I>, a Gram-negative rod oral bacteria. It is a <B>soluble glycoside hydrolase</B> capable of <B>degrading PGA</B>.</P><P>After the enzymes have been decided, we needed to find a promoter responsive to the conditions within a biofilm. The <B>Asr promoter</B> comes from <I>E. coli</I> strain MG1655, and is <B>induced by extracellular low pH</B> of between 4.0~5.0. Perfect for what we need.</P><P>But wait! We’re not quite done yet. Even after being equipped with the Asr promoter, lysostaphin, and dispersin B, our probiotic would still be useless against biofilms unless we <B>export the enzymes out of the cell</B>, where they can do their jobs. Here’s where the signal protein comes in.</P><P><B>YebF</B> is a signal protein naturally secreted by <I>E. coli</I>, though experimental strains rarely secrete this protein, which makes it useful in synthetic biology. Proteins tacked onto YebF can be expected to also <B>be secreted outside of the host cell</B>, so we put this in front of our enzymes.</P><P>At last, our circuit is complete!</P><P>Our expectation is that the three-protein fusion of YebF-Lysostaphin-Dispersin B will be exported out of the cell upon translation by the YebF, where they will be able to do their job of degrading the biofilm, such as in Figure 2.</P><DIV><P style="font-size: 14px; display: block; margin: 0 auto; text-align: center;">Figure 2. </P></DIV><H1 id="2c-3">Putting it to the test!</H1><P>Having a circuit is only the start! Now we must test how well our circuit functions. For the Antibiofilm circuit, we need to test specifically for three things: the function of the Asr promoter, the function of YebF, and the function of lysostaphin and dispersin B.</P><H3>Testing the Function of the Asr Promoter</H3><P>For this test we constructed the following circuit in <I>E. coli.</I> We aim to elucidate the exact pH-activity relation by subjecting the circuit to conditions of various pH, and measuring activity via RFP fluorescence intensity, quantized in OD.</P><P class="noindent">Here is the details of our method:</P><OL><LI>Transform the circuit to <I>E. coli</I>, and grow the modified <I>E.coli</I> on agar plates overnight.</LI><LI>Prepare three of each liquid culture tubes for pH values 3.5, 4, 4.5, 5, and 7, and grow the selected strain in all tubes. Use the growth data from each tube to plot the precise effect of pH on the Asr promoter.</LI><LI>Measure the fluorescence value (RFP), and plot out curves.</LI></OL><H3>Testing the Function of YebF</H3><P>Since we also used YebF in <A href="/Team:NYMU-Taipei/project/1c">[Control: Target]</A>, we have done our functional assays there. Feel free to check it out if you’re interested!</P><H3>Testing the Function of Lysostaphin and Dispersin B</H3><P>For this part of the test, we designed a novel apparatus we called the <A href="/Team:NYMU-Taipei/project/hoperfusion">[HOPErfusion]</A>, which allowed us to culture <I>S. mutans</I> in real saliva (yes, real saliva!) and observe their biofilm formation. The HOPErfusion provided us with the most accurate representation of the human mouth that we could obtain!</P><H3>Experimental Design</H3><P>We have designed two circuits to treat the biofilm formed by <I>S. mutans</I>.</P><P>We cultured <I>S. mutans</I> overnight for 12 hours, and move cultured <I>S. mutans</I> to the modified slide glass which was coated by saliva(37°C, 1 hr). Then we used peristaltic Pump to drop BHI and saliva on slide glass to simulate the flowing atmosphere of oral cavity. All the experiments are conducted in our designed device, which is closed atmosphere mimicking the anoxic atmosphere. </P><P>To test whether our modified <I>E. coli</I> has the ability in decreasing formed biofilm, we put modified <I>E. coli</I> (circuit 1 and 2) to treat <I>S. mutans</I> for 12 hr, 24hr and 36hr. Finally, we did biofilm grow assay via crystal violet stain.
biofilm grow assay: </P><OL><LI>gentally washing with ddH<SUB>2</SUB>O three times.</LI><LI>Air dried at room temperature</LI><LI>Stain with 500ul 0.1% crystal violet for 15min at room temperature</LI><LI>Add ddH<SUB>2</SUB>O three times to wash excess crystal violet</LI><LI>Add 1000 atic acid to extract crystal violet stain and wait for10 min. </LI><LI>Add 20 extract crystal violet stain and 800 ddH<SUB>2</SUB>O to do OD measurement at 595nm wavelength. </LI></OL><DIV style="width: 700px; margin: 0 auto;"><DIV style="display: inline-block;"><P style="font-size: 14px;display: block; margin: 0 auto; text-align: center;"> AIR DRIVED AT ROOM TEMPERATURE</P></DIV><DIV style="display: inline-block; margin-left: 80px;"><P class="noindent" style="font-size: 14px;display: block; margin: 0 auto; text-align: center;"> ADD ddH<SUB>2</SUB>O THREE TIMES TOWASH EXCESS CRYSTAL VIOLET</P></DIV></DIV><H1 id="2c-4">Our results</H1><H3>experiment1: using Hoperfusion to do biofilm growth assay</H3><OL><LI><I>S. mutans</I> growing for 36 hr (control) OD = 0.643</LI><LI><I>S. mutans</I> growing for 36 hr and treat with our modified <I>E. coli</I> for 12hr OD =0.465</LI><LI><I>S. mutans</I> growing for 36 hr and treat with our modified <I>E. coli</I> for 24hr OD =0.391</LI><LI><I>S. mutans</I> growing for 36 hr and treat with our modified <I>E. coli</I> for 36hr OD =0.191</LI></OL><P>Indicated from the graph, we could clearly see that treating our modified <I>E. coli</I> (transformed by Dispersin B circuit and lysostephin circuit ) have a negative impact on biofilm growth of <I>Streptococcus mutans.</I> As a result, we have confidence that our modified <I>E. coli</I> have ability to treat or influence biofilm formation.</P><P>These are the pictures that We co-cultured <I>S. mutans</I> and our modified <I>E. coli</I> through different time span on slide glass coated by saliva and do biofilm grow assay via crystal violet stain and test OD.</P><OL><P>(1)<I>S. mutans</I> growing for 36 hr (control) OD = 0.643</P><P>(2)<I>S. mutans</I> growing for 36 hr and treat with our modified <I>E. coli</I> for 12hr OD =0.465</P><P>(3)<I>S. mutans</I> growing for 36 hr and treat with our modified <I>E. coli</I> for 24hr OD =0.391</P><P>(4)<I>S. mutans</I> growing for 36 hr and treat with our modified <I>E. coli</I> for 36hr OD =0.191</P></OL><H3>experiment2: 24well to test biofilm decrease</H3><P>To test whether our modified e-coli(transformed with dispersin B and lysostephin circuit ) has influence on biofilm growth, we designed function test under two situation in 24 well.</P><P> First one is to co-culture streptococcus mutans and modified e-coli for24hr and do biofilm growth assay. The other one is to culture streptococcus mutans for 12hr and then treat with modified e-coli for 12hr and do biofilm growth assay. 
Also, we compare the streptococcus mutans growth situation in BHI solution with or without sucrose. We used crystal violet stain to do biofilm grow assay.

</P><P>Indicated from the table, we could conclude that under no sucrose, modified e-coli has a great effect on decreasing biofilm formation. Co-culture decreasing rate is up to 64.2% and treat decreasing rate is 51.9% .On the other hand, solution with sucrose will has the bad impact on depleting biofilm, causing low decreasing rate. Overall, we could dedicate that modified e-coli has a good effect on decreasing biofilm formation or depleting biofilm.</P><H1 id="2c-5">Reference</H1></DIV><OL><LI>Na, D., Yoo, S. M., Chung, H., Park, H., Park, J. H., &amp; Lee, S. Y. (January 01, 2013). Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs. Nature Biotechnology, 31, 2, 170-4.</LI><LI>Li, Y.-H., Lau, P. C. Y., Tang, N., Svensater, G., Ellen, R. P., &amp; Cvitkovitch, D. G. (November 15, 2002). Novel Two-Component Regulatory System Involved in Biofilm Formation and Acid Resistance in Streptococcus mutans. Journal of Bacteriology, 184, 22, 6333-6342.</LI><LI>Baev, D., England, R., &amp; Kuramitsu, H. K. (January 01, 1999). Stress-induced membrane association of the Streptococcus mutans GTP-binding protein, an essential G protein, and investigation of its physiological role by utilizing an antisense RNA strategy. Infection and Immunity, 67, 9, 4510-6.</LI><LI>Yoshida, A., &amp; Kuramitsu, H. K. (December 01, 2002). Multiple Streptococcus mutans Genes Are Involved in Biofilm Formation. Applied and Environmental Microbiology, 68, 12, 6283-6291.</LI><LI>Biswas, I., Jha, J. K., &amp; Fromm, N. (August 01, 2008). Shuttle expression plasmids for genetic studies in Streptococcus mutans. Microbiology, 154, 8, 2275-2282.</LI><LI>Li, Yung-Hua, Tang, Nan, Aspiras, Marcelo B., Lau, Peter C. Y., Lee, Janet H., Ellen, Richard P., &amp; Cvitkovitch, Dennis G. (n.d.). A Quorum-Sensing Signaling System Essential for Genetic Competence in Streptococcus mutans Is Involved in Biofilm Formation. American Society for Microbiology.</LI><LI>Wu, J., Cho, M. I., &amp; Kuramitsu, H. K. (January 01, 1995). Expression, purification, and characterization of a novel G protein, SGP, from Streptococcus mutans. Infection and Immunity, 63, 7, 2516-21.</LI></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NYMU-Taipei/project/2c">http://2014.igem.org/Team:NYMU-Taipei/project/2c</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/project/2c" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/project/2c" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/project/2c&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/project/2c&amp;oldid=401399" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>