<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYMU-Taipei_modeling_m3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYMU-Taipei/modeling/m3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYMU-Taipei/modeling/m3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/modeling/m3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYMU-Taipei/modeling/m3&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYMU-Taipei/modeling/m3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYMU-Taipei/modeling/m3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="menu"><UL><LI><A href="/Team:NYMU-Taipei">Home</A></LI><LI><A href="/Team:NYMU-Taipei/project/overview">Project</A></LI><LI><A href="/Team:NYMU-Taipei/project/overview">Overview</A></LI><LI>Control</LI><LI><A href="/Team:NYMU-Taipei/project/1c">Target</A></LI><LI><A href="/Team:NYMU-Taipei/project/1c1">Inhibitor</A></LI></UL><LI>Cleanse</LI><LI><A href="/Team:NYMU-Taipei/project/2c">Antibiofilm</A></LI><LI><A href="/Team:NYMU-Taipei/project/2c1">Attachment</A></LI><LI><A href="/Team:NYMU-Taipei/project/3c">Care</A></LI><LI><A href="/Team:NYMU-Taipei/project/lcasei">L. casei</A></LI><LI><A href="/Team:NYMU-Taipei/project/hoperfusion">HOPErfusion</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/overview">Modeling</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/overview">Overview</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m1">Growth&amp;pH</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m2">Competition</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m3">Threshold</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m4">Stephen curve</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m5">Effectiveness</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m6">new part 1</A></LI><LI><A href="/Team:NYMU-Taipei/modeling/m7">new part 2</A></LI><LI><A href="/Team:NYMU-Taipei/safety">Safety</A></LI><LI><A href="/Team:NYMU-Taipei/notebook/labnotes">Notebook</A></LI><LI><A href="/Team:NYMU-Taipei/notebook/labnotes">Lab notes</A></LI><LI><A href="/Team:NYMU-Taipei/notebook/protocol">Protocol</A></LI><LI><A href="/Team:NYMU-Taipei/hp/overview">Human Practice</A></LI><LI><A href="/Team:NYMU-Taipei/team/member">Team</A></LI><LI><A href="/Team:NYMU-Taipei/team/member">Members</A></LI><LI><A href="/Team:NYMU-Taipei/team/gallery">Gallery</A></LI><LI><A href="/Team:NYMU-Taipei/team/nymu">NYMU</A></LI><LI><A href="/Team:NYMU-Taipei/team/cooperate">Cooperation</A></LI></DIV><DIV id="toothy"><DIV id="convblank"><DIV id="qnaClose">X</DIV><DIV id="qnaNextBt">Next ⇒</DIV><DIV id="qnaPrevBt">⇐ Prev</DIV></DIV><DIV id="convopen">...</DIV></DIV><DIV id="main-content"><H>Threshold model</H><H1>Purpose</H1><DIV class="abstract mdl"><UL><LI>To determine the relationship between <I>S. mutans</I> population and the amount of CSP, competence stimulating peptide.</LI><LI>To verify if gene expression level under different combination of terminators can be shown in mathematic equations.</LI><LI>To find out the type and the amount of terminators to put behind promoter, so that we can have a proper threshold for lysine-protein-producing gene.</LI></UL></DIV><DIV class="cont-panel mdl-panel"><DIV href="#2c1-1"><P>Introduction</P></DIV><DIV href="#2c1-2"><P>Equations</P></DIV><DIV href="#2c1-3"><P>Result</P></DIV><DIV href="#2c1-4"><P>Reference</P></DIV></DIV><DIV class="article modeling indent"><H1 id="2c1-1">Background</H1><P>As we have shown in <A href="/Team:NYMU-Taipei/modeling/m2"><B>competition model</B></A>, it is better to control the population of S. mutans in a proper amount. To achieve this goal, we design a S. mutans population sensing promoter and a threshold in our circuit, and therefore won’t start the killing module, the lysine-protein-producing gene, until S. mutans exceed the <B>optimal amount we have set</B>.</P><P>
NlmC is a <I>S. mutans</I> population sensing promoter. Studies have shown that nlmC is triggered by a transcription factor which is only produced when receptor binds to CSP, a quorum sensing chemical released by every S. mutans[1]. The more S. mutans there are, the higher the concentration of CSP is in the environment, and therefore, the more likely the coding sequence behind the downstream promoter, nlmC, is to be transcribe[2]. However, the certain and quantified connection between the amount of S. mutans and CSP concentration is not well understood. Therefore, we use modelling to do a regression and find out the relation by using an experimental data[3].</P><P>
Nevertheless, using population sensing promoter only doesn’t meet our needs, for we want to have the killing module transcribed only when the amount of S. mutans exceed a certain level. And since the RBS in M102, the phage we plan to manipulate, is not well studied, changing RBS that have different ribosome binding strength is not suitable for our circuit to create a threshold. And therefore we have to find another way to attain the objective. It is well known that terminators have its own efficacy and is possible to have some leakage[4]. As a result, if we put terminators between promoter and coding sequence, the circuit will not be transcribed until the concentration of transcription factor reach a certain level and raise the terminator leakage probability.</P><P>Using the above-described module, we can change the level of threshold by putting different efficacies and numbers of terminators in our circuit. It is reasonable for use to introduce the rule of probability for predicting the result. <B>The higher the efficacy of terminator is, the less likely the coding sequence is to be transcribed. And the more terminators put behind the promoter, the lower the transcription probability is, as the rule of probability multiplication has shown (eq.)</B>. Here, we use modelling to verify this concept and figure out the best combination of terminators for our circuit.</P><H1 id="2c1-2">Models and mathematic equations</H1><P>If we build some terminator in front of the lysine gene mRNA, the concentration of lysis protein won’t increase easily and thus killing the S. mutans in a short time. Although the threshold of concentration of lysine gene that would kill S. mutans is fixed, we can still regulate the velocity of lysine protein reach this threshold by using different amount of terminator with different efficiency.</P><P>$$\frac{\mathrm{d}[\text{lysine gene mRNA}]}{\mathrm{d}t}=\beta \frac{[\text{comE}]^n}{K_d+[\text{comE}]^n} (1-a)^x - [\text{lysine gene mRNA}] K_m$$
$$\frac{\mathrm{d}[\text{lysine protein}]}{\mathrm{d}t}= K_t [\text{lysine gene mRNA}] - K_p [\text{lysis protein}]$$</P><P>This is the simulation of lysine gene mRNA and the lysine protein. The circuit is activated by nlmC promoter, which is turned on by comE molecule. The term $(1-a)^x$ is how the terminator works, lowering the probability that the promoter can successfully transcript the lysine gene mRNA. Note that in addition to using same terminator, we can try combinations of different terminators. The meaning of parameters is shown as below:
1.  $\beta$:max nlmC promoter activity
2.  $K_d$:promoter-TF dissociation constant
3.  $K_m$:Lysine gene mRNA degradation
4.  $K_t$:Translation efficiency
5.  $K_p$:Lysine protein degradation
6.  $n$:hill coefficient
7.  $a$:terminator efficiency
8.  $x$:amount of terminator

      </P><H1 id="2c1-3">Results and model validation</H1><H3>CSP, OD and beta-galactosidase</H3><P>Here, we discuss the graphical representation of competence stimulating peptide(CSP), OD and beta-galactosidase. The aim is to determine the relationship of the different variables, and to reformulate a suitable models. As we can see on figure 1, a graphical representation between OD and beta-galactosidase has presented. It is noticeable that the relationship, OD vs beta-galactosidase, can be approximated using a simple regression model:
$$\text{OD}=\alpha+\beta\text{beta-galactosidase}+\epsilon$$
and the fitted model, the result can be expressed as:
$$\text{OD}=0.425+0.007\text{beta-galactosidase}$$
</P><P style=" text-align: center;">Figure 1 : Graphical representation of OD vs beta-galactosidase(protein)</P><P>This means that on average for a unit change on beta-galactosidase OD can be increased by 0.007. The two blue line curves show a 95% confidence interval of the fitted line (red).</P><P class="Jaxcenter">$$\text{Table 1: Regression model result of OD vs beta-galactosidase}$$</P><PRE>
==============================================================
Estimated parametrs:    | Statistical Tests:
Intercept (alpha)=0.415 | R-squared= 0.95
OD (beta)=0.007         | p-value=0.00011
--------------------------------------------------------------
F-statistic=116.5
===============================================================
</PRE><P>The P.value &lt; 0.05 of the regression model indicated that there is a significant relationship between OD and beta-galactosidase and the R-square value 0.95 shows that their relationship is strong. In other words, 95% of the variable OD in the model can be explained be variable beta-galactosidase and the remain 5% can be explained with other variables (which are not included in the model).</P><P>As for the model of beta-galactosidase and CSP, which turned out to be a nonlinear one, we tried to fit it with exponential model：</P><P>Therefore, we can estimate the amount of CSP with OD from the two models we derived:</P><P>$$\text{CSP}=3.6837 \exp{(0.0039\times\frac{\text{OD}-0.425}{0.007})}$$</P><P>From the competition model, we know that we can optimize our oral condition via activating our killing module when S. mutans reach 72% the amount of its maximum population. From our growth &amp; pH model, we can know that 72% the amount of S. mutans' maximum population is OD600 at about 0.86. The CSP concentration is then computed from the equation above:
$$3.6837 \exp{(0.0039\times\frac{\text{0.86}-0.425}{0.007})}=4.69 \text{(ng/ml)}$$</P><P>To figure out how CSP regulate nlmC promoter, we again use data from literature and did linear transformation beforehand to fit our model and the theoretical endolysin amount. The shows the fitting result:</P><P>Based on the above described results, we are going to find the number and the efficiency of terminators we plan to put in our control circuit, <B>so that endolysin will reach the toxic concentration as soon as CSP concentration is above 4.69 ng/ml</B>. The toxic concentration of endolysin was obtained from literature[5], which pointed out that endolysin at the concentration of 15 U/ml may start to decrease OD600. Table 2 reveals the effect of different numbers and the efficiencies of terminators.</P><P class="Jaxcenter">$$\text{Table 2: Effect of different threshold terminators on endolysin expession}$$</P><TABLE class="wikitable" style="font-size:16px; margin:0 auto;"><TBODY><TR><TH> Number of terminators </TH><TH> BBa_B0012($a$=0.309) </TH><TH> BBa_B0015($a$=0.984)
</TH></TR><TR><TD> 1 </TD></TR><TR><TD> 2 </TD></TR><TR><TD> 3 </TD></TR></TBODY></TABLE><P>It turns out that among all, using 2 BBa_B0012 terminators will meet our needs better, the circuit can start the killer module when CSP is around 4.69 ng/ml. Other terminator combinations would start the killer module either when CSP is too high or too low. By analytical calculation, <B>we figure out that any combination of terminators that reduces transcription rate to about 0.1225% will be suitable for our control circuit.</B> The optimal condition is shown below:</P><P style=" text-align: center;">Appropriate combination of terminators. When CSP is less than 4.69, the killer module won’t be started.</P><P>The concentration of endolysin will exceed toxic concentration and kill the S. mutans as more CSP is secrete by S. mutans.</P><P style=" text-align: center;">Appropriate combination of terminators. When CSP is more than 4.69, the killer module will be started.</P></DIV><H1 id="2c1-4">Reference</H1><OL><LI>Li, Y.-H., et al., A quorum-sensing signaling system essential for genetic competence in Streptococcus mutans is involved in biofilm formation. Journal of bacteriology, 2002. 184(10): p. 2699-2708.</LI><LI>Liu, T., et al., ComCED signal loop precisely regulates nlmC expression in Streptococcus mutans. Annals of Microbiology, 2014. 64(1): p. 31-38.</LI><LI>van der Ploeg, J.R., Regulation of bacteriocin production in Streptococcus mutans by the quorum-sensing system required for development of genetic competence. Journal of bacteriology, 2005. 187(12): p. 3980-3989.</LI><LI>Chen, Y.-J., et al., Characterization of 582 natural and synthetic terminators and quantification of their design constraints. Nature methods, 2013. 10(7): p. 659-664.</LI><LI>Obeso, J.M., et al., Lytic activity of the recombinant staphylococcal bacteriophage ΦH5 endolysin active against&lt; i&gt; Staphylococcus aureus in milk. International journal of food microbiology, 2008. 128(2): p. 212-218.</LI></OL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NYMU-Taipei/modeling/m3">http://2014.igem.org/Team:NYMU-Taipei/modeling/m3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYMU-Taipei/modeling/m3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYMU-Taipei/modeling/m3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYMU-Taipei/modeling/m3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYMU-Taipei/modeling/m3&amp;oldid=395482" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>