Team:Groningen/Notebook/Overproduce yuab
 
Blue Bio Energy
Home
Team
Project
Results
Modeling
Notebook
Parts
Attributions
Collaborations
Design
Human Practices
Regulations
Media Coverage
Events
Card Game
Future
Safety
Measurement
Supervisors
Menu
Home
Team
Project
Results
Modeling
Notebook
Parts
Attributions
Collaborations
Design
Human Practices
Regulations
Media Coverage
Events
Card Game
Future
Safety
Measurement
Supervisors
Overproduce BslA in B. subtilis ComI
BslA is a protein which forms a hydrophobic layer on the biofilm surface.
To overproduce BslA in B. subtilis ComI.
Everything Worked The First Time.
None
00:00, 16 April 2015 - 00:00, 16 April 2015
Primers.
The following primers were designed for bslA (the gene responsible for the  production of BslA) in B. subtilis 168.
#
Name
Sequence
1
bslA for
GCGAATTCGCGGCCGCTTCTAGAGCATTTTTTAGGGGGAATTTTGTTATG
2
bslA overlap for
GATTTGTTAGGAGCTGGCGAATTTAAATTAAAACTG
3
bslA overlap rev
CAGTTTTAATTTAAATTCGCCAGCTCCTAACAAATC
4
bslA rev
CGTACTAGTAGCGGCCGCTGCAGTTATTAGTTGCAACCGCAAGGCTGAG
Primer design for bslA.
Atze van Stralen
00:00, 13 May 2015 - 00:00, 13 May 2015
PCR 1
Prepare dNTP Mastermix with final concentration of 2 mM of  each dNTP.
Ingredient
Amount
100 mM dATP
20 µL
100 mM dTTP
20 µL
100 mM dCTP
20 µL
100 mM dGTP
20 µL
\( \mathrm{H_2 O} \)
920 µL
dNTP Mastermix.
Harm Ruesink
00:00, 13 May 2015 - 00:00, 13 May 2015
Per sample:
Ingredient
Amount
Mastermix PCR 1
30 µL
template DNA (B. subtilis 168 DNA)
0.3 µL
bslA for
0.3 µL
bslA overlap for
0.3 µL
bslA overlap rev
0.3 µL
bslA rev
0.3 µL
Mastermix PCR 1.
Samples
#
Description
1
yuaB for + yuaB overlap rev
2
yuaB overlap for + yuaB rev
Samples.
PCR thermocycle
#
Step
Temperature
Time
1
Initial denaturation
98 °C
2:00
2
Denaturation
98 °C
0:10
3
Annealing
60 °C
0:20
4
Extension
72 °C
0:30
5
Go back to 2 (repeat 30x)
6
Final extension
72 °C
5:00
Recommended thermocycle.
Gel results
Sample 1 formed no product. Sample 2 formed a product of ±200bp visible on gel.  Sample 2 was purified using PCR and Stored in -20°C as  "1.2".
Harm Ruesink
00:00, 15 May 2015 - 00:00, 15 May 2015
PCR 2
The following samples were prepared using B. subtilis 168  DNA.
Ingredient
Quantity
5x buffer
40 µL
dNTP MM (2mM)
2 µL
Phusion polymerase
20 µL
\( \mathrm{\mathrm{H_2 O}} \)
136 µL
Mastermix PCR 2.
Per sample:
Ingredient
Quantity
Mastermix PCR 2
30 µL
Per template DNA
2 µL
Mix per sample.
A sample marked "4" was prepared with the following primers:  yuaB for + yuaB overlap rev. Template DNA was B. subtilis 168 DNA. The  following  thermocycle was performed.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annealing
62 °C
0:30
4
Extention
72°C
1:00
5
Go back to step 2 (repeat 3x)
6
Denaturation
98 °C
0:30
7
Annealing
60 °C
0:30
8
Extention
72 °C
1:00
9
Go back to step 6 (repeat 3x)
10
Denaturation
98 °C
0:30
11
Annaeling
58°C
0:30
12
Extention
72 °C
1:00
13
Go back to step 10 (repeat 3x)
14
Denaturation
98 °C
0:30
15
Annaaling
56 °C
0:30
16
Extention
72 °C
1:00
17
Go back to step 14 (repeat 3x)
18
Denaturation
98 °C
0:30
19
Annealing
54 °C
0:30
20
Extention
72 °C
1:00
21
Go back to step 18 (repeat 3x)
22
Denaturation
98 °C
0:30
23
Annealing
54 °C
0:30
24
Extention
72 °C
1:00
25
Go back to step 22 (repeat 30x)
26
Final Extention
72 °C
10:00
Recommended thermocycle.
Harm Ruesink
00:00, 18 May 2015 - 00:00, 18 May 2015
PCR 3
Ingredient
Quantity
5x buffer
28 µL
dNTP MM (2mM)
14 µL
Phusion polymerase
1.4 µL
\( \mathrm{H_2 O} \)
95.2 µL
Mastermix PCR 3.
Per sample:
Ingredient
Quantity
Mastermix PCR 3
30 µL
Per template DNA
0.3 µL
Per primer
0.3 µL
Mix per sample.
The following samples were prepared: 1. primers: yuaB for +  yuaB rev, template DNA: B. subtilis 168 DNA  2. primers: yuaB for + yuaB overlap rev, template DNA: B. subtilis 168 DNA
#
Primers
Template DNA
1
yuaB for + yuaB rev
B. subtilis 168 DNA
2
yuaB for + yuaB overlap rev
B. subtilis 168 DNA
#
Step
Temperature
Time
1
Initial denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annealing
56 °C
0:30
4
Extention
72 °C
1:00
5
Go back to step 6(repeat 15x)
6
Denaturation
98 °C
0:30
7
Annealing
70 °C
0:30
8
Extention
72 °C
1:00
9
Go back to step 6 (repeat 15x)
10
Final Extention
72 °C
10:00
Recommended thermocycle.
Harm Ruesink
00:00, 19 May 2015 - 00:00, 19 May 2015
Ran a gel with PCR 3 products on 1% agarose gel with EtBr.  Both sample 1 and sample 2 show no bands visible. Probably because yuaB forward  primer doesn’t work.
Harm Ruesink
00:00, 21 May 2015 - 00:00, 21 May 2015
Ordered new primers.
Primer
Sequence
yuaB forward 2 (with RBS)
GCGAATTCGCGGCCGCTTCTAGAGTTAGGGGGAATTTTGTTATGAAACGC
yuaB forward 3 (no RBS)
GCGAATTCGCGGCCGCTTCTAGAGATGAAACGCAAATTATTATCTTCTTTGG
New primers.
Harm Ruesink
00:00, 8 June 2015 - 00:00, 8 June 2015
PCR 5
PCR'd part 1 of yuaB using primers yuaB forward 2 & yuaB  forward 3 and genomic DNA from B. subtilis 168.
Ingredient
Quantity
10x buffer
10 µL
dNTP MM (2mM)
10 µL
Dreamtaq polymerase
1 µL
\( \mathrm{H_2 O} \)
76.5 µL
PCR 5
The following mix was used for each sample.
Ingredient
Quantity
Mastermix PCR 5
0,3 µL per template DNA
Per primer
0,3 µL
Sample contents.
With this mix the following samples were made:
2. primers: yuaB forward 2 + yuaB overlap rev, template DNA:  B. subtilis 168 DNA
3. primers: yuaB forward 3 + yuaB overlap rev, template DNA:  B. subtilis 168 DNA
The following PCR thermocycle was performed.
#
Step
Temperature
Time
7
Initial Denaturation
98 °C
3:00
8
Denaturation
98 °C
0:30
9
Annealing
60 °C
0:30
10
Extention
72 °C
1:00
11
Go back to step 2 (repeat 30x)
12
Final Extension
72 °C
10:00
Recommended thermocycle.
Loaded PCR product on 1% agarose gel with DNA stain G (1:50000) and ran for 1  hour on 100V. Also loaded sample 1.2  (yuaB part 2 of length 209 bp).  The following samples were prepared.
30 µL DNA(tasA)
6 µL 6x buffer
10µL of Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb) was used.
Sample
Result
2
bands visible of ~450 bp
3
bands visible of ~450 bp
1.2
bands visible of ~200 bp
PCR results
Bands were cut from the gel and purified using a PCR purification kit.
Sample
Gel Weight
Binding buffer
2 (1.2)
253 mg
506 µL
3 (1.3)
193 mg
386 µL
1.2 (2)
270 mg
540 µL
Gels.
Samples were stored as
1.2 (49,9 ng/µL Nanodrop value)
1.3 (56,5 ng/µL Nanodrop value)
2 (19,2 ng/µL Nanodrop value)
Harm Ruesink
00:00, 9 June 2015 - 00:00, 9 June 2015
Assembly PCR
Part 1 (450 bp) and part 2 (209 bp) were combined to get the  whole bslA gene.
Ingredient
Quantity
10x buffer
10 µL
dNTP MM (2 mM)
10 µL
Dreamtaq polymerase
1 µL
\( \mathrm{H_2 O}\)
78 µL
PCR Mastermix.
Per sample the following mix was made.
30 µL of mastermix
1 µL per template DNA
0.3 µL per primer
Using this mix two samples were prepared.
Primers
Template DNA
yuab forward 2 + yuaB rev
1.3 + 2
yuaB forward 3 + yuaB rev
1.3 + 2
Samples.
#
Step
Temperature
Time
1
Denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annaeling
60 °C
0:30
4
Extention
72 °C
1:00
5
Go back to step 2 (repeat 30x)
6
Final Extension
72 °C
10:00
Recommended thermocycle.
PCR product was loaded on 1% agarose gel with DNA stain G  (1:50000) and ran for one hour at 100 V. The following samples were  analyzed.
30 µL of DNA (tasA)
6 µL of 6x buffer
10 µL of Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb) was used. Sample 1  showed bands at ~ 650 bp. Sample 2 showed bands at ~650 bp. Band were cut from  the gel and purified using a PCR purification kit.
Sample
Gel Weight
Binding Buffer
1
176 mg
352 µL
2
248 mg
496 µL
Purification.
This was stored (20 µL of elution buffer) as follows.
bslA + RBS, 9-6-15
bsLA - RBS, 96-15
Harm Ruesink
00:00, 10 June 2015 - 00:00, 10 June 2015
Digestion
yuaB was digested using the following mix.
Ingredient
Quantity
DNA (yuaB + RBS)
2 µL
10x buffer (2.1)
2 µL
EcoRI
1 µL
Pstl
1 µL
\( \mathrm{H_2 O} \)
14 µL
Digestion mix.
The mix was incubated for two hours at 37 °C. A 1% agarose + DNA strain G  1:50000 was ran at 120 V.
Ingredient
Quantity
DNA
20 µL
6x buffer
4 µL
Sample.
10 µL of Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb) was used. A band  was visible around 600 bp. This means that the restriction site within the yuaB  gene is gone.
DNA was extracted from the gel using a PCR purification kit.
Sample
Gel Weight
Binding Buffer
yuaB
225 mg
450 µL
Purification.
Eluted in 10 µL elution buffer and stored as "yuaB dig".
Ligation of yuaB in BBa_K823023 was performed using the following ligation  mix.
Ingredient
Quantity
insert DNA (yuaB dig)
10 µL
vector DNA (BBa_K823023)
5 µL
10x buffer
2 µL
T4 ligase
0.5 µL
\(\mathrm{H_2 O}\)
2.5 µL
Ligation mix
This was ligated for 2 hours at room temperature. The DNA was  then transformed into competent E. coli using heat shock transformation. 2  hours  later it was plated on LB ampicilin plates and left in 37 °C overnight.
Harm Ruesink
16:00, 11 June 2015 - 16:30, 11 June 2015
Two white colonies and a lot of red colonies visible on  plate.  Inoculated 2 white colonies and one red colony in liquid LB with ampicilin  overnight.
Harm Ruesink
00:00, 12 June 2015 - 00:00, 12 June 2015
Miniprepped overnight cultures. Stored as
23 yuaB 1
23 yuab 2
23 RFP
Harm Ruesink
00:00, 15 June 2015 - 00:00, 15 June 2015
Digested yuaB from backbone to check ligation using EcoRI and  Pstl. The following mastermix was prepared for 13 samples.
Ingredient
Quantity
10x buffer (2.1)
30 µL
EcorI
10 µL
Pslt
10 µL
\(\mathrm{H_2 O}\)
185 µL
Digestion mastermix.
Per sample the following mix was prepared.
Ingredient
Quantity
DNA
15 µL
Mastermix
15 µL
Sample contents.
The samples were incubated at 37 °C for two hours. They were  then loaded on 1% agarose gel with 1:50000 DNA stain G. To 20 µL of digestion  sample 4 µL of 6x buffer was added. 10µL of Quick-Load® Purple 2-Log DNA Ladder  (0.1 - 10.0 kb) was used for each sample.
Nothing was seen on the gel. The gel may have been too hot when adding the DNA  stain G.
Harm Ruesink
00:00, 16 June 2015 - 00:00, 16 June 2015
Repeated digest of tasA from 15-6-15. Digest for 1.5h at 37  °C.
10 µL of digestion sample was added to 2 L of 6x buffer. This  was run at 100 V for 1 hour with 10 µL of 2log ladder.
No insert was seen on gel. yuaB probably wasn't ligated in the backbone.
Harm Ruesink
00:00, 8 July 2015 - 00:00, 8 July 2015
Ligation of yuaB into pSB1C3
yuaB + RBS with pSB1C3 was digested using EcoRI and Pstl.
Ingredient
Quantity
10x buffer (2.1)
6.6 µL
EcoRI
3.3 µL
Pstl
3.3 µL
\(\mathrm{H_2 O}\)
23.4 µL
Ligation mastermix.
Sample
Ingredient
Quantity
1
yuaB + RBS
10 µL
Mastermix
10 µL
2
pSB1C3
10 µL
Mastermix
10 µL
Samples.
This was digested for 1 h at 37 °C. The DNA was then purified  using a PCR purification kit. yuaB was then ligated in pSB1C3.
Sample
Ingredient
Quantity
2
bslA DNA
10 µL
pSB1C3 DNA
5 µL
T4 ligase
0.5 µL
T4 ligase buffer
2 µL
\(\mathrm{H_2 O}\)
2.5 µL
Sample.
The sample was ligated at room temperature for two hours. The  ligation product was then transformed into 100 µL of NED competent cells using  heat shock transformation. Two plates were made.
90% plate (90 µL)
10% plate (10 µL)
Plates were incubated overnight at 37 °C.
Harm Ruesink
00:00, 9 Juli 2015 - 00:00, 9 Juli 2015
Transformation results
Transformation was a success, both plates show a large number  of colonies. Four colonies from the 10% plate were grown overnight in 3 mL LB  with 6 µL of 1/500 chloramphenicol
Harm Ruesink
00:00, 16 July 2015 - 00:00, 16 July 2015
Ligation of a salt promotor and yuaB into BBa_K823023 was  performed.
Sample
DNA
Restriction enzyme
1
Psalt
EcoRI + Spel
2
bslA (y4, 10-7-15)
Xbal + PstI
3
BBa_K823023
EcoRI + PstI
Digest.
Sample
Ingredient
Quantity
1
DNA (Psalt)
6 µL
EcoRI
1 µL
Spel
1 µL
10x buffer (2.1)
2 µL
\(\mathrm{H_2 O}\)
10 µL
2
DNA (Y4)
3 µL
Xbal
1 µL
PstI
1 µL
10x buffer (2.1)
2 µL
\(\mathrm{H_2 O}\)
13 µL
2
DNA (K823023)
3 µL
EcoRI
1 µL
PstI
1 µL
10x buffer (2.1)
2 µL
\(\mathrm{H_2 O}\)
10 µL
Samples.
Samples were incubated for one hour at 37 °C. They were then  purified using a PCR purification kit (20 µL elution). Samples were stored as  "1 & 2" with "16-7-15" on the side.
Samples 1 + 2 + 3 were ligated.
Ingredient
Quantity
DNA 1
5 µL
DNA 2
5 µL
T4 Ligase
0.5 µL
10x buffer
2 µL
\(\mathrm{H_2 O}\)
5 µL
Ligation mix.
Ligated at room temperature for 30 minutes. Transformed into  10 µL of NEB competent cells with 10 µL of ligation product using heat shock  transformation. The result was plated on LB ampicilin plates and left at 37 °C  overnight.
Harm Ruesink
00:00, 17 July 2015 - 00:00, 17 July 2015
Transformation results
A few white colonies were seen on the plate. A colony PCR  was  performed to check the size of the insert (which is supposed to be bslA +  Psalt). The colony PCR showed possible bslA + Psalt insert in one sample.  Colony  was grown overnight in LB + ampicilin.
Harm Ruesink
00:00, 18 August 2015 - 00:00, 18 August 2015
Miniprep was performed and stored as "22 (17-7-15)".
Harm Ruesink
00:00, 20 July 2015 - 00:00, 20 July 2015
Send in sample 22 for sequencing with prefix (19) and suffix  (20) primers.
Send sample 22 to Macrogen for sequencing.
Ingredient
Quantity
1/20 primer (5 pM/µL)
5 µL
plasmid DNA (22)
5 µL
Samples.
Forward primer (prefix primer): 17C4ZAD516
Reverse primer (suffix primer): 17C4ZAD517
Harm Ruesink
00:00, 21 July 2015 - 00:00, 21 July 2015
Transformed sample 22 into B. subtilis.  Took B.  subtilis 3610 ComI colony from plate (13-7-15 in fridge) and put it in MC 1x  for  5 hours. Optical density at 600 nm was measured to be 0.15. Then 5 µL of  plasmid  DNA (22) was added to 400 µL of cell culture and left in 37 °C  shaking incubator for a further two hours. The culture was then plated on LB  chloramphenicol plates and left overnight at 37 °C.
Harm Ruesink
00:00, 22 July 2015 - 00:00, 22 July 2015
Sequencing results
Results don't show correct insert. Insert is an RFP.  Discarded  transformed Bacillus subtilis due to incorrect insert.
Harm Ruesink
00:00, 23 July 2015 - 00:00, 23 July 2015
Psalt was digsted at for one huur at 37 °C using the  following mix.
Ingredient
Quantity
DNA (Psalt)
10 µL
EcoRI
1 µL
Spel
1 µL
10x buffer (2.1)
2 µL
\(\mathrm{H_2 O}\)
6 µL
Digestion mix
The digestion sample was purified using a PCR purification  kit. Eluted with 20 µL. It was then ligated for two hours at room temperature  using the following mix.
Ingredient
Quantity
DNA 1 (Psalt)
5 µL
DNA 2 (bslA)
5 µL
DNA 3 (K23)
5 µL
T4 Ligase
1 µL
10x buffer
2 µL
\(\mathrm{H_2 O}\)
2 µL
Ligation mix.
A heat shock transformation was then performed. The result  was  plated on LB ampicilin and incubated overnight at 37 °C.
Harm Ruesink
00:00, 24 July 2015 - 00:00, 24 July 2015
Transformation Results
No white colonies were visible on the plate.
Harm Ruesink
00:00, 28 July 2015 - 00:00, 28 July 2015
Adding promoters to Y4 (10-7-15).
Pveg (iGEM distribution kit 2015, plate 1, 20 G)
For the promoters the restriction enzymes EcoRI & Spel were used. For the backbone Xbal & EcoRI were used.
Sample
Ingredient
Quantity
1
Y4 DNA
5 µL
10x buffer
2 µL
EcoRI
1 µL
Xbal
1 µL
\(\mathrm{H_2 O}\)
11 µL
3
Pveg DNA
10 µL
10x buffer
2 µL
EcoRI
1 µL
Spel
1 µL
\(\mathrm{H_2 O}\)
11 µL
Digestion samples
These samples were digested at 37 °C for two hours.  They were then on a gel at 100 V for 45 minutes. 1% agarose with DNA stain G  1:50000 was used. For each sample, 20 µL of DNA was mixed with 4 µL of 6x  buffer. 10 µL of Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb) was used.
Sample
Weight
Binding buffer
1
156 mg
316 µL
3
223 mg
446 µL
4
300 mg
600 µL
Gel cuts.
The DNA was purified using a PCR purification kit (Genejet #K0701, lot  00265976). 25 µL elution buffer was used. The concentrations were checked using  Nanodrop.
Sample
Concentration
Y4 dig (1)
2 ng/µL
Pveg (3)
1.4 ng/µL
Psalt (4)
5 ng/µL
Sample concentrations.
Ligation of promoters in backbones
Ingredient
Quantity
10x buffer
12 µL
T4 Ligase
4 µL
\(\mathrm{H_2 O}\)
8 µL
Ligation mastermix.
Sample
Ingredient
Quantity
1
Y4 dig (1)
12 µL
Pveg (3)
12 µL
Mastermix
6 µL
2
Y4 dig (1)
12 µL
Psalt (4)
12 µL
Mastermix
6 µL
Ligation samples.
Samples were ligated overnight.
Harm Ruesink
10:00, 29 July 2015 - 12:00, 29 July 2015
Heat shock transformation was performed. Samples were plated  on LB chloramphenicol and incubated overnight at 37 °C.
Harm Ruesink
00:00, 30 July 2015 - 00:00, 30 July 2015
Transformation results
No colonies visible on LB chloramphenicol plate.
Y4 was send to Macrogen for sequencing.
5 µL of 1/20 primer (5 pmole/µL)
5 µL of plasmid DNA (Y4)
Sequencing codes Macrogen
Forward primer (prefix (19)): 17C4ZAD660
Reverse primer (suffix (20)): 17C4ZAD661
Harm Ruesink
00:00, 6 August 2015 - 00:00, 6 August 2015
Digestion and ligation of Pliag in front of Y4 was performed.  For the promoters EcoRI and Spel were used. For Y4 Xbal and EcoRI were  used. These enzymes leave a scar.
Sample
Ingredient
Quantity
1
Pliag DNA
5 µL
10x buffer
2 µL
EcoRI
1 µL
Spel
1 µL
\(\mathrm{H_2 O}\) (MQ)
11 µL
2
Y4 DNA
5 µL
10x buffer
2 µL
Xbal
1 µL
EcoRI
1 µL
\(\mathrm{H_2 O}\) (MQ)
11 µL
Samples.
Samples were digested for 2 hours at 37 °C. Samples were then loaded on 1% agarose gel with DNA stain G 1:50000
Ingredient
Quantity
sample DNA
20 µL
6x buffer
4 µL
Samples.
10 µL of NED 2log ladder. Gel was run on 100 V for 30 minutes. It was then cut.
Sample
Weight
Binding Buffer
1
0.300 g
600 µL
2
0.118 g
236 µL
Gel cuts.
The samples were purified using a Thermo Scientific GeneJET PCR Purification Kit. Concentrations were checked using Nanodrop.
Sample
Concentration
Pliag dig
4,6 ng/µL
Y4 dig
4,7ng/µL
Sample concentrations.
Samples were ligated overnight.
Ingredient
Quantity
Pliag DNA
5 µL
Y4 DNA
10 µL
10x T4 buffer
2 µL
T4 Ligase
0.5 µL
\(\mathrm{H_2 O}\)
0.5 µL
Ligation mix.
Harm Ruesink
00:00, 7 August 2015 - 00:00, 7 August 2015
A heat shock transformation was performed. 20 µL of (Y4 + Pliag) DNA was added to 10 µL of competent NEB cells. This was plated on LB chloramphenicol plates and incubated overnight at 37 °C.
Harm Ruesink
00:00, 8 August 2015 - 00:00, 8 August 2015
Transformation results
No colonies were visible.
Harm Ruesink
00:00, 9 August 2015 - 00:00, 9 August 2015
Transformation results
White colonies were visible. A colony PCR was performed. PCR showed possible insert in sample 9 and 13. Cultures were grown overnight to check for the insert. The overnight cultures were grown in LB + chloramphenicol (3mL) at  37 °C, 200rpm.
Harm Ruesink
00:00, 11 August 2015 - 00:00, 11 August 2015
Cultures 9 and 13 were miniprepped and loaded on 1% agarose gel with DNA stain G 1:50000.
Ingredient
Quantity
Sample DNA
1 µL
\(\mathrm{H_2 O}\)
4 µL
6x buffer
1 µL
Mix.
10µL of Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb) was used. The gel was run at 100 V for 30 minutes. Nothing was visible on the gel. DNA concentration should be checked.
Harm Ruesink
00:00, 13 August 2015 - 00:00, 13 August 2015
Concentrations were checked using nanodrop.
Sample
Ingredient
Quantity
9
Sample DNA
4 µL
6x Buffer
1 µL
\(\mathrm{H_2 O}\)
1 µL
13
Sample DNA
5 µL
6x Buffer
1 µL
Y4
Sample DNA
1 µL
6x Buffer
1 µL
\(\mathrm{H_2 O}\)
4 µL
Gel Samples.
Gel was run at 100 V for 30 minutes. The gel showed no bands.
Harm Ruesink
2015 iGEM Groningen
