<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container default screen p1080 details black" id="697ba9800bfd490ea793813ca64040cd"><DIV class="entry header"><DIV class="logo"> </DIV><DIV class="title">Blue Bio Energy</DIV><DIV class="menu"><DIV class="item active"><A class="title" href="https://2015.igem.org/Team:Groningen">Home</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Team">Team</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Description">Project</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Results">Results</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Modeling">Modeling</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Notebook">Notebook</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Parts">Parts</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Attributions">Attributions</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Collaborations">Collaborations</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Design">Design</A></DIV><DIV class="wrapper"><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Human_Practices">Human Practices</A></DIV><DIV class="group"><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Regulations">Regulations</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Media_Coverage">Media Coverage</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Events">Events</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Card_Game">Card Game</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Future">Future</A></DIV></DIV></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Safety">Safety</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Measurement">Measurement</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Supervisors">Supervisors</A></DIV></DIV><DIV class="menu secondary"><DIV class="button"><DIV class="title">Menu</DIV></DIV><DIV class="group"><DIV class="item active"><A class="title" href="https://2015.igem.org/Team:Groningen">Home</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Team">Team</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Description">Project</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Results">Results</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Modeling">Modeling</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Notebook">Notebook</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Parts">Parts</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Attributions">Attributions</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Collaborations">Collaborations</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Design">Design</A></DIV><DIV class="wrapper"><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Human_Practices">Human Practices</A></DIV><DIV class="group"><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Regulations">Regulations</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Media_Coverage">Media Coverage</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Events">Events</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Card_Game">Card Game</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Future">Future</A></DIV></DIV></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Safety">Safety</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Measurement">Measurement</A></DIV><DIV class="item"><A class="title" href="https://2015.igem.org/Team:Groningen/Supervisors">Supervisors</A></DIV></DIV></DIV></DIV><DIV class="entry main"><DIV class="title">Description</DIV><DIV class="text">

The world is changing and so are the energy needs of humanity. Fossil resources are being depleted and it is clear that the transition to clean sustainable energy has to be made. We believe synthetic biology can be an important catalyst in this process. Specifically, we engineer a <I>Bacillus subtilis</I> biofilm to function as a cation exchange membrane. Such a membrane can be used in Reverse Electrodialysis (RED), a technique to generate energy where salt and fresh water mix, for example where rivers flow into the sea. We call this application of the bacterial biofilm Blue Bio Energy. To make the <I>Bacillus subtilis</I> suitable for RED, we first make it more robust by overexpressing the biofilm genes <I>tasA</I> and <I>bslA</I> while preventing reversion to the motile state by knocking out the <I>abrB</I> regulator and overexpressing <I>slrR</I>, another biofilm regulator. Simultaneously, the amino acid polymer gamma-polyglutamic acid was modelled extensively to see if it could be used to make the biofilm ion selective. This was found to be the case. Experiments showed that <I>Bacillus subtilis 3610 comI</I> biofilms are slightly ion selective and, surprisingly, that ion selectivity is improved by knocking out <I>abrB</I> as well as by overexpression of <I>bslA</I> and <I>slrR</I>.

</DIV><DIV class="module content"><DIV class="subtitle">
Growing biofilms
</DIV><DIV class="text">
The word biofilm is used to denote a pleidae of different extracellular structures created by bacteria. The willingness of bacteria to stick together on a surface using secreted extracellular materials is usually considered the defining property of a biofilm, though it is hard to define a clear boundary between closely packed colonies and a low density biofilm. Biofilms are usually grouped according to the type of surface they inhabit, depicted in Table 1.</DIV><DIV class="object data" id="tbl1"><DIV class="wrapper"><DIV class="header"><DIV class="field fw3">Surface</DIV><DIV class="field fw3">Typical biofilm</DIV></DIV><DIV class="record"><DIV class="field fw3">Solid-liquid</DIV><DIV class="field fw3">Thin and uniform</DIV></DIV><DIV class="record"><DIV class="field fw3">Solid-air</DIV><DIV class="field fw7">Thicker and wrinkly, heterogeneous</DIV></DIV><DIV class="record"><DIV class="field fw3">Liquid-air</DIV><DIV class="field fw3">Floating pellicles</DIV></DIV></DIV><DIV class="caption">The effect of the surface on the biofilm phenotype</DIV></DIV><DIV class="text">
Since (by definition) liquid-air pellicles cannot be grown on a solid strength-providing carrier material, they are uninteresting for our project. The solid-air type is the most well-known and can be grown on a simple agar plate. This is convenient, but our setup is nothing like an agar plate, and it is not clear that a biofilm grown on a solid-air interface can cope with the water flow in our setup. The solid-liquid type has the advantage of living at the right kind of surface, but is difficult to grow. In our project we tried growing both the biofilms on the solid-air and on the solid-liquid interface. We found that the solid-air type biofilms were strongest and therefore most suitable for our application.
</DIV><DIV class="subtitle">
Different states of <I>Bacillus subtilis</I></DIV><DIV class="text">
The <I>Bacillus subtilis</I> that is used in our project is able to form a biofilm naturally. However not every <I>Bacillus subtilis</I> with the same genotype is in the biofilm state. One <I>Bacillus subtilis</I> strain knows different behaviours or states. For example, a bacterium can swim around freely, which is called the planktonic state. But <I>B. subtilis</I> can also form spores, or work together in a biofilm community. This biofilm state aids in the survival of the bacteria under harsh conditions. 
A biofilm originates from one or more bacteria that adhere to a surface. By growing in aggregates and producing an extracellular matrix of polymeric substances, a complex community of microbiological life is established. In the biofilm every bacterium has its own role in maintaining the community. Although these bacteria have the same DNA, they express different genes. This phenotypic heterogeneity is the result of certain genetic pathways being activated. When a biofilm matures, some cells start to express different genes and divert into a planktonic state. In this planktonic state the bacterium becomes motile and starts to disperse into the environment. From this planktonic state a new biofilm can be formed by only a few bacteria. The bacteria once again start to change their gene expression and start to express matrix genes to form a biofilm (Vlamakis et al., 2013). 
</DIV><DIV class="subtitle">
Engineering the biofilm state
</DIV><DIV class="text">

Our project requires a stable, robust biofilm. To accomplish this we need to fix the cells in the biofilm (matrix) producing phenotype. The phenotype of <I>Bacillus subtilis</I> is determined by the activity of different genetic pathways. A network of regulators, activators and repressors regulate the expression of genes resulting in a certain phenotype. The expression of biofilm genes is also under control of this complex network. We decided to adopt both a top-down approach (where we modify the genetic pathways controlling expression of biofilm genes and repression of motility genes), and a bottom-up approach (where we overexpress biofilm genes directly). Our biofilm must not only be stable, it must also be ion-selective. Specifically, we wanted to engineer the biofilm to function as a cation exchange membrane. However, selectivity can only be achieved in a reasonably robust biofilm.</DIV><DIV class="subtitle">
Bottom up approach for robustness
</DIV><DIV class="text">
The most obvious way to increase the robustness of the biofilm is the direct overexpression of biofilm genes. This bottom-up approach aims to have continuous expression of biofilm genes as a result, maintaining the bacteria in a biofilm state even if they would otherwise revert to a motile state. Gene candidates are the <I>tasA</I> gene, which results in the formation of TasA amyloid fibres and provides structural integrity to the biofilm (Romero et al, 2010), and the <I>bslA</I> gene, which encodes hydrophobic proteins which form a layer on the surface of a biofilm. This hydrophobic layer of BslA is a major contributor to surface repellency of the biofilm (Kobayashi et al, 2012).  
</DIV><DIV class="subtitle">
Structural fibres (TasA)
</DIV><DIV class="text">
TasA is the main protein component of the <I>Bacillus subtilis</I> biofilm. It forms amyloid fibres which provide structural integrity to the biofilm by binding together the cells in the biofilm (Romero et al, 2009). It is also known that biofilms lacking the TasA amyloid fibres are very fragile (Romero et al, 2010). TasA is part of the <I>tapA-sipW-tasA</I> operon (Kearns et al, 2005). In this operon the <I>sipW</I> gene is responsible for the secretion of the TasA and TapA proteins (Branda et al, 2006). TapA is required for the polymerisation of TasA proteins into amyloid fibres. However TapA is only a minor component of the TasA amyloid fibres (Romero et al, 2011;2014). Since TasA forms the bulk of the amyloid fibres we decided to overexpress the tasA gene specifically. The genes for the TasA fibres were isolated from the B. subtilis 168 genome and were submitted to the parts registry as the K1597002 biobrick. However, early overexpression of <I>tasA</I> might be toxic for the cell because <I>tasA</I> is a prion and also has high metabolic burden on the cell, so a constitutive promoter was not considered an option. To bypass the toxicity of <I>tasA</I>, the production of TasA proteins should be postponed to a later stage in the life of the cell. This is why the choice for an inducible promoter was made. Specifically a salt inducible promoter as seen in figure 1 a &amp; b, which is explained in more detail further on. 
</DIV><DIV class="subtitle">
Hydrophobic coating (BslA)
</DIV><DIV class="text">
BslA is another prominent biofilm protein. It forms a hydrophobic coating on the outer layer of the biofilm. Bacillus subtilis mutants lacking BslA lose the ability to repel water (Kobayashi et al, 2012). Since our biofilm should be robust under water, a continuous expression of this hydrophobic coating gene <I>bslA</I> seems desirable. The <I>bslA</I> hydrophobicity gene was also isolated from the <I>Bacillus subtilis</I> 168 strain and submitted to the parts registry as  biobrick K1597003. Just like <I>tasA</I>, early expression of this gene in the cell life could be toxic. Therefore, a salt promoter was added to regulate gene expression in the same manner as for <I>tasA</I> as seen in figure 1 c &amp; d.
</DIV><DIV class="subtitle">
Salt promoter
</DIV><DIV class="text">
The expression of the <I>tasA</I> and <I>bslA</I> genes should be under control of an inducible promoter since early expression of those genes could result in toxicity for the cells. So to control the expression a salt inducible promoter K1597000 is created, which is already present in the <I>B. Subtilis</I> 168 genome and is is called P<SUB>proH</SUB>. The promoter was isolated and engineered in front of our <I>tasA</I> fibre genes and <I>bslA</I> hydrophobicity genes. This gives us the ability to control the expression of the biofilm genes <I>tasA</I> and <I>bslA</I> by placing the <I>B. subtilis</I> in a salty environment or not. Since our <I>Bacillus subtilis</I> will come in contact with seawater, this provides an elegant way to regulate the expression of the TasA amyloid fibres and BslA hydrophobic coating.</DIV><DIV class="object figure small" id="noid"><DIV class="caption">a) The construct with salt inducible promoter P<SUB>proH</SUB> and fibre forming tasA. b) The construct with salt inducible promoter P<SUB>proH</SUB> and fibre forming tasA. c ) The construct with salt inducible promoter P<SUB>proH</SUB> and hydrophobic coating forming bslA. d) The construct with salt inducible promoter P<SUB>proH</SUB> and hydrophobic coating forming bslA.</DIV></DIV><DIV class="subtitle">
Combining structural fibres (<I>tasA</I>) and hydrophobic coating (<I>bslA</I>) 
</DIV><DIV class="text">
After the creation of biobrick K1597002 for the TasA amyloid fibres and the biobrick K1597003 for the <I>bslA</I> hydrophobic coating, we placed the biobricks behind each other. In front of this we placed our P<SUB>proH</SUB> salt inducible promoter called K1597000. This insert was placed in the BBa_K823023 integration vector from Munich 2012, which uses the <I>amyE</I> locus and replicates in <I>E. coli</I>. This construct was transformed in the <I>Bacillus subtilis</I> strain we selected, which was the <I>Bacillus subtilis 3610 ::comI</I> strain. See figure 2 a &amp; b</DIV><DIV class="object figure small" id="noid"><DIV class="caption">a) The construct with salt inducible promoter P<SUB>proH</SUB>, fibre forming <I>tasA</I> and hydrophobic coating forming <I>bslA</I>. b) The construct with salt inducible promoter P<SUB>proH</SUB>, fibre forming tasA and hydrophobic coating forming <I>bslA</I>. c) The construct desgine with salt inducible promoter P<SUB>proH</SUB> and negative charge forming PGA. d) The construct design with salt inducible promoter P<SUB>proH</SUB> and negative charge forming PGA.</DIV></DIV><DIV class="subtitle">
Top down approach for robustness
</DIV><DIV class="text">
Apart from the bottom up approach, where individual biofilm genes were overexpressed, a top down method was also done. In this method the genetic pathways responsible for the behavioural decision making of <I>Bacillus subtilis</I> get engineered in such a way that biofilm behaviour is constantly on. The reason the biofilm formation is tightly regulated, is because the production of a biofilm requires high investment energetically. Phosphorylation of the transcription factor Spo0A is central to biofilm formation (Branda et al, 2001). The Spo0A transcription factor itself controls two parallel pathways which are responsible for expression of biofilm operons. It also has other functions like repressing cell motility which are beneficial for biofilm formation. However, direct engineering of the Spo0A transcription factor would be difficult, since it controls genetic pathways in a very subtle manner. Very specific amounts of phosphorylated Spo0A have to be present to regulate biofilm formation in a beneficial way (Fujita et al, 2005). The two pathways Spo0A regulates are repressors of biofilm formation. The <I>abrB</I> gene is controlled by Spo0A and directly represses biofilm gene expression. The second pathway is the <I>sinR</I> pathway, which represses biofilm gene expression (Vlamakis et al, 2013). Our goal was to disrupt the activity of those two biofilm repressing pathways. 
</DIV><DIV class="subtitle">
Pathway 1 (<I>abrB</I>)
</DIV><DIV class="text"><I>abrB</I> works by binding directly to the promoter regions of operons responsible for biofilm formation (Hamom et al, 2004). This results in the repression of biofilm gene expression as seen in figure 3a. Under natural circumstances derepression of biofilm operons gets initiated by the alleviation of the repressive effect of abrB by abbA. However instead of making an anti-repressor abbA, an abrB knockout could also be made to have similar effects.  In literature it was also shown that a knock-out of the abrB gene resulted in a 3.3 fold increase in volume of biofilms formed (Bridier et al, 2011). So our plan was to knock out the abrB gene to prevent it from repressing the expression of biofilm operons. Our approach was done by making a knock-out with an antibiotic resistance in the abrB sequence to disrupt it. Since repression of biofilm operons by abrB is now completely gone, this would result in constant biofilm gene transcription through this pathway as seen in figure 3b. However there were two pathways repressing biofilm genes (Vlamakis et al, 2013), so the second pathway also should be engineered.
</DIV><DIV class="subtitle">
Pathway 2 (sinR)
</DIV><DIV class="text">
The second pathway also works by repression of the biofilm operons, specifically the gene sinR acts as the repressor of biofilm operon transcription. It operates in a similar way as the abrB gene. It would be obvious to make a knock-out of this gene too, to inhibit the repression of the biofilm operons in the same way as the abrB. However the presence of sinR is needed to aggregate with another protein called slrR to perform other regulatory functions. In our case the inhibition of cell motility genes is regulated by this aggregation. So knocking out the sinR would also result in the impossibility to repress cell motility, which is required for good biofilm formation. So instead of making a sinR knock-out, the sinR antirepressor slrR was overexpressed constitutively. This was done by making the biobrick BBa_K1597004, which consists of the slrR gene with strong promoter BBa_K090504 and a strong RBS BBa_K780001 in front of it as seen in figure 3c.
</DIV><DIV class="object figure small" id="noid"><DIV class="caption">a) Biofilm operons <I>eps</I>, <I>bslA</I> and <I>tapA-sipW-tasA</I> get repressed by <I>abrB</I> and Spo0A.
b) Biofilm operons <I>eps</I>, <I>bslA</I> and <I>tapA-sipW-tasA</I> get derepressed, by <I>abrB</I> repressor knock out.
c) Biofilm operons <I>eps</I>, <I>bslA</I> and <I>tapA-sipW-tasA</I> get derepressed as <I>sinR</I>  is repressed by <I>slrR</I>.
d) Biofilm operons <I>eps</I>, <I>bslA</I> and <I>tapA-sipW-tasA</I> get derepressed by <I>abrB</I> knock out and by repressing <I>sinR</I> with <I>slrR</I>.</DIV></DIV><DIV class="subtitle">
thrC locus and our integration vector
</DIV><DIV class="text">
Since the tasA and bslA genes were already in the amyE locus, another site for our biobrick with the slrR gene had to be found. abrB doesn’t need an integration vector because it is a knockout of a gene which is in the genome of the B. subtilis.We selected the thrC locus for the overexpression of the slrR gene. The BBa_K823023 integration vector from Munich 2012 was modified by replacing their amyE locus with a thrC locus from pdg1664. Also the chloramphenicol resistance gene was replaced by an erythromycin resistance gene which also came from pdg1664. The integration vector we created is called BBa_K1597001. Via this integration vector we were able to put the slrR biobrick BBa_K1597004 in the thrC locus.
</DIV><DIV class="subtitle">
Combining pathway 1 and 2
</DIV><DIV class="text">
So now the repression of the biofilm operons of pathway 1 has been deactivated by knocking out the <I>abrB</I> gene. Also the repression of biofilm operons through pathway 2 has been deactivated by overexpressing the repressor of the biofilm operon repressing pathway 2 as seen in figure 3d. 
</DIV><DIV class="subtitle">
Combining top down approach and bottom up approach
</DIV><DIV class="text">
So via the bottom up approach specific biofilm genes were overexpressed by a salt inducible promoter. This bottom up construct was placed in the amyE locus via the K823023 backbone from Munich 2012.  This was combined with the top down approach where genetic pathways resulting in biofilm behaviour were derepressed. The top down approach construct was placed in the thrC locus via the K1597001 backbone we created. See figure 4b. This combination resulted in B.subtilis specifically forming a biofilm and the biofilm formed proved to be robust. However to generate energy, the biofilm had to be more ion selective. 
</DIV><DIV class="subtitle">
Selectivity
</DIV><DIV class="text">
So after a robust biofilm was engineered via the top down and bottom up approach, secreted proteins should be engineered to make the biofilm layer ion selective. Specifically cation selective in our case. In literature we found the biofilm matrix could function as a molecular sieve, sequestering away cations, anions and apolar materials from the water (Flemming, H.-C. &amp; Leis,  in Encyclopedia of Environmental Microbiology (ed. Bitton, G.) 2958–2967 (Wiley, New York, 2002).). It is shown that cations from the water can form ionic bonds with negative groups in the biofilm. So we decided to engineer the secretion of negative molecules into the biofilm. We selected the highly negative poly-γ-glutamic acid (PGA), which is already present in the Bacillus subtilis natto strain. To prove that PGA can be cation selective in a layer, its interactions in water with positive and negative ions were modelled  with Molecular Dynamics. The pgsBCA genes from were designed to be behind salt inducible promoter K1597000 we created. Since the amyE locus already had a construct in it, this would fit in the thrC with the slrR by using our integration vector. The  pgsBCA genes, which form PGA, are secreted into the biofilm like in figure 2 c &amp; d.
</DIV><DIV class="subtitle">
Total construct plan
</DIV><DIV class="text">
To combine the genes which enhance stability and robustness ( tasA, bslA, ΔabrB, slrR) with the selectivity enhancing operon pgsBCA the following construct was designed as seen in figure 4c. 

Constitutive overexpression of slrR and inducible overexpression of pgsBCA in the thrC locus by using the K1597001 backbone.
Induced overexpression of bslA and tasA in the amyE locus by using the K823023 backbone.
Knocking out the abrB in the Bacillus subtilis genome.
</DIV><DIV class="object figure large" id="noid"><DIV class="caption">a) Biofilm operons eps, bslA and tapA-sipW-tasA get derepressed by abrB knock outand also by repressing sinR with slrR from the thrC locus. 
b) Biofilm operons eps, bslA and tapA-sipW-tasA get derepressed by knocking out abrB and also by repressing sinR with slrR from the thrC locus. 
c)As in Figure 4b, where the complete robust construct is created. But with addition of the negative charge creating pgsBCA.</DIV></DIV></DIV></DIV><DIV class="entry footer"><DIV class="copyright">2015 iGEM Groningen</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>