Team:Groningen/Notebook/Overproduce TasA in B. subtilis ComI
 
Blue Bio Energy
Home
Team
Project
Results
Modeling
Notebook
Parts
Attributions
Collaborations
Design
Human Practices
Regulations
Media Coverage
Events
Card Game
Future
Safety
Measurement
Supervisors
Menu
Home
Team
Project
Results
Modeling
Notebook
Parts
Attributions
Collaborations
Design
Human Practices
Regulations
Media Coverage
Events
Card Game
Future
Safety
Measurement
Supervisors
Overproduce TasA in B. subtilis ComI
A biofilm is held together by a variety of different extracellular proteins, one of which is TasA. The TasA protein is encoded by the tasA gene. When expressed, the protein forms amyloid-like fibers in the extracellular matrix and provides stability and structural integrity of the biofilm.
Our goal is to overexpress TasA by adding a second copy of the tasA gene under the control of a salt inducible promoter. This overexpression will lead to a stronger biofilm when the biofilm is exposed to salt water.
Malesuada a nisi ac...
Finibus sapien, nec cursus...
00:00, 16 April 2015 - 00:00, 16 April 2015
Primers were designed for tasA in B. subtilis 168.
#
Primer
Sequence
5
tasA for
CGGAATTCGCGGCCGCTTCTAGAGCAATAAAAGGGGAGCTTACCATG
6
tasA rev
CGCTGCAGCGGCCGCTACTAGTATTATTAATTTTTATCCTCGCTATGC
7
tasA EcoRI overlap for
GATTTATCTGCTAAAGAGAACTCAGCGAGTGTGAAC
8
tasA EcoRI overlap rev
GTTCACACTCGCTGAGTTCTCTTTAGCAGATAAATC
9
tasA PstI overlap for
GAGTTTAGGAGTTGCTTCAGCAGCACTAGGATTAG
10
tasA PstI overlap rev
CTAATCCTAGTGCTGCTGAAGCAACTCCTAAACTC
Primers for tasA in B. subtilis 168.
Primers for tasA.
Atze van Stralen
00:00, 13 May 2015 - 00:00, 13 May 2015
PCR 1
B. subtilis 168 DNA was used.
dNTP Mastermix (MM) was prepared with a final concentration of 2 mM of each dNTP by adding water, see Table 1.
Compound
Amount
100 mM from each dNTP (dATP, dTTP, dCTP, dGTP)
20 µL
\( \mathrm{H_2O}\)
920 µL
Preparation Mastermix.
The Mastermix with 2 mM of each dNTP is added to buffer, phusion polymerase and water in the amounts indicated in Table 2.
Compound
Amount
5x buffer
33 µL
dNTP MM (2 mM)
16.5 µL
Phusion polymerase
1.65 µL
\( \mathrm{H_2O}\)
112.2 µL
Mastermix for PCR 1.
The resulting solution was combined with template DNA and primers. See Table 3.
Compound
Amount
Mastermix PCR 1
30 µL
Template DNA
0.3 µL
Each primer
0.3 µL
Components per sample.
The following samples were made. See Table 4.
#
Primer 1
Primer 2
Template DNA
3
5 tasA for
10 PstI overlap rev
B. subtilis 168
4
9 PstI overlap for
8 EcoRI overlap rev
B. subtilis 168
5
7 EcoRI overlap
6 tasA rev
B. subtilis 168
Components per sample.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
2:00
2
Denaturation
98 °C
0:10
3
Annealing
60 °C
0:20
4
Extension
72 °C
0:30
5
Go back to step 2 (30x)
6
Final extension
72 °C
5:00
PCR Thermocycle.
The samples were loaded on 1% agarose gel with EtBr to check for PCR products. The results are listed in Table 6.
#
Product length
3
± 100 bp
4
± 150 bp
5
± 700 bp
PCR Results.
As the product at ± 700 bp was not that clearly visible, the PCR was repeated. PCR purification products of samples 3, 4 and 5 were stored in freezer at -20 °C as 1.3, 1.4 and 1.5.
Harm Ruesink
00:00, 15 May 2015 - 00:00, 15 May 2015
PCR 2
B. subtilis 168 DNA was used.
The Mastermix (MM) with 2 mM of each dNTP is added to buffer, phusion polymerase and water in the amounts indicated in Table 7.
Compound
Amount
5x buffer
33 µL
dNTP MM (2 mM)
16.5 µL
Phusion polymerase
1.65 µL
\( \mathrm{H_2O}\)
112.2 µL
Mastermix for PCR 2.
The resulting solution was combined with template DNA and primers. See Table 8.
Compound
Amount
Mastermix PCR 1
30 µL
Template DNA
0.3 µL
Each primer
0.3 µL
Components per sample.
#
Primer 1
Primer 2
Template DNA
5
7 tasA EcoRI overlap for
6 tasA rev
B. subtilis 168
6
5 tasA for
8 tasA EcoRI overlap rev
PCR product 1.3 + 1.4
7
9 tasA PstI overlap for
6 tasA rev
PCR product 1.4 + 1.5
8
5 tasA for
8 tasA EcoRI overlap rev
PCR product 1.3 + 1.4 + 1.5
1
5 tasA for
8 tasA EcoRI overlap rev
PCR product 1.3
Components per sample.
Sample 1 is a control to check for genomic DNA of PCR 1.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annealing
62 °C
0:30
4
Extension
72 °C
1:00
5
Go back to step 2 (3x)
6
Denaturation
98 °C
0:30
7
Annealing
60 °C
0:30
8
Extension
72 °C
1:00
9
Go back to step 6 (3x)
10
Denaturation
98 °C
0:30
11
Annealing
58 °C
0:30
12
Extension
72 °C
1:00
13
Go back to step 10 (3x)
14
Denaturation
98 °C
0:30
15
Annealing
56 °C
0:30
16
Extension
72 °C
1:00
17
Go back to step 14 (3x)
18
Denaturation
98 °C
0:30
19
Annealing
54 °C
0:30
20
Extension
72 °C
1:00
21
Go back to step 18 (3x)
22
Denaturation
98 °C
0:30
23
Annealing
54 °C
0:30
24
Extension
72 °C
1:00
25
Go back to step 22 (30x)
26
Final extension
72 °C
10:00
PCR Thermocycle.
#
Product length
5
± 700 bp
6
± 250 bp
7
no clear product
8
no clear product
1
± 250 bp
PCR Results.
Sample 7 should have showed a band at ± 850 bp and sample 8 at ± 950 bp, but as this was not observed, the product was discarded. In sample 1 still genomic DNA was present. For further PCR experiments, a gelpurification of the PCR product was done to get rid of the genomic DNA. Sample 1 was discarded, as it had no use in further experiments.
Harm Ruesink
00:00, 18 May 2015 - 00:00, 18 May 2015
Purification of PCR 2 product 5 and 6 with gelpurification on 1% agarose gel with EtBr. Clean up sample with PCR cleanup kit.
PCR product 6 contains part 3 and 4 of TasA.
PCR product 5 contains part 5 of TasA.
Stored in freezer at -20 °C as 2.5 and 2.6.
PCR 3
Compound
Amount
5x buffer
28 µL
dNTP MM (2 mM)
14 µL
Phusion polymerase
1.4 µL
\( \mathrm{H_2O}\)
95.2 µL
Mastermix for PCR 3.
Compound
Amount
Mastermix PCR 3
30 µL
Template DNA
0.3 µL
Each primer
0.3 µL
Components per sample.
#
Primer 1
Primer 2
Template DNA
3
5 tasA for
6 tasA rev
PCR product 2.5 + 2.6
4
5 tasA for
6 tasA rev
PCR product 1.3 + 1.4 + 2.5
Components per sample.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annealing
56 °C
0:30
4
Extension
72 °C
1:00
5
Go back to step 2 (15x)
6
Denaturation
98 °C
0:30
7
Annealing
70 °C
0:30
8
Extension
72 °C
1:00
9
Go back to step 6 (repeat 15x)
10
Final extension
72 °C
10:00
PCR Thermocycle.
The PCR product was stored in the fridge at 4 °C.
Harm Ruesink
00:00, 19 May 2015 - 00:00, 19 May 2015
Run a gel with PCR 3 products on 1% agarose gel with EtBr.
On the gel, for sample 3 two bands were visible, indicating that the PCR assembly of fragement 2.5 and 2.6 did not work. Sample 4 had no product, so also the PCR assembly of product 1.3, 1.4 and 1.5 did not work.
Gibson Assembly
For the Gibson assembly, DNA was used with the concentrations indicated in Table 1. The concentration in the second column is the concentration of DNA based on the gel. The concentrations listed in column 3 are obtained by using Nanodrop.
PCR product
Estimation (based on gel)
Nanodrop
1.3 (100 bp)
250 ng/µL
37 ng/µL
1.4 (150 bp)
170 ng/µL
27.1 ng/µL
2.5 (700 bp)
200 ng/µL
67.7 ng/µL
2.6 (250 bp)
40 ng/µL
42.1 ng/µL
DNA concentration of samples.
Two Gibson reaction samples were made.
#
Component
Amount
1
PCR product 1.3 (100 bp)
0.588 µL
PCR product 1.4 (150 bp)
0.260 µL
PCR product 2.5 (700 bp)
0.143 µL
\( \mathrm{H_2O}\)
4.00 µL
2
PCR product 2.5 (700 bp)
0.588 µL
PCR product 2.6 (250 bp)
0.893 µL
\( \mathrm{H_2O}\)
3.59 µL
Gibson reaction samples.
15 µL of Gibson reaction mixture (Clement) was added to a total volume of 20 µL and incubated for 1h at 50 °C. The Gibson assembly products were purified with the PCR purification kit and stored as tasA DNA 1 and tasA DNA 2 in freezer at -20 °C.
PCR 4
Compound
Amount
5x buffer
14 µL
dNTP MM (2 mM)
7 µL
Phusion polymerase
0.7 µL
\( \mathrm{H_2O}\)
47.6 µL
Mastermix for PCR 4.
Compound
Amount
Mastermix PCR 4
30 µL
Template DNA
0.3 µL
Each primer
0.3 µL
Components per sample.
#
Primer 1
Primer 2
Template DNA
1
5 tasA for
6 tasA rev
tasA DNA 1
2
5 tasA for
6 tasA rev
tasA DNA 2
Components per sample.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annealing
55 °C
0:30
4
Extension
72 °C
0:30
5
Go back to step 2 (30x)
6
Final extension
72 °C
10:00
PCR Thermocycle.
The product was stored in the freezer at -20 °C.
Harm Ruesink
00:00, 20 May 2015 - 00:00, 20 May 2015
PCR 4 was purified with the PCR purification kit. The products were stored as tasA DNA 1 & tasA DNA 2 and PCR 4 prod 20-5-2015 (side). On gel it was checked if the PCR had worked out. A gel was made of 1% agarose with EtBr. The loaded samples consisted of the components listed below. The samples were compared with 3 µL of DNA ladder.
Sample:
1.5 µL 6x loading buffer
2.5 µL \( \mathrm{H_2O}\)
2 µL sample
Ladder:
3 µL GeneRuler™ 1 kb DNA Ladder
On the gel there were no bands visible for sample 1. Apparently the tasA product had not formed. The gel of sample 2 did show a product at ± 950 bp. This fragment was checked with restriction enzymes to find out if the restriction sites were absent in the tasA gene. This product, PCR product 4.2, was digested with the biobrick assembly kit digestion enzymes (NEB) using EcoRI and PstI. The following samples were made. See Table 2.
#
Compound
Amount
1
Buffer 10x
2 µL
EcoRI
1 µL
PCR prod 4.2
2 µL
\( \mathrm{H_2O}\)
15 µL
2
Buffer 10x
2 µL
PstI
1 µL
PCR prod 4.2
2 µL
\( \mathrm{H_2O}\)
15 µL
Components used for digestion of PCR product 4.2.
The following digestion protocol was followed. See Table 3.
#
Step
Temperature
Time
1
Digestion
37 °C
30:00
2
Enzyme inactivation
80 °C
10:00
Digestion protocol.
A gel was loaded with digestion product of sample 1 and 2 on 1% agarose with EtBr. The following components were added to each well.
Sample:
1.5 µL 6x loading buffer
5 µL digestion product
Ladder:
3 µL GeneRuler™ 100 bp DNA Ladder
On the gel there was one band visible for sample 1 at ± 950 bp. Apparently  EcoRI did not cut tasA. A mutation in the EcoRI restriction site of tasA was possibly present. The gel of sample 2 showed a product at ± 950 bp, so pstI did not cut tasA. So a mutation in the PstI restriction site of tasA might have been present.
For the ligation of tasA into the "RFP backbone", first the digestion of tasA (PCR prod 4.2) and "RFP backbone" was carried out.
Compound
Amount
Sample (PCR prod 4.2)
10 µL
EcoRI
1 µL
PstI
1 µL
Buffer 10x
3 µL
\( \mathrm{H_2O}\)
15 µL
Components used for digestion of PCR product 4.2.
Compound
Amount
Sample ("RFP backbone")
5 µL
EcoRI
1 µL
PstI
1 µL
Buffer 10x
3 µL
\( \mathrm{H_2O}\)
20 µL
Components used for digestion of plasmid ("RFP backbone").
The samples were digested for 2 hours at 37 °C. Afterwards the DNA was cleaned up with the PCR purification kit, and stored at -20 °C. The DNA was stored in 20 µL elution buffer.
Sample
Name
Plasmid
ligate RFP BB, digest prod 20-5-15
tasA
ligate tasA 4.2, digest prod 20-5-15
Storage information of purified PCR product.
NOTE: The next time a gel extraction must be done after digestion, as the RFP insert is too large to get rid of with the PCR purification kit.
Harm Ruesink
00:00, 21 May 2015 - 00:00, 21 May 2015
The ligation of the RFP backbone with tasA (4.2) was performed. First a gel extract was made on 1% agarose gel with EtBr to get rid of the RFP insert.
Sample:
4 µL 6x buffer
20 µL sample (RFP backbone)
Ladder:
3 µL GeneRuler™ 1 kb DNA Ladder
The gel was run for ± 45 minutes at 100 V. Afterwards the band for the RFP backbone was cut out to obtain purified RFP backbone without the RFP insert. The RFP backbone DNA was extracted from the gel with the PCR purification kit. For this binding buffer was added to the cut out gel in the ratio 2:1 binding buffer:gel weight. It was stored as BB1, purified backbone (50 µL elution buffer). Afterwards the ligation of tasA in BB1 was carried out. For that the following components were combined.
15 µL BB1
2 µL 10x buffer (2.1)
1 µL tasA DNA (4.2)
2 µL T4 ligase
The mixture was incubated at room temperature for 2 hours. During this time ampicillin plates were poured (100 µg/mL). Afterwards the heat shock transformation was carried out using the following components.
10 µL ligation mixture in competent DH5 alpha
Competent cells Harm (E.coli (1)) (CompH)
Competent cells Clement (CompC)
The transformed cells were plated on LB+amp plates.
90% CompH
10% CompH
90% CompC
10% CompC
The plates were put in the incubator overnight at 37 °C.
Harm Ruesink
00:00, 22 May 2015 - 00:00, 22 May 2015
The transformation was checked. There were 5 colonies visible on the LB+amp plates. 4 colonies on 90% compH and one colony on 10% CompH. Both plates were stored in the fridge at 4 °C.
Harm Ruesink
00:00, 26 May 2015 - 00:00, 26 May 2015
The five transformed colonies were inoculated overnight in liquid LB+amp (30 µL amp per 3 mL LB (1/100)). This should have been 3µL per 3mL amp, so also this concentration was made and used. The culture of the colony from the 10% plate was labeled sample 1 and the cultures of the four colonies from the 90% plate were labeled sample 2,3,4 and 5. The cultures were incubated at 37 °C, at 250 rpm.
Harm Ruesink
00:00, 27 May 2015 - 00:00, 27 May 2015
All cultures grew, of both the 1/100 and the 1/1000 amp concentrations. The cultures were miniprepped overnight according to the miniprep kit manual. The cultures were stored as tubes 1-5. DNA concentrations were determined using Nanodrop.
Tube
DNA (ng/µL)
1
168.4
2
516.7
3
420.1
4
551.5
5
523.9
DNA concentrations.
The miniprep product was digested with EcoRI and PstI to check for insert.
Harm Ruesink
00:00, 27 May 2015 - 00:00, 27 May 2015
All cultures grew, of both the 1/100 and the 1/1000 amp concentrations. The cultures were miniprepped overnight according to the miniprep kit manual. The cultures were stored as tubes 1-5. DNA concentrations were determined using Nanodrop.
Sample
1
2
3
4
5
\( \mathrm{H_2O}\)
14 µL
15 µL
15 µL
15 µL
15 µL
EcoRI
1 µL
1 µL
1 µL
1 µL
1 µL
PstI
1 µL
1 µL
1 µL
1 µL
1 µL
Sample DNA
2 µL
1 µL
1 µL
1 µL
1 µL
10x buffer (2.1)
2 µL
2 µL
2 µL
2 µL
2 µL
DNA concentrations.
The samples were incubated at 37 °C for 10 minutes. Afterwards a gel was run with 1% agarose and EtBr.
For each sample:
2 µL miniprep sample
1 µL 6x buffer
3 µL \( \mathrm{H_2O}\)
Ladder:
3 µL GeneRuler™ 1 kb DNA Ladder
On the gel, multiple bands were visible per plasmid, so the plasmid seems to be wrong. Again tasA was gelpurified and ligated into the backbone. Also the backbone (BB1) was checked by the next steps. The following combinations were made:
EcoRI + PstI + RFP plasmid
EcoRI + SpeI + RFP plasmid
XbaI + PstI + RFP plasmid
XbaI + SpeI + RFP plasmid
uncut RFP plasmid
Digestion Mastermix:
12 µL 10x buffer (2.1)
9 µL DNA (RFP plasmid)
99µL \( \mathrm{H_2O}\)
Per sample:
28 µL Mastermix
2 µL restriction enzymes
The samples were incubated for 30 minutes at 37 °C. Afterwards the samples were loaded on gel, 1% agarose with EtBr.
For each sample:
5 µL sample
1 µL 6x buffer
Ladder:
2 µL GeneRuler™ 1 kb DNA Ladder
On the gel it was seen that BB1 is probably not good, as it still has RFP in it. This was checked with the restriction enzymes of Molgen.
Component
Amount
10x buffer (2.1)
6.6 µL
EcoRI
2.2 µL
PstI
2.2 µL
\( \mathrm{H_2O}\)
50.6 µL
Components for Mastermix digestion.
Component
Amount
RFP plasmid DNA
2 µL
10x buffer
3 µL
EcoRI (Molgen)
1 µL
PstI(Molgen)
1 µL
\( \mathrm{H_2O}\)
23 µL
Components for digestion.
The mixture was incubated at 37 °C for 2 hours. After 2 hours the sample was loaded on a 1% agarose gel with EtBr gel with uncut plasmid. On the gel it was visible that the band of the digested plasmid is lower, but there was no insert visible. Do not use this backbone for further ligations and transformations.
Harm Ruesink
00:00, 2 June 2015 - 00:00, 2 June 2015
The backbones BBa_K823023 and BBa_K823024 were digested and ligated with tasA by the following steps.
#
Component
Amount
1
Mastermix
28 µL
DNA (BBa_K823023)
2 µL
2
Mastermix
28 µL
DNA (BBa_K823024)
2 µL
Digestion mixtures.
The samples were digested for 2 hours at 37 °C. Afterwards they were loaded on 1% agarose gel with SERVA DNA stain G. Also tasA was loaded on the gel for gel purification. 10 µL NEB DNA ladder log was used.
#
Component
Amount
1
DNA (tasA 4.2)
20 µL
6x buffer
4 µL
2
DNA (BBa_K823023)
25 µL
6x buffer
5 µL
3
DNA (BBa_K823024)
25 µL
6x buffer
5 µL
Samples for on the gel.
The gel was run on 100 V for ± 1 hour and the following bands were visible:
tasA at ±800-900 bp
K823023 (±5000 bp) including a band of its RFP insert (±1100 bp)
K823024 (±6000 bp) including a band of its RFP insert (±1100 bp)
These band sizes match the in silico sizes. The bands were cut out of the gel and purified with the PCR purification kit.
Sample
Mass gel(mg)
Binding buffer (µL)
tasA
77
144
K823023
181
362
K823024
133
266
Amounts of DNA and binding buffer added.
The samples were eluted with 20 µL elution buffer and stored at -20 °C as:
tasA DNA ligate
K823023 ligate
K823024 ligate
Harm Ruesink
00:00, 3 June 2015 - 00:00, 3 June 2015
For the ligation of tasA in BBa_K823023, the following components were used.
Component
Amount
Vector DNA (K823023)
2 µL
Insert DNA (tasA)
2 µL
10x buffer
2 µL
T4 ligase
1 µL
\( \mathrm{H_2O}\)
13 µL
Components used for ligation.
The ligation mixture was left 2 hours at room temperature. Afterwards 10 µL of the mixture was put in 100 µL competent E.coli (1). Two mixtures were streaked out on an LB+amp plate:
90% plate (90 µL)
10% plate (10 µL)
Harm Ruesink
00:00, 4 June 2015 - 00:00, 4 June 2015
The transformation was checked. No colonies were visible on either plate so the transformation failed.
Harm Ruesink
00:00, 8 June 2015 - 00:00, 8 June 2015
PCR 5
In this experiment it was attempted to increase the amount of tasA. For this first a Mastermix was made, to which primer and DNA were added.
Component
Amount
10x buffer
10 µL
dNTP MM (2 mM)
10 µL
Dreamtaq polymerase
1 µL
\( \mathrm{H_2O}\)
76,5 µL
Mastermix PCR 5.
Component
Amount
Mastermix PCR 5
30 µL
tasA DNA (4.2)
0.3 µL
5 tasA for
0.3 µL
6 tasA rev
0.3 µL
Components PCR 5 sample.
The following PCR cycle was used.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
3:00
2
Denaturation
98 °C
0:30
3
Annealing
60 °C
0:30
4
Extension
72 °C
1:00
5
Go back to step 2 (repeat 30x)
6
Final extension
72 °C
10:00
PCR thermocycle.
The PCR prduct was loaded on a 1% agarose gel with DNA stain G (1:50000) and run for 1 hour on 100 V with the following samples.
Sample:
30 µL DNA (tasA).
10 µL 6x buffer.
Ladder:
10 µL NEB ladder.
The band of tasA was hardly visible. The band was cut out and purified with the PCR purification kit.
Sample
Mass gel (mg)
Binding buffer (µL)
tasA
270
540
Amounts of DNA and binding buffer added.
The purified DNA was stored as tasA DNA (7.3 ng/µL Nanodrop value).
Harm Ruesink
00:00, 9 June 2015 - 00:00, 9 June 2015
For the digestion of tasA and the plasmids (BBa_K823023 and BBa_K823024), a Mastermix was made, which was added to the DNA.
Component
Amount
10x buffer (2.1)
6.6 µL
EcoRI
3.3 µL
PstI
3.3 µL
\( \mathrm{H_2O}\)
52.8 µL
Components Mastermix digestion.
#
Component
Amount
1
Mastermix
15 µL
DNA (tasA DNA)
10 µL
2
Mastermix
15 µL
DNA (K823023)
10 µL
3
Mastermix
15 µL
DNA (K823024)
10 µL
Components digestion samples.
The samples were incubated for 2 hours at 37 °C. Afterwards they were cleaned up with the PCR purification kit and eluted in 20 µL elution buffer. The purified samples were stored at -20 °C as:
tasA dig, 9-6-15.
23 dig, 9-6-15.
24 dig, 9-6-15.
The ligation of tasA in the plasmids (BBa_K823023 & BBa_K823024) was performed by combining the following components.
Sample
Component
Amount
23 lig
insert (tasA dig)
10 µL
vector(23 dig)
5 µL
10x buffer
2 µL
T4 ligase
0.5 µL
\( \mathrm{H_2O}\)
2.5 µL
24 lig
insert (tasA dig)
10 µL
vector(24 dig)
5 µL
10x buffer
2 µL
T4 ligase
0.5 µL
\( \mathrm{H_2O}\)
2.5 µL
Components of ligation samples.
The mixtures were ligated for 2 hours at room temperature. Afterwards competent E.coli was transformed with the ligation products. A heat shock transformation was used. The cultures were plated on LB+amp plates and incubated overnight at 37°C. They were labeled as:
100% comp E. coli with 23 lig
100% comp E. coli with 24 lig
Harm Ruesink
00:00, 10 June 2015 - 00:00, 10 June 2015
The transformation was checked. One white colony was visible on the 23 lig plate and 7 colonies were visible on the 24 lig plate. The colonies were grown overnight in LB+amp at 37 °C, 200 rpm.
BBa_K823023 + tasA (possible)
BBa_K823023 + RFP
BBa_K823024 + tasA 7x (possible)
BBa_K823024 + RFP
Harm Ruesink
00:00, 11 June 2015 - 00:00, 11 June 2015
The colonies that were grown overnight were miniprepped, by eluting them in 50 µL and stored as 23RFP, 23TA, 24RFP, 24TA (1,2,3,4,5,6&7). All tasA samples were digested to check for the insert. For this a Mastermix for digestion was made.
Component
Amount
10x buffer (2.1)
22 µL
EcoRI
9 µL
PstI
9 µL
\( \mathrm{H_2O}\)
167 µL
Mastermix for digestion.
Component
Amount
Mastermix
18 µL
DNA
2 µL
Sample for digestion.
The samples were incubated at 37 °C for 2 hours and 15 minutes. Afterwards they were put on a 1% agarose gel with 1:50000 DNA stain G. The gel was run on 120 V.
Sample:
5 µL sample
1 µL 6x buffer
Ladder:
10 µL Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb)
On the gel there was not much visible, so the procedure was repeated with more DNA.
Harm Ruesink
00:00, 15 June 2015 - 00:00, 15 June 2015
Digested tasA from backbone using EcoRI and PstI to check ligation. For this, a Mastermix was made for 13 samples (± 300 µL).
Component
Amount
10x buffer (2.1)
30 µL
EcoRI
10 µL
PstI
10 µL
\( \mathrm{H_2O}\)
185 µL
Mastermix digestion.
Component
Amount
DNA
5 µL
Mastermix
15 µL
Components per digestion sample.
The samples were incubated for 2 hours at 37 °C. Afterwards they were loaded on 1% agarose gel with 1:50000 DNA stain G.
Sample:
20 µL digestion sample
4 µL 6x buffer
Ladder:
10 µL Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb)
There was nothing visible on the gel. The gel was probably too hot when adding the DNA stain G.
Harm Ruesink
00:00, 16 June 2015 - 00:00, 16 June 2015
The digestion of tasA of 15 June 2015 was repeated. The samples were digested for 1.5 hours at 37 °C. Afterwards the samples were loaded on gel.
Sample:
10 µL digestion sample
2 µL 6x buffer
Ladder:
10 µL Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb)
A gel was run at 100 V for 1 hour. On the gel 24TA 3 seemed to have the correct insert.
Harm Ruesink
00:00, 18 June 2015 - 00:00, 18 June 2015
24TA 3 was send to Macrogen for sequencing:
5µL 1/20 primer (5 pmol/µL)
5µL plasmid DNA (24TA 3)
Sequencing code Macrogen:
Forward primer (5 tasA for): 1790ZAB067
Reverse primer (6 tasA rev): 1790ZAB068
Harm Ruesink
00:00, 19 June 2015 - 00:00, 19 June 2015
The seqencing results showed the tasA insert without restriction sites. The restriction sites are gone.
Harm Ruesink
00:00, 8 July 2015 - 00:00, 8 July 2015
In this experiment tasA was ligated into pSB1C3. For this first a Mastermix was made for the digestion of 24TA 3 (BBa_K823024 + tasA insert) and pSB1C3 EcoRI and PstI.
Component
Amount
10x buffer (2.1)
6.6 µL
EcoRI
3.3 µL
PstI
3.3 µL
\( \mathrm{H_2O}\)
23.4 µL
Mastermix digestion.
#
Component
Amount
1
24TA 3
10 µL
Mastermix
10 µL
2
pSB1C3
10 µL
Mastermix
10 µL
Components digestion sample.
The sample was digested for one hour at 37 °C. A PCR purification kit was used to clean up the DNA. Afterwards tasA was ligated in pSB1C3.
Component
Amount
tasA DNA
10 µL
pSB1C3 DNA
5 µL
T4 ligase
0.5 µL
T4 ligase buffer
2 µL
\( \mathrm{H_2O}\)
2.5 µL
Components used for ligation.
The sample was ligated at room temperature for 2 hours. Afterwards a heat shock transformation was performed with 100 µL NEB competent cells + ligation product.
90% plate (90µL)
10% plate (10µL)
The plates were incubated overnight at 37 °C.
Harm Ruesink
00:00, 9 July 2015 - 00:00, 9 July 2015
The transformation was checked. On both 10% and 90% plates a large amount of colonies had grown, so the transformation was successful. 4 colonies from the 10% plate was grown overnight in 3 mL LB+cm (6 µL 1/500).
Harm Ruesink
00:00, 10 July 2015 - 00:00, 10 July 2015
The samples were miniprepped and digested to check for insert. It was seen that indeed tasA was inserted successfully into pSB1C3. The insert was stored as T2.
Harm Ruesink
00:00, 16 July 2015 - 00:00, 16 July 2015
The salt-promoter and tasA were ligated into BBa_K823023. The salt promoter from Marieke was used. For the digest the following combinations were made.
#
DNA
Restriction enzymes
1
Psalt
EcoRI + SpeI
2
tasA (T2, 10-7-15)
XbaI + PstI
3
BBa_K823023
EcoRI + PstI
Digestion.
#
Component
Amount
1
DNA (Psalt)
6 µL
EcoRI
1 µL
SpeI
1 µL
10x buffer (2.1)
2 µL
\( \mathrm{H_2O}\)
10 µL
2
DNA (T2)
3 µL
XbaI
1 µL
PstI
1 µL
10x buffer (2.1)
2 µL
\( \mathrm{H_2O}\)
13 µL
3
DNA (K823023)
6 µL
EcoRI
1 µL
PstI
1 µL
10x buffer (2.1)
2 µL
\( \mathrm{H_2O}\)
10 µL
Components samples.
The samples were incubated for one hour at 37 °C. The samples were purified using the PCR purification kit, using 20 µL elution buffer. The samples were stored as 1 and 2, with on the side 16-7-15. The samples 1, 2 and 3 were ligated using the following amounts.
Component
Amount
DNA 1 (Psalt)
5 µL
DNA 2 (T2)
5 µL
DNA 3 (K823023)
2.5 µL
T4 ligase
0.5 µL
10x buffer
2 µL
\( \mathrm{H_2O}\)
5 µL
Components ligation sample.
The sample was ligated at room temperature for 30 minutes. Afterwards the heat shock transformation from the protocol book was performed. For this, 10 µL competent NEB cells with 10 µL ligation product was used. The resulting cells were plated on LB+amp plates and grown overnight at 37 °C.
Harm Ruesink
00:00, 17 July 2015 - 00:00, 17 July 2015
The transformation of the ligation product of the salt promoter with tasA was checked. On the plates a few white colonies were seen. Miniprep and colony PCR were performed to check whether the insert size was the right one. The colony PCR showed possible tasA with Psalt insert in one sample. This colony was grown overnight in LB+amp.
Harm Ruesink
00:00, 18 July 2015 - 00:00, 18 July 2015
The culture of the colony that seemed to have the right insert at 17 July, was miniprepped and stored as 33 (17-1-15).
Wiebrand de Boer
00:00, 20 July 2015 - 00:00, 20 July 2015
Sample 33 was send for sequencing with the prefix and suffix primers (no. 19 and 20) to Macrogen in the following amounts:
Component
Amount
1/20 primer (5 pmol/µL)
5 µL
plasmid DNA (sample 33)
5 µL
Amount of primers and DNA sent for sequencing.
Primer
Code
prefix primer (forward)
17C4ZAD518
suffix primer (reverse)
17C4ZAD519
Sequencing codes Macrogen.
Harm Ruesink
09:24, 20 July 2015 - 16:30, 20 July 2015
Sample 33 was transformed into Bacillus subtilis 3610 comI using Elrike's transformation protocol. For this a comI colony was taken from a plate and put in a tube with 2 mL MC 1x. The tube was put in the shaking incubator for ∼5 hours. After 5 hours of growing the OD at 600 nm was seen to be 0.15. 400 µL of culture was added to a new tube, and 5 µL of plasmid DNA (sample 33) was added. The new tube was placed in the shaking incubator at 37 °C.
Harm Ruesink
00:00, 22 July 2015 - 00:00, 22 July 2015
The sequencing results showed that the insert into the K823023 backbone was not the correct one. RFP was inserted instead of tasA with the salt promoter. The transformed Bacillus was discarded.
Harm Ruesink
00:00, 23 July 2015 - 00:00, 23 July 2015
Psalt, tasA and BBa_K823023 were ligated again. For this to be possible, first the salt promoter (Psalt) had to be digested using EcoRI and SpeI as restriction enzymes. tasA and BBA_K823023 were digested before already.
Component
Amount
DNA (Psalt)
10 µL
EcoRI
1 µL
SpeI
1 µL
10x buffer (2.1)
2 µL
\( \mathrm{H_2O}\)
6 µL
Components for digestion salt promoter.
The sample was digested for one hour at 37 °C. Afterwards the sample was purified using the PCR purification kit. For this 20 µL elution buffer was used. The subsequent ligation was performed with the folowing components.
Component
Amount
DNA 1 (Psalt)
5 µL
DNA 2 (tasA)
5 µL
DNA 3 (K823023)
5 µL
T4 ligase
1 µL
10x buffer
2 µL
\( \mathrm{H_2O}\)
2 µL
Components ligation Psalt, tasA and backbone.
The sample was ligated at room temperature for 2 hours. Afterwards the heat shock transformation form the protocol book was performed and the culture was plated on LB+amp plates and incubated overnight at 37 °C.
Harm Ruesink
00:00, 24 July 2015 - 00:00, 24 July 2015
The transformation of 23 July was checked. One white colony was visible on the plate, but this colony did not have the insert. So the ligation failed.
Harm Ruesink
09:30, 28 July 2015 - 16:00, 28 July 2015
The ligation of the T2 sample of 10 July with two promoters was performed, both with the Pveg promoter from the iGEM distribution kit 2015 (plate 1, 20G) and the Psalt promoter (from Marieke). For the promoters EcoRI and SpeI were used as restriction enzymes, and XbaI and EcoRI for the backbone. The following samples were made.
#
Component
Amount
1
T2 DNA
5 µL
10x buffer
2 µL
EcoRI
1 µL
XbaI
1 µL
\( \mathrm{H_2O}\)
11 µL
2
Pveg DNA
10 µL
10x buffer
2 µL
EcoRI
1 µL
SpeI
1 µL
\( \mathrm{H_2O}\)
6 µL
4
Psalt DNA
5 µL
10x buffer
2 µL
EcoRI
1 µL
SpeI
1 µL
\( \mathrm{H_2O}\)
11 µL
Components digestion samples.
The samples were digested at 37 °C for 2 hours. Afterwards a 1% agarose gel with stain G 1:50000 was run on 100 V for ∼45 minutes. 20 µL of DNA samples were mixed with 4 µL 6x buffer. As a ladder the 10 µL 2log NEB ladder was used. The right bands were cut out, weighed and dissolved in twice as much binding buffer. The samples were purified with the PCR purification kit (Genejet #K0701, lot 00265976) using 25 µL elution buffer. The concentrations were determined with Nanodrop.
Sample
Concentration
T2 dig (2)
4.5 ng/µL
Pveg (3)
1.4 ng/µL
Psalt (4)
5.0 ng/µL
Concentrations of samples determined by Nanodrop.
After the digestion, the promoters were ligated into the backbones. For this a ligation Mastermix was made for four samples.
Component
Amount
10x buffer
12 µL
T4 ligase
4 µL
\( \mathrm{H_2O}\)
8 µL
Ligation Mastermix.
#
Component
Amount
3
T2 dig (2)
12 µL
Pveg (3)
12 µL
Mastermix
6 µL
4
T2 dig (2)
12 µL
Psalt (4)
12 µL
Mastermix
6 µL
Components for ligation samples.
The samples were ligated overnight at room temperature.
Harm Ruesink
10:00, 29 July 2015 - 12:00, 29 July 2015
With the ligation product of 28 July, the heat shock transformation was performed according to the protocol book and the culture was plated on an LB+cm plate and incubated overnight at 37 °C.
Harm Ruesink
00:00, 30 July 2015 - 00:00, 30 July 2015
The transformation of 29 July was checked. No colonies were visible on the LB+cm plate. T2 was send to Macrogen for sequencing.
Component
Amount
1/20 primer (5 pmol/µL)
5 µL
plasmid DNA (T2)
5 µL
Amount of primers and DNA sent for sequencing.
Primer
Code
prefix primer (forward)
17C4ZAD658
suffix primer (reverse)
17C4ZAD659
Sequencing codes Macrogen.
Harm Ruesink
13:22, 6 August 2015 - 18:40, 6 August 2015
PliaG was digested and ligated with T2. Restriction enzymes were used that create a scar. For the digestion the restriction enzymes EcoRI and SpeI were used for the promoter and XbaI and EcoRI for T2. Below are listed the amounts that were added of each component.
#
Component
Amount
1
PliaG DNA
5 µL
10x buffer
2 µL
EcoRI
1 µL
SpeI
1 µL
\( \mathrm{H_2O}\)
11 µL
2
T2 DNA
5 µL
10x buffer
2 µL
XbaI
1 µL
EcoRI
1 µL
\( \mathrm{H_2O}\)
11 µL
Components for digestion.
The sample was digested for 2 hours at 37 °C. Afterwards it was loaded on 1% agarose gel with DNA stain G 1:50000. On the gel were loaded 20 µL of the sample DNA with 4 µL buffer. As a ladder 10 µL of the 2log NEB ladder was used. The gel was run for ∼30 minutes at 100 V. The right bands were cut from the gel and twice as much binding buffer was added. The samples were purificated using the Thermo Scientific GeneJET PCR Purification Kit  and their concentrations were checked using Nanodrop.
Sample
Concentration
PliaG dig
4.6 ng/µL
T2 dig
4.9 ng/µL
Concentrations of samples determined with Nanodrop.
The ligation was carried out overnight at room temperature using the following components.
Component
Amount
PliaG DNA
5 µL
T2 DNA
10 µL
10x T4 buffer
2 µL
T4 ligase
0.5 µL
\( \mathrm{H_2O}\)
2.5 µL
Components for ligation of PliaG with T2.
Harm Ruesink
11:30, 7 August 2015 - 13:30, 7 August 2015
A heat shock transformation was performed by adding 20 µL of DNA (ligation product of T2 with PliaG) to 10 µL competent NEB cells. After 2 hours the culture was plated on LB+cm and incubated overnight at 37 °C.
Harm Ruesink
00:00, 8 August 2015 - 00:00, 8 August 2015
The transformation of 7 August was checked. No colonies were visible.
Harm Ruesink
00:00, 9 August 2015 - 00:00, 9 August 2015
The transformation of 7 August was checked. Some white colonies were visible. With these colonies a colony PCR was carried out. After the PCR the samples were loaded on a 1% agarose gel and it was seen that sample 3 and 7 possibly have the insert. Cultures were made of sample 3 and 7 by putting some of the bacteria in a tube with 3 mL LB+cm. The tubes were put in the shaking incubator at 37 °C with 200 rpm.
Harm Ruesink
12:00, 11 August 2015 - 15:30, 11 August 2015
The cultures of sample 3 and 7 were miniprepped and loaded on 1% agarose gel with DNA stain G 1:50000. For each DNA sample 1 µL was added to the gel, with 4 µL \( \mathrm{H_2O}\) and 1 µL 6x buffer. As a ladder 10 µL of the Quick-Load® Purple 2-Log DNA Ladder (0.1 - 10.0 kb) was used. The gel was run for ∼30 minutes at 100 V. On the gel afterwards nothing was visible, so the DNA concentration needed to be checked again.
Harm Ruesink
10:30, 13 August 2015 - 20:15, 13 August 2015
The concentrations of the samples 3 and 7 were checked again and were found to be the following.
Sample
Concentration
3
3.3 ng/µL
7
9.3 ng/µL
Concentrations of samples determined with Nanodrop.
The samples were loaded on 1% agarose gel + DNA stain G 1:50000.
#
Component
Amount
3
Sample DNA
10 µL
6x buffer
2 µL
7
Sample DNA
3 µL
6x buffer
1 µL
\( \mathrm{H_2O}\)
2 µL
T2
Sample DNA
1 µL
6x buffer
1 µL
\( \mathrm{H_2O}\)
4 µL
Samples on gel.
The gel was run for 30 minutes at 100 V. On the gel no bands were visible. So the digestion was carried out again. For the promoter PliaG, the restriction enzymes SpeI and PstI were used, and for T2, XbaI and PstI. The following digestion samples were prepared.
#
Component
Amount
1
PliaG DNA
10 µL
10x buffer (2.1)
2 µL
SpeI
1 µL
PstI
1 µL
\( \mathrm{H_2O}\)
6 µL
2
T2 DNA
10 µL
10x buffer (2.1)
2 µL
XbaI
1 µL
PstI
1 µL
\( \mathrm{H_2O}\)
6 µL
Components for digestion.
The samples were incubated at 37 °C for 3.5 hours. Afterwards they were loaded on a 1% agarose gel with DNA stain G 1:50000. For the DNA samples 20 µL was used with 4 µL 6x buffer. As a ladder 1 µL of the GeneRuler™ 1 kb DNA Ladder was used. The gel was run for ∼45 minutes at 100 V. The bands with the right sizes were cut from the gel and dissolved in twice as much elution buffer. The GeneJet PCR purification kit was used to extract the DNA, using 21µL elution buffer. With Nanodrop the concentrations were determined.
Sample
Concentration
1
4.0 ng/µL
2
5.1 ng/µL
Concentrations of samples determined with Nanodrop.
The samples were ligated overnight at room temperature.
Component
Amount
DNA sample 1
20 µL
DNA sample 2
20 µL
10x T4 buffer
5 µL
T4 ligase
1 µL
Components for ligation of PliaG with T2.
Harm Ruesink
00:00, 17 August 2015 - 00:00, 17 August 2015
A colony PCR with 6 colonies of the T2+PliaG plate was done. Each colony was taken from the plate and resuspended in 20 µL sterile MQ water. A Mastermix for 16 samples was made.
Component
Amount
Phusion polymerase
4.5 µL
5x phusion buffer
80 µL
DNTP MM (2 mM)
35 µL
\( \mathrm{H_2O}\)
240 µL
VR + VF2 primers
1 µL each
Components Mastermix.
For the PCR samples, 18 µL of Mastermix was added to 4 µL of colony suspension. The following PCR thermocycle was used.
#
Step
Temperature
Time
1
Initial denaturation
99 °C
10:00
2
Denaturation
95 °C
0:30
3
Annealing
58 °C
0:30
4
Extension
72 °C
1:30
5
Go back to step 2 (repeat 30x)
6
Final extension
72 °C
10:00
PCR thermocycle.
The samples were run on a 1% agarose gel with DNA stain G 1:50000. 5 µL of sample DNA was mixed with 1 µL 6x buffer. As a ladder 1 µL of the GeneRuler™ 1 kb DNA Ladder was used. The gel was run for ∼30 minutes at 100 V. No correct PCR product was visible on the gel. All bands are at the same height as the control (T2 without promoter).
Harm Ruesink
00:00, 18 August 2015 - 00:00, 18 August 2015
A new colony PCR was performed with 8 colonies from the T2+PliaG plate and control (T2). Each colony was resuspended in 20 µL MQ water. A Mastermix was made for 9 samples.
Component
Amount
Phusion polymerase
2 µL
5x phusion buffer
36 µL
DNTP MM (2 mM)
18 µL
\( \mathrm{H_2O}\)
120 µL
Primers (pSB1C3 colony PCR)
3 µL each
Components Mastermix.
For the PCR samples, 18 µL of Mastermix was added to 4 µL of colony suspension. The following PCR thermocycle was used.
#
Step
Temperature
Time
1
Initial denaturation
98 °C
10:00
2
Denaturation
95 °C
0:30
3
Annealing
55 °C
0:30
4
Extension
72 °C
1:30
5
Go back to step 2 (repeat 30x)
6
Final extension
72 °C
10:00
PCR thermocycle.
The samples were loaded on a 1% agarose gel with DNA stain G 1:50000. On the gel it was seen that the insert did not have the correct size, meaning that the promoter (PliaG) did not ligate in T2.
Harm Ruesink
2015 iGEM Groningen
