<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_project_degrons"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/project/degrons">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/project/degrons&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/project/degrons&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/project/degrons&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/project/degrons" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/project/degrons</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="topbit" style="margin-bottom: 40px;"><TABLE align="center" style="border-spacing: 5px;"><TBODY><TR><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh">Home</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/team/">Team</A></TD><TD id="navlink"><A href="https://igem.org/Team.cgi?id=1399">Profile</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/logic/">Background</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/project/">Project</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/HP/">Policy and Practices</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/modelling/">Modelling</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/log">Daily log</A></TD></TR></TBODY></TABLE></DIV><DIV id="tourleft"><A href="https://2014.igem.org/Team:Edinburgh/zeigler">← The Zeigler report</A></DIV><DIV id="tourright"><A href="https://2014.igem.org/Team:Edinburgh/project/parts">Our BioBricks→</A></DIV><DIV id="igem"><DIV id="header"><H1>Degrons</H1>Anna Stikāne</DIV><DIV id="degrons"><H2>What are degrons?</H2><P>Synthetic biology aims to build rigid, orthogonal molecular systems with valuable industrial and medical applications. This implies ability to regulate protein production (transcription and translation of mRNA) as well as their lifespan. A convenient method of controlling protein lifetime is marking them for protease-mediated degradation. In bacteria this can be achieved utilizing a widely studied and conserved SsrA-tag mediated protein degradation system. Natural SsrA-tag is added to nascent polypeptides as a way to rescue stalled ribosomes (a process termed trans-translation).<SUP>1</SUP> SsrA tagged proteins are recognised and degraded by proteases within all cellular compartment (i.e., ClpXP and ClpAP in cytoplasm, FtsH within membranes and Tsp in periplasmic space).<SUP>2–4</SUP> The actual degradation rate of SsrA tagged proteins depend on several factors including the sequence of SsrA polypeptide, SsrA binding protein concentration, competition with other interacting molecules and temperature, which all might be altered and engineered to get even more robust and predictable degradation of SsrA tagged proteins.<SUP>5–9</SUP> Engineered protein instability can be advantageous to get specific protein concentrations in vivo and prevent unwanted protein accumulation. This in turn is beneficial for synthetic genetic constructs were protein production timing and concentration thresholds are crucial, e.g., genetic bi-stable switches, oscillators, biosensors, and faster acting bacterial logic gates and computation systems.</P><H3>SsrA-tags and protein degradation</H3><H4>The natural function of SsrA</H4><P>In <EM>Escherichia coli</EM> SsrA (EcoCyc<SUP>10</SUP> accession number EG30100) is a 362 nucleotides long monocistronic gene, encoding for small stable 10Sa RNA.<SUP>11</SUP> SsrA RNA is also referred to as transfer-messenger RNA or tmRNA as it has both mRNA and tRNA structural properties.<SUP>12</SUP> The biological function of SsrA RNA is to recognise and resolve ‘troubled’ translation via process called trans-translation.<SUP>1</SUP> The molecular model of trans-translation  was proposed by Keiler et al. in 1996<SUP>1</SUP> and since then has been further explored and reviewed in much greater details.<SUP>13,14</SUP> To summarize, tmRNA as tRNA is charged with alanine and binds to stalled mRNA-ribosome complexes. Then ribosome switches templates from mRNA to incoming SsrA and finishes the translation.(Figure 1) This process creates a fusion protein ending with 11 amino acids long ‘AANDENYALAA’ peptide tag.<SUP>1,15</SUP> The added SsrA tag then is recognised and rapidly degraded by ClpXP and ClpAP cytoplasmic proteases, FtsH(HflB) membrane protease and Tsp (Prc) periplasmic protease thus preventing build-up of incorrect proteins in all cellular compartments.<SUP>2–4</SUP></P><H4>Cytoplasmic degradation of SsrA tagged proteins</H4><P>The SsrA-tagged protein degradation  can be subdivided into 5 steps: (1) substrate binding to ClpXP or ClpAP, (2) substrate denaturation, (3) translocation; (4) proteolysis, (5)release of peptide fragments.<SUP>16</SUP> (Figure 2)</P><P>In cytoplasm SsrA tagged proteins are degraded by ClpXP and ClpAP.8 ClpX and ClpA are heptameric unfoldases that can both bind directly to SsrA tag.<SUP>17</SUP> In addition, SsrA-tagged proteins can also be directed to ClpXP by ClpX adaptor protein SspB, which physically tethers SsrA-tagged substrates to ClpX.<SUP>18</SUP> (Figure 3)</P><P> Then ClpX and ClpA  use ATP energy to unfold the tagged protein<SUP>17</SUP>. ClpXP uses ~150 molecules of ATP to denature one molecule of substrate however in the absence of ClpP protease even more ATPs are hydrolysed.<SUP>19</SUP>  Interestingly, ClpXP degrades all SsrA-tagged substrates with remarkably similar rates although hyperstable substrates do slow down the ClpXP ATP hydrolysis rate.<SUP>19</SUP></P><P>The denatured protein is then degraded by a serine peptidase ClpP.<SUP>17</SUP> ClpP consists of proteolytic cavity enclosed by two stacked heptameric rings.<SUP>19</SUP> The translocation of substrate into ClpP cavity requires association with ATPases like ClpX and ClpA.<SUP>19</SUP> In addition, proteins unfolded by ClpX and ClpA have higher affinity for degradation by other cellular proteases.<SUP>8</SUP> The analysis of the ClpXP activity dynamics has identified that substrate denaturation is the slow rate limiting step and the actual proteolysis step is very fast (small peptides are degraded with rate &gt;104 min<SUP>-1</SUP>).<SUP>16</SUP>(Figure 2) For more information on ClpXP action see review from Baker and Sauer, (2012).<SUP>20</SUP></P><H4>Engineering SsrA based protein degradation system</H4><P>SsrA-tags are powerful system for controlling protein lifetime. As the tag consists of only 11 amino acids, the tag-encoding nucleotide sequence can be easily added to any protein coding sequence in a single PCR reaction. The protein structural stability or function is not affected by the attached SsrA-tag.<SUP>22</SUP> Moreover, the SsrA tag recognition and following substrate denaturation and degradation by ClpXP and ClpAP is not significantly affected by the substrates structural features and stability.<SUP>19</SUP> In theory this should mean that the minimal SsrA-tagged protein turnover rate within cell will be the SsrA mediated degradation rate. The actual degradation rate for SsrA tagged protein still might be higher if the protein normally is unstable. Thus SsrA-tags could be used to get better control over protein lifetime, which might be especially useful when working with such hyperstable proteins as GFP. In fact several parameters of the system can be further explored and modified to gain a broader spectrum of specified degradation rates. These include altered sequences of SsrA tag and regulation of SspB, ClpXP and ClpAP concentrations.</P><P>Yet another useful feature of the SsrA tag based system is the conservation and presence of SsrA tmRNA throughout the bacterial kingdom including plastid and mitochondrial genomes of some eukaryotes (e.g., algae and protists, respectively) which is no surprise given the importance of SsrA tag biological function as ‘translation quality checkpoint’.<SUP>13</SUP> Table 1 from Karzai et al.,(2000) summarizes phylogenetic information of SsrA tmRNA and associated proteins within eubacterial genomes.<SUP>21</SUP> This in turn means that an optimal SsrA tag motif can be found and used for each particular bacterial chassis.</P><H4>Protein interaction information within SsrA tag</H4><P>Extensive amino acid mutagenesis studies have revealed that protease recognition elements are densely packed within SsrA peptide.<SUP>17</SUP> (Figure 4) Periplasmic Tsp and  membrane FtsH both recognise final hydrophobic amino acids at the C-terminus.<SUP>3,4</SUP> Similarly, cytoplasmic ClpX also recognise the final three amino acids (9-11) as well as the carboxyl group.<SUP>5,17</SUP> In contrast, ClpX enhancer protein SspB recognises first few amino acids (1-4) and amino acid number 7(Tyr).<SUP>17</SUP> In particular, the third amino acid (N; Asn3) is crucial for SspB binding and can only be substituted with H(His) without losing SspB binding. <SUP>17,22</SUP> The other cytoplasmic protease ClpA interacts with amino acids 8-10 and 1-2 at the degron N terminus.<SUP>17</SUP> Knowledge of these protein recognition determinants can be exploited to create synthetic SsrA tags with modified amino acid composition.</P><H4>Modifying the SsrA tag</H4><P>Andersen et al., (1998) showed that changing last three amino acids of the tag can significantly modify the rate of degradation and are often used to specify particular version of the tag.<SUP>5</SUP> Andersen et al., (1998)  tested GFP with five different versions of SsrA-tag and reported the wild type SsrA-LAA and SsrA-LVA leading to the fastest degradation, and SsrA-AAV and SsrA-ASV resulted in slower degradation in E. coli.<SUP>5</SUP> Same GFP-SsrA tagged constructs were tested in Pseudomonas putida and showed that  SsrA-LVA provided the highest degradation rate.5 Amino acid alterations in the rest of SsrA tag can also be used to change SsrA recognition by SspB and ClpA. As SspB causes ClpX to bind more tightly to SsrA tag, SspB improves efficiency of degradation at low substrate concentrations.<SUP>23</SUP> However, SspB and ClpX recognition determinants are located very close within the tag and can lead to steric clash and impaired degradation if both proteins try to bind to the tag.6 Hersch et al., (2004) showed that extending the SsrA tag by repeating amino acids 6-7 (NY) reduced the steric clash between SspB and ClpX and created synthetic degron ‘AANDENYNYALAA’ with 4-5 times higher degradation rate than the natural SsrA LAA without changing  K<SUB>M</SUB> and V<SUB>max</SUB> constants for ClpXP.<SUP>6</SUP> (Figure 5) Our tested SsrA-DAS+2 tag contains two extra amino acids preventing supposed clash between SspB and ClpX and together with SsrA-DAS tag are not expected to be recognised by ClpX.<SUP>7</SUP></P><H4>Degradation rate and protease concentrations</H4><P>Another factor affecting SsrA degradation rate is the concentration of intracellular proteases. Farrel et al., (2005) monitored degradation rate of SsrA tagged GFP in vivo and reported increase in degradation (rate) upon cell entry in stationary growth phase.<SUP>8</SUP> This coincided with an increase in production and concentration of ClpA and ClpP.<SUP>8</SUP> The concentrations of ClpX and SspB remained the same over time.<SUP>8</SUP> One strategy to gain better control over protease abundance caused fluctuations of degradation rate is removal of protease recognition elements of SsrA. For example, McGinness et al., (2006) demonstrated a SsrA-DAS degron with a degradation rate depending only on interaction with SspB.<SUP>7</SUP>  In addition, the proposed SsrA-DAS degron system can be relatively easily transferred to other organisms because it only requires three housekeeping genes (SspB, ClpX, ClpP) and the tagged protein.<SUP>7</SUP> Another way to control protease concentration would be increasing their production by introducing additional copy of ClpX, ClpA or SspB.</P><P>The degradation rate of SsrA tagged substrates is also influenced by molecular interactions between SsrA tag binding proteins (ClpA, ClpX, SspB) and their other substrates. For example, SspB and ClpA have overlapping recognition elements within SsrA tag thus in theory SspB could prevent ClpA binding and direct SsrA tagged proteins to degradation by ClpX.<SUP>17</SUP> This has been shown in vitro but not in vivo as the amount of SspB molecules within the cell rarely outweighs ClpA.<SUP>8</SUP>  Also, molecular competition between SsrA-tag and other ClpXP and ClpAP substrates for binding ClpX and ClpA can interfere with SsrA tagged protein degradation rate. For example, another ClpA mediator ClpS can inhibit ClpAP from recognising SsrA tag.<SUP>8</SUP></P><H4>A few more factors to consider</H4><P>One additional factor to consider when engineering and adding the SsrA tag is the intracellular wash out of native SsrA tags added by trans-translation. For instance, highly expressed protein with synthetic SsrA tag might occupy the bulk of ClpXP and ClpAP proteases thus impeding the removal of such natural protease substrates as damaged proteins.<SUP>5</SUP> Such situation then in turn might lead to reduced growth rate of bacterium.<SUP>5</SUP></P><P>The SsrA-tag mediated protein degradation rate highly depends on the  temperature of environment as this system has none or minimal temperature compensation mechanisms.<SUP>9</SUP> For example, Purcell et al., (2012) determined degradation rate of LacI with attached SsrA LVA and SsrA-LAA at 25oC and 37oC and discovered that at 37 oC LacI was degraded 3-5 times faster.<SUP>9</SUP> To characterise SsrA degron mediated degradation dependence of temperature we performed a set of degradation rate determination experiments in four different temperatures (28⁰C, 32⁰C, 37⁰C and 42⁰C).</P><H4>Future applications, usefulness</H4><P>Looking from synthetic biology perspective SsrA tag system is a tool for controlling the protein lifetime. Protein instability can be advantageous for several reasons.</P><P>First of all, protein instability determines the maximum level of intracellular protein accumulation. Therefore, the amount of required proteins for optimized performance of genetic circuit (e.g., transcription factors, repressors, enzymes) can be produced by selecting the best combination of promoter, ribosomal binding site and degradation tag (e.g., SsrA tag). In addition, the protein overload within a cell can be reduced, which could be used to increase the overall stability of the genetic construct.</P><P>Secondly, controlled protein instability can be used to gain better temporal resolution of intracellular events. Fluorescent proteins (e.g., GFP, RFP, mCherry, YFP) are common and convenient reporters as they require only presence of oxygen for correct maturation and can be fused to any protein of interest without significant changes in protein properties.5 The only drawback of fluorescent proteins is their hyperstability, for example, the degradation rate for GFPmut3 has been estimated to be longer than 24 hours and the unfolding time around 20 years.<SUP>5,16</SUP> Andersen et al., (1998) attached 4 different SsrA tags and gained GFPmut3 variants with half-life ranging between  40 minutes and 2 hours.5 Thus SsrA tagged fluorescent proteins can be used to monitor dynamics of gene expression.<SUP>5</SUP> Similarly, engineered protein instability can help to build (and in some cases have already been used) synthetic constructs where timing of intracellular events is important (e.g., synthetic oscillators<SUP>24</SUP>, biosensors, bacterial computing and logic gates). However, for all these applications an optimal trade-off between maximum protein concentration (e.g., steady state fluorescence intensity) and temporal resolution must be achieved (See our characterisation results).</P><P>Considering all these advantages it is no surprise SsrA tag systems have been used in several iGEM projects generating new BioBricks (some examples summarized in Table 2) and in building new improved synthetic protein degradation regulation systems. One such example is split SspB system presented by Baker’s lab (Figure 6) and subsequently used for inspiration by iGEM13 team <A href="https://2013.igem.org/Team:Bonn">Bonn</A>.<SUP>25</SUP></P><H3>Concluding remarks</H3><P>For more information on SsrA tagging and degradation system see <A href="http://web.mit.edu/bakerlab/pages/tania_baker.html">Dr. Tania Baker Lab</A>, which specialises and studies Clp/Hsp100 ATPases.</P><H2>What did we do with them?</H2><P>In our project we decided to use SsrA degron tagged fluorescent protein (GFP, RFP) reporters to measure the activity of our metabolic wire components. We chose this reporter systems as fluorescence can be easily measured and SsrA degron provide a decent temporal resolution to distinguish how long our metabolic wire transcription factors are active or induced. In addition, as SsrA tag mediated proteolysis degrades all tagged substrates with very similar rate we hypothesised that we can regulate our metabolic wires with attaching similar tags. In our population control system, optimised turnover GFP and RFP signals can be used to monitor growth dynamics in two population E. coli cell mixture.</P><P>Different SsrA tags interact with ClpXP and ClpAP proteases with different affinities and leads to different rates of tagged substrate degradation. In order to find the optimal SsrA tag for our measurement purposes, we built a collection of SsrA tagged RFPs and GFPs. In total we used five different tag sequences found in Parts Registry<SUP>1</SUP> and reported in papers by Andersen et al.,(1998)<SUP>2</SUP> and McGinness et al.,2006<SUP>3</SUP>. These are listed in Table 1.</P><P>We constructed 9 <A href="https://2014.igem.org/Team:Edinburgh/project/biobricks">BioBricks</A> containing four SsrA-tagged RFP versions (RFP-AAV, RFP-LVA, RFP-LAA, RFP-DAS) and five GFP versions (GFP-AAV, GFP-LVA, GFP-LAA, GFP-DAS+2, GFP-DAS).</P><H3>Measurement method</H3><P>We then used recently developed DNA assembly method Paperclip<SUP>4</SUP> to construct measurement pathway(s) consisting of lactose/IPTG inducible promoter (BBa_R0010), ribosomal binding site (BBa_B0034), our constructed SsrA tagged fluorescent protein (FP-SsrA) and a terminator (BBa_B0015) (Biobricks BBa_K1399010 to BBa_K1399018. (Figure1) Similar construct with tetracyclin/aTc inducible promoter was also constructed for SsrA-tagged GFP variants (Biobricks BBa_K1399019 to BBa_K1399023).</P><P>To characterize the rate of SsrA-tagged RFP and GFP degradation we set up two separate experiments: fluorescence induction experiment and fluorescence decay experiment. In fluorescence induction experiment we applied IPTG to diluted overnight cultures and then monitored the increase of fluorescence over time using a microplate reader (FLUOstar Omega, BMG Labtech). The ‘raw’ data were then blank corrected and OD normalised using MARS software (BMG Labtech). An example of the data we got is in Figure 2. The untagged control for GFP and RFP was BBa_J04430 and our own Paperclip<SUP>4</SUP> assembled version of BBa_J04450 respectively. We did 4 replicas for each tagged protein and 3 replicas for untagged control.</P><P>From Fluoresce induction data we observed initial colinearity of the increase of fluorescence (around 1 h), which is consistent to observation that degradation tag does not interfere with promoter activity and protein production rate.</P><P>For the fluorescence decay experiments we induced SsrA-FP transcription for overnight cultures to obtain maximum fluorescence the following morning. Then we washed the cells and switched their medium to one without the inducer (IPTG) molecule and observed fluorescence decrease over time using a microplate reader (FLUOstar Omega, BMG Labtech). The ‘raw’ data were then blank corrected and OD normalised using MARS software (BMG Labtech). An example of the data we got is in Figure 3. Again, we did 4 replicas for each tagged protein and 3 replicas for untagged control. The untagged control for GFP and RFP was BBa_J04430 and our own Paperclip4 assembled version of BBa_J04450 respectively.</P><P>Data from fluorescence decay experiment describes the average fluorescence decrease per cell. This is due to SsrA mediated FP degradation and due to initial FP dilution among subsequent bacterial growth and division (the doubling time was around 80min).</P><H3>Data analysis and results</H3><P>SsrA tagged protein degradation dynamics are influenced by multiple factors including type of tag, protein stability, protease concentration and temperature. As we wanted to use fluorescence to monitor temporal dynamics of our metabolic wire promoter activity and the effectiveness of our population control, we decided to characterise the effect of environmental temperature (28⁰C, 32⁰C, 37⁰C and 42⁰C) on degradation activity as well as the range of fluorescence, we obtained.</P><P>As SsrA degradation tag (degron) cause protein to be susceptible to protease degradation as soon as it is translated, different tags will lead to various maximum and minimum fluorescence steady states within cell. We defined the minimum steady state of fluorescence per OD based on the plateau observed in our fluorescence decay experiment (time points of 4h40min and 5h in Figure 3). To obtain the induced maximum steady state of fluorescence per OD we used the observed plateau fluorescence OD ratio values from our fluorescence induction experiment. The data of minimum and maximum fluorescence per OD steady state for RFP and GFP is summarized in Figure 4 and Figure 5, respectively.</P><P>From these data it is clear to see that each tag provides a qualitative degradation rate that is kept the same in relation to other tags over the range of temperatures. Thus our characterised SsrA degrons can be ranked by their provided degradation rate as follows: the fastest are SsrA-LVA and Ssra-LAA tags followed by SsrA-AAV, than comes SsrA-DAS+2 and the least effective SsrA-DAS.</P><H3>Further data analysis and results</H3><P>We also attempted to quantify the rate of fluorescence/OD decay. According to previous studies fluorescence decrease fits exponential decay models.<SUP>2,5,6</SUP> In order to accomplish better comparison between different SsrA degradation tags, we transformed each data-set into relative fluorescence/OD values. We defined 100% as the maximum fluorescence/OD value extrapolated from fluorescence induction assay experiment, provided that it matches well to data from fluorescence decay experiment. The 100% value was then set as experimental value at time zero. Then relative fluorescence/OD data were added at their corresponding measured time. The corresponding measured time takes into account the time delay between change of culture media and the start of measurements caused by experiment technical issues (e.g., preparation for different stages of the experiment). An example of resulting RFP and GFP data appearance at 37⁰C is provided in Figure 6.</P><P>From these data fits we determined the fluorescence/OD decay rate and half-life. These values represent the SsrA caused degradation rate as well as initial fluorescence dilution among slowly growing cells. However, as our E. coli JM109 during the time course of this experiment showed linear growth, we concluded that this would not affect the rate constant. The obtained fluorescence/OD decay rates and half-lives are reported in tables 2 and 3, respectively.</P><P>From these data we concluded that the fluorescence/OD decay rate is significantly influenced by the temperature. We also observed that at 28⁰C fluorescence/OD decay rate did not match properly to exponential decay model. This could be explained by observation that SsrA-tagged protein degradation follows first order kinetics only when measured as population average, whereas single-cell and in vitro assays reveal zero-order kinetics.7 Thus we hypothesize that degradation rate differences at temperatures lower then optimum somehow influences the population average (e.g., narrows the range of cell-cell behaviour or influences cells for more extreme behaviour).</P><H3>Concluding remarks</H3><P>In this project we have constructed a collection of RFP and GFP variants fused with different SsrA degradation tags (degrons) and characterized them. We also have reviewed the biology behind this system and described several points of modifying and engineering this system to obtain more robust degradation. We thus conclude that different SsrA degrons can be used to effectively destabilise the required protein (GFP and RFP in our case). We also monitored that the environmental temperature is a significant factor influencing the degradation rate. There are several factors that influence the rate of SsrA-tag mediated protein degradation and our experiments monitored only the most accessible and the ones easiest to modify (i.e., type of tag and environmental temperature). For more detailed and robust determination of degradation rate we should move further and include the measurement of ClpXP, SspB and ClpAP concentrations within the cell or maybe supply cell with additional copies of these genes. Anyway we regard that SsrA degradation tag system is a very useful tool for protein destabilisation, and SsrA tagged fluorescent proteins provide tool for easier monitoring of cell dynamics with better time resolution.</P><H2 id="protocols">Protocols</H2><H3>Fluorescence induction over time</H3><H4>Materials</H4><UL><LI>Conical flasks or falcon tubes</LI><LI>0.2mM IPTG</LI><LI>Chloramphenicol (40µg/ml)</LI><LI>LB media (5ml per tube or per construct being assayed)</LI><LI>M9 minimal media (supplemented with glycerol and 0.5% Casamino acids) </LI><LI>Microplate Reader (FLUOstar Omega 3.00 R3, BMG Labtech)</LI></UL><H4>Protocol</H4><OL><SPAN style="text-decoration: underline;">Setting up overnight cultures</SPAN><LI>Prepare and label Falcon tubes</LI><LI>Add Chloramphenicol to LB media (40µg/ml)</LI><LI>Add 5ml of LB media into each tube</LI><LI>Take plates containing single colonies of E. coli JM109 with required constructs</LI><LI>Inoculate labelled tubes with single colonies from the correct plate with the matching label</LI><LI>Leave tubes in a shaker at 200rpm 37°C overnight (These cultures will be referred to as overnight cultures)</LI><SPAN style="text-decoration: underline;">Prepare M9 minimal media for the following morning</SPAN><LI>Followed protocol on http://openwetware.org/wiki/Endy:M9_medium/minimal</LI><LI>For carbon source supplement with glycerol and add 0.5% Casamino acids</LI><LI>Add chloramphenicol and IPTG</LI><LI>Store overnight in +4°C</LI><SPAN style="text-decoration: underline;">Assay</SPAN><LI>Take overnight cultures and wash in  PSB </LI><LI>Then re-suspend cells in M9 minimal media (with IPTG) and measure OD600</LI><LI>Dilute to OD 0.05 in M9 minimal media</LI><LI>Load microplate with 200 µl of diluted cells</LI><LI>Load plate into plate reader and set to take fluorescence and OD600 readings every 20 mins for 5 hours</LI></OL><H3>Fluorescence decay over time</H3><H4>Materials</H4><UL><LI>Conical flasks or falcon</LI><LI>0.2mM IPTG</LI><LI>Chloramphenicol (40µg/ml)</LI><LI>LB media (5ml per tube or per construct being assayed)</LI><LI>M9 minimal media (supplemented with glycerol and 0.5% Casamino acids) </LI><LI>Microplate Reader (FLUOstar Omega 3.00 R3, BMG Labtech)</LI></UL><H4>Protocol</H4><OL><SPAN style="text-decoration: underline;">Setting up overnight cultures</SPAN><LI>Prepare and label Falcon tubes</LI><LI>Add Chloramphenicol (40µg/ml) and IPTG (final concentration 0.2 mM) to LB media </LI><LI>Add 5ml of LB media into each tube</LI><LI>Take plates containing single colonies of E. coli JM109 with required constructs</LI><LI>Inoculate labelled tubes with single colonies from the correct plate with the matching label</LI><LI>Leave tubes in a shaker at 200rpm 37°C overnight (These cultures will be referred to as overnight cultures)</LI><SPAN style="text-decoration: underline;">Prepare M9 minimal media for the following morning</SPAN><LI>Followed protocol on http://openwetware.org/wiki/Endy:M9_medium/minimal</LI><LI>For carbon source supplement with glycerol and add 0.5% Casamino acids</LI><LI>Add chloramphenicol</LI><LI>Store overnight in +4°C
</LI><SPAN style="text-decoration: underline;">Assay</SPAN><LI>Take overnight cultures and wash in  PSB </LI><LI>Then re-suspend cells in M9 minimal media and measure OD600</LI><LI>Dilute to OD 0.05 in M9 minimal media</LI><LI>Load microplate with 200 µl of diluted cells</LI><LI>Load plate into plate reader and set to take fluorescence and OD600 readings every 20 mins for 5 hours</LI></OL><H2>References</H2><OL><LI>Keiler, K. C., Waller, P. R. &amp; Sauer, R. T. Role of a peptide tagging system in degradation of proteins synthesized from damaged messenger RNA. <EM>Science</EM> 271, 990–3 (1996).</LI><LI>Gottesman, S., Roche, E., Zhou, Y. &amp; Sauer, R. T. The ClpXP and ClpAP proteases degrade proteins with carboxy-terminal peptide tails added by the SsrA-tagging system. <EM>Genes Dev.</EM> 12, 1338–47 (1998).</LI><LI>Herman, C., Thévenet, D., Bouloc, P., Walker, G. C. &amp; D’Ari, R. Degradation of carboxy-terminal-tagged cytoplasmic proteins by the Escherichia coli protease HflB (FtsH). <EM>Genes Dev.</EM> 12, 1348–55 (1998).</LI><LI>Keiler, K. C. &amp; Sauer, R. T. Sequence determinants of C-terminal substrate recognition by the Tsp protease. <EM>J. Biol. Chem.</EM> 271, 2589–93 (1996).</LI><LI>Andersen, J. B. et al. New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria. <EM>Appl. Environ. Microbiol.</EM> 64, 2240–6 (1998).</LI><LI>Hersch, G. L., Baker, T. a &amp; Sauer, R. T. SspB delivery of substrates for ClpXP proteolysis probed by the design of improved degradation tags. <EM>Proc. Natl. Acad. Sci. U. S. A.</EM> 101, 12136–41 (2004).</LI><LI>McGinness, K. E., Baker, T. a &amp; Sauer, R. T. Engineering controllable protein degradation. <EM>Mol. Cell</EM> 22, 701–7 (2006).</LI><LI>Farrell, C. M., Grossman, A. D. &amp; Sauer, R. T. Cytoplasmic degradation of ssrA-tagged proteins. <EM>Mol. Microbiol.</EM> 57, 1750–61 (2005).</LI><LI>Purcell, O., Grierson, C. S., Bernardo, M. Di &amp; Savery, N. J. Temperature dependence of ssrA-tag mediated protein degradation. <EM>J. Biol. Eng.</EM> 6, 10 (2012).</LI><LI>Keseler, I. M. et al. EcoCyc: a comprehensive database of Escherichia coli biology. <EM>Nucleic Acids Res.</EM> 39, D583–90 (2011).</LI><LI>Chauhan, a K. &amp; Apirion, D. The gene for a small stable RNA (10Sa RNA) of Escherichia coli. <EM>Mol. Microbiol.</EM> 3, 1481–5 (1989).</LI><LI>Atkins, J. F. &amp; Gesteland, R. F. A case for trans translation. <EM>Nature</EM> 379, 769–71 (1996).</LI><LI>Keiler, K. C. Biology of trans-translation. <EM>Annu. Rev. Microbiol.</EM> 62, 133–51 (2008).</LI><LI>Miller, M. R. &amp; Buskirk, A. R. The SmpB C-terminal tail helps tmRNA to recognize and enter stalled ribosomes. <EM>Front. Microbiol.</EM> 5, 462 (2014).</LI><LI>Tu, G.-F., Reid, G. E., Zhang, J.-G., Moritz, R. L. &amp; Simpson, R. J. C-terminal Extension of Truncated Recombinant Proteins in Escherichia coli with a 10Sa RNA Decapeptide. <EM>J. Biol. Chem.</EM> 270, 9322–9326 (1995).</LI><LI>Kim, Y. I., Burton, R. E., Burton, B. M., Sauer, R. T. &amp; Baker, T. a. Dynamics of substrate denaturation and translocation by the ClpXP degradation machine. <EM>Mol. Cell</EM> 5, 639–48 (2000).</LI><LI>Flynn, J. M. et al. Overlapping recognition determinants within the ssrA degradation tag allow modulation of proteolysis. <EM>Proc. Natl. Acad. Sci. U. S. A.</EM> 98, 10584–9 (2001).</LI><LI>Wah, D. a et al. Flexible linkers leash the substrate binding domain of SspB to a peptide module that stabilizes delivery complexes with the AAA+ ClpXP protease. <EM>Mol. Cell</EM> 12, 355–63 (2003).</LI><LI>Burton, R. E., Siddiqui, S. M., Kim, Y. I., Baker, T. a &amp; Sauer, R. T. Effects of protein stability and structure on substrate processing by the ClpXP unfolding and degradation machine. <EM>EMBO J.</EM> 20, 3092–100 (2001).</LI><LI>Baker, T. a &amp; Sauer, R. T. ClpXP, an ATP-powered unfolding and protein-degradation machine. <EM>Biochim. Biophys.</EM> Acta 1823, 15–28 (2012).</LI><LI>Karzai, a W., Roche, E. D. &amp; Sauer, R. T. The SsrA-SmpB system for protein tagging, directed degradation and ribosome rescue. <EM>Nat. Struct. Biol.</EM> 7, 449–55 (2000).</LI><LI>Levchenko, I. A Specificity-Enhancing Factor for the ClpXP Degradation Machine. Science (80-. ). 289, 2354–2356 (2000).</LI><LI>Wah, D. a, Levchenko, I., Baker, T. a &amp; Sauer, R. T. Characterization of a specificity factor for an AAA+ ATPase: assembly of SspB dimers with ssrA-tagged proteins and the ClpX hexamer. <EM>Chem. Biol.</EM> 9, 1237–45 (2002).</LI><LI>Stricker, J. et al. A fast, robust and tunable synthetic gene oscillator. <EM>Nature</EM> 456, 516–9 (2008).</LI><LI>Davis, J. H., Baker, T. A. &amp; Sauer, R. T. Small-molecule control of protein degradation using split adaptors. <EM>ACS Chem. Biol.</EM> 6, 1205–13 (2011).</LI><OL><DIV id="tourleftbottom"><A href="https://2014.igem.org/Team:Edinburgh/zeigler">← The Zeigler report</A></DIV><DIV id="tourrightbottom"><A href="https://2014.igem.org/Team:Edinburgh/project/parts">Our BioBricks→</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Edinburgh/project/degrons">http://2014.igem.org/Team:Edinburgh/project/degrons</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/project/degrons" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/project/degrons" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/project/degrons&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/project/degrons&amp;oldid=400428" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></OL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>