<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_project_cisgenic"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/project/cisgenic">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/project/cisgenic&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/project/cisgenic&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/project/cisgenic&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/project/cisgenic" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/project/cisgenic</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="topbit" style="margin-bottom: 40px;"><TABLE align="center" style="border-spacing: 5px;"><TBODY><TR><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh">Home</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/team/">Team</A></TD><TD id="navlink"><A href="https://igem.org/Team.cgi?id=1399">Profile</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/logic/">Background</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/project/">Project</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/HP/">Policy and Practices</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/modelling/">Modelling</A></TD><TD id="navlink"><A href="https://2014.igem.org/Team:Edinburgh/log">Daily log</A></TD></TR></TBODY></TABLE></DIV><DIV id="tourleft"><A href="https://2014.igem.org/Team:Edinburgh/project/sugar">← Sugar logic</A></DIV><DIV id="tourright"><A href="https://2014.igem.org/Team:Edinburgh/project/population">Population regulation→</A></DIV><DIV id="igem"><DIV id="header"><H1>Cisgenic wires</H1>Yuma Shino
</DIV><DIV id="cisgenic"><P>This was intended to be constructed from genes only found in strains of <EM>E. coli</EM>. Similar to how it was done by Rafael Silva-Rocha and Victor de Lorenzo in their <A href="http://pubs.acs.org/doi/abs/10.1021/sb400064y">paper</A>, where they used genes found in <EM>Pseudomonas putida</EM> strains, we aimed to create a signalling system using aromatic degradation genes from <EM>E. coli</EM>.</P><P>Contrary to our initial expectations, metabolic pathways regulated by different substrate specific promoters were not as prevalent as claimed and it took a while to find one that satisfied the following requirements:</P><OL><LI>It was controlled by a substrate specific promoter.</LI><LI>The end product of the pathway was the substrate of another pathway, which was also controlled by a substrate specific promoter.</LI><LI>The product was either the co-activator or inducer of the other pathway.</LI><LI>The product of the pathway was freely diffusible.</LI></OL><P>Three pathways were, eventually, chosen to create this system. One pathway would act as a sender and two others were selected for the receiving strain in order to allow for possible failures of one of the other selected pathways. For this the aromatic amine degradation pathway was selected as a sender pathway.<SUP>1, 4, 5, 6</SUP> The two receiver pathways were the phenylacetate (PA) catabolic pathway1, 2, 6 and the 4-hydroxyphenylacetic (HPA) acid catabolic pathway.<SUP>1, 3, 6</SUP></P><P>The system was intended to signal via the transfer of either phenylacetic acid (PA) or 4-hydroxyphenylacetic acid (4HPA) between <EM>E.coli</EM> cells. The sender strain would have the aromatic amine degradation pathway and the receiver strain had the phenylacetate catabolic pathway or the 1-hydroxyphenylacetic acid catabolic pathway.</P><P>The substrates required for this were tyramine and phenylethylamine (PEA) which were converted into 4HPA and PA respectively by the aromatic amine degradation pathway.</P><P>The genes we used were:</P><UL><LI>FeaR</LI><LI>PaaX</LI><LI>HpaR</LI></UL><P>The above are transcription factors.</P><P>FeaR is an activator that binds with an unknown substrate of the aromatic amine degradation pathway.<SUP>1, 4, 5</SUP> The entire aromatic amine degradation pathway can be activated with the substrate PEA or tyramine.<SUP>1, 4, 5</SUP></P><P>PaaX is a repressor of the paa cluster and induced off with PA.1, 2, 6 
HpaR is one of the repressor of the hpa cluster which encodes the genes required for HPA degradation. It could be induced off with 4HPA.<SUP>1, 3, 6</SUP></P><P>The promoters we planned to use were:</P><UL><LI>PtynA</LI><LI>PpaaA</LI><LI>PhpaG</LI></UL><P>These were planned to be assembled with a reporter gene to show the signal had been transferred from the sender strain to the receiver strain. The promoter-RBS-GFP constructs for (Fig.1, 2, 3) were put pSB4C5 to ensure they were low copy, this was to ensure the reporter gene was silenced for the negatively controlled promoters. The repressors were put on pSB3K3 to avoid the possibility of over repression preventing our reporter gene from being expressed. The activator was put on pSB3K3 as well to prevent the possibility of it being overexpressed and binding non-specifically to the genomic DNA of our host possibly interfering with its genomic expression.  The strain of <EM>E. coli</EM> used was switched between MG1655 and BL21 (DE3) due to MG1655 having the phenylacetate catabolic pathway while BL21 (DE3) had the 1-hydroxyphenylacetic acid catabolic pathway.</P><H3>Knock out of degradation pathways to improve signalling</H3><P>As the products of the aromatic amine degradation pathway are often metabolised into the TCA cycle eventually, the idea of removing downstream pathways in the sender strain to increase the amount of signal produced was proposed. This meant any products of the aromatic amine degradation pathway would not be metabolised by the sender strain forcing them to be excreted into the media possibly increasing the concentration of signal produced.</P><P>Similarly the upstream pathway in receiver strains would have to be removed as well. Host strains to our reporter constructs had the aromatic amine degradation pathway. Without removing the upstream pathway to the phenylacetate catabolic pathway, or the 1-hydroxyphenylacetic acid catabolic pathway receiver strains could indirectly signal themselves by digesting the tyramine or PEA into HPA or PA which would activate our reporter construct.</P><P>A knock out cassette of the aromatic amine degradation pathway was designed by Dr. Joseph White of French Labs, MG1655 transformed with lambda red were donated by Dr. Chaokuo Liu of French labs. The lambda red system knockout protocol succeeded and individual colonies grew on Kanamycin plates.</P><H3>Promoter RBS Efficiencies</H3><P>In order to properly characterise the behaviour of a promoter the efficiency of the combination between promoter and RBS must be done. This can be done by observing the rate of expression of the promoter assembled in a plasmid with a different RBS attached to it at varying concentrations of inducer. Different combinations of promoters and RBSs have different rates of expression. A strong RBS may not always provide the strongest expression due to multiple reasons.<SUP>8, 9, 10, 11</SUP></P><P>Six different primers were ordered with the original RBS of the operon, RBS B0030-35, and B0064. These were already within the primers and by using different primers to assemble the constructs via the paperclip<SUP>7</SUP>.</P><P>Assembly attempts failed due to the incorrect concentration of DNA being used.</P><H3>Plasmid diagrams</H3><P>Diagrams of proposed test constructs.</P><P style="font-style: italic;">Fig. 1 Diagram of test construct of sender strain plasmids, RBS, reporter protein, Activator feaR, and Co-activator molecule Tyramine/PEA. Plasmids would be in the sender strain and convert Tyramine and PEA. GFP reporter gene would be in test version to indicate activation of the promoter had occurred. Actual test construct must have the GFP gene removed or replaced with alternative reporter gene to receiver strain.</P><P style="font-style: italic;">Fig. 2 Diagram of test construct of receiver strain plasmids, RBS, reporter protein, Repressor HpaR, and Inducer molecule HPA. Plasmids would be in the receiver strain and convert HPA to pyruvate and TCA cycle metabolites. GFP reporter gene would indicate whether the signal had been received by the receiver strain.</P><P style="font-style: italic;">Fig. 3 Diagram of test construct of receiver strain plasmids, RBS, reporter protein, Repressor PaaX, and Inducer molecule HPA. Plasmids would be in the receiver strain and convert PA to TCA cycle metabolites. GFP reporter gene would indicate whether the signal had been received by the receiver strain.</P><H3>Progression</H3><P>PCR of the genes from genomic DNA was successful, a sample of them can be seen in Fig.4. However, an optimization step was never done. Resulting DNA concentrations were less than 25ng/µl after purification. This caused assembly of the constructs in pSB1C3 for amplification and sequencing using paperclip technique to be difficult.</P><P>Fullparts for the required assemblies had been constructed with the relevant bands visible on PAGE gels as seen in Fig. 5.</P><P>Due to time constraints and insufficient training the gradient PCR required to optimise the PCR conditions was never carried out.</P><P style="font-style: italic;">Fig. 4 Photo of 1.5% Agarose gel wells contain; 50bp ladder PtynA PhpaG</P><P style="font-style: italic;">Fig. 5 Gel photo of Paperclip fullclips and ladders wells containted; 50bp Ladder, pSB1C3 FeaR, pSB1C3 PaaX, HpaR pSB1C3, FeaR pSB1C3, PaaX pSB1C3, pSB1C3 HpaR, pSB1C3 PtynA, pSB1C3 PpaaA, pSB1C3 PhpaG, PtynA pSB1C3, PpaaA pSB1C3, PhpaG with nascent RBS pSB1C3, pSB1C3 Pcons(B0034), 50bp Ladder.</P><H3>References</H3><OL><LI>	Dı́az, E., Ferrández, A., Prieto, M. a. A., &amp; Garcı́a, J. L. (2001). Biodegradation of Aromatic Compounds byEscherichia coli. <EM>Microbiology and Molecular Biology Reviews</EM>, 65(4), 523-569. doi: 10.1128/mmbr.65.4.523-569.2001
</LI><LI>Galán, B., García, J. L., &amp; Prieto, M. A. (2004). The PaaX Repressor, a Link between Penicillin G Acylase and the Phenylacetyl-Coenzyme A Catabolon of Escherichia coli <EM>W. Journal of Bacteriology</EM>, 186(7), 2215-2220. doi: 10.1128/jb.186.7.2215-2220.2004
</LI><LI>Galán, B., Kolb, A., Sanz, J. M., García, J. L., &amp; Prieto, M. A. (2003). Molecular determinants of the hpa regulatory system of Escherichia coli: the HpaR repressor. <EM>Nucleic Acids Research</EM>, 31(22), 6598-6609. doi: 10.1093/nar/gkg851</LI><LI>Hanlon, S. P., Hill, T. K., Flavell, M. A., Stringfellow, J. M., &amp; Cooper, R. A. (1997). 2-Phenylethylamine catabolism by Escherichia coli K-12: gene organization and expression. <EM>Microbiology</EM>, 143(2), 513-518. doi: 10.1099/00221287-143-2-513</LI><LI>Yamashita, M., Azakami, H., Yokoro, N., Roh, J. H., Suzuki, H., Kumagai, H., &amp; Murooka, Y. (1996). maoB, a gene that encodes a positive regulator of the monoamine oxidase gene (maoA) in Escherichia coli. <EM>Journal of Bacteriology</EM>, 178(10), 2941-2947.</LI><LI>Zeng, J., &amp; Spiro, S. (2013). Finely Tuned Regulation of the Aromatic Amine Degradation Pathway in Escherichia coli. <EM>Journal of Bacteriology</EM>, 195(22), 5141-5150. doi: 10.1128/jb.00837-13</LI><LI>Trubitsyna, M., Michlewski, G., Cai, Y., Elfick, A., &amp; French, C. E. (2014). PaperClip: rapid multi-part DNA assembly from existing libraries. <EM>Nucleic Acids Research</EM>. doi: 10.1093/nar/gku829 </LI><LI>Tan, C., Marguet, P., &amp; You, L. (2009). Emergent bistability by a growth-modulating positive feedback circuit. <EM>Nat Chem Biol</EM>, 5(11), 842-848. doi: 10.1038/nchembio.218</LI><LI>Klumpp, S., Zhang, Z., &amp; Hwa, T. (2009). Growth Rate-Dependent Global Effects on Gene Expression in Bacteria. <EM>Cell</EM>, 139(7), 1366-1375. doi: http://dx.doi.org/10.1016/j.cell.2009.12.001</LI><LI>Salis, H. M., Mirsky, E. A., &amp; Voigt, C. A. (2009). Automated design of synthetic ribosome binding sites to control protein expression. <EM>Nat Biotech</EM>, 27(10), 946-950. doi: http://www.nature.com/nbt/journal/v27/n10/suppinfo/nbt.1568_S1.html</LI><LI>Davis, J. H., Rubin, A. J., &amp; Sauer, R. T. (2011). Design, construction and characterization of a set of insulated bacterial promoters. <EM>Nucleic Acids Research</EM>, 39(3), 1131-1141. doi: 10.1093/nar/gkq810</LI></OL></DIV><DIV id="tourleftbottom"><A href="https://2014.igem.org/Team:Edinburgh/project/sugar">← Sugar logic</A></DIV><DIV id="tourrightbottom"><A href="https://2014.igem.org/Team:Edinburgh/project/degrons">Degrons →</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Edinburgh/project/cisgenic">http://2014.igem.org/Team:Edinburgh/project/cisgenic</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/project/cisgenic" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/project/cisgenic" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/project/cisgenic&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/project/cisgenic&amp;oldid=400389" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>