<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ATOMS-Turkiye_Testing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ATOMS-Turkiye/Testing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ATOMS-Turkiye/Testing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Testing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Testing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ATOMS-Turkiye/Testing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ATOMS-Turkiye/Testing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><A href="/wiki/index.php?title=Team:Imperial_College_London/Templates/Header&amp;action=edit&amp;redlink=1" class="new" title="Team:Imperial College London/Templates/Header (page does not exist)">Team:Imperial College London/Templates/Header</A><A href="/wiki/index.php?title=Team:Imperial_College_London/Templates/Chemotaxis&amp;action=edit&amp;redlink=1" class="new" title="Team:Imperial College London/Templates/Chemotaxis (page does not exist)">Team:Imperial College London/Templates/Chemotaxis</A><A href="/wiki/index.php?title=Team:Imperial_College_London/Templates/Accordion&amp;action=edit&amp;redlink=1" class="new" title="Team:Imperial College London/Templates/Accordion (page does not exist)">Team:Imperial College London/Templates/Accordion</A></P><H1>Testing</H1><P style="font-size:1.3em;"><A href="#" onclick="ddaccordion.collapseall('technology'); return false">Collapse all</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false">Expand all</A></P><DIV class="technology">1. Introduction</DIV><DIV class="thelanguage"><P>The assembled PA2652 construct (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>) and non-codon optimised mcpS (in pRK415 backone vector), have been inserted and tested for functionality in <I>E. coli</I> DH5α obtained from New England Biolabs. We carried out several tests in an attempt to characterise the rewired chemotaxis towards L(-)malic acid. We separated testing of chemotaxis towards malate into behavioural, qualitative &amp; quantitative analyses. </P><P> To test bacterial uptake into the roots of the plants, we worked with <I>Arabidopsis thaliana</I> to replicate the experiment by Paungfoo-Lonhienne et al<SUP>[6]</SUP>. <I>Arabidopsis </I> is a common plant model organism. Its genome has been almost completely sequenced and replicates quickly, producing a large number of seeds. Many different mutant strains have been constructed to study its different characteristics<SUP>[1]</SUP>. While <I>Arabidopsis</I> may not represent plant populations naturally occurring in arid areas threatened by desertification, it is a useful model organism which we will be using to study the effect of the auxin plant hormone, indole-3-acetic acid (IAA) on roots, observe chemotaxis towards them, and observe them taking up bacteria as nutrients.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 0); return false">Collapse</A></P></DIV><DIV class="technology">2. Bacterial chemotaxis towards malate</DIV><DIV class="thelanguage"><H3>2.1 Behavioral analysis</H3><P>Bacteria perform two types of movement, smooth swimming and tumbling. In the absence of attractant the result of the two movements lead to random walk without directionality. In the presence of a concentration gradient of attractant, the probabilities of the two movements change and lead to the directional movement known as random biased walk.</P><DIV class="vidbox"><TABLE><TBODY><TR><TD><P><I>Video 1: Mixed population of GFP-expressing control and non-GFP labelled PA2652 expressing </I>E. coli<I>.</I><I> The bacteria have been placed in motility buffer without attractant and therefore there is no observed difference in behaviour between the two. Since no attractant is present, a majority of the bacteria perform tumbling, however there are individual bacteria that perform smooth swimming, a rapid movement. In this video, bacteria decide randomly between tumbling and smooth swimming, which is known as random walk. When bacteria perform chemotactic behaviour towards a concentration gradient(not pictured), the probability of the bacteria performing smooth swimming increases after a transient decrease due to saturation of chemoreceptors. This video has been taken as a set of frames, with one frame per four seconds. Imaging done by Imperial iGEM 2011.</I></P></TD></TR></TBODY></TABLE></DIV><P>During bacterial movement up a concentration gradient of attractant, the probability of the smooth swimming is higher than that of tumbling. Smooth swimming is a fast, uni-directional movement, whereas tumbling is random and although the speed may not be slow, overall velocity is far less than that of smooth swimming. Due to the complicated assay set up with a concentration gradient, we changed strategy to look for uniformity of the bacterial movement. We expected the cells, which are capable of malate recognition to show much more uniform response than those that do not. The bacteria were grown to mid-exponential phase (OD<SUB>600</SUB> 0.4-0.6) before being induced in motility buffer 2 hours prior to observation. Bacteria containing construct PA2652 (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>) have been exposed to motility buffer (reference), 10 mM serine (positive control) &amp; 10 mM malate (test). <I>E. coli</I> DH5α without any construct have been exposed to 10 mM malate (negative control). Observations were taken with a Zeiss Axiovert 200 Inverted Fluorescent Microscope and video collection Volocity software. ImageJ plug-in Manual Tracking was used to collect data and Chemotaxis Tool plug-in to analyze the data.</P><DIV class="imgbox" style="width:800px;margin:0 auto;"><P><I>Figure 1: Probability density function of bacterial number at observed velocities. PA2652 cells exposed to 10 mM malate are more than 90% likely to be moving at just over 2 μm/s. PA2652 cells that were exposed to serine were 90% likely to be moving at a velocity just over 2 μm/s. PA2652 cells that were not exposed to attractant were over 70% likely to be moving at 2 μm/s. Cells without BBa_K515102 construct were less than 50% likely to be moving at a velocity between 2 and 4 μm/s. Data depicts difference in response between PA2652 cells, which were and which were not exposed to an attractant. Also cells without construct show lack of uniform response when exposed to 10 mM malate. Data collected by Imperial iGEM 2011.</I></P></DIV><P>From the data analysis it seems that the bacteria with construct BBa_K515102, when in 10 mM malate perform a very uniform behaviour. This is also confirmed by positive control cells exposed to 10 mM serine, where the response of cells is also highly uniform. Cells with construct PA2652 without exposure to saturating attractant show less uniform movement than PA2652 cells, whether exposed to malate or serine. Also negative control cells fail to show uniformity in their movement suggesting inability to recognise the saturating medium containing 10 mM malate and therefore, performing their movement randomly. </P><H3>2.2 Qualitative analysis</H3><P>Qualitative assays were done to observe the effect of rewiring chemotaxis in <I>E. coli</I> with the engineered constructs PA2652 and mcpS (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>). </P><P>A number of methods exist that show chemotaxis towards a source <SUP>[2]</SUP>. Most of them are based on the properties of semi-solid agar, which allows diffusion of molecules and bacterial movement. We have modified an agar plug assay, which involves plating bacteria at opposite end of attractant on a petri-dish, to observe chemotaxis. The bacteria used were in mid-exponential phase (OD<SUB>600</SUB> 0.4-0.6). Cells suspended in the semi-solid agar were positioned 2 cm away from the attractant source and left overnight to grow. At the end of the assay the plates were imaged using Fujifilm LAS-3000 Imager.</P><P>This assay was done to show the functional chemotaxis of <I> E. coli</I> DH5α transformed with our PA2652 construct (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>) compared to the inability of control <I> E. coli</I> DH5α to do the same. Positive control cells were exposed to increasing concentrations of serine, which is recognised by the endogenous chemoreceptors of <I>E. coli</I>, to observe what the movement we are looking for in cells with our construct. We have also tested cells containing the non-codon optimised mcpS gene for qualitative analysis. </P><P>Results from this assay should show clear differences in the shape of colonies formed. Bacteria attracted to the source move and distort the colony into an elliptical, directed shape towards the source. We expect the control colonies to look circular because bacteria are equally likely to swim into any direction.</P><P><I>Table 1: The concentrations of attractant tested</I></P><TABLE border="1" style="margin-left:20px;"><TBODY><TR><TD><P>Molar range</P></TD><TD><P>0 mM</P></TD><TD><P>0.01 mM</P></TD><TD><P> 0.1 mM</P></TD><TD><P>1 mM</P></TD><TD><P> 10 mM</P></TD><TD><P>100 mM</P></TD></TR><TR><TD><P>Milimolar range</P></TD><TD><P>0 mM</P></TD><TD><P> 5 mM</P></TD><TD><P>10 mM</P></TD><TD><P>15 mM</P></TD><TD><P> 20 mM</P></TD><TD><P> 25 mM</P></TD></TR></TBODY></TABLE><P style="text-align:center;"><B>Positive control</B></P><DIV class="imgbox" style="width:920px;"><P><I>Figure 2: Increasing concentrations of serine were tested. a) 0 mM control - circular colony b) 0.01 mM - circular colony c) 0.1 mM – rendered void due to mis-handling with semi-solid agar  d) 1 mM - circular colony e) 10 mM - elliptical colony f) 100 mM - elliptical colony away from the attractant due to saturation. Data collected by Imperial iGEM 2011.</I></P></DIV><DIV class="imgbox" style="width:920px;"><P><I>Figure 3: Increasing concentrations of serine were tested. a) 0 mM control - circular colony, b)5 mM - elliptical colony c) 10 mM - elliptical colony d) 15 mM - elliptical colony e) 20 mM - elliptical colony f) 25mM - elliptical colony.Data collected by Imperial iGEM 2011.</I></P></DIV><P style="text-align:center;"><B>Negative control</B></P><DIV class="imgbox" style="width:920px;"><P><I>Figure 4: Increasing concentrations of malate were tested. a) 0 mM control - circular colony b) 0.01 mM - circular colony c) 0.1 mM - circular colony d) 1 mM - circular colony e) 10 mM - circular colony f) 100 mM - circular colony. Data collected by Imperial iGEM 2011.</I></P></DIV><DIV class="imgbox" style="width:920px;"><P><I>Figure 5: Increasing concentrations of malate were tested. a) 0 M control - circular colony b)5 mM - circular colony c) 10 mM - circular colony d) 15 mM - circular colony e) 20 mM - circular colony, f) 25mM - circular colony. Data collected by Imperial iGEM 2011.</I></P></DIV><P style="text-align:center;"><B>McpS - pRK415</B></P><DIV class="imgbox" style="width:920px;"><P><I>Figure 6: Increasing concentrations of malate were tested. a) 0 mM control - circular colony b) 0.01 mM - circular colony c) 0.1 mM - circular colony d) 1 mM - circular colony e) 10 mM - circular colony f) 100 mM - circular colony. Data collected by Imperial iGEM 2011.</I></P></DIV><DIV class="imgbox" style="width:920px;"><P><I>Figure 7: Increasing concentrations of malate were tested. a) 0 mM control - circular colony b)5 mM - circular colony c) 10 mM - elliptical colony formed not in the direction expected d) 15 mM - possible elliptical colony e) 20 mM - possible elliptical colony f) 25 mM - circular colony. Data collected by Imperial iGEM 2011.</I></P></DIV><P style="text-align:center;"><B>PA2652 - <A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A></B></P><DIV class="imgbox" style="width:920px;"><P><I>Figure 8: Increasing concentrations of malate were tested. a) 0 M control - circular colony b) 0.01 mM - possible elliptical colony the shape is hard to analyze c) 0.1 mM - strange shape of colony observed, we cannot explain this phenomenon d) 1 mM - colony shape is not perfectly circular however not elliptical either e) 10 mM - circular colony f) 100 mM - circular colony. Data collected by Imperial iGEM 2011.</I></P></DIV><DIV class="imgbox" style="width:920px;"><P><I>Figure 9: Icreasing concentrations of serine were tested. a) 0 mM control - circular colony b)5 mM - colony shape was rendered void due to mishandling with semi-solid agar c) 10 mM - possible elliptical shape colony d) 15 mM - circular colony e) 20 mM - colony shape was rendered void due to mishandling with semi - solid agar f) 25mM - circular colony. Data collected by Imperial iGEM 2011.</I></P></DIV><P> Upon analysis of the data, a definitive conclusion could not be drawn.  This is due to a number of factors in the setup of the assay. Attractant is localised, and it diffuses in all directions, however the plate is finite and therefore loss of concentration gradient occurs over time. Another factor is the semi-solid agar itself, a medium which is very difficult to manipulate. Although the results were vague, a number of points can be drawn. Positive controls have shown that <I>E. coli DH5α</I> are capable of chemotaxis. They have also shown that when the added serine concentration is too high (10<SUP>-1</SUP> M), bacteria do not swim directly towards the attractant source because the surrounding medium is saturated. Instead they perform chemotaxis in a direction towards which the attractant concentration does not saturate the chemoreceptors. This movement still occurs regardless of the fact that the bacteria are moving away from the attractant source. Negative controls have shown that at any tested concentration of L(-)malic acid, <I>E. coli DH5α</I> without PA2652 or mcpS construct do not perform chemotaxis towards attractant. We could not conclude a result for the mcpS or PA2652 construct with this assay since the colonies observed have a range of shapes at different attractant concentrations.</P><H2>2.3 Quantitative Analysis </H2><DIV class="imgbox" style="width:420px; float:right;"><P><I>Figure 10: Outline of the sample setup for the capillary assay. Diagram by Imperial iGEM 2011.</I></P></DIV><P> In comparison to qualitative assays, quantitative assays are more informative as they provide cell count based on different attractant concentrations and therefore allow identification of the optimal attractant concentration for chemotaxis. Our analyses were based on the high-throughput capillary assay<SUP>[3]</SUP>. We wanted to perform data collection using two different cell count methods. One was the traditional CFU (colony forming unit) count, which allows us to normalise the data to a number of cells chemotaxing per mililitre.

</P><P>The other method was based on modern flow cytometry using BD FACScan, which allows us to refer to number of cells per capillary. We tried to use both methods, however due to the high sensitivity of the flow cytometer and the small size of <I>E. coli</I> cells, it was difficult to identify the cells among the particles of the same size which occured in the flow count. Due to this fact we used CFU count for quantification. The assay itself is based on a number of capillary tubes filled with different concentrations of attractant placed into bacterial suspension for a period of 60 minutes. During this period, bacteria containing the PA2652 construct (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>) are expected to swim up the capillary and the cell count can be measured. When performing the capillary assay, the capillaries with different attractant concentrations were exposed to bacterial suspensions containing 6x10<SUP>8</SUP> cells/mL.</P><P><I>Table 2: The concentrations of tested malate attractant.</I></P><TABLE border="1" style="margin-left:20px;"><TBODY><TR><TD><P>Test concentrations</P></TD><TD><P>0 mM</P></TD><TD><P>0.0001 mM</P></TD><TD><P>0.01 mM</P></TD><TD><P>1 mM</P></TD><TD><P>100 mM</P></TD></TR></TBODY></TABLE><DIV class="imgbox" style="width:920px;"><P><I>Figure 11: Dependence of bacterial chemotaxis to varied malate concetrations. Cells containing PA2652 (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>) have shown inreased number of cells in capillaries with inreasing malate concentration, with a peak at 1 mM. The number of cells drops sharply after 1 mM due to saturation. Negative control were cells without contruct. The cell count for negative control in each of the capillaries with increasing attractant concentration has not increased. Data collected by Imperial iGEM 2011.</I></P></DIV><P><B>Figure 11</B> shows that <I>E. coli DH5α</I> with the construct PA2652 is capable of chemotactic response towards L(-)malic acid. Due to the quantitative nature of the assay, we can also see the extent to which cells perform chemotaxis towards a chemoattractant. The highest response towards attractant can be seen at 1 mM malate concentration. Also we can conclude that bacteria without construct are not capable of performing chemotaxis towards malate as the cell count of the control did not increase at any of tested malate concentrations. This shows the functionality of our construct (<A href="http://partsregistry.org/Part:BBa_K515102"><B>BBa_K515102</B></A>).</P><DIV class="imgbox" style="width:420px;float:right;"><P><I>Figure 12: BD FACScan could be used to improve data collection from the capillary assay. (Image by Imperial College London iGEM team 2011).</I></P></DIV><P> After confirming that our construct works, we want to test whether our <I>E. coli</I> cells can respond to concentrations of malate released by the roots. The concentration of malate released from the root depends on soil acidity, which in turn is linked to soil NO<SUB>3</SUB><SUP>-</SUP> levels. In alkaline soils (low in NO<SUB>3</SUB><SUP>-</SUP>) plants have unbalanced excess cation uptake<SUP>[4]</SUP>. In this case the concentration of malate released from the roots is 10<SUP>-6</SUP> M. In neutral soils, where plants have balanced cation/anion or in acidic soils, where plants have unbalanced excess anion uptake, the concentration of malate secreted is lower<SUP>[5]</SUP>.</P><P>We are currently testing the chemotactic response of our chassis to physiologically relevant malate concentration levels below 10<SUP>-6</SUP> M. To obtain these results we have reassessed the data collection method based on flow cytometry, because this should speed up the process of data collection and accuracy. To distinguish our bacteria from the background particles we have cotransformed our chassis with a low copy plasmid pTAC1 containing GFP, so our cells can be uniquely visualised using flow cytometer even if their size is similar to the size of backround particles. Hopefully this new strategy will produce results for presentation in Boston.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 1); return false">Collapse</A></P></DIV><DIV class="technology">3. Uptake of bacteria into roots</DIV><DIV class="thelanguage"><DIV class="imgbox" style="width:370px; float:right;margin-top:10px;"><P><I>Figure 13: GFP-expressing </I>E. coli<I> cells inside </I>Arabidopsis<I> roots (data and imaging by Imperial College iGEM 2011).</I></P></DIV><P>One important part of our project is the uptake of our bacteria into plant roots. The observation that this occurs in both <I>Arabidopsis</I> and tomatoes and that both <I>E. coli</I> and the yeast <I>Saccharomyces cerevisiae</I> can be taken up by roots (albeit under controlled lab settings) is new and was only published last year<SUP>[4]</SUP>. 

</P><P>As the amount of IAA needed for enhancing plant growth depends on whether our bacteria are producing the compound outside or inside the plants, we attempted to replicate these findings.</P><P>In preparation for confocal imaging, we met with Dr Martin Spitaler and Mark Scott who advised us on how to prepare samples and image them. Confocal microscopy is much more precise than conventional light field microscopy as it eliminates background light by focusing the laser through a pinhole (Mark Scott, oral communication). The confocal microscopy we conducted was focused on imaging GFP expressing bacteria inside <I>Arabidopsis</I> roots to show that uptake of the bacteria takes place. </P><P>In an imaging trial run we found that natural fluorescence can be measured in roots in a spectrum that does not interfere with measuring the superfolder GFP expressed by our <I>E. coli</I> cells. In addition, the autofluorescence is strong enough to enable us to identify individual cells <B>(Video 2)</B>. We did therefore not need to dye the roots before imaging.</P><DIV class="imgbox" style="width:590px;margin:0 auto;"><P><I>Video 2: Stack of wt </I>Arabidopsis<I> root. The root can be imaged at around 488 nm. Imaging carried out by Dr Martin Spitaler for Imperial College team iGEM 2011.</I></P></DIV><P>In order to prepare the GFP-expressing bacteria for plant uptake, they were spun down and media was exchanged prior to incubation at 37°C to reach exponential phase. Bacteria were then spun down and resuspended in wash buffer (5 mM MES) to reach OD 30. 8 ml, 4 ml and 2 ml were added to separate flasks, containing 100 ml of half-MS media each. 4 ml and 6 ml of wash buffer were added to the flasks containing 4 ml and 2 ml bacteria, respectively. 8 ml of wash buffer was added to the negative control. Ten <I>Arabidopsis</I> seedlings were distributed into each of the flasks. Incubation was carried out for 15 hours prior to imaging.</P><P>Prior to imaging, roots were washed in PBS to wash off bacteria and facilitate imaging. We imaged the plants incubated with 8 ml of bacteria and were able to find bacteria inside one of the roots. A 3D picture was taken of uninfected roots and roots containing bacteria by taking a Z stack image using confocal microscopy <B>(Video 3)</B>. </P><DIV class="imgbox" style="width:590px;margin:0 auto;"><P><I>Video 3: Bacteria inside of </I>Arabidopsis thaliana<I> roots. These videos were put together using Z-stack images taken on a confocal microscope. These images can be converted into 3D pictures that allow us to verify that the bacteria can indeed be found inside the roots rather than on the surface (data and imaging by Imperial College iGEM 2011).</I></P></DIV><P>We repeated this experiment at a later date with plants that had been allowed to grow for a longer period of time. The bacteria were predominantly found in root hairs and inside of cells on the root surface <B>(Figure 11)</B>. These plants were older than the ones previously used for uptake experiments and bacterial uptake was much more predominant.</P><P>These results show that we would be able to expose plants to IAA from inside the roots themselves. This is extremely important for assessing the secretion of IAA by our bacteria. Nevertheless, we will need to conduct further uptake experiments in soil and with other plant species to confirm that uptake would be possible in the project's eventual implementation stage.

</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 2); return false">Collapse</A></P></DIV><DIV class="technology">4. References</DIV><DIV class="thelanguage"><P>[1] National Institute of Health. (no date) <I>Model Organisms for Biomedical Research.</I> [Online]. Available from: http://www.nih.gov/science/models/arabidopsis/index.html [Accessed 21<SUP>st</SUP> September 2011].</P><P>[2] Jain RK and Pandey J (2010) Chemotactic responses. Timmis VKN, ed. Handbook of hydrocarbon and lipid microbiology. Berlin, Heidelberg: Springer Berlin Heidelberg, pp. 3933-3955. Available at: http://www.springerlink.com/content/x458521h8420l478/ [Cited September 16, 2011].</P><P>[3] Bainer R, Park H and Cluzel P (2003) A high-throughput capillary assay for bacterial chemotaxis. <I>Journal of Microbiological Methods</I><B>55:</B> 315-319.</P><P>[4] Stumpf DK and Burris RH (1981) Organic Acid Contents of Soybean: Age and Source of Nitrogen. <I>Plant Physiology</I><B>68:</B> 989-991.</P><P>[5] Martinoia E and Rentsch D (1994) Malate Compartmentation - Responses to a Complex Metabolism. <I>Annual Review of Plant Physiology and Plant Molecular Biology</I><B>45:</B> 447-467.</P><P>[6] Paungfoo-Lonhienne C et al. (2010) Turning the table: plants consume microbes as a source of nutrients.<I>PLoS One</I><B>5(7):</B> e11915. </P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 4); return false">Collapse</A></P></DIV><H2><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Assembly" style="text-decoration:none;color:#728F1D;float:left;">
M1: Assembly
</A><A href="https://2011.igem.org/Team:Imperial_College_London/Project_Chemotaxis_Future" style="text-decoration:none;color:#728F1D;float:right;">
M1: Future Work
</A></H2><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ATOMS-Turkiye/Testing">http://2014.igem.org/Team:ATOMS-Turkiye/Testing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ATOMS-Turkiye/Testing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ATOMS-Turkiye/Testing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Testing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Testing&amp;oldid=321970" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>