<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ATOMS-Turkiye_Results1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ATOMS-Turkiye/Results1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ATOMS-Turkiye/Results1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Results1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Results1&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ATOMS-Turkiye/Results1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ATOMS-Turkiye/Results1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="menuContent" style=""><NAV class="menuNav"><CENTER><UL><LI><A href="#center">Project</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/At-a-Glance">At A Glance</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Background-Problem">Background/Problem</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Project-Approach">Approach</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/BioBricks">BioBricks</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Future-Plan">Future Plan</A></LI></UL><LI><A href="#center">P&amp;P</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:ATOMS-Turkiye/Practices-Approach">Approach</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M1-Identification">M1-Identification</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M2-Clarification">M2-Clarification</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M3-Specialization">M3-Specialization</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M4-Application">M4-Application</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Outreach">Outreach</A></LI><LI><A href="#center">Safety</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Safety-Approach">Safety Approach</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Form">Safety Form</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Safety-Modeling">Safety Modeling</A></LI><LI><A href="#center">Team</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Who-We-Are">Who We Are</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Collaboration">Collaboration</A></LI><LI><A href="https://igem.org/Team.cgi?id=1456">Official Team Profile</A></LI><LI><A href="#center">Extras</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Interlab-Study">Interlab Study</A></LI><LI><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2014&amp;group=ATOMS-Turkiye">Official Parts Page</A></LI></CENTER></NAV></DIV><DIV class="asoContent"><CENTER><P style="font-size:1.3em;color: rgb(192,0,0) !important;  text-shadow: 0px 0px 1px #000000 !important;        };"><A href="#" onclick="ddaccordion.collapseall('technology'); return false" style="
    color: rgb(192,0,0) !important;  text-shadow: 0px 0px 1px #000000 !important;  margin: 12px !important;  font-size: 18px !important;  text-decoration: none;  };
">[Collapse all]</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false" style="
    color: rgb(192,0,0) !important;  text-shadow: 0px 0px 1px #000000 !important;  margin: 12px !important;  font-size: 18px !important;  text-decoration: none;  };
">[Expand all]</A></P></CENTER><DIV class="technology">Cloning</DIV><DIV class="thelanguage"><LI>Here is our results page, you can analyze our constructed vectors for coding proteins and the agarose gel electrophoresis result of inserts ligated with interest vectors. TetR-VP16 double plasmid system is used in our project as an empowering system of expressing.  
</LI><LI>For Aprotinin, GPx, SOD and tPA; 
</LI><LI>For ODD;
</LI><LI>For HRE and KB regulator genes; 
</LI><LI>This system includes few understructured elements called TetR-VP16 complex and two different plasmids, pTet-off and pTRE vectors. In the pTet-off plasmid, PCMV constitutive promoter codes for TetR-VP16 protein complex in medium strength which can bind to its responding element present in the second plasmid, pTRE. Tetracycline respond element (TetRE) is a protein binding domain which enables the binding of TetR component of the protein complex. Whereas, VP16 component works as a transcription activator for the weak constitutive promoter (PminiCMV) which exists in the pTRE vector. TetR-VP16 protein complex can activate this weak promoter in pTRE vector.
</LI><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 0); return false">[Collapse]</A></P></DIV><H1>Sensing</H1><DIV class="technology">1. HRE</DIV><DIV class="thelanguage"><LI>Hypoxia response elements (HREs) are 234 bp long small DNA sequences present in most of the body cells which work as binding domains for hypoxia inducible factor-1α (HIF-1α), a common transcription factor found in our body cells released during hypoxic conditions. (Semenza et al. 1992)</LI><UL><LI>We aimed to demonstrate its functionality by inserting it into pTRE-Luc vector.</LI><LI>We expect that HRE, as an enhancer, would activate the promoter existing on the downstream region of it, depending on the level of HIF1alfa in the media which is increased in hypoxic conditions. </LI></UL><LI>The design of our vector possessing our part is shown below.</LI><LI>We started our experiments by cloning our parts. The forward primer is designed to produce HRE sequence inserted before the CMV promoter sequence presenting on pTRE vector. Reverse primer is synthesized to produce the cloning region of the vector. HRE-CMV sequence is cloned from pTRE vector via performing PCR. </LI><LI>The PCR product is purified with phenol chloroform method and, afterwards, is cut with XhoI and BamHI restriction enzymes. The product is ligated with pTRE-Luc vector been cut with same enzymes and been treated with Antarctic phosphatase. The ligation product is inserted in DH5alfa strain and we performed colony PCR from the plate. </LI><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>To understand which colony our gene is inserted among the colonies that we transformated pHRE-luciferase vector, we expected the picture above when we perform PCR when we use pTRE-Luc forward and MCS reverse primers.
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>From the samples we perform colony PCR by using pTRE-Luc forward and MCS reverse primers, we obtained a band in 428 bp line. This image proves that our HRE sequnce is inserted into the vector, successfully.
</P></DIV></TD></TR></TBODY></TABLE><H4>Luciferase Assay</H4><LI>The vectors isolated from the colonies we identified correct are co-transfected  into HEK293 and HepG2 cell lines. Transfected cells are incubated in hypoxic and normoxic conditions and the luminescence levels are measured by performing luciferase assay. To conduct hypoxic condition, we used 100 uM CoCl2. Here, the graphics we obtained after the measurement of luminescence is shown below.</LI><LI>According to the results, in hypoxic conditions comparing with normoxia, HRE been inserted into pTRE-luciferase vector shows 7 times more activity by producing more luciferase in HepG2 cell line. In HEK293 cell line, this activity is measured in hypoxia 1,5 times more than normoxia. This results prove that HRE sequence improves the production rate of desired protein in hypoxia comparing with normal oxygen levels.</LI><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 1); return false">[Collapse]</A></P></DIV><DIV class="technology">2. KB</DIV><DIV class="thelanguage"><P>NF-kappaB (NF-kB) proteins comprise a family of structurally-related eukaryotic transcription factors that are involved in the control of a large number of normal cellular and organismal processes, such as immune and inflammatory responses, developmental processes, cellular growth, and apoptosis. In some circumstances, NF-kB/IkB complex can be separated by external effects such as radiation, cellular stress, pathogens, inflammation etc. In this case, NF-kB can enter into nucleus and integrate with compatible kB-RE sites in order to initiate transcription.</P><P>We cloned kB-RE and inserted it into the downstream region of CMV mini promoter as it’s shown above.</P><P>NF-kappaB (NF-kB) was synthesized to GenScript™ company and it came in pUC57 plasmid. We digested it with BamHI &amp; PstI and exposed it to Antarctic phosphatase.  </P><P>Afterwards, we purified our part via phenol chloroform method. We performed the same procedures onto the pTRE-luciferase vector. Eventually, we ligated them. </P><P>We inserted our plasmid (pTRE-luciferase kB-RE) into DH5α strain and performed colony PCR by using CMV forward and kB reverse primers. At the end of this experiment, we expected a band seen in the 20-30 bp line. </P><P>   And we observed correct bands in the expected region.</P><DIV class="withImage" style="float:center;width620px"><P>(p&lt;0,05 and p&lt;0,01 for H2O2 and TNFα induced kappaB-luc,respectively.)</P></DIV><LI>At the end of the experiment,  we could not see an increase in the luciferase assay of cells exposed to H202. We assumed that cell death due to the direct exposal of H2O2 was what limited our luciferase activity expectations. Our assumptions were further supported after the microscope analysis we performed, which showed that our cells were unattached from the plates upon being exposed to H202.</LI>

.


<H3>Reference</H3><LI>1.	 Zhang Y., Yang X., Bian F., et al. TNF-α promotes early atherosclerosis by increasing transcytosis of LDL across endothelial cells: Crosstalk between NF-κB and PPAR-γ. Journal of Molecular and Cellular Cardiology 72 (2014) 85–94</LI><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 2); return false">[Collapse]</A></P></DIV><DIV class="technology">3. ODD</DIV><DIV class="thelanguage"><P><STRONG>ODD SYNTHESİS FROM HEP2G VİA PCR </STRONG></P><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P></DIV></TD><TD><DIV><P><B>Experimented</B></P></DIV></TD></TR></TBODY></TABLE><P>ODD (Oxygen Dependent Degredation) domain of HIF-1α was synthesized through liver cDNA using PCR with Sall enzyme restriction cites placed at the starting and ending points of the domain.</P><P>The PCR product was purified using the Phenol Chloroform method. Following the isolation, the ODD and pTet-Off vector were cut using the Sall restriction enzyme and then ligated. Thus, the ODD insert was placed in between the tetR(DNA binding domain) and VP16(Transactivating domain) of the pTet-Off vector.</P><P><STRONG>COLONY PCR</STRONG></P><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P></DIV></TD><TD><DIV><P><B>Experimented</B></P></DIV></TD></TR></TBODY></TABLE><P>The DH-5α E.coli strains were transformated and, using CMV forward and SV40 polyA reverse primers, colony PCR was conducted and the vectors, in which the inserts were placed, were elected.</P><P><STRONG>CUT-CHECK</STRONG></P><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P></DIV></TD><TD><DIV><P><B>Experimented</B></P></DIV></TD></TR></TBODY></TABLE><P>Since the ODD insert’s contained the same restriction cite on both ends, the colonies that entered the sequence from the right end were cut-checked using EcoRI and BamHI restriction enzymes and the colony containing the desired vector was selected.</P><H4>ODD Results</H4><P>ODD (oxygen dependent degradation) domain, which is present in the HIF-1α (hypoxia inducible factor) protein that it activates in lower oxygen levels and breaks down in intermediate oxygen levels, is a protein domain that plays a key regulatory role in the transcription of the HIF-1α factor. In intermediate oxygen levels, the ODD domain of the HIF-1α protein is hydroxilized by the hydroxilase enzyme and the hydroxilized HIF-1α enzyme breaks down through ubiquitin attachment. Thus, the ODD causes the regulation of a transcription factor, which is active in hypoxic conditions and inactive in normoxic conditions. The Tet-Off is a strong system composed of two strong plasmids. Of the two plasmids that form this system, the TetR-VP16 fusion protein produced by the first plasmid acts as a transcriptionary factor regulating the Tet operator sequence of the second plasmid(pTRE). (TetR: DNA Binding Domain, DBD; VP16: Transactivating domain, TAD).</P><P>The Tet-Off system can be inhibited using tetracyclane.</P><P>In this study, through placing the ODD region of the HIF-1α in between the synthetic TetR - VP16 transcription factors (which are not present in mammallian cells and have been used in molecular biology experiments for a long time), the transcription factor was designed to gain sensitivity to oxygen. The therapeutic genes in the pTRE genes can be synthesized as sensitive to the hypoxic  conditions, controlled by the TetR-ODD-VP16 transcription factor.
</P><P><STRONG>Luciferase Assay</STRONG></P><P>To control the functionality of the TetR-ODD-VP16 system, the HEK 293T and Hep G2 cell lines were cotransfected using the pTET Off-ODD and pTRE-Luc vectors.</P><P>100 µM of CoCl2 was added to the cell medium to establish 1% O2 in the medium. The cells were collected 9 hours later. Luminometric measurement under Thermo Varioscan for 613 nm was done and the data of the following graph were acquired:</P><P>	The observation of the operational level of the TetR-ODD-VP16 did not yield fruitful results as the cells reached out of the dish when 100 µM of CoCl2 were added to HEK 293T medium.</P><P>	When the Tet Off-ODD and pTRE-Luc vectors were cotranfected to Hep G2 cells in hypoxic medium, there was a 4 times increase in Luciferase concentration in respect to normoxic medium.</P><P>	In the light of these results, it is possible to say that the Tet-ODD-VP16 system was successfully synthesized and a novel hypoxia inducible system was introduced for future use in studies indulged into examining hypoxic conditions.</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 3); return false">[Collapse]</A></P></DIV><H1>Therapy</H1><DIV class="technology">4. tPA</DIV><DIV class="thelanguage"><P><B>Tissue plasminogen activator</B> (abbreviated <B>tPA</B> or <B>PLAT</B>) is a protein involved in the breakdown of blood clots. It is a serine protease (EC 3.4.21.68) found in endothelial cells that can be secreted into the plasma as well as the cells that line the blood vessels. As an enzyme, it catalyzes the conversion of plasminogen to plasmin, the major enzyme responsible for clot breakdown. Due to its ability of working on the clotting system, tPA is used in clinical medicine to treat embolic or thrombotic stroke. However, use is contraindicated in hemorrhagic stroke and head trauma. 

</P><P>We questioned the best possible was of measuring the tPA enzyme activity and sought the answer to this question by examining the yield of a tPA catalyzed reaction. We searched the sector and discovered the Human tPA Activity Kit of the company ASSAYPRO which we then ordered to use. In the reaction which catalyzes tPA, our aim was to show that tPA was active when plasminogen was transformed to plasmine.

</P><P>To acquire the tPA gene, the tPA forward and tPA revers primers were synthesized from the cDNA’s we were in possession of. Using these primers, we acquired the tPA genes by performing the PCR of the cDNA. The head and neck cancer cell line was used as the source for cDNA.

</P><H2>GENE SYNTHESIS FROM cDNA OF 64A CELL LINE
</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The base length of tPA is 1762 bp’dir. 
The electrophoresis of the PCR was expected to show a base length of around 1700 bp. The image below shows the expected result of the electrophoresis.
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>Through the primers that we ordered and the cDNA, we acquired tPA inserts. The insert was, as expected, portraying that the base length was around 1700 bp.
</P></DIV></TD></TR></TBODY></TABLE><H2>CLONING CONTROL-1
</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>We used EcoRI and BamHI enzymes to cut the inserts and pTRE vectors which was followed by the ligation of our insert and vector using the ligation process. We then transformed our plasmids to
The E. coli’nin DH5-α strains. Colony PCR was then conducted to verify the accuracy of our ligation.

</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>The specified primers were put into colony PCR. As it can be seen in the results above, a right insert was not achieved. This process was repeated several times but no result was achieved. Seeing that ligation did not provide a solution to the problem, synthetically produced inserts were ordered.

</P></DIV></TD></TR></TBODY></TABLE><P>The tPA gene which was ligated with the synthetically synthesized Puc57 vector was cut using the EcorI and BamHI restriction enzymes and then ligated with pTRE vector to be transformed into the DH5-α strain. We again used Colony PCR to control the accuracy of our ligation.

</P><H2>CLONING CONTROL-2</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The received genes were ligated with the pTRE vector, and then transformated to E.coli strands DH5-α. To test the validity of the transformation, we again applied colony PCR through CMV forward and SV40 reverse primers. The expected base length was 1984 bp.</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>Colony PCR was applied to the inserts acquired from transformation and ligation. As it can be seen in the figure above, the second colony contains the appropriate base length in respect to the ladder.</P></DIV></TD></TR></TBODY></TABLE><H2>WESTERN BLOTTİNG
</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The confirmed genes were transfected into HEK293 cells. The lysates acquired from the cells were run through Western Blot. tPA, that is known to have 63 kD, was expected to have the image on the left.
</P><P>http://www.emdmillipore.com/TR/en/product/Anti-tPA-%28Tissue-Plasminogen-Activator%29-Antibody%2C-clone-GMA-043,MM_NF-05-883
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>The bands in the Western Blot were accurate. The transfection of the inserts were verified and assays of tPA were prepared from the lysates.
</P></DIV></TD></TR></TBODY></TABLE><P>After having proven the presence of tPA expression in the cells transfected with Western Blotting, the concentration of the amount of tPA in the cell and the amount of secreted tPA were measured using the ‘Human tPA Activity Kit’ to show that the expressed proteins are functionally active.

</P><P>The real parameter of the measurement in the Assay was the product of the reaction of the plasmin enzyme. Since tPA shifts the inactive plasminogen to active plasmine, the measured value also presents tPA activity. The yield gives absorbance at 405 nm.

</P><H2>tPA ASSAY</H2><H3>Experimented</H3><P>According to the tPA assay results we obtained, we were able to prove that the tPA enzyme could be produced and secreted from the cell successfully hence also showing that tPA is  functionally active.  

</P><P style="text-align:right;font-size:1.3em;"><A href="#" class="collapseLink" onclick="ddaccordion.collapseone('technology', 4); return false">[Collapse]</A></P></DIV><DIV class="technology">5. Aprotinin</DIV><DIV class="thelanguage"><P>Aprotinin(Bovine Pancrease Tripsine Inhibitor) is a serine protease which inhibits 80% of the formation of superoxide by blocking the transformation of xanhtine dehidrogenase into xanthine oksidase.</P><P>Aprotinin and pTRE vector was digested using EcoRI ve BamHI enzymes following by their ligation which cloned aprotinin into the pTRE vector. The isolated plasmid from the bacteria was then cut-checked using  EcoRI ve BamHI  enzymes and the following results were achieved:</P><H3>Aprotinin Assay</H3><P>	As Aprotinin is a serine protease inhibitor aprotinin activity can be measured using the correlation of serine protease inhibition the aprotinin inhibits.</P><P>HEK 293T cell was transfected with 1 µg pTRE-Aprotinin and cells were obtained after 36 hours which continued with the isolation of proteins. Afterwards, the isolated protein was placed into a tube containing the serine protease and its activity was measured. </P><P>The graphic shows that serine protease inhibition was not observed in the isolated proteins of the cells transfected with the pTRE vectors and our negative controls only.</P><P>The inhibition values obtained by adding 6 mg/dl Bovine Aprotinin(sigmaA1153) samples obtained by cattles and the cotransfected Tet off – pTRE Aprotinin, show that the transfected HEK 293 T cells perform a lower amount of inhibition compared to our standard sample however that is a very minimal difference.  As a result, our transfected aprotinin has accomplished more inhibition than our negative control and has proved that it can pe produced functionally. Afterwards the HEK 293 T cells cotransfected with Tet off – pTRE Aprotinin have been transfected in different amounts to characterise and measure its serine protease activity. </P><H3>Characterisation</H3><P>HEK 293T cells  have been cotransfected with 1 µg, 2 µg, 3µg ve 4 µg Tet off – pTRE Aprotinin. The cells collected and protein isolation was performed to measure the serine protease inhibition. According to the positive control test, these results Show dose-dependent reduction and the production of aprotinin has increased according to the amount of cells successfully transfected.</P><P style="text-align:right;font-size:1.3em;"><A href="" class="collapseLink" onclick="ddaccordion.collapseone('technology', 5); return false">[Collapse]</A></P></DIV><DIV class="technology">6. SOD</DIV><DIV class="thelanguage"><H2>GENE SYNTHESIS FROM HEPG2 CELL LINE cDNA

</H2><P>SOD is a powerful and essential antioxidant enzyme which converts and scavenges free radicals and reactive oxygen species (ROS) into hydrogen peroxide (H2O2), a compound slightly less harmful,in order to convey it to the second step reaction which detoxifies H2O2 into water. 
Once cloning our SOD gene and placing into the pTRE  luciferase vector, we expect to see the expression of  this clone in the eukaryotic HEK(Human Embryonic Kidney) 293 T cell line using the western blotting technique. SOD assay procedure will be used to measure the expression of our SOD gene hence proving that our SOD enzyme is functioning as expected. Below are the results of each strep taken until the functional assay:
</P><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected

</B></P><P>The gel electrophoresis above  shows the expected result of the amplified SOD1 gene using the CMW forward and SV40 reverse primers we obtained from the HEP2G cell line. According to the results , we aim to see a band at 526bp.

</P></DIV></TD><TD><DIV><P><B>Experimented
</B></P><P>We amplified the SOD1 genes using CMV forward and SV40 reverse primers and achieved results by viewing band at 526bp. 

</P></DIV></TD></TR></TBODY></TABLE><P>cDNA obtained from HEP 2G cells have been used to perform PCR to synthesize our desired inserts. EcoRI and BamHI were the choice of enzymes to digest our inserts once their purification was accomplished using phenol chloroform method. Transformation using the DH5α strain was the next step after  ligating our inserts with the pTRE vectors. Colony PCR procedure was conducted using the CMV forward and SV40 reverse primers to ensure that our inserts were cloned correctly. Below are the results of our Colony pcr procedure:
</P><H2>COLONY PCR

</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected
</B></P><P>Upon transforming,  in order to ensure that our insert is placed correctly ,we expect to see the above results using the colony pcr technique. 
We decided to use CMV  forward and  SV40 reverse primers for pcr.
</P></DIV></TD><TD><DIV><P><B>Experimented
</B></P><P>According to the forth colony,
our results show that  sod1-ptre vector  has been inserted correctly.
</P></DIV></TD></TR></TBODY></TABLE><P>In order to prove that our cloning has been carried out correctly we used the restriction-digest process and achieved the expected results.
We cotransfected our vector with HEK 293 T cell line. After incubating our cells we performed western blotting and controlled the cell expression of our SOD1 genes. Below is our western blotting results:

</P><H2>WESTERN BLOTTING 


</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected
</B></P><P><A href="http://www.emdmillipore.com/TR/en/product/Anti-CuZn-SOD-Antibody,MM_NF-07-403">(Link for picture)</A></P></DIV></TD><TD><DIV><P><B>Experimented
</B></P><P>Our western Blotting results show a band at 18kDa meaning that our HEK 293 T cell line has been successful. 
</P></DIV></TD></TR></TBODY></TABLE><P>Functional assay of SOD was performed to test whether our enzyme was functioning as expected.</P><H3>Characterisation</H3><P>Dosage levels were adjusted whilst SOD- Ptre vector was transfected to HEK293 cell.

1.	125ng/24-well  gene transfected to 105 cells present in well. 
2.	250ng/24-well  gene transfected to 105 cells present in well.
3.	500ng/24-well  gene transfected to 105 cells present in well.
The results show that the SOD1 gene is effected by the adjustement of dosage levels which in return proves that as the amount of transfected genes increase the gene expression also increases
</P><H3>SOD ASSAY</H3><P>Superoxide Dismutase 1 changes the oxygen radicals into 02 and H2O2. Using this feature of SOD we placed the oxygen radicals into the same solution as tetrazolium dye which uses the radicals as substrates and enabled them to compete. Hence this allowed us to measure the functional activity of SOD produced by our transfected cells using the spectrophotometer at 560 nm.                                                                                                                                            
</P><P>Transfected Tet off - pTRE and Tet off - pTRE-SOD1 vectors were subjected to SOD assay and measured using the spectrophotometer at 560 nm. Cells which have the Tet off – pTRE SOD1 gene present can degrade the oxygen radicals 21 times faster. SOD assay was performed for the proteins obtained from the transfected SOD1 visible colour change could be seen.
</P><P style="text-align:right;font-size:1.3em;"><A href="" class="collapseLink" onclick="ddaccordion.collapseone('technology', 6); return false">[Collapse]</A></P></DIV><DIV class="technology">7. GPX</DIV><DIV class="thelanguage"><P><B>GPx is a known antioxidant enzyme abundant in human body cells which drives the H2O2 scavenge into water.
</B></P><P>Glutathione (GSH) is a peptide consisting of three amino acid residues, glutamate, cysteine and glycine, respectively. It is a natural cellular component for preventing damage to organelles caused by reactive oxygen species, free radicals or peroxides. 
</P><P>In cell, GSH appears in different components or biochemical structures. GSH is a cofactor for GPx enzyme that the thiol group of cysteine in GSH is able to donate a reducing equivalent (H++ e−) to other unstable molecules, such as reactive oxygen species. In donating an electron, GSH itself becomes reactive, but readily reacts with another reactive GSH to form glutathione disulfide. (GSSG)
</P><P>GSH can be regenerated from GSSG by the enzyme glutathione reductase (GSR) which uses NADPH cofactor. The scheme of reduction cascade can be seen in figure X:
</P><P>In literature, GPx-1 can be overexpressed in transfected cells and produced synthetically. <B>This recombinant enzyme is shown to work against hydrogen peroxide, lipid peroxide and drugs interfering with redox cycle</B></P><H2>GENE SYNTHESIS FROM cDNA OF 64A CELL LİNE
</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The length of the GPx gene is 649 bp. Therefore, the result of the electrophoresis procedure was expected to show a base pair range in between 500-750 bp.
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>GPx inserts were acquired from the ordered genes and cDNA. The results showed the expected range of 500-750 bp.
</P></DIV></TD></TR></TBODY></TABLE><H2>CLONING CONTROL-1

</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The acquired inserts were ligated to the pTRE vector and transformated to E.coli strands DH-5α. The validity of the transformation was again checked through colony PCR using CMV forward and GPx reverse primers. The expected base length was around 781 bp.

</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>Colony PCR was conducted using the specified primers. As it can be seen on the figure, the results were not accurately fitting the base pair range mentioned. The experiment was repeated several times, and after unsuccessful attempts, the parts were ordered for artificial synthesis.

</P></DIV></TD></TR></TBODY></TABLE><H2>CLONING CONTROL-2


</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The ordered genes were ligated with the pTRE vector and then transformated into E.coli strands DH-5α . The validity of the transformation was again checked through colony PCR using CMV forward and GPx reverse primers. The expected base length was around 871 bp.


</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>The ligated and transformated samples were put into colony PCR. As it can be seen on the figure, the right inserts were located in colonies 7,8 and 10.


</P></DIV></TD></TR></TBODY></TABLE><H2>WESTERN BLOTTİNG

</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The genes that the ligation was verified was transfected into HEK293 cells. To show that the transfection had been successfully completed, the proteins isolated from the cells were put under Western Blotting. The expected result for GPx, which was known to be 23 kD, was as the figure on the left.

</P><P>http://www.abcam.com/glutathione-peroxidase-1-antibody-ab22604.html
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>Sadly, the results achieved did not fit the ideal Blotting results.

</P></DIV></TD></TR></TBODY></TABLE><H2>WESTERN BLOTTİNG


</H2><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>http://www.abcam.com/glutathione-peroxidase-1-antibody-ab22604.html
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>Selenium was necessary for the production of the selenocysteine amino acid present in GPx but was not present in the medium of the culture. It was expected to have observed a result such as the figure on the left through adding Na2SeO3 to the cell line. Expression was seen; there was a directly proportional increase in GPx production in respect to selenoite increase(100 uM-200 uM-500 uM-1000 uM).

</P></DIV></TD></TR></TBODY></TABLE><TABLE class="blueborder"><TBODY><TR><TD><DIV><P><B>Expected</B></P><P>The genes that the ligation was verified was transfected into HEK293 cells. To show that the transfection had been successfully completed, the proteins isolated from the cells were put under Western Blotting. The expected result for GPx, which was known to be 23 kD, was as the figure on the left.
</P><P>http://www.abcam.com/glutathione-peroxidase-1-antibody-ab22604.html
</P><P>http://www.abcam.com/glutathione-peroxidase-1-antibody-ab22604.html
</P></DIV></TD><TD><DIV><P><B>Experimented</B></P><P>The bands acquired through the Western Blotting were fitting the expected results. The cell lines, that now had their transfections verified, were purfied for the preparation of GPx assays.

</P></DIV></TD></TR></TBODY></TABLE><P>The amounts of protein acquired were as the charts above and the these data were taken into consideration in the preparation of the assays.
</P><P>A functional assay of GPx was done through NADPH to measure GPx activity. The same amount of genes were transferred to all samples but varying amount of Na2SeO3 was prepared in the samples, with 2 samples containing different Na2SeO3 concentrations and one control group. The GPx gene was not transfected to the control group. From these 3 groups, the assay of the control group gave the lowest yield as expected; the sample of 500 nM Na2SeO3 produced the mid-result. The sample of 1000 nM Na2SeO3 yielded the highest activity results in the assay. These results prove that the GPx production is directly proportional to the amount of added Na2Se2O3 and that the GPx enzyme was produced functionally.
</P><P style="text-align:right;font-size:1.3em;"><A href="" class="collapseLink" onclick="ddaccordion.collapseone('technology', 7); return false">[Collapse]</A></P></DIV><DIV class="technology">Expressional Check of Therapeutical Genes</DIV><DIV class="thelanguage"><DIV class="justImage"><DIV class="justImage"><LI>His tagged Aprotinin (BPTI), SOD1, GPX1 and PLAT (tPA) was overexpressed in HEK 293T cells. 36 hours post transfection cells were collected and subjected to western blotting. Aprotinin, SOD1 and tPA were expressed. However, expression of GPX1 could not be observed. Previously, we realized that GPx-1 is a member of seleno-cystein  proteins. To code seleno-cystein GPx has a internal stop codon. Expression in the absense of seleno-cystein GPX-1 expression  stops at the internal stop codon  thus, the his-tag at the and of GPx-1 can not be expressed. Therefore,we couldn’t see  any band in westernblotting. Next , we planned to add sodiumselenite to induce production of seleno-cystein.  
</LI><DIV class="justImage"><LI>This is comassive blue flourorange view of blotted membrane.

</LI><DIV class="justImage"><LI>The expression level of beta actin is the same among samples.


</LI><DIV class="justImage"><LI>This is comassive blue flourorange view of blotted membrane.

</LI><DIV class="justImage"><LI>We over expressed SOD-1 and tPA with different  doses of SOD-1 and tPA expression vector. SOD-1 well responded to the increasing doses. tPA  gave the same response to all different doses. To induce expression of GPx-1 the cells treated with different doses of sodiumselenite and GPx-1 proportionally response to the increasing doses of sodiumselenite with single dose of expression vector.  


</LI><P style="text-align:right;font-size:1.3em;"><A href="" class="collapseLink" onclick="ddaccordion.collapseone('technology', 14); return false">[Collapse]</A></P></DIV><DIV><CENTER><P style="font-size:1.3em;color: rgb(192,0,0) !important;  text-shadow: 0px 0px 1px #000000 !important;        };"><A href="#" onclick="ddaccordion.collapseall('technology'); return false" style="
    color: rgb(192,0,0) !important;  text-shadow: 0px 0px 1px #000000 !important;  margin: 12px !important;  font-size: 18px !important;  text-decoration: none;  };
">[Collapse all]</A>  | <A href="#" onclick="ddaccordion.expandall('technology'); return false" style="
    color: rgb(192,0,0) !important;  text-shadow: 0px 0px 1px #000000 !important;  margin: 12px !important;  font-size: 18px !important;  text-decoration: none;  };
">[Expand all]</A></P></CENTER></DIV><DIV class="navLR"><A class="navLinkLeft" href="https://2014.igem.org/Team:ATOMS-Turkiye/Modeling">Modeling</A><A class="navLinkRight" href="https://2014.igem.org/Team:ATOMS-Turkiye/Data">Data</A></DIV></DIV><DIV id="footDiv"><NAV class="footNav"><UL><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/At-a-Glance">At A Glance</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Background-Problem">Background/Problem</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Project-Approach">Approach</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/BioBricks">BioBricks</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Modelling">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Data">Data</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Future-Plan">Future Plan</A></LI></UL><LI><A href="#">P&amp;P</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:ATOMS-Turkiye/Practices-Approach">Approach</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M1-Identification">M1-Identification</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M2-Clarification">M2-Clarification</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M3-Specialization">M3-Specialization</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/M4-Application">M4-Application</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Outreach">Outreach</A></LI><LI><A href="#">Safety</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Safety-Approach">Safety Approach</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Form">Safety Form</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Safety-Modeling">Safety Modeling</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Who-We-Are">Who We Are</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Collaboration">Collaboration</A></LI><LI><A href="https://igem.org/Team.cgi?id=1456">Official Team Profile</A></LI><LI><A href="#">Extras</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Interlab-Study">Interlab Study</A></LI><LI><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2014&amp;group=ATOMS-Turkiye">Official Parts Page</A></LI><LI><A href="#">Other Links</A></LI><LI><A href="https://igem.org/Login">Log in</A></LI><LI><A href="https://2014.igem.org/Special:Upload">Upload File</A></LI><LI><A href="https://2014.igem.org/Main_Page">iGEM Main Page</A></LI><LI><A href="http://www.facebook.com/AtomsTurkiye">ATOMS Facebook Page</A></LI><LI><A href="http://twitter.com/Atoms_Turkiye">ATOMS Twitter Page</A></LI></NAV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ATOMS-Turkiye/Results1">http://2014.igem.org/Team:ATOMS-Turkiye/Results1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ATOMS-Turkiye/Results1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ATOMS-Turkiye/Results1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Results1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Results1&amp;oldid=406735" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>