<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USTC_Component"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:USTC/Component">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USTC/Component&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC/Component&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC/Component&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USTC/Component" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USTC/Component</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#FFFFFF;background-color:#66B3FF;" cellpadding="1" cellspacing="2" border="3" width="870px" align="center"><TBODY><TR><TH align="center"><A href="/Team:USTC" title="Team:USTC">Home</A></TH><TH align="center"><A href="/Team:USTC/Team" title="Team:USTC/Team">The Team</A></TH><TH align="center"><A href="/Team:USTC/Project" title="Team:USTC/Project">The Project</A></TH><TH align="center"><STRONG class="selflink">Components</STRONG></TH><TH align="center"><A href="/Team:USTC/Results" title="Team:USTC/Results">Results</A></TH><TH align="center"><A href="/Team:USTC/Parts" title="Team:USTC/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:USTC/Notebook" title="Team:USTC/Notebook">Notebook</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Differetiation_Components"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Differetiation Components</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_cell_density_of_the_colony_on_the_plate"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The cell density of the colony on the plate</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Quorum_Sensing_System"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Quorum Sensing System</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#1..09K082035"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">1.	K082035</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#2..09K082021"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">2.	K082021</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#3..09K082020"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">3.	K082020</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#4._K082045"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">4.       K082045</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Cross-talk"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Cross-talk</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#NOR_gate_.26.26_Cre_recombinase"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">NOR gate &amp;&amp; Cre recombinase</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#NOR_Gate"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">NOR Gate</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Recombinase"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Recombinase</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Report_Components"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Report    Components</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Multi-regulatory_system"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Multi-regulatory system</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#XOR_GATE"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">XOR GATE</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Computational_Protein_Design"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Computational Protein Design</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#References"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Differetiation_Components">Differetiation Components</SPAN></H2><P>
We design a time-dependent gene regulation system to construct a self-organized system. The RhlI/RhlR system is employed to report the cell-population density. The sender cell will appear first when the density reaches a certain level. And it will tell other cells around it to be the receiver cells by sending the ligands into the environment. After receiving the ligands, the Cre recombinase will be expressed to &quot;scissor&quot; the function of sending ligands. In this way, a positional differentiation will be achieved. 
</P><H3><SPAN class="mw-headline" id="The_cell_density_of_the_colony_on_the_plate">The cell density of the colony on the plate</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig 1 .The changes of cell relative density related to the distance at different time points</DIV></DIV></DIV><P>
Cell Automata is employed to mimic the states of a colony growing on the plate. The first seed is placed on the center of the table. Each seed has a child in the next generation, and the child seed is randomly putted into the grid next to its parent. It is the simplest simulation, but it can suggest how the bacteria grow on the plate when the resources are affluent. Fig 1 shows the changes of cell relative density related to distance at different time points.And the result shows that 1) the colony can be divided into two parts, the central region and the peripheral region, according to the density of cells; 2) the central region will expand during the growth of the colony; 3) the difference between the peripheral region and the central region is evident enough to be sensed by the concentration of signal molecules.
</P><H3><SPAN class="mw-headline" id="Quorum_Sensing_System">Quorum Sensing System</SPAN></H3><P>RhlI/RhlR, one type of quorum sensing systems, is used as messengers to inform the cell of the local environment where it lives. RhlI is the enzyme that catalyzes the synthesis of C4HSL and RhlR is the receptor of C4HSL. If the concentration of C4HSL reaches a certain level, the RhlR will elevate the expression level of downstream genes of RhlR promoter to a great extent, which marks the initiation of differentiation. There are three genes downstream: RhlI, LuxI and LacI. RhlI can upregulate the expression of downstream genes by forming a positive feedback cycle. The LuxI gene is transcribed to synthesize the signal molecule 3OC6HSL, sending the differentiation order to other cells. However, LacI acts against AHL inhibiting the expression of differentiation executor. 
In order to build up the system, we have designed and constructed several protein-coding units
as follows, and more will be done in the next step.
</P><H4><SPAN class="mw-headline" id="1..09K082035">1.	K082035</SPAN></H4><P>
In this sequence, we ligated the ptetR, RBS, RhlI and terminator together, so as to produce the C4HSL constitutively. The sequencing result and the electrophoresis both demonstrate that the sequence of this unit is correct. Moreover, although it is too difficult for us to directly test whether RhlI can generate C4HSL as we could not obtain the purified C4HSL molecule, we constructed another sequence comprised by ptetR, RBS, RhlI, RBS, GFP and terminator. The green fluorescence we have observed confirmed the normal expression of RhlI indirectly.
</P><H4><SPAN class="mw-headline" id="2..09K082021">   2.	K082021</SPAN></H4><P>
The sequence codes for RhlR protein and the protein is constitutively expressed.
</P><H4><SPAN class="mw-headline" id="3..09K082020">   3.	K082020</SPAN></H4><P>
Similarly, the sequence expresses the LuxR protein constitutively.
</P><H4><SPAN class="mw-headline" id="4._K082045">   4.       K082045</SPAN></H4><P>
This is the initiation part of differentiation in our project. The sequence codes for RhlI, LacI and LuxI and expresses them at a high level only when there are both RhlR and C4HSL present. In our design, we hope the concentration of C4HSL increases to a certain amount but not too much, so a weaker RBS is used. Moreover, the proteins Lac repressor and AHL are also generated.
</P><H4><SPAN class="mw-headline" id="Cross-talk">Cross-talk</SPAN></H4><P>In our project, we need to use two types of signal transduction systems, and we choose to utilize the RhlI/RhlR and LuxI/LuxR systems. Our selection is based on the proof (reference #10) that the two kinds of systems do not exchange the acyl-homoserine lactone signals. 
</P><H3><SPAN class="mw-headline" id="NOR_gate_.26.26_Cre_recombinase">NOR gate &amp;&amp; Cre recombinase</SPAN></H3><H4><SPAN class="mw-headline" id="NOR_Gate">NOR Gate</SPAN></H4><P>The executor part of differentiation is the Cre recombinase that is regulated by the NOR gate. The designed NOR gate is regulated by two proteins: LacI and LuxR. NOR gate will only be turned on with AHL present and LacI absent. Here the logical gate is utilized to distinguish whether the signal molecule is endogenic or ectogenic. At the beginning, the expression will be inhibited by LuxR. Once LuxI is expressed, AHL will be synthesized and spread throughout the colony. If AHL is endogenic, LacI will be expressed simultaneously so that the executor is not activated. But if AHL is ectogenic, before the concentration of LacI in the cell reaches a sufficient level, the NOR gate is turned on and the executor will be expressed, leading to the shutting off of the AHL generator.
</P><TABLE align="center" border="1"><CAPTION><B>NOR GATE</B></CAPTION><TBODY><TR><TD align="center"><B>LuxR</B></TD><TD align="center"><B>LacI</B></TD><TD align="center"><B>Exp</B></TD></TR><TR><TD align="center"><B>0</B></TD><TD align="center"><B>0</B></TD><TD align="center"><B>1</B></TD></TR><TR><TD align="center"><B>1</B></TD><TD align="center"><B>0</B></TD><TD align="center"><B>0</B></TD></TR><TR><TD align="center"><B>0</B></TD><TD align="center"><B>1</B></TD><TD align="center"><B>0</B></TD></TR><TR><TD align="center"><B>1</B></TD><TD align="center"><B>1</B></TD><TD align="center"><B>0</B></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Recombinase">Recombinase</SPAN></H4><P>
In order to utilize the recombination system repetitively, we design an adaptor where we can put in any part that needs recombining. To accomplish this goal, the restriction sites of SpeI and PstI are placed between lox66 and lox71. In this way, any fragment digested by XbaI and PstI can be inserted between the two recombination sites. We locate the recombination sites into the blank plasmid pSB3C5. The sequencing result as well as the insertion of GFP proves that the system can work successfully.(Fig2,Fig3)
</P><TABLE align="center"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig 2.General recombine system</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fig 3.The overview of the whole system</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Report_Components">   Report    Components</SPAN></H2><P>The response to distance requires that the reporter gene only responds to a narrow range of signal concentrations, which means the reporter gene will not be expressed when the concentration of signals is either too high or too low. There are some different ways to deal with the problem:
</P><H3><SPAN class="mw-headline" id="Multi-regulatory_system">Multi-regulatory system</SPAN></H3><P>The idea comes from the Nature letter “A Synthetic Multicellular System for Programmed Pattern Formation”. In this paper, based on quorum sensing, the engineered ‘receiver’ cells are programmed to form ring-like patterns while the chemical gradient of the acyl-homoserine lactone signal synthesized by ‘sender’ cells is formed. The ‘sender’ and ‘receiver’ cells can be organized to our system.
</P><P>In “sender” cells, the tetR promoter expresses LuxI constitutively so that an AHL gradient is built up around the senders. When the AHL concentration reaches a certain level, the LuxR promoter is activated to a large extent so that LacIM1 protein (a mutant of LacI) and cI protein will be expressed in a large amount. The cI protein will shut off the expression of LacI while the LacIM1 protein will inhibit the expression of GFP regulated by LacR promoter. When the AHL concentration is intermediate, the cI protein is still capable to stop the LacI expression while the LacIM1 protein can not inactivate the LacR promoter any longer, so a certain level of GFP can be observed. On the other side, when the AHL concentration is too low, neither cI nor LacIM1 will be expressed. Without cI protein, the lambda promoter will be activated and LacI protein is produced, which inactivates the LacR promoter so that GFP could be barely observed. All the above are shown as the figures below.
</P><DL><DD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig 4.The low concerntration condition</DIV></DIV></DIV></DD></DL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig 5.The medium concerntration condition</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig 6.The high concerntration condition</DIV></DIV></DIV><P>To organize this part to our system, we should first standard all the genes that don’t include in biobrick parts. We use PCR to clone the genes from plasmids given as gifts fromRon Weiss lab. PCR is also used to mutant the biobrick restriction sites follow the <A href="https://2008.igem.org/USTC/Notebook/Point_mutation_Quick-Change_method" class="external text" rel="nofollow">Quick-Change method</A> described below.
we’ve already cloned GFP(LVA)(GFP with LVA tag <A href="http://nobelprize.org/nobel_prizes/chemistry/laureates/2008/" class="external text" rel="nofollow">GFP GOT THIS YEAR'S NOBEL PRIZE!</A>), LuxR-G2F(a mutant of LuxR), LacI,LacIM1(a mutant of LacI with low affinity to lac promoter), some of which have one or more point mutations to avoid the standard restriction sites. after putting parts such as RBS, promoter and terminator together, we have made some of the protein-coding genes work successfully, e.g. GFP(LVA) under the control of lac promoter lights once stimulated.<A href="https://2008.igem.org/Team:USTC/Results" class="external text" rel="nofollow">(See our Results) </A>Unfortunately, time is not enough for us to put all the units together and test their functions.
</P><H3><SPAN class="mw-headline" id="XOR_GATE">XOR GATE</SPAN></H3><TABLE align="right" border="1"><CAPTION><B>XOR GATE</B></CAPTION><TBODY><TR><TD align="center"><B>LuxR-M1</B></TD><TD align="center"><B>LuxR-M2</B></TD><TD align="center"><B>Exp</B></TD></TR><TR><TD align="center"><B>0</B></TD><TD align="center"><B>0</B></TD><TD align="center"><B>0</B></TD></TR><TR><TD align="center"><B>1</B></TD><TD align="center"><B>0</B></TD><TD align="center"><B>1</B></TD></TR><TR><TD align="center"><B>1</B></TD><TD align="center"><B>1</B></TD><TD align="center"><B>0</B></TD></TR></TBODY></TABLE><P>In another way, the response in a narrow range can be achieved by an XOR gate. The promoter is activated when only one protein binds to the DNA. The downstream genes can not be expressed either when two proteins bind or don't bind the operator. The truth table is shown on the right.And here is a interesting pheonomenon that we can utilize: if we placed pLuxR in the upstream of -35 site, the downstream genes will be transcribed when AHL is present, and if we placed pLuxR between -35 and -10, the downstream genes will be inactivated by AHL. So we can employ two similar proteins that respond to concentration distinctly to work as an XOR gate. The two proteins will not respond both at low concentrations and at the high concentrations. Only in intermediate concentrations, the protein with higher sensitivity will bind DNA and express the downstream genes.
</P><H3><SPAN class="mw-headline" id="Computational_Protein_Design">Computational Protein Design</SPAN></H3><P>How to change protein's sensitivity to ligands? Computational protein design is one easier way. From Protein Data Bank, we can obtain the structure of LuxR and AHL. The analysis tells us the contribution of each residue to docking of the whole protein. So in silico screen we can get the answers about what changes can weaken or strengthen the binding capability.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>1.A Synthetic Multicellular System for Programmed Pattern Formation
</LI></UL><DL><DD>Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss  NATURE  April 28, 2005, 1130-1134
</DD></DL><UL><LI>2.Detection of bacterial quorum sensing N-acyl homoserine lactones in clinical samples
</LI></UL><DL><DD>Anjali Kumari &amp; Patrizia Pasini &amp; Sylvia Daunert  Anal Bioanal Chem (2008) 391:1619–1627
</DD></DL><UL><LI>3.Acyl-homoserine lactone quorum sensing in Gram-negative bacteria: A signaling mechanism involved in associations with higher organisms   
</LI></UL><DL><DD>Matthew R. Parsek and E. Peter Greenberg    PNAS  August 1, 2000,  8789–8793
</DD></DL><UL><LI>4. AiiA, an enzyme that inactivates the acylhomoserine lactone quorum-sensing signal and attenuates the virulence of Erwinia carotovora       
</LI></UL><DL><DD>Yi-Hu Dong, Jin-Ling Xu, Xian-Zhen Li, and Lian-Hui Zhang*   PNAS  March 28, 2000, 3526–3531
</DD></DL><UL><LI>5. Computational redesign of endonuclease DNA binding and cleavage specificity.
</LI></UL><DL><DD>Ashworth, J., Havranek, J.J., Duarte, C.M., Sussman, D., Monnat, R.J., Stoddard, B.L. &amp; Baker, D.Nature, 2006, Vol. 441(7093), pp. 656-659
</DD></DL><UL><LI>6. Improved Pruning algorithms and Divide-and-Conquer strategies for Dead-End Elimination, with application to protein design
</LI></UL><DL><DD>Georgiev, I., Lilien, R.H. &amp; Donald, B.R. Bioinformatics (Oxford, England), 2006, Vol. 22, pp. e174-83
</DD></DL><UL><LI>7. Computing van der Waals energies in the context of the rotamer approximation.
</LI></UL><DL><DD>Grigoryan, G., Ochoa, A. &amp; Keating, A.E. Proteins, 2007, Vol. 68(4), pp. 863-878
</DD></DL><UL><LI>8. Progress in computational protein design.
</LI></UL><DL><DD>Lippow, S.M. &amp; Tidor, B. Curr Opin Biotechnol, 2007, Vol. 18(4), pp. 305-311
</DD></DL><UL><LI>9. New algorithms and an in silico benchmark for computational enzyme design.
</LI></UL><DL><DD>Zanghellini, A., Jiang, L., Wollacott, A.M., Cheng, G., Meiler, J., Althoff, E.A., R?thlisberger, D. &amp; Baker, D.Protein Sci, 2006, Vol. 15(12), pp. 2785-2794
</DD></DL><UL><LI>10. Directed Evolution of Vibrio fischeri LuxR for Improved Response to Butanoyl-Homoserine Lactone.
</LI></UL><DL><DD>Andrew C. Hawkins, Frances H. Arnold, Rainer Stuermer,Bernhard Hauer, and Jared R. Leadbetter1*, 2007, APPLIED AND ENVIRONMENTAL MICROBIOLOGY,  p. 5775–5781.
</DD></DL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:USTC/Component">http://2008.igem.org/Team:USTC/Component</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USTC/Component" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USTC/Component" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USTC/Component&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USTC/Component&amp;oldid=102967" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>