"
Page
Discussion
View source
History
teams
Log in
 
Team:Tufts/ribosponge
From 2014.igem.org
Tufts University
Home
Team
Team Bios
Attributions
Tufts Synthetic Biology
Official Team Profile
Project
Ribosponge
SYNENERGENE Collaboration
Medal Fulfillment
Human Practices
The Future of Phage and Synthetic Biology
Application Scenarios
Techno-Moral Scenarios
Survey
Education
Safety
Sponsors
Jump to
Problem
Background
Methods
Results
Parts
Notebook
Sources
Image from (11)
Problem
Biofilms present one of the largest problems facing healthcare in the United States today. The ability of bacterial pathogens to form biofilms is one of their most important virulence factors, as biofilm formation confers resistance to both antibiotics and natural host defense mechanisms. (1)  These sessile bacterial growths, often of the genus Staphylococcus, form in the lungs of cystic fibrosis patients,  in patients suffering from chronic infections, and on implant materials such as catheters.  The CDC estimates that 65% of infections in developed nations are caused by persistent bacterial biofilms. (2)  They are the leading cause of Healthcare-Associated Infections (HAI’s), of which there are two million instances and 100,000 resulting deaths each year in the United States alone. (3)  The inability to treat biofilms with antibiotics and antimicrobials has lead the Tufts iGEM Team to seek novel methods for disrupting biofilm formation.
Go to top
Background
Not-so-magic bullets
“Two years into the war the Army issued its first meager supplies of penicillin, instructing physicians to use the precious drug sparingly, in doses of about 5,000 units (less than a third of what would be considered a minimal penicillin dose for minor infections in 1993 ). In those early days before bacteria became resistant to antibiotics, such doses were capable of performing miracles, and the Army doctors were so impressed with the powers of penicillin that they collected the urine of patients who were on the drug and crystallized excreted penicillin for reuse on other GIs.” Laurie Garrett - The Coming Plague
Humanity is locked in a perpetual arms race against its pathogens. For the majority of our existence our species has relied upon its innate immune ability in order to survive disease. This has proven effective at a population level so far: no pathogen has driven us to extinction. However, in the last two centuries humanity has added new weapons to its disease fighting arsenal: antibiotics reduced bacterial infections from life-threatening to trivial matters. Antiviral drugs have rescued those infected with HIV from the brink of death. For a time, these drugs were so indisputably effective they were thought of as “magic bullets” and thrown at pathogens indiscriminately. Unfortunately, their efficacy has been diminishing. In a textbook example of selective pressure, the most susceptible bacterial pathogens were wiped out while those able to withstand antibiotics have thrived.
Antibiotics have several modes of action: beta lactams inhibit cell wall synthesis, aminoglycosides interfere with translation and lead to misfolded proteins, and fluoroquinones disrupt DNA gyrase and topoisomerase during cell replication. In response, bacteria have evolved the ability to degrade these molecules with enzymes such as the beta-lactamase, or to actively remove them from the intracellular space by means of efflux pumps. Such pathogens are deemed resistant to antibiotics as they can grow and persist in their presence unperturbed.
Go to top
Laying low
“He who knows when he can fight and when he cannot will be victorious.” – Sun Tzu, The Art of War
Efflux pumps and hydrolytic enzymes are by no means the only evolutionary strategies which pathogenic organisms have in order to escape destruction. Many bacteria have the ability to reduce their metabolism to a minimum and enter a dormant stage in which little to no new cell wall construction, protein translation, or DNA replication occurs. (2) (4) (5) As a result, any cells that have in effect shut down are in practice impervious to the modes of action of modern antibiotics. Such cells do not need to be genetically resistant because their ability to become dormant allows them to tolerate the presence of antibiotics. (2) The strategy is effective not only against toxic chemicals, but also applicable during periods of overcrowding or low nutrient availability.
Cells entering a persistent biofilm undergo a transition from a more motile state to a sessile state, and use their extracellular projections to attach to one another and to their surroundings.  In addition to clumping together, certain bacteria secrete a polymer which forms a protective matrix around the cells. In human wounds infected with P. aeruginosa this exopolymer matrix is effective at denying macrophages access and preventing them from digesting the dormant bacteria. Should antibiotic-susceptible members of the biofilm population be wiped out, the space enclosed by the exopolymer can be repopulated by any individual bacteria which carry resistance genes once therapy is discontinued. The persistence of these bacteria as a biofilm within the lungs of cystic fibrosis patients and in chronic wounds is well known. In addition, the CDC estimates that 65% of infections in developed nations like the United States are caused by persistent bacterial biofilms. (2)
Go to top
Signaling
The ability of bacteria to lie low is an indispensable evolutionary adaptation for P. aeruginosa, M. tuberculosis, and V. cholerae. In effect these bacteria have the ability to wait out and evade our immune systems or persist contrary to adverse environmental factors. Is there a unifying factor to these seemingly similar latent behaviors?
The chemical Bis-(3’-5’)-cyclic dimeric guanosine monophosphate (c-di-GMP) was discovered in the late 1980s. (7) The compound is created when two molecules of guanosine triphosphate are joined in a reaction catalyzed by diguanylate cyclase (DGC). Originally found in Gluconacetobacter xylinus, natural and synthetic versions of the molecule were shown to activate the bacterium’s membrane-bound cellulose synthase. (8) In other bacteria it has been shown to promote the synthesis of adhesins and exopolymers necessary for biofilm formation. (5) (8) In order to test the effects of c-di-GMP research groups working with a range of bacterial species have upregulated the production of DGC enzymes responsible for the secondary messenger’s synthesis. (8) This modification caused the transgenic strains to become sessile and less virulent, implicating that the secondary messenger is the signal which informs the cells to switch to an inactive, sessile, and persistent state. (8) On the other hand, artificially stimulating the phosphodiesterase (PDE) enzymes which break down c-di-GMP resulted in motile populations unable to form a biofilm. (8) (See Figure)
Figure: Cyclic di-GMP, its synthesis and breakdown pathways, and its effects on bacterial phenotype. Taken from (9).
Additional work has also provided further information about the mode of action of the c-di-GMP molecule itself. In silico computations predicted that the PilZ protein domain present in G. xylinus would bind the secondary messenger. As a result of this, PilZ domains have been identified in many bacterial proteins whose activity responds negatively to the presence of c-di-GMP. (7) One example, the YegR protein of free swimming E. coli, contains a PilZ site. Experiments show that the deletion of the phosphodiesterase enzyme responsible for c-di-GMP degradation in the E. coli strain leads to reduced swimming ability. In addition, subsequent deletion of the YegR protein (which binds and responds to the presence of c-di-GMP) restores motility to 80% of that exhibited by the wild type bacteria. (8)
Riboswitch aptamers (ligand-binding nucleic acid molecules) for c-di-GMP have been identified. (9) (10) These RNA aptamers can come before or after a transcribed gene. Examples of “on” and “off” regulatory c-di-GMP riboswitches have been observed in P. aeruginosa, M. tuberculosis, V. cholerae, Y. pestis, and other bacterial species which latent phases which are more tolerant of antibiotics and adverse conditions. (6) (8) (9) (10) It seems that this secondary messenger is the key molecule in persistent cells.
Go to top
Killing the Messenger
“The most important secret in war consists of making one’s self the master of communications.”  – Napoleon Bonaparte, Pensées
In summary, current data shows that if cyclic di-GMP is eliminated from the cells of a number of bacterial species, they cannot enter the persistent state. Theoretically, the pathogens can be prevented from shutting down and becoming tolerant to antibiotics or lack of nutrients. Experiments have already demonstrated this by overproducing the phosphodiesterases which break down the messenger or mutating the dyguanylate cyclases which produce it. (9)
Targeting c-di-GMP across all these species could be done by excising the enzymes which are responsible for synthesizing the molecule or adding more phosphodiesterases to break it down. Genetic editing that is targeted with this degree of specificity is difficult to accomplish in a clinical setting. Furthermore, the addition of new proteins would increase the metabolic burden associated with their translation.
A potential tool is presented by the gram-negative predator of gram-negatives, Bdellovibrio bacteriovorus. This organism also has two distinct life phases. A free-swimming attack phase allows it to roam and locate its prey. After finding a suitable host, B. bacteriovorus attaches to the unfortunate gram negative bacterium and digests it from within.  A recent transcriptome analysis by Karunker and colleagues has shown that the most common RNA identified by sequencing was a non-coding segment which houses a c-di-GMP aptamer. (11) This RNA was only present during the motile attack phase. In addition, it accounted for just as many reads as all protein coding sequences combined and is about as common as structural nucleic acids such as rRNA. These results imply a different mode of action for the bacteriovorus riboswitch. The sequence acts more like a sponge which sequesters available c-di-GMP than a switch which responds to its presence. The discoverers have dubbed this 445nt transcript massively expressed regulatory RNA (merRNA). (11)
Go to top
Methods
The merRNA from Bdellovibrio could be introduced into the genome of other bacterial species and placed under the control of an always-on constitutive promoter. Cyclic di-GMP RNA aptamers are known to bind 1000-fold more tightly to their messenger (KD ~ 1nM) than other regulatory proteins like PilZ (KD ~ 1µM). (10) Thus, these transcripts of the merRNA should in turn sequester the secondary messenger. As a result, the pathways responsible for the transition to a persistent state should not be activated. One advantage of this strategy is that the ribosponge is a non-coding RNA construct; the metabolic burden of transcription should be much lower than that demanded by the transcription and activity of a phosphodiesterase or genome editing enzyme.
We designed a strategy to express merRNA in E. coli and test its effects on biofilm growth using SnapGene molecular biology software. The merRNA sequence was obtained from Karunker et al (11) and identified in the Bdellovibrio bacteriovorus HD100 genome at position 1,073,741−1,074,185 with a length of 445 nucleotides.. Since the goal here was to physically sequester c-di-GMP in cells, we aimed to produce large quantities of the RNA and thus decided to place it under the control of the T7 promoter which is both constitutive and leads to very strong expression. In turn, this sequence is flanked by a set of three strong, naturally occurring terminators from a library characterized by Chen and colleagues (12).
The first of these, ECK120033737, is upstream and serves to prevent any read-through by a polymerase starting before the T7 when the sequence is inserted into a plasmid. Downstream, another pair of terminators, ECK120029600, and ECK120033736, serve to terminate transcription of the merRNA and prevent read-through by a polymerase moving along the reverse strand, and generating reverse-complementary RNA that would base pair with the merRNA and cancel out its effects, respectively. It should be noted that this last, reverse-oriented terminator may be superfluous in most cases, but was necessary for our application as we chose to place the fragment into LITMUS28i which has opposing T7 promoters. It is also likely that the merRNA is self-terminating due to the presence of a poly-U intrinsic terminator. Sites for PstI and BamHI were added to each end of this design in order to facilitate insertion into a plasmid. The resulting 686 base pair sequence was then synthesized by GenScript.
Our initial goal was to express this fragment in bacteria and spread it using M13 bacteriophage. As such, we chose LITMUS28i-I716104 as our vector. This plasmid contains the original NEB LITMUS28i phagemid backbone and a GTG-starting T7 polymerase which itself is under the control of a T7 promoter. Leaky transcription at this T7 promoter starts expression of the T7 RNA polymerase. The plasmid was generated by Dr. Monica Ortiz of the Endy group who shared it with us for use in our project.
Our synthetic DNA and the plasmid were digested with PstI and BamHI and ligated together to form the construct with which all of our biofilm experiments were run. The plasmid was then transformed into TSS competent E. coli JM109 and maintained via ampicillin selection. The plasmid was also transformed into E. coli ZK1056 provided to us by the Kolter group at Harvard Medical School. ZK1056 is a known biofilm-forming strain and its characteristics have been widely studied. In addition to these modifications, we also transformed the two strains with LITMUS28i-I716104.
Biofilm assays were carried according to a modified protocol detailed by O’Toole 2011 (13). First, cultures of JM109, JM109 LITMUS28i-I716104, JM109 LITMUS28i-I716104-merRNA, ZK1056, ZK1056 LITMUS28i-I716104, ZK1056 LITMUS28i-I716104-merRNA were grown up overnight at 37°C in an orbital bath shaker in LB media, or LB media with ampicillin to maintain the plasmid. The cultures were then taken out of the incubator and diluted 1:100 in the fresh, original media. They were left sitting on the bench for 30 minutes after which 250uL were dispensed into untreated 96-well polystyrene plates. Each of these plates was covered with a plastic lid. Plates were then wrapped with a moist Kim Wipe. The plate was then placed in a pipette tip box such that it sat on the grating while the bottom of the box held about a centimeter of water. By slightly closing the lid, the box remains humidified and media do not evaporate. The boxes were then placed on a shelf in a static incubator and held there for twenty-four hours at a given temperature. To process the plates, bacteria were shaken out of them and the plates were rinsed in a reverse osmosis (RO) water bath three times. Each well was then stained with 275uL of 0.1% crystal violet for 30 minutes. At the end of this time the crystal violet stain was shaken out and the plates were rinsed in a RO water baths until the water was clear. Plates were left to dry overnight, after which the stain was solubilized with 300uL 80% ethanol/20% acetone, 250uL of which was then transferred to another plate in order to obtain a readout at 550nm.
Go to top
Results
Expectations
Because the merRNA was thought to sequester c-di-GMP and consequently prevent bacteria from entering into a persistent biofilm state, fewer bacteria were expected to adhere to the sides and bottom of the well plate. It was anticipated that bacteria expressing the merRNA would remain in suspension and therefore be rinsed off prior to staining with crystal violet. Those without the merRNA would adhere to the well and to each much more, which would result in a higher retention of crystal violet.  For bacteria with the merRNA, this reduction in biofilm formation would be indicated by a lower absorbance value at 550 nm of the dissolved crystal violet relative to bacteria without the merRNA.
Fig 1. Absorbance values at 550 nm of cells with and without the merRNA construct. JM109 and ZK1056 are both known to form rather robust biofilms, whereas ELS-30 is not. As such, ELS-30 was used as a control. Cultures were were started from 1:100 dilutions of JM109 and ZK1056 (with and without the merRNA) along with the ELS-30 control. These were grown under the appropriate antibiotic selection. 250 uL of each culture was aliquoted into wells on a nonbinding polystyrene 96-well plate. The cultures were grown at 30°C statically for 24 hours, after which time the planktonic bacteria suspended in LB were rinsed off. The bacteria that remained stuck to the well (those in a biofilm state) were stained with 275 uL 0.1% crystal violet. Excess crystal violet was then rinsed off. The crystal violet was dissolved in 300 uL 80% ethanol/20% acetone. 250 uL of this solution was transferred to a clear-bottomed well plate and the A550 was recorded with a plate reader.
These data indicate that, contrary to expectations, bacteria expressing the merRNA exhibited more robust biofilms than those without the merRNA. An approximately twofold increase in biofilm formation was seen for both JM109 and ZK1056. Since no significant difference was observed in the growth capacity of these cultures, cellular processes (rather than cell density) are believed to be responsible for the differential levels of biofilm formation.
Temperature Dependence
Fig 2. Absorbance at 550 nm of cells with and without the merRNA at 25°C. The same procedure detailed for Figure 1 was repeated at 25°C. Lower temperatures generally increase biofilm formation since cells are under unfavorable growing conditions. Although minor variable preclude direction comparisons to the initial trial at 30°C, the same relationships are seen in this trial.
Fig 3. Absorbance at 550 nm of cells with and without the merRNA at 30°C. The same procedure was repeated at 30°C. A greater difference in biofilm formation in JM109 cells is observed, but the increase in temperature caused ZK1056 to exhibit equally robust biofilm formation with or without the merRNA.
Fig 4. Absorbance at 550 nm of cells with and without the merRNA at 37°C. The same procedure was repeated at 37°C. Since this is a favorable growing temperature, bacteria do not generally enter a biofilm state. An even greater relative increase of JM109 with merRNA was seen, but the ZK1056 once again registered no significant difference in biofilm formation.
The trials at three different temperatures show that the merRNA most consistently promotes robust biofilm formation at lower temperatures since both JM109 and ZK1056 cells exhibited this trend. An increase in temperature widened the difference in JM109 cells with and without merRNA, but the effect was marginalized to a point of insignificance in ZK1056.
Our Hypothesis
The general observation that constitutive expression of the merRNA increases biofilm formation is contrary to the original expectations. However, since such a marked difference in biofilm formation was observed, it is believed that the merRNA is exhibiting a physiological effect. Since the merRNA is known to contain a c-di-GMP aptamer, the likely mechanism of action involves the sequestration of c-di-GMP such that the available intracellular concentration decreases and downstream signalling pathways are not activated.
In Bdellovibrio bacteriovorus, the merRNA causes the transition from a dormant to a motile phase. The same was expected of our E. coli. Instinctively, the observed increase in biofilm formation is incompatible with the expected increase in motility. However, several studies suggest that, in fact, motility is a key factor in enabling the attachment of cells to surfaces and other cells. One study in Pseudomonas aeruginosa reports that flagellar and twitching motility are necessary for biofilm attachment (14). Cells deficient in motility specific genes were unable to attach to surfaces and thus could not enter biofilm states. A paper dealing with E. coli cites the importance of flagella for initial attachment, especially at lower temperatures. The authors write, "In pathogenic E. coli living within the host or on abiotic surfaces (37 °C), type I fimbriae or the adhesin AG43 are involved in initial attachment, PGA stabilizes permanent attachment and also curli fibers, which contribute to surface attachment, can be a predominant matrix component. Bacteria growing in the environment or on abiotic surfaces at lower temperatures (<30 °C), form differently composed biofilms, using flagella for initial attachment and curli, cellulose and colanic acid as a matrix in the mature biofilm" (15). Furthermore, a comprehensive study of various biofilm formation pathways in K12 E. coli concluded, following analysis of 20 different motility-related genes, that flagella are necessary for biofilm attachment, not only initially, but also during later stages of biofilm development (16).
With these findings in mind, we hypothesize that the unexpected effects of merRNA expression are indeed the result of the merRNA acting as a ribosponge and binding to c-di-GMP to lower its active intracellular concentration. This, in turn, increases motility, just as it does in Bdellovibrio bacteriovorus. However, rather than manifesting itself as a population of highly active planktonic bacteria, the up-regulation of motility genes increased flagellar expression which, rather ironically, enabled better attachment of cells to surfaces and facilitated robust biofilm formation.
Applications
Since generating more robust biofilms does not have beneficial applications in a medical setting, the  phagemid deployment of the merRNA was not pursued. There are some applications in which cells with robust biofilm characteristics can be useful, however. Facilitating the growth of beneficial bacteria in a biofilm state can help treat wastewater by destroying harmful microorganisms or even industrial products like gasoline. Additionally, engineering bacteria in a biofilm state will be more effective in microbial leaching applications (17). Since the merRNA is believed to modulate biofilm formation via increases in cellular motility, the merRNA may have uses in facilitating more motile strains of bacteria.
Go to top
Parts
BBa_K1427000
The merRNA (massively expressed regulatory RNA) is a long, non-coding RNA transcript found in Bdellovibrio bacteriovorus which was described by Karnuker et al 2013. The 445 nt transcript contains a cyclic-di-GMP riboswitch sequence and is believed to mediate a change from growth phase to attack phase in Bdellovibrio by sequestering the second messenger c-di-GMP. When expressed constitutively under a T7 promoter in K12 E. coli, the merRNA increased biofilm formation. This is thought to be the result of the merRNA binding to and sequestering c-di-GMP molecules. This causes an increase in motility and flagella expression. The additional flagella adhere to surfaces and other cells, thereby forming a robust biofilm. Teams can insert this part directly downstream of a promoter sequence to cause stronger biofilm formation.
BBa_K1427001
This part contains BBa_K1427001 under the transcriptional control of a constitutive T7 promoter. Additionally, terminators developed by the Voigt Lab flank the merRNA sequence. There is an upstream ECK120033737 terminator which stops read-through by other genes. Downstream of the merRNA sequence, bidirectional termination is ensured with ECK120029600 and ECK120033736. It is likely these are redundant, as the poly-U tail of the merRNA suggests that is contains an intrinsic terminator.
Go to top
Notebook
  Lab Notebook - June
  Lab Notebook - July
  Lab Notebook - August
  Lab Notebook - September
  Lab Notebook - October
Go to top
Works Cited
1. Staphylococcus epidermidis surfactant peptides promote biofilm maturation and dissemination of biofilm-associated infection in mice. Rong Wang ... Shu Y. Queck, Michael Otto.  January 4, 2011 J Clin Invest. 2011;121(1):238-248. doi:10.1172/JCI42520.
http://www.jci.org/articles/view/42520
2. Persister cells, dormancy and infectious disease. Lewis, Kim. January 2007, Nature, pp. 48-56.
3. NIH Scientists Identify Mechanism Responsible for Spreading Biofilm Infections Finding Could Lead to Treatment to Prevent Infection Associated with Catheters and Medical Implants. Monday, Dec. 6, 2010. National Institute of Allergy and Infectious Diseases (NIAID)
http://www.niaid.nih.gov/news/newsreleases/2010/Pages/biofilmMechanism.aspx
4. BACTERIAL BIOFILMS: FROM THE NATURAL ENVIRONMENT TO INFECTIOUS DISEASES. Hall-Stoodley, Luanne, Costerton, J. William and Stoodley, Paul. February 2004, Nature Reviews | Microbiology, pp. 95-108.
5. Signals, Regulatory Networks, and Materials That Build and Break Bacterial Biofilms. Karatan, Ece and Watnick, Paula. 2, 2009, Microbiology and Molecular Biology Reviews, Vol. 73, pp. 310-347.
6. Phytoplankton-linked viable non-culturable Vibrio cholerae O1 (VNC) from rivers in Tucuman, Argentina. Seeligmann, Claudia, et al., et al. 4, 2008, Journal of Plankton Research, Vol. 30, pp. 367-377.
7. Cyclic diguanylate (c-di-GMP) regulates Vibrio cholerae biofilm formation. Tischler, Anna and Camilli, Andrew. 3, 2004, Molecular Microbiology, Vol. 53, pp. 857-869.
8. Roles of Cyclic Diguanylate in the Regulation of Bacterial Pathogenesis. Tamayo, Rita, Pratt, Jason and Camilli, Andrew. 2007, Annual Review of Microbiology, pp. 131-148.
9. Principles of c-di-GMP signalling in bacteria. Hengge, Regina. April 2009, Nature Reviews | Microbiology, pp. 263-273.
10. Riboswitches in Eubacteria Sense the Second Messenger Cyclic Di-GMP. Sudarsan, N, et al., et al. July 18, 2008, Science, pp. 411-413.
11. A Global Transcriptional Switch between the Attack and Growth Forms of Bdellovibrio bacteriovorus. Karunker, Iris, et al., et al. April 16, 2013, PLOS|ONE.
12. Characterization of 582 natural and synthetic terminators and quantification of their design constraints. Chen, et al., June 02, 2013, Nature Methods.
13. Microtiter dish biofilm formation assay. O'Toole, GA.,January 30, 2011.J Vis Exp.
14. Flagellar and twitching motility are necessary for Pseudomonas aeruginosa biofilm development.O'Toole, GA and R Kolter., October 30, 1998. Mol Microbiol.
15. Small Regulatory RNAs in the Control of Motility and Biofilm Formation in E. coli and Salmonella. Mika, F and R Hengge., February 26, 2013. Int J Mol Sci.
16. Temporal gene-expression in Escherichia coli K-12 biofilms. Domka,J; J Lee, T Bansal1 and T Wood. February, 2007. Environ Microbiol.
17. How do Biofilms Impact Our World? http://www.cs.montana.edu/webworks/projects/stevesbook/contents/chapters/chapter001/section005/green/page001.html
Works Referenced
Todar, Kenneth. Mycobacterium tuberculosis and Tuberculosis. Todar's Online Textbook of Bacteriology. [Online] [Cited: December 16, 2013.] http://textbookofbacteriology.net/tuberculosis_2.html.
Tuberculosis: What We Don't Know Can, and Does, Hurt Us. Russel, David, Barry, Clifton and Flynn, JoAnne. May 14, 2010, Science, pp. 852-856.
Mycobacterium tuberculosis: success through dormancy. Gengenbacher, Martin and Kaufmann, Stefan. 2012, Federation of European Microbiological Societies, pp. 514-532.
Structural and mechanistic determinants of c-di-GMP signalling. Schirmer, Tilman and Jenal, Urs. 2009, Nature Reviews, pp. 724-735.
Cyclic di-GMP mediates Mycobacterium tuberculosis dormancy and pathogenicity. Hong, Yuzhi, et al., et al. 2013, Tuberculosis, pp. 625-634.
Three-component serotype conversion in Pseudomonas aerugionsa by bacteriophage D3. Newton, Gregory, et al., et al. 5, 2001, Molecular Microbiology, Vol. 39, pp. 1237-1247.
DNA sequence, structure and gene expression of mycobacteriophage L5: a phage system for mycobacterial genetics. Sarkis, Gary J and Hatfull, Graham F. 3, 1993, Molecular Microbiology, Vol. 7, pp. 395-405.
Mycobacteriophage and their application to disease control. McNerney, R. and Traore, H. 2005, Journal of Applied Microbiology, pp. 223-233.
Ortiz, Monica. Enabling population-level programming via DNA messaging. Northeastern University. [Online] [Cited: December 20, 2013.]
http://madrid.ccs.neu.edu/biocom2/wp-content/uploads/2013/01/Monica_Ortiz.pdf.
Engineered cell-cell communication via DNA messaging. Ortiz, Monica E and Endy, Drew. 2012, Journal of Biological Engineering.
New England BioLabs. LITMUS 28i Vector. New England BioLabs. [Online] [Cited: December 23, 2013.]
https://www.neb.com/products/n3528-litmus-28i-vector#pd-properties-usage.
M13KO7 Helper Phage. New England BioLabs. [Online] [Cited: December 22, 2013.] https://www.neb.com/products/n0315-m13ko7-helper-phage#tabselect1.
A strategy of gene overexpression based on tandem repetitive promoters in Escherichia Coli. Li, Mingjli, et al., et al. 2012, Microbial Cell Factories.
Knight, Tom. BioBrick RFC[10]. Registry of Standard Biological Parts. [Online] [Cited: Dec 23, 2013.]
http://parts.igem.org/Help:Standards/Assembly/RFC10.
Part:BBa_B0015. Registry of Standard Biological Parts. [Online] 07 17, 2003. [Cited: December 21, 2013.]
http://parts.igem.org/Part:BBa_B0015.
iGEM07_Berkeley_UC. T7rnap (T7 RNA Polymerase) - GTG Start Codon. Registry of Standard Biological Parts. [Online] 06 09, 2007. [Cited: December 21, 2013.]
http://parts.igem.org/Part:BBa_I716104.
New England BioLabs. Phusion High-Fidelity PCR Master Mix with HF Buffer. New England BioLabs. [Online] [Cited: Dec 20, 2013.]
https://www.neb.com/products/m0531-phusion-high-fidelity-pcr-master-mix-with-hf-buffer.
Gibson Assembly Cloning Kit. New England BioLabs. [Online] [Cited: December 20, 2013.]
https://www.neb.com/products/e5510-gibson-assembly-cloning-kit.
Hha, YbaJ, and OmpA, Regulate Eschericha coli K12 Biofilm Formation and Conjugation Plasmid Abolish Motility. Gonzalez Barrios, Andres F, et al., et al. Nov 29, 2005, Biotechnology and Bioengineering, pp. 189-200.
College of Veterinary Medicine, Michigan State University. Mode of Action. Antimicrobial Resistance Learning Site. [Online] [Cited: December 21, 2013.]
http://amrls.cvm.msu.edu/pharmacology/antimicrobials/mode-of-action.
Division of Tuberculosis Elimination. Basic TB Facts. Centers for Disease Control and Prevention. [Online] March 13, 2013. [Cited: December 18, 2013.]
http://www.cdc.gov/tb/topic/basics/default.htm.
Go to top
Retrieved from " http://2014.igem.org/Team:Tufts/ribosponge "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
