<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Notebook_Week4"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Notebook/Week4">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Notebook/Week4&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week4&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week4&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Notebook/Week4" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Notebook/Week4</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="wrap"><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:Cambridge">Home</A></LI><LI><A href="/Team:Cambridge/TheTeam">Meet the Team</A></LI><LI><A href="/Team:Cambridge/Photos">Photo Gallery</A></LI><LI><A href="/Team:Cambridge/Videos">Videos</A></LI><LI><A href="/Team:Cambridge/BioBricks">Parts submitted</A></LI></UL><LI><A class="head" href="#">Project Firefly</A></LI><LI><A href="/Team:Cambridge/Bioluminescence">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background_Firefly">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Colour">Coloured outputs</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Luciferin_Regeneration">Luciferin recovery</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Project Vibrio</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Luciferases">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/G28">The LuxBrick</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">Codon optimisation</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:Cambridge/Tools/Gibson">Gibthon</A></LI><LI><A href="/Team:Cambridge/Tools/GenBank">BioBrick → GenBank</A></LI><LI><A href="/Team:Cambridge/Tools/Ligate">Ligation Calculator</A></LI><LI><A href="/Team:Cambridge/Tools/Eglometer">The E.glometer</A></LI><LI><A class="head" href="#">Gibson Assembly</A></LI><LI><A href="/Team:Cambridge/Gibson/Introduction">Introduction</A></LI><LI><A href="/Team:Cambridge/Gibson/Mechanism">How it works</A></LI><LI><A href="/Team:Cambridge/Gibson/Protocol">Protocol</A></LI><LI><A href="/Team:Cambridge/Gibson/RFC">RFC</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:Cambridge/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Cambridge/Safety">Safety</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:Cambridge/Tools/microMeasure">Biosensors</A></LI><LI><A href="/Team:Cambridge/Tools/Lighting">Lighting</A></LI><LI><A class="head" href="/Team:Cambridge/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:Cambridge/Notebook/Summary">Summary</A></LI><LI><A href="/Team:Cambridge/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:Cambridge/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:Cambridge/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:Cambridge/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:Cambridge/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:Cambridge/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:Cambridge/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:Cambridge/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:Cambridge/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:Cambridge/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:Cambridge/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:Cambridge/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:Cambridge/Notebook/FurtherWork">Beyond Week 12</A></LI><LI><A href="/Team:Cambridge/Protocols">Protocols</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">Our Partners</A></LI><LI><A href="/Team:Cambridge/Partners">Sponsors</A></LI><LI><A href="/Team:Cambridge/iGEM_Teams">iGEM Teams</A></LI><LI><A href="/Team:Cambridge/Departments">University</A></LI><LI><A href="/Team:Cambridge/Publicity">Publicity</A></LI></DIV><DIV id="right"><DIV style="width:730px; background-color:#fad72a;" class="secheader">Notebook: Week 4</DIV><DIV style="position:relative; height:30px;border-top:5px solid white;"><DIV style="width:492px;position:absolute; background-color:#96d446;" class="selecteddiv" id="projectlink"><P>Viewing project diary</P></DIV><DIV style="width:246px; position:absolute; background-color:#fb5c2b;left:497px;" class="selectordiv" id="lablink"><P>View protocols and data</P></DIV></DIV><DIV style="overflow:hidden; height:2200px; width:730px; position:relative;" id="maskingbox"><DIV style="width:1320px; position:relative;" id="superwide"><TABLE style="width:1480px" id="widetable"><TBODY><TR><TD style="padding-right:50px; padding-left:10px; width:630px;vertical-align:top"><DIV align="center">Week 4: Monday 2nd - Sunday 8th August</DIV><H2><SPAN class="mw-headline" id="Monday"> Monday </SPAN></H2><P>We transformed some of the competent TOP10 cells we made the previous week with plasmid phK724, which was kindly  sent to us by <A href="http://departments.kings.edu/biology/lux/" class="external text" rel="nofollow">Jim Slock</A></P><P>We also placed E. coli containing phK555 in 5ml LB broth in a rotating 37°C incubator to grow up overnight.
</P><P>In terms of dry work, we discussed what we wanted to have synthesised.  We decided to dismiss PPDK (Pyruvate Orthophosphate Dikinase) and to include the Japanese firefly (<I>Luciola cruciata</I>) because of the ease with which we could change the colour of its emission.
</P><H2><SPAN class="mw-headline" id="Tuesday"> Tuesday </SPAN></H2><P>We made cells from 4 different hns mutants: W3110 hns93-1, BW25113 Δhns::kan, W3110 hns-205::Tn10, GM230 hns-205::Tn10.  This was prompted by a paper suggesting increased bioluminescence in strains with reduced hns. These cells were stored at -80°C  for later use
</P><P>We continued to discuss synthesis requirements
</P><H2><SPAN class="mw-headline" id="Wednesday"> Wednesday </SPAN></H2><P>Hannah and Theo tested the previous day's preparations for competence.  Unfortunately they plated out only 20ul, not enough to get sufficient colonies for an accurate perception of competence. Meanwhile, Peter and Paul transformed E.coli containing phK555 with phK724 and placed the cells in a 30C incubator. (V. fischeri lux proteins denature above this temperature)
</P><P>Bill, Emily, Anja tried to add a prokaryotic RBS and promoter to the firefly luciferase already in the registry.  They performed miniprep and restriction digest then ligation, but used incorrect restriction enzymes, such that we could not exclude religated plasmid backbones. 
</P><H2><SPAN class="mw-headline" id="Thursday"> Thursday </SPAN></H2><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P>GM230 hns-205 ::Tn10 TetR (black) - somewhat competent
</P><P>W3110 hns 93-1 (blue) - no transformants
</P><P>BW25113 Δ  hns ::kan (red) - somewhat
</P><P>W3110 hns-205::Tn10 Tet<SUP>R</SUP> - no tansformants
</P><H3><SPAN class="mw-headline" id="Theo.27s_plan">Theo's plan</SPAN></H3><OL><LI> Miniprep plasmid DNA from luciferase and backbone transformed TOP10 following Qiagen protocol
</LI><LI> Digest luciferase Biobrick with: XbaI, PstI to produce two fragments:
</LI><LI> P end---old backbone---E--Xend
</LI><LI> X<SUB>end</SUB>--luciferase---S--P<SUB>end</SUB><STRONG>(desired)</STRONG></LI></OL><LI> Digest backbone Biobrick with SpeI, PstI to produce:
</LI><LI> S<SUB>end</SUB>--short fragment--P<SUB>end</SUB></LI><LI> P<SUB>end</SUB>---backbone--E--X--promoter--RBS--S<SUB>end</SUB><STRONG>(desired)</STRONG></LI><LI> Run both digest products on a gel
</LI><LI> Extract suitable lengthed fragments from gel:
</LI><LI> Promoter + backbone should be 2153 bp
</LI><LI> Luciferase should be 1653 bp (its 3426 bp backbone should be discarded)
</LI><LI> Perform ligation
</LI><LI>S and X are compatible so we get P<SUB>end</SUB>---backbone--E--X--promoter--RBS--SX<SUB>scar</SUB>--luciferase---S--P<SUB>end</SUB></LI><LI>The two P<SUB>end</SUB>s will also ligate, producing a circular desired plasmid
</LI><LI> Transform competent TOP10 cells with ligation product
</LI><H3><SPAN class="mw-headline" id="What_happened">What happened</SPAN></H3><OL><LI> In the morning we were a bit disconsolate because there was nothing visible on the plates containing phK724 and phK555, and our transformation tests for the variant hns strains were less successful than we had hoped. 
</LI><LI>However our mood changed when Jim A told us we could not hope for many transformants with 20ul of strains that were not TOP10.  We decided to retest the strains with 100 ul plated out.{{
</LI><LI>AND we checked the 30 C incubator again and saw 2 colonies on one of the pHK555 plates, viewed under the camera they were clearly glowing so we gathered in the cold room and passed them around, as our eyes adapted we saw the colonies.  We even saw a third glowing colony which we had not spotted on the plate.
</LI></OL><H2><SPAN class="mw-headline" id="Friday"> Friday </SPAN></H2><P>Ligation appeared to have failed.
Jim A suggests that we heat to 65 degrees to melt short DNAs after restriction digest, plus that we clean up the DNA with a spin column without gel.
</P><H3><SPAN class="mw-headline" id="PM">PM</SPAN></H3><UL><LI> Hannah and Theo transformed TOP10, red, and black containing phK555 with pHK724 using Jim Slock's EZ Competence buffer
</LI><LI> Anja, Emily drew out various things on petri dishes.
</LI><LI> Set off for trip to lake district
</LI></UL><H2><SPAN class="mw-headline" id="Saturday"> Saturday </SPAN></H2><P>We spent the day in the Lake District, a beautiful part of the UK 250 miles North of Cambridge. 
</P><H2><SPAN class="mw-headline" id="Sunday"> Sunday </SPAN></H2><P>Went in to the lab on return:
Found that:
</P><UL><LI> TOP10 containing 5/7 (Theo&amp;Hannah, Friday) had been successfully transformed, glowing
</LI><LI> There were some colonies growing on ligation plate
</LI><LI> Logos were mostly dim, except at the edges (weekend incubation too long)
</LI></UL><P>Theo and Peter:
</P><UL><LI> Redrew logos from *very* turbid liquid culture of pHK555/724
</LI><LI> Prepared overnight cultures of:
</LI><LI> Two hns strains (details?)
</LI><LI> The potential ligations
</LI><LI> The firely luciferase, and backbones, for miniprep if necessary
</LI></UL><LI> + made up the 5mlsish of LB with pHK555/724 (Slock's strain) with 200ml LB
</LI></TD><TD style="width:630px; vertical-align:top"><H2><SPAN class="mw-headline" id="Monday_2">Monday</SPAN></H2><H3><SPAN class="mw-headline" id="7._Experiment:_Transformation_of_TOP10_cc_.28Ben.2C_Emily.2C_Bill.2C_Hannah.2C_Will_.26_Anja.29">7. Experiment: Transformation of TOP10 cc (Ben, Emily, Bill, Hannah, Will &amp; Anja)</SPAN></H3><P>Top10 cc taken out of -80°C freezer and tawed on ice. 1ml pipette tip cut with scissors sterilised with ethanol and flamed. Using cut pipette tip ~50μl of TOP10cc were transferred to 1.5ml Eppendorf tubes (3x)
</P><OL><LI>2μl of resuspended (in 10μl deionised water) BBa_J13002 (plasmiid with Tet<SUP>R</SUP> represed PoPs/RIPs generator) was added.
</LI><LI>2μl of resuspended (in 10μl deionised water) BBa_I712019 (plasmid with firefly luciferase) was added
</LI><LI>1.5μl of pHK724 (plasmid containing lux4 gene) was addded
</LI></OL><P>Ce]]s were held on ice for 30min. Heat shocked for 60s at 42°C (waterbath). As a control TOP10cc that had not been transformed with anything (no plasmid DNA added) were subjected to same treatment (as cells to be transformed). Put on ice for ~2min. Added 250μl pre-wardmed (in 37°C) SOC. Incubated at 37°C for Ph with rotation (Eppendorf tubes---&gt; 12ml falcon tubes, tape over top). Plated 50micorl on pre-warmed (in 37°C incubator) LB agar plates with Amp (with blue L-shaped spreader). Grow colonies overnight at 37°2.
</P><H3><SPAN class="mw-headline" id="8.Experiment:_Set_up_overnight_culture_of_E.coli.2FpHK555_.28Will_.26_Anja.29">8.Experiment: Set up overnight culture of E.coli/pHK555 (Will &amp; Anja)</SPAN></H3><P>Inoculated single bacterial colony (E.coli/pHK555) in 5ml LB in a 12ml falcon tube. Incubated at 37°C with rotation overnight.
(Since it was difficult to see the colony sent from Jim Slock, we picked twice from where we anticipated colony to be and set up 2x5ml LB overnight cultures)
</P><H3><SPAN class="mw-headline" id="9.Experiment:_Set_up_overnight_cultures_of_W3110_hns_93-1.2C_BW25113_.CE.94hns::kan.2C_W3110_hns-205::Tn10_TetR.2C_GM230_hns-205::Tn10_TetR_.28Ben_.26_Anja.29">9.Experiment: Set up overnight cultures of W3110 hns 93-1, BW25113 Δhns::kan, W3110 hns-205::Tn10 Tet<SUP>R</SUP>, GM230 hns-205::Tn10 Tet<SUP>R</SUP> (Ben &amp; Anja)</SPAN></H3><P>Inoculated single bacterial colonies in 5ml SOB (i.e. 4 different cultures), incubated at RT with shaking overnight.
</P><H2><SPAN class="mw-headline" id="Tuesday_2">Tuesday</SPAN></H2><H3><SPAN class="mw-headline" id="10._Experiment:_Preparing_chemically_competent_cells_.28W3110_hns_93-1.2C_BW25113_.CE.94hns::kan.2CW3110_hns-205::Tn10_TetR.2C_GM230_hns-205::Tn10_TetR_.28Ben.2C_Will.2C_Paul.2C_Bill.2C_Emily.2C_Hannah_.26_Anja.29">10. Experiment: Preparing chemically competent cells (W3110 hns 93-1, BW25113 Δhns::kan,W3110 hns-205::Tn10 Tet<SUP>R</SUP>, GM230 hns-205::Tn10 Tet<SUP>R</SUP> (Ben, Will, Paul, Bill, Emily, Hannah &amp; Anja)</SPAN></H3><P>(followed protocol for 'TOP10 chemically competent cells cells' from OpenWetWare)
Inoculated 0.5ml of the 4 bacterial strains (overnight cultures) in 85ml SOB each. Incubated at 37°C with shaking (180rpm). put on at 11:55am
</P><P>Bacterial Strains:
</P></TD></TR><TR><TH> ID
</TH><TH> Type
</TH></TR><TR><TD><FONT color="black">(1)</FONT></TD><TD> GM230 hns-205::Tn10 Tet<SUP>R</SUP></TD></TR><TR><TD><FONT color="red">(2)</FONT></TD><TD> BW25113 Δhns::kan
</TD></TR><TR><TD><FONT color="blue">(3)</FONT></TD><TD> W3110 hns 93-1
</TD></TR><TR><TD><FONT color="green">(4)</FONT></TD><TD> W3110 hns-205::Tn10 Tet<SUP>R</SUP></TD></TR></TBODY></TABLE><P>OD<SUB>600</SUB> measurements:
</P><TABLE class="wikitable"><TBODY><TR><TH> Time
</TH><TH> OD<SUB>600</SUB><FONT color="black">(1)</FONT></TH><TH> OD<SUB>600</SUB><FONT color="red">(2)</FONT></TH><TH> OD<SUB>600</SUB><FONT color="blue">(3)</FONT></TH><TH> OD<SUB>600</SUB><FONT color="green">(4)</FONT></TH></TR><TR><TD> 1:00pm
</TD><TD> 0.019
</TD><TD> -0.008
</TD><TD> -0.020
</TD><TD> 0.049
</TD></TR><TR><TD> 2:00pm
</TD><TD> 0.034
</TD><TD> 0.027
</TD><TD> 0.001
</TD><TD> 0.124
</TD></TR><TR><TD> 3:00pm
</TD><TD> 0.059
</TD><TD> 0.114
</TD><TD> 0.049
</TD><TD> 0.268
</TD></TR><TR><TD> 3:25pm
</TD><TD> 0.092
</TD><TD> 0.235
</TD><TD> 0.116
</TD><TD> -
</TD></TR><TR><TD> 3:40pm
</TD><TD> -
</TD><TD> 0.297
</TD><TD> -
</TD><TD> -
</TD></TR><TR><TD> 4:00pm
</TD><TD> 0.159
</TD><TD> -
</TD><TD> 0.267
</TD><TD> -
</TD></TR><TR><TD> 4:25pm
</TD><TD> 0.203
</TD><TD> -
</TD><TD> -
</TD><TD> -
</TD></TR><TR><TD> 4:40pm
</TD><TD> 0.256
</TD><TD> -
</TD><TD> -
</TD><TD> -
</TD></TR></TBODY></TABLE><P>Cooled cells and (newly prepared) CCMB80 Buffer in ice bath for ~20 minutes. <FONT color="green">(4)</FONT> was on ice for over an hour. <FONT color="red">(2)</FONT> was on ice for ~40 minutes. <FONT color="blue">(3)</FONT> was on ice for ~40 minutes.
</P><P>2 x 30-35ml of each of the  four 85ml cultures were poured into 50ml falcon tubes. Centrifuged at 3000g at 4°C for 10 min. Discarded supernatant. Resuspended cell pellet in 20ml tube CCMB80 buffer (ice cold). On ice for 20 min. Centrifuged at 3000g at 4°C for 10 min. Discarded supernatant. Resuspended cell pellet in 5ml tube CCMB80 buffer. Incubated on ice for 20 min. Aliquoted 200μl portions into Eppendorf tubes colour-coded according to the above listed colour labels. Stored at -80°C
</P><H3><SPAN class="mw-headline" id="11._Experiment:_Streaking_out_of_bacterial_cultures_.28E._coli.2FpHK555.29_.28Paul_.26_Anja.29">11. Experiment: Streaking out of bacterial cultures (<I>E. coli</I>/pHK555) (Paul &amp; Anja)</SPAN></H3><P>On LB agar plates with Chloranphenicol: <I>E. coli</I>/pHK555 (from both overnight cultures) on two individual plates incubated at 37°C overnight.
</P><H3><SPAN class="mw-headline" id="12._Experiment:_Set_up_overnight_cultures_.28TOP10_with_BBa-J13002.2C_TOP10_with_BBa_I712019.29_.28Ben_.26_Anja.29">12. Experiment: Set up overnight cultures (TOP10 with BBa-J13002, TOP10 with BBa_I712019) (Ben &amp; Anja)</SPAN></H3><P>Results from transformations on 02/08/10
</P><UL><LI> Colonies grew for TOP10 cells transformed with BBa_J13002 (promoter + rbs) and those transformed with BBa_I712019 (firefly luciferase) :-)
</LI><LI> no cells grew for untransformed TOP10 cells plated on LB agar + Amp plates (neg control)
</LI><LI> small (but plenty) colonies grew for TOP10 cells transformed with pHK724, these were left in the incubator for another 24h, and then put at 4°C
</LI></UL><P>Inoculated single bacterial (TOP10 with BBa-J13002 and TOP10 with BBa_I712019 individually) colony in 5ml LB, incubated at 37°C with shaking (180rpm) overnight
</P><H2><SPAN class="mw-headline" id="Wednesday_2">Wednesday</SPAN></H2><H3><SPAN class="mw-headline" id="13._Experiment:_Measuring_competency_.28W3110_hns_93-1.2C_BW25113_.CE.94hns::kan.2C_W3110_hns-205::Tn10_TetR.2C_GM230_hns-205::Tn10_TetR.29_.28Theo_.26_Hannah.29">13. Experiment: Measuring competency (W3110 hns 93-1, BW25113 Δhns::kan, W3110 hns-205::Tn10 Tet<SUP>R</SUP>, GM230 hns-205::Tn10 Tet<SUP>R</SUP>) (Theo &amp; Hannah)</SPAN></H3><P>Followed protocol under 'TOP10 chemically competent cells' from OpenWetware
</P><P>Competent cells from all 4 different strains taken out of -80°C freezer and thawed on ice. 1ml pipette tips  cut with scissors dipped in ethanol and flamed. For each strain 50μl of cells were transferred to a separate 1.5ml Eppendorf tube. 1μl pUC19 (standard plasmid) was added to each 50μl of cells. Held on ice for 30 min. Heat shocked at 42°C for 60s (water bath). On ice for ~2min. 250μl SOC was added to each of 4 tubes. Each Eppendorf tube was put in a 12ml falcon with tape over the top and incubated at 37°C for 1h with rotation. 20μl of each of the 4 different transformed strains were plated on LB agar plages with Amp. Colonies were grown overnight at 37°C.
</P><H3><SPAN class="mw-headline" id="14._Experiment:_Making_competent_and_transforming_E._coli.2FpHK555_.28Peter_.26_Paul.29">14. Experiment: Making competent and transforming <I>E. coli</I>/pHK555 (Peter &amp; Paul)</SPAN></H3><P>50μl EZ Comp  buffer transferred to Eppendorf tube and placed on ice. Single <I>E. coli</I>/pHK555 colony resuspended in EZ Comp buffer(2x single colony in 50μl EZ each, 1 streak of colonies in another 50μl EZ). Vortex if needed to suspend clumps. 3μl of pHK724 plasmid added to solution, mixed and incubated on ice for 20 min.
</P><P>Heaat shocked at 42°C for 90s (water bath). Incubated at RT for 5 min. 1ml LB added. Eppendorf tubes put in 12ml falcon tubes with tape over top. Incubated at 37°C for 1h with rotation. Plated 100μl on LB agar plates with Cm and Amp. Incubated at 30°C for 48h.
</P><H3><SPAN class="mw-headline" id="15._Experiment_BioBrick_Standard_Assembly_.28Emily_.26_Bill.2C_Paul_.26_Anja.29">15. Experiment BioBrick Standard Assembly (Emily &amp; Bill, Paul &amp; Anja)</SPAN></H3><P>Took overnight cultures of 
</P><OL><LI> TOP10 with BBa_J13002 (plasmid with promoter + rbs)
</LI><LI> TOP10 with BBa_I712019 (plasmid with firefly luciferase)
</LI></OL><H4><SPAN class="mw-headline" id="Plasmid_DNA_Purification">Plasmid DNA Purification</SPAN></H4><P>following the &quot;QIAprrep Spin Miniprep Kit using a Microcentrifuge&quot; protocol.
</P><H4><SPAN class="mw-headline" id="Restriction_enzyme_digest">Restriction enzyme digest</SPAN></H4><P>Prepared at RT in the listed order:
</P><TABLE class="wikitable"><TBODY><TR><TH> promoter + rbs
</TH><TH> luciferase
</TH></TR><TR><TD> nuclease-free H<SUB>2</SUB>O
</TD><TD> 15μl
</TD><TD> 14μl
</TD></TR><TR><TD> 10x Fast Digest Buffer
</TD><TD> 2μl
</TD><TD> 2μl
</TD></TR><TR><TD> Plasmid DNA
</TD><TD> 2μl
</TD><TD> 2μl
</TD></TR><TR><TD> FD enxyme SpeI
</TD><TD> 1μl
</TD><TD> 1μl
</TD></TR><TR><TD> FD enzyme XbaI
</TD><TD> -
</TD><TD> 1μl
</TD></TR></TBODY></TABLE><P>It was checked (Nanodrop!) that 2μl of plasmid DNA would not contain more than 1μg of DNA. The reaction mixtures were mixed gently (flick tube) and spun in the microcentrifuge for 15s (13,000 rpm). It was then incubated at 37°C (water bath) for 30 min.
</P><H4><SPAN class="mw-headline" id="Gel_Electrophoresis">Gel Electrophoresis</SPAN></H4><P>An E-gel EX Agarose 1% was mounted on the transilluminator. Nanodrop readings were taken for both restriction enzyme digests:
</P><UL><LI>Plasmid with promoter + rbs: 21.4 ng/μl =&gt; 5μl gives 100ng
</LI><LI>Plasmid with luciferase: 19.7ng/μl =&gt; 5μl gives 100ng
</LI></UL><P>For each digest the following mixture was made up
</P><UL><LI> 3μl 6X orange loading Dye
</LI><LI> 5μl plasmid DNA (restriction digest)
</LI><LI> 12μl deionised H<SUB>2</SUB>O
</LI></UL><P>Gel was loaded according to the scheme below:
</P><TABLE class="wikitable"><TBODY><TR><TD> Easyladder II
</TD><TD> promoter + rbs
</TD><TD> promoter + rbs
</TD><TD> luciferase
</TD><TD> luciferase
</TD></TR><TR><TD> 10μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD></TR></TBODY></TABLE><P>Empty wells were filled with 20μl deionised H<SUB>2</SUB>O.
Gel was run and DNA gragments of appropriate sizes were observed:
</P><UL><LI> plasmid with promoter andd rbs =&gt; 2153b
</LI><LI> plasmid without luciferase =&gt; 3426b
</LI><LI> firefly luciferase =&gt; 1653b
</LI></UL><P>The plasmid with promoter + rbs as well as te firefly luciferase bands were cut out of the gel (gel cut pipette tips) and DNA was extracted separately from the two gel pieces by following the QIAquick Gel Extraction Kit protocol.
</P><H4><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H4><P>Fermentas 5X Rapid ligation buffer was taken out of -20°C fridge, thawed and mixed thoroughly prior to use. Nanodrop measurements of the linearised vector DNA as well as the insert DNA were taken:
</P><UL><LI> plasmid with promoter + rbs: 56.8ng/μl
</LI><LI> firefly luciferase: 29.6ng/μl
</LI></UL><P>10-100ng of linearised vector DNA should be added to the ligation mix=&gt;1μl. 3:1 molar exess  of insert DNA should be added to the ligation mix.
</P><P>(56.8ng.2153b) * 1653b * 3 = 131ng =&gt; 4.5μl
</P><P>Added to a mirocentrifuge tube:
</P><TABLE class="wikitable"><TBODY><TR><TD> Plasmid with promoter and rbs
</TD><TD> 1μl
</TD></TR><TR><TD> Firefly luciferase
</TD><TD> 4.5μl
</TD></TR><TR><TD> 5X Rapid ligation buffer
</TD><TD> 4μl
</TD></TR><TR><TD> T4 DNA ligase
</TD><TD> 1μl
</TD></TR><TR><TD> nuclease-free H<SUB>2</SUB>O
</TD><TD> 9.5μl
</TD></TR><TR><TD> 20μl
</TD></TR></TBODY></TABLE><P>The mix was vortexed and spun for 15s in the microcentrifuge (13,000 rpm). It was incubated at 22°C (RT) for 30 min and stored at 4°C.
</P><H3><SPAN class="mw-headline" id="16._Experiment:_Set_up_new_overnight_cultures_.28TOP10_with_BBa_J13002.2C_TOP10_with_Ba_I712019.29_.28Peter_.26_Anja.29">16. Experiment: Set up new overnight cultures (TOP10 with BBa_J13002, TOP10 with Ba_I712019) (Peter &amp; Anja)</SPAN></H3><H2><SPAN class="mw-headline" id="Thursday_2">Thursday</SPAN></H2><H3><SPAN class="mw-headline" id="17._Experiment:_Measuring_competency_.28W3110_hns_93-1.2C_BW25113_.CE.94hns:_kan.2C_W3110_hns-205::Tn10_TetR.2C_GB_230_hns-205::Tn10_TetR.29_.28Theo_.26_Peter.29">17. Experiment: Measuring competency (W3110 hns 93-1, BW25113 Δhns: kan, W3110 hns-205::Tn<SUP>10</SUP> Tet<SUP>R</SUP>, GB 230 hns-205::Tn<SUP>10</SUP> Tet<SUP>R</SUP>) (Theo &amp; Peter)</SPAN></H3><P>After 24h the pUC19 transfmration from 04/08/10 had yielded:
</P><TABLE style="wikitable"><TBODY><TR><TD> W3110 hns 93-1
</TD><TD> 0 colonies/plate
</TD></TR><TR><TD> BW25113 Δhns::kan
</TD><TD> 13 colonies/plate
</TD></TR><TR><TD> W3110 hns-205::Tn10 Tet<SUP>R</SUP></TD><TD> 1 colony/plate
</TD></TR><TR><TD> GM230 hns-205::Tn10 Tet<SUP>R</SUP></TD><TD> 10 colonies/plate (many v small ones-&gt; prob result of degraded Amp)
</TD></TR></TBODY></TABLE><P>pUC19 transformations were repeated following the protocl outlined on 03/08/10, with the exeption that not 20μl but 100μl of the transfomrd strains were plated on LB agar plates with Amp.
As a control 100μl of the transformed strains were plated on LB agar plates without anti-biotics.
</P><H3><SPAN class="mw-headline" id="18._Experiment:_Standard_BioBrick_Assembly_.28Emily.2C_Bill_.26_Anja.29">18. Experiment: Standard BioBrick Assembly (Emily, Bill &amp; Anja)</SPAN></H3><P>Took oversight cultures of:
</P><OL><LI>TOP10 with BBa_J13002 (plasmid with promoter + rbs)
</LI><LI>TOP10 with BBa_I712019 (plasmid with firefly luciferase)
</LI></OL><H4><SPAN class="mw-headline" id="Plasmid_DNA_purification_2">Plasmid DNA purification</SPAN></H4><P>following the &quot;QIAprep Spin Miniprep Kit using a Microcentrifuge&quot; protocol
</P><H4><SPAN class="mw-headline" id="Restriction_enzyme_digest_2">Restriction enzyme digest</SPAN></H4><P>Prepared at RT in the listed order:
</P><TABLE class="wikitable"><TBODY><TR><TH> promoter + rbs
</TH><TH> luciferase
</TH></TR><TR><TD> nuclease-free H<SUB>2</SUB>O
</TD><TD> 15μl
</TD><TD> 14μl
</TD></TR><TR><TD> 10x Fast Digest Buffer
</TD><TD> 2μl
</TD><TD> 2μl
</TD></TR><TR><TD> Plasmid DNA
</TD><TD> 2μl
</TD><TD> 2μl
</TD></TR><TR><TD> FD enxyme SpeI
</TD><TD> 1μl
</TD><TD> -
</TD></TR><TR><TD> FD enzyme PstI
</TD><TD> 1μl
</TD><TD> 1μl
</TD></TR><TR><TD> FD enzyme XbaI
</TD><TD> -
</TD><TD> 1μl
</TD></TR><TR><TD> 20μl
</TD><TD> 20μl
</TD></TR></TBODY></TABLE><P>It was checked (Nanodrop!) that 2μl of plasmid DNA would not contain more than 1μg of DNA. The reaction was mixed gentlyy, spun down and incubated at 37°C for 30 min
</P><H4><SPAN class="mw-headline" id="Gel_electrophoresis_2">Gel electrophoresis</SPAN></H4><P>E-gel EX Agarose 1% was mounted on the transilluminator. Nanodrop readings were taken for both reaction enzyme digests
</P><UL><LI>plasmid with promoter + rbs 27.7ng/μl =&gt; 4μl give 100ng
</LI><LI>plasmid with luciferase 29.1ng/μl =&gt; 4μl give 100ng
</LI></UL><P>Prepared following mixture for both digests
</P><UL><LI> 3μl 6X orange loading Dye
</LI><LI> 5μl plasmid DNA (restriction digest)
</LI><LI> 12μl deionised H<SUB>2</SUB>O
</LI></UL><P>Gel was loaded according to the scheme below:
</P><TABLE class="wikitable"><TBODY><TR><TD> Easyladder II
</TD><TD> promoter + rbs
</TD><TD> promoter + rbs
</TD><TD> luciferase
</TD><TD> luciferase
</TD></TR><TR><TD> 10μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD></TR></TBODY></TABLE><P>Empty wells were filled with 20μl deionised H<SUB>2</SUB>O. Gel was run and DNA bands of length corresponding to plasmid with promoter + rbs (2153b) as well as firefly luciferase (16553b) were cut out the gel. DNA was extracted separately from the two gel pieces by following the QIAquick Gel Extraction Kit Protocol
</P><H4><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H4><P>Fermentas 5X Rapid ligation buffer was taken out the -20°C freezer, thawed on ice and mixed thoroughly.
</P><P>Nanodrop measurements
</P><TABLE class="wikitable"><TBODY><TR><TD> Plasmid with promoter + rbs
</TD><TD> 2.0ng/μl
</TD></TR><TR><TD> Firefly luciferase
</TD><TD> 3.1ng/μl
</TD></TR></TBODY></TABLE><P>10-100ng of linearised vector DNA should be used in ligation mix =&gt; 10μl (20ng). 3:1 molar excess of insert DNA should be added to ligation mix.
</P><P>(20ng/2153b) * 1653b * 3 = 46ng =&gt; 15μl (46.5ng)
</P><P>Added to a microcentrifuge tube:
</P><TABLE class="wikitable"><TBODY><TR><TD> Plasmid with promoter + rbs
</TD><TD> 10μl
</TD></TR><TR><TD> Firefly luciferase
</TD><TD> 15μl
</TD></TR><TR><TD> 5X Rapid ligation buffer
</TD><TD> 8μl
</TD></TR><TR><TD> T4 DNA ligase
</TD><TD> 2μl
</TD></TR><TR><TD> Nuclease-free H<SUB>2</SUB>O
</TD><TD> 5μl
</TD></TR><TR><TD> 40μl
</TD></TR></TBODY></TABLE><P>Vortexed, spun briefly and incubated at 22°C (RT) fr 30 min
</P><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>TOP10 cc taken out of -80°C freezer and thawed on ice, 1ml pipette tip cut with scissors sterilised with ethanol and flamed. 50μl of TOP10cc were transferred to 1.5ml Eppendorf tube. 2μl of ligation reaction were added. Cells were held on ice for 30 min. Heat shocked for 60s at 42°C (water bath). Put on ice for ~2min. Added 250μl SOC. Incubated at 37°C for 1h with rotation. Plated 100μl on LB agar plates with Amp. Incubated overnight at 37°C.
</P><H3><SPAN class="mw-headline" id="19._Experiment:_Set_up_overnight_cultures_.28E._coli.2FpHK555_transformed_with_pHK724.29_.28Theo_.26_Anja.29">19. Experiment: Set up overnight cultures (<I>E. coli</I>/pHK555 transformed with pHK724) (Theo &amp; Anja)</SPAN></H3><P>2 cyan-glowing colonies of <I>E. coli</I>/pHK555 transformed with pHK724 had grown.
</P><P>One of these was inoculated in 5ml LB and incubated at 30°C overnight.
</P><H3><SPAN class="mw-headline" id="20._Experiment:_Isolation_of_pHK555_and_pHK724_and_transformation_of_TOP10.2C_BW25113_.CE.94hns::kan_and_GM230_hns-205::Tn10_TetR_.28Hannah_.26_Will.29">20. Experiment: Isolation of pHK555 and pHK724 and transformation of TOP10, BW25113 Δhns::kan and GM230 hns-205::Tn10 Tet<SUP>R</SUP> (Hannah &amp; Will)</SPAN></H3><P>Took an <I>E. coli</I>/pHK555 and a TOP10/pHK724 colony and separately purified plasmids following the &quot;QIAprep Spin Miniprep Kit using a Microcentrifuge&quot; protocol. Nanodrop measurements.
TOP10, BW25113 Δhns::kan and GM230 hns-205::Tn10 Tet<SUP>R</SUP> cells were transformed with both plasmids together and separately. Control cells were left untransformed.
Transformation was performed according to the protocol in experiment 18 (instead of ligation reaction, pHK555 (6μl) and pHK724 (7μl) were added to the commp cells. TOP10 cc transformed with both plasmids got 5μl of pHK724 only). Transformed cells were plated on different antibiotic-containing plates as listed in the table below:
</P><TABLE class="wikitable"><TBODY><TR><TH> TOP10
</TH><TH> BW25113 Δhns::kan
</TH><TH> GM230 hns-205::Tn10 Tet<SUP>R</SUP></TH></TR><TR><TD> pHK555 &amp;</TD><TD> Cm+Amp
</TD><TD> Cm+Amp
</TD><TD> Cm+Amp
</TD></TR><TR><TD> pHK725
</TD><TD> Cm || Amp
</TD><TD> Cm || Amp
</TD><TD> Cm || Amp
</TD></TR><TR><TD> No antibiotic
</TD><TD> No antibiotic
</TD><TD> No antibiotic
</TD></TR><TR><TD> pHK555
</TD><TD> Cm
</TD><TD> Cm
</TD><TD> Cm
</TD></TR><TR><TD> No antibiotic
</TD><TD> No antibiotic
</TD><TD> No antibiotic
</TD></TR><TR><TD> pHK724
</TD><TD> Amp
</TD><TD> Amp
</TD><TD> Amp
</TD></TR><TR><TD> No antibiotic
</TD><TD> No antibiotic
</TD><TD> No antibiotic
</TD></TR><TR><TD> none
</TD><TD> No antibiotic
</TD><TD> No antibiotic
</TD><TD> No antibiotic
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="21._Experiment:_Set_up_overnight_cultures_.28E._coli.2FpHK555_and_TOP10.2FpHK724.29_.28Hannah_.26_Will.29">21. Experiment: Set up overnight cultures (<I>E. coli</I>/pHK555 and TOP10/pHK724) (Hannah &amp; Will)</SPAN></H3><H2><SPAN class="mw-headline" id="Friday_2">Friday</SPAN></H2><H3><SPAN class="mw-headline" id="22._Results:_Measuring_competency_.28W3110_hns_93-1.2C_BW25113_.CE.94hns::kan.2C_W3110_hns-205::Tn10_TetR.2C_GM230_hns-205::Tn10_TetR">22. Results: Measuring competency (W3110 hns 93-1, BW25113 Δhns::kan, W3110 hns-205::Tn10 Tet<SUP>R</SUP>, GM230 hns-205::Tn10 Tet<SUP>R</SUP></SPAN></H3><P>After 24h the pUC19 transformation from Thursday had yielded:
</P><TABLE class="wikitable"><TBODY><TR><TD> W3110 hns 93-1
</TD><TD> 1 colony/plate
</TD></TR><TR><TD> BW25113 Δhns;:kan
</TD><TD> ~60 colonies/plate
</TD></TR><TR><TD> W33110 hns-205::Tn10 Tet<SUP>R</SUP></TD><TD> 6 colonies/plate (many tiny ones)
</TD></TR><TR><TD> GM230 hns-205::Tn10 Tet<SUP>R</SUP></TD><TD> ~30 colonies/plate (many tiny ones)
</TD></TR></TBODY></TABLE><P>Control: All cells grew fine on LB agar plates without antiiotic.
</P><H3><SPAN class="mw-headline" id="23._Results:_Transformation_of_ligated_promoter_.2B_rbs_and_luciferase">23. Results: Transformation of ligated promoter + rbs and luciferase</SPAN></H3><P>After 24h no colonies weree observed on the LB agar + Amp plate.
</P><H3><SPAN class="mw-headline" id="24._Results:_Overnight_culture_of_glowing_E._coli.2FpHK555_transformed_with_pHK724">24. Results: Overnight culture of glowing <I>E. coli</I>/pHK555 transformed with pHK724</SPAN></H3><P>The culture glowed (cyan) in the dark.
</P><H3><SPAN class="mw-headline" id="25._Results:_Transformation_of_TOP10.2C_BW25113_.CE.94hns::kan_and_GM230_hns-205::Tn10_TetR_with_pHK555_and_pHK724">25. Results: Transformation of TOP10, BW25113 Δhns::kan and GM230 hns-205::Tn10 Tet<SUP>R</SUP> with pHK555 and pHK724</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> TOP10
</TH><TH> bw25113 Δhns::kaan
</TH><TH> GM230 hns-205
</TH></TR><TR><TD> Cm &amp; Amp
</TD><TD> No colonies
</TD><TD> No colonies
</TD><TD> No colonies
</TD></TR><TR><TH> pHK555 &amp;</TH><TD> Cm
</TD><TD> Few (5)
</TD><TD> Few, normal size
</TD><TD> Few
</TD></TR><TR><TH> pHK724
</TH><TD> Amp
</TD><TD> Many, small
</TD><TD> Many, small
</TD><TD> No colonies
</TD></TR><TR><TD> No antibiotic
</TD><TD> Smear
</TD><TD> Smear
</TD><TD> Smear
</TD></TR><TR><TH> pHK724
</TH><TD> Amp
</TD><TD> Many, small
</TD><TD> Many, small
</TD><TD> Many diff sizes
</TD></TR><TR><TD> No antibiotic
</TD><TD> Smear
</TD><TD> Smear
</TD><TD> Smear
</TD></TR><TR><TH> pHK555
</TH><TD> Cm
</TD><TD> Large colonies
</TD><TD> Many, large
</TD><TD> None
</TD></TR><TR><TD> No antibiotic
</TD><TD> Smear
</TD><TD> Smear
</TD><TD> Smear
</TD></TR><TR><TH> none
</TH><TD> No antibiotic
</TD><TD> Smear
</TD><TD> Smear
</TD><TD> Smear
</TD></TR></TBODY></TABLE><P>No bacteria from any of the three strains took up both plasmids simultaneously, although when both the plasmids were added to a strain simultaneously there was growth on both of the individual resistance plates – i.e. some bacteria had taken up one and some the other but the probability that any would take up both was so low that no colonies were observed on the ChlAmp plate.
No bacteria strains were killed by the heat shocking as smears of bacteria were seen on every plate without antibiotics added that also had a transformed strain plated. 
The takeup of plasmid 7 was successful in Invitrogen Top 10 strain and in the Red Strain.
</P><P>In the Black strain, the results from the Black strain are less clear – when we added both plasmids there was no colony growth on the Ampicillin plate (plasmid 7 indicates ampicillin resistance) therefore no plasmid 7 was taken up, however we observed growth on the plate which had been transformed by only plasmid 7 then added to an Amp plate. The colony size was extremely variable though, and although there was no colour differences visible we hypothesise that it may have been contaminated during the procedure. 
</P><H3><SPAN class="mw-headline" id="26._Experiment:_Streaking_out_of_E._coli.2FpHK555_trasnformed_with_pHK724_.28Paul.2C_Emily_.26_Anja.29">26. Experiment: Streaking out of <I>E. coli</I>/pHK555 trasnformed with pHK724 (Paul, Emily &amp; Anja)</SPAN></H3><P>Streaked out on a LB agar plate with Cm and Amp
</P><P>Used to write out 2x 'Cambridge', 'Sterilin' &amp; 'iGEM 2010' on LB agar plates with Amp.
</P><H3><SPAN class="mw-headline" id="27._Experiment:_of_plasmids_pHK724_.26_pHK555_from_TOP10_.26_SLOCK_10_cells_respectively_.28Will_.26_Emily.29">27. Experiment: of plasmids pHK724 &amp; pHK555 from TOP10 &amp; SLOCK 10 cells respectively (Will &amp; Emily)</SPAN></H3><P>Followed &quot;QIAprep Spin Miniprep Kit&quot; protocl from QIAGEN
4 ml of overnight cultures from each (prepared by Will &amp; Hanah) of TOP10/pHK724 and <I>E. coli</I>/pHK555 were used to extract plasmids.
</P><H4><SPAN class="mw-headline" id="Results_2">Results</SPAN></H4><P>50μl of sultion was obtained, 1μl of which was used to NanoDrop
</P><TABLE class="wikitable"><TBODY><TR><TH> plasmid
</TH><TH> Nucleic acid conc<SUP>n</SUP></TH></TR><TR><TD> pHK555
</TD><TD> 34.1ng/μl
</TD></TR><TR><TD> pHK724
</TD><TD> 15.7ng/μl
</TD></TR></TBODY></TABLE><P>plasmids were stored at 4°C
</P><H3><SPAN class="mw-headline" id="28._Experiment:_Making_long-term_stocks_of_pHK555_.28in_Slock10.29_.26_pHK724_.28in_TOP10.29_.28Peter_.26_Emily.29">28. Experiment: Making long-term stocks of pHK555 (in Slock10) &amp; pHK724 (in TOP10) (Peter &amp; Emily)</SPAN></H3><H4><SPAN class="mw-headline" id="Preparation_of_40.25_glycerol_solution">Preparation of 40% glycerol solution</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TD> 100% glycerol
</TD><TD> 40ml
</TD></TR><TR><TD> Deionized sterilised water
</TD><TD> 60ml
</TD></TR></TBODY></TABLE><P>For pHK555 &amp; pHK724, separately:
</P><UL><LI> Added 1ml of 40% glycerol solution to cryogenic vial
</LI><LI> Added 1ml of sample culture to vial
</LI><LI> Gently vortexed vial to mix
</LI></UL><P>Stored in -80°C freezer in 'Black-GM230 hns-205' storage box. Vials are labelled with stickers: '7' for pHK724, '5' for pHK555.
</P><H3><SPAN class="mw-headline" id="29._Experiment:_Transformation_of_TOP10.2C_Red_strain_and_Black_strain_with_plasmid_pHK724._They_have_each_already_been_transformed_with_plasmid_pHK555_.28Hannah_.26_Theo.29">29. Experiment: Transformation of TOP10, Red strain and Black strain with plasmid pHK724. They have each already been transformed with plasmid pHK555 (Hannah &amp; Theo)</SPAN></H3><H4><SPAN class="mw-headline" id="Aim">Aim</SPAN></H4><P>We hope to see each colony glowing, with different brightnesses
</P><H4><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H4><UL><LI>Followed Jim Stock's protocol using his EX comp buffer
</LI><LI>Left growing in 30°C incubator over a few days
</LI></UL><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> Strain
</TH><TH> Plasmid Added
</TH><TH> Glowing
</TH><TH> Colonies
</TH></TR><TR><TD> TOP10 with pHK555
</TD><TD> pHK724
</TD><TD> Yes
</TD><TD> Yes
</TD></TR><TR><TD> Red strain with pHK555
</TD><TD> pHK724
</TD><TD> No
</TD><TD> Yes
</TD></TR><TR><TD> Black strain with pHK555
</TD><TD> pHK724
</TD><TD> No
</TD><TD> Yes
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H4><P>Only TOP10 successful however colonies seen on both of red and black strain plates.
</P><H4><SPAN class="mw-headline" id="Further_Work">Further Work</SPAN></H4><P>Plate out red and black strain colonies - possible that they had been left to incubate for too long and glowed then stopped glowing.
</P><H2><SPAN class="mw-headline" id="Sunday_2">Sunday</SPAN></H2><H3><SPAN class="mw-headline" id="30._Experiment:_Various">30. Experiment: Various</SPAN></H3><UL><LI> Wrote out in Slock strain, placed in 30°C incubation overnight
</LI><LI> iGEM 2010
</LI><LI> Sterilin
</LI><LI> Cambridge
</LI><LI> We got Glowth
</LI></UL><UL><LI> Made up the ~3ml of Slock growth strain to 200ml, added antibiotics (A+C)
</LI></UL><UL><LI> Inoculated 5ml of LB with
</LI><LI> W3110 hns-205::Tet<SUP>R</SUP> (+T)
</LI><LI> GM230 hns-205::Tn10 Tet<SUP>R</SUP> (+T)
</LI><LI> pHK555 trans 724 (glowth in Slock) (A+C)
</LI><LI> Top10 trans promoter + rbs (A)
</LI><LI> Top10 trans luciferase (A)
</LI><LI> Top10 ligation (A)
</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Cambridge/Notebook/Week4">http://2010.igem.org/Team:Cambridge/Notebook/Week4</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Notebook/Week4" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Notebook/Week4" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week4&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week4&amp;oldid=52936" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>