<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Notebook_Week6"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Notebook/Week6">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Notebook/Week6&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week6&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week6&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Notebook/Week6" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Notebook/Week6</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="wrap"><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:Cambridge">Home</A></LI><LI><A href="/Team:Cambridge/TheTeam">Meet the Team</A></LI><LI><A href="/Team:Cambridge/Photos">Photo Gallery</A></LI><LI><A href="/Team:Cambridge/Videos">Videos</A></LI><LI><A href="/Team:Cambridge/BioBricks">Parts submitted</A></LI></UL><LI><A class="head" href="#">Project Firefly</A></LI><LI><A href="/Team:Cambridge/Bioluminescence">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background_Firefly">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Colour">Coloured outputs</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Luciferin_Regeneration">Luciferin recovery</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Project Vibrio</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Luciferases">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/G28">The LuxBrick</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">Codon optimisation</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:Cambridge/Tools/Gibson">Gibthon</A></LI><LI><A href="/Team:Cambridge/Tools/GenBank">BioBrick → GenBank</A></LI><LI><A href="/Team:Cambridge/Tools/Ligate">Ligation Calculator</A></LI><LI><A href="/Team:Cambridge/Tools/Eglometer">The E.glometer</A></LI><LI><A class="head" href="#">Gibson Assembly</A></LI><LI><A href="/Team:Cambridge/Gibson/Introduction">Introduction</A></LI><LI><A href="/Team:Cambridge/Gibson/Mechanism">How it works</A></LI><LI><A href="/Team:Cambridge/Gibson/Protocol">Protocol</A></LI><LI><A href="/Team:Cambridge/Gibson/RFC">RFC</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:Cambridge/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Cambridge/Safety">Safety</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:Cambridge/Tools/microMeasure">Biosensors</A></LI><LI><A href="/Team:Cambridge/Tools/Lighting">Lighting</A></LI><LI><A class="head" href="/Team:Cambridge/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:Cambridge/Notebook/Summary">Summary</A></LI><LI><A href="/Team:Cambridge/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:Cambridge/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:Cambridge/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:Cambridge/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:Cambridge/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:Cambridge/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:Cambridge/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:Cambridge/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:Cambridge/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:Cambridge/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:Cambridge/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:Cambridge/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:Cambridge/Notebook/FurtherWork">Beyond Week 12</A></LI><LI><A href="/Team:Cambridge/Protocols">Protocols</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">Our Partners</A></LI><LI><A href="/Team:Cambridge/Partners">Sponsors</A></LI><LI><A href="/Team:Cambridge/iGEM_Teams">iGEM Teams</A></LI><LI><A href="/Team:Cambridge/Departments">University</A></LI><LI><A href="/Team:Cambridge/Publicity">Publicity</A></LI></DIV><DIV id="right"><DIV style="width:730px; background-color:#fad72a;" class="secheader">Notebook: Week 6</DIV><DIV style="position:relative; height:30px;border-top:5px solid white;"><DIV style="width:492px;position:absolute; background-color:#96d446;" class="selecteddiv" id="projectlink"><P>Viewing project diary</P></DIV><DIV style="width:246px; position:absolute; background-color:#fb5c2b;left:497px;" class="selectordiv" id="lablink"><P>View protocols and data</P></DIV></DIV><DIV style="overflow:hidden; height:2200px; width:730px; position:relative;" id="maskingbox"><DIV style="width:1320px; position:relative;" id="superwide"><TABLE style="width:1480px" id="widetable"><TBODY><TR><TD style="padding-right:50px; padding-left:10px; width:630px;vertical-align:top"><DIV align="center">Week 6: Monday 16th - Sunday 22nd August</DIV><H2><SPAN class="mw-headline" id="Monday"> Monday</SPAN></H2><P>In the morning we checked the BMG microplate reader which had been run over the weekend and provided interesting growth curves, quantifying the extent to which adding chloramphenicol to the medium slowed growth (this was by c. 50%).  Bill miniprepped lux R DNA from the registry. James told us that to use the standard Gibson protocol we needed 40bp of overlap, not the 20bp we planned which required rethinking oligos.  Peter prepared a number of cultures to test the effect of lumazine on emissions.
</P><P>In the evening Hannah, Peter, Anja and Theo combined take-aways and beer with primer design, which went surprisingly well.
</P><H2><SPAN class="mw-headline" id="Tuesday"> Tuesday </SPAN></H2><P>In the morning we sent off an order to Biolegio for oligo synthesis, in order to PCR out the Vibrio fischeri lux operon.  Will also designed oligos for the site-directed mutagenesis of Luciola cruciata luciferase into different colours, ready from when it arrives from DNA 2.0.
</P><P>We decided to have a synthetic lux operon synthesised, relieving HNS repression.  Theo and Anja transformed TOP10 cells with Enhanced Yellow Fluorescent protein and a TetR repressed promoter from the registry, the latter will be used to control the firely operons.  Will and Hannah set up an experiment in the BMG plate reader which was our first test of its ability to detect bioluminescent emissions.
</P><H2><SPAN class="mw-headline" id="Wednesday"> Wednesday </SPAN></H2><P>Bill wrote a piece of software in Python to simplify primer design for Gibson Assembly.  Theo and Hannah designed a synthetic V. fischeri lux operon with codon usage and GC content optimised. Will prepared DNA for use with the oligos we hope will arrive tomorrow.  Paul analysed results from the BMG plate reader.
</P><H2><SPAN class="mw-headline" id="Thursday"> Thursday </SPAN></H2><P>Anja continued transferring the RBS and luciferase part into psB1C3 for submission to the registry.  Theo and Hannah sent their synthetic lux operon to Mr. Gene for synthesis.  Peter and Ben (who has returned from Edinburgh!) designed an experiment for characterising the existing registry firefly luciferase.
</P><H2><SPAN class="mw-headline" id="Friday">Friday</SPAN></H2><H2><SPAN class="mw-headline" id="Saturday">Saturday</SPAN></H2><H2><SPAN class="mw-headline" id="Sunday">Sunday</SPAN></H2></TD><TD style="width:630px; vertical-align:top"><H2><SPAN class="mw-headline" id="Monday_2">Monday</SPAN></H2><H3><SPAN class="mw-headline" id="42._Experiment:_Transforming_TOP10_with_pBAD_promoter_.28Hannah.29">42. Experiment: Transforming TOP10 with pBAD promoter (Hannah)</SPAN></H3><P>Took plasmid from Shuna, followed protocol from earlier, growing up stock to make more plasmid so we can PCR from it later. 
</P><P>Plated out on Kanomycin plates
</P><H3><SPAN class="mw-headline" id="43._Experiment:_Plasmid_extraction_-_pHK555_.2B_LuxR_from_registry_.28Hannah.29">43. Experiment: Plasmid extraction - pHK555 + LuxR from registry (Hannah)</SPAN></H3><P>Followed Qiagen miniprep protocol
</P></TD></TR><TR><TH> Plasmid
</TH><TH> Concentration (ng/μl)
</TH></TR><TR><TD> pHK555
</TD><TD> 20.9
</TD></TR><TR><TD> 17.3
</TD></TR><TR><TD> LuxR
</TD><TD> 43.9
</TD></TR><TR><TD> 89.2
</TD></TR></TBODY></TABLE><P>These are in storage in the -20°C
</P><H3><SPAN class="mw-headline" id="44._Experiment:_Growth_of_Slock10.2C_TOP10_pHK555.2BpHK724_and_phosphoreum_to_make_glycerol_stocks_for_tomorrow_.28Hannah.29">44. Experiment: Growth of Slock10, TOP10 pHK555+pHK724 and phosphoreum to make glycerol stocks for tomorrow (Hannah)</SPAN></H3><UL><LI>Used antibiotic resistance markers to select for bacteria in Slock10 + TOP10
</LI><LI>These were put in the 37°C incubator
</LI></UL><H3><SPAN class="mw-headline" id="45._Experiment:_Growing_up_of_10ml_overnight_cultures_of_old_glowth_.28from_non-glowing_flask.29_and_BBa_216007_.28Peter.29">45. Experiment: Growing up of 10ml overnight cultures of old glowth (from non-glowing flask) and BBa_216007 (Peter)</SPAN></H3><P>Made 4 vials:
</P><UL><LI>Amp + Chl      glowth?
</LI><LI>Amp            glowth?
</LI><LI>Amp            glowth? + lumazine (BBa_216007), double inoculation
</LI><LI>Amp            lumazine
</LI></UL><P>Placed in incubator at 30°C
</P><H2><SPAN class="mw-headline" id="Tuesday_2">Tuesday</SPAN></H2><H3><SPAN class="mw-headline" id="46._Experiment:_Transformation_of_TOP10cc_with_BBa_R0040_.28TetR_repressed_promoter.29.2C_BBa_E0030_.28enhanced_yellow_fluorescent_protein.29_separately._.28Theo_.26_Anja.29">46. Experiment: Transformation of TOP10cc with BBa_R0040 (TetR repressed promoter), BBa_E0030 (enhanced yellow fluorescent protein) separately. (Theo &amp; Anja)</SPAN></H3><P>BBa_R0040 (Plate 1, Well 6L) and BBa_E0030 (Plate 2, Well 24E) were resuspended in 10μl deionised H20. 
</P><P>TOP10cc were taken out of -80°C freezer and thawed on ice. 1ml pipette tip cut with scissors (sterile). 2x50μl of TOP10cc were transformed to 1.5ml eppendorf tubes. 
</P><P>2μl of resuspended BBa_R0040 and 2μl of resuspended BBa_E0030 were added to separate eppendorf tubes with TOP10cc. Cells were held on ice for 30mins. Heat shocked for 60s at 42°C (waterbath). Put on ice for ~2mins. Added 250μl pre-warmed (37°C) soc. Incubated at 37°C for 1h with rotation. Plated 50μl on pre-warmed (in 37°C incubator) LB agar plates with:
</P><UL><LI>Amp for BBa_R0040
</LI><LI>Kan for BBa_E0030
</LI></UL><P>Grow colonies overnight at 37°C. 
</P><H3><SPAN class="mw-headline" id="46._Experiment:_Streaking_out_and_setting_up_overnight_cultures_of_TOP10_with_promoter_.2B_rbs_.2B_luc_.28Anja.29">46. Experiment: Streaking out and setting up overnight cultures of TOP10 with promoter + rbs + luc (Anja)</SPAN></H3><P>Streaked out from a single colony on an LB agar plate with Amp and incubate at 37°C overnight. 
</P><P>Inoculated single bacterial colony in 5ml LB with 5μl (100mg/ml) Ampicillin, incubated at 37°C with shaking overnight. 
</P><H3><SPAN class="mw-headline" id="47._Experiment:_Using_plate_reader_to_characterise_light_output_of_the_pHK724.2BpHK555_in_Slock10_and_TOP10_as_they_grew._.28Paul.29">47. Experiment: Using plate reader to characterise light output of the pHK724+pHK555 in Slock10 and TOP10 as they grew. (Paul)</SPAN></H3><P>Grew the following (1/2 Slock10, 1/2 TOP10):
</P><UL><LI>10 x Cells + Medium LB + Amp
</LI><LI>10 x Cells + Medium LB + Chl
</LI><LI>10 x Cells + Medium LB + Amp + Chl
</LI></UL><P>Final concentrations of Amp = 100μg/ml, Chl = 20μg/ml
</P><P>Growth - at 30°C as we hope this will maximise light output. 
</P><P>Volume per well - 100μl of total (= LB + Chl/Amp + Cells)
</P><P>Cells used - grown up in broth on previous day by Paul in LB + antibiotic broth. 
</P><H2><SPAN class="mw-headline" id="Wednesday_2">Wednesday</SPAN></H2><H3><SPAN class="mw-headline" id="48._Experiment:_Making_long-term_stock_of_TOP10_with_promoter_.2B_rbs_.2B_p.p.luc_.28Peter.29">48. Experiment: Making long-term stock of TOP10 with promoter + rbs + p.p.luc (Peter)</SPAN></H3><P>Added 0.5ml 40% glycerol solution and 0.5ml sample culture to cryogenic vial (x2). Gently vortexed. 
</P><P>Stored in -80°C freezer in 'Black-GM230hns-205' storage box. Vials labelled with sticker 'TetR luc'. 
</P><H3><SPAN class="mw-headline" id="49._Experiment:_Transfer_of_promoter_.2B_rbs_.2B_p.p.luc_into_psB1C3_for_submission_to_registry._.28Will.2C_Peter_.26_Anja.29">49. Experiment: Transfer of promoter + rbs + p.p.luc into psB1C3 for submission to registry. (Will, Peter &amp; Anja)</SPAN></H3><P>1.5ml of TOP10 cells with promoter+rbs+luc grown overnight in LB+Amp were transferred to eppendorf tubes (x2) and cells were spun down in microcentrifuge (1min, 13000rpm). Suernatant was discarded and plasmid was purified following the 'QIAprep Spin Miniprep Kit using a microcentrifuge' protocol. 
</P><P>Nanodrops showed 135.1ng/μl DNA present in purified plasmid prep (EB as blank). 
</P><H4><SPAN class="mw-headline" id="Restriction_enzyme_digest">Restriction enzyme digest</SPAN></H4><P>Prepared at room temperature in the listed order:
</P><TABLE class="wikitable"><TBODY><TR><TD> nuclease-free H20
</TD><TD> 14μl
</TD></TR><TR><TD> 10x Fast Digest Buffer
</TD><TD> 2μl
</TD></TR><TR><TD> Plasmid DNA
</TD><TD> 2μl
</TD></TR><TR><TD> FD Enzyme EcoRI
</TD><TD> 1μl
</TD></TR><TR><TD> FD Enzyme PstI
</TD><TD> 1μl
</TD></TR><TR><TD> 20μl
</TD></TR></TBODY></TABLE><P>It was checked (Nanodrop!) that 2μl of plasmid DNA would not contain more than 1μg of DNA. The reaction was mixed gently, spun down and incubated at 37°C (PCR machine) for 30mins. Enzymes were inactivated by heating at 80°C for 10mins (in PCR Machine). 
</P><H4><SPAN class="mw-headline" id="DNA_Purification">DNA Purification</SPAN></H4><P>was carried out following the 'QIAquick PCR Purification Kit Protocol (using a microcentrifuge)' The DNA was stored overnight at -20°C. 
</P><H2><SPAN class="mw-headline" id="Thursday_2">Thursday</SPAN></H2><H3><SPAN class="mw-headline" id="50._Experiment:_Continued_transfer_of_promoter_.2B_rbs_.2B_p.p.luc_into_psB1C3_for_submission_to_registry._.28Will_and_Anja.29">50. Experiment: Continued transfer of promoter + rbs + p.p.luc into psB1C3 for submission to registry. (Will and Anja)</SPAN></H3><H4><SPAN class="mw-headline" id="Gel_Electrophoresis">Gel Electrophoresis</SPAN></H4><P>Nanodrop reading was taken for the restriction enzyme digest. 
</P><P>Plasmid and promoter + rbs + luc:   1ng/μl
</P><P>1% Agarose gels were poured following the below protocol:
</P><H4><SPAN class="mw-headline" id="Preparation_1.25_Agarose_Gels_.282_gels.29">Preparation 1% Agarose Gels (2 gels)</SPAN></H4><P>1g (agarose)/100ml (TAE - better resolution &gt; 4kb / TBE - better resolution 0.1-3kb) --&gt; 0.5 - 10kb optimal DNA resolution
</P><UL><LI>Measure agarose into beaker with buffer (50ml for 7x10cm gel box)
</LI><LI>Microwave until agarose fully melted
</LI><LI>Let agarose cool on bench
</LI><LI>Seal open edges of gel box with tape
</LI><LI>Add 10μl Sybr Safe Dye to agarose sultion, swirl
</LI><LI>Pour agarose solution in taped gel box, remove bubbles, put in comb, letit cool &amp; solidify
</LI></UL><P>Due to the low DNA concentration, it was decided to repeat the restriction enzyme digest (x3) following the same protocol as on p.31 without the enzyme inativation step. 
</P><P>Nanodrop measurements were taken for the restriction enzyme digests:
</P><P>
Plasmid and promoter + rbs + luc 1   --   34.9ng/μl
</P><P>Plasmid and promoter + rbs + luc 2   --   14.4ng/μl
</P><P>Plasmid and promoter + rbs + luc 3   --   35.8ng/μl
</P><P>
Gel loading mixture was mude up: (4x)
3μl    6x orange loading dye
7μl    plasmid DNA (restriction digest) --&gt; 245ng DNA
10μl   deionised H20
20μl
</P><P>Gel was loaded as follows:
</P><TABLE class="wikitable"><TBODY><TR><TD> Easyladder II
</TD><TD> Plasmid &amp; promoter + rbs + luc (4 wells)
</TD></TR><TR><TD> 10μl
</TD><TD> 20μl
</TD></TR></TBODY></TABLE><P>Gel was run for 40mins at 100V. No bands were observed. 
</P><P>Therefore, the restriction enzyme digest was repeated as on p.32. 
</P><P>Nanodrop readings: 
</P><P>
Plasmid and promoter + rbs + luc 1   --   14.6ng/μl
</P><P>Plasmid and promoter + rbs + luc 2   --   43.8ng/μl
</P><P>Plasmid and promoter + rbs + luc 3   --   18.1ng/μl
</P><P>
4 gel loading mixtures were taken up. 
</P><TABLE class="wikitable"><TBODY><TR><TD> Digest 1
</TD><TD> Digest 2a
</TD><TD> Digest 2b
</TD><TD> Digest 3
</TD></TR><TR><TD> 6x orange LD (μl)
</TD><TD> 3
</TD><TD> 3
</TD><TD> 3
</TD><TD> 3
</TD></TR><TR><TD> Plasmid DNA (μl)
</TD><TD> 4
</TD><TD> 10
</TD><TD> 5
</TD><TD> 5
</TD></TR><TR><TD> Deionised H20 (μl)
</TD><TD> 13
</TD><TD> 7
</TD><TD> 12
</TD><TD> 12
</TD></TR><TR><TD> 58.4ng DNA
</TD><TD> 438ng DNA
</TD><TD> 219ng DNA
</TD><TD> 90ng DNA
</TD></TR></TBODY></TABLE><P>Gel was loaded as follows:
</P><TABLE class="wikitable"><TBODY><TR><TD> Easyladder II
</TD><TD> Digest 1
</TD><TD> Digest 2a
</TD><TD> Digest 2b
</TD><TD> Digest 3
</TD></TR><TR><TD> 10μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD><TD> 20μl
</TD></TR></TBODY></TABLE><P>Empty wells were loaded with deionised H20. The gel was run and two bands were observed. One just around 2kb --&gt; plasmid backbone (2079b) and one a little lighter --&gt; promoter+rbs+luc (1727b). The lower band was cut out of the gel and DNA was extracted following the QIAquick Gel Extraction Kit Protocol. 
</P><H4><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H4><P>Nanodrop measurement of insert DNA
</P><P>Promoter + rbs + luc:    4.5ng/μl
pSB1C3 is supplied at:   25ng/μl  Chl. resistance
</P><P>10-100ng of linearised vector DNA should be added to ligation mix; 3μl gives 75ng. 3:1 molar excess of insert DNA should be added. 
</P><P>75ng/2035b x 1727b x 3 = 190ng --&gt; 42μl (189ng)
</P><P>Added to a microcentrifuge tube:
</P><TABLE class="wikitable"><TBODY><TR><TD> Promoter + rbs + luc
</TD><TD> 42μl
</TD></TR><TR><TD> pSB1C3
</TD><TD> 3μl
</TD></TR><TR><TD> 5x Rapid Ligation Buffer
</TD><TD> 12μl
</TD></TR><TR><TD> T4 DNA ligase
</TD><TD> 3μl
</TD></TR><TR><TD> 60μl
</TD></TR></TBODY></TABLE><P>Mix was vortexed, spun down for 15s at 13000rpm and incubated at 22°C (Room temperature) for 30 mins. 
</P><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>Followed protocol on p.13+14, but transformed 2 x 50μl of TOP10cc. Also plated on LB agar plates with Chl. 
</P><P>Results: No colonies after 1 night. After 4 nights on one of the 4 plates there was 1 potential colony.
</P><H2><SPAN class="mw-headline" id="Friday_2">Friday</SPAN></H2><H2><SPAN class="mw-headline" id="Saturday_2">Saturday</SPAN></H2><H2><SPAN class="mw-headline" id="Sunday_2">Sunday</SPAN></H2></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Cambridge/Notebook/Week6">http://2010.igem.org/Team:Cambridge/Notebook/Week6</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Notebook/Week6" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Notebook/Week6" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week6&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week6&amp;oldid=52942" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>