"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Bioluminescence/Bacterial Codon optimisation
From 2010.igem.org
Introduction
Home
Meet the Team
Photo Gallery
Videos
Parts submitted
Project Firefly
Introduction
Background
Coloured outputs
Luciferin recovery
Modelling
Characterisation
Project Vibrio
Introduction
Background
The LuxBrick
Codon optimisation
Modelling
Characterisation
Tools
Gibthon
BioBrick → GenBank
Ligation Calculator
The E.glometer
Gibson Assembly
Introduction
How it works
Protocol
RFC
Considerations
Human Practices
Safety
Future applications
Biosensors
Lighting
Notebook
Summary
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
Beyond Week 12
Protocols
Our Partners
Sponsors
iGEM Teams
University
Publicity
Project Vibrio: Bacterial Codon Optimisation
What is codon usage?
One of the fascinating features of life is the universal genetic code.  In all known organisms, from bacteria to man, the same triplets of DNA bases code for the same amino acids.  However this does not mean that all species encode their genomes in exactly the same way.   The code is redundant: a number of triplets code for the same amino acid.  While all species are able to translate any sequence of DNA interchangeably, E. coli prefers to use certain triplets to code for certain amino acids which may be different to the ones we use.  This 'preference' is reflected in the levels of tRNA which match such a triplet.  In this project we resynthesised a number of genes de novo and thus were able to codon optimise them for expression in E. coli.
Improved translational speed
Starting with the DNA sequence of the Vibrio fischeri lux operon found on the NCBI database, we used a number of tools to replace the codons used with the most common codons found in the E.coli genome. To achieve optimal expression of the Lux operon in E.coli, we had the operon re-synthesized after optimising the usage of codons. This conserves the sequence of amino acids in the gene products, but improves the rate of translation, as more common tRNAs are recruited. Codon usage optimization can yield dramatic increases in the expression of foreign genes, especially if they are introduced from less closely related species Gustafsson et al. 2004
Altered G-C content
DNA curvature is increased by sequences rich in A-T or G-C pairs. The natural V.fischeri Lux operon, and especially its intergenic regions, contains stretches rich in A-T, resulting in the curvature that H-NS proteins bind to preferentially. Changing the coding DNA sequence also meant changing the curvature of the DNA, which affects the binding affinity of H-NS proteins. To alleviate the repression that H-NS exerts, we took care to raise the G-C content of intergenic regions and coding sequences (at times resorting to suboptimal codons). According to a computational prediction, this resulted in greatly reduced DNA curvature, and thus hopefully to a reduced affinity for H-NS proteins.
Change in G-C(red)/A-T(blue) content by codon optimization. The occurence of long A-T rich stretches is reduced in the re-synthesised version of the Lux operon.
Computational prediction of DNA curvature before and after codon optimization. (click to enlarge)
Differential Expression
In Vibrio fischeri, LuxA and B are expressed at five times the levels of LuxC, D, E and G. Since all these genes are transcribed on the same mRNA, but have their own Ribosome Binding Sites, this is probably due to differences in codon usage ( Meighen et al. 1988 ). A number of rare codons are found in Lux C, D, E and G, but not in LuxA and B. Since we did not receive the newly synthesised LuxA and B in time, we constructed a new operon using our LuxC, D, E and G, and the LuxAB BioBrick that was put into the registry by the Edinburgh iGEM team 2009 (BBa_216008). These genes originate from Xenorhabdus luminescens. Compared to Vibrio fischeri, there is only limited amino acid identity in the Lux A and B genes (52% and 66% respectively). Yet the literature describes both as using the same substrates. Schmidt et al. 1989 describe positive complementation test using components from V. fisheri and Xenorhabdus luminescens. To use foreign genes from two very different donor species in one pathway in E.coli is an exciting test of our understanding of the processes involved in bacterial bioluminescence and of the power of synthetic biology in general. In this construct, LuxC, D, E and G are codon optimised, but LuxA and B are not. In order to adjust the ratio of gene expression between these genes to the state found in nature, we chose to put LuxA and B under a very strong, phage derived promoter (plambda) to be constitutively expressed. The other genes can now be put under any promoter to create a PoPS-to-light device. In conjunction with an inducible or repressible promoter, this could be used as a reporter device. To test the system, we placed the LuxCDEG under an arabinose induced pbad promoter (BBa_i0500).
Parts submitted to the registry
Ideally we would have submitted a codon-optimised version of the entire Lux operon. Unfortunately Mr Gene, the company we employed for synthesis, still had not completed the order two months after placing, and at the point of the wiki-freeze we still have not received the optimised versions of LuxA and B. In order to complete our aim of creating a PoPs->light device that can be placed under any promoter, we combined the optimised Vibrio LuxCDEG genes with the Edinburgh 2009 Xenorhabdus LuxAB. The assembly was achieved using the Gibson method . Since we only received LuxCD and LuxEG from Mr Gene two weeks before the documentation deadline, we could not properly characterise these parts. For a complete list of the BioBricks we submitted see the BioBricks section .
Codon optimised bacterial luminescence parts:
luxCD (BBa_K325902) derived from V. fischeri, but optimised for E.coli
luxEG (BBa_K325903) derived from V. fischeri, but optimised for E.coli
CDEG pLambda AB (BBa_K325905) a PoPs->light device that can be placed under any promoter, hybrid device containing promoterless V. fisheri derived CDEG (optimised) and Xenorhabdus AB under a strong constitutive promoter
pbad CDEG pLambda AB (BBa_K325906) the above part under the arabinose induced pbad promoter
Retrieved from " http://2010.igem.org/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
