"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/LabBook/Week8
From 2010.igem.org
Week 8: Monday 30th August - Sunday 5th September
Contents
1 Monday
1.1 69. Expt: Growing up of cell cultures of above experimental results & of results on p49 (Will)
1.2 70. Expt: Miniprep pSB1C3 from colonies (Anja)
1.3 71. Expt: Gibson assembly of pBAD, luxCD,AB,EG and pSB1C3 (Anja)
1.3.1 Gel electrophoresis
1.3.2 Gel Extraction
2 Tuesday
2.1 72. Expt: Continued Gibson assembly of pBAD, luxCD,AB,EG and pSB1C3 (Anja)
2.1.1 Transformation
2.2 73. Expt: Prepare DNA sequencing material for BioScience (Theo and Anja)
3 Wednesday
3.1 74. Expt: Prepare DNA sequencing material for BioScience (Anja)
3.2 75. Expt: Repeat of PCR of pBAD+luxCDABEG using modified reaction conditions (Will)
3.3 76. Night of 1 September - Plate Reader
4 Thursday
4.1 77. Expt: PCR of pBAD, luxCD,AB,EG and pSB1C3 (Will and Anja)
4.1.1 Gel Electrophoresis
4.2 78. Expt: Firefly Luciferase/Luciferin overnight plate reader experiments (+CHBT test) (Will)
4.2.1 Results
4.3 79. Expt: Transforming reporters (Theo)
4.4 80. Expt: Gel extraction of pBAD, luxCD,AB,EG and pSB1C3 (continued from p65) (Will)
4.5 81. Testing colonies from transformation on 31.08.10 for presence of Gibson assembled plasmid (Anja)
4.5.1 Colony PCR
4.5.2 Gel Electrophoresis
5 Friday
5.1 82. Expt: Gibson assembly and transformation of pBAD, luxCD,AB,EG in pSB1C3 (continued from gel extraction on p.67) (Anja)
5.1.1 Gibson assembly reaction
5.1.2 Transformation
5.2 83. Expt: Digestions of rbs Luc, pBAD Slock Lux (G28), rbs+EYFP, rbs+ECFP (Theo)
6 Saturday
6.1 84. Expt: Recovering P. phosphoreum from old cultures for making glycerol stocks (Peter)
7 Sunday
7.1 85. Expt: Glycerol stocks of TOP10 with tetR repressed rbs+luc+pSB1C3 (Peter)
Monday
Result (from Expt. 68):
No growth overnight, 2 growths out of 3 plates (+1 fungus) after weekend of growing. No glow.
69. Expt: Growing up of cell cultures of above experimental results & of results on p49 (Will)
Strains G1, j1 and G28wh were grown up in 5ml LB + 2µl Chloramphenicol, and on plates containing LB+Cm.
Left to grow at 30°C for 24h.
Results: Strains grew but did not glow after 48h.
70. Expt: Miniprep pSB1C3 from colonies (Anja)
Suspended a 'streak' of bacteria transformed with BBa_J04450 (registry part in pSB1C3) in 250µl buffer P1. Followed 'Bench protocol: Qiagen Spin Miniprep Kit Using a Microcentrifuge'.
Nanodrop 12.8ng/µl
71. Expt: Gibson assembly of pBAD, luxCD,AB,EG and pSB1C3 (Anja)
PCR:
Template
Primer f.
Primer r.
Amplified Fragment
I0500
prefix.f.pBadstart
luxCstart.r.pBADend
pBAD (A)
pJS555
pBADend.f.luxCstart
luxAstart.r.luxDend
luxCD (B)
pJS555
luxDend.f.luxAstart
luxEstart.r.luxBend
luxAB (C)
pJS555
luxBend.f.luxEstart
suffix.rev.luxGend
luzEG (D)
BBa_J04450
luxGend.for.suffix
pBADstart.r.prefix
pSB1C3 (E)
BBa_J04450: once purified (E), once colony PCR (EC)
Followed protocol on p38 for PCR mixtures and programs.
Gel electrophoresis
Self-cast 1% agarose gel loaded as follows:
Easyladder II
A
A
B
B
C
C
D
D
E
E
EC
EC
10µl
17µl
17µl
17µl
17µl
17µl
17µl
17µl
17µl
17µl
17µl
17µl
17µl
Run at 120V.
No bands were visible. Not even in the marker lane. Since a gel was that had been sitting on the bench for a few days and SYBR Safe Dye is light sensitive it is assumed that the dye had become non-functional.
A 1% Agarose gel was loaded as follows:
Easyladder II
A
A
B
C
D
E?
EC?
10µl
12µl
12µl
12µl
12µl
12µl
12µl
12µl
PCR Rxn
3µl
3µl
3µl
3µl
3µl
3µl
3µl
6x LD
5µl
5µl
5µl
5µl
5µl
5µl
5µl
Nuclease-free H20
These were all mixed and spun down prior to gel loading.
Bands of correct sizes were observed in all cases.
Gel Extraction
DNA was extracted from the gel following the "Bench protocol: MinElute Gel Extraction Microcentrifuge protocol".
Tuesday
72. Expt: Continued Gibson assembly of pBAD, luxCD,AB,EG and pSB1C3 (Anja)
Fresh Gibson 1.33x Master Mix was prepared following the protocol on p40.
A Gibson Assembly reaction was prepared according to the protocol on p40, but taking twice the volume of the Master Mix (ie 30µl) and each fragment (ie 2µl). The reaction was incubated for 1h at 50°C.
Transformation
TOP10 cells, red strain and black strain were transformed with 10µl of Gibson reaction following protocol on p13+14. Plated 150µl on LB agar plates with Chl. (Since we were short of LB+Agar+Chl plates, two old LB agar + 5µg/ml Chl plates from PJ were used). Incubated at 30°C overnight.
73. Expt: Prepare DNA sequencing material for BioScience (Theo and Anja)
Extracted plasmids from TOP10 cells were transformed with pBAD, luxCD,AB,EG in pSB1C3 following the 'Qiaprep Spin Miniprep Kit Using a Microcentrifuge' protocol.
Experiment was cancelled (to be repeated on 01/09/10)
Wednesday
74. Expt: Prepare DNA sequencing material for BioScience (Anja)
Extracted plasmids from TOP10 cells were transformed with pBAD, luxCD,AB,EG in pSB1C3 following the 'Qiaprep Spin Miniprep Kit Using a Microcentrifuge' protocol.
Nanodrop measurements: 88.7ng/µl
Primers are at 100µM = 100pmol/µl  --> 30x dilution --> 3.2pmol/µl
BioScience requests DNA concentrations of 100ng/µl for plasmids (Volume: 6µl) and 3.2pmol/µl (Volume: 10µl) for primers.
Tube 1: 6µl of TetR repr. prom, rbs, P.P.luc in pSB1C3 (purified) at 88.7ng/µl
Tube 2: 10µl Forward primer of 3.2pmol/µl (made up from 0.33µl 100pmol/µl primer and 9.67µl nuclease-free H20)
Tube 3: 10µl Reverse primer of 3.2pmol/µl (made up from 0.33µl 100pmol/µl primer and 9.67µl nuclease-free H20)
75. Expt: Repeat of PCR of pBAD+luxCDABEG using modified reaction conditions (Will)
Added on ice:
2x Phusion Master Mix
25µl
Forward primers
0.25µl
Reverse primers
0.25µl
Template DNA
2µl
Distilled (nuclease-free) H20
22.5µl
notation as on p38
Fragment
Template
Reaction
110°C heated lid, 1m30s denaturation
Promoter
I0500 purified (James Brown)
35 cycles
CD
pJS555
30s 98°C, 30s 53°C, 1m45s 72°C
AB
pJS555
7m30 elongation
EG
pJS555
pSB1C3
Plasmid extract from BBa_J04450 strain
Same as above only 71°C
In addition EG was run in touchdown PCR.
Heated lid
112°C
Denaturation
98°C
1m30
Touchdown
70°C to 45°C
45 cycles
Elongation
72°C
5min
PCR products were loaded into 1g/100ml agarose gel with 20µl/100ml SYBR SAFE strain.
Per well 25µl product + 5µl Gel loading dye
Wells: Hyperladder I, A, B, C, D, E, Dtouchdown, Hyperladder I
76. Night of 1 September - Plate Reader
In two parts:
(A)
Arabinose induction of G28 (lux operon under pBAD)
Rows A and B: all LB+Chl (100µl)
Column
Inoculated with
Arabinose Conc
x val
1
G28
0
x1, 11
2
G28
1µM
x2, 12
3
G28
10µM
x3, 13
4
G28
100µM
x4, 14
5
G28
1mM
x5, 15
6
G28
100mM
x6, 16
7
pSB1C3
0
x7, 17
8
Nothing - LB only
0
x8, 18
9
Slock10
0
x9, 19
10
Empty well
x10, 20
Incubated at 30°C
Plate reader protocol:
Lum reads: gain 3200, time = 10s
OD reads at 100µl (595nm)
10 mins between readings
Layout:
1
8
12
D
x21
x28
x32
E
x33
x40
Thursday
77. Expt: PCR of pBAD, luxCD,AB,EG and pSB1C3 (Will and Anja)
PCR mixtures were made up according to p38.
Notation as follows:
Amplified fragment
Template used
A
pBAD
I0500
B
luxCD
pJS555
C
luxAB
pJS555
D
luxEG
pJS555
Dp
luxEG
previously gel extracted D
E
pSB1C3
BBa_J04450
Ec
pSB1C3
colony with BBa_J04450
Ep
pSB1C3
previously gel ext. E
A,B and C were run at the following conditions:
110°C heated lid
Initial denaturation
98°C
1.30min
Denaturation
98°C
30s
Start cycle (35x)
Annealing
53°C
30s
Elongation
72°C
1.45min
End cycle
Final elongation
72°C
7.30min
Store at 10°C
D was run at:
110°C heated lid
Initial denaturation
98°C
1.30min
Denaturation
98°C
30s
Start cycle (35x)
Annealing
59°C
30s
Elongation
72°C
1.45min
End cycle
Final elongation
72°C
7.30min
Store at 10°C
Dp, E, Ec & Ep were run at:
110°C heated lid
Initial denaturation
98°C
1.30min
Denaturation
98°C
30s
Start cycle (35x)
Annealing
64°C
30s
Elongation
72°C
1.45min
End cycle
Final elongation
72°C
7.30min
Store at 10°C
Gel Electrophoresis
In self-cast 1% agarose gels (with 30µl SYBR safe dye per gel) ran: 25µl PCR reaction + 5µl 6x orange LD (mixed prior to loading).
Ladder: Hyperladder I
A,B,C,E,Dp were cut out for gel extraction. To improve the yield of fragment E, bands from both gels were cut out and pooled.
78. Expt: Firefly Luciferase/Luciferin overnight plate reader experiments (+CHBT test) (Will)
Plate reader: OD, 100ml, luminescence gain 3200, read length 10s, reads every 10 minutes, overnight (from 1st to 2nd Sept)
Plate layout (X21 to 40):
1
2
3
4
5
6
7
8
9
10
11
12
D
X21
22
23
24
25
26
27
28
29
30
31
32
200µM
200µM
100µM
100µM
40µM
40µM
200µM
200µM
100µM
100µM
40µM
40µM
E
33
34
35
36
37
38
39
40
200µM
200µM
100µM
100µM
40µM
40µM
*
*
'*' = 50µl broth + 50µl PBS
All have 50µl cell in broth + 50µl substrate solution.
X22-26 CHBT substrate solution: 0.4mM (400µM) in 5% DMSO in PBS
X27-32 Luciferin substrate solution: 0.4mM in 5% DMSO in PBS
For 200µl:
12.1µl luciferin stock
10µl DMSO
177.9µl PBS 1x
X33-38 Luciferin substrate solution: 0.4mM in PBS
Luciferin, sodium salt, molar mass = 302g/mol
Stock at 2mg/µl = 0.002g/ml
0.002g/302g/mol = 6.62x10^-6 mol/ml or 6.62x10^-3 Molar solution (6.62mM)
We want 0.4mM so 6.62/0.4 = 16.5x dilution
Results
CHBT: no light
79. Expt: Transforming reporters (Theo)
Theo transformed TOP10cc with two translational units:
BBa_I15017 EYFP translational unit: B0032 RBS - plate 122M
BBa_I15016 ECFP translational unit - plate 122K
He plated out and made overnight cultures.
80. Expt: Gel extraction of pBAD, luxCD,AB,EG and pSB1C3 (continued from p65) (Will)
Bands were cut out of the gels and DNA was extracted following the 'Bench protocol: MinElute Gel Extraction Microcentrifuge Protocol'. The extracted DNA was stored at -20°C overnight.
81. Testing colonies from transformation on 31.08.10 for presence of Gibson assembled plasmid (Anja)
Colonies had grown on all plates from the transformation on 31.01.10, however, only after 3 days of incubating at 30°C.
Are these colonies just a result of degraded chloramphenicol or are they containing the desired plasmid?
A 10mM dNTP mix was prepared: 10µl 100mM dATP, dGTP, dTTP & dCTP + 60µl nuclease-free H20.
Colony PCR
Make up a master mix for 10 PCR reactions
10X NH4 Reaction Buffer
50µl
50mM MgCl2
10µl
dNTPs (10mM)
10µl
Forward Primer
10µl
Reverse Primer
10µl
BioTaw Polymerase
7.5µl
nuclease-free H20
400µl
~500µl
Aimed to amplify luxEG region, used according to primers (see p38).
Also prepared 10µM solutions of the two primers (using these rather than the 100µM stocks to reduce danger of contamination).
The Master Mix was aliquated in 50µls, to which bacteria from one colony each were added (dip pipette tip 1st into colony then aliquated PCR mix, suck up PCR mix in pipette tip, pipette up and down a few times).
PCR program (saved as BIOTAQ COL PCR) on G-storm:
110°C heated lid
Initial denaturation
96°C
1:30min
Denaturation
96°C
10s
Start cycle (25x)
Annealing
60°C
20s
Elongation
72°C
1:30min (45s/kb)
End cycle
Final elongation
72°C
5min
Store at 10°C
Gel Electrophoresis
1% Agarose E-gel was used. Loading mixtures contained 3µl 6x Orange LD, 7µl nuclease-free H20, 10µl PCR reaction mix
There were lots of primer dimers => BEN WAS RIGHT!
There is no band in any of the colonies of a size of ~1.8kb (corresponding to luxEG) => colonies do NOT contain desired plasmid :-(
Friday
82. Expt: Gibson assembly and transformation of pBAD, luxCD,AB,EG in pSB1C3 (continued from gel extraction on p.67) (Anja)
Gibson assembly reaction
30µl
Gibson 1.33x Master Mix
2µl
A
2µl
B
2µl
C
2µl
D
2µl
E
40µl
Incubated for 1h at 50°C
Transformation
Transformed 3x50µl TOP10 with 10µl Gibson reaction each. Following the protocol on p13+14. Plated 150µl on LB agar plates with Cm and incubated overnight at 37°C.
Results: On the 04.09.10 two of three transformations were observed to yield colonies (ie colonies on two of three plates), most of plates which (not all!) were pink. ==> Need to identify what plasmid they contain.
83. Expt: Digestions of rbs Luc, pBAD Slock Lux (G28), rbs+EYFP, rbs+ECFP (Theo)
Theo miniprepped these:
DNA (ng/µl)
G28
76
Luc
32
C
4.8
Y
12.7
Digestion:
G28 - SP (SpeI, PstI) - 13µl DNA
Luc - SP (SpeI, PstI) - 17µl DNA
C - XP (XbaI, PstI) - 17µl DNA
Y - XP (XbaI, PstI) - 17µl DNA
Fast digested for 30 mins
Gel run with Hyperladder I. Interpretation:
G28 is actually religated by pSB1C3 (but minus RFP)
Luc failed (but why?)
too little DNA from EYFP, ECFP => predictable (not from colonies)
Saturday
84. Expt: Recovering P. phosphoreum from old cultures for making glycerol stocks (Peter)
Grew up overnight culture from the old Falcon 4 in the morning the culture was turbid but not glowing on photos.
Streaked out two of the old phosphoreum lawns to achieve single, glowing colonies.
5/9/10 => no colonies visible
6/9/10 => tiny colonies visible, but not glowing on photos
Sunday
85. Expt: Glycerol stocks of TOP10 with tetR repressed rbs+luc+pSB1C3 (Peter)
Liquid cultures had grown in 37°C revolving incubator for two days. Broth appeared overgrown and sedimenting => resuspended sediment and streaked out onto Chl plates (37°C), placed liquid cultures into fridge.
6/9/10 put 9 single colonies in plate reader with some D-luciferin, after 6 hours ODs were up to ~1.5 but no glowing was detectable.
Retrieved from " http://2010.igem.org/Team:Cambridge/LabBook/Week8 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
