<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Notebook_Week9"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Notebook/Week9">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Notebook/Week9&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week9&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week9&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Notebook/Week9" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Notebook/Week9</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="wrap"><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:Cambridge">Home</A></LI><LI><A href="/Team:Cambridge/TheTeam">Meet the Team</A></LI><LI><A href="/Team:Cambridge/Photos">Photo Gallery</A></LI><LI><A href="/Team:Cambridge/Videos">Videos</A></LI><LI><A href="/Team:Cambridge/BioBricks">Parts submitted</A></LI></UL><LI><A class="head" href="#">Project Firefly</A></LI><LI><A href="/Team:Cambridge/Bioluminescence">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background_Firefly">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Colour">Coloured outputs</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Luciferin_Regeneration">Luciferin recovery</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Project Vibrio</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Luciferases">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/G28">The LuxBrick</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">Codon optimisation</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:Cambridge/Tools/Gibson">Gibthon</A></LI><LI><A href="/Team:Cambridge/Tools/GenBank">BioBrick → GenBank</A></LI><LI><A href="/Team:Cambridge/Tools/Ligate">Ligation Calculator</A></LI><LI><A href="/Team:Cambridge/Tools/Eglometer">The E.glometer</A></LI><LI><A class="head" href="#">Gibson Assembly</A></LI><LI><A href="/Team:Cambridge/Gibson/Introduction">Introduction</A></LI><LI><A href="/Team:Cambridge/Gibson/Mechanism">How it works</A></LI><LI><A href="/Team:Cambridge/Gibson/Protocol">Protocol</A></LI><LI><A href="/Team:Cambridge/Gibson/RFC">RFC</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:Cambridge/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Cambridge/Safety">Safety</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:Cambridge/Tools/microMeasure">Biosensors</A></LI><LI><A href="/Team:Cambridge/Tools/Lighting">Lighting</A></LI><LI><A class="head" href="/Team:Cambridge/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:Cambridge/Notebook/Summary">Summary</A></LI><LI><A href="/Team:Cambridge/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:Cambridge/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:Cambridge/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:Cambridge/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:Cambridge/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:Cambridge/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:Cambridge/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:Cambridge/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:Cambridge/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:Cambridge/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:Cambridge/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:Cambridge/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:Cambridge/Notebook/FurtherWork">Beyond Week 12</A></LI><LI><A href="/Team:Cambridge/Protocols">Protocols</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">Our Partners</A></LI><LI><A href="/Team:Cambridge/Partners">Sponsors</A></LI><LI><A href="/Team:Cambridge/iGEM_Teams">iGEM Teams</A></LI><LI><A href="/Team:Cambridge/Departments">University</A></LI><LI><A href="/Team:Cambridge/Publicity">Publicity</A></LI></DIV><DIV id="right"><DIV style="width:730px; background-color:#fad72a;" class="secheader">Notebook: Week 9</DIV><DIV style="position:relative; height:30px;border-top:5px solid white;"><DIV style="width:492px;position:absolute; background-color:#96d446;" class="selecteddiv" id="projectlink"><P>Viewing project diary</P></DIV><DIV style="width:246px; position:absolute; background-color:#fb5c2b;left:497px;" class="selectordiv" id="lablink"><P>View protocols and data</P></DIV></DIV><DIV style="overflow:hidden; height:2200px; width:730px; position:relative;" id="maskingbox"><DIV style="width:1320px; position:relative;" id="superwide"><TABLE style="width:1480px" id="widetable"><TBODY><TR><TD style="padding-right:50px; padding-left:10px; width:630px;vertical-align:top"><DIV align="center">Week 9: Monday 6th - Sunday 12th September</DIV><H2><SPAN class="mw-headline" id="Monday"> Monday </SPAN></H2><P>This morning was very exciting because both our orders from DNA2.0 had arrived.  These were the firefly operons, both containing both a luciferase and a luciferin regenerating enzyme.  However we became incredibly paranoid, since the two tiny discs of filter paper were worth a great deal!
</P><P>DNA 2.0 had supplied a protocol for extraction but we wanted to OK it with our advisors first and there was no-one in the lab, so we regretfully decided to leave extraction and transformation for Tuesday.
</P><P>However we did see interesting results in that G28, an attempt Will had made to Gibson assemble the Vibrio fischeri lux operon into E. coli had glowed in the plate reader.  Therefore Theo made up plates containing arabinose in LB with chloramphenicol and plated some out.
</P><H2><SPAN class="mw-headline" id="Tuesday"> Tuesday </SPAN></H2><P>G28 had indeed been induced and glowed, see the pic on right - the first photograph of one of our finished BioBricks emitting light (since nothing else has been in pSB1C3).  We still need to sequence it to show it is definitely correct.
Today we also started a number of standard Biobrick assembly operations.  We wanted to move the registry luciferase under a TetR promoter into pSB1C3 and also to add a fluorescent reporter to this operon, as well as G28.
As such we performed a restriction digest, ran gels and performed gel extraction. We put this DNA in the freezer and turned our attention to our order from DNA 2.0.
This was extracted from filter paper adding Tris-HCl and centrifuging.  Then we transformed competent cells, we performed 4 transformations for each construct just in case, we plated them all out on kanamycin plates to select for the DNA 2.0 in house vector <A href="https://www.dna20.com/index.php?pageID=278" class="external text" rel="nofollow">pJ207</A>.
</P><H2><SPAN class="mw-headline" id="Wednesday"> Wednesday </SPAN></H2><P>To our immense relief the DNA 2.0 transformations had indeed grown up!  We prepared overnight cultures for BioBricking.
</P><P>Also Paul continued in his attempts to extract the exisiting E. coli thioesterase gene from genomic DNA for BioBricking.  Unfortunately the DNA was so faint that it was best visualised by putting a cardboard box over the transilluminator.
</P><P>Theo ligated the fluorescent proteins onto the two operons, continuing Tuedays experiment and plated these out.
</P><H2><SPAN class="mw-headline" id="Thursday"> Thursday </SPAN></H2><H2><SPAN class="mw-headline" id="Friday">Friday</SPAN></H2><H2><SPAN class="mw-headline" id="Saturday">Saturday</SPAN></H2><H2><SPAN class="mw-headline" id="Sunday">Sunday</SPAN></H2></TD><TD style="width:630px; vertical-align:top"><DIV align="center">Week 9: Monday 6th September - Sunday 12th September</DIV><H2><SPAN class="mw-headline" id="Monday_2">Monday</SPAN></H2><H3><SPAN class="mw-headline" id="86._Expt:_Extract_CDABEG_from_pJS555">86. Expt: Extract CDABEG from pJS555</SPAN></H3><UL><LI>Using 2 primers
</LI><LI>prefix start of O
</LI><LI>suffix end of G
</LI></UL><LI>Hope that it will glow in its own right without luxR+I
</LI><LI>Need to put under a new promoter
</LI><H3><SPAN class="mw-headline" id="87._Expt:_Gibson_transformation_.28cont._from_p.70.29">87. Expt: Gibson transformation (cont. from p.70)</SPAN></H3><P>3 colonies on one plate were not pink. These were streaked out on new Chl plated and put into liquid cultures along with 2 of the red colonies. 
</P><P>Next check for the right sized fragment with colony PCR and try to induce with arabinose. 
</P><H3><SPAN class="mw-headline" id="88._Expt:_Plate_Reader_G28">88. Expt: Plate Reader G28</SPAN></H3></TD></TR><TR><TD>Well
</TD><TD>Arabinose conc.
</TD></TR><TR><TD>A1
</TD><TD>0
</TD><TD>x1,9
</TD></TR><TR><TD>A2
</TD><TD>1µM
</TD></TR><TR><TD>A3
</TD><TD>5µM
</TD></TR><TR><TD>A4
</TD><TD>10µM
</TD></TR><TR><TD>A5
</TD><TD>100µM
</TD></TR><TR><TD>A6
</TD><TD>Blank
</TD></TR><TR><TD>A7
</TD><TD>pSB1C3
</TD></TR><TR><TD>A8
</TD><TD>Blank
</TD><TD>x8,16
</TD></TR></TBODY></TABLE><P>Reads every 10 mins
</P><H3><SPAN class="mw-headline" id="89._Expt:_Extracting_YFP.2C_CFP.2C_Luc_tetR.2C_G28_via_miniprep_.28Paul.29">89. Expt: Extracting YFP, CFP, Luc tetR, G28 via miniprep (Paul)</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD>Nanodrop readings (ng/µl)
</TD></TR><TR><TD>YFP
</TD><TD>33.3, 55.4
</TD></TR><TR><TD>CFP
</TD><TD>17.9, 20.4
</TD></TR><TR><TD>Luc tetR
</TD><TD>33.7
</TD></TR><TR><TD>G28
</TD><TD>125.3, 96.9
</TD></TR><TR><TD>G28?
</TD><TD>17.9
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="90._Expt:_Colony_PCR_to_extract_the_thioesterase_gene_from_E._coli_K12_.28Paul.29">90. Expt: Colony PCR to extract the thioesterase gene from E. coli K12 (Paul)</SPAN></H3><P>We used TOP10 cells, a substrain of DH10B, which is a substrain of K12. 
</P><P>We did a colony PCR to isolate the gene: 
</P><TABLE class="wikitable"><TBODY><TR><TD>3 replicates with:
</TD><TD>1 negative control:
</TD></TR><TR><TD>2x Phusion Mastermix
</TD><TD>10µl
</TD><TD>10µl
</TD></TR><TR><TD>Template DNA
</TD><TD>1µl
</TD><TD>0
</TD></TR><TR><TD>Primer 1
</TD><TD>1µl</TD><TD>1µl
</TD></TR><TR><TD>Primer 2
</TD><TD>1µl
</TD><TD>1µl
</TD></TR><TR><TD>Nuclease-free H20
</TD><TD>7µl
</TD><TD>8µl
</TD></TR></TBODY></TABLE><P>The negative control was to check for primer/dimer and contamination. 
</P><P>PCR protocol:
</P><UL><LI>Initial denaturation: 5min @ 98°C
</LI><LI>Touchdown: 16 cycles
</LI><LI>Denaturation: 10s @ 98°C
</LI><LI>Annealing: 20s @ 65°C to 57°C
</LI><LI>Elongation: 15s @ 72°C
</LI></UL><LI>30 cycles
</LI><LI>Denaturation: 10s @ 98°C
</LI><LI>Annealing: 20s @ 50°C
</LI><LI>Elongation: 15s @ 72°C
</LI><LI>End and hold at 10°C
</LI><UL><LI>We then ran result on a gel with SyberSafe at 6x lodaing Dye and Hyperladder IV
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>Lane: 1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD></TR><TR><TD>Ladder
</TD><TD>Tube 1
</TD><TD>Tube 2
</TD><TD>Tube 3
</TD><TD>Negative Control
</TD></TR></TBODY></TABLE><P>Failed: Something obtained in control...
</P><P>Nothing in other lanes
</P><P>We repeated without touchdown PCR:
</P><UL><LI>Denaturation 5min @ 98°C
</LI><LI>35 Cycles
</LI><LI>Denaturation: 12s @ 98°C
</LI><LI>Annealing: 20s @ 58.5°C
</LI><LI>Elongation: 15s @ 72°C
</LI></UL><LI>Hold at 10°C
</LI><P>Failed again: Product in negative control observed again...
</P><H2><SPAN class="mw-headline" id="Tuesday_2">Tuesday</SPAN></H2><H3><SPAN class="mw-headline" id="91._Expt:_Further_testing">91. Expt: Further testing</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD>Column
</TD><TD>Arabiniose/µl
</TD><TD>Well names
</TD></TR><TR><TD>Repeat 1
</TD><TD>Repeat 2
</TD><TD>Repeat 3
</TD></TR><TR><TD>1
</TD><TD>0
</TD><TD>X1
</TD><TD>X12
</TD><TD>X23
</TD></TR><TR><TD>2
</TD><TD>0
</TD><TD>X2
</TD><TD>X13
</TD><TD>X24
</TD></TR><TR><TD>3
</TD><TD>0
</TD><TD>X3
</TD><TD>X14
</TD><TD>X25
</TD></TR><TR><TD>4
</TD><TD>5
</TD><TD>X4
</TD><TD>X15
</TD><TD>X26
</TD></TR><TR><TD>5
</TD><TD>10
</TD><TD>X5
</TD><TD>X16
</TD><TD>X27
</TD></TR><TR><TD>6
</TD><TD>50
</TD><TD>X6
</TD><TD>X17
</TD><TD>X28
</TD></TR><TR><TD>7
</TD><TD>100
</TD><TD>X7
</TD><TD>X18
</TD><TD>X29
</TD></TR><TR><TD>8
</TD><TD>1000
</TD><TD>X8
</TD><TD>X19
</TD><TD>X30
</TD></TR><TR><TD>9
</TD><TD>10,000
</TD><TD>X9
</TD><TD>X20
</TD><TD>X31
</TD></TR><TR><TD>10
</TD><TD>Blank
</TD><TD>X10
</TD><TD>X21
</TD><TD>X32
</TD></TR><TR><TD>11
</TD><TD>Blank
</TD><TD>X11
</TD><TD>X22
</TD><TD>X33
</TD></TR></TBODY></TABLE><P>Inoculation: 1ml overnight added to 3ml, Amp + Cm
</P><H3><SPAN class="mw-headline" id="92._Expt:_Restriction_Enzyme_Digests">92. Expt: Restriction Enzyme Digests</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TD>DNA
</TD><TD>Enzymes used
</TD><TD>Subsequent Nanodrop readings/ng/µl
</TD></TR><TR><TD>1.
</TD><TD>Linear Plasmid
</TD><TD>EP
</TD><TD>5.6
</TD></TR><TR><TD>2.
</TD><TD>Luc TetR
</TD><TD>ES
</TD><TD>4.0
</TD></TR><TR><TD>3.
</TD><TD>Luc TetR
</TD><TD>EP
</TD><TD>2.8
</TD></TR><TR><TD>4.
</TD><TD>RBS YFP
</TD><TD>XP
</TD><TD>19.4
</TD></TR><TR><TD>5.
</TD><TD>RBS CFP
</TD><TD>XP
</TD><TD>2.9
</TD></TR><TR><TD>6.
</TD><TD>G28 lux
</TD><TD>SP
</TD><TD>6.1
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="93._Expt:_Extracting_DNA_2.0_from_Registry">93. Expt: Extracting DNA 2.0 from Registry</SPAN></H3><UL><LI>Remove filter from plastic bag and place on a sterile and clean surface.
</LI><LI>Add 100µl of 10mM Tris-HCl, pH 7.5 directly to the center of the filter
</LI><LI>Incubate at room temperature for 2minutes.
</LI><LI>Puncture the bottom of at 0.6ml tube using a syringe.
</LI><LI>Place filter in the 0.6ml tube and place 0.6ml tube in a 1.5ml tube.
</LI><LI>Place the 1.5ml tube(now containing punctured 0.6ml tube with filter) in tabletop centrifuge.
</LI><LI>Spin 1 minute at full speed. The DNA containing liquid will transfer from the filter in the 0.6ml tube into the 1.5ml tube.
</LI><LI>Discard the 0.6ml tube with filter. The 1.5ml tube now contains ~90µl buffer+DNA.
</LI><LI>Carefully remove supernatant. there may be a small pellet consisting of filter debris. This pellet does NOT contain any of the DNA. The supernatant should contain approximately 2µl plasmid DNA(~20ng/µl).
</LI></UL><P>The isolated DNA can subsequently be transformed, cut with restriction enzymes, or sequenced without further purification.
</P><P>Then transformed using standard protocol
</P><P>Result: Colonies all grew
</P><H2><SPAN class="mw-headline" id="Wednesday_2">Wednesday</SPAN></H2><H3><SPAN class="mw-headline" id="94._Expt:_Ligating_restriction_digests_from_yesterday">94. Expt: Ligating restriction digests from yesterday</SPAN></H3><P>Theo wished to prepare three constructs:
</P><UL><LI>A: P[TetR repressed] - WT luciferase - YFP in pSB1C3
</LI><LI>B: pBad Lux aoperon (G28) - YFP in pSB1C3
</LI><LI>C: P[TetR repressed] - WT luciferase
</LI></UL><P>Volumes all in µl
</P><TABLE class="wikitable"><TBODY><TR><TD>pSB1C3 linearised cut with EP.
</TD><TD>Luc cut with ES.
</TD><TD>Luc cut with EP.
</TD><TD>G28.
</TD><TD>YFP.
</TD><TD>Rapid ligation buffer.
</TD><TD>T4 ligase.
</TD></TR><TR><TD>A
</TD><TD>4
</TD><TD>9
</TD><TD>0
</TD><TD>0
</TD><TD>2
</TD><TD>4
</TD><TD>1
</TD></TR><TR><TD>B
</TD><TD>4
</TD><TD>0
</TD><TD>0
</TD><TD>9
</TD><TD>2
</TD><TD>4
</TD><TD>1
</TD></TR><TR><TD>C
</TD><TD>2
</TD><TD>0
</TD><TD>13
</TD><TD>0
</TD><TD>0
</TD><TD>4
</TD><TD>1
</TD></TR></TBODY></TABLE><P>Theo then incubated for 5mins at 22°C and following transformation protocol, plating out on Chlor plates.
</P><H3><SPAN class="mw-headline" id="95._Expt:_Colony_PCR_of_cells_transformed_with_the_Gibson_assembly_product_.28pages_70.2C73.29_.28Done_by_Peter.29">95. Expt: Colony PCR of cells transformed with the Gibson assembly product (pages 70,73) (Done by Peter)</SPAN></H3><P>Took 2 colonies of plates ABCDE each
</P><P>ran PCR on G-storm ('Phusion Lng rng clny PCR')
</P><P>27 cycles
</P><UL><LI>melting: 98µl 15s
</LI><LI>annealing: 60µl 10s
</LI><LI>extension: 72µl 3min
</LI></UL><P>final elongation 10min
</P><P>Ran on E-gel:
</P><P>faint band at ~2000bp
</P><P>no band visible at the expected 6.6kbp
</P><P>PCR tubes placed in Freezer, ~30µl left
</P><H3><SPAN class="mw-headline" id="96._Expt:_Colony_PCR_to_extract_the_thioesterase_gene_from_E._coli_K-12_.28repeat_using_new_protocol.29">96. Expt: Colony PCR to extract the thioesterase gene from E. coli K-12 (repeat using new protocol)</SPAN></H3><P>We used K-12 strains from Veio collection ( JW3582 and JW5313) but during the cell lysis protcols writing on tubes was erased so we couldn't identify the 2 strains.
</P><P>Cell lysis
</P><UL><LI>Heat at 98 for 10 min
</LI><LI>Freeze at -80 for 10min
</LI><LI>Vortex for 2min
</LI></UL><P>Cells were collected from plates and put in 20 of dH<SUB>2</SUB>0
</P><P>We performed 2 PCR (normal and touchdown) with 2 replicates for each strain and one negative control:
</P><TABLE class="wikitable"><TBODY><TR><TD>negative control
</TD></TR><TR><TD>7µl of dH<SUB>2</SUB>0
</TD><TD>8µl of dH<SUB>2</SUB>0
</TD></TR><TR><TD>10µl of 2x Phusion Mastermix
</TD><TD>10µl of 2x Phusion Mastermix
</TD></TR><TR><TD>1µl of primer 1 @0.5mM
</TD><TD>1µl of primer 1 @0.5mM
</TD></TR><TR><TD>1µl of primer 2 @0.5mM
</TD><TD>1µl of primer 2 @0.5mM
</TD></TR><TR><TD>1µl of Cell suspension
</TD></TR></TBODY></TABLE><P>PCR
</P><UL><LI>Denaturation: 5min @ 98°C
</LI><LI>35 cycles:
</LI><LI>12s @ 98°C : Denaturation
</LI><LI>20s @ 58°C : Annealing
</LI><LI>20s @ 72°C : Elongation
</LI></UL><LI>Elongation: 5min @ 72°C
</LI><LI>Hold at 10
</LI><P>Touchdown PCR
</P><UL><LI>10 cycles
</LI><LI>Denaturation: 12s @ 98°C
</LI><LI>Annealing: 20s @ 65 to 55°C
</LI><LI>Elongation: 20s @ 72°C
</LI></UL><UL><LI>35cycles
</LI><LI>Denaturation: 12s @ 98°C
</LI><LI>Annealing: 20s @ 55°C
</LI><LI>Elongation: 20s @ 72°C
</LI></UL><UL><LI>Elongation 5min @ 72°C
</LI></UL><P>We then ran a gel
</P><P>Products were obtained for strain 2 in both Touchdown lanes and in one normal PCR lane
</P><H2><SPAN class="mw-headline" id="Thursday_2">Thursday</SPAN></H2><H3><SPAN class="mw-headline" id="97._Expt:_Restriction_Digest_of_DNA2.0_stuff.2C_tetR_and_pSB1C3_with_RFP">97. Expt: Restriction Digest of DNA2.0 stuff, tetR and pSB1C3 with RFP</SPAN></H3><P>Miniprepped:
</P><UL><LI>DNA2.0 P. pyralis (x2)
</LI><LI>DNA2.0 L. cruciata (x2)
</LI><LI>TetR promotor
</LI><LI>pSB1C3 + RFP
</LI></UL><P>Nanodrop results: 
</P><P>On the side of the tubes in red pen
</P><P>Restriction enzymes:
</P><UL><LI>DNA2.0 P.P. --&gt; E + P
</LI><LI>DNA2.0 P.P. --&gt; X + P
</LI><LI>DNA2.0 L.C. --&gt; E + P
</LI><LI>DNA2.0 L.C. --&gt; X + P
</LI><LI>TetR --&gt; E + S
</LI><LI>pSB1C3 RFP --&gt; E + P
</LI></UL><P>Ran on gel, results in lab book. Look correct. 
</P><H3><SPAN class="mw-headline" id="98._Expt:_PCR_to_add_20bp_to_pSB1C3_to_be_able_to_Gibson_it_with_Thioesterase_from_p83">98. Expt: PCR to add 20bp to pSB1C3 to be able to Gibson it with Thioesterase from p83</SPAN></H3><P>PCR protocol similar to p83 but with:
</P><UL><LI>Denaturation 30s @ 98°C
</LI><LI>35 cycles:
</LI><LI>Denaturation 12s @ 98°C
</LI><LI>Annealing 20s @ 70°C
</LI><LI>Elongation 45s @ 72°C
</LI></UL><LI>Elongation 7:30min @ 72°C
</LI><LI>Hold at 10°C
</LI><P>It failed. 
</P><P>2nd attempt with annealing @ 69°C
</P><P>We obtained results which were then extracted from the gel by Ben. We also added pure plasmid to compare intensities. 
</P><UL><LI>Lane 1: 1.8ng/µl
</LI><LI>Lane 2: 2.3ng/µl
</LI><LI>Lane 3: 4.5ng/µl
</LI><LI>Lane 4: 24.5ng/µl
</LI></UL><H3><SPAN class="mw-headline" id="99._Expt:_Gibson_assembly_of_fragments_from_p87_and_p84_.28pSB1C3_and_Thioesterase.29">99. Expt: Gibson assembly of fragments from p87 and p84 (pSB1C3 and Thioesterase)</SPAN></H3><P>Readings for thioesterase extracted from expt p84 were as follows:
</P><UL><LI>Lane 4: 5.7ng/µl
</LI><LI>Lane 8: 12.7ng/µl
</LI><LI>Lane 9: 12.4ng/µl
</LI></UL><P>We performed more assembling:
</P><UL><LI>Plasmid 4 --&gt; Thio8
</LI><LI>Plasmid 4 --&gt; Thio9
</LI><LI>Plasmid 3 --&gt; Thio4
</LI><LI>Plasmid 4 --&gt; Thio4
</LI><LI>Plasmid 1 --&gt; Thio8
</LI><LI>Plasmid 1 --&gt; Thio9
</LI><LI>Plasmid 3 --&gt; Thio8
</LI><LI>Plasmid 3 --&gt; Thio9
</LI><LI>Plasmid 2 --&gt; Thio8
</LI></UL><P>We used the standard protocol (OpenWetWare). We then ran gel with Lane 10 pSB1C3 from the registry as control. 
</P><P>We extracted the DNA samples marked in red (as the resulting plasmid should be circular so gel speed could be variable).
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Cambridge/Notebook/Week9">http://2010.igem.org/Team:Cambridge/Notebook/Week9</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Notebook/Week9" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Notebook/Week9" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week9&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week9&amp;oldid=62559" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>