<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Notebook_Week10"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Notebook/Week10">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Notebook/Week10&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week10&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week10&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Notebook/Week10" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Notebook/Week10</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="wrap"><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:Cambridge">Home</A></LI><LI><A href="/Team:Cambridge/TheTeam">Meet the Team</A></LI><LI><A href="/Team:Cambridge/Photos">Photo Gallery</A></LI><LI><A href="/Team:Cambridge/Videos">Videos</A></LI><LI><A href="/Team:Cambridge/BioBricks">Parts submitted</A></LI></UL><LI><A class="head" href="#">Project Firefly</A></LI><LI><A href="/Team:Cambridge/Bioluminescence">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background_Firefly">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Colour">Coloured outputs</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Luciferin_Regeneration">Luciferin recovery</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Project Vibrio</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Luciferases">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/G28">The LuxBrick</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">Codon optimisation</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:Cambridge/Tools/Gibson">Gibthon</A></LI><LI><A href="/Team:Cambridge/Tools/GenBank">BioBrick → GenBank</A></LI><LI><A href="/Team:Cambridge/Tools/Ligate">Ligation Calculator</A></LI><LI><A href="/Team:Cambridge/Tools/Eglometer">The E.glometer</A></LI><LI><A class="head" href="#">Gibson Assembly</A></LI><LI><A href="/Team:Cambridge/Gibson/Introduction">Introduction</A></LI><LI><A href="/Team:Cambridge/Gibson/Mechanism">How it works</A></LI><LI><A href="/Team:Cambridge/Gibson/Protocol">Protocol</A></LI><LI><A href="/Team:Cambridge/Gibson/RFC">RFC</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:Cambridge/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Cambridge/Safety">Safety</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:Cambridge/Tools/microMeasure">Biosensors</A></LI><LI><A href="/Team:Cambridge/Tools/Lighting">Lighting</A></LI><LI><A class="head" href="/Team:Cambridge/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:Cambridge/Notebook/Summary">Summary</A></LI><LI><A href="/Team:Cambridge/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:Cambridge/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:Cambridge/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:Cambridge/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:Cambridge/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:Cambridge/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:Cambridge/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:Cambridge/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:Cambridge/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:Cambridge/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:Cambridge/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:Cambridge/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:Cambridge/Notebook/FurtherWork">Beyond Week 12</A></LI><LI><A href="/Team:Cambridge/Protocols">Protocols</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">Our Partners</A></LI><LI><A href="/Team:Cambridge/Partners">Sponsors</A></LI><LI><A href="/Team:Cambridge/iGEM_Teams">iGEM Teams</A></LI><LI><A href="/Team:Cambridge/Departments">University</A></LI><LI><A href="/Team:Cambridge/Publicity">Publicity</A></LI></DIV><DIV id="right"><DIV style="width:730px; background-color:#fad72a;" class="secheader">Notebook: Week 10</DIV><DIV style="position:relative; height:30px;border-top:5px solid white;"><DIV style="width:492px;position:absolute; background-color:#96d446;" class="selecteddiv" id="projectlink"><P>Viewing project diary</P></DIV><DIV style="width:246px; position:absolute; background-color:#fb5c2b;left:497px;" class="selectordiv" id="lablink"><P>View protocols and data</P></DIV></DIV><DIV style="overflow:hidden; height:2200px; width:730px; position:relative;" id="maskingbox"><DIV style="width:1320px; position:relative;" id="superwide"><TABLE style="width:1480px" id="widetable"><TBODY><TR><TD style="padding-right:50px; padding-left:10px; width:630px;vertical-align:top"><DIV align="center">Week 10: Monday 13th - Sunday 19th September</DIV><H2><SPAN class="mw-headline" id="Monday"> Monday </SPAN></H2><P>To our pleasant surprise our plates of P. pyralis in pSB1C3, L. cruciata in pSB1C3 and L. cruciata with pBAD in SB1C3 had all grown successfully over the weekend. Only P. pyralis with pBAD had not grown, which Hannah and Emily (who was back from holiday) repeated that experiment today.
</P><P>In the meantime, Bill investigated flights to America and Theo wrote a draft abstract which the team discussed over coffee. The thioesterase plates had also grown over the weekend as well, which was very exciting! Ben set up a plate reader experiment going overnight to test whether L. cruciata luciferase worked and Theo took very pretty pictures of light bulbs glowing with bacteria! 
</P><P>In trying to be organised, we've also grown up broths of the original DNA2.0 plasmids and the products of the plates to make glycerol stocks tomorrow. Ben put on a plate reader experiment in the evening while Theo made overnight cultures that could be sent for sequencing.
</P><H2><SPAN class="mw-headline" id="Tuesday"> Tuesday </SPAN></H2><P>We sent a number of sequences for sequencing, courtesty of Source BioSciences.  Unfortunately in our hurry to meet the deadline we broke a pipette, and we missed it anyway. But the analysis of the plate reader experiment showed us that the Luciola cruciata luciferase was indeed working.  Hannah and Emily performed standard Biobrick assembly to try to ligate the pBAD promoter onto the EPIC mutant of Photinus pyralis luciferase and LRE operon, which failed before.  We got very excited about blog posts by our very own <A href="http://labrat.fieldofscience.com/2010/09/bacterial-lightbulbs.html" class="external text" rel="nofollow">lab rat</A> and also <A href="http://scienceblogs.com/oscillator/2010/09/bacterial_lightbulb.php" class="external text" rel="nofollow">Oscillator</A>.  Theo plated out Luciola cruciata on an arabinose containing plate in order to make a red bulb to add to our blue.
</P><H2><SPAN class="mw-headline" id="Wednesday"> Wednesday </SPAN></H2><P>The ligation had failed so Emily and Hannah redid their experiment.  Meanwhile, Ben attempted to use site-specific PCR mutagenesis followed by Gibson assembly to mutate the red mutant of Luciola cruciata luciferase back to wild type.  This is a first step needed so that we can mutate it into a number of other colours.
</P><H2><SPAN class="mw-headline" id="Thursday"> Thursday </SPAN></H2><P>Since Hannah and Emily's experiment did not seem to have produced a viable ligation, Theo tried this yet again.  But over the course of the day we noticed small colonies growing on a plate from the first ligation, which was an encouraging result.  Ben retried the Gibson assembly and the team rehearsed a Gibson Assembly song Theo had written the previous evening.  Ben arranged it for ukelele!
</P><H2><SPAN class="mw-headline" id="Friday"> Friday </SPAN></H2><P>Today we got the robot in Jim's lab to do some work for us, it drew iGEM in Pixels on a 96 well plate!  We hope to get this working with all the firefly colours eventually, and hopefully to mix them together.
We also tidied the lab, and continued construction of a bacterial bubble lamp.
</P><H2><SPAN class="mw-headline" id="Saturday">Saturday</SPAN></H2><P>Theo and Bill came into the lab.  Bill performed a number of Gibson assemblies attempting to separate LRE and luciferase BioBricks.  Theo tried continuing Ben's work on site directed mutagenesis of Luciola cruciata luciferase back to wild-type.
</P><H2><SPAN class="mw-headline" id="Sunday">Sunday</SPAN></H2></TD><TD style="width:630px; vertical-align:top"><DIV align="center">Week 10: Monday 13th September - Sunday 19th September</DIV><H2><SPAN class="mw-headline" id="Monday_2">Monday</SPAN></H2><H3><SPAN class="mw-headline" id="100._Expt:_Adding_pBAD_to_P.P._pSB1C3_.28Hannah_and_Emily.29">100. Expt: Adding pBAD to P.P. pSB1C3 (Hannah and Emily)</SPAN></H3><UL><LI>Took P.P. pSB1C3 out of fridge
</LI><LI>Extracted pBAD
</LI><LI>Restricted pBAD with E + S (using protocol from Fermentas below)
</LI><LI>Restricted P.P. pSB1C3
</LI><LI>Ran on gel
</LI></UL><P>Here are gel columns:
</P></TD></TR><TR><TD>Lane 1
</TD><TD>Lane 2
</TD><TD>Lane 3
</TD><TD>Lane 4
</TD><TD>Lane 5
</TD><TD>Lane 6
</TD><TD>Lane 7
</TD></TR><TR><TD>EZ Ladder II
</TD><TD>P.P.pSB1C3
</TD><TD>P.P.pSB1C3
</TD><TD>pBAD (1)
</TD><TD>pBAD (1)
</TD><TD>pBAD (2)
</TD><TD>pBAD (2)
</TD></TR><TR><TD>5µl
</TD><TD>24µl
</TD><TD>12µl
</TD><TD>12µl
</TD><TD>12µl
</TD><TD>12µl
</TD><TD>12µl
</TD></TR></TBODY></TABLE><P>Results of gel: P.P.+pSB1C3 worked but pBAD could not be seen. P.P.+pSB1C3 was around 5kb as expected. Ben performed gel extraction on this. 
</P><P>We used pBAD and P.P.+pSB1C3 from a previous experiment (from the freezer) and performed ligation using the Fermentas protocol below, but using the following quantities:
</P><TABLE class="wikitable"><TBODY><TR><TD>Volume (µl)
</TD></TR><TR><TD>P.P. pSB1C3
</TD><TD>1
</TD></TR><TR><TD>pBAD
</TD><TD>1
</TD></TR><TR><TD>DI H20
</TD><TD>13
</TD></TR><TR><TD>5x Rapid ligation Buffer
</TD><TD>4
</TD></TR><TR><TD>T4 DNA Ligase
</TD><TD>1
</TD></TR><TR><TD>20
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Fermentas_Restriction_Digest_Protocol">Fermentas Restriction Digest Protocol</SPAN></H4><P>1. Combine the following reaction components at room temperature in the order indicated:
</P><TABLE class="wikitable"><TBODY><TR><TD>Plasmid DNA
</TD></TR><TR><TD>Water, nuclease-free
</TD><TD>15µl
</TD></TR><TR><TD>10x FastDigest Buffer
</TD><TD>2µl
</TD></TR><TR><TD>DNA
</TD><TD>2µl (up to 1µg)
</TD></TR><TR><TD>FastDigest enzyme
</TD><TD>1µl
</TD></TR><TR><TD>20µl
</TD></TR></TBODY></TABLE><P>2. Mix gently and spin down
</P><P>3. Incubate at 37°C in a heat block or water thermostat for 5min (we actually left them for 30 mins). Optional: Inactivate the enzyme by heating for 5min at 80°C. 
</P><H4><SPAN class="mw-headline" id="Fermentas_Ligation_Protocol">Fermentas Ligation Protocol</SPAN></H4><P>1. Add the following to a microcentrifuge tube:
</P><TABLE class="wikitable"><TBODY><TR><TD>Linearized vector DNA
</TD><TD>10-100ng
</TD></TR><TR><TD>Insert DNA (at 3:1 Molar excess over vector)
</TD><TD>variable
</TD></TR><TR><TD>5x Rapid Ligation Buffer
</TD><TD>4µl
</TD></TR><TR><TD>T4 DNA Ligase, 5u/µl
</TD><TD>1µl
</TD></TR><TR><TD>Water, nuclease-free
</TD><TD>to 20µl
</TD></TR></TBODY></TABLE><P>2. Vortex and spin briefly to collect drops.
</P><P>3. Incubate the mixture at 22°C (RT) for 5 mins (we used 30 mins)
</P><P>4. Use 2-5µl of the ligation mixture for transformation
</P><P>The reaction mixture can be stored at 0-4°C until used for transformation. 
</P><H3><SPAN class="mw-headline" id="101._Expt:_Transformation_of_plasmid_obtained_in_experiment_on_p88">101. Expt: Transformation of plasmid obtained in experiment on p88</SPAN></H3><P>We transformed TOP10 cells with the plasmid extracted from the experiment on p88 using the standard protocol. We incubated overnight on Cm plates and obtained colonies on plates from plasmids of lanes: bottom; 2 top; 8; 5; 7; 8; 9. 
</P><P>We then grew overnight LB cultures for plasmid extraction. 
</P><P>The colonies had a slightly different aspect suggesting some possible contamination. I've therefore extracted plasmids from 3 lanes: 
</P><UL><LI>Lane 1 bottom: 19.8ng/µl
</LI><LI>Lane 2 top: 5.6ng/µl
</LI><LI>Lane 5: 16.8ng/µl
</LI></UL><P>These 3 have been sent to sequencing.
</P><H3><SPAN class="mw-headline" id="102._Expt:_Making_glycerol_stock">102. Expt: Making glycerol stock</SPAN></H3><P>In CC iGEM 2010 Green Box in -80°C Freezer:
</P><P>1. DNA2.0 PP in TOP10 from source
</P><P>2. DNA2.0 LC in TOP10 from source
</P><P>3. PP + pSB1C3
</P><P>4. LC + pSB1C3
</P><P>5. LC + pBAD + pSB1C3
</P><P>6. Thio Lane 1 bottom
</P><P>7. Thio Lane 2 top
</P><P>8. Thio Lane 5
</P><P>9. Thio Lane 7
</P><H2><SPAN class="mw-headline" id="Tuesday_2">Tuesday</SPAN></H2><H3><SPAN class="mw-headline" id="103._Expt:_Mutagenesis_of_Luciola_cruciata">103. Expt: Mutagenesis of Luciola cruciata</SPAN></H3><P>Will's mutagenesis primers will all be sorted in the green box in the freezer once diluted. Ordered primers diluted to 100µM as suggested by Biolegio. 
</P><P>Working stocks will be made (to reduce freeze-thaw damage to the main stocks) at 10mM. 
</P><P>Primers in reactions should be at 0.5µM (approx) so 1µl of dilute primer stock per 20µl reaction. 
</P><H4><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H4><UL><LI>Lyse cells first, take a loopful of colony and add 20µl of ddH20 (in a PCR tube)
</LI><LI>Heat at 98°C for 10 mins
</LI><LI>Cool to -80°C for ~10 mins
</LI><LI>Vortex for ~2mins
</LI></UL><P>PCR mix: 
</P><TABLE class="wikitable"><TBODY><TR><TD>ddH20
</TD><TD>7µl
</TD></TR><TR><TD>Phusion Mast Mix (2x)
</TD><TD>10µl
</TD></TR><TR><TD>F Primer
</TD><TD>1µl
</TD></TR><TR><TD>R Primer
</TD><TD>1µl
</TD></TR><TR><TD>Lysate
</TD><TD>1µl
</TD></TR><TR><TD>20µl reactions
</TD></TR></TBODY></TABLE><P>Used mutagenesis.f.LC and mut286.r.LC --&gt; Labelled (1)
</P><P>and mutagenesis.r.LC and mut286.f.LC (to mutate back to wild type) --&gt; Labelled (2)
</P><P>Started Gibson assembly with (1) and (2), products from colony PCR with 15µl Gibson master mix, 2.5µl (1), 2.5µl (2)
</P><H2><SPAN class="mw-headline" id="Wednesday_2">Wednesday</SPAN></H2><H3><SPAN class="mw-headline" id="104._Expt:_Adding_pBAD_to_P.P._pSB1C3_.28Hannah_.2B_Emily.29">104. Expt: Adding pBAD to P.P. pSB1C3 (Hannah + Emily)</SPAN></H3><UL><LI>Restriction: Use Fermentas protocol on p88 using the quantities below. Cut P.P. pSB1C3 used from experiment yesterday. Restriction performed on pBAD using E and S. 
</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>Volume (µl)
</TD></TR><TR><TD>10x FD Buffer
</TD><TD>2
</TD></TR><TR><TD>pBAD DNA
</TD><TD>6
</TD></TR><TR><TD>FD EcoRI
</TD><TD>1
</TD></TR><TR><TD>FD SpeI
</TD><TD>1
</TD></TR><TR><TD>Nuclease-free H20
</TD><TD>10
</TD></TR><TR><TD>20
</TD></TR></TBODY></TABLE><UL><LI>Gel: Showed 3 bands around 5100, 5000 and 1200, as expected. The gel was loaded as follows:
</LI></UL><P>Easy Ladder II, pBAD, pBAD, pBAD, pBAD
</P><UL><LI>Ligation: After gel extraction, pBAD results from nanodrop were unreliable, but eventually gave stable readings of 8.2ng/µl
</LI></UL><P>Ligation was performed using the protocol on p90 using the following quantities:
</P><TABLE class="wikitable"><TBODY><TR><TD>Volume (µl)
</TD></TR><TR><TD>5x Rapid Ligation Buffer
</TD><TD>4
</TD></TR><TR><TD>T4 DNA Ligase
</TD><TD>1
</TD></TR><TR><TD>P.P. pSB1C3
</TD><TD>6.5
</TD></TR><TR><TD>pBAD
</TD><TD>8.5
</TD></TR></TBODY></TABLE><P>Transformation failed :(
</P><H3><SPAN class="mw-headline" id="105._Expt:_Luminescence_measurements_with_plate_reader_to_measure_effect_of_Arabinose.3B_luciferin_and_D-cysteine">105. Expt: Luminescence measurements with plate reader to measure effect of Arabinose; luciferin and D-cysteine</SPAN></H3><P>We used 96-well plate and tested the effect of varying concentrations of Arabinose and luciferin with and without D-cysteine.
</P><P>There repeats were made each time
</P><TABLE class="wikitable"><TBODY><TR><TD>Luc
</TD></TR><TR><TD>10µM
</TD><TD>100µM
</TD><TD>1mM
</TD></TR><TR><TD>Arabinose
</TD><TD>10µM
</TD><TD>1-3
</TD><TD>4-6
</TD><TD>7-9 with DC
</TD></TR><TR><TD>10-12
</TD><TD>13-15
</TD><TD>16-18 without DC
</TD></TR><TR><TD>100µM
</TD><TD>19-21
</TD><TD>22-24
</TD><TD>25-27 with DC
</TD></TR><TR><TD>28-30
</TD><TD>31-33
</TD><TD>34-36 without DC
</TD></TR><TR><TD>1mM
</TD><TD>37-39
</TD><TD>40-42
</TD><TD>43-45 with DC
</TD></TR><TR><TD>46-48
</TD><TD>49-51
</TD><TD>52-54 without DC
</TD></TR></TBODY></TABLE><P>55-57: 100 luc, 0 Ara, No DC
58-60: 0 luc, 100 Ara, No DC
61-63: 100 luc, 100 Ara --&gt; No cells
</P><P>Cells were taken from 2 colonies of LC + pBAD + pSB1C3 and mixed in 10ml of LB. 
</P><P>75µl of LB + cells was added in each well + DI H20 and necessary Arabinose/luciferin in the right concentrations. 
</P><P>D-cysteine was added to ~1mM
</P><H2><SPAN class="mw-headline" id="Thursday_2">Thursday</SPAN></H2><H3><SPAN class="mw-headline" id="106._Expt:_Gibson_to_separate_luciferase_and_LRE_in_P._pyralis_and_L._cruciata_.28Bill.2C_Emily_and_Hannah.29">106. Expt: Gibson to separate luciferase and LRE in P. pyralis and L. cruciata (Bill, Emily and Hannah)</SPAN></H3><P>We diluted primers from Biolegio using the stated quantities of water on the product sheets. Then made a 1 in 10 dilution of this and followed the Finnzymes Phusion Master Mix protocol. The following was mixed for solutions A to P:
</P><TABLE class="wikitable"><TBODY><TR><TD>Volume (µl)
</TD></TR><TR><TD>DI H20
</TD><TD>19
</TD></TR><TR><TD>2x Phusion Master Mix
</TD><TD>25
</TD></TR><TR><TD>Primer 1
</TD><TD>2.5
</TD></TR><TR><TD>Primer 2
</TD><TD>2.5
</TD></TR><TR><TD>Template DNA
</TD><TD>1
</TD></TR><TR><TD>50
</TD></TR></TBODY></TABLE><P>Primer 1, primer 2 and template DNA were as follows for each solution:
</P><TABLE class="wikitable"><TBODY><TR><TD>Solution
</TD><TD>To extract
</TD><TD>Primer 1
</TD><TD>Primer 2
</TD><TD>Template DNA
</TD></TR><TR><TD>Uneven length outputs
</TD><TD>A
</TD><TD>LRE
</TD><TD>1
</TD><TD>6
</TD><TD>PP
</TD></TR><TR><TD>B
</TD><TD>Ext
</TD><TD>4
</TD><TD>7
</TD><TD>pSB1C3
</TD></TR><TR><TD>C
</TD><TD>Luc
</TD><TD>5
</TD><TD>2
</TD><TD>PP
</TD></TR><TR><TD>D
</TD><TD>Ext
</TD><TD>3
</TD><TD>8
</TD><TD>pSB1C3
</TD></TR><TR><TD>E
</TD><TD>LRE
</TD><TD>1
</TD><TD>10
</TD><TD>LC
</TD></TR><TR><TD>F
</TD><TD>Ext
</TD><TD>11
</TD><TD>4
</TD><TD>pSB1C3
</TD></TR><TR><TD>G
</TD><TD>Luc
</TD><TD>9
</TD><TD>2
</TD><TD>LC
</TD></TR><TR><TD>H
</TD><TD>Ext
</TD><TD>3
</TD><TD>12
</TD><TD>pSB1C3
</TD></TR><TR><TD>Even length outputs
</TD><TD>I
</TD><TD>LRE
</TD><TD>f
</TD><TD>6
</TD><TD>PP
</TD></TR><TR><TD>J
</TD><TD>LRE
</TD><TD>7
</TD><TD>r
</TD><TD>PP
</TD></TR><TR><TD>K
</TD><TD>Luc
</TD><TD>5
</TD><TD>r
</TD><TD>PP
</TD></TR><TR><TD>L
</TD><TD>Luc
</TD><TD>f
</TD><TD>8
</TD><TD>PP
</TD></TR><TR><TD>M
</TD><TD>LRE
</TD><TD>f
</TD><TD>10
</TD><TD>LC
</TD></TR><TR><TD>N
</TD><TD>LRE
</TD><TD>11
</TD><TD>r
</TD><TD>LC
</TD></TR><TR><TD>O
</TD><TD>Luc
</TD><TD>f
</TD><TD>12
</TD><TD>LC
</TD></TR><TR><TD>P
</TD><TD>Luc
</TD><TD>9
</TD><TD>r
</TD><TD>LC
</TD></TR></TBODY></TABLE><P>where f = forward mutagenesis primer, r = reverse mutagenesis primer, PP = P. pyralis, LC = L. cruciata
</P><P>These solutions were run in the PCR machine using the following program: 
</P><TABLE class="wikitable"><TBODY><TR><TD>Temp (°C)
</TD><TD>Duration
</TD></TR><TR><TD>Initial Denaturation
</TD><TD>98
</TD><TD>30s
</TD></TR><TR><TD>35 cycles
</TD></TR><TR><TD>Denaturation
</TD><TD>98
</TD><TD>8s
</TD></TR><TR><TD>Annealing
</TD><TD>67-75
</TD><TD>25s
</TD></TR><TR><TD>Elongation
</TD><TD>72
</TD><TD>2m30s
</TD></TR><TR><TD>End cycle
</TD></TR><TR><TD>Final elongation
</TD><TD>72
</TD><TD>10mins
</TD></TR><TR><TD>Hold
</TD><TD>4
</TD></TR></TBODY></TABLE><P>This experiment was first performed on 16th September and the diagnostic gel showed it had failed =&gt; the bands were all in the same places and wrong. This was because the lid had not been closed properly on the PCR machine; the hot plate was not touching the tube lids. Also, the hot lid had not been switched on so condensation formed at the tops of the tubes. 
</P><H2><SPAN class="mw-headline" id="Friday_2">Friday</SPAN></H2><P>When repeated on Friday there was no condensation and the gel showed solutions A to H had all the correct lengths as expected. However, only K, M and P had worked for the remaining solutions. 
</P><P>So, Ben gel extracted solutions A to H. 
</P><P>Bill transformed and plated out the results over the weekend.
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Cambridge/Notebook/Week10">http://2010.igem.org/Team:Cambridge/Notebook/Week10</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Notebook/Week10" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Notebook/Week10" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week10&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week10&amp;oldid=62560" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>