<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Bioluminescence_Bacterial_Codon_optimisation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Bioluminescence/Bacterial Codon optimisation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="wrap"><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#"> Introduction</A></LI><LI><A href="/Team:Cambridge">Home</A></LI><LI><A href="/Team:Cambridge/TheTeam">Meet the Team</A></LI><LI><A href="/Team:Cambridge/Photos">Photo Gallery</A></LI><LI><A href="/Team:Cambridge/Videos">Videos</A></LI><LI><A href="/Team:Cambridge/BioBricks">Parts submitted</A></LI></UL><LI><A class="head" href="#">Project Firefly</A></LI><LI><A href="/Team:Cambridge/Bioluminescence">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background_Firefly">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Colour">Coloured outputs</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Luciferin_Regeneration">Luciferin recovery</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Firefly_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Project Vibrio</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Luciferases">Introduction</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Background">Background</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/G28">The LuxBrick</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">Codon optimisation</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Modelling">Modelling</A></LI><LI><A href="/Team:Cambridge/Bioluminescence/Vibrio_Characterisation">Characterisation</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:Cambridge/Tools/Gibson">Gibthon</A></LI><LI><A href="/Team:Cambridge/Tools/GenBank">BioBrick → GenBank</A></LI><LI><A href="/Team:Cambridge/Tools/Ligate">Ligation Calculator</A></LI><LI><A href="/Team:Cambridge/Tools/Eglometer">The E.glometer</A></LI><LI><A class="head" href="#">Gibson Assembly</A></LI><LI><A href="/Team:Cambridge/Gibson/Introduction">Introduction</A></LI><LI><A href="/Team:Cambridge/Gibson/Mechanism">How it works</A></LI><LI><A href="/Team:Cambridge/Gibson/Protocol">Protocol</A></LI><LI><A href="/Team:Cambridge/Gibson/RFC">RFC</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:Cambridge/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Cambridge/Safety">Safety</A></LI><LI><A class="head" href="#">Future applications</A></LI><LI><A href="/Team:Cambridge/Tools/microMeasure">Biosensors</A></LI><LI><A href="/Team:Cambridge/Tools/Lighting">Lighting</A></LI><LI><A class="head" href="/Team:Cambridge/Notebook/Week1">Notebook</A></LI><LI><A href="/Team:Cambridge/Notebook/Summary">Summary</A></LI><LI><A href="/Team:Cambridge/Notebook/Week1">Week 1</A></LI><LI><A href="/Team:Cambridge/Notebook/Week2">Week 2</A></LI><LI><A href="/Team:Cambridge/Notebook/Week3">Week 3</A></LI><LI><A href="/Team:Cambridge/Notebook/Week4">Week 4</A></LI><LI><A href="/Team:Cambridge/Notebook/Week5">Week 5</A></LI><LI><A href="/Team:Cambridge/Notebook/Week6">Week 6</A></LI><LI><A href="/Team:Cambridge/Notebook/Week7">Week 7</A></LI><LI><A href="/Team:Cambridge/Notebook/Week8">Week 8</A></LI><LI><A href="/Team:Cambridge/Notebook/Week9">Week 9</A></LI><LI><A href="/Team:Cambridge/Notebook/Week10">Week 10</A></LI><LI><A href="/Team:Cambridge/Notebook/Week11">Week 11</A></LI><LI><A href="/Team:Cambridge/Notebook/Week12">Week 12</A></LI><LI><A href="/Team:Cambridge/Notebook/FurtherWork">Beyond Week 12</A></LI><LI><A href="/Team:Cambridge/Protocols">Protocols</A></LI><LI><A href="/Team:Cambridge/Partners" class="head">Our Partners</A></LI><LI><A href="/Team:Cambridge/Partners">Sponsors</A></LI><LI><A href="/Team:Cambridge/iGEM_Teams">iGEM Teams</A></LI><LI><A href="/Team:Cambridge/Departments">University</A></LI><LI><A href="/Team:Cambridge/Publicity">Publicity</A></LI></DIV><DIV id="right"><DIV style="width:730px; background-color:#386abc;" class="secheader">Project Vibrio: Bacterial Codon Optimisation</DIV><DIV class="contwrap"><H2 class="topheader">What is codon usage?</H2><P>One of the fascinating features of life is the <B>universal genetic code</B>.  In all known organisms, from bacteria to man, the same triplets of DNA bases code for the same amino acids.  However this does not mean that all species encode their genomes in exactly the same way.  
The code is <I>redundant</I>: a number of triplets code for the same amino acid.  While all species are able to translate any sequence of DNA interchangeably, E. coli prefers to use certain triplets to code for certain amino acids which may be different to the ones we use.  This 'preference' is reflected in the levels of tRNA which match such a triplet.  In this project we resynthesised a number of genes <I>de novo</I> and thus were able to codon optimise them for expression in E. coli.
</P><H1><SPAN class="mw-headline" id="Improved_translational_speed">Improved translational speed</SPAN></H1><P>Starting with the DNA sequence of the <I>Vibrio fischeri</I> lux operon found on the NCBI database, we used a number of tools to replace the codons used with the most common codons found in the E.coli genome. To achieve optimal expression of the Lux operon in E.coli, we had the operon re-synthesized after optimising the usage of codons. This conserves the sequence of amino acids in the gene products, but improves the rate of translation, as more common tRNAs are recruited. Codon usage optimization can yield dramatic increases in the expression of foreign genes, especially if they are introduced from less closely related species <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6TCW-4C8NKCY-3&amp;_user=6094838&amp;_coverDate=07%2F31%2F2004&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_origin=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_searchStrId=1511316646&amp;_rerunOrigin=scholar.google&amp;_acct=C000053194&amp;_version=1&amp;_urlVersion=0&amp;_userid=6094838&amp;md5=c970323b521fa1ec9d3050d4c4970eb1&amp;searchtype=a" class="external text" rel="nofollow">Gustafsson et al. 2004</A></P><H1><SPAN class="mw-headline" id="Altered_G-C_content">Altered G-C content</SPAN></H1><P>DNA curvature is increased by sequences rich in A-T or G-C pairs. The natural <I>V.fischeri</I> Lux operon, and especially its intergenic regions, contains stretches rich in A-T, resulting in the curvature that H-NS proteins bind to preferentially. Changing the coding DNA sequence also meant changing the curvature of the DNA, which affects the binding affinity of H-NS proteins. To alleviate the repression that H-NS exerts, we took care to raise the G-C content of intergenic regions and coding sequences (at times resorting to suboptimal codons). According to a computational prediction, this resulted in greatly reduced DNA curvature, and thus hopefully to a reduced affinity for H-NS proteins.
</P><TABLE><TBODY><TR><TD><B>Change in G-C(red)/A-T(blue) content by codon optimization. The occurence of long A-T rich stretches is reduced in the re-synthesised version of the Lux operon.</B></TD><TD><B>Computational prediction of DNA curvature before and after codon optimization. (click to enlarge)</B></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Differential_Expression">Differential Expression</SPAN></H1><P>In <I>Vibrio fischeri</I>, LuxA and B are expressed at five times the levels of LuxC, D, E and G. Since all these genes are transcribed on the same mRNA, but have their own Ribosome Binding Sites, this is probably due to differences in codon usage (<A href="http://www.annualreviews.org/doi/pdf/10.1146/annurev.mi.42.100188.001055" class="external text" rel="nofollow">Meighen et al. 1988</A>). A number of rare codons are found in Lux C, D, E and G, but not in LuxA and B. Since we did not receive the newly synthesised LuxA and B in time, we constructed a new operon using our LuxC, D, E and G, and the LuxAB BioBrick that was put into the registry by the Edinburgh iGEM team 2009 (BBa_216008). These genes originate from Xenorhabdus luminescens. Compared to <I>Vibrio fischeri</I>, there is only limited amino acid identity in the Lux A and B genes (52% and 66% respectively). Yet the literature describes both as using the same substrates. <A href="http://aem.asm.org/cgi/content/abstract/55/10/2607" class="external text" rel="nofollow">Schmidt et al. 1989</A> describe positive complementation test using components from <I>V. fisheri</I> and <I>Xenorhabdus luminescens</I>. To use foreign genes from two very different donor species in one pathway in E.coli is an exciting test of our understanding of the processes involved in bacterial bioluminescence and of the power of synthetic biology in general.
In this construct, LuxC, D, E and G are codon optimised, but LuxA and B are not. In order to adjust the ratio of gene expression between these genes to the state found in nature, we chose to put LuxA and B under a very strong, phage derived promoter (plambda) to be constitutively expressed. The other genes can now be put under any promoter to create a PoPS-to-light device. In conjunction with an inducible or repressible promoter, this could be used as a reporter device. To test the system, we placed the LuxCDEG under an arabinose induced pbad promoter (BBa_i0500).
</P><H1><SPAN class="mw-headline" id="Parts_submitted_to_the_registry">Parts submitted to the registry</SPAN></H1><P>Ideally we would have submitted a codon-optimised version of the entire Lux operon. Unfortunately Mr Gene, the company we employed for synthesis, still had not completed the order two months after placing, and at the point of the wiki-freeze we still have not received the optimised versions of LuxA and B. In order to complete our aim of creating a PoPs-&gt;light device that can be placed under any promoter, we combined the optimised <I>Vibrio</I> LuxCDEG genes with the Edinburgh 2009 <I>Xenorhabdus</I> LuxAB. The assembly was achieved using the <A href="https://2010.igem.org/Team:Cambridge/Gibson/Introduction" class="external text" rel="nofollow">Gibson method</A>. Since we only received LuxCD and LuxEG from Mr Gene two weeks before the documentation deadline, we could not properly characterise these parts. For a complete list of the BioBricks we submitted see the <A href="https://2010.igem.org/Team:Cambridge/BioBricks" class="external text" rel="nofollow">BioBricks section</A>.
</P><P>
Codon optimised bacterial luminescence parts:
</P><P><A href="http://partsregistry.org/Part:BBa_K325902" class="external text" rel="nofollow">luxCD (BBa_K325902)</A> derived from V. fischeri, but optimised for E.coli
</P><P><A href="http://partsregistry.org/Part:BBa_K325903" class="external text" rel="nofollow">luxEG (BBa_K325903)</A> derived from V. fischeri, but optimised for E.coli
</P><P><A href="http://partsregistry.org/Part:BBa_K325905" class="external text" rel="nofollow">CDEG pLambda AB (BBa_K325905)</A> a PoPs-&gt;light device that can be placed under any promoter, hybrid device containing promoterless V. fisheri derived CDEG (optimised) and Xenorhabdus AB under a strong constitutive promoter
</P><P><A href="http://partsregistry.org/Part:BBa_K325906" class="external text" rel="nofollow">pbad CDEG pLambda AB (BBa_K325906)</A> the above part under the arabinose induced pbad promoter

</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation">http://2010.igem.org/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Bioluminescence/Bacterial_Codon_optimisation&amp;oldid=202014" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>