"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Non-Biologist
From 2010.igem.org
Introduction
Home
Meet the Team
Photo Gallery
Videos
Parts submitted
Project Firefly
Introduction
Background
Coloured outputs
Luciferin recovery
Modelling
Characterisation
Project Vibrio
Introduction
Background
The LuxBrick
Codon optimisation
Modelling
Characterisation
Tools
Gibthon
BioBrick → GenBank
Ligation Calculator
The E.glometer
Gibson Assembly
Introduction
How it works
Protocol
RFC
Considerations
Human Practices
Safety
Future applications
Biosensors
Lighting
Notebook
Summary
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
Beyond Week 12
Protocols
Our Partners
Sponsors
iGEM Teams
University
Publicity
For the non-biologist
Synthetic biology is a new discipline which has arisen as scientists have developed better techniques for manipulating genetic material, and a better understanding of how existing organisms work. In almost every case it involves taking the genes which exist naturally after billions of years of evolution and combining and changing them in novel ways.
Engineering disciplines rely on standardisation of parts and one of the chief aims of iGEM is to create a Registry of Standard Parts for synthetic biology. Teams must submit all the genetic constructs they use in their projects to this central repository from which they are distributed to future teams.  Gradually a full toolkit is built up.
The work of the Cambridge team this year is a prime example of both these aspects of iGEM.  The team began the project by scouring the light producer's that had been produced by the natural world.  They considered a large number of candidates, including glow worms, jellyfish and a bacterium called Photobacterium phosphoreum.
But in the end they settled on three species:
The North American Firefly, Photinus pyralis was selected because it is known for its brightness, and because papers had recently been published which suggested more progress could be made
The Japanese Firefly, Luciola cruciata was selected because papers showed that it could be easily changed to emit a number of different colours
Vibrio fischeri, a bacterium which inhabits squid, and allows them to emit light (a symbiotic relationship ) was selected for its blue colour and because unlike firefly genes it required nothing but nutrients to emit light.
The team then needed the DNA to work with, a number of approaches were taken here.
Vibrio fischeri=
Genes had already been extracted from Vibrio fischeri and stored in circles of DNA known as plasmids.  We contacted James Slock at King's College who offered to send us his plasmids.  But when they arrived we still had a great deal of work to do.   Vibrio fischeri has a rather interesting way of life.  It can either be free-living or live in the light organs of large marine organisms.  These two distinct lifestyles require different proteins to be produced by the bacteria.  It is useless to waste resources emitting light when floating around as a single cell in the sea.  The bacterium has a clever control mechanism - each bacteria constantly produces a small amount of a chemical called AHL which is free to diffuse into the liquid outside.  Only at high levels of this chemical will the genes for light emission be activated.
This activation process is controlled by a particular region of DNA known as a promoter which comes just before the light producing genes. We wanted to end the reliance on AHL for light output, to achieve this we made a new BioBrick using a promoter called pBAD from the Registry.  This allows light output to be triggered by adding a small amount of a sugar called arabinose.  We also synthesised a BioBrick containing just the light emitting genes with no promoter, this will allow future teams to use light output in any way they which.  For example they could use a carbon-monoxide sensitive promoter to make a sensor.
North American Firefly
The team followed two approaches for the North American firefly.  Fireflies use a protein called luciferase to emit light. There was already a luciferase in the Registry which had been sent to us.  It had been made by the team from Ljubjana in a previous year.  But some work had to be done.  We were using as our "chassis" the bacterium E. coli, a workhorse of molecular biology.  E.coli are only distantly related to bacteria.  It is a fascinating phenomenon that despite this deep separation the basic language that genes are written in is interchangeable between the two species.  However some control elements are different.  The team had to replace a eukaryotic (eukaryotes are a branch of the tree of life that include us) signal called an RBS with the equivalent RBS for bacteria.  RBS stands for Ribosome Binding Site and is the signal for a complex called the ribosome to begin turning the messages produced by a gene into a protein.
Luciferase is not the only thing needed to emit light, it needs a fuel known as luciferin.  An analogy can be drawn with a candle with luciferase as the wick and luciferin as the wax.  The team also wanted to investigate how to make the fuel last longer.  There is a protein in fireflies called luciferin regenerating enzyme (LRE) which returns spent fuel (oxyluciferin) into a form in which it is ready for use again.  However no samples of physical DNA were available to the team for this.  Therefore we sent the string of letters which makes it up to one of our sponsors, DNA 2.0.  Their facilities are able to turn this digital information into physical DNA which our cells can use to make proteins.  This synthesis process also meant that we were able to tailor the DNA so that it was especially suited for expression in E. coli (a process known as codon optimisation .
Retrieved from " http://2010.igem.org/Team:Cambridge/Non-Biologist "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
