<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-Software_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-Software/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Document</TITLE></P><DIV class="container" id="head_1"><DIV class="row top_nav" id="top_area"><DIV class="col-lg-5 col-md-5 col-sm-5 col-xs-5 my_nav"><DIV class="row "><DIV class="col-lg-3 col-md-2 col-sm-3 col-xs-4 xl"><A href="https://2018.igem.org/Team:UESTC-Software">Home</A></DIV><DIV class="col-lg-2 col-md-3 col-sm-3 col-xs-4 xl"><A href="#">Project</A><UL class="menu" style="left: -10px;"><LI><A href="https://2018.igem.org/Team:UESTC-Software/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Applied_Design">Applied_Design</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Validation">Validation</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Contribution">Contribution</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-lg-3 col-md-2 col-sm-3 col-xs-4 xl"><A href="https://2018.igem.org/Team:UESTC-Software/Model">Model</A></DIV><DIV class="col-lg-4 col-md-5 col-sm-3 col-xs-12 xl"><A href="https://2018.igem.org/Team:UESTC-Software/Human Practices">Human Practice</A><UL class="menu" style="left: -5px; height:90px"><LI><A href="https://2018.igem.org/Team:UESTC-Software/Human_Practices">Integrated HP
</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Public_Engagement">Public_Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Collaborations">Collaborations</A></LI></UL></DIV></DIV></DIV><DIV class="col-lg-5 col-md-5 col-sm-5 col-xs-5 my_nav"><DIV class="row "><DIV class="col-lg-3 col-md-3 col-sm-4 col-xs-4 xl"><A href="https://2018.igem.org/Team:UESTC-Software/Attributions">Attributions</A></DIV><DIV class="col-lg-3 col-md-3 col-sm-2 col-xs-4 xl"><A href="https://2018.igem.org/Team:UESTC-Software/Team">Team</A></DIV><DIV class="col-lg-3 col-md-3 col-sm-3 col-xs-4 xl"><A href="https://2018.igem.org/Team:UESTC-Software/Awards">Awards</A></DIV><DIV class="col-lg-3 col-md-3 col-sm-3 col-xs-12 xl"><A href="https://2018.igem.org/Team:UESTC-Software/Safety">Safety</A></DIV></DIV></DIV></DIV></DIV><NAV class="navbar navbar-default" role="navigation" id="head_2"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example-navbar-collapse"><SPAN class="sr-only">切换导航</SPAN></BUTTON><A class="navbar-brand" href="https://2018.igem.org/Team:UESTC-Software">uestc<SPAN>software</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="example-navbar-collapse"><UL class="nav navbar-nav"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
               Project </A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Applied_Design">Applied_Design</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Validation">Validation</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Contribution">Contribution</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Notebook">Notebook</A></LI></UL><LI><A href="#">Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
               Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Integrated HP">Integrated HP</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Public_Engagement">Public_Engagement</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Awards">Awards</A></LI><LI><A href="https://2018.igem.org/Team:UESTC-Software/Safety">Safety</A></LI></DIV></NAV><DIV class="nav_model"><UL class="nav"><LI><A href="#Overview">Overview</A></LI><LI><A href="#blast">BLAST model</A></LI><LI class="nav_down"><A href="#introduction">Introduction</A></LI><LI class="nav_down"><A href="#results">BLAST results</A></LI><LI style="height:30px" class="nav_down"><A href="#screening">Screening</A></LI><LI><A href="#prediction">Prediction Model</A></LI><LI class="nav_down"><A href="#CNN">CNN</A></LI><LI style="height:30px" class="nav_down"><A href="#Multi-Classifier System">Multi-Classifier System</A></LI><LI style="height:30px" class="nav_down"><A href="#Global">Global Search</A></LI><LI><A href="#Reference">Reference</A></LI></UL></DIV><SECTION><DIV id="all-con-model"><DIV class="container"><DIV class="row"><DIV class="col-lg-11 col-lg-offset-1 col-md-10 col-md-offset-2 col-sm-10 col-sm-offset-2 col-xs-10 col-xs-offset-2"><H1 id="Overview">Overview</H1><P>We spend a lot of time and energy to construct our models, which played an important role in both parts of our project. For database integration, the first step of integration is to find the entry corresponding to the biobrick in the database to be integrated. For this we use the ncbi BLAST tool. However, the original BLAST matching results are not usable, since there are problems like redundancy and low similarity. In order to screen the BLAST results, we used JS divergence for feature extraction and innovatively employed logistic regression classifier with polynomial features to classify BLAST results, which performed pretty well. For promoter prediction, we used machine learning method to construct promoter classifier. Then, we constructed multiple classifier system to improve the performance. With large-scale search model, we can identify promoter in a large-scale sequence. With these, we constructed a promoter prediction model that can be used in both of prokaryote and eukaryote, which benefit biologists to find more promoters.</P><H1 id="blast">Get the power of BLAST</H1><H2 id="introduction">Introduction</H2><P>NCBI BLAST is one of the most well-known tools in the field of nucleic acid and protein sequence alignment. In order to efficiently analyze a large number of sequence data, we chose the local version: BLAST+2.7.1 as our sequence alignment tool <B>[1]</B>. The raw BLAST results are not usable, since there are redundancy and not all of the matched items are in high similarity, so we developed this model to select the trustworthy matches, then treating these matching sequences as identical or homologous sequences, and then performing subsequent information docking extensions. Through our manual sampling test, the two sequences matched by the BLAST have a high degree of consistency in function, so the accuracy of data expansion can be guaranteed to some extent.</P><H2 id="results">BLAST results</H2><P>BLAST+ provides 18 output formats, in order to simplify the procession, we choosed the tabular format. Here is part of the BLAST results:</P><P id="mingming">Table.1.BLAST results
                        </P><H3>Important values in BLAST</H3><P>Score -- sum of the scores in the traceback process of alignment, it directly indicates the similarity of two sequences. Higher score corresponds to higher similarity.</P><P>Evalue -- the expected number of HSPs(high score pairs) with score at least S. It’s the main threshold that can be used to limit the BLAST output. The smaller the evalue, the less the number of HSPs, and correspondingly, the higher the similarity of the two sequences.</P><DIV class="row"><DIV class="img_model col-lg-4 col-md-5 col-sm-5"><P id="mingming">Fig.1.BLAST parameters</P></DIV><P>Calculating: <SPAN>\(E=Kmne^{-\lambda S}\)</SPAN></P><P class="col-lg-7 col-lg-offset-1 col-md-6 col-md-offset-1 col-sm-5">Where: m and n are the lengths of the sequences aligned. K and lambda can be thought of simply as natural scales for the search space size and the scoring system respectively. <B>[2]</B>pident -- Percentage of identical matches, besides which is the mismatch area and gaps in the alignment arealength-- Alignment lengthqlen-- length of query sequence
                            </P></DIV><H2 id="screening">Screening the BLAST results</H2><H3>Problems</H3><P>Protein coding sequences is a big category in iGEM parts database, which contains 8454 parts(the total number of parts in the database is less than 30,000) and has longer sequence than other kind of parts on average. It’s both important and difficult for us to BLAST.</P><P>when doing BLAST between iGEM parts database and uniprot database to match the high similarity items, we’ve set the evalue to 1e-6. However, by artificial validation we’ve found that many of the matched pairs in the BLAST output is in low similarity in function or even have nothing to do with each other, which means the match is misleading.</P><P>Despite all of the matched pairs has evalue pretty low(most of them are far lower than 1e-6),but the ‘pident’ of them are varying from 19% to 100%, and the value of length/qlen is varying from a very low number to 100%--which badly needed filtrating.</P><H4>There are several facts important for analyzing the BLAST results:</H4><H5>1. Homologous:</H5><P>According to the theories of evolution, similarity of DNA sequence between different organisms is very common, and the level of similarity is related to the evolutionary distance; hence, the similar sequence identified by BLAST may be the homologous gene in two different species, most of which are similar in function.</P><H5>2. Allele:</H5><P>Even for one given gene in the same species, there may be several variant forms, which is called allele. Different alleles are highly similar in sequence, but they can result in different observable phenotypic traits, such as different pigmentation.</P><H4>Characteristics of the coding sequences in the iGEM database:</H4><P>1. Most of them are artificially modified or cut from existing proteins, in this case, the BLAST result is pretty nice.</P><P>2. Synthetic sequences: artificially constructed sequences, almost have no similar protein coding sequences.</P><P>3. Combined with several protein coding sequences’  fragments. In this case, one query sequence may hit several different nonhomologous proteins, but the similarity of every hit is not high.</P><P>4. Redundant and mistakes in the biobricks’ sequences: As is mentioned by many papers, redundant and even mistakes are possibly appearing in the biobricks’ sequences, which is a big trouble for us to recognize the right sequence when integrating other databases with the iGEM database.
                        </P><H3>Solution</H3><H4>Preliminary screening</H4><P>As the BLAST results are given in descending order of the similarity, we choose the first hit of every specific fragment in a gene as the probable match and abandon the others.</P><P id="mingming">Fig.2.Preliminary screening</P><P>By doing this, the amount of the results has been reduced to 8000+ while the origin amount is 61000+</P><H4>Logistic regression classifying</H4><P>Linear regression is a basic regression algorithm. For a regression problem:</P><P id="gongshi"><SPAN>\(Y=X\cdot W+\varepsilon\)</SPAN><SPAN style="position:relative; float:right;">(1)</SPAN></P><P>Where:</P><SPAN>$$X=[x_1,\ x_2,\ x_3,\ ...,\ x_n]^T$$ 
                        <P id="gongshi"><SPAN>\(W=[w_1,\ w_2,\ w_3,\ ...,\ w_n]^T\)</SPAN><SPAN style="position:relative; float:right;">(2)</SPAN></P>
                        $$\varepsilon=[\varepsilon_1,\ \varepsilon_2,\ \varepsilon_3,\ ...,\ \varepsilon_n]^T$$</SPAN><P>The aim is to search for the best W that minimize the mean of e.</P><P>With the help of logistic function: <SPAN>\(Logistic(z)=\frac{1}{1+\mathbf{e}^{-z}}\)</SPAN>,logistic regression is created to solve classification problem.</P><H4>probability to be ‘1’:</H4><P id="gongshi"><SPAN>\(\hat p=\sigma(\theta^T \cdot x_b)=\frac{1}{1+\mathbf{e}^{-{\theta^{T \cdot x_b}}}}\)</SPAN><SPAN style="position:relative; float:right;">(3)</SPAN></P><P>Where:</P><SPAN>$$\theta^T=[\theta_1,\ \theta_2,\ \theta_3,\ ...,\ \theta_m]$$</SPAN><P id="gongshi"><SPAN>\(x_b=[x_1,\ x_2,\ x_3,\ ...,\ x_m]\)</SPAN><SPAN style="position:relative; float:right;top:-20px">(4)</SPAN></P><H4>label:</H4><P id="gongshi"><SPAN>\(\hat y=\begin{cases}
1,\ \hat p&gt;0.5\\
0,\ \hat p&lt;0.5\\
\end{cases}
\)</SPAN><SPAN style="position:relative; float:right;top:15px">(5)</SPAN></P><H4>cost function:</H4><P id="gongshi"><SPAN>\(J(\theta)=-\frac{1}{m}\displaystyle\sum_{i=1}^m y^{(i)}\log(\hat p^{(i)})+(1-y^{(i)})\log(1-\hat p^{(i)})\\
=-\frac{1}{m}\displaystyle\sum_{i=1}^m y^{(i)}\log(\sigma(-X_b^{(i)}\theta))+(1-y^{(i)})\log(1-\sigma(-X_b^{(i)}\theta))\\
\)</SPAN><SPAN style="position:relative; float:right;top:50px">(6)</SPAN></P><H4>Regularization:</H4><P>Overfitting is a disturbing problem in machine learning, using penalty can help reduce it significantly.</P><P>Cost with L1 penalty: <SPAN>\(cost=C \cdot J(\theta)+||\theta^T||\)</SPAN></P><P>Cost with L2 penalty: <SPAN>\(cost=C \cdot J(\theta)+0.5\min{||\theta^T||}^2\)</SPAN></P><P>(C is a coefficient used to regulate the degree of regularization)</P><H4>Coordinate descent:</H4><P>The core task of this classifier is to minimize the cost function, coordinate descent is one of the methods. After trying several methods we've found that coordinate descent algorithm is the most appropriate one for our issue.</P><P>Given a differentiable convex function \(f:\mathbb{R}^n \to \mathbb{R}\) , this method assumes that if there’s a value x0 that minimize the f(x) on each coordinate, then f(x0) is the minimum value of this function.</P><P>Coordinate descent starts with initial variable values: <SPAN>\(x=(x_1^0,\ x_2^0,\ x_3^0,\ ...,\ x_n^0)\)</SPAN>, then loop k times for every coordinate respectively to find the minimum value.</P><P id="gongshi"><SPAN>\(x_1^{(k)}\in \displaystyle{\arg\min}_{x_1}{f(x_1,\ x_2^{(k-1)},\ x_3^{(k-1)},\ ...,\ x_n^{(k-1)})}\\
x_2^{(k)}\in \displaystyle{\arg\min}_{x_2}{f(x_1^{(k)},\ x_2,\ x_3^{(k-1)},\ ...,\ x_n^{(k-1)})}\\
x_3^{(k)}\in \displaystyle{\arg\min}_{x_3}{f(x_1^{(k)},\ x_2^{(k)},\ x_3,\ ...,\ x_n^{(k-1)})}\\
......\\
x_n^{(k)}\in \displaystyle{\arg\min}_{x_n}{f(x_1^{(k)},\ x_2^{(k)},\ x_3^{(k)},\ ...,\ x_n)}\\
\)</SPAN><SPAN style="position:relative; float:right;top:90px">(7)</SPAN></P><H4>Polynomial features</H4><P>original logistic regression is limited to linear situation, however, our problem is a nonlinear one. Polynomial features can help to make the decision boundary more complex that can fit the training sample better.</P><H4>(1)Feature extraction</H4><P>BLAST output provides more than 10 parameters, but not all of them are important for similarity screening, such as sites and sequence length. At the same time, some of the parameters are closely related to each other, so the features used for training don’t need to contain all of them.</P><P>To select features with high information content, we used JS divergence(Jensen-Shannon divergence). JS divergence is a method to measure the similarity of two probability distributions. </P><P id="mingming">Fig.3.Relationship between JS divergence and similarity</P><P>We divided the continuous numerical features into discrete segments, and the frequencies in which the different segments appear in the positive and negative samples were taken as the corresponding probabilities, and calculated the JS divergence. The greater the JS divergence of the feature, the greater the impact on the classification result, in other words, the more important the feature is. We arranged all features in descending order of JS divergence values. The result is as follow:</P><P id="mingming">Table.2.JS divergence of BLAST parameters</P><P>Finally, according to the JS divergence order, the correlation between features and the specific requirements of screening, the following three parameters are selected:</P><H5>1.int_evalue=-\(\log_{10}\)(evalue)</H5><P>Evalue is the main threshold parameter, unfortunately the value, which varying from 1e-6 to 0,is too small for SVM to do classification. To solve this, we use the int_evalue to replace it.</P><H5>2.pident</H5><P>int_evalue limits the similarity of the hits by control the probability of the hits’ appearance in the whole searching space, for every specific hit, we use pident.</P><H5>3.pqlen=length/qlen</H5><P>To filtrate the short fragments, we bring in the ratio parameter pqlen.</P><H5>High similarity requires high value of all three parameters</H5><H4>(2) Constructing training set</H4><P>We‘ve randomly chose 1/10 items from BLAST result and identified them by manually comparing the consistency of the two sequences in the databases they belong to, with labeling the idented items with 1 and unidented with 0.</P><H4>(3) Training</H4><P>Sklearn is a well-known machine learning package for python, which is powerful and convenient to use, we use it to do our classification. <B>[3][4]</B></P><H5>gridsearch</H5><P>We use gridsearch strategy to search for the optimal parameters, for coding sequence, the result is: degree=3,C=0.44,penalty=l1, which make the accuracy of 3-fold cross validation reach up to 0.985.</P><P id="mingming">Fig.4.Decision boundary</P><H3>Result</H3><P>The result of 3 fold cross-validation shows the accuracy of this model reaches up to  0.985. For coding sequence, we labeled the 8590 items after the preliminary screening with this model and get the result as follows:True:6404 False:2186
                        </P><P id="mingming">Fig.5.Prediction result</P><P>We compared this classifier with SVM(support vector machine) classifier, the 3-fold validation accuracy of SVM classifier is around 0.93, lower than the value of logistic regression classifier(about 0.98). We plotted the ROC(receiver operating characteristic) curve of this two classifier, the result is as follows: </P><P id="mingming">Fig.6.ROC curve</P><P>The AUC(Area Under roc Curve) value of logistic regression classifier is 0.99, while the number of SVM(support vector machine) is 0.94, which shows that the former one is better.</P><H1 id="prediction">Prediction Model</H1><H2 id="CNN">CNN</H2><P>Based on the LeNet-5 structure, we constructed a convolutional neural network to predict promoter. Through the movement of the convolution kernel on the input data, it will extract the most basic features. </P><P>The DNA sequence converted by the One-Hot Encoding input to the convolutional layer for feature extraction. After the internal weight calculation, the final result will be presented in the last two neurons, corresponding to the promoter and the non-promoter</P><P id="mingming">Fig.7.Use CNN to identify promoter in DNA sequence.</P><H2 id="Multi-Classifier System">Multi-Classifier System</H2><P>The feature of the promoter region is very different from other genomic regions such as exons, introns, 3'UTR and internal regions of the gene <B>[5]</B>. Therefore, when identifying eukaryotic promoters, we established three classifiers (promoter and exon, intron, 3UTR) to classify promoters at the same time.</P><P id="mingming">Fig.8.The nature of promoter is different from exon, intron, 3utr.</P><P>We put three CNN classifiers together by majority voting rules. A gene sequence \(x\) is simultaneously transferred into three classifiers to obtain the outputs of the three classifiers:   <SPAN>\(f^{r} (x),\ r=1,\ 2,\ 3\)</SPAN>. In order to determine if \(x\) is a promoter, we need to combine these outputs together and use majority voting rules <B>[6]</B>.</P><P id="gongshi"><SPAN>\(y=\begin{cases}
+1,\ if\displaystyle\sum_{r=1}^{3} \frac{f^{r} (x^r)+1}{2}\geq2\\
-1,\quad Otherwise\\
\end{cases}
\)</SPAN><SPAN style="position:relative; float:right;top:35px">(8)</SPAN></P><P>Where <SPAN>\(y\)</SPAN> is the simulated value of \(x\). If <SPAN>\(y=x+1\)</SPAN>, then \(x\) is a promoter, otherwise \(x\) is a non-promoter.</P><P id="mingming">Fig.9.Multiple Classifier System</P><H2 id="Global">Global Search</H2><P>We use a sliding window approach to identify promoter regions in large genome sequences. By moving a fixed-length window in a long sequence, every short sequence is identified to whether it is a promoter sequence. The window length of prokaryotes is 81 bp, and the window length of eukaryotes is 251 bp.</P><P id="mingming">Fig.10.Slide window is used for identify promoter in large scale sequence.</P><P>Due to the small difference between adjacent sequences, adjacent windows are likely to have the same classification result. Therefore, we put the region with more than 10 consecutive hits into the candidate set <SPAN>\(G={\{g_1,\ g_2,\ g_3,\ ...,\ g_n\}}\)</SPAN>, and select only one window with the highest score to report as the promoter region:</P><P id="gongshi"><SPAN>\(g=\displaystyle\max_{n} score(g_1,\ g_2,\ g_3,\ ...,\ g_n)\)</SPAN><SPAN style="position:relative; float:right;top:15px">(9)</SPAN></P><H2 id="Result">Result</H2><P>We tested the effect of our model with an independent validation set. In the E. coli Sigma70 promoter prediction, the sensitivity reached 89.1%, the specificity reached 95.2%, and the accuracy reached 92.2%. The performance in eukaryotes is as follows:</P><P id="mingming">Table.3.The performance compared with other method by Fickett &amp; Hatzigeorgiou's evaluation criterion.</P><P>Based on Fickett &amp; Hatzigeorgiou's datasets and evaluation criterion <B>[7]</B>, we evaluated the effect of our tool in human genes. Since our tool do not have strand specificity, we treated all approaches as “not strand specific”. <B>[8]</B> Its performance is better than most of the tools selected.</P><P id="mingming">Table.4.The performance in eukaryotic with independent validation sets.</P><H1 id="Reference">Reference</H1><P>[1] blast+2.7.1. Download from ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.7.1/</P><P>[2] eTutorials.org. Basic local alignment search tool (blast). Retrieved from  http://etutorials.org/Misc/blast/Part+III+Practice/Chapter+7.+A+BLAST+Statistics+Tutorial/7.1+Basic+BLAST+Statistics/</P><P>[3] scikit-learn.org. Supervised learning. Retrieved from http://scikit-learn.org/stable/supervised_learning.html#supervised-learning</P><P>[4] Fan,R; Chang,K. (2008). LIBLINEAR: A Library for Large Linear Classification</P><P>[5] Zeng, Jia, Xiao-Qin Cao, and Hong Yan. &quot;Human promoter recognition using Kullback-Leibler divergence.&quot; Machine Learning and Cybernetics, 2007 International Conference on. Vol. 6. IEEE, 2007.</P><P>[6] Xu, Wenxuan, Li Zhang, and Yaping Lu. &quot;SD-MSAEs: promoter recognition in human genome based on deep feature extraction.&quot; Journal of biomedical informatics 61 (2016): 55-62.</P><P>[7] Fickett, James W., and Artemis G.  Hatzigeorgiou. &quot;Eukaryotic promoter recognition.&quot; Genome research 7.9 (1997): 861-878.</P><P>[8] Scherf, Matthias, Andreas Klingenhoff, and Thomas Werner. &quot;Highly specific localization of promoter regions in large genomic sequences by PromoterInspector: a novel context analysis approach.&quot; Journal of molecular biology 297.3 (2000): 599-606.</P></DIV></DIV></DIV></DIV></SECTION><P><TITLE>Document</TITLE></P><DIV id="footer_main"><DIV class="container"><DIV class="row"><DIV class="col-lg-6 col-md-6 col-sm-6 col-xs-6 col-lg-offset_3 col-md-offset-3 col-sm-offset-3 col-xs-8 col-xs-offset-2"><H3 style="text-align:center;">Contact Us</H3><P style="text-align:center; font-size:18px; color:rgb(38,38,38)">uestcsoftware@vip.qq.com</P><P style="text-align:center; font-size:20px;color:rgb(38,38,38)">NO.2006.XiYuan Avenue,UESTC,Chengdu,China</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>