<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJUT-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJUT-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:ZJUT-China - 2018.igem.org</TITLE></P><DIV id="globalWrapper"><DIV id="top_menu_14" class="noprint">Loading menubar.....</DIV><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJUT-China</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="topbar"><DIV class="dropbkgd"><DIV class="dropposition"><DIV class="dropdown1"><DIV class="drop"><A href="https://2018.igem.org/Team:ZJUT-China">Home</A></DIV></DIV><DIV class="dropdown2"><DIV class="drop">
              Team
            <DIV class="dropdown-content"><A href="https://2018.igem.org/Team:ZJUT-China/Team">Team Members</A><A href="https://2018.igem.org/Team:ZJUT-China/Attributions">Attributions</A><A href="https://2018.igem.org/Team:ZJUT-China/Collaborations">Collaborations</A></DIV></DIV></DIV><DIV class="dropdown3"><DIV class="drop">
              Project
            <DIV class="dropdown-content"><A href="https://2018.igem.org/Team:ZJUT-China/Description">Description</A><A href="https://2018.igem.org/Team:ZJUT-China/Design">Design</A><A href="https://2018.igem.org/Team:ZJUT-China/Results">Results</A><A href="https://2018.igem.org/Team:ZJUT-China/Demonstrate">Demonstrate</A></DIV></DIV></DIV><DIV class="dropdown4"><DIV class="drop">
              Parts
            <DIV class="dropdown-content"><A href="https://2018.igem.org/Team:ZJUT-China/Parts">All parts</A><A href="https://2018.igem.org/Team:ZJUT-China/Improve">Improve</A></DIV></DIV></DIV><DIV class="dropdown5"><DIV class="drop">
            Lab
            <DIV class="dropdown-content"><A href="https://2018.igem.org/Team:ZJUT-China/Notebook">Notebook</A><A href="https://2018.igem.org/Team:ZJUT-China/Protocols2018">Protocols</A><A href="https://2018.igem.org/Team:ZJUT-China/Safety">Safety</A><A href="https://2018.igem.org/Team:ZJUT-China/InterLab">Interlab</A></DIV></DIV></DIV><DIV class="dropdown6"><DIV class="drop">
              Human Practices
            <DIV class="dropdown-content"><A href="https://2018.igem.org/Team:ZJUT-China/Human_Practices">Integrated hp</A><A href="https://2018.igem.org/Team:ZJUT-China/Public_Engagement">Public Engagement &amp; Education</A></DIV></DIV></DIV><DIV class="dropdown7"><DIV class="drop"><A href="https://2018.igem.org/Team:ZJUT-China/Model"> Modeling</A></DIV></DIV><DIV class="dropdown8"><DIV class="drop"><A href="https://2018.igem.org/Team:ZJUT-China/Achievement2018"> Judging</A></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="col-10"><H1>
      Overview
    </H1><P>
        The antimicrobial resistance is mostly caused by the expression of related antibiotic resistance genes(ARGs), which are widely being used as screening markers. But no effective method has been applied to ensure that ARGs would not be released into the environment yet. This situation poses a great threat to the society and is exactly the problem we want to solve.
    The goal of our project is to eliminate the ARGs carried by the plasmids of the bacteria themselves. The way of elimination is to cleave the target ARGs by the light-controlled expression of the Cas9 protein through the guidance of sgRNA. So we designed the following gene circuits:
    </P><P>
      Our goal is to eliminate the ARGs carried on the plasmids used in laboratories and fermentation plants. But it is difficult to build our system on this plasmid because our system is too huge. So weould need a second plasmid as a vector for the light-controlled cleavage system. In this case we plan to move the different ARGs on the different two plasmids, and achieve their occurrence in a certain order. To this end, we have improved the original design:
    </P><P>
      The sgRNA1 transcribed by the guide1 will guide Cas9 to cleave ARG1 on the plasmid1, and the sgRNA2 transcribed by the guide2 will guide Cas9 to cleave ARG2 on Plasmid 2. When ARG1 on Plasmid 1 is not cleaved, the plasmid is intact, and the normally expressed Repressor will repress the transcription of the guide2. when ARG1 is cleaved, Repressor cannot be expressed normally, resulting in the transcription of the guide2. This leads to ARG2 degradation by Cas9 on the plasmid2. Our design allows the two plasmids to be cleaved with a certain order to ensure self-destruction after elimination of the target gene.
    </P><P>
      What’s more, in our final design, cells end up with a inducible-lysis after ARGs were cut. To achieve this we designed the following gene circuit for lysis part and it will be added to the genome using CRISPR/Cas9 genome editing tool.
    </P><DIV><DIV class="btn"><BUTTON type="button" name="button" id="btn1">Light-controlled system</BUTTON></DIV><DIV class="btn"><BUTTON type="button" name="button" id="btn21">sgRNA-cm</BUTTON></DIV><DIV class="btn"><BUTTON type="button" name="button" id="btn31">P<SUB>BAD</SUB>-Cas9</BUTTON></DIV><DIV class="btn"><BUTTON type="button" name="button" id="btn41">Light-controlled Cas9 system</BUTTON></DIV><DIV class="btn"><BUTTON type="button" name="button" id="btn51">Repression system</BUTTON></DIV><DIV class="btn"><BUTTON type="button" name="button" id="btn61">Lysis</BUTTON></DIV><DIV id="div1"><H1>Light-controlled system</H1><H2>1.Primary light control system</H2><H2>1.1.Introduction of primary light control system</H2><P>
            The primary light control system is shown in figure 1. Under dark condition, the phosphate group is transferred from Yf1 protein to FixJ protein, the phosphorylated FixJ protein activates the P<SUB><I>fixK2</I></SUB> promoter and then activates the downstream gene expression. When induced by light, the phosphorylation of Fixj protein will be blocked, and the expression of genes regulated by P<SUB><I>fixK2</I></SUB> promoter will be inhibited.
          </P><DIV class="note">Figure 1. primary light control system</DIV><H2>1.2.Experimental design</H2><P>
            To further verify the effectiveness of the system, we plan to transfer plasmids (dusk-eGFP-pUC57) with the system to different hosts. Cultured under blue and dark conditions after a period of time, then detected the fluorescence value of eGFP. The efficiency of primary light control system in different hosts was compared.
          </P><H2>1.3.Experimental method</H2><H2>1.3.1.Extraction of dusk-eGFP-pUC57 plasmid</H2><P>Because the 2017 ZJUT team had already constructed the dusk-eGFP-pUC57 plasmid,we used a shaking flask containing 50ml LB liquid medium to incubate bacteria carrying the plsmid overnight, and then extracted the plasmids using the DNA extraction kit. See the specification of the kit for specific operation.
          </P><H2>1.3.2.Preparation of competent cells</H2><P>
            After discussion, we decided to select DH5α, BL21, MG1655 and BW25113 as the hosts of the primary light control system. When the host cells were cultured to a OD<SUB>600</SUB> value of about 0.5, the effect of preparing the competent cells was the best.
          </P><H2>
            1.3.3.Transformation
          </H2><P>The transformation was carried out at 42℃. The transformation solution was coated on the LB solid medium containing Amp and cultured overnight at 37℃. Observe the number of transformants.
          </P><H2>
            1.3.4.Primary light control system effect detection
          </H2><P>The successful transformants and their corresponding hosts without primary light control system were cultured under blue light and dark light respectively. The expression of eGFP was detected every other time.
          </P><H2>2.Improvement of primary light control system</H2><P>
            After being cultured at 37℃ for 24 to 48 hours in darkness, the expression of eGFP protein is not very high, showing our light control system lacks efficiency. So we wanted to improve the light control system and had a discussion with the team members. We decided to replace the J23100 promoter with the T7 promoter.
          </P><H2>2.1.Replacement of a strong promoter</H2><P>
            We planned to replace the J23100 promoter with the T7 promoter to reconstruct the light control system (Fig.3), and determine the relationship between the culture time and the expression of eGFP protein. Different from the primary light control system, the T7 promoter of the secondary light controlled system is regulated by lactose analogues. We planed to use IPTG to regulate the T7 promoter to regulate the efficiency of the light control system.
          </P><DIV class="note">
            Figure 3. secondary light controlled system
          </DIV><H2>2.2.Experimental design</H2><P>
            We plan to obtain the T7 promoter from the pET28b plasmid, delete the PJ23100 promoter from the dusk-eGFP-pUC57 plasmid, construct the T7-dusk-eGFP-pUC57 plasmid by one step clone method, transfer it into the BL21 host and induce it by adding different amount of IPTG. The expression of eGFP was detected by culturing under blue light and dark light for a period of time.
          </P><H2>2.3.Experimental method</H2><H2>2.3.1.Acquisition of T7 Promoter</H2><P>
            We obtained the T7 promoter from the pET28b plasmid by PCR. The primers and PCR program are as follows:
            pET28b-F：5’-ttgtactgagagtgcaccattaagtgcggcgacgatag-3’
            pET28b-R：5’-agtagctagcactgtacctggaattgttatccgctcac-3’
          </P><TABLE border="1"><TBODY><TR><TH>PCR</TH><TH>pET28b</TH></TR><TR><TH>2×Phanta Max Master Mix</TH><TH>25μl</TH></TR><TR><TH>10mM pET28b-F</TH><TH>2μl</TH></TR><TR><TH>10mM pET28b-R</TH><TH>2μl</TH></TR><TR><TH>pET28b plasmid DNA</TH><TH>20~30ng</TH></TR><TR><TH>ddH<SUB>2</SUB>O</TH><TH>to 50μl</TH></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>PCR condition setting</TH><TH>pET28b</TH></TR><TR><TH>Step one</TH><TH>95℃ 180s</TH></TR><TR><TH>Step two</TH><TH>95℃ 15s</TH><TH>25 cycles</TH></TR><TR><TH>Step three</TH><TH>59℃ 15s</TH><TH>25 cycles</TH></TR><TR><TH>tep four</TH><TH>72℃ 180s</TH><TH>25 cycles</TH></TR><TR><TH>Step five</TH><TH>72℃ 300s</TH></TR></TBODY></TABLE><P>
            PCR results were detected by electrophoresis gel.
          </P><H2>2.3.2.Delete J23100 promoter from dusk-eGFP-pUC57 plasmid</H2><P>
            We deleted the J23100 promoter from the dusk-eGFP-pUC57 plasmid by PCR. The primers and the PCR program were designed as follows:
            pUC57-F：5’-tgtgagcggataacaattccaggtacagtgctagctact-3’
            pUC57-R：5’-gactatcgtcgccgcacttaatggtgcactctcagtac-3’
          </P><TABLE border="1"><TBODY><TR><TH>PCR</TH><TH>dusk-eGFP-pUC57</TH></TR><TR><TH>2×Phanta Max Master Mix</TH><TH>25μl</TH></TR><TR><TH>10mM pUC57-F</TH><TH>2μl</TH></TR><TR><TH>10mM pUC57-R</TH><TH>2μl</TH></TR><TR><TH>pUC57-dusk-eGFP plasmid DNA</TH><TH>20~30ng</TH></TR><TR><TH>ddH<SUB>2</SUB>O</TH><TH>to 50μl</TH></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>PCR condition setting</TH><TH>pET28b</TH></TR><TR><TH>Step one</TH><TH>95℃ 180s</TH></TR><TR><TH>Step two</TH><TH>95℃ 15s</TH><TH>25 cycles</TH></TR><TR><TH>Step three</TH><TH>59℃ 15s</TH><TH>25 cycles</TH></TR><TR><TH>tep four</TH><TH>72℃ 360s</TH><TH>25 cycles</TH></TR><TR><TH>Step five</TH><TH>72℃ 300s</TH></TR></TBODY></TABLE><P>
            PCR results were detected by electrophoresis gel.
          </P><H2>
            2.3.3.Template elimination
          </H2><P>
            PCR products need to be treated with Dpn Ⅰ enzyme to remove template plasmids on subsequent experiments. The reaction system is as follows:
          </P><P>
            Result of template elimination were detected by electrophoresis gel.
          </P><H2>
            2.3.3.One-step cloning and transformation
          </H2><P>
            The insertion fragments and linearized carriers were purified by purification kit, then connected by one-step cloning kit, and then transferred into BL21 competent cells for overnight culture at 37 ℃. The number of transformants were counted. The reaction system of one-step cloning is as follows:
          </P><H2>2.3.4.Secondary light control system effect detection</H2><P>The successful transformants were added to different amounts of IPTG, and cultured under blue light and dark conditions for 24 hours to detect the expression of eGFP.
          </P><DIV class="note1">
            References：
          [1] Wang G, Lu X, Zhu Y, et al. A light-controlled cell lysis system in bacteria.[J]. Journal of Industrial Microbiology &amp; Biotechnology, 2018:1-4.
          [2] Wu H, Wang Y, Wang Y, et al. Quantitatively relating gene expression to light intensity via the serial connection of blue light sensor and CRISPRi[J]. Acs Synthetic Biology, 2014, 3(12):979.
          [3] Gardner L, Deiters A. Light-Controlled Synthetic Gene Circuits[J]. Current Opinion in Chemical Biology, 2012, 16(3-4):292-299.
          </DIV><BUTTON type="button" name="button" id="btn2">Back</BUTTON></DIV><DIV id="div21"><H1>Constructing a plasmid expressing sgRNA which target chloramphenicol resisitance gene</H1><H2>▶First step: Test CRISPR/Cas9 system</H2><H2>Overview</H2><P>
            We constructed a CRISPR/Cas9 system for degrading ARGs. In this system, plasmid pTargetF expresses sgRNA, to guide Cas9 protein to cleave target gene. Firstly, We tested the CRISPR/Cas9 system. We selected two different kinds of <I>E. coli</I>.The one is wild type, the other is a <I>panD</I> mutant. We transferred pCas plasmid and pTargetF plasmid into the two strains. We observed that the wild type died and the <I>panD</I> mutant grew. The results proved that the CRISPR/Cas9 system was effective.
          </P><H2>
            Functional description
          </H2><P>
            The plasmid pTargetF-<I>panD</I> expresses sgRNA which targets on the <I>panD</I>, which guides Cas9 to the <I>panD</I> gene and cause double-strand DNA break in the genome of <I>E. coli</I> MG1655.
          </P><H2>Functional verification</H2><P>
            The strain we used is <I>E. coli</I> MG1655 Δ<I>panD</I> and <I>E. coli</I> MG1655 wild type. Firstly, we need to prepare <I>E. coli</I> MG1655 Δ<I>panD</I> and <I>E. coli</I> MG1655 wild type competent cells and transform pCas plasmid into it. Then, we need to transform pTargetF-<I>panD</I> plasmid into the above transformants. Finally, the plates were incubated at 30℃ overnight.
          </P><P> pTargetF-<I>panD</I> can guide Cas9 to the <I>panD</I> gene and cause double-strand DNA break in the genome of <I>E. coli</I> MG1655 wide type, while pTargetF-<I>panD</I> can not guide Cas9 to any place in the genome of <I>E. coli</I> MG1655 Δ<I>panD</I>. Transformants of <I>E. coli</I> MG1655 Δ<I>panD</I> grew, Transformants of <I>E. coli</I> MG1655 wild type died.
          </P><H2>▶Second step: Construct pTargetF-cm</H2><H2>Overview</H2><P>
            In order to achieve degradation of ARGs, we have been trying to construct a resistant cutting system by using CRISPR/Cas9 system. In this system, plasmid pTargetF expresses sgRNA, to guide Cas9 protein to the target gene. Secondly, We have been construct pTargetF-cm, which can express sgRNA which targets chloramphenicol resistance gene. And sequencing results showed successful construction.
          </P><H2>
            Functional description
          </H2><P>The plasmid pTargetF-cm can expresses sgRNA which targets on the chloramphenicol resistance gene. </P><H2>Functional verification</H2><P> After PCR, we got the plasmid pTargetF-cm, then transformed it into <I>E. coli</I> DH5α competent cells.The construction was confirmed by sequancing.
          </P><H2>▶Third step: Test pTargetF-cm</H2><H2>Overview</H2><P>In order to achieve degradation of ARGs, we have been trying to construct a resistant cutting system by using CRISPR/Cas9 system. In this system, plasmid pTargetF expresses sgRNA, for guiding Cas9 protein to cleavage target gene. Thirdly, we tested the function of the plasmid pTargetF-cm.We introduced pSU20 which has a chloramphenicol resistance gene and pCas which has a kanamycin resistance gene into <I>E. coli</I> MG1655.
          Finally, the transformant can grow in LB supplemented with kanamycin and spectinomycin, while it can not grow in LB supplemented with chloramphenicol. The result showed that pTargetF-cm worked successfully.
          </P><H2>Functional description</H2><P>
            The plasmid pTargetF-cm expresses sgRNA which targets on the chloramphenicol in <I>E. coli</I> MG1655-pSU20. Therefore, pTargetF-cm can guide Cas9 to the chloramphenicol resistance gene and cause double-strand DNA break in the pSU20 plasmid.
          </P><H2>Functional verification</H2><P>The strain we used is <I>E. coli</I> MG1655 wild type. First, we need to prepare <I>E. coli</I> MG1655 wild type competent cells and transform pSU20 plasmid into it. Then, we need to culture the transformants and prepare them into competent cells, and then transform pCas plasmid. Then, we need to culture the transformants and prepare them into competent cells, and then transform pTargetF-cm plasmid, respectively.
          </P><P>pTargetF-cm can guide Cas9 to the chloramphenicol resistance gene and cause double-strand DNA break in the pSU20 plasmid. After grown at 30℃ overnight,After incubation at 30 C overnight, <I>E. coli</I> MG1655 wild type was able to grow in LB supplemented with kanamycin and spectinomycin but failed to grow in the culture supplemented with chloramphenicol because the corresponding resistance gene in pSU20 had been eliminated.

          </P><DIV class="note1">
            References：
            [1] Jiang W, Bikard D, Cox D, et al. CRISPR-assisted editing of bacterial genomes[J]. Nature Biotechnology, 2013, 31(3):233-239. 
            [2] Citorik R J, Mimee M, Lu T K. Sequence-specific antimicrobials using efficiently delivered RNA-guided nucleases[J]. Nature Biotechnology, 2014, 32(11):1141-1145. 
            [3] Didovyk A, Borek B, Hasty J, et al. Orthogonal Modular Gene Repression in Escherichia coli Using Engineered CRISPR/Cas9[J]. Acs Synthetic Biology, 2016, 5(1):81-88. 
            [4] Doench J G, Fusi N, Sullender M, et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9[J]. Nature Biotechnology, 2016, 34(2):184-191.
          </DIV><BUTTON type="button" name="button" id="btn22">Back</BUTTON></DIV><DIV id="div31"><H1>P<SUB>BAD</SUB>-Cas9</H1><P>Before we used the light-controlled system to regulate the expression of Cas9, we first used P<SUB>BAD</SUB> to regulate the expression of Cas9 to verify the cleavage function of Cas9.
            </P><P>
              The arabinose-induced promoter offers the possibility to regulate the expression of Cas9 protein in <I>E. coli</I> by adding or not adding a certain amount of arabinose into the culture.
            </P><H2>Experiment design:</H2><P>
              We built this part on pGLO and named it pGLO-Cas9. To verify whether the expressed Cas9 protein has the function of cutting the target gene, we used <I>E. coli</I> MG1655 as a chassis. In addition, we will use the pTargetF plasmid to transcribe the sgRNA that targets the <I>panD</I> gene. Since the original pTargetF is not compatible with pGLO, we have built a pTargetF-p15A that is compatible with pGLO.
               Finally, we characterized the function of the Cas9 gene based on the growth curve of the bacteria.
              When the function of Cas9 was verified, we replaced the sgRNA targeting <I>panD</I> with the sgRNA targeting the chloramphenicol gene. The plasmid we constructed was named pTargetF-p15A-cm. And we used <I>E. coli</I> MG1655 containing the chloramphenicol gene on genome for validation experiments.
              </P><H2>The experiment is divide into 5 steps:</H2><P>
                the regulation of an inducible promoter. This is based on a given plasmid pTargetF, we changed its replicon to make it compatible with P<SUB>BAD</SUB>-Cas9.
                  1) Construct the P<SUB>BAD</SUB>-Cas9 plasmid based on a given plasmid pGLO.
                  2) Construct a plasmid pTargetF-p15A to express sgRNA to test if the Cas9 protein is function under the regulation of an inducible promoter. This is based on a given plasmid pTargetF, we changed its replicon to make it compatible with P<SUB>BAD</SUB>-Cas9.
                 3) Transform both plasmid to <I>E. coli</I> MG1655
                4) Get the growth curve result to prove after adding arabinose, the Cas9 protein will express and perform its function.
               5) Construct and test a plasmid pTargetF-p15A-cm to perform ARGs cut.
               
                *<I>panD</I> gene is a gene on the genome of <I>E. coli</I>, in the previous research of our lab, we obtained a <I>panD</I>-deficient strain of <I>E. coli.</I></P><DIV class="note">
              References：
              [1] Jiang W, Bikard D, Cox D, et al. CRISPR-assisted editing of bacterial genomes[J]. Nature Biotechnology, 2013, 31(3):233-239. 
              [2] Citorik R J, Mimee M, Lu T K. Sequence-specific antimicrobials using efficiently delivered RNA-guided nucleases[J]. Nature Biotechnology, 2014, 32(11):1141-1145. 
              [3] Didovyk A, Borek B, Hasty J, et al. Orthogonal Modular Gene Repression in Escherichia coli Using Engineered CRISPR/Cas9[J]. Acs Synthetic Biology, 2016, 5(1):81-88. 
              [4] Doench J G, Fusi N, Sullender M, et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9[J]. Nature Biotechnology, 2016, 34(2):184-191.
            </DIV><BUTTON type="button" name="button" id="btn32">Back</BUTTON></DIV><DIV id="div41"><H1>Light-controlled Cas9 system</H1><H2> 1.1Functional description</H2><P>
            Under the dark condition, YF1 phosphorylates FixJ and phospholylated FixJ activates the P<SUB><I>fixK2</I></SUB> promoter, which activates the expression of Cas9.Under the blue light condition YF1 don’t phosphorylate FixJ. Therefore P<SUB><I>fixK2</I></SUB> promoter can not initiate the expression of Cas9.
          </P><P>The plasmid pTargetF-cm expresses sgRNA which targets on the chloramphenicol resistance gene . In a <I>panD</I> mutant <I>E. coli</I> Bl21+Δ<I>panD</I>, the original <I>panD</I> gene was replaced by the chloramphenicol resistance gene in the genome. Therefore, pTargetF-cm can guide Cas9 to the chloramphenicol gene and cause double-strand DNA break in the genome of <I>E. coli</I> Bl21+Δ<I>panD</I>.
          </P><P>Under dark conditions, <I>cas9</I> gene can be expressed, so pTargetF-cm can guide Cas9 to the chloramphenicol resistance gene and cause double-strand DNA break in the genome.However, in blue light,the expression of Cas9 is inhibited, double-strand DNA will not be cut.
          </P><DIV class="note">Figure 1. Light-controlled CRISPR/Cas9 system</DIV><H2>1.2Experimental design</H2><H2>1.2.1Design of plasmid needed </H2><P>
            In this experiment, we need dusk-Cas9-pUC57 plasmid,pTargetF-cm plasmid, pSU20 plasmid and pTargetF-<I>panD</I> plasmid. We used dusk-eGFP-pUC57 plasmid and pTargetF-<I>panD</I> plasmid to construct dusk-Cas9-pUC57 plasmid and pTargetF-cm plasmid. For detailed experimental procedure,
             please refer to our notebook.
              <A href="https://static.igem.org/mediawiki/2018/c/c6/T--ZJUT-China--mao714.pdf">7.14</A><A href="https://static.igem.org/mediawiki/2018/e/eb/T--ZJUT-China--mao715.pdf">7.15</A><A href="https://static.igem.org/mediawiki/2018/4/41/T--ZJUT-China--mao716.pdf">7.16</A></P><H2>
            1.2.2Functional verification
          </H2><P>
            The strain we used is <I>E. coli</I> BL21 Δ<I>panD</I>, which is a <I>panD</I> mutant, the original <I>panD</I> gene was replaced by the chloramphenicol resistance gene in the genome(cm). First, we need to prepare <I>E. coli</I> BL21 Δ<I>panD</I> competent cells and transform dusk-Cas9-pUC57 plasmid into it.Then, we transformed pTargetF-cm plasmid, pSU20 plasmid (as control) and pTargetF-<I>panD</I> plasmid (as control) into <I>E. coli</I> BL21 Δ<I>panD</I> dusk-Cas9-pUC57 separately. The transformants were incubated overnight under dark or blue light conditions.
          </P><P>  Under dark conditions, pTargetF-cm can guide Cas9 to the chloramphenicol gene and cause double-strand DNA break in the genome of <I>E. coli</I> Bl21 Δ<I>panD</I>, while pTargetF-<I>panD</I> can not guide Cas9 to any place in the genome of <I>E. coli</I> Bl21 Δ<I>panD</I>. The damage to the bacterial genome due to CRISPR/Cas9 can be determined by transformation efficiency. When light was provided, the light-controlled gene expression system was suppressed and therefore the transformation efficiency was higher.
          </P><DIV class="note">Table 1.Transformation experiments in each group </DIV><DIV class="note">Competent cells were prepared from BL21 ΔpanD containing plasmid dusk-Cas9-pUC57.</DIV><DIV class="note1">
              References：
[1] Wang G, Lu X, Zhu Y, et al. A light-controlled cell lysis system in bacteria.[J]. Journal of Industrial Microbiology &amp; Biotechnology, 2018:1-4. 
[2] Wu H, Wang Y, Wang Y, et al. Quantitatively relating gene expression to light intensity via the serial connection of blue light sensor and CRISPRi[J]. Acs Synthetic Biology, 2014, 3(12):979. 
[3] Citorik R J, Mimee M, Lu T K. Sequence-specific antimicrobials using efficiently delivered RNA-guided nucleases[J]. Nature Biotechnology, 2014, 32(11):1141-1145. 
[4] Jiang W, Bikard D, Cox D, et al. CRISPR-assisted editing of bacterial genomes[J]. Nature Biotechnology, 2013, 31(3):233-239.
            </DIV><BUTTON type="button" name="button" id="btn42">Back</BUTTON></DIV><DIV id="div51"><H1>Repression system</H1><H2>▶First step</H2><H2>Overview</H2><P>
            In order to achieve the sequential degradation of ARGs, we have been trying to construct a repression system. In this system, the arabinose-controlled repressor protein was utilized to regulate the transcription of sgRNA. We chose two types of inhibitor-promoter: <I>lacI</I>-P<SUB><I>lac</I></SUB>, <I>cI</I>-P<SUB>R</SUB> to test just in case. The LacI repressor is a DNA-binding protein which can bind to the <I>lac</I> operator to inhibit transcription in <I>E. coli</I>. This inhibition can be relieved by adding lactose or isopropyl-beta-D-thiogalactopyranoside (IPTG). The P<SUB>R</SUB> promoter has two DNA binding sites for CI repressor. And transcription would be repressed when CI protein binds to the sites. After experiment, we selected the <I>lacI</I>-P<SUB><I>lac</I></SUB> to make up the repression system due to the results.
          </P><H2>Design</H2><H2>(1)<I>lacI</I>-P<SUB><I>lac</I></SUB></H2><P>The genetic circuit is shown in Fig.1.</P><DIV class="note"> Fig.1 The genetic circuit of lacI-P<SUB><I>lac</I></SUB></DIV><P>
              This part is mainly composed of three elements: 
            </P><P>①Arabinose induced promoter P<SUB>BAD</SUB> to express the gene of downstream. This part contains the promoter as well as the coding sequence for the repressor AraC which is transcribed in the opposite direction.By binding to L (+)-arabinose, AraC changes its conformation and induce the transcription of downstream genes.
            </P><P>②This part contains the <I>lac</I> operator as well as the coding sequence for the repressor lacI. The lacI repressor binds to the <I>lac</I> operator to inhibit transcription in <I>E. coli</I>. This inhibition can be relieved by adding lactose or isopropyl -beta -D-thiogalactopyranoside (IPTG).
            </P><P>③GFP, as the report gene, can show the degree of repression by repressor.
            </P><H2>（2）cI-P<SUB>R</SUB></H2><P>The genetic circuit is shown in Fig.2.</P><DIV class="note"> Fig.2 The genetic circuit of cI-P<SUB>R</SUB></DIV><P>      ①Arabinose induced promoter P<SUB>BAD</SUB> to express the gene of downstream.
          </P><P>  ②This part contains the P<SUB>R</SUB> promoter as well as the coding sequence for the repressor cI. The cI repressor binds to the operator to inhibit transcription in <I>E. coli</I>.
          </P><P> ③GFP, as the report gene, can verify the degree of repression by repressor cI .
          </P><H2>Description</H2><P>Different concentrations of arabinose will induce these parts with different concentration of repressor proteins. By adding different concentration of arabinose, we can measure the fluorescence intensity of GFP and find out the relationship between the concentration of arabinose and RFU (Relative fluorescence unit) intensity. And by adding IPTG，the LacI protein is able to be released from the <I>lac</I> operator .It is a characteristic that can be utilized to further verify the function of the lacI-P<SUB><I>lac</I></SUB> by adding IPTG or not.
          </P><H2>▶Second step</H2><H2>Overview</H2><P>
            Based on the results we have achieved, we chose <I>lacI</I>-P<SUB><I>lac</I></SUB> to carry out the next step. In this system, the arabinose-controlled repressor protein is utilized to regulate the transcription of sgRNA.
          </P><P> Single-guide RNA (sgRNA) is an artificial RNA which is designed to bind a certain DNA sequence. It contains the 20-bp complementary region (N20) with the requisite NGG PAM matching genomic loci of interest and the sequence of tracrRNA. It will guide Cas9 to the target. We design the sgRNA which specifically targets the gene <I>panD</I> on the <I>E. coli</I> genome in order to demonstrate our CRISPR/Cas9 system.
          </P><H2>●lacI-P<SUB><I>lac</I></SUB>-sgRNA</H2><P>The genetic circuit is shown in Fig.3.</P><DIV class="note"> Fig.3 The genetic circuit of <I>lacI</I>-P<I>lac</I>-sgRNA. guide is the gene will be transcribed to sgRNA which can guide the Cas9 protein to target gene. </DIV><P>
            We combined lacI repression system (BBa_K2556031) which is controlled by the AraC-Ara promoter (BBa_K808000) with sgRNA (BBa_K2556041) so that we can regular the transcription of sgRNA. It’s important to activate the CRISPR/Cas9 system only under certain proper conditions.
          </P><H2>Description</H2><P>
            When activating the AraC-Ara promoter by adding L (+)-arabinose, the LacI repressor will express and the transcription of sgRNA will be inhibited. But this inhibition can be relieved after induction with lactose or isopropyl-beta-D-thiogalactoside (IPTG). So when the inhibition is relieved, the sgRNA will transcribe successfully and combine with Cas9 to cut the target gene <I>panD</I>. That means the <I>E. coli</I> won’t survive. We can demonstrate the difference of the transcriptional level of sgRNA by analyzing the growth curve in four different conditions (① +Ara +IPTG ② +Ara -IPTG ③ -Ara +IPTG  ④ -Ara -IPTG)
          </P><DIV class="note1">
            References：
            [1] Deltcheva E, Chylinski K, Sharma C M, et al. CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III[J]. Nature, 2011, 471(7340):602-607. 
            [2] Citorik R J, Mimee M, Lu T K. Sequence-specific antimicrobials using efficiently delivered RNA-guided nucleases[J]. Nature Biotechnology, 2014, 32(11):1141-1145.
          </DIV><BUTTON type="button" name="button" id="btn52">Back</BUTTON></DIV><DIV id="div61"><H1>Lysis</H1><P>The aim of our project is cutting ARGs in engineered bacteria and reduce ARGs. To achieve this goal, we used the part <A href="http://parts.igem.org/Part:BBa_K2277000">BBa_K2277000</A>, which had been constructed by 2017 iGEM team ZJUT-China. But in most cases, ARGs have been added to plasmids in order to screen transformants. So we integrated the <I>lysin</I> gene directly into the genome. Firstly, it can reduce the number of plasmids that need to be transformed in the engineering bacteria, which can reduce the additional metabolic pressure. Secondly, it can avoid the addition of new resistance.
            </P><P>The arabinose-induced promoter offers the possibility to regulate the expression of lysin protein in <I>E. coli</I> by adding or not adding a certain amount of arabinose into the culture, therefore control the lysin system.</P><H2>Experiment design:</H2><P>
              We first built this part on plasmid T vector. To verify whether the expressed lysin protein has the function of cell lysis, we used <I>E. coli</I> MG1655 as a chassis. we transformed this plasmid into MG1655 and characterized the function of the <I>lysin</I> gene based on the growth curve of the bacteria. 
            </P><P>When the function of lysin protein was verified, we started to construct this part in the genome of <I>E.coil</I> MG1655 by using CRISPR/Cas9 system. In this system, pTargetF plasmid will express sgRNA which guide Cas9 protein cut target gene, causing double-strand DNA break. At this point, the donor DNA with the homologous arms will be inserted into the cutting site by homologous recombination. We designed the pTargetF plasmid which can transcribe the sgRNA targeting at specified site in the genome. We also constructed the donor DNA, which is composed of AraC-P<SUB>BAD</SUB>-lysis and two homologous arms of the genome of MG1655. After construction, we characterized the function of the <I>lysin</I> gene based on cell growth on plates and in liquid culture. 
            </P><H2>The experiment is divide into 4 steps:</H2><P>1.Construct T-MG1655 plasmid (contain 2000bp homologous arm of genome) by TA Cloning</P><P>2. Construct T-MG1655-Lysis plasmid by one-step cloning</P><P>3. Construct pTargetF-Lysis plasmid targeting <I>E. coli</I> MG1655 genome</P><P>4. Transform pCas and pTargetF-Lysis and fragment of AraC-P<SUB>BAD</SUB>-Lysin into <I>E. coli</I>MG1655 to construct th desired recombinant strain</P><P>You can gain more details from our <A href="https://2018.igem.org/Team:ZJUT-China/Notebook">notebook</A></P><DIV class="note1">
              References：
      [1] Jiang W, Bikard D, Cox D, et al. CRISPR-assisted editing of bacterial genomes[J]. Nature Biotechnology, 2013, 31(3):233-239. 
      [2] Doench J G, Fusi N, Sullender M, et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9[J]. Nature Biotechnology, 2016, 34(2):184-191. 
      [3] Wang G, Lu X, Zhu Y, et al. A light-controlled cell lysis system in bacteria.[J]. Journal of Industrial Microbiology &amp; Biotechnology, 2018:1-4. 
      [4] Park T, Struck D K, Dankenbring C A, et al. The Pinholin of Lambdoid Phage 21: Control of Lysis by Membrane Depolarization[J]. Journal of Bacteriology, 2007, 189(24):9135.
            </DIV><BUTTON type="button" name="button" id="btn62">Back</BUTTON></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="foottop"><DIV class="footmessage-right"><H3>Contact</H3><P>
        College of Biotechnology and Bioengineering
        Zhejiang University of Technology
        Chaowang Rd. 18, 310014, Hangzhou, China
        E-mail: cbb@zjut.edu.cn Tel: +86-571-88320391
        </P></DIV></DIV><DIV class="footbottom"><DIV class="note">
      Copyright © 2018 ZJUT-China
      </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>