Team:ZJUT-China/Results
CRISPR
Team:ZJUT-China - 2018.igem.org
Loading menubar.....
Team:ZJUT-China
Home
Team
Team Members Attributions Collaborations
Project
Description Design Results Demonstrate
Parts
All parts Improve
Lab
Notebook Protocols Safety Interlab
Human Practices
Integrated hp Public Engagement & Education
Modeling
Judging
Light-controlled system
sgRNA-cm
PBAD-Cas9
Light-controlled Cas9 system
Repression system
Lysis
Light-controlled system
3.Experimental results
3.1.Comparison of the efficiency of the light-controlled gene expression system in different host E. coli strains
Figure 5. Light-controlled gene expression system in DH5α
Figure 6. Light-controlled gene expression system in BL21
Figure 7. Light-controlled gene expression system in MG1655
Figure 8. Light-controlled gene expression system in BW25113
3.2.Replacing the promoter PJ23100 with T7
Figure 9. PCR amplification of T7 promoter and linearization of the vector for expressing the light-controlled system
3.3.Elimination of the template plasmid with DpnI
Fig. 10 Degrading the template plasmid in the PCR product
3.4.The results of T7-involved light-controlled system effect detection
Figure 11. Effect of IPTG on T7 regulated light-controlled system
Back
sgRNA-cm
Test CRISPR/Cas9 system
We evaluated a CRISPR/Cas9 system with MG1655 and MG1655 ΔpanD. The two strains were transformed with pCas, which expresses Cas9. The sgRNA which targets to a site in the panD gene was expressed in a plasmid pTargetF-panD. The pTargetF-panD was able to transform MG1655 ΔpanD pCas9, but not MG1655 pCas9, indicating that CRISPR/Cas9 was able to recognize and cut DNA in the chromosomal gene.
Table 1. Transformants culture result
Our target:
GTCCTAGGTATAATACTAGTGGCAATGAAAGACGGTGAGCGTTTTAGAGCTAGAAATAGC
sequencing results :
Sequencing results showed successful construction.
Test pTargetF-cm
To test whether the CRISPR/Cas9 can cut an ARG, we designed an sgRNA which targeted to the chloramphenicol resistance gene. The sgRNA was expressed in pTargetF-cm. which was transformed into MG1655 ΔpanD carrying a chloramphenicol resistance gene. We observed that all transformants grew in the medium without chloramphenicol, but failed to grow in the medium with chloramphenicol. The experiment showed that the CRISPR/Cas9 system was able to cut ARG in the genome.
Figure 1. The result of culture
Back
PBAD-Cas9
Fig.1 Growth curve, add 10mM ara after 3 hours of culture
Fig.2 Growth curve, add ara after 5 hours of culture
In our experiment we required transformation data and two growth curve in the end. Fig.1 shows that after adding arabinose, E. coli MG1655 with PBAD-Cas9 plasmid stopped growing. We used OD600 to characterize the cells amount so to proving that the expression of Cas9 protein can be regulated by 10mM arabinose. Fig.2 shows how different concentrations of arabinose affects the Cas9 expression.
In the chloramphenicol gene cleavage experiment, we used a plate containing ara for transformation experiments, and the cleavage of the chloramphenicol gene could be characterized by transformation efficiency. The results are showed in Table 1:
Our results show that inducing the expression of Cas9 using ara has no effect on the transformation efficiency of wild type, and the transformation efficiency of E. coli cmR is greatly reduced. Therefore, we can conclude that the transformation efficiency is reduced because Cas9 cleaves the chloramphenicol gene on the genome under the guidance of sgRNA-cm. Moreover, when we did not add Ara, the transformation efficiency of E. coli cmR was also lower than that of E. coli wild type. This suggests that Cas9 exhibits high gene cleavage efficiency in bacteria because PBAD is a well-regulated promoter.
Back
Light-controlled Cas9 system
Under light and dark conditions, the transformation efficiencies with pTargetF-panD were 1.185 and 1.295 folds of thoese with pTargetF-cm respectively, demonstrating that pTargetF-cm could guide Cas9 to the cm gene on the genome and resulted in the decrease of transformation efficiency. The result also reflects that the blue light provided cannot completely suppress the expression of CRISPR/Cas9. It is also remarkable that the transformation efficiency with pTargetF-cm under the dark condition was lower than that under the light condition, indicating that the CRISPR/Cas9 system showed stronger activity under dark condition, achieving the purpose of cutting a resistance gene with the light-controlled CRISPR/Cas9 system.
Figure1. Transformation of the plasmid carrying the light-controlled CRISPR/Cas9 system under dark and light conditions
Back
Repression system
▶First step
Aim
We’d like to achieve the following purposes:
1、Construct a part with repression system.
2、Maximize the repression effect.
3、Select a more suitable system through comparison.
Experimental details
●lacI-Plac
（1）Obtain Objective strain
For testing this device we used DH5α cells which contain the plasmid named pGLO-lacI. This plasmid contains the parts (BBa_K2556032) we wanted to test. The steps are as follow:
①pCAS was cultured in the test tube with liquid LB medium. After 12 hours, we extracted the plasmid DNA and measured the concentration.
②We prepared the insertion fragment and the linearized carrier（pGLO）by PCR. Then the PCR products were de-templated by DpnⅠ, then purified by TaKaRa kit to prepare for ligation.
③Construct plasmids named pGLO-lacI by One step cloning
④Transform the pGLO-lacI into E. coli DH5α by Heat Shock and 10-100μL plated on antibiotic selective media(AMP).
⑤Colony PCR and ideal colonies were selected, sequenced by company. We preserved the strain and began to test the effectiveness of the repression system.
Verify the function of repression of lacI
①Strain contained target plasmid was cultured in the test tubes with liquid minimal medium. Different conditions are set in each test tube and the conditions can be seen clearly from the chart below. (Parallel three groups)
②OD600 and GFP values of the bacterial culture were measured periodically with the automatic microplate reader. For green fluorescence, excitation and emission wavelengths were 395 and 509 nm.
③Draw charts using measured data to verify the function of repression of lacI
（3）Maximize the repression effect.
①Strain contained target plasmid was cultured in the test tubes with liquid minimal medium. And the concentration gradients of arabinose were set in these tubes(2×10-5M,4×10-5M,8×10-5M,1.6×10-4M,3.2×10-4M）.
②OD600 and GFP values of the bacterial culture were measured periodically with the automatic microplate reader. For green fluorescence, excitation and emission wavelengths were 395 and 509 nm.
③Draw charts using measured data to observe the trends of repression.
●cI-PR
The same as above.
●lacI-Plac
(1)Verify the function of repression of lacI
The time history plot of Relative Fluorescence Unit of the objective strain(The strain contains the plasmid named pGLO-lacI） under different conditions is shown in Fig.1
Fig.1 The Relative Fluorescence Unit of the objective strain
(2)Maximize the repression effect
The time history plot of Relative Fluorescence Unit of the objective strain(The strain contains the plasmid named pGLO-lacI） under different concentration of arabinose is shown in Fig.2
Fig.2 The Relative Fluorescence Unit of the objective strain
●cI-PR
The time history plot of Relative Fluorescence Unit of the objective strain(The strain contains the plasmid named pGLO-cI） under concentration of arabinose is shown in Fig.3
Fig.3 The Relative Fluorescence Unit of the objective strain
Conclusions and analysis
①We construct both two parts with repression function (lacI-Plac, cI-PR) successfully.
lacI-Plac: By adding arabinose to induce the expression of LacI protein, the Relative Fluorescence Unit of the objective strain is lower which illustrate the effect of repressor. And, when adding the IPTG, the Relative Fluorescence Unit of the objective strain is higher which also prove the expression of LacI protein.
cI-PR：By adding arabinose to induce the expression of cI protein, the Relative Fluorescence Unit of the objective strain is lower which illustrate the effect of repressor.
②We gain two time history plots which are shown in Fig.2 and Fig.3 respectively.
lacI-Plac: When the concentration of arabinose is 2×10-5M and 8×10-5M,the repression effect of LacI repressor is relatively close. Compared with these concentration, repression effect of LacI repressor increases after 30h under the concentration of arabinose is 16×10-5M and decreases before 40h under the concentration of arabinose is 4×10-5M .
cI-PR：Only when the concentration of arabinose is 2×10-5M, this part could inhibit the transcription of downstream.
②We select lacI-Plac as our final repression system. Because lacI-Plac can maintain the stable effect of repression at different concentrations but cI-PR didn’t show that.
▶Second step
Aim
We aim to use the lacI repression system to regular the transcription of sgRNA which is designed to specifically bind to the gene panD on the E. coli genome.
Experimental details
●lacI-Plac-sgRNA
We constructed two compatible plasmids containing the sgRNA-regulated transcriptional system (BBa_K2556042) and the Cas9 protein, respectively. And then we introduced two plasmids into E. coli MG1655. After obtaining the transformants, we tried to demonstrate the difference of the transcriptional level of sgRNA by analyzing the growth curves in four different conditions (①+2×10-5M Ara +2×10-5M IPTG ②+2×10-5M Ara ③+2×10-5M IPTG ④None) . If sgRNA transcribe correctly, the CRISPR/Cas9 system will be activated. That means the target gene panD will be cut and the E. coli couldn’t survive.
Results
●lacI-Plac-sgRNA
We did three sets of parallel experiments and the result is shown in Fig.4.
Fig.4 The growth curves in four different conditions
Conclusions and analysis
Analysis of the experimental results revealed that the lacI repression system couldn’t regulate the transcription of sgRNA successfully, because the four growth curves in different conditions are very similar. We suspect the lac operator which directly links to the sgRNA affects its conformation so that the sgRNA can’t be unable to function properly.
After consulting the literatures, we demonstrated it was exactly a DNA sequence before the sgRNA that prevented it from maturation. And it also didn’t work when we added a repeat artificially before sgRNA. Because it has been published that tracrRNA directs the maturation of crRNAs by the activities of the widely conserved endogenous RNase III and the CRISPR-associated Csn1 protein. This limits the use of the CRISPR/Cas9 system in E. coli.
Here we present two solutions. First, try to find a suitable repression system to replace the lacI repression system, which means sgRNA should be linked directly to the transcription initiation site (TSS). Second, activate the type I CRISPR/Cas system in E. coli because of the less limitation in comparison with the type II CRISPR/Cas system.
Back
Lysis
Before genome editing, we first tested the cleavage function of the part containing the genomic homology arm on the plasmid. Our part is built on the T vector. The result is shown in Fig 1:
Fig.1 Growth curve
Our results show that when we added arabinose(the concentration is 10mM) to induce lysin gene expression, the OD600 value of the bacterial culture was significantly lower than that of the control group without arabinose induction. And comparing with adding arabinose after 4.5 h of culture, when we added arabinose at the beginning, the OD600 is lower.
Our vector experiments have initially confirmed that the expression of the lysin gene is effective for cell lysis. Based on the results of this experiment, we inserted the lysis part into the E. coli genome by CRISPR/Cas technology. In order to obtain a transformant that was successfully inserted the part into genome, we screened by plate streaking, and the experimental results are shown in Fig 2:
Fig.2 Result of plate streaking
We selected 34 transformants for plate streaking, and 5 of them showed lysis effects on arabinose containing plates. They were probably the strain we have insert lysis part into genome. And finally we got a strain with a significant lysis effect0-9 after arabinose induction, and we named it E. coli MG1655-Lysis. You can see the result in Fig. 3.
Fig.3 E. coli MG1655-Lysis cultured in tubes
Back
Contact
College of Biotechnology and Bioengineering
Zhejiang University of Technology
Chaowang Rd. 18, 310014, Hangzhou, China
E-mail: cbb@zjut.edu.cn  Tel: +86-571-88320391
Copyright © 2018 ZJUT-China
