"
Page
Discussion
View source
History
teams
Log in
 
Team:SCU China/Project
From 2013.igem.org
Animation
Home
Team
Project
Modeling
Parts
Safety
Notebook
Attributions
iGEM 2013
 
Contents
1Background
2Our project design
3Five systems in our design
3.1Quorum Sensing system
3.1.1Functionality
3.1.2Design
3.1.3Test
3.1.4Achievements
3.2Controlled DNA Invertase device insertted in genome
3.2.1Functionality
3.2.2Design
3.2.3Test
3.2.4Achievements
3.3Real-time fluorescence reporter
3.3.1Functionality
3.3.2Construct process
3.4Three regulators and its responding promoters
3.4.1Functionality
3.4.2(1)The first regulator was T7 polymerase and T7 promoter.
3.4.3(2)The second regulator was riboregulator.
3.4.4(3)The third regulator was delta activator from phiR73 phage BBa_I746352
3.5F factor and its incompatible plasmid
3.5.1Functionality
3.5.2Design
4Result
4.1Result 1 Test the function of Riboregulator
4.2Result 2 Test the function of the Riboregulator and PhiR73 activator together
4.3Result 3 Test the function of Part BBa_K1087016 BBa_K1087016
Sexual differentiation is the most common and natural phenomenon in multicellutar organism, Usually, two genders’ combination through fertilization is essential for their proliferation. However, in the most primitive organisms, no sexual differentiation existed. With the evolution from protist to metazoan such as animals, definite sexual differentiation came into being. So, there is a question that when and how sexual differentiation happened. The special F factor in E. coli enables its gene communication and determination of the sex what we defined as “female” and “male”. Though this kind of gene communication is not genuinely sexual differentiation, we could make a hypothesis that the F factor is the transitory stage to true sexual differentiation. So we want to imitate sexual differentiation in unicellular organism to help us understand the process of sexual differentiation. We believe that this imitation would be very interesting and attract more people to do related research.
Background
At first, we want to introduce some background in the field of using bacteria to imitate the whole developmental procedure from a single cell to a well-organized organism.
 
Life begins from fertilization according to developmental biology. After that, the development process includes differentiation between Germline and Soma,  cell migration, pattern formation and so on. The whole course is self-organized. Imitations of above five steps except fertilization have been explored by 2007 Paris( https://2007.igem.org/Paris ), 2011 USTC( https://2011.igem.org/Team:USTC-China ), Nature 434,1130-34,28 April 2005 ( [1] ) and 2008 USTC( https://2008.igem.org/Team:USTC/Project ), respectively. However, no comprehensive studies on the imitations of gametogenesis and sexual differentiation.
 
 
Our project design
In our project, we intended to construct male and female E. coli separately (called G+ and G- cells). According the natural characteristic of E.coli, the G+ cells possessed F plasmids. On the other hand, the G- cells were constructed by using another plasmid incompatible with F factor.
When cultured separately, the G+/G- cells would divide and reach a threshold of cell density  . Then, the quorum sensing system would turn on its responding promoters, and make G+/G- differentiate into gametes (P+ and P-, respectively), which could not divide any more but were capable of transferring or accepting plasmid.
 
After that, male and female E. coli were mixed. Just like fertilization, the male gametes would recognize the female cells and began to transfer modified F plasmids into female gametes through sex pili. The conjugation made female gametes return to the state of undifferentiation (called G cells), which meant that they could divide again but were not sexually determined.
 
After several rounds of cell divisions, F factor and its incompatible plasmid in G cell would seperate.  Consequencely, the G cell differentiated into a G+ or G-, which, like zygote, maybe contained genes from both male and female gametes.
 
Simply, our project included differentiation from mature E. coli to gametes, conjugation between male and female gametes, which resulted in de-differentiation, and sex differentiation through plasmid incompatiblity.
 
Five systems in our design
 
Quorum Sensing system in high copy plasmid;
Controlled DNA Invertase device inserted into genome;
Real-time fluorescence reporters;
Three regulators and its responding promoters;
F  factor and its incompatible plasmid.
Quorum Sensing system
Functionality
This part was designed to sense the density of E. coli and to start gametogenesis at an appropriate time. So , it could imitate the growth and maturity of the animal.
 
Design
This part was constructed by assembling standard biobricks: BBa_F2622, BBa_C0261, and terminator  BBa_B0015.
In this part, luxr was controlled by inducible promoter R0010.
IPTG was added to the culture of G+/G- so that this part could work. And IPTG could be washed away to stop quorum sensing before mixing male and female gametes(quorum sensing would disturb conjugation between gametes).
 
Test
 
To test this system, we also insert a GFP after LuxI to detect the strength of luxpR and to report the time when cell density reaches the threshold.This was achieved by assembling BBa_F2622,    BBa_C0261, and BBa_I13504.
 
Achievements
We have constructed above two devices, however, sequence results suggested the whole devices except the first biobrick BBa_F2622 we used was right. Maybe some unknown problems came up when we transformed BBa_F2622 from the distribution kit.
 
 
Controlled DNA Invertase device insertted in genome
Functionality
This part was a crutial sysytem for our project since it was designed to control the differentiation from mature E.coli to gametes. This device used cre and flpe invertases, and could “invert only once at one time”. Moreover, it is reversible and could invert again after conjugation. Therefore, this part is very suitable for the control of differention and de-differentiation in our project.
 
Design
This part was derived from DNA Invertase Cascade (DIC) Counter (reference: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2690711/ ), and have been redesigned to be used in our system.
In this part, there were a reverse Lox site and a reverse PO promoter between Ptet and the RBS, and a forward Lox site after luxpR.
PO promoter was from phage (see following) and FinOP is a conjugation inhibition system consisting of an antisense RNA (finP) and a small protein (finO) (reference: https://2009.igem.org/Team:UNICAMP-Brazil ).
 
LuxpR was the quorum sensing promoter, once the cell density of G+/G- reached the threshold, the luxpR promoter would express cre invertase (with an LAA-ssrA degradation tag). Cre invertase would recognize two oppositely-oriented Lox sites, and invert the DNA sequence between them. Due to the inverted orientation of the recombination gene with respect to luxpR promoter, further expression of the recombinase protein ceases and DNA orientation is fixed. (PS: PO promoter didn’t express in G+/G- cells)
Inverted DNA sequence:
The invertion stopped the expression of FinOP, so that the inhibition of conjugation was eliminated and male E.coli could construct sex pili, which meant that it differentiated into gametes.
 
 
Construct :
In the part,reverse Lox site and reverse PO promoter were constructed by PCR using PO promoter BBa_I746361 as template. The biobrick suffix was added onto the 5’ end of the upstream primer, Xbal site and reverse Lox site were added onto the 5’ end of the downstream primer. Therefore, the PCR product was “reverse Lox site + reverse PO promoter”.
Then, we assembled Ptet with “reverse Lox site + reverse PO promoter”
The LAA-ssrA degradation tag of cre-ssrA was chosed among the four ssrA tags with different degeradation rate. The LAA was the fast one. (reference: [2] )
To construct RBS-cre-ssrA, BBa_I718008 was used as a template and the LAA tag was added onto the downstream primer.
 
Test
To test the function of above controlled DNA Invertase device, we also intended to constructed the following construct:
RFP reporter was put between the two oppositely-oriented promoters, and GFP-ssrA was put after cre-ssrA, Plac was used to replace luxpR.
Therefore, we can simply construct this device, and test its function without the need for other systems. We could use IPTG to induce the Plac, and detect the fluorescence of GFP and RFP to test whether the “Controlled DNA Invertase device” works.
Achievements
We have successfully construct Ptet-reverse Lox-reverse PO-RFP( BBa_K1087016 ), and sequence results were good. What’s more, RFP were normally expressed, obsevred and measured. See results:
We also build a model about the function of  Controlled DNA Invertase device, see model:
 
Real-time fluorescence reporter
Functionality
Real-time reporters was used to report the changes in E.coli immediately and in time. In our project, GFP and RFP were expressed in G+ and G- respectively. When G+ differentiated into P+ because of the DNA invertion in “Controlled DNA Invertase device”, we need real-time reporters to tell us it’s time to move to the next step.
 
Construct process
Since mature GFP is very stable, we need a GFP with ssrA degradation tag. (reference: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC106306/ ). We constructed the real-time GFP reporter BBa_K1087013 by assembly four biobricks from distribution kits: BBa_I712074, B0034, BBa_E0044,and teminator BBa_B0015.
 
We also constructed some other reporters to be used:
T7 promoter + constitutive promoter + mRFP1, BBa_K1087020
In this part, the reporter mRFP1 was controlled by both the strong T7 promoter and one weak constitutive promoter BBa_J23117. Thus, it was strongly expressed when cotransfected with T7 polymerase while less expressed alone.
 
T7 promoter-B0034-amilCP-teminator, BBa_K1087003
AmilCP is a very strong reporter, even can be seen by naked eyes (Reference BBa_K592025, https://2012.igem.org/Team:Groningen/Construct )
 
 
Three regulators and its responding promoters
Functionality
In our project, we need three regulators and their responding promoters to control the expression of target genes.
(1)The first regulator was T7 polymerase and T7 promoter.
T7 polymerase was put in the “Controlled DNA Invertase device”. In G+ cells, T7 polymerase was in the right orientation and expressed by Ptet. Threefore, Pt7 could express real-time reporters. When T7 polymerase sequence was inverted in P+ cells, further expression of real-time would cease.
 
(2)The second regulator was riboregulator.
(Reference: https://2006.igem.org/wiki/index.php/Berkeley2006-RiboregulatorsMain )
Functionality:
In our design, BBa_K145215 (“TetR promoter + J23066 RiboKey”) was put in G- on the plasmid incompatible with F factor.
 
 
The Ribolock “TetR promoter + lock3d” was constructed and put in G+ on the F plasmid.
 
We improved an existing biobrick part BBa_I714070([pTet][Lock3]). We could not  transforme this part from the distribution kit, and the QC results for this part in the registry were bad, too. Bad sequence, inconsistent resistence, and bad gel results.
Therefore, we constructed this part by ourselves. We assembled two basic parts Ptet BBa_R0040 and [lock3d] BBa_J23032 to create a new part: BBa_K1087004
Sequence results were confired the successful construclion, and we did lots of tests to prove its function.
 
We assembled BBa_K1087004 with mRFP to create BBa_K1087015 , and we ligate BBa_K1087015 with RiboKey BBa_K145215 to create the new BBa_K1087021 .
BBa_K1087015    
BBa_K1087021    
Positive control    
 
Negetive control:  BBa_J61046
We transformed BBa_K1087015 , BBa_K1087021 into E.coli DH5α, respectively. And we used Ptet + mRFP1 (BBa_I13521) as positive control, irrelevant BBa_J61046 as negative control. The transformed cells were cultured separately in the same condition and measured the intensity of fluorescence at the same time. See results.
 
(3)The third regulator was delta activator from phiR73 phage BBa_I746352
The third regulator was delta activator from phiR73 phage BBa_I746352, which can induce strongly PO promoter BBa_I746361 from P2 phage.
 (Reference: https://2009.igem.org/Team:Cambridge/Project/Amplification/Characterisation ).
Test
We also built some bigger constructs to test the function of Riboregulator and PhiR73 activator .
We ligated PO promoter with mRFP1 to create BBa_K1087018 so that we can test the basal level of PO promoter.
We also ligated “TetR promoter + lock3d” with phiR73 activator to create BBa_K1087007 . Then, we assembled BBa_K1087007 with BBa_K1087018 to create BBa_K1087022 .
 
BBa_K1087018                             
 
BBa_K1087022            
 
BBa_K1087023  
See results.
 
 
F factor and its incompatible plasmid
Functionality
F plasmid was intended to be constructed in G+, and would be transferred to P- when conugation.The RiboKey on F plasmid would be open the Ribolock on the plasmid in P-.Then ,the phiR73 activator would be expressed and induce PO promoter on the “Controlled DNA Invertase device”. Finally,the DNA sequence between Lox sites would be inverted to the original state. Cells returned to the state of un-differentiation (called G cells).
 
Design
We intended to modify F palsmid using RED sysytem (reference: [3] ).
 
And we intended to make use of the stardard biobrick plasmids (reference: [4] ) that were incompatible with F Factor.
 
Result
Result 1 Test the function of Riboregulator
 
We assembled BBa_K1087004 with mRFP to create BBa_K1087015 , band we assembled BBa_K1087015 with RiboKey BBa_K145215 to create BBa_K1087021 .
BBa_K1087015    
BBa_K1087021    
Positive control   
 
Negative control:  BBa_J61046
 
We transformed BBa_K1087015 and BBa_K1087021 into E. coli DH5, respectively. And we used Ptet+mRFP1 (BBa_I13521) as positive control, irrelevant BBa_J61046 with no fluorescence gene as negative control.
 
All of the constructs were expressed in high copy plasmid PSB1X3.
The transformed cells were cultured in 37℃, 200rpm.
 
We used fluorescence microscope to observe the fluorescence at 16h, and we also used Thermo Varioskan Flash to quantitatively measure the fluorescence intensity at 16h, 18h, and 19h, respectively. 150ul culture fluid was added into each well of the 96-well black plate to measure the fluorescence intensity, and OD600 were measured using spectrophotometer at the same time one by one to quantify the concentration of bacteria.
Each sample had three duplicates when cultured. And the fluorescence value was handled through mathematical method. The fluorescence intensity of above transformants was divided by the OD600 value and minus that of negative control.
  Data  Qualitative representation of  Riboregulator 1, color change with time revealed the expression level  of corresponding parts.
 
13h
14.5h
16h
17.5
19h
20.5h
5H1
white
light pink
pink
pink
pink
rose red
5H2
white
light pink
pink
pink
pink
rose red
5H3
white
light pink
pink
pink
pink
rose red
 
 
 
 
 
 
 
 
 
Figure 1.  fluorescence observation using fluorescence microscope.
A. Negative control: Coletit DH5αtransformed with B Ba_J61046 in PSB1X3.     
B. Coletit DH5αtransformed with BBa_K1087015 .      
C. Coletit DH5αtransformed with BBa_K10870 21.
D. Positive control: Coletit DH5αtransformed with B Ba_I13521.
 
 
Figure2.  quantitatively measurement of the fluorescence intensity.
Red: Coletit DH5αtransformed with BBa_K1087015 .
Blue: Coletit DH5αtransformed with BBa_K10870 21.
Black: Positive control: Coletit DH5αtransformed with B Ba_I13521.
PS: the fluorescence intensity of above transformants was divided by the OD600 value and minus that of negative control.
 
 
According to above figures, it’s obvious that the ribolock BBa_K1087004 worked well, and its fluorescence intensity was only 2.6531% compared to positive control even at 19h, while that of lock & key ( BBa_K1087021 ) was 91.7138% at 19h. The results suggested that the key BBa_K145215 could hugely improve the expression level of the lock BBa_K1087004.
 
Data:
Fluorescence per OD
BBa_K1087015
BBa_K10870 21
BBa_I13521
16h
0.089
6.8595
14.849
18h
0.486
16.011
18.363
19h
0.629
21.7435
23.708
percentage
BBa_K1087015
BBa_K10870 21
BBa_I13521
16h
0.00599367
0.4619503
1
18h
0.026466264
0.871916354
1
19h
0.026531129
0.917137675
1
Result 2 Test the function of the Riboregulator and PhiR73 activator together
 
We assembled PO promoter with mRFP1 to create BBa_K1087018 ，so that we can test the basal expression level of PO promoter.
We also assembled “TetR promoter + lock3d” with phiR73 activator to create BBa_K1087007 . Then, we assembly BBa_K1087007 with BBa_K1087018 to create BBa_K1087022 .
Finally, we assembled RiboKey BBa_K145215 with BBa_K1087022 to create BBa_K108702 3.
 
BBa_K1087018                             
 
BBa_K1087022            
 
BBa_K108702 3
We transformed BBa_K1087018 , BBa_K1087022 , and BBa_K108702 3 into E. coli DH5α, respectively. And we used irrelevant BBa    _J61046 with no fluorescence gene as negative control. The transformed cells were cultured separately in the same condition and measured fluorescence intensity at same time.
 
 
The measurement was done mainly the same as result 1.
  Data  Qualitative representation of  Riboregulator 2, color change with time revealed the expression level  of corresponding parts.
 
13h
14.5h
16h
17.5h
19h
20.5h
7H+22O 1
white
light yellow
light yellow
light yellow
light pink
light pink
7H+22O 2
white
light yellow
light yellow
light yellow
light pink
light pink
6I+3N+12G+23L+7H+22O 1
white
light yellow
light yellow
light yellow
light yellow
light yellow
6I+3N+12G+23L+7H+22O 2
white
light yellow
light yellow
light yellow
light yellow
light yellow
13J+6I+3N+12G+23L+7H+22O 1
white
light pink
light pink
light pink
 pink
rose red
13J+6I+3N+12G+23L+7H+22O 2
white
Light pink
Light pink
Light pink
pink
Rose red
 
 
 
 
 
 
 
 
 
 
Figure 1. Figure 1.  fluorescence observation using fluorescence microscope.
A. Negative control: Ecoli DH5αtransformed with BBa_J61046 in PSB1X3.     
B. E. coli DH5αtransformed with BBa_K1087018 .      
C. E. coli DH5αtransformed with BBa_K10870 22.
D. E. coli DH5αtransformed with BBa_K10870 23.
 
 
Figure2. Quantitatively measurement of the fluorescence intensity.
Blue: E. coli DH5αtransformed with BBa_K10870 23.
Green: E. coli DH5αtransformed with BBa_K10870 18.
Pink: E. coli DH5αtransformed with BBa_K108722.
PS: the fluorescence intensity of above transformants was divided by the OD600 value and minus that of negative control.
 
We could see clearly from above figures that the basal expression level of PO promoter was really low, and its strength could be improved about 11 times when induced by PhiR73 controlled by ribokey and lock.
Data:
Fluorescence per OD
BBa_K10870 23
BBa_K108722
BBa_K10870 18
16h
19.55141667
1.29
2.311
18h
49.09341667
1.5785
3.885
19h
66.099625
1.43575
5.043
fold
BBa_K10870 23
BBa_K108722
BBa_K10870 18
16h
8.460154334
0.558199913
1
18h
12.63665809
0.406306306
1
19h
13.1072
0.2847
1
 
 
 
 
 
 
Result 3 Test the function of Part BBa_K1087016 BBa_K1087016
 
Normally, we picked two colonies from the agar plate for BBa_K108701 6, called A and B. We transformed BBa_K108701 6A and B into E. coli DH5, respectively. And we use Ptet+mRFP1(BBa_I13521) as positive control, irrelevant BBa_J61046 with no fluorescence gene as negative control. The transformed cells were cultured separately in the same condition and measured fluorescence intensity at same time.
 
 
The measurement was done mainly the same as result 1.
 
Data  Qualitative representation of  Riboregulator 3 , color change with time revealed the expression level  of corresponding parts.
 
 
13.5h
15h
16.5h
18h
2-1
white
white
white
Light pink
2-2
3-1
Light pink
pink
Rose red
Rose red
3-2
4-1
white
white
Light pink
pink
4-2
 
 
 
 
 
 
 
 
 
 
 
 
Figure1. fluorescence observation using fluorescence microscope.
A. Negative control: E. coli DH5αtransformed with BBa_J61046 in PSB1X3.     
B. E. coli DH5αtransformed with BBa_K108701 6A.      
C. E. coli DH5αtransformed with BBa_K108701 6B.
D. Positive control: E. coli DH5αtransformed with BBa_I13521.
 
 
 
 
Figure2. Quantitatively measurement of the fluorescence intensity.
Blue: E. coli DH5αtransformed with BBa_K108701 6A.
Black: Positive control, E. coli DH5αtransformed with BBa_I13521.
Green: E. coli DH5αtransformed with BBa_K108701 6B.
PS: the fluorescence intensity of above transformants was divided by the OD600 value and minus that of negative control.
 
From above figures, we could conclude that the Ptet and mRFP1 in BBa_K108701 6 worked well. It was comparable to even stronger than the positive control BBa_I13521 in fluorescence intensity. For BBa_K108701 6A, it expressed about twice fluorescence of BBa_I13521. Results for sequencing suggested that the first half of the Ptet was lost in BBa_K108701 6A, which may account for its increase in strength.
 
Data:
Fluorescence per OD
BBa_K108701 6B
BBa_K108701 6A
BBa_I13521
16h
5.4215
19.23766667
14.849
18h
11.37075
36.49133333
18.363
19h
15.27725
47.6205
23.708
fold
BBa_K108701 6B
BBa_K108701 6A
BBa_I13521
16h
0.365108762
1.295553011
1
18h
0.619220716
1.987220679
1
19h
0.644392188
2.00862578
1
 
 
Retrieved from " http://2013.igem.org/Team:SCU_China/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
