"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Parts/Characterization
From 2010.igem.org
HOME
TEAM
PROJECT
PARTS
MODELING
NOTEBOOK
HUMAN PRACTICE
BIOSAFETY
  Project Home
  Project Discription
  Biosensor
  Bioabsorbent
  Application
  Expansion
   Personal Notes
  Protocols
   Vocabulary
   Parts Submitted to the Registry
   Characterization
   Existing Parts' Improvement
   Students and Instructors
   Peking University
   Collaboration
   Gallery
   Acknowledgement
   Previous Years
   Modelling Home
   Calculating Process
   Analyses and Results
   Advanced Model
   HP Home
   HGT Review
   Proof of Concept
   HGT Investigation
   Acknowledgement
Overview
  Parts
Parts Submitted
Charaterization
Existing Parts' Improvement
CountDown
Vistors From...
   Parts Characterization
         Parts >Characterization
Characterization of our Favourite Parts
Abstract and Navigation
Name
Part type
Discription
Designer
Length
Part:BBa_K346000
Translational unit
RBS(B0032)+T3 DNA-directed RNA polymerase
Haoqian Zhang & Weiye Wang
2674
Part:BBa_K346001
Translational unit
RBS (B0034) + MerR (mercury-responsive transcription factor)
Ao Liu & Ying Sheng
453
Part:BBa_K346002
Ragulatory
PmerT promoter (mercury-responsive)
Qianzhu Wu & Mei Chen
57
Part:BBa_K346003
Tranlational unit
RBS(B0032)+MBP(mercury metal binding peptide engineered from MerR))
Yang Hu & Xin Teng
342
Part:BBa_K346004
Translational unit
RBS(B0034)_MBP(lead metal binding peptide egineered from PbrR)+Terminator(B0015)
Junyi Jiao
479
Part:BBa_K346005
Device
Mercury (II) ions absorption device
Junyi Jiao, Xin Teng, Donghai Liang & Yang Hu
2424
Part:BBa_K346007
Coding
Antigen43
Miao Jing
3120
Part:BBa_K346000
Contents
Part:BBa_K346000: main introduction
Part:BBa_K346000: Part Design
Part:BBa_K346000: Experience
Part:BBa_K346000: main introduction
The RNA polymerase of bacteriophage T3 is DNA-directed RNA polymerase with both high template affinity and specificity. Similar to T7 polymerase, it also possesses quite simple structure compared with the bacterial RNA polymerase so that it can be easily wide-utilized in synthetic biology.
Previous work confirmed this concept and showed that although the RNAPs encoded by T3 and T7 bacteriophages have high sequence[1] , they exhibit nearly exclusive template specificities. According to the previous work, the two amino acid sequences have more than 82% identical residues[2].  Furthermore, promoters of the T7 and T3 RNAPs are also quite similar with highly conserved 23 bp sequences that differ significantly only over an about 3 bp region[2].  Despite these similarities, neither Polymerase (T3 RNA polymerase or T7 RNA polymerase) can transcribe the heterologous promoter efficiently[3,4] . Namely, the T3 RNA polymerase is orthogonal to T7 RNA polymerase, which is an excellent alternative of T7 polymerase for complex genetic circuit designed in the near future.
Figure 2. Predicted interaction between T3 polymerase and its cognate T3 promoter, based the data gained from T7 polymerase.
a. The interaction between bacterial RNA polymerase and the promoter DNA.
1. The -10 region interacts with the sigma factor;
2. The -35 region also interacts with the sigma factor;
3. The alpha CTD contacts the CRP protein.
b. The interaction between T3 polymerase and the T3 promoter.
1. The -1 to -4 TATA interacts with the 228 to 245 intercalating hairpin.
2. The -12 to -17 region of the promoter interacts with a beta loop of the T3 RNA polymerase.
3. The -5 to -12 region of the promoter interacts with the core specificity determinant.
Figure 3: The expression of T3 polymerase in BL21 strain. The strain bears a plasmid that has T7 promoter upstream of T3 polymerase. The left three columns represent strains that were not inducted by IPTG, and from left to right is protein expression in whole cell, cytosol, or inclusive body. The fourth column is the protein marker. The right three columns represent strains that were induced by 10^-3 M IPTG for 30 hours under 18℃. Note that there is a significant band in columns representing the induced strains as indicated by arrows, and the result shows that the T3 polymerase was mainly expressed in the cytosol.
Figure 4: The strength of different T3 promoters under the same expression level of T3 polymerase. BBa_E0240 was combined with 11 T3 promoters from the genome of T3 phage. The x axis denotes strains that only differ in the T3 promoter strength, in accordance with the order from strong to weak. The Y axis denotes the GFP intensity normalized by OD600. We can see that these T3 promoters can be divided into 3 subgroups: weak, medium and strong. (You can find more details about the promoter intensity characterization on the Experience Page)
Following is the key to the partsregistry number of each T3 promoter we characterized and their primary names in T3 phage genome.
phiOL
[ BBa_K346039 ];
phiOR
[ BBa_K346040 ];
phi1.05
[ BBa_K346041 ];
phi1.1
[ BBa_K346042 ];
phi1.3
[ BBa_K346043 ];
phi1.5
[ BBa_K346044 ];
phi2.5
[ BBa_K346045 ];
phi4.3
[ BBa_K346046 ];
phi6.5
[ BBa_K346047 ];
phi9
[ BBa_K346048 ];
phi10
[ BBa_K346049 ];
phi11
[ BBa_K346050 ];
phi13
[ BBa_K346051 ];
phi3.8
[ BBa_K346052 ];
-
-
-
-
Notice:  phi13 has the same sequence with phi6.5.
It should be also noticed that as T3 RNA polymerase has dramatically high template affinity, even a little leakage expression of T3 polymerase might cause a significant basal level at T3 promoter. Therefore, when T3 polymerase is exploited in genetic circuit, its expression must be tightly controlled to reduce the basal level.
Reference
1. Davis, R. W. & Hyman, R. W (1971)J. Mol. Biol 62, 287-301.
2. McGraw, N. J., Bailey, J. N., Cleaves, G. R., Dembinski, D. R., Gocke, C. R., Joliffe, L. K., MacWright, R. S.&McAllister, W. T. (1985) Nucleic Acids Res. 13, 6753–6766.
3. Klement, J. F., Moorefield, M. B., Jorgensen, E. D., Brown, J. E., Risman, S. & McAllister, W. T. (1990) J. Mol. Biol. 215, 21–29.
4. Lee, S. S. & Kang, C. W. (1992) Biochem. Int. 26, 1–5.
5. Li, T., Ho, H. H., Maslak, M., Schick, C. & Martin, C. T. (1996) Biochemistry 35, 3722–3727.
6. Chapman, K. A. & Burgess, R. R. (1987) Nucleic Acids Res. 15, 5413–5432.
[ TOP ]
Part:BBa_K346000: Part Design
T3 RNA polymerase has a dramatically high transcription rating; therefore a strong RBS is not necessary as a medium RBS (BBa_B0032) will be enough to reach the maximum of transcription activation at T3 promoters.
What’s more, we observed that maximum transcription rating might result in protein misfolding sometimes and the apparent transcription activation fold will decrease instead. Therefore we strongly recommend a low expression intensity of T3 polymerase in the case of genetic circuit design, or the combination of T3 polymerase with a low-strength T3 promoter, such as promoter phi 1.3 or 4.3. You can find more info about the T3 promoter intensity in the following Figure
Figure 5: The strength of different T3 promoters under the same expression level of T3 polymerase. BBa_E0840 was combined with 11 T3 promoters from the genome of T3 phage. The x axis denotes strains that only differ in the T3 promoter strength, in accordance with the order from strong to weak. The Y axis denotes the GFP intensity normalized by OD600. We can see that these T3 promoters can be divided into 3 subgroups: weak, medium and strong. (You can find more details about the promoter intensity characterization on the Experience Page)
[ TOP ]
Part:BBa_K346000: Experience
We constructed a two-plasmid system to characterize the T3 polymerase and T3 promoters, using IPTG, as the inducer of T3 polymerase expression and GFP as the reporter gene.
Fig 1. The two-plasmid system we constructed to characterize T3 RNAP/T3 promoter pairs. T3 promoter was prefixed BBa_E0840 by insertion of annealing primers which would form EcoRI and SpeI sticky end.  Induction temperature was 28-30℃
Induction time was 6-8h
IPTG concentration was 10^-3M; culture with no IPTG supplement as the negative control
Spin down the cells under 5000rpm, and discard the supernatant. Resuspend the pellet with PBS. The the intensity GFP was recorded by the microplate reader. Parameters were selected as suggested in BBa_E0840.
Figure 4: The strength of different T3 promoters under the same expression level of T3 polymerase (10^-3M IPTG). BBa_E0840 was combined with 11 T3 promoters from the genome of T3 phage. The x axis denotes strains that only differ in the T3 promoter strength, in accordance with the order from strong to weak. The Y axis denotes the GFP intensity normalized by OD600. We can see that these T3 promoters can be divided into 3 subgroups: weak, medium and strong.
[ TOP ]
Part:BBa_K346001
Contents
Part:BBa_K346001: main introduction
Part:BBa_K346001: Part Design
Part:BBa_K346001: Experience
Part:BBa_K346001: main introduction
<partinfo>BBa_K346001 short</partinfo>
This part was designed as a translational unit for MerR expression.
The mercury resistance operon, mer, can enable bacteria to avoid and remove toxic metal Hg under the regulation of the MerR family transcriptional factor MerR. MerR acts as an activator of mer genes in response to the presence of Hg (II), while it will be reversed into a weak repressor in the absence of Hg (II), to maintain its own expression at certain level.
Fig.1. MerR dimer structure. We can see from its structure, upper part is the metal binding domain. When two monomers come together in the presence of mercury, to form a dimer, mercury ions will be captured inside.
The MerR protein, in the form of homodimer, binds to the mer operon between the RNA polymerase binding sites of the promoter region. The attachment is independent on the presence of Hg(II). When the apo-MerR dimer bind to the dyad symmetrical operator DNA between the -35 and – 10 elements of mercury inducible promoter, PmerT, which has a unusually long spacer of 19 bp for MerR to bind on, the binding of RNA polymerase is inhibited. When Hg(II) is available in the environment, the ion binds to merR between the two subunits. The Hg-bound MerR can result in an a structural distortion of PmerT, allowing the RNA polymerase contacts to be made, leading to the expression of down-stream genes(Fig.2).
Fig.2. A generalized mechanism of MerR family regulator transcriptional activation. A: The dimeric MerR regulator binds to the operator region of the promoter and recruits RNA polymerase, forming a ternary complex. Transcription is slightly repressed because the apo-MerR regulator dimer has bent the promoter DNA such that RNA polymerase does not contact it properly. B: Upon binding the cognate metal ions (shown as cyan circles) the metallated MerR homodimer causes a realignment of the promoter such that RNA polymerase contacts the -35 and -10 sequences leading to open complex formation and transcription. Modified from Brown et al.
Part:BBa_K346001: Part Design
The native RBS of MerR is not strong enough and its intensity can not be predicted as there is no such a corresponding RBS part in Registry. In order to facilitate the use and further characterization of MerR for future team, we prefixed an RBS part  BBa_B0034 from Partsregistry.
The model of MerR controlling PmerT tracscription indicates that the apo-merR and Hg-bound merR have a competition. We speculate that the threshold of MerR response can be also manipulated by controlling the concentration of MerR in cytosol. As with the bacteria in natural environment, the concentration of MerR is stabilized at a certain level. In our design, merR was isolated from the operon and assembled with constitutive promoters of certain strength to maintain its expression intensity at certain level, then the sensitivity of PmerT under different MerR concentrations can be denoted by mercury threshold concentration at which reporter (GFP) expression emerges. (Fig.3).
Fig.3. The construction of bioreporter. This biosensor construct was made by fusing PmerT and a reporting system, gfp, along with a plasmid structure that constitutive promoters prefixed before merR coding sequence.
We observed that cells with different MerR intensity exhibited correspondingly different sensitivity to mercury, indicating that the stronger the expression level of MerR is, a higher threshold is represented(Fig.4).
Fig.4. The expression intensity of MerR significantly determines the threshold of sensitivity to mercury (II). Five representative lines are selected and it can be seen that the thresholds have varied apparently. The letter in the bracket after the promoter name denotes the backbone (pSB3K3 or pSB1A2) where Pc-RBS-merR was cloned. The deeper the colour, the stronger the expression level of MerR is, leading to a higher threshold.
Source
The coding sequence of MerR comes from Tn21, prefixed by an RBS part BBa_B0034 from Partsregistry
References
Brown, N. L., J. V. Stoyanov, et al. (2003). "The MerR family of transcriptional regulators." FEMS Microbiol Rev 27(2-3): 145-163.
Hobman, J. L., J. Wilkie, et al. (2005). "A design for life: prokaryotic metal-binding MerR family regulators." Biometals 18(4): 429-436.
Diana M. Ralston, Tomas V. O'Halloran, et al. (1990). “Ultrasensitivity and heavy-metal selectivity of the allostericalyl modulated MerR transcription complex” Proc. Natl. Acad. Sci. USA, Vol. 87, pp. 3846-3850,
Park, S. J., J. Wireman, et al. (1992). "Genetic analysis of the Tn21 mer operator-promoter." J Bacteriol 174(7): 2160-2171.
[ TOP ]
Part:BBa_K346001: Experience
We speculate that the threshold of MerR response can be also manipulated by controlling the concentration of MerR in cytosol. As with the bacteria in natural environment, the concentration of MerR is stabilized at a certain level. In order to verify the speculation, promoters from partsregistry constitutive promoter library with different strength were prefixed before BBa_B0034+MerR coding sequence to exogenously maintain MerR expression at different intensity (Fig.6). The sensitivity of PmerT under different MerR concentrations can be denoted by mercury threshold concentration at which reporter (GFP) expression emerges. Furthermore, we used pSB1A2 and pSB3K3 as backbones because of their different copy numbers which could also result in different MerR expression intensity (Fig.7), which was confirmed by an additional experiment.
Fig.6. The construction of the system used for MerR operation characterization. Promoters from the partsregistry with different intensities were selected, leading to varying MerR concentrations. The transcription of PmerT is controlled by the percentage of Hg-bound MerR dimer, thus the expression of GFP can reflex the concentration of Hg (II).
Fig.7. The strategy of backbones swiching. We used pSB3K3, a low-copy number backbone, and pSB1A2, high-copy, to differ the expression of merR.
We carefully tested every combination (the strength of constitutive promoter controlling MerR expression and the backbone copy number) under induction with a Hg (II) concentration gradient ranging from 10^-9M to 10^-5M. The concentration gradient of Hg (II) is: 0, 1E-9, 3E-9, 5E-9, 8E-9, 1E-8, 3E-8, 5E-8, 8E-8, 1E-7, 3E-7, 5E-7, 8E-7, 1E-6, 3E-6, 5E-6, 8E-6, 1E-5, 3E-5, and 5E-5. Cell culture was cultivated for 8 hours and then dilute with fresh LB in a ratio of 1:100 and continue to incubate until the OD600 reached 0.4~0.6.
The result is shown in Fig.8.
Fig.8. The expression intensity of MerR significantly determines the threshold of sensitivity to mercury (II). Five representative lines are selected and it can be seen that the thresholds have varied apparently. The letter in the bracket after the promoter name denotes the backbone (pSB3K3 or pSB1A2) where Pc-RBS-merR was cloned. The deeper the colour, the stronger the expression level of MerR is, leading to a higher threshold.
The sensitivity of PmerT under different MerR concentrations can be denoted by mercury threshold concentration at which reporter (GFP) expression emerges. Data demonstrates that cells with different MerR intensity exhibited correspondingly different sensitivity to mercury, indicating that the stronger the expression level of MerR is, a higher threshold is represented.
[ TOP ]
Part:BBa_K346002
Contents
Part:BBa_K346002: main introduction
Part:BBa_K346002: Part Design
Part:BBa_K346002: Experience
Part:BBa_K346002: main introduction
This part, PmerT, is a promoter from Tn21 mercury resistance (mer) operon. The mer operon of Tn21 consists of two tightly overlapped, divergently oriented promoters – Pr and Ptpad.(Park, Wireman et al. 1992). Pr is the promoter of the regulatory protein gene, merR, and Ptpcad is for the transcription of the structural gene – merPTAD. They are called merOP as a whole.
merOP is bound by MerR, a regulatary protein, as a homodimer and the occupancy of Ptpad by RNA polymerase is enhanced regardless of the presence of Hg(II), although only after Hg(II)’s binding can the dimer stop preventing the formation of the open complex by RNA polymerase(Fig.1).
Fig.1. A generalized mechanism of MerR family regulator transcriptional activation. (A) The dimeric MerR regulator binds to the operator region of the promoter and recruits RNA polymerase, forming a ternary complex. Transcription is slightly repressed because the apo-MerR regulator dimer has bent the promoter DNA such that RNA polymerase does not contact it properly. (B) Upon binding the cognate metal ions (shown as cyan circles) the metallated MerR homodimer causes a realignment of the promoter such that RNA polymerase contacts the -35 and -10 sequences leading to open complex formation and transcription. Modified from Brown et al.
The key sequence for MerR’s binding is a region of interrupted dyad symmetry (19bp) located between the -35 and -10 haxamers of Ptpcad (The top strand). And the structure of Pr (botton strand) is similar to Ptpcad in a divergent orientation. The -10 hexamers of Ptpcad and Pr actually overlap by four bases. When the apo-MerR dimer bind to the dyad symmetrical operator DNA between the -35 and – 10 elements of mercury inducible promoter, PmerT, which has a unusually long spacer of 19 bp for MerR to bind on, the binding of RNA polymerase is inhibited(Fig.2). When Hg(II) is available in the environment, the ion binds to merR between the two subunits. The Hg-bound MerR can result in an a structural distortion of PmerT, allowing the RNA polymerase contacts to be made, leading to the expression of down-stream genes. This mode of transcription activation indicates that as the apo-MerR and Hg-bound MerR have a competing relationship. The initiation of PmerT is controlled by the expression level of MerR, and the activity of which can be denoted by reporter genes. Thus the sensitivity of Hg(II) in cell can be manipulated.
Fig.2. DNA sequence of the Tn21 mer operator promoter region. The MerR binding site on PmerT is marked by a box. The -35 and -10 regions for both PmerR and PmerTPAD are marked with boxes, and the dyad symmetrical DNA sequence that MerR recognizes and binds to is marked with arrows under the DNA sequence. The divergently oriented promoters are marked by blue box and purple box, respectively.
For more information, please check 'Experience' and our wiki!
[ TOP ]
Part:BBa_K346002: Part Design
Design Notes
In our design, merR was isolated from the operon and assembled with constitutive promoters of certain strength to maintain its expression intensity at certain level. For the same reason, the divergent promoter Pr was also removed by deletion of its -35 region(Fig.3).
Fig.3. DNA sequence of the Tn21 mer operator promoter region. The MerR binding site on PmerT is marked by a box. The -35 and -10 regions for both PmerR and PmerTPAD are marked with boxes, and the dyad symmetrical DNA sequence that MerR recognizes and binds to is marked with arrows under the DNA sequence. (A) The divergently oriented promoters are marked by blue box and purple box, respectively.  (B) In our project, the expression intensity of MerR should be maintained exogenously, so the divergent promoter Pr (of MerR transcript) was also removed by deletion of its -35 region. The resulted promoter sequence is marked with a dark purple box Modiﬁed from (Hobman, Wilkie et al. 2005)
By annealing PmerT-forward (5’-3’) and PmerT-reverse (5’-3’) primers, Ptpcad carrying a sticky end of EcoRI and SpeI was cloned upstream of BBa_E0840, a GFP generator. With exogenous expression of  MerR in bacteria, GFP’s expression could be induced by Hg (II) in a dose response manner(Fig.4).
Fig.4. The construction of bioreporter. This biosensor construct was made by fusing PmerT and a reporting system, gfp, along with a plasmid structure that constitutive promoters prefixed before merR coding sequence.
Source
PmerT is a promoter from Tn21 mercury resistance (mer) operon. By annealing PmerT-forward (5’-3’) and PmerT-reverse (5’-3’) primers, PmerT carrying a sticky end of EcoRI and SpeI was cloned into Psb1C3. The sequence is as below.
TTCCATATCGCTTGACTCCGTACATGAGTACGGAAGTAAGGTTACGCTATCCAATCC
References
Brown, N. L., J. V. Stoyanov, et al. (2003). "The MerR family of transcriptional regulators." FEMS Microbiol Rev 27(2-3): 145-163.
Hobman, J. L., J. Wilkie, et al. (2005). "A design for life: prokaryotic metal-binding MerR family regulators." Biometals 18(4): 429-436.
Liebert, C. A., R. M. Hall, et al. (1999). "Transposon Tn21, flagship of the floating genome." Microbiol Mol Biol Rev 63(3): 507-522.
Nakaya, R., A. Nakamura, et al. (1960). "Resistance transfer agents in Shigella." Biochem Biophys Res Commun 3: 654-659.
Park, S. J., J. Wireman, et al. (1992). "Genetic analysis of the Tn21 mer operator-promoter." J Bacteriol 174(7): 2160-2171.
[ TOP ]
Part:BBa_K346002: Experience
Promoter Characterization
By annealing PmerT-forward (5’-3’) and PmerT-reverse (5’-3’) primers, PTPCAD carrying a sticky end of EcoRI and SpeI was cloned upstream of BBa_E0840, a GFP generator. With exogenous expression of  MerR in bacteria, GFP’s expression could be induced by Hg (II) in a dose response manner. The PTPCAD-E0840 was then cloned into pSB3K3 backbone and the BBa_J23103 (constitutive promoter)-merR into pSB1A3.
Protocols for promoter characterization
1.An overnight culture of bacteria carrying the two plasmids pSB3K3 and pSB1A3 were grown in LB broth with ampicillin and kanamycin at 37°C was reactivated by diluting the culture in a ratio of 1:100 with fresh LB.
2.When OD600 reached 0.4-0.6, the bacteria was disposed to several EP tubes, each owning 500uL, and different dose of Mercuric chloride solution was spplied with 3 duplicates and the final concentration varied from 0 to 1.0E-6 mol/L. 100uL of the 500uL was added to the black-96-well plates for GFP intensity’s measurement and another 100uL was added to the transparent-96-well plates for OD600 measurement.
3.In-plate culture fluorescence and OD600 was recorded at 20min intervals from 0 to 275min and the GFP intensity of 20min before inducement was also measured. Temperature was constant at 37°C.
The result is shown in Fig.5.
Fig.5. Time and dose response of  GFP and OD 600. The data was measured every 20 minutes since 20 minutes before supplement of different dose of Hg(II) to the wells and the plates was incubated in the shaker at 37℃during the interval of measurement. For both plates, the volume of LB medium with bacteria was 100uL per well. A: GFP intensity was measured by Tecan Microplate Reader with excitation wavelength at 470nm and emission wavelength at 509nm. A black 96-well plate was used to minimize the interference of different well. B: OD 600 was also measured by Tecan Microplate Reader in a transparent 96-well plate, since the depth of 100uL in the well did not reach 1 centimeter, the OD 600 value here was smaller than that was measured by a spectrophotometer.
Construction of merOP Library
In our project, we planned to analyze the behavior of merOP thoroughly, and it could be expected that mutation at the dyad domain II would change the Ka of MerR-DNA interactions. Thus we mutated the semiconserved sites and left the conserved sites constant.
The approach is generally based upon degenerate primer PCR, with the combination of a ‘DNA shuffling’ procedure, that is performed on the target DNA sequence; the resulting library of variants is then screened for the desired feature, and selected isolates are subjected to a repeated procedure(Fig.6).
Fig.6. Library construction. We performed a degenerate primer PCR on the merTPCAD wild-type promoter region of the alterable sites. The resulting PCR fragments, each potentially contained one or more mutation sites at a restricted location.
Fig.7. Dose-response curves of mutants. Among all these candidates, mutant1, 3, 25, 44, 85, 88 (also shown in Fig 3) were selected for the final careful characterization during which a higher concentration resolution was exploited. Then the dose-response curves were fitted by Hill function.
Fig.8 Sequenc comparison of the mutants
It can be observed that mutations at the semiconserved region of PmerT promoter significantly influenced the response behavior of MerR/PmerT pair(Fig.7). It is probably because that the mutations alter the binding affinity between MerR and PmerT promoter. As a result of this, different sensitivity (higher or lower than the wild-type) PmerT promoters were gained by us. More details about these mutants suffixed with GFP(BBa_E0840) were described in the parts BBa_K346020 - BBa_K346025.
[ TOP ]
Part:BBa_K346003
Designed by Huyang Tengxin   Group: iGEM10_Peking   (2010-10-14)
Contents
Part:BBa_K346003: main introduction
Part:BBa_K346003: Part Design
Part:BBa_K346003: Experience
RBS(B0032)+MBP(mercury metal binding peptide engineered from MerR)
This part is a combination of a rbs, a mbp, the coding sequence of a mercury metal binding peptide egineered from MerR.It was designed to function as mercury(II) binding peptide in our project.It is expressed in the cytosol and when fused with the periplasm protein Dsba or the membrane protein lpp-ompa, it will be expressed and translocated to the periplasm and membrane.In short, this part function as the fundamental metal binding peptide of mercury in our project.
Part:BBa_K346003: main introduction
MerR, the mercury-responsible transcription factor(figure on the left), acts as an effective mercury accumulator in aquatic environment.However, as a transcription regulator, over-expression of MerR in bacteria may lead to some unpredictable side effect. Earlier work suggested that the truncated peptide only consisting of the metal binding domain can form a stable dimer with its mercury binding affinity remained and DNA binding domain and metal binding domain can function individually.Based on all these above and carefully structure analysis of MerR via 3D structure modeling, we directly tandemed two copies of metal binding domain of MerR together, to implement a mercury metal binding peptide (MBP) as is shown in the figure on the right.
The figure shows the structure of MerR and MBP. The left figure is MerR, the mercury-responsive transcription factor. The right figure shows the predicted structure of resulted metal binding peptide. Mercury ions are indicated as black balls in metal binding pockets.
[ TOP ]
Part:BBa_K346003: Part Design
To achieve the goal of making a high performance MBP, we constructed a single polypeptide consisting of two dimerization helixes and metal binding loops of MerR, to form an antiparallel coiled coil MBP mimicking the dimerized metal binding domains of the wild-type as described in the Fig. We amplified the N-terminal and C-terminal of MBP directly from full length MerR by PCR, and then cloned them into the backbone together in one step.Then the constructed mbp coding sequence is inserted to the plasmid
Source
The MerR is on the plasmid NR1, which is provided by Anne O. Summers.
References
Ralston, D. M. & Halloran, T. O. Ultrasensitivity and heavy-metal selectivity of the allosterically modulated MerR transcription complex. Proc. Natl. Acad. Sci. 87, 3846-3850 (1990).
Brocklehurst, K. R., Hobman, J. R., Lawley, B., Blank, L., Marshall, L. J., Brown, N. L. & Morby, A. P. ZntR is a Zn(II)-responsive  MerR-like transcriptional regulator of zntA in Escherichia coli. Mol. Microbiol. 31, 893-902 (1999).
Zeng, Q., Stalhandske, C., Anderson, M. C., Scott, R. A. & Summers, A. O. The core metal-recognition domain of MerR. Biochemistry 37, 15885-15895 (1998).
Changela, A., Chen, K., Xue, Y., Holschen, J., Outten, C. E., Halloran, T. V. & Mondrago, A. Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR. Science 301, 1383-1387 (2003).
Chen, P. R. & He, C. Selective recognition of metal ions by metalloregulatory proteins. Curr. Opin. Chem. Biol. 12,214-221 (2008).
Shewchuk, L. M., Verdine, G. L., Nash, H. & Walsh, C.T. Mutagenesis of the cysteines in the metalloregulatory protein MerR indicates that a metal-bridged dimer activates transcription. Biochemistry 28, 6140-6145 (1989).
Wright, J. G., Tsang, H. T., Penner-Hahn, J. E. & O’Halloran T.V. Coordination chemistry of the Hg-MerR metalloregulatory protein: evidence for a novel tridentate Hg-cysteine receptor sites. J. Am. Chem. Soc. 112, 2434-2435 (1990).
Changela, A., Chen, K., Xue, Y., Holschen, J., Outten, C. E., Halloran, T. V. & Mondrago, A. Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR. Science 301, 1383-1387 (2003).
Chen, P. R. & He, C. Selective recognition of metal ions by metalloregulatory proteins. Curr. Opin. Chem. Biol. 12,214-221 (2008).
Shewchuk, L. M., Verdine, G. L., Nash, H. & Walsh, C.T. Mutagenesis of the cysteines in the metalloregulatory protein MerR indicates that a metal-bridged dimer activates transcription. Biochemistry 28, 6140-6145 (1989).
Wright, J. G., Tsang, H. T., Penner-Hahn, J. E. & O’Halloran T.V. Coordination chemistry of the Hg-MerR metalloregulatory protein: evidence for a novel tridentate Hg-cysteine receptor sites. J. Am. Chem. Soc. 112, 2434-2435 (1990).
[ TOP ]
Part:BBa_K346003: Experience
To be specific, the entire coding region of the MBP for standard part was amplified by PCR from full length MerR with two pairs of primers. Two of these primers encoded a three-residue bridge, SSG, which does not occur in MerR and was added to afford some flexibility in the loop connecting the two dimerization helix. The two PCR products were digested with EcoR I / BamH I, or BamH I / Pst I and cloned into EcoR I / Pst I -digested pSB1K3 in one step (fig), which was verified by DNA sequencing.Then the RBS and terminator is added.
Based on the same strategy, MBP-His6 was constructed by using two different pairs of primers, which is used for MBP expression test by western blot. The two PCR products were digested with Nde I / BamH I, or BamH I / Xho I and cloned into Nde I / Xho I -digested pET 21a, which contains a region encoding six histines, in one step to construct pET 21a – mbp , which was verified by DNA sequencing.
Expression Experiment and Function Test:
To test the function of this part, both expression experiment and function test is necessary. We have verified the size of the expressed proteins with SDS-page and Western blot. Besides, to test the efficiency of mercury binding, we also carried out the function test with ICP-AES with the mercury gradient from 10^-8M to 10^-6M.
Results:
Expression of proteins
The plasmid PET21a-mbp is transferred to E.coli strain BL21, which can generate T7polyerase when induced with IPTG. Both induced cells and uninduced cells(as control) are centrifuged to get the cytosol, the periplasm and the membrane separated. The SDS-page and Western blot of the expressed proteins show that induced cells expressed an identical IPTG-inducible protein at the proper place with the size of ~12kD which consists with the predicted size, indicating that the engineered MBP can be expressed in the cytosol.
Function test Having made sure that the protein can express normally in the cytosol, the function tests experiment are carried out with ICP-AES. To test the efficiency of mercury absorption of MBP in different concentration of mercury, the concentration gradient is set from 10^-8M to 10^-6M, the results are shown in figure 3. It is obvious that the efficiency of MBP increases with the increase of the mercury concentration and it can binding Hg(II) with high efficiency and high sensitivity from the concentration of 10^-7 M compared to that of the control.
[ TOP ]
Part:BBa_K346004
Contents
Part:BBa_K346004: main introduction
Part:BBa_K346004: Part Design
Part:BBa_K346004: Experience
RBS(B0034)_MBP(lead metal binding peptide egineered from PbrR)+Terminator(B0015)
RBS+mbp(codes for the lead binding peptide)+terminator.
This part was designed to function as lead binding peptide in our project.It is expressed in the cytosol and when fused with the periplasm protein Dsba and membrane protein lpp-ompa, it will be expressed and translocated to the periplasm and membrane.In short, this part function as the fundamental metal binding peptide of lead in our project.
Part:BBa_K346004: main introduction
MerR family TFs share a high similarity at the C-terminal metal binding domain , which indicates a similar metal recognition mechanism and metal-protein complex structure.Previous work shows that C-terminal metal binding domain can act as a metal accumulator without the help of the N-terminal DNA binding domain.Function test of mercury MBP showed that our whole-cell bioabsorbents can absorb more than 50% of 10-6 M Hg (II) in 120 minutes, indicating that cells expressing certain metal binding peptide engineered via our method can decontaminate corresponding metal ion from aquatic environment. This design has been successfully applied to another homolog of MerR family – PbrR, the lead responsive regulator.In our project, we try to tandem two metal binding domains together to make a high performance and less energy consuming metal binding peptide. Our bacteria achieved an equivalent lead accumulation capacity as the mercury MBP, which proved validness of our engineering strategy. This part works as the core part in the lead bioabsorption device.
The figure shows the structure of PbrR and MBP(lead). The left figure is PbrR  a member of MerR family and the right figure shows the predicted structure of resulted metal binding peptide of lead, which is engineered from PbrR. lead ions are indicated as black balls in metal binding pockets.
[ TOP ]
Part:BBa_K346004: Part Design
Similar method as MBP(mercury) is constructed was applied to PbrR. Firstly,Sequence alignment of MerR and PbrR has been carried out. Previous work showed that MerR family TFs share a highly conserved homology at their metal binding domains (Brown et al., 2003; Hobman, 2007), which implies that our strategies of bioabsorbent engineering might be applicable to other cases of heavy metals. We again design a single polypeptide consisting of two dimerization helixes and metal binding loops of PbrR, to form an antiparallel coiled coil MBP mimicking the dimerized metal binding domains of the wild-type as Figure(B) shows.Figure (C) shows that the MBP was constructed by fusing two copies of metal binding domain of PbrR in tandem via the same method with mercury MBP.
Source
Pro. Chuan He
References
Brown, N. L., Stoyanov, J. V. & Kidd, S. P. & Hobman, J. L. The MerR family of transcriptional regulators. FEMS Microbiol. Rev. 27, 145-163 (2003).
Zeng, Q., Stalhandske, C., Anderson, M. C., Scott, R. A. & Summers, A. O. The core metal-recognition domain of MerR. Biochemistry 37, 15885-15895 (1998).
Mejare, M. & Bulow, L. Metal-binding proteins and peptides in bioremediation and phytoremediation of heavy metals. Trends in biotechnol. 19, 67-73(2001).
Ralston, D. M. & Halloran, T. O. Ultrasensitivity and heavy-metal selectivity of the allosterically modulated MerR transcription complex. Proc. Natl. Acad. Sci. 87, 3846-3850 (1990).
Brocklehurst, K. R., Hobman, J. R., Lawley, B., Blank, L., Marshall, L. J., Brown, N. L. & Morby, A. P. ZntR is a Zn(II)-responsive MerR-like transcriptional regulator of zntA in Escherichia coli. Mol. Microbiol. 31, 893-902 (1999).
Zeng, Q., Stalhandske, C., Anderson, M. C., Scott, R. A. & Summers, A. O. The core metal-recognition domain of MerR. Biochemistry 37, 15885-15895 (1998).
Changela, A., Chen, K., Xue, Y., Holschen, J., Outten, C. E., Halloran, T. V. & Mondrago, A. Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR. Science 301, 1383-1387 (2003).
Chen, P. R. & He, C. Selective recognition of metal ions by metalloregulatory proteins. Curr. Opin. Chem. Biol. 12,214-221 (2008).
Shewchuk, L. M., Verdine, G. L., Nash, H. & Walsh, C.T. Mutagenesis of the cysteines in the metalloregulatory protein MerR indicates that a metal-bridged dimer activates transcription. Biochemistry 28, 6140-6145 (1989).
Wright, J. G., Tsang, H. T., Penner-Hahn, J. E. & O’Halloran T.V. Coordination chemistry of the Hg-MerR metalloregulatory protein: evidence for a novel tridentate Hg-cysteine receptor sites. J. Am. Chem. Soc. 112, 2434-2435 (1990).
[ TOP ]
Part:BBa_K346004: Experience
Just as what we have done to construct the mercury Metal Binding Peptide, the entire coding region of the MBP for standard part was amplified by PCR from full length MerR with two pairs of primers. Two of these primers encoded a three-residue bridge, SSG, which does not occur in MerR and was added to afford some flexibility in the loop connecting the two dimerization helix. The two PCR products were digested with EcoR I / BamH I, or BamH I / Pst I and cloned into EcoR I / Pst I -digested pSB1K3 in one step (fig), which was verified by DNA sequencing.Then the RBS and terminator is added.
Based on the same strategy, MBP-His6 was constructed by using two different pairs of primers, which is used for MBP expression test by western blot. The two PCR products were digested with Nde I / BamH I, or BamH I / Xho I and cloned into Nde I / Xho I -digested pET 21a, which contains a region encoding six histines, in one step to construct pET 21a – mbp , which was verified by DNA sequencing.
Expression Experiment and Function Test:
To test the function of this part, both expression experiment and function test is necessary. We have verified the size of the expressed proteins with SDS-page and Western blot. Besides, to test the efficiency of mercury binding, we also carried out the function test with ICP-AES with the mercury gradient from 10^-8M to 10^-6M.
Results:
Expression of proteins
The plasmid PET21a-mbp is transferred to E.coli strain BL21, which can generate T7polyerase when induced with IPTG. Both induced cells and uninduced cells(as control) are centrifuged to get the cytosol, the periplasm and the membrane separated. The SDS-page and Western blot of the expressed proteins show that induced cells expressed an identical IPTG-inducible protein at the proper place with the size of ~12kD which consists with the predicted size, indicating that the engineered MBP can be expressed in the cytosol.
The specific band in western blot for his-tag fused MBP of about 12 kD confirmed that the MBP is expressed as expected. Considerable amount of MBP expressed in cytosol can be indicated from the result of SDS-PAGE.
Function test
Having made sure that the protein can express normally in the cytosol, the function tests experiment are carried out with ICP-AES. To test the efficiency of mercury absorption of MBP in different concentration of mercury, the concentration gradient of lead is set from 10^-7M to 10^-5M （the results are not shown in the following figure). The results of the efficiency of different parts containing MBP(lead) is shown here. It is apparent that under the concentration of 10^-5M,there is a notable absorption of lead compared to the control. What's more, the device containing three parts are more efficient than the parts alone.
Figure 2 Different amount of lead absorbed by bacteria with MBP expressed in different subcellular compartments cultured for ~40h in 10-5 mol/L Pb (II) medium.
[ TOP ]
Part:BBa_K346005
Contents
Part:BBa_K346005: main introduction
Part:BBa_K346005: Part Design
Part:BBa_K346005: Experience
Part:BBa_K346005: main introduction
Hg(II) Bioabsorption Device
Dsba-mbp(mercury metal binding peptide)+mbp(mercury metal binding peptide)+lpp-ompa-mbp(mercury metal binding peptide)
This part was designed to function as mercury(II) ions absorption device in our project. If T7 polymerase is inductively expressed, this device will be switched on and metal bind peptide will be dramatically expressed and translocated to cytosol, periplasm and outer membrane surface. Namely, bacteria bearing this device will function as whole-cell bioabsorbent in the presence of T7 polymerase.
Description:
As the main part of mercury bioabsorbant of our project, this device is designed to assemble three subparts----the T7promoter-rbs-Dsba-mbp-terminator, T7promoter-rbs-mbp-terminator and T7promoter-rbs-lpp-ompa-mbp-terminator (figure1), which can bind mercury separately in periplasm, cytosol and the membrane of E.coli, to make full use of the space and maximize the absorption of mercury. Since all these subparts are driven by their own T7 promotors, they can be expressed when T7polymerase exists.
1 Metal Binding Pepside(MBP)
MBP was designed as a single polypeptide that could fold into an antiparallel coiled coil. Previous work shows that artificial MBP chain still kept the in vivo metal-binding ability comparable to dimeric, full-length MerR, while it comprises less amino acids and will cost less for large-scale expression. Since our ultimate goal is to design a high-performance and less energy-consuming bioabsorbent, the MBP is an excellent candidate for the absorbent effector.The construction and structure of MBP are shown as followed.
2 Dsba-MBP
MBP was fused with DsbA, a periplasmic expression signal protein to construct periplasmic MBP.
3 Lpp-OmpA-MBP
Lpp-OmpA-mbp is designed as a fusion protein consisting of the signal sequence and first 9 amino acid of Lpp, residue 46~159 of OmpA and the metal binding peptide(MBP). The signal peptide of the N-termini of this fusion protein targets the protein on the membrane while the trans-membrane domain of Ompa serves as an anchor. MBP is on the externally exposed loops of OmpA, which can be anchored to the outer membrane.
4 assemble
To make full use of the spaces and reduce the energy consumption to the least while improve the efficiency of metal binding, we assemble the three genes together: mbp, Dsba-mbp, lpp-ompa-mbp, which are expressed separately in cytoplasm, periplasmic space and on the membrane.More information about this design will be given in the "Part design".
[ TOP ]
Part:BBa_K346005: Part Design
Design Notes
This part is designed to combine three subparts----the T7promoter-rbs-Dsba-mbp-terminator, T7promoter-rbs-mbp-terminator and T7promoter-rbs-lpp-ompa-mbp-terminator.
Metal binding pepside(MBP)
To achieve the goal of making a high performance MBP, we constructed a single polypeptide consisting of two dimerization helixes and metal binding loops of MerR, to form an antiparallel coiled coil MBP mimicking the dimerized metal binding domains of the wild-type as described in the Fig. We amplified the N-terminal and C-terminal of MBP directly from full length MerR by PCR, and then cloned them into the backbone together in one step. After that, RBS, T7 promoter and terminator are added.
Dsba-MBP
Dsba-mbp is a fusion protein aiming to transport the MBP protein to the periplasm. Dsba is a signal peptide, which can be recognized and transported to the periplasm.
LPP-OmpA-MBP
LPP-OmpA-MBP is designed as a fusion protein consisting of the signal sequence and first 9 amino acid of Lpp, residue 46~159 of OmpA and the metal binding peptide(MBP). The signal peptide of the N-termini of this fusion protein targets the protein on the membrane while the trans-membrane domain of Ompa serves as an anchor. MBP is on the externally exposed loops of OmpA, which can be anchored to the outer membrane.
Assemble
For the Lpp-OmpA-MBP which is displayed on the surface, previous work shows that over 20000 copies of mbps are expressed on the membrane per cell, with metal stoichiometries of ~1.0 Hg(II) per MBP monomer. But at the same time, the membrane protein consumes more energy. In contrast, the copy number of cytoplasmic MBP is 560000~800000 per cell, with a stoichiometries of ~0.455Hg(II) per MBP monomer at low concentration of Hg(II) and ~1.12±0.18 Hg(II) per MBP monomer at high concentration of Hg(II) and the energy needed to express the protein is less than the membrane protein. The data for Dsba-MBP, which is expressed in the periplasmic space is between MBP and Lpp-Ompa-MBP. However, given that our bioabsorbent is aimed to tackle the water with trace of Hg(II), we find us are trapped in the dilemma due to the contradiction between copy number of proteins and the absorption capacity if only one type of MBP is chosen. To make full advantage of the space in the cell and reduce the energy-assumption to the least, we assemble these three parts together, expecting MBP to express and play roles in cytoplasm, periplasm space and on the membrane simultaneously.
Source
MerR is from plasmid NR1
lpp-ompa-mbp is from plasmid PSD-MBD
both these two plasmids are offered by Anne O. Summers.
References
[1]Yamaguchi, K., Yu, F. & Inouye, M. (1988) Cell 53, 423-432.
[2]Francisco, J. A., Earhart, C. F. & Georgiou, G. (1992). Transport and anchoring of beta-lactamase to the external surface of Escherichia coli. Proc Natl Acad Sci U S A 89, 2713–2717.
[3]Francisco, J. A., Campbell, R., Iverson, B. L. & Georgiou, G. (1993). Production and fluorescence-activated cell sorting of Escherichia coli expressing a function antibody fragment on the external surface. ProcNatl Acad Sci U S A 90, 10444–10448
[4]Daugherty, P. S., Olsen, M. J., Iverson, B. L. & Georgiou, G. (1999).Development of an optimized expression system for the screening of antibody libraries displayed on the Escherichia coli surface. Protein Eng 12, 613–621.
[5]Song, L., Caguiat, J., Li, Z., Shokes, J., Scott, R. A., Olliff, L. &Summers, A. O. (2004). Engineered single-chain, antiparallel,coiled coil mimics the MerR metal binding site. J Bacteriol 186,1861–1868.
[6]Jie Qin,Lingyun Song,Hassan Brim, Michael J. Daly and Anne O. Summers(2006) Hg(II) sequestration and protection by the MerR metal-binding domain(MBD).Microbiology 15, 709–719
Mulligan, C. N., Yong, R. N. & Gibbs, B. F. Remediation technologies for metal-contaminated soils and groundwater: an evaluation. Eng. Geol. 60, 193-207 (2000).
Matlock, M. M., Henke, K. R. & Atwood, D. A. Effectiveness of commercial reagents for heavy metal removal from water with new insights for future chelate designs. J. Hazard Mater. B92, 129-142 (2002).
Gavrilescu, M. Removal of Heavy metals from the Environment by Biosorption. Eng. Life Sci. 4, 219-232 (2004).
Mejare, M. & Bulow, L. Metal-binding proteins and peptides in bioremediation and phytoremediation of heavy metals. Trends in biotechnol. 19, 67-73(2001).
Brown, N. L., Stoyanov, J. V. & Kidd, S. P. & Hobman, J. L. The MerR family of transcriptional regulators. FEMS Microbiol. Rev. 27, 145-163 (2003).
Shewchuk, L. M., Verdine, G. L., Nash, H. & Walsh, C.T. Mutagenesis of the cysteines in the metalloregulatory protein MerR indicates that a metal-bridged dimer activates transcription. Biochemistry 28, 6140-6145 (1989).
Changela, A., Chen, K., Xue, Y., Holschen, J., Outten, C. E., Halloran, T. V. & Mondrago, A. Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR. Science 301, 1383-1387 (2003).
Song, L., Caguiat, J., Li, Z., Shokes, J., Scott, R. A., Olliff, L. & Summers, A. O. Engineered Single-Chain, Antiparallel, Coiled Coil Mimics the MerR Metal Binding Site. J. Bacteriol. 186, 1861–1868 (2004).
Silver, S. & Phung, L. T. A bacterial view of the periodic table: genes and proteins for toxic inorganic ions. J. Ind. Microbiol. Biotechnol. 32, 587-605 (2005).
Barkay, T., Miller, S. M. & Summers, A. O. Bacterial mercury resistance from atoms to ecosystems. FEMS Microbiol. Rev. 27, 355-384 (2003).
Woude, M. W. & Henderson, I. R. Regulation and Function of Ag43 (Flu). Annu. Rev. Microbiol. 62, 153-169 (2008).
[ TOP ]
Part:BBa_K346005: Experience
All parts of our bioabsorbent are driven by T7 promoter, followed by an RBS in the upstream of the part. T7 polymerase is constitutively expressed with the constitutive promoter Ptet.It is notable that RBS B0030, a weaker ribosome binding site was used as to avoid the over expression of DsbA-MBP because it might saturate the co-translational transporter and inhibit the translocation of other proteins. To avoid leakage and interference between different parts, a strong terminator of BBa_B0015 is added to the downstream of each part. It is necessary to notice that DsbA-mbp has to be located on the upstream of other parts, for there is a PstI enzyme digestion site, resulting in the only way to digest DsbA-mbp with EcoRI and SpeI. To test the function of the device, both expression experiment and function test is necessary. As a result,the size of the expressed proteins have been verified with SDS-page and Western blot. Besides, to verify the efficiency of mercury binding, we also carried out the function test with ICP-AES, which can test the quantity of mercury binding by the bacteria with the device.
Results:
Expression of proteins
The Dsba-mbp, mbp and lpp-ompa-mbp are inserted into the commercial plasmid PET21A. Then the plasmid is transferred to E.coli strain BL21, which can generate T7polyerase when induced with IPTG. Both induced cells and uninduced cells(as control) are centrifuged to get the cytosol, the periplasm and the membrane separated. The SDS-page and Western blot of the expressed proteins in these three parts(figure2) show that induced cells expressed an identical IPTG-inducible protein at the proper place with the size of ~12kD for MBP, ~40kD for DsbA-MBP and ~27kD for LPP-OMPA-MBP, all of which are consist with the predicted size, indicating that all these three coding sequence can be expressed normally in the right place.
Figure 1: The result of SDS-PAGE and Western blotting, indicating the expression of MBP, DsbA-MBP, lpp-OmpA-MBP in different location.
Function test
We mainly used two methods to evaluate the capability and efficacy of our bioabsorbent:
For qualitative measurement, we invented a creative method to directly visualize the detoxification process of our bioabsorbent.  Firstly, we synthesized a water-soluble metal indicator, TritonX-100-PAN-S, according to the method in reference. Treated with pure sulfuric acid by stirring reaction overnight at room temperature, 1-(2-Pyridylazo)-2-naphthol (PAN) was sulfonated to make it water-soluble. (Scheme 2-1) After ether sedimentation and washing, we obtained the raw product, PAN-S, which was a reddish-brown solid powder. (Figure 2-2A) Then we mixed PAN-S and TritonX-100 by 1:20 mass ratio, and added ddH2O for thorough dissolving to get the final working solution, TritonX-100-PAN-S (TPS), with bright orange color. (Figure 2-2B)
Secondly, we conducted the characterization for our self-synthesized water-soluble metal indicator with expected colorimetric selectivity for heavy metals. We started with pH and metal concentration titration. By adding TPS into different pH solution at different mercury concentration, we found the proper color transition point at pH=~8. The lower limit of metal concentration for color transition was 0.8×10-5M. The color of solution changed from bright yellow to rosy color. (Figure 2-3)
Thirdly, we used our characterized indicator for direct visualization of mercury detoxification process. We constructed an integrated plasmid consisting of a constitutive promoter and metal binding peptides localized at different cell compartments. After transformation and overnight culturing, the bacteria were used to treat sample solution with 10-5M mercury and TPS at 37℃ for 10 min. The result was intriguing. Evident distinction could be observed between experiment group and control groups, which indicated the high efficacy of our bioabsorbent. (Figure 2-4)
For quantitative evaluation, we used inductively coupled plasma atomic emission spectrometry (ICP-AES) to precisely measure the different concentration of heavy metal before and after the treatment of bioabsorbent.
We constructed several plasmids for different localization of metal binding peptides and transformed them into BL21 (DE3) for expression, respectively. After approximately 40 hours of induction at the presence of different concentration of heavy metals, we collected the bacteria, washed with ddH2O for several times, and freeze-dried the pellet overnight before precise measuring their dry weights. Then we performed microwave digestion to completely dissolve all metal ions into solution before ICP-AES measurements. (Figure 2-5A, B)
The results are shown in the following figures:
These four parts are tested with the mercury concentration of 10^-5M to compare with each other, with the results shown in figure 4. It is necessary to point that that the device consisting of the three subparts seems to be less efficient than that of the surface display part Lpp-OmpA-MBP but it is better than the MBP in the cytoplasm and DsbA-MBP in the periplasm. This “unusual” phenomenon can be explained as that with the number of exogenous protein increases, the efficiency of expression of protein decreases quickly, for the hard burden due to these proteins.
Figure 3 && Figure 4The figure on the left showed the mercury binding capacity in different mercury concentration. We can see that the bacteria can absorbed more mercury in higher mercury concentration. But the bacteria had no significant mercury binding capacity when cultured with 0.1 uM mercury. But it is the detection limit of our biosensor. The figure on the right showed the mercury binding capacity of bacteria expressing differently localized MBPs. The surface displayed MBPs appear to have highest binding capacity while the pyramiding of MBP expression does not work well. This is because it is hard for the hydrophilic mercury ions to pass through the hydrophobic inner membrane.
[ TOP ]
Part:BBa_K346007
Contents
Part:BBa_K346007: main introduction
Part:BBa_K346007: Part Design
Part:BBa_K346007: main introduction
Introduction: Antigen 43 is a unique autotransporter that promote bacterial cell-to-cell aggregation. Antigen 43 can be expressed on the E.coli cell surface in large quantities, up to 50000 copies per cell.[1] The structure analysis of antigen 43 revealed that antigen 43 has an N-terminal signal peptide; an N-proximal passenger domain that is secreted, which could also be called α domain; an autochaperone domain that facilitates folding of the passenger domain; and a C-terminal β-barrel domain that forms an integral outer membrane protein, also called β domain[2]. The passenger domain(αdomain) confers the autoaggregation phenotype and it is bound to the surface via non-covalent interaction with the βdomain.
Figure 1: The structure of the coding sequence of antigen 43. Antigen 43 contains a signal peptide, a passenger domain, an autochaperone domain and a translocation unit, which are all indicated in this figure.
Antigen 43 mediated aggregation is a distinct phenotype that can be visualized macroscopically as flocculation and settling of cells in static liquid suspensions. Since we use E.coli to detect and absorb heavy metal ions, it is of great importance that we consider the disposal,which requires minimal effort to reclaim the bacteria, thus purifying the polluted water.
Materials and methods: 1: construction of standard plasmid with antigen 43 gene: in order to put antigen 43 gene into standard plasmid, we first cloned it from the genome of K12 strain of E.Coli using nest PCR procedure. Primers were designed using Primer Premier 5 and with standard digest sites, EcoRI,XbaI forward and SpeI, PstI reverse. This gene showed an apparent 3K band when electrophoresis in 1% agarose gel in both two steps of PCR. This gene is then put into standard plasmid by standard digestion.
Figure 2：the gel image of antigen 43. The middle is Trans2K DNA marker. Antigen 43 is about 3k large.
2: construction of the expression plasmid of antigen 43: in order to express antigen 43, we put in downstream the delay and amplify part which are mentioned before, which is constructed by PhiR73 and Po promoter. This PhiR73 is under T7 promoter, so antigen 43 gene is indirectly controlled by T7 promoter. We then put this plasmid into BL21 strains, and test the function of antigen 43 by IPTG induction.
Figure 3: the clone plasmid that can achieve the function of antigen 43. Green circle indicate the ribosome binding sites. Red hexagon indicates the terminator. PhiR73 and Po promoter are used as a delay and amplify module.
3：function analysis of antigen 43: Because antigen 43 can cause auto-aggregation, we adapted a well-established auto-aggregation assay to test the function of antigen 43[4]. This assay is to follow the bacterial settling kinetics over time. BL21 strains were overnight cultured and were diluted 1:100 in 50ml LB and then grown to OD600=0.4-0.6., at which point expression of antigen 43 was induced by the addition 0.001% IPTG. The cultures were grown to a final OD600=1(standardized) and 5000rp for 5min, then use 1% PBS which 0.15mM NaCl was added to re-suspense it. this cultures were vigorously shaken before experiment. At regular time interval, a 100ul samples was  taken approximately 0.5cm from the surface and transferred into a microplate maintained on ice. Every 10 minutes 100ul samples was taken with same method. At the end of the experiment, OD600 were measured using microplate reader. at last the plot was drawn by OD600 with time.
Result：
We induced the expression of antigen 43 by adding IPTG. After 4 hours we put the tube on the table to see the auto-aggregation phenomenon. After 20 minutes there is significant aggregation which can be easily observed by eyes. We then took photograph to show this phenomenon as figure 4.
Figure 4: Autoaggregation caused by antigen 43. The  right one is strain expressing antigen 43, which shows significant autoaggregation.
Then the result of the auto-aggregation assay is shown in figure 5. The strain that did not add IPTG also shows significant drop in OD600. We propose this is because the leakage expression of T7 polymerase in BL21 strains and due to the amplifier the expression of antigen 43 is in great amount.
FIgure 5: The OD of the supernatant to time. There is a significant drop of OD after nearly 20 minutes in those strains that expressing antigen 43, and those do not express antigen 43 show constant OD of the supernatant.
Reference
1.Marjan W.van der Woude & Ian R.Henderson(2008).Annu.Rev.Microbiol.62:153-169.
2.Kristian K & Henrik H(2002).Journal of Bacteriology.184.15.4197-4204.
3.Ian R.& Herderson(1997).FEMS Microbiology Letters.149.115-120.
4.Glen C.Ulett & Richard I.W(2006).Microbiology.152,2101-2110.
[ TOP ]
Part:BBa_K346007: Part Design
'Design Notes
Since the original antigen 43's RBS is regulated by intracellular factors such as the oxidative particles, and in our bioabsorbant kit we need to express some proteins such as DsbA which can affect the oxidative condition in the cell, it is unconvenient for us to use antigen 43. We therefore replace the RBS by a standard RBS B0034 which is not regulated by so many factors in the cell. By this design, we are able to express antigen 43 without considering the condition within the cell.
By using nest PCR, we were able to get the coding region of antigen 43 by two step PCR. We designed the primer using Primer Premier 5.0 and add standart digestion sites into them.
Source
K12 strain of E.Coli
[ TOP ]
Retrieved from " http://2010.igem.org/Team:Peking/Parts/Characterization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
