<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Peking_Project_Bioabsorbent_MBPExpression"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Peking/Project/Bioabsorbent/MBPExpression">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Peking/Project/Bioabsorbent/MBPExpression&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking/Project/Bioabsorbent/MBPExpression&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Peking/Project/Bioabsorbent/MBPExpression&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Peking/Project/Bioabsorbent/MBPExpression" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Peking/Project/Bioabsorbent/MBPExpression</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="Layer1"><TABLE width="550" height="36" border="0"><TBODY><TR><TD width="46"><DIV align="center"><A href="https://2010.igem.org/Team:Peking"><FONT color="#ffffff">HOME</FONT></A></DIV></TD><TD width="41" onmouseover="MM_showHideLayers('Layer2','','hide');MM_showHideLayers('Layer5','','show')" onmouseout="MM_showHideLayers('Layer5','','show');MM_showHideLayers('Layer5','','hide')" onfocus="MM_showHideLayers('Layer5','','hide')"><DIV align="center" onmouseover="MM_showHideLayers('Layer5','','show');MM_showHideLayers('Layer2','','hide')"><A href="https://2010.igem.org/Team:Peking/Team"><FONT color="#ffffff">TEAM</FONT></A></DIV></TD><TD width="64" onmouseover="MM_showHideLayers('Layer2','','show')" onmouseout="MM_showHideLayers('Layer2','','hide')" onfocus="MM_showHideLayers('Layer2','','hide')"><DIV align="center" onmouseover="MM_showHideLayers('Layer5','','hide')"><A href="https://2010.igem.org/Team:Peking/Project"><FONT color="#ffffff">PROJECT</FONT></A></DIV></TD><TD width="49" onmouseover="MM_showHideLayers('Layer2','','hide');MM_showHideLayers('Layer4','','show')" onmouseout="MM_showHideLayers('Layer4','','hide')"><DIV align="center" onmouseover="MM_showHideLayers('Layer6','','hide')"><A href="https://2010.igem.org/Team:Peking/Parts"><FONT color="#ffffff">PARTS</FONT></A></DIV></TD><TD width="80" onmouseover="MM_showHideLayers('Layer3','','hide');MM_showHideLayers('Layer4','','hide');MM_showHideLayers('Layer6','','show')" onfocus="MM_showHideLayers('Layer6','','show')"><DIV align="center" onmouseover="MM_showHideLayers('Layer6','','show')"><A href="https://2010.igem.org/Team:Peking/Modeling"><FONT color="#ffffff">MODELING</FONT></A></DIV></TD><TD width="77" onmouseover="MM_showHideLayers('Layer3','','show')" onmouseout="MM_showHideLayers('Layer3','','hide')"><DIV align="center" onmouseover="MM_showHideLayers('Layer6','','hide')"><A href="https://2010.igem.org/Team:Peking/Notebook"><FONT color="#ffffff">NOTEBOOK</FONT></A></DIV></TD><TD width="78" onmouseover="MM_showHideLayers('apDiv1','','show')" onmouseout="MM_showHideLayers('apDiv1','','hide')"><DIV align="center"><A href="https://2010.igem.org/Team:Peking/Humanpractice"><FONT color="#ffffff">HUMAN PRACTICE </FONT></A></DIV></TD><TD width="81"><DIV align="center"><A href="https://2010.igem.org/Team:Peking/Biosafety"><FONT color="#ffffff">BIOSAFETY</FONT></A></DIV></TD></TR></TBODY></TABLE></DIV><DIV id="Layer2" onmouseover="MM_showHideLayers('Layer2','','show')" onmouseout="MM_showHideLayers('Layer2','','hide')"><TABLE width="200" height="150" border="0"><TBODY><TR><TD height="22"><A href="https://2010.igem.org/Team:Peking/Project">  Project Home</A></TD></TR><TR><TD height="22"><A href="https://2010.igem.org/Team:Peking/Project/ProjectDiscription">  Project Discription</A></TD></TR><TR><TD height="23"><A href="https://2010.igem.org/Team:Peking/Project/Biosensor">  Biosensor</A></TD></TR><TR><TD height="15"><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent">  Bioabsorbent</A></TD></TR><TR><TD height="15"><A href="https://2010.igem.org/Team:Peking/Project/Application">  Application</A></TD></TR><TR><TD height="15"><A href="https://2010.igem.org/Team:Peking/Project/Expansion">  Expansion</A></TD></TR></TBODY></TABLE></DIV><DIV id="Layer3" onmouseover="MM_showHideLayers('Layer3','','show')" onmouseout="MM_showHideLayers('Layer3','','hide')"><TABLE width="172" height="77" border="0"><TBODY><TR><TD height="21">  <A href="https://2010.igem.org/Team:Peking/Notebook">Personal Notes</A></TD></TR><TR><TD height="21"><A href="https://2010.igem.org/Team:Peking/Notebook/Protocols">  Protocols</A></TD></TR><TR><TD height="21">  <A href="https://2010.igem.org/Team:Peking/Notebook/Vocabulary">Vocabulary</A></TD></TR></TBODY></TABLE></DIV><DIV id="Layer4" onmouseover="MM_showHideLayers('Layer4','','show')" onmouseout="MM_showHideLayers('Layer4','','hide')"><TABLE width="245" height="70" border="0"><TBODY><TR><TD height="15"> <A href="https://2010.igem.org/Team:Peking/Parts"> Parts Submitted to the Registry</A></TD></TR><TR><TD height="15">  <A href="https://2010.igem.org/Team:Peking/Parts/Characterization">Characterization</A></TD></TR><TR><TD height="15">  <A href="https://2010.igem.org/Team:Peking/Parts/Existed">Existed Parts' Improvement</A></TD></TR></TBODY></TABLE></DIV><DIV id="Layer5" onfocus="MM_showHideLayers('Layer5','','hide')"><TABLE width="200" border="0" onmouseover="MM_showHideLayers('Layer5','','show')" onmouseout="MM_showHideLayers('Layer5','','hide')"><TBODY><TR><TD> <A href="https://2010.igem.org/Team:Peking/Team"> Students and Instructors</A></TD></TR><TR><TD> <A href="http://english.pku.edu.cn/" target="_blank"> Peking University</A></TD></TR><TR><TD> <A href="https://2010.igem.org/Team:Peking/Team/Collaboration"> Collaboration</A></TD></TR><TR><TD> <A href="https://2010.igem.org/Team:Peking/Team/Gallery"> Gallery</A></TD></TR><TR><TD> <A href="https://2010.igem.org/Team:Peking/Team/Acknowledgement"> Acknowledgement</A></TD></TR><TR><TD> <A href="https://2010.igem.org/Team:Peking/Team/PreviousYears"> Previous Years</A></TD></TR></TBODY></TABLE></DIV><DIV id="Layer6" onmouseover="MM_showHideLayers('Layer6','','show')" onmouseout="MM_showHideLayers('Layer6','','hide')"><TABLE width="200" height="105" border="0"><TBODY><TR><TD height="26"> <A href="https://2010.igem.org/Team:Peking/Modeling"> Modelling Home</A></TD></TR><TR><TD height="24"> <A href="https://2010.igem.org/Team:Peking/Modeling/CalculationProcess"> Calculating Process</A></TD></TR><TR><TD height="22"> <A href="https://2010.igem.org/Team:Peking/Modeling/Analysis"> Analyses and Results</A></TD></TR><TR><TD height="22"> <A href="https://2010.igem.org/Team:Peking/Modeling/Advanced_Model"> Advanced Model</A></TD></TR></TBODY></TABLE></DIV><DIV id="apDiv1" onmouseover="MM_showHideLayers('apDiv1','','show')" onmouseout="MM_showHideLayers('apDiv1','','hide')"><TABLE width="186" border="0"><TBODY><TR><TD height="23"> <A href="https://2010.igem.org/Team:Peking/Humanpractice"> HP Home</A></TD></TR><TR><TD height="23"> <A href="https://2010.igem.org/Team:Peking/Humanpractice/HGTReview"> HGT Review</A></TD></TR><TR><TD height="23"> <A href="https://2010.igem.org/Team:Peking/Humanpractice/PC"> Proof of Concept</A></TD></TR><TR><TD height="23"> <A href="https://2010.igem.org/Team:Peking/Humanpractice/HGTInvestigation"> HGT Investigation</A></TD></TR><TR><TD height="23"> <A href="https://2010.igem.org/Team:Peking/Humanpractice/Acknowledgement"> Acknowledgement</A></TD></TR></TBODY></TABLE></DIV><DIV id="lsidebar"><P><FONT size="4"><FONT color="#585858"><FONT face="Franklin Gothic Demi Cond">Overview of the Heavy Metal Decontamination Kit</FONT></FONT></FONT><TABLE align="left" cellspacing="0" cellpadding="0"><TBODY><TR><TD><UL id="menu"><LI>  Biosensor
    </LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Biosensor">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Biosensor/PromoterCharacterization">Promoter Characterization</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Biosensor/OperationCharacterization"><FONT size="2">Operation Characterization</FONT></A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Biosensor/Bioreporter">Bioreporter</A></LI><LI>  Bioabsorbent
    </LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent/MBPconstruction">MBP construction</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent/MBPExpression">MBP Expression</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent/FacilitationModule">Facilitation Module</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent/InductiveAggregation">Inductive Aggregation</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent/ModuleAssembly">Module Assembly</A></LI><LI>  Application
    </LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Application/PreservationOptimization">Preservation Optimization</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Application/KitOperation">Kit Operation</A></LI><LI>  Expansion
    </LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Expansion">Introduction</A></LI><LI><A href="https://2010.igem.org/Team:Peking/Project/Expansion/LeadBioabsorbent">Lead Bioabsorbent</A></LI></UL></TD></TR></TBODY></TABLE></P><P><FONT size="5"><FONT color="#585858"><FONT face="Franklin Gothic Demi Cond">CountDown</FONT></FONT></FONT></P><P><FONT size="5"><FONT color="#585858"><FONT face="Franklin Gothic Demi Cond">Vistors From...</FONT></FONT></FONT></P></DIV></DIV><DIV id="twhite"><FONT size="5"><FONT color="000"><FONT face="Franklin Gothic Demi Cond">   MBP Expression</FONT></FONT></FONT>        <A href="/Team:Peking/Project" title="Team:Peking/Project">Project</A>&gt;<A href="/Team:Peking/Project/Bioabsorbent" title="Team:Peking/Project/Bioabsorbent">Bioabsorbent</A>&gt;<STRONG class="selflink">MBP Expression</STRONG></DIV><DIV id="middlewhite">

As was mentioned above, in order to implement the mercury-binding function in bacteria with as least cost as possible, we constructed a single polypeptide consisting of two repeats of MerR with a flexible linker in between (<A href="/Team:Peking/Project/Bioabsorbent/MBPconstruction" title="Team:Peking/Project/Bioabsorbent/MBPconstruction">Details can be seen in the MBP Construction Part</A>).
<P>Since our ultimate goal is to design a high-performance and less energy-consuming bioabsorbent, the MBP will be an excellent candidate for the absorbent effector. MBP was then fused with DsbA, a periplasmic translocated protein with a signal sequence at its N-terminal, and OmpA, a membrane protein, to construct periplasmic MBP and surface displayed MBP, in order to maximize the bacterial capability of mercury binding. Finally, mercury MBP was constitutively expressed on surface, periplasm and cytosol of E.coli cells via carefully designed genetic circuits, to guarantee the maximum of Hg absorption (Fig 1). 
</P><P><B>Fig 1. The genetic circuit we designed to guarantee the maximum of Hg absorption. The production of T7 RNA polymerase is constitutive. T7 polymerases will active high rating transcription at T7 promoters. Thus Hg (II) will be highly effectively accumulated by substantial amount of MBPs which are translocated to cytosol, periplasm and cell surface of the bacteria.</B></P><H2><SPAN class="mw-headline" id="Cytoplasmic_Expression_of_MBP">Cytoplasmic Expression of MBP</SPAN></H2><P>To design the module of cytosol expression, MBP in pSB3K3 was then to coloned into pSB1A2, prefixed by T7 promoter and RBS BBa_B0034 (Fig 2), verified by DNA sequencing. 
</P><P><B>Fig 2. Construction of the cytosol expression module. MBP was cloned into the pSB1A2 step by step, prefixed by T7 promoter and RBS.</B></P><P>The resulting construct, T7 promoter + BBa_B0034+ MBP was transformed into BL21 (DE3) competent cells. The expression of MBP has been verified by SDS-PAGE and western blotting. (Fig 3) 
</P><P><B>Fig 3. The expression of MBP has been verified by SDS-PAGE and Western Blotting. IPTG was used as the inducer. (A) Overexpression band at about 10 KD was detected. (B) Positive band of the expected molecular weight could be detected in the cytosol</B></P><P>We detected overexpression band at about 10 kD, which was likely to be MBP. Then we fused a his-tag to our target protein and conducted western blotting to further verify their expression. Positive band of the expected molecular weight could be detected in the cytosol, which confirmed expression and localization of the target protein. We can also indicate from the SDS-PAGE that there are indeed a large number of MBPs existing in cytosol, ready to bind mercury ions.
</P><H2><SPAN class="mw-headline" id="Periplasmic_Translocation_of_MBP">Periplasmic Translocation of MBP</SPAN></H2><P>Though contributing to the removal of Hg2+, over-expression of the mercury-binding protein may be inefficient because of its limitation of mercury uptake . Then we pay our attention to the translocation of MBP to the periplasm or surface of the bacteria, a promising strategy that not only eliminates the limitation of the capacity of accumulating Hg2+  but also makes full use of the spaces in the bacteria besides cytosolic MBPs, thus increasing the speed of absorbability and removal. 
</P><P>DsbA, the commonly used signal protein which can export proteins fused to it into the periplasmic space, was selected as the candidates for the periplasmic fusion MBPs. According to previous work, DsbA, a SecA dependent signal protein , can export its C-terminal fusion protein into periplasm on the signal recognition particle (SRP) pathway , which is made up of six proteins and an RNA molecule and directs rapid co-translational translocation of many proteins .Since some protein with a rapid protein folding pathway often assembles into its stable three-dimensional structure before it has a chance to be exported, Maltose Binding Protein thus inevitably suffers from its inefficient posttranslational export . In contrast, DsbA perfectly bypasses such problem due to its cotranslational translocation that obviates the inhibitory effect of protein folding on exportation. Hence DsbA overmatches Maltose Binding Protein with a more efficient and rapid way to export the target protein (for example, the metal binding peptide) to the periplasmic space, making it the best choice for the periplasmic design (Fig. 4)
</P><P><B>Fig 4. Result of 3D modeling for DsbA-MBP Fusion Protein. DsbA is translocated into periplasm by the co-translational pathway, which is friendly for protein folding.</B></P><P>Then a genetic circuit was designed as was shown in Fig 4. Bacteria will express large copy number of DsbA-MBPs and finally they will fill up the periplasmic space. In addition, RBS B0030, a weaker ribosome binding site was used as to avoid the overexpression of DsbA-MBP because it might saturate the co-translational transporter and inhibit the translocation of other proteins. 
</P><P>As for the standardization of the periplasmic translocation module, the entire coding region of DsbA and MBP was cloned into pSB1K3 with standard restriction enzyme sites. Particularly, the PstI restriction site inside DsbA was mutated synonymously. 
</P><P><B>Fig 5. Procedure of DsbA-MBP construction.</B></P><P>As to construct the fusion of DsbA-MBP, a commercial plasmid, pET-39b (+), which contains the gene encoding DsbA, was used as the backbone. The entire coding region of the MBP was amplified by PCR from full length MerR with two pairs of primers. The two PCR products were digested with Xba I / BamH I, or BamH I / Xho I, followed by cloning these 2 fragments into Spe I / Xho I digested pET-39b(+) in one step (Fig. 5) to construct pET-39b(+)-DsbA-MBP.
</P><P><B>Fig 6. Standardization procedure of DsbA-MBP.</B></P><P>DsbA and MBD gene was amplified by PCR from pET-39b (+)-DsbA-MBP, with the primer containing T7 promoter, RBS and SD restriction sites, as shown in Fig 6. The PCR product was digested with EcoR I / Pst I and then cloned into EcoR I / Pst I double digested pSB1K3, to achieve the goal of standardization of the fusion protein (Fig. 6).
</P><H3><SPAN class="mw-headline" id="RESULT">RESULT</SPAN></H3><P>Periplasmic Expression DsbA-MBPs was verified by SDS-PAGE and western blotting (Fig 7).
</P><P><B>Fig 7. Result of SDS-PAGE and Western blotting</B></P><P>Over-expression band in SDS-PAGE result and bands specific to his-tag in western blot result were found at about 40 kD. The presence of these bands confirmed that we have got the expected protein expressed in periplasm. But there are large amount of proteins detected in lysate pellet, this was because of the high intensity of T7 promoter. When induced with 0.5 mM IPTG, our target protein were expressed so rapidly that some of them cannot fold properly and accumulated in the inclusion bodies. This problem can be solved by using lower IPTG concentration, lower induction temperature or shorter induction time.
</P><P><B>Fig 8. Lpp-OmpA-MBP was designed as a fusion protein consisting of the signal sequence and first 9 amino acid of Lpp, residue 46~159 of OmpA and the metal binding peptide (MBP). The signal peptide of the N-termini of this fusion protein targets the protein to the membrane while the transmembrane domain of OmpA serves as an anchor. MBP is on the externally exposed loops of OmpA, which can be anchored to the outer membrane. </B></P><P>Lpp-OmpA(46-159) fusions were proved to be an effective surface display system to anchor a variety of proteins such as ,B-lactamase [6], a cellulose binding protein, alkaline phosphatase and a single chain Fv antibody fragment(scFv)[7] on the external surface of E. coli. For OmpA, targeting and correct assembly into the outer membrane appear to be distinct events. Extensive studies on it suggest that the region between residue 154 and 180 is crucial for localization on the membrane, while the region between residue 46 and 159 ,the fragment constitute our fusion protein, contains five transmembrane β-strands of an eight-stranded β-barrel, providing an sufficient region to assemble into the membrane[6].
</P><P>Additionally, a signal peptide is also needed to mediate the proper localization of the fusion protein on the outer membrane. The N-termini of the outer membrane lipoprotein (Lpp) can serve as the signal peptide, which is considered to be firmly associated with the membrane due to its extreme hydrophobicity [6]. To conclude this, the fusion protein of Lpp-OmpA , with their target and assembly function, can serve as a powerful tool in surface display [9].
</P><H3><SPAN class="mw-headline" id="CONSTRUCTION">CONSTRUCTION</SPAN></H3><P>We constructed a fusion protein consisting of the signal sequence and first 9 amino acid of Lpp, residue 46~159 of OmpA and our metal binding peptide (MBP). The signal peptide of the N-termini of this fusion protein targets the protein to the membrane while the transmembrane domain of OmpA serves as an anchor. MBP is on the externally exposed loops of OmpA. 
To test the expression of the fusion protein and the function, it was essential to construct both the standard plasmid and commercial plasmid. 
</P><P>Firstly, we used PCR to obtain Lpp-OmpA gene from the template of PSD-MBD which was provided by Anne O Summers as a gift. The primers were designed based the literature (Fig 9). And the MBP gene was also obtained by PCR. Then the two fragment were cut with EcoRI(or NdeI), SalI and SalI, PstI (or XhoI), respectively, as inserts and the standard plasmid PSB1A2 was cut with EcoRI and PstI(for the commercial plasmid PET21a, the enzymes were NdeI and XhoI) as the vector. Ultimately, three fragments (Lpp-OmpA, MBP and the vector) were combined to get the standard and commercial plasmid bearing Lpp-OmpA-MBP, as is shown in Fig 9 and Fig 10.
</P><P><B>Fig 9. Procedures of the construction of standard plasmid. </B></P><P><B>Fig 10. Procedures of Construction of Commercial Plasmid. </B></P><P>After the construction of the plasmid with the fusion protein gene Lpp-OmpA-MBP, We prefixed T7 promoter and BBa_B0030 upstream of Lpp-OmpA-MBP. Additionally, a strong terminator BBa_B0015 was suffixed. 
</P><P><B>Fig 11. Results of SDS-PAGE and Western Blotting. The overexpression band in SDS-PAGE and specific band in western blotting comfirmed the correct expression and localization of our fusion protein. Specific band cannot be detected in the cytosol which indicates the excellent translocation efficiency of OmpA.</B></P><P><B>Fig 12. Overview of various localization of MBP. </B></P><P>In summary, as is shown in Fig 12, MBP will be translocated in three different compartments in cell, directly into cytosol, into periplasm with the help of DsbA, and display on the surface using OmpA fusion. We would also like to compare the effects to evaluate the different efficacy of these modules, as is shown below.
</P><P>We mainly used two methods to evaluate the capability and efficacy of our bioabsorbent:
</P><P>For qualitative measurement, we invented a creative method to directly visualize the detoxification process of our bioabsorbent.
</P><P>Firstly, we synthesized a water-soluble metal indicator, TritonX-100-PAN-S, according to the method in reference. [10] Treated with pure sulfuric acid by stirring reaction overnight at room temperature, 1-(2-Pyridylazo)-2-naphthol (PAN) was sulfonated to make it water-soluble. (Scheme 1) After ether sedimentation and washing, we obtained the raw product, PAN-S, which was a reddish-brown solid powder. (Figure 13A) Then we mixed PAN-S and TritonX-100 by 1:20 mass ratio, and added ddH2O for thorough dissolving to get the final working solution, TritonX-100-PAN-S (TPS), with bright orange color. (Figure 13B)[11]
</P><P><B>Scheme 1. Synthesis of PAN-S.</B></P><P><B>Fig 13. A: PAN-S as reddish-brown solid powder. B: TritonX-100-PAN-S (TPS) as bright orange solution. </B></P><P>
Secondly, we conducted the characterization for our self-synthesized water-soluble metal indicator with expected colorimetric selectivity for heavy metals. We started with pH and metal concentration titration. By adding TPS into different pH solution at different mercury concentration, we found the proper color transition point at pH=~8. The lower limit of metal concentration for color transition was 0.8×10-5M. The color of solution changed from bright yellow to rosy color. (Figure 14)
</P><P><B>Fig 14. Evident color change of TPS upon addition of mercury. A: Before addition of mercury.   B: After addition of mercury.</B></P><P>
Thirdly, we used our characterized indicator for direct visualization of mercury detoxification process. We constructed an integrated plasmid consisting of a constitutive promoter and metal binding peptides localized at different cell compartments. After transformation and overnight culturing, the bacteria were used to treat sample solution with 10-5M mercury and TPS at 37℃ for 10 min. The result was intriguing. Evident distinction could be observed between experiment group and control groups, which indicated the high efficacy of our bioabsorbent. (Figure 15)
</P><P><B>Fig 15. Direct visualization of mercury detoxification process.  A: 500uL HEPES buffer +10uL TPS; B: 500uL HEPES buffer +10uL TPS +10uM mercury + bacteria bioabsorbent; C: 500uL HEPES buffer +10uL TPS +10uM mercury. Three groups were shaking cultured at 37℃ for 10 min and followed by centrifugation.</B></P><P>
For quantitative evaluation, we used inductively coupled plasma atomic emission spectrometry (ICP-AES) to precisely measure the different concentration of heavy metal before and after the treatment of bioabsorbent.
We constructed several plasmids for different localization of metal binding peptides and transformed them into BL21 (DE3) for expression, respectively. After approximately 40 hours of induction at the presence of different concentration of heavy metals, we collected the bacteria, washed with ddH2O for several times, and freeze-dried the pellet overnight before precise measuring their dry weights. Then we performed microwave digestion to completely dissolve all metal ions into solution before ICP-AES measurements. (Figure 16A, B) 
</P><P><B>Fig 16. A: High Throughput Closed Microwave Digestion System (Mars, USA).  B: Leeman Labs Profile inductively coupled plasma (ICP) spectrometer (Leeman, USA).</B></P><P>
The results are shown in the following figures:
</P><P><B>Fig 17. Different amount of mercury absorbed by bacteria with integrated plasmid consisting of MBP, DsbA-MBP and Lpp-OmpA-MBP (MDL) cultured with different Hg(II) concentration. </B></P><P><B>Fig 18. Different amount of mercury absorbed by bacteria with MBP expressed in different subcellular compartments cultured for ~40h in 10-5 mol/L Hg (II) medium.</B></P><H2><SPAN class="mw-headline" id="Reference">Reference</SPAN></H2><P>1.Chen, S., and D. B. Wilson. 1997. Construction and characterization of Escherichia coli genetically engineered for bioremediation of Hg2-contaminated environments. Appl. Environ. Microbiol. 63:2442–2445.
</P><P>2.Chen, S., and D. B. Wilson. 1997. Genetic engineering of bacteria and their potential for Hg2 bioremediation. Biodegradation 8:97–103.
</P><P>3.Chen, S., and D. B. Wilson. 1997. Construction and characterization of Escherichia coli genetically engineered for bioremediation of Hg2-contaminated environments. Appl. Environ. Microbiol. 63:2442–2445.
</P><P>4.Luirink, J., and B. Dobberstein. 1994. Mammalian and Escherichia coli signal recognition particles. Mol. Microbiol. 11:9–13.
</P><P>5.Debarbieux, L., and J. Beckwith. 1998. The reductive enzyme thioredoxin acts as an oxidant when it is exported to the Escherichia coli periplasm. Proc.Natl. Acad. Sci. USA. 95:10751–10756.
</P><P>6.Francisco, J. A., Earhart, C. F. &amp; Georgiou, G. (1992). Transport and anchoring of beta-lactamase to the external surface of Escherichia coli. Proc Natl Acad Sci U S A 89, 2713–2717.
</P><P>7.Francisco, J. A., Campbell, R., Iverson, B. L. &amp; Georgiou, G. (1993). Production and fluorescence-activated cell sorting of Escherichia coli expressing a function antibody fragment on the external surface. Proc Natl Acad Sci U S A 90, 10444–10448
</P><P>8.Yamaguchi, K., Yu, F. &amp; Inouye, M. (1988) Cell 53, 423-432.
</P><P>9.Jie Qin,Lingyun Song,Hassan Brim, Michael J. Daly and Anne O. Summers(2006) Hg(II) sequestration and protection by the MerR metal-binding domain(MBD).Microbiology 15, 709–719.
</P><P>10.Jingping Wang, Zhongming Wu.(2004) Simultaneous titration determination of mercury and lead with 1-(2-Pyridylazo)-2-naphthol sulphonic acid as a complexometric indicator. Chinese Journal of Analysis Laboratory 23, 60-62.
</P><P>11.Min Li, Guojun Lian.(2004) The effect of TritonX-100-PAN-S as a complexometric titration indicator in determining copper. Guangdong Trace Element Science 11, 56-58.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Peking/Project/Bioabsorbent/MBPExpression">http://2010.igem.org/Team:Peking/Project/Bioabsorbent/MBPExpression</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Peking/Project/Bioabsorbent/MBPExpression" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Peking/Project/Bioabsorbent/MBPExpression" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Peking/Project/Bioabsorbent/MBPExpression&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Peking/Project/Bioabsorbent/MBPExpression&amp;oldid=193934" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>