"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/Bioabsorbent
From 2010.igem.org
HOME
TEAM
PROJECT
PARTS
MODELING
NOTEBOOK
HUMAN PRACTICE
BIOSAFETY
  Project Home
  Project Discription
  Biosensor
  Bioabsorbent
  Application
  Expansion
   Personal Notes
  Protocols
   Vocabulary
   Parts Submitted to the Registry
   Characterization
   Existed Parts' Improvement
   Students and Instructors
   Peking University
   Collaboration
   Gallery
   Acknowledgement
   Previous Years
   Modelling Home
   Calculating Process
   Analyses and Results
   Advanced Model
   HP Home
   HGT Review
   Proof of Concept
   HGT Investigation
   Acknowledgement
Overview of the Heavy Metal Decontamination Kit
  Biosensor
Introduction
Promoter Characterization
Operation Characterization
Bioreporter
  Bioabsorbent
Introduction
MBP construction
MBP Expression
Facilitation Module
Inductive Aggregation
Module Assembly
  Application
Preservation Optimization
Kit Operation
  Expansion
Introduction
Lead Bioabsorbent
CountDown
Vistors From...
   BIOABSORBENT
        
              *MBP construction
              *MBP Expression
              *Facilitation Module
              *Inductive Aggregation
              *Module Assembly
   INTRODUCTION
         Project >Bioabsorbent
Bioabsorbent Introduction
    As mentioned before, heavy metals including mercury and lead can cause significant damage to the environment and human health as a result of their mobilities and solubilities[1]. Present technology including immobilization and extraction relies upon chemical precipitation or chelating agents[2], both of which are commercially impractical, lack of selectivity or sensitivity and not eco-friendly[3]. High performance and inexpensive methods to decontaminate heavy metals in water body, therefore, are highly desired.
    There are natural proteins, which can immobilize heavy metals to reduce the toxicity, exist in eukaryote and prokaryote. Most of them are metallothioneins[4] with cysteines acting as the binding sites. MerR family transcription factors are among of them. As was discussed, MerR family is a group of transcriptional activators that can bind different metal (or drug) specifically at extremely low concentration. The overall structure of the family is a combination of the N-terminal DNA binding domain and C-terminal metal binding domain; and homodimer is their active form[5]. It should be noticed that MerR family TFs share a highly conserved homology at the C-terminal metal binding domain (Fig. 1), which indicates a similar metal recognition mechanism and metal-protein complex structure.
Figure 1. Structure based sequence similarity of various metal responsive regulators of MerR family. CadR: cadmium, CueR: copper, ZntR: zinc, MerR: mercury, PbrR:lead. Those amino acids showed in bold are metal binding sites according to solved crystal structures [6] or mutagenesis analysis [7]. The metal binding site of CadR or PbrR can be speculated based on the high similarity of MerR family and the geometry of ligand field of metal ions.
    Grounded on this concept, we conducted a 3D structure modeling for MerR according the structure data of CueR, a homolog of MerR (Fig.2). Results demonstrated that the metal binding domain and DNA binding domain were probably modular for each other, which means that metal binding was independent of DNA binding. We directly tandemed two metal binding domain of MerR including dimerization helix and metal binding loop together, to make a mercury metal binding peptide ( MBP ) (fig. 3).
Figure 2. The results of 3D structure modeling for MerR based on data of CueR. (A) Crystal structure of CueR[ ], which represent the overall structure of the MerR family. The ribbon diagram depicts one monomer in color and the other in gray, with DNA-binding domain in blue, the dimerization helix in red, the metal binding domain in purple and the metal ion in cyan. (B) The predicted three dimensional conformation of Hg(II) bound MerR structure by our 3D modeling. Two monomers are indicated in green and cyan, respectively. (C) and (D) were obtained after rotation of the structure in (B) by 90° about the z-axis and x- axis, respectively. The C-terminal metal binding domain and the N-terminal DNA binding domain are likely to be modular for each other. (To Read More…)
    Previous work shows that artificial MBP chain still kept the in vivo metal-binding ability comparable to dimeric, full-length MerR[8], while it comprises less amino acids and will cost less for large-scale expression. Since our ultimate goal is to design a high-performance and less energy-consuming bioabsorbent, the MBP will be an excellent candidate for the absorbent effector. MBP was then fused with DsbA, a periplasmic expression signal protein, and OmpA, a membrane protein, to construct periplasmic MBP and surface displayed MBP, in order to maximize the bacterial capability of mercury binding. Finally, mercury MBP was constitutively expressed on surface, periplasm and cytosol of E.coli cells via carefully designed genetic circuits, to guarantee the maximum of Hg absorption.
Figure 3. Scheme of MBP construction and the predicted MBP structure. (A) Linear structure of MerR coding region. (B) Construction of MBP. The MBP was constructed by fusing two copies of metal binding domain of MerR in tandem with a flexible SSG bridge. Blue bars indicate the dimerization helix in the metal binding domain of MerR, and gray bars indicate other α-helices of MerR. The red line indicates the SSG linker, and the blue and green lines indicate the loop after the dimerization loop and the region after the loop, respectively. Orange dots indicate cysteines involved in Hg(II) binding[8]. (C) Predicted structure of resulted metal binding peptide. Mercury ions are indicated as black balls in metal binding pockets. (To Read More…)
    Hg (II) ions can enter the bacteria cell by diffusion through the outer membrane [9], however, the hydrophobic inner membrane prevents them from passing through the inner membrane. To guarantee the binding efficiency of our bioabsorbent, we also designed the facilitation module to assist the MBPs to chelate enough Hg (II). MerT is a transmembrane protein which can import Hg (II) ions from periplasm into cytosol with the help of MerP; while MerC can directly transport Hg (II) into cytosol [10]. Therefore, we constructed constitutively expressed merP, merT, merC (Fig 4). This design was expected to expand the capacity of our bioabsorbent and the baseline of Hg (II) concentration that our biokit can be applied to.
Figure 4: The overall structure of the facilitation module. The expression of MerT, MerP and MerC are amplified by T7 polymerase activated T7 promoter, which maximizes the bacterial Hg absorption capacity via actively transportation of mercury ions into cytosol. (To Read More…)
    Following function test showed that our whole-cell bioabsorbents can absorb more than 50% of 10-6 M Hg (II) in 120 minutes. Therefore, cells expressing certain metal binding peptide engineered via our method may act as whole-cell bioabsorbents that can decontaminate corresponding metal ion from aquatic environment. This design was successfully applied to another homolog of MerR family – PbrR, the lead responsive regulator (Fig 5). Our bacteria achieved an equivalent lead accumulation capacity as the mercury MBP, which proved validness of our engineering strategy.
Figure 5 Our method was applied to PbrR. (A) Sequence alignment of MerR and PbrR. MerR family TFs share a highly conserved homology at their metal binding domains (Brown et al., 2003; Hobman, 2007), which implies that our strategies of bioabsorbent engineering might be applicable to other cases of heavy metals. (B) and (C) Construction of lead MBP. The MBP was constructed by fusing two copies of metal binding domain of MerR in tandem via the same method with mercury MBP. (To Read More…)
    However, there is still a problem remained in the application of our bioabsorbent. It is not convenient to simply remove our bacteria from water or gather up them for recycling process after enough Hg (II) has been absorbed. Therefore, a time-delayed autoaggregation design was introduced. We chose Antigen43 (Ag43), which mediates visible autoaggregation conferred on the bacteria population in static liquid [11], as our autoaggregation effector. We designed a time-delayed device in which the expression of Ag43 was in response to the presence of mercury with a time delay (Fig 6). Besides time-delayed function, this design also acts an amplifier that can promote the expression level of antigen 43 as well.
Figure 6: The overall structure of the time-delay device. PmerT is a mercury-responsive promoter. In the presence of mercury, the genetic cassette will amplify the signal and express Ag43 with a time delay and high intensity. (To Read More…)
    In summary, we’ve developed a streamlined method to decontaminate heavy metal in water. Due to the easy-manipulated, robust, high performance and inexpensive properties of our bioabsorbent, it might play certain role in heavy metal decontamination in the near future.
Reference
1. Mulligan, C. N., Yong, R. N. & Gibbs, B. F. (2000). Remediation technologies for metal-contaminated soils and groundwater: an evaluation. Eng. Geol. 60, 193-207.
2. Matlock, M. M., Henke, K. R. & Atwood, D. A. (2002). Effectiveness of commercial reagents for heavy metal removal from water with new insights for future chelate designs. J. Hazard Mater. B92, 129-142.
3. Gavrilescu, M. (2004). Removal of Heavy metals from the Environment by Biosorption. Eng. Life Sci. 4, 219-232.
4. Mejare, M. & Bulow, L. (2001). Metal-binding proteins and peptides in bioremediation and phytoremediation of heavy metals. Trends in biotechnol. 19, 67-73.
5. Brown, N. L., Stoyanov, J. V. & Kidd, S. P. & Hobman, J. L. (2003). The MerR family of transcriptional regulators. FEMS Microbiol. Rev. 27, 145-163.
6. Changela, A., Chen, K., Xue, Y., Holschen, J., Outten, C. E., Halloran, T. V. & Mondrago, A. (2003). Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR. Science 301, 1383-1387.
7. Shewchuk, L. M., Verdine, G. L., Nash, H. & Walsh, C.T. (1989). Mutagenesis of the cysteines in the metalloregulatory protein MerR indicates that a metal-bridged dimer activates transcription. Biochemistry 28, 6140-6145.
8. Song, L., Caguiat, J., Li, Z., Shokes, J., Scott, R. A., Olliff, L. & Summers, A. O. (2004). Engineered Single-Chain, Antiparallel, Coiled Coil Mimics the MerR Metal Binding Site. J. Bacteriol. 186, 1861–1868.
9. Silver, S. & Phung, L. T. (2005). A bacterial view of the periodic table: genes and proteins for toxic inorganic ions. J. Ind. Microbiol. Biotechnol. 32, 587-605.
10. Barkay, T., Miller, S. M. & Summers, A. O. (2003). Bacterial mercury resistance from atoms to ecosystems. FEMS Microbiol. Rev. 27, 355-384.
11. Woude, M. W. & Henderson, I. R. (2008). Regulation and Function of Ag43 (Flu). Annu. Rev. Microbiol. 62, 153-169.
Retrieved from " http://2010.igem.org/Team:Peking/Project/Bioabsorbent "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
