"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/Bioabsorbent/MBPconstruction
From 2010.igem.org
(Redirected from Team:Peking/MBPconstruction )
HOME
TEAM
PROJECT
PARTS
MODELING
NOTEBOOK
HUMAN PRACTICE
BIOSAFETY
  Project Home
  Project Discription
  Biosensor
  Bioabsorbent
  Application
  Expansion
   Personal Notes
  Protocols
   Vocabulary
   Parts Submitted to the Registry
   Characterization
   Existed Parts' Improvement
   Students and Instructors
   Peking University
   Collaboration
   Gallery
   Acknowledgement
   Previous Years
   Modelling Home
   Calculating Process
   Analyses and Results
   Advanced Model
   HP Home
   HGT Review
   Proof of Concept
   HGT Investigation
   Acknowledgement
Overview of the Heavy Metal Decontamination Kit
  Biosensor
Introduction
Promoter Characterization
Operation Characterization
Bioreporter
  Bioabsorbent
Introduction
MBP construction
MBP Expression
Facilitation Module
Inductive Aggregation
Module Assembly
  Application
Preservation Optimization
Kit Operation
  Expansion
Introduction
Lead Bioabsorbent
CountDown
Vistors From...
   MBP Construction
         Project > Bioabsorbent >MBP Construction
Early biophysical work shows that homodimer is the active form of MerR. The metal binding domain lies at the C-terminal half of the protein, as predicted by earlier genetic, biochemical, and biophysical work, while the Helix-Turn-Helix DNA binding domain locates at the N-terminal (Fig 1A). To be specific, MerR can response to as low as 10-9 M Hg (II) even in the presence of 1 to 5 mM competing thiol ligands[1] ,which indicates that MerR may act as an effective mercury accumulator in aquatic environment.
However, since MerR is a transcription regulator, over-expression of MerR in bacteria may lead to some unpredictable side effect. Earlier work suggested that the truncated peptide only consisting of the metal binding domain can form a stable dimer with its mercury binding affinity remained [2]; and DNA binding domain and metal binding domain can function individually [3]. Based on all these above and carefully structure analysis of MerR via 3D structure modeling, we directly tandemed two copies of metal binding domain of MerR together, to implement a mercury metal binding peptide (MBP) (Fig 2).
STRUCTURE ANALYSIS
As described before, MerR family transcription factors can be separated into two modular domains: the metal binding domain and DNA binding domain. The signature of the family is a helix-turn-helix (HTH) motif followed by a coiled-coil region, namely, a similar N-terminal helix-turn-helix DNA binding domains and a C-terminal effector binding domains that are specific to the effector (eg. metal ions)to be recognized[4] (Fig. 1A, Fig 3). These two domains communicate with each other by the intervening region (hinge). The crystal structure of several members of MerR family have been solved which provides us evidence for MBP design.
Figure 1. The results of 3D structure modeling for MerR based on data of CueR. (A) Crystal structure of CueR[5], which represent the overall structure of the MerR family. The ribbon diagram depicts one monomer in color and the other in gray, with DNA-binding domain in blue, the dimerization helix in red, the metal binding domain in purple and the metal ion in cyan. (B) The predicted three dimensional conformation of Hg (II) bound MerR structure by our 3D modeling. Two monomers are indicated in green and cyan, respectively. (C) and (D) were obtained after rotation of the structure in (B) by 90° about the z-axis and x- axis, respectively. The C-terminal metal binding domain and the N-terminal DNA binding domain are likely to be modular for each other.
Figure 2. Scheme of MBP construction and the predicted MBP structure. (A) Linear structure of MerR coding region. (B) Construction of MBP. The MBP was constructed by fusing two copies of metal binding domain of MerR in tandem with a flexible SSG bridge. Blue bars indicate the dimerization helix in the metal binding domain of MerR, and gray bars indicate other α-helices of MerR. The red line indicates the SSG linker, and the blue and green lines indicate the loop after the dimerization loop and the region after the loop, respectively. Orange dots indicate cysteines involved in Hg(II) binding[6]. (C) Predicted structure of resulted metal binding peptide. Mercury ions are indicated as black balls in metal binding pockets.
Earlier work provides us a deeper insight into the metal recognition mechanism of MerR family. Kathryn et al. constructed a hybrid regulatory protein containing the DNA binding domain of MerR and the Metal binding domain of ZntR, which showed an altered specificity to MerR’s promoter but responding to zinc[3], which suggested that the DNA binding domain and metal binding domain of MerR family function modularly to some extent. Since our bioabsorbent needs high metal binding affinity and selectivity, we then took a closer look at the metal binding mechanism and the structure of the C-terminal binding domain.
MerR family TFs share a high similarity at the C-terminal metal binding domain (Fig. 3), which indicates a similar metal recognition mechanism and metal-protein complex structure. The key factor of the remarkable selectivity and sensitivity seems to be the preorganization of geometries suited for specific metal ions through folding of the metal-binding domains in these proteins [7]. To be specific, previous mutagenesis study shows that MerR dimer binds one Hg(II) ion in a bridge fashion between the two monomers[8], and Hg (II) adopts a three-coordinate Hg-(S-Cys)3-binding mode by extended X-ray absorption fine structure (EXAFS) spectroscopy[9], though there is no crystal structure available. As is known, single α-helix has a high tendency to be oligomerized. The outer electron configuration of Hg (II) is 5d106s0, and it tends to form a complex, especially with sulfur. All these evidences above indicate that the C-terminal metal binding domain can act as a mercury accumulator without the help of the N-terminal DNA binding domain. Luckily, Qiandong Zeng et al. constructed an N-terminal deletion mutant (contain only residue 80-120) that can form a stable dimer and retain high affinity for Hg (II) [2]. This work reinforces the idea of tandeming two metal binding domains together to make a high performance and less energy consuming metal binding peptide.
Figure 3. Structure based sequence similarity of various metal responsive regulators of MerR family. CadR: cadmium, CueR: copper, ZntR: zinc, MerR: mercury, PbrR:lead. Those amino acids showed in bold are metal binding sites according to solved crystal structures [5] or mutagenesis analysis [8]. The metal binding site of CadR or PbrR can be speculated based on the high similarity of MerR family and the geometry of ligand field of metal ions.
METAL BINDING PEPTIDE (MBP) CONSTRUCTION
To achieve the goal of making a high performance MBP, we constructed a single polypeptide consisting of two dimerization helixes and metal binding loops of MerR, to form an antiparallel coiled coil MBP mimicking the dimerized metal binding domains of the wild-type as described in Fig 2. We amplified the N-terminal and C-terminal of MBP directly from full length MerR by PCR, and then cloned them into the backbone together in one step (Fig. 4).
A
B
Figure 3. MBP Construction procedure. A: Standard part. B. Expression detection part.
To be specific, the entire coding region of the MBP for standard part was amplified by PCR from full length MerR with two pairs of primers. Two of these primers encoded a three-residue bridge, SSG, which does not occur in MerR and was added to afford some flexibility in the loop connecting the two dimerization helix (fig. 1). The two PCR products were digested with EcoR I / BamH I, or BamH I / Pst I and cloned into EcoR I / Pst I -digested pSB1K3 in one step (fig. 3A), which was verified by DNA sequencing.
Based on the same strategy, MBP-His6 was constructed by using two different pairs of primers, which is used for MBP expression test by western blot. The two PCR products were digested with Nde I / BamH I, or BamH I / Xho I and cloned into Nde I / Xho I -digested pET 21a, which contains a region encoding six histines, in one step to construct pET 21a – MBP (fig. 3B), which was verified by DNA sequencing.
Reference
1. Ralston, D. M. & Halloran, T. O. (1990). Ultrasensitivity and heavy-metal selectivity of the allosterically modulated MerR transcription complex. Proc. Natl. Acad. Sci. 87, 3846-3850.
2. Brocklehurst, K. R., Hobman, J. R., Lawley, B., Blank, L., Marshall, L. J., Brown, N. L. & Morby, A. P. (1999). ZntR is a Zn(II)-responsive MerR-like transcriptional regulator of zntA in Escherichia coli. Mol. Microbiol. 31, 893-902.
3. Zeng, Q., Stalhandske, C., Anderson, M. C., Scott, R. A. & Summers, A. O. (1998). The core metal-recognition domain of MerR. Biochemistry 37, 15885-15895.
4. Brown, N. L., Stoyanov, J. V., Kidd, S. P. & Hobman, J. L. (2003). The MerR family of transcriptional regulators. FEMS Microbiol. Rev. 27, 145-163.
5. Changela, A., Chen, K., Xue, Y., Holschen, J., Outten, C. E., Halloran, T. V. & Mondrago, A. (2003). Molecular Basis of Metal-Ion Selectivity and Zeptomolar Sensitivity by CueR. Science 301, 1383-1387 .
6. Qin, J., Song, J., Brim, H., Daly, M. J. & Summers, A. O. (2006). Hg(II) sequestration and protection by the MerR metal-binding domain (MBD). Micro. 152, 709–719.
7. Chen, P. R. & He, C. (2008). Selective recognition of metal ions by metalloregulatory proteins. Curr. Opin. Chem. Biol. 12,214-221.
8. Shewchuk, L. M., Verdine, G. L., Nash, H. & Walsh, C.T. (1989). Mutagenesis of the cysteines in the metalloregulatory protein MerR indicates that a metal-bridged dimer activates transcription. Biochemistry 28, 6140-6145.
9. Wright, J. G., Tsang, H. T., Penner-Hahn, J. E. & O’Halloran T.V. (1990). Coordination chemistry of the Hg-MerR metalloregulatory protein: evidence for a novel tridentate Hg-cysteine receptor sites. J. Am. Chem. Soc. 112, 2434-2435.
Retrieved from " http://2010.igem.org/Team:Peking/Project/Bioabsorbent/MBPconstruction "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
