"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/Expansion/LeadBioabsorbent
From 2010.igem.org
HOME
TEAM
PROJECT
PARTS
MODELING
NOTEBOOK
HUMAN PRACTICE
BIOSAFETY
  Project Home
  Project Discription
  Biosensor
  Bioabsorbent
  Application
  Expansion
   Personal Notes
  Protocols
   Vocabulary
   Parts Submitted to the Registry
   Characterization
   Existed Parts' Improvement
   Students and Instructors
   Peking University
   Collaboration
   Gallery
   Acknowledgement
   Previous Years
   Modelling Home
   Calculating Process
   Analyses and Results
   Advanced Model
   HP Home
   HGT Review
   Proof of Concept
   HGT Investigation
   Acknowledgement
Overview of the Heavy Metal Decontamination Kit
  Biosensor
Introduction
Promoter Characterization
Operation Characterization
Bioreporter
  Bioabsorbent
Introduction
MBP construction
MBP Expression
Facilitation Module
Inductive Aggregation
Module Assembly
  Application
Preservation Optimization
Kit Operation
  Expansion
Introduction
Lead Bioabsorbent
CountDown
Vistors From...
   PbrR-based lead bioabsorbent
         Project > Expansion >LeadBioabsorbent
After the completion of MerR-based mercury decontamination kit, we decided to take another example to verify the validness of our design for metal binding peptide. Based on the previous homology study of MerR family proteins, we took a lead-sensing protein, PbrR, as the second research target.
Lead contamination is a serious threat to human health and the environment. Lead poisoning is still one of the most common environmentally caused diseases in the world today.[1] As the concentration of such toxic ions is generally low, which present a huge challenge for environmental engineers to both detect and to absorb the pollutant with traditional chemical methods. A revolutionary strategy was taken into consideration, which took the advantage of metalloregulatory proteins with capability of sensing and absorbing the Pb(II) ions.
Figure 1 Sequence alignment of MerR and PbrR. MerR family TFs share a highly conserved homology at their metal binding domains (Brown et al., 2003; Hobman, 2007), which implies that our strategies of bioabsorbent engineering might be applicable to other cases of heavy metals.
Nature has evolved numerous such regulating proteins to control the concentrations of beneficial or toxic metal ions with extraordinary sensitivity and selectivity.[1] As is known, the MerR family is a group of transcriptional activators with similar N-terminal helix-turn-helix DNA binding regions and C-terminal effector binding regions that are specific to the effector recognized.[2] The majority of regulators in the family respond to environmental stimuli, such as oxidative stress, heavy metals or antibiotics. A subgroup of the family activates transcription in response to metal ions. This subgroup shows sequence similarity in the C-terminal effector binding region as well as in the N-terminal region. PbrR is a MerR family protein found in Ralstonia metallidurans CH34, a bacterium specifically adapted to survive under toxic heavy metal environment. The PbrR protein is responsible for regulation of lead(II) efflux pumps involved in lead detoxification inside R. Metallidurans.
Due to the highly conserved homology of protein MerR and PbrR (Fig.1), we were able to apply the strategy used for MerR engineering to the development of lead bioabsorbent. Based on the crystal structure study of MerR, the metal binding domain of PbrR was recognized by sequence alignment with MerR.(Fig 2) [3] Further, 3D structure was also conducted.(Fig3)
Figure 2 Pb binding domain predicted by sequence alignment. Three Cys-residues located at 79,114,123 specifically binds Pb(II) ions by forming a metal binding pocket.
Then we designed PbrR metal binding peptide, with consisted of two tandem duplications of α-helix 5 linked by a flexible linker, SSG, and followed by a short peptide sequence.(Fig 4) These direct tandem α-helices fold back on each other into an antiparallel, coiled-coil hairpin. Three Cys-residues located at 79,114,123 specifically binds Pb(II) ions by forming disulfide bonds within the engineered dimer.
Figure 4 Design of PbrR metal binding peptide and structure prediction. Within the engineered dimer, six Cys-residues centralize 2 metal binding pocket, each of which specifically binds Pb(II) ions by forming disulfide bonds .
The principles were same when we considered the construction of PbrR metal binding peptide (MBP). It was accomplished by fusing two copies of alpha-helices 5 of PbrR in tandem with three nonnative amino acids, SSG, as a bridge. PbrR MBP was then constructed and cloned into pSB1C3 backbone as a standard part for function test and the pET21a backbone as the commercial plasmid for the western blotting, as is shown in Fig 5.
Fig 5. Construction procedure of PbrR (lead) MBP. Top: Standard part; Bottom: Expression detection part.
As proved previously, when the mercury MBP was fused to DsbA, it would be efficiently translocated to the periplasm and works comparable to those targeting to the cytosol. Therefore, we fused lead MBP using the same method as it was in mercury MBP construction (Fig 6). Particularly, the PstI restriction site inside DsbA was mutated synonymously.
Fig 6. Procedure of DsbA-MBP construction.
In order to standardize the module, with Nest PCR, RBS (BBa_B0034) and T7 promoter are prefixed with the DsbA-MBP fusion, as is shown in Fig 5. A His-tag was fused at the C-terminal for further western blotting.
Fig 7. Standardization procedure of DsbA-MBP.
DsbA and MBD gene was amplified by PCR from pET-39b (+)-DsbA-MBP, with the primer containing T7 promoter, RBS and SD restriction sites, as shown in Fig 6. The PCR product was digested with EcoR I / Pst I and then cloned into EcoR I / Pst I double digested pSB1K3, to achieve the goal of standardization of the fusion protein (Fig. 7).
Like surface display of MerR MBP, Lpp-OmpA-MBP was designed as a fusion protein consisting of the signal sequence and first 9 amino acid of Lpp, residue 46~159 of OmpA and the PbrR metal binding peptide (MBP). The signal peptide of the N-termini of this fusion protein targets the protein to the membrane while the transmembrane domain of OmpA serves as an anchor. MBP is on the externally exposed loops of OmpA, which can be anchored to the outer membrane. As the method shown in mercury bioabsorbent construction, we directly fused the PbrR-MBP at the C-terminal of Lpp-OmpA protein through a flexible SG rich linker (Fig 8).
Fig 8. Result of 3D modeling for our fusion protein construction. The transmembrane domain of OmpA serves as an anchor and MBP is on the externally exposed loops of OmpA.
Fig 9. Procedures of the construction of standard plasmid with Lpp-OmpA-MBP as the insert.
Fig 10 Procedures of Construction of Commercial Plasmid.
After the construction of the plasmid with the fusion protein gene Lpp-OmpA-MBP, We prefixed T7 promoter and BBa_B0030 upstream of Lpp-OmpA-MBP. Additionally, a strong terminator BBa_B0015 was suffixed.  When the construction of three proteins was completed, T7 promoter+RBS and terminator were prefixed or suffixed to each protein coding sequence, respectively. T7 polymerase from T7 phage was designed to be constitutively expressed, thus to constitutively activate transcription at T7 promoter, in order to guarantee the expression of MBP regardless of the genetic background of bacteria strain.
For the same reason as in Hg (II) MBP construction, we assembled the three modules [Fig. 11]: MBP, DsbA-MBP and Lpp-OmpA-MBP, which would translocate MBP to the cytoplasm, periplasmic space and on the outer membrane in pSB1C3 for further function test.
Fig 11. The lead absorption device we designed to guarantee the maximum of Pb absorption. Top: The final overall structure of lead absorption device as an insert in pSB1C3. Middle and Bottom: The production of T7 RNA polymerase is constitutive. T7 polymerases will active high rating transcription at T7 promoters. Thus Hg (II) will be highly effectively accumulated by substantial amount of MBPs which are translocated to cytosol, periplasm and cell surface of the bacteria. All the components were assembled together and cloned into pSB3K3.
Protein Expression and Function Test
The cytoplasmic expression, periplasmic translocation and surface display of PbrR-MBP were verified by SDS-PAGE and Western Blotting. The specific band in western blotting for his-tag fused MBP of about 12 kD confirmed that the MBP was expressed as expected (Fig 12). Considerable amount of MBP expressed in cytosol can also be indicated from the result of SDS-PAGE. Overexpression band in SDS-PAGE result and specific band in the western blotting in size of expected molecular weight also indicated that the fusion proteins were translocated into periplasm and displayed on the surface as expected (Fig 13). Therefore, the bioabsorbent of lead employs same pathways to serve the decontamination goal as mecury bioabsorbent: to express MBP in the cytosol, in the periplasm and on the surface. Their expression was under the regulation of T7 promoter, driven by constitutively expressed T7 polymerase.
Fig 12. The specific band in western blot for his-tag fused MBP of about 12 kD confirmed that the MBP is expressed as expected. Considerable amount of MBP expressed in cytosol can be indicated from the result of SDS-PAGE.
Fig 13. There are overexpression band in SDS-PAGE result and specific band in the western result at the expected molecular weight, which indicate that the fusion proteins are translocated into periplasm as expected.
Besides, we conducted 3D structure modeling to overview them and their localization (Fig 14).
Fig 14. Overview of various localization of MBP engineered from PbrR.
After verifying that the PbrR-MBP could be expressed and translocated as expected, the function test was carried out with ICP-AES, using the method described at MBP Expression Page. The result was similar to that of mercury MBP. The lead binding capacity of MBP with different localization was indicated in Fig 15. The surface displayed MBP appeared to have highest binding capacity while the pyramiding of MBP expression did not function as expected.
Fig 15 Different amount of lead absorbed by bacteria with MBP expressed in different subcellular compartments cultured for ~40h in 10-5 mol/L Pb (II) medium.
reference
[1] Peng Chen, Bill Greenberg, Safiyh Taghavi, Christine Romano, Daniel van der Lelie, and Chuan He, An exceptionally selective lead (II)-regulatory protein from Ralstonia metallidurans: development of a fluorescent lead (II) probe, Angew. Chem. 117, 2005,  2775 –2779.
[2] Nigel L. Brown, Jivko V. Stoyanov, Stephen P. Kidd, Jon L. Hobman, The MerR family of transcriptional regulators, FEMS Microbiology Reviews, 27, 2003, 145-163.
[3] Lingyun Song, Jonathan Caguiat, Zhongrui Li, Jacob Shokes, Robert A. Scott, Lynda Olliff, and Anne O. Summers, Engineered Single-Chain, Antiparallel, Coiled Coil Mimics the MerR Metal Binding Site, Journal of Bacteriology, 186(6), 2004,  1861–1868.
[4] Song, L., Caguiat, J., Li, Z., Shokes, J., Scott, R. A., Olliff, L. &Summers, A. O. (2004). Engineered single-chain, antiparallel,coiled coil mimics the MerR metal binding site. J Bacteriol 186,1861–1868.
Retrieved from " http://2010.igem.org/Team:Peking/Project/Expansion/LeadBioabsorbent "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
