"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Notebook/WYWang
From 2010.igem.org
HOME
TEAM
PROJECT
PARTS
MODELING
NOTEBOOK
HUMAN PRACTICE
BIOSAFETY
  Project Home
  Project Discription
  Biosensor
  Bioabsorbent
  Application
  Expansion
   Personal Notes
  Protocols
   Vocabulary
   Parts Submitted to the Registry
   Characterization
   Existed Parts' Improvement
   Students and Instructors
   Peking University
   Collaboration
   Gallery
   Acknowledgement
   Previous Years
   Modelling Home
   Calculating Process
   Analyses and Results
   Advanced Model
   HP Home
   HGT Review
   Proof of Concept
   HGT Investigation
   Acknowledgement
NOTEBOOK HOME
  Protocols
Protocols
  Vocabulary
Vocabulary
  Personal Notes
Haoqian Zhang
Ao Liu
Xin Teng
Qianzhu Wu
Ying Sheng
Mei Chen
Miao Jing
Heng Pan
Tianze Zhu
Yang Hu
Junyi Jiao
Boxuan Zhao
Donghai Liang
Yiwei Chen
Zairan Liu
Weiye Wang
Zhenzhen Yin
Yuheng Lu
CountDown
Vistors From...
   Weiye Wang's Notes
                                                                                                                                                
My wetlab mission is to select a T3 promoter with medium expression level from T3 phage genome: Constructing 3 multi-plasmid systems, using IPTG, mercury and arabinose as the inducer of T3 polymerase’s expression, so that to characterize the response properties of these T3 polymerase dependent promoters. The design and construction of our team's wiki is also under my charge.
download her notes
Contents
July, 2010
August, 2010
September, 2010
October, 2010
July
Mon
Tue
Wed
Thu
Fri
Sat
Sun
-
-
-
-
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
[ TOP ]
7.2
get designed primers
T3 promters+fragments of GFP(BBa_E0240)(Forward), and name  them in my own order:
phiOL For: ccg gaattc TATTTACCCTCACTAAAGGGAAT tcaca cagga aagta ctaga tgcgt aaagg
phiOR For: ccg gaattc CATTAACCCTCACTAAAGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi1.05 For: ccg gaattc CATTAACCCTCACTAACGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi1.1 For: ccg gaattc AGTTAACCCTCACTAACGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi1.3 For: ccg gaattc TAATAACCCTCACTAACAGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi1.5 For: ccg gaattc CATTAACCCTCACTAACAGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi2.5 For: ccg gaattc TAATTACCCTCACTAAAGGGAAC tcaca cagga aagta ctaga tgcgt aaagg
phi4.3 For: ccg gaattc AATTAACCCTCACTAACGGGAAC tcaca cagga aagta ctaga tgcgt aaagg
phi6.5 For: ccg gaattc AATTAACCCTCACTAAAGGGAAG tcaca cagga aagta ctaga tgcgt aaagg
phi9 For: ccg gaattc TAATTACCCTCACTAAAGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi10 For: ccg gaattc AATTAACCCTCACTAAAGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi11 For: ccg gaattc CTTTAACCCTCACTAACAGGAGG tcaca cagga aagta ctaga tgcgt aaagg
phi13 For: ccg gaattc AATTAACCCTCACTAAAGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
phi3.8 For: ccg gaattc AATTAACACTCACTAAAGGGAGA tcaca cagga aagta ctaga tgcgt aaagg
notice:
1) No.13(phi13) has the same sequence with No. 9(phi6.5)
2) E0240: B0032 RBS (medium) +GFP(E0040)+ terminator(B0010+B0012) is 876 bps long
3) The sequence of T3 promoters are extracted directly from T3 phage genome, and some of the T3 promoters may already have an RBS on their tails.
4) No.9 (phi6.5) is the Most Commonly used T3 promoter, which always exists on commercialized vectors (such as EZ-T).
fragments of GFP(BBa_E0240) (Reverse):
Rev Standard E0840: aaa ctgcag cggccgc t actagt a TATAAACGCA GAAAGGCCCA
BBa_G00000+T3 RNA polymerase(forward)
T3 RNA polymerase(reverse)+BBa_G00001
7.5
1. Dissolve the primers got days before; PCR reaction (using 20μl phusion system) to get 12 T3 promoter+GFP(E0240) fragments and EX+T3 RNA polymerase+SP fragments .
20uL reaction system:
5x PHF buffer II           4uL
ddNTP Mixture           1.6uL
For                    1uL
Rev                    1uL
Template plasmid         0.2uL
phusionHF              0.2uL
ddH2O                 12 uL
phusion system: 1000bp/15s
2. Electrophoresis the PCR reaction products in 1.5% agarose gel. (Using trans 2K plus as marker)
3. Excise the gel slice and extract the T3 promoter+GFP(E0240) fragments, which is about 2.6 k bps’ long. Preserving those fragments in -20 ℃ fridge.
Result:  NO.7 & NO.13 didn’t see the string, PCR failed, only 12 T3P+GFP fragments were collected.
7.6
1. PCR reaction to get NO.7 & NO.13 T3P+GFP fragments. (20μl phusion system still)
2. Electrophoresis the PCR reaction products in 1.5% agarose gel. (Using trans 2K plus as marker), successfully extracted the NO.7 T3P+GFP fragments. Preserve them in -20 ℃ fridge.NO.13 failed again.
3.Picking colonies of following 2 strains in the afternoon(16:00pm): 2 colonies of pSB6A1, & 2 colonies of pSB4K5 in order to get enough plasmid backbones. Cultivate at 37℃ overnight.
7.7
1. Miniprep of Psb1A2 & Psb4K5 plasmids.
2. PCR reaction to get NO.13 T3P+GFP fragments. This time using 20uL Taq reaction system:
20uL Taq reaction system:
2x EASY TAQ buffer           10uL
For                    1uL
Rev                    1uL
Template plasmid         0.5uL
ddH2O                 7.5 uL
phusion system:~ 1000bp/60s
3. Electrophoresis the PCR reaction products in 1.5% agarose gel. (Using trans 2K plus as marker), successfully extracted the NO.13 T3P+GFP fragments.
4. Digestion and identification by Electrophoresis:
1) EcorI & PstI cut T3P+GFP fragments(14 tubes totally, each in 20uL digestion system)
2)XbaI & PstI cut EX+T3pol+SP fragments
3)EcorI & PstI cut Psb6A1 &Psb4K5
7.8
1. Purified the digested fragments from gel, and ligation for 1.5h.
1) 14*T3P+GFP(E,P cut) with Psb4K5( E,Pcut)
2)EX+T3 pol+SP(E,S cut) with RBS(B0032)+SP (cutted backbone of Psb1A2)
2. Transformation of ligation mixture. (10:30 pm)
7.9
1. 14:00pm, all colonies on the plates are red, which means the enzyme-digestion failed the day before yesterday. Check the experimental notes and realized that I forgot to add buffer into the digestion system.
2. re-digestion of the fragments and plasmids:
1) EcorI & PstI cut T3P+GFP fragments(14 tubes totally, each in 20uL digestion system)
2)XbaI & PstI cut EX+T3pol+SP fragments
3)EcorI & PstI cut Psb6A1 &Psb4K5
Enyzme 1.5uL each
Sample 15uL
1 0x NEB buffer 2uL
3.Electrophoresis and Purification of the digested fragments.
7.10
1. Ligating the digested fragments again.(NO.1 T3P+GFP splashed = =|||)
2. Transform the ligation mixture into trans5a competent cells.
7.11
1. Re-ligation of NO.1 T3P+GFP and Psb4K5 digested backbone.
2. Transformation of the NO.1 T3P+GFP ligation mixture.
3. Picking colonies from the other 13 T3P+GFP(pSB4K5)plates and cultivating; each T3P+GFP picking 3 candidates. Picking colonies from RBS(B0032)+T3 pol(1A2): also 3 candidates.
7.12
1. No colony existed on RBS_T3pol #1, and few colonies appeared on three NO.4 T3P+GFP(Psb4K5) plates.
2. Miniprep 2+13*3 tubes of plasmids.
3. Picking colonies from the NO.1 T3P+GFP(pSB4K5)plate: 3 candidates. Shaking.
7.13
1. Sent the two RBS+T3pol plasmid candidates to sequence company.
2. Culture result wasn’t very satisfied: Exam the shaking table and found the temperature inside is 39℃, which is too high for bacteria to grow.
3. Miniprep 3 candidates of NO.1 T3P+GFP.
4. PCR reaction to exam whether all the 45 T3P+GFP candidates have the wanted sequence.(using T3P primers I received on July 2ed):
Electrophoresis result show that all 45 candidates have the wanted sequence ( ~0.9k bps)
7.15
1. Get SpeI+PstI cutted T7P(on Psb1A2) from Heng Pan, and digest RBS_T3pol(already got 2 candidates: RBS_T3pol_2 & RBS_ T3pol_3) with XbaI +PstI( in order to link the RBS_T3pol downstream the T7P)
2. Electrophoresis and Purify the digestion product
3. Ligation: SP cut T7P + XP cut RBS_T3pol
4. Transform the ligation mixture into competent cells
7.16
1. T7P+RBS_T3pol_3’s transformation failed, no colony appeared.
2. Pick 3 colonies from T7P+RBS_T3pol_2’s plate and culture it in shaking table.
3. The result of sequence (RBS_T3pol two candidates) failed, no signal.
7.17
1. Miniprep the 3 candidates of T7P+RBS_T3pol_2 and sent the plasmid to sequence company.
2. Using 3 candidates of T7P+RBS_T3pol_2 and 6 T3P_GFP candidates which is randomly chose to do the co-transformation ( transform two different plasmids simultaneously), in order to see which one of the 3 T7P+RBS_T3pol_2 candidates has the function.
Selected T3P_GFP: 1_1; 2_2; 3_3; 11_1; 12_2; 13_3;
So 3*6=18 pairs was made to co-transform into trans1-T1 competent cells( which is phage-resisted); 18 plates were put into oven.
7.18
1. 12/18 plates failed to grow colonies. ( antibiotics added too much?)
2. Induction for the 1st time: pick 1 colony of each co-transformation plate, culture in 37℃ shaking table, cultured until O.D600 level become 0.98( actually 0.4-0.6 maybe better). Add IPTG and induce the expression of T7 RNAP.
The induction failed finally, no GFP signal was observed.
3. Method II: transfer 1 plasmid first (T7P+RBS_T3pol on Psb1A2), then make the transformed cell become competent again, and transform the second plasmid(T3P_GFP on Psb4K5) into it.
7.19
1. The result of T7P+RBS_T3pol showed that the digestion on July 15th is failed, and the RBS_T3pol on Psb1A2 failed to be cut down. The results also showed that the RBS_T3pol sequence is correct.
2. Digest the RBS_T3pol again and do the ligation.
3. Pick 6 colonies of T3P_GFP(1_1; 2_2; 3_3; 11_1; 12_2; 13_3;) and RBS_T3pol, cultured for miniprep to extend the plasmid resource.
7.20
1. Transform the two RBS_T3pol+T7Pbackbone ligation mixture in the morning.
2. No colony grew on one of the two plates (RBS_T3pol_2 failed) in the evening. Pick 2 candidates from T7P+RBS_T3pol_3, cultured and miniprep in the evening, named them T7P_T3pol 3.1 &T7P_T3pol3.2.
3. Found a way to exam whether T7P is successfully link upstream the RBS_T3pol: using SFC_I enzyme, whose recognition site is only inside T7P( no recognition site on RBS_T3pol and Psb1A2 backbone)
4. Receive the sequence result of T3P_GFPs: T3P_GFP 1.1 failed.
5. Digest the T7P_T3pol 3.1 &T7P_T3pol3.2 with SfcI and PstI, if T7P is successfully linked, a 3000 bps string will appear on the Electrophoresis gel.
7.21
1. Electrophoresis result show no signal. Pick T7P_T3pol candidates again from the plate grow yesterday; miniprep them in the afternoon.
Digest with SfcI enzyme, and using uncut RBS_T3pol as a negative control.
2. Miniprep:
1)RBS_T3pol_2 and RBS_T3pol_3
2)T3P_GFPs
3. Sent T3P_GFP candidates to the sequence company: NO.4_1;NO.5_1;NO.6_!;NO.7_1;NO.8_1
4. Transform RBS_T3pol sequenced plasmid into trans1T-1competent cells to keep the strain.
5. Digest RBS_T3pol_2 and RBS_T3pol_3 using XbaI and PstI again. The Electrophoresis result show that the objected 2.6k string is mixed with the Psb1A2 backbone’s string (which is 2.1k long), thus become hard to distinguish and separate them.
6. About the SfcI +PstIexam: SfcI’s recognition site:CTRYAG
There are more than 4 recognition site if the T7P is successfully linked upstream the RBS_T3pol on Psb1A2.
7.22
1. Miniprep to get more RBS_T3pol plasmids.
2. Sent T3P_GFP candidates to sequence company: NO.9_1;NO.10_1;NO.14_1
3. Digestion of RBS_T3pol with XbaI and PstI and do the ligation again.
4. Consult An Xiao for more advise:
1) The amount of enzyme in digestion system cannot be more than 1/10
2)10*BSA is demanded if required
3) TAE used in electrophoresis is needed refresh frequently.
5. Go no SfcI+PstI exam of the T7P_T3pol candidates.
The electrophoresis result wasn’t very good. Maybe one possible candidate is selected; named it T7P_T3pol neo.
7.23
1. Transform the T7P_T3pol neo in to trans1-T1 cells.
2. XbaI and PstI digestion to make show the reserved RBS_T3pol is the right plasmid.(2.6k bps fragments are observed)
3.Pick more T7P_T3pol candidates and cultured.
7.24
1. Miniprep to the T7P_T3pol candidates.(10 totally)
2. SfcI single-digest-system to exam whether the T7P is existed or not.
If T7P is existed, a ~1k bps fragment will be observed by Electrophoresis.
This time 7 candidates show the positive Electrophoresis results. Select 4 of them and sent them to the sequence company.
7.26
1. The result of sequence is received, which show that 2 candidates is correct: T7P_RBS_T3pol is successfully constructed.
2. Do the induction pre-experiment again(like July 18th ), co-transform two plasmid into BL21 competent cells.( randomly choose T3P_GFP 2_2;3_3;11_1;12_2; totally 2*4=8 pairs)
7.27
1. Tragedy! All the 8 plates failed to grow colonies!!
1) BL21 competent cell isn’t robust, cannot product enough antibiotic-resistant plasmid?
2) low-efficiency of co-transformation
3) BL21 competent cell is affected by the phage?
2. Transform the correct sequenced T7P_RBS_T3pol into trans1-T1 to restore the strain.
3. Sent T3P_GFP candidates to sequence:NO1_3;NO.6_2;NO7_2;NO.8_2;NO.9_2
7.28
1. Pick colonies from T7P_RBS_T3pol(tran1-T1) to get more plasmids. Pick colonies of T7P_RBS_T3pol(BL21) to product competent cell.
2. Scribe and restore some already-sequenced T3P_GFP:NO2_2;NO.3_3;NO.11_1;NO12_2
7.29
1. Turn T7P_RBS_T3pol(BL21) into competent cells.
2. Sequence result failed, re-pick T3P_GFP#1,5,6,7,8,9,10 from plates cultured on July 11th.
3. Found a serious problem: The plates applied on the morning is covered by bacteria lawn.
1) Kan+ antibiotic lose its function
2) The transformation efficiency of DIY competent cells are too high
3) The bacteria lawn is not the wanted BL21 cells, the plates are polluted
So new plates(K+) are made, and picked more colonies of T7P_RBS_T3pol(BL21) for tomorrow’s DIY competent cells.
7.30
1. DIY T7P_RBS_T3pol(BL21) competent cells again, and transfer T3P_GFP plasmids into the cells(1_1,2_2,3_3,4_1,5_3,6_3,7_3,8_3,9_3,10_3,11_1,12_2,14_1)
2. Unfortunately, the bacteria lawn began to grow again 5h after plate-apply.
3. In order to exam whether these lawn are bacteria pollution or not, try to pick several colonies from the plates cultivated yesterday, cultured and miniprep to get its plasmids. Using EcorI and PstI to digest it.
The Digestion result show the lawn contain the wanted T7P_RBS_T3pol(BL21) cells.(~3k fragments detected)
7.31
1.3rd time to DIY T7P_RBS_T3pol(BL21) cells and transfer T3P_GFP into them. Lawn appeared again, digestion show several unwanted fragments.
2. Consult An Xiao: the reason of lawn’s growing maybe result from T7P’s leakage, which lead to the downstream Amp+ ‘s over expression.
1)book BL21(DES)plySs competent cells, which have little T7 RNAP leakage
2)link a terminator after T3 pol.(using terminator B0015, 100bps)
August
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
-
-
-
-
[ TOP ]
8.1
Ligation of B0015 to T7P_RBS_T3pol, transformation into Trans1-T1(2plates)
8.2
1. One plates grow better. Pick colonies and miniprep in the evening.
Digest them by NdeI+PstI
If the ligation succeeded, 4k fragment and 890bps fragments will be seen.
If failed, 4k fragments and 7.9 fragments.
2. BL21(DES)plySs arrive, transfer T7P_RBS_T3pol into them.
8.3
Exam the digestion product. (succeed~~get 3 possible candidates, sent them to sequence)
8.4
1. Got sequencing’s results of T3P_GFPs
Now I’ve got T3P_GFPNO.1, 2, 3, 4, 5, 7, 8, 10,11, 12, 13, 14;
No. 6 failed, results show the sample sent has the same sequence with No.11
No. 9 failed, results show the sample sent has the same sequence with No.5
2. Got sequencing’s results of T7P_RBS_T3pol_terminator 15_1, correct.
Transfer the plasmids into BL21(DES)plySs. (failed…)
8.5
More sequence result give out, which show T7P_RBS_T3pol_terminator 15_1, 15_2, 15_3 are all correct.
8.6-8.11
Go back to home in Shanghai ~~
Visit the EXPO
8.12
1. Transfer T7P_RBS_T3pol_terminator 15_1 into BL21(DES)plySs, again.
8.13
1. Pick 3 colonies of the BL21(DES)plySs+ T7P_RBS_T3pol_terminator 15_1, cultivate in tubes, then turn them into DIY competent cells.
Transfer T3P_GFP NO.1, NO.2, NO.3, NO.4, NO.5, NO.7, NO.8, NO.10, NO.11, NO.12, NO.14.
8.14
1. The plates transferred yesterday: 4/11 failed
2. Using co-transformation ( 2 plasmids being transfered into competent cells simultaneously )to deal with the low efficiency of transformation which using DIY competent cells.
8.15
IPTG induction ~~!! Under 37 degree, shaking for 2h. Collect the O.D. very 30min.
Failed. Almost no fluorescent signal detected. But NO.4 NO.5 seems to have some weak signal.
8.16
Induction again. Under 37 degree, shaking for 8h. Collect the O.D. very 30min.
Failed, still no GFP fluorescent signal.
8.17
1. Colony PCR of the inducted bacteria to see whether these bacteria are the wanted bacteria(T7P_RBS_T3pol_terminator 15_1+T3P GFP)
Using hc primers to exam whether there are T3 pol sequence ( recogonization site only on T3 polymerase; positive results will have see a 1.6k bps fragment):
hc-T7P-F
AGCACTTCAAGAAGCACG
hc-T7P-R
TGAGCCAGTTCATCGCCT
Using T3P_GPF primers to exam the existence of T3P_GFP(800-900bp fragment)
Only NO.4, NO.5, NO.7 have the target T3P_GFP positive result, which seems to fit with the 8.15 Induction’s result.
8.18
1. Induction again (2h) . Still failed.
Dissolve fresh IPTG. ( preserve away from light)
2. Transfer T3P_GFP into DIY T7P_RBS_T3pol_terminator 15_1 BL21(DES)plySs competent cells.
8.19
1. Pick colonies from plates transferred yesterday. Preserve the strain, and do the colony PCR to check the existence of the two plasmids.
Notice: bacteria lawn appeared again in spite of the terminator’s existence.
change the backbone of T7P_RBS_T3pol_terminator 15_1 into Psb1C3.
EP digest and check whether bacteria existence happened again: positive(T7P_RBS_T3pol_terminator 15_1’s existence checked).
PCR exam try to confirm the existence of T3pol and T3_GFP
8.20
1. Check the PCR result, no positive result.-bacteria polluted.
the possible reason of lawn’s appearance:
O.D. level-too high: 0.2-0.6 is better (mine is 0.9-1.0)
plasmids volume too little: only 5μl-better 10μl?
2. Check the sequence result again- T7P_RBS_T3pol_terminator 15_1 has 1 point mutation (although maybe the sequencing error)!!! And T7P_RBS_T3pol_terminator 15_2 & T7P_RBS_T3pol_terminator 15_3 is correct.
Transfer T7P_RBS_T3pol_terminator 15_2 & T7P_RBS_T3pol_terminator 15_3 into BL21(DES)plySs ~~
3. Transfer the backbone-changed T7P_RBS_T3pol_terminator 15_1 into trans1-T1(abandoned, since the T7P_RBS_T3pol_terminator 15_1 may be a wrong copy)
8.21
1. SP digest the RBS-T3pol, link with XP cut terminator
2. XP digest T7P_RBS_T3pol_terminator 15_2 to change the backbone into Psb1C3.
3.in order to confirm the expression of T3 RNAP, induce the T7P_RBS_T3pol_terminator 15_1 BL21 bacteria(only one plasmid inside) to do the PAGE exam
37℃ , 2h
4. Transfer the T3P_GFP NO.2, NO.4, NO.5, NO.7, NO.8, NO.9, NO.10, NO.14 into the DIY competent cells.
8.22
1.Induction again, freeze and extract RNA from the bacteria, in order to have RT-PCR-EXAM to confirm the expression of T3 RNAP.
2. Transfer T3P_GFPs into DIY competent cells.
8.23
1. miniprep the backbone-changed  T7P_RBS_T3pol_terminator 15_2 & T7P_RBS_T3pol_terminator 15_3 bacteria. Sent to sequence.
2. Low efficiency of transformation still…
8.24
1.Try transformation again; PCR and Enzyme Digestion again to confirm the existence of each plasmids.
Successfully confirm the following 5 T3P_GFP+ T7P_RBS_T3pol_terminator strains:
NO.7 NO.8 NO.4 NO.5 NO.10
8.25
1. IPTG induction again(NO.7 and NO.8)(37℃, 5h, failed again)
Insult Xing Teng- induction should be practiced under a lower temperature, such as 18℃, and longer time, such as 16h.
8.26-27
1. Get more confirmed two-plasmid strains.(NO.9 & NO.11 obtained~ NO.9 is the most widely used commercial T3P~~)
2. Follow Xing Teng’s advice, using 0.001M/L IPTG to induce under 18℃ for 16h.
PAGE result confirmed the expression of T3 RNAP.
8.30
Using  IPTG to induce under 32℃ for 5h(NO.9 & NO 11).(Concentration gradient from 10^-7 to 10^-1 M IPTG)
This time NO.9 and No.11 detected fluorescent signal in 0.001 M IPTG!!!!
8.31
Induction again. (NO.8 NO.9 NO.11), 30℃, 6h, (Concentration gradient from 10^-7 to 10^-1 M IPTG)
Result:
10^-3M IPTG highest signal
Unfortunately, the leakage problem was very serious.
Two plans to avoid the leakage:
Using merR system ,induce the expression of T7 RNAP with Hg(II) .
Problem: in merR system, plasmids conflict existed(Ori competition)= =|||   The backbone-change of PmerT_T3pol(1C3) into3C5 is required.
Substitute T7P with PBAD, and using arabinose to induce the expression of T7 RNAP
September
Mon
Tue
Wed
Thu
Fri
Sat
Sun
-
-
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
-
-
[ TOP ]
9.1-9.10
IPTG induction.
Statistics of NO.4, NO.5, NO.7, NO.8, NO.9, NO.10, NO.11, NO.12, NO.14 (NO.13 are the same as No.11) are collected
9.11-9.20
Construct the ara-induction system and Hg-induction system, but unfortunately failed…= =|||
9.21-10.27
Began to focus on wiki construction. Miao Jing took the major charge of the left bench work.
==go to her page==         
Retrieved from " http://2010.igem.org/Team:Peking/Notebook/WYWang "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
