"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking/Project/Expansion
From 2010.igem.org
HOME
TEAM
PROJECT
PARTS
MODELING
NOTEBOOK
HUMAN PRACTICE
BIOSAFETY
  Project Home
  Project Discription
  Biosensor
  Bioabsorbent
  Application
  Expansion
   Personal Notes
  Protocols
   Vocabulary
   Parts Submitted to the Registry
   Characterization
   Existed Parts' Improvement
   Students and Instructors
   Peking University
   Collaboration
   Gallery
   Acknowledgement
   Previous Years
   Modelling Home
   Calculating Process
   Analyses and Results
   Advanced Model
   HP Home
   HGT Review
   Proof of Concept
   HGT Investigation
   Acknowledgement
Overview of the Heavy Metal Decontamination Kit
  Biosensor
Introduction
Promoter Characterization
Operation Characterization
Bioreporter
  Bioabsorbent
Introduction
MBP construction
MBP Expression
Facilitation Module
Inductive Aggregation
Module Assembly
  Application
Preservation Optimization
Kit Operation
  Expansion
Introduction
Lead Bioabsorbent
CountDown
Vistors From...
   EXPANSION
        
              *Lead Bioabsorbent
   INTRODUCTION
         Project >Expansion
Expansion Introduction
    The MerR family of transcription activators includes those that respond to the presence of essential and toxic metals; and those that respond to the presence of drugs and other chemical species which cause cellular damage(Brown et al., 2003). The bacterial metal sensing MerR regulators directly bind metals and then activate transcription from unusual promoters as a result of metal binding. A recent phylogenetic analysis of the predicted protein sequences of metal responsive MerR regulators (COG0789) classifies them into several groups – a subgroup of which sense Hg2+ (MerR), Cu+ (CueR/SctR), Cd2+ (CadR), Pb2+ (PbrR), or Zn2+ (ZntR) with high specificity (Fig 1) etc. (Permina et al., 2006).
Fig 1. Selective metal ion recognition by metalloregulatory proteins. Adapted from Peng Chen and Chuan He, 2008.
    It is notable that the MerR family is a group of transcriptional activators with similar N-terminal helix-turn-helix DNA binding regions and C-terminal effector binding regions that are specific to the effector recognized. Interestingly, the subgroup shows sequence similarity in the C-terminal effector binding region as well as in the N-terminal region, but it is not yet clear how metal discrimination occurs (Fig 2 and Fig 3). This subgroup of MerR family regulators includes MerR itself and may have evolved to generate a variety of specific metal-responsive regulators by fine-tuning the sites of metal recognition.
Figure 2 Sequence alignment of MerR and PbrR. MerR family TFs share a highly conserved homology at their metal binding domains (Brown et al., 2003; Hobman, 2007), which implies that our strategies of bioabsorbent engineering might be applicable to other cases of heavy metals.
    Inspired by the fact that MerR family TFs share a highly conserved homology at their metal binding domains (Brown et al., 2003; Hobman, 2007), we developed to engineer strategies of bioreporter and bioabsorbent which may be applicable to other cases. Therefore, we then expanded our reverse engineering strategy to another common toxic heavy metal, lead (Borremans et al., 2001; Chakraborty et al., 2008; Chen et al., 2005; Julian et al., 2009; Mergeay et al., 2003). We primarily took lead resistance operon apart, followed by analyze of function, structure and operation of PbrR, the lead responsive regulator. Then modeling was conducted to analyze the characteristics of PbrR and topology of its regulation behavior. Information collected confirmed the possibility to design genetic circuit that makes lead sensor and regulator components more efficient and robuster. We also engineered PbrR into single-chain coiled coil (MBD) via the same method as MerR. Exhilaratingly, following expression of PbrR MBD on the surface, periplasm and in cytosol of bacteria and the lead (II) absorption test showed that our bacteria can absorb more than 50% of 10^-6M Pb (II) in 2 hours, which is comparable to mercury MBD expression in E.coli, proving validness of our engineering strategy.
Reference
Borremans, B., Hobman, J.L., Provoost, A., Brown, N.L., and van Der Lelie, D. (2001). Cloning and functional analysis of the pbr lead resistance determinant of Ralstonia metallidurans CH34. J Bacteriol 183, 5651-5658.
Brown, N.L., Stoyanov, J.V., Kidd, S.P., and Hobman, J.L. (2003). The MerR family of transcriptional regulators. FEMS Microbiol Rev 27, 145-163.
Chakraborty, T., Babu, P.G., Alam, A., and Chaudhari, A. (2008). GFP expressing bacterial biosensor to measure lead contamination in aquatic environment. Current Science 94, 800-805.
Chen, P., Greenberg, B., Taghavi, S., Romano, C., van der Lelie, D., and He, C. (2005). An exceptionally selective lead(II)-regulatory protein from Ralstonia metallidurans: development of a fluorescent lead(II) probe. Angew Chem Int Ed Engl 44, 2715-2719.
Hobman, J.L. (2007). MerR family transcription activators: similar designs, different specificities. Mol Microbiol 63, 1275-1278.
Julian, D.J., Kershaw, C.J., Brown, N.L., and Hobman, J.L. (2009). Transcriptional activation of MerR family promoters in Cupriavidus metallidurans CH34. Antonie Van Leeuwenhoek International Journal of General and Molecular Microbiology 96, 149-159
Mergeay, M., Monchy, S., Vallaeys, T., Auquier, V., Benotmane, A., Bertin, P., Taghavi, S., Dunn, J., van der Lelie, D., and Wattiez, R. (2003). Ralstonia metallidurans, a bacterium specifically adapted to toxic metals: towards a catalogue of metal-responsive genes. Fems Microbiology Reviews 27, 385-410
Permina, E.A., Kazakov, A.E., Kalinina, O.V., and Gelfand, M.S. (2006). Comparative genomics of regulation of heavy metal resistance in Eubacteria. BMC Microbiol 6, 49.
Retrieved from " http://2010.igem.org/Team:Peking/Project/Expansion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
