<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIOSINT_Mexico_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIOSINT Mexico/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-fixed" style="opacity:0.7; position:absolute; height:100px; overflow:hidden;"><NAV class="N/A transparent"><DIV class="nav-wrapper #000000 black"><UL id="nav-mobile" class="left hide-on-med-and-down"><LI id="buton" class="dropdown-button" data-activates="dropdown1"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico">Main</A></LI><LI class="dropdown-button" data-activates="dropdown2"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Project">Project</A></LI><LI class="dropdown-button" data-activates="dropdown3"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Practices">Policy &amp; Practices</A></LI><LI class="dropdown-button" data-activates="dropdown4"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Achivements">Achievements</A></LI><LI class="dropdown-button" data-activates="dropdown5"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Team">Team</A></LI><LI class="dropdown-button" data-activates="dropdown6"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Notebook">Notebook</A></LI><LI class="dropdown-button"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Attributions">Attributions</A></LI><LI class="dropdown-button"><A class="header" href="https://2015.igem.org/Team:BIOSINT_Mexico/Collaborations">Collaborations</A></LI></UL></DIV></NAV></DIV><UL id="dropdown1" class="dropdown-content #000000 black"><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico#abstract">Abstract</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico#video">Video</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico#whyart">Why Art?</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico#map">Map Of The Site</A></LI></UL><UL id="dropdown2" class="dropdown-content #000000 black"><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#coliroid">Coliroid</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#background">Background</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#colorcoliroid">Color Coliroid</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#assembly">Assembly</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#cannon">Light Cannon</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Project#results">Results</A></LI></UL><UL id="dropdown3" class="dropdown-content #000000 black"><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Practices#Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Practices#Cooperation">Cooperation</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Practices#ArtGallery">Art Gallery</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Practices#SpreadingIGEM">Spreading IGEM</A></LI></UL><UL id="dropdown4" class="dropdown-content #000000 black"><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Achivements#results">Results</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Achivements#parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Achivements#camp">Camp</A></LI></UL><UL id="dropdown5" class="dropdown-content #000000 black"><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Team#members">Members</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Team#gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Team#sponsors">Sponsors</A></LI><LI><A href="https://igem.org/Team.cgi">Official Page</A></LI></UL><UL id="dropdown6" class="dropdown-content #000000 black"><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Notebook#LabRecord">Lab Record</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Notebook#Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:BIOSINT_Mexico/Notebook#References">References</A></LI></UL><DIV class="container"><DIV id="coliroid" style="position: relative;"><H1>
				Coliroid
			</H1><H3>What´s a coliroid?</H3><H5>Has it ever come into your mind that biology can be used as a photographic device?</H5><H5>Well that's not a  very far- fetched idea. In fact this has already been done by the joint  UT-Austin/UCSF iGEM  team in 2004. This team came up with a project of bacterial photography commonly known as Coliroid or E.coliroid.  But what exactly does that mean?
E. Coliroid is a bacterial system which switches on and off in response to red light and acts like a bacterial Polaroid camera. Thanks to this polaroid like function the unique name of Coliroid came into being by combining the words  E.coli and Polaroid.</H5><H5>Figure. Coliroid images.  The left panel shows the first coliroid image done “Hello World “ published in Levskaya et al., Nature, 2005 and the right panel shows a  coliroid portait of Andy Ellington
But how does this work?
The system consist of a synthetic sensor  based on 2 devices as shown in figure, that allows a lawn of bacteria in an agar medium to project a two dimensional chemical image, based on the projection of a pattern of light on to the bacterial plate. </H5><H5>Figure. Two device Synthetic Sensor. The left panel shows a light sensor that takes red light as an input and produces PoPs as an output and the right panel shows the color generator that takes in a PoPs signal as the input and displays a colored output.The system works by  producing a black chromoprotein (LacZ) in the absence of red light, so the gel section overshadowed by the image becomes dark. This mechanism can be observed in the image below:</H5></DIV><DIV id="background"><H1>
				Background
			</H1><H5>
In 2004, the University of Texas designed a bacterial system that is switched between different states by red light. The system consists of a synthetic sensor kinase that allows a lawn of bacteria to function as a biological film, such that the projection of a pattern of light onto the bacteria produces a high-definition (about 100 megapixels per square inch), two-dimensional chemical image.

Plants and some bacteria use a class of protein photoreceptors known as phytochromes to control phototaxis, photosynthesis and the production of protective pigments. Phytochrome consists of two identical chains. Each chain has a PAS domain and GAF domain. The PAS domain serves as a signal sensor and the GAF domain is responsible for binding to cGMP ( regulator of ion channel conductance, glycogenolysis, and cellular apoptosis) and also senses light signals. Together, these subunits form the phytochrome region, which regulates physiological changes in plants to changes in red and far red light conditions. Photoreceptors are not found in enterobacteria, such as Escherichia coli. So UT Austin group designed the red sensor cph8 using the cph1 (photoreceptor) and EnvZ (response regulator of OmpR). 

In 2011, the Uppsala University design systems capable of detecting multiple wavelengths (red light, blue light and green light), focuses on improving the existing multichromatic sensing systems by expanding the number of useful wavelengths. The system can regulate the expression of three different genes independently from each other using three different wavelengths. These &quot;multichromatic coliroids&quot; upgrades the present coliroids, much like upgrading black-and-white movies to color TV.

Considering the design by the University of Texa, the same red sensor (cph8) was used and only changed the production of LacZ for RFP to give the red. For Blue, Uppsala use a new blue light sensor (YF1) designed by Andreas Möglich in 2008. YF1 is a fusion protein of YtvA (B subtilis) and FixL (B japonicum). The chromoprotein that was used in YF1 is amilCP. For Yellow, Uppsala used the green light sensitive sensor (ccaS) from Synechocystis sp. PCC6803. The chromoprotein that was used in ccaS is amilGFP.

</H5></DIV><DIV id="overview"><H1>
				Overview
			</H1><H5>
This project aims to bioengineer  E. coli  in order to  serve as a biosensor that expresses chromoprotein reporters. Two light receptors were used, one for red light-sensing, cph8, and one for the blue light-sensing, YF1. 
Both of this membrane receptor phosphorylate the transcription factors respectively associated to the promoters, pompC and FixK2, when they are not in the presence of a light stimuli. 
ho1 and pcyA are essential coding sequences involved in the biosynthesis of phycocyanobilin (PCB), which is involved in the creation of cph8 fusion protein.
AmilGFP and spisPink are proteins isolated from coral-reef that act as reporter proteins due to its colorimetric properties.

When the bacteria is irradiated with an specific wavelength of 660nm, amilGFP´s expression is repressed. The same  thing  happens with a wavelight of 470 nm and the  production of the chromoprotein spisPink is repressed. When the system is in total darkness both proteins are expressed and a third color is formed.</H5></DIV><DIV id="colorcoliroid"><H1>
				Color Coliroid
			</H1><H5>
Our project uses E. coli strain k-12 as chassis and a gene circuit that allow the bacteria to sense specific light waves. Two photoreceptor cph8 and YF1, that had to be cloned and expressed in a constitutively manner, are sensitive to red and blue light 660 and 470 nm respectively.

ho1 produce biliverdin IXalpha (BV) that is converted to  phycocyanobilin (PCB) by phycocyanobilin:ferredoxin oxidoreductase (PcyA), PCB is the immediate precursor for chromophore formation and essential  for the assembly of cph8.

ho1 is a gene from cyanobacteria that produces a heme oxygenase . Biliverdin IXα is produced when heme undergoes reductive ring cleavage at the α-methene bridge catalyzed by heme oxygenase. 

PcyA and is a member of the ferredoxin-dependent bilin reductase family. After the ho1 activity PcyA reduces D-ring (exo) and A-ring (endo) vinyl groups of biliverdin IXα (BV) to yield phycocyanobilin. 

cph8 change ompR to its phosphorylated state. ompR is a endogenous transcription factor in E. coli and constitutively expressed. When this happens, production mediated by pompC promoter is stop. YF1 works in a similar manner that cph8 with the difference that the respective transcription factor to be phosphorylated is not naturally expressed in E. coli and had to be added with a constitutive promoter.

cph8 is a fusion protein form by a phytochrome Cph1 and a histidine kinase domain, requires the biosynthesis of phycocyanobilin. The resulting molecule is a transmembrane protein. the cph1 domain it´s outside the membrane and the EnvZ domain dephosphorylated ompR with the stimuli of red light.  

The transcription factor OmpR positively regulates the promoter. Phosphorylated OmpR binds to operator sites and activates transcription. This state is achieved by the EnvZ site of cph8.
The EnvZ/OmpR regulatory components are a dimeric histidine kinase EnZ, serving as a signal sensor. After a signal the envZ autophosphorylates and the transformed to EnvZ-P to produce Phosphorylated OmpR or OmpR-P. OmpR-P is a transcription factor that binds to promoter regions and regulates it´s expression. 

YF1 is a fusion protein from a covalent bond between FixL and YtvA. It was reported by Möglich et.al. (2008), as a control for gene expression in a light-dependent manner in vivo. 
YF1 was conceived by oxygen-sensitive, light-inert histidine kinase by replacing its chemosensor domain by a light-oxygen-voltage (LOV) photosensor domain.
When expressed, this protein migrates to the cell membrane and becomes able to receive luminal stimuli, it get excited when photons in a wavelength of 470 nm hit the outer part of the protein, and stops the phosphorylation reaction. 

In corals the key color determinants are GFP-like fluorescent proteins (FP´s). FP´s from coral are, approximately, 230 amino acids long and have acquired the ability to synthesize different color proteins. The FP´s are responsible of practically all color diversity in corals. 

amilGFP is a yellow chromoprotein optimized for E.coli k12 and is parts of a group of proteins isolated from Acropora species. The amilGFP derivative from Acropora millepora, present a yellow pigment when it’s expressed. Also, the protein has a maximum absorption of 512 nm.This version is codon optimized for the expression in E.coli K12.
spisPinkK12: Pink chromoprotein optimized for E.coli k12. This protein, expressed in Stylophora pistillata, was evolving independently from the rest of coral chromoproteins. The protein has a maximum absorption of 560 nm. This characteristic confers a blue shift by at least 14 nanometers in comparison to other known coral chromoproteins, inducing the production of pink color rather than purple appearance. This version is codon optimized for the expression in E.coli K12.
When the system is in total darkness both spisPink (pink color protein) and amilGFP (yellow color protein) are expressed and a third color should be form by it´s co expression.</H5></DIV><DIV id="assembly"><H1>
				Assembly
			</H1><H5>For the assembly, we are working with two different constructs, both in pSB1C3, with differentiated functions. The plasmids are show below.</H5><H5>Figure N° 1. pSB1C3 with construct for constitutive expression for PCB (Ho1 and pcyA), and regulated expression of spisPink attached upstream to FixK2 promoter.</H5><H5>Figure N° 2. pSB1C3 with construct for constitutive expression of red and blue light photoreceptors (cph8 and YF1), and regulated expression of amilGFP attached upstream to FixK2 promoter.</H5><H3>Devices</H3><H5>Several devices were designed and built, as shown in the next image.</H5><H5>Figure N° 5. Color-Coliroid. The figure above show how the system works. </H5><H3>Photoreceptors</H3><H5>YF1 and Cph8 are two different photoreceptors, chimeric proteins These Are That Have the Ability to phosphorylate or dephosphorylate Their respective transcription factors. For Their constant expression inside the cell.
For this, first the coding sequences joined a sequence RBS (BBa_J61100), and then were joined together, in addition to which was added a terminator (BBa_B0030) at the 3 'end and a constitutive promoter (BBa_J23101) upstream, in the 5 '</H5><H3>Transcription factors</H3><H5>YF1 and Cph8 are responsible for regulating the activity of two different promoters, by phosphorylation of two transcription factors. Thus, a device capable of producing FixJ transcription factor associated with YF1 and FixK2 was designed. Also, ompR is a transcription factor associated with cph8, however, since endogenous ompR use, the device was assembled to produce PCB (HO1 and pcyA), which helps the assembly of this transcription factor.To this they are joined HO1 (BBa_I15008) pcyA (BBa_I15009) and FixJ (BBa_K592005) with RBS (BBa_J61100) upstream. All these parts are joined together with a strong constitutive promoter (BBa_J23101) and a terminator at the end (BBa_B0030).</H5><H3>Chromoproteins</H3><H5>Chromoproteins Both were linked to a repressible promoter, that is associated with transcription factors, to produce light when there is.

spisPink optimized for E. coli K12 (BBa_K1684000) are joined together to FixK2 promoter (BBa_K592006), an RBS (BBa_J61100), and a terminator (BBa_B0030)


amilGFP optimized for E. coli K12 (BBa_K1684001) and joined together to FixK2 promoter (BBa_K592006), an RBS (BBa_J61100), and a terminator (BBa_B0030)
</H5></DIV><DIV id="cannon"><H1>
				Light Cannon
			</H1><H5>A device which meets the needs of the project was designed.This device consists of a box in which a system for temperature control is implemented also a mechanism in which not to do many things manually is implemented; only you need to move a knob to the figure you want to project in the Petri dish is changed.The mechanism is commonly known as Geneva mechanism, it is a mechanism steps. The original mechanism consists of a rotary movement in which every time the circular piece rotates one revolution the other piece is moved one step. You can design the mechanism so that it has three or more steps:The Geneva mechanism that was designed consists of four steps in which each step an image that will not let light pass through it will. In each quarter revolution will move one step design that was given. A knob to make the move from outside the box, if you move the knob one full turn will become the first step and therefore the first figure is attached. This knob is manually moved to allow time for the bacteria to receive wavelengths.Care in designing the components of the pieces do not collide with each other to perform the rotary movement had:The box consists of a drawer that he can change the petri dish:Top box has a hole through which a projector is attached to occur with an RGB combination the desired wavelength:The projector, the top hole of the box and figure to be projected are aligned.The material of the box must be insulated, because you do not want to have a heat exchange with the surroundings.Have a resistance on the back of the box, it is desired to increase the temperature inside the box. Between the top where the Geneva mechanism and the laboratory dish is separated from the back in case the resistance emits unwanted wavelengths becomes hot is located. It should be a material with high thermal conductivity between the two sections to be rapidly spread.The sensor would be at the top of the box.The electronic system consists of a temperature sensor is one that would be used as predesigned. Obtained from the LM335 datasheet from Texas Instruments:The circuit must be outside the box by security components except heat resistance and the temperature sensor.</H5></DIV><DIV id="results"><H1>
				Results
			</H1><H3>spisPink &amp; amilGFP optimized to Escherichia coli K12 </H3><H5>After completing the insertion of gBlocks (IDT) designed (spisPink and amilGFP), into the plasmid pSB1C3, by following the relevant protocols; competent cells were successfully transformed and consequently the removal of plasmids (spisPink into pSB1C3) BBa_K1684000 and (amilGFP into pSB1C3) BBa_K1684001 was performed.The Figure 6 presents a 1% agarose gel which contains two samples, by triplicate, next to the ladder. In line 1, 2 and 3 are located the triplicate samples of spisPink piece, optimized for E. coli K12 (BBa_K1684000), with a size of 2789 bp. Likewise on the line 4,5 and 6 are repeated samples of the piece amilGFP optimized for E. coli K12 (BBa_K1684001). For both BioBricks, the electrophoresis results correspond to the theoretical size, greater than 2500 bp and less than 3000 bp; also it can be seen that there is no variation between samples, replicas, indicating the reliability of E. coli K12 transformations.<DIV align="center"><P>Figure 6: Agarose gel with spisPink and amigGFP samples, optimized for E. coli K12 (x3)</P></DIV></H5><H3>RBS and Optimized biobricks</H3><H5>In Figure 7 different samples of parts used for transformation of competent cells; these two parts are composite biobricks, the first is composed by RBS (BBa_B0034) and the optimized chromoprotein sequence spisPink for E. coli K12 (BBa_K1684000), while the second is composed by a RBS (BBa_B0034) and the optimized chromoprotein sequence amilGFP (BBa_K1684001 ). The agarose gel shown in Line 1, the negative control; Line 2, belongs to the purified DNA of pGLO (5371 bp), was run as a positive control. Line 3 and Line 4 contain duplicate products of BBa_K1684002 (BBa_B0034 + BBa_K1684000) biobrick, both results match the size of the plasmids, 2800 bp. Line 5 and Line 6 contain duplicate BBa_K1684003 (BBa_B0034 + BBa_K1684001) biobrick; however, a positive result is observed only in Line 5, with size of 2822 bp, while in line 6 only presents contamination, without the presence of any genetic material. <DIV align="center"><P>Figure 7: Agarose gel with BBa_K1684002 (x2) and BBa_K1684003(x2) samples, positive and negative controls</P></DIV></H5><H3>Optimized biobricks with Terminator</H3><H5>Finally, in Figure 8 samples composite parts are presented, the first BBa_K1684004, is composed of the chromoprotein sequence optimized for E. coli K12 spisPink (BBa_K1684000) and a terminator (BBa_B0010); the second part is composed of the chromoprotein optimized sequence amilGFP (BBa_K1684001) and the same terminator (BBa_B0010). This agarose gel shown on Line 1 and Line 2, the biobrick BBa_K1684004, in the first case the band run corresponding to its size, 2869 bp; however, on Line 2 the result was not the expected and only contamination was obtained.In Line 3 and 4 results, was run the composite BioBrick BBa_K1684005, in the first case the result was not successful; In Line 4 however, the band is present with the respective size, 2890 bp, as a positive result. Finally on Line 5 is presented only the negative control.<DIV align="center"><P>Figure 8: Agarose gel with BBa_K1684004 (x2) and BBa_K1684005(x2) samples, and negative control.</P></DIV></H5></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>