<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CIEI-BJ_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CIEI-BJ/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Template</TITLE></P><NAV id="mainNav" class="navbar navbar-default navbar-inverse" style="font-family:Lato;font-size:22px;background-color:#2196f3;-webkit-transform: translateZ(0);z-index: 100;position: fixed;box-shadow: 0px 0px 10px black;"><DIV id="TopBar" class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:CIEI-BJ">Home Page</A></LI><LI><A id="t1" onmouseover="javascript:showsecond(this,'mydiv2');" onmouseout="dalayedHide1(this,'mydiv1')">Team</A></LI><LI><A id="t2" onmouseover="javascript:showfirst(this,'mydiv1');" onmouseout="dalayedHide2(this,'mydiv2')">Project</A></LI><LI><A id="t3" onmouseover="javascript:showthird(this,'mydiv3');" onmouseout="dalayedHide3(this,'mydiv3')">Parts</A></LI><LI><A id="t4" onmouseover="javascript:showforth(this,'mydiv4');" onmouseout="dalayedHide4(this,'mydiv4')">Safety</A></LI><LI><A id="t6" onmouseover="javascript:showsixth(this,'mydiv6');" onmouseout="dalayedHide6(this,'mydiv6')">Human Practices</A></LI></UL></DIV></DIV></NAV></DIV><DIV class="container well" id="mydiv2" style="border:1px solid silver;display:none; margin-top: 0px;"><DIV class="row"><P> </P><P> </P><DIV class="col-lg-4 col-md-4 col-sm-6"><DIV class="media"><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Team">Meet the Team</A></H2><P>Meet that passionate team!!</P></DIV></DIV></DIV><DIV class="col-lg-4 col-md-4 col-sm-6"><DIV class="media"><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Collaborations">Collaboration</A></H2><P>See how we work together.</P></DIV></DIV></DIV><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Attributions">Attributions</A></H2><H2 class="media-heading"> </H2></DIV></DIV></DIV><DIV class="container well" id="mydiv4" style="border:1px solid silver;display:none;margin-top: 0px;"><DIV class="row"><P> </P><P> </P><DIV class="col-lg-4 col-md-4 col-sm-6"><DIV class="media"><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Safety">Safety First!</A></H2>
          Safety is always our first priority.</DIV></DIV></DIV><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="container well" id="mydiv5" style="border:1px solid silver;display:none;margin-top: 0px;"><DIV class="row"><DIV class="col-lg-4 col-md-4 col-sm-6"><P> </P><P> </P><P> </P><DIV class="media"><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Attributions">Attributions</A></H2>
          The Attribution requirement tells what we did ourselves and what we had help with. </DIV></DIV></DIV><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="container well" id="mydiv6" style="border:1px solid silver;display:none;margin-top: 0px;"><DIV class="row"><DIV class="col-lg-4 col-md-4 col-sm-6"><P> </P><P> </P><DIV class="media"><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Human_Practices">Human Practices</A></H2>
          This part is our social and environmental context, in order to help us better understand issues that might influence the design and use of our technologies. </DIV></DIV></DIV><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="container well" id="mydiv7" style="border:1px solid silver;display:none;margin-top: 0px;"><DIV class="row"><DIV class="col-lg-4 col-md-4 col-sm-6"><P> </P><P> </P><DIV class="media"><DIV class="media-body"><H2 class="media-heading"><A href="https://2016.igem.org/Team:CIEI-BJ/Model">Model</A></H2>
          Here's our model to prove our results and help us finish the experiment.</DIV></DIV></DIV><P> </P><P> </P><P> </P><P> </P></DIV></DIV><P><A href="#0" class="cd-top cd-is-visible cd-fade-out">Top</A></P><DIV class="container well" id="mydiv1" style="border:1px solid silver;display:none;margin-top: 0px;"><P>Tanshinone will decrease blood pressure, dilate the arteries and reduce blood clotting.</P><TABLE width="947" height="168" style="border:1px #f5f5f5 solid;"><TBODY><TR><TH width="99" scope="col" style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Description#description">Description</A></TH><TH width="88" scope="col" style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Description#solution">Design</A></TH><TH width="136" scope="col" style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Experiments">Experiments</A></TH><TH width="117" scope="col" style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Notebook">Notebook</A></TH></TR><TR><TD style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Description#description"> Demonstrate</A></TD><TD style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Proof">Proof</A></TD><TD style="border:1px transparent solid;"><A href="https://2016.igem.org/Team:CIEI-BJ/Results">Results</A></TD><TD style="border:1px transparent solid;"> </TD><TD style="border:1px transparent solid;"> </TD></TR></TBODY></TABLE></DIV><P> </P><DIV class="container well" id="mydiv3" style="border:1px solid silver;margin-top: 0px;display:none;"><TABLE width="952" height="80" style="border:1px transparent solid;"><TBODY><TR><TH width="860" scope="col" style="border:1px transparent solid;"><H1><A href="https://2016.igem.org/Team:CIEI-BJ/Parts">All Parts</A></H1></TH></TR></TBODY></TABLE></DIV><P><TITLE>Safety</TITLE></P><DIV class="container" style="top:0%;"><DIV id="bodycontainer" class="row card"><DIV class="col-sm-10"><DIV class="row" id="environmental_safety"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header red" style="padding-top: 50px"><STRONG>Procotol</STRONG></H3><UL><LI><P align="center"><STRONG>Preparing LB and LB agar</STRONG></P></LI><LI>The following is required for preparing LB (liquid broth) agar</LI><LI>Tryptone 1%</LI><LI>Yeast extracted 0.5%</LI><LI>NaCl 1%</LI><LI>Deionized water</LI><LI>Add some amount of NaOH to adjust the pH value to 7.0</LI><LI>Add 300 μL antibiotic</LI><LI>Heat the solution from step 2 in a microwave oven. </LI><LI>Centrifuge at 180 rpm for 40 min. </LI><LI>Centrifuge at 7500 rpm for 1 min.</LI><LI>Seal the conical flask and autoclave for 20 minutes at 121℃, 1.01 atm. For LB agar, prepare empty plates while autoclaving.</LI><LI>Pour the LB agar till the plate is half-full. Store in 4℃ freezer after the LB agar was solidified.</LI></UL><P> </P><P align="center"><STRONG>Culturing single colony</STRONG></P><UL><LI>Make sure there are sample tubes, pipette, alcohol lamp, and marker inside the sterile hood.</LI><LI>Sterilize liquid broth (LB), antibiotics, culture plate, tweezers, and your hands with alcohol. Place the equipment mentioned in the sterile hood. Light up the alcohol lamp to disinfect tweezers.</LI><LI>Choose the single colony desired for culture from the plate and mark clearly. </LI><LI>Label group number, date, sample name, and resistance on the sample tube. </LI><LI>Add 3 mL of LB into the sample tube. </LI><LI>Add 150 μL of antibiotic into the sample tube, gently pipet to mix. </LI><LI>Use the disinfected tweezers to clip a pipette tip and pick out the marked single colony. </LI><LI>Place the pipette tip inside the sample tube. </LI><LI>Incubate the sample tube in shaking bed for 12 hrs at 37°C. </LI></UL><P align="left"> </P><P align="left"> </P><P align="center"><STRONG>PCR</STRONG></P><UL><LI>Add the following into the PCR tubes</LI></UL><P align="left"><STRONG>PCR of target gene</STRONG></P><UL><LI>Forward primer 0.5 μL</LI><LI>Reverse primer 0.5 μL</LI><LI>Sample 1 μL</LI><LI>2xQ5 Taq mix 6 μL</LI><LI>dd H2O 7 μL</LI></UL><P align="left"><STRONG>Verification of positive clone:</STRONG></P><UL><LI>Forward primer 0.5 μL</LI><LI>Reverse primer 0.5 μL</LI><LI>Sample1 μL</LI><LI>2xTaq mix 5 μL</LI><LI>dd H2O 8 μL</LI><LI>Setting the PCR thermal cycle</LI></UL><P align="left"><STRONG>Target gene:</STRONG></P><UL><LI>Denature: 98℃ 30s </LI><LI>Annealing (32 cycles): 98℃ 10s, 62℃ 30s, 72℃ 1min </LI><LI>Elongation: 72℃ 10min </LI></UL><P align="left"><STRONG>Verification of positive clone:</STRONG></P><UL><LI>Denature: 94℃ 3min </LI><LI>Annealing (30 cycles): 94℃ 30 s, 53℃ 30 s, 72℃ 30 s </LI><LI>Elongation: 72℃ 10min </LI><LI>Run gel electrophoresis after the PCR is complete. </LI></UL><P align="left"> </P><P align="center"><STRONG>Terminal adding A reaction</STRONG></P><UL><LI>Add the following to a centrifugal tube</LI><LI>sample 4 μL </LI><LI>1mm dATP 2 μL </LI><LI>10*buffer 1 μL </LI><LI>Taq enzyme 0.2 μL </LI><LI>dd H2O 2.8 μL </LI><LI>Incubate at 72℃ for 30 minutes. 
						      <P align="center"><STRONG>Plasmid extraction</STRONG></P><P align="left">We used Agarose Gel DNA Recovery Kit to extract the plasmid. The experiment procedure is as follow: </P></LI><LI>Add all RNase A into solution P1; store it in the 4℃ freezer.</LI><LI>Add ethyl alcohol absolute into Buffer PW; store it at room temperature.</LI><LI>Check if P2, P3 solution contains any precipitate, if so, water bath at 37℃ till the precipitate dissolves.</LI><LI>Add 1-5 mL of the bacteria solution into the centrifugal tube. Centrifuge for 1 minute at 10,000g, room temperature. Decant the supernatant.</LI><LI>Add 250 μL of P1 into the centrifugal tube. Vortex till pellet disappears.</LI><LI>Add 250 μL of P2, mix by inversion for 6-8 times. (Do not mix violently)</LI><LI>Add 350 μL of P3, mix by inversion for 6-8 times until white precipitate appears. Centrifuge at 10,000g for 10 minutes, room temperature.</LI><LI>Transfer the supernatant to spin column. Centrifuge for 30-60 s at 10,000 g, room temperature. Discard the flow-through. </LI><LI>Add 700 μLof buffer PW. Centrifuge for 30-60 s at 10,000 g, room temperature. Discard the flow-through.</LI><LI>Add 500 μL of buffer PW. Centrifuge for 30-60 s at 10,000g, room temperature. Discard the flow-through.</LI><LI>Centrifuge for an additional 2 minutes at 10,000 g, room temperature, to completely remove any residue buffer PW. Leave the lid open for a while to allow any residual ethanol to evaporate.</LI><LI>Transfer the spin column to a clean centrifugal tube. Add 30-100 μL buffer EB to the center of the absorption film. Leave it at room temperature for 2 minutes, then centrifuge for 2 minutes at 10,000 g, room temperature. </LI><LI>The solution in the centrifugal tube now contains the plasmid. Discard the spin column. Store the plasmid DNA at -20℃.</LI></UL><P align="center"><STRONG> </STRONG></P><P align="center"><STRONG>Gel Electrophoresis</STRONG></P><P align="left"><STRONG>Making the gel</STRONG></P><UL><LI>Preparing the mixture</LI><LI>Agarose powder 1%</LI><LI>TAE buffer</LI><LI>Melt the mixture in a microwave oven. </LI><LI>Pour it into the gel-casting tray and insert the comb. Wait 30 minutes for the gel to solidify.</LI><LI>Remove the comb with care, and place the gel along with the gel-casting tray inside the electrophoresis tank.</LI></UL><P align="left"><STRONG>Running gel electrophoresis</STRONG></P><UL><LI>1of 1xloading buffer need to be added for every 10 μL solution. Calculate and add appropriate amount of loading buffer to increase the sample weight.</LI><LI>Add 1μL dye and 1μL Bromophenol blue for each sample.</LI></UL><P align="left">※ The dye makes the strips visible under the UV light. Bromophenol blue is used for the indication of the position of each gene. </P><UL><LI>Pour 1xTAE buffer into the electrophoresis tank until the gel is immersed. Close the lid to the electrophoresis tank and connect the electrodes. Adjust running time according to the size of the DNA sample. Pay attention to the dye line to avoid running overtime.</LI></UL><P align="left"> </P><P align="center"><STRONG>Gel extraction and purification</STRONG></P><P align="left">Agarose Gel DNA Recovery Kit:</P><UL><LI>Cut the targeted DNA stripe off the gel under ultraviolet light, discard as many residues as possible. Place the gel into a clean centrifuge tube and measure its mass.</LI><LI>Add binding buffer PN to the tube, the volume of PN should be two times more than that of the gel. Water bath at 55℃ for 10 minutes, overturn the tube mildly and consistently during this process.</LI></UL><P align="left"><STRONG>Note:</STRONG> If the binding buffer appears to be pink, add 10 μL 3 M NaAc pH 5.2 to the solution.</P><UL><LI>Wait till the temperature drop back to room temperature and add the solution into the absorption column CA1, leave it at room temperature for 2 minutes, centrifuge at 10,000 g for 30-60 s. Discard the liquor in the collection tube and put CA1 back into the tube.</LI></UL><P align="left"><STRONG>Note:</STRONG> If the volume of the gel is greater than 700μL, then place it into two CA1s. Make sure all the solution is in the absorption column.</P><UL><LI>Add 700 μL Buffer PW into CA1, centrifuge at 10,000 g for 30-60 s, discard the liquor and put CA1 back into the tube.</LI><LI>Repeat step 4.</LI><LI>Centrifuge at 10,000 g for 2 minutes. Open the lid and leave it at room temperature for 2 minutes so the ethanol will volatilize.</LI><LI>Place the absorption column inside a clean centrifuge tube. Add 20 μL Buffer EB to the center of the absorption film. Leave it at room temperature for 2 minutes, then centrifuge at 10,000 g for 2 minutes. </LI><LI>The solution in the centrifuge tube now contains the DNA we need. Discard CP and store the centrifuge tube at -20℃.</LI></UL><P align="left"><STRONG> </STRONG></P><P align="center"><STRONG>Digestion</STRONG></P><UL><LI>Prepare proper pipets and labeled centrifugal tubes.</LI><LI>Add the following to the labeled centrifugal tube on ice.</LI><LI>10×buffer 1 μL</LI><LI>Sample 1 0.5 μL</LI><LI>Sample 2 0.5 μL</LI><LI>Sample 4 μL</LI><LI>dd H2O 4 μL</LI><LI>Incubate for 3~5 hrs at the appropriate temperature. (Depends on the type of bacteria and the target of the experiment, usually at 37℃. </LI></UL><P> </P><P align="center"><STRONG>Ligation</STRONG></P><UL><LI>Prepare proper pipets and labeled centrifugal tubes.</LI><LI>Premix the following solution in a labeled centrifugal tube:</LI><LI>T4 ligase 0.1 μL10xT4 ligase buffer 1μLSample 1 2 μL </LI><LI>Sample 2 4 μL </LI><LI>dd H2O 3 μL </LI><LI>Place the solution at 4℃ overnight. 
						          <P align="center"><STRONG> </STRONG></P><P align="center"><STRONG>Transformation</STRONG></P></LI><LI>Preparation:</LI><LI>Prepare LB agar plate</LI><LI>Set the water bath to 42℃</LI><LI>Prepare an ice bucket</LI><LI>Take competent cell from the -80℃ freezer for thawing</LI><LI>Pipette 250 μL of competent cells into each centrifugal tube. Incubate on ice for 30 minutes. Store the remaining competent cell back into the -80℃ freezer.</LI><LI>Take 10 μL of ligation product from -20℃ freezer. Gently mix by pipetting.</LI><LI>Heat-shock at 42℃ for 90 seconds, and then incubate on ice for 5 minutes. </LI><LI>Pipette 450 μL of bacteria-free LB medium (contains correspondent antibiotic) into each tube. </LI><LI>Centrifuge for 40 min at 180 rmp, room temperature.</LI><LI>Centrifuge for 1min at 7500 rmp, room temperature.</LI><LI>Decant 500~600 μL supernatant.</LI><LI>Disinfect the spreader using alcohol lamp and wait for the spreader to cool. Pipette 50~100 μL<EM> E.coli</EM> onto the agar plate. Spread the cells evenly on the plate.</LI><LI>Incubate the plates for 12~16 hours at 37℃.</LI></UL><P align="left"><STRONG> </STRONG></P><P align="center"><STRONG>Western blot</STRONG></P><P align="left"><STRONG>SDS-page</STRONG></P><UL><LI>Set up the system using the brochure provided by Mini-PROTEAN  tetra system (Bio-Rad)</LI><LI>Making 8% spacer gel</LI><LI>dd H2O 4.6 mL</LI><LI>1.5 M Tris-HCL (pH8.8) 2.5 mL</LI><LI>30% Solarbio 2.7 mL</LI><LI>10% SDS 100 μL</LI><LI>10% AP (Sigma) 100 μL</LI><LI>TEMED (Sigma) 6 μL</LI><LI>Total 10 mL</LI></UL><P align="left">Mix the liquid immediately after adding TEMED. Using a pipette to add the mixture into the container. Add a layer of isopropanol to seal the gel. Wait for 20 minutes until the gel solidify.</P><UL><LI>After the gel solidified, pour out the isopropanol. Use dd H2O to clean the surface.</LI><LI>Making 5% separation gel</LI><LI>dd H2O 2.7 mL</LI><LI>1M Tris-HCL (pH6.8) 0.5 mL</LI><LI>30% Solarbio 0.67 mL</LI><LI>10% SDS 40 μL</LI><LI>10% AP 40 μL</LI><LI>TEMED 4 μL</LI><LI>Total 4 mL</LI></UL><P align="left">Place a 1mm thick comb into the glass. </P><UL><LI>Wait for 20 minutes, and then take out the comb vertically. Use dd H2O to wash the gel.</LI><LI>Place the gel into the gel-casting tray. Fill the box with 1xrunning buffer (25mM Tris, 192 mM glycine, 0.1% w/v SDS).</LI><LI>Add 4 μL marker (SM0671, Fermentas). Add 10~15 μL of each specimen. Set the voltage at 80V and change it into 100 V after 20 minutes. Stop running when Bromophenol blue reached the bottom of the gel. </LI></UL><P align="left"><STRONG>Transmembrane and Antibody incubation</STRONG></P><UL><LI>After SDS-page, cut the bottom part of the gel and put it into the transfer buffer (40mM Tris, 39 mM glycine, 20% v/v methanol) for 20 minutes. </LI><LI>Prepare the filter paper and the PVDF membrane according to the size of the gel. Soak the PVDF membrane in 100% methanol for 5 minutes, and then transfer it together with the filter paper into the buffer for 15 minutes. </LI><LI>Place them in the transmembrane clip in the sequence of black plane - cotton - filter paper - gel - PVDF membrane - cotton - transparent plane. The front of the membrane should face the gel. Make sure there is no bubble between the gel and the membrane.</LI><LI>Add buffer till the Blotting line. Place the gel-casting tray into 4℃ freezer, 100 V for 80 minutes.</LI><LI>After transferring, use 5% milk sealing fluid made from TBST (100mM Tris-HCL pH7.5, 150mM NaCl, 0.05% Tween-2-) to incubate overnight at 4℃.</LI><LI>After the incubation, use TBST to wash the membrane for 3 times, 5 minutes each. Place the PVDF membrane on the Parafilm membrane facing upwards. Dilute the primary antibody (Anti-GST 1:1000) with 1% milk sealing fluid made from TBST, the pipette it onto the PVDF membrane. Incubate for 1 hour at room temperature.</LI><LI>After the incubation, use TBST to wash the membrane for 4 times, 10 minutes each. </LI><LI>Place the PVDF membrane on the Parafilm membrane facing upwards. Dilute the secondary antibody (Goat Anti-Mouse lgG (H&amp;L)-HRP, 1:1000) with 1% milk sealing fluid made from TBST, then pipette it onto the PVDF membrane. Incubate for 1 hour at room temperature.</LI><LI>Use TBST to wash the membrane for 4 times, 10 minutes each.</LI><LI>Place a preservative film at the bottom of the cassette. Use paper to remove the buffer on the PVDF membrane. Place the PVDF membrane onto the preservative film facing upwards. Mix A, B Thermo with the ratio of 1:1 and pipette in onto the membrane. Place another layer of preservative film and incubate for 5 minutes, protect form light.</LI><LI>Coloration: Close all the light in the room, take out XBT-1, Kodak, mark the corner of the film and place it into the cassette. Set the exposure time. Place into the developer until the strips appear, place it into the water for a few seconds, and transfer in into fixative. When the film turns black, take it out from the fixative and wait for it to dry. </LI></UL></DIV></DIV></DIV><DIV class="row" id="environmental_safety2"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header green" style="padding-top: 50px"><STRONG>Contruction </STRONG>of Vector</H3><UL><LI><P align="left"><STRONG>Cloning of <EM>GFP</EM> gene</STRONG></P></LI><LI>PCR</LI><LI>Design primers that contain correspondent restriction enzyme cutting site (Bam H1 and Nco 1)</LI><LI>Amplify <EM>GFP </EM>gene. </LI><LI>After the PCR is complete, run gel electrophoresis to confirm that the target gene (GFP) was successfully replicated.</LI><LI>Gel extraction and purification</LI></UL><P align="left">Use Agarose Gel DNA Recovery Kit to recover the product.</P><UL><LI>Terminal adding A reaction</LI></UL><P align="left">The DNA polymerase used is Q5, so a terminal adding A reaction needs to be performed to turn the target DNA from blunt end to sticky end.
            The sample used is the <EM>GFP</EM> product after gel extraction and purification.</P><UL><LI>Ligation of <EM>GFP </EM>gene and T vector</LI><UL><LI>Add the following to a centrifugal tube</LI></UL><LI>The product after adding A reaction 3 μL </LI><LI>PEASY-T simple vector 1 μL </LI><UL><LI>Incubate at 25℃ for 5 minutes.</LI><LI>Transformation to <EM>E. coli</EM><EM>DH5α</EM></LI><LI>The LB medium should contain Amp and x-gal+2PTG.</LI><LI>Culturing single colony</LI><LI>The bacteria in the agar plate are either white or blue. A white colony is required for the culturing.
                </LI><LI>Amplification culture</LI><LI>Sequence the gene to confirm that it contains the desired bacteria.</LI><LI>Amplification culture in shaking bed at 37℃ overnight.</LI><LI>Plasmid extraction
                    <P align="left"><STRONG>Construction of pGEX-KG vector</STRONG></P></LI><LI>Digestion of the T vector that contains <EM>GFP </EM>gene.</LI></UL><P align="left">Enzyme: Bam H1, Nco 1.</P><UL><LI>Digestion of pGEX-KG</LI></UL><P align="left">Enzyme: Bam H1, Nco 1.</P><UL><LI>Ligation of <EM>GFP </EM>gene and pGEX-KG</LI></UL><P align="left"> </P><UL><LI>Transformation to <EM>E. coli</EM><EM>DH5α</EM></LI></UL><UL><LI>Culturing single colony</LI><LI>Amplification culture</LI><LI>PCR to confirm positive clones.</LI><LI>Amplification culture in shaking bed at 37℃ overnight.</LI><LI>Plasmid extraction
  <P align="left"><STRONG>Prokaryotic expression</STRONG></P></LI><LI>Transformation to <EM>E. coli BL21</EM></LI><LI>Culturing single colony</LI></UL><UL><LI>Observe the green fluorescent under fluorescent microscope. </LI></UL></UL></DIV></DIV><DIV class="row" id="environmental_safety4"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header blue" style="padding-top: 50px"><STRONG>Construction of SmPC1（CPS1）prokaryotic expression system</STRONG></H3><UL><LI><P align="center"><STRONG>Add <EM>GFP </EM>gene to pGEM-KG vector with TEV restriction enzyme cutting site</STRONG></P><P><STRONG>Cloning of <EM>GFP </EM>gene with TEV</STRONG></P></LI><LI>PCR</LI><LI>Design primers that contains correspondent restriction enzyme cutting site (Sal 1 and Hind 3)</LI><LI>Amplify <EM>GFP</EM> gene with TEV</LI><LI>After the PCR is complete, run gel electrophoresis to confirm that the target gene is successfully replicated.</LI><LI>Gel extraction and purification</LI></UL><P align="left">Use Agarose Gel DNA Recovery Kit to recover the product.</P><UL><LI>Terminal adding A reaction</LI></UL><P align="left">The sample used is the <EM>GFP</EM> product with TEV after gel extraction and purification.</P><UL><LI>Ligation of <EM>GFP</EM> and T vector</LI><LI>Add the following to a centrifugal tube</LI><LI>The product after adding A reaction 3 μL </LI><LI>PEASY-T simple vector 1 μL </LI><LI>Incubate at 25℃ for 5 minutes. </LI><LI>Transformation to <EM>E. coli</EM><EM>DH5α</EM></LI></UL><P align="left">The LB medium should contain Amp and x-gal+IPTG.</P><UL><LI>Culturing single colony</LI></UL><P align="left">The bacteria in the agar plate are either white or blue. A white colony is required for the culturing.</P><UL><LI>Amplification culture</LI><LI>Sequence the gene to confirm that it contains the desired bacteria.</LI><LI>Amplification culture in shaking bed at 37℃ overnight. </LI><LI>Plasmid extraction 
              <P align="left"><STRONG>Construction of pGEX-KG vector</STRONG></P></LI><LI>Digestion of the T vector that contains <EM>GFP</EM> with TEV</LI></UL><P align="left">Enzyme: Bam H1, Nco 1. </P><UL><LI>Digestion of pGEX-KG</LI></UL><P align="left">Enzyme: Sal 1, Hind 3. </P><UL><LI>Ligation of <EM>GFP</EM> with TEV and pGEX-KG</LI><LI>Transformation to <EM>E. coli</EM><EM>DH5α</EM></LI></UL><UL><LI>Culturing single colony</LI><LI>Amplification culture</LI></UL><UL><LI>PCR to confirm positive clones.</LI><LI>Amplification culture in shaking bed at 37℃ overnight. </LI></UL><UL><LI>Plasmid extraction
                  <P align="left"><STRONG>Prokaryotic expression</STRONG></P></LI><LI>Transformation to <EM>E. coli</EM><EM>BL21</EM></LI><LI>Culturing single colony</LI><LI>Observe the green fluorescent under fluorescent microscope. </LI></UL></DIV></DIV><DIV class="row" id="environmental_safety5"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header green" style="padding-top: 50px"><STRONG>Add SmCPS1 onto the vector</STRONG></H3><UL><LI><P><STRONG>Cloning of <EM>SmCPS1</EM></STRONG></P></LI><LI>PCR</LI><LI>Amplify <EM>SmCPS1</EM> gene</LI><LI>After the PCR is complete, run gel electrophoresis to confirm that the target gene is successfully replicated.</LI><LI>Gel extraction and purification</LI></UL><P align="left">Use Agarose Gel DNA Recovery Kit to recover the product.</P><UL><LI>Terminal adding A reaction</LI></UL><P align="left">The sample used is <EM>SmCPS1</EM> after gel extraction and purification.</P><UL><LI>Ligation of <EM>SmCPS1 </EM>and T vector</LI><LI>Add the following to a centrifugal tube</LI><LI>The product after adding A reaction 3 μL </LI><LI>PEASY-T simple vector 1 μL </LI><LI>Incubate at 25℃ for 5 minutes. </LI><LI>Transformation to <EM>E. coli</EM><EM>DH5α</EM></LI></UL><P align="left">The LB medium should contain Amp and x-gal+2PTG.</P><UL><LI>Culturing single colony</LI></UL><P align="left">The bacteria in the agar plate are either white or blue. A white colony is required for the culturing.</P><UL><LI>Amplification culture</LI><LI>Sequence the gene to confirm that it contains the desired bacteria.</LI><LI>Amplification culture in shaking bed at 37℃ overnight. </LI><LI>Plasmid extraction</LI></UL><P align="left"><STRONG>Site-directed mutagenesis</STRONG>
              Since <EM>SmCPS1</EM> contains the same restriction enzyme cutting site as the biobricks provided, we need to perform site-directed mutagenesis. </P><P><STRONG> </STRONG></P><P><STRONG> </STRONG></P><P><STRONG> </STRONG></P></DIV></DIV><DIV class="row" id="environmental_safety6"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header blue" style="padding-top: 50px"><STRONG>Cloning of SmCPS1 gene</STRONG></H3><UL><LI>PCR </LI><LI>Design primers that contains correspondent restriction enzyme cutting site (Sma 1 and Xho 1)</LI><LI>Amplify <EM>SmCPS1</EM> gene</LI><LI>After the PCR is complete, run gel electrophoresis to confirm that the target gene is successfully replicated.</LI></UL><UL><LI>Gel extraction and purification</LI></UL><P align="left">Use Agarose Gel DNA Recovery Kit to recover the product.</P><UL><LI>Terminal adding A reaction</LI></UL><P align="left">The sample used is <EM>SmCPS</EM>1 with Sma 1 and Xho 1 after gel extraction and purification.</P><UL><LI>Ligation of <EM>SmCPS1</EM> and T vector</LI><LI>Add the following to a centrifugal tube</LI><LI>The product after adding A reaction 3 μL </LI><LI>PEASY-T simple vector 1 μL </LI><LI>Incubate at 25℃ for 5 minutes. </LI><LI>Transformation to <EM>E. coli DH5α</EM></LI></UL><P align="left">The LB medium should contain Amp and x-gal+2PTG.</P><UL><LI>Culturing single colony</LI></UL><P align="left">The bacteria in the agar plate are either white or blue. A white colony is required for the culturing.</P><UL><LI>Amplification culture</LI><LI>Sequence the gene to confirm that it contains the desired bacteria.</LI><LI>Amplification culture in shaking bed at 37℃ overnight. </LI><LI>Plasmid extraction        </LI></UL></DIV></DIV><DIV class="row" id="environmental_safety7"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header green" style="padding-top: 50px"><STRONG>Ligation of SmCPS1 and pGEX-KG with TEV</STRONG></H3><UL><LI>Digestion of the T vector that contains <EM>SmCPS1</EM> with Sma 1 and Xho 1 Enzyme: Sma 1, Xho 1. </LI><LI>Perform gel electrophoresis to confirm the product.</LI><LI>Gel extraction and purification (Use Agarose Gel DNA Recovery Kit to recover the product.)</LI><LI>Digestion of pGEX-KG - Xho 1 and Sal 1 are isocaudarner, so the product after digestion can be ligased. (Enzyme: Sma 1, Sal 1.) </LI><LI>Ligation of <EM>SmCPS1 </EM>gene with TEV and pGEX-KG</LI><LI>Transformation to <EM>E. coli</EM><EM>DH5α</EM></LI><LI>Culturing single colony</LI><LI>Amplification culture -- A. PCR to confirm positive clones. -- B. Amplification culture in shaking bed at 37℃ overnight. </LI><LI>Plasmid extraction        </LI></UL></DIV></DIV><DIV class="row" id="environmental_safety8"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header red" style="padding-top: 50px"><STRONG>Prokaryotic expression</STRONG></H3><UL><LI>Transformation to <EM>E. coli</EM><EM>BL21</EM></LI><LI>Culturing single colony</LI><LI>Observe the green fluorescent under fluorescent microscope.        </LI></UL></DIV></DIV><DIV class="row" id="environmental_safety9"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header green" style="padding-top: 50px"><STRONG>Prokaryotic expression</STRONG></H3><UL><LI>Transformation to <EM>E. coli</EM><EM>BL21</EM></LI><LI>Culturing single colony</LI><LI>Observe the green fluorescent under fluorescent microscope.        </LI></UL></DIV></DIV><DIV class="row" id="environmental_safety10"><DIV class="col-sm-10 col-sm-offset-1"><H3 class="block_header blue" style="padding-top: 50px"><STRONG>Protein expression</STRONG></H3><UL><LI>Culture and amplify bacteria with a bacteria contained media/fresh media ratio of 1:100. Add IPTG when OD600 is 0.8-1.0 to change the concentration into 0.2 mM, 0.5 mM and 1 mM. Incubate at 16℃ overnight. </LI><LI>Pick out a small amount of bacteria, and add 100 μL 2SDS buffer. Boling for 5 minutes. Perform SDS-page using the supernatant. Choose optimum concentration of IPTG.</LI><LI>Use the optimum IPTG to culture 200 mL bacteria. Centrifuge to collect the bacteria and store at -80℃. </LI><LI>Centrifuge the amplification product for 10 minutes at 4℃, 5000rmp. Collect the bacteria. </LI><LI>Add 20 mL Binding buffer to every 1 L of bacteria. </LI><LI>Add PMSF (50 mM) and fully mix it. </LI><LI>Ultrasonication in an ice bucket for 30 minutes. Perform 10 seconds and then rest for 8 seconds at 30% intensity.</LI><LI>Centrifuge for 40 minutes at 4℃, 14000 g. Collect the supernatant.        </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>