"
Page
Discussion
View source
History
teams
Log in
 
Team:Washington/PRIMER DESIGN
From 2013.igem.org
Home
UW 2013
UW 2012
UW 2011
UW 2010
UW 2009
UW 2008
iGEM Homepage
Light Sensing Experiments
Background
Cloning
Our Systems
Methods
Results
Scientific Sources
Future Plans
Parts Submitted
e.colightTune
Background
App
Methods
Results Summary
Outreach
Activities
Pictures
Protocols
General
Cloning
Light assays
Links
Safety Form
Parts Registry
Parts Documentation
Funding
'
Primer Design
IDT’s Oligo Analyzer
Finnezymes Multiple primer analyzer
Tyler’s primer design video- Primer Design for PCR
General Guidelines:
PCR Primer Design Guidelines
Primer Design
Forward Primer:
Take the beginning of the 18~35 bases of your targeted sequence according to the melting temperature for the gene primer bind to the plasmid you want to transform into. Tms should be around 58*C to match VF2 and VR primers, unless making “hanging primers” by adding the prefix and suffix for cloning - these Tms can be a bit lower to make the combined oligo shorter.
Add the *prefix “gaattcgcggccgcttctagag” at the beginning (extend if needed)
Reverse primer:
Add the *suffix “tactagtagcggccgctgcag” to the end of the targeted sequence
Then reverse complement the whole thing (end of the 20 bases + suffix). Both primers must be listed 5’->3’ for ordering format purposes.
General notes:
-When finding the reverse complement of the end sequence, instead of writing out the reverse complement either use http://www.bioinformatics.org/sms/rev_comp.html or highlight the sequence in Ape, right click and click “Copy Rev-Com”
- The primer has to be at least 18 base pairs long ( NO LESS )
- The primer should end on an G/C for stability
When possible use the Oligo Analyzer or Multiple primer analyzer (ignore Tms):
- Click on hairpin, and looking at the structures the site pops out with. If there is a delta G value of <-2.0 add more base pairs till all values of any resulting structures are >-2.0. The reason for this is because of free energy, the primer might loop onto itself and not function anymore.
Using Ape:
- The highlighted region that you will use for primers will show “Tm” = mel temps. It should be between 40-50 degrees (THIS HAS MORE PRIORITY THAN THE delta G)
*the prefix and the suffix can be found in the following website:
http://partsregistry.org/Help:BioBrick_Prefix_and_Suffix
(use the prefix for the non-inframe cloning, i.e. the longer one)
*match the melting temperature of forward and reverse primer
*A good way to check whether you design the primer correctly is used “find” (control +F) on a new plasmid file you construct by inserting the segment between the prefix and suffix.
Retrieved from " http://2013.igem.org/Team:Washington/PRIMER_DESIGN "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
