"
Page
Discussion
View source
History
teams
Log in
 
Team:BIT-China/Notebook
From 2014.igem.org
project
LAB NOTES
1.Layered AND gates
May 4, 2014—May 11, 2014
We finished the gene synthesis of ipgC, mxiE, exsC, sicA* and invF.They have be proved to have the correct sequence.
May 11, 2014—May 18, 2014
We constructed the standard parts of ipgC, mxiE, exsC, sicA* and invF.
May 18, 2014—May 25, 2014
We designed primers to repair the exsD gene.
May 25, 2014—June 1, 2014
We transformed the terminator B0015,ribosome binding site B0034,Promoter pBAD, J23105 and gene arac.They have be proved to have the correct sequence.We finished the gene ligation of J23105 with B0034 and screened the positive clones.
June 1, 2014—June 8, 2014
We finished the gene ligation of J23105+B0034 with tetR, B0015 with pBAD, B0015 with pTac, B0015 with pTet and screened the positive clones.
June 8, 2014—June 15, 2014
We finished the gene synthesis of exsD and constructed its standard part.
June 15, 2014—June 22, 2014
We finished the gene ligation of J23105+B0034 with araC, J23105+B0034 with lacI, J23105+B0034+tetR with B0015 with pTet and screened the positive clones.
June 22, 2014—June 29, 2014
We finished the gene ligation of J23105+B0034+araC with B0015 +pBAD,J23105+B0034+lacIwithB0015+pTac,J23105+B0034+tetR+B0015+pTet with B0034+E1010 and screened the positive clones.
June 29, 2014—July 6, 2014
We finished the gene ligation of J23105+B0034+araC+B0015 +pBAD withB0034+E1010, J23105+B0034+lacI+B0015+pTac with B0034+K592009 and screened the positive clones.
July 6, 2014—July 13, 2014
We prepared the inducer Ara and IPTG.We verified the function of circuit J23105+B0034+araC+B0015+pBAD+B0034+E1010 and the function of  circuit J23105+B0034+lacI+B0015+pTac+B0034+K592009.
July 13, 2014—July 20, 2014
We finished the gene ligation of J23105+B0034+lacI+B0015+pTac with B0034+E1010 and screened the positive clones.We verified the function of circuit J23105+B0034+araC+B0015+pBAD+B0034+E1010 and the function of  circuit J23105+B0034+lacI+B0015+pTac+B0034+E1010 and the function of circuit J23105+B0034+tetR+B0015+pTet+B0034+E1010.
July 20, 2014—July 27, 2014
We redesigned the reverse primers of ipgC, mxiE, invF and sicA* .We finished the gene synthesis of ipgC, mxiE, invF and sicA*.
July 27, 2014—August 3, 2014
We constructed the standard parts of ipgC, mxiE, invF and sicA*.
August 3, 2014—August 10, 2014
We finished the gene ligation of ipgC with B0015, mxiE with B0015,  invF with B0015, sicA* with B0015 and screened the positive clones.
August 10, 2014—August 17, 2014
We finished the gene ligation of B0034 with ipgC+B0015, B0034 with mxiE+B0015, B0034 with invF+B0015, B0034 with sicA*+B0015 and screened the positive clones.
August 17, 2014—August 24, 2014
We finished the gene ligation of J23105+B0034+araC+B0015+pBAD+B0034+E1010withB0034+ipgC+B0015 , J23105+B0034+lacI+B0015+pTac+B0034+E1010 with B0034+ mxiE+B0015, B0034+invF+B0015 with psicA, and screened the positive clones.
August 24, 2014—August 31, 2014
We finished the construction of plasmid I.
August 31, 2014—September 7, 2014
We finished the gene ligation of pipaH* with B0034+E1010, J23105+B0034+tetR+B0015+pTet+B0034+E1010withB0034+invF+B0015+psicA and screened the positive clones.
September 7, 2014—September 14, 2014
We co-constructed the plasmid I with pipaH* +B0034+E1010 and screened the positive clones.We verified the function of plasmid I.
September 14, 2014—September 21, 2014
We designed primers to get the mutants of replacing promoter J23105 with J23119.
2.MIN system
June 1, 2014—June 8, 2014
We started our experiment and decided to use hok system to control the concentration of bacteria.
June 8, 2014—June 15, 2014
Through the PCR, we got the HSP from the E.coli’s genome.
June 15, 2014—June 22, 2014
Through the PCR, we got the PS from the E.coli’s genome.At the same time, we failed to construct the HSP to the standard part.
June 22, 2014—July 6, 2014
We tried to construct the HSP and PS to the standard parts many times, but it failed.
July 6, 2014—July 13, 2014
Due to we couldn’t construct the hok system as standard parts, so we changed our system. We decided to use Min system to control the concentration of the bacteria.
July 13, 2014—July 20, 2014
We finished the gene ligation of J23119+B0034.
July 20, 2014—July 27, 2014
We finished the gene ligation of R0063+B0034 and JB+C0062.
July 27, 2014—August 3, 2014
We finished the gene ligation of J23119+B0034 and constructed the JBC(J23119+B0034+C0062) and RB(R0063+B0034) standard parts.
August 3, 2014—August 10, 2014
We finished the gene ligation of J23119+B0034 and tried to ligate it with C0060.
August 10, 2014—August 17, 2014
We ligated the J23119+B0034+C0062+R0063+B0034 by OEpcr, but we failed to construct its standard part.
August 17, 2014—August 24, 2014
Through the pcr, we got the MinC from the E.coli’s genome and constructed its standard part.
August 24, 2014—August 31, 2014
Through the pcr, we got the MinD from the E.coli’s genome and constructed its standard part.
August 31, 2014—September 7, 2014
Because the mutation of  RB(R0063+B0034), we had to ligate the R0063 and B0034 again.
September 7, 2014—September 14, 2014
We finished the gene ligation of  R0063+B0034 again.
September 14, 2014—September 21, 2014
We finished the gene ligation of J23119+B0034+C0060 and B0034+MinD.
3.sRNA system
May 4, 2014—May 11, 2014
We designed primers for synthesis gene MicC.
May 11, 2014—May 18, 2014
We finished the gene synthesis of regulation box1.It have be proved to have the correct sequence.
May 18, 2014—May 25, 2014
We constructed the standard devices of “box1” and “box2”.
May 25, 2014—June 1, 2014
We constructed the “sRNA regulation system” which contain standard devices of “box1” and “box2”.
June 1, 2014—June 8, 2014
We designed primers for add gene ipgC to box2.
June 8, 2014—June 15, 2014
We constructed the ipgC to box2.
June 15, 2014—June 22, 2014
We designed primers for add gene sicA to box2.
June 22, 2014—June 29, 2014
We constructed the sicA to box2.
June 29, 2014—July 6, 2014
We designed primers for add sequence anti-ipgC-24bp to box1.
July 6, 2014—July 13, 2014
We constructed the anti-ipgC-24bp to box1.
July 13, 2014—July 20, 2014
We designed primers for add sequence anti-sicA-24bp to box1.
July 20, 2014—July 27, 2014
We constructed the anti-sicA-24bp to box1.
July 27, 2014—August 3, 2014
We designed primers for add gene MinC to box2.
August 3, 2014—August 10, 2014
We constructed the MinC to box2.
August 10, 2014—August 17, 2014
We designed primers for add gene MinD to box2.
August 17, 2014—August 24, 2014
We constructed the MinD to box2.
August 24, 2014—August 31, 2014
We designed primers for add sequence anti-MinC-24bp to box1.
August 31, 2014—September 7, 2014
We constructed the anti-MinC-24bp to box1.
September 7, 2014—September 14, 2014
We designed primers for add sequence anti-MinD-24bp to box1.
September 14, 2014—September 21, 2014
We constructed the anti-MinD-24bp to box1.
September 22, 2014—September 28, 2014
Measured the β- galactosidase activity.
PROTOCOL
1.Artificial synthesis the genes
Gene synthesis is the step during which oligonucleotides (oligos) are combined into DNA fragments of several hundred bases in length. Numerous protocols have been described and extensively reviewed. In our project, we use the polymerase chain assembly (PCA) to synthetic ipgC, InvF, sicA and mixE.
We finished the gene synthesis of ipgC, mxiE, exsC, sicA* and invF.They have be proved to have the correct sequence.
First, we disassemble target gene into several oligos using GENE DESIGN, and we synthesis these oligos in GENEWIZ.
Then, the oligos that are synthesized to build the target gene. The pool of oligos is assembled in equimolar amounts and allowed to anneal. The annealed oligos are extended in the 3' direction until the end of their partner oligo is reached. The double-stranded DNA is melted and reannealed with extension products and any remaining oligos. Each extension reaction results in progressively longer products and full length products are eventually synthesized. At this step, the terminal oligos are added to the reaction and full length products from the previous reactions are amplified by PCR and subsequently cloned and sequenced.
Reference ：Stemmer,W.P. et al. (1995) Single-step assembly of a gene and entire plasmid from large numbers of oligo deoxyribonucleotides. Gene164, 49–53
2.One-step Site-directed Mutagenesis
We insert target sequence into plasmid to construct our desired synthetic small RNAs. Our protocol is rather simple. We just use PCR to perform it, one-step site-directed mutagenesis. In our experiments, we used it many times. For instance, one of steps is that we inserted ipgc sequence into Box2 (blue target in Fig. 2-1).
IPGC sequence is ATGTCTTTAAATATCACCGAAAAT
To construct box2 for testing silencing effect, we should insert the 24-nt-long sequence ATGTCTTTAAATATCACCGAAAAT between the RBS and the lacZ gene. (Fig. 2)
The two oligonucleotide primers we used for one-step site-directed mutagenesis to construct the Box2 are shown in following figure.
A new plasmid inserted into ipgc sequence can be constructed by conventional PCR by using designed primers and the PSB1C3 plasmid containing Box1 and Box2, as Fig.4, is a template. The PCR results in the generation, containing ipgc sequences.
LAB NOTES
PROTOCOL
Retrieved from " http://2014.igem.org/Team:BIT-China/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
