<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tokyo_Tech_Projects_Urea-cooler_index_htm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tokyo_Tech/Projects/Urea-cooler/index.htm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/Urea-cooler/index.htm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/Urea-cooler/index.htm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tokyo_Tech/Projects/Urea-cooler/index.htm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tokyo Tech/Projects/Urea-cooler/index.htm</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Tokyo Tech 2011</TITLE></P><DIV class="top"><DIV id="navigation"><UL id="TopMenu"><LI id="menu_Home"><A href="https://2011.igem.org/Team:Tokyo_Tech">Home</A></LI><LI id="menu_Project">
				Project
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/index.htm">RPS-Game</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/making-rain/index.htm">Make it Rain</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm">Urea Coolers</A></LI></UL><LI id="menu_data_page"><A href="https://2011.igem.org/Team:Tokyo_Tech/DataPage.htm">Data page</A></LI><LI id="menu_Modeling">
				Modeling
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/RPS-game/RPS-game">RPS-Game</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/urea-cooler">Urea Coolers</A></LI><LI id="menu_HumanPractice"><A href="https://2011.igem.org/Team:Tokyo_Tech/HumanPractice.htm">Human Practice</A></LI><LI id="menu_Sitemap"><A href="https://2011.igem.org/Team:Tokyo_Tech/sitemap.htm">Sitemap</A></LI><LI id="menu_Extra">
				More
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Attribution_and_Contributions.htm">Attribution and Contributions</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/notebook">NoteBook</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Sponsers.htm">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Collaboration.htm">Collaboration</A></LI></DIV></DIV><DIV style="min-height: 10700px; float: left;"><DIV id="LeftMenu"><UL><LI><A href="#abst">1. Abstruct</A></LI><LI><A href="#2.Gene">2. Genetic Engineering for Urea Production</A></LI><LI><A href="#2.1Intro">2.1 Introduction</A></LI><LI><A href="#2.2">2.2 Charcterization of <I>roc</I>F and Arg box</A></LI><LI><A href="#2.3">2.3 Characterization of Ptrc-RBS-<I>rocF</I>-Arg box</A></LI><LI><A href="#2.4">2.4 Use of mutation in <I>argR</I> gene</A></LI></UL><LI><A href="#Flux">3. Flux analysis for providing more urea </A></LI><LI><A href="#abstract">3.1 Abstract</A></LI><LI><A href="#intro">3.2 Introduction</A></LI><LI><A href="#modes">3.2.1 What is Elementary Flux Modes?</A></LI><LI><A href="#analyzing">3.2.2 Analyzing the function of the compounds involved in the Urea Cycle by determining the elementary flux modes</A></LI><LI><A href="#finding">3.2.3 Finding Modes to Increase the Urea production by <I>E. coli</I></A></LI><LI><A href="#results">3.3 Results</A></LI><LI><A href="#funciton">3.3.1 Analyzing the function of the compounds involved in the Urea Cycle by determining the elementary flux modes</A></LI><LI><A href="#production">3.3.2 Finding Modes to Increase the Urea production by <I>E. coli</I></A></LI><LI><A href="#future">3.4 Future Work</A></LI></DIV></DIV><DIV class="main"><H1> Urea cooler </H1><H2 id="abst">1. Abstract</H2><P>
			Coolers can be made by dissolving urea in water because it is an endothermic reaction. So, in order to obtain urea, we made the urea cycle in<I> E.coli</I> by introducing <I>rocF</I> gene encoding arginase, which is an enzyme that converts L-arginine to L-ornithine and urea. But just by introducing <I>rocF</I>, only a little amount of urea can be produced because arginine biosynthesis is repressed by the arginine repressor. Therefore, we tried to derepress arginine biosynthesis in two ways. One is by introducing arginine operator sequences(Arg boxes), which bind the arginine repressor. The other is by using a strain that carries a mutation in a gene which encodes arginine repressor. Furthermore, we studied elementary flux modes to provide more urea. As a result, we found out that the artificial urea production system is robust in a stoichiometric point of view. The analysis also revealed that supplementation of arginine, glutamic acid and aspartic acid would increase urea production rate.<TABLE style="text-align:center;" align="center"><TBODY><TR><TD>
						Fig. 3 Urea concentration in growth media 1 hour after IPTG induction 
					</TD><TD>
						Fig. 5  The reactions related with the urea cycle
					</TD></TR></TBODY></TABLE></P><H2 id="2.Gene">2. Genetic Engineering for Urea Production</H2><H3 id="2.1Intro">2.1 Introduction</H3><P>
			Coolers can be made by dissolving urea in water, since dissolution of urea in water is an endothermic reaction (-57.8 cal/g). So we came up with an idea of creating <I>E. coli</I> that can synthesize urea. Originally, <I>E. coli</I> has all the enzymes in the urea cycle except for arginase, which converts L-arginine into L-ornithine and urea(Fig. 1). So we attempted to complete the urea cycle in <I>E. coli</I> by introducing a gene which encodes arginase and obtain urea.
			In this work, introduction of the <I>Bacillus subtilis</I><I>rocF</I> gene which encodes arginase on a standardized plasmid completed urea cycle and enabled<I> E.coli</I> to produce urea as reported by TUCHMAN <I>et al</I>.,(1997).
 		</P><CENTER><DIV class="graph_title">
	 		Fig. 1 Addition of a gene which encoding arginase completes urea cycle in <I>E.coli</I>.
		</DIV></CENTER><P>
			However, just by introducing arginase, <I>E. coli</I> produces 
			only a little amount of urea. TUCHMAN <I>et al</I>. proposed that catabolite 
			repression in arginine biosynthesis pathway is the main reason for 
			the low efficiency of production(TUCHMAN <I>et al</I>., 1997). The bacterial 
			arginine biosynthetic genes are all regulated via a common repressor 
			protein (arginine repressor) encoded by the <I>argR</I> gene. The arginine represor is activated in the presence of arginine (Fig. 2).
TUCHMAN <I>et al</I>. circumvented the repression by introduction of 
			arginine operator sequences (Arg boxes), which bind the arginine repressor.  
			With the arginine repressor binding to Arg boxes, the amount of the arginine 
			repressor which can repress arginine biosynthesis is reduced.  
			In this work, we tried two ways of derepressing arginine biosynthesis. One way 
			is by introducing the Arg boxes as previous work. The other way is by using an 
			<I>E. coli</I> strain that has an <I>argR</I> deletion genotype so that 
			the repressor is not synthetized.  
		</P><CENTER><DIV class="graph_title">
			Fig. 2 Arginine biosynthesis is repressed by the arginine repressor in the presence of arginine.
		</DIV></CENTER><H3 id="2.2">2.2 Charcterization of <I>rocF</I> and Arg box</H3><P>
                Introduction of <I>rocF</I> gene on our part <A href="http://partsregistry.org/Part:BBa_K649301">BBa_K649301</A> led to production of more urea compared to negative controls. Therefore, we confirmed that insertion of <I>rocF</I> gene resulted in arginase production and completed the urea cycle in <SPAN class="name">E. coli</SPAN> as expected.
                </P><DIV class="graph_title">
	 		Fig. 3 Urea concentration in growth media when <I>rocF</I> or Arg box were introduced strain MG1655.
		</DIV><P>
			As a way to derepress arginine biosynthesis we introduced Arg box on our part <A href="http://partsregistry.org/Part:BBa_K649401">BBa_K649401</A> in addition to <I>rocF</I> gene (<A href="http://partsregistry.org/Part:BBa_K649301">BBa_K649301</A>). This led to production of even more urea. These results show that Arg boxes are effectively derepressing arginine production by deactivating the arginine repressor. <A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/data#1.1">Detailed information is shown here</A>.
		</P><H3 id="2.3">2.3 Characterization of Ptrc-RBS-<I>rocF</I>-Arg box</H3><P>
			In this study Arg boxes and <I>rocF</I> were introduced separately on high-copy-number plasmids (pSB1C3) and low-copy-number plasmids (pSB3K3). We also tested the effect of Arg boxes when they were introduced downstream of <I>rocF</I> gene on low-copy-number plasmids (pSB3K3 and pSB6A1). Here, Arg boxes didn’t work effectively. This is probably because a low-copy-number plasmid is not capable of introducing enough number of Arg boxes to effectively deactivate the arginine repressor.<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/data#2.1"> Detailed information is shown here</A>.
		</P><H3 id="2.4">2.4 Use of mutation in <SPAN class="gene">argR</SPAN> gene</H3><CENTER><DIV class="graph_title">
	 		Fig. 4 Urea concentration in growth media when <I>rocF</I> gene on pSB3K3 was introduced in strain MG1655 (<SPAN class="gene">argR</SPAN> +) or JE6852 (<SPAN class="gene">argR</SPAN> -).
		</DIV></CENTER><P>
			As another way to derepress arginine biosynthesis, we used an <SPAN class="name">E. coli</SPAN> strain that has an <SPAN class="gene">argR</SPAN> deletion genotype (JE6852, This strain was obtained from National Institute of Genetics). In our assay results, much more urea was produced when <I>rocF</I> gene was introduced in JE6852 than in MG1655 as expected.<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/data#2.1"> See more details about this experiment.</A></P><H2 id="Flux"> 3. Flux analysis for providing more urea </H2><H3 id="abstract">3.1 Abstract </H3><P>
			This section is about a metabolic engineering study we did about the urea cycle. On the first part we show how we used “elementary flux modes” (Schuster <I>et al</I>., 2000) to analyze the function of the compounds involved in the urea cycle. Mainly we deduced which compounds act as sources of carbon and sources of nitrogen for the production of urea. On the second part of this study we show how we determined elementary flux modes of the urea cycle to find ways to increase the yield of urea. We focused on a strategy which involves increasing the concentration of four components of the cycle and which we concluded would yield more urea. To confirm our results future experiments will be done.
		</P><DIV class="graph_title">
			Fig. 5 The reactions related with the urea cycle
		</DIV>
		*The orange letters are the abbreviated names of the enzymes involved. The red letters are the enzyme expressed by introducing <SPAN class="gene"><I>rocF</I></SPAN> gene. For complete names of the enzymes see <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/tables#table3">Table 3</A>.


		<H3 id="intro">3.2 Introduction</H3><H4 id="modes">3.2.1 What is Elementary Flux Modes?</H4><P>
			In Metabolic Engineering, mathematical modeling is an effective way to increase the products of a reaction. In particular, Flux Analysis, which is based on the hypothesis that the system is in a steady state, is effective to find how to increase these products.  The concept of elementary flux mode provides a mathematical tool to define and comprehensively describe all metabolic routes that are both stoichiometrically and thermodynamically feasible for a group of enzymes. As a method of metabolic flux analysis, it is based on the hypothesis that the concentration of the reactants and products involved in the cycle does not change.
		</P><H4 id="analyzing">3.2.2 Analyzing the function of the compounds involved in the Urea Cycle by determining the elementary flux modes</H4><P>
			By determining the elementary flux modes of a cycle we can have a more clear view of the function of each of the compounds involved in the cycle being analyzed. Based on the elementary flux modes of the urea cycle, in this study we could deduce that HCO<SUB>3</SUB><SUP>-</SUP> acts as the source of carbon for urea production and that both L-glutamine and NH<SUB>3</SUB> act as nitrogen sources for the formation of urea.
 		</P><CENTER><SPAN style="font-size:12px;">→</SPAN></CENTER><H4 id="finding">3.2.3 Finding Modes to Increase the Urea production by <I>E. coli</I></H4><P>
			In this study we determined elementary flux modes to maximize urea production by <I>E. coli</I>. We found that there are two main strategies to increase urea production: one is to increase the amount of carbamoyl phosphate (which formation is known to be the rate-limiting step of the urea cycle). The other one is to increase the concentration of four components of the urea cycle: L-ornithine, L-citrulline, N-(L-arginino)succinate and L-arginine. We deduced the latter strategy by determining the elementary modes of the urea cycle, and therefore in this study we will focus on the description of this strategy.
		</P><DIV class="graph_title">
			Fig. 8 Two ways to increase urea production 
		</DIV><H3 id="results">3.3 Results</H3><P>
			In our study, we considered the enzymatic reactions shown in <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/tables#table3">Table 3</A> to determine the elementary flux modes related to urea production by <I>E. coli</I>. The scheme the overall reaction system is shown in Fig. 5 below.
		</P><DIV class="graph_title">
			Fig. 5 The reactions related with the urea cycle
		</DIV><P>
			*The orange letters are the abbreviated names of the enzymes involved. The red letters are the enzyme expressed by introducing <SPAN class="gene">rocF</SPAN> gene. For complete names of the enzymes see <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/tables#table3">Table 3</A>.
		</P><P>
			By determining the elementary flux modes to produce urea inside <I>E. coli</I>, we found two important results:
		</P><OL><LI><P>
				We confirmed both L-glutamine and NH<SUB>3</SUB> act as nitrogen providers in the urea cycle, as well as deducing that HCO<SUB>3</SUB><SUP>-</SUP> acts as the source of carbon for urea production. These modes did not make use of organic intermediates. Even though L-glutamine is consumed in order to to transfer the side-chain ammonium group needed for the production of carbamoyl phosphate (which in turn transfers the ammonium group to the urea cycle), free ammonium ion can restore L-glutamine from L-glutamate (which is a byproduct of the reaction that yields carbamoyl phosphate as a product).
			</P></LI><LI><P>
				We concluded that increasing the concentration of L-ornithine will increase the concentration of three related compounds (L-citrulline, N-(L-arginino)succinate, and L-arginine) and this will ultimately lead to an increase in the production of urea. We also noted that since the L-aspartate amino acid, which is needed in the urea cycle we considered(Fig. 5), is normally consumed in protein biosynthesis, so it should be supplied in the culture medium or synthetized by <SPAN class="name">E. coli</SPAN> in order to be able to increase the amount of urea and to maintain the cycles that produce it.
			</P></LI></OL><P>
			Below is a detailed description of these three results.
		</P><H4 id="funciton">3.3.1 Analyzing the function of the compounds involved in the Urea Cycle by determining the elementary flux modes</H4><P>
			The first step was to determine the flux modes which need of L-glutamine as an input (Mendel <I>et al</I>., 1996). We did this by calculations based on a matrix as the tableau shown below.
		</P><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/method" align="right">Details about the calculations can be found here</A><P>
		We found eight modes that can produce urea without using organic intermediates. These are shown in <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/figures#Elem1">Fig. 6</A>. Each reaction formula is shown in <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/tables#table4">Table 4</A>. In particular, we focused on one the mode displayed in Fig. 7.
	</P><DIV class="graph_title"><DIV style="font-size:larger"><B>2NH<SUB>3</SUB> + HCO<SUB>3</SUB><SUP>-</SUP> + 3ATP + H<SUB>2</SUB>O + NADPH + NAD<SUP>+</SUP>→ Urea + 2ADP + AMP + 2Pi + PPi + NADP<SUP>+</SUP> + NADH</B></DIV>
		Fig. 7 One of the urea producing cycles leaded by the concept of elementary flux modes
		*The numbers indicate the relative flux carried by the enzymes.
	</DIV><P>
		As shown in Fig. 7, we deduced that the carbon atom of urea is provided from HCO<SUB>3</SUB><SUP>-</SUP> , which is a byproduct of respiration and therefore is already an abundant compound in the bacterial cytoplasm. On the other hand, we also confirmed that carbamoyl phosphate is a nitrogen source for urea production.We also found that the function of L-glutamine in the urea cycle is to provide nitrogen for urea production via carbamoyl phosphate, because ammonium ion can restore L-glutamine from L-glutamate (which is a
byproduct of the reaction that yields carbamoyl phosphate as a product).This conclusion was confirmed experimentally by Mendel <I>et al</I>. (1996).  Also, since only providing a nitrogen source is enough to increase urea production by <I>E. coli</I>, we can also conclude that the aritificial urea cycle in <I>E. coli</I> is stoichiometrically well designed. By comparing Fig. 5 and Fig. 7 we can also observe that, in Fig. 7, the reaction which converts L-glutamate to L-ornithine is not needed for urea production.
	</P><H4 id="production">3.3.2 Finding Modes to Increase the Urea production by <I>E. coli</I></H4><P>
		There are two ways to obtain more products from a cycle of reactions: increasing the speed the reactions and increasing the concentration of the reactants. This becomes obvious if we think of the cycle as a track which is travelled by cars (the reactants), and the products as the total sum of the number of laps made by every car. If we double the speed of the cars the number of laps will also double (Fig. 8, lower left). Similarly, if we double the number of cars the number of laps will double as well (Fig. 8, lower right). We applied this analogy to the urea cycle, where the metabolites in the cycle are represented by the cars and the total number of laps represents the total urea yield (as shown in the figure below).
		Increasing the velocity of the cars corresponds to increasing the amount of carbamoyl phosphate in the urea cycle, because the reaction which converts L-glutamine to carbamoyl phosphate is the rate-limiting reaction of the cycle. On the other hand, increasing the number of the cars correspond to increasing the concentration of the compounds of the urea cycle. We focused on increasing the concentration the compounds of the urea cycle to find ways to increase the urea yield.
	</P><DIV class="graph_title"><P>Fig. 8 Two ways to increase urea production</P></DIV><P>
		L-ornithine, L-citrulline, N-(L-arginino)succinate and L-arginine are four important compounds of the urea cycle. As can be seen I Fig.7, these compounds form a sub-cycle that directly yields urea. Therefore, by increasing the yield of this cycle we can increase the production of urea in <I>E. coli</I>.
	</P><P>
		We determined the elementary modes which produce these four important compounds. 
		All elementary flux modes which produce these compounds from L-glutamine or from compounds in TCA cycle produce L-ornithine as intermediate or final product (these modes are shown in <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/figures#Elem2">Fig. 9</A> and each reaction formula is shown in <A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/tables#table5">Table 5</A>, it can be concluded that increasing the concentration of L-ornithine will increase the production of urea. One of the L-ornithine producing modes is shown in Fig. 10.
	</P><DIV class="graph_title"><DIV style="font-size:larger"><SMALL><B>2-oxoglutarate + NH<SUB>3</SUB> + acetyl-CoA + ATP + 3NADPH + 3H<SUP>+</SUP>→ L-ornithine + CoASH + acetate + ADP + Pi + H<SUB>2</SUB>O + 3NADP<SUP>+</SUP></B></SMALL></DIV>
			Fig. 10 One of the L-ornithine producing pathways from intermediates of TCA cycle
			*The numbers indicate the relative flux carried by the enzymes.
		</DIV><P>
		The reactions we determined increase the above mentioned four compounds of the urea cycle are shown in Fig. 9. All modes include the reaction that yields L-ornithine by converting L-glutamate to L-ornithine. 
		We also confirmed that <SPAN class="name">E. coli</SPAN> has no feasible routes for production of these four components other than those indicated in Fig. 5. Therefore, we can conclude that the reaction which converts L-glutamate to L-ornithine is a key reaction to increase the reaction rates in the urea cycle and thereby to increase urea production. It should be noted that one of the reactions of the cycle shown in Fig. 5 (the one in the lowest part of the image) requires ATP, NADPH, Acetyl-CoA, and L-glutamate. With the exception of L-glutamate, all of these compounds are already abundant in the cell. Therefore, in future wet experiments, we will focus on studying the effects of supplying L-glutamate to <I>E. coli</I>. We will confirm that by supplying L-glutamate the concentration of intermediates like L-ornithine can be increased and therefore urea production can be increased.
	</P><P>
		Furthermore, to supply L-glutamine, L-glutamate and L-arginine is effective way to increase the amount of ornithine.(Fig. 11)
	</P><DIV align="center"><SPAN class="graph_title">Fig. 11 Ornithine is made from L-glutamine, L-glutamate and L-arginine</SPAN></DIV><P>
		We also noted that since L-aspartate is consumed in protein biosynthesis, this amino acid should be supplied from in the medium or produced by <I>E. coli</I> itself not only for increasing the amount of urea production, but also for maintaining the cycle.
	</P><P>
		In conclusion, increasing the concentration of L-glutamine, L-glutamate, L-arginine and L-aspartate is an effective way to increase the amount of urea produced.
	</P><H3 id="future">3.4 Future Work</H3><P>
		As a future work, we will experimentally confirm our results to show that activating the reactions which supply these amino acids is an effective way to increase the production of urea by <I>E. coli</I>.
	</P><H3 id="reference">Reference</H3>
	[1] Stefan Schuster, <I>et al.</I> A general definition of metabolic pathways useful for systematic organization and analysis of complex metabolic network, Nat Biotechnol(2000) 18:326-32
	[2] Mendel Tuchman, <I>et al.</I> Enhanced production of arginine and urea by genetically engineered Escherichia coli K-12 strains, Apple Environ Microbiol(1997) 63: 38-8</DIV><DIV style="bottom: 0;left: 30; position:fixed; background-color: #000000; opacity:0.8;"><A onclick="scrollTo(0,0);return false;" onfocus="this.blur();" style="color:#FFFFFF;">Return to Page Top</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm">http://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/Urea-cooler/index.htm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/Urea-cooler/index.htm&amp;oldid=261306" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>