<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tokyo_Tech_Projects_RPS-game_index_htm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tokyo_Tech/Projects/RPS-game/index.htm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tokyo_Tech/Projects/RPS-game/index.htm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/index.htm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/index.htm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tokyo_Tech/Projects/RPS-game/index.htm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tokyo Tech/Projects/RPS-game/index.htm</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Tokyo Tech 2011</TITLE></P><DIV class="top"><DIV id="navigation"><UL id="TopMenu"><LI id="menu_Home"><A href="https://2011.igem.org/Team:Tokyo_Tech">Home</A></LI><LI id="menu_Project">
				Project
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/index.htm">RPS-Game</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/making-rain/index.htm">Make it Rain</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm">Urea Coolers</A></LI></UL><LI id="menu_data_page"><A href="https://2011.igem.org/Team:Tokyo_Tech/DataPage.htm">Data page</A></LI><LI id="menu_Modeling">
				Modeling
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/RPS-game/RPS-game">RPS-Game</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/urea-cooler">Urea Coolers</A></LI><LI id="menu_HumanPractice"><A href="https://2011.igem.org/Team:Tokyo_Tech/HumanPractice.htm">Human Practice</A></LI><LI id="menu_Sitemap"><A href="https://2011.igem.org/Team:Tokyo_Tech/sitemap.htm">Sitemap</A></LI><LI id="menu_Extra">
				More
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Attribution_and_Contributions.htm">Attribution and Contributions</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/notebook">NoteBook</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Sponsers.htm">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Collaboration.htm">Collaboration</A></LI></DIV></DIV><DIV style="min-height: 20600px; float: left;"><DIV id="LeftMenu"><UL><LI><A href="#1">1. The Hands</A></LI><LI><A href="#2">2. The Judges</A></LI><LI><A href="#2.1">2.1 Using AND-Gate promoters to create Judges</A></LI><LI><A href="#2.2">2.2 Creating Parts that responded correctly to our set of Signaling Molecules</A></LI><LI><A href="#2.3">2.3 Improving PlsrA</A></LI><LI><A href="#2.4">2.4 Improving Plas</A></LI></UL><LI><A href="#3">3. The Randomizers</A></LI><LI><A href="#3.1">3.1 Single Colony Isolation</A></LI><LI><A href="#3.2">3.2 Conditional Knockout by Recombination</A></LI><LI><A href="#3.2.1">3.2.1 The Requirements</A></LI><LI><A href="#3.2.2">3.2.2 The Mechanism</A></LI><LI><A href="#3.2.3">3.2.3 Testing the <I>Lox</I> Cassettes</A></LI><LI><A href="#3.2.4">3.2.4 Playing Fair: Future Work</A></LI><LI><A href="#3.3">3.3 Survival of one strain</A></LI><LI><A href="#3.3.1">3.3.1 Introduction: Very small differences determine who will survive</A></LI><LI><A href="#3.3.2">3.3.2 Adjusting the Model to create a True Randomizer</A></LI><LI><A href="#3.3.3">3.3.3 How the Three Types of Bacteria Compete for Survival</A></LI><LI><A href="#3.3.4">3.3.4 The Old Model</A></LI><LI><A href="#3.3.5">3.3.5 Our New Model</A></LI><LI><A href="#3.3.6">3.3.6 The Biological Meaning of our Model</A></LI><LI><A href="#3.3.7">3.3.7 Making it Obvious</A></LI></DIV></DIV><DIV class="main"><H1> Rock-Paper-Scissors game </H1><H2> Introduction </H2><DIV align="center"><SPAN class="top"> The Hands </SPAN></DIV><P>
		The first step towards making an RPS game that can be played 
		between humans and bacteria is giving each player a set of 
		signaling molecules through which they can communicate their 
		choice of rock, paper or scissors. For that purpose we created 
		two sets of three signaling molecules corresponding each to rock, paper or scissors. 
		For humans we used IPTG, aTc and salicylate, respectively. 
		For <SPAN class="name">E. coli</SPAN> we used 3OC6-HSL, 3OC12-HSL and AI-2, respectively.
	</P><DIV align="center"><SPAN class="top"> The Judges </SPAN></DIV><P>
		Although we defined a set of six signaling molecules that can be used to 
		play the RPS game, we still need to find a way to know who wins the game. 
		To know who the winner of each game is, we designed a set of 
		<SPAN class="name">E. coli</SPAN> that act as judges. Because there were no working parts related to AI-2 or 3OC12-HSL, we constructed new working  lasI promoter, lsrA promoter and LsrR coding gene. 
	</P><DIV align="center"><SPAN class="top"> The Randomizers </SPAN></DIV><P>
		Although our set of six signaling molecules allows us to play RPS with 
		<SPAN class="name">E. coli</SPAN>, we must make sure <SPAN class="name">E. coli</SPAN> 
		can choose any of its three signaling molecules with the same probability 
		in order to be able to play RPS fairly and properly. 
		To do so, we designed three kinds of randomizers.
	</P><H2 id="1">1. The Hands</H2><P>
			The first step towards making an RPS game that can be played between 
			humans and bacteria is giving each player a set of signaling molecules 
			through which they can communicate their choice of rock, paper or scissors. 
			For that purpose we created two sets of three signaling molecules 
			corresponding each to rock, paper or scissors. For humans we used IPTG, 
			aTc and salicylate, respectively. 
			For <SPAN class="name">E. coli</SPAN> we used 3OC6-HSL, 3OC12-HSL and AI-2, respectively.
		</P><H2 id="2" style="clear:both;">2. The Judges</H2><P>
			Although we defined a set of six signaling molecules that can be used 
			to play the RPS game, we still need to find a way to know who wins the game. 
			To know who the winner of each game is, 
			we designed a set of <SPAN class="name">E. coli</SPAN> that act as judges. 
			Each Judge <SPAN class="name">E. coli</SPAN> has an AND-gate promoter and 
			a fluorescent protein gene that is expressed when the AND-gate promoter 
			is activated. In this way, the Judge <SPAN class="name">E. coli</SPAN> can 
			let us know its decision by producing GFP, RFP or CFP to indicate whether 
			humans win, lose or it is a tie, respectively.
		</P><H3 id="2.1" style="clear:both;">2.1 Using AND-Gate promoters to create Judges</H3><P>
			The first step to make the Judge <SPAN class="name">E. coli</SPAN> was to 
			find a logic device which could allow the Judge to decide who the winner of the RPS game was. 
			We found that the AND-gate promoters would fit perfectly for that purpose, 
			since they can take two signaling molecules as inputs and produce one indicator 
			as output. Since each of the players has a set of three different signaling 
			molecules, we need a set of nine Judges, each of which has an AND-gate promoter 
			that is activated only by one of the nine possible pairs of signaling molecules. 
			These combinations are shown in the image below.
		</P><P>
			Our next mission was then to check if there were AND-gate promoters BioBricks 
			that we could use. We searched in the Registry and found a potential AND-gate 
			promoter designed by iGEM 2007's team Tokyo_Tech. This potential AND-gate 
			promoter is designed to be activated by the addition of both IPTG and 3OC6-HSL. 
			However, there was no data showing the IPTG dependency of this promoter, 
			so we did experiments and confirmed this dependency for the first time in iGEM. 
			We concluded that the addition of both IPTG and 3OC6-HSL regulates the activity 
			of this AND-gate promoter. In this way, we completed the construction of one of 
			the Judges <SPAN class="name">E. coli</SPAN>, which proves in principle that 
			our game is feasible. To know the detailed method about this assay, 
			please see <A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#4.">here</A>.
		</P><DIV style="float:left"><SPAN class="graph_title">Fig. 2.1 Tokyo_Tech AND-gate promoter</SPAN></DIV><P>
			This Plux-lac hybrid promoter contains two LacI operators, a LuxR operator and luxR. 
			We introduced this part into LacI expressing <SPAN class="name">E. coli</SPAN> strain. 
			Because IPTG controls the binding of LacI to two LacI-operator parts and 3OC6-HSL 
			controls the binding of LuxR to a LuxR-operator part, the <SPAN class="gene">gfp</SPAN> 
			gene activity of the reporter part is dually regulated by IPTG and 3OC6-HSL. 
			We used promoterless pSB3K3-<SPAN class="gene">gfp</SPAN> (BBa_J54103) as a negative control, 
			and pAC-Pλ-<SPAN class="gene">gfp</SPAN> (chloramphenicol-resistance), which constitutively expressed GFP, 
			as a positive control. To know about the mechanism of this promoter click 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#4.0.">here</A>.
		</P><H3 id="2.2" style="clear:both;">2.2 Creating Parts that responded correctly to our set of Signaling Molecules</H3><P>
			In the process of constructing enough AND-gates that could suffice the needs 
			of our RPS game design, we discovered two faulty BioBricks: lsrA promoter (BBa_K117002) and las promoter (BBa_J64010). 
			Because of these faulty parts, the Judge <SPAN class="name">E. coli</SPAN> 
			set we had designed could only sense the Player <SPAN class="name">E. coli</SPAN>'s 
			signaling molecule 3OC6-HSL (Rock). This ultimately led to an unfair game 
			because humans could win every time they played with the Paper signaling molecule 
			(for more on the “Sad story of the Rock Player” click 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/the_sad_stroy_of_the_rock_player">here</A>).
		</P><P>
			To fix this problem, we improved the old defective las and lsrA promoters parts 
			by making new parts that work! As can be seen in the experimental information 
			below (see “Improving lsrA promoter” and “Improving las promoter”), 
			we confirmed our lasI promoter (<A href="http://partsregistry.org/Part:BBa_K649000">BBa_K649000</A>) 
			and lsrA promoter (<A href="http://partsregistry.org/Part:BBa_K649100">BBa_K649100</A>) work perfectly!
		</P><P>
			Since the lsrA promoter plays a key role in the correct functioning of AI-2, 
			fixing these parts now allows us to use AI-2 as a signaling molecule, which is a 
			promising advance because of the characteristics of 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#5.0.">the AI-2 mechanism</A>.
			This mechanism prevents AI-2 from cross-talking with other signaling molecules 
			such as AHL. Hence, this signaling molecule is a very powerful 
			tool to build complex Synthetic Biology systems.
		</P><P>
			Finally, one thing we would like outline is that although the promoters we made 
			are single input promoters, confirmation of their activity is required as a 
			reference to construct AND gate promoters. Therefore, we solved important issues 
			and made significant advances towards constructing AND-gate promoters. This allows 
			<SPAN class="name">E. coli</SPAN> to also choose the signaling molecules 
			corresponding to Paper and Scissors, so we have again a working RPS game design.
		</P><H3 id="2.3">2.3 Improving PlsrA</H3><DIV style="float: left;"><SPAN class="graph_title">Fig. 2.2 Not working lsrA promoter(<A href="http://partsregistry.org/Part:BBa_K117002">BBa_K117002</A>)  activity and our new lsrA promoter(<A href="http://partsregistry.org/Part:BBa_K649104">BBa_K649104</A>)</SPAN> activity
		</DIV><P>
			We confirmed that the lsrA promoter (<A href="http://partsregistry.org/Part:BBa_K117002">BBa_K117002</A>) does not work properly 
			(samples used our experiment are listed in Table 2.1 below). The fluorescence intensity of GFP of lsrA promoter-<SPAN class="gene">gfp</SPAN>((<A href="http://partsregistry.org/Part:BBa_K117002">BBa_K117002</A>)-<SPAN class="gene">gfp</SPAN>) was lower 
			even than those of the negative control (Fig. 2.2), which clearly shows that 
			lsrA promoter((<A href="http://partsregistry.org/Part:BBa_K117002">BBa_K117002</A>) does not work as expected. In this experiment, 
			we measured transcriptional activity of lsrA promoter by introducing a 
			<SPAN class="gene">gfp</SPAN> gene downstream of this promoter (Fig. 2.3).
		</P><TABLE style="clear:both;margin-top: 20px;" border="1" align="center"><CAPTION>Table 2.1 Samples used our experiment</CAPTION><TBODY><TR><TH>Name</TH><TH>Strain</TH><TH>Plasmid</TH></TR><TR><TD>sample1</TD><TD rowspan="4">JD22597</TD><TD>Ptet-<SPAN class="gene">gfp</SPAN> on pSB1A2</TD></TR><TR><TD>sample2</TD><TD>Promoterless-<SPAN class="gene">gfp</SPAN> on pSB6A1</TD></TR><TR><TD>sample3</TD><TD>PlsrA-<SPAN class="gene">gfp</SPAN> on pSB1A2 (<A href="http://partsregistry.org/Part:BBa_K649104">BBa_K649104</A>)</TD></TR><TR><TD>sample4</TD><TD>PlsrA-<SPAN class="gene">gfp</SPAN> on pSB1A2 ((<A href="http://partsregistry.org/Part:BBa_K117002">BBa_K117002</A>)-<SPAN class="gene">gfp</SPAN>)</TD></TR></TBODY></TABLE><DIV align="center"><SPAN class="graph_title">Fig. 2.3 lsrA promoter-<SPAN class="gene">gfp</SPAN>((<A href="http://partsregistry.org/Part:BBa_K117002">BBa_K117002</A>)-<SPAN class="gene">gfp</SPAN>)</SPAN></DIV><P>
			To solve this problem, we created the first working iGEM lsrA promoter (<A href="http://partsregistry.org/Part:BBa_K649100">BBa_K649100</A>). 
			Its fluorescence intensity was much higher than that from a promoter-less 
			<SPAN class="gene">gfp</SPAN> negative control plasmid, showing that our new 
			lsrA promoter works(Fig. 2.5). In this experiment, we measured the transcriptional 
			activity of our lsrA promoter by introducing a <SPAN class="gene">gfp</SPAN> gene downstream of 
			the promoter(<A href="http://partsregistry.org/Part:BBa_K649104">BBa_K649104</A>, Fig. 2.4). Details about this experiment can be found 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#5.">here</A>.
		</P><DIV align="center"><SPAN class="graph_title">Fig. 2.4 lsrA promoter-<SPAN class="gene">gfp</SPAN>(<A href="http://partsregistry.org/Part:BBa_K649104">BBa_K649104</A>)</SPAN></DIV><DIV align="center"><SPAN class="graph_title">Fig. 2.5 LsrR represses lsrA promoter.</SPAN></DIV><P>
			Moreover, this promoter can be repressed by our new LsrR part(<A href="http://partsregistry.org/Part:BBa_K649105">BBa_K649105</A>). 
			(samples used our experiments are listed in Table 2.2 below) The fluorescence 
			intensity of GFP of sample 3 was three times as large as 
			that of sample 4. This result shows that LsrR successfully repressed 
			lsrA promoter. In this experiment, we measured LsrR repression activity by 
			introducing a <SPAN class="gene">gfp</SPAN> gene downstream of lsrA promoter ((<A href="http://partsregistry.org/Part:BBa_K649105">BBa_K649105</A>, Fig. 2.6).
			Details about this experiment can be found 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#6.">here</A>.
		</P><TABLE align="center" border="1"><CAPTION>Table2.2 Samples used our experiment</CAPTION><TBODY><TR><TH>Name</TH><TH>Strain</TH><TH>Plasmid</TH></TR><TR><TD>sample1</TD><TD rowspan="2">JM2.300</TD><TD>Ptet-<SPAN class="gene">gfp</SPAN> on pSB6A1</TD></TR><TR><TD>sample2</TD><TD>Promoterless-<SPAN class="gene">gfp</SPAN> on pSB3K3</TD></TR><TR><TD>sample3</TD><TD rowspan="2">MG1655</TD><TD>PlsrA-<SPAN class="gene">gfp</SPAN> on pSB3K3</TD></TR><TR><TD>sample4</TD><TD>PlsrA-<SPAN class="gene">gfp</SPAN>-PlsrR-<SPAN class="gene">lsrR</SPAN> on pSB3K3</TD></TR></TBODY></TABLE><DIV align="center"><SPAN class="graph_title">Fig. 2.6 lsrA promoter-<SPAN class="gene">gfp</SPAN>-lsrR promoter-<SPAN class="gene">lsrR</SPAN>(<A href="http://partsregistry.org/Part:BBa_K649105">BBa_K649105</A>)</SPAN></DIV><H3 id="2.4">2.4 Improving las promoter</H3><TABLE align="center"><TBODY><TR><TD><SPAN class="graph_title">Fig. 2.7 (a)</SPAN></TD><TD><SPAN class="graph_title">(b)</SPAN></TD></TR></TBODY></TABLE><P>
			Fig. 2.7 (a): Not working lasI promoter (BBa_J64010). 
			Fig. 2.7 (b): New working lasI promoter (BBa_K649000) we made. 
			We confirmed it works as expected. In our assay, we used the same LasR 
			regulator part used in the assay of BBa_ J64010. Clearly, for our 
			part the fluorescence intensity of 3OC12-HSL+ was higher than that of 3OC12-HSL-.
			To know detailed methods about these lasI promoter assay, please see here, 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#2.">previous part</A> 
			and <A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#3.">new part</A>. 
		</P><P>
			To prove that the LasR regulator used in our PlasI assay works, we did another assay. 
			Details about this assay can be found 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#1.">here.</A></P><H2 id="3">3. The Randomizers</H2><P>
			Although our set of six signaling molecules allows us to play RPS with 
			<SPAN class="name">E. coli</SPAN>, we must make sure <SPAN class="name">E. coli</SPAN> 
			can choose any of its three signaling molecules with the same probability 
			in order to be able to play RPS fairly and properly. To do so, we designed 
			three kinds of randomizers: one kind which needs of three types of bacteria 
			(each of which produces one of the three RPS signaling molecules), 
			and the other kind that needs of only one type of bacteria which can 
			synthetize each of the three signaling molecules one at a time and randomly. 
			Namely, the randomizers are Single Colony Isolation, 
			Survival of one Strain and Conditional Knockout by Recombination.
		</P><H3 id="3.1">3.1 Single Colony Isolation</H3><P>
			This is our simplest randomizer design. To make sure <SPAN class="name">E. coli</SPAN> 
			chooses any of its signaling molecules with equal probability, we put the constructs 
			for each molecule inside a different bacterium, so we create three types of bacteria: 
			one synthetizing the corresponding signaling molecule for rock, other synthetizing 
			the corresponding signaling molecule for paper, and lastly one synthetizing the 
			corresponding signaling molecule for scissors. By randomly isolating a single colony 
			out of the many colonies that result from the mixing between the three types of 
			<SPAN class="name">E. coli</SPAN>, we get a random output as 
			<SPAN class="name">E. coli</SPAN>'s choice for the RPS game.
		</P><H3 id="3.2" style="clear:both">3.2 Conditional Knockout by Recombination</H3><P>
			Our second randomizer differs from our other two randomizers in that all the 
			three signaling molecules are produced one at a time and randomly by only one type 
			of bacteria. We were inspired by a paper about “brainbow” research 
			on mice to create this randomizer (Livet J <I>et al.</I>, 2007), which is based 
			on the recombination mechanism of the enzyme Cre and the lox sequences. 
			We designed a Cre-Lox system which allows <SPAN class="name">E. coli</SPAN> 
			to express one of its three signaling molecules by means of conditional knockout. 
			The design is depicted in Fig. 1. It should be noted this randomizer is designed 
			to be used at a single-cell level. When this randomizer is used in groups of cells, 
			the different signals released by the cells will mix. In this case, by using microfluidic 
			devices or isolating single colony of bacteria, we can obtain only one of the signaling 
			molecules produced by the initial group of cells.
		</P><DIV class="graph_title">
			Fig. 3.1 - (a) Cre recombinase construction. (b) lox cassettes distribution for the randomizer design
		</DIV><H4 id="3.2.1">3.2.1 The Requirements</H4><P>
			Basically, each pair of <I>lox</I> sites (indicated by the same color) mark the points 
			which the enzyme Cre will excise (they will be cut off the backbone along with the 
			sequence between them). For a design that allows choosing randomly one of 
			<SPAN class="name">E. coli</SPAN>’s three signaling molecules, at least two 
			cassettes of lox sites are needed. When these two cassettes of lox sites and 
			protein coding sequences are arranged as in Fig. 3.1(b), 
			only one signaling molecule is produced.
		</P><P>
			Our design also required a way to control <SPAN class="gene">luxI</SPAN> gene's expression. 
			To do so, we used an inducible promoter instead of a constitutive promoter. 
			A constitutive promoter would have caused <SPAN class="gene">luxI</SPAN> gene to be expressed beforehand and 
			could lead to an <SPAN class="name">E. coli</SPAN> producing two signaling molecules 
			at the same time (the equivalent of showing two hands in the RPS game). In contrast, 
			the inducible promoter prevents a particular gene from expressing preferentially.
		</P><P>
			One last but not less important requirement for our randomizer is that it should 
			express each of the three signals not only one at a time, but also with the same 
			probability. This makes the game fair in the sense that 
			<SPAN class="name">E. coli</SPAN>’s choice in the RPS game is not predictable.
		</P><H4 id="3.2.2">3.2.2 The Mechanism</H4><P>
			When the blue cassette of <I>Lox</I> sites is excised (Fig. 3.1), the signaling molecule 
			coded by lasI (3OC6-HSL) will be produced. Likewise, when the black <I>Lox</I> cassette 
			is excised, the signaling molecule coded by luxS (AI-2) will be produced. On the other hand, 
			a third possible outcome is that recombination does not take place. In this case, 
			the signaling molecule coded by <SPAN class="gene">luxI</SPAN> (3OC6-HSL) will be produced. Also note that excision 
			of one kind of lox cassette removes the remaining cassette, 
			thereby preventing further recombination.
		</P><P>
			As mentioned before, one of the requirements for our randomizer was to have at least 
			two lox cassettes. This prevents excision of <I>lox</I> sites from different cassettes 
			(for example one blue <I>lox</I> site and one black <I>lox</I> site). Because the <I>lox71-lox66</I> cassette and the l<I>ox2272-lox2272</I> cassette are incompatible (Zorana Carter and Daniela Delneri <I>et al.</I>, Yeast 2010), we can use them to build our randomizer.
		</P><H4 id="3.2.3">3.2.3 Testing the Lox Cassettes</H4><P>
			As stated above, we need two lox cassettes of different recombination frequency 
			for randomizer. Because there was no working <I>lox</I> parts in registry, we constructed 
			three original BioBricks for testing whether <I>lox2272</I> and <I>lox71/66</I> cassettes work. For 
			the convenience of testing, fluorescence expressing genes were used in place of signal 
			molecular expressing genes in construction. After figuring out their working <I>in vitro</I>, we tested them <I>in vivo</I> by detecting red and green fluorescence through fluoro imager and flow cytometer. Furthermore, we compared the relative recombination frequency of two cassettes . Our <I>lox</I> Cassettes constructions 
			were working properly, and their recombination frequency were different from each other.
		</P><UL><LI>
				PlacIQ-<I>lox2272</I>-<SPAN class="gene">gfp</SPAN>-<I>lox2272</I><A href="http://partsregistry.org/Part:BBa_K649200">(BBa_K649200)</A></LI><LI>
				PlacIQ-<I>lox2272</I>-<SPAN class="gene">rfp</SPAN>-<I>lox2272</I>-<SPAN class="gene">gfp</SPAN><A href="http://partsregistry.org/Part:BBa_K649201">(BBa_K649201)</A></LI><LI>
				PlacIQ-<I>lox71</I>-<SPAN class="gene">rfp</SPAN>-<I>lox66</I>-<SPAN class="gene">gfp</SPAN><A href="http://partsregistry.org/Part:BBa_K649202">(BBa_K649202)</A></LI></UL><P>
			The <I>in vitro</I> assay with K649200 was made in advance. The preliminary experiment allowed 
			us to confirm that the Cre-mediated recombination on <I>lox2272</I> cassette works as designed. 
			In the assay, Cre recombinase was added to the linear DNA and incubated for 0.5, 2, and 4 hours.  
			Images of the experiments have been added below.
		</P><P>
			When checking the result by electrophoresis, there were several bands in samples to 
			which Cre was added(1st, 2nd lane from right which corresponds to 4 hr and 2 hr respectively). 
			It indicates that excision of the <I>lox</I> sites successfully occurred. 
			To know detailed about this assay, please see here, 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#7."><I>in vitro</I> assay for <I>lox2272</I></A></P><P>
			For the <I>in vivo</I> assay, by detecting fluorescenc levels of GFP and mCherry, 
			we could determine whether recombination occured in K649201 and K649202 and compare 
			relative recombination frequency between two of them. We prepared a competent cell 
			JM2.300 into which P<SUB>BAD</SUB>/araC-Cre(pSB1A2, BBa_I718008) had been constructed. 
			Subsequently, our BioBrick was constructed into the cell.
		</P><TABLE border="1" align="center"><TBODY><TR><TH colspan="2">sample</TH><TH>arabinose</TH></TR><TR><TD>1</TD><TD>PlacIQ-<I>lox</I>-<SPAN class="gene">rfp</SPAN>-<I>lox</I>-<SPAN class="gene">gfp</SPAN>(pSB3K3)P<SUB>BAD</SUB>/araC-Cre(pSB1A2)</TD><TD style="text-align:center;">+</TD></TR><TR><TD>2</TD><TD>PlacIQ-<I>lox</I>-<SPAN class="gene">rfp</SPAN>-<I>lox</I>-<SPAN class="gene">gfp</SPAN>(pSB3K3)P<SUB>BAD</SUB>/araC-Cre(pSB1A2)</TD><TD style="text-align:center;">-</TD></TR><TR><TD>3</TD><TD>PlacIQ-<I>lox</I>-<SPAN class="gene">rfp</SPAN>-<I>lox</I>-<SPAN class="gene">gfp</SPAN>(pSB3K3) : negative control </TD><TD style="text-align:center;">+</TD></TR></TBODY></TABLE><P>
			The strain was grown in a 3 mL liquid culture, and 75 µL of 2 M arabinose was added 
			to induce Cre expression. We used two controls for the experiment. One was the same strain 
			without arabinose induction, and the other was JM2.300 strain which was induced by arabinose 
			and had only our BioBrick. All the strains were cultured each for periods of 0.5, 1, 2, and 4 hours, 
			and in each case the florescence levels were measured by flow cytometer and FLA. 
			To know the detailed method about this assay, please see here 
			<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#8."><I>in vivo</I> assay for <I>lox</I> cassettes</A>.
		</P><P>
			We confirmed our results optically by taking florescence images. 
			K649201 transformants with with 0.5 hr-induction of Cre in liquid medium and its two control strains were                    plated and incubated in 37°C for 12 hours. Images of the three conditions were taken using red 
			florescence filter, green florescence filter and no filter as shown below, respectively.
		</P><TABLE align="center"><TBODY><TR><TD style="text-align:center;">(a)</TD></TR><TR><TD style="text-align:center;">(b)</TD></TR><TR><TD style="text-align:center;">(c)</TD></TR></TBODY></TABLE><P>
			Fig. 3.2 Cre-meditated recombination at <I>lox2272</I> cassette. Cre-induction period of 0.5 hr
                        (a)Overlay of Green and Red channel. The leftmost is a negative control which don't have Cre-expressing            
                        plasmid. The center is an arabinose induced sample which has both Cre plasmid and BioBrick K649201. 
			The rightmost is a uninduced strain which has both plasmid like as the center. 
			(b)Detection of GFP. The order of samples is same as above. 
			(c)Detection of mCherry. The order of samples is same as above.
		</P><P>
			On the sample with the P<SUB>BAD</SUB>/araC-Cre construction, we found that recombination occurred 
			when arabinose was added. In contrast to this result, when we measured the levels of the sample 
			without the P<SUB>BAD</SUB>/araC-Cre construction, we found that the GFP levels were far lower than 
			those of the sample with the P<SUB>BAD</SUB>/araC-Cre construction. This clearly proves that our 
			<I>lox</I> constructions, both in K649201 and K649202, respond correctly to the effects of Cre recombinase. 
			A slight detection of green florescence in plate absence of P<SUB>BAD</SUB>/araC-Cre can be explained
                        that there happened cross-talk to green channel by FMN(Flavin mononucleotide) or expression of GFP according
                        to malfunction of terminator before <SPAN class="gene">gfp</SPAN>. We could also observe recombination occurred when arabinose was not 
                        added, which can be explained due to a leaking in the P<SUB>BAD</SUB>/araC promoter.You can find that K649202 also works well from the images of K649202 on<A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay#8."> here</A>.
		</P><P>
			Furthermore, we could observe that the arabinose(+) sample of K649202 has higher green/red ratio than that of K649201, which implying the frequency of <I>lox71/66</I> casette is higher than that of <I>lox2272</I>.
		</P><DIV align="center"><SPAN class="graph_title">Fig. 3.3 Image of six samples of K649201 (up) and K649202 (down) at period of 0.5 hr</SPAN></DIV><TABLE border="0"><TBODY><TR><TD style="text-align:center;">(a)</TD><TD style="text-align:center;">(b)</TD></TR><TR><TD>
					Fig. 3.4 identical plates with Fig. 3.3
					(a)expression levels of red and green florescence of K649201
					(b)expression levels of red and green florescence of K649202
					(c)examined area for comparing between red and green florescence at each plate</TD></TR><TR><TD style="text-align:center;">(c)</TD></TR></TBODY></TABLE><P>
			As we examining green florescence in comparison to red florescence, green expression level was lower than red in 
			K649201(Fig. 3.4(a)), which meaning that considerable plasmids in those cells yet. In contrast, green	expression level exceeded red in K649202(Fig. 3.4(b)). This result implies that recombination frequency of 
			<I>lox71/66</I> cassette is relatively high than that of <I>lox2272</I> cassette.</P><P><SPAN class="graph_title">Fig. 3.5 Green fluorescence level of each cell was detected by flow cytometer.
                        (a)arabinose induced strain containing only K649201 and cre-expressing plasmid (b)arabinose supplied strain containing only K649201 (c)arabinose induced starin containing K649202 and cre-expressing plasmid (d)arabinose supplied strain containing only K649202</SPAN></P><P>
                The higher recombination efficiency of <I>lox71/66</I> compare to <I>lox2272</I> was confirmed also by flow cytometer intensity of <I>lox71/66</I> was higher than that of lox2272, which supports the result detected by FLA. 
                </P><H4 id="3.2.4">3.2.4 Playing Fair: Future Work</H4><P>
			To make each of the outcomes (R, P, and S) equally probable, we are going to quantify the recombination frequency of each lox cassette. This information and adequate Cre induction will be likely to allow us to have an RPS player 
			<SPAN class="name">E. coli</SPAN> whose choice of either of rock, paper or scissors cannot be predicted.
		</P><P>
			In our next experiments, we are going to vary the reaction time and the distance between the lox sites 
			of each cassette. We believe precise modification of this two parameters must lead to our goal of 
			making a randomizer in which each of the signaling molecules can be expressed with the same frequency 
			(which results in each of the outcomes being expressed with the same probability).
		</P><H3 id="3.3">3.3 Survival of One strain</H3><H4 id="3.3.1">3.3.1 Introduction: Very small differences determine who will survive</H4><P>
			In this section we will show a shocking scenario of evolution: 
			the future of each of three different rival strains (whether the strain will die or survive) 
			is marked by very small differences between the initial population densities of the strains (a phenomenon also known as the “butterfly effect”). 
			Furthermore, we will also show that we can apply this very interesting result to create a randomizer 
			that can be used in our Rock-Paper-Scissors game, due to the fact that only one of the rival strains 
			will survive. More specifically, we assign to each of the three rival strains either of Rock, Paper or Scissors, 
			make them compete for survival and take the surviving strain to represent the bacteria's choice for the RPS game.
		</P><H4 id="3.3.2">3.3.2 Adjusting the Model to create a True Randomizer</H4><P>
			The idea for creating this randomizer was born from a paper written in 
			1996 by Durrett and Levin. In it, the authors described a system of three types of bacteria that competed for 
			survival in dynamic that resembled a Rock-Paper-Scissors (RPS) game. 
			However, the model proposed in this paper is not fully appropriate for our RPS randomizer, 
			since one of the three types of bacteria cannot ultimately survive 
			(although it can dominate the system, i.e. have the highest population density, for definite periods of time). 
			We will discuss more on the limitations we found in this model to be adopted as a 
			randomizer and the modifications we made to create a true randomizer.
		</P><H4 id="3.3.3">3.3.3 How the Three Types of Bacteria Compete for Survival</H4><P>
			The three types of bacteria that compete for survival use three tactics to outcompete their rivals: 
			the production of a toxin (a bacteriocin called colicin) that is toxic to other strains, 
			resistance to the toxin produced by other strains, and a higher birth rate than their rival strains. 
			Namely, the three types of bacteria are: colicin-producing <SPAN class="name">E. coli</SPAN> (R), 
			colicin-resistant <SPAN class="name">E. coli</SPAN> (P) 
			and colicin-sensitive <SPAN class="name">E. coli</SPAN> (S). 
			The colicin-producer outcompetes the colicin-sensitive by producing the colicin. The colicin-sensitive 
			bacteria outcompetes the colicin-resistant because its birth rate is higher than that of the colicin-resistant. 
			The colicin-resistant outcompetes the colicin producer because its birth rate is higher 
			than that of the colicin producer. The colicin resistant bacteria are also able to produce colicin, 
			but at a lower energetic cost, which allows them to have a higher birth rate.
		</P><P>
			The system was described by the following general differential equations
		</P><P>
			Where
		</P><H4 id="3.3.4">3.3.4 The Old Model</H4><P>
			In the model described by Durrett and Levin’s paper the equations were as follows:
		</P><DIV align="center"><P>Producer</P><P>Resistant</P><P>Sensitive</P></DIV><P>
			These equations show that the colicin-resistant bacteria are completely immune to colicin 
			(there is not death factor associated to colicin in the equation for du<SUB>2</SUB>/dt). 
			However, as will be explained afterwards, this results in a loss of 
			balance that does not allow building a true randomizing system.
		</P><P>
			Now, setting the parameters as follows, the graph below was created by Durrett and Levin.
		</P><H4 id="3.3.5">3.3.5 Our New Model</H4><P>
			As mentioned before, the model proposed by Durrett and Levin has critical limitations as a randomizer for the RPS game. 
			To be able to create a true randomizer, we modified the differential equations of the model taking care to give it 
			a biological meaning. With our new differential equations, any of the three types of bacteria can ultimately survive 
			by outcompeting the other two strains, which will die. More specifically, we limited the resistance of the 
			colicin-resistant bacteria in the sense that it would produce a type of bacteriocin that is only toxic to itself and 
			to the sensitive strain, and additionally the resistant strain would also be vulnerable to the colicin produced by 
			the colicin-producer.  Since which strain will be the one that survives is determined by very small differences in 
			the initial concentrations of the three different populations of bacteria, in practice this systems becomes a 
			randomizer because of the imprecisions in the measurements that result, for example, when using micropipettes. 
			This randomizer describes a new competition dynamic that could not be reproduced in the previous model proposed 
			by Durrett and Levin due to the instability along the
			axis.
		</P><P>
			If we set the parameters as follows
		</P><P>
			and we graph this equations using a Matlab program, 
			we get a graph which clearly shows there are stable points on each of the three axes 
			(Fig. 1, Up).
		</P><DIV align="center">
			Fig. 1 Up: Our New model. Down: The Old Model
		</DIV><P>
			These stable points (u<SUB>1</SUB>,0,0), (0,u<SUB>2</SUB>,0) and (0,0,u<SUB>3</SUB>) indicate that for the equations 
			we have set all of the three strains may ultimately survive for infinite peiriods of time. 
			The differences between our model and the model of Durrett and Levin can be seen graphically 
			in Fig. 1. These graphs were plotted using Matlab. 
		</P><P>
			Note that the parameters we have set for our equations satisfy the initail conditions 
			of the model proposed by Durrett and Levin (indicated in black font) 
			</P><H4 id="3.3.6">3.3.6 The Biological Meaning of our Model</H4><P>
			From a biological perspective, our model describes the existence of two strains of 
			bacteria that produce two different types of bacteriocins. One of these strains is not 
			completely resistant to its own bacteriocin nor to the bacteriocin produced by its 
			rival strain. This can be justified as the consequence of insufficient/ineffective 
			resistance protein production by the “resistant” strain. This limitation in the 
			production of resistance protein could be thought of as a consequence of the 
			“resistant” strain being a mutant of a colicin-sensitive strain.
		</P><H4 id="3.3.7">3.3.7 Making it Obvious</H4><P>
			From the graph of our new model (Fig. 1, left) it can be deduced that there are paths 
			that converge at stable points (u<SUB>1</SUB>,0,0), (0,u<SUB>2</SUB>,0) and (0,0,u<SUB>3</SUB>), and that this paths all 
			have an approximately common origin. In this section we would like to show that the 
			origin of these paths is practically the same, and that in that sense we have designed 
			a true randomizer (since, as mentioned before, the imprecisions that result in the 
			experimental measurements will make it impossible to make the initial 
			population density of the three strains the same). 
		</P><P>
			In the following set of graphs we will make it obvious that each of the three 
			different strains of E. coli to survive in a random fashion by very small
			differences on the initial concentrations of each strain.
		</P><P>
			We modeled our results using Matlab.  
			As can be seen in the graphs below, each of the strains can survive if their 
			initial density in only tree hundredths (a.u.) greater than the other two strains' 
			initial concentrations.
		</P><TABLE><TBODY><TR><TD rowspan="2">
					With these graphs it becomes clear that the imprecisions in experimental measurements 
					(i.e. pipetting) are enough to cause the outcome of Rock, Paper or Scissors signaling 
					molecule to be random. Consequently, we can conclude that this randomizer is not 
					only feasible but also practical and effective (let alone interesting).
				</TD></TR></TBODY></TABLE><H2>References</H2><P><UL><LI>Patrick C. Seed <I>et al</I>. “Activation of the Pseudomonas aeruginosa lasI Gene by LasR and the Pseudomonas Autoinducer PAI: an Autoinduction Regulatory Hierarchy” JOURNAL OF BACTERIOLOGY (1995)</LI><LI>Shotaro Ayukawa <I>et al</I>. “Construction of a genetic AND gate under a new standard for assembly of genetic parts” BMC Genomics (2010)</LI><LI>Robert Sidney Cox, III <I>et al</I>. “Programming gene expression with combinatorial promoters” molecular system biology (2007)</LI><LI>Rolf Lutz <I>et al</I>. “Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements” Nucleic Acids Research (1997)</LI><LI>Karina B.Xavier and Bonnie L.Bassler 2004. Regulation of Uptake and Processing of the Quorum-Sensing Autoinducer AI2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Jan. 2005, p. 238-248</LI><LI>Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou, Baolon Sun 2009. LsrR-binding site recognition and regulatory characteristics in Escherichia coli AI-2 quorum sensing. Cell Research (2009), p.1258-1268</LI><LI>Liang Wang, Yoshifumi Hashimoto, Chen-Yu Tsao, James J.Valdes, and William E.Bentley 2004. Cyclic AMP(cAMP) and cAMP Receptor Protein Influence both Synthesis and Uptake of Extracellular Autoinducer 2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Mar. 2005, p. 2066-2076</LI><LI>Jean Livet, Tamily A. Weissman, Hyuno Kang, Ryan W. Draft, Ju Lu, Robyn A. Bennis, Joshua R. Sanes and Jeff W. Lichtman (2007). Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system. Nature.</LI><LI>Kimi Araki, Masatake Araki1 and Ken-ichi Yamamura (2002). Site-directed integration of the cre gene mediated by Cre recombinase using a combination of mutant <I>lox</I> sites, Nucleic Acids Research</LI><LI>Durrett, R., Levin, S. Allelopathy in Spatially Distributed Populations (1997). Journal of Theoretical Biology, 185, 165-171. </LI></UL></P></DIV><DIV style="bottom: 0;left: 30; position:fixed; background-color: #000000; opacity:0.8;"><A onclick="scrollTo(0,0);return false;" onfocus="this.blur();" style="color:#FFFFFF;">Return to Page Top</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/index.htm">http://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/index.htm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tokyo_Tech/Projects/RPS-game/index.htm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tokyo_Tech/Projects/RPS-game/index.htm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/index.htm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/index.htm&amp;oldid=261495" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>