<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tokyo_Tech_Projects_RPS-game_assay"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tokyo_Tech/Projects/RPS-game/assay">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tokyo_Tech/Projects/RPS-game/assay&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/assay&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/assay&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tokyo_Tech/Projects/RPS-game/assay" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tokyo Tech/Projects/RPS-game/assay</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Tokyo Tech 2011</TITLE></P><DIV class="top"><DIV id="navigation"><UL id="TopMenu"><LI id="menu_Home"><A href="https://2011.igem.org/Team:Tokyo_Tech">Home</A></LI><LI id="menu_Project">
				Project
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/index.htm">RPS-Game</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/making-rain/index.htm">Make it Rain</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Projects/Urea-cooler/index.htm">Urea Coolers</A></LI></UL><LI id="menu_data_page"><A href="https://2011.igem.org/Team:Tokyo_Tech/DataPage.htm">Data page</A></LI><LI id="menu_Modeling">
				Modeling
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/RPS-game/RPS-game">RPS-Game</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Modeling/Urea-cooler/urea-cooler">Urea Coolers</A></LI><LI id="menu_Human Practice"><A href="https://2011.igem.org/Team:Tokyo_Tech/HumanPractice.htm">Human Practice</A></LI><LI id="menu_Sitemap"><A href="https://2011.igem.org/Team:Tokyo_Tech/sitemap.htm">Sitemap</A></LI><LI id="menu_Extra">
				More
				</LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Attribution_and_Contributions.htm">Attribution and Contributions</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/notebook">NoteBook</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Sponsers.htm">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Tokyo_Tech/Collaboration.htm">Collaboration</A></LI></DIV></DIV><DIV style="min-height: 17000px; float: left;"><DIV id="LeftMenu"><UL><LI><A href="#1.">1. LasR repression</A></LI><UL><LI><A href="#1.1.">1.1 Plasmid transformed into <I>E.coli</I></A></LI><LI><A href="#1.2.">1.2 Method</A></LI><LI><A href="#1.3.">1.3 Results</A></LI><LI><A href="#1.4.">1.4 Discussion</A></LI></UL><LI><A href="#2.">2. Previous lasI promoter activity</A></LI><UL><LI><A href="#2.1.">2.1 Plasmid transformed into <I>E.coli</I></A></LI><LI><A href="#2.2.">2.2 Method</A></LI><LI><A href="#2.3.">2.3 Results</A></LI><LI><A href="#2.4.">2.4 Discussion</A></LI></UL><LI><A href="#3.">3. New lasI promoter activity</A></LI><UL><LI><A href="#3.1.">3.1 Plasmid transformed into <I>E.coli</I></A></LI><LI><A href="#3.2.">3.2 Method</A></LI><LI><A href="#3.3.">3.3 Results</A></LI><LI><A href="#3.4.">3.4 References</A></LI></UL><LI><A href="#4.">4. lux-lac hybrid promoter</A></LI><UL><LI><A href="#4.0.">4.0 The AND-gate Mechanism</A></LI><LI><A href="#4.1.">4.1 Plasmid transformed into <I>E.coli</I></A></LI><LI><A href="#4.2.">4.2 Method</A></LI><LI><A href="#4.3.">4.3 Results</A></LI><LI><A href="#4.4.">4.4 Discussion</A></LI><LI><A href="#4.5.">4.5 References</A></LI></UL><LI><A href="#5.">5. Previous lsrA promoter activity and our new lsrA promoter activity</A></LI><UL><LI><A href="#5.0.">5.0 AI-2 working system</A></LI><LI><A href="#5.1.">5.1 Sample</A></LI><LI><A href="#5.2.">5.2 Method</A></LI><LI><A href="#5.3.">5.3 Results</A></LI><LI><A href="#5.4.">5.4 Discussion</A></LI><LI><A href="#5.5.">5.5 References</A></LI></UL><LI><A href="#6.">6. LsrR repression</A></LI><UL><LI><A href="#6.1.">6.1 Sample</A></LI><LI><A href="#6.2.">6.2 Method</A></LI><LI><A href="#6.3.">6.3 Construction of PlsrA-<I>gfp</I>-PlsrR-<I>lsrR</I></A></LI><LI><A href="#6.4.">6.4 Results</A></LI><LI><A href="#6.5.">6.5 References</A></LI></UL><LI><A href="#7.">7. <SPAN class="name">in vitro</SPAN> assay for <I>lox2272</I></A></LI><UL><LI><A href="#7.1.">7.1 Construction of <I>lox2272</I>-<I>gfp</I>-<I>lox2272</I></A></LI><LI><A href="#7.2.">7.2 Sample</A></LI><LI><A href="#7.3.">7.3 Method</A></LI><LI><A href="#7.4.">7.4 Results</A></LI><LI><A href="#7.5.">7.5 References</A></LI></UL><LI><A href="#8.">8. <I>in vivo</I> assay for <I>lox</I> cassettes</A></LI><UL><LI><A href="#8.1.">8.1 Sample</A></LI><LI><A href="#8.2.">8.2 Method</A></LI><LI><A href="#8.3.">8.3 Results</A></LI><LI><A href="#8.4.">8.4 Discussions</A></LI><LI><A href="#8.5.">8.5 References</A></LI></UL></UL></DIV></DIV><DIV class="main"><H1> RPS detailed method and results </H1><H2 id="1.">1. LasR repression</H2><H3 id="1.1.">1.1 Plasmid transformed into <I>E.coli</I></H3><P><UL><LI>Sample</LI><LI>
					Ptrc-rbs-<SPAN class="gene">lasR</SPAN>-TT-PlasI-rbs-<SPAN class="gene">cI</SPAN> on pBR</LI><LI>
					Pλ-rbs-<SPAN class="gene">gfp</SPAN> on pAC</LI><LI>negative control</LI><LI>
					Ptrc-rbs-<SPAN class="gene">lasR</SPAN>-TT on pBR</LI><LI>
					promoterless-rbs-<SPAN class="gene">gfp</SPAN>-TT on pSB3K3
				</LI></UL></P><H3 id="1.2.">1.2 Method</H3><P><OL><LI>Overnight culture of sample strain grown at 37°C in LB medium containing carbenicillin and chloramphenicol were diluted 1:20 in the medium, and overnight culture of promoterless negatgive control grown at 37°C in LB medium containing carbenicillin and kanamycin were diluted 1:300 in the medium and then they were incubated at 37°C as fresh cultures.</LI><LI>After their OD<SUB>600</SUB> reached 0.2, we added 3 µL of 500 µM 3O-C12-HSL (3OC12-HSL+) or 3 µL of DMSO (3OC12-HSL-) into the fresh cultures.</LI><LI>After 3-hour incubation at 37°C (OD approximately reached 1.50.), 0.25 mL of each culture was harvested by centrifugalization and suspended by adding 1 mL of PBS (phosphate-buffered saline).</LI><LI>We dispensed 500 µL of each suspension into a disposable tube through a cell strainer, and its fluorescence intensity was measured with a flow cytometer of Becton, Dickinson and Company.</LI></OL></P><H3 id="1.3.">1.3 Results</H3><H3 id="1.4.">1.4 Discussion</H3><P>
        	The above construction proves that the <I>lasR</I> part is working properly, which leads to our conclusion that the PlasI of the <A href="http://partsregistry.org/Part:BBa_J64010">BBa_J64010</A> part is a defective promoter. The regulator part has the constitutive promoter Ptrc, so the regulator LasR is constantly being produced. LasR needs of 3OC12-HSL to bind to Plas and activate it. Therefore, if the <I>lasR</I> part is working, it should activate the Plas promoter when 3OC12-HSL is added, and cI should be produced. The reporter part is has a λ promoter, which is a constitutive promoter. This means <SPAN class="gene">gfp</SPAN> is constantly being expressed and fluorescence is observed. Since cI represses Pλ, a working LasR part should lead to a dropdown of the levels of fluorescence. As can be seen below in the results from our experiments, the fluorescence levels dropped down when we added 3OC12-HSL, which leads to the conclusion that the LasR part we used works properly.
        </P><H2 id="2.">2. Previous lasI promoter activity</H2><H3 id="2.1.">2.1 Plasmid transformed into <I>E.coli</I></H3><P><UL><LI>Sample</LI><LI>
					PlasI<A href="http://partsregistry.org/Part:BBa_J64010">(BBa_J64010)</A>-rbs-<SPAN class="gene">gfp</SPAN>-TT on pSB3K3</LI><LI>
					Ptrc-rbs-<SPAN class="gene">lasR</SPAN>-TT on pBR</LI><LI>negative control</LI><OL><LI>
					promoterless-<SPAN class="gene">gfp</SPAN> on pSB3K3
				</LI><LI>
					Ptrc-rbs-<SPAN class="gene">lasR</SPAN>-TT-PlasI-rbs-<SPAN class="gene">cI</SPAN>-TT on pBR</LI></OL></UL></P><H3 id="2.2.">2.2 Method</H3><P><OL><LI>
			Overnight culture of sample strain grown at 37°C in LB medium containing carbenicillin and kanamycin, and overnight culture of promoterless negative control grown at 37°C in LB medium containing carbenicillin and kanamycin were diluted 1:300 in the medium, and then they were incubated at 37°C as fresh cultures.
		</LI><LI>
			After their OD<SUB>600</SUB> reached 0.2, we added 3 µL of 500 µM 3O-C12-HSL (3OC12-HSL+) or 3 µL of DMSO (3OC12-HSL-) into the fresh cultures.
		</LI><LI>
			After 3-hour incubation at 37°C (OD approximately reached 1.50.), 0.25 mL of each culture was harvested by centrifugalization and suspended by adding 1 mL of PBS (phosphate-buffered saline).
		</LI><LI>
			We dispensed 500 µL of each suspension into a disposable tube through a cell strainer, and its fluorescence intensity was measured with a flow cytometer of Becton, Dickinson and Company.
		</LI></OL></P><H3 id="2.3.">2.3 Results</H3><H3 id="2.4.">2.4 Discussion</H3><P>
        	Because the regulator part used in our lasI promoter assay has been constructed from the regulator part used in our LasR repression assay, the regulator part used in lasI promoter assay is working. Nevertheless, fluorescence intensity of PlasI<A href="http://partsregistry.org/Part:BBa_J64010">(BBa_J64010)</A>-rbs-<SPAN class="gene">gfp</SPAN>-TT was not changed before and after 3O-C12-HSL induction. From this result, we considered that las promoter <A href="http://partsregistry.org/Part:BBa_J64010">(BBa_J64010)</A> was not regulated by lasR and 3O-C12-HSL.
        </P><H2 id="3.">3. New lasI promoter activity</H2><H3 id="3.1.">3.1 Plasmid transformed into <I>E.coli</I></H3><P><UL><LI>Sample</LI><LI>
					Ptrc-rbs-<SPAN class="gene">lasR</SPAN>-TT on pBR</LI><LI>
					PlasI<A href="http://partsregistry.org/Part:BBa_K649000">(BBa_K649000)</A>-rbs-<SPAN class="gene">gfp</SPAN>-TT on pSB3K3</LI><LI>negative control
			</LI><LI>
					Ptrc-rbs-<SPAN class="gene">lasR</SPAN>-TT on pBR</LI><LI>promoterless-rbs-<SPAN class="gene">gfp</SPAN>-TT on pSB3K3</LI></UL></P><H3 id="3.2.">3.2 Method</H3><P><OL><LI>
			Overnight cultures of sample strain grown at 37°C in LB medium containing carbenicillin and kanamycin were diluted 1:100 in the medium, and overnight cultures of promoterless negative control strain grown at 37°C in LB medium containing carbenicillin and kanamycin were diluted 1:200 in the medium , and then they were incubated at 37°C as fresh cultures.
		</LI><LI>
			After their OD<SUB>600</SUB> reached 0.2, we added 3 µL of 500 µM 3O-C12-HSL (3OC12-HSL+) or 3 µL of DMSO (3OC12-HSL-) into the fresh cultures.
		</LI><LI>
			After 3-hour incubation at 37°C (OD reached approximately 1.80.), 0.25 mL of each culture was harvested by centrifugalization and suspended by adding 1 mL of PBS (phosphate-buffered saline).
		</LI><LI>
			We dispensed 500 µL of each suspension into a disposable tube through a cell strainer, and its fluorescence intensity was measured with a flow cytometer of Becton, Dickinson and Company.
		</LI></OL></P><H3 id="3.3.">3.3 Results</H3><H3 id="3.4.">3.4 References</H3><P><OL><LI>
			Patrick C. Seed <I>et al</I>. “Activation of the Pseudomonas aeruginosa <I>lasI</I> Gene by LasR and the Pseudomonas Autoinducer PAI: an Autoinduction Regulatory Hierarchy” JOURNAL OF BACTERIOLOGY (1995) 
		</LI></OL></P><H2 id="4.">4. lux-lac hybrid promoter activity</H2><H3 id="4.0.">4.0 The AND-gate Mechanism</H3><P>
		We designed AND-gate promoters that use two operators: one that uses activators as regulators, and other that uses repressors as regulators. After an activator binds to an inducer the resulting complex binds to the corresponding operator and activates it. However, since the AND-gate promoter needs its both operators to be active, transcription will not start until the operator that uses repressors as regulators has been de-repressed by the appropriate signaling molecule. This mechanism assures that transcription will not start until both players have added the corresponding signaling molecules to the place where the Judge bacterium is, so it fits perfectly in our RPS game design. To see the game results easily, we added a reporter gene downstream of the AND-gate promoter, so either of <SPAN class="gene">gfp, rfp</SPAN> or <SPAN class="gene">cfp</SPAN> is expressed when humans win, lose or tie the game, respectively.）
	</P><H3 id="4.1.">4.1 Plasmid transformed into <I>E.coli</I></H3><P><UL><LI>Sample</LI><LI>
					I751101 on pSB3K3</LI><LI>pTrc99A</LI><LI>negative control</LI><OL><LI>promoterless-rbs-<SPAN class="gene">gfp</SPAN>-TT on pSB3K3</LI><LI>pTrc99A</LI></OL></UL></P><H3 id="4.2.">4.2 Method</H3><P><OL><LI>
			Overnight culture of BBa_I751101 grown at 37°C in LB medium containing carbenicillin and kanamycin, and overnight culture of promoterless negative control grown at 37°C in LB medium containing carbenicillin and kanamycin were diluted 1:100 in the medium, and then they were incubated at 37°C as fresh cultures.
		</LI><LI>
			After their OD<SUB>600</SUB> reached 0.2, we added inducers into the fresh culture: 1 mM IPTG and/or 10 nM 3OC6-HSL.
		</LI><LI>
			After 3-hour incubation at 37°C (OD approximately reached 1.70.), 0.25 mL of each culture was harvested by centrifugalization and suspended by adding 1 mL of PBS (phosphate-buffered saline).
		</LI><LI>
			We dispensed 500 µL of each suspension into a disposable tube through a cell strainer, and its fluorescence intensity was measured with a flow cytometer of Becton, Dickinson and Company.
		</LI></OL></P><H3 id="4.3.">4.3 Results</H3><H3 id="4.4.">4.4 Discussion</H3><P>
		In the absence of the both inducers, the culture with Plux-lac hybrid promoter-<I>gfp</I> showed the background–fluorescence intensity generated by promoterless-rbs-<SPAN class="gene">gfp</SPAN> on pSB3K3. The presence of either IPTG or AHL alone had little effect on increasing the fluorescence intensity. In the presence of both inducers, the culture showed about 100-fold higher fluorescence intensity than that in the absence of both inducers. This result confirmed that the assembly of the two LacI operators and the LuxR operator integrated the inputs of IPTG and 3OC12-HSL into the output of GFP transcription. In other words, combination of the two LacI operators and the LuxR operator functioned as a genetic AND gate.
	</P><H3 id="4.5.">4.5 References</H3><P><OL><LI>
			Shotaro Ayukawa <I>et al</I>. “Construction of a genetic AND gate under a new standard for assembly of genetic parts” BMC Genomics (2010)
		</LI><LI>
			Robert Sidney Cox, III <I>et al</I>. “Programming gene expression with combinatorial promoters” molecular system biology (2007)
		</LI><LI>
			Rolf Lutz <I>et al</I>. “Independent and tight regulation of transcriptional units in <I>Escherichia coli</I> via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements” Nucleic Acids Research (1997)
		</LI></OL></P><H2 id="5.">5. Previous lsrA promoter activity and our new lsrA promoter activity</H2><H3 id="5.0.">5.0 AI-2 working system</H3><P>
		First AI-2 is synthesized by <I>luxS</I> and accumulates outside the cell. Then AI-2 is imported into the cell by the LsrACDB transporter. Inside the cell, AI-2 is phosphorylated by the LsrK kinase. Phospho-AI2 relieves LsrR inhibition from the lsr operon, therefore activating lsrA promoter.
	</P><H3 id="5.1.">5.1 Sample</H3><P><UL><LI>
			Ptet-<SPAN class="gene">gfp</SPAN> on pSB1A2(JD22597)(positive control)</LI><LI>
			Promoterless-<SPAN class="gene">gfp</SPAN> on pSB6A1(JD22597)(negative control)</LI><LI>
			PlsrA-<SPAN class="gene">gfp</SPAN> on pSB1A2 <A href="http://partsregistry.org/Part:BBa_K649104">(BBa_K649104)</A>(JD22597)</LI><LI>
			PlsrA-<SPAN class="gene">gfp</SPAN> on pSB1A2 (<A href="http://partsregistry.org/Part:BBa_K117002">(BBa_K117002)</A>-gfp)(JD22597)</LI></UL>
	JD22597 is a strain lacking lsrR.</P><H3 id="5.2.">5.2 Method</H3><P><OL><LI>
			Overnight cultures of reporter strains grown at 37°C in LB medium containing appropriate antibiotics were diluted 1:100 into 3 mL of LB medium and were incubated at 37°C as fresh cultures.
		</LI><LI>
			After their OD590 reached 0.15, the fresh cultures were diluted 1:100.
		</LI><LI>
			After 4-hour incubation at 37°C, 1 mL of each culture was moved to 1.6 mL tube and its fluorescence intensity was measured with a flow cytometer.
		</LI></OL></P><H3 id="5.3.">5.3 Results</H3><P>
		Fig 5.1 Not working lsrA promoter<A href="http://partsregistry.org/Part:BBa_K117002">(BBa_K117002)</A> activity and our new lsrA promoter activity<A href="http://partsregistry.org/Part:BBa_K649100">(BBa_K649100)</A></P><H3 id="5.4.">5.4 Discussion</H3><P>
		To confirm that lsrA promoter<A href="http://partsregistry.org/Part:BBa_K117002">(BBa_K117002)</A> does not work properly, we  introduced a <I>gfp</I> gene<A href="http://partsregistry.org/Part:BBa_J54103">(BBa_J54103)</A> downstream of the promoter and transformed this construct into JD22597 lacking <SPAN class="gene">lsrR</SPAN>. As a consequence, fluorescence intensity of lsrA promoter-<SPAN class="gene">gfp</SPAN>(<A href="http://partsregistry.org/Part:BBa_K117002">(BBa_K117002)</A>-gfp) was almost the same as promoterless-<SPAN class="gene">gfp</SPAN>(negative control), showing that lsrA promoter <A href="http://partsregistry.org/Part:BBa_K117002">(BBa_K117002)</A> does not work properly. In spite of no LsrR repression, gene transcription does not take place sufficiently. On the other hand, fluorescence intensity  of our lsrA promoter-<SPAN class="gene">gfp</SPAN><A href="http://partsregistry.org/Part:BBa_K649104">(BBa_K649104)</A> was much higher than that of  promoterless-<SPAN class="gene">gfp</SPAN>(negative control), showing that our new lsrA promoter <A href="http://partsregistry.org/Part:BBa_K649100">(BBa_K649100)</A> works.  Gene transcription takes place sufficiently. The difference between lsrA promoter<A href="http://partsregistry.org/Part:BBa_K117002">(BBa_K117002)</A> and  lsrA promoter <A href="http://partsregistry.org/Part:BBa_K649100">(BBa_K649100)</A> is whether promoter contains CRP binding site(Fig 5.2) or not. Our  lsrA promoter<A href="http://partsregistry.org/Part:BBa_K649100">(BBa_K649100)</A> contains this site. According to Wang (2004) <I>et al</I>.(ref.3), cAMP-CRP directly binds to the upstream of promoter and stimulates expression of the lsr operon.
	</P><P>
		Fig 5.2 CRP recognition sites are shown in capital letter. Our new promoter lsrA<A href="http://partsregistry.org/Part:BBa_K649100">(BBa_K649100)</A> contains this sites
	</P><H3 id="5.5.">5.5 References</H3><P><OL><LI>Karina B.Xavier and Bonnie L.Bassler 2004. Regulation of Uptake and Processing of the Quorum-Sensing Autoinducer AI2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Jan. 2005, p. 238-248</LI><LI> Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou, Baolon Sun 2009. LsrR-binding site recognition and regulatory characteristics in Escherichia coli AI-2 quorum sensing. Cell Research (2009), p.1258-1268</LI><LI>Liang Wang, Yoshifumi Hashimoto, Chen-Yu Tsao, James J.Valdes, and William E.Bentley 2004. Cyclic AMP(cAMP) and cAMP Receptor Protein Influence both Synthesis and Uptake of Extracellular Autoinducer 2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Mar. 2005, p. 2066-2076</LI></OL></P><H2 id="6.">6. LsrR repression</H2><H3 id="6.1.">6.1 Sample</H3><P><UL><LI>
			Ptet-<SPAN class="gene">gfp</SPAN> on pSB6A1(JM2.300)(positive control)</LI><LI>
			Promoterless-<SPAN class="gene">gfp</SPAN> on pSB6A1(JM2.300)(negative control)</LI><LI>
			PlsrA-<SPAN class="gene">gfp</SPAN> on pSB3K3(MG1655)<A href="http://partsregistry.org/Part:BBa_K649104">(BBa_K649104)</A></LI><LI>
			PlsrA-<SPAN class="gene">gfp</SPAN>-PlsrR-<SPAN class="gene">lsrR</SPAN> on pSB3K3<A href="http://partsregistry.org/Part:BBa_K649105">(BBa_K649105)</A>(MG1655)</LI></UL></P><H3 id="6.2.">6.2 Method</H3><P><OL><LI>
			Overnight cultures of reporter strains grown at 37°C in LB medium containing appropriate antibiotics were diluted 1:100 into 3 mL of LB medium and were incubated at 37°C as fresh cultures.
		</LI><LI>
			After their OD<SUB>590</SUB> reached 0.15, the fresh cultures were diluted 1:10.
		</LI><LI>
			After 4-hour incubation at 37°C, 1 mL of each culture was moved to 1.6 mL tube and its fluorescence intensity was measured with a flow cytometer.
		</LI></OL></P><H3 id="6.3.">6.3 Construction of PlsrA-<SPAN class="gene">gfp</SPAN>-PlsrR-<SPAN class="gene">lsrR</SPAN></H3><P>
		PlsrA<A href="http://partsregistry.org/Part:BBa_K649100">(BBa_K649100)</A> and PlsrR-<SPAN class="gene">lsrR</SPAN><A href="http://partsregistry.org/Part:BBa_K649101">(BBa_K649101)</A> were amplified by PCR using the MG1655 as a template. PlsrA was ligated with igem part<A href="http://partsregistry.org/Part:BBa_E5500">(BBa_E5500)</A>.
Then PlsrR-<SPAN class="gene">lsrR</SPAN><A href="http://partsregistry.org/Part:BBa_K649101">(BBa_K649101)</A> was ligated into PlsrA-RBS-<SPAN class="gene">gfp</SPAN><A href="http://partsregistry.org/Part:BBa_K649104">(BBa_K649104)</A>.
This part<A href="http://partsregistry.org/Part:BBa_K649105">(BBa_K649105)</A> is described as PlsrA-<SPAN class="gene">gfp</SPAN>-PlsrR-<SPAN class="gene">lsrR</SPAN> because <SPAN class="gene">lsrK</SPAN> of <A href="http://partsregistry.org/Part:BBa_K649101">BBa_K649101</A> has mutation and does not work and other genes work. </P><H3 id="6.4.">6.4 Results</H3><P>Fig 6.1 the intensity level of GFP fluorescence</P><H3 id="6.5.">6.5 References</H3><P><OL><LI>Karina B.Xavier and Bonnie L.Bassler 2004. Regulation of Uptake and Processing of the Quorum-Sensing Autoinducer AI2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Jan. 2005, p. 238-248</LI><LI> Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou, Baolon Sun 2009. LsrR-binding site recognition and regulatory characteristics in <I>Escherichia coli</I> AI-2 quorum sensing. Cell Research (2009), p.1258-1268</LI><LI>Liang Wang, Yoshifumi Hashimoto, Chen-Yu Tsao, James J.Valdes, and William E.Bentley 2004. Cyclic AMP(cAMP) and cAMP Receptor Protein Influence both Synthesis and Uptake of Extracellular Autoinducer 2 in <I>Escherichia coli</I>. JOURNAL OF BACTERIOLOGY, Mar. 2005, p. 2066-2076</LI></OL></P><H2 id="7.">7. <I>in vitro</I> assay for <I>lox2272</I></H2><H3 id="7.1.">7.1 Construction of <I>lox2272</I>-<I>gfp</I>-<I>lox2272</I></H3><P>
		PlacIQ-<I>lox2272</I>-<I>gfp</I>-<I>lox2272</I><A href="http://partsregistry.org/Part:BBa_K649200">(BBa_K649200)</A> is composed of <I>lox2272</I><A href="http://partsregistry.org/Part:BBa_K649204">(BBa_K649204)</A> and PlacIQ-<I>gfp</I><A href="http://partsregistry.org/Part:BBa_J54202">(BBa_J54202)</A>. PlacIQ-<I>gfp</I> contains Bgl II and Mlu I restriction site on upstream of <I>gfp</I>. These restriction sites allows insertion of the upstream <I>lox2272</I>, designed to be flanked by the sticky ends of BamH I and Mlu I. This <I>lox2272</I> was inserted to the PlacIQ-<I>gfp</I> digested with Bgl II and Mlu I. Likewise, the down stream <I>lox2272</I> was inserted between Spe l and Pst l by same means.
</P><H3 id="7.2.">7.2 Sample</H3><P><CENTER><TABLE border="1"><TBODY><TR><TH><CENTER>sample</CENTER></TH><TH><CENTER>volume</CENTER></TH></TR><TR><TD><CENTER>PlacIQ-<I>lox2272</I>-<I>gfp</I>-<I>lox2272</I>(pSB1C3)</CENTER></TD><TD><CENTER>50 ng/µL × 4 µL</CENTER></TD></TR><TR><TD><CENTER>10× Cre recombination Buffer</CENTER></TD><TD><CENTER>1 µL</CENTER></TD></TR><TR><TD><CENTER>ddH<SUB>2</SUB>O</CENTER></TD><TD><CENTER>4 µL</CENTER></TD></TR><TR><TD><CENTER>Cre recombinase</CENTER></TD><TD><CENTER>1,000 units/mL × 1 µL</CENTER></TD></TR><TR><TH><CENTER>total</CENTER></TH><TH><CENTER>10 µL</CENTER></TH></TR></TBODY></TABLE></CENTER></P><H3 id="7.3.">7.3 Method</H3><P><OL><LI>
			The part on pSB1C3 was made linear by EcoRV restriction site which is far from <I>lox</I> sites.
		</LI><LI>
			Three identical samples and one negative control supplied with ddH<SUB>2</SUB>O instead of Cre-recombinase had been prepared and those were incubated in 37°C for 0.5hr, 2hr and 4hr respectively. Negative control was incubated for 4hr.
		</LI><LI>
			After a period of time, the samples were kept in -20°C until the whole samples are arranged. The result was checked by electrophoresis(0.8% EtBr(+) agarose gell, 100 V, 400 mA, 30 min).
		</LI></OL></P><H3 id="7.4.">7.4 Results</H3><P>
		From left, each lane corresponds to negative control, 0.5hr, 2hr and 4hr respectively. It was confirmed by earlier control experiment that several bands appear when recombination happens. As you can see in the picture, there are three bands shown in period of 2hr and 4hr. It can be said that excision at <I>lox2272</I> sites did happen.
	</P><H3 id="7.5.">7.5 References</H3><P><OL><LI>Shotaro Ayukawa, Akio Kobayashi, Yusaku Nakashima, Hidemasa Takagi, Shogo Hamada, Masahiko Uchiyama, Construction of a genetic AND gate under a new standard for assembly of genetic parts,BMC Genomics. 2010; 11</LI><LI>Kimi Araki, Masatake Araki1 and Ken-ichi Yamamura, Site-directed integration of the cre gene mediated by Cre recombinase using a combination of mutant lox sites, Nucleic Acids Research, 2002.</LI></OL></P><H2 id="8.">8. <I>in vivo</I> assay for <I>lox</I> cassettes</H2><H3 id="8.1.">8.1 Sample</H3><P>
		Two kinds of BioBricks were prepared for the assay. 
		One was PlacIQ-<I>lox2272</I>-<I>rfp</I>-<I>lox2272</I>-<I>gfp</I><A href="http://partsregistry.org/Part:BBa_K649201">(BBa_K649201)</A>, and the other was PlacIQ-<I>lox71</I>-<I>rfp</I>-<I>lox66</I>-<I>gfp</I><A href="http://partsregistry.org/Part:BBa_K649202">(BBa_K649202)</A>
The assay for these parts was carried out simultaneously.
	<CENTER><TABLE border="1"><TBODY><TR><TH colspan="2"><CENTER>sample</CENTER></TH><TH><CENTER>arabinose</CENTER></TH></TR><TR><TD><CENTER>1</CENTER></TD><TD><CENTER>PlacIQ-<I>lox</I>-<I>rfp</I>-<I>lox</I>-<I>gfp</I>(pSB3K3)  +  P<SUB>BAD</SUB>/araC-Cre(pSB1A2)</CENTER></TD><TD><CENTER>+</CENTER></TD></TR><TR><TD><CENTER>2</CENTER></TD><TD>PlacIQ-<I>lox</I>-<I>rfp</I>-<I>lox</I>-<I>gfp</I>(pSB3K3)  +  P<SUB>BAD</SUB>/araC-Cre(pSB1A2)</TD><TD><CENTER>-</CENTER></TD></TR><TR><TD><CENTER>3</CENTER></TD><TD>PlacIQ-<I>lox</I>-<I>rfp</I>-<I>lox</I>-<I>gfp</I>(pSB3K3) : negative control</TD><TD><CENTER>+</CENTER></TD></TR></TBODY></TABLE></CENTER></P><H3 id="8.2.">8.2 Method</H3><P>
		The method for two BioBricks' assay was same as below. 
	<OL><LI>
			Strain JM2.300 with both P<SUB>BAD</SUB>/araC-Cre and PlacIQ-<I>lox</I>-<I>rfp</I>-<I>lox</I>-<I>gfp</I> in it was cultured at 37°C in 3 mL of LB medium containing ampicillin (6 µL) and kanamycin(3.6 µL). Same kind of strain having only PlacIQ-<I>lox</I>-<I>rfp</I>-<I>lox</I>-<I>gfp</I> was cultured in same volume of LB containing kanamycin (3.6 µL) and carbenicillin(4 µL). These were cultured until OD 1.6.
		</LI><LI>
			Each cultured medium was 6 times diluted in the medium and three samples (3 mL each)  were dispensed from those. Among them, two were induced by arabinose (2 M, 75 µL).
		</LI><LI>
			After 30 min from induction, 1000 times dilution and 100000 times dilution of each sample were plated and incubated in 37°C about 12 hours. The florescence of plates was examined by FLA and it was taken picture. About the rest of the samples, strains were harvested by centrifugation and suspended by adding 1 mL of PBS (phosphate-buffered saline). The last OD of PBS solution was approximately 0.4. We dispensed 700 µL of each suspension into a disposable tube through a cell strainer, and fluorescence intensity of each cell was measured with a flow cytometer of Becton, Dickinson and Company.
		</LI><LI>
			In the same way as 3, 9 more samples(for each BioBrick) were examined at period 1hr, 2hr, and 4hr.
		</LI></OL></P><H3 id="8.3.">8.3 Results</H3><P><CENTER>(a)
	</CENTER><CENTER>(b)
	</CENTER><CENTER>(c)
	</CENTER></P><P>
		Fig 8.1 Cre-meditated recombination at <I>lox2272</I> cassette. Cre-induction period of 0.5 hr.(a) Overlay of Green and Red channel.  The leftmost is a negative control which don't have Cre-expressing plasmid. The center is an arabinose induced sample which has both Cre plasmid and BioBrick K649201. The rightmost is a uninduced strain which has both plasmid like as the center. (b)Detection of GFP. The order of samples is same as above.(c)Detection of mCherry. The order of samples is same as above.
	</P><CENTER>(a)
	</CENTER><CENTER>(b)
	</CENTER><CENTER>(c)
	</CENTER><P>
		Fig 8.2 Cre-meditated recombination at <I>lox71/66</I> cassette. Cre-induction period of 0.5 hr.(a) Overlay of Green and Red channel.  The leftmost is a negative control which don't have Cre-expressing plasmid. The center is an arabinose induced sample which has both Cre plasmid and BioBrick K649202. The rightmost is a uninduced strain which has both plasmid like as the center. (b)Detection of GFP. The order of samples is same as above.(c)Detection of mCherry. The order of samples is same as above.
	</P><P><CENTER>Fig 8.3 Images of six samples of K649201 (up) and K649202 (down) Cre-induced period of 0.5 hr</CENTER></P><TABLE border="0"><TBODY><TR><TD><CENTER>(a)</CENTER></TD><TD><CENTER>(b)</CENTER></TD></TR><TR><TD><CENTER>(c)</CENTER></TD><TD>
				Fig 8.4 identical plates with Fig 8.3
				(a)expression levels of red and green florescence of K649201
				(b)expression levels of red and green florescence of K649202
				(c)examined area for comparing between red and green florescence at each plate
			</TD></TR></TBODY></TABLE><P>
		Fig 8.5 Green florescence level of each cell was detected by flow cytometer. (a)arabinose induced strain containig K649201 and cre-expressing plasmid (b)arabinose supplied strain containing only K649201 (c)arabinose induced strain containing K649202 and cre-expressing plasmid (d)arabinose supplied strain containing only K649202 
	</P><H3 id="8.4.">8.4 Discussions</H3><P>
		We made it clear at the previous page that two cassettes of different excision frequency are necessary for the adequate randomizer. We confirmed that our <I>lox71/66</I> and <I>lox2272</I> cassettes work in strain. In addition, we are informed that the recombination efficiency of <I>lox71/66</I> is superior to that of <I>lox2272</I> from the results of the experiments. It is clear that excision occurs more frequently as longer as the sequence flanked by lox cassette is. Therefore, it is reason to design the randomizer with these two cassettes as following.
	</P><P><I>lox71</I>-<I>lox2272</I>-<I>encoding gene</I>-<I>lox66</I>-<I>encoding gene</I>-<I>lox2272</I></P><P>
		By designing sequence flanked by <I>lox2272</I> longer than that by <I>lox71/66</I>, we may operate the frequency equally.
	</P><H3 id="8.5.">8.5 References</H3><P><OL><LI>Jean Livet, Tamily A. Weissman, Hyuno Kang, Ryan W. Draft, Ju Lu, Robyn A. Bennis, Joshua R. Sanes and Jeff W. Lichtman, Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system, Nature 2007.</LI><LI>Kimi Araki, Masatake Araki1 and Ken-ichi Yamamura, Site-directed integration of the cre gene mediated by Cre recombinase using a combination of mutant lox sites, Nucleic Acids Research, 2002.</LI></OL></P></DIV><DIV style="bottom: 0;left: 30; position:fixed; background-color: #000000; opacity:0.8;"><A onclick="scrollTo(0,0);return false;" onfocus="this.blur();" style="color:#FFFFFF;">Return to Page Top</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay">http://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/assay</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tokyo_Tech/Projects/RPS-game/assay" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tokyo_Tech/Projects/RPS-game/assay" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/assay&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/RPS-game/assay&amp;oldid=261864" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>