"
Page
Discussion
View source
History
teams
Log in
 
Team:Tokyo Tech/Projects/RPS-game/index.htm
From 2011.igem.org
Tokyo Tech 2011
Home
Project
RPS-Game
Make it Rain
Urea Coolers
Data page
Modeling
RPS-Game
Urea Coolers
Human Practice
Sitemap
More
Safety
Attribution and Contributions
NoteBook
Team
Sponsors
Collaboration
1. The Hands
2. The Judges
2.1 Using AND-Gate promoters to create Judges
2.2 Creating Parts that responded correctly to our set of Signaling Molecules
2.3 Improving PlsrA
2.4 Improving Plas
3. The Randomizers
3.1 Single Colony Isolation
3.2 Conditional Knockout by Recombination
3.2.1 The Requirements
3.2.2 The Mechanism
3.2.3 Testing the Lox Cassettes
3.2.4 Playing Fair: Future Work
3.3 Survival of one strain
3.3.1 Introduction: Very small differences determine who will survive
3.3.2 Adjusting the Model to create a True Randomizer
3.3.3 How the Three Types of Bacteria Compete for Survival
3.3.4 The Old Model
3.3.5 Our New Model
3.3.6 The Biological Meaning of our Model
3.3.7 Making it Obvious
Rock-Paper-Scissors game
Introduction
The Hands
The first step towards making an RPS game that can be played    between humans and bacteria is giving each player a set of    signaling molecules through which they can communicate their    choice of rock, paper or scissors. For that purpose we created    two sets of three signaling molecules corresponding each to rock, paper or scissors.    For humans we used IPTG, aTc and salicylate, respectively.    For E. coli we used 3OC6-HSL, 3OC12-HSL and AI-2, respectively.
The Judges
Although we defined a set of six signaling molecules that can be used to    play the RPS game, we still need to find a way to know who wins the game.    To know who the winner of each game is, we designed a set of E. coli that act as judges. Because there were no working parts related to AI-2 or 3OC12-HSL, we constructed new working  lasI promoter, lsrA promoter and LsrR coding gene.
The Randomizers
Although our set of six signaling molecules allows us to play RPS with E. coli, we must make sure E. coli can choose any of its three signaling molecules with the same probability    in order to be able to play RPS fairly and properly.    To do so, we designed three kinds of randomizers.
1. The Hands
The first step towards making an RPS game that can be played between     humans and bacteria is giving each player a set of signaling molecules     through which they can communicate their choice of rock, paper or scissors.     For that purpose we created two sets of three signaling molecules     corresponding each to rock, paper or scissors. For humans we used IPTG,     aTc and salicylate, respectively.     For E. coli we used 3OC6-HSL, 3OC12-HSL and AI-2, respectively.
2. The Judges
Although we defined a set of six signaling molecules that can be used     to play the RPS game, we still need to find a way to know who wins the game.     To know who the winner of each game is,     we designed a set of E. coli that act as judges.     Each Judge E. coli has an AND-gate promoter and     a fluorescent protein gene that is expressed when the AND-gate promoter     is activated. In this way, the Judge E. coli can     let us know its decision by producing GFP, RFP or CFP to indicate whether     humans win, lose or it is a tie, respectively.
2.1 Using AND-Gate promoters to create Judges
The first step to make the Judge E. coli was to     find a logic device which could allow the Judge to decide who the winner of the RPS game was.     We found that the AND-gate promoters would fit perfectly for that purpose,     since they can take two signaling molecules as inputs and produce one indicator     as output. Since each of the players has a set of three different signaling     molecules, we need a set of nine Judges, each of which has an AND-gate promoter     that is activated only by one of the nine possible pairs of signaling molecules.     These combinations are shown in the image below.
Our next mission was then to check if there were AND-gate promoters BioBricks     that we could use. We searched in the Registry and found a potential AND-gate     promoter designed by iGEM 2007's team Tokyo_Tech. This potential AND-gate     promoter is designed to be activated by the addition of both IPTG and 3OC6-HSL.     However, there was no data showing the IPTG dependency of this promoter,     so we did experiments and confirmed this dependency for the first time in iGEM.     We concluded that the addition of both IPTG and 3OC6-HSL regulates the activity     of this AND-gate promoter. In this way, we completed the construction of one of     the Judges E. coli, which proves in principle that     our game is feasible. To know the detailed method about this assay,     please see here .
Fig. 2.1 Tokyo_Tech AND-gate promoter
This Plux-lac hybrid promoter contains two LacI operators, a LuxR operator and luxR.     We introduced this part into LacI expressing E. coli strain.     Because IPTG controls the binding of LacI to two LacI-operator parts and 3OC6-HSL     controls the binding of LuxR to a LuxR-operator part, the gfp gene activity of the reporter part is dually regulated by IPTG and 3OC6-HSL.     We used promoterless pSB3K3-gfp (BBa_J54103) as a negative control,     and pAC-Pλ-gfp (chloramphenicol-resistance), which constitutively expressed GFP,     as a positive control. To know about the mechanism of this promoter click here .
2.2 Creating Parts that responded correctly to our set of Signaling Molecules
In the process of constructing enough AND-gates that could suffice the needs     of our RPS game design, we discovered two faulty BioBricks: lsrA promoter (BBa_K117002) and las promoter (BBa_J64010).     Because of these faulty parts, the Judge E. coli set we had designed could only sense the Player E. coli's     signaling molecule 3OC6-HSL (Rock). This ultimately led to an unfair game     because humans could win every time they played with the Paper signaling molecule     (for more on the “Sad story of the Rock Player” click here ).
To fix this problem, we improved the old defective las and lsrA promoters parts     by making new parts that work! As can be seen in the experimental information     below (see “Improving lsrA promoter” and “Improving las promoter”),     we confirmed our lasI promoter ( BBa_K649000 )     and lsrA promoter ( BBa_K649100 ) work perfectly!
Since the lsrA promoter plays a key role in the correct functioning of AI-2,     fixing these parts now allows us to use AI-2 as a signaling molecule, which is a     promising advance because of the characteristics of the AI-2 mechanism .    This mechanism prevents AI-2 from cross-talking with other signaling molecules     such as AHL. Hence, this signaling molecule is a very powerful     tool to build complex Synthetic Biology systems.
Finally, one thing we would like outline is that although the promoters we made are single input promoters, confirmation of their activity is required as a     reference to construct AND gate promoters. Therefore, we solved important issues     and made significant advances towards constructing AND-gate promoters. This allows E. coli to also choose the signaling molecules     corresponding to Paper and Scissors, so we have again a working RPS game design.
2.3 Improving PlsrA
Fig. 2.2 Not working lsrA promoter( BBa_K117002 )
activity and our new lsrA promoter( BBa_K649104 ) activity
We confirmed that the lsrA promoter ( BBa_K117002 ) does not work properly     (samples used our experiment are listed in Table 2.1 below). The fluorescence intensity of GFP of lsrA promoter-gfp(( BBa_K117002 )-gfp) was lower     even than those of the negative control (Fig. 2.2), which clearly shows that     lsrA promoter(( BBa_K117002 ) does not work as expected. In this experiment,     we measured transcriptional activity of lsrA promoter by introducing a gfp gene downstream of this promoter (Fig. 2.3).
Table 2.1 Samples used our experiment
Name
Strain
Plasmid
sample1
JD22597
Ptet-gfp on pSB1A2
sample2
Promoterless-gfp on pSB6A1
sample3
PlsrA-gfp on pSB1A2 ( BBa_K649104 )
sample4
PlsrA-gfp on pSB1A2 (( BBa_K117002 )-gfp)
Fig. 2.3 lsrA promoter-gfp(( BBa_K117002 )-gfp)
To solve this problem, we created the first working iGEM lsrA promoter ( BBa_K649100 ).     Its fluorescence intensity was much higher than that from a promoter-less gfp negative control plasmid, showing that our new     lsrA promoter works(Fig. 2.5). In this experiment, we measured the transcriptional     activity of our lsrA promoter by introducing a gfp gene downstream of     the promoter( BBa_K649104 , Fig. 2.4). Details about this experiment can be found here .
Fig. 2.4 lsrA promoter-gfp( BBa_K649104 )
Fig. 2.5 LsrR represses lsrA promoter.
Moreover, this promoter can be repressed by our new LsrR part( BBa_K649105 ).     (samples used our experiments are listed in Table 2.2 below) The fluorescence     intensity of GFP of sample 3 was three times as large as     that of sample 4. This result shows that LsrR successfully repressed     lsrA promoter. In this experiment, we measured LsrR repression activity by     introducing a gfp gene downstream of lsrA promoter (( BBa_K649105 , Fig. 2.6).    Details about this experiment can be found here .
Table2.2 Samples used our experiment
Name
Strain
Plasmid
sample1
JM2.300
Ptet-gfp on pSB6A1
sample2
Promoterless-gfp on pSB3K3
sample3
MG1655
PlsrA-gfp on pSB3K3
sample4
PlsrA-gfp-PlsrR-lsrR on pSB3K3
Fig. 2.6 lsrA promoter-gfp-lsrR promoter-lsrR( BBa_K649105 )
2.4 Improving las promoter
Fig. 2.7 (a)
(b)
Fig. 2.7 (a): Not working lasI promoter (BBa_J64010).     Fig. 2.7 (b): New working lasI promoter (BBa_K649000) we made.     We confirmed it works as expected. In our assay, we used the same LasR     regulator part used in the assay of BBa_ J64010. Clearly, for our     part the fluorescence intensity of 3OC12-HSL+ was higher than that of 3OC12-HSL-.
To know detailed methods about these lasI promoter assay, please see here, previous part and new part .
To prove that the LasR regulator used in our PlasI assay works, we did another assay.     Details about this assay can be found here.
3. The Randomizers
Although our set of six signaling molecules allows us to play RPS with E. coli, we must make sure E. coli can choose any of its three signaling molecules with the same probability     in order to be able to play RPS fairly and properly. To do so, we designed     three kinds of randomizers: one kind which needs of three types of bacteria     (each of which produces one of the three RPS signaling molecules),     and the other kind that needs of only one type of bacteria which can     synthetize each of the three signaling molecules one at a time and randomly.     Namely, the randomizers are Single Colony Isolation,     Survival of one Strain and Conditional Knockout by Recombination.
3.1 Single Colony Isolation
This is our simplest randomizer design. To make sure E. coli chooses any of its signaling molecules with equal probability, we put the constructs     for each molecule inside a different bacterium, so we create three types of bacteria:     one synthetizing the corresponding signaling molecule for rock, other synthetizing     the corresponding signaling molecule for paper, and lastly one synthetizing the     corresponding signaling molecule for scissors. By randomly isolating a single colony     out of the many colonies that result from the mixing between the three types of E. coli, we get a random output as E. coli's choice for the RPS game.
3.2 Conditional Knockout by Recombination
Our second randomizer differs from our other two randomizers in that all the     three signaling molecules are produced one at a time and randomly by only one type     of bacteria. We were inspired by a paper about “brainbow” research     on mice to create this randomizer (Livet J et al., 2007), which is based     on the recombination mechanism of the enzyme Cre and the lox sequences.     We designed a Cre-Lox system which allows E. coli to express one of its three signaling molecules by means of conditional knockout.     The design is depicted in Fig. 1. It should be noted this randomizer is designed     to be used at a single-cell level. When this randomizer is used in groups of cells,     the different signals released by the cells will mix. In this case, by using microfluidic     devices or isolating single colony of bacteria, we can obtain only one of the signaling     molecules produced by the initial group of cells.
Fig. 3.1 - (a) Cre recombinase construction. (b) lox cassettes distribution for the randomizer design
3.2.1 The Requirements
Basically, each pair of lox sites (indicated by the same color) mark the points     which the enzyme Cre will excise (they will be cut off the backbone along with the     sequence between them). For a design that allows choosing randomly one of E. coli’s three signaling molecules, at least two     cassettes of lox sites are needed. When these two cassettes of lox sites and     protein coding sequences are arranged as in Fig. 3.1(b),     only one signaling molecule is produced.
Our design also required a way to control luxI gene's expression.     To do so, we used an inducible promoter instead of a constitutive promoter.     A constitutive promoter would have caused luxI gene to be expressed beforehand and     could lead to an E. coli producing two signaling molecules     at the same time (the equivalent of showing two hands in the RPS game). In contrast,     the inducible promoter prevents a particular gene from expressing preferentially.
One last but not less important requirement for our randomizer is that it should     express each of the three signals not only one at a time, but also with the same     probability. This makes the game fair in the sense that E. coli’s choice in the RPS game is not predictable.
3.2.2 The Mechanism
When the blue cassette of Lox sites is excised (Fig. 3.1), the signaling molecule     coded by lasI (3OC6-HSL) will be produced. Likewise, when the black Lox cassette     is excised, the signaling molecule coded by luxS (AI-2) will be produced. On the other hand,     a third possible outcome is that recombination does not take place. In this case,     the signaling molecule coded by luxI (3OC6-HSL) will be produced. Also note that excision     of one kind of lox cassette removes the remaining cassette,     thereby preventing further recombination.
As mentioned before, one of the requirements for our randomizer was to have at least     two lox cassettes. This prevents excision of lox sites from different cassettes     (for example one blue lox site and one black lox site). Because the lox71-lox66 cassette and the lox2272-lox2272 cassette are incompatible (Zorana Carter and Daniela Delneri et al., Yeast 2010), we can use them to build our randomizer.
3.2.3 Testing the Lox Cassettes
As stated above, we need two lox cassettes of different recombination frequency     for randomizer. Because there was no working lox parts in registry, we constructed     three original BioBricks for testing whether lox2272 and lox71/66 cassettes work. For     the convenience of testing, fluorescence expressing genes were used in place of signal     molecular expressing genes in construction. After figuring out their working in vitro, we tested them in vivo by detecting red and green fluorescence through fluoro imager and flow cytometer. Furthermore, we compared the relative recombination frequency of two cassettes . Our lox Cassettes constructions     were working properly, and their recombination frequency were different from each other.
PlacIQ-lox2272-gfp-lox2272 (BBa_K649200)
PlacIQ-lox2272-rfp-lox2272-gfp (BBa_K649201)
PlacIQ-lox71-rfp-lox66-gfp (BBa_K649202)
The in vitro assay with K649200 was made in advance. The preliminary experiment allowed     us to confirm that the Cre-mediated recombination on lox2272 cassette works as designed.     In the assay, Cre recombinase was added to the linear DNA and incubated for 0.5, 2, and 4 hours.      Images of the experiments have been added below.
When checking the result by electrophoresis, there were several bands in samples to     which Cre was added(1st, 2nd lane from right which corresponds to 4 hr and 2 hr respectively).     It indicates that excision of the lox sites successfully occurred.     To know detailed about this assay, please see here, in vitro assay for lox2272
For the in vivo assay, by detecting fluorescenc levels of GFP and mCherry,     we could determine whether recombination occured in K649201 and K649202 and compare     relative recombination frequency between two of them. We prepared a competent cell     JM2.300 into which PBAD/araC-Cre(pSB1A2, BBa_I718008) had been constructed.     Subsequently, our BioBrick was constructed into the cell.
sample
arabinose
1
PlacIQ-lox-rfp-lox-gfp(pSB3K3)
PBAD/araC-Cre(pSB1A2)
+
2
PlacIQ-lox-rfp-lox-gfp(pSB3K3)
PBAD/araC-Cre(pSB1A2)
-
3
PlacIQ-lox-rfp-lox-gfp(pSB3K3)
: negative control
+
The strain was grown in a 3 mL liquid culture, and 75 µL of 2 M arabinose was added     to induce Cre expression. We used two controls for the experiment. One was the same strain     without arabinose induction, and the other was JM2.300 strain which was induced by arabinose     and had only our BioBrick. All the strains were cultured each for periods of 0.5, 1, 2, and 4 hours,     and in each case the florescence levels were measured by flow cytometer and FLA.     To know the detailed method about this assay, please see here in vivo assay for lox cassettes .
We confirmed our results optically by taking florescence images.     K649201 transformants with with 0.5 hr-induction of Cre in liquid medium and its two control strains were                    plated and incubated in 37°C for 12 hours. Images of the three conditions were taken using red     florescence filter, green florescence filter and no filter as shown below, respectively.
(a)
(b)
(c)
Fig. 3.2 Cre-meditated recombination at lox2272 cassette. Cre-induction period of 0.5 hr                         (a)Overlay of Green and Red channel. The leftmost is a negative control which don't have Cre-expressing                                     plasmid. The center is an arabinose induced sample which has both Cre plasmid and BioBrick K649201.     The rightmost is a uninduced strain which has both plasmid like as the center.     (b)Detection of GFP. The order of samples is same as above.     (c)Detection of mCherry. The order of samples is same as above.
On the sample with the PBAD/araC-Cre construction, we found that recombination occurred     when arabinose was added. In contrast to this result, when we measured the levels of the sample     without the PBAD/araC-Cre construction, we found that the GFP levels were far lower than     those of the sample with the PBAD/araC-Cre construction. This clearly proves that our lox constructions, both in K649201 and K649202, respond correctly to the effects of Cre recombinase.     A slight detection of green florescence in plate absence of PBAD/araC-Cre can be explained                         that there happened cross-talk to green channel by FMN(Flavin mononucleotide) or expression of GFP according                         to malfunction of terminator before gfp. We could also observe recombination occurred when arabinose was not                          added, which can be explained due to a leaking in the PBAD/araC promoter.You can find that K649202 also works well from the images of K649202 on here .
Furthermore, we could observe that the arabinose(+) sample of K649202 has higher green/red ratio than that of K649201, which implying the frequency of lox71/66 casette is higher than that of lox2272.
Fig. 3.3 Image of six samples of K649201 (up) and K649202 (down) at period of 0.5 hr
(a)
(b)
Fig. 3.4 identical plates with Fig. 3.3
(a)expression levels of red and green florescence of K649201
(b)expression levels of red and green florescence of K649202
(c)examined area for comparing between red and green florescence at each plate
(c)
As we examining green florescence in comparison to red florescence, green expression level was lower than red in     K649201(Fig. 3.4(a)), which meaning that considerable plasmids in those cells yet. In contrast, green expression level exceeded red in K649202(Fig. 3.4(b)). This result implies that recombination frequency of lox71/66 cassette is relatively high than that of lox2272 cassette.
Fig. 3.5 Green fluorescence level of each cell was detected by flow cytometer.
(a)arabinose induced strain containing only K649201 and cre-expressing plasmid (b)arabinose supplied strain containing only K649201 (c)arabinose induced starin containing K649202 and cre-expressing plasmid (d)arabinose supplied strain containing only K649202
The higher recombination efficiency of lox71/66 compare to lox2272 was confirmed also by flow cytometer intensity of lox71/66 was higher than that of lox2272, which supports the result detected by FLA.
3.2.4 Playing Fair: Future Work
To make each of the outcomes (R, P, and S) equally probable, we are going to quantify the recombination frequency of each lox cassette. This information and adequate Cre induction will be likely to allow us to have an RPS player E. coli whose choice of either of rock, paper or scissors cannot be predicted.
In our next experiments, we are going to vary the reaction time and the distance between the lox sites     of each cassette. We believe precise modification of this two parameters must lead to our goal of     making a randomizer in which each of the signaling molecules can be expressed with the same frequency     (which results in each of the outcomes being expressed with the same probability).
3.3 Survival of One strain
3.3.1 Introduction: Very small differences determine who will survive
In this section we will show a shocking scenario of evolution:     the future of each of three different rival strains (whether the strain will die or survive)     is marked by very small differences between the initial population densities of the strains (a phenomenon also known as the “butterfly effect”).     Furthermore, we will also show that we can apply this very interesting result to create a randomizer     that can be used in our Rock-Paper-Scissors game, due to the fact that only one of the rival strains     will survive. More specifically, we assign to each of the three rival strains either of Rock, Paper or Scissors,     make them compete for survival and take the surviving strain to represent the bacteria's choice for the RPS game.
3.3.2 Adjusting the Model to create a True Randomizer
The idea for creating this randomizer was born from a paper written in     1996 by Durrett and Levin. In it, the authors described a system of three types of bacteria that competed for     survival in dynamic that resembled a Rock-Paper-Scissors (RPS) game. However, the model proposed in this paper is not fully appropriate for our RPS randomizer,     since one of the three types of bacteria cannot ultimately survive     (although it can dominate the system, i.e. have the highest population density, for definite periods of time).     We will discuss more on the limitations we found in this model to be adopted as a     randomizer and the modifications we made to create a true randomizer.
3.3.3 How the Three Types of Bacteria Compete for Survival
The three types of bacteria that compete for survival use three tactics to outcompete their rivals:     the production of a toxin (a bacteriocin called colicin) that is toxic to other strains,     resistance to the toxin produced by other strains, and a higher birth rate than their rival strains.     Namely, the three types of bacteria are: colicin-producing E. coli (R),     colicin-resistant E. coli (P)     and colicin-sensitive E. coli (S).     The colicin-producer outcompetes the colicin-sensitive by producing the colicin. The colicin-sensitive     bacteria outcompetes the colicin-resistant because its birth rate is higher than that of the colicin-resistant.     The colicin-resistant outcompetes the colicin producer because its birth rate is higher     than that of the colicin producer. The colicin resistant bacteria are also able to produce colicin,     but at a lower energetic cost, which allows them to have a higher birth rate.
The system was described by the following general differential equations
Where
3.3.4 The Old Model
In the model described by Durrett and Levin’s paper the equations were as follows:
Producer
Resistant
Sensitive
These equations show that the colicin-resistant bacteria are completely immune to colicin     (there is not death factor associated to colicin in the equation for du2/dt).     However, as will be explained afterwards, this results in a loss of     balance that does not allow building a true randomizing system.
Now, setting the parameters as follows, the graph below was created by Durrett and Levin.
3.3.5 Our New Model
As mentioned before, the model proposed by Durrett and Levin has critical limitations as a randomizer for the RPS game.     To be able to create a true randomizer, we modified the differential equations of the model taking care to give it     a biological meaning. With our new differential equations, any of the three types of bacteria can ultimately survive     by outcompeting the other two strains, which will die. More specifically, we limited the resistance of the     colicin-resistant bacteria in the sense that it would produce a type of bacteriocin that is only toxic to itself and     to the sensitive strain, and additionally the resistant strain would also be vulnerable to the colicin produced by     the colicin-producer.  Since which strain will be the one that survives is determined by very small differences in     the initial concentrations of the three different populations of bacteria, in practice this systems becomes a     randomizer because of the imprecisions in the measurements that result, for example, when using micropipettes.     This randomizer describes a new competition dynamic that could not be reproduced in the previous model proposed     by Durrett and Levin due to the instability along the
axis.
If we set the parameters as follows
and we graph this equations using a Matlab program,     we get a graph which clearly shows there are stable points on each of the three axes     (Fig. 1, Up).
Fig. 1 Up: Our New model. Down: The Old Model
These stable points (u1,0,0), (0,u2,0) and (0,0,u3) indicate that for the equations     we have set all of the three strains may ultimately survive for infinite peiriods of time.     The differences between our model and the model of Durrett and Levin can be seen graphically     in Fig. 1. These graphs were plotted using Matlab.
Note that the parameters we have set for our equations satisfy the initail conditions     of the model proposed by Durrett and Levin (indicated in black font)
3.3.6 The Biological Meaning of our Model
From a biological perspective, our model describes the existence of two strains of     bacteria that produce two different types of bacteriocins. One of these strains is not     completely resistant to its own bacteriocin nor to the bacteriocin produced by its     rival strain. This can be justified as the consequence of insufficient/ineffective     resistance protein production by the “resistant” strain. This limitation in the     production of resistance protein could be thought of as a consequence of the “resistant” strain being a mutant of a colicin-sensitive strain.
3.3.7 Making it Obvious
From the graph of our new model (Fig. 1, left) it can be deduced that there are paths     that converge at stable points (u1,0,0), (0,u2,0) and (0,0,u3), and that this paths all     have an approximately common origin. In this section we would like to show that the     origin of these paths is practically the same, and that in that sense we have designed     a true randomizer (since, as mentioned before, the imprecisions that result in the     experimental measurements will make it impossible to make the initial     population density of the three strains the same).
In the following set of graphs we will make it obvious that each of the three     different strains of E. coli to survive in a random fashion by very small    differences on the initial concentrations of each strain.
We modeled our results using Matlab.      As can be seen in the graphs below, each of the strains can survive if their     initial density in only tree hundredths (a.u.) greater than the other two strains'     initial concentrations.
With these graphs it becomes clear that the imprecisions in experimental measurements       (i.e. pipetting) are enough to cause the outcome of Rock, Paper or Scissors signaling       molecule to be random. Consequently, we can conclude that this randomizer is not       only feasible but also practical and effective (let alone interesting).
References
Patrick C. Seed et al. “Activation of the Pseudomonas aeruginosa lasI Gene by LasR and the Pseudomonas Autoinducer PAI: an Autoinduction Regulatory Hierarchy” JOURNAL OF BACTERIOLOGY (1995)
Shotaro Ayukawa et al. “Construction of a genetic AND gate under a new standard for assembly of genetic parts” BMC Genomics (2010)
Robert Sidney Cox, III et al. “Programming gene expression with combinatorial promoters” molecular system biology (2007)
Rolf Lutz et al. “Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements” Nucleic Acids Research (1997)
Karina B.Xavier and Bonnie L.Bassler 2004. Regulation of Uptake and Processing of the Quorum-Sensing Autoinducer AI2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Jan. 2005, p. 238-248
Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou, Baolon Sun 2009. LsrR-binding site recognition and regulatory characteristics in Escherichia coli AI-2 quorum sensing. Cell Research (2009), p.1258-1268
Liang Wang, Yoshifumi Hashimoto, Chen-Yu Tsao, James J.Valdes, and William E.Bentley 2004. Cyclic AMP(cAMP) and cAMP Receptor Protein Influence both Synthesis and Uptake of Extracellular Autoinducer 2 in Escherichia coli. JOURNAL OF BACTERIOLOGY, Mar. 2005, p. 2066-2076
Jean Livet, Tamily A. Weissman, Hyuno Kang, Ryan W. Draft, Ju Lu, Robyn A. Bennis, Joshua R. Sanes and Jeff W. Lichtman (2007). Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system. Nature.
Kimi Araki, Masatake Araki1 and Ken-ichi Yamamura (2002). Site-directed integration of the cre gene mediated by Cre recombinase using a combination of mutant lox sites, Nucleic Acids Research
Durrett, R., Levin, S. Allelopathy in Spatially Distributed Populations (1997). Journal of Theoretical Biology, 185, 165-171.
Return to Page Top
Retrieved from " http://2011.igem.org/Team:Tokyo_Tech/Projects/RPS-game/index.htm "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
