Team:Queens Canada/error2
Error Prone PCR · Benchling
Error Prone PCR
Introduction
Error prone PCR is a method by which random mutants may be inserted into any piece of DNA.
Materials
DNA primers for PCR amplification
DNA template
Four separate solutions of 20 mM dATP, TTP, dCTP, and dGTP
Taq DNA polymerase, stored at -20 C
10 X PCR buffer: 100 mM Tris-HCl, pH 8.3, 15 mM MgCl2, and 500 mM KCl
1 MgCl2
50 mM MnCl2
Agarose
10 X TBE buffer: 1 M Tris base, 1 M boric acid, and 20 mM EDTA, pH 8.0
10 mg/mL ethidium bromide
Loading buffer: 10 mM Tris-HCl, pH 7.6, 0.15 % orange G, 0.03% xylene cyanol FF, 60% glycerol, 60 mM EDTA
DNA mass ladder for quantification
Gel purification kit
Procedure
Prepare a stock solution of the DNA template (~50-100 ng/μL) in water.
Label 16 thin-walled PCR tubes as "reactions 1 through 16".
Combine the reagents listed in Table 1 in a conical vial and label it "PCR reaction mixture"
Dispense the PCR reaction mixture from step 3 into tubes 1 through 16 by 96 μL to tube 1 and add 88 μL to tubes 2-16.
Add 2 μL of the DNA template to tube 1.
Place tube 1 in the thermocycler and start the PCR program (Table 2).
A
A
B
B
C
C
D
D
1
1
Reagents
Stock Concentration
Volume
Final Concentration
2
2
Forward Primer
100 μM
15 μL
1 μM
3
3
Reverse Primer
100 μM
15 μL
1 μM
4
4
dCTP and dTTP
20 mM
75 μL/ea
1 mM
5
5
dATP and dGTP
20 mM
15 μL/ea
0.2 mM
6
6
PCR buffer w/ Mg2+
10X
150 μL
1x
7
7
MgCl2
1M
8 μL
~5.5 mM
8
8
DMDC H2O
1098 μL
9
9
Final Volume
1491 μL
PCR Reagent Mixture
A
A
B
B
C
C
D
D
1
1
Step
Temperature
Duration
2
2
Template Denaturation
95 - 98°C
1 min
One time only
3
3
Oligo Denaturation
94°C
1 min
Repeat next 3 steps x30
4
4
Annealing
58°C
1 min
This may change depending on primers
5
5
Extension
72°C
2 min
6
6
Final Elongation
72°C
5 min
7
7
Product Storage
4°C
forever
Repeat steps 3-5 for 30 cycles
Table1
Once the PCR program has reached the annealing temperature, add 1 μL of freshly prepared MnCl2 solution and 1 μL of Taq DNA polymerase to the PCR reaction tube.
Perform 4 cycles of PCR amplification using the hot start procedure (see Note 3).
Remove the PCR tube from the thermocycler
Place the tube on ice.
Transfer 10 μL of the PCR reaction from tube 1 to tube 2.
Place tube 2 in the thermocycler and start the PCR program.
Once the PCR program has reached the annealing temperature, add 1 μL of freshly prepared MnCl2 and 1 μL of Taq DNA polymerase to the PCR reaction tube.
Perform 4 cycles of PCR amplification using the host start procedure.
Remove the PCR tube from the thermocycler
Place the tube on ice.
Repeat steps 11 through 16 using tubes 3 through 16 to create a mutagenic library by serial dilution amplification (image down below)
Verify the quality of each PCR reaction by agarose gel electrophoresis.
Combine 50 μL of each PCR reaction into a single tube and store on ice; this is your DNA library
Purify the DNA library by agarose gel electrophoresis
Recover the DNA library using a gel purification kit.
Quantify the DNA library by agarose gel electrophoresis
NOTES
Primer should be complemetary and have similar melting techniques: http://bioweb.uwlax.edu/GenWeb/Molecular/seq_anal/primer_design/primer_design.htm
Optimal PCR conditions should be determined prior to the use of this mutagenic PCR protocol. The doubling efficiency for the normal PCR reaction should be ∼1.7–1.9 per PCR cycle and can be analyzed by running the PCR product after each cycle on an agarose gel
Taq DNA polymerase should be stored at −20°C and kept on ice or in a frozen metal block when taken out of the freezer. The hot start procedure refers to adding the Taq DNA polymerase to the PCR reaction mixture after the mixture has reached the annealing temperature. Either standard or “hot start” Taq DNA polymerase will work in the described method.
It is important to prepare a fresh MnCl2 solution for the error-prone PCR experiment. This solution should be stored on ice and combined with the PCR reaction mixture at the start of each serial dilution.
It is important to prepare a fresh MnCl2 solution for the error-prone PCR experiment. This solution should be stored on ice and combined with the PCR reaction mixture at the start of each serial dilution.
The method described here was optimized for a 400-nucleotide gene. We highly recommend that experimenters read ref. 7 for information on the predicted average number of doublings and mutations expected as a function of template length
Agarose gel electrophoresis should be performed by standard methods. A DNA ladder with an appropriate base pair range should be used to identify and estimate the quantity of the PCR product.
It is recommended to save a portion of the PCR product from each tube for further analysis or amplification.
The concentration of the DNA product can be determined by comparing the intensity of the product bands to the intensity of the bands from the DNA quantification ladder from commercial sources.
Transformation of Libraries
GIbson Assembely can be performed if the primers of the linearized vector are complimentery to the error-prone PCR product.
References
Random Mutagenesis by Error-Prone PCR McCullum E Williams B Zhang J Chaput J Methods in molecular biology (Clifton, N.J.) 2010 vol: 634 pp: 103-109
