<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Queens_Canada_Applied_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Queens Canada/Applied Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="containerMenu"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right" style="margin-right:50px"><LI class="dropdown" style="color:#f9f9f9"><A href="https://2018.igem.org/Team:Queens_Canada/Description" style="color:#f9f9f9">Project</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Description" style="color:#f9f9f9">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Background" style="color:#f9f9f9">Background</A></LI></UL><LI class="dropdown" style="color:#f9f9f9"><A href="https://2018.igem.org/Team:Queens_Canada/Lab" style="color:#f9f9f9">Laboratory</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Lab" style="color:#f9f9f9">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Design" style="color:#f9f9f9">Design</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Experiments" style="color:#f9f9f9">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Demonstrate" style="color:#f9f9f9">Results</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Improve" style="color:#f9f9f9">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Parts" style="color:#f9f9f9">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Safety" style="color:#f9f9f9">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Notebook" style="color:#f9f9f9">Notebook</A></LI><LI class="dropdown" style="color:#f9f9f9"><A href="https://2018.igem.org/Team:Queens_Canada/Model" style="color:#f9f9f9">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Model" style="color:#f9f9f9">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Molecular_Dynamic_Simulations" style="color:#f9f9f9">Molecular Dynamic Simulations</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Linker_Development" style="color:#f9f9f9">Linker Development</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Michaelis-Menten_Kinetics" style="color:#f9f9f9">Michaelis-Menten Kinetics</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Fluid_Dynamics" style="color:#f9f9f9">Fluid Dynamics</A></LI><LI class="dropdown" style="color:#f9f9f9"><A href="https://2018.igem.org/Team:Queens_Canada/Hardware" style="color:#f9f9f9">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Hardware" style="color:#f9f9f9">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Pacifier" style="color:#f9f9f9">Pacifier</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Luminometer" style="color:#f9f9f9">Luminometer</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Thermistor" style="color:#f9f9f9">Thermistor</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:Queens_Canada/Software" style="color:#f9f9f9">Software </A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Software" style="color:#f9f9f9">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Luminometer_Software" style="color:#f9f9f9">Luminometer</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Linker_Software" style="color:#f9f9f9">Linker Software</A></LI><LI class="dropdown" style="color:#f9f9f9"><A href="https://2018.igem.org/Team:Queens_Canada/Human_Practices" style="color:#f9f9f9">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Human_Practices" style="color:#f9f9f9">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Interviews" style="color:#f9f9f9">Expert Interviews</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Market" style="color:#f9f9f9">Market Analysis</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Public_Engagement" style="color:#f9f9f9">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Collaborations" style="color:#f9f9f9">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/InterLab" style="color:#f9f9f9">InterLab</A></LI><LI class="dropdown" style="color:#f9f9f9"><A href="https://2018.igem.org/Team:Queens_Canada/Team" style="color:#f9f9f9">QGEM</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Team" style="color:#f9f9f9">Team</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Attributions" style="color:#f9f9f9">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Sponsors" style="color:#f9f9f9">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Entrepreneurship" style="color:#f9f9f9">Entrepreneurship</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2018.igem.org/Team:Queens_Canada"> iGEM TEAM QUEENS CANADA 2018</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">Project</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Description">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Background">Background</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Laboratory</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Lab">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Notebook">Notebook</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Molecular_Dynamic_Simulations">Molecular Dynamic Simulations</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Linker_Development">Linker Development</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Michaelis-Menten_Kinetics">Michaelis-Menten Kinetics</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Fluid_Dynamics">Fluid Dynamics</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Hardware">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Pacifier">Pacifier</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Luminometer">Luminometer</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Thermistor">Thermistor</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Software">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Luminometer_Software">Luminometer</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Linker_Software">Linker Software</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Interviews">Expert Interviews</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Market">Market Analysis</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/InterLab">InterLab</A></LI><LI><A data-toggle="collapse" data-target="#Team_side">QGEM</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Sponsors">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:Queens_Canada/Entrepreneurship">Entrepreneurship</A></LI></NAV><H2 style="width:70%;margin-left:26%">Design Approaches</H2><DIV class="column full_size" style="width:70%;margin-left:15%"><P style="width:70%;margin-left:15%;font-size:18pt">We have taken two approaches to the development of biological sensors, or biosensors, for the measurement of Cortisol:

Firstly, we have constructed a reagentless, and continuous glucocorticoid sensor which utilizes changes in Fluorescence Resonance Energy Transfer (FRET) to detect hormones. 

Secondly, we have begun developing a novel, and portable biological sensor which utilizes intein splicing to produce a luminescent signal, for hormone quantification. 

We will explore both of these methods in depths, as well as their advantages and disadvantages.
</P></DIV><DIV class="column full_size" style="width:70%;margin-left:15%"><H3 style="margin-left:15%">Approach 1: Fluorescence Resonance Energy Transfer</H3><P style="width:70%;margin-left:15%;font-size:18pt">Fluorescence Resonance Energy Transfer (FRET), is a mechanism of energy transfer between light-sensitive molecules, such as fluorescent proteins. Fluorescent proteins work by absorbing light at a peak wavelight, called the excitation wavelength, and emitting light at a higher wavelength, called the emission wavelength. When two fluorescent proteins are within close proximity of each other, it is possible to excite the first fluorescent protein at its excitation wavelength, and produce emission of the wavelength from the second fluorescent protein. For example, the excitation of Cyan Fluorescent Protein is 436 nm, and its emission is 488 nm, while the excitation of Yellow Fluorescent Protein is 517 nm and its Emission is 528 nm. If Cyan Fluorescent Protein, and Yellow Fluorescent Protein are in close proximity, excitation at 436 nm will result in emissions at both 488 nm and 528 nm. The efficiency of this energy transfer process is dependent on the proximity of the two fluorophores, and can therefore be used to quantify minor changes in the structure and activity of proteins.
</P></DIV><DIV class="column full_size" style="width:70%;margin-left:15%"><H3 style="margin-left:15%">Our FRET Biosensor</H3><P style="width:70%;margin-left:15%;font-size:18pt"> The glucocorticoid receptor (GR) regulate numerous physiological processes to maintain homeostasis, and accordingly GRs is expressed in nearly every cell of the body and is necessary for life. Glucocorticoids are capable of producing diverse and pronounced effects on homeostasis remarkably diverse, exhibiting profound variability in specificity and sensitivity. Following glucocorticoid binding, GR induces or represses the transcription of target genes which can comprise huge amounts of the human genome. 

Nuclear receptors are a family of evolutionarily conserved proteins that functions as a 
ligand-dependent transcription factor [1]. After binding certain ligands, the receptor undergoes a conformational change which activates them, 
and allows them to bind directly to DNA to alter gene transcription [1]. Circulating steroid hormones, like cortisol, are able to activate the 
receptor and mediate processes such as stress response, energy metabolism and immune responses [2]. The ligand binding domain of nuclear 
receptors generally consists of eleven alpha-helices and two beta-sheets that enable the formation of a three-layered protein structure [2]. 
There also exists a regulatory C-terminal helix, titled &quot;helix 12”, that is essential for hormone binding. There are conserved residues within 
these helices which form critical interactions with the ligand allowing for specificity within the interaction [2]. Upon binding of a agonist to the receptor, helix-12 undergoes a drastic conformational change that is required for the receptor to now bind DNA and induce transcription. Upon binding of an antagonist to the receptor, helix 12 undergoes a different conformational change that prevents induction of transcription. 

Our design takes advantage of the conformational change in helix 12 of nuclear receptors upon ligand binding to produce a measurable signal. We sought to employ the sensitivity of FRET to measure changes in conformation. FRET is highly sensitive to the distance between the first and second fluorescent reagents, and should therefore respond to the conformational change in nuclear receptors upon ligand binding. In the absence of agonist, the first fluorescent reagent, bound to the nuclear receptor or ligand binding domain thereof, will start in close proximity to the second fluorescent reagent such that FRET will be observed. In the presence of agonist, conformational changes occurs thus bringing the first and the second fluorescent reagents further apart, allowing FRET to occur less and the change in FRET can be observed. In the presence of antagonist, conformational changes occurs thus bringing the first and the second fluorescent reagents even further apart than the agonist bound conformation, allowing FRET to occur even less and the change in FRET can be observed.
One example of the way we can use this biosensor is detecting, identifying, and/or quantifying a ligand of a nuclear receptor in a biological sample. The nuclear receptor or ligand binding domain thereof is provided for the hormone of interest, here we chose the glucorticoid receptor which can bind be used to bind cortisol. The receptor is labeled with an N- terminal fluorescent protein and a C-terminal fluorescent protein. Such a construct is combined with the sample under conditions favourable for binding of the ligand, e.g., hormone, to bind to the receptor or ligand binding domain thereof. The magnitude of FRET between the N- terminal and C-terminal fluorescent proteins in the absence and presence of the biological sample is then determined, wherein a change (e.g., reduction) in FRET magnitude in the presence of the biological sample indicates that the biological sample comprises the ligand (e.g., hormone) of interest. By quantifying the change in FRET magnitude and relating it to a quantity (e.g., concentration) of the ligand, the amount or concentration of ligand in the biological sample may be determined. The biological sample obtained from a subject may be, for example, urine, saliva, blood, or serum. 


  	In principle, FRET may be measured by monitoring either (a) a decrease in the emission of the donor fluorescent reagent following stimulation at the donor's absorption wavelength, and/or (b) an increase in the emission of the acceptor reagent following stimulation at the donor's absorption wavelength. In practice, FRET is most effectively measured by emission ratioing. Emission ratioing monitors the change in the ratio of emission by the acceptor over emission by the donor. An increase in this ratio signifies that energy is being transferred from donor to acceptor and thus that FRET is occurring. The specific FRET pair of acGFP1, and mCherry were selected due to their successful previous application in a continous, and quantifiable, glucose biosensor.[3] AcGFP1 and mCherry possess a strong fluorescent quantum yield, appropriate overlapping in their excitation spectra, as well they demonstrate resistance to photo bleaching, which enables continuous monitoring.[3]
</P></DIV><DIV class="column full_size" style="width:70%;margin-left:15%"><H3 style="margin-left:15%">Approach 2: Inteins</H3><P style="width:70%;margin-left:15%;font-size:18pt">Found in organisms from all domains, inteins (intervening proteins) are auto-processing proteins that function both in endogenous and exogenous contexts [4]. These proteins are involved in the cleavage and formation of peptide bonds during a unique process where they excise themselves from a polypeptide and ligate the flanking extein (external protein) [4]. This spontaneous splicing process occurs post-translationally and is most commonly observed in proteins involved in DNA transcription, replication and maintenance processes within a cell [4]. Inteins have such great potential as biosensors because of their rapidness. The induced splicing and joining of the exteins to create a functional extein protein is much quicker than the typical transcription and translation process.
</P></DIV><DIV class="column full_size" style="width:70%;margin-left:15%"><H3 style="margin-left:15%">Small Molecule Triggered Intein Splicing</H3><P style="width:70%;margin-left:15%;font-size:18pt">The <EM>mycobacterium tuberculosis</EM> RecA intein was selected for use as it has been shown to splice in a wide variety of protein contexts [5]. Small molecule triggered intein splicing allows the production of a “molecular switch” which is only activated in the presence of the designated ligand. To function, this system requires that the intein is able to bind with a high affinity to its specific ligand, and that the resulting conformational change initiates the process of protein splicing [5]. The above graphic demonstrates a scheme for directed evolution of small-molecule triggered intein splicing, applied to the estrogen receptor [6]. We will utilize the same system, but instead of the estrogen receptor ligand binding domain, we will attempt to evolve the glucocorticoid receptor to induce splicing upon binding to cortisol. For the initial application of our biosensor system, we will be using the binding of cortisol to the human glucocorticoid receptor as the initiating reaction that triggers intein splicing.</P></DIV><DIV class="column full_size" style="width:70%;margin-left:15%"><H3 style="margin-left:15%">Customizable Biosensors</H3><P style="width:70%;margin-left:15%;font-size:18pt"> The ability to interchange nuclear receptor in our modular intein system, results in the potential to detect hundreds of biological molecules. A few examples include, estrogen, retinoic acid, and cortisol. The ability to interchange the extein spliced together upon ligand binding, results in the capability to produce a signal in a variety of forms. A few examples include, luminescence, growth in antibiotic media, and florescence. We decided to start by constructing the 4-Hydroxytamoxifen dependent estrogen receptor intein construct, developed by Buskirk et al. (2004). We chose to began with the construct in the Kanamcyin Resistance context, so that we could characterize the ability of the small-molecule triggered intein splicing system in bacteria, when it had been initially demonstrated in yeast [5]. We would simultaneously construct a novel glucocorticoid receptor intein in the Kanamycin Resistance context, so that we could select bacteria which were capable of successfully performing small-molecule triggered intein splicing. If no colonies formed, we planned to perform directed evolution until the system was capable of producing a splicing event.
</P></DIV><DIV class="column full_size" style="width:70%;margin-left:15%"><H3 style="margin-left:15%">Split NanoLuc Luciferase Domain</H3><P style="width:70%;margin-left:15%;font-size:18pt">Upon producing cortisol-dependent intein splicing, we can then swap out the flanking extein sequence with a quantitative reporter. We therefore chose  NanoLuc® Luciferase as the extein and quantitative reporter for our construct. NanoLuc® by Promega is a luciferase derived via directed evolution from the luminous shrimp, Oplophorus gracilirostris [3]. The enzyme was obtained from deep-sea shrimp and optimized following the discovery of a novel substrate, furimazine, which allows for the production of visible light with less background activity than other luciferases [7,8]. NanoLuc® is a 19.1 kDa monomeric protein that is both soluble and ATP-independent [7]. Compared to firefly (Lampyridae) and sea pansy (Renilla) luciferases, this novel protein offers many advantages reflected by its increased stability, smaller size, and &gt;150-fold increase in luminescence [8]. The unique characteristics of this enzyme construct combined with its high luminescence activity allow for the production of a very sensitive diagnostic assay. The split site for NanoLuc® Luciferase was chosen between the amino acids 52 and 53, as this split was previously determined to produce the highest luminescence upon successful rejoining of the N and C termini [9]. 


</P><H3 style="margin-left:15%">Comparison of the two Biosensor</H3><P style="width:70%;margin-left:15%;font-size:18pt"> Each biosensor has its specific advantages. The FRET biosensor is reagent-less, continuous, small, but not portable, as it requires a fluorometer or a confocal laser imaging microscope. The FRET biosensor could best be used in an offboard system for detection of hormones. It also posses potential to be used in pharmaceutical research to detect agonists or antagonists of nuclear receptors. Because the system is reagent-less, and not used up in the detection process, the biosensor could potentially be used to detect hormone in one sample, be washed, and then used to test another sample.

The intein based system is small, portable, but undergoes a irreversible reaction, and requires a luciferin substrate to produce a signal. Therefore, the intein system could theoretically be employed in our diagnostic pacifier to quantify salivary hormones wirelessly. However, the internal detection will have to be replaced as the intein is spliced, and the luciferin substrate is expended.</P><H3 style="width:70%;margin-left:15%">Application of the Biosensors in a Diagnostic Pacifier</H3><P style="width:70%;margin-left:15%; font-size: 18pt"> Complimentary and simultaneous to our work in the laboratory, has been our work on a computer-aided design pacifier which has the ability to passively collect saliva from the users mouth. The FRET based system can be utilized for salivary hormone quantification through an off-board system. Saliva samples would be passively collected, then cortisol present would interact with internally housed FRET biosensors glucocorticoid ligand binding domain, while the individual uses the pacifier. After sufficient saliva has been collected, the sample can be taken out of the pacifier, and tested with a fluorometer, or confocal microscope to quantify FRET emission and intensity indicative of cortisol concentration.

The intein biosensor design would work through our engineered protein construct binding to salivary cortisol, at its glucocorticoid ligand binding domain, resulting in a conformational change in the receptors structure. The confrontational change brings the terminals of the intein in close proximity such that a splicing event can occur, linking together the halves of the split NanoLuc, producing luminescence. The resulting luminescent signal would be detected by our specialized built-in luminometer and the resulting signal is transmitted to a smartphone application. Therefore, presenting a simple and portable method of analyzing salivary hormones.</P></DIV><DIV class="column half_size" style="width:70%;margin-left:15%"><H4>References</H4><OL id="noIndent"><LI>
T. Kino, Glucocorticoid Receptor. MDText.com, Inc., 2000.</LI><LI>H. Reyer, S. Ponsuksili, E. Kanitz, R. Pöhland, K. Wimmers, and E. Murani, “A Natural Mutation in Helix 5 of the Ligand Binding Domain of Glucocorticoid Receptor Enhances Receptor-Ligand Interaction,” PLoS One, vol. 11, no. 10, p. e0164628, Oct. 2016.</LI><LI>Veetil JV, Jin S, Ye K. A Glucose Sensor Protein for Continuous Glucose Monitoring. Biosensors &amp; bioelectronics. 2010;26(4):1650-1655. doi:10.1016/j.bios.2010.08.052.</LI><LI>N. H. Shah and T. W. Muir, “Inteins: Nature’s Gift to Protein Chemists., Chem. Sci., vol. 5, no. 1, pp. 446–461, 2014.</LI><LI>A. R. Buskirk, Y.-C. Ong, Z. J. Gartner, and D. R. Liu,&quot; “Directed evolution of ligand dependence: small-molecule-activated protein splicing., Proc. Natl. Acad. Sci. U. S. A., vol. 101, no. 29, pp. 10505–10, Jul. 2004.</LI><LI>Peck SH, Chen I, Liu DR. Directed Evolution of a Small Molecule-Triggered Intein with Improved Splicing Properties in Mammalian Cells. Chemistry &amp; biology. 2011;18(5):619-630. doi:10.1016/j.chembiol.2011.02.014.</LI><LI>C. G. England, E. B. Ehlerding, and W. Cai, “NanoLuc: A Small Luciferase Is Brightening Up the Field of Bioluminescence., Bioconjug. Chem., vol. 27, no. 5, pp. 1175–1187, 2016.&quot;</LI><LI>. Boute, P. Lowe, S. Berger, M. Malissard, A. Robert, and M. Tesar, “NanoLuc Luciferase - A Multifunctional Tool for High Throughput Antibody Screening., Front. Pharmacol., vol. 7, p. 27, 2016.</LI><LI>L. G. G. C. Verhoef, M. Mattioli, F. Ricci, Y.-C. Li, and M. Wade, “Multiplex detection of protein–protein interactions using a next generation luciferase reporter,” Biochim. Biophys. Acta - Mol. Cell Res., vol. 1863, no. 2, pp. 284–292, Feb. 2016. </LI></OL></DIV><FOOTER style="background-color: #212121;height:90px;"><DIV class="container"><DIV class="row"><DIV class="col-md-4" style="margin-top:5px"><UL class="list-inline quicklinks"><LI><A href="https://igem.org/Main_Page" style="color:white; font-size:16pt">iGEM Headquarters</A></LI></UL></DIV><DIV class="col-md-4" style="margin-top:5px"><UL class="list-inline quicklinks" style="color: white;font-size:16pt;"><LI><A href="http://www.queensu.ca/" style="color:white;margin-left:100px;">Queen's University</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>