"
Page
Discussion
View source
History
teams
Log in
 
Team:BostonU/NotebookQS
From 2013.igem.org
Our Team
Team
Summer Fun
Project
Project Overview and Abstract
MoClo DNA Assembly
Quorum Sensing
Datasheet App
Clotho
Results Summary
Achievements
Data Collected
Parts Submitted
MoClo Kit
Medal Fulfillment
Notebook
Protocols
Characterization Notebook
Quorum Sensing Notebook
Software Notebook
Troubleshooting
Considerations
New England iGEM Regional Meeting
Human Practices and Safety Forms
Acknowledgements
Collaborations
Acknowledgements
Quorum Sensing Notebook
Goal:
To show feasibility of creating a working LuxR and LuxI homologue in E. coli from Chromobacterium violaceum.
Create MoClo parts for CviI and CviR completed
Clone pVioA as a MoClo part completed
Build transcriptional units for a quorum sensing device in progress
Show orthogonality with LuxI/LuxR system in progress
June
June 2013
We planned out that we will transform E. coli with the CviI/CviR system found in Chromobacterium violaceum. We will design MoClo devices that include these systems. In the meantime, we are beginning to design primers that will make the CviI/CviR system into a MoClo system by adding the the fusion site in addition to the BBS and SpeI sites. Then, we will develop a PCR strategy.
June 18, 2013
Our advisor, Traci, emailed different labs and PIs that work with Chromobacterium violaceum.  Dr. Jim Thoden from the Holden Lab at the University of Wisconsin sent us genomic DNA.
June 20, 2013
We designed forward and reverse primers for CviR/CviI systems and ordered those primers.  We are currently waiting on acquiring the primers to begin PCR.
July
July 8, 2013
Today, we did Phusion PCR using the genomic DNA of Chomobacterium violaceum as the template for the CviI and CviR systems.  We ran the PCR product on a gel and the CviI did not appear on the gel.  The CviR system appeared and we gel extracted the parts.
The gel concentrations are as follows:
July 9, 2013
We redid the PCR for CviI.  We ran the PCR product on a gel and we did the gel extraction.  we quantified the concentration on the NanoDrop.
We ran a Level 0 MoClo reaction with the higher concentration of CviI and CviR.
July 10, 2013
We did a transformation using Bioline Gold E. coli and plated the samples on IPTG + Xgal.
July 11, 2013
We pulled the plates and picked three colonies and plated them on a stab plate and set up liquid cultures for incubation.
July 12, 2013
We set up CviI_CD and CviR_CD for sequencing in triplicate.
July 13, 2013
We analyzed the sequences for CviI and CviR and all of the samples we sent in matched the expected sequence.
July 17, 2013
We made glycerols and archived the highest concentration of each CviI and CviR.
July 22, 2013
Traci ordered the promoter pVioA_AB from IDT.  It should come in from 4-10 business days.  We are to treat it as PCR product and do L0 MoClo reaction.  We can build the following four Level 1 devices and combine them together for a Level 2 device.
July 25, 2013
We are still waiting for the pVioA promoter.  In the meantime, we are making I13453_GB (pBAD)and sending four samples out for sequencing. Here is an image of its transformation.
July 26, 2013
One of the I13453_GB samples were a match so we can make glycerol stocks and miniprep plasmid and archive the parts in our registry.
July 30, 2013
We ran Level 1 MoClo reactions for:
E-J23100_EB-BCD2_BC-CviR_CD-B0015_DF-F
F-I13453_FB-BCD2_BC-CviI_CD-B0015_DG-G
G-I13453_GB-BCD2_BC-XFP_CD-B0015_DH-H
July 31, 2013
Today we transformed yesterday’s reactions using Bioline competent cells and plated the transformations on IPTG and Xgal for blue white screening.
August
August 1, 2013
We picked white colonies for overnight cultures.
August 2, 2013
The cultures were miniprepped and sent for sequencing but they were unfortunately wrong.
August 19, 2013
pVioA finally arrived so we can create four transcriptional units using MoClo. First we had to make the Level 0 part for pVioA promoter in the AB an EB format.
August 21, 2013
With the sequence of pVioA_AB and pVioA_EB verified, we set up the level 1 reaction for:
pVioA_AB+BCD2_BC+E1010_CD+B0015-DE+DVL1_AE
J23100_EB+BCD2_BC+CviR_CD+B0015_DF+DVL1_EF
I13453_FB+BCD2_BC+CviI_CD+B0015_DG+DVL1_FG
I13453_GB+BCD2_BC+E0040_CD+B0015_DH+DVL1_GH
August 23, 2013
After sequencing, we seem to have had a mix up of promoter and CDS. CviR is paired with I13453 instead of J23100, while CviI is paired with J23100 instead of I13453. Other lab mates are redoing the reactions today to ensure no other errors occur.
August 25, 2013
The Level 1 reactions were redone, checked on a gel, and sent for sequencing because last week's reactions were unsuccessful.
August 26, 2013
We did the sequence analysis for the Level 1 parts and set up glycerol stocks.
We realized that we will need a fifth Level 1 reaction to incorporate arabinose as a small molecule.  We will do this using the araC CDS.  We also decided that we are going to change the J23100 promoter to J23104.  The reactions are now as follows:
QS1 = pVioA_AB+BCD2_BC+E1010_CD+B0015_DE+DVL1_AE
QS2 = J23104_EB+BCD2_BC+CviR_CD+B0015_DF+DVL1_EF
QS3 = I13453_FB+BCD2_BC+CviI_CD+B0015_DG+DVL1_FG
QS4 = I13453_GB+BCD2_BC+E0040_CD+B0015_DH+DVL1_GH
QS5 = J23104_HB+BCD2_BC+C0080_CD+B0015_DJ+DVL1_HJ
QS6 = QS1+QS2+QS3+QS4+QS5
Also did PCR and gel extraction for C0061_CD, C0062_CD and R0062_CD. They are parts necessary for the LuxI/R quorum sensing that is theoretically orthogonal to the C. violaceum system being put together.
September 1, 2013
We analyzed the sequences for C0061_CD and C0062_CD.  C0062_CD was confirmed but the C0061_CD sequence included only one of the PCR product inserts.
Retrieved from " http://2013.igem.org/Team:BostonU/NotebookQS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
