"
Page
Discussion
View source
History
teams
Log in
Â 
Team:BostonU/QS
From 2013.igem.org
Our Team
Team
Summer Fun
Project
Project Overview and Abstract
MoClo DNA Assembly
Quorum Sensing
Datasheet App
Clotho
Results Summary
Achievements
Data Collected
Parts Submitted
MoClo Kit
Medal Fulfillment
Notebook
Protocols
Characterization Notebook
Quorum Sensing Notebook
Software Notebook
Troubleshooting
Considerations
New England iGEM Regional Meeting
Human Practices and Safety Forms
Acknowledgements
Collaborations
Acknowledgements
Quorum Sensing
Quorum sensing is a system that controls population density using stimulus and response. These types of systems can be commandeered for use in synthetic biology and re-engineering to control gene expression in genetic circuits that are implanted in bacteria. The bacteria utilize quorum sensing to coordinate gene expression.  To do so, the bacteria produce and secrete signaling molecules.  These bacteria have receptors that detect the signaling molecule.  When the signaling molecule binds to the receptor, it induces gene expression.  The presence of one or more proteins will induce a promoter that controls the production of signal proteins, such as fluorescent proteins, or of other functional genes to link synthetic genetic circuits together.
The most commonly used quorum sensing systems in synthetic biology are the acyl-homoserine lactone(AHL) based systems. Below is a figure from Fuqua and Greenberg, (2002) that shows how the LuxR/LuxI system functions. These genes are originally from Vibrio fischeri but have been successfully utilized in E. coli based synthetic circuits, the most well known arguably being the edge detecting system published by Tabor et. al, (2009) .
A LuxI/LuxR-type quorum sensing system has been detected in Chromobacterium violaceum (Stauff et al., 2011) , where CviR is homologous to LuxR and CviI is homologous to LuxI. The goal is to introduce a new AHL quorum sensing system to synthetic biology via MoClo. Chromobacterium violaceum is a gram-negative bacteria found in flora from water and soil in tropical and subtropical regions and uses the CviR/I system as a means of population control, just like V. fischeri uses the LuxR/LuxI system. The promoter containing the binding site for CviR is pVioA (Stauff et al., 2011) .
Circuit Design
We've designed the circuit below to test the CviI/CviR system using the MoClo assembly method. This will require us to generate three new Level 0 Parts (CviI, CviR, and pVioA). We have obtained Chromobacterium violaceum genomic DNA from the Holden Lab at the University of Wisconsin and will utilize PCR to amplify these parts. This also requires a fifth transcriptional unit, which will expand the library beyond the 4-TU system we're currently working with. We will need to generate new destination vectors prior to making the Level 0 parts for the promoter and terminator as shown in the circuit design below.
New Level 0 Parts
We have successfully made three new parts while working on the QS project. CviI and CviR were submitted to the Registry as Level 0 MoClo Parts. The pVioA promoter was confirmed after the DNA submission deadline and was not included in our submission. All three parts are now sequence verified.
We have created one new destination vector but are still working on the second destination vector for the fifth transcriptional unit. We hope to have these vectors confirmed soon and will continue working on creating the final transcriptional unit for our design.
Our next steps involve testing the CviI/CviR genes for function and checking them for cross-reactivity with LuxI/LuxR.
pVioA as a Constitutive Promoter
We have successfully characterized the pVioA promoter as a constitutive promoter controlling RFP expression (first transcriptional unit in diagram above). For more information on how we ran our characterization experiment, please check out our Data Collected page.
The graphs below show MEFL converted RFP measurements (done using the TASBE tools). The data is binned by fluorescence intensity and each bin also includes the cell count. The bottom graph shows the same data as the top one above but it is ordered by cell count, from highest (dark purple) to lowest (pale purple).
We can now see that within the population of cells, the smallest binned group shows the highest level of expression. This may be due to higher copy counts of plasmids within that subpopulation of cells.
We plan to continue with this work and hope to have an update on our progress at the Jamboree!
References
[1] Stauff, D.L., and Bassler, B.L. (2011) "Quorum Sensing in Chromobacterium violaceum: DNA Recognition and Gene Regulation by the CviR Receptor." Journal of Bacteriology 193(15):3871-3878. doi: 10.1128/JB.05125-11.
[2] Fuqua, C., and Greenberg, E.P. (2002) "Listening in on Bacteria: Acyl-Homoserine Lactone Signalling." Nature Reviews Molecular Cell Biology 3, 685-695. doi:10.1038/nrm907.
[3] Tabor, J.J., Salis, H.M., Simpson, Z.B., Chevalier, A.A., Levskaya, A., Marcotte, E.M., Voigt, C.A., and Ellington, A.D. (2009) "A Synthetic Genetic Edge Detection Program." Cell 137 (7) 1272-1281. doi:10.1016/j.cell.2009.04.048.
Retrieved from " http://2013.igem.org/Team:BostonU/QS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
