Team:SJTU-BioX-Shanghai/Improve
HOME
PROJECT
Description
Design
Results
Demonstration
Loader
EXPERIMENT
Parts
Notebook
Protocol
Improvement
Interlab
MODEL
Overview
Adhesion
Ultrasonography
Gene Circuit
PRACTICES
Background
Integrated
Engagement
Biosafety
Interview
Price Model
Application
TEAM
Members
Attributions
Collaborations
AWARDS
Usage and Biology
Experiment design
Protein structure
Results
Reference
Improve
Usage and Biology
It’s an improvement of OmpA(BBa_K1489002 ). OmpA is widely utilized to anchor otherwise soluble proteins in the outer cell membrane of bacteria like E.coli. Thus, OmpA functions as a carrier that carry different protein or domain to the surface of bacterial cells.
Considering more and more teams are focusing on cancer targeting, we added a 15-AA oligo peptide to the C-terminal of OmpA to give it the ability to target cancer. The oligo peptide has the ability to bind to Thomsen–Friedenreich antigen (T antigen) that exist on some kinds of cancer cells(like colorectal cancer) by a mechanism similar to antigen-antibody binding reaction. On the other hand, the oligo peptide is so short that its influence on the function of OmpA is limited. That is, when OmpA is carrying other peptides or protein domains at the C terminal, there is little chance that the peptide will cause adverse effects such as misfolding.
Experiment design
Experiments have been conducted by Team SJTU-BioX-Shanghai. We used cell-climbing cover glasses as the solid basement on which the cell-bacteria adhesion experiment was carried out. Cell line HT29(human colorectal cancer cell) was chosen for the experiment, which grew over the glasses. We certificate the expression of T antigen by FACS previously. After cell HT29 grew and covered the glasses， we mixed it with E.coli BL21(DE3) cultures which expressed OmpA-peptide fusion protein (egfp was expressed as reporter gene ). The mixture glasses were incubated at 4°C for hours(results of 12h are shown) after which cells were cleaned with normal saline to wash away unconjugated bacteria. These cover glasses were carefully observed via the fluorescent microscope.
Protein structure
We use Swiss Model to predict the structure of our OmpA-peptide fusion protein. The result shows us that the oligo peptide don’t influence the correct folding of the β-barrel of OmpA.
Fig 1. 3D structure of OmpA - peptide T1
Results
Reference
[1] Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R., ... & Lepore, R. (2018). SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic acids research.
[2] Guex, N., Peitsch, M. C., & Schwede, T. (2009). Automated comparative protein structure modeling with SWISS‐MODEL and Swiss‐PdbViewer: A historical perspective. Electrophoresis, 30(S1), S162-S173.
[3] Bienert, S., Waterhouse, A., de Beer, T. A., Tauriello, G., Studer, G., Bordoli, L., & Schwede, T. (2016). The SWISS-MODEL Repository—new features and functionality. Nucleic acids research, 45(D1), D313-D319.
[4] Benkert, P., Biasini, M., & Schwede, T. (2010). Toward the estimation of the absolute quality of individual protein structure models. Bioinformatics, 27(3), 343-350.
[5] Bertoni, M., Kiefer, F., Biasini, M., Bordoli, L., & Schwede, T. (2017). Modeling protein quaternary structure of homo-and hetero-oligomers beyond binary interactions by homology. Scientific reports, 7(1), 10480.
Copyright © 2018 SJTU-BioX-Shanghai. All rights reserved.
