<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Protocol skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SJTU-BioX-Shanghai/Protocol</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_bar_pc"><DIV class="menu_item_container"><UL id="nav--desktop" class="menu_item_list"><LI class="menu_item"><A title="HOME" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai">
                    HOME
                </A></LI><LI class="menu_item dropdown_main_menu"><SPAN class="nav_font" valign="middle">PROJECT</SPAN></LI><LI class="submenu_item"><A title="Description" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Description">
                        Description
                        </A></LI><LI class="submenu_item"><A title="Design" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Design">
                        Design    
                        </A></LI><LI class="submenu_item"><A title="Results" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Results">
                        Results
                        </A></LI><LI class="submenu_item"><A title="Demonstrate" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Demonstrate">
                        Demonstration
                        </A></LI><LI class="submenu_item"><A title="Demonstrate" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Loader">
                        Loader
                        </A></LI></UL><LI class="menu_item dropdown_main_menu"><SPAN class="nav_font">EXPERIMENT</SPAN></LI><LI class="submenu_item"><A title="Parts" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Parts">
                        Parts
                        </A></LI><LI class="submenu_item"><A title="Notebook" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Notebook">
                        Notebook
                        </A></LI><LI class="submenu_item"><A title="Protocol" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Protocol">
                        Protocol
                        </A></LI><LI class="submenu_item"><A title="Improve" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Improve">
                        Improvement
                        </A></LI><LI class="submenu_item"><A title="Interlab" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/InterLab">
                        Interlab
                        </A></LI><LI class="menu_item dropdown_main_menu"><SPAN class="nav_font">MODEL</SPAN></LI><LI class="submenu_item"><A title="Overview" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Model">
                        Overview
                        </A></LI><LI class="submenu_item"><A title="Adhesion" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Adhesion">
                        Adhesion
                        </A></LI><LI class="submenu_item"><A title="Ultrasonic Imaging" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Ultrasonic_Imaging">
                        Ultrasonography
                        </A></LI><LI class="submenu_item"><A title="Gene Circuit" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Gene_Circuit">
                        Gene Circuit
                        </A></LI><LI class="menu_item dropdown_main_menu"><SPAN class="nav_font">PRACTICES</SPAN></LI><LI class="submenu_item"><A title="Background" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Background">
                        Background
                        </A></LI><LI class="submenu_item"><A title="Integrated Human Practice" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Human_Practices">
                        Integrated
                        </A></LI><LI class="submenu_item"><A title="Education &amp; Public engagement" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Public_Engagement">
                        Engagement
                        </A></LI><LI class="submenu_item"><A title="Biosafety" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Safety">
                        Biosafety
                        </A></LI><LI class="submenu_item"><A title="Interview" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Interview">
                        Interview
                        </A></LI><LI class="submenu_item"><A title="Price Model" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Price_Model">
                        Price Model
                        </A></LI><LI class="submenu_item"><A title="Application" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Application">
                        Application
                        </A></LI><LI class="menu_item dropdown_main_menu"><SPAN class="nav_font">TEAM</SPAN></LI><LI class="submenu_item"><A title="Members" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Team">
                        Members
                        </A></LI><LI class="submenu_item"><A title="Attributions" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Attributions">
                        Attributions
                        </A></LI><LI class="submenu_item"><A title="Collaborations" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Collaborations">
                        Collaborations
                        </A></LI><LI class="menu_item"><A title="Awards" href="https://2018.igem.org/Team:SJTU-BioX-Shanghai/Awards">
                AWARDS
                </A></LI></DIV></DIV><DIV class="content"><DIV class="content_text"><DIV class="text"><H1>Protocol</H1><DIV class="storage_box"><UL class="storage_box_list"><LI class="storage_box_ele"><DIV class="storage_header"><P>Molecular Experiment</P></DIV><DIV class="storage_text"><P>PCR：
Materials:
10×Buffer
MgSO4 solution
dNTPs
Template
F/R-primer
DNA polymerase
Methods:
1.Combination of all reagents:
Name of reagent                volume (μl)
10×Buffer                            5
MgSO4  solution                     3
Template               20 ng for plasmid or 200 ng for genome
F/R-primer(10μmol)                  1.5 for each
DNA polymerase                      1
dNTPs                               5   
Sterile Water            make reaction system up to 50μ in total

Set the PCR machine to run the following steps:
Stage             Temperature (℃)            Time                 
initial denaturation     98                    2 min                      96 (denaturation)      15 seconds
30 cycles         40-65 (around Tm  annealing) 30 seconds                68 (extension)        30 seconds per 1000bp
Final extension           68                     7 min
Incubate                12                   Infinite
Restriction Digestion
Materials:
Restriction Enzyme: Thermo Fisher FastDigest Restriction Enzyme
10×FD Green Buffer
DNA samples
Sterile Water

Methods:
Note: the components listed below are for double digestion!
Component	volume (μl)
10×FD Green Buffer	2
DNA sample	600 ng for plasmid or up to 200 ng for PCR production
Restriction Enzyme    1 for each
Sterile Water          make reaction up to 20μl in total
	
	

Set up the reaction following the above table and incubate at 37℃ for 15-30 min and do the gel electrophoresis or purification later
Ligation：
Materials:
T4 DNA Ligase
5×T4 DNA Ligase Buffer
Vector Plasmid
Insert DNA
Sterile Water

Methods:
Component	volume or mass
5×T4 DNA Ligase Buffer 	4μl
Vector plasmid	50ng
Insert DNA	require molar ratio of 3:1 to 7:1 (insert:vector)
T4 DNA Ligase	0.4 μl
Sterile Water	make reaction system up to 20μl in total
Make the reaction system and incubate at room temperature for 4-6 hours.
Note: for how to calculate volumes of vector and insert DNA, we highly recommend NEBbioCalculator
DNA Gel Electrophoresis
Materials:
Agarose
TBE Buffer
Gel board
EB
DNA×10 or×6 Loading Buffer
Electrophoresis tank
sample
marker 2000 or 15000
Methods:
1.Make a 1.0%-1.5% Agarose gel according to the volume of your gel board (eg. add 0.45g Agarose in 30ml TBE if choose a concentration of 1.5%)
2.Heat the container with the mixture until it is complete dissolved. When the solution starts boiling, stop heating, shake the container until materials mix well and heat again until the mixture become clear
3.Wait until the solution cools down to 50℃,add 0.01% EB (eg. 3μin 30ml) or non-poisonous dyes instead
4.Pour the solution into a gel board and insert the comb
5.Set the gel at room temperature and wait until it completely solidifies
6.Remove the comb and transfer the gel to the electrophoresis tank
7.Add samples into pores with 40% DNA Loading Buffer (eg. 2μLoading Buffer in 5μDNA sample)
8.Run the gel for 20 min at 100V, 100mA
Transformation
Materials:
Competent cells
LB broth
Ice
Selection plates with corresponding antibiotics.
Methods:
1.Put 50µL competent E. coli cells on ice for 10 minutes
2.Add 20 µl DNA from a ligation reaction mix or 10-100 ng plasmid 
3.Incubate the mixture on ice for 30 minutes
4.Heat shock at exactly 42°C for exactly 45 seconds（DH5α），90 seconds (BL21(DE3))
5.Place on ice for 3 minutes
6.add the mixture into 500 µL LB broth
7.Incubate at 37°C and 200-250 rpm for one hour
8.For ligation reaction DNA: 500 µl of each transformation reaction onto a selection plate. For known plasmid: 100-200 µL of each transformation reaction onto a selection plate 
9.Incubate overnight at 37°C with plates upside down.
Purification and Gel Extraction &amp; Plasmid Extraction:
Purification of PCR or restriction products was performed according to the OMEGA Cycle Pure Kit D6492.
Gel Extraction was performed according to the OMEGA Gel Extraction Kit D2500.
Plasmid Extraction was performed according to the BIOMIGA Plasmid Miniprep Kit.
Induction:
Materials:
1M IPTG stock solution.
LB broth
Antibiotics
Plates grew over night

Methods:
1.	Pick clones from the plate into 5ml LB broth
2.	Incubate overnight
3.	Transfer some bacterial culture into new LB broth(1:100-1:5) and incubate at 37℃ until O.D.600 reach 0.6.
4.	Add appropriate amount of IPTG solution(final concentration 0.5-1mM)
5.	Incubate at 30℃ for hours.(40h is needed for ARGs)</P></DIV></LI><LI class="storage_box_ele"><DIV class="storage_header"><P>Western Blotting protocol</P></DIV><DIV class="storage_text"><P><STRONG>1、Prepare of Proteins</STRONG>
1.1 Culture <I>E.coli</I> BL21(DE3) with LB(l) medium overnight at 37℃, 200rpm
1.2 Add 1ml overnight culture into 5ml new LB(l) medium for 1 hour at 37℃, 200rpm
1.3 Induce culture by IPTG (with final concentration of 0.5mM) more than 4 hours, while the OD600 is 0.6 
1.4 (Lysis) Add 1.5ml lysozyme (1mg/ml) into 2ml centrifuged and the supernatant discadred strain, 30℃ for 30 minutes
1.5 (Degeneration) Add SDS loading buffer and put the centrifugal tube into boiled water for 10 minutes 
2、<STRONG>Prepare SDS-PAGE gel, Running buffer, Trans buffer and TBST Washing buffer</STRONG><STRONG>SDS-PAGE:</STRONG>
12% Separation Gel(the lower gel):
30%Acr-Bis3.2ml
1.5M Tris-HCl(pH=8.8)2ml
10%SDS 80μl
10%AP 80μl
TEMED3μl
H2O2.64ml
Total8ml
5% Spacer Gel(the upper gel):
30%Acr-Bis500μl
1.5M Tris-HCl(pH=6.8)500μl
10%SDS 40μl
10%AP30μl
TEMED3μl
H2O2ml
Total3ml<STRONG>1×Running buffer:</STRONG>
Tris3.03g
Glycine14.4g
10%SDS10ml
Add ultra pure water to 1L 
Store at 4℃ <STRONG>1×Trans buffer:</STRONG>
Tris3.03g
Glycine14.4g
10%SDS  10ml
Methanol  200ml
Add ultra pure water to 1L
Store at 4℃<STRONG>1×TBS buffer:</STRONG>
Tris      2.4g
NaCl    8.7g
Adjust pH=7.6 with HCl 
Add ultra pure water to 1L <STRONG>TBST Washing buffer:</STRONG>
TBS      500ml
Tween-20  250μl<STRONG>Blocking buffer:</STRONG>
Add 2.5g no-fat dry milk into 50ml TBST Washing buffer
Prepare Blocking buffer only when using it<STRONG>3、Protein transfer and Immunodetection</STRONG>
3.1 Add 15μl sample and 5μl Protein marker 
3.2 Add Running buffer for electrophoresis with a constant voltage at 100V for about 25 minutes, then change voltage into 200V for 40 minutes (adjust the former time by the marker)
3.3 Open the package and cut the upper spacer gel
3.4 Prepare the same size PVDF membrane and put into Trans buffer for 5 minutes before trans-membrane
3.5 Open gel cassettes and put the gel on the PVDF membrane sandwiched between two pieces of blotting paper(always filter paper)<STRONG>Mind the order of gel and PVDF membrane between the electrodes, black end with gel for negative electrode and white end with PVDF membrane for positive electrode.</STRONG>
3.6 Add Trans buffer for electrophoresis with a constant current at 350mA for 65 minutes(Crowd with ice at a 4℃ temperature )
3.7 After trans-membrane, wash PVDF membrane with 10ml TBST for 10 minutes with gentle shaking
3.8 Add 10ml Blocking buffer with gentle shaking for 1 hour at room temperature or 4℃ overnight
3.9 Wash PVDF membrane with 10ml TBST for 10 minutes with gentle shaking
3.10 Add 10ml diluted(1:5000) primary antibody for 2 hours at room temperature or 4℃ overnight with gentle shaking (primary antibody can be used repeatedly for 10 times)
3.11 repeat step 3.9
3.12 Add 10ml diluted secondary conjugated antibody for 2 hours at room temperature or 4℃ overnight with gentle shaking (seconfary antibody needn’t be repeatedly used)
3.13 repeat step 3.9
3.14 With the ratio of 1:1 to mix Luminous substrate A and B, typically 200-400μl mixture added on PVDF membrane
3.15 Take photos by specific machine and store it</P></DIV></LI><LI class="storage_box_ele"><DIV class="storage_header"><P>CRC induction</P></DIV><DIV class="storage_text"><P><STRONG>Subject: </STRONG>C57BL/6 mice strain (6~8 week old)<STRONG>Material:</STRONG> Azoxymethane (AOM) solution (1mg/ml)Dextran Sulfate Sodium Salt (DSS) solution (2.0% &amp; 2.5%)<STRONG>Experiment aim:</STRONG>After analyzing the qualities of our device that requires testifying meanwhile thoroughly considering the 3Rs principle, we have reduced the need of animal experiments to the least, but for certain qualities the testifying procedure is difficult to replace, refine and reduce experimental animals use. As a result the colorectal cancer mouse model was induced in the minimum amount just to be adequate for product testifying procedure.<STRONG>Experiment design:</STRONG>Azoxymethane (AOM) is a chemical agent that can initiate cancer by alkylation of DNA, there by facilitating base mispairings, while Dextran Sulfate Sodium Salt (DSS) is a substance that causes inflammations that will eventually lead to enteritidis.After being injected into the subject, AOM introduces a carcinogenic environment in vivo, along with DSS consumed, enteritidis is likely to occur and soon develop into colorectal cancer.<STRONG>Experimental procedure:</STRONG>After the mice strain stabilizes and being weighed, intraperitoneal injection of AOM solution is given in the amount of 10mg/kg and let rest for a week.With the AOM injection completed, the DSS induction process begins. One induction cycle lasts for 3 weeks while the complete induction process includes 3 cycles of which. In the first week of the three-week cycle the DSS solution (2.0% &amp; 2.5%) was given as drinking water to our subject. DSS solution was mixed right before it was given to subject to ensure the best results. After one week of consuming DSS, the drinking water for the next two weeks was switched to normal clear water. Repeat the cycle two more times while carefully monitor the weight change of the subject.<STRONG>Citation:</STRONG>[1]Ameet I. Thaker, Anisa Shaker, M. Suprada Rao, Matthew A. Ciorba,Modeling Colitis-Associated Cancer with Azoxymethane (AOM) and Dextran Sulfate Sodium (DSS) ,Journal of Visualized Experiments,9/11/2012[2]Clemens Neufert, Christoph Becker &amp; Markus F Neurath,An inducible mouse model of colon carcinogenesis for the analysis of sporadic and inﬂammation-driven tumor progression,NatureProtocol, 9/08/2007</P></DIV></LI><LI class="storage_box_ele"><DIV class="storage_header"><P>Structure observation of CRC models—paraffin sectioning and HE staining</P></DIV><DIV class="storage_text"><P><STRONG>Subject:</STRONG> Colorectal tissue from CRC induced mouse model and normal ones<STRONG>Material: </STRONG>PBS, different concentration of ethanol (50%, 75%, 85%, 95%, 100%), xylene, paraffin wax, haematoxylin and eosin (HE stain)<STRONG>Experiment aim: </STRONG>Before using our CRC mouse model for any further experiment, it is essential to first testify the existence of the disease. The exquisite imaging of the cancerous area is also critical in modeling as well as device improvement.<STRONG>Experiment design:</STRONG>To maintain the original structure of the tissue, the paraffin sectioning was carry out. After receiving a clear and complete biopsy of the sample, haematoxylin and eosin (HE) stain was performed to get the best vision of the histological structure.<STRONG>Experiment procedure:</STRONG>First, a section of colon or rectum was removed from the subjects and longitudinally cut open to form a rectangular piece of tissue. The tissue was then placed on a small cardboard and soaked in paraformaldehyde (PFA) solution overnight. Then the tissue was washed with phosphate buffered saline solution (PBS) three times (10minutes each time). With the tissue cleared from PFA, a series of soaking process was done on decoloring shaker. The soaking process was shown in table below for better understanding.</P>
emsp
<DIV class="img_in_text zoom_out_able"><P class="fig_illustration">Table 1. Table of soaking process for paraffin sectioning</P></DIV><P>After the soaking process the samples were then embedded in paraffin wax and sliced into biopsy of 10μm thick. They were gently flushed with PBS before the haematoxylin was applied and let sit in the dark for 4 minutes. Haematoxylin was then removed and the samples were rinsed and flushed in ddH2O for 5 minutes. Add 4 drops of HCl-Ethanol (1ml HCl+400ml ethanol) and rapidly dip the sample in ddH2O where a 2-minute flush as well as a 2-minute soaking were performed. Eosin was applied and last on the biopsy for 15 seconds before it was flushed out gently using ddH2O. Similar to the paraffin sectioning, a series of soaking process on decoloring shaker was involved and mentioned in table below. The biopsy was let dry in room temperature and sealed with neutral balsam to complete the HE stained slide.</P><DIV class="img_in_text zoom_out_able"><P class="fig_illustration">Table 2. Table of soaking process for H&amp;E Staining</P></DIV></DIV></LI><LI class="storage_box_ele"><DIV class="storage_header"><P>Examination of T antigen expression level</P></DIV><DIV class="storage_text"><P>The examination involved two examining method: the cellular level examination and the
biopsy examination.<STRONG>Subject: </STRONG>Colorectal tissue from CRC induced mouse model and normal ones
Material: Trypsin, PNA-FITC, PBS, ddH2O<STRONG>Experiment aim:</STRONG>Through document material we knew that the T antigen expression level of cancerous area is much higher than normal tissue, and this characteristic plays an important role in our project design. So before we started to testify our product’s quality, it is essential to solidify the foundation of our product design. 
Experiment design:In this experiment we use the fluorescent level of PNA-FITC to indicate the expression of T antigen. PNA-FITC is a plant lectin protein that binds with the sugar that contains carbohydrate sequence Gal-β(1-3)-GalNAc, a sequence also carries by our testifying target, T antigen. Also, the frozen section of the tissue makes the binding with cellular dye better to observe. After adequate binding time being given, we can compare the fluorescent level of CRC mouse model with normal ones to reach to our conclusion. <STRONG>
Experimental procedure:</STRONG><STRONG>C.1 cellular level examination</STRONG>First, the colorectal tissue was removed from the subjects and digested by trypsin under 37°C for 15~20 minutes. The volume of the trypsin used is four times of the sample with the concentration of 0.25%. The liquid should then be centrifuged for 10 minutes in 1000XG and remove the supernatant to eliminate further damage by trypsin remained. Phosphate buffered saline (PBS) was added to the remains before the samples were resuspended. The mixture were then filtered through 50μm filter membrane and transport to new lightproof sample tubes. A drop of the sample was drawn and observed under microscope to ensure the density of which is around 106cell/ml. If the cell number is lower than target, the samples should be centrifuged again and withdraw certain amount of the supernatant. While in higher cell count situation, extra PBS should be added to dilute the sample to the ultimate density.Then PNA-FITC(1μg/ml) was added to the samples and was incubated under 37°C for 30 minutes. Centrifuge the samples for 10 minutes in 1000XG and replace the supernatant with clear PBS to eliminate the possible interrupting signals caused by unbunded PNA-FITC. Our samples were then measured using flow cytometry. When using the cytometry we circled out the living cells on the graph and compare the fluorescent level chart between CRC models and normal mouse.<STRONG>C.2 Biopsy examination</STRONG>The tissue was fixed by frozen section before we apply PNA-FITC on to the biopsy. So first, a section of colon or rectum was removed from the subjects and longitudinally cut open to form a rectangular piece of tissue. The tissue was then placed on a small cardboard and soaked in paraformaldehyde (PFA) solution overnight. The soaking solution was then switched to 30% sucrose solution and last for another night. Put the tissue in freezing solution and freeze under -20°C overnight to prepare the following biopsy. The properly frozen tissue were sliced into biopsy of 10μm thick.In the following procedure we refer PNA-FITC to as the primary antibody and carry out the standard procedure of immunohistochemical test. The biopsy was first flushed with 0.1% tween-20 phosphate buffered saline solution (PBST) three times (10minutes each time), and then the blocking liquid (5% FBS+0.1% Triton X-100) was added with the incubation last for an hour. After removing the blocking liquid the samples require another three-time flushing by PBST. At last PNA-FITC (50μg/ml) was added and the samples were incubated in 4°C overnight. Before being observed under fluorescent microscope, a three-time PBST flushing was done again. The final samples were sealed with mounting medium that contains 4′,6-diamidino-2-phenylindole (DAPI) for better vision on the structure as well.
</P><P>
[1]Abed. J ,Emgard.JEM ,Fap2 Mediates Fusobacterium nucleatum Colorectal Adenocarcinoma Enrichment by Binding to Tumor-Expressed Gal-GalNAc,CELL HOST &amp; MICROBE, Vol20 215-225,08/10/2016 



</P></DIV></LI><LI class="storage_box_ele"><DIV class="storage_header"><P>Bacterial adhesion property examination</P></DIV><DIV class="storage_text"><P><STRONG>Subject:</STRONG> Colorectal tissue from CRC induced mouse model and normal ones
<STRONG>Material: </STRONG>Two types of engineered bacteria: A. bacteria with surface peptide expression and GFPB. bacteria with only GFPPBS<STRONG>Experiment aim: </STRONG>After solidifying the distinguishing T antigen expression level in experiment B as well as the adhesion property in cellular level, it is time to apply our product directly on tissue to determine the adhesion property in real situation.<STRONG>Experimental design:</STRONG>Since the overall examination of fluorescent level of the tissue is difficult to perform, we’ve decided to apply our device straight on the tissue before performing a frozen section. Frozen section comparing to paraffin section enables the tissue to maintain the binding with bacteria better so it was performed soon after the bacterial adhesion procedure. <STRONG>Experimental procedure:</STRONG>First, a section of colon or rectum was removed from the subjects and longitudinally cut open to form a rectangular piece of tissue. The tissue were then placed on a small cardboard and soaked in paraformaldehyde (PFA) solution overnight. The soaking solution was then switched to sucrose solution (40%) and last for another night. Meanwhile our engineered bacteria were incubated and induced until the final OD600=1 with obvious GFP observed under fluorescent microscope. Then, the tissue were separately put in the two kind of bacteria media and were incubated under 37°C for 30 minutes. After the incubation the tissue were soaked in PBS and gently washed for three times using PBS. Put the tissue in freezing solution and freeze under -20°C overnight to prepare for the following biopsy. The properly frozen tissue were slice into pieces of 10μm thick to form a biopsy. The biopsy was then observed under fluorescent microscope to compare the aggregation of the glowing bacteria. </P></DIV></LI><LI class="storage_box_ele"><DIV class="storage_header"><P>Ultrasound signal examination </P></DIV><DIV class="storage_text"><P><STRONG>Subject: </STRONG>colorectal section from CRC induced mouse model and normal ones<STRONG>Materials:</STRONG> Two types of engineered bacteria: A. Vesicles induced bacteriaB. Vesicles induced bacteria with surface peptide expressionC. Bacteria with no vesicles (with only EFP)<STRONG>Experiment aim:</STRONG>The ultimate goal of our device is to enable vitro detection of CRC via ultrasound, so the demonstration of the ultrasound signal results from our device is undoubtedly essential. Since the signal from bacterium gel was previously testified, we’ve then moved on to observe signal from our mouse models in order to analyze the quality and feasibility of our product. This experiment involves two different procedures in order to reach different conclusions.  For the first procedure which uses vesicles induced bacteria and bacteria with no vesicles, we would like to testify whether the vesicle signal results from our device is visible and breakable under ultrasound; as for the second procedure involving vesicles induced bacteria and vesicles induced bacteria with surface peptide expression, the specificity of bindings between surface peptide and T antigen as well as the actual signal difference were being determined. <STRONG>Experimental design: </STRONG>Before introducing engineered bacteria into our mouse models digesting system, we would like to start with applying our device on removed colorectal section in vitro. Also in order to push our device step by step toward completeness as well as to lay solid foundation for further demonstration, our experiments were first conducted on an ideal model. To create the ideal model, we’ve removed the possible interruption inside the intestines meanwhile creating a system where ultrasound could be easily observed. <STRONG>Experimental procedure:</STRONG>A section of colon or rectum (2cm~3cm) was removed from the subjects with the faeces inside removed. The whole section was then washed with PBS while the testinal mucosa remained. The flowing procedure differs between bacteria types.<STRONG>E.1 Vesicles induced bacteria comparing with bacteria with no vesicles.</STRONG>In this part of experiment our subjects were colorectal sections from normal mice. Before introducing bacteria into the intestines, the bacterial suspension was mixed with agarose to form a 0.2% agarose solution. The solution was then pumped into the colorectal section carefully using syringe. After the injection the intestine was clamped at both sides to let set the solution. We chose 6-well culture plate as the container for our samples and sealed our sample in 0.4% agarose for better ultrasound observation. The samples were then observed under ultrasound with the frequency of 21mHz. </P><DIV class="img_in_text zoom_out_able"><P class="fig_illustration">Fig 1. Schematic of ultrasound sampling for E.1 experiment.</P></DIV><P><STRONG>E.2 Comparison of vesicles induced bacteria with and without surface peptide expression</STRONG>As for this part of experiment our subjects included colorectal sections from both normal mice and CRC induced mouse models. After realizing signal observation inside the intestines, we moved onto testifying the specificity of peptide binding. Similar to the procedure mentioned above, the bacterial suspension was mixed with agarose to form a 0.2% agarose solution and pumped into our samples to form a gel inside the intestines. The difference lied in the way of how our samples were placed in the container. First we poured 0.5% agarose gel into the well with a round model standing in the middle, forming a sunken space. The samples were then placed in this space to build a “standing” sample. Slowly pour 0.2%agarose into the gap to stabilize the sample. After solidification the projecting part above the gel were removed to form a smooth gel service for ultrasound examination. The samples were then observed under ultrasound with the frequency of 21mHz. 
</P><DIV class="img_in_text zoom_out_able"><P class="fig_illustration">Fig 2. Schematic of ultrasound sampling for E.2 experiment</P></DIV></DIV><DIV id="footer"><P>Copyright © 2018 SJTU-BioX-Shanghai. All rights reserved.</P></DIV></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>