<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNITS_Trieste_Data"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNITS_Trieste/Data">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNITS_Trieste/Data&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITS_Trieste/Data&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNITS_Trieste/Data&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNITS_Trieste/Data" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNITS Trieste/Data</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="access"><DIV class="wrapper"><DIV class="menu"><UL><LI class="page_item"><A href="https://2011.igem.org/Team:UNITS_Trieste" title="Home" class="current_page_item">Home</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Team" title="Team:UNITS Trieste/Team">Team</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Team#overview" title="Team:UNITS Trieste/Team">Overview</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Team#location" title="Team:UNITS Trieste/Team">Location</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Team#press" title="Team:UNITS Trieste/Team">Press</A></LI></UL><LI class="page_item"><A href="/Team:UNITS_Trieste/Project" title="Team:UNITS Trieste/Project">Project</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Project#overview" title="Team:UNITS Trieste/Project">Overview</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Project#modeling" title="Team:UNITS Trieste/Project">Modeling</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Parts" title="Team:UNITS Trieste/Parts">Parts</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Parts#overview" title="Team:UNITS Trieste/Parts">Overview</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Parts#conclusion" title="Team:UNITS Trieste/Parts">Conclusion</A></LI><LI class="page_item"><STRONG class="selflink">Data</STRONG></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Notebook" title="Team:UNITS Trieste/Notebook">Notebook</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Safety" title="Team:UNITS Trieste/Safety">Safety</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Acknowledgements" title="Team:UNITS Trieste/Acknowledgements">AKs</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Acknowledgements#judging" title="Team:UNITS Trieste/Acknowledgements">Judging</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Acknowledgements#attributions" title="Team:UNITS Trieste/Acknowledgements">Attributions</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Acknowledgements#cooperations" title="Team:UNITS Trieste/Acknowledgements">Cooperations</A></LI><LI class="page_item"><A href="/Team:UNITS_Trieste/Outreach" title="Team:UNITS Trieste/Outreach">Outreach</A></LI></DIV></DIV></DIV><DIV id="header"><DIV class="wrapper"><DIV id="project-description" role="description"><DIV class="site-title" id="Data"><SPAN>SINBIOME DATA</SPAN></DIV></DIV></DIV></DIV><DIV id="main"><DIV class="wrapper"><DIV id="container"><DIV id="header-data"><H1>SYNBIOME</H1><SPAN>Click for details on image clickable areas!</SPAN></DIV><DIV id="text-model"><TABLE id="datatable" style="font-size: 10px;"><TBODY><TR><TD class="tableh">Produced AHL</TD><TD class="validated">3OC12 AHL</TD><TD class="validated">3OC8 AHL</TD><TD>-</TD></TR><TR><TD class="tableh">Responds to</TD><TD>-</TD><TD class="validated">3OC12 AHL</TD><TD class="validated">3OC8 AHL</TD></TR><TR><TD class="tableh">Generated reward</TD><TD>-</TD><TD class="validated">Β-glucosidase(Cellobiose to glucose)</TD><TD class="validated">Secreted 	Β-lactamase</TD></TR><TR class="tablegray"><TD>Direct dependence</TD><TD>B&amp;C</TD><TD>A&amp;C</TD><TD>A&amp;B</TD></TR></TBODY></TABLE><SPAN class="validated">Validated</SPAN></DIV><DIV id="text-data"><DIV id="pTraBox" class="data_box"><H1>pTraBox</H1><DIV><H3>Excision of CMV from pIRES2-EGFP and following riligation of the backbone</H3>

                            pIRES2-EGFP supplied by Clontech has been digested in AseI and NheI (Fig.1) in order to remove the constitutive CMV promoter and then the linearized backbone has been purified using the &quot;Wizard Gel Clean Up System&quot; by Promega.
                            <CENTER><SPAN class="refNews">Figure 1</SPAN></CENTER>
                            The extremities of the linearized backbone have been blunted in order to allow its self ligation.
                            
                            XL10-GOLD competent cells have been transformed with the products of ligation and then minipreps have been done.
                            
                            The colonies have been checked by enzymatic digestion with NdeI and BamHI, the positives must show only one excised fragment of 600bp (Fig.2).<CENTER><SPAN class="refNews">Figure 2</SPAN></CENTER><H3>Excision of TraBox-CMV from pSEAP</H3>
                            pSEAP has been double digested with EcorI and NotI in Buffer EcoRI plus BSA in 30ul total.
                            
                            The digestion has been checked on agarose Gel 0.8% W/V.
                            
                            The fragment corrensponding to the TraBox/CMVmin has been purified using the &quot;Wizard Gel Clean Up System&quot; by Promega.
                        <H3>Cloning TraBox-CMVmin in pCDNA3 using NotI- EcoRI sites:</H3>
                            
                            pCDNA3 has been previously cut in EcorI and NotI in order to obtain the linearized backbone ready for the cloning of TraBox-CMVmin.
                            
                            Different Condition of ligation has been performed looking for the best efficiency.
                            
                            The colonies obtained in this way have previously been screened by colony pcr and then checked by enzymatic digestion.
                            
                            All the digested colonies were positive, the fragment excised by the EcorI/XhoI double digeston is the TraBox-CMVmin. (Fig.3)
                            <CENTER><SPAN class="refNews">Figure 3</SPAN></CENTER>
                            One of the positives has been chosen and then amplified by trasformation in XL10-GOLD competent cells. The plasmidic DNA has been purified using a commercial Kit supplied by Promega.
                            
                            The Plasmidic DNA has been subsequently digested in EcoRI and XhoI in order to obtain the same insert previously cloned provided by the XhoI sites.
                            
                            The insert TraBox-CMVmin has been purified using the &quot;Wizard Gel Clean Up System&quot; by Promega.
                        <H3>Cloning TRABOXCMVmin in pIRES2-EGFP/CMV- using EcorI/XhoI sites in order to obtain pTraBOX-IRES-EGFP</H3>                        	
                            
                            TraBox-CMVmin has to be cloned in the pIRES2-EGFP/CMV- previously digested in EcoRI XhoI.(Fig.4)<CENTER><SPAN class="refNews">Figure 4</SPAN></CENTER>
                            The linearized backbone has been purified using the &quot;Wizard Gel Clean Up System&quot; by Promega and then ligated with the TraBox-CMVmin as insert.
                            
                            Different Condition of ligation has been performed looking for the best efficiency.
                            
                            XL10-GOLD competent cells have been transformed with the products of ligation and then minipreps has been done.
                         
                            The plasmidic DNA so obtained has been screened by enzymatic digestion using EcoRI and XhoI. The positives have to show the TraboxCMVmin excised in agarose gel electrophoresis separation (Fig5).
                            
                            Colony N°2 and 4 has been chosen as positive and amplified in order to obtain more plasmidic DNA.<CENTER><SPAN class="refNews">Figure 5</SPAN></CENTER><H3>Cloning sBLA in pTraBox-IRES-EGFP</H3>
                            
                            sBLA has to be cloned in the pTRABOX-IRES-EGFP previously digested in EcoRI - BamHI.
                            
                            The linearized backbone has been purified using the &quot;Wizard Gel Clean Up System&quot; by Promega and then ligated with the sBLA as insert.
                            
                            Different Condition of ligation has been performed looking for the best efficiency.
                            
                            XL10-GOLD competent cells have been transformed with the products of ligation and then minipreps has been done.
                            
                            The plasmidic DNA so obtained has been screened by enzymatic digestion using EcoRI and BamHI.
                            
                            The positives have to show the sBLA excised in agarose gel electrophoresis separation(Fig.6).
                            <CENTER><SPAN class="refNews">Figure 6</SPAN></CENTER>
                            Colony N°4 and N°5 have been chosen as positive and amplified in order to obtain more plasmidic DNA. 
                        <H3>Checking the final constructs pTRABOX-sBLA-IRES-EGFP</H3>
                            In order to check the final constructs both the plamidic DNA obtained by the clone N°4 and 5 has been digested with:<EM> -   <B>EcoRI-BamHI</B>: sBLA has to be excised
                                
                                -   <B>EcoRI-XhoI</B>: TraBoxCMVmin has to be excised
                                
                                -   <B>NdeI-BamHI</B>: The construct has to be linearized
                                </EM>
                            All the digestions have been checked in Gel electrophoresis separation on Agarose 1% W/V (Fig.7)<CENTER><SPAN class="refNews">Figure 7</SPAN></CENTER></DIV></DIV><DIV id="P65-TraR" class="data_box"><H1>P65-TraR</H1><DIV><CENTER><H3>AHL Sensible Eukaryotic Switch</H3></CENTER><P>
We decided to use p65-TraR (Neddermann P. et al., 2003), kindly provided by Dr. R. Cortese's group, as a eukaryotic genetic switch sensible to AHL-OXOC8. This chimerical protein is composed by a portion of the human p65, a NLS signal and the whole TraR. (Fig.1)
                        </P><CENTER><SPAN class="refNews">Figure 1</SPAN></CENTER><P>
The gene received by Dr. R. Cortese's group doesn't fit the RCF10 standard because it presents several EcoRI, PstI and XbaI restriction sites (RS) inside the coding sequences. Based on this evidence we decided to design a new p65-TraR that would fit the RCF10 standards in terms of RS inside the coding sequence and the presence of Prefix and Suffix. In this way the BBa_K553023 has been generated. DNA2.0 performed the P65-TraR gene synthesis and optimization for the expression in mammals systems.
                        </P><CENTER><SPAN class="refNews">Figure 2</SPAN></CENTER><P>
The P65-TraR in our system represents the most important element because it makes possible the communication between the two kingdoms. Using the TraR protein it can recognize both AHL-OXOC8 and the TraBox region and thanks to the NLS signal and the p65 it can migrate in the nucleus, where it acts as an eukaryotic trans-activator. (Fig.3)
                        </P><CENTER><SPAN class="refNews">Figure 3</SPAN></CENTER></DIV></DIV><DIV id="LASnlator" class="data_box"><H1>LASnlator</H1><DIV><P style="width:320px; float:right;">
                            This part, made of two composite Biobrick, provides the continuous presence of LasR. 
                            
                            This trans-activator is ready to bind OC8 AHL and then it positively regulates the transcription of both the cellobiosidase and LasI, the OC8 AHL synthase present on the same plasmid.
                        </P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/1/19/Units-bkgG.png)"><SPAN class="centered"><B>Generation of: Constitutive Promoter - RBS - LasR - Terminator</B></SPAN><SPAN>To build this plasmid we used the following parts:
                                    -  Constitutive Promoter <B>BBa_J23100</B> - 35 bp
                                    -  RBS <B>BBa_B0034</B> - 12 bp
                                    -  Las R (coding region) <B>BBa_C0179</B> - 723 bp
                                    -  Terminator <B>BBa_B0015</B> - 129 bp
                                </SPAN><SPAN>These BioBricks transformed into DH5α, as suggested by the iGEM protocol. BBa_C0179 was digested with EcoRI/SpeI in order to isolate the LasR. BBa_B0015 was linearized with an EcoRI/XbaI-digestion and LasR was ligated ahead of the terminator. The ligated product was than transformed into DH5α and seeded in the presence of the appropriate antibiotic. The growing colonies were tested with a colony PCR (Vf-Vr2 primers), following this protocol:</SPAN><SPAN class="centered">93°  5' | 30x (95° 30&quot; | 50° 30&quot; |72° 60'') | 72° 7'| 4° ∞</SPAN><SPAN>The positives, highlighted through electrophoresis, have been expanded and the plasmid extracted.
                                
                                The same steps were followed to verify the ligation of  LasR-Terminator (XbaI/PstI digested) downstream the RBS (BBa_B0034). The positive colonies were extracted and the plasmids have been digested with XbaI/PstI. The RBS-LasR-Terminator fragments were inserted in the plasmid BBa_J23119 at first but, as seen with other constructs, this promoter didn't work as expected. So, finally we ligated them inside a different promoter, BBa_J23100, previously digested with SpeI/PstI in order to eliminate the RFP reporter and linearize the backbone. As always, the ligation product was tested with the colony PCR and the positive colonies were inoculated to extract the plasmids. 
                            </SPAN><SPAN>The final BioBrick was tested with an EcoRI/PstI cut, giving our insert long 921 bp, whilst the plasmid is 2000 bp (see figure below).</SPAN><SPAN>To make LAS- &quot;n&quot;lator suitable for our system, this plasmid was ligated to the PromLasR - TraI BioBrick and inserted into the kanamycin resistant vector pBBR1MCS-3.</SPAN></P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/0/08/Units-bkgY.png)"><SPAN class="centered"><B>Generation of: PromLasI - RBS - TraI - Terminator</B></SPAN><SPAN>To build this plasmid we used the following parts:
                                    -  Promoter Las R regulated <B>BBa_R0079</B> - 157 bp
                                    -  RBS <B>BBa_B0034</B> - 12 bp
                                    -  TraI from <B><I>A. tumefaciens</I></B> (New!) -  639 bp
                                    -  Terminator <B>BBa_B0015</B> - 129 bp
                            </SPAN><SPAN>The BioBricks BBa_R0079, B0034 and B0015 were resuspended and transformed into DH5α cells. The PromLasR was digested with EcoRI/SpeI and checked on gel electrophoresis. The purified fragments were ligated upstream of the EcoRI/XbaI-digested RBS and then transformed into DH5α. Then a colony PCR was run to check the transformed colonies (primers: Vf2 &amp; Vr). The protocol used was the following:</SPAN><SPAN class="centered">93° 5' | 25x(93° 30&quot; | 50° 30&quot; | 72° 40&quot;) | 72° 7' | 4° ∞</SPAN><SPAN>The positive colonies were expanded, their plasmids extracted and then linearized with a SpeI/PstI digestion.
                                
                                Meanwhile, a PCR amplification of TraI was made from the gDNA of <I>A. tumefaciens</I> with TaqPol, following this PCR protocol (oligos on parts page):
                            </SPAN><SPAN class="centered">95° 5' | 10x(93° 30&quot; | 56° 30&quot; | 72° 40&quot;) | 23x (93° 30&quot; | 65° 30&quot; | 72° 40&quot;) | 72° 7' | 4° ∞</SPAN><SPAN>The TraI amplification, to which the EcoRI and PstI restriction sites were added through the PCR primers, was verified on gel electrophoresis, extracted and then digested with EcoRI/PstI. 
                                
                                The digested fragment was ligated into pBSIIK and transformed into DH5α. Positive white colonies growing on Xgal-LB agar plates were checked for the presence of TraI with the following colony PCR protocol (PTraI oligo on parts page):
                            </SPAN><SPAN class="centered">93° 5' | 25x(95° 30&quot; | 65° 30&quot; | 72° 40&quot;) | 72° 7' | 4° ∞</SPAN><SPAN>The PCR amplification was checked on gel electrophoresis, and the positive colonies selected for sequencing. When the sequencing was completed, we analyzed and selected the mutation-free samples.
                                
                                TraI was digested with EcoRI/SpeI in order to ligate it in ahead of the terminator, previously digested with EcoRI/XbaI. The ligated product was transformed into cells and the resulting colonies were checked with a colony PCR (prmers: vf2 and vr):
                            </SPAN><SPAN class="centered">93° 5' | 25x(95° 30&quot; | 50° 30&quot; | 72° 60&quot;) | 72° 7' | 4° ∞</SPAN><SPAN>The positive colonies were expanded and their plasmids extracted. 
                                
                                The TraI-Terminator fragment was isolated with a XbaI/PstI digestion, and then ligated into the previously linearized plasmid containing the PromLasI-RBS. We thus transformed DH5α  and subsequently performed a colony PCR with the same protocol used before, in order to expand the positive colonies and extract the plasmids.
                            </SPAN><SPAN>The completed BioBrick was checked through an EcoRI/PstI digestion and sequencing. The digested fragment of B3 was of the appropriate length (937 bp), as confirmed with gel electrophoresis (see figure below).</SPAN></P></DIV></DIV><DIV id="pLasI-glucosidase" class="data_box"><H1>pLasI-glucosidase</H1><DIV><P style="width:320px; float:right;">
                            This plasmid hosts an OC12 AHL inducible promoter: LasI (BBa_R0079). Once OC8 AHL binds the LasR trans-activator (BBa_C0179) the cellobiosidase transcription is activated thus the bacteria itself can transform the cellobiose in glucose and use it as a source of energy.
                            
                            The plasmid provides also to constitutively generate the GFP and host a kanamycin resistance.
                        </P><CENTER><H3>Generation of: PromLasI - RBS - Glucosidase - terminator</H3></CENTER><P class="full-descr-bat">
                            To build up this plasmid we used the following parts:
                            <SPAN class="list">-</SPAN>Promoter Las I <B>BBa_R0079</B>
                            - 157 bp
                            <SPAN class="list">-</SPAN>RBS <B>BBa_J15001</B>
                            + Glucosidase <B>BBa_K392008</B>
                            composite Biobrick - 1681 bp
                            <SPAN class="list">-</SPAN>Terminator <B>BBa_B0015</B>
                            - 129 bp
                            </P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/1/19/Units-bkgG.png)"><SPAN>The double terminator BioBrick (BBa_B0015) has been resuspended and then amplified through transformation into DH5α competent cells. The plasmid was then extracted and digested (EcoRI/XbaI).</SPAN><SPAN>The RFC10 compatible RBS-Glucosidase plasmid (BBa_J15001 + BBa_K392008), a gift from Dr. C. French (University of Edinburgh), was transformed into DH5α, extracted and digested (EcoRI/SpeI). The glucosidase was then ligated inside the BBa_B0015 vector and transformed into DH5α. The transformed colonies were screened with colony PCR (primers: Vf and Vr2), using the following protocol:</SPAN><SPAN class="centered">93° 5' | 30x(95° 30&quot; | 50° 30&quot; | 72° 60&quot;) | 72° 7' | 4° ∞</SPAN><SPAN>As seen with other constructs, the first ligation made with the constitutive promoter was not successful: the DNA sequencing confirmed that. The construct was then built again from the beginning.</SPAN></P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/0/08/Units-bkgY.png)"><SPAN>Unlike PromTraI-Glucosidase, the EcoRI/XbaI-digested RBS-glucosidase-terminator vector was ligated with PromLasI, previously cut with EcoRI/SpeI. Different ligation strategies were attempted and later DH5α were transformed. We then performed a colony PCR with the following protocol (PromTrFw and PromTraRev): </SPAN><SPAN class="centered">93° 5' | 30x(95° 30&quot; | 50° 30&quot; | 72° 70&quot;) | 72° 7' | 4° ∞</SPAN><SPAN>Positive colonies were expanded and their plasmids extracted.
                                We then proceeded with the quality control: the plasmid was control-digested with EcoRI/PstI and checked on gel electrophoresis, showing the expected pattern with the construct at about 1850 bp and the plasmid backbone at less than 3000 bp.</SPAN><SPAN>This plasmid was sent to sequence and the results were analyzed and compared with the irregular construct we'd previously made. The composite construct is correct but we observed some mutations in the glucosidase sequence. </SPAN></P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/9/97/Units-bkgB.png)"><SPAN>To complete our construct, the BBa_I13522 PTet GFP was resuspended, transformed, extracted and digested with XbaI/PstI. This fragment was then ligated into our SpeI/PstI-digested construct. The ligation product was transformed into DH5α, giving some positive green colonies expressing GFP.</SPAN></P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/b/b2/Units-bkgR.png)"><SPAN>Positive colonies were expanded and the extracted and digested with EcoRI/PstI. The digested fragments were finally ligated into the kanamycin-resistant vector pSB1K3, then transformed into DH5α and seeded with the appropriate antibiotic, in order to allow the selective growth only of the colonies carrying our construct B1 ligated in the new plasmid.</SPAN></P><P class="full-descr-bat"><B>Notes:</B>
It has been planned to join in this high copy number plasmid also the parts present in the other medium copy number plasmid (PLasI-TraI and PConst-LasR). This strategy was performed with success during the last week.
</P></DIV></DIV><DIV id="Pcost-LasI" class="data_box"><H1>Pcost-LasI</H1><DIV><P style="width:480px; float:right;">
                        	This plasmid contains a constitutive promoter (BBa_J23100) that guarantee a continuous production of the LasI synthase (BBa_K081016) that in turn produces the 3OC12 AHL autoinducer.
                        </P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/1/19/Units-bkgG.png)"><SPAN>The LasI generator device present in the registry – BBa_K081016 - has been resuspended and then amplified through transformation into DH5α competent cells. The plasmid was then extracted and digested (XbaI/PstI).
At the same time the constitutive promoter containg plasmid was processed in the same way except for the digestion (SpeI/ PstI).</SPAN></P><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/0/08/Units-bkgY.png)"><SPAN>The obtained composit part is then transfered to a medium copy plasmid (pBBR)</SPAN></P><P class="full-descr-bat"><B>Notes:</B>
It has been planned to insert the RFP reporter (BBa_I13522) downstream this part and to obtain all the elements on a single plasmid. This strategy was performed with success during the last week of work.
</P></DIV></DIV><DIV id="Pcost-RFP" class="data_box"><H1>Pcost-RFP</H1><DIV><P class="descr-bat" style="background-image:url(https://static.igem.org/mediawiki/2011/1/19/Units-bkgG.png)"><SPAN>High copy number plasmid (pSB1K3) containing mRFP (BBa_I3521).</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="units-footer"><DIV class="wrapper"><DIV class="links"><H3><SPAN class="mw-headline" id="Links">Links</SPAN></H3><UL><LI><A href="https://2011.igem.org">IGEM</A></LI><LI><A href="http://www.units.it">UNITS</A></LI><LI><A href="http://www.icgeb.org">ICGEB</A></LI></UL><H3><SPAN class="mw-headline" id="Search">Search</SPAN></H3><DIV class="noprint" id="controls-search"><FORM id="searchform" action="/Special:Search"> </FORM></DIV></DIV><DIV class="links"><H3><SPAN class="mw-headline" id="Main_Navigation">Main Navigation</SPAN></H3><UL><LI><A href="https://2011.igem.org/Team:UNITS_Trieste" title="Home">Home</A></LI><LI><A href="/Team:UNITS_Trieste/Team" title="Team:UNITS Trieste/Team">Team</A></LI><LI><A href="/Team:UNITS_Trieste/Project" title="Team:UNITS Trieste/Project">Project</A></LI><LI><A href="/Team:UNITS_Trieste/Parts" title="Team:UNITS Trieste/Parts">Parts</A></LI><LI><STRONG class="selflink">Data</STRONG></LI><LI><A href="/Team:UNITS_Trieste/Notebook" title="Team:UNITS Trieste/Notebook">Notebook</A></LI><LI><A href="/Team:UNITS_Trieste/Safety" title="Team:UNITS Trieste/Safety">Safety</A></LI><LI><A href="/Team:UNITS_Trieste/Acknowledgements#attributions" title="Team:UNITS Trieste/Acknowledgements">Attributions</A></LI><LI><A href="/Team:UNITS_Trieste/Acknowledgements#cooperations" title="Team:UNITS Trieste/Acknowledgements">Cooperations</A></LI><LI><A href="/Team:UNITS_Trieste/Acknowledgements#judging" title="Team:UNITS Trieste/Acknowledgements">Judging</A></LI><LI><A href="/Team:UNITS_Trieste/Outreach" title="Team:UNITS Trieste/Outreach">Outreach</A></LI></UL></DIV><DIV class="offices"><H3>Follow Us!</H3></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:UNITS_Trieste/Data">http://2011.igem.org/Team:UNITS_Trieste/Data</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNITS_Trieste/Data" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNITS_Trieste/Data" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNITS_Trieste/Data&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNITS_Trieste/Data&amp;oldid=252981" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>