"
Page
Discussion
View source
History
teams
Log in
 
Team:UNITS Trieste/Notebook
From 2011.igem.org
Home
Team
Overview
Location
Press
Project
Overview
Modeling
Parts
Overview
Conclusion
Data
Notebook
Safety
AKs
Judging
Attributions
Cooperations
Outreach
SINBIOME NOTEBOOK
Diary, Friday 17 December 2010
Giulio: Do you know IGEM?
Luca: No, what’s it?
Giulio: It’s a competition, we will design and develop our own project in12 weeks giving a proof of concept ...
Luca: It sounds interesting… let me think about it ...
10 minutes later...
Luca: Ok ... we will rock it!
$$$ Funding, building up a team, Finding Advisors-Instructors, 6 month of theoretical work, tens of sleepless nigths, hundreds of papers, Imagination, Creativity, Development and so ... SYNBIOME
 
1st Week 2nd Week 3rd Week 4th Week 5th Week 6th Week 7th Week  
8th Week 9th Week 10th Week 11th Week 12th Week Last Month  
1st Week (28 June–3 July)
Our lab work starts off with the arrival of the two plasmids pcD/p65NTraR and pSEAP-(tra box)(1-7) (Neddermann P. et al., 2003), kindly provided by Dr. R. Cortese’s group, and of the commercial vector pIRES2-EGFP (Clontech), in which we are going to subclone all the parts needed to engineer our eukaryotic cells.
Different competent E.coli strains (XL10-Gold and DH5-alpha) have been transformed with the above mentioned plasmids and selected with appropriate culture conditions (p65NTraR and pSEAP grow on ampicillin, pIRES2-EGFP grows on kanamicin).
Subsequently the amplified plasmids have been purified following a regular Miniprep protocol.
Concurrently, we have designed and ordered two oligos needed in order to remove the CMV immediate early promoter (CMVie) from pIRES2-EGFP, and to allow the correct insertion of the secreted mammalian beta-lactamase (sBLA) coding sequence into the vector.
We then started a series of enzymatic digestions in order to extrude CMVie from pIRES2-EGFP and just take its backbone, and to isolate sBLA from pSEAP.
In the meantime the plasmid pWW1015 we requested from Dr. M. Fussenegger arrived and was immediatedly amplified through transformation of DH5-alpha bacteria, while we also ordered the sequencing of the p65NTraR plasmid (BMR Genomics).
2nd Week (4–10 July)
The oligos we requested the earlier week arrived, and we proceeded with the annealing protocol for the two pairs of complementary strands.
Due to the irregular running pattern pSEAP displayed on agarose gel, we set out to do several control digestions in order to determine its integrity and we also sent a DNA sample to sequence its  multiple cloning site (MCS).
Meanwhile we also started the prokaryotic part of this work by transforming competent bacteria with all the Biobricks needed in this side of the project, and by PCR-amplifying the TraR-regulated promoter, TraI gene and TraR gene sequences from A.tumefaciens genomic DNA.
This Tra family has been cloned into pBluescript and then used to transform bacteria on Xgal-containing Petri dishes. Finally, a colony PCR has been carried out to further amplify the Tra family DNA sequences.
Moreover we transformed new bacteria in order to amplify a plasmid containing the enzyme beta-glucosidase (bGluc) we received from Dr. C. French.
3rd Week (11–17 July)
We obtained pIRES2-EGFP without CMVie but could not ligate our oligo inside it, so we devised new ligation strategies to amplify our backbone, removing all the unnecessary DNA sequences.
We also started working on some of the constructs needed to engineer our bacterial strains, namely: the bacterial construct A0, composed of a TraR-regulated promoter (P-TraR) followed by a beta-glucosidase (BBa_K392008), an E.coli ribosome binding site (RBS, BBa_J15001) and a double terminator (BBa_B0015); the bacterial construct B1, identical to A0 but under the transcriptional control of a LasR-regulated promoter (P-LasR, BBa_R0079); the bacterial construct B2, consisting of a constitutive promoter (BBa_J23119), an RBS (BBa_B0034), a LasR activator (BBa_C0179) and a terminator; and the bacterial construct B3, made of P-LasR, an RBS, the TraI gene and a terminator.
4th Week (18–24 July)
We finally obtained and amplified pIRES2-EGFP without CMVie and without the gene Rac2, that we discovered was cloned inside the MCS.
We then proceeded to amplify an aliquot of pSEAP of which we received the full sequence, in order to transfect HeLa cells and assay it’s activity.
We amplified and ligated TraI with BBa_B0015 in order to subsequently complete the bacterial construct B3. The construction of A0 and B2 is approaching completion as well, while we finished working on constrct B1: we’ll now need to verify the correctness of its sequence.
Meanwhile we also started working on our bacterial construct A2 by ligating the previously amplified P-TraR into BBa_K081016 (LasI comprising an RBS and a terminator up- and downstream, respectively).
5th Week (25–31 July)
We sent the sequences to have the secreted beta-lactamase (sBLA) gene by synthesis, comprising all the restriction sites needed to clone it into our vectors. It should be here next week
Also, we tested the activity of pSEAP after co-transfection with p65-TraR into HeLa cells. To assay this we employed the Great EscAPe secreted alkaline phosphatase detection kit (Clontech), obtaining promising results confirming the efficiency of the Tra box-CMVminimal promoter sequence. We then proceeded to isolate this DNA sequence and ligate it into pIRES2-EGFP (lacking its constitutive CMV promoter), through a series of sequential cloning steps.
The bacterial constructs A2 and B2 have been completed and sent to sequence, while B1 was control-digested but found not to be of the expected dimensions. The same problem occurred with A0, probably due to the beta-glucosidase we used for both the constructs. We thus sent A0 and B1 to be sequenced as well, and in meantime started working to construct it again from the start. The construction of B3 proceeded with the ligation of the TraI/terminator segment with P-LasR/RBS and was followed by the transformation of competent bacteria to subsequently verify the possible completion of the construct.
6th Week (1–7 August)
The Tra box-CMVmin sequence was cloned into pIRES2-EGFP (without CMV), so we wanted to repeat the earlier week’s experiment and see if, following the correct AHL signal, cells would express EGFP. Curiously only a light fluorescence was detected, either due to an error during the cloning steps, or because of the low translation efficiency of the IRES sequence.
Meanwhile the sBLA we ordered by synthesis arrived, so we set out to amplify this construct in order to clone it in an appropriate vector and assay it’s activity.
The results from last week’s sequencing confirmed the successful completion of the bacterial constructs A2 and B2. The sequencing of A0 and B1, on the other hand, did not give the same results. On the contrary, we found a few discrepancies between BBa_K392008’s sequence and that which we were making reference to till now, so the work to re-construct A0 and B1 is still ongoing.
Finally, the construct B3 was completed, controlled, and then ligated into B2.
7th Week (8–14 August)
The synthetic sBLA construct was cloned into pSEAP downstream of the Tra box-CMVmin sequence. This plasmid was then tested after transfection into HeLa cells and successive AHL induction in order to verify the secretion of the enzyme. Early analyses showed negative results, probably due to the low levels of protein produced or to errors during the cloning design. We then set out to do more clonings into other vectors in order to further test sBLA activity. In particular we cloned the synthetic sBLA into pIRES2-EGFP (either with and without CMV), and we also amplified pWW1015 to clone its sBLA into pcDNA3.
Meanwhile we started working on the bacterial construct A1, similar to B2 but with a TraR activator instead of the LasR activator under the constitutive promoter.
The bacterial constructs A0 and B1 have been completed, ligated to two different reporters (RFP and GFP, respectively) and assayed for their activity in response to the appropriate AHL signals (OC8 and OC12, respectively). This assay, which gave good results, showed that these constructs coding for an AHL-inducible beta-glucosidase allow bacteria to survive in a glucose free medium supplied with cellobiose alone. Finally, we also tested our bacteria for the production of AHL on thin layer chromatography.
8th Week (15–21 August)
Analyses on the eukaryotic construct showed some errors in its sequence due to an incorrect excision of the CMVie sequence. We then set out to reassemble the construct from the start.
The bacterial construct A0-RFP was cloned in the pSB1K3 vector. After transforming DH5α bacteria with this plasmid, bacteria were newly made competent in order to successively transform them with a second plasmid containing A1-A2. Meanwhile we tried to ligate the constructs B2 and B3 together.
9th Week (22–28 August)
We finished working on our eukaryiotic construct containing a Trabox-CMVmin sequence followed by the synthetic sBLA cloned into pIRES2-EGFP lacking the constitutive CMVie.
Doing some assays to evaluate the communication between bacteria and HeLa cells we successfully obtained eukaryotic responsivity to the prokaryotic lactones, but had some problems with the co-culture of the two type of cells.
The construct B1-GFP was cloned into pSB1K3, meanwhile we ligated the glucosidase to a new constitutive promoter, in order to further test its activity.
10th Week (29 August – 4 September)
The completed eukaryotic construct (Trabox-CMVmin-sBLA-IRES-GFP) was correctly verified with a series of control digestions and through DNA sequencing. HeLa cells positive to the cotransfection of this construct and the transactivator p65-TraR were isolated through FACS, in order to try and expand a stable cell line.
We successfully ligated the bacterial constructs A1 with A2 and B2 with B3. The constructs A1-A2 and B2-B3 thus obtained were cloned into a pBBR vector, while we also transformed competent bacteria with either A0-RFP or B1-GFP (the beta-glucosidase coding constructs with the respective reporter). These bacteria were then made competent again, in preparation of their further transformation with either A1-A2 or B2-B3.
11th Week (5–11 September)
Constructs A1-A2 and B2-B3 were successfully cloned into pBBR and verified through control digestions. After that, we transformed with these constructs the competent bacteria containing A0-RFP and B1-GFP obtained previously. The bacteria containing B2-B3 were then tested for the production of OC8, giving positive results, while we also started analyzing the production of OC12 from A1-A2-containing bacteria on TLC.
Meanwhile we further verified the activity of our eukaryotic construct, detecting through Western blot the beta-lactamase secreted in the culture medium following p65-TraR-mediated OC8 induction. Finally, we cloned this construct into pSB1C3, but had some problems cloning into this vector several other parts we need to submit.
12th Week (12–18 September)
After having several problems, we finished cloning all our BioBricks and set out to further validate the glucosidase-producing ones in a glucose-free medium supplemented with cellobiose. Meanwhile we kept assaying our bacteria for the production of AHL through TLC.
Finally, much work has been done to complete our site and all the documents for our submissions to the Registry of Standard Biological Parts. For this purpose we also created to each BioBrick its own datasheet, so that their properties and characterization could be more easily accessible to the whole iGEM community and any other synthetic biologist.
Last Month (4-28 October)
In these last weeks following the European Jamboree we focussed on the parts we needed to fix in our system. We modified and optimized the parts needed to engineer bacterial strain A, while we also tested the two strains together to have further evidence of the communication between them.
On the eukaryotic end we're still trying to expand a stable cell line cotransfected with the two eukaryotic constructs. In particular, we finally got p65-TraR synthesized without all the "illegal" restriction sites hindering us from submitting this key part to the Registry.
Working hard on perfecting the wiki, re-designing the poster, rehearsing the presentation and much more till the very last minute!
...Waiting for Boston!
Diary, Saturday 17 September 2011
Veronica: We're on the verge of the end of this venture and still working at full blast!
Niels: We're almost there, huh?
Gigi: Yeah, 12 weeks really flew by
Giulio: Right, and we went all out despite the short time!
Francesca: But it surely was a thrilling experience, we should do it again next year!
Luca: ...Yeah, right, but let's first see if we come back alive from Amsterdam!
Links
IGEM
UNITS
ICGEB
Search
 
Main Navigation
Home
Team
Project
Parts
Data
Notebook
Safety
Attributions
Cooperations
Judging
Outreach
Follow Us!
Retrieved from " http://2011.igem.org/Team:UNITS_Trieste/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
