"
Page
Discussion
View source
History
teams
Log in
 
Team:UNITS Trieste/Acknowledgements
From 2011.igem.org
Home
Team
Overview
Location
Press
Project
Overview
Modeling
Parts
Overview
Conclusion
Data
Notebook
Safety
AKs
Judging
Attributions
Cooperations
Outreach
SINBIOME ACKNOWLEDGEMENTS
JUDGING
We believe our team deserves the Gold medal because we met the following criteria:
Requirements for a Bronze Medal:
1.Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
2.Successfully complete and submit this iGEM 2011 Judging form.
3.Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
4.Plan to present a Poster and Talk at the iGEM Jamboree.
5.Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
5.1.Primary nucleaic acid sequence
5.2.Description of function
5.3.Authorship
5.4.Safety notes, if relevant.
5.5.Acknowedgment of sources and references
6.Submit DNA for at least one new BioBrick Part or Device to the Registry.
Additional Requirements for a Silver Medal:
1.Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected;
2.Characterize the operation of your new part/device.
3.Enter this information and other documentation on the part's 'Main Page' section of the Registry. Part Number(s):
BBa_K553000
BBa_K553001
BBa_K553002
BBa_K553003
BBa_K553004
BBa_K553005
BBa_K553007
BBa_K553008
BBa_K553009
BBa_K553010
BBa_K553020
BBa_K553021
BBa_K553022
BBa_K553023
Additional Requirements for a Gold Medal:
1.Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system. Click here for details.
2.Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation. Click here for details.
iGEM Prizes
Best Human Practice Advance
This has been the first presence of Trieste in iGEM and this meant quite a hard venture in building up a team, finding money, finding a hosting lab, designing and developing our project.
We students have made it all, obviously with the support of the University of Trieste but mostly with our passion and determination.Here in Trieste, for its history and location, we found an ideal place in which young students like us could undertake this innovative approach to research that iGEM represents.
We invested a great amount of energy on human practices because we aimed at creating the right environment to make iGEM a core activity among the first steps of the future researchers.
We decided to talk about iGEM and about our experiences as iGEM team through different channels and in different ways referring to a wide range of audience.
You can find a detailed description of UNITS_TRIESTE team human practices here!
Team_Parts
BBa_K553008
BBa_K553020
BBa_K553021
BBa_K553023
~ iGEM Safety ~
~ Attribution and Contributions ~
Comments
IGEM more than a competition is a life experience; it involves you at all, sometimes it is hard sometimes funny but above all it is by far the best experience which a student could done during his undergraduate studies. The UNITS_TRIESTE Team is an “Handmade” team, no advisors working in synthetic biology field, no company linked to synthetic biology business only 6 students and their mighty wish to take their place in a worldwide acknowledged competition as IGEM is.
Synbiome is made of  "undergraduate" thoughts  and it has been realized by "undergraduate" hands.
As any researchers know three months are nothing, only 90 days tu build up a new, innovative and competitive project to face the world of synthetic biology.
We decided to do it as it must be do, no shortcuts: 6 month of preliminary work making reality the wish to partecipate at igem, learning about  everything as biobricks, sponsors, registry, registration,  human practice, tracks, jamboree, team and whatever else but mostly we thought about the project, tons of rough ideas falling down as bricks (Not yet so “Bio”J), looking for the best way to handle our first partecipation. At the end of june Synbiome was birth, son of all our minds and father of all our expectation.
Thanks to Synbiome we had the chance to involve scientists originally unrelated to the field of synthetic biology as our instructors. They believed so much our project that we’re not only working at Synbiome  in a competitive optical but mostly looking forward  to reach a publication at the end of the year.
These rows have been written only to give you all the feelings that this experience has meant for us, we really hope that you will enjoy Synbiome.
Looking for Boston
During the Regional Jamboree in Amsterdam we focused our attention in collecting any kind of feedbacks from everybody: judges, instructors, advisors and other igemmers.
This has been really useful also to plan our work during this supplementary month.
Looking for Boston we decided to change the genetic circuit inside the bacterial strain A.
At the beginning of this experience we believed that forcing a strong mutualism between the two bacterial strains would represent the best way to build a stable consortium. Following this purpose we designed for Amsterdam a bacterial strain A, which in response to OXOC8 produces OXOC12 and glucosidase and a bacterial strain B, which produces OXOC8 and glucosidase in presence of OXOC12. Both A and B are not sensible to their own language and so A needs B and vice versa. This kind of circuit guarantees a strong mutualism but will also generate a kind of useless redundancy in which both A and B produce the same  reward for the whole system.
Moreover this genetic circuit needs an initial external injection of or OXOC8 or OXOC12 to start the system.
Our feelings suggested us to keep Symbiome simple and more effective associating to each characters a specific and unique task.
On these new evidences we decided to give to cell A the role of starting up the system by constitutively producing OC12, leaving to cells B and C the production of the rewards, respectively the beta-glucosidase and the secreted beta-lactamase.
ATTRIBUTIONS
This is a list of people who helped us making synbiome possible...
Research Group
Who
What
Mauro Giacca,  Director,  ICGEB Trieste Component,  Group Leader
Molecular Medicine,  ICGEB
Miguel Mano,  Postdoc
Molecular Medicine,  High Throughput Facility
Data acquisition
Bruna Marini,  PhD Student
Molecular Medicine
Microscopy
Vittorio Venturi,  Group Leader
Bacteriology,  ICGEB
Daniel Passos,  PhD Student
Bacteriology and plant bacteriology
Flow cytometry
Sandor Pongor,  Group leader
Protein structure and bioinformatics,  ICGEB
Adam Kerenyi,  Staff Scientist
Biological Research Center,  Szeged,  Hungary
Modeling
Dr. Riccardo Sgarra
Department of Life Sciences,  University of Trieste,  Trieste,  Italy
Mass Spectrometry
Dr. C. French
The University of Edinburgh
BBa_K392008
Dr. M. Fussenegger
Department of Biosystems Science and Engineering
PWW0015
Dr. Cortese & P. Neddermann
IRBM Science Park
P65-TraR
TraBOX-CMVmin
Maddalena Fragnito
Video editing
COOPERATIONS
Edinburgh IGEM team
During the theoretical work on Synbiome we decided to use a β-glucosidase to provide our Interkingdom Consortium with a usable carbon source. We asked Chris French to send us the part BBa_K392008, made by team Osaka in 2010 and encoding a β-glucosidase from the bacterium Cellulomonas fimi. Firstly, we tested the part and it worked, but the DNA sequencing we decided to make during the generation of our parts BBa_K553005 and BBa_K553006 underlined the presence of a frameshift at the beginning of the coding sequence, introducing a stop codon. We shared this information with team Edinburgh and together we elaborated a solution to this strange problem (more details can be found on the cooperation page of team Edinburgh’s website). They were also very kind to provide us with a protocol of an assay that can be used to test the activity of this β-glucosidase using 4-methylumbelliferyl-β-D-glucuronide.
Beta Glucosidase
CYP Analysis
Wageningen IGEM team
We also collaborated with team Wageningen, analyzing a bacterial strain expressing CFP with Fluorescence Microscopy. During the summer they typed on Twitter looking for a Confocal Microscope with the proper laser and filter for CFP’s wavelength. Since we had it here at ICGEB, we positively replied to team Wageningen’s request and helped them analyzing their samples, as they also reported in their website’s cooperation page.
Links
IGEM
UNITS
ICGEB
Search
 
Main Navigation
Home
Team
Project
Parts
Data
Notebook
Safety
Attributions
Cooperations
Judging
Outreach
Follow Us!
Retrieved from " http://2011.igem.org/Team:UNITS_Trieste/Acknowledgements "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
