"
Page
Discussion
View source
History
teams
Log in
 
Team:Kent/modelling
From 2014.igem.org
Univeristy of Kent iGEM
 
Modelling
 
This is the template structure to which zingiberene synthase, R-linalool synthase and  limonene synthase were modelled. The models were made using Phyre2[1]  based on the amino acid sequences available from uniprot [2]. The genes used in this project have been curated (Swissprot) so are more reliable than hypothetical proteins identified from homology. The colours show the overlap between the 3 sequences. This suggests that despite having very low sequence identity and producing different groups of terpenes, they are based on the same structure and therefore will have a similar mechanism of action.
 
 
Figure 1: Three protein models superimposed. They are based on the same template from PDB
 
Consurf [4,5] was used to identify sequence conserved residues amongst all the limonene synthase genes available in the protein data bank (PDB). The darker shades of pink highlight the location of residues which are highly conserved. White residues have an average level of conservation (similar or not completely different functional groups in residues) and the blue residues show variable regions. This highlights areas of functional importance in the enzymes. Limonene synthase was used as a model due to lack of sequence information regarding the other synthases. A large number of sequences are required to provide data with a higher confidence level. 61 different limonene synthase genes were used to generate this map.
 
Figure 2: Model of limonene synthase showing sequence conservation amongst the 61 variants in the PDB. Blue (variable), white (average) and pink (conserved).
 
This model suggests there are several domains important to the function of terpene synthases. The substrate-binding active site is of critical importance and therefore the residues around the site which are important in function are likely to be conserved.
 
Figure 3: Model of limonene synthase showing the metal-binding domain (red).
 
The red highlighted region of the protein shows the DDXXD motif common to all terpene synthases. It is the metal binding domain responsible for binding Mg2+ or Mn2+ ions as co-factors for enzyme activity.
An understanding of the functional regions of the synthases could allow the potential for site-directed mutagenesis to increase the activity of an enzyme. Altering residues could increase the Vmax values and or increase the affinity of the enzyme for its substrate. It could also increase the specificity of products from the reaction. For example zingiberene synthase produces low amounts of other compounds such as β-sequiphellandrene and α-bergamotene [6]. Mutagenesis at specific residues may increase the ratio of desired product and make the extraction of our desired product a simpler task.
 
 
References:
[1] Kelley LA and Sternberg MJE (2009) Protein structure prediction on the web: a case study using the Phyre server. Nature Protocols 4, 363 – 371.
[2] Website: Uniprot protein database. Available at URL: http://www.uniprot.org/
[3] Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. (2011) Molecular systems biology 7 :539 PMID: 21988835
[4] Celniker G, Nimrod G, Ashkenazy H, Glaser F, Martz E, Mayrose I, Pupko T and Ben-Tal N (2013) ConSurf: Using Evolutionary Data to Raise Testable Hypotheses about Protein Function Isr. J. Chem. 2013 March 10, doi: 10.1002/ijch.201200096 [ABS],[PDF]
[5] Landau M, Mayrose I, Rosenberg Y, Glaser F, Martz E, Pupko T and Ben-Tal N (2005) ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures.
Nucl. Acids Res. 33:W299-W302. [ABS],[PDF]
[6] Website: Uniprot protein database. Entry C5YHH7 - Zingiberene synthase sorghum vulgare. Available at URL: http://www.uniprot.org/uniprot/C5YHH7 (last modified 01/10/2014).
 
 
Retrieved from " http://2014.igem.org/Team:Kent/modelling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
