<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kent_sequencealign"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Kent/sequencealign">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kent/sequencealign&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kent/sequencealign&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kent/sequencealign&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kent/sequencealign" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kent/sequencealign</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Univeristy of Kent iGEM </TITLE></P><DIV class="container"><DIV class="sidebar1"><TABLE width="100%" border="0"><TBODY><TR><TD> </TD></TR></TBODY></TABLE></DIV><DIV class="content"><TABLE width="950" border="0"><TBODY><TR><TD><DIV align="center"><H1>Sequences, Sequence Alignment and Primer Design</H1></DIV></TD></TR><TR><TD><P align="justify"><STRONG>Overview:
          
          </STRONG></P><P align="justify"><STRONG>Sequence Alignment</STRONG>: </P><P align="justify">After researching fragrance-producing enzymes in plants, we selected two terpene synthase genes from a protein database (Uniprot). These genes were Zingerberene Synthase and R-linalool Synthase respectively. We modified these sequences to optimize translation of the gene product according to the frequency of codon usage in bacteria. The sequences were then further modified to remove endogenous restriction enzyme sites, allowing us to carry out restriction digests without cutting the coding region. We then added a 50 base pair sequence prefix and suffix region to the 5' and 3' ends of our construct respectively corresponding to the vector pSB1C3, allowing us to clone our construct into this plasmid using Gibson Assembly. </P><P align="justify"><STRONG>Gibson Assembly Schematic:
        </STRONG></P><P align="justify">Where A (in red) corresponds to the vector 50 bp pSB1C3 prefix region, B (in green) corresponds to the pSB1C3 suffix region, and the yellow region corresponding to either the R-linalool Synthase or Zingiberene Synthase gene. (Figure1.) </P></TD></TR><TR><TD> </TD></TR><TR><TD> </TD></TR><TR><TD><P align="justify"><STRONG>Primers:</STRONG></P><P align="justify">The red and green arrows represent the forward and reverse primers respectively that correspond to the forward and reverse complement of the sequences in A (region containing prefix) and B (region containing suffix) from the pSB1C3 plasmid. Since these regions were incorporated at the 3' and 5' ends of both our constructs, these primers can therefore be used to amplify either one of our gene fragments as well the pSB1C3 vectors using the polymerase chain reaction (PCR).
        We designed the primers using IDT oligo analyzer1 to ensure that they had a suitable melting temperature, did not form secondary structures and were of a suitable length. The forward and reverse primer sequences and parameters are outlined below. </P></TD></TR><TR><TD> </TD></TR><TR><TD> </TD></TR><TR><TD><P align="justify">The annealing sites of the oligonucleotides are outlined in white in the construct sequences for R-linalool Synthase and Zingiberene Synthase in the next sections.</P><P align="justify">Our Genes:
</P><P align="justify">R linalool synthase:</P><P align="justify"> This gene encodes an enzyme that catalyzes the conversion of the mevalonate pathway intermediate Geranyl diphosphate (GPP) into R-linalool. This particular isomer of Linalool has a fragrance resembling that of a woody lavender smell. We found the gene sequence (taken from Lavender) on Uniprot:        </P><P align="justify">This sequence was then codon optimised for use in E.coli by running the sequence through the IDT codon optomisation tool2. Running this new codon optimised sequence through NEB cutter3 then identified endogenous restriction enzyme sites. Restriction sites corresponding to SpeI, EcoRI, XbaI, and PstI where then removed by creating a one base silent substitution at each endogenous site. Appropriate bases for the substitutions were selected via a codon usage table optimised for E.coli. A fifty base pair sequence corresponding to the prefix and upstream region of the vector PSB1C3 was added to the 5' end of the codon optimized gene such that the first base of the start codon overlapped with the final base of the XbaI restriction site. The 50bp suffix region of the same vector (PSB1C3) was added to the 3'end of the gene to obtain our final construct. The coding part of the final construct was run through Expasy Translate tool4 and aligned with the uniprot protein primary sequence with a 100% match using EBI Clustal Omega5 sequence alignment software. We ordered this sequence as a fragment from the DNA synthesis company Life Technologies and, on receiving the artificially synthesized fragment, cloned it into the pSB1C3 vector provided by iGEM HQ using Gibson Assembly method. </P><P align="justify"><STRONG>Final Optimized Sequence:
R-Linalool Synthase:</STRONG></P><P align="justify"><A href="http://www.uniprot.org/uniprot/Q2XSC5">http://www.uniprot.org/uniprot/Q2XSC5</A><A href="http://www.ebi.ac.uk/ena/data/view/ABB73045">http://www.ebi.ac.uk/ena/data/view/ABB73045</A></P></TD></TR><TR><TD> </TD></TR><TR><TD> </TD></TR><TR><TD> </TD></TR><TR><TD><P align="justify"><STRONG>Zingiberene Synthase:</STRONG></P><P align="justify">This gene product produces Zingiberene (ginger scent) from (2E,6E)-farnesyl diphosphate (FPP) and is found in the long grass Sorghum Bicolor. 
          <A href="http://www.uniprot.org/uniprot/C5YHH7">http://www.uniprot.org/uniprot/C5YHH7</A>
          
          
          The sequence was taken from Uniprot, however, incomplete mRNA sequencing meant that we had to use the amino acid sequence of the protein to determine the missing codons via a codon usage table. This sequence was then processed in a similar manner to R-linalool synthase using the same IDT codon optimisation tool and then removing restriction sites using the same process as outlined above for R-linalool Synthase.     
          
          
          Final Optimised Sequence:          </P><P align="justify">Zingiberene Synthase:
            <A href="http://www.uniprot.org/uniprot/C5YHH7">http://www.uniprot.org/uniprot/C5YHH7</A></P></TD></TR><TR><TD> </TD></TR><TR><TD> </TD></TR><TR><TD><P align="justify"><STRONG>Online tools used:
          
          </STRONG></P><P align="justify">1.	IDT Oligo Analyzer: <A href="https://www.idtdna.com/analyzer/Applications/OligoAnalyzer/">https://www.idtdna.com/analyzer/Applications/OligoAnalyzer/</A></P><P align="justify">2.	IDT Codon Optimization Tool: <A href="https://www.idtdna.com/CodonOpt">hhttps://www.idtdna.com/CodonOpt</A></P><P align="justify">3.	NEB Cutter: <A href="http://tools.neb.com/NEBcutter2/">http://tools.neb.com/NEBcutter2/</A></P><P align="justify">4.	Expasy Translate Tool: <A href="http://web.expasy.org/translate/">http://web.expasy.org/translate/</A></P><P align="justify">5.	Clustal Omega Mulitple Sequence Alignment: <A href="http://www.ebi.ac.uk/Tools/msa/clustalo/">http://www.ebi.ac.uk/Tools/msa/clustalo/</A></P></TD></TR></TBODY></TABLE><P> </P></DIV><DIV class="footer"><P> </P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Kent/sequencealign">http://2014.igem.org/Team:Kent/sequencealign</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kent/sequencealign" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kent/sequencealign" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kent/sequencealign&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kent/sequencealign&amp;oldid=360357" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>