Team:St Andrews/Notebook
Home
Team
Team members
Collaborations
Project
Description & Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
Human practice
Human practices
Education & Engagement
Safety
Judging form
Notebook
⤵
May
June
July
August
September
October
May
During May, the initial stages of project planning occurred, and the team began research on what was feasible and what would be the best way to approach the question posed.
June
Continued planning for various parts of the project, researching necessary protocols and finalising sequences for the first stages of experiments.
Cell lysis:
The team determined which constructs should contain which promoter (weak/strong), which fluorescent protein should be used as the signal part of the system, and later which part of the mNeonGreen protein and which should include a tag for binding affinity. Constructs were ordered.
Biofilm:
Researched properties of the constitution of biofilms that were specific to Pseudomonas Aeruginosa and could be targeted by the system we aimed to create.
Other:
InterLab began initial calibrations for performing the cell measurement and CFU counting experiment, and prepared the initial amount of backbone pSB1C3 provided by iGEM.
July
Cell lysis:
Initially constructs 1-8 were inserted into backbone puc19 with restriction enzymes Nde1 and Pci1, and then transformed into E. coli strain DH5α. Once inserted, constructs were sent for sequencing and were successful. Backbone pHISTEV was prepared by digestion with enzymes Nco1 and Xho1 to be used with construct 9as a standard for measuring fluorescence of other constructs. Constructs 3 and 7 were sent for sequencing. iGEM backbone was prepared for insertion of constructs 1-8 by digesting with EcoR1 and Spe1. Initial attempts to insert the constructs into pSB1C3 failed.
Biofilm:
The team performed PCR to add the necessary restriction sites (EcoR1 and Spe1)
Other:
InterLab prepared competent cells for the wetlab team to use, and performed the cell measurement and CFU counting protocols as provided by iGEM.
August
Cell lysis:
Restriction enzymes Spe1 and EcoR1 were added to constructs 1-9. Constructs 1-8 were inserted into pSB1C3 and 9 was inserted into pHISTEV, and then transformed into DH5α.  Multiple attempts to measure fluorescence of construct 9 induced with ITPG in LB were made, the first few failures due to the addition of the wrong antibiotic (used chloramphenicol when kanamycin was needed). Initial attempts to generate absorbance through combinations of export and retain constructs lysing with bugbuster failed, so switched to lysing via sonification. After switching to this method, constructs 1 and 5 were successfully grown and combined in minimal media (M9). Constructs 2 and 6 were also co-cultured to compare the effectiveness of the added tag. To test if the mNeonGreen protein was present in the constructs, cultures of constructs 1-7 (8 was not tested due to there not being enough lanes on the gel) were run on a Western Blot. Construct 9 production of mNeonGreen (His-tagged) was quantified using a nickel chromatography pull-down.
Biofilm:
Constructs 1-8 were inserted into puc19, after various failed attempts to clone the constructs into pSB1C3 it was decided to try to insert the constructs into the backbone using Gibson Assembly, so primers were designed for biofilm constructs 1-4. This attempt was successful only for construct 2, which was confirmed by sequencing.
Other:
InterLab carried out the optional flow cytometry measurement and redid the cell measurement part of the protocol as the initial measurement data was rejected due to an issue with the negative control. Gibson Assembly primers were designed and ordered for part characterisation experiment and insertion of biofilm constructs into pHISTEV and pSB1C3.
September
Cell lysis:
Various combinations of constructs 1, 2, 5, and 6 were made and used to determine the time necessary for the separate parts of mNeonGreen to combine and fluoresce in a time trial.
Biofilm:
First attempts at Gibson failed and reattempted, and constructs 3 and 4 were sent off to be sequenced.
Other:
PCR of part characterisation gene block and insertion into pSB1C3 using Gibson Assembly. Part improvement experiment begun by ordering primers for Gibson Assembly of constructs with mOrange and the improved part, mOrange2, into pSB1C3 with the restriction sites Xho1 and Nco1 added.
October
Cell lysis:
To prepare for submission, the team attempted to insert mNeonGreen 1-10 and mNeonGreen 11 separately into pSB1C3 to be submitted as basic parts as well as being submitted as composite parts. A single colony grew on the plate of mNeonGreen 1-10, which was submitted without sequencing. Test of system under realistic conditions to be run using an antibiotic to lyse cells retaining mNeonGreen instead of using sonification.
Biofilm:
Construct 2 (alginate binding construct) submitted to the registry.
Other:
The team performed PCR on mOrange and mOrange2 to prepare the gene blocks for insertion into pHISTEV
