<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Worldshaper-Wuhan_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Worldshaper-Wuhan/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="canvas"><DIV id="mobileNav" class="menu-open"><DIV class="wrapper"><NAV class="main-nav mobileNav"><UL><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan">Home</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/about-us">About Us</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Background">Background</A></LI><LI class="folder-collection folder"><LABEL for="mobile-folder-toggle-59916eb759cc681bb4584222" class="folder-toggle-label" onclick="" data-href="/project/">Project</LABEL></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/solution-and-design">Solution and Design</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Results">Results</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Model">Model</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Parts">Parts</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Demonstrate">Measurement</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/procedure">Procedure</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Notebook">Notebook</A></LI></UL></NAV></DIV><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Safety">Safety</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Human-Practice">Human Practice</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Collaborations">Collaboration</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/InterLab">InterLab</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Attributions">Attributions</A></LI><LI class="folder-collection folder"><LABEL for="mobile-folder-toggle-59926c70f7e0ab4ab234fd0e" class="folder-toggle-label" onclick="" data-href="/outside-support/">Awards</LABEL></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/HP/Gold_Integrated">Achievement</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Engagement">Survey Result</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Applied_Design">Applied Design</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/HP/Silver">Silver</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/gold">Gold</A></LI></DIV></DIV></DIV><DIV id="mobileMenuLink"><A>Menu</A></DIV><HEADER id="header" class="clearfix"><DIV class="site-info" data-content-field="business-information"><DIV class="site-address">Street Address</DIV><DIV class="site-city-state">City, State, Zip</DIV><DIV class="site-phone">Phone Number</DIV></DIV><DIV class="custom-info"><DIV class="sqs-layout sqs-grid-12 columns-12" data-layout-label="Header Subtitle: Custom Content" data-type="block-field" data-updated-on="1381761023501" id="customInfoBlock"><DIV class="row sqs-row"><DIV class="col sqs-col-12 span-12"><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-81eee4bbda3d9efd0834"><DIV class="sqs-block-content"><P class="text-align-center">Your Custom Text Here</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="topNav"><NAV class="main-nav" data-content-field="navigation"><UL><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan">Home</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/about-us">About</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Background">Background</A></LI><LI class="folder-collection folder"><A aria-haspopup="true" href="#" onclick="return false;">Project</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/solution-and-design">Solution and Design</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Results">Results</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Model">Model</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Parts">Parts</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Demonstrate">Measurement</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/procedure">Procedure</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Notebook">Notebook</A></LI></UL></NAV></DIV><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Safety">Safety</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Human-Practice">Human Practice</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Collaborations">Collaboration</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/InterLab">InterLab</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Attributions">Attributions</A></LI><LI class="folder-collection folder"><A aria-haspopup="true" href="#" onclick="return false;">Awards</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/HP/Gold_Integrated">Achievement</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Engagement">Survey Result</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/Applied_Design">Applied Design</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/HP/Silver">Silver</A></LI><LI class="page-collection"><A href="https://2017.igem.org/Team:Worldshaper-Wuhan/gold">Gold</A></LI></HEADER></DIV></DIV><DIV class="container"><SECTION id="page" class="clear" role="main"><DIV class="sqs-layout sqs-grid-12 columns-12" data-type="page"><DIV class="row sqs-row"><DIV class="col sqs-col-12 span-12"><DIV class="sqs-block html-block sqs-block-html"><DIV class="sqs-block-content"><H1 class="text-align-center">Result</H1></DIV><DIV class="sqs-block-content"><DIV class="row sqs-row" id="yui_3_17_2_1_1509572305608_121"><DIV class="col sqs-col-2 span-2"><DIV class="sqs-block spacer-block sqs-block-spacer sized vsize-1" data-block-type="21" id="block-1a5bf660f105f25ba7a9"><DIV class="sqs-block-content"> </DIV></DIV></DIV><DIV class="col sqs-col-9 span-9" id="yui_3_17_2_1_1509572305608_120"><DIV class="sqs-block spacer-block sqs-block-spacer sized vsize-1" data-block-type="21" id="block-4ade93f13ea914e436a9"><DIV class="sqs-block-content"> </DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-77a6b083575b3c254a6d"><DIV class="sqs-block-content"><P><STRONG>1.     Construction of the plasmids</STRONG></P><P>  We constructed five plasmids-pEGFP-miR-21-sponge-6s; pEGFP-miR-21-sponge-2s; pHAGE-pre-miR-21; pSB1C3-miR-21-sponge-2s and pSB1C3-miR-21-sponge-6s. The plasmids of pEGFP-miR-21-sponge-6s and pEGFP-miR-21-sponge-2s were used to monitor the expression of miR-21 in cells. We also used pHAGE-pre-miR-21 to manipulate miR-21 expression in cells. And the plasmids of pSB1C3-miR-21-sponge-6s and pSB1C3-miR-21-sponge-2s were submitted to 2017IGEM (part number: BBa_K2514000 and BBa_K2514001).
														</P><P><STRONG>1.1 Construction of pEGFP-miR-21-sponge-6s and pEGFP-miR-21-sponge-2s plasmid</STRONG></P><P>
															1.1.1 Amplification of miR-21-sponge-6s and miR-21-sponge-2s fragment.
														</P><P>
															To monitor the expression level of miR-21 in different cells, we made two plasmids of miR-21 sponges in the 3’UTR of GFP. Firstly, we designed miR-21 sponges contains six miR-21 binding sites with 3-nt spacers for bulged sites (pEGFP-miR-21-sponge-6s) and miR-21 sponges contains two miR-21 binding sites with 3-nt spacers for bulged sites (pEGFP-miR-21-sponge-2s) based on the sequence of hsa-miR-21 according to the previous study (Ebert MS, Neilson JR, Sharp PA. MicroRNA sponges: competitive inhibitors of small RNAs in mammalian cells. Nat Methods. 2007 Sep;4(9):721-6). Then we amplified miR-21-sponge-6s and miR-21-sponge-2s using primers. After that, we purified the PCR products by PCR Purification Kit and digested them with restriction enzymes Xho I and Hind III (Fig 1 A and B).
														</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_93_1509069313608_37436"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_119"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_118"><DIV class="intrinsic" style="max-width:468.0px;" id="yui_3_17_2_1_1509572305608_117"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 1. Electrophoresis of miR-21-sponge-6s (A: lanes 1-3) and miR-21-sponge-2s (B: lanes 4-6) PCR product.</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_93_1509069313608_37715"><DIV class="sqs-block-content"><P>1.1.2     Digested pEGFP-C1 vector</P><P>
														We know that microRNA can bind the 3 'UTR of genes to regulate the expression, so we choose EGFP to be the detector of miR-21. We digested the pEGFP-C1 vectors with restriction enzymes Xho I and Hind III (Fig 2).
													</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_93_1509069313608_68774"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_139"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_138"><DIV class="intrinsic" style="max-width:193.0px;" id="yui_3_17_2_1_1509572305608_137"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 2. Electrophoresis of the digested pEGFP-c1 vector.</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_93_1509069313608_69052"><DIV class="sqs-block-content"><P>
													1.1.3 Ligation of purified miR-21-sponge-6s and miR-21-sponge-2s fragments  to pEGFP-c1 vector
												</P><P>
													miR-21-sponge-6s fragments and miR-21-sponge-2s fragments were ligated to pEGFP-c1 vector, respectively. Then we selected the positive clones by PCR and sequencing (Fig 3 and 4)
												</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_17356"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_173"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_172"><DIV class="intrinsic" style="max-width:318.0px;" id="yui_3_17_2_1_1509572305608_171"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P>C</P><P><EM>
																	Fig 3. Electrophoresis of the PCR identifications of pEGFP- miR-21-sponge-6s (A: lane 1-3) and  pEGFP- miR-21-sponge-2s (B: lane 4-6) vector and sequencing map.
																</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_21117"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_190"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_189"><DIV class="intrinsic" style="max-width:193.0px;" id="yui_3_17_2_1_1509572305608_188"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
																Fig 4. Electrophoresis of pEGFP- miR-21-sponge-6s (lane 1,2) and  pEGFP- miR-21-sponge-2s (lane 3,4) vector.
															</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_94_1509069313608_12051"><DIV class="sqs-block-content"><P><STRONG>1.2     Construction of miR-21-</STRONG><STRONG>precursor</STRONG><STRONG>expression plasmid</STRONG></P><P>
											1.2.1      Amplification of miR-21- precursor fragment.
										</P><P> 
											To set up a model to determent the amount of miR-21 in cells, we made a construct to express miR-21.  We designed the specific primers of human miR-21-precursor based on the sequences of miR-21 precursor (
											<A href="http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=MI0000077">http://www.miRbase.org/cgi-bin/miRna_entry.pl?acc=MI0000077</A>
											) and amplified 72bp fragment by PCR. After that, we digested the purified PCR products with restriction enzymes Sal I and Not I (Fig 5).
										</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_88253"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_207"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_206"><DIV class="intrinsic" style="max-width:198.0px;" id="yui_3_17_2_1_1509572305608_205"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 5. Electrophoresis of miR-21-</EM>
														precursor
														<EM>PCR product. The lanes 1-3 showed the miR-21-</EM>
														precursor
														<EM>bands.</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_94_1509069313608_88531"><DIV class="sqs-block-content"><P>
										1.2.2  Digested pHAGE-puro vector
									</P><P>
										For
										<A href="file:///D:/Users/Administrator/AppData/Local/Youdao/Dict/7.5.0.0/resultui/dict/?keyword=eukaryotic">eukaryotic</A><A href="file:///D:/Users/Administrator/AppData/Local/Youdao/Dict/7.5.0.0/resultui/dict/?keyword=expression">expression</A>
										of has-miR-21 in cells, we digested the
										<A href="file:///D:/Users/Administrator/AppData/Local/Youdao/Dict/7.5.0.0/resultui/dict/?keyword=eukaryotic">eukaryotic</A><A href="file:///D:/Users/Administrator/AppData/Local/Youdao/Dict/7.5.0.0/resultui/dict/?keyword=expression">expression</A>
										vector pHAGE-puro with the same restriction enzymes Sal I and Not I  (Fig 6).
									</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_105946"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_224"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_223"><DIV class="intrinsic" style="max-width:198.0px;" id="yui_3_17_2_1_1509572305608_222"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 6. Electrophoresis of the digested pHAGE-puro vector.\</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_94_1509069313608_106224"><DIV class="sqs-block-content"><P>
									1.2.3 Ligation of purified miR-21-precursor fragment to pHAGE-puro vector
						</P><P>
									To sequence the miR-21-precursor cloning we got, the purified miR-21-precursor fragment was ligated to pHAGE-puro vector. Then we selected the positive clones by PCR and sequencing (Fig 7 and 8).
								</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_121180"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_241"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_240"><DIV class="intrinsic" style="max-width:192.0px;" id="yui_3_17_2_1_1509572305608_239"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P>A</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_139550"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_258"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_257"><DIV class="intrinsic" style="max-width:340.0px;" id="yui_3_17_2_1_1509572305608_256"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P>B</P><P><EM>
												Fig 7. Electrophoresis of the PCR identified production of pHAGE- pre-miR-21 vector (A) and sequencing map (B) .
											</EM></P><P> </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_94_1509069313608_178499"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_275"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_274"><DIV class="intrinsic" style="max-width:182.0px;" id="yui_3_17_2_1_1509572305608_273"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 8. Electrophoresis of pHAGE-pre-miR-21 vector.</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_94_1509069313608_178776"><DIV class="sqs-block-content"><P><STRONG>
							1.3  Construction of pSB-miR-21-sponge-6s and pSB-miR-21-sponge-2s plasmid
						</STRONG>
						1.3.1 Amplification of miR-21-sponge-6s and miR-21-sponge-2s fragment.
					</P><P>
						In order to submit the parts to IGM2017, we amplified miR-21-sponge-6s and miR-21-sponge-2s using primers. After that, we purified the PCR products by PCR Purification Kit and digested them with restriction enzymes EcoR I and Pst I (Fig 9 A and B).
					</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_19010"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_292"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_291"><DIV class="intrinsic" style="max-width:564.0px;" id="yui_3_17_2_1_1509572305608_290"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
										Fig 9. Electrophoresis of miR-21-sponge-6s (A: lanes 1-3) and miR-21-sponge-2s (B: lanes 4-6) PCR product.
									</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_4_1509162483656_19289"><DIV class="sqs-block-content"><P>1.3.2      Digested pSB1C3 vector</P><P>
					We digested the pSB1C3 vector with the same restriction enzymes EcoR I and Pst I  (Fig 10).
				</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_30874"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_309"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_308"><DIV class="intrinsic" style="max-width:268.0px;" id="yui_3_17_2_1_1509572305608_307"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 10. Electrophoresis of the digested pSB1C3 vector.</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_4_1509162483656_31152"><DIV class="sqs-block-content"><P>
				1.3.3      Ligation of purified miR-21-sponge-6s and miR-21-sponge-2s fragment  to pSB1C3 vector
			</P><P>
				We ligated the purified miR-21-sponge-6s or miR-21-sponge-2s fragment to pSB1C3 vector, respectively. Then we selected the positive clones by PCR and sequencing (Fig 11 and 12).
			</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_59772"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_360"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_359"><DIV class="intrinsic" style="max-width:259.0px;" id="yui_3_17_2_1_1509572305608_358"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
								Fig 12. Electrophoresis of pSB1C3- miR-21-sponge-6s (lane 1, 2) and  pSB1C3- miR-21-sponge-2s (lane 3, 4) vector.
							</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_4_1509162483656_60050"><DIV class="sqs-block-content"><P><STRONG>1.4</STRONG> The solid plates of each plasmid constructed (Fig 13)
		</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_75826"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_377"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_376"><DIV class="intrinsic" style="max-width:779.0px;" id="yui_3_17_2_1_1509572305608_375"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>Fig 13 Solid plates of five plasmids</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_4_1509162483656_76104"><DIV class="sqs-block-content"><P><STRONG>2. The effect of miR-21 sponges in cells</STRONG></P><P> To detect the validity of our detection device, we transfected pEGFP-miR-21-sponge-6s or pEGFP-miR-21-sponge-2s (0.25ug /well) with pHAGE-puro (0.25ug /well, as negative control) into 293T cells in 24 well-plate, respectively. At same times, we transfected pEGFP-miR-21-sponge-6s or pEGFP-miR-21-sponge-2s (0.25ug/well) with pHAGE-pre-miR-21 (0.25ug/well) into 293T cells. Then the GFP fluorescence was observed under fluorescence microscopy (Fig 14 A,B,C,D). The result suggested our detection device worked very well, and overexpression of miR-21 can decrease the fluorescence intensity of pEGFP-miR-21-sponge-6s and pEGFP-miR-21-sponge-2s.
	</P><P>
		And we also detected the fluorescence of GFP after transfection pEGFP-miR-21-sponge-6s or pEGFP- miR-21-sponge-2s with or without pHAGE-pre-miR-21 for 24 h by plate reader (SpectraMax i3), and observed that endogenous miR-21 in cells decreased the fluorescence of GFP (Fig 15). The fluorescence of GFP in cells transfect with pEGFP-miR-21-sponge-6s was lower in that of pEGFP-miR-21-sponge-2s, because pEGFP-miR-21-sponge-6s contains six miR-21 binding sites. These results suggested that sponges with six binding sites produced stronger derepressiveeffects than sponges with two binding sites.
	</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_86044"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_394"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_393"><DIV class="intrinsic" style="max-width:334.0px;" id="yui_3_17_2_1_1509572305608_392"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
						Fig 14.  The images of 293T cells transfected different plasmids A. pEGFP- miR-21-sponge-6s and pHAGE-puro co-transfected. B. pEGFP-miR-21-sponge-2s and pHAGE-puro co-transfected. C. pEGFP-miR-21-sponge-6s and pHAGE-pre-miR-21 co-transfected. D. pEGFP- miR-21-sponge-2s and pHAGE-pre-miR-21 co-transfected, Bar = 100mm .
					</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_100258"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_411"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_410"><DIV class="intrinsic" style="max-width:495.0px;" id="yui_3_17_2_1_1509572305608_409"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
					Fig 15. The fluorescence of GFP after co-transfection pEGFP-miR-21-sponge-6s or pEGFP- miR-21-sponge-2s with or without pHAGE-pre-miR-21 .
				</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_4_1509162483656_100536"><DIV class="sqs-block-content"><P><STRONG>
	2.1 The effect of miR-21 sponges as a monitor to detect the expression of miR-21
</STRONG></P><P>     To test the effect of miR-21 sponges as a monitor to detect the expression of miR-21, 293T cells were transfected with pEGFP-miR-21-sponge-6s or pEGFP-miR-21-sponge-2s with different concentration of pHAGE-pre-miR-21. We used the plasmid of pHAGE-pre-miR-21 to quantify miR-21 expression and calculate the copy numbers of miR-21 by using the formula listed below.
</P><P><STRONG><EM>     copies/ul= (6.02×1023)×(plasmids concentrations ng/ul×10-9)/(DNA length×660)
	</EM></STRONG></P><P>
The standard curve of miR-21-sponge-6s and miR-21-sponge-2s were made by EXCEL, respectively (Fig 16 and 17). We find that when miR-21 copies changed the slope of the standard curve of miR-21-sponge-2s was greater than miR-21-sponge-6s which indicated miR-21-sponge-2s is more sensitive than miR-21-sponge6s. However, the correlation coefficient (R2 value) of miR-21-sponge-6s was 0.9993 and R2 value of miR-21-sponge-6s was 0.8995. The R2 value of miR-21-sponge-6s is better than miR-21-sponge-2s.
</P></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_114255"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_428"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_427"><DIV class="intrinsic" style="max-width:401.0px;" id="yui_3_17_2_1_1509572305608_426"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
				Fig 16. The effect of pEGFP-miR-21-sponge-6s or pEGFP-miR-21-sponge-2s on the correlative between fluorescence value of GFP and copy numbers of miR-21.
			</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block image-block sqs-block-image sqs-text-ready" data-block-type="5" id="block-yui_3_17_2_4_1509162483656_124171"><DIV class="sqs-block-content" id="yui_3_17_2_1_1509572305608_445"><DIV class="image-block-outer-wrapper layout-caption-below design-layout-inline" id="yui_3_17_2_1_1509572305608_444"><DIV class="intrinsic" style="max-width:405.0px;" id="yui_3_17_2_1_1509572305608_443"><DIV class="image-caption-wrapper"><DIV class="image-caption"><P><EM>
			Fig 17. The standard curve of miR-21-sponge-6s and miR-21-sponge-2s.
		</EM></P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-yui_3_17_2_4_1509162483656_114533"><DIV class="sqs-block-content"><P><STRONG>Conclusions and perspectives</STRONG></P><P>
The results reported here demonstrate that miRNA sponge may be a monitor to detect the expression of miRNA in cells. And we also suggested the slope of the standard curve of miR-21-sponge-2s is better than miR-21-sponge-6s, suggesting the more sensitive of miR-21-sponge-2s as a monitor. Our results for the first time to evaluate the possibility of miRNA sponge for detecting miRNA and provide a new method for diagnosis miRNA in the future. However, there are still lots of issues relative to this new method that we need to consider. Firstly, we should evaluate the sensitivity and specificity of this technique by compare with other method including real-time PCR. Next, whether miRNA sponge contained two binding sites is the best monitor should be considered and tested. More strategies should be thought to improve the sensitivity and specificity of this technique by synthetic biology.
</P><P> </P><P> </P></DIV></DIV><DIV class="sqs-block spacer-block sqs-block-spacer sized vsize-1" data-block-type="21" id="block-264b5aac7b86608a98a6"><DIV class="sqs-block-content"> </DIV></DIV></DIV><DIV class="col sqs-col-1 span-1"><DIV class="sqs-block spacer-block sqs-block-spacer sized vsize-1" data-block-type="21" id="block-ddabbec28d6ec1ef4b6a"><DIV class="sqs-block-content"> </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV><FOOTER id="footer" class="clear"><DIV class="sqs-layout sqs-grid-12 columns-12" data-layout-label="Footer Content" data-type="block-field" data-updated-on="1502788432753" id="footerBlock"><DIV class="row sqs-row"><DIV class="col sqs-col-12 span-12"><DIV class="row sqs-row"><DIV class="col sqs-col-6 span-6"><DIV class="sqs-block html-block sqs-block-html" data-block-type="2" id="block-126febe67777dbcbc0b4"><DIV class="sqs-block-content"><P>Wuhan, China .   New York, USA</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>