<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="background-color:#FFFFFF; color:white"><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Development_of_a_Tailings_Pond_Clean_up_Kit"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Development of a Tailings Pond Clean up Kit</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Project_Description"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Project Description</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Background_and_Rationale"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Background and Rationale</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Catechol_as_the_entry_point_into_Escherichia_coli_metabolism"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Catechol as the entry point into Escherichia coli metabolism</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Compartmentalization_for_optimization"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Compartmentalization for optimization</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Removal_of_aqueous_heavy_metals"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Removal of aqueous heavy metals</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Accelerating_clay_particle_sedimentation"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Accelerating clay particle sedimentation</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Environmentally_safe_organisms"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Environmentally safe organisms</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#Objectives_of_Proposed_Research"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Objectives of Proposed Research</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Research_Design_and_Methods"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Research Design and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Significance_and_Future_Directions"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Significance and Future Directions</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Development_of_a_Tailings_Pond_Clean_up_Kit"><FONT color="black">Development of a Tailings Pond Clean up Kit</FONT></SPAN></H1><H2><SPAN class="mw-headline" id="Project_Description"><FONT color="black">Project Description</FONT></SPAN></H2><P>Managing byproducts of the extraction and refinement processes is a common problem in harvesting natural resources, such as oil. In most cases, tailings ponds are used for storing the toxic water byproducts, which not only have severe negative environmental impacts but also by using current methods can take decades before they can be reclaimed. The current remediation methods need to be improved to provide economical, effective and efficient processes to decrease the negative environmental impact of the tailings ponds. The tailings ponds contain toxic organic compounds, heavy metals and fine clay particles, all which require improved methods of treatment. We will produce a tailings pond clean up kit that uses environmentally safe methods to accelerate the decontamination of toxic organic molecules, heavy metals, and settle the fine clay particles at an increased rate. Toxic compounds will be degraded into metabolizable compounds at increased rates by using proteins that act within a common degradation pathway co-localized within a microcompartment in the form of an easily distributed dry powder. Removal of heavy metals, such as iron, from samples of tailings ponds water will be achieved by inducing the formation of iron nanoparticles, which can be removed together with the generated biomass. The rapid formation of fine clay sediments will be facilitated by the use of bacteria cell aggregates, increasing sedimentation rates from many decades to days or even hours. The kit will consist of either cell-free components or genetically modified organisms (GMO) that pose no threat to the environment as they will have been programmed with a method of rendering the cell inert and destroying its DNA once the desired action is completed. The methods within the tailings pond clean up kit will be applicable for large-scale treatment facilities as well as <I>in situ</I> tailings pond treatment.
</P><H2><SPAN class="mw-headline" id="Background_and_Rationale">Background and Rationale</SPAN></H2><P>Oil reserves are found in many corners of the globe and all nations must address the challenges of (I) extraction, (II) refinement, and (III) managing the byproducts of these processes. The byproducts are stored in the form of tailings ponds where the water contains (1) toxic organic compounds such as naphthenic acids and catechol, (2) heavy metals, and (3) fine clay sediment<SUP>1</SUP>. The tailings ponds raise critical environmental issues as the water cannot be reused and is harmful to organisms. Traditional methods used for tailings pond remediation can take decades to complete with the first reclaimed tailings pond only being completed in September 2010 after 40 years<SUP>2</SUP>.  Our goal is to address the challenge of decreasing the time necessary for tailings ponds remediation by creating a tailings pond clean up (TPCU) kit that can be applied (A) directly to the ponds or (B) in a large-scale treatment facility.
</P><H3><SPAN class="mw-headline" id="Catechol_as_the_entry_point_into_Escherichia_coli_metabolism">Catechol as the entry point into <I>Escherichia coli</I> metabolism</SPAN></H3><P>To address the decontamination of (1) toxic organic compounds such as naphthene and catechol found in the tailings ponds, natural pathways that can break down the toxic chemicals can be utilized. The toxic organic compounds can be broken down into molecules that are metabolizable by all organisms, such as 2-hydroxymuconic semialdehyde (2-HMS)<SUP>3</SUP>. Catechol is a prime candidate to function as a hub for the funneling and subsequent degradation of other toxic compounds found in abundance, such as naphthene, anthracene, and fluorene into 2-HMS (Figure 1)<SUP>3,4,5</SUP>. Catechol 2,3-dioxygenase (xylE) is found in <I>Pseudomonas putida</I> and hydrolyzes catechol into 2-HMS, which can enter the citric acid cycle<SUP>6</SUP>. In 2010, the Lethbridge iGEM team confirmed that xylE when introduce to <I>E. coli</I> hydrolyzes catechol to 2-HMS demonstrating and that this critical step can successfully be performed to convert toxic chemicals to metabolic intermediates<SUP>7</SUP>. XylE is inefficient on its own, especially <I>in vitro</I>, due to instability caused by oxygen sensitivity, and oxidation of the active site iron molecule during it’s functional cycle rendering the enzyme inactive<SUP>8</SUP>. It was demonstrated by the Lethbridge 2010 iGEM team that xylE only exhibits single turnover kinetics, and therefore, each protein molecule is capable of hydrolyzing only one molecule of catechol<SUP>7</SUP>. In natural systems, the <I>xylT</I> gene is located upstream of <I>xylE</I>. <I>XylT</I> codes for a ferredoxin protein (xylT) that stabilizes xylE by reducing the iron molecule back to its previous active state, allowing xylE to continue degrading catechol<SUP>9</SUP>. Creating a multiple turnover system would allow one protein to degrade multiple catechol molecules thereby increasing the efficiency of the system while increasing potential real-world applications. We have performed a computational metabolic flux analysis to determine possible bottlenecks and choke points in the feeding of 2-HMS into the citric acid cycle. The results indicated that the bottleneck in the pathway is not located in the citric acid cycle but in the breakdown of 2-HMS by the native <I>E. coli</I> enzyme xylF, indicating another step that can be used to optimize the catechol degradation system (Figure 1)<SUP>10</SUP>.
</P><P><FONT color="black">Figure 1.  Schematic of the tailings pond toxic organic molecule degradation pathways and their entry into <I>E. coli’s</I> natural metabolic pathways. Fluorene, anthracene and naphthalene can be degraded into catechol using a series of enzymatic steps (orange). Catechol can be degraded to 2-HMS by xylE, which can enter the citric acid cycle (green). Alternatively, catechol-1,2-dioxygenase can degrade catechol into cis,cis-muconate that can enter the citric acid cycle following a series of 5 enzymatic steps (red). The bottleneck as demonstrated by computational metabolic flux analysis is located in the three-step breakdown of 2-HMS to acetaldehyde and pyruvate and is the xylF enzyme. The blue box outlines the location we will optimize for catechol degradation, the entry point of toxic organic compound degradation into the cells’ natural metabolic pathways. Orange outlines the pathways we are investigating for breakdown of the more toxic organic molecules found in the tailings ponds.<SUP>3,4,5,6,10,24</SUP></FONT></P><H3><SPAN class="mw-headline" id="Compartmentalization_for_optimization">Compartmentalization for optimization</SPAN></H3><P>Compartmentalization is a method that can serve to optimize systems and can be applied to the decontamination of toxins from the tailings ponds. Localizing proteins of a pathway together will decrease cross-talk with other systems and concentrate the proteins of interest in close proximity. Lumazine synthase (LS) from <I>Aquifex aeolicus</I> forms icosahedral microcompartment (MC) assemblies of 60 or 180 monomeric units that self assemble and are capable of isolating proteins from their local environment<SUP>11</SUP>. As shown in previously published work, the LS protein has been mutated so that the interior of the MC is negatively charged, and the Lethbridge 2009 iGEM team has submitted the mutated LS gene to the parts registry<SUP>11</SUP>. A negatively charged interior allows for preferential compartmentalization of positively charged molecules, which can easily be engineered through the addition of a poly-arginine tag to a target protein<SUP>11</SUP>. The poly-arginine C- and N-terminal tags have also been submitted to the registry by the Lethbridge 2009 iGEM team. Recent work on LS using directed evolution to optimize the interior surface of the MC created a new variant of LS that allows for the increased loading capacity of the MC<SUP>12</SUP>. It has been found that this LS variant can successfully encapsulate toxic compounds within <I>E. coli</I> cells without nonspecific association of contaminants or target molecules to the exterior of the MC<SUP>12</SUP>. This LS variant will be advantageous for selective compartmentalization of larger or multiple target proteins involved in catechol degradation. Compartmentalization will also allow for the introduction of pathways that may contain toxic intermediates to the chassis, while keeping them separate from the chassis’ other processes.
</P><H3><SPAN class="mw-headline" id="Removal_of_aqueous_heavy_metals">Removal of aqueous heavy metals</SPAN></H3><P>Decontamination of (2) heavy metals found in the tailings ponds can be performed by the Mms6 protein found in <I>Magnetospirillum magneticum</I> that produces nanoparticles<SUP>13,14</SUP>. Mms6 can facilitate the removal of iron from tailings ponds water through the formation of symmetric octahedral iron crystal nanoparticles<SUP>15</SUP>. The production of nanoparticles from soluble heavy metals will effectively remove the heavy metals from the tailings ponds with subsequent removal of the magnetic nanoparticles using simple methods such as a magnet<SUP>13</SUP>. Previous work done by the Lethbridge 2010 iGEM team on the Mms6 protein demonstrated that the expression of Mms6 induced low cell growth rates and low expression levels, indicating that the protein is toxic to the <I>E. coli</I> chassis<SUP>16</SUP>. A method to circumvent the toxic protein problem for the chassis is to have the protein secreted from the cell to prevent accumulation within the cell.
</P><H3><SPAN class="mw-headline" id="Accelerating_clay_particle_sedimentation">Accelerating clay particle sedimentation</SPAN></H3><P>Without intervention, the (3) fine clay sediment found in the tailings ponds can take decades to solidify, but with the application of molecules such as polysaccharides that are found on the surface of bacterial membranes, the clay particles can be aggregated<SUP>17,18</SUP>. Once the aggregates of cell and clay have been formed, sediments can be formed by using an antigen that promotes rapid bacterial aggregation<SUP>19</SUP>. The aggregates will then settle out, resulting in clearer water. The Peking 2010 iGEM team successfully characterized Antigen 43 (Ag43) that successfully formed sediments 20 minutes after induction<SUP>20</SUP>. The Ag43 was used to sediment the chassis after absorbing mercury from a solution.
</P><H3><SPAN class="mw-headline" id="Environmentally_safe_organisms">Environmentally safe organisms</SPAN></H3><P>To prevent environmental contamination by any GMO’s DNA, measures must be taken. One method is by expressing an endonuclease within the chassis causing the genomic DNA to be degraded and therefore removing the risk of the GMO’s DNA being released into the environment. This method is especially appealing because the potential of horizontal gene transfer between other species present in the tailings ponds is eliminated, while still ensuring relevant enzymes will not be destroyed. The UC Berkley 2007 iGEM team successfully characterized an endonuclease that degraded the genomic DNA of <I>E. coli</I> when induced<SUP>21</SUP>. The ability to induce the endonuclease will allow for the fully-functional organism to carry out the mechanism for which it is designed, whether that be (1) degradation of organic molecules, (2) decontamination of heavy metals, or (3) sedimentation of clay particles, before the organism is rendered inert by the degradation of genomic DNA. 
</P><H2><SPAN class="mw-headline" id="Objectives_of_Proposed_Research">Objectives of Proposed Research</SPAN></H2><P>The overall goal of the project is to produce a prototype tailings ponds clean up kit (Figure 2). The tailings ponds clean up kit will decontaminate (1) toxic organic molecules, (2) heavy metals, (3) remove the clay sediment, and (4) prevent GMO’s DNA contamination of the environment. The kit will allow for an efficient and environmentally friendly method of tailings ponds remediation. The methods used will also be applicable for large-scale treatment facilities to treat the tailings water before it is deposited in ponds thereby decreasing their negative environmental impact. 
<B>(1)	Our aim is to produce optimized enzymes found in nature that will decontaminate toxic organic molecules found in the tailings ponds.</B> In order to do so, we will further characterize and optimize the project started by the Lethbridge 2010 iGEM team that focused on converting toxins in the tailings ponds into compounds that cells can metabolize. To optimize the system, we will introduce additional enzymes that will improve the efficiency of degradation and increase the degradation capabilities of the current toxin-degrading enzyme, based on our metabolic flux analysis. Further optimization will be performed by isolating the new two-enzyme system in compartments that will prevent their interaction with other naturally occurring systems and allow for their isolation and application to the tailings ponds.
<B>(2)	Our aim is to use enzymes already present in nature for removing the heavy metals found in tailings ponds by for example, magnetic nanoparticle formation.</B> We will do this by working with the enzyme submitted by the Lethbridge 2009 iGEM team and optimizing its production conditions to allow for maximal removal of toxic heavy metals.
<B>(3)	Our aim is to rapidly form sediment composed of the fine clay particles found in the tailings ponds by using the natural properties of bacterial cells to form aggregates.</B> The engineered bacterial cell will have the properties required to attach to and form sediment with the fine clay particles found in the tailings ponds. This system will allow for rapid clay particle removal from the tailings ponds by increasing the rate of sedimentation of the clay particles by adding biomass and aggregations <I>in situ</I> or in a treatment facility.
<B>(4)    Our aim is to produce an environmentally safe tailings pond clean up kit by ensuring all components do not contain the GMO’s DNA, which could potentially have an adverse affect on the environment.</B> We will achieve this by including a naturally occurring enzyme within the bacterial cells that will degrade the DNA when triggered. This property will make the kit environmentally safe by ensuring that the potentially harmful GMO’s DNA will not reach the environment and risk contamination.
</P><P><FONT color="black">Figure 2. Schematic of the environmentally safe tailings ponds clean up kit. The kit will contain the materials necessary for (1) toxic organic molecule degradation, (2) heavy metals decontamination, (3) removal of fine clay particles, and (4) degradation of GMO’s DNA.
</FONT></P><H2><SPAN class="mw-headline" id="Research_Design_and_Methods">Research Design and Methods</SPAN></H2><P><B>(1)    Degradation of toxic organic molecules found within the tailings ponds will be achieved by optimizing the system characterized by the Lethbridge 2010 iGEM team.</B>The Lethbridge 2010 iGEM team demonstrated that the degradation of catechol to 2-HMS by xylE is successful using <I>E. coli</I> as the chassis<SUP>7</SUP>. The same spectroscopy methods used to characterize xylE will be used to detect the production of 2-HMS – an easily detectable compound due to its yellow colour – in the presence or absence of xylT as a measure of catechol degradation. To further optimize the xylE/xylT system, compartmentalization of the proteins into biological MCs will also be performed. Close proximity of xylE and xylT within the MC will facilitate rapid reactivation of xylE by xylT, increasing the efficiency of catechol degradation. 

The protein, lumazine synthase (LS), will be used for compartmentalization of xylE and xylT. To target xylE and xylT into the negative interior of the LS MC, a positively charged poly-arginine tag will be used. However, the positive tag may impair the function of the target protein, xylE or xylT, depending on its position at the N-terminus or C-terminus. It has been demonstrated that adding a poly-arginine tag to the C-terminus of a protein does not affect its function, while an N-terminal tag may significantly reduce a protein’s activity22. The Lethbridge 2010 iGEM team confirmed this by individually tagging the N- and C-termini of both cyan and yellow fluorescent protein (CFP and YFP, respectively) and demonstrated that the N-terminally tagged proteins were inactive. Together, CFP and YFP are a well known fluorescence resonance energy transfer (FRET) pair that will be used as a proof-of-principle demonstrating successful co-localization in the LS MC. FRET is a distance dependent phenomenon that involves the transfer of energy from an excited donor fluorophore to an acceptor fluorophore and results in a decrease in the fluorescence signal from the donor coupled to an increase in the fluorescence signal from the acceptor when the fluorophores are in close proximity. 

With successful co-localization demonstrated using the CFP/YFP FRET system, optimized conditions will be used to co-localize xylE and xylT within the MC. BioBricks for xylE, xylT, and both N- and C-terminal poly-arginine tagged xylE and xylT will be assembled into constructs, tested, and submitted to the parts registry. The activity of the tagged xylE and xylT proteins will be compared to that of the wild type xylE and xylT, respectively, using spectroscopy methods. A final BioBrick construct containing LS and tagged versions of xylE and xylT will be assembled to test localization of the proteins within the MC. A simple assay to verify co-localization will be to use gradient centrifugation to isolate the LS MC followed by SDS-PAGE with markers for LS, xylE, and xylT. Co-localization will be successful if all three proteins of the LS MC purification are visualized by SDS-PAGE and extracellular application of the MC will also be tested.
<B>(2)    Decontamination of heavy metals found in the tailings ponds will be performed using tools already found in nature.</B> The bacteria <I>M. magneticum</I> produce the protein Mms6, which possesses  the ability to reduce heavy metals into magnetic nanoparticles<SUP>23</SUP>. Further work will now be done to characterize the formation of the magnetic nanoparticle from the Mms6 BioBrick found in the registry. Expression conditions of Mms6 in vivo will be optimized, exploring conditions such as iron concentration and Mms6 cellular localization. To investigate the effect of Mms6 over-expression in different cellular locations, signal sequences can be used to facilitate targeting of Mms6 to the periplasm or its secretion out of the cell. Four signal sequences are available in the registry that were submitted by the Lethbridge 2010 iGEM team and will be added to the <I>mms6</I> gene construct to control localization of the over-expressed protein product. Cell growth rates are easily tracked through absorbance measures of cell cultures, and over-expression of Mms6 can be verified by SDS-PAGE. Since cell growth rates can be slowed when under metal ion stress, removing aqueous metals via nanoparticle formation will have an added benefit of improving cell viability while decreasing heavy metal ion stress in the tailings ponds.
<B>(3)    Sedimentation of clay particles found in the tailings ponds can be accelerated by the use of naturally occurring polysaccharides found on the surface of bacterial cell membranes that bind clay particles and form aggregates<SUP>18</SUP>.</B> The cell-clay aggregates can then form sediments using Antigen 43 (Ag43). Ag43 is an autotransporter that promotes bacterial cell-to-cell aggregation, which accelerates sedimentation rates<SUP>19</SUP>. The Peking 2010 iGEM team successfully characterized Ag43 in the <I>E. coli</I> chassis. We will take advantage of native <I>E. coli</I> surface polysaccharides’ clay-binding properties and use them in conjunction with Ag43 to efficiently settle clay particles more rapidly than the current conventional methods. The team will introduce Ag43 into <I>E. coli</I> and apply the bacteria to a clay solution. The bacteria will aggregate the clay and form sediment at the bottom of the container and the removal of clay will be observed by monitoring the optical density of the solution. This experimental parameter can be measured using spectrophotometric methods and changes in optical density over time of the solution will indicate the efficiency of the auto-aggregation system.
<B>(4)    To prevent contamination of the environment with GMO’s DNA we will utilize natural methods of DNA degradation to degrade the genomic DNA of our chassis to ensure that our tailings pond clean up kit will not contaminate the environment with DNA.</B> We will utilize the part characterized by the UC Berkeley 2007 iGEM team that degrades the genomic DNA of the organism. The gene, <I>BamH</I>I, codes for a restriction endonuclease that effectively degrades the genomic material of the cell when induced, while having no effect on the function of other proteins within the cell<SUP>21</SUP>. Thus, the expression of this gene will allow for the pathways we introduce to the <I>E. coli</I> chassis to remain functional while removing the risk of DNA contamination. By having the ability to destroy the chassis’ DNA on command, we gain tight control over regulating the engineered bacterial cells. We will be able to degrade the genomic DNA and create an inert organism unable to adversely affect the environment and ensure that all parts of the kit are free of GMO DNA. The viability of the chassis once the DNA has been degraded will be tested under different conditions to determine the best conditions for use.
</P><H2><SPAN class="mw-headline" id="Significance_and_Future_Directions">Significance and Future Directions</SPAN></H2><P>Production of an environmentally safe tailings pond clean up kit, which contains methods applicable to large-scale treatment facilities, has multiple advantages. Use of the kit will both accelerate remediation of current tailings ponds in addition to providing methods of water treatment directly after industrial use. The kit will be an environmentally safe prototype that contains biological components for tailings ponds decontamination of (1) toxic organic molecules, (2) heavy metals, and (3) fine clay particles. Future work on the project will include extending the types of toxic organic molecules that it is capable of degrading to molecules that are metabolized by cells (Figure 1). These molecules include naphthene, anthracene, and fluorene, which can be further degraded to catechol. The ability to degrade these chemicals to catechol will complement the prototype kit and increase its effectiveness at tailings pond remediation. The current heavy metals we are able to target for decontamination are iron and cobalt. We will target additional metals by using <I>in vitro</I> evolution of the Mms6 enzyme to change its specificity for different heavy metals. In addition, optimization of the kit for different stages of tailings pond remediation can be performed. Kit optimization for different tailings pond stages and conditions will allow for universal use in a variety of tailings ponds found in every corner of the globe.
</P><H2><SPAN class="mw-headline" id="References"><FONT color="black">References</FONT></SPAN></H2><P>1.  Kato, T., Haruki, M., Imanaka, T., Morikawa, M., and Kanaya, S. (2001). Isolation and characterization of psychotrophic bacteria from oil-reservoir water and oil sands. Applied Microbiol Biotechnology. 55: 794-800.

2.  Suncor Energy A. (2011, May 10). Oil Sands Tailings. Retrieved from <A href="http://sustainability.suncor.com/2010/en/responsible/3508.aspx" class="external free" rel="nofollow">http://sustainability.suncor.com/2010/en/responsible/3508.aspx</A>
3.  Austen, R., and Dunn, N. (1980). Regulation of the plasmid-specified naphthalene catabolic pathway of <I>Pseudomonas putida</I>. Journal of General Microbiology. 117: 521-528. 

4.  Timoney, K., and Lee, P. (2009). Does the Alberta tar sands industry pollute? The scientific evidence. The Open Conservation Biology Journal. 3: 65-81.

5.  SalmonellaBase (2011, May 10). Benzoate Degradation via Hydroxylation. Retrieved from http://salmonellabase.com/admin/pathwayimages/benzoate%20degradation%20via%20hydroxylation.png

6.  Gomes, N., Kosheleva, I., Abraham, W.-R., and Smalla, K. (2005). Effects of the inoculant strain <I>Pseudomonas putida</I> KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community. FEMS Microbiology Ecology. 54 (1): 21–33.

7.  iGEM 2010 A (2011, May 10). Catechol Degradation. Retrieved from <A href="https://2010.igem.org/Team:Lethbridge/Results#Catechol_Degradation" class="external free" rel="nofollow">https://2010.igem.org/Team:Lethbridge/Results#Catechol_Degradation</A>
8.  Hugo, N., Armengaud, J., Gaillard, J., Timmis K., and Jouanneau Y. (1998). A novel [2Fe-2S] ferredoxin from <I>Pseudomonas putida</I> mt2 promotes the reductive reactivation of catechol 2,3-dioxygenase. Journal of Biological Chemistry. 273(16): 9622-9629.

9.  Polissi, A., and Harayama, S. (1993). In vivo reactivation of catechol 2,3-dioxygenase mediated by a chloroplast-type ferredoxin: a bacterial strategy to expand the substrate specificity of aromatic degradative pathways. EMBO J., 12(8); 3339-3347.

10.  Diaz, E., and Timmis, K. (1995). Identification of functional residues in 2-hydroxymuconic semialdehyde hydrolase. Journal of Biological Chemistry. 270(1): 6403-6411.

11.  Seebeck, F., Woycechowsky, K., Zhuang, W., Rabe, J., and Hilvert, D. (2006). A simple tagging system for protein encapsulation. Journal of the American Chemical Society. 128: 4516-4517.

12.  Wörsdörfer, B., Woycechowsky, K., and Hilvert, D. (2011). Directed evolution of a protein container. Science. 331(6017): 589-592.

13.  Arakaki, A., Webb, J., and Matsunaga, T. (2003). A novel protein tightly bound to bacterial magnetic particles in <I>Magnetospirillum magneticum</I> Strain AMB-1. Journal of Biological Chemistry. 278: 8745-50.

14.  Schleifer, K., and Ludwig, W. (1991). The genus magnetospirillum gen-nov – description of magnetospirillum-gryphiswaldense sp-nov and trandfer of aquaspirillum-magnetotacticum to magnetospirillum-magnetotacticum comb-nov. Systematic and Applied Microbiology 14: 379-85.

15.  Tanaka, M., Mazuyama, E., Arakaki, A., and Matsunaga, T. (2011). Mms6 protein regulates crystal morphology during nano-sized magnetite biomineralization in vivo. Journal of Biological Chemistry. 286: 6386-6392.

16.  iGEM 2010 B (2011, May 10). Magnetic Nanoparticles Parts. Retrieved from <A href="https://2010.igem.org/Team:Lethbridge/Results#Magnetic_Nanoparticles_Parts" class="external free" rel="nofollow">https://2010.igem.org/Team:Lethbridge/Results#Magnetic_Nanoparticles_Parts</A>
17.  Suncor Energy B. (2011, May 10). Oil Sands Tailings. Retrieved from <A href="http://sustainability.suncor.com/2010/en/responsible/1779.aspx" class="external free" rel="nofollow">http://sustainability.suncor.com/2010/en/responsible/1779.aspx</A>
18.  Dorioz, J.M., Robert, M., and Chenu, C. (1993). The role of roots, fungi and bacteria on clay particle organization. An experimental approach. Geoderma. 56: 179–194.

19.  van der Woude, M., and Henderson, I. (2008). Regulation and function of Ag43 (Flu). Annual Review of Microbiology. 62:153-169.

20.  iGEM 2010 C (2011, May 10). Parts Characterization. Retrieved from <A href="https://2010.igem.org/Team:Peking/Parts/Characterization" class="external free" rel="nofollow">https://2010.igem.org/Team:Peking/Parts/Characterization</A>
21.  iGEM 2007. (2011, May 10). BerkiGEM2007Present5. Retrieved from <A href="https://2007.igem.org/BerkiGEM2007Present5" class="external free" rel="nofollow">https://2007.igem.org/BerkiGEM2007Present5</A>
22.  Bachmair, A., Finley, D., and Varshavsky, A. (1986). In vivo half-life of a protein is a function of its amino-terminal residue. Science 234(4773); 179-186.

23.  Arakaki, A., Masuda, F., Amemiya, Y., Tanaka, T., Matsunaga, T. (2010). Control of the morphology and size of magnetite particles with peptides mimicking the Mms6 protein from magnetotactic bacteria. Journal of Colloid and Interface Science. 343:65-70.
</P></BLOCKQUOTE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Lethbridge/Project">http://2011.igem.org/Team:Lethbridge/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/Project&amp;oldid=263237" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>