<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Lethbridge_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Lethbridge/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Lethbridge/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Lethbridge/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Lethbridge/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="background-color:#FFFFFF; color:white"><BLOCKQUOTE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Part:BBa_K542008_-_Lumazine_Synthase_Microcompartment_Device"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Part:BBa_K542008 - Lumazine Synthase Microcompartment Device</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Construct"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Construct</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#Characterization"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Electron_Microscopy"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Electron Microscopy</SPAN></A></LI><LI class="toclevel-4 tocsection-6"><A href="#Materials_and_Methods"><SPAN class="tocnumber">1.2.1.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#Results"><SPAN class="tocnumber">1.2.1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#Discussion_and_Conclusion"><SPAN class="tocnumber">1.2.1.3</SPAN><SPAN class="toctext">Discussion and Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Fluorescence_Microscopy"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Fluorescence Microscopy</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Materials_and_Methods_2"><SPAN class="tocnumber">1.2.2.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Results_2"><SPAN class="tocnumber">1.2.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-12"><A href="#Conclusion"><SPAN class="tocnumber">1.2.2.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Scanning_Electron_Microscopy"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Scanning Electron Microscopy</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Fluorescent_Resonance_Energy_Transfer_.28FRET.29"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Fluorescent Resonance Energy Transfer (FRET)</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#Materials_and_Methods_3"><SPAN class="tocnumber">1.2.4.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-16"><A href="#Results_3"><SPAN class="tocnumber">1.2.4.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-17"><A href="#Conclusion_2"><SPAN class="tocnumber">1.2.4.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-4 tocsection-18"><A href="#Discussion"><SPAN class="tocnumber">1.2.4.4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#References"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Part:_BBa_K542010_-_Enhanced_Lumazine_Synthase_.28ELS.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Part: BBa_K542010 - Enhanced Lumazine Synthase (ELS)</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Introduction_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Characterization_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Overexpression"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Overexpression</SPAN></A></LI><LI class="toclevel-4 tocsection-24"><A href="#Materials_and_Methods_4"><SPAN class="tocnumber">2.2.1.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-25"><A href="#Results_4"><SPAN class="tocnumber">2.2.1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-26"><A href="#Conclusion_3"><SPAN class="tocnumber">2.2.1.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Protein_Purification"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Protein Purification</SPAN></A></LI><LI class="toclevel-4 tocsection-28"><A href="#Materials_and_Methods_5"><SPAN class="tocnumber">2.2.2.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-29"><A href="#Results_5"><SPAN class="tocnumber">2.2.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-30"><A href="#Conclusion_4"><SPAN class="tocnumber">2.2.2.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Transmission_Electron_Microscopy"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Transmission Electron Microscopy</SPAN></A></LI><LI class="toclevel-4 tocsection-32"><A href="#Materials_and_Methods_6"><SPAN class="tocnumber">2.2.3.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-33"><A href="#Results_6"><SPAN class="tocnumber">2.2.3.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-4 tocsection-34"><A href="#Conclusion_5"><SPAN class="tocnumber">2.2.3.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#References_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#Part:_BBa_K542009_.E2.80.93_Antigen_43"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Part: BBa_K542009 – Antigen 43</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Introduction_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Construct_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Construct</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Sedimentation_by_Antigen_43"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Sedimentation by Antigen 43</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Materials_and_Methods_7"><SPAN class="tocnumber">3.3.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Results_7"><SPAN class="tocnumber">3.3.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Conclusion_6"><SPAN class="tocnumber">3.3.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-43"><A href="#Part:_BBa_I716462_.E2.80.93_BamHI"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Part: BBa_I716462 – BamHI</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#Introduction_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#Characterization_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Characterization</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#Cell_Survival"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Cell Survival</SPAN></A></LI><LI class="toclevel-4 tocsection-47"><A href="#Materials_and_Methods_8"><SPAN class="tocnumber">4.2.1.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-48"><A href="#Results_8"><SPAN class="tocnumber">4.2.1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-49"><A href="#Analysis_of_Genomic_DNA"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Analysis of Genomic DNA</SPAN></A></LI><LI class="toclevel-4 tocsection-50"><A href="#Materials_and_Methods_9"><SPAN class="tocnumber">4.2.2.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-4 tocsection-51"><A href="#Results_9"><SPAN class="tocnumber">4.2.2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#Conclusion_7"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#References_3"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-54"><A href="#Collaboration_-_Cell_Viability_in_Tailings_Pond_Water"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Collaboration - Cell Viability in Tailings Pond Water</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#Materials_and_Methods_10"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#Results_10"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#Conclusion_8"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Part:BBa_K542008_-_Lumazine_Synthase_Microcompartment_Device"><FONT>Part:BBa_K542008 - Lumazine Synthase Microcompartment Device</FONT></SPAN></H1><P><FONT color="black">Datasheet for <A href="http://partsregistry.org/Part:BBa_K542008"><FONT color="blue">Part BBa_K542008</FONT></A> in <I>E. coli</I> strain DH5alpha. 
</FONT></P><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>The intended purpose of the Lumazine Synthase microcompartment device is to specifically target proteins that have been tagged with a positively charged oligopeptide into the cavity formed by the microcompartment. Furthermore, multiple unique proteins may be targeted into the cavity (1) for the purposes of increasing the efficiency of a metabolic pathway, to name just one example. This test construct is designed to demonstrate that proteins with positively charged tags can be localized into the cavity of the microcompartment formed by the oligomerization of Lumazine Synthase monomers. 

The expression of Lumazine Synthase and the fluorescent proteins are independently regulated by two separate promoters. Lumazine Synthase is regulated by the pLacI promoter and the fluorescent proteins are regulated by the pBAD inverter. Since the two are independently controlled, Lumazine Synthase microcompartments may be formed in the presence or absence of the fluorescent proteins (ie. absence or presence of arabinose, respectively).

Because the fluorescent proteins are tagged with a positively-charged poly-arginine tag and the Lumazine Synthase is mutated with a negative interior (BBa_K249002 and Lethbridge 2009 Modeling), enhanced cyan fluorescent protein (ECFP) and enhanced yellow fluorescent protein (EYFP) should be targeted into the microcompartments. ECFP and EYFP are a well known FRET pair.
</P><H3><SPAN class="mw-headline" id="Construct">Construct</SPAN></H3><P>Part Number BBa_K542008

The construct was made by assembling BBa_K542004 with BBa_K542005. 
</P><P><B>Figure 1.</B>    BBa_K542008. This construct contains pLacI Regulated Lumazine Synthase and pBAD Inverse-Regulated Arg-tagged ECFP and EYFP.
</P><H2><SPAN class="mw-headline" id="Characterization">Characterization</SPAN></H2><H3><SPAN class="mw-headline" id="Electron_Microscopy">Electron Microscopy</SPAN></H3><P><I>E. coli</I> DH5α cells expressing BBa_K542008 and controls were viewed using Transmission Electron Microscopy (TEM). 
</P><H4><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H4><P><I>E. coli</I> DH5α cells containing BBa_K542008 as well as control cultures containing only plasmid pSB1C3 were grown overnight at 37°C. Approximately 400 mL of these cells were then resuspended in 5 mL SOC media, and induced with 5 mL or 10 mL of 1 M IPTG. Cells were again incubated at 37°C for approximately 4 hours. Cells were then spun down at 6000 x g for 2 min. 

Cells were resuspended in 500 mL gluteraldehyde/formaldehyde fixative and incubated at room temperature for 2 hours. The solution was then spun at 6000 x g for 2 min. 

The cell pellet was resuspended in 0.1 M sodium cacodylate buffer and incubated for 10 min. The solution was centrifuged for 2 min at 6000 x g. This step was repeated an additional 2 times. Cells were then resuspended in 1% osmium tetroxide fixative in 0.1 M CaC buffer and incubated at room temperature for 1 hour. The solution was centrifuged at 6000 x g for 2 min. Cells were then resuspended in MilliQ H2O. 

The cells were spun down at 10000 x g for 5 min, and resuspended in 4% BIO-RAD standard low melting temperature agarose at 60°C. A drop of this cell-agarose mixture was then added to slides in order to harden, and incubated in MilliQ H<SUB>2</SUB>O overnight. 

Agarose-cell mixtures were cut into cubes of approximately 1 mm x 1 mm x 1 mm. They were then incubated in increasing alcohol concentrations up to 100% anhydrous ethyl alcohol. The cubes were then incubated in decreasing concentrations of alcohol and increasing concentrations of resin, up to 100% resin. The cubes were left in 100% resin to harden overnight. 

The resin-embedded cubes of cells were thin-sectioned and placed on a carbon grid. They were stained with uranyl acetate and then positively stained with lead citrate and viewed under TEM.  
</P><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P><B>Figure 2.</B><I>E. coli</I> DH5α control cells (left) and cells expressing BBa_K542008 (right) as viewed under transmission electron microscope. 
</P><P><B>Figure 3.</B><I>E. coli</I> DH5α cells expressing BBa_K542008 as viewed under transmission electron microscope. 
</P><H4><SPAN class="mw-headline" id="Discussion_and_Conclusion"> Discussion and Conclusion</SPAN></H4><P>We found that while the controls had a fairly uniform interior, the lumazine expressing cells did not (Figure 2). In the lumazine cells there were homogenous areas of grey as well as darker spots, whereas the controls contained a more uniform interior. 

We hypothesized that the lumazine accumulated together, which caused the isolation of other cellular elements. If this is the case, the microcompartments should most likely be the homogenous areas of grey not seen in the controls, and can’t be distinguished by contrast staining because they are aggregated together. 

To see if this is the case we will look at cells expressing BBa_K542008 under the spectral confocal (fluorescence) microscope. We hypothesize that if the microcompartments are segregated from other cellular elements then the fluorescent proteins will also be segregated and therefore we should see areas within the cell with brighter fluorescence.
</P><H3><SPAN class="mw-headline" id="Fluorescence_Microscopy">Fluorescence Microscopy</SPAN></H3><P><I>E. coli</I> DH5α cells containing BBa_K542008 were viewed with fluorescence microscopy. This was done in order to determine if the distribution of fluorescent proteins within the cells was similar to the homogenous grey regions seen in the cells under TEM. 
</P><H4><SPAN class="mw-headline" id="Materials_and_Methods_2">Materials and Methods</SPAN></H4><P><I>E. coli</I> DH5α cells containing the BBa_K542008 construct were grown overnight at 37°C. They were spun down at 10,000 rpm and then resuspended in 1X phosphate buffer saline (PBS). The solution was centrifuged at 10,000 rpm, decanted and resuspended in 4% paraformaldehyde (PFA) overnight. This solution was then centrifuged at 10000 rpm and cells were resuspended in glycerol. The glycerol-cell mixtures were then placed on slides, coverslipped and sealed. 

The slides were viewed with an Olympus FV1000 spectral confocal at 60X magnification. Pictures were taken using the ECFP and EYFP filters. 
</P><H4><SPAN class="mw-headline" id="Results_2">Results</SPAN></H4><P><B>Figure 4.</B><I>E. coli</I> DH5a cells containing the BBa_K542008 construct as viewed under spectral confocal microscope with 60X magnification. The left picture corresponds to the ECFP filter and the right picture corresponds to the EYFP filter.
</P><H4><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H4><P>Figure 4 shows that the fluorescent proteins were expressed in the cells. It is also evident that the fluorescence was segmented in a way that looks similar to that found in the homogenous grey areas in the TEM micrographs of the cells.
</P><H3><SPAN class="mw-headline" id="Scanning_Electron_Microscopy">Scanning Electron Microscopy</SPAN></H3><P>We attempted to view lumazine synthase microcompartments under scanning electron microscopy. However, because our microcompartments were in solution, the moisture remaining caused the electron beam to heat the sample. This destroyed the entire sample, including the carbon material it was placed on.
</P><P><B>Figure 1.</B>    Scanning electron micrograph of lumazine synthase. The microcompartments are not visible due to the electron beam destroying the sample, as seen in the above dark spots.
</P><H3><SPAN class="mw-headline" id="Fluorescent_Resonance_Energy_Transfer_.28FRET.29">Fluorescent Resonance Energy Transfer (FRET)</SPAN></H3><H4><SPAN class="mw-headline" id="Materials_and_Methods_3">Materials and Methods</SPAN></H4><P><I>Preparation of Cell Cultures and Controls</I>
A total of six 5ml cell cultures of Escherichia coli DH5α cells containing the BBa_K542008 construct were grown under different conditions overnight at 37°C. As controls parts BBa_K542006, BBa_K331031, BBa_K331033, BBa_K542001 were grown normally overnight at 37°C.</P><P><B>Table 1.</B> E.coli DH5α cells cultures grown in LB media and incubated overnight with shaking at 37°C. Samples #1-6 was grown up differently with specific treatments for the expression of certain parts within the BBa_K542008 construct. Samples #7-10 was grown as controls. </P><P>The following morning, six 50mL cultures were inoculated with 1mL of overnight culture grown to sample 6 (with nothing expressed), and four 50mL cultures were inoculated with samples 7-10. The 50mL cultures were allowed to grow with shaking at 37oC until they reached an OD600 of 0.6 (approximately 3 hours). Samples 1-3 and 6-10 were immediately pelleted by centrifugation (4oC, 5000xg, 20min) and placed on ice. Samples 4 and 5 were treated as described in table above. Pellets were weighed, and resuspended in 10mL/mg wet weight of the following buffer:
</P><UL><LI>50mM TrisHCl (pH8.0)
</LI><LI>250mM NaCl
</LI><LI>5mM 2ME
</LI></UL><P>In order to evaluate expression levels, 1mL of cells was removed from each culture, and analysed using SDS-PAGE (12%).</P><P><I>Lysis of Cells and Clearing of Cell Lysate</I>
Resuspended cell cultures were incubated on ice, and subjected to 3x20 pulse sonication bursts, with 3 minute rests on ice.
Lysate was cleared by centrifugation (4C, 30000xg, 1 hour). Cleared cell lysate was removed and retained for fluorescent analysis.<I>Scanning of Cell Lysate using a Spectrofluorometer</I>
Using a spectrofluorometer, 3 mL of cell lysate was used for each scan. Each scan was done at 5nm excitation and 5nm emission slits. For the analysis of the expression of Förster resonance energy transfer (FRET) within the micro compartment, each cell lysate was excited at 439nm and the emission spectra was read from 444nm to 650nm.
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><P><I>Expression levels of Lumazine Synthase and fluorescent proteins</I>
Load order for the 12% denaturing polyacrylamide gel is as follows:</P><OL><LI>	Low Range Molecular Marker
</LI><LI>	K542008 – No Expression (Sample 6)
</LI><LI>	K542001 – Lumazine Synthase Control (Sample 8)
</LI><LI>	K331031 – YFP Control (Sample 9)
</LI><LI>	K331033 – CFP Control (Sample 10)
</LI><LI>	K542006 – Fluorescent Proteins (Both YFP and CFP) Control (Sample 7)
</LI><LI>	K542008 – Expression of fluorescent proteins only (Sample 1)
</LI><LI>	K542008 – Expression of Lumazine Synthase only (Sample 2)
</LI><LI>	K542008 – Co-expression of Lumazine Synthase and fluorescent proteins (Sample 3)
</LI><LI>	K542008 – Fluorescent proteins first, then Lumazine Synthase (Sample 4)
</LI><LI>	K542008 – Lumazine Synthase first, then fluorescent proteins (Sample 5)
</LI></OL><P><B>Figure 1.</B> Expression patterns of K542008 and controls
In every K542008 sample, regardless of treatment, lumazine synthase (16.6kDa) is expressed. This means that we are unable to control production of lumazine synthase as we had hoped. By altering the promoter, we hope to be able to control the production of lumazine synthase in order to evaluate how the expression patterns of lumazine synthase and arginine tagged proteins affect efficiency of co-localization of tagged proteins within the compartment.
The expression of the fluorescent proteins (each approximately 27kDa) can easily been seen in lanes 2 and 3. Although it is difficult to see fluorescent proteins in K542008 samples, fluorescence observed in FRET experiments suggests that protein is expressed.</P><P><I>FRET</I>
By using sample 7 (fluorescent protein control; BBa_K542006) as the baseline for expression of fluorescent proteins in the cleared cell lysates, we evaluated samples 1-5 for FRET by comparing fluorescence at 475nm (CFP) and 528nm (YFP).  </P><P><B>Figure 2.</B> Relative change in fluorescence at 475nm and 528nm, when excited by light at 439nm. &quot;LS then FP&quot; is Sample 5; &quot;FP Only&quot; is Sample 1; &quot;LS Only&quot; is Sample 2; &quot;FP then LS&quot; is Sample 3; &quot;LS then FP&quot; is Sample 4.</P><P>In each sample, we observed a decrease in fluorescence at 475nm and a simultaneous increase in fluorescence at 528nm. We believe that this decrease in CFP fluorescence (observed at 475nm) and concomitant increase in YFP fluorescence (observed at 528nm) is indicative of FRET interactions occurring between CFP and YFP. The presence of FRET interactions naturally leads us to the conclusion that the oligo-arginine-tagged CFP and YFP are being co-localized within the lumazine synthase microcompartment. 
Within the FRET results, the condition where lumazine synthase was expressed before the fluorescent proteins were expressed shows lower levels of fluorescent change than the other conditions, suggesting that fluorescent proteins have difficulty entering the completely formed microcompartments.
</P><H4><SPAN class="mw-headline" id="Conclusion_2">Conclusion</SPAN></H4><OL><LI> Proteins that have an oligo-arginine tail fused to their C-termini are localized within the microcompartment formed by the oligomerization of lumazine synthase proteins
</LI><LI> Proteins can localize within the compartment more easily when they are co-expressed with lumazine synthase
</LI></OL><DL><DD>Proteins have a more difficult time entering the compartment when the compartment is fully formed
</DD></DL><H4><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H4><P>While we hoped to be able to control the expression of the fluorescent proteins and lumazine synthase temporally, our SDS-PAGE gel indicates that even in the absence of IPTG, the <I>lac</I>-inducible lumazine synthase is being expressed. This means we are not able to achieve one of our conditions – fluorescent proteins expressed first, then lumazine synthase. The constant expression of lumazine synthase gave us several measurements with co-expression of lumazine synthase and fluorescent proteins. Furthermore, co-expression approximates the condition we were attempting for our first condition – namely, the formation of microcompartments around tagged proteins. We were also able to measure the condition where lumazine synthase was expressed in the absence of fluorescent proteins, which were expressed after the production of the lumazine synthase microcompartments. We interpret the results to indicating that tagged proteins have more difficulty entering pre-formed lumazine synthase microcompartments.</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>(1)	Seebeck, F., Woycechowsky, K., Zhuang, W., Rabe, J., and Hilvert, D. (2006). A simple tagging system for protein encapsulation. Journal of the American Chemical Society. 128: 4516-4517.
</P><H1><SPAN class="mw-headline" id="Part:_BBa_K542010_-_Enhanced_Lumazine_Synthase_.28ELS.29"><FONT color="black">Part: BBa_K542010 - Enhanced Lumazine Synthase (ELS)</FONT></SPAN></H1><P>Datasheet for <A href="http://partsregistry.org/Part:BBa_K542010"><FONT color="blue">Part BBa_K542010</FONT></A> in <I>E. coli</I> strain BL21 (DE3). 
</P><H2><SPAN class="mw-headline" id="Introduction_2">Introduction</SPAN></H2><P>Lumazine synthase (LS) from <I>Aquifex aeolicus</I> forms icosahedral microcompartment (MC) assemblies of 60 or 180 monomeric units that self assemble and are capable of isolating proteins from their local environment. As shown in previously published work (1), the LS protein has been mutated so that the interior of the MC is negatively charged; the UL 2009 iGEM team has submitted the mutated LS gene to the parts registry (BBa_K249002). A negatively charged interior allows for preferential compartmentalization of positively charged molecules, which can easily be engineered through the addition of a poly-arginine tag to a target protein (1). The size of the cavity of the Lumazine Synthase microcompartment was enlarged and the negative charge was increased via directed evolution (2). The compartment formed by this polypeptide has a larger cavity and a larger net negative charge, thus allowing a larger loading capacity into the cavity of the compartment.

The part was synthesized by Bio Basic Inc. into the pET28a plasmid vector (which allows for expression of ELS with an N-terminal His-tag). The Enhanced Lumazine Synthase (ELS) was moved into the pSB1C3 vector by the Lethbridge 2011 team.
</P><H2><SPAN class="mw-headline" id="Characterization_2">Characterization</SPAN></H2><H3><SPAN class="mw-headline" id="Overexpression">Overexpression</SPAN></H3><P>The His-tagged ELS in pET28a was transformed into <I>E. coli</I> BL21 (DE3) and overexpressed. 
</P><H4><SPAN class="mw-headline" id="Materials_and_Methods_4">Materials and Methods</SPAN></H4><P>4 X 500 mL of LB media in 2 L flasks were inoculated with <I>E. coli</I> BL21 (DE3) cells expressing ELS. Cells were grown at 37°C with shaking until the culture reached an OD<SUB>600</SUB> of 0.683. The cultures were then induced with 10 mM IPTG. Culture samples containing equal amounts of cells were taken before induction and 30 min, 1 hour, 2 hours, and 3 hours post induction for analysis by SDS-PAGE. At 5 hours post induction the cells were harvested by centrifugation at 5000 x g for 5 min. The cell lysate was resuspended in 42 mL of buffer containing 50 mM Tris pH 8.0, 60 mM NH<SUB>4</SUB>CL, 7 mM β-mercaptoethanol, 1 mM PMSF, 7 mM MgCl<SUB>2</SUB>, 300 mM KCl, 10 mM imidazole, and 15% glycerol. Afterwards, 0.05 g of crystallized lysozyme was added to the cell suspension and incubated on ice for 1 hour. The lysate was then centrifuged for 70 minutes at 30000 x g to obtain the S30 fraction.
</P><H4><SPAN class="mw-headline" id="Results_4">Results</SPAN></H4><P><B>Figure 1.</B>    12.5% SDS-PAGE of His-tagged ELS Overexpressed in <I>E. coli</I> BL21 (DE3). Samples were taken at 1 hour increments after IPTG induction. Lane 1: zero hours; Lane 2: 1 hour; Lane 3: 2 hours; Lane 4: 3 hours. The expected molecular weight of enhanced lumazine synthase is 18.5 kD. 
</P><H4><SPAN class="mw-headline" id="Conclusion_3">Conclusion</SPAN></H4><P>Figure 1 clearly displays an increase in expression of a protein smaller than 25.0 kD, which is approximately the expected size of enhanced lumazine synthase monomers (18.5 kD).
</P><H3><SPAN class="mw-headline" id="Protein_Purification">Protein Purification</SPAN></H3><P>Enhanced lumazine synthase microcompartments were purified using size exclusion chromatography (SEC) to determine if the purified ELS formed a homogenous mixture of compartments, heterogenous mixture of microcompartments, or remained in monomer form.
</P><H4><SPAN class="mw-headline" id="Materials_and_Methods_5">Materials and Methods</SPAN></H4><P>In order to obtain a sample of ELS that was near homogeneity for characterization with SEC, the ELS sample was first purified by immobilized metal-ion affinity chromatography (IMAC). Ni<SUP>2+</SUP>-Sepharose beads supplied by Sigma Aldrich were pre-swelled in buffer containing 20% ethanol and 1mM NaCl. 3.75 mL of the Ni<SUP>2+</SUP>-Sepharose slurry was added to a 50 mL falcon tube. The beads were centrifuged at 500 x g for 2 min and the ethanol-containing buffer decanted. The beads were washed with 3 bed volumes of sterile water. The column was then washed in 6 bed volumes of Buffer A containing 50 mM Tris pH 8.0, 60 mM NH<SUB>4</SUB>CL, 7 mM β-mercaptoethanol, 1 mM PMSF, 7 mM MgCl<SUB>2</SUB>, 300 mM KCl, 10 mM imidazole, and 15% glycerol. S30 cell extract was applied to the column and mixed gently. The resin-S30 extract suspension was incubated on ice for 1 hour. The mixture was centrifuged at 500 x g for 2 min and the supernatant decanted. The supernatant was kept for further analysis by SDS-PAGE. The resin was washed 3 times with a full falcon tube volume (50 mL) of Buffer A. The mixture was centrifuged at 500 x g for 2 min and the supernatant decanted. The supernatant was kept for further analysis by SDS-PAGE. The column was washed 4 times with a full falcon tube volume (50 mL) of Buffer B (Buffer A plus 20 mM imidazole). The mixture was centrifuged at 500 x g for 2 minutes and the supernatant decanted. The supernatant was kept for further analysis by SDS-PAGE. The protein was eluted 9 times using 90% bed volume of Buffer E (Buffer A plus 250 mM imidazole). After each wash the sample was kept for analysis by SDS-PAGE.

Fractions 1 – 9 were pooled and concentrated using a Vivaspin Column with a 10 kDa molecular weight cut-off.

Concentrated protein samples from the Ni<SUP>2+</SUP>-Sepharose column were applied to a Sephacryl S400 size exclusion chromatography column at a flow rate of 0.4 mL/minute. 150 mL of SEC Buffer (50 mM sodium phosphate, 5 mM EDTA, 200 mM NaCl, pH 8.0, with 20% glycerol) was pumped through the column. The fractions eluted were collected and the absorbance was measured at 280 nm (see Fig 3 and 4).
</P><H4><SPAN class="mw-headline" id="Results_5">Results</SPAN></H4><P><B>Figure 2.</B>     SDS-PAGE (15%) of ELS purification by IMAC. Lane 1: protein marker. Lanes 2-10: Elution Fractions 1 – 9. 
</P><P><B>Figure 3.</B>     (A) Concentrated protein sample from the Ni<SUP>2+</SUP>-Sepharose column applied to a Sephacryl S400 size exclusion chromatography column at a flow rate of 0.4 mL/minute. The absorbance of the eluting solution was measured at 280nm. (B) Magnified view of chromatogram of the two peaks seen in (A). 
</P><H4><SPAN class="mw-headline" id="Conclusion_4">Conclusion</SPAN></H4><P>Figure 2 shows that with each wash proteins of a similar size were eluted off the resin. These proteins run at the expected molecular weight of enhanced lumazine synthase monomers (18.5 kD).

The chromatogram (Figure 3) from the chromatography experiment displays two regions of increased absorbance at 280 nm, between 82 and 135 mL of buffer eluted. These fractions were pooled, concentrated, and analyzed using transmission electron microscopy.
</P><H3><SPAN class="mw-headline" id="Transmission_Electron_Microscopy">Transmission Electron Microscopy</SPAN></H3><P>Purified samples of enhanced lumazine synthase from size exclusion chromatography (SEC) were characterized using transmission electron microscopy. We viewed samples of each of the two peaks from the SEC; however only one sample contained microcompartments. 
</P><H4><SPAN class="mw-headline" id="Materials_and_Methods_6">Materials and Methods</SPAN></H4><P>Purified Samples in solution were placed on a carbon grid and negatively stained using uranyl acetate. Carbon grids containing the samples were then viewed with a Hitachi H-7500 Transmission Electron Microscope.  
</P><H4><SPAN class="mw-headline" id="Results_6">Results</SPAN></H4><P><B>Figure 4.</B>     Transmission electron microscopy of Enhanced Lumazine Synthase microcompartments.  Microcompartments of approximately 40 nm can be seen at x 20K magnification. 
</P><P><B>Figure 5.</B>     Transmission electron microscopy of Enhanced Lumazine Synthase microcompartments.  Microcompartments of approximately 40 nm can be seen at x 100K magnification. 
</P><H4><SPAN class="mw-headline" id="Conclusion_5">Conclusion</SPAN></H4><P>As expected, TEM micrographs show polyhedral particles of approximately 40 nm, corresponding to the size described by Wörsdörfer et al. (2011). This confirms that <I>E. coli</I> is capable of producing enhanced lumazine synthase.
</P><H2><SPAN class="mw-headline" id="References_2">References</SPAN></H2><P>(1)	Viadiu H, Aggarwal AK (2000). Structure of BamHI bound to nonspecific DNA: a model for DNA sliding. Mol. Cell 5 (5): 889-895.

(2)	Wörsdörfer, B., Woycechowsky, K.J., and Hilvert, D. (2011). Directed Evolution of a Protein Container. Science. 331: 589-592.
</P><H1><SPAN class="mw-headline" id="Part:_BBa_K542009_.E2.80.93_Antigen_43"><FONT>Part: BBa_K542009 – Antigen 43</FONT></SPAN></H1><P>Datasheet for <A href="http://partsregistry.org/Part:BBa_K542009"><FONT color="blue">Part BBa_K542009</FONT></A> in <I>E. coli</I> strain BL21 (DE3). 
</P><H2><SPAN class="mw-headline" id="Introduction_3">Introduction</SPAN></H2><P>The Peking 2010 iGEM team characterized Antigen 43 (AG43) using a T7 promoter expression system in <I>E. coli</I> BL21(DE3) strain and submitted the coding sequence with no promoter or ribosome binding site (RBS) to the parts registry (BBa_K364007). In order for AG43 to be used, BBa_K542009 was assembled to include the required regulatory elements. The pLacI promoter and RBS were added to the coding sequence for AG43.
</P><H2><SPAN class="mw-headline" id="Construct_2">Construct</SPAN></H2><P>Part Number: BBa_K542009
</P><P><B>Figure 1.</B>    BBa_K542009. The inductive aggregation construct was made by assembling BBa_J04500 and BBa_K346007. 
</P><H2><SPAN class="mw-headline" id="Sedimentation_by_Antigen_43">Sedimentation by Antigen 43</SPAN></H2><P><I>E. coli</I> DH5α cells expressing BBa_K542009 were monitored for sedimentation and compared to sedimentation of uninduced cells.
</P><H3><SPAN class="mw-headline" id="Materials_and_Methods_7">Materials and Methods</SPAN></H3><P>Overnight cultures <I>E. coli</I> DH5α containing BBa_K542009 or BBa_J04500 (blank control) were grown with the appropriate antibiotics. Three 50 mL LB cultures – BBa_J04500 (control), BBa_K542009 (induced), and BBa_K542009 (uninduced) – with the appropriate antibiotics were inoculated to an OD<SUB>600</SUB> of 0.1 and incubated at 37°C with shaking.

At OD<SUB>600</SUB> of 0.4-0.6, the cultures were induced (excluding one of the BBa_K542009 cultures) with IPTG to a concentration of 0.001% and growth was monitored to a final OD<SUB>600</SUB> of 1. The cells were centrifuged at 5000 x g for 5 min and the supernatant discarded. The cells were resuspended with 5 mL of 1% PBS (with 0.15 mM NaCl) and transferred into test tubes. The cells in the test tubes were agitated and 100 µL was taken ~0.5 cm from the surface every 10 min for the first 60 min; then every hour for the next 3 hours and again at 18 hours and 18.5 hours. The samples were also subjected to OD<SUB>590</SUB> measurements using a microplate and the SoftMax Pro Software.
</P><H3><SPAN class="mw-headline" id="Results_7">Results</SPAN></H3><P><B>Figure 2.</B><I>E. coli</I> DH5a cells containing BBa_K542009. Cultures were first agitated and then sedimentation rates were monitored. (a) Initial reading at the beginning of the experiment for the induced and uninduced. (b) Initial reading for the blank and 30 min after agitation for the induced and uninduced. (c) 3.5 hr after agitation for the blank and 4 hr after agitation for the induced and uninduced. (d) 17.5 hr after agitation for the blank and 18 hr after agitation for the induced and uninduced. 
</P><P><B>Figure 3.</B>    Microplate readings at 590nm for samples taken from the sedimentation assay using SoftMax Pro Software. Increase in absorbance at 150 min may have been due to the aggregation of the cells on the microplate prior to taking the measurement. 
</P><H3><SPAN class="mw-headline" id="Conclusion_6">Conclusion</SPAN></H3><P>There is a significant difference between the aggregation of cells with the Antigen43 construct (BBa_K542009) and the cells without the construct after 18 hours. The pLacI promoter is known to be leaky. This was illustrated by the similarity between the uninduced and induced samples (Figures 2 and 3). These experiments support the conclusion that BBa_K542009 works as expected and the induced cells exhibit sedimentation.
</P><H1><SPAN class="mw-headline" id="Part:_BBa_I716462_.E2.80.93_BamHI"><FONT color="black">Part: BBa_I716462 – <I>Bam</I>HI</FONT></SPAN></H1><P>Datasheet for <A href="http://partsregistry.org/Part:BBa_I716462"><FONT color="blue">Part BBa_I716462</FONT></A> in <I>E. coli</I> strain DH5alpha. 
</P><H2><SPAN class="mw-headline" id="Introduction_4">Introduction</SPAN></H2><P>Part I716462 was previously characterized by the UC Berkeley iGEM team in 2007. The purpose of this part was to prevent unwanted cell proliferation by engineering a genetic self-destruct mechanism into bacteria that upon induction would express a genetic material-degrading toxin, the endonuclease BamHI that would kill cells yet preserve their physical integrity. <I>Bam</I>HI has a restriction cut site of G'GATCC, generating sticky ends (1). The restriction endonuclease was placed under the control of the arabinose- inducible promoter, pBAD. In 2007, the UC Berkeley iGEM team was able to show that cells lost their ability to reproduce but did not lyse upon induction with arabinose (2). The University of Lethbridge 2011 iGEM team wished to further characterize this part to degrade the genomic DNA of E. coli in order to have a functional chassis for our tailings pond clean-up kit without the risk of DNA contamination. In this way, tight control and regulation over our engineered bacterial cells is possible.

Registry Entry - BBa_I716462

Length – 1918 bp 
</P><H2><SPAN class="mw-headline" id="Characterization_3">Characterization</SPAN></H2><H3><SPAN class="mw-headline" id="Cell_Survival">Cell Survival</SPAN></H3><H4><SPAN class="mw-headline" id="Materials_and_Methods_8">Materials and Methods</SPAN></H4><P>The <I>Bam</I>HI construct was overexpressed in <I>E. coli</I> DH5α cells. Cultures were induced with arabinose at an OD<SUB>600</SUB> of 0.6 and grown overnight. Samples of the induced culture and an uninduced culture were plated on LB agar plates to monitor growth of colony forming units.
</P><H4><SPAN class="mw-headline" id="Results_8">Results</SPAN></H4><P><B>Figure 1.</B>   Growth curves of <I>E. coli</I> DH5α cells induced (blue) and uninduced (red) for expression of <I>Bam</I>HI. Cultures were induced at an OD<SUB>600</SUB> of 0.60.
</P><P><B>Figure 2.    </B><I>E. coli</I> DH5α cells containing the <I>Bam</I>HI construct from overnight cultures. The samples were plated on LB agar plates and the number of colony forming units (CFU) was counted. Left: culture induced for <I>Bam</I>HI expression with arabinose, right: uninduced culture.  
</P><H3><SPAN class="mw-headline" id="Analysis_of_Genomic_DNA">Analysis of Genomic DNA</SPAN></H3><H4><SPAN class="mw-headline" id="Materials_and_Methods_9">Materials and Methods</SPAN></H4><P><I>E. coli</I> DH5α cells were grown and induced to overexpress one of three samples: <I>Bam</I>HI construct induced with arabinose (10µM) at an OD<SUB>600</SUB> of 0.6, <I>Bam</I>HI construct uninduced, non-<I>Bam</I>HI construct induced with arabinose (10µM) at OD<SUB>600</SUB> of 0.6. Cells were grown in LB media with 100 ng/mL ampicillin.

Qiagen Blood and Tissue Kit was used to isolate the genomic DNA of gram-negative bacteria. A 0.8% agarose gel was ran at 150 V for 90 minutes to separate the genomic DNA of <I>E. coli</I> DH5α cells collected at different time points (pre-induced and 30 min, 1 hr, 2 hr, and 3 hr post induction) for all three samples. One absorbance unit of cells were collected and analyzed by SDS-PAGE to monitor cellular protein levels.
</P><H4><SPAN class="mw-headline" id="Results_9">Results</SPAN></H4><P><B>Figure 3.</B>     Induced <I>Bam</I>HI. 12% SDS-PAGE of cell lysate from <I>E. coli</I> DH5α cells containing BBa_I716462 induced for overexpression of <I>Bam</I>HI. Left to right: protein marker, 0 h, 0.5 h, 1 h, 2 h, and 3 h after induction with lactose.
</P><P><B>Figure 4.</B>    Induced <I>Bam</I>HI. 12% SDS-PAGE of cell lysate from uninduced <I>E. coli</I> DH5α cells containing BBa_I716462. Left to right: protein marker, 0 h, 0.5 h, 1 h, 2 h, and 3 h after induction with lactose.
</P><P><B>Figure 5.</B>     Uninduced <I>Bam</I>HI. 12% SDS-PAGE of cell lysate from <I>E. coli</I> DH5α cells. Left to right: protein marker, 0 h, 0.5 h, 1 h, 2 h, and 3 h after induction with lactose.
</P><H2><SPAN class="mw-headline" id="Conclusion_7">Conclusion</SPAN></H2><P>Upon induction with arabinose, <I>E. coli</I> DH5α cells containing the <I>Bam</I>HI construct did not reproduce as the overexpression of the <I>Bam</I>HI endonuclease digested the genomic DNA of the host cell. This is why in Figure 2 there are only 4 colonies seen as opposed to the control which shows a lawn of growth. In Figure 1, the optical density of the <I>Bam</I>HI expressing cells reaches a plateau; however, the same sample which has not been induced kept increasing, which indicates that the expression of <I>Bam</I>HI is affecting normal growth of the cells.

Samples analyzed by SDS-PAGE after induction of <I>Bam</I>HI expression show that protein concentration is not significantly affected (Figures 3-5).
</P><H2><SPAN class="mw-headline" id="References_3">References</SPAN></H2><P>(1)	Viadiu H, Aggarwal AK (2000). Structure of BamHI bound to nonspecific DNA: a model for DNA sliding. Mol. Cell 5 (5): 889-895.

(2)	iGEM 2007. (2011, May 10). BerkiGEM2007Present5. Retrieved from <A href="https://2007.igem.org/BerkiGEM2007Present5" class="external free" rel="nofollow">https://2007.igem.org/BerkiGEM2007Present5</A>. 
</P><H1><SPAN class="mw-headline" id="Collaboration_-_Cell_Viability_in_Tailings_Pond_Water">Collaboration - Cell Viability in Tailings Pond Water</SPAN></H1><P>We were asked by the Calgary iGEM team to to test the viability of their chassis to survive in tailings pond water. If <I>E. coli</I> DH5α cells containing one of their constructs can grow normally in medium prepared with tailings water, we can be confident that the chassis can be applied to tailings pond water in general.
</P><H2><SPAN class="mw-headline" id="Materials_and_Methods_10">Materials and Methods</SPAN></H2><P>50 mL of LB medium was made with tailings water and filtered through MILLEX filter Unit GS MF-Millipore MCE Membrane 0.22 µm in order to sterilize it and appropriate antibiotics were added. The tailings LB was inoculated with <I>E. coli</I> DH5α cells containing BBa_K331009 to an OD<SUB>600</SUB> of 0.1. The flask was incubated at 37˚C with shaking for 7 hours, during which photometric readings were taken every 30. Photometric readings were taken using the Pharmacia Biotech Ultrospec 3000, as the OD<SUB>600</SUB> readings reached 1.0 the solutions was diluted with LB media to keep the OD<SUB>600</SUB> reading between 0.1 and 1.0. The same protocol was used to observe cell growth of <I>E. coli</I> DH5α cells containing BBa_K331009 in 50 mL of LB medium made with MilliQ H<SUB>2</SUB>O.
</P><H2><SPAN class="mw-headline" id="Results_10">Results</SPAN></H2><P><B>Figure 1.</B>    OD<SUB>600</SUB> of <I>E. coli</I> DH5α cells containing BBa_K331009 cultured in LB medium made with tailings water (red) and LB medium made with MilliQ H<SUB>2</SUB>O (black).  Performed by our team.
</P><P><B>Figure 2.</B>    OD<SUB>600</SUB> of <I>E. coli</I> DH5α cells containing BBa_K331009 cultured in LB medium made with tailings water (red and blue) and LB medium made with MilliQ H<SUB>2</SUB>O (black).  Performed independently by the 2011 Calgary iGEM Team.
</P><H2><SPAN class="mw-headline" id="Conclusion_8">Conclusion</SPAN></H2><P>As seen in Figure 1, <I>E. coli</I> DH5α cells were able to grow in LB medium made with tailings water, suggesting that the chassis will survive for use in tailings pond water.
</P></BLOCKQUOTE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Lethbridge/Results">http://2011.igem.org/Team:Lethbridge/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Lethbridge/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Lethbridge/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Lethbridge/Results&amp;oldid=263238" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>