<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Slovenia_HS_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Slovenia HS/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team Slovenia_HS: Notebook</TITLE></P><DIV id="page-wrapper"><HEADER id="header"><DIV class="logo container"><DIV><H1><A href="https://2015.igem.org/Team:Slovenia_HS" id="logo">iGEM HIGH SCHOOL</A></H1><P>team Slovenia</P></DIV></DIV></HEADER><NAV id="nav"><UL><LI><A href="https://2015.igem.org/Team:Slovenia_HS">Main</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Extended_abstract">Extended abstract</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Conclusions">Conclusions</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Construction">Construction</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Testing">Testing</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Parts">Parts</A></LI><LI class="current"><A href="#">Lab &amp; Safety</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Recipes">Recipes</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Safety">Safety</A></LI><LI><A href="#">People</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Funding">Funding</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Slovenia_HS/Judging">Judging</A></LI></NAV><DIV id="main-wrapper"><DIV id="main" class="container"><DIV class="row"><DIV class="12u"><DIV class="content"><ARTICLE class="box page-content"><HEADER><H2>Notebooks</H2></HEADER><H3>Our Journey</H3><HEADER>
 <CENTER><H3>''If you want to go quickly, go alone. If you want to go far, go

together.''</H3><P><I>- African proverb -</I></P></CENTER></HEADER>
 

<B>The beginning of our journey</B>

 

<P align="justify">When we first saw that The National Institute of Chemistry was looking for team members

to present the colours of Slovenia iGEM HS Division, we thought it would be a great

opportunity to do what something we were interested in and learn something new. We

applied to the call for applications and hoped for the best. Parts of the application

procedure were writing a motivational letter, attending an interview, taking part in a

personality test and, the toughest of them all, solve a scientific problem.

 <B><I>14 months ago, we knew nothing about synthetic biology because we only learned the

basics in our biology and chemistry classes.</I></B>We had no clue how demanding the project is

going to be at the time.</P><B>Building a knowledge basis</B>

 

<P align="justify">When we started with our team meetings in September and October 2014, <B><I> we first had to

learn the basics of biochemistry and biotechnology. </I></B> Through autumn and winter we have

had having lessons two afternoons per week after school. We learned about cloning,

plasmids, PCR and biobricks. Moreover, we learned how to edit wiki pages, use

bioinformatics programs and data bases. At the same time we were also studying the

additional literature and developing our ideas. We learned about E. coli, conditions for its

optimal growth and bioreactors. Finally, we got to enter the labs where we practiced basic

lab techniques and protocols, such as pipetting with micropipettes. We also cut the vectors

with restriction enzymes for the first time and tested our samples using agarose gel

electrophoresis.  <B><I>It needless to say how fascinated we were when we were actually able to

see several pieces of DNA of different lengths. </I></B>

It was a great experience because we were tutored by university professors and scientists

who were really able to explain to us everything thoroughly.</P><B>So Let’s get to work!</B>

 

<P align="justify">We visited the the company AlgEn, which was a unique exprence which helped us shape our ideas for our project and design the experiments better.
Afterwards, we started regularly working in labs of the National Institute of Chemistry and the Faculty of Chemistry and Chemical Technology.  


 

When we first actually started working in the lab we wanted to obtain the genes needed for

building our construct from a mixed culture which we knew also contained <I>Clostridium</I>. We

wanted to do so because The Laboratory for Environmental Sciences and Engineering

already used this culture for their own experiments and we wanted to make use of it as

well. As our DNA purification from mixed culture was unsuccessful we decided to go a step

further and order pure culture and purify the DNA out of it. However, this was also

unsuccessful and we decided to order synthetic DNA segments from IDT Technologies.

These arrived in late June and since then we spent more or less every day in the lab.</P>

 

<B>Summer of tears and sweat (and empty AGE gels)</B><P align="justify">We spent the following months sweating in the lab. Our days consisted of cloning, doing

colony PCRs, testing our samples, hoping for positive results of agarose gel electrophoresis

and crying as this was rarely the case. At the same time we prepared different media to

see what best suits for E. coli's growth, measured absorbance, worked in both aerobic and

anaerobic conditions (the following were especially fun), carried out the ecotoxicological

tests, isolated n-butanol, prepared different media again and analysed tons of results.</P><B>There is life outside the lab too…</B>

 

<P align="justify">Besides the lab work we also filmed a short clip about our project and presented our project

and principles of synthetic biology to a wider audience. <B><I>We met the Slovenian prime

minister while doing so and made sure he understood what we were trying to accomplish.</I></B>

We established our very first crowdfunding campaign on Indiegogo, wrote articles,

collaborated with other iGEM teams, searched for sponsors and designed our wiki.</P><B>The lessons we learned</B><P align="justify"><B><I>To be honest, we have never imagined how demanding it would be to manage such a

project. </I></B>In fact, it turned out to be much more than just a scientific research. <B><I>Being a part of

this project was a great opportunity to experience real-life scientific teamwork, develop

organisational and leadership skills.</I></B>

On the other hand we had to overcome a lot of obstacles, from having troubles with

bureaucracy to not being able to get our vectors into bacteriae. It was then when we

realized only we are the ones responsible for the project and that it is on us to reach out for

help. We learned that in the end you always have to do what you have to do, no matter how

hard it may seem.<B><I> Perhaps the crucial lesson of this summer was about the importance of

collaboration with others or, to put it another way, to stick together through thick and

thin.</I></B></P><SECTION><H2>Construction Notebook</H2><H3>1.4.2015</H3>•DNA cloning with lyophilized bacteria•Our template were 2 mixed cultures of lyophilized bacteria cells (from National Institute of Chemistry) which were resuspended in water and sonified.•We attempted to clone 2 genes:
CtfAB and
BdhA

2 concentrations of each culture were prepared:•original concentration•10x dilluted•PCR from cultures•Agarose gel electrophoresis, PCR was not successful<H3>2.4 2015</H3>•DNA extraction from sludge.
<H3>8.4.2015</H3>•DNA cloning with isolated DNA•Our template was DNA previously isolated from active sludge and the active sludge itself.
Two concentrations of isolated DNA were prepeared:•the original concentration•200x diluted

We attempted to clone the following genes:•BdhB•BdhE2•AdhE•CtfAB•BdhA

We tried to optimise PCR by varying the annealing temperature of primers: 40, 45, 50 and 55 °C.<H3>22.4.2015</H3>•Gene isolation from <I>Clostridium</I> acetobutylicum
We ordered new specific oligonucleotides (BdhA, CtfAB).•Our goal was the isolation of genes reffering to <I>Clostridium</I> acetobutylicum. 
 
As oligonucleotides we have used BdhA (R), Bdha (F), ctfAB (R), ctfAB (F), 16S (F), 16S (R). 
Isolation sources:•    broth•    10x diluted broth•    DNA•    100x diluted DNA
 
PCR (master mix preparation)
Agarose gel electrophorsis preparation









			


AGE: 1- broth, 2-100x diluted broth, 3- DNA, 4- 100x diluted DNA
 We can see that there are some of the products, but they do not correspond of all the masses of the products. CfAB and BdhA are both 1000 bp long. 16S is 600 bp long. The reaction has to be repeated.<H3>6.5.2015</H3>•DNA cloning with pure culture•Our template was pure <I>Clostridium</I> acetobutylicum culture. Two concentrations were prepared:•the original concentration•50x diluted
We attempted to clone the following genes:•16S•CtfAB•AdhE•AdhE2•BdhA•BdhB
Agarose gel electrophoresis showed that only 16S was successfully cloned.

   
Only 16S was successfully cloned.<H3>7.5.2015</H3>

DNA cloning
Our template was pure <I>Clostridium</I> acetobutylicum. A fresh master mix for cloning the following genes was prepared for:•16S•CtfAB•AdhE•AdhE2•BdhA•BdhB
Agarose gel electrophoresis showed that only 16S was successfully cloned.<H3>20.5.2015</H3>•iGEM transformation efficiency kit test<H3>28.5.2015</H3>•preparation of transformed E. coli for negative controls.
Constructs:•promotor + RBS•only terminator<H3>22. 6. 2015</H3>•Ligation with pJET 1.2 VECTOR, overnight incubation, PCR <H3>24.6.2015</H3>•Isolation of plasmid DNA from liquid medium via protocol.<H3>30. 6. 2015</H3>•inoculation of the negative controls into liquid LB media, DNA purification (AdhE1 and AdhE2), preparation of cloning vectors (with inserts BdhA, CtfA, CtfB) and their transformation into E. coli <H3>1.7. 2015</H3>•colony PCR (transformed cells with BdhA, CtfA, CtfB) <H3>2.7.2015</H3>•Agarose gel electrophoresis•Colony PCR of samples•We were repeating agarose gel electrophoresis •unsccessful gene isolation from sludge due to Electrophoresis for CtfA was succesfull•Plasmid DNA isolation using centrifuges<H3>3. 7. 2015</H3>•AGE of yesterday's samples, DNA purification (CtfA), colony PCR (BdhA, CtfB) <H3>6. 7. 2015</H3>•AGE of the last colony PCR samples, DNA purification (CtfB), colony PCR (BdhA), AGE of the samples (colony PCR of BdhA insert), BdhA ligation and transformation repeated <H3>7. 7. 2015</H3>•AGE of the samples (colony PCR with BdhA insert of the yesterday's transformed cells). Colony PCR repeated with new primers and new master mix. Inoculation of the transformed cells (with BdhB, AdhE1, AdhE2, CtfB, CtfA inserts) into liquid LB media. Restriction of the purified DNA (plasmids with CtfA, CtfB, BdhB, AdhE1 and AdhE2 inserts). <H3>8. 7. 2015</H3>•AGE of the samples (last colony PCR and restriction samples). DNA purification repeated (BdhB, AdhE1, AdhE2, CtfA, CtfB)<H3>13.7.2015</H3>•Plasmid DNA isolation using centrifuges•measuring of absorbance of constructs•Control restriction of genes, linearization

AGE of restrictions<H3>20. 7. 2015</H3>•colony PCR of CtfB, BdhB and CtfA cells, AGE of the samples. Inoculation of E. coli with BdhB, CtfB, AdhE2, CtfA plasmids and an empty pSB1c vector in a liquid LB media <H3>21.7.2015</H3>•CtfA, CtfB, BdhB restriction with XbaI and PstI•Restriction mixture was incubated at 37°C for 3h.
PCR of samples: markings: 1-10 CtfA, 11-20 CtfB, 21-30 BdhB
Agarose gel electrophoresis of PCR samples

Expected lenghts of genes:•CtfA: 735 bp•CtfB: 744 bp•BdhA: 1250 bp•AdhE1: 1666 bp•AdhE2: 1020 BP


Electrophoresis was successful for the restriction products. Fragments with appropriate lenghts were isolated from the gel.•DNA gel extraction•Ligation•overnight incubation on 37°C.
These fragments were then inserted into the expression vector pSB1C3.<H3>22.7.2015</H3>

Transformation of plasmid pSB1C3 into cells E.coli DH5α was conducted accordingly to Heat shock<H3>3.8.2015</H3>•Agarose gel electrophoresis  of PCR samples•Pouring of plates, 400 mL•Transformation of (35, 43, 51) appropriate samples , smearing on the plates (200 µL)•Inoculation of transformed cells•Measuring of absorbance<H3>4.8.2015</H3>•Plasmid DNA isolation (CtfB, BdhB)•Measuring of absorbance•Inoculating on liquid medium•Protein isolation (CtfB, BdhB)•Restriction for pSB1C3+P+RBS+CtfB+dT plasmid (with SpeI, PstI) and pSB1C3+P+RBS+BdhB+dT plasmid (with XbaI, PstI)•Overnight incubation at 37 °C<H3>23. 7. 2015</H3>
colony PCR of the transformed cells, AGE of the samples, colony PCR and AGE repeated <H3>24. 7. 2015</H3>
restrictions of vectors
CtfA forward insert
CtfB forward insert
BdhB forward insert
CtfA back insert
CtfB back insert
BdhB back insert
negative control 1
negative control 2
AGE of samples
DNA purification out of gel
ligation into pSB1c
transformation of cells<H3>25. 7. 2015</H3>
colony PCR of yesterday’s samples
repeating the yesterdays’ unsuccessful restrictions
AGE of the samples
DNA purification out of gel
ligation<H3>26. 7. 2015</H3>•colony PCR of yesterday’s samples•repeating the yesterdays’ unsuccessful restrictions •AGE of the samples•DNA purification out of gel<H3>27. 7. 2015</H3>•ligation of yesterdays’ samples into pSB1c•transformation of the cells<H3>30. 7. 2015</H3>•DNA purification using kit
transformation of AdhE1 and AdhE2 plasmids into the cells
restrictions•CtfA forward insert•CtfB forward insert•Bdhb forward insert•negative control 1 back insert•AdhE1•AdhE2<H3>31. 7. 2015</H3>•colony PCR of the samples•DNA purification using kit•ligation of negative controls<H3>5.8.2015</H3>•CtfA gene grew•Plasmid isolation using centrifuges•AGE restrictions•Measuring of absorbance•Agarose gel electrophoresis •Ctf•Antibody incubation•Ligation mixture (CtfB, BdhB)•Restriction of CtfA and CtfB •Overnight incubation on 37 °C<H3>6.8.2015</H3>•Inoculating of constructs CtfB and BdhB on plates. •Measuring of absorbance•Isolation of proteins  (CtfA, CtfB and BdhB)<H3>7.8.2015</H3>•Restriction of CtfA(Spe1), CtfB (EcoRI)•Adding CtfB, BdhB in AGE gel. •Measuring of absorbance<H3>10.8.2015</H3>•Isolation of proteins<H3>11.8.2015</H3>•Colony PCR: 35 (CtfA), 43 (CtfB), 51 (BdhB)<H3>12.8.2015</H3>•AGE of constructs (62 samples)•Western blot•Restriction of pSB1C3 with EcoRI, PstI for biobricks•Overnight incubation on 37 °C•Inoculating CtfA, CtfB, BdhB on liquid medium•Transformation in pJET<H3>13.8.2015</H3>•Agarose gel electrophoresis (CtfA, CtfB, BdhB, pSB1C3)•Plasmid isolation using centrifuges (Pjet), inoculating, because of unsuccessful growth•Colouring of proteins•Transformation of BdhB in Pjet. <H3>14.8.2015</H3>•Colouring of proteins (CtfA, CtfB)•Protein electrophoresis•Plasmid isolation using centrifuges (BdhB, CtfB)•Inoculating on medium (CtfA, CtfB, BdhB)<H3>18.8.2015</H3>•Colony PCR for AdhE in pJET. •Agarose gel electrophoresis•Transformation with ligations PCR with AdhE1•Transformation<H3>19. 8.2015</H3>•colony PCR and AGE•1-10  pSB1c3+ctfA•11-20 pSB1c3+ctfB•21-30 pSB1c3+bdhB•31-40 pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt•R: Unsuccesful•colony PCR and AGE•1-30 pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt•31 pSB1c3+prom+RBS+ctfA+dt•32 pSB1c3+ prom+RBS+ctfB+dt•R: only ctfB is succesful<H3>20.8.2015</H3>•restriction Adhe1 and Adhe2 with Kpnl, AGE, isolation from gel and restriction with Apal•R: Apal doesn't cut well•restriction of pSB1c3+prom+RBS+ctfA+dt(BV) and pSB1c3+prom+RBS+ctfB+dt (BI) over night<H3>21. 8. 2015</H3>•restriction of pSB1c3+prom+RBS+ctfB+dt(FV) and pSB1c3+prom+RBS+ctfB+dt (FI)•AGE of the restrictions for forward ligation and restrictions for back ligation from yesterday•R: unsuccesful<H3>24. 8.2015</H3>•cross cleveage (cutting each plasmid with only one restriction enzyme instead of double digest) with samples. The samples are cut with one restriction enzyme and only a part of the sample is loaded on gel. If the restriction is succesful, the other restriction enzyme (in parenthesisI will be added to complete the restriction•pSB1c3+prom+RBS+ctfA+dt with SpeI (EcoRI)•pSB1c3+prom+RBS+ctfA+dt with EcoRI (SpeI)•pSB1c3+prom+RBS+ctfA+dt with SpeI (PstI)•pSB1c3+prom+RBS+ctfA+dt with PstI (SpeI)•pSB1c3+ prom+RBS+ctfB+dt with XbaI (PstI)•pSB1c3+ prom+RBS+ctfB+dt with PstI (XbaI)•pSB1c3+ prom+RBS+ctfB+dt with XbaI (EcoRI)•pSB1c3+ prom+RBS+ctfB+dt with EcoRI (XbaI)•pSB1c3+prom+RBS+ctfB+dt with SpeI (EcoRI)•pSB1c3+prom+RBS+ctfB+dt with EcoRI (SpeI)•pSB1c3+prom+RBS+ctfB+dt with SpeI (PstI)•pSB1c3+prom+RBS+ctfB+dt with PstI (SpeI)•pSB1c3+ prom+RBS+bdhB+dt with XbaI (PstI)•pSB1c3+ prom+RBS+bdhB+dt with PstI (XbaI)•pSB1c3+ prom+RBS+bdhB+dt with XbaI (EcoRl)•pSB1c3+ prom+RBS+bdhB+dt with EcoRl (XbaI)

R: restrictions with the first enzyme were succesful, so the second enzyme was added over night<H3>25. 8.2015</H3>•we have new competent cells, so we are repeating Transformation •Efficency Test
R: no colonies grew•AGE of yesterday's restrictions
R: mostly unsuccesful, we isolate only pSB1c3+prom+RBS+ctfB+dt as back insert (BI)
R: unsuccesful<H3>26. 8.2015</H3>•Repeating the cross cleveage with the same samples, AGE, adding the second enzyme over night•Repeating the Transformation Efficency Test
R: succesful<H3>27. 8.2015</H3>•AGE of yesterday's restrictions
R: succesful, we isolate pSB1c3+prom+RBS+ctfA+dt (BV), pSB1c3+prom+RBS+ctfB+dt (BI), pSB1c3+prom+RBS+ctfB+dt (BV), pSB1c3+prom+RBS+bdhB+dt (BI), pSB1c3+prom+RBS+ctfB+dt (FI), pSB1c3+prom+RBS+bdhB+dt (FV) •ligation for 2 hours at room temperature and transformation (for eachmixture we transform 2 tubes of cells, one with 2 ul, the other with 6 ul of ligation mixture; the rest of ligation mixture is left over night at 16•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with  prom+RBS+ctfB+dt (BI)
pSB1c3+prom+RBS+ctfB+dt (BV) ligated with prom+RBS+bdhB+dt (BI)
prom+RBS+ctfB+dt (FI) ligated with pSB1c3+prom+RBS+bdhB+dt (FV)
R: the next day nothing grew<H3>28. 8.2015</H3>•as the cells transformed with ligation mixture did not grow, we again transform new cells with the same ligation mixture that was left on 20 °C over night•SDS-page of the cultures  pSB1c3+prom+RBS+ctfA+dt, pSB1c3+prom+RBS+ctfB+dt, pSB1c3+prom+RBS+bdhB+dt that were grown for 3 days on 16 °C in hopes that the produced proteins would appear in the soluble fraction
R: all genes appear to be active, however they remain in the unsoluble fraction<H3>31. 8.2015</H3>•we recieved constructs from Aalto Helsinki (pSB1c3+GFP), which we transform the cells with•transformation of cells with the biobrick BBa_K863010 (lacase) and an empty vector pSB1c3 as control
restriction of:•pSB1c3+prom+RBS+ctfA+dt as BV, FI•pSB1c3+prom+RBS+ctfB+dt as BI, BV, FI, FV•pSB1c3+prom+RBS+bdhB+dt as BI, FV
R: forward vectors and inserts were not succesful, but back inserts and vectors were
we isolate from the gel:•pSB1c3+prom+RBS+ctfA+dt as BV•pSB1c3+prom+RBS+ctfB+dt as BI, BV•pSB1c3+prom+RBS+bdhB+dt as BI•we prepare the ligation for 2 hours at room temperature and then transform the cells
pSB1c3+prom+RBS+ctfA+dt (BV) ligated with +prom+RBS+ctfB+dt (BI) pSB1c3+prom+RBS+ctfB+dt (BV) ligated with prom+RBS+bdhB+dt (BI)
R: the next day nothing grew<H3>1. 9.2015</H3>restriction and AGE of:•pSB1c3+prom+RBS+ctfA+dt as BV, FI•pSB1c3+prom+RBS+ctfB+dt as BI, BV, FI, FV•pSB1c3+prom+RBS+bdhB+dt as BI, FV
R: except for pSB1c3+prom+RBS+ctfA+dt (FI) the restrictions were succesful
We isolate from the gel:•pSB1c3+prom+RBS+ctfA+dt as BV•pSB1c3+prom+RBS+ctfB+dt as BI, BV, FI•pSB1c3+prom+RBS+bdhB+dt as BI, FV•we set the ligation on 16 °C over night of:•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfB+dt (BV) ligated with prom+RBS+bdhB+dt (BI)•prom+RBS+ctfB+dt (FI) ligated with pSB1c3+prom+RBS+bdhB+dt (FV)


SDS-page of:•pSB1c3+prom+RBS+ctfA+dt •pSB1c3+prom+RBS+ctfB+dt •pSB1c3+prom+RBS+bdhB+dt
R:two gels were made for each gene, one was later colored and with the other a western blot was preformed•we preform western blot for two of the gels and leave the membrane in milk over night in the cold room<H3>2. 9.2015</H3>•we transform the cells with the ligations we set last night•transformation of BBa_K863010 (lacase)
we set the restriction for and complete the AGE for:•pSB1c3+bdhB+dt (BI)•pSB1c3+prom+RBS (BV) •because we have problems expressing the same constructs we have made before,
pSB1c3+GFP (BV) and
pSB1c3+ctfB+dt (BI)
for our collaboration with Aalto
R: all restrictions were succesful•we isolate all the restrictions from the gel and set the ligation for 2 hours at room temperature, later transforming the cells
pSB1c3+prom+RBS (BV) ligated with bdhB+dt (BI)
pSB1c3+GFP (BV) ligated with ctfB+dt (BI) •we stain the two membranes from yesterday
R: construct with ctfB appears and is partialy in the soluble fraction, the other two construct do not appear for an unknown reason•we check the expression for both lacases with SDS-page, making two geles, one to be coloured and the other for western blot, storing the membrane in milk over the night in the cold room
R: nothing of note appears on the gels<H3>3. 9.2015</H3>•we stain the membrane from yesterday
R: the membrane does not colour
colony PCR and AGE:•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (back ligation)•pSB1c3+prom+RBS+ctfB+dt +prom+RBS+bdhB+dt (forward ligation)
R: some samples pSB1c3+prom+RBS+ctfB+dt +prom+RBS+bdhB+dt (back ligation) seem to succed<H3>4. 9.2015</H3>
colony PCR and AGE of:•pSB1c3+prom+RBS+bdhB+dt and
•pSB1c3+GFP+ctfB+dt
R: the construct with bdhB succeded•expression of BBa_K733012 and BBa_K863010
R: unsuccesful•we set restriction and AGE for:
pSB1c3+ctfB+dt as BI and
pSB1c3+GFP as BV
R:restriction is succesful•we isolate both samples from the gel and set ligation over night
pSB1c3+GFP (BV) ligated with ctfB+dt (BI)<H3>5. 9.2015</H3>•expression of BBa_K733012 and BBa_K863010
R: unsuccesful•we set the restriction of
pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt as BI,
pSB1c3+prom+RBS+ctfB+dt as BI and
pSB1c3+prom+RBS+ctfA+dt as BV 
R: restriction is succesful•we isloate restrictions from the gel and set ligation over night•pSB1c3+prom+RBS+ctfA+dt(BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)
pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI) •we transform the cels with yesterday's ligation pSB1c3+GFP (BV) ligated with ctfB+dt (BI)•SDS-page with different samples from yesterday's colony PCR and AGE, we make two gels, one to be coloured, the other for western blot, storing the membrane in milk over night<H3>6. 9.2015</H3>•we colour yesterday's membrane
R: it does not colour•we transform cells with yesterday's ligations•pSB1c3+prom+RBS+ctfA+dt(BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI)
•colony PCR and AGE of
pSB1c3+GFP+ctfB+dt
R: some of the samples are succesful
we set restriction and AGE of:•pSB1c3+GFP+ctfB+dt as BI•pSB1c3+prom+RBS as BV•pSB1c3+prom+RBS+bdhB+dt as BI•pSB1c3+prom+RBS+ctfA+dt as BV•pSB1c3+ prom+RBS+ctfB+dt+prom+RBS+bdhB+dt as BI
R: restrictions are succesful
we isolate the samples from the gel and ligate them over night:•pSB1c3+prom+RBS (BV) ligated with GFP+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfB+dt (BV) ligated with prom+RBS+bdhB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)<H3>7. 9.2015</H3>
we transform the cells with yesterday's ligations:•pSB1c3+prom+RBS (BV) ligated with GFP+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfB+dt (BV) ligated with prom+RBS+bdhB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)
colony PCR and AGE:•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt•pSB1c3+GFP+ctfB+dt
R: some constructs with GFP are succesful•we set restriction for pSB1c3+prom+RBS+ctfA+dt as BV, do an AGE, isolate the sample from the gel and freeze it<H3>8. 9.2015</H3>•we set restriction for prom+RBS+ctfB+dt+prom+RBS+bdhB+dt as BI, do an AGE, isolate the sample from the gel and set a ligation over night
pSB1c3+prom+RBS+ctfA+dt (BV) ligated with pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
We make colony PCR and AGE of:•pSB1c3+prom+RBS+GFP+ctfB+dt•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
R: some of the samples pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt and pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt are succesful<H3>9. 9.2015</H3>
we set restrictions and AGE of:•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt as BI•pSB1c3+prom+RBS+ctfA+dt as BV•pSB1c3+prom+RBS+ctfB+dt as BI
R: restrictions are succeful
we isolate the samples from the gel and set ligation over night:•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with •prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)
SDS-page, we make two gels, one to be coloured, the other for bloting:
•pSB1c3+prom+RBS+GFP+ctfB+dt
R: some bands of appropriate size appear
western blot and colouring of pSB1c3+prom+RBS+GFP+ctfB+dt
R: nothing shows on the membrane
we set restrictions over night:•pSB1c3+GFP as BI•pSB1c3+prom+RBS+ctfA+dt as BV•pSB1c3+prom+RBS+ctfB+dt as BI•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt as BI<H3>10. 9. 2015</H3>
we do the AGE of yesterday's restrictions, isolate them from the gel and set ligations for two hours:•pSB1c3+prom+RBS (BV) ligated with GFP (BI)•pSB1c3+prom+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)
we transform the cells with today's and yesterday's ligations:•pSB1c3+prom+RBS (BV) ligated with GFP (BI)•pSB1c3+prom+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt (BI)•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with prom+RBS+ctfB+dt+prom+RBS+bdhB+dt (BI)•colony PCR and AGE of
pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
R: some samples seem to be succesful<H3>11. 9.2015</H3>
SDS-page, we make two gels, one for colouring, other for bloting:•pSB1c3+prom+RBS+ctfB+dt•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
R: all samples seem to show the same results
we do western blot on one of the gels and colour the memebrane 
R: the membrane shows nothing of note
we set restrictions and AGE, we cut all samples with EcoRl and Pstl to check the size of fragments:•pSB1c3+prom+RBS+bdhB+dt•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
R: they all cut the same, which means the colony PCR were false pfitives and we have not managed to ligate anything togheter
colony PCR and AGE:•pSB1c3+prom+RBS+GFP•pSB1c3+prom+ctfA+dt+prom+RBS+ctfB+dt•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
R: some samples seem to be succesful
expression and SDS-page of BBa_K733012 and BBa_K863010
R: unsuccesful<H3>12. 9.2015</H3>
transformation of  BBa_K863005 
SDS-page: •pSB1c3+ctfB+dt•pSB1c3+prom+RBS+ctfB+dt•pSB1c3+prom+RBS+GFP•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt
R: bands appear
we set restriction and AGE:•pSB1c3+prom+RBS+ctfA+dt as BV•pSB1c3+prom+RBS+ctfB+dt as BI, BV•pSB1c3+prom+RBS+bdhB+dt as BI
R: restrictions are succesful
we isolate the samples<H3>13. 9.2015</H3>
we set ligation over night:•pSB1c3+prom+RBS+ctfA+dt (BV) ligated with +prom+RBS+ctfB+dt (BI) pSB1c3+prom+RBS+ctfB+dt (BV) ligated with prom+RBS+bdhB+dt (BI)<H3>14. 9. 2015</H3>•we transform the cells with yesterday's ligations
<H3>15. 9.2015</H3>
colony PCR and AGE:•pSB1c3+prom+RBS+ctfA+dt+prom+RBS+ctfB+dt•pSB1c3+prom+RBS+ctfB+dt+prom+RBS+bdhB+dt•expression of BBa_K863005
<H2>Growth Optimization Notebook</H2><H3>23. 3.2015</H3>
 fermentative production of hydrogen, sampling and analysis on HPLC<H3>24. 3.2015</H3>
 sampling and analysis on HPLC<H3>25. 3. 2015</H3>
sampling and analysis on HPLC<H3>26. 3.2015</H3>
 sampling and analysis on HPLC<H3>28. 4. 2015</H3>
analysis of previous results<H3>2. 5.2015</H3>
 fermentative production of hydrogen, sampling and analysis on HPLC<H3>3. 5. 2015</H3>
sampling and analysis on HPLC<H3>8. 5.2015</H3>
 learning the basics of working with bacteria, aerobic preparation of 4 different media, inoculation, measuring absorbance using spectrophotometry, analysis of previous results<H3>9. 5.2015</H3>
 learning the basics of working with bacteria, aseptic transfer of bacteria, measuring absorbance using spectrophotometry<H3>10. 5. 2015</H3>
learning the basics of working with bacteria, aseptic transfer of bacteria, measuring absorbance using spectrophotometry<H3>11. 5.2015</H3>
 learning the basics of working with bacteria, aseptic transfer of bacteria, measuring absorbance using spectrophotometry<H3>13. 5. 2015</H3>
aseptic transfer of bacteria, aerobic preparation of 4 different media, measuring absorbance using spectrophotometry<H3>14. 5. 2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>21. 5. 2015</H3>
learning the basics of working with bacteria, aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>25. 5.2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>26. 5. 2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>27. 5. 2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>28. 5.2015</H3>
 aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>29. 5.2015</H3>
 aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>3. 6. 2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>4. 6. 2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>11. 6. 2015</H3>
aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>12. 6.2015</H3>
 aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>15. 6.2015</H3>
 aseptic transfer of bacteria into liquid medium, measuring absorbance using spectrophotometry<H3>24. 6. 2015</H3>
aerobic and anaerobic preparation of 48 different media and saline soluton,  aseptic transfer of bacteria onto solid medium plates, aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry<H3>25. 6.2015</H3>
 measuring absorbance using  spectrophotometry, analysis of results<H3>26. 6.2015</H3>
 anaerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>29. 6.2015</H3>
 aerobic and anaerobic preparation of 72 different media and saline soluton,  aseptic transfer of bacteria onto solid medium plates, aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>30. 6.2015</H3>
 measuring absorbance using  spectrophotometry, analysis of results<H3>1. 7.2015</H3>
 measuring absorbance using  spectrophotometry, analysis of results<H3>2. 7.2015</H3>
 anaerobic  inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>3. 7.2015</H3>
 measuring absorbance using  spectrophotometry, analysis of results<H3>6. 7. 2015</H3>
aerobic and anaerobic preparation of 72 different media,  aseptic transfer of bacteria onto solid medium plates, aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>9. 7. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>10. 7. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>13. 7.2015</H3>
 aerobic and anaerobic preparation of 72 different media,  aseptic transfer of bacteria onto solid medium plates, aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, testing the new Perkin Elmer spectrophotometer, sterilization of working space<H3>14. 7.2015</H3>
 measuring absorbance using  spectrophotometry, testing the new Perkin Elmer spectrophotometer, sterilization of working space, analysis of results<H3>15. 7. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>16. 7.2015</H3>
 aseptic transfer of bacteria onto solid medium plates, anaerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, sterilization of working space, analysis of results<H3>17. 7. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>20. 7. 2015</H3>
aerobic and anaerobic preparation of 72 different media, aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry<H3>21. 7.2015</H3>
 analysis of results, ecotoxicological tests, preparing LB agar medium, isolation of n-butanol, measuring absorbance using  spectrophotometry<H3>22. 7.2015</H3>
 analysis of results, ecotoxicological tests, isolation of n-butanol, measuring absorbance using  spectrophotometry<H3>23. 7. 2015</H3>
analysis of results, ecotoxicological tests, measuring absorbance using  spectrophotometry<H3>24. 7. 2015</H3>
analysis of results, measuring absorbance using  spectrophotometry<H3>3. 8. 2015</H3>
aerobic and anaerobic preparation of 72 different media and saline soluton,  aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>4. 8. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>5. 8.2015</H3>
 aseptic transfer of bacteria onto solid medium plates, measuring absorbance using  spectrophotometry, analysis of results<H3>6. 8. 2015</H3>
anaerobic preparation of 72 different media,  anaerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>7. 8. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>10. 8. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results<H3>11. 8. 2015</H3>
aseptic transfer of bacteria onto solid medium plates, measuring absorbance using  spectrophotometry, analysis of results<H3>19. 8. 2015</H3>
aerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry<H3>20. 8.2015</H3>
 measuring absorbance using  spectrophotometry, preparation of Davis minimal medium, preparation of solutions for HPLC and GC methods<H3>21. 8. 2015</H3>
measuring absorbance using  spectrophotometry, analysis of results, preparation of solutions for HPLC method, measuring the prepared solutions on HPLC and GC<H3>24. 8. 2015</H3>
preparation of minimal medium solutions<H3>25. 8.2015</H3>
 measuring the prepared solutions on HPLC, analysis of results<H3>3. 9. 2015</H3>
anaerobic inoculation of bacteria into well plates, measuring absorbance using  spectrophotometry, analysis of results<H3>4. 9.2015</H3>
 measuring absorbance using  spectrophotometry<H3>8. 9.2015</H3>
 measuring absorbance using  spectrophotometry, analysis of results</SECTION></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>