<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toulouse_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toulouse/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Toulouse 2015</TITLE></P><DIV class="wrapper row0 fixed" style="max-height:87px;margin-top:-10px"><HEADER id="header" class="clear"><NAV id="mainav" class="center"><UL class="clear" style="padding-top:15px;"><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse"><B>Home</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse#context">Context</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse#achievements">Achievements</A></LI></UL><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Team" id="team"><B>Team</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Team">Presentation</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/FunFacts">Fun Facts</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description" id="project"><B>Project</B></A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description">Project: Description</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Attract">Design: Attract</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Eradicate">Design: Eradicate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Regulation">Design: Regulate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Design">Device: TrapiColi</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Results" id="Results"><B>Results</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#attract">Attract</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#eradicate">Eradicate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#regulate">Regulate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#TPX">Containment</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#biolo">Growth tests</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#tox">Cytoxicity</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Parts" id="parts"><B>Parts</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#biobricks">New BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#1">Module: Attract</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#2">Module: Eradicate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#3">Module: Regulate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Modeling" id="modeling"><B>Modeling</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part1">Metabolic networks</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part2">FBA</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part5">Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part6">Optimisation</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Practices" id="practices"><B>Human Practice</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Collaborations">Meetups &amp; Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Ethics">Ethics &amp; IP</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Safety">Safety</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Notebook" id="notebook"><B>Wet lab</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Experiments">Protocols</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Attributions" id="attributions"><B>Attributions</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Supports">Supports</A></LI></NAV></HEADER></DIV><DIV class="wrapper row0 bgded" style="background-image:url('https://static.igem.org/mediawiki/2015/8/8f/TLSE_bg_9.jpg')"><DIV class="shout"><DIV class="shout-content clear"><DIV class="maintitle"><CENTER><H3>Biobricks</H3></CENTER></DIV></DIV></DIV></DIV><DIV class="wrapper row3"><MAIN class="container clear"><DIV class="title"><A href="#main1"><H3>Content</H3></A></DIV><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#1">- Attraction (butyrate pathway)</A></LI><LI><A href="#2">- Eradication (formate pathway)</A></LI><LI><A href="#3">- Circadian switch</A></LI></UL></DIV></CENTER><DIV class="group center" id="biobricks"><P class="text" style="padding:10px;">
In the list below you will find an overview over the BioBrick parts created by the iGEM Toulouse 2015 team and added to the IGEM registry. 

For our project, we worked on three main biological modules : Attract, Eradicate, and Regulate to alternatively produce the two molecules of interest (butyrate by day, formate by night). 
</P></DIV><DIV class="group center"><TABLE class="result"><TBODY><TR><TH>Name</TH><TH>Type</TH><TH>Genic construction</TH><TH>Module</TH><TH>Length (bp)</TH><TH>Sequencing</TH><TH>References</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587000" target="_blank&quot;">BBa_K1587000</A></TD><TD>Composite (with RBS)</TD><TD>RBS-<I>ho1</I></TD><TD>Regulate</TD><TD>744</TD><TD>Ok</TD><TD>[13] [14]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587001" target="_blank&quot;">BBa_K1587001</A></TD><TD>Basic part</TD><TD><I>tesB</I></TD><TD>Attract</TD><TD>861</TD><TD>Ok</TD><TD>[3][4][5]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587002" target="_blank&quot;">BBa_K1587002</A></TD><TD>Composite (with RBS)</TD><TD>RBS-<I>pcyA</I></TD><TD>Regulate</TD><TD>796</TD><TD>Ok</TD><TD>[13][14]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587003" target="_blank&quot;">BBa_K1587003</A></TD><TD>Basic part</TD><TD><I>crt</I></TD><TD>Attract</TD><TD>786</TD><TD>Sequenced until 736 pb : ok </TD><TD>[1][2][3][4][7]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587004" target="_blank&quot;">BBa_K1587004</A></TD><TD>Device</TD><TD>P(Bla)-RBS-<I>ccr</I>-RBS-<I>hbd</I>-RBS-<I>crt</I>-RBS-
<I>tesB</I>-RBS-<I>atoB</I>-Terminator
</TD><TD>Attract</TD><TD>5192</TD><TD>Ok</TD><TD>[1][2][3][4][6]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587005" target="_blank&quot;">BBa_K1587005</A></TD><TD>Device</TD><TD>P(Bla)-RBS-<I>hbd</I>-RBS-<I>crt</I>-RBS-
<I>tesB</I>-RBS-<I>atoB</I>-Terminator</TD><TD>Attract</TD><TD>3827</TD><TD>Ok</TD><TD>[1][2][3][4][6]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587006" target="_blank&quot;">BBa_K1587006</A></TD><TD>Device</TD><TD>P<SUB>OmpC</SUB>-<I>LacI</I>box-RBS-<I>cI</I></TD><TD>Regulate</TD><TD>905</TD><TD>Ok</TD><TD>[12][13][14]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587007" target="_blank&quot;">BBa_K1587007</A></TD><TD>Device</TD><TD>RBS-<I>pflB</I>-RBS-<I>pflA</I>-Terminator</TD><TD>Eradicate</TD><TD>3093</TD><TD>Ok</TD><TD>[7][8][9][10] </TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587008" target="_blank&quot;">BBa_K1587008</A></TD><TD>Device</TD><TD>Strong Promotor-RBS-<I>cph8</I></TD><TD>Regulate</TD><TD>2288</TD><TD>Ok</TD><TD>[11][13][14]</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1587009" target="_blank&quot;">BBa_K1587009</A></TD><TD>Composite (with RBS)</TD><TD>P<SUB>OmpC</SUB>-<I>LacI</I>box-RBS-<I>cI</I>-RBS-<I>pflB</I>-RBS-<I>pflA</I>-Terminator</TD><TD>Eradicate</TD><TD>4006</TD><TD>-</TD><TD>[7][8][9][10][11][12][13][14]</TD></TR></TBODY></TABLE></DIV><CENTER><DIV class="title" id="1"><H3>Attraction (butyrate pathway)</H3></DIV></CENTER><DIV class="group center"><P class="text">
The chassis we used is <I>Escherichia coli</I>, and this bacterium is not able to naturally produce butyrate. That is why we introduced genes from others bacterial strains to synthesize this molecule.
</P></DIV><DIV class="subtitle"><H3>Basic parts</H3></DIV><DIV class="subsubtitle"><H3><I>tesB</I><A href="http://parts.igem.org/Part:BBa_K1587001" target="_blank&quot;">(BBa_K1587001)</A></H3></DIV><DIV class="group center"><P class="text">
A gene from <I>Escherichia coli</I> (Accession Number: <A href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10995" target="_blank">EG10995</A>) involved in the butyrate pathway that enables its production directly from acyl-coAs. This group of enzymes catalyzes the hydrolysis of acyl-CoAs into free fatty acid (in our case, butyryl-coA into butyrate) plus reduced coenzyme A (CoA-SH).
</P></DIV><DIV class="subsubtitle"><H3><I>crt</I><A href="http://parts.igem.org/Part:BBa_K1587003" target="_blank&quot;">(BBa_K1587003)</A></H3></DIV><DIV class="group center"><P class="text">
A gene from <I>Clostridium acetobutylicum</I> (Accession Number: <A href="http://www.biocyc.org/CACE272562/NEW-IMAGE?type=GENE&amp;object=GJIH-2688" target="_blank">GJIH-2688</A>) introduced in our 
bacterium after codon optimization in order to obtain a better 
expression in our strain. The <I>crt</I> enzyme substrate is 3-hydroxybutyryl CoA, 
and the product is Crotonyl CoA. This reaction does not need any coenzyme. 
</P></DIV><DIV class="subtitle"><H3>Other parts</H3></DIV><DIV class="subsubtitle"><H3><I>ccr-Butyrate pathway</I><A href="http://parts.igem.org/Part:BBa_K1587004" target="_blank&quot;">(BBa_K1587004)</A></H3></DIV><DIV class="group center"><P class="text">

This BioBrick construction is composed of a constitutive promoter P(Bla)
 (<A href="http://parts.igem.org/Part:BBa_I14018" target="_blank&quot;">BBa_I14018</A>) and 5 genes from three different micro-organisms : 
 in yellow are the <I>E. coli</I> genes, in blue those from <I>Clostridium 
 acetobutylicum</I> and finally, the purple gene is from <I>Streptomyces 
 collinus</I>. A Ribosome Binding Site (RBS; represented by a green circle; <A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>), is added between each gene in order to improve the proteic
 synthesis.  Finally, a strong terminator <A href="http://parts.igem.org/Part:BBa_B1006" target="_blank&quot;">BBa_B1006)</A> represents the end of the sequence.   
<A href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10995" target="_blank"><I>tesB</I></A> and <A href="http://www.biocyc.org/CACE272562/NEW-IMAGE?type=GENE&amp;object=GJIH-2688" target="_blank"><I>crt</I></A> have been described above. <A href="http://www.uniprot.org/uniprot/Q53865" target="_blank"><I>ccr</I></A> encodes a crotonyl-CoA reductase (an oxidoreductase which acts on the double bond CH=CH). <A href="http://www.biocyc.org/CACE272562/NEW-IMAGE?type=GENE&amp;object=GJIH-2684" target="_blank"><I>hbd</I></A> from <I>Clostridium acetobutylicum</I> encodes a 3-hydroxybutyryl-CoA 
dehydrogenase (an oxidoreductase which catalyses the formation of alcohol function). <A href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG11672" target="_blank"><I>atoB</I></A>, from <I>E. coli</I>, encodes an acetyl-CoA acetyltransferase which catalyses the condensation of two acetyl-CoA. 

</P></DIV><DIV class="subsubtitle"><H3><I>Butyrate pathway wihout ccr</I><A href="http://parts.igem.org/Part:BBa_K1587005" target="_blank&quot;">(BBa_K1587005)</A></H3></DIV><DIV class="group center"><P class="text">  
This BioBrick construction is the same as the previous one, but for the fact that it does not 
contain the <I>ccr</I> gene from <I>Streptomyces collinus</I>. 
It is composed of a constitutive promoter P(Bla) (<A href="http://parts.igem.org/Part:BBa_I14018" target="_blank&quot;">BBa_I14018</A>) and of 4 of the 5 genes present in the previous construction. Genes are issued from two micro-organisms : in yellow from 
<I>E. coli</I>, and in blue from <I>Clostridium acetobutylicum</I>. The green 
circles correspond to the strong RBS sequence (<A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>) based on Ron Weiss thesis and the red one is the terminator (<A href="http://parts.igem.org/Part:BBa_B1006" target="_blank&quot;">BBa_B1006</A>).  
</P></DIV><CENTER><DIV class="title" id="2"><H3>Eradication (formate pathway)</H3></DIV></CENTER><DIV class="group center"><P class="text">
The second module is dedicated to formate production and composed of two key genes : 
 <A href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10028" target="_blank"><I>pflA</I></A> and <A href="http://ecocyc.org/ECOLI/NEW-IMAGE?type=GENE&amp;object=EG10701" target="_blank"><I>pflB</I></A>.  
  </P></DIV><DIV class="subtitle"><H3>Formate pathway (<A href="http://parts.igem.org/Part:BBa_K1587007" target="_blank&quot;">BBa_K1587007</A>)</H3></DIV><DIV class="group center"><P class="text"><I>pflB</I> encodes a pyruvate formate lyase, an enzyme which catalyses
 the cutting between C1 and C2 carbons of pyruvate. This enzyme 
 is oxygen-sensitive and is only active in microaerobic or anaerobic 
 conditions. 
<I>pflA</I> encodes the pyruvate formate lyase activase, an enzyme 
which activates <I>pflB</I>. 


 
 
The formate compound is naturally produced by <I>E. coli</I>, that is why we 
decided to overexpress these two genes. For this, <I>pflB</I> and <I>pflA</I> are put 
together with two RBS (<A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>) in front of them to allow the proteic synthesis. A strong terminator (<A href="http://parts.igem.org/Part:BBa_B1006" target="_blank&quot;">BBa_B1006</A>) ends up the sequence.
</P></DIV><CENTER><DIV class="title" id="3"><H3>Circadian switch</H3></DIV></CENTER><DIV class="group center"><P class="text">
We aimed to produce butyric acid in our trap during
 the day and formic acid during the night. 
 We designed a light response system which is improved to obtain an 
 on/off switch of genic expression.
 
The center of the light sensor is composed of membrane proteins PCB 
(chromophore phycocyanobilin) and the hybrid protein Cph8 (EnvZ and Cph1)[11].

PCB protein comes from a cyanobacterium <I>Synechocystis sp.</I> PCC 6803 
and to be synthesized, it needs the expression of two genes: 
heme oxygenase (<I>Ho1</I>) and biliverdin reductase (<I>PcyA</I>)[11]. 
</P></DIV><DIV class="subtitle"><H3><I>Ho1</I> with RBS (<A href="http://parts.igem.org/Part:BBa_K1587000" target="_blank&quot;">BBa_K1587000</A>)</H3></DIV><DIV class="group center"><P class="text">
A gene required for chromophore synthesis in photosynthetic 
light-harvesting complexes, photoreceptors, and circadian clocks. 
<I>ho1</I>, along with <I>pcyA</I>, converts heme into the chromophore 
phycocyanobillin (PCB). The gene comes from the cyanobacterium <I>Synechocystis sp. </I>PCC 6803.

The biobrick is composed of a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>) and <I>Ho1</I> coding region (<A href="http://parts.igem.org/Part:BBa_K566022" target="_blank&quot;">BBa_K566022</A>). 
</P></DIV><DIV class="subtitle"><H3><I>PcyA</I> with RBS (<A href="http://parts.igem.org/Part:BBa_K1587002" target="_blank&quot;">BBa_K1587002</A>)</H3></DIV><DIV class="group center"><P class="text">
A gene required for chromophore synthesis in photosynthetic 
light-harvesting complexes, photoreceptors, and circadian clocks. The gene comes from the cyanobacterium <I>Synechocystis sp.</I> PCC 6803.

The biobrick is composed of a strong RBS (<A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>) and <I>pcyA</I> 
coding region (<A href="http://parts.igem.org/Part:BBa_K566023" target="_blank&quot;">BBa_K566023</A>).
</P></DIV><DIV class="subtitle"><H3>P<SUB>OmpC</SUB>-<I>LacIbox</I>-RBS-<I>cI</I> (<A href="http://parts.igem.org/Part:BBa_K1587006" target="_blank&quot;">BBa_K1587006</A>) </H3></DIV><DIV class="group"><P class="text">
This biobrick contains the OmpC promoter (<A href="http://parts.igem.org/Part:BBa_R0082" target="_blank&quot;">BBa_R0082</A>), a lacI box 
separated from <I>cI</I> gene by a RBS sequence (<A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>). 
</P></DIV><DIV class="subtitle"><H3>Red light biosensor (<A href="http://parts.igem.org/Part:BBa_K1587008" target="_blank&quot;">BBa_K1587008</A>)  </H3></DIV><DIV class="group center"><P class="text">
This is an assembly between the strong promoter <A href="http://parts.igem.org/Part:BBa_J23119" target="_blank&quot;">BBa_J23119</A>, an RBS (<A href="http://parts.igem.org/Part:BBa_B0030" target="_blank&quot;">BBa_B0030</A>), and the chimeric Red light receptor Cph8 <A href="http://parts.igem.org/Part:BBa_I15010" target="_blank&quot;">BBa_I15010</A>.</P></DIV><DIV class="subtitle"><H3>Regulated formate production (<A href="http://parts.igem.org/Part:BBa_K1587009" target="_blank&quot;">BBa_K1587009</A>)  </H3></DIV><DIV class="group center"><P class="text">
An assembly between biobricks <A href="http://parts.igem.org/Part:BBa_K1587006" target="_blank&quot;">BBa_K1587006</A> 
and <A href="http://parts.igem.org/Part:BBa_K1587007" target="_blank&quot;">BBa_K1587007</A>, which have been both successfully sequenced, for the regulation of formate
 production by the circadian switch system.
</P></DIV></MAIN></DIV><DIV class="wrapper row4"><DIV class="container clear" style="padding-top:30px;"><CENTER><P class="maintitle">		  
References
</P></CENTER><DIV class="clear"><UL><LI>
[1] Layton DS &amp; Trinh CT (2014) Engineering modular ester fermentative pathways in <I>Escherichia coli</I>. Metabolic Engineering 26: 77–88
</LI><LI>
	
[2] Saini M, Hong Chen M, Chiang C-J &amp; Chao Y-P (2015) Potential production platform of n-butanol in <I>Escherichia coli</I>. Metabolic Engineering 27: 76–82
</LI><LI>
[3] Volker AR, Gogerty DS, Bartholomay C, Hennen-Bierwagen T, Zhu H &amp; Bobik TA (2014) Fermentative production of short-chain fatty acids in <I>Escherichia coli</I>. Microbiology (Reading, Engl.) 160: 1513–1522 
</LI><LI>
[4] Saini M, Wang ZW, Chiang C-J &amp; Chao Y-P (2014) Metabolic Engineering of <I>Escherichia coli</I> for Production of Butyric Acid. J. Agric. Food Chem. 62: 4342–4348 
</LI><LI>
[5] Hunt MC &amp; Alexson SEH (2002) The role Acyl-CoA thioesterases play in mediating intracellular lipid metabolism. Prog. Lipid Res. 41: 99–130 
</LI><LI>
[6] Aboulnaga E-H, Pinkenburg O, Schiffels J, El-Refai A, Buckel W &amp; Selmer T (2013) Effect of an oxygen-tolerant bifurcating butyryl coenzyme A dehydrogenase/electron-transferring flavoprotein complex from <I>Clostridium difficile</I> on butyrate production in <I>Escherichia coli</I>. J. Bacteriol. 195: 3704–3713
</LI><LI>
[7]  Thesis: SUNYA Sirichai. July 2012. Dynamique de la réponse physiologique d’Escherichia coli à des perturbations maîtrisées de son environnement : vers le développement de nouveaux outils de changement d’échelle. Ingénieries Enzymatique et Microbienne. 
</LI><LI>
[8]  CEA-CNRS- Aix Marseille Université. February 2015. Paris . Activation d’enzymes bactériennes pour convertir le CO2 en source d’énergie renouvelable.  
</LI><LI>
[9] Crain AV &amp; Broderick JB (2014) Pyruvate formate-lyase and its activation by pyruvate formate-lyase activating enzyme. J. Biol. Chem. 289: 5723–5729 
</LI><LI>
[10] Wei X-X, Zheng W-T, Hou X, Liang J, Li Z-J, Wei X-X, Zheng W-T, Hou X, Liang J &amp; Li Z-J (2015) Metabolic Engineering of <I>Escherichia coli</I> for Poly(3-hydroxybutyrate) Production under Microaerobic Condition. BioMed Research International, BioMed Research International 2015, 2015: e789315 
</LI><LI>
[11] Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, Davidson EA, Scouras A, Ellington AD, Marcotte EM &amp; Voigt CA (2005) Synthetic biology: Engineering <I>Escherichia coli</I> to see light. Nature 438: 441–442
</LI><LI>
[12] Okada K (2009) HO1 and PcyA proteins involved in phycobilin biosynthesis form a 1:2 complex with ferredoxin-1 required for photosynthesis. FEBS Lett. 583: 1251–1256
</LI><LI>
[13] Lee JM, Lee J, Kim T &amp; Lee SK (2013) Switchable gene expression in <I>Escherichia coli</I> using a miniaturized photobioreactor. PLoS ONE 8: e52382 
</LI><LI>
[14] Gambetta GA &amp; Lagarias JC (2001) Genetic engineering of phytochrome biosynthesis in bacteria. PNAS 98: 10566–10571 
</LI></UL></DIV></DIV></DIV><DIV class="wrapper row5"><FOOTER id="footer" class="clear"><DIV class="group"><DIV class="one_third first"><H6 class="title">iGEM Toulouse 2015</H6><ADDRESS class="btmspace-15">
        INSA Toulouse
        135, Avenue de Rangueil
        31400 TOULOUSE
		France</ADDRESS></DIV><DIV class="one_third"><H6 class="title">Follow us</H6></DIV><DIV class="one_third"><H6 class="title">For more informations</H6><A href="mailto:igemtoulouse2015@gmail.com" style="border:1px solid #fff;-webkit-border-radius: 5px; 
		-moz-border-radius: 10px; border-radius: 5px; padding: 7px 16px 5px; color:white; 
		text-decoration:none; font-size:17px; background:none;  display:inline-block;">Contact</A></DIV></DIV><CENTER><P>Website by <A href="https://2015.igem.org/Team:Toulouse">Team iGEM Toulouse 2015</A>&amp;<A href="http://www.os-templates.com/" title="Free Website Templates">OS Templates</A></P></CENTER></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>