Team:Toulouse/Experiments
iGEM Toulouse 2015
Home
Context
Achievements
Team
Presentation
Gallery
Fun Facts
Project
Project: Description
Design: Attract
Design: Eradicate
Design: Regulate
Device: TrapiColi
Results
Attract
Eradicate
Regulate
Containment
Growth tests
Cytoxicity
Parts
New BioBricks
Module: Attract
Module: Eradicate
Module: Regulate
Modeling
Metabolic networks
FBA
Diffusion
Optimisation
Human Practice
Meetups & Collaborations
Ethics & IP
Education
Safety
Wet lab
Notebook
Protocols
Attributions
Attributions
Supports
Experiments & Protocols
Content
- Protocols for varroa tests
- Protocols for culture tests
- Protocol for Polymerase Chain Reaction
- Protocols for TPX permeability tests
- Transformation Protocol: RbCl method
- Cloning protocol
- InFusion cloning protocol
Protocols for varroa tests
- Sampling of varroa
- Standardization of varroas and sampling
- Attraction test on varroas
- Mortality test on varroas
Sampling of varroa
To run tests with varroas it is necessary to get them back from beehive directly because they cannot live without bees.
Materials
Bee hive
Beekeeper suit
Gloves
Smoker
Dry twigs
Tweezers
Big brush
Small brush
Petri dishes
Ø x h = 35 x 15 mm
Methods
Slip beekeeper suit and gloves on and go to beehive
Fire dry twigs in smoker
Open bee hive and activate smoker to get bees inside the hive
Take a frame out the hive and remove bees with big brush and smoker
Close beehive
In the lab, put the frame on a table against the wall
With tweezer drill hole into one beehive cell
Remove larvae and look for varroas on larvae and on beehive cell
If there are varroas, take them with a small brush and put them on Petri dishes
Make sure there are two or three larvae on Petri dishes in order to allow survival of varroas
Start again step 7 to 9 until you have enough varroas
Steps 1, 4 & 7: Our teams members gathering varroas on infected larvae
Standardization of varroas and sampling
When we take varroas directly from frame, as it is described in  protocol “Sampling Varroas”, we have varroas in different phases. In order to have varroas  in the same phase it is necessary to add one step and it is important for reproducibility of the experiments. With this method we place varroas on adult bees so all varroas will be in phoretic phase.
Materials
Bees in box with aeration and glucose
Varroas from protocol “Sampling varroas”
Gas cylinder of CO2
Small brush
Tweezers
Petri dishes
Ø x h = 35 x 15 mm
Methods
With small brush take varroas from Petri dish and put them on bees in box through aeration holes
Place the box in a 35 °C incubator overnight. Make sure you have a bowl with water in order to have enough humidity in incubator
Take the box out of incubator
Add CO2 from gas cylinder into the box until all bees fall down
Open the box, take a bee with tweezer and look for varroas
When you find a varroa take him with small brush and replace bee in the box
Start again step 5 and 6 until you have enough varroas
Steps 2 & 5: Varroas gathering on infected bees
Attraction test on varroas
In order to test the attraction effect of butyric acid  on varroas a Y test tube was built, as it is showed below. A glass pipe was chosen because on plastic varroas could load themselves with electrostatics and die.  For butyric acid, the concentration chosen 4 % (V/V) because this is the concentration used in the patent quoted (see “Attribution” part).
Materials
Pump wich expels air
15 mL Flacon tube
Plastic pipe
Ø = 10 mm
Glass T pipe
Ø = 10 mm, made by a glassworker
Plastic separator
Carded cotton
Absorbent cotton
5mL 4% (V/V) Butyric acid
5mL Water
Standardized varroas
Methods
Put a cotton on Petri dish and add 400 µL of one acid formic solution
Place three varroas on this Petri dish and close it
Start again step1 and 2 for each formic acid solution and water
Each 30 minutes check if varroas are alive. To do that:
When varroa heads for one side of Glass T tube and covers more than 2 cm test is over and we write down the side choosen by varroa (Butyric acid or Water)
Two tests can be made in the same time thanks to the plastic separator
br>
Glass T-tube: Varroa is going to butyric acid (at left)
Mortality test on varroas
Test the toxicity of formic acid  on varroas. When beekeepers use formic acid for long  treatment they place a diffuser at the top of the hive and formic acid concentration  was assessed at 200 ppm 1 on average which is equivalent to 7.8 mmol.m-3.  As gas concentration is difficult to evaluate we calculated the liquid concentration balance   using the ideal gas law and Henry’s law. To simplify calculation we noted down formic acid A.
$$ P\cdot V = n\cdot R\cdot T, \textrm{ideal gaz law} $$ $$ P_A = C_A\cdot R\cdot T = 7.826\cdot10^{-3}\times8.314\times293=19.96 Pa $$
PA: partial pressure of A in Pa
CA: Concentration of A in air in mol.m-3
R: perfect gaz constant = 8.314 J.mol-1.K-1
T: temperature in °K
$$ P_A = H_A\cdot C_{A,eq}, \textrm{Henry's law} $$ $$ C_{A,eq} = \frac{19.964}{0.019} = 1.019 mol.L^{-1}$$
CA,eq: equivalent concentration in liquid in mol.L-1
HA: Henry's constant = 0.019 Pa.m3mol-1
We chose a positive control with a higher concentration, 2 mol.L-1, and then decreasing concentration in order to identify which minimum concentration could kill varroa. We used water as a negative control. For this test we use varroas directly from frames because we did not have enough standardized varroas.
Materials
Petri dishes
Ø x h = 35 x 15 mm
Varroas form “Sampling varroas”
Cotton
Acid formic solutions:
2 mol.L-1
10 mmol.L-1
1 mmol.L-1
500 µmol.L-1
50 µmol.L-1
Water
Methods
Put a cotton on Petri dish and add 400 µL of one acid formic solution
Place three varroas in this Petri dish and close it
Start again step1 and 2 for each formic acid solution and water
Every 30 minutes check if varroas are alive. To do that:
Tap on Petri dish and see if varroa moves. If it does varroa is still alive, if not see below
Observe through a binocular magnifier if varroa move. If it does, it is still alive.
Varroa mortality experiment
Protocols for culture tests
- Cytotoxicity tests
- Culture on erlenmeyers and TubeSpin® Bioreactors
- NMR analysis
- Culture on 48 wells plates
- Enzyme kinetic
- Acids production test
- Test of gas concentration
Cytotoxicity tests
Choice of concentrations
In the beginning we tested high and low concentrations and we further adapted the concentrations. In the end we worked with these concentrations:
Butyric acid : 218 mM, 109 mM, 10.9 mM, 5.45 mM and 1.09 mM
Formic acid : 100 mM, 10 mM, 1 mM 500 µM, 100 µM, 50 µM and 25 µM
Materials
Optical reader, OPTIMA MARS Analysis
48 wells plates
Pre-culture of E. coli BW 25113
Acid solutions
Medium : LB, M9 15 mM of glucose or 30 mM of glucose
Methods
Add 400 µL of medium in each well
Add 50 µL of pre-culture
Add 50 µL of acid solution
Place the 48 well plate in the optical reader
Adjust parameters on computer.
Usually we set 250 cycles around 24 hours so we have an OD measurement every six minutes
Note: Each condition is tested in three replicates
Culture on erlenmeyers and TubeSpin® Bioreactors
      - Inoculation and sampling
Materials
Pre-culture of E. coli BW 25113 in LB
Spectrophotometer
1mL Spectrophotometer cuvettes
Centrifuge
Erlenmeyers
TubeSpin® Bioreactors from TPP brand
Medium : M9 15 mM of glucose or 30 mM of glucose
Incubators at 37 °C, 130 rpm and without agitation
1.5 mL Eppendorf
0.2 µm filters
Methods
Add 50 mL of medium on erlenmeyer and TubeSpin® Bioreactor
Inoculate from pre-culture to have OD600nm=0.1.
To do that centrifuge the appropriate volume of pre-culture, then remove LB medium and resuspend sediment with M9 medium to inoculate.
Note: This step permits to eliminate substrates from LB medium which could interfere during NMR analysis.
Place erlenmeyers in incubator 37 °C 130 rpm and TubeSpin® Bioreactor in incubator 37 °C without agitation
Sampling every two hours the first day:
Take 1 mL of culture in 1.5 mL Eppendorf.
For TubeSpin® Bioreactor use needle and syringe in order not to let air enter.
Add 100 µL of sample in spectrophotometer cuvette, complete with 900 µL water and measure OD600 nm with spectrophotometer
Centrifuge the rest of samples at 13,000 rpm during 3 minutes
Filter the supernatant through a 0.2 µm filter and conserve it at -20 °C
Days follow sample once a day with method below and plate on Petri dish an appropriate dilution in order to know if bacteria are alive
      - NMR analysis
Materials
Culture supernatants from -20°C
2.5 mM TSP (Trimethylsilyl propanoic acid) diluted in D2O
0.5 mm NMR tubes
1.5 mL Eppendorf
Spinners (5mm)
500 MHz Bruker Avance Spectrometer
Methods
Add 400 µL of culture supernatant in 1.5 mL Eppendorf
Add 100 µL of TSP solution
Place the mix in 0.5 mm NMR tubes
Place NMR tube into spinner, sample is ready to analyse
Micro-aerobic culture, filtration and NMR samples
500MHz NMR Spectrometer used for culture supernatant analysis
Culture on 48 wells plates
In order to determine the right concentration of polysaccharide and enzyme of  BioSilta kit we have to do several cultures at the same time. So, we use an optical  reader and 48 wells plates.
Materials
Optical reader, OPTIMA MARS Analysis
48 wells plates
Pre-culture of E. coli BW 25113
Different concentrations of BioSilta medium
For one concentration of BioSilta medium different concentrations of enzyme
Methods
Add 450 µL of medium in each well
Add 50 µL of pre-culture
Place the 48 well plate in the optical reader
Adjust parameters on computer. We tested culture between one day and ten days
Note: Each condition is tested in two replicates.  According to our results we adapt concentrations of Biosilta medium and enzyme,  results are exposed in device part.
Enzyme kinetic
Materials
Spectrophotometer
BioSilta medium
BioSilta enzyme solution named Reagent A (3000U/L)
Bradford’s reagent
1.5 mL Eppendorf
Standard solutions of glucose
Methods
For each standard solutions : in a 1.5 mL Eppendorf sample 10 µL and add 1 mL Bradford’s Reagent, wait 20 minutes and measure OD505 nm
Plot glucose concentration in function of OD505nm and determine the linear region
Add 1.5 mL of BioSilta medium and 45 µL of reagent A (50 U/L) in an Eppendorf
Sampling every 30 minutes:
Take 10 µL and add 1 mL of Bradford’s reagent in an Eppendorf
Wait 20 minutes
Measure OD505 nm
If OD505 nm is over linear region dilute sample and measure OD505 nm again
Stop sampling when glucose concentration no longer change
Acids production test
In order to test if E. coli produces formic acid and butyric acid with genes added, we made culture test with modified bacteria. We used the same protocol as “Culture on Erlenmeyers and TubeSpin® Bioreactors” with some changes:
Volume of culture : 30 mL
Add Ampicillin at 25 µg/mL to have selection pressure
The number of samples:
Sample at the beginning
Sample at the end of the first day
Sample after 24 hours culture and 48 hours culture
Test of gas concentration
The objective of our device is to produce gas, so we would like to know the gas composition of our culture.  So, we developed a system in order to recover acids gas.
Materials
4 hours culture in 50 mL Falcon in M9 medium with 15 mM of glucose
Silicon plugs adapted to 50 mL Falcon
Needles
0.2 µm filters
10 mL Syringes
Neoprene pipes Ø=0.8 mm
10 mM NaHCO3
1.5 mL Eppendorf
1 mL Sterile cone
Incubator 3 °C without agitation
Methods
Replace Falcon plug with silicon plug
Adjust filter on needle and peg it into silicon plug. Do it twice
Adjust neoprene pipe into each filter
Add 700 µL of NaHCO3 in an Eppendorf
At the end of first pipe put a sterile cone and immerse it into Eppendorf with NaHCO3
At the end of second pipe put a 10 mL syringe
After 24 hours culture, press 10 mL syringe in order to expel gas in NaHCO3 solution
Conserve samples at -20 °C before NMR analysis (see protocol foregoing)
Note 1: We used culture in M9 because with the “Acids production tests” we had data on this medium.
Note 2: 10 mM NaHCO3 solution was used because pH was 8.3 so it would permit acid gas solubilisation.
Photo 8: Gas concentration test with falcons
Protocol for Polymerase Chain Reaction (PCR), From Thermo Scientific™ DreamTaq™ Green PCR Master Mix
Materials
MilliQ water nuclease free (QSP)
PCR Mix 2X
Forward primer
Reverse primer
Template DNA
Thin walled PCR tube
Ice
Methods
Gently vortex and briefly centrifuge the PCR mix after thawing
Place a thin-walled PCR tube on ice and add the different components for a 50 μL PCR reaction
Gently vortex the samples
Perform PCR using the recommended thermal cycling conditions
The PCR Mix from Thermo Scientific contains Taq DNA polymerase, Green Buffer, MgCl2, dNTPs but also two tracking dyes and a density reagent that allows for direct loading of the PCR product on a migration gel.
The template DNA concentration has to be adapted in order to be between 10 pg and 1 μg in the final volume of 50 μL. The template DNA can come from a miniprep solution or from a single colony. The primer concentrations have to be between 0.1 μM and 1 μM.
Each PCR reaction has to be adapted to the length of the PCR products, and to the melting temperature Tm of the primers. The extension step lasts 1 min for PCR products up to 2 kb. For longer products, the extension time has to be prolonged by 1 min/kb.
Step
Temperature (°C)
Time
Number of cycles
Initial denaturation
95
1-3 min
1
Denaturation
95
30s
25-40
Annealing
Tm – 5°C
30s
Extension
72
Adapt to the length
Final extension
72
5-15 min
1
Protocols for TPX® permeability tests
- Preparation of TPX® bag
- Permeability test
- Sterility test of TPX® bag
- Culture test of E. coli in TPX® bag
Preparation of TPX® bag
Materials
TPX®, gas permeable plastic
Fusing machine
2 mM Formic acid solution
4 % (V/V) Butyric acid solution
Methods
Prepare plastic bag in sticking on 3 sides over 4 with fusing machine
Add 7 mL of appropriate solution in plastic bag
Stick on the last side with fusing machine
Permeability test
To test gas permeability of TPX® plastic, we use the same protocol as “Test of gas concentration”.  The only change is that no filters were used because the sterility is not necessary.
Photo 9: The device used for the permeability test
Sterility test of TPX® bag
    Demonstrate that the TPX® bag is impermeable to bacteria
Materials
TPX bags
M9 defined Medium
E. coli BW 25113
Steril laboratory glass bottle
Methods
Overnight culture of E. coli BW 25113 at 37 °C
Inoculate a small TPX® bag at OD600 nm = 0.1 in LB medium (Final Volume = 8 mL)
Negative Control: Fill a small TPX® bag with M9 medium (Final Volume=8 mL)
Dispose each small bag in a Steril glass measuring cylinder containing M9 medium
Incubate at 37 °C
Measure OD600 nm twice a day
Culture test of E. coli in TPX® bag
Materials
TPX bags
LB Medium
Steril clips
E. coli BW 25113
Steril laboratory flask
Methods
Overnight culture of E. coli BW 25113 at 37 °C
Inoculate a small TPX® bag at OD600 nm = 0.1 in LB medium (Final Volume = 8 mL)
Close the small bag via fusing machine and Put the closed small bag in a Steril laboratory flask
Positive Control: Inoculate a culture tube at OD600 nm = 0.1 in LB medium (Final Volume = 20 mL)
Incubate at 37 °C
Measure OD600 nm twice a day
Transformation Protocol: RbCl method
- Media and solution
- Preparation of Competent Cells
- Transformation of Competent Cells
- Minipreps
Media and solution
YETM 500 mL
TFB1 200 mL
TFB2 200 mL
2.5 g Yeast Extract
10 g Tryptone
5 g MgSO4.7H2O
Adjust pH to 7.5 with KOH
For Plates: add 7.5 g of Agar
0.59 g KOAc
2.42 g RbCl
0.29 g CaCl2.2H2O
1.98 g MnCl2.4H2O
Adjust to pH 5.8 with 0.2 M acetic acid
Add dH2O to 200 mL
Filter sterilize
Store refrigerated at 4°C
0.42 g MOPS
2.21 g CaCl2.2H20
0.24 g RbCl
30 g Glycerol
Adjust to pH 6.5 with KOH
Add dH2O to 200 mL
Filter sterilize
Store refrigerated at 4 °C
Preparation of Competent Cells
1. Streak cells from frozen stock onto YETM plate. Incubate overnight at 37°C
2. Pick a single fresh colony to inoculate 5 mL of YETM medium. Grow over night at 37°C.
Do not vortex cells at any time after this point in the procedure
3. Dilute 1 mL of culture into 50 mL YETM medium prewarmed to 37 °C
Grow at 37 °C for 2 hours with agitation
Volumes can be scaled up 5X and all of the 5 mL overnight culture can be used
4. Transfer culture to sterile 50 mL tube. Chill on ice/water 10-15 minutes
5. Centrifuge for 10 minutes at 2,000 rpm at 4 °C. Immediately aspirate off all of the supernatant
Do not allow cells to warm above 4 °C at any time in this procedure
6. Resuspend cells in 10 mL of ice-cold TFB1 with gentle re-pipetting. Use chilled glass or plastic pipette
7. Incubate cells on ice for 5 minutes
8. Repeat step 5
9. Resuspend cells in 2 mL of ice-cold TFB2 with gentle re-pipetting. Use micropipet tip (plastic)
10. Incubate cells on ice for 15 minutes
Cells may be used for transformation or frozen
To freeze: aliquot cell in 200 μL volumes into prechilled 0.5 mL microfuge tube (on ice)
Freeze immediately in liquid nitrogen
Store cells frozen at -80 °C
Transformation of Competent Cells
1. If starting with frozen competent cells, warm tube/cells by gently twirling between your fingers until just thawed.
Immediately place on ice for about 5 minutes
2. Add to 1,5 mL eppendorf on ice: 2-3 μL iGEM plate or 1 μL plasmid or 10 μL ligation.
3. Add 100 μL of competent cells and mix by gentle re-pipetting
4. Incubate cells on ice for 20-30 minutes
5. Heat shock the cells exactly 90 seconds at 42 °C
6. Return cells on ice for 2 minutes
7. Add 1 mL of YETM medium. Incubate at 37 °C for 45-60 minutes with slow gentle shaking
8. Plate 0.1-0.2 mL of transformed cells on LB-plate containing the appropriate antibiotic. Incubate  overnight at 37°C
Minipreps
1. Resuspend 4 to 12 colonies from the plate and name each colony taken on the tubes and on the plate (A, B, C, …)
2. Resuspend one colony per culture tube in 5 mL of LB medium with antibiotic
3. Let the culture grow overnight at 37 °C in a shaking incubator
4. Use the QIAprep spin Miniprep Kit for each culture tube. The last step consisting in the elution of the DNA is made with elution buffer or water at 55 °C
5. Keep the tubes at -20 °C
Cloning
- Digestion: both parts have the same antibiotic resistance
- Digestion: The two parts have a different antibiotic resistance
- Migration and gel extraction
- Ligation
- Transformation
First step: Digestion
Both parts have the same antibiotic resistance
Vector
Insert
Digestion control first enzyme
Digestion control second enzyme
Volume equivalent to 1 µg of vector miniprep
Volume equivalent to 1 µg of insert miniprep
Volume equivalent to 1 µg of vector miniprep
Volume equivalent to 1 µg of vector miniprep
1 µL of each restriction enzymes
1 µL of each restriction enzymes
1 µL of the first restriction enzyme
1 µL of the second restriction enzyme
2 µL of Fast Digest Green Buffer (Thermo Scientific™)
2 µL of Fast Digest Green Buffer (Thermo Scientific™)
2 µL of Fast Digest Green Buffer (Thermo Scientific™)
2 µL of Fast Digest Green Buffer (Thermo Scientific™)
Up to 20 µL of Milli-Q water
Up to 20 µL of Milli-Q water
Up to 20 µL of Milli-Q water
Up to 20 µL of Milli-Q water
Incubate 15 minutes at 37 °C
The two parts have a different antibiotic resistance
Both parts
Volume equivalent to 1 µg of DNA miniprep
1 µL of each restriction enzymes
2 µL of Fast Digest Green Buffer (Thermo Scientific™)
Up to 20 µL of Milli-Q water
Incubate 15 minutes at 37°C
Migration and gel extraction
1. Prepare a 1 % or 2 % electrophoresis agarose gel with 0.5 X TAE buffer
2. Put 20 µL of sample + 6 µL of marker (1 kb for 1 % gel and 100 pb for 2 %) into the well
3. Migration for 30 min at 100 V or 1 hour at 50 V
4. Bath 10 minutes in BET
5. Wash in water for 5 minutes
6. The gel extraction is realized thanks to the QIAGEN Gel Extraction Kit
Two ways to inactivate the enzymes for the vector
Use of DNA Clean up kit for the DNA fragment above 200 pb
Heat inactivation at 95 °C for 10 minutes
Second step: Ligation
Mix
Negative Control
Positive Control
Volume equivalent to 3 molecules of insert (for one molecule of vector)
no insert
Volume equivalent to 3 molecules of insert (for one molecule of vector)
Volume equivalent to 50 ng of digested vector
Volume equivalent to 50 ng of digested vector
Volume equivalent to 50 ng of undigested vector
2 µL of 10X T4 buffer
2 µL of 10X T4 buffer
2 µL of 10X T4 buffer
0.5 µL of T4 ligase
0.5 µL of T4 ligase
0.5 µL of T4 ligase
Up to 20 µL of Milli-Q water
Up to 20 µL of Milli-Q water
Up to 20 µL of Milli-Q water
 Incubate the ligation mix 15 minutes at room temperature (22°C)
 Keep the tubes in ice or at -20 °C to prepare the transformation
Third step: Transformation
1a. Take 10 µL of the ligation mix for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol
1b. Positive control: take 10 µL of the ligation mix for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol
1c. First restriction enzyme digestion control: take 10 µL of the corresponding digestion mix (First step) for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol
1d. Second restriction enzyme digestion control: take 10 µL of the corresponding digestion mix (First step) for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol
1e. Negative control: take 10 µL of the corresponding mix for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol
2. Plate the solution on selective medium overnight at 37 °C
InFusion cloning protocol
Principle of In Fusion cloning
1. Design the tailed-oligonucleotides for the vector and the inserts. The tail of the 5’oligonucleotide must be the 20 last nucleotides of the previous fragment and the tail of the 3’oligonucleotide must be the 20 first nucleotides of the next fragment.
2. Amplify the different fragments with the previously designed oligonucleotides
3. Clean the PCR products either using a spin column purification kit or by digesting with DpnI. For NEB DpnI, mix the different PCR products together, add 10X CutSmart Buffer and DpnI (1 µL in a 20 µL mix); incubate at 37°C for 20 minutes and inactive at 80 °C during 30 minutes.
4. Set up the In Fusion cloning reaction :
Ligation
Control
5X In-Fusion HD Enzyme Premix
2 µL
-
Clean PCR mix
4 µL
4 µL
MilliQ water nuclease free
4 µL
6 µL
Total
10 µL
10 µL
5. Incubate at 50 °C during 15 min and then cool on ice
6. Transform commercial ultra-competent cells (108 to 109 cfu/µg DNA) with 2.5 µL of the ligation using provided with the competent cells. Plate several 10-fold dilutions of the transformation mix.
iGEM Toulouse 2015
INSA Toulouse
135, Avenue de Rangueil
31400 TOULOUSE
France
Follow us
For more informations
Contact
Website by Team iGEM Toulouse 2015 & OS Templates
