<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toulouse_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toulouse/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Toulouse 2015</TITLE></P><DIV class="wrapper row0 fixed" style="max-height:87px;margin-top:-10px"><HEADER id="header" class="clear"><NAV id="mainav" class="center"><UL class="clear" style="padding-top:15px;"><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse"><B>Home</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse#context">Context</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse#achievements">Achievements</A></LI></UL><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Team" id="team"><B>Team</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Team">Presentation</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/FunFacts">Fun Facts</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description" id="project"><B>Project</B></A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description">Project: Description</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Attract">Design: Attract</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Eradicate">Design: Eradicate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Regulation">Design: Regulate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Design">Device: TrapiColi</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Results" id="Results"><B>Results</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#attract">Attract</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#eradicate">Eradicate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#regulate">Regulate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#TPX">Containment</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#biolo">Growth tests</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#tox">Cytoxicity</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Parts" id="parts"><B>Parts</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#biobricks">New BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#1">Module: Attract</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#2">Module: Eradicate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#3">Module: Regulate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Modeling" id="modeling"><B>Modeling</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part1">Metabolic networks</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part2">FBA</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part5">Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part6">Optimisation</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Practices" id="practices"><B>Human Practice</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Collaborations">Meetups &amp; Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Ethics">Ethics &amp; IP</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Safety">Safety</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Notebook" id="notebook"><B>Wet lab</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Experiments">Protocols</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Attributions" id="attributions"><B>Attributions</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Supports">Supports</A></LI></NAV></HEADER></DIV><DIV class="wrapper row0 bgded" style="background-image:url('https://static.igem.org/mediawiki/2015/3/33/TLSE_bg_7.png')"><DIV class="shout"><DIV class="shout-content clear"><DIV class="maintitle"><CENTER><H3>Experiments &amp; Protocols</H3></CENTER></DIV></DIV></DIV></DIV><DIV class="wrapper row3"><MAIN class="container clear"><DIV class="title"><A href="#main1"><H3>Content</H3></A></DIV><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#varroatest">- Protocols for varroa tests</A></LI><LI><A href="#culturetest">- Protocols for culture tests</A></LI><LI><A href="#PCR">- Protocol for Polymerase Chain Reaction</A></LI></UL></DIV><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#TPX">- Protocols for TPX permeability tests</A></LI><LI><A href="#transfo">- Transformation Protocol: RbCl method</A></LI><LI><A href="#cloning">- Cloning protocol</A></LI></UL></DIV><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#infusion">- InFusion cloning protocol</A></LI></UL></DIV></CENTER><DIV class="group center"><DIV class="title" id="varroatest"><H3>Protocols for varroa tests</H3><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#samplingV">- Sampling of varroa</A></LI><LI><A href="#standardV">- Standardization of varroas and sampling</A></LI><LI><A href="#attractV">- Attraction test on varroas</A></LI><LI><A href="#mortalV">- Mortality test on varroas</A></LI></UL></DIV></CENTER></DIV></DIV><DIV class="group"><DIV class="subtitle" id="samplingV"><H3>Sampling of varroa</H3></DIV></DIV><DIV class="group"><P align="justify" style="font-size:15px;">To run tests with varroas it is necessary to get them back from beehive directly because they cannot live without bees. </P></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Bee hive </LI><LI>Beekeeper suit</LI><LI>Gloves</LI><LI>Smoker</LI><LI>Dry twigs</LI><LI>Tweezers</LI><LI>Big brush</LI><LI>Small brush</LI><LI>Petri dishes  Ø x h = 35 x 15 mm</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Slip beekeeper suit and gloves on and go to beehive</LI><LI>Fire dry twigs in smoker</LI><LI>Open bee hive and activate smoker to get bees inside the hive</LI><LI>Take a frame out the hive and remove bees with big brush and smoker</LI><LI>Close beehive</LI><LI>In the lab, put the frame on a table against the wall</LI><LI>With tweezer drill hole into one beehive cell</LI><LI>Remove larvae and look for varroas on larvae and on beehive cell</LI><LI>If there are varroas, take them with a small brush and put them on Petri dishes</LI><LI>Make sure there are two or three larvae on Petri dishes in order to allow survival of varroas</LI><LI>Start again step 7 to 9 until you have enough varroas</LI></OL></DIV><CENTER><P class="legend">
Steps 1, 4 &amp; 7: Our teams members gathering varroas on infected larvae
</P></CENTER><DIV class="group"><DIV class="subtitle" id="standardV"><H3>Standardization of varroas and sampling</H3></DIV></DIV><DIV class="group center"><P align="justify" style="font-size:15px;">When we take varroas directly from frame, as it is described in 
protocol “Sampling Varroas”, we have varroas in different phases. In order to have varroas 
in the same phase it is necessary to add one step and it is important for reproducibility of the
experiments. With this method we place varroas on adult bees so all varroas will be in phoretic phase.</P></DIV><DIV class="group" style="padding-top:10px"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Bees in box with aeration and glucose </LI><LI>Varroas from protocol “Sampling varroas”</LI><LI>Gas cylinder of CO<SUB>2</SUB></LI><LI>Small brush </LI><LI>Tweezers</LI><LI>Petri dishes  Ø x h = 35 x 15 mm</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>With small brush take varroas from Petri dish and put them on bees in box through aeration holes</LI><LI>Place the box in a 35 °C incubator overnight. Make sure you have a bowl with water in order to have enough humidity in incubator</LI><LI>Take the box out of incubator</LI><LI>Add CO<SUB>2</SUB> from gas cylinder into the box until all bees fall down</LI><LI>Open the box, take a bee with tweezer and look for varroas</LI><LI>When you find a varroa take him with small brush and replace bee in the box</LI><LI>Start again step 5 and 6 until you have enough varroas</LI></OL></DIV><CENTER><P class="legend">
Steps 2 &amp; 5: Varroas gathering on infected bees
</P></CENTER><DIV class="group"><DIV class="subtitle" id="attractV"><H3>Attraction test on varroas</H3></DIV></DIV><DIV class="group center"><P align="justify" style="font-size:15px;">In order to test the attraction effect of butyric acid 
on varroas a Y test tube was built, as it is showed below. A glass pipe was chosen because on plastic varroas could load themselves with electrostatics and die. 
For butyric acid, the concentration chosen 4 % (V/V) because this is the concentration used in the patent quoted (see “Attribution” part). </P></DIV><DIV class="group" style="padding-top:10px"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Pump wich expels air</LI><LI>15 mL Flacon tube</LI><LI>Plastic pipe  Ø = 10 mm</LI><LI>Glass T pipe  Ø = 10 mm, made by a glassworker</LI><LI>Plastic separator</LI><LI>Carded cotton</LI><LI>Absorbent cotton</LI><LI>5mL 4% (V/V) Butyric acid</LI><LI>5mL Water</LI><LI>Standardized varroas </LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Put a cotton on Petri dish and add 400 µL of one acid formic solution</LI><LI>Place three varroas on this Petri dish and close it</LI><LI>Start again step1 and 2 for each formic acid solution and water</LI><LI>Each 30 minutes check if varroas are alive. To do that:</LI><LI>When varroa heads for one side of Glass T tube and covers more than 2 cm test is over and we write down the side choosen by varroa (Butyric acid or Water)</LI><LI>Two tests can be made in the same time thanks to the plastic separator</LI></OL></DIV>br&gt;
<CENTER><P class="legend">
Glass T-tube: Varroa is going to butyric acid (at left) 
</P></CENTER><DIV class="group"><DIV class="subtitle" id="mortalV"><H3>Mortality test on varroas</H3></DIV></DIV><DIV class="group center"><P align="justify" style="font-size:15px;">Test the toxicity of formic acid 
on varroas. When beekeepers use formic acid for long 
treatment they place a diffuser at the top of the hive and formic acid concentration 
was assessed at 200 ppm<A target="_blank" href="http://www.agroscope.admin.ch/imkerei/00316/00329/02079/index.html?lang=en"><SUP>1</SUP></A>on average which is equivalent to 7.8 mmol.m<SUP>-3</SUP>. 
As gas concentration is difficult to evaluate we calculated the liquid concentration balance  
using the ideal gas law and Henry’s law. To simplify calculation we noted down formic acid A.
 </P></DIV><DIV class="group center"><P style="font-size:15px;">
$$ P\cdot V = n\cdot R\cdot T, \textrm{ideal gaz law} $$

$$ P_A = C_A\cdot R\cdot T = 7.826\cdot10^{-3}\times8.314\times293=19.96 Pa $$
</P></DIV><UL style="font-size:15px;"><LI>P<SUB>A</SUB>: partial pressure of A in Pa</LI><LI>C<SUB>A</SUB>: Concentration of A in air in mol.m<SUP>-3</SUP></LI><LI>R: perfect gaz constant = 8.314 J.mol<SUP>-1</SUP>.K<SUP>-1</SUP></LI><LI>T: temperature in <SUP>°</SUP>K </LI></UL><DIV class="group center" style="padding-top:10px;"><P style="font-size:15px;">
$$ P_A = H_A\cdot C_{A,eq}, \textrm{Henry's law} $$

$$ C_{A,eq} = \frac{19.964}{0.019} = 1.019 mol.L^{-1}$$
</P></DIV><UL style="font-size:15px;"><LI>C<SUB>A,eq</SUB>: equivalent concentration in liquid in mol.L<SUP>-1</SUP></LI><LI>H<SUB>A</SUB>: Henry's constant = 0.019 Pa.m<SUP>3</SUP>mol<SUP>-1</SUP></LI></UL><DIV class="group center"><P align="justify" style="font-size:15px;">We chose a positive control with a higher concentration, 2 mol.L<SUP>-1</SUP>, and then decreasing concentration in order to identify which minimum concentration could kill varroa. We used water as a negative control.
For this test we use varroas directly from frames because we did not have enough standardized varroas.
</P></DIV><DIV class="group" style="padding-top:10px"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Petri dishes  Ø x h = 35 x 15 mm</LI><LI>Varroas form “Sampling varroas”</LI><LI>Cotton</LI><LI>Acid formic solutions: </LI><UL align="justify" style="font-size:15px;"><LI>2 mol.L<SUP>-1</SUP></LI><LI>10 mmol.L<SUP>-1</SUP></LI><LI>1 mmol.L<SUP>-1</SUP></LI><LI>500 µmol.L<SUP>-1</SUP></LI><LI>50 µmol.L<SUP>-1</SUP></LI></UL><LI>Water</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Put a cotton on Petri dish and add 400 µL of one acid formic solution</LI><LI>Place three varroas in this Petri dish and close it</LI><LI>Start again step1 and 2 for each formic acid solution and water</LI><LI>Every 30 minutes check if varroas are alive. To do that: </LI><OL align="justify" style="font-size:15px;"><LI>Tap on Petri dish and see if varroa moves. If it does varroa is still alive, if not see below</LI><LI>Observe through a binocular magnifier if varroa move. If it does, it is still alive. </LI></OL></OL></DIV><CENTER><P class="legend">
Varroa mortality experiment
</P></CENTER><CENTER><DIV class="title" id="culturetest"><H3>Protocols for culture tests</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#CytotoxicityTests">- Cytotoxicity tests</A></LI><LI><A href="#erlencult">- Culture on erlenmeyers and TubeSpin<SUP>®</SUP> Bioreactors</A></LI><LI><A href="#NMR">- NMR analysis</A></LI><LI><A href="#platecult">- Culture on 48 wells plates</A></LI></UL></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#enzymekinetic">- Enzyme kinetic</A></LI><LI><A href="#acidsprod">- Acids production test</A></LI><LI><A href="#gasconcentration">- Test of gas concentration</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="CytotoxicityTests"><H3>Cytotoxicity tests</H3></DIV><DIV class="subsubsubtitle"><H3>Choice of concentrations</H3></DIV><DIV class="group"><P align="justify" style="font-size:15px;">
In the beginning we tested high and low concentrations and we further adapted the concentrations. In the end we worked with these concentrations:
</P></DIV><DIV class="group"><UL style="font-size:15px;" align="left"><LI>Butyric acid : 218 mM, 109 mM, 10.9 mM, 5.45 mM and 1.09 mM</LI><LI>Formic acid : 100 mM, 10 mM, 1 mM 500 µM, 100 µM, 50 µM and 25 µM </LI></UL></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Optical reader, OPTIMA MARS Analysis</LI><LI>48 wells plates</LI><LI>Pre-culture of <I>E. coli</I> BW 25113</LI><LI>Acid solutions</LI><LI>Medium : LB, M9 15 mM of glucose or 30 mM of glucose</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Add 400 µL of medium in each well</LI><LI>Add 50 µL of pre-culture</LI><LI>Add 50 µL of acid solution</LI><LI>Place the 48 well plate in the optical reader</LI><LI>Adjust parameters on computer.  
			Usually we set 250 cycles around 24 hours so we have an OD measurement every six minutes</LI></OL></DIV><P align="justify" style="font-size:15px;"><U>Note:</U> Each condition is tested in three replicates
</P><DIV class="subtitle" id="erlencult"><H3>Culture on erlenmeyers and TubeSpin<SUP>®</SUP> Bioreactors</H3><H3>      - Inoculation and sampling</H3></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Pre-culture of <I>E. coli</I> BW 25113 in LB</LI><LI>Spectrophotometer</LI><LI>1mL Spectrophotometer cuvettes</LI><LI>Centrifuge</LI><LI>Erlenmeyers</LI><LI>TubeSpin<SUP>®</SUP> Bioreactors from TPP brand</LI><LI>Medium : M9 15 mM of glucose or 30 mM of glucose</LI><LI>Incubators at 37 °C, 130 rpm and without agitation</LI><LI>1.5 mL Eppendorf</LI><LI>0.2 µm filters</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Add 50 mL of medium on erlenmeyer and TubeSpin<SUP>®</SUP> Bioreactor</LI><LI>Inoculate from pre-culture to have OD<SUB>600nm</SUB>=0.1. 
			To do that centrifuge the appropriate volume of pre-culture, then remove LB medium and resuspend sediment with M9 medium to inoculate.<U>Note:</U> This step permits to eliminate substrates from LB medium which could interfere during NMR analysis.</LI><LI>Place erlenmeyers in incubator 37 °C 130 rpm and TubeSpin<SUP>®</SUP> Bioreactor in incubator 37 °C without agitation</LI><LI>Sampling every two hours the first day:</LI><UL><LI>Take 1 mL of culture in 1.5 mL Eppendorf. 
			For TubeSpin<SUP>®</SUP> Bioreactor use needle and syringe in order not to let air enter.
</LI><LI>Add 100 µL of sample in spectrophotometer cuvette, complete with 900 µL water and measure OD<SUB>600 nm</SUB> with spectrophotometer</LI><LI>Centrifuge the rest of samples at 13,000 rpm during 3 minutes</LI><LI>Filter the supernatant through a 0.2 µm filter and conserve it at -20 °C
</LI></UL><LI>Days follow sample once a day with method below and plate on Petri dish an appropriate dilution in order to know if bacteria are alive</LI></OL></DIV><DIV class="subtitle" id="NMR"><H3>      - NMR analysis</H3></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Culture supernatants from -20°C</LI><LI>2.5 mM TSP (Trimethylsilyl propanoic acid) diluted in D<SUB>2</SUB>O
</LI><LI>0.5 mm NMR tubes</LI><LI>1.5 mL Eppendorf</LI><LI>Spinners (5mm)</LI><LI>500 MHz Bruker Avance Spectrometer</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Add 400 µL of culture supernatant in 1.5 mL Eppendorf
</LI><LI>Add 100 µL of TSP solution</LI><LI>Place the mix in 0.5 mm NMR tubes</LI><LI>Place NMR tube into spinner, sample is ready to analyse</LI></OL></DIV><CENTER><P class="legend">
Micro-aerobic culture, filtration and NMR samples
</P></CENTER><CENTER><P class="legend">
500MHz NMR Spectrometer used for culture supernatant analysis
</P></CENTER><DIV class="subtitle" id="platecult"><H3>Culture on 48 wells plates</H3></DIV><DIV class="group center"><P align="justify" style="font-size:15px">
In order to determine the right concentration of polysaccharide and enzyme of 
BioSilta kit we have to do several cultures at the same time. So, we use an optical 
reader and 48 wells plates. 
</P></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Optical reader, OPTIMA MARS Analysis</LI><LI>48 wells plates
</LI><LI>Pre-culture of <I>E. coli</I> BW 25113</LI><LI>Different concentrations of BioSilta medium</LI><LI>For one concentration of BioSilta medium different concentrations of enzyme
</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Add 450 µL of medium in each well
</LI><LI>Add 50 µL of pre-culture</LI><LI>Place the 48 well plate in the optical reader
</LI><LI>Adjust parameters on computer. We tested culture between one day and ten days</LI></OL></DIV><DIV class="group center"><P align="justify" style="font-size:15px"><U>Note:</U> Each condition is tested in two replicates. 
According to our results we adapt concentrations of Biosilta medium and enzyme, 
results are exposed in device part.
</P></DIV><DIV class="subtitle" id="enzymekinetic"><H3>Enzyme kinetic</H3></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>Spectrophotometer
</LI><LI>BioSilta medium
</LI><LI>BioSilta enzyme solution named Reagent A (3000U/L)
</LI><LI>Bradford’s reagent</LI><LI>1.5 mL Eppendorf</LI><LI>Standard solutions of glucose</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>For each standard solutions : in a 1.5 mL Eppendorf sample 10 µL and add 1 mL Bradford’s Reagent, wait 20 minutes and measure OD<SUB>505 nm</SUB></LI><LI>Plot glucose concentration in function of OD<SUB>505nm</SUB> and determine the linear region
</LI><LI>Add 1.5 mL of BioSilta medium and 45 µL of reagent A (50 U/L) in an Eppendorf
</LI><LI>Sampling every 30 minutes: </LI><OL><LI>Take 10 µL and add 1 mL of Bradford’s reagent in an Eppendorf
</LI><LI>Wait 20 minutes
</LI><LI>Measure OD<SUB>505 nm</SUB></LI><LI>If OD<SUB>505 nm</SUB> is over linear region dilute sample and measure OD<SUB>505 nm</SUB> again
</LI></OL><LI>Stop sampling when glucose concentration no longer change</LI></OL></DIV><DIV class="subtitle" id="acidsprod"><H3>Acids production test</H3></DIV><DIV class="group center"><P align="justify" style="font-size:15px;"> In order to test if <I>E. coli</I> produces formic acid and butyric acid with genes added, we made culture test with modified bacteria. We used the same protocol as “Culture on Erlenmeyers and TubeSpin® Bioreactors” with some changes: </P></DIV><DIV class="group"><DIV class="one_half"><UL style="font-size:15px;" align="left"><LI>Volume of culture : 30 mL
</LI><LI>Add Ampicillin at 25 µg/mL to have selection pressure</LI><LI>The number of samples: 
</LI><UL style="font-size:15px;" align="left"><LI>Sample at the beginning</LI><LI>Sample at the end of the first day
</LI><LI>Sample after 24 hours culture and 48 hours culture</LI></UL></UL></DIV></DIV><DIV class="subtitle" id="gasconcentration"><H3>Test of gas concentration</H3></DIV><DIV class="group center"><P style="font-size:15px;" align="justify">
The objective of our device is to produce gas, so we would like to know the gas composition of our culture. 
So, we developed a system in order to recover acids gas.
</P></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>4 hours culture in 50 mL Falcon in M9 medium with 15 mM of glucose</LI><LI>Silicon plugs adapted to 50 mL Falcon</LI><LI>Needles
</LI><LI>0.2 µm filters</LI><LI>10 mL Syringes
</LI><LI>Neoprene pipes Ø=0.8 mm
</LI><LI>10 mM NaHCO<SUB>3</SUB></LI><LI>1.5 mL Eppendorf</LI><LI>1 mL Sterile cone</LI><LI>Incubator 3 °C without agitation</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Replace Falcon plug with silicon plug
</LI><LI>Adjust filter on needle and peg it into silicon plug. Do it twice</LI><LI>Adjust neoprene pipe into each filter</LI><LI>Add 700 µL of NaHCO<SUB>3</SUB> in an Eppendorf</LI><LI>At the end of first pipe put a sterile cone and immerse it into Eppendorf with NaHCO<SUB>3</SUB></LI><LI>At the end of second pipe put a 10 mL syringe</LI><LI>After 24 hours culture, press 10 mL syringe in order to expel gas in NaHCO<SUB>3</SUB> solution</LI><LI>Conserve samples at -20 °C before NMR analysis (see protocol foregoing) </LI><DIV class="group"><P style="font-size:15px;" align="justify"><U>Note 1:</U> We used culture in M9 because with the “Acids production tests” we had data on this medium.<U>Note 2:</U> 10 mM NaHCO<SUB>3</SUB> solution was used because pH was 8.3 so it would permit acid gas solubilisation.
</P></DIV><CENTER><P class="legend">
Photo 8: Gas concentration test with falcons
</P></CENTER><CENTER><DIV class="title" id="PCR"><H3>Protocol for Polymerase Chain Reaction (PCR), From Thermo Scientific™ DreamTaq™ Green PCR Master Mix </H3></DIV></CENTER><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>MilliQ water nuclease free
(QSP)</LI><LI>PCR Mix 2X</LI><LI>Forward primer
</LI><LI>Reverse primer</LI><LI>Template DNA
</LI><LI>Thin walled PCR tube</LI><LI>Ice</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Gently vortex and briefly centrifuge the PCR mix after thawing</LI><LI>Place a thin-walled PCR tube on ice and add the different components for a 50 μL PCR reaction</LI><LI>Gently vortex the samples</LI><LI>Perform PCR using the recommended thermal cycling conditions</LI><DIV class="group center"><P class="text">
The PCR Mix from Thermo Scientific contains Taq DNA polymerase, Green Buffer, MgCl2, dNTPs but also two tracking dyes and a density reagent that allows for direct loading of the PCR product on a migration gel.

The template DNA concentration has to be adapted in order to be between 10 pg and 1 μg in the final volume of 50 μL. The template DNA can come from a miniprep solution or from a single colony. The primer concentrations have to be between 0.1 μM and 1 μM. 

Each PCR reaction has to be adapted to the length of the PCR products, and to the melting temperature Tm of the primers. The extension step lasts 1 min for PCR products up to 2 kb. For longer products, the extension time has to be prolonged by 1 min/kb. 
</P></DIV><DIV class="group center"><TABLE class="df" style="font-size:15px;"><THEAD><TR><TH>Step</TH><TH>Temperature (°C)</TH><TH>Time</TH><TH>Number of cycles</TH></TR></THEAD><TBODY><TR><TD><P>Initial denaturation</P></TD><TD><P>95</P></TD><TD><P>1-3 min</P></TD><TD><P>1</P></TD></TR><TR><TD><P>Denaturation</P></TD><TD><P>95</P></TD><TD><P>30s</P></TD><TD rowspan="3"><P>25-40</P></TD></TR><TR><TD><P>Annealing</P></TD><TD><P>Tm – 5°C</P></TD><TD><P>30s</P></TD></TR><TR><TD><P>Extension</P></TD><TD><P>72</P></TD><TD><P>Adapt to the length</P></TD></TR><TR><TD><P>Final extension</P></TD><TD><P>72</P></TD><TD><P>5-15 min</P></TD><TD><P>1</P></TD></TR></TBODY></TABLE></DIV><CENTER><DIV class="title" id="TPX"><H3>Protocols for TPX® permeability tests</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#prepTPX">- Preparation of TPX® bag</A></LI><LI><A href="#permeabTPX">- Permeability test</A></LI><LI><A href="#sterilTPX1">- Sterility test of TPX® bag</A></LI></UL></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#cultTPX">- Culture test of <I>E. coli</I> in TPX® bag</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="prepTPX"><H3>Preparation of TPX® bag</H3></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>TPX®, gas permeable plastic</LI><LI>Fusing machine</LI><LI>2 mM Formic acid solution</LI><LI>4 % (V/V) Butyric acid solution</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Prepare plastic bag in sticking on 3 sides over 4 with fusing machine</LI><LI>Add 7 mL of appropriate solution in plastic bag</LI><LI>Stick on the last side with fusing machine</LI><DIV class="subtitle" id="permeabTPX"><H3>Permeability test</H3></DIV><DIV class="group center"><P style="font-size:15px;" align="justify">
To test gas permeability of TPX® plastic, we use the same protocol as “Test of gas concentration”. 
The only change is that no filters were used because the sterility is not necessary.
</P></DIV><CENTER><P class="legend">
Photo 9: The device used for the permeability test
</P></CENTER><DIV class="subtitle" id="sterilTPX2"><H3>Sterility test of TPX® bag</H3><H3 style="font-size:18px">    Demonstrate that the TPX® bag is impermeable to bacteria</H3></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>TPX bags</LI><LI>M9 defined Medium</LI><LI><I>E. coli</I> BW 25113</LI><LI>Steril laboratory glass bottle</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Overnight culture of <I>E. coli</I> BW 25113 at 37 °C</LI><LI>Inoculate a small TPX® bag at OD<SUB>600 nm</SUB> = 0.1 in LB medium (Final Volume = 8 mL)
</LI><LI>Negative Control: Fill a small TPX® bag with M9 medium (Final Volume=8 mL)
</LI><LI>Dispose each small bag in a Steril glass measuring cylinder containing M9 medium</LI><LI> Incubate at 37 °C</LI><LI>Measure OD<SUB>600 nm</SUB> twice a day</LI><DIV class="subtitle" id="cultTPX"><H3>Culture test of <I>E. coli</I> in TPX® bag</H3></DIV><DIV class="group"><DIV class="one_quarter first"><DIV class="subsubsubtitle"><H3>Materials</H3></DIV><UL align="justify" style="font-size:15px;"><LI>TPX bags</LI><LI>LB Medium</LI><LI>Steril clips</LI><LI><I>E. coli</I> BW 25113</LI><LI>Steril laboratory flask</LI><DIV class="three_quarter"><DIV class="subsubsubtitle"><H3>Methods</H3></DIV><OL align="justify" style="font-size:15px;"><LI>Overnight culture of E. coli BW 25113 at 37 °C</LI><LI>Inoculate a small TPX® bag at OD<SUB>600 nm</SUB> = 0.1 in LB medium (Final Volume = 8 mL)
</LI><LI>Close the small bag via fusing machine and Put the closed small bag in a Steril laboratory flask

</LI><LI>Positive Control: Inoculate a culture tube at OD<SUB>600 nm</SUB> = 0.1 in LB medium (Final Volume = 20 mL)
</LI><LI> Incubate at 37 °C</LI><LI>Measure OD<SUB>600 nm</SUB> twice a day</LI><CENTER><DIV class="title" id="transfo"><H3>Transformation Protocol: RbCl method</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#mediaAndSol">- Media and solution</A></LI><LI><A href="#competentCell">- Preparation of Competent Cells</A></LI><LI><A href="#transfoCell">- Transformation of Competent Cells </A></LI><LI><A href="#minipreps">- Minipreps</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="mediaAndSol"><H3> Media and solution</H3></DIV><DIV class="group center"><TABLE class="df"><THEAD><TR><TH>YETM 500 mL</TH><TH>TFB1 200 mL</TH><TH>TFB2 200 mL</TH></TR></THEAD><TBODY><TR><TD><UL style="font-size:15px;" align="left"><LI> 2.5 g Yeast Extract </LI><LI> 10 g Tryptone </LI><LI> 5 g MgSO<SUB>4</SUB>.7H<SUB>2</SUB>O </LI><LI> Adjust pH to 7.5 with KOH </LI><LI><B>For Plates</B>: add 7.5 g of Agar </LI></UL></TD><TD><UL style="font-size:15px;" align="left"><LI>  0.59 g KOAc </LI><LI> 2.42 g RbCl </LI><LI> 0.29 g CaCl<SUB>2</SUB>.2H<SUB>2</SUB>O </LI><LI> 1.98 g MnCl<SUB>2</SUB>.4H<SUB>2</SUB>O </LI><LI> Adjust to pH 5.8 with 0.2 M acetic acid </LI><LI> Add dH<SUB>2</SUB>O to 200 mL </LI><LI> Filter sterilize </LI><LI> Store refrigerated at 4°C </LI></UL></TD><TD><UL style="font-size:15px;" align="left"><LI> 0.42 g MOPS </LI><LI> 2.21 g CaCl<SUB>2</SUB>.2H<SUB>2</SUB>0 </LI><LI> 0.24 g RbCl </LI><LI> 30 g Glycerol </LI><LI> Adjust to pH 6.5 with KOH </LI><LI> Add dH<SUB>2</SUB>O to 200 mL </LI><LI> Filter sterilize </LI><LI> Store refrigerated at 4 °C </LI></UL></TD></TR></TBODY></TABLE></DIV><DIV class="subtitle" id="competentCell"><H3> Preparation of Competent Cells </H3></DIV><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>1. Streak cells from frozen stock onto YETM plate. Incubate overnight at 37°C </LI><LI>2. Pick a single fresh colony to inoculate 5 mL of YETM medium. Grow over night at 37°C.</LI><LI><B>Do not vortex cells at any time after this point in the procedure</B></LI><LI>3. Dilute 1 mL of culture into 50 mL YETM medium prewarmed to 37 °C</LI><UL align="justify" style="font-size:15px;"><LI> Grow at 37 °C for 2 hours with agitation </LI><LI> Volumes can be scaled up 5X and all of the 5 mL overnight culture can be used </LI></UL><LI> 4. Transfer culture to sterile 50 mL tube. Chill on ice/water 10-15 minutes </LI><LI> 5. Centrifuge for 10 minutes at 2,000 rpm at 4 °C. Immediately aspirate off all of the supernatant </LI><LI><B> Do not allow cells to warm above 4 °C at any time in this procedure </B></LI><LI> 6. Resuspend cells in 10 mL of ice-cold TFB1 with gentle re-pipetting. Use chilled glass or plastic pipette </LI><LI> 7. Incubate cells on ice for 5 minutes </LI><LI> 8. Repeat step 5</LI><LI> 9. Resuspend cells in 2 mL of ice-cold TFB2 with <B>gentle</B> re-pipetting. Use micropipet tip (plastic)</LI><LI> 10. Incubate cells on ice for 15 minutes </LI><LI> Cells may be used for transformation or frozen </LI><UL align="justify" style="font-size:15px;"><LI> To freeze: aliquot cell in 200 μL volumes into prechilled 0.5 mL microfuge tube (on ice) </LI><LI> Freeze immediately in liquid nitrogen </LI><LI> Store cells frozen at -80 °C </LI></UL></UL></DIV><DIV class="subtitle" id="transfoCell"><H3> Transformation of Competent Cells </H3></DIV><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>1. If starting with frozen competent cells, warm tube/cells by gently twirling between your fingers until just thawed. Immediately place on ice for about 5 minutes </LI><LI>2. Add to 1,5 mL eppendorf on ice: 2-3 μL iGEM plate or 1 μL plasmid or 10 μL ligation.</LI><LI>3. Add 100 μL of competent cells and mix by gentle re-pipetting</LI><LI> 4. Incubate cells on ice for 20-30 minutes </LI><LI> 5. Heat shock the cells exactly 90 seconds at 42 °C </LI><LI> 6. Return cells on ice for 2 minutes </LI><LI> 7. Add 1 mL of YETM medium. Incubate at 37 °C for 45-60 minutes with slow gentle shaking </LI><LI> 8. Plate 0.1-0.2 mL of transformed cells on LB-plate containing the appropriate antibiotic. Incubate  overnight at 37°C</LI></UL></DIV><DIV class="subtitle" id="minipreps"><H3> Minipreps </H3></DIV><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>1. Resuspend 4 to 12 colonies from the plate and name each colony taken on the tubes and on the plate (A, B, C, …) </LI><LI>2. Resuspend one colony per culture tube in 5 mL of LB medium with antibiotic</LI><LI>3. Let the culture grow overnight at 37 °C in a shaking incubator</LI><LI> 4. Use the QIAprep spin Miniprep Kit for each culture tube. The last step consisting in the elution of the DNA is made with elution buffer or water at 55 °C </LI><LI> 5. Keep the tubes at -20 °C</LI><DIV class="group center"><CENTER><DIV class="title" id="cloning"><H3>Cloning</H3></DIV></CENTER></DIV><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#sameAntibio">- Digestion: both parts have the same antibiotic resistance</A></LI><LI><A href="#diffAntibio">- Digestion: The two parts have a different antibiotic resistance</A></LI></UL></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#migration">- Migration and gel extraction</A></LI><LI><A href="#ligation">- Ligation</A></LI><LI><A href="#cloningtransfo">- Transformation</A></LI></UL></DIV></CENTER><DIV class="subtitle"><H3>First step: Digestion</H3></DIV><DIV class="subsubsubtitle" id="sameAntibio"><H3>Both parts have the same antibiotic resistance</H3></DIV><DIV class="group center"><TABLE class="df" style="font-size:15px;"><THEAD><TR><TH>Vector</TH><TH>Insert</TH><TH>Digestion control first enzyme</TH><TH>Digestion control second enzyme</TH></TR></THEAD><TBODY><TR><TD><P>Volume equivalent to 1 µg of vector miniprep </P></TD><TD><P>Volume equivalent to 1 µg of insert miniprep </P></TD><TD><P>Volume equivalent to 1 µg of vector miniprep </P></TD><TD><P>Volume equivalent to 1 µg of vector miniprep </P></TD></TR><TR><TD><P>1 µL of each restriction enzymes</P></TD><TD><P>1 µL of each restriction enzymes</P></TD><TD><P>1 µL of the first restriction enzyme</P></TD><TD><P>1 µL of the second restriction enzyme</P></TD></TR><TR><TD><P>2 µL of Fast Digest Green Buffer (Thermo Scientific™)</P></TD><TD><P>2 µL of Fast Digest Green Buffer (Thermo Scientific™)</P></TD><TD><P>2 µL of Fast Digest Green Buffer (Thermo Scientific™)</P></TD><TD><P>2 µL of Fast Digest Green Buffer (Thermo Scientific™)</P></TD></TR><TR><TD><P>Up to 20 µL of Milli-Q water</P></TD><TD><P>Up to 20 µL of Milli-Q water</P></TD><TD><P>Up to 20 µL of Milli-Q water</P></TD><TD><P>Up to 20 µL of Milli-Q water</P></TD></TR><TR><TD colspan="4"><P>Incubate 15 minutes at 37 °C</P></TD></TR></TBODY></TABLE></DIV><DIV class="subsubsubtitle" id="diffAntibio"><H3>The two parts have a different antibiotic resistance</H3></DIV><DIV class="group center"><TABLE class="df" style="font-size:15px;"><THEAD><TR><TH>Both parts</TH></TR></THEAD><TBODY><TR><TD><P>Volume equivalent to 1 µg of DNA miniprep </P></TD></TR><TR><TD><P>1 µL of each restriction enzymes</P></TD></TR><TR><TD><P>2 µL of Fast Digest Green Buffer (Thermo Scientific™)</P></TD></TR><TR><TD><P>Up to 20 µL of Milli-Q water</P></TD></TR><TR><TD><P>Incubate 15 minutes at 37°C</P></TD></TR></TBODY></TABLE></DIV><DIV class="subsubsubtitle" id="migration"><H3>Migration and gel extraction</H3></DIV><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>1. Prepare a 1 % or 2 % electrophoresis agarose gel with 0.5 X TAE buffer </LI><LI>2. Put 20 µL of sample + 6 µL of marker (1 kb for 1 % gel and 100 pb for 2 %) into the well</LI><LI>3. Migration for 30 min at 100 V or 1 hour at 50 V</LI><LI> 4. Bath 10 minutes in BET</LI><LI> 5. Wash in water for 5 minutes </LI><LI> 6. The gel extraction is realized thanks to the QIAGEN Gel Extraction Kit</LI><LI><U> Two ways to inactivate the enzymes for the vector</U></LI><UL align="justify" style="font-size:15px;"><LI> Use of DNA Clean up kit for the DNA fragment above 200 pb</LI><LI> Heat inactivation at 95 °C for 10 minutes </LI></UL></UL></DIV><DIV class="subtitle" id="ligation"><H3>Second step: Ligation</H3></DIV><DIV class="group center"><TABLE class="df" style="font-size:15px;"><THEAD><TR><TH>Mix</TH><TH>Negative Control</TH><TH>Positive Control</TH></TR></THEAD><TBODY><TR><TD><P>Volume equivalent to 3 molecules of insert (for one molecule of vector)</P></TD><TD><P>no insert</P></TD><TD><P>Volume equivalent to 3 molecules of insert (for one molecule of vector)</P></TD></TR><TR><TD><P>Volume equivalent to 50 ng of digested vector</P></TD><TD><P>Volume equivalent to 50 ng of digested vector</P></TD><TD><P>Volume equivalent to 50 ng of undigested vector</P></TD></TR><TR><TD><P>2 µL of 10X T4 buffer</P></TD><TD><P>2 µL of 10X T4 buffer</P></TD><TD><P>2 µL of 10X T4 buffer</P></TD></TR><TR><TD><P>0.5 µL of T4 ligase</P></TD><TD><P>0.5 µL of T4 ligase</P></TD><TD><P>0.5 µL of T4 ligase</P></TD></TR><TR><TD><P>Up to 20 µL of Milli-Q water</P></TD><TD><P>Up to 20 µL of Milli-Q water</P></TD><TD><P>Up to 20 µL of Milli-Q water</P></TD></TR><TR><TD colspan="3"><P> Incubate the ligation mix 15 minutes at room temperature (22°C)</P></TD></TR><TR><TD colspan="3"><P> Keep the tubes in ice or at -20 °C to prepare the transformation</P></TD></TR></TBODY></TABLE></DIV><DIV class="subtitle" id="cloningtransfo"><H3>Third step: Transformation</H3></DIV><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>1a. Take 10 µL of the ligation mix for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol </LI><LI>1b. Positive control: take 10 µL of the ligation mix for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol </LI><LI>1c. First restriction enzyme digestion control: take 10 µL of the corresponding digestion mix (First step) for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol </LI><LI>1d. Second restriction enzyme digestion control: take 10 µL of the corresponding digestion mix (First step) for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol </LI><LI>1e. Negative control: take 10 µL of the corresponding mix for 100 µL of competent cells and use the Toulouse iGEM Team 2015 transformation protocol </LI><LI>2. Plate the solution on selective medium overnight at 37 °C</LI></UL></DIV><DIV class="group center"><CENTER><DIV class="title" id="infusion"><H3>InFusion cloning protocol</H3></DIV></CENTER></DIV><CENTER><P class="">
		Principle of In Fusion cloning
		</P></CENTER><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>1. Design the tailed-oligonucleotides for the vector and the inserts. The tail of the 5’oligonucleotide must be the 20 last nucleotides of the previous fragment and the tail of the 3’oligonucleotide must be the 20 first nucleotides of the next fragment. </LI><LI>2. Amplify the different fragments with the <A href="#PCR">previously designed oligonucleotides</A></LI><LI>3. Clean the PCR products either using a spin column purification kit or by digesting with DpnI. For NEB DpnI, mix the different PCR products together, add 10X CutSmart Buffer and DpnI (1 µL in a 20 µL mix); incubate at 37°C for 20 minutes and inactive at 80 °C during 30 minutes.</LI><LI>4.	Set up the In Fusion cloning reaction :</LI><CENTER><TABLE class="result" style="padding-left:50px;"><TBODY><TR><TH>Ligation</TH><TH>Control</TH></TR><TR><TD>5X In-Fusion HD Enzyme Premix</TD><TD>2 µL</TD><TD>-</TD></TR><TR><TD>Clean PCR mix</TD><TD>4 µL</TD><TD>4 µL</TD></TR><TR><TD>MilliQ water nuclease free</TD><TD>4 µL</TD><TD>6 µL</TD></TR><TR><TD>Total</TD><TD>10 µL</TD><TD>10 µL</TD></TR></TBODY></TABLE></CENTER><LI>5.	Incubate at 50 °C during 15 min and then cool on ice</LI><LI>6.	Transform commercial ultra-competent cells (10<SUP>8</SUP> to 10<SUP>9</SUP> cfu/µg DNA) with 2.5 µL of the ligation using provided with the competent cells. Plate several 10-fold dilutions of the transformation mix.</LI></UL></DIV></UL></DIV></OL></DIV></UL></DIV></DIV></OL></DIV></UL></DIV></DIV></OL></DIV></UL></DIV></DIV></OL></DIV></UL></DIV></DIV></OL></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></MAIN></DIV><DIV class="wrapper row5"><FOOTER id="footer" class="clear"><DIV class="group"><DIV class="one_third first"><H6 class="title">iGEM Toulouse 2015</H6><ADDRESS class="btmspace-15">
        INSA Toulouse
        135, Avenue de Rangueil
        31400 TOULOUSE
		France</ADDRESS></DIV><DIV class="one_third"><H6 class="title">Follow us</H6></DIV><DIV class="one_third"><H6 class="title">For more informations</H6><A href="mailto:igemtoulouse2015@gmail.com" style="border:1px solid #fff;-webkit-border-radius: 5px; 
		-moz-border-radius: 10px; border-radius: 5px; padding: 7px 16px 5px; color:white; 
		text-decoration:none; font-size:17px; background:none;  display:inline-block;">Contact</A></DIV></DIV><CENTER><P>Website by <A href="https://2015.igem.org/Team:Toulouse">Team iGEM Toulouse 2015</A>&amp;<A href="http://www.os-templates.com/" title="Free Website Templates">OS Templates</A></P></CENTER></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>