Team:Toulouse/Description/Attract
iGEM Toulouse 2015
Home
Context
Achievements
Team
Presentation
Gallery
Fun Facts
Project
Project: Description
Design: Attract
Design: Eradicate
Design: Regulate
Device: TrapiColi
Results
Attract
Eradicate
Regulate
Containment
Growth tests
Cytoxicity
Parts
New BioBricks
Module: Attract
Module: Eradicate
Module: Regulate
Modeling
Metabolic networks
FBA
Diffusion
Optimisation
Human Practice
Meetups & Collaborations
Ethics & IP
Education
Safety
Wet lab
Notebook
Protocols
Attributions
Attributions
Supports
Attract
Content
How to attract Varroa destructor?
Butyrate attraction test
How to produce butyrate with E.Coli?
How to attract Varroa destructor?
Just before capping, bee larvaes produce a wide range of molecules, those molecules warn the mite about the upcoming capping and motivate  it to enter the cell [1].   Of all these molecules, scientific studies have shown that one can  significantly attract varroa: butyrate [2].
Butyrate is a volatile acid which is non-toxic for honeybees     nor for the human being, because it is already present at physiologic     concentrations in their digestive tract. Moreover this molecule     is naturally     produced by some bacterial strains,     which is an asset     for identifying key enzymatic activities for this synthetic biology project [3].
Therefore, based on the patent US 8647615 (Figure 1), we decided to     modify E. coli so it will synthesize    butyrate in order to attract varroa [4].
Figure 1: Results of butyrate attraction     test with quadrants method, US 8647615 B1 [4].
Butyrate attraction test
Figure 2: Butyrate attraction test using     T tube, with varroa mite in the middle
To confirm this previous finding and make sure we can handle such experiment, a new protocol using a custom-made glass T-tube has been developed (Figure 2).  In the first branch, there is a cotton soaked with 50 µL of water,  in the second a cotton with  50 µL of butyrate at 4%, and finally the  last one contains the varroa.
Butyrate being very volatile, our  system  used a pump to renew air, producing a concentration gradient as seen here .
How to produce butyrate with E.coli?
In this project, an Escherichia coli strain is used for its known   simplicity of genetic manipulation and its adequacy with butyrate   synthesis [5]. Indeed, among the five enzymes we selected to produce butyrate from acetyl-CoA,   two enzymes are naturally produced by the bacteria. The following   engineered butyrate pathway has been designed:
Figure 3: Engineered butyrate pathway
The initial substrate is glucose which is decomposed into   acetyl-CoA during glycolysis. Butyrate pathway   begins with acetyl-CoA: five genes are required with two  homologous and three heterologous genes.
atoB present in E.coli (Accession Number: EG11672 ), coding for acetyl-CoA    acetyltransferase, an acetyltransferase catalyzing the combination    of two acetyl-CoA.
Figure 4: Reaction catalyzed by acetyl-CoA   acetyltransferase
hbd present in Clostridium acetobutylicum (Accession Number: GJIH-2684 ) coding for    3-hydroxybutyryl-CoA dehydrogenase, an oxidoreductase catalyzing    the formation of an alcohol function.
Figure 5: Reaction catalyzed by   3-hydroxybutyryl-CoA dehydrogenase
crt present in C.acetobutylicum (Accession Number: GJIH-2688 )    coding for 3-hydroxybutyryl-CoA dehydratase,   a lyase cleaving carbon-oxygen bond.
Figure 6:  Reaction catalyzed by 3-hydroxybutyryl-CoA deshydratase
ccr present in Streptomyces collinus (Accession Number: Q53865-1 ) coding   for crotonyl-CoA reductase,   an oxidoreductase acting on   CH=CH double bond and successfully used in E. coli [6].
Figure 7: Reaction   catalyzed by crotonyl-CoA reductase
tesB present in E.coli (Accession Number: EG10995 )   coding for acyl-CoA transferase 2,   a thiolase which enables coenzyme A transfer.
Figure 8: Reaction   catalyzed by acyl-CoA transferase 2
Concerning heterologous genes (hbd, crt and ccr), codon optimizations have been performed in order to enable a correct expression of these genes in E. coli.  The genetic construction is then done by assembling the five genes presented earlier, which are placed under the control of the P(Bla) constitutive promoter (BBa_I14018). Ribosome binding sites (RBS ; BBa_B0030) are added before the coding sequences to ensure protein expression. A strong terminator (BBa_B1006) is used to end this construction. The final biobrick was cloned into a pSB1C3 vector ( here ).
READ MORE
Project Description
Eradicate
Regulation
Device: TrApiColi
Results
References
[1] Le Conte Y, Arnold G, Trouiller J, Masson C, Chappe B & Ourisson G (1989) Attraction of the parasitic mite varroa to the drone larvae of honey bees by simple aliphatic esters. Science 245: 638–639
[2] Peter EA Teal, Adrian J. Duehl, Mark J. Carroll. The United States Of America, As Represented By The Secretary Of Agriculture. 2014. Methods for attracting honey bee parasitic mites, US 8647615 B1.
[3] Louis P & Flint HJ (2009) Diversity, metabolism and microbial ecology of butyrate-producing bacteria from the human large intestine. FEMS Microbiol. Lett. 294: 1–8
[4] Peter EA Teal, Adrian J. Duehl, Mark J. Carroll. The United States Of America, As Represented By The Secretary Of Agriculture. 2014. Methods for attracting honey bee parasitic mites, US 8647615 B1.
[5] Atsumi S, Cann AF, Connor MR, Shen CR, Smith KM, Brynildsen MP, Chou KJY, Hanai T & Liao JC (2008) Metabolic engineering of Escherichia coli for 1-butanol production. Metabolic Engineering 10: 305–311
[6] Wallace KK, Bao ZY, Dai H, Digate R, Schuler G, Speedie MK & Reynolds KA (1995) Purification of crotonyl-CoA reductase from Streptomyces collinus and cloning, sequencing and expression of the corresponding gene in Escherichia coli. Eur. J. Biochem. 233: 954–962
iGEM Toulouse 2015
INSA Toulouse
135, Avenue de Rangueil
31400 TOULOUSE
France
Follow us
For more informations
Contact
Website by Team iGEM Toulouse 2015 & OS Templates
