<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Toulouse_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Toulouse/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Toulouse 2015</TITLE></P><DIV class="wrapper row0 fixed" style="max-height:87px;margin-top:-10px"><HEADER id="header" class="clear"><NAV id="mainav" class="center"><UL class="clear" style="padding-top:15px;"><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse"><B>Home</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse#context">Context</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse#achievements">Achievements</A></LI></UL><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Team" id="team"><B>Team</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Team">Presentation</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/FunFacts">Fun Facts</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description" id="project"><B>Project</B></A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description">Project: Description</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Attract">Design: Attract</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Eradicate">Design: Eradicate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Description/Regulation">Design: Regulate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Design">Device: TrapiColi</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Results" id="Results"><B>Results</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#attract">Attract</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#eradicate">Eradicate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#regulate">Regulate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#TPX">Containment</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#biolo">Growth tests</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Results#tox">Cytoxicity</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Parts" id="parts"><B>Parts</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#biobricks">New BioBricks</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#1">Module: Attract</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#2">Module: Eradicate</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Parts#3">Module: Regulate</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Modeling" id="modeling"><B>Modeling</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part1">Metabolic networks</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part2">FBA</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part5">Diffusion</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Modeling#part6">Optimisation</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Practices" id="practices"><B>Human Practice</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Collaborations">Meetups &amp; Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Ethics">Ethics &amp; IP</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Practices#Safety">Safety</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Notebook" id="notebook"><B>Wet lab</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Experiments">Protocols</A></LI><LI><A class="drop" href="https://2015.igem.org/Team:Toulouse/Attributions" id="attributions"><B>Attributions</B></A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Toulouse/Supports">Supports</A></LI></NAV></HEADER></DIV><DIV class="wrapper row0 bgded" style="background-image:url('https://static.igem.org/mediawiki/2015/f/f5/TLSE_bg_1.png')"><DIV class="shout"><DIV class="shout-content clear"><DIV class="maintitle"><CENTER><H3>Results</H3></CENTER></DIV></DIV></DIV></DIV><DIV class="wrapper row3"><MAIN class="container clear"><DIV class="title"><H3>Content</H3></DIV><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#attract">○ Attract</A></LI><LI><A href="#eradicate">○ Eradicate</A></LI><LI><A href="#regulate">○ Regulate</A></LI><LI><A href="#TPX">○ ApiColi containment and culture</A></LI></UL></DIV><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#biolo">○ Growth tests</A></LI><LI><A href="#carbon">○ Choice of carbon source</A></LI><LI><A href="#tox">○ Testing acids toxicity</A></LI></UL></DIV></CENTER><CENTER><DIV class="title" id="attract"><H3>Attract</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#varrotest">- Test on varroas</A></LI><LI><A href="#cloningbutyrate">- Cloning butyrate genes</A></LI><LI><A href="#butyproduct">- Test of butyrate production</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="varrotest"><H3>Tests on varroas</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to prove again varroa attraction for butyrate</B></FONT></DIV><DIV class="group center"><P class="text">
  In the US patent <A target="_blank" href="http://www.google.com.ar/patents/US8647615">US 8647615 B1</A>, the concentration of butyric acid that attracts varroa mites is 4 % (V/V), but the final description specified the concentration is efficient from 0.00001 %. 
   </P></DIV><DIV class="group center"><P class="text">
  In order to verify the results presented in this patent and to make sure we are able to perform such experiment, we designed an attraction test for varroas. Champollion University in Albi welcomed us in their lab to get access to varroas. At the time of this test, there were not a lot of varroas available, so we add to make only one test in order to hope a significant result.
 Hence, we used the mentioned 4 % butyric acid concentration, like the one first mentioned in the patent.
   </P></DIV><CENTER><P class="legend">
Figure 1: Butyric acid test pie chart and statistical test
</P></CENTER><DIV class="group center"><P class="text">
  This test demonstrated that a solution of 4 % butyric acid attracts varroas. However, in the <A href="https://2015.igem.org/Team:Toulouse/Results#tox">Cytotoxicity</A>
  part of the results, we also showed that this 4% concentration is lethal for bacteria. We therefore decided to aim for a minimal concentration of 0.00001 % butyric acid, even if further experimentation will have to demonstrate the efficiency of this lower threeshold. 
   </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: Our own protocol to test varroa attraction for butyrate is functionnal</B></FONT></DIV><DIV class="subtitle" id="cloningbutyrate"><H3>Cloning butyrate genes</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to create the genetic tool to implement a butyrate pathway in <I>E. coli</I></B></FONT></DIV><DIV class="group"><P class="text">
  We designed and ordered synthetic genes to express the full pathway minus the <I>ccr</I> gene. The latter was already 
  placed behind <I>lacI</I> to be ready for circadian circle regulation. Thus, to build a biobrick with the full pathway, we cloned the <I>ccr</I> gene with all the genes necessary for butyrate production, as presented below: 
   </P></DIV><CENTER><P class="legend">
Figure 2: assembly of the synthetic pathway leading to butyrate production (5192 Kb). 
The first arrow represents the promoter, the others represent genes. The green circles are for the RBS and the red circle is for the terminator. Purple genes are originated from <I>Streptomyces collinus</I>, blue genes from <I>Clostridium acetobutylicum</I> and yellow genes from <I>Escherichia coli</I>.
</P></CENTER><DIV class="group"><P class="text">
  This construction <A target="_blank" href="http://parts.igem.org/Part:BBa_K1587004">(BBa_K1587004)</A> was inserted in the plasmid pSB1C3 to 
  be provided to the iGEM in the biobrick format. 
   </P></DIV><CENTER><P class="legend">
Figure 3: Gel electrophoresis of the digestion of the <A target="_blank" href="http://parts.igem.org/Part:BBa_K1587004">(BBa_K1587004)</A>. The size
 of the first DNA fragment matches  5192  pb (butyrate construction) and the size of the second one (2070 pb) matches linearized pSB1C3.
</P></CENTER><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: The synthetic butyrate polycistron is ready to be used in <I>E. coli</I></B></FONT></DIV><DIV class="subtitle" id="butyproduct"><H3>Test of butyrate production</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to produce butyrate in <I>E. coli</I></B></FONT></DIV><DIV class="group"><P class="text">
In order to test butyrate production, we cultivated ApiColi under micro-aerobic conditions to simulate the growth conditions in the device. After having harvested the supernatant of the culture, we filtrated it prior to do NMR analysis.
   </P></DIV><DIV class="group"><P class="text">
  We tested our genetic construction in the <I>E. coli</I> strain BW25113 but we did not detect butyrate production. To improve the production of butyrate, we repeated the experiment in a strain with a deletion of the phosphate acetyltransferase gene. This strain produces lower amount of acetate, a fermentation product which production is likely to compete with butyrate.
</P></DIV><CENTER><P class="legend">
Figure 4: Test of butyrate production in <I>E. coli</I> strain deleted for <I>pta</I> gene. Culture under micro-aerobic conditions in 10 mL falcon, result obtained after 28.5 hours of culture.
</P></CENTER><DIV class="group"><P class="text">
Unfortunately we could not detect butyric acid in our NMR analysis, but we observed significant differences for other fermentation products, proving that the synthetic pathway is actually interfering with the fermentation products metabolism. Additional experiments will be needed to go further (test of our enzymes expression, measurements of intracellular pools to identify metabolic locks, growth at other pH, deletions to prevent the accumulation of other fermentation products as explained in [1]...).
 </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: butyrate production has to be improved in our strain and under our conditions</B></FONT></DIV><DIV class="group center"><P class="highlight" style="padding:10px;"><B>In a nutshell:</B> Butyrate is an effective attractant for varroas. Its synthetic pathway has been implemented in <I>E. coli</I>. Butyrate production from this pathway remains to be demonstrated prior to testing the strain with varroas.
</P></DIV><CENTER><DIV class="title" id="eradicate"><H3>Eradicate</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#varrotest2">- Test on varroas</A></LI><LI><A href="#cloningformate">- Cloning formate genes</A></LI><LI><A href="#formaproduct">- Test of formate production</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="varrotest2"><H3>Tests on varroas</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: determinate the sufficient formate concentration to kill <I>Varroa destructor</I></B></FONT></DIV><DIV class="group"><P class="text">
In order to determinate the amount of formic acid required to kill the mite, we tested different concentrations of formic acid on varroas as explained in the <A target="_blank" href="https://2015.igem.org/Team:Toulouse/Experiments#mortalV">Protocol</A> part.
</P></DIV><CENTER><P class="legend">
Figure 5: Mortality of varroas as a function of time for different formic acid concentrations
</P></CENTER><CENTER><P class="legend">
Figure 6: Histogram representing mortality of varroas after 2 hours and after 7 hours
</P></CENTER><DIV class="group"><P class="text">
Figure 7 presents a dose-dependency of formic acid on varroa mortality. At 10mM formic acid, all varroas 
died before three hours but as we explained in the <A targer="_blank" href="https://2015.igem.org/Team:Toulouse/Experiments#mortalV">Protocol</A> part, varroas also stop moving at lesser concentrations. Figure 6 shows that even with 50 µM of formate, around 30 % varroas died after 7 hours (so close to a night duration). We therefore set our production goal around 50 µmol.L<SUP>-1</SUP>.
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: 50 µmol.L<SUP>-1</SUP> could be sufficient to kill varroas</B></FONT></DIV><DIV class="subtitle" id="cloningformate"><H3>Cloning formate genes</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to create the genetic tool to implement a formate pathway in <I>E. coli</I></B></FONT></DIV><DIV class="group"><P class="text">
  We designed and ordered synthetic genes to express the full pathway. Thus, we only had to clone the synthesized genes into pSB1C3 to be sent to the iGEM registry. 
   </P></DIV><CENTER><P class="legend">
Figure 8: assembly of the synthetic pathway leading to formate production (3090 Kb). 
The first arrow represents the promoter, the others represent genes. The green circles are for the RBS and the red circle is for the terminator. 
</P></CENTER><DIV class="group"><P class="text">
  This construction <A target="_blank" href="http://parts.igem.org/Part:BBa_K1587007">(BBa_K1587007)</A> was inserted in the plasmid pSB1C3 to be provided to the iGEM in the biobrick format. 
   </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: The synthetic formate operon is ready to be used in <I>E. coli</I></B> and sent to the registry
 </FONT></DIV><DIV class="subtitle" id="formaproduct"><H3>Test of formate production</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to increase the production of formate in <I>E. coli</I></B></FONT></DIV><DIV class="group"><P class="text">
For formate production, we designed a synthetic polycistron coding for pyruvate formate lyase and its activator protein. 
The polycistron was cloned in a pUC57 plasmid to be tested for formate production. 
We then made a biobrick by subcloning the polycistron in the pSB1C3 plasmid (<A target="_blank" href="http://parts.igem.org/Part:BBa_K1587007">BBa_K1587007</A>).
</P></DIV><CENTER><P class="legend">
Figure 9: extracellular substrate and product concentrations with or without formate polycistron overexpression under micro-aerobic conditions.
</P></CENTER><DIV class="group"><P class="text">
Figure 9 shows that the only difference between ApiColi and the control is in the formate accumulation, 
so we plotted the specific histogram for formate after 3 days (Figure 10). This highlights that formate production is increased significantly by 10%.
</P></DIV><CENTER><P class="legend">
Figure 10: Formate production tests after 3 days cultivation under micro-aerobic conditions.
</P></CENTER><DIV class="group"><P class="text">
As mentioned earlier, our goal was to produce 50µM of formic acid in 7 hours. 
To reach this quantity, we needed to produce 77mM of formate. We observed a maximal formate concentration of 25mM (Figure 9).
Therefore our production level is not far from the target. By improving the strain and/or conditions (as for <A href="https://2015.igem.org/Team:Toulouse/Results#butyproduct">Test of butyrate production</A>), we can reasonably hope to reach this goal. 
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: formate production was increased with our synthetic construction</B></FONT></DIV><DIV class="group center"><P class="highlight" style="padding:10px;"><B>In a nutshell:</B> formate is effective to kill varroas from as low as 50 µmol.L<SUP>-1</SUP>. Overexpressing its biosynthetic genes enables obtention of a 10 % increase in production. Optimizing this production will have to be achieved before testing the strain capacity to kill varroas.
</P></DIV><CENTER><DIV class="title" id="regulate"><H3>Regulate</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#logic">- NOT logic gate</A></LI><LI><A href="#sensor">- Light sensor</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="logic"><H3>NOT logic gate</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: NOT logic gate construction</B></FONT></DIV><DIV class="group center"><P class="text">
We want to express the butyrate synthetic pathway by day and the formate synthetic pathway by night. As a first step to develop our regulation module, we replaced the butyrate pathway by the RFP reporter gene and the formate pathway by GFP reporter gene. Doing so, it should be easy to assess the functioning of the light sensor and the NOT logic gate since the colonies should be red by day and green by night.
To do so, we synthesized the pOMPC-cI gene (new biobrick <A href="http://parts.igem.org/Part:BBa_K1587006">BBa_K1587006</A>)Through subcloning the GFP reporter gene (<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>) was placed under control of this promoter. Likewise, the RFP reporter gene (<A href="http://parts.igem.org/Part:BBa_K081014">BBa_K081014</A>) was cloned behind the PL-lacI synthetic gene. Both constructions were placed together on the same plasmid but unfortunately sequencing revealed that there was some incoherence in the RFP sequence, preventing further testing.
</P></DIV><DIV class="group center"><P class="legend">
   Figure 11: NOT logic gate test construction
  </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: the strain can grow and survive over 10 days in the TPX® bag.</B></FONT></DIV><DIV class="subtitle" id="sensor"><H3>Light sensor</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: Light sensor system construction.</B></FONT></DIV><DIV class="group center"><P class="text">
For our project we need to implement a light sensor system in our strain. To that end, we needed to clone the three genes (<I>cph8, hoI, pcyA</I>) of this system behind a strong promoter. We received the strain containing the three genes on different plasmids as courtesy from Dr Clark Lagarias, Christopher Voigt, and Nico Claassens [2]. These genes were amplified by PCR and subcloned in pSB1C3 (at this step, a RBS was added to sequence during the PCR). The new biobricks for <I>hoI</I> and <I>pcYA</I> were sent to the registry under the accession number <A href="http://parts.igem.org/Part:BBa_K1587000">BBa_K1587000</A> and <A href="http://parts.igem.org/Part:BBa_K1587002">BBa_K1587002</A>, respectively. The <I>cph8</I> biobrick was not obtained but we managed to build a biobrick (<A href="http://parts.igem.org/Part:BBa_K1587008">BBa_K1587008</A>) with the <I>cph8</I> encoding sequence under the control of a strong promoter originated from the biobrick (<A href="http://parts.igem.org/Part:BBa_J23119">BBa_J23119</A>).
</P></DIV><DIV class="group center"><P class="legend">
   Figure 12: Targeted light sensor construction
  </P></DIV><DIV class="group center"><P class="legend">
   Figure 13: <I>:cp</I> biobrick
  </P></DIV><DIV class="group center"><P class="text">
 We did not succeed in associating the three genes together behind a strong promoter in pSB1C3 using the In Fusion cloning technique before the iGEM deadline. Consequently, we were not able to test our light sensor system soon enough for this project.
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: the light sensor system is not yet achieved.</B></FONT></DIV><DIV class="group center"><P class="highlight"><B>In a nutshell:</B> About 70% of the clonings for the light sensor and the NOT logic gate has been done, but the last cloning steps remain to be finished before the tests.
</P></DIV><CENTER><DIV class="title" id="TPX"><H3>ApiColi containment and culture</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#growth">- Growth tests in TPX bag</A></LI><LI><A href="#gaz">- Gaz diffusion tests</A></LI><LI><A href="#safety">- Safety tests</A></LI></UL></DIV></CENTER><DIV class="subtitle" id="growth"><H3>Growth tests in TPX bag</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to perform the growth of the modified strain in a sealed container</B></FONT></DIV><DIV class="group center"><P class="text">
  We investigated if the bacteria could grow inside a small bag of TPX®. Thus, the strain <I>E. coli</I> BW 25113 has been inoculated in a small bag that was sealed. The bag was placed inside a tube and incubated at 37 °C. Growth was visualy assessed in the bags after 17 hours at 37°C (figure 14), or with a culture in tube with agitation as a control over the same period (figure 15). This showed that the cells were able to proliferate in the plastic bag.
   </P></DIV><DIV class="group center"><P class="legend">
   Figure 14: Growth test of <I>E. coli</I> BW 25113 inside a small bag of TPX®.  (t = 17 hours, 37 °C). 
  </P></DIV><DIV class="group center"><P class="legend">
   Figure 15: Control of figure 14 with a growth test of <I>E. coli</I> BW 25113 in a culture tube. 
The culture tube contains bacteria growing in parallel of the biological sample tubes shown above (t = 17 hours, 37 °C, 130 rpm).
 </P></DIV><DIV class="group"><P class="text">
   We then assessed the biomass evolution during long term growth in the TPX® bag (7 to 10 days). Figure 16 displays the result. The growth tendency is toward a continous increase over the experiment time.
   </P></DIV><DIV class="group center"><P class="legend">
   Figure 16: Growth test in TPX® bag by monitoring OD at 600 nm over 7 days to 10 days. </P></DIV><DIV class="group center"><P class="text">
  We then checked the viability of the cells after 10 days of growth in the TPX® bag (tube 1, 1' and 2) or in the control tube (tube 3), by spreading diluted volume of the cultures on Petri plates. Similar numbers of colonies were obtained, indicating that the cells survival is the same in both culture conditions.

 </P></DIV><DIV class="group center"><P class="legend">
   Figure 17: Colonies of <I>E. coli</I> BW 25113 on Petri dishes after an overnight incubation at 37°C to check survivability. </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: the strain can grow and survive over 10 days in the TPX® bag.</B></FONT></DIV><DIV class="subtitle" id="gaz"><H3>Gas diffusion tests</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: ensuring that formate and butyrate permeate through the TPX® bag</B></FONT></DIV><DIV class="group"><P class="text">
  For our system to be efficient, it was necessary to check that butyrate and formate could permeate through the TPX® bag. This was  verified as described in the <A target="_blank" href="https://2015.igem.org/Team:Toulouse/Experiments#TPX">Protocol</A> section. Shortly, a solution of butyrate or formate was placed in the TPX® bag. The molecules exchanged between the bag and the surrounding air were desorbed in a solution of sodium bicarbonate. This resulting solution was analysed by NMR.
  </P></DIV><DIV class="group"><P class="text">
  On a first assay, we did not detect any permeation of butyric acid through the TPX® bag. The same experiment was therefore performed by placing directly the bag in the solution of sodium bicarbonate. The control is an injection of a 4% (V/V) solution of butyric acid in water. 
  </P></DIV><DIV class="group center"><P class="legend">
  
  Figure 18: NMR Spectrum of butyric acid liquid control 
  in red and butyric acid liquid which passed through
  TPX bag in blue. <SUP>*</SUP> Blue curve is zoomed 1340 times more 
  than red curve. Each condition was tested in two replicates.
  </P></DIV><DIV class="group center"><P class="legend">
  
  Table 1: Concentrations of butyric acid calculated from the NMR spectrum
  </P></DIV><DIV class="group"><P class="text">
  From these results, we can conclude that the TPX® enables butyric acid 
  to weakly pass outside the bag. We detected only a small quantity but
  an optimization of the device could be made with a plastic containing 
  bigger pores. 
  
 For the formic acid, we were able to detect it in the gas, probably because its pKa is lower than the butyric acid one. 
  </P></DIV><DIV class="group center"><P class="legend">
  Figure 19: NMR spectrum of formic acid gas control in red and formic 
  acid gas which passed through TPX bag in blue.
  </P></DIV><DIV class="group center"><P class="legend">
  
  Table 2: Concentrations of formic acid corresponding to NMR spectrum
  </P></DIV><DIV class="group center"><P class="text">
  According to these results, TPX® lets 56% of formic acid 
  to pass outside the bag in the gas phase. Therefore, formic acid permeats through the TPX plastic. Using a more porous plastic 
  as proposed for the butyrate, we supposed this percentage to further increase.
  </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: formate permeates through the TPX® while butyrate exchange will need to be improved</B></FONT></DIV><DIV class="subtitle" id="safety"><H3>Safety tests</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: demonstration of TPX® capacity to retain the bacteria</B></FONT></DIV><DIV class="group center"><P class="text">
The bacteria impermeability of the <B>TPX®</B> was tested through inoculation of the strain  
  <I>E. coli</I> BW 25113 in M9 defined medium complemented with glucose inside the bag. Then, the inoculated bag was immersed in a glass measuring cylinder containing M9 medium with glucose. OD600 nm served to monitor a putative growth in the external medium. </P></DIV><DIV class="group center"><P class="legend">
  
  Figure 20: Measuring cylinders used for the safety test of the TPX® polymer. 
The cylinder on the left contains the TPX® bag with <I>E. coli</I> BW 25113 immersed in M9 medium after 27 hours of growth at 37 °C. On the right, the negative control cylinder contains a bag of TPX® without bacteria after 27 hours of growth at 37 °C.
  </P></DIV><DIV class="group"><P class="text">
 Over this time, no growth was detected in the medium surrounding the plastic bag. We concluded that the bacteria are contained in the bag. 
  </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: bacteria cannot permeate through the TPX® material</B></FONT></DIV><DIV class="group center"><P class="highlight" style="padding:10px;"><B>In a nutshell:</B> The TPX® material successfully contains growing bacteria while allowing the diffusion of formate and, in a lesser extend, of butyrate. These tests will profit to all Igem teams looking for a solution to contain their strains .
</P></DIV><CENTER><DIV class="title" id="biolo"><H3>Growth tests</H3></DIV></CENTER><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#biomass">- Biomass, substrate and products</A></LI><LI><A href="#survival">- Bacteria survival</A></LI></UL></DIV></CENTER><DIV class="group center"><P class="text">
  Our final objective was to prepare a bag containing bacteria producing alternatively butyric acid (during the day)
and formic acid (during the night) and for a period of time of at least ten days (so that beekeepers don't have to change them every morning...). 

So we faced some biological questions:
  </P></DIV><DIV class="group"><UL align="justify" style="font-size:15px;"><LI>Can bacteria live during ten days under micro-aerobic conditions?</LI><LI>Which carbon source is suitable to produce continuously acids?</LI><LI>Are formic and butyric acids toxic for <I>E. coli</I>?</LI></UL></DIV><DIV class="group center"><P class="text">
  In order to characterize the <I>E. coli</I> strain growth in the conditions we plan to use in our device, we used a culture under
  aerobic and micro-aerobic conditions, measured the growth rate (OD) and analyzed the supernatant to measure the concentration of fermentation products. 
  </P></DIV><DIV class="group center"><P class="text">
  Micro-aerobic conditions were obtained with culture in specific Falcon tubes with holes and covered with a membrane that let the oxygen pass through. They were incubated at 37 °C 

without agitation (mimicking the aerobic and lack of agitation condition present in our device).
  
  </P></DIV><DIV class="group"><P class="text">
  Aerobic conditions were obtained with a classical Erlenmeyer incubated at 37 °C with agitation.
  </P></DIV><DIV class="group"><P class="text">
We used a minimal M9 medium to identify formic or butyric acid production by NMR.
  </P></DIV><DIV class="subtitle" id="biomass"><H3>Biomass, substrate and products</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to measure formate production of wild-type strains in microaerobic conditions</B></FONT></DIV><DIV class="group"><P class="text">
In order to plot biomass concentration we converted the OD values with the following equation:
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
$$ X=OD_{600nm}\times 0,4325 $$
</P><P style="font-size:15px;">
Where X is the cell concentration (g.L<SUP>-1</SUP>)
</P></DIV><DIV class="group"><P class="text">
Substrate and products concentration was inferred from the peak area of each molecule on the NMR spectrum. 

Concentrations were calculated with the following equation:
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
$$[A]=\frac{Area_{molecule}}{Area_TSP} \times [TSP] \times \frac{\textrm{TSP proton number}}{\textrm{A proton number}} \times DF $$
  </P></DIV><UL style="font-size:15px;"><LI>[A] = concentration of molecule in our solution in mM</LI><LI>Area<SUB>TSP</SUB> = 1</LI><LI>[TSP] = 1.075 mM concentration of Trimethylsilyl propanoic acid in NMR tube, internal reference for 

quantification</LI><LI>TSP proton number = 9</LI><LI>DF = Dilution Factor = 1.25</LI></UL><CENTER><P class="text">
Biomass and concentration of various molecules were issued from fermentation under aerobic and microaerobic conditions.
  </P></CENTER><CENTER><P class="legend">
Figure 21: Results of micro-aerobic and aerobic culture after 5 days. Culture of E. coli BW25113 in M9 medium with [glucose] = 15 mM, at 37 °C
</P></CENTER><DIV class="group center"><P class="text">
Glucose was consumed approximately at the same rate for both conditions but was converted differently depending on the condition. 
Under aerobic conditions the biomass reached 3 g/L whereas under micro-aerobic conditions there was six times less biomass. 
On the contrary, there were far less products in aerobic conditions.

 </P></DIV><DIV class="group center"><P class="text">
Our results were in line with our objective to produce acids in a microporous bag because we demonstrated that with microaerobic conditions, 
bacteria grew slowly and produced fermentation products.
 </P></DIV><DIV class="group"><P class="text">
To convert the concentration of formate into formic acid we used the famous Henderson Hasselbalch equation:


</P></DIV><DIV class="group center"><P style="font-size:15px;">
$$ pH=pKa+log \left(\frac{C_{b}}{C_{a}} \right) $$
  </P></DIV><UL style="font-size:15px;"><LI>pH = 7 (pH of the M9 culture).</LI><LI>pKa: 3.7 for formic acid and 4.8 for butyric acid</LI><LI>C<SUB>b</SUB>: base concentration</LI><LI>C<SUB>a</SUB>: acid concentration</LI></UL><DIV class="group center"><P class="text">
As mentioned in the <A target="_blank" href="https://2015.igem.org/Team:Toulouse/Experiments#mortalV">Eradicate</A> part, our goal was to produce 50 µM of formic acid to kill varroa, and this corresponds to 77,7 mM of formate.
</P></DIV><DIV class="group center"><P class="text">
Under our microaerobic conditions, the wild type <I>E. coli</I> strain produced 32 mM of formate. 
It was therefore necessary to further improve the metabolic production by adding genes involved in formate production in order to increase 
it by a factor of 240%. For a perfect regulation it would have been necessary to delete the chromosomic version 
of <I>pflB</I> in <I>E. coli</I> genome to avoid formate production during the day.
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: the wild type <I>E. coli</I> strain produces formate in microaerobic conditions but production has to be further improved</B></FONT></DIV><DIV class="subtitle" id="survival"><H3>Bacteria survival</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to verify how <I>E. coli</I> survives in long culture conditions</B></FONT></DIV><DIV class="group center"><P class="text">
As explained <A target="_blank" href="https://2015.igem.org/Team:Toulouse/Experiments#platecult">here</A> calculation of the survival of bacteria was perfomed via colony counting after plating on solid agar medium. Wild-type bacteria can easily survive at least 15 days in aerobic or microaerobic conditions without a carbon source. 
We hypothesized that in presence of a carbon source, we may be able to extend even further their survival time period.
</P></DIV><DIV class="group center"><P class="legend">Figure 22: Bacteria survival test performed with the BW25113 strain on M9 with 15 mM of glucose during 15 days. </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: <I>E. coli</I> can survive at least 15 days under normal culture conditions</B></FONT></DIV><DIV class="group center"><P class="highlight" style="padding:10px;"><B>In a nutshell:</B><I>E. coli</I> can produce important amount of fermentation products, among which formate, when cultured in microaerobiosis. The strain can survive up to two weeks 
in normal culture conditions. This validates the possible use of long lasting cultures in our device. 
</P></DIV><CENTER><DIV class="title" id="carbon"><H3>Choice of the carbon source to sustain formic and butyric acids production during 10 days</H3><H3><CENTER><DIV id="breadcrumb" class="clear" style="float: center;"><UL><LI><A href="#biosilta">- Characteristics of Biosilta kit</A></LI><LI><A href="#grosilta">- Growth culture with Biosilta kit</A></LI><LI><A href="#acid">- Modelling of formic and butyric acids production</A></LI><LI><A href="#conc">- Testing different concentrations of Biosilta kit</A></LI></UL></DIV></CENTER></H3><H3>Characteristics of the Biosilta kit</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: optimizing the release of glucose using Biosilta kit</B></FONT></DIV><DIV class="group center"><P class="text">
Biosilta kit is a technology which enables production of a lot of 
recombinant proteins owing to a slow carbon source delivery during 
24 hours. This technology is based on polymer degradation by an depolymerisation enzyme 
leading to a perfect control of the quantity of substrate released in the medium for a given period of time.
 We wanted to use this technology to cultivate our cells during one 
 or two weeks in conditions allowing the production of butyrate and formate.
 The medium with the polymer was solid and contained in separate bags. 
 In order to quantify the production of butyrate and formate in these conditions, we 
 first performed a test with a high concentration of the enzyme (50 U/L) and measured the kinetics of the polymer degradation.
 </P></DIV><DIV class="group center"><P class="legend">
Figure 23: Kinetics of Biosilta polymer degradation measured with glucose release.
 
 </P></DIV><DIV class="group"><P class="text">
  In order to have a global idea of the release of glucose per unit of time we then calculated the average rate of release with the following formula:
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
 $$ v_{glucose1}=\frac{[glucose]}{time}=\frac{11.1}{3.97}=2.80 g.L^{-1}.h^{-1}
, for [E]_{1}=50 U.L^{-1} (4) $$
</P></DIV><DIV class="group"><P class="text">
With a final glucose concentration of 13 g/L for one bag of polymer, 
the rate of glucose release necessary to have glucose during 13 days can be calculated with the following formula.
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
 $$ v_{glucose2}=\frac{13}{13 days}=\frac{13}{322 hours}=0.0403 
 g.L^{-1}.h^{-1} (5) $$
</P></DIV><DIV class="group"><P class="text">
The reduction factor was calculated:
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
 $$ RF=\frac{v_{glucose1}}{v_{glucose2}}=\frac{2.8}{0.0403}=69.44 (6)$$
</P></DIV><DIV class="group"><P class="text">
So, the concentration of enzyme that we had to use should have been:
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
 $$ [E]_{2}=\frac{[E]_{1}}{RF}=\frac{50}{69.44}=0.72 U.L^{-1} (7)$$
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: It is possible with a Bio Silta Kit to have a continuous release of glucose for 13 days</B></FONT></DIV><DIV class="subtitle" id="grosilta"><H3>Growth assessment using the Biosilta kit</H3><H3><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: optimizing a long term linear growth of <I>E. coli</I> using Biosilta kit</B></FONT></DIV><DIV class="group center"><P class="text">
  As we do not know any growth characteristics of bacteria in the Biosilta 
  medium we tested different enzyme concentrations and not 
  only the one calculated previously.  Acquisition of the growth stopped due to a problem with the plate reading system where the bacteria where growing. Hence, there is a break at 5 days.
  </P></DIV><CENTER><P class="legend">
  Figure 24: Bacteria growth as a function of different enzyme
  concentrations in Biosilta medium. Growth tests were performed in 48 wells
  plates attached to an OD reader.
  </P></CENTER><DIV class="group center"><P class="text">
 Except for the 1.5 U/L enzyme concentration, there was an increase of OD in all conditions during 12 days. This means that glucose 
 releasing worked as expected. At the beginning there is 
 an exponential growth because some glucose is directly available in the
 medium. This phase was therefore not the one to analyse. After 2 days and until the end, the growth
 rate changes gradually and becomes constant, at the highest between 2 and 4 days and decreasing thereafter. 
  </P></DIV><DIV class="group center"><P class="text">
We therefore knew that it was possible to have continuous 
growth during at least 12 days. However, in our control 
without enzyme, there was also some bacterial growth so either another substrate was available 
or bacteria could realy degrade the polymer. More tests will be needed to verify if and how <I>E. coli</I> can use this polymer. 
  </P></DIV><DIV class="group center"><P class="text">
In order to answer these questions we performed some cultures in Falcon tubes 
to analyse the products and the evolution of the polymer quantity. Culture without enzyme and with a concentration of enzyme of 0.72 U/L were performed. 
  </P></DIV><CENTER><P class="legend">
Figure 25: Results of BW25113 culture on Biosilta medium after 4 days. Test of two conditions: without enzyme and with [Enzyme] = 0.72 U/L
</P></CENTER><DIV class="group center"><P class="text">
  Figure 25 (right panel) shows that the polymer is not degraded by <I>E. coli</I> which is probably capable of finding another carbon source within the Biosilta medium. The concentration of products (left panel) is clearly different in the two conditions: with the enzyme, more fermentation products are formed. 
  
  Fermentation products have high concentrations in comparison to the cultures in performed in M9 (15 mM glucose), around 20 times more for lactate, 3 times more for acetate and 2 times more for ethanol.
  We hypothesized that by deleting the various production pathways for lactate, acetate and ethanol and the degradation pathway for
  formate we should be able to produce enough formate. 
  </P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: Fermentation products concentration can be improved with the enzymatic degradation of the Biosilta polymer. </B></FONT></DIV></H3><H3>Modelling of formic and butyric acid production</H3></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: modeling the production of formic and butyric acids as a function of the glucose release rates.</B></FONT></DIV><DIV class="group center"><P class="text">
  With the rate of glucose release calculated above, an FBA and FVA simulation were launched as explained in <A target="_blank" href="https://2015.igem.org/Team:Toulouse/Modeling#part2">Modeling part</A>. 
  Some conversion between the model and the real condition are necessary and they are also explained there. 

In order to model production in the most similar conditions to fit real experiment we chose a glucose rate of 0.0403 g.L<SUP>-1</SUP>.h<SUP>-1</SUP> that correspond to 0.72 U/L of enzyme.

To convert formate production into formic acid concentration we used equation (3). 
</P></DIV><CENTER><P class="legend">
  Figure 26: Modeling of formic acid production as a function of different growth rates for a glucose release of 0.0403 g.L<SUP>-1</SUP>.h<SUP>-1</SUP>.
  </P></CENTER><DIV class="group center"><P class="text">
  Our goal was to produce at least 50 µmol/L. The graph clearly shows a maximum production of 6 µmol/L. To reach our goal we then have to change the growth rate.
  </P></DIV><CENTER><P class="legend">
 Figure 27: Modelling formic acid production as a function of glucose release for different growth rates (in h<SUP>-1</SUP>).
  </P></CENTER><DIV class="group center"><P class="text">
  To produce 50 µmol/L of formic acid different strategies are possible. 
  If we choose a low growth rate then a low glucose rate is be necessary and vice-versa. 
  As bacteria have to live during at least ten days it was better to have a continuous slow growth rate. 
  Moreover, it would consume less glucose per hour so we would need a lower polymer concentration in our bag at the beginning. 
  Thus, we chose a growth rate of 0.2h<SUP>-1</SUP>, and we could determinate the glucose release value needed.
  </P></DIV><DIV class="group center"><P style="font-size:15px;">
 [Formic acid] (μmol.L<SUP>-1</SUP> )=166.88 ×[Glucose]  (g.L<SUP>-1</SUP>.h<SUP>-1</SUP> ) (9)
 
 $$ [Glucose]  =  \frac{50}{166.88}=0.3 g.L^{-1}.h^{-1}    (10)  $$ 
</P></DIV><DIV class="group"><P class="text">
  Now, we will see which butyric acid concentration we can theoretically produce.
  </P></DIV><CENTER><P class="legend">
 Figure 28: Modelling of butyric acid production as a function of glucose release for different growth rates.
  </P></CENTER><DIV class="group center"><P class="text">
  According to the modelling results described in figure 28, we have to produce around 100µmol/L of butyric acid
  that corresponds to 0.0092% (V/V). As explained in the &quot;Results&quot; part, <A href="https://2015.igem.org/Team:Toulouse/Results#varrotest">&quot;attract&quot;</A> section,
  our objective was to produce at least 0.00001%, so the modelling indicates that we can theoretically reach it.

Nevertheless, in order to have this glucose release production, it was necessary to calculate 
how much polymer and enzyme concentration are needed. 

With the same equations as we used in <A target="_blank" href="https://2015.igem.org/Team:Toulouse/Results#biosilta">Characteristics of Biosilta kit</A> 
we determined which total quantity of glucose is needed for a fortnight. 
</P></DIV><DIV class="group center"><P style="font-size:15px;">
$$[Glucose]=v_{glucose}\times time=0.3\times 322=96.6 g.L^{-1}    (11)$$
</P></DIV><DIV class="group center"><P class="text">
Knowing that one Biosilta kit contains the equivalence of 13 g/L of glucose, we should therefore concentrate the medium 7 times. 

To obtain a glucose release of 0.3 g.L<SUP>-1</SUP>.h<SUP>-1</SUP> value, we nee 5 U/L of enzyme.

Thus, we tested different concentrations of Biosilta medium with different enzyme concentrations.
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: Biosilta medium has to be used in a more concentrated manner to reach our growth objectives.</B></FONT></DIV><DIV class="subtitle" id="conc"><H3>Testing different concentrations of Biosilta medium</H3><H3><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to determine if we can concentrate Biosilta medium to improve the concentration of fermentation products.</B></FONT></DIV><DIV class="group center"><P class="text">
We do not have the exact composition of Biosilta medium and cannot ascertain innocuity of higher titers of the polymer on growth. 
The best approach was to actually test it directly.

</P></DIV><CENTER><P class="legend">
Figure 29: Bacteria growth as a function of time for different concentrations of Biosilta medium. Culture with BW25113 on 48 wells plate and optical reader
</P></CENTER><DIV class="group center"><P class="text">
From these three figures it is evident that we cannot reach the growth level that is expected and that concentrated Biosilta is actually detrimental to growth.
</P></DIV><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Conclusion: concentrating the Biosilta medium is not compatible with <I>E. coli</I> growth.</B></FONT></DIV><DIV class="group center"><P class="highlight" style="padding:10px;"><B>In a nutshell:</B>The Biosilta medium was successfully used to have a continuous release of glucose over two weeks. Optimizing the protocol settings allows improving the production of fermentation product. The modeling suggests that concentration the Biosilta medium should even improve formate and butyrate productions but the experimental validation of this production has shown that this is not compatible with <I>E. coli</I> growth.
</P></DIV></H3><H3>Testing acids toxicity</H3><H3><DIV style="font-size:14px;"><FONT color="#EF9D2A"><B>Aim: to test butyric and formic acid effects on <I>E. coli</I> growth.</B></FONT></DIV><DIV class="group center"><P class="text">
In order to measure BW25113 resistance to different acids concentrations we tested two medium: LB and M9 with 15 mM glucose.
</P></DIV><CENTER><P class="legend">
 Figure 31: Optical density in function of time for different <B>formic acid</B> concentrations. LB medium is represented with green curves and M9 medium with blue curves. Each condition is tested in three replicates and standard deviation is represented in orange.
 </P><P class="legend">
 Figure 32: Optical density in function of time for different <B>butyric acid</B> concentrations and two medium. LB medium is represented with green curves and M9 medium with blue curves. Each condition is tested in three replicates and standard deviation is represented in orange.
 </P></CENTER><DIV class="group center"><P class="text">
In M9 medium, the growth is slower at the beginning of the growth curve, however the maximum value of OD reached is almost the same for both medium. 

There is a visible difference between the two culture media. Growth in LB is clearly less tolerant to either formic or butyric acid. As M9 (and not LB) is a buffered culture medium we measured the pH of both media with different acids concentrations. 
</P></DIV><CENTER><P class="legend">
 Figure 33: pH of the two media in function of the concentration of formic acid and butyric acid. LB medium is represented with green curves and M9 medium with blue curves. pH was measured with pH paper. Each condition was tested three times.
 </P></CENTER><DIV class="group center"><P class="text">
It is clear that in M9 medium the pH remains at pH 7 at high acid concentrations but not in the LB medium. Moreover we now know that bacteria do not grow when the pH is around 5. We will now see if buffering our medium has a positive action on growth.
</P></DIV><P class="highlight" style="padding:10px;"><B>In a nutshell:</B>Butyric and formic acids production does not stop the growth within our targeted ranges of concentrations. 

</P></H3><H3>READ MORE</H3></DIV></CENTER><DIV class="group center"><DIV style="width:20%;"><A href="https://2015.igem.org/Team:Toulouse/Description"><DIV class="title"><H3>Project Description</H3></DIV></A></DIV><DIV style="width:20%;"><A href="https://2015.igem.org/Team:Toulouse/Description/Attract"><DIV class="title"><H3>Attract</H3></DIV></A></DIV><DIV style="width:20%;"><A href="https://2015.igem.org/Team:Toulouse/Description/Eradicate"><DIV class="title"><H3>Eradicate</H3></DIV></A></DIV><DIV style="width:20%;"><A href="https://2015.igem.org/Team:Toulouse/Description/Regulation"><DIV class="title"><H3>Regulation</H3></DIV></A></DIV><DIV style="width:20%;"><A href="https://2015.igem.org/Team:Toulouse/Design"><DIV class="title"><H3>Device: TrApiColi</H3></DIV></A></DIV></DIV></MAIN></DIV><DIV class="wrapper row4"><DIV class="container clear" style="padding-top:30px;"><CENTER><P class="maintitle">		  
References
</P></CENTER><DIV class="clear"><UL><LI>
[1] Saini M, Wang ZW, Chiang C-J &amp; Chao Y-P (2014) Metabolic Engineering of Escherichia coli for Production of Butyric Acid. J. Agric. Food Chem. 62: 4342–4348
</LI><LI>
[2] Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, Davidson EA, Scouras A, Ellington AD, Marcotte EM &amp; Voigt CA (2005) Synthetic biology: Engineering <I>Escherichia coli</I> to see light. Nature 438: 441–442
</LI></UL></DIV></DIV></DIV><DIV class="wrapper row5"><FOOTER id="footer" class="clear"><DIV class="group"><DIV class="one_third first"><H6 class="title">iGEM Toulouse 2015</H6><ADDRESS class="btmspace-15">
        INSA Toulouse
        135, Avenue de Rangueil
        31400 TOULOUSE
		France</ADDRESS></DIV><DIV class="one_third"><H6 class="title">Follow us</H6></DIV><DIV class="one_third"><H6 class="title">For more informations</H6><A href="mailto:igemtoulouse2015@gmail.com" style="border:1px solid #fff;-webkit-border-radius: 5px; 
		-moz-border-radius: 10px; border-radius: 5px; padding: 7px 16px 5px; color:white; 
		text-decoration:none; font-size:17px; background:none;  display:inline-block;">Contact</A></DIV></DIV><CENTER><P>Website by <A href="https://2015.igem.org/Team:Toulouse">Team iGEM Toulouse 2015</A>&amp;<A href="http://www.os-templates.com/" title="Free Website Templates">OS Templates</A></P></CENTER></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>