Team:Toulouse/Parts
iGEM Toulouse 2015
Home
Context
Achievements
Team
Presentation
Gallery
Fun Facts
Project
Project: Description
Design: Attract
Design: Eradicate
Design: Regulate
Device: TrapiColi
Results
Attract
Eradicate
Regulate
Containment
Growth tests
Cytoxicity
Parts
New BioBricks
Module: Attract
Module: Eradicate
Module: Regulate
Modeling
Metabolic networks
FBA
Diffusion
Optimisation
Human Practice
Meetups & Collaborations
Ethics & IP
Education
Safety
Wet lab
Notebook
Protocols
Attributions
Attributions
Supports
Biobricks
Content
- Attraction (butyrate pathway)
- Eradication (formate pathway)
- Circadian switch
In the list below you will find an overview over the BioBrick parts created by the iGEM Toulouse 2015 team and added to the IGEM registry.
For our project, we worked on three main biological modules : Attract, Eradicate, and Regulate to alternatively produce the two molecules of interest (butyrate by day, formate by night).
Name
Type
Genic construction
Module
Length (bp)
Sequencing
References
BBa_K1587000
Composite (with RBS)
RBS-ho1
Regulate
744
Ok
[13] [14]
BBa_K1587001
Basic part
tesB
Attract
861
Ok
[3][4][5]
BBa_K1587002
Composite (with RBS)
RBS-pcyA
Regulate
796
Ok
[13][14]
BBa_K1587003
Basic part
crt
Attract
786
Sequenced until 736 pb : ok
[1][2][3][4][7]
BBa_K1587004
Device
P(Bla)-RBS-ccr-RBS-hbd-RBS-crt-RBS- tesB-RBS-atoB-Terminator
Attract
5192
Ok
[1][2][3][4][6]
BBa_K1587005
Device
P(Bla)-RBS-hbd-RBS-crt-RBS- tesB-RBS-atoB-Terminator
Attract
3827
Ok
[1][2][3][4][6]
BBa_K1587006
Device
POmpC-LacIbox-RBS-cI
Regulate
905
Ok
[12][13][14]
BBa_K1587007
Device
RBS-pflB-RBS-pflA-Terminator
Eradicate
3093
Ok
[7][8][9][10]
BBa_K1587008
Device
Strong Promotor-RBS-cph8
Regulate
2288
Ok
[11][13][14]
BBa_K1587009
Composite (with RBS)
POmpC-LacIbox-RBS-cI-RBS-pflB-RBS-pflA-Terminator
Eradicate
4006
-
[7][8][9][10][11][12][13][14]
Attraction (butyrate pathway)
The chassis we used is Escherichia coli, and this bacterium is not able to naturally produce butyrate. That is why we introduced genes from others bacterial strains to synthesize this molecule.
Basic parts
tesB (BBa_K1587001)
A gene from Escherichia coli (Accession Number: EG10995 ) involved in the butyrate pathway that enables its production directly from acyl-coAs. This group of enzymes catalyzes the hydrolysis of acyl-CoAs into free fatty acid (in our case, butyryl-coA into butyrate) plus reduced coenzyme A (CoA-SH).
crt (BBa_K1587003)
A gene from Clostridium acetobutylicum (Accession Number: GJIH-2688 ) introduced in our  bacterium after codon optimization in order to obtain a better  expression in our strain. The crt enzyme substrate is 3-hydroxybutyryl CoA,  and the product is Crotonyl CoA. This reaction does not need any coenzyme.
Other parts
ccr-Butyrate pathway (BBa_K1587004)
This BioBrick construction is composed of a constitutive promoter P(Bla)  ( BBa_I14018 ) and 5 genes from three different micro-organisms :   in yellow are the E. coli genes, in blue those from Clostridium   acetobutylicum and finally, the purple gene is from Streptomyces   collinus. A Ribosome Binding Site (RBS; represented by a green circle; BBa_B0030 ), is added between each gene in order to improve the proteic  synthesis.  Finally, a strong terminator BBa_B1006) represents the end of the sequence.
tesB and crt have been described above. ccr encodes a crotonyl-CoA reductase (an oxidoreductase which acts on the double bond CH=CH). hbd from Clostridium acetobutylicum encodes a 3-hydroxybutyryl-CoA  dehydrogenase (an oxidoreductase which catalyses the formation of alcohol function). atoB , from E. coli, encodes an acetyl-CoA acetyltransferase which catalyses the condensation of two acetyl-CoA.
Butyrate pathway wihout ccr (BBa_K1587005)
This BioBrick construction is the same as the previous one, but for the fact that it does not  contain the ccr gene from Streptomyces collinus.  It is composed of a constitutive promoter P(Bla) ( BBa_I14018 ) and of 4 of the 5 genes present in the previous construction. Genes are issued from two micro-organisms : in yellow from E. coli, and in blue from Clostridium acetobutylicum. The green  circles correspond to the strong RBS sequence ( BBa_B0030 ) based on Ron Weiss thesis and the red one is the terminator ( BBa_B1006 ).
Eradication (formate pathway)
The second module is dedicated to formate production and composed of two key genes : pflA and pflB .
Formate pathway ( BBa_K1587007 )
pflB encodes a pyruvate formate lyase, an enzyme which catalyses  the cutting between C1 and C2 carbons of pyruvate. This enzyme   is oxygen-sensitive and is only active in microaerobic or anaerobic   conditions.
pflA encodes the pyruvate formate lyase activase, an enzyme  which activates pflB.
The formate compound is naturally produced by E. coli, that is why we  decided to overexpress these two genes. For this, pflB and pflA are put  together with two RBS ( BBa_B0030 ) in front of them to allow the proteic synthesis. A strong terminator ( BBa_B1006 ) ends up the sequence.
Circadian switch
We aimed to produce butyric acid in our trap during  the day and formic acid during the night.   We designed a light response system which is improved to obtain an   on/off switch of genic expression.
The center of the light sensor is composed of membrane proteins PCB  (chromophore phycocyanobilin) and the hybrid protein Cph8 (EnvZ and Cph1)[11].
PCB protein comes from a cyanobacterium Synechocystis sp. PCC 6803  and to be synthesized, it needs the expression of two genes:  heme oxygenase (Ho1) and biliverdin reductase (PcyA)[11].
Ho1 with RBS ( BBa_K1587000 )
A gene required for chromophore synthesis in photosynthetic  light-harvesting complexes, photoreceptors, and circadian clocks. ho1, along with pcyA, converts heme into the chromophore  phycocyanobillin (PCB). The gene comes from the cyanobacterium Synechocystis sp. PCC 6803.
The biobrick is composed of a strong RBS ( BBa_B0030 ) and Ho1 coding region ( BBa_K566022 ).
PcyA with RBS ( BBa_K1587002 )
A gene required for chromophore synthesis in photosynthetic  light-harvesting complexes, photoreceptors, and circadian clocks. The gene comes from the cyanobacterium Synechocystis sp. PCC 6803.
The biobrick is composed of a strong RBS ( BBa_B0030 ) and pcyA coding region ( BBa_K566023 ).
POmpC-LacIbox-RBS-cI ( BBa_K1587006 )
This biobrick contains the OmpC promoter ( BBa_R0082 ), a lacI box  separated from cI gene by a RBS sequence ( BBa_B0030 ).
Red light biosensor ( BBa_K1587008 )
This is an assembly between the strong promoter BBa_J23119 , an RBS ( BBa_B0030 ), and the chimeric Red light receptor Cph8 BBa_I15010 .
Regulated formate production ( BBa_K1587009 )
An assembly between biobricks BBa_K1587006 and BBa_K1587007 , which have been both successfully sequenced, for the regulation of formate  production by the circadian switch system.
References
[1] Layton DS & Trinh CT (2014) Engineering modular ester fermentative pathways in Escherichia coli. Metabolic Engineering 26: 77–88
[2] Saini M, Hong Chen M, Chiang C-J & Chao Y-P (2015) Potential production platform of n-butanol in Escherichia coli. Metabolic Engineering 27: 76–82
[3] Volker AR, Gogerty DS, Bartholomay C, Hennen-Bierwagen T, Zhu H & Bobik TA (2014) Fermentative production of short-chain fatty acids in Escherichia coli. Microbiology (Reading, Engl.) 160: 1513–1522
[4] Saini M, Wang ZW, Chiang C-J & Chao Y-P (2014) Metabolic Engineering of Escherichia coli for Production of Butyric Acid. J. Agric. Food Chem. 62: 4342–4348
[5] Hunt MC & Alexson SEH (2002) The role Acyl-CoA thioesterases play in mediating intracellular lipid metabolism. Prog. Lipid Res. 41: 99–130
[6] Aboulnaga E-H, Pinkenburg O, Schiffels J, El-Refai A, Buckel W & Selmer T (2013) Effect of an oxygen-tolerant bifurcating butyryl coenzyme A dehydrogenase/electron-transferring flavoprotein complex from Clostridium difficile on butyrate production in Escherichia coli. J. Bacteriol. 195: 3704–3713
[7]  Thesis: SUNYA Sirichai. July 2012. Dynamique de la réponse physiologique d’Escherichia coli à des perturbations maîtrisées de son environnement : vers le développement de nouveaux outils de changement d’échelle. Ingénieries Enzymatique et Microbienne.
[8]  CEA-CNRS- Aix Marseille Université. February 2015. Paris . Activation d’enzymes bactériennes pour convertir le CO2 en source d’énergie renouvelable.
[9] Crain AV & Broderick JB (2014) Pyruvate formate-lyase and its activation by pyruvate formate-lyase activating enzyme. J. Biol. Chem. 289: 5723–5729
[10] Wei X-X, Zheng W-T, Hou X, Liang J, Li Z-J, Wei X-X, Zheng W-T, Hou X, Liang J & Li Z-J (2015) Metabolic Engineering of Escherichia coli for Poly(3-hydroxybutyrate) Production under Microaerobic Condition. BioMed Research International, BioMed Research International 2015, 2015: e789315
[11] Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, Davidson EA, Scouras A, Ellington AD, Marcotte EM & Voigt CA (2005) Synthetic biology: Engineering Escherichia coli to see light. Nature 438: 441–442
[12] Okada K (2009) HO1 and PcyA proteins involved in phycobilin biosynthesis form a 1:2 complex with ferredoxin-1 required for photosynthesis. FEBS Lett. 583: 1251–1256
[13] Lee JM, Lee J, Kim T & Lee SK (2013) Switchable gene expression in Escherichia coli using a miniaturized photobioreactor. PLoS ONE 8: e52382
[14] Gambetta GA & Lagarias JC (2001) Genetic engineering of phytochrome biosynthesis in bacteria. PNAS 98: 10566–10571
iGEM Toulouse 2015
INSA Toulouse
135, Avenue de Rangueil
31400 TOULOUSE
France
Follow us
For more informations
Contact
Website by Team iGEM Toulouse 2015 & OS Templates
