<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pittsburgh skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pittsburgh</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Team</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Project</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Software">Software Tool</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Parts">Parts</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Safety">Safety</A></LI></DIV></DIV><DIV class="foreground" id="middle"><P style="font-size: 1.2em">
Biological records are ubiquitous throughout nature, from tree rings to sediment deposits. They allow scientists to delve back into the past and recover information that a human could not easily observe. Until recently, attempts at designing synthetic biological systems to perform this type of function at the molecular level have proven ineffective. However, the efficiency and versatility of CRISPR-Cas9 in gene editing has led to the development of promising new tools. Conventional CRISPR-Cas9 targets a specific sequence of DNA and makes a double stranded break, but it relies on host mechanisms for repair, which results in stochastic changes in the genome. Improving upon CRISPR-Cas9’s gene editing abilities, a fusion base editor complex (BE3) was developed consisting of the DNA base modification enzyme cytidine deaminase (CDA), Cas9 nickase (nCas9), and a base excision repair inhibitor called uracil DNA glycosylase inhibitor (ugi). This complex can produce a single nucleotide change, resulting in a much more certain alteration. The high fidelity allows for researchers to design complex systems while still retaining the targeting specificity of the original CRISPR-Cas9. 
</P><P style="font-size: 1.2em">
In the past year, researchers have developed two systems that utilize base editors to record stimuli1,2. These systems demonstrate the power of base editing, but recording capability was limited to logging an average concentration of stimuli over a period of time. Our system, termed CUTSCENE, builds upon these foundations by designing a method of true chronological event recording. Because it is based in bacterial cells, it provides the benefit of functioning in a variety of environments without the need of external power or guidance. Additionally, writing changes directly to the DNA allows for high information storage capacity and resistance to change. 
</P></DIV><DIV class="banner smallTitle text-center" id="pic1"><SPAN class="smallTitle-Text">CUTSCENE</SPAN></DIV><DIV class="foreground"><P style="font-size: 1.2em">
CUTSCENE consists of E. coli containing a low-copy writing plasmid and high-copy recording plasmids. A recording plasmid can be thought of as a roll of unexposed film, with each frame being the equivalent to a short, repeating sequence of DNA. The plasmid also contains two sgRNAs which direct the base editor. These sgRNAs are controlled by separate inducible promoters. The first inducer controls the production of sgRNA #1 and sets the cell into recording mode. This sgRNA directs the base editor to move along the DNA repeats, making mutations at a timed rate and constantly shifting which frame is in front of our base editor and available to record. The presence of a stimulus activates the promoter for sgRNA #2. Expression of this sgRNA directs the base editor to mark the current frame with a unique mutation. This edit creates a restriction enzyme cut site and stops the recording process for that specific plasmid (the other plasmids continue recording). Once the recording process finishes, a restriction digest occurs, followed by gel electrophoresis. The readout from our gel provides a quick and cost effective way of visualizing the time at which a stimulus occurred. Longer fragments correspond to later time points and vice versa. 
</P><P style="font-size: 1.2em">
This system allows for noninvasive, real-time monitoring of analytes in a system. As this system is limited only by the promoters that can be used to induce the second sgRNA, a variety of signals can be recorded, making the system a universal diagnostic tool. Some useful signals that can be measured include certain cytokines, pH, and pollutants. These could be utilized in diagnosing diseases such as peptic ulcers and early-stage cancers. CUTSCENE could be an invaluable tool for researchers as well. By tracking the levels of multiple molecules over the course of a cell’s life, investigators can classify dependent relationships. This information allows biologists to construct vastly improved cellular models, accelerating the rate of scientific discovery. 
</P><P style="font-size: 1.2em">
[1] Tang, W., &amp; Liu, D. R. (2018). Rewritable multi-event analog recording in bacterial and mammalian cells. Science, 360(6385). doi:10.1126/science.aap8992
</P><P style="font-size: 1.2em">
[2] Farzadfard, F. et al. (2018). Single-Nucleotide-Resolution Computing and Memory in Living Cells. BioRxiv. doi:10.1101/263657
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>