<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pittsburgh_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pittsburgh/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Team</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Project</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Software">Software Tool</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Parts">Parts</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="https://2018.igem.org/Team:Pittsburgh">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Public_Engagement">Education &amp; Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Pittsburgh/Safety">Safety</A></LI></DIV></DIV><DIV style="background-color: #efefef; padding: 90px"><DIV class="sidenav"><A href="#section1">Antibiotics and Inducers</A><A href="#section2">TE Buffer</A><A href="#section3">Liquid LB Media</A><A href="#section4">LB Agar Plates</A><A href="#section5">Liquid Cultures</A><A href="#section6">Transformation: DH5a</A><A href="#section7">Making Competent Cells</A><A href="#section8">Transformation S1030</A><A href="#section9">TBE Buffer</A><A href="#section10">Gel Electrophoresis</A><A href="#section11">Gel Extraction</A><A href="#section12">Miniprep</A><A href="#section13">PCR</A><A href="#section14">DNA Clean</A><A href="#section15">Annealing DNA Oligos</A><A href="#section16">Digetion</A><A href="#section17">Ligation</A><A href="#section18">Gibson</A><A href="#section19">TOPO Cloning</A><A href="#section23">Annealing RNA oligos</A><A href="#section24">Cas9 digestion</A><A href="#section20">CAMERA</A><A href="#section21">Base Editor Improvement</A><A href="#section22">CrisPy</A></DIV><DIV id="section1" class="main"><DL><DT>Antibiotics and Inducers</DT><DD><TABLE><TBODY><TR><TH>Antibiotic/Inducer</TH><TH>Stock Concentration</TH><TH>Working Concentration</TH><TH>Solvent</TH><TH>Storage</TH></TR><TR><TD>Ampicillin/Carbenicillin (Amp/Carb)</TD><TD>100 mg/mL</TD><TD>100 μg/mL</TD><TD>ddH2O</TD><TD>-20℃</TD></TR><TR><TD>Chloramphenicol (CAM)</TD><TD>35 mg/mL</TD><TD>35 μg/mL</TD><TD>100% ethanol</TD><TD>-20℃</TD></TR><TR><TD>Spectinomycin (Spec)</TD><TD>100 mg/mL</TD><TD>100 μg/mL</TD><TD>ddH2O</TD><TD>-20℃</TD></TR><TR><TD>Streptomycin (Strep)</TD><TD>100 mg/mL</TD><TD>50 μg/mL</TD><TD>ddH2O</TD><TD>-20℃</TD></TR><TR><TD>kanamycin (Kan)</TD><TD>50 mg/mL</TD><TD>50 μg/mL</TD><TD>ddH2O</TD><TD>-20℃</TD></TR><TR><TD>anhydrous tetracycline (aTc)</TD><TD>250 μg/mL</TD><TD>250 ng/mL</TD><TD>50% ethanol</TD><TD>-20℃, in foil wrap (avoid light exposure)</TD></TR><TR><TD>Isopropyl β-D-1-thiogalactopyranoside (IPTG)</TD><TD>0.1 M (24 mg/mL)</TD><TD>0.1 mM</TD><TD>ddH2O</TD><TD>-20℃</TD></TR><TR><TD>Arabinose</TD><TD>20% (0.2 g/mL)</TD><TD>0.2%</TD><TD>ddH2O</TD><TD>Room temperature</TD></TR><TR><TD>Xylose</TD><TD>20%</TD><TD>2%</TD><TD>ddH2O</TD><TD>Room temperature</TD></TR></TBODY></TABLE></DD><DIV id="section2" class="main"><DT>TE Buffer<SUP>[1]</SUP></DT><DD><TABLE><TBODY><TR><TH>Component</TH><TH>Volume (Total 500 mL)</TH></TR><TR><TD>1M Tris (pH 8)</TD><TD>5 mL</TD></TR><TR><TD>0.5 M EDTA (pH 8)</TD><TD>1 mL</TD></TR><TR><TD>distilled water</TD><TD> 496 mL</TD></TR></TBODY></TABLE></DD></DIV><DIV id="section3" class="main"><DT>Liquid LB Media</DT><DD>1. To make 1L liquid LB media, weigh 25g of anhydrous LB Broth and add it to 1000 mL of the Millipore water.  
2. Screw the cap partially and place the media bottle in the autoclave tray along with a water bath about 3-5 cm high. Apply Autoclave tape to the top of the bottle.  
3. Autoclave the bottle under the “liquid” setting.  
4. Allow to cool on bench top before use or adding antibiotics.
  </DD></DIV><DIV id="section4" class="main"><DT>LB Agar plates</DT><DD>1. Turn on the water bath to 60℃.  
2. To make 500 mL media, add 12.5g of anhydrous LB Broth and 7.5g of BD BactoTM Agar to 500 mL of the Millipore water.  
3. Screw the cap partially and place the media bottle in the autoclave tray along with a water bath about 3-5 cm high. Apply Autoclave tape to the top of the bottle.  
4. Autoclave the bottle under the “liquid” setting.  
5. Allow to cool in water bath before use or adding antibiotics.  
6. Add antibiotic for selectivity as desired and distribute LB broth onto the plates to cover about halfway. 
7. Allow the plates to cool on the counter and store inverted in the 4℃ refrigerator.
  </DD></DIV><DIV id="section5" class="main"><DT>Liquid Culture</DT><DD>1. Aliquot 5ml of LB into each 15 mL conical tube. Add antibiotic as desired. 
2. Remove plates from the fridge and use a marker to label the desired single origin colonies that are easy to pick.  
3. Using the p10 pipette tip, gently pick the colony and place it in the tube with liquid media.  
4. Screw the cap partially and apply lab tape to the top of the tube. &lt;.br&gt; 
5. Place the liquid culture tubes on the shaker at 37℃ and 220 rpm overnight (16-18 hours).
  </DD></DIV><DIV id="section6" class="main"><DT>Transformation of E.coli DH5a Cells</DT><DD>1. Thaw chemically-competent DH5a cells on ice.  
2. Add 1-5 μL containing 1 pg-100 ng of plasmid DNA to 50 μL of competent cells.  
3. Place the mixture on ice for 30 minutes.  
4. Heat shock at 42℃ for 30 seconds.  
5. Transfer tubes to ice for 5 minutes.  
6. Add 950 μl of room temperature LB or SOC media to each tube.  
7. Incubate the tube at 37℃ for 60 minutes. Shake vigorously (250 rpm).  
8. Centrifuge the cells at 6800 rpm for 1 minute.  
9. Remove 900μL of the supernatant. Resuspend the cells and pipette 100 μL onto the selection plates.  
10. Place the plates upside down and incubate overnight at 37℃ (18-20 hours).
  </DD></DIV><DIV id="section7" class="main"><DT>Preparation of chemically competent E. coli cells</DT><DD>Using Zymo Mix &amp; Go E. coli Transformation Kit with a few differences: 
1. Use 0.5 mL of fresh, overnight E.coli culture to inoculate 50 mL LB in a flask. Shake culture vigorously (150-250 rpm) at 37℃ until the OD600 is 0.4-0.6. 
2. Prepare fresh buffer on ice: Add 2.5 mL Dilution Buffer and 2.5 mL 2X stock Wash Buffer to make 5 mL 1X Wash Buffer. Add 2.5 mL Dilution Buffer and 2.5 mL 2X stock Competent Buffer to make 5 mL 1X Competent Buffer. 
3. Transfer the culture from step 1 to ice. After 10 minutes, pellet the cell by centrifuge at 3300 rpm for 10 minutes at 4℃. 
4. Remove the supernatant and resuspended the cell in 5 mL 1X ice-cold Wash Buffer. Repellet the cells as step 3.
5. Completely remove the supernatant and resuspended the cell in 5 mL 1X ice-cold Competent Buffer. 
6. Aliquot 0.1 mL of cell suspension into sterile eppendorfs. Cells are now ready for transformation or can be stored in -80℃.
  </DD></DIV><DIV id="section8" class="main"><DT>Transformation of E. coli S1030 cells</DT><DD>1. Add 1-5 μL plasmid DNA to a tube of thawed Mix &amp; Go cells on ice.
2. When selecting with Ampicillin, spread 100 μl of the mixture onto a pre-warmed (37°C) plate. 
3. When selecting with other antibiotics, incubate the cells on ice for 5-10 minutes. Add 400 μL of SOC and incubate for 1 hour at 37°C with gentle shaking at 250 rpm. Then, spread 100 μL mixture directly onto pre-warmed plates.
  </DD></DIV><DIV id="section9" class="main"><DT>TBE Buffer</DT><DD>1. To make 1L 5x TBE stock solution, add 54g of Tris Base, 27.5g of boric acid and 20 mL of 0.5M EDTA. 
2. Dissolve in 800 mL ddH2O. 
3. Using stir bar and heat plate set to low. 
4. Bring final volume to 1L. 
5. Dilute to 1x TBE buffer before using for gel electrophresis. </DD></DIV><DIV id="section10" class="main"><DT>Gel Electrophresis</DT><DD>1. Put 0.5g of agar powder and 50 mL of New 1x TBE. Swirl gently and microwave for 45 seconds.  
2. Add 1uL of ethidium bromide to the hot gel and pour the gel with the desired comb.  
3. After the gel is solidify, add running buffer (Used 1x TBE) to completely submerge the gel.  
4. Add loading dye in a 5:1 DNA/loading dye ratio. (Ex. 25μL of  DNA and 5μL of dye)  
5. Add 5-10 μL of ladder to the first well. Add controls and digested DNA to the other wells as desired.  
6. Run gel at 120 V for desired time. 
  </DD></DIV><DIV id="section11" class="main"><DT>Gel Extraction - Zymoclean Gel DNA Recovery Kit</DT><DD>1. Excise the DNA fragment from the agarose gel and transfer it into an eppendorf. 
2. Add 3 volumes of ADB to each volume of agarose excised from the gel.  
3. Incubate at 55℃ for 5-10 minutes until the gel slice is completely dissolved.  
4. Transfer the melted agarose solution to a Zymo-Spin™  Column in a Collection Tube.  
5. Centrifuge at maximum speed for 45 seconds. Discard the flow-through.  
6. Add 200 μL of DNA Wash Buffer to the column and centrifuge 30 seconds. Discard the flow-through. Repeat the wash step. 
7. Add 10-20 μL DNA Elution Buffer or nuclease free water directly to the column matrix. Place column into an eppendorf and centrifuge for 30 seconds to elute DNA. 
8. Nanodrop and record the concentration.
  </DD></DIV><DIV id="section12" class="main"><DT>Miniprep - Zyppy™ Plasmid Miniprep Kit</DT><DD>1. Add 1.5 mL of bacterial culture grown in LB medium to a eppendorf and centrifuge for 30 seconds at maximum speed. Discard the supernatant. Repeat this step as needed.
2. Add 600 μL of TE buffer to resuspend the pellet. 
3. Add 100 μL of 7X Lysis Buffer (Blue) and mix by inverting the tube 4-6 times. 
4. Add 350 μL of cold Neutralization Buffer (Yellow) and mix thoroughly. A yellowish precipitate will form. 
5. Centrifuge at maximum speed for 4 minutes. 
6. Transfer the supernatant (at most 800 μL) into the provided Zymo-Spin™ IIN column. Place the column into a Collection Tube and centrifuge for 15 seconds. 
7. Discard the flow-through. Reat Step 6 if necessary. 
8. Add 200 μL of Endo-Wash Buffer to the column. Centrifuge for 30 seconds. 
9. Add 400 μL of Zyppy™ Wash Buffer to the column. Centrifuge for 1 minute. 
10. Transfer the column into a labeled eppendorf then add 30 μL of Zyppy™ Elution Buffer or nuclease free water directly to the column matrix and let stand for one minute at room temperature. 
11. Centrifuge for 1 minute to elute the plasmid DNA. 
12. Nanodrop and record the conentration. </DD></DIV><DIV id="section13" class="main"><DT>PCR - NEB Q5® High-Fidelity 2X Master Mix</DT><DD>Reaction Setup: <TABLE><TBODY><TR><TH>Component</TH><TH>25 μL Reaction</TH><TH>50 μL Reaction</TH></TR><TR><TD>Q5 High-Fidelity 2X Master Mix</TD><TD>12.5 µL</TD><TD>25 µL</TD></TR><TR><TD>10 µM Forward Primer</TD><TD>1.25 µL</TD><TD>2.5 µL</TD></TR><TR><TD>10 µM Reverse Primer</TD><TD>1.25 µL</TD><TD>2.5 µL</TD></TR><TR><TD>Template DNA (~25/50 ng)</TD><TD>variable</TD><TD>variable</TD></TR><TR><TD>Nuclease-Free Water</TD><TD>to 25 µL</TD><TD>to 50 µL</TD></TR></TBODY></TABLE>
Thermocycling Conditions for a Routine PCR: <TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD rowspan="3">30 Cycles</TD><TD>98°C</TD><TD>10 seconds</TD></TR><TR><TD>Ta for specific primers</TD><TD>20 seconds</TD></TR><TR><TD>72°C</TD><TD>Elongation time 1 min/kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>nfinity</TD></TR></TBODY></TABLE></DD></DIV><DIV id="section14" class="main"><DT>Column Purification -  Zymo DNA Clean &amp; Concentrator (DCC) Kit</DT><DD>1. For PCR product, add 5 volumes of DNA Binding Buffer to each volume of DNA sample. Mix briefly by vortexing. 
2. Transfer mixture to a provided Zymo-Spin™ Column in a Collection Tube.
3. Centrifuge for 30 seconds. Discard the flow-through. 
4. Add 200 µL DNA Wash Buffer to the column. Centrifuge for 30 seconds. Repeat
the wash step. 
5. Add ≥ 6 µL DNA Elution Buffer or nuclease-free water directly to the column matrix and incubate at room temperature for one minute. Transfer the column to an eppendorf and centrifuge for 30 seconds to elute the DNA.
  </DD></DIV><DIV id="section15" class="main"><DT>Annealing Oligos</DT><DD><TABLE><TBODY><TR><TH>Component</TH><TH>Amount</TH></TR><TR><TD>Phosphorylated oligos</TD><TD>500 ng each (1 μg total)</TD></TR><TR><TD>NEB 10X T4 Ligase Buffer</TD><TD>15 μL</TD></TR><TR><TD>Nuclease-free water</TD><TD> To 50 μL</TD></TR></TBODY></TABLE>
Incubate the mixed oligonucleotides to 85°C for 10 minutes and cool down slowly (do 30 min ramp from 85 down to 20).
  </DD></DIV><DIV id="section16" class="main"><DT>Restriction Enzyme Digestion</DT><DD>1. Reaction Setup: <TABLE><TBODY><TR><TH>Component</TH><TH>50 μL Reaction</TH></TR><TR><TD>Purified DNA</TD><TD>250-1000 ng</TD></TR><TR><TD>NEB CutSmart Buffer</TD><TD>5 μL</TD></TR><TR><TD>Restriction Enzyme</TD><TD>1 μL of each</TD></TR><TR><TD>Nuclease-free water</TD><TD>to 50 μL</TD></TR></TBODY></TABLE>
2. Incubate at 37°C for 30 minutes if doing diagnostic digestion or 2 hours if getting the DNA. 
  </DD></DIV><DIV id="section17" class="main"><DT>Ligation</DT><DD>1. Reaction Setup: <TABLE><TBODY><TR><TH>Component</TH><TH>15 μL Reaction</TH></TR><TR><TD>NEB T4 DNA Ligase Buffer (10X)</TD><TD>1.5 μL</TD></TR><TR><TD>Vector DNA</TD><TD>50 ng</TD></TR><TR><TD>Insert DNA</TD><TD>X ng</TD></TR><TR><TD>Nuclease-free water</TD><TD>to 15 μL</TD></TR><TR><TD>NEB T4 DNA Ligase*</TD><TD>1 μL</TD></TR></TBODY></TABLE>
*T4 DNA Ligase should be added last.
2. Incubate at 16°C overnight or room temperature for 30 minutes. 
3. Chill on ice and transform 5 μL of the reaction into 50 μL competent cells.
  </DD></DIV><DIV id="section18" class="main"><DT>Gibson Assembly - NEBuilder HiFi DNA Assembly</DT><DD>1. Reaction Setup on Ice: <TABLE><TBODY><TR><TH>2-3 Fragment Assembly</TH><TH>4-6 Fragment Assembly</TH><TH>Positive Control</TH></TR><TR><TD>Recommended DNA Molar Ratio</TD><TD>vector:insert = 1:2</TD><TD>vector:insert = 1:1</TD><TD>-</TD></TR><TR><TD>Total Amount of Fragments</TD><TD>0.015 - 0.1 pmols*</TD><TD>0.1 - 0.25 pmols</TD><TD>5 μL</TD></TR><TR><TD>NEBuilder HiFi DNA Assembly Master Mix</TD><TD>5 μL</TD><TD>5 μL</TD><TD>5 μL</TD></TR><TR><TD>ddH2O</TD><TD>Volume to 10 μL</TD><TD>Volume to 10 μL</TD><TD>0</TD></TR><TR><TD>Total Volume </TD><TD>10 μL**</TD><TD>10 μL**</TD><TD>10 μL</TD></TR></TBODY></TABLE>
*pmols = (weight in ng) x 1,000 / (base pairs x 650 daltons) 
**Can be cut down to half reaction. 
2. Incubate samples in a thermocycler at 50℃: 15 minutes - when 2 or 3 fragments are being assembled; 60 minutes - when 4–6 fragments are being assembled. 
3. Transformation: 
Follow the Transformation of E.coli DH5a Cells protocol with a few difference: 
Add 2 μL of product to 50 μL of competent cells. 
After heat shock, incubate on ice for 2 minutes instead of 5 minutes. </DD></DIV><DIV id="section19" class="main"><DT>TOPO cloning - Zero Blunt™ TOPO™ PCR Cloning Kit</DT><DD>1. Reaction Setup*: <TABLE><TBODY><TR><TH>Component</TH><TH>6 μL Reaction</TH></TR><TR><TD>Fresh Blunt-end PCR Product</TD><TD>0.5-4 μL</TD></TR><TR><TD>Salt Solution</TD><TD>1 μL</TD></TR><TR><TD>Sterile Water</TD><TD>to 6 μL</TD></TR><TR><TD>pCR™-Blunt II-TOPO®</TD><TD>1 μL</TD></TR></TBODY></TABLE>
*For transformation of chemically competent E. coli only. 
2. Mix gently and incubate for 5 minutes at room temperature. 
3. Place on ice. Proceed to transformation. 
  </DD></DIV><DIV id="section23" class="main"><DT>Annealing RNA oligos - GenScript CRISPR Ribonucleoprotein User Manual</DT><DD>1. Centrifuge RNA oligos at 12000 rpm for 2 minutes at 4℃. 
2. Resuspend oligos in nuclease free water to a final volume of 100 μM. 
3. Vortex for 15 seconds and Centrifuge for 1 minute at 12000 rpm. 
4. Reaction Setup: To anneal components of 25 μM final duplex concentration<TABLE><TBODY><TR><TH>Component</TH><TH>40 μL Reaction</TH></TR><TR><TD>Nuclease-free water</TD><TD>12 μL</TD></TR><TR><TD>Annealing Buffer (5X)*</TD><TD>8 μL</TD></TR><TR><TD>crRNA Oligo (100 μM)</TD><TD>10 μL</TD></TR><TR><TD>tracrRNA Oligo (100 μM)</TD><TD>10 μL</TD></TR></TBODY></TABLE>
5. Heat at 95℃ for 5 minutes. 
6. Remove from heat and put in 60℃ water, and let it to cool to room temperature. 
7. Store at -20℃. 
*Annealing Buffer (5X): 300mM KCl, 30mM HEPES, 1mM MgCl2, pH7.5
  </DD></DIV><DIV id="section24" class="main"><DT>NEB In vitro digestion of DNA with Cas9 Nuclease</DT><DD>1. Reaction Setup at room temperature: <TABLE><TBODY><TR><TH>Component</TH><TH>30 μL Reaction</TH></TR><TR><TD>Nuclease-free water</TD><TD>20 μL</TD></TR><TR><TD>10X Cas9 Nuclease Reaction Buffer</TD><TD>3 μL</TD></TR><TR><TD>300nM sgRNA</TD><TD>3 μL</TD></TR><TR><TD>1 µM Cas9 Nuclease, S. pyogenes (M0386S)</TD><TD>1 μL</TD></TR><TR><TD colspan="2">Pre-incubate for 10 minutes at 25⁰C</TD></TR><TR><TD>30 nM substrate DNA</TD><TD>3 μL</TD></TR></TBODY></TABLE>
2. Mix thoroughly and pulse-spin in a microfuge. 
3. Incubate at 37°C for 15 minutes. 
4. Add 1 ul of Proteinase K to each sample, Mix thoroughly and pulse-spin in a microfuge. 
5. Incubate at room temperature for 10 minutes. 
6. Proceed with fragment analysis. </DD></DIV><DIV id="section20" class="main"><DT>Test CAMERA Base Editing System</DT><DD>Goal: See that base editor (CAMERA W2.1) works with CAMERA R1. Confirm that mutation rate is linear and test the base editing efficiency. 
96 deep well plate:
Steps: 
1. To start the layout, fill every well expect C5-C8 with 800 μL of sterile water. Fill C5-C8 with 800 μL of fresh LB and appropriate amount of antibiotics (1000X Carb and Spec). Add inducers (200 ng/mL aTc and 0.1 mM IPTG) into C5 and C7. Add overnight cultures into C5-C8 and make sure they are 1:500 dilutions. Secure with 96 well plate film. Allow the plate to shake at 37℃ for 24 hours. 
2. After 24 hours. take out the water in D5-D8 and refill these wells with 800 μL of fresh LB. Dilute the cultures in previous row (C5-C8) 500-fold into fresh LB media containing the corresponding inducers and antibiotics. Miniprep the rest of cultures in the previous row. Secure the plate with film and shake at 37℃ for another 24 hours. 
3. Repeat Step 2 in the following two days. 
4. Sanger sequencing results will then be analyzed by our software tool CrisPy. 
  </DD></DIV><DIV id="section21" class="main"><DT>Base Editing Improvement</DT><DD>Goals: Test if shortening the time between the dilutions would improve the editing rate. 
96 deep well plate: 
Steps: Follow CAMERA protocol. Dilute the cultures every 12 hours. Send for sequencing every 24 hours. 
  </DD></DIV><DIV id="section22" class="main"><DT>CrisPy</DT><DD>Goals: Test the ability of our software tool, CrisPy, for characterizing CRISPR on-target and off-target edits through Sanger sequencing. 
96 deep well plate: 
Steps: Follow CAMERA protocol, however change the antibiotics to CAM and Spec. 
  </DD></DIV></DL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>