Team:Pittsburgh/Results
Results
Overview
CAMERA
Improve
Base Editor
Individual gRNA
Oscillatory gRNA
CrisPy
Overview
Characterize a previous base editor system (CAMERA)
Improve editing rate of the base editor
Assemble our own base editor
Assemble individual gRNA design system
Model our system to visualize readout
Utilize that model to redesign our project in order to get clearer readouts
Assemble a new oscillatory gRNA design system
Build and characterize a software tool to analyze Sanger sequencing readouts and analyze off-target sequences
Build and characterize nCas9 part for the registry
CAMERA
Firstly, we tested a previously characterized CRISPR/Cas9 base editing system obtained from the Addgene repository of Dr. Liu’s lab at Harvard University. The system consisted of a high copy number recording plasmid (R1) and a low copy number writing plasmid with a base editor and one gRNA (W2.1). To assemble this system we transformed the two plasmids into Escherichia coli S1030 obtained from Addgene. This strain of E. coli is a version of the K12 strain with regulatory elements for the 2 inducible promoters PtetO and PLacO encoded into the genome.
The results of the single nucleotide mutations were analyzed through sequencing. The original paper utilized High Throughput Sequencing (HTS) to analyze single nucleotide mutations produced by the base editor. However, due to budget restrictions, we utilized Sanger Sequencing combined with analysis from our software tool CrisPy. For analysis of this software please see our software tool page.
Figure 3 below displays the mutation frequency (the percent of total plasmids edited) over time. This matches the expected results, in that we see a linear relationship of editing over time. However, we did not reach the same amount of overall editing that we demonstrated in the paper. Our data shows an overall editing rate of about 12% editing efficiency after 96 hours, which is similar to the paper who got about 55% editing after four days.
In order to optimize our protocol for testing the CAMERA base editing system, we made two modifications to the previous protocol. We used the corresponding antibiotics for each plasmid to mainly ensure that the low copy number plasmid would not be kicked out by the bacterium. Additionally, we used fresh medium and inducers each time we diluted instead of filling all the wells with medium at the beginning and then diluting into them.
From this new data, Figure 5 and 6 below,  we can see that our optimizations from the previous protocol increased our editings efficiency. Once again this data matches the expected results, in that we see a linear relationship of editing over time. The data shows an overall editing rate of about 40% editing efficiency after 96 hours. Figure 6 demonstrates the control data, which is uninduced editing, there is about 20% baseline editing because of the leaky transcription of PtetO and PLacO which is backed by promoter characterization in the literature.
We performed 2-way ANOVA tests to demonstrate that this data represents a significant editing difference between each time point.
The purpose of these experiments were 2-fold. First, we needed to demonstrate the functionality of the base editor. Secondly, a fundamental part of our project utilizes the linear editing property of the base editor in order to establish a time scale. The ability of our project to establish a time point readout of a stimulus is what separates our project from previous work that utilizes a base editor as a recording mechanism for cellular stimuli.
Improve Base Editing Efficiency
Based on feedback we got from Dr. Seth Childers during our Human Practices we set out to test a theory that increasing the replication rate of our plasmids would increase our base editing rate. From the literature, we discovered that the limiting factor to editing was not the matter of the Cytidine Deaminase modifying the base but rather the release of the complex from the DNA so that it could edit another plasmid. This base editor utilized dCas9 to localize the complex to the correct location for a base change. dCas9 is the catalytically inactivated version of Cas9; meaning that once it binds to the DNA it does not have any function. dCas9 is commonly used by scientists for the specific purpose that once bound it will stay bound. This is called CRISPR interfering or CRISPRi and is used to disable genes or block binding domains such as those for an RNA polymerase. Moreover, due to the fact that it is staying bound, we are not getting proficient use of the enzyme to edit multiple plasmids. Finally, in our research, we discovered that during replication of a plasmid all bound proteins are released as the polymerase replicates the plasmid.
To take advantage of this discovery, we wanted to test if shortening the time between the dilutions would improve the editing rate. By diluting every 12 hours instead of every 24 hours we not allowing the cells to reach stationary phase and therefore are constantly replicating in the exponential phase. We used our CAMERA protocol and adapted it to dilute into fresh medium with antibiotics and inducers every 12 instead of 24 hours.
The difference in mutation frequencies from the two experiments (24 hr dilutions and 12 hr dilutions) was found at each time point. The standard error of the mean was also calculated. Because each difference lies more than two deviations from zero, we are at least 95% confident that the more frequent dilutions increase the mutation rate.
Based on these results we are confident that our hypothesis, keeping the cells in exponential phase would increase the rate of base editing, is true. With this information, all of our future experiments with base editing will involve diluting every 12 hours instead of 24 hours. Alternatively, we can utilize a bioreactor, which will constantly supply the bacteria with new media and therefore keep the cells in exponential phase and improve base editing efficiency.
Base Editor
Due to the fact that the base editor we used did not conform to iGEM’s BioBrick (RFC 10) compatibility, we synthesized DNA from IDT for the CDA and ugi proteins, which were codon optimized to remove BioBrick sites, and utilized a dCas9 from the registry BBa_K1323002 in an effort to assemble our own base editor.
Our first attempt at assembling this plasmid involved using Gibson assembly with 30 bp overhangs between each part and the pSB4A5 low copy number backbone. We attempted varying concentrations, amounts, and ratios of pieces, but none of our experiments produced colonies.
After multiple attempts at assembly using Gibson, we transitioned into using overlap extension PCR. Then we attempted golden gate assembly. Using PCR we added BsaI or BbsI type II restriction enzymes sites to the ends of each of our parts each with a unique 4 bp overhang. These parts would then be ligated together and transformed. Once again we attempted using various concentrations and ratios of pieces; however, we never produced colonies with the correct assembly. We additionally attempted assembling the base editor stepwise, 1 piece at a time, but to no avail.
We learned from our advisors that sometimes plasmids will not successfully transform or be grown up when in a low copy backbone. So we transitioned into assembling it into the pSB1C3 backbone before we transitioned it into its pSB4A5 backbone for final use. Once again we continued to have no success. Finally, we assembled each of the individual parts into a TOPO vector along with the restriction enzymes sites necessary for golden gate assembly to ensure that our sequences from IDT were correct and that our enzymes were successfully cutting the parts before ligation. Assembling in TOPO confirmed that our sequences were correct and our restriction enzymes were cutting correctly. However, our ligations sill never produced any colonies.
After going through numerous attempts of assembly and seeking the advice from various experts about our issues we were unsuccessful in assembling our Base Editor. However, we still had the opportunity the use the base editor plasmid obtained from Addgene for our experiments, but it would not be able to be submitted to iGEM due to violations of the BioBrick RFC10 compatibility.
Individual gRNA System
All recording plasmids for our MC gRNA system were ordered from IDT as blocks and assembled into pSB1C3 using Gibson assembly. Due to the fact that DNA synthesis is problematic in the production of repeating sequences of DNA, we resorted to designing our gBlocks with only 2 repeats and inserting more repeat sequences using type II restriction enzyme assembly (BsaI) and DNA oligos. The following plasmids were all fully assembled and sequence confirmed.
BBa_K2785000 - Multiple Cell Recording (0 Repeat Control)
BBa_K2785001 - Multiple Cell Recording (0 Repeat)
BBa_K2785002 - Multiple Cell Recording (1 Repeat)
BBa_K2785003 - Multiple Cell Recording (2 Repeat)
BBa_K2785004 - Multiple Cell Recording (3 Repeat)
BBa_K2785005 - Multiple Cell Recording (4 Repeat)
BBa_K2785006 - Multiple Cell Recording (5 Repeat)
Our two gRNAs for this system were ordered from IDT as separate gBlocks to be annealed using Gibson assembly. Due to the fact that repeating sequences are difficult to synthesize, IDT was not able to create a sample of gRNA 1 which met their quality control standards. They sent us their best effort attempt of producing it which contained an estimated 80% sample of correct sequence. In order to isolate the correct sequence, we cloned the gRNA into pSB1C3. All samples that we sent for sequencing did not contain the correct sequence of gRNA 1. We repeated this process multiple times, but could not find a correct sequence for gRNA 1. Furthermore, in an attempt to isolate a correct sequence of gRNA 1 we used TOPO cloning to insert our gRNA 1 into the TOPO vector in order to isolate a sequence-confirmed sample of gRNA 1 but had no success finding a correct sequence.
Due to the fact that our base editor could not be assembled, we attempted to incorporate our gRNA into the CAMERA W2.1 base editor. Using PCR we added restriction enzyme sites to W2.1 and our gRNA and attempted assembly using standard restriction enzyme cloning. Unfortunately, our ligations never produced colonies that had an insertion of our gRNAs.
Due to our struggles in cloning the gRNA and the fact that our model demonstrated that this system would not produce the outcomes desired for this system to work we decided to redirect our focus to the oscillatory gRNA system.
Oscillatory gRNA System
Our single cell oscillatory system recording and writing plasmids were ordered from Twist Bioscience as complete plasmids. However once again we had to insert more repeat sequences into the recording plasmid using type II restriction enzyme assembly (BsaI) and DNA oligos as we did with our individual gRNA system. Additionally, we had to remove a spacer sequence from our writing plasmid using type II restriction enzyme assembly (BsaI). This was successfully assembled and sequence-confirmed. Additionally, we successfully modified the CAMERA W2.1 base to remove the gRNA in order to work with this system because our own base editor could not be assembled.
Once all plasmids were assembled and sequence confirmed, we attempted to co-transform all three plasmids into our chassis organism E. coli S1030. However, after multiple attempts, we were unsuccessful in isolated colonies that had successfully up took all three plasmids.
We are continuing to work on co-transforming these plasmids so that we can start this experiment.
CrisPy
Our plasmid containing three off-target gRNA recognition sequences along with 1 target sequence was fully assembled using standard restriction enzyme protocol into the pSB1C3 backbone and sequence confirmed. This plasmid was co-transformed with CAMERA W2.1 base editor plasmid in S1030. We are currently awaiting results and analysis of our on and off-target editing efficiencies, which will be analyzed with our CrisPy software tool. This data will be used to characterize the tools accuracy in identifying and interpreting gRNA binding sequences.
Go to top
Team
Team Members
Collaborations
Attributions
Project
Description
Design
Experiments
Results
Notebook
InterLab
Modeling
Model
Software Tool
Parts
Human Practices
Human Practices
Education & Engagement
Safety
