"
Page
Discussion
View source
History
teams
Log in
 
Team:UTK-Knoxville
From 2013.igem.org
Team:UK-Knoxville - 2013.igem.org
Home
Project
Overall Project
Material & Methods
Parts Origin
Construct
Parts
Submitted Parts ->
Aer
HemAT
RcoM
NarX
TodS
LasR
(EnvZ)
(NtrC)
Primers
Characterization
Discussion
Overview
Background
Application
Notebook
Daily Entries
Material & Methods
Team Meetings
Seminar on DETAILS
oUTreach
Eastman HITES
Coming Soon
Team
Overview
Advisors
Undergraduates
University
Sponsors
Abstract
The major limitation in synthetic biology today is the lack of numerous, well characterized sensors.  Our project aims to provide a reliable scaffold to test potential sensing domains with unknown substrates.  We have created a standard platform to test a range of intracellular and transmembrane domains.  Positive results are reported with green fluorescent protein for easy identification which can be done with high throughput methods like 96 well plates.  We test our platform on sensors with interesting known responses.  The chimera proteins are also useful in creating signals orthogonal to the cell.
Project Description
The 2013 UTK-Knoxville iGem team is creating chimeric biosensors to provide fast and easy reporting of useful ligands, and also creating a proof of concept standard platform with which to characterize new biosensors.
We are constructing chimeric proteins of five well studies sensors.  The five sensors are Aer, HemAT, TodS, NarX, and RcoM.  Aer senses redox states.  HemAT directly senses O2 oxygen.  TodS senses toluene as well as benzene, xylene, chlorotoluene, and other aromatics, most of which are toxic.  NarX senses nitrate, which is used in explosives.  RcoM senses CO, NO, CO2, and NO2, which are all toxic at high levels.  Each of these chimera sensors circuit creates a fluorescent protein to report a positive signal and provide easy confirmation of the presence of the ligand.  These sensors could be used in microfluidic blood, water pollution, and soil tests.
The broader power of this project is that it can provide a way to characterize sensors with unknown stimuli.  Currently around 25,000 sensors have been identified in genomes while only around 100 have been studied to find what stimuli they sense.  Chimera proteins can provide a stable reporting platform to characterize the sensing domains without changing the reporting machinery (kinase, promoter, reporter gene).  High throughput screens can then be done in conjunction with flow cytometry to understand how the sensors interact with an array of chemicals or other stimuli.  This process traditionally takes years to complete.  For example, E. coli has five chemotaxis receptors (sensors for ribose, aspartate, serine, peptides, and redox) and it took about 40 years to characterize these.
Characterizing sensors with unknown stimuli would also shed light onto the desired environment of unculturable microorganisms through the evolved sensors present in their genome.
[ back ]
Retrieved from " http://2013.igem.org/Team:UTK-Knoxville "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
