<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HSiTAIWAN_Project_CYP skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HSiTAIWAN/Project/CYP</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="bot" style="position:fixed;bottom:5px;left:5px;z-index:99999;"><DIV class="drag" id="Drherb" style="bottom:5px;left:5px;"><DIV id="convblank"><DIV id="qnaClose">X</DIV><DIV id="dialog"><DIV id="box" style="overflow:hidden;height:230px;width:610px;"><DIV style="width:610px;"><DIV class="Herbsay"><P>　Oh hello there,I am Charlie,a trusty member of the &quot;Herb Tasters&quot; and also the brainiest E. coli in the colony.
       
       　I know all the secrets of Chinese herbs and their magical healing powers.
       
       　If you are up to a challenge,find me at team HSiTW at the jamboree.
       
       　I am the one in a straw hat,showing them pearls.I will be waiting.
       
       　Muhahahahaha</P></DIV><DIV class="Herbsay"><P>　Hi there! My name is Nu Zhen Chi. This is how I look like.
       
       　　Take a closer look; guess which part of me is used as medicine?
       (1) the root (2) the stem
       (3) the leaf
       (4) the seed

	</P></DIV><DIV class="Herbsay"><P><B>Ans.</B>(4) the seed
       <B>Name:</B> 女貞子 (Nu Zhen Chi)
       <B>Botanical Name:</B> Ligustrum lucidum Aiton

       　I can treat people who are yin deficient, and liver problems that cause dizziness,cataract of the eyes,
       lower back pain, premature graying of the hair and tinnitus.
	</P></DIV><DIV class="Herbsay"><P>　Hello! My name is Chuan Xiong. This is how I look like. 
       
       　Make a guess, which part of me is used as medicine?
       (1) the root (2) the stem
       (3) the leaf
       (4) the seed

	</P></DIV><DIV class="Herbsay"><P><B>Ans.</B>(1) the root
       <B>Name:</B> 川芎 (Chuan Xiong)
       <B>Botanical Name:</B> Ligusticum chuanxiong Hort

       　I help with blood regulation to prevent relevant to blood stasis and non-stop bleeding.I can also strengthen your qi circulation.
       
       　In addition, I relieve you of physical pain,
         such as headaches, abdominal aches, chest pain, and muscle pain.
       
       　Finally, I free the ladies of menstrual disorders and amenorrhea.
	</P></DIV><DIV class="Herbsay"><P>　What’s up? My name is Dang Gui. I can: 
       (1) stop coughing (2) regulate mense
       (3) reduce internal heat

	</P></DIV><DIV class="Herbsay"><P><B>Ans.</B>(2) Regulate mense
       <B>Name:</B> 當歸 (Dang Gui)
       <B>Botanical Name:</B> Angelica sinensis (Oliv.) Diels

       　I can remove blood stasis and clots, so I am usually used to regulate menses,lubricate intestines to correct constipation, reduce swelling, expel pus.

	</P></DIV><DIV class="Herbsay"><P><B>　Reference</B>
       　臧堃堂 (2005) 中華材輕百科-現代版本草綱目，山岳文化出版社，台北
       <A href="http://brion.org.tw/index.php" target="_blank">Non-Profit Organization Brion Research Institute of Taiwan.</A><A href="https://www.jadeinstitute.com/index.php" target="_blank">Chinese Herb Gallery. Jade Institute</A><A href="http://www.shen-nong.com/eng/front/index.html" target="_blank">Herbal Glossary. Shen-Nong- Chinese Traditional Medicine</A><B>　Acknowledgement</B>
       　Thank you for Non-Profit Organization Brion Resaerch Institute of Taiwan that provide us Chinese herbs and photos.
	</P></DIV></DIV></DIV></DIV><DIV id="qnaNextBt" onclick="slidebar(1,'box');">Next ⇒</DIV><DIV id="qnaPrevBt" onclick="slidebar(-1,'box');">⇐ Prev</DIV></DIV><DIV id="convopen">...</DIV></DIV></DIV><DIV class="myContent"><DIV id="contentList"><LI class="listOne"><P id="list1" onclick="window.scrollTo(0,document.getElementById('FirstTitle').offsetTop-50)">Biosensor of Aflatoxin</P></LI><LI class="listTwo" id="listfirst1" onclick="window.scrollTo(0,document.getElementById('1-1').offsetTop-50)">Introduction</LI><LI class="listTwo" id="listfirst2" onclick="window.scrollTo(0,document.getElementById('1-2').offsetTop-50)">Regulation</LI><LI class="listTwo" id="listfirst3" onclick="window.scrollTo(0,document.getElementById('1-3').offsetTop-50)">Circuit Design and Parts</LI><LI class="listTwo" id="listfirst4" onclick="window.scrollTo(0,document.getElementById('1-4').offsetTop-50)">Results</LI></DIV><DIV id="mainContent"><P class="titleOne" id="FirstTitle">CYP: Biosensor of Aflatoxin</P><P class="titleTwo" id="1-1">Introduction</P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">1. Aflatoxin</P><P class="contentP">Aflatoxins are a group of natural mycotoxins that are secondary metabolites produced from different species of fungi, especially Aspergillus species<SUP>1</SUP>.  At least 14 different aflatoxins are produced in nature. Aflatoxin B1 is considered the most toxic and is produced by both Aspergillus flavus and Aspergillus parasiticus.</P><P class="contentP">Figure 1. Main Types of Aflatoxins<SUP>2</SUP></P><P class="contentP">Aspergillus flavus and A. parasiticus grow very well at 28-30°C and 25-35°C.  Aflatoxin commonly contaminate cereals and grains such as rice, maize, and many others during the storage and poor processing conditions.  Afs are chemically stable during processing and storage, the optimum temperature is difficult factor to avoid AF formation during production, harvesting, transport and storage, which is not usually possibe in practice.  In addition, Afs are stable at high temperature with little damage occuring during cooking.  Aflatoxin-food contamination is a global problem especially in the tropical and subtropical regions of the world where warm temperatures and humidity<SUP>3</SUP>.</P><P class="contentP">According to the US FDA, Alfatoxin poisoning is relatively rare but dangerous, leading to risks of liver cancer, mental impairments, digestive reactions, coma, hemorrhages and malabsorption .  Long term exposure leads to food allergies, autoimmune disease, inflammation affecting heart, organ damange such as liver and kidneys, growth and development impairment, and so on</P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">2. Cytochromes P450 and aflatoxins metabolism</P><P class="contentP">The process of metabolism and excretion of chemical substances from the body involves two main phases: Phase I metabolism involves the addition of a small polar group containing both positive and negative charges that are added to the xenobiotics by the process of oxidation, reduction, acetylation and hydrolysis and render it harmless.  Phase II metabolism involves sulfate, glucuronide, glutathione and amino acid conjugation re actions.  Cytochrome P450 enzymes (CYP) play critical roles in the first phase of AFB1 metabolism.  CYP are primarily membrane-associated proteins that metabolize thousands of endogenous and exogenous chemicals.  Cytochrome P450 enzymes are present in most tissues of the body, and play important roles in metabolizing toxic compounds, including drugs and products of endogenous metabolism, principally in the liver<SUP>5</SUP>. Among human CYPs, CYP1A2 and CYP3A4 are the major AFB1-metabolizing enzymes in human liver.  Human lung is also able to activate AFB1, and study has found that CYP2A13  activates AFB1 to form AFB1-N7 -guanine adducts<SUP>6</SUP>.</P><P class="contentP">The major carcinogenic and mutagenic metabolites of AFB1 are AFB1-8,9-epoxide and AFM1-8,9-epoxide, although the latter is relatively less active in mutagenesis test.  The figure shows that CYP isoenzymes metabolize AFB1 to the 8,9-epoxide, which reacts with DNA. While a number of products are formed, the initial major adduct is AFB1-N7-Gua. This adduct has a destabilized glycosidic bond and depurinates to the AP site shown. Alternatively, the primary adduct can suffer opening of its imidazole ring, giving rise to the chemically and biologically stable formamidopyrimidine adduct, AFB1-FAPY.  These three DNA lesions, AFB1-N7-Gua, AFB1-FAPY and the AP sites, are candidate precursors to the mutations induced by AFB1<SUP>7</SUP>.</P><P class="contentP">Figure 2. Pathway of metabolic activation leading to DNA adduct formation by AFB1</P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">3. SOS response</P><P class="contentP">The SOS response is a global response to DNA damage in which the cell cycle is arrested and DNA repair and mutagenesis are induced.  SOS induction was first observed in cells where the replication fork encounters DNA lesions which it tries to replicate<SUP>8</SUP>. In Escherichia coli, the SOS system consists of more than 40 genes.  Two proteins play key roles in the regulation of the SOS response: a repressor named LexA and an inducer, the RecA filament.</P><P class="contentP">RecA is a protein in nearly all bacteria and all organisms.   Following a break on a single-stranded DNA (ssDNA), the exonuclease RecJ cleaves single-stranded DNA.  The single-strand DNA-binding protein (SSB) binds to the daughter strand gap, preserving the integrity of the DNA.  Recombinase mediator proteins RecF, RecO, and RecR act together to promote loading of RecA onto single stranded DNA.  Rec A binds single-stranded DNA (ssDNA), forming a nucleoprotein filament that promotes LexA cleavage (thereby inducing the SOS response).</P><P class="contentP">Figure 3 The FOR pathway and RecA filament in the process of SOS response. (This figure is from Naofumi Handa, et al, 2009)<SUP>9</SUP></P><P class="contentP">Repressor LexA or LexA is a transcriptional repressor that represses SOS response genes.  Normally, the LexA repressor binds to a specific sequence—the SOS box, present in the promoter region of SOS genes—and prevents their expression<SUP>10</SUP>. When the cell senses the presence of an increased level of DNA damage, DNA-damaging agents induce the formation of DNA single-strand gaps, a nucleoprotein complex—the RecA filament— induces the LexA cleavage reaction and induce the SOS response.</P><P class="contentP">Figure 4. RecA_filament_Activates_the_Lex_A_repressor_in_the_Process_SOS_Response. (This figure is from is Dan I. Andersson and Diarmaid Hughes, 2014)<SUP>11</SUP></P><P class="titleTwo" id="1-2">Regulation</P><P class="contentP"> Our government identifies 37 kinds of Chinese herb medicine for aflatoxin detection.  The maximum would for Aflatoxin B1 is 5 ppb(parts per billion) and total aflatoxins (B1, B2, G1, and G2) is 10 ppb.   On October 1, 2016, our government expanded the scope of aflatoxin detection to all Chinese herb medicine.  </P><P class="titleTwo" id="1-3">Circuit Design and Parts</P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">1. Idea of aflatoxin biosensor</P><P class="contentP">The basic design of aflatoxin biosensor consists of two parts: (1) CYP2A13 transcription and (2) SOS system response with GFP reporter gene.   The first part produces CYP2A13, CYP1A2 or CYP3A4 that can metabolizes Aflatoxin B1 to aflatoxin B1 exo-8,9-epoxide(AFBO).      </P><P class="contentP"> Figure 5. Cytochromes CYP2A13 and Aflatoxin Metabolism</P><P class="contentP">Aflatoxin B1 exo-8,9-epoxide will cause DNA damage.  The single-strand DNA-binding protein (SSB) binds to the DNA, preserving the integrity of the DNA.  RecF, RecO, and RecR act together to form a complex which promotes loading of RecA onto single stranded DNA.  RecA binds single-stranded DNA (ssDNA), forming a nucleoprotein filament that promotes LexA cleavage and thereby inducing the SOS response.     </P><P class="contentP"> Figure 6. FOR pathway, RecA and LexA repressor in the process of SOS response.</P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">2.Circuits and Parts</P><P class="titleTwo" style="color:#538135 ;font-size: 18px;">New Single Parts</P><P class="contentP">We submitted 5 new single parts related to aflatoxin metabolism and the FOR pathway related to SOS response.</P><TABLE class=""><TBODY><TR><TH>Part</TH><TH>Type</TH><TH>Description</TH></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961000">BBa_K1961000</A></TD><TD>coding</TD><TD><B>Cytochrome P450, family 2, subfamily A, polypeptide 13(CYP2A13)</B><B>To metablize aflatoxin and related mycotoxin.</B> 

            Like CYP3A4, CYP2A13 is a member of the cytochrome P450 proteins, which oxidizes a variety of structurally unrelated compounds, including steroids, fatty acids, and xenobiotics.  CYP3A4, CYP3A5 and CYP1A2 are mainly in liver.  CYP2A13 is in lung.
</TD></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961001">BBa_K1961001</A></TD><TD>coding</TD><TD><B>RecF protein-SOS mediator protein</B><B>To enhance DNA replication and repair</B>

            RecF protein is involved in the RecFOR pathway for DNA metabolism; it is required for DNA replication and normal SOS inducibility. 
</TD></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961002">BBa_K1961002</A></TD><TD>coding</TD><TD><B>RecO protein-SOS mediator protein</B><B>To enhance DNA replication and repair</B>

            The RecO protein is involved in the RecFOR pathway for DNA metabolism; it is required for DNA replication and normal SOS inducibility.
</TD></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961003">BBa_K1961003</A></TD><TD>coding</TD><TD><B>RecR protein-SOS mediator protein</B><B>To enhance DNA replication and repair</B>

            The RecR protein is involved in the RecFOR pathway for DNA metabolism; it is required for DNA replication and normal SOS nducibility.</TD></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961007">BBa_K1961007</A></TD><TD>coding</TD><TD><B>Cytochrome P450, family 1, subfamily A, polypeptide 2(CYP1A2)</B><B>To metablize aflatoxin and related mycotoxin.</B> 

            Like CYP3A4 and CYP2A13, CYP1A2 is a member of the cytochrome P450 proteins, which oxidizes a variety of structurally unrelated compounds, including steroids, fatty acids, and xenobiotics.  CYP3A4, CYP3A5 and CYP1A2 are mainly in liver.
</TD></TR></TBODY></TABLE><P class="titleTwo" style="color:#538135 ;font-size: 18px;">New Composite Parts</P><P class="contentP">We submitted 2 new composite parts related to the FOR pathway related to SOS response.</P><TABLE class=""><TBODY><TR><TH>Part</TH><TH>Type</TH><TH>Description</TH></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961008">BBa_K1961008</A></TD><TD>generator </TD><TD><B>Generator of RecF protein-SOS mediator protein</B><B>To enhance DNA replication and repair</B>

            RecF protein is involved in the RecFOR pathway for DNA metabolism; it is required for DNA replication and normal SOS inducibility. 
</TD></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1961009">BBa_K1961009</A></TD><TD>generator</TD><TD><B>Generator of RecO protein-SOS mediator protein</B><B>To enhance DNA replication and repair</B>

            The RecO protein is involved in the RecFOR pathway for DNA metabolism; it is required for DNA replication and normal SOS inducibility.
</TD></TR></TBODY></TABLE><P class="titleTwo" style="color:#538135 ;font-size: 18px;">Test the circuit of SOS response(BBa_K079050)  </P><P class="contentP">We used the Aflatoxin detector (BBa_K1613007) to metabolize aflatoxin. We used the SOS part(BBa_K079050, iGEM 2008 Bologna) to test the SOS response induced by actived-Rec A protein, which facilitates the cleavage of LexA transcription repressor on SOS response. </P><TABLE class=""><TBODY><TR><TH>Part</TH><TH>Type</TH><TH>Description</TH></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K1613007">BBa_K1613007</A></TD><TD>device</TD><TD>CYP1A2 aflatoxin metabolism

            A composite part that contain CYP1A2 that can matabolize aflatoxin 

            J23119+B0034+K1613004+B0010+B0012</TD></TR><TR class="" id=""><TD><A href="http://parts.igem.org/Part:BBa_K079050">BBa_K079050</A></TD><TD>device</TD><TD><B>DNA damage reporter</B>

            GFP reporter protein under the control of the J23100 constitutive promoter and LexA 2 operator

            J23100+K079040+B0034+J04031+B0010+B0012</TD></TR></TBODY></TABLE><P class="titleTwo" style="color:#538135 ;font-size: 18px;">Design of Composite Part for Modelling      </P><P class="contentP">Because CYP2A13 is a large protein from human lung, we would like to control the CYP2A13 generation by measuring the expression of GFP downstream.  We designed an autoregulated CYP2A13 generator for modeling the control of CYP2A13 generation .  This composite part consists of the TetR and CYP2A13 generators, both controlled by PTet, which has TetR binding sites.  TetRs bind onto PTet and repress the transcription of TetR and CYP2A13.  We used tetracycline to induce the gene expression of CYP2A13.     </P><P class="contentP"> Figure 7 Autoregulation of CYP2A13 generator.</P><P class="titleTwo" id="1-4">Results</P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">1.Functional Test of E. Coli with SOS plasmid</P><P class="contentP">It is known that hydrogen peroxide is an important reactive oxygen.  It can impose on cell an oxidative stress, resulting in cell damage.  Literature showed that in E. coli, low concentration of hydrogen peroxide resulted in SOS gene induction.<SUP>12,13</SUP></P><P class="contentP">Objective:use H2O2 to produce SSDNA to test the SOS function of E. coli  </P><P class="contentP">Material: E. Coli with SOS plasmid (BBa_K079050 , 2008 Bologna) in SOC under 16  °C</P><P class="contentP">Independent variable1: H2O2 concentration (0mM, 1mM, 1.25mM, 1.50mM, 1.75mM)</P><P class="contentP">Independent variable 2: time (0, 5, 10, 15 minutes) </P><P class="contentP">Dependent variable: fluorescent/od </P><P class="titleTwo" style="color:#0070c0;font-size: 20px;">Results:</P><P class="contentP"> Figure 8 Fluorescence/OD of E. coli with SOS genes in different concentrations of H2O2 solution</P><P class="contentP">Our study used different E. coli strain to test the H2O2 concentration for SOS induction.  Our results are somewhat a little bit different from that of study done by Imay and Linn(1987).  Their study showed the SOS killing rate was stable after 6 minutes, and the SOS killing borderline was 1.25mM.   In order to prolong the E. coli survival time, we lowered the temperature.  Our study showed SOS killing time was about 15 minutes and the SOS killing borderline was not 1.25mM, at least 1.75mM in our study result. </P><P class="contentP">Figure 9 The survival fraction of E.coli in different concentrations of H2O2 done by  Imlay and Linn 1987</P><P class="titleTwo" id="1-4">Reference</P><P class="contentP">[1]  Godfrey S. Bbosa, David Kitya, John Odda, Jasper Ogwal-Okeng.  Aflatoxins metabolism, effects on epigenetic mechanisms and their role in carcinogenesis.  Health 2013. 5(10A): 14-34</P><P class="contentP">[2]  Mohamed E. Zain. Impact of mycotoxins on humans and animals.  Journal of Saudi Chemical Society 2011. 15: 129-144 </P><P class="contentP">[3]  Kiran Lakkireddy, Kasturi K., Sambasiva Rao K.R.S. Aflatoxins in Food and Feed: The Science of Safe Food. Research &amp; Reviews: Journal of Food Science and Technology  2014. 3(2): 6-11</P><P class="contentP">[4]  BBB – Aflatoxins. Bad Bug Book: Foodborne Pathogenic Microorganisms and Natural Toxins Handbook Aflatoxins. U.S. Food and Drug Administration. http://www.fda.gov/Food/FoodborneIllnessContaminants/CausesOfIllnessBadBugBook/ucm071020.htm </P><P class="contentP">[5]  Cytochrome P450. Wikipedia. https://en.wikipedia.org/wiki/Cytochrome_P450   </P><P class="contentP">[6]  Xiao-Yang He, Lili Tang, Shou-Lin Wang, Qing-Song Cai, Jia-Sheng Wang and Jun-Yan Hong. Efﬁcient activation of aﬂatoxin B1 by cytochrome P450 2A13, an enzyme predominantly expressed in human respiratory tract. International Journal of Cancer (2006). 118: 2665-2671  </P><P class="contentP">[7]  Maryann E. Smela, Sophie S. Currier, Elisabeth A. Bailey, John M. Essigmann. The chemistry and biology of aflatoxin B1 from mutational spectrometry to carcinogenesis. Carcinogenesis 2001. 22(4): 535-545  </P><P class="contentP">[8]  Radman M.  SOS repair hypothesis: Phenomenology of an inducible DNA repair which is accompanied by mutagenesis. Basic Life Sci 5A(1975): 355– 367.  </P><P class="contentP">[9]  Naofumi Handa, Katsumi Morimatsu, Susan T. Lovett, Stephen C. Kowalczykowski. Reconstitution of initial steps of dsDNA break repair by the RecF pathway of E. coli. GENES &amp; DEVELOPMENT (2009). 23(10):1234–1245   </P><P class="contentP">[10] Repressor lexA. Wikipedia. https://en.wikipedia.org/wiki/Repressor_lexA   </P><P class="contentP">[11] Dan I. Andersson, Diarmaid Hughes. Microbiological effects of sublethal levels of antibiotics.  Nature Reviews Microbiology 2014. 12: 465-478   </P><P class="contentP">[12] Imlay, JA and Linn, Stuart. Mutagenesis and Stress Response Induced in Escherichia coli by Hydrogen Peroxide.  Journal of Bacteriology 1987. 169(7)2967-2976   </P><P class="contentP">[13] Imlay, JA and Stuart Linn. Bimodal Pattern of Killing of DNA-Repair-Defective or Anoxically Growth of Escherichia coli by Hydrogen Peroxide.  Journal of Bacteriology 1986. 166(2): 519-527   </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>