Team:HSiTAIWAN/Model
X
Oh hello there,I am Charlie,a trusty member of the "Herb Tasters" and also the brainiest E. coli in the colony.
I know all the secrets of Chinese herbs and their magical healing powers.
If you are up to a challenge,find me at team HSiTW at the jamboree.
I am the one in a straw hat,showing them pearls.I will be waiting.
Muhahahahaha
Hi there! My name is Nu Zhen Chi. This is how I look like.
Take a closer look; guess which part of me is used as medicine?
(1) the root
(2) the stem
(3) the leaf
(4) the seed
Ans.(4) the seed
Name: 女貞子 (Nu Zhen Chi)
Botanical Name: Ligustrum lucidum Aiton
I can treat people who are yin deficient, and liver problems that cause dizziness,cataract of the eyes,
lower back pain, premature graying of the hair and tinnitus.
Hello! My name is Chuan Xiong. This is how I look like.
Make a guess, which part of me is used as medicine?
(1) the root
(2) the stem
(3) the leaf
(4) the seed
Ans.(1) the root
Name: 川芎 (Chuan Xiong)
Botanical Name: Ligusticum chuanxiong Hort
I help with blood regulation to prevent relevant to blood stasis and non-stop bleeding.I can also strengthen your qi circulation.
In addition, I relieve you of physical pain,          such as headaches, abdominal aches, chest pain, and muscle pain.
Finally, I free the ladies of menstrual disorders and amenorrhea.
What’s up? My name is Dang Gui. I can:
(1) stop coughing
(2) regulate mense
(3) reduce internal heat
Ans.(2) Regulate mense
Name: 當歸 (Dang Gui)
Botanical Name: Angelica sinensis (Oliv.) Diels
I can remove blood stasis and clots, so I am usually used to regulate menses,lubricate intestines to correct constipation, reduce swelling, expel pus.
Reference
臧堃堂 (2005) 中華材輕百科-現代版本草綱目，山岳文化出版社，台北
Non-Profit Organization Brion Research Institute of Taiwan.
Chinese Herb Gallery. Jade Institute
Herbal Glossary. Shen-Nong- Chinese Traditional Medicine
Acknowledgement
Thank you for Non-Profit Organization Brion Resaerch Institute of Taiwan that provide us Chinese herbs and photos.
Next ⇒
⇐ Prev
...
Modeling of Lead and CYP Biosensor
Introduction
Regulation
Circuit Design
Simulation
Reference
Modeling of Lead and CYP Biosensor
Introduction
The brilliant advances in genetic engineering technologies based on proteins and non-coding small RNAs are enabling the systematic biological functions for both prokaryote and eukaryote. In this study, we utilize the mean field theory and the 4th Runge-Kutta method to simulate the engineering synthetic genetic circuits in Escherichia coli (E. coli), including the small RNA (sRNA) regulations and autoregulation circuits. For sRNA level in E. coli, the near simultaneously regulations that decrease (or increase) the target mRNA disable (enable) the target protein function. Therefore, the phase space for induction-regulation plays a key role and should be thoroughly investigated by simulations. In comparison with the sRNA level, the simulations and experimental methods based on the autoregulation circuit (Ptet-TetR) are designed to quantitatively describe the reporter protein or fluorescence response. The Hill coefficient is also checked to disclose the intrinsic properties of the autoregulation circuits in E. coli.
Regulation
The introduction for genetic circuits
There are three types of commonly used genetic regulations:
Negative regulation —Protein  (repressor) inhibits transcription (Ex. LacI, TetR protein). Inducer– binds to repressor, alters form, reduces affinity for target, allows expression of gene. Sometimes, small molecule required for repressor activity.
Positive regulation —Activator protein increases transcription rate. Generally bound to a smaller signal molecule. (Ex. XylR protein activates Pu promoter).
sRNA-mediated gene silencing -(sRNA) are small (50-250 nucleotide) non-coding RNA molecules produced by bacteria; they are highly structured and contain several stem-loops. sRNAs can either bind to protein targets, and modify the function of the bound protein, or bind to mRNA targets and regulate gene expression. (Ex. RyhB and sodB)
Figure 1. Negative regulation: The TetR protein/Tc in ref [1].
Figure 2. sRNA-mediated regulation: The RyhB-sodB RNA. RyhB is a small RNA that regulates iron-using proteins. It performs sRNA-mediated regulation by binding to the target mRNA sodB.
According to the negative regulation method and sRNA mediated circuit, we are able to design three types of regulation circuits, like the constitutive expression in Fig. 3, autoregulation expression in Fig. 4 and sRNA-mediated regulation in Fig. 5.
Figure 3. The Constitutive expression: The Pcon-tetR. O1 and O2 symbolizes operators, while Pconst represents promoter that constitutive expresses the proteins.
Figure 4. The autoregulation switch: The Ptet-tetR system. In fact, autoregulation includes two types of auto-repression and induction. Hereafter, we use the autoregulation to represent auto-repression for convenience.
Figure 5. The sRNA-mediated regulation circuit
There are more than fifty types of small RNA in E. coli, ref [2], in which sRNA performs “RNA silencing” by binding to target mRNA and causing its bio-function as shown in Table 1.
Table 1. The famous sRNA in E. coli
Circuit Design of Lead Biosensor
As shown in previous section of Lead project, we design the Constitutive PbrR circuit in Fig. 6 and Autoregulation PbrR circuit in Fig. 7.
Figure 6. Constitutive PbrR is used to measure the background expression of PbrR in E. Coli without any inducer
Figure 7. Autoregulation PbrR circuit: the generation of PbrR measured by GFP expression downstream and is under the control of TetR and promoter PTet
The introduction for Hill coefficient
According to the analysis of Hill coefficient, we are able to get the intrinsic property of genetic circuit. Therefore, we discuss the origin of Hill coefficient in Fig. 8, equation in Equation 1 and response from Hill n in Fig. 9 in the following.
The dissociation constant K
Figure 8. The cartoon picture illustrating the dissociation constant K and origin of Hill coefficient n where n = 2. The TetR proteins are used to describe how to bind and dissociate with inducers.
The Hill coefficient n determine the response sharpness of [Reporter] vs. [Inducer].
Equation 1: The Hill equation: [R] as the concentration of reporter protein, [I] as the concentration of inducer and KM as the concentration of inducer when [R] reaches to the half of [Rmax]
Figure 9. Based on Equation 1, we illustrate the various response of [R] vs. [I] in which the various responses are determined by the different Hill coefficient n. This number n implies the different types of cooperative networking. As the value of n increases (n>1), the variation slope of the graph increases.
Modeling method
The sRNA based genetic circuits
We also design the sRNA-mediated genetic circuit to control the expression of reporter CYP. With the more precise control, we are able to study the bio-function of CYP clearly. The designed circuits are illustrated in Fig. 10 and 11.
Figure 10. The complex designed regulation: The RyhB-sodB : w/o sodB and reporter as the CYP protein (cytochrome P450 family).
Figure 11. The complex designed regulation: The RyhB-sodB : w/ sodB and reporter as the CYP protein.
Figures 10-11 demonstrates the difference between circuits with and without RyhB-sodB regulation. The reporter as the CYP protein in which its bio-function is to metabolize Aflatoxin B1 to aflatoxin B1 exo-8,9-epoxide(AFBO), as shown in the previous section of CYP project. CYP 1A1 is shown in Fig. 12 and  CYP 1A2 shown in Fig. 13
Figure 12. The cytochrome P450 family: CYP 1A1 (RCSB number: 4i8v)
Figure 13. The cytochrome P450 family: CYP 1A2 (RCSB number: 2hi4).
Equation 2: The mean field theory is used to describe the sRNA-mediated regulation circuit. With this circuit, it is easy to control the express of CYP protein.
Equation 3: The steady state analysis for sRNA-mediated regulation circuit.
For both system of Ptet-tetR and PLac-LacI, we run the simulation for the gR function for various [inducer] with different Hill coefficient m and n, where the dissociation constant as KpLac-LacI = 500 nM [11], KLacI-IPTG = 30 uM [11], KPtet-TetR = 50 nM [10] and KTetR-aTc = 15 nM [12]. For PLac-LacI and Ptet-TetR, we consider both of the inducer Hill coefficient m=2, while binding to DNA is set to n=2 or 3. In Fig. 14(b) and 14(d), we will get the sharp response in comparison with 14(a) and 14(b). The higher n and sharper response.
Figure 14(a). The gR function for different Hill coefficient (m=2, n=2) with PLac-LacI.
Figure 14(b). The gR function for different Hill coefficient (m=2, n=3) with PLac-LacI.
Figure 14(c). The gR function for different Hill coefficient (m=2, n=2) with Ptet-TetR.
Figure 14(d). The gR function for different Hill coefficient (m=2, n=3) with Ptet-TetR.
Equation 4: The approximation approach based on the certain condition of transcription rate
Constitutive circuit
Figure 15(a). The Constitutive circuit and the related rate equations
A regular constitutive circuit contains multiple inducers and repressor with unidirectional regulations.
Autoregulation circuit
Figure 15(b). The Autoregulation circuit and the related rate equations
The repressor of an autoregulation circuits can not only react with other reporters but also regulate itself.
Equation 5: The fold analysis of Constitutive Circuit
Equation 6: The fold analysis of Autoregulation Circuit
Equation 7: The fold analysis of Autoregulatory and Constitutive circuits When [I] goes to 0, solve numerically.
Equation 5-7 shows how regulation function gR is affected by the change of inducer I (Omega represents the fold of circuits.)
Simulation by Runge-Kutta fourth-order
Considering the rate equations in Equation 5, we can write down the trajectory y=f(x) of the linear ordinary differential equation as y'=F(x,y) by using Runge-Kutta fourth-order method in Equation 8. The simulation is to use the x0 (y0=f(x0)) to predict the xn by the set of rate equations in which we can use the 4 kinds of slope (F(x,y):m1, m2, m3, m4) to get the mean slope in Fig.16.
Equation 8: The linear ordinary differential equation y'=F(x,y) using Runge-Kutta fourth-order method
Figure 16. The 4 kinds of slope (m1, m2, m3, m4) are used to get the mean slpoe.
According to the trajectory of linear ordinary differential equation y'=F(x,y), we are able to run the phase diagram that is helpful for researchers to uncover the intrinsic properties of genetic circuits, illustrated in Fig. 17(a) and (b). All the source codes are described in detail in the section of Software. (code)
Figure 17(a). The 2-D phase space for sRNA mediated circuit
Figure 17(b). The 3-D phase space for sRNA mediated circuit
In Figures 18 and 19, we mimic the autoregulatory and constitutive circuits to estimate the fold change with different KR/KE ratio. It is easy to find out that the fold change of autoregulatory circuit is slower than that of  constitutive circuits because of the auto-repression effect. This is reason why a lot of genetic circuits belong to the autoregulation in E. coli.
Figure 18. The parameter relationship for circuits
Figure 19. Fold relationship for autoregulatory and constitutive circuits
Reference
[ 1] Dan S. Bolintineanu, Katherine Volzing, Victor Vivcharuk, Abdallah Sayyed-Ahmad, Poonam Srivastava, and Yiannis N. Kaznessis, Investigation of Changes in Tetracycline Repressor Binding upon Mutations in the Tetracycline Operator, J. Chem. Eng. Data,  59 (2014), 3167–3176.
[ 2] Nellen, Wolfgang, Hammann, Christian, Small RNAs: Analysis and Regulatory Functions. Springer-Verlag Berlin Heidelberg (2006).
[ 3] E. Levine, Z. Zhang, T. Kuhlman, T. Hwa, Quantitative Characteristics of Gene Regulation Mediated by small RNA, PLoS Biol 5 (2007) 1998-2010.
[ 4] David Braun et al. Parameter estimation for two synthetic gene networks: A case study. ICASSP (2005) 769-772.
[ 5] Timothy S. Gardner Charles R. Cantor & James J. Collins, Construction of a genetic toggle switch in Escherichia coli, Nature 403 (2000) 339-342.
[ 6] O. Scholz, P. Schubert, M. Kintrup and W. Hillen, Tet repressor induction without Mg2+, Biochemistry 39 (2000) 10914–10920.
[ 7] Peter Orth et al., Conformational changes of the Tet repressor induced by tetracycline trapping, J. Mol. Biol. 279 (1998) 439-447.
[ 8] Thomas A Geissmann and Daniele Touati, Hfq, a new chaperoning role: binding tomessenger RNA determines access for small RNA regulator, The EMBO J. 23 (2004) 396–405.
[ 9] Nicolae RaduZabet, Negative feedback and physical limits of genes, J. of Theoretical Biology 284 (2011) 82–91.
[10] Braun D, S. Basu,R. Weiss, Parameter Estimation for Two Synthetic Gene Networks: A Case Study, ICASSP 5 (2005) 769-772.
[11] Timothy S. Gardner, Charles R. Cantor, James J. Collins, Construction of a genetic toggle switch in Escherichia coli, Nature 403 (2000) 339-342.
[12] Scholz O, Schubert P, Kintrup M, Hillen W., Tet repressor induction without Mg2+, Biochemistry 39 (2000) 10914-10920.
[13] Harley H. McAdams and Adam Arkin, Gene regulation: Towards a circuit engineering discipline, Current Biology 10 (2000) R318–R320.
[14] JOSHUA L. CHERRY AND FREDERICK R. ADLER, How to make a Biological Switch, J. theor. Biol. 203 (2000) 117-133.
[15] Ellen M. Judd, Michael T. Laub, and Harley H. McAdams, Toggles and oscillators: new genetic circuit designs, BioEssays 22 (2000) 507-509.
[16] Michael B. Elowitz, Stanislas Leibler, A synthetic oscillatory network of transcriptional regulators, NATURE 403 (2000) 335-338.
[17] Timothy S. Gardner, Charles R. Cantor & James J. Collins, Construction of agenetic toggle switch in Escherichia coli, NATURE 403 (2000) 339-342.
[18] Attila Becskei & Luis Serrano, Engineering stability in gene networks by autoregulation, NATURE 405 (2000) 590-593.
[19] Karen M. Wassarman, Small RNAs in Bacteria: Diverse Regulators of Gene Expression in Response to Environmental Changes, Cell 109 (2002) 141-144.
[20] Ralf Metzler, Peter G. Wolynes, Number ﬂuctuations and the threshold model of kinetic switches, Chemical Physics 284 (2002) 469-479.
[21] Shai S. Shen-Orr, Ron Milo, Shmoolik Mangan, Uri Alon, Network motifs in the transcriptional regulation network of Escherichia coli, Nature Genetics 31 (2002) 64-68.
[22] Raik Gru¨nberg and Luis Serrano, Strategies for protein synthetic biology, Nucleic Acids Research 38 (2010) 2663-2675.
[23] Nagarajan Nandagopal , Synthetic Biology: Integrated Gene Circuits, SCIENCE 333 (2011) 1244-1248.
[24] Josef Panek and et al., The suboptimal structures find the optimal RNAs: homology search for bacterial non-coding RNAs using suboptimal RNA structures, Nucleic Acids Research 39 (2011) 3418-3426.
[25] Jacob Beal, Ting Lu, Ron Weiss, Automatic Compilation from High-Level Biologically Oriented Programming Language to Genetic Regulatory Networks, PLoS ONE 6 (2011) e22490.
[26] Adrian L. Slusarczy, Allen Lin and Ron Weiss, Foundations for the design and implementation of synthetic genetic circuits, NATURE 13 (2012) 406-420.
[27] Huseyin Tas, Cac T. Nguyen, Ravish Patel, Neil H. Kim, Thomas E. Kuhlman, An Integrated System for Precise Genome Modification in Escherichia coli, PLOS ONE (2015) 1-19.
