<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HSiTAIWAN_Notebook_FA_Pb skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HSiTAIWAN/Notebook/FA/Pb</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="bot" style="position:fixed;bottom:5px;left:5px;z-index:99999;"><DIV class="drag" id="Drherb" style="bottom:5px;left:5px;"><DIV id="convblank"><DIV id="qnaClose">X</DIV><DIV id="dialog"><DIV id="box" style="overflow:hidden;height:230px;width:610px;"><DIV style="width:610px;"><DIV class="Herbsay"><P>　Oh hello there,I am Charlie,a trusty member of the &quot;Herb Tasters&quot; and also the brainiest E. coli in the colony.
       
       　I know all the secrets of Chinese herbs and their magical healing powers.
       
       　If you are up to a challenge,find me at team HSiTW at the jamboree.
       
       　I am the one in a straw hat,showing them pearls.I will be waiting.
       
       　Muhahahahaha</P></DIV><DIV class="Herbsay"><P>　Hi there! My name is Nu Zhen Chi. This is how I look like.
       
       　　Take a closer look; guess which part of me is used as medicine?
       (1) the root (2) the stem
       (3) the leaf
       (4) the seed

	</P></DIV><DIV class="Herbsay"><P><B>Ans.</B>(4) the seed
       <B>Name:</B> 女貞子 (Nu Zhen Chi)
       <B>Botanical Name:</B> Ligustrum lucidum Aiton

       　I can treat people who are yin deficient, and liver problems that cause dizziness,cataract of the eyes,
       lower back pain, premature graying of the hair and tinnitus.
	</P></DIV><DIV class="Herbsay"><P>　Hello! My name is Chuan Xiong. This is how I look like. 
       
       　Make a guess, which part of me is used as medicine?
       (1) the root (2) the stem
       (3) the leaf
       (4) the seed

	</P></DIV><DIV class="Herbsay"><P><B>Ans.</B>(1) the root
       <B>Name:</B> 川芎 (Chuan Xiong)
       <B>Botanical Name:</B> Ligusticum chuanxiong Hort

       　I help with blood regulation to prevent relevant to blood stasis and non-stop bleeding.I can also strengthen your qi circulation.
       
       　In addition, I relieve you of physical pain,
         such as headaches, abdominal aches, chest pain, and muscle pain.
       
       　Finally, I free the ladies of menstrual disorders and amenorrhea.
	</P></DIV><DIV class="Herbsay"><P>　What’s up? My name is Dang Gui. I can: 
       (1) stop coughing (2) regulate mense
       (3) reduce internal heat

	</P></DIV><DIV class="Herbsay"><P><B>Ans.</B>(2) Regulate mense
       <B>Name:</B> 當歸 (Dang Gui)
       <B>Botanical Name:</B> Angelica sinensis (Oliv.) Diels

       　I can remove blood stasis and clots, so I am usually used to regulate menses,lubricate intestines to correct constipation, reduce swelling, expel pus.

	</P></DIV><DIV class="Herbsay"><P><B>　Reference</B>
       　臧堃堂 (2005) 中華材輕百科-現代版本草綱目，山岳文化出版社，台北
       <A href="http://brion.org.tw/index.php" target="_blank">Non-Profit Organization Brion Research Institute of Taiwan.</A><A href="https://www.jadeinstitute.com/index.php" target="_blank">Chinese Herb Gallery. Jade Institute</A><A href="http://www.shen-nong.com/eng/front/index.html" target="_blank">Herbal Glossary. Shen-Nong- Chinese Traditional Medicine</A><B>　Acknowledgement</B>
       　Thank you for Non-Profit Organization Brion Resaerch Institute of Taiwan that provide us Chinese herbs and photos.
	</P></DIV></DIV></DIV></DIV><DIV id="qnaNextBt" onclick="slidebar(1,'box');">Next ⇒</DIV><DIV id="qnaPrevBt" onclick="slidebar(-1,'box');">⇐ Prev</DIV></DIV><DIV id="convopen">...</DIV></DIV></DIV><DIV class="myContent"><DIV id="contentList"><LI class="listOne"><P id="listFA">Functional Array</P></LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/FA/Pb'">Pb</LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/FA/Cu'">Cu</LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/FA/As'">As</LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/FA/CYP'">CYP</LI><LI class="listOne"><P id="listCM">Chinese Medicine testing</P></LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/CM/Pb'">Pb</LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/CM/Cu'">Cu</LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/CM/As'">As</LI><LI class="listTwo" onclick="location.href='https://2016.igem.org/Team:HSiTAIWAN/Notebook/CM/CYP'">CYP</LI></DIV><DIV id="mainContent"><P class="titleOne" id="FA">Functional Array Pb</P><P class="titleTwo" id="151112" style="line-height:120%">2015/11/12</P><P class="Topic">Topic:Bacteria Culturing</P><P class="LabnoteTitle">Material:</P><P class="Labnote">old agar with E coli</P><P class="Labnote">coating bar 2</P><P class="Labnote">new agar plate 2</P><P class="Labnote">chloramphenicol</P><P class="Labnote">curette 2</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Gather bacteria from old agar plate with curette</P><P class="Labnote">2.Put bacteria in shaking incubator for 15 minutes</P><P class="Labnote">3.Add more chloramphenicol and bacteria to one of thenew agar plate and add only bacteria to the other plate</P><P class="Labnote">4.Spread the bacteria evenly with coating bar</P><P class="Labnote">5.But new agar plates in incubator</P><P class="LabnoteTitle">Results: are expectednext week</P><P class="Labnote">　The number of reconstructedbacteriawas far less then we had expected. The previous agar plate showed abnormal colonies, and the DNA density results ran out low. We assume the antibiotic wearing off over time, or the cloning vectors unable to successfully be inserted. Therefore, we tried adding more antibiotics to solve the solution.</P><P class="titleTwo" id="151119" style="line-height:120%">2015/11/19</P><P class="Topic">Topic: PCR &amp; agar making</P><P class="LabnoteTitle">Material:</P><P class="Labnote">10X PCR buffer *10μl</P><P class="Labnote">50mM MgCl2 *3μl</P><P class="Labnote">10mM dNTP *2μl</P><P class="Labnote">5U/μlDNA Polymerase *0.4μl</P><P class="Labnote">dd<SUB>2</SUB> *74.6μl</P><P class="Labnote">10μM forward primer *5μl</P><P class="Labnote">10μM reverse primer *5μl</P><P class="Labnote">Blue dye *2μl</P><P class="Labnote">10g/500 ml LB *10ml</P><P class="Labnote">LB *4g</P><P class="Labnote">Agar bacteriological *3g</P><P class="Labnote">dd<SUB>2</SUB> *200ml</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Mix the bacteria that have been incubated since last week with PCR  buffer,  dNTP,  DNA  polymerase,  MgCl2,  dd<SUB>2</SUB>,  and primers into a tube.</P><P class="Labnote">2.Separate the mixture into six smaller tubes and place them into the thermal cycler.</P><P class="Labnote">3.Prepare the gel electrophoresis. Build theplate, and thenheat the agar roll.</P><P class="Labnote">4.Add red dye to the agar roll and spread them across the built module. Reduce the bubbles in the agar roll.</P><P class="Labnote">5.Retrieve the six tubes after PCR, mix them with the blue dye, and add them in individual wells.</P><P class="Labnote">6.Turn on the electricity for (120V, 400mA) 35 minutes.</P><P class="Labnote">7.Scan the gel under infrared ray and select the RNAwe are interested inexamining.</P><P class="Labnote">8.Target  the  colonies  that are  consistent with  the  gel  RNA. Culture them in LB for two tubes(5ml each).</P><P class="Labnote">1.Add LB, agar bacteriological, and dd<SUB>2</SUB> into one single tube.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">　We found two colonies of bacterium that matchthetargeted RNA. Accordingly, we are planning to culturethem and check ifthey may convey the right plasmid.</P><P class="titleTwo" id="151126" style="line-height:120%">2015/11/26</P><P class="Topic">Topic: Agarose Gel Extraction</P><P class="LabnoteTitle">Material:</P><P class="Labnote">nsert*800ng</P><P class="Labnote">Vector*500ng</P><P class="Labnote">10XBuffer (CutSmart)*2</P><P class="Labnote">10X BSA</P><P class="Labnote">Spe1*3.0μl</P><P class="Labnote">Xba1*1.5μl</P><P class="Labnote">dd<SUB>2</SUB>*52.8μl</P><P class="Labnote">1/10000 gel green</P><P class="Labnote">Agarose*40ml</P><P class="Labnote">Dye*12μl</P><P class="Labnote">PE buffer *0.75ml</P><P class="Labnote">3μl/1mg QG buffer</P><P class="Labnote">1μl/1mg isoprophanol</P><P class="Labnote">DNA ligase</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Add the restriction enzymes, buffer, and BSA into two tubes. One tube containsthe insert, and another containsthe vector.Adddd<SUB>2</SUB> and make up to 40μlin both tubes.</P><P class="Labnote">2.Incubate both tubes at 37°C for 1 hour.</P><P class="Labnote">3.Add 6μl of dye into each tube. Prepare the agarose with gel green dye.</P><P class="Labnote">4.Run the gel electrophoresis at 120V, 400mA for 30 minutes.</P><P class="Labnote">5.Slice  the  glowing  DNA  under  UV  light  and  remove  excess agarose.</P><P class="Labnote">6.Put the DNA slices in the clean tubes. Add QR buffer and heat the tubes at 23°C until the gel slices are completely dissolved.</P><P class="Labnote">7.After the gel slices are liquified, add isoprophanol.</P><P class="Labnote">8.Centrifuge it for three times. Firstly, centrifuge it and discard flow-through  agarose.Then, centrifuge it again  and  add 0.75ml PE buffer.Finally, centrifugeitagain and discard the residual ethanol.</P><P class="Labnote">9.Mix  10X  buffer,  DNA  ligase,  vector*4μl, and insert*13μlaltogether. Place the mixturein a 16°C water incubator for ligation.</P><P class="LabnoteTitle">Results: are expectednext week</P><P class="Labnote">　The number of reconstructedbacteriawas far less then we had expected. The previous agar plate showed abnormal colonies, and the DNA density results ran out low. We assume the antibiotic wearing off over time, or the cloning vectors unable to successfully be inserted. Therefore, we tried adding more antibiotics to solve the solution.</P><P class="titleTwo" id="1511127" style="line-height:120%">2015/11/27</P><P class="Topic">Topic: Transformation</P><P class="LabnoteTitle">Material:</P><P class="Labnote">ligated plasmid*10μl</P><P class="Labnote">DH5αbacterium*100μl</P><P class="Labnote">LB broth(without antibody) *900μl</P><P class="Labnote">Agar plates*2</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Add the ligated plasmid to the tube of DH5αbacterium. Then put the mixture onice for 30 minutes.</P><P class="Labnote">2.Heat shock the tube of mixturefor 60 second ina warm water bath at 42∘C. Then, place itonice for 3minutes.</P><P class="Labnote">3.Add the 900μl of LB broth into the tube.</P><P class="Labnote">4.Put thetubein a shaking incubator for 1 hour.</P><P class="Labnote">5.Centrifugefor 2 minutes at 6000rpmand remove 200μl of excess broth on the top.</P><P class="Labnote">6.Divide the mixture into 50μland 150μlrespectively, and spread evenly across two agar plates.</P><P class="titleTwo" id="151210" style="line-height:120%">2015/12/10</P><P class="Topic">Topic: PCR of Final Plasmid</P><P class="LabnoteTitle">Material:</P><P class="Labnote">10X Bfr*35μl</P><P class="Labnote">50mM MgCl2*10.5μl</P><P class="Labnote">10mM dNTP*7μl</P><P class="Labnote">5U/μl DNA Polymerase*1.4μl</P><P class="Labnote">dd<SUB>2</SUB>*261μl</P><P class="Labnote">10μM forward primer(I721002)*17.5μl</P><P class="Labnote">10μM reverse primer(B0015)*17.5μl</P><P class="Labnote">Agarose*1.5g</P><P class="Labnote">0.5X TAE*98.5g</P><P class="Labnote">1/20000 red dye*2μl</P><P class="Labnote">Marker*4μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Pick out white colonies from the agar plate. </P><P class="Labnote">2.Run PCR.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">　98% of the E coli on the agar plate wecultivated last time has shown a red fluorescent. Supposedly, a successfully implantedE coli should not have expressedthis characteristic unless they react with lead ions. Accordingly, weassumed that the backbone stucktogether without containing the insert genes. Thus, we picked out the colonies that werepale white. After PCR, our results showedno signs of successful plasmid.</P>Interested RNA length:169<P class="Labnote">Results: None</P><P class="titleTwo" id="151217" style="line-height:120%">2015/12/17</P><P class="Topic">Topic: PCR, Non-gel Extraction, Ligation</P><P class="LabnoteTitle">Material:</P><P class="Labnote">10X Bfr*20μl</P><P class="Labnote">50mM MgCl2*6μl</P><P class="Labnote">10mM dNTP*4μl</P><P class="Labnote">5U/μl DNA Polymerase*0.8μl</P><P class="Labnote">dd<SUB>2</SUB>*149.2μl</P><P class="Labnote">10μM forward primer(I721002)*10μl</P><P class="Labnote">10μM reverse primer(B0015)*10μl</P><P class="Labnote">Agarose*1.5g</P><P class="Labnote">0.5X TAE*98.5g</P><P class="Labnote">1/20000 red dye*2μl</P><P class="Labnote">Marker*4μl</P><P class="Labnote">Positive control(B00321~I721001)</P><P class="Labnote">Vector fragment(backbonePSB1C3)*18μl</P><P class="Labnote">10X cutsmart*2μl</P><P class="Labnote">CIP(calf intestinalphosphatase)*0.5μl</P><P class="Labnote">EB bfr*40μl</P><P class="Labnote">QG bfr*300μl</P><P class="Labnote">DNA ligase*1μl</P><P class="Labnote">10X bfr*2μl</P><P class="Labnote">Insert(B00321~I721001)*12μl</P><P class="Labnote">dd<SUB>2</SUB>*1μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Mix the vector fragments(backbone)with CIP and place in the water incubator for 30 minutes at 37degrees.</P><P class="Labnote">2.Extract the sample to get rid ofCIP.Because there is no gel   in the sample, so no need to remove the gel.</P><P class="Labnote">3.Ligase the sample with the interested part(B00321~I721001).</P><P class="Labnote">4.Pick out the white colonies from the cultural plates, from previous agar platesthat were cultured on 11/26and 12/10respectively.</P><P class="Labnote">5.Run PCR of the white colonies.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">　Last time, we didn’t find any strips of DNA that matchedour interested DNA. Today, however, several colonies that we cultured last time turn out to be white,which was a possible feature of a successful E. coli. So we run a PCR to test whether the white E. coli has been successfully implanted. The results werenegative. We still don’t have a completed cell. We thought that the binding of the backbone’s two ends might affect the results. Thus, we decided to knock the sticky ends out and try to ligase the plasmid again.</P><P class="Labnote">Interested DNA lenth:170bp</P><P class="Labnote">Results: none</P><P class="titleTwo" id="151224" style="line-height:120%">2015/12/24</P><P class="Topic">Topic: Plasmid Mini, DNA Restriction, Gel Electrophoresis</P><P class="LabnoteTitle">Material:</P><P class="Labnote">PS1 Buffer</P><P class="Labnote">Ethanol*100ml</P><P class="Labnote">RNase A</P><P class="Labnote">W2 Buffer</P><P class="Labnote">50% Glycerol</P><P class="Labnote">PS2 Buffer*200μl</P><P class="Labnote">PS3 Buffer*300μl</P><P class="Labnote">Wash Buffer*1000μl</P><P class="Labnote">dd<SUB>2</SUB>*50μl</P><P class="Labnote">DNA(final plasmid)*10μl</P><P class="Labnote">10X Buffer(cutsmart)*10μl</P><P class="Labnote">10X BSA*10μl</P><P class="Labnote">Restriction enzyme(Spe1)*2.5μl</P><P class="Labnote">Restriction enzyme(Hpa1)*2.5μl</P><P class="Labnote">dd<SUB>2</SUB>*65μl</P><P class="Labnote">Agarose*1.5g</P><P class="Labnote">0.5X TAE*98.5g</P><P class="Labnote">1/20000 red dye*2μl</P><P class="Labnote">Marker*4μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Extract the DNA from the six chosen colonies with the plasmid mini kit. Collect them in six individual tubes.</P><P class="Labnote">2.Cut DNA at the restriction site of Spe1~Hpa1(approximately 100bp).</P><P class="Labnote">3.Measure the concentration of DNA from each tube. Eliminate the sample with a low concentration(17.7ng/μl).</P><P class="Labnote">4.Run the remaining DNA samplesin a gel electrophoresis.</P><P class="Labnote">5.Searchfor DNA parts at the length of 100bp.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">　Our white E. coli colonies could have two possible results. First, the plasmid might have a reversed insert site. In this case, there should be a DNA length about 500bp when we run the infrared sensor. Second, theplasmid has been successfully inserted.If this is the case, we should see a DNA length about 100bp.</P><P class="Labnote">　Our results came out with no DNA length near 100. However, we found four plasmids showing a sequence of DNA with the same length, which we assume itis the backbone. Three ofthe plasmids show nothing else;while one plasmid shows a DNA with the length around 500bp. The last plasmid is believed to be the one with a wrong insert.Because we did not find the DNA, we hypothesized that the three plasmids have a different DNA sequence than the last one. In other words, we think that the three plasmids convey the correct DNA. Yet, the infrared sensor device did not show any DNA we were looking for. So, we presumed that the enzyme(Hpa1) could be mild, causing poor visibility of the DNA. Next time, we plan to enhance the quantity of the enzyme. Also, we will increase the density of gel so that the DNA could be more concentrated.</P><P class="Labnote">The plasmid to the right shows an additional DNA.</P><P class="Labnote">The four plasmids on the right show a same DNA part.</P><P class="Labnote">Interested DNA:100bp</P><P class="Labnote">Results: none</P><P class="titleTwo" id="151231" style="line-height:120%">2015/12/31</P><P class="Topic">Topic: Gel Electrophoresis</P><P class="LabnoteTitle">Material:</P><P class="Labnote">DNA(sample1 DNA density: 107.9ng/μl)*3μl</P><P class="Labnote">DNA(sample3 DNA density: 413.4ng/μl)*0.8μl</P><P class="Labnote">DNA(sample5 DNA density: 821.4ng/μl)*0.4μl</P><P class="Labnote">10X BSA*6μl</P><P class="Labnote">10X Bfr*6μl</P><P class="Labnote">Spe1*2.4μl</P><P class="Labnote">Hpa1*2.4μl</P><P class="Labnote">dd<SUB>2</SUB>*39μl</P><P class="Labnote">Green dye*2μl</P><P class="Labnote">1.5% Agarose*25μl</P><P class="Labnote">100bp Marker*2μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Choose three colonies that differ the most. Measure the density of DNA in each.</P><P class="Labnote">2.Aquire thecorrect amountof samples to make the equal amount of DNA.</P><P class="Labnote">3.Run the gel electrophoresis.</P><P class="LabnoteTitle">Results</P><P class="Labnote">Although we have added more enzymes, there is still no sign of DNA around 100bp. The result is similar to that from last time, but with a clearer view of the DNA. Due to the fact that features of the colonies we have chosen accord to a successfully inserted colony.There should be a DNA strain around 100bp, unless, the sequence of our E coli. contradicts logically. We will check whether there is any problem in our sequence before the RFP.</P><P class="Labnote">Interested DNA:100bp</P><P class="Labnote">Results: none</P><P class="titleTwo" id="160114" style="line-height:120%">2016/01/14</P><P class="Labnote">We now face difficulties of getting the correct DNA length. Thus, we ran a primer on two different plasmids, from the RFP site to the Lead Promoter site. After matching the DNA sequence, we couldn’ t find the Lead Promoter site. However,we found the DNA sequence of B0032 at 471~483bp, the sequence of I721002 at 491~889bp and an incomplete part of B0015 at 897~. This result (figure 1)fits perfectly if we consider the whole insert to be reversed. The reason for the missing sequence of the Lead Promoter may be because the primer couldn’t go any further. So, we took a closer look at the three DNA sequences we matched. The B0032 sequence was negative(reversed), but the other two were positive(not reversed). To shorten up, the B0032 has been reversed, and the whole insert has also been reversed in the backbone. Next time, we will reinsert the insert partsand make sure we have the right plasmid (figure 2)through quick PCR.</P><P class="titleTwo" id="160125" style="line-height:120%">2016/01/25</P><P class="Topic">Topic: Gene Extraction, Ligation</P><P class="LabnoteTitle">Material:</P><P class="Labnote">CIP*1μl</P><P class="Labnote">restriction enzymes(Spe1)*5μl</P><P class="Labnote">restriction enzymes(Xba1)*2.5μl</P><P class="Labnote">10X cutsmart*2μl</P><P class="Labnote">1/20000red dye*2μl</P><P class="Labnote">blue dye</P><P class="Labnote">Marker*4μl</P><P class="Labnote">Agarose*1.5g</P><P class="Labnote">0.5X TAE*98.5g</P><P class="Labnote">Vector(backbone)*5μl</P><P class="Labnote">Insert(B0032~I721001)*12μl</P><P class="Labnote">3μl/mg QGbfr*630μl</P><P class="Labnote">DEbfr *750μl</P><P class="Labnote">dd<SUB>2</SUB>*80μl</P><P class="Labnote">10X bfr*2μ</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Redo the ligation process and then extract the insert and vector from individual plasmids.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">None. Results areexpected after transformation and PCR.</P><P class="titleTwo" id="160126" style="line-height:120%">2016/01/26</P><P class="Topic">Topic: Transformation</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Ligated plasmid*10μl</P><P class="Labnote">DH5αbacterium*100μl</P><P class="Labnote">LB broth*900μl</P><P class="Labnote">Agar petri dish*1</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Transform the plasmid into the DH5α</P><P class="Labnote">2.Culture the bacterium.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">*The amount of bacterium shown after centrifugation was unusuallylittle.</P><P class="Labnote">Results are expected after PCR.</P><P class="titleTwo" id="160127" style="line-height:120%">2016/01/27</P><P class="Topic">Topic: PCR of final plasmid</P><P class="LabnoteTitle">Material:</P><P class="Labnote">10X bfr*(35+35)μl</P><P class="Labnote">50mM MgCl2*(10.5+0)μl</P><P class="Labnote">dNTP*(7+7)μl</P><P class="Labnote">F primer(E1010)*(17.5+7)μl</P><P class="Labnote">R primer(B0032)*(17.5+7)μl</P><P class="Labnote">Polymerase*(1.4+4.2)μl</P><P class="Labnote">dd<SUB>2</SUB>*(261.1+289.8)μl</P><P class="Labnote">0.8X TAE*140ml</P><P class="Labnote">Agarose</P><P class="Labnote">1/1000 AMP*8μl</P><P class="Labnote">LB broth*8ml</P><P class="Labnote">*We tried a new buffer that included MgCl2, so we didn’t add in any MgCl2 the second time we did PCR</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Pick out 62 colonies from the former petri dish.</P><P class="Labnote">2.Do PCR.</P><P class="Labnote">3.Pick out 2from the 9 colonies that have our interested DNA length.</P><P class="Labnote">4.Culture the 2 colonies in LB broth.</P><P class="Labnote">5.Send the samples for sequencing.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">Every colony on the agar plateappears to be white, which is what a correctly ligased E. coli should express. To increase the chance of succeedingin finding the correct colonies, we picked 62 colonies for PCR. 9(1-2, 1-3, 1-4, 1-5, 1-6, )out of the 62 tend to have the rightlength of DNA(about 700bp). And to make sure that the sequences are accurate, we chose 2 colonies(1-6, 2-18)which are most valid after infrared  lightscanning. We  will  send  the  samples  in  for  further sequencing.</P><P class="titleTwo" id="160201" style="line-height:120%">2016/02/01</P><P class="Labnote">After sequencing, we confirmed that the DNA is indeed what we need. To testthe  E.  coli  accessibility, we are  going  to performethree experiments,including howthe  E.coli react  to  Pb  in  different concentrations([Pb]), what factors may alterthe expression of the E. coli, and the survival ratesof E. coli in different living conditions. We makea table that lists four variables(restriction of growth, E. coli amount, time,and[Pb]) to record different results. It was our original plan (Figure1and Table 1).</P><P class="Labnote">In order to assure that there are equal amountsof bacteriain the tubes,we cultured the E. coli in a single tube, andthenevenly divided into all tubes. Next, we added5differentconcentrations of Pb into the 5tubes. However, at this point, the professor reminded us to be aware of the Pb ions penetration-time. The different concentration of Pb might affect thebacteria’s absorption speed, thus influence the amount of Pb in the cells. To control these potential biases, he advisedus to observe Pb in different concentrations,  and  control  three  variables  in  terms  of  restriction  of growth, time duration, and amount of E coli. We then changed our plan and performed the new strategies.</P><P class="Labnote">First, we prepare 10 tubesthat contain different [Pb]concentration levels, andevenly add the E. coli in each tube(Figure2and Table 2). Then, culture the tubes for one nightto ensure all bacteriahaveabsorbedthe maximum amount of Pb ion. Then, we will observe the remaining E. coli  (some  might  die in  high  concentration  of  Pb)  and  examine  the differencesbetween one another via O.D. Finally, we will choose the most suitable Pb concentration as the goldstandard to conduct the remaining experiments. By completing these processes, we can both figure out the best situation to collect our results and test the survival rate of our E. coli.</P><P class="Topic">Topic: Preparing E. coli</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Ampicillin*6μl</P><P class="Labnote">LB broth (without antibiotics)*6ml</P><P class="Labnote">E. coli (cloned)*3μl</P><P class="Labnote">E. coli as vector (J23102)*3μl</P><P class="Labnote">Tube*2</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Mix two tubes of LB broth with Ampicillin.</P><P class="Labnote">2.Add the cloned E. coli and E. coli(J23102)into 2 individual tubes, respectively.</P><P class="titleTwo" id="160202" style="line-height:120%">2016/02/02</P><P class="Topic">Topic: Functional Array</P><P class="LabnoteTitle">Material:</P><P class="Labnote">E. coli (cloned)*45μl</P><P class="Labnote">E. coli as vector(J23102)*5μl</P><P class="Labnote">LB broth (without antibiotics)*10000μl</P><P class="Labnote">Ampicillin*10μl</P><P class="Labnote">10<SUP>4</SUP>ppm Pb (aq)*(100+100)μl</P><P class="Labnote">ddH<SUB>2</SUB>O*9000μl</P><P class="Labnote">Cultivation tube*10</P><P class="Labnote">Dilution tube*9</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Dilute the Pb aqueous solution by adding ddH<SUB>2</SUB>O to make 9different  levels of  concentration,respectively,  10<SUP>4</SUP>(ppm), 10<SUP>3</SUP>, ..., 10<SUP>-4</SUP>.</P><P class="Labnote">2.Prepare LB broth by adding ampicillin.</P><P class="Labnote">3.Make 9 cultrivation tubes that contain 9 differentlevels of Pb concentration.[formula: 100μl of Pb<SUB>(aq)</SUB>,5μl of bacterium and 900μl LB to form 1c.c of fresh mixture]</P><P class="Labnote">*Because the aqueous solution has been diluted again in the mixture, the true concentration inthe tubes should drop one foldof ten.
    </P><P class="Labnote">4.Cultivate  the  10  tubes(inclulding  one  control  tube)in  a shaking incubatorovernight.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">Results are expected the next day.</P><P class="titleTwo" id="160203" style="line-height:120%">2016/02/03</P><P class="Topic">Topic: Functional Array2</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Pb(NO<SUB>3</SUB>)<SUB>2</SUB>*5g</P><P class="Labnote">ddH<SUB>2</SUB>O*45ml</P><P class="Labnote">LB broth*4500μl</P><P class="Labnote">Ampicillin*4.5μl</P><P class="Labnote">E. coli (cloned)*(200+20)μl</P><P class="Labnote">E. coli as vector (J23102)*5μl</P><P class="Labnote">Petri dish*2</P><P class="Labnote">Coating bar*2</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Run an O.D test on the tubes from last time.</P><P class="Labnote">2.Check the E. coli under UV light.</P><P class="Labnote">3.Make a fresh tube of Pb aqueoussolutionin the concentration of 10<SUP>5</SUP>ppm.</P><P class="Labnote">4.Dilute  the  solution  into  8  different  levels.(10<SUP>5</SUP>,  10<SUP>4</SUP>,  ..., 10<SUP>-2</SUP>ppm)</P><P class="Labnote">5.Spread 100μl of E. coli per fresh agar dish. Add 4 different Pb solution on 4  corners  of  the  plate.  (Do  not shake  the  petri dishes!)</P><P class="Labnote">6.Make 4 cultivation tubes like last time with the concentration of 10, 10<SUP>3</SUP>, 10<SUP>2</SUP>, 10<SUP>1</SUP>ppm and a tube for reference without Pb. (formula  of  1c.c. mixture:  LB  broth*900μl,  Amp*0.9μl, bacterium*5μl, Pb<SUB>(aq)</SUB>*100μl)</P><P class="LabnoteTitle">Results:</P><P class="Labnote">OD results</P><P class="Labnote">　According to the results in the table above, the concentration of Pb between 101to 10-7ppm doesn’t affect the survival of E. colimuch. There isnosignificant finding in this experiment.</P><P class="Labnote">　Compare to the control tube, the E. coli in the experimental tubes didn’t express any obvious glow(Figure 1 and 2). There were two Victor Shen 2016/2/3possibilities. First, the experiment indeed failed. Second, the E. coli actually did react, but was just too faint for our naked eyes to observe. Surprisingly,  our  bacterium  did  express  a  hazy fluorescent(Figure 3).So, we assumed the process is accessible, but the level ofPb ions might be too low. Much to our shock, we mistook the former [Pb] 102folds higher. Every solution was 102lower than the what we mean to prepare.</P><P class="Labnote">　Meanwhile, we concerned that LB broth might somehow react with the Pb solution, and then alter the proper functionof the E. coli.  Hence, we  tried  anothermethod  to  test  the results  of bacteriain different [Pb] levels.We first spread our E. coli onto the agar plates to make sure they have the necessary survival requirements.  Next,  we  dripped  small  amounts  of  Pb  solution onto the surface of our E. coli. By doing this, we expect no other materials from  the  broth could interfere the  reaction.</P><P class="titleTwo" id="160218" style="line-height:120%">2016/02/18</P><P class="Topic">Topic: Fluorescence Spectrometry</P><P class="LabnoteTitle">Material:</P><P class="Labnote">LB broth*1750μl</P><P class="Labnote">Control bacteria(J61002)*250μl</P><P class="Labnote">100μl/well Old bacterial fluid in different levels of [Pb]*100*14μl</P><P class="Labnote">Thermo Varioskan Flash</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Mix  the  bacteria  clusters  at  the  bottom  of  the  Eppendorf tubes.</P><P class="Labnote">2.Dilute the control bacteria 2 times into 7 tubes. (1, 2<SUP>-1</SUP>, 2<SUP>-2</SUP>, 2<SUP>-3</SUP>, 2<SUP>-4</SUP>, 2<SUP>-5</SUP>, 2<SUP>-6</SUP>)</P><P class="Labnote">3.Add  the  control  and  bacterial  fluid  into  the  wells  of  a photographic plate.</P><P class="Labnote">4.Place the plate into the fluorescence spectrometry machine. (excitation 585, emission 610)</P><P class="LabnoteTitle">Results:</P><P class="Labnote">Table 1 shows that the absorption levels and concentration levels increase at a direct proportion.Figure 1 shows that the spectrometer is precise and the relation between the absorption level and the concentration level.As shown in Table 1 and Figure1, we can see that the detecting device is precise and the deviation mean is low. Thus, our calculations can be accounted accurate.</P><P class="Labnote">In Table2, any bacteria with Pb solution added shows a higher fluorescence absorption compared to that without any solution amplified. There is a slight difference between the bacteria with Pb solution added and none. But the distribution of the absorption results appears to be random.Hence, we knowthat our E. coli does express a red fluorescence when combined with Pb ions. However, there is no significant relevance between the Pb ion levels and the E. coli’s fluorescence expressions.</P><P class="Labnote">In conclusion, our E. coli is capable of detecting the existenceof Pb ions, but is not able to differentiate the changes in the concentration levels of the solutions.</P><P class="titleTwo" id="160225" style="line-height:120%">2016/02/25</P><P class="Topic">Topic: Functional Array 2</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Zn(NO<SUB>3</SUB>)<SUB>2-6H<SUB>2</SUB>O*500mg</SUB></P><P class="Labnote">NiSO<SUB>4</SUB>-xH<SUB>2</SUB>O*500mg</P><P class="Labnote">CuSO<SUB>4</SUB>-5H<SUB>2</SUB>O*500mg</P><P class="Labnote">MnSO<SUB>4</SUB>-H<SUB>2</SUB>O*500mg</P><P class="Labnote">Pb(NO<SUB>3</SUB>)<SUB>2</SUB>*500mg</P><P class="Labnote">LB broth*900*34+100μl</P><P class="Labnote">Bacteria*5*34μ</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Prepare 5solutions  that  consist  of  5different  metal  ionsrespectively(Zn<SUP>2+</SUP>, Ni<SUP>2+</SUP>, Cu<SUP>2+</SUP>, Mn<SUP>2+</SUP>, Pb<SUP>2+</SUP>).</P><P class="Labnote">2.Dilute  every  tube  of  solution  into  5  different  levels  of concentration (10<SUP>-6</SUP>, 10<SUP>-4</SUP>, 10<SUP>-2</SUP>, 10<SUP>0</SUP>, 10<SUP>2</SUP>ppm).</P><P class="Labnote">3.Dilute the lead ion solution further more. Make three other tubes of solution at the concentration of 10<SUP>-7</SUP>, 10<SUP>-8</SUP>, 10<SUP>-9</SUP>ppm.</P><P class="Labnote">4.Make a mixture of bacteria and metal ions. Formula:900μl of fresh LB broth with antibiotics, 100μl of diluted ion solution, and 5μl of bacteria.</P><P class="Labnote">5.Culture the tube in a shaking incubator.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">We wanted to examine whether our E. coli has a strong singularity and if the presence of other metal ions may influence the bacteria’s absorption ability. So, we tried out 4 othermetal ions in additional to leadions. Furthermore, we wanted to test the detecting ranges of our E. coli. According to our previous functional tests, we were able to conclude that our bacterial device was capable of detecting Pb ions less than 10<SUP>-7</SUP>ppm. Therefore, we will try to challenge its lowest limitthis time.</P><P class="titleTwo" id="160226" style="line-height:120%">2016/02/26</P><P class="Topic">Topic: Functional Array2</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Bacteria as control (J23102)*250μl</P><P class="Labnote">LB broth used as blank*250μl</P><P class="Labnote">E. coli used as blank*250μl</P><P class="Labnote">Different solutions consisting different metal ionsand BBa_K1613017</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Vortex to mix the bacteria clusters at the bottom of the Eppendorf tubes.</P><P class="Labnote">2.</P><P class="Labnote">3.Add two control sets and experimental set into the wells of a photographic plate.</P><P class="Labnote">4.Place the plate into the fluorescence spectrometry machine. (excitation 585, emission 610)</P><P class="LabnoteTitle">Results:</P><P class="Labnote">In this study, we used functional arrays to examine the singularity. We expected to find that E coli only responded to lead ions. But we did not get the results we expected. Table 1 shows the control samples we made. In our experiment, we made two sets of control:one was fresh LB broth and E. coli; another one was J231002 in different solutions at the concentration level of 10-3ppm. Table 2 shows the fluorescence absorption of different metal ions. The results suggest that our E coli were significantly responded to lead ions. However, our E coli were also significantly responded to other metal ions. We suspected that the bacteria havea poor singularity bond between lead ions, so they also react with other metal substances. Accordingly, we decide to expend length in time,allowing sufficient time for the E. coli to interact with lead ions, and distinguish the differences of lead from other metal ions.In the previous experiment, the reaction duration was 24hours. This time, we plan to prolong the reactionduration to 1 week.</P><P class="titleTwo" id="160303" style="line-height:120%">2016/03/03</P><P class="Topic">Topic: Functional Array3 (fluorescence microscope)</P><P class="LabnoteTitle">Results:</P><P class="Labnote">　*Exposure Time: 350ms</P><P class="Labnote">　*During  editing,  we  increased  the  brightness  of  every picture 100% in order to enhance the difference between the background color.</P><P class="Labnote">　After discussion, we suspected that living bacteria might disrupt the  detecting  of  fluorescence  protein. A  fluorescence  spectrometry machine doesn’t only detect the fluorescence level, but it also calculates several other factors. So, we decided to examine the bacteria through a fluorescence microscope, in that way, we could rule out other factors except the fluorescence of the bacteria.</P><P class="Labnote">　Based on the picturesbelow, they obviously showthat the bacteria consisting of lead expressesa clear glow compared to other experiment samples.Thus, we can conclude that our E. coli dohave a specific characteristic.</P><P class="titleTwo" id="160428" style="line-height:120%">2016/04/28</P><P class="Topic">Topic: Preparing Toxic Solution</P><P class="LabnoteTitle">Material:</P><P class="Labnote">CuSO<SUB>4</SUB>-5H<SUB>2</SUB>O*2.0g</P><P class="Labnote">10<SUP>5</SUP>ppm Pb(NO<SUB>3</SUB>)<SUB>2</SUB>*10ml</P><P class="Labnote">Tube*13</P><P class="Labnote">ddH<SUB>2</SUB>O*118ml</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Measure 2g of Cu<SUP>2+</SUP>crystal and pour it into a tube. Add ddH<SUB>2</SUB>O to 10ml.</P><P class="Labnote">2.Dilute the solution by 10 times in the sequence of 10<SUP>5</SUP> 10<SUP>4</SUP> 10<SUP>3</SUP> 10<SUP>2</SUP> 10<SUP>1</SUP> 1 10<SUP>-1</SUP>ppm in concentration.</P><P class="Labnote">3.Do the same as for lead solution.</P><P class="LabnoteTitle">Note:</P><P class="Labnote">oday,  our  plan  was  to  culture  E.coli  in  five  different concentrations, ranging from 10<SUP>3</SUP>to 10<SUP>-1</SUP>. Thus, I first diluted the five concentrations.  However,  my  advisor  reminded  me  that  the ion solution would be further diluted when added into the culturing broth. Either  I  prepare  the  exact  same  concentration  in  LB  broth,  or,  I prepare a more concentrated water solution. So, I chose the latter and prepared the solution up to 10<SUP>5</SUP>ppm. Also, I was advised thatE. coli would stop growing after 14 hours, hence, I would have to collect my sample by then. If the bacteria still remain in the 37°C incubator, eventually the cells would all die out. Another thing I learnt was that the fluorescent proteinwould begin the process of protein degradation. (Protein should be stored at -80°C, in other words, 37°C is not a nice temperature for storing protein). Given these two conditions, I should not culture the E coli today because I have school classestomorrowand I do not have two consecutive days.Therefore, I plan to culture the E. coli over the weekend.</P><P class="titleTwo" id="160505" style="line-height:120%">2016/05/05</P><P class="Topic">Topic: Preparing Solution and Bacteria Culturing</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Na<SUB>2</SUB>HAsO<SUB>4</SUB>·7H<SUB>2</SUB>O*1g</P><P class="Labnote">ddH<SUB>2</SUB>O*19ml</P><P class="Labnote">Petri dish(ampicillin)*1</P><P class="Labnote">Petri dish(Chloramphenicol)*1</P><P class="Labnote">Alcohol lamp(70% full)*1</P><P class="Labnote">Steel coating bar*1</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Prepare one tube of arsenic solution inthe concentration of 10<SUP>5</SUP>ppm.</P><P class="Labnote">2.Dilute the solution by tenth folds, making another solution in the concentration of 10<SUP>4</SUP>ppm.</P><P class="Labnote">3.Wipe  the laboratory  counter with  adequate alcohol. Then, light an alcohol lamp in the middle of the table to provide a temporary germ-free environment (1 square meter).</P><P class="Labnote">4.Burn  the  steel  coating  bar  until  it  turns  bright  red  for sterilization. Wait about 20 seconds for the coating bar to cool off, or else, the high temperature will destroy the germ colonies later on.</P><P class="Labnote">5.Using  the  coating  bar,dab  a  colony  from  a  former  plate. Zigzag the germ colony on a new petri dish.</P><P class="Labnote">6.Burn the steel coating bar again.</P><P class="Labnote">7.Spread the colony on the fresh petri dish further. In that case, the colonies will be more scattered. Thus, it will be easy for us to pick colonies in the future.</P><P class="Labnote">8.Place the petri dishes into the incubator to culture.</P><P class="titleTwo" id="160506" style="line-height:120%">2016/05/06</P><P class="Topic">Topic: Plasmid functional array</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Cu<SUP>2+</SUP>solution in the concentration of 2*10<SUP>5</SUP>to 2*10<SUP>-1</SUP>ppm</P><P class="Labnote">Pb<SUP>2+</SUP>solution in the concentration of 10<SUP>5</SUP>to 10<SUP>-1</SUP>ppm</P><P class="Labnote">E. coli (K1555000) (K1613017)</P><P class="Labnote">Ampicillin*10μl</P><P class="Labnote">Chloramphenicol*10μl</P><P class="Labnote">LB Broth*20ml</P><P class="Labnote">Culturing tube*10</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Create a germ-free environment with an alcohol lamp.</P><P class="Labnote">2.Divide the broth into two tubes. Add the appropriate antibiotic into the broth at a ratio of 1:1000.</P><P class="Labnote">3.Distribute the broth into the other remaining tubes.One tube should require 2ml of broth.</P><P class="Labnote">4.Poke the colonies on the agar plate gently with a plastic tip. Dip the tip inside the broth.</P><P class="Labnote">5.Add the corresponding solution into the tubes at a ratio of 1:100.</P><P class="Labnote">6.Place the culturing tubes into a shaking incubator overnight.</P><P class="LabnoteTitle">Note:</P><P class="Labnote">Before we open, or close the caps of containers containing LB, be sure to burn the cap for a split second. That way, we can ensure the opening to be clean and uncontaminated.</P><P class="titleTwo" id="160512" style="line-height:120%">2016/05/12</P><P class="Topic">Topic: Listing of Chinese Medicine and Preparing SolutionMaterials</P><P class="LabnoteTitle">Material:</P><P class="Labnote">黃岑、甘草、知母、茯苓、酸棗仁湯、薄荷、黃耆、牡丹皮、川芎、當歸、白芍、女貞子、炙甘草、炮薑、桭子、柴胡、逍遙散、四物湯、霍香</P><P class="Labnote">Eppendorf*20</P><P class="Labnote">ddH<SUB>2</SUB>O</P><P class="Labnote">Dry bath incubator</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Analyze the medicine into three categories, liquid, thick broth, or solid chunks.</P><P class="Labnote">2.Heat the thick broth at the temperature of 100°C for 10 minutes in a dry bath incubator. Wait until the broth liquefies.</P><P class="Labnote">3.Dilute  the  fluid  by  1/10  into  individual  Eppendorf. The formula should be: 150μl of Chinese medicine and 1350μl of ddH<SUB>2</SUB>O.*Thick broth often clogs up easily. So be sure to be fast.</P><P class="Labnote">4.Smash the solid chunks into powder, the dainty the better.</P><P class="Labnote">5.Measure 15mg of powder. Add ddH<SUB>2</SUB>O in an Eppendorf to 1.5ml. Mix the two thoroughly.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">Some of the medicine stew was too thick that it did not liquefy even after heating for 20 minutes (Figure 1). And some of the medicine chunk was too hard to break down (Figure 2). So, we decided to turn to a herbal medicine professor for advice. Then, we can figure out how to deal with this situation.</P><P class="titleTwo" id="160519" style="line-height:120%">2016/05/19</P><P class="Topic">Topic: Preparing solution, planning the experiments</P><P class="LabnoteTitle">Material:</P><P class="Labnote">岩精*140mg</P><P class="Labnote">硃砂*15mg</P><P class="Labnote">木香*15mg</P><P class="Labnote">ddH<SUB>2</SUB>O*17ml</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Add the samples into different Eppendorf.</P><P class="Labnote">2.Add ddH<SUB>2</SUB>O to the concentration of 10mg/ml.</P><P class="LabnoteTitle">Future Project:</P><P class="Labnote">We plannedto test whether Chinese medicine would suppress the growth of the E. coli. Thus, we have decided to add Chinese medicine into the culturing environment of the bacteria. Essentially, there are three methodsof conducting this experiment. First, spread Chinese medicine across the surface of the agar plates while culturing. Second, mix Chinese medicine into the agar gel. Then, culture the bacteria as usual. Third, culture the bacteria in a culturing tube and add Chinese medicine. We chose the latestmethodfor two considerations. One the one hand, the first two methods requirecounting colonies. Onthe other hand, the last method analyzes the bacteria’s growth via optical densitywhich is more reliable. Using a more objective analyzing system, we can increase the accuracy of our experiment.</P><P class="Labnote">Further, we dilutedChinese medicine into 5 different levels of concentration, ranging widely. Because we intend to test the survival rate of the E. coli,the concentrationrange should be broader to save time and resources.</P><P class="titleTwo" id="160" style="line-height:120%">2016/0/</P><P class="Topic">Topic: Plasmid Fluorescence, ODFunctional Array 2</P><P class="LabnoteTitle">Material:</P><P class="Labnote">LB Broth*30ml</P><P class="Labnote">Antibiotic(Ampicillin)*1.3μl</P><P class="Labnote">Antibiotic(Chloramphenicol)*1.3μl</P><P class="Labnote">E. coli(k1555000)(Cu<SUP>2+</SUP>at 10<SUP>-7</SUP>~10<SUP>2</SUP>ppm)*30μl each</P><P class="Labnote">E. coli(k1613017)(Pb<SUP>2+</SUP>at 10<SUP>-7</SUP>~10<SUP>2</SUP>ppm)*30μl each
ddH<SUB>2</SUB>O*(13+1.3)*2μl</P><P class="Labnote">Copper ion solution(10<SUP>-5</SUP>, 10<SUP>-3</SUP>, 10<SUP>-1</SUP>, 10<SUP>1</SUP>, 10<SUP>2</SUP>ppm) *10μl each</P><P class="Labnote">Lead ion solution(10<SUP>-5</SUP>, 10<SUP>-3</SUP>, 10<SUP>-1</SUP>, 10<SUP>1</SUP>, 10<SUP>2</SUP>ppm)*10μl each</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Prepare blank samples with LB broth, ddH<SUB>2</SUB>O, ion solution, and antibiotics. (Formula: 1300μl of LB and 13μl of ddH2O, 1300μl of LB and 1.3μl of ddH2O, 1300μl of LB and 1.3μl of antibiotics)</P><P class="Labnote">2.Dilute the cultured bacterial fluid from last night by 1/10, 1/ 50.  Be  sure  to  vortex  and  then  centrifuge  the fluid  briefly before dilution, that way, the E. coli will distribute more evenly.</P><P class="Labnote">3.Load the corresponding fluid into the well(Figure 1, 2). (200μl per well)The well for testing optical density is transparent, while the well for testing fluorescence is black.</P><P class="titleTwo" id="160602" style="line-height:120%">2016/06/02</P><P class="Topic">Topic: Analysisof OD, GFP, RFP</P><P class="LabnoteTitle">Results:</P><P class="Labnote">According to the bar charts(Figure 1 &amp; Figure2), we can see that as the same bacterial fluid gets diluted, the optical density drops.Also, from the charts, we can assume that metal ion concentration levels are not relevant to the growth of E. coli. Hence, the instrument for measuring OD is functional and accurate.</P><P class="Labnote">In terms of the fluorescence sample, results are not going well. Aside from the sample with thehighest concentration of copper ion (10<SUP>2</SUP>ppm), all the other testing samples have no significant absorption difference with the negative control sample (Figure 3). What’s worse, the  fluorescence  absorption  in  RFP  (biosensor  for  lead)  appears random(Figure4). Thus, we cannot even find the slightest connection between the concentration of lead ion and the intensity of RFP, whereas the negative control sample shows a stronger fluorescence than the sample with the highest ion concentration. Based on the results, we are guessing that the instrument for detecting fluorescence absorption is not reliable. So, we have to use another device in our next experiment.</P><P class="titleTwo" id="160616" style="line-height:120%">2016/06/16</P><P class="Topic">Topic: Transformation</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Competent cell (DH5a)*15*3μ</P><P class="Labnote">Plasmid (K608010, K608012, K516132)*2μl for each</P><P class="Labnote">SOB*200*3μl</P><P class="Labnote">Agar plate*3μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Add  15μl  of  competent  cell  with  2μl  of  plasmid  into  an Eppendorf.</P><P class="Labnote">2.Ice the bacteria in ice for 5 minutes.</P><P class="Labnote">3.Heatshock the bacteria in an incubator at 37°C for 3 minutes.</P><P class="Labnote">4.Place the bacteria in ice to recover for 2 minutes.</P><P class="Labnote">5.Add 200μlof SOBinto the bacteria.</P><P class="Labnote">6.Put the bacteria in a shaking incubator for 4 hours.</P><P class="Labnote">7.Spread the bacteria on an agar plate and distribute the fluid evenly with a coating bar.</P><P class="Labnote">8.Place the petri dish into an incubator overnight.</P><P class="titleTwo" id="160718" style="line-height:120%">2016/07/18</P><P class="Topic">Topic: Culturing Frozen Bacteria</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Frozen bacteria (k1613017, k1555000, J23102)*100μl</P><P class="Labnote">Ampicillin*50μl</P><P class="Labnote">Chloramphenicol*50μl</P><P class="Labnote">Agar plate*3</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Add antibiotics in the antibiotic free agar plates.</P><P class="Labnote">2.Distribute the bacteria evenly on the plate with acoating bar.</P><P class="titleTwo" id="160" style="line-height:120%">2016/0/</P><P class="Topic">Topic: Growth RatePretest, Fluorescence Preparation, Lead Solution Preparation, Strick Out</P><P class="LabnoteTitle">Material:</P><P class="Labnote">LB*20ml</P><P class="Labnote">Ampicillin*20μl</P><P class="Labnote">Former agar plate (k1613017)</P><P class="Labnote">LB*6000+490*48μl</P><P class="Labnote">Ampicillin*3*2μl</P><P class="Labnote">Former agar plate (RFP positive control, k1613017)</P><P class="Labnote">24well plate*1</P><P class="Labnote">Culturing tube*24</P><P class="Labnote">Pb(NO<SUB>3</SUB>)<SUB>2</SUB>*8g</P><P class="Labnote">ddH<SUB>2</SUB>O*500+0.9*10ml</P><P class="Labnote">Coating bar*1</P><P class="Labnote">Fresh agar plate*3</P><P class="Labnote">Former agar plate (RFP positive control, k1613017, k1555000)</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">(For growth rate pretest)</P><P class="Labnote">　1.Pick a colony (k1613017) and add it in 20μl of Ampicillin and 20mlof LB.</P><P class="Labnote">　2.Culture  for  the  first  4  hours  without  testing  optical density(OD). *Because the bacteria haven’t gone into a log phase, the growth appears to be slow. There is no use in testing OD at this time.</P><P class="Labnote">　3.Test the optical density next every 1 hour.</P><P class="Labnote">(For fluorescence preparation)</P><P class="Labnote">　1.Pick two colonies (k1613017, RFP positive control) and culture them separately in 3ml of LB and 3μl of Ampicillin.</P><P class="Labnote">　2.Culture the two tubes in an orbiting incubator for 8 hours.</P><P class="Labnote">　3.Add 490μl LB, 5μl of lead solution, and 3μl of bacterial fluid into a 24well plate and culturing tubes.</P><P class="Labnote">　4.There should be six different samples in each experiment set. Negative control, positive control (RFP), 10<SUP>1</SUP>, 10<SUP>-2</SUP>, 10<SUP>-7</SUP>, 10<SUP>-8</SUP>ppm concentration of lead ion.</P><P class="Labnote">　5.Incubate the two sets (well plate, culturing tube) in a shaking incubator for 16 hours.</P><P class="Labnote">(For lead solution preparation)</P><P class="Labnote">　1.Add 8g of Pb(NO<SUB>3</SUB>)<SUB>2</SUB> into 500ml of ddH<SUB>2</SUB>O to create a lead ion solution in the concentration of 10<SUP>4</SUP>ppm.</P><P class="Labnote">　2.Dilute the solution by 10 folds, from 10<SUP>4</SUP>to 10<SUP>-6</SUP>ppm. *900μl of ddH<SUB>2</SUB>O+100μl of previous solution.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">Although the bacteria colony we took contained a large variety of E. coli, we can still take the growth rate as a reference. We wanted to examine the growth phase of our bacteria, that way, we can make sure that our following experimentswill acquire healthy and fresh bacteria, which are in the log phase. In the graph below, after culturing about 7 hours, the bacteria approached 0.6, which is the perfect condition for testing. Thus, we will design our further experiments to be based on bacteria which are cultured for 7 hours or so</P><P class="titleTwo" id="160720" style="line-height:120%">2016/07/20</P><P class="Topic">Topic: Functional Testing of Frozen Bacteria</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Lead ion solution*2*5μl (10<SUP>3<SUP>, 10<SUP>-2</SUP>, 10<SUP>-5</SUP>, 10<SUP>-6</SUP>ppm)</SUP></SUP></P><P class="Labnote">LB*14*500μl</P><P class="Labnote">Ampicillin*7μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Culture the frozen bacteria in different concentrations of lead ion for 8 hours.</P><P class="Labnote">2.Prepare the blank set with LBand antibiotic.</P><P class="Labnote">3.Test the fluorescence intensity of the sample.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">From the table above, we sought that the fluorescence of bacteria did not correspond to the concentration of lead ion, which is contrary to what we tested before. Thus, we assumed that the frozen bacteria might somehow be contaminated or degraded. But, I guessed that there might be something wrong with the construct of the plasmid, or maybe the test results we did before was simply a coincidence. So, I would try to recheck the sequence that we ran before to check whether the frames have shifted. For now, we decided to re-transform the plasmid into a fresh group of E. coli, and hope to test the function again.</P><P class="titleTwo" id="160725" style="line-height:120%">2016/07/25</P><P class="Topic">Topic: Mini Extraction, Gel Electrophoresis, Liquid Culture, Making TAE Buffer</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Mini Extraction Kit*1</P><P class="Labnote">0.8% Agarose*49ml</P><P class="Labnote">50X TAE Buffer*1ml</P><P class="Labnote">ddH<SUB>2</SUB>O</P><P class="Labnote">Blue Dye*12μl</P><P class="Labnote">EDBT</P><P class="Labnote">LB*4ml</P><P class="Labnote">Ampicillin*4μl</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">(For mini extraction)</P><P class="Labnote">　1.Add 1.5ml of bacteria into an Eppendorf.</P><P class="Labnote">　2.Centrifuge the sample at 12000rcf for 1 minute.</P><P class="Labnote">　3.Remove excess fluidfrom above the tube.</P><P class="Labnote">　4.Add 200μl of MX1 buffer onto the pellet. Vigorously vortex the sample.</P><P class="Labnote">　5.Add 250μl of MX2 buffer into the mixture. Shake gently by rotating your hands.</P><P class="Labnote">　6.Place the sample for 2~5 minutes at room temperature.</P><P class="Labnote">　7.Add  350μl  of  MX3  buffer.  Shake  the  sample  immediately, there will appear to be a white cloudy suspension floating above the tube.</P><P class="Labnote">　8.Centrifuge the sample at 10000rcf for 5~10 minutes.</P><P class="Labnote">　9.Drip thefluid through a mini Plustmcolumn carefully. Do not damage the film between. Discard the pallet beneath.</P><P class="Labnote">　10.Centrifuge the column at 7000rcf for approximately 1 minute.</P><P class="Labnote">　11.Remove excess fluid inside the tube below.</P><P class="Labnote">　12.Drip 500μl of WN buffer through the film.</P><P class="Labnote">　13.Centrifuge the sample at 7000rcf for approximately 1 minute.</P><P class="Labnote">　14.Remove excess fluid inside the tube below.</P><P class="Labnote">　15.Drip 700μlof WS buffer through the film.</P><P class="Labnote">　16.Centrifuge the sample at 7000rcf for approximately 1 minute.</P><P class="Labnote">　17.Remove excess fluid inside the tube below. </P><P class="Labnote">　18.Centrifuge  again  at  10000rcf  for  3  minutes  to  get  rid  of remaining ethanol in the pallet.</P><P class="Labnote">　19.Change the tube of the column into an Eppendorf.</P><P class="Labnote">　20.Add 50μlof Elution buffer.</P><P class="Labnote">　21.Wait for 3 minutes.</P><P class="Labnote">　22.Centrifuge  the  sample  at  10000rcf  for  3~5  minutes.  The remaining fluid will be the extraction of DNA plasmid.</P><P class="Labnote">　23.Either use the plasmid instantly or freeze the plasmid inside a 4°C freezer.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">We ran a gel electrophoresis to check whether we have successfully transformed our interested gene into our bacteria. So, we usedthe plasmid  we  transformed  to  serve  as  a  reference  to  the  extracted plasmid we got. The results are as the followingfigure 1. Although there is a long smear in the original plasmid, while our extracted plasmid seems clean-cut; the numbers of base pair are significantly near and can be assumed to be a successful transformation. However, since the plasmid was used 4 to 5 months ago, we cannot be sure if the plasmid has mutated or shifted frames. Thus, we decide to culture a colony  of  successfully  transformed  bacteria,  and  test  its  function tomorrow. If the results do not run out well, we will then try western blot to see if PbrR protein was ideally transcript. If that doesn’t run well either, we might choose to reconstruct our DNA. </P><P class="titleTwo" id="160726" style="line-height:120%">2016/07/26</P><P class="Topic">Topic: Fluorescence Spectrometry</P><P class="LabnoteTitle">Material:</P><P class="Labnote">LB (for blank)</P><P class="Labnote">Ampicillin (for blank)</P><P class="Labnote">Bacteria (k1613017, J23102)</P><P class="Labnote">Infinite M200 Pro (TECAN) fluorescence spectrometry</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Create  a  sterilized environment  by  lighting  fire  from  an alcohol lamp.</P><P class="Labnote">2.Prepare a series of Blank samples.</P><P class="Labnote">3.Test the optical density and fluorescence intensity, with an absorption wave length of 600nm; and an excitation length of 584nm and emission length of 607nm.</P><P class="LabnoteTitle">Results:</P><P class="Labnote">As in the table below, we can see that the experiment sampledid not differ with the negative control, which has no lead ion in the solution. Also, the sample will E. coli shows less fluorescence than our blank sample, containing only LB. Furthermore, the fluorescence of the bacteria did not grow strong overtime. Thus, we decided to culture a longer time, and if that does not prove to be well, we shall run a western blot to make sure that our bacteria is capable of producing PbrR protein.</P><P class="titleTwo" id="160817" style="line-height:120%">2016/08/17</P><P class="Topic">Protocol E.coli growth rate inpbsolution</P><P class="LabnoteTitle">Objective:</P><P class="LabnoteTitle">Material:</P><P class="Labnote">Transformed DH5α pcDNA (ctrl)on SOCagar plates</P><P class="Labnote">SOC broth</P><P class="Labnote">Amp (1000 X)</P><P class="Labnote">15 ml tube、50 ml tube</P><P class="Labnote">96 well plate</P><P class="Labnote">Eppendorf</P><P class="Labnote">Lead solution</P><P class="Labnote">Pipette(1000ul、200ul、20ul)</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">Day 1</P><P class="Labnote">　1.Culture 4 tubes of DH5α(lead biosensor) in 15ml centrifuge tubeovernight: 2ml SOC with 2ulrpm:110)</P><P class="Labnote">　2.Prepare 1000ppm, 100ppm, 50ppm, 20ppm, 10ppm, 5ppmlead solution by serial dilution</P><P class="Labnote">Day 2</P><P class="Labnote">　1.Preparesix 15ml centrifugetube, each containing 1ml of SOC, and 2ul Amp.</P><P class="Labnote">　2.Add 1mlofovernight culturedbacteria liquidinto each 15ml centrifuge tube,which makes all of them contain 1ml of SOC, and 2ul Amp, and 1ml bacteria liquid</P><P class="Labnote">　3.Culture them for another two hours to make sure if they are able to grow.</P><P class="Labnote">　4.Prepare five kinds of concentration oflead solution, 10μg/ml, 3μg/ml, 1μg/ml, 0.3μg/ml, 0.1μg/mlby serial dilution.(Which is prepared at day 1)</P><P class="Labnote">　5.Prepare eighteen50ml centrifungal, each contains 29.1ml of SOC, 0.6ml of bacteria liquid, and 0.3ml lead solution(with five kinds of solution).</P><P class="Labnote">　6.Examine the OD of each tube every 30 minutes</P><P class="titleTwo" id="160823" style="line-height:120%">2016/08/23</P><P class="Topic">Topic:Transform Pb plasmid</P><P class="LabnoteTitle">Material:</P><P class="Labnote">E. coli(DH5alfa)</P><P class="Labnote">Pb<SUP>2+</SUP>plasmid</P><P class="Labnote">SOC</P><P class="Labnote">Chloramphenicol(antibiotic)</P><P class="Labnote">Pipette</P><P class="Labnote">Eppendorf</P><P class="Labnote">15ml centrifuge tube</P><P class="Labnote">37°C incubator shaker</P><P class="Labnote">Agardish</P><P class="Labnote">Ice bucket</P><P class="Labnote">37°C shaker</P><P class="LabnoteTitle">Procedure:</P><P class="Labnote">1.Take out 60ul of E.coli DH5a from the -80 fridge and the plasmid in the -20 fridge</P><P class="Labnote">2.Thaw  the  E.coli  DH5a  in  a  ice  bucket  until  it  becomes transparent</P><P class="Labnote">3.Add  1ul  plasmid  liquid  and  20ul  DH5a  liquid  into  a eppendorf</P><P class="Labnote">4.Put it in a ice bucket for 5 minutes</P><P class="Labnote">5.Heat shock the E.coli in 37 shaker for 3 minutes.</P><P class="Labnote">6.Put it in a ice bucket for 2 minutes(recovery)</P><P class="Labnote">7.Add 200ul SOC into each eppendorf</P><P class="Labnote">8.Put the eppendorfs into 37°C shaker for 4 hours.</P><P class="Labnote">9.Culture the baceteria in the eppendorfs into agar plates.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>