<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Chalmers-Gothenburg/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Chalmers-Gothenburg/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Chalmers-Gothenburg/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Chalmers-Gothenburg/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav"><LI><A href="/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Team">Who we are</A></LI><LI><A href="https://igem.org/Team.cgi?id=753">Official team page</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Contacts">Contact Us</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Biodetection_of_hCG">Project description</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Methods">Methods</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Results">Results</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Discussion">Discussion and future perspectives</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Modelling">Modelling</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Notebook">Notebook</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Safety_page">Safety</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Media">Media</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Education">Education</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Collaboration">Collaboration</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI></DIV><DIV class="box2"><DIV align="left" style="margin-left:10px; margin-right:10px; "><B><FONT size="9"><B>Sponsors</B></FONT></B></DIV></DIV><DIV align="justify" style="width:auto; margin-left:0px; margin-right:300px;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Results_Summary"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Results Summary</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Survival_of_yeast_in_urine"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Survival of yeast in urine</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Deletion_of_CWP2_gene"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Deletion of CWP2 gene</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Gene_Deletion_according_to_the_Bipartite_method"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Gene Deletion according to the Bipartite method</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Colony_PCR"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Lyticase_assay"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Lyticase assay</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Introduction_of_genes_for_indigo-synthesizing_enzymes"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Introduction of genes for indigo-synthesizing enzymes</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Construction_of_pIndigo"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Construction of pIndigo</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Test_with_tryptophan_added_to_media"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Test with tryptophan added to media</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Enzymatic_assay_of_tryptophanase"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Enzymatic assay of tryptophanase</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Test_with_indole_added_to_media"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Test with indole added to media</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Day1"><SPAN class="tocnumber">4.4.1</SPAN><SPAN class="toctext">Day1</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Day2"><SPAN class="tocnumber">4.4.2</SPAN><SPAN class="toctext">Day2</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Day3"><SPAN class="tocnumber">4.4.3</SPAN><SPAN class="toctext">Day3</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Absorption_spectra"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Absorption spectra</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Construction_of_pFIG1-Indigo"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">Construction of pFIG1-Indigo</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Expression_of_the_gene_for_the_LH.2FCG_receptor_in_yeast"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Expression of the gene for the LH/CG receptor in yeast</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Obtaining_and_cloning_the_yLHCGR_and_the_hLHCGR_gene"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Obtaining and cloning the yLHCGR and the hLHCGR gene</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Sequencing"><SPAN class="tocnumber">5.1.1</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Evaluating_the_functionality_of_the_receptor"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Evaluating the functionality of the receptor</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Initial_tests:_Screening_for_GFP_expression"><SPAN class="tocnumber">5.2.1</SPAN><SPAN class="toctext">Initial tests: Screening for GFP expression</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Final_test_of_the_completed_system:_screening_for_color_change"><SPAN class="tocnumber">5.2.2</SPAN><SPAN class="toctext">Final test of the completed system: screening for color change</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Results_Summary"><B>Results Summary</B></SPAN></H1><P>Spot test analysis and growth measurements of yeast in urine were performed in order to test the survival of yeast cells in urinary medium. These initial experiments showed that yeast cells are able to survive in urinary medium which is an important prerequisite for the biosensor. 
</P><P>The gene CWP2, encoding a mannoprotein in the cell wall was successfully deleted in the yeast strain IMFD-73. The deletion was confirmed by verification PCR. In addition, a lyticase assay showed that [IMFD-73 Δ<I>cwp2</I>::kanMX] was degraded faster than IMFD-73 and this leads us to the conclusion that the cell wall is somewhat weakened in the strains with the deleted cell wall mannoprotein CWP2. 
</P><P>Two different plasmids both containing the receptor gene LHCGR but one with the human signal peptide and the other one with a yeast signal peptide was created. Both plasmids were successfully cloned into IMFD-73 and [IMFD-73 Δ<I>cwp2</I>::kanMX] but in neither strains could the receptor be proved as functional in detection of hCG.
</P><P>The Indigo group managed to obtain different colour changes in different growth media by introducing a plasmid containing the genes tnaA and fmo (pIndigo). For instance, blue bubbles could be seen in one medium and other media turned into different shades of brown. These changes of colour were strictly dependent on the presence of pIndigo and could not be detected in any of the cultures with the strain containing the control plasmid. This indicates an at least partial activity of the pathway. However, we did not manage to confirm the presence of bio-indigo and to determine exactly which compound or reaction that was responsible for the colourful bubbles or colour changes. 
</P><P>[IMFD-73 Δ<I>cwp2</I>::kanMX] was successfully transformed with both receptor genes and genes required for bio-indigo production. However, the system was never functional as a pregnancy test kit since it did not give any significant response in the presence of hCG. More detailed analysis of the results are shown in the section below and the discussion can be found <A href="https://2012.igem.org/Team:Chalmers-Gothenburg/Discussion" class="external text" rel="nofollow">here</A>.
</P><H1><SPAN class="mw-headline" id="Survival_of_yeast_in_urine"><B>Survival of yeast in urine</B></SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:232px;"><DIV class="thumbcaption">Figure 1: Growth curve for cells in urine and YPD medium. The OD remains the same in the urine i.e. the cells are not growing.</DIV></DIV></DIV>
An initial test was performed in order to test the survival of yeast cells in urine, which is an important property of our biosensor if it should function as a pregnancy test kit. Pre-cultures of the lab yeast strain CEN.PK 113-11C were prepared in 5 ml YPD and grown O/N at 30 °C. The day after, the cells were centrifuged, resuspended in 40 ml YPD and 40 ml filter sterilized urine respectively and grown in shake flasks at 30°C. OD<SUB>600</SUB> measurements were taken at two different points of time during the day. The growth curve can be seen in Figure 1. The cells in urine did not grow; the OD remained the same during the whole <DIV class="thumb tright"><DIV class="thumbinner" style="width:232px;"><DIV class="thumbcaption">Figure 2: Spot test analysis of cells grown in YPD and urine for 4h. The cells survive in the urine and grow normal after being plated on a YPD plate.</DIV></DIV></DIV> day, while the cells in YPD were proliferating. 
<P>
In order to test if the cells remained alive in the urin, 300 µl of each culture (after 4h) were taken and OD<SUB>600</SUB> was adjusted to 0.5. The samples were diluted 3x, 9x, 27x and 81x and 10 µl aliquots of each dilution were spotted on an YPD plate (Figure 2). The cells from both the YPD and the urine media could grow, this means that the cells survived 4 h in urine and grew normal after being spotted on a YPD plate. Summarizing, yeast cells are unable to proliferate in the urine medium but do survive under these conditions.
</P><H1><SPAN class="mw-headline" id="Deletion_of_CWP2_gene"><B>Deletion of <I>CWP2</I> gene</B></SPAN></H1><P>One task of our iGEM project was the deletion of the <I>CWP2</I> gene, which is encoding a cell wall mannoprotein. By removing it, we aimed for higher cell wall permeability and thus enhanced chances of our ligand hCG to pass the cell wall and to bind to the membrane-bound receptor.
</P><H2><SPAN class="mw-headline" id="Gene_Deletion_according_to_the_Bipartite_method"><B>Gene Deletion according to the Bipartite method</B></SPAN></H2><P>The gene deletion was performed according to the bipartite method.  The results from the first PCR reactions, in which we amplified the overlapping fragments, can be seen in Figure 3. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 3: A) A schematic illustration of the first four PCR reactions. The 5' and 3' end of the kanMX (kanamycin resistance) cassette, flanked to loxP sites and available in vector pUG6h, were amplified by PCR (PCR reaction 1 and 2). In addition, 500 bp of the up- and downstream sequence of the <I>CWP2</I> gene were amplified from genomic DNA using primers with 5' extensions that are complementary to the loxP sites as indicated with lines above. B) The results of the four PCR reactions on gel. The expected sizes of the kanMX and <I>CWP2</I> fragments are about 1000 bp and 500 bp respectively which correspond to the sizes observed on gel.</DIV></DIV></DIV></DIV><P>The next step was to fuse together the overlapping fragments. The results of the fusion PCR are shown in the Figure 4. By transforming yeast with these fragments, homologous recombination should occur which will lead to the exchange of the <I>CWP2</I> gene with the kanMX cassette.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 4: A) A schematic illustration of the fusion PCR, the second step in the bipartite method. The <I>CWP2</I> upstream fragment was fused with the 5’ end of the kanMX cassette and the <I>CWP2</I> downstream fragment was fused with the 3’ end of the kanMX cassette.  B) The results of the fusion PCR on gel. Two samples (A and B) of each PCR reaction are shown. The expected size of both fusion fragments is about 1500 bp which corresponds to the sizes observed on gel (indicated with an arrow).</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Colony_PCR"><B>Colony PCR</B></SPAN></H2><P>Yeast strain IMFD-73 was transformed with 150 ng of each fragment and grown on G418 plates. Colonies could be observed, which were purified, and colony PCR was then performed in order to test whether the kanMX cassette was integrated randomly in the genome or had replaced the <I>CWP2</I> gene as desired. PCR was run with a forward primer outside of the inserted region and reverse primers within the <I>CWP2</I> gene or kanMX cassette respectively. As a control, a wild-type strain and a Δ<I>cwp2</I> strain from a deletion library were also tested. The results of the PCR reactions indicated that the deletion was correct (see Figure 5).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Figure 5: A) Schematic illustration of the colony PCR. A forward primer was designed to bind upstream in the genomic DNA of the 500 bp region that had been used in the construction of the bipartite fragment. Two reverse primers were designed to hybridize within the <I>CWP2</I> gene and in the kanMX cassette respectively. The primers are indicated with arrows. B) The results of the PCR reactions on gel. In every reaction, the cwp2-fw primer but different reverse primers were used. 1: The Δ<I>cwp2</I> deletion strain (bipartite) with the kanMX-rev primer. 2: The Δ<I>cwp2</I> deletion strain (from deletion library) with the kanMX-rev primer. 3: The Δ<I>cwp2</I> deletion strain (bipartite) with the cwp2-rev primer. 4: The Δ<I>cwp2</I> deletion strain (from deletion library) with the cwp2-rev primer. 5. A wild type yeast strain with the cwp2-rev primer. Since the <I>CWP2</I> gene is missing and the kanMX cassette is inserted instead, bands are expected when PCR is run with the kanMX-rev primer (1, 2, expected size 1800 bp) but no bands should be observed when PCR is performed with the cwp2-rev primer (3, 4). On the contrary, a clear band of about 800 bp should be observed when running PCR with the wild type strain and the cwp2-rev primer. This corresponds to the result on gel which indicates that the <I>CWP2</I> deletion was successful.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Lyticase_assay"><B>Lyticase assay</B></SPAN></H2><P>A lyticase assay was performed primarily in order to check the activity of lyticase. Another goal was to compare the rapidity of the cell wall degradation between the IMFD-73 and the [IMFD-73 Δ<I>cwp2</I>::kanMX] strain. Lyticase was added to the cells and the cell wall degradation (i.e. protoplast formation) could be displayed by adding SDS and then measuring OD at 600 nm (SDS leads to lysis of protoplasts, while cells with intact cell wall remain unaffected). SDS was added and OD values were taken at different time points. The two graphs (Figure 6 and 7) below show the decrease of the amount of intact cells in % over a time period of 1h with two different amounts of lyticase. The percentages of intact cells were calculated as OD<SUB>600</SUB>(t=x)/OD<SUB>600</SUB>(t=0)*100.  One can observe that a slightly smaller amount of cells of the deletion strain were intact between 30 and 60 minutes (except for one outlier at 50 min) which means that the cell wall degraded quicker in the [IMFD-73 Δ<I>cwp2</I>::kanMX] strain. This lead to the conclusion that the cell wall is somewhat weakened in the strains with the deleted cell wall mannoprotein gene <I>CWP2</I>. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 6: Assessment of cell wall degradation by lyticase (50 µl enzyme solution). Prior to the each OD measurement, SDS was added. The percentage of intact cells was calculated as OD<SUB>600</SUB>(t=x)/OD<SUB>600</SUB>(t=0)*100.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 7: Assessment of cell wall degradation by lyticase (20 µl enzyme solution). Prior to the each OD measurement, SDS was added. The percentage of intact cells was calculated as OD<SUB>600</SUB>(t=x)/OD<SUB>600</SUB>(t=0)*100.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Introduction_of_genes_for_indigo-synthesizing_enzymes"><B>Introduction of genes for indigo-synthesizing enzymes</B></SPAN></H1><P>Another big tasks in this project was the introducing of two enzymes needed for indigo synthesizes in yeast. In the following sections the construction of the plasmid for indigo production and the testing of the same will be presented.
</P><H2><SPAN class="mw-headline" id="Construction_of_pIndigo"><B>Construction of pIndigo</B></SPAN></H2><P>To enable production of indigo in yeast, two genes, <I>tnaA</I> and <I>fmo</I>, had to be cloned into the yeast strain. In order to achieve that, a plasmid containing both genes under the control of two different promoters had to be constructed. First we used PCR to amplify the genes. <I>TnaA</I>, encoding the enzyme tryptophanase, was amplified from <I>E. coli</I> genomic DNA by colony-PCR and fmo, encoding the flavin-containg monooxygenase, was amplified by PCR from plasmid kindly provided to us by Si Wouk Kim from Chosun University in South Korea. After a few attempts both genes were successfully amplified. TnaA and fmo were digested with <I>Bam</I>HI and <I>Xho</I>I and <I>Not</I>I and <I>Pac</I>I respectively and the genes were separately cloned into pIYC04 which had been digested in the same way as respective gene. The plasmids were now called pIYC04-tnaA and pIYC04-fmo. After the ligation <I>E. coli</I> DH5α cells were transformed with pIYC04-tnaA and pIYC04-fmo respectively.  Both transformations were successful and the plasmids were purified. 
</P><P>Digested <I>tnaA</I> was ligated with pIYC04-fmo, and digested <I>fmo</I> was ligated with pIYC04-tnaA and the plasmids were now called pIndigo. The reason that pIndigo was constructed in both ways was in order to increase the chances of getting a functional plasmid. DH5α were transformed with pIndigo. The amplification was successful and pIndigo was purified. pIndigo now contained <I>tnaA</I> under control of the constitutive <I>PGK1</I> promoter and <I>fmo</I> under the control of the constitutive <I>TEF1</I> promoter. The plasmid was verified by sequencing (data not shown). Sequencing showed that the plasmid contained both <I>tnaA</I> and <I>fmo</I> and there were no signs of mutations.
The yeast strain CEN.PK 113-11C was transformed with pIndigo and the presence of the plasmid should theoretically enable the cells to produce indigo, but no blue cells were seen on the plates. To see if the <I>tnaA</I> and the <I>fmo</I> gene had been correctly introduced into the cells, a colony-PCR with a following gel electrophoresis was performed for the genes. The expected bands were approximately 1400 bp for both genes. As one can see on the gel picture in Figure 8, bands with right lengths were present for both genes. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 8: Verification of transformation of CEN.PK 113-11C with pIndigo through gel electrophoresis. For both tnaA and fmo was one band expected to be seen with a length of approximately 1400 bp. As one can see on gel picture, bands with right lengths were present for both genes.</DIV></DIV></DIV></DIV><P>After the CEN PK 113-11C had been transformed with the pIndigo the cells were transferred to SD His- plates along with two controls consisting of CEN PK 113-11C without the plasmid and CEN PK 113-11C with empty plasmid. The control cells that lacked the plasmid were not able to grow on the plates, while the transformed ones did, so the transformation was considered to be successful. After more than one week of cultivation in room temperature and 30 ºC no blue cells were detected. After some time the cells were also transferred to fresh SD His- plates, and further cultivated, but this did not give blue cells on the plate. The strains were also cultivated in SD His- media in room temperature, but no blue colour was obtained.
</P><H2><SPAN class="mw-headline" id="Test_with_tryptophan_added_to_media"><B>Test with tryptophan added to media</B></SPAN></H2><P>The CEN PK 113-11C + pIndigo strain was cultivated in liquid media containing tryptophan, the precursor from which Indigo is produced, to see if that would enable the cells to produce indigo. The media volumes were 45 ml and consisted of; 6.9 g/l YNB, 20 g/l glucose, 770 mg/l CSM and X mg/l L-tryptophan. Three different media were prepared and X for these were; 50 mg/l, 100 mg/l respectively 1000 mg/l. The cells had been precultivated overnight in 5 ml SD His- media the main cultures were cultivated in room temperature. After one day of cultivation, the colours of the media in the flasks had not turned blue but interestingly into different shades of brown (data not shown).
</P><H2><SPAN class="mw-headline" id="Enzymatic_assay_of_tryptophanase"><B>Enzymatic assay of tryptophanase</B></SPAN></H2><P>In order to investigate the activity of the <I>tna</I> gene product, tryptophanase, an enzymatic assay was performed. It was focused on the reaction between tryptophan and the enzyme tryptophanase, which forms indole. The general idea was to measure the decrease of tryptophan in a reaction mixture that contained cell free extract of CEN PK 113-11C + pIndigo. The concept of the enzyme essay was to follow the tryptophan concentration by measuring the absorbance at 280 nm.  The first attempt failed due to malfunctional spectrophotometer. Two more attempts were performed for the enzymatic assay but the assays did not succeed. The OD was measured for some different concentrations of pure tryptophan solutions and they showed that there were only slight differences in absorbance, even with a 20 time dilution and this was probably the reason that the assay did not work. 
</P><H2><SPAN class="mw-headline" id="Test_with_indole_added_to_media"><B>Test with indole added to media</B></SPAN></H2><P>A mayor experiment with 20 different cultivations was prepared, in order to study the indigo production of CEN PK 113-11C + pIndigo. The cultivation media had different compositions. In ten flasks the base of the media was 45 ml SD His- and 2% glucose, whilst in the other ten it was 45 ml SD His-, 2% galactose and 0.2% glucose. Amongst these subgroups of ten, 50% of the cultivations were made with CEN PK 113-11C + pIndigo and the others were the control. The cell culture used for the control was CEN PK 113-11C + pIYC04, where pIYC04 is an empty plasmid (negative control). In these subgroups of five, 5 ml L-tryptophan 20 g/l was added to one of the flasks, 5 ml Indole 2 g/l to another, 5 ml L-tryptophan 20 g/l to another after 24 hours of cultivation, 2 drops Indole 2 g/l to another after 24 hours of cultivation and the last flask did not receive an additional chemical. 
</P><P>Two different carbon sources were chosen, since glucose had previously been shown to inhibit tryptophanase. Tryptophan was added as pathway precursor, indole as pathway intermediate in case the monooxygenase, but not the tryptophanase was active.
</P><H3><SPAN class="mw-headline" id="Day1"><B>Day1</B></SPAN></H3><P>During day 1 no major changes in colour were observed for the samples (see Figure 9 and 10). All cultivations had a foggy yellow colour. The samples with added L-tryptophan had initially a slight darker shade of yellow.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 9: . Cultivations in galactose media day 1, all cultivations were made with the strain CEN PK 113-11C . The order of the cultivations is, from left to right; pIndigo, pIYC04, pIndigo with 5 ml L-tryptophan 20 g/l added to medium, pIYC04 with 5 ml L-tryptophan 20 g/l added to medium, pIndigo with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIYC04 with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIndigo with 5 ml indole 2 g/l added to medium, pIYC04 with 5 ml indole 2 g/l added to medium, pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation and pIYC04 with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 10: . Cultivations in glucose media day 1, all cultivations were made with the strain CEN PK 113-11C . The order of the cultivations is, from left to right; pIndigo, pIYC04, pIndigo with 5 ml L-tryptophan 20 g/l added to medium, pIYC04 with 5 ml L-tryptophan 20 g/l added to medium, pIndigo with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIYC04 with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIndigo with 5 ml indole 2 g/l added to medium, pIYC04 with 5 ml indole 2 g/l added to medium, pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation and pIYC04 with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Day2"><B>Day2</B></SPAN></H3><P>At 9 am on day 2, see Figure 11 and 12, some changes had happened overnight. The culture containing CEN PK 113-11C + pIndigo with 5 ml L-tryptophan 20 g/l added to medium had turned brownish for both galactose and glucose medium. This effect is something that has occurred several times for this type of cultivation after about 24 hours. Another short remark that can be done about these pictures is that the cultivations with the initially added indole are quite clear, an indication that it is harder for the cells to survive in the chosen concentration of indole. In the flasks that indole was added on day 2, CEN PK 113-11C + pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation and CEN PK 113-11C + pIYC04 with 2 drops of indole 2 g/l added after 24 hours of cultivation, the solution turned slightly darker after indole addition. Finally it can be said that the result for media containing glucose respectively galactose, after 24 hours, were similar and the controls were unchanged.
</P><P>At 4 pm on day 2, see Figure 11-13, all colours were preserved from the morning except for the cultivations of CEN PK 113-11C + pIndigo with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation and CEN PK 113-11C + pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation. They had turned brown for the cultivations in galactose, and green for the ones in glucose.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 11: Cultivations in galactose media day 2, all cultivations were made with the strain CEN PK 113-11C . The order of the cultivations is, from left to right; pIndigo, pIYC04, pIndigo with 5 ml L-tryptophan 20 g/l added to medium, pIYC04 with 5 ml L-tryptophan 20 g/l added to medium, pIndigo with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIYC04 with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIndigo with 5 ml indole 2 g/l added to medium, pIYC04 with 5 ml indole 2 g/l added to medium, pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation and pIYC04 with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 12: Cultivations in glucose media day 2, all cultivations were made with the strain CEN PK 113-11C . The order of the cultivations is, from left to right; pIndigo, pIYC04, pIndigo with 5 ml L-tryptophan 20 g/l added to medium, pIYC04 with 5 ml L-tryptophan 20 g/l added to medium, pIndigo with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIYC04 with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIndigo with 5 ml indole 2 g/l added to medium, pIYC04 with 5 ml indole 2 g/l added to medium, pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation and pIYC04 with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 13: Comparison of cultivation flasks at 16.00 day 2. Cultivations flaks at back row contained galactose media and front row flasks contained glucose media. All cultivations were made with the strain CEN PK 113-11C. Both rows, the flasks are arranged in the same order. From left to right; pIndigo, pIYC04, pIndigo with 5 ml L-tryptophan 20 g/l added to medium, pIYC04 with 5 ml L-tryptophan 20 g/l added to medium, pIndigo with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIYC04 with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation, pIndigo with 5 ml indole 2 g/l added to medium, pIYC04 with 5 ml indole 2 g/l added to medium, pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation and pIYC04 with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><P>Interestingly, when the two flasks containing CEN PK 113-11C + pIndigo with 2 drops of indole 2 g/l added after 24 hours of cultivation in glucose medium and CEN PK 113-11C + pIndigo in galactose medium were shaken blue bubbles occurred in both flasks (see Figure 14). The bubbles were pipetted from the flasks and were examined under microscope. In between the cells big blue sheets bigger than the cells were observed but we could not figure out what the bubbles consisted of. When the flask was shaken again, new blue bubbles were formed.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 14: CEN PK 113-11C + pIndigo in glucose medium with 2 drops of indole 2 g/l added after 24 hours of cultivation and CEN PK 113-11C + pIndigo in galactose medium, both flasks were shaken blue bubbles occurred in both flasks.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Day3"><B>Day3</B></SPAN></H3><P>The following pictures show a detailed comparison of the different cultures after 3 days of cultivation. What can be concluded from these results is:
1. A clear colour change was observed in several cultures, which was strictly dependent on the presence of pIndigo and could not be detected in any of the cultures with the strain containing the control plasmid (e.g. comparison figure 17 and 18). This indicates an at least partial activity of the pathway.
2. Clear colour changes only occurred when pathway precursors (tryptophan) or intermediates (indole) were added to the medium (e.g. comparison figure 15 and 17) assuming that they represent a limiting factor in the non-supplemented cultures.  
3. A colour difference could be seen between cells grown in glucose and galactose, respectively (figures 19 and 23), which might however be due to differences in cell density.
4. Although the pIndigo cultures showed a colour change none of them exhibited the expected blue colour typical of indigo. We therefore assume that the detected colours we caused by not yet identified pathway intermediates or by-products.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 15: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIndigo in glucose medium. Right flask: CEN PK 113-11C + pIndigo in galactose medium.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 16: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIYC04 in glucose medium. Right flask: CEN PK 113-11C + pIYC04 in galactose medium.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 17: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIndigo in glucose medium with 5 ml L-tryptophan 20 g/l added to medium. Right flask: CEN PK 113-11C + pIndigo in galactose medium with 5 ml L-tryptophan 20 g/l added to medium.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 18: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIYC04 in glucose medium with 5 ml L-tryptophan 20 g/l added to medium. Right flask: CEN PK 113-11C + pIYC04 in galactose medium with 5 ml L-tryptophan 20 g/l added to medium.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 19: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIndigo in glucose medium with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation. Right flask: CEN PK 113-11C + pIndigo in galactose medium with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 20: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIYC04 in glucose medium with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation. Right flask: CEN PK 113-11C + pIYC04 in galactose medium with 5 ml L-tryptophan 20 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 21: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIndigo in glucose medium with 5 ml indole 2 g/l added to medium. Right flask: CEN PK 113-11C + pIndigo in galactose medium with 5 ml indole 2 g/l added to medium. Not that the flasks are incorrectly labelled, the concentration of indole should be 0.2 g/L instead of 2 g/L.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 22: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIYC04 in glucose medium with 5 ml indole 2 g/l added to medium. Right flask: CEN PK 113-11C + pIYC04 in galactose medium with 5 ml indole 2 g/l added to medium. Not that the flasks are incorrectly labelled, the concentration of indole should be 0.2 g/L instead of 2 g/L.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 23: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIndigo in glucose medium with 2 drops of indole 2 g/l added after 24 hours of cultivation. Right flask: CEN PK 113-11C + pIndigo in galactose medium with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 24: Comparision of cultivation flasks at 13.30 day 3. Left flask: CEN PK 113-11C + pIYC04 in glucose medium with 2 drops of indole 2 g/l added after 24 hours of cultivation. Right flask: CEN PK 113-11C + pIYC04 in galactose medium with 2 drops of indole 2 g/l added after 24 hours of cultivation.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Absorption_spectra"><B>Absorption spectra</B></SPAN></H2><P>In order to try to determine what could have caused the colour change, absorption spectra between 250-700nm were determined for supernatants of cultures from different growth conditions: 
(1) CEN PK 113-11C + pIndigo in glucose media with 2 g/L L-tryptophan added after 24 hours: yellow-brown colour, (2) 1:4 dilution of CEN PK 113-11C + pIndigo in glucose media with 2 g/L L-tryptophan added after 24 hours: yellow-brown colour, (3) CEN PK 113-11C + pIYC04 in glucose media with 2 g/L L-tryptophan added after 24 hours, (4) CEN PK 113-11C + pIndigo in galactose media with 2 g/L L-tryptophan added after 24 hours: orange-brown colour, (5) 1:4 dilution of CEN PK 113-11C + pIndigo in galactose media with 2 g/L L-tryptophan added after 24 hours: orange-brown colour, (6) CEN PK 113-11C + pIYC04 in galactose media with 2 g/L L-tryptophan added after 24 hours.
</P><P>For glucose containing media peaks could be seen at ~400nm and ~520nm while galactose containing media only resulted in dispersion spectrum, which can be seen in Figure 25. The dilutions resulted in the same absorption patterns as the respective undiluted supernatants.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 25: Absorbance spectra of supernatant from (1) CEN PK 113-11C + pIndigo in glucose media with 2 g/L L-tryptophan added after 24 hours,(2) 1:4 dilution of CEN PK 113-11C + pIndigo in glucose media with 2 g/L L-tryptophan added after 24 hours, (3) CEN PK 113-11C + pIYC04 in glucose media with 2 g/L L-tryptophan added after 24 hours, (4) CEN PK 113-11C + pIndigo in galactose media with 2 g/L L-tryptophan added after 24 hours: orange-brown colour, (5) 1:4 dilution of CEN PK 113-11C + pIndigo in galactose media with 2 g/L L-tryptophan added after 24 hours: orange-brown colour, (6) CEN PK 113-11C + pIYC04 in galactose media with 2 g/L L-tryptophan added after 24 hours.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Construction_of_pFIG1-Indigo"><B>Construction of pFIG1-Indigo</B></SPAN></H2><P>To make the system work as planned, with indigo production as an output signal for detection of hCG, the constitutive PGK1 promoter had to be exchanged with the inducible <I>FIG1</I> promoter, which is activated by the pheromone pathway. The <I>FIG1</I> promoter was amplified with yeast colony-PCR. After amplification, the <I>FIG1</I> PCR fragment and pIndigo were both digested with <I>Bam</I>HI and <I>Fse</I>I and were then ligated with each other. The new plasmid was now called pFIG1-Indigo. Transformation of <I>E. coli</I> DH5α cells with pFIG1-Indigo was successful and pFIG1-Indigo could afterwards be purified with. Purified pFIG1-Indigo was tested by sequencing but unfortunately the sequencing failed. The pFIG1-Indigo and then used to transform strains [IMFD-73 Δ<I>cwp2</I>::kanMX] + hLHCGR and [IMFD-73 Δ<I>cwp2</I>::kanMX] + yLHCGR. We managed to transform the strains with pFIG1-Indigo but the system was never proved to be functional. The <I>FIG1</I> promoter was then successfully cloned into a BioBrick which was verified by PCR and gel electrophoresis (data not shown).
</P><H1><SPAN class="mw-headline" id="Expression_of_the_gene_for_the_LH.2FCG_receptor_in_yeast"><B>Expression of the gene for the LH/CG receptor in yeast</B></SPAN></H1><P>Two different versions of the LH/CG receptor were to be evaluated and expressed in yeast during this project: one with the original human signal peptide fused to the receptor and one with the yeast Ste3 signal peptide fused to the receptor. Both versions were obtained using the gene construct illustrated in Figure 26. The construct was ordered from and synthesized by GenScript.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 26: A schematic illustration of the gene construct used to obtain the two different versions of the LHCGR gene: one fused with the native human signal peptide sequence and one fused with the yeast Ste3 signal peptide sequence.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Obtaining_and_cloning_the_yLHCGR_and_the_hLHCGR_gene"><B>Obtaining and cloning the yLHCGR and the hLHCGR gene</B></SPAN></H2><P>The yeast signal peptide version of the receptor gene, hereafter referred to as “yLHCGR”, was obtained with PCR using the original gene construct (Figure 26) and 5’ phosphorylated primers (lhcgr-rev and lhcgr-fw), as can be seen in Figure 27 A. The yLHCGR construct was then religated and when cut with restriction enzymes <I>Not</I>I and <I>Pac</I>I and run on a gel, the obtained band displayed a correct length of approximately 2100 bp (see Figure 27 C). The band was cut out and the yLHCGR gene was then ligated into the shuttle plasmid pSP-GM1 and amplified in <I>E. coli</I>.
</P><P>The human signal peptide version of the receptor gene, hereafter referred to as “hLHCGR”, was obtained by digesting the original gene construct with <I>Not</I>I and <I>Pac</I>I (Figure 27 B). Gel electrophoresis was run and the obtained fragment displayed a correct length of approximately 2100 bp (see Figure 27 C). The band was cut out, ligated into the pSP-GM1 plasmid and amplified in <I>E.coli</I>. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure 27: Cloning and isolating the yLHCGR and hLHCGR genes. A) A schematic illustration of how the yLHCGR gene was obtained by PCR using the original gene construct and 5’ phosphorylated primers. The resulting PCR product was religated and then digested with restriction enzymes NotI and PacI and run on a gel in order to verify that the correct product had been obtained. B) The hLHCGR gene was obtained by digesting the original gene construct with NotI and PacI. C) Picture from gel electrophoresis. (L)  GeneRuler™ 1 kb DNA Ladder, (1) Digested vector carrying yLHCGR, expected lengths: 2132bp (yLHCGR, has been cut out of the gel) and 2710bp (vector backbone), (2) Digested vector carrying hLHCGR, expected lengths: 2122 bp (hLHCGR, has been cut out of the gel) and 2710 bp (vector backbone), (3) Digested pSP-GM1 plasmid, expected length: 7404bp (has been cut out of the gel)</DIV></DIV></DIV></DIV><P><I>Saccharomyces cerevisiae</I> strains IMFD-70, IMFD-72, IMFD-73 and IMFD-74 were then successfully transformed with the yLHCGR and hLHCGR expression plasmids respectively. All these strains have been optimized for the coupling of human GPCRs to the pheromone signaling pathway and they differ in their expression of different G alpha proteins. IMFD-70 has the endogenous G alpha protein <A href="#Importance">[1]</A> while the other strains express different versions of yeast/human chimeric G alpha proteins. These chimeric proteins have had the five last amino acid residues of the C terminal end replaced with the corresponding human amino acids [Unpublished data]. IMFD-73 has a chimeric alpha subunit of such type that it should have the best chances of interacting with the LH/CG receptor and was therefore the strain that was mainly used during this project. All strains were kindly provided by the Kondo group at Kobe University, Japan. 
</P><H3><SPAN class="mw-headline" id="Sequencing"><B>Sequencing</B></SPAN></H3><P>In order to confirm that the yLHCGR gene and the hLHCGR gene not only have the correct length but also the correct sequence, the yLHCGR and the hLHCGR plasmid were sent in for sequencing.  Results indicated that the sequences were correct for both genes.
</P><H2><SPAN class="mw-headline" id="Evaluating_the_functionality_of_the_receptor"><B>Evaluating the functionality of the receptor</B></SPAN></H2><P>The ability of the receptor to detect hCG (i.e to bind the hormone and transmit the signal) when expressed in yeast, was evaluated several times under different conditions. Initial tests were performed by screening for fluorescence after the addition of hCG to the yeast cells. A final test also included screening for indigo production in the completed biosensor system.
</P><H3><SPAN class="mw-headline" id="Initial_tests:_Screening_for_GFP_expression"><B>Initial tests: Screening for GFP expression</B></SPAN></H3><P>The yeast strains that have been transformed with the gene for the LH/CG receptor have the gene for GFP placed under the control of the pheromone-induced FIG1 promoter. An initial test to see if the LH/CG receptor could detect hCG and activate the pheromone pathway, was therefore done by screening for fluorescence. Below, results from three of these screens are presented.
</P><P><B>IMFD-70, IMFD-72, IMFD-73 and IMFD-74, grown at 30°C</B></P><P>One experiment was carried out using all four, IMFD-70, -72, -73 and -74 strains, transformed with yLHCGR, hLHCGR and the empty pSP-GM1 plasmid respectively. Cells were cultivated at 30°C and were treated with lyticase in order to degrade the cell wall. 200 IU of hCG was then added to each strain. Screening for fluorescence was done using a fluorescence microscope. See Figure 28, 29, 30 and 31 for pictures that have been selected as representative of all results obtained during this experiment. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 28: Fluorescence microscopy of the IMFD-70 strain transformed with the yLHCGR plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 3 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 29: Fluorescence microscopy of the IMFD-72 strain transformed with the empty pSP-GM1 plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 3 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 30: Fluorescence microscopy of the IMFD-73 strain transformed with the yLHCGR plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 2 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 31: Fluorescence microscopy of the IMFD-74 strain transformed with the hLHCGR plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately3 hours after the addition of hCG.</DIV></DIV></DIV></DIV><P>Results obtained for the cells not treated with hCG and cells transformed with the negative control plasmid, indicate that there is relatively high level of background noise of GFP expressions in all IMFD strains. Generally, the fluorescence intensity is approximately equal between samples that have had the hCG hormone added to them and samples that did not have hCG added to them. It was therefore concluded that the LH/CG receptor, when expressed in these yeast strains and tested under the given conditions, is unable to detect hCG or at least that the receptor it is not functioning in a way that produces a sufficiently strong GFP output signal to be distinguished from the background noise. 
</P><P><B>IMFD-73, grown at room temperature</B></P><P>In a second test, the transformed IMFD-73 strain with either the yLHCGR, hLHCGR or the empty pSP-GM1 plasmid, were cultivated at room temperature. Gron wth at lower temperatures is often used in heterologous protein production since it may improve protein folding. Cells were then treated with cell wall-degrading enzyme lyticase and 200 IU of hCG was added. Results from the fluorescence microscope can be seen in Figure 32, 33 and 34.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 32: Fluorescence microscopy of the IMFD-73 strain transformed with the hLHCGR plasmid. Cells were grown at room temperature. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 2 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 33: Fluorescence microscopy of the IMFD-73 strain transformed with the empty pSP-GM1 plasmid. Cells were grown at room temperature. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 2 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 34: Fluorescence microscopy of the IMFD-73 strain transformed with the yLHCGR plasmid. Cells were grown at room temperature. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 2 hours after the addition of hCG.</DIV></DIV></DIV></DIV><P>Once again, high levels of background fluorescence were present. Results indicated that also when cells are grown at room temperature, the LH/CG receptor is unable to detect hCG or uncapable to produce an output signal that can be distinguished from the background noise in response to detecting hCG.
</P><P><B>IMFD-73-ΔCWP2, grown at 30°C</B></P><P>In a third test, the deletion strain IMFD-73-ΔCWP2 transformed with yLHCGR, hLHCGR and the empty pSP-GM1 plasmid respectively was used. The IMFD-73-ΔCWP strain potentially has a weakened cell wall. Cells were grown at 30°C and were not treated with lyticase before the addition of 200 IU hCG. Fluorescence microscope pictures can be seen in Figure 35, 36 and 37.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 35: Fluorescence microscopy of the IMFD-73 ΔCWP2 strain transformed with the hLHCGR plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 3 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 36: Fluorescence microscopy of the IMFD-73 ΔCWP2 strain transformed with the empty pSP-GM1 plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 3 hours after the addition of hCG.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 37: Fluorescence microscopy of the IMFD-73 ΔCWP2 strain transformed with the yLHCGR plasmid. Cells were grown at 30° C. GFP fluorescence is shown on the right. The upper panel shows cells that have not been treated with hCG. The lower panel shows cells that have had 200 IU hCG added to the sample. Photos are taken approximately 3 hours after the addition of hCG.</DIV></DIV></DIV></DIV><P>As previously, the yeast transformed with the LH/CG receptor gene and that had the hCG hormone added to them, did not produce an output signal that could be distinguished from the fluorescence intensity of cells that did not have hCG added to them or that did not contain the LH/CG gene.
</P><H3><SPAN class="mw-headline" id="Final_test_of_the_completed_system:_screening_for_color_change"><B>Final test of the completed system: screening for color change</B></SPAN></H3><P>One last test was performed using the completed biosensor system: IMFD-73-ΔCWP2+pFig-Indigo+hLHCGR or yLHCGR. The cultures were grown over night before adding hCG in the morning. 3 hours later more hCG was added.  From the colorometic assay (see &quot;Results: indigo&quot;), it was concluded that it takes approximately 7 hours for the pIndigo cells to produce a visible color change when grown at 30°C. However, 7 hours after adding hCG to the finalized system, no color change could be observed in any of the cultures (see Figure 38 and 39) and it was concluded that the biosensor did not succeed in detecting the hormone. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 38: Test of the finalized system. A) [IMFD-73+hLHCGR+pFig-Indigo]. hCG was added to the sample to the left, while sample to the left were left untreated. B) [IMFD-73+hLHCGR+pIYCO4]. hCG was added to the sample to the left, while sample to the left were left untreated.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure 39: Test of the finalized system. A) [IMFD-73+yLHCGR+pFig-Indigo]. hCG was added to the sample to the left, while sample to the left was untreated. B) [IMFD-73+yLHCGR+pIYCO4]. hCG was added to the sample to the left, while sample to the left was untreated.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="References"><B>References</B></SPAN></H1>[1] Togawa F, Ishii J,Ishikura A, Tanaka T, Ogino C, Kondo A. Importance of asparagine residues at positions 13 and 26 on the amino-terminal domain of human somatostatin receptor subtype-5 in signalling. The Journal of Biochemistry. 2010;147(6):867-873.
[2] Zhang M, Liang Y, Zhang X, Xu Y, Dai H, Xiao W. Deletion of yeast CWP genes enhances cell permeability to genotoxic agents. Toxicological sciences: an official journal of the Society of Toxicology. 2008;130(1):68-76.
[3] Ishii J, Moriguchi M, Hara KY, Shibasaki S, Fukuda H, Kondo A. Improved identification of agonist-mediated Gαi-specific human G-protein-coupled receptor signaling in yeast cells by flow cytometry. Analytical Biochemistry. 2012;426(2):129-133.
[4] Fukuda N, Ishii J, Kaishima M, Kondo A. Amplification of agonist stimulation of human G-protein-coupled receptor signaling in yeast. Analytical Biochemistry. 2011;417(2):182-187.
[5] Petäja-Repo UE, Merz WE, Rajaniemi HJ. Significance of the carbohydrate moiety of the rat ovarian luteinizing hormone/chorionic-gonadotropin receptor for ligand-binding specificity and signal transduction. The Biochemical Journal. 1993;292(3):839-844.
[6] Ladds G, Goddard A, Davey J. Functional analysis of heterologous GPCR signaling pathways in yeast. Trends in Biotechnology. 2005;23(7):367-373
</DIV><DIV id="footer" style="background-color:#FFFFFF; height:300px;"><P>With many thanks to our generous sponsors:
</P><DIV class="floatright"><A href="/File:GATC.gif" title="File:GATC.gif">File:GATC.gif</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Chalmers-Gothenburg/Results">http://2012.igem.org/Team:Chalmers-Gothenburg/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Chalmers-Gothenburg/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Chalmers-Gothenburg/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Results&amp;oldid=216469" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>