<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Chalmers-Gothenburg/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Chalmers-Gothenburg/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Methods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Chalmers-Gothenburg/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Chalmers-Gothenburg/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav"><LI><A href="/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Team">Who we are</A></LI><LI><A href="https://igem.org/Team.cgi?id=753">Official team page</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Contacts">Contact Us</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Biodetection_of_hCG">Project description</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Methods">Methods</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Results">Results</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Discussion">Discussion and future perspectives</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Modelling">Modelling</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Notebook">Notebook</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Safety_page">Safety</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Media">Media</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Education">Education</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Collaboration">Collaboration</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI></DIV><DIV class="box2"><DIV align="left" style="margin-left:10px; margin-right:10px; "><B><FONT size="9"><B>Sponsors</B></FONT></B></DIV></DIV><DIV align="justify" style="width:auto; margin-left:0px; margin-right:300px;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Methods"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Expression_of_the_LHCGR_gene_in_yeast_strain_IMFD-73"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Expression of the LHCGR gene in yeast strain IMFD-73</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Plasmid_construction"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Plasmid construction</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformation_of_IMFD-73_and_degradation_of_the_cell_wall"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Transformation of IMFD-73 and degradation of the cell wall</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Deletion_of_CWP2"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Deletion of CWP2</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Expression_of_monooxygenase_and_tryptophanase_genes"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Expression of monooxygenase and tryptophanase genes</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Plasmid_construction_and_transformation_of_a_yeast_lab_strain"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Plasmid construction and transformation of a yeast lab strain</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Replacement_of_the_PGK1_with_the_FIG1_promoter"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Replacement of the PGK1 with the FIG1 promoter</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Combination_of_the_components_to_a_functional_biosensor"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Combination of the components to a functional biosensor</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Methods"><B>Methods</B></SPAN></H2><P>The work required for the construction of the biosensor is divided into three main tasks: (1) expression of the LH/CG receptor, (2) knock-out of the <I>CWP2</I> gene encoding a cell wall mannoprotein and (3) expression of indigo synthesizing enzymes tryptophanase and monooxygenase. These tasks will be performed in parallel and in the end, the different parts will be combined into one system. The three main tasks are described in detail in the following sections. 
</P><H2><SPAN class="mw-headline" id="Expression_of_the_LHCGR_gene_in_yeast_strain_IMFD-73"><B>Expression of the LHCGR gene in yeast strain IMFD-73</B></SPAN></H2><P>The first task is to express the LH/CG receptor in yeast. The receptor contains several amino acid residues that are essential for efficient ligand binding, both in the extracellular and in the transmembrane domains. In addition, the C-terminus and the intracellular loop III are considered to bind to the associated Gα protein of the receptor, and thereby constitute an important part of the signal transduction <A href="#Ryu">[1]</A>. In order to ensure correct conformational change following ligand binding and to guarantee a proper signal transduction, the full-length LH/CG receptor will be expressed. The LHCGR gene will be inserted into the yeast strain IMFD-73 and be placed under the control of a strong, constitutive promoter to ensure constant expression. To enhance translation efficiency, a Kozak sequence (AAAACA) will also be inserted upstream of the LHCGR gene. In addition, a signal peptide will be added to the N-terminus of the receptor sequence to ensure transportation to the yeast cell membrane. 
</P><P>Previous research does not provide general guidelines for which signal peptide should be used when expressing human GPCRs in yeast. In this project, both the native signal of the LH/CG receptor, consisting of its 22 first amino acids <A href="#Dufau">[2]</A>, and a yeast signal peptide will be tested separately. The 5' end of the LH/CG receptor, which will be used as the native signal peptide, is originally glycosylated and glycosylation in yeast differs significantly from the one in human <A href="#Hamilton">[3]</A>. The native signal will, however, be tested even though it is not sure if glycosylation of these amino acid residues is required for correct transportation of the receptor to the membrane <A href="#Petaja">[4]</A>. In order to test a yeast signal, the idea was to fuse the signal peptide of Ste2, the receptor to be replaced, with the LH/CG receptor N-terminus. However, Ste2 lacks an identified signal sequence <A href="#Data">[5]</A>. Instead, the signal peptide from Ste3, the MATα type pheromone receptor, will be used. It consists of the first 25 amino acid residues of the receptor <A href="#Data">[5]</A>.
</P><H4><SPAN class="mw-headline" id="Plasmid_construction">Plasmid construction</SPAN></H4><P>The LHCGR gene construct will be ordered in a vector. Figure 1 provides a schematic illustration of this construct. The signal peptide of Ste3 and a Kozak sequence will be fused with a Kozak sequence at the 5' site of the native signal and the gene. In addition, <I>Not</I>I and one <I>Pac</I>I restriction sites will be added as indicated in Figure 1.
</P><P style="width:auto; margin-left:80px; margin-right:80px;"><FONT face="verdana"><SMALL><B> Figure 1: Schematic illustration of the LHCGR gene construct. </B> Using this construct enables expression of the LHCGR gene with each signal peptides separately. For the expression of the gene with the native signal, the vector can be digested directly with <I>Not</I>I and <I>Pac</I>I. When testing the expression with the yeast signal, PCR can be run with 5' phosphorylated primers indicated by arrows in order to exclude the native signal and its associated Kozak sequence and PacI restriction site. The resulting linear fragment can be religated.    </SMALL></FONT></P><P>The aim of using this particular construct is to enable expression of the LHCGR gene with both signal peptides separately. For the expression with the native signal, the vector containing the LHCGR gene will be digested directly with <I>Not</I>I and <I>Pac</I>I restriction enzymes. The fragments will be purified on a gel and then ligated into the corresponding sites of the expression plasmid pSP-GM1 with bidirectional promoters TEF1 and PGK1, a <I>URA3</I> marker gene and an ampicillin resistance gene. For expression of the LHCGR with the Ste3 signal peptide, PCR will be run with 5' phosphorylated primers indicated with arrows in Figure 1 (lhcgr-rev and lhcgr-fw). This will result in a linear fragment that lacks the native signal and its associated <I>Not</I>I site and Kozak sequence. This fragment will be purified on a gel, religated and amplified in <I>E. coli</I>. After that, it will be digested with <I>Not</I>I and <I>Pac</I>I restriction enzymes, column purified and then ligated into the expression plasmid pSP-GM1. 
</P><P>In both cases, the gene will be set under the control of the strong and constitutive TEF1 promoter. The resulting plasmids will be named pLHCGRn (native signal peptide) and pLHCGRy (yeast signal peptide). Figure 2 shows a map of the plasmid pLHCGRn. pLHCGRy is similar to pLHCGRn and only differs in length of the LHCGR gene construct, which is 9 bp longer in the pLHCGRy plasmid. For amplification, <I>E. coli</I> strain DH5α will be transformed separately with the two plasmids and an empty control plasmid by heatshocking. The cells will be cultivated in ampicillin-containing LB medium.
</P><P style="width:auto; margin-left:80px; margin-right:80px;"><FONT face="verdana"><SMALL><B>  Figure 2: Plasmid map of pLHCGRn derived from pSP-GM1. </B> The plasmid contains a yeast origin of replication, 2μ ori, and a yeast marker, <I>URA3</I>. The parts that are important for transforming <I>E. coli</I> are the ampicillin resistance gene, AmpR and the E. coli origin of replication, pUC ori. The LHCGR gene construct with the yeast signal peptide is anked by <I>Not</I>I and <I>Pac</I>I. P TEF1 is a strong, constitutive promoter and T ADH1 is a terminator. </SMALL></FONT></P><H4><SPAN class="mw-headline" id="Transformation_of_IMFD-73_and_degradation_of_the_cell_wall">Transformation of IMFD-73 and degradation of the cell wall</SPAN></H4><P>After amplification in DH5α, the plasmids will be purified from <I>E. coli</I>. The yeast strain IMFD-73 will be transformed with either pLHCGRn or pLHCGRy using the High Efficiency Transformation (TRAFO) method. As a control, a third transformation of IMFD-73 with the empty expression plasmid will also be performed. The cells will be cultivated and selected in uracil-deficient media. After transformation, hCG hormone, ordered from a company, will be added and screening for fluorescence will be performed. As mentioned previously, proteins consisting of 70 or more amino acids have very limited possibilities of penetrating the yeast cell wall <A href="#Ladds">[6]</A>. Since the hCG hormone has two subunits consisting of 92 and 145 amino acid residues, respectively, it is likely that it will not be able to pass the cell wall. Therefore, Zymolyase, a cell wall degrading enzyme, will be added and the screening procedure will be repeated. For further experiments, the plasmid (either pLHCGRn or pLHCGRy) expressing the more functional receptor will be chosen. The fluorescence levels will therefore be compared and the plasmid of the strain displaying the higher fluorescence intensity will be used. The strain with the empty control plasmid will be used to examine the background fluorescence signal and serves as a reference. 
</P><H2><SPAN class="mw-headline" id="Deletion_of_CWP2"><B>Deletion of CWP2</B></SPAN></H2><P>The second task focuses on modifying the IMFD-73 strain in such a way that its cell wall is weakened. The aim is for the hCG hormone to be able to pass the cell wall without the need of adding Zymolyase, which is impractical if the aim for the test kit to be used at home. Furthermore, removing the cell wall with enzymes results in less stable cells and this is not desirable if the construction should be used as a biosensor. Hence, instead of removing the whole cell wall, the IMFD-73 strain will have the gene <I>CWP2</I> deleted in order to decimate the cell wall. This gene encodes a mannoprotein that is a major constituent of the cell wall and important for its stabilization <A href="#Data">[5]</A>. Previously, it has been reported that CWP mutants have an increased cell wall permeability <A href="#Zhang">[7]</A> which is a desired feature of the hCG biosensor. 
</P><P>The deletion of <I>CWP2</I> will be done using the Bipartite method, which is outlined in Figure 3. For this, four DNA fragments will be needed. Two of those will consist of 500 bp from the upstream respectively downstream region of the <I>CWP2 gene</I>. The other two fragments will consist of the 5' respectively 3' end of the kanMX cassette. This cassette contains a kanamycin resistance gene under control of the TEF1 promoter and terminator. Each of these two fragments will also contain a loxP site. All four fragments will first be amplified in separate PCR reactions (denoted “PCR1” reactions in Figure 3). The kanMX fragments, which will be available in already existing plasmids, are amplified using the primers kanMx-1-fw, kanMx-1-rev, kanMx-2-fw and kanMx-2-rev. The <I>CWP2</I> fragments will need to be obtained and amplified from yeast via colony-PCR using primers cwp2up-fw, cwp2up-rev, cw2pdown-fw and cwp2down-rev. The primers cwp2up-rev and cwp2down-fw have 5' extensions that are reverse complement (indicated with lines in Figure 3) to the loxP sites of the kanMX fragments. The PCR1 reaction with the <I>CWP2</I> up and downstream sequences will consequently generate two fragments that are overlapping the loxP sites in the kanMX fragments. 
</P><P>The next step will be to fuse the <I>CWP2</I> up- and downstream region to the kanMX gene by fusion PCR as indicated in Figure 3. In this second PCR reaction, the <I>CWP2</I> upstream fragment that resulted from PCR1 and the overlapping kanMX 5' end fragment will together serve as templates. PCR will be run with primers cwp2up-fw and kanMx-1-rev. The same principle will be applied in order to fuse the kanMX 3' end with the <I>CWP2</I> downstream fragment. The resulting two fragments of PCR2 will then be purified and inserted into IMFD-73, using the TRAFO method. Through homologous recombination, the <I>CWP2</I> gene will then be replaced by the kanMX cassette. The cells will be selected on G418 medium, where only cells containing the kanMX gene will be able to grow. The deletion will also be verified by verification PCR. The loxP site can be used to remove the marker after selection, but this will not be necessary in this case.
</P><P style="width:auto; margin-left:80px; margin-right:80px;"><FONT face="verdana"><SMALL><B> Figure 3: Overview of the Bipartite method.</B> 500 bp of the up- and downstream sequence of the <I>CWP2</I> gene will be amplified from genomic DNA using primers with 5' extensions that are complement to the loxP sites as indicated with lines above. The 5' and 3' end of the kanMX (kanamycin resistance) gene flanked to loxP sites, available in vectors, will also be amplified by PCR. The fragments from these PCR1 reactions will be fused by Fusion PCR. The resulting fragments of this reaction can then be inserted into yeast. Homologous recombination will lead to the exchange of <I>CWP2</I> with kanMX in the yeast's genome. The cells can be selected on G418 medium.</SMALL></FONT></P><H2><SPAN class="mw-headline" id="Expression_of_monooxygenase_and_tryptophanase_genes"><B>Expression of monooxygenase and tryptophanase genes</B></SPAN></H2><P>The third task in this project is to manipulate the yeast cell in order to produce indigo. The dye indigo is chosen as the output signal in the pregnancy test due to its high intensity. A small amount of this molecule will produce a powerful stain. Two enzymes are needed for the synthesis of indigo from the precursor tryptophan: tryptophanase and a mono- or a dioxygenase. Expression of the fmo gene encoding the avin-containing monooxigenase in <I>E. coli</I> has resulted in the most successful bio-indigo production  <A href="#Han">[8]</A>. Therefore, the fmo gene will be used in this project. A gene with similarities to fmo has been identified in yeast  <A href="#Choi">[9]</A> but due to lack of available research records, the fmo gene from <I>Methylophaga</I> will be used instead. The tryptophanase gene (tnaA) will be cloned from <I>E. coli</I>. Both genes will then be inserted into <I>S. cerevisiae</I> and will at first be set under the control of constitutive promoters. This is done in order to test if it is possible to produce indigo in yeast. If bio-indigo production is successful, the constitutive promoter for the tnaA gene will be exchanged by the <I>FIG1</I> promoter in order to couple the bio-indigo production with the pheromone pathway. The plasmid can then be used to transform IMFD-73.
</P><H4><SPAN class="mw-headline" id="Plasmid_construction_and_transformation_of_a_yeast_lab_strain">Plasmid construction and transformation of a yeast lab strain</SPAN></H4><P>Two cloning steps are needed to construct the plasmid with the tnaA and fmo genes. E. coli strains contain the gene coding for tryptophanase (tnaA) and the sequence will be amplified from genomic DNA using colony-PCR. The used primers are tnaa-fw and tnaa-rev. To the 5' end of the primers, <I>Bam</I>HI and <I>Xho</I>I sequences are added in order to flank the gene sequence with these restriction sites. The PCR product will be purified on gel and digested with <I>Bam</I>HI and <I>Xho</I>I. After digestion, the fragment will be purified on a column and ligated into the corresponding restriction sites of the expression plasmid pIYC04 with the bidirectional, constitutive promoters TEF1 and PGK1, a <I>HIS3</I> marker gene and an ampicillin resistance gene. The tnaA gene will be set under the control of the PGK1 promoter. For amplification of the plasmid, <I>E. coli</I> strain DH5α will be transformed by heatshocking and the cells will be cultivated in ampicillin-containing media. Afterwards, the plasmid will be purified. 
</P><P>The gene for the monooxygenase (fmo), derived from Methylophaga, is kindly provided by Si Wouk Kim from Chosun University in South Korea. The sequence will be amplified by PCR with primers fmo-fw and fmo-rev. Restriction sites <I>Not</I>I and <I>Pac</I>I are inserted into the primers. The PCR product will be purified on gel and digested with restriction enzymes <I>Not</I>I and <I>Pac</I>I. The digested fragments will then be purified on a column and ligated into the expression plasmid containing the tnaA gene. Thus, the fmo gene is set under the control of the TEF1 promoter. The resulting plasmid will be called pIndigo and amplified in <I>E. coli</I>. A plasmid map of pIndigo is shown in Figure 4. After amplification, the plasmid will be used to transform the normal yeast lab strain CEN.PK 113-11C using the TRAFO method. If the experiments are successful, indigo will be produced. If indigo is not detected, it might be due to low intracellular levels of tryptophan, which will therefore be added to the culture and screening for indigo will be repeated.
</P><P style="width:auto; margin-left:80px; margin-right:80px;"><FONT face="verdana"><SMALL><B>Figure 4: Plasmid map of pIndigo derived from pIYC04.</B> The plasmid contains a yeast origin of replication, 2μ ori and a yeast marker, <I>HIS3</I>. The important parts for transforming <I>E. coli</I> are the ampicillin resistance gene, AmpR, and the <I>E. coli</I> origin of replication, pUC ori. The fmo gene is flanked by <I>Not</I>I and <I>Pac</I>I and is set under the control of the TEF1 promoter. The tnaA gene is flanked by <I>Bam</I>HI and <I>Xho</I>I and is set under the control of the PGK1 promoter. T CYC1 and T ADH1 are terminators.
 </SMALL></FONT></P><H4><SPAN class="mw-headline" id="Replacement_of_the_PGK1_with_the_FIG1_promoter">Replacement of the PGK1 with the FIG1 promoter</SPAN></H4><P>The next step is to replace the PGK1 with the <I>FIG1</I> promoter. 450 bp of the 5' upstream regulatory sequence of the <I>FIG1</I> gene will be amplified from genomic DNA from a normal yeast strain using colony-PCR. The primers fig-fw and fig-rev will used. Into the primers, <I>Bam</I>HI and <I>Fse</I>I restriction sites are inserted. The PCR product will be purified on gel, digested with the restriction enzymes and ligated into the pIndigo, hence removing PPGK1 and inserting P(FIG1). This plasmid will be named pFig-Indigo and is illustrated in Figure 5.
</P><P style="width:auto; margin-left:80px; margin-right:80px;"><FONT face="verdana"><SMALL><B>Figure 5: Plasmid map of pFig-Indigo derived from pIndigo.</B> The plasmid contains a yeast origin of replication, 2μ ori and a yeast marker, <I>HIS3</I>. The important parts for transforming <I>E. coli</I> are the ampicillin resistance gene, AmpR, and the <I>E. coli</I> origin of replication, pUC ori. The fmo gene is flanked by <I>Not</I>I and <I>Pac</I>I and is set under the control of TEF1 promoter. The tnaA gene is now set under the control of the <I>FIG1</I> promoter. T CYC1 and T ADH1 are terminators.
 </SMALL></FONT></P><H2><SPAN class="mw-headline" id="Combination_of_the_components_to_a_functional_biosensor"><B>Combination of the components to a functional biosensor</B></SPAN></H2><P>The final step of the project will be to combine the parts described above in one single host strain. The IMFD-73 with the deleted <I>CWP2</I> gene from task 2 will be transformed with either pLHCGRy or pLHCGRn using TRAFO and the cells will be selected on uracil-deficient media. After that, the pFig Indigo will also be inserted using TRAFO and the cells will be grown on histidine-deficient media. The ability of the resulting yeast to function as a biosensor will then be tested by adding different concentrations of the hCG hormone and screening for output signals will be done.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><A href="http://ac.els-cdn.com/S0303720796039512/1-s2.0-S0303720796039512-main.pdf?_tid=dc7f17ba396e939eeaf08479655a57c9&amp;acdnat=1341397743_aa342de7247ab8f2a9adffd446423cb8" class="external autonumber" rel="nofollow">[1]</A> Ryu KS, Ji L, Chang L, Ji TH. Molecular mechanism of LH/CG-receptor activation. Molecular and Cellular Endocrinology. 1996;125(1-2):93-100.
<A href="http://search.proquest.com.proxy.lib.chalmers.se/docview/222535499/fulltextPDF?accountid=10041" class="external autonumber" rel="nofollow">[2]</A> Dufau ML. The luteininizing hormone receptor. Annual review of Physiology. 1998;60(1):461-496.
<A href="http://ac.els-cdn.com/S0958166907001139/1-s2.0-S0958166907001139-main.pdf?_tid=e8b772f387e19d3343415ee73bea9a2a&amp;acdnat=1341398513_c6e13108ff4f982aadd1fca30cc779c8" class="external autonumber" rel="nofollow">[3]</A> Hamilton SR, Gerngross TU. Glycosylation engineering in yeast: the advent of fully humanized yeast. Current Opinion in Biotechnology. 2007;18(5):387-392.
<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1134190/pdf/biochemj00109-0219.pdf" class="external autonumber" rel="nofollow">[4]</A> Petäja-Repo UE, Merz WE, Rajaniemi HJ. Significance of the carbohydrate moiety of the rat ovarian luteinizing-hormone/chorionic-gonadotropin receptor for ligand-binding specificity and signal transduction. The Biochemical Journal. 1993;292(3):839-844.
<A href="http://www.yeastgenome.org" class="external autonumber" rel="nofollow">[5]</A> Saccharomyces Genome Database [Online]. Stanford: National Human Genome Research Institute; 1997 [updated 2012 Feb 9; cited 2012 May 13]. Available from: <A href="http://www.yeastgenome.org/" class="external free" rel="nofollow">http://www.yeastgenome.org/</A>.
<A href="http://ac.els-cdn.com/S0167779905001356/1-s2.0-S0167779905001356-main.pdf?_tid=72c8be5e65ef3c59b099e60def061454&amp;acdnat=1341398710_2488dd15464fdb0f634d8b8d96659bfa" class="external autonumber" rel="nofollow">[6]</A> Ladds G, Goddard A, Davey J. Functional analysis of heterologous GPCR signaling pathways in yeast. Trends in Biotechnology. 2005;23(7):367-373.
<A href="http://toxsci.oxfordjournals.org.proxy.lib.chalmers.se/content/103/1/68.full.pdf+html" class="external autonumber" rel="nofollow">[7]</A> Zhang M, Liang Y, Zhang X, Xu Y, Dai H, Xiao W. Deletion of yeast CWP genes enhances cell permeability to genotoxic agents. Toxicological sciences : an official journal of the Society of Toxicology. 2008;130(1):68-76.
<A href="http://ac.els-cdn.com/S0141022908000598/1-s2.0-S0141022908000598-main.pdf?_tid=61734e15218e3231cf528f84f9d3a9e9&amp;acdnat=1341398925_c384fe6ee32ee8a669126dd28851f716" class="external autonumber" rel="nofollow">[8]</A> Han GH, Shin HJ, Kim SW. Optimization of bio-indigoproduction by recombinant E. coli harboring fmo gene. Enzyme and Microbial Technology. 2008;42(7):617-623.
<A href="http://ac.els-cdn.com/S0006291X03010878/1-s2.0-S0006291X03010878-main.pdf?_tid=a83acce8c1f2e57aa628e936a295a853&amp;acdnat=1341399014_e92588fb6cf9840564b395083ca57556" class="external autonumber" rel="nofollow">[9]</A> Choi HS, Kim JK, Cho HE, Kim YC, Kim JI, Kim SW. A novel  flavin-containing monooxygenase from Methylophaga sp.strain SK1 and its indigo synthesis in Escherichia coli. Biochemical and Biophysical Research Communications. 2003;306(4):930-936.
</DIV><DIV id="footer" style="background-color:#FFFFFF; height:300px;"><P>With many thanks to our generous sponsors:
</P><DIV class="floatright"><A href="/File:GATC.gif" title="File:GATC.gif">File:GATC.gif</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Chalmers-Gothenburg/Methods">http://2012.igem.org/Team:Chalmers-Gothenburg/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Chalmers-Gothenburg/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Chalmers-Gothenburg/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Methods&amp;oldid=58914" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>