<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Discussion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Chalmers-Gothenburg/Discussion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Chalmers-Gothenburg/Discussion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Discussion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Discussion&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Chalmers-Gothenburg/Discussion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Chalmers-Gothenburg/Discussion</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="nav"><LI><A href="/Team:Chalmers-Gothenburg">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Team">Who we are</A></LI><LI><A href="https://igem.org/Team.cgi?id=753">Official team page</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Contacts">Contact Us</A></LI><LI><A href="#">Project</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Biodetection_of_hCG">Project description</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Theory">Theory</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Methods">Methods</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Results">Results</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Discussion">Discussion and future perspectives</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Modelling">Modelling</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Notebook">Notebook</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Safety_page">Safety</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Attributions">Attributions</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Media">Media</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Education">Education</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Collaboration">Collaboration</A></LI><LI><A href="/Team:Chalmers-Gothenburg/Sponsors">Sponsors</A></LI></DIV><DIV class="box2"><DIV align="left" style="margin-left:10px; margin-right:10px; "><B><FONT size="9"><B>Sponsors</B></FONT></B></DIV></DIV><DIV align="justify" style="width:auto; margin-left:0px; margin-right:300px;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Discussion_of_results_and_future_perspectives"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Discussion of results and future perspectives</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Production_of_indigo_in_yeast"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Production of indigo in yeast</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Reflections_of_the_pIndigo_system"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Reflections of the pIndigo system</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Possible_improvements_of_methods"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Possible improvements of methods</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#The_LH.2FCG_receptor"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The LH/CG receptor</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#1._The_hCG_hormone_could_not_pass_the_yeast_cell_wall"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">1. The hCG hormone could not pass the yeast cell wall</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#2.The_receptor_only_mediates_weak_signals"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">2.The receptor only mediates weak signals</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#3._The_receptor_is_not_functional_when_expressed_in_yeast"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">3. The receptor is not functional when expressed in yeast</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Discussion_of_results_and_future_perspectives"><B>Discussion of results and future perspectives</B></SPAN></H1><P>In this section we will discuss our results and things that can be improved. The discussion is divided in to big parts, the indigo production and the receptor. 
</P><H2><SPAN class="mw-headline" id="Production_of_indigo_in_yeast"><B>Production of indigo in yeast</B></SPAN></H2><P>After analyzing the output signal, we analyzed the colour changes and tried to verify which colours that was produced. In the next section we will discuss our observations and what kind of improvements that can be done.
</P><H3><SPAN class="mw-headline" id="Reflections_of_the_pIndigo_system"><B>Reflections of the pIndigo system</B></SPAN></H3><P>One interesting aspect of our work is if we can draw any conclusions about the reactions that are occuring during our cultivations. If we can do this, then we can conclude something about the bio-indigo producing system pIndigo, and finally pFig-Indigo. 
</P><P>A common observation has been the change of colour, from yellow to brown, for cultivations of CEN PK 113-11C + pIndigo in both glucose and galactose containing media, with L-tryptophan added. In the reaction from tryptophan to indigo the enzyme tryptophanase is utilized in the first step. The change of colour would then indicate that the tryptophan in the solution reacts, since this is the only difference in the media compared to the cultivation CEN PK 113-11C + pIndigo in glucose media, which does not change colour. If tryptophan reacts, then we can conclude that the tryptophanase gene in pIndigo works, since it only can react with the tryptophanase. 
</P><P>However, during the above mentioned conditions, since the media turns brown it seems that the indole, produced after tryptophan reacts with tryptophanase, does not react with <I>fmo</I>. This reaction would, in presence of oxygen, produce indigo which would result in a blue media. There could of course be a very small concentration of indigo present in the media, but it is more likely that indole reacts to something completely different, that induces this brown colour. In conclusion it can be said that, during these conditions, it is likely that the <I>tnaA</I> gene is working but it is not clear in any way if the <I>fmo</I> gene is working or not.
</P><P>The cultivation of CEN PK 113-11C + pIndigo and 5 ml L-tryptophan 20 g/l (added after 24 hours of cultivation) in glucose containing medium gives another result though. This cultivation does not take the brown colour, but instead a kind of forest green. This could possible imply that the cells need to grow for, in our case, 24 h before the system is fully activated, in regards to transcription and translation of the <I>fmo</I> gene. When this has been achieved and the chemical is added a hypothesis would be that the precursor, tryptophan, reacts all through to indigo and the blue colour observed is a mixture of the indigo blue and the natural yellow colour of the media.
</P><P>A final thing that indicates that the system is working is the blue bubbles observed in CEN PK 113-11C + pIndigo in galactose containing medium, and the CEN PK 113-11C + pIndigo and 2 drops of indole 2 g/l (added after 24 hours of cultivation) in glucose containing medium. It occurs in two different flaks with the same initial conditions (the indole had not been added at the time of the picture) and the system should not produce any other chemical with this distinct colour. One could argue that some contamination was present, but when the experiment was repeated these blue bubbles appeared again. It has not been proved that it is indigo, but it seems likely. If there was more time to be able to keep working on the experiment the next step would have been to scientifically prove that indigo was produced by our system, for the above mentioned conditions. 
</P><H3><SPAN class="mw-headline" id="Possible_improvements_of_methods"><B>Possible improvements of methods</B></SPAN></H3><P>During the enzymatic assay some sort of separation of the enzymes that was studied could, and probably should, have been performed before the reaction with tryptophan was initiated. It is preferable that the reaction occurs in the cell free extract, but since only the presence of tryptophanase was to be proved this was probably not necessary.
</P><P>Since different codons for the same amino acid can occur in different ratios, depending on the kind of organism, an improvement could be found by studying the genes in the system. If some codons could be optimized for the host organisms then this would increase the efficiency for the transcription and translation of our system. 
</P><P>Indigo can be synthesized from tryptophan in two main steps and each requires a catalysing enzyme. The first step is catalysed by tryptophanase and the last reaction step is actually several steps where the first reaction is the conversion of indole to indoxyl, catalysed by a mono- or dioxygenase. Indoxyl is then converted to indigo by several spontaneous reactions in the presence of oxygen <A href="#Han">[1]</A>, <A href="#Berry">[2]</A>. In our system we have chosen to use <I>fmo</I>, a gene encoding flavin-containing monooxygenase to catalyse the conversion of indole to indoxyl. It is possible that this gene does not work in yeast and a final approach would therefore be to substitute the <I>fmo</I> gene for one that encodes for another oxygenase or to analyze and try to improve the initial conditions for the co-factors of our current enzymes.
</P><H2><SPAN class="mw-headline" id="The_LH.2FCG_receptor"><B>The LH/CG receptor</B></SPAN></H2><P>When analyzing our results, we have identified several possible reasons that may have led to the biosensor being unable to detect the hCG hormone. Below, we discuss these and also present some ideas on how to take our project further and improve our study. 
</P><H3><SPAN class="mw-headline" id="1._The_hCG_hormone_could_not_pass_the_yeast_cell_wall"><B>1. The hCG hormone could not pass the yeast cell wall</B></SPAN></H3><P>We were aware of the risk that the hCG hormone, due to its size and due to the limited permeability of the yeast cell wall, would not be able to pass the cell wall and therefore not reach the receptor. This is the reason that the ΔCWP2 strain was created and that the initial fluorescence screenings were done using cells that had been treated with cell wall-degrading enzymes. Despite these efforts, there is a small risk that the hCG hormone never reached the receptor, which would then explain the lack of a successful detection. However, we deem this to be relatively unlikely. At least the lyticase treatment should have allowed some hCG to reach the receptor.
</P><P>If, however, further investigations would show that the cell wall permeability of the Δ<I>cwp2</I> strain is still too limited for hCG to pass, one could also try to delete the gene CWP1 in order to increase the permeability additionally <A href="#Zhang">[3]</A>. But there are also several other possibilities to weaken the cell wall, for example the deletion of genes whose corresponding proteins contribute to the assembly of the cell wall or knock-out of other cell wall components.
</P><H3><SPAN class="mw-headline" id="2.The_receptor_only_mediates_weak_signals"><B>2.The receptor only mediates weak signals</B></SPAN></H3><P>The receptor might be able to bind hCG but, due to inefficient interaction between the receptor and the yeast G alpha, only partially induce the activation of the pheromone pathway. This is a common problem when expressing human GPCRs in yeast and that is the reason why strains with chimeric G alpha proteins have been used when expressing the LH/CG receptor. Since these chimeric G alpha proteins have had the last 5 C terminal amino acid residues of the yeast G alpha protein replaced by the corresponding human amino acids, they should enable the human receptor to couple with the G alpha subunit while still preserving the G alpha ability to interact with G betagamma complex. Still, it might be so that the LH/CG receptor interacts inefficiently with the yeast G protein and thus only produces a very weak output signal. This may then be the reason why our biosensor was unable to produce a noticeable signal in response to detecting hCG. 
</P><P>If the LH/CG receptor only produces a weak output signal, the background noise of GFP expression when examining our cells in the fluorescence microscope would have made it difficult to identify cells that actually displayed signal-induced expression of GFP. To improve analysis, a next step could therefore be to use flow cytometry. This method enables comparative quantification of the GFP signaling in whole cell populations <A href="#flowcy">[4]</A> and may then enable even weak signal-induced GFP expression to be distinguished from the background noise.
</P><P>Another possible step to solve the potential problem of a weak signal would be to amplify the signal. A method designed to this was developed by Fukada et al in 2011. Their strategy involves invoking positive feedback in response to activation of the signaling pathway. This is done by placing both the gene for the yeast G beta subunit and the reporter gene under the control of the FIG1 promoter. Thus, when a ligand binds to the receptor, the Gbeta is overexpressed and can compete with the endogenous Gbeta for bindinf the G alpha subunit. This competition would allow G betagamma complexes to be released and to activate the signaling pathway once again thereby amplifying the weak signal and increasing the expression of the reporter gene <A href="#fukuda">[5]</A>. 
</P><H3><SPAN class="mw-headline" id="3._The_receptor_is_not_functional_when_expressed_in_yeast"><B>3. The receptor is not functional when expressed in yeast</B></SPAN></H3><P>There is always a risk when expressing a protein in another host that the protein will not be functional. This might be due to an incorrect fold or that the protein did not get the proper post-translational modifications. The LH/CG receptor, when expressed in yeast, might have lost its correct conformation, been unable to integrate properly into the yeast cell membrane and therefore also be unable to bind the hCG hormone. Mammalian proteins will be glycosylated in a significantly different manner when expressed in yeast. This might be a potential problem for the LH/CG receptor, which has a number of glycosylated sites. According to literature, these glycosylations are not important for ligand binding and signal transduction, but they might be important for proper integration of the receptor into the membrane<A href="#pet.C3.A4ja">[6]</A>. This may then explain why the receptor is non-functional in yeast. 
</P><P>As a further step, we would like to determine the location of the LH/CG receptor when expressed in yeast. This could be done by, for example, placing a fluorescent tag on the receptor and would enable us to see whether or not it has been integrated in the yeast membrane or if it has been retained in the cytosol. 
</P><P>If it turns out that LH/CG successfully has been integrated in the membrane but that the receptor is unable to function properly, we would also like to try to reengineer the receptor. In previous studies deletion of foremost intracellular loops has proven to increase the stability and activity of some GPCRs when expressed in yeast<A href="#ladds">[7]</A>. This might be something that could be applied to the LH/CG receptor as well.
</P><H1><SPAN class="mw-headline" id="References"><B>References</B></SPAN></H1>[1] Han GH, Shin HJ, Kim SW. Optimization of bio-indigoproduction by recombinant E. coli harboring fmo gene. Enzyme and Microbial Technology. 2008;42(7):617-623.
[2] Berry A, Dodge TC, Pepsin M, Weyle W. Application of metabolic engineering to improve both the production and use of biotech indigo. Journal of Industrial  Microbiology &amp; Biotechnolog.2002;28(3):127-133.
[3] Zhang M, Liang Y, Zhang X, Xu Y, Dai H, Xiao W. Deletion of yeast CWP genes enhances cell permeability to genotoxic agents. Toxicological sciences: an official journal of the Society of Toxicology. 2008;130(1):68-76.
[4] Ishii J, Moriguchi M, Hara KY, Shibasaki S, Fukuda H, Kondo A. Improved identification of agonist-mediated Gαi-specific human G-protein-coupled receptor signaling in yeast cells by flow cytometry. Analytical Biochemistry. 2012;426(2):129-133.
[5] Fukuda N, Ishii J, Kaishima M, Kondo A. Amplification of agonist stimulation of human G-protein-coupled receptor signaling in yeast. Analytical Biochemistry. 2011;417(2):182-187.
[6] Petäja-Repo UE, Merz WE, Rajaniemi HJ. Significance of the carbohydrate moiety of the rat ovarian luteinizing hormone/chorionic-gonadotropin receptor for ligand-binding specificity and signal transduction. The Biochemical Journal. 1993;292(3):839-844.
[7] Ladds G, Goddard A, Davey J. Functional analysis of heterologous GPCR signaling pathways in yeast. Trends in Biotechnology. 2005;23(7):367-373.
</DIV><DIV id="footer" style="background-color:#FFFFFF; height:300px;"><P>With many thanks to our generous sponsors:
</P><DIV class="floatright"><A href="/File:GATC.gif" title="File:GATC.gif">File:GATC.gif</A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Chalmers-Gothenburg/Discussion">http://2012.igem.org/Team:Chalmers-Gothenburg/Discussion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Chalmers-Gothenburg/Discussion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Chalmers-Gothenburg/Discussion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Discussion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Chalmers-Gothenburg/Discussion&amp;oldid=221773" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>