"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Edinburgh/Results/BABEL
From 2008.igem.org
Return to Protocols page
BABEL: BioBrickTM Assembly with Blunt-Ended Ligation
BABEL is an alternative, restriction enzyme-free method for generating BioBricksTM. It is fully compatible with the BioBrickTM 1.0 standard and also with all of the proposed successor standards, and is itself a potential successor to the BioBrickTM 1.0 standard in that it can be used for fully scarless assembly. Babel is based on blunt-ended ligation of PCR products followed by fusion PCR and self-ligation. Here we describe the use of BABEL to generate fully compliant BioBricksTM in pSB1A2, a procedure referred to in our laboratory as 'SOB', or 'Son Of Babel'. Possible extensions of this concept to scarless assembly should be obvious. The 'Son of Babel' procedure is as follows:
Generate a blunt-ended linear version of pSB1A2 by PCR using appropriate primers which contain the entire BioBrickTM prefix and suffix. Primers we have used for this are: sobf2 tactagtagcggccgctgcag and sobr1 ctagaagcggccgcgaattc. Since these primers lie entirely within the prefix and suffix regions, they should work for any BioBrickTM vector. Be sure to use a proof-reading polymerase which will generate a blunt-ended product. We use Kod (Invitrogen), which is as accurate as Pfu and 4 to 5 times faster. Purify your product and store it in the freezer, as it can be used for all of your BABEL cloning experiments.
Design primers for your insert. These should include only the insert, no non-complementary tails (apart from the extra TAA that may be added to the end of a coding sequence BioBrickTM), no prefix or suffix. For example, for a coding sequence BioBrickTM, the forward primer should start with ATG... and the reverse primer with TTATTA.. (giving the double TAATAA stop codon). Since there is no requirement for prefix or suffix sequence, the primers can be shorter and cheaper than those for standard BioBrickTM cloning.
Perform PCR to obtain your blunt-ended insert PCR product (again, be sure to use a proof-reading enzyme which will give a blunt-ended product - not Taq, which usually leaves an A overhang). Check the product on a gel and purify it.
Ligate your insert PCR product to your linear vector PCR product in a reaction mixture which includes both T4 DNA ligase and T4 Polynucleotide Kinase (PNK). PNK is required to add a 5'-phosphate to the DNA to allow ligation, since standard oligonucleotide primers have a free 5'-hydroxyl group. Fortunately, PNK works well in T4 DNA ligase buffer, which also provides the necessary ATP.
You can now transform with this primary ligation, but since the linear vector PCR product can self-ligate, most of the colonies will probably just be vector, so unless there is an easy screen or selection for those containing the insert, we recommend doing a second PCR as described below (we did make a vector allowing blue-white selection of insert-containing colonies, but it did not generate a standard BioBrickTM vector so would not have allowed direct submission of our products to the Registry).
Perform a secondary PCR reaction using the insert forward primer and vector reverse primer, using a small amount of the primary ligation as a template. This should give you a single PCR product consisting of vector and insert fused together. At this stage, you can also add a ribosome binding site to a coding sequence BioBrickTM by using a variant reverse primer which contains the ribosome binding site sequence. We used primer sobrbs1, ctagtacctcctctagaagcggccgcaattc, which adds ribosome binding site BBa_K1180012, but found that this was much more likely than the standard reverse primer to generate multiple PCR products.
Check the secondary PCR product on a gel, and purify it if it looks OK. Then self-ligate it in a reaction mixture containing T4 DNA ligase and T4 PNK as described above.
Transform E. coli with the ligation mixture. Since self-ligation is highly efficient, even with blunt ends, the great majority of colonies should contain the desired construct.
Miniprep a few colonies, check insert size by restriction digest, and sequence to be sure that the ends are intact, with no missing bases.
Advantages and disadvantages of BABEL
Since this method does not use restriction digests, any gene can be cloned straight into a BioBrickTM vector, even if it contains multiple forbidden restriction sites which would prevent cloning using standard BioBrickTM prefix and suffix cloning sites. MABEL (qv) can then be performed to remove the offending restriction sites.
By choice of primer in the secondary PCR, it is possible to recover the insert in both forward and reverse orientations, which is not possible using standard BioBrickTM cloning methods. There are situations where it might be advantageous to recover a promoter or a coding sequence in both the forward and reverse orientation; for example, in the Edinburgh 2007 cell-division-detector project.
Prefix and suffix are in the vector PCR product, and are not associated with the insert. This means that the same procedure can be used for scarless assembly, obviating the problems with fusion proteins which are the main impetus for discussions as to a replacement standard for BioBrickTM 1.0 (see the BioBrickTM Foundation Technical Standards Working Group discussion for July archived here .
One potential disadvantage of BABEL is that multiple rounds of PCR are required, increasing the risk of PCR induced mutations. However, with highly accurate proof-reading polymerases this does not seem to be such a big issue: we have genes which have been through BABEL plus several rounds of MABEL mutagenesis (which works in a similar way) and have detected no undesired mutation events.
User Experience
We used Son-Of-Babel in this project to clone glgC, crtE, and the maize isoamylase genes in pSB1A2. In all cases we were ultimately successful in recovering the desired clones, but in some cases the process was not as smooth as we might have hoped. In particular, use of the alternate reverse primer with the ribosome binding site pre-added had a strong tendency to generate incorrect PCR products, suggesting that this primer might also be binding spuriously to the other end of the vector sequence. Although we do not have good evidence at this stage, we suspect the GC-rich NotI sites at both ends of the vector may play a part in this problem. Since the NotI sites are no longer used, we understand that they may be removed from the next specification for BioBricksTM (see discussion site link above). Also, in the case of the maize genes, which for some reason were unexpectedly hard to amplify in any case (requiring an extended denaturation time and the presence of 10% v/v glycerol for efficient amplification), a subset of the clones had missing bases at the end suggesting some exonuclease activity, though we only had to sequence three clones in each case to get the correct product.
Retrieved from " http://2008.igem.org/Team:Edinburgh/Results/BABEL "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
