<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Results_Bacillobricks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Edinburgh/Results/Bacillobricks">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Results/Bacillobricks&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Results/Bacillobricks&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Results/Bacillobricks&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Results/Bacillobricks" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Results/Bacillobricks</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><B><A href="/Team:Edinburgh/Protocols" title="Team:Edinburgh/Protocols">Return to Protocols Page</A></B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Bacillobricks:_Introduction_of_BioBricksTM_into_Bacillus_subtilis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Bacillobricks: Introduction of BioBricks<SUP>TM</SUP> into Bacillus subtilis</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction_of_BioBricksTM_into_B._subtilis_on_a_plasmid"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction of BioBricks<SUP>TM</SUP> into B. subtilis on a plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Integration_of_BioBricksTM_onto_the_B._subtilis_chromosome"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Integration of BioBricks<SUP>TM</SUP> onto the B. subtilis chromosome</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#References"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Bacillobricks:_Introduction_of_BioBricksTM_into_Bacillus_subtilis">Bacillobricks: Introduction of BioBricks<SUP>TM</SUP> into <I>Bacillus subtilis</I></SPAN></H1><P><I>Bacillus subtilis</I> is potentially superior to <I>E. coli</I> as a host for some projects, for several reasons:
</P><UL><LI> It is much more effective at secreting proteins into the medium, as <I>E. coli</I> lacks the Main Terminal Branch of the General Secretory Pathway.
</LI><LI> As a Gram positive bacterium, it lacks the toxic lipopolysaccharide (endotoxin) of Gram negative bacteria such as <I>E. coli</I></LI><LI> The cells are considerably larger, making it easier to visualise intracellular components.
</LI><LI><I>B. subtilis</I> forms endospores, a highly stable, heat and dessication resistant resting state which can be stored dry for years or decades, and will then germinate in less than 30 minutes when added to a suitable growth medium.
</LI><LI><I>B. subtilis</I> is not pathogenic and has even been used as a probiotic organism in human foods.
</LI></UL><P>However, standard BioBrick<SUP>TM</SUP> vectors do not allow introduction of BioBricks<SUP>TM</SUP> into <I>B. subtilis</I>. We felt that <I>B. subtilis</I> was potentially a suitable host for our 'Edinburgh Process' of conversion of cellulose to starch, due mainly to its ability to secrete enzymes such as cellulases. We therefore investigated two processes to allow introduction of BioBricks<SUP>TM</SUP> to <I>B. subtilis</I>, either on a plasmid or by integration into the genome.
These experiments were carried out mainly by C. French (instructor) and Nimisha Joshi (advisor).
</P><H2><SPAN class="mw-headline" id="Introduction_of_BioBricksTM_into_B._subtilis_on_a_plasmid">Introduction of BioBricks<SUP>TM</SUP> into <I>B. subtilis</I> on a plasmid</SPAN></H2><P>A standard method of transferring BioBricks<SUP>TM</SUP> between <I>E. coli</I> and <I>B. subtilis</I> would be the use of a shuttle vector which could replicate in both organisms, and in fact the Edinburgh iGEM 2007 team did demonstrate this using the <I>Lactobacillus</I> plasmid <B>pTG262</B>. pTG262 replicates in both <I>E. coli</I> and <I>B. subtilis</I> from the same versatile origin of replication, and has a multi-cloning site with EcoRI and PstI sites allowing convenient incorporation of BioBricks<SUP>TM</SUP>. pTG262 was submitted to the Registry as BBa_I742103, but our understanding, from one team that tried to acquire it from the Registry, was that it is not currently available as the transformation failed. Which brings us to the problem with pTG262 - it does not transform lab strains of <I>E. coli</I> with high efficiency, at least not in our hands. (For more information about pTG262 and its ancestors <B>pWV01</B> and <B>pSH71</B>, see deVos and Simons (1994), and this  <A href="http://www.openwetware.org/wiki/Cfrench:BioBrickVectors1" class="external text" rel="nofollow">summary</A>.
</P><P>To get around this problem, we decided to check whether we could ligate a BioBrick<SUP>TM</SUP> into pTG262 and then transform <I>B. subtilis</I> directly with the ligation mixture, rather than initially preparing DNA from <I>E. coli</I>. Initially we tested this idea using BioBrick<SUP>TM</SUP><B>BBa_J33204</B>, bearing the reporter gene <I>xylE</I> encoding <B>catechol-2,3-dioxygenase</B> (which converts catechol, a cheap, colourless substrate, to bright yellow 2-hydroxy-<I>cis,cis</I>-muconic semialdehyde). We have previously demonstrated that this reporter gene works well in <I>B. subtilis</I>. The insert was cut out with EcoRI and PstI and ligated with pTG262 cut with the same two enzymes. The ligation was then used to transform <I>B. subtilis</I> using a <A href="http://www.openwetware.org/wiki/Cfrench:BacTrans1" class="external text" rel="nofollow">standard procedure</A> and cells were plated on LB with chloramphenicol (10 mg/l). Chloramphenicol-resistant colonies were obtained, and the presence of <I>xylE</I> was confirmed by PCR, but no XylE activity was detected (ie, no visible yellow colour on addition of a drop of 10 mM catechol to the colonies). It was therefore concluded that BioBricks<SUP>TM</SUP> can be introduced into <I>B. subtilis</I> by this method, and that expression does not occur in the absence of a promoter.
</P><P>The experiment was repeated using a composite BioBrick<SUP>TM</SUP> consisting of <B>BBa_J33207</B> (<I>lac</I> promoter from <I>E. coli</I>) with BBa_J33204 (<I>xylE</I> reporter gene). Again, chloramphenicol-resistant colonies were obtained but no XylE expression was observed, suggesting that this promoter was not active in this context, even though the <I>lacI</I> repressor gene was not present.
</P><P>To check that expression could be achieved, we finally turned to the only BioBricked native <I>B. subtilis</I> promoter in our freezer, <B>BBa_J33206</B> (<I>B. subtilis ars</I> promoter, induced by sodium arsenate, cloned as part of the Edinburgh iGEM 2006 arsenic biosensor project). We made a composite BioBrick<SUP>TM</SUP> consisting of J33206+J33204, ligated this with pTG262 and transformed <I>B. subtilis</I> as above. Since the pSB1A2 vector band could not be separated from the Pars+xylE BioBrick<SUP>TM</SUP> on a gel (as they were the same size), it was expected that half of the colonies would contain the correct insert. In fact, one of four clones tested showed evidence of XylE activity (ie, a yellow pigment produced when a drop of 10 mM catechol was added to a colony) and only this clone showed the presence of <I>xylE</I> by PCR.
</P><P>A plasmid of the expected size was detected in plasmid DNA preps from this clone, but since the construct contains a significant amount of native <I>B. subtilis</I> sequence, we cannot at present exclude the possibility that the construct may have integrated into the genome at the <I>ars</I> locus. This can be tested by PCR to check the size of this locus.
</P><P>Interestingly, this clone showed <B>highly sensitive arsenic-dependent induction of XylE activity</B> and was capable of detecting arsenic at the WHO recommended threshold level of 10 ppb, unlike our previous attempts at non-BioBrick<SUP>TM</SUP>-based <I>B. subtilis</I> arsenic biosensors. This makes it potentially a useful biosensor for use in developing countries such as Bangladesh, where arsenic in groundwater is a major public health problem, since the biosensor can be stored and distributed in the form of dried endospores, which is not possible with <I>E. coli</I>-based biosensors. Here are some sample data from an experiment in which the 'Bacillosensor' clone was incubated overnight in 50% v/v LB, 50% v/v water  with 10 mg/l chloramphenicol and various concentrations of arsenic (as sodium arsenate, the most environmentally relevant form of this toxin). Incubation was at 37 C with shaking. The following morning, catechol was added to a final concentration of 0.5 mM and the samples were incubated at room temperature without shaking for several hours. Cells were then removed by centrifugation, and the absorbance at 377 nm (peak absorbance of the yellow product) of a 1/10 dilution was measured against a water blank.
</P><UL><LI> sterile growth medium: 0.027
</LI><LI> no arsenic: 0.048
</LI><LI> 10 ppb arsenic (WHO safety limit): 0.090
</LI><LI> 25 ppb arsenic: 0.117
</LI><LI> 50 ppb arsenic (Bangladesh safety limit): 0.184
</LI><LI> 75 ppb arsenic: 0.228
</LI><LI> 100 ppb arsenic: 0.309
</LI></UL><P>The yellow colour was clearly visible by eye, making this potentially a useful addition to our range of arsenic biosensors, especially for use in developing countries, where the ability to store and distribute the organism in a dry form as spores will be especially useful.
</P><H2><SPAN class="mw-headline" id="Integration_of_BioBricksTM_onto_the_B._subtilis_chromosome">Integration of BioBricks<SUP>TM</SUP> onto the <I>B. subtilis</I> chromosome</SPAN></H2><P><I>B. subtilis</I> becomes naturally competent at certain stages of its life cycle, and will readily take up linear DNA from the medium and incorporate it into the chromosome by homologous recombination (Sonenshein <I>et al</I>, 1993). We therefore sought to develop a method to prepare linear DNA fragments bearing a BioBrick<SUP>TM</SUP> and chloramphenicol resistance gene flanked by upstream and downstream DNA. The first step was to choose a locus of integration. The <I>amyE</I> locus, encoding amylase, is commonly used for this purpose, but since we hoped to test cellulose degradation genes, we were reluctant to disrupt a polysaccharide degradation locus (even though this might have been appropriate for this particular project, since starch hydrolysis would be undesirable in a starch-producing organism). Westers <I>et al</I> (2003) have presented a list of 332 genes which are dispensable in <I>B. subtilis</I>, and after examination of this list, we decided to incorporate our genes into a locus associated with lytic genes of the prophage PBSX. Our upstream DNA site would be 1 kb of DNA preceding the <I>xepA</I> region, and our downstream DNA would be 1 kb of DNA in the <I>xlyB-spoIISB</I> region. The inserted DNA would replace the lytic genes <I>xepA, xhlA, xhlB</I>, and the 5' end of <I>xlyB</I>, but <I>spoIISB</I> would be left intact. The following PCR primers were designed:
</P><UL><LI><B>upstreamF:</B> ggactt ggatcc gccattgg (BamHI) 
</LI><LI><B>upstreamR:</B> ggt gaattc tttatactggtcagc (EcoRI) 
</LI><LI><B>downstreamF:</B> gaaaaaccc gagctc tatcc (SacI) 
</LI><LI><B>downstreamR:</B> gaac ggatcc atgtttattatgg (BamHI)
</LI></UL><P>Additionally, the following primers were designed to clone the chloramphenicol resistance gene of pTG262:
</P><UL><LI><B>catF:</B> atg ctgcag tctatcccggcaatag (PstI) weak no
</LI><LI><B>catR:</B> ttt gagctc tttccggcgaggc (SacI)
</LI></UL><P>PCR was performed to obtain the three products: upstream DNA with BamHI-EcoRI sites; chloramphenicol resistance gene with PstI-SacI sites; downstream DNA with PstI-SacI sites. These three fragments were then digested: 'upstream' with BamHI; 'cat' with SacI; and 'downstream' with SacI and BamHI. The products were mixed together and ligated in a triple ligation, which was used as template for a PCR reaction using primers catF and upstreamR. This generated a single PCR product of about 4 kb with this structure:
</P><P><B>PstI - CML RESISTANCE - SacI - DOWNSTREAM DNA - BamHI - UPSTREAM DNA - EcoRI</B></P><P>This was then stored as <B>Bacillobrick linear vector 1</B>. The following protocol was then used to generate linear BioBrick<SUP>TM</SUP>-containing DNA for transformation of <I>B. subtilis</I>:
</P><UL><LI> Digest BioBrick<SUP>TM</SUP> (in pSB1A2 or similar vector with EcoRI and PstI, excise the BioBrick<SUP>TM</SUP> band from a gel and purify.
</LI><LI> Digest linear vector DNA with EcoRI and PstI.
</LI><LI> Ligate the two pieces of DNA.
</LI><LI> Use a small amount of ligation reaction as template for PCR with primers upstreamF and downstreamR. This should generate a single product of size 4 kb plus the size of the BioBrick<SUP>TM</SUP>. The product should have this structure:
</LI></UL><P><B>UPSTREAM DNA - EcoRI - BIOBRICK<SUP>TM</SUP> - PstI - CML RESISTANCE - SacI - DOWNSTREAM DNA</B></P><UL><LI> Check this product on a gel, and if it looks OK, use it to transform <I>B. subtilis</I> according to a <A href="http://www.openwetware.org/wiki/Cfrench:BacTrans1" class="external text" rel="nofollow">standard procedure</A>.
</LI></UL><P>Initially we tested this protocol using BBa_J33204 which carries the promoterless reporter gene <I>xylE</I>, as described above. Chloramphenicol (10 mg/l)-resistant colonies were obtained, whereas control transformations without DNA did not yield colonies. XylE activity was not detected, but the presence of <I>xylE</I> was confirmed by colony PCR. We then repeated the procedure using a composite BioBrick<SUP>TM</SUP> BBa_J33207+BBa_J33204, which has the <I>xylE</I> reporter gene under control of a <I>lac</I> promoter. The result was the same: transformants were obtained, and the presence of <I>xylE</I> was confirmed by PCR, but no XylE activity was detected. Finally, we used composite BioBrick<SUP>TM</SUP> BBa_J33206+BBa_J33204, which, as noted above, includes the native <I>B. subtilis ars</I> promoter with the <I>xylE</I> reporter gene. The result was again the same: chloramphenicol-resistant clones carrying <I>xylE</I>, but no XylE activity detected, even with induction by arsenate, which, as noted above, led to detectable XylE activity in the plasmid version of this experiment. We therefore conclude that this method works well for introducing BioBricks<SUP>TM</SUP> into the chromosome of <I>B. subtilis</I>, but that we have yet to demonstrate expression of genes introduced in this way (although the chloramphenicol resistance gene is clearly expressed). We plan to test the use of stronger <I>B. subtilis</I> promoters and alternative loci of integration.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> de Vos, W.M. and Simons, G.F.M. 1994. Gene cloning and expression systems in Lactococci. Chapter 2 (pages 52 to 105) in 'Genetics and Biotechnology of Lactic Acid Bacteria', edited by M.J. Gasson and W.M. de Vos, Blackie Academic and Professional, London
</LI></UL><UL><LI> Sonenshein, A.L. <I>et al</I>. 1993. <I>Bacillus subtilis</I>and other Gram positive bacteria: biochemistry, physiology and molecular genetics.
</LI></UL><UL><LI>Westers, H., Dorenbos, R., van Dijl, J.M. <I>et al</I>. 2003. Genome Engineering reveals large dispensable regions in <I>Bacillus subtilis</I>. Molecular Biology and Evolution <B>20</B>, 2076-2090.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Edinburgh/Results/Bacillobricks">http://2008.igem.org/Team:Edinburgh/Results/Bacillobricks</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Results/Bacillobricks" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Results/Bacillobricks" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Results/Bacillobricks&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Results/Bacillobricks&amp;oldid=101230" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>