<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Notebook_Cellulose_degradation_and_related_notes"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Notebook/Cellulose degradation and related notes</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#3_July_2008"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">3 July 2008</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#References"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#References_2"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#References_3"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Cellulose_degradation:"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Cellulose degradation:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="3_July_2008"> 3 July 2008 </SPAN></H2><P>This as a starting place for looking up pathways:
Clostridium thermocellum ATCC 27405 Starch and sucrose metabolism
KEGG
</P><P>== 1~2 July 2008 == - Secretion of Cellulomonas cellulase enzymes
We have already obtained designed primers for all four Cellulomonas fimi cellulase genes: cenA, cenB, cenC and cex. I am currently investigating how the products of these genes might be secreted by E. coli. Based on the predictions of PSORT (psort.org) and papers documenting the expression of cenC and cex by E. coli, it would appear that existing signal sequences in the gene products result in (1) extracellular secretion in Cellulomonas fimi and (2) secretion into the periplasm of E. coli.
</P><P>Hence, my objective now is to find a means by which the gene products could be induced to cross the outer membrane of E. coli. The easiest method might be fusing our gene(s) of interest with the C-terminal region of haemolysin A, which 'contains all the information required for efficient translocation and can therefore be used as a signal sequence for recombinant protein targetting' (Mergulhao et al). Alternatively, we could use the signal peptide from E. coli's outer membrane protein A (OmpA). Signal sequences from Bacillus have also been shown to direct extracellular secretion in E. coli.
</P><P>Meanwhile, Andy has a rather exciting proposal based on last year's Paris project that has lots of potential for modelling.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>Yamabhai et al. Secretion of recombinant Bacillus hydrolytic enzymes using E. coli expression systems 
Mergulhao et al. Recombinant protein secretion in E. coli 
25 June 2008 - Cellulomonas update
'Endo- and exo-glucanases [act] synergistically in the breakdown of the cellulose chain to cellobiose and beta-glucosidase... hydrolyses the cellobiose to glucose' (O'Neill et al, Gene 44:325-330).
</P><P>Two endoglucanase genes - cenA and cenC - and an exoglucanase gene - cex - from Cellulomonas fimi have been sequenced. All three enzymes have been expressed in E. coli. We will need to order a stock of Cellulomonas fimi and design primers for cex and an endoglucanase. Please see me (Adler) for the relevant papers/sequences as they are not freely available online.
</P><P>The nucleotide sequence for Clostridium thermocellum beta-glucosidase is also available. The existence of beta-glucosidase in Cellulomonas fimi has been confirmed and a putative amino acid sequence has been released. However, I have not found any papers documenting its nucleotide sequence*. We should decide whether we want to use beta-glucosidase from Clostridium (probably easier, since we have the complete gene sequence) or Cellulomonas fimi (probably more effective, since all three enzymes would come from the same species, but also more time-consuming).
</P><H3><SPAN class="mw-headline" id="References_2">References</SPAN></H3><P>Mayer et al. Characterization of a ¿-N-acetylhexosaminidase and a ¿-N-acetylglucosaminidase/¿-glucosidase from Cellulomonas fimi 
Gräbnitz et al. Structure of the beta-glucosidase gene bglA of Clostridium thermocellum. Download PDF.
</P><UL><LI>Cellulomonas fimi beta-glucosidase [sequence|<A href="http://srs.ebi.ac.uk/srsbin/cgi-bin/wgetz?-e+" class="external free" rel="nofollow">http://srs.ebi.ac.uk/srsbin/cgi-bin/wgetz?-e+</A>[EMBL:M94865]+-newId] at EMBL - not suceeding in cleaning up this link - feel free to do so. 
</LI></UL><P>25 June 2008 - Cellulolysis by Cytophaga hutchinsonii
&quot;Many genes thought to encode proteins involved in cellulose utilization were identified. These include candidate endo-¿-1,4-glucanases and ¿-glucosidases. Surprisingly, obvious homologs of known cellobiohydrolases were not detected. Since such enzymes are needed for efficient cellulose digestion by well-studied cellulolytic bacteria, C. hutchinsonii either has novel cellobiohydrolases or has an unusual method of cellulose utilization.&quot; - Link
</P><P>C. hutchinsonii has binds to cellulolytic fibres and glides along them. It is hypothesised that it has to be attached to the medium to be able to utilise it. The enzymes involved in the cellulolysis are unknown though. There are candidate genes, but nothing concrete.
</P><P>Based on this information, I think we should concentrate on trying to introduce Cellulomonas genes into E. coli. It might be worth us questioning Chris on his PhD students' progress with C. hutchinsonii. Cellulose degradation is the most ambitious part of our plan, and we should accept that we might have to give it up in practice and pursue an alternative mentioned below.
</P><H3><SPAN class="mw-headline" id="References_3">References</SPAN></H3><P>2008: Three Microbial Strategies for Plant Cell Wall Degradation
2007: Genome Sequence of the Cellulolytic Gliding Bacterium Cytophaga hutchinsonii
</P><P>
== 24 June 2008 == - Notes from meeting with Chris
I spoke with Chris today about our plan, and this is the gist of what he had to say: We should hope to achieve in the lab 10% of what we design in theory.
</P><H3><SPAN class="mw-headline" id="Cellulose_degradation:">Cellulose degradation:</SPAN></H3><P>Cellulose degradation is really complicated. There are various different mechanisms that various different organisms use. (What is traditionally known is derived from work done on Trichoderma reesei, a fungus which is easilly cultured, but there are many other mechanisms.) Over the past ~50 years attempts to introduce cellulases into E. coli have yielded poor results. 
That isn't to say that we should give up on cellulose degredation. 
Cytophaga hutchinsoniiis a cellulolytic bacterium whose genome has been sequenced. PhD students working under Chris French are investigating cellulases from this bacterium. We could use this (needs some research). A talk by the PhD students has been suggested, and would probably be useful for us even if we have already decided by then not to use C. hutchinsonii. 
Chris isn't opposed to the idea of ordering a stock of Cellulomonas if we know the gene we need, and we can get the right strain. A strain would be best ordered from NCIMB in the UK, but there are other companies in Europe and America (from which the strain would take long to arrive and would probably be more expensive). Chris says he would be quite keen to have his PhD students working Cellulomonas anyway (hence his approvable of ordering it). 
There are alternatives to actually using our chassis organism to degrade cellulose though:
</P><P>E. coliis naturally able to metabolise pentose sugars (arabinose, xylose etc.), which are the products of hemicellulose hydrolysis. Hemicellulose is similar to cellular and present in a lot of waste biomass. It can be broken down into pentose sugars by the action of dilute acid. We could grow our modified strain of E. coli on a hemicellulose hydrolysate medium. 
E. coliis able to metabolise lactose, a sugar which can be retrieved from whey. Whey is a waste product of the dairy industry, of which, because of its biological oxygen demand (BOD), dairies have to pay companies to dispose of. So we could probably get whey for free, although there probably isn't much of it hanging around the 3rd world! 
The final idea is to used mixed cultures. This would mean using a cellulytic organism to break down cellulose. Because the demand for C, N and P are roughly in a 100:10:1 ratio, and cellulose is has them in a roughly 100:0:0 ratio, the cellulytic organisms secrete high-C waste products that can then be naturally metabolised by E. coli. Chris is currently investigating the feasibility of this, culturing E. coli with C. hutchinsonii and paper. 
Related things to consider:
</P><P>To get a bacterial gene, it's best to order a strain of said bacteria and use PCR to amplify the gene ourselves. An unfavourable alternative is to order custom-made genes from GeneArt, which would be expensive and could take as long as 2 months to get here (in theory it shouldn't take longer than a few weeks, but it was 2 months when they tried it last summer). 
Cellulase enzymes need to be secreted. The secretory pathway of E. coli is apparently poorly understand (unlike that of B. subtilis), and is unlike the canonical bacterial secretory pathway. If we are using E. coli and cloning cellulase enzymes into them, we need to find out how to secrete the enzymes. 
A method may be to fuse cellulase to the C-terminus of Haemolysin A (a secreted E. coliprotein). We could make a secretory biobrick construct whereby potentially any protein could be coupled to the N-terminal part of HA. (This would be good for the registry.)
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes">http://2008.igem.org/Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Cellulose_degradation_and_related_notes&amp;oldid=11617" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>