"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Project/Labwork Summary/17 June 08
From 2008.igem.org
iGEM 2008 Labwork Summary
Tuesday 17 June 08
Prepared competent JM109 cells using TSS method. 25 tubes of 0.2 ml were prepared, labelled 'iGEM 17-6-8 Jnn' where nn is a number from 01 to 25. These are in the pink box in Garry's -80 C freezer.  Thursday 19 June 08 Eluted DNA for BioBrick E0240 (GFP) from square 1001, well 4B and used this to transform half of tube J25; other half was transformed with 1.5 microlitres of J33207 (Edinbrick1) as a control to test the competence of the cells. Plated 100 microlitres of each to LA+amb+BW. Incubate at 37 C overnight.  Plate 1: JM109 transformation with E0240 eluted DNA. Plate 2: JM109 transformation with J33207 plasmid DNA (control).
Friday 20 June 08
Result of transformation: the positive control was highly successful, with hundreds of colonies, but no colonies were present on the plate transformed with the BioBrick DNA indicating some problem with the DNA elution. Need to check that we got the method right.  Ordered primers for dxs, appY and glgC. There are no forbidden restriction sites in the first two, so primers with full prefix and partial suffix were ordered to clone as EcoRI-SpeI fragments, but glgC has two EcoRI sites which will have to be mutated out. We therefore had a choice of cloning it initially as a XbaI-PstI fragment, or attempting the BABEL system, and decided to try BABEL first. If it proves too unreliable, we will need to order new primers to clone it the old-fashioned way.  Primer dxsf1: gat gaattc gcggccgc t *tctaga+ tg agt ttt gat att gcc Primer dxsr1: gc t actagt a tt a tta tgc cag cca ggc ctt g Primer appyf1: gat gaattc gcggccgc t tctaga tg gat tat gtt tgc tcc Primer appyr1: gct actagt a tta tt a gtc aat tgt ttt gtt tat tcc Primer glgcf1: atg gtt agt tta gag aag aac gat c Primer glgcr1: tta tta tcg ctc ctg ttt atg ccc taa c
Tuesday 24 June 08
Sarah S wants to revive one of the BioBricks from the Registry (pBad-GFP), so we'll see whether she can get it to work.
Wednesday 25 June 08
Primers arrived. Made up 500 uM stock solutions in EB and 10 uM working solutions (f and r together) in water. Performed PCR with Kod, annealing at 55 C and extending for 38 seconds (expected sizes are: dxs 1862+prefix and suffix; appY 749+prefix and suffix; glgC 1295 with no prefix or suffix (except the extra TAA). Result: all looked good, nice pure bands.  Gel 1: markers, empty lane, dsx, appY, glgC, appY (repeat), glgC (repeat), markers.
Thursday 26 June 08
Purified DNA from PCR reactions. Used 20 microlitres of glass beads and eluted to 40 microlitres of TE.  Sample P1: dsx PCR product Sample P2: appY PCR product Sample P3: glgC PCR product
Set up digests to clone appY and dxs in Edinbrick1. Digests with 32 microlitres water, 5 microlitres buffer E, 4 microlitres Edinbrick1 DNA, 4 microlitres purified PCR product, 2.5 microlitres SpeI, 2.5 microlitres EcoRI. Incubated at 37 C. Purified. Set up ligations:  Ligation L1: dsx + Edinbrick1, E/S Ligation L2: appY + Edinbrick1, E/S Ligation L3: glgC (5 ul) +linear Babel1 (16-2-8, 2 ul) with PNK Ligation L4: glgC (5 ul) +linear Babel2 (18-2-8, 2 ul) with PNK
Incubate ligations at 16 C overnight.
Friday 27 June 08
Ordered Cellulomonas fimi ATCC 484 (NCIB 8980, DSM 20113) from DSMZ.  Transformations of the iGEM competent JM109 cells with ligations L1 to L4. In each case, 100 microlitres of cells were transformed with 5 microlitres of ligation, and the remaining 5 microlitres of each ligation was frozen so that it could be analysed if the transformations fail. Fresh Blue-White Ampicillin plates were prepared.  Plate 3: Ligation L1, 100 microlitres Plate 4: Ligation L2, 100 microlitres Plate 5: Ligation L3, 100 microlitres Plate 6: Ligation L4, 100 microlitres Plate 7: Ligation L1, 900 microlitres Plate 8: Ligation L2, 900 microlitres Plate 9: Ligation L3, 900 microlitres Plate 10: Ligation L4, 900 microlitres
Saturday 28 June 08
Result of transformations:  Plate 3: 1 white; Plate 7: 5 white, 13 blue. Plate 4: no growth; Plate 8: no growth. Plate 5: 1 white; Plate 9: 3 white, 20 blue, possible signs of phage. Plate 6: maybe one tiny white; Plate 10: 7 white, 9 blue.
The white colonies were transferred to fresh plates of the same medium:
Plate 11: possible Edinbrick-dxs transformants. Plate 12: possible Babel1-glgC transformants. Plate 13: possible Babel2-glgC transformants.
The new plates were incubated at 37 C. The old plates were left at room temperature to see if any further colonies would appear.
Sunday 29 June 08
Streaks on plate 12 show strong signs of phage infection and are unusable apart from the single white one from plate 5. At least one of the streaks on plate 13 also has a couple of plaques, but plate 11 looks fine. This suggests that phage may have come from the Babel DNA stock (or the PNK, which seems unlikely). Set up overnight cultures (2.5 ml LB in 20 ml bijoux) for minipreps. Numbers 1 to 6 are Edinbrick-dxs from plate 11, all white, number 7 is from plate 12 (the white clone from plate 5), and 8 to 12 are from plate 13, white or pale blue. Incubated at 37 C with shaking.  Also note: a couple more colonies turned up on plates 5 and 6. The one on 6 looks like an escape, but the one on 5 looks OK. Subbed them both to a fresh plate (Plate 14). Previous plates, apart from plates 4 and 8 which had no growth, were transferred to the cold storage room.  Monday 30 June 08 Plasmid DNA minipreps:  M1 to M6: Edinbrick-dxs white colonies M7: Babel1-glgC white colony M8 to M12: Babel2-glgC white and pale blue colonies
Minipreps M1 to M6 were digested with EcoRI and PstI (Gel 2). The expected pattern is pSB1A2 vector band at 2.04 kb (2079 bp less prefix and suffix) and dxs insert band around 1.9 kb. M1 showed bands around 1.1 and 2.1 kb. M5 showed no DNA. The other four showed a vector-like band around 2 kb and a fainter, fuzzier band around 3 kb, possibly a 'ghost' band. Thus none of the clones show the expected pattern of bands (although it is conceivable, since the vector and insert bands are so close in size, that they may be lying on top of each other; dxs has an internal EcoRV site at +504 and two HindIII sites at +606 and +1230, whereas pSB1A2 lacks such sites so this could be used to check). In conjunction with the total lack of growth on the appY plates, this suggests that something went wrong in the cloning procedure. The next step is to run the remaining ligation material on a gel and see what it looks like.
Tuesday 1 July 2008
Gel 3: minipreps M2, M3, M4 and M6 (pSB1A2-dxs clones) digested with EcoRI alone; lanes 5 and 6, 2.5 microlitres of ligations L1 and L2. Result: all 4 minipreps now give a 4.2 kb band (plus the same 3.2 kb 'ghost' band as before) consistent with pSB1A2 carrying a 2 kb insert. Would be nice to confirm identity using HindIII (2 internal sites). Both ligations show clear signs of insert and vector bands (oddly, the dsx insert band overlaps the pSB1A2 vector band whereas the appY insert band overlaps the lacZ' insert excised from Edinbrick1). However, no ligated bands are visible. Thus the DNA purification is fine; if there was a a problem, it is with the ligase or ligase buffer.
Gel 4: minipreps M7 to M12 (supposed to be glgC in Babel vectors) digested with EcoRI and PstI to excise the inserts. M10 and M11 have a single 3 kb band consistent with vector, but no insert band at 1.2 kb. M7, M8, M9 and M12 all show a single band at about 2.4 kb which is not consistent with Babel vectors if properly digested. Unless the digests totally failed, none of these plasmids would seem to contain glgC.
Wednesday 2 July 2008
Digests of M10 and M11 with EcoRI, to determine orientation, were not very clear (Gel 7). Probably simpler just to sequence them, since we will need to check that the ends are intact in any case. Ordered mutagenic primers to remove the EcoRI sites:  primer glgcm1f: tgttgaaaaacctgctaaccc primer glgcm1r: aattcgataattttatcgttctc primer glgcm2f: ctcattctgcaacattgattcc primer glgcm2r: ttcacgcgaacgcgcgag
Thursday 3 July 2008
Clones M2, M10 and M11 were submitted for sequencing using primers pSB1A2f1 and pSB1A2r1 (for M2) and pTG262f1 and pTG262r1 (for M10 and M11). We should get the results on Monday.  Ordered primers to make carotenoid biosynthesis BioBricks:  primer crtIf2: gat gaattc gcggccgc t tctag atg aaa cca act acg gta att g 22 matches, 8 GC = 32 C, 14 AT = 28 C, total 60 C, total length = 45 83.2¿C, moderate, no
primer crtIr2: gct actagt a tta tt a tat cag atc ctc cag cat c 20 matches, 9 GC = 36, 11 AT = 22, total 58 C, length = 35 66.1¿C, weak, no
primer crtBf2: gat gaattc gcggccgc t tctag atg aat aat ccg tcg tta ctc 21 matches, 8 GC = 32, 13 AT = 26, total 58 C. length = 44 82.9¿C, moderate, no
primer crtYf2: gat gaattc gcggccgc t tctag atg caa ccg cat tat gat ctg 21 matches, 9 GC = 36,12 AT = 24, total 60 C, length = 44 86.4¿C, very strong, no
Note that we already have a compliant crtE BioBrick (I hope), and the reverse primers crtBr2 and crtYr2 should be fine (see iGEM2007 lab book, page 90).
Retransformed JM109 with the remainder of the appY ligation (I re-ligated it, and also heat-treated it before the transformation, since Tom Knight reports that this can increase transformation efficiency). Plated this to Plates 15 and 16.
Friday 4 July 2008
Repeat appY transformation has failed: no colonies on plates 15 and 16. It seems unlikely that there is a major problem with the competent cells since other transformations have worked. Other than this, the only way to get no colonies (not even recircularized vector) is if something went seriously wrong in the original digest.  Mutagenic primers for glgC arrived. Prepared 500 uM stock solutions and 10 uM working solutions. Tried PCR with both primer pairs using M11 (0.5 microlitres) as template, just to check that the primers are OK. Annealing was at 57 C (lowest melting temperature of a primer) and extension for 110 secondsusing Kod (PCR reactions P6 and P7). Products (5 microlitres) were run on Gel 8. P6 (m1) shows a single rather faint band at 4.2 kb; P7 (m2) shows multiple bands, with the 4.2 kb band strongest. Probably both pairs are fine for MABEL. Now we just need to wait for the sequencing results on Monday.  Also ran the other 5 microlitres left from the religated L2 on gel 8. No DNA was visible. Did another digest (30 microlitres total volume with 2 microlitres each P2 and Edinbrick1, using Buffer E with EcoRi and SpeI, digesting at 37 C for about 5 hours, mainly because I forgot about it). Purified and ligated: Ligation L5.  Saturday 5 July 2008. Streaked Pantoea ananatis DSM 30080 from cryogenic stock to LA (Plate 17) and incubated at 30 C so that we will have a fresh stock of cells to make the crt BioBricks. Transformed competent cells tube J21 with ligation L5 (appY-pSB1A2)(Plates 18 and 19).
Sunday 6 July 2008
Result of appY transformation: one white colony (subcultured to plate 20). Several more possible small white ones right at the edge: plate left overnight to grow a bit more.
Week 4
Monday 7 July 2008
Several more white colonies have grown on plate 19, but quite small and close together. Tried to sub these to Plate 21.  Nimisha and the team did two maxipreps - X1 (J33201), and X2 (the dxs clone of miniprep M2). X2 digested with HindIII (single restriction digest).  Purified the mutagenic PCR reactions P6 and P7, and set up self ligations (L6 and L7).  Sequence results for glgC clones M10 and M11 show that both are in reverse orientation. The result for the dxs clones M2 looks OK at first glance, but can't see restriction sites since sequencing primers pSB1A2insf1 and pSB1A2insr1 were used rather than pSB1A2f2 and pSB1A2r2.
Tuesday 8 July 2008
New primers for crt BioBricks arrived. Performed PCR with Kod using primer pairs crtBf2 (new) + crtBr2 (old)(P8), crtIf2 + crtIr2 (both new) (P9), and crtYf2 (new) + crtYr2 (old)(P10). The template for P8 and P10 was a suspension of Pantoea ananatis cells from plate 17; for P9, Douglas's maxiprep DNA from 19 Feb (with the PstI sites mutated out) was used. In all cases annealing was at 55 C and extension for 30 seconds. Gel 9 showed that P8 and P9 had worked, with pure products of the expected size seen, but no product was seen for P10.  glgC mutagenesis: Transformed competent cells with ligation products L6 and L7; plated cells on Plates 22 and 23 (L6) and Plates 24 and 25 (L7).  dxs: Ran gel (Gel 9) to check size of X1 (J33201) and X2 (pSB1A2-dxs) HindIII restriction digest. The digest looked fine.
Wednesday 9 July 2008
Results of L6 and L7 transformations: no colonies for L6, but 9 colonies were obtained from L7. These were subcultured to plates 26 and 27. By the end of the day, these had grown sufficiently for 6 overnight cultures to be set up for minipreps tomorrow. In theory, some or all of these should be glgC with EcoRI site 2 mutated out.  Minipreps M13 to M18 of possible pSB1A2-appY transformants. Digests with EcoRI/PstI were ron on Gel 10. All but M13 looked OK (insert band on M14 was so faint as to be ambiguous, but M15 to M18 all look good).  Digests and ligation L8 for addition of a synthetic ribosome binding site M2 (which has now been entered into the Registry as BBa_K118000).
Thursday 10 July 2008
Subitted M15 (possible pSB1A2-appY) for sequencing with primers pSB1A2insf2 and pSB1A2insr2.  Minipreps M19 to M24 of possible glgC mutants with EcoRI site 2 removed. EcoRI digests were run on Gel 11. The insert bands are a bit fuzzy, but it looks as though M19, M21 and M22 may have lost the site, and M20, M23 and M24 may still have it.  PCR P11 using ligation L8 as template to generate a fusion product of rbs+dxs. Run on Gel 11, last lane. No product visible.  Digested P8 and P9 (crtB and crtI(mutant) PCR products) with EcoRI/SpeI together with Edinbrick1, purified and ligated (ligations L9 and L10) at 16 C overnight.
Friday 11 July 2008
Submitted M19, M21 and M22 for sequencing using primer pTG262f1, to check for clean removal of the EcoRI site.  Revived Cellulomonas fimi in NB and plated to NA and NA+glucose. Plates 28 and 29 incubated at 30 C.  ZY and XH repeated rbs+dxs ligation (L11).  Transformation of JM109 with L9 and L10 (Edinbrick1-crtB and crtI mutant). Plates 30 to 33.  possible rbs+dxs clones from un-numbered plates were streaked to plate 34.
Saturday 12 July 2008
Result of transformation: crtB 100 microlitres, 22 white and 10 blue; 900 microlitres, too numerous to count, about two thirds white; a few blue showed signs of phage-like lysis. For crtI mutant, 100 microlitres, 4 white and 2 blue; 900 microlitres, 143 white and 25 blue. White crtB colonies were patched to Plate 35, crtI to Plate 36.  Set up overnight cultures of possible rbs+dxs clones from plate 34 for minipreps.
Sunday 13 July 2008
Signs of growth on Cellulomonas plates 28 and 29.  Minipreps M25 to M30 of possible rbs+dxs clones from plate 34.  PCR P12 to obtain rbs+dxs fusion product in case minipreps didn't work. Template was ligation L11, and primers were rbs2clonf1 and pSB1A2insr1. Annealing was 58 C, extension 40 seconds using Kod.
Week 5
Monday 14 July 08
Digests and ligation L12 to add rbs to appY clone M15 (sequence back today, seems OK).  Minipreps M31 to M42 of crtB and crtI clones. Digests with E/P.
Tuesday 15 July 08
PCR P15: fusion PCR of rbs+appY using L15 as template. Extension 20 seconds, annealing 58 C, same primer set as before. Gel 13 shows that it worked OK.  Gel 13 also shows digests of M31 to M42 from yesterday.  Transformation of JM109 with ligations L13 and L15 (glgC mutation and rbs+dxs cloning). Plated to plates 39 to 41.
Wednesday 16 July 08
Meeting with other UK iGEM teams.
Thursday 17 July 08
PCR for cenA, cenB, cenC and cex (P16, 17, 18, 19). Annealing 58 C, extension 115 seconds (cenB and cenC are both over 3 kb) and included 5% v/v glycerol in the reactions since in the past that has been helpful with high GC templates. PCR products purified to be run on gel tomorrow (against wisdom, since they should have been run on gel both before and after purification).  Results of transformations: plenty of colonies for glgC mutation (mutating site 1 in the mutant which already has site 2 mutated). Subbed some of these to plates 42, 44 and 45 (poor communications). Only a few colonies for rbs+dxs, and only one of these was white. Subbed this to plate 43. Apparently there may have been some problem with the purification of this digest - may need to repeat.   Re-digested M32, M36 (crtB), M39 and M40 (crtI) and ran on Gel 15. Gel unsuccessful ~ stain smeared over 0.5~2 kb region, exactly where crtB and crtI bands are expected to appear. Perhaps too much loading buffer or problem with the tank (tank with 'loose wire' was used).  appY+Edinbrick 1 cultures set up in beaker for maxiprep tomorrow (X3).
Friday 18 July 08
Made appY+Edinbrick 1 Maxiprep X3.  Purified PCR products for cenA, cenB, cenC and cex (P16, 17, 18, 19) run on Gel 16. No visible bands.  Second attempt at PCR for cenA, cenB, cenC and cex (P20, 21, 22, 23). Annealing 55 C, extension 85 seconds; did not include glycerol. Ran unpurified PCR products on Gel 17 ~ no bands apart from thick bands in the <0.5kb region = primers. Hence, PCR was unsuccessful.
Saturday 19 July 08
Minipreps M43 to M47 of possible glgC double mutants (both EcoRI sites gone), and M48 of the only white colony from the rbs+dxs transformation. Gel 18 shows E/P digests of 3 microlitres. M43 to M47 all look about right: send one for sequencing. M48 is ambiguous, could be incomplete deigestion, need to repeat. (CF)  PCR P24 and P25, repeat PCR for cenA and cex (the two shorter genes) usingPfu polymerase instead of Kod, adding 5% glycerol - this worked well for high-GC genes from Saccharothrix espanaensis last year. (CF)
Sunday 20 July 08
Ran P24 and P25 on Gel 19. No amplification products. Time to do some trouble shooting. The fault is in either the primers, the template, or the reactions. All previous reactions have been fine, so most likely the primers or template. To check this, we need to do a positive control using universal primers such as fd1 and rd1 (amplify 16S rRNA gene rrnB) which should definitely give a product. If these fail, then it is most likely that the template is at fault - we should prepare genomic DNA and try using this as template rather than cells; also do a Gram stain to confirm that the cells look OK. If the control reaction works, try remaking the primer solutions, though it seems unlikely that all four sets could be wrong, unless I was having a really bad day. (CF)  Redid digest of M48 with SacI-Spe1. Does not look good. Also repeated digests of crtB and crtI clones M36 and M42; alternating single digests (E) and double digests (EP) to check for inserts. Look OK. Submit for sequencing. (CF)  Also: I have a plan for in vitro assembly to cut down on transformations and minipreps. Could have a try of this next week if there are some students with some free time.
Week 6
Summary of samples
PCR products
P1: dxs from E. coli JM109 (or K12?) P2: appY from E. coli JM109 (or K12?) P3: glgC from E. coli JM109 (or K12?) P4: fusion PCR for Babel1-glgC P5: fusion PCR for Babel2-glgC P6: mutagenic PCR to remove EcoRI site 1 from glgC P7: mutagenic PCR to remove EcoRI site 2 from glgC P8: crtB (from P. ananatis cells) P9: crtI (from Douglas's maxiprep of the mutated crtIB plasmid) P10: crtY (from P. ananatis cells). Failed. P11: fusion PCR for rbs+dxs. Failed. P12: repeat fusion PCR for rbs+dxs, using L11 and primers rbs2fclon1+pSB1A2insr1. P13 and P14: MABEL mutation of site 1 on glgC mutant clones M19 and M22. P15: fusion PCR for rbs+appY P16, P17, P18 and P19: first attempts at cenA, cenB, cenC and cex. P20, P21, P22 and P23: second attempts at cenA, cenB, cenC and cex. P24, P25: repeat PCR for cenA and cex using Pfu. Failed.
Ligations
L1: Edinbrick1-dxs L2: Edinbrick1-appY L3: Babel1-glgC L4: Babel2-glgC L5: Edinbrick1-appY repeat (initially labelled L4 in error) L6: Self ligation of mutagenic PCR P6 L7: Self ligation of mutagenic PCR P7 L8: Ligation of synthetic ribosome binding site to M2 (BBa_K118000) L9: Edinbrick1-crtB L10: Edinbrick1-crtI (mutant lacking PstI sites, I hope) L11: repeat ligation of synthetic ribosome binding site to M2 (BBa_K118000) L12: rbs ligated with appY. L13 and L14: self ligations of glgC mutation reactions P13 and P14. L15: ligation of Edinbrick1 with rbs+appY fusion PCR product.
Minipreps
M1 to M6: pSB1A2-dxs transformants. M2, M3, M4 and M6 all looked correct on a gel. M2 was sequenced to confirm identity and was maxiprepped as X2. M7: pBabel1-glgC transformant. Did not look right on gel. M8 to M12: pBabel2-glgC transformants. M10 and M11 both looked good on a gel, but sequencing showed that both had the insert in the reverse orientation. M13 to M18: pSB1A2-appY transformants. Gel 10 showed that M15 to M18, and possibly M14, all seemed to have inserts the right size. M19 to M24: Babel2-glgC clones after mutation of EcoRI site 2. EcoRI digests on Gel 11 suggest that M19, M21 and M22 may have lost the EcoRI site. M25 to M30: possible rbs+dxs transformants. M31 to M36: crtB clones. M37 to M42: crtI clones. M43 to M47: possible glgC double mutants (both EcoRI sites gone) M48: a possible rbs+dxs clone.
Maxipreps
X1: BioBrick J33201 X2: pSB1A2-dxs clone (as M2; BBa_K118000) X3: pSB1A2-appY clone (as M15; BBa_K118001)
Plates
1 and 2: transformation with a BioBrick from the Registry stock: failed. 3 and 7: L1 transformation, 100 and 900 microlitres 4 and 8: L2 transformation, 100 and 900 microlitres. Failed. 5 and 9: L3 transformation, 100 and 900 microlitres 6 and 10: L4 transformation, 100 and 900 microlitres Plate 11: possible Edinbrick-dxs transformants. Plate 12: possible Babel1-glgC transformants. Plate 13: possible Babel2-glgC transformants. Plate 14: more subcultures from plates 5 and 6. Plates 15 and 16: retransformation of religated L2. Failed again. Plate 17: Pantoea ananatis for crt PCR reactions. Plates 18 and 19: L5 transformation, 100 and 900 microlitres. Plates 20 and 21: subculturees from 18 and 19. Plates 22 and 23: L6 transformation. No colonies. Plates 24 and 25: L7 transformation. Plates 26 and 27: subcultures from plates 24 and 25 (L7). Plates 28 and 29: Cellulomonas from initial stock. Plates 30 and 31: possible pSB1A2-crtB transformants. Plates 32 and 33: possible pSB1A2-crtI(mutant) transformants. Plate 34: possible rbs+dxs clones from Nimisha's plates. Plate 35: patches of possible pSB1A2-crtB transformants from Plate 30. Plate 36: patches of possible pSB1A2-crtI(mutant) transformants from Plates 32 and 33. Plate 37: subcultuure of Cellulomonas fimi on NA with filter paper (cellulose) Plates 38 and 39: L15 transformation (rbs+dxs PCR product with Edinbrick1) Plates 40 and 41: L13 transformation (glgC M19 PCR produce to mutate site 1) Plates 42, 44 and 45: subcultures from plate 41 (glgC mutation) Plate 43: subculture from plate 39 (possible rbs+dxs)
Gels
Gel 1: PCR reactions P1, P2, P3 Gel 2: minipreps M1 to M6, EcoRI/PstI Gel 3: minipreps M2, M3, M4, M6, EcoRI, plus L1 and L2 Gel 4: minipreps M7 to M12, undigested Gel 5: minipreps M7 to M12, EcoRI/PstI Gel 6: minipreps M2, M3, M4, M6, HindIII; also P4 and P5 (fusion PCR) Gel 7: minipreps M10 and M11, EcoRI (two versions of this gel) Gel 8: PCR products P6 and P7 (mutagenic PCR on M11) Gel 9: maxipreps X1 and X2, etc. Gel 10: minipreps M13 to M18, E/P digests. Gel 11: minipreps M19 to M24, EcoRI digests, plus PCR P11. Gel 12: minipreps M25 to M30 of possible rbs+dxs clones: none look right. Gel 13: PCR P13 and P14 to mutated second EcoRI site on glgC clones M19 and M22. Gel 14: minipreps M31 to M42 (crtB and crtI clones) E/P, and P15 (rbs+appY) Gel 15: redigest of M32, M36 (crtB), M39, M40 (crtI) E/P; unsuccessful. Gel 16: purified PCR products for cenA, cenB, cenC, cex (P16, 17, 18, 19); unsuccessful. Gel 17: PCR products for cenA, cenB, cenC, cex (P20, 21, 22, 23) ~ second attempt; unsuccessful.
Retrieved from " http://2008.igem.org/Team:Edinburgh/Project/Labwork_Summary/17_June_08 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
