<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Plan"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Edinburgh/Plan">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Plan&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Plan&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Plan&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Plan" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Plan</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="header"><TITLE>Edinburgh iGEM 2008</TITLE><DIV id="colortab" class="ddcolortabs"><UL><LI><A href="https://2008.igem.org/Team:Edinburgh" title="Home"><SPAN>Home</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Plan" title="Plan of the biology" rel="dropmenu1_a"><SPAN>Plan</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Protocols" title="New Protocols developed" rel="dropmenu1_a"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Notebook" title="Details of labwork, in realtime"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Results" title="Results" rel="dropmenu1_a"><SPAN>Results</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Parts" title="Our submitted BioBrick&lt;sup&gt;TM&lt;/sup&gt; parts" rel="dropmenu1_a"><SPAN>Parts</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Modelling" title="Modelling" rel="dropmenu2_a"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Software" title="Software" rel="dropmenu2_a"><SPAN>Software</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/ELSI" title="Ethical, legal and social implications of our project"><SPAN>ELSI</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/FAQ" title="Frequently asked questions"><SPAN>FAQ</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Team" title="Meet the team"><SPAN>The Team</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Acknowledgements" title="Acknowledgements"><SPAN>Acknowledgements</SPAN></A></LI></UL></DIV><DIV class="ddcolortabsline"> </DIV><DIV id="dropmenu1_a" class="dropmenudiv_a"><A href="https://2008.igem.org/Team:Edinburgh/Team">Overview</A><A href="https://2008.igem.org/Team:Edinburgh/Team">Step1</A><A href="https://2008.igem.org/Team:Edinburgh/Team">Step2</A></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#An_ambitious_plan"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">An ambitious plan</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_chassis:_E._coli_vs._B._subtilis"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The chassis: E. coli vs. B. subtilis</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Primary_Objective:_The_production_of_starch_from_cellulose"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Primary Objective: The production of starch from cellulose</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Secondary_Objective:_The_synthesis_of_.CE.B2-carotene"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Secondary Objective: The synthesis of β-carotene</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Further_considerations"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Further considerations</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#An_inducible_RNAse"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">An inducible RNAse</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Lemon_flavour"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Lemon flavour</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#System_overview"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">System overview</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Cellulose_degradation"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Cellulose degradation</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Bacterial_cell_lysis"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Bacterial cell lysis</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Glycogen_overproduction"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Glycogen overproduction</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Starch_biosynthesis"><SPAN class="tocnumber">1.6.4</SPAN><SPAN class="toctext">Starch biosynthesis</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Synthesis_of_carotenoids"><SPAN class="tocnumber">1.6.5</SPAN><SPAN class="toctext">Synthesis of carotenoids</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Synthesis_of_limonene_and_other_terpenes"><SPAN class="tocnumber">1.6.6</SPAN><SPAN class="toctext">Synthesis of limonene and other terpenes</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="An_ambitious_plan"> An ambitious plan </SPAN></H1><H2><SPAN class="mw-headline" id="The_chassis:_E._coli_vs._B._subtilis"> The chassis: <I>E. coli</I> vs. <I>B. subtilis</I></SPAN></H2><P>We were initially torn between using <I>Escherichia coli</I> or <I>Bacillus subtilis</I> as a chassis for development of MicroMaize:
</P><UL><LI> For the labwork we decided to use <I>E. coli</I> JM109 as the chassis. This was due to the greater availability of tested registry parts for <I>E. coli</I> compared to <I>B. subtilis</I>, a higher level of understanding of the biochemistry of <I>E. coli</I>, and more in-house experience of engineering <I>E. coli</I>.
</LI></UL><UL><LI> We decided that <I>B. subtilis</I> would have advantages in the case of our system working and industrialisation being an option. This was because of increased ease of transport (due to its stable spore-forming nature), its having a better characterised secretory system and being regarded in higher favour than <I>E. coli</I> by the general public (see section of ethical, legal and social implications). Another possibility for the future would be to port the system to yeast, <I>Saccharomyces cerevisiae</I>; this would be more difficult to achieve but might ultimately be more desirable as yeast is Generally Regarded As Safe for food use, and as a eukaryote, might be able to accumulate larger amounts of starch than bacteria.
</LI></UL><H2><SPAN class="mw-headline" id="Primary_Objective:_The_production_of_starch_from_cellulose"> Primary Objective: The production of starch from cellulose </SPAN></H2><P><B>Firstly, why starch?</B></P><P>Although one of our aims is to provide a substrate for the biofuels industry, which can easily ferment glucose to ethanol, we decided to try to synthesise starch instead, even though this requires extra steps. The reason for this was a question of purification: glucose is soluble, and therefore difficult to purify from solution, as is glycogen. Starch on the other hand is highly insoluble making purification of this end product relatively straight forward. As well as this, glucose becomes inhibitory at high concentrations due to osmotic pressure, and may also cause product inhibition of glucose-producing reactions. Starch, being insoluble, can potentially accumulate at much higher quantities without disrupting the biochemistry of the cell. 
</P><P><B>The production of starch from cellulose requires two main steps:</B></P><OL><LI><A href="https://2008.igem.org/Team:Edinburgh/Plan/Cellulolysis" class="external text" rel="nofollow">Cellulolysis</A></LI><LI><A href="https://2008.igem.org/Team:Edinburgh/Plan/StarchSynthesis" class="external text" rel="nofollow">Starch synthesis</A></LI></OL><H2><SPAN class="mw-headline" id="Secondary_Objective:_The_synthesis_of_.CE.B2-carotene"> Secondary Objective: The synthesis of β-carotene </SPAN></H2><P>β-carotene is the pigment that gives carrots their orange colour. It is produced as an intermediate of the lycopene synthesis pathway and is converted into vitamin A by the human body. We added a number of genes from <I>Pantoea ananatis</I> (formerly <I>Erwinia uredovora</I>) to <I>E. coli</I>, in essence transferring the beta-carotene synthesis pathway of <I>P. ananatis</I> to our chasis organism. We also cloned two <I>E. coli</I> genes in order to upregulate the transgenic β-carotene synthesis pathway.
</P><DL><DD><UL><LI><A href="https://2008.igem.org/Team:Edinburgh/Plan/Beta-Carotene" class="external text" rel="nofollow">Overview of the genes and pathway</A></LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Further_considerations"> Further considerations </SPAN></H2><P>As well as designing the major components of our system, we had to consider how we would make MicroMaize a safe and appealling product (see <A href="https://2008.igem.org/Team:Edinburgh/ELSI" class="external text" rel="nofollow">ELSI page</A> for more details).
</P><H3><SPAN class="mw-headline" id="An_inducible_RNAse"> An inducible RNAse </SPAN></H3><P>Although we can easilly kill <I>E. coli</I> through heat treatment, the cells contain a lot of RNA, which can cause gout. To overcome this we decided to use the <A href="https://2007.igem.org/BerkiGEM2007Present5" class="external text" rel="nofollow">&quot;genetic self-destruct&quot; system</A> (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I716463" class="external text" rel="nofollow">BBa_I716463</A>) developed by <A href="https://2007.igem.org/Berkeley_UC" class="external text" rel="nofollow">UC Berkley's '07 team</A>.
Briefly, this is an arabinose-inducible RNAse. Its use for us is that we can grow our cells to the required starch or β-carotene content, then induce transcription of the RNAse Barnase. Barnase will then hydrolyse the RNA into single nucleotides. We envisage that these will then diffuse through the bacterial cell wall after disruption through heat treatment.
</P><H3><SPAN class="mw-headline" id="Lemon_flavour"> Lemon flavour </SPAN></H3><P>We decided that to make MicroMaize more appealling we would make it smell and taste of lemons. To do this we used <A href="http://partsregistry.org/wiki/index.php/Part:BBa_I742111" class="external text" rel="nofollow">BioBrick<SUP>TM</SUP> I742111</A> developed by <A href="https://2007.igem.org/Edinburgh/Yoghurt/Design#Lemon_Flavour_Production" class="external text" rel="nofollow">Edinburgh '07</A>. This is the BioBrick<SUP>TM</SUP> for <I>LIMS1</I>, which encodes Limonene Synthase of <I>Citrus limon</I>. Limonene synthase catalyses the production of (+) limonene (thought to be responsible for ~90% of the lemon taste) from geranyl diphosphate.
</P><P>To complement and increase β-carotene synthesis we decided to incorporate <I>LIMS1</I> into a short operon, preceeded by <I>dxs</I> and followed by <I>appY</I>.
</P><H2><SPAN class="mw-headline" id="System_overview"> System overview </SPAN></H2><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><H3><SPAN class="mw-headline" id="Cellulose_degradation">Cellulose degradation</SPAN></H3><UL><LI> Chaudhary, P., Kumar, N.N., Deobagkar, D.N. 1997. The glucanases of <I>Cellulomonas</I>. <I>Biotechnology Advances</I><B>15</B>(2), 315-331.
</LI></UL><UL><LI> Lam, T.L., Wong, R.S.C., Wong, W.K.R. 1997. Enhancement of extracellular production of a <I>Cellulomonas fimi</I> exoglucanase in <I>Escherichia coli</I> by the reduction of promoter strength. <I>Enzyme and Microbial Technology</I><B>20</B>, 482-488. (An approach to achieving secretion of cellulose-degrading enzymes in <I>E. coli</I>).
</LI></UL><UL><LI> Lynd, L.R., Weimer, P.J., van Zyl, W.H., and Pretorius, I.S. 2002. Microbial Cellulose Utilization: Fundamentals and Biotechnology. <I>Microbiology and Molecular Biology Reviews</I><B>66</B>, 506-577. (An excellent review by the leading researchers in the field).
</LI></UL><UL><LI> Stoll, D. 2001. Mapping of genes encoding glycoside hydrolases on the chromosome of <I>Cellulomonas fimi</I>. <I>Canadian Journal of Microbiology</I><B>47</B>, 1063-1067. (Some information on the genome size and known cellulose-degrading enzymes of the cellulolytic bacterium <I>C. fimi</I>).
</LI></UL><UL><LI> Tomme, P., Kwan, E., Gilkes, N.R., Kilburn, D.G., and Warren, R.A.J. 1996. Characterization of CenC, an enzyme from Cellulomonas fimi with both endo- and exoglucanase activities. <I>Journal of Bacteriology</I><B>178</B>, 4216-4223. (A potentially useful enzyme from <I>C. fimi</I>, which we unfortunately did not have time to pursue in this project, as it contains multiple forbidden restriction sites; maybe something to look at for the future).
</LI></UL><UL><LI> Wilson, D.B. 2008. Three Microbial Strategies for Plant Cell Wall Degradation. <I>Annals of the New York Academy of Sciences</I><B>1125</B>, 289-297.
</LI></UL><UL><LI> Xie, G., Bruce, D.C., Challacombe, J.F., Chertkov, O., Detter, J.C., Gilna, P., Han, C.S., Lucas, S., Misra, M., Myers, G.L., Richardson, P., Tapia, R., Thayer, N., Thompson, L.S., Brettin, T.S., Henrissat, B., Wilson, D.B., McBride, M.J. 2007. Genome sequence of the cellulolytic gliding bacterium <I>Cytophaga hutchinsonii</I>. <I>Applied and Environmental Microbiology</I><B>73</B>(11), 3536-3546.
</LI></UL><H3><SPAN class="mw-headline" id="Bacterial_cell_lysis"> Bacterial cell lysis </SPAN></H3><UL><LI> Barnard. A., Wolfe, A. and Busby, S. 2004. Regulation at complex bacterial promoters: how bacteria use different promoter organizations to produce different regulatory outcomes. <I>Current Opinion in Microbiology</I><B>7</B>(2), 102-108.
</LI></UL><UL><LI> Berka, R.M., Hahn, J., Albano, M., Draskovic, I., Persuh, M., Cui, X., Sloma, A., Widner, W. and Dubnau, D. 2002. Microarray analysis of the <I>Bacillus subtilis</I> K-state: genome-wide expression changes dependent on ComK. <I>Molecular Microbiology</I><B>43</B>(5), 1331-1345. (Discription of a transcription factor that activates its own expression in <I>B. subtilis</I>.)
</LI></UL><UL><LI> Rhodius, V.A. and Busby, S.J.W. 1998. Positive activation of gene expression. <I>Current Opinion in Microbiology</I><B>1</B>(2), 152-159.
</LI></UL><UL><LI> Snyder, L. and Champness, W. 2007. Molecular Genetics of Bacteria (3rd Edition). ASM Press
</LI></UL><UL><LI> Young, R., Wang, I.-N. and Roof, W.D. 2000. Phages will out: strategies of host cell lysis. <I>Trends in Microbiology</I><B>8</B>(3), 120-128.
</LI></UL><H3><SPAN class="mw-headline" id="Glycogen_overproduction">Glycogen overproduction</SPAN></H3><UL><LI> Damotte M., Cattanéo, J., Sigal, N. and Puig, J. 1968. Mutants of <I>Escherichia coli</I> K12 altered in theiry ability to store glycogen. <I>Biochemical and Biophysical Research Communications</I><B>32</B>(6), 916-920. (Description of a quantitative glycogen assay.)
</LI></UL><UL><LI> Dedhia, N., Chen, W., and Bailey, J.E. 1996. Design of expression systems for metabolic engineering: coordinated synthesis and degradation of glycogen. <I>Biotechnology and Bioengineering</I><B>55</B>, 419-426.
</LI></UL><UL><LI> Eydallin, G., Viale, A.M., Moran-Zorzano, M.T., Munoz, F.J., Montero, M., Baroja-Fernandez, E., and Pozueta-Romero, J. 2007. Genome-wide screening of genes affecting glycogen metabolism in <I>Escherichia coli</I> K-12. <I>FEBS Letters</I><B>581</B>, 2947-2953.
</LI></UL><UL><LI> Fernandez-Banares, I., Clotet, J., Arino, J., and Guinovart, J.J. 1991. Glycogen hyperaccumulation in <I>Saccharomyces cerevisiae RAS2</I> mutant - a biochemical study. <I>FEBS Letters</I><B>290</B>, 38-42. (A potential method for overproducing glycogen in yeast, if we ultimately decided to go down that route rather than using a bacterial host).
</LI></UL><UL><LI> Govons, S., Vinopal, R., Ingraham, J. and Preiss, J. 1969. Isolation of mutants of <I>Escherichia coli</I> B altered in their ability to synthesize glycogen. <I>Journal of Bacteriology</I><B>97</B>(2), 970-972. (Description of an assay to quantify glycogen production by <I>E. coli</I>.)
</LI></UL><UL><LI> Leung, P., Lee, Y.M., Greenberg, E., Esch, K., Boylan, S., and Preiss, J. 1986. Cloning and expression of the <I>Escherichia coli glgC</I> gene from a mutant containing an ADP-glucose pyrophsophorylase with altered allosteric properties. <I>Journal of Bacteriology</I><B>167</B>, 82-88. (Initial description of the <I>glgC16</I> mutant.)
</LI></UL><UL><LI> Meyer, C.R., Bork, J.A., Nadler, S.N., Yirsa, J. and Preiss, J. 1998. Site-Directed Mutagenesis of a Regulatory Site of <I>Escherichia coli</I> ADP-Glucose Pyrophosphorylase: The Role of Residue 336 in Allosteric Behaviour. <I>Archives of Biochemistry and Biophysics</I><B>351</B>(1), 152-159. (Further description of the <I>glgC16</I> mutant.)
</LI></UL><H3><SPAN class="mw-headline" id="Starch_biosynthesis">Starch biosynthesis</SPAN></H3><UL><LI> Ball, S.G. and Morell, M.K. 2003. From bacterial glycogen to starch: understanding the biogenesis of the plant starch granule. <I>Annual Reviews in Plant Biology</I><B>54</B>, 207-233. (A comprehensive review of the state of knowledge regarding glycogen and starch biosynthesis in bacteria and plants).
</LI></UL><UL><LI> Delatte, T., Trevisan, M., Parker, M.L., and Zeeman, S.C. 2005. Arabidopsis mutants Atisa1 and Atisa2 have identical phenotypes and lack the same multimeric isoamylase, which influences the branch point distribution of amylopectin during starch synthesis. <I>Plant Journal</I><B>41</B>, 815-830. (Probable role of the heteromeric isoamylase in converting a glycogen-like precursor to starch).
</LI></UL><UL><LI> Wattebled, F., Dong, Y., Dumez, S., Delvalle, D., Planchot, R., Berbezy, P., Vyas, D., Colonna, P., Chatterjee, M., Ball, S., and D'Hulst, C. 2005. Mutants of Arabidopsis lacking a chloroplastic isoamylase accumulate phytoglycogen and an abnormal form of amylopectin. <I>Plant Physiology</I><B>138</B>, 184-195. (Probable role of the heteromeric isoamylase in converting a glycogen-like precursor to starch).
</LI></UL><UL><LI> Kubo, A., Rahman, S., Utsumi, Y., Li, Z.Y., Mukai, Y., Yamamoto, M., Ugaki, M., Harada, K., Satoh, H., Konik-Rose, C., Morell, M., and Nakamura, Y. 2005. Complementation of sugary-1 phenotype in rice endosperm with the wheat isoamylase1 in gene in supports a direct role for isoamylase1 amylopectin biosynthesis. <I>Plant Physiology</I><B>137</B>, 45-36. (Probable role of the heteromeric isoamylase in converting a glycogen-like precursor to starch).
</LI></UL><UL><LI> Utsumi, Y., and Nakamura, Y. 2006. Structural and enzymatic characterization of the isoamylase1 homo-oligomer and the isoamylase1-isoamylase2 hetero-oligomer from rice endosperm. <I>Planta</I><B>225</B>, 75-87. (Expression of functional isoamylase subunits in <I>E. coli</I> shows that this enzyme can be expressed in a functional form in bacteria).
</LI></UL><H3><SPAN class="mw-headline" id="Synthesis_of_carotenoids">Synthesis of carotenoids</SPAN></H3><UL><LI> Sandmann, G. 2006. Production of carotenoids by gene combination in <I>Escherichia coli</I>. pages 143-153 in 'Food Biotechnology', 2nd edition, eds. K. Shetty, G. Paliyath, A. Pometto and R.E. Levin, CRC Press, Taylor &amp; Francis group, Boca Raton, FL. (A useful review).
</LI></UL><UL><LI> Misawa, N., Nakagawa, N., Kobayashi, K., Yamano, S., Nakamura, K., and Harashima, K. 1990. Elucidation of the <I>Erwinia uredovora</I> carotenoid biosynthetic pathway by functional analysis of gene products expressed in <I>Escherichia coli</I>. <I>Journal of Bacteriology</I><B>172</B>, 6704-612. (Demonstration of the functional expression of the <I>Pantoea ananatis (= Erwinia uredovora)</I> genes in <I>E. coli</I>)
</LI></UL><UL><LI> Kang, M.J., Lee, Y.M., Yoon, S.H., Kim, J.H., Ock, S.W., Jung, K.H., Shin, Y.C., Keasling, J.D., and Kim, S.W. 2005. Identification of genes affecting lycopene accumulation in <I>Escherichia coli</I> using a shot-gun method. <I>Biotechnology and Bioengineering</I><B>91</B>, 636-642 (role of <I>dxs</I> and <I>appY</I> in increasing carotenoid biosynthesis).
</LI></UL><H3><SPAN class="mw-headline" id="Synthesis_of_limonene_and_other_terpenes">Synthesis of limonene and other terpenes</SPAN></H3><UL><LI> Lucker, J., El Tamer, M.K., Schwab, W., Verstappen, F.W.A., van der Plas, L.H.W., Bouwmeester, H.J., and Verhoeven, H.H. 2002. Monoterpene biosynthesis in lemon (<I>Citrus limon</I>). <I>European Journal of Biochemistry</I><B>269</B>, 3160-3171. (Functional expression of the LIMS1 gene in <I>E. coli</I>).
</LI></UL><UL><LI> Reiling, K.K., Yoshikuni, Y., Martin, V.J.J., Newman, J., Bohlmann, J., and Keasling, J.D. 2004. Mono and diterpene production in <I>Escherichia coli</I>. <I>Biotechnology and Bioengineering</I><B>87</B>, 200-212. (Generation of various monoterpenes, including limonene, in <I>E. coli</I> and methods for detection).
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Edinburgh/Plan">http://2008.igem.org/Team:Edinburgh/Plan</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Plan" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Plan" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Plan&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Plan&amp;oldid=100985" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>