"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Edinburgh/Protocols/Edinbrick1
From 2008.igem.org
< Back to Protocols index
Use of Edinbrick1 in preparation of BioBricksTM from PCR products
Edinbrick1 was developed by the Edinburgh 2006 iGEM team and was used extensively in both 2007 and 2008 to facilitate BioBrickTM construction. Although it was not developed by the 2008 team, we describe it here to increase awareness of it in case anyone else might find it useful.
Edinbrick1 consists of the standard BioBrickTM vector pSB1A2 with insert BBa_J33207 which includes a lac promoter and lacZ' minigene encoding the N-terminal 77 amino acids of the beta-galactosidase gene from E. coli B. A SacI site, GAGCTC, has also been added at the start of this insert overlapping the XbaI site (TCTAGA) of the prefix. Edinbrick1 is designed to facilitate cloning of PCR products in pSB1A2 to make BioBricksTM. The standard protocol we use is as follows:
Design primers for the PCR. One primer should contain a full prefix or suffix, the other a partial prefix or suffix. Thus the PCR product can be cloned as an EcoRI/SpeI or XbaI/PstI fragment. Alternatively, if there is no internal SacI site in the insert, and if you do not have a precise requirement for the first few bases of the insert (as with coding sequence BioBricksTM), shorter primers can be used with a SacI site in the forward primer and an SpeI site in the reverse primer.
Perform PCR, test the reaction product on a gel, and purify it.
Set up a single-tube digestion reaction which contains both Edinbrick1 vector and the PCR product with the appropriate set of restriction enzymes: either EcoRI/SpeI, XbaI/PstI, or SacI/SpeI.
When the digestion is complete, purify the DNA using your favorite method (we use glass beads and sodium iodide) and add ligase buffer and T4 DNA ligase.
Transform E. coli with the ligation mixture and plate on LB plates with ampicillin, Xgal and IPTG. Blue colonies should be Edinbrick1, white colonies should be pSB1A2 with the desired insert replacing the BBa_J33207 insert of Edinbrick1.
In our hands this procedure is quick and reliable and does not require purification of bands from a gel.
One problem with this procedure is that it cannot be used if the BioBrickTM being made includes lacZ, as in this case the desired colonies would also be blue. To get around this, Edinbrick2 (BBa_J33204 in pSB1A2) can be used. This includes xylE from Pseudomonas putida S13, which encodes catechol-2,3-dioxygenase, a reporter gene which converts catechol (a cheap, colourless, soluble subsrate) to bright yellow 2-hydroxy-cis,cis-muconic semialdehyde. BBa_J33204 does not include a promoter but we have found that in pSB1A2 (which lacks the flanking terminators of pSB1A3) there is sufficient expression in colonies to detect the activity - just add a drop of 10 mM catechol in water to each colony, and those which are XylE+ will go bright yellow within a few minutes. Of course, if you are confident that lacZ' will be expressed in your construct, you can just plate on Xgal and choose the blue colonies.
Retrieved from " http://2008.igem.org/Team:Edinburgh/Protocols/Edinbrick1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
