"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Edinburgh/Protocols/Bacillobricks
From 2008.igem.org
< Back to Protocols index
Contents
1 Bacillobricks: Introduction of BioBricksTM into Bacillus subtilis
1.1 Introduction of BioBricksTM into B. subtilis on a plasmid
1.2 Integration of BioBricksTM onto the B. subtilis chromosome
1.3 References
Bacillobricks: Introduction of BioBricksTM into Bacillus subtilis
Bacillus subtilis is potentially superior to E. coli as a host for some projects, for several reasons:
It is much more effective at secreting proteins into the medium, as E. coli lacks the Main Terminal Branch of the General Secretory Pathway.
As a Gram positive bacterium, it lacks the toxic lipopolysaccharide (endotoxin) of Gram negative bacteria such as E. coli
The cells are considerably larger, making it easier to visualise intracellular components.
B. subtilis forms endospores, a highly stable, heat and dessication resistant resting state which can be stored dry for years or decades, and will then germinate in less than 30 minutes when added to a suitable growth medium.
B. subtilis is not pathogenic and has even been used as a probiotic organism in human foods.
However, standard BioBrickTM vectors do not allow introduction of BioBricksTM into B. subtilis. We felt that B. subtilis was potentially a suitable host for our 'Edinburgh Process' of conversion of cellulose to starch, due mainly to its ability to secrete enzymes such as cellulases. We therefore investigated two processes to allow introduction of BioBricksTM to B. subtilis, either on a plasmid or by integration into the genome. These experiments were carried out mainly by C. French (instructor) and Nimisha Joshi (advisor).
Introduction of BioBricksTM into B. subtilis on a plasmid
A standard method of transferring BioBricksTM between E. coli and B. subtilis would be the use of a shuttle vector which could replicate in both organisms, and in fact the Edinburgh iGEM 2007 team did demonstrate this using the Lactobacillus plasmid pTG262. pTG262 replicates in both E. coli and B. subtilis from the same versatile origin of replication, and has a multi-cloning site with EcoRI and PstI sites allowing convenient incorporation of BioBricksTM. pTG262 was submitted to the Registry as BBa_I742103, but our understanding, from one team that tried to acquire it from the Registry, was that it is not currently available as the transformation failed. Which brings us to the problem with pTG262 - it does not transform lab strains of E. coli with high efficiency, at least not in our hands. (For more information about pTG262 and its ancestors pWV01 and pSH71, see deVos and Simons (1994), and this summary .
To get around this problem, we decided to check whether we could ligate a BioBrickTM into pTG262 and then transform B. subtilis directly with the ligation mixture, rather than initially preparing DNA from E. coli. Initially we tested this idea using BioBrickTM BBa_J33204, bearing the reporter gene xylE encoding catechol-2,3-dioxygenase (which converts catechol, a cheap, colourless substrate, to bright yellow 2-hydroxy-cis,cis-muconic semialdehyde). We have previously demonstrated that this reporter gene works well in B. subtilis. The insert was cut out with EcoRI and PstI and ligated with pTG262 cut with the same two enzymes. The ligation was then used to transform B. subtilis using a standard procedure and cells were plated on LB with chloramphenicol (10 mg/l). Chloramphenicol-resistant colonies were obtained, and the presence of xylE was confirmed by PCR, but no XylE activity was detected (ie, no visible yellow colour on addition of a drop of 10 mM catechol to the colonies). It was therefore concluded that BioBricksTM can be introduced into B. subtilis by this method, and that expression does not occur in the absence of a promoter.
The experiment was repeated using a composite BioBrickTM consisting of BBa_J33207 (lac promoter from E. coli) with BBa_J33204 (xylE reporter gene). Again, chloramphenicol-resistant colonies were obtained but no XylE expression was observed, suggesting that this promoter was not active in this context, even though the lacI repressor gene was not present.
To check that expression could be achieved, we finally turned to the only BioBricked native B. subtilis promoter in our freezer, BBa_J33206 (B. subtilis ars promoter, induced by sodium arsenate, cloned as part of the Edinburgh iGEM 2006 arsenic biosensor project). We made a composite BioBrickTM consisting of J33206+J33204, ligated this with pTG262 and transformed B. subtilis as above. Since the pSB1A2 vector band could not be separated from the Pars+xylE BioBrickTM on a gel (as they were the same size), it was expected that half of the colonies would contain the correct insert. In fact, one of four clones tested showed evidence of XylE activity (ie, a yellow pigment produced when a drop of 10 mM catechol was added to a colony) and only this clone showed the presence of xylE by PCR.
A plasmid of the expected size was detected in plasmid DNA preps from this clone, but since the construct contains a significant amount of native B. subtilis sequence, we cannot at present exclude the possibility that the construct may have integrated into the genome at the ars locus. This can be tested by PCR to check the size of this locus.
Interestingly, this clone showed highly sensitive arsenic-dependent induction of XylE activity and was capable of detecting arsenic at the WHO recommended threshold level of 10 ppb, unlike our previous attempts at non-BioBrickTM-based B. subtilis arsenic biosensors. This makes it potentially a useful biosensor for use in developing countries such as Bangladesh, where arsenic in groundwater is a major public health problem, since the biosensor can be stored and distributed in the form of dried endospores, which is not possible with E. coli-based biosensors. Here are some sample data from an experiment in which the 'Bacillosensor' clone was incubated overnight in 50% v/v LB, 50% v/v water with 10 mg/l chloramphenicol and various concentrations of arsenic (as sodium arsenate, the most environmentally relevant form of this toxin). Incubation was at 37 C with shaking. The following morning, catechol was added to a final concentration of 0.5 mM and the samples were incubated at room temperature without shaking for several hours. Cells were then removed by centrifugation, and the absorbance at 377 nm (peak absorbance of the yellow product) of a 1/10 dilution was measured against a water blank.
sterile growth medium: 0.027
no arsenic: 0.048
10 ppb arsenic (WHO safety limit): 0.090
25 ppb arsenic: 0.117
50 ppb arsenic (Bangladesh safety limit): 0.184
75 ppb arsenic: 0.228
100 ppb arsenic: 0.309
The yellow colour was clearly visible by eye, making this potentially a useful addition to our range of arsenic biosensors, especially for use in developing countries, where the ability to store and distribute the organism in a dry form as spores will be especially useful.
Integration of BioBricksTM onto the B. subtilis chromosome
B. subtilis becomes naturally competent at certain stages of its life cycle, and will readily take up linear DNA from the medium and incorporate it into the chromosome by homologous recombination (Sonenshein et al, 1993). We therefore sought to develop a method to prepare linear DNA fragments bearing a BioBrickTM and chloramphenicol resistance gene flanked by upstream and downstream DNA. The first step was to choose a locus of integration. The amyE locus, encoding amylase, is commonly used for this purpose, but since we hoped to test cellulose degradation genes, we were reluctant to disrupt a polysaccharide degradation locus (even though this might have been appropriate for this particular project, since starch hydrolysis would be undesirable in a starch-producing organism). Westers et al (2003) have presented a list of 332 genes which are dispensable in B. subtilis, and after examination of this list, we decided to incorporate our genes into a locus associated with lytic genes of the prophage PBSX. Our upstream DNA site would be 1 kb of DNA preceding the xepA region, and our downstream DNA would be 1 kb of DNA in the xlyB-spoIISB region. The inserted DNA would replace the lytic genes xepA, xhlA, xhlB, and the 5' end of xlyB, but spoIISB would be left intact. The following PCR primers were designed:
upstreamF: ggactt ggatcc gccattgg (BamHI)
upstreamR: ggt gaattc tttatactggtcagc (EcoRI)
downstreamF: gaaaaaccc gagctc tatcc (SacI)
downstreamR: gaac ggatcc atgtttattatgg (BamHI)
Additionally, the following primers were designed to clone the chloramphenicol resistance gene of pTG262:
catF: atg ctgcag tctatcccggcaatag (PstI) weak no
catR: ttt gagctc tttccggcgaggc (SacI)
PCR was performed to obtain the three products: upstream DNA with BamHI-EcoRI sites; chloramphenicol resistance gene with PstI-SacI sites; downstream DNA with PstI-SacI sites. These three fragments were then digested: 'upstream' with BamHI; 'cat' with SacI; and 'downstream' with SacI and BamHI. The products were mixed together and ligated in a triple ligation, which was used as template for a PCR reaction using primers catF and upstreamR. This generated a single PCR product of about 4 kb with this structure:
PstI - CML RESISTANCE - SacI - DOWNSTREAM DNA - BamHI - UPSTREAM DNA - EcoRI
This was then stored as Bacillobrick linear vector 1. The following protocol was then used to generate linear BioBrickTM-containing DNA for transformation of B. subtilis:
Digest BioBrickTM (in pSB1A2 or similar vector with EcoRI and PstI, excise the BioBrickTM band from a gel and purify.
Digest linear vector DNA with EcoRI and PstI.
Ligate the two pieces of DNA.
Use a small amount of ligation reaction as template for PCR with primers upstreamF and downstreamR. This should generate a single product of size 4 kb plus the size of the BioBrickTM. The product should have this structure:
UPSTREAM DNA - EcoRI - BIOBRICKTM - PstI - CML RESISTANCE - SacI - DOWNSTREAM DNA
Check this product on a gel, and if it looks OK, use it to transform B. subtilis according to a standard procedure .
Initially we tested this protocol using BBa_J33204 which carries the promoterless reporter gene xylE, as described above. Chloramphenicol (10 mg/l)-resistant colonies were obtained, whereas control transformations without DNA did not yield colonies. XylE activity was not detected, but the presence of xylE was confirmed by colony PCR. We then repeated the procedure using a composite BioBrickTM BBa_J33207+BBa_J33204, which has the xylE reporter gene under control of a lac promoter. The result was the same: transformants were obtained, and the presence of xylE was confirmed by PCR, but no XylE activity was detected. Finally, we used composite BioBrickTM BBa_J33206+BBa_J33204, which, as noted above, includes the native B. subtilis ars promoter with the xylE reporter gene. The result was again the same: chloramphenicol-resistant clones carrying xylE, but no XylE activity detected, even with induction by arsenate, which, as noted above, led to detectable XylE activity in the plasmid version of this experiment. We therefore conclude that this method works well for introducing BioBricksTM into the chromosome of B. subtilis, but that we have yet to demonstrate expression of genes introduced in this way (although the chloramphenicol resistance gene is clearly expressed). We plan to test the use of stronger B. subtilis promoters and alternative loci of integration.
References
de Vos, W.M. and Simons, G.F.M. 1994. Gene cloning and expression systems in Lactococci. Chapter 2 (pages 52 to 105) in 'Genetics and Biotechnology of Lactic Acid Bacteria', edited by M.J. Gasson and W.M. de Vos, Blackie Academic and Professional, London
Sonenshein, A.L. et al. 1993. Bacillus subtilisand other Gram positive bacteria: biochemistry, physiology and molecular genetics.
Westers, H., Dorenbos, R., van Dijl, J.M. et al. 2003. Genome Engineering reveals large dispensable regions in Bacillus subtilis. Molecular Biology and Evolution 20, 2076-2090.
Retrieved from " http://2008.igem.org/Team:Edinburgh/Protocols/Bacillobricks "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
