"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Plan
From 2008.igem.org
Edinburgh iGEM 2008
Home
Plan
Protocols
Notebook
Results
Parts
Modelling
Software
ELSI
FAQ
The Team
Acknowledgements
 
Overview Step1 Step2
Contents
1 An ambitious plan
1.1 The chassis: E. coli vs. B. subtilis
1.2 Primary Objective: The production of starch from cellulose
1.3 Secondary Objective: The synthesis of β-carotene
1.4 Further considerations
1.4.1 An inducible RNAse
1.4.2 Lemon flavour
1.5 System overview
1.6 References
1.6.1 Cellulose degradation
1.6.2 Bacterial cell lysis
1.6.3 Glycogen overproduction
1.6.4 Starch biosynthesis
1.6.5 Synthesis of carotenoids
1.6.6 Synthesis of limonene and other terpenes
An ambitious plan
The chassis: E. coli vs. B. subtilis
We were initially torn between using Escherichia coli or Bacillus subtilis as a chassis for development of MicroMaize:
For the labwork we decided to use E. coli JM109 as the chassis. This was due to the greater availability of tested registry parts for E. coli compared to B. subtilis, a higher level of understanding of the biochemistry of E. coli, and more in-house experience of engineering E. coli.
We decided that B. subtilis would have advantages in the case of our system working and industrialisation being an option. This was because of increased ease of transport (due to its stable spore-forming nature), its having a better characterised secretory system and being regarded in higher favour than E. coli by the general public (see section of ethical, legal and social implications). Another possibility for the future would be to port the system to yeast, Saccharomyces cerevisiae; this would be more difficult to achieve but might ultimately be more desirable as yeast is Generally Regarded As Safe for food use, and as a eukaryote, might be able to accumulate larger amounts of starch than bacteria.
Primary Objective: The production of starch from cellulose
Firstly, why starch?
Although one of our aims is to provide a substrate for the biofuels industry, which can easily ferment glucose to ethanol, we decided to try to synthesise starch instead, even though this requires extra steps. The reason for this was a question of purification: glucose is soluble, and therefore difficult to purify from solution, as is glycogen. Starch on the other hand is highly insoluble making purification of this end product relatively straight forward. As well as this, glucose becomes inhibitory at high concentrations due to osmotic pressure, and may also cause product inhibition of glucose-producing reactions. Starch, being insoluble, can potentially accumulate at much higher quantities without disrupting the biochemistry of the cell.
The production of starch from cellulose requires two main steps:
Cellulolysis
Starch synthesis
Secondary Objective: The synthesis of β-carotene
β-carotene is the pigment that gives carrots their orange colour. It is produced as an intermediate of the lycopene synthesis pathway and is converted into vitamin A by the human body. We added a number of genes from Pantoea ananatis (formerly Erwinia uredovora) to E. coli, in essence transferring the beta-carotene synthesis pathway of P. ananatis to our chasis organism. We also cloned two E. coli genes in order to upregulate the transgenic β-carotene synthesis pathway.
Overview of the genes and pathway
Further considerations
As well as designing the major components of our system, we had to consider how we would make MicroMaize a safe and appealling product (see ELSI page for more details).
An inducible RNAse
Although we can easilly kill E. coli through heat treatment, the cells contain a lot of RNA, which can cause gout. To overcome this we decided to use the "genetic self-destruct" system ( BBa_I716463 ) developed by UC Berkley's '07 team .
Briefly, this is an arabinose-inducible RNAse. Its use for us is that we can grow our cells to the required starch or β-carotene content, then induce transcription of the RNAse Barnase. Barnase will then hydrolyse the RNA into single nucleotides. We envisage that these will then diffuse through the bacterial cell wall after disruption through heat treatment.
Lemon flavour
We decided that to make MicroMaize more appealling we would make it smell and taste of lemons. To do this we used BioBrickTM I742111 developed by Edinburgh '07 . This is the BioBrickTM for LIMS1, which encodes Limonene Synthase of Citrus limon. Limonene synthase catalyses the production of (+) limonene (thought to be responsible for ~90% of the lemon taste) from geranyl diphosphate.
To complement and increase β-carotene synthesis we decided to incorporate LIMS1 into a short operon, preceeded by dxs and followed by appY.
System overview
References
Cellulose degradation
Chaudhary, P., Kumar, N.N., Deobagkar, D.N. 1997. The glucanases of Cellulomonas. Biotechnology Advances 15(2), 315-331.
Lam, T.L., Wong, R.S.C., Wong, W.K.R. 1997. Enhancement of extracellular production of a Cellulomonas fimi exoglucanase in Escherichia coli by the reduction of promoter strength. Enzyme and Microbial Technology 20, 482-488. (An approach to achieving secretion of cellulose-degrading enzymes in E. coli).
Lynd, L.R., Weimer, P.J., van Zyl, W.H., and Pretorius, I.S. 2002. Microbial Cellulose Utilization: Fundamentals and Biotechnology. Microbiology and Molecular Biology Reviews 66, 506-577. (An excellent review by the leading researchers in the field).
Stoll, D. 2001. Mapping of genes encoding glycoside hydrolases on the chromosome of Cellulomonas fimi. Canadian Journal of Microbiology 47, 1063-1067. (Some information on the genome size and known cellulose-degrading enzymes of the cellulolytic bacterium C. fimi).
Tomme, P., Kwan, E., Gilkes, N.R., Kilburn, D.G., and Warren, R.A.J. 1996. Characterization of CenC, an enzyme from Cellulomonas fimi with both endo- and exoglucanase activities. Journal of Bacteriology 178, 4216-4223. (A potentially useful enzyme from C. fimi, which we unfortunately did not have time to pursue in this project, as it contains multiple forbidden restriction sites; maybe something to look at for the future).
Wilson, D.B. 2008. Three Microbial Strategies for Plant Cell Wall Degradation. Annals of the New York Academy of Sciences 1125, 289-297.
Xie, G., Bruce, D.C., Challacombe, J.F., Chertkov, O., Detter, J.C., Gilna, P., Han, C.S., Lucas, S., Misra, M., Myers, G.L., Richardson, P., Tapia, R., Thayer, N., Thompson, L.S., Brettin, T.S., Henrissat, B., Wilson, D.B., McBride, M.J. 2007. Genome sequence of the cellulolytic gliding bacterium Cytophaga hutchinsonii. Applied and Environmental Microbiology 73(11), 3536-3546.
Bacterial cell lysis
Barnard. A., Wolfe, A. and Busby, S. 2004. Regulation at complex bacterial promoters: how bacteria use different promoter organizations to produce different regulatory outcomes. Current Opinion in Microbiology 7(2), 102-108.
Berka, R.M., Hahn, J., Albano, M., Draskovic, I., Persuh, M., Cui, X., Sloma, A., Widner, W. and Dubnau, D. 2002. Microarray analysis of the Bacillus subtilis K-state: genome-wide expression changes dependent on ComK. Molecular Microbiology 43(5), 1331-1345. (Discription of a transcription factor that activates its own expression in B. subtilis.)
Rhodius, V.A. and Busby, S.J.W. 1998. Positive activation of gene expression. Current Opinion in Microbiology 1(2), 152-159.
Snyder, L. and Champness, W. 2007. Molecular Genetics of Bacteria (3rd Edition). ASM Press
Young, R., Wang, I.-N. and Roof, W.D. 2000. Phages will out: strategies of host cell lysis. Trends in Microbiology 8(3), 120-128.
Glycogen overproduction
Damotte M., Cattanéo, J., Sigal, N. and Puig, J. 1968. Mutants of Escherichia coli K12 altered in theiry ability to store glycogen. Biochemical and Biophysical Research Communications 32(6), 916-920. (Description of a quantitative glycogen assay.)
Dedhia, N., Chen, W., and Bailey, J.E. 1996. Design of expression systems for metabolic engineering: coordinated synthesis and degradation of glycogen. Biotechnology and Bioengineering 55, 419-426.
Eydallin, G., Viale, A.M., Moran-Zorzano, M.T., Munoz, F.J., Montero, M., Baroja-Fernandez, E., and Pozueta-Romero, J. 2007. Genome-wide screening of genes affecting glycogen metabolism in Escherichia coli K-12. FEBS Letters 581, 2947-2953.
Fernandez-Banares, I., Clotet, J., Arino, J., and Guinovart, J.J. 1991. Glycogen hyperaccumulation in Saccharomyces cerevisiae RAS2 mutant - a biochemical study. FEBS Letters 290, 38-42. (A potential method for overproducing glycogen in yeast, if we ultimately decided to go down that route rather than using a bacterial host).
Govons, S., Vinopal, R., Ingraham, J. and Preiss, J. 1969. Isolation of mutants of Escherichia coli B altered in their ability to synthesize glycogen. Journal of Bacteriology 97(2), 970-972. (Description of an assay to quantify glycogen production by E. coli.)
Leung, P., Lee, Y.M., Greenberg, E., Esch, K., Boylan, S., and Preiss, J. 1986. Cloning and expression of the Escherichia coli glgC gene from a mutant containing an ADP-glucose pyrophsophorylase with altered allosteric properties. Journal of Bacteriology 167, 82-88. (Initial description of the glgC16 mutant.)
Meyer, C.R., Bork, J.A., Nadler, S.N., Yirsa, J. and Preiss, J. 1998. Site-Directed Mutagenesis of a Regulatory Site of Escherichia coli ADP-Glucose Pyrophosphorylase: The Role of Residue 336 in Allosteric Behaviour. Archives of Biochemistry and Biophysics 351(1), 152-159. (Further description of the glgC16 mutant.)
Starch biosynthesis
Ball, S.G. and Morell, M.K. 2003. From bacterial glycogen to starch: understanding the biogenesis of the plant starch granule. Annual Reviews in Plant Biology 54, 207-233. (A comprehensive review of the state of knowledge regarding glycogen and starch biosynthesis in bacteria and plants).
Delatte, T., Trevisan, M., Parker, M.L., and Zeeman, S.C. 2005. Arabidopsis mutants Atisa1 and Atisa2 have identical phenotypes and lack the same multimeric isoamylase, which influences the branch point distribution of amylopectin during starch synthesis. Plant Journal 41, 815-830. (Probable role of the heteromeric isoamylase in converting a glycogen-like precursor to starch).
Wattebled, F., Dong, Y., Dumez, S., Delvalle, D., Planchot, R., Berbezy, P., Vyas, D., Colonna, P., Chatterjee, M., Ball, S., and D'Hulst, C. 2005. Mutants of Arabidopsis lacking a chloroplastic isoamylase accumulate phytoglycogen and an abnormal form of amylopectin. Plant Physiology 138, 184-195. (Probable role of the heteromeric isoamylase in converting a glycogen-like precursor to starch).
Kubo, A., Rahman, S., Utsumi, Y., Li, Z.Y., Mukai, Y., Yamamoto, M., Ugaki, M., Harada, K., Satoh, H., Konik-Rose, C., Morell, M., and Nakamura, Y. 2005. Complementation of sugary-1 phenotype in rice endosperm with the wheat isoamylase1 in gene in supports a direct role for isoamylase1 amylopectin biosynthesis. Plant Physiology 137, 45-36. (Probable role of the heteromeric isoamylase in converting a glycogen-like precursor to starch).
Utsumi, Y., and Nakamura, Y. 2006. Structural and enzymatic characterization of the isoamylase1 homo-oligomer and the isoamylase1-isoamylase2 hetero-oligomer from rice endosperm. Planta 225, 75-87. (Expression of functional isoamylase subunits in E. coli shows that this enzyme can be expressed in a functional form in bacteria).
Synthesis of carotenoids
Sandmann, G. 2006. Production of carotenoids by gene combination in Escherichia coli. pages 143-153 in 'Food Biotechnology', 2nd edition, eds. K. Shetty, G. Paliyath, A. Pometto and R.E. Levin, CRC Press, Taylor & Francis group, Boca Raton, FL. (A useful review).
Misawa, N., Nakagawa, N., Kobayashi, K., Yamano, S., Nakamura, K., and Harashima, K. 1990. Elucidation of the Erwinia uredovora carotenoid biosynthetic pathway by functional analysis of gene products expressed in Escherichia coli. Journal of Bacteriology 172, 6704-612. (Demonstration of the functional expression of the Pantoea ananatis (= Erwinia uredovora) genes in E. coli)
Kang, M.J., Lee, Y.M., Yoon, S.H., Kim, J.H., Ock, S.W., Jung, K.H., Shin, Y.C., Keasling, J.D., and Kim, S.W. 2005. Identification of genes affecting lycopene accumulation in Escherichia coli using a shot-gun method. Biotechnology and Bioengineering 91, 636-642 (role of dxs and appY in increasing carotenoid biosynthesis).
Synthesis of limonene and other terpenes
Lucker, J., El Tamer, M.K., Schwab, W., Verstappen, F.W.A., van der Plas, L.H.W., Bouwmeester, H.J., and Verhoeven, H.H. 2002. Monoterpene biosynthesis in lemon (Citrus limon). European Journal of Biochemistry 269, 3160-3171. (Functional expression of the LIMS1 gene in E. coli).
Reiling, K.K., Yoshikuni, Y., Martin, V.J.J., Newman, J., Bohlmann, J., and Keasling, J.D. 2004. Mono and diterpene production in Escherichia coli. Biotechnology and Bioengineering 87, 200-212. (Generation of various monoterpenes, including limonene, in E. coli and methods for detection).
Retrieved from " http://2008.igem.org/Team:Edinburgh/Plan "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
