"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Results/MABEL
From 2008.igem.org
Return to Protocols page
Site-drected mutagenesis using the MABEL protocol
MABEL (Mutagenesis with Blunt-Ended Ligation) is a fast, cheap, simple and reliable alternative to QuickChange for site-directed mutagenesis to remove undesired restriction sites or to add specific mutations. In our project we used MABEL to remove two PstI sites from crtI (this was performed by Mr. Douglas Armstrong prior to the official start of the iGEM project), two EcoRI sites from glgC, and a PstI site from cex, and also to introduce the G336D mutation into glgC to abolish feedback inhibition. The MABEL procedure is as follows:
Design two non-overlapping divergent primers around the target site for the mutagenesis. The base to be altered should be at the 5' end of one of the two primers, and the other primer should start at the next base along in the sequence (see example below). Since the primers do not contain non-complementary tails and do not overlap, they can be quite short: 18 base pairs or so should usually be sufficient.
Perform PCR using a very small amount of plasmid DNA as the template, using a fast and highly accurate proof-reading polymerase. We recommend Kod polymerase (Invitrogen) for this, as it is at least as accurate as Pfu and 4 to 5 times faster. Note that proof-reading polymerases such as Kod will generate a blunt-ended PCR product.
Check the PCR product on a gel and purify it.
Self-ligate the blunt-ended PCR product in a reaction which includes both T4 DNA Ligase and T4 Polynucleotide Kinase (PNK). The PNK is required to add 5'-phosphate groups to the PCR product to allow ligation, since oligonucleotide primers generally have a free 5'-hydroxyl group rather than a phosphate. Fortunately PNK works well in ligase buffer, which also provides the necessary ATP. A typical ligation mixture using Promega reagents might contain 3.5 microlitres PCR product 3.5 microlitres water, 1 microlitre 10 x T4 DNA ligase buffer, 1 microlitre T4 DNA ligase, and 1 microlitre T4 Polynucleotide Kinase, incubated overnight at 16 C.
Transform E. coli with the ligation. Since self-ligation is more efficient than intermolecular ligation, the colonies should predominantly contain the desired mutation.
Check a few minipreps for loss of the restriction site. Sequence to confirm that the mutation has been made cleanly with no bases added or deleted. This is a recommended safety precaution, but in all our mutagenesis experiments, we never detected any such undesired changes.
Example
MABEL was used to remove this PstI site at codons 261 and 262 of the cex gene encoding exoglucanase of Cellulomonas fimi:
gac ttc cgg cag aac ctg cag cgg ttc gcg gac ctg ggc gtg gac
Since a silent change was wanted, the desired new sequence was:
gac ttc cgg cag aac ctg caa cgg ttc gcg gac ctg ggc gtg gac
Therefore, mutagenic primers were designed with the new A base being at the 5' end of the forward primer, and with the reverse primer starting with the complement of the rest of this codon, so that the two primer ends abut but do not overlap:
Forward A CGG TTC GCG GAC CTG G (anneal 58°C)
Reverse TG CAG GTT CTG CCG GAA G (anneal 58°C)
Retrieved from " http://2008.igem.org/Team:Edinburgh/Results/MABEL "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
