<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Utah_State/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overall_project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overall project</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Details"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Dual_Luciferase"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Dual Luciferase</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Promoters_Tested"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Promoters Tested</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Dual_luciferase_construct"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Dual luciferase construct</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Bioproduct_Pathways"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Bioproduct Pathways</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Pathway_1_-_Fatty_Alcohol"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Pathway 1 - Fatty Alcohol</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Pathway_2_-_Wax_Ester"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Pathway 2 - Wax Ester</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Pathway_3_-_Alkane_Alkene"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">Pathway 3 - Alkane Alkene</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#Experiments"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overall_project"><B>Overall project</B></SPAN></H2><P><B>Abstract</B>:
</P><P>The field of Synthetic Biology is continuously increasing our understanding of the complexity of all living systems, one gene at a time. Our project focuses on producing high value bioproducts in the form of fatty alcohols, wax esters, and alkanes/alkenes using Synechocystis sp PCC 6803. In addition, our team is investigating the strength of 23 promoters using a newly designed dual luciferase expression device. 
</P><P><B>Project Description</B>:
</P><P>Building upon the CyanoBrick toolkit developed by the 2010 Utah State iGEM team, our project will focus on producing valuable bioproducts using <I>Synechocystis sp.</I> PCC 6803. Our project will attempt to produce 3 different bioproducts: fatty alcohols, wax esters, and alkanes/alkenes. Fatty alcohols and wax esters are used to produce cosmetics, lubricants, and various pharmaceuticals. Fatty alcohols with a 20-carbon chain are used in the cold sore medication Abreva[reference needed], and long chain fatty alcohols have been linked to improved heart health [reference needed]. Alkanes and alkenes are utilized as hydrocarbon fuels. The production of these bioproducts in cyanobacteria will greatly reduce their cost and increase their availability.
</P><P>In addition to producing bioproducts, our project will also provide a more detailed characterization of the promoters and ribosome binding sites (RBS) produced by the 2010 Utah State iGEM Team. Utilizing a dual luciferase expression measurement construct, and reference promoters from <I>E. coli</I> and <I>Synechocystis</I>, our team will more precisely characterize the expression levels of 23 promoters under standard growth conditions, compared with the GFP-based measurements made in 2010. We will also produce useful intermediate parts, which are currently not available through the registry, allowing the dual luciferase expression measurement system to be easily adapted to new organisms and new reference promoters.
</P><P><B>Justification</B>:
</P><P>Global energy requirements are heavily dependent on fossil fuels such as oil, coal, and natural gas. Efforts to lower our dependence on foreign oil, along with the anticipation of fossil fuels being exhausted in the future, novel strategies need to be discovered for alternative energy generation. Renewable hydrocarbons (Schirmer et al., 2010), fatty alcohols, and wax esters (Keasling et al., 2010) may provide “drop in” compatible fuels and chemicals. The production of alkanes has been reported in a wide variety of microorganisms, particularly photoautotrophic cyanobacteria (Schirmer et al., 2010). Our group is focused on genetically engineering the necessary pathways needed to produce fatty alcohols (Figure 1), wax esters (Figure 2), and alkanes/alkenes (Figure 3) using Synechocystis sp PCC 6803 as the model organism. In addition, we are investigating the strength of 23 promoters by measuring a dual luciferase expression construct.
</P><P><B><I>Synechocystis</I> sp PCC 6803</B>:
</P><P>Synechocystis sp. PCC 6803 (hereafter referred to as PCC 6803) is a cyanobacterium, a photosynthetic bacterium, and one of the major model organisms in microbiology. The photosynthetic pathway of PCC 6803 has been extensively studied and its genome has been completely sequenced. It is capable of acting as both an autotroph, obtaining its energy from light and its carbon from the atmosphere, and as a heterotroph, using glycolysis of sugars as both an energy source and a carbon source. It also possesses the ability to uptake DNA present in its environment through natural transformation, making it a useful species for genetic manipulation (Zhang et al., 2007). PCC 6803 possesses many industrial applications including lipid recovery in biofuel production, PHB collection, and as a toxin biosensor (Rouillon et al., 1999; Wu et al., 2001). Heptadecane, an alkane hydrocarbon, production has been observed within PCC 6803 (Schirmer et al., 2010).
</P><H2><SPAN class="mw-headline" id="Project_Details"> Project Details</SPAN></H2><H3><SPAN class="mw-headline" id="Dual_Luciferase"> Dual Luciferase </SPAN></H3><P>The dual luciferase system integrates two efficient reporter genes to evaluate regulatory gene expression. This system incorporates the firefly (Photinus pyralis) and the Renilla (Renilla reniformis) luciferases, which have different biochemical requirements for luminescence (Sherf et al., 1996). Dual reporter systems are commonly used to improve the accuracy of reporter assays when analyzing regulatory gene expression, particularly in mammalian cells. The luciferase assay is very sensitive assay that is performed by sequentially measuring both luciferase activities of the same sample (McNabb et al., 2005).
</P><P>The expression level of your promoter of interest (cloned in front of Firefly Luciferase) is then compared to the expression level of the reference promoter in front of Renilla Luciferase (this promoter is generally the Anderson 1.0 reference promoter BBa_J23100, but can be changed to other promoters for use in other species). The ratio of Firefly to Renilla expression indicates the relative strength of the promoter of interest. To control for differences in the expression of the two genes due to codon bias and other differences, the Firefly:Renilla ratio of the promoter of interest to the reference promoter is compared to a Dual Luciferase construct with the reference promoter controlling expression of both luciferase genes. This adjusted ratio gives the true expression level of the promoter.
</P><P>
This system has the following advantages over the commonly used RFP expression measurement system:
</P><OL><LI><B>Copy number control</B> – Plasmids, and particularly high copy number plasmids, generally do not have a single set number of copies in a cell, but can vary in a wide range. The iGEM standard plasmids are high copy number plasmids that can have between 100-300 copies in a cell (Example: pSB1C3). By having both the promoter of interest and the reference promoter controlled genes on the same plasmid (rather than in two different cell cultures like the RFP system) the genes are present in equal numbers, controlling this variable factor.</LI><LI><B>Cell cycle phase control</B> – Since both genes are present in each cell, the reference will still be a viable control regardless of cell cycle phase. In the RFP system, if the two cultures containing the testing promoter and the reference promoter were in different phases, the levels of expression would not be comparable and the promoter strengths would be incorrectly calculated.</LI><LI><B>Cell density control</B> – Since both genes are present in each cell in the culture, the density of the culture does not affect the ratio of the promoter’s expression. In the RFP system, if cell densities were different between the control and reference cultures, the expression levels of the promoters would be incorrectly measured (the more dense culture having a higher apparent expression level), and the promoter strength calculation would be inaccurate. 
</LI></OL><P>
This system does require some additional supplies that the RFP system does not, but these additional supplies are not prohibitively difficult to obtain:
</P><OL><LI><B>Luminometer</B> – Since luciferase proteins convert chemical energy to light, standard spectrophotometers, which rely on the amount of light that a sample absorbs, will not correctly measure the light output. Luminometers are designed to measure emitted light levels at specific wavelengths, and are likely to be found on most college campuses. </LI><LI><B>Chemical Energy (Luciferin)</B> – Luciferase proteins produce light by causing chemical conversions of chemicals called luciferins. These luciferins need to be added to lysed cells solutions in order to measure the expression of the proteins, unlike RFP which can be measured within living cells and with only the addition of light of the proper wavelength. There are several commercial kits available specifically for dual luciferase assays including the Promega Dual Luciferase Reporter Assay System.</LI></OL><H3><SPAN class="mw-headline" id="Promoters_Tested"> Promoters Tested </SPAN></H3><P><B>Table 1</B>. Comprehensive list of all promoters tested.

</P><H3><SPAN class="mw-headline" id="Dual_luciferase_construct">Dual luciferase construct</SPAN></H3><P><B>Construct 1</B>. 
</P><H3><SPAN class="mw-headline" id="Bioproduct_Pathways"> Bioproduct Pathways </SPAN></H3><H5><SPAN class="mw-headline" id="Pathway_1_-_Fatty_Alcohol"> Pathway 1 - Fatty Alcohol </SPAN></H5><P>FadD Fatty Acyl-CoA Synthetase (Ligase) from E. coli. Contains a 8-Histidine tag immediately before the stop codon to allow gel identification and extraction. It converts fatty acids into fatty acyl-CoA using ATP. Fatty Acyl-CoA Synthetase is used in conjunction with Thioesterase to greatly increase the concentration of Fatty Acyl-CoA present in the cell, to increase the amount of bioproducts produced. It is used to produce Fatty Alcohols and Wax Esters in the 2011 Utah_State iGEM project.
</P><H5><SPAN class="mw-headline" id="Pathway_2_-_Wax_Ester"> Pathway 2 - Wax Ester </SPAN></H5><P>Wax Ester Synthase from Marinobacter aquaeolei. Contains a 8-Histidine tag immediately before the stop codon to allow gel identification and extraction. The Wax Ester Synthase converts fatty acyl-CoA and fatty alcohols into wax esters. Wax Ester Synthase is used to produce Wax Esters, a valuable bioproduct, in the 2011 Utah_State iGEM project.
</P><H5><SPAN class="mw-headline" id="Pathway_3_-_Alkane_Alkene"> Pathway 3 - Alkane Alkene </SPAN></H5><P>Fatty Aldehyde Decarbonylase from Synechocystis sp. PCC 6803. Contains a 8-Histidine tag immediately before the stop codon to allow gel identification and extraction. The Fatty Aldehyde Decarbonylase gene is annotated as a hypothetical gene, but is believed to converts fatty aldehydes into alkanes and alkenes. Fatty Aldehyde Decarbonylase is used in conjunction with Fatty Acyl-ACP Reductase to produce alkenes and alkanes, hydrocarbon fuels, in the 2011 Utah_State iGEM project.
</P><H3><SPAN class="mw-headline" id="Experiments"> Experiments </SPAN></H3><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P><B>Table 2. Promoter Strengths calculated from Firefly expression and from Firefly/Renilla expressions</B></P><P>Using only the Firefly strengths we get approximately the same results as the Anderson promoter set that uses RFP. We get 0.080+/-0.017 for the Anderson 0.10 and 0.32+/-0.001 for the Anderson 0.01. When we standardized our results to the ratio of the E.coli control (which had the Anderson promoter in front of both Luciferases) in order to compensate for difference in expression for the two Luciferases are estimates for promoter strength changed dramatically. The Anderson 0.1 gave a relative strength of 6.8+/-0.9 and the 0.01 gave a relative strength of 24+/-1.6.
</P><P><B>Graph 1. Firefly/Renila ratio (log10) vs. Time (hr), there was a total of 14 data points collected at each time point, each from different promoter testing samples.</B></P><P>Each of the constructs were grown in 50ml LB shaker flasks and measurements (Optical density and Luminescence) were taken at 0,3,6,9, and 24 hours. Also shown in this graph is the Firefly/Renilla ratio vs. time (hrs). The data suggests that for all tested constructs, the Firefly/Renilla ratio decreases as the culture ages.
</P><P><B>Table 3. E.coli control, Anderson 0.10, and Anderson 0.01 compared to DH5α (negative control)</B></P><P>The experimental results shown in Table 3 were averages based on triplicate data collection. Table 3 shows that the E.coli control, Anderson 0.10, and Anderson 0.01 are higher than the DH5α (negative control). The E.coli control, Anderson 0.10 and Anderson 0.01 had Firefly averages of 45715166, 3695748, and 1461220 respectively compared to the DH5α (negative control) having a background Firefly average of 74.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>1.	Keasling, J.D., Steen, E.J., Kang, Y.S., Bokinsky, G., Hu, Z.H., Schirmer, A., McClure, A., del Cardayre, S.B., (2010) Microbial production of fatty-acid-derived fuels and chemicals from plant biomass. Nature 463, 559-U182.
</P><P>2.	McNabb, D.S., Reed, R., Marciniak, R.A., (2005) Dual luciferase assay system for rapid assessment of gene expression in Saccharomyces cerevisiae. Eukaryot Cell 4, 1539-1549.
</P><P>3.	Rouillon, R., Avramescu, A., Carpentier, R., (1999) Potential for use of a cyanobacterium Synechocystis sp immobilized in poly(vinylalcohol): Application to the detection of pollutants. Biotechnol Tech 13, 559-562.
</P><P>4.	Schirmer, A., Rude, M.A., Li, X.Z., Popova, E., del Cardayre, S.B., (2010) Microbial Biosynthesis of Alkanes. Science 329, 559-562.
</P><P>5.	Sherf, B. A., S. L. Navarro, R.R. Hannah, and K. V. Wood., (1996) Dualluciferase reporter assay: an advanced co-reporter technology integrating ﬁreﬂy and Renilla luciferase assays. Promega Notes 57:2–8. [Online.] <A href="http://www.promega.com/pnotes/57/5573a/5573a.html" class="external free" rel="nofollow">http://www.promega.com/pnotes/57/5573a/5573a.html</A>.
</P><P>6.	Wu, G.F., Wu, Q.Y., Shen, Z.Y., (2001) Accumulation of poly-beta-hydroxybutyrate in cyanobacterium Synechocystis sp PCC6803. Bioresour Technol 76, 85-90.
</P><P>7.	Zhang, X.C., Zang, X.N., Liu, B., Liu, S.M., Arunakumara, K.K.I.U., (2007) Optimum conditions for transformation of Synechocystis sp PCC 6803. J Microbiol 45, 241-245.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Utah_State/Project">http://2011.igem.org/Team:Utah_State/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;oldid=201686" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>