<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BNU-China</TITLE></P><DIV id="w_nav" style="top:15px;"><UL><LI id="w_Project"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Description">Project</A></LI><LI id="w_Modeling"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Model">Modeling</A></LI><LI id="w_Practice"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/HP">Practice</A></LI><LI id="w_home"><A class="w_a" style="opacity:0;" href="https://2017.igem.org/Team:BNU-China">Home</A></LI><LI id="w_Achievement"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Judging">Achievement</A></LI><LI id="w_Lab"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Team">Lab</A></LI><LI id="w_Safety"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Safety">Safety</A></LI></UL><DIV id="w_navMenu" style="text-align:right;"><P>Menu▼</P><UL id="w_navMenuContent"><LI id="w_Project"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Description">Project</A></LI><LI id="w_Modeling"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Model">Modeling</A></LI><LI id="w_Practice"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/HP">Practice</A></LI><LI id="w_Achievement"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Judging">Achievement</A></LI><LI id="w_Lab"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Team">Lab</A></LI><LI id="w_Safety"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Safety">Safety</A></LI></UL></DIV></DIV><DIV id="w_content" style="width:100%;"><DIV id="w_Panel" style="text-align:left;"><UL id="w_menu"><LI id="w_menu1"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Team">Team</A></LI><LI id="w_menu2"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Notebook">Notebook</A></LI><LI id="w_menu3"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Experiments">Protocol</A></LI><LI id="w_menu4"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Attributions">Attributions</A></LI><LI id="w_menu4"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Acknowlegement">Acknowlegement</A></LI></UL></DIV></DIV><DIV id="w_text"><H1>Protocol</H1><H2>1. PCR system</H2><H3>1) 20μL system is used for verification</H3><P>Taq: </P><TABLE><TBODY><TR><TD>Enzyme</TD><TD>Plasmid(genome)</TD><TD>Primer1</TD><TD>Primer2</TD><TD>DNTP</TD><TD>Buffer</TD><TD>ddH2O</TD></TR><TR><TD>0.2μl</TD><TD>20ng(40-80ng)</TD><TD>1μl</TD><TD>1μl</TD><TD>2μl</TD><TD>2μl</TD><TD>fill the system to 20μl</TD></TR></TBODY></TABLE><P>Super mix: </P><TABLE><TBODY><TR><TD>Plasmid(genome)</TD><TD>Primer1</TD><TD>Primer2</TD><TD>Super mix</TD><TD>ddH2O</TD></TR><TR><TD>20ng(40-80ng)</TD><TD>1μl</TD><TD>1μl</TD><TD>10μl</TD><TD>fill the system to 20μl</TD></TR></TBODY></TABLE><H3>2) 50μL system for PCR target genes</H3><P>
        Pfu mix: 
        DNA, plasmid, 50ng, genome, 100-200ng</P><TABLE><TBODY><TR><TD>Plasmid(genome)</TD><TD>Primer1</TD><TD>Primer2</TD><TD>Pfumix</TD><TD>ddH2O</TD></TR><TR><TD>50ng(100-200ng)</TD><TD>2μl</TD><TD>2μl</TD><TD>25μl</TD><TD>fill the system to 50μl</TD></TR></TBODY></TABLE><H3>3) 20μL PCR system for microbes</H3><P>
        Super mix: </P><P>Bacteria (after picking on the board, back up on the new board and rinse it in the system that has been prepared) </P><TABLE><TBODY><TR><TD>Primer1</TD><TD>Primer2</TD><TD>Super mix</TD><TD>ddH2O</TD></TR><TR><TD>1μl</TD><TD>1μl</TD><TD>10μl</TD><TD>fill the system to 20μl</TD></TR></TBODY></TABLE><H2>2. Plasmid concentration</H2><OL class="l_finalol"><LI>Dilute collected DNA to 400μl. (If the plasmid is eluted by ddH2O, then it is diluted by ddH2O as well; If the plasmid is eluted by elution buffer, then it is diluted by elution buffer as well)</LI><LI>Add 900μl absolute ethanol to precipitate DNA, and 40μl NaAc (3M PH=5.2) to help precipitation. Be sure to mix it up.</LI><LI>Put the reactant in ice-bath for 1 hour, or put it under -20 degrees Celsius overnight. </LI><LI>Centrifuge the reactant under 12000rpm for 10 minutes, then throw away the supernatant. </LI><LI>Add 1ml 75% ethanol (the 75% ethanol is prepared by using absolute ethanol rather than ethanol for disinfection) and mix up to wash away salt ions. </LI><LI>Centrifuge the reactant under 12000rpm for 5 minutes, then throw away the supernatant. </LI><LI>Centrifuge the reactant for another 3-4 seconds and use transfer liquid gun to throw away residual supernatant. </LI><LI>Centrifuge the reactant for another 3-4 seconds and use transfer liquid gun to throw away residual supernatant. </LI><LI>Open the centrifuge tube and expose the reactant to air. Put the centrifuge tube into incubator at 37 degrees Celsius for 15 minutes until the plasmid turn into transparent. </LI><LI>Add appropriate volume of ddH2O to dilute the plasmid according to precipitation capacity. </LI><LI>Use 1μl solution and add 9μl ddH2O to dilute it, then use electrophoresis to measure its quantity. </LI></OL><H2>3. Plasmid purification</H2><OL class="l_finalol"><LI>Culture of E-coli: Select single colonies from plate culture medium then inoculate them into 1-3ml liquid nutrient medium containing antibiotics, then culture them at 37 degrees Celsius overnight. (Usually the culture time is 12-16 hours, because if the cells have been cultured over 16 hours, it is hard to lysis the cells so that plasmid production will decrease. Besides, the medium should not be excessive, because too many bacteria are difficult to be fully lysised so that plasmid purity is affected.) </LI><LI>Collect 1-4ml overnight culture of bacterial fluid, then centrifuge it at 12000rpm for 2 minutes and throw away the filtrate. </LI><LI>Use 250μl Solution | (be sure that RNase A is added to Solution |) to fully suspend the bacterial precipitation. (Pay attention not to leave over small pieces of bacteria. Vortex can be used to make the bacteria fully suspended) </LI><LI>Add 250μl Solution || and gently blend the solution by turning it up and down for 5-6 times to make the bacteria fully lysis until the solution becomes transparent. (Be sure to blend the solution gently instead of oscillating fiercely, and this step should be no more than 5 minutes.) </LI><LI>Add 350μl Solution ||| which was pre-cooled at 4 degrees Celsius, gently blend the solution by turning it up and down for 5-6 times until tight agglutination block is formed, then stand it under room temperature for 2 minutes. </LI><LI>Centrifuge the solution at 12000rpm for 10 minutes and keep the filtrate. (Centrifuge at 4 degrees Celsius is not benefit to sedimentation.) </LI><LI>Place the Spin Column on the Collection Tube. </LI><LI>Transfer the supernatant from Step (6) to the Spin Column, then centrifuge it at 12000rpm for 1 minutes and throw away the filtrate. </LI><LI>Add 500μl Buffer WA to the Spin Column then centrifuge it at 12000rpm for 30 seconds and throw away the filtrate. </LI><LI>Add 700μl Buffer WB to the Spin Column then centrifuge it at 12000rpm for 30 seconds and throw away the filtrate. (Make sure that specified volume of 100% ethanol has been added to the Buffer WB.) </LI><LI>Repeat Step (10) once. </LI><LI>Place the Spin Column on the Collection Tube, then centrifuge it at 12000rpm, under room temperature for 1 minutes and throw away the filtrate. </LI><LI>Place the Spin Column on a new 1.5ml centrifuge tube. Add 50μl sterile water or Elution Buffer to the center of the membrane of the Spin Column and stand the solution under room temperature for 1 minutes. (If the sterile water or Elution Buffer is heated to 60 degrees Celsius, then the elution efficiency can be improved.) </LI><LI>Centrifuge the solution at 12000rpm, under room temperature for 1 minutes to elute DNA. </LI></OL><H2>4. Enzyme digestion</H2><H3> (1) Enzyme digestion test (10μl system) </H3><TABLE><TBODY><TR><TD>Buffer X</TD><TD>Enzyme</TD><TD>Plasmid DNA</TD><TD>ddH2O</TD></TR><TR><TD>1μl</TD><TD>0.2μl</TD><TD>y μl</TD><TD>fill the system to 10μl</TD></TR></TBODY></TABLE><P>X: look up the enzyme digestion tableY: If the length of the minimal fragment is a, the length of the plasmid is b, the concentration of the plasmid is c, then: 20ng/a=(m ng)/b；y=m/c</P><H3> (2) Enzyme digestion to get linear plasmid (50μl system) </H3><TABLE><TBODY><TR><TD>Buffer X</TD><TD>Enzyme</TD><TD>Plasmid DNA</TD><TD>ddH2O</TD></TR><TR><TD>5μl</TD><TD>1μl</TD><TD>y μl</TD><TD>fill the system to 50μl</TD></TR></TBODY></TABLE><P>X: look up the enzyme digestion tabley: 2.5μg</P><H2>5. Infusion</H2><H3>1) Primer design</H3><OL class="l_finalol"><LI>Each In-Fusion primer must contain two parts: the primer 5 'end must be included and will be connected to the DNA fragment (such as carrier or another fragment) 15 ends completely homologous base. Primers 3 must contain primer sequences of target genes. </LI><LI>Each primer 3 'should be: 
        - with gene specificity
        - length is 18 - 25, the content of GC is 40 - 60%
        - annealing temperature (Tm) is of 58 - 65 C. Forward and reverse primers of the Tm value difference should not exceed 4 degrees, otherwise the amplification effect is not ideal. Note that the evaluation of Tm values should be based on primer 3 (gene specificity), rather than on the whole primer. If Tm is too low, we can appropriately extend the part of primer with gene specificity until Tm reaches 58 - 65 degrees C.
        - do not contain successive identical nucleotides. The last 5 nucleotides of each primer '3' should not contain more than 2 guanine (G) or cytosine (C). </LI><LI>The complementary sequence between the primers is avoided to prevent the primer itself from producing hairpin structure. </LI><LI>You can do BLAST search to make sure that the 3’ end of each primer is highly specific. (www.ncbi.nlm.nih.gov/BLAST/)</LI></OL><H3>2) Reaction system and steps</H3><TABLE><TBODY><TR><TD>5, x, In-Fusion, HD, Enzyme, Premix</TD><TD>Linear plasmid</TD><TD>Target gene</TD><TD>ddH2O</TD></TR><TR><TD>1μl</TD><TD>25-50ng</TD><TD>25-50ng</TD><TD>fill the system to 5μl</TD></TR></TBODY></TABLE><P>* linear plasmid and target gene volume should be smaller than 2μL<OL class="l_finalol"><LI>mixing system</LI><LI>static in 50 degrees Celsius environment, reaction for 15min</LI><LI>convert on ice, or -20 preservation</LI><LI>2.5μL reaction liquid was added into 50μL competent cells during transformation</LI></OL></P><H2>6. DNA fragment purification</H2><OL class="l_finalol"><LI>Add Buffer DC as much as 3 times the volume of the PCR reaction liquid to the PCR reaction liquid (or other enzymatic reaction fluid), then mix them up. (If the needed Buffer DC is less than 100μl, then add to 100μl.) </LI><LI>Place the Spin Column on the Collection Tube. </LI><LI>Transfer the solution in Step (1) into the Spin Column, then centrifuge it at 12000rpm, under room temperature for 1 minutes and throw away the filtrate. (If the filtrate is added back to the Spin Column to centrifuge again, the recovery rate of the DNA can be improved.) </LI><LI>Add 700μl Buffer WB to the Spin Column then centrifuge it at 12000rpm, under room temperature for 30 seconds and throw away the filtrate. (Make sure that specified volume of 100% ethanol has been added to the Buffer WB.) </LI><LI>Repeat Step (4) once. </LI><LI>Place the Spin Column on the Collection Tube, then centrifuge it at 12000rpm, under room temperature for 1 minutes. </LI><LI>Place the Spin Column on a new 1.5ml centrifuge tube. Add 25-30μl sterile water or Elution Buffer to the center of the membrane of the Spin Column and stand the solution under room temperature for 1 minutes. (If the sterile water or Elution Buffer is heated to 60 degrees Celsius, then the elution efficiency can be improved.) </LI><LI>Centrifuge the solution at 12000rpm, under room temperature for 1 minutes to elute DNA. </LI></OL><H2>7. Gel electrophoresis</H2><H3>1) Prepare agarose gel</H3><P>Experimental supplies: 1×TAE, agarose<OL class="l_finalol"><LI>Weigh agarose according to its working concentration. Usually 0.8% or 1% is suitable for running quantitative gel or testing PCR product. </LI><LI>Using 1×TAE to dissolve the agarose by careful warming. (Heat it in microwave oven and boil it two times so that the agarose dissolves completely) </LI><LI>When the temperature decreases to 55 degrees Celsius, add Gelstain and shake it up. (1μl Gelstain per 10 ml gel) </LI><LI>Pour the prepared gel into the electrophoresis bath, wait for the gel to concrete and reserve it. </LI></OL></P><H3>2) Add sample</H3><OL class="l_finalol"><LI>For running quantitative gel, add 1μl sample; For testing PCR product, add 3-5μl sample. </LI><LI>Mix up the sample and 10×loading buffer, add the mixture into spotting hole. Be careful not to leave any air bubbles, which will change the volume of the sample. </LI><LI>Add corresponding volume of Marker according to the manual. (Usually 5μl Marker per 10μl spotting hole) </LI><LI>Set up constant voltage 170V, switch on for about 15 minute. When the indicator line reach 1/2-2/3 of the whole gel, switch off. </LI><LI>Use Gel imager to show the imaging of the gel. </LI></OL><H2>8. Agarose Gel DNA Extraction</H2><OL class="l_finalol"><LI>Agarose gel is prepared using TAE buffer or TBE buffer, then agarose gel electrophoresis is performed on the target DNA. </LI><LI>Cut the agarose gel containing the target DNA under the UV lamp and suck the liquid on the gel surface with a paper towel. At this point, we should pay attention to the removal of the gel which does not contain the target DNA, minimize the gel volume and increase the DNA recovery rate. When glue blocks exceed 300 mg, use more than one Column Recycling, otherwise seriously affecting the yield. Note: when cutting, please be careful not to expose the DNA to UV light for a long time to prevent DNA damage. </LI><LI>Shredding the gel block. When the adhesive block is chopped up, the dissolution time of the block in the operation step 6 can be accelerated, and the DNA recovery rate can be improved. </LI><LI>Weighing the weight of the glue block and calculating the volume of the glue block. When calculating the volume of the block, 1mg=1μL is used to calculate. </LI><LI>Buffer GM are added into the gel block, and the amount of solution Buffer GM. </LI><LI>After mixing evenly at room temperature 15-25 degrees C dissolve glue block (glue concentration is more difficult to dissolve and can be heated at 37 C). At this point should be interrupted oscillation mixing, so that the block fully dissolved (about 5~10 minutes). Note: the block must be fully dissolved, otherwise it will seriously affect the recovery rate of DNA. High concentration gel can prolong the time appropriately. </LI><LI>When the gel is completely dissolved, observe the solution color, if the solution color changed from yellow to orange or pink, add 3 M sodium acetate solution to the rubber block solution (pH5.2) 10μL, mixed solution to restore yellow. When the DNA fragment is less than 400bp, the isopropanol with a final concentration of 20% should be added to the solution. </LI><LI>Place the Spin Column in the reagent kit on the Collection Tube. </LI><LI>Transfer the solution of the operation step 7 to the Spin Column with 12000 rpm centrifugation for 1 minutes, filtrate rejection. Note: if the filtrate is added to the Spin Column to be centrifuged once, it can be improved DNA recovery rate. </LI><LI>Add 700μL Buffer WB to Spin Column at room temperature 12000 rpm centrifuge for 30 seconds, discard filtrate. Note: please confirm that Buffer WB has added 100% ethanol of the specified volume. </LI><LI>Repeat step 10. </LI><LI>Place the Spin Column on the Collection Tube at room temperature 12000rpm Centrifuge for 1 minutes.</LI><LI>Place the Spin Column on the new 1.5 ml centrifuge tube at Spin. Add 30μl of sterile water or Elution Buffer at the center of the Column film and leave it for 1 minutes at room temperature. Note: heating the sterilized water or Elution Buffer to 60 degrees can improve the elution efficiency when used. </LI><LI>12000 rpm centrifugations at room temperature, 1 minute elution DNA. </LI></OL><H2>9. Saccharomyces cerevisiae genomic extraction</H2><OL class="l_finalol"><LI>Mix the bacterial fluid into the 2ml centrifuge tube and collect the cells by centrifugation at 12000rpm 2min; </LI><LI>Add 1ml sterile water or washing buffer and wash 1 time; </LI><LI>Adding equal volume glass beads, and adding 100μL TENTS, using a cell crusher to break cells for 1min; </LI><LI>Add 600 L TENTS, mix upside down, then put in room temperature for 1min; </LI><LI>Centrifuge 8000 rpm, 1 min, pipette extract supernatant, transfer to the new 2ml EP tube; </LI><LI>Add the volume of phenol: chloroform (1:1) [pay attention to exact the lower layer], mixing evenly, place in room temperature10 min; </LI><LI>Centrifuged 10 min 120000 rpm, carefully remove the supernatant with a 200 L pipette [pay attention to not to the middle of the protein layer]; </LI><LI>Add 2-3 times the volume of anhydrous ethanol, 1/10 volume of NaAc, mixing evenly, ice bath for 40 min-1h; </LI><LI>Centrifuged 10 min 12000 rpm and discard supernatant; </LI><LI>Add 1 ml 75% ethanol, mix evenly [wash precipitation], centrifuge 5 min 12000 rpm; </LI><LI>Discard the supernatant [caution not to drop the bottom white], Short, the residue is removed with a pipette, and the 37 degrees incubator is dried for 20 min; </LI><LI>Precipitation is transparent and add 100μL ddH2O dissolve DNA [this step can be 4 degrees overnight], and then add 1.5μl RNase, 37 degrees [remove RNA] reaction 1-2 h; </LI><LI>Added 100μL ddH2O per tube [dilute the DNA to reduce the loss of dilution, extraction process], with an equal volume of phenol: chloroform (1:1), upside down mixing, 12000 rpm centrifugal for 10min, carefully pipette supernatant with 200μL[note the protein in the middle layer]; </LI><LI>Repeat step 13 once; </LI><LI>Add 2-3 times the volume of anhydrous ethanol, 1/10 volume of NaAc, upside down, mixing evenly, ice bath 40 min-1 h; </LI><LI>12000 rpm, centrifuge 10 min and discard supernatant; </LI><LI>Add 1 ml 75% ethanol, mix evenly [wash precipitation], 12000 rpm, centrifuge 5 min; </LI><LI>Discard the supernatant [caution not to drop the bottom white], Short, the residue is removed with a pipette, and the 37 degrees incubator is dried for 20 min; </LI><LI>Precipitation is transparent and add 20-30 L ddH2O dissolve DNA [this step can be 4 degrees overnight]. </LI></OL><H2>10. Preservation in Glycerol</H2><H3>1) The manufacture of glycerin tubes</H3><P>Configuration 50% glycerin (Glycerol: ddH2O = 1:1) 
        Packaged in 2ml storage tubes and 550μl per tube
        Sterilization at 121 degrees for 20min</P><H3>2) Preservation</H3><P>In advance, the target strains were cultured in the liquid medium containing antibiotics (E. coli in LB medium, yeast in YPD) for 12-14h
        Secondly, the liquid fraction of 10ml~20ml was centrifuged twice, and the bacterial precipitate was suspended by 1ml without the corresponding liquid medium of μ antibiotics. 
        Add the suspension to the sterilized glycerin tube, blow it several times and screw down the lid. </P><H2>11. Inversion</H2><OL class="l_finalol"><LI>Add appropriate amount of plasmid (no more than 4μL) into 50μL competence. </LI><LI>Place the reactant on ice for 30 minutes. </LI><LI>Put the reactant under 42 degrees Celsius for 45s-90s. </LI><LI>Place the reactant on ice for 2 minutes. </LI><LI>Add 950μL Luria-Bertani into the reactant inside the super-clean bench. </LI><LI>Shaking culture the reactant for 1 hour, at 37 degrees Celsius, 200rpm. </LI><LI>Centrifuge the reactant for 5 minutes at 3000-4000rpm to collect the cell. Throw away 500μL supernatant, then re-suspend the rest reactant. Use 100μL reactant to spread plate(Luria-Bertani + appropriate antibiotic), and the rest is reserved under 4 degrees Celsius. </LI><LI>Culture the cell overnight under 37 degrees Celsius. (If chloramphenicol is added, then the culture time can be longer) </LI></OL><H2>12. Prepare competent cell</H2><OL class="l_finalol"><LI>Inoculate strains in 5ml YPD medium and culture them at 200rpm, 30 degrees Celsius overnight. </LI><LI>Add 500μl mother culture into 50ml YPD and culture them at 200rpm, 30 degrees Celsius for 12-14 hours, then make sure the OD value of the fungal fluid is between 1.3-1.5. </LI><LI>Put the fungal fluid into 50ml sterilized centrifuge tube and make sure the solution is balanced before centrifuging. Centrifuge the fungal fluid at 4000rpm, 4 degrees Celsius for 5 minutes and then collect the sediment. Add 3ml icy sterile water, then use the transfer liquid gun to mix up and re-suspend the sediment. Add 25ml icy sterile water to wash the sediment and centrifuge it at 4000rpm, 4 degrees Celsius for 5 minutes for two times. (Do not touch the fungal fluid with hands to make sure it is kept under low temperature during the process.) </LI><LI>Add 10ml icy sterile water to re-suspend the sediment, then add 1ml pH=7.510×TE loading buffer and mix them up on ice on the super-clean bench. Add 10×LiAc and shaking culture it at 90rpm, 30 degrees Celsius for 45 minutes. </LI><LI>Add 0.4ml 1mol/L DTT and shaking culture it at 90rpm, 30 degrees Celsius for 15 minutes. </LI><LI>Centrifuge the fungal fluid at 4000rpm, 4 degrees Celsius for 5 minutes and then throw away the supernatant. Use 25ml icy sterile water to wash the sediment. (The same washing method as step (3).) </LI><LI>Use 10ml 1mol/L icy sorbitol to wash the sediment. Centrifuge it at 4000rpm, 4 degrees Celsius for 5 minutes and then throw away the supernatant. </LI><LI>Add 200μl sorbitol into each centrifuge tube to dissolve the sediment and transfer it into EP tube (180μl per tube). Use 2 tube of EPY100 for electroporation, and add 10μl glycerol into each of the rest tubes and preserve it under -80 degrees Celsius.</LI></OL><H2>13. Electric transfer</H2><H3>the cleaning of the electric rotor: </H3><P>Clean with distilled water, immerse with alcohol in the clean bench for 10min, add sterile water, use a syringe suction cleaning for 3 times, absorb liquid and open the cover (lid should be washed) under UV for 1H, place in ice precooling.
        </P><OL class="l_finalol"><LI>Add 5~10μg DNA (volume &lt;10μL) to the competent cell, blow with a gun (gently); (9 L PYD1, 1- alpha 1 g/ L), transfer to the pre-cooled electric cup, and put aside for 5min; </LI><LI>Set shock parameters: 1.5Kv, 25 F, 200Ω; </LI><LI>Immediately add 1ml precooling sorbitol, transfer to the EP tube, 30 degrees and be static for 1h; </LI><LI>4000rpm and 5min are set to centrifuge at normal temperature, and then supernatant is abandoned and add 1ml YPD, centrifuge at 30 degrees for 200rpm and 2h; </LI><LI>4000rpm, 5min at room temperature centrifugation, suction 550μL supernatant, 150μL/ plate used for coating board [before coating board, culture board should be drying]. </LI></OL><H2>14. SDS-PAGE</H2><H3>Materials</H3><TABLE><TBODY><TR><TD>Gel</TD><TD>Tris-HCl </TD><TD>Acr/Bis 30%</TD><TD>SDS 10%</TD><TD>ddH2O</TD><TD>TEMED</TD><TD>AP 10%</TD></TR><TR><TD>Stacking Gel(4%)</TD><TD>pH=6.8 500 μl</TD><TD>500μl</TD><TD>25μl</TD><TD>1350μl</TD><TD>2.5μl</TD><TD>12.5μl</TD></TR><TR><TD>Running Gel(12%)</TD><TD>pH=8.8 1250μl</TD><TD>2000μl</TD><TD>50μl</TD><TD>1675μl</TD><TD>2.5μl</TD><TD>25μl</TD></TR><TR><TD>Running Gel(18%)</TD><TD>pH=8.8 1250μl</TD><TD>3000μl</TD><TD>50μl</TD><TD>675μl</TD><TD>2.5μl</TD><TD>25μl</TD></TR></TBODY></TABLE><H3>Running Buffer</H3><TABLE><TBODY><TR><TD>Reagent</TD><TD>Tris-HCl</TD><TD>Glycine</TD><TD>(w/v) SDS</TD></TR><TR><TD>Dosage</TD><TD>25mmol/L</TD><TD>0.192mol/L</TD><TD>0.1%</TD></TR></TBODY></TABLE><OL class="l_finalol"><LI>The SDS polyacrylamide gels are prepared in the so-called PerfectBlue™ Twin Double Gel System. </LI><LI>After ensuring that the equipment is waterproof, the 12% (or 18%) running gel is mixed and filled into the chamber. Pipetting about 1 ml of H2O on top of the running gel to seal the gel. </LI><LI>After polymerization, the remaining H2O is removed and the 12% stacking gel is filled on top. Insert a comb to create sample pockets. </LI><LI>After the stacking gel also polymerized, 1x running buffer is used to run the Double Gel System via the SDS gel. </LI><LI>After loading the generated pockets with the samples, the stacking gel is run at 100 V and then running gel at 120 V. </LI></OL><H2>15. Western Blot</H2><H3>System</H3><P>TBST:1000 mL(pH=7.4) </P><TABLE><TBODY><TR><TD>Reagent</TD><TD>NaCl(137mM)</TD><TD>KCl(2.7mM)</TD><TD>Na2HPO4(10mM)</TD><TD>K2HPO4(2mM)</TD><TD>Tween-20</TD></TR><TR><TD>Dosage</TD><TD>8g</TD><TD>0.2g</TD><TD>1.44g</TD><TD>0.24g</TD><TD>0.5ml</TD></TR></TBODY></TABLE><P>Imprint buffer：2000 mL (pH=8.3) Transfer Buffer</P><TABLE><TBODY><TR><TD>Reagent</TD><TD>Tris</TD><TD>Gly</TD><TD>Methanol</TD></TR><TR><TD>Dosage</TD><TD>6.06g</TD><TD>28.8g</TD><TD>400ml</TD></TR></TBODY></TABLE><H3>Transfer</H3><P>(Prepare transfer Buffer just before glue leaking, and precool at -20℃). </P><OL class="l_finalol"><LI>Put the transfer Buffer and the black subface of transfer splint downward, and lay a sponge in it. Several filter paper(three pieces of filter paper), glue(except Stacking Gel). </LI><LI>Activation PVDF membrane in advance with anhydrous ethanol, and put it on the membrane. </LI><LI>Three layers of filter paper, sponge, Squeeze out of the bubbles, turn tight. </LI><LI>The black subface electric rotary groove stick to each other, put in ice. </LI><LI>110V, 120min. </LI><LI>5% skim milk powder (prepared by TBST), block for a night. </LI><LI>Dilute Primary antibody at the proportion of 1:2000 with 3% skim milk powder (add 0.02% sodium azide), incubate 1h at the room temperature. </LI><LI>TBST elute, wash with shocking for 5min, three times. </LI><LI>Dilute Secondary antibody at the proportion of 1:2000 with 3% skim milk powder, incubate 1h at the room temperature. </LI><LI>TBST elute, wash with shocking for 5min, three times. </LI><LI>Color development. </LI></OL><H2>16. Detection of proteins</H2><OL class="l_finalol"><LI>Remove the film, blot the water with the filter paper and wrap it in a dark room. </LI><LI>Exposure: prepare develop and fix buffer, pull the curtain, open the lamp. Take the plate and cut it into two parts. Open the blind clip, put the film in the NC film, cover it, and expose for 10 seconds to 5 minutes. </LI><LI>Take the film and place it in the develop buffer for 2 minutes. </LI><LI>Leave the develop liquid in the fixative for 1 minutes, and the ribbon appears. </LI></OL><H2>17. Target protein purification</H2><P>Using Ni Sepharose 6 Fast Flow</P><TABLE><TBODY><TR><TD>Binding buffer</TD><TD>20mM sodium phosphate</TD><TD>0.5M NaCl</TD><TD>pH 7.4</TD></TR><TR><TD>Wash buffer</TD><TD>20mM sodium phosphate</TD><TD>0.5M NaCl</TD><TD>0 to 30mM imidazole, pH 7.4</TD></TR><TR><TD>Elution buffer</TD><TD>20mM sodium phosphate</TD><TD>0.5M NaCl</TD><TD>500mM imidazole, pH 7.4</TD></TR></TBODY></TABLE><OL class="l_finalol"><LI>If the column contains 20% ethanol, wash it with 5 column volumes of distilled water. Use a flow velocity of 50 to 100 cm/h. Refer to Appendix 8 for flow rate calculations. </LI><LI>Equilibrate the column with at least 5 column volumes of binding buffer at a flow velocity of 150 to 600 cm/h. </LI><LI>Load the sample at a flow velocity of 150 to 600 cm/h. </LI><LI>Wash with 20 column volumes of wash buffer at a flow velocity of 150 cm/h. </LI><LI>Elute with elution buffer using a one-step procedure. Five column volumes of elution buffer is usually sufficient. Alternatively, a linear elution gradient (10 to 20 column volumes) may give higher purity, at the expense of lower target protein concentration in eluted fractions. Use a flow velocity of 150 cm/h. </LI></OL><H2>18. Extracting flagellin</H2><OL class="l_finalol"><LI>Inoculate E.coli K12 into 600mL LB fluid medium, culture E.coli K12 for 16 hours in a shaking table at 37℃.</LI><LI>Centrifugal the bacteria with the fluid medium for 20 minutes at 5,000rmp/min at 4℃ and throw away liquid supernatant. Resuspend the sediments of bacteria with a moderate amount of sterilized normal saline and then adjust suspension to pH 2.0 (concentrated hydrochloric acid acquired). Stir the bacteria liquid with a magnetic stirrer for 6 minutes at 4℃ in order to make the flagellum fall off.</LI><LI>Centrifugal the bacterium fluid at 10,000rmp/min at 4℃ for 40 minutes. Absorb the supernatant fluid and filter it with 0.22μm diameter filter to remove impurities.</LI><LI>Adjust suspension to pH 7.0, add copper sulfate solid into liquid supernatant to a final concentration of 40% with gently stirring. After 8 hours at 4℃, flagellin precipitation from the liquid gradually.</LI><LI>Centrifugal the liquid at 12,000rmp/min at 4℃ for 40 minutes, throw away the waste liquid and resuspend the sediments with 5mL PBS, after filtration, store at -80℃ for later use.</LI></OL><H2>19. Extracting tubulin from porcine brains</H2><OL class="l_finalol"><LI>Pick up 20 porcine brains from Beijing No.5 Meat Processing. For tubulin extraction experiment, the brains should be as fresh as possible. Take an ice box to store the brains. Avoid contact between the brains.</LI><LI>While getting the brains, another student should stay in the lab and prepared the centrifuge (set one at 4℃ and another at 37℃, also pre-warm the rotor). Put electronic balance, grinder, a graduated cylinder in refrigerator and pre-warm the glycerol and PEM in 37℃. prepare the fresh ATP and GTP buffer in the morning.</LI><LI>Clean the brain by tearing off the meninges and blood clots using kim wipes or by hand.</LI><LI>After cleaning, weigh the brains, put the brains in the blender, then add the same volume buffer PEM(with 1 mM DTT) in it accordingly.</LI><LI>Homogenate the brain for 3s, 10 times, time interval between two homogenate is 5s, in order to avoid destroy the tubulin because of high thermos.</LI><LI>Pour the homogenate into a flask, incubate in 4℃ for 30 min to depolymerize microtubules.</LI><LI>Pour the homogenate into tubes for Type 45Ti rotor and balance each tube.</LI><LI>Centrifuge at 8000 rpm for 40min at 4℃. Filter the supernatant with 4 gauzes. Then centrifuge the filtrate at 40000 g for 40min at 4℃.</LI><LI>Add 1/2 volume of warmed glycerol drop-wise with continuous shaking, mix gently but thoroughly. Add GTP (final concentration 0.1 mmol/L), MgCl2 (final concentration 3 mmol/L), and EGTA (final concentration 1 mmol/L). Incubate in a 37℃ water bath for 1h, shake gently and occasionally.</LI><LI>Balance each tube and centrifuge at 100000 g for 40 min at 35℃.Discard supernatant, the pellet is crude extracts. Split charge them into 50 mL centrifuge tubes, each tube contains 5 g. Snap freeze the tubulin in 15 μL aliquots in liquid nitrogen and further stored at -80℃.</LI><LI>When going to do refined depuration, melt the freezing crude extract at 4℃ refrigerator on ice over night. Pop out 1 g of the pellet out of the tubes with a spatula. Put the pellets in the Dounce grinder, then add cold PEM in the tube to wash off residual pellets.</LI><LI>Re-suspend the pellets with grinder, keep the grinder on ice, and grinding occasionally. After 30min, pour out the solution and rinse the grinder with cold PEM. Total re-suspended volume is 5 mL.</LI><LI>Add GTP (final concentration 0.1 mmol/L). Place it on ice for 1h to depolymerize. Shake it occasionally.</LI><LI>Centrifuge the depolymerized tubulin at 100000 g for 40 min at 4℃.</LI><LI>Recover the supernatant and pour it into a flask. Add equal volume of warmed PIPES( pH=6.9 ), DMSO(final concentration 10%), GTP (final concentration 0.1 mmol/L), MgCl2 (final concentration 1 mmol/L) and EGTA (final concentration 1 mmol/L). Mix gently but thoroughly.</LI><LI>Incubate in a 37℃ water bath for 1 h. the solution would look cloudy.</LI><LI>Balance each tube and centrifuge at 100000 g for 1h at 35℃.</LI><LI>Discard the supernatant. Risen the pellet briefly with cold PEM, add GTP (final concentration 0.1 mmol/L). Place it on ice for 1h to depolymerize. Shake it occasionally.</LI><LI>Centrifuge the depolymerized tubulin at 100000 g for 40min at 4℃.</LI><LI>Recover the supernatant and pour it into a flask. Add equal volume of warmed PIPES ( pH=6.9 ), DMSO(final concentration 10%), GTP (final concentration 0.1 mmol/L), MgCl2(final concentration 1 mmol/L) and EGTA (final concentration 1 mmol/L). Mix gently but thoroughly.</LI><LI>Incubate in a 37℃ water bath for 1 h. the solution would look cloudy.</LI><LI>Balance each tube and centrifuge at 100000 g for 1h at 35℃.</LI><LI>Discard the supernatant. Risen the pellet briefly with cold PEM, add GTP (final concentration 0.1 mmol/L). Place it on ice for 1 h to depolymerize. Shake it occasionally.</LI><LI>Centrifuge the depolymerized tubulin at 100000 g for 40 min at 4℃.</LI><LI>Recover the supernatant, add equal volume polymerize buffer (containing 100 mmol/L PIPES-KOH, 2 mmol/L EGTA, 2 mmol/L MgCl2, 2 mmol/L GTP and 60% glycerol).</LI><LI>Incubate in a 37℃ water bath for 1 h. Adding a series concentrations of taxol.</LI><LI>Balance each tube and centrifuge at 100000 g for 1 h at 35℃. The pellet is fine purified product.</LI></OL><H2>20. FilC Polymerization Condition</H2><TABLE><TBODY><TR><TD>Components</TD><TD>Initiation phase</TD><TD>Elongation phase</TD></TR><TR><TD>Flagellin(FliC)</TD><TD>&gt;4mg/ml </TD><TD>&gt;5mg/ml </TD></TR><TR><TD>NaCl</TD><TD>0.04M</TD><TD>none</TD></TR><TR><TD>phosphate buffer</TD><TD>0.01-0.03M</TD><TD>0.01-0.03M</TD></TR><TR><TD>pH</TD><TD>6.5±0.2</TD><TD>6.5±0.2</TD></TR><TR><TD>Na2SO4</TD><TD>0.43-0.64M</TD><TD>0.43-0.64M</TD></TR><TR><TD>KCl</TD><TD>none</TD><TD>0.2-0.3M</TD></TR><TR><TD>Temp</TD><TD>20-30 degrees Celsius (26 degrees Celsius)</TD><TD>20-30 degrees Celsius (26 degrees Celsius)</TD></TR></TBODY></TABLE><H2>21. Tubulin Polymerization</H2><OL class="l_finalol"><LI>Turn on fluorimeter and enter experimental parameters. </LI><LI>Place the 96 well plate in the fluorimeter and allow to warm to 37 degrees Celsius for 10 minutes. </LI><LI>Thaw out the paclitaxel stock solution and aliquot 5μl of this into 325μl of room temperature sterile distilled water. This gives you a 30μM 10× paclitaxel solution. Keep this at room temperature until use. </LI><LI>Prepare 10×stock solutions of your test compounds or protein of interest. </LI><LI>Do not continue until the compounds are fully ready. </LI><LI>Defrost 1.5ml of Buffer 1 and place on ice. </LI><LI>Defrost 20μl of GTP stock and place on ice. </LI><LI>Remove the Tubulin Glycerol Buffer from 4 degrees Celsius and place on ice. </LI><LI>Defrost 88μl of Tubulin in a room temperature water bath until liquid, then immediately place on ice. </LI><LI>Immediately mix the assay components as shown below to make the Tubulin </LI></OL><H3>Reaction Mix: </H3><TABLE><TBODY><TR><TD>Component</TD><TD>Volumes for Standard Conditions</TD><TD>Volumes for Inhibitor Detection</TD><TD>Volumes for Enhancer Detection</TD><TD>Final Concentration</TD></TR><TR><TD>Buffer 1</TD><TD>243μl</TD><TD>205μl</TD><TD>355μl</TD><TD>1×</TD></TR><TR><TD>Tubulin Glycerol Buffer</TD><TD>112μl</TD><TD>150μl</TD><TD>none</TD><TD>Varies</TD></TR><TR><TD>GTP stock (100mM)</TD><TD>4.4μl</TD><TD>4.4μl</TD><TD>4.4μl</TD><TD>1mM</TD></TR><TR><TD>Tubulin stock (10 mg/ml)</TD><TD>85μl</TD><TD>85μl</TD><TD>85μl</TD><TD>2mg/ml</TD></TR></TBODY></TABLE><OL class="l_finalol"><LI value="11">Pipette 5μl of control buffer (exact match to your compound or protein buffer) into the first duplicate wells A1&amp;B1. Then pipette 5μl of the 10× paclitaxel stock into C1&amp;D1, followed by 5μl of the test compound into E1&amp;F1 etc, repeat until all compounds are aliquoted. </LI><LI>Place plate back into warm plate reader for 1 minute, but no longer as the 5μl volume will evaporate quickly. </LI><LI>Pipette 50μl of the tubulin reaction mix into each of the eight wells using a single channel pipettor. Work rapidly, use medium pipetting speed and keep the tip of the pipettor on the wall of the well. This technique avoids bubble formation which will disrupt absorbance readings. </LI><LI>Start plate reader immediately. If the software crashes in the first 5 minutes it is worthwhile to restart the process because some data can be rescued in most cases. </LI></OL><H2>22. Using DNS for determination of reducing sugars</H2><P>3,5-Dinitrosalicylic acid (DNS or DNSA, IUPAC name 2-hydroxy-3,5-dinitrobenzoic acid) is an aromatic compound that reacts with reducing sugars and other reducing molecules to form 3-amino-5-nitrosalicylic acid, which strongly absorbs light at 540 nm.</P><OL class="l_finalol"><LI>Take six clean test tubes, adding various reagents in the following order:</LI><OL class="l_finalol" style="list-style-type:disc;margin-left: 180px;font-size: 15px;"><LI>Xylose solution </LI><LI>Distilled water </LI><LI>DNS reagent </LI></OL></OL><TABLE><TBODY><TR><TD>The amount of sugar contained</TD><TD>0 mg/ml,    0.2 mg/ml,    0.4 mg/ml,    0.6 mg/ml,    0.8 mg/ml,    1.0 mg/ml</TD></TR><TR><TD>Distilled water</TD><TD>1.0 mg/ml,    0.8 mg/ml,    0.6 mg/ml,   0.4 mg/ml,    0.2 mg/ml,    0 mg/ml</TD></TR><TR><TD>DNS reagent</TD><TD>2ml</TD></TR><TR><TD>Heating</TD><TD>Heat 5 minutes（90℃）</TD></TR><TR><TD>Cooling</TD><TD>Immediately with the flow of cold water to cool</TD></TR><TR><TD>Distilled water</TD><TD>Add distilled water again：9ml</TD></TR></TBODY></TABLE><OL class="l_finalol"><LI value="2">Mix the above solution, each tube after the photoelectric colorimeter (540 nm) for colorimetric determination of the blank control solution with zero adjustment, recording the optical density value. Each group of samples detect three times in parallel, xylose concentration as the abscissa, the ordinate is optical density standard curve plotted.</LI><LI>Extract the protein from EBY100（each tube 200 microlitre）, add 5% xylan solution 800 microlitre in each tube，incubate 10minutes in 50℃ metal bath.</LI><LI>Detectitsopticaldensityvalue as mentionedabove.</LI></OL><DIV style="background-color:#F7EDB7;text-align:center;"><P style="text-align: center;color: #e5b051;padding-top: 5px;margin-top:75px;">Copyright © 2017 <A href="https://2017.igem.org/Team:BNU-China" style="color: #9E2B20;text-decoration: none;opacity:0.8;">BNU-China</A> All rights reserved.</P><P style="text-align: center;color: #e5b051;margin-bottom:-20px;">If you like this page, you can contact us: <SPAN style="color:#9E2B20;text-decoration: none;opacity:0.8;">bnu_igem@163.com</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>