<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>BNU-China</TITLE> &quot;&gt;
    </P><DIV id="w_nav" style="top:15px;"><UL><LI id="w_Project"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Description">Project</A></LI><LI id="w_Modeling"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Model">Modeling</A></LI><LI id="w_Practice"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/HP">Practice</A></LI><LI id="w_home"><A class="w_a" style="opacity:0;" href="https://2017.igem.org/Team:BNU-China">Home</A></LI><LI id="w_Achievement"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Judging">Achievement</A></LI><LI id="w_Lab"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Team">Lab</A></LI><LI id="w_Safety"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Safety">Safety</A></LI></UL><DIV id="w_navMenu" style="text-align:right;"><P>Menu▼</P><UL id="w_navMenuContent"><LI id="w_Project"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Description">Project</A></LI><LI id="w_Modeling"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Model">Modeling</A></LI><LI id="w_Practice"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/HP">Practice</A></LI><LI id="w_Achievement"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Judging">Achievement</A></LI><LI id="w_Lab"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Team">Lab</A></LI><LI id="w_Safety"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Safety">Safety</A></LI></UL></DIV></DIV><DIV id="w_content" style="width:100%;"><DIV id="w_Panel" style="text-align:left;"><UL id="w_menu"><LI id="w_menu1"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Description">Background</A></LI><LI id="w_menu2"><A class="w_a" href="#">Design</A></LI><LI><P href="https://2017.igem.org/Team:BNU-China/Design" data-id="#title1">Overview</P></LI><LI><P href="https://2017.igem.org/Team:BNU-China/Design" data-id="#title2">Microtubule</P></LI><LI><P href="https://2017.igem.org/Team:BNU-China/Design" data-id="#title3">Flagellar Filament</P></LI></UL><LI id="w_menu5"><A class="w_a" href="https://2017.igem.org/Team:BNU-China/Demonstrate">Results</A></LI></DIV><DIV id="w_text"><H1>Design</H1><H2 id="title1">Overview</H2><H3>Materials</H3><P>To achieve our goal, the materials we use should be self-assembling protein subunits that can polymerize into long fibrous proteins. After searching and comparing many materials, we finally chose α-/β-tubulin and flagellin.
                </P><H3>Approach</H3><P>After a thorough discussion of various plans, we finally decided to build two parts: the display module and the secretory module. So here, with the special structure of surface display system, we can display the particular subunit onto the yeast surface as a linkage site and secrete other subunits into the extracellular environment to create a subunit-rich surroundings. With the optimal condition, those subunits could reconstitute into polymers.
                </P><H3>Display module</H3><P><B>A-agglutinin System</B>Among various GPI-anchored proteins, α/a-agglutinin is one of the most common proteins that have been utilized for the display of proteins on a yeast cell surface (Liu, 2016). The native a and α agglutinin receptors are believed to act as adhesion molecules to stabilize cell-cell interactions during mating and to facilitate fusion between the a and α haploid yeast cells. A-agglutinin has a core subunit encoded by AGA1 and a binding subunit encoded by AGA2. Both a-agglutinin and the core subunit of a-agglutinin consist of a secretion-signal region, a functional region, a support region, and a putative GPI anchor-attachment signal. After genetic engineering, foreign proteins can be fused to the C-terminal of the binding subunit of a-agglutinin. As AGA1 protein is secreted from the cell and becomes covalently attached to the β-glucan (Invitrogen, 2002) in the extracellular matrix of the yeast cell wall, AGA2 protein fused with a foreign protein on its C-terminus will then bind to aga1p through two disulfides bonds. As a result, the heterologous proteins are covalently linked with the glucan-layer.
                </P><P>It is worth mentioning that AGA1 gene is controlled by GAL10 promotor in the genome of <I>S. cerevisiae</I> EBY100 strain, while AGA2 gene is controlled by the GAL1 promotor on the vector pYD1. (see Figure 1)
            </P><H4>Figure 1 The structure of yeast surface display system
      (Source: pYD1 Yeast Display Vector Kit Handbook Version D, 2002)</H4><H3>Secretory module</H3><P><B>Alpha-factor</B> We use α-Factor to facilitate the secretion of our exogenous proteins. (see Figure 2)
                 α-Factor is a 13-residue peptide, secreted by cells of α mating type, that acts on cells of the opposite mating type to promote efficient conjugation between the two cell types, which leads to the formation of aα diploid cells. α-factor is synthesized as a precursor of 165 amino acids containing an 83-residue leader and four α-factor coding regions (Lin-Cereghino GP, 2013), each preceded by a short spacer peptide.
            </P><H4>Figure 2 Predicted structure of the α-mating factor pre-pro secretion signal.
      The first helix is part of the pre-region, whereas the following two helixes are part of the pro-region.
      (Source: Lin-Cereghino GP, Stark CM, Kim D, et al. 2013)</H4><P>As a natural signal peptide that is widely present in yeast, α-factor has strong and potent secretory properties. Studies have shown that its secretion is almost 100% (Lin-Cereghino GP, 2013), which can prevent the secretion of exogenous tubulin from interrupting the yeast's inherent microtubule system.
                </P><H2 id="title2">Armed with Microtubules</H2><H3>Overview</H3><P><B>What is microtubule?</B> Microtubules are fibrous elements in the cytoplasm of eukaryotic cells, where they play comparably diverse and important roles. The microtubule building block is the tubulin αβ heterodimer (Caudron, 2002). A typical microtubule is a hollow water-filled tube with an outer diameter of 25 nm (Fygenson, 1994) and an inner diameter of 15 nm. Alpha-beta heterodimers associate head-to-tail to form protofilaments running lengthwise along the microtubule wall with the beta-tubulin subunit facing the microtubule plus end conferring a structural polarity (Alberts, 2014). Microtubules usually have 13 protofilaments but different protofilament numbers can be found in some organisms and specialized cells. (see Figure 3)
                </P><H4>Figure 3 (A) Tubulin heterodimers α/β. GTP molecules are shown in red. (B) Schematics of one subunit and one protofilament. (C) Stiff hollow microtubule consists of 13 protofilaments. (D) A short segment view of microtubule under electron microscope.(E) A view of ring structure constituted by 13 protofilaments under electron microscope.
      (Source: Alberts, et al. Molecular Biology of the Cell 6th, 2014)</H4><P>From previous literature research, we found that microtubules can self-assemble in vitro under appropriate conditions. The assembly of microtubule from purified tubulin consists of two steps: microtubule nucleation and microtubule elongation.
                 Whenever in vivo or in vitro, microtubule dynamics is profoundly influenced by the binding and hydrolysis of nucleotide—GTP. The tubulin heterodimers are known to bind GTP in different stability (Tian, 1999): α-tubulin binds GTP nonexchangablely, whereas the GTP-binding sites on the β-tubulin is exchangeable. The hydrolysis of GTP is coupled with the assembly of microtubule such that variation in the size of the &quot;cap&quot; of GTP-bound subunits on a given microtubule determines whether it will continue to grow or undergo transition to a rapidly depolymerizing phase. Generally, the change from growth to shrinkage is called a <I>catastrophe</I>, while the change to growth is called a <I>rescue</I> (Alberts, 2014). Therefore, due to the unique features of its <I>dynamical behaviors</I>, suitable condition is needed for tubulin dimers to reassemble into microtubules <I>in vitro</I>.
                </P><P><B>Why microtubule?</B> Microtubules, a component of the cytoskeleton, are widely found in eukaryotic cells. They are very important in a number of cellular processes, such as maintaining cell morphology, intracellular transport and cell division. So it has been a hot spot for research. Displaying microtubules on yeast surface will greatly promote the study of microtubules.
                 Moreover, Microtubules can act as substrates for motor proteins. Maybe we can use microtubule molecular motors to create a micro-factory based on the extracellular microtubule network we built.
                 Additionally, due to the important role microtubule plays in mitosis, it is also a major target in cancer treatment. As microtubule-stabilizing agents become more and more important in cancer chemotherapy, it's really necessary to find more antitumor agents such as Taxol<SUP>®</SUP>, or other agents that can also stabilize microtubules. Displaying microtubules on the yeast surface, we can easily observe the interaction between small molecules and microtubules, which is very helpful for cancer treatment.
                </P><H4>Figure 4 The protein system used to simulate the interaction between microtubules and Taxol<SUP>®</SUP>. GTP and GDP are shown in cyan and gold, respectively. The M-loop for this monomer is colored yellow and the interacting H1-S2 loop of the adjacent monomer is colored red.
      (Source: Arpita M, et al. Biophysical Journal, 2008.)</H4><P><B>What do we plan to do using microtubule?</B> Thanks to the comprehensive human practice work, we learned a crucial fact: considering the poor stability of microtubules and their strict polymerization conditions, we'd better not use microtubules in large-scale industrial applications, but should rather use it in a relatively mild laboratory environment.
                 Thus, we regard microtubule as an excellent tool for high-throughput screening of microtubule-stabilizing agents. Furthermore, we attempt to construct a screening platform on our microtubule display technique.
   </P><H3>Display module-parts design</H3><P>Here we choose the vector pYD1 to fulfill the display function. As shown on the Figure 5, pYD1 vector contains AGA2 gene, which has described above, codes AGA2 protein. And our target protein is fused on the C-terminus of AGA2p, which is shown as multiple cloning sites in Figure 5.
                 So after inducing by galactose, our target gene expression can be started by GAL1 promotor. Additionally, there are also several tags on the plasmid such as V5 tag and His-tag, which will be used in further analysis.
                 In this display module, we designed two basic parts: pYD1-α-tubulin <A href="">(BBa_K2220019)</A> and pYD1-β-tubulin <A href="">(BBa_K2220020)</A>, which means to insert α-tubulin or β-tubulin gene sequences respectively.
                </P><H4>Figure 5 (A) Map of pYD1. (Source: pYD1 Yeast Display Vector Kit Handbook Version D, 2002) 
      (B) Maps of pYD1-α-tubulin and pYD1-β-tubulin</H4><H3>Secretory module-parts design</H3><P>In the secretory module, we choose GAL1-controlled expression vector, which contains an alpha-factor signal peptide to facilitate the secretion of our exogenous proteins. (see Figure 6)
                </P><H4>Figure 6 Map of pYES2/CT/α-Factor</H4><P>And then, we insert varied target gene sequences respectively into the multiple cloning site shown as a square box below. (see Figure 7)
                 Consequently, there will be four types of recombinant proteins. They are alpha-tubulin<A href="">(pYCα-α tubulin: BBa_K2220022)</A>, β-tubulin <A href="">(pYCα-β tubulin: BBa_K2220023)</A>, mcherry fused with α-tubulin<A href="">(pYCα-mCherry-α tubulin: BBa_K2220024)</A> and β-tubulin fused with mGFP<A href="">(pYCα-β-tubulin-mGFP: BBa_K2220025)</A>. (see Figure 7) We also designed a vector which contains alpha-factor fused with mCherry as our positive control in protein expression analysis. This positive control part is pYCα-mCherry. <A href="">(BBa_K2220021)</A></P><H4>Figure 7 Schematic of secretory module bio-bricks</H4><H3>Polymerization</H3><H4>Figure 8 The schematics of microtubule polymerization on yeast cell surface</H4><P>Mixing the yeast of module 1 and the production of module 2 in one system, we can then accomplish the self-assembly of microtubules on the yeast cell wall in suitable conditions. Once the assembly is stabled by Taxol<SUP>®</SUP>, the fluorescent intensity of the sample can be different. So we can wash the remaining free tubulin dimers and test the affinity effect by the fluorescent intensity difference.
                </P><H2 id="title3">Armed with Flagellar Filaments</H2><H3>Overview</H3><P><B>What is flagellar filament?</B> As we all know, the flagella filament is a helical structure composed of subunits of flagellin. And among various flagellin, the FliC has its own unique features. The N- and C-terminal regions of the FliC are important for filament formation (Klein, 2012), which are the D0 and D1 domain in Figure 8. The D3 domain is a central region of the Flic which plays no role in filament formation. Additionally, when FliC proteins are in the polymeric state, the D3 domain can be exposed on to the surface. Therefore, we can replace the central region of flagellin by other proteins without disturbing self-assembly. If we fuse the enzyme and FliC, we can obtain a filamentous material with high surface density of catalytic sites.
                </P><H4>Figure 9 Construction of the FliC fusion protein. (A) Arrangement of flagellin subunits within the flagellar filament. D3 domain is exposed on to the surface. (B) The C<SUB>α</SUB> backbone trace of flagellin C protein and eGFP. eGFP replaces D3 domain in the recombinant protein. eGFP can also be replaced by other enzymes. The N- and C- terminal of the flagellin C protein are D0 and D1 domain, which play an essential role in flagellin polymerization.(Source: Klein, Ágnes, et al. &quot;A polymerizable GFP variant.&quot; Protein Engineering Design &amp; Selection Peds, 2012) </H4><P><B>Why flagellar filament?</B> Flagellar filament is a quite stable material which only depolymerizes in extreme conditions such as pH lower than 2.0 or extremely high temperatures (Klein, 2015). Additionally, the polymerization of flagellar filament is operable <I>in vitro</I>, and we can simply add different concentrations of precipitants to control the nucleation and elongation process of flagellin monomers. So far, the successful expression of FliC-enzyme fusion proteins has been well documented, so we can make further efforts on exploring the possibility of yeast surface display system by using flagellar filament. Furthermore, considering biosafety issue, we choose Flagellin protein which origin from <I>E. coli</I> K-12. Because <I>E. coli </I>K-12 is a commonly used strain in lab and the FliC protein of its flagellin does not have antigenicity.
                </P><P><B>What we plan to do with flagellar filament?</B> Due to the stability of flagellar filaments, our MonKeYeast armed with flagellar filament is qualified to play a role in industry.
                 We believe that its greatest value lies in whole-cell multiple Biocatalysts. So we try to build extracellular metabolic pathways, coordinated with intracellular metabolic pathways, to reduce the burden of bacteria. The successful construction of this system will lead to a new era in the field of fermentation engineering.
                 To fulfill our ambitions, we choose these following five enzymes: xylanase A, PETase (which has been mutated and displayed by TJUSLS-2016), endo-1,4-β-D-glucanohydrolase (EG), exo-1,4-β-D-glucannase (CBH), and β-1,4-glucosidase (BG).
                </P><H3>Display module-parts design</H3><P>Same as the microtubule section, here we choose the vector pYD1 to fulfill the display function. As shown in Figure 5, pYD1 vector contains AGA2 gene, which, as has been described above, codes AGA2 protein. Our target protein is fused on the C-terminus of AGA2p, which is shown as multiple cloning sites in Figure 5.
                 So after being induced by galactose, our target gene expression can be initiated by GAL1 promotor. Additionally, there are also several tags on the plasmid such as V5 tag and His-tag, which will be used in further analysis.
                 Here we also divided parts design into two parts: displaying the building block—FliC, and displaying several monomer enzymes respectively to test and compare the catalytic activities of the monomeric display enzyme and the polymeric display enzyme. There is a small difference between these two parts. If peptide is fused on the N-terminus or C-terminus of flagellin, the polymerization will be significantly affected. So we cut down the AGA2 gene from the vector pYD1, and inserted it into FliC gene to replace the central region. Research shows that the His-tag should be fused on the C-terminus of the target protein so as not to affect the polymerization.
                 Therefore, we designed the following parts: pYD1-FliC <A href="">(BBa_K2220002)</A>, pYD1-XynA<A href="">(BBa_K2220004)</A>, pYD1-PETase <A href="">(BBa_K2220005)</A>, pYD1-BG
                <A href="">
                    (BBa_K2220007)</A>, pYD1-EG <A href="">(BBa_K2220006)</A>, and pYD1-CBH <A href="">(BBa_K2220008)</A>, which means to fuse the target gene sequences with AGA2 gene respectively. We also designed pYD1-FliC (eGFP) <A href="">(BBa_K2220003)</A> as our positive control. The maps of the parts are presented below. (See Figure 10 &amp; Figure 11)
                       </P><H4>Figure 10 Schematic of FliC-GFP display bio-brick</H4><H4>Figure 11 Schematic of parts in monomeric display module</H4><H3>Secretory module-parts design</H3><P>In the secretory module, we choose GAL1-controlled expression vector, which contains an alpha-factor signal peptide to facilitate the secretion of our exogenous protein. (see Figure 6) Considering the structure of the yeast surface display module mentioned above, we redesign the gene and tag locations on the vector pYES2/CT/α-Factor to ensure the further reassembly of recombinant FliC and enzyme fusion proteins.
                 Here we change the location of AGA2, His-tag, which are inserted into the central region of FliC. In this way, the structure of recombinant Enzyme-FliC fusion proteins fit the flagellin polymerization process.
                 We designed the following parts: pYCα-FliC (XynA) <A href="">(BBa_K2220011)</A>, pYCα-FliC (PETase) <A href="">(BBa_K2220012)</A>, pYCα-FliC (BG) <A href="">(BBa_K2220014)</A>, pYCα-FliC (EG) <A href="">(BBa_K2220013)</A>, pYCα-FliC (CBH) <A href="">(BBa_K2220015)</A></P><H4>Figure 12 Schematics of secretory module bio-bricks</H4><H3>Polymerization</H3><P>The monomeric flagellin can nucleate in high concentrations of sulfate solution. And it can elongate in the elongation solution, after these two steps, we can obtain flagellar filaments of different lengths.
                </P><H4>Figure 13 The schematics of flagellar filaments polymerization on yeast cell surface</H4><DIV class="refer"><P>Alberts, Bruce, and Alexander Johnson. &quot;MICROTUBULES.&quot; Molecular Biology of the Cell, Garland Science, 2015, pp. 925–929. 
               </P><P>Caudron, N, et al. &quot;Microtubule nucleation from stable tubulin oligomers.&quot; Journal of Biological Chemistry 277.52(2002):50973-9.</P><P>Fygenson, D. K., E. Braun, and A. Libchaber. &quot;Phase diagram of microtubules.&quot; Phys Rev E Stat Phys Plasmas Fluids Relat Interdiscip Topics 50.2(1994):1579-1588.</P><P>Invitrogen. &quot;pYD1 Yeast Display Vector Kit, Version D.&quot;(2002):1-16.</P><P>Klein, A, et al. &quot;A polymerizable GFP variant.&quot; Protein Engineering Design &amp; Selection Peds 25.4(2012):153-7.</P><P>Klein, Á, et al. &quot;Xylan-Degrading Catalytic Flagellar Nanorods.&quot; Molecular Biotechnology 57.9(2015):814-819.</P><P>Kuwajima, G. &quot;Construction of a minimum-size functional flagellin of Escherichia coli.&quot; Journal of Bacteriology 170.7(1988):3305-9.</P><P>Linse, K, and E. M. Mandelkow. &quot;The GTP-binding peptide of beta-tubulin. Localization by direct photoaffinity labeling and comparison with nucleotide-binding proteins.&quot; Journal of Biological Chemistry 263.29(1988):15205-10.</P><P>Liu, Z., et al. &quot;Recent advances in yeast cell-surface display technologies for waste biorefineries.&quot; Bioresource Technology 215(2016):324.</P><P>Mitra A, Sept D. Taxol<SUP>®</SUP> allosterically alters the dynamics of the tubulin dimer and increases the flexibility of microtubules[J]. Biophysical Journal, 2008, 95(7):3252.</P><P>Shibasaki, S, H. Maeda, and M. Ueda. &quot;Molecular display technology using yeast--arming technology.&quot; Analytical Sciences the International Journal of the Japan Society for Analytical Chemistry 25.1(2009):41.</P><P>Tian, G., et al. &quot;Tubulin folding cofactors as GTPase-activating proteins. GTP hydrolysis and the assembly of the alpha/beta-tubulin heterodimer.&quot; Journal of Biological Chemistry 274.34(1999):24054.</P><P>Wakabayashi, K, H. Hotani, and S. Asakura. &quot;Polymerization of Salmonella flagellin in the presence of high concentrations of salt.&quot; BBA - Protein Structure 175.1(1969):195-203.</P></DIV><DIV style="background-color:#F7EDB7;text-align:center;"><P style="text-align: center;color: #e5b051;padding-top: 5px;margin-top:75px;">Copyright © 2017 <A href="https://2017.igem.org/Team:BNU-China" style="color: #9E2B20;text-decoration: none;opacity:0.8;">BNU-China</A> All rights reserved.</P><P style="text-align: center;color: #e5b051;margin-bottom:-20px;">If you like this page, you can contact us: <SPAN style="color:#9E2B20;text-decoration: none;opacity:0.8;">bnu_igem@163.com</SPAN></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>