Team:TU Dresden/Part Collection
Project
Peptidosomes
β-Lactam Biosensor
Signal Peptide Toolbox
Evaluation Vector
Secretion
Communication
Results
Demonstration
Parts
Human Practices
Outreach
Integrated
Collaborations
iGEM goes green
Green Teams
Resources
Our Achievements
Our Experience
About Us
Team
Attributions
Financing & Sponsors
Lab Work
Experiments & Protocols
Safety
Judging
A handful of stars.
Best Part Collection
Part Collection
Signal Peptides
AmyE
AspB
BglS
Bpr
CccA
CitH
Csn
DacB
DacF
DltD
Epr
FliL
FliZ
GlpQ
LipA
LytB
LytC
LytD
LytR
Mdr
Mpr
NprE
PbpX
Pel
PelB
PenP
PhoA
PhoB
PhrA
PhrC
PhrF
PhrG
PhrK
RpmG
SacB
SacC
SleB
SpoIID
SpoIIP
SpoIIQ
SpoIIR
TyrA
Vpr
WapA
YbbC
YbbE
YbbR
YbdG
YbdN
YbfO
YbxI
YdbK
YdhT
YdjM
YdjN
YfhK
YfjS
YfkD
YfkN
YhcR
YhdC
YhfM
YhjA
YjcN
YjdB
YjfA
YjiA
YkoJ
YkvV
YkwD
YlaE
YlbL
YlqB
YlxF
Evaluation Vector
The collection of SPs was combined with the Evaluation Vector .
Short Description
The biotechnological production of extracellular enzymes is a multi-billion dollar market. This process requires and is often limited by efficient protein secretion. Due to its impressive secretory capacity and ease of genetic manipulation, the Gram-positive bacterium Bacillus subtilis is at the forefront of this process. Its Sec pathway orchestrates secretion via Signal Peptides (SP), which are located at the N-terminus of secreted proteins. Their efficiency is not determined by an optimal consensus sequence but instead is rather target protein-specific, necessitating individual screens for each new protein. Therefore, we established a parts collection to evaluate all combinational possibilities of each SP with a protein of interest: The Signal Peptide Toolbox contains 74 distinct Sec-dependent SPs for Bacillus subtilis that were thoroughly evaluated for functionality. This toolbox will enable future iGEM teams to boost the secretion of their preferred protein by significantly enhancing the optimization procedure.
Background
Over the course of the last decades the quality, amount and spectrum of heterologous (and recombinant) proteins has drastically increased and therefore the need for techniques to easily express and purify these proteins has emerged. We find such proteins as ingredients of detergents (proteases), medical treatments (insulin) or food and beverage products (amylases). Simply put, heterologous proteins are ubiquitously present. [1]
In order to tackle this demand we chose to apply the genetic tools of the model organism Bacillus subtilis. It is already one of the most frequently used hosts for overproduction of proteins throughout academia and industry because of its tremendous capacity to secret proteins, which can be exploited to increase the overall yield.
B. subtilis has four different secretion pathways, however the majority of proteins are being secreted via the general Sec pathway (Figure 1). This pathway has been identified in playing a crucial role in protein secretion as a common element among all domains of life [2] . In the Sec pathway, the secretion of proteins into the surrounding supernatant is orchestrated by signal peptides (SP). These SPs are composed of approximately 60 to 180 nucleotides and they are located upstream of the protein to be secreted. Intracellularly, the SP is translationally fused to the specific protein but cut off during the membrane translocation process releasing the protein into the supernatant without the signal peptide attached to it. [3]
Figure 1: The Sec pathway of B. subtilis. 1 The protein (blue) and its N-terminally fused SP (red) are ribosomally synthesized (purple). 2 A Signal Recognition Particle (orange) transports the protein to the membrane. 3 At the membrane, the translocation complex (light and dark green) takes over the protein. 4 During the translocation process, the protein is hold in the translocation complex but the SP is cut off by a peptidase (grey). 5 Afterwards, the protein is released into the supernatant where it reaches its native fold and the SP is degraded.
Currently, approximately 170 SPs belonging to the Sec pathway of B. subtilis have been annotated but unfortunately, no correlations on sequence levels have been identified that link efficient protein secretion with a distinct SP. [4]
As part of our project EncaBcillus, we aimed at creating a platform for heterologous protein overexpression combined with their efficient secretion without releasing any cells into the environment. Hereby, we encapsulated our model organism B. subtilis in the Peptidosomes which serve as a new innovative method to keep the producing bacteria separated from the desired compounds. (For more details see Peptidosomes .)
To address the vision of creating this new platform, we first wanted to establish a fast and easy screening procedure to evaluate all combinational possibilities of each SP with a protein of interest (POI) – The Signal Peptide Toolbox.
As SPs of B. subtilis have been successfully adapted to GRAM-positive [5] and GRAM-negative bacteria [6] , the Signal Peptide Toolbox, as an organism-independent genetic measurement tool, can be applied to any bacterial host. Every future iGEM team will be able to use this fully partsregistry availabe tool to increase their protein secretion.
Design
Bacillus subtilis contains approximately 170 Sec Signal Peptides (SPs) that re-direct proteins for secretion via the Sec pathway. For our toolbox we were able to clone 74 Sec SPs (Table 1). Each SP was amplified from genomic DNA of B. subtilis wild type with the primers found in the primer collection table at the end of the Design section. After amplification, each SP was digested using the restriction enzymes EcoRI and PstI, stored into the pSB1C3 backbone and submitted to the partsregistry.
Table 1: Submitted Sec SPs of B. subtilis.
AmyE
AspB
BglS
Bpr
CccA
CitH
Csn
DacB
DacF
DltD
Epr
FliL
FliZ
GlpQ
LipA
LytB
LytC
LytD
LytR
Mdr
Mpr
NprE
PbpX
Pel
PelB
PenP
PhoA
PhoB
PhrA
PhrC
PhrF
PhrG
PhrK
RpmG
SacB
SacC
SleB
SpoIID
SpoIIP
SpoIIQ
SpoIIR
TyrA
Vpr
WapA
YbbC
YbbE
YbbR
YbdG
YbdN
YbfO
YbxI
YdbK
YdhT
YdjM
YdjN
YfhK
YfjS
YfkD
YfkN
YhcR
YhdC
YhfM
YhjA
YjcN
YjdB
YjfA
YjiA
YkoJ
YkvV
YkwD
YlaE
YlbL
YlqB
YlxF
This powerful collection of SPs can now be combined with our Evaluation Vector and any protein of interest (POI) for a shotgun cloning approach to identify the best combination of SP and POI.
Though having access to these SPs via the registry, does not solve the problem of having to create one clone for each single SP and POI. This also means, sufficient amounts of each SP are necessary tu guarantee efficient cloning. Thus, we followed up on the idea of multi-template PCR. Since all SPs are stored in the pSB1C3 vector and flanked by the same pre- and suffix sequences, we should be able to amplify all SPs via PCR using the BioBrick pre- and suffix as primers. To cope with this issue of having 74 SPs, we decided to break down the 74 SP into Signal Peptide Mixes (SPM), each consisting of maximal 20 SPs (Table 2). We have carefully evaluated the maximal number of SPs within each mix to increase the robustness of the PCR (see the Results section for more details). Followed by the amplification of all SPM subsets, they can be easily purified and applied in a shotgun ligation approach with your protein of interest and the EV.
We demonstrate the applicability of this approach with three different proteins: the α-Amylase of B. subtilis, sfGFP and mCherry. For each protein we applied a protein dependent assay and tested supernatants of positive transformed B. subtilis clones. We correlated protein activities with the secretion efficiency. In a final step, we sequenced strains wich showed highest protein activities in order to identify the SPs. Thus, we were able to identify the most potent combination for a distinct SP with the given POI.
Detailed methods can be found in our protocol collection section . All primers used can be found in our primer collection table down below.
Primer collection table
Results
Amplification of the Signal Peptide Mixes via optimized multi-template PCR
So far, no direct correlation between the perfect combination of signal peptide and downstream sequence to gain optimal secretion levels is known. Thus, the problem of having to create one clone per combination of SP and protein of interest remains. Therefore, we created the so-called Signal Peptide Mixes (SPMs), a set of libraries with each containing equal concentrations of up to twenty distinct SPs which can be easily enriched via multi-template PCR. The amplified SPs can then be combined with our Signal Peptide Evaluation Vector (SP-EV) and the gene of interest (For more details see the protocol in the end of the Results section).
Figure 3: PCR results from amplifying a SPM containing twenty SPs. All twenty distinct SPs could be amplified in individual reactions from the same template.  Primers used were specific for each SP. ZR 50Bp DNA ladder from Zymo Research was used.
Figure 2: PCR evaluation to identify the maximum number of SP in a SPM. A PCR of SPMs consisting of 53 distinct SPs, shown in triplicates. PCR artifacts are visible at 250 Bp. B SPMs consisting only of 20 distinct SPs. We varied primer concentrations (µM) to enhance the PCR reaction and adjusted the template amounts (v-%). Set 1 = 1.5µM / 3%, Set 2 = 1µM / 3%, Set 3 = 1.5µM / 2%, Set 4 = 1µM / 2%. ZR 50Bp DNA ladder from Zymo Research was used.
In a first approach, we evaluated the multi-template PCR by varying the number of different SPs in one mix. Our aim was to amplify all SPs equally for the downstream cloning procedures. To test this, a SPM subset containing 53 SPs, was amplified using the RFC10 prefix and suffix as primers, we expected a band at around 100-200 Bp (size range of the SPs). Unfortunately, we also observed a second dominant band (at about 250 Bp) (Figure 2, A), which most likely derives from PCR artifacts. Thus, we concluded that a SPM consisting of 53 SPs was not suitable for our purposes.
Form this we decided to reduce the number of different SPs to a maximum of 20 and also to increase the primer concentrations in the PCR reaction. These improvements lead to a specific amplification of our SPs (Figure 2, B). To double check, if all 20 SPs were indeed amplified, we conducted a second PCR using the first PCR as template with specific primers for each SP in the corresponding SPM. We could show that all 20 distinct SPs of the SPM subset were amplified during the first PCR (Figure 3). From these observations, we decided to split up the 74 SPs into subset-mixes of each containing up to 20 SPs max.
All 74 SPs which we provide were therefore aliquoted to 0.5 ng/μL and assigned to one (a, b, c or d) SPM subsets. The table below gives an overview.
Table 2: SPM subsets a-d of all submitted Sec-dependent SPs.
SPM subset a
AmyE
AspB
BglS
Bpr
CccA
CitH
Csn
DacB
DacF
DltD
Epr
FliL
FliZ
GlpQ
LipA
LytB
LytC
LytD
SPM subset b
LytR
Mdr
Mpr
NprE
PbpX
Pel
PelB
PenP
PhoA
PhoB
PhrA
PhrC
PhrF
PhrG
PhrK
RpmG
SPM subset c
SacB
SacC
SleB
SpoIID
SpoIIP
SpoIIQ
SpoIIR
TyrA
Vpr
WapA
YbbC
YbbE
YbbR
YbdG
YbdN
YbfO
YbxI
YdbK
YdhT
YdjM
SPM subset d
YdjN
YfhK
YfjS
YfkD
YfkN
YhcR
YhdC
YhfM
YhjA
YjcN
YjdB
YjfA
YjiA
YkoJ
YkvV
YkwD
YlaE
YlbL
YlqB
YlxF
Cloning Signal Peptides with the Signal Peptide-Evaluation Vector
Following the evaluation of the multi-template PCR amplification of the SPs, we established a Standard Operating Procedure (SOP) protocol for cloning the SPs using the Signal Peptide-Evaluation Vector (SP-EV). This SOP describes the high-throughput approach of screening SPs with a POI. While our project is based in B. subtilis and we use E. coli the cloning host, we believe this SOP can be applied to any organism which is able to perform secretion via the Sec pathway.
Figure 4: Animated cloning stages using our EV. The promoter can be exchanged via EcoRI and BsaI (=results in XbaI overhang), to insert Signal Peptides use XbaI and AgeI and use NgoMIV and PstI to exchange of the gene of interest.
We provide the EV with a xylose inducible promoter (PxylA), which is well characterised for the use in B. subtilis. All the following experiments were performed using this promoter to drive our expression. As a first approach, we wanted to investigate if we could boost the secretion of the native amyE in B. subtilis. We amplified amyE (without its native signal peptide) using the primers iG17P180 and iG17P062, restriction enzymes interfering with the RFC25 standard were removed via mutagenesis PCR using iG17P057 - iG17P062. After sequencing, we stored the biobricked RFC25 compatible amyE version in the pSB1C3 backbone and submitted it to the parts registry ( BBa_K2273103 ). We then sub-cloned this part into our EV, via NgoMIV and PstI, replacing the lacZα. Next, we amplified all SPs from each SPM subset using the RFC10 (pre- and suffix) sequences as primers (TM4487 and iG17P039). After the amplification, each subset was digested using XbaI and AgeI. Now, we had everything ready to perform our shotgun ligation approach: we digested the EV (containing the BioBrick amyE) with BsaI and AgeI and ligated equal molar amounts with each SPM subset (a, b, c and d). The ligation mixes were directly transformed into B. subtilis .
The detailed SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox can be found down below at the end of the Results section "High throughput screening procedure for B. subtilis". The Gif (Figure 4) above summarizes graphically the steps neccessary to set up your individual SP-EV.
High throughput screening procedure for B. subtilis
After various adjustments to improve the applicability of the Signal Peptide Toolbox, we developed a high throughput screening procedure tailored to fit our model organism B. subtilis and proceeded to identify the most potent SPs for highest secretion of B. subtilis' α-Amylase.
Since we wanted to evaluate amylase secretion efficiency, we performed our transformation into a starch degradation-deficient B. subtilis strain (TMB3547). This strain contains a disruption of the amyE gene, due to the insertion of Pveg-lacZ. Fortunately, the strain still contains the necessary flanking regions for homologous recombination of the pBS1C vector. Thus, positive integration of the pBS1C-SPM-amyE construct lead to white colonies, when plated on X-Gal containing agar plates. (Figure 5, A)
We obtained colonies from each transformation (SPM a, b, c or d withamyE in the EV) and transferred colonies to: a starch containing screening agar plate (to check for vector integration into the B. subtilis genome) and a second backup agar plate which was spiked with chloramphenicol (to maintain each colony). In our setup, we included a negative control (TMB3547, non starch degrading) and a positive control (W168, native amyE and thereby native amylase secretion).
Figure 5: Handling of B. subtilis after transformation shown on agar plates. A The transformation agar plate (chloramphenicol and X-Gal). No blue colonies indicate a high transformation efficiency. B The screening agar plate. Zones of degradation indicate successful integration events. The negative control (TMB3547) has been marked with a black box, the positive control (W168 wild type) with a white box respectively. C The backup agar plate containing chloramphenicol. Neither the negative control (black box), nor the positive control (white box).
To verify positive integration of our EV-SPM-amyE constructs, we poured Lugol’s Iodine solution on the screening plates, which where spiked with starch. Normally, the integration of the EV into the amyE locus of B. subtilis, results in a disruption of the native gene leading to a loss of the enzymatic activity. Therefore, successfully transformed clones would not be able to degrade starch visualisable by no brightened zone of degradation (Figure 5B, black box). But as we performed our transformation into a starch degradation-deficient B. subtilis strain and used amylase as POI, successfully transformed colonies were again able to degrade starch. Thereby, a brightened zone of degradation on the screening agar plate indicated promising colonies (Figure 5, B e.g. top row). The position of successfully transformed colonies was then marked on the backup agar plates (Figure 5, C). Surprisingly, some chloramphenicol resistance clones did not show any zone of starch degradation (compare clone 4 on Figure 5, C and check with the same position on Figure 5, B). We believe that in these cases, only single homologous recombination events occurred.
Having now identified clones with positive integrated EV-SPM-amyE constructs, we proceeded with testing their supernatants in an amylase enzyme assay. [8] We therefore, incubated approximately 30 clones of each transformation (with the four different SPM subset) in a 96 well plate using 2xYT medium . We also included the negative control (TMB3547) and the wild type. We inoculated the 96 well plates and incubated the B. subtilis cultures for 8 hours at 37°C with 220 rpm using the plate reader.
Figure 6: Screening of signal peptides in front of amyE. Results of the plate reader based starch hydrolysis assay to identify the most potent combinations of SPs and B. subtilis’ α-Amylase of each SPM subset. The shown data was normalized over OD600. Data shown in A, B, C and D represent clones derived from SPM subsets a, b, c and d.
Figure 7: Sequenced signal peptides in front of amyE. Fold change in secretion efficiency (amylase activity) over wild type. Depicted candidates were identified by sequencing.
After the first 8 hours of incubation, we reinoculated the cells in fresh 2xYT medium supplemented with a final concentration of 1% xylose to induce the expression of our constructs driven by PxylA. We incubated the B. subtilis cultures for 16 hours at 37°C with 220 rpm in the plate reader.
Before separating the supernatant from the cells via centrifuging the 96 well plates, we measured the OD600 to normalize our data in the end (assuming a direct correlation between bacterial density and secreted protein). Using the harvested supernatants, we applied a microplate reader based starch hydrolysis assay [8] and normalized the generated data with the OD600 values to identify the most potent combinations of SPs and B. subtilis’ α-Amylase of each SPM subset (Figure 6).
Finally, we picked a range of promising colonies and amplified the SP containing region via standard PCR using the primers TM4487 and iG17P058. The resulting fragments were sequenced, thus we could identify the most potent SPs for α-Amylase (Figure 7).
We were very much excited to identify several SP which lead to a almost 1.5 fold higher amylase activity when compared to the wilde type! Interestingly we also identified the amyE SP as more potent than the wild type. We explain this, by the way we cloned our construct: due to the RFC25 we by default added two amino acids (AA) (from the scar site derived from the fusion of NgoMIV and AgeI) between the SP and amyE. Apparently, by having this two AA linker, the activity of Amylase could be enhanced.
We also identified SP which lead to less Amylase secretion. Overall, we were able to demonstrate a fully functional high-throughput approach to identify most potent partners between Sec dependant SP and a POI.
We also went one step further and proved the applicability of the Signal Peptide Toolbox by evaluating two more proteins: sfGFP and mCherry, according to the procedure described above.
The detailed SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox can be found down below. The protocol selection explains the use of the Evaluation Vector , the Signal Peptide-Evaluation Vector and the high throughput screening procedure for B. subtilis.
SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox
Identification of the most potent Signal Peptides for sfGFP and mCherry
The screening procedure for identifying the best SPs for highest secretion of sfGFP and mCherry were conducted as stated in the detailed SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox above.
For sfGFP, we applied a microplate reader based fluorescence assay where we performed an endpoint measurement at wavelengths set to 480 nm for excitation and 510 nm emission and normalized the generated data via the clones' OD600 values to identify the most potent combinations of SPs and sfGFP. For mCherry, we performed a second microplate reader based fluorescence assay, an endpoint measurement at wavelengths set to 585 nm for excitation and 615 nm for emission. The data was normalized over the clones' OD600 values. In a final step, we identified the most potent SP‑sfGFP and SP‑mCherry combinations via sequencing (Figure 8, 9).
Figure 9: Sequenced signal peptides in front of mCherry. Fold change in secretion efficiency (fluorescence) over wild type. Depicted candidates were identified by sequencing.
Figure 8: Sequenced signal peptides in front of sfGFP. Fold change in secretion efficiency (fluorescence) over wild type. Depicted candidates were identified by sequencing.
Conclusion
Our team developed and proved the applicability of a powerful toolbox to quickly screen via a high throughput procedure for improved secretion of proteins in bacteria - the Signal Peptide Toolbox. We are very much sure that the vision to facilitate Peptidosomes as protein production platform can be achieved. The promising combination of increased protein secretion and physical separation of production host and end-product has endless possible applications.
References
[1]
van Dijl J. M. and Hecker M. (2013) Bacillus subtilis: from soil bacterium to super-secreting cell factory. Mircobial Cell Factories, 3 (1-6).
[2]
Papanikou E., Karamanou S. and Economou A. (2007) Bacterial protein secretion through the translocase nanomachine. Nature Reviews Microbiology, 11 (839-851).
[3]
Fu L. L., Xu Z. R., Li W. F., Shuai J. B., Lu P. and Hu C. X. (2006) Protein secretion pathways in Bacillus subtilis: implication for optimization of heterologous protein secretion. Biotechnology advances 1 (1-12).
[4]
Brockmeier U., Caspers M., Freudl R., Jockwer A., Noll T. and Eggert T. (2013) Systematic screening of all signal peptides from Bacillus subtilis: a powerful strategy in optimizing heterologous protein secretion in Gram-positive bacteria. Journal of molecular biology, 3 (393-402).
[5]
Hemmerich J., Rohe P., Kleine B., Jurischka S., Wiechert W., Freudl R. and Oldiges M. (2016) Use of a Sec signal peptide library from Bacillus subtilis for the optimization of cutinase secretion in Corynebacterium glutamicum. Mircobial Cell Factories, 208.
[6]
Pechsrichuang P., Songsiriritthigul C., Haltrich D., Roytrakul S., Namvijtr P., Bonaparte N., Yamabhai M. (2016) OmpA signal peptide leads to heterogenous secretion of B. subtilis chitosanase enzyme from E. coli expression system. Springerplus, 1200.
[7]
Radeck, J., Kraft, K., Bartels, J., Cikovic, T., Dürr, F., Emenegger, J., Kelterborn, S., Sauer, C., Fritz, G., Gebhard, S., and Mascher, T. (2013) The Bacillus BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with Bacillus subtilis. J Biol Eng 7, 29.
[8]
Xiao Z., Storms R. and Tsang A. (2007) A quantitative starch-iodine method for measuring alpha-amylase and glucoamylase activities. Analytical Biochemistry, 1 (146-148).
Beam me up!
iGEM-Team TU Dresden
Institut für Mikrobiologie, AG Thorsten Mascher
Zellescher Weg 20b, 01217 Dresden
Germany
@igem.tud2017
@igem_tud_2017
igem2017@mailbox.tu-dresden.de
