<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Project_Communication skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Project/Communication</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2017.igem.org/Team:TU_Dresden/Description" class="nav-text dropbtn">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes" class="nav-text">Peptidosomes</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Biosensor" class="nav-text">β-Lactam Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Measurement" class="nav-text">Signal Peptide Toolbox</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part" class="nav-text">Evaluation Vector</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion" class="nav-text">Secretion</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Communication" class="nav-text">Communication</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU_Dresden/Results" class="nav-text dropbtn">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Demonstrate" class="nav-text">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Parts" class="nav-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Human_Practices" class="nav-text dropbtn">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Silver" class="nav-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Integrated</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Collaborations" class="nav-text">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green" class="nav-text dropbtn">iGEM goes green</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams" class="nav-text">Green Teams</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Resources" class="nav-text">Resources</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Our Achievements</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Experience" class="nav-text">Our Experience</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/About_Us" class="nav-text dropbtn">About Us</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Team" class="nav-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Attributions" class="nav-text">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Sponsors" class="nav-text">Financing &amp; Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Lab_Work" class="nav-text dropbtn">Lab Work</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Experiments" class="nav-text">Experiments &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Safety" class="nav-text">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Judging" class="nav-text dropbtn">Judging</A></LI></DIV></NAV><DIV id="pagebanner" style="background-image: url(https://static.igem.org/mediawiki/2017/3/3e/T--TU_Dresden--planet--communication.png);"><DIV><DIV id="bannerquote">Contacting Extra-Peptidosomal Life</DIV><DIV id="projecttitle">Communication</DIV><DIV class="contentbox at-a-glance"><H1 class="box-heading">At a Glance</H1><FIGURE><H4>Motivation:</H4><P>Demonstrate that two different strains of encapsulated bacteria could exchange soluble compounds across the Peptidosome barrier.</P><H4>Approach:</H4><P>Develop a sender-receiver strain pair in <I>Bacillus subtilis</I> that communicates via peptide-dependent quorum sensing.</P><H4>Achievements:</H4><P>(I) A functional <A href="#sestrest" class="hashlink">sender-receiver</A>  pair was established, based on the ComX pheromone and the ComP-ComA-P<SUB><I>srfA</I></SUB> signaling system. (II) The sender could activate the receiver in <A href="#coculture" class="hashlink">co-culturing</A> experiments. (III) The same behavior was observed in <A href="#peptidosomes" class="hashlink">Peptidosome</A> cultivations. (IV) 6 novel basic <A href="#BioBricks" class="hashlink">BioBrick parts</A> were generated and fully evaluated for functionality.</P></FIGURE></DIV></DIV><MAIN><DIV class="contentbox"><H1 id="communication" class="box-heading">Short Description</H1><P>
By using Peptidosomes we introduce a new powerful platform for co-culturing. This technique physically separates bacterial populations without limiting their ability to communicate with each other via signaling molecules. This part of EncaBcillus is focused on proving the concept of communication between encapsulated bacteria by making use of the native regulatory system for competence development in <I>Bacillus subtilis</I> which is based on quorum sensing.
</P></DIV><DIV class="contentbox"><H1 id="background" class="box-heading">Background</H1><P> 
In order to solve complex tasks, it is reasonable to break down a problem into small subunits that can be addressed by specialists individually. This concept is widely implemented in modern economy but can also be applied to synthetic 
biology <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24829281">[1]</A>. In nature, such strategies have been evolved over millions of years. A good example is the formation of biofilms by the gram-positive bacterium <I>Bacillus subtilis</I>. Here, cell-cell interactions lead to cellular differentiation processes, which results in an effective division of labour <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4188541/">[2]</A>.
However, development of synthetic consortia consisting of several different bacterial strains, with each executing distinct tasks remains challenging <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18675483">[3]</A>. 
</P><P>
By encapsulating subpopulations within molecular cages, called <A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes">Peptidosomes</A>, we want to physically separate co-working bacteria while still assuring their cooperativity to fulfil one major task. In this context communication between the bacteria is indispensable, thus we had to make sure that the Peptidosome membrane is permeable for small molecules and that the bacteria are still able to interact with each other. In this part of EncaBcillus we will show that the encapsulation of one population does not limit its ability to communicate with cells in the surrounding medium.</P><H3>The regulatory system for competence development in <I><B>B. subtilis</B></I></H3><FIGURE><FIGURE class="makeresponsive floatright" style="width: 60%;"><FIGCAPTION><B>Figure 1: The regulatory system for competence development in <I><B>Bacillus subtilis</B></I>.</B> The development of genetic competence is stimulated by quorum sensing due to the secretion of the ComX pheromone and its accumulation in the surrounding medium. (Modified from: <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/22511326">Jung <I>et al.</I></A>)</FIGCAPTION></FIGURE><P>
The well-studied regulatory system for competence development in <I>B. subtilis</I> provided a genetic set-up based on quorum sensing that we used as a proof of principle (Figure 1). <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/12576575">[4]</A><I>B. subtilis</I> constantly secretes the ComX pheromone, a 9- to 10-amino acid oligopeptide, as a signaling molecule <B>(a)</B>. By rising cell-density, the ComX-concentration in the surrounding medium increases until it reaches a threshold and activates ComP, a membrane-spanning protein kinase <B>(b)</B>. 
The kinase reacts to the accumulation of ComX by phosphorylating the response regulator ComA <B>(c)</B> which then works as a transcription factor by binding to several promoters and enhancing their activity <B>(d)</B><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26582911">[5</A>,<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18585392">6]</A>.</P><P>
The most important promoter regulated by ComA and involved in competence development is the promoter of the <I>srfA</I> operon. This operon contains not only genes for the production of the antibiotic surfactin <B>(e)</B> but also for ComS <B>(f)</B>, another small peptide that prevents the degradation of the autoregulated transcription factor ComK <B>(g)</B>.
ComK activates expression of more than 100 genes, including <I>comG</I>, that is part of the transformation machinery <B>(h)</B><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16091051">[7,</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/14762007">8]</A>. Consequently, <I>B. subtilis</I> can take up DNA from the environment either as direct nutrient source or incorporate the DNA via homologues recombination into its own genome.</P></FIGURE><H3>Communication between encapsulated bacteria and bacteria in the surroundings </H3><FIGURE><FIGURE class="makeresponsive floatright" style="width:60%;"><FIGCAPTION><B>Figure 2: Illustration of co-cultured sender (ReSt) and receiver strain (SeSt) using Peptidosomes.</B> The SeSt in the surrounding medium secretes the ComX quorum sensing pheromone that diffuses through the membrane into the Peptidosome and stimulates the encapsulated, comX-deficient ReSt.  As a result, the ReSt produces luminescence driven by a ComX-dependant promoter upstream the lux operon.</FIGCAPTION></FIGURE><P>
Based on this genetic background, we wanted to set up a simple experiment that shows communication between two <I>B. subtilis</I> strains. The basic idea is comparable to a radio broadcast: you need a transmitter that sends out the radio waves and a receiver that receives the waves and converts them into an acoustical signal. Transferring this idea over to our project means that we need a sender strain (SeSt) that secretes the ComX pheromone, and a receiver strain (ReSt) that reacts to the ComX-stimulation with an easy-detectable output (Figure 2).</P><P>
In our setup, we fused target promoters of the competence machinery of <I>B. subtilis</I> to the <I>lux</I> operon and measured luminescence output. Additionally, the ReSt needed to be <I>comX</I>-deficient to prevent autoinduction. We encapsulated the ReSt within Peptidosomes and incubated it together with the SeSt (either wild type or a ComX-overproducing strain) in the surrounding medium. Thus, we expected ComX that is produced outside the Peptidosme to diffuse through the membrane and stimulate the ReSt. As a result, we should obtain a glowing Peptidosome mediated by the <I>lux</I> operon, which is only present in the ReSt.</P></FIGURE></DIV><DIV id="design" class="contentbox"><H1 class="box-heading">Design</H1><P>After extensive research, an iGEM project starts with designing genetic constructs and planning experiments. It takes some time until the strains are finished and the first assays are running. Anyway, it is no secret that the most important results come in the last few weeks before the Giant Jamboree. Figure 3 gives an overview of this subproject in chronological order.</P><FIGURE class="makeresponsive" style="padding-left: 25%; padding-right: 25%;"><FIGCAPTION><B>Figure 3: Workflow diagram and timeline of the subproject “Communication”. </B> Blue arrows indicate the way towards the sender strain (SeSt) and pink arrows towards the receiver strain (ReSt). For Promoter evaluation, we used plate reader assays (PR-Assay). Associated protocols are gathered at <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Experiments &amp; Protocols</A>. Collected data is presented in the results section.</FIGCAPTION></FIGURE><H3>Genetic design and cloning</H3><DIV style="display: flex; align-items: flex-start; flex-wrap: wrap;"><DIV class="makeresponsive" style="width: 50%;"><H4>Sender strain (SeSt)</H4><FIGURE><FIGURE class="makeresponsive floatleft" style="width:40%;"><FIGCAPTION><B>Figure 4: Inducible copy of <I><B>B. subtilis</B></I> ComX pheromone.</B></FIGCAPTION></FIGURE><P>As SeSt, we used <I>B. subtilis</I> W168 (WT) with an additional inducible copy of <I>comX</I> (Table 1) in order to increase the concentration of ComX in the medium independently from cell density. Overproduction of ComX was achieved due to a xylose-inducible promoter (P<SUB><I>xylA</I></SUB>, Figure 4). For that reason, we used the pBS2EP<SUB><I>xylA</I></SUB>, an integrative plasmid of the <I>Bacillus</I> BioBrick Box 2.0 (Popp <I>et al.</I> 2017, accepted). </P></FIGURE></DIV><DIV class="makeresponsive" style="width: 50%;"><H4>Receiver strain (ReSt)</H4><FIGURE><FIGURE class="makeresponsive floatleft" style="width:40%;"><FIGCAPTION><B>Figure 5: Target promoters of the competence machinery of <I><B>B. subtilis</B></I> (P<SUB><I><B>X</B></I></SUB> = P<SUB><I><B>srfA</B></I></SUB> or P<SUB><I><B>rapA</B></I></SUB> or P<SUB><I><B>rapF</B></I></SUB> or P<SUB><I><B>comG</B></I></SUB> or P<SUB><I><B>comK</B></I></SUB> mut) fused to the <I><B>lux</B></I> operon.</B></FIGCAPTION></FIGURE><P>To create a strain suitable as ReSt we screened various promoters that are part of the quorum response to ComX. We have chosen three promoters, P<SUB><I>srfA</I></SUB>, P<SUB><I>rapA</I></SUB> and P<SUB><I>rapF</I></SUB>, that are controlled by the response regulator ComA as well as the the ComK-dependent promoter P<SUB><I>comG</I></SUB> and the mutated promoter of the competence transcription factor ComK (P<SUB><I>comK</I></SUB> mut) itself (Table 1).
All promoters were cloned into the pBS3C<I>lux</I>, an integrative plasmid of the <I>Bacillus</I> BioBrick Box  containing the <I>luxABCDE</I> reporter to measure promoter activities (Figure 5) <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[9]</A>.</P></FIGURE></DIV></DIV><FIGURE class="jonathanstables" id="BioBricks"><FIGCAPTION><B>Table1: Basic BioBricks created for the subproject &quot;Communication&quot;.</B></FIGCAPTION><TABLE class="makeresponsive"><TBODY><TR><TD><B>Name</B></TD><TD><B>BioBrick</B></TD><TD><B>Remarks</B></TD></TR><TR><TD><I>comX</I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273003">BBa_K2273003</A></TD><TD>None</TD></TR><TR><TD>P<I><SUB>srfA</SUB></I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273000">BBa_K2273000</A></TD><TD>None</TD></TR><TR><TD>P<I><SUB>rapA</SUB></I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273001">BBa_K2273001</A></TD><TD>None</TD></TR><TR><TD>P<I><SUB>rapF</SUB></I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273002">BBa_K2273002</A></TD><TD>None</TD></TR><TR><TD>P<I><SUB>comG</SUB></I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273012">BBa_K2273012</A></TD><TD>None</TD></TR><TR><TD>P<I><SUB>comK</SUB></I> mut</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273013">BBa_K2273013</A></TD><TD>In order to remove a SpeI restriction site we had to exchange one nucleotide (nt 32, A to T).</TD></TR></TBODY></TABLE></FIGURE><P>All parts have been amplified via PCR from gDNA of <I>Bacillus subtilis </I> W168, verified by sequencing and cloned into the pSB1C3 for storage and submission to the iGEM registry. Cloning was done according to <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">standard protocols</A>. Plasmids have been multiplied with <I>Escherichia coli</I> DH10β.  All assays have been conducted with <I>B. subtilis</I> W168 carrying one or more of the earlier-described constructs.</P><H3>Experimental design</H3><P>
In order to investigate quorum sensing-dependent effects we determined the promoter activities of  P<SUB><I>srfA</I></SUB>, P<SUB><I>rapA</I></SUB>, P<SUB><I>rapF</I></SUB>, P<SUB><I>comG</I></SUB> and P<I><SUB>comK</SUB></I> mut in WT and in <I>comX</I>-deficient strains by monitoring the luminescence. In the knockout-strains <I>comX</I> has been replaced with either an erythromycin or a kanamycin resistance cassette (<I>comX::ery</I> or <I>comX::kanR</I>, for simplicity from now on referred to as Δ<I>comX</I>). </P><P>Furthermore, we examined the influence of different media on promoter activity using Luria-Bertani (LB) broth as full medium as well as two minimal media: MCSE and MNGE medium. The latter is commonly used for the <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">transformation</A> of <I>B. subtilis</I>. All plate reader assays (PR-Assays) have been performed with the CLARIOstar® microplate reader from BMG Labtech.
</P><P>For co-cultivation of ReSt and SeSt we used either ThinCerts™ - TC inserts (6 well, 0.4 µm pores, transparent) by Greiner or Peptidosomes. Peptidosomes have been prepared according to  <A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes#creation">standard procedures</A>.</P><DIV style="display: flex; align-items: flex-start; flex-wrap: wrap;"><DIV class="makeresponsive" style="width: 40%;"><P>For the co-culture assays with SeSt and ReSt using ThinCerts™ (Figure 6), separate day cultures were inoculated (1:1000) from overnight cultures and incubated at 37°C and 220 rpm until they reached an optical density at 600 nm (OD<SUB>600</SUB>) of 0.2 – 0.4. From this day cultures, fresh MNGE medium was inoculated to an OD<SUB>600</SUB> of 0.05 and distributed to the well plate according to the pipette scheme (Table 2) in a final volume of 2 mL for each insert and well respectively. After one hour of incubation (37°C, 200 rpm) SeSt were induced with 1% xylose (if necessary). Luminescence was detected directly after induction (t0) and once every following hour (t1 – t5) using a chemiluminescence imaging system.</P></DIV><DIV class="makeresponsive" style="width: 60%;"><FIGURE class="makeresponsive floatright" style="width: 100%;"><FIGCAPTION><B>Figure 6:</B> Schematic view of a well plate with ThinCerts™. These inserts can be used to co-culture distinct population and to study possible interactions. The membrane provides a physical barrier for cells whereas molecules are still able to diffuse. (Modified from: <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24829281">Goers <I>et al.</I></A>)</FIGCAPTION></FIGURE><FIGURE class="makeresponsive floatright" style="width: 100%"><FIGCAPTION><B>Table 2: Pipette scheme for co-culture assay using ThinCerts™.</B> The assay was performed in a 6-well plate. For co-culture of sender strain (SeSt) and receiver strain (ReSt), we used cell culture inserts (A, B, C). Sender Strains were induced with 1% xylose (+xyl) if necessary. For the negative controls (D, E, F) inserts have been omitted.</FIGCAPTION></FIGURE></DIV></DIV><P>
Co-culture assays using Peptidosomes were performed as well scan experiments according to <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">&quot;Plate reader well scan Peptidosome&quot;</A> protocol, with the following changes:</P><P>
Fresh MNGE medium was inocculated with SeSt from overnights in a final volume of 500 µL. Peptidosomes were inocculated with ReSt from overnight to an OD<SUB>600</SUB> of 10. After one hour of incubation strains were induced with 1% xylose (if necessary). Luminescence was detected directly after induction and once every following hour using the CLARIOstar® microplate reader from BMG Labtech. Plates were constantly incubated at 37°C under shaking conditions (100rpm). 
</P></DIV><DIV class="contentbox"><H1 id="screening" class="box-heading">Results</H1><H3>Promoter activity screening</H3><P>
Although quorum sensing induces various changes in gene expression, the engineering of the ReSt initially required an extensive screening of several promoters regarding their ComX-dependent activity (Figure 7).</P><FIGURE><FIGURE class="makeresponsive floatright" style="width:70%;"><FIGCAPTION><B>Figure 7:  Growth (A) and promoter activity of P<SUB><I><B>srfA</B></I></SUB>, P<SUB><I><B>rapA</B></I></SUB>, P<I><SUB><B>rapF</B></SUB></I>, P<I><SUB><B>comG</B></SUB></I> and P<I><SUB><B>comK</B></SUB></I> mut (B) in MNGE medium.</B> Wild type (WT, pink) and <I>comX</I>-knockout strains (Δ<I>comX</I>, dark blue) contain a promoter-<I>luxABCDE</I> fusion. </FIGCAPTION></FIGURE><P>Since the ComX pheromone itself does not act as a transcription factor we examined promoters that are either under the control of the response regulator ComA (P<SUB><I>srfA</I></SUB>, P<SUB><I>rapA</I></SUB>&amp; P<SUB><I>rapF</I></SUB>) or regulated by the competence transcription factor ComK (P<I><SUB>comK</SUB></I> and P<I><SUB>comG</SUB></I>). Surprisingly P<SUB><I>rapA</I></SUB> and P<SUB><I>rapF</I></SUB> seem to be constitutive as well as P<I><SUB>comK</SUB></I> mut that is regulated by ComK itself. The unexpected behaviour of P<I><SUB>comK</SUB></I> mut could be a consequence of the single nucleotide exchange that was necessary to remove a restriction site since small changes of a promoter sequence can influence the binding of regulatory elements. After an initial decrease of activity P<I><SUB>comG</SUB></I> exhibits a weak increase of activity during stationary phase. However, for our purpose the most promising promoter is P<SUB><I>srfA</I></SUB>. Although it possesses a high basal activity in the wild type (WT) it reveals a tenfold increase of expression during the transition to stationary phase. In the associated <I>comX</I>-knockout strain (Δ<I>comX</I>) the promoter activity only shows basal activity. This clearly demonstrates the ComX-dependent expression of the <I>srfA</I> operon.
</P></FIGURE><H3>Influence of different media on the activity of P<SUB><I><B>srfA</B></I></SUB></H3><FIGURE><FIGURE class="makeresponsive floatright" style="width:65%"><FIGCAPTION><B>Figure 8: Growth (A) and promoter activity of P<SUB><I><B>srfA</B></I></SUB> (B) in different media.</B> Wild type (WT, pink) and <I>comX</I>-knockout strains (Δ<I>comX</I>, dark blue) contain a P<SUB><I>srfA</I></SUB><I>luxABCDE</I> fusion. </FIGCAPTION></FIGURE><P>We continued by examining the influence of different media on the activity of P<SUB><I>srfA</I></SUB>. This step is important because competence development is not only dependent on quorum sensing but also on nutritional conditions. As expected MNGE, the transformation medium of <I>B. subtilis</I>, turned out to be the most convenient medium for our purposes (Figure 8). In rich medium (LB) we observed the lowest promoter activity for the knockout strain as well as the lowest increase of activity in wild type. However, in both minimal media (MCSE and MNGE), we could observe a higher increase of luminescence, although there is a temporary decline of activity after 3 hours of incubation in MCSE-medium. Therefore, we decided to continue using MNGE-medium for all following experiments.</P></FIGURE><H3 id="sestrest">Overproduction of ComX</H3><FIGURE><FIGURE class="makeresponsive floatright" style="width:50%"><FIGCAPTION><B>Figure 9: Growth (A) and promoter activity of P<SUB><I><B>srfA</B></I></SUB> (B) in dependence on xylose-induced ComX production.</B> Wild type (WT, pink and light blue) and <I>comX</I>-knockout strain (<I>ΔcomX</I>, black and dark blue) contain a P<SUB><I>srfA</I></SUB><I>luxABCDE</I> fusion. After 1 hour of incubation (dashed line) the cultures were induced with 1% xylose (light blue and black) or with distilled water (dH2O, pink and dark blue).
</FIGCAPTION></FIGURE><P>After screening of promoter activity and media-dependence of P<SUB><I>srfA</I></SUB> we examined the influence of ComX-overproduction via induction with xylose (Figure 9). In strains with an additional copy of comX (WT), an induction with xylose (+ xylose) did not lead to higher luminescence but to an earlier increase in comparison to the non-induced WT (+ dH2O). In knockout strains (∆<I>comX</I>), we could compensate the <I>comX</I>-deficiency via an induction of ComX-production. Surprisingly in those cultures the luminescence increased even earlier as in WT with an additional induction of the ComX-production. The increase of luminescence in the comX-deficient strain without xylose-induction (∆<I>comX</I> + dH2O) served as negative control. In conclusion, we could demonstrate an impact of ComX-overproduction.</P></FIGURE><H3 id="coculture">Co-culture of SeSt and ReSt using ThinCerts™</H3><FIGURE><FIGURE class="makeresponsive floatright" style="width: 60%;"><FIGCAPTION><B>Figure 10: Co-culture of sender strain (SeSt) and receiver strain (ReSt) using ThinCert™ cell culture inserts.</B> Luminescence of the ReSt (well) co-cultured with the SeSt (insert) induced with xylose <B>(A)</B>, luminescence of ReSt (well) co-cultured with the SeSt (insert) without xylose-induction <B>(B)</B> and luminescence of ReSt (insert) co-cultured with the SeSt (well) induced with xylose <B>(C)</B> was documented by a chemoluminescence imaging system. Cultures containing only SeSt <B>(D)</B>, ReSt <B>(E)</B> or MNGE <B>(F)</B> served as negative controls.</FIGCAPTION></FIGURE><P>Via plate reader assays, we have already proven the functionality of our genetic constructs combined in one single strain. So, the next step was to demonstrate that the system also works if the constructs are separated in two different strains. Therefore, we continued with co-cultures of SeSt and ReSt using cell culture inserts (ThinCerts<SUP>TM</SUP>). Thereby, we could finally show that the SeSt stimulates the ReSt, containing a P<SUB><I>srfA</I></SUB>-<I>luxABCDE</I> fusion, by using luminescence as output (Figure 10). In the control, without SeSt, the <I>comX</I>-deficient ReSt exhibited only a weak luminescence signal because of the promoter’s basal activity. Hence, our system, consisting of SeSt and ReSt, is qualified as a proof of principle for communication between two distinct strains.</P></FIGURE><H3 id="peptidosomes">Co-culture of SeSt and ReSt using Peptidosomes</H3><P>The last step to an overall success of this subproject was the demonstration of microbial communication between bacteria encapsulated in Peptidosomes and bacteria in the surrounding medium. Therefore, we repeated the co-culture assay as described before but instead of separating the distinct strains using cell culture inserts we encapsulated the ReSt within a Peptidosome while we cultivated the SeSt in the surrounding medium. Luminescence was detected via well scan to precisely detect even small changes as well as the spatial extent. We could show that there is only a rise of luminescence if you cultivate the encapsulated ReSt together with the SeSt in the surrounding medium.</P><FIGURE><FIGURE class="makeresponsive floatright" style="width:60%;"><FIGCAPTION><B>Figure 11: Co-culture of sender strain (SeSt) and receiver strain (ReSt) using Peptidosomes. A</B>, <B>B</B> and <B>C</B> illustrate the experimental set-up whereas <B>D</B>, <B>E</B> and <B>F</B> display the results of the well scan. There is only an increase of luminescence (≥ 40 RLU, pink), restricted to the Peptidosomes, if the encapsulated ReSt is cultivated within the SeSt <B>(A, D)</B>. Cultures containing either SeSt <B>(B, E)</B> or encapsulated ReSt <B>(C, F)</B> served as negative controls.
</FIGCAPTION></FIGURE><P>In Figure 11 A, B and C illustrate the experimental set-up whereas D, E and F display the results of the well scan analysis. While areas with luminescence less or equal the maximal luminescence measured for SeSt (≤ 40 RLU) are blue-colored, areas with luminescence above SeSt level (≥ 40 RLU) are pink-colored. Therefore, we can show that the increase of luminescence is restricted to the Peptidosome (Figure 11, E). The mean luminesescence determined for the encapsulated ReSt is 90 RLU. There are few areas with values that are negligibly above the threshold (41 to 62 RLU) causing pink-colored squares located outside the expected position of the Peptidosome. This is a consequence of the chosen threshold and provides no evidence for cells leaving the Peptidosome. </P></FIGURE><H2>Conclusion</H2><P class="survey-quote"><B>Our team has made substantial progress in the evaluation of Peptidosomes as a tool for co-cultivation and studies of microbial interactions. We could prove communication between encapsulated bacteria and bacteria in the surroundings by exchanging signaling molecules (=ComX). Further research about pore sizes of the Peptidosome membrane and diffusion rates needs to be done to tap the full potential of Peptidosomes as a powerful co-culture technique.</B></P></DIV><DIV class="contentbox"><H1 class="box-heading">References</H1><TABLE class="source"><TBODY><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24829281">[1]</A></TD><TD> Goers, L., Freemont, P. and Polizzi, K.M. (2014) Co-culture systems and technologies: taking synthetic biology to the next level.<I> J. R. Soc. Interface 11</I>, 20140065</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4188541/">[2]</A></TD><TD> Mielich‐Suss, B. and Lopez, D. (2015) Molecular mechanisms involved in <I>Bacillus subtilis</I> biofilm formation. <I> Environ. Microbiol. 17</I>, 555–565</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18675483">[3]</A></TD><TD> Brenner, K., You, L. and Arnold, F.H. (2008) Engineering microbial consortia: a new frontier in synthetic biology.<I> Trends Biotechnol. 26</I>, 483-489</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/12576575">[4]</A></TD><TD> Hamoen, L.W., Venema G. and Kuipers O.P. (2003) Controlling competence in <I>Bacillus subtilis</I>: shared use of regulators.<I> Microbiology. 149</I>, 9-17</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/26582911">[5]</A></TD><TD> Wolf, D., Rippa, V., Mobarec, J.C., Sauer, P., Adlung, L., Kolb, P. and Bischofs, I.B. (2016) The quorum-sensing regulator ComA from <I>Bacillus subtilis</I> activates transcription using topologically distinct DNA motifs.<I> Nucleic Acids Res. 44(5)</I>, 2160-72</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18585392">[6]</A></TD><TD>  Griffith, K.L. and Grossman, A.D. (2008) A degenerate tripartite DNA-binding site required for activation of ComA-dependent quorum response gene expression in <I>Bacillus subtilis. J Mol Biol. 381(2)</I>, 261-75</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16091051">[7]</A></TD><TD> Comella, N. and Grossman, A.D. (2005) Conservation of genes and processes controlled by the quorum response in bacteria: characterization of genes controlled by the quorum-sensing transcription factor ComA in <I>Bacillus subtilis. Mol Microbiol. 57(4)</I>,1159-74</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/14762007">[8]</A></TD><TD> Susanna, K.A., van der Werff, A.F., den Hengst, C.D., Calles, B., Salas, M., Venema, G., Hamoen, L.W. and Kuipers, O.P. (2004) Mechanism of transcription activation at the comG promoter by the competence transcription factor ComK of <I>Bacillus subtilis. J Bacteriol. 186(4)</I>, 1120-8</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[9]</A></TD><TD>Radeck, J., Kraft, K., Bartels, J., Cikovic, T., Dürr, F., Emenegger, J., Kelterborn, S., Sauer, C., Fritz, G., Gebhard, S., and Mascher, T. (2013) The <I>Bacillus</I> BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with <I>Bacillus subtilis. J Biol Eng 7</I>, 29.</TD></TR></TBODY></TABLE></DIV></MAIN></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end;">Beam me up!</A></DIV><DIV class="footerbar"><TABLE><TBODY><TR><TD><A href="https://www.google.de/maps/place/Zellescher+Weg+20,+01217+Dresden" target="_blank">iGEM-Team TU Dresden<SMALL>Institut für Mikrobiologie, AG Thorsten MascherZellescher Weg 20b, 01217 Dresden</SMALL>Germany</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><A href="https://www.facebook.com/igem.tud2017" target="_blank">@igem.tud2017</A></TD></TR><TR><TD><A href="https://www.instagram.com/igem_tud_2017" target="_blank">@igem_tud_2017</A></TD></TR><TR><TD><A href="mailto:igem2017@mailbox.tu-dresden.de">igem2017@mailbox.tu-dresden.de</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>