<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2017.igem.org/Team:TU_Dresden/Description" class="nav-text dropbtn">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes" class="nav-text">Peptidosomes</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Biosensor" class="nav-text">β-Lactam Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Measurement" class="nav-text">Signal Peptide Toolbox</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part" class="nav-text">Evaluation Vector</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion" class="nav-text">Secretion</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Communication" class="nav-text">Communication</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU_Dresden/Results" class="nav-text dropbtn">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Demonstrate" class="nav-text">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Parts" class="nav-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Human_Practices" class="nav-text dropbtn">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Silver" class="nav-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Integrated</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Collaborations" class="nav-text">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green" class="nav-text dropbtn">iGEM goes green</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams" class="nav-text">Green Teams</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Resources" class="nav-text">Resources</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Our Achievements</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Experience" class="nav-text">Our Experience</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/About_Us" class="nav-text dropbtn">About Us</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Team" class="nav-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Attributions" class="nav-text">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Sponsors" class="nav-text">Financing &amp; Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Lab_Work" class="nav-text dropbtn">Lab Work</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Experiments" class="nav-text">Experiments &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Safety" class="nav-text">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Judging" class="nav-text dropbtn">Judging</A></LI></DIV></NAV><DIV id="pagebanner" style="background-image: url(https://static.igem.org/mediawiki/2017/3/34/T--TU_Dresden--planet--vector.png);"><DIV><DIV id="bannerquote">BBa_K2273107 - The New Star</DIV><DIV id="projecttitle">Evaluation Vector</DIV><DIV class="contentbox at-a-glance"><H1 class="box-heading">At a Glance</H1><FIGURE><H4>Motivation:</H4><P>Develop a combinatorial platform for convenient screening of N- and C-terminal translational fusion proteins.</P><H4>Approach:</H4><P>Develop a vector that allows translationally fusing N-terminal tags (such as signal peptide-encoding sequences) with protein-encoding genes in a highly controlled manner.</P><H4>Achievements:</H4><P>(I) A suitable BioBrick-compatible <A href="#design" class="hashlink">vector</A> design was successfully implemented. (II) Functionality and efficiency of the evaluation vector was demonstrated by combining <A href="https://2017.igem.org/Team:TU_Dresden/Measurement#threeproteins" class="hashlink">3 different genes</A> with <A href="https://2017.igem.org/Team:TU_Dresden/Measurement">Signal Peptide Toolbox</A>. (II) 1 basic BioBrick (<A href="#protocol" class="hashlink">the vector!</A>) was generated and comprehensively evaluated.</P></FIGURE></DIV></DIV><MAIN><DIV class="contentbox"><H1 class="box-heading">Short Description</H1><P>Peptidosomes in combination with <I>Bacillus subtilis</I> offer a perfect platform for enhanced protein overproduction by the means of efficient protein secretion provided through <I>B. subtilis</I> and the easy purification due to the physical separation of bacteria and the end-product in the supernatant facilitated by the Peptidosomes. Naturally, <I>B. subtilis</I> is a strong secretion host and in order to take full advantage of this great potential it is necessary to evaluate all possible combinations of the <I>B. subtilis’ </I>  secretion signal peptides and the proteins of interest. Therefore, we developed the <B>Evaluation Vector (EV)</B> which is a powerful genetic tool containing a multiple cloning site (MCS) specifically designed to easily exchange translational fusions composed of the desired protein and a secretion signal peptide.</P></DIV><DIV class="contentbox"><H1 id="background" class="box-heading">Background</H1><P>Tool development and their proper evaluation are core aspects of Synthetic Biology. In our project EncaBcillus one main idea was to establish Peptidosomes with encapsulated bacteria as efficient protein overproduction platform. We took advantage of <I>B. subtilis’</I> ability to efficiently secrete proteins into its environment in order to increase overall yields and to simplify the purification of the desired proteins.Therefore, we developed a general expression <B>Evaluation Vector (EV)</B> with easily exchangeable units:  I) allowing the replacement of the promoter (which drives the system) and II) a multiple cloning site enabling to work with translationally fused composite parts. In our case, a typical composite part consists of a signal peptide (for secretion in <I>B. subtilis</I>) and a protein of interest.In summary, our EV was designed to fulfill the following distinct features:
</P><UL class="henristyle"><LI>Exchangeable promoter region</LI><LI>Insertion of basic or composite parts as expression units</LI><LI>Fulfilling the RFC10 and RFC25 BioBrick standard</LI><LI>Easy cloning and screening procedure in <I>Escherichia coli</I></LI></UL><P>As our project is based on the Gram-positive model organism <I>B. subtilis</I>, we decided to use a previously well-evaluated <I>B. subtilis</I> vector as source for our Evaluation Vector: the integrative vector pBS1C1. <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[1]</A> In brief, the vector has the following features for cloning in <I>E.coli</I>: an ori of replication and the <I>bla</I>  gene mediating resistance against ampicillin. The <I>B. subtilis</I> specific part of the vector contains the multiple cloning site (MCS) in RFC10 standard, a <I>cat</I> cassette providing resistance against chloramphenicol and flanking regions needed for integration into the <I>amyE</I> locus. After integration into  α-amylase, the resulting disruption of the native gene leads to a loss of this enzymatic activity, thereby making it a vector easy to screen for by performing a starch test for positive integration events. (For a detailed description of the original vector features please have a look at <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[1]</A> and our Design section of the EV.)
</P></DIV><DIV class="contentbox"><H1 id="design" class="box-heading">Design</H1><P>As described above, we chose to modify the backbone of pBS1C1 <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[1]</A> to create our new <B>Evaluation Vector (EV)</B>, by engineering the multiple cloning site (MCS) according to the scheme below (Figure 1).</P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 65%;"><FIGCAPTION><B>Figure 1: New layout of the multiple cloning site in our Evaluation Vector.</B> The crosses indicate restriction enzyme sites: E= EcoRI, N= NotI, X=XbaI, S= SpeI and P= PstI. Please note: cutting with BsaI will result in an XbaI overhang.</FIGCAPTION></FIGURE><P>At first, we removed a BsaI restriction enzyme site on the backbone of the vector by PCR based mutagenesis using primers TM3161 and TM3164 because it was interfering with our design. The confirmed BsaI free vector was then cut with EcoRI and XbaI to insert the xylose inducible promoter P<SUB><I>xylA</I></SUB><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[1]</A> which was prior amplified using the primers iG17P051 and iG17P052 followed by digestion with EcoRI and BsaI (resulting in an XbaI overhang) to maintain the BioBrick prefix in front of the promoter. Next, we had to create an entirely new multiple cloning site (MCS): We synthesized a new RFP based on the sequence of the RFP found in the pSB1C3 backbone. The expression of this RFPsyn2 is still driven by the IPTG inducible P<SUB><I>lacI</I></SUB> promoter but lacks any restriction enzyme sites interferring with the RFC25 standard.
</P><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 80%;"><FIGCAPTION><B>Figure 2: Cloning scheme of the Evaluation Vector.</B> The detailed cloning workflow which led to the finished Evaluation Vector construct with the pBS1C backbone.</FIGCAPTION></FIGURE><P>Additionally, we added an AgeI restriction enzyme site downstream of the RFP coding sequence which is necessary for translational fusions. Furthermore, we amplified a <I>lacZα</I> fragment with AgeI and NgoMIV restriction enzyme sites upstream of the coding sequence and the RFC10 BioBrick standard as suffix using the primers iG17P055 and iG17P056.Finally, we combined our new MCS, by ligating the digested RFPsyn2 (cut with XbaI and AgeI) with the <I>lacZα</I> fragment (cut with AgeI and PstI). This MCS was inserted into the pBS1C-P<SUB><I>xylA</I></SUB> backbone, which was prior opened using BsaI (resulting in an XbaI overhang) and PstI (Figure 2). The final construct of our EV was verified by sequencing.We decided to additionally also provide this MCS as a BioBrick. Therefore we cloned it into the pSB1C3 backbone (via EcoRI and PstI digest) and verified the construct by sequencing. It has been submitted to the parts registry under <A target="_blank" href="http://parts.igem.org/Part:BBa_K2273107">BBa_K2273107</A>.
</P><P>Detailed methods can be found in our <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">protocol collection section</A>. All primers used can be found in our primer collection table down below.
</P><P><A class="pdf-resources" href="https://static.igem.org/mediawiki/2017/b/b9/T--TU_Dresden--M_Primerlist.pdf">Primer collection table</A></P></FIGURE></FIGURE></DIV><DIV class="contentbox"><H1 id="protocol" class="box-heading">Results</H1><FIGURE><FIGURE class="makeresponsive floatright" style="width: 34.9%;"><FIGCAPTION><B>Figure 4: Three color stages of the EV.</B> All agar plates shown contain X-Gal and IPTG. <B>A</B> The EV without any inserts. <B>B</B> The EV with an inserted SP and gene of interest. <B>C</B> The EV with RPFsyn2 and an inserted gene of interest.</FIGCAPTION></FIGURE><FIGURE class="makeresponsive floatright" style="width: 26%;"><FIGCAPTION><B>Figure 3: Vector map of the EV.</B> The MCS is indicated in colors, grey elements refer to features necessary for cloning in <I>E. coli</I> and the white elements refer to <I>B. subtilis</I> specific vector parts.</FIGCAPTION></FIGURE><P>We constructed the <B>Evaluation Vector (EV)</B> to quickly screen for the secretion of a protein of interest. This vector contains a specifically designed multiple cloning site (MCS) equipped with reporters to quickly identify positive replacements by insert integration (Figure 3).  Additionally, this vector can be applied for the expression of any other fusion protein of interest regulated by a promoter of your choice. For more details on the application related with our project check out our <A href="https://2017.igem.org/Team:TU_Dresden/Measurement">Signal Peptide Toolbox</A> and <A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion">Secretion project</A>.</P><P>Due to the unique setup of the MCS which provides easy access to powerful cloning, we provide the MCS as a part stored in the pSB1C3 backbone for the iGEM community (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2273107">BioBrick BBa_K2273107</A>).</P><P>As stated above in the Background section of the EV, we aimed for an easy cloning and screening procedure in our cloning host <I>Escherichia coli</I>. To accomplish that, we chose to set the construct RFPsyn2 as placeholder for the N-terminally fused protein and the gene <I>lacZα</I> for the C-terminally fused protein, respectively. Therefore, the blue color of <I>lacZα</I> carrying colonies and thereby X-Gal degrading colonies masks the red color of the RFPsyn2 on X-Gal containing agar plates (Figure 4, A). However, on not X-Gal containing agar plates, the red color of the RFPsyn2 will be visible. This applies to colonies not carrying <I>lacZα</I>, too (Figure 4, B). <I> E. coli</I> colonies carrying neither <I>lacZα</I> nor RPFsyn2 will stay whitish as common <I> E. coli</I> colonies (Figure 4, C). By applying this setup, successfully transformed <I>E. coli</I> colonies can be identified easily, as stated below in the standard operating procedure (SOP) protocol.</P><P>Based on our design, we established detailed SOP protocols, which can be found below, for working with the <B>Evaluation Vector</B> and the <A href="https://2017.igem.org/Team:TU_Dresden/Measurement"> Signal Peptide Toolbox</A> as the EV was evaluated in the course of it.</P><P><A class="pdf-resources" href="https://static.igem.org/mediawiki/2017/d/db/SOP_protocols_for_cloning_with_the_Evaluation_Vector_and_the_Signal_Peptide_Toolbox.pdf">SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox</A></P></FIGURE><H2>Conclusion</H2><P class="survey-quote"><B>Our team developed a unique multiple cloning site for the Evaluation Vector. The distinct features of the Evaluation Vector allow for easy insertion of both, a promoter and two basic or composite parts while providing an easy cloning and screening procedure. We evaluated and proved the applicability of the system multiple times and are sure, that this special multiple colng site can be of great value for future cloning.</B></P></DIV><DIV class="contentbox"><H1 class="box-heading">References</H1><TABLE class="source"><TBODY><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[1]</A></TD><TD>Radeck, J., Kraft, K., Bartels, J., Cikovic, T., Dürr, F., Emenegger, J., Kelterborn, S., Sauer, C., Fritz, G., Gebhard, S., and Mascher, T. (2013) The <I>Bacillus</I> BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with <I>Bacillus subtilis. J Biol Eng 7</I>, 29.</TD></TR></TBODY></TABLE></DIV></MAIN></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end;">Beam me up!</A></DIV><DIV class="footerbar"><TABLE><TBODY><TR><TD><A href="https://www.google.de/maps/place/Zellescher+Weg+20,+01217+Dresden" target="_blank">iGEM-Team TU Dresden<SMALL>Institut für Mikrobiologie, AG Thorsten MascherZellescher Weg 20b, 01217 Dresden</SMALL>Germany</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><A href="https://www.facebook.com/igem.tud2017" target="_blank">@igem.tud2017</A></TD></TR><TR><TD><A href="https://www.instagram.com/igem_tud_2017" target="_blank">@igem_tud_2017</A></TD></TR><TR><TD><A href="mailto:igem2017@mailbox.tu-dresden.de">igem2017@mailbox.tu-dresden.de</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>