<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Project_Secretion skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Project/Secretion</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2017.igem.org/Team:TU_Dresden/Description" class="nav-text dropbtn">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes" class="nav-text">Peptidosomes</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Biosensor" class="nav-text">β-Lactam Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Measurement" class="nav-text">Signal Peptide Toolbox</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part" class="nav-text">Evaluation Vector</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion" class="nav-text">Secretion</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Communication" class="nav-text">Communication</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU_Dresden/Results" class="nav-text dropbtn">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Demonstrate" class="nav-text">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Parts" class="nav-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Human_Practices" class="nav-text dropbtn">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Silver" class="nav-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Integrated</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Collaborations" class="nav-text">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green" class="nav-text dropbtn">iGEM goes green</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams" class="nav-text">Green Teams</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Resources" class="nav-text">Resources</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Our Achievements</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Experience" class="nav-text">Our Experience</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/About_Us" class="nav-text dropbtn">About Us</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Team" class="nav-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Attributions" class="nav-text">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Sponsors" class="nav-text">Financing &amp; Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Lab_Work" class="nav-text dropbtn">Lab Work</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Experiments" class="nav-text">Experiments &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Safety" class="nav-text">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Judging" class="nav-text dropbtn">Judging</A></LI></DIV></NAV><DIV id="pagebanner" style="background-image: url(https://static.igem.org/mediawiki/2017/d/dd/T--TU_Dresden--planet--secretion.png);"><DIV><DIV id="bannerquote">Get ready for take off!</DIV><DIV id="projecttitle">Secretion</DIV><DIV class="contentbox at-a-glance"><H1 class="box-heading">At a Glance</H1><FIGURE><H4>Motivation:</H4><P>Demonstrate that encapsulated bacteria could release proteins to the environment that surrounds the Peptidosomes.
</P><H4>Approach:</H4><P>Apply the SpyTag-SpyCatcher system to secrete proteins that form complexes in the environment.</P><H4>Achievements:</H4><P>(I) The SpyTag-SpyCatcher system, originally developed by the 2013 team of <A target="_blank" href="http://parts.igem.org/Part:BBa_K1159200">TU-Munich</A> was improved and adapted to secretion in <I>Bacillus subtilis</I>. (II) Secretion and interaction of SpyTag-SpyCatcher system was  <A href="#results" class="hashlink">demonstrated</A>. (III) 8 <A href="#basic" class="hashlink">basic BioBrick  parts</A> were improved and 8 <A href="#composite" class="hashlink">composite parts</A> were generated and evaluated.</P></FIGURE></DIV></DIV><MAIN><DIV class="contentbox"><H1 id="secretion" class="box-heading">Short Description</H1><P>In combining <I>Bacillus subtilis</I> powerful secretion capacity with Peptidosomes as a new platform for functional co-cultivation we aim to produce multiprotein complexes. Various strains - each secreting distinct proteins of interest - can be cultivated in one reaction hub while being physically separated. In this part of EncaBcillus we study extracellular protein interaction mediated by the SpyTag/SpyCatcher system. This set-up bears the potential for an effective production of customizable biomaterials or enzyme complexes.</P></DIV><DIV class="contentbox"><H1 id="background" class="box-heading">Background</H1><P>Efficient and low cost production of valuable natural compounds, like proteins, has developed into a leading industry. Starting from the choice of a suitable production host to the establishment of a profitable downstream process, every step is constantly optimized to increase overall yields. </P><P>When it comes to choose the production host, <I>Bacillus subtilis</I> is particularly interesting: the Gram-positive model organism can be easily genetically modified and has powerful secretion capacities.<A target="_blank" href="https://www.researchgate.net/profile/Reindert_Nijland/publication/23656731_Optimization_of_Protein_Secretion_by_Bacillus_subtilis/links/09e415136142e1554c000000/Optimization-of-Protein-Secretion-by-Bacillus-subtilis.pdf">[1]</A></P><P>In this part of EncaBcillus we aimed at making use of <I> B. subtilis</I> native advantages and combining them with Peptidosomes – a new innovative platform for functional co-cultivation. There is a possibility to create multiple Peptidosomes, each encapsulating one specific strain that secrets a protein of interest. By doing so, the production of multiprotein complexes could be easily achieved in one reaction hub with different subpopulations. This system is illustrated in Figure 1. </P><FIGURE class="makeresponsive"><FIGCAPTION><B>Figure 1: Production of multiprotein complexes with Peptidosomes.</B> Depicted are two different strains of <I>B. subtilis</I> each producing a distinct protein of interest, that is either fused with SpyTag or mini. SpyCatcher. After diffusion into the medium outside of the Peptidosomes, a covalent bonding of the proteins is mediated. </FIGCAPTION></FIGURE><P>To ensure the assembly of proteins outside of the Peptidosomes we further characterized the SpyTag/SpyCatcher system. Theses functional units can be attached to any protein of interest and upon secretion will result in a covalent isopeptide bond between the SpyTag/SpyCatcher partners.<A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/28230977">[2]</A>
The system originates from <I>Streptococcus pyogenes</I> and still remains under constant developments.<A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/22366317&gt;[3]&lt;/a&gt; In our project we applied codon-adapted &lt;i&gt; B. subtilis&lt;/i&gt; specific tags and a SpyCatcher that is reduced in length to enhance it´s usability when translationally fused to a protein of interest. Thus, decreasing the chances of the tag interfering with overall protein folding.&lt;a target=" _blank="">[4]</A></P><P>To demonstrate the applicability of both tags we fused them to a green (sfGFP) and a red (mCherry) fluorescent protein, enabling an easy detectable output. (For more details please check our Design section)
Since a core part of this project involves secretion, we included a signal peptide in front of all our constructs. (click <A href=" https://2017.igem.org/Team:TU_Dresden/Measurement">here</A> to learn more about our Signal Peptide Toolbox).</P><P>To evaluate the efficiency of the secretion process we monitored the fluorescence in supernatants harvested from <I> B. subtilis</I> strains carrying our constructs and compared them to supernatants obtained from the wild type. In order to prove the functionality of the SpyTag/SpyCatcher system we co-incubated supernatants derived from different strains and performed a SDS-Page where we could demonstrate the formation of a fusion protein.</P></DIV><DIV class="contentbox"><H1 id="design" class="box-heading">Design</H1><P>All of the composite parts necessary for the genetic constructs were equipped with the RFC 25 standard, cloned into pSB1C3 backbone and submitted to the registry. All cloning was done according to standard protocols and the plasmids were stored in <I>Escherichia coli</I> DH10β. All constructs were verified by sequencing.</P><P id="basic">The gene encoding the mini. SpyCatcher (<A target="_blank" href=" http://parts.igem.org/Part:BBa_K2273015">BBa_K2273015</A>) was chemically synthesized.
The codon optimized SpyTag (<A target="_blank" href=" http://parts.igem.org/Part:BBa_K2273014">BBa_K2273014</A>) was generated via overlapping primers iG17P049 and G17P050 and amplified using the primers TM4487 and iG17P039. We used a <I>sfGFP</I> (<A target="_blank" href=" http://parts.igem.org/Part:BBa_K2273033">BBa_K2273033</A>) that was codon optimized for <I>Streptococcus pneumoniae</I>, which has been demonstrated to work best in <I>Bacillus subtilis</I>.<A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/23956387">[5]</A> The used <I>mCherry</I> (<A target="_blank" href=" http://parts.igem.org/Part:BBa_K2273034">BBa_K2273034</A>) was codon adapted for <I>B. subtilis</I> (Popp et al., 2017, accepted). The His-tag, necessary for protein purification was included in the reverse primers (Table 1).</P><FIGURE class="jonathanstables" style="width:100%;"><FIGCAPTION><B>Table 1: Overview of the constructed basic parts.</B></FIGCAPTION><TABLE style="width:100%; display: table;"><TBODY style="max-width:inherit;"><TR><TD><B> Gene </B></TD><TD><B>BioBrick</B></TD><TD><B>forward Primer</B></TD><TD><B>reverse Primer </B></TD></TR><TR><TD>SpyTag_His-tag </TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273016">BBa_K2273016</A></TD><TD> TM4487</TD><TD> iG17P068</TD></TR><TR><TD>mini. SpyCatcher_His-Tag </TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273017">BBa_K2273017</A></TD><TD>TM4487</TD><TD> iG17P067</TD></TR><TR><TD> sfGFP_His-tag</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273021">BBa_K2273021</A></TD><TD>TM4487</TD><TD> iG17P065</TD></TR><TR><TD>mCherry_His-tag </TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273022">BBa_K2273022</A></TD><TD>TM4487</TD><TD>iG17P066</TD></TR></TBODY></TABLE></FIGURE><P>In order to identify the best combination of SpyTag/SpyCatcher and FP fusion, we constructed all N- and C-terminal combinations (Table 2).</P><FIGURE class="jonathanstables" style="width:100%;" id="composite"><FIGCAPTION><B>Table 2: List of SpyTag/SpyCatcher and FP composite parts.</B></FIGCAPTION><TABLE style="width:100%; display: table;"><TBODY style="max-width:inherit;"><TR><TD><B> Gene </B></TD><TD><B>BioBrick</B></TD></TR><TR><TD> pSB1C3-mCherry-SpyTag-His </TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273035">BBa_K2273035</A></TD></TR><TR><TD> pSB1C3-mCherry-SpyCatcher-His</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273036">BBa_K2273036</A></TD></TR><TR><TD> pSB1C3-SpyTag-mCherry-His </TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273037">BBa_K2273037</A></TD></TR><TR><TD>pSB1C3-SpyCatcher-mCherry-His</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273038">BBa_K2273038</A></TD></TR><TR><TD>pSB1C3-sfGFP-SpyTag-His</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273039">BBa_K2273039</A></TD></TR><TR><TD>pSB1C3-sfGFP-SpyCatcher-His</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273040">BBa_K2273040</A></TD></TR><TR><TD>pSB1C3-SpyTag-sfGFP-His</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273041">BBa_K2273041</A></TD></TR><TR><TD>pSB1C3-SpyCatcher-sfGFP-His</TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273042">BBa_K2273042</A></TD></TR></TBODY></TABLE></FIGURE><P>For the final construct we added the signal peptide sequence of amyE (<A href="http://parts.igem.org/Part:BBa_K2273023">BBa_K2273023</A>) upstream of all constructs (Table 2) and cloned them into the single copy integrative <I>B. subtilis</I> vector pBS2EP<I><SUB>xylA</SUB></I>. In brief, the vector has the following features for cloning in <I>E.coli</I>: an ori of replication and the <I>bla</I> gene mediating resistance against ampicillin. The <I>B. subtilis</I>specific part of the vector contains the multiple cloning site (MCS) in RFC10 standard with a P<I><SUB>xylA</SUB></I> promoter upstream of the BioBrick prefix, a <I>erm</I> cassette providing resistance against erythromycin /lincomycin and flanking regions needed for integration into the <I>lacA</I> locus. </P><P>An overview of the final constructs is depicted in Figure 2 and Figure 3.</P><P>The sequenced plasmids were transformed into <I>B. subtilis</I> strain WB800N, a protease deficient strain. Fluorescence measurements of the supernatants derived from strains harbouring our constructs (Table 2) in the <I>B. subtilis</I> vector, was performed according to the protocol below. </P><FIGURE class="makeresponsive"><FIGCAPTION><B>Figure 2: Genetic constructs with sfGFP.</B> Depicted are translational fusion constructs downstream of the P<I>xylA</I> promotor, that were cloned in the multiple cloning site of the pBS2EP<I><SUB>xylA</SUB></I> vector. The constructs contain a signal peptide sequence, the gene coding for sfGFP, either c- or n-terminally fused SpyTag or mini. SpyCatcher and a his-tag.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B><B>Figure 3: Genetic constructs with mCherry.</B> Depicted are translational fusion constructs downstream of the P<I>xylA</I> promotor, that were cloned in the multiple cloning site of the pBS2EP<I><SUB>xylA</SUB></I> vector. The constructs contain a signal peptide sequence, the gene coding for mCherry, either c- or n-terminally fused SpyTag or mini. SpyCatcher and a his-tag.</B></FIGCAPTION></FIGURE></DIV></MAIN><P>SpyTag/SpyCatcher and FP (mCherry with C-terminal mini. SpyCatcher and C-terminal SpyTag, sfGFP with N-terminal SpyTag) were subcloned into the medium copy (15-20 copies per cell) <I>B. subtilis</I> vector pBS0E (Popp et al., 2017, accepted). Via Q5-PCR using the primers TM4067 and TM3082 and the corresponding pBS2E-plasmids as template. The PCR products were digested with BsaI und PstI and ligated with the EcoRI and PstI digested pBS0E vector backbone. The sequenced plasmids were transformed into <I>B. subtilis</I> strain WB800N. Fluorescence assay was performed as described above. Protein purification from the supernatants to check the functionality of the SpyTag/SpyCatcher and FP fusions was performed according to the <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">protocol</A>.</P><H3>Fluorescence assay:</H3><P>The protocol for the implemented assay can be found under the <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">experiments &amp; protocol</A> page. For all the assays biological duplicates and technical triplicates were used.</P><H3>Purification and SDS-PAGE:</H3><P>
To prove the functionality of the SpyTag and the mini. SpyCatcher as a part of the fusion-proteins, the supernatants were purified with a quick <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">protocol</A> using agarose beads. The samples were mixed with a loading buffer containing a reducing agent and heated for 5 min at 95°C. 10 μl were then loaded onto a 12,5% SDS gel, which was run at 200 V for 45 min. and stained in Coomassie Blue over night. </P></DIV><DIV class="contentbox"><H1 id="results" class="box-heading">Results</H1><H3>Fluorescence assay of supernatants derived from strains with single copy genes</H3><P>The first obstacle that we had to overcome was establishing a suitable protocol to boost the secretion capacities of our generated <I>Bacillus subitilis</I> producer strains. After initially testing various media and incubation periods we performed all experiments using 2xYT medium and an incubation time of 16 h.</P><P>Applying this protocol, we observed good fluorescence in the supernatants harvested from the eight strains (combinations of fluorescence proteins (FP) with SpyTag/SpyCather (Tag) constructs) secreting either mCherry or sfGFP.  We were able to detect higher fluorescence when compared to supernatants derived from the wild-type (Figure 4). </P><P>The supernatant of the wild-type shows a relatively high fluorescence in case of sfGFP , because of the autofluorescence derived from the medium (data not shown). When compared with each other, all of the different fusion possibilities of a FP and the Tags show a similar fluorescence, with the exception of the N-terminally fused mini. SpyCatcher constructs. Apparently, this fusion either inhibits the native folding of the FP or decreases the secretion efficiency. </P><FIGURE class="makeresponsive"><FIGCAPTION><B>Figure 4: Endpoint measurement of the fluorescence from supernatants carrying our constructs and the wild type. </B>
Expression of the single copy mCherry or sfGFP fusion SpyTag/SpyCather constructs (purple) was induced with 1% xylose and the supernatants were harvested after 16 h of incubation. Wild type supernatant is shown as a control (pink). Excitation wavelength for sfGFP was set to 480 nm and emission was recorded at 510 nm and for mCherry excitation wavelength was set to 585 nm and emission was recorded at 615 nm. The fluorescence was normalized by the optical density (OD<SUB>600</SUB>). Graph shows mean values and standard deviations of at least two biological and three technical replicates. 
 </FIGCAPTION></FIGURE><FIGURE><FIGURE class="makeresponsive floatright" style="width: 50%;"><FIGCAPTION><B>Figure 5: Endpoint measurement of the fluorescence obtained under different conditions.</B><I>B. subtilis</I> strain WB800N with a single copy of mCherry- mini. SpyCatcher is compared to the wild type after 16 h of incubation and initial induction with 1% xylose. Pink bars show the harvested supernatant, purple bars refer to pelleted cells resuspended in fresh medium and blue bars to non-processed cell solution. Excitation wavelength was set to 585 nm and emission was recorded at 615 nm. Graph shows mean values and standard deviations of at least two biological and three technical replicates.</FIGCAPTION></FIGURE><P>To check the efficiency of the secretion process, we also determined the fluorescence of the cells. The supernatant was separated from the cells by centrifugation.</P><P>The pelleted cells were resuspended in fresh medium, immediately followed by a fluorescence measurement (assuming the recorded fluorescence exclusively derives from FPs still inside the cells). We compared the relative fluorescence of both samples and also included untreated 16h old cell culture in this assay.</P><P>Figure 5 shows data generated for one strain (mCherry-mini. SpyCatcher) as an example. While the relative fluorescence in the unprocessed cell culture and the supernatant do not differ significantly, the emission of the fresh-resuspended cells show a clear drop of fluorescence (comparable to the wild-type level). </P><P>This clearly demonstrates that indeed the measured fluorescence intensities derive from secreted FPs. </P></FIGURE><H3 id="dual">Fluorescence assay of supernatants derived from strains with multi copy genes:</H3><P>To further increase the secretion capacity, three constructs were sub-cloned into the pBS0E vector which is a medium copy number <I>B. subtilis</I> specific vector. When performing the same fluorescence assay as bevor, we obtained eight to ten times higher fluorescence intensities in the supernatants when compared to the supernatants derived from single copy (Figure 6). </P><FIGURE class="makeresponsive"><FIGCAPTION><B>Figure 6: Endpoint measurement of the fluorescence from supernatants. </B>
Expression of the multi copy mCherry or sfGFP fusion constructs (purple) was induced with 1% Xylose and the supernatants were harvested after 16 h of incubation. Wild type supernatant is shown as a control (pink). Excitation wavelength for sfGFP was set to 480 nm and emission was recorded at 510 nm and for mCherry excitation wavelength was set to 585 nm and emission was recorded at 615 nm . The fluorescence was normalized over the optical density (OD<SUB>600</SUB>). Graphs show mean values and standard deviations of at least two biological and three technical replicates. 

 </FIGCAPTION></FIGURE><P>The sfGFP fusion proteins were visible in the supernatant when excited with blue light (Figure 7) and the typical red colour of mCherry could even be observed under daylight conditions (Figure 8). These protein rich supernatants were used for stability tests, spectral analysis, purification and SDS-PAGE.</P><DIV style="display: flex; align-items: flex-start; flex-wrap: wrap;"><DIV class="makeresponsive" style="width:50%;"><FIGURE class="makeresponsive" style="padding-left:10%; padding-right:10%;"><FIGCAPTION><B>Figure 7: Supernatants of <I><B>B. subtilis</B></I> cultures excited with blue light.</B>
Wild-type supernatant (left) and a SpyTag-sfGFP secreting strain (middle and right). The expression of the multi-copy sfGFP was induced with 1% Xylose and the supernatant was harvested after 16 h of incubation. The fluorescence was induced with a “Dark Reader Transilluminator”.
</FIGCAPTION></FIGURE></DIV><DIV class="makeresponsive" style="width:50%;"><FIGURE class="makeresponsive floatleft" style="padding-left:10%; padding-right:10%;"><FIGCAPTION><B>Figure 8: Supernatants of <I><B>B. subtilis</B></I> cultures.</B>
Wild-type supernatant (left) and a mCherry-mini. SpyCatcher secreting strain (right). The expression of the multi-copy mCherry was induced with 1% Xylose and the supernatant was harvested after 16 h of incubation.
</FIGCAPTION></FIGURE></DIV></DIV><H3>Stability:</H3><P>After incubating the supernatants containing the fluorescent fusion protein for 24h they still showed the same fluorescence intensity, implying that the protein is quite stable in 2xYT medium (graph included in pdf below).  Using the <I>B. subtilis</I> strain WB800N as host contributes to an increased stability, as it is not secreting any proteases that may degrade the protein of interest.</P><A target="_blank" class="pdf-resources" href="https://static.igem.org/mediawiki/2017/a/a3/T--TU_Dresden--secretion--results--stability.pdf">Stability of secreted FP</A><H3>Emission spectra:</H3><P>The emission spectra of the supernatants containing fusion protein producing cultures were analysed to prove that the measured fluorescence’s originates from sfGFP or mCherry. In Figure 9 the emission spectra of the sfGFP-fusion protein is clearly visible, having an emission peak at ca. 510 nm. The mCherry-fusion protein displays the correct emission spectrum as well, having a peak at 615 nm. </P><P>The 2xYT medium that was used for cultivation shows an overall high auto-fluorescence when sfGFP wavelengths were measured. As already mentioned before, this influences the total fluorescent emissions of the supernatant. Likewise, it needs to be considered that the wild-type supernatant has a lower fluorescence than the medium, suggesting a quenching effect caused by <I>B. subtilis</I>. On the other hand, only very low auto-fluorescents or quenching effects could be observed when the mCherry spectrum was recorded. Thus, we decided to carry on with mCherry constructs to prove the applicability of the SpyTag/SpyCatcher systems. </P><FIGURE class="makeresponsive"><FIGCAPTION><B>Figure 9: Emission spectra of supernatants and medium. </B> Expression of the multi copy mCherry or sfGFP fusion constructs was induced with 1% xylose and the supernatants were harvested after 16 h of incubation. The sfGFP is N-terminal fused with Spytag, the mCherry is C-terminal fused with mini. SpyCatcher. The supernatants containing fluorescent proteins (pink) were diluted 1:4. Spectra of wilde-type supernatant (blue) and medium (purple) are shown as a control. Excitation wavelength for sfGFP was set to 585 nm and for mCherry to 615 nm. The endpoint fluorescence was measured for each wavelength in steps of one nm ranging from 490 nm to 600 nm for sfGFP and from 580 nm to 700 nm for mCherry. 
 </FIGCAPTION></FIGURE><H3 id="interplay">Purification and SDS-PAGE:</H3><P>By running SDS-PAGE with the purified supernatants of mCherry fusion protein producing cultures, we could prove the functionality of the SpyTag/SpyCatcher partners. In Figure 10 it is clearly visible, that the supernatants from the production strains (lane 4 and 6) contain protein bands of high concentration that the wild-type (lane 2) lacks. These proteins band are showing up again in the elution after purification, proving that the bands are referring to the his-tagged target proteins, mCherry-mini. SpyCatcher having a weight of 36,6 kDa and mCherry-SpyTag weighting 31,9 kDa.</P><P> Surprisingly, the supernatant with mCherry-SpyTag contained a band of the wrong weight, and the mCherry-mini. SpyCatcher even three bands the wrong weight, indicating accumulation or degradation of the protein of interest.</P><P>Nonetheless, upon combining two supernatants and incubating for 4 h, a new protein band with ca. 70 kDA shows up (lane 8 and 9). This is the definite proof, that the mini. SpyCatcher and the Spytag are functional and mediate the covalent bonding of the mCherry constructs. </P><FIGURE class="makeresponsive"><FIGCAPTION><B>Figure 10: SDS gel with crude and purified supernatants.</B> Expression of the multi copy mCherry constructs was induced with 1% Xylose and the supernatants were harvested after 16 h of incubation. The his-tagged proteins were purified with Ni-NTA agarose beads. Lane 1 was loaded with 3 	µl of NEB´s “Color Prestained Protein Standard Broad Range” ladder. Crude (c) and purified (p) supernatant of wild-type (WT) are shown as a control in lane 2 and 3. Lane 4 and 5 contain the supernatant of <I>B. subtilis</I> producing mCherry-mini. SpyCatcher fusion protein (36,6 kDa). Lane 4 and 5 contain the supernatant of <I>B. subtilis</I> producing mCherry-SpyTag fusion protein (31,9 kDa).  The crude supernatants of the two mCherry producing strains were combined, incubated for 4 h, purified and loaded onto lane 8 and 9. The fusion product of the mCherry constructs is visable in the crude and purified supernatant.
 </FIGCAPTION></FIGURE><H2 id="conclusion">Conclusion</H2><P class="survey-quote"><B>Our team succeeded in the production of self conjugating protein complexes using <I><B>B. subtilis</B></I>secretion capacity’s. Even though the system could not yet be tested in Peptidosomes, we are very much sure that our vision to facilitate the production process of tuneable protein complexes can be achieved. Peptidosomes as a novel platform for cultivation and <I><B>B. subtilis</B></I> powerful secretion are a promising combination that should be studied further.</B></P></DIV><DIV class="contentbox"><H1 class="box-heading">References</H1><TABLE class="source"><TBODY><TR><TD><A target="_blank" href="https://www.researchgate.net/profile/Reindert_Nijland/publication/23656731_Optimization_of_Protein_Secretion_by_Bacillus_subtilis/links/09e415136142e1554c000000/Optimization-of-Protein-Secretion-by-Bacillus-subtilis.pdf">[1]</A></TD><TD>Nijland, Reindert &amp; Kuipers, Oscar. (2008). Optimization of Protein Secretion by <I>Bacillus subtilis</I>. <I>Recent patents on biotechnology</I></TD></TR><TR><TD><A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/28230977">[2]</A></TD><TD>Gilbert et. all (2017) Extracellular Self-Assembly of Functional and Tunable Protein Conjugates from <I>Bacillus subtilis. ACS Synth. Biol</I>.</TD></TR><TR><TD><A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/22366317">[3]</A></TD><TD>Zakeri et. All (2012) Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. <I>Applied Microbiology and Biotechnology</I>.</TD></TR><TR><TD><A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/24161952">[4]</A></TD><TD>Li et. All (2013) Structural Analysis and Optimization of the Covalent Association between SpyCatcher and a Peptide Tag . <I>J. Mol. Biol.</I></TD></TR><TR><TD><A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/23956387">[5]</A></TD><TD>Overkamp, W. et al. Benchmarking various green fluorescent protein variants in <I>Bacillus subtilis, Streptococcus pneumoniae</I>, and <I>Lactococcus lactis</I> for live cell imaging. <I>Appl. Environ. Microbiol.</I> 79, 6481–6490 (2013).</TD></TR></TBODY></TABLE></DIV></DIV></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end;">Beam me up!</A></DIV><DIV class="footerbar"><TABLE><TBODY><TR><TD><A href="https://www.google.de/maps/place/Zellescher+Weg+20,+01217+Dresden" target="_blank">iGEM-Team TU Dresden<SMALL>Institut für Mikrobiologie, AG Thorsten MascherZellescher Weg 20b, 01217 Dresden</SMALL>Germany</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><A href="https://www.facebook.com/igem.tud2017" target="_blank">@igem.tud2017</A></TD></TR><TR><TD><A href="https://www.instagram.com/igem_tud_2017" target="_blank">@igem_tud_2017</A></TD></TR><TR><TD><A href="mailto:igem2017@mailbox.tu-dresden.de">igem2017@mailbox.tu-dresden.de</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>