<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Project_Biosensor skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Project/Biosensor</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2017.igem.org/Team:TU_Dresden/Description" class="nav-text dropbtn">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes" class="nav-text">Peptidosomes</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Biosensor" class="nav-text">β-Lactam Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Measurement" class="nav-text">Signal Peptide Toolbox</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part" class="nav-text">Evaluation Vector</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion" class="nav-text">Secretion</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Communication" class="nav-text">Communication</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU_Dresden/Results" class="nav-text dropbtn">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Demonstrate" class="nav-text">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Parts" class="nav-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Human_Practices" class="nav-text dropbtn">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Silver" class="nav-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Integrated</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Collaborations" class="nav-text">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green" class="nav-text dropbtn">iGEM goes green</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams" class="nav-text">Green Teams</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Resources" class="nav-text">Resources</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Our Achievements</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Experience" class="nav-text">Our Experience</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/About_Us" class="nav-text dropbtn">About Us</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Team" class="nav-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Attributions" class="nav-text">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Sponsors" class="nav-text">Financing &amp; Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Lab_Work" class="nav-text dropbtn">Lab Work</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Experiments" class="nav-text">Experiments &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Safety" class="nav-text">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Judging" class="nav-text dropbtn">Judging</A></LI></DIV></NAV><DIV id="pagebanner" style="background-image: url(https://static.igem.org/mediawiki/2017/2/27/T--TU_Dresden--planet--biosensor.png);"><DIV><DIV id="bannerquote">Luminescence Alert!</DIV><DIV id="projecttitle">β-Lactam Biosensor</DIV><DIV class="contentbox at-a-glance"><H1 class="box-heading">At a Glance</H1><FIGURE><H4>Motivation:</H4><P>Demonstrate that encapsulated bacteria could respond to the environment that surrounds the peptidosomes.</P><H4>Approach:</H4><P>Develop a novel whole-cell biosensor that responds to a variety of β-lactam antibiotics (input) with bioluminescence (output).</P><H4>Achievements:</H4><P>(I) A fully functional <A href="#design" class="hashlink">BlaR1I-P<SUB><I>blaZ</I></SUB></A> system from <I>Staphylococcus aureus</I> was heterologously established in <I>Bacillus subtilis</I>. (II) The resulting whole-cell biosensor responded specifically to the presence of a <A href="#detection" class="hashlink">diverse range of β-lactam antibiotics</A> in a dose-dependent manner. (III) <A href="#peptidosomes" class="hashlink">Encapsulated biosensor</A> cells could respond to antibiotics provided to the outside. (IV) 8 novel basic <A href="#BioBricks" class="hashlink">BioBrick parts</A> were generated and fully evaluated for functionality.</P></FIGURE></DIV></DIV><MAIN><DIV class="contentbox"><H1 class="box-heading">Short Description</H1><P>Worldwide, multidrug-resistant bacteria are on the rise and provoke the intensive search for novel effective compounds. To simplify the search for new antibiotics and to track the antibiotic pollution in water samples, whole-cell biosensors constitute a helpful investigative tool. In this part of EncaBcillus, we developed a functional and independent heterologous β-lactam biosensor in <I>Bacillus subtilis</I>. These specialised cells are capable of sensing a compound of the β-lactam family and will respond by the production of an easily measurable luminescence signal. We analysed the detection range and sensitivity of the biosensor in response to six different β-lactam antibiotics from various subclasses. The evaluated biosensor was then encapsulated into Peptidosomes to prove the concept of our project EncaBcillus. The encapsulation of engineered bacteria allows a simplified handling and increased biosafety, potentially raising the chances for their application e.g. sewage treatment plants.</P></DIV><DIV class="contentbox"><H1 id="background" class="box-heading">Background</H1><FIGURE><FIGURE class="makeresponsive floatright" style="width: 55%;"><FIGCAPTION><B>Figure 1: Commonly used β-lactam antibiotics and their chemical structure.</B> All of them share the so-called β-lactam ring structure (here shown as square structure containing nitrogen).<A target="_blank" href="https://en.wikipedia.org/wiki/Β-lactam_antibiotic">[4]</A></FIGCAPTION></FIGURE><P>Antibiotics represent the most effective treatment against bacterial infections. Since the discovery of penicillin by Alexander Fleming in 1928, many new antibiotics have been constantly developed and were successfully applied to treat life-threatening diseases. This significant advancement in medicine saved millions of lives and still does today. However, fighting microorganisms has never been a completed task, but rather an ongoing race between drug discovery and pathogens developing resistances. Thus, multi-drug resistant bacteria still constitute a major threat for humanity, as infectious diseases represent the second leading cause of death worldwide.<A target="_blank" href="https://www.aerzteblatt.de/archiv/52563">[1]</A><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422635/">[2]</A></P><FIGURE class="makeresponsive floatright" style="width: 55%"><FIGCAPTION><B>Table 1: β-lactams and controls tested in this project and their respective classification.<A target="_blank" href="https://en.wikipedia.org/wiki/Β-lactam_antibiotic">[4]</A></B></FIGCAPTION></FIGURE><P>One major reason, for the steady increase of antimicrobial resistances is the “inappropriate use of antimicrobials”. Due to excessive prescription and application in livestock farming, little amounts of antibiotics are nearly found everywhere, even in drinking water. These low-dose and non-lethal concentrations containing habitats, allow bacteria to adjust and develop resistances.<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422635/">[2]</A><A target="_blank" href="http://www.who.int/mediacentre/factsheets/fs194/en/">[3]</A></P><P>As β-lactams make up a large percentage of all antibiotics used, the project preferentially focused on this class of broad-spectrum antibiotics. Carbapenems, penicillin derivatives, cephalosporins and monobactams represent the four main classes of the β-lactams that sum up to over 100 different active substances. All compounds of this particular group can be easily identified by their common chemical structure: the β-lactam ring (see Figure 1).<A target="_blank" href="https://en.wikipedia.org/wiki/Β-lactam_antibiotic">[4]</A></P><P>To address the increasing development of multi-drug resistant bacteria our iGEM Team aims at developing a novel β-lactam biosensor in <I>Bacillus subtilis</I> based on the genetics of the <I>bla</I>-operon found in <I>Staphylococcus aureus</I> (for a detailed description consider our Design section below). 
The genetically engineered biosensor will help to <B>(I)</B> reliably detect even minimal antibiotic concentrations of compounds from the β-lactam family in waste and drinking water and <B>(II)</B> unravel producer strains of yet unknown β-lactam related antibiotics. After extensive characterization of the detection range and sensitivity, the greater goal is to combine the functional β-lactam biosensor with our Peptidosomes. Thereby, we would prove the applicability of EncaBcillus as a completely new cultivation platform. Encapsulation of this whole-cell biosensor, will allow an easier and safer handling of the bacteria and thus making them more appealing for field applications, like for example in sewage treatment plants. As a proof of principle we used six β-lactams and two controls (water and bacitracin) to evaluate our biosensor (Table 1).</P></FIGURE></DIV><DIV class="contentbox"><H1 id="design" class="box-heading">Design</H1><H3><B>Genetic engineering</B></H3><P>To achieve our goal of encapsulating bacteria into Peptidosomes that can sense antibiotics of the β-lactam family, we first needed to develop a reliable biosensor strain. 
In <I>Staphylococcus aureus</I> the <I>bla</I>-operon encodes a one-component system, which is responsible for sensing and mediating resistance against β-lactam antibiotics. The idea was to transfer the regulatory elements of this operon to <I>Bacillus subtilis</I> and replace the native output – being the β-lactamase <I>BlaZ</I> – by an easy detectable signal. Thus, making <I>Bacillus subtilis</I> a β-lactam sensing biosensor. (see Figure 2).</P><DIV style="display: flex; align-items: flex-start; flex-wrap: wrap;"><DIV class="makeresponsive" style="width:50%;"><FIGURE class="makeresponsive" style="padding-left:10%; padding-right:10%;"><FIGCAPTION><B>Figure 2: Overall concept showing the components and the molecular mechanism of the biosensor in <I><B>B. subtilis</B></I></B>. Upon binding of a β-lactam to the receptor BlaR1 <B>(1)</B>, due to the receptors c-terminal proteolytic activity, the repressor BlaI is degraded and frees the target promoter <B>(2)</B> enabling the expression of an easy detectable reporter <B>(3)</B>.
</FIGCAPTION></FIGURE></DIV><DIV class="makeresponsive" style="width:50%;"><FIGURE class="makeresponsive floatleft" style="padding-left:10%; padding-right:10%;"><FIGCAPTION><B>Figure 3: Genetic constructs necessary for the functional biosensor strain.</B><B>(A)</B><I>blaR1</I> under control of the constitutive promoter P<SUB><I>veg</I></SUB> or the inducible promoter P<SUB><I>xylA</I></SUB>,<B>(B)</B><I>blaI</I> downstream of the promoter P<SUB><I>lepA</I></SUB>,  and <B>(C)</B> the promoters P<SUB><I>blaZ</I></SUB> or P<SUB><I>blaR1I</I></SUB> controlling the expression of the <I>luxABCDE</I> operon.</FIGCAPTION></FIGURE></DIV></DIV><P>For the creation of our biosensor in <I>B. subtilis</I>, the <I>bla</I>-operon from <I>S. aureus</I> was split into three genetic constructs: <B>(A)</B> The Receptor gene <I>blaR1</I> under control of a strong constitutive promotor (P<SUB><I>veg</I></SUB>), <B>(B)</B> the repressor gene <I>blaI</I> under control moderate strong constitutive promoter (P<SUB><I>lepA</I></SUB>) and <B>(C)</B> the target promoter region of the <I>bla</I>-operon (P<SUB><I>blaZ</I></SUB> and P<SUB><I>blaR1I</I></SUB>) in front of the <I>lux</I>-operon (<I>luxABCDE</I>) (see Figure 3). In addition, an inducible version of the <I>blaR1</I> construct was made by inserting the P<SUB><I>xylA</I></SUB> promoter upstream of the <I>blaR1</I> gene <B>(A)</B>.<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2942778/">[5]</A></P><P>All genetic constructs and plasmids have been created using the <A target="_blank" href="http://parts.igem.org/Help:Standards/Assembly/RFC10">RFC10</A> and/or <A target="_blank" href="http://parts.igem.org/Assembly_standard_25">RFC25</A> cloning standard. Enzymes used were obtained from New England BioLabs<SUP>©</SUP>. Cloning procedures were carried out according to the manufacturer`s protocols. </P><P>For submission of our parts to the registry, all Biobricks were cloned into the pSB1C3 backbone. The created genetic constructs were verified by sequencing (Eurofins or GATC sequencing services).  All designed plasmids were stored in <I>Escherichia coli</I> DH10β (see <A target="_blank" href="https://2017.igem.org/Team:TU_Dresden/Experiments">Experiments and Protocols</A> for details). In this project, we used integrative single-copy <I>B. subtilis</I> specific vectors that stably integrate into the genome at designated loci.<A target="_blank" href="https://jbioleng.biomedcentral.com/articles/10.1186/1754-1611-7-29">[6]</A></P><FIGURE class="jonathanstables" style="width:100%;"><FIGCAPTION id="BioBricks"><B>Table 2: Overview of the basic parts designed for the biosensor project in the pSB1C3 backbone .</B></FIGCAPTION><TABLE style="width:100%; display: table;"><TBODY style="max-width:inherit;"><TR><TD><B>Part in pSB1C3 backbone</B></TD><TD><B>BioBrick Number</B></TD></TR><TR><TD>pSB1C3-<I>blaR1</I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273108">BBa_K2273108</A></TD></TR><TR><TD>pSB1C3-<I>blaZ</I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273109">BBa_K2273109</A></TD></TR><TR><TD>pSB1C3-<I>blaI</I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273110">BBa_K2273110</A></TD></TR><TR><TD>pSB1C3-P<SUB><I>blaZ</I></SUB></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273111">BBa_K2273111</A></TD></TR><TR><TD> pSB1C3-P<SUB><I>blaR1I</I></SUB></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273112">BBa_K2273112</A></TD></TR><TR><TD> pSB1C3-P<SUB><I>penP(short)</I></SUB></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273113">BBa_K2273113</A></TD></TR><TR><TD>pSB1C3-<I>penP</I></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273114">BBa_K2273114</A></TD></TR><TR><TD> pSB1C3-P<SUB><I>penP(long)</I></SUB></TD><TD><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273116">BBa_K2273116</A></TD></TR></TBODY></TABLE></FIGURE><H3><B>Biosensor Characterization</B></H3><P>First, we investigated the detection range towards different β-lactam families as well as the sensitivity of the created biosensor. Therefore, we conducted plate reader experiments and disk diffusion assays to test our biosensor in liquid as well as on solid conditions. We recorded the luminescence signal and growth behavior (see <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Experiments and Protocols</A> for details) of our biosensor strains in the presence of six different β-lactam antibiotics. We also included physiological controls that lack one or two of the genetic constructs of the complete biosensor machinery. 
Furthermore, we analyzed the impact of deleting the <I>Bacillus subtilis</I> gene <I>penP</I> - encoding a β-lactamase (which has not been studied intensively yet) - on the luminescence output. The strain W168 <I>penP::kan<SUP>R</SUP></I> was created via Long-Flanking Homology PCR (see <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Experiments and Protocols</A> for details). We also investigated, if the different β-lactam antibiotics induce the promoter driving <I>penP</I>.<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684797/">[7]</A></P><H3><B>Encapsulation into Peptidosomes</B></H3><FIGURE><P>Finally, we could demonstrate a fully functional biosensor encapsulated in <A target="_blank" href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes">Peptidosomes</A>. For this we used the strain showing the strongest response, broadest detection range and the highest viability when tested with different β-lactams. Peptidosomes containing the biosensor, were exposed to different β-lactams and development of the luminescence signal was obtained every hour over a time period of 5 hours.</P></FIGURE></DIV><DIV class="contentbox"><H1 id="biosensor" class="box-heading">Results</H1><H3><B>1. Determination of Inhibitory Antibiotic Concentrations</B></H3><P>Before starting the actual tests regarding the functionality of the β-lactam biosensor, several pretests have been conducted to determine the optimal antibiotic concentration for the subsequent experiments. Therefore, we analyzed the concentration dependent effect of six different β-lactam antibiotics on the growth of <I>Bacillus subtilis</I> W168 and a strain lacking the <I>B. subtilis</I> native β-lactamase PenP (W168 <I>penP::kan<SUP><I>R</I></SUP></I>). We decided to test the following β-lactams in our assays: ampicillin, carbenicillin, cefoperazone, cefoxitin, cefalexin and penicillin G. As controls we chose water (dH<SUB>2</SUB>O) and the peptide antibiotic bacitracin, which does not belong to the group of β-lactams. </P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 60%"><FIGCAPTION><B>Table 3: Antibiotic concentrations in [µg µl<SUP>-1</SUP>] (final concentration in the well) used in the plate reader assay.
</B></FIGCAPTION></FIGURE><P>In pre-tests, we investigated the growth of <I>B. subtilis</I> wild type and the <I>penP</I> mutant, upon exposure to different concentrations of each tested β-lactam (data not shown). After that, we narrowed these down to two concentrations per antibiotic (see Table 3). Since we did not want to kill our biosensors, we focused on β-lactam concentrations which result in a slight inhibition of growth. These concentrations were used for all following experiments to further characterise our biosensors and the effect of the <I>B. subtilis</I> native β-lactamase (PenP) (see Figure 4). <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Plate reader</A> experiments were performed in <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Mueller Hinton</A> (MH) media, induction with the antibiotics was carried out after one hour of incubation at 37˚C in the plate reader. Growth was monitored every five minutes for at least 18h.</P></FIGURE><P>We expected a higher growth inhibition with rising antibiotic concentrations. In the <I>penP</I> mutant we expected an increased sensitivity towards the β-lactam antibiotics. Addition of water to the culture should not show any effect on the growth and serves as a control. We included a non β-lactam (the peptide antibiotic bacitracin) in all our assays to demonstrate the specificity of the biosensor.</P><FIGURE class="makeresponsive floatright" style="width: 100%"><FIGCAPTION><B>Figure 4: Growth curves of W168 <B>(1a-f)</B> and W168 <I><B>penP::kan<SUP><B><I>R</I></B></SUP></B></I><B>(2a-f)</B> showing an effect after treatment with the tested antibiotics indicated in the legend above the graphs. </B>Samples were induced after 1 hour (indicated by the black line) with <B>(a)</B> ampicillin, <B>(b)</B> carbenicillin, <B>(c)</B> cefoperazone, <B>(d)</B> cefalexin, <B>(e)</B> cefoxitin as well as with <B>(f)</B> penicillin G. Bacitracin and dH<SUB>2</SUB>O serve as controls. Number 1 and 2 represent the two different concentrations tested of each antibiotic, referring to Table 3. Mean values and standard deviation are depicted from at least three biological replicates.
</FIGCAPTION></FIGURE><FIGURE><FIGURE class="makeresponsive floatright" style="width: 40%"><FIGCAPTION><B>Table 4: Antibiotic concentrations in [µg µl<SUP>-1</SUP>] (final concentration in the well) used in all further plate reader experiments.
</B></FIGCAPTION></FIGURE><P>As expected, the data from the pretest performed in triplicates in Figure 4 show, that the presence of the β-lactamase PenP plays a key role in facilitating survival at higher antibiotic concentrations. The wild type strain <I>B. subtilis</I> W168 is therefore able to grow at higher antibiotic concentrations as the mutant W168 <I>penP::kan<SUP>R</SUP></I> when treated with ampicillin, carbenicillin, cefoperazone, cefoxitin and penicillin G (see Figure 4). However, this was not the case for cefalexin and bacitracin. Here, we could not observe any difference in growth inhibition between the wild type and the <I>penP</I> mutant in regard to the tested concentrations (see Figure 4). Cefalexin showed a very strong inhibitory effect on the growth of both wildtype and the mutant. For this reason, we chose a relatively weak final concentration (see Table 4 below). Furthermore, we noticed a growth inhibition of the mutant W168 <I>penP::kan<SUP>R</SUP></I> during the stationary phase, especially when treated with carbenicillin, cefalexin and penicillin G (data not shown). For this reason, we selected weaker antibiotic concentrations for the upcoming experiments with this mutant.</P><P>From these first experiments, we selected the final antibiotic concentrations for the upcoming plate reader experiments with the biosensor strains (see Table 4).</P></FIGURE><H3 id="detection"><B>2. Analysis of Detection Range and Sensitivity</B></H3><H4><B>2.1 Assessing the Detection Range via Plate Reader Assays</B></H4><P>During this project, we generated several strains to investigate the functionality of the heterologous constructs constituting the biosensor in <I>Bacillus subtilis</I>. This result section though will focus on the evaluation of the strains shown in Table 5 as these represent the most interesting ones.</P><P>In our first experiment, we performed plate reader assays in a 96 well plate format and measured growth (OD<SUB>600</SUB>) and luminescence output for 18 hours every 5 minutes. Induction with the β-lactam antibiotics occurred after 1 hour. All strains have been tested in triplicates under the same conditions. Strains with the genotype <I>penP::kan<SUP>R</SUP></I> have been induced with lower concentrations compared to the wild type strain W168 (see Table 4 above).</P><P>After induction, we anticipate a strong increase in luminescence signal for strains containing the full set of constructs (P<SUB><I>blaZ</I></SUB>_<I>lux</I> or P<SUB><I>blaR1I</I></SUB>_<I>lux</I>, P<SUB><I>veg</I></SUB>_<I>blaR1</I> or P<SUB><I>xyl</I></SUB>_<I>blaR1</I>, P<SUB><I>lepA</I></SUB>_<I>blaI</I>), thus representing functional biosensors. Besides the biosensor constructs, we also tested all physiological controls missing one essential composite of the biosensors' heterologous one-component system (data not shown). The control strain W168 (wild type) and control 1, will presumably not show any luminescence output, while the positive control 2 is expected to show a steady luminescence signal regardless of the presence of any antibiotic compound.</P></DIV></MAIN><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 65%"><FIGCAPTION><B>Table 5: Strains of interest with their names and important genotype remarks for differentiation.</B></FIGCAPTION></FIGURE><P>As the β-lactamase PenP confers resistance to β-lactam antibiotics in <I>B. subtilis</I>, a rising luminescence signal is estimated for the P<SUB><I>penP</I></SUB>_<I>lux</I> constructs post induction. Generally, the P<SUB><I>xyl</I></SUB> promoter is activated by adding xylose as an inducer to the medium. In the plate reader experiments, 0.2% xylose was added to the cells (beginning with the day culture) to activate expression of <I>blaR1</I>.</P><P>Further we propose biosensor strains carrying the genotype remark <I>penP::kan<SUP>R</SUP></I> to give a stronger signal in presence of β-lactam compounds, as they cannot be degraded by the PenP enzyme.</P></FIGURE><P>We could not observe a substantial activation of the P<SUB><I>blaR1I</I></SUB> promoter by the β-lactam compounds, which is why we are not taking it into account in the evaluation below. The bar charts in Figure 5 illustrate the best biosensor constructs identified in the plate reader experiments and compare the RLU/OD<SUB>600</SUB> values of the strains 2 hours post induction with the antibiotics.</P><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 55%"><FIGCAPTION><B>Figure 5: RLU/OD<SUB><I>600</I></SUB> values of the different biosensors and the controls are shown 2 hours after induction with the six β-lactams, bacitracin and dH<SUB><B>2</B></SUB>O</B>. Graphs show the Wild-type (<B>black</B>), control 1 (<B>light gray</B>), control 2 (<B>dark gray</B>), biosensor 1 (<B>pink</B>), biosensor 2 (<B>purple</B>), biosensor 3 (<B>white and black</B>) and biosensor 3 Xylose induced (<B>dark blue</B>). Luminescence (RLU/OD<SUB>600</SUB>) output is shown two hours after β-lactam antibiotic induction. Mean values and standard deviation are depicted from at least three biological replicates. </FIGCAPTION></FIGURE><P>As shown in Figure 5, the wildtype W168 (black with white dots) shows no increase in RLU/OD<SUB>600</SUB> values when induced with the different β-lactam antibiotics and controls. Control 1 (black tight stripes) behaves similarly to the wild type strain. The slight decrease of control 2 (light grey) in the bar chart where induction with ampicillin and carbenicillin happened, is mostly explained by the high growth inhibition caused by the chosen concentrations for W168 (with functional PenP). Most of the times, the constitutive expression of the <I>lux</I> operon resulted in an RLU/OD<SUB>600</SUB> of over 1.3 million for control 2 (see Figure 5).</P><P>Biosensor 1 gives an overall good signal for all β-lactam antibiotics tested, but also shows a higher basal activity in absence of the β-lactam compounds of 40.000- 90.000 RLU/OD<SUB>600</SUB> (see Figure 5, bar chart with bacitracin and dH<SUB>2</SUB>O). Further, we could observe a difference in signal intensity dependent on the β-lactam antibiotic tested.  Therefore, biosensor 1 gives the highest signal in presence of penicillin G, cefoxitin and cefoperazone with up to 2.7 million RLU/OD<SUB>600</SUB>. Ampicillin and penicillin G again show a weaker increase in signal produced by biosensor 1, which could be due to the same reason as for control 2 (see Figure 5).</P><P>For biosensor 2, the detection range and sensitivity is comparable to biosensor 1. This strain strongly senses cefoxitin, ampicillin and cefoperazone reaching up to 2.4 million RLU/OD<SUB>600</SUB>. Even the basal activity of the P<SUB><I>blaZ</I></SUB> promoter in biosensor 2, as shown in the bar charts with bacitracin and dH<SUB>2</SUB>O, conforms with the one from biosensor 1.</P></FIGURE><FIGURE><FIGURE class="makeresponsive floatright" style="width: 55%"><FIGCAPTION><B>Figure 6: Luminescence (RLU/OD<SUB><B>600</B></SUB>) output of the P<SUB><I><B>penP</B></I></SUB> promoter versions, two hours after β-lactam induction.</B> Strains carrying different PenP promoter versions (<B>long</B> or <B>short</B> / <B>1</B> and <B>2</B>) or with varying genetic backgrounds (either <B>W168</B> or <I><B>penP::kan<SUP>R</SUP></B></I> / <B>a</B> and <B>b</B>). Mean values and standard deviation are depicted from at least three biological replicates.
</FIGCAPTION></FIGURE><P>The activity of biosensor 3 can be induced by adding xylose (0.2% final concentration) to the sample. In absence of xylose, the signal is very weak as can be seen in the bar with the dark blue stripes (see Figure 5). This is due to the fact, that without xylose, nearly no receptor molecules are localized in the bacteria`s inner cell membrane and the signal cannot be detected and amplified. For induction with cefoperazone though, the weak receptor density on the surface because of the leaky expression of <I>blaR1</I>, seems to be enough to cause a quite powerful signal already. From this can be concluded, that the receptor seems to have a high affinity to cefoperazone. In the bar charts of bacitracin and dH<SUB>2</SUB>O (see Figure 5), biosensor 3 shows the lowest basal promoter activity compared to the other two biosensors tested. Here, we could also observe a high sensitivity for penicillin G, cefoxitin, cefoperazone and ampicillin. In general, all biosensors show good performance in liquid MH-Medium. They able to detect the six β-lactams reliably by giving a signal way stronger than the basal luminescence.</P><P>Furthermore, the analysis of the induction of P<I><SUB>penP</SUB></I> by different β-lactam antibiotics unfolded that this promoter seems to be constitutively active during exponential phase (see Figure 6). As the exact promoter length and potential regulatory regions upstream are still unidentified, two versions (short and long) of the promoter have been examined.</P><P>The RLU/OD<SUB>600</SUB> values shown in Figure 6 indicate a moderate promoter activity during exponential growth. There is no noticeable difference between the strains with a functional PenP enzyme <B>(a)</B> and the <I>penP</I> mutant <B>(b)</B> (see Figure 6). We could not observe a particular activation by β-lactam antibiotics, which suggests that this enzyme is produced might have other functionalities, too.</P></FIGURE><H4><B>2.2 Analyzing the biosensors' behavior on solid medium conducting Disk Diffusion assays</B></H4><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 40%"><FIGCAPTION><B>Table 6: Antibiotic concentrations in [µg µl<SUP>-1</SUP>] (final concentration in the well) used in the Disk Diffusion assays.
</B></FIGCAPTION></FIGURE><P>To determine whether the results obtained in the plate reader assays under liquid conditions can be reproduced on solid agar plates, we performed disk diffusion assays. This assay visualises the detection range of the biosensor by a glowing halo at the edge of the growth inhibition zones, where bacteria are exposed to sub-lethal antibiotic concentrations. We tested the same β-lactam antibiotics and controls as in the previous experiments, but chose higher concentrations since <I>B. subtilis</I> tends to be more resistant against antibiotics when tested on solid media. By increasing the antibiotic concentrations we still obtained slight growth defects but remained in the detection range of the biosensor. The concentrations used in the disk diffusion assays can be extracted from Table 6.</P></FIGURE><FIGURE><FIGURE class="makeresponsive floatright" style="width: 55%"><FIGCAPTION><B>Figure 7: Photographs of the plates from the disk diffusion assay. </B>The upper rows (Panel A and C) show pictures of the plates with the strains under daylight conditions, while the row beneath (Panel B and D) shows the plate after detection of bioluminescence (2 minutes exposure time). At the bottom in Panel E, the disk layout and the most important remarks of the genotype of all strains are indicated. 
</FIGCAPTION></FIGURE><P>We expected the control substances (water and bacitracin) to not cause any luminescence signal at the edge of the inhibition zones. The β-lactam antibiotics should lead to a glowing halo when tested with the three different biosensor versions. The wildtype strain and control 1 should not show any signal, since both strains are lacking the <I>lux</I> operon. In the case of control 2, a luminescence signal should be spread over the whole plate, due to the constitutive expression (P<SUB><I>veg</I></SUB>) of luciferase. Figure 7 sums up the results of the disk diffusion assay for all strains tested. After 24 hours of incubation at 37°C, plates were photographed under day light conditions and under a chemiluminescence dock (with two minutes exposure time).</P><P>As expected, the wildtype and control 1, show no luminescence signal, while control 2 leads to a strong luminescence signal spread across the entire plate (Figure 7, Panel B). Neither bacitracin, nor dH<SUB>2</SUB>O lead to an detectable output, accounting for all strains tested. While in liquid medium biosensor 1 behaves similar compared to biosensor 2, there is a tremendous difference in the detection capability. Biosensor 2 showed detection for all β-lactams tested (Figure 7, Panel D). Au contraire, biosensor 1 only showed a luminescence signal for cefoperazone, cefoxitin and cefalexin (Figure 7, Panel D). Further, the luminescence halo around the cefoxitin disk is quite broad compared to the others, indicating an increased diffusion of the compound into the lawn. Although, biosensor 1 was activated by penicillin G in liquid medium, we could not observe an induction on plate (Figure 7, Panel D).</P><P>Biosensor 2 was activated by all of the β-lactam compounds tested (Figure 7, Panel D). Ampicillin, cefoxitin, cefalexin and cefoperazone strongly activate the system, while penicillin G and carbenicillin just show a weak induction of the signal on plate. These findings go along with the results obtained in liquid medium in the previous experiments.
On the plate with the lawn of biosensor 3 (Panel D), all β-lactams could be detected efficiently when 0.2% xylose was added. In contrast to biosensor 1 and 2, there is a very weak luminescence halo around the cefalexin disk. Also, this halo seems not to be directly at the edge where the cells are in contact with the antibiotic, but rather a bit further off the inhibition zone. Without induction of biosensor 3 with 0.2% xylose, we could not detect any luminescence signal, demonstrating that the receptor (BlaR1) is crucial for detection and signal transduction, standing in line with results obtained in liquid medium (data not shown). The following Table 7 contains the measured diameters of all inhibitions zones caused by the antibiotic. As expected, no inhibition zones around the negative control dH<SUB>2</SUB>O were observed.</P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 100%"><FIGCAPTION><B>Table 7: Measured diameters in mm of the inhibition zones around the disks with the different antibiotics on the plates from the disk diffusion assays.</B></FIGCAPTION></FIGURE></FIGURE><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 55%"><FIGCAPTION><B>Figure 8: Results from the Disk Diffusion Assay with the P<SUB><I><B>penP(long)</B></I></SUB> reporter strains.</B> Daylight pictures (top row) and luminescence detection using a chemiluminescence dock (with two minutes exposure time). Top right scheme shows the disk layout together with the antibiotics applied in the assay. Please note: only results for the long version of the promoter are shown, as the short version behaves accordingly.</FIGCAPTION></FIGURE><P>After evaluating our biosensor versions, we were also curious to follow up on the native <I>B. subtilis</I>β-lactamase PenP. Thus, we performed disk diffusion assays with the P<SUB><I>PenP</I></SUB> (short and long version) reporter strains and checked if any of the β-lactams would lead to a luminescence signal  (Figure 8). Unfortunately, none of the tested substance lead to a notable luminescence. We again could only observe a weak basal promoter activity (as in liquid) with both reporter strains. The measured diameters of the inhibition zones are summarised in Table 8. Taking these results together with the observations in liquid (Figure 6), we can state that the native β-lactamase in <I>B. subtilis</I> dose not respond to any of our tested β-lactams. Yet, we could observe increased sensibility of our biosensors, when <I>penP,</I> is knock-out, accounting for ampicillin in liquid conditions (Figure 5) and for all antibiotics tested on solid agar plates (Figure 7). We could also clearly demonstrate increased sensitivity in terms of resistance against β-lactams, when <I>B. subtilis</I> is lacking PenP (Figure 4).</P><P><FIGURE class="makeresponsive floatleft" style="width: 100%"><FIGCAPTION><B>Table 8: Measured diameter of the inhibition zones in [mm] from the Disk Diffusion Assay with the P<SUB><I><B>penP(long)</B></I></SUB> reporter strains.</B></FIGCAPTION></FIGURE></P></FIGURE><H4 id="dose">2.3 Examining the Sensitivity of the Biosensor</H4><FIGURE><FIGURE class="makeresponsive floatright" style="width: 60%"><FIGCAPTION><B>Figure 9: Dose-Response Curves of the six different β-lactam antibiotics of biosensor 2.</B> Observed luminescence signal (two hours after antibiotic exposure) was plotted according to each tested antibiotic concentration. Please note both axes are depicted logarithmic. Mean values and standard deviation are depicted from at least three biological replicates.
</FIGCAPTION></FIGURE><P>As a final characterisation of our biosensor, we carried out Dose Response assays in order to investigate the detection sensitivity. In this experiment, eleven different concentrations of the β-lactam antibiotics and bacitracin (control) have been tested together with the biosensor 2. This biosensor was chosen due to its reliable performance in liquid and on solid medium and because all necessary components are constitutively expressed without the addition of any further substances (as with biosensor 3).</P><P>The obtained results (se Figure 9) indicate that biosensor 2 shows the highest dynamic range for cefoperazone and is capable to even sense very low concentrations of this compound.  Biosensor 2 shows the weakest dose response towards cefalexin, which is in agreement with all previous experiments (see Figure 9). The antibiotics cefalexin, ampicillin and penicillin G lead to a decrease in luminescence signal for concentrations higher than 10<SUP>-1</SUP>, due to the growth inhibition at these concentrations (see Figure 9). Over all, a high dynamic range upon antibiotic exposure can be detected for all compounds starting at concentrations above 10<SUP>-3</SUP>.</P></FIGURE><H3 id="peptidosomes">3. Encapsulation of the Biosensor into Peptidosomes – Proving the Application Potential</H3><P>After evaluation of the biosensor we probed its activity when encapsulated in Peptidosomes. An overnight culture was inoculated in Fmoc-FF-Solution with a final OD<SUB>600</SUB>=10. Peptidosomes were prepare containing no bacteria (A), W168 (B), control 2 (C) and biosensor 2 (D) (see Figure 10 below) and underwent 3 washing steps. Afterwards, the Peptidosomes were transferred to a 12-well plate, incubated at 37˚C and luminescence was detected every hour. Induction with 0.2 µg µl<SUP>-1</SUP> cefoperazone happened after 1 hour of growth.
</P><P>The Peptidosomes without cells and the wild type W168 are expected to show no luminescence signal at all times (A and B). We estimate control 2 to reach a luminescence signal under non-induced as well as under induced conditions (C). This signal should be weaker than that of the induced biosensor 2 (D, +AB). No signal is expected for the encapsulated biosensor in absence of cefoperazone (D, -AB).</P><FIGURE class="makeresponsivet" style="width: 100%"><FIGCAPTION><B>Figure 10: Encapsulation experiment with biosensor 2.</B> The pictures in the upper row show the distribution of the Peptidosomes at the time point of luminescence detection, which was immediately performed afterwards using a chemiluminescence dock (bottom row). Pink arrows indicate Peptidosomes with a luminescence signal deriving from the encapsulated biosensor. Upper row of well plates contain non-induced samples. Lower row of well plates were induced with cefoperazone (0.2 µg µl<SUP>-1</SUP>).
</FIGCAPTION></FIGURE><P>In this experiment, we successfully encapsulated biosensor 2 into Peptidosomes and demonstrated its ability to sense the β-lactam cefoperazone diffusing into the Peptidosome.  Already 2 hours post induction, there is a luminescence signal detectable for control 2 and the encapsulated biosensor 2 (see Figure 10, middle, C and D). Thereby, we could validate the hypothesis that antibiotic compounds can enter the Peptidosomes and trigger the activation of the biosensor. We also showed, that the performance of the biosensor is not compromised by the encapsulation.  </P><H2>Conclusion</H2><P class="survey-quote"><B>In this part of EncaBcillus we successfully created and analyzed three biosensors that are able to detect six different β-lactam compounds in liquid and on solid medium. We recommend biosensor 2 as our final β-Lactam biosensor, as its performed best: showing the highest detection range of different β-lactams and having the highest sensitivity. In addition to the evaluation of the different biosensors, we even demonstrated the possibility to encapsulate the biosensor in Peptidosomes. By doing so, we confirmed our hypothesis that the antibiotics can diffuse into the Peptidosome and activated the biosensor, which was visible by the luminescence signal. Finally, our encapsulated biosensor is now ready to be used in many potential field applications. </B></P></FIGURE></DIV><DIV class="contentbox"><H1 class="box-heading">References</H1><TABLE class="source"><TBODY><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422635/">[1]</A></TD><TD> C. Lee Ventola, MS (2015) The antibiotic resistance crisis: part 2: management strategies and new agents.<I> Pharmacy and Therapeutics 40(5)</I>, 344–352</TD></TR><TR><TD><A target="_blank" href="https://www.aerzteblatt.de/archiv/52563">[2]</A></TD><TD> www.aerzteblatt.de, visited 08/23/17 (5:34pm) </TD></TR><TR><TD><A target="_blank" href="http://www.who.int/mediacentre/factsheets/fs194/en/">[3]</A></TD><TD> www.who.int, visited 09/04/17 (3:21pm)</TD></TR><TR><TD><A target="_blank" href="https://en.wikipedia.org/wiki/Β-lactam_antibiotic">[4]</A></TD><TD> https://en.wikipedia.org/wiki/Β-lactam_antibiotic, visited 10/27/17 (4:42pm)</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2942778/">[5]</A></TD><TD> Leticia I. Llarrull, Mary Prorok, and Shahriar Mobashery (2010) Binding of the Gene Repressor BlaI to the <I>bla</I> Operon in Methicillin-Resistant <I>Staphylococcus aureus</I>.<I> Biochemistry 49(37)</I>, 7975–7977</TD></TR><TR><TD><A target="_blank" href="https://jbioleng.biomedcentral.com/articles/10.1186/1754-1611-7-29">[6]</A></TD><TD> Radeck, J., Kraft, K., Bartels, J., Cikovic, T., Dürr, F., Emenegger, J., Kelterborn, S., Sauer, C., Fritz, G., Gebhard, S., and Mascher, T. (2013) The <I>Bacillus</I> BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with <I>Bacillus subtilis</I>.<I> J Biol Eng 7(29)</I>, </TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4684797/">[7]</A></TD><TD> Toth, M., Antunes, N.T., Stewart, N.K., Frase, H., Bhattacharya, M., Smith, C.  and Vakulenko, S. (2016) Class D β-lactamases do exist in Gram-positive bacteria. <I>Nature Chemical Biology 12(1)</I>,9-14</TD></TR></TBODY></TABLE></DIV></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end;">Beam me up!</A></DIV><DIV class="footerbar"><TABLE><TBODY><TR><TD><A href="https://www.google.de/maps/place/Zellescher+Weg+20,+01217+Dresden" target="_blank">iGEM-Team TU Dresden<SMALL>Institut für Mikrobiologie, AG Thorsten MascherZellescher Weg 20b, 01217 Dresden</SMALL>Germany</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><A href="https://www.facebook.com/igem.tud2017" target="_blank">@igem.tud2017</A></TD></TR><TR><TD><A href="https://www.instagram.com/igem_tud_2017" target="_blank">@igem_tud_2017</A></TD></TR><TR><TD><A href="mailto:igem2017@mailbox.tu-dresden.de">igem2017@mailbox.tu-dresden.de</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>