<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2017.igem.org/Team:TU_Dresden/Description" class="nav-text dropbtn">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes" class="nav-text">Peptidosomes</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Biosensor" class="nav-text">β-Lactam Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Measurement" class="nav-text">Signal Peptide Toolbox</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part" class="nav-text">Evaluation Vector</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion" class="nav-text">Secretion</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Communication" class="nav-text">Communication</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU_Dresden/Results" class="nav-text dropbtn">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Demonstrate" class="nav-text">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Parts" class="nav-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Human_Practices" class="nav-text dropbtn">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Silver" class="nav-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Integrated</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Collaborations" class="nav-text">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green" class="nav-text dropbtn">iGEM goes green</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams" class="nav-text">Green Teams</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Resources" class="nav-text">Resources</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Our Achievements</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Experience" class="nav-text">Our Experience</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/About_Us" class="nav-text dropbtn">About Us</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Team" class="nav-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Attributions" class="nav-text">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Sponsors" class="nav-text">Financing &amp; Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Lab_Work" class="nav-text dropbtn">Lab Work</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Experiments" class="nav-text">Experiments &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Safety" class="nav-text">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Judging" class="nav-text dropbtn">Judging</A></LI></DIV></NAV><DIV id="pagebanner" style="background-image: url(https://static.igem.org/mediawiki/2017/2/25/T--TU_Dresden--planet--spt.png);"><DIV><DIV id="bannerquote">Secretion is no rocket science!</DIV><DIV id="projecttitle">Signal Peptide Toolbox</DIV><DIV class="contentbox at-a-glance"><H1 class="box-heading">At a Glance</H1><FIGURE><H4>Motivation:</H4><P>Provide a collection of signal peptides to screen for efficient Sec-dependent protein secretion in any bacterial species in a high-throughput manner.</P><H4>Approach:</H4><P>Extract and provide Sec-dependent signal peptides from <I>Bacillus subtilis</I> in a BioBrick-compatible, modified RFC25 standard.</P><H4>Achievements:</H4><P>(I) 74 basic <A href="https://2017.igem.org/wiki/index.php?title=Team:TU_Dresden/Measurement#BioBricks">BioBrick parts</A> are provided with this toolbox.  (II) A robust <A href="#protocol" class="hashlink">PCR-based protocol</A> for amplifying this signal peptide library was developed. (III) A robust secretion efficiency protocol was developed by evaluating combinations of the library with <A href="#threeproteins" class="hashlink">3 different genes</A>, using the <A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part">Evaluation Vector</A>. (IV) This toolbox can be applied to <A href="#anyorganism" class="hashlink">ANY microorganism</A>.</P></FIGURE></DIV></DIV><MAIN><DIV class="contentbox"><H1 class="box-heading">Short Description</H1><P>In bacteria, protein secretion is mainly orchestrated by the Sec Pathway via Signal Peptides (SP), which are located at the N-terminus of secreted proteins. The secretion efficiency is not determined by the sequence of the SP alone, but instead is the combined result of an SP with its specific target protein. This necessitates establishing efficient screening procedures to evaluate all possible SP/target protein combinations. We developed such an approach for our Signal Peptide Toolbox, which contains 74 Sec-dependent SPs. It combines combinatorial construction with highly reproducible, quantitative measurements. By applying this procedure, we demonstrated the secretion of three different proteins and succeeded in identifying the most potent SP-protein combination for each of them. This thoroughly evaluated measurement tool, in combination with our SP toolbox (fully available via the partsregistry) enables an organism-independent, straightforward approach to identifying the best combination of SP with any protein of interest.</P></DIV><DIV class="contentbox"><H1 id="background" class="box-heading">Background</H1><P>Over the course of the last decades the quality, amount and spectrum of heterologous (and recombinant) proteins has drastically increased and therefore the need for techniques to easily express and purify these proteins has emerged. We find such proteins as ingredients of detergents (proteases), medical treatments (insulin) or food and beverage products (amylases). Simply put, heterologous proteins are ubiquitously present. <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23311580">[1]</A></P><P>In order to tackle this demand we chose to apply the genetic tools of the model organism <I>Bacillus subtilis</I>. It is already one of the most frequently used hosts for overproduction of proteins throughout academia and industry because of its tremendous capacity to secret proteins, which can be exploited to increase the overall yield.</P><P><I>B. subtilis</I> has four different secretion pathways, however the majority of proteins are being secreted via the general Sec pathway (Figure 1). This pathway has been identified in playing a crucial role in protein secretion as a common element among all domains of life <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/17938627">[2]</A>. In the Sec pathway, the secretion of proteins into the surrounding supernatant is orchestrated by signal peptides (SP). These SPs are composed of approximately 60 to 180 nucleotides and they are N-terminally attached to the protein thereby orchestrating the secretion. Intracellularly, the SP is translationally fused to the specific protein but cut off during the membrane translocation process releasing the protein into the supernatant without the signal peptide attached to it. <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16997527">[3]</A></P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 70%;"><FIGCAPTION><B>Figure 1: The Sec pathway of <I><B>B. subtilis</B></I>.</B><B>1</B> The protein (blue) and its N-terminally fused SP (red) are ribosomally synthesized (purple). <B>2</B> A Signal Recognition Particle (orange) transports the protein to the membrane. <B>3</B> At the membrane, the translocation complex (light and dark green) takes over the protein. <B>4</B> During the translocation process, the protein is hold in the translocation complex but the SP is cut off by a peptidase (grey). <B>5</B> Afterwards, the protein is released into the supernatant where it reaches its native fold and the SP is degraded.</FIGCAPTION></FIGURE><P>Currently, approximately 170 SPs belonging to the Sec pathway of <I>B. subtilis</I> have been annotated but unfortunately, no correlations on sequence levels have been identified that link efficient protein secretion with a distinct SP. <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16930615">[4]</A></P><P>As part of our project EncaBcillus, we aimed at creating a platform for heterologous protein overexpression combined with their efficient secretion without releasing any cells into the environment. Hereby, we encapsulated our model organism <I>B. subtilis</I> in the Peptidosomes which serve as a new innovative method to keep the producing bacteria separated from the desired compounds. (For more details see <A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes">Peptidosomes</A>.)</P><P>To address the vision of creating this new platform, we first wanted to establish a fast and easy screening procedure to evaluate all combinational possibilities of each SP with a protein of interest (POI) – The Signal Peptide Toolbox.</P><P>As SPs of <I>B. subtilis</I> have been successfully adapted to GRAM-positive <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27927208">[5]</A> and GRAM-negative bacteria <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27516938">[6]</A>, the Signal Peptide Toolbox, as an organism-independent genetic measurement tool, can be applied to any bacterial host. Every future iGEM team will be able to use this fully partsregistry availabe tool to increase their protein secretion.</P></FIGURE></DIV><DIV class="contentbox"><H1 id="design" class="box-heading">Design</H1><P><I>Bacillus subtilis</I> contains approximately 170 Sec Signal Peptides (SPs) that re-direct proteins for secretion via the Sec pathway. For our toolbox we were able to clone 74 Sec SPs (Table 1). Each SP was amplified from genomic DNA of <I>B. subtilis</I> wild type with the primers found in the primer collection table at the end of the Design section. After amplification, each SP was digested using the restriction enzymes EcoRI and PstI, stored into the pSB1C3 backbone and submitted to the partsregistry.
</P><FIGURE class="tablefigure" style="width: 100%;" id="BioBricks"><FIGCAPTION><B>Table 1: Submitted Sec SPs of <I><B>B. subtilis</B></I>.
</B></FIGCAPTION><TABLE style="width: 100%;"><TBODY><TR><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273023">AmyE</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273024">AspB</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273025">BglS</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273027">Bpr</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273026">CccA</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273032">CitH</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273023">Csn</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273029">DacB</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273030">DacF</A></TD><TD style="padding: 5px; width:10%; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273031">DltD</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273004">Epr</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273005">FliL</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273006">FliZ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273007">GlpQ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273008">LipA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273009">LytB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273010">LytC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273011">LytD</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273043">LytR</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273044">Mdr</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273046">Mpr</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273048">NprE</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273051">PbpX</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273052">Pel</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273053">PelB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273054">PenP</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273055">PhoA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273056">PhoB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273057">PhrA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273058">PhrC</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273059">PhrF</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273060">PhrG</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273061">PhrK</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273062">RpmG</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273063">SacB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273064">SacC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273065">SleB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273066">SpoIID</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273067">SpoIIP</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273068">SpoIIQ</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273069">SpoIIR</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273070">TyrA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273071">Vpr</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273072">WapA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273073">YbbC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273074">YbbE</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273075">YbbR</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273076">YbdG</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273077">YbdN</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273078">YbfO</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273079">YbxI</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273080">YdbK</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273081">YdhT</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273082">YdjM</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273083">YdjN</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273084">YfhK</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273085">YfjS</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273086">YfkD</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273087">YfkN</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273088">YhcR</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273089">YhdC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273090">YhfM</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273091">YhjA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273092">YjcN</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273093">YjdB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273094">YjfA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273095">YjiA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273096">YkoJ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273097">YkvV</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273098">YkwD</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273099">YlaE</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273100">YlbL</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273101">YlqB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273102">YlxF</A></TD></TR></TBODY></TABLE></FIGURE><P>This powerful collection of SPs can now be combined with our <A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part">Evaluation Vector</A> and any protein of interest (POI) for a shotgun cloning approach to identify the best combination of SP and POI.
</P><P>Though having access to these SPs via the registry, does not solve the problem having to create one clone for each single SP and POI. This also means, sufficient amounts of each SP are necessary to guarantee efficient cloning. Thus, we followed up on the idea of <B>multi-template PCR</B>. Since all SPs are stored in the pSB1C3 vector and flanked by the same pre- and suffix sequences, we should be able to amplify all SPs via PCR using the BioBrick pre- and suffix as primers.
To cope with this issue of having 74 SPs, we decided to break down the 74 SP into Signal Peptide Mixes (SPM), each consisting of maximal 20 SPs (Table 2). We have carefully evaluated the maximal number of SPs within each mix to increase the robustness of the PCR (see the Results section for more details). Followed by the amplification of all SPM subsets, they can be easily purified and applied in a shotgun ligation approach with the protein of interest and the EV. </P><P id="threeproteins">We demonstrate the applicability of this approach with three different proteins: the α-Amylase of <I>B. subtilis</I>, sfGFP and mCherry. For each protein we applied a protein dependent assay and tested supernatants of positive transformed <I>B. subtilis</I> clones. We correlated protein activities with the secretion efficiency. In a final step, we sequenced strains which showed highest protein activities in order to identify the SPs. Thus, we were able to identify the most potent combination for a distinct SP with the given POI. 
</P><P>Detailed methods can be found in our <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">protocol collection section</A>. All primers used can be found in our primer collection table down below.
</P><P><A class="pdf-resources" href="https://static.igem.org/mediawiki/2017/b/b9/T--TU_Dresden--M_Primerlist.pdf">Primer collection table</A></P></DIV><DIV class="contentbox"><H1 id="protocol" class="box-heading">Results</H1><H3>Amplification of the Signal Peptide Mixes via optimized multi-template PCR</H3><P>So far, no direct correlation between the perfect combination of signal peptide and downstream sequence to gain optimal secretion levels is known. Thus, the problem of having to create one clone per combination of SP and protein of interest remains. Therefore, we created the so-called Signal Peptide Mixes (SPMs), a set of libraries with each containing equal concentrations of up to twenty distinct SPs which can be easily enriched via multi-template PCR. The amplified SPs can then be combined with our Signal Peptide Evaluation Vector (SP-EV) and the gene of interest. (For more details see the protocol in the end of the Results section.)</P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 49%;"><FIGCAPTION><B>Figure 3: PCR results from amplifying a SPM containing twenty SPs.</B> All twenty distinct SPs could be amplified in individual reactions from the same template.  Primers used were specific for each SP. ZR 50bp DNA ladder from Zymo Research was used.</FIGCAPTION></FIGURE><FIGURE class="makeresponsive floatright" style="width: 34.8%;"><FIGCAPTION><B>Figure 2: PCR evaluation to identify the maximum number of SP in a SPM.</B><B>A</B> PCR of SPMs consisting of 53 distinct SPs, shown in triplicates. PCR artifacts are visible at 250 bp. <B>B</B> SPMs consisting only of 20 distinct SPs. We varied primer concentrations (µM) to enhance the PCR reaction and adjusted the template amounts (v-%). Set 1 = 1.5µM / 3%, Set 2 = 1µM / 3%, Set 3 = 1.5µM / 2%, Set 4 = 1µM / 2%. ZR 50bp DNA ladder from Zymo Research was used.</FIGCAPTION></FIGURE><P>In a first approximation, we evaluated the multi-template PCR by varying the number of different SPs in one mix. Our aim was to amplify all SPs equally for the downstream cloning procedures. To test this, a SPM subset containing 53 SPs, was amplified using the RFC10 prefix and suffix as primers, we expected a band at around 100-200 bp (size range of the SPs). Unfortunately, we also observed a second dominant band (at about 250 bp) (Figure 2, A), which most likely derives from PCR artifacts. Thus, we concluded that a SPM consisting of 53 SPs was not suitable for our purposes.</P><P>From this we decided to reduce the number of different SPs to a maximum of 20 and also to increase the primer concentrations in the PCR reaction. These improvements lead to a specific amplification of our SPs (Figure 2, B). To double check, if all 20 SPs were indeed amplified, we conducted a second PCR using the first PCR as template with specific primers for each SP in the corresponding SPM. We could show that all 20 distinct SPs of the SPM subset were amplified during the first PCR (Figure 3). From these observations, we decided to split up the 74 SPs into subset-mixes of each containing up to 20 SPs max.</P></FIGURE><P>All 74 SPs which we provide were therefore aliquoted to 0.5 ng/μL and assigned to one (a, b, c or d) SPM subsets. The table below gives an overview.</P><FIGURE class="makeresponsive" style="width: 100%;"><FIGCAPTION><B>Table 2: SPM subsets a-d of all submitted Sec-dependent SPs.</B></FIGCAPTION><FIGURE class="makeresponsive floatleft" style="width:50%"><TABLE class="makeresponsive" style="width:100%;"><TBODY><TR><TD colspan="10" style="padding: 5px; width:100%;"><P>SPM subset a</P></TD></TR><TR><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273023">AmyE</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273024">AspB</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273025">BglS</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273027">Bpr</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273026">CccA</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273032">CitH</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273023">Csn</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273029">DacB</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273030">DacF</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273031">DltD</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273004">Epr</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273005">FliL</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273006">FliZ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273007">GlpQ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273008">LipA</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273009">LytB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273010">LytC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273011">LytD</A></TD></TR></TBODY></TABLE></FIGURE><FIGURE class="makeresponsive floatright" style="width:50%"><TABLE class="makeresponsive" style="width:100%;"><TBODY><TR><TD colspan="10" style="padding: 5px; width:100%;"><P>SPM subset b</P></TD></TR><TR><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273043">LytR</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273044">Mdr</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273046">Mpr</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273048">NprE</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273051">PbpX</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273052">Pel</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273053">PelB</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273054">PenP</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273055">PhoA</A></TD><TD style="padding: 5px; border: 1px solid; width:5%;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273056">PhoB</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273057">PhrA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273058">PhrC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273059">PhrF</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273060">PhrG</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273061">PhrK</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273062">RpmG</A></TD></TR></TBODY></TABLE></FIGURE><FIGURE class="makeresponsive floatleft" style="width:50%"><TABLE class="makeresponsive" style="width:100%;"><TBODY><TR><TD colspan="10" style="padding: 5px; width:100%;"><P>SPM subset c</P></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273063">SacB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273064">SacC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273065">SleB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273066">SpoIID</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273067">SpoIIP</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273068">SpoIIQ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273069">SpoIIR</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273070">TyrA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273071">Vpr</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273072">WapA</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273073">YbbC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273074">YbbE</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273075">YbbR</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273076">YbdG</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273077">YbdN</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273078">YbfO</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273079">YbxI</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273080">YdbK</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273081">YdhT</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273082">YdjM</A></TD></TR></TBODY></TABLE></FIGURE><FIGURE class="makeresponsive floatright" style="width:50%"><TABLE class="makeresponsive" style="width:100%;"><TBODY><TR><TD colspan="10" style="padding: 5px; width:100%;"><P>SPM subset d</P></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273083">YdjN</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273084">YfhK</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273085">YfjS</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273086">YfkD</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273087">YfkN</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273088">YhcR</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273089">YhdC</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273090">YhfM</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273091">YhjA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273092">YjcN</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273093">YjdB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273094">YjfA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273095">YjiA</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273096">YkoJ</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273097">YkvV</A></TD></TR><TR><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273098">YkwD</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273099">YlaE</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273100">YlbL</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273101">YlqB</A></TD><TD style="padding: 5px; border: 1px solid;"><A target="_blank" href="http://parts.igem.org/Part:BBa_K2273102">YlxF</A></TD></TR></TBODY></TABLE></FIGURE></FIGURE><H3 id="anyorganism">Cloning Signal Peptides with the Signal Peptide-Evaluation Vector</H3><P>Following the evaluation of the multi-template PCR amplification of the SPs, we established a Standard Operating Procedure (SOP) protocol for cloning the SPs using the Signal Peptide-Evaluation Vector (SP-EV). This SOP describes the high-throughput approach of screening SPs with a POI. While our project is based in <I>B. subtilis</I> and we use <I>E. coli</I> as cloning host, we believe this SOP can be applied to any organism which is able to perform secretion via the Sec pathway.</P><FIGURE><FIGURE class="makeresponsive floatleft" style="width:60%; margin-bottom:-10px;"><FIGCAPTION><B>Figure 4: Animated cloning stages using our EV.</B> The promoter can be exchanged via EcoRI and BsaI (=results in XbaI overhang), to insert Signal Peptides use XbaI and AgeI and use NgoMIV and PstI to exchange of the gene of interest.</FIGCAPTION></FIGURE><P>We provide the EV with a xylose inducible promoter (P<SUB><I>xylA</I></SUB>), which is well characterised for the use in <I>B. subtilis</I>. All the following experiments were performed using this promoter to drive our expression. As a first approach, we wanted to investigate if we could boost the secretion of the native <I>amyE</I> in <I>B. subtilis</I>. We amplified <I>amyE</I> (without its native signal peptide) using the primers iG17P180 and iG17P062, restriction enzymes interfering with the RFC25 standard were removed via mutagenesis PCR using iG17P057 - iG17P062. After sequencing, we stored the biobricked RFC25 compatible <I>amyE</I> version in the pSB1C3 backbone and submitted it to the parts registry (<A target="_blank" href="http://parts.igem.org/Part:BBa_K2273103">BBa_K2273103</A>). We then sub-cloned this part into our EV, via NgoMIV and PstI, replacing the <I>lacZα</I>. Next, we amplified all SPs from each SPM subset using the RFC10 (pre- and suffix) sequences as primers (TM4487 and iG17P039). After the amplification, each subset was digested using XbaI and AgeI. Now, we had everything ready to perform our shotgun ligation approach: we digested the EV (containing the BioBrick <I>amyE</I>) with BsaI and AgeI and ligated equal molar amounts with each SPM subset (a, b, c and d). The ligation mixes were directly <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">transformed into <I>B. subtilis</I></A>.
</P></FIGURE><P>The detailed SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox can be found down below at the end of the Results section &quot;High throughput screening procedure for <I>B. subtilis</I>&quot;. The GIF (Figure 4) above summarizes graphically the steps neccessary to set up your individual SP-EV.</P><H3 id="secretion">High throughput screening procedure for <I><B>B. subtilis</B></I></H3><P>After various adjustments to improve the applicability of the Signal Peptide Toolbox, we developed a high throughput screening procedure tailored to fit our model organism <I>B. subtilis</I> and proceeded to identify the most potent SPs for highest secretion of <I>B. subtilis'</I>α-Amylase.</P><P>Since we wanted to evaluate amylase secretion efficiency, we performed our transformation into a starch degradation-deficient <I>B. subtilis</I> strain (TMB3547). This strain contains a disruption of the <I>amyE</I> gene, due to the insertion of P<SUB><I>veg</I></SUB>-<I>lacZ</I>. Fortunately, the strain still contains the necessary flanking regions for homologous recombination of the pBS1C vector. Thus, positive integration of the pBS1C-SPM-<I>amyE</I> construct lead to white colonies, when plated on X-Gal containing agar plates. (Figure 5, A)</P><P>We obtained colonies from each transformation (SPM a, b, c or d with <I>amyE</I> in the EV) and transferred colonies to: a starch containing screening agar plate (to check for vector integration into the <I>B. subtilis</I> genome) and a second backup agar plate which was spiked with chloramphenicol (to maintain each colony). In our setup, we included a negative control (TMB3547, non starch degrading) and a positive control (W168, native <I>amyE</I> and thereby native amylase secretion).</P><FIGURE><FIGURE class="makeresponsive floatright" style="width:70%;"><FIGCAPTION><B>Figure 5: Handling of <I><B>B. subtilis</B></I> after transformation shown on agar plates.</B><B>A</B> The transformation agar plate (chloramphenicol and X-Gal). No blue colonies indicate a high transformation efficiency. <B>B</B> The screening agar plate. Zones of degradation indicate successful integration events. The negative control (TMB3547) has been marked with a black box, the positive control (W168 wild type) with a white box respectively. <B>C</B> The backup agar plate containing chloramphenicol. Neither the negative control (black box), nor the positive control (white box).</FIGCAPTION></FIGURE><P>To verify positive integration of our EV-SPM-<I>amyE</I> constructs, we poured <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Lugol’s Iodine solution</A> on the screening plates, which where spiked with starch. Normally, the integration of the EV into the <I>amyE</I> locus of <I>B. subtilis</I>, results in a disruption of the native gene leading to a loss of the enzymatic activity. Therefore, successfully transformed clones would not be able to degrade starch visualisable by no brightened zone of degradation (Figure 5, B black box). But as we performed our transformation into a starch degradation-deficient <I>B. subtilis</I> strain and used amylase as POI, successfully transformed colonies were again able to degrade starch. Thereby, a brightened zone of degradation on the screening agar plate indicated promising colonies (Figure 5, B e.g. top row). The position of successfully transformed colonies was then marked on the backup agar plates (Figure 5, C). Surprisingly, some chloramphenicol resistance clones did not show any zone of starch degradation (compare clone 4 on Figure 5, C and check with the same position on Figure 5, B). We believe that in these cases, only single homologous recombination events occurred.</P><P>Having now identified clones with positive integrated EV-SPM-<I>amyE</I> constructs, we proceeded with testing their supernatants in an amylase enzyme assay. <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16500607">[8]</A> We therefore, incubated approximately 30 clones of each transformation (with the four different SPM subset) in a 96 well plate using <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">2xYT medium</A>. We also included the negative control (TMB3547) and the wild type. We inoculated the 96 well plates and incubated the <I>B. subtilis</I> cultures for 8 hours at 37°C with 220 rpm using the plate reader.</P></FIGURE><FIGURE><FIGURE class="makeresponsive floatright" style="width:60%;"><FIGCAPTION><B>Figure 6: Screening of signal peptides in front of <I><B>amyE</B></I>.</B> Results of the plate reader based starch hydrolysis assay to identify the most potent combinations of SPs and <I>B. subtilis’</I>α-Amylase of each SPM subset. The shown data was normalized over OD<SUB>600</SUB>. Data shown in <B>A</B>, <B>B</B>, <B>C</B> and <B>D</B> represent clones derived from SPM subsets a, b, c and d.</FIGCAPTION></FIGURE><FIGURE class="makeresponsive floatright" style="width:60%;"><FIGCAPTION><B>Figure 7: Sequenced signal peptides in front of <I><B>amyE</B></I>.</B> Fold change in secretion efficiency (amylase activity) over wild type. Depicted candidates were identified by sequencing.</FIGCAPTION></FIGURE><P>After the first 8 hours of incubation, we reinoculated the cells in fresh 2xYT medium supplemented with a final concentration of 1% xylose to induce the expression of our constructs driven by P<SUB><I>xylA</I></SUB>. We incubated the <I>B. subtilis</I> cultures for 16 hours at 37°C with 220 rpm in the plate reader.</P><P>Before separating the supernatant from the cells via centrifuging the 96 well plates, we measured the OD<SUB>600</SUB> to normalize our data in the end (assuming a direct correlation between bacterial density and secreted protein). Using the harvested supernatants, we applied a microplate reader based starch hydrolysis assay <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16500607">[8]</A> and normalized the generated data with the OD<SUB>600</SUB> values to identify the most potent combinations of SPs and <I>B. subtilis’</I>α-Amylase of each SPM subset (Figure 6).</P><P>Finally, we picked a range of promising colonies and amplified the SP containing region via standard PCR using the primers TM4487 and iG17P058. The resulting fragments were sequenced, thus we could identify the most potent SPs for α-Amylase secretion (Figure 7).</P><P>We were very much excited to identify several SP which lead to a almost 1.5 fold higher amylase activity when compared to the wilde type! Interestingly we also identified the <I>amyE</I> SP as more potent than the wild type. We explain this, by the way we cloned our construct: due to the RFC25 we by default added two amino acids (AA) (from the scar site derived from the fusion of NgoMIV and AgeI) between the SP and <I>amyE</I>. Apparently, by having this two AA linker, the activity of Amylase could be enhanced.</P><P>We also identified SP which lead to less Amylase secretion. Overall, we were able to demonstrate a fully functional high-throughput approach to identify most potent partners between Sec dependant SP and a POI.
</P><P>We also went one step further and proved the applicability of the Signal Peptide Toolbox by evaluating two more proteins: sfGFP and mCherry, according to the procedure described above.</P><P>The detailed SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox can be found down below. The protocol selection explains the use of the <A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part">Evaluation Vector</A>, the Signal Peptide-Evaluation Vector and the high throughput screening procedure for <I>B. subtilis</I>.</P></FIGURE><P><A class="pdf-resources" href="https://static.igem.org/mediawiki/2017/d/db/SOP_protocols_for_cloning_with_the_Evaluation_Vector_and_the_Signal_Peptide_Toolbox.pdf">SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox</A></P><H3>Identification of the most potent Signal Peptides for <I><B>sfGFP</B></I> and <I><B>mCherry</B></I></H3><P>The screening procedure for identifying the best SPs for highest secretion of sfGFP and mCherry were conducted as stated in the detailed SOP protocols for working with the Evaluation Vector and the Signal Peptide Toolbox above.</P><P>For <I>sfGFP</I>, we applied a microplate reader based fluorescence assay where we performed an endpoint measurement at wavelengths set to 480 nm for excitation and 510 nm emission and normalized the generated data via the clones' OD<SUB>600</SUB> values to identify the most potent combinations of SPs and <I>sfGFP</I>. For <I>mCherry</I>, we performed a second microplate reader based fluorescence assay, an endpoint measurement at wavelengths set to 585 nm for excitation and 615 nm for emission. The data was normalized over the clones' OD<SUB>600</SUB> values. In a final step, we identified the most potent SP‑<I>sfGFP</I> and SP‑<I>mCherry</I> combinations via sequencing (Figures 8, 9).</P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 61.3%;"><FIGCAPTION><B>Figure 9: Sequenced signal peptides in front of <I><B>mCherry</B></I>.</B> Fold change in secretion efficiency (fluorescence) over wild type. Depicted candidates were identified by sequencing.</FIGCAPTION></FIGURE><FIGURE class="makeresponsive floatleft" style="width: 38.7%;"><FIGCAPTION><B>Figure 8: Sequenced signal peptides in front of <I><B>sfGFP</B></I>.</B> Fold change in secretion efficiency (fluorescence) over wild type. Depicted candidates were identified by sequencing.</FIGCAPTION></FIGURE></FIGURE><H2 id="conclusion">Conclusion</H2><P class="survey-quote"><B>Our team developed and proved the applicability of a powerful toolbox to quickly screen via a high throughput procedure for improved secretion of proteins in bacteria - the Signal Peptide Toolbox. We are very much sure that the vision to facilitate Peptidosomes as protein production platform can be achieved. The promising combination of increased protein secretion and physical separation of production host and end-product has endless possible applications.</B></P></DIV><DIV class="contentbox"><H1 class="box-heading">References</H1><TABLE class="source"><TBODY><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/23311580">[1]</A></TD><TD>van Dijl J. M. and Hecker M. (2013) <I>Bacillus subtilis</I>: from soil bacterium to super-secreting cell factory. <I>Mircobial Cell Factories</I>, 3 (1-6).</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/17938627">[2]</A></TD><TD>Papanikou E., Karamanou S. and Economou A. (2007) Bacterial protein secretion through the translocase nanomachine. <I>Nature Reviews Microbiology</I>, 11 (839-851).</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16997527">[3]</A></TD><TD>Fu L. L., Xu Z. R., Li W. F., Shuai J. B., Lu P. and Hu C. X. (2006) Protein secretion pathways in <I>Bacillus subtilis</I>: implication for optimization of heterologous protein secretion. <I>Biotechnology advances</I> 1 (1-12).</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16930615">[4]</A></TD><TD>Brockmeier U., Caspers M., Freudl R., Jockwer A., Noll T. and Eggert T. (2013) Systematic screening of all signal peptides from <I>Bacillus subtilis</I>: a powerful strategy in optimizing heterologous protein secretion in Gram-positive bacteria. <I> Journal of molecular biology</I>, 3 (393-402).</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27927208">[5]</A></TD><TD>Hemmerich J., Rohe P., Kleine B., Jurischka S., Wiechert W., Freudl R. and Oldiges M. (2016) Use of a Sec signal peptide library from <I>Bacillus subtilis</I> for the optimization of cutinase secretion in <I>Corynebacterium glutamicum</I>. <I>Mircobial Cell Factories</I>, 208.</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/27516938">[6]</A></TD><TD>Pechsrichuang P., Songsiriritthigul C., Haltrich D., Roytrakul S., Namvijtr P., Bonaparte N., Yamabhai M. (2016) OmpA signal peptide leads to heterogenous secretion of <I>B. subtilis</I> chitosanase enzyme from <I>E. coli</I> expression system. <I>Springerplus</I>, 1200.</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/24295448">[7]</A></TD><TD>Radeck, J., Kraft, K., Bartels, J., Cikovic, T., Dürr, F., Emenegger, J., Kelterborn, S., Sauer, C., Fritz, G., Gebhard, S., and Mascher, T. (2013) The <I>Bacillus</I> BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with <I>Bacillus subtilis</I>.<I> J Biol Eng 7</I>, 29.</TD></TR><TR><TD><A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/16500607">[8]</A></TD><TD>Xiao Z., Storms R. and Tsang A. (2007) A quantitative starch-iodine method for measuring alpha-amylase and glucoamylase activities. <I>Analytical Biochemistry</I>, 1 (146-148).</TD></TR></TBODY></TABLE></DIV></MAIN></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end;">Beam me up!</A></DIV><DIV class="footerbar"><TABLE><TBODY><TR><TD><A href="https://www.google.de/maps/place/Zellescher+Weg+20,+01217+Dresden" target="_blank">iGEM-Team TU Dresden<SMALL>Institut für Mikrobiologie, AG Thorsten MascherZellescher Weg 20b, 01217 Dresden</SMALL>Germany</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><A href="https://www.facebook.com/igem.tud2017" target="_blank">@igem.tud2017</A></TD></TR><TR><TD><A href="https://www.instagram.com/igem_tud_2017" target="_blank">@igem_tud_2017</A></TD></TR><TR><TD><A href="mailto:igem2017@mailbox.tu-dresden.de">igem2017@mailbox.tu-dresden.de</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>