<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Dresden_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Dresden/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV><DIV class="wrapper"><UL class="navbar"><LI><A href="https://2017.igem.org/Team:TU_Dresden/Description" class="nav-text dropbtn">Project</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Peptidosomes" class="nav-text">Peptidosomes</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Biosensor" class="nav-text">β-Lactam Biosensor</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Measurement" class="nav-text">Signal Peptide Toolbox</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Composite_Part" class="nav-text">Evaluation Vector</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Secretion" class="nav-text">Secretion</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Project/Communication" class="nav-text">Communication</A></LI></UL><LI><A href="https://2017.igem.org/Team:TU_Dresden/Results" class="nav-text dropbtn">Results</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Demonstrate" class="nav-text">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Parts" class="nav-text">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Human_Practices" class="nav-text dropbtn">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Silver" class="nav-text">Outreach</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Integrated</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Collaborations" class="nav-text">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green" class="nav-text dropbtn">iGEM goes green</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams" class="nav-text">Green Teams</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Resources" class="nav-text">Resources</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/HP/Gold_Integrated" class="nav-text">Our Achievements</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Experience" class="nav-text">Our Experience</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/About_Us" class="nav-text dropbtn">About Us</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Team" class="nav-text">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Attributions" class="nav-text">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Sponsors" class="nav-text">Financing &amp; Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Lab_Work" class="nav-text dropbtn">Lab Work</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Experiments" class="nav-text">Experiments &amp; Protocols</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Safety" class="nav-text">Safety</A></LI><LI><A href="https://2017.igem.org/Team:TU_Dresden/Judging" class="nav-text dropbtn">Judging</A></LI></DIV></NAV><DIV id="pagebanner" style="background-image: url(https://static.igem.org/mediawiki/2017/6/62/Tudresden_bestbasicpart.jpeg);"><DIV id="bannerspace"><DIV id="bannerquote">Our best basic part</DIV></DIV><MAIN><DIV class="contentbox"><H1 class="box-heading">Introduction</H1><P>As part of the EncaBcillus project, we developed a novel and complete heterologous biosensor for  β-lactam antibiotics in <I>Bacillus subtilis</I>. This biosensor is based on a one-component system encoded in the so-called <I>bla-operon</I> naturally found in <I>Staphylococcus aureus</I>. The biosensor is composed of three composites from this operon: The  β-lactam receptor BlaR1 receptor and the repressor BlaI which have been codon-adapted for expression in <I>B. subtilis</I> as well as the <A target="_blank" href="http://parts.igem.org/Part:BBa_K2273111">P<SUB><I>blaZ</I></SUB> promoter [BBa_K2273111]</A>(see Figure 2). This promoter was inserted upstream of the <I>lux</I>-operon, our reporter of choice. Figure 1 displays the molecular mechanism of the established biosensor. In case a  β-lactam is bound to BlaR1, the receptor`s proteolytic c-terminal domain degrades the BlaI repressor, thereby releasing the P<SUB><I>blaZ</I></SUB> promoter. This enables binding of the transcription machinery to the promoter and therefore the expression of the <I>luxABCDE</I> genes, resulting in a luminescence signal produced by the bisosensor. </P><DIV style="display: flex; align-items: flex-start; flex-wrap: wrap;"><DIV class="makeresponsive" style="width:50%;"><FIGURE class="makeresponsive" style="padding-left:10%; padding-right:10%;"><FIGCAPTION><B>Figure 1: Overall concept showing the components and the molecular mechanism of the biosensor in <I><B>B. subtilis</B></I></B>. Upon binding of a β-lactam to the receptor BlaR1 <B>(1)</B>, due to the receptors c-terminal proteolytic activity, the repressor BlaI is degraded and frees the target promoter <B>(2)</B> enabling the expression of an easily detectable reporter <B>(3)</B>.
</FIGCAPTION></FIGURE></DIV><DIV class="makeresponsive" style="width:50%;"><FIGURE class="makeresponsive floatleft" style="padding-left:10%; padding-right:10%;"><FIGCAPTION><B>Figure 2: Genetic constructs necessary for the functional biosensor strain.</B><B>(A)</B><I>blaR1</I> under control of the constitutive promoter P<SUB><I>veg</I></SUB> or the inducible promoter P<SUB><I>xylA</I></SUB>,<B>(B)</B><I>blaI</I> downstream of the promoter P<SUB><I>lepA</I></SUB>,  and <B>(C)</B> the promoters P<SUB><I>blaZ</I></SUB> or P<SUB><I>blaR1I</I></SUB> controlling the expression of the <I>luxABCDE</I> operon.</FIGCAPTION></FIGURE></DIV></DIV><P>This biosensor project turned out to be successful as our biosensor showed a great performance in all conducted experiments. For this reason, we created this section to apply for “best basic part” with the <A target="_blank" href="http://parts.igem.org/Part:BBa_K2273111">P<SUB><I>blaZ</I></SUB> promoter [BBa_K2273111]</A>. As this promoter showed high activity and reliability when induced by β-lactams, a clear differentiation between background and the desired signal was possible. The results demonstrated in the paragraphs below, validate the functionality of the biosensor and thus also the functionality of its composites.</P></DIV><DIV class="contentbox"><H1 class="box-heading">Proving the functionality of P<SUB><I>blaZ</I></SUB></H1><H3>1. Assessing the activity of P<SUB><I><B>blaZ</B></I></SUB> in liquid medium</H3><FIGURE><FIGURE class="makeresponsive floatright" style="width: 40%"><FIGCAPTION><B>Table 1: Antibiotic concentrations in [µg µl<SUP>-1</SUP>] (final concentration in the well) used in all further plate reader experiments.
</B></FIGCAPTION></FIGURE><P>First, we investigated the detection range towards different β-lactam families as well as the sensitivity of the created biosensor. Therefore, we conducted plate reader experiments to test our biosensor in liquid conditions. We recorded the luminescence signal and growth behavior (see <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Experiments and Protocols</A> for details) of our biosensor strains in the presence of six different β-lactam antibiotics. We also included physiological controls that lack one or two of the genetic constructs of the complete biosensor machinery (data not shown).  
Furthermore, we analyzed the impact of deleting the <I>Bacillus subtilis</I> gene <I>penP</I> - encoding a β-lactamase (which has not been studied intensively yet) - on the luminescence output. The strain W168 <I>penP::kan<SUP>R</SUP></I> was created via Long-Flanking Homology PCR (see <A href="https://2017.igem.org/Team:TU_Dresden/Experiments">Experiments and Protocols</A> for details). </P><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 70%"><FIGCAPTION><B>Table 2: Strains of interest with their names and important genotype remarks for differentiation.</B></FIGCAPTION></FIGURE><P>The control strain W168 (wild type) and control 1, will presumably not show any luminescence output, while the positive control 2 is expected to show a steady luminescence signal regardless of the presence of any antibiotic compound. Generally, the P<SUB><I>xyl</I></SUB> promoter is activated by adding xylose as an inducer to the medium. In the plate reader experiments, 0.2% xylose was added to the cells (beginning with the day culture) to activate expression of <I>blaR1</I>.</P><P>Further we propose biosensor strains carrying the genotype remark <I>penP::kan<SUP>R</SUP></I> to give a stronger signal in presence of β-lactam compounds, as they cannot be degraded by the PenP enzyme.</P></FIGURE><P>We could not observe a substantial activation of the P<SUB><I>blaR1I</I></SUB> promoter by the β-lactam compounds, which is why we are not taking it into account in the evaluation below. The bar charts in Figure 3 illustrate the best biosensor constructs identified in the plate reader experiments and compare the RLU/OD<SUB>600</SUB> values of the strains 2 hours post induction with the antibiotics.</P><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 55%"><FIGCAPTION><B>Figure 3: RLU/OD<SUB><I>600</I></SUB> values of the different biosensors and the controls are shown 2 hours after induction with the six β-lactams, bacitracin and dH<SUB><B>2</B></SUB>O</B>. Graphs show the Wild-type (<B>black</B>), control 1 (<B>light gray</B>), control 2 (<B>dark gray</B>), biosensor 1 (<B>pink</B>), biosensor 2 (<B>purple</B>), biosensor 3 (<B>white and black</B>) and biosensor 3 Xylose induced (<B>dark blue</B>). Luminescence (RLU/OD<SUB>600</SUB>) output is shown two hours after β-lactam antibiotic induction. Mean values and standard deviation are depicted from at least three biological replicates. </FIGCAPTION></FIGURE><P>As shown in Figure 3, the wildtype W168 (black with white dots) shows no increase in RLU/OD<SUB>600</SUB> values when induced with the different β-lactam antibiotics and controls. Control 1 (black tight stripes) behaves similarly to the wild type strain. The slight decrease of control 2 (light grey) in the bar chart where induction with ampicillin and carbenicillin happened is mostly explained by the high growth inhibition caused by the chosen concentrations for W168 (with functional PenP). Most of the times, the constitutive expression of the <I>lux</I> operon resulted in an RLU/OD<SUB>600</SUB> of over 1.3 million for control 2 (see Figure 3).</P><P>Biosensor 1 gives an overall good signal for all β-lactam antibiotics tested, but also shows a higher basal activity in absence of the β-lactam compounds of 40.000- 90.000 RLU/OD<SUB>600</SUB> (see Figure 3, bar chart with bacitracin and dH<SUB>2</SUB>O). Further, we could observe a difference in signal intensity dependent on the β-lactam antibiotic tested.  Therefore, biosensor 1 gives the highest signal in presence of penicillin G, cefoxitin and cefoperazone with up to 2.7 million RLU/OD<SUB>600</SUB>. Ampicillin and penicillin G again show a weaker increase in signal produced by biosensor 1, which could be due to the same reason as for control 2 (see Figure 3).</P><P>For biosensor 2, the detection range and sensitivity is comparable to biosensor 1. This strain strongly senses cefoxitin, ampicillin and cefoperazone reaching up to 2.4 million RLU/OD<SUB>600</SUB>. Even the basal activity of the P<SUB><I>blaZ</I></SUB> promoter in biosensor 2, as shown in the bar charts with bacitracin and dH<SUB>2</SUB>O, conforms with the one from biosensor 1.</P></FIGURE><H3>2. Investigating the promoter activity on solid medium</H3><FIGURE><FIGURE class="makeresponsive floatleft" style="width: 40%"><FIGCAPTION><B>Table 3: Antibiotic concentrations in [µg µl<SUP>-1</SUP>] (final concentration in the well) used in the Disk Diffusion assays.
</B></FIGCAPTION></FIGURE><P>To determine whether the results obtained in the plate reader assays under liquid conditions can be reproduced on solid agar plates, we performed disk diffusion assays. This assay visualises the detection range of the biosensor by a glowing halo at the edge of the growth inhibition zones, where bacteria are exposed to sub-lethal antibiotic concentrations. We tested the same β-lactam antibiotics and controls as in the previous experiments, but chose higher concentrations since <I>B. subtilis</I> tends to be more resistant against antibiotics when tested on solid media. By increasing the antibiotic concentrations we still obtained slight growth defects but remained in the detection range of the biosensor. The concentrations used in the disk diffusion assays can be extracted from Table 3.</P></FIGURE><FIGURE><FIGURE class="makeresponsive floatright" style="width: 55%"><FIGCAPTION><B>Figure 4: Photographs of the plates from the disk diffusion assay. </B>The upper rows (Panel A and C) show pictures of the plates with the strains under daylight conditions, while the row beneath (Panel B and D) shows the plate after detection of chemiluminescence (2 minutes exposure time). At the bottom in Panel E, the disk layout and the most important remarks of the genotype of all strains are indicated. 
</FIGCAPTION></FIGURE><P>We expected the control substances (water and bacitracin) to not cause any luminescence signal at the edge of the inhibition zones. The β-lactam antibiotics should lead to a glowing halo when tested with the three different biosensor versions. The wildtype strain and control 1 should not show any signal, since both strains are lacking the <I>lux</I> operon. In the case of control 2, a luminescence signal should be spread over the whole plate, due to the constitutive expression (P<SUB><I>veg</I></SUB>) of luciferase. Figure 4 sums up the results of the disk diffusion assay for all strains tested. After 24 hours of incubation at 37°C, plates were photographed under daylight conditions and under a chemiluminescence dock (with two minutes exposure time).</P><P>As expected, the wildtype and control 1, show no luminescence signal, while control 2 leads to a strong luminescence signal spread across the entire plate (Figure 4, Panel B). Neither bacitracin nor dH<SUB>2</SUB>O lead to a detectable output, accounting for all strains tested. While in liquid medium biosensor 1 behaves similar compared to biosensor 2, there is a tremendous difference in the detection capability. Biosensor 2 showed detection for all β-lactams tested (Figure 4, Panel D). Au contraire, biosensor 1 only showed a luminescence signal for cefoperazone, cefoxitin, and cefalexin (Figure 4, Panel D). Further, the luminescence halo around the cefoxitin disk is quite broad compared to the others, indicating an increased diffusion of the compound into the lawn. Although biosensor 1 was activated by penicillin G in liquid medium, we could not observe an induction on plate (Figure 4, Panel D).</P><P>Biosensor 2 was activated by all of the β-lactam compounds tested (Figure 4, Panel D). Ampicillin, cefoxitin, cefalexin, and cefoperazone strongly activate the system, while penicillin G and carbenicillin just show a weak induction of the signal on the plate. These findings go along with the results obtained in liquid medium in the previous experiments.
On the plate with the lawn of biosensor 3 (Figure 4, Panel D), all β-lactams could be detected efficiently when 0.2% xylose was added. In contrast to biosensor 1 and 2, there is a very weak luminescence halo around the cefalexin disk. Also, this halo seems not to de directly at the edge where the cells are in contact with the antibiotic, but rather a bit further off the inhibition zone. Without induction of biosensor 3 with 0.2% xylose, we could not detect any luminescence signal, demonstrating that the receptor (BlaR1) is crucial for detection and signal transduction, standing in line with results obtained in liquid medium (data not shown). The following Table 4 contains the measured diameters of all inhibitions zones caused by the antibiotic. As expected, no inhibition zones around the negative control dH<SUB>2</SUB>O were observed.</P><FIGURE><FIGURE class="makeresponsive floatright" style="width: 100%"><FIGCAPTION><B>Table 4: Measured diameters in mm of the inhibition zones around the disks with the different antibiotics on the plates from the disk diffusion assays.</B></FIGCAPTION></FIGURE></FIGURE><H3>3. Examining the dose-response relationship</H3><FIGURE><FIGURE class="makeresponsive floatright" style="width: 60%"><FIGCAPTION><B>Figure 5: Dose-Response Curves of the six different β-lactam antibiotics of biosensor 2.</B> Observed luminescence signal (two hours after antibiotic exposure) was plotted according to each tested antibiotic concentration. Please note both axes are depicted logarithmic. Mean values and standard deviation are depicted from at least three biological replicates.
</FIGCAPTION></FIGURE><P>As a final characterization of our biosensor, we carried out Dose Response assays in order to investigate the detection sensitivity. In this experiment, eleven different concentrations of the β-lactam antibiotics and bacitracin (control) have been tested together with the biosensor 2. This biosensor was chosen due to its reliable performance in liquid and on solid medium and because all necessary components are constitutively expressed without the addition of any further substances (as with biosensor 3).</P><P>The obtained results (see Figure 5) indicate that biosensor 2 shows the highest dynamic range for cefoperazone and is capable to even sense very low concentrations of this compound.  Biosensor 2 shows the weakest dose response towards cefalexin, which is in agreement with all previous experiments (see Figure 5). The antibiotics cefalexin, ampicillin and penicillin G lead to a decrease in luminescence signal for concentrations higher than 10<SUP>-1</SUP>, due to the growth inhibition at these concentrations (see Figure 5). Overall, a high dynamic range upon antibiotic exposure can be detected for all compounds starting at concentrations above 10<SUP>-3</SUP>.</P></FIGURE><H3 id="peptidosomes">4. Encapsulation of the Biosensor into Peptidosomes – Proving the Application Potential</H3><P>After evaluation of the biosensor, we probed its activity when encapsulated in Peptidosomes. An overnight culture was inoculated in Fmoc-FF-Solution with a final OD600=10. Peptidosomes were prepared containing no bacteria (A), W168 (B), control 2 (C) and biosensor 2 (D) (see Figure 6 below) and underwent 3 washing steps. Afterwards, the Peptidosomes were transferred to a 12-well plate, incubated at 37˚C and luminescence was detected every hour. Induction with 0.2 µg µl<SUP>-1</SUP> cefoperazone happened after 1 hour of growth.
</P><P>The Peptidosomes without cells and the wild-type W168 are expected to show no luminescence signal at all times (A and B). We estimate control 2 to reach a luminescence signal under non-induced as well as under induced conditions (C). This signal should be weaker than that of the induced biosensor 2 (D, +AB). No signal is expected for the encapsulated biosensor in absence of cefoperazone (D, -AB).</P><FIGURE class="makeresponsivet" style="width: 100%"><FIGCAPTION><B>Figure 6: Encapsulation experiment with biosensor 2.</B> The pictures in the upper row show the distribution of the Peptidosomes at the time point of luminescence detection, which was immediately performed afterwards using a chemiluminescence dock (bottom row). Pink arrows indicate Peptidosomes with a luminescence signal deriving from the encapsulated biosensor. The upper row of well plates contains non-induced samples. Lower row of well plates were induced with cefoperazone (0.2 µg µl<SUP>-1</SUP>).
</FIGCAPTION></FIGURE><P>In this experiment, we successfully encapsulated biosensor 2 into Peptidosomes and demonstrated its ability to sense the β-lactam cefoperazone diffusing into the Peptidosome.  Already 2 hours post induction, there is a luminescence signal detectable for control 2 and the encapsulated biosensor 2 (see Figure 6, middle, C and D). Thereby we could validate the hypothesis that antibiotic compounds can enter the Peptidosomes and trigger the activation of the biosensor. We also showed, that the performance of the biosensor is not compromised by the encapsulation.  </P></FIGURE></FIGURE></DIV><DIV class="contentbox"><H1 class="box-heading">Summary</H1><P>Taking together all the results obtained in this project, we can conclude that all three biosensors show excellent functionality under various conditions. All strains are able to detect the six β-lactams, though the biosensors 2 and 3 perform better on solid MH-medium. Generally speaking, the P<SUB><I>blaZ</I></SUB> promoter, as part of the biosensor strains, generates a high luminescence signal that can be easily detected in liquid and on solid media. Further, our results show high reproducibility of the strong promoter activity in the conducted experiments evaluated in the section above.</P><P>Another potential application for the P<SUB><I>blaZ</I></SUB> promoter other than in the context of a biosensor would be in the framework of an expression system. As already very low concentrations of e.g. cefoperazone are leading to strong activation of the promoter by the BlaR1-BlaI system, you could think of replacing the lux-operon by any gene of interest. This promoter reached even higher activities than the constitutive promoter P<SUB><I>veg</I></SUB>. For this reason, we also propose this system for the overexpression of proteins of interest. </P></DIV></MAIN></DIV></DIV></DIV><FOOTER><DIV id="ufo"><A class="nav-text back-to-top" href="#" style="align-self: flex-end;">Beam me up!</A></DIV><DIV class="footerbar"><TABLE><TBODY><TR><TD><A href="https://www.google.de/maps/place/Zellescher+Weg+20,+01217+Dresden" target="_blank">iGEM-Team TU Dresden<SMALL>Institut für Mikrobiologie, AG Thorsten MascherZellescher Weg 20b, 01217 Dresden</SMALL>Germany</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><A href="https://www.facebook.com/igem.tud2017" target="_blank">@igem.tud2017</A></TD></TR><TR><TD><A href="https://www.instagram.com/igem_tud_2017" target="_blank">@igem_tud_2017</A></TD></TR><TR><TD><A href="mailto:igem2017@mailbox.tu-dresden.de">igem2017@mailbox.tu-dresden.de</A></TD></TR></TBODY></TABLE></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>