<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_VictoriaBC_Labprotocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:VictoriaBC/Labprotocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:VictoriaBC/Labprotocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:VictoriaBC/Labprotocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:VictoriaBC/Labprotocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:VictoriaBC/Labprotocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:VictoriaBC/Labprotocols</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TABLE cellspacing="0"><TBODY><TR><TD width="210" bgcolor="#990000" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC">HOME</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#990033" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Project">PROJECT</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#993300" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Overview">OVERVIEW</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#993333" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Design">DESIGN</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#996600" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Testing">TESTING</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#996633" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Results">RESULTS</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#999900" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Conclusions">CONCLUSIONS</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#999933" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Team">TEAM</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#99cc00" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Labnotebook">LAB NOTEBOOK</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#99cc33" valign="top"><CENTER><A href="https://2009.igem.org/Team:VictoriaBC/Labprotocols">LAB PROTOCOLS</A></CENTER></TD></TR><TR><TD width="210" bgcolor="#99ff00" valign="top"><CENTER><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=VictoriaBC">PARTS</A></CENTER></TD></TR></TBODY></TABLE></P><P><B>Lab Protocols</B><A href="javascript:unhide('plates');">Plate Pouring Protocol</A></P><DIV id="plates" class="hidden"><P>For 1 L ~ 60 plates</P><P>20 g LB powder (follow label directions)</P><P>Add 15 g/L agar, fill to 1L water</P><P>Autoclave, let cool till touchable - Don't let solidify!</P><P>Add antibiotic:</P><P>Amp: from 50 mg/mL stock, to a final concentration of 250 ug/mL, so 2.5 mL per 500 mL beaker</P><P>Kan: from 12 mg/mL stock, to a final concentration of 30 ug/mL, so 1.25 mL per 500 mL beaker</P><P>Chl: from 35 ug/mL stock, to a final concentration of the same, so 1 mL per 1 L beaker</P><P>Agitate and pour plates</P></DIV><A href="javascript:unhide('top10');">Top 10 Preparation of Competent Cells Protocol</A><DIV id="top10" class="hidden"><P>Taken from <A href="http://openwetware.org/wiki/TOP10_chemically_competent_cells">http://openwetware.org/wiki/TOP10_chemically_competent_cells</A></P><P> </P><H3> </H3><H3><SPAN class="mw-headline">Preparing seed stocks</SPAN></H3><UL><LI>Streak TOP10 cells on an <A href="http://openwetware.org/wiki/SOB" title="SOB">SOB</A> plate and grow for single colonies at 23°C
    </LI><LI>room temperature works well</LI></UL><LI>Pick single colonies into 2 ml of SOB medium and shake overnight at 23°C
    </LI><LI>room temperature works well</LI><LI>Add glycerol to 15%</LI><LI>Aliquot 1 ml samples to Nunc cryotubes</LI><LI>Place tubes into a zip lock bag, immerse bag into a dry ice/ethanol bath for 5 minutes
    </LI><LI>This step may not be necessary</LI><LI>Place in -80°C freezer indefinitely.</LI><H3> </H3><H3><SPAN class="mw-headline">Preparing competent cells</SPAN></H3><UL><LI>Inoculate 250 ml of <A href="http://openwetware.org/wiki/SOB" title="SOB">SOB</A> medium with 1 ml vial of seed stock and grow at 20°C to an OD600nm of 0.3
    </LI><LI>This takes approximately 16 hours.</LI><LI>Controlling the temperature makes this a more reproducible process, but is not essential.</LI><LI>Room temperature will work. You can adjust this temperature somewhat to fit your schedule</LI><LI>Aim for lower, not higher OD if you can't hit this mark</LI></UL><LI>Centrifuge at 3000g at 4°C for 10 minutes in a flat bottom centrifuge bottle.
    </LI><LI>Flat bottom centrifuge tubes make the fragile cells much easier to resuspend</LI><LI>It is often easier to resuspend pellets by mixing <I>before</I> adding large amounts of buffer</LI><LI>Gently resuspend in 80 ml of ice cold CCMB80 buffer
    </LI><LI>sometimes this is less than completely gentle. It still works.</LI><LI>Incubate on ice 20 minutes</LI><LI>Centrifuge again at 4°C and resuspend in 10 ml of ice cold CCMB80 buffer.</LI><LI>Test OD of a mixture of 200 μl SOC and 50 μl of the resuspended cells.</LI><LI>Add chilled CCMB80 to yield a final OD of 1.0-1.5 in this test.</LI><LI>Incubate on ice for 20 minutes</LI><LI>Aliquot to chilled screw top 2 ml vials or 50 μl into chilled microtiter plates</LI><LI>Store at -80°C indefinitely.
    </LI><LI>Flash freezing does not appear to be necessary</LI><LI>Test competence (see below)</LI><LI>Thawing and refreezing partially used cell aliquots dramatically reduces transformation efficiency by about 3x the first time, and about 6x total after several freeze/thaw cycles.</LI><P> </P><H3><SPAN class="mw-headline">Measurement of competence</SPAN></H3><UL><LI>Transform 50 μl of cells with 1 μl of standard pUC19 plasmid (Invitrogen)
    </LI><LI>This is at 10 pg/μl or 10<SUP>-5</SUP>μg/μl</LI><LI>This can be made by diluting 1 μl of NEB pUC19 plasmid (1 μg/μl, NEB part number N3401S) into 100 ml of TE</LI></UL><LI>Hold on ice 0.5 hours</LI><LI>Heat shock 60 sec at 42C</LI><LI>Add 250 μl <A href="http://openwetware.org/wiki/SOC" title="SOC">SOC</A></LI><LI>Incubate at 37 C for 1 hour in 2 ml centrifuge tubes rotated
    </LI><LI>using 2ml centrifuge tubes for transformation and regrowth works well because the small volumes flow well when rotated, increasing aeration.</LI><LI>For our plasmids (pSB1AC3, pSB1AT3) which are chloramphenicol and tetracycline resistant, we find growing for 2 hours yields many more colonies</LI><LI>Ampicillin and kanamycin appear to do fine with 1 hour growth</LI><LI>Plate 20 μl on AMP plates using sterile 3.5 mm glass beads
    </LI><LI>Good cells should yield around 100 - 400 colonies</LI><LI>Transformation efficiency is (dilution factor=15) x colony count x 10<SUP>5</SUP>/µgDNA</LI><LI>We expect that the transformation efficiency should be between 5x10<SUP>8</SUP> and 5x10<SUP>9</SUP> cfu/µgDNA</LI><P> </P><P> </P><H2><SPAN class="mw-headline">Materials</SPAN></H2><UL><LI>Detergent-free, sterile glassware and plasticware (see procedure)</LI><LI>Table-top OD600nm spectrophotometer</LI><LI><A href="http://openwetware.org/wiki/SOB" title="SOB">SOB</A></LI></UL><H3><SPAN class="mw-headline">CCMB80 buffer</SPAN></H3><UL><LI>10 mM KOAc pH 7.0 (10 ml of a 1M stock/L)</LI><LI>80 mM CaCl<SUB>2</SUB>.2H<SUB>2</SUB>O (11.8 g/L)</LI><LI>20 mM MnCl<SUB>2</SUB>.4H<SUB>2</SUB>O (4.0 g/L)</LI><LI>10 mM MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O (2.0 g/L)</LI><LI>10% glycerol (100 ml/L)</LI><LI>adjust pH DOWN to 6.4 with 0.1N HCl if necessary
    </LI><LI>adjusting pH up will precipitate manganese dioxide from Mn containing solutions.</LI></UL><LI>sterile filter and store at 4°C</LI><LI>slight dark precipitate appears not to affect its function</LI></DIV><A href="javascript:unhide('cells');">Preparation of Competent Cells Protocol</A><DIV id="cells" class="hidden"><P><U>BCMB 301B Protocol</U></P><P> </P><P>Cells must be kept cold at all times.</P><P> </P><P>1.) An overnight culture of E. coli DH5α is subcultured by diluting 1/100 in LB medium and grown in a shaking water bath at 37oC until the OD600 is between 0.4 and 0.6 (2-3 hr).</P><P>2.) Chill 20 mL of this subculture on ice for 15 min.</P><P>3.) Centrifuge cells for 5 min at 7500 rpm. </P><P>4.) Decant supernatant and resuspend pellet in 1/5 growth volume (4 mL) of CM1 buffer.</P><P>5.) Centrifuge cells for 5 min at 7500 rpm. </P><P>6.) Decant supernatant and resuspend pellet in 1/50 growth volume (0.4 mL) of CM2 buffer.</P><P>7.) Transfer two 200 uL aliquots to small sterile test tubes, and two 200 uL aliquots to epi tubes.  Keep competent cells on ice.</P><P> </P><P> </P><P><U>Buffer preparations</U></P><P> </P><P>CM1:</P><P>NaOAc = 0.1 mL of 1 M pH 5.6 stock to produce 10 mM</P><P>MnCl2 = 0.5 mL of 1 M stock to produce 50 mM</P><P>NaCl = 0.2 mL of 1 M stock to produce 5 mM</P><P>Top to 10 mL with water and chill on ice.</P><P> </P><P>CM2:</P><P>NaOAc = 0.1 mL of 1 M pH 5.6 stock to produce 10 mM</P><P>Glycerol = 0.5 mL of 99.5% stock to produce 5%</P><P>MnCl2 = 0.05 mL of 1 M stock to produce 5 mM</P><P>CaCl = 0.7 mL of 1 M stock to produce 70 mM</P><P>Top to 10 mL with water and chill on ice.</P></DIV><A href="javascript:unhide('rehydration');">Rehydration Protocol</A><DIV id="rehydration" class="hidden"><P>Biobrick parts are shipped from the registry in a dehydrated from. As such they must be rehydrated before they can be used.</P><UL><LI>Puncture a hole through the foil with a pipette tip into the well that corresponds to the Biobrick - standard part that you want</LI><LI>Add 15 μL of diH20 (deionized water)</LI><LI>Let the water sit for 5 minutes</LI><LI>Take 2 μL DNA and transform into your desired competent cells, plate out onto a plate with the correct antibiotic and grow overnight. Your goal here is to obtain single colonies.</LI></UL></DIV><A href="javascript:unhide('transformation');">Transformation Protocol</A><DIV id="transformation" class="hidden"><P>1) Add 2ul of rehydrated plasmid to 100ul of competent cells, let sit on ice for 30 minutes</P><P>2) Heat shock at 42 degrees for exactly 2 minutes in a hot water bath</P><P>3) Chill on ice for 1-2 minutes</P><P>4) Add 400ul of pre-warmed LB</P><P>5) Incubate at 37 degrees for 1 hr 15 min under medium agitation for amp plates, and 2 hr for kan and cm plates</P><P>6) Plate 200 ul and incubate at 37 degrees</P><P> </P></DIV><A href="javascript:unhide('broth');">Broth Culture Protocol</A><DIV id="broth" class="hidden"><P>Next Day Broth Culture Protocol:</P><P>A.</P><P>From a single colony on a plate, use a sterile stick to pick and then swirl in 3 mL of LB + antibiotic.</P><P> </P><P>Antibiotic quantities:</P><P>Amp: add 15uL to 3 mL LB</P><P>Kan: add 7.5uL to 3 mL LB</P><P>Cm: add 3uL to 3 mL LB</P><P> </P><P> </P><P>From: <A href="http://www1.qiagen.com/literature/handbooks/literature.aspx?id=1000248&amp;r=1880">http://www1.qiagen.com/literature/handbooks/literature.aspx?id=1000248&amp;r=1880</A></P><P>Pick a single colony from a freshly streaked selective plate and inoculate a culture of</P><P>1–5 ml LB medium containing the appropriate selective antibiotic. Incubate for</P><P>12–16 h at 37°C with vigorous shaking.</P><P>Growth for more than 16 h is not recommended since cells begin to lyse and plasmid</P><P>yields may be reduced. Use a tube or flask with a volume of at least 4 times</P><P>the volume of the culture.</P><P> </P><P>B.</P><P>From a single colony on a plate, use a sterile stick to pick and then swirl in 50 mL of LB.</P><P> </P><P> </P><P><U>IPTG induction protocol</U></P><P> </P><P>Made 105uL of 1M IPTG from 0.025g IPTG in 105uL sterile deionized water.</P><P> </P><P>Taken from <A href="http://openwetware.org/wiki/IPTG">http://openwetware.org/wiki/IPTG</A></P><P>Isopropyl-beta-D-thiogalactopyranoside</P><P>Induces transcription from promoters regulated by LacI repressor.</P><P>Molecular weight is 238.31 g/mol. The chemical formula is <A rel="nofollow" title="http://www.fermentas.com/catalog/reagents/img/iptg.gif" class="external text" href="http://www.fermentas.com/catalog/reagents/img/iptg.gif">here</A>.</P><OL><LI>Dissolve 1g in 4196 μL deionized water to make 1M solution.</LI><LI>Filter sterilize with syringe and 0.22μm filter</LI></OL><P>Generally a 1mM solution is an effective amount to induce the pLac promoter region. It should be noted that this may vary over cell strains. For example:</P><UL><LI>lacIq: cell strains which overproduce LacI repressor (ie. <A title="E. coli genotypes" href="http://openwetware.org/wiki/E._coli_genotypes"><I>E. coli</I> cell strain</A> type <A title="E. coli genotypes" href="http://openwetware.org/wiki/E._coli_genotypes#D1210">D1210</A>, aka. <A title="registry:Part:BBa_V1003" class="extiw" href="http://partsregistry.org/Part:BBa_V1003">BBa_V1003</A>)</LI></UL><P>For including IPTG in LB agar plates, a typical amount people recommend is 0.1-0.5mM IPTG.</P><P> </P><P> </P></DIV><A href="javascript:unhide('miniprep');">Qiagen Miniprep Protocol</A><DIV id="miniprep" class="hidden"><P>From <A href="http://openwetware.org/wiki/Miniprep/Qiagen_kit">http://openwetware.org/wiki/Miniprep/Qiagen_kit</A></P><P> </P><P>Procedure</P><OL><LI>Pellet 1-5mL of an overnight culture by spinning 1.5 mL at 8000 for 10 min.</LI><LI>Resuspend pelleted bacterial cells in 250 µl Buffer P1 (kept at 4 °C) and transfer to a microcentrifuge tube.
    <P>Ensure that RNase A has been added to Buffer P1. No cell clumps should be visible after resuspension of the pellet.</P></LI><LI>Add 250 μl Buffer P2 and gently invert the tube 4–6 times to mix.
    <P>Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.</P><EM>Even if all you care about is the plasmid DNA, don't vortex it - this will cause smearing on your gel and hide the plasmid band.</EM></LI><LI>Add 350 μl Buffer N3 and invert the tube <STRONG>immediately</STRONG> but gently 4–6 times.
    <P>To avoid localized precipitation, mix the solution gently but thoroughly, immediately after addition of Buffer N3. The solution should become cloudy.</P></LI><LI>Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.
    <P>A compact white pellet will form.</P></LI><LI>Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.</LI><LI>Centrifuge for 30–60 s. Discard the flow-through.
    <P><I>Spinning for 60 seconds produces good results.</I></P></LI><LI>(Optional): Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through.
    <P>This step is necessary to remove trace nuclease activity when using endA+ strains such as the JM series, HB101 and its derivatives, or any wild-type strain, which have high levels of nuclease activity or high carbohydrate content. Host strains such as XL-1 Blue and DH5α™ do not require this additional wash step.</P><P><I>Although they call this step optional, it does not really hurt your yield and you may think you are working with an endA- strain when in reality you are not. Again for this step, spinning for 60 seconds produces good results.</I></P></LI><LI>Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.
    <P><I>Spinning for 60 seconds produces good results.</I></P></LI><LI>Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer.
    <P>IMPORTANT: Residual wash buffer will not be completely removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE may inhibit subsequent enzymatic reactions. <I>They are right about this.</I></P></LI><LI>Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
    <P><I>If you are concerned about the concentration of the DNA, you can alternatively add 30 μL water to the center of the column, incubate at room temperature on the bench for 5 mins and then centrifuge for 1 min. This will increase the concentration of DNA in your final sample which can be useful in some cases. See notes below for why you should elute in water rather than the Buffer EB they recommend if you plan to sequence your sample. Even if you are not sequencing, it may be beneficial to elute in water. For instance, if you elute in buffer EB and you are using this DNA in a restriction digest, then the additional salts in your sample can affect the salt content of your digest. This may matter with some finicky enzymes.</I></P></LI></OL><P> </P><P><A href="/Agarose-Gel-Electrophoresis-Protocol">Agarose Gel Electrophoresis Protocol</A> </P><P> </P><P> </P><P><STRONG><U>Materials</U></STRONG></P><P> </P><P>Do not autoclave solutions containing isopropanol or MOPS; use sterile filtration if necessary.</P><P> </P><P>Buffer P1 - Alkaline prep buffer</P><UL><LI>50 mM Tris-HCl pH 8.0</LI><LI>10 mM EDTA</LI><LI>100 μg/ml RNaseA</LI></UL><P>The buffer and RNaseA can also be ordered from Qiagen separately (catalog numbers 19051 and 19101).</P><P>(Denatures DNA and protein, chromosomal DNA separated from plasmid because it's bound to cell wall.  Plasmid DNA remains in clear supernatant.)</P><P> </P><P>Buffer P2 - Lysis buffer</P><UL><LI>200 mM NaOH</LI><LI>1% SDS</LI></UL><P>Check Buffers P2 and P3 for salt precipitation and redissolve at 37 degrees C if neccessary.</P><P>(This is why we mix slowly, to prevent contamination of plasmid with chromosomal DNA (There is no other step that separates them).)</P><P> </P><P>Buffer P3 (not for spin columns, but for Qiatips, midi, maxi, giga kits)</P><UL><LI>3.0 M potassium acetate pH 5.5</LI></UL><P> </P><P>Buffer N3 - Neutralizing buffer</P><UL><LI>4.2 M Gu-HCl</LI><LI>0.9 M potassium acetate</LI><LI>pH 4.8</LI></UL><P> </P><P>Buffer PB - Endonuclease wash</P><UL><LI>5 M Gu-HCl</LI><LI>30% ethanol</LI><LI>(maybe add 10mM Tris-HCL PH 6.6, and that is better)</LI></UL><P>(Removes endonucleases which may degrade target DNA.)</P><P> </P><P>Buffer PE - Ethanol wash</P><UL><LI>10 mM Tris-HCl pH 7.5</LI><LI>80% ethanol</LI></UL><P>(Removes salts.)</P><P> </P><P>Buffer QBT equilibration buffer</P><UL><LI>750 mM NaCl</LI><LI>50 mM MOPS pH 7.0</LI><LI>15% isopropanol</LI><LI>0.15% triton X-100</LI></UL><P> </P><P>Buffer QC wash buffer</P><UL><LI>1.0M NaCl</LI><LI>50 mM MOPS pH 7.0</LI><LI>15% isopropanol</LI></UL><P> </P><P>Buffer QF elution buffer</P><UL><LI>1.25M NaCl</LI><LI>50 mM Tris-HCl pH 8.5</LI><LI>15% isopropanol</LI></UL><P> </P><P>Buffer QN</P><UL><LI>1.6M NaCl</LI><LI>50 mM MOPS pH 7.0</LI><LI>15% isopropanol</LI></UL><P>Buffer FWB2</P><UL><LI>1M potassium acetate, pH 5.0</LI></UL></DIV><A href="javascript:unhide('cryo');">Cryopreservation Protocol</A><DIV id="cryo" class="hidden"><H2><SPAN class="mw-headline">Materials</SPAN></H2><UL><LI>40% glycerol solution</LI><LI>Cryogenic vials</LI></UL><H2><SPAN class="mw-headline">Method</SPAN></H2><UL><LI>Make a 1/100 dilution subculture of an overnight subculture, or pick from overnight plates and inoculate</LI><LI>Allow to grow from some hours, so it will be in logarithmic growth phase</LI><LI>Add 1 ml of 40% glycerol in H<SUB>2</SUB>O to a cryogenic vial.</LI><LI>Add 1 ml sample from the culture of bacteria to be stored.</LI><LI>Gently vortex the cryogenic vial to ensure the culture and glycerol is well-mixed.
    </LI><LI>Alternatively, pipet to mix.</LI></UL><LI>Use a tough spot to put the name of the strain or some useful identifier on the top of the vial.</LI><LI>On the side of the vial list all relevant information - part, vector, strain, date, researcher, etc.</LI><LI>Store in a freezer box in a -80C freezer. Remember to record where the vial is stored for fast retrieval later.</LI><H2><SPAN class="mw-headline">Notes</SPAN></H2><UL><LI>While it is possible to make a long term stock from cells in stationary phase, ideally your culture should be in logarithmic growth phase.</LI></UL><P> </P></DIV><A href="javascript:unhide('gel');">Agarose Gel Electrophoresis Protocol</A><DIV id="gel" class="hidden"><P><U>Gel preparation</U></P><P>1.) Prepare 500 mL of 1X TAE buffer (10 mL from 50X TAE: 2M Tris, 1M Acetic acid, 50 mM EDTA)</P><P>2.) Prepare a 60 mL 1% powdered agarose gel (0.6g)  (or 80mL with 0.8g for larger tray)</P><P>3.) Combine agarose and TAE in 125 mL flask and plug with scrunched up Kimwipe.</P><P>4.) Dissolve in microwave for 50 sec., swirl.</P><P>5.) If cloudy, reheat for 15 sec.</P><P>6.) Repeat with 5 sec. heating times until crystal clear.  Be Careful!</P><P>7.) When it has cooled enough to touch add 8 uL of a 2 mg/mL solution of ethidium bromide.</P><P>8.) Immediately pour in to the tray and insert comb.</P><P>9.) Set for at least 20 min.</P><P> </P><P><U>Electrophoresis</U></P><P>1.) Mix 2 uL undigested plasmid DNA with 2 uL 6X sample buffer and 8 uL dH2O. (For digested DNA use 4 uL DNA with 2 uL 6X sample buffer and 6uL dH2O.)</P><P>2.) Load the mixed 12 uL samples into lanes.</P><P>3.) Apply constant voltage of 100V and run for 45 min in 1X TAE.</P><P> </P><P>Method 2.</P><P>EtBr soak for 10-20 minutes after gel has run instead of adding to gel itself.</P><P>Final concentration of 0.2 ug/mL, so for 500mL, add 10uL of 10mg/mL EtBr. </P><P> </P><P> </P><P>We're thinking that the <A href="f/DNA+ladder.jpg">DNA ladder</A> from Ivitrogen (<A href="http://products.invitrogen.com/ivgn/en/US/adirect/invitrogen?cmd=catProductDetail&amp;showAddButton=true&amp;productID=15615016&amp;_bcs_=H4sIAAAAAAAAAL3QUUvDMBAA4Ef7MwrDPji3NOpwmxQZU4c45qDqq4Tk2gbSJCRpR%2F%2B96ew2NvYovuRyx13yJQMcrI1iFXU2jMIUTM0p2LO1wjk9RWiz2Qy5rLkzKgc5pKpEljtAlUUgUaFKQLvpAZFssJtHM60Fp8RxJe2wcKUIer3AmQqCizjAMUaT27sYXwVzJf3Zwl6HqfMHEMM8g5T6IVwSxsDsJbq75oQzHceI17lsOZ8pIowboA4dmh5pyZLXr8XKY5641YI0c%2BIgV6aJfOkNmmRLaX0ZEfYY6NdRy1xVVACn4Yxy9q%2By0ZEMB5n7xvfelfkwGbe0WU6MshAuQITPwl9hlC6UAcv%2FRnjQpa4RELXJdpekXOYCllCDOAuP3tdJxoUDE%2F2GdnRFSkg6c9%2Bb%2ByfmrvWj0ZDgLkm6V1%2FevBzevf%2BVH3Ko3TnWAgAA&amp;returnURL=http%3A%2F%2Fproducts.invitrogen.com%3A80%2Fivgn%2Fen%2FUS%2Fadirect%2Finvitrogen%3Fcmd%3DcatDisplayStyle%26catKey%3D93501%26filterType%3D1%26OP%3Dfilter%26filter%3Dft_1701%252Ff_179801*">this page</A>) corresponds to our ladder.</P><P> </P></DIV><A href="javascript:unhide('3a');">3A Assembly Protocol</A><DIV id="3a" class="hidden"><P>Taken from <A href="http://ginkgobioworks.com/support/">http://ginkgobioworks.com/support/</A></P><P> </P><P>Red - Destination Plasmid</P><P>Blue/Purple - Upstream Part</P><P>Green/Yellow - Downstream Part</P><P> </P><P><STRONG><U>Digest</U></STRONG></P><P>Vortex everything before adding</P><P>1. Prepare 3 tubes for: Upstream, Downstream and destination Plasmid</P><P>2. Add 22.5 uL sterile, deionized water (=42.5 uL - 500 ng DNA with DNA at 25 ng/uL)</P><P>3. Add 20 uL DNA (500 ng at 25 ng/uL)</P><P>4. Add 5 uLNEBuffer 2</P><P>5. Add 0.5 uL BSA</P><P>6. Add 1 uL 1st restriction enzyme</P><P>7. Add 1 uL 2nd restriction enzyme</P><P>   =This should equal 50 uL.</P><P>8. Mix by flicking tube</P><P>9. Incubate at 37oC for 15 min</P><P>10. Incubate at 80oC for 20 min</P><P> </P><P>You can now freeze these digests at -20oC or proceed</P><P>(You can also run a 1% agarose gel with 20 uL of the digest to test for digested part lengths)</P><P> </P><P> </P><P><STRONG><U>Ligate</U></STRONG></P><P>1. Agitate 10X T4 DNA Ligase Reaction Buffer until all precipitate is in solution</P><P>2. Prepare a tube for each addition</P><P>3. Add 11 uL sterile, deionized water</P><P>4. Add 2 uL from each digest: Upstream, Downstream, and Plasmid (6 uL total)</P><P>5. Add 2 uL 10X T4 DNA Ligase Reaction Buffer</P><P>6. Add 1 uL T4 DNA Ligase</P><P>   =This should equal 20 uL</P><P>7. Incubate at room temperature for 10 min</P><P>8. Incubate at 80oC for 20 min</P><P> </P><P>You can now freeze these ligation mixes at -20oC or proceed to Transformation.
</P></DIV><A href="javascript:unhide('clean');">Cleaning Miniprep Spin Columns Protocol</A><DIV id="clean" class="hidden"><P>- soak column 24-48 hr in 1M HCl - try make sure no air bubbles at filter surface</P><P>- store in acid to inhibit contamination - say 0.1M HCl</P><P>- before use, rinse and reequilibrate</P><P>  - 3-5x 0.75mL dH<SUB>2</SUB>O spin thru, discard</P><P>  - 1x 0.75mL PE spin thru, discard and spin again</P><P> </P><P>full reference to come</P><P>stub: BioTechniques 42:186-192 (February 2007)</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:VictoriaBC/Labprotocols">http://2009.igem.org/Team:VictoriaBC/Labprotocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:VictoriaBC/Labprotocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:VictoriaBC/Labprotocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:VictoriaBC/Labprotocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:VictoriaBC/Labprotocols&amp;oldid=126538" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>