"
Page
Discussion
View source
History
teams
Log in
 
Team:Rice University/CONSTRUCTS
From 2008.igem.org
OUR TEAM ::: SUMMARY  ::: BACKGROUND  ::: STRATEGY  ::: CONSTRUCTS ::: RESULTS  ::: ONGOING WORK
Constructs
Saccharomyces cerevisiae is widely used for baking and brewing, a versatile eukaryotic model system,  and particularly useful for synthesizing metabolites under fermentation conditions. The microaerobic conditions of fermentation impede the oxidation of sensitive bioreactive compounds and are optimal for the de novo synthesis of resveratrol. To achieve our project goals and expand the yeast synthetic biology toolbox, we have constructed BioBricks encoding 3 yeast promoters, 3 yeast terminators, a 2-micron origin of replication, 2 selectable markers, 2 metabolic enzymes, and a yeast integration plasmid. We have also submitted two additional parts representing foundational tools, including a gene encoding an amber suppressed RFP biobrick for screening of SupF+ (Amber suppressor) genotype and an amber suppressor tRNA biobrick.
Yeast Promoters
BBa_K122000
pPGK1
~1500 bp upstream of the PGK1 coding region. Strongly induced during fermentation.
1497bp
BBa_K122002
pADH1
700bp upstream of ADH1 promoter region. Constitutive promoter under aerobic and anaerobic conditions.
701bp
BBa_K122017
pGAL1 + tetO
Glucose repressible / galactose inducible GAL1 promoter.  An additional TetR operator site was included to allow repression by TetR.
484bp
Yeast Terminators
BBa_K122003
tCYC1
300bp downstream the CYC1 coding region in a standard yeast strain.
300bp
BBa_K122004
tADH1
300bp downstream the ADH1 coding region in a standard yeast strain.
300bp
BBa_K122013
tPGK1
1000bp downstream the PGK1 coding region in an industrial yeast strain.
1000bp
 
Selectable Markers
BBa_K122018
ZeoR
Zeocin/Bleocin Resistance Gene
300bp
BBa_K122008
BleoR
Bleocin Resistance Gene under pTet promoter
80bp-ish
BBa_K122014
ORI+HisTag
2 Micron ORI and auxotrophic histidine marker
<9000bp
 
Project Specific Constructs
BBa_K122001
[pGAL1][tetO][ZeoR]
874bp
BBa_K122005
Tyrosine Ammonia Lyase
1933bp
BBa_K122010
4-coumarate CoA ligase :: Stilbene Synthase Fusion Protein
4000bp
BBa_K122012
[pPGK1][4CL:STS][tCYC1]
5497bp
BBa_K122015
[pGAL1][tetO][ZeoR][tADH1]
1175bp
BBa_K122021
[pADH1][TAL][tPGK1]
2651bp
BBa_K122019
[pPGK1][4CL:STS][tCYC1][pGAL1][tetO][ZeoR][tADH1]
1824bp
 
Additional Bacterial Parts
Novel Zero Leak Inverter ( K122007 and K122006 )
Amber Suppressor tRNA
K122007
The supF construct, an amber suppressor tRNA, allows for read-through at native amber (TAG) stop codons.  Charges with tyrosine.
211bp
K122006
Point Mutation of RFP(13521) with incorporation of an amber stop codon at the native tyrosine required for fluorophore maturation.
923bp
Through incorporation of amber stop codons or point mutations of tyrosine codons (TAC) to TAG within the coding region, a genetic circuit can be used to add an additional level of regulation and determine whether a full protein or partial peptide with be synthesized. This design has high signal-to-noise ratio, with virtually no leaky expression.
The Amber suppressed Red Florescent Protein (ARFP) has no virtually expression in SupF- (2) cells while visually apparent levels of red pigment is present in SupF+ cells (4). ARFP expression was compared against wtRFP expression in both cell types (1 and 3).
OUR TEAM  ::: SUMMARY  ::: INTRODUCTION  ::: STRATEGY  ::: RESULTS  ::: ONGOING WORK  ::: GALLERY
Retrieved from " http://2008.igem.org/Team:Rice_University/CONSTRUCTS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
