<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sherbrooke_BioResults skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sherbrooke/BioResults</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Sherbrooke">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Description">PROJECT
					</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Description">Description</A></LI><LI class="largeMenu"><A href="https://2015.igem.org/Team:Sherbrooke/Experiments">Hardware Experiments &amp; Protocols</A></LI><LI class="largeMenu"><A href="https://2015.igem.org/Team:Sherbrooke/BioExperiments">Biology Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Results">Hardware Results</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/BioResults">Biology Results</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Design">Hardware Design</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/PartsDesign">Parts Design</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sherbrooke/Parts">PARTS
					</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Measurement">MEASUREMENT</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Software">SOFTWARE</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Entrepreneurship">ENTREPRENEURSHIP</A></LI></DIV><DIV id="contentContainer"><H1> Biology Results</H1><SPAN id="Biology_results"> </SPAN><H2>Biology Results</H2><UL><LI><A href="#toxins_test">Toxin Efficiency Test</A></LI><LI><A href="#toxins_integrated_test">Integrated Toxin Efficiency Test</A></LI><LI><A href="#pKD4">pKD4 cassette insertion in pVCR94</A></LI><LI><A href="#vcrx027">vcrx027 deletion with BBa_K1744000</A></LI><LI><A href="#vcrx027_clean">Clean deletion of BBa_K1744000</A></LI><LI><A href="#BBa_K1744001_insertion">BBa_K1744001 insertion in LacZ and phenotype</A></LI></UL><SPAN id="toxins_test"> </SPAN><H3>Toxin Efficiency Test</H3><P style="text-align:justify">
Here are the data for the killswitch test done with the cells containing the plasmids pBAD30-[<I>mosT</I>;<I>ccdB</I>;<I>mazF</I>;<I>vcrx028</I>], for experiment details, please click <A href="BioExperiments#Recom_test"> here</A>:
</P><H3>pBAD30-mosT</H3><H3>pBAD30-ccdB</H3><H3>pBAD30-mazF</H3><H3>pBAD30-vcrx028</H3><P style="text-align:justify">
Note that the data were normalized with the initial OD[600]. The horizontal axis shows a time lapse and the vertical one the optical density (OD) measured at 600 nm. The optical density serves as an indicator of cell concentration in the medium, the more concentrated the cells are, the higher the OD. Cells were inoculated in LB broth with glucose as a control to keep the kill switches repressed. It was also measured with culture in 1% arabinose LB broth to show the effect of toxin induction on the cells.  We can see that <I>mosT</I> and <I>vcrx028</I>  shown the best behavior with absence of growth and cell mortality with arabinose (induced killswitch). This shows that cells die more rapidly than they are able to mutate the toxin sequence or gain resistance. It also shows that both <I>mosT</I> and <I>vcrx028</I> are the most deadly among the tested toxins.
</P><A href="#Biology_results">Back to top</A><SPAN id="toxins_integrated_test"> </SPAN><H3>Integrated Toxin Efficiency Test</H3><P>
Here are the data for the integrated killswitch test, for experiment details, please click <A href="BioExperiments#Recom_test"> here</A> :
</P><H3>E. coli BW25113-mosT</H3><H3>E. coli BW25113-vcrx028</H3><P style="text-align:justify">
Note that the data were normalized with the initial OD[600]. The horizontal axis shows a time lapse and the vertical one the optical density (OD) measured at 600 nm. The optical density serves as an indicator of cell concentration in the medium, the more concentrated the cells are, the higher the OD. Cells were inoculated in LB broth with glucose as a control to keep the kill switches repressed. It was also measured with culture in 1% arabinose LB broth to show the effect of toxin induction on the cells. We can see that after 10-12 hours, all the tested clones of <I>mosT</I> recombinants and 1 of the <I>vcrx028</I> ones started growing even if arabinose was present.  Therefore, we conclude that <I>vcrx028</I> is the most fitted counter-selectable marker for our clean deletion system.
</P><A href="#Biology_results">Back to top</A><SPAN id="pKD4"> </SPAN><H3>pKD4 cassette insertion in pVCR94</H3><P style="text-align:justify">
This section shows the results for pKD4’s cassette insertion in pVCR94 and deletion of <I>vcrx028</I>. This was achieved through PCR screening as described in <A href="BioExperiments#Cyclic_deletion">the experiment section</A>.
</P><P style="text-align:justify">
As it can be seen on the above gel, all of the clone exhibit the correct size band. The only thing that is not good about this result is that there is non-specific amplification in all of the clones. Thing is that we restarted the PCR verification multiple times but we could not get rid of the non-specific binding. It might be primer off-targets. In the other hand, it would be unlikely that the predicted band could be non-specific because we screened with two primer pairs for confirmation. For the first series of amplification, the clones were tested for 5’ end of the insertion, with a primer binding in the 3’ of the cassette. For the right side of the gel, the same clones in the same order were screened for the 3’ end of the insertion with a primer binding in the 5’ of the inserted cassette. Therefore, no product should be visible if the cassette was not inserted at the right place. We kept clone # 3 for further experiments because it was the one with the lowest background of non-specific amplification.
</P><A href="#Biology_results">Back to top</A><SPAN id="vcrx027"> </SPAN><H3>vcrx027 deletion with BBa_K1744000</H3><P style="text-align:justify">
This section shows the results of BBa_K1744000 integration in pVCR94, leading to <I>vcrx027</I> and pKD4 knock-out. For the details on the experiments, please click <A href="BioExperiments#Cyclic_deletion">here</A>.
</P><P style="text-align:justify">
The above gel shows a representative success rate of BBa_K1744000. The presence of a band in the red rectangle means the clone is good. The clone order is the same on 5’ and 3’ screening.  As you can see, the success rate is really high. There is no bad clones in my screening of 5 different colonies. This represent a 100% success rate. This is confirmed by our wild-type control showing no amplification. However, false positives are still possible. I would recommend to screen at least 5 clones to make sure you get at least one good clone. In our case, clone #2 will be used for further experiments.
</P><A href="#Biology_results">Back to top</A><SPAN id="vcrx027_clean"> </SPAN><H3>Clean deletion of BBa_K1744000</H3><P>
This section shows the reliability of <I>vcrx028</I> as a counter-selection marker.for clean deletion recombineering. For the details on the experiments, please click <A href="BioExperiments#Cyclic_deletion">here</A>.
</P><P style="text-align:justify">
The above picture shows the screening effort used to obtain good recombinant. From our experience, it seems all recombinants after clean deletion of BBa_K1744000 are good after screening. You can see that only the control exhibits a different amplicon size, and that all clones have the sized band. This means that vcrx028 can be successfully used as a highly efficient killswitch to counter select cells that did not recombine. The success rate of the method is, so far, of 100%.
</P><A href="#Biology_results">Back to top</A><SPAN id="BBa_K1744001_insertion"> </SPAN><H3>BBa_K1744001 insertion in LacZ and phenotype</H3><P>
This section will discuss BBa_K1744001 functionality as a reporter for recombineering. For details about the experiments leading to these results, please click <A href="BioExperiments#Insertion_BBa_K1744001">here</A>.
</P><P style="text-align:justify">
In order to assay the expression of amilCP in single copy, it was inserted through recombineering in lacZ gene in E. coli K-12 substr. BW25113. The result shown above demonstrate that the insertion had a 100% rate of success. In fact, we screened with two primer pairs. The first one (at left) produce a 875 bp amplicon if insertion of the part in the genome is successful but no bands if it failed. The second primer pair (at right) produces a 1.7 kb band if kanR-amilCP was successfully inserted in the genome and a 1.2 kb band if it did not. Using both results, we can conclude that 100% of the clones are positive. The cells were selected on LB kanamycin 50 µg/mL and ampicilin 100 µg/ml (for the selection BBa_K1744001 pSIM6, a plasmid important for recombineering). The cells were photographed to show the difference between cells with the plasmidic form of amilCP gene and the inserted version of the gene:
</P><P style="text-align:justify">
In this previous image, you can see the appearance of colonies on LB agar when they carry plasmidic or genomic amilCP-KanR cassette. As you can see, the expression of <I>amilCP</I> in single copy is not sufficient to make the colonies blue even faintly. But, this is not as dramatic as it looks, <I>amilCP</I> can still be used in our experiment by exploiting this aspect. The plasmid carrying <I>amilCP</I> can be detected easily because it makes the colonies deep blue. In a recombineering experiment, the longest part if often the screening. There can be a lot of false positive colonies. Some of those can be plasmidic background. But, with <I>amilCP</I>, the plasmidic background will be eliminated rapidly because the colonies will be deep blue instead of white therefore accelerating the screening and eliminating a good proportion of the background. In an automation perspective, this is really important because a robot often manipulate in a more straightforward manner than a real person and false positive colonies could mean restarting all over again.
</P><A href="#top_menu_under">Back to top</A><H2>Also in &quot;Project&quot;</H2><UL><LI><A href="Description">Project Description</A></LI><LI><A href="Experiments">Hardware Experiments &amp; Protocols</A></LI><LI><A href="BioExperiments">Biology Experiments &amp; Protocols</A></LI><LI><A href="Results">Hardware Results</A></LI><LI><A href="Design">Hardware Design</A></LI><LI><A href="PartsDesign">Parts Design</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>