<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sherbrooke_BioExperiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sherbrooke/BioExperiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Sherbrooke">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Description">PROJECT
					</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Description">Description</A></LI><LI class="largeMenu"><A href="https://2015.igem.org/Team:Sherbrooke/Experiments">Hardware Experiments &amp; Protocols</A></LI><LI class="largeMenu"><A href="https://2015.igem.org/Team:Sherbrooke/BioExperiments">Biology Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Results">Hardware Results</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/BioResults">Biology Results</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Design">Hardware Design</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/PartsDesign">Parts Design</A></LI></UL><LI><A href="https://2015.igem.org/Team:Sherbrooke/Parts">PARTS
					</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Measurement">MEASUREMENT</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Software">SOFTWARE</A></LI><LI><A href="https://2015.igem.org/Team:Sherbrooke/Entrepreneurship">ENTREPRENEURSHIP</A></LI></DIV><DIV id="contentContainer"><H1>Biology Experiments &amp; Protocols</H1><SPAN id="Biology"> </SPAN><H2>Biology section</H2><P style="text-align:justify">
In this section, we will detail the experiments leading to a fully working clean deletion system through recombineering. This system will ultimately serve as the basics for automated recombineering using BIOBOT, even if we haven't had the time to actually do it with the robot because of the early deadlines.
</P><UL><LI><A href="#Bio_Exp">Experiments section</A></LI><LI><A href="#Bio_Protocol">Biology Protocols</A></LI></UL><SPAN id="Bio_Exp"> </SPAN><H2>Biology Experiments</H2><UL><LI><A href="#Recom_construct">Recombineering cassette construction</A></LI><LI><A href="#Recom_test">Recombineering cassette test</A></LI><LI><A href="#Cyclic_deletion">Cyclic deletion cassette test</A></LI><LI><A href="#Cassette_opti">Deletion cassette optimisation</A></LI></UL><SPAN id="Recom_construct"> </SPAN><H3>Recombineering cassette construction</H3><P style="text-align:justify">
In order to test our BIOBOT platform (which could ultimately do automated MAGE experiment), a recombineering experiment has to be set-up. While a lot of reliable selectable markers are known, we can’t say the same about counter selectable ones. Many counter selectable markers for recombineering are actually toxins isolated from conjugative plasmids. pVCR94 is a plasmid isolated during the 1994 cholerae outbreak in a Rwanda refugee camp. This conjugative plasmid carries resistance to a lot of antibiotics and its regulation was recently investigated by 
<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4207636/"> Carraro et al. 2015</A>.
</P><P style="text-align:justify">
 Like most conjugative plasmids, it possesses many stabilisation and partition systems. One of which is a powerful toxin-anti-toxin system. The toxin was firstly tested for its ability to kill cells compared to other known toxins. To do so, mosT, ccdB, mazF and pVCR94’s toxin: vcrx028 were cloned in pBAD30 to obtain arabinose triggered kill switches. To do so, <A href="PartsDesign#Primers">primers described in the design sections</A> were used on different template: pSXT, ptac-ccdb-cassette, gBlock_FG1, pVCR94. Specifically, primer pair vcrx028-pBAD30-F and vcrx028-pBAD30-R were used on pVCR94 to amplify<I> vcrx028</I>’s ORF as an insert for pBAD30; ccdB-pBAD30-F and ccdB-pBAD30-R on ptac-ccdb-cassette to amplify <I>ccdb</I>’s ORF as an insert for pBAD30; mosT-pBAD30-F and mosT-pBAD30-R to amplify <I>mosT</I>’s ORF as an insert for pBAD30; mazF-pBAD30-F and mazF-pBAD30-R to amplify <I>mazF</I>’s ORF as an insert for pBAD30. The pBAD30 vector (in purified DNA miniprep form), was digested with EcoRI. After that, PCR products and digestions of pBAD30 were purified by SPRI technique with Agencourt beads ratio 1:1. The inserts and the vector were then mixed for <A href="#Gibson"> Gibson experiment</A>. After the gibson, DNA was transformed using chemically competent cell into <I>E. coli EC100 </I> strain. For schematique details on the resulting plasmid, please visit:
<A href="PartsDesign#Toxins"> Arabinose inducible toxin design section</A>.
</P><P style="text-align:justify">
The clones for each construction were screened by digestion (results not shown) and good clones for each constructions were obtained. Those clones will be used for phenotypic test of each toxins.
</P><A href="#Biology">Back to top</A><SPAN id="Recom_test"> </SPAN><H3>Recombineering cassette test</H3><P style="text-align:justify">
The different kill switches (pBAD30-[mosT;ccdB;mazF;vcrx028]) were tested (see <A href="#toxins_test">here</A> for the protocol) to investigate if they worked and to know which one was the best. We found that mosT and vcrx028 were the best counter-selectable markers (see <A href="BioResults#toxins_test">here</A> for the results)
</P><P style="text-align:justify">
Our multi-copy plasmid results shown that mosT and vcrx028 were the best candidates for our system, but we needed to test them in single-integrated-copy for it to be more representative. Therefore, we amplified cassettes containing all the needed elements of the kill switches from pBAD30-[mosT;vcrx028] (which are basically AraC with its promoter, inducible toxin with promoter and terminator and bla gene from pBAD30) with primers (killswitch_ins_LacZ_F and killswitch_ins_LacZ_R) adding homologies that allow lambda-red mediated recombination in the lacZ truncated gene of Escherichia coli BW25113. After confirmation of the recombinants, they were tested and it shown that both killswitch were still working even in single copy. The main difference is that after 10-12 hours, some mutants that inactivated the integrated kill switches began growing. All the tested clones for mosT shown that behavior, but only 1 of the 3 vcrx028 clones did. <A href="#toxins_test">See the protocol</A> or <A href="BioResults#toxins_integrated_test">See the results</A></P><A href="#Biology">Back to top</A><SPAN id="Cyclic_deletion"> </SPAN><H3>Cyclic deletion cassette test</H3><P style="text-align:justify">
Since vcrx028 seems to work better than other toxins, we decide to use it in order to test a cyclic recombineering cassette using the 
 <A href="PartsDesign#Toxins"> pBAD30-vcrx028 plasmid</A> 
as template. Our target genes for recombineering will quite ironically be vcrx028 and vcrx027 within pVCR94 (deltaX2 version see Carraro <I>et al.</I> 2015 for details, this version had the multiple resistance genes region deleted and replaced by Spectinomycin resistance). We want to delete these gene for two principal project related purpose. Firstly, by doing so, we will enable cyclic deletion using our cassette on pVCR94, which is the only plasmidic that we could do so beacause expresses the anti-toxin of <I>vcrx028</I> which is <I>vcrx027</I> and would resist to toxin induction. Secondly, the goal of this experiment is to test the recombineering in a relatable context. Deleting gene in low copy plasmid can be a challenging task since they can contain multiple plasmid copy per cell and never segregate properly due to their capacity (in our case) to conjugate. Furthermore, we know that recombineering works in pVCR94 since there are quite a few articles that have used this technique to modify its sequence (one of these is Carraro <I>et al.</I> 2015. We will firstly need to take out <I>vcrx028</I>  to take out <I>vcrx027</I> with our cassette and make our cassette “pop-out” with a fusion PCR of both adjacent regions.
</P><P style="text-align:justify">
The first thing to do is to delete <I>vcrx028</I> from pVCR94. To do so, we cannot use our newly constructed recombineering cassette because the toxin will not be able to counter select with a functional <I>vcrx027</I>(the anti-toxin) in the background. We therefore need another selection marker. Traditionally, the recombineering technique use an antibiotic resistance gene as a positive selection system. We decided to use <I>Aph3’-II</I>, the resistance gene to kanamycin from pKD4, a commercially available plasmid that have the characteristic to have flanking FRT sequence at both sides of the <I>Aph3’-II</I> gene. So, the pKD4 casette was amplified using vcrx028-KanR-del-F and vcrx028-KanR-del-R (<A href="PartsDesign#Primers">see primer details in the design section</A>).The cassette was then purified by <A href="#SPRI">SPRI</A> with agencourt bead ratio 1:1 and <A href="#restriction_digest">digested with DpnI</A> to get rid of plasmidic background. It was then purified again and used for <A href="#Recombineering">recombineering</A>. The recombineering experiment uses (in our case) pSIM5 (chloramphenicol resistant) as the thermo-inducible plasmid carrying the lambda red system. The cells were plated on LB kanamycin, streptinomycin and chloramphenicol at <A href="#Medium">working concentration</A> and incubated overnight. After that, the isolated colonies are picked and inoculated in LB broth containing the same antibiotics amounts. The DNA of each clones were prepared using the <A href="#Chelex">Chelex method</A> and were screened using vcrx028-KanR-del-F with vcrx027-28-del-verif-R; and vcrx027-28-del-verif-F with vcrx028-KanR-del-R. The advantage of using those two pairs of primer is that it gives a double confirmation of good insertion. In each case, one of the two primer in internal to the pKD4 cassette and will only bind to pVCR94 if the pKD4 cassette was successfully integrated. See the results for the cassette insertion in <A href="BioResults#pKD4">pKD4 insertion result section</A>.
</P><P style="text-align:justify">
We can now test our cassette through <I>vcrx027</I> deletion. Since the kanamycin marker is no longer wanted in pVCR94 and is adjacent to the gene to delete, we will delete it as well. The primers (vcrx027-del-F and vcrx027-del-R) listed in 
<A href="PartsDesign#Primers"> the primer table </A>
 of the design section  are used to amplify the cassette as described in the same section. The cassette basically contain an <I>araC</I> gene for repression, an arabinose inducible <I>vcrx028</I> toxin and a resistance gene for ampicillin (bla),<A href="Parts#BBa_K1744000"> see the part description for full design information</A>. The cassette was then purified by <A href="#SPRI">SPRI</A> with Agencourt beads ratio 1:1 and <A href="#restriction_digest">digested with DpnI</A> to get rid of plasmidic background. It was then purified again and used for <A href="#Recombineering">recombineering</A>. The plasmid used was still pSIM5.The cells were plated on LB ampicillin, streptinomycin, chloramphenicol and glucose at <A href="#Medium">working concentration</A> and incubated overnight. After that, the isolated colonies are picked and inoculated in LB broth containing the same antibiotics and glucose amounts. The DNA of each clones were prepared using the <A href="#Chelex">Chelex method</A> and were screened using 5'_pVCR94_del_verif_F with 5'_pVCR94_del_verif_R; and 3'_pVCR94_del_verif_F with 3'_pVCR94_del_verif_R. Again, one primer on two binded in the cassette and the other one binded in pVCR94. As the primer name says, the first pair screened the 5’ end of the insertion and the second pair screened the 3’ end section of the insertion. The results of  <I>vcrx027</I> deletion are shown in <A href="BioResults#vcrx027">vcrx027 deletion result section</A>.
</P><P style="text-align:justify">
The cyclic deletion cassette was then tested for its counter-selectivity. To do so, adjacent 5’ region of vcrx027 was amplified on about 200 bp using vcrx027-28-clean-del-5'-F and vcrx027-28-clean-del-5'-R. The 3 ‘ region was also amplified on the same distance with vcrx027-28-clean-del-3'-F and vcrx027-28-clean-del-3'-R. Using both PCR products (1 µL each) a fusion PCR was started using vcrx027-28-clean-del-5'-F and vcrx027-28-clean-del-3'-R as primer. All of these primers are described in <A href="PartsDesign#Primers"> the primer table </A>. The resulting cassette was then <A href="#SPRI">SPRIed</A> using Agencourt beads and used in the <A href="#Recombineering">recombineering</A> experiment. The recombineering used again the same plasmid pSIM5.The cells were plated on LB streptinomycin, chloramphenicol and arabinose at <A href="#Medium">working concentration </A> and incubated overnight. After that, the isolated colonies are picked and inoculated in LB broth containing the same antibiotics and arabinose amounts. The DNA of each clones were prepared using the <A href="#Chelex">Chelex method</A> and were screened using vcrx027-28-del-verif-F and vcrx027-28-del-verif-R. The results for the clean deletion was perfect. All of the 20 clones screened using toxin induction counter-selection were showing the good pattern on gel. You can see the result in <A href="BioResults#vcrx027_clean">vcrx027 clean deletion results</A></P><A href="#Biology">Back to top</A><SPAN id="Cassette_opti"> </SPAN><H3>Deletion cassette optimisation</H3><P style="text-align:justify">
One major aspect not discussed in the previous sections about our cassette is that it is really big. This is a problem on many levels: it’s harder to get a clean PCR amplification, the recombineering efficiency and specificity is impaired with long cassette, there is some elements that can be changed or are unnecessary and ampicillin resistance is not a good positive selection marker since it allows satellite colonies to grow within an overnight time lapse due to antibiotic fast degradation. To assess these concern, the cassette needs to be modified in order to be smaller, more efficient in PCR amplification and recombineering and easier to work with in screening assays. That be said, we still have a working cassette with strong results to send to IGEM Headquarters as a part. We just have to change the backbone. Unfortunately, it won’t be Biobrick compatible. Our team is sorry about that, but since the cassette will be optimised, we have put more effort on actually improving the cassette than making it Biobrick proof. Thing is you can still digest it if you want, but the easiest way to obtain your construction done is by far gibson assembly. This section will discuss how we have planned to improve the deletion system and how we assembled the shipped parts. Preliminary results will not be shown for they are not necessary to the comprehension of this section. Only real phenotypic-like test will be shown.
</P><A href="#Biology">Back to top</A><H4>BBa_K1744000’s construction</H4><P style="text-align:justify">
The construction of part <A href="Parts#BBa_K1744000">BBa_K1744000</A> was pretty easy in term of manipulation. This part is basically just the previously described cassette, but transferred in pSB1C3 for IGEM shipping of the part. To do so, pBAD30-vcrx028 was<A href="#miniprep"> miniprep</A> and diluted 1/1000 for <A href="#PCR"> PCR</A>. The primer used for amplification of the cassette were pBAD30-vcrx028-igem-ship-F and pBAD30-vcrx028-igem-ship-R. They were used as described in <A href="PartsDesign#Primers"> the primer table </A> in the design section. After that, we purified the PCR product using <A href="#SPRI">SPRI method</A> and assembled the construction by <A href="#Gibson">gibson assembly</A> using pSB1C3 linearised version provided by the IGEM team. The gibson reaction was then entirely transformed in chemically competent cells (EC100Dpir+ strain) and plated on LB ampicillin, chloramphenicol and glucose 5%. The clones were confirmed by digestion and lyophilised for IGEM shipping.
</P><A href="#Biology">Back to top</A><SPAN id="Insertion_BBa_K1744001"> </SPAN><H4>BBa_K1744001’s construction</H4><P style="text-align:justify"><A href="Parts#BBa_K1744001">This part</A> is the optimisation conclusion of the final deletion cassette. It contains kanamycin resistance from pOK12 (a commercially available plasmid) which is a truncated, shorter version of <I>Aph3’-I</I> and <I>amilCP</I>, a deep blue chromoprotein Biobrick standardised by a previous IGEM team from part BBa_K592009. We came out with this idea because we wanted the insertion cassette to be shorter and easier to screen. It turns out that if the good colonies are blue, or, if any blue colonies can be associated with a clear result (like plasmid background for example), it would be much easier to design an automated system in the BIOBOT to seek and pick only the right colonies for screening. The chromoprotein would therefore serve as a detection device for a future robot update.  Even if adding this chromoprotein to the cassette makes it bigger it can still be worth it if the difference between negative and positive clones can become more visual.
</P><P style="text-align:justify">
To build this part, we ordered a gBlock from IDT containing a P1U8 driven amilCP ORF. P1U8 is a promoter + RBS combination characterised to drive strong expression of both GFP and RFP in plasmidic form. The test has been done by Mutalik <I>et al.</I> in 2013 (see reference section in team parts section). To assemble the part, we amplified the P1U8-amilCP gBlock (named gBlock_IGEM_2015_2) using primers kanR-amilCP-igem-ship-F2 and kanR-amilCP-igem-ship-R2. We also amplified pOK12 with kanR-amilCP-igem-ship-F1 and kanR-amilCP-igem-ship-R1 to obtain the kanamycin resistance gene. Then, we used 1 µL of both PCR products to do a fusion PCR using kanR-amilCP-igem-ship-F1 and kanR-amilCP-igem-ship-R2 (see <A href="PartsDesign#Primers"> the primer table </A> for details on the primers).After that, we purified the PCR product using <A href="#SPRI">SPRI method</A> and assembled the construction by <A href="#Gibson">gibson assembly</A> using pSB1C3 linearised version provided by the IGEM team. The gibson reaction was then entirely transformed in chemically competent cells (EC100Dpir+ strain) and plated on LB kanamycin and chloramphenicol. The clones were confirmed by digestion and lyophilised for IGEM shipping. The aspects of the colonies was indeed deep blue. We had great hope that maybe in single copy we could get pale blue colonies. So we designed primer to insert BBa_K1744001 in BW25113’s genome at LacZ gene. To do so BBa_K1744001 was amplified using kanR-amilCP-lacZ-ins-F and kanR-amilCP-lacZ-ins-R. Then 1 µL of a 1/100 dilution of PCR product was used in a second round of PCR to enlarge the homologies to 80 bp approximately. The primer used are lacZ-homology-ext-F and lacZ-homology-ext-R. Then, the cassette was integrated in the genome through <A href="#Recombineering">recombineering</A> using a pSIM6 containing BW25113 strain. The cells were then plated on LB Kanamycin and let to grow overnight. Some colonies were then picked and inoculated in LB broth containing kanamycin. The DNA of those clones was then extracted using <A href="#Chelex">Chelex method</A> and screened through PCR using 2 different pairs of primers. The first pair includes lacZ-verif-F and amilCP-R and should yield amplification only if there was an appropriate insertion. The second pair of primers include lacZ-verif-F and lacZ-verif-R and should amplify a product of 1.7 or 1.2 kb if it worked or if it did not, respectively. As shown in the <A href="BioResults#BBa_K1744001_insertion"> results</A>, all the recombinants tested were good. You can also see in the <A href="BioResults#BBa_K1744001_insertion">results</A> that the colonies with BBa_K1744001 inserted in single copy in the genome were not really blue. Thus, the marker <I>amilCP</I>, with our promoter and RBS, can not be used as a positive marker in recombineering experiments, but it could serve us as a plasmidic marker to avoid plasmid background and only the gene <I>aph3’-I</I> would serve us as a positive recombineering marker to help us select the good recombinants. Using the part in that fashion, if the colonies would be blue, it would be plasmidic background and if they would be white, then they would be recombinants (since the expression of amilCP in single copy is not visible).
</P><A href="#Biology">Back to top</A><H4>BBa_K1744002’s construction</H4><P style="text-align:justify"><A href="Parts#BBa_K1744002">This part</A> contains a portion of BBa_K1744000 and the whole sequence of BBa_K1744001. It was designed to contain the advantages of both preceding parts : the working <I>vcrx028</I> kill switch and the dual positive marker amilCP-aph. To build it, we used BBa_K1744000 as template in a PCR with the primers pBAD30-vcrx028-igem-ship-F and vcrx028-kan-amilCP-IGEM-ship-R1 to obtain a part of this plasmid that contains the kill switch without the resistance marker. Then another PCR was done on BBa_K1744001 with the primers vcrx028-kan-amilCP-IGEM-ship-F2 and vcrx028-kan-amilCP-IGEM-ship-R2 (see <A href="PartsDesign#Primers"> the primer table </A> for details on the primers) to get its whole sequence. Both products were purified by <A href="#SPRI">SPRI</A>. Then the two purified products were used in a <A href="#Gibson">gibson assembly</A> with the linearized pSB1C3 provided by the IGEM team to obtain the whole sequence of BBa_K1744002 in pSB1C3. The gibson reaction was then entirely transformed in chemically competent cells (EC100Dpir+ strain) and plated on LB kanamycin, chloramphenicol and glucose 5%. The clones were confirmed by digestion and lyophilised for IGEM shipping. They were not tested for cassette efficiency this time because we did not have time to do it. Unfortunately, we had problems amplifying the cassette in order to integrate it in the genome so we did not have the chance to test the cassette. Although it is not tested, we are confident that the cassette should work at least as good as BBa_K1744000.
</P><A href="#Biology">Back to top</A><SPAN id="Bio_Protocol"> </SPAN><H2>Biology Protocols</H2><UL><LI><A href="#toxins_test">Recombineering cassette test</A></LI><LI><A href="#Medium">Medium and reagent use</A></LI><LI><A href="#PCR">PCR general protocol</A></LI><LI><A href="#Chem_transfo">Chemical transformation</A></LI><LI><A href="#Electro_transfo">Electroporation</A></LI><LI><A href="#Recombineering">Recombineering</A></LI><LI><A href="#miniprep">DNA miniprep</A></LI><LI><A href="#restriction_digest">DNA digestion with restriction enzymes</A></LI><LI><A href="#Gibson">Gibson assembly</A></LI><LI><A href="#SPRI">SPRI</A></LI><LI><A href="#Chelex">Chelex DNA extraction</A></LI></UL><SPAN id="toxins_test"> </SPAN><H3>Recombineering cassette test</H3><P style="text-align:justify">
To test the effect of the toxins on the cells containing the different kill swiches:
</P><UL><LI>2 ul of culture was used to inoculate 198 ul of LB-ampicillin-Glucose 5% or LB-ampicillin-arabinose 1%</LI><LI>That was done in a 96-wells plate put at 30°C</LI><LI>The OD[600] was measured at different intervals during 16-24 hours using a plate reader</LI></UL><A href="#Biology">Back to top</A><SPAN id="Medium"> </SPAN><H3>Medium and reagent use</H3><H4>Medium</H4><P style="text-align:justify">
The only medium used in this project was LB in broth and agar form. Both come from BIOBASIC, their catalog numbers are SD7002(S518) and SD7003(S519).
</P><H4>Antibiotics</H4><P style="text-align:justify">
Here are the complete list of antibiotics used in this project with the working concentration for each:
</P><UL><LI>Ampicillin: 100 µg/mL</LI><LI>Chloramphenicol: 34 µg/mL</LI><LI>Kanamycin: 50 µg/mL</LI><LI>Spectinomycin: 50 µg/mL</LI></UL><H4>Other Reagents</H4><UL><LI>D-Glucose: used at 5% w/v</LI><LI>L-Arabinose: used at 1% w/v</LI><LI>Molecular Grade Water</LI><LI>TFBI:30 mM KOAc, 100 mM RbCl, 10 mM CaCl2, 50 mM MnCl2, 15 % glycerol, pH 5.8 (adjusted with acetic acid), 0.22 µm filtered.</LI><LI>TFBII:10 mM MOPS or PIPES, 75 mM CaCl2, 10 mM RbCl, 15 % glycerol, pH 6.5 (adjust with KOH), 0.22 µm filtered.</LI></UL><H4>Strains</H4><UL><LI><I>Escherichia coli K-12 substrain EC100Dpir+</I></LI><LI><I>Escherichia coli K-12 substrain BW25113</I></LI></UL><A href="#Biology">Back to top</A><SPAN id="PCR"> </SPAN><H3>PCR general protocol</H3><P style="text-align:justify">
The polymerase used for all experiment was Veraseq from Enzymatics<SUB>™</SUB>. This polymerase adds 2 kb of DNA per minute. PCR were always done following this fact. All PCR except those specified in the text were made with 50 Celsius degrees as annealing temperature. The recipe for all PCR mix is as follow:
</P><TABLE id="”t01”"><TBODY><TR><TH>Component</TH><TH>Volume for 1 Reaction</TH></TR><TR><TD>Molecular Grade Water</TD><TD>17.5 µL</TD></TR><TR><TD>5X Veraseq Buffer</TD><TD>5 µL</TD></TR><TR><TD>10 mM dNTP</TD><TD>0.5 µL</TD></TR><TR><TD>Forward Primer</TD><TD>0,5 µL</TD></TR><TR><TD>Reverse Primer</TD><TD>0.5 µL</TD></TR><TR><TD>Template DNA (1 ng/µL)</TD><TD>1 µL</TD></TR><TR><TD>Veraseq Polymerase</TD><TD>0.25 µL</TD></TR></TBODY></TABLE><P style="text-align:justify">
The PCR mixes were all done on ice and we followed manufacturer’s recommendation for denaturation and elongation temperature.
</P><A href="#Biology">Back to top</A><SPAN id="Chem_transfo"> </SPAN><H3>Chemical transformation</H3><P style="text-align:justify">
For TFBI and TFBII composition, please refer to other reagents section. Sometimes, chemical transformation can be an advantage compared to electroporation. Basically, the DNA source of the transformation doesn’t need to be pure for a chemical transformation and can be highly concentrated in salt. You can put as much a 10% of the total volume in DNA in each transformation. Here is the total procedure used in our lab to prepare chemically competent cells:
</P><UL><LI>Inoculate a single colony into 5 ml LB broth.</LI><LI>Incubate 37°C O/N with 200 rpm agitation.</LI><LI>Subculture the O/N 1:100 in LB + 6 mM MgSO4 (typically 250 ml).</LI><LI>Grow to OD600 = 0,48 (0,4-0,6 is good, should take 2-3h).</LI><LI>Centrifuge 6,000 rpm 5 min at 4°C.</LI><LI>Gently resuspend pellet in 1/2,5 volume unit ice cold TFBI. (for 250 ml, use 100 ml TFBI ; 50 ml/bottle.</LI><LI>. Combine the resuspended cells in one bottle. Keep all steps on ice. Incubate on ice for 5 min.</LI><LI>Centrifuge 5,000 rpm 5 min 4°C.</LI><LI>Resuspend pellet in 1/25 original volume ice cold TFBII (for 250 ml original, use 10 ml TFBII).</LI><LI>Incubate on ice 15-60 min. Put 100 ul per tube and flash freeze tubes with liquid nitrogen. Store at -80°C.</LI><LI>When needed, cells can be thawed 15 minutes on ice before using them for transformation</LI><LI>Add as much as 10 µL of DNA to a cell aliquot</LI><LI>Incubated 45 seconds at 42 Celsius degrees then add 1 mL of cold LB</LI><LI>Let cells recuperate 1 hour at 37 degrees before plating them on the right selective plate.</LI></UL><A href="#Biology">Back to top</A><SPAN id="Electro_transfo"> </SPAN><H3>Electroporation</H3><P style="text-align:justify">
To execute this protocol, make sure you have access to an electroporator and be carefull handling the device, as its function is to give electric pulses. 
<UL><LI>Start an overnight pre-culture of your cells to be electroporated at 30 Celsius degrees.</LI><LI>Once fully grown (after overnight incubation), inoculated a 4 mL LB broth per needed electroporation by diluting the pre-culture 1:50.</LI><LI>Incubate the cells at 30 Celsius degrees with 200 rpm agitation until it has an optical density (OD) of between 0.4 and 0.8 using a 1 cm optical cuvette at 600 nm in a spectrophotometer.</LI><LI>Then, transfer your culture in a falcon and incubated on ice for 15 minutes</LI><LI>Spin your cells at 7000 x g for 5 minutes.</LI><LI>Aspirate the supernatant and wash the pellet with 1 mL of cold sterile water.</LI><LI>Transfer the cells in a 1.5 mL microtube and centrifuge at 10 000 x g for 1 minute</LI><LI>Get rid of the supernatant and wash again with 1 mL of cold sterile water</LI><LI>Centrifuge at 10 000 x g for 1 minute</LI><LI>Get rid of the supernatant and wash again with 1 mL of cold sterile water</LI><LI>Centrifuge at 10 000 x g for 1 minute</LI><LI>Get rid of the supernatant and resuspend your cells in 40 µL of cold sterile water per needed electroporation (which brings them to 1000X concentration)</LI><LI>Dispatch 40 µL of cells in microtubes and add 1 µL of DNA 50 ng/µL (if possible).</LI><LI>Transfer the cells into 1 mm electroporation cuvettes.</LI><LI>Electroporate using 1,8 kV, 200 Ω, 25 µF and target 5.0 ms.</LI><LI>Resuspend cells in 1 mL of LB and let them recuperate 1 hour at 30 Celsius degrees (or 37 Celsius degrees).</LI><LI>Plate 200 µL of the transformation on a LB agar plate containing the right selection antibiotic.</LI></UL><A href="#Biology">Back to top</A><SPAN id="Recombineering"> </SPAN></P><H3>Recombineering</H3><P style="text-align:justify">
The recombineering technique resembles a lot the electroporation one. There is two major adds and changes to the protocol. This comes from the fact that recombineering uses lambda red system. This system is (in our case) carried by a pSIM plasmid. Those plasmid are thermosensible (the origin of replication do not work at 37 degrees Celsius) and the lambda red system is heat induced. Those two fact explains the major changes in the protocol. The recombineering technique was developed by <A herf="”http://www.ncbi.nlm.nih.gov/pubmed/10829079”">Datsenko <I>et al.</I> (2000)</A> and uses three protein for the lambda phage: <I>bet, gam, exo</I>. <I>exo</I> firstly degrades one of the two strands of a double stranded DNA fragment through its 5’-&gt;3’ exonuclease activity. Then,<I> bet</I> will bind the single stranded DNA and insert it in the genome. <I> gam </I>, for instance, will inhibit endogenous nucleases.
</P><UL><LI>Start an overnight pre-culture of your cells to be electroporated at 30 Celsius degrees.</LI><LI>Once fully grown (after overnight incubation), inoculated a 4 mL LB broth per needed electroporation by diluting the pre-culture 1:50.</LI><LI>Incubate the cells at 30 Celsius degrees with 200 rpm agitation until it has an optical density (OD) of between 0.4 and 0.8 using a 1 cm optical cuvette at 600 nm in a spectrophotometer.</LI><LI>Transfer the culture into a 42 Celsius degrees agitative bath and incubate 15 minutes with 180 rpm agitation.</LI><LI>Then, transfer your culture in a falcon and incubated on ice for 15 minutes</LI><LI>Spin your cells at 7000 x g for 5 minutes.</LI><LI>Aspirate the supernatant and wash the pellet with 1 mL of cold sterile water.</LI><LI>Transfer the cells in a 1.5 mL microtube and centrifuge at 10 000 x g for 1 minute</LI><LI>Get rid of the supernatant and wash again with 1 mL of cold sterile water</LI><LI>Centrifuge at 10 000 x g for 1 minute</LI><LI>Get rid of the supernatant and wash again with 1 mL of cold sterile water</LI><LI>Centrifuge at 10 000 x g for 1 minute</LI><LI>Get rid of the supernatant and resuspend your cells in 40 µL of cold sterile water per needed electroporation (which brings them to 1000X concentration)</LI><LI>Dispatch 40 µL of cells in microtubes and add 1 µL of DNA 50 ng/µL (if possible).</LI><LI>Transfer the cells into 1 mm electroporation cuvettes.</LI><LI>Electroporate using 1,8 kV, 200 Ω, 25 µF and target 5.0 ms.</LI><LI>Resuspend cells in 1 mL of LB and let them recuperate 1 hour at 30 Celsius degrees (or 37 Celsius degrees if you want to loose the pSIM plasmid).</LI><LI>Plate all of the transformation on a LB agar plate containing the right selection antibiotic (you can spin the cells and resuspend them in 100 µL to concentrate them).</LI></UL><A href="#Biology">Back to top</A><SPAN id="miniprep"> </SPAN><H3> DNA miniprep</H3><P style="text-align:justify">
The DNA minipreps were all done using the EZ-10 Spin Colomn Miniprep Kit<SUB>™</SUB>. Those are available commercially and distributed by Biobasic<SUB>™</SUB>. We have strictly followed manufacturer’s recommendation and obtained 50 µL of 50 ng/µL on average.
</P><A href="#Biology">Back to top</A><SPAN id="restriction_digest"> </SPAN><H3>DNA digestion with restriction enzymes</H3><P style="text-align:justify">
It may be necessary in some cases to digest the vector in order to construct a new part. We have sometimes used restriction enzymes to do so. All restriction enzymes used were from NEB and digestions were all carried in 1X CutSmart Buffer and incubated 1 hour at 37 Celsius degrees.
</P><UL><LI>8 µL of Template DNA</LI><LI>1 µL of 10X Cutsmart Buffer</LI><LI>1 µL of restriction enzymes</LI><LI>Mix and incubate at 37 Celsius degrees 1 hour</LI></UL><A href="#Biology">Back to top</A><SPAN id="Gibson"> </SPAN><H3>Gibson assembly</H3><P style="”text-align:justify”">
Gibson assembly was used for all our plasmid construction. It is now a well known technique that uses an exonuclease to degrade 5’-&gt;3’ one strand of DNA, permitting the homology regions to merge. Than a polymerase will fill the sequence left empty by the exonuclease and a ligase will join both strands together. In our lab, we are using NEBuilder HiFi DNA Assembly cloning kit from NEB. The procedures are specified by the manufacturer, but here is how we do it:
</P><UL><LI>Design your primers to have at least 20 bp of homology on both sides for the adjacent part on your futur DNA construction. You can use synthetic tags if you want to maximise the success rate.</LI><LI>Prepare your insert(s) and vector the way you want (digestion, PCR amplification, gBlock synthesis, etc). As long as they have all 20 bp of homology for each other</LI><LI>Purify all your DNA samples. In our lab, we use Agencourt’s SPRI beads with 1:1 ratio.</LI><LI>Dose the DNA with a precise method (we use Nanodrop dosage).</LI><LI>Mix all the DNA fragments at 0,08 pmol of DNA each.</LI><LI>add water to have 5 µL total.</LI><LI>Add 5 µL of NEBuilder mix.</LI><LI>Incubate 1 hour at 55 Celsius degrees.</LI><LI>Transform all 10 µL in chemically competent cells (heat shock at 42 Celsius degrees for 45 seconds, then resuspend the cells in 1 mL of LB and then recuperate 1 hour at 37 celsius degrees).</LI><LI>Plate all the cells on a LB agar plate containing the right selective antibiotics</LI></UL><A href="#Biology">Back to top</A><SPAN id="SPRI"> </SPAN><H3>SPRI DNA purification</H3><P style="”text-align:justify”">
The SPRI method for DNA purification that uses the negative charge of the DNA to bind it with PEG onto magnetic beads. The beads (carrying DNA) can be washed of PEG to release DNA when we need it. Furthermore, the longer the DNA molecule, the more affinity it will have for the beads because it carries more negative charge than smaller DNA fragments. So, the method can be used, not only to purify DNA, but also to get rid of primers and non-specific bands that are often smaller than the band of interest. The DNA yield of this method is known to be far better than column of band excision techniques. The full protocol is available with the kit purchase.
</P><UL><LI>IMPORTANT NOTE: This technique can be affected by the buffer in which the DNA lies. Dilute digestion samples 1:2 at least to get better DNA yield.</LI><LI>Mix your DNA 1:1 volume with the washed beads.</LI><LI>Incubate 5 minutes at room temperature.</LI><LI>Place your tube on a magnetic rack and wait for the beads to aggregate on the side of the tube.</LI><LI>Get rid of the supernatant and wash with twice the volume of EtOH 80% two times.</LI><LI>Get rid of all the EtOH and let dry at room temperature.</LI><LI>Once dry, resuspend the beads in molecular grade water in the volume needed for further experiments.</LI><LI>Transfers your tube on the magnetic rack again and wait for the beads to aggregate on the side of the tube.</LI><LI>Transfers the supernatant in another tube (it now contains DNA)</LI><LI>You can dose your DNA with your favorite method.</LI></UL><A href="#Biology">Back to top</A><SPAN id="Chelex"> </SPAN><H3>Chelex DNA purification</H3><P style="”text-align:justify”">
Chelex DNA purification is a quick and cheap method for DNA purification. It uses Chelex beads (available commercially) and heat to extract poor quality DNA for PCR screening only. The principle under this method is that chelex beads display chelator molecules that bind to the ions released by the cells that could inhibit PCR. It also inhibit nuclease by chelating the needed co-factor so it preserves DNA longer. The cells are heated up to denature it's proteins and lysate it so in the supernatant we get only the partially denatured DNA with denatured proteins.
</P><UL><LI>Mix 10 µL of dense culture (overnight is far more than sufficient) to 100 µL of Chelex (5%w/v in water)</LI><LI>Incubate 25 minutes at 56 degrees Celsius</LI><LI>Incubate 10 minutes at 100 degrees Celsius (you can put your tube in the PCR to make a program that does both incubations for you)</LI><LI>Vortex mix your tubes.</LI><LI>Spin down the chelex beads in a quick-spin microtube centrifuge.</LI><LI>Transfers 50 µL of the supernatant in another microtube. You can use 1 µL for each PCR as template DNA</LI></UL><A href="#top_menu_under">Back to top</A><H2>Also in &quot;Project&quot;</H2><UL><LI><A href="Description">Project Description</A></LI><LI><A href="Experiments">Hardware Experiments &amp; Protocols</A></LI><LI><A href="Results">Hardware Results</A></LI><LI><A href="BioResults">Biology Results</A></LI><LI><A href="Design">Hardware Design</A></LI><LI><A href="PartsDesign">Parts Design</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>