"
Page
Discussion
View source
History
teams
Log in
 
Team:Columbia-Cooper-NYC/Columbia notebook 2
From 2012.igem.org
Home
Team
Biographies Facilitators The University(s) Official Profile
Project
Overview Data and Conclusions Future Directions References
Parts
Parts Submitted
Lab Notebook
Genetics Copper Etching Protocols
Human Practice
Outreach
Safety
Safety
Sponsors
Sponsor Us Acknowledgements
iGEM
Columbia Genetics Lab Notebook
July, 2012
Thursday, 5th
Re-hydrated plasmids with 50µl of LB and Kanamycin solution
Stored solution at 37°C incubator overnight
Friday, 6th
Purified pET26b vector using standard DNA purification protocol
Monday, 9th
Received kill gene Bba-K124017 from plate 3, 20M
Re-hydrated DNA according to standard iGEM re-hydration protocol
Tuesday, 10th
Contacted professors at Germany in hopes to receive copies of fungal phytochrome FphA
Wednesday, 11th
Received confirmation by professors at Germany for FphA to be sent to Columbia University
Conducted transformation using electroporation with competent bacteria (marked by resistance to Kanamycin)
Control: 1µl of deionized water with abt. and 60µl of bacteria cells
Variable: 1µl of re-hydrated kill gene and 60µl of bacteria cells
Placed both samples after electroporation into 200µl of preprepared LB
Placed samples in shaker 37°C for 30 minutes
Thursday, 12th
Grew 1 colony of transformed bacteria in 5mL of Kanamycin and LB solution
Note: Using pET20b vector over pET26b vector from glycerol stock solution
Friday, 13th
Isolated 4 samples of plasmid using standard plasmid isolation protocol
2 samples: kill gene
2 samples: pET20b vector
Monday, 16th
Re-hydrated two biobrick parts in plasmid pSB2K3 according to standard iGEM re-hydration protocol
BBa-I16009 (PcyA) from plate 1, 20F
BBa-I16008 (ho1) from plate 2, 13J
Electroporated 1µl of each biobrick into separate E. coli at 1800V
Added 100µl LB broth into each sample
Placed samples at 33.4°C for 20 minutes
Samples were plated to be grown overnight
Tuesday, 17th
Placed 5ml each of LB/Kan into two centrifuge tube for PCB creation
Label P: PcyA
Label h: ho1
Placed samples in 37°C incubator
Wednesday, 18th
Purified ho1 and PcyA plasmids using standard DNA purification protocol
Placed purified DNA into glycerol stock (LB/Kan) and stored at -80°C
Thursday, 19th
Purified GFP using standard DNA purification protocol
Prepared glycerol stock solution (500µl GFP/500µl 80% glycerol) and stored at -80°C
Tuesday, 24th
Re-hydrated four biobrick parts according to standard iGEM re-hydration protocol
Inducible plasmid (pSB1AK3-J04500) from plate 4, 12A
GFP (pSB1A2-E0040) from plate 1, 14K
High copy plasmid pSB1T3-J044500 from plate 1, 7A
Low copy plasmid (pSB3C5-J044500) from plate 1, 3C
Electroporated competent E. coli with each of the four above genes separetely
Created Kan, Amp, Cam, Tetra, and Amp/Kan plates
Wednesday, 25th
Streaked pSB1T3-J04450
Created LB solution with Kan or Amp or Cam
Thursday, 26th
Prepared Glycerol stock for inducible promoter, GFP, and low-copy plasmid
Picked a single colony from pSB1T3-J04450 and let it grow overnight in 37C
Recorded and measured the DNA concentrations of following at 260nm:
Kill gene
PcyA
ho1
GFP
Inducible promoter
Low copy CAM plasmid
Followed digestion and ligation protocol; setup explained below:
Upstream: ho1; Downstream: PcyA; Destination plasmid: Low-copy CAM
Upstream: Inducible promoter; Downstream: GFP; Destination plasmid: Low-copy CAM
Upstream: Inducible promoter; Downstream: Kill; Destination plasmid: High-copy TET (pSB1T3)
After completion, store samples at -20C
Friday, 27th
Electroporated ligated samples from previous day: GFP, PCB, Kill
Followed standard plasmid isolation protocol for pSB1T3-J04450 (TET plasmid)
Saturday, 28th
Check plates from electroporation from previous day
Monday, 30th
Stored pif3 and phyB that arrived from Sweden
Relocated and reorganized the iGEM biobrick kit and glycerol stocks
Picked colonies for following DNA:
Inducible promotor (pSB1AK3_J04500)
GFP (pSB1A2_E0040)
Low copy CAM plasmid (pSB3C5_J04150)
Added 10µl of ho1 and PcyA to 5ml of antibiotics
Electroporated following DNA:
Inducible promoter IPTG/kill gene in BL21 cells
Inducible promoter IPTG/GFP in BL21 cells
PCB in α cells
Transferred successful electroporated cells to eppendorf tube with 100μL of LB and placed in 37C shaker
Tuesday, 31st
Prepared glycerol stock of the following:
GFP
Inducible promoter (IPTG)
Low copy CAM plasmid
pcyA
ho1
Isolated ho1 and PcyA using standard plasmid isolation protocol
August, 2012
Wednesday, 1st
Applied IPTG to samples of bacteria with GFP or kill gene and placed back in incubator at 37C
Electroporated the following plasmids:
IPTG inducible promoter/kill gene into BL21 cells
IPTG inducible promoter/GFP into BL21 cells
phyB into α select cells
pif3 into α select cells
Followed the PCR purification protocol for samples 1, 2 listed above
Chemically transformed the following using standard heat shock protocol provided by Bioline:
IPTG inducible promoter/kill gene into BL21 cells
IPTG inducible promoter/GFP into BL21 cells
Plated PCB onto LB/CAM plate
Plated Pif3 and phyB on KAN plates
Plated 500μL of GFP containing cells among two plates each LB/CAM (with and without IPTG)
Placed all samples in 37C incubator
Thursday, 2nd
Selected colonies from PhyB from the LB/Kan plate
Prepared CAM plates
Selected colonies from GFP control
Received agar stabs from Uppsala
Friday, 3rd
Placed 60μL of IPTG to half of control plate for reconfirmation of results
Streaked GFP (with IPTG inducible promoter) and Pif3 on plates
Saturday, 4th
Streaked all 6 Uppsala (labelled below) parts from stabs onto plates:
Upps 1: pSB1K3-B0034-YF1-B0034-FixJ
Upps 2: pSB1K3-YF1
Upps 3: pSB1K3-FixJ
Upps 4: pSB1C3-PfixK2
Upps 5: pSB1A3-amilCP
Upps 6: pSB1C3 - amilGFP
Prepared glycerol stock for phyB (ETHZ)
Isolated phyB by following standard plasmid isolation protocol
Electroporated fphA and ho1 from Germany
Created LB/TET, LB/CAM, LB/Amp, LB/Kan plates
Sunday, 5th
Pulled colonies from 6 iGEM parts and ho1 from Germany
Streaked fphA from Germany
Pulled colonies for Upps 2, 3, 4, 5, 6
Streaked Upps 1, 3, 4, 6
Parafilmed all plates and placed in 4C
Sorted and threw out unnecessary plates
Monday, 6th
Pulled colonies from fphA and Upps 1
Isolated plasmids for the following samples following standard plasmid isolation protocol:
6 iGEM parts
Upps 2-6
ho1
Reconfirmed that TET plates are valid
Tuesday, 7th
Prepared glycerol stock and isolated the plasmid using standard plasmid isolation protocol for following:
fphA from Germany
Upps 1 (pSB1K3-B0034-YF1-B0034-FixJ)
Prepared digestion by measuring OD at 260nm for following:
Upps 1
Upps 4
Upps 5
Upps 6
Kill Gene
Inducible promoter containing plasmid
Prepared digestions and ligated with following setup using the 3A assembly protocol provided from Bioline:
Upstream: Upps 4; Downstream: Upps 5; Destination Plasmid: High Copy TET plasmid
Upstream: Upps 4; Downstream: Upps 6; Destination Plasmid: High Copy TET plasmid
Upstream: Upps 4; Downstream: Kill gene; Destination Plasmid: Low Copy CAM plasmid
Upstream: Inducible promoter; Downstream: Upps 5; Destination Plasmid: High Copy TET plasmid
Upstream: Inducible promoter; Downstream: Upps 6; Destination Plasmid: High Copy TET plasmid
Upstream: Inducible promoter; Downstream: Kill gene; Destination Plasmid: High Copy TET plasmid
Wednesday, 8th
Pulled colony from GFP streak
Isolated following using standard plasmid isolation protocol:
Low Copy CAM plasmid
High Copy TET plasmid
Inducible promoter
Checked optical density and applied necessary dilutions for GFP
Prepared two samples of GFP: sample with IPTG and sample without IPTG
Place samples in shaker to grow overnight
Thursday, 9th
Diluted GFP solutions to match proper optical density
Pulled colony for ho1 as backup and place in 37C inbucator
Friday, 10th
Remade 200 mL each of antibiotic solutions for CAM, TET, Kan, Amp
Conducted digestions and ligations using the 3A assembly method following the protocols provided by bioline of following:
Upstream: Upps 4; Downstream: Kill gene; Destination Plasmid: High Copy TET plasmid
Upstream: Inducible promoter; Downstream: Upps 5; Destination Plasmid: Low Copy CAM plasmid
Upstream: Inducible promoter; Downstream: Upps 6; Destination Plasmid: Low Copy CAM plasmid
Upstream: Inducible promoter; Downstream; Kill gene; Destination Plamid: Low Copy CAM plasmid
Followed the butanol purification protocol for ligated material containing inducible promoter from 7th and 10th
Saturday, 11th
Reviewed the solutions for diluted GFP and observed no significant results
Reorganized samples in fridges and incubators
Monday, 13th
Chemically transformed competent cells (BL21) with plasmids below using bioline protocol (used 1/2 of recommended amount)
IPTG-Upps 5-Low Copy (CAM)
IPTG-Upps 6-Low Copy (CAM)
IPTG-Kill gene-Low Copy (CAM)
CAM control plasmid
PUC19 control plasmid
Electroporated competent cells (α-select) with plasmids below using bioline protocol
Upps 4-Kill gene-High Copy (TET)
Upps 4-Upps 5-High Copy (TET)
Upps 4-Upps 6-High Copy (TET)
PUC19 control plasmid
High copy TET control plasmid
Note 1: TET control sparked
Note 2: Upps 4-Upps 5-TET sparked
Note 3: Original DNA for Upps 4-Upps 5-TET was pink
Placed transformed samples in growth media and placed in 37°C shaker
Made 60ml of 1% agar gel for running gel electrophoresis (2 rows of 12 wells each noted below for gel) to check digestion and ligation
First row
2 log ladder
Upps 4
blank
High Copy TET plasmid
Upps 4 (2)
Upps 6
High Copy TET plasmid (2)
Upps 4 (3)
Kill gene
High Copy TET plasmid (3)
PCB
High Copy TET plasmid (4)
Second row
2 log ladder
Inducible promoter-IPTG
Kill gene (2)
High Copy TET plasmid (5)
Inducible promoter-IPTG (2)
Upps 6 (2)
High Copy TET plasmid (6)
Inducible promoter-IPTG (3)
Upps 5 (2)
High Copy TET plasmid (7)
Ran gel for 25 minutes at constant 150V
Took picture under UV light
Created TET and CAM plates
Tuesday, 14th
Note: Work done below was conducted at the Kanbar Lab at the Cooper Union
Prepared CAM from .250g of 25mg/mL CAM powder with 10mL of EtOH
Prepared LB/glucose media
Heat shocked DH5α competent cells with Inducible promoter/GFP/Low Copy CAM plasmid
Wednesday, 15th
Diluted bacteria cultures with following plasmids to 200x LB/CAM and placed in 37°C shaker
IPTG-Upps 5-Low Copy (CAM)
IPTG-Upps 6-Low Copy (CAM)
IPTG-Kill gene-Low Copy (CAM)
Purified above plasmids and CAM control plasmid using standard purification protocol and prepared glycerol stocks
Added appropriate buffers to IPTG-Upps 5 and IPTG-Upps 6 and centrifuged for 10 minutes to determine for a pellet
Measured OD 600 of following samples
IPTG-Upps 5-Low Copy (CAM): .160A
IPTG-Upps 6-Low Copy (CAM): .038A
Inserted 1µl of 1M IPTG into cultures
Placed all samples in 37°C overnight
Note: conducted the below procedures at the Kanbar Lab
Observed no growth for inducible promoter/GFP/low copy CAM plasmid
Thursday, 16th
Measured OD 600 for 200x diluted bacterial solution containing plasmids with promoter inducible with IPTG
IPTG-Upps 5-Low Copy (CAM): .040A
IPTG-Upps 6-Low Copy (CAM): .032A
IPTG-Kill gene-Low Copy (CAM): .028A
Noted that cell concentration was dense, decided to dilute solution with 75µl cells and 925µl LB/CAM solution
Placed diluted cell solution into 37°C incubator for 40 minutes
Inserted 1µl of 1M IPTG into cultures
Placed all samples in 37°C overnight
Friday, 17th
Note: IPTG induced I-U5 and I-U6 appear to give no color change (no expression)
Electroporate the following genes:
Upps 4-Upps 6-High copy TET (sparked first time, redid trial)
Upps 4-Kill-High copy TET
Upps 4-Kill-Low copy CAM
pUC19 control DNA (AMP resistance)
Placed all samples in 37°C overnight
Note2: Only adding 25µl of competent cells instead of 50µl mentioned in the
Saturday, 18th
Observed none of the electroporated DNA grew, but observed colonies for control
Placed all samples back in the 37°C incubator
Monday, 20th
Measured the optical density for the following:
Upps 6 (with inducible promoter)
Kill gene
GFP
Upps 5 (with inducible promoter)
Diluted the concentrated primer into a primer stock that would be used for DNA sequencing
Sent following DNA for sequencing (at Genewiz)
Inducible promoter-GFP-Low Copy CAM plasmid
Inducible promoter-Upps 5-Low Copy CAM plasmid
Inducible promoter-Upps 6-Low Copy CAM plasmid
Inducible promoter-Kill gene-Low Copy CAM plasmid
Analyzed gel and concluded following:
The digestion for TET high copy plasmid did not work
The digestion for Upps 4, 5, 6 appeared to have worked
All other digestions are inconclusive results
Tuesday, 21st
Made additional Amp/CAM and Amp/TET plates
Measured the optical density for the following:
Upps 1
Kill gene
High copy TET plasmid
Followed the standard digestion protocol provided by bioline or the following:
Upps 1 (Upstream)
Kill gene (Downstream)
High copy TET plasmid (Destination plasmid)
Re-organized iGEM boxes by throwing out previous digestions of High Copy TET plasmids
Re-hydrated following biobricks from standard rehydration protocol:
Constitutive promoter-BBa_J130002 (P1, 13B)
Low-copy TET plasmid-PSB3T5 (P1, 7C)
High-copy CAM plasmid-PSB1C3 (P1, 3A)
High-copy Amp/CAM plasmid-PSB1AC3 (P1, 9A)
High-copy Amp/TET plasmid-PSB1AT3 (P1, 13A)
Made 60ml of 1% agar gel for running gel electrophoresis (2 rows of 12 wells each noted below for gel) to check digestion and ligation
First row
2 log ladder
old TET high copy plasmid
old Kill gene
Upps 5
IPTG inducible promoter
Second row
2 log ladder
newly digested TET high copy plasmid
newly digested Kill gene
Upps 1
Ran gel for 25 minutes at constant 150V
Took picture under UV light
Wednesday, 22nd
Conducted ligations for the following using a vector to insert ratio of 1:3
Sample 1: Inducible promoter (upstream insert)-Kill gene (Downstream insert)-pSB3C5 (vector)
Sample 2: Inducible promoter (upstream insert)-Upps 5 (Downstream insert)-pSB3C5 (vector)
Sample 3: Inducible promoter (upstream insert)-Upps 6 (Downstream insert)-pSB3C5 (vector)
Sample 4: Upps 4 (upstream insert)-Kill gene (Downstream insert)-pSB1T3 (vector)
Sample 5: Upps 4 (upstream insert)-Upps 5 (Downstream insert)-pSB1T3 (vector)
Sample 6: Upps 4 (upstream insert)-Upps 6 (Downstream insert)-pSB1T3 (vector)
Sample 7: Control pSB3C5 (1:0 vector to insert ratio)
Sample 8: Control pSB1T3 (1:0 vector to insert ratio)
Note: above ligations will be done with T4 buffer and ligase from both the Biobrick kit and Columbia's lab
Note 2: label all sample using iGEM ligase and buffer with prefix N
Note 3: label all sample using Columbia lab's ligase and buffer with B
Electroporated the following using the standard protocol provided by Bioline:
Constitutive promoter-BBa_J130002
Low-copy TET plasmid-PSB3T5
High-copy CAM plasmid-PSB1C3
High-copy Amp/CAM plasmid-PSB1AC3
High-copy Amp/TET plasmid-PSB1AT3
PUC19 control plasmid
Chemically transformed the following using the heat shock protocol:
N1, N2, N3, N7, B1, B2, B3, B7 into BL21 (DE) pLysZ orange cells
GFP IPTG given by graduate student at Columbia into BL21 (DE) green cells
PUC19 into BL21 (DE) pLysZ orange cells
PUC19 into BL21 (DE) green cells
Purified ligated protocol for electroporation following QIAquick PCR purification protocol
Plated all chemically transformed and electroporated cells
Thursday, 23rd
Note: Observed no growth with pSB1T3, BL21 pLyse
Note 2: Observed growth with PUC19 and other BL21
Streaked sample N5
Found following 3 restriction sites (to be double checked by graduate student):
NgomIV: CCGCCGGC
EcoRI: GAATTC
PstI: CTGCAG
Friday, 24th
Diluted IPTG driven cultures to 200x with fresh LB/antibiotic solution
Followed standard plasmid isolation protocol for everything except the following:
1 sample of graduate student's GFP
Sample of pSB1C3 (the high copy CAM plasmid)
Checked optical densities of samples B1-B3 and graduate student's GFP
Created two subsamples each of B1-B3; applied 1M IPTG to half of the subsamples
Applied 1M IPTG to into one of the samples of graduate student's IPTG-GFP
Saturday, 25th
Checked IPTG "induced" samples and observed no expression
Measured optical density for following samples and diluted the samples:
GFP by graduate student-pre-dilution: 2.0, post dilution: 0.710
IPTG-kill (N1)-pre-dilution: 2.4, post dilution: 0.736
IPTG-U5 (N2)-pre-dilution: 1.9, post dilution: 0.740
IPTG-U6 (N5)-pre-dilution: 2.4, post dilution: 0.630
IPTG-U5 (B2)-pre-dilution: 1.9, post dilution: 0.681
IPTG-U6 (B3)-pre-dilution: 1.9, post dilution: 0.651
Applied 1.5μL of 1M IPTG with 1.5mL of each sample and place in 25C shaker
Followed standard plasmid isolation protocol for N1 and N5
Monday, 27th
Observe expression of graduate student's GFP
Measured optical densities for following:
Ligated samples:
N1-N3
B1-B3
Mini-prepped:
Upps1-Upps6
pSB1C3
pSB1T3
pSB1AT3
pSB1AC3
pSB1A2
Sent above samples for sequencing
Followed protocol from bioline for digestion:
Downstream: Upps 1
Upstream: Constitutive promoter
Destination plasmid: pSB3T5
Destination plasmid: pSB1C3
Destination plasmid: pSB1AC3
Destination plasmid: pSB1AT3
Streaked out Upps 1 from agar stab
Tuesday, 28th
Reviewed results from the sequencing from Genewiz
Design and send new primer (nev) for sequencing
Conducted ligations for the following using a vector to insert ratio of 1:3
Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB1C3 (vector)
Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB3T5 (vector)
Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB1AT3 (vector)
Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB1AC3 (vector)
Electroporated the above 4 ligations without purifying the ligations
Plated cells into respective plates
Picked colonies for Upps 1-YF1/FixJ
Wednesday, 29th
Conducted ligations for the following using a vector to insert ratio of 1:3
Upps 4 (upstream insert)-Upps 5 (downstream insert)-pSB1C3 (vector)
Upps 4 (upstream insert)-Upps 6 (downstream insert)-pSB1C3 (vector)
Upps 4 (upstream insert)-Kill gene (downstream insert)-pSB1C3 (vector)
Upps 4 (upstream insert)-Upps 5 (downstream insert)-pSB1AC3 (vector)
Upps 4 (upstream insert)-Upps 6 (downstream insert)-pSB1AC3 (vector)
Upps 4 (upstream insert)-Kill gene (downstream insert)-pSB1AC3 (vector)
Extracted Upps 1 (Kan) from glycerol stock from -80C freezer
Followed graduate student's quickchange protocol and labelled N1-N4, E1-E4, P1-P4 with the labeling as following:
Sample 1: HF only
Sample 2: HF and 1.5μL DMSO
Sample 3: GC only
Sample 4: GC and 1.5μL DMSO
Prefix N: NgoMIV mutation
Prefix E: EcoRI mutation
Prefix P: PsH mutation
Thursday, 30th
Followed standard plasmid isolation protocol for Upps 1
Transformed U4-U5, U4-U6, U4-kill in 2 different plasmids
Ran gels of Quickchange product
Column 1: miniprepped plasmid
Column 2-5: N1-N4
Column 6-9: E1-E4
Column 10-13: P1-P4
Re-ligated Constitutive promoter-Upps 1
September, 2012
Wednesday, 5th
Note: All work done from this day on was done at the Kanbar lab at the Cooper Union
Single digested the J13002
Placed the digest above and other digests to be run on gels into -20C freezer
Made 4 50mL 1% agarose gels
Followed standard plasmid isolation protocol for kill gene
Friday, 7th
Ran two gels to check digests
Gel Number 1
Marker
Upps 1
Inducible promoter 1
Inducible promoter 2
pSB3T5
pSB1C3
Low Copy Vector 1
Low Copy Vector 2
Gel Number 2
Marker
Kill gene
PcyA
TetR 1
TetR 2
GFP from biobrick kit
GFP from Tushar
GFP from Kevin
Conducted digestions for the following:
Inducible promoter digested with EcoRI&Spe with Buffer E and Buffer Multi (for 30 min.)
TetR digested with EcoRI with Buffer H
TetR digested with Spe with Buffer B
Upps 1 digested with Xba&Pst with Buffer H
Kill gene digested with Xba&Pst with Buffer H
GFP digested with Xba&Pst with Buffer H
pSB3T5 with EcoRI&Pst with Buffer H
Low copy plasmid 2 digested with EcoRI&Pst with Buffer H
pSB1C3 digested with EcoR&Pst with Buffer H
Note: Digests 7-9 was done in Antartic Phosphotase Treat 30 min in 37C and 15 min in 65C
Conducted ligations for the following (followed by chemical transformations:
Upstream: Inducible promoter; Downstream: Upps 1; Destination plasmid: Low copy plasmid 2
Upstream: Inducible promoter; Downstream: Kill; Destination plasmid: Low copy plasmid 2
Upstream: Inducible promoter; Downstream: GFP; Destination plasmid: Low copy plasmid 2
Conducted additional chemical transformations with the following vectors:
pET20b
pET26b
pUC19
Conducted following additional ligations, reference numbers correlate to numbers shown above for this day's digestions
Label A: 2 and 9
Label B: 3 and 4 and 9
Label C: 1 and 4 and 8
Label D: 1 and 5 and 8
Label E: 1 and 6 and 8
Label F: 3 and 4
Label G: 3 and 5
Label H: 3 and 6
Tuesday, 11th
Observed the following:
Inducible promoter-GFP-Low Copy CAM Plasmid had 3 pink clones (IPTG applied)
Inducible promoter-GFP-Low Copy CAM Plasmid had 3 pink clones (NO IPTG)
Inducible promoter-Upps 1-Low Copy CAM Plasmid had 32 pink clones
Inducible promoter-Kill gene-Low Copy CAM Plasmid had 50 pink clones
Note 1: Destination plasmid/vector contained J04450 biobrick part, explained pink clones
Note 2: All vectors were all treated with Antartic Phophotase, expect no religation of J04450 with LC vector
Digested Kill gene and GFP with E and P
AP Treat Kill gene with X and P, GFP with X and P
Conducted ligations for the following:
GFP (E and P) with pSB1C3 (E and P, AP treated)
GFP (X and D, AP treated) with TetR/s
Condcuted transformations for the following:
Ligation F from 9/7, TetR, Upps 1(X and P)
Kill gene, pSB1C3
GFP, pSB1C3
GFP, TetR
Mini-cultured pET20b and pET26b
Wednesday, 12th
Observed following from the transformations from 9/11
TetR/GFP-successful with about 250 colonies
TetR/Upps 1-successful with about 200 colonies
No growth with Kill&pCB1C3, GFP&pCB1C3
Prepared colony PCR for TetR&GFP, and TetR&Upps 1 on Amp grid plate (had A-D columns and 1-4 rows)
Resuspended single colonies in 4μL deionized water
Transferred 3μL colony suspension to Amp grid
Stored remaining 1μL as PCR template
Grew Amp grid plate in 37C overnight
Note 1: GFP/TetR: A and B with 1-4 each
Note 2: Upps 1/TetR: C and D with 1-4 each
PCR reaction contained following mixture:
1x PCR Rxn
1.0μL of 10x buffer
0.4μL of dNTP
0.4μL of Primer F
0.4μL of Primer R
1.0μL of template mentioned above
0.4μL of tag
6.4μL of deionized water
17x PCR Rxn
17μL of 10x buffer
6.8μL of dNTP
6.8μL of Primer F
6.8μL of Primer R
6.8μL of tag
108.8μL of deionized water
PCR reactions conducted with following protocol:
Denature at 95C for 10 min.
Denature at 95C for additional 30 sec.
Anneal at 56C for 30 sec.
Elongation at 70C for 60 sec.
Elongation at 72C for 20 min.
Hold at 4C for 39 cycles
Set up second half composite construction digests for following:
Upps 1; Enzyme: EcoRI and SpeI; Buffer E*; Ratio: 1:2; Reaction vol.: 30μL
Upps 4; Enzyme: EcoRI and SpeI; Buffer E*; Ratio: 1:2; Reaction vol.: 30μL
Upps 5; Enzyme: Xba and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
Upps 6; Enzyme: Xba and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
Kill gene; Enzyme: Xba and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
Linearlized pSB1T3; Enzyme: EcoRI and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
Linearlized pSB1K3; Enzyme: EcoRI and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
Promoterless GFP; Enzyme: Xba; Buffer H; Ratio: N/A; Reaction vol.: 20μL
pUC19; Enzyme: EcoRI; Buffer H; Ratio: N/A; Reaction vol.: 20μL
Note 3: for * buffers, will use buffer 2 or 4 in future
Reactions took place for 1 hr. in 37C incubation and placed in -20C
Isolated pGLO and pET20b vectors using standard plasmid isolation protocol
Prepared glycerol stocks for pET20b and pET26b
Thursday, 13th
Conducted fphA PCR mutogenesis with following (template = pASKfphA 753bp)
1x PCR Rxn
2.0μL of 10x Buffer
0.4μL of dNTP
0.4μL of Primer F (EcoRI)
0.4μL of Primer R (EcoRI)
1.0μL of template
0.4μL of tag
15.4μL of deionized water
3x PCR Rxn
6.0μL of 10x Buffer
1.2μL of dNTP
1.2μL of Primer F (EcoRI)
1.2μL of Primer R (EcoRI)
1.2μL of tag
46.2μL of deionized water
Set up 4 min. extension time with 55C for annealing temp for PCR program
Digested above mixture with Dpn1 at 37C for 1hr. after PCR
Transformed product into DH5α
Heat inactivated samples with Spe1 restriction digests from 9/12 at 80C for 20 min. and rest at 65C for 20 min.
Antartic treated the following (15 min. 37C, 5 min. 65C):
Upps 4
Promoterless GFP
pUC19
Conducted ligations for the following for a total volume of 10μL
Upps 4-Kill gene-pSB1K3
Upps 4-Upps 5-pSB1K3
Upps 4-Upps 6-pSB1K3
TetR/Spe (from 9/7)-promoterless GFP
pUC19
Transformed ligated DNA into DH5α
Ran gels for PCR Rxn and observed following:
A3, B3 had 300 bp (estimated) product
B1, B4 had 700 bp (estimated) product
Everything else (including marker failed)
Can not differentiate between ±TetR
Minicultured 4 TetR-Upps 1 cultures randomly
Friday, 14th
Sterilized glycerol for stocks
Created more LB/Amp plates
Created more sterile miniculture tubes
Ran 100 bp optimization gel
Note 1: While 2µl DNA visble, should use minimum of 5µl/lane for system (SYBR)
Repeated VF2-VR PCR on TetR&promoterless GFP (denoted A) and TetR&Upps 1 (denoted B)
1x PCR Rxn
1.0μL of 10x Buffer
0.4μL of dNTP
0.4μL of Primer F (EcoRI)
0.4μL of Primer R (EcoRI)
1.0μL of template
0.4μL of tag
6.4μL of deionized water
17x PCR Rxn
17.0μL of 10x Buffer
6.8μL of dNTP
6.8μL of Primer F (EcoRI)
6.8μL of Primer R (EcoRI)
6.8μL of tag
108.8μL of deionized water
Note 2: Will pick 7 colonies from each and have 4 minicultures of B-will use 1µl±1 colony
After review of primers, changed PCR program as follows:
Increased annealing temp to 60C
Increased extension temp to 70C
Reduced cycles to 30
Thursday, 20th
Reviewed order of custom primer from Invitrogen (primer ordered: 9/17)
Rehydrated primers
Ran PCR Rxn with following:
Annealing temperature: 61C
Elongation time: 3 minutes
30 cycles
Monday, 24th
Reviewed plates and observed the following:
Noted plenty of colonies on both plates, as expected
Plates with pSB1C3-J04450 based vector had many pink colonies and 8-10 white clones
Considering both vectors were alkaline phosphatase treated, will assume treatment failed for both vectors
Will currently disregard Upps 3 based vector transformations
Picked following colonies for miniculture:
pSB1C3-FphA
Upps 4-Kill gene
Upps 4-Upps 6
TetR-Upps 1
Minicultured the following from glycerol stocks
Upps 1
Upps 4
TetR
Tuesday, 25th
Ran a gel to verify following plasmids:
Sizing Gel 1
Kill gene 1 (OK)
Kill gene 2 (OK)
FphA1 (too small - discard)
FphA2 (OK)
Upps 1 (too big - discard)
B0034 RBS (OK, faint)
Sizing Gel 2
Upps 4-Kill gene 1 (OK)
Upps 4-Kill gene 2 (OK)
Upps 4-Upps 6 (OK)
Upps 4-Upps 6 (too big - discard)
Upps 4 (OK)
B0030 RBS.1 (OK, but really faint)
Note 1: the parenthesis denote observations/analysis of gel
Isolated plasmids using standard miniprep protocol for the minicultures from previous day
Set up PCR site directed mutagenesis using pSB1C3-FphA as template
Note 2: used protocol from 9/13 and EcoRI primer set
Rehydrated the following using standard iGEM rehydration protocol
BBa_J23100 (plate 1, 18C [Amp])
BBa_J23101 (plate 1, 18E [Amp])
Bba_J23102 (plate 1, 18G [Amp])
Transformed the above rehydrated biobricks into DH5α
Wednesday, 26th
Digested the PCR Rxn with DpnI
Transformed pSB1C3-FphA into DH5α
Digested the following:
BBa_B0034 RBS with SpeI
Upps4 with XbaI
Kill gene with XbaI
Ligated the following (was predigested with EcoRI):
RBS-Upps 4
RBS-Kill gene
Gel extracted the bands of the following:
RBS-Upp4 Target 2352bp, erroneous 2155bp
RBS-Kill Target 3441bp, erroneous 2155bp
Minicultured the following:
Upps 1
Promoterless GFP
Promoters 100-102
Wednesday, 27th
Made minipreps from minicultures from previous day, ran qualitative gel, and concluded the following:
Upps 1 had poor yield
Promoterless GFP showed no growth
Conducted digestions of the following: (yielded 2x volume)
RBS with SpeI
Promoter 102 with SpeI
Upps 1 with XbaI
Promoterless GFP with XbaI
Conducted ligations of the following using digestion with 1x volume of EcoRI: (yielded 2x volume)
Promoter 102-Upps 1
Promoter 102-Promoterless GFP
RBS-Upps 4
Ribosome Binding Site-Kill gene
Ran 1x volume on gel to purify and gel extract target band
Transformed the above in DH5α using standard heat shock protocol
Observed the PCR site mutagenesis yielded no colonies
Friday, 28th
Observed that Ribosome Binding Site-Kill gene worked and everything else did not
Conducted minicultures for the following:
Ribosome Bidning Site-Kill gene
Promoterless GFP with Kan resistance
Upps 1
Upps 4
Conducted digestions of the following:
Upps 1 with XbaI
Inducible Promoter with SpeI
Upps 1 with EcoRI/SpeI
Upps 4 with XbaI/PstI
pSB1A3 (linearized DNA) with EcoRI/PstI
pSB3T5 with EcoRI/PstI
Conducted ligations of the following:
Promoter 100-Upps 1
Promoter 101-Upps 1
Promoter 102-Upps 1
Inducible Promoter-Upps 1
Promoter 102-promoterless GFP
Upps 1-Upps 4-pSB1A3
Upp1-Upp4-pSB3T5
Transformed the above ligated products into DH5α using heat shock protocol
Saturday, 29th
Observed that Promoter 102-GFP worked and everythign else did not
Conducted digestions of the following:
Upps 1 with EcoRI&SpeI
Upps 4-Kill with XbaI&PstI
Upps 4-Upps 6 with XbaI&PstI
pSB1C3-J04450 with EcoRI&PstI
Conducted ligations of the following:
Upps 1-Upps 4-Kill gene-pSB1C3 destination plasmid
Upps 1-Upps 4-Upps6-pSB1C3 destination plasmid
Transformed the above ligated products into DH5α using a heat shock protocol
Transformed the following into JM109:
Promoter 100-Upps 1
Promoter 101-Upps 1
Promoter 102-Upps 1
Inducible Promoter-Upps 1
October, 2012
Monday, 1st
Observed that all of the plates grew colonies
Prepared minicultures of the following:
Upps 1-Upps 4-Kill gene-pSB1C3 destination plasmid
Upps 1-Upps 4-Upps 6-pSB1C3 destination plasmid
Promoter 102-Upps 1
Inducible promoter-Upps 1
Tuesday, 2nd
Isolated following plasmids by following standard plasmid isolation protocol and submitted parts:
pSB1C3-FphA
Upps 4-Kill gene
Upps 1-Upps 4-Kill gene
Upps 1-Upps 4-Upps 6
Promoter 102-Upps 1
Inducible promoter-Upps 1
Retrieved from " http://2012.igem.org/Team:Columbia-Cooper-NYC/Columbia_notebook_2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
