<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Columbia-Cooper-NYC_Columbia_notebook_2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Columbia-Cooper-NYC/Columbia_notebook_2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Columbia-Cooper-NYC/Columbia_notebook_2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Columbia-Cooper-NYC/Columbia_notebook_2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Columbia-Cooper-NYC/Columbia_notebook_2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Columbia-Cooper-NYC/Columbia_notebook_2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Columbia-Cooper-NYC/Columbia notebook 2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><UL id="sddm" style="width:1000px;margin:0 auto"><LI><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC" onmouseover="mopen('m1')" onmouseout="mclosetime()">Home</A></LI><LI><A href="#" onmouseover="mopen('m2')" onmouseout="mclosetime()">Team</A><DIV id="m2" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Team">Biographies</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Facilitators">Facilitators</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Universities">The University(s)</A><A href="https://igem.org/Team.cgi?year=2012">Official Profile</A></DIV></LI><LI><A href="#" onmouseover="mopen('m3')" onmouseout="mclosetime()">Project</A><DIV id="m3" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Overview">Overview</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Data_and_Conclusions">Data and Conclusions</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Future_Directions">Future Directions</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/References">References</A></DIV></LI><LI><A href="#" onmouseover="mopen('m4')" onmouseout="mclosetime()">Parts</A><DIV id="m4" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Parts_Submitted">Parts Submitted</A></DIV></LI><LI><A href="#" onmouseover="mopen('m5')" onmouseout="mclosetime()">Lab Notebook</A><DIV id="m5" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Columbia_notebook_2">Genetics</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Columbia_notebook_1">Copper Etching</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Protocols">Protocols</A></DIV></LI><LI><A href="#" onmouseover="mopen('m6')" onmouseout="mclosetime()">Human Practice</A><DIV id="m6" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Outreach">Outreach</A></DIV></LI><LI><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Safety" onmouseover="mopen('m7')" onmouseout="mclosetime()">Safety</A><DIV id="m7" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Safety">Safety</A></DIV></LI><LI><A href="#" onmouseover="mopen('m8')" onmouseout="mclosetime()">Sponsors</A><DIV id="m8" onmouseover="mcancelclosetime()" onmouseout="mclosetime()"><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Sponsor_Us">Sponsor Us</A><A href="https://2012.igem.org/Team:Columbia-Cooper-NYC/Acknowledgements">Acknowledgements</A></DIV></LI><LI><A href="https://2012.igem.org/Main_Page" onmouseover="mopen('m9')" onmouseout="mclosetime()">iGEM</A></LI></UL></P><H1><SPAN class="mw-headline" id="Columbia_Genetics_Lab_Notebook"> Columbia Genetics Lab Notebook </SPAN></H1><DIV class="content-header-box"><H2><SPAN class="mw-headline" id="July.2C_2012"> July, 2012 </SPAN></H2></DIV><DIV class="content-body-box"><H4><SPAN class="mw-headline" id="Thursday.2C_5th"> Thursday, 5th </SPAN></H4><UL><LI> Re-hydrated plasmids with 50µl of LB and Kanamycin solution
</LI><LI> Stored solution at 37°C incubator overnight
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_6th"> Friday, 6th </SPAN></H4><UL><LI> Purified pET26b vector using standard DNA purification protocol
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_9th"> Monday, 9th </SPAN></H4><UL><LI> Received kill gene Bba-K124017 from plate 3, 20M
</LI><LI> Re-hydrated DNA according to standard iGEM re-hydration protocol
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_10th"> Tuesday, 10th </SPAN></H4><UL><LI> Contacted professors at Germany in hopes to receive copies of fungal phytochrome FphA
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_11th"> Wednesday, 11th </SPAN></H4><UL><LI> Received confirmation by professors at Germany for FphA to be sent to Columbia University
</LI><LI> Conducted transformation using electroporation with competent bacteria (marked by resistance to Kanamycin)
</LI><LI> Control: 1µl of deionized water with abt. and 60µl of bacteria cells
</LI><LI> Variable: 1µl of re-hydrated kill gene and 60µl of bacteria cells
</LI><LI> Placed both samples after electroporation into 200µl of preprepared LB
</LI><LI> Placed samples in shaker 37°C for 30 minutes
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_12th"> Thursday, 12th </SPAN></H4><UL><LI> Grew 1 colony of transformed bacteria in 5mL of Kanamycin and LB solution
</LI><LI> Note: Using pET20b vector over pET26b vector from glycerol stock solution
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_13th"> Friday, 13th </SPAN></H4><UL><LI> Isolated 4 samples of plasmid using standard plasmid isolation protocol
</LI><LI> 2 samples: kill gene
</LI><LI> 2 samples: pET20b vector
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_16th"> Monday, 16th </SPAN></H4><UL><LI> Re-hydrated two biobrick parts in plasmid pSB2K3 according to standard iGEM re-hydration protocol 
</LI><LI> BBa-I16009 (PcyA) from plate 1, 20F
</LI><LI> BBa-I16008 (ho1) from plate 2, 13J
</LI><LI> Electroporated 1µl of each biobrick into separate E. coli at 1800V
</LI><LI> Added 100µl LB broth into each sample
</LI><LI> Placed samples at 33.4°C for 20 minutes
</LI><LI> Samples were plated to be grown overnight
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_17th"> Tuesday, 17th </SPAN></H4><UL><LI> Placed 5ml each of LB/Kan into two centrifuge tube for PCB creation
</LI><LI> Label P: PcyA
</LI><LI> Label h: ho1
</LI><LI> Placed samples in 37°C incubator
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_18th"> Wednesday, 18th </SPAN></H4><UL><LI> Purified ho1 and PcyA plasmids using standard DNA purification protocol
</LI><LI> Placed purified DNA into glycerol stock (LB/Kan) and stored at -80°C
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_19th"> Thursday, 19th </SPAN></H4><UL><LI> Purified GFP using standard DNA purification protocol
</LI><LI> Prepared glycerol stock solution (500µl GFP/500µl 80% glycerol) and stored at -80°C
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_24th"> Tuesday, 24th </SPAN></H4><UL><LI> Re-hydrated four biobrick parts according to standard iGEM re-hydration protocol
</LI><LI> Inducible plasmid (pSB1AK3-J04500) from plate 4, 12A
</LI><LI> GFP (pSB1A2-E0040) from plate 1, 14K
</LI><LI> High copy plasmid pSB1T3-J044500 from plate 1, 7A
</LI><LI> Low copy plasmid (pSB3C5-J044500) from plate 1, 3C
</LI><LI> Electroporated competent E. coli with each of the four above genes separetely
</LI><LI> Created Kan, Amp, Cam, Tetra, and Amp/Kan plates
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_25th"> Wednesday, 25th </SPAN></H4><UL><LI> Streaked pSB1T3-J04450
</LI><LI> Created LB solution with Kan or Amp or Cam
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_26th"> Thursday, 26th </SPAN></H4><UL><LI> Prepared Glycerol stock for inducible promoter, GFP, and low-copy plasmid
</LI><LI> Picked a single colony from pSB1T3-J04450 and let it grow overnight in 37C
</LI><LI> Recorded and measured the DNA concentrations of following at 260nm:
</LI><LI> Kill gene
</LI><LI> PcyA
</LI><LI> ho1
</LI><LI> GFP
</LI><LI> Inducible promoter
</LI><LI> Low copy CAM plasmid
</LI><LI> Followed digestion and ligation protocol; setup explained below:
</LI><LI> Upstream: ho1; Downstream: PcyA; Destination plasmid: Low-copy CAM
</LI><LI> Upstream: Inducible promoter; Downstream: GFP; Destination plasmid: Low-copy CAM
</LI><LI> Upstream: Inducible promoter; Downstream: Kill; Destination plasmid: High-copy TET (pSB1T3)
</LI><LI> After completion, store samples at -20C
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_27th"> Friday, 27th </SPAN></H4><UL><LI> Electroporated ligated samples from previous day: GFP, PCB, Kill
</LI><LI> Followed standard plasmid isolation protocol for pSB1T3-J04450 (TET plasmid)
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_28th"> Saturday, 28th </SPAN></H4><UL><LI> Check plates from electroporation from previous day
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_30th"> Monday, 30th </SPAN></H4><UL><LI> Stored pif3 and phyB that arrived from Sweden
</LI><LI> Relocated and reorganized the iGEM biobrick kit and glycerol stocks
</LI><LI> Picked colonies for following DNA:
</LI><LI> Inducible promotor (pSB1AK3_J04500)
</LI><LI> GFP (pSB1A2_E0040)
</LI><LI> Low copy CAM plasmid (pSB3C5_J04150)
</LI><LI> Added 10µl of ho1 and PcyA to 5ml of antibiotics
</LI><LI> Electroporated following DNA:
</LI><LI> Inducible promoter IPTG/kill gene in BL21 cells
</LI><LI> Inducible promoter IPTG/GFP in BL21 cells
</LI><LI> PCB in α cells
</LI><LI> Transferred successful electroporated cells to eppendorf tube with 100μL of LB and placed in 37C shaker
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_31st"> Tuesday, 31st </SPAN></H4><UL><LI> Prepared glycerol stock of the following:
</LI><LI>GFP
</LI><LI>Inducible promoter (IPTG)
</LI><LI>Low copy CAM plasmid
</LI><LI>pcyA
</LI><LI>ho1
</LI><LI> Isolated ho1 and PcyA using standard plasmid isolation protocol
</LI></UL></DIV><H2><SPAN class="mw-headline" id="August.2C_2012"> August, 2012 </SPAN></H2><DIV class="content-body-box"><H4><SPAN class="mw-headline" id="Wednesday.2C_1st"> Wednesday, 1st </SPAN></H4><UL><LI> Applied IPTG to samples of bacteria with GFP or kill gene and placed back in incubator at 37C
</LI><LI> Electroporated the following plasmids:
</LI><LI> IPTG inducible promoter/kill gene into BL21 cells
</LI><LI> IPTG inducible promoter/GFP into BL21 cells
</LI><LI> phyB into α select cells
</LI><LI> pif3 into α select cells
</LI><LI> Followed the PCR purification protocol for samples 1, 2 listed above
</LI><LI> Chemically transformed the following using standard heat shock protocol provided by Bioline:
</LI><LI> IPTG inducible promoter/kill gene into BL21 cells
</LI><LI> IPTG inducible promoter/GFP into BL21 cells
</LI><LI> Plated PCB onto LB/CAM plate
</LI><LI> Plated Pif3 and phyB on KAN plates
</LI><LI> Plated 500μL of GFP containing cells among two plates each LB/CAM (with and without IPTG)
</LI><LI> Placed all samples in 37C incubator
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_2nd"> Thursday, 2nd </SPAN></H4><UL><LI> Selected colonies from PhyB from the LB/Kan plate
</LI><LI> Prepared CAM plates
</LI><LI> Selected colonies from GFP control
</LI><LI> Received agar stabs from Uppsala
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_3rd"> Friday, 3rd </SPAN></H4><UL><LI> Placed 60μL of IPTG to half of control plate for reconfirmation of results
</LI><LI> Streaked GFP (with IPTG inducible promoter) and Pif3 on plates
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_4th"> Saturday, 4th </SPAN></H4><UL><LI> Streaked all 6 Uppsala (labelled below) parts from stabs onto plates:
</LI><LI> Upps 1: pSB1K3-B0034-YF1-B0034-FixJ
</LI><LI> Upps 2: pSB1K3-YF1
</LI><LI> Upps 3: pSB1K3-FixJ
</LI><LI> Upps 4: pSB1C3-PfixK2
</LI><LI> Upps 5: pSB1A3-amilCP
</LI><LI> Upps 6: pSB1C3 - amilGFP
</LI><LI> Prepared glycerol stock for phyB (ETHZ)
</LI><LI> Isolated phyB by following standard plasmid isolation protocol
</LI><LI> Electroporated fphA and ho1 from Germany
</LI><LI> Created LB/TET, LB/CAM, LB/Amp, LB/Kan plates
</LI></UL><H4><SPAN class="mw-headline" id="Sunday.2C_5th"> Sunday, 5th </SPAN></H4><UL><LI> Pulled colonies from 6 iGEM parts and ho1 from Germany
</LI><LI> Streaked fphA from Germany
</LI><LI> Pulled colonies for Upps 2, 3, 4, 5, 6
</LI><LI> Streaked Upps 1, 3, 4, 6
</LI><LI> Parafilmed all plates and placed in 4C
</LI><LI> Sorted and threw out unnecessary plates
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_6th"> Monday, 6th </SPAN></H4><UL><LI> Pulled colonies from fphA and Upps 1
</LI><LI> Isolated plasmids for the following samples following standard plasmid isolation protocol:
</LI><LI> 6 iGEM parts
</LI><LI> Upps 2-6
</LI><LI> ho1
</LI><LI> Reconfirmed that TET plates are valid
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_7th"> Tuesday, 7th </SPAN></H4><UL><LI> Prepared glycerol stock and isolated the plasmid using standard plasmid isolation protocol for following:
</LI><LI> fphA from Germany
</LI><LI> Upps 1 (pSB1K3-B0034-YF1-B0034-FixJ)
</LI><LI> Prepared digestion by measuring OD at 260nm for following:
</LI><LI> Upps 1
</LI><LI> Upps 4
</LI><LI> Upps 5
</LI><LI> Upps 6
</LI><LI> Kill Gene
</LI><LI> Inducible promoter containing plasmid
</LI><LI> Prepared digestions and ligated with following setup using the 3A assembly protocol provided from Bioline:
</LI><LI> Upstream: Upps 4; Downstream: Upps 5; Destination Plasmid: High Copy TET plasmid
</LI><LI> Upstream: Upps 4; Downstream: Upps 6; Destination Plasmid: High Copy TET plasmid
</LI><LI> Upstream: Upps 4; Downstream: Kill gene; Destination Plasmid: Low Copy CAM plasmid
</LI><LI> Upstream: Inducible promoter; Downstream: Upps 5; Destination Plasmid: High Copy TET plasmid
</LI><LI> Upstream: Inducible promoter; Downstream: Upps 6; Destination Plasmid: High Copy TET plasmid
</LI><LI> Upstream: Inducible promoter; Downstream: Kill gene; Destination Plasmid: High Copy TET plasmid
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_8th"> Wednesday, 8th </SPAN></H4><UL><LI> Pulled colony from GFP streak
</LI><LI> Isolated following using standard plasmid isolation protocol:
</LI><LI> Low Copy CAM plasmid
</LI><LI> High Copy TET plasmid
</LI><LI> Inducible promoter
</LI><LI> Checked optical density and applied necessary dilutions for GFP
</LI><LI> Prepared two samples of GFP: sample with IPTG and sample without IPTG
</LI><LI> Place samples in shaker to grow overnight
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_9th"> Thursday, 9th </SPAN></H4><UL><LI> Diluted GFP solutions to match proper optical density
</LI><LI> Pulled colony for ho1 as backup and place in 37C inbucator
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_10th"> Friday, 10th </SPAN></H4><UL><LI> Remade 200 mL each of antibiotic solutions for CAM, TET, Kan, Amp
</LI><LI> Conducted digestions and ligations using the 3A assembly method following the protocols provided by bioline of following:
</LI><LI> Upstream: Upps 4; Downstream: Kill gene; Destination Plasmid: High Copy TET plasmid
</LI><LI> Upstream: Inducible promoter; Downstream: Upps 5; Destination Plasmid: Low Copy CAM plasmid
</LI><LI> Upstream: Inducible promoter; Downstream: Upps 6; Destination Plasmid: Low Copy CAM plasmid
</LI><LI> Upstream: Inducible promoter; Downstream; Kill gene; Destination Plamid: Low Copy CAM plasmid
</LI><LI> Followed the butanol purification protocol for ligated material containing inducible promoter from 7th and 10th
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_11th"> Saturday, 11th </SPAN></H4><UL><LI> Reviewed the solutions for diluted GFP and observed no significant results
</LI><LI> Reorganized samples in fridges and incubators
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_13th"> Monday, 13th </SPAN></H4><UL><LI> Chemically transformed competent cells (BL21) with plasmids below using bioline protocol (used 1/2 of recommended amount)
</LI><LI> IPTG-Upps 5-Low Copy (CAM)
</LI><LI> IPTG-Upps 6-Low Copy (CAM)
</LI><LI> IPTG-Kill gene-Low Copy (CAM)
</LI><LI> CAM control plasmid
</LI><LI> PUC19 control plasmid
</LI><LI> Electroporated competent cells (α-select) with plasmids below using bioline protocol
</LI><LI> Upps 4-Kill gene-High Copy (TET)
</LI><LI> Upps 4-Upps 5-High Copy (TET)
</LI><LI> Upps 4-Upps 6-High Copy (TET)
</LI><LI> PUC19 control plasmid
</LI><LI> High copy TET control plasmid
</LI></UL><P><I>Note 1: TET control sparked</I></P><P><I>Note 2: Upps 4-Upps 5-TET sparked</I></P><P><I>Note 3: Original DNA for Upps 4-Upps 5-TET was pink</I></P><UL><LI> Placed transformed samples in growth media and placed in 37°C shaker
</LI><LI> Made 60ml of 1% agar gel for running gel electrophoresis (2 rows of 12 wells each noted below for gel) to check digestion and ligation
</LI><LI> First row
</LI><LI> 2 log ladder
</LI><LI> Upps 4
</LI><LI> blank
</LI><LI> High Copy TET plasmid
</LI><LI> Upps 4 (2)
</LI><LI> Upps 6
</LI><LI> High Copy TET plasmid (2)
</LI><LI> Upps 4 (3)
</LI><LI> Kill gene
</LI><LI> High Copy TET plasmid (3)
</LI><LI> PCB
</LI><LI> High Copy TET plasmid (4)
</LI><LI> Second row
</LI><LI> 2 log ladder
</LI><LI> Inducible promoter-IPTG
</LI><LI> Kill gene (2)
</LI><LI> High Copy TET plasmid (5)
</LI><LI> Inducible promoter-IPTG (2)
</LI><LI> Upps 6 (2)
</LI><LI> High Copy TET plasmid (6)
</LI><LI> Inducible promoter-IPTG (3)
</LI><LI> Upps 5 (2)
</LI><LI> High Copy TET plasmid (7)
</LI></UL><LI> Ran gel for 25 minutes at constant 150V
</LI><LI> Took picture under UV light
</LI><LI> Created TET and CAM plates
</LI><H4><SPAN class="mw-headline" id="Tuesday.2C_14th"> Tuesday, 14th </SPAN></H4><P><I>Note: Work done below was conducted at the Kanbar Lab at the Cooper Union</I></P><UL><LI> Prepared CAM from .250g of 25mg/mL CAM powder with 10mL of EtOH
</LI><LI> Prepared LB/glucose media
</LI><LI> Heat shocked DH5α competent cells with Inducible promoter/GFP/Low Copy CAM plasmid
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_15th"> Wednesday, 15th </SPAN></H4><UL><LI> Diluted bacteria cultures with following plasmids to 200x LB/CAM and placed in 37°C shaker
</LI><LI> IPTG-Upps 5-Low Copy (CAM)
</LI><LI> IPTG-Upps 6-Low Copy (CAM)
</LI><LI> IPTG-Kill gene-Low Copy (CAM)
</LI><LI> Purified above plasmids and CAM control plasmid using standard purification protocol and prepared glycerol stocks
</LI><LI> Added appropriate buffers to IPTG-Upps 5 and IPTG-Upps 6 and centrifuged for 10 minutes to determine for a pellet
</LI><LI> Measured OD 600 of following samples
</LI><LI> IPTG-Upps 5-Low Copy (CAM): .160A
</LI><LI> IPTG-Upps 6-Low Copy (CAM): .038A
</LI><LI> Inserted 1µl of 1M IPTG into cultures
</LI><LI> Placed all samples in 37°C overnight
</LI></UL><P><I>Note: conducted the below procedures at the Kanbar Lab</I></P><UL><LI> Observed no growth for inducible promoter/GFP/low copy CAM plasmid
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_16th"> Thursday, 16th </SPAN></H4><UL><LI> Measured OD 600 for 200x diluted bacterial solution containing plasmids with promoter inducible with IPTG
</LI><LI> IPTG-Upps 5-Low Copy (CAM): .040A
</LI><LI> IPTG-Upps 6-Low Copy (CAM): .032A
</LI><LI> IPTG-Kill gene-Low Copy (CAM): .028A
</LI><LI> Noted that cell concentration was dense, decided to dilute solution with 75µl cells and 925µl LB/CAM solution
</LI><LI> Placed diluted cell solution into 37°C incubator for 40 minutes
</LI><LI> Inserted 1µl of 1M IPTG into cultures
</LI><LI> Placed all samples in 37°C overnight
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_17th"> Friday, 17th </SPAN></H4><P><I>Note: IPTG induced I-U5 and I-U6 appear to give no color change (no expression)</I></P><UL><LI> Electroporate the following genes:
</LI><LI> Upps 4-Upps 6-High copy TET (sparked first time, redid trial)
</LI><LI> Upps 4-Kill-High copy TET
</LI><LI> Upps 4-Kill-Low copy CAM
</LI><LI> pUC19 control DNA (AMP resistance)
</LI><LI> Placed all samples in 37°C overnight
</LI></UL><P><I>Note2: Only adding 25µl of competent cells instead of 50µl mentioned in the</I></P><H4><SPAN class="mw-headline" id="Saturday.2C_18th"> Saturday, 18th </SPAN></H4><UL><LI> Observed none of the electroporated DNA grew, but observed colonies for control
</LI><LI> Placed all samples back in the 37°C incubator
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_20th"> Monday, 20th </SPAN></H4><UL><LI> Measured the optical density for the following:
</LI><LI> Upps 6 (with inducible promoter)
</LI><LI> Kill gene
</LI><LI> GFP
</LI><LI> Upps 5 (with inducible promoter)
</LI><LI> Diluted the concentrated primer into a primer stock that would be used for DNA sequencing
</LI><LI> Sent following DNA for sequencing (at Genewiz)
</LI><LI> Inducible promoter-GFP-Low Copy CAM plasmid
</LI><LI> Inducible promoter-Upps 5-Low Copy CAM plasmid
</LI><LI> Inducible promoter-Upps 6-Low Copy CAM plasmid
</LI><LI> Inducible promoter-Kill gene-Low Copy CAM plasmid
</LI><LI> Analyzed gel and concluded following:
</LI><LI> The digestion for TET high copy plasmid did not work
</LI><LI> The digestion for Upps 4, 5, 6 appeared to have worked
</LI><LI> All other digestions are inconclusive results
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_21st"> Tuesday, 21st </SPAN></H4><UL><LI> Made additional Amp/CAM and Amp/TET plates
</LI><LI> Measured the optical density for the following:
</LI><LI> Upps 1
</LI><LI> Kill gene
</LI><LI> High copy TET plasmid
</LI><LI> Followed the standard digestion protocol provided by bioline or the following:
</LI><LI> Upps 1 (Upstream)
</LI><LI> Kill gene (Downstream)
</LI><LI> High copy TET plasmid (Destination plasmid)
</LI><LI> Re-organized iGEM boxes by throwing out previous digestions of High Copy TET plasmids
</LI><LI> Re-hydrated following biobricks from standard rehydration protocol:
</LI><LI> Constitutive promoter-BBa_J130002 (P1, 13B)
</LI><LI> Low-copy TET plasmid-PSB3T5 (P1, 7C)
</LI><LI> High-copy CAM plasmid-PSB1C3 (P1, 3A)
</LI><LI> High-copy Amp/CAM plasmid-PSB1AC3 (P1, 9A)
</LI><LI> High-copy Amp/TET plasmid-PSB1AT3 (P1, 13A)
</LI><LI> Made 60ml of 1% agar gel for running gel electrophoresis (2 rows of 12 wells each noted below for gel) to check digestion and ligation
</LI><LI> First row
</LI><LI> 2 log ladder
</LI><LI> old TET high copy plasmid
</LI><LI> old Kill gene
</LI><LI> Upps 5
</LI><LI> IPTG inducible promoter
</LI><LI> Second row
</LI><LI> 2 log ladder
</LI><LI> newly digested TET high copy plasmid
</LI><LI> newly digested Kill gene
</LI><LI> Upps 1
</LI></UL><LI> Ran gel for 25 minutes at constant 150V
</LI><LI> Took picture under UV light
</LI><H4><SPAN class="mw-headline" id="Wednesday.2C_22nd"> Wednesday, 22nd </SPAN></H4><UL><LI> Conducted ligations for the following using a vector to insert ratio of 1:3
</LI><LI> Sample 1: Inducible promoter (upstream insert)-Kill gene (Downstream insert)-pSB3C5 (vector)
</LI><LI> Sample 2: Inducible promoter (upstream insert)-Upps 5 (Downstream insert)-pSB3C5 (vector)
</LI><LI> Sample 3: Inducible promoter (upstream insert)-Upps 6 (Downstream insert)-pSB3C5 (vector)
</LI><LI> Sample 4: Upps 4 (upstream insert)-Kill gene (Downstream insert)-pSB1T3 (vector)
</LI><LI> Sample 5: Upps 4 (upstream insert)-Upps 5 (Downstream insert)-pSB1T3 (vector)
</LI><LI> Sample 6: Upps 4 (upstream insert)-Upps 6 (Downstream insert)-pSB1T3 (vector)
</LI><LI> Sample 7: Control pSB3C5 (1:0 vector to insert ratio)
</LI><LI> Sample 8: Control pSB1T3 (1:0 vector to insert ratio)
</LI></UL><P><I>Note: above ligations will be done with T4 buffer and ligase from both the Biobrick kit and Columbia's lab</I></P><P><I>Note 2: label all sample using iGEM ligase and buffer with prefix N</I></P><P><I>Note 3: label all sample using Columbia lab's ligase and buffer with B</I></P><UL><LI> Electroporated the following using the standard protocol provided by Bioline:
</LI><LI> Constitutive promoter-BBa_J130002
</LI><LI> Low-copy TET plasmid-PSB3T5 
</LI><LI> High-copy CAM plasmid-PSB1C3
</LI><LI> High-copy Amp/CAM plasmid-PSB1AC3 
</LI><LI> High-copy Amp/TET plasmid-PSB1AT3 
</LI><LI> PUC19 control plasmid
</LI><LI> Chemically transformed the following using the heat shock protocol:
</LI><LI> N1, N2, N3, N7, B1, B2, B3, B7 into BL21 (DE) pLysZ orange cells
</LI><LI> GFP IPTG given by graduate student at Columbia into BL21 (DE) green cells
</LI><LI> PUC19 into BL21 (DE) pLysZ orange cells
</LI><LI> PUC19 into BL21 (DE) green cells
</LI><LI> Purified ligated protocol for electroporation following QIAquick PCR purification protocol
</LI><LI> Plated all chemically transformed and electroporated cells
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_23rd"> Thursday, 23rd </SPAN></H4><P><I>Note: Observed no growth with pSB1T3, BL21 pLyse</I></P><P><I>Note 2: Observed growth with PUC19 and other BL21</I></P><UL><LI> Streaked sample N5
</LI><LI> Found following 3 restriction sites (to be double checked by graduate student):
</LI><LI> NgomIV: CCGCCGGC
</LI><LI> EcoRI: GAATTC
</LI><LI> PstI: CTGCAG
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_24th"> Friday, 24th </SPAN></H4><UL><LI> Diluted IPTG driven cultures to 200x with fresh LB/antibiotic solution
</LI><LI> Followed standard plasmid isolation protocol for everything except the following:
</LI><LI> 1 sample of graduate student's GFP
</LI><LI> Sample of pSB1C3 (the high copy CAM plasmid)
</LI><LI> Checked optical densities of samples B1-B3 and graduate student's GFP
</LI><LI> Created two subsamples each of B1-B3; applied 1M IPTG to half of the subsamples
</LI><LI> Applied 1M IPTG to into one of the samples of graduate student's IPTG-GFP
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_25th"> Saturday, 25th </SPAN></H4><UL><LI> Checked IPTG &quot;induced&quot; samples and observed no expression
</LI><LI> Measured optical density for following samples and diluted the samples:
</LI><LI> GFP by graduate student-pre-dilution: 2.0, post dilution: 0.710
</LI><LI> IPTG-kill (N1)-pre-dilution: 2.4, post dilution: 0.736
</LI><LI> IPTG-U5 (N2)-pre-dilution: 1.9, post dilution: 0.740
</LI><LI> IPTG-U6 (N5)-pre-dilution: 2.4, post dilution: 0.630
</LI><LI> IPTG-U5 (B2)-pre-dilution: 1.9, post dilution: 0.681
</LI><LI> IPTG-U6 (B3)-pre-dilution: 1.9, post dilution: 0.651
</LI><LI> Applied 1.5μL of 1M IPTG with 1.5mL of each sample and place in 25C shaker
</LI><LI> Followed standard plasmid isolation protocol for N1 and N5
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_27th"> Monday, 27th </SPAN></H4><UL><LI> Observe expression of graduate student's GFP
</LI><LI> Measured optical densities for following:
</LI><LI> Ligated samples:
</LI><LI> N1-N3 
</LI><LI> B1-B3
</LI><LI> Mini-prepped:
</LI><LI> Upps1-Upps6
</LI><LI> pSB1C3
</LI><LI> pSB1T3
</LI><LI> pSB1AT3
</LI><LI> pSB1AC3
</LI><LI> pSB1A2
</LI></UL><LI> Sent above samples for sequencing
</LI><LI> Followed protocol from bioline for digestion:
</LI><LI> Downstream: Upps 1 
</LI><LI> Upstream: Constitutive promoter 
</LI><LI> Destination plasmid: pSB3T5
</LI><LI> Destination plasmid: pSB1C3
</LI><LI> Destination plasmid: pSB1AC3
</LI><LI> Destination plasmid: pSB1AT3
</LI><LI> Streaked out Upps 1 from agar stab
</LI><H4><SPAN class="mw-headline" id="Tuesday.2C_28th"> Tuesday, 28th </SPAN></H4><UL><LI> Reviewed results from the sequencing from Genewiz
</LI><LI> Design and send new primer (nev) for sequencing
</LI><LI> Conducted ligations for the following using a vector to insert ratio of 1:3
</LI><LI> Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB1C3 (vector)
</LI><LI> Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB3T5 (vector)
</LI><LI> Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB1AT3 (vector)
</LI><LI> Constitutive promoter (upstream insert)-Upps 1 (downstream insert)-pSB1AC3 (vector)
</LI><LI> Electroporated the above 4 ligations without purifying the ligations
</LI><LI> Plated cells into respective plates
</LI><LI> Picked colonies for Upps 1-YF1/FixJ
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_29th"> Wednesday, 29th </SPAN></H4><UL><LI> Conducted ligations for the following using a vector to insert ratio of 1:3
</LI><LI> Upps 4 (upstream insert)-Upps 5 (downstream insert)-pSB1C3 (vector)
</LI><LI> Upps 4 (upstream insert)-Upps 6 (downstream insert)-pSB1C3 (vector)
</LI><LI> Upps 4 (upstream insert)-Kill gene (downstream insert)-pSB1C3 (vector)
</LI><LI> Upps 4 (upstream insert)-Upps 5 (downstream insert)-pSB1AC3 (vector)
</LI><LI> Upps 4 (upstream insert)-Upps 6 (downstream insert)-pSB1AC3 (vector)
</LI><LI> Upps 4 (upstream insert)-Kill gene (downstream insert)-pSB1AC3 (vector)
</LI><LI> Extracted Upps 1 (Kan) from glycerol stock from -80C freezer
</LI><LI> Followed graduate student's quickchange protocol and labelled N1-N4, E1-E4, P1-P4 with the labeling as following:
</LI><LI> Sample 1: HF only
</LI><LI> Sample 2: HF and 1.5μL DMSO
</LI><LI> Sample 3: GC only
</LI><LI> Sample 4: GC and 1.5μL DMSO
</LI><LI> Prefix N: NgoMIV mutation
</LI><LI> Prefix E: EcoRI mutation
</LI><LI> Prefix P: PsH mutation
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_30th"> Thursday, 30th </SPAN></H4><UL><LI> Followed standard plasmid isolation protocol for Upps 1
</LI><LI> Transformed U4-U5, U4-U6, U4-kill in 2 different plasmids
</LI><LI> Ran gels of Quickchange product
</LI><LI> Column 1: miniprepped plasmid
</LI><LI> Column 2-5: N1-N4
</LI><LI> Column 6-9: E1-E4
</LI><LI> Column 10-13: P1-P4
</LI><LI> Re-ligated Constitutive promoter-Upps 1
</LI></UL></DIV><H2><SPAN class="mw-headline" id="September.2C_2012"> September, 2012 </SPAN></H2><DIV class="content-body-box"><H4><SPAN class="mw-headline" id="Wednesday.2C_5th"> Wednesday, 5th </SPAN></H4><P><I>Note: All work done from this day on was done at the Kanbar lab at the Cooper Union</I></P><UL><LI> Single digested the J13002
</LI><LI> Placed the digest above and other digests to be run on gels into -20C freezer
</LI><LI> Made 4 50mL 1% agarose gels
</LI><LI> Followed standard plasmid isolation protocol for kill gene
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_7th"> Friday, 7th </SPAN></H4><UL><LI> Ran two gels to check digests
</LI><LI> Gel Number 1
</LI><LI> Marker
</LI><LI> Upps 1
</LI><LI> Inducible promoter 1
</LI><LI> Inducible promoter 2
</LI><LI> pSB3T5
</LI><LI> pSB1C3
</LI><LI> Low Copy Vector 1
</LI><LI> Low Copy Vector 2
</LI><LI> Gel Number 2
</LI><LI> Marker
</LI><LI> Kill gene
</LI><LI> PcyA
</LI><LI> TetR 1
</LI><LI> TetR 2
</LI><LI> GFP from biobrick kit
</LI><LI> GFP from Tushar
</LI><LI> GFP from Kevin
</LI></UL><LI> Conducted digestions for the following:
</LI><LI> Inducible promoter digested with EcoRI&amp;Spe with Buffer E and Buffer Multi (for 30 min.)
</LI><LI> TetR digested with EcoRI with Buffer H
</LI><LI> TetR digested with Spe with Buffer B
</LI><LI> Upps 1 digested with Xba&amp;Pst with Buffer H
</LI><LI> Kill gene digested with Xba&amp;Pst with Buffer H
</LI><LI> GFP digested with Xba&amp;Pst with Buffer H
</LI><LI> pSB3T5 with EcoRI&amp;Pst with Buffer H
</LI><LI> Low copy plasmid 2 digested with EcoRI&amp;Pst with Buffer H
</LI><LI> pSB1C3 digested with EcoR&amp;Pst with Buffer H
</LI><P><I>Note: Digests 7-9 was done in Antartic Phosphotase Treat 30 min in 37C and 15 min in 65C</I></P><UL><LI> Conducted ligations for the following (followed by chemical transformations:
</LI><LI> Upstream: Inducible promoter; Downstream: Upps 1; Destination plasmid: Low copy plasmid 2
</LI><LI> Upstream: Inducible promoter; Downstream: Kill; Destination plasmid: Low copy plasmid 2
</LI><LI> Upstream: Inducible promoter; Downstream: GFP; Destination plasmid: Low copy plasmid 2
</LI><LI> Conducted additional chemical transformations with the following vectors:
</LI><LI> pET20b
</LI><LI> pET26b
</LI><LI> pUC19
</LI><LI> Conducted following additional ligations, reference numbers correlate to numbers shown above for this day's digestions
</LI><LI> Label A: 2 and 9
</LI><LI> Label B: 3 and 4 and 9
</LI><LI> Label C: 1 and 4 and 8
</LI><LI> Label D: 1 and 5 and 8
</LI><LI> Label E: 1 and 6 and 8
</LI><LI> Label F: 3 and 4
</LI><LI> Label G: 3 and 5
</LI><LI> Label H: 3 and 6
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_11th"> Tuesday, 11th </SPAN></H4><UL><LI> Observed the following:
</LI><LI> Inducible promoter-GFP-Low Copy CAM Plasmid had 3 pink clones (IPTG applied)
</LI><LI> Inducible promoter-GFP-Low Copy CAM Plasmid had 3 pink clones (NO IPTG)
</LI><LI> Inducible promoter-Upps 1-Low Copy CAM Plasmid had 32 pink clones
</LI><LI> Inducible promoter-Kill gene-Low Copy CAM Plasmid had 50 pink clones
</LI></UL><P><I>Note 1: Destination plasmid/vector contained J04450 biobrick part, explained pink clones</I></P><P><I>Note 2: All vectors were all treated with Antartic Phophotase, expect no religation of J04450 with LC vector</I></P><UL><LI> Digested Kill gene and GFP with E and P
</LI><LI> AP Treat Kill gene with X and P, GFP with X and P
</LI><LI> Conducted ligations for the following:
</LI><LI> GFP (E and P) with pSB1C3 (E and P, AP treated)
</LI><LI> GFP (X and D, AP treated) with TetR/s
</LI><LI> Condcuted transformations for the following:
</LI><LI> Ligation F from 9/7, TetR, Upps 1(X and P)
</LI><LI> Kill gene, pSB1C3
</LI><LI> GFP, pSB1C3
</LI><LI> GFP, TetR
</LI><LI> Mini-cultured pET20b and pET26b
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_12th"> Wednesday, 12th </SPAN></H4><UL><LI> Observed following from the transformations from 9/11
</LI><LI> TetR/GFP-successful with about 250 colonies
</LI><LI> TetR/Upps 1-successful with about 200 colonies
</LI><LI> No growth with Kill&amp;pCB1C3, GFP&amp;pCB1C3 
</LI><LI> Prepared colony PCR for TetR&amp;GFP, and TetR&amp;Upps 1 on Amp grid plate (had A-D columns and 1-4 rows)
</LI><LI> Resuspended single colonies in 4μL deionized water
</LI><LI> Transferred 3μL colony suspension to Amp grid
</LI><LI> Stored remaining 1μL as PCR template
</LI><LI> Grew Amp grid plate in 37C overnight
</LI></UL><P><I>Note 1: GFP/TetR: A and B with 1-4 each</I></P><P><I>Note 2: Upps 1/TetR: C and D with 1-4 each</I></P><UL><LI> PCR reaction contained following mixture:
</LI><LI> 1x PCR Rxn
</LI></UL><OL><LI> 1.0μL of 10x buffer
</LI><LI> 0.4μL of dNTP
</LI><LI> 0.4μL of Primer F
</LI><LI> 0.4μL of Primer R
</LI><LI> 1.0μL of template mentioned above
</LI><LI> 0.4μL of tag
</LI><LI> 6.4μL of deionized water
</LI></OL><UL><LI> 17x PCR Rxn
</LI></UL><OL><LI> 17μL of 10x buffer
</LI><LI> 6.8μL of dNTP
</LI><LI> 6.8μL of Primer F
</LI><LI> 6.8μL of Primer R
</LI><LI> 6.8μL of tag
</LI><LI> 108.8μL of deionized water
</LI></OL><LI> PCR reactions conducted with following protocol:
</LI><LI> Denature at 95C for 10 min.
</LI><LI> Denature at 95C for additional 30 sec.
</LI><LI> Anneal at 56C for 30 sec.
</LI><LI> Elongation at 70C for 60 sec.
</LI><LI> Elongation at 72C for 20 min.
</LI><LI> Hold at 4C for 39 cycles
</LI><LI> Set up second half composite construction digests for following:
</LI><LI> Upps 1; Enzyme: EcoRI and SpeI; Buffer E*; Ratio: 1:2; Reaction vol.: 30μL
</LI><LI> Upps 4; Enzyme: EcoRI and SpeI; Buffer E*; Ratio: 1:2; Reaction vol.: 30μL
</LI><LI> Upps 5; Enzyme: Xba and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
</LI><LI> Upps 6; Enzyme: Xba and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
</LI><LI> Kill gene; Enzyme: Xba and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
</LI><LI> Linearlized pSB1T3; Enzyme: EcoRI and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
</LI><LI> Linearlized pSB1K3; Enzyme: EcoRI and Pst; Buffer H; Ratio: 1:1; Reaction vol.: 20μL
</LI><LI> Promoterless GFP; Enzyme: Xba; Buffer H; Ratio: N/A; Reaction vol.: 20μL
</LI><LI> pUC19; Enzyme: EcoRI; Buffer H; Ratio: N/A; Reaction vol.: 20μL
</LI><P><I>Note 3: for * buffers, will use buffer 2 or 4 in future</I></P><UL><LI> Reactions took place for 1 hr. in 37C incubation and placed in -20C
</LI><LI> Isolated pGLO and pET20b vectors using standard plasmid isolation protocol
</LI><LI> Prepared glycerol stocks for pET20b and pET26b
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_13th"> Thursday, 13th </SPAN></H4><UL><LI> Conducted fphA PCR mutogenesis with following (template = pASKfphA 753bp)
</LI><LI> 1x PCR Rxn
</LI></UL><OL><LI> 2.0μL of 10x Buffer
</LI><LI> 0.4μL of dNTP
</LI><LI> 0.4μL of Primer F (EcoRI)
</LI><LI> 0.4μL of Primer R (EcoRI)
</LI><LI> 1.0μL of template
</LI><LI> 0.4μL of tag
</LI><LI> 15.4μL of deionized water
</LI></OL><UL><LI> 3x PCR Rxn
</LI></UL><OL><LI> 6.0μL of 10x Buffer
</LI><LI> 1.2μL of dNTP
</LI><LI> 1.2μL of Primer F (EcoRI)
</LI><LI> 1.2μL of Primer R (EcoRI)
</LI><LI> 1.2μL of tag
</LI><LI> 46.2μL of deionized water
</LI></OL><LI> Set up 4 min. extension time with 55C for annealing temp for PCR program
</LI><LI> Digested above mixture with Dpn1 at 37C for 1hr. after PCR
</LI><LI> Transformed product into DH5α
</LI><LI> Heat inactivated samples with Spe1 restriction digests from 9/12 at 80C for 20 min. and rest at 65C for 20 min.
</LI><LI> Antartic treated the following (15 min. 37C, 5 min. 65C):
</LI><LI> Upps 4
</LI><LI> Promoterless GFP
</LI><LI> pUC19
</LI><LI> Conducted ligations for the following for a total volume of 10μL
</LI><LI> Upps 4-Kill gene-pSB1K3
</LI><LI> Upps 4-Upps 5-pSB1K3
</LI><LI> Upps 4-Upps 6-pSB1K3
</LI><LI> TetR/Spe (from 9/7)-promoterless GFP
</LI><LI> pUC19
</LI><LI> Transformed ligated DNA into DH5α
</LI><LI> Ran gels for PCR Rxn and observed following:
</LI><LI> A3, B3 had 300 bp (estimated) product
</LI><LI> B1, B4 had 700 bp (estimated) product
</LI><LI> Everything else (including marker failed)
</LI><LI> Can not differentiate between ±TetR
</LI><LI> Minicultured 4 TetR-Upps 1 cultures randomly 
</LI><H4><SPAN class="mw-headline" id="Friday.2C_14th"> Friday, 14th </SPAN></H4><UL><LI> Sterilized glycerol for stocks
</LI><LI> Created more LB/Amp plates
</LI><LI> Created more sterile miniculture tubes
</LI><LI> Ran 100 bp optimization gel
</LI></UL><P><I>Note 1: While 2µl DNA visble, should use minimum of 5µl/lane for system (SYBR)</I></P><UL><LI> Repeated VF2-VR PCR on TetR&amp;promoterless GFP (denoted A) and TetR&amp;Upps 1 (denoted B)
</LI><LI> 1x PCR Rxn
</LI><LI> 1.0μL of 10x Buffer
</LI><LI> 0.4μL of dNTP
</LI><LI> 0.4μL of Primer F (EcoRI)
</LI><LI> 0.4μL of Primer R (EcoRI)
</LI><LI> 1.0μL of template
</LI><LI> 0.4μL of tag
</LI><LI> 6.4μL of deionized water
</LI><LI> 17x PCR Rxn
</LI><LI> 17.0μL of 10x Buffer
</LI><LI> 6.8μL of dNTP
</LI><LI> 6.8μL of Primer F (EcoRI)
</LI><LI> 6.8μL of Primer R (EcoRI)
</LI><LI> 6.8μL of tag
</LI><LI> 108.8μL of deionized water
</LI></UL><P><I>Note 2: Will pick 7 colonies from each and have 4 minicultures of B-will use 1µl±1 colony</I></P><UL><LI> After review of primers, changed PCR program as follows:
</LI><LI> Increased annealing temp to 60C
</LI><LI> Increased extension temp to 70C
</LI><LI> Reduced cycles to 30
</LI></UL><H4><SPAN class="mw-headline" id="Thursday.2C_20th"> Thursday, 20th </SPAN></H4><UL><LI> Reviewed order of custom primer from Invitrogen (primer ordered: 9/17)
</LI><LI> Rehydrated primers
</LI><LI> Ran PCR Rxn with following:
</LI><LI> Annealing temperature: 61C
</LI><LI> Elongation time: 3 minutes
</LI><LI> 30 cycles
</LI></UL><H4><SPAN class="mw-headline" id="Monday.2C_24th"> Monday, 24th </SPAN></H4><UL><LI> Reviewed plates and observed the following:
</LI><LI> Noted plenty of colonies on both plates, as expected
</LI><LI> Plates with pSB1C3-J04450 based vector had many pink colonies and 8-10 white clones
</LI><LI> Considering both vectors were alkaline phosphatase treated, will assume treatment failed for both vectors
</LI><LI> Will currently disregard Upps 3 based vector transformations
</LI></UL><LI> Picked following colonies for miniculture:
</LI><LI> pSB1C3-FphA
</LI><LI> Upps 4-Kill gene
</LI><LI> Upps 4-Upps 6
</LI><LI> TetR-Upps 1
</LI><LI> Minicultured the following from glycerol stocks
</LI><LI> Upps 1
</LI><LI> Upps 4
</LI><LI> TetR
</LI><H4><SPAN class="mw-headline" id="Tuesday.2C_25th"> Tuesday, 25th </SPAN></H4><UL><LI> Ran a gel to verify following plasmids:
</LI><LI> Sizing Gel 1
</LI><LI> Kill gene 1 (OK)
</LI><LI> Kill gene 2 (OK)
</LI><LI> FphA1 (too small - discard)
</LI><LI> FphA2 (OK)
</LI><LI> Upps 1 (too big - discard)
</LI><LI> B0034 RBS (OK, faint)
</LI><LI> Sizing Gel 2
</LI><LI> Upps 4-Kill gene 1 (OK)
</LI><LI> Upps 4-Kill gene 2 (OK)
</LI><LI> Upps 4-Upps 6 (OK)
</LI><LI> Upps 4-Upps 6 (too big - discard)
</LI><LI> Upps 4 (OK)
</LI><LI> B0030 RBS.1 (OK, but really faint)
</LI></UL><P><I>Note 1: the parenthesis denote observations/analysis of gel</I></P><UL><LI> Isolated plasmids using standard miniprep protocol for the minicultures from previous day
</LI><LI> Set up PCR site directed mutagenesis using pSB1C3-FphA as template 
</LI></UL><P><I>Note 2: used protocol from 9/13 and EcoRI primer set</I></P><UL><LI> Rehydrated the following using standard iGEM rehydration protocol
</LI><LI> BBa_J23100 (plate 1, 18C [Amp])
</LI><LI> BBa_J23101 (plate 1, 18E [Amp])
</LI><LI> Bba_J23102 (plate 1, 18G [Amp])
</LI><LI> Transformed the above rehydrated biobricks into DH5α
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_26th"> Wednesday, 26th </SPAN></H4><UL><LI> Digested the PCR Rxn with DpnI
</LI><LI> Transformed pSB1C3-FphA into DH5α
</LI><LI> Digested the following: 
</LI><LI> BBa_B0034 RBS with SpeI
</LI><LI> Upps4 with XbaI
</LI><LI> Kill gene with XbaI
</LI><LI> Ligated the following (was predigested with EcoRI): 
</LI><LI> RBS-Upps 4
</LI><LI> RBS-Kill gene
</LI><LI> Gel extracted the bands of the following:
</LI><LI> RBS-Upp4 Target 2352bp, erroneous 2155bp
</LI><LI> RBS-Kill Target 3441bp, erroneous 2155bp
</LI><LI> Minicultured the following:
</LI><LI> Upps 1
</LI><LI> Promoterless GFP
</LI><LI> Promoters 100-102
</LI></UL><H4><SPAN class="mw-headline" id="Wednesday.2C_27th"> Wednesday, 27th </SPAN></H4><UL><LI> Made minipreps from minicultures from previous day, ran qualitative gel, and concluded the following:
</LI><LI> Upps 1 had poor yield
</LI><LI> Promoterless GFP showed no growth
</LI><LI> Conducted digestions of the following: (yielded 2x volume)
</LI><LI> RBS with SpeI
</LI><LI> Promoter 102 with SpeI 
</LI><LI> Upps 1 with XbaI
</LI><LI> Promoterless GFP with XbaI
</LI><LI> Conducted ligations of the following using digestion with 1x volume of EcoRI: (yielded 2x volume)
</LI><LI> Promoter 102-Upps 1
</LI><LI> Promoter 102-Promoterless GFP
</LI><LI> RBS-Upps 4
</LI><LI> Ribosome Binding Site-Kill gene
</LI><LI> Ran 1x volume on gel to purify and gel extract target band
</LI><LI> Transformed the above in DH5α using standard heat shock protocol
</LI><LI> Observed the PCR site mutagenesis yielded no colonies
</LI></UL><H4><SPAN class="mw-headline" id="Friday.2C_28th"> Friday, 28th </SPAN></H4><UL><LI> Observed that Ribosome Binding Site-Kill gene worked and everything else did not 
</LI><LI> Conducted minicultures for the following: 
</LI><LI> Ribosome Bidning Site-Kill gene
</LI><LI> Promoterless GFP with Kan resistance
</LI><LI> Upps 1
</LI><LI> Upps 4
</LI><LI> Conducted digestions of the following: 
</LI><LI> Upps 1 with XbaI
</LI><LI> Inducible Promoter with SpeI
</LI><LI> Upps 1 with EcoRI/SpeI
</LI><LI> Upps 4 with XbaI/PstI
</LI><LI> pSB1A3 (linearized DNA) with EcoRI/PstI
</LI><LI> pSB3T5 with EcoRI/PstI
</LI><LI> Conducted ligations of the following:
</LI><LI> Promoter 100-Upps 1
</LI><LI> Promoter 101-Upps 1
</LI><LI> Promoter 102-Upps 1
</LI><LI> Inducible Promoter-Upps 1
</LI><LI> Promoter 102-promoterless GFP
</LI><LI> Upps 1-Upps 4-pSB1A3
</LI><LI> Upp1-Upp4-pSB3T5
</LI><LI> Transformed the above ligated products into DH5α using heat shock protocol
</LI></UL><H4><SPAN class="mw-headline" id="Saturday.2C_29th"> Saturday, 29th </SPAN></H4><UL><LI> Observed that Promoter 102-GFP worked and everythign else did not
</LI><LI> Conducted digestions of the following:
</LI><LI> Upps 1 with EcoRI&amp;SpeI
</LI><LI> Upps 4-Kill with XbaI&amp;PstI
</LI><LI> Upps 4-Upps 6 with XbaI&amp;PstI
</LI><LI> pSB1C3-J04450 with EcoRI&amp;PstI
</LI><LI> Conducted ligations of the following:
</LI><LI> Upps 1-Upps 4-Kill gene-pSB1C3 destination plasmid
</LI><LI> Upps 1-Upps 4-Upps6-pSB1C3 destination plasmid
</LI><LI> Transformed the above ligated products into DH5α using a heat shock protocol
</LI><LI> Transformed the following into JM109:
</LI><LI> Promoter 100-Upps 1
</LI><LI> Promoter 101-Upps 1
</LI><LI> Promoter 102-Upps 1
</LI><LI> Inducible Promoter-Upps 1
</LI></UL></DIV><H2><SPAN class="mw-headline" id="October.2C_2012"> October, 2012 </SPAN></H2><DIV class="content-body-box"><H4><SPAN class="mw-headline" id="Monday.2C_1st"> Monday, 1st </SPAN></H4><UL><LI> Observed that all of the plates grew colonies
</LI><LI> Prepared minicultures of the following:
</LI><LI> Upps 1-Upps 4-Kill gene-pSB1C3 destination plasmid 
</LI><LI> Upps 1-Upps 4-Upps 6-pSB1C3 destination plasmid
</LI><LI> Promoter 102-Upps 1 
</LI><LI> Inducible promoter-Upps 1
</LI></UL><H4><SPAN class="mw-headline" id="Tuesday.2C_2nd"> Tuesday, 2nd </SPAN></H4><UL><LI> Isolated following plasmids by following standard plasmid isolation protocol and submitted parts:
</LI><LI> pSB1C3-FphA
</LI><LI> Upps 4-Kill gene
</LI><LI> Upps 1-Upps 4-Kill gene
</LI><LI> Upps 1-Upps 4-Upps 6
</LI><LI> Promoter 102-Upps 1
</LI><LI> Inducible promoter-Upps 1
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Columbia-Cooper-NYC/Columbia_notebook_2">http://2012.igem.org/Team:Columbia-Cooper-NYC/Columbia_notebook_2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Columbia-Cooper-NYC/Columbia_notebook_2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Columbia-Cooper-NYC/Columbia_notebook_2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Columbia-Cooper-NYC/Columbia_notebook_2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Columbia-Cooper-NYC/Columbia_notebook_2&amp;oldid=299471" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>