"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Project/Primer Design
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Primer Design
With our minimal essential gene list, it was now required to take these genes and insert them into the pAB and pBA plasmids which allow for BioByte Assembly (Please see the Standard Plasmids section under the DNA Assembly tab for more information on pAB/pBA and the BioBytes method).  In order to do this, PCR was used to amplify the individual genes from MG1655 genomic DNA along with extensions to produce restriction sites.
Prefix and Suffix to be Added For PCR Amplification
These restriction sites were designed to be complementary to the restrictions sites found in pAB and pBA (PstI and XbaI).  Due to the use of restriction enzymes, it was important to check each gene sequence for those same restriction sites.  Genes which contained at least one PstI site, used the NsiI enzyme to be cloned into our standard plasmids (it produces the same overhang as PstI).  Similarly if NsiI and PstI were in the gene sequence, SbfI was used. If all three of these sites were present in a gene sequence, other enzymes which produce the same overhang could be used.  Genes which contained XbaI could use AvrII, NheI, as well as a multitude of others.  In the attached file, 314 primers have been produced with their sequence information.  188 of these primers have been tested and amplified and their sequences uploaded onto the parts registry.  These primers are additionally characterized by the restriction sites which they contain (the sites found in the primers were ensured to not appear in the gene sequence).  In the wetlab we have used PCR to amplify 185 of these genes already. All primers that produced PCR products of the predicted size have been submited as biobrick parts.
To view our primer list please click here .
To view details of our primer testing, click here .
Vector NTI
Vector NTI was used to produce all of the primers in the above list.  Each primer's thermodynamic properties were examined using Vector NTI's hairpin and dimerzation programs.  Primers were optimized to fit as many of the following criteria as possible:
Find the shortest possible sequence, reducing the cost to produce the primer.
Produce the highest score value possible.
Produce the closest Tm's possible
Produce hairpins with dG values >-5
Produce dimers with dG values >-10
If these values could not be attained, the quality values within the PCR Setup window were altered to try and produce a better primer.  Ultimately, if the optimized primers will still not within the above parameters, they were added to the list and noted.
The file which loads our Primer Settings into Vector NTI can be found here.
Retrieved from " http://2009.igem.org/Team:Alberta/Project/Primer_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
