"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/DNAanchor
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
DNA Anchor/Terminator
Anchoring System
A vital component of the BioBytes method is the use of a biotinylated DNA Anchor in order to allow unidirectional assembly of the Bytes on paramagnetic beads by sequestering the 5' ends of Bytes, leaving only the 3' ends available to bind incomding Bytes.  The Anchor itself has three vital components:  A 5’ biotinylation, a double stranded DNA (dsDNA) portion that incorporates a release mechanism in order to liberate the construct from the beads, and A or B overhangs to allow Bytes to bind to the Anchor.  Our team has considered a number anchor systems, each with their own set of advantages and disadvantages.  The three main anchor systems we investigated are summarized below. See Table 1.
Table 1: Overview of three different anchoring systems that were considered for BioBytes. See the section on Anchor Variants and Binding Capacity below for further information on these systems. 5' Biotin (BTN), single stranded DNA (ssDNA), nucleotide (nt), double stranded DNA (dsDNA).
The current BioBytes anchor system utilizes a 5’-biotin which anchors the construct to beads by binding non-covalently, but with great strength, to the covalently linked streptavidin on the surface of the paramagnetic beads.  There is also a 5’-15 nucleotide spacer region of ssDNA which facilitates more efficient binding of the Anchor to the bead as the binding pocket of streptavidin is deep and thus a highly flexible ssDNA linker is recommended to allow the biotin to effectively bind into this deep pocket.  A 21 bp double stranded portion of the Anchor contains the I-SceI recognition sequence, which when digested with I-SceI produces 4 base overhangs, but also includes four deoxyuracil residues.  These uracils are excised by New England Biolab’s USERTM system to generate single nucleotide gaps in the top strand.  USERTM digestion thus effectively destroys the Anchor and produces a 21 base 3’ overhang which becomes important for recircularization of the construct.  Finally the Anchor contains the A or B 3’ overhangs complementary to those of the Bytes, allowing their binding to the Anchor. See Figure 1.
Figure 1: pAB and pBA multiple cloning sites with highlighted primers pAB_F/R and pBA_F/R annealing regions.
Termination System
Once the construct has been completed, i.e. the last Byte has been added, the construct may be released from the beads as is by a simple I-SceI digestion or a USERTM digestion, thus yielding a linear construct.  If a circular construct, such as a plasmid, is desired then a final "Terminator" piece must be added.  This piece is similar in construction to the Anchor, whereby there is a dsDNA I-SceI recognition sequence with four deoxyuracils incorporated into it on one strand, as well as an A or B 3' overhang.  The Terminator binds to the last Byte and release is once again achieved by I-SceI digestion or USERTM digestion.  In either case both the Anchor and Terminator develop sticky ends that are complementary to each other: 4 bases if I-SceI digestion is utilized, or 21 bases if USER is used.  USERTM digestion is obviously preferred since 21 bp of interaction will form spontaneously and without ligation, and thus transformation of the construct can proceed immediately. See Figure 1.
Anchor and Terminator Oligo Sequences
The following sequences (Figure 2) are for the oligonucleotides one must order and anneal to generate the full set of Anchors and Terminators.  The Anchor_A piece must be used in an Anchor that is meant to bind an AB Byte and Anchor_B for a BA Byte.  Terminators containing the Term_A piece will bind BA Bytes, whereas those with a Term_B will bind AB Bytes; remembering that Terminators bind the the 3' end of Bytes and Anchors to the 5' end of Bytes. The Term_Comp and Anchor_Comp sequences are the complementary sequences that anneal to both the Term_A/B and Anchor_A/B pieces respectively to give the 21 bp dsDNA portion of both the Anchor and Terminator.  Thus if you want to make an Anchor with an A overhang, you must anneal Anchor_A with Anchor_comp, etc.
Figure 2: Sequences of the oligonucleotides used to make the Anchor and Terminator.
Anchor Variants and Binding Capacity
Anchor Version #1
As seen in Table 1, we considered three main types of anchor systems.  The simplest being a 5' biotinylated 20 nt ssDNA anchor.  The product brochure for the paramagnetic streptavidin beads from NEB (Cat. # S1420S) claimed that the binding capacity of such an oligo is 500 pmol mg-1 beads.  We did not bother to confirm this because this anchor does not allow for ligation of incoming Bytes, since there is no complementary strand to which the incoming Byte may ligate. There is also no mechanism to release the construct from the bead, other than boiling the beads (which is not desireable).
Anchor Version #2
A simple anchor system we had initially developed was one whereby we use 5'biotinylated forward primers (pAB/pBA forward sequencing primer) without the incorporation of deoxyuracils, and uracil containing universal reverse primers (pAB_R and pBA_R).  PCR is conducted in the presence of the Byte you wish to make your Anchor and the result is amplified 5' biotinylated pAB/pBA insert (gene) with a 3' uracil containing end.  The characteristic 12 base overhangs are generated by USER digestion, however since the biotinylated forward primers do not contain uracils, only the 3' end of the PCR product is acted upon by USER, thus only the 3' end of the "Byte" has an overhang.  This was then directly bound to the beads.  The one advantage of this system is the anchor itself is a Byte and contributes directly to the final construct size.  However, it can only be released by NotI digestion (a consequence of the forward primer sequences used).  Most importantly, it was found that this method of anchoring had terrible binding capacities, depending on the size of the anchor piece (2-8 pmol mg-1 beads) due to the lack of a ssDNA spacer region between the 5' biotin and the dsDNA region, and that fact that binding capacity has an inverse relationship with anchor size.
Anchor Version #3: The BioByte Anchor
The anchoring system we decided on was the one described above.  It's small size and the presence of the ssDNA spacer region gives this anchor a high binding capacity of about 200 pmol mg-1 beads.  The presence of the dsDNA region allows Bytes to be ligated to the anchor and the incorporation of a deoxyuracil containing I-SceI site allows the construct to be released via the two methods described already(USER and I-SceI digestion).
Binding Capacity Determination
Anchor system #2, whose protocol for binding capacity is not shown, was tested by binding a known high concentration of the biotinylated DNA to a known amount of beads.  Binding capacity was determined by quantifying both the decrease in concentration of free anchor in solution after binding to the bead as well as measuring the amount of DNA is solution following enxymatic release from the bead, a direct measurement of the ng of DNA bound. This system of anchor, that is biotinylated gene-size dsDNA, had a paltry binding capacity of only 2-8 pmol mg-1 beads.
The bead binding capacity for the BioBytes anchor system was determined, see the protocol here . The results are shown below.
The graphs show a strong hyperbolic relationship of binding capacity between amount of beads used and a linear relationship to the concentration of DNA used. Thus the assay we have outlined in the protocols section is flawed.  Having run the experiment multiple times and calculated similar binding capacities for the various volumes of beads we decided to just use the binding capacity for 40 uL of beads, since this is the amount we use for all our assemblies. The binding capacity for the BioBytes anchor system is 209 ± 20 pmol mg-1 beads.
Retrieved from " http://2009.igem.org/Team:Alberta/DNAanchor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
