"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Project/ModelValid
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Model Validation
The majority of essential gene experiments, in literature, involve single gene knock outs. Our model is novel, in that, it has a unique ability to predict what combinations of gene deletions will be lethal. It is important to determine how accurate our model is in determining multiple gene deletion results.  Due to this, the Model Validation attempts to answer the question: Is a series of multiple deletions that cause death in our model, accurately reflected in vivo?
We are currently in the process of improving this model and hope to have it completed prior to competition.  Please see our poster at the competition for additional information on the validation experiments.
The Experiment
The purpose is to predict a series of double deletions, via computational analysis, that accurately predict the model's effects in vivo. The MATLAB program that was used can identify genes that work in combination.  The DualDeletionTest.m file is designed to slowly delete reactions one at a time to determine which reactions are essential to the cell.  The data which was saved, was the genes (listed by blattner number) which catalyze the reaction.  The list was cross referenced with the PEC database to ensure that all genes were considered unessential.  Once completed, the list was narrowed down to reactions which used only two genes.  Finally, the two genes were deleted in the model and their resulting growth rate was recorded.  This was done to clarify that the growth rate was either zero or incredibly low (as predicted by the model).  Two sets of genes were selected to perform wetlab experiments, cysM and cysK, as well as, metH and metD.  (To see data produced by DualDeletionTest.m and the processed data click here ).
In order to delete one of the genes of interest, cells from the Keio Knockout Collection were procured which contained knockouts of CysK and MetH.  These cell lines replaced each gene with a Kanamycin resistance cassette flanked by two FRT sites allowing for its removal with FLP recombinase.  In order to remove the second gene, an ampicillin resistance cassette was produced via PCR with 50 bp of homology on either side which corresponds to the sequences on either side of the second gene to be knocked out. (Click here to see the PCR primers).
pKD46 carries the lambda recombinase genes which allow for the ampicillan homology cassette to recombine into the genome.  Lambda recombinase is a temperature sensitive plasmid and its activity is induced by arabinose.  The cell should no longer be viable after insertion of the cassette. To test this a control plasmid was used to ensure that the dual gene deletion is causing death.  The Keio Knockout Collection uses the BW25113 E.coli strain.  This was used as our control cell strain.  In theory, insertion of the cassette should not cause death in the control strain since there will only be one deletion made.  This will ensure that the cassette is being inserted correctly and it is the cause of the Keio Collection cell line's death.
Control Plasmid Experiment
Keio Knockout Plasmid Experiment
The above diagrams only demonstrated the experiment for the cysK and cysM genome, however, the same experiment would occur with metH and metD.  The results of this experiment will demonstrate the effectiveness of the model.  Although simple, this will hopefully validate the use of the model in the production of the BioBytes essential genome list.
Retrieved from " http://2009.igem.org/Team:Alberta/Project/ModelValid "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
