"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Optimization
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Optimizing Linear Assembly
Much work has been done to try and increase the efficiency by which we generate the Bytes, anchor them, assemble them, and terminate them.  A general outline of the optimizations we have considered and worked on are shown below as well as their effects on the process.
The Uracil Dilemma
The original format of the universal primers did not have the uracils distributed evenly within the primer.  The result was poor efficiency in construction on a bead.  Our hypothesis was that the uracils, if they were distributed more evenly, would create smaller pieces of ssDNA that would more easily melt off the Byte to generate fully ssDNA 12 base overhangs.  The first version of our USER ends is shown below.  By changing primers to their current form we have consequently increased efficiency of construction 2.5 times that of the first version.
PCR Optimization
The first step in producing workable quantities of BioBytes is PCR with the universal deoxyuracil-containing primers.  Only slight tweaking of PCR conditions was required to produce ideal quantities of Bytes.
The Primers
HPLC purified primers were ordered and PCR was attempted with them.  The motivation for this was the possibility that during oligo synthesis only a fraction of the synthesized primers are full length.  We thought that this may have an effect on the Byte end generation.  Unfortunately not only were the primers very expensive to have HPLC purified and that they were in very low yield, but also that the resulting Bytes showed no improved efficiency in ligation.
The dNTPs
Concentration effects of dNTPs were considered.  We ran six PCR reactions in total: 3 sets for each AB/BA primer pair.  The three conditions tested were 0.2, 0.6, 0.8, 1.2, and 1.5 µM final concentration of dNTPs.  Optimal PCR yield was at 0.2 µM dNTPs.
BioByte Processing
An essential step in assembly with BioBytes is the preparation of the Bytes.  Following PCR the product is USERTM digested to nick the DNA.  Finally, the Bytes are purified away from these small ssDNA pieces to prevent their binding to the sticky ends during assembly and consequently negatively influencing the efficiency of construction.  The following describes the results of optimization experiments conducted to increase efficiency of BioBytes.
USERTM Digestion
The amount of USERTM in the digest was investigated by digesting two sets of BA/AB Bytes: either 4% or 10% total volume of USERTM was added to the PCR product tubes.  The 4%, for instance, corresponds to 1 µL of USERTM mix in a 25 µL PCR reaction.  The conclusion was that slight improvement in ligation efficiency was seen for 10% over 4% USERTM addition.
USERTM digestion duration was considered at 0.5, 1, 1.5, and 2 hours.  By examining the efficiency of ligation of AB and BA Bytes processed under these conditions that no changes in efficiency were observed after 1 hour.
PCR purification conditions were considered in terms of the temperature at which the purification, using a spin column kit, was conducted. 23°C(room temperature), 37°C and 50°C conditions were tried.  We observed maximal efficiency in terms of yield and ligation efficiency of Bytes by PCR purifying at 37°C.
Alternative purification techniques proved better than PCR purification.  We found that by running the USERTM digested PCR products on a gel then gel extracting the band of interest not only prevented carry over of template DNA from PCR and extraneous PCR products (minimal anyway), but also showed better efficiency for ligation of the Bytes. To optimize even further we found that adding pH 5.2 3M sodium acetate to the dissolved gel pieces gave increased yields.
BioByte Assembly
Once the Bytes have been amplified, digested, and purified they can be assembled onto the bead.  The process of building the constructs on the bead is simple but critical since by this point you have invested money and time into getting your Bytes ready.  Thus it was important to optimize the assembly as much as we could in the short duration of the project.
Ligation
The Ligase itself is an important part of Byte assembly on a bead.  We tried both T4 and Taq DNA ligases and found that by far T4 was better. T4's increased efficiency allowed large constructs to be assembled with higher relative yields.  We also tried ligation at room temperature versus at 4°C, hypothesizing that at reduced temperature ligation efficiency will go down but the 3' 12 base Byte ends may anneal more stably.  We ligated for nearly 1.5 hours under both conditions and found that ligation at room temperature is preferable.
Incubation time was tested by allowing two complementary Bytes to ligate for 15, 30, 45, and 60 minutes.  A control was done in the absence of ligase, where no ligated product was observed.  The results indicated that 15 minutes is sufficient but playing it conservatively we went with 20 minute ligation times.
Assembly
The effect of BioByte Excesses on the efficiency of assembly was investigated by binding 5 µmol of anchor version#2 (see the Anchor/Terminator section upon which 1x, 2x, and 5x excesses of a second Byte were added.  There was a clear and drastic improvement in ligation efficiency and thus as large a molar excess of successive Bytes should be added to the growing construct as one can muster.
The volume of washes between Byte addition to the beads is of concern.  Large washes tend to slow bead pelleting and thus leads to loss of beads since some will be aspirated off.  On the other hand, small washes are not effective. By simple experimentation we found that 75 µL washes of 40 µL worth of 4 mg mL-1 beads works the best.
The number of washes was optimized by binding Bytes to anchor bound beads.  The aspirated washes were saved and run on a gel.  Seven consecutive 75 µL washes with wash/binding buffer were done.  It was evident that after even just one wash most of the excess DNA was gone and just two washes removes all traces of excess DNA.
Alternatives to the USERTM System
Originally, project BioBytes required the use of a few different restriction enzymes to generate the 12 base 3' sticky ends. Also, we had experimented with termination PCR to generate our ends using the universal deoxyuracil primers.  However, the current system of BioBytes remains the most effective to date. Alternatives to the current system are still being considered.
The "Nicking" System
The Nicking system was the precursor to the current BioBytes method using uracil containing primers.  This alternative method is what we have built BioBytes upon, though it has been changed dramatically.  The Nicking system required the cloning of genes into pAB and pBA, just as the current system does, however it also required 3 different restriction enzymes to cut and nick the template. The system was tested but showed much lower efficiency than the current system.  It was also hard to obtain Bytes at high enough workable concentration.
uPASTE
Due to the need for the obscure DNA polymerase Pfu Cx and the USERTM mix as well as the money and time to use these things, it was an impetus to try to develop an alternative to the current BioBytes method.  The alternative that was being developed was dubbed uPASTE (Uracil PCR Amplified Sticky Ends).  It was hoped that the sticky ends could be generated in the PCR step, foregoing the rest of the processing steps of the current system.  Pfu contains a uracil binding domain that prevents it's reading through of uracil containing DNA.  We figured we could use this to terminate PCR with primers containing a single deoxyuracil.  However, further research and consultation showed us why the system failed after even significant attempts to optimize the PCR. Pfu polymerase binds uracils irreversibly, thus inhibiting the polymerase.
Retrieved from " http://2009.igem.org/Team:Alberta/Optimization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
