"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/ByteCreation
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Byte Creation
Standard Plasmids
Each part to be assembled via the BioBytes method needs to have its ends altered to either the AB or the BA Byte type. To do this, unique plasmids were developed and named pAB and pBA (Figure 1). These plasmids were designed and constructed from pUC19 and contain the pMB1 high copy origin. Unique cassettes were designed containing PstI, XbaI, and  primer annealing regions complementary to the ‘A’ and ‘B’ ends. The cassettes (Figure 2) were synthesized and inserted using two restriction sites (EcoRI and NsiI). This left an EcoRI site on the final plasmid as well a PstI scar site. The primer annealing regions in pAB are reverse to that in pBA so that compatible sticky ends can be produced in either plasmid. Genes can be inserted using XbaI and PstI (or an enzyme which produces a compatible sticky end). The plasmids were originally designed for a previous but now obsolete system, where the sticky ends were generated via nicking enzymes.
Figure 1.
Figure 2.
How To Create a Byte
In order to format a part as an AB or BA form Byte, the part first needs to be cloned in to pAB or pBA, respectively. This is done using a XbaI and PstI digest of both the insert and backbone, followed by ligation to place the part inside of the AB or BA cassette.
Once the part is cloned, universal PCR primers containing deoxyuridine residues are used to amplify the part. To create an AB Byte from pAB, the universal primers pAB_F and pAB_R are used. For the creation of a BA Byte from pBA, the primers pBA_F and pBA_R are used (Figure 3).
Figure 3.
After amplification, treatment with USERTM mix (available from New England Biolabs) creates a nucleotide gap at the position of the uracil by first excising the uracil base by Uracil DNA Glycosylase and then cleaving the phosphodiester backbone at the apyrimidinic site via Endonuclease VIII. The resulting short oligonucleotides are then purified away from the PCR product to produce mature 12 base 3' overhangs of the AB or BA form Byte (Figure 4).
Figure 4.
The protocol for amplifying and digesting AB and BA Bytes can be found in our lab section .
Retrieved from " http://2009.igem.org/Team:Alberta/ByteCreation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
