"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Alberta/Project/Overview
From 2009.igem.org
Home
Team
Undergraduates Team Advisors Faculty Consultants
Project
Bio Informatics Automation Assembly Process Recombineering Cell Reboot Parts Outreach
Notebook
References
Keyword Author Title Year
Links
External Site
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Thank You To Our Generous Sponsors
Project Overview
This year's iGEM project can be subdivided into two sub-projects. The first and most important of which is the Life Bytes chromosome assembly system. This system refers to a mechanism for rapid and reliable construction of plasmids (i.e.: artificial gene sets) in vitro. The second, The Minimal Genome project, refers to the ultimate goal of rapid and reliable DNA assembly, that is, the construction of an artificial E. coli chromosome. Furthermore, it includes the strategy of gene selection, arrangement, artificial chromosome insertion and the destruction of the host's chromosome.
Life Bytes Chromosome Assembly
Current methods for DNA assembly are incredulously slow and complicated, and tend to break down with large scale additions. The current method, in brief, is to incorporate genes constructed in the form of "bricks", put them into a helper plasmid, amplify them within the bacterial host E. coli, and purify them before beginning the next cycle of additions. At each addition step, the construct must be structurally and functionally evaluated. The entire cycle is routinely completed over three to four days, while troubleshooting and correction require considerable effort. It is also pertinent to know that the system has a tendency to break down after five or more additions.
These issues have left a void in genetic engineering that the Life Bytes chromosome assembly system hopes to fill. Our system involves the linking of Streptavidin to a carefully structured set of magnetic beads. Streptavidin strongly binds to biotin, a moiety that can be synthetically added to double stranded DNA molecules. This Bead-Streptavidin-Biotin-Double Stranded DNA platform provides a secure base for genetic information to be added sequentially. Genes are to be created in a standardized format so that they can added one after another, with each gene binding to the last, to form a long chain of DNA as per Figure 1 (See below). This is a far simpler method than the current method of inserting single genes into an already circular piece of DNA. In one final step, the single stranded DNA is released from the biotin and recircularized with the end of the final brick. This is illustrated in Figure 2 (See below). The Life Bytes chromosome assembly system requires ~15 minutes per gene addition cycle, a marked improvement over the current three day methods.
Figure 1: A means for linking consecutive genes using the Life Bytes chromosome assembly system.
Figure 2: A means for recircularizing the artificially constructed chromosome once all the desired bricks have been sequentially added.
The Minimal Genome Project
The minimal E. coli genome has been a holy grail of biology for a number of years. E. coli is the most widely used cellular research tool by the Molecular Biology community. Since scientific research is based upon reductionism and simplification for understanding, a simplified version of an experimental model organism such as E. coli is, in principle, preferred as a chassis for experimentation. To reduce the E. coli genome to roughly 10% its original size shows a great simplification of this model organism.
To create such an organism, we plan on building an artificial E. coli chromosome using the Life Bytes chromosome assembly system and inserting it into living E. coli. We then intend to remove the host chromosome by making it incapable of division. This allows only the artificial, inserted chromosome to propagate through multiple generations as the cells grow and divide. This is markedly different than the current, time-consuming method of knocking out inessential genes, one at a time, in an effort to produce the minimal genome. It is this difference that we hope to exploit in our attempt to win the race to produce the minimal E. coli genome.
Retrieved from " http://2009.igem.org/Team:Alberta/Project/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
