"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Project/Bioinformatics
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Why a Minimal Genome?
One of the most useful applications of the BioBytes assembly method is the production of entire genomes.  This brings the synthetic biology research community closer to one of its holy grails, the production of a viable synthetic minimal organism.  For this reason the BioBytes team has attempted to create the tools and design principles needed to produce a minimal genome.
A minimal genome provides many benefits to the scientific community.
Genomes are extremely complex. Producing a minimal genome allows for a better understanding of the function and interaction of key cellular components needed for life.
A minimal cell provides a chassis for future research with minimal intracellular inteferents.  This makes it the optimum research vector.
Genome Design
Due to complexity of producing a minimal genome, its development has been shortened into three sections:
The selection of essential genes to be used in the genome
Building the genome via the BioBytes Assembly Method
Using recombination to eliminate the original host chromosome and replace it with the minimal chromosome
Why E. coli?
The E. coli bacterium (strain MG1655) was chosen as the model organism for the production of our essential genome.  Although other organisms have smaller genomes (E. coli contains over 4500 genes) Escherichia coli is the most commonly used laboratory organism.  This means that it is one of the most widely studied and understood organisms.  This gives us the greatest success in producing a minimal genome, while simultaneously producing the most useful research vector for the scientific community.
Determining Essential Genes
E. coli has over 4,500 genes.  The size and complexity of this genome makes it almost impossible to manually process.  An ''in silico'' approach allows for this complex data to be more easily collected, manipulated, and interpreted.  Bioinformatics has aided us in accomplishing the following:
Review lists of essential genes in the literature and existing databases and compile a preliminary essential gene list
Model the metabolic reactions and net growth rate of E. coli with given gene sets. This identified additional metabolic genes essential to a minimal genome.
Identify knock out combinations that could be tested in the wet lab, to verify the accuracy of our metabolic model.
Select standardized promoters and terminators that would replace the natural promoters and terminators of essential genes.
Determine which promoter should be used with which gene, by analyzing expression level data.
Design primers to amplify all essential genes from genomic DNA.
These steps have all been completed, and are described on the following pages.
Gene Selection
In order to produce a preliminary genome list, various databases and papers were used.  These were determined through a variety of different experimental methods and have very limited overlap.  Each gene must was carefully considered and a gene list of 332 genes was produced.  Additionally, 29 genes were found to be essential for the RNA's.
Click here for more... .
Metabolic Modeling
To verify that all genes necessary for metabolism are included in our essential gene list, a computer model was used.  The Model was produced by the Palson group at the University of San Diego and was used in conjunction with the Cobra Toolbox developed by the System's Biology Research Group.  It provides a new "in silico" approach to identifying essential genes.  The results from the computational analysis suggests that many more genes are required in order to produce a viable minimal genome.  This added an additional 118 essential genes.  Together with the Literature Research, 450 genes were found to make up our essential gene list.  In order to accomplish this a series of programs were developed to be used with the Cobra Toolbox.  These programs allow for '''the determination of any organism's minimal metabolic network.'''  The results of the metabolic modeling is currently being researched in the wetlab to demonstrate its accuracy.
Click here for more...
Retrieved from " http://2009.igem.org/Team:Alberta/Project/Bioinformatics "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
