"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/MedalRequirements
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Medals and Area Prize Achievements
Bronze
Design and document BioBrick parts: We’ve entered 422 BioBrick parts into the Registry of Standard Biological Parts. In summary, we’ve submitted:
2 backbone plasmids
12 promoters in pAB or pBA
2 transcriptional terminators in pAB or pBA
3 anchor sequence components
3 chain terminator components
22 reporters, selectable markers and genes in pAB or pBA
2 primers for sequencing out of pAB and pBA
4 USER primers for the BioByte assembly method
2 primers for amplifying an origin with Biobyte ends
185 primer pairs for essential E. coli genes
All submitted parts have been verified as the correct size using test digests and gel electrophoresis. In addition, over 26 parts have been sequenced to verify correct insertion into pAB or pBA. Sequencing files are posted on the Parts Registry as advanced sequence analyses.
Submit DNA:
We’ve submitted DNA for 26 parts, including 24 parts in pAB or pBA, and both the pAB and pBA plasmids.
Silver
Demonstrate that at least one new part works as expected:
The BioBytes rapid assembly system, consisting of the pAB and pBA plasmids and the universal uracil primers, has been demonstrated to function for rapid on-bead assembly. This is the principal achievement of the 2009 U of A team.
In addition, all 185 pairs of primers for essential genes submitted to the Parts Registry have been tested in PCR's and shown to give a product of the correct size. The essentiality of all these genes for a minimal genome was predicted using a computer model. Also, pAB/BA primers for sequencing inserts in pAB/BA worked as intended when used to verify part insertion.
Characterize the operation of at least one new BioBrick part and enter this information:
The pAB and pBA plasmids and USER primers have been extensively tested in order to optimize the BioBytes assembly method, as documented in the DNA Assembly section of this wiki.  We’ve also tested several components in pAB and pBA using the BioBytes assembly method, and verified by gel electrophoresis and sequencing that they assemble properly.  Moreover, we’ve characterized the behavior of parts for Biobyte assembly in a microfluidic chip, demonstrating successful construct production on a micro-scale.
Gold
Only one of the following criteria must be met for Gold. However, our team has met all four possible criteria:
Characterize or improve an existing BioBrick part or device:
The GFP and RFP parts we have submitted in pAB and pBA are derived from preexisting Biobrick parts. By adapting these parts to the BioBytes plasmids, we allow them to be used in the BioBytes assembly method, expanding their applications.
Furthermore, we adapted the Anderson collection of promoters to the Biobytes method. We removed NheI and AvrII cut sites from the consensus promoter and added two nucleotides downstream of the -10 region to place an A as the +1 nucleotide. These promoters are currently being tested by cloning them upstream of a ribosome binding site-red fluorescent protein segment, and checking for red colonies.
Help another iGEM team:
We constructed an expression plasmid for MMS6 (part K249019 ) for the University of Lethbridge team.
Develop and document a new technical standard:
We have written and submitted a new RFC, BBF RFC 47, detailing how the BioBytes assembly method can be used, and have included detailed protocol for how to perform this assembly method.  RFC #47 The BioBytes Assembly Standard can be found here .
Outline and detail a new approach to the issue of human practices:
As a novel approach to human practices, we worked with eight internationally competitive debaters to produce a debate about the ethical and societal implications of developing artificially engineered organisms. This debate was filmed and is posted and summarized on our wiki for the benefit of the public. The extensive experience of the debaters brings an expertise about policy issues not commonly seen among science and engineering students. The diverse background of the debaters allows a wide range of opinions to be reflected that could not have been imagined by an iGEM team alone. The format of debate allows both sides of an issue equal opportunity, and requires the use of well-reasoned arguments and evidence.  Moreover, debates are fast-paced and engaging, capturing an audience’s attention.  The debate is a valuable resource for educating synthetic biologists about public reactions, assessing public knowledge about genetic engineering, helping policy-makers make well-reasoned decisions, and helping the public form their own opinion of synthetic biology.
See U of A Debate
Moreover, we reached out to high schools, summer camps, libraries and colleges, doing demonstration debates about ethics and presentations on the science of synthetic biology. We’ve reached over 230 students already and are scheduled to reach over 400 students by January. We’ve developed and posted resources for teaching synthetic biology to a wide range of age groups. From the feedback we’ve collected and analyzed, students almost unanimously feel learning about synthetic biology is valuable.
See Presentations
Special Prizes
Best Model:
We’ve developed a novel system for comprehensively analyzing the entirety of the known metabolic network of E. coli. Through this system, one can assess the net biomass production of an E. coli in which any combination of metabolic gene are present, and thus assess which combination of pathways are necessary for cell survival. We used this model to predict the minimal set of metabolic genes required for E.coli survival, towards the goal of producing a minimal E. coli genome. This minimal gene list included 117 essential genes never previously identified as essential for a minimal genome. Moreover, a graphical user interface has been developed for exploring the results of our minimal metabolism modeling.
See Modeling
Best New Standard and Best Foundational Advance:
The BioBytes RFC (BBF RFC 47) outlines what is currently the only method for DNA assembly that is fast, modular, sequential and in vitro.
Synthetic biology needs more than minor modifications to existing evolutionary plans. We’ve developed a method of gene assembly allowing complete genome re-design. The speed and automation of the Biobytes method makes possible the maximization of modularity on a grand scale. Imagine a synthetic genome grouping common pathway components and components with similar levels of expression. This degree of organism control would be a milestone marking synthetic biology as a mature field. The Biobytes method of gene assembly allows us to efficiently test, optimize and correct genome scale design principles.
There are currently two alternatives for gene assembly. The first, BioBricks, is modular but slow. The second, the use of unique long sticky ends for each piece, is fast but non-modular.
BioBytes is the only method that is fast, modular, sequential and in vitro:
Fast: The addition of each DNA segment takes only 20min, a roughly 200-fold increase in speed from traditional cloning. Moreover, we’ve demonstrated that the Biobytes method is automatable and can be performed on microfluidic chips.
Modular: Our method allows standard parts such as the backbone plasmids and USER primers to be reused, greatly reducing expenses for large scale projects. Once parts are in pAB or pBA, they can be rapidly assembled in any order, allowing easy testing of alternative designs.
Sequential: Biobytes allows tight control over the order of gene assembly. New DNA segments can add only to the unanchored end, and in only one orientation. Moreover, using two different sets of complementary ends prevents concatamerization of parts before assembly.
In vitro: Using an organism as an intermediate is time-consuming and limits one’s ability to control and assess the changes being made. For this reason, an in vitro method such as Biobytes is essential. Genome-sized constructs can be transformed into an organism after construction is complete.
Overall, the BioBytes method gives synthetic biology the tools to understand and organize complexity, standardize  robust parts, use modular strategies and rapidly test rational designs and computational models. With BioBytes we can start asking the most fundamental questions: to what extent do the rules of engineering hold true for biology? To what degree does life equal the sum of its parts?
See DNA Assembly
Retrieved from " http://2009.igem.org/Team:Alberta/MedalRequirements "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
