"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/References/Publications/One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products
From 2009.igem.org
Kirill A. Datsenko and Barry L. Wanner
Proc Natl Acad Sci U S A. 2000 Jun 6;97(12):6640-5
Abstract: We have developed a simple and highly efficient method to disrupt chromosomal genes in Escherichia coli in which PCR primers provide the homology to the targeted gene(s). In this procedure, recombination requires the phage λ Red recombinase, which is synthesized under the control of an inducible promoter on an easily curable, low copy number plasmid. To demonstrate the utility of this approach, we generated PCR products by using primers with 36- to 50-nt extensions that are homologous to regions adjacent to the gene to be inactivated and template plasmids carrying antibiotic resistance genes that are flanked by FRT (FLP recognition target) sites. By using the respective PCR products, we made 13 different disruptions of chromosomal genes. Mutants of the arcB, cyaA, lacZYA, ompR-envZ, phnR, pstB, pstCA, pstS, pstSCAB-phoU, recA, and torSTRCAD genes or operons were isolated as antibiotic-resistant colonies after the introduction into bacteria carrying a Red expression plasmid of synthetic (PCR-generated) DNA. The resistance genes were then eliminated by using a helper plasmid encoding the FLP recombinase which is also easily curable. This procedure should be widely useful, especially in genome analysis of E. coli and other bacteria because the procedure can be done in wild-type cells.
Link: NCBI
Retrieved from " http://2009.igem.org/Team:Alberta/References/Publications/One-step_inactivation_of_chromosomal_genes_in_Escherichia_coli_K-12_using_PCR_products "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
