"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Project/BeadBindingCapacity
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Thank You To Our Generous Sponsors
Anchor Binding Capacity
What you will need:
Paramagnetic Streptavidin Beads
Annealed Anchor
Binding Capacity
Prepare a 20 μM solution of the Anchor (same as "Annealing Anchor/Terminator" but using 30 μL water instead of 80 μL)
Set up a set of microcentrifuge tubes with 10, 20, 30, 40, 50, 60, 70, 80 μL of 4 mg mL-1 beads dispensed into the respectively labelled tubes.
Wash the beads twice with 100 μL Wash/Binding buffer and once with 100 μL water, then aspirate water off the beads.
Prepare 4 μL of the 20 μM Anchor solution to separate tubes (corresponding to which bead-containing tube they will be added to) along with enough water to bring the final solution volume up to half the dispensed bead volume. (i.e. to the 40 μL tube add 4 μL Anchor + 16 μL water).
Add half of this solution to the tubes containing the washed beads. Retain the other half.
Suspend the beads in the DNA solution.
Let bind at room temperature for 10 minutes.
Pellet the beads with a magnet and aspirate the solution.
Take readings at A260 with a UV-Vis Spectrophotometer of the DNA solutions prior to binding to bead (the half you retained) and after binding to bead (the aspirated solution). (ng of DNA can also be found by in-gel band intensity quantification methods)
Calculation
Using the A260 readings, calculate respective ng of DNA by multiplying volume of DNA applied to beads.
Calculate the difference of DNA for each set of readings per bead volume to give ng of DNA bound to bead.
Substitute the ng DNA bound into the following equation to get pmol of DNA:
(ng DNAbound x 109) / (21519.1 g mol-1 x 1012)
Divide this number by the μL of beads used to get binding capacity:
μL Beads x 103 x 4 mg mL-1
Disclaimer
By doing the binding capacity in this manner we found that binding capacity is highly dependant on DNA and bead concentration (see the graphs "here" ).  Thus, we simply used the calculated binding capacity for 40 μL of beads since this is the volume we use for all builds.  A better binding capacity assay should be developed, but due to time constraints this method proved good enough since multiple runs of this experiment yielded numbers with small standard deviation:
209 ± 20 pmol mg-1
Retrieved from " http://2009.igem.org/Team:Alberta/Project/BeadBindingCapacity "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
