<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_Project_ModelValid"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Alberta/Project/ModelValid">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/Project/ModelValid&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Project/ModelValid&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Project/ModelValid&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/Project/ModelValid" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/Project/ModelValid</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>University of Alberta - BioBytes</TITLE></P><DIV id="header"><DIV id="mainNav"><UL id="ddNav"><LI><A href="https://2009.igem.org/Team:Alberta">  Home  </A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Team">  Team  </A><DIV><A href="https://2009.igem.org/Team:Alberta/Team">Undergraduates</A><A href="https://2009.igem.org/Team:Alberta/Team#Team_Ad">Team Advisors</A><A href="https://2009.igem.org/Team:Alberta/Team#Fac_Con">Faculty Consultants</A></DIV></LI><LI><A href="https://2009.igem.org/Team:Alberta"> Project </A><DIV><A href="https://2009.igem.org/Team:Alberta/Project/assemblyoverview">DNA Assembly</A><A href="https://2009.igem.org/Team:Alberta/Project/AutomationOverview">Automation</A><A href="https://2009.igem.org/Team:Alberta/Project/Bioinformatics">Organism Design</A><A href="https://2009.igem.org/Team:Alberta/Project/Chromosome_Assembly">Chromosome Assembly</A><A href="https://2009.igem.org/Team:Alberta/Project/Outreach">Human Practices</A></DIV></LI><LI><A href="https://2009.igem.org/Team:Alberta/MedalRequirements"> Achievements </A><DIV><A href="https://2009.igem.org/Team:Alberta/MedalRequirements">Medals &amp; Prizes</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=Alberta" target="_blank">Parts Registry</A></DIV></LI><LI><A>   Lab   </A><DIV><A href="https://2009.igem.org/Team:Alberta/Calendar">Lab Notebook</A><A href="https://2009.igem.org/Team:Alberta/Blackboard">Lab Blackboard</A><A href="https://2009.igem.org/Team:Alberta/Protocols">Protocols</A></DIV></LI><LI><A href="https://2009.igem.org/Team:Alberta/References/Keyword"> References </A></LI></UL></DIV></DIV><P><TABLE align="left" cellspacing="0" cellpadding="0"><TBODY><TR><TD><UL id="menu"><LI>DNA Assembly
    <OL><LIE><A href="https://2009.igem.org/Team:Alberta/Project/assemblyoverview">Overview</A></LIE></OL></LI><LI><A href="https://2009.igem.org/Team:Alberta/ByteCreation">Byte Creation</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/DNAanchor">Anchor/Terminator</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Optimization">Optimization</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/linearconstruct">Linear Constructs</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/plasmidconstruct">Plasmid Constructs</A></LI><LI>Automation
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/AutomationOverview">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Microfluidics">Microfluidics</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Automation">DIY Automation</A></LI><LI>Organism Design
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Bioinformatics">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Gene_Selection">Gene Selection</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Modeling">Modeling</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/ModelValid">Model Validation</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Promoters_&amp;_Terminators">Promoter/Terminator</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Primer_Design">Primer Design</A></LI><LI>Chromosome Assembly
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Chromosome Assembly">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Recombineering">Recombineering</A></LI><LI>Human Practices
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Outreach">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Presentations">Presentations</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Surveys">Feedback</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/UofADebate">U of A Debate</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Debate_Outreach"> HS Debate Outreach</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Safety">Safety</A></LI></UL></TD></TR></TBODY></TABLE></P><DIV class="all"><DIV style="background:#FFFFFF"><TABLE width="75%" style="background:#FFFFFF; padding:2px;"><TBODY><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>Model Validation</H1><DIV align="justify"><FONT size="2"><P> The majority of essential gene experiments, in literature, involve single gene knock outs. Our model is novel, in that, it has a unique ability to predict what combinations of gene deletions will be lethal. It is important to determine how accurate our model is in determining multiple gene deletion results.  Due to this, the Model Validation attempts to answer the question: Is a series of multiple deletions that cause death in our model, accurately reflected in vivo?  </P><P>We are currently in the process of improving this model and hope to have it completed prior to competition.  Please see our poster at the competition for additional information on the validation experiments.

 

</P></FONT></DIV></DIV></DIV></TD></TR><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>The Experiment</H1><DIV align="justify"><FONT size="2"><P>The purpose is to predict a series of double deletions, via computational analysis, that accurately predict the model's effects in vivo. The MATLAB program that was used can identify genes that work in combination.  The <A href="https://2009.igem.org/Team:Alberta/Modeling/DualDeletionTest"> DualDeletionTest.m</A> file is designed to slowly delete reactions one at a time to determine which reactions are essential to the cell.  The data which was saved, was the genes (listed by blattner number) which catalyze the reaction.  The list was cross referenced with the PEC database to ensure that all genes were considered unessential.  Once completed, the list was narrowed down to reactions which used only two genes.  Finally, the two genes were deleted in the model and their resulting growth rate was recorded.  This was done to clarify that the growth rate was either zero or incredibly low (as predicted by the model).  Two sets of genes were selected to perform wetlab experiments, cysM and cysK, as well as, metH and metD.  (To see data produced by DualDeletionTest.m and the processed data click <A href="https://2009.igem.org/Image:UofA_DualDeletionData.xls"> here</A>).</P><P>In order to delete one of the genes of interest, cells from the Keio Knockout Collection were procured which contained knockouts of CysK and MetH.  These cell lines replaced each gene with a Kanamycin resistance cassette flanked by two FRT sites allowing for its removal with FLP recombinase.  In order to remove the second gene, an ampicillin resistance cassette was produced via PCR with 50 bp of homology on either side which corresponds to the sequences on either side of the second gene to be knocked out. (Click <A href="https://2009.igem.org/Image:UofA_Matlab_Primers.xls"> here</A> to see the PCR primers).</P><P>pKD46 carries the lambda recombinase genes which allow for the ampicillan homology cassette to recombine into the genome.  Lambda recombinase is a temperature sensitive plasmid and its activity is induced by arabinose.  The cell should no longer be viable after insertion of the cassette. To test this a control plasmid was used to ensure that the dual gene deletion is causing death.  The Keio Knockout Collection uses the BW25113 E.coli strain.  This was used as our control cell strain.  In theory, insertion of the cassette should not cause death in the control strain since there will only be one deletion made.  This will ensure that the cassette is being inserted correctly and it is the cause of the Keio Collection cell line's death.</P><H4>Control Plasmid Experiment</H4><H4>Keio Knockout Plasmid Experiment</H4><P>The above diagrams only demonstrated the experiment for the cysK and cysM genome, however, the same experiment would occur with metH and metD.  The results of this experiment will demonstrate the effectiveness of the model.  Although simple, this will hopefully validate the use of the model in the production of the BioBytes essential genome list.</P></FONT></DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Alberta/Project/ModelValid">http://2009.igem.org/Team:Alberta/Project/ModelValid</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/Project/ModelValid" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/Project/ModelValid" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/Project/ModelValid&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/Project/ModelValid&amp;oldid=138038" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>