"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Project/Recombineering
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
What is Recombineering?
Recombineering refers to the strategic use of recombination in vivo in order to reach a defined goal. In the case of BioBytes, a method is required to target the final construct for insertion at a specific place on the E. coli chromosome.
To do this successfully, three components must be taken into account:
There must be a system for targeting the construct to a specific site of insertion
Activation of the recombination system must be under experimenter control
It must be possible to select for and verify colonies in which the insertion was successful
Targeting
We have chosen to employ the Red Recombination System from bacteriophage lambda. Lambda Red recombinase causes specific crossover events between the ends of a linear  and homologous chromosomal DNA. If the ends of a fragment contain at least 50 bp of homology to two separate sites on the E. coli chromosome, the genetic material between these two homologous regions will be exchanged. This provides the basis for our chromosome assembly system.
Figure 1.
The homologous regions must be a minimum of 50 base pairs in length for recombination to occur at a significant frequency. These homologous regions can be produced in different ways:
Regions corresponding to homologous genomic sequences can be added to any gene through PCR amplification with 5' extended primers.  However, our synthetic constructs are rather large and long-extension PCR proves rather finicky. PCR amplification across an entire linear construct in order to add regions of homology regions is not a viable option.
An alternative is to use the flanking essential genes on either end of our linear construct as the homologous regions.  This provides much greater sequence homology and should provide increased recombination efficiency.
Inducible Recombination System
The lambda Red recombinase system consists primarily of three proteins: lambda exonuclease, which progressively removes 5' nucleotides from linear dsDNA; beta protein, which binds to ssDNA and promotes strand annealing; and gamma protein, which binds to the bacterial RecBCD enzyme to inhibit its activity.  These three genes are contained on the plasmid pKD46 downstream from an arabinose promoter.  For further regulation, this plasmid contains the RepA temperature sensitive origin.  Therefore, one can induce chromosomal integration of a transformed linear DNA fragment with arabinose induction, then cure the cell of the pKD46 plasmid with growth at 42°C.  This system leaves a host cell with a specific chromosomal mutation, and free of lambda Red recombinase in order to prevent future random, unwanted recombination within the cell.
Our Efforts at Recombineering
Attempt I:
Our first attempt at recombineering entailed the integration of an ampicillin resistance cassette into a non-coding, non-regulatory region of the host chromosome.  AmpR cassette primers were engineered with 50 bp extensions that were homologous to flanking portions of the region of the chromosome to be excised.  The linear construct produced through PCR amplification with these primers was then electroporated into cells containing pKD46 that were pre-induced with arabinose.  Cells were left to incubate at 30°C for four hours, then plated under selection for ampicillin resistance.
Colonies with the intended recombination event were screened for using PCR.  PCR amplification across the expected integration region was expected to produce markedly different fragment sizes between wildtype and ampicillin-resistant cells if integration of the AmpR cassette occurred.
PCR verification showed no difference in fragment size between wildtype and ampicillin-resistant cells.  This led us to believe that non-specific recombination occurred and that the 50 bp of homology used was not great enough for site-specific recombination using this method.
Attempt II:
In order to decrease the degree of non-specific recombination, we attempted to integrate a linear construct with whole-gene homology into the chromosome.  To do this, we flanked a chloramphenicol resistance cassette with an essential gene on either side.  The region that would be excised during integration contained no known essential genes. The construct was built from five bytes using the BioBytes assembly method
Figure 2.
The gel-purified construct was electroporated into cells containing pKD46 that were pre-induced with arabinose.  Cells were left to incubate at 30°C for four hours, then plated under selection for chloramphenicol resistance. However, no colonies grew.
This indicated either that recombination did not occur, or that the cassette was non-functional.  We did not have time for further troubleshooting.
Retrieved from " http://2009.igem.org/Team:Alberta/Project/Recombineering "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
