<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_Project_Microfluidics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Alberta/Project/Microfluidics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/Project/Microfluidics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Project/Microfluidics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Project/Microfluidics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/Project/Microfluidics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/Project/Microfluidics</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>University of Alberta - BioBytes</TITLE></P><DIV id="header"><DIV id="mainNav"><UL id="ddNav"><LI><A href="https://2009.igem.org/Team:Alberta">  Home  </A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Team">  Team  </A><DIV><A href="https://2009.igem.org/Team:Alberta/Team">Undergraduates</A><A href="https://2009.igem.org/Team:Alberta/Team#Team_Ad">Team Advisors</A><A href="https://2009.igem.org/Team:Alberta/Team#Fac_Con">Faculty Consultants</A></DIV></LI><LI><A href="https://2009.igem.org/Team:Alberta"> Project </A><DIV><A href="https://2009.igem.org/Team:Alberta/Project/assemblyoverview">DNA Assembly</A><A href="https://2009.igem.org/Team:Alberta/Project/AutomationOverview">Automation</A><A href="https://2009.igem.org/Team:Alberta/Project/Bioinformatics">Organism Design</A><A href="https://2009.igem.org/Team:Alberta/Project/Chromosome_Assembly">Chromosome Assembly</A><A href="https://2009.igem.org/Team:Alberta/Project/Outreach">Human Practices</A></DIV></LI><LI><A href="https://2009.igem.org/Team:Alberta/MedalRequirements"> Achievements </A><DIV><A href="https://2009.igem.org/Team:Alberta/MedalRequirements">Medals &amp; Prizes</A><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=Alberta" target="_blank">Parts Registry</A></DIV></LI><LI><A>   Lab   </A><DIV><A href="https://2009.igem.org/Team:Alberta/Calendar">Lab Notebook</A><A href="https://2009.igem.org/Team:Alberta/Blackboard">Lab Blackboard</A><A href="https://2009.igem.org/Team:Alberta/Protocols">Protocols</A></DIV></LI><LI><A href="https://2009.igem.org/Team:Alberta/References/Keyword"> References </A></LI></UL></DIV></DIV><P><TABLE align="left" cellspacing="0" cellpadding="0"><TBODY><TR><TD><UL id="menu"><LI>DNA Assembly
    <OL><LIE><A href="https://2009.igem.org/Team:Alberta/Project/assemblyoverview">Overview</A></LIE></OL></LI><LI><A href="https://2009.igem.org/Team:Alberta/ByteCreation">Byte Creation</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/DNAanchor">Anchor/Terminator</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Optimization">Optimization</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/linearconstruct">Linear Constructs</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/plasmidconstruct">Plasmid Constructs</A></LI><LI>Automation
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/AutomationOverview">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Microfluidics">Microfluidics</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Automation">DIY Automation</A></LI><LI>Organism Design
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Bioinformatics">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Gene_Selection">Gene Selection</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Modeling">Modeling</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/ModelValid">Model Validation</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Promoters_&amp;_Terminators">Promoter/Terminator</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Primer_Design">Primer Design</A></LI><LI>Chromosome Assembly
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Chromosome Assembly">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Recombineering">Recombineering</A></LI><LI>Human Practices
    </LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Outreach">Overview</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Presentations">Presentations</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Surveys">Feedback</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/UofADebate">U of A Debate</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Debate_Outreach"> HS Debate Outreach</A></LI><LI><A href="https://2009.igem.org/Team:Alberta/Project/Safety">Safety</A></LI></UL></TD></TR></TBODY></TABLE></P><DIV class="all"><DIV style="background:#FFFFFF"><TABLE width="75%" style="background:#FFFFFF; padding:2px;"><TBODY><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>Lab-on-a-Chip Byte Assembly</H1><DIV align="justify"><FONT size="2"><P><B>Figure 1.</B></P><P>
Microfluidic chips bring several advantages to the table.  These so called Lab-on-a-Chip devices require only small amounts of reagents.  They are portable, inexpensive, typically faster, more efficient, and able to be automated.  We have built and tested prototype chips to execute our novel <A href="https://2009.igem.org/Team:Alberta/Project/assemblyoverview"> Byte Assembly Process</A>.  With these custom made chips, we have successfully demonstrated the quick and efficient assembly of 5 Bytes.</P></FONT></DIV></DIV></DIV></TD></TR><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>How The Chip Works</H1><DIV align="justify"><FONT size="2"><P>
Many design configurations were considered.  For the proof of concept, the simplest design was chosen.  The chip consists of a central washing chamber that is connected to 8 surrounding chambers through micro-channels.  None of the outer chambers are directly connected to each other.  The washing chamber is in the middle because it needs to be accessed after every Byte addition, and because this is the easiest way to set up Laplace flow towards all the outer chambers.  

</P><P><B>Figure 2.</B></P>
To assemble  the Bytes into a multi-gene construct, all of the chambers are filled with their appropriate solutions (see diagram above).  The beads are dragged with an external robotically controlled magnet from the initial chamber into the washing chamber, then into the Anchor chamber.  After 10 minutes, the Anchor is attached to the beads and the beads are dragged out of the Anchor chamber and into the washing chamber.  The beads are then moved into the 1st Byte chamber.  After 15 minutes, the 1st Byte is ligated to the Anchor on each bead.  The beads are then moved to the next Byte chamber where the next Byte is ligated.  The beads keep on moving from chamber to chamber in this manner until all of the Bytes are assembled into one complete construct.  After assembly is done, the beads are recovered and digested to release the construct from the bead.  The assembled product is then ready for transformation.
<P>Prior to Byte assembly, the software is programmed to organize the order in which the robotic arm moves the beads from chamber to chamber. Using this method, we can choose beforehand the order the Bytes are assembled in the construct.</P></FONT></DIV></DIV></DIV></TD></TR><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>Results</H1><DIV align="justify"><P> We have successfully demonstrated the assembly of 5 Bytes within one of our microfluidic chips.  The construct was made up of the anchor and 5 Bytes (alternating AB and BA Bytes).  Each Byte chamber held 4.2 uL of the specific Byte along with ligase and ligase buffer (totalling 5 uL in each chamber).  This is a very small amount compared to the volumes used in our standard off-chip protocol.  The total mass of the magnetic beads was 8 micrograms.  The procedure described in the previous section was carried out in less than 2 hours and at room temperature.  The beads were recovered from their original chamber.  The constructs were cleaved off of the beads with USER enzyme, and afterward ran on an agarose gel. 
</P><P><B>Figure 3.</B></P>
The gel showed 5 very dim but distinct bands (far right of gel).  The contents of some chambers were also run on the same gel as a control, to verify that there was no contamination from other Byte chambers.  The wash buffer used to wash the beads after they were recovered was also run on the gel.  The gel showed some self ligation in the Byte chambers, but also showed that there was no visible amount of un-linked DNA coming off of the chip with the beads.  The size of the construct interpreted by the number and location of the bands on the gel proved successful assembly of the bytes into a construct.  Repeat trials gave consensus results ergo, further proving the effectiveness of the technique.

</DIV></DIV></DIV></TD></TR><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>The Robot</H1><DIV align="justify"><P>The magnet-moving XY stage in the robot and its accompanying software were designed and built by a group of undergraduate students from the U of A as a capstone design project.  Since then, this XY stage has been incorporated into another machine meant to combine many lab-on-a-chip functions including sample preparation, PCR and capillary electrophoresis for genetic amplification and detection into one single device and one chip.  That is why the machine is so big; the internal laser, camera, and extra electronics take up a lot of space.  The software can be controlled from a Mac or PC.  Since the chip's channels were curved, it was somewhat cumbersome to attain the XY positions that the magnet needed to go to.  But once the positions were known, the rest was simple.
</P><P><B>Figure 4.</B></P></DIV></DIV></DIV></TD></TR><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>FAQs</H1><DIV align="justify"><P><B>Q. What stops the DNA in the outer wells from diffusing into the washing chamber and contaminating all of the other chambers?</B></P><P>
A. We set up Laplace flow originating from the central washing chamber.</P><P><B>Q. What is Laplace flow?</B></P><P>
A. Laplace flow is the term used for the flow of liquid caused by differences in pressure.  So by filling the central washing chamber to a height slightly higher than the outer chambers, a slow and steady flow against the concentration gradient of the DNA was accomplished.</P><P><B>Q. How do you stop the chamber contents from evaporating?</B></P><P>
A. We normally cap the chambers with a small volume of oil.  Without the oil, the small reaction volumes can evaporate in less than 5 minutes.</P><P><B>Q. What are some of the physical characteristics of the microfluidic chip?</B></P><P><B>Figure 5.</B></P><P>
A. The chip is made of 2 Borofloat glass layers.  The micro-channels are etched onto the top of the bottom layer, and the chambers are part of the top layer.  The channels are 45 micrometers by 100 micrometers.  The chip itself is approximately 1 inch by 3 inches.  Each outer chamber can hold 5 uL of liquid.  The central washing chamber volume varied from chip to chip ranging from ~2 uL to 20 uL.  Some chips had a closed central washing chamber, but were no longer used after unintentional Laplace flow was found to be largely uncontrollable.  The chips are reusable after a quick wash.
</P><P><B>Q. How do you fill the channels?</B></P><P>
A. Once liquid touches the central washing chamber, capillary flow rapidly fills the channels.  The outer chambers are then ready to be filled.</P><P><B>Q. Why were the channels made to curve like they do? Why are they not simply coming straight out of the washing chamber towards the outer chambers?</B></P><P>
A. The first reason was to increase the length that the DNA had to travel if diffusion turned out to be a problem.  Since there was limited space, curving the channels is all we could do to increase channel length.  The second reason was so that we could visually confirm that Laplace flow was working by filling the outer chambers with orange dye, and the central washing chamber with blue dye.  Shorter channel lengths would have made this hard to see.</P><P><B>Q. How do you know that the beads do not drag unwanted DNA around to contaminate other chambers?</B></P><P>
A. In our tests, we've recovered the contents of the chambers to run them on a gel and see if any DNA from a different chamber has entered.  No measurable amount of DNA was found to contaminate other chambers (including the wash chamber).  This suggests that the Laplace flow from the washing chamber is enough to cleanse the beads of DNA that has not ligated.</P><P><B>Q. Can you give me the gist of how the chips are made?</B></P><P>
A. In a CAD program, a mask is designed with the channels and chambers that we want.  Photolithography is used to pattern a mask that will be used as a template for the chips.  Borofloat glass is chemically etched using the designed mask.  The glass substrate is then cut into many ready to use microfluidic chips.  They were made at the University of Alberta's <A href="http://www.ece.ualberta.ca/~aml/">Applied Miniaturisation Laboratory (AML)</A>.</P></DIV></DIV></DIV></TD></TR><TR><TD style="height: 400; padding-left: 10px; padding-right: 10px; padding-top: 11px;"><DIV class="Outreach"><DIV style="height: 400; background:#FFFFFF; colorou line-height:100% padding: 3px 0px;"><H1>The Future of Lab-on-a-Chip Byte Assembly</H1><DIV align="justify"><P>Many of the procedures needed for Byte assembly have been accomplished on a microfluidic chip in labs across the world.  Although not used for Byte assembly and not all accomplished on the same chip, these on-chip protocols have shown that PCR, purification, digestion, electromagnetic bead manipulation, ligation, cell separation, and electroporation (transformation) can all be achieved on a microfluidic chip.  The future will bring many of these procedures together onto one chip allowing the researcher to choose their artificial plasmid on the computer, and upload the program to the microfluidic chip's microprocessor.
</P><P><B>Figure 6.</B></P><P>
The diagrams show what a Biofab-on-a-chip might look like.  It would be a large scale integration of previously independent lab-on-a-chip procedures.  Starting on the left of the first labeled diagram:
<LI>
 Bytes are PCR'd out of their plasmids and their sticky ends are created after the USER enzyme is pumped in.  The solution needs to be PCR purified and is pumped through the wavy channels.  These wavy channels contain thousands upon thousands of micro-pillars that selectively bind DNA.  The DNA does not make it through at first; the waste is made to flow into the waste chamber.  Soon after, the purification buffers are pumped through allowing the Bytes to collect in the Byte chambers.  Prior to this, magnetic beads were prepared by being dragged into the anchor chamber (either by an external magnet, or on-chip micro-electromagnets).  
</LI><LI>
When the Bytes are ready in their chambers, the supply of beads is split into groups by the external magnet or the internal micro-electromagnets.  Each portion is transferred to a different Byte chamber and attaches a Byte to the anchor.  After ligation is done, each portion of beads is led to another chamber (through the wash canal)  where each bead portion attaches a second Byte.  The process continues until a certain amount of Bytes is on each bead.  The next steps are to combine the separate Byte constructs into one &quot;MegaByte.&quot;

</LI></P><P><B>Figure 7.</B></P><LI>
Two bead groups are moved to independent MegaByte chambers where the USER enzyme cuts the constructs off of the beads.  The beads are transferred back to their storage chamber while the construct is left free in the MegaByte chamber.  The remaining bead portions that still have constructs attached are transferred to these freshly cut constructs and soon ligate.  There are now two bead portions left, each having a construct.  One of them is cut with USER, and the remaining bead portion is moved to that chamber where the constructs ligate.  At last, this large Byte construct is cut with USER and is ready for electroporation.  (Heating in the MegaByte chambers between construct additions and ligations denatures the enzymes to allow for proper conditions for MegaByte assembly.)  The newly made MegaByte is pumped into the electroporation chamber where the MegaByte is transformed into a cell and ready for extraction.


<P>
See the microfluidics section of our <A href="https://2009.igem.org/Team:Alberta/References/Keyword"> References</A> page for papers describing the above protocols.

</P></LI></DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Alberta/Project/Microfluidics">http://2009.igem.org/Team:Alberta/Project/Microfluidics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/Project/Microfluidics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/Project/Microfluidics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/Project/Microfluidics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/Project/Microfluidics&amp;oldid=169168" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>