"
Page
Discussion
View source
History
teams
Log in
 
Team:Alberta/Project/Chromosome Assembly
From 2009.igem.org
University of Alberta - BioBytes
  Home  
  Team  
Undergraduates Team Advisors Faculty Consultants
 Project 
DNA Assembly Automation Organism Design Chromosome Assembly Human Practices
 Achievements 
Medals & Prizes Parts Registry
    Lab    
Lab Notebook Lab Blackboard Protocols
 References 
DNA Assembly
Overview
Byte Creation
Anchor/Terminator
Optimization
Linear Constructs
Plasmid Constructs
Automation
Overview
Microfluidics
DIY Automation
Organism Design
Overview
Gene Selection
Modeling
Model Validation
Promoter/Terminator
Primer Design
Chromosome Assembly
Overview
Recombineering
Human Practices
Overview
Presentations
Feedback
U of A Debate
HS Debate Outreach
Safety
Chromosome Assembly
A powerful implementation of BioBytes technology is the construction of artificial chromosomes. One of the goals of our project was the design of a minimalized Escherichia coli genome (see Organism Design ). However, BioBytes alone can only facilitate the in vitro construction of a synthetic chromosome. A method is needed to insert the construct into a cell and displace the original chromosome.
One approach would be to fabricate the entire construct in vitro in the form of a bacterial artificial chromosome (BAC), insert the BAC, and then inactivate the host chromosome. However, we have adopted the approach of piecing together the chromosome in vivo by recombining synthetic sections into the original host chromosome.  This provides a step-wise means of testing the functionality of smaller gene subsets rather than attempting to find errors in an entire minimal chromosome.
In Vivo Construction
Figure 1.
This method requires a technique known as Lambda Red recombination (see Recombineering ). Synthetic sections produced via the BioBytes method can be transformed through electroporation as linear fragments. Once a fragment is in a cell, the Red recombination genes direct the section to a double crossover event at regions on the chromosome homologous to regions flanking the ends of the synthetic section. This results in the replacement of a large portion of the original chromosome with a synthetic construct. For a fully synthetic genome, this process can be repeated until only the original origin of replication remains.
Figure 2.
Advantages of Recombination Over Building BAC's
For one, it can be very difficult to transform a 400 kb construct into E. coli as a single piece. An in vivo approach allows for the transformation of a number of much smaller constructs.
By building the synthetic chromosome onto the original chromosome, there is no need for a complicated method of “rebooting” the cell. Transforming the entire new chromosome would require a way to smoothly inactivate or destroy the host chromosome without killing the cell in the process. in vivo construction allows for a smooth transition from the original to the artificial cell.
The step wise integration of synthetic material into the host genome also allows for greater ability to test the functionality of subsets of the artificial genome. If the cells die upon integration of a synthetic fragment, then there may be a problem with the new construct, or the construct has displaced an essential gene without replacing it.
Retrieved from " http://2009.igem.org/Team:Alberta/Project/Chromosome_Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
