Team:BUCT-China/Model
HOME
TEAM
Team Members
Collaborations
PROJECT
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
PARTS
Parts Overview
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Human Practices
Education & Engagement
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Plant
Software
JUDGING FORM â‡—
Model
We have built three models to showcase our parts.
Fluorescence Model
In the case of adding arabinose to all samples, we set up eight fatty acids with different concentration gradients, which were added to the test tubes, and a set of fluorescence was measured every 10 minutes. The x-axis was the time after the addition of fatty acids, and the Y-axis was added the concentration of fatty acids, the Z axis is the measured fluorescence value.
Through the model, we can see that the initial measured fluorescence value is relatively high, then it decreases sharply and then gradually increases. We speculate that this is caused by the Fade gene in E. coli, which will affect the added fatty acid and thus affect Expression of fluorescent proteins.
The figure below shows the measured fluorescence model data.
Biological Model
The picture on the right shows the biological model of Fadr. Fadr achieves its mediated regulatory function by binding to an operator sequence. In the absence of LCFA, FadR inhibits FA degradation by binding to a site in the fad gene promoter, and by binding to the promoters of the fabA and fabB genes. The site to activate UFA biosynthesis. In the presence of LCFA, they are transported into cells by FadL and activated to LCFA-CoAs by FadD. The resulting activated LCFA-CoAs bind to FadR, causing a conformational change that causes it to release from the DNA and cause derepression and inability to activate its regulated promoter. This model clearly demonstrates the interaction of FadR with DNA. The blue part represents the amino acid and its position, and the red part indicates its interaction with the corresponding site of DNA and the distance between the two points and the bond length. In order to eliminate the influence of fadE gene on fatty acids, we used RED recombination system to knock out the fadE gene, then transferred it to our constructed plasmid, re-measured the fluorescence value, and verified the fatty acid ec operon.
Logical model
We all know that operons are the switch of gene expression, and synthetic biology means that people connect "genes" into a network, allowing cells to accomplish the tasks that the designer envisions.[1] So we modeled the plasmid we constructed into a circuit network as a model. Of course, this is an ideal state model, and the factors in the cell will be  more complicated.
A:  Add arabinose A = 1                       No arabinose  A = 0                   B:  Adding fatty acid / glyoxylic acid B = 1                       No fatty acid / glyoxylate         B = 0                   S:  The switch is turned on, and the synthesis of the repressor protein representing the arabinose                           operon hinders  gene transcription.    OR1 door:  arabinose operon                      Y = 1 turns on transcription, followed by fatty acid operon repressor gene expression to produce                                repressor.                      Y = 0 arabinose operon binds to repressor and blocks gene transcription.     NOT gate:  a repressor of the trans-acting factor fatty acid/glyoxylate operon, which hinders gene transcription                           after synthesis, cannot be synthesized, and does not hinder gene transcription.     OR2 gate:  Switching effect of fatty acid/glyoxylic acid operon                      X = 1 fatty acid / glyoxylate operon does not bind to repressor protein, no hindrance                      X = 0 fatty acid/glyoxylate operon binds to repressor and blocks gene transcription     AND gate:  continuity of prokaryotic genes, directionality of transcription.                      Z = 1 transcription is smooth, fluorescent protein gene expression, producing fluorescent protein,                                            emitting fluorescence.                      Z = 0 transcription is blocked, fluorescent protein gene can not be expressed, can not produce fluorescent  protein fluorescence.
Citation
[1]Ciaran M. Lamont,Frank Sargent. Design and characterisation of synthetic operons for biohydrogen technology[J]. Archives of Microbiology,2017,199(3).
