Team:UT-Knoxville/Project
<!DOCTYPE html>
Team:UT-Knoxville/Team - 2016.igem.org
Team:UT-Knoxville/Team
Team UT Knoxville
UTK
iGEM
HOME
TEAM
PROJECT
Motivation
Pu Promoter
Toxicity Efflux Pumps
Enzyme Homologues
References
NOTEBOOK
Calendar
Methods
Protocols
ATTRIBUTIONS
Sponsors
Group
HUMAN PRACTICES
Engagement
Silver HP Requirement
JUDGING
Medal Requirements
Collaborations
Interlab
Submitted Parts
Motivation
Due to the complex nature of metabolic engineering, there remains limitless possibilities regarding shortcomings in a metabolic network. To reconcile some metabolic lapses, our team strategized this year’s project around optimizing the pAMB pathway from our 2016 iGEM project . While proud of our platform, we were keenly aware of a few obstacles hindering bioconversion.  Foremost, our Pu promoter was never able to regulate the production module. While IPTG was an effective inducer, it is expensive to scale. Additionally, we suspected that our terminal product was toxic to the cells and inhibits further bioconversion. Lastly, the AMB pathway has a high degree of specificity. To develop a more robust library of products, we would need to explore enzymatic homologs. In summary, we intend to develop a more robust microorganism for the bioconversion of aromatic waste to aromatic aldehydes.
This year, we set out to tackle these metabolic shortcomings in the following ways:
Develop a functioning Pu promoter element to fine-tune protein production in the presence of aromatic waste.
Express efflux pumps to alleviate toxic effects.
Explore enzymatic homologs to develop a wider library of products.
Pu Promoter
To fine-tune protein production, our team designed the following composite parts. BBa_K2451013 was designed for a general proof of concept, and BBa_K24510015 is directed towards fine tuning our team’s 2016 production circuit (BBa_K1966000 and BBa_K1966001).
Figure 1: Pu Promoter Parts Submitted to Registry
While we first designed these parts in 2016, our past characterizations led to inconclusive results with respect to literature [1]. After an extensive search of the registry and various experimental parameters, we set out to re-develop our regulatory Pu promoter. After various constructions and experiments, we eventually arrived at construct BBa_K2451013, depicted above. This construct displays promising results for m-xylene induction. The xylR coding sequence was synthesized during our 2016 project, and the Pu promoter was amplified from 2013 Peking’s BBa_K1031803. Confirmed sequences of these parts are available on the registry.
The experiments were performed as follows. Strains were inoculated from frozen stock and grown overnight in LB media. The following morning, the cultures were washed and reseeded to an OD=0.1 in M9 + 4 g/L glucose media. Literature has previously reported that this regulatory network is coupled with the σ54 factor [2], a nitrogen limitation protein [3]. Thus, we modulated the nitrogen content for this experiment.  As the cultures entered early/mid log phase, they were induced with 0.1 mM IPTG and 1 mM m-xylene[4]. Samples were measured for GFP fluorescence (excitation 485/20, emission 528/20 nm) and optical density (OD600nm), and data was collected over the next 24 hours on a Biotek Synergy HT microplate reader. Additionally, RFU = “Relative Fluorescence Units” = (Media GFP value)/(OD600).
Figure 2: Pu & m-xylene characterization in [1.0x] nitrogen
Figure 3: Pu & m-xylene characterization in [0.1x] nitrogen
Results:
1 mM m-xylene induces a 4-fold increase in fluorescence.
Low nitrogen concentrations marginally increase fluorescence.
To test specificity the of the promoter, we performed similar experiments in the presence of another substrate, 1 mM toluene, and 1 mM benzaldehyde, a product. The experiments were performed the same as the m-xylene conditions, but in LB media as opposed to M9.  A more detailed protocol can be found in the Methods section.
Figure 4: Pu & toluene characterization
Figure 5: Pu & benzaldehyde characterization
Results:
Both 1.0 mM toluene and benzaldehyde were unable to induce the Pu promotor.
Figure 6:  All three analytes were extracted in hexane and measured using a single 1 uL injection.
Figure 7: m-Xylene, 3-methyl Benzyl Alcohol, and m-Tolualdehyde had retention times at 7.1, 9.7, and 10.28 minutes respectively. The internal Standard ethyl benzoate had a retention time of 10.64 minutes. Signals are directly overlapped in order to show differences in concentrations.
The production of m-tolualdehyde using the m-xylene induced production circuit BBa_K2451015 also termed UTK05 was analyzed through GC/MS using an Agilent 7890A gas chromatograph equipped with a 7693A automatic liquid sampler, an HP-5ms capillary column (30 m long × 0.25 mm inside diameter with a 0.25-µm capillary film of 5% phenyl methylsilicone) and a 5975C mass-sensitive detector.  As expected our production module was not capable of producing any m-tolualdehyde or the intermediate product, 3-methyl Benzyl Alcohol, without the exogenous addition of m-xylene as shown in Figure 1. Similarly, our negative controls were not capable of producing the final product even after induction. Importantly, the inclusion of the Pu promoter to the production module did not lower the production of m-tolualdehyde. The 7.25 mg/ml of m-tolualdehyde our m-xylene induced module was able to produce matches favorably with the ~5 mg/mL produced by our IPTG induced module. This provides good evidence that helping to regulate the expression of production module conserves cellular resources by limiting expression early in growth and by basing expression on the concentration of m-xylene. Figure 2 is a representative chromatogram from the GC analysis of the production module. As the chromatogram shows, we are seeing no presence of the intermediate or the final product. Our analysis leads us to conclude that our production circuit is not only working, but is being induced by the simultaneous expression of xylR and the presence m-xylene activating the Pu promoter.
Conclusions
Given the presence of 1 mM m-xylene, our mutant strain (BL21(DE3) + BBa_K2451013) exhibited a 4-fold increase in fluorescence. This validates our construct, and we coupled it with our 2016 production circuit. In theory, we have neared our end goal- such that catalytic enzymes (xylAMB) are produced only in the presence of aromatic hydrocarbons. While we cloned this construct (BBa_K2451015), time constraints did not allow us to characterize the circuit in its entirety.  Additionally, our data suggests the nitrogen content of the media can marginally increase Pu Promoter strength per σ54 availability. This is another promising lead to further fine tune enzyme formation. Lastly, our results suggest a high degree of specificity for the Pu promotor.
Setbacks
The Pu Promoter circuit proved our most difficult part to construct, and took on a variety of forms on its way to functionality. Ultimately, the lack of an untranslated region (UTR) (about 7 to 8 base pairs) between the RBS and start codon proved inhibitory enough to prevent any perceivable m-xylene induction. Salis Lab’s RBS calculator proved critical to troubleshooting this dilemma [5][6]. The following table shows the workflow our team underwent to arrive at our functioning construct.
Figure 8: Pu construct workflow
Toxicity Efflux Pumps
Strategy
A common obstacle in metabolic engineering is the buildup of toxic products. This can be extra debilitating for strains designed for heterologous protein expression. However, bioengineered efflux systems have been demonstrated to be a practical solution that carries wide metabolic engineering applications [7]. Our 2016 platform (BBa_K1966003) demonstrated conversion of aromatic hydrocarbons to aldehydes. However, wild-type E. coli exhibits a toxic susceptibility to many of our desired products, namely benzaldehyde.  To develop a more robust microorganism, we set out to overexpress native transcriptional regulators directed to increase organic solvent tolerance (OST) [8], and alleviate benzaldehyde’s toxic effects. Several external stresses are detected by the Mar/Sox/Rob regulon, and it serves to modulate several different pathways, one of which increases expression of the AcrAB-TolC efflux pump [9]. Overexpression of the transcriptional activators (MarA+, SoxS+, Rob+) and the broadly non-specific efflux pump AcrAB-TolC has been previously demonstrated to confer resistance to variety of organic solvents and antibiotics [8].
Toxicity Parts
To increase tolerance to aromatic aldehydes, our team designed the following composite parts. The composite parts function as transcriptional regulator protein generators.
Figure 9: Toxicity parts submitted to the registry
The above constructs were all submitted by the UT-Knoxville 2017 IGEM team for this year’s project. Each gene of interest was amplified from E. coli MG1655 genomic DNA, and Gibson Assembled into a linearized vector consisting of pSB1C3, BBa_B0034, and BBa_B0010. The constructs were first transformed into Top10, and characterized and sequenced in BL21(DE3). Confirmed sequences of these parts are available in the registry. It should be noted that pRob (BBa_K2451006) is subject to denial/exception to the registry due to EcoRI and PstI in the coding sequence. However, the pRob mutant was characterized alongside the other three mutant strains, and the data remains relevant to the overall analysis. The following strains were characterized.
Strain
Plasmid
BL21(DE3)
None
BL21(DE3)
pSB1C3 (empty vector)
BL21(DE3)
pMarA (BBa_K241002)
BL21(DE3)
pSoxS (BBa_K2451004)
BL21(DE3)
pRob (BBa_K2451006)
BL21(DE3)
pAcrR (BBa_K2451008)
Figure 10: Strains characterized
A liquid Ampicillin toxicity test was performed to observe activity for the transcriptional regulators. Log-phase cells were inoculated at an OD=0.1 in various ampicillin concentrations, and optical densities were measured over 24 hours on a Thermo-Scientific Genysys30 Vis-spectrophotometer at wavelength 600 nm. Due to the relative volatility of benzaldehyde, the experiment was performed under semi-aerobic conditions (capped headspace). The following strains were characterized, with BL21(DE3) WT and BL21(DE3) + pSB1C3 serving as controls. A more detailed experimental protocol can be found in the Methods section.
Figure 11: Liquid ampicillin toxicity test
Our transcriptional regulators performed as expected. The overexpression of the transcriptional activators (MarA+/SoxS+/Rob+) conferred ampicillin resistance, and the repressor (AcrR+) displayed increased susceptibility. These phenotypes have been previously established in literature [8]. To further validate our results, a solid media assay was also performed on the mutant strains BL21(DE3) + pMarA and BL21(DE3) + pSoxS with BL21(DE3) + pSB1C3 serving as control. LB media + 1% agar was autoclaved and cooled in 55˚C water bath. 10 mM MgSO4 and various of concentrations of ampicillin were added to the media; which was poured into plates.  Frozen stocks were used to inoculate overnight cultures; which were resuspended in 0.9% NaCl sterile water to an OD600 = 1.0. Cell count was varied by serial dilutions of the following factors: 102, 104, and 106. The plates grew at 37˚C for 24 hours at which point the following pictures were taken.
Figure 12: Cell count dilutions for droplet placement
Figure 13: pSB1C3 solid media ampicillin test
BL21(DE3) + pSB1C3 serves as a positive control for antibiotic effect.
BL21(DE3)  + pSB1C3 is able at each cell count in no ampicillin.
BL21(DE3)  + pSB1C3 is only able to grow in 3 μg/mL at the highest cell count (OD=1.0).
Figure 14: pMarA solid media ampicillin test
BL21(DE3) + pMarA tolerates ampicillin concentration up to 9 μg/mL.
Figure 15: pSoxS solid media ampicillin test
BL21(DE3) + pSoxS tolerates ampicillin concentration up to 9 μg/mL.
After seeing our transcriptional regulators react to an ampicillin regiment, a liquid benzaldehyde toxicity test was used to characterize our transcriptional regulators. Log-phase cells were inoculated at an OD=0.1 in various benzaldehyde concentrations, and optical densities were measured over 24 hours on a Thermo-Scientific Genysys30 Vis-spectrophotometer at wavelength 600 nm. This experiment was performed under the same protocol as the ampicillin assay, and a more detailed experimental protocol can be found in the methods section.
Figure 16: Liquid benzaldehyde toxicity test
Figure 17: Liquid benzaldehyde toxicity test controls
Overexpressing the transcriptional activators (MarA+/SoxS+/Rob+) marginally lowered benzaldehyde tolerance.
Overexpressing the transcriptional repressor AcrR+ marginally increased benzaldehyde tolerance.
Conclusions
Our results suggest that these regulators play a complex role in E.coli’s native defense network [10][11], and for our purposes- proved difficult to directed towards benzaldehyde tolerance. We believe that the antibiotic resistance conferred by the transcriptional activators (MarA+/SoxS+/Rob+) independently validates our benzaldehyde results. Most interesting, our results suggest the AcrR+ mutant repressor strain performs inversely to the activators in respect to both ampicillin and benzaldehyde tolerance. This relationship is displayed in the following table. We assert that overexpressing AcrR is this study’s best candidate for conferring benzaldehyde resistance, but the benefits are marginal at best. While our results display distinct phenotypes, we hope to repeat these experiments with constructs that include a UTR between the RBS and start codon in order to increase translational efficiency.
Strain
Plasmid
Benzaldehyde
Ampicillin
BL21(DE3)
pMarA (BBa_K241002)
-
+
BL21(DE3)
pSoxS (BBa_K2451004)
-
+
BL21(DE3)
pRob (BBa_K2451006)
-
+
BL21(DE3)
pAcrR (BBa_K2451008)
+
-
Figure 18: Construct phenotypes
Enzyme Homologues
The xylAMB genes from the 2016 project showed great specificity, which limited the number of products that could be formed. To expand our library of products, we searched for enzymatic homologues that could catalyze a wider range of substrates and/or increase the conversion of BTX compounds to their respective aromatic aldehydes.
Through literature search, a gene cluster coding for a protein called toluene/o-xylene monooxygenase (ToMO) from the organism Pseudomonas stutzeri OX1 [12] was found. The six open reading frames corresponding to this gene were designated as touABCDEF. An advantage of this enzyme was that it metabolizes o-xylene. Our xylene monooxygenase (xylAM) enzyme from last year failed to metabolize o-xylene, while it successfully metabolized m-xylene and p-xylene. The substrates oxidized by ToMO and the products formed are highlighted below. Cresols are widely used as chemical intermediates in the pharmaceutical industry, solvents, preservatives, and various other applications [13].
Toluene -> o-cresol, m-cresol, and p-cresol -> 3-methylcatechol and 4-methylcatechol
o-xylene -> 2,3-dimethylphenol and 3,4-dimethylphenol -> dimethylcatechols
Also able to oxidize benzene, ethylbenzene, m and p-xylene, styrene and naphthalene
The benzyl alcohol dehydrogenase (xylB) enzyme homologue was found from the organism Acinetobacter calcoaceticus. The most effective substrate for the xylB homologue was benzyl alcohol [14]. Also, para-substituted substrates were more effective substrates for the enzyme [14]. Thus, this enzyme homologue was directed towards increasing the conversion of benzyl alcohol (and other related aromatic alcohols used in last year’s project) into their respective products.
The constructs designed for the enzymatic homologues project are shown below. A “mix and match” strategy was used and new enzyme homologues were going to be incorporated with the genes from last year and vice versa to observe which combination of genes yielded the largest amount of product.
Figure 16: Intended constructs for enzyme homologues
Many challenges were encountered with cloning of these genes into the pSB1C3 backbone. All enzyme homologues were synthesized by IDT. The touABCDEF cluster had to be broken down into two segments because of the ~4.5 kb length. During synthesis, the touABC gBlock failed IDT quality control standards, and thus created troubles with cloning the gene. The xylB gene was cloned multiple times with positive results with colony PCR, but negative results when the gene was sequenced each time. Thus, due to the cloning challenges, and the large amount of time spent troubleshooting with no success, the enzymatic homologues project was unsuccessful.
References
Kulakova, A. N., Kulakov, L. A., McGrath, J. W. and Quinn, J. P. (2009), The construction of a whole-cell biosensor for phosphonoacetate, based on the LysR-like transcriptional regulator PhnR from Pseudomonas fluorescens 23F. Microbial Biotechnology, 2: 234–240. doi:10.1111/j.1751-7915.2008.00082.x
Bertoni, G., Fujita, N., Ishihama, A., & de Lorenzo, V. (1998). Active recruitment of sigma54-RNA polymerase to the Pu promoter of Pseudomonas putida: role of IHF and alphaCTD. The EMBO Journal, 17(17), 5120–5128.
Hunt, T. P., & Magasanik, B. (1985). Transcription of glnA by purified Escherichia coli components: core RNA polymerase and the products of glnF, glnG, and glnL. Proceedings of the National Academy of Sciences of the United States of America, 82(24), 8453–8457.
Kim, M. N., Park, H. H., Lim, W. K., and Shin, H. J. (2005). Construction and comparison of Escherichia coli whole-cell biosensors capable of detecting aromatic compounds. Journal of Microbiological Methods, 60(2), 235-245. https://doi.org/10.1016/j.mimet.2004.09.018
Salis, H. M., Mirsky, E. A., & Voigt, C. A. (2009). Automated Design of Synthetic Ribosome Binding Sites to Precisely Control Protein Expression. Nature Biotechnology, 27(10), 946–950. http://doi.org/10.1038/nbt.1568
Espah Borujeni, A., Channarasappa, A. S., & Salis, H. M. (2014). Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites. Nucleic Acids Research, 42(4), 2646–2659. http://doi.org/10.1093/nar/gkt1139
Jones, C. M., Hernández Lozada, N. J., & Pfleger, B. F. (2015). Efflux Systems in Bacteria and their Metabolic Engineering Applications. Applied Microbiology and Biotechnology, 99(22), 9381–9393. http://doi.org/10.1007/s00253-015-6963-9
Alekshun, M., Levy, S. (1999) The mar regulon: multiple resistance to antibiotics and other toxic chemicals. Trends in Microbiology, Volume 7, Issue 10, 410 – 413. DOI: http://dx.doi.org/10.1016/S0966-842X(99)01589-9
Barbosa, T., Pomposiello, P. “The mar Regulon.” Frontiers in Antimicrobial Resistance: a tribute to Stuart B. Levy. Ed. D. G. White, M. N. Aleskshun, P. F. McDermott. Washington DC: ASM Press, 2005. Chapter 15: 209-223
White, D. G., Goldman, J. D., Demple, B., & Levy, S. B. (1997). Role of the acrAB locus in organic solvent tolerance mediated by expression of marA, soxS, or robA in Escherichia coli. Journal of Bacteriology, 179(19), 6122–6126.
Lee JO, Cho KS, Kim OB. 2014. Overproduction of AcrR increases organic solvent tolerance mediated by modulation of SoxS regulon in Escherichia coli. Appl Microbiol Biotechnol 98:8763–8773. doi:10.1007/s00253-014-6024-9.
Bertoni, G., Martino, M., Galli, E., Barbieri, P. Analysis of the Gene Cluster Encoding Toluene/o-Xylene Monooxygenase from Pseudomonas stutzeri OX1. October 1998. Applied and Environmental Microbiology, Vol. 64, No. 10
Global Information, Inc.: Premium market research reports. Cresols Market (Meta-cresols, Para-cresols, and Ortho-cresols) for Chemical Intermediates, Solvents, Preservatives, Antioxidants and Other Applications - Global Industry Analysis, Size, Share, Growth, Trends and Forecast 2015 – 2023. January 2016.
MacKintosh, R. W. and Fewson, C. A. Benzyl alcohol dehydrogenase and benzaldehyde dehydrogenase II from Acinetobacter calcoaceticus: substrate specificities and inhibition studies. 1998. Biochem. J., 255:653-661.
×
Loading ...
