<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Glossary skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Glossary</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_area"><DIV class=" header"><DIV class="mainmenu floatright"><NAV class="main-navigation pull-right hidden-xs hidden-sm" id="pageHighlighter"><UL><LI><A href="https://2016.igem.org/Team:Aachen">Home</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description#characterization">Improvement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Glossary">Glossary</A></LI></UL><LI><A href="#" class="has-submenu" style="text-decoration:none;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Basic_Part">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Aachen/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab/Protocols">Protocols/Methods</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Beyond_Basics">Beyond the Basics</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supervisors">Supervisors</A></LI><LI><A href=" https://2016.igem.org/Team:Aachen/Attributions#attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supporters">Supporters</A></LI></NAV></DIV></DIV></DIV><P><TITLE>Welcome to iGEM Aachen 2016</TITLE></P><DIV class="main_content_area "><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Glossary</H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">During iGEM we have got into the habit of using special scientific terms of synthetic biology which are not self-explanatory. But for those who do not have a biological background or want to explore an unfamiliar area, a glossary of all these terms is shown below.</P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><BIG><B><SPAN style="color:#005C04 ;">Aminoacyl-tRNA synthetase</SPAN></B></BIG>
An aminoacyl-tRNA synthetase (aaRS) is an enzyme that attaches the appropriate amino acid onto its tRNA. It does so by catalysing the esterification of a specific amino acid or its precursor to one of all its compatible equivalent tRNAs to form an aminoacyl-tRNA.<BIG><B><SPAN style="color:#005C04 ;">BioBrick</SPAN></B></BIG>
BioBrick standard biological parts are DNA sequences of defined structure and function; These building blocks are used to design and assemble larger synthetic biological circuits from individual parts and combinations of parts with defined functions, which would then be incorporated into living cells such as <I>Escherichia coli</I> cells to construct new biological systems. They represent an effort to introduce the engineering principles of abstraction and standardization into synthetic biology. Examples of BioBrick parts include promoters, ribosomal binding sites (RBS), coding sequences and terminators.<BIG><B><SPAN style="color:#005C04 ;">Codon</SPAN></B></BIG>
A sequence of three adjacent nucleotides that corresponds to a specific amino acid during protein synthesis. There is at least one tRNA for any codon, and sometimes multiple codons translate into the same amino acid. Many tRNAs are compatible with several codons. <BIG><B><SPAN style="color:#005C04 ;">Codon optimization</SPAN></B></BIG>
The vast majority of amino acids are encoded by multiple codons, which means that there are multiple tRNAs that carry any given amino acid. Of any group of &quot;redundant&quot; tRNAs, one is usually much more abundant. Codon optimization is switching the codons used in a transgene without changing the amino acid that it encodes for -- this typically dramatically increases the abundance of the protein, which is encoded by the codon optimized gene; because it replaces “rare” with abundant codons.<BIG><B><SPAN style="color:#005C04 ;">Dephosphorylation</SPAN></B></BIG>
The process by which phosphate groups are removed from a molecule by an enzyme called phosphatase. Removal of phosphate groups from a DNA fragment can prevent self-ligation.<BIG><B><SPAN style="color:#005C04 ;">DNA (deoxyribonucleic acid) </SPAN></B></BIG>
They are double standard biomolecules which acts as the genetic material for almost all the living organisms. It is made up of basic units called nucleotides. 
<BIG><B><SPAN style="color:#005C04 ;">Enzyme</SPAN></B></BIG>
Proteins that catalyze any complex reaction are called enzymes. They are highly specific on where and under which conditions they react (temperature, pH) and are biodegradable.
Some of the commonly used enzymes in laboratory includes:</P><OL style=" list-style-type: disc; padding-left:1.5cm; padding-right:1.5cm;  align-content: center"><LI><SPAN>Restriction enzymes - also known as molecular scissors, which are used to cleave DNA at a targeted site.</SPAN></LI><LI><SPAN>Ligase - enzymes which are used to fuse two ends of DNA.</SPAN></LI><LI><SPAN>DNA Polymerase – an enzyme which assembles the nucleotides to create a DNA molecules.</SPAN></LI><LI><SPAN>Protease – enzymes which breaks down proteins (in our case, Subtilisin E).</SPAN></LI></OL><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">Some of the commonly used enzymes in day-to-day life are:</P><OL style=" list-style-type: disc; padding-left:1.5cm; padding-right:1.5cm;  align-content: center"><LI><SPAN>Rennet - an enzyme that acts on milk proteins to form curd.</SPAN></LI><LI><SPAN>Lactase – enzyme used in ice cream production.</SPAN></LI><LI><SPAN>Trypsin – enzymes which remove hair from leather products.</SPAN></LI></OL><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><BIG><B><SPAN style="color:#005C04 ;"><I>Escherichia coli</I> (Abbr. <I>E. coli</I>) </SPAN></B></BIG><I>Escherichia coli </I>(abbreviated as <I>E. coli</I>) are bacteria found in the environment, foods, and intestines of people and animals. It is used extensively in recombinant DNA research because it has been genetically well characterised, which makes them desirable organisms for the researchers to work with.<BIG><B><SPAN style="color:#005C04 ;">Esterification </SPAN></B></BIG>
The process of formation of ester from the chemical reaction of an alcohol and a carboxylic acid usually in the presence of hydrochloric acid or sulfuric acid. Esters are common in organic chemistry and biological materials, and often have a characteristic pleasant, fruity odour.<BIG><B><SPAN style="color:#005C04 ;">Expression</SPAN></B></BIG>
The process by which the genetic information in our DNA are converted into a functional product, such as a protein. It mainly involves two steps:
<OL style=" list-style-type: disc; padding-left:1.5cm; padding-right:1.5cm;  align-content: center"><LI><SPAN>Transcription – process of conversion of DNA to RNA.</SPAN></LI><LI><SPAN>Translation – process of conversion of RNA to protein.</SPAN></LI></OL></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><BIG><B><SPAN style="color:#005C04 ;">Genetically modified organism (GMO) </SPAN></B></BIG>
A genetically modified organism, or GMO, is an organism which had its genetic material altered through genetic engineering to change a particular characteristic to a desirable one.<BIG><B><SPAN style="color:#005C04 ;">Genetic Engineering</SPAN></B></BIG>
The science of modification of genetic material (DNA) of an organism in order to alter its characteristics in a particular way. 
<BIG><B><SPAN style="color:#005C04 ;">Ligation</SPAN></B></BIG>
The joining of two DNA strands or other molecules by a phosphate ester linkage.<BIG><B><SPAN style="color:#005C04 ;">Marker</SPAN></B></BIG>
A selectable marker is a gene introduced into a cell, especially a bacterium or to cells in culture, that confers a trait suitable for artificial selection. They are a type of reporter gene used in laboratory microbiology, molecular biology, and genetic engineering to indicate the success of a transfection or other procedure meant to introduce foreign DNA into a cell. Selectable markers are often antibiotic resistance genes; bacteria that have been subjected to a procedure to introduce foreign DNA are grown on a medium containing an antibiotic, and those bacterial colonies that can grow have successfully taken up and expressed the introduced genetic material or in other words, these markers make the organism resistant to that particular antibiotic. Normally the genes encoding resistance to antibiotics such as ampicillin, chloramphenicol, tetracycline or kanamycin, etc., are considered useful selectable markers for <I>E. coli</I>.<BIG><B><SPAN style="color:#005C04 ;">Molecular Cloning </SPAN></B></BIG>
Cutting a piece of DNA (restriction/ digestion) from one organism and inserting it into a vector (ligation), where it can be replicated by a host organism <BIG><B><SPAN style="color:#005C04 ;"> Non-canonical amino acid </SPAN></B></BIG>
In biochemistry, non-canonical, non-coded, non-proteinogenic, or &quot;unnatural&quot; amino acids are those not naturally encoded or found in the genetic code of any organisms. Naturally, an organism could encode for 20 amino acids.</P><P align="justify" style="padding-left:3.0cm; padding-right:1.0cm; font-size: 16px;"><B><SPAN style="color:#005C04 ;">4,5-dimethoxy-2-nitrobenzylserine (DMNBS)</SPAN></B>
Serine with O-nitrobenzyl protection group, a non-canonical amino acid (“caged” serine).<B><SPAN style="color:#005C04 ;">Ortho-nitrobenzyl tyrosine (ONBY)</SPAN></B>
Tyrosine with O-nitrobenzyl protection group, a non-canonical amino acid (“caged” tyrosine).</P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">They are used in the process of photocaging with its corresponding orthogonal tRNA/ synthetase pair.<BIG><B><SPAN style="color:#005C04 ;">Nucleotides </SPAN></B></BIG>
It is the basic structural unit of nucleic acids such as DNA and RNA. Each nucleotide consists of: a 5-carbon sugar, a phosphate group and nitrogenous bases like Adenine (A), Guanine (G), Thymine (T)/ Uracil (U) and Cytosine (C).
<BIG><B><SPAN style="color:#005C04 ;">Optical Density</SPAN></B></BIG>
Optical Density of a substance is defined as the amount of light which it allows to pass through. The higher the optical density, the lesser the light would pass through or the more the light is absorbed. It also refer to the refraction of substance. It can be measured as a logarithmic ratio of intensity of transmitted light and the intensity of incident light. <BIG><B><SPAN style="color:#005C04 ;">Photocaging</SPAN></B></BIG>
Photo-removable or photo-labile protecting groups are molecules that are covalently linked to a target molecule to inhibit its activity. Upon application of light treatment, the target molecule is released from the caged structure, thus activating its function. The process of inhibiting (caging) the function of a molecule through a protection group (which can be removed by light) is called as photocaging.[<A href="#ref1glossary"><U style="color:#0000EE; ">1</U></A>]<BIG><B><SPAN style="color:#005C04 ;">Plasmid</SPAN></B></BIG>
A small, circular DNA structure that is separate from the cell's genome and can replicate independently from the host cell. Plasmids are used in the laboratory to deliver specific DNA sequences into a cell.<BIG><B><SPAN style="color:#005C04 ;">Polymerase chain reaction (PCR) </SPAN></B></BIG>
PCR (polymerase chain reaction) is a method to analyze a short sequence of DNA (or RNA). PCR is used to amplify selected sections of DNA or RNA. It requires:
<OL style=" list-style-type: disc; padding-left:1.5cm; padding-right:1.5cm;  align-content: center"><LI><SPAN>Two &quot;primers&quot;, short single-stranded DNA sequences that are synthesized to correspond to the beginning and ending of the DNA stretch to be copied; and also to assign the starting position for the polymerase to bind on.</SPAN></LI><LI><SPAN>An enzyme called polymerase that moves along the segment of DNA, reading its code and assembling a copy.</SPAN></LI><LI><SPAN>A pile of DNA building blocks (dNTPs) that the polymerase needs as basic building blocks of the DNA.</SPAN></LI></OL></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
Its application includes genotyping, cloning, mutation detection, sequencing, microarrays, forensics, and paternity testing.<BIG><B><SPAN style="color:#005C04 ;">Propeptide</SPAN></B></BIG>
Many secretory proteins and peptides are synthesized as inactive precursors that in addition to signal peptide cleavage undergo post-translational processing to become biologically active poly-peptides. Precursors are usually cleaved at sites composed of single or paired basic amino acid residues by members of the subtilisin/kexin-like pro-protein convertase (PC) family. In mammals, seven members have been identified, with furin being the one first discovered and best characterized.[<A href="#ref2glossary"><U style="color:#0000EE; ">2</U></A>] <BIG><B><SPAN style="color:#005C04 ;">Proteolytic Activity</SPAN></B></BIG>
The process of hydrolysis of proteins into simpler, soluble substances (polypeptides or amino acids) by the action of enzymes caused by enzymes called proteases.<BIG><B><SPAN style="color:#005C04 ;">Recombinant DNA</SPAN></B></BIG>
A vector containing a foreign DNA is called recombinant DNA. <BIG><B><SPAN style="color:#005C04 ;">Reporter</SPAN></B></BIG>
In molecular biology, a reporter gene (often simply reporter) is a gene that researchers attach to a regulatory sequence of another gene of interest in bacteria, cell culture, animals or plants. Certain genes are chosen as reporters because the characteristics they confer on organisms expressing them are easily identified and measured, or because they are selectable markers. Reporter genes are often used as an indication of whether a certain gene has been taken up by or expressed in the cell or organism population.
A common reporter that we use are the <B>Green Fluorescent Protein (GFP)</B> causes cells that express it to glow green under UV light.<BIG><B><SPAN style="color:#005C04 ;">Restriction enzyme</SPAN></B></BIG>
An enzyme which is used to make cuts at specific sites of a DNA sequence. The resulting fragments can then be spliced (joined) together to form recombinant DNA, which can be inserted into a plasmid or separated out on a gel.<BIG><B><SPAN style="color:#005C04 ;">RNA (Ribonucleic acid) </SPAN></B></BIG>
They are single standard biomolecules, presents in every living cells and is a basic unit for some viruses. Nucleotides acts as its basic units. <BIG><B><SPAN style="color:#005C04 ;">Serine proteases </SPAN></B></BIG>
Enzymes that cleave peptide bonds in proteins, which has serine at its active site. The reactivity of the serine residue is ensured by the vicinity of a histidine and an aspartate residue (catalytic triad), all three residues are required for the charge relay system to take place. In mammals, serine proteases perform many important functions, especially in digestion, blood clotting, and the complement system.<BIG><B><SPAN style="color:#005C04 ;">Shuttle vector</SPAN></B></BIG>
A shuttle vector is a vector (usually a plasmid) constructed in such a way that it can propagate in two different host species. Therefore, DNA inserted into a shuttle vector can be confirmed or manipulated in two different cell types. It has two origin of replication each of which is specific to a host. Due to its ability to replicate in two different hosts, it is also called as bifunctional hosts.<BIG><B><SPAN style="color:#005C04 ;">Site-directed mutagenesis (SDM) </SPAN></B></BIG>
A molecular biology method to create specific, targeted changes (mutations) in double stranded plasmid DNA. There are many reasons to make specific DNA alterations (insertions, deletions and substitutions), including:
<OL style=" list-style-type: disc; padding-left:1.5cm; padding-right:1.5cm;  align-content: center"><LI><SPAN>To study changes in protein activity that occur due to the change in DNA sequence.</SPAN></LI><LI><SPAN>To select or screen for mutations that have a desired property at the DNA, RNA or protein level.</SPAN></LI><LI><SPAN>To insert or remove restriction sites or tags.</SPAN></LI></OL></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><BIG><B><SPAN style="color:#005C04 ;">Stop Codon</SPAN></B></BIG>
In the genetic code, a <B>stop codon</B> (or termination codon) is a nucleotide triplet within messenger RNA that signals a termination of translation. Proteins are based on polypeptides, which are unique sequences of amino acids. UAG (&quot;amber&quot;), UAA (&quot;ochre&quot;) and UGA (&quot;opal&quot;) codes for stop codon.<BIG><B><SPAN style="color:#005C04 ;">Subtilisin E</SPAN></B></BIG>
Subtilisin is a non-specific, extracellular alkaline serine protease (a protein-digesting enzyme) initially obtained from <I>Bacillus subtilis</I>. It catalyzes the hydrolysis of proteins and peptide amides<BIG><B><SPAN style="color:#005C04 ;">Synthetic Biology</SPAN></B></BIG>
Synthetic biology can be defined as engineering in biology, in order to design and construct biological parts, pathways or systems, which are non-existing in nature and re-design the existing natural biological pathways or systems.<BIG><B><SPAN style="color:#005C04 ;">Template</SPAN></B></BIG>
A strand of DNA or RNA (mRNA) that specifies the base sequence of a newly synthesized strand of DNA or RNA, the two strands being complementary.<BIG><B><SPAN style="color:#005C04 ;">Vector</SPAN></B></BIG>
Artificially modified plasmids, which acts as a carrier to carry our gene of interest into a target cell to get the desired product through expression. <BIG><B><SPAN style="color:#005C04 ;">Wild type</SPAN></B></BIG>
A strain, gene, or characteristic which persists among individuals in natural conditions, as distinct from an uncommon mutant type. </P><H2 style="border-bottom: 5px solid #005b04; padding-left: 1.0cm;">References</H2><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="ref1glossary" class="anchor" style="color:Black">[1]<SPAN style="padding-left: 0.5cm;">P. P. Goswami, “Chemical biology: Photocages activated by green light,” <I>Nat. Methods</I>, vol. 12, no. 5, pp. 388–388, May 2015.</SPAN></A><A name="ref2glossary" class="anchor" style="color:Black">[2]<SPAN style="padding-left: 0.5cm;">Center for Biological Sequence Analysis, “CBS Data Sets”, 2004. [Online]. Available: http://www.cbs.dtu.dk/databases/propeptides. [Accessed: 19-Oct-2016]</SPAN></A></P></DIV></DIV><DIV class=" footer_area"><DIV class=" footer"><DIV class=" single_footer_container"><DIV class=" single_footer" style="float: left;"><P style="padding-left: 1cm;"><A href="http://igem.rwth-aachen.de/" style="text-decoration:none;color:#7fb983;">Link to our website</A><A href="https://2014.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki 2014 Aachen</A><A href="https://2015.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki  2015 Aachen</A><A href="#" style="text-decoration: none;color:#7fb983;">igem@rwth-aachen.de</A></P></DIV><DIV class=" single_footer" style="float: right;">                 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>