<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_area"><DIV class=" header"><DIV class="mainmenu floatright"><NAV class="main-navigation pull-right hidden-xs hidden-sm" id="pageHighlighter"><UL><LI><A href="https://2016.igem.org/Team:Aachen">Home</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description#characterization">Improvement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Glossary">Glossary</A></LI></UL><LI><A href="#" class="has-submenu" style="text-decoration:none;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Basic_Part">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Aachen/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab/Protocols">Protocols/Methods</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Beyond_Basics">Beyond the Basics</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supervisors">Supervisors</A></LI><LI><A href=" https://2016.igem.org/Team:Aachen/Attributions#attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supporters">Supporters</A></LI></NAV></DIV></DIV></DIV><P><TITLE>Welcome to iGEM Aachen 2016</TITLE></P><DIV class="main_content_area "><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Parts</H1></DIV><TABLE align="center" style="width:90%; border:2pt solid black;margin-left: auto; margin-right: auto;"><TBODY><TR><TD bgcolor="#7fb983"><B>BioBrick</B></TD><TD bgcolor="#7fb983"><B>Description</B></TD></TR><TR><TD><A href="#K2020000" style="color:#0000EE;">K2020000</A></TD><TD>subtilisin E gene, optimized for <I>E. coli</I> codon usage</TD></TR><TR><TD><A href="#K2020001" style="color:#0000EE;">K2020001</A></TD><TD>subtilisin E gene, optimized for <I>E. coli</I> codon usage, with leader sequence pelB </TD></TR><TR><TD><A href="#K2020002" style="color:#0000EE;">K2020002</A></TD><TD>expression system for subtilisin E in <I>E. coli</I></TD></TR><TR><TD><A href="#K2020003" style="color:#0000EE;">K2020003</A></TD><TD>mutated expression system for subtilisin E in <I>E. coli</I> (S221Y)</TD></TR><TR><TD><A href="#K2020004" style="color:#0000EE;">K2020004</A></TD><TD>mutated expression system for subtilisin E in <I>E. coli</I> (S221X)</TD></TR><TR><TD><A href="#K2020005" style="color:#0000EE;">K2020005</A></TD><TD>mutated expression system for subtilisin E in <I>E. coli</I> (Y77W)</TD></TR><TR><TD><A href="#K2020006" style="color:#0000EE;">K2020006</A></TD><TD>mutated expression system for subtilisin E in <I>E. coli</I> (Y77X)</TD></TR><TR><TD><A href="#K2020026" style="color:#0000EE;">K2020026</A></TD><TD>leader sequence MFalpha (different version of the biobrick <A href="http://parts.igem.org/Part:BBa_K792002"><U style="color:#0000EE;">K792002</U></A>)</TD></TR><TR><TD><A href="#K2020040" style="color:#0000EE;">K2020040</A></TD><TD>screening plasmid for incorporation of non-canonical amino acids → pRXG (<A href="http://parts.igem.org/Part:BBa_K1416004"><U style="color:#0000EE;">twin pFRY</U></A>)</TD></TR><TR><TD><A href="#K2020042" style="color:#0000EE;">K2020042</A></TD><TD>tRNA specific for tyrosine and UAG codon in <I>E. coli</I></TD></TR><TR><TD><A href="#K2020043" style="color:#0000EE;">K2020043</A></TD><TD>tRNA synthetase specific for the ncAA AzF and UAG codon in <I>E. coli</I> → AzF-RS</TD></TR><TR><TD><A href="#K2020045" style="color:#0000EE;">K2020045</A></TD><TD>tRNA synthetase specific for the ncAA NitroY and UAG codon in <I>E. coli</I> → NitroY-RS</TD></TR><TR><TD><A href="#K2020046" style="color:#0000EE;">K2020046</A></TD><TD>tRNA synthetase specific for the ncAA CNF and UAG codon in <I>E. coli</I> → CNF-RS</TD></TR><TR><TD><A href="#K2020050" style="color:#0000EE;">K2020050</A></TD><TD>tRNA synthetase specific for tyrosine and UAG codon in <I>E. coli</I> → Y-RS</TD></TR><TR><TD><A href="#K2020051" style="color:#0000EE;">K2020051</A></TD><TD>mutated tRNA synthetase specific for tyrosine and UAG codon in <I>E. coli</I> → Y-RS with Y32G</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020052</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 1</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020053</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 2</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020054</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 3</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020055</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 4</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020056</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 5</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020057</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 6</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020058</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 7</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020059</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 8</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020060</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 9</TD></TR><TR><TD><A href="#K2020052" style="color:#0000EE;">K2020061</A></TD><TD>tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 10</TD></TR></TBODY></TABLE></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="basic" class="anchor">Best New Basic Part</A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020042" class="anchor" style="color:#005C04 ;"><BIG><B>K2020042</B></BIG> - tRNA specific for tyrosine and UAG codon in <I>E. coli</I></A>
For incorporating non-canonical amino acids into a protein, an orthogonal tRNA/synthetase pair is needed, that does not crossreact with the cognate tRNA/synthetase pairs. This tRNA can be assembled with a variety of synthetases into a plasmid to incorporate ncAA in <I>E. coli</I> in response to an amber stop codon. 
This tRNA derives from the wild type tyrosyl <I>Methanococcus janaschii</I> tRNA/synthetase pair. It was proven to not crossreact with the cognate <I>E. coli</I> tRNA/synthetase pairs. The tRNA is used together with a tRNA synthetase. It has been proven to work with various synthetases for incorporation of ncAA by <A href="https://2014.igem.org/Team:Austin_Texas iGEM-Team Austin, Texas 2014"><U style="color:#0000EE;">iGEM Team Austin Texas 2014</U></A><OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI><A href="http://parts.igem.org/Part:BBa_K2020050"><U style="color:#0000EE;">Y-RS, canonical amino acid</U></A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1416000"><U style="color:#0000EE;">ONBY-RS</U></A></LI><LI><A href="http://parts.igem.org/Part:BBa_K2020043"><U style="color:#0000EE;">AzF-RS</U></A></LI><LI><A href="http://parts.igem.org/Part:BBa_K2020046"><U style="color:#0000EE;">CNF-RS</U></A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1416001"><U style="color:#0000EE;">IodoY-RS</U></A></LI><LI><SPAN>5HTP-RS</SPAN></LI><LI><A href="http://parts.igem.org/Part:BBa_K2020045"><U style="color:#0000EE;">NitroY-RS</U></A></LI><LI><SPAN>AminoY-RS</SPAN></LI></OL></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
Furthermore, <A href="https://2016.igem.org/Team:TU_Darmstadt iGEM-Team TU Darmstadt 2016"><U style="color:#0000EE;">iGEM Team TU Darmstadt</U></A> was working with this tRNA and the OMeY-synthetase.
Our Team is using the tRNA in this project to incorporate the canonical amino acid tyrosine with <A href="http://parts.igem.org/Part:BBa_K2020050"><U style="color:#0000EE;">Y-RS</U></A> in response to an amber stop codon as well as an ONBY with an <A href="http://parts.igem.org/Part:BBa_K1416000"><U style="color:#0000EE;">ONBY-RS</U></A> in <I>E. coli</I>. Furthermore, DMNBS synthetase variants undergo screening for incorporation efficiency and fidelity, which are done by using this BioBrick. These variants are proven to work with the tRNA and are all listed in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A>.

Incorporation of ncAA:
The tRNA contains an amber anticodon for incorporating the ncAA in response to a recoded amber termination codon. It has been used previously in an amberless <I>E. coli</I> strain C321.∆A.expb as well as in BL21 DE3 gold. When working with a recoded amber codon in BL21 DE3, the tRNA is competing with the release factor1 at the amber stop codon. As amber stop codons add up to only about 7% of total stop codos, amber suppression still works appropriately. An application of the tRNA is either the incorporation of the ncAA into a protein or usage with a reporter plasmid (for example <A href="http://parts.igem.org/Part:BBa_K2020040"><U style="color:#0000EE;">pFRY</U></A>) for probing ncAA tRNA/synthetase pair variants regarding efficiency and fidelity.

Assembly in a synthetase plasmid for incorporation of ncAA:<FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 1:  pACYC derived plasmid with tRNA and a cognate synthetase</B></FIGCAPTION></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
Most synthetases are used with low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator (fig. 1).

Elements of orthogonality:
<OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI><SPAN>C1-G72: most important element for orthogonality, recognized by Arg<SUP>174</SUP>, Arg<SUP>132</SUP>, Met<SUP>178</SUP>, Lys<SUP>175</SUP> within the synthetase </SPAN></LI><LI><SPAN>A73: recognized by Val<SUP>195</SUP></SPAN></LI><LI><SPAN>G71: recognized by Arg<SUP>132</SUP></SPAN></LI></OL></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
Recognition between tRNA and ncAA-synthetases:
The <I>Methanococcus janaschii</I> wild type tyrosyl tRNA consists of two arms: Firstly, the acceptor-minihelix, where the ncAA will be attached to the 3' end and secondly, the anticodon containg arm. Synthetases interact mainly with the acceptor minihelix of the tRNA. Due to the lack of a recognizing element within the anticodon-containing section, a mutation of an anticodon base has a comparatively small effect on the aminoacylation efficiency and may explain why a variety of ncAA can be incorporated with this tRNA.

Example measurement proving incorporation of amino acids:<CENTER><FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 2:  normalized fluorescence spectrum of mRFP1</B></FIGCAPTION></CENTER><CENTER><FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 3:  normalized fluorescence spectrum of sfGFP as a sign for successful amino acid incorporation via tRNA/synthetase pair for tyrosine via amber supression</B></FIGCAPTION></CENTER></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A href="http://parts.igem.org/Part:BBa_K2020050"><U style="color:#0000EE;">The wild type <I>Methanococcus janaschii</I> tRNA/synthetase pair</U></A> for incorporation of tyrosine at an amber termination codon (the pair containing the tRNA mentioned above) is co-transformed with <A href="http://parts.igem.org/Part:BBa_K2020040"><U style="color:#0000EE;">pFRY</U></A>, a flourescent reporter plasmid for measurement of incorporation of ncAA into BL21 DE3 gold, which is induced by 100 µM ITPG.
pFRY is one part of the two plasmid reporter system for measurement of incorporation efficiency of ncAA via amber supression. It consists of two fluorescent domains connected through a linker sequence containing an amber stop. When induced with IPTG, a the fluorescence can be measured. A red fluorescence is always visible upon induction, and if an amino acid is incorporated in response to the recoded amber stop codon, a green fluorescence intensity is also measurable. Fidelty and efficiency of the incorporation can be determined by comparison of fluorescent levels. 
This experiment is performed in order to obtain fluorescence spectra of mRFP1 and sfGFP. As shown above (fig. 3) GFP formation is measured as a result of successful amino acid incorporation via amber supression. 
Excitation and emission spectra of mRFP1 (fig. 2), and sfGFP (fig. 3) were obtained through measurements with a modified Biolector set-up.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">Registry page</U></A>.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="composite" class="anchor">Best New Composite Part</A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020002" class="anchor" style="color:#005C04;"><BIG><B>K2020002</B></BIG> – expression system for subtilisin E in <I>E. coli</I></A>
This expression system consists of the promoter <A href="http://parts.igem.org/Part:BBa_R0011"><U style="color:#0000EE;">BBa_R0011</U></A>, the ribosome binding site <A href="http://parts.igem.org/Part:BBa_B0034"><U style="color:#0000EE;">BBa_B0034</U></A>, the newly created BioBrick <A href="http://parts.igem.org/Part:BBa_K2020001"><U style="color:#0000EE;">K2020001</U></A> and the terminator <A href="http://parts.igem.org/Part:BBa_B0010"><U style="color:#0000EE;">BBa_B0010</U></A>. BioBrick <A href="http://parts.igem.org/Part:BBa_K2020001"><U style="color:#0000EE;">K2020001</U></A> is a composite part itself and includes the secretion tag pelB (<A href="http://parts.igem.org/Part:BBa_J32015"><U style="color:#0000EE;">BBa_J32015</U></A>) and a subtilisin E gene optimized for <I>Escherichia coli</I> codon usage (<A href="http://parts.igem.org/Part:BBa_K2020000"><U style="color:#0000EE;">BBa_K2020000</U></A>). Once introduced into <I>E. coli</I>, this BioBrick is able to produce subtilisin E, an alkaline serine protease, which non-specifically digests proteins, and simultaneously secrets the enzyme into the periplasm of the cell. Caused by the lacI regulated promoter <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A>, the expression system can be induced by addition of IPTG. 
With the iGEM promoter <A href="http://parts.igem.org/Part:BBa_R0011"><U style="color:#0000EE;">BBa_R0011</U></A>, which was integrated in our sequence at first, it was not possible to successfully express subtilisin E due to fatal mutations inside the expression system in all analyzed colonies. Either there have been single base deletions or insertions in the pro-peptide, which led to a frameshift of the whole protein, or a 23 base pair deletion in the promoter. Both types of mutations result in an incorrect expression system, so that an expression of the protease is impossible. Since the promoter <A href="http://parts.igem.org/Part:BBa_R0011"><U style="color:#0000EE;">BBa_R0011</U></A> is leaky and induces the expression even without addition of IPTG, it can be assumed that subtilisin E is toxic for <I>E. coli</I>. 
Hence, we exchanged the promoter against <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A>. For achieving this, we carried out a polymerase chain reaction (PCR) to extract everything but the promoter and the RBS and simultaneously extend the remaining DNA sequence with the pre-fix of iGEM. Afterwards, we assembled it with the BioBrick <A href="http://parts.igem.org/Part:BBa_J04500"><U style="color:#0000EE;">J04500</U></A> and in parallel cloned it into the vector pSB1C3 - by cutting RFP out of the BioBrick <A href="http://parts.igem.org/Part:BBa_J04450"><U style="color:#0000EE;">J04450</U></A>. The implemented BioBrick <A href="http://parts.igem.org/Part:BBa_J04500"><U style="color:#0000EE;">J04500</U></A> itself contains another IPTG inducible promoter (<A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A>) and the same RBS (<A href="http://parts.igem.org/Part:BBa_B0034"><U style="color:#0000EE;">BBa_B0034</U></A>). An expression with the newly integrated promoter <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A> led to a colony with the correct sequence in opposition to our trial of gaining a positive clone while working with the first promoter <A href="http://parts.igem.org/Part:BBa_R0011"><U style="color:#0000EE;">BBa_R0011</U></A>.
We continued our experiments by performing a skim milk assay on agar plates. Therefore, we poured LB skim milk agar plates containing IPTG and the needed antibiotic and streaked out the <I>E. coli</I> BL21 cells containing the plasmid with the expression system.<FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 4: Skim milk plates assay.</B> Cells containing the empty backbone (left) and cells containing the expression system for native subtilisin E (right) after incubation for 3 days at 30°C.</FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">
Comparing the clearance of the skim milk plates, a proteolytic activity could be proven for the cells containing the expression system for native subtilisin E. As a result, we concluded that within three days these cells are able to produce the native protease, which will then digest the skim milk in the agar plates, resulting in a clearance.
In conclusion, we were able to express subtilisin E in <I>E. coli</I> and to prove its proteolytic activity via skim milk assay.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020002"><U style="color:#0000EE;">Registry page</U></A>.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="collection" class="anchor">Parts Collection</A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020043" class="anchor" style="color:#005C04;"><BIG><B>K2020043</B></BIG> – tRNA synthetase specific for the ncAA AzF and UAG codon in <I>E. coli</I> → AzF-RS</A>
This is the 4-Azido-L-phenylalanine-synthetase to be used as an orthogonal tRNA synthetase in <I>E. coli</I>. It can be used in combination with the cognate tRNA <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">BBa_K2020042</U></A> to incorporate AzF in response to an amber stop codon. 

Assembly in a tRNA/synthetase plasmid for incorporation of ncAA:
Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020043"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020045" class="anchor" style="color:#005C04;"><BIG><B>K2020045</B></BIG> – tRNA synthetase specific for the ncAA NitroY and UAG codon in <I>E. coli</I> → NitroY-RS</A>
This is the 3-Nitro-L-tyrosine-synthetase to be used as an orthogonal tRNA synthetase in <I>E. coli</I>. It can be used in combination with the cognate tRNA <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">BBa_K2020042</U></A> to incorporate NitroY in response to an amber stop codon. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020045"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020046" class="anchor" style="color:#005C04;"><BIG><B>K2020046</B></BIG> – tRNA synthetase specific for the ncAA CNF and UAG codon in <I>E. coli</I> → CNF-RS</A>
This is the 4-Cyano-L-phenylalanine-synthetase to be used as an orthogonal tRNA synthetase in <I>E. coli</I>. It can be used in combination with the cognate tRNA <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">BBa_K2020042</U></A> to incorporate CNF in response to an amber stop codon. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020046"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020050" class="anchor" style="color:#005C04;"><BIG><B>K2020050</B></BIG> - tRNA synthetase specific for tyrosine and UAG codon in <I>E. coli</I> → Y-RS</A>
This is the wild type tyrosyl synthetase derived from <I>Methanococcus janaschii</I> to be used as an orthogonal tRNA synthetase in <I>E. coli</I>. It can be used in combination with the cognate tRNA <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">BBa_K2020042</U></A> to incorporate tyrosine in response to an amber stop codon. 

Incorporation of ncAA with amber codon supression:
Photo-cleavable non-canonical amino acids offer the opportunity to control protein function on a non-invasive basis. Working with non-canonical amino acids requires an additional, orthogonal pair of a tRNA and a cognate synthetase i.e. which does not crossreact with the endogenous tRNA/synthetase pairs. The tRNA's anticodon contains was mutated to an amber stop anticodon. Hence, it is possible to incorporate an amino acid at a chosen position in a protein via amber codon suppression. While working with amber codon supression, the <I>Methanococcus</I> wild type tyrosyl-tRNA synthetase is present even in minimal media.

Assembly in a synthetase plasmid for incorporation of (n)cAA:
Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator.

Recognition between tRNA and (n)cAA-synthetases derived from Mj:
The <I>Methanococcus janaschii</I> wild type tyrosyl tRNA consists of two arms: Firstly, the acceptor-minihelix, where the ncAA will be attached to the 3' end and secondly, the anticodon containg arm. Synthetases interact mainly with the acceptor minihelix of the tRNA. Due to the lack of a recognizing element within the anticodon-containing section, a mutation of an anticodon base has a comparatively small effect on the aminoacylation efficiency and may explain why a variety of ncAA can be incorporated with this tRNA. 

Example measurement proving incorporation of amino acids:
The wild type <I>Methanococcus janaschii</I> tRNA/synthetase pair for incorporation of tyrosine at an amber termination codon (the pair containing the tRNA mentioned above) is co-transformed with <A href="http://parts.igem.org/Part:BBa_K2020040"><U style="color:#0000EE;">pFRY</U></A>, a flourescent reporter plasmid for measurement of incorporation of ncAA into BL21 DE3 gold, which is induced by 100 µM ITPG.
pFRY is one part of the two plasmid reporter system for measurement of incorporation efficiency of ncAA via amber supression. It consists of two fluorescent domains connected through a linker sequence containing an amber stop. When induced with IPTG, a the fluorescence can be measured. A red fluorescence is always visible upon induction, and if an amino acid is incorporated in response to the recoded amber stop codon, a green fluorescence intensity is also measurable. Fidelty and efficiency of the incorporation can be determined by comparison of fluorescent levels. 
This experiment is performed in order to obtain fluorescence spectra of mRFP1 and sfGFP. GFP formation is measured as a result of successful amino acid incorporation via amber supression. 
Excitation and emission spectra of mRFP1, and sfGFP were obtained through measurements with a modified Biolector set-up. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020050"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020051" class="anchor" style="color:#005C04;"><BIG><B>K2020051</B></BIG> - mutated tRNA synthetase specific for tyrosine and UAG codon in <I>E. coli</I> → Y-RS with Y32G</A>
This is a DMNBS-synthetase to be used as an orthogonal tRNA synthetase in <I>E. coli</I>. It can be used in combination with the cognate tRNA <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">K2020042</U></A> to incorporate tyrosine in response to an amber stop codon. This synthetase has Y32 mutated to glycine and is therefore a template for further mutations for the purpose of changing amino acid specificity. 

Assembly in a synthetase plasmid for incorporation of ncAA:
Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020051"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020052" class="anchor" style="color:#005C04;"><BIG><B>K2020052 to K2020061</B></BIG> – tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 1 to 10</A>
All of these parts are variants of a DMNBS-synthetase to be used as an orthogonal synthetase in <I>E. coli</I>. Each of them can be used in combination with the cognate tRNA <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">BBa_K2020042</U></A> to incorporate DMNBS in response to an amber stop codon.</P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
Incorporation of ncAA:
Photo-cleavable non-canonical amino acids offer the opportunity to control protein function on a non-invasive basis. Working with non-canonical amino acids requires an additional, orthogonal pair of a tRNA and a cognate synthetase i.e. which does not crossreact with the endogenous tRNA/synthetase pairs. The tRNA's anticodon is mutated to an amber stop anticodon. Hence, it is possible to incorporate an amino acid at a chosen position in a protein via amber codon suppression.
A previously reported tRNA/synthetase pair for O-(4,5-dimethoxy-2-nitrobenzyl)-L-serine (DMNBS), which derived from <I>Escherichia coli</I> and was used in <I>Saccharomyces cerevisiae</I> leads to the lack of a possibility to work with non-canonical amino acids replacing serine in <I>E. coli</I> by using a 21st amino acid.
Based on computational modeling a tRNA synthetase for DMNBS in <I>E. coli</I> was designed by creating a semi rational mutation library. 

Assembly in a synthetase plasmid for incorporation of ncAA:
Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator. 

Host organism:
This reporter plasmid and the measurement of its corresponding proteins expressed are previously used both in an amberless <I>E. coli</I> strain and BL21 DE3 gold. The latter results in competition of the suppressor tRNA with RF1 at the amber stop codon.  

Screening Results:<FIGCAPTION style="text-align: center; font-size: 15px; "><B>Figure 5: DMNBS graph efficiency</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
If the levels of optical density of the tRNA synthetase to be evaluated and a working synthetase are equal, a first approximation of efficiency and fidelity can be made by normalizing GFP levels. Thus, one can eliminate the biogenic background fluorescence levels and compare the variants with each other. 
All details about these parts can be seen on their Registry pages as listed below:<OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020052"><U style="color:#0000EE;">BioBrick K2020052</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020053"><U style="color:#0000EE;">BioBrick K2020053</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020054"><U style="color:#0000EE;">BioBrick K2020054</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020055"><U style="color:#0000EE;">BioBrick K2020055</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020056"><U style="color:#0000EE;">BioBrick K2020056</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020057"><U style="color:#0000EE;">BioBrick K2020057</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020058"><U style="color:#0000EE;">BioBrick K2020058</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020059"><U style="color:#0000EE;">BioBrick K2020059</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020060"><U style="color:#0000EE;">BioBrick K2020060</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020061"><U style="color:#0000EE;">BioBrick K2020061</U></A></LI></OL></P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Expression in <I>E. coli</I></H1></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 0.8cm;">Basic Building Blocks</H2><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020000" class="anchor" style="color:#005C04;"><BIG><B>K2020000</B></BIG> – subtilisin E gene, optimized for <I>E. coli</I> codon usage</A>
The gene of this BioBrick can be used to express subtilisin E, which is an alkaline serine protease, which non-specifically digests proteins, in <I>Escherichia coli</I>. To use this part a suitable leader sequence has to be placed in front as the sequence of this BioBrick does not contain a start codon.
Its sequence was originally obtained from a wild type <I>Bacillus subtilis</I> but was codon optimized for <I>E. coli</I>.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020000"><U style="color:#0000EE;">Registry page</U></A> and in the section of the related BioBrick <A href="#K2020002" style="color:#0000EE;">K2020002</A>.<A name="K2020001" class="anchor" style="color:#005C04 ;"><BIG><B>K2020001</B></BIG> – subtilisin E gene, optimized for <I>E. coli</I> codon usage, with leader sequence pelB</A>
This composite part consists of the leader sequence pelB (<A href="http://parts.igem.org/Part:BBa_J32015"><U style="color:#0000EE;">BBa_J32015</U></A>) and the subtilisin E gene (<A href="http://parts.igem.org/Part:BBa_K2020000"><U style="color:#0000EE;">K2020000</U></A>) from our first BioBrick. It can be used to express subtilisin E in <I>Escherichia coli</I> and simultaneously secret the enzyme into the periplasm of the cell. Subtilisin E is an alkaline serine protease, which non-specifically digests proteins.
To generate a functional coding sequence that can be expressed in <I>E. coli</I> the leader sequence pelB, which begins with a start codon, was placed in front of the subtilisin E gene.
Caused to subtilisin E’s partly toxicity for <I>E. coli</I>, this BioBrick should be cloned into an expression system with an inducible promoter. This should hinder the organism in the expression of the protease during its growing period. With the iGEM promoter <A href="http://parts.igem.org/Part:BBa_R0011"><U style="color:#0000EE;">BBa_R0011</U></A> it was not possible to successfully express subtilisin E. Hence, we exchange the promoter against <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A>.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020001"><U style="color:#0000EE;">Registry page</U></A> and in the section of the related BioBrick <A href="#K2020002" style="color:#0000EE;">K2020002</A>.<SPAN style="color:#005C04;"><BIG><B>K2020002</B></BIG> – expression system for subtilisin E in <I>E. coli</I></SPAN>
Details about this part can be seen above in the section <A href="#composite" style="color:#0000EE;">Best New Composite Part</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020002"><U style="color:#0000EE;">Registry page</U></A>.</P><H2 style="border-bottom: 5px solid #005b04;padding-left: 0.8cm;">Mutated Versions</H2><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020003" class="anchor" style="color:#005C04 ;"><BIG><B>K2020003</B></BIG> – mutated expression system for subtilisin E in <I>E. coli</I> (S221Y)</A>
The BioBrick named <A href="#K2020002" style="color:#0000EE;">K2020002</A> for the expression of subtilisin E in <I>E. coli</I> is the basic component of this new part. Therefore, it consists of the promoter <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A>, the ribosome binding site <A href="http://parts.igem.org/Part:BBa_B0034"><U style="color:#0000EE;">BBa_B0034</U></A>, the leader sequence pelB <A href="http://parts.igem.org/Part:BBa_J32015"><U style="color:#0000EE;">BBa_J32015</U></A>, the newly created BioBrick <A href="http://parts.igem.org/Part:BBa_K2020000"><U style="color:#0000EE;">K2020000</U></A> and the terminator <A href="http://parts.igem.org/Part:BBa_B0010"><U style="color:#0000EE;">BBa_B0010</U></A> like the expression system itself. The whole expression is based on the usage in <I>E. coli</I>, so the sequence of the subtilisin E gene from a wild type <I>Bacillus subtilis</I> was optimized for <I>E. coli</I> codon usage. 
The sequence was partly ordered from IDT (<A href="http://parts.igem.org/Part:BBa_K2020001"><U style="color:#0000EE;">BBa_K2020001</U></A> + <A href="http://parts.igem.org/Part:BBa_B0010"><U style="color:#0000EE;">BBa_B0010</U></A>) and then cloned into <A href="http://parts.igem.org/Part:BBa_J04500"><U style="color:#0000EE;">BBa_J04500</U></A>, a protein expression backbone, which already carries the LacI promoter <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A> and the ribosome binding site <A href="http://parts.igem.org/Part:BBa_B0034"><U style="color:#0000EE;">BBa_B0034</U></A>. Afterwards, a mutation in the active site of the enzyme was introduced by performing site-directed mutagenesis (SDM). The codon AGC of serine<SUP>221</SUP> was substituted with TAC, which codes for tyrosine, so serine was exchanged against tyrosine in the catalytic triade of the enzyme.
We were not able to exactly detect the expression of the modified proteases via SDS gel, so we proceeded by executing a skim milk assay on agar plates containing IPTG and the needed antibiotics. Therefore, we streaked out the cells containing the modified expression systems on these plates and incubated at 30°C for three days.<FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 6: Skim milk plates assay.</B> Cells containing the empty backbone (1) in comparison to either cells producing the native protease (2), cells with the SDM 1-modified (3) and the SDM 3-modified expression system (4) after incubation for 3 days at 30°C.</FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">
Neither the empty backbone nor the SDM 1 modified expression system did seem to cause a proteolytic activity. A clearance and therefore a proteolytic activity could only be observed for the native protease (as demonstrated in the section of BioBrick <A href="#K2020002" style="color:#0000EE;">K2020002</A>).
By demonstrating that this modification doesn’t result in a clearance of the skim milk plates, we were now able to prove that serine is essential for the proteolytic activity of the protease and that exchanging it would inactivate the enzyme. Hence, we demonstrated that exchanging serine against a photo-labile, non-canonical amino acid will inactivate subtilisin E and therefore proved the principle of our project. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020003"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020004" class="anchor" style="color:#005C04 ;"><BIG><B>K2020004</B></BIG> – mutated expression system for subtilisin E in <I>E. coli</I> (S221X)</A>
This part is a variation of the BioBrick <A href="#K2020003" style="color:#0000EE;">K2020003</A>. In opposition to the exchange of serine<SUP>221</SUP> against tyrosine, the named serine was exchanged against an amber codon, the least used stop codon in <I>Escherichia coli</I>. The codon AGC of serine<SUP>221</SUP> was substituted with TAG, which codes for DMNB-serine, if the corresponding aminoacyl tRNA/synthetase pair is added. So serine will be exchanged against DMNBS in the catalytic triade of the enzyme in the presence of the needed DMNBS tRNA/synthetase pair. By irradiation with light, this non-canonical amino acid can be converted to natural serine in the active site of the enzyme and the cleaved off photo-labile group. Thereby, the protease will reach its activity just by shining light on it and will be reversibly inactivated successfully.
Unfortunately, we were not able to execute all of the experiments that we planned due to a lack of time. But as our <A href="#K2020003" style="color:#0000EE;">proof of principle</A> of exchanging serine against a larger amino acid namely tyrosine worked, it can be assumed, that this part will also operate correctly. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020004"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020005" class="anchor" style="color:#005C04 ;"><BIG><B>K2020005</B></BIG> – mutated expression system for subtilisin E in <I>E. coli</I> (Y77W)</A>
The BioBrick named <A href="#K2020002" style="color:#0000EE;">K2020002</A> for the expression of subtilisin E in <I>E. coli</I> is the basic component of this new part. Therefore, it consists of the promoter <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A>, the ribosome binding site <A href="http://parts.igem.org/Part:BBa_B0034"><U style="color:#0000EE;">BBa_B0034</U></A>, the leader sequence pelB <A href="http://parts.igem.org/Part:BBa_J32015"><U style="color:#0000EE;">BBa_J32015</U></A>, the newly created BioBrick <A href="http://parts.igem.org/Part:BBa_K2020000"><U style="color:#0000EE;">K2020000</U></A> and the terminator <A href="http://parts.igem.org/Part:BBa_B0010"><U style="color:#0000EE;">BBa_B0010</U></A> like the expression system itself. The whole expression is based on the usage in <I>E. coli</I>, so the sequence of the subtilisin E gene from a wild type <I>Bacillus subtilis</I> was optimized for <I>E. coli</I> codon usage. 
The sequence was partly ordered from IDT (<A href="http://parts.igem.org/Part:BBa_K2020001"><U style="color:#0000EE;">BBa_K2020001</U></A> + <A href="http://parts.igem.org/Part:BBa_B0010"><U style="color:#0000EE;">BBa_B0010</U></A>) and then cloned into <A href="http://parts.igem.org/Part:BBa_J04500"><U style="color:#0000EE;">BBa_J04500</U></A>, a protein expression backbone which already carries the LacI promoter <A href="http://parts.igem.org/Part:BBa_R0010"><U style="color:#0000EE;">BBa_R0010</U></A> and the ribosome binding site <A href="http://parts.igem.org/Part:BBa_B0034"><U style="color:#0000EE;">BBa_B0034</U></A>. Afterwards, a mutation between the pro-peptide and the subtilisin E gene was introduced by performing site-directed mutagenesis. The codon TAT of tyrosine<SUP>77</SUP> was substituted with TGG, which codes for tryptophan, so tyrosine was exchanged against tryptophan at the connection site of pro-peptide and subtilisin E gene. Caused by the mutation in the pro-peptide and not in the direct subtilisin E gene, the actual position of the modified tyrosine was counted from the beginning of the pro-peptide compared to common annotations counted from the N-terminus of the actual enzyme.
We were not able to exactly detect the expression of the modified proteases via SDS gel, so we proceeded by executing a skim milk assay on agar plates containing IPTG and the needed antibiotics. Therefore, we streaked out the cells containing the modified expression systems on these plates and incubated at 30°C for three days.<FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 6: Skim milk plates assay.</B> Cells containing the empty backbone (1) in comparison to either cells producing the native protease (2), cells with the SDM 1-modified (3) and the SDM 3-modified expression system (4) after incubation for 3 days at 30°C.</FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">
The empty backbone didn't cause a proteolytic activity. A clearance and therefore a proteolytic activity could be observed for the native protease (as demonstrated in the section of BioBrick <A href="#K2020002" style="color:#0000EE;">K2020002</A>) but also for the SDM 3 modified expression system.
As SDM 3 had been executed to exchange tyrosine in the pro-peptide cleavage site against tryptophan, a proteolytic activity could be assumed caused by the clearance observed. Contrary to our former beliefs, it could now be deduced that exchanging tyrosine doesn’t result in a change of activity. Consequently, tyrosine in the pro-peptide cleavage site is not essential for the activity of subtilisin E.
Unfortunately, we were not able to prove the principle of our project regarding tyrosine in the pro-peptide cleavage site, as it seems to be not essential for the proteolytic activity. Thus, exchanging tyrosine against a photo-labile, non-canonical amino acid more precisely ONB-tyrosine will not influence the activity of the enzyme. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020005"><U style="color:#0000EE;">Registry page</U></A>.<A name="K2020006" class="anchor" style="color:#005C04 ;"><BIG><B>K2020006</B></BIG> – mutated expression system for subtilisin E in <I>E. coli</I> (Y77X)</A>
This part is a variation of the BioBrick <A href="#K2020005" style="color:#0000EE;">K2020005</A>. In opposition to the exchange of tyrosine<SUP>77</SUP> against tryptophan, the named tyrosine was exchanged against an amber codon, the least used stop codon in <I>Escherichia coli</I>. The codon TAT of tyrosine<SUP>77</SUP> was substituted with TAG, which codes for ONB-tyrosine, if the corresponding tRNA/synthetase pair is added. So tyrosine will be exchanged against ONBY in the pro-peptide cleavage site of the enzyme in the presence of the needed ONBY tRNA/synthetase pair. By irradiation with light, this non-canonical amino acid can be converted to natural tyrosine in the pro-peptide cleavage site of the enzyme and the cleaved off photo-labile group. Thereby, the protease will reach its activity just by shining light on it. Caused by the mutation in the pro-peptide and not in the direct subtilisin E gene, the actual position of the modified tyrosine was counted from the beginning of the pro-peptide compared to common annotations counted from the N-terminus of the actual enzyme. 
Unfortunately, we were not able to execute all of the experiments that we planned due to a lack of time. But as our <A href="#K2020005" style="color:#0000EE;">proof of principle</A> of exchanging tyrosine against a larger amino acid namely tryptophan showed no difference in the activity compared to our working subtilisin E <A href="#K2020002" style="color:#0000EE;">expression system</A>, tyrosine<SUP>77</SUP> in general not influences the proteolytic activity of the enzyme. And the replacement of tyrosine against ONBY will not cause an inactivation of the protease.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020006"><U style="color:#0000EE;">Registry page</U></A>.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Expression in <I>S. cerevisiae</I></H1></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 0.8cm;">Improvement of an Existing Part</H2><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020026" class="anchor" style="color:#005C04 ;"><BIG><B>K2020026</B></BIG> – leader sequence MFalpha (different version of the biobrick K792002)</A>
This part is a different version of the leader sequence MFalpha (<A href="http://parts.igem.org/Part:BBa_K792002"><U style="color:#0000EE;">BBa_K792002</U></A>) for <I>Saccharomyces cerevisiae</I>. It can be cloned directly in front of the protein, which will be secreted. It will be relocated into the medium and the tag will be cleaved off during this process. This alpha mating factors secretion tag is naturally occurring in the genome of <I>S. cerevisiae</I>. Futhermore, the sequence contained an illegal restriction site, which was corrected via PCR by us.
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020026"><U style="color:#0000EE;">Registry page</U></A>.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Evolution of a New Synthetase</H1></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 0.8cm;">Screening System</H2><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="K2020040" class="anchor" style="color:#005C04 ;"><BIG><B>K2020040</B></BIG> – screening plasmid for incorporation of non-canonical amino acids → pRXG</A>
This part is an improvement of the existing BioBrick <A href="http://parts.igem.org/Part:BBa_K1416004"><U style="color:#0000EE;">K1416004</U></A>, called pFRY by the iGEM Team Austin Texas 2014. It is an flourescent reporter for measurement of incorporation of ncAA.
pRXG is one part of a two plasmid reporter system for measurement of incorporation of ncAA via amber supression. It consists of two flourescent domains connected through a linker sequence containing and amber stop. When IPTG induced and expressed, the flourescence intensity can be measured. A red flourescence is always visible upon induction, and if an amino acid is incorporated as response to the amber stop codon, then a green flourescence intensity is measurable. Fidelty and efficiacy of the incorporation can be determined with comparison of flourescent level.
This reporter plasmid is one of a two plasmids containing screening system for determining efficiancy and fidelty of non-canonical amino acids (ncAA) incorporation via amber termination supression. One plasmid contains tRNA and corresponding aminoacylation-synthetase. The other one is this plasmid presented herein. It consists of am mRFP1 domain which is connected through a linker sequence containing a recoded amber stop codon with a sfGFP domain. The expression of the plasmid gives either a red fluorescence, or - if the ncAA will be incorporated at the recoded amber stop codon within the linker site - both a red and a green fluorescence.
Synthetase and tRNA are constitutivly expressed in a plasmid with low copy replicon, taking into account that expression results in metabolic stress, but are not under IPTG control for he purpose of avoiding abrupt and unpredictable effects considering extra time and energy needed for their assembly. Whereas the reporter plasmid containig two fluorescence proteins, is kept under operon control for IPTG induction likewise on a low copy plasmid with ColE1.<FIGCAPTION style="text-align: center; font-size: 15px; "><B>Figure 7: reporter plasmid pRXG</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
This reporter plasmid and the corresponding measurement of protein formation is previously used in both an amberless E.coli strain and BL21 DE3 gold. The use of the latter is resulting in competion of the supressor tRNA with release factor one at the amber stop codon at the usual 321 amber stop codons. 
Most synthetases are used with low copy plasmids (e.g. pACYC). The reporter should also be used with a low copy replicon. Assemble the part provided here into a low copy plasmid, make sure to use replicons from different incompatibility groups, eg. ColE1 and p15A and different selection markers. Keep under operon control for induction. 
All details about this part can be seen on its <A href="http://parts.igem.org/Part:BBa_K2020040"><U style="color:#0000EE;">Registry page</U></A>.</P><H2 style="border-bottom: 5px solid #005b04;padding-left: 0.8cm;">tRNA and Synthetases</H2><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><SPAN style="color:#005C04;"><BIG><B>K2020042</B></BIG> – tRNA specific for tyrosine and UAG codon in <I>E. coli</I></SPAN>
Details about this part can be seen above in the section <A href="#basic" style="color:#0000EE;">Best Basic Part</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020042"><U style="color:#0000EE;">Registry page</U></A>.<SPAN style="color:#005C04;"><BIG><B>K2020043</B></BIG> – tRNA synthetase specific for the ncAA AzF and UAG codon in <I>E. coli</I> → AzF-RS</SPAN>
Details about this part can be seen above in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020043"><U style="color:#0000EE;">Registry page</U></A>.<SPAN style="color:#005C04;"><BIG><B>K2020045</B></BIG> – tRNA synthetase specific for the ncAA NitroY and UAG codon in <I>E. coli</I> → NitroY-RS</SPAN>
Details about this part can be seen above in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020045"><U style="color:#0000EE;">Registry page</U></A>.<SPAN style="color:#005C04;"><BIG><B>K2020046</B></BIG> – tRNA synthetase specific for the ncAA CNF and UAG codon in <I>E. coli</I> → CNF-RS</SPAN>
Details about this part can be seen above in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020046"><U style="color:#0000EE;">Registry page</U></A>.<SPAN style="color:#005C04;"><BIG><B>K2020050</B></BIG> – tRNA synthetase specific for tyrosine and UAG codon in <I>E. coli</I> → Y-RS</SPAN>
Details about this part can be seen above in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020050"><U style="color:#0000EE;">Registry page</U></A>.<SPAN style="color:#005C04;"><BIG><B>K2020051</B></BIG> – mutated tRNA synthetase specific for tyrosine and UAG codon in <I>E. coli</I> → Y-RS with Y32G</SPAN>
Details about this part can be seen above in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A> and on its <A href="http://parts.igem.org/Part:BBa_K2020051"><U style="color:#0000EE;">Registry page</U></A>.<SPAN style="color:#005C04;"><BIG><B>K2020052 to K2020061</B></BIG> - tRNA synthetase specific for DMNB-serine and UAG codon in <I>E. coli</I> → version 1 to 10</SPAN>
Details about these parts can be seen above in the section <A href="#collection" style="color:#0000EE;">Parts Collection</A> and on their Registry pages as listed below:<OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020052"><U style="color:#0000EE;">BioBrick K2020052</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020053"><U style="color:#0000EE;">BioBrick K2020053</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020054"><U style="color:#0000EE;">BioBrick K2020054</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020055"><U style="color:#0000EE;">BioBrick K2020055</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020056"><U style="color:#0000EE;">BioBrick K2020056</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020057"><U style="color:#0000EE;">BioBrick K2020057</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020058"><U style="color:#0000EE;">BioBrick K2020058</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020059"><U style="color:#0000EE;">BioBrick K2020059</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020060"><U style="color:#0000EE;">BioBrick K2020060</U></A></LI><LI style="padding-right: 0.3cm;"><A href="http://parts.igem.org/Part:BBa_K2020061"><U style="color:#0000EE;">BioBrick K2020061</U></A></LI></OL></P></DIV></DIV><DIV class=" footer_area"><DIV class=" footer"><DIV class=" single_footer_container"><DIV class=" single_footer" style="float: left;"><P style="padding-left: 1cm;"><A href="http://igem.rwth-aachen.de/" style="text-decoration:none;color:#7fb983;">Link to our website</A><A href="https://2014.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki 2014 Aachen</A><A href="https://2015.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki  2015 Aachen</A><A href="#" style="text-decoration: none;color:#7fb983;">igem@rwth-aachen.de</A></P></DIV><DIV class=" single_footer" style="float: right;">                 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>