<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_area"><DIV class=" header"><DIV class="mainmenu floatright"><NAV class="main-navigation pull-right hidden-xs hidden-sm" id="pageHighlighter"><UL><LI><A href="https://2016.igem.org/Team:Aachen">Home</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description#characterization">Improvement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Glossary">Glossary</A></LI></UL><LI><A href="#" class="has-submenu" style="text-decoration:none;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Basic_Part">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Aachen/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab/Protocols">Protocols/Methods</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Beyond_Basics">Beyond the Basics</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supervisors">Supervisors</A></LI><LI><A href=" https://2016.igem.org/Team:Aachen/Attributions#attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supporters">Supporters</A></LI></NAV></DIV></DIV></DIV><P><TITLE>Welcome to iGEM Aachen 2016</TITLE></P><DIV class="main_content_area "><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Project Idea</H1></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Motivation</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">The aim of this year’s project is to replace a harmful ingredient from laundry detergents with the help of synthetic biology. 
Modern detergent formulations contain a variety of enzymes to target stains specifically and efficiently. This helps to significantly reduce the amount of surfactants needed. 
Laundry detergents contain proteases, whose long-time storage is especially problematic due to self-degradation. Later they even degrade other useful enzymes in the final mixture. Thus, they are inhibited by adding an equal mass of boric acid. This chemical has been classified as a <B>“substance of very high concern”</B> by the European Chemicals Agency, as studies indicate its reproductive toxicity and teratogenicity  [<A href="#ref1PI"><U style="color:#0000EE; ">1</U></A>]. Consequently, many attempts have been made to replace it.  But up to now the approaches were either not efficient enough or had severe disadvantages regarding safety. 

In 2014, 595 000 tons of laundry detergents were  consumed solely within Germany  [<A href="#ref2PI"><U style="color:#0000EE; ">2</U></A>]. 
Boric acid is naturally present in drinking water and the environment in traces, but use of washing detergents makes its amount exceed the natural occurrence [<A href="#ref3PI"><U style="color:#0000EE; ">3</U></A>]. We want to avoid the usage of boric acid, to stop its influx into the water cycle, because of the continuously high amount of washing detergents consumed. 

The method we consider perfect for this challenging task is photocaging of enzymes.  
It allows activation through light waves, which are the fastest and most convenient way to get an activation signal to its destination. The method of photocaging is conceptually simple and in theory, applicable for every protein. Conveniently, this would also render the handling of tons of chemicals for enzyme stabilization unnecessary. So far though, the method has only been used in intracellular studies for investigation of enzyme function [<A href="#ref4PI"><U style="color:#0000EE; ">4-6</U></A>].
With this project we hope to demonstrate the versatility of photocaging for industrially applicable enzymes by example go laundry detergents proteases. </P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Theory</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">

Our aim is to produce a photocaged protease for washing detergents. Photocaging allows expressing a protease, inhibited by only one covalently bound molecule. Directly prior to the washing process, it could be activated with light by cleaving off this molecule. 
The key of this method is attaching a chemical protection group to a normally reactive molecule which is thereby shielded from reacting and can be cleaved off when irradiated with the respective spectrum of light. By replacing an amino acid which is crucial to the function of the enzyme with the corresponding photocaged amino acid, a protected enzyme would be generated. Exposure to the correct spectrum of light would lead to cleavage of the protection group and yield the original amino acid and therefore a functional enzyme. 
The introduction of this non-canonical amino acid can be achieved via genetic code expansion. The host organism is provided with an orthogonal tRNA/synthetase pair that is capable of incorporating the non-canonical amino acid of choice at a UAG codon. The UAG or “amber” codon is the least used stop codon in most organisms which makes it perfect for being reassigned. [<A href="#ref4PI"><U style="color:#0000EE; ">4</U></A>] 
More and more orthogonal tRNA/synthetase pairs are being created, but researchers are limited to available ones, if time does not allow the development of the synthetase that meets one’s special needs. As this is the case for our project, the following strategies have been developed to achieve reversible inactivation. 
</P><TABLE width="80%" align="center" style="border-collapse:separate;border:0px;border-spacing:40px;margin-right:20px; margin-left:80px;"><TBODY><TR><TD><FIGCAPTION style="text-align:center; font-size: 15px; "><B><I>S. cerevisiae</I></B></FIGCAPTION></TD><TD><FIGCAPTION style="text-align: center; font-size:15px; "><B><I>E. coli</I></B></FIGCAPTION></TD><TD><FIGCAPTION style="text-align: center; font-size:15px; "><B>Synthetase</B></FIGCAPTION></TD><TD><FIGCAPTION style="text-align: center; font-size:15px; "><B>Hardware</B></FIGCAPTION></TD></TR></TBODY></TABLE></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="scerevisiae" class="anchor">Photocaging - Targeting Serine in <I>S. cerevisiae</I></A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
Subtilisin is a protease from <I>Bacillus subtilis</I> whose derivatives are being used in many washing detergents, which makes it suitable for a proof of principle. 
Like a serine protease it catalyzes the cleavage of a peptide bond by means of a catalytic triad consisting of serine<SUP>221</SUP>, aspartic acid<SUP>32</SUP> and histidine<SUP>64</SUP>. [<A href="#ref7PI"><U style="color:#0000EE; ">7</U></A>]<FIGCAPTION style="text-align: center; font-size:15px; "><B>Figure 1: Mechanism of reaction catalyzed by the serine protease subtilisin E</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
Previous studies proved, that replacing the serine with an alanine, which equals the loss of the reactive group, reduces the activity by factors of 2∙10<SUP>6</SUP>. As the protection group would shield the hydroxyl group the inhibition should be at least equally effective.
An orthogonal tRNA/synthetase pair which is able to introduce the photo-liable serine derivative O-(4,5-dimethoxy-2-nitrobenzyl)-L-serine (DMNB-serine) at an amber stop codon, exists in <I>Saccharomyces cerevisiae</I>. It was developed and successfully used by P. G. Schultz [<A href="#ref5PI"><U style="color:#0000EE; ">5</U></A>]. 
DMNB-serine is converted to serine by irradiation with low energy blue light.
The first challenge is the production of subtilisin E in <I>S. cerevisiae</I>, thus, an expression system has to be built. We chose to try three different approaches to compare efficiency of production: A plasmid with a constitutive promotor, one with an inducible and an inducible genome integration vector. Then, by providing the yeast cells with the tRNA/synthetase pair for DMNB-serine, the respective non-canonical amino acid can be incorporated. 
Mutation of the codon for serine<SUP>221</SUP> to an amber stop codon in the subtilisin gene should lead to production of a photocaged and consequently inactive variant. 
Also the naturally occurring secretion tag from <I>B. subtilis</I>, that allows relocation of the enzyme to the medium shall be exchanged with a secretion tag working in <I>S. cerevisiae</I>, to make harvesting more convenient. 
As previous efforts to express a protease from <I>B. subtilis</I> in <I>S. cerevisiae</I> failed due to hyperglycosylation, [<A href="#ref8PI"><U style="color:#0000EE; ">8</U></A>] which resulted in an inactive enzyme, we also pursued a second strategy. Thus the idea to develop a DMNBS-synthetase that works in <I>Escherichia coli</I> evolved. </P><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="ecoli" class="anchor"> Photocaging - Targeting Tyrosine in <I>E. coli</I></A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><I>Escherichia coli</I> is widely used in synthetic biology. It offers the advantage of being a comparatively simple and well-understood model organism while being easy to handle in the laboratory environment. Also, an expansion of the genetic code has already been successfully implemented in <I>E. coli</I> multiple times [<A href="#ref9PI"><U style="color:#0000EE; ">9-12</U></A>] by introducing an orthogonal tRNA/synthetase pair.
Therefore, working with <I>E. coli</I> is an obvious choice.

Due to a limited range of tRNA/synthetase pairs for non-canonical amino acids in general and especially for those that act orthogonally in <I>E. coli</I>, photocaging serine in the active site of subtilisin E with DMNB-serine is currently not possible. Hence, another strategy is needed to produce temporarily inactive proteases. This part of the project focuses on utilizing the maturation process of subtilisin E.</P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
Subtilisin E is an alkaline serine protease found in <I>Bacillus subtilis</I> that has to autoprocess itself to become functional. At first, the enzyme exists as a precursor, namely the pre-pro-subtilisin. The pre-sequence serves as a recognition sequence for secretion across the cytoplasmic membrane and is cleaved off in the course of the process. The pro-peptide acts as an intramolecular chaperone (IMC) and facilitates the folding of the protease. Folding is essential for the activity of an enzyme. Still, the maturation process of Subtilisin E is not completed, as the pro-peptide covers the substrate binding site and inhibits activity. However, enough proteolytic activity is achieved to autoprocess the IMC-domain and therefore cleave off the pro-peptide. Yet, the C-terminal end of the pro-peptide continues to block the substrate binding site. After the degradation of the pro-peptide, the substrate-binding site is cleared and the protease becomes effectively active. [<A href="#ref13PI"><U style="color:#0000EE;">13</U></A>]
</P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;margin-bottom: 0px">This mechanism can be used to implement a novel inactivation method. 
O-(2-Nitrobenzyl)-L-tyrosine (ONBY) is a derivative of the canonical amino acid tyrosine. It carries a photo-labile protection group that can be cleaved off by irradiation with UV-light (365nm, [<A href="#ref14PI"><U style="color:#0000EE; ">14</U></A>]).<FIGCAPTION style="text-align: center; font-size:15px; "><B>Figure 2: structural formula of O-(2-Nitrobenzyl)-L-tyrosine and its cleavage reaction.</B></FIGCAPTION></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
By adding ONBY to the genetic code of <I>E. coli</I> and incorporating said amino acid in the pro-peptide cleavage-site of subtilisin E the maturation process is disturbed. Due to its size ONBY sterically hinders the protease [<A href="#ref15PI"><U style="color:#0000EE; ">15</U></A>]. The pro-peptide cannot be cleaved from the enzyme and subtilisin E is not able to achieve its full proteolytic activity. A temporarily inactive protease is produced.
After removal of the protection group the maturation process can be completed and subtilisin E acquires its full proteolytic activity.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="synthetase" class="anchor">Synthetase - Incorporating DMNBS in <I>E. coli</I></A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><B><SPAN style="color:#005C04;">Introduction</SPAN></B>
Photocleavable non-canonical amino acids offer the opportunity to control protein function on a non-invasive basis. Working with non-canonical amino acids requires an additional, orthogonal pair of a tRNA and a cognate synthetase which does not crossreact with the endogenous tRNA/synthetase pairs [<A href="#ref5PI"><U style="color:#0000EE; ">5</U></A>]. 
A previously reported tRNA/synthetase pair for O-(4,5-dimethoxy-2-nitrobenzyl)-L-serine (DMNBS) derived from <I>Escherichia coli</I> and was used in <I>Saccharomyces cerevisiae</I> [<A href="#ref5PI"><U style="color:#0000EE; ">5</U></A>]. However, this tRNA/synthetase pair can not be used in <I>E. coli</I> due to orthogonality.
Nevertheless, the shape of the binding pocket which is adapted to this specific non-canonical serine can be used as  template to create an orthogonal tRNA/synthetase pair for the use in <I>E. coli</I>
based on computational modeling, a synthetase for DMNBS in <I>E. coli</I> is designed, mainly because of three reasons:
<OL type="1" style=" padding-left:1cm; padding-right:1cm; "><LI><SPAN>Serine is a crucial amino acid for enzyme catalysis and therefore an obvious target for pathway analysis and control of protein activity via photocaging [<A href="#ref5PI"><U style="color:#0000EE; ">5</U></A>]</SPAN></LI><LI><SPAN><I>E. coli</I> is the most used organism in the field of biotechnology.</SPAN></LI><LI><SPAN>DMNBS has been proven to work very well as a non-canonical amino acid in yeast and is superior to oNBY regarding the physicochemical properties, e.g. a higher quantum yield for photocleavage and an absorption wavelength apart from the UV spectrum. [<A href="#ref5PI"><U style="color:#0000EE; ">5</U></A>]</SPAN></LI></OL></P><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;">
For that purpose, the well described tyrosyl-tRNA/synthetase pair of <I>Methanococcus janaschii</I> is mutated to selectively incorporate DMNBS into proteins in <I>E. coli</I>. 
The cognate tRNA's anticodon contains an amber stop anticodon. Hence, it is possible to incorporate an amino acid at a chosen position in a protein via amber codon suppression.

<B><SPAN style="color:#005C04;">Design of DMNBS-tRNA/synthetase pair</SPAN></B><FIGCAPTION style="text-align: center; font-size:15px; "><B>Figure 4: tyrosine binding pocket of wild type tyrosyl synthetase</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">

As the basis for designing a DMNBS specific synthetase the wild type tyrosyl-synthetase of <I>M. janaschii</I> (Mj-Tyr-RS) is chosen due to its well characterized properties, its resemblance to the <I>E. coli</I>-leucyl-synthetase (Ec-Leu-RS) derived DMNBS-synthetase used in yeast and its availability through the <A href="https://2014.igem.org/Team:Austin_Texas"><U style="color:#0000EE; ">iGEM-Team Austin, Texas, 2014</U></A>.
It was also previously shown, that this pair is orthogonal to endogenous <I>E. coli</I> synthetases [<A href="#ref16PI"><U style="color:#0000EE; ">16</U></A>].

The determination of the alterations to be made for changing the amino acid specificity are based on the comparison of the crystal structures of Mj-tyrosyl- and Ec-Leu-tRNA/synthetase pairs. Highly conserved amino acids Y<SUP>32</SUP>, L<SUP>65</SUP> and D<SUP>158</SUP> are known to build H-bonds with the natural ligand tyrosine [<A href="#ref17PI"><U style="color:#0000EE; ">17</U></A>]. Furthermore, these sites resemble the mutated sites within <I>E. coli</I>-leucyl-synthetase which was already changed to a DMNBS-specificity and is successfully used in yeast to incorporate DMNBS [<A href="#ref16PI"><U style="color:#0000EE; ">16,18</U></A>]. Some more positions are chosen to be mutated due to their location within the 2Å-radius of bound DMNBS. <FIGCAPTION style="text-align: center; font-size: 15px; "><B>Figure 5: visualization of DMNBS binding to designed DMNBS-tRNA/synthetase</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">

For creating a mutant library, sites to mutate were determined and partially randomized codons for site saturated mutagenesis, also with respect to the codon usage of <I>E. coli</I>, were chosen.<B><SPAN style="color:#005C04;">Characterization of Synthetase Plasmids</SPAN></B><FIGCAPTION style="text-align: center; font-size: 15px;"><B>Figure 6: tyrosyl-tRNA/synthetase pair or DMNBS-tRNA/synthetase pair plasmid </B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">

A low copy number plasmid (p15A, 10-12 copies per cell) is used to express the DMNBS-synthetase variants and <I>Methanococcus janaschii</I>wild type tyrosyl-tRNA in order to test incorporation values of DMNBS via amber codon suppression. Expression is used to be constitutive. A gentamicin resistance reporter gene is used for selection.<B><SPAN style="color:#005C04;">Characterization of pRXG (<A href="http://parts.igem.org/Part:BBa_K1416004"><U style="color:#0000EE; ">Part:BBa_K1416004</U></A> called pFRY)</SPAN></B><FIGCAPTION style="text-align: center; font-size: 15px; "><B>Figure 7: reporter plasmid pRXG</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
Once the screening method for measuring the fidelity and incorporation efficiency of the created tRNA/synthetase pairs, a fluorescence-based method as established by the the <A href="https://2014.igem.org/Team:Austin_Texas"><U style="color:#0000EE; ">iGEM-Team Austin, Texas, 2014</U></A> is used: a two-plasmid system - one plasmid (pRXG) contains an IPTG-inducible RFP-sfGFP reporter protein construct and the additional plasmid the tRNA/synthetase pair. 
The pRXG plasmid consists of a RFP domain which is connected by a linker sequence containing an amber stop codon with a sfGFP domain. The expression of the plasmid results either in red fluorescence, or - if the ncAA is incorporated at the amber stop codon within the linker site - in both: red and green fluorescence.
By comparison of fluorescence levels it is possible to determine incorporation efficiency of the generated synthetase variants.
A kanamycin resistance reporter gene is used for selection of the reporter plasmid. Furthermore, the expression of the fluorescence proteins is kept under lac-operon control for IPTG induction.
The reporter plasmid, as described above, has been created by iGEM Team Austin, Texas, 2014. A BioBrick has been built from it and characterized for further use:

<A href="http://parts.igem.org/Part:BBa_K1416004"><U style="color:#0000EE; ">by iGEM-Team Austin, Texas, 2014</U></A><B><SPAN style="color:#005C04;">Screening</SPAN></B>
For both, positive and negative screening, M9 minimal medium is used, because it has a lower optical density itself and contains buffer to stabilize the fluorescence proteins.
As a pretest, the excitation and emission wavelengths of the fluorescence proteins are measured, along with the OD<SUB>600</SUB>. Thus, the wavelengths for the screening process are identified.<FIGCAPTION style="text-align: center; font-size: 15px; "><B>Figure 8: wavelengths scan of mRFP1 and sfGFP</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
To transform the mutation library into <I>E. coli</I>, the following strategy is applied: The reporter plasmid is transformed into <I>E.coli</I> BL21 DE3 gold from which subsequently chemical competent cells are made. The library then is transformed into those cells and onto M9 minimal solid medium with gentamycin and kanamycin for selection.
A preculture is inoculated each well with one colony in 96-well microtiter plates in M9 minimal medium.  Microtiter plates are sealed with microporous tape sheets, because fluorescence proteins mature under the influence of oxygen. 

As an intermediate step the precultures are replicated onto a next microtiter plate for the purpose of homogenizing the cell density. 

Thereof, two separate screening microtiter plates are inoculated: Firstly, a positive screening plate with M9 minimal medium, addition of 2mM DMNBS as well as 100µM IPTG for expression. Secondly a negative screening plate without DMNBS. All microtiter plates are measured after the same amount of time. Incubation was performed at 900rpm, 30°C and pH 7.6. 

OD<SUB>600</SUB> and both, red and green fluorescence levels are measured as an endpoint measurement. Thus, the most efficient clones can be determined to selectively incorporate DMNBS at an amber stop codon. 

To obtain final results, the new-found, working DMNBS tRNA/synthetase pair clones undergo an online measurement rescreening with each nine replicates and the previously mentioned growth and screening conditions. 

Results can be found <A href="https://2016.igem.org/Team:Aachen/Demonstrate"><U style="color:#0000EE; ">here</U></A></P><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="characterization" class="anchor">Characterization of Existing Parts</A></H1></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;"> Characterization of pYES2 Galactose Inducible Vector (<A href="http://parts.igem.org/Part:BBa_K555009"><U style="color:#0000EE; ">Part:BBa_K555009</U></A>)</H2><FIGCAPTION style="text-align: center; font-size: 15px; "><B>Figure 9: <I>S. cerevisiae</I> vector pYES2 </B></FIGCAPTION><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"> To get an overview of the growth of <I>S. cerevisiae</I> CENPK2-1D cells, we compared the growth of the yeast cells carrying the empty vector in the induced and the non-induced conditions.
We used the growth profiler GP960 (EnzyScreen, Heemstede, The Netherlands), which is a temperature controlled shaker for microtiterplates with a scanner for growth detection. 

</P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"><FIGCAPTION style="text-align: center; padding-left: 1.0cm; padding-right: 1.0cm; font-size: 15px; "><B>Figure 10: comparative growth of induced and not induced pYES2 empty vector and pyes2 with Mfalpha (BBa_K2020026) and mCherry (BBa_E2060).</B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">
The logarithmic growth phase was determined to be from 4 to 7 hours after induction. 

Comparing the induced and non-induced cultures, it can be seen that they grow much better with additional galactose, since <I>S. cerevisiae</I> can metabolize it as a carbon source.
In order to improve the significance of these measurements, addition of glucose to the non-induced cells can be an outlook for further work. 

Furthermore, the cells carrying the MFalpha+mCherry inserts (<A href="http://parts.igem.org/Part:BBa_K2020026"><U style="color:#0000EE; ">Part:BBa_K2020026</U></A> and <A href="http://parts.igem.org/Part:BBa_E2060"><U style="color:#0000EE; ">Part:BBa_E2060</U></A>) are not affected in their growth after induction as the blue and yellow curves show. </P></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;"> Characterization of pRXG (<A href="http://parts.igem.org/Part:BBa_K1416004"><U style="color:#0000EE; ">Part:BBa_K1416004</U></A> called pFRY)</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;"><B><SPAN style="color:#005C04;">How the Reporter Plasmid works</SPAN></B>
This reporter plasmid is one of the two plasmids containing screening system for determining efficiency and fidelity of non-canonical amino acids´ (ncAA) incorporation via amber termination suppression. One plasmid contains tRNA and its corresponding aminoacylation-synthetase. The other one is this plasmid presented herein. It consists of a mRFP1 domain, which is connected through a linker sequence containing a recoded amber stop codon with a sfGFP domain. The expression of the plasmid gives either a red fluorescence, or - if the ncAA is incorporated at the recoded amber stop codon within the linker site - both a red and a green fluorescence.<B>Synthetase and tRNA</B> are constitutively expressed in a plasmid with low copy replicon p15A under metabolic stress, but are not under IPTG control for the purpose of avoiding abrupt and unpredictable effects considering the time and energy required for their assembly. Whereas the <B>reporter plasmid</B> containing two fluorescence proteins, is kept under operon control for IPTG induction like a low copy plasmid with ColE1.<B><SPAN style="color:#005C04;">Cultivation Conditions with High Throughput Measurement</SPAN></B>
In order to evolve a new aminoacylation synthetase for DMNBS in E.coli and transforming a mutant library into competent cells by using the following order of events to get a maximum output and equal optical densities:<OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><SPAN><B>Transform</B> into (BL21 DE3 gold + pRXG) on M9 solid, growth: 2 days, 37°C</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><B>Pick</B> into M9 liquid: <B>Masterplate</B>, growth: 2 days at 30°C, 900rpm,  shaking diameter: 50 mm</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><B>Replicate</B> into M9 liquid, growth 2 days at 30°C, 900 rpm, shaking diameter: 50 mm</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><B>Replicate</B> into M9 liquid <B>Screening plate</B>, growth 2 days at 30°C, 900 rpm, shaking diameter: 50 mm</SPAN></LI></OL></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
It was shown previously, that sufficient GFP expression was achieved by supplementing IPTG and ncAA, but causes a decelerated growth (1). In fact, cultivation of BL21 DE3 gold containing two different plasmids show reduced growth rates, when adding 100µM IPTG and 2 mM DMNBS to M9 minimal medium results in a growth phase of 42-48 h at 30°C to reach maximum cell density.<B><SPAN style="color:#005C04;">Host Organism</SPAN></B>
This reporter plasmid and the measurement of its corresponding proteins expressed are previously used both in an amberless E.coli strain and BL21 DE3 gold. The latter results in competition of the suppressor tRNA with RF1 at the amber stop codon.<B><SPAN style="color:#005C04;">Measurement: Wavelength</SPAN></B>
As a pre-screening setup, an endpoint detection of OD and fluorescence intensities with microtiter-plate reader is chosen. Excitation and emission spectra of mRFP1 and sfGFP were obtained from a previously conducted measurement (fig 8). For screening with Tecan Plate Reader, the following settings were used:<OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><SPAN>OD: 600 nm </SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>sfGFP 480/508 nm </SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>mRFP1 584/605 nm</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Slid: 8 nm</SPAN></LI></OL></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
While rescreening for the detection of endpoint, the same plate reader setup was used. Additionally an online measurement monitoring sfGFP and mRFP1 formation as well as scattered light measurement at 650 nm, is used. This is achieved by a screening platform based on the established Biolector setup. [<A href="#ref19PI"><U style="color:#0000EE;">19</U></A>]<B><SPAN style="color:#005C04;">Measurement: Evaluation</SPAN></B>
If the levels of optical density of the synthetase to be evaluated and a working synthetase are equal, a first approximation of efficiency and fidelity can be made by normalizing GFP levels. Thus one can eliminate the biogenic background fluorescence levels and compare the clones with each other. This mutant corresponds to No.: 1</P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"><FIGCAPTION style="text-align: center; padding-left: 1.0cm; padding-right: 1.0cm; font-size: 15px; "><B>Figure 11: Efficiency (dark blue) and fidelity (light blue) of various DMNBS mutants compared to wild type <I>Mj</I> tyrosyl synthetase (grey) </B></FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;"><B><SPAN style="color:#005C04;">With the reporter plasmid evaluated synthetases</SPAN></B><OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020050"><U style="color:#0000EE; ">Y-RS, canonical amino acid </U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K1416000"><U style="color:#0000EE; ">oNBY-RS </U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020043"><U style="color:#0000EE; ">AzF-synthetase </U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020046"><U style="color:#0000EE; ">CN-F synthetase </U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K1416001"><U style="color:#0000EE; ">Iodo-Y synthetase </U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020045"><U style="color:#0000EE; ">Nitro-Y synthetase </U></A></SPAN></LI><LI style="padding-right: 0.3cm&gt;&lt;span&gt;Amino-Y synthetase &lt;/span&gt;&lt;/li&gt;
&lt;li style=" padding-right:="" 0.3cm=""><SPAN>OMe-Y-RS</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>5HT-P synthetase</SPAN></LI></OL></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">
DMNBS mutants:<OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020052"><U style="color:#0000EE; ">DMNBS-RS Clone 1</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020053"><U style="color:#0000EE; ">DMNBS-RS Clone 2</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020054"><U style="color:#0000EE; ">DMNBS-RS Clone 3</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020055"><U style="color:#0000EE; ">DMNBS-RS Clone 4</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020056"><U style="color:#0000EE; ">DMNBS-RS Clone 5</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020057"><U style="color:#0000EE; ">DMNBS-RS Clone 6</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020058"><U style="color:#0000EE; ">DMNBS-RS Clone 7</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020059"><U style="color:#0000EE; ">DMNBS-RS Clone 8</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020060"><U style="color:#0000EE; ">DMNBS-RS Clone 9</U></A></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020061"><U style="color:#0000EE; ">DMNBS-RS Clone 10</U></A></SPAN></LI></OL></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;"><B><SPAN style="color:#005C04;">Links</SPAN></B><OL style=" list-style-type: disc; padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part K2020040"><U style="color:#0000EE; ">Fluorescence reporter for measurement of incorporation of ncAA</U></A> Whereas you find hereunder the whole plasmid as a biobrick, we provide the two flourescent proteins connected by the linker sequence as a biobrick for self-assembly in a low copy plasmid.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="http://parts.igem.org/Part:BBa_K2020040"><U style="color:#0000EE; ">DMNBS-RS Clone 10</U></A> - <A href=" https://2014.igem.org/Team:Austin_Texas/kit "><U style="color:#0000EE; ">Team Austin, Texas 2014 </U></A> built this measurement kit and probed various synthetases.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><A href="https://2016.igem.org/Team:Aachen "><U style="color:#0000EE; "> Team Aachen 2016</U></A> evolved a new synthetase for incorporation of DMNBS in <I>E. coli</I>.</SPAN></LI></OL></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;"><B><SPAN style="color:#005C04;">Attributions</SPAN></B>
iGEM Team Austin, Texas: Thanks for making the measurement kit available to us.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="hardware" class="anchor">Hardware</A></H1></DIV><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm;font-size:16px;">One of the things which we need to consider when we choose to work with light-sensitive materials is an apt environment. Light sensitive materials on exposure to stray light experience changes in properties on exposure to stray light what makes them problematic to work with under normal conditions. It requires a dedicated room where the lighting is controlled, usually by using filtered red or amber safelight. But due to economic, time and spatial limitations, not every school, university and institute can have this dark room. Hence, we have developed an affordable, convenient and portable workspace to handle these light-sensitive materials, called “Dark-bench”. It can be assembled and disassembled at any place with little effort.
After resolving a safe place to work with, we required a device with which we can study the photo cleavage reaction by activating the caged amino acid by irradiating at a specific wavelength. Since the real-world application of our project is to activate the caged subtilisin in the liquid detergent, the uncaging device, the so called LIPS stick, should be cheap and easy-to-use. That is made possible by incorporating pumps and a control circuit for optimal light exposure by adjusting various parameters automatically. With further miniaturization it can also be integrated into washing machines in the future.
Both of the devices were developed not only to satisfy our project needs but also were made with affordable and readily available components so as to make it accessible to all. More information about the hardware aspect of our project can be found <A href="https://2016.igem.org/Team:Aachen/Hardware"><SPAN style="color:#0000EE; ">here</SPAN></A>.</P></DIV><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;"><A name="synthetase" class="anchor">References</A></H1></DIV><P align="justify" style="padding-left:1.0cm; padding-right:1.0cm; font-size: 16px;"><A name="ref1PI" class="anchor" style="color:Black">[1]<SPAN style="padding-left: 0.5cm;">European Chemicals Agency, “Member State Commitee Draft Support Document for Identification of Boric Acid as a Substance of Very High Concern Because of Its CMR Properties”, 2011. [Online]. Available: https://echa.europa.eu. [Accessed: 16-Oct-2016].
</SPAN></A><A name="ref2PI" class="anchor" style="color:Black">[2]<SPAN style="padding-left: 0.5cm;">Industrieverband Körperpflege- und Waschmittel e. V., “Bericht Nachhaltigkeit in der Wasch-, Pflege- und Reinigungsmittelbranche in Deutschland 2013–2014”, 2014. [Online]. Available: http://www.ikw.org. [Accessed: 07-Oct-2016].</SPAN></A><A name="ref3PI" class="anchor" style="color:Black">[3]<SPAN style="padding-left: 0.5cm;">Landesanstalt für Umwelt, Messungen und Naturschutz Baden-Württemberg, „Bor - Ableitung einer Geringfügigkeitsschwelle zur Beurteilung von Grundwasserverunreinigungen“, 2002. [Online]. Available: http://www.fachdokumente.lubw.baden-wuerttemberg.de. [Accessed 07-Oct-2016].
</SPAN></A><A name="ref4PI" class="anchor" style="color:Black">[4]<SPAN style="padding-left: 0.5cm;">D. P. Nguyen, M. Mahesh, S. J. Elsässer, S. M. Hancock, C. Uttamapinant, and J. W. Chin, “Genetic Encoding of Photocaged Cysteine Allows Photoactivation of TEV Protease in Live Mammalian Cells,” <I>J. Am. Chem. Soc.</I>, vol. 136, no. 6, pp. 2240–2243, Feb. 2014.</SPAN></A><A name="ref5PI" class="anchor" style="color:Black">[5]<SPAN style="padding-left: 0.5cm;">E. A. Lemke, D. Summerer, B. H. Geierstanger, S. M. Brittain, and P. G. Schultz, “Control of protein phosphorylation with a genetically encoded photocaged amino acid,” <I>Nat. Chem. Biol.</I>, vol. 3, no. 12, pp. 769–772, Dec. 2007.</SPAN></A><A name="ref6PI" class="anchor" style="color:Black">[6]<SPAN style="padding-left: 0.5cm;">A. S. Baker and A. Deiters, “Optical Control of Protein Function through Unnatural Amino Acid Mutagenesis and Other Optogenetic Approaches,” <I>ACS Chem. Biol.</I>, vol. 9, no. 7, pp. 1398–1407, Jul. 2014</SPAN></A><A name="ref7PI" class="anchor" style="color:Black">[7]<SPAN style="padding-left: 0.5cm;">P. Carter and J. A. Wells, “Dissecting the catalytic triad of a serine protease,” <I>Nature</I>, vol. 332, no. 6164, pp. 564–568, Apr. 1988.</SPAN></A><A name="ref8PI" class="anchor" style="color:Black">[8]<SPAN style="padding-left: 0.5cm;">L. F. Wang and R. J. Devenish, “Expression of Bacillus subtilis neutral protease gene (nprE) in Saccharomyces cerevisiae,” <I>J. Gen. Microbiol.</I>, vol. 139, no. 2, pp. 343–347, Feb. 1993.</SPAN></A><A name="ref9PI" class="anchor" style="color:Black">[9]<SPAN style="padding-left: 0.3cm;">J. W. Chin, A. B. Martin, D. S. King, L. Wang, and P. G. Schultz, “Addition of a photocrosslinking amino acid to the genetic code of Escherichia coli,” <I>Proc. Natl. Acad. Sci.</I>, vol. 99, no. 17, pp. 11020–11024, 2002.</SPAN></A><A name="ref10PI" class="anchor" style="color:Black">[10]<SPAN style="padding-left: 0.3cm;">J. W. Chin, “Reprogramming the genetic code,” <I>EMBO J.</I>, vol. 30, no. 12, pp. 2312–2324, 2011.</SPAN></A><A name="ref11PI" class="anchor" style="color:Black">[11]<SPAN style="padding-left: 0.3cm;">R. A. Mehl et al., “Generation of a Bacterium with a 21 Amino Acid Genetic Code,” <I>J. Am. Chem. Soc.</I>, vol. 125, no. 4, pp. 935–939, Jan. 2003.</SPAN></A><A name="ref12PI" class="anchor" style="color:Black">[12]<SPAN style="padding-left: 0.3cm;">L. Wang and P. G. Schultz, “Expanding the Genetic Code,” <I>Angew. Chem. Int. Ed.</I>, vol. 44, no. 1, pp. 34–66, Jan. 2005.</SPAN></A><A name="ref13PI" class="anchor" style="color:Black">[13]<SPAN style="padding-left: 0.3cm;">X. Fu, M. Inouye, and U. Shinde, “Folding Pathway Mediated by an Intramolecular Chaperone,” <I>J. Biol. Chem.</I>, vol. 275, no. 22, pp. 16871–16878, 2000.</SPAN></A><A name="ref14PI" class="anchor" style="color:Black">[14]<SPAN style="padding-left: 0.3cm;">M. S. Kim and S. L. Diamond, “Photocleavage of o-nitrobenzyl ether derivatives for rapid biomedical release applications,” <I>Bioorg. Med. Chem. Lett.</I>, vol. 16, no. 15, pp. 4007–4010, Aug. 2006.</SPAN></A><A name="ref15PI" class="anchor" style="color:Black">[15]<SPAN style="padding-left: 0.3cm;">C. Chou, D. D. Young, and A. Deiters, “A Light-Activated DNA Polymerase,” <I>Angew. Chem. Int. Ed.</I>, vol. 48, no. 32, pp. 5950–5953, Jul. 2009.</SPAN></A><A name="ref16PI" class="anchor" style="color:Black">[16]<SPAN style="padding-left: 0.3cm;">A. Deiters, D. Groff, Y. Ryu, J. Xie, and P. G. Schultz, “A Genetically Encoded Photocaged Tyrosine,” <I>Angew. Chem. Int. Ed.</I>, vol. 45, no. 17, pp. 2728–2731, Apr. 2006.</SPAN></A><A name="ref17PI" class="anchor" style="color:Black">[17]<SPAN style="padding-left: 0.3cm;">A. Pavelka, E. Chovancova, and J. Damborsky, “HotSpot Wizard: a web server for identification of hot spots in protein engineering,” <I>Nucleic Acids Res.</I>, vol. 37, no. Web Server issue, pp. W376-383, Jul. 2009.</SPAN></A><A name="ref18PI" class="anchor" style="color:Black">[18]<SPAN style="padding-left: 0.3cm;">N. Wu, A. Deiters, T. A. Cropp, D. King, and P. G. Schultz, “A Genetically Encoded Photocaged Amino Acid,” <I>J. Am. Chem. Soc.</I>, vol. 126, no. 44, pp. 14306–14307, Nov. 2004.</SPAN></A><A name="ref19PI" class="anchor" style="color:Black">[19]<SPAN style="padding-left: 0.3cm;">G. Wandrey et al., “Probing unnatural amino acid integration into enhanced green fluorescent protein by genetic code expansion with a high-throughput screening platform,” <I>J. Biol. Eng.</I>, vol. 10, Sep. 2016.</SPAN></A></P></DIV></DIV><DIV class=" footer_area"><DIV class=" footer"><DIV class=" single_footer_container"><DIV class=" single_footer" style="float: left;"><P style="padding-left: 1cm;"><A href="http://igem.rwth-aachen.de/" style="text-decoration:none;color:#7fb983;">Link to our website</A><A href="https://2014.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki 2014 Aachen</A><A href="https://2015.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki  2015 Aachen</A><A href="#" style="text-decoration: none;color:#7fb983;">igem@rwth-aachen.de</A></P></DIV><DIV class=" single_footer" style="float: right;">                 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>