Team:Aachen/Notebook
Home
Project Idea
Project Idea
Improvement
Applied Design
Glossary
Lab
Lab
Lab Book
Hardware
Collaborations
Parts 
Interlab Study
Protocols/Methods
Human Practices
Human Practices
Engagement
Integrated Practices
Beyond the Basics
Safety
Achievements
Achievements
Medal Criteria
Proof of Concept
Results
Team
Members
Advisors
Supervisors
Attributions
Supporters
Welcome to iGEM Aachen 2016
Lab Book
To express a photocaged protease, in our case subtilisin E, we developed three lab strategies. Their milestones can be seen below in three different timelines, one for every sub-division. Details about reaching these milestones will become visible, if you click on the corresponding circles. On the page Project Idea the theory beneath our lab work is described and on the subpage Protocols & Methods the exact protocols can be seen, if you would like to repeat the experiments. It can also be helpful to have a look at our primers list .
Photocaging - Targeting Serine in Saccharomyces cerevisiae
Expression of Subtilisin E
As the first step to reach our aim of producing an inactive photocaged subtilisin E variant, which can be activated by light, we researched on the recombinant expression of subtilisin E in S. cerevisiae . The native subtilisin E is a gene of Bacillus subtilis which we could receive from the institute. In the following steps our procedures can be traced.
Determination of Parameters
Strain Selection
S. cervisiae
* the synthetase is made for this strain
* needs to have the deficiencies of both chosen vectors
Characterization of Strand [ 1 ]
Name
Auxotrophies
Prototrophies
Mating Type
CENPK2-1D
his3D1; leu2-3_112; ura3-52; trp1-289
MAL2-8c; SUC2
MATalpha
Growth – Temperature
We tested the growth in different temperatures for optimization:
Temperature
OD at start
OD after 24 h
25°C
0
0
30°C
0
>2
37°C
0
0
The S. cerevisiae strand only grew at 30°C.
Growth – Media
As our CENPK2-1D strand was growing poorly in the SC-U medium, we got the advice to try the same medium but with a higher amount of amino acids (SD-U).
Medium
OD at start
OD after 24 h
SC-U
0.1
2.9
SD-U
0.1
9.1
After OD measurements it was obvious, that the SD-U medium gave the CENPK2-1D better conditions for growth
Growth  – Protection against Contamination
To prevent bacterial contaminations, we made a comparison of the growth of our used strand in medium with and without ampicillin.
Medium
OD at start
OD after 24 h
YPD
0
2.7
YPD+Amp
0
2.6
The growth in both media seems equal. So we decided to only use medium with additional ampicillin in future.
Selection of Vector
Requirements for Vector
1. Secretion
Subtilisin E must be secreted as we do not know if it can be expressed active in cytoplasm. For industry secretion is a big advantage as it allow continuous production
2. Shuttle vector
For amplifying the vector easily it needs to have an ori and selective marker in E. coli
3. Selection system in yeast
different than the selection of synthetase plasmid which is tryptophane auxotrophy
auxotrphy must be different from trytophane
Available Vectors for Us:
Figure 1: pTEF-MF
Figure 2: pYes2
Figure 3: pcfb255 [ 2 ]
Vector
pTEF-MF
pYES2
pCFB255 [ 2 ]
pYES2Mfalpha
pCFB255MfalphaTEF1
pESC-Trp
selection in S. cerevisiae
ura3
ura3
ura3
ura3
ura3
trp
selection in E. coli
Amp
Amp
Amp
Amp
Amp
Amp
Secretion
yes
no
no
yes
yes
no
Genome Integration
no
no
yes
no
yes
no
Length
6.6 kb
5.9 kb
-
6.2 kb
-
6.5 kb
Promoter
TEF1
GAL
TEF1
GAL
TEF1
GAL
References
[1]http://www.euroscarf.de/plasmid_details.php?accno=30000B [2]Jensen et al. (2013). “EasyClone: method for iterative chromosomal integration of multiple genes in Saccharomyces cerevisiae” FEMS Yeast Research 14(2): 238-248. Provided by Irina Borodina, Technical University of Denmark
Cloning
Cloning of Subtilisin E Gene from Bacillus subtilis into an E. coli - S. cerevisiae Shuttle Vector
In the beginning we received the PHY300PKL plasmid with the subtilisin E gene. We chose an uracil deficient Saccharomyces cerevisiae strand and a shuttle vector carrying the URA gene. The vector has the secretion tag MF alpha to make harvesting of produced protein more convenient. After cloning the subtilisin E gene into the vector in Escherichia coli we want to transform it in S. cerevisiae and make a test expression. A scheme of the cloning can be seen below:
Figure 1: Scheme of cloning the Bacillus gene for subtilisin E into pTEF-MF
For the detailed labbook entries see this PDF .
Cloning of Codon Optimized Subtilisin E in pTEF-MF
As the test expression of the Subtilisin E gene from bacillus subtilis was not successful we used a codon optimized variant of the gene we had on the PHY300PKL plasmid. As a host we chose the S. cerevisiae strand CENPK2-1D again as it has the deficiencies we need for selection. As a vector backbone we tried the pTEF-MF vector again. A scheme of the cloning can be seen below:
Figure 1: scheme of cloning the codon optimized gene for subtilisin E into pTEF-MF
For the detailed labbook entries see this PDF .
Cloning into the inducible vector pYes2
After day 10 of cloning the codon optimized subtilisin E into pTEF-MF we tried a parallel strategy in the PYES2 vector. A scheme of the cloning can be seen below:
Figure 1: scheme of cloning the codon optimized gene for subtilisin E with MFalpha into pYES2
For the detailed labbook entries see this PDF .
Genome Integration (pCFB255) [ 1 ]
After codon optimization we planned to have the subtilisin E inside an integrative vector pCFB255 [ 1 ] for better expression in Saccharomyces cerevisiae
For the detailed labbook entries see this PDF .
References
[1]Jensen et al. (2013). “EasyClone: method for iterative chromosomal integration of multiple genes in Saccharomyces cerevisiae” FEMS Yeast Research 14(2): 238-248. Provided by Irina Borodina, Technical University of Denmark
Test Expression
Introduction
After cloning the subtilisin E into the vectors pTEF-MF, pyes2 and the genome integration vector pCfB255 [ 1 ] a test expression was made. We used the following methods to test for proteolytic activity:
Skim Milk Assay
The skim milk assay is a non specific test for proteolytic activity. Lysis of casein leads to a clear solution while the solution remains white when the proteolytic activity is missing.
AAPF Assay
The AAPF assay is a very specific test for proteolytic activity in serine proteases.
TCA Precipitation
It is used to precipitate proteins from cell culture supernatant to gain a higher concentration.
SDS Gel
An SDS gel can be used to make proteins visible. SDS (sodiumdodecyl-sulfate) charges all proteins negative, so that they migrate in an electric field. It is a size based sorting method.
Results in pTEF-MF
In the pTEF-MF the secretion tag MFalpha should lead to a secretion. We tested the supernatant of 24 h, 48 h and 72 h cultures with the methods described above but did not get any positive results. We also crushed the cells to test them with the same methods after 24 h, 48 h and 72 h but there was no positive result either. Subtilisin E expression was not possible in this expression system.
Results in pYES2
In the pyes2 vector with additional MFalpha secretion tag Subtilsin E also should be secreted.  We tested the vector like pTEF-MF except we induced it first with 20 % Galactose and fed 20 % again after every 24h.
Results in pCFB255[ 1 ]
Tests were performed just like the pTEF-MF, but also genome integration did not result in secretion of an active protease.
References
[1]Jensen et al. (2013). “EasyClone: method for iterative chromosomal integration of multiple genes in Saccharomyces cerevisiae” FEMS Yeast Research 14(2): 238-248. Provided by Irina Borodina, Technical University of Denmark.
SDM Ser to UAG
Introduction To make insertion of the DMNB-serine into the enzyme possible, an amber stop codon has to be at the respective site. In order to achieve this, we made a quikchange of our gene.
For all agarose gels we used 1% agarose and 1 kb Thermo scientific ladder 08:
Day 1
SDM Subtilsin E: Serine 221 to amber codon
We made a SDM (site directed mutagenesis) with YP0010 and YP0011. As template we used 1µl of sequenced subtilisin E in pTEF-MF (day 14 from cloning of subtilisin E gene from Bacillus subtilis) with a concentration of 13,6 ng/µl per 50 µl reaction.
Programm 1 (cycles: 3 D-E)
ID
98°C
30 sec
D
98°C
10 sec
A
65-74°C
45 sec
E
72°C
6 min
S
8°C
storage
Programm 2 (cylces: 15 D-E)
ID
98°C
30 sec
D
98°C
10 sec
A
65-74°C
45 sec
E
72°C
5 min
FE
72°C
10 min
S
8°C
storage
We used colums 4-9
Colum
4
5
6
7
8
9
Temperature
66.4°C
67.4°C
68.6°C
69.8°C
71°C
72.1°C
Agarose Gel
We made an agarose gel to see if our modified subtilsin E was amplified with the primers. The expected size is 7 700 bp.
1:  PCR product of column 4
2:  PCR-product of column 5
3:  PCR-product of column 6
4:  PCR-product of column 7
5:  PCR-product of column 8
6:  PCR-product of column 9
0:  ladder
There are clear bands at all the six annealing temperatures, so the PCR worked well.
DpnI Digestion
We did a DpnI digestion overnight of the quik change products.
Day 2
PCR-Clean-Up
We made a PCR-clean-up of the DpnI digested products to prepare them for transformation in E. coli Dh5alpha.
Transformation in DH5alpha
We transformed the purified SDM product into E. coli DH5 alpha and plated them on LB + Amp plates.
Day 3
Transformation Results
The transformation was successful.
Overnight Cultures
We made overnight cultures of 4 of the grown colonies in 5 ml LB + Ampicillin.
Day 4
Plasmid Isolation
We isolated the plasmids of the quikchanged subtilisin E in pTEF-MF overnights.
Nanodrop
We measured the concentration of isolated plasmids via nanodrop:
1.  912.6 ng/µl
2.   910 ng/µl
3.   866 ng/µl
4.   660.4 ng/µl
Sequencing
We sent the four samples of isolated plasmids in for sequencing with YP0007, YP0008 and YP0009.
Day 5
Sequencing Results
All four samples showed the right sequence.
Results
We did not continue after this step as the subtilisin E expression did not work.
Test Expression
As our previous efforts did not produce usable results, we did not reach this point.
Expression of Synthetase
In order to incorporate our DMNB-serine into the protease we aim to produce, we have to provide the organism with an orthogonal tRNA/Synthetase pair specific for this non canonical amino acid. We found that this has been created by the Schultz lab and contacted them. We were grateful that they agreed to send us their plasmid and thus made our work possible in the first place.
Determination of Parameters
Strain selection
S. cervisiae
* Synthetase is made for this strand
* needs to have the deficiencies of both chosen vectors
Name
Auxotrophies
Prototrophies
Mating Type
CENPK2-1D
his3D1; leu2-3_112; ura3-52; trp1-289
MAL2-8c; SUC2
MATalpha
Vector selection
The lab of Peter G. Schultz kindly sent us the plasmid carrying the genes for the tRNA/Synthetase pair (pESCTrpLeuRSBH5T252A) orthogonal in Saccharomyces Cerevisiae and capable of integration the photolabile unnatural amino acid 4,5-Dimethoxy-2-nitrobenzyl-serine.
Check Genes
Introduction
Here we introduced the tRNA/synthetase plasmid into the organism we sought to express subtilisin E in, in order to be able to integrate the noncanonical amino acid into the protease .
For all agarose gels we used 1% agarose and 1kb Thermo scientific ladder 08
Day 1
Transformation
Unfortunately, the tube was open upon receipt and no traces of dried DNA material were visible. Thus it was decided to transfer competent cells directly into the tube during the process of transformation in E. coli DH5alpha.
Day 2
Overnights
We made 5 ml LB + Amp overnight cultures of three different colonies grown on yesterday’s transformation plates.
Day 3
Cryo Cultures and Plasmid Isolation
We prepared cryo cultures with 1 ml of yesterday´s overnights and isolated the plasmids from the remaining culture. Nanodrop
We measured the concentration of the isolated plasmids: 1: 202 ng/µl 2: 214 ng/µl 3: 263 ng/µl
Control Digest
The plasmid was digested with EcoRI and checked on an agarose gel.
1: Colony 1 2: Colony 2 3: Colony 3 0: Ladder
Day 4
Preparation of competent yeast cells
We prepared competent cells out of a Saccharomyces CENPK21D strand
Day 5
Transformation of pESCTrpLeuRSBH5T252A into Saccharomyces Cerevisiae
We transformed pESCTrpLeuRSBH5T252A into an Uracil and Tryptophan deficient Saccharomyces cerevisiae strand (CENPK21D).
Day 8
Transformation Results
The transformation grew but at the control plates where untransformed competent cells where plated on SD-Trp medium also a few colonies grew with no obvious reason.
Overnight Culture
5 ml Overnight cultures of 5 of the cultures derived from the transformations two days ago were made in SD-U medium with ampicillin.
Day 9
Cryo Cultures and Plasmid Isolation
We prepared cryo cultures with 1 ml of yesterday´s overnights and isolated the plasmids from the remaining culture. As an alternate first step the yeast cell walls were lysed with beads by vortexing cultures in buffer with them for 30 minutes instead of the bacterial lysis buffer.
Nanodrop
We measured the concentration of the isolated plasmids:
1: 400 ng/µl 2: 316.7 ng/µl
Agarose Gel
The isolated plasmids were digested with EcoRI and put on a gel.
S = Synthetase plasmid for control 1-4 = Colony 1-4 0 = Ladder
No plasmids were visible. Advice from our advisors who worked with yeast suggested that we transformed the plasmids we isolated into E. coli for amplification and then look at it on a gel again.
Day 10
Transformation
The isolated plasmids the yeast cells were transformed into E. coli DH5alpha.
Day 11
Overnights
Overnights of two colonies from yesterday’s transformation were made.
Day 12
Plasmid Isolation + Cryos
We made cryo cultures from yesterday’s overnights and isolated plasmids from the remaining culture. Nanodrop
The concentration of the isolated plasmids was measured with a nanodrop:  1. 807,8 ng/ul  2. 815,5 ng/ul Digestion
We to cut the samples 2a-4b plus the control vectors with confirmed sequence with EcoRI.
Agarose Gel
Oddly, the agarose gel shows that the plasmids are not at the expected size.
3a, 3b: isolated plasmids 1 + 2 CSyn: Control (synthetase plasmid) (other slots are not relevant for this section)
Sequencing
We sent the original plasmids for sequencing with primers YP0022 and YP0023, that bind to the respective promotors and primers to find out the sequence of the plasmid, as this was not given to us along with the plasmid. We hoped to find out what went wrong in the transformation process into yeast like this. Also we contacted a research group, who worked with this plasmid for sequences of primers, as we could not reach the group we originally obtained it from, and got YP0016b, YP0017b, YP0018b and YP0019b. With these we also sequenced.
Day 13
Results
The sequencing revealed that we did not have the plasmid we expected, as the primers did not bind. Our efforts to receive the right sequence of this plasmids somehow were unsuccessful. As we had no way to control, if what we were working with was the right thing and without mutations, we sadly could not use the synthetase in our work.
Test Expression
As we are still dealing with challenges of the previous step, we did not yet continue to this point.
Co-transformation
As our previous efforts did not produce usable results, we did not reach this point.
Test Expression
As our previous efforts did not produce usable results, we did not reach this point.
Test Photocleavage and Activity
As our previous efforts did not produce usable results, we did not reach this point.
Photocaging - Targeting Tyrosine in Escherichia coli
Expression of subtilisin E in E. coli
Our aim here was to express the natural subtilisin E in E. coli and to determine the optimal conditions.
Building the expression system
Day 1
Our aim was to create a functional expression system for subtilisin E in Escherichia coli. At first, we had to optimize the sequence of the subtilisin E gene from Bacillus subtilis for Escherichia coli codon usage. For this purpose, our DNA sequence was improved with the DNA and protein sequence analysis tool ‘Geneious’ and the ‘Codon Optimization Tool’ from IDT. In the following, we proceeded to design the expression system digitally.
The expression system consists of commonly used BioBricks:
Figure 1: Visualization of the system designed for the expression of subtilisin E in E. coli
Sub-part
BioBrick number
Additional basepairs
BioBrick prefix
Promoter
BBa_R0011
Ribosome-binding site
BBa_B0034
Secretion tag pelB
BBa_J32015
Pro-peptide  +  subtilisin E
BBa_K2020000
Terminator
BBa_B0010
BioBrick suffix
Additional basepairs
This construct was ordered at IDT as a gblock and named ‘EI0002’.
We decided to use the pSB1C3-backbone from iGEM. To amplify the backbone, we transformed BBa_J04450 (RFP expression system in a pSB1C3 backbone) in Escherichia coli BL21.
Figure 3: BBa_J04450 in pSB1C3. Picture taken from the Registry
Day 2
The ordered DNA sequence EI0002 was resuspended to a concentration of 20 ng/µL and amplified via PCR with the primers shown below.
Figure 4: Amplification of EI0002.
Multiple identical samples were prepared.
temperature
duration
cycles
Initial denaturation
98°C
30 s
Denaturation
98°C
10 s
24 cycles
Annealing
Gradient from 55°C to 65°C
30 s
Elongation
72°C
90 s
Final elongation
72°C
10 min
Figure 5: PCR program.
Afterwards, the size of the PCR products was examined via agarose-gelelectrophoresis.
Figure 6: Agarose gel picture. PCR-amplified insert (column 1-12), DNA ladder #SM0312.
As all samples showed the correct size, the DNA concentration was increased by purification of multiple samples through one column during PCR clean-up.
In addition to this, multiple overnight cultures of 5 mL were prepared with the plates of the transformation from BBa_J04450 in E. coli BL21 from the day before. As controls, overnight cultures with E. coli BL21 wildtype were prepared, with and also without addition of antibiotic.
Day 3
Growth?
Ligation product
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Figure 7: Results of the overnight cultures. All controls were correct.
We prepared cryo cultures of the overnight cultures of BBa_J04450 in E. coli BL21 and isolated the plasmids from the cells for further experiments.
Day 4
In the following, we cloned the PCR-amplified, linear expression system EI0002 from day 1 into the pSB1C3 backbone of BBa_J04450. Both samples were digested with the enzymes EcoRI and PstI. After that, the samples were ligated for one hour at room temperature and then transformed into E. coli BL21. All controls were prepared according to the protocol.
Day 5
Growth?
EI0002 in pSB1C3
Yes
Control 1: dephosphorylated backbone
No
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 8: Transformation results. All controls were correct.
To find a successfully ligated plasmid, we screened 24 colonies via Colony PCR with primers that bind outside of the insert to pSB1C3 and in parallel prepared overnight cultures of the same colonies. The samples were tested via agarose-gelelectrophoresis after the PCR.
Figure 9: Agarose gel picture of the colony-PCR. PCR-amplified insert (column 1-12), DNA ladder #SM0312, positive clones can be seen at the top in column 1, 11 and 12
A few colonies showed the correct size and were therefore used for further experiments. The overnight cultures of all negative colonies were discarded.
Day 6
Growth?
EI0002 in pSB1C3
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Control 3: BL21 WT without insert
No
Figure 10: Results of the overnight cultures. All controls were correct.
We prepared cryo cultures of the overnight cultures of the ligation product in E. coli BL21 and isolated the plasmids from the cells. The purified plasmids were sent in for sequencing.
Day 7
All sequenced samples show fatal mutations. Either frameshifts in the coding sequence or multiple deletions in the promoter occurred.
Despite repeating the experiment above multiple times, we were not able to obtain an expression system for subtilisin E without fatal mutations. Combined with the experience of other iGEM teams that the promoter BBa_R0011 shows some degree of expression before the addition of the inducer, we assumed that subtilisin E prohibits cell growth of E. coli if expressed in its active form.
As a result, we progressed by exchanging the promoter.
Day 8
To exchange to promoter of our expression system, we started with extracting parts of it via PCR. Hence, we used the primers shown below to obtain a sequence with the secretion tag, the subtilisin E gene and a terminator that is flanked by the BioBrick prefix and suffix.
Figure 11: Extraction of a part of the expression system via PCR.
Afterwards, we cloned this sequence in the BioBrick BBa_J04500 with the restriction sites EcoRI and PstI. BBa_J04500 is a protein expression backbone which consists of the lacI promoter BBa_R0010 and the ribosome-binding site BBa_B0034 integrated in the BioBrick standard vector pSB1C3. Then, we transformed the obtained construct in E. coli BL21 together with all needed controls.
Day 9
Growth?
PCR extracted sequence in BBa_J04500
Yes
Control 1: dephosphorylated backbone
No
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 12: Transformation results. All controls were correct.
We prepared 5ml overnight cultures of the colonies and the usual controls.
Day 10
Growth?
PCR extracted sequence in BBa_J04500
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Figure 13: Results of the overnight cultures. All controls were correct.
At first, we prepared cryo cultures of the overnight cultures and then isolated the plasmids from the cells. In the following, we linearized the purified plasmids with EcoRI and tested their size via agarose-gelelectrophoresis.
Figure 14: Agarose gel picture of expression system with new promoter. Ligation products (column 1-5), DNA ladder #SM0312. Positive clones can be seen in column 3, 4 and 5.
The samples with the correct size were sent in for sequencing.
Testing the Expression
Day 1
For the analysis of the expression level we needed an empty backbone control.  Hence, we digested BBa_J04450 with XbaI and SpeI and ligated the sample without addition of a further insert. The ligation product was then transformed in E. coli BL21.
Day 2
Growth?
Empty backbone
Yes
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 1: Transformation results. All controls were correct.
Some colonies of the ligation product appeared to be pink. This was only possible due to a re-ligation of the RFP-expression cassette into pSB1C3. Therefore, these colonies weren’t used for further experiments. Only white colonies were used for the preparation of 5ml overnight cultures.
Growth?
Empty backbone
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Figure 2: Results of the overnight cultures. All controls were correct.
We prepared cryo cultures of the overnights of the empty backbone and purified the plasmids from the cells. Afterwards, we checked the size of the plasmids via agarose-gelelectrophoresis.
Figure 3: Agarose gel picture of the empty backbone pSB1C3 (column 1-2). DNA ladder #SM0312. Both clones are positive.
One of the samples that showed the correct size was sent in for sequencing.
Day 4
One of the samples of the PCR extracted expression system in BBa_J04500 that was sequenced beforehand and also the sample with the empty backbone showed the correct sequence and could therefore be used for further experiments.
To analyze the expression level of subtilisin E we prepared 20ml pre-cultures of the positive sample. In addition, we prepared pre-cultures of the E. coli BL21 wildtype and the pSB1C3 empty backbone in E. coli BL21 as controls.
Day 5
The pre-cultures from the day before were used to inoculate two 50ml main-cultures each to an OD600 of 0.1. This main-cultures were then incubated at 37°C, 250 rpm until OD 0.3 was reached. Then, the main-cultures were induced by adding 0.5mM IPTG. Afterwards, one of the main-cultures was incubated at 30°C and the other at 25°C. We took samples of 500µL hourly and continued the incubation overnight.
Day 6
We took one sample of each main-culture from the day before and then lysed all samples in a sonication bath for about 30 minutes. The supernatant and the pellet of these samples were then tested separately via SDS-gelelectrophoresis.
Unfortunately, we were not able to obtain any clear results from the SDS gel and thus could not verify the expression of subtilisin E.
Testing the folding and activity
Day 1
We continued our experiments by performing a skim milk assay on agar plates. This assay utilizes the clearance of skim milk due to proteolysis to detect proteolytic activity. Therefore, we poured LB skim milk agar plates containing IPTG and the needed antibiotic and streaked the E. coli BL21 cells containing the plasmid with the expression system.
Figure 1: Skim milk plates assay. Cells containing the empty backbone (right) and cells containing the expression system for native subtilisin E (left) after incubation for 3 days at 30°C.
Comparing the clearance of the skim milk plates, a proteolytic activity could be observed for the cells containing the expression system for native subtilisin E. As a result, we concluded that within three days these cells are able to produce the native protease which will then digest the skim milk in the agar plates, resulting in a clearance.
Conclusion
We were able to express subtilisin E in E. coli in amount high enough to be detected via skim milk assay.
Mutating the expression system
This phase of the project had two aims:
At first, we wanted to show that both the active site and the pro-peptide cleavage site of subtilisin E are essential for the activity of the enzyme. Thus, we planned to simulate the incorporation of ONBY and DMNBS by exchanging both amino acids at the target sites against bigger amino acids and then to analyze the activity of the mutated enzyme as a proof of principle.
Our second goal was to mutate our current expression system in order to exchange serine in the active site and tyrosine in the pro-peptide cleavage site against the UAG amber stop codon in preparation for the expression of a photocaged enzyme.
Site-directed mutagenesis
Day 1
To prepare the expression of a photocaged protease and to prove our principle, we performed a site-directed mutagenesis. Four different strategies were pursued:
SDM
amino acid
DNA
amino acid
DNA
1.
serine  >  tyrosine
AGC  >  TAC
2.
serine  >  DMNB-serine
TAC  >  TAG
3.
tyrosine  >  tryptophan
TAT  >  TAG
4.
tyrosine  >  ONB-tyrosine
TAT  >  TGG
Figure 1: Overview of different SDM strategies.
As a proof of principle we exchanged serine in the active site against tyrosine and tyrosine in the pro-peptide cleavage site against tryptophan. In preparation for the expression of a photocaged enzyme we exchanged the codons for the targeted amino acids against the UAG amber stop codon.
To achieve this, we simultaneously performed the site-directed mutagenesis three times (SDM 1, 3 and 4) with fitting primers and our expression system after the exchange of promoters as a template. SDM 2 had to be performed separately and with SDM 1 as a template at a later time.
For SDM 1, 3 and 4 we executed the following PCR:
temperature
duration
cycles
Initial denaturation
98°C
30s
Denaturation
98°C
10s
3 cycles
Annealing
Gradient from 70°C to 90°C
30s
Elongation
72°C
3min 25s
Figure 2: PCR program 1.
temperature
duration
cycles
Initial denaturation
98°C
30s
Denaturation
98°C
10s
15 cycles
Annealing
Gradient from 70°C to 90°C
30s
Elongation
72°C
3min 25s
Final elongation
72°C
10min
Figure 3: PCR program 2.
>
Multiple samples were prepared for each SDM. After completion of both PCR programs the size of all samples was verified via agarose-gelelectrophoresis
Figure 4: Agarose gel pictures of SDM 1 (top), 3 (middle) and 4 (bottom). DNA ladder #SM0312.
As all samples showed the correct size, identical samples were purified through one column during PCR clean-up to increase the DNA concentration.
After that, the DpnI digest was set up and incubated at 37°C overnight.
Day 2
In the beginning we performed a heatkill with the DpnI-digested samples of SDM 1, 3 and 4. Then, all samples were transformed in E. coli BL21 together with all needed controls.
Day 3
Growth?
SDM 1, 3 and 4
Yes
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 5: Transformation results . All controls were correct.
Multiple 5ml overnight cultures were prepared for each SDM approach together with all needed controls.
Day 4
Growth?
SDM 1, 3 and 4
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Figure 6: Results of the overnight cultures. All controls were correct.
We prepared cryo cultures of the overnight cultures of SDM 1, 3 and 4 in E. coli BL21 and isolated the plasmids from the cells to use them for further experiments. A few samples were sent in for sequencing.
Day 5
The sequencing results showed that we were able to obtain samples with the desired, modified sequence. Hence, we used one sample with a confirmed sequence of SDM 1 as a template for SDM 2. In the following, we executed the following PCR:
temperature
duration
cycles
Initial denaturation
98°C
30s
Denaturation
98°C
10s
3 cycles
Annealing
Gradient from 70°C to 90°C
30s
Elongation
72°C
3min 25s
Figure 7: PCR program 1.
temperature
duration
cycles
Initial denaturation
98°C
30s
Denaturation
98°C
10s
15 cycles
Annealing
Gradient from 70°C to 90°C
30s
Elongation
72°C
3min 25s
Final elongation
72°C
10min
Figure 8: PCR program 2.
Multiple identical samples were prepared. After completion of both PCR programs the size of all samples was verified via agarose-gelelectrophoresis.
Figure 9: Agarose gel picture of expression system plasmid after SDM 2. DNA ladder #SM0312.
As all samples showed the correct size, identical samples were purified through one column during PCR clean-up to increase the DNA concentration.
After that, the DpnI digest was set up and incubated at 37°C overnight.
Day 6
We performed a heatkill with the DpnI-digested samples of SDM 2. Then, the sample was transformed in E. coli BL21 together with all needed controls.
Day 7
Growth?
SDM 2
Yes
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 10: Transformation results. All controls were correct.
Multiple 5ml overnight cultures were prepared together with all needed controls.
Day 8
Growth?
SDM 2
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Figure 11: Results of the overnight cultures. All controls were correct.
We prepared cryo cultures of the overnight cultures of SDM 2 in E. coli BL21 and isolated the plasmids from the cells. Some samples were sent in for sequencing.
Day 9
The sequencing results showed that in some samples we successfully modified our expression system with the desired mutations.
Testing the expresion
In the following we wanted to analyze the expression level of the different, mutated versions of subtilisin E.
To express a photocaged enzyme not only a stop codon at the target site is needed but also an orthogonal tRNA/synthetase pair that incorporates the desired non-canonical amino acid in response to the amber stop codon. While targeting the pro-peptide cleavage site, we wanted to use the ONBY-tRNA/synthetase pair for E. coli which derives from the tyrosyl-tRNA/synthetase pair from Methanococcus janaschii. This year, the DNA sequence for the ONBY-tRNA/synthetase-pair was available on the iGEM kit plate as BioBrick BBa_K1416000 . However, this part was only available in pSB1C3. This means, that the plasmids for both our expression system and the tRNA/synthetase-pair have the same origin of replication and therefore are competing against each other in terms of which of them will be replicated. Therefore, we weren’t able to use the BioBrick BBa_K1416000 for our experiments.
Fortunately, we were able to contact members of the former iGEM team Texas, Austin from 2014 who originally created the part BBa_K1416000. They sent us the ONBY-tRNA/synthetase pair in a backbone that carries a gentamicin resistance and is replicated with an ori that is not part of the same incompatibility group as the ori of pSB1C3.
Figure 1: Schematic representation of the recieved tRNA/synthetase pair in its backbone. Graphic obtained through iGEM team Texas, Austin 2014.
This allowed us to use the ONBY-tRNA/synthetase pair that is described as BBa_K1416000 without the need of cloning it into a different backbone.
Moreover, the experiments to test the incorporation efficiency of the ONBY-tRNA/synthetase pair showed that this pair is also able to incorporate DMNB-serine to a certain degree. This is why we decided to try to incorporate DMNBS in our SDM 4-modified enzyme with this tRNA/synthetase pair.
Day 1
We received the ONBY-tRNA/synthetase pair from team Texas, Austin 2014 as dried DNA on filter paper. At first, we cut the marked region from the filter paper, put it in a 1.5ml tube and immersed it in 100µl Tris-HCl pH 8.0. After incubation for 10 minutes, we used 2µl for a transformation in E. coli BL21 and in parallel transformed the usual controls.
Day 2
Growth?
ONBY-tRNA/synthetase pair
Yes
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 2: Transformation results. All controls were correct.
Multiple 5ml overnight cultures were prepared for each SDM approach together with all needed controls.
Day 3
Growth?
ONBY-tRNA/synthetase pair
Yes
Control 1: BL21 WT with antibiotic
No
Control 2: BL21 WT without antibiotic
Yes
Figure 3: Results of the overnight cultures. All controls were correct.
Cryo cultures of the ONBY-tRNA/synthetase pair in E. coli BL21 were prepared and we isolated the plasmids from the cells. One sample was sent in for sequencing.
Day 4
The sequencing results showed that the sample had the correct sequence and could therefore be used in further experiments.
In the following we executed a co-transformation in E. coli BL21:
SDM 2-modified expression system and ONBY-tRNA/synthetase pair
SDM 4-modified expression system and ONBY-tRNA/synthetase pair
During the transformation 2µl of each plasmid sample were used and the cells were plated on LB agar plates with two antibiotics.
Day 5
Growth?
Co-transformation
Yes
Control 2: RFP
Yes
Control 3: BL21 WT without insert
No
Figure 4: Transformation results. All controls were correct.
We prepared multiple 5ml overnight cultures of each co-transformation together with the needed controls.
As the transfer from the solid complex medium to liquid minimal medium causes a high stress-level for the E. coli cells, we intended to change at first from solid complex medium to solid minimal medium and then to liquid minimal medium. Hence, we streaked the same colonies that we used for the overnight cultures on M9 agar plates.
Day 6
Growth?
Co-transformation
Yes
Control 1: BL21 WT with antibiotic 1
No
Control 2: BL21 WT with antibiotic 2
No
Control 3: BL21 WT with antibiotic 1 + 2
No
Control 4: BL21 WT without antibiotics
Yes
Figure 5: Results of the overnight cultures. All controls were correct.
To back up the co-transformed cells in M9 medium we prepared cryo cultures of the overnight cultures. As the cells were able to grow in medium with two antibiotics, both plasmids must have been successfully transformed into them. As a consequence, these cells can be used for further experiments. Continuing the transfer to liquid minimal media, we prepared 5ml overnight cultures with the positive clones from the M9 agar plates in liquid M9 medium.
Day 7
In the following we wanted to start with the analysis of the expression level. For that reason, we prepared different pre-cultures:
SDM 1-modified expression system in complex media
SDM 2-modified expression system + ONBY-tRNA/synthetase pair in minimal media
SDM 3-modified expression system in complex media
SDM 4-modified expression system + ONBY-tRNA/synthetase pair in minimal media
Empty backbone pSB1C3 in complex media
Empty backbone pSB1C3 in minimal media
E. coli BL21 wildtype in complex media
E. coli BL21 wildtype in minimal media
All pre-cultures were incubated at 37°C, 250 rpm overnight.
Day 8
We then proceeded to prepare two 50ml main-cultures for each pre-culture from the day before. All main-cultures were prepared with the same medium as the pre-culture. To the cultures with minimal medium we added the non-canonical amino acid solved in 50% DMSO. Also, these cultures had to be handled in the dark after addition of the non-canonical amino acids due to their light-sensitivity.
Main-culture
Non-canonical amino acid
SDM 2 + ONBY-tRNA/synthetase pair
ONB-tyrosine
SDM 4 + ONBY-tRNA/synthetase pair
DMNB-serine
Empty backbone pSB1C3
ONB-tyrosine
E. coli BL21 wildtype
ONB-tyrosine
Figure 6: Overview of addition of non-canonical amino acids to the main-cultures.
All main-cultures were inoculated to an OD600 of 0.1 and incubated at 37°C, 250 rpm to OD600 0.3. Then, the main-cultures were induced by adding 0.5mM IPTG. Afterwards, one of each type of main-culture was incubated at 30°C and the other at 25°C. We took five identical samples of 500µL of each main-culture hourly and continued the incubation overnight.
Day 9
We took five identical samples of each main-culture from the day before and then lysed all samples with different cell lysis methods:
Lysozyme solution (1h, 37°C, 250 rpm)
Lysozyme solution (1h, 37°C, 250 rpm) + sonication bath (30 minutes)
Sonicator (3 minutes)
Glass beads mill (10 minutes)
The supernatant of these samples was then tested via SDS-gelelectrophoresis. Unfortunately, we were not able to obtain any clear results from the SDS gel and thus could not verify the expression of subtilisin E.
Testing the folding and activity
Day 1
As we were not able to detect the expression of the modified proteases via SDS gel, we proceeded by executing a skim milk assay on agar plates containing IPTG and the needed antibiotics. Therefore, we streaked the cells containing the modified expression systems on these plates and incubated at 30°C for three days.
Figure 1: Skim milk plates assay. Cells producing the native protease (picture A-C, left) in comparison to either cells containing the empty backbone (picture A, right) or cells carrying the SDM 3-modified (picture B, right) and the SDM 1-modified expression system (picture C, right) after incubation for 3 days at 30°C.
Neither the empty backbone nor the SDM 1-modified expression system did seem to cause a proteolytic activity. A clearance and therefore a proteolytic activity could only be observed for the native protease (as demonstrated before) and the SDM 3-modified expression system.
Via SDM 1 we exchanged the targeted serine in the active site against a bigger amino acid, namely tyrosine. By demonstrating that this modification doesn’t result in a clearance of the skim milk plates, we were now able to prove that serine is essential for the proteolytic activity of the protease and that exchanging it would inactivate the enzyme.
SDM 3 has been executed to exchange tyrosine in the pro-peptide cleavage site against tryptophan, a bigger amino acid. As seen on the pictures above, a clearance has occurred for those modified cells and a proteolytic activity could be assumed. Contrary to our former beliefs, it could now be deduced that exchanging tyrosine doesn’t result in a change of activity. Consequently, tyrosine in the pro-peptide cleavage site is not essential for the activity of subtilisin E.
Conclusion
We were able to proof that serine in the active site would result in a loss of activity. Hence, we demonstrated that exchanging serine against a photo-labile, non-canonical amino acid will inactivate subtilisin E and therefore proved the principle of our project.
Unfortunately, we were not able to prove the principle of our project regarding tyrosine in the pro-peptide cleavage site, as it seems to be not essential for the proteolytic activity. Thus, exchanging tyrosine against ONB-tyrosine will not influence the activity of the enzyme.
Photoclevage
This part of the project was dedicated to the actual photocaging of the protease. Unfortunately, we were not able to execute all of the experiments that we planned due to a lack of time. But as we were able to prove the concept of our project, we will not only show our progress so far but also our plans for the experiments that we weren't able to conduct.
Irradiation
Day 1
To characterize O-(2-nitrobenzyl)-L-tyrosine (ONBY), we decided to test the photocleavage with the light box provided by the iGEM team Düsseldorf 2016 and to analyze it, using gas chromatography-mass spectrometry (GC-MS). To achieve this, we dissolved 5mg of ONBY powder in 100µl pyridine and incubated at 37°C for 10 minutes. Then, the analysis was executed via GC-MS with the following conditions:
Injection Temperature: 80°C; Holding Time: 2 minutes
Final Temperature: 250°C; Temperature Ramp: 8°C
The sample was then exposed to UV light of intensity 14.8 mW/cm2 with different exposure time of 1, 3, 5 and 10 minutes using the light box and then analyzed using the same GC-MS method described above. The exposure resulted in photocleavage, and produced 2-nitroso benzaldehyde which has a molecular weight of 132.122 g/mol. Compounds with similar molecular weight have an elution time of 5.3 minutes.
Figure 1: Chromatograms of the analysis via GC-MS of ONBY. Chromatogram of pyridine as a blank (A) and ONBY after 0 (B), 1 (C), 3 (D), 5 (E) and 10 minutes (F) of exposure time.
The chromatogram of the analyzed samples shows the appearance of a peak at 5.22 minutes close to the expected elution time of 5.3 minutes. We observed that, as the exposure time was incremented from 1 minutes to 10 minutes (figure 1, graph C-F), the intensity of the peak also amplified indicating an increase in the amount of cleavage product. Also the absence of this peak in pyridine (blank) and the sample without exposure (figure 1, graph A and B), supports the absence of cleavage product. However, due to the unavailability of standard 2-nitroso benzaldehyde, we could not quantify the amount of cleaved product produced on UV exposure. The spectrum of the cleaved product show peaks at 51, 52, 73 and 79 m/z (figure 2) which is almost similar to the data available. [1]
Figure 2: Spectrum of 2-nitroso benzaldehyde for peak at 5.22 minutes.
A rough estimate of the change in concentration was made by measuring the area under the curve for the peak at 5.22 minutes (figure 3).
Figure 3: Increase in area under curve for peak at 5.22 minutes.
Also based on similar measurements, we observed a decrease in area for the peak at 19.02 minutes, indicating the presence and decrease in the concentration of ONBY (figure 4).
Figure 4: Decrease in area under curve for peak at 19.02 minutes.
As a conclusion, we deduced that the amount of cleavage product increases as the exposure time is extended.
Day 2
We intended to use the results of the ONBY photocleavage characterization to irradiate our samples S221X- and Y77X-mutated proteases which were obtained through cultivation of the SDM 2 and SDM 4-modified expression system in E. coli BL21. Furthermore, we planned to perform a skim milk assay with these samples before and after irradiation and then the degradation efficiencies of the proteases were compared by the extent of clearance. With these results we wanted to prove that due to the incorporation of DMNB-serine in the catalytic triade of the protease would lose its proteolytic activity and would only re-gain it after irradiation which would result into photocleavage.
Synthetase - Incorporating DMNBS in E. coli
Research on screening system. Decision to work with kit from iGEM Team Austin, Texas 2014
The development of a screening system, which was needed to test for the activity of a synthetase incorporating a ncAA, is a difficult and time consuming task. Therefore, we used a reporter plasmid, which was established and characterized by iGEM Team Austin, Texas, 2014 . It offers the opportunity to approximate incorporation values based on fluorescence proteins. The reporter plasmid codes for mRFP1 and sfGFP which are IPTG inducible. mRFP1 is linked to sfGFP by a sequence which includes the amber stop codon. mRFP1 is (if induced) expressed constitutively and if the orthogonal mutated synthetase is capable of recognizing the ncAA and loading it on the tRNA with the amber anticodon, is followed by sfGFP. Else, the expression stops at the amber stop codon in the linker.
Definition of Incubation Conditions
Decision for host strain
An amberless E. coli strain has been developed which only expresses proteins with stop codons excluding the amber stop codon. For our purpose we chose not to work with this strain. In our working strain E. coli BL21 DE3 gold the release factor 1 competes with the tRNA loaded with the ncAA. Still enough proteins with the amber as a stop codon are expressed funtionally. Moreover, the affinity of the tRNA to the amber stop codon on the mRNA is high enough to incorporate the ncAA in any target protein.
Arrival and Preparation of Test Kit with DNA from Texas
After receiving the test kit the plasmids were recovered in TRIS buffer pH 8.5. The diluted DNA was then transformed in E. coli BL21 DE3 gold cells which were cultivated on LB solid plates. After 1 Day of growth the cells were prepared as overnight cultures. Following the incubation, plasmids were isolated according to protocol.
Preparation of Chemical Competent Cells hosting the reporter plasmid pRXG
The reporter plasmid pRXG was transformed onto LB plates. Subsequently, some colonies were picked for overnight cultures. Thereof cryo cultures were prepared. The plasmids were isolated and sent for sequencing. The cryo cultures were then used to inoculate precultures and prepare chemical competent cells according to the protocol.
Determining Concentration of Antibiotics in LB
The growth of E. coli cells transformed with plasmids containing wild type tyrosyl-tRNA/synthetase and oNBY-tRNA/synthetase on LB solid as well as in LB liquid is not affected by concentrations of gentamicin up to 30 µg/ml, whereas a wild type E. coli BL21 DE3 gold is affected already at 5 µg/ml. Hence, appropriate concentration for gentamicin plates and cultures is 30 µg/ml. This observation is also valid for 50 µg/ml kanamycin tested with transformed reporter plasmids in E. coli BL21 DE3 gold.
Furthermore the combination of the two antibiotics is evaluated. Since the cells showed a proper growth rate in LB with 30 µg/ml gentamicin + 50 µg/ml kanamycin when transformed with a reporter plasmid and a synthetase plasmid the assessed concentrations are kept.
Figure 1: antibiotics test - growth on plates with gentamicin in concentrations: 30, 35, 40, 45, 50 µg/ml
Determining Concentration of Antibiotics in M9 Minimal Medium
By cultivation of cells transformed with one of the two types of plasmids (antibiotic resistances: kanamycin/gentamicin) and incubating on plates with different antibiotics concentrations the correct working concentrations were found. After having tested the growth on plates we cultivated the cells in liquid medium with the same antibiotics concentrations to also check it under this conditions. For this determination we evaluated the growth on the plates and in the test tubes. To make sure the cells kept the plasmids we performed a colony PCR (plates) or plasmid isolation (liquid) followed by gel electrophoresis.(picture) With the found concentrations we performed the same experiment but with both plasmids and antibiotics at the same time to find a proper concentration combination of both antibiotics. It was high enough to have an appropriate pressure on the cells, but was low enough, so that cultivation was  possible in minimal medium. The growth on microtiter plates was as well evaluated via plate reader.
Our chosen concentration where E. coli is growing and reliably keeps both plasmids can be found in the following list.
Concentrations used in M9  solid medium with E. coli BL21 DE3 gold:
Gentamicin 10 µg/ml
Kanamycin 50 µg/ml
Concentrations used in M9  liquid medium with E. coli BL21 DE3 gold:
Gentamicin 5 µg/ml and
Kanamycin 10 µg/ml
Testing Transformation Efficiency
To get a maximum of colonies with transformation of the mutant library, the following tranformation methods were tested:
A) co-transformation of both plasmids at the same time: reporter + tRNA/synthetase plasmid B) transformation of chemical competent cells (already hosting the reporter plasmid) with tRNA/synthetase plasmid Conclusion: To have both plasmids in one cell, the best solution is the transformation of the tRNA/synthetase plasmid as PCR product (deriving from mutagenesis PCR) in competent cells which already contain the reporter plasmid pRXG. Colonies appear smaller in comparison to E. coli BL21, because they have two additional plasmids resulting in metabolic burden and hence a lower growth rate. Compared to co-transformation, the number of cells is higher by a factor of 10.
Testing Various Cultivation Steps
As transformation on LB solid with subsequent picking in M9 liquid resulted in almost no growth, proper cell growth is achieved by the following method:
Pick into M9 liquid: Masterplate, growth: 2 days at 30°C, 900rpm,
Replicate into M9 liquid Screening plate, growth 2 days at 30°C, 900 rpm
For positive screening:
Induction with 100 µM IPTG
2mM DMNBS
For negative screening:
Induction with 100 µM IPTG
Cryo cultures are made from all intermediate steps.
Dissolving DMNB-serine (DMNBS)
DMNBS is soluble in 50% DMSO at basic pH. A stock of 50mM is prepared and used with a final concentration in M9 of 2 mM resulting in a pH value of 7.55.
Determining Growth with DMNBS
Cultivating E. coli BL21 DE3 gold containing two different plasmids shows lower growth rates when adding 100µM IPTG and 2mM DMNBS to M9 minimal medium, resulting in an overall growth time of 42-48h.
Modelling of Binding Pocket of DMNBS-Synthetase
We did research on already published mutated synthetases and chose the wild type Methanococcus jannaschii tyrosyl tRNA/synthetase pair to mutate for DMNBS specificity. This pair has already been reported to be orthogonal in E. coli and was mutated for the use with other ncAAs. By computational analysis and comparison to the E. coli leucyl tRNA/synthetase pair which has been changed to DMNBS affinity, amino acids in the binding pocket were identified. These needed to be mutated to achieve DMNBS specificity in the Methanococcus jannaschii tyrosyl tRNA/synthetase pair. We were lucky that the synthetase backbone was not affected by the size of the ncAA. The other mutation sites were determined with respect to chemical properties, space for the photoprotection group and codon usage of E. coli.
Figure 1: ncAA (DMNB-serine) in binding pocket of Methanococcus jannaschii tyrosyl synthetase
SDM Site Directed Mutagenesis
Tyrosine at position 32 was mutated to glycine by site directed mutagenesis with a full gradient two step PCR. Elongation time was set to 50 sec/kbp.
temperature
duration
cycles
Initial denaturation
98°C
30s
Denaturation
98°C
10s
3 cycles
Annealing
Gradient from 61°C to 71°C
45s
Elongation
5 min
Figure 1: PCR program 1
temperature
duration
cycles
Initial denaturation
98°C
30s
Denaturation
98°C
10s
15 cycles
Annealing
Gradient from 61°C to 71°C
45s
Elongation
72°C
5 min
Final elongation
72°C
10min
Figure 2: PCR program 2
Gel electrophoresis of the PCR product showed a successful amplification at the expected length.
Figure 3: Gel electrophoresis of Site Directed Mutagenesis showing a PCR product at the expected length at 5321 bp
0: ladder 1-11: temperature gradient: 61-71°C The tubes were pooled, cleaned and transformed into E. coli BL21 DE3 gold. The plates were used to prepare overnight cultures and on the following day, the plasmids were isolated. Sequencing results showed in 4 out of 4 colonies the successful replacement of the tyrosine by the glycine codon.
SSM1 Site Saturated Mutagenesis 1
The isolated plasmid from SDM (Tyr32Gly) was used as template for further site saturated mutagenesis. A full gradient two step PCR with an elongation time of 50 sec/kbp was performed on site L65 due to its location within the 2 Å radius of DMNBS. The subsequent gel electrophoresis showed the PCR product at the expected length.
Figure 1: Gel electrophoresis of Site Saturated Mutagenesis 1 showing a PCR product at the expected length of 5321 bp
The tubes were pooled and cleaned and the product was transformed into E. coli BL21 DE3 gold. The sequencing of 4 colonies showed in all 4 samples the successful mutation at the SSM1 site to various amino acids as well as the glycine codon mutation at position 32.
SSM2 Site Saturated Mutagenesis 2
With the purpose of performing Site Saturated Mutagenesis the colonies of SSM1 were washed off the transformation plates. The isolated plasmid was used as template for SSM2 (mutation sites D158 and I159). The first site was to be mutated due to known H-bonds of the tRNA/synthetase complex with the former ligand tyrosine. Second site had a sidechain, which was located within the 2 Å radius of DMNBS. The primers contain two partially randomized codons. A full gradient two step PCR was performed with an elongation time of 50 sec/kbp. Subsequent gel electrophoresis showed a PCR product at the expected length. The tubes were pooled and the product was isolated and subsequently transformed into E. coli BL21 DE3 gold. 4 out of 4 colonies were sequenced and the results still showed a glycine codon in SDM position as well as different mutations in various combinations at the three SSM sites.
Figure 1: Gel electrophoresis of Site Saturated Mutagenesis 2 showing a PCR product at the expected length at 5321 bp
SSM3 Site Saturated Mutagenesis 3
Colonies were washed off the transformation plates from SSM2. Plasmids were isolated and used as template for Site Saturation Mutagenesis 3. A full gradient two step PCR is performed with 50 sec/kbp elongation time. Gel electrophoresis showed the PCR product to be at the expected length.
Figure 1: Gel electrophoresis of Site Saturated Mutagenesis 3 showing a PCR product at the expected length at 5321 bp
Mutant Library
For purposes of sequencing  12 transformations of the mutation library into E. coli BL21 DE3 gold were made and all results showed different codons in various combinations at all mutation sites, respectively a glycine at position 32. That means, all of our mutagenesis PCRs worked.
The samples were pooled and the cleaned up PCR product was transformed into the competent cells already containing the reporter plasmid. They were streaked on M9 plates with the assessed concentrations of antibiotics for selection. After two days of incubation appropriate colonies were picked into microtiter plates containing M9 liquid Medium and respective antibiotics. Thus over 8000 clones were collected. The microtiterplates were incubated and used for inoculation of a second microtiter plate. A final inoculation into black microtiter plates with transparent bottom for screening followed this preculture. Induction with 100 µM IPTG as well as supplementation of 2 mM DMNBS was carried out at the beginning of this final cultivation (see results).
Figure 1: Sequencing results showing various mutations in saturated mutation spots with different combinations of codons. Aligned with Geneious 9.0.5.
Determine measurement conditions and parameters
Excitation and emission spectra of mRFP1 and sfGFP were obtained from fluorescence scan measurements (figure 1, figure 2). To get green and red fluorescence signals E. coli BL21 DE3 gold was transformed with the reporter plasmid pRXG and synthetase plasmid (wild type tyrosyl-tRNA/synthetase pair). The tRNA/synthetase pair coded by this plasmid (YRS) is the natural synthetase for incorporation of tyrosine, whereas the anticodon of the tRNA is mutated to recognize the amber stop codon. As a result, the linker downstream of mRFP1 is synthesized, harbouring tyrosine at the amber codon site. Following the linker, sfGFP is also expressed an its fluorescence signal is used to determine the correct wavelength for screening.
Figure 1:  normalized fluorescence spectrum of mRFP1 - dotted line representing excitation and continuous line representing emission
Figure 2:  normalized fluorescence spectrum of sfGFP - signal for successful amino acid incorporation by tRNA/synthetase pair via amber supression with dotted line representing excitation and continuous line representing emission
For screening the following wavelength settings were used: OD: 600 nm sfGFP: excitation: 480 nm emission: 508 nm mRFP1: excitation: 584 nm emission: 605 nm
Screening
As a prescreening setup an endpoint detection of OD and fluorescent intensities with a plate reader was chosen. Excitation and emission spectra of mRFP1 and sfGFP were obtained from a previously conducted measurement.
Thus a preselection of DMNBS tRNA/synthetase mutants was determined. A rescreening of mutants with promising effiency and fidelity was performed, to which an additional online measurement was made to obtain scattered light values as well as fluorescence intensities over time. This was carried out by an online measurement screening platform.
Analysis
The DMNBS synthetase variants were evaluated by normalizing GFP fluorescence levels to the wild type tyrosyl synthetase and comparing the mutants to each other. Thus incorporation efficiency and fidelity values were obtained.
Figure 1: Efficiency (dark green) and fidelity (light green) of various DMNBS mutants normalized to wild type Mj tyrosyl synthetase
These variants were sent for sequencing to obtain and discuss properties of amino acids which occur at the mutation spots.  See this link for results.
Link to our website
Wiki 2014 Aachen
Wiki  2015 Aachen
igem@rwth-aachen.de
                    
