<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_area"><DIV class=" header"><DIV class="mainmenu floatright"><NAV class="main-navigation pull-right hidden-xs hidden-sm" id="pageHighlighter"><UL><LI><A href="https://2016.igem.org/Team:Aachen">Home</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description#characterization">Improvement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Glossary">Glossary</A></LI></UL><LI><A href="#" class="has-submenu" style="text-decoration:none;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Basic_Part">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Aachen/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab/Protocols">Protocols/Methods</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Beyond_Basics">Beyond the Basics</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supervisors">Supervisors</A></LI><LI><A href=" https://2016.igem.org/Team:Aachen/Attributions#attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supporters">Supporters</A></LI></NAV></DIV></DIV></DIV><P><TITLE>Welcome to iGEM Aachen 2016</TITLE></P><DIV class="main_content_area "><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Results</H1></DIV><P align="justify" style="text-align:center; padding-left:1.0cm; padding-right:1.0cm; font-size: 30px;"><A href="https://2016.igem.org/Results"><SPAN style="color: #000000;"><B>Recipient of a Gold Medal</B><B>Nomination for Best New Application Project, Overgrad</B><B>Nomination for Best Hardware, Overgrad</B></SPAN></A></P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Recombinant Expression of Subtilisin E</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;">Before photocaging, we needed to express subtilisin E recombinantly in <I>Escherichia coli</I> or <I>Saccharomyes cerevisiae</I> as we had to use an already existing tRNA/synthetase pair for our first attempts.
<B><SPAN style="color: #005C04;">In <I>S. cerevisiae</I></SPAN></B>
Unfortunately, we were not able to express subtilisin E in <I>S. cerevisiae</I>.  That is not greatly surprising, as it originates from a prokaryote. If we would have been able to, the possibility of glycosylation causing the enzyme to be inactive, would still be quite high.
<B><SPAN style="color: #005C04;">In <I>E. coli</I></SPAN></B>
In the course of our project we were able to express native subtilisin E in <I>E. coli</I>. 
As the production of the protease interfered with the well-being of the organism it took a long time to see proteolytic activity. But we are positive that inhibiting the enzyme would improve the growth conditions and therefore yield faster results and a higher production rate.
For the verification of our results, we performed a skim milk assay on agar plates. Therefore, we poured LB skim milk agar plates containing IPTG and the needed antibiotic and streaked out the <I>E. coli</I> BL21 cells containing the plasmid with the expression system.<FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 1: Skim milk plates assay.</B> Cells containing the empty backbone (right) and cells containing the expression system for native subtilisin E (left) after incubation for 3 days at 30°C.</FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">
Comparing the clearance of the skim milk plates, a proteolytic activity could be proven for the cells containing the expression system for native subtilisin E. As a result, we concluded that within three days these cells are able to produce the native protease, which will then digest the skim milk in the agar plates, resulting in a clearance.
In conclusion, we were able to express subtilisin E in <I>E. coli</I> and to prove its proteolytic activity via skim milk assay.</P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Photocaging of Subtilisin E</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;">To avoid the immoderate use of boric acid for liquid laundry detergents, we aimed to develop a subtilisin E variant which is inactivated via introduction of a photocaged amino acid into the protein in vivo. 
<B><SPAN style="color: #005C04;">In <I>S. cerevisiae</I></SPAN></B>
Targeting the serine<SUP>221</SUP> in the active center could not be tested in <I>Saccharomyes cerevisiae</I>, as expression of the protease was unsuccessful.
<B><SPAN style="color: #005C04;">In <I>E. coli</I></SPAN></B>
With <I>E. coli</I> we were able to simulate the integration of a photo-labile, non-canonical amino acid both in the catalytic triad and the pro-peptide cleavage site by exchanging the targeted amino acids against larger amino acids. The results of these experiments showed that incorporating DMNB-serine would definitely lead to a reversible inhibition. Due to a lack of time, it was not possible to perform further investigations.
To prove the principle of our project idea, we performed a site-directed mutagenesis on each of our targeted sites to simulate the integration of a photo-labile, non-canonical amino acid. At first, we exchanged serine<SUP>221</SUP> in the active site of subtilisin E against tyrosine. In addition to this, we substituted tyrosine<SUP>77</SUP> in the pro-peptide cleavage site against tryptophan.
Then, the cells with a modified version of the expression system for subtilisin E in <I>E. coli</I> that had been proven to work beforehand were streaked on skim milk agar plates containing the inducer IPTG and the needed antibiotics.<FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; "><B>Figure 1: Skim milk plates assay.</B> Cells producing the native protease (picture A-C, left) in comparison to either cells containing the empty backbone (picture A, right) or cells carrying the SDM 3-modified (picture B, right) and the SDM 1-modified expression system (picture C, right) after incubation for 3 days at 30°C.</FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">

Neither the empty backbone nor the expression system with the modified catalytic triad seems to cause a proteolytic activity. A clearance of the skim milk plates and therefore a proteolytic activity can only be observed for the native protease (as it has been demonstrated before) and the Y77W-mutated enzyme. 
Through this experiment, we are now able to prove that serine is essential for the proteolytic activity of the protease and that exchanging it would inactivate the enzyme.
As seen on the pictures above, a clearance had occurred for the cells modified to express subtilisin with tryptophan instead of tyrosine<SUP>77</SUP>. As a result, a proteolytic activity can be assumed. Contrary to our former beliefs, it can now be deduced that exchanging tyrosine doesn’t result in a change of activity. Consequently, tyrosine in the pro-peptide cleavage site is not essential for the activity of subtilisin E. </P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Development of a New Synthetase</H2><DIV class="text-overflow" style="margin-bottom: -5px;"><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;">For making the introduction of a photocaged serine in a prokaryote possible, which would be an ideal approach to our goal, and for improving the photocaging method in general by enlarging its applications, we intended to extend the genetic code in <I>Escherichia coli</I>.
<B>Summary of achievements</B>
Expanding the photocaging method with a new tool  as a replacement for serine in <I>E. coli </I>is achieved by  creating a mutant library and screening over 8000 variants.
</P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"><FIGCAPTION style="text-align: center; padding-left: 1.0cm; padding-right: 1.0cm; font-size: 15px; "><B>Figure 3: Effieciency (dark green) and fidelity (light green) of various DMNBS mutants compared to wild type <I>Methanococcus jannaschii</I> tyrosyl tRNA/synthetase pair</B></FIGCAPTION><TABLE align="center" style="width:90%; border:2pt solid black;margin-left: auto; margin-right: auto;"><TBODY><TR><TD style="border:2pt solid black;">DMNBS-RS variant</TD><TD style="border:2pt solid black;">Incorporation rate with DMNBS (%)</TD><TD style="border:2pt solid black;">Incorporation rate without DMNBS (%)</TD><TD style="border:2pt solid black;">Incorporation ratio</TD><TD style="border:2pt solid black;">Mutation site 65</TD><TD style="border:2pt solid black;">Mutation site 158</TD><TD style="border:2pt solid black;">Mutation site 159</TD><TD style="border:2pt solid black;">Mutation site 176</TD><TD style="border:2pt solid black;">Mutation site 177</TD></TR><TR><TD style="border:2pt solid black;">WT</TD><TD style="border:2pt solid black;">100</TD><TD style="border:2pt solid black;">100</TD><TD style="border:2pt solid black;">1</TD><TD style="border:2pt solid black;">L</TD><TD style="border:2pt solid black;">D</TD><TD style="border:2pt solid black;">I</TD><TD style="border:2pt solid black;">I</TD><TD style="border:2pt solid black;">H</TD></TR><TR><TD style="border:2pt solid black;">1</TD><TD style="border:2pt solid black;">85</TD><TD style="border:2pt solid black;">28</TD><TD style="border:2pt solid black;">3,04</TD><TD style="border:2pt solid black;">H</TD><TD style="border:2pt solid black;">G</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">A</TD><TD style="border:2pt solid black;">A</TD></TR><TR><TD style="border:2pt solid black;">2</TD><TD style="border:2pt solid black;">90</TD><TD style="border:2pt solid black;">42</TD><TD style="border:2pt solid black;">2,14</TD><TD style="border:2pt solid black;">G</TD><TD style="border:2pt solid black;">R</TD><TD style="border:2pt solid black;">T</TD><TD style="border:2pt solid black;">V</TD><TD style="border:2pt solid black;">A</TD></TR><TR><TD style="border:2pt solid black;">3</TD><TD style="border:2pt solid black;">85</TD><TD style="border:2pt solid black;">42</TD><TD style="border:2pt solid black;">2,02</TD><TD style="border:2pt solid black;">H</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">-</TD><TD style="border:2pt solid black;">-</TD></TR><TR><TD style="border:2pt solid black;">4</TD><TD style="border:2pt solid black;">87</TD><TD style="border:2pt solid black;">41</TD><TD style="border:2pt solid black;">2,12</TD><TD style="border:2pt solid black;">G</TD><TD style="border:2pt solid black;">S</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">V</TD><TD style="border:2pt solid black;">A</TD></TR><TR><TD style="border:2pt solid black;">5</TD><TD style="border:2pt solid black;">93</TD><TD style="border:2pt solid black;">40</TD><TD style="border:2pt solid black;">2,33</TD><TD style="border:2pt solid black;">L</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">-</TD><TD style="border:2pt solid black;">-</TD></TR><TR><TD style="border:2pt solid black;">6</TD><TD style="border:2pt solid black;">59</TD><TD style="border:2pt solid black;">27</TD><TD style="border:2pt solid black;">2,19</TD><TD style="border:2pt solid black;">R</TD><TD style="border:2pt solid black;">G</TD><TD style="border:2pt solid black;">S</TD><TD style="border:2pt solid black;">F</TD><TD style="border:2pt solid black;">A</TD></TR><TR><TD style="border:2pt solid black;">7</TD><TD style="border:2pt solid black;">19</TD><TD style="border:2pt solid black;">3</TD><TD style="border:2pt solid black;">6,33</TD><TD style="border:2pt solid black;">G</TD><TD style="border:2pt solid black;">A</TD><TD style="border:2pt solid black;">S</TD><TD style="border:2pt solid black;">V</TD><TD style="border:2pt solid black;">A</TD></TR><TR><TD style="border:2pt solid black;">8</TD><TD style="border:2pt solid black;">60</TD><TD style="border:2pt solid black;">24</TD><TD style="border:2pt solid black;">2,5</TD><TD style="border:2pt solid black;">H</TD><TD style="border:2pt solid black;">T</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">A</TD><TD style="border:2pt solid black;">V</TD></TR><TR><TD style="border:2pt solid black;">9</TD><TD style="border:2pt solid black;">86</TD><TD style="border:2pt solid black;">32</TD><TD style="border:2pt solid black;">2,69</TD><TD style="border:2pt solid black;">H</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">P</TD><TD style="border:2pt solid black;">F</TD></TR></TBODY></TABLE></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"><FIGCAPTION style="text-align:center; font-size: 15px; padding-left:2cm;padding-right:2cm; ">Table 1: Table of Incorporation rates of DMNBS synthetase mutants with and without supplementation of DMNBS. <B>Mutant No</B>: No of mutant. WT = wild type Y-RS,  
<B> Incorporation rate with DMNBS (%)</B>: Incorporation value of DMNBS normalized to incorporation value of Mj tyr-RS of tyrosine,  
<B> Incorporation rate without DMNBS (%)</B>: Incorporation value of other substrates normalized to Mj tyr-RS, 
<B>Incorporation ratio</B>: Ratio of DMNBS incorporation and other incorporation, 
<B>Mutation site 65, 158, 159, 176, 177</B>: amino acid occurring in this site
</FIGCAPTION></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"> The two plasmid screening system for testing incorporation efficiency and fidelity of ncAA herein is used to get a first approximation of properties of DMNBS synthetase clones for E.coli, and moreover offers a good control with RFP always being formed if the reporter plasmid is present and induced, on the contrary to other screening systems.
As basis for comparison we chose to normalize fluorescence levels with wild type tyrosyl synthetase. This is possible due to equal optical densities. Furthermore, the biogenic background fluorescence levels are eliminated.  

As shown in <I>table 1</I> mutants occur which incorporate DMNBS more selectively than others. By comparing evaluated sequencing with ncAA incorporation values, the following statements can be made:

Position L65 is in most cases mutated to histidine or glycine. Arginine is also occurring. The previous modeling with Yasara 16.4.6 shows, that nitro group of DMNBS will most likely be in the vicinity of this residue possibly resulting in positively charged histidine or arginine stabilizing the nitro group. This is in accordance with previously reported evolved p-Nitrophenylalanine synthetase which also requires a histidine residue at this site [<A href="#ref_3"><U style="color:#0000EE">3</U></A>]. 

Preferred residues at position 158 and 159 are in most cases either one or two adjacent prolines. As proline has been determined prior in other synthetases [<A href="#ref_3"><U style="color:#0000EE">1,2,4</U></A>] for ncAA, the occurance in the DMNBS synthetase might have a beneficial spatial or functional aspect e.g. enlargement of the binding pocket.

Residues 176 and 177 are in most cases mutated to an amino acid with no sterical hindrance, giving also space to the bulky photoprotection group.

Mutant 7  (<I>table 1</I>, L65G, D158A, I159S, I176V, H177A) shows a very low incorporation value, also a low incorporation efficiency at all. Sequencing results confirm the exchange of amino acids in a way, that reduces specificity for the original substrate way more than for the desired one. This means incorporation of DMNBS still exceeds the one of tyrosine, while being very low overall.

In general a positively charged amino acid seems to be necessary either in position 65 or 158 due to the nitrogroup of DMNBS. Furthermore an amino acid showing no sterical hindrance is required to give space to the photoprotection group. Otherwise one or two proline residues might bend the protein structure to enlarge the binding pocket.

The general approach herein is used to get an approximation of the DMNBS synthetase mutants’ efficiency and fidelity. In order to improve one of the now existing tRNA/synthetase pairs, determinination of impact factors on incorporation values and cell growth rate will be made.

Though good fluorescent signals are obtained via plate reader, an online measurement is significantly increasing the accuracy of collected data, as the exact end-point detection is possible. 

Furthermore the influence of mutating positions 176 and 177 is to be evaluated in detail, as well as probing various combinations of here identified beneficial residues at all sites. According to the project idea the incorporation of DMNBS in subtilisin E to replace serine within the active site at position 221 will follow.<B style="color:#005b04;"> Conclusion </B></P><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"> Wild type <I>Methanococcus janaschii </I> tyrosyl synthetase is successfully muted to incorporate DMNBS in <I>E.coli</I> with reasonable efficiency by using a computational engineering method as well as an already established fluorescence based screening system. 
This is the first tRNA/synthetase pair to incorporate a photocaged serine in <I>E. coli</I> by using amber codon suppression.

</P></DIV><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Making Light Isolated Work More Comfortable</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;">
A lot of chemicals are extremely sensitive to light and therefore require a dark work area for scientists. Working in dark rooms is very inconvenient and can also affect health. We wanted to build a tool that allows you to stay in the daylight while your chemicals can remain in a protective box.

We were able to build a device called Dark Bench, which is affordable and convenient. Also the possibility to assemble and disassemble the device, makes it portable. Use of opaque Plexiglas as a building material and UV foil as a protective cover for the viewing window, makes Dark Bench light proof 
Thus, the Dark Bench is a light proof device and can provide a suitable environment to work with light sensitive materials.
</P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Activation of Inhibited Protease Before Washing</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;">As an important step for the development of the light inducible proteases we were in need to find a solution for the activation of the photocaged proteases.

For that, we built a device which can cleave the protection group off the caged protease, thus activating the protease. The inclusion of inexpensive and simple components makes the LIPs-Stick highly economical and compact, which in the near future will facilitate the installation of the device in washing machines.

</P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Summary</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;">
Expression of subtilisin E in <I>S. cerevisiae</I> was not possible so the approach to target the serine in the active center could not be tested. 
In <I>E. coli</I> expression was successful, but photocaging of the propeptide cleavage site did not lead to inhibition. 
Fortunately, we were able to develop a tRNA/synthetase pair for incorporation of photocaged serine in <I>E. coli</I>. Using this, the first approach can be revived in a host that is able to produce subtilisin E. 
This is very likely to work, as we proved that exchanging the serine in the active center with a tyrosine leads to loss of proteolytic activity. 
In future experiments we will have to determine whether the complete activity can be restored by uncaging the photocaged proteases. </P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">New Biobrick Parts</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;"><A href="https://2016.igem.org/Team:Aachen/Basic_Part"><U style="color: #0000EE;">Click here </U></A> to see all the Biobricks we created in the course of our project.

</P><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;" id="ref_3">References</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-Size:16px;">[1] Xie et al, 2006, A chemical toolkit for proteins — an expanded genetic code
[2] Tsao et al, 2006, The Genetic Incorporation of a Distance Probe into Proteins in Escherichia coli
[3] Peters et al, 2009, Photocleavage of the Polypeptide Backbone by 2-Nitrophenylalanine
[4] Schultz et al, 2002, An efficient system for the evolution of aminoacyl-tRNA synthetase specificity</P></DIV></DIV><DIV class=" footer_area"><DIV class=" footer"><DIV class=" single_footer_container"><DIV class=" single_footer" style="float: left;"><P style="padding-left: 1cm;"><A href="http://igem.rwth-aachen.de/" style="text-decoration:none;color:#7fb983;">Link to our website</A><A href="https://2014.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki 2014 Aachen</A><A href="https://2015.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki  2015 Aachen</A><A href="#" style="text-decoration: none;color:#7fb983;">igem@rwth-aachen.de</A></P></DIV><DIV class=" single_footer" style="float: right;">                 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>