Team:Aachen/Lab/Protocols
Home
Project Idea
Project Idea
Improvement
Applied Design
Glossary
Lab
Lab
Lab Book
Hardware
Collaborations
Parts 
Interlab Study
Protocols/Methods
Human Practices
Human Practices
Engagement
Integrated Practices
Beyond the Basics
Safety
Achievements
Achievements
Medal Criteria
Proof of Concept
Results
Team
Members
Advisors
Supervisors
Attributions
Supporters
Protocols
General
Polymerase Chain Reaction (PCR) 
Colony-PCR using Alkaline PEG [1]
Preparation of alkaline PEG reagent:
Combine 60 g PEG 200 (Sigma-Aldrich or equivalent) with 0.93 mL 2 M KOH and 39 mL water. If desired, KOH can be substituted by NaOH in the reagent.
Note: PEG 200 is measured by mass rather than volume because of the viscosity of the liquid.
Adjust the pH to 13.3–13.5..
Note: Due to storage, some batches of PEG 200 have an acidic, rather than neutral pH. In this case, add an additional amount of alkali to reach the target pH range.
Execution of the Colony PCR:
Pick a bit of cell material of colony with a pipette tip and put it in a PCR tube with 50 µL of the reagent. Keep the pipette tip there during incubation time.
Lyse the samples by incubation them in the prepared PEG reagent for 3-15 min at room temperature.
Use 1-5 µL of the lysates as template in a 20-50 µL PCR reaction.
Note: The aliquot of the sample lysate should not exceed 10% of the final volume of a PCR mixture.
Colony-PCR with Heatshock [2]
Pick colonies from the transformed plates.
Streak one part on a masterplate.
Suspend the other part in 10 µL water.
Incubate the suspended sample at 90°C for 5-6 minutes.
Perform a PCR with primers that bind to the insert and with 1-5 µL of the lysate as template in a 20-50 µL PCR reaction.
Extended Colony-PCR [2]
Aim:find a colony which carries the correct insert in the vector.
Pick colonies of your plates.
Streak a part of the colony on a master plate.
Suspend the other part in master sample.  (20 µL of ddH2O)
Repeat step 1 eight times → eight colonies on each master plate and in each master sample.
Repeat step 2 four times (or more) → four (or more) master plates/master samples.
incubate the master plates at 37°C.
Heat the master samples at 90°C for 6 minutes.
Perform a Colony PCR with the master samples and with primers that anneal to the correct insert.
Perform an agarose-gelelectrophoresis with the results from the colony PCR.
If any sample shows a band with the correct size, then use this sample for the next step.
Perform a Colony PCR with all eight colonies (taken from the master plate) that were part of the positive tested master sample.
Perform another agarose-gelelectrophoresis and analysis with these samples.
If any sample shows a band with the correct size, then use this sample for further experiments.
Note:This experiment allows you to test a huge amount of colonies at the same time, and then to identify the exact colony which carries the correct insert.
Precipitation 
TCA precipitation [3]
Solutions
SDS sample buffer:
2% (w/v) 2-Mercaptoethanol or 3.1% (w/v) DTT or 125 mM Tris
20% (v/v) Glycerol
0.001% (w/v) Bromophenol Blue
4% (w/v) SDS
50% TCA: 50% (w/v) Trichloroacetic Acid
NaOH/Mercaptoethanol Solution:
1% (v/v) 2-Mercaptoethanol
0.25 M NaOH
Grow yeast cells to a cell density of approximately an Optical Density at 600 nm (OD600) of 1 (1 X 107 cells/ml).
Centrifuge the 1.5 ml of cells at 5,000 rpm in a microcentrifuge for 5 min.
Remove the supernatant and resuspend the cells in 0.25 ml of NaOH/Mercaptoethanol solution.
Incubate the cells for 10 min on ice.
Add 0.16 ml of 50% TCA.
Incubate for 10 min on ice.
Centrifuge at full speed in a microcentrifuge for 10 min.
Remove the supernatant and resuspend the pellet in 1 ml of ice-cold Acetone.
Centrifuge at full speed in a microcentrifuge for 10 min.
Remove the supernatant and allow the pellet to air dry. Resuspend the pellet in 100-200 μl of SDS Sample Buffer.
Proceed with SDS-PAGE and Western Blotting.
Cell Lysis [2] 
Lysozym
Centrifuge the samples for 4 min at 4°C and 14000 rpm.
Freeze the "dried" pellet overnight.
Add 150 µL buffer on the pellet, resuspend it (e.g. with LB-medium) and incubate for 5 min at room temperature.
Add 150 µL lysozym solution (c = 5-8 mg/mL).
Incubate the samples for 1 h at 37°C and 250 rpm.
Centrifuge the samples for 4 min at 4°C and 14000 rpm.
Decant the supernatant.
Sonication
conditions:
time:            3 min
pulse on:    30 s
pulse off:    20 s
amplitude:  50%
Glass beads
Put some small glass beads into your sample tubes.
Vortex your samples (5-10 min at level 2-6), avoid foam on the top of your sample.
Put the used beads into a waste bottle filling with 70% EtOH for.
Transformation 
in Eschericha coli [4]
Thaw 100 µL E. coli DH5α or E. coli BL21(DE3) Gold on ice
Add DNA (2-400 ng/µL: 50 ng if Plasmid DNA; 2 µL (~20 ng) ligation mixture; 1-4 µL PLICing-reaction) to the competent cells and swirl gently
Incubate for 15-30 min on ice.
Perform heatshock of the competent cells using a preheated water bath at <42°C for 45s (DH5α, BL21).
Samples were immediately cooled down 5 min on ice.
Fill up the transformation mix to 1 mL with SOC media and incubate at 37°C at 250rpm for 45 min (45-60 min) for recovery of the cells.
Plate cells on LB agar plate with antibiotic respectively and dry them under the clean bench.
200 µL
resuspended pellet (from centrifugation of the leftover)
Incubate the agar plates at 37°C overnight (16-18 hours).
Count single colonies of each agar plate on the next day.
in Saccharomyces cerevisiae [5]
Combine 0.5 - 1 µg each of desired fragments, bring to a total volume of 34uL with ddH2O (for plasmids, use 100 - 300ng of plasmid DNA)
2. Prepare transformation mixture (Can be scaled up and prepared as master mix):
240 ul 50% PEG 3350
36ul 1M LiAc
50ul single stranged carrier DNA
Centrifuge aliquot of competent cells at 16000xG for 15s.
Resuspend cells in 34ul (Plasmid) DNA solution.
Add transformation mixture and vortex gently.
Incubate tubes at 30°C for 30 minutes.
Invert tubes, then incubate for precisely 30 min at 42°C.
Centrifuge cells again at 16000xg for 15s.
Resuspend pellet in 100ul dd H2O, spread on selection medium
Note:If antibiotic marker is used, then outgrowth is needed:
Resuspend cells in 500ul YPD medium and incubate for precisely 30 min at 42°C.
centrifuge cells at 16000 x G, 15s.
resuspend cells on 100ul ddH2O, then spread on medium
Preparation of Chemical Competent Cells  
in Eschericha coli [4]
Preparation of TFB1 and TFB2  solution;
TFB1:
mix the components together:
30mM k-Acetat
50mM  MnCl2 X 3H2O
100mM  RbCl2
10mM  CaCl2 x 2H2O
15%  glycerin in H2O
pH=6.8, adjusted with KOH/ HCl
sterile filtrate it under the clean bench
TFB2:
mix the components together:
10mM  MOPS
75mM CaCl2
10mM RbCl2
15% glycerin in ddH2O
pH=6.8, adjusted with KOH/ HCl
sterile filtrate it under the clean bench
Making competent E. coli cells:
Prepare the TFB1 and TFB2 solution.
Inoculate 5 mL overnight culture in LB-media and incubate at 37°C, 250rpm for 14h.
Inoculate the main culture of 200 mL LB-media with 1-2% of the preculture (OD600 =0.1 at the beginning) in a 1 L flask and incubate it at 37°C, 250rpm until the OD600 = 0.4.
Cool down the culture on ice and centrifuge it at 4°C, 3000 rpm, for 10 min.
All following steps were carried out on ice. The pellet is resuspended in 15 mL sterile filtered cooled TFB1 solution and incubated for 10 min. Resuspend by gently swirling the tube, avoid to pipette the solution up and down.
Centrifuge cells at 4°C, 3000 rpm for 10 min.
Resuspend the pellets in 2-8 mL sterile filtered cooled TFB2.
Divide competent cells in 50 µL or 100 µL aliquots in precooled Eppis and store the aliquots at -80°C immediately after freezing in liquid nitrogen
in Saccharomyces cerevisiae [5]
Material: 0.1 M LiOAc (should be filter sterilized by using 0.2mm filter).
Competent Cells:
Prepare yeast overnight culture in YPD media.
Inoculate main culture (50ml) with prepared overnight culture with OD600=0.2.
Incubate for 4-5 h, until OD600=1 is reached.
Centrifuge the culture at 2500xg for 5 min.
Wash with 25 mL sterile ddH2O.
Resuspend pellet in 1 mL 0.1 M LiOAc and transfer into a 1.5 mL Eppi.
Centrifuge your sample at 10,000xg for 15 s.
Resuspend pellet in 400 µL 0.1 M LiOAc.
Divide the resuspended samples into 50 µL aliquots, store them at 4°C.
Note: Fresh made cells can be stored for up to 1 week.
Analytics
Gelelectrophoresis [2] 
Carefully remove the comb of the prepared solid agarose gel.
Put the gel into the gel-electrophoresis set up filled with buffer.
Pipette 1 µL loading dye on a piece of parafilm.
Add 5 µL of the sample to it and mix it by pipetting it up and down (avoid bubbles).
Pipette the whole solution (6 µL), without air in the pipette tip, slowly into one well of the gel.
Pipette 2.5 µL ladder into another well as a comparison (e.g. ladder 08: #SM0311).
Let it run for 35 min: 300 mA, 90V, 50W.
Take out the gel, analyze it and then discard the gel afterwards.
Analysis of Expression Level [2] 
Preparation of pre-culture
Fill 20 mL LB media into a (small) flask, add the corresponding antibiotics.
Inoculate the pre-culture.
Incubate your culture at 37°C, 250 rpm for 16-18 hours.
Preparation of main culture
Fill 200 mL LB media into a 1000 mL flask and add the corresponding antibiotics.
Inoculate the main culture with 2-5 mL pre-culture, as to reach an OD600 of 0.1 in the main culture.
Incubate your main culture at 37°C, 250 rpm till it reaches the OD600 of 1 (2-4 hours).
Take a sample (500 µL) out of your main culture and store it at -20°C.
Induce the expression of your main culture by adding a final concentration of each 0.5 / 1 / 1.5 mM IPTG.
Incubate at 37°C, 250 rpm for an hour.
Take a sample hourly afterwards. (as described above)
Analysis
Perform a cell lysis with the samples which were taken and frozen at -20°C.
Perform a Skim Milk Assay with every lysed sample.
Perform a SDS-gel with all lysed samples.
SDS 
An intact SDS PAGE electrophoresis system  should include: a tank, lid with power cables, electrode assembly, cell buffer dam,   casting stands, casting frames, combs(usually 10-well or 15-well), and glass plates (a thin glass plate and a thicker one).
The SDS PAGE gel in a single electrophoresis run can be divided into stacking gel and separating gel.. Stacking gel is poured on top   of the separating gel (after solidification) and a gel comb is inserted in the stacking gel.   The acrylamide percentage in SDS PAGE gel depends on the size of the target protein in the   sample (details shown below).
Acrylamide  M.W. Range
7%  50 kDa - 500 kDa
10%  20 kDa - 300 kDa
12% 10 kDa - 200 kDa
15% 3 kDa - 100 kDa
Preparing the Gel:
Content of separation gel:
H2O  : 2.1 ml
30% acrylamide mix     : 1.5 ml
1,5M Tris (ph 8.8)         : 1.3 ml
10% SDS                   : 0.05 ml
10% ammonium persulfate : 0.05 ml
TEMED     : 0.002 ml
Content Stacking gel
H2O                                          : 0.72 ml
30% acrylamide mix             : 0.13 ml
1,0 M Tris (ph 6,8)                : 0.13 ml
10% SDS                                 : 0.01
10% ammonium persulfate: 0.01 ml TEMED                                    : 0.001 ml
SDS Protocol: [6]
Use one thin glass plate in front of a thicker one.
Put these two glass plates on a rubber sealing in the casting frame (make sure that you close the casting frame).
Proof the closeness with water.
Pour out the water and clean the device by using a papel towel.
After water proofing of the casting frame, the separation gel (12%) could be prepared in a separate small beaker according to the protocol above.
Note: Ammonium Persulfate and TEMED should be added at last, because TEMED will solidify the solution very fast. And be noticed that TEMED smells very bad, so use it fast and close the bottle rapidly.
After making the separation gel solution should be filled between the two glass plates until it reaches the mark of the casting frame.
You can straighten the surface by adding a thin layer of isopropanol to your separation gel.
Keep the rest of your solution in the small beaker for cheaking the solidification of your separation gel.
After the separation gel is solidified, the isopropanol should be discarded. Then  clean the glass plates with water and discard the warter afterwards.
Prepare the stacking gel in a separate small beaker according to the protocol above.
Note: Ammonium Persulfate and TEMED should be added at last, because TEMED will solidify the solution very fast. And be noticed that TEMED smells very bad, so use it fast and close the bottle rapidly.
Put the comb into the stacking gel.
Let the stacking gel solidifying by following the same procedure from the protocol above.
Note: Make sure a complete gelation of the stacking gel and take out the comb.
Running Setup
Take the glass plates out of the casting frame and set them in the cell buffer dam.
Pour the running buffer (electrophoresis buffer) into the inner chamber and keep pouring after overflow untill the buffer surface reaches the required level in the outer chamber.
Prepare the samples:
Mix your samples with sample buffer (loading buffer).
Heat them in boiling water for 5-10 min.
Loading Samples and Runing the Gel
Load prepared samples slowly into wells and make sure not to overflow. Don't forget loading protein marker (ladder).
Then cover the top and connect the anodes.
Set an appropriate volt and run the electrophoresis when everything's done.
Note: As for the total running time, stop SDS-PAGE running when the downmost sign of the protein marker (if no visible sign, inquire the manufacturer) almost reaches the foot line of the glass plate. Generally, about 1 hour for a 120V voltage and a 12% separating gel. For a separating gel posessing higher percentage of acylamide, the time will be longer.
Preparing gel after running
Take out the sealed glass plates carefully.
Clean the casting frame and used devices.
Put the used running buffer into the used running buffer beaker for further usage.
SDS gel staining-destaining procedure
Fix the gel with destaining solution for 10 min.
Staining the gel with staining solution for 10-15 min.
Perform the first destaining with the used destanng solution to for 15 min, pour back to the used destaning solution afterwards.
Perform the second destaining with the fresh destaining solution and destain for 20-30 min. Afterwards pour back to the used destaining solution bottle.
Use storage solution to destain for 1-2h, then discard the solution. ( optional)
Skim Milk Assay [7] 
Principle
Protease activity can be detected by the clearance of the skim milk. This assay is suitable for library screening in microtiterplates.
Preparation of solutions
0.1M Tris/HCl buffer
For a solution with 500mL dissolve6.05g Tris in 300mL ddH2O
adjust the pH to 8.6 using 1M HCl.
Fill up the mixture to a volume of 500mL with ddH2O.
Substrate solution:2%skim milk
Weigh 0.5g skim milk in a beaker
Stir for 2 minutes and directly start the reaction.
and add 25ml of 0.1M Tris/HCl buffer.
Procedure
Load 5-10 µL of enzyme solution in a 96-well flat bottom microtiter plate. Also use blank reactions.
Add 190-195 µL of substrate solution<.
Incubate for 15-20min at 23°C MTP spectrophotometer.
Absorbance is measured at 650nm every minute (without shaking).
Clearance(650nm)=Blank(650nm)–Sample(650nm)
Cloning
Enzymatic Digestion 
Single digestion / linearisation (10 µL sample)
Mix:
2 µL plasmid DNA (~100 ng)
1 µL restriction enzyme (e.g. EcoRI)
1 µL buffer (10x) for enzyme (e.g. Cutsmart for EcoRI)
6 µL ddH2O
Incubate it for minimum one hour at 37°C.
Double digestion (20 µL sample)
Mix:
2 µL Enzyme buffer (19x) (e.g.CutSmart)
0.4 µL restriction enzyme 1
0.4 µL restriction enzyme 2
10 µL DNA (~200 ng)
7.2 µL ddH2O
Incubate it for minimum one hour at 37°C.
Dephosphorylation [8] 
Dephosphorylation after restriction
Add 1 unit of rSAP for every 1 pmol of DNA ends (about 1 μg of a 3 kb plasmid) after restriction (example: 1 µL for 2000bp) and according buffer (volume according to concentration of buffer).
Incubate at 37°C for 30–60 minutes.
Stop reaction by heat-inactivation of rSAP at 65°C for 5 minutes.
Store at -20°C.
Cloning of BioBricks 
Preparation
Amplify the insert and the backbone, either with a PCR or a transformation and check concentrations (backbone should have a concentration of 200 ng/µL or more, insert of 100 ng/µL or more).
Prepare agar plates and SOC media.
First single digest of backbone and insert.
Mix:
20 µL  DNA template
2 µL  restriction enzyme 1
2 µL  buffer
Incubate both for 1h at 37°C, a heatkill is not necessary.
PCR clean-up with additional nanodrop
Elute in the end with 20 µL buffer.
A high concentration is needed for further steps.
Second single digest
Mix:
20 µL  DNA template
2 µL  restriction enzyme 2
2 µL  buffer
Incubate both for 1h at 37°C, a heatkill is not necessary.
PCR clean-up
Elute in the end with 20 µL buffer.
A high concentration is needed for further steps.
Dephosphorylation of backbone
Dephosporylation of backbone with rSAP (according to protocol “Dephosphorylation”)
PCR clean-up with additional Nanodrop of dephosphorylated backbone
Elute in the end with 20 µL buffer.
A high concentration is needed for further steps.
Ligation of dephosphorylated and purified backbone with insert
Mix:
2.5 µL  T4 ligase buffer X µL  backbone (In the end in this mix should have a mass ratio backbone to insert of 1:10)
20 µL  insert
2 µL  T4 ligase
It is recommended to prepare a control (1) sample with the same composition but with water instead of the insert.
Incubate the samples at room temperature for one hour.
Perform a heatkill for 10min at 65°C.
Transformation
Transform the ligation mix according to protocol “Transformation”
Recommended controls:
Control 1: Transformation of dephosphorylated backbone to check if dephosphorylation worked
Control 2: Transformation of another plasmid with same antibiotics resistance to check if transformation worked
Control 3: Plate wild type organism to check if antibiotics are working
Site Directed Mutagenesis (SDM) and Site Saturation Mutagenesis (SSM) [9] 
Step 1
Mastermix (50 µl reaction)
Set up the mastermix on ice in a 0.2ml tube for PCR according to the table below. Thaw all non-enzyme components at room temperature, mix them by short vortex and collect them by short centrifugation. It is recommended to add PfuS DNA polymerase last in order to prevent any primer degradation caused by the 3' --> 5' exonuclease activity. Polymerase may be diluted in 1x buffer just prior to use in order to reduce pipetting errors.
Tube 1 - Forward Primer
Amount
Final
Plasmid template(sequencing or column grade plasmid preparation)
X µl = 10-15 ng, max 50 ng
PDeionized water with PCR quality
39.5 µl-X µl
10 x PfuS buffer
5 µl
final 1x
X Forward primer (5 µM) [1]
5 µl
final 400 nM
dNTP-mix (10 mM each)
1 µl
Final 0.2 mM each nucleotide
PfuS(2U/µl) [2]
0.5 µl
Final 0.02 U/ul
Tube 2 - Forward Primer
Amount
Final
Plasmid template(sequencing or column grade plasmid preparation)
X µl = 10-15 ng, max 50 ng
PDeionized water with PCR quality
39.5 µl-X µl
10 x PfuS buffer
5 µl
final 1x
X Forward primer (5 µM) [1]
5 µl
final 400 nM
dNTP-mix (10 mM each)
1 µl
Final 0.2 mM each nucleotide
PfuS(2U/µl) [2]
0.5 µl
Final 0.02 U/ul
PCR Program 1
3 cycles
tube control with 50 µl
heated lid to 105°C
Step
Time
temperature
Cycles
ID
Initial Denaturation
30 sec
98°C
3 cycles
D
Denaturation
10 sec [3]
98°C
A
Annealing
15-30 sec
X°C(45-72°C) [4]
Included in the 3 cycles
E
Extension
30-50 sec/kb [5]
72°C
Included in the 3 cycles
S
Storage
8°C
Note: Extension time can be increased up to 2 min/kb if difficult template.
Step 2
Proceed with step 2 immediately after step 1 without freezing in between.
Mix the equal amount of foward (tube1) and reverse (tube2) reaction.
Add 1U=0.5µl PfuS DAN polymerase per 50 µl mixture.
PCR Program 2
15 cycles
tube control with 50 µl
105°C heated lid
Step
Time
temperature
Cycles
ID
Initial Denaturation
30 sec
98°C
D
Denaturation
10 sec [3]
98°C
15 cycles
A
Annealing
15-30 sec
X°C(45-72°C) [4]
E
Extension
30-50 sec/kb [5]
72°C
FE
Final Elongation [6]
5-10 min
72°C
S
Storage
8°C
Note: Extension time can be increased up to 2 min/kb and/or 20 cycles if difficult templates, 1µl of PfuS might be necessary.
Gel and Purification
Load all samples on a 0.8% TAE agarose gel: 100V, 300 mA for 35min.
Check for optimal annealing temperature and if enough PCR product is present.
Purify the PCR products with the PCR Purification Kit. [7]
Dpnl digest the PCR product
Add 5 µl NEB bufferand 1 µl  DpnI to the 45 µl purified PCR product.
Incubate at least for 2h at 37°C. [8]
Purify the PCR products with the PCR Purification Kit and elute in 30-50µl NE buffer.
Note: Efficiency can be increased by purification of the PCR-product before Dpnl digestion, especially when Phusion from NEB was used.
Transformation
The purified PCR products will be transformed into competent cells. See the protocol Transformation.
Helpful Notes
Numbers in brackets and with green background.
Primers: Oligonucleotide primers are generally 20-40 nucleotides long and ideally have a GC-content of 40-60%. The final concentration of each in a PCR reaction may be 0.2-1mM while 0.4µM is recommended.
PfuS DNA polymerase concentration: recommended use of 1 U per 50 µl reaction. However, the optimal concentration of PfuS DNA polymerase may vary from 0.5-2 U per 50 µl reaction depending on ampicon length and difficulty. DO not exceed 2U/50 µl reaction, especially for amplicons longer than 5kb.
Keep the denaturation as short as possible. Usually 5-10 seconds at 98°C is enough for most templates. Note: The denaturation time and temperature may vary depending on the ramp rate and temperature control mode of the cycler.
The PfuS DNA polymerase has the ability to stabilize primer-template hybridization. As a basic rule for primers >20 nt, anneal for 10-30sec at a Tm +3°C of the lower primer. For primers <20 nt use an annealing temperature equal to the Tm of the lower primer. Calculate Tm for matching sequence with IDT Oligoanalyzer
Extension time depends on ampicon length and complexity. For low complexity DNA (plasmid, lambda or BAC DNA) use extension time of 30sec per 1kb. For high complexity the extension time can be extended to 2min per 1kb.
Final Extension is an optional step which usually does not influence the outcome and can be skipped by default.
PCR purification can be skipped, depending on the buffer which is used during PCR. Dpnl is not active in Phusion HF-buffer from NEB.
Incubation time can be increased up to overnight for completely digestion.
Media
LB Medium [4] 
liquid
For 1000mL of liquid LB medium mix:
10g Peptone/ Tryptone (1%)
10g NaCl (1%)
5g Yeast extract (0.5%)
Fill up with ddH2O to 1000 mL.
Autoclave the mixture.
Add desired antibiotics (under sterile conditions) when bottle is slightly cooled down.
solid
For 1000mL of solid LB medium mix:
10g Peptone/ Tryptone (1%)
10g NaCl (1%)
5g Yeast extract (0.5%)
20g Agar
Fill up with ddH2O to 1000 mL.
SC-U/SD-U Minimal Medium [10] 
liquid
For 1000mL medium mix:
6.7g yeast nitrogen base (without amino acids)
0.95g amino acid mix
uracil
0.005% amino acid mix (aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine)
Dissolve this mix in 800mL ddH2O.
Autoclave at 15 psi, 121° C for 20min.
Cool to 50°C and add 100mL of filter-sterilized 20% glucose or 200ml of filter-sterilized 10% raffinose.
Note: If you are making induction medium, follow Steps 1–3 above except dissolve the reagents only in 800 ml of deionized water. Cool the medium to 50°C and add 100 ml of filter-sterilized 20% galactose and 100 ml of filter-sterilized 10% raffinose to the medium when you want to induce your cells.
solid
For 1000mL medium mix:
6.7g yeast nitrogen base (without amino acids)
0.95g amino acid mix
uracil
0.005% amino acid mix (aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine)
For SD-U:
63mg histidine
75mg tryptophan
250mg leucine
SOC Medium [4] 
liquid
For 500mL solution mix:
10g Tryptone
1g NaCl
2.5g  yeast extract
480mL ddH2O
Autoclave the mixture at 15psi, 121°C for 20min.
Prepare three stocks:
1.0165g MgCl2(x6H2O) in 5mL ddH2O
1.2324g MgSO4(x7H2O) in 5mL ddH2O
1.9817g glucose monohydrate in 10mL ddH2O
Sterile filtrate the solutions under sterile conditions.
Add the three solutions to first 480mL mix under clean conditions as soon as the autoclaved mix is cooled down.
Aliquote the medium and store at -20°C.
M9 Medium [11] 
liquid
Prepare seven stock solutions for mixing the M9 medium in the end.
M9 salt solution (10x)
For a 1L stock dissolve in 800mL ddH2O:
75.2g  Na2HPO4(x2H2O)
30g  KH2PO4
5g  NaCl
5g  NH4Cl
Adjust the pH value to 7.2 with NaOH.
Fill up the mixture to a final volume of 1L with ddH2O.
Autoclave for 15min at 121°C.
20% Glucose stock (w/v)
For 500mL stock solution add 100g glucose to 440 ml ddH2O
Autoclave for 15min at 121°C.
1M MgSO4 stock
For 100mL stock solution dissolve 24.65g MgSO4(x7H2O) in 87mL ddH2O.
Autoclave for 15min at 121°C.
1M CaCl2 stock
For 100mL stock solution dissolve 14.7g CaCl2(x2H2O) in 94.5mL ddH2O.
Autoclave for 15min at 121°C.
Biotin stock (1mg/mL)
For 50mL stock mix 50mg biotin with 45mL ddH2O.
Add small aliquots of 1N NaOH until the biotin is dissolved.
Add ddH2O to a final volume of 50mL.
Sterilize the solution over a 0.22µm filter.
Prepare 1mL aliquots and store at -20°C.
Thiamin stock (1mg/mL)
For a 50mL stock mix 50mg thiamin with 50mL ddH2O.
Sterilize the solution over a 0.22µm filter.
Prepare 1mL aliquots and store at -20°C.
100x trace elements solution
Prepare following 4 solutions for this stock solution.
0.1 M CuCl2 (1.7g/100mL): For 10mL of this solution dissolve 0.170g CuCl2(x2H2O) in 10mL ddH2O.
0.2 M CoCl2 (4.76g/100mL): For 10mL of this solution dissolve 0.476g CoCl2(x6H2O) in 10mL ddH2O.
0.1 M H3BO3 (0.62g/100mL): For 10mL of this solution dissolve 0.062g H3BO3 in 10mL ddH2O.
1 M MnCl2 (19.8g/100mL): For 1mL of this solution dissolve 0.198g MnCl2(x4H2O) in 1mL ddH2O.
For a 1L volume of the stock solution dissolve 5g EDTA in 800mL ddH2O.
Adjust the pH to 7.5 with NaOH.
Add:
498 mg   FeCl3
84 mg   ZnCl2
765 µL   0.1M CuCl2 solution
210 µL  0.2M CoCl2 solution
1.6 mL  0.1M H3BO3 solution 8.1 µL  1M MnCl2 solution.
Fill up the mix to a final volume of 1L with ddH2O.
Sterilize the solution over a 0.22µm filter.
For 1L* M9 mineral medium add to 867mL* sterile ddH2O:
0.3mL   1M CaCl2 solution
100mL   M9 salt solution (10x)
20mL   20% glucose solution
1mL   1M MgSO4 solution
1mL   biotin solution (1mg/mL)
1mL   thiamin solution (1mg/mL)
10mL   trace elements solution
*For 500mL of a concentrated version of this medium (needed for preparation of M9 plates)  add the same amounts to 367 mL of sterile ddH2O.
solid
For preparation of 1L M9 medium for plates prepare 500mL concentrated liquid M9 medium and warm it up to 60°C.
Dissolve 15g agar in 500mL ddH2O and autoclave this mixture for 15min at 121°C.
Add the autoclaved agar solution to the M9 concentrate.
Add antibiotics.
Pour plates under sterile conditions.
YPD Medium 
liquid
For 1000mL mix:
20g peptone (1%)
20g dextrose (2%)
10g yeast extract (0.5%)
Fill up to 1000mL mark with ddH2O.
Autoclave the mixture at 15psi, 121°C for 20min.
solid
For 1000mL mix:
10g  peptone (1%)
20g  Glc (2%)
5g Yeast extract (0.5%)
20g Agar
Fill up to 1000mL mark with ddH2O.
Autoclave the mixture at 15psi, 121°C for 20min.
When medium cooled down add according antibiotics and make plates.
Devices
BioLector, in-house construction based on the established BioLector setup
Conductivity meter
Flow Cytometer
Molecular Viewer
NanoDrop
Plate Reader Tecan M1000
Ultra Sonicator
Kits
Plasmid Isolation 
NucleoSpin® Plasmid (NoLid) kit from Macherey-Nagel, 12/2015, Rev. 09, protocol p. 16-17
PCR Clean-up 
NucleoSpin® Gel and PCR Clean-up kit from Macherey-Nagel, 01/2012, Rev. 02, protocol p. 18-19
DNA Extraction from Agarose Gel 
NucleoSpin® Gel and PCR Clean-up kit from Macherey-Nagel, 01/2012, Rev. 02, protocol p. 20-21
JET Cloning 
Thermo Scientific CloneJET PCR Cloning Kit, protocol p. 5
References
[1]"Alkaline polyethylene glycol-based method for direct PCR from bacteria, eukaryotic tissue samples, and whole blood" (Chomczynski and Rymaszewsk, BioTechniques 2006)
[2] Information we got during our work in the Schwaneberg and Blank Lab
[3] http://www.koko.gov.my/CocoaBioTech/Purification%20Precipitation3.html
[4] Sambrook et al. 1989
[5] Joska et al. 2014
[6] http://www.assay-protocol.com/molecular-biology/electrophoresis/denaturing-page
[7] 2.4.5 Skim milk detection system http://publications.rwth-aachen.de/record/229540/files/4772.pdf
[8] NEB protocol for CIP alkaline phosphatase
[9] Wang, W. Y. and Malcom, B.A.: Two-stage PCR protocol allowing introduction of multiple mutations, deletions and insertions using QuikchangeTM site-directed mutagenesis. Biotechniques, 1999. 26(4): p.680-682
[10] Thermofisher pYES2 instructions
[11]  Arie Geerlof Helmholtz Center Munich
Link to our website
Wiki 2014 Aachen
Wiki  2015 Aachen
igem@rwth-aachen.de
                    
