<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Lab_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Lab/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_area"><DIV class=" header"><DIV class="mainmenu floatright"><NAV class="main-navigation pull-right hidden-xs hidden-sm" id="pageHighlighter"><UL><LI><A href="https://2016.igem.org/Team:Aachen">Home</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description">Project Idea</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Description#characterization">Improvement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Design">Applied Design</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Glossary">Glossary</A></LI></UL><LI><A href="#" class="has-submenu" style="text-decoration:none;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Notebook">Lab Book</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Basic_Part">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Aachen/InterLab">Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Lab/Protocols">Protocols/Methods</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Beyond_Basics">Beyond the Basics</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="#" class="has-submenu" style="text-decoration:none;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supervisors">Supervisors</A></LI><LI><A href=" https://2016.igem.org/Team:Aachen/Attributions#attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Aachen/Attributions#supporters">Supporters</A></LI></NAV></DIV></DIV></DIV><DIV class="main_content_area "><DIV class=" content_area structure"><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Protocols</H1></DIV><H2 style="border-bottom: 5px solid #005b04; padding-left: 1.0cm;">General</H2><OL id="category-tabs" style=" padding-left:0.1cm; list-style-type: none;"><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Polymerase Chain Reaction (PCR)</B> </A></LI><LI><B>Colony-PCR using Alkaline PEG</B> [1]
</LI><LI style=" padding-left: 0.5cm; color:Black;"><B>Preparation of alkaline PEG reagent:</B></LI><OL type="1" style=" padding-left:0.1cm; padding-right:0.1cm; align-content: center"><LI style=" padding-right: 0.3cm;"><SPAN>Combine <B>60 g PEG 200</B> (Sigma-Aldrich or equivalent) with <B>0.93 mL 2 M KOH </B> and <B>39 mL water</B>. If desired, KOH can be substituted by NaOH in the reagent. 
<SPAN style="color: #7fb983;">Note:</SPAN> PEG 200 is measured by mass rather than volume because of the viscosity of the liquid.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Adjust the <B>pH to 13.3–13.5.</B>.
<SPAN style="color: #7fb983;">Note:</SPAN> Due to storage, some batches of PEG 200 have an acidic, rather than neutral pH. In this case, add an additional amount of alkali to reach the target pH range.</SPAN></LI></OL><LI style=" padding-left: 0.5cm; color:Black;"><B>Execution of the Colony PCR:</B></LI><OL type="1" style=" padding-left:0.1cm; padding-right:0.1cm; align-content: center"><LI style=" padding-right: 0.3cm;"><SPAN>Pick a bit of cell material of colony with a pipette tip and put it in a PCR tube with <B>50 µL</B> of the <B>reagent</B>. Keep the pipette tip there during incubation time.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Lyse the samples by incubation them in the prepared PEG reagent for <B>3-15 min</B> at room temperature.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Use <B>1-5 µL</B> of the lysates as template in a <B>20-50 µL PCR reaction</B>.<SPAN style="color: #7fb983;">Note:</SPAN> The aliquot of the sample lysate should not exceed 10% of the final volume of a PCR mixture.</SPAN></LI></OL><LI><B>Colony-PCR with Heatshock</B> [2]
</LI><LI style=" padding-right: 0.3cm;"><SPAN>Pick colonies from the transformed plates.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Streak one part on a masterplate.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Suspend the other part in <B>10 µL water</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate the suspended sample at <B>90°C</B> for <B>5-6 minutes</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a PCR with primers that bind to the insert and with <B>1-5 µL</B> of the <B>lysate</B> as template in a <B>20-50 µL PCR reaction</B>.</SPAN></LI></OL><LI><B>Extended Colony-PCR</B> [2]
</LI><LI style=" padding-left: 1cm;padding-right: 0.3cm;"><SPAN><SPAN style="color: #7fb983;">Aim:</SPAN>find a colony which carries the correct insert in the vector.
</SPAN></LI><OL type="1" style=" padding-left:0.1cm; padding-right:0.1cm; align-content: center"><LI style=" padding-right: 0.3cm;"><SPAN>Pick colonies of your plates.
</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Streak a part of the colony on a master plate.
</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Suspend the other part in master sample.  (<B>20 µL of ddH2O</B>)
</SPAN></LI></OL><LI style=" padding-right: 0.3cm;"><SPAN>Repeat step 1 <B>eight times</B> → eight colonies on each master plate and in each master sample.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Repeat step 2 <B>four times</B> (or more) → four (or more) master plates/master samples.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>incubate the master plates at <B>37°C</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Heat the master samples at <B>90°C</B> for <B>6 minutes</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a <B>Colony PCR</B> with the master samples and with primers that anneal to the correct insert.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform an <B>agarose-gelelectrophoresis</B> with the results from the colony PCR.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>If any sample shows a band with the correct size, then use this sample for the next step.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a <B>Colony PCR</B> with <B>all eight colonies</B> (taken from the master plate) that were part of the positive tested master sample.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform another <B>agarose-gelelectrophoresis</B> and analysis with these samples.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>If any sample shows a band with the correct size, then use this sample for further experiments.
<SPAN style="color: #7fb983;">Note:</SPAN>This experiment allows you to test a huge amount of colonies at the same time, and then to identify the exact colony which carries the correct insert.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Precipitation</B> </A></LI><LI><B>TCA precipitation</B> [3]
</LI><LI style=" padding-right: 0.3cm; padding-left: 0.5cm;color:Black;"><B>Solutions</B></LI><LI style="padding-right: 0.3cm;"><SPAN><B>SDS sample buffer</B>:
2% (w/v) 2-Mercaptoethanol or 3.1% (w/v) DTT or 125 mM Tris 
20% (v/v) Glycerol 
0.001% (w/v) Bromophenol Blue 
4% (w/v) SDS
</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN><B>50% TCA</B>: 50% (w/v) Trichloroacetic Acid</SPAN></LI><LI style="padding-right: 0.3cm; "><SPAN><B>NaOH/Mercaptoethanol Solution</B>:
1% (v/v) 2-Mercaptoethanol
0.25 M NaOH
</SPAN></LI><OL type="1" style=" padding-left:0.1cm; padding-right:0.1cm; align-content: center"><LI style=" padding-right: 0.3cm;"><SPAN>Grow yeast cells to a cell density of approximately an Optical Density at 600 nm (OD600) of 1 (1 X 107 cells/ml). </SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge the 1.5 ml of cells at 5,000 rpm in a microcentrifuge for 5 min. </SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Remove the supernatant and resuspend the cells in 0.25 ml of <B>NaOH/Mercaptoethanol </B> solution. </SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate the cells for 10 min on ice.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add 0.16 ml of <B>50% TCA</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate for 10 min on ice.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge at full speed in a microcentrifuge for 10 min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Remove the supernatant and resuspend the pellet in 1 ml of ice-cold <B>Acetone</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge at full speed in a microcentrifuge for 10 min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Remove the supernatant and allow the pellet to air dry. Resuspend the pellet in 100-200 μl of <B>SDS Sample Buffer</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Proceed with <B>SDS-PAGE</B> and <B>Western Blotting</B>.</SPAN></LI></OL><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Cell Lysis </B> [2] </A></LI><LI><B>Lysozym</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge the samples for <B>4 min</B> at <B>4°C</B> and <B>14000 rpm</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Freeze the &quot;dried&quot; pellet overnight.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add <B>150 µL</B> buffer on the pellet, resuspend it (e.g. with LB-medium) and incubate for <B>5 min</B> at room temperature.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add <B>150 µL</B> lysozym solution <B>(c = 5-8 mg/mL)</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate the samples for <B>1 h</B> at <B>37°C</B> and <B>250 rpm</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge the samples for <B>4 min</B> at <B>4°C</B> and <B>14000 rpm</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Decant the supernatant.</SPAN></LI><LI><B>Sonication</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>conditions:</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>time:            3 min</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>pulse on:    30 s</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>pulse off:    20 s</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>amplitude:  50%</SPAN></LI><LI><B>Glass beads</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Put some <B>small</B> glass beads into your sample tubes.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Vortex your samples <B>(5-10 min at level 2-6)</B>, <B>avoid foam</B> on the top of your sample.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Put the used beads into a waste bottle filling with <B>70%</B> EtOH for.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Transformation</B> </A></LI><LI><B>in Eschericha coli</B> [4]
</LI><LI style=" padding-right: 0.3cm;"><SPAN>Thaw 100 µL <B>E. coli DH5α</B> or <B>E. coli BL21(DE3)</B> Gold on ice</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add <B>DNA</B> (2-400 ng/µL: 50 ng if Plasmid DNA; 2 µL (~20 ng) ligation mixture;
1-4 µL PLICing-reaction) to the competent cells and swirl gently</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate for <B>15-30 min on ice.</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform <B>heatshock</B> of the competent cells using a preheated water bath at &lt;42°C for 45s (DH5α, BL21).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Samples were immediately cooled down <B>5 min on ice.</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Fill up the transformation mix to 1 mL with <B>SOC media</B> and
incubate at <B>37°C at 250rpm for 45 min </B> (45-60 min) for recovery of the cells.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Plate cells on <B>LB agar plate</B> with antibiotic respectively and dry them under the clean bench.
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>200 µL</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>resuspended pellet (from centrifugation of the leftover)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate the agar plates at <B>37°C overnight</B> (16-18 hours).</SPAN></LI><LI><SPAN>Count single colonies of each agar plate on the next day.</SPAN></LI><LI><B>in Saccharomyces cerevisiae</B> [5]

</LI><LI style=" padding-right: 0.3cm;"><SPAN>Combine 0.5 - 1 µg each of desired fragments, bring to a total volume of 34uL with <B>ddH2O</B> (for plasmids, use 100 - 300ng of plasmid DNA)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>2.	Prepare <B>transformation mixture</B> (Can be scaled up and prepared as master mix):240 ul 50% PEG 3350
36ul 1M LiAc
50ul single stranged carrier DNA
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge aliquot of competent cells at 16000xG for 15s.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Resuspend cells in 34ul (Plasmid) <B>DNA solution.</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add transformation mixture and vortex gently.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate tubes at 30°C for 30 minutes.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Invert tubes, then incubate for precisely 30 min at 42°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge cells again at 16000xg for 15s.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Resuspend pellet in 100ul dd H2O, spread on <B>selection medium</B><SPAN style="color: #7fb983;">Note:</SPAN>If <B>antibiotic marker</B> is used, then outgrowth is needed:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Resuspend cells in 500ul YPD medium and incubate for precisely 30 min at 42°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>centrifuge cells at 16000 x G, 15s.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>resuspend cells on 100ul ddH2O, then spread on medium</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Preparation of Chemical Competent Cells </B> </A></LI><LI><B>in Eschericha coli</B> [4]
</LI><LI style=" padding-right: 0.3cm; padding-left: 0.5cm;color:Black;"><B>Preparation of TFB1 and TFB2 
solution;</B>
TFB1:</LI><OL type="1" style=" style=" padding-left:0.1cm=""><LI style=" padding-right: 0.3cm;"><SPAN>mix the components together:
30mM	k-Acetat
50mM 	MnCl2 X 3H2O
100mM 	RbCl2	
10mM 	CaCl2 x 2H2O
15% 	glycerin in H2O<B>pH=6.8</B>, adjusted with KOH/ HCl</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>sterile filtrate it under the clean bench</SPAN></LI></OL><LI style=" padding-right: 0.3cm; padding-left: 0.5cm;color:Black;">TFB2:</LI><OL type="1" style=" style=" padding-left:0.1cm=""><LI style=" padding-right: 0.3cm;"><SPAN>mix the components together:
10mM 	MOPS
75mM	CaCl2
10mM	RbCl2
15%	glycerin in ddH2O<B>pH=6.8</B>, adjusted with KOH/ HCl
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>sterile filtrate it under the clean bench</SPAN></LI></OL><LI style="color:Black;"><B>Making competent E. coli cells:</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare the <B>TFB1</B> and <B>TFB2</B> solution.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Inoculate 5 mL overnight culture in LB-media and incubate at 37°C, 250rpm for 14h.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Inoculate the main culture of 200 mL <B>LB-media</B> with 1-2% of the preculture (OD600 =0.1 at the beginning) in a 1 L flask and incubate it at 37°C, 250rpm until the OD600 = 0.4.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Cool down the culture on ice and centrifuge it at 4°C, 3000 rpm, for 10 min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>All following steps were carried out on ice. The pellet is resuspended in 15 mL sterile filtered cooled <B>TFB1 solution</B> and incubated for 10 min. Resuspend by gently swirling the tube, avoid to pipette the solution up and down.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge cells at 4°C, 3000 rpm for 10 min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Resuspend the pellets in 2-8 mL sterile filtered cooled <B>TFB2</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Divide competent cells in 50 µL or 100 µL aliquots in precooled Eppis and store the aliquots at -80°C immediately after freezing in liquid nitrogen</SPAN></LI><LI><B>in Saccharomyces cerevisiae</B> [5]
</LI><LI style=" padding-right: 0.3cm;padding-left: 0.3cm;"><SPAN>Material: <B>0.1 M LiOAc</B> (should be filter sterilized by using 0.2mm filter).</SPAN></LI><LI style=" padding-right: 0.3cm; padding-left: 0.5cm;color:Black;"><B> Competent Cells:</B></LI><OL type="1" style=" style=" padding-left:0.1cm=""><LI style=" padding-right: 0.3cm;"><SPAN>Prepare yeast overnight culture in <B>YPD media</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Inoculate main culture (50ml) with prepared overnight culture with OD600=0.2.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate for 4-5 h, until OD600=1 is reached.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge the culture at 2500xg for 5 min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Wash with 25 mL sterile <B>ddH2O</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Resuspend pellet in 1 mL <B>0.1 M LiOAc</B> and transfer into a 1.5 mL Eppi.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Centrifuge your sample at 10,000xg for 15 s.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Resuspend pellet in 400 µL <B>0.1 M LiOAc</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Divide the resuspended samples into 50 µL aliquots, store them at 4°C.
<SPAN style="color: #7fb983;">Note:</SPAN> Fresh made cells can be stored for up to 1 week.</SPAN></LI></OL><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Analytics</H2><OL id="category-tabs" style=" padding-left: 0.1cm; list-style-type: none;"><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Gelelectrophoresis </B> [2] </A></LI><LI style=" padding-right: 0.3cm;"><SPAN>Carefully remove the comb of the prepared solid agarose gel.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Put the gel into the gel-electrophoresis set up filled with buffer.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Pipette <B>1 µL loading dye</B> on a piece of parafilm.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add <B>5 µL</B> of the <B>sample</B> to it and mix it by pipetting it up and down (avoid bubbles).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Pipette the whole solution (6 µL), without air in the pipette tip, slowly into one well of the gel.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Pipette <B>2.5 µL ladder</B> into another well as a comparison (e.g. ladder 08: #SM0311).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Let it run for 35 min: <B>300 mA</B>, <B>90V</B>, <B>50W</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Take out the gel, analyze it and then discard the gel afterwards.</SPAN></LI></OL><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Analysis of Expression Level </B> [2] </A></LI><LI><B>Preparation of pre-culture</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Fill <B>20 mL LB media</B> into a (small) flask, add the corresponding antibiotics.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Inoculate</B> the pre-culture.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Incubate</B> your culture at <B>37°C</B>, <B>250 rpm</B> for <B>16-18 hours</B>.</SPAN></LI><LI><B>Preparation of main culture</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Fill <B>200 mL LB media</B> into a 1000 mL flask and add the corresponding antibiotics.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Inoculate</B> the main culture with <B>2-5 mL pre-culture</B>, as to reach an <B>OD</B>600 of <B>0.1</B> in the main culture.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Incubate</B> your main culture at <B>37°C</B>, <B>250 rpm</B> till it reaches the <B>OD</B>600 of <B>1</B> (2-4 hours).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Take a sample (<B>500 µL</B>) out of your main culture and store it at -20°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Induce</B> the <B>expression</B> of your main culture by adding a final concentration of each <B>0.5 / 1 / 1.5 mM IPTG</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate at <B>37°C</B>, <B>250 rpm</B> for an hour.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Take a sample <B>hourly</B> afterwards. (as described above)</SPAN></LI><LI><B>Analysis</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a <B>cell lysis</B> with the samples which were taken and frozen at -20°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a <B>Skim Milk Assay</B> with every lysed sample.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a <B>SDS-gel</B> with all lysed samples.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>SDS</B> </A></LI><LI style=" padding-right: 0.3cm;"><SPAN>An intact SDS PAGE electrophoresis system
 should include: <B>a tank, lid with power cables, electrode assembly, cell buffer dam, 
 casting stands, casting frames, combs(usually 10-well or 15-well), and glass plates (a thin glass plate and a thicker one)</B>.
The SDS PAGE gel in a single electrophoresis run can be divided into
 <B>stacking gel </B> and <B>separating gel.</B>. Stacking gel is poured on top 
 of the separating gel (after solidification) and a gel comb is inserted in the stacking gel. 
 The acrylamide percentage in SDS PAGE gel depends on the size of the target protein in the 
 sample (details shown below).</SPAN></LI><OL style="list-style-type: none; padding-left:0.1cm; padding-right:0.1cm; align-content: center;"><LI style=" padding-right: 0.3cm;color: Black;"><B>Acrylamide</B> <B>M.W. Range</B>
7%		50 kDa - 500 kDa
10%		20 kDa - 300 kDa
12%	10 kDa - 200 kDa
15%	3 kDa - 100 kDa
</LI></OL><LI style=" padding-right: 0.3cm; color: #7fb983;"><SPAN><B>Preparing the Gel</B>:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Content of separation gel</B>:
H2O  : 2.1 ml 
30% acrylamide mix     : 1.5 ml
1,5M Tris (ph 8.8)         : 1.3 ml
10% SDS                   : 0.05 ml
10% ammonium persulfate : 0.05 ml
TEMED     : 0.002 ml</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Content Stacking gel</B>
H2O                                          : 0.72 ml
30% acrylamide mix             : 0.13 ml
1,0 M Tris (ph 6,8)                : 0.13 ml
10% SDS                                 : 0.01
10% ammonium persulfate: 0.01 ml
TEMED                                    : 0.001 ml</SPAN></LI><LI style=" padding-right: 0.3cm;"><B>SDS Protocol:</B> [6]
</LI><LI style=" padding-right: 0.3cm;"><SPAN>Use one <B>thin</B>  glass plate in front of a <B>thicker</B>  one.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Put these two glass plates on a rubber sealing in the casting frame (make sure that you close the casting frame).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Proof the closeness with water.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Pour out the water and clean the device by using a papel towel.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>After water proofing of the casting frame, the separation gel (12%) could be prepared in a separate small beaker according to the protocol above.
<SPAN style="color: #7fb983;">Note:</SPAN> Ammonium Persulfate and <B> TEMED should be added at last</B>, because TEMED will solidify the solution very fast. And be noticed that TEMED smells very bad, so use it fast and close the bottle rapidly.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>After making the <B>separation gel</B> solution should be filled between the two glass plates until it reaches the mark of the casting frame.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>You can <B>straighten</B> the <B>surface </B> by <B>adding</B> a thin layer of <B>isopropanol </B> to your separation gel.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Keep the rest of your solution in the small beaker for cheaking the solidification of your separation gel. </SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>After the separation gel is solidified, the isopropanol should be discarded. Then  clean the glass plates with water and discard the warter afterwards. </SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare the <B>stacking gel</B> in a separate small beaker according to the protocol above.
<SPAN style="color: #7fb983;">Note:</SPAN> Ammonium Persulfate and <B>TEMED should be added at last</B>, because TEMED will solidify the solution very fast. And be noticed that TEMED smells very bad, so use it fast and close the bottle rapidly.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Put</B> the <B>comb</B> into the stacking gel.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Let the stacking gel <B>solidifying</B> by following the same procedure from the protocol above.<SPAN style="color: #7fb983;">Note:</SPAN> Make sure a complete gelation of the stacking gel and take out the comb.</SPAN></LI><LI style=" padding-right: 0.3cm;"><B>Running Setup</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Take the glass plates out of the casting frame and set them in the cell buffer dam.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Pour the <B>running buffer</B> (electrophoresis buffer) into the inner chamber and keep pouring after overflow untill the buffer surface reaches the required level in the outer chamber. </SPAN></LI><LI style=" padding-right: 0.3cm;"><B>Prepare the samples: </B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Mix your samples with sample buffer (loading buffer).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Heat them in boiling water for 5-10 min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><B>Loading Samples and Runing the Gel</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Load prepared samples <B>slowly</B> into wells and make sure not to overflow. Don't forget loading<B> protein marker</B> (ladder). </SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Then cover the top and connect the anodes.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Set an <B>appropriate volt</B> and run the electrophoresis when everything's done. <SPAN style="color: #7fb983;">Note:</SPAN> As for the total running time, stop SDS-PAGE running when the downmost sign of the protein marker (if no visible sign, inquire the manufacturer) almost reaches the foot line of the glass plate. Generally, about <B>1 hour</B> for a <B>120V</B> voltage and a <B>12% separating gel</B>. For a separating gel posessing higher percentage of acylamide, the time will be longer. </SPAN></LI><LI style=" padding-right: 0.3cm;"><B>Preparing gel after running</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Take out the sealed glass plates carefully.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Clean the casting frame and used devices.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Put the used running buffer into the used running buffer beaker for further usage.</SPAN></LI><LI style=" padding-right: 0.3cm;"><B>SDS gel staining-destaining procedure</B></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Fix</B> the gel with <B>destaining solution</B> for <B>10 min</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Staining</B> the gel with <B>staining solution</B> for <B>10-15 min</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform the <B> first destaining</B> with the <B>used destanng solution</B> to for <B> 15 min</B>, pour back to the used destaning solution afterwards.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform the <B>second destaining</B> with the<B> fresh destaining solution </B> and destain for <B>20-30 min</B>. Afterwards pour back to the used destaining solution bottle.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Use <B>storage solution</B> to destain for <B>1-2h</B>, then discard the solution. (<B> optional</B>)</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Skim Milk Assay</B> [7] </A></LI><LI style=" padding-right: 0.3cm; "><SPAN><B>Principle</B></SPAN></LI><LI style=" padding-right: 0.3cm; "><SPAN>Protease activity can be detected by the clearance of the skim milk. This assay is suitable for library screening in microtiterplates.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Preparation of solutions</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.1M <B>Tris/HCl buffer</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For a solution with 500mL dissolve<B>6.05g Tris</B><B>in 300mL ddH2O</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>adjust the pH to 8.6 using 1M HCl.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Fill up the mixture to a volume of 500mL with ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Substrate solution</B>:2%<B>skim milk</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Weigh <B>0.5g skim milk in a beaker</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Stir for 2 minutes</B> and <B>directly start</B> the <B>reaction</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>and add 25ml of 0.1M <B>Tris/HCl buffer</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Procedure</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Load 5-10 µL of enzyme solution in a 96-well flat bottom microtiter plate. Also use blank reactions.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add 190-195 µL of substrate solution&lt;.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate for 15-20min at 23°C MTP spectrophotometer.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Absorbance is measured at 650nm every minute (without shaking).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Clearance(650nm)=Blank(650nm)–Sample(650nm)</SPAN></LI><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Cloning</H2><OL id="category-tabs" style=" padding-left: 0.1cm; list-style-type: none;"><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Enzymatic Digestion</B> </A></LI><LI><B>Single digestion / linearisation (10 µL sample)</B></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Mix</B>:</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>2 µL plasmid DNA (~100 ng)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>1 µL restriction enzyme (e.g. EcoRI)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>1 µL buffer (10x) for enzyme (e.g. Cutsmart for EcoRI)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>6 µL ddH2O</SPAN></LI></OL><LI style=" padding-right: 0.3cm;"><SPAN><B>Incubate</B> it for minimum <B>one hour at 37°C</B>.</SPAN></LI><LI><B>Double digestion (20 µL sample)</B></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Mix</B>:</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>2 µL Enzyme buffer (19x) (e.g.CutSmart)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.4 µL restriction enzyme 1</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.4 µL restriction enzyme 2</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10 µL DNA (~200 ng)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>7.2 µL ddH2O</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Incubate</B> it for minimum <B>one hour at 37°C</B>.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Dephosphorylation</B> [8] </A></LI><LI><B>Dephosphorylation after restriction</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add <B>1 unit of rSAP</B> for every 1 pmol of DNA ends (about 1 μg of a 3 kb plasmid) after restriction (example: 1 µL for 2000bp) and according <B>buffer</B> (volume according to concentration of buffer).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate at <B>37°C</B> for <B>30–60 minutes</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Stop reaction by <B>heat-inactivation</B> of rSAP at <B>65°C</B> for <B>5 minutes</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Store at -20°C.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Cloning of BioBricks</B> </A></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Preparation</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Amplify</B> the <B>insert</B> and the <B>backbone</B>, either with a PCR or a transformation and check concentrations (backbone should have a concentration of 200 ng/µL or more, insert of 100 ng/µL or more).</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare <B>agar plates</B> and<B> SOC media</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>First single digest</B> of backbone and insert.
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Mix:
20 µL 	DNA template 
2 µL 	restriction enzyme 1
2 µL 	buffer
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate both for 1h at 37°C, a heatkill is not necessary.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>PCR clean-up</B> with additional <B>nanodrop</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Elute in the end with <B>20 µL buffer</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>A high concentration is needed for further steps.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Second single digest</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Mix:
20 µL 	DNA template 
2 µL 	restriction enzyme 2
2 µL 	buffer
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Incubate both for 1h at 37°C, a heatkill is not necessary.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>PCR clean-up</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Elute in the end with <B>20 µL buffer</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>A high concentration is needed for further steps.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Dephosphorylation</B> of backbone
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Dephosporylation of backbone with rSAP (according to protocol “Dephosphorylation”)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>PCR clean-up</B> with additional <B>Nanodrop</B> of dephosphorylated backbone
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Elute in the end with <B>20 µL buffer</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>A high concentration is needed for further steps.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Ligation</B> of dephosphorylated and purified backbone with insert
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Mix:
2.5 µL 	T4 ligase buffer
X µL 	backbone (In the end in this mix should have a mass ratio backbone to insert of <B>1:10</B>)
20 µL 	insert
2 µL 	T4 ligase
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>It is recommended to prepare a <B>control (1) </B> sample with the same composition but with water instead of the insert.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Incubate</B> the samples at <B>room temperature for one hour</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Perform a <B>heatkill</B> for <B>10min</B> at <B>65°C</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><B><SPAN>Transformation</SPAN></B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Transform the ligation mix according to <B> protocol “Transformation” </B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Recommended <B>controls</B>:<B>Control 1</B>: Transformation of dephosphorylated backbone to check if dephosphorylation worked<B>Control 2</B>: Transformation of another plasmid with same antibiotics resistance to check if transformation worked<B>Control 3</B>: Plate wild type organism to check if antibiotics are working
</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Site Directed Mutagenesis (SDM) and Site Saturation Mutagenesis (SSM) 
</B> [9] </A></LI><LI><SPAN><B>Step 1</B></SPAN></LI><LI style="padding-right:1cm;"><B>Mastermix (50 µl reaction)</B><SPAN>Set up the mastermix <B>on ice</B> in a <B>0.2ml</B> tube for PCR according to the table below. Thaw all non-enzyme components at room temperature, mix them by short vortex and collect them by short centrifugation. It is recommended to <B>add PfuS</B> DNA polymerase last in order to prevent any primer degradation caused by the 3' --&gt; 5' exonuclease activity. <B>Polymerase may be diluted in 1x buffer</B> just prior to use in order to reduce pipetting errors.</SPAN><B>Tube 1 - Forward Primer</B><TABLE style=" margin-right: 50px;"><TBODY><TR><TH style="border:1pt solid black;background-color:white;">Amount</TH><TH style="border:1pt solid black;background-color:white;">Final</TH></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>Plasmid template</B>(sequencing or column grade plasmid preparation)</TH><TD style="border:1pt solid black; padding: 2px;"><B>X µl</B> = 10-15 ng, max 50 ng</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>PDeionized water with PCR quality</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>39.5 µl-X µl</B></TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>10 x PfuS buffer</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>5 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">final 1x</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>X Forward primer (5 µM) [<SPAN style="color:#7fb983;">1</SPAN>]</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>5 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">final 400 nM</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>dNTP-mix (10 mM each)</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>1 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">Final 0.2 mM each nucleotide</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>PfuS(2U/µl) [<SPAN style="color:#7fb983;">2</SPAN>]</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>0.5 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">Final 0.02 U/ul</TD></TR></TBODY></TABLE><B>Tube 2 - Forward Primer</B><TABLE style=" margin-right: 50px;"><TBODY><TR><TH style="border:1pt solid black;background-color:white;">Amount</TH><TH style="border:1pt solid black;background-color:white;">Final</TH></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>Plasmid template</B>(sequencing or column grade plasmid preparation)</TH><TD style="border:1pt solid black; padding: 2px;"><B>X µl</B> = 10-15 ng, max 50 ng</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>PDeionized water with PCR quality</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>39.5 µl-X µl</B></TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>10 x PfuS buffer</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>5 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">final 1x</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>X Forward primer (5 µM) [<SPAN style="color:#7fb983;">1</SPAN>]</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>5 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">final 400 nM</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>dNTP-mix (10 mM each)</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>1 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">Final 0.2 mM each nucleotide</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;"><B>PfuS(2U/µl) [<SPAN style="color:#7fb983;">2</SPAN>]</B></TH><TD style="border:1pt solid black; padding: 2px;"><B>0.5 µl</B></TD><TD style="border:1pt solid black; padding: 2px;">Final 0.02 U/ul</TD></TR></TBODY></TABLE></LI><LI><B>PCR Program 1</B></LI><LI style="padding-right: 0.3cm;"><SPAN>3 cycles</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>tube control with 50 µl</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>heated lid to 105°C </SPAN></LI><TABLE style=" margin-right: 100px;"><TBODY><TR><TH style="border:1pt solid black;background-color:white;">Step</TH><TH style="border:1pt solid black;background-color:white;">Time</TH><TH style="border:1pt solid black;background-color:white;">temperature</TH><TH style="border:1pt solid black;background-color:white;">Cycles</TH></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">ID</TH><TD style="border:1pt solid black; padding: 2px;">Initial Denaturation</TD><TD style="border:1pt solid black;">30 sec</TD><TD style="border:1pt solid black;">98°C</TD><TD>3 cycles</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">D</TH><TD style="border:1pt solid black; padding: 2px;">Denaturation</TD><TD style="border:1pt solid black; padding: 2px;">10 sec [<SPAN style="color:#7fb983;">3</SPAN>]</TD><TD style="border:1pt solid black;">98°C</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">A</TH><TD style="border:1pt solid black; padding: 2px;">Annealing</TD><TD style="border:1pt solid black; padding: 2px;">15-30 sec</TD><TD style="border:1pt solid black; padding: 2px;">X°C(45-72°C) [<SPAN style="color:#7fb983;">4</SPAN>]</TD><TD>Included in the 3 cycles</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">E</TH><TD style="border:1pt solid black; padding: 2px;">Extension</TD><TD style="border:1pt solid black;"> 30-50 sec/kb [<SPAN style="color:#7fb983;">5</SPAN>]</TD><TD style="border:1pt solid black;">72°C</TD><TD>Included in the 3 cycles</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">S</TH><TD style="border:1pt solid black; padding: 2px;">Storage</TD><TD style="border:1pt solid black;">8°C</TD></TR></TBODY></TABLE><SPAN style="color:#7fb983; "><B>Note: </B></SPAN><SPAN style="padding-right:1cm;"> Extension time can be increased up to 2 min/kb if difficult template.</SPAN><LI><SPAN><B>Step 2</B></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Proceed with step 2 immediately after step 1 without freezing in between.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Mix the equal amount of foward (tube1) and reverse (tube2) reaction.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Add 1U=0.5µl PfuS DAN polymerase per 50 µl mixture.</SPAN></LI><LI><B>PCR Program 2</B></LI><LI style="padding-right: 0.3cm;"><SPAN><B>15 cycles</B></SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>tube control with 50 µl</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>105°C heated lid </SPAN></LI><TABLE style=" margin-right: 100px;"><TBODY><TR><TH style="border:1pt solid black;background-color:white;">Step</TH><TH style="border:1pt solid black;background-color:white;">Time</TH><TH style="border:1pt solid black;background-color:white;">temperature</TH><TH style="border:1pt solid black;background-color:white;">Cycles</TH></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">ID</TH><TD style="border:1pt solid black; padding: 2px;">Initial Denaturation</TD><TD style="border:1pt solid black;">30 sec</TD><TD style="border:1pt solid black;">98°C</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">D</TH><TD style="border:1pt solid black; padding: 2px;">Denaturation</TD><TD style="border:1pt solid black; padding: 2px;">10 sec [<SPAN style="color:#7fb983;">3</SPAN>]</TD><TD style="border:1pt solid black;">98°C</TD><TD rowspan="3">15 cycles</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">A</TH><TD style="border:1pt solid black; padding: 2px;">Annealing</TD><TD style="border:1pt solid black; padding: 2px;">15-30 sec</TD><TD style="border:1pt solid black; padding: 2px;">X°C(45-72°C) [<SPAN style="color:#7fb983;">4</SPAN>]</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">E</TH><TD style="border:1pt solid black; padding: 2px;">Extension</TD><TD style="border:1pt solid black;">30-50 sec/kb [<SPAN style="color:#7fb983;">5</SPAN>]</TD><TD style="border:1pt solid black;">72°C</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">FE</TH><TD style="border:1pt solid black; padding: 2px;">Final Elongation [<SPAN style="color:#7fb983;">6</SPAN>]</TD><TD style="border:1pt solid black;">5-10 min</TD><TD style="border:1pt solid black;">72°C</TD></TR><TR style="padding: 2px;"><TH scope="row" style="border:1pt solid black; background-color:white;">S</TH><TD style="border:1pt solid black; padding: 2px;">Storage</TD><TD style="border:1pt solid black;">8°C</TD></TR></TBODY></TABLE><LI style="padding-right: 1cm;"><SPAN style="color:#7fb983; "><B>Note: </B></SPAN><SPAN style="padding-right:1cm;">Extension time can be increased up to 2 min/kb and/or 20 cycles if difficult templates, 1µl of PfuS might be necessary.</SPAN></LI><LI><B>Gel and Purification</B></LI><LI style="padding-right: 0.3cm;"><SPAN>Load all samples on a 0.8% TAE agarose gel: 100V, 300 mA for 35min.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Check for optimal annealing temperature and if enough PCR product is present.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Purify the PCR products with the PCR Purification Kit. [<SPAN style="color:#7fb983;">7</SPAN>]</SPAN></LI><LI><B>Dpnl digest the PCR product </B></LI><LI style="padding-right: 0.3cm;"><SPAN> Add  <B>5 µl NEB buffer</B>and <B>1 µl  DpnI</B> to the 45 µl purified PCR product.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Incubate at least for <B>2h</B> at <B>37°C.</B> [<SPAN style="color:#7fb983;">8</SPAN>]</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Purify the PCR products with the PCR Purification Kit and elute in 30-50µl NE buffer. </SPAN></LI><LI><SPAN style="color:#7fb983;"><B>Note: </B></SPAN><SPAN>Efficiency can be increased by purification of the PCR-product before Dpnl digestion, especially when Phusion from NEB was used.</SPAN></LI><B>Transformation</B><SPAN>The purified PCR products will be transformed into competent cells. See the protocol Transformation.</SPAN><LI><B>Helpful Notes
</B><SPAN>Numbers in brackets and with green background.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Primers: Oligonucleotide primers are generally 20-40 nucleotides long and ideally have a GC-content of 40-60%. The final concentration of each in a PCR reaction may be 0.2-1mM while 0.4µM is recommended.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>PfuS DNA polymerase concentration: recommended use of 1 U per 50 µl reaction. However, the optimal concentration of PfuS DNA polymerase may vary from 0.5-2 U per 50 µl reaction depending on ampicon length and difficulty. DO not exceed 2U/50 µl reaction, especially for amplicons longer than 5kb.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Keep the denaturation as short as possible. Usually 5-10 seconds at 98°C is enough for most templates. Note: The denaturation time and temperature may vary depending on the ramp rate and temperature control mode of the cycler.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>The PfuS DNA polymerase has the ability to stabilize primer-template hybridization. As a basic rule for primers &gt;20 nt, anneal for 10-30sec at a Tm +3°C of the lower primer. For primers &lt;20 nt use an annealing temperature equal to the Tm of the lower primer. Calculate Tm for matching sequence with IDT Oligoanalyzer</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Extension time depends on ampicon length and complexity. For low complexity DNA (plasmid, lambda or BAC DNA) use extension time of 30sec per 1kb. For high complexity the extension time can be extended to 2min per 1kb.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Final Extension is an optional step which usually does not influence the outcome and can be skipped by default.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>PCR purification can be skipped, depending on the buffer which is used during PCR. Dpnl is not active in Phusion HF-buffer from NEB.</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Incubation time can be increased up to overnight for completely digestion.</SPAN></LI><H2 style="border-bottom: 5px solid #005b04;padding-left: 1.0cm;">Media</H2><OL id="category-tabs" style=" padding-left: 0.1cm; list-style-type: none;"><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>LB Medium </B> [4] </A></LI><LI><B>liquid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 1000mL of liquid LB medium <B>mix</B>:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g Peptone/ Tryptone (1%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g NaCl (1%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>5g Yeast extract (0.5%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Fill up</B> with <B>ddH2O</B> to 1000 mL.</SPAN></LI></OL><LI style=" padding-right: 0.3cm;"><SPAN><B>Autoclave</B> the mixture.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Add</B> desired <B>antibiotics</B> (under sterile conditions) when bottle is slightly <B>cooled down</B>.</SPAN></LI><LI><B>solid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 1000mL of solid LB medium <B>mix</B>:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g Peptone/ Tryptone (1%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g NaCl (1%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>5g Yeast extract (0.5%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>20g Agar</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Fill up</B> with <B>ddH2O</B> to 1000 mL.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>SC-U/SD-U Minimal Medium</B> [10] </A></LI><LI><B>liquid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 1000mL medium <B>mix</B>:

</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>6.7g yeast nitrogen base (without amino acids)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.95g amino acid mix</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>uracil</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.005% amino acid mix (aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Dissolve this mix in 800mL <B>ddH2O</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Autoclave</B> at 15 psi, 121° C for 20min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Cool to <B>50°C</B> and add 100mL of <B>filter-sterilized 20% glucose</B> or 200ml of filter-sterilized 10% <B>raffinose</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><SPAN style="color: #7fb983;">Note:</SPAN> If you are making induction medium, follow Steps 1–3 above except dissolve the reagents only in 800 ml of deionized water. Cool the medium to 50°C and add 100 ml of filter-sterilized 20% galactose and 100 ml of filter-sterilized 10% raffinose to the medium when you want to induce your cells.
</SPAN></LI><LI><B>solid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 1000mL medium <B>mix</B>:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>6.7g yeast nitrogen base (without amino acids)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.95g amino acid mix</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>uracil</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>0.005% amino acid mix (aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine)</SPAN></LI><LI><B>For SD-U:</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>63mg histidine</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>75mg tryptophan</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>250mg leucine</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>SOC Medium</B> [4] </A></LI><LI><B>liquid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 500mL solution <B>mix</B>:</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g Tryptone</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>1g	NaCl</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>2.5g 	yeast extract</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>480mL	ddH2O</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Autoclave</B> the mixture at 15psi, 121°C for 20min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare three <B>stocks</B>:</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>1.0165g MgCl2(x6H2O) in 5mL ddH2O</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>1.2324g MgSO4(x7H2O) in 5mL ddH2O</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>1.9817g glucose monohydrate in 10mL ddH2O</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Sterile filtrate</B> the solutions under sterile conditions.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Add</B> the three solutions to first 480mL mix under clean conditions as soon as the autoclaved mix is <B>cooled down</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Aliquote</B> the medium and store at -<B>20°C</B>.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>M9 Medium</B> [11] </A></LI><LI><B>liquid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare <B>seven stock solutions</B> 
for mixing the M9 medium in the end.</SPAN></LI><OL type="1" style=" padding-left:0.1cm; padding-right:0.1cm; align-content: center"><LI style=" padding-right: 0.3cm;"><SPAN><B>M9 salt solution (10x) </B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For a 1L stock dissolve in 800mL ddH2O:
75.2g 	Na2HPO4(x2H2O)
30g 	KH2PO4
5g 	NaCl
5g 	NH4Cl
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Adjust the pH value to 7.2 with NaOH.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Fill up the mixture to a final volume of 1L with ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Autoclave for 15min at 121°C.</SPAN></LI></OL><LI style=" padding-right: 0.3cm;"><SPAN><B>20% Glucose stock (w/v) </B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 500mL stock solution add 100g glucose to 440 ml ddH2O</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Autoclave for 15min at 121°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>1M MgSO4 stock</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 100mL stock solution dissolve 24.65g MgSO4(x7H2O) in 87mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Autoclave for 15min at 121°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>1M CaCl2 stock</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 100mL stock solution dissolve 14.7g CaCl2(x2H2O) in 94.5mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Autoclave for 15min at 121°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Biotin stock (1mg/mL) </B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 50mL stock mix 50mg biotin with 45mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add small aliquots of 1N NaOH until the biotin is dissolved.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add ddH2O to a final volume of 50mL.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Sterilize the solution over a 0.22µm filter.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare 1mL aliquots and store at -20°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Thiamin stock (1mg/mL) </B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For a 50mL stock mix 50mg thiamin with 50mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Sterilize the solution over a 0.22µm filter.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare 1mL aliquots and store at -20°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>100x trace elements solution</B></SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Prepare following 4 solutions for this stock solution.
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>0.1 M CuCl2</B> (1.7g/100mL): For 10mL of this solution dissolve 0.170g CuCl2(x2H2O) in 10mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>0.2 M CoCl2</B> (4.76g/100mL): For 10mL of this solution dissolve 0.476g CoCl2(x6H2O) in 10mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>0.1 M H3BO3 </B> (0.62g/100mL): For 10mL of this solution dissolve 0.062g H3BO3 in 10mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>1 M MnCl2</B> (19.8g/100mL): For 1mL of this solution dissolve 0.198g MnCl2(x4H2O) in 1mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>For a <B>1L volume of the stock solution</B> dissolve 5g EDTA in 800mL ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Adjust the pH to 7.5 with NaOH.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add:
498 mg 		FeCl3
84 mg 		ZnCl2
765 µL 		0.1M CuCl2 solution
210 µL		0.2M CoCl2 solution
1.6 mL		0.1M H3BO3 solution
8.1 µL		1M MnCl2 solution.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Fill up the mix to a final volume of 1L with ddH2O.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Sterilize the solution over a 0.22µm filter.</SPAN></LI><LI style=" padding-right: 0.3cm; "><SPAN>For <B>1L* M9 mineral medium</B> add to 867mL* sterile ddH2O:

0.3mL 		1M CaCl2 solution
100mL 		M9 salt solution (10x)
20mL 		20% glucose solution
1mL 		1M MgSO4 solution
1mL 		biotin solution (1mg/mL)
1mL 		thiamin solution (1mg/mL)
10mL 		trace elements solution</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>*For 500mL of a concentrated version of this medium (needed for preparation of M9 plates) 
add the same amounts to 367 mL of sterile ddH2O.</SPAN></LI><LI><B>solid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For preparation of 1L M9 medium for plates prepare <B>500mL concentrated liquid M9 medium</B> and warm it up to 60°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Dissolve 15g <B>agar</B> in 500mL ddH2O and autoclave this mixture for 15min at 121°C.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add the autoclaved agar solution to the M9 concentrate.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Add <B>antibiotics</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>Pour plates under sterile conditions.</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>YPD Medium</B> </A></LI><LI><B>liquid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 1000mL <B>mix</B>:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>20g peptone (1%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>20g dextrose (2%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g yeast extract (0.5%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Fill up</B> to 1000mL mark with <B>ddH2O</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Autoclave</B> the mixture at 15psi, 121°C for 20min.</SPAN></LI><LI><B>solid</B></LI><LI style=" padding-right: 0.3cm;"><SPAN>For 1000mL <B>mix</B>:
</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>10g 	peptone (1%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>20g 	Glc (2%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>5g Yeast extract (0.5%)</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>20g Agar</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Fill up</B> to 1000mL mark with <B>ddH2O</B>.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN><B>Autoclave</B>  the mixture at 15psi, 121°C for 20min.</SPAN></LI><LI style=" padding-right: 0.3cm;"><SPAN>When medium cooled down add according <B>antibiotics</B>  and make plates.</SPAN></LI><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Devices</H1></DIV><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"><OL type="1" style=" padding-left:1cm; padding-right:1cm; align-content: center"><LI style="padding-right: 0.3cm;"><SPAN>BioLector, in-house construction based on the established BioLector setup</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN> Conductivity meter </SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Flow Cytometer </SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Molecular Viewer</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN> NanoDrop</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN>Plate Reader Tecan M1000</SPAN></LI><LI style="padding-right: 0.3cm;"><SPAN> Ultra Sonicator</SPAN></LI></OL></P><DIV class="single_header_title"><H1 style="padding-left: 0.8cm;">Kits</H1></DIV><OL id="category-tabs" style=" padding-left: 0.3cm; list-style-type: none;"><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>Plasmid Isolation</B> </A></LI><LI><SPAN>NucleoSpin® Plasmid (NoLid) kit from Macherey-Nagel, 12/2015, Rev. 09,
protocol p. 16-17
</SPAN></LI></OL><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>PCR Clean-up</B> </A></LI><LI style=" padding-right: 0.3cm;"><SPAN>
NucleoSpin® Gel and PCR Clean-up kit from Macherey-Nagel, 01/2012, Rev. 02, protocol p. 18-19
</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>DNA Extraction from Agarose Gel</B> </A></LI><LI style=" padding-right: 0.3cm;"><SPAN>
NucleoSpin® Gel and PCR Clean-up kit from Macherey-Nagel, 01/2012, Rev. 02, protocol p. 20-21
</SPAN></LI><LI><A href="#" style="text-decoration: none; font-size: 18px"><B>JET Cloning</B> </A></LI><LI style=" padding-right: 0.3cm;"><SPAN>
Thermo Scientific CloneJET PCR Cloning Kit, protocol p. 5
</SPAN></LI><H2 style="border-bottom: 5px solid #005b04; padding-left: 1.0cm;">References</H2><P align="justify" style="padding-left: 1.0cm; padding-right: 1.0cm; font-size:16px;"> 

[1]&quot;Alkaline polyethylene glycol-based method for direct PCR from bacteria, eukaryotic tissue samples, and whole blood&quot; (Chomczynski and Rymaszewsk, BioTechniques 2006)
[2] Information we got during our work in the Schwaneberg and Blank Lab 
[3] http://www.koko.gov.my/CocoaBioTech/Purification%20Precipitation3.html
[4] Sambrook et al. 1989
[5] Joska et al. 2014
[6] http://www.assay-protocol.com/molecular-biology/electrophoresis/denaturing-page
[7] 2.4.5 Skim milk detection system http://publications.rwth-aachen.de/record/229540/files/4772.pdf
[8] NEB protocol for CIP alkaline phosphatase
[9] Wang, W. Y. and Malcom, B.A.: Two-stage PCR protocol allowing introduction of multiple mutations, deletions and insertions using Quikchange<SUP>TM</SUP> site-directed mutagenesis. Biotechniques, 1999. 26(4): p.680-682
[10] Thermofisher pYES2 instructions
[11]  Arie Geerlof Helmholtz Center Munich</P></DIV></DIV><DIV class=" footer_area"><DIV class=" footer"><DIV class=" single_footer_container"><DIV class=" single_footer" style="float: left;"><P style="padding-left: 1cm;"><A href="http://igem.rwth-aachen.de/" style="text-decoration:none;color:#7fb983;">Link to our website</A><A href="https://2014.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki 2014 Aachen</A><A href="https://2015.igem.org/Team:Aachen" style="text-decoration: none;color:#7fb983;"> Wiki  2015 Aachen</A><A href="#" style="text-decoration: none;color:#7fb983;">igem@rwth-aachen.de</A></P></DIV><DIV class=" single_footer" style="float: right;">                 </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>