Team:Aachen/Part Collection
Home
Project Idea
Project Idea
Improvement
Applied Design
Glossary
Lab
Lab
Lab Book
Hardware
Collaborations
Parts 
Interlab Study
Protocols/Methods
Human Practices
Human Practices
Engagement
Integrated Practices
Beyond the Basics
Safety
Achievements
Achievements
Medal Criteria
Proof of Concept
Results
Team
Members
Advisors
Supervisors
Attributions
Supporters
Welcome to iGEM Aachen 2016
Parts
BioBrick
Description
K2020000
subtilisin E gene, optimized for E. coli codon usage
K2020001
subtilisin E gene, optimized for E. coli codon usage, with leader sequence pelB
K2020002
expression system for subtilisin E in E. coli
K2020003
mutated expression system for subtilisin E in E. coli (S221Y)
K2020004
mutated expression system for subtilisin E in E. coli (S221X)
K2020005
mutated expression system for subtilisin E in E. coli (Y77W)
K2020006
mutated expression system for subtilisin E in E. coli (Y77X)
K2020026
leader sequence MFalpha (different version of the biobrick K792002 )
K2020040
screening plasmid for incorporation of non-canonical amino acids → pRXG ( twin pFRY )
K2020042
tRNA specific for tyrosine and UAG codon in E. coli
K2020043
tRNA synthetase specific for the ncAA AzF and UAG codon in E. coli → AzF-RS
K2020045
tRNA synthetase specific for the ncAA NitroY and UAG codon in E. coli → NitroY-RS
K2020046
tRNA synthetase specific for the ncAA CNF and UAG codon in E. coli → CNF-RS
K2020050
tRNA synthetase specific for tyrosine and UAG codon in E. coli → Y-RS
K2020051
mutated tRNA synthetase specific for tyrosine and UAG codon in E. coli → Y-RS with Y32G
K2020052
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 1
K2020053
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 2
K2020054
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 3
K2020055
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 4
K2020056
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 5
K2020057
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 6
K2020058
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 7
K2020059
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 8
K2020060
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 9
K2020061
tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 10
Best New Basic Part
K2020042
- tRNA specific for tyrosine and UAG codon in E. coli For incorporating non-canonical amino acids into a protein, an orthogonal tRNA/synthetase pair is needed, that does not crossreact with the cognate tRNA/synthetase pairs. This tRNA can be assembled with a variety of synthetases into a plasmid to incorporate ncAA in E. coli in response to an amber stop codon. This tRNA derives from the wild type tyrosyl Methanococcus janaschii tRNA/synthetase pair. It was proven to not crossreact with the cognate E. coli tRNA/synthetase pairs. The tRNA is used together with a tRNA synthetase. It has been proven to work with various synthetases for incorporation of ncAA by iGEM Team Austin Texas 2014
Y-RS, canonical amino acid
ONBY-RS
AzF-RS
CNF-RS
IodoY-RS
5HTP-RS
NitroY-RS
AminoY-RS
Furthermore, iGEM Team TU Darmstadt was working with this tRNA and the OMeY-synthetase. Our Team is using the tRNA in this project to incorporate the canonical amino acid tyrosine with Y-RS in response to an amber stop codon as well as an ONBY with an ONBY-RS in E. coli. Furthermore, DMNBS synthetase variants undergo screening for incorporation efficiency and fidelity, which are done by using this BioBrick. These variants are proven to work with the tRNA and are all listed in the section Parts Collection . Incorporation of ncAA: The tRNA contains an amber anticodon for incorporating the ncAA in response to a recoded amber termination codon. It has been used previously in an amberless E. coli strain C321.∆A.expb as well as in BL21 DE3 gold. When working with a recoded amber codon in BL21 DE3, the tRNA is competing with the release factor1 at the amber stop codon. As amber stop codons add up to only about 7% of total stop codos, amber suppression still works appropriately. An application of the tRNA is either the incorporation of the ncAA into a protein or usage with a reporter plasmid (for example pFRY ) for probing ncAA tRNA/synthetase pair variants regarding efficiency and fidelity. Assembly in a synthetase plasmid for incorporation of ncAA:
Figure 1:  pACYC derived plasmid with tRNA and a cognate synthetase
Most synthetases are used with low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator (fig. 1). Elements of orthogonality:
C1-G72: most important element for orthogonality, recognized by Arg174, Arg132, Met178, Lys175 within the synthetase
A73: recognized by Val195
G71: recognized by Arg132
Recognition between tRNA and ncAA-synthetases: The Methanococcus janaschii wild type tyrosyl tRNA consists of two arms: Firstly, the acceptor-minihelix, where the ncAA will be attached to the 3' end and secondly, the anticodon containg arm. Synthetases interact mainly with the acceptor minihelix of the tRNA. Due to the lack of a recognizing element within the anticodon-containing section, a mutation of an anticodon base has a comparatively small effect on the aminoacylation efficiency and may explain why a variety of ncAA can be incorporated with this tRNA. Example measurement proving incorporation of amino acids:
Figure 2:  normalized fluorescence spectrum of mRFP1
Figure 3:  normalized fluorescence spectrum of sfGFP as a sign for successful amino acid incorporation via tRNA/synthetase pair for tyrosine via amber supression
The wild type Methanococcus janaschii tRNA/synthetase pair for incorporation of tyrosine at an amber termination codon (the pair containing the tRNA mentioned above) is co-transformed with pFRY , a flourescent reporter plasmid for measurement of incorporation of ncAA into BL21 DE3 gold, which is induced by 100 µM ITPG. pFRY is one part of the two plasmid reporter system for measurement of incorporation efficiency of ncAA via amber supression. It consists of two fluorescent domains connected through a linker sequence containing an amber stop. When induced with IPTG, a the fluorescence can be measured. A red fluorescence is always visible upon induction, and if an amino acid is incorporated in response to the recoded amber stop codon, a green fluorescence intensity is also measurable. Fidelty and efficiency of the incorporation can be determined by comparison of fluorescent levels. This experiment is performed in order to obtain fluorescence spectra of mRFP1 and sfGFP. As shown above (fig. 3) GFP formation is measured as a result of successful amino acid incorporation via amber supression. Excitation and emission spectra of mRFP1 (fig. 2), and sfGFP (fig. 3) were obtained through measurements with a modified Biolector set-up. All details about this part can be seen on its Registry page .
Best New Composite Part
K2020002
– expression system for subtilisin E in E. coli This expression system consists of the promoter BBa_R0011 , the ribosome binding site BBa_B0034 , the newly created BioBrick K2020001 and the terminator BBa_B0010 . BioBrick K2020001 is a composite part itself and includes the secretion tag pelB ( BBa_J32015 ) and a subtilisin E gene optimized for Escherichia coli codon usage ( BBa_K2020000 ). Once introduced into E. coli, this BioBrick is able to produce subtilisin E, an alkaline serine protease, which non-specifically digests proteins, and simultaneously secrets the enzyme into the periplasm of the cell. Caused by the lacI regulated promoter BBa_R0010 , the expression system can be induced by addition of IPTG. With the iGEM promoter BBa_R0011 , which was integrated in our sequence at first, it was not possible to successfully express subtilisin E due to fatal mutations inside the expression system in all analyzed colonies. Either there have been single base deletions or insertions in the pro-peptide, which led to a frameshift of the whole protein, or a 23 base pair deletion in the promoter. Both types of mutations result in an incorrect expression system, so that an expression of the protease is impossible. Since the promoter BBa_R0011 is leaky and induces the expression even without addition of IPTG, it can be assumed that subtilisin E is toxic for E. coli. Hence, we exchanged the promoter against BBa_R0010 . For achieving this, we carried out a polymerase chain reaction (PCR) to extract everything but the promoter and the RBS and simultaneously extend the remaining DNA sequence with the pre-fix of iGEM. Afterwards, we assembled it with the BioBrick J04500 and in parallel cloned it into the vector pSB1C3 - by cutting RFP out of the BioBrick J04450 . The implemented BioBrick J04500 itself contains another IPTG inducible promoter ( BBa_R0010 ) and the same RBS ( BBa_B0034 ). An expression with the newly integrated promoter BBa_R0010 led to a colony with the correct sequence in opposition to our trial of gaining a positive clone while working with the first promoter BBa_R0011 . We continued our experiments by performing a skim milk assay on agar plates. Therefore, we poured LB skim milk agar plates containing IPTG and the needed antibiotic and streaked out the E. coli BL21 cells containing the plasmid with the expression system.
Figure 4: Skim milk plates assay. Cells containing the empty backbone (left) and cells containing the expression system for native subtilisin E (right) after incubation for 3 days at 30°C.
Comparing the clearance of the skim milk plates, a proteolytic activity could be proven for the cells containing the expression system for native subtilisin E. As a result, we concluded that within three days these cells are able to produce the native protease, which will then digest the skim milk in the agar plates, resulting in a clearance. In conclusion, we were able to express subtilisin E in E. coli and to prove its proteolytic activity via skim milk assay. All details about this part can be seen on its Registry page .
Parts Collection
K2020043
– tRNA synthetase specific for the ncAA AzF and UAG codon in E. coli → AzF-RS This is the 4-Azido-L-phenylalanine-synthetase to be used as an orthogonal tRNA synthetase in E. coli. It can be used in combination with the cognate tRNA BBa_K2020042 to incorporate AzF in response to an amber stop codon. Assembly in a tRNA/synthetase plasmid for incorporation of ncAA: Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator. All details about this part can be seen on its Registry page .
K2020045
– tRNA synthetase specific for the ncAA NitroY and UAG codon in E. coli → NitroY-RS This is the 3-Nitro-L-tyrosine-synthetase to be used as an orthogonal tRNA synthetase in E. coli. It can be used in combination with the cognate tRNA BBa_K2020042 to incorporate NitroY in response to an amber stop codon. All details about this part can be seen on its Registry page .
K2020046
– tRNA synthetase specific for the ncAA CNF and UAG codon in E. coli → CNF-RS This is the 4-Cyano-L-phenylalanine-synthetase to be used as an orthogonal tRNA synthetase in E. coli. It can be used in combination with the cognate tRNA BBa_K2020042 to incorporate CNF in response to an amber stop codon. All details about this part can be seen on its Registry page .
K2020050
- tRNA synthetase specific for tyrosine and UAG codon in E. coli → Y-RS This is the wild type tyrosyl synthetase derived from Methanococcus janaschii to be used as an orthogonal tRNA synthetase in E. coli. It can be used in combination with the cognate tRNA BBa_K2020042 to incorporate tyrosine in response to an amber stop codon. Incorporation of ncAA with amber codon supression: Photo-cleavable non-canonical amino acids offer the opportunity to control protein function on a non-invasive basis. Working with non-canonical amino acids requires an additional, orthogonal pair of a tRNA and a cognate synthetase i.e. which does not crossreact with the endogenous tRNA/synthetase pairs. The tRNA's anticodon contains was mutated to an amber stop anticodon. Hence, it is possible to incorporate an amino acid at a chosen position in a protein via amber codon suppression. While working with amber codon supression, the Methanococcus wild type tyrosyl-tRNA synthetase is present even in minimal media. Assembly in a synthetase plasmid for incorporation of (n)cAA: Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator. Recognition between tRNA and (n)cAA-synthetases derived from Mj: The Methanococcus janaschii wild type tyrosyl tRNA consists of two arms: Firstly, the acceptor-minihelix, where the ncAA will be attached to the 3' end and secondly, the anticodon containg arm. Synthetases interact mainly with the acceptor minihelix of the tRNA. Due to the lack of a recognizing element within the anticodon-containing section, a mutation of an anticodon base has a comparatively small effect on the aminoacylation efficiency and may explain why a variety of ncAA can be incorporated with this tRNA. Example measurement proving incorporation of amino acids: The wild type Methanococcus janaschii tRNA/synthetase pair for incorporation of tyrosine at an amber termination codon (the pair containing the tRNA mentioned above) is co-transformed with pFRY , a flourescent reporter plasmid for measurement of incorporation of ncAA into BL21 DE3 gold, which is induced by 100 µM ITPG. pFRY is one part of the two plasmid reporter system for measurement of incorporation efficiency of ncAA via amber supression. It consists of two fluorescent domains connected through a linker sequence containing an amber stop. When induced with IPTG, a the fluorescence can be measured. A red fluorescence is always visible upon induction, and if an amino acid is incorporated in response to the recoded amber stop codon, a green fluorescence intensity is also measurable. Fidelty and efficiency of the incorporation can be determined by comparison of fluorescent levels. This experiment is performed in order to obtain fluorescence spectra of mRFP1 and sfGFP. GFP formation is measured as a result of successful amino acid incorporation via amber supression. Excitation and emission spectra of mRFP1, and sfGFP were obtained through measurements with a modified Biolector set-up. All details about this part can be seen on its Registry page .
K2020051
- mutated tRNA synthetase specific for tyrosine and UAG codon in E. coli → Y-RS with Y32G This is a DMNBS-synthetase to be used as an orthogonal tRNA synthetase in E. coli. It can be used in combination with the cognate tRNA K2020042 to incorporate tyrosine in response to an amber stop codon. This synthetase has Y32 mutated to glycine and is therefore a template for further mutations for the purpose of changing amino acid specificity. Assembly in a synthetase plasmid for incorporation of ncAA: Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator. All details about this part can be seen on its Registry page .
K2020052 to K2020061
– tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 1 to 10 All of these parts are variants of a DMNBS-synthetase to be used as an orthogonal synthetase in E. coli. Each of them can be used in combination with the cognate tRNA BBa_K2020042 to incorporate DMNBS in response to an amber stop codon.
Incorporation of ncAA: Photo-cleavable non-canonical amino acids offer the opportunity to control protein function on a non-invasive basis. Working with non-canonical amino acids requires an additional, orthogonal pair of a tRNA and a cognate synthetase i.e. which does not crossreact with the endogenous tRNA/synthetase pairs. The tRNA's anticodon is mutated to an amber stop anticodon. Hence, it is possible to incorporate an amino acid at a chosen position in a protein via amber codon suppression. A previously reported tRNA/synthetase pair for O-(4,5-dimethoxy-2-nitrobenzyl)-L-serine (DMNBS), which derived from Escherichia coli and was used in Saccharomyces cerevisiae leads to the lack of a possibility to work with non-canonical amino acids replacing serine in E. coli by using a 21st amino acid. Based on computational modeling a tRNA synthetase for DMNBS in E. coli was designed by creating a semi rational mutation library. Assembly in a synthetase plasmid for incorporation of ncAA: Most synthetases are used in low copy plasmids (e.g. pACYC). The tRNA and the synthetase have to be assembled in a low copy plasmid, each one with its own promoter and only one terminator. Host organism: This reporter plasmid and the measurement of its corresponding proteins expressed are previously used both in an amberless E. coli strain and BL21 DE3 gold. The latter results in competition of the suppressor tRNA with RF1 at the amber stop codon. Screening Results:
Figure 5: DMNBS graph efficiency
If the levels of optical density of the tRNA synthetase to be evaluated and a working synthetase are equal, a first approximation of efficiency and fidelity can be made by normalizing GFP levels. Thus, one can eliminate the biogenic background fluorescence levels and compare the variants with each other. All details about these parts can be seen on their Registry pages as listed below:
BioBrick K2020052
BioBrick K2020053
BioBrick K2020054
BioBrick K2020055
BioBrick K2020056
BioBrick K2020057
BioBrick K2020058
BioBrick K2020059
BioBrick K2020060
BioBrick K2020061
Expression in E. coli
Basic Building Blocks
K2020000
– subtilisin E gene, optimized for E. coli codon usage The gene of this BioBrick can be used to express subtilisin E, which is an alkaline serine protease, which non-specifically digests proteins, in Escherichia coli. To use this part a suitable leader sequence has to be placed in front as the sequence of this BioBrick does not contain a start codon. Its sequence was originally obtained from a wild type Bacillus subtilis but was codon optimized for E. coli. All details about this part can be seen on its Registry page and in the section of the related BioBrick K2020002 .
K2020001
– subtilisin E gene, optimized for E. coli codon usage, with leader sequence pelB This composite part consists of the leader sequence pelB ( BBa_J32015 ) and the subtilisin E gene ( K2020000 ) from our first BioBrick. It can be used to express subtilisin E in Escherichia coli and simultaneously secret the enzyme into the periplasm of the cell. Subtilisin E is an alkaline serine protease, which non-specifically digests proteins. To generate a functional coding sequence that can be expressed in E. coli the leader sequence pelB, which begins with a start codon, was placed in front of the subtilisin E gene. Caused to subtilisin E’s partly toxicity for E. coli, this BioBrick should be cloned into an expression system with an inducible promoter. This should hinder the organism in the expression of the protease during its growing period. With the iGEM promoter BBa_R0011 it was not possible to successfully express subtilisin E. Hence, we exchange the promoter against BBa_R0010 . All details about this part can be seen on its Registry page and in the section of the related BioBrick K2020002 .
K2020002
– expression system for subtilisin E in E. coli Details about this part can be seen above in the section Best New Composite Part and on its Registry page .
Mutated Versions
K2020003
– mutated expression system for subtilisin E in E. coli (S221Y) The BioBrick named K2020002 for the expression of subtilisin E in E. coli is the basic component of this new part. Therefore, it consists of the promoter BBa_R0010 , the ribosome binding site BBa_B0034 , the leader sequence pelB BBa_J32015 , the newly created BioBrick K2020000 and the terminator BBa_B0010 like the expression system itself. The whole expression is based on the usage in E. coli, so the sequence of the subtilisin E gene from a wild type Bacillus subtilis was optimized for E. coli codon usage. The sequence was partly ordered from IDT ( BBa_K2020001 + BBa_B0010 ) and then cloned into BBa_J04500 , a protein expression backbone, which already carries the LacI promoter BBa_R0010 and the ribosome binding site BBa_B0034 . Afterwards, a mutation in the active site of the enzyme was introduced by performing site-directed mutagenesis (SDM). The codon AGC of serine221 was substituted with TAC, which codes for tyrosine, so serine was exchanged against tyrosine in the catalytic triade of the enzyme. We were not able to exactly detect the expression of the modified proteases via SDS gel, so we proceeded by executing a skim milk assay on agar plates containing IPTG and the needed antibiotics. Therefore, we streaked out the cells containing the modified expression systems on these plates and incubated at 30°C for three days.
Figure 6: Skim milk plates assay. Cells containing the empty backbone (1) in comparison to either cells producing the native protease (2), cells with the SDM 1-modified (3) and the SDM 3-modified expression system (4) after incubation for 3 days at 30°C.
Neither the empty backbone nor the SDM 1 modified expression system did seem to cause a proteolytic activity. A clearance and therefore a proteolytic activity could only be observed for the native protease (as demonstrated in the section of BioBrick K2020002 ). By demonstrating that this modification doesn’t result in a clearance of the skim milk plates, we were now able to prove that serine is essential for the proteolytic activity of the protease and that exchanging it would inactivate the enzyme. Hence, we demonstrated that exchanging serine against a photo-labile, non-canonical amino acid will inactivate subtilisin E and therefore proved the principle of our project. All details about this part can be seen on its Registry page .
K2020004
– mutated expression system for subtilisin E in E. coli (S221X) This part is a variation of the BioBrick K2020003 . In opposition to the exchange of serine221 against tyrosine, the named serine was exchanged against an amber codon, the least used stop codon in Escherichia coli. The codon AGC of serine221 was substituted with TAG, which codes for DMNB-serine, if the corresponding aminoacyl tRNA/synthetase pair is added. So serine will be exchanged against DMNBS in the catalytic triade of the enzyme in the presence of the needed DMNBS tRNA/synthetase pair. By irradiation with light, this non-canonical amino acid can be converted to natural serine in the active site of the enzyme and the cleaved off photo-labile group. Thereby, the protease will reach its activity just by shining light on it and will be reversibly inactivated successfully. Unfortunately, we were not able to execute all of the experiments that we planned due to a lack of time. But as our proof of principle of exchanging serine against a larger amino acid namely tyrosine worked, it can be assumed, that this part will also operate correctly. All details about this part can be seen on its Registry page .
K2020005
– mutated expression system for subtilisin E in E. coli (Y77W) The BioBrick named K2020002 for the expression of subtilisin E in E. coli is the basic component of this new part. Therefore, it consists of the promoter BBa_R0010 , the ribosome binding site BBa_B0034 , the leader sequence pelB BBa_J32015 , the newly created BioBrick K2020000 and the terminator BBa_B0010 like the expression system itself. The whole expression is based on the usage in E. coli, so the sequence of the subtilisin E gene from a wild type Bacillus subtilis was optimized for E. coli codon usage. The sequence was partly ordered from IDT ( BBa_K2020001 + BBa_B0010 ) and then cloned into BBa_J04500 , a protein expression backbone which already carries the LacI promoter BBa_R0010 and the ribosome binding site BBa_B0034 . Afterwards, a mutation between the pro-peptide and the subtilisin E gene was introduced by performing site-directed mutagenesis. The codon TAT of tyrosine77 was substituted with TGG, which codes for tryptophan, so tyrosine was exchanged against tryptophan at the connection site of pro-peptide and subtilisin E gene. Caused by the mutation in the pro-peptide and not in the direct subtilisin E gene, the actual position of the modified tyrosine was counted from the beginning of the pro-peptide compared to common annotations counted from the N-terminus of the actual enzyme. We were not able to exactly detect the expression of the modified proteases via SDS gel, so we proceeded by executing a skim milk assay on agar plates containing IPTG and the needed antibiotics. Therefore, we streaked out the cells containing the modified expression systems on these plates and incubated at 30°C for three days.
Figure 6: Skim milk plates assay. Cells containing the empty backbone (1) in comparison to either cells producing the native protease (2), cells with the SDM 1-modified (3) and the SDM 3-modified expression system (4) after incubation for 3 days at 30°C.
The empty backbone didn't cause a proteolytic activity. A clearance and therefore a proteolytic activity could be observed for the native protease (as demonstrated in the section of BioBrick K2020002 ) but also for the SDM 3 modified expression system. As SDM 3 had been executed to exchange tyrosine in the pro-peptide cleavage site against tryptophan, a proteolytic activity could be assumed caused by the clearance observed. Contrary to our former beliefs, it could now be deduced that exchanging tyrosine doesn’t result in a change of activity. Consequently, tyrosine in the pro-peptide cleavage site is not essential for the activity of subtilisin E. Unfortunately, we were not able to prove the principle of our project regarding tyrosine in the pro-peptide cleavage site, as it seems to be not essential for the proteolytic activity. Thus, exchanging tyrosine against a photo-labile, non-canonical amino acid more precisely ONB-tyrosine will not influence the activity of the enzyme. All details about this part can be seen on its Registry page .
K2020006
– mutated expression system for subtilisin E in E. coli (Y77X) This part is a variation of the BioBrick K2020005 . In opposition to the exchange of tyrosine77 against tryptophan, the named tyrosine was exchanged against an amber codon, the least used stop codon in Escherichia coli. The codon TAT of tyrosine77 was substituted with TAG, which codes for ONB-tyrosine, if the corresponding tRNA/synthetase pair is added. So tyrosine will be exchanged against ONBY in the pro-peptide cleavage site of the enzyme in the presence of the needed ONBY tRNA/synthetase pair. By irradiation with light, this non-canonical amino acid can be converted to natural tyrosine in the pro-peptide cleavage site of the enzyme and the cleaved off photo-labile group. Thereby, the protease will reach its activity just by shining light on it. Caused by the mutation in the pro-peptide and not in the direct subtilisin E gene, the actual position of the modified tyrosine was counted from the beginning of the pro-peptide compared to common annotations counted from the N-terminus of the actual enzyme. Unfortunately, we were not able to execute all of the experiments that we planned due to a lack of time. But as our proof of principle of exchanging tyrosine against a larger amino acid namely tryptophan showed no difference in the activity compared to our working subtilisin E expression system , tyrosine77 in general not influences the proteolytic activity of the enzyme. And the replacement of tyrosine against ONBY will not cause an inactivation of the protease. All details about this part can be seen on its Registry page .
Expression in S. cerevisiae
Improvement of an Existing Part
K2020026
– leader sequence MFalpha (different version of the biobrick K792002) This part is a different version of the leader sequence MFalpha ( BBa_K792002 ) for Saccharomyces cerevisiae. It can be cloned directly in front of the protein, which will be secreted. It will be relocated into the medium and the tag will be cleaved off during this process. This alpha mating factors secretion tag is naturally occurring in the genome of S. cerevisiae. Futhermore, the sequence contained an illegal restriction site, which was corrected via PCR by us. All details about this part can be seen on its Registry page .
Evolution of a New Synthetase
Screening System
K2020040
– screening plasmid for incorporation of non-canonical amino acids → pRXG This part is an improvement of the existing BioBrick K1416004 , called pFRY by the iGEM Team Austin Texas 2014. It is an flourescent reporter for measurement of incorporation of ncAA. pRXG is one part of a two plasmid reporter system for measurement of incorporation of ncAA via amber supression. It consists of two flourescent domains connected through a linker sequence containing and amber stop. When IPTG induced and expressed, the flourescence intensity can be measured. A red flourescence is always visible upon induction, and if an amino acid is incorporated as response to the amber stop codon, then a green flourescence intensity is measurable. Fidelty and efficiacy of the incorporation can be determined with comparison of flourescent level. This reporter plasmid is one of a two plasmids containing screening system for determining efficiancy and fidelty of non-canonical amino acids (ncAA) incorporation via amber termination supression. One plasmid contains tRNA and corresponding aminoacylation-synthetase. The other one is this plasmid presented herein. It consists of am mRFP1 domain which is connected through a linker sequence containing a recoded amber stop codon with a sfGFP domain. The expression of the plasmid gives either a red fluorescence, or - if the ncAA will be incorporated at the recoded amber stop codon within the linker site - both a red and a green fluorescence. Synthetase and tRNA are constitutivly expressed in a plasmid with low copy replicon, taking into account that expression results in metabolic stress, but are not under IPTG control for he purpose of avoiding abrupt and unpredictable effects considering extra time and energy needed for their assembly. Whereas the reporter plasmid containig two fluorescence proteins, is kept under operon control for IPTG induction likewise on a low copy plasmid with ColE1.
Figure 7: reporter plasmid pRXG
This reporter plasmid and the corresponding measurement of protein formation is previously used in both an amberless E.coli strain and BL21 DE3 gold. The use of the latter is resulting in competion of the supressor tRNA with release factor one at the amber stop codon at the usual 321 amber stop codons. Most synthetases are used with low copy plasmids (e.g. pACYC). The reporter should also be used with a low copy replicon. Assemble the part provided here into a low copy plasmid, make sure to use replicons from different incompatibility groups, eg. ColE1 and p15A and different selection markers. Keep under operon control for induction. All details about this part can be seen on its Registry page .
tRNA and Synthetases
K2020042
– tRNA specific for tyrosine and UAG codon in E. coli Details about this part can be seen above in the section Best Basic Part and on its Registry page .
K2020043
– tRNA synthetase specific for the ncAA AzF and UAG codon in E. coli → AzF-RS Details about this part can be seen above in the section Parts Collection and on its Registry page .
K2020045
– tRNA synthetase specific for the ncAA NitroY and UAG codon in E. coli → NitroY-RS Details about this part can be seen above in the section Parts Collection and on its Registry page .
K2020046
– tRNA synthetase specific for the ncAA CNF and UAG codon in E. coli → CNF-RS Details about this part can be seen above in the section Parts Collection and on its Registry page .
K2020050
– tRNA synthetase specific for tyrosine and UAG codon in E. coli → Y-RS Details about this part can be seen above in the section Parts Collection and on its Registry page .
K2020051
– mutated tRNA synthetase specific for tyrosine and UAG codon in E. coli → Y-RS with Y32G Details about this part can be seen above in the section Parts Collection and on its Registry page .
K2020052 to K2020061
- tRNA synthetase specific for DMNB-serine and UAG codon in E. coli → version 1 to 10 Details about these parts can be seen above in the section Parts Collection and on their Registry pages as listed below:
BioBrick K2020052
BioBrick K2020053
BioBrick K2020054
BioBrick K2020055
BioBrick K2020056
BioBrick K2020057
BioBrick K2020058
BioBrick K2020059
BioBrick K2020060
BioBrick K2020061
Link to our website
Wiki 2014 Aachen
Wiki  2015 Aachen
igem@rwth-aachen.de
                    
