<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USTC_Modeling_b"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:USTC/Modeling/b">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USTC/Modeling/b&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC/Modeling/b&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC/Modeling/b&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USTC/Modeling/b" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USTC/Modeling/b</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>An Integrated Platform Based on Bacterial Microcompartment for de novo Proteinaceous Artificial Organelles</TITLE></P><DIV id="transparent"><H2><SPAN class="mw-headline" id="Overview"> Overview </SPAN></H2><P><SPAN class="firstletter">I</SPAN>
n this section, we mainly focus on the co-evolution relationship between the <B>pdu-pdu proteins</B>.</P><P>
Based on these modeling, we want to know more about the evolution status of pdu shell microcompartment. And furthermore, we want to prove that our artificial organelle is open to a variety of signal peptide, thus making its somehow more<B> prone to contain disparate reactions</B>.
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P><B>Co-evolution of pdu-pdu proteins.</B></P><P>Use the method from previous mature algorithm <A href="http://www.ncbi.nlm.nih.gov/pubmed/10860738" class="external autonumber" rel="nofollow">[1]</A>, we check the pdu shell microcompartment proteins as interactive partners <I>in vivo</I>.
</P><H2><SPAN class="mw-headline" id="Method"> Method </SPAN></H2><P><B>1.sequence analysis</B>
Sequence related to all the components of Pdu microCompartments were retrieved using PSI-BLAST <A href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="external autonumber" rel="nofollow">[2]</A>with only 100 alignments and other parameters default. Multiple sequence alignments of them were constructed based directly on the PSI-BLAST alignments. The ClustalW phylogeny program was used to calculate a distance matrix with the neighbor-joining method<A href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="external autonumber" rel="nofollow">[3]</A>.
</P><P><B>2.Correlation analysis</B>
Distance matrices were generated from the multiple alignments using ClustalW <A href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="external autonumber" rel="nofollow">[4]</A>,we employed a linear regression analysis measuring the correlation between pairwise evolutionary distances among all proteins in a multiple sequence alignment.
We computed the linear correlation coefficient r(Pearson's correlation coefficient defined as:

with r ranges from -1 to 1, where Xbar is the mean of all X<SUB>ij</SUB> -values and Y is the mean of all Y<SUB>ij</SUB>-values. In our context, X<SUB>ij</SUB> and Y<SUB>ij</SUB> are pairwise sequence similarity distances between shell-inside components and shell components. Positive values of r would indicate a positive co-evolution; i.e. receptors that appear to be evolutionarily close, have ligands that, in turn, are more closely related than other pairs of any two ligands .By contrast, r-values of around zero would indicate no correlation, and negative values of r would indicate anti-correlation.
</P><P><B>3.Estimation of statistical significance of correlation</B>
The significance of the computed value r was assessed by calculating the r of two randomized distance matrix. We repeat this process 10000 times to get a distribution of r as figured. From the resulting 1000 values rrand,a z-score for the actual observed value r was calculated as where s is the standard deviation of rrand and rrandbar is the mean (effectively zero for truly random data)
</P><H2><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H2><P>We calculate the correlation coefficient between the shell components and the inside components. We also cut the 20 amino acids of the <B>N-terminal</B> of the inside components to see if the N-terminal of the inside components can be more correlated with the inside components. To calculate the correlation coefficient, two methods, pearson and spearman （see supplementary）are utilized. To see the results more clearly we calculated the z-score of each value and draw the heat map. From the results, we can see that the components of the shell components and inside components are co-evolved, as we expected. Also, the scores of PduK are not very high suggesting that it might be less probable to interact with N-terminal of inside components, which might be proportional to its low copies in the microcompartment.
</P><H2><SPAN class="mw-headline" id="Supplementary"> Supplementary </SPAN></H2><P>1. To see the original data of pdu-pdu co-evolution. go to <A href="https://2010.igem.org/Image:Pdu_microcompartment_co-evolution_DATA.zip" class="external autonumber" rel="nofollow">[5]</A></P><P>2. The results of spearman correlation coefficient:
</P><H2><SPAN class="mw-headline" id="Reference"> Reference </SPAN></H2><P>[1] Chern-Sing Goh et al., Co-evolution of proteins with their interaction partners, <I>Journal of Molecular Biology</I>, 2000 Jun 2; <B>299</B>(2):283-93.  
</P><P>[2] Stephen F. Altschul et al., Gapped BLAST and PSI-BLAST: a new generation of protein database search programs, <I>Nucleic Acids Research</I>, 1997; <B>25</B>(17): 3389 -3402.
</P><P>[3] N Saitou and M Nei, The neighbor-joining method: a new method for reconstructing phylogenetic trees., <I>Molecular Biology and Evolution</I>, 1987 Jul; <B>4</B>(4):406-25.
</P><P>[4] Julie D. Thompson, Desmond G. Higgins, and Toby J. Gibson, CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice, <I>Nucleic Acids Research</I>, 1994 Nov 11; <B>22</B>(22):4673-80.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:USTC/Modeling/b">http://2010.igem.org/Team:USTC/Modeling/b</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USTC/Modeling/b" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USTC/Modeling/b" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USTC/Modeling/b&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USTC/Modeling/b&amp;oldid=205812" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>