"
Page
Discussion
View source
History
teams
Log in
 
Team:Kyoto/Material and Method
From 2014.igem.org
count down
Magnetosome
DMS
Parts
Primers
Modeling
Protocol
Magnetosome
DMS
Our Question
Our Apporach
Future
Human Practice
Safety
For Judges
Acknowledgement
Sponsors
Official Team Profile
Member
Instructor & Advisor
Contact
Material & Method
Parts
Primers
Magnetosome
DMS
Modeling
Protocol
PCR
Mutation PCR
RNA Extraction
Making Competent Cells
Miniprep
Ethanol Precipitation
Electrophoresis
Gel Extraction and PCR Purification
Ligation
Restriction Enzyme Digestion
Media
Buffer TB
Solubilization of Antibiotics
Transformation
Sequence
qRT-PCR
SDS-PAGE
Western Blotting
Transmission Electron Microscopy Observation
HPLC
DMS Detecting Tube Assay
Material & Method
Parts
If you see our team's parts in part.igem.org, Click Here .
<groupparts>iGEM014 Kyoto</groupparts>
Primers
Magnetosome
mamL
mamL-AMB-F
GCAGTATTGGCGTCAGCTTGGAG
mamL-AMB-R
AACCGCTCTTCCTCATCCTTACTCAC
Xba-RBS-mamL-F
GCTCTAGAAAAGAGGAGAAAAGTGGCATGGTAAGATTGATCG
Pst-Spe-His-mamL-R
AACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGGCGCTTGATGACGATGCTC
mamQ
mamQ-AMB-F
CCGCATTTCAAGAAGGCCAATGAAC
mamQ-AMB-R
AACGAATCAGGCCCAGCACC
Xba-RBS-mamQ-F
GCTCTAGAAAAGAGGAGAAAGCGGATATGGCATTAGGCG
Pst-Spe-His-mamQ-R
AACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGTTTCTTGATGTCCTGCGCATG
mamB
mamB-AMB-F
GGGACGATCAAGGCGAAGAAGG
mamB-AMB-R
ACGGCTCAACATACGCTCTGG
Eco-Xba-RBS-mamB-F
CGGAATTCGCGGCCGCTTCTAGAGAAAGAGGAGAAAGAACCGATGAAGTTCGAAAATTGCA
Spe-mamB-R
GGACTAGTGGCTCAACATACGCTCTGG
Eco-Pcon-Xba-RBS-mamB-F
CGGAATTCTTGACGGCTAGCTCAGTCCTAGGTACAGTGCTAGCTCTAGAAAAGAGGAGAAAGAACCGATGAAGTTCGAAAATTGCA
Spe-His-mamB-R
GGACTAGTCTAGTGGTGATGGTGATGATGGACGAACAGGCGGATGTCG
mutation-mamB-F
CTGCAAACCGCCTGGCAGGTGGCCA
mutation-mamB-R
CCAGGCGGTTTGCAGCAATTCCGTATCC
Sequence
pLQB-seq-1
AGCTGCTTGCCCAGTTG
pQLB-seq-2
TTTGATGGCGGTGGTGG
pQLB-seq-3
TCGGCTACGGCAACATC
pLQB-seq-4
AGTTCGACATCCGCCTG
DMS
AT
Fw AT cloning primer 61
GGAATAGAAACAAAGTGGCAGCAACG
Rv AT cloning primer 61
GCTACCAGGAGAAGTTGGGTTGAC
AT OE primer A 59
CCGGAATTCGCGGCCGCTTCTAGATGGTATGTGTACAGTATATGATTCTCAGT
AT OE primer B 60
TCGGTCGACTACCGGGACTGAGAATCATATACTG
AT OE primer C 59
GTCCCGGTAGTCGACCGAAATGAAAACTTTG
AT OE primer D 61
AACTGCAGCGGCCGCTACTAGTAATTTGAATAAAAATAGTCCATTGCTTTAG
AT2 OE primer B 61
GTGGGGTTGTTAGGAGAACATAAATAGACGACATCG
AT2 OE primer C 60
TTATGTTCTCCTAACAACCCCACTGGTGCTGC
Rv AT cloning for His 60
CTAAAACTGCAGATCTTTCTTCATTGTAACTTTCAATCG
AT Mutation Insert Fw 55
TCCCGGTATCGACCGAAATGAAAACT
AT Mutation Insert Rv 55
CTAAAACTGCAGATCTTTCTTCATTGTAACTTTCAATCG
AT Deletion Mutation Fw 63
GCCGATGTCgTCTATTTATGTTCTCCTAACAACCCCAC
AT Deletion Mutation Rv 63
ATAGAGGACATCGGCACGGGGTAA
present from Santa
ACCTCGCTCTGCTAATCCTGTTAC
REDOX
Fw REDOX cloning primer 59
CATTCGCATCTCACATCGTCTCATAG
Rv REDOX primer Ver.2 64
CCTTCTCATTCTTAACGTGCAGGTAGG
Fw Pre-REDOX 59
CCGGAATTCGCGGCCGCTTCTAGATGTCGTCCAAAAATGTGGTTC
Rv Suf-REDOX 58
AACTGCAGCGGCCGCTACTAGTATGTATTATATTGCATTGTCCCTCC
Rv REDOX cloning for His 59
CTAAAACTGCAGTGGTATTCCATATGCTTTTTTTTG
SAMmt
Fw SAMmt cloning primer 58
ACGACGATGACGATGACGAC
Rv SAMmt cloning primer 57
AGCATGGAAGAACGCATCAG
SAMmt OE primer A 60
CCGGAATTCGCGGCCGCTTCTAGATGATTGGTGCTTCATTCTATGCT
SAMmt OE primer B 59
ACGGGTAGTCGGTACAAAAGATTTTGAATATTCTC
SAMmt OE primer C 60
ATCTTTTGTACCGACTACCCGTACATGGTCACATTCTA
SAMmt OE primer D 61
AACTGCAGCGGCCGCTACTAGTAATGGGAACAGAACATGTTAAGTCTTG
Fw SAMmt mutation PstI 62
TGTACCAGTGCAACATTCCATACC
Rv SAMmt mutation PstI 61
TGTTGCACTGGTACAATAAATAATTGTT
Rv SAMmt cloning for His 61
CTAAAACTGCAGTATTTTTTCTATAACTAAATAGGAATGGACG
dddD
F dddD cloning 66
GTTCAAGCAGGCAGGCAAG
R dddD cloning 65
AGCAACGGCACATATTCGAG
Fw Pre-dddD cloning Ver.2 68
CCGGAATTCGCGGCCGCTTCTAGATGCAGAATCGCCCGCTTTCC
Rv SpeI-dddD cloning ver3 68
CTACTAGTACGAGGCGTGGTTGCTGGTCC
Fw dddD PstI mutation 78.7
TGCAACGTGAAAGCGAAGGCGAGATCGAC
Rv dddD PstI mutation 78.4
CGCTTTCACGTTGCAGGCCCGCATC
T7-RBS-His-pSB1C3
Fw Add His to T7-RBS Ver.2 65
CATCATCACCATCACCACTAATCCGGCAAAAAAGGGCAA
Rv Add His to T7-RBS Ver.2 64
TTAGTGGTGATGGTGATGATGCTGCAGCGGCCGCTACTAG
Sequence
Sequence primer VF2 56.4
TGCCACCTGACGTCTAAGAA
Sequence primer No.1 57.9
TGATCGAGAACTTCCGCCC
Sequence primer No.2 57.3
GCGAGCAGATCGAGGACTAC
Sequence primer No.3 57.6
GAATGGGAGGAGCTGTTCG
Sequence primer No.4 56.3
TATTGGATCGACACTGGCAC
Sequence primer No.5 57.2
GGAAACACCCGAGGAACAC
Sequence primer No.6 56.4
ACTGGATGCGAGCCTTTG
SAMmt sequence No.1 56
ATTGTGAAAGCTGTTCGTGAA
AT sequence No.1
TAGTGACGGTGCTAAATGTGATA
AT sequence No.2
TGTTCCAGCGGGATTGA
REDOX sequence No.1
GTGACACCCGAGTATAACCATT
Modeling
We calculated the theoretical VOR of the E. coli group, assuming that every individual has one vesicle (i.e. the part efficiency = 100%).
We calculated the measured VOR of the pLQ and pLQB transformants from the section sliced vertically to x-axis (Represented by circule in the figure), to reduce the effect of the 3-D direction of the cells in the slice.
Finally, we determined the part efficiency from the theoretical VOR and the measured VOR to calculate the part efficiency of the negative control, pLQ, and pLQB.
The caluculation of the theoretical VOR
1. Measure transformants' height and radius and determine the values of "b" and "c".
*Assuming that the vesicle is detectable when the shortest distance of the center of modeled vesicle (a spherical shape) to the face of the slice is lower than the vesicle's radius (The value of "a" is lower than that of "b"/2).
2. Subsititute them in the fomula.( Fig. 1 )
Fig. 1 Our model of E. coli
The calcuration of the measured VOR
1. Pick up sections that are considered to be sliced vertically to x-axis.
Count the section only when the ratio to long axis and short axis of the section is lower than 2.0.( Fig. 2 )
2.  Count the sections and the number of vesicles of them.
3. Divide the number of vesicles by the number of the sections
Fig. 2 The criterion of the section counted
We count left but did't count right.
The calculation of the part efficiency
Divide the value of measured VOR by the value of theoretical VOR
Protocol
Contents
1 PCR
2 Mutation PCR
3 RNA Extraction
4 Making Competent Cells
5 Miniprep
6 Ethanol Precipitation
7 Electrophoresis
8 Gel Extraction and PCR Purification
9 Ligation
10 Restriction Enzyme Digestion
11 Media
12 Buffer TB
13 Solubilization of Antibiotics
14 Transformation
15 Sequence
16 qRT-PCR
17 SDS-PAGE
18 Western Blotting
19 Transmission Electron Microscopy Observation
20 High Performance Liquid Chromatograpy
21 DMS Detecting Tube Assay
PCR
Mix the following.
Name
Volume
25mM MgSO4
1.5 µL
2mM dNTPs
2.5 µL
10xBuffer for KOD -Plus- ver.2
2.5 µL
Template DNA
properly
Primer Forward (10 µM)
0.75 µL
Primer Reverse (10 µM)
0.75 µL
KOD -Plus-
0.5 µL
MilliQ
up to 25 µL
Total
25 µL
Put samples into Thermal Cyclers and run the following steps.
PreDenature
Denature
Anealing
Extension
cycle
94 °C
98 °C
Tm-5 °C
68 °C
--
2 min
10 sec
30 sec
1 min/kb
30 cycles
Agarose Gel Electrophoresis for confirmation.
PCR: ToYoBo Quick Taq HS DyeMix
Mix the following.
Name
Volume
Template DNA
properly
Primer Forward (10 µM)
0.2 µL
Primer Reverse (10 µM)
0.2 µL
2x Quick Taq HS DyeMix
5 µL
MilliQ
up to 10 µL
Total
10 µL
Put samples into Thermal Cyclers and run the following steps.
PreDenature
Denature
Anealing
Extension
cycle
94 °C
94 °C
Tm-5 °C
68 °C
--
2 min
30 sec
30 sec
1 min/kb
30 cycles
Agarose Gel Electrophoresis for confirmation.
Mutation PCR
Use TaKaRa PrimeSTAR Mutagenesis BasaI Kit
Dilute the concentration of template DNA with MilliQ.
Mix the following
PrimeSTAR Max Premix (2x)
25µL
Primer Forward
10pmol
Primer Reverse
10pmol
Template DNA (1ng/µL)
1~2µL
MilliQ
up to 50µL
Total
50µL
30 cycles for 10s at 98°C, for 15s 55°C, and for 40sec at 72°C.
Agalose Gel Electrophoresis for confirmation.
Negative Control: Use nothing.
Add 2L PCR products and 20L competent cells for transformation.
RNA Extraction
Use ISOGEN－LS（NIPPON GENE,311-02621
Add 1mL ISOGEN-LS to sample and vortex.
Store for 5min on ice.
Add 250µL chloroform and shake vigorously for 15 sec.
Store for 3min. on ice.
Centrifuge 17400xg for 10min. at 4°C.
Transfer aqueous phase to another tube and add 0.8 volume isopropanol.
Store for 10min. on ice.
Centrifuge 17400xg for 10min. at 4°C.
Discard the supernatant.
Add 800µL 80% ethanol and vortex.
Centrifuge 7500xg for 5min. at 4°C.
Discard the supernatant.
Dry briefly.
Dissolve in nuclease-free water.
Making Competent Cells
Streak E.coli cells on an LB plate
Allow cells to grow at 37°C overnight
Place one colony in 3 mL LB media (+antibiotic selection if necessary), grow overnight at 37°C
Take 2 ml LB media and save for blank. Transfer 500 µL overnight culture into 50 mL LB media in 500 mL flask
Allow cell to grow at 37°C (250 rpm), until OD600= 0.4 (~2-3 hours)
Place cells on ice for 30 mins
Transfer cells to a centrifuge tube (50 mL), and centrifuge cells in High Speed refrigirated centrifuge at 4°C for 10 mins at 3,000 g.
Pour off media and resuspend cells in 12 mL of cold TB.
Centrifuge cells at 4°C for 10 mins at 3,000 g (2500 rpm)
Pour supernatant and resuspend cells (by pipetting) in 5 mL of cold TB and 350 µL of DMSO. Transfer 20 µL to 1.5 mL tube
Ependorff tubes placed on ice. Freeze the cells in liquid nitrogen. Cells stored at -80°C can be used for transformation for up to ~6 months
Miniprep
Use LaboPass Plasmid Mini Purification Kit.
Pick a single colony from a freshly streaked selective plate and inoculate a culture of about 3mL Plusgrow medium containing the appropriate selective antibiotic.
Incubate at 160 rpm for 8 h at 37 °C with vigorous shaking.
Pellet the bacterial culture by centrifugation for 1 min at 15,000 g in a tabletop centrifuge.
Discard the supernatant as much as possible.
Resuspend pelleted bacterial cells in 250 µL of Buffer S1.
Transfer the suspension to a new 1.5 mL tube.
Add 250 µL of Buffer S2 and mix by inverting the tube 4 times (do not vortex).
Add 350 µL of Buffer S3 and immediately mix by inverting the tube 4-6 times (do not vortex).
Centrifuge for 10min.
Transfer carefully the supernatant to a spin column and centrufuge for 1 min.
Remove the spin column, discard the flowthrough, and re-insert the spin column to the collection tube.
Apply the 750 µL of Buffer PW and centrifuge for 1 min.
Remove the spin column, discard the flowthrough, and re-insert the spin column to the collection tube.
Discard the flowthrough, and centrifuge for an additional 1 min to remove residual wash buffer.
Transfer the spin column to a new 1.5 mL tube.
Add 30 µL of MilliQ, let stand for 1 min, and centrifuge for 1 min.
Ethanol Precipitation
Use Ethachinmate (NIPPON GENE、312-01791).
Add 3.3 µL of 3M Sodium Acetate (attached with Ethachinmate) into 100µL of DNA solution.
Add 1µL of Ethachinmate.
Vortex.
Add ethanol, 200-250µL.
Vortex.
Centrifuge at 12000xg for 5min.
Precipitation.
Electrophoresis
Prepare 200mL of a 1.0% agarose solution:
Measure 2.0g agarose into a beaker.
Add 200mL 1xTAE buffer.
Wrap the top of the beaker with plastic wrap.
Punch a hole through the wrap with a pipette tip (To let out steam).
Dissolve the agarose by heating in microwave and swirling without boiling.
Allow the agarose to cool.
Pour the agarose solution into a gel tray on a gel maker.
If there is air bubbles, pushing them with a pipette tip.
Place comb in the maker.
Cover the maker with a plastic wrap.
Let stand for about 45min.
Remove the comb carefully.
Store in the Tupperware in the refrigerator.
Place the tray in electrophoresis chamber.
Cover the tray with 1xTAE buffer.
To prepare samples for electrophoresis, add 1µL of 10x Loading Buffer for every 9µL of DNA solution and mix well.
Load 6µL of the DNA solution per well.
Electrophoresis at 100V for about 30min until Loading Buffer have migrated approximately three-quarters of the gel.
Stain the gel in ethidium bromide.
Place a plastic wrap on the transilluminator in the cabinet of Printgraph.
Place the gel on the transilluminator.
Turn on the transilluminator and confirm the position of the gel.
Shoot the picture.
Turn off the transilluminator.
Dispose of the gel.
Gel Extraction and PCR Purification
Gel Extraction
Use LMP agarose gel
Perform electrophoresis using an established protocol, using 1% LMP agarose gel.
Visualize and photograph the DNA using a long-wavelength UV lamp and an intercalating dye such as ethidium bromide. To reduce nicking, irradiate the gel for the absolute minimum time as possible. Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade.
Transfer the gel slice to a microcentrifuge tube.
If gel volume is less than 200mL, add 600mL of MilliQ. If not, add 900mL of MilliQ.
Vortex the mixture and incubate at 70°C for 20 minutes or until the gel slice is completely dissolved.
Vortex the tube every few minutes to increase the rate of agarose gel melting.
Add same volume of phenol as that of gel solution, and vortex the tube.
Centrifuge the tube in a microcentrifuge at 15,000 rpm for 5 min.
Transfer the supernatant to new microcentrifuge tube.
Centrifuge the tube in a microcentrifuge at 15,000 rpm for 5 min.
To avoid with contaminating phenol, transfer the one-eighth from the top of supernatant to new microcentrifuge tube.
Add same volume of butanol as that of solution, and vortex the tube.
Centrifuge the tube in a desktop small-sized centrifuge for few seconds.
Discard the supernatant. Repeat the extraction process by butanol until water layer reduce less than 100 µL.
Add 10 µL of 3M Sodium Acetate, 1 µL of glycogen and 250 µL of 99.5% ethanol.
Cool the tube at -30°C for 10 minutes.
Centrifuge the tube in a microcentrifuge at 15,000 rpm for 10 min.
Discard the supernatant.
Add 200 µL of 70% ethanol, and Centrifuge the tube in a microcentrifuge at 15,000 rpm for 10 min.
Discard the supernatant.
Open the cap and centrifuge the tube in a microcentrifuge at 15,000 rpm for 10 min.
Add 10 µL of MilliQ, and dissolve the DNA.
Use Promega Wizard SV Gel and PCR Clean-Up System.
Perform electrophoresis using an established protocol.
Weigh a 1.5 mlL microcentrifuge tube for each DNA fragment to be isolated and record the weight.
Visualize and photograph the DNA using a long-wavelength UV lamp and an intercalating dye such as ethidium bromide. To reduce nicking, irradiate the gel for the absolute minimum time possible. Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade.
Transfer the gel slice to the weighed microcentrifuge tube and record the weight. Subtract the weight of the empty tube from the total weight to obtain the weight of the gel slice.
Add Membrane Binding Solution at a ratio of 10 µL of solution per 10 mg of agarose gel slice.
Vortex the mixture and incubate at 52 °C for 10 minutes or until the gel slice is completely dissolved. Vortex the tube every few minutes to increase the rate of agarose gel melting. Once the agarose gel is melted, the gel will not resolidify at room temperature.
Proceed to General.
PCR Purification
Amplify target of choice using standard amplification conditions.
Add an equal volume of Membrane Binding Solution to the PCR amplification.
Proceed to General.
General
Place one SV Minicolumn in a Collection Tube for each dissolved gel slice or PCR amplification.
Transfer the dissolved gel mixture or prepared PCR product to the SV Minicolumn assembly and incubate for 1 minute at room temperature.
Centrifuge the SV Minicolumn assembly in a microcentrifuge at 16,000 g for 1 min. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube.
Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol, to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 min at 16,000 g.
Remove the SV Minicolumn assembly from the centrifuge, being careful not to wet the bottom of the column with the flowthrough. Empty the Collection Tube and recentrifuge the column assembly for 1 min with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.
Carefully transfer the SV Minicolumn to a clean 1.5 mL microcentrifuge tube. Apply 30 mL of MilliQ directly to the center of the column without touching the membrane with with the pipette tip. Incubate at room temperature for 1 min. Centrifuge for 1 min at 16,000 g.
Discard the SV Minicolumn and store the microcentrifuge tube containing the eluted DNA at 4 °C or 20 °C
Ligation
Use ToYoBo Ligation High Ver.2.
Mix the vector DNA and the insert DNA (the vector and the insert at 1 : 5-10).
Add a half volume of Ligation High Ver.2 to the DNA solution.
Incubate at 16 °C for 1 hour (or at 4 °C for overnight).
Restriction Enzyme Digestion
Use EcoRI, XbaI, SpeI, PstI (TaKaRa).
Mix the following.
Name
Volume
Sample DNA
2 µg
Restriction Enzyme
0.5µL
10x Buffer
3 µL
(If use XbaI,) 10x BSA
3 µL
MilliQ
up to 30 µL
total
30 µL
Let stand for 1 hour at 37 °C.
Use EcoRI, XbaI, SpeI, PstI (Promega).
Mix the following.
Name
Volume
Sample DNA
2 µg
Restriction Enzyme
0.5µL
10x Buffer
3 µL
100x BSA
3 µL
MilliQ
up to 30 µL
total
30 µL
Let stand for 1 hour at 37 °C.
Media
M9 medium
Stir Na2HPO4 6 g, KH2PO4 3 g, NaCl 0.5 g and NH4Cl 1 g with water 500 mL.
If you make M9 plates, add agar 15 g.
After autoclave, add 1 mL filter sterilized 1 M MgSO4, 5.6 mL 2 M glucose, 0.1 mL 1 M CaCl2, and 1 mL 1 % thiamine-HCl.
If you need, add 10 mL filter sterilized 20 % casamino acid.
LB medium
Stir BactoTryptone 2 g, Bacto-yeast extract 1 g, NaCl 1 g and 1M NaOH 200 µL with water 200mL.
If you make LB plates, add agar 2 g.
Autoclave.
SOB medium
Stir BactoTryptone 20 g and Bacto-yeast extract 5 g with water.
Add 2 mL 5 M NaCl and 840 µL 3 M KCl and add water up to 1 L.
After autoclave, add 10 mL filter sterilized 1 M MgSO4 and 1 M MgCl2.
SOC medium
Add 2 M glucose 1 mL to 100 mL SOB.
Plusgrow Medium
Stir plusgrow 20 g and with water 500 mL.
Autoclave.
YTSS medium
Stir BactoTryptone 0.5 g,Bacto-yeast extract 0.8 g and Red sea salt(Red sea) 40g with water 1000mL.
Autoclave.
Buffer TB
Stir 0.6g PIPES and 30mg CaCl2 with 10 mL water.
Add 2.5mL 2M KCl.
Add KOH and adjust pH 6.7.
Add 0.218g MnCl2・4H2O.
Add water up to 20 mL.
Filter sterilize.
Solubilization of Antibiotics
Mix the following (Final concentration is 50mg/mL).
Ampicillin
Ampicillin
1.0g
MilliQ
20mL
Kanamycin
Kanamycin
0.5g
MilliQ
10mL
Chloramphenicol
Chloramphenicol
0.5g
99.5% EtOH
10mL
Dispense 1.1mL of the solution into 1.5mL tubes.
Store in the -20°C freezer.
Transformation
Unfreeze conpetent cells on ice.
Dry a plate by letting the plate upside down and partly open in incubator.
Add 1~20µL DNA solution and 10~50µL competent cells to 1.5mL tube, let stand for 30min on ice. If few colonies are observed, increase the amount of the competent cells or DNA, but make the amount of DNA not to get over that of the competent cells.
Heatshock for 45s at 42°C.
Let stand for 2min on ice.
Culture for 1h in preculture medium (LB or SOC medium), and plate by using spreader. Do not heat spreader too much because E.coli will dead for heat.
Culture overnight at 37°C.
Sequence
Use Big Dye Terminator 3.1（ABI）
Mix the following
5xBuffer
1.75µL
Primer (3.2µM)
0.5µL
Template Plasmid
400ng
Big Dye Terminator 3.1
0.5µL
MilliQ
up to 10.5µL
Let stand for 1min at 96°C.
30 cycles for 5s at 98°C, for 5s 50°C, and for 4min at 68°C.
Add 25µL 100% ethanol and 1µL NAOAC
qRT-PCR
Use QuantiTect SYBR green PCR kit Cat. No. 204143 by QIAGEN
Dilute primer to 1.5µM.
Dilute RT products.
Mix the following;
2x QuantiTect SYBR Green PCR Master Mix
22.5µL
1/20xRT products
4.5µL
MilliQ
9µL
total
36µL
Mix the reaction mix thoroughly, and dispense 36µL into 96 wells plate.
Add primer set 9µL.
Mix by inverting and voltex.
Dispense 10µL into 384 wells plate and centrifuge.
Let stand for 2min at 50°C and for 15min for 95°C.
40 cycles for 15sec at 95°, for 30sec at 60°, and for 1min at 72°C.
Let stand for 15sec at 95°.
SDS-PAGE
Make the Running gel
Mix the following
20% gel (Running gel) for 1gel
30% AcrylAmide
5.3mL
1.5M Tris (pH8.8)
2mL
MilliQ
500mL
10% SDS
80µL
10% APS
80µL
TEMED
4.8µL
Total
8mL
Insert Isopropanol on the running gel.
Let stand for about 20 min
Make the Stacking Gel
Mix the following
Stacking Gel for 1gel
30% AcrylAmide
0.41mL
1.5M Tris (pH8.8)
0.21mL
MilliQ
18.2mL
10% SDS
25µL
10% APS
25µL
TEMED
2.5µL
Total
2.5mL
Discard the Isopropanol
Place the stacking gel on the running gel
Place the comb, 12 well
Let stand for 20 min
Load the gel into the phoresis tank
Fill 1xSDS buffer into the phoresis tank
If there is air bubbles, pushing them with a syringe.
Load the proper amount of protein sample solution per well
Electrophoresis at 25mA for about 60 min
Western Blotting
Soak the filter paper and the gel in transfer-buffer.
Soak the Membrane in Methanol and in transfer-buffer.
Put 3 filter paper, membrane, gel, 3 filter paper on the transcription machinary.
Transcript at 200 mA for 73 min.
Wash the membrane by TBS-T buffer.
Let shake for about 10 min 3 times.
Block the Membrane for about 60 min.
The Blocking reagent is "DS Pharma Biomedical Co..Ltd, Block Ace Powder."
Let shake for 60 min.
Antibody is "MBL,Anti-His-tag mAb-HRP-DirecT"
Make the antibody 5000 times dilution by TBS-T buffer.
Mix the antibody with membrane.
Let shake for 60 min.
Wash the membrane by TBS-T buffer.
Let shake for about 10 min.
Place the membrane on the Western Lightening.
Shoot the picture.
Dispose the membrane.
Transmission Electron Microscopy Observation
Ultramicrotome -> LEICA ULTRACUT UCT
Transmission electron microscopy -> JEM-1005X
Prefix solution
Final Conc.
Stock Sol.
Glutaraldehyde (20 %)
2 %
1.25 mL
Phosphate buffer (pH 7.4)
0.05 M
5 mL
DW
3.75 mL
Total
10 mL
0.1 M Phosphate buffer (pH 7.4)
NaH2PO4·H2O
1.38 g
NaH2PO4·H2O
14.34 g
DW
up to 500 mL
Postfix solution
Final Conc.
Stock Sol.
osmic acid
1 %
0.5 mL
Phosphate buffer
0.05 M
0.5 mL
Total
1 mL
TAAB EPON812 RESIN KIT
TAAB EPON812 RESIN
24 g
TAAB MNA
20 g
TAAB DDSA
6 g
TAAB DMP-30
1 g
(Deaerate for 15 minute by Vacuum pumps)
lead citrate (Reynold method)
Measure 1.33g lead nitrate and 1.76g sodium citrate into a beaker.
Add 30mL DW and shake hard for 1 minute.
Let stand the beaker for 30 minute as shaking the tube every few minutes.
Add 8mL 1N-NaOH and DW up to 50mL.
1. Pick a single colony from a freshly streaked selective plate and inoculate a culture of about 4mL LB medium containing the appropriate selective antibiotic. Incubate at 160 rpm at 37 °C with vigorous shaking until reaching appropriate concentration.
2. Transfer the 1.5mL cultured medium to microcentrifuge tube.
3. Pellet the bacterial culture by centrifugation for 10 min at 3,000 g in a tabletop centrifuge.
4. Discard the supernatant as much as possible.
5. Add the 1.2mL prefix solution, and resuspent by pipetting.
6. Incubate at 4 °C overnight.
7. Rinse (Centrifugation: 10 min, 3,000 g and resuspention) by 0.1M Phosphate buffer twice.
8. Add 50 µL postfix solution, and incubate on ice for 2 hours.
9. Discard the postfix solution, and add 50 µL 50% ethanol. Incubate on ice for 15 min.
10. Replace the solution with 50 µL 70% ethanol. Incubate at room temperature for 15 min.
11.  Replace the solution with 50 µL 90% ethanol. Incubate at room temperature for 15 min.
12. Replace the solution with 50 µL 95% ethanol. Incubate at room temperature for 15 min.
13. Replace the solution with 50 µL 99.5% ethanol. Incubate at room temperature for 15 min.
14. Replace the solution with 50 µL 100% ethanol (using molecular sieve). Incubate at room temperature for 15 min.
15. Replace the solution with 50 µL 100% ethanol . Incubate at room temperature for 15 min.
16. Replace the solution with 50 µL propylene oxide. Incubate at room temperature for 15 min.
17. Replace the solution with 50 µL propylene oxide. Incubate at room temperature for 15 min.
18. Replace the solution with 50 µL half and half mixture of propylene oxide and Epon 812 Resin Kit mixed liquor. Incubate at room temperature for 1 hour.
19. Replace the solution with 50 µL Epon 812 Resin Kit mixed liquor. Incubate in the desiccator at 4°C overnight.
20. Incubate at 60°C for 2 days.
21. Trim the sample into 0.5~2mm square and slice into 1 micro meter thick by a glass knife.
22. Observe the sections, and trim smaller to make ultra-thin section.
23. To make the smooth aspect, slice into 1 micro meter thick by a new glass nife.
24. Slice into 90~100nm thick by diamond knife.
25. Collect the section by collodion coated mesh (VECO GRID ,NISSIN EM ,Japan).
26. Stain the sample by 1% uranyl acetate for 30 min.
27. Stain the sample by lead citrate for 3 min.
28. Observe by TEM, and take the photograph.
High Performance Liquid Chromatograpy
Culture
1. Cultivate E.coli in LB medium till OD600 becomes 0.8. LB contains 1% Methionine and 1mM 2-oxoglutarate.
2. When OD600 become 0.8, add IPTG to medium for its density become 10mM
3.Cultivate E.coli for 14 hours.
HPLC
1. Centrifuge 1mL of culture of E.coli 120rpm,2min to pelletize cells.
2. Add 250μL of Lysis Buffer(50mM Tris, 1mM EDTA, 1mM DTT).
3. Sonicate 30sec 3times.
4. Centrifuge 15000rpm 15min at 4℃.
5. Apply the supernatant to 3K cut off membrane(Amicon Ultra-0.5 mL Centrifugal Filters for Protein Purification and Concentration,Merck Millipore)15000rpm 30min at 4℃.
6. 10 fold dilute and add equivalent DMB Solution(composition will show below;) and incubate 2h30min at 50℃.
DMB Solution
3.15mg DMB
160.8μL Acetic acid
105.6μL 2-mercaptoethanol
18μL 2M Na-hydrosulfonide
641.6μL Distilled water
7. 10 fold dilute and inject 3μL of the sample into column. HPLC condition will show below;
HPLC: SHIMADZU LC-10A
Column: ZORBAX Eclipse® XDB-C18
Total Flow: 0.5mL/min
Gradient: Pump A: acetonitrile: MeOH: H2O=9:7:84  Pump B: 50:7:43
0:01 B Percentage=5%
0:01 Total flow=0.5mL/min
0:01 Start
20:00 B Percentage=20%
25:00 B Percentage=100%
30:00 B Percentage=100%
35:00 B Percentage=5%
45:00 Stop
DMS Detecting Tube Assay
1.Cultivate E.coli in LB medium for  hours.
In the experimental, medium contains 1mM IPTG, 1nM DMSP, and E.coli which was transformed DddD generator.
In the positive control, medium contains 1mM IPTG, 1nM DMSP, 1mM DMS and E.coli which was transformed DddD generator.
In the negative control 1, medium contains 1mM IPTG, 1mM DMSP and E.coli which wasn't transformed.
In the negative control 2, medium contains 1mM IPTG and E.coli which was transformed DddD generator.
In the negative control 3, medium contains 1mM DMSP and E.coli which was transformed DddD generator.
Falcon tube in which samples incubated was sealed by Parafilm.
2.After cultivating E.coli, sample 30mL vapor in the syringe and inject it in DMS Detector tube.
We used Tokyo Gas Engineering Co.,Ltd, DMS Detecting tube .
Retrieved from " http://2014.igem.org/Team:Kyoto/Material_and_Method "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
