<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kyoto_Material_and_Method"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Kyoto/Material_and_Method">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kyoto/Material_and_Method&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/Material_and_Method&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/Material_and_Method&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kyoto/Material_and_Method" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kyoto/Material and Method</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="kyoto-alert"><DIV id="kyoto-alert-display">count down</DIV></DIV><DIV class="kyoto-centering-wrapper" id="kyoto-menu-centering"><DIV class="kyoto-nav"><UL class="kyoto-menu"><LI><A href="https://2014.igem.org/Team:Kyoto/Project/Magnetosome_Formation#title">Magnetosome</A></LI><LI class="kyoto-menu-bottom"><A href="https://2014.igem.org/Team:Kyoto/Project/DMS_Synthesis#title">DMS</A></LI></UL><LI><A href="https://2014.igem.org/Team:Kyoto/Material_and_Method#parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Material_and_Method#primers">Primers</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Material_and_Method#modeling">Modeling</A></LI><LI class="kyoto-menu-bottom"><A href="https://2014.igem.org/Team:Kyoto/Material_and_Method#protocol">Protocol</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Notebook/Magnetosome_Formation#title">Magnetosome</A></LI><LI class="kyoto-menu-bottom"><A href="https://2014.igem.org/Team:Kyoto/Notebook/DMS_Synthesis#title">DMS</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Policy_and_Practice#question">Our Question</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Policy_and_Practice#approach">Our Apporach</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Policy_and_Practice#future">Future</A></LI><LI class="kyoto-menu-bottom"><A href="https://2014.igem.org/Team:Kyoto/Policy_and_Practice#human_practice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Consideration#safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Consideration#for_judges">For Judges</A></LI><LI style="font-size: 15px;"><A href="https://2014.igem.org/Team:Kyoto/Consideration#acknowledgement">Acknowledgement</A></LI><LI class="kyoto-menu-bottom"><A href="https://2014.igem.org/Team:Kyoto/Consideration#sponsors">Sponsors</A></LI><LI style="font-size: 14px;"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Kyoto">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Team#member">Member</A></LI><LI style="font-size: 14px;"><A href="https://2014.igem.org/Team:Kyoto/Team#instructor">Instructor &amp; Advisor</A></LI><LI class="kyoto-menu-bottom"><A href="https://2014.igem.org/Team:Kyoto/Team#contact">Contact</A></LI></DIV></DIV><DIV id="kyoto-container"><DIV id="kyoto-side_menu"><UL style="width: 180px;"><LI><A id="kyoto-side-title" href="#title">Material &amp; Method</A></LI><LI><A href="#parts">Parts</A></LI><LI><A href="#primers">Primers</A></LI><LI><A href="#partsmagnet">Magnetosome</A></LI><LI><A href="#partsdms">DMS</A></LI></UL><LI><A href="#modeling">Modeling</A></LI><LI><A href="#protocol">Protocol</A></LI><LI><A href="#pcr">PCR</A></LI><LI><A href="#mutation_pcr">Mutation PCR</A></LI><LI><A href="#rna_extraction">RNA Extraction</A></LI><LI><A href="#competent_cells">Making Competent Cells</A></LI><LI><A href="#miniprep">Miniprep</A></LI><LI><A href="#ethanol_precipitation">Ethanol Precipitation</A></LI><LI><A href="#electrophoresis">Electrophoresis</A></LI><LI><A href="#gel_extraction">Gel Extraction and PCR Purification</A></LI><LI><A href="#ligation">Ligation</A></LI><LI><A href="#digestion">Restriction Enzyme Digestion</A></LI><LI><A href="#media">Media</A></LI><LI><A href="#buffer_tb">Buffer TB</A></LI><LI><A href="#solubilization_of_antibiotics">Solubilization of Antibiotics</A></LI><LI><A href="#transformation">Transformation</A></LI><LI><A href="#sequence">Sequence</A></LI><LI><A href="#qrt-pcr">qRT-PCR</A></LI><LI><A href="#sds-page">SDS-PAGE</A></LI><LI><A href="#western_blotting">Western Blotting</A></LI><LI><A href="#tem">Transmission Electron Microscopy Observation</A></LI><LI><A href="#hplc">HPLC</A></LI><LI><A href="#dms-detecting">DMS Detecting Tube Assay</A></LI></DIV><DIV id="kyoto-content_page"><H1 id="kyoto-title">Material &amp; Method</H1><H2>Parts</H2><P>If you see our team's parts in part.igem.org, <A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2014&amp;group=Kyoto">Click Here</A>.</P>&lt;groupparts&gt;iGEM014 Kyoto&lt;/groupparts&gt;<H2>Primers</H2><H3>Magnetosome</H3><TABLE class="kyoto-primers"><TBODY><TR><TH colspan="2">mamL</TH></TR><TR><TD>mamL-AMB-F</TD><TD>GCAGTATTGGCGTCAGCTTGGAG</TD></TR><TR><TD>mamL-AMB-R</TD><TD>AACCGCTCTTCCTCATCCTTACTCAC</TD></TR><TR><TD>Xba-RBS-mamL-F</TD><TD>GCTCTAGAAAAGAGGAGAAAAGTGGCATGGTAAGATTGATCG</TD></TR><TR><TD>Pst-Spe-His-mamL-R</TD><TD>AACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGGCGCTTGATGACGATGCTC</TD></TR><TR><TH colspan="2">mamQ</TH></TR><TR><TD>mamQ-AMB-F</TD><TD>CCGCATTTCAAGAAGGCCAATGAAC</TD></TR><TR><TD>mamQ-AMB-R</TD><TD>AACGAATCAGGCCCAGCACC</TD></TR><TR><TD>Xba-RBS-mamQ-F</TD><TD>GCTCTAGAAAAGAGGAGAAAGCGGATATGGCATTAGGCG</TD></TR><TR><TD>Pst-Spe-His-mamQ-R</TD><TD>AACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGTTTCTTGATGTCCTGCGCATG</TD></TR><TR><TH colspan="2">mamB</TH></TR><TR><TD>mamB-AMB-F</TD><TD>GGGACGATCAAGGCGAAGAAGG</TD></TR><TR><TD>mamB-AMB-R</TD><TD>ACGGCTCAACATACGCTCTGG</TD></TR><TR><TD>Eco-Xba-RBS-mamB-F</TD><TD>CGGAATTCGCGGCCGCTTCTAGAGAAAGAGGAGAAAGAACCGATGAAGTTCGAAAATTGCA</TD></TR><TR><TD>Spe-mamB-R</TD><TD>GGACTAGTGGCTCAACATACGCTCTGG</TD></TR><TR><TD>Eco-Pcon-Xba-RBS-mamB-F</TD><TD style="font-size: 0.8em;">CGGAATTCTTGACGGCTAGCTCAGTCCTAGGTACAGTGCTAGCTCTAGAAAAGAGGAGAAAGAACCGATGAAGTTCGAAAATTGCA</TD></TR><TR><TD>Spe-His-mamB-R</TD><TD>GGACTAGTCTAGTGGTGATGGTGATGATGGACGAACAGGCGGATGTCG</TD></TR><TR><TD>mutation-mamB-F</TD><TD>CTGCAAACCGCCTGGCAGGTGGCCA</TD></TR><TR><TD>mutation-mamB-R</TD><TD>CCAGGCGGTTTGCAGCAATTCCGTATCC</TD></TR><TR><TH colspan="2">Sequence</TH></TR><TR><TD>pLQB-seq-1</TD><TD>AGCTGCTTGCCCAGTTG</TD></TR><TR><TD>pQLB-seq-2</TD><TD>TTTGATGGCGGTGGTGG</TD></TR><TR><TD>pQLB-seq-3</TD><TD>TCGGCTACGGCAACATC</TD></TR><TR><TD>pLQB-seq-4</TD><TD>AGTTCGACATCCGCCTG</TD></TR></TBODY></TABLE><H3>DMS</H3><TABLE class="kyoto-primers"><TBODY><TR><TH colspan="2">AT</TH></TR><TR><TD>Fw AT cloning primer 61</TD><TD>GGAATAGAAACAAAGTGGCAGCAACG</TD></TR><TR><TD>Rv AT cloning primer 61</TD><TD>GCTACCAGGAGAAGTTGGGTTGAC</TD></TR><TR><TD>AT OE primer A 59</TD><TD>CCGGAATTCGCGGCCGCTTCTAGATGGTATGTGTACAGTATATGATTCTCAGT</TD></TR><TR><TD>AT OE primer B 60</TD><TD>TCGGTCGACTACCGGGACTGAGAATCATATACTG</TD></TR><TR><TD>AT OE primer C 59</TD><TD>GTCCCGGTAGTCGACCGAAATGAAAACTTTG</TD></TR><TR><TD>AT OE primer D 61</TD><TD>AACTGCAGCGGCCGCTACTAGTAATTTGAATAAAAATAGTCCATTGCTTTAG</TD></TR><TR><TD>AT2 OE primer B 61</TD><TD>GTGGGGTTGTTAGGAGAACATAAATAGACGACATCG</TD></TR><TR><TD>AT2 OE primer C 60</TD><TD>TTATGTTCTCCTAACAACCCCACTGGTGCTGC</TD></TR><TR><TD>Rv AT cloning for His 60</TD><TD>CTAAAACTGCAGATCTTTCTTCATTGTAACTTTCAATCG</TD></TR><TR><TD>AT Mutation Insert Fw 55</TD><TD>TCCCGGTATCGACCGAAATGAAAACT</TD></TR><TR><TD>AT Mutation Insert Rv 55</TD><TD>CTAAAACTGCAGATCTTTCTTCATTGTAACTTTCAATCG</TD></TR><TR><TD>AT Deletion Mutation Fw 63</TD><TD>GCCGATGTCgTCTATTTATGTTCTCCTAACAACCCCAC</TD></TR><TR><TD>AT Deletion Mutation Rv 63</TD><TD>ATAGAGGACATCGGCACGGGGTAA</TD></TR><TR><TD>present from Santa</TD><TD>ACCTCGCTCTGCTAATCCTGTTAC</TD></TR><TR><TH colspan="2">REDOX</TH></TR><TR><TD>Fw REDOX cloning primer 59</TD><TD>CATTCGCATCTCACATCGTCTCATAG</TD></TR><TR><TD>Rv REDOX primer Ver.2 64</TD><TD>CCTTCTCATTCTTAACGTGCAGGTAGG</TD></TR><TR><TD>Fw Pre-REDOX 59</TD><TD>CCGGAATTCGCGGCCGCTTCTAGATGTCGTCCAAAAATGTGGTTC</TD></TR><TR><TD>Rv Suf-REDOX 58</TD><TD>AACTGCAGCGGCCGCTACTAGTATGTATTATATTGCATTGTCCCTCC</TD></TR><TR><TD>Rv REDOX cloning for His 59</TD><TD>CTAAAACTGCAGTGGTATTCCATATGCTTTTTTTTG</TD></TR><TR><TH colspan="2">SAMmt</TH></TR><TR><TD>Fw SAMmt cloning primer 58</TD><TD>ACGACGATGACGATGACGAC</TD></TR><TR><TD>Rv SAMmt cloning primer 57</TD><TD>AGCATGGAAGAACGCATCAG</TD></TR><TR><TD>SAMmt OE primer A 60</TD><TD>CCGGAATTCGCGGCCGCTTCTAGATGATTGGTGCTTCATTCTATGCT</TD></TR><TR><TD>SAMmt OE primer B 59</TD><TD>ACGGGTAGTCGGTACAAAAGATTTTGAATATTCTC</TD></TR><TR><TD>SAMmt OE primer C 60</TD><TD>ATCTTTTGTACCGACTACCCGTACATGGTCACATTCTA</TD></TR><TR><TD>SAMmt OE primer D 61</TD><TD>AACTGCAGCGGCCGCTACTAGTAATGGGAACAGAACATGTTAAGTCTTG</TD></TR><TR><TD>Fw SAMmt mutation PstI 62</TD><TD>TGTACCAGTGCAACATTCCATACC</TD></TR><TR><TD>Rv SAMmt mutation PstI 61</TD><TD>TGTTGCACTGGTACAATAAATAATTGTT</TD></TR><TR><TD>Rv SAMmt cloning for His 61</TD><TD>CTAAAACTGCAGTATTTTTTCTATAACTAAATAGGAATGGACG</TD></TR><TR><TH colspan="2">dddD</TH></TR><TR><TD>F dddD cloning 66</TD><TD>GTTCAAGCAGGCAGGCAAG</TD></TR><TR><TD>R dddD cloning 65</TD><TD>AGCAACGGCACATATTCGAG</TD></TR><TR><TD>Fw Pre-dddD cloning Ver.2 68</TD><TD>CCGGAATTCGCGGCCGCTTCTAGATGCAGAATCGCCCGCTTTCC</TD></TR><TR><TD>Rv SpeI-dddD cloning ver3 68</TD><TD>CTACTAGTACGAGGCGTGGTTGCTGGTCC</TD></TR><TR><TD>Fw dddD PstI mutation 78.7</TD><TD>TGCAACGTGAAAGCGAAGGCGAGATCGAC</TD></TR><TR><TD>Rv dddD PstI mutation 78.4</TD><TD>CGCTTTCACGTTGCAGGCCCGCATC</TD></TR><TR><TH colspan="2">T7-RBS-His-pSB1C3</TH></TR><TR><TD>Fw Add His to T7-RBS Ver.2 65</TD><TD>CATCATCACCATCACCACTAATCCGGCAAAAAAGGGCAA</TD></TR><TR><TD>Rv Add His to T7-RBS Ver.2 64</TD><TD>TTAGTGGTGATGGTGATGATGCTGCAGCGGCCGCTACTAG</TD></TR><TR><TH colspan="2">Sequence</TH></TR><TR><TD>Sequence primer VF2 56.4</TD><TD>TGCCACCTGACGTCTAAGAA</TD></TR><TR><TD>Sequence primer No.1 57.9</TD><TD>TGATCGAGAACTTCCGCCC</TD></TR><TR><TD>Sequence primer No.2 57.3</TD><TD>GCGAGCAGATCGAGGACTAC</TD></TR><TR><TD>Sequence primer No.3 57.6</TD><TD>GAATGGGAGGAGCTGTTCG</TD></TR><TR><TD>Sequence primer No.4 56.3</TD><TD>TATTGGATCGACACTGGCAC</TD></TR><TR><TD>Sequence primer No.5 57.2</TD><TD>GGAAACACCCGAGGAACAC</TD></TR><TR><TD>Sequence primer No.6 56.4</TD><TD>ACTGGATGCGAGCCTTTG</TD></TR><TR><TD>SAMmt sequence No.1 56</TD><TD>ATTGTGAAAGCTGTTCGTGAA</TD></TR><TR><TD>AT sequence No.1</TD><TD>TAGTGACGGTGCTAAATGTGATA</TD></TR><TR><TD>AT sequence No.2</TD><TD>TGTTCCAGCGGGATTGA</TD></TR><TR><TD>REDOX sequence No.1</TD><TD>GTGACACCCGAGTATAACCATT</TD></TR></TBODY></TABLE><H2>Modeling</H2><P>We calculated the theoretical VOR of the <I>E. coli</I> group, assuming that every individual has one vesicle (<I>i.e.</I> the part efficiency = 100%).</P><P>We calculated the measured VOR of the pLQ and pLQB transformants from the section sliced vertically to x-axis (Represented by circule in the figure), to reduce the effect of the 3-D direction of the cells in the slice.</P><P>Finally, we determined the part efficiency from the theoretical VOR and the measured VOR to calculate the part efficiency of the negative control, pLQ, and pLQB.</P><H4>The caluculation of the theoretical VOR</H4><P>1. Measure transformants' height and radius and determine the values of &quot;b&quot; and &quot;c&quot;.</P><P>*Assuming that the vesicle is detectable when the shortest distance of the center of modeled vesicle (a spherical shape) to the face of the slice is lower than the vesicle's radius (The value of &quot;a&quot; is lower than that of &quot;b&quot;/2).</P><P>2. Subsititute them in the fomula.(<A href="#fig1">Fig. 1</A>)</P><FIGURE><FIGCAPTION><SPAN class="kyoto-fig-title">Fig. 1 Our model of <I>E. coli</I></SPAN></FIGCAPTION></FIGURE><H4>The calcuration of the measured VOR</H4><P>1. Pick up sections that are considered to be sliced vertically to x-axis. </P><P>Count the section only when the ratio to long axis and short axis of the section is lower than 2.0.(<A href="#fig2">Fig. 2</A>)</P><P>2.  Count the sections and the number of vesicles of them.</P><P>3. Divide the number of vesicles by the number of the sections</P><FIGURE><FIGCAPTION><SPAN class="kyoto-fig-title">Fig. 2 The criterion of the section counted</SPAN>We count left but did't count right.</FIGCAPTION></FIGURE><H4>The calculation of the part efficiency</H4><P>Divide the value of measured VOR by the value of theoretical VOR</P><H2>Protocol</H2><DIV id="kyoto-protocol"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#PCR"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#Mutation_PCR"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Mutation PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#RNA_Extraction"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">RNA Extraction</SPAN></A></LI><LI class="toclevel-1"><A href="#Making_Competent_Cells"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Making Competent Cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Miniprep"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Miniprep</SPAN></A></LI><LI class="toclevel-1"><A href="#Ethanol_Precipitation"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Ethanol Precipitation</SPAN></A></LI><LI class="toclevel-1"><A href="#Electrophoresis"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-1"><A href="#Gel_Extraction_and_PCR_Purification"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Gel Extraction and PCR Purification</SPAN></A></LI><LI class="toclevel-1"><A href="#Ligation"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1"><A href="#Restriction_Enzyme_Digestion"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Restriction Enzyme Digestion</SPAN></A></LI><LI class="toclevel-1"><A href="#Media"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Media</SPAN></A></LI><LI class="toclevel-1"><A href="#Buffer_TB"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Buffer TB</SPAN></A></LI><LI class="toclevel-1"><A href="#Solubilization_of_Antibiotics"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Solubilization of Antibiotics</SPAN></A></LI><LI class="toclevel-1"><A href="#Transformation"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1"><A href="#Sequence"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Sequence</SPAN></A></LI><LI class="toclevel-1"><A href="#qRT-PCR"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">qRT-PCR</SPAN></A></LI><LI class="toclevel-1"><A href="#SDS-PAGE"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">SDS-PAGE</SPAN></A></LI><LI class="toclevel-1"><A href="#Western_Blotting"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Western Blotting</SPAN></A></LI><LI class="toclevel-1"><A href="#Transmission_Electron_Microscopy_Observation"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Transmission Electron Microscopy Observation</SPAN></A></LI><LI class="toclevel-1"><A href="#High_Performance_Liquid_Chromatograpy"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">High Performance Liquid Chromatograpy</SPAN></A></LI><LI class="toclevel-1"><A href="#DMS_Detecting_Tube_Assay"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">DMS Detecting Tube Assay</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="PCR">PCR</SPAN></H3><UL><LI>Mix the following.</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>25mM MgSO4</TD><TD>1.5 µL
</TD></TR><TR><TD>2mM dNTPs</TD><TD>2.5 µL
</TD></TR><TR><TD>10xBuffer for KOD -Plus- ver.2</TD><TD>2.5 µL
</TD></TR><TR><TD>Template DNA</TD><TD>properly
</TD></TR><TR><TD>Primer Forward (10 µM)</TD><TD>0.75 µL
</TD></TR><TR><TD>Primer Reverse (10 µM)</TD><TD>0.75 µL
</TD></TR><TR><TD>KOD -Plus-</TD><TD>0.5 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 25 µL
</TD></TR><TR><TD>Total</TD><TD>25 µL
</TD></TR></TBODY></TABLE><UL><LI>Put samples into Thermal Cyclers and run the following steps.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>PreDenature</TH><TH>Denature</TH><TH>Anealing</TH><TH>Extension</TH><TH>cycle
</TH></TR><TR><TD>94 °C</TD><TD>98 °C</TD><TD>Tm-5 °C</TD><TD>68 °C</TD><TD>--
</TD></TR><TR><TD>2 min</TD><TD>10 sec</TD><TD>30 sec</TD><TD>1 min/kb</TD><TD>30 cycles
</TD></TR></TBODY></TABLE><UL><LI>Agarose Gel Electrophoresis for confirmation.</LI></UL><P><B>PCR: ToYoBo Quick Taq HS DyeMix</B></P><UL><LI>Mix the following.</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>Template DNA</TD><TD>properly
</TD></TR><TR><TD>Primer Forward (10 µM)</TD><TD>0.2 µL
</TD></TR><TR><TD>Primer Reverse (10 µM)</TD><TD>0.2 µL
</TD></TR><TR><TD>2x Quick Taq HS DyeMix</TD><TD>5 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 10 µL
</TD></TR><TR><TD>Total</TD><TD>10 µL
</TD></TR></TBODY></TABLE><UL><LI>Put samples into Thermal Cyclers and run the following steps.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>PreDenature</TH><TH>Denature</TH><TH>Anealing</TH><TH>Extension</TH><TH>cycle
</TH></TR><TR><TD>94 °C</TD><TD>94 °C</TD><TD>Tm-5 °C</TD><TD>68 °C</TD><TD>--
</TD></TR><TR><TD>2 min</TD><TD>30 sec</TD><TD>30 sec</TD><TD>1 min/kb</TD><TD>30 cycles
</TD></TR></TBODY></TABLE><UL><LI>Agarose Gel Electrophoresis for confirmation.</LI></UL><H3><SPAN class="mw-headline" id="Mutation_PCR">Mutation PCR</SPAN></H3><P>Use TaKaRa PrimeSTAR Mutagenesis BasaI Kit</P><UL><LI>Dilute the concentration of template DNA with MilliQ.</LI><LI>Mix the following</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>PrimeSTAR Max Premix (2x)</TD><TD>25µL
</TD></TR><TR><TD>Primer Forward </TD><TD>10pmol
</TD></TR><TR><TD>Primer Reverse</TD><TD>10pmol
</TD></TR><TR><TD>Template DNA (1ng/µL)</TD><TD>1~2µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 50µL
</TD></TR><TR><TD>Total</TD><TD>50µL
</TD></TR></TBODY></TABLE><UL><LI>30 cycles for 10s at 98°C, for 15s 55°C, and for 40sec at 72°C.</LI><LI>Agalose Gel Electrophoresis for confirmation.</LI><LI>Negative Control: Use nothing.</LI><LI>Add 2L PCR products and 20L competent cells for transformation.</LI></UL><H3><SPAN class="mw-headline" id="RNA_Extraction">RNA Extraction</SPAN></H3><P>Use　ISOGEN－LS（NIPPON　GENE,311-02621</P><UL><LI>Add 1mL ISOGEN-LS to sample and vortex.</LI><LI>Store for 5min on ice.</LI><LI>Add 250µL chloroform and shake vigorously for 15 sec.</LI><LI>Store for 3min. on ice.</LI><LI>Centrifuge 17400xg for 10min. at 4°C.</LI><LI>Transfer aqueous phase to another tube and add 0.8 volume isopropanol.</LI><LI>Store for 10min. on ice.</LI><LI>Centrifuge 17400xg for 10min. at 4°C.</LI><LI>Discard the supernatant.</LI><LI>Add 800µL 80% ethanol and vortex.</LI><LI>Centrifuge 7500xg for 5min. at 4°C.</LI><LI>Discard the supernatant.</LI><LI>Dry briefly.</LI><LI>Dissolve in nuclease-free water.</LI></UL><H3><SPAN class="mw-headline" id="Making_Competent_Cells">Making Competent Cells</SPAN></H3><UL><LI>Streak <I>E.coli</I> cells on an LB plate</LI><LI>Allow cells to grow at 37°C overnight</LI><LI>Place one colony in 3 mL LB media (+antibiotic selection if necessary), grow overnight at 37°C</LI><LI>Take 2 ml LB media and save for blank. Transfer 500 µL overnight culture into 50 mL LB media in 500 mL flask</LI><LI>Allow cell to grow at 37°C (250 rpm), until OD600= 0.4 (~2-3 hours)</LI><LI>Place cells on ice for 30 mins</LI><LI>Transfer cells to a centrifuge tube (50 mL), and centrifuge cells in High Speed refrigirated centrifuge at 4°C for 10 mins at 3,000 g.</LI><LI>Pour off media and resuspend cells in 12 mL of cold TB.</LI><LI>Centrifuge cells at 4°C for 10 mins at 3,000 g (2500 rpm)</LI><LI>Pour supernatant and resuspend cells (by pipetting) in 5 mL of cold TB and 350 µL of DMSO. Transfer 20 µL to 1.5 mL　tube</LI><LI>Ependorff tubes placed on ice. Freeze the cells in liquid nitrogen. Cells stored at -80°C can be used for transformation for up to ~6 months</LI></UL><H3><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H3><P>Use LaboPass Plasmid Mini Purification Kit.</P><UL><LI>Pick a single colony from a freshly streaked selective plate and inoculate a culture of about 3mL Plusgrow medium containing the appropriate selective antibiotic.</LI><LI>Incubate at 160 rpm for 8 h at 37 °C with vigorous shaking.</LI><LI>Pellet the bacterial culture by centrifugation for 1 min at 15,000 g in a tabletop centrifuge.</LI><LI>Discard the supernatant as much as possible.</LI><LI>Resuspend pelleted bacterial cells in 250 µL of Buffer S1.</LI><LI>Transfer the suspension to a new 1.5 mL tube.</LI><LI>Add 250 µL of Buffer S2 and mix by inverting the tube 4 times (do not vortex).</LI><LI>Add 350 µL of Buffer S3 and immediately mix by inverting the tube 4-6 times (do not vortex).</LI><LI>Centrifuge for 10min.</LI><LI>Transfer carefully the supernatant to a spin column and centrufuge for 1 min.</LI><LI>Remove the spin column, discard the flowthrough, and re-insert the spin column to the collection tube.</LI><LI>Apply the 750 µL of Buffer PW and centrifuge for 1 min.</LI><LI>Remove the spin column, discard the flowthrough, and re-insert the spin column to the collection tube.</LI><LI>Discard the flowthrough, and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Transfer the spin column to a new 1.5 mL tube.</LI><LI>Add 30 µL of MilliQ, let stand for 1 min, and centrifuge for 1 min.</LI></UL><H3><SPAN class="mw-headline" id="Ethanol_Precipitation">Ethanol Precipitation</SPAN></H3><P>Use Ethachinmate (NIPPON GENE、312-01791).</P><UL><LI>Add 3.3 µL of 3M Sodium Acetate (attached with Ethachinmate) into 100µL of DNA solution.</LI><LI>Add 1µL of Ethachinmate.</LI><LI>Vortex.</LI><LI>Add ethanol, 200-250µL.</LI><LI>Vortex.</LI><LI>Centrifuge at 12000xg for 5min.</LI><LI>Precipitation.</LI></UL><H3><SPAN class="mw-headline" id="Electrophoresis">Electrophoresis</SPAN></H3><UL><LI>Prepare 200mL of a 1.0% agarose solution:</LI><LI>Measure 2.0g agarose into a beaker.</LI><LI>Add 200mL 1xTAE buffer.</LI><LI>Wrap the top of the beaker with plastic wrap.</LI><LI>Punch a hole through the wrap with a pipette tip (To let out steam).</LI><LI>Dissolve the agarose by heating in microwave and swirling without boiling.</LI><LI>Allow the agarose to cool.</LI><LI>Pour the agarose solution into a gel tray on a gel maker.</LI><LI>If there is air bubbles, pushing them with a pipette tip.</LI><LI>Place comb in the maker.</LI><LI>Cover the maker with a plastic wrap.</LI><LI>Let stand for about 45min.</LI><LI>Remove the comb carefully.</LI><LI>Store in the Tupperware in the refrigerator.</LI><LI>Place the tray in electrophoresis chamber.</LI><LI>Cover the tray with 1xTAE buffer.</LI><LI>To prepare samples for electrophoresis, add 1µL of 10x Loading Buffer for every 9µL of DNA solution and mix well.</LI><LI>Load 6µL of the DNA solution per well.</LI><LI>Electrophoresis at 100V for about 30min until Loading Buffer have migrated approximately three-quarters of the gel.</LI><LI>Stain the gel in ethidium bromide.</LI><LI>Place a plastic wrap on the transilluminator in the cabinet of Printgraph.</LI><LI>Place the gel on the transilluminator.</LI><LI>Turn on the transilluminator and confirm the position of the gel.</LI><LI>Shoot the picture.</LI><LI>Turn off the transilluminator.</LI><LI>Dispose of the gel.</LI></UL><H3><SPAN class="mw-headline" id="Gel_Extraction_and_PCR_Purification">Gel Extraction and PCR Purification</SPAN></H3><P><B>Gel Extraction</B>
Use LMP agarose gel
</P><UL><LI>Perform electrophoresis using an established protocol, using 1% LMP agarose gel.
</LI><LI>Visualize and photograph the DNA using a long-wavelength UV lamp and an intercalating dye such as ethidium bromide. To reduce nicking, irradiate the gel for the absolute minimum time as possible. Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade.
</LI><LI>Transfer the gel slice to a microcentrifuge tube.
</LI><LI>If gel volume is less than 200mL, add 600mL of MilliQ. If not, add 900mL of MilliQ.
</LI><LI>Vortex the mixture and incubate at 70°C for 20 minutes or until the gel slice is completely dissolved.
</LI><LI>Vortex the tube every few minutes to increase the rate of agarose gel melting.
</LI><LI>Add same volume of phenol as that of gel solution, and vortex the tube.
</LI><LI>Centrifuge the tube in a microcentrifuge at 15,000 rpm for 5 min.
</LI><LI>Transfer the supernatant to new microcentrifuge tube.
</LI><LI>Centrifuge the tube in a microcentrifuge at 15,000 rpm for 5 min.
</LI><LI>To avoid with contaminating phenol, transfer the one-eighth from the top of supernatant to new microcentrifuge tube.
</LI><LI>Add same volume of butanol as that of solution, and vortex the tube.
</LI><LI>Centrifuge the tube in a desktop small-sized centrifuge for few seconds. 
</LI><LI>Discard the supernatant. Repeat the extraction process by butanol until water layer reduce less than 100 µL.
</LI><LI>Add 10 µL of 3M Sodium Acetate, 1 µL of glycogen and 250 µL of 99.5% ethanol.
</LI><LI>Cool the tube at -30°C for 10 minutes.
</LI><LI>Centrifuge the tube in a microcentrifuge at 15,000 rpm for 10 min.
</LI><LI>Discard the supernatant.
</LI><LI>Add 200 µL of 70% ethanol, and Centrifuge the tube in a microcentrifuge at 15,000 rpm for 10 min.
</LI><LI>Discard the supernatant.
</LI><LI>Open the cap and centrifuge the tube in a microcentrifuge at 15,000 rpm for 10 min.
</LI><LI>Add 10 µL of MilliQ, and dissolve the DNA.
</LI></UL><P>
Use Promega Wizard SV Gel and PCR Clean-Up System.</P><UL><LI>Perform electrophoresis using an established protocol.</LI><LI>Weigh a 1.5 mlL microcentrifuge tube for each DNA fragment to be isolated and record the weight.</LI><LI>Visualize and photograph the DNA using a long-wavelength UV lamp and an intercalating dye such as ethidium bromide. To reduce nicking, irradiate the gel for the absolute minimum time possible. Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade.</LI><LI>Transfer the gel slice to the weighed microcentrifuge tube and record the weight. Subtract the weight of the empty tube from the total weight to obtain the weight of the gel slice.</LI><LI>Add Membrane Binding Solution at a ratio of 10 µL of solution per 10 mg of agarose gel slice.</LI><LI>Vortex the mixture and incubate at 52 °C for 10 minutes or until the gel slice is completely dissolved. Vortex the tube every few minutes to increase the rate of agarose gel melting. Once the agarose gel is melted, the gel will not resolidify at room temperature.</LI><LI>Proceed to General.</LI></UL><P><B>PCR Purification</B></P><UL><LI>Amplify target of choice using standard amplification conditions.</LI><LI>Add an equal volume of Membrane Binding Solution to the PCR amplification.</LI><LI>Proceed to General.</LI></UL><P><B>General</B></P><UL><LI>Place one SV Minicolumn in a Collection Tube for each dissolved gel slice or PCR amplification.
</LI><LI>Transfer the dissolved gel mixture or prepared PCR product to the SV Minicolumn assembly and incubate for 1 minute at room temperature.</LI><LI>Centrifuge the SV Minicolumn assembly in a microcentrifuge at 16,000 g for 1 min. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube.</LI><LI>Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol, to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 min at 16,000 g.</LI><LI>Remove the SV Minicolumn assembly from the centrifuge, being careful not to wet the bottom of the column with the flowthrough. Empty the Collection Tube and recentrifuge the column assembly for 1 min with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.</LI><LI>Carefully transfer the SV Minicolumn to a clean 1.5 mL microcentrifuge tube. Apply 30 mL of MilliQ directly to the center of the column without touching the membrane with with the pipette tip. Incubate at room temperature for 1 min. Centrifuge for 1 min at 16,000 g.</LI><LI>Discard the SV Minicolumn and store the microcentrifuge tube containing the eluted DNA at 4 °C or 20 °C</LI></UL><H3><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H3><P>Use ToYoBo Ligation High Ver.2.
</P><UL><LI>Mix the vector DNA and the insert DNA (the vector and the insert at 1 : 5-10).</LI><LI>Add a half volume of Ligation High Ver.2 to the DNA solution.</LI><LI>Incubate at 16 °C for 1 hour (or at 4 °C for overnight).</LI></UL><H3><SPAN class="mw-headline" id="Restriction_Enzyme_Digestion">Restriction Enzyme Digestion</SPAN></H3><P><B>Use EcoRI, XbaI, SpeI, PstI (TaKaRa).</B></P><UL><LI>Mix the following.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>Sample DNA</TD><TD>2 µg
</TD></TR><TR><TD>Restriction Enzyme</TD><TD>0.5µL
</TD></TR><TR><TD>10x Buffer</TD><TD>3 µL
</TD></TR><TR><TD>(If use XbaI,) 10x BSA</TD><TD>3 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 30 µL
</TD></TR><TR><TD>total</TD><TD>30 µL
</TD></TR></TBODY></TABLE><UL><LI>Let stand for 1 hour at 37 °C.</LI></UL><P><B>Use EcoRI, XbaI, SpeI, PstI (Promega).</B></P><UL><LI>Mix the following.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>Sample DNA</TD><TD>2 µg
</TD></TR><TR><TD>Restriction Enzyme</TD><TD>0.5µL
</TD></TR><TR><TD>10x Buffer</TD><TD>3 µL
</TD></TR><TR><TD>100x BSA</TD><TD>3 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 30 µL
</TD></TR><TR><TD>total</TD><TD>30 µL
</TD></TR></TBODY></TABLE><UL><LI>Let stand for 1 hour at 37 °C.</LI></UL><H3><SPAN class="mw-headline" id="Media">Media</SPAN></H3><P><B>M9 medium</B></P><UL><LI>Stir Na2HPO4 6 g, KH2PO4 3 g, NaCl 0.5 g and NH4Cl 1 g with water 500 mL.</LI><LI>If you make M9 plates, add agar 15 g.</LI><LI>After autoclave, add 1 mL filter sterilized 1 M MgSO4, 5.6 mL 2 M glucose, 0.1 mL 1 M CaCl2, and 1 mL 1 % thiamine-HCl.</LI></UL><P>If you need, add 10 mL filter sterilized 20 % casamino acid.<B>LB medium</B></P><UL><LI>Stir BactoTryptone 2 g, Bacto-yeast extract 1 g, NaCl 1 g and 1M NaOH 200 µL with water 200mL.</LI><LI>If you make LB plates, add agar 2 g.</LI><LI>Autoclave.</LI></UL><P><B>SOB medium</B></P><UL><LI>Stir BactoTryptone 20 g and Bacto-yeast extract 5 g with water.</LI><LI>Add 2 mL 5 M NaCl and 840 µL 3 M KCl and add water up to 1 L.</LI><LI>After autoclave, add 10 mL filter sterilized 1 M MgSO4 and 1 M MgCl2.</LI></UL><P><B>SOC medium</B></P><UL><LI>Add 2 M glucose 1 mL to 100 mL SOB.</LI></UL><P><B>Plusgrow Medium</B></P><UL><LI>Stir plusgrow 20 g and with water 500 mL.</LI><LI>Autoclave.</LI></UL><P><B>YTSS medium</B></P><UL><LI>Stir BactoTryptone 0.5 g,Bacto-yeast extract 0.8 g and Red sea salt(Red sea) 40g with water 1000mL.</LI><LI>Autoclave.</LI></UL><H3><SPAN class="mw-headline" id="Buffer_TB">Buffer TB</SPAN></H3><UL><LI>Stir 0.6g PIPES and 30mg CaCl2 with 10 mL water.</LI><LI>Add 2.5mL 2M KCl.</LI><LI>Add KOH and adjust pH 6.7.</LI><LI>Add 0.218g MnCl2・4H2O.</LI><LI>Add water up to 20 mL.</LI><LI>Filter sterilize.</LI></UL><H3><SPAN class="mw-headline" id="Solubilization_of_Antibiotics">Solubilization of Antibiotics</SPAN></H3><UL><LI>Mix the following (Final concentration is 50mg/mL).</LI></UL><P><B>Ampicillin</B></P><TABLE class="wikitable"><TBODY><TR><TD>Ampicillin</TD><TD>1.0g
</TD></TR><TR><TD>MilliQ</TD><TD>20mL
</TD></TR></TBODY></TABLE><P><B>Kanamycin</B></P><TABLE class="wikitable"><TBODY><TR><TD>Kanamycin</TD><TD>0.5g
</TD></TR><TR><TD>MilliQ</TD><TD>10mL
</TD></TR></TBODY></TABLE><P><B>Chloramphenicol</B></P><TABLE class="wikitable"><TBODY><TR><TD>Chloramphenicol</TD><TD>0.5g
</TD></TR><TR><TD>99.5% EtOH</TD><TD>10mL
</TD></TR></TBODY></TABLE><UL><LI>Dispense 1.1mL of the solution into 1.5mL tubes.</LI><LI>Store in the -20°C freezer.</LI></UL><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><UL><LI>Unfreeze conpetent cells on ice.</LI><LI>Dry a plate by letting the plate upside down and partly open in incubator.</LI><LI>Add 1~20µL DNA solution and 10~50µL competent cells to 1.5mL tube, let stand for 30min on ice. If few colonies are observed, increase the amount of the competent cells or DNA, but make the amount of DNA not to get over that of the competent cells.</LI><LI>Heatshock for 45s at 42°C.</LI><LI>Let stand for 2min on ice.</LI><LI>Culture for 1h in preculture medium (LB or SOC medium), and plate by using spreader. Do not heat spreader too much because <I>E.coli</I> will dead for heat.</LI><LI>Culture overnight at 37°C.</LI></UL><H3><SPAN class="mw-headline" id="Sequence">Sequence</SPAN></H3><P>Use Big Dye Terminator 3.1（ABI）</P><UL><LI>Mix the following</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>5xBuffer</TD><TD>1.75µL
</TD></TR><TR><TD>Primer (3.2µM)</TD><TD>0.5µL
</TD></TR><TR><TD>Template Plasmid</TD><TD>400ng
</TD></TR><TR><TD>Big Dye Terminator 3.1</TD><TD>0.5µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 10.5µL
</TD></TR></TBODY></TABLE><UL><LI>Let stand for 1min at 96°C.</LI><LI>30 cycles for 5s at 98°C, for 5s 50°C, and for 4min at 68°C.</LI><LI>Add 25µL 100% ethanol and 1µL NAOAC</LI></UL><H3><SPAN class="mw-headline" id="qRT-PCR">qRT-PCR</SPAN></H3><P>Use QuantiTect SYBR green PCR kit Cat. No. 204143 by QIAGEN</P><UL><LI>Dilute primer to 1.5µM.</LI><LI>Dilute RT products.</LI><LI>Mix the following;</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>2x QuantiTect SYBR Green PCR Master Mix</TD><TD>22.5µL
</TD></TR><TR><TD>1/20xRT products</TD><TD>4.5µL
</TD></TR><TR><TD>MilliQ</TD><TD>9µL
</TD></TR><TR><TD>total</TD><TD>36µL
</TD></TR></TBODY></TABLE><UL><LI>Mix the reaction mix thoroughly, and dispense 36µL into 96 wells plate.</LI><LI>Add primer set 9µL.</LI><LI>Mix by inverting and voltex.</LI><LI>Dispense 10µL into 384 wells plate and centrifuge.</LI><LI>Let stand for 2min at 50°C and for 15min for 95°C.</LI><LI>40 cycles for 15sec at 95°, for 30sec at 60°, and for 1min at 72°C.</LI><LI>Let stand for 15sec at 95°.</LI></UL><H3><SPAN class="mw-headline" id="SDS-PAGE">SDS-PAGE</SPAN></H3><P>Make the Running gel</P><UL><LI>Mix the following</LI></UL><P><B>20% gel (Running gel) for 1gel</B></P><TABLE class="wikitable"><TBODY><TR><TD>30% AcrylAmide</TD><TD>5.3mL
</TD></TR><TR><TD>1.5M Tris (pH8.8)</TD><TD>2mL
</TD></TR><TR><TD>MilliQ</TD><TD>500mL
</TD></TR><TR><TD>10% SDS</TD><TD>80µL
</TD></TR><TR><TD>10% APS</TD><TD>80µL
</TD></TR><TR><TD>TEMED</TD><TD>4.8µL
</TD></TR><TR><TD>Total</TD><TD>8mL
</TD></TR></TBODY></TABLE><UL><LI>Insert Isopropanol on the running gel.
</LI><LI>Let stand for about 20 min
</LI></UL><P>Make the Stacking Gel
</P><UL><LI>Mix the following
</LI></UL><P><B>Stacking Gel for 1gel</B></P><TABLE class="wikitable"><TBODY><TR><TD>30% AcrylAmide</TD><TD>0.41mL
</TD></TR><TR><TD>1.5M Tris (pH8.8)</TD><TD>0.21mL
</TD></TR><TR><TD>MilliQ</TD><TD>18.2mL
</TD></TR><TR><TD>10% SDS</TD><TD>25µL
</TD></TR><TR><TD>10% APS</TD><TD>25µL
</TD></TR><TR><TD>TEMED</TD><TD>2.5µL
</TD></TR><TR><TD>Total</TD><TD>2.5mL
</TD></TR></TBODY></TABLE><UL><LI>Discard the Isopropanol
</LI><LI>Place the stacking gel on the running gel
</LI><LI>Place the comb, 12 well
</LI><LI>Let stand for 20 min
</LI></UL><UL><LI>Load the gel into the phoresis tank
</LI><LI>Fill 1xSDS buffer into the phoresis tank
</LI><LI>If there is air bubbles, pushing them with a syringe.
</LI><LI>Load the proper amount of protein sample solution per well
</LI><LI>Electrophoresis at 25mA for about 60 min
</LI></UL><H3><SPAN class="mw-headline" id="Western_Blotting">Western Blotting</SPAN></H3><UL><LI>Soak the filter paper and the gel in transfer-buffer.
</LI><LI>Soak the Membrane in Methanol and in transfer-buffer.
</LI><LI>Put 3 filter paper, membrane, gel, 3 filter paper on the transcription machinary.
</LI><LI>Transcript at 200 mA for 73 min.
</LI><LI>Wash the membrane by TBS-T buffer.
</LI><LI>Let shake for about 10 min 3 times.
</LI><LI>Block the Membrane for about 60 min.
</LI><LI>The Blocking reagent is &quot;DS Pharma Biomedical Co..Ltd, Block Ace Powder.&quot;
</LI><LI>Let shake for 60 min.
</LI><LI>Antibody is &quot;MBL,Anti-His-tag mAb-HRP-DirecT&quot;
</LI><LI>Make the antibody 5000 times dilution by TBS-T buffer.
</LI><LI>Mix the antibody with membrane.
</LI><LI>Let shake for 60 min.
</LI><LI>Wash the membrane by TBS-T buffer.
</LI><LI>Let shake for about 10 min.
</LI><LI>Place the membrane on the Western Lightening.
</LI><LI>Shoot the picture.
</LI><LI>Dispose the membrane.
</LI></UL><H3><SPAN class="mw-headline" id="Transmission_Electron_Microscopy_Observation">Transmission Electron Microscopy Observation</SPAN></H3><P>Ultramicrotome -&gt; LEICA　ULTRACUT UCT
</P><P>Transmission electron microscopy -&gt; JEM-1005X
</P><P><B>Prefix solution</B></P><TABLE class="wikitable"><TBODY><TR><TH>Final Conc.</TH><TH>Stock Sol.
</TH></TR><TR><TD>Glutaraldehyde (20 %)</TD><TD>2 %</TD><TD>1.25 mL
</TD></TR><TR><TD>Phosphate buffer (pH 7.4)</TD><TD>0.05 M</TD><TD>5 mL
</TD></TR><TR><TD>DW</TD><TD>3.75 mL
</TD></TR><TR><TD>Total</TD><TD>10 mL
</TD></TR></TBODY></TABLE><P><B>0.1 M Phosphate buffer (pH 7.4)</B></P><TABLE class="wikitable"><TBODY><TR><TD>NaH2PO4·H2O</TD><TD>1.38 g
</TD></TR><TR><TD>NaH2PO4·H2O</TD><TD>14.34 g
</TD></TR><TR><TD>DW</TD><TD>up to 500 mL
</TD></TR></TBODY></TABLE><P><B>Postfix solution</B></P><TABLE class="wikitable"><TBODY><TR><TH>Final Conc.</TH><TH>Stock Sol.
</TH></TR><TR><TD>osmic acid</TD><TD>1 %</TD><TD>0.5 mL
</TD></TR><TR><TD>Phosphate buffer</TD><TD>0.05 M</TD><TD>0.5 mL
</TD></TR><TR><TD>Total</TD><TD>1 mL
</TD></TR></TBODY></TABLE><P><B>TAAB EPON812 RESIN KIT</B></P><TABLE class="wikitable"><TBODY><TR><TD>TAAB EPON812 RESIN</TD><TD>24 g
</TD></TR><TR><TD>TAAB MNA</TD><TD>20 g
</TD></TR><TR><TD>TAAB DDSA</TD><TD>6 g
</TD></TR><TR><TD>TAAB DMP-30</TD><TD>1 g
</TD></TR></TBODY></TABLE><P>(Deaerate for 15 minute by Vacuum pumps)
</P><P><B>lead citrate (Reynold method)</B></P><UL><LI>Measure 1.33g lead nitrate and 1.76g sodium citrate into a beaker.
</LI><LI>Add 30mL DW and shake hard for 1 minute.
</LI><LI>Let stand the beaker for 30 minute as shaking the tube every few minutes.
</LI><LI>Add 8mL 1N-NaOH and DW up to 50mL.
</LI></UL><UL><LI>1. Pick a single colony from a freshly streaked selective plate and inoculate a culture of about 4mL LB medium containing the appropriate selective antibiotic.　Incubate at 160 rpm at 37 °C with vigorous shaking until reaching appropriate concentration.
</LI><LI>2. Transfer the 1.5mL cultured medium to microcentrifuge tube.
</LI><LI>3. Pellet the bacterial culture by centrifugation for 10 min at 3,000 g in a tabletop centrifuge.
</LI><LI>4. Discard the supernatant as much as possible.
</LI><LI>5. Add the 1.2mL prefix solution, and resuspent by pipetting.
</LI><LI>6. Incubate at 4 °C overnight.
</LI><LI>7. Rinse (Centrifugation: 10 min, 3,000 g and resuspention) by 0.1M Phosphate buffer twice.
</LI><LI>8. Add 50 µL postfix solution, and incubate on ice for 2 hours.
</LI><LI>9. Discard the postfix solution, and add 50 µL 50% ethanol. Incubate on ice for 15 min.
</LI><LI>10. Replace the solution with 50 µL 70% ethanol. Incubate at room temperature for 15 min.
</LI><LI>11.  Replace the solution with 50 µL 90% ethanol. Incubate at room temperature for 15 min.
</LI><LI>12. Replace the solution with 50 µL 95% ethanol. Incubate at room temperature for 15 min.
</LI><LI>13. Replace the solution with 50 µL 99.5% ethanol. Incubate at room temperature for 15 min.
</LI><LI>14. Replace the solution with 50 µL 100% ethanol (using molecular sieve). Incubate at room temperature for 15 min.
</LI><LI>15. Replace the solution with 50 µL 100% ethanol . Incubate at room temperature for 15 min.
</LI><LI>16. Replace the solution with 50 µL propylene oxide. Incubate at room temperature for 15 min.
</LI><LI>17. Replace the solution with 50 µL propylene oxide. Incubate at room temperature for 15 min.
</LI><LI>18. Replace the solution with 50 µL half and half mixture of propylene oxide and Epon 812 Resin Kit mixed liquor. Incubate at room temperature for 1 hour.
</LI><LI>19. Replace the solution with 50 µL Epon 812 Resin Kit mixed liquor. Incubate in the desiccator at 4°C overnight.
</LI><LI>20. Incubate at 60°C for 2 days.
</LI><LI>21. Trim the sample into 0.5~2mm square and slice into 1 micro meter thick by a glass knife.
</LI><LI>22. Observe the sections, and trim smaller to make ultra-thin section.
</LI><LI>23. To make the smooth aspect, slice into 1 micro meter thick by a new glass nife. 
</LI><LI>24. Slice into 90~100nm thick by diamond knife.
</LI><LI>25. Collect the section by collodion coated mesh (VECO GRID ,NISSIN EM ,Japan).
</LI><LI>26. Stain the sample by 1% uranyl acetate for 30 min.
</LI><LI>27. Stain the sample by lead citrate for 3 min.
</LI><LI>28. Observe by TEM, and take the photograph.
</LI></UL><H3><SPAN class="mw-headline" id="High_Performance_Liquid_Chromatograpy">High Performance Liquid Chromatograpy</SPAN></H3><UL><LI>Culture
</LI></UL><UL><LI>1. Cultivate E.coli in LB medium till OD600 becomes 0.8. LB contains 1% Methionine and 1mM 2-oxoglutarate.
</LI></UL><UL><LI>2. When OD600 become 0.8, add IPTG to medium for its density become 10mM
</LI></UL><UL><LI>3.Cultivate E.coli for 14 hours.
</LI></UL><UL><LI>HPLC
</LI></UL><UL><LI>1. Centrifuge 1mL of culture of E.coli 120rpm,2min to pelletize cells.
</LI></UL><UL><LI>2. Add 250μL of Lysis Buffer(50mM Tris, 1mM EDTA, 1mM DTT).
</LI></UL><UL><LI>3. Sonicate 30sec 3times.
</LI></UL><UL><LI>4. Centrifuge 15000rpm 15min at 4℃.
</LI></UL><UL><LI>5. Apply the supernatant to 3K cut off membrane(Amicon Ultra-0.5 mL Centrifugal Filters for Protein Purification and Concentration,Merck Millipore)15000rpm 30min at 4℃.
</LI></UL><UL><LI>6. 10 fold dilute and add equivalent DMB Solution(composition will show below;) and incubate 2h30min at 50℃.
</LI></UL><UL><LI>DMB Solution
</LI></UL><UL><LI>3.15mg DMB
</LI></UL><UL><LI>160.8μL Acetic acid
</LI></UL><UL><LI>105.6μL 2-mercaptoethanol
</LI></UL><UL><LI>18μL 2M Na-hydrosulfonide
</LI></UL><UL><LI>641.6μL Distilled water
</LI></UL><UL><LI>7. 10 fold dilute and inject 3μL of the sample into column. HPLC condition will show below;
</LI></UL><UL><LI>HPLC: SHIMADZU LC-10A
</LI></UL><UL><LI>Column: ZORBAX Eclipse® XDB-C18
</LI></UL><UL><LI>Total Flow: 0.5mL/min
</LI></UL><UL><LI>Gradient: Pump A: acetonitrile: MeOH: H2O=9:7:84  Pump B: 50:7:43
</LI></UL><UL><LI>0:01 B Percentage=5%
</LI><LI>0:01 Total flow=0.5mL/min
</LI><LI>0:01 Start
</LI><LI>20:00 B Percentage=20%
</LI><LI>25:00 B Percentage=100%
</LI><LI>30:00 B Percentage=100%
</LI><LI>35:00 B Percentage=5%
</LI><LI>45:00 Stop
</LI></UL><H3><SPAN class="mw-headline" id="DMS_Detecting_Tube_Assay">DMS Detecting Tube Assay</SPAN></H3><UL><LI>1.Cultivate E.coli in LB medium for  hours.  
</LI></UL><UL><LI>In the experimental, medium contains 1mM IPTG, 1nM DMSP, and E.coli which was transformed DddD generator.  
</LI></UL><UL><LI>In the positive control, medium contains 1mM IPTG, 1nM DMSP, 1mM DMS and E.coli which was transformed DddD generator.  
</LI></UL><UL><LI>In the negative control 1, medium contains 1mM IPTG, 1mM DMSP and E.coli which wasn't transformed. 
</LI></UL><UL><LI>In the negative control 2, medium contains 1mM IPTG and E.coli which was transformed DddD generator. 
</LI></UL><UL><LI>In the negative control 3, medium contains 1mM DMSP and E.coli which was transformed DddD generator. 
</LI></UL><UL><LI>Falcon tube in which samples incubated was sealed by Parafilm. 
</LI></UL><UL><LI>2.After cultivating E.coli, sample 30mL vapor in the syringe and inject it in DMS Detector tube. 
</LI></UL><UL><LI>We used Tokyo Gas Engineering Co.,Ltd, DMS Detecting tube .
</LI></UL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Kyoto/Material_and_Method">http://2014.igem.org/Team:Kyoto/Material_and_Method</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kyoto/Material_and_Method" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kyoto/Material_and_Method" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kyoto/Material_and_Method&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kyoto/Material_and_Method&amp;oldid=384349" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>