"
Page
Discussion
View source
History
teams
Log in
 
Team:Kyoto/Project/DMS Synthesis
From 2014.igem.org
count down
Magnetosome
DMS
Parts
Primers
Modeling
Protocol
Magnetosome
DMS
Our Question
Our Apporach
Future
Human Practice
Safety
For Judges
Acknowledgement
Sponsors
Official Team Profile
Member
Instructor & Advisor
Contact
Project
MAGNETOSOME FORMATION
DMS SYNTHESIS
Introduction
Experiments
Conclusion
Future Work
Reference
DMS SYNTHESIS
Introduction
Don't you think it is fantastic if the small organism, E. coli, create something huge, like a cloud? You might think we are daydreamers, but we found a clue in some scientists' words. In our project, forming cloud, we tried to make E. coli which can produce dimethyl sulfide (DMS), a source of Cloud Condensation Nuclei (CCN). In wetlands or sea, specific marine bacteria and coral produce DMS, so we try to obtain the genes related to this synthetic pathway and introduce them into E. coli.
How is cloud produced from DMS?
DMS is a simple volatile material formed through multi-step reactions by some marine organisms. After formed in the ocean, it is volatilized, and decomposed by the ultraviolet rays form the sky. Then it is converted to sulfate aerosol. In nature, this sulfate aerosol plays a role as a Cloud Condensation Nuclei (CCN), tiny particles which condense water vapor around itself and form clouds ( Fig. 1 ) [1] .
Fig. 1 The flow from organisms to cloud: The marine organisms produce DMS and DMSP. DMS becomes sulfate aerosol, and then plays a role of CCN.
Methionine (Met) - DimethylSulfidePropionate (DMSP) - DMS route is known as one of the DMS biosynthetic pathways. This route consists of Met-DMSP synthetic pathway of certain diatoms and corals and DMSP-DMS metabolic pathway of marine bacteria [2] ( Fig. 1 ).
Fig. 2 The synthetic pathway of Met to DMS
The structure of each intermediate chemical had been already clarified( Fig. 2 ).
They are 4-Methylthio-2-oxobutyrate (MTOB), 4-Methylthio-2-hydroxybutyrate (MTHB) and 4-Dimethylsulfonio-2-hydroxy-butyrate (DMSHB). However, genes responsible for each intermediate reaction of Met-DMSP pathway are still unknown ( Fig. 2 ). And only the candidate genes from Fragilariopsis cylindrus are suggested by Barbara R. Lyon et al. [3] . This organism is a model sea-ice diatom and can produce DMSP in Met-DMSP synthetic pathway.
Fig. 3-1 F. cylindrus uses DMSP in order to control their osmotic pressure.
Fig. 3-2 Barbara R. Lyon et al. compared the expression level under two different conditions (high-salinity and general-salinity) and chose the 5 candidate genes of Met-DMSP synthetic pathway.
The osmotic pressure of F. cylindrus is controlled by DMSP density ( Fig. 3-1 ). Thus, the authors of the paper hypothesized that proteins which increases under the condition of F. cylindrus producing a lot of DMSP, seem to be enzymes which catalyze the DMSP biosynthetic pathway. They compared the proteome of F. cylindrus cultured under high-salinity with another one under general salinity by using 2-dimensional electrophoresis. As a result, they found out that the amount of some proteins increased under high-salinity condition compared to general condition ( Fig. 3-2 ). Taking advantage of chemical reaction types in the Met-DMSP biosynthetic pathway and knowledge on substances of known enzymes, achieved from the mass spectrometry of these proteins, they speculated 5 candidate genes (AT, REDOX, SAMmt, DECARB, DiDECARB) that might be assigned to each step.
On the other hand, a gene concerned with DMSP-DMS pathway is identified from another organism. The enzyme which catalyzes the one-step reaction is encoded by the dddD gene. It is reported that dddD gene of Marinomonas sp., one of the marine bacteria, is functional in E. coli. [4]
Depending on these previous works, we speculate that introducing the 5 candidate genes of F. cylindrus and dddD enables E. coli to produce DMS. But it is still unclear whether each of these genes can be finely expressed in E. coli. So we tried to introduce these genes separately into E. coli to verify its expression.  In order to investigate whether the candidate proteins properly catalyze each reaction step, we used High Performance Liquid Chromatography (HPLC) to detect each reaction product of Met-DMSP synthetic route. To detect DMS, we used a DMS gas detecting tube.
Experiments
As described in the Introduction section, 5 candidate genes (AT, REDOX, SAMmt, DECARB, and DiDECARB) are considered to be involved in the Met-DMSP synthesis pathway. Also, one gene, dddD, is known to be involved in the DMSP-DMS metabolic pathway. We planned to introduce each gene into E. coli, in order to check the function of each gene one by one ( Fig. 4 ).
Fig. 4 DMS synthesis pathway and its candidate genes.
1. Cloning of 5 genes from genomic DNAs
First of all, we tried to obtain the target genes from the genomic DNA of each organism. For the Met-DMSP biosynthesis pathway, we purchased the genomic DNA of F. cylindrus, which is used in the preceding study [3] . With respect to dddD gene, which is involved in the final step of DMS synthesis, the function of dddD homolog from Marinomonas sp. MWYL1 strain was verified in E. coli in the preceding study [4] . However, we used Ruegeria pomeroyi genome for the source of dddD cloning [4] , since the genome of Marinomonas sp. MWYL1 was less commonly available.
Fig. 5 We cloned the candidate genes (Met-DMSP)form F. cylindrus and dddD gene from R. pomeroyi
We decided to start our project from the confirmation of the R. pomeroyi dddD homolog because its function has not been experimentally verified so far. We first compared the protein sequence of the dddD gene of R. pomeroyi and the homolog from Marinomonas sp. These proteins showed high homology to each other based on the global alignment (41% identity, 59% similarity).
Fig. 6 Protein sequence from dddD gene of R. pomeroyi has 41% identity to that of Marinomonas sp.
Moreover, as a common feature, these proteins share a domain structure with the CaiB protein, which is a predicted acyl-CoA transferases/carnitine dehydratase.
From the genomic DNAs, we amplified the selected genes (AT, REDOX, SAMmt, DECARB, DiDECARB, and dddD) by PCR. However, genes of AT, REDOX, SAMmt, DECARB, and DiDECARB contain introns, because F. cylindrus is an eukaryote. To remove these introns, we performed Overlap Extension PCR (OE-PCR) ( Fig. 7 ). Compared to the PCR products, the bands of the OE-PCR products showed a shorter length, indicating that the introns were removed ( Fig. 7 ). Thus, we attained our parts composed of only exons from each gene. We sequenced these parts and confirmed that the introns were precisely removed.
Fig. 7 Cloning of 5 genes from genomic DNAs
Fig. 8 Introns were removed.
2. Plasmid construction for expressing the genes in E. coli
We constructed the expression plasmids using the amplified PCR products. To date, we completed the DddD generator and the AT generator ( Fig. 9 ). We chose T7 promoter to express the interested genes, so that we could overexpress AT gene and dddD gene using E. coli BL21(DE3).
Fig. 9 Expression vector: T7 is a T7 promoter which is a strong expression promoter. In E. coli which is lysogen of bacteriophage DE3, this promoter is ordinarily repressed, and activated when Isopropyl β-D-1-thiogalactopyranoside (IPTG) exist. All plasmids are composed of pSB1C3 which is high copy plasmid. AT generator is the plasmid for expressing AT gene. DddD generator is the plasmid for expressing dddD gene.
3. Examination
Initially, we were planning to verify the function of AT generator by directly detecting MTOB, the product of AT reaction, by the UV absorption with HPLC. However, it turned out that E. coli produces too many chemicals with similar characteristics to MTOB, rendering the direct detection of MTOB difficult. We searched for an alternative method and found a protocol for fluorescent labeling of MTOB.
DMB (1,2-diamino-4,5-methylenedioxybenzene) reacts with α-keto acid specifically and emits strong fluorescence ( Fig. 10 ) [6] [7] [8] . MTOB is an α-keto acid that should be a good substrate of the labeling. Fortunately, α-keto acids seem to be scarce in the E. coli lysate, making our MTOB detection assay easy.
Fig. 10 MTOB was fluorescent labeled by DMB and detected by HPLC.
To evaluate our assay system for MTOB detection, we checked if, with HPLC, we can clearly detect MTOB in experimental medium containing E. coli lysate, methionine, IPTG, 2-oxoglutarate (necessary for the reaction from Met to MTOB [5] ) and so on. Pure chemicals (sample 1 and 2) or E. coli crude extract (sample 3) were separated by a C-18 column with acetonitrile gradient.
Fig. 11  Establishing a bioassay system.
In Sample 1, a single strong peak at 35.233 minutes was observed, indicating that this peak represents DMB. Similarly, in addition to the DMB peak, there is another strong peak at 34.070 minutes in Sample 2. This peak represents MTOB labeled by DMB, since free MTOB is not fluorescent. Note that the peak for free DMB (35.233 minutes) is less abundant in Sample 2 when compared to Sample1, suggesting the conversion of free DMB to DMB-MTOB conjugates.
As clearly observed, the peak around 34.070 is missing in Sample 3, demonstrating that all components of MTOB formation without AT generator produce no significant signal at this specific retention time. These results show that MTOB labeling by DMB would be an ideal method to detect MTOB synthesis by AT generator transformants.
Starting from the genomic DNA of F. cylindrus, we have completed the construction of the AT generator. However, unfortunately, we did not use the generator in our detection system described above, as we could not prepare the generator until very recently. So, the much-anticipated results of the AT reaction assay will be obtained in the future. We believe that we can present the detailed results in the coming jamboree this year.
Next, we transformed the DddD generator into E. coli and tried to cultivate the cells in a medium containing DMSP and IPTG, to verify the conversion of DMSP to DMS by the dddD gene product. However, also unfortunately, we did not conduct the experiment because DMSP didn't arrive very recently. So, the reaction of dddD reaction assay also will be obtained future.
Conclusion
We have completed two plasmids for the partial reconstitution of DMS synthesis in E. coli. We have also established the effective system for the detection of MTOB, a intermediate of DMSP synthesis, by using fluorescent labeling of MTOB in E. coli lysates.
Future Work
The examination of the AT generator and dddD generator are finally possible and we believe that it works. The next step of this project is to introduce all of the four candidate genes hypothetically involved in this pathway to E. coli. We will try to detect DMS formation directly by adding excess quantity of methionine to the growth media.
What can we do if we achieve the creation of DMS producing E. coli? In nature, biologically produced DMS is evaporated and converted to sulfate aerosol by UV irradiation, which forms cloud condensation nuclei (CCN). What we can try first is to emulate this process in vitro. Experimental demonstration of these reactions will become an intriguing show for the understanding of the global climate system.
Taking a long view, it might become possible for us to spread the engineered bacteria, to manipulate the local climate. However, it is absolutely necessary to establish effective security system before we use engineered organism in nature.
Reference
[1] Ippei Nagao, Progress and current status of research on dimethylsulfide, Low Temperature Science, 2014/3/31, (1-14)
[2] Gage et al., A new route for synthesis of dimethylsulphoniopropionate in marine algae, Nature, 26 JUNE 1997, (891-894)
[3] Barbara et al., Proteomic Analysis of a Sea-Ice Diatom: Salinity Acclimation Provides New Insight into the Dimethylsulfoniopropionate Production Pathway, American Society of Plant Biologists, December 2011, (1926-1941)
[4] Jonathan et al., Structural and Regulatory Genes Required to Make the Gas Dimethyl Sulfide in Bacteria, Science, 2 February 2007, (666-669)
[5] Summers et al., Identification and Stereospecificity of the First Three Enzymes of 3 Dimethylsulfoniopropionate Biosynthesis in a Chlorophyte Alga, Plant Physiol, January 1998, (369-378)
[6] Hara et al., Fluorimetric determination of α-keto acids with 4, 5-dimethoxy-1, 2-diaminobenzene and its application to high-performance liquid chromatography, Anal. Chim. Acta, 1985, (167-173).
[7] Hara et al., Fluorescent Products of Reaction between α-Keto Acids and 1, 2-Diamino-4, 5-dimethoxybenzene, Chem. Pharm. Bull., 1985, (3493-3498).
[8] Fluorescent Products of Reaction between α-Keto Acids and 1, 2-Diamino-4, 5-dimethoxybenzene, S. Hara, M. Yamaguchi, Y. Takemori and Y. Ohkura, Chem. Pharm. Bull., 35, 687(1987).
Retrieved from " http://2014.igem.org/Team:Kyoto/Project/DMS_Synthesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
