<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TecCEM_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TecCEM/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM_TECCEM</TITLE></P><DIV class="collapse navbar-collapse rightMenu" id="navbarSupportedContent"><UL class="navbar-nav mr-md-6 paddingNavBar"><LI class="nav-item active"><A class="nav-link whiteFont" href="https://2017.igem.org/Team:TecCEM">Home<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/Description" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Design">Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Experiments">Experiments</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Notebook">Notebook</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/InterLab">Interlab</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Model">Model</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Improve">Improve</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/Parts" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Parts</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Basic_Part">Basic Parts</A></DIV></LI><LI class="nav-item active"><A class="nav-link whiteFont" href="https://2017.igem.org/Team:TecCEM/Safety">Safety<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/PublicEngagement" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/HP/Silver">Silver HP</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/HP/Gold_Integrated">Integrated and Gold</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Engagement">Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/software" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Awards</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Hardware">Hardware</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Model">Model</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Software">Software</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/team" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Collaborations">Collaborations</A></DIV></LI></UL></DIV><TITLE>Experiments</TITLE><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-12"><H1 class="titleUbuntu">Protocols</H1></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Chemocompetent cell preparation
                </BUTTON><DIV class="dropdown-menu"><H3>Preparation of chemocompetent cells</H3><UL><H4>Materials</H4><LI>LB Medium</LI><LI>CaCl<SUB>2</SUB> 0.1 M</LI><LI>CaCl<SUB>2</SUB> 0.1M + 15% glycerol</LI><LI>Desired bacterial strain</LI></UL><OL><H4>Steps</H4><LI>Inoculate an isolated colony of the desired bacterial strain in a sterile tube containing 10 mL of LB medium and incubate overnight at 37°C and 260 rpm.</LI><LI>Inoculate 100 mL of LB broth with 1 mL of the previous culture (1:100 proportion).</LI><LI>Incubate for 3 hours at 37 °C and 260 rpm until it reaches a 0.6 O.D.</LI><LI>Place on ice for 10 minutes.</LI><LI>Divide the culture in sterile Falcon tubes and centrifuge at 5000 rpm for 5 minutes.</LI><LI>Remove the supernatant, resuspend the pellets in 10 mL of cold 0.1 M CaCl<SUB>2</SUB> and incubate on ice for 10 minutes.</LI><LI>Centrifuge at 4000 rpm for 10 minutes.</LI><LI>Remove the supernatant, resuspend the pellet in 2 mL of CaCl<SUB>2</SUB> 0.1M + 15% glycerol solution and incubate on ice for 10 minutes.</LI><LI>Prepare 200 μl aliquots and store at -80 °C.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Chemocompetent cell transformation
                </BUTTON><DIV class="dropdown-menu"><H3>Chemocompetent cell transformation</H3><UL><H4>Materials</H4><LI>Chemocompetent cells</LI><LI>Plasmid DNA</LI><LI>LB or SOC medium</LI></UL><UL><H4>Previous Steps</H4><LI>Prepare a water bath at 42°C.</LI></UL><OL><H4>Steps</H4><LI>Thaw a tube of competent cells on ice.</LI><LI>Add 5-10 μL of DNA (concentration between 1 pg/mL- 100 ng) to 125μL the competent cells.</LI><LI>Incubate the mixture in ice for 30 minutes.</LI><LI>Place in the water bath at 42°C for 30 seconds and immediately incubate on ice for 5 minutes.</LI><LI>Add 950 μL with SOC or LB medium under sterile conditions.</LI><LI>Incubate at 37°C for 60 minutes at 250 rpm.</LI><LI>Centrifuge at 5000 rpm for 2 minutes to obtain a pellet.</LI><LI>Discard 800 μL from the supernatant and resuspend the pellet in the remaining 200 μL.</LI><LI>Spatulate the volume on LB plates with the selection antibiotic and incubate for 16 hours at 37°C.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Annealing of oligonucleotides
                </BUTTON><DIV class="dropdown-menu"><H3>Annealing of oligonucleotides</H3><UL><H4>Materials</H4><LI>Oligonucleotides</LI><LI>Injectable water</LI><LI>NEBuffer 2.1</LI></UL><UL><H4>Previous Steps</H4><LI>Prepare a water bath at 95°C.</LI></UL><OL><H4>Steps</H4><LI>Perform the corresponding calculations for a desired concentration (μg/μL) and the volume needed to use 5 μg of each oligonucleotide.</LI><LI>For each oligonucleotide, make a mix with 1X NEBuffer 2.1 for a total volume of 20 μl with the following materials:</LI><UL><LI>Injectable water</LI><LI>NEBuffer 2.1</LI><LI>Sense oligonucleotide</LI><LI>Antisense oligonucleotide</LI></UL><LI>Incubate the mixture at 95°C for 5 minutes on the water bath.</LI><LI>Leave in the water bath to cool down until room temperature is reached.</LI><LI>Place on ice until further use.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Plasmid digestion
                </BUTTON><DIV class="dropdown-menu"><H3>Plasmid digestion</H3><UL><H4>Materials</H4><LI>NEBuffer 2.1</LI><LI>Restriction enzymes</LI><LI>Injectable water</LI></UL><OL><H4>Steps</H4><LI>Make a Master Mix with the following:</LI><UL><LI>18 μL of injectable water</LI><LI>5 μL of NEBuffer 2.1</LI><LI>1 μL of Enzyme 1</LI><LI>1 μL of Enzyme 2</LI></UL><LI>Add 6 μL of the Master Mix to 6 μL of linearized plasmid in a sterile tube.</LI><LI>Incubate at 37° for 2 hours.</LI><LI>Inactivate in water bath at 80°C for 20 minutes.</LI><LI>Incubate on ice for immediate use or store at -20 oC.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Ligation
                </BUTTON><DIV class="dropdown-menu"><H3>Ligation</H3><UL><H4>Materials</H4><LI>Injectable water</LI><LI>T4 DNA ligase buffer (10X)</LI><LI>T4 DNA ligase</LI><LI>Aligned oligonucleotides</LI><LI>Digested linearized plasmid (25ng/μL)</LI></UL><OL><H4>Steps</H4><LI>Prepare a master mix (if necessary) with the following components:</LI><UL><LI>Digested plasmid</LI><LI>Water °MB</LI><LI>T4 DNA ligase</LI><LI>T4 DNA ligase buffer</LI></UL><LI>Distribute the master mix among the reaction tubes and place the oligonucleotides aligned to have a final volume of 10 μL per reaction.</LI><LI>Leave incubating at room temperature overnight and transform the next day.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Lowry Method por protein quantification
                </BUTTON><DIV class="dropdown-menu"><H3>Lowry Method for protein quantification</H3><OL><H4>Standard curve protocol</H4><LI>To identify the protein concentration, quantify a BSA standard curve.</LI><LI>Preparation of samples with known proteins for a BSA stock (1mg/ml):</LI><P><B>Table 1.</B> Known protein concentrations with the volumes of each reactant for its preparation.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>Sample</TH><TH>BSA (μg)</TH><TH>BSA vol (1ug/ul)</TH><TH>H2O (ul)</TH><TH>C Reactant (ul)</TH><TH>Folin Reactant (ul)</TH></TR><TR><TH>1</TH><TH>0</TH><TH>0</TH><TH>100</TH><TH>500</TH><TH>50</TH></TR><TR class="parBackgroundColor"><TH>2</TH><TH>10</TH><TH>5</TH><TH>95</TH><TH>500</TH><TH>50</TH></TR><TR><TH>3</TH><TH>20</TH><TH>10</TH><TH>90</TH><TH>500</TH><TH>50</TH></TR><TR class="parBackgroundColor"><TH>4</TH><TH>30</TH><TH>15</TH><TH>85</TH><TH>500</TH><TH>50</TH></TR><TR><TH>5</TH><TH>40</TH><TH>20</TH><TH>80</TH><TH>500</TH><TH>50</TH></TR><TR class="parBackgroundColor"><TH>6</TH><TH>50</TH><TH>25</TH><TH>75</TH><TH>500</TH><TH>50</TH></TR><TR><TH>7</TH><TH>60</TH><TH>30</TH><TH>70</TH><TH>500</TH><TH>50</TH></TR><TR class="parBackgroundColor"><TH>8</TH><TH>70</TH><TH>35</TH><TH>65</TH><TH>500</TH><TH>50</TH></TR><TR><TH>9</TH><TH>80</TH><TH>40</TH><TH>60</TH><TH>500</TH><TH>50</TH></TR><TR class="parBackgroundColor"><TH>10</TH><TH>110</TH><TH>55</TH><TH>45</TH><TH>500</TH><TH>50</TH></TR><TR><TH>11</TH><TH>120</TH><TH>60</TH><TH>40</TH><TH>500</TH><TH>50</TH></TR></TBODY></TABLE><LI>After the samples of each concentration are prepared, add 500 μl of solution “C” (Biuret reactant) to each one and incubate for 10 minutes at room temperature.</LI><LI>Add 50 μl of Folin reactant to each sample and then incubate 45 minutes at room temperature.</LI><LI>Add 150μl per sample in a 96-well plate and read absorbance.</LI></OL><OL><H4>Protein quantification for the curve sample protocol</H4><LI>Resuspend samples in lysis buffer RIPA (200μl).</LI><LI>Dilute 1:20 using 10μl of sample in 190μl of milliQ water.</LI><LI>Add 1ml of “solution C” to the diluted samples and incubate for 10 minutes at room temperature.</LI><LI>Add 10μl of Folin reactant per sample and resuspend. Incubate 45 min at room temperature.</LI><LI>In a 96 well plate add 150μl per sample and read absorbance at 660.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Plasmid extraction
                </BUTTON><DIV class="dropdown-menu"><H3>Plasmid extraction</H3><UL><H4>Materials</H4><LI>Transformed E. coli DH5a cells.</LI><LI>LB + CAM</LI><LI>STET Buffer</LI><LI>Lysozyme</LI><LI>RNAse A</LI><LI>3 M Sodium acetate</LI><LI>Isopropanol</LI><LI>70% ethanol</LI><LI>Injectable water</LI></UL><OL><H4>Steps</H4><LI>Seed an isolated colony of each transformed Escherichia coli strain in 10 mL of LB+CAM in a test tube and incubate for 16 hours at 37oC at 260 rpm.</LI><LI>Centrifuge all the culture in 2 mL sterile microtubes at 13,000 rpm for 1 minute in order to retrieve the pellet.</LI><LI>Remove the supernatant and wash the pellets without resuspending with 350μL of STET Buffer (0.1M NaCl, 10 mM TRIS-HCl pH 8, 1 mM EDTA pH 8 and Triton X-100 5%)</LI><LI>Resuspend with 350 μL of STET Buffer and add 25μL of fresh lysozyme (2 mg/mL).</LI><LI>Incubate for 4 minutes at room temperature, and then place in a water bath at 100oC for 1 minute.</LI><LI>Centrifuge at 12,000 rpm for 10 minutes and remove the pellet with a stick.</LI><LI>Add 5 μL of RNAse A and incubate for 20 minutes at 37°C .</LI><LI>Add 75 μL of sodium acetate 3M pH 5.2 and 250 μL of isopropanol and incubate for 10 minutes at room temperature.</LI><LI>Centrifuge at 12,400 rpm for 10 minutes and discard the supernatant.</LI><LI>Wash the pellet twice with 750 μL of 70% ethanol to eliminate salts.</LI><LI>Resuspend the pellet in 150 μL of water for injections.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">SDS-PAGE
                </BUTTON><DIV class="dropdown-menu"><H3>SDS-PAGE</H3><OL><H4>Steps</H4><LI>Resuspend each sample in Laemmli buffer (B-Mercaptoethanol, 0.1%, Bromophenol blue 0.0005% ,Glycerol, 10%, SDS (electrophoresis-grade) 2% and Tris-HCl, 63 mM (pH 6.8). Make the necessary calculations so all samples have the same amount of protein.</LI><LI>Homogenize samples with the polytron.</LI><LI>Place the samples on a water bath at 100oC for 5 minutes.</LI></OL><OL><H4>Pouring the resolving gel</H4><LI>Clean glass plates with soap and water, then with ethanol. Assemble the glass plates and spacers.</LI><LI>Once the gel holder has been assembled, it is verified that there is no leak, the boundary of the separating phase is then marked 0.5 cm below the teeth of the comb.</LI><LI>Prepare 10 mL of separating phase. Due to the size of RFP (26 kDa), 10% or 12% gel would do.</LI><P>Table 1: Reagents required to prepare the separating phase in different % gels.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>Acylamide percentage</TH><TH>6%</TH><TH>8%</TH><TH>10%</TH><TH>12%</TH><TH>15%</TH></TR><TR><TH>H<SUB>2</SUB>O</TH><TH>5.2 ml</TH><TH>4.6 ml</TH><TH>3.8 ml</TH><TH>3.2 ml</TH><TH>2.2 ml</TH></TR><TR class="parBackgroundColor"><TH>Acrylamide/Bis-acrylamide (30%/0.8% w/v)</TH><TH>2 ml</TH><TH>2.6 ml</TH><TH>3.4 ml</TH><TH>4 ml</TH><TH>5 ml</TH></TR><TR><TH>1.5M Tris (pH 8.8)</TH><TH>2.6 ml</TH><TH>2.6 ml</TH><TH>2.6 ml</TH><TH>2.6 ml</TH><TH>2.6 ml</TH></TR><TR class="parBackgroundColor"><TH>10% w/v SDS</TH><TH>0.1 ml</TH><TH>0.1 ml</TH><TH>0.1 ml</TH><TH>0.1 ml</TH><TH>0.1 ml</TH></TR><TR><TH>10% w/v ammonium persulfate</TH><TH>100 μl</TH><TH>100 μl</TH><TH>100 μl</TH><TH>100 μl</TH><TH>100 μl</TH></TR><TR class="parBackgroundColor"><TH>TEMED</TH><TH>10 μl</TH><TH>10 μl</TH><TH>10 μl</TH><TH>10 μl</TH><TH>10 μl</TH></TR></TBODY></TABLE><LI>Add 5 mL the separating phase mixture taking care not to exceed the previously marked limit.</LI><LI>So that the phase is even and without bubbles, add a layer of ethanol, which is not miscible with the above mixture.</LI><LI>When the separating phase has gelled, the ethanol is removed with the help of a filter paper.</LI><LI>Prepare 5 mL of the concentrating phase.</LI><P>Table 2: Reagents required to prepare the concentrating phase.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>H<SUB>2</SUB>O</TH><TH>2.97 ml</TH></TR><TR><TH>0.5 M Tris-HCl (pH 6.8)</TH><TH>1.25 ml</TH></TR><TR class="parBackgroundColor"><TH>10% w/v SDS</TH><TH>0.05 ml</TH></TR><TR><TH>Acrylamide/Bis-acrylamide (30%/0.8% w/v)</TH><TH>0.67 ml</TH></TR><TR class="parBackgroundColor"><TH>10% w/v ammonium persulfate</TH><TH>0.05 ml</TH></TR><TR><TH>TEMED</TH><TH>0.005 ml</TH></TR></TBODY></TABLE><LI>The concentrating phase is added and the comb is placed so that there are no empty spaces. After waiting until it gels, the comb is removed.</LI><LI>Place the gel in the electrophoresis, cover with the running buffer.</LI><LI>Load the samples and the molecular weight in the wells.</LI><LI>Run the gel at given conditions.</LI><LI>Stain with Coomassie Blue dye overnight.</LI><LI>Destain with Destaining Solution (30% Methanol, 7% acetic acid) for 30 minutes. After the time is up, replace the solution with fresh one. Repeat until the gel is visible.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Plasmid extraction with Monarch Miniprep Kit
                </BUTTON><DIV class="dropdown-menu"><H3>Plasmid extraction with Monarch ® Plasmid DNA Miniprep Kit from New England Biolabs TM</H3><UL><H4>Previous step</H4><LI>Prepare Plasmid Wash Buffer 2, add 4 volumes of ethanol (&gt; 95%) to one volume of Buffer. Add 24 mL of ethanol for a final volume of (30 mL) of Plasmid Wash Buffer 2.</LI><P>*All centrifugations steps should be carried out at 13,000 rpm.</P></UL><OL><H4>Steps</H4><LI>Pellet 10 ml bacterial culture by centrifugation for 30 seconds. Discard supernatant.</LI><LI>Resuspend pellet in 400 μL plasmid Resuspension Buffer (B1). Vortex to ensure cells are completely resuspended. There should be no visible clumps.</LI><LI>Add 400μl Plasmid Lysis Buffer (B2), gently invert tube 5-6 times, and incubate at room temperature for 1 minute. Color should change to dark pink, and solution will become transparent and viscous. Do NOT vortex.</LI><LI>Add 800μl of Plasmid Neutralization Buffer (B3), gently invert tube until neutralized, and incubate at room temperature for 2 minutes. Sample is neutralized when color is uniformly yellow and precipitate forms. Do NOT vortex.</LI><LI>Centrifuge lysate for 10 minutes.</LI><LI>Carefully transfer supernatant to the spin column and centrifuge for 1 minute. Discard flow-through.</LI><LI>Re-insert column in the collection tube and add 200 μL of Plasmid Wash Buffer 1. Centrifuge for 1 minute. Discarding the flow-through is optional.</LI><LI>Add 400 μL of Plasmid Wash Buffer 2 and centrifuge for 1 minute.</LI><LI>Transfer column to a clean 1.5ml microfuge tube. Use care to ensure that the tip of the column does NOT come into contact with the flow-through. If there is any doubt, re-spin the column for 1 min.</LI><LI>Add 100 μl of injectable water to the center of the matrix. Wait for 1 minute, then spin for 1 min to elute DNA. A second eluted DNA was realized with 50 μl of water.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">DNA band purification
                </BUTTON><DIV class="dropdown-menu"><H3>DNA band purification with GenElute TM Gel Extraction Kit from Sigma-Aldrich.</H3><UL><H4>Materials</H4><LI>Scalpel</LI><LI>Ethanol 100%</LI><LI>Isopropanol 100%</LI><LI>3M Sodium Acetate Buffer, pH 5.2</LI><LI>Water bath at 60°C</LI><LI>Molecular Biology water</LI><LI>All the centrifuge are performed at 16,000 xg or 13,155 rpm</LI></UL><UL><H4>Previous steps</H4><LI>Prepare the Wash Solution Concentrate G with 48 mL of EtOH 100%</LI></UL><OL><H4>Steps</H4><LI>Excise the DNA fragment of interest from the agarose gel with the scalpel.</LI><LI>Weigh the gel slice in a microtube.</LI><LI>For every 100 mg of agarose gel, add 300 mL of Gel Solubilization Solution.</LI><LI>Incubate the gel mixture at 50-60 °C for 10 minutes, or until the agarose has dissolved. Vortex briefly every 2-3 minutes during incubation to help dissolve the gel.</LI><LI>Prepare the binding column in a microtube of 2 mL, add 500 μL of the Column Preparation Solution, centrifuge for 1 min and discard flow-through liquid.</LI><LI>If the mixture is red, add 10 μL of Sodium Acetate Buffer, until the mixture turns yellow.</LI><LI>Add 1 gel volume isopropanol and mix.</LI><LI>Load the gel solution to the binding column. If the volume of the gel mixture is &gt;700 mL, load the sample onto the column in 700 mL portions. Centrifuge for 1 minute after loading the column each time. Discard the flow-through liquid.</LI><LI>Add 700 μL of Wash Solution to the binding column, centrifuge 1 minute and discard the flow- through. Centrifuge again 1 minute.</LI><LI>Elute DNA, transfer the column to a new microtube, add 50 μL of Elute Solution, incubate for 1 minute and centrifuge 1 minute.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Polymerase Chain Reaction
                </BUTTON><DIV class="dropdown-menu"><H3>Polymerase Chain Reaction</H3><OL><H4>Steps</H4><LI>Prepare the Master Mix for the PCR reaction adding</LI><UL><LI>0.75μL 50 mM MgCl2</LI><LI>22. 5 μL PCR Buffer 10X</LI><LI>0.5 μL 10 mM dNTP Mix</LI><LI>1 μL 10 mM FWD Primer</LI><LI>1 μL 10 mM REV Primer</LI><LI>0.2 μL Taq DNA Polymerase</LI><LI>18.05 μL MBo Water</LI></UL><LI>Add the Master Mix and 1 μL of DNA template for each reaction. Final volume: 45 μL per reaction.</LI><LI>Reactions are placed in the thermal cycler with the following program:</LI><UL><LI>94°C for 5 minutes</LI><LI>31 cycles:</LI><UL><LI>94°C for 30 seconds</LI><LI>55°C for 30 seconds</LI><LI>72°C for 30 seconds</LI></UL><LI>72°C for 5 minutes</LI><LI>12°C indefinitely</LI></UL><LI>For PCR product analysis, an agarose gel electrophoresis must be performed.</LI><P>*Note: Number of cycles and alignment temperatures may vary in each case.</P></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Schneider's Insect Medium preparation
                </BUTTON><DIV class="dropdown-menu"><H3>Schneider’s Insect Medium Preparation Protocol</H3><OL><H4>Steps</H4><LI>Measure 80% of final volume of water at 15-20°C.</LI><LI>While gently stirring the water, add the powdered medium. Stir until dispersed.</LI><LI>Rinse original package with a small amount of water to remove all traces of powder. Add to solution in step 2.</LI><LI>To the solution in step 3, add 0.4 g of sodium bicarbonate for each liter of solution.</LI><LI>While stirring, adjust the pH to at least 9.2 ± 0.2 with 1N NaOH. Stir for a minimum of 10 minutes. Solution may become turbid.</LI><LI>While stirring, adjust the pH to 6.7 ± 0.2 with 1N HCl, solution will clear.</LI><LI>Prepare a calcium chloride solution by dissolving 0.6 g of anhydrous calcium chloride in 50 mL of tissue culture grade water for each liter of final volume being prepared. Slowly add the calcium chloride solution dropwise to the medium with rapid mixing to avoid precipitate formation.</LI><LI>While stirring, adjust the pH of the medium to 0.1-0.3 pH units below the desired pH with 1N HCl or NaOH.</LI><LI>Add additional water to bring the solution to final volume.</LI><LI>Sterilize immediately by filtration using a membrane with a porosity of 0.22 microns or less.</LI><LI>Aseptically dispense medium into sterile container.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Sequencing Biobricks
                </BUTTON><DIV class="dropdown-menu"><H3>Sequencing Biobricks</H3></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">BSLA SiRNA confirmation protocol
                </BUTTON><DIV class="dropdown-menu"><H3>BSLA SiRNA confirmation protocol</H3><OL><H4>Steps</H4><LI>Transform BSLA devices in HT115</LI><LI>Culture a blue colony of HT115 with BSLA devices for 2 days in liquid medium</LI><LI>Extract RNA (Without using RNAse)</LI><LI>Quantify the RNA samples</LI><LI>Prepare a hybridization buffer (30 mM Na2HPO4 pH 8, 0.3 M NaCl, 10 mM EDTA)</LI><LI>Place 16 μL fo the hybridization buffer in a 0.6 ml microtube and then add 4 μL of oligos (100 μM)</LI><LI>Add 2 μg of RNA sample</LI><LI>Mix the samples with the micropipette</LI><LI>Incubate the samples at 94°C for 5 minutes</LI><LI>Incubate the samples at 42°C for 60 minutes</LI><LI>Add 20 units of exonuclease 1</LI><LI>Incubate at 37°C for 30 minutes</LI><LI>Place the samples in ice or store they at -80°C until its use</LI><LI>Run a non-denaturing gel</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-12"><H1 class="titleUbuntu">Experiments</H1></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">RFP characterization curve for DH5a strain
                </BUTTON><DIV class="dropdown-menu"><H3>RFP characterization curve for DH5a strain</H3><OL><H4>IPTG induction</H4><LI>Prepare the seed culture by resuspending an isolated colony of transformed Escherichia coli DH5a+BBa_J04450 in 10 mL of liquid LB broth with chloramphenicol (25μg/mL) and incubate it at 37°C overnight.
                        </LI><LI>Inoculate 100 mL of LB + CAM (25μg/mL) with 1 ml of the seed culture and incubate at 37°C and 260 RPM until the OD reaches 0.6.</LI><LI>Divide the culture in two different flasks (50 mL each one).</LI><LI>Induce one of the flasks with 1 mL of a 1 M stock of IPTG, resulting in a final concentration of 20 mM.
                        </LI><LI>Divide the content of each flask in 8 glass tubes (1 per hour of the kinetics) with the objective of not interrupting the bacterial growth and incubated for 3 and a half hours at 37°C and 260 RPM.</LI><LI>Each hour, take a single tube from the incubator.</LI><LI>Use 100 μl to measure absorbance at 600 nm using the multiwell plate and 100 μl to measure fluorescence using the Synergy H1 Hybrid Multi-Mode Reader of BioTek. Program the excitation and emission wavelength at 550 nm and 584 nm respectively.</LI><LI>Divide the remaining volume in two different microtubes and centrifuge them at 5000 rpm for 2 minutes, and discard the supernatant.</LI><LI>Conserve the sample at -20°C for further analysis by Lowry Method and SDS-PAGE.</LI><LI>Repeat steps 7, 8 and 9 for each hour of the experiment.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Chitosan encapsulation
                </BUTTON><DIV class="dropdown-menu"><H3>Chitosan encapsulation</H3><UL><H4>Materials</H4><LI>Chitosan 0.15 g</LI><LI>Tripoliphosphate 0.05 g</LI><LI>Pluronic F-68 0.5 g</LI><LI>H<SUB>2</SUB>O BM 100ml</LI><LI>Acetic Acid 1 ml</LI></UL><OL><H4>Steps</H4><LI>Prepare a 1% acetic acid solution and adjust to pH 4 using NaOH.</LI><LI>Divide the solution in two 50 mL containers.</LI><LI>In one container, add and hydrate the chitosan and mix until dissolution.</LI><LI>Filter the chitosan using filter paper.</LI><LI>Add Pluronic F-68 to the filtered solution, hydrate and mix avoiding the formation of foam.</LI><LI>To the other 50 mL acetic acid 1% solution, add the tripolyphosphate and mix.</LI></OL><OL><H4>RacI siRNA encapsulation</H4><LI>Add 1.25 mL chitosan solution to a sterile 15 mL Falcon tube.</LI><LI>Add 250 uL of RacI siRNA 100 ng/uL.</LI><LI>Add 1.25 mL of tripolyphosphate solution.</LI><LI>Mix by vortex until the color changed from colorless to translucent white.</LI><LI>Store at 5°C.</LI></OL><OL><H4>Awd+Alexa siRNA encapsulation</H4><LI>Add 225 μL chitosan solution was added to a 15ml falcon tube.</LI><LI>Add 50 μL of Awd+Alexa siRNA 100 ng/μL.</LI><LI>Add 225 μL of tripolyphosphate solution.</LI><LI>Mix by vortex until the color changed from colorless to translucent white.</LI><LI>Store at 5°C.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Extraction of DNA from filter paper
                </BUTTON><DIV class="dropdown-menu"><H3>Extraction of DNA from filter paper</H3><OL><H4>Steps</H4><LI>Cut out the area of the filter with spotted cDNA (circle marked on the paper).</LI><LI>Put the piece of paper in a sterile microtube.</LI><LI>Apply 75 µl of Water Molecular Biology Grade.</LI><LI>Vortex 30 seconds.</LI><LI>Centrifuge 1 min to elute the DNA and pellet the solution and paper.</LI><LI> Store the solution toghether with the filter paper at 4°C.</LI><LI>Transform E. coli with 1 µl of the plasmid DNA.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown"><I>Diaphorina citri</I> Total RNA Extraction
                </BUTTON><DIV class="dropdown-menu"><H3>Diaphorina citri Total RNA Extraction</H3><UL><H4>Materials</H4><LI>20 <I>Diaphorina citri</I> live specimens</LI><LI>Liquid nitrogen</LI><LI>Trizol</LI><LI>Chloroform</LI><LI>RNaseOUT</LI><LI>Isopropanol</LI><LI>75% ethanol</LI><LI>Injectable water</LI></UL><OL><H4>Steps</H4><LI>Place 20 individuals at 4°C for 10 minutes.</LI><LI>Place on a mortar with liquid nitrogen, and crush with pestle until homogeneous.</LI><LI>Add 500 μL of Trizol and mix.</LI><LI>Transfer the mixture to a 1.5 mL Eppendorf tube. To ensure the transfer of all the material, any remnants in the mortar may be washed with an additional 300 μL of Trizol.</LI><LI>Leave the mixture for 5 minutes at room temperature to allow complete dissociation.</LI><LI>Add 160 μL of chloroform to the mixture and leave for 2 to 3 minutes.</LI><LI>Centrifuge at 12,000 g and 4°C for 15 minutes.</LI><LI>Transfer the aqueous phase to a new microtube and add 5 μL of RNAseOUT.</LI><LI>Add 400 μL of isopropanol and incubate for 10 minutes at room temperature.</LI><LI>Centrifuge at 10,000 g and 4°C for 10 minutes.</LI><LI>Discard the supernatant with a micropipette and resuspend in 800 μL of 75% ethanol.</LI><LI>Vortex the sample briefly.</LI><LI>Centrifuge at 7,500 g and 4°C for 5 minutes.</LI><LI>Discard the supernatant with a micropipette and let the pellet dry by placing the tube downwards on a clean sheet of paper.</LI><LI>Resuspend the pellet in 30 μL of injectable water.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Two-Step Reverse Transcriptase PCR
                </BUTTON><DIV class="dropdown-menu"><H3>Two-Step Reverse Transcriptase Polymerase Chain Reaction</H3><OL><H4>Steps</H4><LI>Prepare the RNA using RevertAid H Minus First Strand cDNA Synthesis Kit from ThermoFisher:</LI><UL><LI>3 μL (250 ng/ μL) of Diaphorina citri RNA</LI><LI>1 μL of Reverse primer (The mix was made using 10 μL of each primer)</LI><LI>8 μL of injectable water.</LI></UL><LI>As only 38.5 % of D. citri genome is composed by GC’s the step of incubating the sample at 65°C for 5 minutes can be omitted.</LI><LI>Prepare the Mix for cDNA synthesis as follows:</LI><P>Table 1: Preparation of the Master Mix using RevertAid H Minus First Strand cDNA Synthesis Kit.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>Reagent</TH><TH>1X</TH></TR><TR><TH>Buffer 5X</TH><TH>4 μL</TH></TR><TR class="parBackgroundColor"><TH>Ribolock RNAse inhibitor (20 U/μL)</TH><TH>1 μL</TH></TR><TR><TH>dNTPs Mix (10 μM)</TH><TH>2 μL</TH></TR><TR class="parBackgroundColor"><TH>Reverse Transcriptase (200 U/μL)</TH><TH>1 μL</TH></TR></TBODY></TABLE><LI>Give a spin in the centrifuge.</LI><LI>Incubate at 52°C for 1 hour in the thermal cycler.</LI><LI>Prepare the Mix for PCR as follows:</LI><P>Table 2: Preparation of the Master Mix for PCR.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>Reagent</TH><TH>1X</TH></TR><TR><TH>Buffer 10X</TH><TH>2.5 μL</TH></TR><TR class="parBackgroundColor"><TH>MgCl2 10 mM</TH><TH>0.75 μL</TH></TR><TR><TH>dNTP Mix 10 μM</TH><TH>1 μL</TH></TR><TR class="parBackgroundColor"><TH>Forward Primer (10 μM)</TH><TH>1 μL</TH></TR><TR><TH>Reverse Primer (10 μM)</TH><TH>1 μL</TH></TR><TR class="parBackgroundColor"><TH>Taq DNA polymerase</TH><TH>0.25 μL</TH></TR><TR><TH>D. citri cDNA</TH><TH>2 μL</TH></TR><TR class="parBackgroundColor"><TH>Injectible water</TH><TH>14.5 μL</TH></TR></TBODY></TABLE><LI>Add the Master Mix to the different microtubes, and complete the volume to 25 μL with 1 μL of each corresponding primer.</LI><LI>Place the reactions in the thermal cycler in the following conditions:</LI><UL><LI>94°C for 3 minutes</LI><LI>40 cycles:</LI><UL><LI>94°C for 30 seconds</LI><LI>54°C for 30 seconds</LI><LI>72°C for 20 seconds</LI></UL><LI>72°C for 5 minutes</LI><LI>12°C indefinitely</LI></UL><LI>A 2.5%-3% agarose gel electrophoresis is required to verify PCR products.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Diaphorina citri siRNA soaking
                </BUTTON><DIV class="dropdown-menu"><H3>Diaphorina citri siRNA soaking</H3><OL><H4>Steps</H4><LI>Collect 10 Diaphorina citri per essay.</LI><LI>Starve them for 2 hours.</LI><LI>Cool the Diaphorina citri at 4°C, and keep them on ice while working with them.</LI><LI>Take 8 μL of concentration of 100 ng/μL of siRNA.</LI><LI>Use a cool clean petri dish, place the Diaphorina citri and quickly put the droplet of SiRNA in the middle abdomen.</LI><LI>Wait 5 minutes, then remove the remaining water with a filter paper.</LI><LI>Place the psyllids on an apical meristem or a fresh shoot.</LI><OL><LI>If it is for a real time PCR test, harvest them after 4 days, perform the RNA protocol as done before.
                            </LI><LI>If it is for mortality analysis watch and report the number of dead individualsevery 6 hours.</LI></OL></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Diaphorina citri RNA extraction
                </BUTTON><DIV class="dropdown-menu"><H3>Transfected Diaphorina citri RNA extraction</H3><OL><H4>Steps</H4><LI>Collect 10 Diaphorina citri from each soaking essay.</LI><LI>Prepare the utensil to smash the psyllid. It is recommended to use a micropipette tip, this could be done by burning the tip and smashing it in order to close the hole of the tip.</LI><LI>Put the Diaphorina citri in the microtube and submerge it liquid nitrogen.</LI><LI>Take the tip and grind over the Diaphorina citri until an homogenous mix is obtained.</LI><LI>Take 250 μL of Trizol<SUP>TM</SUP> and add it to the microtube, wash the tip.</LI><LI>Mix and grind.</LI><LI>Add 250 μL of TRIzol<SUP>TM</SUP> and wash the tip again.</LI><LI>Centrifuge the lysate for 5 minutes at 12,000 × g at 4–10°C.</LI><LI>Incubate for 5 minutes to permit complete dissociation of the nucleoproteins complex.</LI><LI>Add 0.2 mL of chloroform per 1 mL of TRIzol<SUP>TM</SUP> Reagent used for lysis, then securely cap the tube.</LI><LI>Incubate for 2–3 minutes.</LI><LI>Centrifuge the sample for 15 minutes at 12,000 × g at 4°C. The mixture separates into a lower red phenol-chloroform, and interphase, and a colorless upper aqueous phase.</LI><LI>Transfer the aqueous phase containing the RNA to a new tube.</LI><LI>Transfer the aqueous phase containing the RNA to a new tube by angling the tube at 45° and pipetting the solution out.</LI><LI>Add 0.5 mL of isopropanol to the aqueous phase, per 1 mL of TRIzol<SUP>TM</SUP> Reagent used for lysis.
                        </LI><LI>Incubate for 10 minutes.</LI><LI>Centrifuge for 10 minutes at 12,500 × g at 4°C. Total RNA precipitate forms a white gel-like pellet at the bottom of the tube.</LI><LI>Discard the supernatant pouring the solution out.</LI><LI>Resuspend the pellet in 1 mL of 70% ethanol per 1 mL of TRIzol<SUP>TM</SUP> Reagent used for lysis.</LI><LI>Centrifuge for 5 minutes at 12500× g at 4°C. c.</LI><LI>Discard the supernatant, make sure the pellet is still in your microtube.</LI><LI>Air dry the RNA pellet for 5–10 minutes resuspend in 20-40 μL of water depending on the color of the aqueous phase in step 13, if it is almost transparent add 20 μL.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Two-Step qRT-PCR
                </BUTTON><DIV class="dropdown-menu"><H3>Two-Step Quantitative Reverse Transcriptase Polymerase Chain Reaction</H3><OL><H4>Steps</H4><LI>Prepare the RNA using RevertAid H Minus First Strand cDNA Synthesis Kit from ThermoFisher:</LI><UL><LI>250 ng/ μL of Diaphorina citri RNA-siRNA.</LI><LI>2 μL of Reverse primer (The mix was made using 10 μL of each primer)</LI><LI>Injectable water up to 24 μL.</LI></UL><LI>As only 38.5 % of D. citri genome is composed by GC’s the step of incubating the sample at 65oC for 5 minutes can be omitted.</LI><LI>Prepare the Mix for cDNA synthesis as follows:</LI><P>Table 1: Preparation of the Master Mix using RevertAid H Minus First Strand cDNA Synthesis Kit.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>Reagent</TH><TH>1X</TH></TR><TR><TH>Buffer 5X</TH><TH>4 μL</TH></TR><TR class="parBackgroundColor"><TH>Ribolock RNAse inhibitor (20 U/μL)</TH><TH>1 μL</TH></TR><TR><TH>dNTPs Mix (10 μM)</TH><TH>2 μL</TH></TR><TR class="parBackgroundColor"><TH>Reverse Transcriptase (200 U/μL)</TH><TH>1 μL</TH></TR></TBODY></TABLE><LI>Give a spin in the centrifuge.</LI><LI>Incubate at 52°C for 1 hour in the thermal cycler.</LI><LI>Prepare the Mix for qPCR as follows:</LI><P>Table 2: Preparation of the Master Mix for PCR.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH>Reagent</TH><TH>1X</TH></TR><TR><TH>Maxima SYBR Green/ROX qPCR Master Mix (2X)</TH><TH>12.5 μL</TH></TR><TR class="parBackgroundColor"><TH>Forward Primer (10 μM)</TH><TH>1 μL</TH></TR><TR><TH>Reverse Primer (10 μM)</TH><TH>1 μL</TH></TR><TR class="parBackgroundColor"><TH>D. citri cDNA</TH><TH>2.5 μL</TH></TR><TR><TH>Injectible water</TH><TH>Adjust to 25 μL</TH></TR></TBODY></TABLE><LI>Add the Master Mix to the different microtubes.</LI><LI>Place the reactions in the thermal cycler in the following conditions:</LI><UL><LI>94°C for 3 minutes</LI><LI>40 cycles:</LI><UL><LI>94°C for 30 seconds</LI><LI>54°C for 30 seconds</LI><LI>72°C for 20 seconds</LI></UL><LI>72°C for 5 minutes</LI><LI>12°C indefinitely</LI></UL><LI>A 2.5%-3% agarose gel electrophoresis is required to verify PCR products in addition to the generated plots.</LI></OL></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8"><DIV class="dropdown paragraphU paddingButton"><BUTTON class="btn btn-primary dropdown-toggle responsiveImage" type="button" data-toggle="dropdown">Primary cell culture
                </BUTTON><DIV class="dropdown-menu"><H3>Establishment of Diaphorina citri primary cell culture</H3><OL><H4>Steps</H4><LI>Collect 50 Diaphorina citri and keep in ice.</LI><LI>Pick each with sterile tweezers.</LI><LI>Wash in .11% sodium hypochlorite for 10 seconds.</LI><LI>Wash in 75% of ethanol for 30 seconds.</LI><LI>Wash in sterile H20MiliQ water for 20 seconds.</LI><LI>Place on an sterile petri dish under a stereoscopic microscope.</LI><LI>Grab the head of the insect with tweezers then slowly remove the lower abdomen. If removed slowly, green strings will appear indicating the digestive tract. In order to avoid contamination, remove completely.</LI><LI>Collect the removed digestive tracts and place in a separate container.</LI><LI>Collect the head and rest of the body and place in a microtube.</LI><P>For the digestive tract:</P><LI>Place a Corning 10 μm cell strain filter over a sterile 50 mL Falcon Tube and place the material in it.</LI><LI>Using a 5 ml syringe plunger, grind the tissue until it is completely homogenous.</LI><LI>Add 500 ul of Hank's Salt Solution.</LI><LI>Keep grinding the remains.</LI><LI>Add another 500ul of Hank's Salt Solution.</LI><LI>In a different 50mL Falcon Tube, place a 40 μm Corning cell strain filter and filter the previously obtained solution.</LI><LI>Add 300 uL of Schneider's Insect medium and place on a cell culture plate.</LI><P>For the rest of the body:</P><LI>Centrifuge at 250 g for 5 minutes.</LI><LI>Remove the supernatant with a micropipette and place on a 10 μm cell strain filter.</LI><LI>Grind with a syringe plunger until homogeneous.</LI><LI>Wash the filter with 400 uL of Hank’s Salt Solution.</LI><LI>Transfer the filtered liquid to a 40 μm cell strain filter and add 200 uL of Hank’s Salt Solution.</LI><LI>Centrifuge the filtered liquid at 250 g for 10 minutes.</LI><LI>Add the supernatant to a cell culture plate well, then resuspend the pellet in 300 uL of Schneider’s Insect medium and add to a different well.</LI></OL></DIV></DIV></DIV></DIV></DIV><TITLE>IGEM_TECCEM</TITLE><FOOTER class="textoUbuntu" style="padding-top:25px"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-3 align-self-center"><DIV class="row"><DIV class="col-10"><P> Mail:
								<A href="mailto:igem.tec.cem@gmail.com">igem.tec.cem@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>