<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TecCEM_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TecCEM/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM_TECCEM</TITLE></P><DIV class="collapse navbar-collapse rightMenu" id="navbarSupportedContent"><UL class="navbar-nav mr-md-6 paddingNavBar"><LI class="nav-item active"><A class="nav-link whiteFont" href="https://2017.igem.org/Team:TecCEM">Home<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/Description" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Design">Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Experiments">Experiments</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Notebook">Notebook</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/InterLab">Interlab</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Model">Model</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Demonstrate">Demonstrate</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Improve">Improve</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Attributions">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/Parts" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Parts</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Basic_Part">Basic Parts</A></DIV></LI><LI class="nav-item active"><A class="nav-link whiteFont" href="https://2017.igem.org/Team:TecCEM/Safety">Safety<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/PublicEngagement" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/HP/Silver">Silver HP</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/HP/Gold_Integrated">Integrated and Gold</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Engagement">Public Engagement</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/software" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Awards</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Applied_Design">Applied Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Hardware">Hardware</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Model">Model</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Software">Software</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle whiteFont" href="https://2017.igem.org/Team:TecCEM/team" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</A><DIV class="dropdown-menu smallU whiteFont dropdown-menu-navbar" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:TecCEM/Collaborations">Collaborations</A></DIV></LI></UL></DIV><TITLE>Results</TITLE><DIV class="container"><DIV class="row"><DIV class="col-12"><H1 class="titleRed">Results</H1></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop">siRNA confirmation</H1><P>The siRNA confirmation protocol was made to test, with the real time PCR, the presence of the siRNAs designed by the iGEM Tec Cem team. The four devices capable of producing the siRNAs were tested: BSLA-SOD, BSLA-WNT, BSLA-RACI and BSLA -AWD. The protocol used for this purpose is based on the liquid northern hybridization proposed by Wang X., Tong Y. and Wang S. (2010). The protocol is described as a novel form to detect miRNAs. We expect to detect the siRNAs that our different constructs can produce taking advantage of their hybridization with the forward and reverse templates of their DNA sequences and the ability of the exonuclease to digest the non-hybridized RNA sequences.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-5 paragraphU smallP"><P>Figure 1. RNA from the HTT115 strain witouth the siRNA construct and with the siRNA construct (BSLA-AWD) with different hybridization oligos and treatments in a 12% no-denaturing gel at 100 Volts..</P></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 2. RNA from the HTT115 strain with the siRNA construct (BSLA-SOD, BSLA-WNT and BSLA-RACI) with different hybridization oligos and treatments in a 12% no-denaturing gel at 100 Volts.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>Second BSLA siRNA confirmation</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-5 paragraphU smallP"><P>Figure 3. RNA from the HTT115 strain witouth the siRNA construct and with the siRNA construct (BSLA-AWD) with different hybridization oligos and treatments in a 12% no-denaturing gel at 100 Volts.</P></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 4. RNA from the HTT115 strain with the siRNA construct (BSLA-SOD, BSLA-WNT and BSLA-RACI) with different hybridization oligos and treatments in a 12% no-denaturing gel at 100 Volts.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>The first test of the control (only RNA of HTT115) revealed the presence of two bands in the no-denaturing gel when no exonuclease is added to the reaction of approximately 150 pb and 100 pb respectively. When exonuclease is added to the same sample a little band is observed below the 50 bp. When 4 μL of a 100μM dna oligo was added to the same sample, the samples seem to migrate better than the last two and a great amount of material stayed in the low part of the gel, probably because of the high amount of non-hybridized oligo.
                    For the BSLA samples, interestingly, the behavior between the samples treated with the forward and reverse DNA template was different. The RNA samples treated with the forward template sequence migrated less than the ones treated with the reverse, probably because they really hybridized with the siRNA. The RNA samples treated with the reverse sequence migrated really similar to the RNA of HT115 treated with the oligo and exonuclease, a behavior that might mean that the oligos didn´t hybridize.
                    We expected to observe hybridization with both templates (forward and reverse), since the construction for each siRNA has their own sense and anti-sense sequence. The results in both confirmations might mean that the siRNA might hybridize with the forward sequence but no with the reverse. Together with the real time PCR we can conclude that we have a certain amount of siRNA but to characterize their hybridization, we should changes the amount of material for the experiment.
                    The second confirmation of siRNA was repeated with less RNA material (2 μg instead of 5 μg) and more units of exonuclease (20 instead of 10) expecting to distinguish more defined bands. Nevertheless, since it was used a lower dilution of GelRed seems brighter than the first confirmation and the observed material is similar. Next studies might use less RNA and oligos as in the protocol proposed by Wang X., Tong Y. and Wang S. (2010) where they used 1 μg of RNA.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop">Blue BSLA colonies</H1><P>The following figures show blue fluorescent colonies of <SPAN class="italicText">E. coli</SPAN> HT115. The color of the bacteria represents the presence of the BSLA construct and production of siRNA. </P></DIV></DIV><DIV class="row"><DIV class="col-md-10"><DIV class="row"><DIV class="col-md-4 paragraphU smallP"><P>Figure 11. First transformation in DH5a of the BSLA-AWD device from Genscript. The characteristic blue color from the blue chromoprotein is observed in the colonies.</P></DIV><DIV class="col-md-4 paragraphU smallP"><P>Figure 12. First transformation in DH5a of the BSLA-AWD device in PSB1C3. The characteristic blue color from the blue chromoprotein is observed in the colonies. </P></DIV><DIV class="col-md-4 paragraphU smallP"><P>Figure 13. First transformation in HT115 of the BSLA-AWD device in PSB1C3. The characteristic blue color from the blue chromoprotein is observed in the colonies. </P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop">Sequencing</H1><P>It is fundamental for us to make sure that all our BioBricks have the correct sequence before sending them to Boston for the parts registry. Therefore, we sequenced every single piece, to confirm that the ligated products into the pSB1C3 backbone correspond to the original sequence.The BioBricks cloned by the iGEM TEC CEM 2017 team in PSB1C3 were sequenced in order to test if the inserts for the siRNAs and T7 terminator were really ligated. For the following samples of BSLA devices (Blue chromoprotein, forward siRNA, loop and reverse siRNA): 1, 2, 3, 4 and 5 the sequencing was made using the VF2 primer. Although it was possible to confirm the presence of the blue chromoprotein, the reaction couldn´t show us the presence of the siRNAs constructs because of the length of the protein gene before them. In these cases, a reaction using the VR primer is recommended since there are approximately in each case a total of 220-230 pb since the reverse primer to the siRNAs constructs. The single presence of the siRNAs sequence (5’-3’) in PSCB1C3 without all the machinery (blue chromoprotein, loop and siRNA anti sense) was confirmed for the RacI and SOD constructs. The Wnt construct wasn’t sequenced and the results for the Awd construct weren´t conclusive. Also, the presence of the sequenced used as the control of the siRNAs (CNT) was confirmed to be cloned in PSB1C3 (sample 7). The sample 8 correspond to the single sequence of the T7-Terminator used during the construction of the BSLA devices, it was confirmed to be cloned in the PSB1C3. For the samples of BSLA device, we have to repeat the sequencing using the VR primers for the BSLA-SOD and BSLA-RACI, so for this samples we have the sequence forward and reverse with the objective to seek the siRNA construction before the blue chromoprotein, however the results weren't conclusive since the alignment showed the correct base pairs just after the reverse primer but not the same sequence before the terminator. And finally the sequencing for BSLA-AWD in PSB1C3 wasn’t repeated, since it was given to the team by the Gene script sponsorship, and once cloned in PSB1C3 the colonies became blue.*All the samples were sequenced in a ABI PRISM 310 equipment
                    </P></DIV></DIV><DIV class="row"><DIV class="col-md-10"><H1 class="subTitleUbuntu"><SPAN class="italicText">In vivo</SPAN> transfection</H1></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>The <SPAN class="italicText">In vivo</SPAN> transfection was done by submerging the psyllids for 4 minutes in a 20 ul siRNA drop. The results were used to extract RNA to perform the real time PCR.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU smallP"><P>Figure 21. A specimen was placed in a microtube and then the siRNA drop was added.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu">RNA Extraction</H1><P>For the RNA extraction, psyllids were collected and stored at -80°C. The extraction had a high concentration using this method and the RNA samples were used to perform a RT-PCR. The process for this protocol is shown below.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10"><DIV class="row"><DIV class="col-md-6 paragraphU smallP"><P>Figure 19. 1% agarose gel with TBE Buffer 1X for the Diaphorina citri RNA extraction. The gel was loaded with Fast-DNA Ladder in lanes 3 and 7, and lanes 4 and 8 with RNA. Lanes 3 and 4 were loaded first and ran for 30 minutes at 100 V. Due to the high concentration, lanes 7 and 8 were loaded with less RNA sample and ran for 30 minutes at 100 V.</P></DIV><DIV class="col-md-6 paragraphU smallP"><P>Figure 20. Left, the psyllids were captured and placed on liquid nitrogen for instant death. Right, the yellow supernatant is indicative that the protocol was done succesfully.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop">cDNA Synthesis</H1></DIV></DIV><DIV class="row table"><DIV class="col-md-5 paragraphU"><P>Table 1. Expected products for the amplifications in the Two-step RT-PCR.</P><TABLE><TBODY><TR class="parBackgroundColor"><TH> Product</TH><TH>Expected size</TH></TR><TR><TH> Tubulin </TH><TH> 195 bp </TH></TR><TR class="parBackgroundColor"><TH> Rac I</TH><TH> 193 bp </TH></TR><TR><TH> WNT</TH><TH> 200 bp</TH></TR><TR class="parBackgroundColor"><TH> AWD </TH><TH> 180 bp </TH></TR><TR><TH> SOD </TH><TH> 199 bp</TH></TR></TBODY></TABLE></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 10. 2.5 % agarose gel with TBE Buffer 1X for the RT-PCR. The gel was loaded with Fast Ladder in lane 2 and 8 and with the samples listed in table N. The gel was stained with GelRed and run for 60 minutes at 100 volts.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>As we can see in the figure, all the products for the Tubulin, AWD, SOD and WNT rely within the 0.150 and 0.300 kb marker bands, corresponding to the expected base pair sizes. The only product that remains outside the expected size is the RacI amplification product, which was observed in last week’s gel. The primers when tested in silico using PrimerBlast show the 193 bp product, but this does not happen in the gels. Since the reported sequence states “like-sequence” it is possible that there are mismatches between the reported sequence and the real one. It is still unclear why do the primers not generate the expected product. There are other predicted sequences such as XM_008482020.1 and XM_017443825.1, as this sequences are not specific the primers do not amplify this sequences either.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu">Real Time PCR Melt Curves</H1><P>SYBR® Green dye will detect any double-stranded DNA, including: primer dimers, contaminating DNA, PCR product due to mis-annealed primers and the PCR product of interest. By viewing a dissociation/melt curve, you ensure that the desired amplicon was detected. The presence of a single peak indicates that you have a specific product that can be corroborated by electrophoresis. For all amplicons we got a single peak that is in accordance with the expected size of the amplicons except for RacI. We checked primer specificity with blast tools and after a revisión of NCBI databases of transcriptomics we found that RacI have other isoform. It suggests that the fragment of 50 bp could belong to a different isoform for which the primers were designed. Furthermore both isoforms share the region where siRNA was designed.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8 paragraphU smallP"><P>Figure 5. Melting curves for Tubulin, Awd, RacI, SOD and WNT.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop">Real Time PCR</H1></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>
                    Now knowing that our siRNA molecules generate all the expected products (except RacI), we proceeded to the qRT-PCR, which allowed us to measure the amount of mRNA for the corresponding genes at any given time in the cells. We used SYBR Green as an intercalating agent and got the following results. The experiment results were divided on three groups:
                    Experiment A: We performed the soaking method with all the siRNA, the synthetic ones; AWD and RAC1. The ones we created using our raw BSLA Method; SOD and WNT. The purified samples of SOD and WNT, the sp and wp stand for pure. And finally a mix of RAC1, WNT, AWD and SOD to learn about the expression when these siRNA are mixed. 
                    Experiment B: By applying chitosan to the leaves and feeding the diaphorina we tested two genes RAC1 and AWD. 
                    Experiment C: We tested different concentrations to observe which one was the best option for our essay, we also performed this experiment in order to prove our mathematical modeling. 
                    The following figures represent our Real Time PCR results.</P></DIV></DIV><DIV class="row"><DIV class="col-md-5 paragraphU smallP"><P>Figure 6: Effect of the different siRNA molecules over the mRNA production in <SPAN class="“" italictext="">Diaphorina citri</SPAN> after <SPAN class="“" italictext="">in vivo</SPAN> soaking application.</P></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 7: Effect of the siRNA molecules mix over the mRNA production in <SPAN class="“" italictext="">Diaphorina citri</SPAN> after <SPAN class="“" italictext="">in vivo</SPAN> soaking application.</P></DIV></DIV><DIV class="row"><DIV class="col-md-5 paragraphU smallP"><P>Figure 8: Effect of the chitosan-encapsulated siRNA molecules over mRNA production in <SPAN class="“" italictext="">Diaphorina citri</SPAN> after direct application over the plant leaves.</P></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 9: Effects of different concentrations (ng/uL) of siRNA-AWD tested in <SPAN class="“" italictext="">Diaphorina citri</SPAN> after in vivo soaking application to prove the proposed mathematical modeling.
                    </P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>
                    The results from Figure 6 show that both synthetic and E. coli HT115 produced siRNA molecules were able to hybridize with its target sequence, inducing effective gene silencing by reducing the mRNA production. As it can be appreciated, the WNT siRNA reduced its target mRNA up to three times, being the highest-impact molecule, followed by AWD.
                    Figure 7 shows the effect of treating the psyllids with a mixture of the different siRNA molecules rather than with each siRNA separately. Once again, the WNT gene showed the strongest level of mRNA reduction, but in a much higher scale than with the solo treatment. The AWD siRNA is second, with almost the same impact as in the previous figure. The importance of this graph relies within the siRNA molecules for WNT and SOD produced with the novel BSLA BioBrick in Escherichia coli HT115 are functional on their own, which means that the BSLA has a great potential for future siRNA production in bacterial systems. It is important to notice that the siRNA have a higher activity in the Diaphorina citri mRNA production when applied together in a mix rather than individually.
                    The results observed in Figure 8 give us crucial information about the possible upscaling of our project via chitosan-encapsulated siRNA molecules and their direct application over the plant leaves. The main question was if the psyllids were attracted to the molecules and could feed on them. Since we observe reduction in the levels of mRNA, we can say that Diaphorina citri in fact fed on the chitosan. When comparing the treated psyllid with the negative control, it can be concluded that the AWD siRNA has the greatest reduction of the mRNA, while RacI has a lower direct impact in the mRNA expression.
                    In Figure 9, we tested different concentrations of the AWD siRNA to prove the mathematical modeling. All the concentrations yielded mRNA change expressions in the psyllids, but the most effective one was 60 ng/uL, followed by 20 ng/uL. The math model was developed to graph amount of siRNA to mRNA expression this helps us validate the results by analyzing the reduction of this expression, it also helps us determine which is the best concentration to be used to control the HLB plague.
qRT calculations
</P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop"><SPAN class="italicText">Diaphorina citri</SPAN> primary cell culture</H1><P>The main objetive was to develop a primary culture of Diaphorina citri cells, to be transfected with our siRNA and analyzed through flow cytometry. The transfected siRNA sequences were tagged with Alexafluor so they would be visible. This protocol was repeated several times under different conditions, and different compositions of the culture medium were used, which can be seen below. The cells that were cultured in Medium B were able to survive after twelve hours, and replicated at a slow rate. Because of this, the flow cytometry was not performed.
                    
                    Table 2. Composition of different mediums for cell culture.
                </P></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><TABLE style="padding-bottom:30px"><TBODY><TR class="parBackgroundColor"><TH>Reagent</TH><TH>Medium A</TH><TH>Medium B</TH></TR><TR><TH>200 mM L-Glutamine</TH><TH>1 mL</TH><TH>1 mL</TH></TR><TR class="parBackgroundColor"><TH>Penicillin and Streptomycin 1000X</TH><TH>1.5 mL</TH><TH>1.5 mL</TH></TR><TR><TH>Fungizone 500X</TH><TH>0.2 mL</TH><TH>0.2 mL</TH></TR><TR class="parBackgroundColor"><TH>Schneider Medium</TH><TH>72 mL</TH><TH>80 mL</TH></TR><TR><TH>199 Medium 10X with Hanks’ salts</TH><TH>6 mL</TH><TH>-----</TH></TR><TR class="parBackgroundColor"><TH>Fetal Bovine Serum (heat-inactivated)</TH><TH>20 mL</TH><TH>20 mL</TH></TR><TR><TH>0.06 M Histidine solution</TH><TH>1 mL</TH><TH>-----</TH></TR></TBODY></TABLE></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10"><DIV class="row"><DIV class="col-md-4 paragraphU smallP"><P>Figure 14. Cell culture 12 hours after extraction.</P></DIV><DIV class="col-md-4 paragraphU smallP"><P>Figure 15. Cell culture 2 days after extraction, observed under an inverted microscope at 10x. No significant cell replication can be observed compared to the first day,</P></DIV><DIV class="col-md-4 paragraphU smallP"><P>Figure 16. Cell culture 2 weeks after extraction, observed under an inverted microscope at 10x. Cell replication can be observed.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU smallP"><P>Figure 17. The body was separated in three parts: 1. head and wings, 2. intestines, 3. abdomen. Cell cultures were made with parts 1 and 3, while number 2 was discarded to avoid contamination.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu">Chitosan test</H1><P>The test consists in encapsulating siRNA, in this case of Rac1 and AWD. The concentration is of 100ng/ul of the siRNA used.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10 paragraphU"><P>The results of the silencing can be viewed in the protocol involving the real time PCR, but it is important to note that the insect does it the leave and it does show mRNA reduction.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10 paragraphU smallP"><P>Figure 18. Damage observed in the leaves following chitosan application.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10 paragraphU"><P>The only problem of the chitosan protocol is suspected to be the chitosan concentration as it produces a thin film over the leaves, which caused them to be blocked and the leaves died.</P></DIV></DIV><DIV class="row" style="padding-top:40px"><DIV class="col-md-10 paragraphU"><H1 class="subTitleUbuntu paddingTop">Diaphorina citri laboratory development</H1><P>Special cages were built for rearing and reproduction of Diaphorina citri specimens across three months. Each cage was double lined with fine white mesh to prevent the insects from escaping while allowing us to inspect them. On top, a set of bulbs was placed and connected to a timer, to control daily light hours, which were set at 16 hours of light and 8 of darkness. Additionally, two fan heaters were placed at a distance of 1 meter, which increased the temperature inside the cages to up to 30ºC. While this is not the ideal temperature for reproduction, the presence of eggs and nymphs could be observed after two weeks.
                    The initial population across all cages was of 80 insects, and after approximately one month an increase of almost double was observed. However, due to the psyllid's lifespan, a rapid decline was observed shortly after. In addition to this, the increase in population made it harder for each psyllid to feed on one plant, which increased mortality.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10"><DIV class="row"><DIV class="col-md-4 paragraphU smallP"><P>Figure 22. Detail of mesh lining the interior of the cages. The size of the mesh prevents the insects from escaping while providing light, air flow and visibility.</P></DIV><DIV class="col-md-4 paragraphU smallP"><P>Figure 23. Presence of dew in the plant's leaves, resulting from nymphs feeding on the plant, which indicates reproduction.</P></DIV><DIV class="col-md-4 paragraphU smallP"><P>Figure 24. Detail of plant leaves after two weeks of rearing. The presence of eggs in the stems of the plant can be observed.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-5 paragraphU smallP"><P>Figure 25. Complete set-up for rearing of Diaphorina citri, consisting in two mesh-lined cages and complete with light and temperature conditioning.</P></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 26. Presence of adult Diaphorina citri specimens after one month of rearing.</P></DIV></DIV><DIV class="row"><DIV class="col-md-10"><H1 class="subTitleUbuntu">Morphology</H1></DIV></DIV><DIV class="row"><DIV class="col-md-10 paragraphU"><P>Different specimens were observed under a stereoscopic microscope and identified by gender.</P></DIV><DIV class="row"><DIV class="col-md-5 paragraphU smallP"><P>Figure 27. Male specimen.</P></DIV><DIV class="col-md-5 paragraphU smallP"><P>Figure 28. Female specimen.</P></DIV></DIV></DIV><TITLE>IGEM_TECCEM</TITLE><FOOTER class="textoUbuntu" style="padding-top:25px"><DIV class="container-fluid"><DIV class="row"><DIV class="col-md-3 align-self-center"><DIV class="row"><DIV class="col-10"><P> Mail:
								<A href="mailto:igem.tec.cem@gmail.com">igem.tec.cem@gmail.com</A></P></DIV></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>