<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Results_Recombinant"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Results/Recombinant">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Results/Recombinant&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Results/Recombinant&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Results/Recombinant&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Results/Recombinant" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Results/Recombinant</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Characterization_of_recombinant_effector_proteins">Characterization of recombinant effector proteins</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:162px;"><DIV class="thumbcaption"><B>Figure 1:</B> Effectors</DIV></DIV></DIV><P>For the development of a transgenic water filter it is an essential task to create a collection of well described and functional effector proteins which are either able to bind (<A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation" class="external text" rel="nofollow">BioAccumulation</A>) or to degrade (<A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation" class="external text" rel="nofollow">BioDegradation</A>) xenobiotics which are present in the aquatic environment. This task was completed by the production of relevant effector proteins in <I>E. coli</I> and their subsequent purification and characterization. Deliberately we have chosen some well established BioBricks from the last years such as a laccase (&lt;partinfo&gt;BBa_K1159002&lt;/partinfo&gt;) or the catechol dioxigenase (&lt;partinfo&gt;BBa_K648011&lt;/partinfo&gt;) to improve these BioBricks. Beside these improvements we also added new BioBricks to the registry which we characterized <I>in vitro</I> such as the erythromycin esterase (EreB) (&lt;partinfo&gt;BBa_K1159000&lt;/partinfo&gt;) or the NanoLuc luciferase (&lt;partinfo&gt;BBa_K1159001&lt;/partinfo&gt;) which will be a useful tool for subsequent generations of iGEM students. For technical questions on our experiments, please see <A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods" class="external text" rel="nofollow">protein biochemical methods</A> for further information. Beside these experiments with recombinant proteins we also characterized our stable transformed moss strains (<A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss" class="external text" rel="nofollow">see PhyscoFilter section</A>).
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 1:</B> Investigated Proteins
</CAPTION><TBODY><TR><TH>Protein
</TH><TH>BioBrick
</TH><TH>RFC
</TH><TH>Size [kDa]
</TH><TH>Disulfide bridges
</TH></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#Eryhtromycin_Esterase_.28EreB.29" class="external text" rel="nofollow"><B>Erythromycin esterase</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1159000" class="external text" rel="nofollow">BBa_K1159000</A></TD><TD>RFC[25]
</TD><TD align="right"> 48.5
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#Laccase" class="external text" rel="nofollow"><B>Laccase</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1159002" class="external text" rel="nofollow">BBa_K1159002</A></TD><TD>RFC[25]
</TD><TD align="right"> 58.8
</TD><TD>yes
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#Nano_Luciferase" class="external text" rel="nofollow"><B>Nano Luciferase</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1159001" class="external text" rel="nofollow">BBa_K1159001</A></TD><TD>RFC[25]
</TD><TD align="right"> 19.4
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#Catechol_Dioxigenase_.28XylE.29" class="external text" rel="nofollow"><B>XylE</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K648011" class="external text" rel="nofollow">BBa_K648011</A></TD><TD>RFC[10]
</TD><TD align="right"> 4 x 35.4
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#Protein_phosphatase_1_.28from_Homo_sapiens.29" class="external text" rel="nofollow"><B>PP1</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1159004" class="external text" rel="nofollow">BBa_K1159004</A></TD><TD>RFC[25]
</TD><TD align="right"> 37.4
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#DDT-Dehydrochlorinase" class="external text" rel="nofollow"><B>DTT Dehydrochlorinase</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K620000?title=Part:BBa_K620000" class="external text" rel="nofollow">BBa_K620000</A></TD><TD>RFC[10]
</TD><TD align="right"> 23.4
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#SpyCatcher_.26_SpyTag" class="external text" rel="nofollow"><B>SpyCatcher</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1159200" class="external text" rel="nofollow">BBa_K1159200</A></TD><TD>RFC[25]
</TD><TD align="right"> 12.6
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#SpyCatcher_.26_SpyTag" class="external text" rel="nofollow"><B>SpyTag</B></A></TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159201" class="external text" rel="nofollow">BBa_K1159201</A></TD><TD>RFC[25]
</TD><TD align="right"> 1.9
</TD><TD>none
</TD></TR><TR><TD><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant#YFP_TEV_CFP" class="external text" rel="nofollow"><B>YFP_TEV_CFP</B></A></TD><TD><A href="http://parts.igem.org/Part:BBa_K1159112" class="external text" rel="nofollow">BBa_K1159112</A></TD><TD>RFC[25]
</TD><TD align="right"> 55.2
</TD><TD>none
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Erythromycin_Esterase_.28EreB.29">Erythromycin Esterase (EreB)</SPAN></H2><P>The erythromycin esterase (EreB) is an enzyme found in some strains of <I>Escherichia coli</I> and which was introduced to the Parts Registry by the TU Munich iGEM Team 2013 in RFC[25]. As this BioBrick is derived from <I>Escherichia coli</I> it was to expect that it is well expressing in our experiments on recombinant effector proteins. For further information on the theoretical background of EreB please see our <A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation" class="external text" rel="nofollow">BioDegradation page</A>.
</P><H3><SPAN class="mw-headline" id="Production_and_purification_of_recombinant_EreB">Production and purification of recombinant EreB</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure A:</B> Streptavidin affinity chromatography for the erythromycin esterase</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure B:</B> Analytical size exclusion chromatography on a Superdex 200 10/30 column</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure C:</B> Preparative size exclusion chromatography on a Superdex 75 16/60 column</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure D</B>: SDS-gel of recombinant EreB with the marker (M) followed by the concentrated throughput of the streptavidin affinity column and 6 fractions collected from the elution peak</DIV></DIV></DIV><P>The recombinant production and purification was carried out twice, in a first attempt 2 L of LB-media were used for an analytical purpose whereas in the second attempt we produced enough purified enzyme for all subsequent experiments. This preparation was carried out in 6 x 2L of LB media. Protein production was in both cases induced at OD = 0.8 by adjusting the cell culture to 5 mM of arabinose and was carried out for 4 h for the first and 5 h for the second preparation. Cell disruption was performed by ultrasonic sound in both cases. The cell lysate was then dialyzed against 5 L of SA-buffer and subsequently applied to streptavidin affinity columns. After the application of the protein, the column was washed with SA-buffer until a base line was reached. Afterwards the protein was eluted using 5 mM biotin. During the first preparation 2-mercapto-ethanol was added after the chromatographic steps. In order to avoid oxidation of cysteine residues to disulphid-bridges, which is not desired for the cytosolic EreB protein, the preparative purification was carried out with buffers containing 5 mM of 2-mercapto-ethanol in all buffers. When comparing the size exclusion chromatograms, obtained from the analytical and the preparative purification, it can be stated that there is still a considerable aggregation peak near the void volume (Fig. B) of the column in the first attempt, which was nearly not the case for the preparative preparation (Fig. C). Therefore we would give the advise to use strictly reducing conditions while working with recombinant EreB. The finally resulting yields of the preparative purification have been determined by absorption measurement of the aromatic amino acids at 280. The total yield was determined to 25 mg of pure protein which is 2.1 mg/L of LB culture.
</P><H3><SPAN class="mw-headline" id="Kirby-Bauer_Assay:_Measuring_remaining_erythromycin_on_a_pertri_dish">Kirby-Bauer Assay: Measuring remaining erythromycin on a pertri dish</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 2:</B> Kirby-Bauer assay of recombinant EreB incubated with erythromycin</DIV></DIV></DIV><P>The Kirby-Bauer assay is an <B>agar diffusion test</B>, with which it is very easy to examine the decomposition activity of the enzyme. We also analyzed the decomposition activity of the recombinant EreB by <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss#The_PhyscoFilter_for_Erythromycin" class="external text" rel="nofollow">LC-MS</A>. The recombinant protein is incubated with the antibiotic and the reaction is stopped with methanol. To be sure that the enzyme is inhibited and the reaction does not go on, we additionally heated the reaction mixture (see table 2) after stopping for two minutes at 50Â°C and then shock froze the mixture in liquid nitrogen. The effect of methanol, heating and shock freezing on the bacterial strain and the substrate were checked before and have no influence. </P><P>Then bacteria from a quite dense liquid culture are plated on a LB agar plate without antibiotics under sterile conditions and spread with sterile cotton tip applicators. We used the bacterial strain <I>Micrococcus luteus</I> which was generously donated from the <A href="http://www.micbio.wzw.tum.de/cms/docs/scherer-anzeigen.php" class="external text" rel="nofollow">chair of microbiology by Prof. Scherer`s group</A> and is mentioned to be specifically sensitive to Erythromycin [<A href="http://pubs.acs.org/doi/abs/10.1021/bi201790u?mi=0&amp;af=R&amp;pr..." class="external text" rel="nofollow">Wright et al., 2012</A>]. Now several 6 mm filter paper  discs are placed on the bacterial lawn in adequate intervals and 8 Âµl of the spinned down reaction mixture is added onto one disc. The mixture diffuses from the filter paper into the agar. The concentration of the compound will be highest next to the disk, and will decrease with increasing distance. If the compound is effective against bacteria at a certain concentration, no colonies will grow where the concentration in the agar is greater than or equal to the effective concentration. This is called the <B>zone of inhibition</B>.
</P><TABLE><CAPTION><B>Table 2:</B> Reaction mixture for Kirby-Bauer assay
</CAPTION><TBODY><TR><TH> substance
</TH><TH> amount
</TH><TH> stock solution
</TH></TR><TR><TD> EreB recombinant protein 40 nM
</TD><TD align="right"> 0.18 Âµl
</TD><TD> 11 ÂµM EreB in PBS with 10 mM Ã-mercaptoethanol, 2% glycerol (v/v) and 300 mM NaCl
</TD></TR><TR><TD> Erythromycin 0.36 mM
</TD><TD align="right"> 3 Âµl
</TD><TD> Erythromycin in ethanol, 6 mM
</TD></TR><TR><TD> Tris-HCl buffer pH 7.5 100 mM
</TD><TD align="right"> 10 Âµl
</TD><TD> Tris-HCl buffer pH 7.5, 500 mM
</TD></TR><TR><TD> NaCl 0.08 M
</TD><TD align="right"> 4 Âµl
</TD><TD> NaCl in water, 1 M
</TD></TR><TR><TD> ddH20
</TD><TD align="right"> 32.82 Âµl
</TD><TD> ddH20
</TD></TR><TR><TD><B>TOTAL:</B></TD><TD align="right"><B>50 Âµl</B></TD></TR></TBODY></TABLE><P>As negative controls there is one mixture containing no substrate or enzyme and two mixtures without the enzyme but with antibiotic, which were incubated for 0 minutes and 6 hours. 
</P><P>As expected the assay shows a gradually decreasing zone of inhibtion with increasing reaction time. In the first half hour the zone is more or less constant although the enzyme is constantly degrading the antibiotic. This can be explained by the fact that the mixture only diffuses a certain range into the agar and in the first few time steps of the reaction the concentration of the antibiotic within this range is above the <A href="http://en.wikipedia.org/wiki/Minimum_inhibitory_concentration" class="external text" rel="nofollow">minimum inhibitory concentration (MIC)</A>, so the bacteria is inhibited independently of the exact concentration. After 3 hours all Erythromycin was degraded so there is no zone of inhibition anymore. Also we can see that the antibiotic does not degrade by itself significantly over 6 hours since the zones of inhibition in both negative controls are pretty much the same.
</P><H3><SPAN class="mw-headline" id="Characterisation_of_the_enzyme">Characterisation of the enzyme</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 3:</B> Reaction mechanism of EreB hydrolyzing the model ester p-NPB into a chromogenic product</DIV></DIV></DIV><P>To characterize the enzyme we performed a spectrophotometric assay using the model ester p-nitrophenyl-butyrate (p-NPB) as a substrate for EreB. The chromogenic product p-Nitrophenyl shows characteristic absorbance at 405 nm. The reaction takes place even faster than hydrolysis of erythromycin since p-NPB is already highly activated and therefore readily hydrolyzed. This is also the reason why it degrades quite rapidly by itself so reactions have to be measured soon after being set up and taking a negative control is especially important.
</P><H4><SPAN class="mw-headline" id="Kinetics">Kinetics</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4:</B> 96-well plate showing the substrate serial dilution incubated with EreB (1-4) and as negative control with water (0)</DIV></DIV></DIV><P>To determine the kinetic parameters of EreB we prepared a 100 Âµl serial dilution of the substrate, starting with 1.2 mM p-NPB (in 0.4% Triton, 50 mM buffer Tris-HCl pH 7.5, 0.2 M NaCl) to which 100 Âµl of the enzyme (11 ÂµM) were added. The experiment was performed in quadruplets and the mean value was then plotted (see figure 5) against substrate concentrations.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 5:</B> Reaction of EreB with a serial dilution of p-NPB (left) and negative control without the enzyme (right)</DIV></DIV></DIV><P><B>Result</B>:  The higher the substrate concentration the faster the substrate is hydrolyzed, which can be seen in an increasing gradient and is self-evident as the enzyme encounters the substrate more often and faster the higher its concentration is in the solution. This data was also used for an exact determination of the kinetic parameters in <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme#Erythromycinesterase_.28Substrate_Dependence.29" class="external text" rel="nofollow">our modelling section</A>. At these reaction conditions (pH 7.5) almost no decomposition of p-NPB takes place which is confirmed in the following experiment.
</P><H4><SPAN class="mw-headline" id="PH_optimum">PH optimum</SPAN></H4><P>We prepared a buffer series from pH 4.5 to pH 10 using Sodium acetate + acetic acid (pH 4, 4.5, 5), MES (pH 5.5, 6), HEPES (6.5, 7), Tris-HCl (7.5, 8, 8.5) and Ches (9, 9.5, 10), each 100 Âµl in a 96-well plate. To the buffer we added 50 Âµl Substrate p-NPB (2 mM in 0.8% Triton and 400 mM NaCl) and finally 100 Âµl of the enzyme (11 ÂµM) to start the reaction. The reaction was followed with a spectrophotometer at 405 nm for 1 hour every 15 seconds with the reaction mixture being shaken before every read. The experiment was performed threefold and the mean value was then plotted (see figure 6) against the pH-value of the buffer.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 6:</B> Reaction of EreB with p-NPB in different pH buffers (left) and negative control without the enzyme (right)</DIV></DIV></DIV><P><B>Result</B>: From pH 4.5 to 6.5 there was no enzyme activity because the enzyme degraded, which was clearly visible as a dreary solution. The small absorption can be traced back to the degraded enzyme, so the conclusion is that EreB is unstable in alcaline surroundings. This is also the reason why <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme#Erythromycinesterase_.28Substrate_Dependence.29only" class="external text" rel="nofollow">cytoplasmatic localized enzyme works in our transformed moss</A>, because the Knop medium in which <I>P. patens</I> is grown has a pH of 5.8. Since natural aquatic environment is generally on the alcalic side the degradation of antibiotics with EreB can only be realized intracellular. P-NPB in contrast seems to be very stable at low pH as there is no absorption measured in the negative control. With increasing pH the substrate increasingly degrades by itself while the enzyme's activity is also increasing. These two factors lead to a rapidly growing conversion of p-NPB. In a nutshell EreB works optimally in an acidic environment.
</P><H4><SPAN class="mw-headline" id="Ionic_strength_optimum">Ionic strength optimum</SPAN></H4><P>A serial dilution of NaCl starting with 1.5 M was prepared (50 Âµl) and another 100 Âµl of p-NPB (2 mM in 0.8Â % Triton) were added before the reaction was started with 50 Âµl of the enzyme (11 ÂµM). The reaction was followed with a spectrophotometer at 405 nm for 1 hour every 20 seconds with the reaction mixture being shaken before every read. The experiment was performed threefold and the mean value was then plotted (see figure 7) against the varying salt concentrations.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 7:</B> Reaction of EreB with p-NPB at different salt concentrations and negative control without the enzyme (right)</DIV></DIV></DIV><P><B>Result</B>: The lower the salt concentration the higher is the enzymatic activity while simultaneously the decomposition of p-NPB increases. However, the reaction does not reach saturation as in the reaction with different pH (see figure 6). This is intuitive since the double amount of substrate and half of the enzyme's amount were incubated.
</P><H2><SPAN class="mw-headline" id="Laccase">Laccase</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_Animation_Laccasex.gif" title="File:TUM13 Animation Laccasex.gif">File:TUM13 Animation Laccasex.gif</A><DIV class="thumbcaption"><B>Figure 8:</B> Animation of protein structure of Laccase</DIV></DIV></DIV><P>Laccases are an important group of effector proteins that can be utilized in bioremediation to degrade xenobiotics. They are copper-containing enzymes which catalyze the oxidation of a wide range of substances. It was interesting for us as the two important xenobiotics ethinylestradiol and diclofenac can be degraded using laccases and the general function of the enzyme can easily be investigated using the commonly used substrate ABTS which results in a chromogenic product.
During the planing phase of our project we decided not to add a new laccase to the Parts Registry but instead to continue the work on the best availible laccase. The iGEM team <A href="https://2012.igem.org/Team:Bielefeld-Germany" class="external text" rel="nofollow">Bielefeld Germany</A> 2012 has created BioBricks for five different laccases during the last summer. So we contacted this iGEM team and got the advise that the laccase derived from <I>Bacillus pumilus</I>&lt;partinfo&gt;BBa_K863000&lt;/partinfo&gt; seems to be the best choice for our effector collection. As we wanted to try this effector protein in different localisations such as cytoplasmatic, secreted, or receptor bound we converted this enzyme to RFC[25] in order to make it compatible for protein fusions.
</P><H3><SPAN class="mw-headline" id="Bioinformatics:_Laccase_-_a_secreted_enzyme">Bioinformatics: Laccase - a secreted enzyme</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 9:</B> Structural analysis of Laccase by homology</DIV></DIV></DIV><P>For the production of effector proteins it is always essential to know whether they are naturally cytoplasmatic or whether they are secreted. Cytoplasmatic proteins usually contain a higher portion of cysteine residues as no disulfide bridges can be formed under the reducing conditions of the cytoplasm. In contrast secreted proteins often do contain disulfide briges which make them more stable to environmental stress. The disulfide formation is possible for the secreted proteins as the oxidizing extracellular conditions facilitate the bridge formation. On the other hand there is a selection pressure for secreted proteins not to contain exposed cysteine residues that are not involved in the formation of disulfide bridges. Such free cysteine residues would lead to aggregation of proteins which would probably hamper their activity. Therefore secreted enzymes generally only contain a smaller amount of cysteine residues. 
Thus it is important to find out whether the used laccase BPUL is a secreted enzyme as this would increase the chance to express it in a functional form on the extracellular part of our moss. To solve this question we did an alignment of the laccase BioBrick and the closest related solved structure. The structure contains disulfide bridges for example Cys<SUP>229</SUP>-Cys<SUP>322</SUP>. The comparison of the position of these residues within the amino acid sequence alignment shows that these residues are conserved and it can thus be assumed that the laccase BioBrick we used also contains disulfide bridges that have to be formed under oxidizing conditions and will increase the stability of our effector protein.
We thus used 10 mM of potassium ferrocyanide which is a milde oxidization reagent subsequent to the cell disruption and did not use any 2-mercaptoethanol as a reducing agent during the protein purification.
</P><H3><SPAN class="mw-headline" id="Analytical_preparation">Analytical preparation</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 10:</B> Streptavidin affinity chromatography</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 11a:</B> Size exclusion chromatography on a Superdex 200 10/30 column</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 11b:</B> SDS-gel of recombinant Laccase with the marker (M) followed by the the cell lysate, concentrated protein throughput of the streptavidin affinity column, 1 fraction of the eluate unconcentrated and concentrated</DIV></DIV></DIV><P>As with all other effector proteins we performed at first an analytical preparation in a 2 L LB-media scale in order to learn about the protein and then up-scaled the production to 6 x 2 L of LB media. The production was carried out cytoplasmatically in <I>E. coli</I> BL21. The culture was grown to OD 0.8 and then induced using 5 mM of arabinose and the protein production was carried out for 5 hours. Cell disruption was performed using ultrasonic sound and subsequently the cell lysate was dialyzed against 1 x SA-Buffer that contained additional 5 mM of the milde oxidation reagent potassium ferrocyanide in order to form the disulfide bridges present in the laccase structure (see previous paragraph).
The streptavidin affinity chromatography yielded a satisfying elution peak that was then concentrated using a centrifugal filter unit (MWCO: 30 kDa). The concentrated protein was then applied to an ÃKTA purifier equipped with a Superdex 200 10/30 size exclusion chromatography column. The chromatogram shows three major peaks with a first peak near the void volume that corresponds to aggregated protein &gt;600 kDa, a second elution peak of high molecular protein and a third elution peak corresponding to the desired laccase protein. This assumption was made on the knowledge that on this column proteins of arround 50 kDa elute at an elution volume of 14 to 15 ml. This assumption was subsequently confirmed by ABTS activity assays with all elution fractions (see next section).
</P><H3><SPAN class="mw-headline" id="Activity_determination_using_ABTS">Activity determination using ABTS</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_ABTS-oxidation.png" title="File:TUM13 ABTS-oxidation.png">File:TUM13 ABTS-oxidation.png</A><DIV class="thumbcaption"><B>Figure 12:</B> Mechanism of the oxidation of ABTS which is a model substrate for laccase activity</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 13:</B> Correlation of size exclusion chromatography (blue) and activity determination using ABTS (red). Relative values are shown.</DIV></DIV></DIV><P>The enzymatic activity of the purified laccase was determined by the ABTS-assay. In a first pre experiment the appropriate dilution factor was determined to be 100-fold. The elution fractions obtained from size exclusion chromatography were diluted 1:100 in PBS and in an ELISA plate. 100 Âµl of the enzyme and 100 Âµl of ABTS substrate were mixed and a kinetic measurement at 405 nm was performed. The absorption at 280 nm in the SEC chromatogramm (blue) identifies three main protein peaks, with a first peak corresponding to aggregated protein, a shoulder which also corresponds to higher molecular protein and a single peak which was proposed to be the monomeric laccase. The relative activity obtained for the different elution fractions was plotted in the same diagramm and shows a clear peak which matches the laccase peak in the SEC. Beside this major peak a second smaller peak of active fraction was visible which appeared in earlier elution fractions and might correspond to dimerized laccase. As the laccase is a secreted enzyme which also bears dislufide bridges it was produced in the cytoplasm and subsequently it was oxidized to form the proper disulfide bonds. As this process might be only partial there is a possiblity for the formation of disulfide dimers. Nevertheless the fractions 14 to 17 were pooled for further experiments as they showed the highest enzymatic activity. The protein concentration of the pooled fraction was determined to be 0.48 mg/ml after SEC.
</P><H3><SPAN class="mw-headline" id="Characterization_of_the_enzyme">Characterization of the enzyme</SPAN></H3><P>The enzymatic activity of the laccase BioBrick was investigated under different conditions as this was important for our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter" class="external text" rel="nofollow">filter model</A> in which we try to simulate the application of the laccase in our remediation raft to clean highly contaminated rivers. The first activity assays were dilution series experiments in which we wanted to determine the appropriate amount of protein which we had to apply in the assays. We found that a 1:100 dilution of the prepared protein (0.49 mg/ml) to be ideal and thus performed all experiments with this concentration. 
</P><H4><SPAN class="mw-headline" id="Kinetic_assay_.28Substrate_dependency.29">Kinetic assay (Substrate dependency)</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 14:</B> 96-well plate showing dilutin series of substrate (ABTS) for kinetic experiments</DIV></DIV></DIV> For the kinetic assay a 1:1 dilution series of the substrate was set up. The oxidation of the ABTS substrate was then monitored over time observing the absorbance at 405 nm. The experiment was performed in triplicates (see Figure 14) and as a control the dilution series of the substrate was incubated with PBS instead of the diluted protein. It can be seen that there is no autolytic oxidation of the substrate. The resulting kinetic parameters are summarized in table 3.
<TABLE><CAPTION><B>Table 3:</B> Best fit parameters
</CAPTION><TBODY><TR><TH> Parameter
</TH><TH> Best fit
</TH></TR><TR><TD> k<SUB>f</SUB></TD><TD> 1.606 E+05 1/s
</TD></TR><TR><TD> k<SUB>r</SUB></TD><TD> 3.450 E-08 1/s
</TD></TR><TR><TD> k<SUB>cat</SUB></TD><TD> 63.522 1/s
</TD></TR><TR><TD> K<SUB>M</SUB></TD><TD> 0.396 mM
</TD></TR></TBODY></TABLE><P>For a detailed evaluation of the results please see our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme#Laccase_.28Substrate_Dependence.29" class="external text" rel="nofollow">enzyme kinetics</A> modeling.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 15:</B> Reaction of Laccase with a serial dilution of ABTS (left) and negative control without the enzyme (right)</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Activity_dependency_on_pH_values">Activity dependency on pH values</SPAN></H4><P>Secondly we did a series of experiments in which we determined the effect of the pH on the enzymatic activity (see Figure 16). For these experiments 100 Âµl of various 500 mM buffer (various pH), 50 Âµl of ABTS substrate (0.5 mg/ ml in ABTS-bufer from Roche) and 50 Âµl of laccase, diluted 1:100, were incubated. The absorbance at 405 nm was then recorded over the time in a 96-well plate. 
The change in absorbance over time is shown for every condition as the average of three measurements with standard deviation. The resulting plot shows that there is an optimum for the pH which is around 5. This matches the acidic conditions present in the apoplast of the plant, which is mainly between pH 5 and pH 6.5, so localizing the enzyme in the interspace between cell wall and membrane is theoretically possible. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 16:</B> Reaction of Laccase with ABTS in different pH buffers. Absorbance curve with laccase on the left; control with PBS on the right</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Influence_of_ionic_strength">Influence of ionic strength</SPAN></H4><P>Beside the substrate dependence and the influence of the pH value, the dependence of the Laccase on the ionic strength of the media was of interest for our project as well as is is for further users of this BioBrick. We thus incubated the recombinant laccase (1:100 dilution) with ABTS substrate and various concentrations of sodium chloride. This was done in triplicates and for a control without the enzyme with PBS instead.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 17:</B> Reaction of laccase with ABTS at different salt concentrations. Absorbance curve with laccase on the left; control with PBS on the right</DIV></DIV></DIV><P>As it can clearly be seen the enzymatic activity is increasing with decreasing salt concentration. For a detailed analysis of the results please look at our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme" class="external text" rel="nofollow">enzyme modelling page</A>.
</P><H4><SPAN class="mw-headline" id="Influence_of_the_temperature">Influence of the temperature</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 18:</B> Reaction of laccase with ABTS at different temperatures</DIV></DIV></DIV><P>To analyze the temperature dependency of the laccase we determined the katalytic activity with a 1:100 enzyme dilution and 0.12 mM of the substrate ABTS. The enzymatic conversion could be followed by the absorbance of the resulting product at 405 nm. The enzyme and substrate were both incubated at the specific temperatures for half an hour before starting the reaction. 
As expected the enzymes efficiency is increasing with temperature, but it is also acceptable at lake and river temperatures which are about 10 to 15 Â°C. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 19:</B> Dependence of the activity of the laccase on the ambient temperature</DIV></DIV></DIV><P>For our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter" class="external text" rel="nofollow">filter modelling</A> we determined k<SUB>cat</SUB>/K<SUB>M</SUB> from these measurements by looking at the initial gradient. We normalized this by equating the enzyme activity at 20Â° C with the activity determined by our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme" class="external text" rel="nofollow">modeling</A> of the substrate dependence of the laccase. This can be assumed, because in the experimental setup the substrate concentration is a lot less than the K<SUB>M</SUB> value we determined, which then simplifies the Michaelis-Menten equation to give, that the initial gradient is proportional to the enzyme activity k<SUB>cat</SUB>/K<SUB>M</SUB>.This gave us the activity dependence shown in Figure 19.
</P><H4><SPAN class="mw-headline" id="Half-life_of_the_laccase_in_river_water">Half-life of the laccase in river water</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 20:</B> Laccase reaction after 0 hours of incubation in river water</DIV></DIV></DIV><P>Laccase with an initial concentration of 0,3125 ÂµM was diluted with 1:1 steps at the initial time, then ABTS at a concentration of 97.2 ÂµM was added to each dilution and the kinetic of the ABTS reaction was recorded using a photometer at 405 nm. Figure 20 shows the results.
</P><P>Then the laccase was dialyzed against river water, taken from the local river Isar. After 96 hours and again after 144 hours, the initial dilution experiment was repeated giving figure 21 any 22, respectively. It is apparent that the enzymes activity has been reduced. 
From these measurements we <B>estimated the half-life to be 36.3 hours</B>, corresponding to an inactivation rate of 5.3 10<SUP>-6</SUP> s<SUP>-1</SUP>, which we then used in our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter" class="external text" rel="nofollow">filter model</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 21:</B> Laccase reaction after 96 hours of incubation in river water</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 22:</B> Laccase reaction after 144 hours of incubation in river water</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Nano_Luciferase">Nano Luciferase</SPAN></H2><P>The Nano Luciferase (NanoLuc) which was introduced in 2013 by Promega is a new member of the luciferase reporter gene/protein familiy and shows some advantages compared to the other family members. The NanoLuc is very small (19 kDa) compared to the firefly luciferase (61 kDa) and the <I>Renilla</I> luciferase (36 kDa). On the other hand it is also said that the specific activity of the NanoLuc is about 150-fold stronger compared to conventional luciferases and the background caused by autoluminescense of the substrate is said to be remarkably smaller. 
</P><H3><SPAN class="mw-headline" id="Production_in_E._coli_and_purification">Production in <I>E. coli</I> and purification</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 23:</B> Analytical size exclusion chromatography on a Superdex 200 10/30 column showing a single elution peak for the NanoLuc.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 24:</B> Preperative size exclusion chromatography on a Superdex 75 10/30 column showing a single elution peak for the NanoLuc.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 25: </B>Streptavidin affinity chromatography for NanoLuc</DIV></DIV></DIV><P>Therefore the NanoLuc was synthesized as a BioBrick in RFC[25] and was produced in <I>E. coli</I> using the pBad expression system with a C-terminal <I>Strep</I>-tag. After the production (2 l of LB-media for analytical and 12 l for preparative preparations) the cells were disrupted using sonification and the lysate was dialysed against 5 l of 1x SA-buffer. Afterwards the lysate was applied to a Streptavidin-Affinity (SA) column and was subsequently washed using SA-Buffer until a baseline was reached and the protein was then eluted using 5 mM of biotin (Attention: These are special columns which are not availible commercially. If you are using commercial colum material you have to use d-Desthiobiotin because usual biotin will elute your protein but you will not be able to regenerate the column after your chromatography). After the SA-chromatography the protein was concentrated using centrifugal concentration units (MWCO: 10 kDa). The concentrated protein was then applied onto a Superdex S200/75 size exclusion chromatography. The chromatogram of both preparations show a single peak in the chromatogram which elutes at an expected elution volume of 15 ml. The absence of any notable aggregation peak shows the high stability of this protein and the ease of production.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 26:</B>SDS-gel of recombinant nLuc with the marker (M) followed by the concentrated throughput of the streptavidin affinity column and 6 fractions collected from the elution peak</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Structure_of_the_Nano_Luciferase">Structure of the Nano Luciferase</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 27:</B> Circular dichroism spectrum of the recombinant produced NanoLuc luciferase</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_Annimated_test.gif" title="File:TUM13 Annimated test.gif">File:TUM13 Annimated test.gif</A><DIV class="thumbcaption"><B>Figure 28:</B> Homologous structure (3ppt_A) of the NanoLuc luciferase</DIV></DIV></DIV><P>There is no structure available for the <A href="http://parts.igem.org/Part:BBa_K1159001" class="external text" rel="nofollow">NanoLuc Luciferase</A> in the <A href="http://www.rcsb.org/pdb/home/home.do" class="external text" rel="nofollow">Protein Data Bank</A>. In our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions" class="external text" rel="nofollow">protein modelling</A> we used homolgy search and identified the structure <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3PPT" class="external text" rel="nofollow">3ppt_A</A> as the solved structure with the highest homology to the NanoLuc which has only 21% identity with a similarity of only 0.359. The result of the homology search is shown as annimated gif in Figure 28 (please see our <A href="https://2013.igem.org/Team:TU-Munich/Results/How_To" class="external text" rel="nofollow">How To</A> for an introduction). The protein was dialysed against 1x CD-buffer and subsequently a circular dichroism spectroscopy was taken (learn about <A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods#Circular_Dichroism_Spectroscopy" class="external text" rel="nofollow">CD spectroscopy</A>). The CD spectrum was used to predict the secondary structure content of the NanoLuc which could be determined to 35.1% helix, 27.6% b-strand, 18.5% turn and 18.8% random. As there is only a poor homology present, a detailed comparison of the determined  and the predicted secondary structure is not possible. But it can be stated that both show a balanced content of different secondary structures and that the produced protein is present in a folded conformation. The mixed secondary structure content is also in consistance with the predicted secondary structure shown in the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">AutoAnnotator</A> sequence window (<A href="http://parts.igem.org/Part:BBa_K1159001" class="external text" rel="nofollow">click here</A>).
</P><H3><SPAN class="mw-headline" id="Activity_determination_of_Luminescense">Activity determination of Luminescense</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 29:</B> Activity dependence of the NanoLuc luciferase on the pH.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 30:</B> Activity dependence of the NanoLuc luciferase on the ionic strength.</DIV></DIV></DIV><P>The activity of the produced NanoLuc was investigated by its luminescense. The assay was performed in white 96-well plates (Nund) and the bioluminescense was determined at 460 nm in a BioTeK II plate reader for 1 sec per well. For every reaction 50 Âµl of the NanoGlow substrate (sponsored by Promega) was mixed with 50 Âµl of the NanoLuc preparation. In a first experiment a dilution series of the NanoLuc was performed in order to find the apropriate dilution of the enzyme (data not shown). Finally the 1:100 000 dilution was used for all experiments as it resulted in intermediate luminescense values within the dynamic range of the plate reader. 
The dependency of the luminescense reaction on the pH value of the buffer was assayed by an experiment in which the NanoGlow substrate was diluted in water instead of the provided buffer and the pH was adapted by 100 Âµl of 500 mM buffers. The resulting luminescense is shown in Figure 23 and shows a clear optimum around pH 8.0 to 8.5. A luminescense value of more than 50% can be expected for pH values between 6 and 9 which is sufficient for most assays. 
For the dependency of the luminescence on the ionic strength 50 Âµl of diluted NanoLuc, 50 Âµl of NanoGlow substrate and 100 Âµl of sodium chloride dilution series were incubated and afterwards the luminescense was quantified in triplicates. The result is impressing and again speaks for the NanoLuc luciferase as an innovative reporter protein. The obtained luminescense is constant from 0 to 250 mM of NaCl and upon 1 M of NaCl at least 50% of the maximal luminescense can be expected.
</P><H2><SPAN class="mw-headline" id="Catechol_Dioxigenase_.28XylE.29">Catechol Dioxigenase (XylE)</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_Animation_XylEx.gif" title="File:TUM13 Animation XylEx.gif">File:TUM13 Animation XylEx.gif</A><DIV class="thumbcaption"><B>Figure 25:</B> Animation of protein structure of XylE</DIV></DIV></DIV><P>The catechol dioxigenase (&lt;partinfo&gt;BBa_K648011&lt;/partinfo&gt;) is a BioBrick which was characterized by the <A href="https://2010.igem.org/Team:Imperial_College_London" class="external text" rel="nofollow">Imperial College 2010</A> iGEM team. As we also want to use this BioBrick we produced it as a recombinant protein. Given the fact that previous teams did a great job in characterizing this proteins kinetic parameters we decided to focus on other aspects.
Catecholdioxygenases are a wide variety of enzymes degrading catechol and various of its derivatives. The xylE gene we use is from <I>Pseudomonas putida</I> and has a ferrous ion ligand in each subunit. The enzyme has the structure of a homotetramer
</P><H3><SPAN class="mw-headline" id="Analytical_preparation_2">Analytical preparation</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 26:</B> Streptavidin affinity chromatography for the catecholdioxygenase</DIV></DIV></DIV><P>To investigate if our recombinant BL21 <I>E.coli</I> cells were producing the catecholdioxygenase we started with an analytical preparation in 2 l LB-medium. After we confirmed the succesful transformation of our cells we produced another 2 x 2 l flasks full of recombinant cells in order to have enough enzyme to work with. To induce protein production we used 5 mM of arabinose when we measured an OD of 0.8. We disrupted the cells by sonification and after dialysis in 1 x SA-buffer we performed streptavidin affinity chromatography.
After elution of the desired protein in 1 x SA-buffer with 5 mM 2-mercaptoethanol and 10% glycerol we concentrated it using a centrifugal filter unit (MWCO: 30 kDa). The concentrated protein was then applied to a ÃKTA purifier equipped with a Superdex 75 16/60 size exclusion chromatography column. As seen in the figure below the protein was eluted without any aggregation products and was ready to use.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 27:</B> Size exclusion chromatography for the catecholdioxygenase</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Protein_phosphatase_1_.28from_Homo_sapiens.29">Protein phosphatase 1 (from <I>Homo sapiens</I>)</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_Animation_PP1.gif" title="File:TUM13 Animation PP1.gif">File:TUM13 Animation PP1.gif</A><DIV class="thumbcaption"><B>Figure 28:</B> Animation of protein structure of PP1</DIV></DIV></DIV><P>The production of recombinant protein phosphatase 1 was part of our collaboration with the Dundee iGEM team 2013. They developed this BioBrick which naturally binds <A href="http://de.wikipedia.org/wiki/Microcystine" class="external text" rel="nofollow">microcystine</A>, an important environmental toxin. 
</P><H3><SPAN class="mw-headline" id="Production_and_purification_of_recombinant_PP1_protein">Production and purification of recombinant PP1 protein</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 29:</B> Streptavidin affinity chromatography of protein phosphatase 1</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 30:</B> Size exclusion chromatography of protein phosphatase 1 on Superdex 200 10/30 column</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Figure 31:</B> SDS-gel of recombinant PP1 with the marker (M) followed by the the cell lysate, concentrated protein throughput of the streptavidin affinity column, 1 fraction of the eluate unconcentrated and concentrated</DIV></DIV></DIV><P>We converted this BioBrick to RFC[25] and subsequently cloned it into the expression vector pBad_C-terminal_<I>Strep</I>. Beside the recombinant characterisation we also created a <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss" class="external text" rel="nofollow">transgenic moss transformed with a receptor containing PP1</A> in its extracellular domain. The recombinant protein production was carried out in <I>E. coli</I> BL-21 which was grown in 2 L of LB-media. The protein production was induced at OD 0.8 by addition of arabinose to a concentration of 5 mM. The cells were harvested after 4 h and subsequently resuspended in 20 mL SA-buffer with 5 mM 2-mercapto ethanol. Cell disruption was performed using ultrasonic sound. The dialysed cell extracte was applied to a streptavidin affinity column which was washed until a base line was reached and was subsequently eluted using SA-buffer containing 5 mM of biotin and 5 mM of 2-mercapto ethanol. The elution peak recorded by the continuous measurement of the absorbance at 280 nm indicated a good yield of recombinant protein (Fig. 29). As a second purification step after the streptavidin affinity chromatography we concentrated the protein in centrifugal filter units in order to apply it to a size exclusion chromatography. During the concentration process there were clear signal for precipitated protein which appeared as white flakes in the concentration filter unit. This effect was by far the most drastic precipitation of recombinant that was detected during this iGEM project. Anyhow the concentrated protein was centrifuged for 5 min at 13 200 RPM to remove particles of precipitated protein and was then applied to an ÃKTA purified with a Superdex 200 10/30 column (Fig. 30). There were four different peaks present in the chromatogramm which most probably correspond to (1) a peak created by aggregated protein which runs in the voit volume of the size exclusion colum, (2) a diffuse peak which might correspond to multimeric PP1, (3) a sharp peak at the elution volume where the recombinant PP1 was expected and (4) a peak which is most probably caused by low molecular buffer substances. The general signal intensity obtained in the size exclusion was very low indicating low protein concentrations. The maximal peak obtained for PP1 had an absorbance of 50 milli Absorption Units whereas this was for example for the NanoLuc in the range of 1000 for a comparable experiment. The fact that this protein does not tolerate the concentration procedure gives an indication that it is fragile and tends to aggregation when stress is applied. Finally recombinant PP1 protein was prepared although the yield was very low compared with other effector proteins.
</P><H3><SPAN class="mw-headline" id="Possible_reasons_for_the_instability_of_PP1_in_vitro">Possible reasons for the instability of PP1 <I>in vitro</I></SPAN></H3><P>The high aggregation and denaturation tendency of the protein phosphatase 1 which was observed in the analytical preparation motivated the search for a possible explanation. As a first hint the amino acid composition calculated by the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">AutoAnnotator</A> was examined and it can be seen that 13 cysteine residues are present in the sequence of the protein phosphatase. This corresponds to 4% of the total number of amino acid residues comapred to 2.8% cystein residues in an average protein. This information taken together with the molecular function of the protein phosphatase 1 clearly shows that this BioBrick is a cytoplasmatic protein which fulfills its function in the reducing milieu of the cytoplasm and tends to aggregation when exposed to an oxidizing milieu such a the periplasm of <I>E. coli</I>. 
Secondly the amino acid sequence produced by the AutoAnnotator was fed into the bioinformatic tool <A href="http://web.expasy.org/cgi-bin/protparam/protparam" class="external text" rel="nofollow">ProtParam</A> which gives a stability prediction for proteins. The result was: &quot;The instability index (II) is computed to be 42.99 - This classifies the protein as unstable.&quot; 
Taken both these indications into account it still seems a good idea to express the protein phosphatase 1 cytoplasmatically to bind microcystine but it is not advisable to think about a secreted version of PP1 as the protein seems to unstable for this application.
</P><H2><SPAN class="mw-headline" id="DDT-Dehydrochlorinase">DDT-Dehydrochlorinase</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_Animation_DDT.gif" title="File:TUM13 Animation DDT.gif">File:TUM13 Animation DDT.gif</A><DIV class="thumbcaption"><B>Figure 32:</B> Animation of protein structure of DDT</DIV></DIV></DIV><P>The DDT dehydrochlorinase (&lt;partinfo&gt;BBa_K620000&lt;/partinfo&gt;) BioBrick was designed in RFC[10] by the <A href="https://2011.igem.org/Team:Caltech" class="external text" rel="nofollow">Caltech 2011</A> iGEM team and as a localization of this effector outside the cytoplasm isn't advisable we didn't improve this BioBrick to RFC[25]. The DDT dehydrochlorinase is dependent on the continuous supply of glutathione which conjugates to xenobiotics in order to inactivate them. The disadvantage of an RFC[10] BioBrick is that it can not be fused to an affinity tag which can subsequently be used to purify the protein after its recombinant production. 	 
Beside the experiments already performed in iGEM we found monochlorobimane to be an interesting assay for experiments in bioremeditation. This commercially availible substrate can be conjugated to glutathione. The conjugation of monochlorobimane can be followed by the fluorescense of the conjugation product (excitation: 380 nm / emission: 461 nm). We tryed this assay with transgenic plants but could not see results as the autofluorescense of the photosystem in <I>Physcomitrella</I> was too strong in order to see the emergence of the fluorescense caused by the conjugation product.
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="http://udel.edu/~gshriver/pdf/Pimenteletal1997.pdf" class="external text" rel="nofollow">Pmentel et al., 1997</A>] Pimentel, D., Wilson, C., McCullum, C., Huang, R., Dwen, P., Flack, J. Tran, Q., Saltman, T., Cliff, T. (1997). Economic and environmental benefits of biodiversity. <I>BioScience</I>, Vol. 47, No. 11., pp. 747-757.
</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Results/Recombinant">http://2013.igem.org/Team:TU-Munich/Results/Recombinant</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Results/Recombinant" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Results/Recombinant" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Results/Recombinant&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Results/Recombinant&amp;oldid=360252" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>