<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Notebook_Labjournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Notebook/Labjournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Notebook/Labjournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Labjournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Labjournal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Notebook/Labjournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Notebook/Labjournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Labjournal"> Labjournal </SPAN></H2><FORM id="labselect"><FIELDSET class="ui-widget ui-widget-content ui-corner-left"><B>Display:</B><B style="color: rgb(115, 208, 255);">General</B><B style="color: rgb(166, 126, 166);">Vector Design</B><B style="color: rgb(116, 183, 112);">Effectors</B><B style="color: rgb(255, 122, 97);">Vitality Sensor</B><B style="color: rgb(0, 32, 96);">Safety</B><A href="#" id="ExAll">Expand All ...</A><A href="#" id="ColAll">Collapse All ...</A><B>Jump to:</B><A href="#Week_1">Week 1</A> 22.4-28.4<A href="#Week_2">Week 2</A> 29.4-05.5<A href="#Week_3">Week 3</A> 06.5-12.5<A href="#Week_4">Week 4</A> 13.5-19.5<A href="#Week_5">Week 5</A> 20.5-26.5<A href="#Week_6">Week 6</A> 27.5-02.6<A href="#Week_7">Week 7</A> 03.6-09.6<A href="#Week_8">Week 8</A> 10.6-16.6<A href="#Week_9">Week 9</A> 17.6-23.6<A href="#Week_10">Week 10</A> 24.6-30.6<A href="#Week_11">Week 11</A> 01.7-07.7<A href="#Week_12">Week 12</A> 08.7-14.7<A href="#Week_13">Week 13</A> 15.7-21.7<A href="#Week_14">Week 14</A> 22.7-28.7<A href="#Week_15">Week 15</A> 29.7-04.8<A href="#Week_16">Week 16</A> 05.8-11.8<A href="#Week_17">Week 17</A> 12.8-18.8<A href="#Week_18">Week 18</A> 19.8-25.8<A href="#Week_19">Week 19</A> 26.8-01.9<A href="#Week_20">Week 20</A> 02.9-08.9<A href="#Week_21">Week 21</A> 09.9-15.9<A href="#Week_22">Week 22</A> 16.9-22.9

        </FIELDSET></FORM><DIV id="ladder" class="ui-widget ui-widget-content ui-corner-right"><B><U>1 kbp GeneRuler:</U></B><B><U>100 bp GeneRuler:</U></B><B><U>PageRuler Plus:</U></B></DIV><DIV class="labbook"><H1><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H1><DIV class="week" id="WWeek_1"><H2><SPAN class="mw-headline" id="Monday.2C_April_22nd"><B>Monday, April 22nd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29">  Transformation of <I>E. coli</I> XL1 blue with Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3) </SPAN></H3><P><B>Investigator: Jeff, Rosario</B></P><P><B>Aim of the experiment:</B> Transformation of Phytochrome B for protein fusion.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_pTUM100_with_pGAL.2C_pTEF1.2C_pTEF2.2C_pADH_and_RFC25_compatible_RFP_generator">  Miniprep of pTUM100 with pGAL, pTEF1, pTEF2, pADH and RFC25 compatible RFP generator </SPAN></H3><P><B>Investigator: Jeff, Rosario</B></P><P><B>Aim of the experiment:</B> Miniprep of pTUM100 with pGAL, pTEF1, pTEF2, pADH and RFC25 compatible RFP generator
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_RFP-Generator_.28RFC25.2C_pSB1C3.29"> Sequencing of RFP-Generator (RFC25, pSB1C3) </SPAN></H3><P><B>Investigator: Jeff, Rosario</B></P><P><B>Aim of the experiment:</B> Sequencing of RFP-Generator (RFC25, pSB1C3)
</P><P><B>Procedure:</B></P><P>Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer)
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_April_23rd"><B>Tuesday, April 23rd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_with_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29"> Picking of of <I>E. coli</I> XL1 blue with Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3) </SPAN></H3><P><B>Investigator: Jeff, Rosario, Florian</B></P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue with Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> pSB1C3 plasmid with BBa_K801031 (PhyB 2 - 908 aa, RFC25): Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 4 colonies were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_RFP-generator_.28RFC25.2C_pSB1C3.2C_P4_.26_P5.29"> Analytical digestion and gelelectrophoresis of RFP-generator (RFC25, pSB1C3, P4 &amp; P5) </SPAN></H3><P><B>Investigator: Jeff, Rosario, Florian</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of RFP-generator (RFC25, pSB1C3, P4 &amp; P5).
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P4 with NgoMIV+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P4
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P5 with NgoMIV+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P5
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder DNA ladder
</TD><TD><B>P4</B></TD><TD><B>P5</B></TD></TR><TR><TD><B>Mutation successful</B></TD><TD><B>Mutation successful!</B></TD></TR></TBODY></TABLE><UL><LI> Parts are compliant and do not contain RFC25 forbidden restriction sites.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_pTUM_vectors_with_pGAL.2C_pADH.2C_pTEF1.2C_pTEF2"> Sequencing of pTUM vectors with pGAL, pADH, pTEF1, pTEF2 </SPAN></H3><P><B>Investigator: Jeff, Rosario, Florian</B></P><P><B>Aim of the experiment:</B> Sequencing of pTUM vectors with pGAL, pADH, pTEF1, pTEF2
</P><P><B>Procedure:</B></P><P>Sequencing batch were prepared after manufacturer's protocol. (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different vectors we sequenced received the following barcodes:
</P><P>- ADH in pTUM100:   FR01002265
</P><P>- TEF1 in pTUM100:  FR01002266
</P><P>- TEF2 in pTUM100:  FR01002266
</P><P>- GAL in pTUM100:   FR01002268
</P><P>
Sequencing of TEF2 in pTUM100 was not interpretable. The other sequences were consistent with the sequences in the parts registry. 
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_April_24th"><B>Wednesday, April 24th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29">  Miniprep of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3)  </SPAN></H3><P><B>Investigator: Jeff, Florian</B></P><P><B>Aim of the experiment:</B> Miniprep of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_Phytochrome_B_.282-908_N-terminal_amino_acids.29_.28BBa_K801031.2C_RFC25.2C_pSB1C3.29.2C_P7_-_P10"> Analytical digestion and gelelectrophoresis of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3), P7 - P10 </SPAN></H3><P><B>Investigator: Jeff, Florian</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of Phytochrome B (2-908 N-terminal amino acids) (BBa_K801031, RFC25, pSB1C3), P7 - P10.
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion for P7 with NgoMIV+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P7
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P8 with NgoMIV+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P8
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P9 with NgoMIV+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P9
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion for P10 with NgoMIV+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P10
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder DNA ladder
</TD><TD><B>P7</B></TD><TD><B>P8</B></TD><TD><B>P9</B></TD><TD><B>P10</B></TD></TR><TR><TD><B>Part is correct</B></TD><TD><B>Part is correct</B></TD><TD><B>Part is correct</B></TD><TD><B>Part is correct</B></TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_EreA_and_EreB_.28pDEST14.29">  Transformation of <I>E. coli</I> XL1 blue with EreA and EreB (pDEST14) </SPAN></H3><P><B>Investigator: Jeff, Florian</B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with EreA and EreB (pDEST14).
</P><P><B>Procedure:</B></P><UL><LI> Plasmid DNA was received dried in paper from McMaster University.
</LI></UL><UL><LI> DNA was resuspended in ddH2O
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The transformated cells were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on chlorampenicol and ampicillin plates.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H1><DIV class="week" id="WWeek_2"><H2><SPAN class="mw-headline" id="Monday.2C_April_29th"><B>Monday, April 29th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_pRK792.2C_pRK793.2C_pRIL.2C_ereA.2C_ereB">  Miniprep of pRK792, pRK793, pRIL, ereA, ereB </SPAN></H3><P><B>Investigator: Rosario</B></P><P><B>Aim of the experiment:</B> Miniprep of pRK792, pRK793, pRIL, ereA, ereB
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI> Concentrations were determined by measuring the absorption:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>c [ng/µl]</B></TD></TR><TR><TD>pRK792
</TD><TD>214,5
</TD></TR><TR><TD>pRK793
</TD><TD>154,3
</TD></TR><TR><TD>pRIL
</TD><TD>6,1
</TD></TR><TR><TD>ereA
</TD><TD>183,4
</TD></TR><TR><TD>ereB
</TD><TD>98,3
</TD></TR></TBODY></TABLE><UL><LI> The procedure will have to be repeated for pRIL
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Midiprep_of_RFC25_compatible_RFP_generator_in_.28pSB1C3.29">  Midiprep of RFC25 compatible RFP generator in (pSB1C3) </SPAN></H3><P><B>Investigator: Andreas, Florian</B></P><P><B>Aim of the experiment:</B> Midiprep of RFC25 compatible RFP generator in (pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Midiprep was performed after manufacturer's protocol (QIAprep Midiprep, QIAGEN)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_April_30th"><B>Tuesday, April 30th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_mINF1_and_Leptin_with_HindIII_and_EheI"> Preparative digestion and gelelectrophoresis of mINF1 and Leptin with HindIII and EheI </SPAN></H3><P><B>Investigator: Louise, Johanna</B></P><P><B>Aim of the experiment:</B> Testing the enzymes, since the earlier digestion with probably older enzymes did not work (no bands on gel) .
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion for mINF1 with HindIII and EheI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Plasmid DNA mINF11
</TD></TR><TR><TD>5 µl
</TD><TD>Tango Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>HindIII (10 U/µl)
</TD></TR><TR><TD>3 µl
</TD><TD>EheI (10 U/µl)
</TD></TR><TR><TD>30 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion for Leptin with HindIII and EheI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA Leptin
</TD></TR><TR><TD>5 µl
</TD><TD>Tango Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>HindIII (10 U/µl)
</TD></TR><TR><TD>3 µl
</TD><TD>EheI (10 U/µl)
</TD></TR><TR><TD>20 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_PSH21_.28P17.29">  Transformation of <I>E. coli</I> XL1 blue with PSH21 (P17) </SPAN></H3><P><B>Investigator: Johanna, Louise</B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with PSH21.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice for 10 minutes.
</LI></UL><UL><LI> 5 µl of DNA was added to 250 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 60 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding the DNA to 2 ml LB-medium.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The transformated cells were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on chlorampenicol and ampicillin plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_2nd"><B>Thursday, May 2nd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_pSH21_.28pAutoRex8.29_containing_AlcR.2FAlcA_promotor_system">  Miniprep of pSH21 (pAutoRex8) containing AlcR/AlcA promotor system </SPAN></H3><P><B>Investigator: Katrin</B></P><P><B>Aim of the experiment:</B> Miniprep of pSH21 (pAutoRex8) containing AlcR/AlcA promotor system
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI> The concentration was measured (900 ng/µl)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_3rd"><B>Friday, May 3rd</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_pSB1C3-GFP.2C_pSB1C3-mkate2.2C_pSB1C3-mCherry">  Transformation of <I>E. coli</I> XL1 blue with pSB1C3-GFP, pSB1C3-mkate2, pSB1C3-mCherry </SPAN></H3><P><B>Investigator: Andreas</B></P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with pSB1C3-GFP, pSB1C3-mkate2, pSB1C3-mCherry.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice for 10 minutes.
</LI></UL><UL><LI> 5 µl of DNA was added to 250 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 60 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding the DNA to 2 ml LB-medium.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The transformated cells were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on chlorampenicol and ampicillin plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Plating_of_received_E..C2.A0coli_containing_biobricks"> Plating of received <I>E. coli</I> containing biobricks </SPAN></H3><P><B>Investigator: Jeff, Andreas</B></P><P><B>Aim of the experiment:</B> The received biobricks were already transformed in <I>E. coli</I> and were in an agar stabs. These <I>E. coli</I> cells were transferred with an inoculation loop on antibiotic selection plates and were incubated over night.
</P><P><B>Operational sequence:</B></P><UL><LI> Bacterias containing plasmids with biobricks were transferred with a sterile inoculation loop on antibiotic plates and were incubated at 37 °C overnight.
</LI></UL><UL><LI> The biobricks were:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Name
</TH><TH> Function
</TH></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K157004" class="external text" rel="nofollow"><B>BBa_K157004</B></A>]
</TD><TD> Fluoresceine A -binding derivative of a Lipocalin
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K863000" class="external text" rel="nofollow"><B>BBa_K863000</B></A>]
</TD><TD> bpul (laccase from Bacillus pumilus) with T7 promoter, RBS and HIS tag
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K909009" class="external text" rel="nofollow"><B>BBa_K909009</B></A>]
</TD><TD> cDNA of UV-B sensing protein UVR8 from Arabidopsis thaliana
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K337013" class="external text" rel="nofollow"><B>BBa_K337013</B></A>]
</TD><TD> constitutive promoter SV40
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K747096" class="external text" rel="nofollow"><B>BBa_K747096</B></A>]
</TD><TD> constitutive promoter CMV
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K414002" class="external text" rel="nofollow"><B>BBa_K414002</B></A>]
</TD><TD> constitutive promoter CaMV35S
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K147002" class="external text" rel="nofollow"><B>BBa_K147002</B></A>]
</TD><TD> xylE gene coding for catechol-1,2-dioxygenase
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_E2020" class="external text" rel="nofollow"><B>BBa_E2020</B></A>]
</TD><TD> Cerulean
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K404319" class="external text" rel="nofollow"><B>BBa_K404319</B></A>]
</TD><TD> mCFP
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_E0040" class="external text" rel="nofollow"><B>BBa_E0040</B></A>]
</TD><TD> GFPmut3b
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K592010" class="external text" rel="nofollow"><B>BBa_K592010</B></A>]
</TD><TD> amilGFP
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_E2050" class="external text" rel="nofollow"><B>BBa_E2050</B></A>]
</TD><TD> mOrange
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K399001" class="external text" rel="nofollow"><B>BBa_K399001</B></A>]
</TD><TD> mStrawberry
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow"><B>BBa_E1010</B></A>]
</TD><TD> mRFP1
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_E2060" class="external text" rel="nofollow"><B>BBa_E2060</B></A>]
</TD><TD> mCherry
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K592012" class="external text" rel="nofollow"><B>BBa_K592012</B></A>]
</TD><TD> eforRed
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K592011" class="external text" rel="nofollow"><B>BBa_K592011</B></A>]
</TD><TD> cjBlue
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K864401" class="external text" rel="nofollow"><B>BBa_K864401</B></A>]
</TD><TD> aeBlue
</TD></TR><TR><TD> [<A href="http://partsregistry.org/Part:BBa_K592009" class="external text" rel="nofollow"><B>BBa_K592009</B></A>]
</TD><TD> amilCP, blue chromoprotein
</TD></TR><TR><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K414000" class="external text" rel="nofollow"><B>BBa_K414000</B></A>]
</TD><TD> RD29A Promoter + Strong Plant Kozak (RBS) + RFP
</TD></TR><TR><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K414006" class="external text" rel="nofollow"><B>BBa_K414006</B></A>]
</TD><TD> 35S Promoter + (Strong Plant RBS + GFP) From K414001
</TD></TR><TR><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K414007" class="external text" rel="nofollow"><B>BBa_K414007</B></A>]
</TD><TD> DREB1C promoter
</TD></TR><TR><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K678001" class="external text" rel="nofollow"><B>BBa_K678001</B></A>]
</TD><TD> alcA promoter
</TD></TR><TR><TD>[<A href="http://partsregistry.org/Part:BBa_K243004" class="external text" rel="nofollow"><B>BBa_K243004</B></A>]
</TD><TD>Short linker
</TD></TR><TR><TD>[<A href="http://partsregistry.org/Part:BBa_K243005" class="external text" rel="nofollow"><B>BBa_K243005</B></A>]
</TD><TD>Middle linker
</TD></TR><TR><TD>[<A href="http://partsregistry.org/Part:BBa_K243006" class="external text" rel="nofollow"><B>BBa_K243006</B></A>]
</TD><TD>Long Linker
</TD></TR><TR><TD>[<A href="http://partsregistry.org/Part:BBa_K243029" class="external text" rel="nofollow"><B>BBa_K243029</B></A>]
</TD><TD> GSAT Linker
</TD></TR><TR><TD>[<A href="http://partsregistry.org/Part:BBa_K243030" class="external text" rel="nofollow"><B>BBa_K243030</B></A>]
</TD><TD> SEG Linker
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_May_5th"><B>Sunday, May 5th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_the_plated_E..C2.A0coli_containing_biobricks_and_the_parts_received_from_LMU"> Picking of the plated <I>E. coli</I> containing biobricks and the parts received from LMU </SPAN></H3><P><B>Investigator: Florian</B></P><P><B>Aim of the experiment:</B> The colonies from the plates were transferred into liquid LB medium, so that minipreps can be performed the next day.
</P><P><B>Operational sequence:</B></P><UL><LI> Tubes with 5 ml LB medium and the corresponding antibiotic were prepared.
</LI></UL><UL><LI> A single colony from each plate was transferred into a tube with a pipet tip.
</LI></UL><UL><LI> The biobricks were:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Function
</TH></TR><TR><TD> p19
</TD><TD> mCherry in pSB1C3
</TD></TR><TR><TD> p20
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K823039" class="external text" rel="nofollow"><B>BBa_K823039</B></A>]
</TD><TD> GFP in pSB1C3
</TD></TR><TR><TD> p21
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K823029" class="external text" rel="nofollow"><B>BBa_K823029</B></A>]
</TD><TD> mKate2 in pSB1C3
</TD></TR><TR><TD> p22
</TD><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K414006" class="external text" rel="nofollow"><B>BBa_K414006</B></A>]
</TD><TD> 35S Promoter + (Strong Plant RBS + GFP) From K414001
</TD></TR><TR><TD> p23
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K414002" class="external text" rel="nofollow"><B>BBa_K414002</B></A>]
</TD><TD> constitutive promoter CaMV35S
</TD></TR><TR><TD> p24
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K909009" class="external text" rel="nofollow"><B>BBa_K909009</B></A>]
</TD><TD> cDNA of UV-B sensing protein UVR8 from Arabidopsis thaliana
</TD></TR><TR><TD> p25
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K399001" class="external text" rel="nofollow"><B>BBa_K399001</B></A>]
</TD><TD> mStrawberry
</TD></TR><TR><TD> p26
</TD><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K414007" class="external text" rel="nofollow"><B>BBa_K414007</B></A>]
</TD><TD> DREB1C promoter
</TD></TR><TR><TD> p27
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K863000" class="external text" rel="nofollow"><B>BBa_K863000</B></A>]
</TD><TD> bpul (laccase from Bacillus pumilus) with T7 promoter, RBS and HIS tag
</TD></TR><TR><TD> p28
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K337013" class="external text" rel="nofollow"><B>BBa_K337013</B></A>]
</TD><TD> constitutive promoter SV40
</TD></TR><TR><TD> p29
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K592011" class="external text" rel="nofollow"><B>BBa_K592011</B></A>]
</TD><TD> cjBlue
</TD></TR><TR><TD> p30
</TD><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K678001" class="external text" rel="nofollow"><B>BBa_K678001</B></A>]
</TD><TD> alcA promoter
</TD></TR><TR><TD> p31
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K592012" class="external text" rel="nofollow"><B>BBa_K592012</B></A>]
</TD><TD> eforRed
</TD></TR><TR><TD> p32
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K592010" class="external text" rel="nofollow"><B>BBa_K592010</B></A>]
</TD><TD> amilGFP
</TD></TR><TR><TD> p33
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K864401" class="external text" rel="nofollow"><B>BBa_K864401</B></A>]
</TD><TD> aeBlue
</TD></TR><TR><TD> p34
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K404319" class="external text" rel="nofollow"><B>BBa_K404319</B></A>]
</TD><TD> mCFP
</TD></TR><TR><TD> p35
</TD><TD> [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K414000" class="external text" rel="nofollow"><B>BBa_K414000</B></A>]
</TD><TD> RD29A Promoter + Strong Plant Kozak (RBS) + RFP
</TD></TR><TR><TD> p36
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K592009" class="external text" rel="nofollow"><B>BBa_K592009</B></A>]
</TD><TD> amilCP, blue chromoprotein
</TD></TR><TR><TD> p37
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K747096" class="external text" rel="nofollow"><B>BBa_K747096</B></A>]
</TD><TD> constitutive promoter CMV
</TD></TR><TR><TD> p38
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K147002" class="external text" rel="nofollow"><B>BBa_K147002</B></A>]
</TD><TD> XylE dioxygenase (catechol dioxygenase)
</TD></TR><TR><TD> p39
</TD><TD>[<A href="http://partsregistry.org/Part:BBa_K243005" class="external text" rel="nofollow"><B>BBa_K243005</B></A>]
</TD><TD>Middle linker
</TD></TR><TR><TD> p40
</TD><TD>[<A href="http://partsregistry.org/Part:BBa_K243029" class="external text" rel="nofollow"><B>BBa_K243029</B></A>]
</TD><TD> GSAT Linker
</TD></TR><TR><TD> p41
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_E0040" class="external text" rel="nofollow"><B>BBa_E0040</B></A>]
</TD><TD> GFPmut3b
</TD></TR><TR><TD> p42
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K243006" class="external text" rel="nofollow"><B>BBa_K243006</B></A>]
</TD><TD> Long Linker
</TD></TR><TR><TD> p43
</TD><TD>[<A href="http://partsregistry.org/Part:BBa_K243030" class="external text" rel="nofollow"><B>BBa_K243030</B></A>]
</TD><TD> SEG Linker
</TD></TR><TR><TD> p44
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K243004" class="external text" rel="nofollow"><B>BBa_K243004</B></A>]
</TD><TD> Short linker
</TD></TR><TR><TD> p45
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_K157004" class="external text" rel="nofollow"><B>BBa_K157004</B></A>]
</TD><TD> Fluoresceine A -binding derivative of a Lipocalin
</TD></TR><TR><TD> p46
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_E2060" class="external text" rel="nofollow"><B>BBa_E2060</B></A>]
</TD><TD> mCherry
</TD></TR><TR><TD> p47
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_E2020" class="external text" rel="nofollow"><B>BBa_E2020</B></A>]
</TD><TD> Cerulean
</TD></TR><TR><TD> p48
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_E1010" class="external text" rel="nofollow"><B>BBa_E1010</B></A>]
</TD><TD> mRFP1
</TD></TR><TR><TD> p49
</TD><TD> [<A href="http://partsregistry.org/Part:BBa_E2050" class="external text" rel="nofollow"><B>BBa_E2050</B></A>]
</TD><TD> mOrange
</TD></TR></TBODY></TABLE><UL><LI>There were no colonies on the plate with K864401 (p33), therefore the LB medium could not be infected
</LI></UL><UL><LI>The tubes were placed into the incubator at 37 °C
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_3">Week 3</SPAN></H1><DIV class="week" id="WWeek_3"><H2><SPAN class="mw-headline" id="Monday.2C_May_6th"><B>Monday, May 6th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_the_prepared_E..C2.A0coli_containing_biobricks_and_the_parts_from_LMU"> Miniprep of the prepared <I>E. coli</I> containing biobricks and the parts from LMU </SPAN></H3><P><B>Investigators:  Florian, Johanna, Jefferey</B></P><P><B>Aim of the experiment:</B> A miniprep of the overnight culture of our <I>E. coli</I> containig our biobricks and LMU parts was performed using the Qiagen miniprep kit. 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI> Resulting concentrations were determined by measuring the absorption:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Concentration [ng/µl]
</TH></TR><TR><TD> p19
</TD><TD> mCherry in pSB1C3
</TD><TD> 40,2
</TD></TR><TR><TD> p20
</TD><TD> GFP in pSB1C3
</TD><TD> 58,5
</TD></TR><TR><TD> p21
</TD><TD> mKate2 in pSB1C3
</TD><TD> 131,3
</TD></TR><TR><TD> p22
</TD><TD> 35S Promoter + (Strong Plant RBS + GFP) From K414001
</TD><TD> 42,8
</TD></TR><TR><TD> p23
</TD><TD> constitutive promoter CaMV35S
</TD><TD> 39,3
</TD></TR><TR><TD> p24
</TD><TD> cDNA of UV-B sensing protein UVR8 from Arabidopsis thaliana
</TD><TD> 30,0
</TD></TR><TR><TD> p25
</TD><TD> mStrawberry
</TD><TD> 46,6
</TD></TR><TR><TD> p26
</TD><TD> DREB1C promoter
</TD><TD> 34,8
</TD></TR><TR><TD> p27
</TD><TD> bpul (laccase from Bacillus pumilus) with T7 promoter, RBS and HIS tag
</TD><TD> 52,5
</TD></TR><TR><TD> p28
</TD><TD> constitutive promoter SV40
</TD><TD> 58,7
</TD></TR><TR><TD> p29
</TD><TD> cjBlue
</TD><TD> 44,4
</TD></TR><TR><TD> p30
</TD><TD> alcA promoter
</TD><TD> 26,9
</TD></TR><TR><TD> p31
</TD><TD> eforRed
</TD><TD> 64,8
</TD></TR><TR><TD> p32
</TD><TD> amilGFP
</TD><TD> 59,3
</TD></TR><TR><TD> p34
</TD><TD> mCFP
</TD><TD> 78,4
</TD></TR><TR><TD> p35
</TD><TD> RD29A Promoter + Strong Plant Kozak (RBS) + RFP
</TD><TD> 61,2
</TD></TR><TR><TD> p36
</TD><TD> amilCP, blue chromoprotein
</TD><TD> 89,2
</TD></TR><TR><TD> p37
</TD><TD> constitutive promoter CMV
</TD><TD> 83,6
</TD></TR><TR><TD> p38
</TD><TD> XylE dioxygenase (catechol dioxygenase)
</TD><TD> 57,4
</TD></TR><TR><TD> p39
</TD><TD>Middle linker
</TD><TD> 64,3
</TD></TR><TR><TD> p40
</TD><TD> GSAT Linker
</TD><TD> 109,6
</TD></TR><TR><TD> p41
</TD><TD> GFPmut3b
</TD><TD> 118,1
</TD></TR><TR><TD> p42
</TD><TD> Long Linker
</TD><TD> 133,0
</TD></TR><TR><TD> p43
</TD><TD> SEG Linker
</TD><TD> 107,5
</TD></TR><TR><TD> p44
</TD><TD> Short linker
</TD><TD> 60,2
</TD></TR><TR><TD> p45
</TD><TD> Fluoresceine A -binding derivative of a Lipocalin
</TD><TD> 82,7
</TD></TR><TR><TD> p46
</TD><TD> mCherry
</TD><TD> 230,2
</TD></TR><TR><TD> p47
</TD><TD> Cerulean
</TD><TD> 25,7
</TD></TR><TR><TD> p48
</TD><TD> mRFP1
</TD><TD> 299,3
</TD></TR><TR><TD> p49
</TD><TD> mOrange
</TD><TD> 181,2
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_FluA.2C_SV40.2C_CaMV35S.2C_xylE.2C_LMU_mKate2.2C_LMU_GFP.2C_LMU_mCherry.2C_DREB1C.2C_RD29A">  Sequencing of FluA, SV40, CaMV35S, xylE, LMU mKate2, LMU GFP, LMU mCherry, DREB1C, RD29A </SPAN></H3><P><B>Investigators: Andreas, Florian, Johanna, Jefferey, Rosario</B></P><P><B>Aim of the experiment:</B> Sequencing of genes for which no prior sequencing results were available.
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode
</TH></TR><TR><TD> p45
</TD><TD> FluA
</TD><TD> FR01002355
</TD></TR><TR><TD> p28
</TD><TD> SV40
</TD><TD> FR01002356
</TD></TR><TR><TD> p23
</TD><TD> CaMV35S
</TD><TD> FR01002357
</TD></TR><TR><TD> p38
</TD><TD> xylE
</TD><TD> FR01002358
</TD></TR><TR><TD> p21
</TD><TD> LMU mKate2
</TD><TD> FR01002359
</TD></TR><TR><TD> p20
</TD><TD> LMU GFP
</TD><TD> FR01002360
</TD></TR><TR><TD> p19
</TD><TD> LMU mCherry
</TD><TD> FR01002345
</TD></TR><TR><TD> p46
</TD><TD> mCherry
</TD><TD> FR01002346
</TD></TR><TR><TD> p26
</TD><TD> DREB1C
</TD><TD> FR01002347
</TD></TR><TR><TD> p35
</TD><TD> RD29A
</TD><TD> FR01002348
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_ordered_primers">  Preparation of ordered primers </SPAN></H3><P><B>Investigators: Andreas, Jefferey, Rosario</B></P><P><B>Aim of the experiment:</B> Dissolving of primer pallets to provide a concentration of 100 pmol/µl
</P><P><B>Procedure:</B> We added an amount of water which gave us a final primer concentration of 100 pmol/µl. 
</P><P>Primers were labeled as follows:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Oligonr.
</TH></TR><TR><TD> 6
</TD><TD> EreA_for
</TD></TR><TR><TD> 7
</TD><TD> EreA_rev
</TD></TR><TR><TD> 8
</TD><TD> EreA_SDM1_for
</TD></TR><TR><TD> 9
</TD><TD> EreA_SDM1_rev
</TD></TR><TR><TD> 10
</TD><TD> EreA_SDM2_for
</TD></TR><TR><TD> 11
</TD><TD> EreA_SDM2_rev
</TD></TR><TR><TD> 12
</TD><TD> EreA_SDM3_for
</TD></TR><TR><TD> 13
</TD><TD> EreA_SDM3_rev
</TD></TR><TR><TD> 14
</TD><TD> EreB_for
</TD></TR><TR><TD> 15
</TD><TD> EreB_rev
</TD></TR><TR><TD> 16
</TD><TD> EreB_SDM1_for
</TD></TR><TR><TD> 17
</TD><TD> EreB_SDM1_rev
</TD></TR><TR><TD> 18
</TD><TD> EreB_SDM2_for
</TD></TR><TR><TD> 19
</TD><TD> EreB_SDM2_rev
</TD></TR><TR><TD> 20
</TD><TD> EreB_SDM3_for
</TD></TR><TR><TD> 21
</TD><TD> EreB_SDM3_rev
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_May_7th"><B>Tuesday, May 7th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR.2C_purification_and_analytical_geleletrophoresis_of_EreA_.28P15.29_and_EreB_.28P16.29"> PCR, purification and analytical geleletrophoresis of EreA (P15) and EreB (P16) </SPAN></H3><P><B>Investigator: Jeff, Louise, Florian, Rosario, Andi, Johanna</B></P><P><B>Aim of the experiment:</B>  PCR of EreA (P15) and EreB (P16).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P15: O46 (EreA_for); P16: O31 (EreB_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P15: O47 (EreA_rev); P16: O32 (EreB_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P15; P16)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The gradient PCR program was performed after following scheme with following conditions (T<SUB>m</SUB>=54 °C; ΔG=2 °C; P15 in row 11(=52 °C); P16 in row 1 (=54.0 °C)):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>T<SUB>m</SUB>=54 °C; ΔG=2 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>80 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> 9 µl of the purified PCR products were mixed with 1 µl DNA loading buffer (10x) for analytical gelelectrophoresis
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min in 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>F1
</TD><TD>F2
</TD></TR><TR><TD>Fragment-size like expected
</TD><TD>Fragment-size like expected
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_PhyB_.28P9.29_and_PIF3-20aa_linker_.28P50.29_with_AgeI_and_PstI_as_well_as_20aalinker-PIF3_.28P51.29_with_EcoRI_and_NgoMIV"> Preparative digestion and gelelectrophoresis of PhyB (P9) and PIF3-20aa linker (P50) with AgeI and PstI as well as 20aalinker-PIF3 (P51) with EcoRI and NgoMIV</SPAN></H3><P><B>Investigator: Jeff, Andi, Florian, Rosario, Louise, Johanna</B></P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of PhyB (P9) and PIF3-20aa linker (P50) with AgeI and PstI as well as 20aalinker-PIF3 (P51) with EcoRI and NgoMIV .
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PhyB (P9) and PIF3-20aa linker (P50) with AgeI and PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P9/P50
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of 20aalinker-PIF3 (P51) with EcoRI and NgoMIV
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P51
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the reaction batches after digestion and were loaded on an 1% low-melting agarose gel for preparative gelelectrophoresis.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V for 90 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P9 digestion = F3
</TD><TD>1 kbp ladder
</TD><TD>P50 digestion = F4
</TD><TD>P51 digestion = F5
</TD></TR><TR><TD>Fragment-size like expected; band was cut out
</TD><TD>Fragment-size like expected; band was cut out
</TD><TD>Fragment-size like expected; band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_May_8th"><B>Wednesday, May 8th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_PCR_product_of_EreA_.28F1.29_and_EreB_.28F2.29_with_XbaI_.26_AgeI"> Preparative digestion and gelelectrophoresis of PCR product of EreA (F1) and EreB (F2) with XbaI &amp; AgeI </SPAN></H3><P><B>Investigator: Jeff, Andi, Florian, Rosario, Louise</B></P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of PCR product of EreA (F1) and EreB (F2) with XbaI &amp; AgeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR product of EreA (F1) and EreB (F2) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR product F1/F2
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P52 for preperation of pSB1C3 vector:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P52
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the reaction batches for F1 &amp; F2 and 4 µl of DNA loading buffer (10x) were added to the reaction batches for Pxx after digestion and were loaded on an 1% agarose gel for preparative gelelectrophoresis.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>F1 digestion = F6
</TD><TD>F2 digestion = F7
</TD><TD>1 kbp ladder
</TD><TD>P52 digestion = F8
</TD></TR><TR><TD>Length was like expected
</TD><TD>Length was like expected
</TD><TD>Length was like expected; the lower band was extracted
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F8.2BF6_.28EreA_in_pSB1C3.29.2C_F8.2BF7_.28EreB_in_pSB1C3.29"> Ligation of F8+F6 (EreA in pSB1C3), F8+F7 (EreB in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Florian, Rosario, Louise
</P><P><B>Aim of the experiment:</B> Ligation of F8+F6 (EreA in pSB1C3), F8+F7 (EreB in pSB1C3). Quickchanges have still to be performed.
</P><P><B>Procedure:</B></P><P>Ligation batch for F8+F6 (EreA in pSB1C3), F8+F7 (EreB in pSB1C3):
</P><UL><LI> Ligation batch for F8+F6
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.35 µl
</TD><TD>F8 (74.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>6.76 µl
</TD><TD>F6 (25.9 ng/µl, 1218 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>8.89 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F8+F7
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.35 µl
</TD><TD>F8 (74.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>8.78 µl
</TD><TD>F7 (20.6 ng/µl, 1257 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>6.87 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>
Lid temperature = 37 °C
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Midiprep_of_cloning_vector_RFC25_RFP_generating_device"> Midiprep of cloning vector RFC25 RFP generating device </SPAN></H3><P><B>Investigators:</B> Andreas, Rosario
</P><P><B>Aim of the experiment:</B> Midiprep of our cloning vector RFC25 RFP generating device.
</P><P><B>Procedure:</B> Midiprep was performed after manufacturer's protocol (QIAprep Midiprep, QIAGEN)
</P><P>The yield of our vector was 1854 ng.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_9th"><B>Thursday, May 9th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F8.28pSB1C3.29.2BF6.28EreA.29_and_F8.28pSB1C3.29.2BF7.28EreB.29">  Transformation of <I>E. coli</I> XL1 blue with F8(pSB1C3)+F6(EreA) and F8(pSB1C3)+F7(EreB) </SPAN></H3><P><B>Investigator:</B> Florian, Jeff, Louise, Katrin, Ingmar
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F8(pSB1C3)+F6(EreA) and F8(pSB1C3)+F7(EreB). Quickchanges has now to be performed to remove forbidden restriction sites.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation products F8+F6, F8+F7, F8 negative control were added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_10th"><B>Friday, May 10th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_mutations_found_by_sequencing_in_our_gene_constructs">  Quick Change mutagenesis to remove mutations found by sequencing in our gene constructs  </SPAN></H3><P><B>Investigator: Andi, Jeff, Rosario</B></P><P><B>Aim of the experiment: </B>Deletion of found mutations.
</P><P><B>Operational sequence</B></P><P><B>1. PCR general setup</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>12
</TD><TD>95 °C
</TD><TD> 30 s
</TD></TR><TR><TD>55 °C
</TD><TD> 1 min
</TD></TR><TR><TD>68 °C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI></UL><UL><LI> After the PCR was finished 1 µl of the DpnI restriction enzyme (10 U/µl)was added
</LI></UL><UL><LI> Now the reaction was mixed gently and thoroughly, spinned down in a microcentrifuge for 1 minute, and immediately incubated at 37 °C for 1 hour to digest the parental supercoiled dsDNA 
</LI></UL><P><B>2. Used constructs and primers</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Quickchange number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Geneconstruct</B></TD><TD align="center" style="background:#f0f0f0;"><B>Oligo number</B></TD></TR><TR><TD> QC 1</TD><TD>P27</TD><TD>BPul_Laccase+ pSB1C3</TD><TD>O26 &amp; O27
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmids_F8_.2B_F6"> Picking of transformed Plasmids F8 + F6 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the study:</B> Picking from the overnight transformation of the ligated F8+F6, to see whether ligation is successful or not.
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_TEV_.28P14.2C_pRK793.29"> PCR of TEV (P14, pRK793) </SPAN></H3><P><B>Investigator: Katrin, Jeff</B></P><P><B>Aim of the experiment:</B>  PCR, purification and analytical geleletrophoresis of TEV (P14, pRK793) to produce TEV with RFC25 pre- and suffx (still contains forbidden SpeI-site).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (O34 (TEV_fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (O35 (TEV_rv))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P14)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme with following conditions:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>150 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> Resulting product was labeled as F12.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Digestion_of_P61_with_S1_Nuclease"> Digestion of P61 with S1 Nuclease </SPAN></H3><P><B>Investigator:</B> Katrin
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Oligohybridization_of_O22_.28MinMCS_for.29_and_O23_.28MinMCS_rev.29">  Oligohybridization of O22 (MinMCS_for) and O23 (MinMCS_rev) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment: </B>Oligohybridization of MinMCS_for (O22) and MinMCS_rev (O23
</P><P><B>Procedure:</B></P><UL><LI>25 µL of 100 pmol/µl of  MiniMCS fw  and 25 µL of 100 pmol/µl  MiniMCS rev 
</LI></UL><UL><LI>Heating up to 95 °C for 5 min 
</LI></UL><UL><LI>Cooling at RT in a styropor box overnight
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_ordered_primers_2">  Preparation of ordered primers </SPAN></H3><P><B>Investigators: Andreas, Rosario, Katrin, Ingmar</B></P><P><B>Aim of the experiment:</B> Dissolving of primer pallets to provide a concentration of 100 pmol/µl</P><P><B>Procedure:</B> We added an amount of water which gave us a final primer concentration of 100 pmol/µl. 
</P><P>Primers were labeled as follows:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Oligonr.
</TH></TR><TR><TD> 24
</TD><TD> BPU_for
</TD></TR><TR><TD> 25
</TD><TD> BPU_rev
</TD></TR><TR><TD> 26
</TD><TD> BPU_SDM1_for
</TD></TR><TR><TD> 27
</TD><TD> BPU_SDM1_rev
</TD></TR><TR><TD> 28
</TD><TD> BPU_SDM2_for
</TD></TR><TR><TD> 29
</TD><TD> BPU_SDM2_rev
</TD></TR><TR><TD> 30
</TD><TD> BPU_SDM3_for
</TD></TR><TR><TD> 31
</TD><TD> BPU_SDM3_rev
</TD></TR><TR><TD> 32
</TD><TD> BPU_SDM4_for
</TD></TR><TR><TD> 33
</TD><TD> BPU_SDM4_rev
</TD></TR><TR><TD> 34
</TD><TD> TEV_for
</TD></TR><TR><TD> 35
</TD><TD> TEV_rev
</TD></TR><TR><TD> 36
</TD><TD> TEV_t387c_for
</TD></TR><TR><TD> 37
</TD><TD> TEV_t387c_rev
</TD></TR><TR><TD> 38
</TD><TD> N_Split_TEV_for
</TD></TR><TR><TD> 39
</TD><TD> N_Split_TEV_rev
</TD></TR><TR><TD>40
</TD><TD>C_Split_TEV_for
</TD></TR><TR><TD>41
</TD><TD>C_Split_TEV_rev
</TD></TR><TR><TD>42
</TD><TD>PIF6_2-100_for
</TD></TR><TR><TD>43
</TD><TD>PIF6_2-100_rev
</TD></TR><TR><TD>44
</TD><TD>p104AgeI a71g fw
</TD></TR><TR><TD>45
</TD><TD>p104AgeI a71g rev
</TD></TR><TR><TD>46
</TD><TD>xylE_for
</TD></TR><TR><TD>47
</TD><TD>xylE_rev
</TD></TR><TR><TD>48
</TD><TD>xylE_c312a_for
</TD></TR><TR><TD>49
</TD><TD>xylE_c312a_rev
</TD></TR><TR><TD>50
</TD><TD>xylE_g483a_for
</TD></TR><TR><TD>51
</TD><TD>xylE_g483a_rev
</TD></TR><TR><TD>52
</TD><TD>xylE_a834g_for
</TD></TR><TR><TD>53
</TD><TD>xylE_a834g_rev
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_May_11th"><B>Saturday, May 11th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_PCR_product_of_S219V_pRK793_.28F12.29_with_XbaI_.26_AgeI"> Preparative digestion of PCR product of S219V pRK793 (F12) with XbaI &amp; AgeI </SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Preparative digestion of PCR product of S219V pRK793 (F12) with XbaI &amp; AgeI. Quickchange has still to be performed (forbidden SpeI restriction site).
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR product S219V pRK793 (F12) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>PCR product F12
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C
</LI></UL><UL><LI> Digested products were purified to remove nucleotides and enzymes with QIAquick PCR Purification Kit, QIAGEN.
</LI></UL><UL><LI> The digested fragment was labelled F15
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E..7Ecoli_XL1-Blue_with_F8_.28pSB1C3.29_.2B_F6_.28EreA.29">  Miniprep of overnight culture of transformated <I>E.~coli</I> XL1-Blue with F8 (pSB1C3) + F6 (EreA)  </SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E.~coli</I> XL1-Blue with F8 (pSB1C3) + F6 (EreA) of 2 clones picked the day before 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI> The plasmids were labelled as P63 and P64
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P63_and_P64_with_AgeI-HF_and_XbaI">  Analytical digestion and gelelectrophoresis of P63 and P64 with AgeI-HF and XbaI  </SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P63 and P64 with AgeI-HF and XbaI
</P><P><B>Procedure:</B></P><UL><LI> Batch for analytical digestion of P63 and P64 with AgeI-HF and XbaI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA P63/P64
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI(10 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Analytical digestion of P63 with AgeI-HF and XbaI
</TD><TD>1 kbp ladder
</TD><TD>Analytical digestion of P64 with AgeI-HF and XbaI
</TD></TR><TR><TD><B>Insertion successful</B></TD><TD><B>Insertion successful</B></TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_PCR_product_of_EreB_.28F2.29_with_XbaI_.26_AgeI"> Preparative digestion of PCR product of EreB (F2) with XbaI &amp; AgeI </SPAN></H3><P><B>Investigator: Louise, Johanna</B></P><P><B>Aim of the experiment:</B> Preparative digestion of PCR product EreB (F2) with XbaI &amp; AgeI
(second try since there did not grow any transformed bacteria in the first time).
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR product and EreB (F2) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>PCR product F2
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>22.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C
</LI><LI> The digested fragment was labelled F14
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Purification_and_ligation_of_digested_PCR_product_EreB_.28F14.29"> Purification and ligation of digested PCR product EreB (F14) </SPAN></H3><P><B>Investigator:</B> Louise, Rosario
</P><P><B>Aim of the experiment:</B> PCR Purification and ligation of digested PCR product EreB (F14).
</P><P><B>Procedure:</B>
PCR Purification of EreB PCR (F14)
</P><UL><LI> To purify the digested Fragment(F14) the QIAquick PCR Purification Kit was used. The concentration of the purified product was found to be 5 ng/µl via Nano drop.
</LI></UL><P>Ligation of F14 + F8(pSB1C3)
</P><UL><LI> Ligation batch for F8+F14 (positive control)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.35 µl
</TD><TD>F8 (74.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>15.65 µl
</TD><TD>F14 (5 ng/µl, 1257 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch.
</LI><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>
Lid temperature = 37 °C
</P></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_products_of_F10_.2B_F11_into_E.coli_Xl1-Blue"> Transformation of ligation products of F10 + F11 into <I>E.coli</I> Xl1-Blue</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Transformation of the ligation products of F10 + F11 and the  negative control in pTUM104 in XL1 Blue.
</P><P><B>Operation Sequence</B></P><UL><LI> melting of 2x 200 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> Preparing 4 Tubes with 100 µl of Ca-competent  <I>E.coli</I> XL1-Blue cells
</LI><LI> addition of 5 µl of the ligation products
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_Quickchange_Product_P62_into_E.coli_Xl1-Blue"> Transformation of Quickchange Product P62 into <I>E.coli</I> Xl1-Blue</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Transformation of the Quickchange Product P62 and the  negative control in XL1 Blue. 
</P><P><B>Operation Sequence</B></P><UL><LI> melting of 1x 200 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> Preparing 4 Tubes with 50 µl of Ca-competent  <I>E.coli</I> XL1-Blue cells
</LI><LI> addition of 1 µl of the Quickchange Product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quickchange_mutagenesis_of_pTUM104">  Quickchange mutagenesis of pTUM104  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment:</B> Sequencing results showed a forbiden AgeI restriction site in this vector. The sdm will delete this restricition site 
</P><P><B>Procedure:</B><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P6 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O44 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P6 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O45 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. The resulting product was named fragmet 14.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_May_12th"><B>Sunday, May 12th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F15_.28digested_TEV_PCR_product.29"> Analytical gelelectrophoresis of F15 (digested TEV PCR product) </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F15 (digested TEV PCR product) to check whether PCR has worked.
</P><P><B>Procedure:</B></P><UL><LI> 5 µl of F15 was mixed with 0.55 µl of DNA loading buffer (10x) and analytical gelelectrophoresis was performed at 1% agarose gel for 60 min at 90 V.
</LI></UL><UL><LI> PCR product has expected length
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_ligation_product_F8.2BF15_.28TEV_in_pSB1C3.2C_RFC25.29">  Transformation of <I>E. coli</I> XL1 blue with ligation product F8+F15 (TEV in pSB1C3, RFC25)  </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with ligation product F8+F15 (TEV in pSB1C3, RFC25). Further quickchanges still have to be done.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of the ligation products were added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_ligation_products_F8.2BF14_.28EreB_in_pSB1C3.2C_RFC25.29">  Transformation of <I>E. coli</I> XL1 blue with ligation products F8+F14 (EreB in pSB1C3, RFC25)  </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with ligation product F8+F14 (EreB in pSB1C3, RFC25). Further quickchanges still have to be done.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of the ligation products were added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P23">  Transformation of <I>E. coli</I> XL1 blue with P23 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P23 for insertion of miniMCS.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_F10.2BF11_.28IRES_from_polio_virus.2C_blunt_ligated_with_pSB1C3_for_sequencing.29"> Picking of transformed Plasmid F10+F11 (IRES from polio virus, blunt ligated with pSB1C3 for sequencing) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid F10+F11 (AlcR blunt ligated with pSB1C3 for sequencing).
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL><UL><LI> 10 colonies were picked.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_4">Week 4</SPAN></H1><DIV class="week" id="WWeek_4"><H2><SPAN class="mw-headline" id="Monday.2C_May_13th"><B>Monday, May 13th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E..7Ecoli_XL1-Blue_with_F10.2BF11">  Miniprep of overnight culture of transformated <I>E.~coli</I> XL1-Blue with F10+F11  </SPAN></H3><P><B>Investigator: Rosario, Jeffery, Johanna, Flo </B></P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E.~coli</I> XL1-Blue F10+F11 of 10 clones picked the day before 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analyzing_sequencing_results"> Analyzing sequencing results </SPAN></H3><P><B>Investigator: Johanna </B></P><P><B>Aim of the experiment:</B> Comparison of the sequencing results with the Biobrick (BB) sequences.
</P><P><B>Procedure:</B> Using Geneious to align the sequencing results with the BB sequences.
</P><P><B>LMU  GFP in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A>(p20)</B></P><UL><LI> The provided sequence in the parts registry is totally wrong.
</LI><LI> The comparison of the amino acid seqeunces reveals several point mutations.
</LI></UL><P><B>FluA in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> (p45)</B></P><UL><LI>Did not yield sequencing results (does the primer bind ?)
</LI><LI>No presence of RCF 25 standard restriction enzymes
</LI></UL><P><B>mCherry in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> (p46)</B></P><UL><LI>Did not yield sequencing results (does the primer bind ?)
</LI><LI>No presence of RCF 25 standard restriction enzymes
</LI></UL><P><B>CaMV35S in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> (p23)</B></P><UL><LI>high identity 
</LI><LI>high match with HQ699544 (Blast)
</LI></UL><P><B>LMU mKate2in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> (p21)</B></P><UL><LI>no continous ORF
</LI><LI>Did not yield sequencing results
</LI></UL><P><B>DREB1C promoter in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> (p26)</B></P><UL><LI>Present of fluorecent protein gene
</LI></UL><P><B>xylE in RCF 25 (p38)</B></P><UL><LI>Did not yield sequencing results
</LI></UL><P><B>SV40 in RCF 25 (p28)</B></P><UL><LI>sequencing result agrees with the laccases
</LI><LI>the sequencing result can't be the one of SV40
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quickchange_mutagenesis_SDMI_of_p27">  Quickchange mutagenesis SDMI of p27  </SPAN></H3><P><B>Investigator: Rosario, Jeff, Flo, Andi</B></P><P><B>Aim of the experiment:</B> Sequencing results showed a forbidden restriction site in this vector. The sdm will delete this restricition site. The resulting Plasmid was labeled 65. 
</P><P><B>Procedure:</B><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid P27 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O26 (10 pmol/µL)
</TD></TR><TR><TD>20. 5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid P27 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O27 (10 pmol/µL)
</TD></TR><TR><TD>20.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 4 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied, but the amount of cycles was increased to 20 
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quickchange_mutagenesis_SDMI_of_p38">  Quickchange mutagenesis SDMI of p38  </SPAN></H3><P><B>Investigator: Jeff, Flo, Andi, Rosario</B></P><P><B>Aim of the experiment:</B> Sequencing results showed a forbidden restriction site in this vector. The sdm will delete this restricition site. The resulting Plasmid was labeled 67.
</P><P><B>Procedure:</B><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0,5 µl
</TD><TD>Plasmid P38 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O48 (10 pmol/µL)
</TD></TR><TR><TD>20.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid P38 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O49 (10 pmol/µL)
</TD></TR><TR><TD>20.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 4 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied, but the amount of cycles have been increased to 20 
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quickchange_mutagenesis_SDMI_of_P63">  Quickchange mutagenesis SDMI of P63  </SPAN></H3><P><B>Investigator: Jeff, Rosario, Flo, Andi</B></P><P><B>Aim of the experiment:</B> Sequencing results showed a forbidden restriction site in this vector. The sdm will delete this restricition site. The resulting Plasmid was labeled P66. 
</P><P><B>Procedure:</B><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>Plasmid P63 template; 1:2 dilution
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O8 (10 pmol/µL)
</TD></TR><TR><TD>20.75 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>Plasmid P63 template; 1:2 dilution
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O9 (10 pmol/µL)
</TD></TR><TR><TD>20.75 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied, the amount of cycles was raised to 20
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P65">  Transformation of <I>E. coli</I> XL1 blue with P65 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario, Johanna, Andi, Flo
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P65 .
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P66">  Transformation of <I>E. coli</I> XL1 blue with P66 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario, Flo, Johanna, Andi
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P66.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P67">  Transformation of <I>E. coli</I> XL1 blue with P67 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario, Johanna, Andi, Flo
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P67.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_May_14th"><B>Tuesday, May 14th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P15_and_P19_to_P52"> Analytical digestion of P15 and P19 to P52</SPAN></H3><P><B>Investigator:</B> Florian, Johanna
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of  P15 and P19 to P52 to check the sequencing results.
</P><P><B>Procedure:</B></P><UL><LI>Batch for analytical digestion of P15 and P19 to P52
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>5 µl
</TD><TD>Mastermix
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Batch of Mastermix
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>40 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>NotI(10 U/µl)
</TD></TR><TR><TD>148 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=200 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><UL><LI><B>Discussion:</B></LI><LI> Gel 1, Row 1:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>lane</B></TD><TD><B>part</B></TD><TD><B>plasmid</B></TD><TD><B>band 1</B></TD><TD><B>band 2</B></TD><TD><B>result</B></TD></TR><TR><TD>2
</TD><TD>EreA
</TD><TD>pDEST14
</TD><TD>3,25 kb
</TD><TD>no cut, no restriction sites in pDEST14
</TD></TR><TR><TD>3
</TD><TD>mCherry(LMU)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>750 bp
</TD><TD>probably right
</TD></TR><TR><TD>4
</TD><TD>GFP(LMU)(845 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>750 bp
</TD><TD>maybe switched with mKate2(LMU)
</TD></TR><TR><TD>5
</TD><TD>mKate2(LMU)(702 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>900 bp
</TD><TD>maybe switched with GFP(LMU)
</TD></TR><TR><TD>6
</TD><TD>35S + Strong Plant + GFP(1775 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>750 bp
</TD><TD>?
</TD></TR><TR><TD>7
</TD><TD>UVR8(1332 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>1,1 kb
</TD><TD>?
</TD></TR><TR><TD>8
</TD><TD>mStrawberry(708 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>1,4 kb
</TD><TD>?
</TD></TR><TR><TD>9
</TD><TD>DREB1C promoter(463 bp)
</TD><TD>pSB1C3
</TD><TD>2,5 kb
</TD><TD>probably only one cut -&gt; one NotI site mutated
</TD></TR><TR><TD>10
</TD><TD>Laccase(1587 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>1,3 kb
</TD><TD>?
</TD></TR><TR><TD>11
</TD><TD>SV40(419 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>1,6 kb
</TD><TD>this could be the laccase
</TD></TR><TR><TD>12
</TD><TD>cjBlue(696 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,5 kb
</TD><TD>?
</TD></TR></TBODY></TABLE><UL><LI> Gel 1, Row 2:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>lane</B></TD><TD><B>part</B></TD><TD><B>plasmid</B></TD><TD><B>band 1</B></TD><TD><B>band 2</B></TD><TD><B>result</B></TD></TR><TR><TD>2
</TD><TD>alcA (843 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,7 kb
</TD><TD>?
</TD></TR><TR><TD>3
</TD><TD>eforRed (681 bp)
</TD><TD>pSB1C3
</TD><TD>plasmid
</TD><TD>plasmid bands (coiled, supercoiled, etc.)
</TD></TR><TR><TD>4
</TD><TD>amilGFP (699 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,7 kb
</TD><TD>right part
</TD></TR><TR><TD>5
</TD><TD>mCFP (714 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,7 kb
</TD><TD>right part
</TD></TR><TR><TD>6
</TD><TD>RD29A promoter (1535 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>1,5 kb
</TD><TD>right part
</TD></TR><TR><TD>7
</TD><TD>amilCP (669 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,7 kb
</TD><TD>right part
</TD></TR><TR><TD>8
</TD><TD>CMV promoter (580 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,6 kb
</TD><TD>right part
</TD></TR><TR><TD>9
</TD><TD>middle linker (24 bp)
</TD><TD>pMA-BBFR
</TD><TD>2,4 kb
</TD><TD>short insert went through the gel
</TD></TR><TR><TD>10
</TD><TD>GSAT (108 bp)
</TD><TD>pMA-BBFR
</TD><TD>2,4 kb
</TD><TD>0,1 kb
</TD><TD>right part
</TD></TR><TR><TD>11
</TD><TD>GFP(720 bp)
</TD><TD>pSB1A2
</TD><TD>?
</TD></TR><TR><TD>12
</TD><TD>long linker (36 bp)
</TD><TD>pMA-BBFR
</TD><TD>2,4 kb
</TD><TD>short insert went through the gel
</TD></TR></TBODY></TABLE><UL><LI> Gel 2:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>lane</B></TD><TD><B>part</B></TD><TD><B>plasmid</B></TD><TD><B>band 1</B></TD><TD><B>band 2</B></TD><TD><B>result</B></TD></TR><TR><TD>2
</TD><TD>SEG linker (108 bp)
</TD><TD>pMA-BBFR
</TD><TD>2,4 kb
</TD><TD>short insert went through the gel
</TD></TR><TR><TD>3
</TD><TD>short linker (12 bp)
</TD><TD>pMA-BBFR
</TD><TD>2,4 kb
</TD><TD>short insert went through the gel
</TD></TR><TR><TD>4
</TD><TD>FluA (522 bp)
</TD><TD>pMA-BBFR
</TD><TD>2,4 kb
</TD><TD>0,5 kb
</TD><TD>right part
</TD></TR><TR><TD>5
</TD><TD>mCherry (769 bp)
</TD><TD>pSB2K3
</TD><TD>4,4 kb
</TD><TD>0,75 kb
</TD><TD>right part
</TD></TR><TR><TD>6
</TD><TD>Cerulean (778 bp)
</TD><TD>pSB2K3
</TD><TD>?
</TD></TR><TR><TD>7
</TD><TD>mRFP1 (706 bp)
</TD><TD>pSB2K3
</TD><TD>4,4 kb
</TD><TD>0,7 kb
</TD><TD>right part
</TD></TR><TR><TD>8
</TD><TD>mOrange (769 bp)
</TD><TD>pSB2K3
</TD><TD>4,4 kb
</TD><TD>0,75 kb
</TD><TD>right part
</TD></TR><TR><TD>9
</TD><TD>PIF3-20aa (366 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,35 kb
</TD><TD>right part
</TD></TR><TR><TD>10
</TD><TD>20aa-PIF3 (366 bp)
</TD><TD>pSB1C3
</TD><TD>2,0 kb
</TD><TD>0,35 kb
</TD><TD>two plasmids and two insert can be seen
</TD></TR><TR><TD>11
</TD><TD>LexA (ca. 6 kb)
</TD><TD>pSB1C3
</TD><TD>&gt; 8 kb
</TD><TD>only one cut, huge plasmid
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_Plasmid_P23_.28CaMV35S.29_and_Ligation_Product_F8_.2B_F15_.28TEV.29"> Picking of Plasmid P23 (CaMV35S) and Ligation Product F8 + F15 (TEV) </SPAN></H3><P><B>Investigator:</B> Jeff, Florian
</P><P><B>Aim of the study:</B> Picking of Plasmid P23 (CaMV35S) and Ligation Product F8 + F15 (TEV) for Miniprep.
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL><UL><LI> 1 colony of P23 were picked.
</LI></UL><UL><LI> 7 colonies of F8 + F15 were picked.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Sequencing_of_P63_.26_P64">  Sequencing of P63 &amp; P64 </SPAN></H3><P><B>Investigators: Louise</B></P><P><B>Aim of the experiment:</B> Sequencing of genes for which no prior sequencing results were available.
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> p63
</TD><TD> EreA
</TD><TD> FR01002349
</TD><TD> FR01002350
</TD></TR><TR><TD> p64
</TD><TD> EreA
</TD><TD> FR01002351
</TD><TD> FR01002352
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_May_15th"><B>Wednesday, May 15th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_psB1C3-RFP-generator_.28P60.29_and_another_psB1C3_.28P8.29_as_well_as_P64_.28EreA_in_defect_psB1C3_.28F8.29.29_with_XbaI_.26_AgeI"> Preparative digestion of psB1C3-RFP-generator (P60) and another psB1C3 (P8) as well as P64 (EreA in defect psB1C3 (F8)) with XbaI &amp; AgeI </SPAN></H3><P><B>Investigator: Louise,Johanna</B></P><P><B>Aim of the experiment:</B>Preparative digestion of psB1C3-RFP-generator (P60) and another psB1C3 (P8) as well as P64 (EreA in defect psB1C3 (F8)) with XbaI &amp; AgeI to recover EreA and prepare two new vectors for cloning.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR product and EreB (F2) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid (P8,P60,P64)
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C
</LI><LI> The digested fragments were labelled F17(P8), F18(P60), F19(P64)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR.2C_purification_and_analytical_geleletrophoresis_of_and_EreB_.28P16.29"> PCR, purification and analytical geleletrophoresis of and EreB (P16) </SPAN></H3><P><B>Investigator: Jeff, Louise</B></P><P><B>Aim of the experiment:</B> PCR of EreB (P16).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (O31 (EreB_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (O32 (EreB_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P15; P16)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program TMKS was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>80 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> 4 µl of the purified PCR products were mixed with 0.4 µl DNA loading buffer (10x) for analytical gelelectrophoresis
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 30 min in 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>F16
</TD></TR><TR><TD>Fragment-size like expected
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Purification_and_ligation_of_digested_products_F17.2C_F18.2C_F19"> Purification and ligation of digested products F17, F18, F19 </SPAN></H3><P><B>Investigator:</B> Jeff, Louise
</P><P><B>Aim of the experiment:</B> Purification and ligation of digested products F17, F18, F19.
</P><P><B>Procedure:</B></P><UL><LI> To purify the digested Fragments the QIAquick PCR Purification Kit was used. The concentration of the purified product was found to be XX ng/µl via Nano drop.
</LI></UL><P>Ligation of F19 + FXX(pSB1C3)
</P><UL><LI> Ligation batch for F19+FXX (positive control)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.35 µl
</TD><TD>F8 (74.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>X µl
</TD><TD>F14 (5 ng/µl, 1257 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch.
</LI><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_analytical_gelelectrophoresis_of_P90-P97_.28TEV_protease.2C_XbaI.2BAgeI.29_and_P97_.28CaMV_p35S.2C_XbaI.2BPstI.29"> Analytical digestion and analytical gelelectrophoresis of P90-P97 (TEV protease, XbaI+AgeI) and P97 (CaMV p35S, XbaI+PstI) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and analytical gelelectrophoresis of P90-P97 (TEV protease, XbaI+AgeI) and P97 (CaMV p35S, XbaI+PstI).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P90-P96 with XbaI+AgeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>16 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.6 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>118.4 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=140 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for digestion of P90-P96 with XbaI+AgeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>17.5 µl
</TD><TD>Mastermix
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for digestion of P90-P96 with XbaI+AgeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>Digestion of P90</B></TD><TD><B>Digestion of P91</B></TD><TD><B>Digestion of P92</B></TD><TD><B>Digestion of P93</B></TD><TD><B>Digestion of P94</B></TD><TD><B>Digestion of P95</B></TD><TD><B>Digestion of P96</B></TD><TD><B>Digestion of P97</B></TD></TR><TR><TD>Insert has expected length
</TD><TD>Insert has expected length
</TD><TD>Insert has expected length
</TD><TD>Insert has expected length
</TD><TD>Corrupt
</TD><TD>Insert has expected length
</TD><TD>Insert has expected length
</TD><TD>Corrupt
</TD></TR></TBODY></TABLE><UL><LI> The inserts for TEV semm to be right in the most cases, but the backbone has mutated and it should be cloned into a new pSB1C3 with new <B>same restriction enzymes (XbaI+AgeI)</B>!
</LI></UL><UL><LI> The forward sequencing of CaMV p35S is right but the analytical digestion is wrong. Maybe further parts are downstream of the promoter?! Reverse sequencing should be performed for further analysis.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_16th"><B>Thursday, May 16th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_pSH21_.28P61.29.2C_psB1A2.28P41.29_and_P96_.28TEV_S19V.29">  Preparative digestion of pSH21 (P61), psB1A2(P41) and P96 (TEV S19V)   </SPAN></H3><P><B>Investigator: Jeff, Katrin </B></P><P><B>Aim of the experiment:</B> Preparative digestion of pSH21 (P61)and psB1A2(P41) with EcoRI and P96 (TEV S19V) with XbaI &amp; AgeI for subsequent ligation.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of pSH21 with EcoRI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P61
</TD></TR><TR><TD>4 µl
</TD><TD>Buffer 4(10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of psB1A2 with EcoRI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P41
</TD></TR><TR><TD>4 µl
</TD><TD>Buffer 4(10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P96 (TEV S219V) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P96
</TD></TR><TR><TD>4 µl
</TD><TD>Buffer 4(10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> The resulting fragments were named: F23 (P41) and F22 (P61)
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the reaction batch containing pSH21 (P61) after digestion and were loaded on an 1% agarose gel for preparative gelelectrophoresis. 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V for 90 min. 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P61 digestion = F23
</TD></TR><TR><TD>Fragment-size like expected; band was cut out
</TD></TR></TBODY></TABLE><UL><LI> The band had the expected length. They were cut out and purified using the QIAquick Gel Extraction Kit, QIAGEN
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_ereB_.28P64.29_with_XbaI_.26_AgeI.">  Preparative digestion of ereB (P64)  with XbaI &amp; AgeI.  </SPAN></H3><P><B>Investigator: Jeff, Louise, Katrin </B></P><P><B>Aim of the experiment:</B>Preparative digestion of EreB (P64) with XbaI &amp; AgeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR product and EreB (F2) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C
</LI><LI> The digested fragments were labelled 
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_F23">  Dephosphorylation of F23  </SPAN></H3><P><B>Investigator: Jeff, Katrin </B></P><P><B>Aim of the experiment:</B> Dephosphorylation of cut vector psB1A2 to prevent re-ligation
</P><P><B>Procedure:</B></P><UL><LI> the digestion product was purified with QIAquick PCR Purification Kit, Qiagen in order to change the buffer
</LI></UL><UL><LI> Batch for the dephosphorylation of F23
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>Plasmid DNA F23 (2 µg)
</TD></TR><TR><TD>4 µl
</TD><TD>10X Fast AP buffer
</TD></TR><TR><TD>2 µl
</TD><TD>FastAP Thermosensitive Alkaline
<P>Phosphatase
</P></TD></TR><TR><TD>4 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> the reaction batch was spun briefly and incubated at 37 °C for 10 minutes
</LI><LI> the reaction was stopped by heating to 75 °C for 5 minutes
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_pSH21_.28F22.29_with_psB1A2.28F23.29.2C_TEV_.28P96.29.2C_EreA_.28P64.29.2C_pSB1A2.28P41.29">  Ligation of pSH21 (F22) with psB1A2(F23), TEV (P96), EreA (P64), pSB1A2(P41)   </SPAN></H3><P><B>Investigator: Jeff,  Katrin </B></P><P><B>Aim of the experiment:</B> ligation of pSH21 (F22) with pSB1A2(F23), TEV (F20) with pSB1C3 (F17), EreA (F19)with pSB1C3 (F17) and EreB (F21) with pSB1C3 (F17)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for EreB (F21) with pSB1C3 (F17)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.6 µl
</TD><TD>F17 (61,4 ng/µl)
</TD></TR><TR><TD>15,4  µl
</TD><TD>F21 (8,3 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for TEV (F20) with pSB1C3 (F17)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.6 µl
</TD><TD>F17 (61,4 ng/µl)
</TD></TR><TR><TD>4,1  µl
</TD><TD>F20 (24,7  ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11,3 µl
</TD><TD>ddH20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for pSH21 (F22) with pSB1A2(F23)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,85  µl
</TD><TD>F23 (118,1 ng/µl)
</TD></TR><TR><TD>2,3  µl
</TD><TD>F22 (119,0  ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13,85 µl
</TD><TD>ddH20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for EreA (F19) with pSB1C3 (F17)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.6 µl
</TD><TD>F17 (61,4 ng/µl)
</TD></TR><TR><TD>6,4  µl
</TD><TD>F19 (8,3 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9  µl
</TD><TD>ddH20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch.
</LI><LI> the ligation was performed at 16 °C overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_17th"><B>Friday, May 17th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F17.2BF19.2C_F17.2BF20.2C_F17.2BF21.2C_F22.2BF23.2C_F17_.28negative_control.29.2C_F23_.28negative_control.29">  Transformation of <I>E. coli</I> XL1 blue with F17+F19, F17+F20, F17+F21, F22+F23, F17 (negative control), F23 (negative control) </SPAN></H3><P><B>Investigator: Jeff, Katrin </B></P><P><B>Aim of the experiment:</B> Transformation of ligation products F17+F21 (pSB1C3, EreB), F17+F20 (pSB1C3, TEV), F17+F19 (pSB1C3, EreA), F23+F22 (pSB1A2, pSH21) and negative controls
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on chloramphenicol plates (F17+F19, F17+F20, F17+F21, F17 negative) or ampicillin plates (F23+F22, F23 negative control)
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was discarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates (F17+F19, F17+F20, F17+F21) or ampicillin plates (F23+F22)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_mutations_found_by_sequencing_in_our_gene_constructs_2">  Quick Change mutagenesis to remove mutations found by sequencing in our gene constructs  </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment: </B>removal of forbidden restiction sites from laccase
</P><P><B>Operational sequence</B></P><P><B>PCR general setup</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>DNA template (p28)
</TD></TR><TR><TD>0,5 µl each
</TD><TD>1:10 dilution of appropriate Oligos (1: O26 + O27, 2: O28 + O29, 3: O30 + O31, 4: O32 + O33 )
</TD></TR><TR><TD>42 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>1 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>12
</TD><TD>95 °C
</TD><TD> 30 s
</TD></TR><TR><TD>55 °C
</TD><TD> 1 min
</TD></TR><TR><TD>68 °C
</TD><TD> 4 min
</TD></TR><TR><TD>3
</TD><TD>1
</TD><TD>4 °C
</TD><TD> hold
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_the_4_Quickchangeproducts_using_1:_O26_.2B_O27.2C_2:_O28_.2B_O29.2C_3:_O30_.2B_O31.2C_4:_O32_.2B_O33">  Transformation of <I>E. coli</I> XL1 blue with the 4 Quickchangeproducts using 1: O26 + O27, 2: O28 + O29, 3: O30 + O31, 4: O32 + O33 </SPAN></H3><P><B>Investigator: Jeff, Andi, Johanna </B></P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with the 4 Quickchangeproducts using 1: O26 + O27, 2: O28 + O29, 3: O30 + O31, 4: O32 + O33
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on chloramphenicol plates 1: O26 + O27, 2: O28 + O29, 3: O30 + O31, 4: O32 + O33
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was discarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates (1: O26 + O27, 2: O28 + O29, 3: O30 + O31, 4: O32 + O33)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_May_18th"><B>Saturday, May 18th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_F17.2BF19.2C_F17.2BF20.2C_F17.2BF21.2C_F22.2BF23"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with F17+F19, F17+F20, F17+F21, F22+F23  </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid F17+F19, F17+F20, F17+F21, F22+F23 .
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR/AmpR LB-medium)
</LI></UL><UL><LI> 2 colonies were picked for every Transformation product.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_May_19th"><B>Sunday, May 19th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E..C2.A0coli_XL1_blue_with_F17.2BF19.2C_F17.2BF20.2C_F17.2BF21.2C_F22.2BF23">  Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with F17+F19, F17+F20, F17+F21, F22+F23  </SPAN></H3><P><B>Investigator: Jeff, Andi </B></P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with F17+F19, F17+F20, F17+F21, F22+F23 of 2 clones picked the day before 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_analytical_gelelectrophoresis_of_F17.2BF19.2C_F17.2BF20.2C_F17.2BF21_and_F22.2BF23"> Analytical digestion and analytical gelelectrophoresis of F17+F19, F17+F20, F17+F21 and F22+F23 </SPAN></H3><P><B>Investigator:</B> Jeff, Andi
</P><P><B>Aim of the experiment:</B> Analytical digestion and analytical gelelectrophoresis of ligation products F17+F19, F17+F20, F17+F21, F22+F23.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of F17+F19, F17+F20, F17+F21 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>14 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1.75 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1.75 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>103.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=122.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for digestion of F22+F23 with EcoRI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>15.25 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed two times for each ligation product at 90 V for 60 min.
</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>Digestion of F17+F19</B></TD><TD><B>Digestion of F17+F19</B></TD><TD><B>Digestion of F17+F20</B></TD><TD><B>Digestion of F17+F20</B></TD><TD><B>Digestion of F17+F21</B></TD><TD><B>Digestion of F17+F21</B></TD><TD><B>Digestion of F22+F23</B></TD><TD><B>Digestion of F22+F23</B></TD></TR><TR><TD>Insert has expected length
</TD><TD>Insert has expected length, the first line represents the rest of the uncut Plasmid
</TD><TD>Insert has expected length
</TD><TD>Insert has expected length, the first line represents the rest of the uncut Plasmid
</TD><TD>Insert has expected length
</TD><TD>Insert has expected length
</TD><TD>Insert has expected length, insert and cut Plasmid overlay each other, because of the same length
</TD><TD>Corrupt
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_of_P28_.28Laccase.29.2C_P100_.28TEV_S219V.29.2C_P102_.28EreB.29">  Quick Change mutagenesis to remove forbidden restriction sites of P28 (Laccase), P100 (TEV S219V), P102 (EreB) </SPAN></H3><P><B>Investigator: Jeff, Andi</B></P><P><B>Aim of the experiment: </B> Removal of forbidden restiction sites from P28 (Laccase), P100 (TEV S219V), P102 (EreB).
</P><P><B>Operational sequence</B></P><P><B>PCR general setup</B></P><P><B>Procedure:</B></P><P><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid template (P28/P100/P102)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O26 (for P28, 10 µM)/O16 (for P102, 10 µM)/O36 (for P100, 10 µM)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P38 template
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O27 (for P28, 10 µM)/O17 (for P102, 10 µM)/O37 (for P100, 10 µM)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR></TBODY></TABLE><UL><LI> Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied, but the amount of cycles have been increased to 20.
</LI></UL><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_QuickChange_products_of_P100.2C_P102_and_P28_to_check_whether_the_QuickChange_PCR_and_DpnI_digestion_were_successful"> Analytical gelelectrophoresis of QuickChange products of P100, P102 and P28 to check whether the QuickChange PCR and DpnI digestion were successful  </SPAN></H3><P><B>Investigator:</B></P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_the_Quickchange_products_with_P28.2C_P100_and_P102_into_E..C2.A0coli_XL1_blue">  Transformation of the Quickchange products with P28, P100 and P102 into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Jeff, Andi
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P28, P102 and P100.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H1><DIV class="week" id="WWeek_5"><H2><SPAN class="mw-headline" id="Tuesday.2C_May_21st"><B>Tuesday, May 21st</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMI_-_of_P28_.28Laccase.29.2C_P100_.28TEV_S219V.29.2C_P102_.28EreB.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMI - of P28 (Laccase), P100 (TEV S219V), P102 (EreB) </SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Johanna
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction sites from P28 (Laccase), P100 (TEV S219V), P102 (EreB).
</P><P><B>Operational sequence</B></P><P><B>PCR general setup</B></P><P><B>Procedure:</B></P><P><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid template (P28/P100/P102)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O26 (for P28, 10 µM)/O16 (for P102, 10 µM)/O36 (for P100, 10 µM)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P38 template
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O27 (for P28, 10 µM)/O17 (for P102, 10 µM)/O37 (for P100, 10 µM)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR></TBODY></TABLE><UL><LI> Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied, but the amount of cycles have been increased to 20.
</LI></UL><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_QuickChange_products_of_P100.2C_P102_and_P28_to_check_whether_the_QuickChange_PCR_and_DpnI_digestion_were_successful_2"> Analytical gelelectrophoresis of QuickChange products of P100, P102 and P28 to check whether the QuickChange PCR and DpnI digestion were successful  </SPAN></H3><P><B>Investigator:</B></P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_the_Quickchange_products_with_P28.2C_P100_and_P102_into_E..C2.A0coli_XL1_blue_2">  Transformation of the Quickchange products with P28, P100 and P102 into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Johanna
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P28, P102 and P100.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P105.2C_P28.2C_P100.2C_P102_.26_P98">  Sequencing of P105, P28, P100, P102 &amp; P98 </SPAN></H3><P><B>Investigators: Jeff, Andi,Johanna</B></P><P><B>Aim of the experiment:</B> Sequencing of genes for which no prior sequencing results were available.
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> p105
</TD><TD> IRES
</TD><TD> FR01002288
</TD><TD> FR01002269
</TD></TR><TR><TD> p102
</TD><TD> EreB
</TD><TD> FR01002270
</TD><TD> FR01002275
</TD></TR><TR><TD> p100
</TD><TD> TEV S219V
</TD><TD> FR01002278
</TD><TD> FR01002279
</TD></TR><TR><TD> p98
</TD><TD> EreA
</TD><TD> FR01002276
</TD><TD> FR01002277
</TD></TR><TR><TD> p28
</TD><TD> Laccase
</TD><TD> FR01002280
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_Knop-Medium_for_Moss">Preparation of Knop-Medium for Moss</SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Johanna, Rosario
</P><P><B>Aim of the experiment:</B> Preparation of Knop-Medium for Moss
</P><P><B>Procedure:</B></P><UL><LI> All four prepared stocks have been: 25g/L KH2PO4; 25g/L KCl; 25g/L MgSO4 x 7 H2O; 100g/L Ca(NO3)2  
</LI><LI> Volume of prepared Medium: 1L
</LI><LI> 10 ml of each stock were converted into a 1,8L Erlenmeyer flask and filled up to 1L with destilled water (ELGA); 12,5mg FeSO4 x 7 H2O were added; the pH was set to 5.8 with KOH
</LI><LI> The prepared Volume of 1L was shared equally to 500ml Erlenmeyer flasks, so each Erlenmeyer flask contained 200 ml of the Knop-Medium 
</LI><LI> All flasks have been autoclaved 
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_QuickChange_products_of_P100_.28TEV.2C_QC_with_O36.2FO37.29_and_P102_.28EreB.2C_QC_with_O16.2FO17.29"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QuickChange products of P100 (TEV, QC with O36/O37) and P102 (EreB, QC with O16/O17) </SPAN></H3><P><B>Investigator:</B> Andi, Jeff, Johanna, Rosario
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QuickChange products of P100 (TEV, QC with O36/O37) and P102 (EreB, QC with O16/O17) from Sunday, May 19th.
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL><UL><LI> 3 colonies for P100QC (TEV, QC with O36/O37) and 5 colonies from P102QC (EreB, QC with O16/O17) were picked from the plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_May_22nd"><B>Wednesday, May 22nd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E..C2.A0coli_XL1_blue_with_QuickChange_products_of_P100_.28TEV.2C_QC_with_O36.2FO37.29_and_P102_.28EreB.2C_QC_with_O16.2FO17.29">  Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with QuickChange products of P100 (TEV, QC with O36/O37) and P102 (EreB, QC with O16/O17) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with QuickChange products of P100 (TEV, QC with O36/O37) and P102 (EreB, QC with O16/O17).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_QuickChange_products_of_P28_.28Laccase.2C_QC_with_O26.2FO27.29"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QuickChange products of P28 (Laccase, QC with O26/O27) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QuickChange products of P28 (Laccase, QC with O26/O27).
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL><UL><LI> 8 colonies for P28QC (Laccase, QC with O26/O27) were picked from the plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_analytical_gelelectrophoresis_of_P106.2C_P107.2C_P108.2C_P109.2C_P110.2C_P111.2C_P112.2C_P113"> Analytical digestion and analytical gelelectrophoresis of P106, P107, P108, P109, P110, P111, P112, P113 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and analytical gelelectrophoresis of Quickchanges to check whether the transformation of <B>E. coli</B> Xl1 Blue with QuickChanges products of P100 (TEV, QC with O36/O37: P111, P112, P113) and P102 (EreB, QC with O16/O17: P106, P107, P108, P109, P110) were successful
</P><P><B>Procedure:</B></P><P>
Mastermix for analytical digestion of P106, P107, P108, P109, P110, P111, P112, P113 with AgeI-HF and XbaI:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2.5 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>2.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>148 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>= 175 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,5 µl Mastermix with 2,5 µl Plasmid
</LI></UL><P>
Mastermix for analytical digestion of P111, P112, P113 with XbaI and SpeI and P106, P107, P108, P109, P110 with XbaI and PstI-HF:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>148 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>= 172,5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,25 µl Mastermix with 2,5 µl Plasmid and 0,25 µl SpeI or PstI respectively
</LI></UL><P>
Mastermix for analytical digestion of  with P106, P107, P108, P109, P110 EcoRI and SpeI:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1,2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1,5 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1,5 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>88.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>= 105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,5 µl Mastermix with 2,5 µl Plasmid
</LI></UL><P>
Mastermix for analytical digestion of  with P106, P107, P108, P109, P110 XbaI and SpeI:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1,2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1,5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1,5 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>88.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>= 105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,5 µl Mastermix with 2,5 µl Plasmid
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>P111 XbaI + AgeI</B></TD><TD><B>P112 XbaI + AgeI</B></TD><TD><B>P113 XbaI + AgeI</B></TD><TD><B>P111 XbaI + SpeI</B></TD><TD><B>P112 XbaI + SpeI</B></TD><TD><B>P113 XbaI + SpeI</B></TD></TR><TR><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>QC succesful
</TD><TD>QC failed
</TD><TD>QC succesful
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>P106 XbaI + AgeI</B></TD><TD><B>P107 XbaI + AgeI</B></TD><TD><B>P108 XbaI + AgeI</B></TD><TD><B>P109 XbaI + AgeI</B></TD><TD><B>P110 XbaI + AgeI</B></TD><TD><B>P106 XbaI + PstI</B></TD><TD><B>P107 XbaI + PstI</B></TD><TD><B>P108 XbaI + PstI</B></TD><TD><B>P109 XbaI + PstI</B></TD><TD><B>P110 XbaI + PstI</B></TD></TR><TR><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>PstI inside as should be; useless experiment
</TD><TD>PstI inside as should be; useless experiment
</TD><TD>PstI inside as should be; useless experiment
</TD><TD>PstI inside as should be; useless experiment
</TD><TD>PstI inside as should be; useless experiment
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>P106 XbaI + SpeI</B></TD><TD><B>P107 XbaI + SpeI</B></TD><TD><B>P108 XbaI + SpeI</B></TD><TD><B>P109 XbaI + SpeI</B></TD><TD><B>P110 XbaI + SpeI</B></TD><TD><B>P106 EcoRI + SpeI</B></TD><TD><B>P107 EcoRI + SpeI</B></TD><TD><B>P108 EcoRI + SpeI</B></TD><TD><B>P109 EcoRI + SpeI</B></TD><TD><B>P110 EcoRI + SpeI</B></TD></TR><TR><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>Ctrl
</TD><TD>QC succesful
</TD><TD>QC succesful
</TD><TD>QC succesful
</TD><TD>QC succesful
</TD><TD>QC failed
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_23rd"><B>Thursday, May 23rd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P111.2C_P113_and_P109_.28.3F.29"> Sequencing of P111, P113 and P109 (?) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Sequencing of P111, P113 and P109 to check whether the QuickChange brought further unwanted mutations.
</P><P><B>Procedure:</B></P></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_P111_.28TEV_S219V.2C_SpeI-less.29_to_generate_Split-TEV"> PCR of P111 (TEV S219V, SpeI-less) to generate Split-TEV </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR of P111 (TEV S219V, SpeI-less) to generate Split-TEV. To generate N-TEV, the primer O38 and O39 were used. To generate C-TEV, the primer O40 and O41 were used.
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (O38 (N_Split_TEV_fw) for generating N-TEV; O40 (C_Split_TEV_fw) for generating C-TEV)
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (O39 (N_Split_TEV_rv) for generating N-TEV; O41 (C_Split_TEV_rv) for generating C-TEV)
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P111)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme with following conditions:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>51 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>30 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> Resulting product was labeled as F24 (N-TEV) and F25 (C-TEV).
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F24_and_F25_.28N-TEV_and_C-TEV.29"> Analytical gelelectrophoresis of F24 and F25 (N-TEV and C-TEV) </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F24 and F25 (N-TEV and C-TEV) to check whether PCR has worked.
</P><P><B>Procedure:</B></P><UL><LI> 4 µl of F24 and F25 were mixed with seperately with 0.44 µl of DNA loading buffer (10x) and analytical gelelectrophoresis was performed at 1% agarose gel for 60 min at 90 V.
</LI></UL><UL><LI> PCR products has expected length, but there are unspecific products. So we have to be cautious and have to check if we ligate the right products.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_PCR_products_.28N-TEV_and_C-TEV.29_of_P111_.28TEV_S219V.2C_SpeI-less.29_with_XbaI_.26_AgeI"> Preparative digestion of PCR products (N-TEV and C-TEV) of P111 (TEV S219V, SpeI-less) with XbaI &amp; AgeI </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Preparative digestion of PCR products (N-TEV and C-TEV) of P111 (TEV S219V, SpeI-less) with XbaI &amp; AgeI to clone it into pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR products (N-TEV and C-TEV) of P111 (TEV S219V, SpeI-less) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR product F24/F25
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 150 min at 37 °C.
</LI></UL><UL><LI> Digested products were purified with QIAquick PCR Purification Kit, QIAGEN.
</LI></UL><UL><LI> The digested fragments were labelled F26 (digestion of F24 with XbaI&amp;AgeI) and F27 (digestion of F25 with XbaI&amp;AgeI)
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F17.2BF26_.28N-TEV_in_pSB1C3.29_and_F17.2BF27_.28C-TEV_in_pSB1C3.29"> Ligation of F17+F26 (N-TEV in pSB1C3) and F17+F27 (C-TEV in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Ligation of F17+F26 (N-TEV in pSB1C3) and F17+F27 (C-TEV in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F17+F26:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.63 µl
</TD><TD>F17 (61.4 ng/µl, 2086 bp)
</TD></TR><TR><TD>3.20 µl
</TD><TD>F26 (15.9 ng/µl, 354 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.17 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F17+F27 (positive control)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.63 µl
</TD><TD>F17 (61.4 ng/µl, 2086 bp)
</TD></TR><TR><TD>2.41 µl
</TD><TD>F27 (21.1 ng/µl, 354 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.96 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Lid temperature = 37 °C
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E..C2.A0coli_XL1_blue_with_the_first_Quickchange_.28SDMI.29_Product_of_P28">  Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with the first Quickchange (SDMI) Product of P28 </SPAN></H3><P><B>Investigator: Jeff, Andi, Rosario, Johanna </B></P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with the first Quickchange Product of P28. 8 clones were picked the day before. 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_analytical_gelelectrophoresis_of_P28.2C_P117.2C_P118.2C_P119.2C_P120.2C_P121.2C_P122.2C_P123"> Analytical digestion and analytical gelelectrophoresis of P28, P117, P118, P119, P120, P121, P122, P123 </SPAN></H3><P><B>Investigator:</B> Rosario, Andi, Johanna, Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and analytical gelelectrophoresis of P28 (promoter SV40) and quickchange products P117, P118, P119, P120, P121, P122, P123 (all QC - SDMI -  of Bpul Laccase to remove forbidden AgeI).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P28, P117, P118, P119, P120, P121, P122, P123 AgeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2.5 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>150.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=175 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed two times for each ligation product at 90 V for 60 min.
</LI></UL><P><B>Results:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>Digestion of P28</B></TD><TD><B>Digestion of P116</B></TD><TD><B>Digestion of P117</B></TD><TD><B>Digestion of P118</B></TD><TD><B>Digestion of P119</B></TD><TD><B>Digestion of P120</B></TD><TD><B>Digestion of P121</B></TD><TD><B>Digestion of P122</B></TD><TD><B>Digestion of P123</B></TD></TR><TR><TD>as expected
</TD><TD>QC successful
</TD><TD>QC successful
</TD><TD>QC successful
</TD><TD>QC successful
</TD><TD>QC successful
</TD><TD>QC successful
</TD><TD>QC successful
</TD><TD>QC successful
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMII_-_of_P28_.28Laccase.29.2C_P106_.28EreB.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMII - of P28 (Laccase), P106 (EreB) </SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Johanna, Rosario
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction sites from P28 (Laccase), P106 (EreB).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch - P106</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2 µl
</TD><TD>Plasmid template (P106 - 1:10 dilution)
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O18 (for P106, 10 µM)
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O19 (for P106, 10 µM)
</TD></TR><TR><TD>38.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>1 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P106</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>18
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL><P><B>Reaction batch - P116 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2 µl
</TD><TD>Plasmid template - 1:5 dilution
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O28 for P116
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O29 for P116
</TD></TR><TR><TD>38.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>1 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P116</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>18
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_the_Quickchange_products_with_P28.2C_P100_and_P102_into_E..C2.A0coli_XL1_blue_3">  Transformation of the Quickchange products with P28, P100 and P102 into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Johanna, Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with Quickchangeproduct SDM II of P106 and P116.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_24th"><B>Friday, May 24th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_ligation_products_F17.2BF26_.28N-TEV_S219V_in_pSB1C3.29_and_F17.2BF27_.28C-TEV_S219V_in_pSB1C3.29"> Transformation of <I>E. coli</I> XL1 blue with ligation products F17+F26 (N-TEV S219V in pSB1C3) and F17+F27 (C-TEV S219V in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff, Florian
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with ligation products F17+F26 (N-TEV S219V in pSB1C3) and F17+F27 (C-TEV S219V in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl<SUB>2</SUB> competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on a new chlorampenicol plates.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMII_-_of_P116_.28Laccase.29.2C_P109_.28EreB.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMII - of P116 (Laccase), P109 (EreB) </SPAN></H3><P><B>Investigator:</B> Volker, Louise, Katrin, Flo
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction sites from P116 (Laccase), P109 (EreB).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch - P109</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P109 - 1:10 dilution)
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O18 (for P109, 10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O19 (for P109, 10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P109</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>21
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min 50 s
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL><P><B>Reaction batch - P116 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template - 1:5 dilution
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O28 for P116
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O29 for P116
</TD></TR><TR><TD>18.0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P116</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>21
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min 50 s
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_the_Quickchange_products_into_E..C2.A0coli_XL1_blue">  Transformation of the Quickchange products into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Volker
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with Quickchangeproduct SDM II of P109 and P116.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_May_25th"><B>Saturday, May 25th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_QuickChange_products_of_P116_and_P109_to_check_whether_the_QuickChange_PCR_and_DpnI_digestion_were_successful"> Analytical gelelectrophoresis of QuickChange products of P116 and P109 to check whether the QuickChange PCR and DpnI digestion were successful  </SPAN></H3><P><B>Investigator: Volker, Louise, Katrin, Flo</B></P><P><B>Aim of the experiment:  Analytical gelelectrophoresis of QuickChange products of P116 and P109 to check whether the QuickChange PCR and DpnI digestion were successful</B></P><P><B>Procedure:</B></P><UL><LI> 5 µl of the PCR Quickchange products were either mixed with with 0.5 µl of DNA loading buffer (10x) and analytical gelelectrophoresis was performed at 1% agarose gel for 60 min at 90 V.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_QuickChange_products_of_P116_.28Laccase.2C_QC_with_O28.2FO29.29_and_P109_.28EreB.2C_QC_with_O18.2FO19.29"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QuickChange products of P116 (Laccase, QC with O28/O29) and P109 (EreB, QC with O18/O19)</SPAN></H3><P><B>Investigator:</B> Volker
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QuickChange products of P116 (Laccase, QC with O28/O29) and P109 (EreB, QC with O18/O19).
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL><UL><LI> 5 colonies for P116QC (Laccase, QC with O28/O29) and 6 colonies for P109QC (EreB, QC with O18/O19) were picked from the plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1_blue_with_ligation_products_F17.2BF26_.28N-TEV_S219V_in_pSB1C3.29_and_F17.2BF27_.28C-TEV_S219V_in_pSB1C3.29"> Picking of <I>E. coli</I> XL1 blue with ligation products F17+F26 (N-TEV S219V in pSB1C3) and F17+F27 (C-TEV S219V in pSB1C3)</SPAN></H3><P><B>Investigator:</B> Volker
</P><P><B>Aim of the study:</B> Picking of <I>E. coli</I> XL1 blue with ligation products F17+F26 (N-TEV S219V in pSB1C3) and F17+F27 (C-TEV S219V in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (CamR LB-medium)
</LI></UL><UL><LI> 5 colonies were picked from plates for <I>E. coli</I>s XL1 blue transformed with ligation products F17+F26 (N-TEV S219V in pSB1C3) and F17+F27 (C-TEV S219V in pSB1C3).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_May_26th"><B>Sunday, May 26th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_QuickChange_products_of_P116_.28Laccase.2C_QC_with_O28.2FO29.29_and_P109_.28EreB.2C_QC_with_O18.2FO19.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with QuickChange products of P116 (Laccase, QC with O28/O29) and P109 (EreB, QC with O18/O19) </SPAN></H3><P><B>Investigator:</B> Katrin, Louise
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with QuickChange products of P116 (Laccase, QC with O28/O29) and P109 (EreB, QC with O18/O19).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_to_check_whether_the_transformation_of_E..C2.A0coli_Xl1_Blue_with_QuickChanges_products_of_P116_.28Laccase.2C_QC_with_O28.2FO29.29_and_P109_.28EreB.2C_QC_with_O18.2FO19.29_were_successful"> Analytical digestion and gelelectrophoresis to check whether the transformation of <I>E. coli</I> Xl1 Blue with QuickChanges products of P116 (Laccase, QC with O28/O29) and P109 (EreB, QC with O18/O19) were successful </SPAN></H3><P><B>Investigator:</B> Katrin, Louise
</P><P><B>Aim of the experiment:</B>Analytical digestion and gelelectrophoresis to check whether the transformation of <B>E. coli</B> Xl1 Blue with QuickChanges products of P116 (Laccase, QC with O28/O29) and P109 (EreB, QC with O18/O19)
</P><P><B>Procedure:</B></P><UL><LI> analytical digestion of Quickchange products of P109 (EreB, QC with O18/O19)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>NEB CutSmart Buffer (10x)
</TD></TR><TR><TD>0,25  µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>2,5 µl
</TD><TD>DNA from Miniprep
</TD></TR><TR><TD>15,25 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> in order to check if the QC was successful, P109 (before QC) was also digested
</LI></UL><UL><LI> analytical digestion of Quickchange products of P116 (Laccase, QC with O28/O29)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>NEB CutSmart Buffer (10x)
</TD></TR><TR><TD>0,25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0,25  µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>2,5 µl
</TD><TD>DNA from Miniprep
</TD></TR><TR><TD>15  µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> in order to check if the QC was successful, P116 (before QC) was also digested
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B>#
</TD><TD><B>P116 (Laccase before QCII) digested with EcoRI-HF, NgoMIV</B></TD><TD><B>Laccase after QCII (P128) digested with EcoRI-HF, NgoMIV</B></TD><TD><B>Laccase after QCII (P129) digested with EcoRI-HF, NgoMIV</B></TD><TD><B>Laccase after QCII (P130) digested with EcoRI-HF, NgoMIV</B></TD><TD><B>Laccase after QCII (P131) digested with EcoRI-HF, NgoMIV</B></TD><TD><B>Laccase after QCII (P124) digested with EcoRI-HF, NgoMIV</B></TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><P>Laccase P124 was chosen for sequencing
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD><B>EreB after QCII (P125) digested with PstI</B></TD><TD><B>EreB after QCII (P132) digested with PstI</B></TD><TD><B>EreB after QCII (P133) digested with PstI</B></TD><TD><B>EreB after QCII (P134) digested with PstI</B></TD><TD><B>EreB after QCII (P135) digested with PstI</B></TD><TD><B>EreB after QCII (P136) digested with PstI</B></TD><TD><B>EreB before QCII (P109) digested with PstI</B></TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><P>EreB P125 was chosen for sequencing
</P></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_of_N-TEV_.28F17.2BF26.29_and_C-TEV_.28F17.2BF25.29_with_pSB1C3">  Miniprep of ligation products of N-TEV (F17+F26) and C-TEV (F17+F25) with pSB1C3  </SPAN></H3><P><B>Investigator:</B> Katrin, Louise
</P><P><B>Aim of the experiment:</B> Miniprep of ligation products of N-TEV (F17+F26) and C-TEV (F17+F25) with pSB1C3 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_ligation_products_F17_.2B_F26.2C_F17_.2B_F27"> Analytical digestion and gelelectrophoresis of ligation products F17 + F26, F17 + F27</SPAN></H3><P><B>Investigator:</B> Katrin, Louise
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis to check whether the ligation of N-TEV (F17+F26) and C-TEV (F17+F25) in pSB1C3 were successful
</P><P><B>Procedure:</B></P><UL><LI> analytical digestion of ligation products of N-TEV (F17+F26) and C-TEV (F17+F25)in pSB1C3
</LI></UL><P>Mastermix:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>22 µl
</TD><TD>NEB Buffer 4 (10x)
</TD></TR><TR><TD>2,2  µl
</TD><TD>BSA (100X)
</TD></TR><TR><TD>2,75 µl
</TD><TD>XbaI
</TD></TR><TR><TD>2,75 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>162,8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=192,5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 2,5 µl of miniprep products were added to 17,5  µl Mastermix
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B>#
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD><B>N-TEV ligated into pSB1C3 (F17 + F26) (P126) digested with XbaI, AgeI-HF</B></TD><TD><B>N-TEV ligated into pSB1C3 (F17 + F26) (P137) digested with XbaI, AgeI-HF</B></TD><TD><B>N-TEV ligated into pSB1C3 (F17 + F26) (P138) digested with XbaI, AgeI-HF</B></TD><TD><B>N-TEV ligated into pSB1C3 (F17 + F26) (P139) digested with XbaI, AgeI-HF</B></TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><P>N-TEV ligated into pSB1C3 (F17 + F26) (P126) was chosen for sequencing
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD><B>C-TEV ligated into pSB1C3 (F17 + F27) (P127) digested with XbaI, AgeI-HF</B></TD><TD><B>C-TEV ligated into pSB1C3 (F17 + F27) (P140) digested with XbaI, AgeI-HF</B></TD><TD><B>C-TEV ligated into pSB1C3 (F17 + F27) (P141) digested with XbaI, AgeI-HF</B></TD><TD><B>C-TEV ligated into pSB1C3 (F17 + F27) (P142) digested with XbaI, AgeI-HF</B></TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD><TD>belongs to other experiment
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><P>C-TEV ligated into pSB1C3 (F17 + F27) (P127) was chosen for sequencing
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P124-P127">  Sequencing of P124-P127 </SPAN></H3><P><B>Investigators: Louise, Katrin</B></P><P><B>Aim of the experiment:</B> Sequencing of QCII products (EreB, Laccase) and ligation product of split-TEV
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> p124
</TD><TD> Miniprep of transformation of QC product of P116 (Laccase, QC with O28/O29)(QC - SDMII -  of Bpul Laccase to remove forbidden NgoMIV)
</TD><TD> FR01002337 (VR)
</TD><TD> FR01002338 (VF2)
</TD></TR><TR><TD> p125
</TD><TD> Miniprep of transformation of QC product of P109 (EreB, QC with O18/O19) (QC - SDMII -  of EreB to remove forbidden PstI)
</TD><TD> FR01002339 (VR)
</TD><TD> FR01002340 (VF2)
</TD></TR><TR><TD> p126
</TD><TD> Miniprep of ligation product F17 + F26 (N-TEV)
</TD><TD> FR01002341 (VR)
</TD><TD> FR01002342 (VF2)
</TD></TR><TR><TD> p127
</TD><TD> Miniprep of ligation product F17 + F27 (C-TEV)
</TD><TD> FR01002343 (VR)
</TD><TD> FR01002344 (VF2)
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_6">Week 6</SPAN></H1><DIV class="week" id="WWeek_6"><H2><SPAN class="mw-headline" id="Monday.2C_May_27th"><B>Monday, May 27th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Oligohybridization_of_MiniMCS_.28O56.2CO57.29_and_Spytag_.28O54.2CO55.29">  Oligohybridization of MiniMCS (O56,O57) and Spytag (O54,O55) </SPAN></H3><P><B>Investigator: Johanna</B></P><P><B>Aim of the experiment: </B>Oligohybridization of MiniMCS (MiniMCSII_fw (O56) and MiniMCSII_rv (O57) and of Spytag (SpyTag_fw (O54) and SpyTag_rv (O55))
</P><P><B>Operational sequence</B></P><UL><LI>25 µL of 100 pmol/µl of  MiniMCS fw  and 25 µL of 100 pmol/µl  MiniMCS rev in one Eppi. 25 µL of 100 pmol/µl of  Spytag fw  and 25 µL of 100 pmol/µl Spytag rev in a second Eppi
</LI></UL><UL><LI>Heating up to 95 °C for 5 min 
</LI></UL><UL><LI>Cooling at RT in a styropor box overnight
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_npt-Kasette_.28P115.29.2C_PIF3_.28P51.29.2C_pActin_.28P114.29_and_t35S_.28P114.29"> PCR of npt-Kasette (P115), PIF3 (P51), pActin (P114) and t35S (P114) </SPAN></H3><P><B>Investigator: Johanna</B></P><P><B>Aim of the experiment:</B>  PCR of npt-Kasette (P115), PIF3 (P51), pActin (P114) and t35S (P114).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction GC Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P114:O62 (pActin_for); P114:O60 (t35S_for); P115:O58 (npt-Kasette_for); P51:O66 (PIF3_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P114:O63 (pActin_rev); P114:O61 (t35S_rev); P115:O59 (npt-Kasette_rev); P51:O67 (PIF3_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P114; P115; P51)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>49 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>100 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><P><B>Results</B>:
</P><UL><LI> PCR did not work properly because primer concentration was wrong.
</LI></UL></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P45_.28FluA-Psb1C3.29_with_XbaI_.26_AgeI.2C_P39_with_AgeI_.26_PstI.2C_P20_with_NgoMIV_.26_PstI"> Preparative digestion and gelelectrophoresis of P45 (FluA-Psb1C3) with XbaI &amp; AgeI, P39 with AgeI &amp; PstI, P20 with NgoMIV &amp; PstI </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Preparative digestion of P45 (FluA-Psb1C3) with XbaI &amp; AgeI, P39 with AgeI &amp; PstI, P20 with NgoMIV &amp; PstI to clone fragment of P45 into F17 (pSB1C3) and  to fusion Middle linker of P39 with GFP of P20. 
</P><P><B>Procedure:</B></P><P>Batch for preparative digestion of  P45 (FluA-Psb1C3) with XbaI &amp; AgeIwith XbaI &amp; AgeI.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P45
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI
</TD></TR><TR><TD>11.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Batch for preparative digestion of  P39 (Middle-Linker) with PstI and AgeI.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P39
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>2 µl
</TD><TD>PstI
</TD></TR><TR><TD>11.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Batch for preparative digestion of  P20 (GFP-Psb1C3) with NgoMIV and PstI.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P45
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>2 µl
</TD><TD>PstI
</TD></TR><TR><TD>11.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 120 min at 37 °C.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1.5 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P20 digested with NgoMIV&amp;PstI (=F30)
</TD><TD>P20 digested with NgoMIV&amp;PstI (=F30)
</TD><TD>P39 digested with AgeI&amp;PstI (=F29)
</TD><TD>P39 digested with AgeI&amp;PstI (=F29)
</TD><TD>P45 digested with XbaI&amp;AgeI (=F28)
</TD><TD>P45 digested with XbaI&amp;AgeI (=F28)
</TD></TR><TR><TD>lower band was cut out
</TD><TD>lower band was cut out
</TD><TD>band was cut out
</TD><TD>band was cut out
</TD><TD>lower band was cut out
</TD><TD>lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIII_-_of_P124.28Laccase.29.2C_P125_.28EreB.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMIII - of P124(Laccase), P125 (EreB) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction sites from P124 (Laccase), P125 (EreB).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch - P124</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P124 - 1:6 dilution)
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O30 (for P125, 10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O31 (for P125, 10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch - P125</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P125 - 1:6 dilution)
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O20 (for P125, 10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O21 (for P125, 10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters for both batches</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>21
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min 50 s
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_ordered_primers_3">  Preparation of ordered primers </SPAN></H3><P><B>Investigators: Andi</B></P><P><B>Aim of the experiment:</B> Dissolving of primer pallets to provide a concentration of 100 pmol/µl
</P><P><B>Procedure:</B> We added an amount of water which gave us a final primer concentration of 100 pmol/µl. 
</P><P>Primers were labeled as follows:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Oligonr.
</TH></TR><TR><TD> O54
</TD><TD> Spytag_fw
</TD></TR><TR><TD> O55
</TD><TD> Spytag_rev
</TD></TR><TR><TD> O56
</TD><TD> MiniMCSII_fw
</TD></TR><TR><TD> O57
</TD><TD> MiniMCSII_rev
</TD></TR><TR><TD> O58
</TD><TD> Npt_for
</TD></TR><TR><TD> O59
</TD><TD> Npt_rev
</TD></TR><TR><TD> O60
</TD><TD> t35S_for
</TD></TR><TR><TD> O61
</TD><TD> t35S_rev
</TD></TR><TR><TD> O62
</TD><TD> pACT_for
</TD></TR><TR><TD> O63
</TD><TD> pACT_rev
</TD></TR><TR><TD> O64
</TD><TD> NucA_for
</TD></TR><TR><TD> O65
</TD><TD> NucA_rev
</TD></TR><TR><TD> O66
</TD><TD> Pif3_for
</TD></TR><TR><TD> O67
</TD><TD> Pif3_rev
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_P45.2C_P51.2C_P100_and_P102_into_E..C2.A0coli_XL1_blue">  Transformation of P45, P51, P100 and P102 into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P45, P51, P102 and P100.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_P114.2C_P115"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P114, P115</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P114, P115.
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (AmpR LB-medium)
</LI></UL><UL><LI> 5 colonies of P114 and P115 were picked from the plates.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F29.2BF30_.28GFP_in_pSB1C3_after_middle_linker.2C_.28_Gly-Gly-Ser-Gly.29x2.29_and_F17.2BF27_.28FluA_in_pSB1C3.29"> Ligation of F29+F30 (GFP in pSB1C3 after middle linker, ( Gly-Gly-Ser-Gly)x2) and F17+F27 (FluA in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F29+F30 (GFP in pSB1C3 after middle linker, ( Gly-Gly-Ser-Gly)x2) and F17+F27 (FluA in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F29+F30:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.22 µl
</TD><TD>F29 (23.7 ng/µl, 2110 bp)
</TD></TR><TR><TD>10.48 µl
</TD><TD>F30 (9.5 ng/µl, ~700 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>2.3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F17+F28:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.63 µl
</TD><TD>F17 (61.4 ng/µl, 2086 bp)
</TD></TR><TR><TD>7.8 µl
</TD><TD>F28 (9.6 ng/µl, 522 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>7.57 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Lid temperature = 37 °C
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_May_28th"><B>Tuesday, May 28th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_PCR_products_of_P114_.28O62.26O63.29.2C_P114_.28O60.26O61.29.2C_P115_.28O58.26O59.29.2C_P51_.28O66.26O67.29"> Analytical gelelectrophoresis of PCR products of P114 (O62&amp;O63), P114 (O60&amp;O61), P115 (O58&amp;O59), P51 (O66&amp;O67)  </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of PCR products of P114 (O62&amp;O63), P114 (O60&amp;O61), P115 (O58&amp;O59), P51 (O66&amp;O67).
</P><P><B>Procedure:</B></P><UL><LI> 4 µl of PCR products of P114 (O62&amp;O63), P114 (O60&amp;O61), P115 (O58&amp;O59), P51 (O66&amp;O67) were mixed with seperately with 0.44 µl of DNA loading buffer (10x) and analytical gelelectrophoresis was performed at 1% agarose gel for 40 min at 90 V.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>PCR products of P114 (O62&amp;O63), P114 (O60&amp;O61), P115 (O58&amp;O59), P51 (O66&amp;O67)
</TD><TD>PCR products of P114 (O60&amp;O61)
</TD><TD>PCR products of P115 (O58&amp;O59)
</TD><TD>PCR products of P51 (O66&amp;O67)
</TD></TR><TR><TD>PCR did work, but strong primer dimer signal
</TD><TD>PCR did work, but strong primer dimer signal
</TD><TD>PCR did not work
</TD><TD>PCR did work, but strong primer dimer signal
</TD></TR></TBODY></TABLE><UL><LI> Wrong buffer (GC buffer, instead of standard reaction buffer, was used and 10x too much primers were also used)
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_Quickchangeproduct_SDMIII_of_P124_.28Laccase.29_and_P125_.28EreB.29_.2B_ligation_product_of_F29.2BF30_.2B_ligation_product_of_F17.2BF19_into_E..C2.A0coli_XL1_blue">  Transformation of Quickchangeproduct SDMIII of P124 (Laccase) and P125 (EreB) + ligation product of F29+F30 + ligation product of F17+F19 into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Transformation of Quickchangeproduct SDMIII of P124 (Laccase) and P125 (EreB) + ligation product of F29+F30 + ligation product of F17+F19 + ligation product of F17+NK + ligation product of F29+NK into <I>E. coli</I> XL1 blue.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Sequencing_of_P133_and_P134">  Sequencing of P133 and P134 </SPAN></H3><P><B>Investigators: Louise, Johanna</B></P><P><B>Aim of the experiment:</B> Sequencing of two more QCII products (EreB) 
<B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> p133
</TD><TD> Miniprep of transformation of QC product of P109 (EreB, QC with O18/O19) (QC - SDMII -  of EreB to remove forbidden PstI)
</TD><TD> FR01002287 (fw)
</TD><TD> FR01002296 (rev)
</TD></TR><TR><TD> p134
</TD><TD> Miniprep of transformation of QC product of P109 (EreB, QC with O18/O19) (QC - SDMII -  of EreB to remove forbidden PstI)
</TD><TD> FR01002295 (fw)
</TD><TD> FR01002294 (rev)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_of_P114_and_P115">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of P114 and P115 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of P114 and P115.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_P45.2C_P51.2C_P100.2C_P102"> Picking of of <I>E. coli</I> XL1 blue transformed with P45, P51, P100, P102 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with P45, P51, P100, P102.
</P><P><B>Procedure:</B></P><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x) (P51, P100, P102)/4 µL Ampicillin (1000x) (P45).
</LI></UL><UL><LI> 1 colony for each plasmid were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_npt-Kasette_.28P144.29.2C_PIF3_.28P51.29.2C_pActin_.28P143.29_and_t35S_.28P143.29"> PCR of npt-Kasette (P144), PIF3 (P51), pActin (P143) and t35S (P143) </SPAN></H3><P><B>Investigator:</B> Rosario, Jeff
</P><P><B>Aim of the experiment:</B>  PCR of npt-Kasette (P144), PIF3 (P51), pActin (P143) and t35S (P143).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P143:O62 (pActin_for); P143:O60 (t35S_for); P144:O58 (npt-Kasette_for); P51:O66 (PIF3_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P143:O63 (pActin_rev); P143:O61 (t35S_rev); P144:O59 (npt-Kasette_rev); P51:O67 (PIF3_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P114; P115; P51)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>47 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>120 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F33.2C_F34.2C_F35.2C_F36"> Analytical gelelectrophoresis of F33, F34, F35, F36 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F33, F34, F35, F36.
</P><P><B>Procedure:</B></P><UL><LI> 4 µl of F33, F34, F35 and F36 were mixed with seperately with 0.44 µl of DNA loading buffer (10x) and analytical gelelectrophoresis was performed at 1% agarose gel for 40 min at 90 V.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>F33
</TD><TD>F33
</TD><TD>F33
</TD><TD>F33
</TD></TR><TR><TD>PCR did not work, maybe high GC content?
</TD><TD>PCR products has expected length, some byproducts
</TD><TD>PCR products has expected length, some byproducts
</TD><TD>PCR products has expected length
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_May_29th"><B>Wednesday, May 29th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_SDMIII_product_of_P124.2C_ligation_product_of_F29.2BF30_and_ligation_product_of_F17.2BF18">  Transformation of <I>E. coli</I> XL1 blue with SDMIII product of P124, ligation product of F29+F30 and ligation product of F17+F18</SPAN></H3><P><B>Investigator: Andi</B></P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with SDMIII product of P124, ligation product of F29+F30 and ligation product of F17+F18.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_of_P100.2C_P102.2C_P51.2C_P45">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of P100, P102, P51, P45 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of P100, P102, P51, P45 .
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_ligation_product_of_F17.2BF28"> Picking of of <I>E. coli</I> XL1 blue transformed with ligation product of F17+F28</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with ligation product of F17+F28.
</P><P><B>Procedure:</B></P><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x) (
</LI></UL><UL><LI> 3 colonies were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIII_-_of_P133_.28EreB.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMIII - of P133 (EreB)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction site from P133 (EreB).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P133 - 1:10 dilution)
</TD><TD>need to get to 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O20 (for P133, 10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O21 (for P133, 10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P133</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>3 min 30 s
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the reaction batch and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_the_Quickchange_product_into_E..C2.A0coli_XL1_blue">  Transformation of the Quickchange product into <I>E. coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with Quickchangeproduct SDM III of P133.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_t35s.28F34.29_with_XbaI_.26_PstI.2C_npt.28F25.29_with_EcoRI_.26_SpeI.2C_PIF3.28F36.29_with_XbaI_.26_AgeI-HF_and_P10_with_XbaI_.26_PstI_as_well_as_with_EcoRI_.26_SpeI">  Preparative digestion of t35s(F34) with XbaI &amp; PstI, npt(F25) with EcoRI &amp; SpeI, PIF3(F36) with XbaI &amp; AgeI-HF and P10 with XbaI &amp; PstI as well as with EcoRI &amp; SpeI</SPAN></H3><P><B>Investigator: Louise </B></P><P><B>Aim of the experiment:</B> Preparative digestion of t35s(F34) with XbaI &amp; PstI, npt(F25) with EcoRI &amp; SpeI, PIF3(F36) with XbaI &amp; AgeI-HF and P10 with XbaI &amp; PstI as well as with EcoRI &amp; SpeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of t35s(F34) with XbaI and PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA P61
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of npt(F35) with EcoRI and SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA P61
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of PIF3(F36) with XbaI and AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA P61
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P10 with XbaI &amp; PstI for backbone isolation.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P96
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P10 with EcoRI and SpeI for backbone isolation.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P96
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> The resulting fragments were named: ?????
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches and 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis. 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 90 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>Digestion of PCR product t35s (F34) with XbaI &amp; PstI = F38
</TD><TD>Digestion of PCR product npt casette (F25) with EcoRI &amp; SpeI = F39
</TD><TD>Digestion of PCR product PIF3 (F36) with XbaI &amp; AgeI-HF = F40
</TD></TR><TR><TD>as expected, lower band was cut out
</TD><TD>as expected, bright band was cut out
</TD><TD>as expected, bright band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>Digestion of P10 with EcoRI &amp; SpeI = F41
</TD><TD>Digestion of P10 with XbaI &amp; PstI = F42
</TD></TR><TR><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_pSB1C3_containing_a_NLS_from_SV40_.28BBa_K801030.29">  Transformation of <I>E. coli</I> XL1 blue with pSB1C3 containing a NLS from SV40 (BBa_K801030) </SPAN></H3><P><B>Investigator:</B> Florian, Jeff
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with pSB1C3 containing a NLS from SV40 (BBa_K801030).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_pActin_.28P143.29"> PCR of pActin (P143) </SPAN></H3><P><B>Investigator:</B> Florian, Jeff
</P><P><B>Aim of the experiment:</B>  PCR of pActin (P143).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P143:O62 (pActin_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P143:O63 (pActin_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>51 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>80 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><P><B>Results</B>:
</P><UL><LI> After analytical gelelectrophoresis there was no product visible. PCR did't work!
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F17.2BF40_.28PIF3_in_pSB1C3.28RFC25.29.29.2C_F42.2BF38_.28t35S_in_pSB1C3.28RFC10.29.29_and_F41.2BF39.28npt-casette_in_pSB1C3.28RFC10.29.29"> Ligation of F17+F40 (PIF3 in pSB1C3(RFC25)), F42+F38 (t35S in pSB1C3(RFC10)) and F41+F39(npt-casette in pSB1C3(RFC10)) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F17+F40 (PIF3 in pSB1C3(RFC25)), F42+F38 (t35S in pSB1C3(RFC10)) and F41+F39(npt-casette in pSB1C3(RFC10)).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F17+F40:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.63 µl
</TD><TD>F17 (61.4 ng/µl, 2086 bp)
</TD></TR><TR><TD>1.88 µl
</TD><TD>F40 (22.8 ng/µl, ~297 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.49 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F42+F38:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.36 µl
</TD><TD>F42 (73.5 ng/µl, 2070 bp)
</TD></TR><TR><TD>3.42 µl
</TD><TD>F38 (9.2 ng/µl, 217 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.22 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F41+F39:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.11 µl
</TD><TD>F41 (89.9 ng/µl, 2070 bp)
</TD></TR><TR><TD>13.74 µl
</TD><TD>F39 (16.0 ng/µl, 1517 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>2.15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Lid temperature = 37 °C
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_May_30th"><B>Thursday, May 30th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Gradient_PCR_of_pActin_.28P143.29"> Gradient PCR of pActin (P143) </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B>  PCR of pActin  
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O62 (pActin_for)
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O63 (pActin_rev)
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA P143
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq GC Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O62 (pActin_for)
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O63 (pActin_rev)
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA P143
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>47 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>120 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>A temperature gradient was applied beginning at 47°C and ranging to 51°C. For each integer (47, 48, 49, 50 and 51°C) a single PCR batch was used.
</P><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><P><B>Analytical gelelectrophoresis</B><B>Procedure:</B></P><UL><LI> 4 µl of each PCR product were mixed seperately with 0.44 µl of DNA loading buffer (10x) and analytical gelelectrophoresis was performed at 1% agarose gel for 60 min at 90 V.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>F45 GC buffer
</TD><TD>F46 GC buffer
</TD><TD>F47 GC buffer
</TD><TD>F47 GC buffer
</TD><TD>F49 GC buffer
</TD><TD>normal buffer
</TD><TD>normal buffer
</TD><TD>normal buffer
</TD><TD>normal buffer
</TD><TD>normal buffer
</TD><TD>1 kbp ladder
</TD></TR><TR><TD>PCR products has expected length
</TD><TD>PCR products has expected length
</TD><TD>PCR products has expected length
</TD><TD>PCR products has expected length
</TD><TD>PCR products has expected length
</TD><TD>PCR products has expected length, some byproducts
</TD><TD>PCR products has expected length, some byproducts
</TD><TD>PCR products has expected length, some byproducts
</TD><TD>PCR products has expected length, some byproducts
</TD><TD>PCR products has expected length, some byproducts
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_of_F17.2BF28_.28FluA_in_pSB1C3_RFC25.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of F17+F28 (FluA in pSB1C3 RFC25) </SPAN></H3><P><B>Investigator:</B> Jeff, Flo
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of F17+F28 (FluA in pSB1C3 RFC25).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_ligation_products_F17.2BF28_.28FluA_in_pSB1C3_RFC25.29"> Analytical digestion and gelelectrophoresis of ligation products F17+F28 (FluA in pSB1C3 RFC25) </SPAN></H3><P><B>Investigator:</B> Flo, Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of ligation products F17+F28 (FluA in pSB1C3 RFC25).
</P><P><B>Procedure:</B></P><UL><LI> Analytical digestion of ligation products of F17+F28 (FluA in pSB1C3 RFC25) with XbaI &amp; AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl P150/P151/P152/P153
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B>#
</TD><TD>Digestion of P150 with XbaI&amp;AgeI-HF
</TD><TD>Digestion of P151 with XbaI&amp;AgeI-HF
</TD><TD>Digestion of P152 with XbaI&amp;AgeI-HF
</TD><TD>Digestion of P153 with XbaI&amp;AgeI-HF
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_oligohybridization_products_F31_.26_F32"> Preparative gelelectrophoresis of oligohybridization products F31 &amp; F32 </SPAN></H3><P><B>Investigator:</B> Flo, Jeff
</P><P><B>Aim of the experiment:</B> Preparative gelelectrophoresis of oligohybridization products F31 &amp; F32.
</P><P><B>Procedure:</B></P><UL><LI> 50 µl of the 1:10 dilution of F31 and F32 were mixed with 5 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed 1.5 h on a 2% agarose gel at 90 V.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>F31
</TD><TD>F32
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL><UL><LI> Gelpurified F31 and F32 were labelled as F43 and F44.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F17.2BF40_.28PIF3_in_pSB1C3.28RFC25.29.29.2C_F42.2BF38_.28t35S_in_pSB1C3.28RFC10.29.29_and_F41.2BF39.28npt-casette_in_pSB1C3.28RFC10.29.29">  Transformation of <I>E. coli</I> XL1 blue with F17+F40 (PIF3 in pSB1C3(RFC25)), F42+F38 (t35S in pSB1C3(RFC10)) and F41+F39(npt-casette in pSB1C3(RFC10)) </SPAN></H3><P><B>Investigator:</B> Florian, Jeff
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with F17+F40 (PIF3 in pSB1C3(RFC25)), F42+F38 (t35S in pSB1C3(RFC10)) and F41+F39(npt-casette in pSB1C3(RFC10)).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on a new chlorampenicol plates.
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F42.2BF43_and_F17.2BF32"> Ligation of F42+F43 and F17+F32 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Ligation of F42+F43 and F17+F32.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F42+F43:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.36 µl
</TD><TD>F42 (73.5 ng/µl, 2070 bp)
</TD></TR><TR><TD>1.51 µl
</TD><TD>F43 (5.4 ng/µl, 34 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>14.13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F17+F32:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.63 µl
</TD><TD>F17 (61.4 ng/µl, 2086 bp)
</TD></TR><TR><TD>0.25 µl
</TD><TD>F32 (2510.8 ng/µl, 49 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>15.12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Lid temperature&amp;</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_EreB_SDMIII_.28P149.29_and_BBa_K801030_.28SV40_NLS.29"> Picking of of <I>E. coli</I> XL1 blue transformed with EreB SDMIII (P149) and BBa_K801030 (SV40 NLS) </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with EreB SDMIII (P149) and BBa_K801030 (SV40 NLS).
</P><P><B>Procedure:</B></P><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 5 colonies of EreB SDMIII (P149) and 2 colonies of BBa_K801030 (SV40 NLS) were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_May_31st"><B>Friday, May 31st</B></SPAN></H2><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_P149_.28QCIII_of_EreB.29_and_BBa_K801030_.28SV40_NLS.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P149 (QCIII of EreB) and BBa_K801030 (SV40 NLS) </SPAN></H3><P><B>Investigator:</B> Louise, Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P149 (QCIII of EreB) and BBa_K801030 (SV40 NLS) 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P149_.28QCIII_of_EreB.29_with_EcoRI-HF_and_PstI-HF"> Analytical digestion and gelelectrophoresis of P149 (QCIII of EreB) with EcoRI-HF and PstI-HF </SPAN></H3><P><B>Investigator:</B> Louise, Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P154-P158 (QCIII of EreB) with EcoRI-HF and PstI-HF to check if QCII was successful
</P><P><B>Procedure:</B></P><UL><LI> Analytical digestion of P154-P158 (QCIII of EreB) with EcoRI-HF and PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P149-P158/P133 (control before QCIII)
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B>#
</TD><TD>Digestion of P133 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P154 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P155 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P156 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P157 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P158 with EcoRI-HF and PstI-HF
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P157_and_P160">  Sequencing of P157 and P160 </SPAN></H3><P><B>Investigators:</B> Louise, Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Sequencing of QCIII product (EreB) and BBa_K801030 (SV40 NLS) 
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P157
</TD><TD> Miniprep of transformation of QCIII product of P149 (QC - SDMIII -  of EreB to remove forbidden PstI)
</TD><TD> FR01002289 (VR)
</TD><TD> FR01002290 (VF2)
</TD></TR><TR><TD> P160
</TD><TD> Miniprep of transformation of BBa_K801030 (SV40 NLS)
</TD><TD> FR01002291 (VR)
</TD><TD> FR01002292 (VF2)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F29.2BF30"> Ligation of F29+F30</SPAN></H3><P><B>Investigator:</B>  Katrin
</P><P><B>Aim of the experiment:</B> Ligation of F29+F30
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.22 µl
</TD><TD>F29 (23.7 ng/µl, 2110 bp))
</TD></TR><TR><TD>10.48 µl
</TD><TD>F30 (9,5 ng/µl, about 700 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>2.3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>
No negative control was prepared because there was not enough vector backbone left
Ligation was performed at 37 °C overnight
</P></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIII_-_of_P124_.28Laccase.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMIII - of P124 (Laccase)</SPAN></H3><P><B>Investigator:</B> Louise, Katrin
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction site from P124 (Laccase).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P124 - 1:10 dilution)
</TD><TD>need to get between 5 and 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O30 (for P124, 10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O31 (for P124, 10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P124</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><P>After the PCR; 1 µl of Dpn1 was added and the reaction mixture was incubated at 37°C for one hour
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_pActin.28F46.29_with_XbaI_.26_PstI">  Preparative digestion of pActin(F46) with XbaI &amp; PstI</SPAN></H3><P><B>Investigator: Ingmar </B></P><P><B>Aim of the experiment:</B> Preparative digestion of pActin(F46) in order to ligat this fragment in pSB1C3 later on.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of pActint(F46) with XbaI and PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>F46 PCR product of pActin
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batchesand loaded on a 1% agarose gel for preparative gelelectrophoresis. 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 90 min. 
</LI></UL><UL><LI>The band was extracted by QIAquick Gel Extraction Kit, QIAGEN. The resulting fragment was named: F50
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F42.2BF50_.28pActin_in_pSB1C3.29"> Ligation of F42+F50 (pActin in pSB1C3)</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Ligation of F42+F50 (pActin in pSB1C3(RFC25).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.37 µl
</TD><TD>F42
</TD></TR><TR><TD>13.13 µl
</TD><TD>F50
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>2.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Lid temperature = 37 °C
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_F42.2BF38_.28t35S.29.2C_F17.2BF40_.28Pif_3.29_and_F41.2BF39_.28npt.29"> Picking of of <I>E. coli</I> XL1 blue transformed with F42+F38 (t35S), F17+F40 (Pif 3) and F41+F39 (npt)</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with F42+F38 (t35S), F17+F40 (Pif 3) and F41+F39 (npt).
</P><P><B>Procedure:</B></P><UL><LI> The picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover containing 5 mL of LB-medium + 5 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 3 colonies of each agar plate were picked. 
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_ligation_products_F42.2BF43_and_F17.2BF32">  Transformation of <I>E. coli</I> XL1 blue with ligation products F42+F43 and F17+F32 </SPAN></H3><P><B>Investigator:</B> Louise, Jeff
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with ligation products F42+F43 and F17+F32.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_1st"><B>Saturday, June 1st</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_ligation_products_F29.2BF30_.28GFP_in_pSB1C3.29.2C_F42.2BF50_.28actin_promoter_in_pSB1C3.29.2C_F42.2BF50_.28negative_control.29">  Transformation of <I>E. coli</I> XL1 blue with ligation products F29+F30 (GFP in pSB1C3), F42+F50 (actin promoter in pSB1C3), F42+F50 (negative control) </SPAN></H3><P><B>Investigator:</B>  Katrin
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with ligation products F29+F30 (GFP in pSB1C3), F42+F50 (actin promoter in pSB1C3), F42+F50 (negative control)
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on a new chlorampenicol plates.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_QCIII_product_of_Laccase_.28SDMIII_of_P124.29">  Transformation of <I>E. coli</I> XL1 blue with QCIII product of Laccase (SDMIII of P124)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with QCIII product of Laccase
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on a new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_F39_.2B_F41_.28npt-casette_in_pSC1C3.29.2CF42.2BF38_.28t35s_in_pSB1C3.29.2C_F17.2BF40_.28PIF3_in_pSB1C3.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with F39 + F41 (npt-casette in pSC1C3),F42+F38 (t35s in pSB1C3), F17+F40 (PIF3 in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with F39 + F41 (npt-casette in pSC1C3),F42+F38 (t35s in pSB1C3), F17+F40 (PIF3 in pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P161-163_.28F39_.2B_F41.2C_npt-casette_in_pSB1C3.29.2C_P164-166_.28F42.2BF38.2C_t35s_in_pSB1C3.29.2C_P167-169_.28F17.2BF40.2C_PIF3_in_pSB1C3.29"> Analytical digestion and gelelectrophoresis  of P161-163 (F39 + F41, npt-casette in pSB1C3), P164-166 (F42+F38, t35s in pSB1C3), P167-169 (F17+F40, PIF3 in pSB1C3)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P161-163 (F39 + F41, npt-casette in pSC1C3)  with EcoRI-HF and SpeI-HF, P164-166 (F42+F38, t35s in pSB1C3) with EcoRI-HF and SpeI-HF and P167-169 (F17+F40, PIF3 in pSB1C3) with AgeI-HF and XbaI
</P><P><B>Procedure:</B></P><UL><LI> Analytical digestion of P161-163 (F39 + F41, npt-casette in pSC1C3), P164-166 (F42+F38, t35s in pSB1C3) with EcoRI-HF and SpeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P161-P166
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><UL><LI> Analytical digestion of P167-169 (F17+F40, PIF3 in pSB1C3) with AgeI-HF and XbaI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P167-P169
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B>#
</TD><TD>Digestion of P161 with EcoRI-HF and SpeI-HF
</TD><TD>Digestion of P162 with EcoRI-HF and SpeI-HF
</TD><TD>Digestion of P163 with EcoRI-HF and SpeI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and SpeI-HF
</TD><TD>Digestion of P165 with EcoRI-HF and SpeI-HF
</TD><TD>Digestion of P166 with EcoRI-HF and SpeI-HF
</TD><TD>Digestion of P167 with AgeI-HF and XbaI
</TD><TD>Digestion of P168 with AgeI-HF and XbaI
</TD><TD>Digestion of P169 with AgeI-HF and XbaI
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_2nd"><B>Sunday, June 2nd</B></SPAN></H2><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_QCIII_product_of_P124_and_the_ligation_products_of_F17.2BF44_and_F42.2BF32"> Picking of of <I>E. coli</I> XL1 blue transformed with QCIII product of P124 and the ligation products of F17+F44 and F42+F32</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with QCIII product of P124 and the ligation products of F17+F32 and F42+F32.
</P><P><B>Procedure:</B></P><UL><LI> The picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover containing 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 4 colonies of each agar plate were picked. 
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight.
</LI></UL></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H1><DIV class="week" id="WWeek_7"><H2><SPAN class="mw-headline" id="Monday.2C_June_3rd"><B>Monday, June 3rd</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P170-P173_.28F17.2BF32.2C_SpyTag_in_pSB1C3_RFC25.29.2C_P174-P177_.28F42.2BF43.2C_miniMCSII_in_pSB1C3_RFC10.29.2C_P178-181_.28QCIII_von_P142.2C_Laccase_in_pSB1C3_RFC10.29"> Analytical digestion and gelelectrophoresis  of P170-P173 (F17+F32, SpyTag in pSB1C3 RFC25), P174-P177 (F42+F43, miniMCSII in pSB1C3 RFC10), P178-181 (QCIII von P142, Laccase in pSB1C3 RFC10) </SPAN></H3><P><B>Investigator:</B> Andi, Johanna, Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P170-P173 (F17+F32, SpyTag in pSB1C3 RFC25), P174-P177 (F42+F43, miniMCSII in pSB1C3 RFC10), P178-181 (QCIII von P142, Laccase in pSB1C3 RFC10).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P170-P173 (F17+F32, SpyTag in pSB1C3 RFC25), P174-P177 (F42+F43, miniMCSII in pSB1C3 RFC10) with XbaI+PstI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>18 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>2.25 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>135 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=157.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P170-P177)
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><UL><LI> Analytical digestion of P178-181 (QCIII von P142, Laccase in pSB1C3 RFC10):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P178-P181
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>Digestion of P170 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P162 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P163 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and PstI-HF
</TD><TD>Digestion of P164 with EcoRI-HF and PstI-HF
</TD></TR><TR><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD><TD>as expected, insert too small to be visible
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>Digestion of P142 with NgoMIV and PstI-HF
</TD><TD>Digestion of P178 with NgoMIV and PstI-HF
</TD><TD>Digestion of P179 with NgoMIV and PstI-HF
</TD><TD>Digestion of P180 with NgoMIV and PstI-HF
</TD><TD>Digestion of P181 with NgoMIV and PstI-HF
</TD></TR><TR><TD>Ctrl (before QuickChange III)
</TD><TD>as expected, plasmid is only linearized, all NgoMIV sites are removed
</TD><TD>as expected, plasmid is only linearized, all NgoMIV sites are removed
</TD><TD>as expected, plasmid is only linearized, all NgoMIV sites are removed
</TD><TD>as expected, plasmid is only linearized, all NgoMIV sites are removed
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_4th"><B>Tuesday, June 4th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P162.28npt-cassete.29.2C_P165.28t35s.29.2C_P168.28PIF3.29.2C_P179.28LaccQCIII.29.2C_P177.28MiniMCS.29.2C_P170.28Spytag.29">  Sequencing of P162(npt-cassete), P165(t35s), P168(PIF3), P179(LaccQCIII), P177(MiniMCS), P170(Spytag) </SPAN></H3><P><B>Investigators:</B> Louise, Johanna
</P><P><B>Aim of the experiment:</B> Sequencing of P162(npt-cassete), P165(t35s), P168(PIF3), P179(LaccQCIII), P177(MiniMCS), P170(Spytag). 
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P162
</TD><TD> Miniprep of ligation product F39 + F41 (npt-casette in pSB1C3)
</TD><TD> FR01002331 (VR)
</TD><TD> FR01002330 (VF2)
</TD></TR><TR><TD> P165
</TD><TD> Miniprep of ligation product F42+F38 (t35s in pSB1C3)
</TD><TD> FR01002332 (VF2)
</TD></TR><TR><TD> P168
</TD><TD> Miniprep of ligation product F17+F40 (PIF3 in pSB1C3)
</TD><TD> FR01002333 (VF2)
</TD></TR><TR><TD> P179
</TD><TD> Miniprep of ligation product Laccase QCIII (P142)
</TD><TD> FR01002335 (VR)
</TD><TD> FR01002334 (VF2)
</TD></TR><TR><TD> P177
</TD><TD> Miniprep of ligation product MiniMCSII F42+F43
</TD><TD> FR01002336 (VF2)
</TD></TR><TR><TD> P170
</TD><TD> Miniprep of ligation product F17+F32 (SpyTag)
</TD><TD> FR01002328 (VF2)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P39.28middle-linker.29_with_PstI_and_AgeI-HF.2C_P20.28GFP-Psb1C3.29_with_NgoMIV_and_PstI.2C_F45.28pActin.29_with_XbaI_and_PstI">  Preparative digestion and gelelectrophoresis of P39(middle-linker) with PstI and AgeI-HF, P20(GFP-Psb1C3) with NgoMIV and PstI, F45(pActin) with XbaI and PstI </SPAN></H3><P><B>Investigator:</B> Louise, Johanna
</P><P><B>Aim of the experiment:</B> Preparative digestion of P39(middle-linker) with PstI and AgeI-HF, P20(GFP-Psb1C3) with NgoMIV and PstI, F45(pActin) with XbaI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P39(middle-linker) with PstI and AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P39
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P20(GFP-Psb1C3) with NgoMIV and PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P20
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of F45(pActin) with XbaI and PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Fragment DNA F45
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches and 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis. 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 90 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>Digestion of P20 with NgoMIV &amp; PstI = F53
</TD><TD>Digestion of P39 with AgeI &amp; PstI = F52
</TD><TD>Digestion of F45 with XbaI &amp; PstI = F51
</TD></TR><TR><TD>as expected, lower band was cut out
</TD><TD>as expected, band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIV_-_of_P179_.28Laccase.29">  Quick Change mutagenesis to remove forbidden restriction sites - SDMIV - of P179 (Laccase)</SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction site SDM IV from P179 (Laccase).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P179 - 1:6 dilution)
</TD><TD>need to get between 5 and 50 ng/µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O32 (for P179, 10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O33 (for P179, 10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P179</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><P>After the PCR; 1 µl of Dpn1 was added and the reaction mixture was incubated at 37°C for one hour
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P182_.28Gensynthesis_1.29_with_XbaI_and_AgeI-HF.2C_P183_.28Gensynthesis_2.29_with_XbaI_and_AgeI-HF">  Preparative digestion and gelelectrophoresis of P182 (Gensynthesis 1) with XbaI and AgeI-HF, P183 (Gensynthesis 2) with XbaI and AgeI-HF </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P182 (Gensynthese 1) with XbaI and AgeI-HF, P183 (Gensynthese 2) with XbaI and AgeI-HF 
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P182 with XbaI and AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P182
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P183 with XbaI and AgeI- HF 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P183
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on a 1.5% agarose gel for preparative gelelectrophoresis. 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V for 60 min. 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>Digestion of Gensynthesis 1 with XbaI &amp; AgeI
</TD><TD>100 bp DNA ladder
</TD><TD>Digestion of Gensynthesis 2 with XbaI &amp; AgeI
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>as expected, fragments (78 bp and 530 bp) were cut out
</TD><TD>as expected, fragments (111 bp, 206 bp and 359 bp) were cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P9_with_XbaI_and_AgeI-HF">  Preparative digestion and gelelectrophoresis of P9 with XbaI and AgeI-HF </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P9 with XbaI and AgeI-HF 
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P9 with XbaI and AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>Plasmid DNA P9
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> The resulting fragments were named: ?????
</LI></UL><UL><LI>4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis. 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V for 60 min.
</LI></UL><UL><LI> Gelfragmentation was like expected, both bands were cut out (upper band = PhyB, lower band = pSB1C3 RFC25)
</LI></UL><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_Quickchange_product_SDM_IV_of_P179_.28Laccase.29">  Transformation of <I>E. coli</I> XL1 blue with Quickchange product SDM IV of P179 (Laccase) </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with Quickchange product SDM IV of P179 (Laccase).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2x 10 µl of DNA was added seperately into 2x 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P39">  Transformation of <I>E. coli</I> XL1 blue with P39 </SPAN></H3><P><B>Investigator:</B> Louise, Johanna
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with P39
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added seperately into 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on new chlorampenicol plates.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P39_with_AgeI_.26_PstI.2C_P20_with_NgoMIV_.26_PstI"> Preparative digestion and gelelectrophoresis of P39 with AgeI &amp; PstI, P20 with NgoMIV &amp; PstI </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Where is the snapshot of the gel =DDDD</DIV></DIV></DIV><P><B>Investigator:</B> Johanna, Louise 
</P><P><B>Aim of the experiment:</B>Preparative digestion of P39 with AgeI &amp; PstI, P20 with NgoMIV &amp; PstI to fusion Middle linker of P39 with GFP of P20. 
</P><P><B>Procedure:</B></P><P>Batch for preparative digestion of  P39 (Middle-Linker) with PstI and AgeI.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P39
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>2 µl
</TD><TD>PstI
</TD></TR><TR><TD>11.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Batch for preparative digestion of  P20 (GFP-Psb1C3) with NgoMIV and PstI.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P45
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>2 µl
</TD><TD>PstI
</TD></TR><TR><TD>11.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 150 min at 37 °C.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><P><A href="/File:FOTOFOTO.png" title="File:FOTOFOTO.png">File:FOTOFOTO.png</A></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P20 digested with NgoMIV&amp;PstI (=F30)
</TD><TD>P39 digested with NgoMIV&amp;PstI (=F30)
</TD></TR><TR><TD>lower band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F58.2BF59.2C_F55.2BF59.2C_F54.2BF59.2C_F56.2BF59.2C_F57.2BF59.2C_F52.2BF53.2C_F42.2BF51"> Ligation of F58+F59, F55+F59, F54+F59, F56+F59, F57+F59, F52+F53, F42+F51</SPAN></H3><P><B>Investigator:</B> Jeff, Andi, Johanna, Louise
</P><P><B>Aim of the experiment:</B> Ligation of F53+F59, F58+F59, F55+F59, F54+F59, F56+F59, F57+F59, F52+F53, F42+F51.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F54+F59 :
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.2 µl
</TD><TD>F54 (5.9 ng/µl, 90 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F55+F59:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>11.6 µl
</TD><TD>F55 (6.4 ng/µl, 520 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>3.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F56+F59:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.3 µl
</TD><TD>F56 (4.7 ng/µl, 110 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.2 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F57+F59:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.1 µl
</TD><TD>F57 (9.1 ng/µl, 200 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.4 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F58+F59:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.4 µl
</TD><TD>F58 (12.9 ng/µl, 400 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11.1 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative Control Ligation batch for F59:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>15.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F52+F53:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>F52 (9.9 ng/µl, XXX bp)
</TD></TR><TR><TD>7 µl
</TD><TD>F53 (12.2 ng/µl, XXX bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F42+F51:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.36 µl
</TD><TD>F42 (XXX ng/µl, XXX bp)
</TD></TR><TR><TD>15.64 µl
</TD><TD>F51 (8.2 ng/µl, 1237 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>99 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Lid temperature = 37 °C
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Plating_of_received_E..C2.A0coli_containing_biobricks_2"> Plating of received <I>E. coli</I> containing biobricks </SPAN></H3><P><B>Investigator: Jeff, Andreas</B></P><P><B>Aim of the experiment:</B> The received biobricks were already transformed in <I>E. coli</I> and were in an agar stabs. These <I>E. coli</I> cells were transferred with an inoculation loop on antibiotic selection plates and were incubated over night.
</P><P><B>Operational sequence:</B></P><UL><LI> Bacterias containing plasmids with biobricks were transferred with a sterile inoculation loop on antibiotic plates and were incubated at 37 °C overnight.
</LI></UL><UL><LI> The biobricks were:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Name
</TH><TH> Function
</TH></TR><TR><TD>BBa_J61032
</TD><TD>Alkaline Phosphatase
</TD></TR><TR><TD>BBa_K426020
</TD><TD>Self lysis device
</TD></TR><TR><TD>BBa_K729004
</TD><TD>Nuclease from S. Aureus
</TD></TR><TR><TD>BBa_K365001
</TD><TD>PIF6
</TD></TR><TR><TD>BBa_K648011
</TD><TD>RBS B0034 with cI repressor C0051 under the control of constitutive promoter J23113
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_5th"><B>Wednesday, June 5th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1_blue_transformed_with_F59.2BF54.2C_F59.2BF55.2C_F59.2BF56.2C_F59.2BF57.2C_F59.2BF58.2C_F42.2BF51_and_F52.2BF53"> Transformation of E. coli XL1 blue transformed with F59+F54, F59+F55, F59+F56, F59+F57, F59+F58, F42+F51 and F52+F53  </SPAN></H3><P><B>Investigator:</B> Florian, Rosario, Louise, Jeff
</P><P><B>Aim of the experiment:</B> Transformation of E. coli XL1 blue transformed with F59+F54, F59+F55, F59+F56, F59+F57, F59+F58, F42+F51 and F52+F53.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on plates containing antibiotics.
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_BBa_K365001_.28PIF6.29.2C_BBa_K729004_.28Nuclease_from_Staphylococcus_aureus.29.2C_BBa_K648011_.28XylE_RFC25.29.2C_BBa_J61032_.28Alkaline_phosphotase.29.2C_BBa_K426020_.28BBa_K426020.29"> Picking of of <I>E. coli</I> XL1 blue transformed with BBa_K365001 (PIF6), BBa_K729004 (Nuclease from Staphylococcus aureus), BBa_K648011 (XylE RFC25), BBa_J61032 (Alkaline phosphotase), BBa_K426020 (BBa_K426020) </SPAN></H3><P><B>Investigator:</B> Rosario, Florian, Louise, Jeff
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with BBa_K365001 (PIF6), BBa_K729004 (Nuclease from Staphylococcus aureus), BBa_K648011 (XylE RFC25), BBa_J61032 (Alkaline phosphotase), BBa_K426020 (BBa_K426020).
</P><P><B>Procedure:</B></P><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x) or ampicillin(1000x).
</LI></UL><UL><LI> 2 colonies of each plate were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_6th"><B>Thursday, June 6th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_P39.2C_P20.2C_genesynthesis_1.262_and_BBa_J61032.2C_BBa_K426020.2C_BBa_K729004.2C_BBa_K365001.2C_BBa_K648011">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P39, P20, genesynthesis 1&amp;2 and BBa_J61032, BBa_K426020, BBa_K729004, BBa_K365001, BBa_K648011 </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B>Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P39, P20, genesynthesis 1&amp;2 and BBa_J61032, BBa_K426020, BBa_K729004, BBa_K365001, BBa_K648011
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> P39</TD><TD>P184
</TD></TR><TR><TD> P39</TD><TD>P185
</TD></TR><TR><TD> P20</TD><TD>P186
</TD></TR><TR><TD> P20</TD><TD>P187
</TD></TR><TR><TD> BBa_J61032</TD><TD>P188
</TD></TR><TR><TD> BBa_J61032</TD><TD>P189
</TD></TR><TR><TD> BBa_K426020</TD><TD>P190
</TD></TR><TR><TD> BBa_K426020</TD><TD>P191
</TD></TR><TR><TD> BBa_K729004</TD><TD>P192
</TD></TR><TR><TD> BBa_K729004</TD><TD>P193
</TD></TR><TR><TD> BBa_K365001</TD><TD>P194
</TD></TR><TR><TD> BBa_K365001</TD><TD>P195
</TD></TR><TR><TD> BBa_K648011</TD><TD>P196
</TD></TR><TR><TD> BBa_K648011</TD><TD>P197
</TD></TR><TR><TD> Genesynthesis 2</TD><TD>P198
</TD></TR><TR><TD> Genesynthesis 2</TD><TD>P199
</TD></TR><TR><TD> Genesynthesis 1</TD><TD>P200
</TD></TR><TR><TD> Genesynthesis 1</TD><TD>P201
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P188-P197_.28latest_received_biobricks:_BBa_J61032.2C_BBa_K426020.2C_BBa_K729004.2C_BBa_K365001.2C_BBa_K648011.29"> Analytical digestion and gelelectrophoresis of P188-P197 (latest received biobricks: BBa_J61032, BBa_K426020, BBa_K729004, BBa_K365001, BBa_K648011) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P188-P197 (latest received biobricks: BBa_J61032, BBa_K426020, BBa_K729004, BBa_K365001, BBa_K648011).
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Mastermix (12x)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>24 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>3 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>03 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>180 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=210 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>17.5 µl Mastermix + 2.5 µl Plasmid each
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kb Marker
</TD><TD>P188 Alk.Phos.
</TD><TD>P189 Alk.Phos.
</TD><TD>P190 self lysis device
</TD><TD>P191 self lysis device
</TD><TD>P192 Nuclease
</TD><TD>P193 Nuclease
</TD><TD>P194 PIF6
</TD><TD>P195 PIF6
</TD><TD>P196 RBS
</TD><TD>P197 RBS
</TD></TR><TR><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_the_ligation_products_of_F52.2BF53.2C_F59.2BF58.2C_F59.2BF54.2C_F59.2BF55.2C_F59.2BF57_and_F59.2BF56"> Picking of of <I>E. coli</I> XL1 blue transformed with the ligation products of F52+F53, F59+F58, F59+F54, F59+F55, F59+F57 and F59+F56 </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with the ligation products of F52+F53, F59+F58, F59+F54, F59+F55, F59+F57 and F59+F56
</P><P><B>Procedure:</B></P><UL><LI> The picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover containing 4 mL of LB-medium + 4 µL chloramphenicol(1000x) or ampecillin (1000x).
</LI></UL><UL><LI> 2 colonies of each agar plate were picked. 
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overday.
</LI><LI>The ligation of F42 and F51 was not successful and has to be repeated. One reason for the failure could be secondary structures of the promoter. Therefore the new ligation batch should be heated to 95°C for a while before adding the T4 ligase.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_P192_.28Thermonuclease.2C_O64_.26_O65.29_and_P194_.28PIF6.2C_O42_.26_O43.29"> PCR of P192 (Thermonuclease, O64 &amp; O65) and P194 (PIF6, O42 &amp; O43) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  PCR of P192 (Thermonuclease, O64 &amp; O65) and P194 (PIF6, O42 &amp; O43).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P192: O64 (NucA_fw); P194: O42 (PIF6_fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P192: O65 (NucA_rv); P194: O43 (PIF6_rv))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>1:1000 dilution of plasmid DNA (P192; P194)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The gradient PCR program was performed after following scheme with following conditions:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>30 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> PCR product of P192 = F61; PCR product of 194 = F62
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="QuikChange_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIV_-_of_P179_.28Laccase.29">  QuikChange mutagenesis to remove forbidden restriction sites - SDMIV - of P179 (Laccase)</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment: </B> Removal of forbidden restiction site from P179 (Laccase).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid template (P179 - 1:10 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O32 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O33 (10 µM)
</TD></TR><TR><TD>18.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P179</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><P>After the PCR; 1 µl of Dpn1 was added and the reaction mixture was incubated at 37°C for one hour
</P></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F61_.28PCR_product_of_P192.2C_Thermonuclease.29.2C_F62_.28PCR_product_of_P194.2C_PIF6.29_and_the_DpnI_digestion_of_the_SDMIV_product_of_P179"> Analytical gelelectrophoresis of F61 (PCR product of P192, Thermonuclease), F62 (PCR product of P194, PIF6) and the DpnI digestion of the SDMIV product of P179 </SPAN></H3><P><B>Investigator:</B> Rosario, Andi, Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F61 (PCR product of P192, Thermonuclease), F62 (PCR product of P194, PIF6) and the DpnI digestion of the SDMIV product of P179.
</P><P><B>Procedure:</B></P><UL><LI> 4 µl of F61 or F62 or the DpnI digestion of the SDMIV product of P179 were mixed with 0.444 µl of DNA loading buffer (10x)
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h on a 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>PCR product F61
</TD><TD>PCR product F62
</TD><TD>DpnI digestion of QCIV of P179
</TD></TR><TR><TD>successful
</TD><TD>successful
</TD><TD>successful
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F42.2BF51"> Ligation of F42+F51</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Ligation of F42+F51.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F42+F51:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.68 µl
</TD><TD>F42 (73.5 ng/µl, 2070 bp)
</TD></TR><TR><TD>15.64 µl
</TD><TD>F51 (10.93 ng/µl, 1237 bp)
</TD></TR><TR><TD>5.39 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI><LI>The reaction batch was  heated up to 95°C for 5 min before the enzyme and buffer were added.
</LI><LI> The ligation was performed for two hours at room temperature.
</LI></UL><UL><LI> Lid temperature = 37 °C
</LI></UL></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1_blue_with_F42.2BF51_.28pActin_in_pSB1C3_RFC10.29_and_DpnI_digestion_of_the_SDMIV_product_of_P179_.28Laccase.29"> Transformation of <I>E. coli</I> XL1 blue with F42+F51 (pActin in pSB1C3 RFC10) and DpnI digestion of the SDMIV product of P179 (Laccase)  </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F42+F51 (pActin in pSB1C3 RFC10 )and DpnI digestion of the SDMIV product of P179 (Laccase).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on plates containing antibiotics (CamR).
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_Biobricks_P188.2F189_.28alk.Phos..2C_1475.C2.A0bp.29.2C_P190.2F191_.28self_lysis_device.2C_2910.C2.A0bp.29.2C_P192.2F193_.28Nuclease.2C_561.C2.A0bp.29.2C_P194.2F195_.28PIF6.2C_300.C2.A0bp.29.2C_P196.2F197_.28RBS.2C_918.C2.A0bp.29">  Sequencing of Biobricks P188/189 (alk.Phos., 1475 bp), P190/191 (self lysis device, 2910 bp), P192/193 (Nuclease, 561 bp), P194/195 (PIF6, 300 bp), P196/197 (RBS, 918 bp) </SPAN></H3><P><B>Investigators:</B> Johanna
</P><P><B>Aim of the experiment:</B> Sequencing of the Biobrick MiniPreps
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM):
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Konz. ng/µl
</TH><TH> Vol. BB µl
</TH><TH> Vol H2O µl
</TH></TR><TR><TD> P188
</TD><TD> 160.6
</TD><TD> 5
</TD><TD> 10
</TD></TR><TR><TD> P189
</TD><TD> 123.6
</TD><TD> 7.5
</TD><TD> 7.5
</TD></TR><TR><TD> P190
</TD><TD> 319.2
</TD><TD> 3
</TD><TD> 12
</TD></TR><TR><TD> P191
</TD><TD> 127.0
</TD><TD> 10
</TD><TD> 5
</TD></TR><TR><TD> P192
</TD><TD> 336.9
</TD><TD> 3
</TD><TD> 12
</TD></TR><TR><TD> P193
</TD><TD> 230.6
</TD><TD> 4
</TD><TD> 11
</TD></TR><TR><TD> P194
</TD><TD> 156.6
</TD><TD> 5
</TD><TD> 10
</TD></TR><TR><TD> P195
</TD><TD> 197.4
</TD><TD> 4
</TD><TD> 11
</TD></TR><TR><TD> P196
</TD><TD> 90.3
</TD><TD> 10
</TD><TD> 5
</TD></TR><TR><TD> P197
</TD><TD> 230.2
</TD><TD> 4
</TD><TD> 11
</TD></TR></TBODY></TABLE><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P188
</TD><TD> alk.Phos.
</TD><TD> FR01002327 (O3)
</TD><TD> FR01002326 (O4)
</TD></TR><TR><TD> P189
</TD><TD> alk.Phos.
</TD><TD> FR01002325 (O3)
</TD><TD> FR01002320 (O4)
</TD></TR><TR><TD> P190
</TD><TD> self lysis device
</TD><TD> FR01002319 (O3)
</TD><TD> FR01002318 (O4)
</TD></TR><TR><TD> P191
</TD><TD> self lysis device
</TD><TD> FR01002317 (O3)
</TD><TD> FR01002316 (O4)
</TD></TR><TR><TD> P192
</TD><TD> Nuclease
</TD><TD> FR01002315 (O3)
</TD><TD> - (O4)
</TD></TR><TR><TD> P193
</TD><TD> Nuclease
</TD><TD> FR01002314 (O3)
</TD><TD> - (O4)
</TD></TR><TR><TD> P194
</TD><TD> PIF6
</TD><TD> FR01002313 (O3)
</TD><TD> - (O4)
</TD></TR><TR><TD> P195
</TD><TD> PIF6
</TD><TD> FR01002312 (O3)
</TD><TD> - (O4)
</TD></TR><TR><TD> P196
</TD><TD> RBS
</TD><TD> FR01002324 (O3)
</TD><TD> FR01002323 (O4)
</TD></TR><TR><TD> P197
</TD><TD> RBS
</TD><TD> FR01002322 (O3)
</TD><TD> FR01002321 (O4)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_F61_.28Thermonuclease.29.2C_F62_.28PIF6.29_.2C_P165_.28t35S.29.2C_F31_.28miniMCSII.29"> Preparative digestion and gelelectrophoresis of F61 (Thermonuclease), F62 (PIF6) , P165 (t35S), F31 (miniMCSII) </SPAN></H3><P><B>Investigator:</B> Rosario, Andi, Florian, Johanna, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of F61 (Thermonuclease), F62 (PIF6) , P165 (t35S), F31 (miniMCSII.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of F61 (Thermonuclease) with XbaI &amp; AgeI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR product F61
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of F62 (PIF6) with XbaI &amp; AgeI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR product F62
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P165 (t35S) with XbaI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P165
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of F31 (miniMCSII):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Oligohybridization F31
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>34 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches and 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on an 2% agarose gel for preparative gelelectrophoresis (1% agarose gel for digestion of P165). 
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>Digestion of F31 with SpeI-HF
</TD><TD>Digestion of F61 with XbaI &amp; AgeI-HF
</TD><TD>Digestion of F62 with XbaI &amp; AgeI-HF
</TD></TR><TR><TD>Fragment length as expected; band was cut out
</TD><TD>Fragment length as expected; band was cut out
</TD><TD>Fragment length as expected; band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>Digestion of P165 with XbaI
</TD></TR><TR><TD>Fragment length as expected; band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_preparative_digestion_of_P165_with_FastAP">  Dephosphorylation of preparative digestion of P165 with FastAP  </SPAN></H3><P><B>Investigator:</B> Rosario, Jeff
</P><P><B>Aim of the experiment:</B> Dephosphorylation of preparative digestion of P165 with FastAP to prevent re-ligation.
</P><P><B>Procedure:</B></P><UL><LI> After preparative digestion, gelelectrophoresis and gelextraction of P165, the digestion product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> Batch for the dephosphorylation of digestion product of P165
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>Digestion product of P165
</TD></TR><TR><TD>4 µl
</TD><TD>10X Fast AP buffer
</TD></TR><TR><TD>2 µl
</TD><TD>FastAP Thermosensitive Alkaline Phosphatase
</TD></TR><TR><TD>4 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The reaction batch was spun briefly and incubated at 37 °C for 10 min.
</LI></UL><UL><LI> The reaction was stopped by heating to 75 °C for 5 min.
</LI></UL><UL><LI> The dephosphorylized digestion product of P165 product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> The resulting product was labelled as F63
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F58.2BF59.2C_F55.2BF59.2C_F54.2BF59.2C_F56.2BF59.2C_F57.2BF59.2C_F52.2BF53.2C_F42.2BF51_2"> Ligation of F58+F59, F55+F59, F54+F59, F56+F59, F57+F59, F52+F53, F42+F51</SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Ligation of F53+F59, F58+F59, F55+F59, F54+F59, F56+F59, F57+F59, F52+F53, F42+F51.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F59+F65 (Thermonuclease in pSB1C3 RFC25):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2086 bp)
</TD></TR><TR><TD>4.56 µl
</TD><TD>F65 (14.1 ng/µl, 447 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>10.94 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F59+F66:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2086 bp)
</TD></TR><TR><TD>3.78 µl
</TD><TD>F66 (11.3 ng/µl, 297 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11.72 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F63+F64:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12.05 µl
</TD><TD>F63 (8.3 ng/µl, 2287 bp)
</TD></TR><TR><TD>4.95 µl
</TD><TD>F64 (4.3 ng/µl, 22 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_7th"><B>Friday, June 7th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_QC_IV_Laccase"> Picking of QC IV Laccase </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the study:</B> Picking of QC IV Laccase (concentrated) and QC IV Laccase (diluted)
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl CamR  and 4 ml LB-medium)
</LI></UL><UL><LI> 3 colonies were picked for every Transformation product.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_Knop-Medium_for_Moss_2">Preparation of Knop-Medium for Moss</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparation of Knop-Medium for Moss
</P><P><B>Procedure:</B></P><UL><LI> All four prepared stocks have been: 25g/L KH2PO4; 25g/L KCl; 25g/L MgSO4 x 7 H2O; 100g/L Ca(NO3)2  
</LI><LI> Volume of prepared Medium: 2.6L
</LI><LI> 10 ml of each stock were converted into a 5L Erlenmeyer flask and filled up to 2.6L with destilled water (ELGA); 32.5mg FeSO4 x 7 H2O were added; the pH was set to 5.8 with KOH/NaOH
</LI><LI> 200ml of the prepared medium was filled into three 500ml Erlenmeyer flasks, so each Erlenmeyer flask contained 200 ml of the Knop-Medium; the rest (2L) was left in the 5L EM flask
</LI><LI> All flasks have been autoclaved 
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_F52.2BF53.2C_F59.2BF54.2C_F59.2BF55.2C_F59.2BF56.2C_F59.2BF57.2C_F59.2BF58">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  with F52+F53, F59+F54, F59+F55, F59+F56, F59+F57, F59+F58</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  with F52+F53, F59+F54, F59+F55, F59+F56, F59+F57, F59+F58
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> F52+F53 I</TD><TD>P202
</TD></TR><TR><TD> F52+F53 II</TD><TD>203
</TD></TR><TR><TD> F59+F54 I</TD><TD>P204
</TD></TR><TR><TD> F59+F54 II</TD><TD>P205
</TD></TR><TR><TD> F59+F55 I</TD><TD>P206
</TD></TR><TR><TD> F59+F55 II</TD><TD>P207
</TD></TR><TR><TD> F59+F56 I</TD><TD>P208
</TD></TR><TR><TD> F59+F56 II</TD><TD>P209
</TD></TR><TR><TD> F59+F57 I</TD><TD>P210
</TD></TR><TR><TD> F59+F57 II</TD><TD>P211
</TD></TR><TR><TD> F59+F58 I</TD><TD>P212
</TD></TR><TR><TD> F59+F58 II</TD><TD>P213
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1_blue_with_F59_.2B_F65_.28Thermonuclease_in_pSB1C3.29.2C_F59_.2B_F66_.28PIF6_in_pSB1C3.29_and_F63_.2B_F64_.28MiniMCS_.2B_t35S_in_pSB1C3.29"> Transformation of <I>E. coli</I> XL1 blue with F59 + F65 (Thermonuclease in pSB1C3), F59 + F66 (PIF6 in pSB1C3) and F63 + F64 (MiniMCS + t35S in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F59 + F65 (Thermonuclease in pSB1C3), F59 + F66 (PIF6 in pSB1C3) and F63 + F64 (MiniMCS + t35S in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product were added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of each tube were plated on agarose plates containing antibiotics (CamR).
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on plates containing antibiotics (CamR).
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P204_.28Ligation_Product_F59.2BF54.29.2C_P206_.28Ligation_Product_F59.2BF55.29.2C_P208_.28Ligation_Product_F59.2BF56.29.2C_P210_.28Ligation_Product_F59.2BF57.29.2C_P212_.28Ligation_Product_F59.2BF58.29">  Sequencing of P204 (Ligation Product F59+F54), P206 (Ligation Product F59+F55), P208 (Ligation Product F59+F56), P210 (Ligation Product F59+F57), P212 (Ligation Product F59+F58) </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">A monster is born!</DIV></DIV></DIV><P><B>Investigators:</B> Louise
</P><P><B>Aim of the experiment:</B> Sequencing of Ig-Kappa (P204), Nanoluciferase (P206), SigP (P208), Transmembrand. (P210) and SpyCatcher (P212) Ligations in pSB1C3 RFC25
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode (O3)
</TH></TR><TR><TD>P204
</TD><TD>Ig-Kappa-SigP + pSB1C3 RFC25
</TD><TD>FR01002304
</TD></TR><TR><TD>P206
</TD><TD>Nanoluciferase + pSB1C3 RFC25
</TD><TD>FR01002303
</TD></TR><TR><TD>P208
</TD><TD>SERK-SigP + pSB1C3 RFC25
</TD><TD>FR01002302
</TD></TR><TR><TD>P210
</TD><TD>Transmembrand + pSB1C3 RFC25
</TD><TD>FR01002301
</TD></TR><TR><TD>P212
</TD><TD>SpyCatcher + pSB1C3 RFC25
</TD><TD>FR01002300
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P202-P213"> Analytical digestion and gelelectrophoresis  of P202-P213 </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P202-P213
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P202-P213 with XbaI+AgeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>3.75 µl
</TD><TD>XbaI
</TD></TR><TR><TD>3.75 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>225 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=262.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P202-P203)
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Inaddition, P203 was digested for 4 and for 8 seconds in the microwave, a negative control was incubated at room temperature
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>Digestion of P202 with XbaI und AgeI (GFP + linker)
</TD><TD>Digestion of P203 with XbaI und AgeI (GFP + linker)
</TD><TD>Digestion of P204 with XbaI und AgeI
</TD><TD>Digestion of P205 with XbaI und AgeI
</TD><TD>Digestion of P206 with XbaI und AgeI
</TD><TD>Digestion of P207 with XbaI und AgeI
</TD><TD>Digestion of P208 with XbaI und AgeI
</TD><TD>Digestion of P209 with XbaI und AgeI
</TD><TD>Digestion of P210 with XbaI und AgeI
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>as expected (SERK-TMD)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>1 kbp ladder DNA ladder</B></TD><TD>Digestion of P211 with XbaI und AgeI
</TD><TD>Digestion of P212 with XbaI und AgeI
</TD><TD>Digestion of P213 with XbaI und AgeI
</TD><TD>Digestion of P203 with XbaI und AgeI (microwave 4 seconds)
</TD><TD>Digestion of P203 with XbaI und AgeI (microwave 8 seconds)
</TD><TD>Digestion of P203 with XbaI und AgeI (control)
</TD></TR><TR><TD>as expected (SERK-TMD)
</TD><TD>not as expected
</TD><TD>as expected
</TD><TD>interesting
</TD><TD>interesting
</TD><TD>interesting
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_8th"><B>Saturday, June 8th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_QC_IV_Laccase">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  QC IV Laccase </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  QC IV Laccase
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI> The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> QC IV Laccase clone 1</TD><TD>P214
</TD></TR><TR><TD> QC IV Laccase clone 2</TD><TD>P215
</TD></TR><TR><TD> QC IV Laccase clone 3</TD><TD>P216
</TD></TR><TR><TD> QC IV Laccase clone 4</TD><TD>P217
</TD></TR><TR><TD> QC IV Laccase clone 5</TD><TD>P218
</TD></TR><TR><TD> QC IV Laccase clone 6</TD><TD>P219
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_F59_.2B_F65_.28Thermonuclease_in_pSB1C3.29.2C_F59_.2B_F66_.28PIF6_in_pSB1C3.29_and_F63_.2B_F64_.28MiniMCS_.2B_t35S_in_pSB1C3.29"> Picking of F59 + F65 (Thermonuclease in pSB1C3), F59 + F66 (PIF6 in pSB1C3) and F63 + F64 (MiniMCS + t35S in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the study:</B> Picking of F59 + F65 (Thermonuclease in pSB1C3), F59 + F66 (PIF6 in pSB1C3) and F63 + F64 (MiniMCS + t35S in pSB1C3)
<B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl CamR  and 4 ml LB-medium)
</LI></UL><UL><LI> 2 colonies were picked for every Transformation product.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_F47_.28pActin.2C_fragment_named_F68.29_with_XbaI.2FSpeI.2C_P202_.28GFP.2Blinker.2C_fragment_named_F69.29_with_NgoMIV.2FSpeI.2C_P210_.28SERK-TMD.2C_fragment_named_F67.29"> Preparative digestion and gelelectrophoresis of F47 (pActin, fragment named F68) with XbaI/SpeI,  P202 (GFP+linker, fragment named F69) with NgoMIV/SpeI, P210 (SERK-TMD, fragment named F67) </SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of F47 (pActin, fragment named F68) with XbaI/SpeI,  P202 (GFP+linker, fragment named F69) with NgoMIV/SpeI, P210 (SERK-TMD, fragment named F67) with AgeI/SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of F47 (pActin, fragment named F68) with XbaI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR product F47
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P202 (GFP+linker, fragment named F69) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>plasmid P202
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P210 (SERK-TMD, fragment named F67) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA P210
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of F47 (pActin, fragment named F68) with XbaI/SpeI
</TD></TR><TR><TD>Fragment length as expected; band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P202 (GFP+linker, fragment named F69) with NgoMIV/SpeI
</TD></TR><TR><TD>Fragment length as expected; band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P210 (SERK-TMD, fragment named F67) with AgeI/SpeI
</TD></TR><TR><TD>Fragment length as expected; band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI><LI> Concentration determined with Nanodrop:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>F68
</TD><TD>pActin
</TD><TD>10,4 ng/µl
</TD></TR><TR><TD>F69
</TD><TD>GFP+linker
</TD><TD>13,0 ng/µl
</TD></TR><TR><TD>F67
</TD><TD>SERK-TMD
</TD><TD>14,3 ng/µl
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P7_.28PhyB_in_pSB1C3.2C_fragment_named_F75.29_with_XbaI.2FSpeI_and_P8_.28PhyB_in_pSC1C3.2C_fragment_named_F76.29_with_NgoMIV.2FSpeI"> Preparative digestion and gelelectrophoresis of P7 (PhyB in pSB1C3, fragment named F75) with XbaI/SpeI and P8 (PhyB in pSC1C3, fragment named F76) with NgoMIV/SpeI</SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P7 (PhyB in pSB1C3, fragment named F75) with XbaI/SpeI and P8 (PhyB in pSC1C3, fragment named F76) with NgoMIV/SpeI (backbones for ligation)
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P7 (PhyB in pSB1C3, fragment named F75) with XbaI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid P7
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P8 (PhyB in pSC1C3, fragment named F76) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>plasmid P8
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> before gelelectrophoresis, P7 was dephosphorylated
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches after digestion and were loaded on a 0.5% agarose gel for preparative gelelectrophoresis
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P7 (PhyB in pSB1C3, fragment named F75) with XbaI/SpeI
</TD><TD>digestion of P8 (PhyB in pSC1C3, fragment named F76) with NgoMIV/SpeI
</TD></TR><TR><TD>Fragment lengths as expected; lower band was cut out
</TD><TD>Fragment lengths as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI><LI> Concentration determined with Nanodrop:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>F75
</TD><TD>dephosphorylated Backbone (P7)
</TD><TD>39,4 ng/µl
</TD></TR><TR><TD>F76
</TD><TD>dephosphorylated Backbone (P8)
</TD><TD>52,0 ng/µl
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_F75">  Dephosphorylation of F75  </SPAN></H3><P><B>Investigator: Jeff, Katrin </B></P><P><B>Aim of the experiment:</B> Dephosphorylation of cut vector pSB1C3 prevent re-ligation
</P><P><B>Procedure:</B></P><UL><LI> the digestion product was purified with QIAquick PCR Purification Kit, Qiagen in order to change the buffer
</LI></UL><UL><LI> Batch for the dephosphorylation of F75
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>Plasmid DNA F75 from digestion
</TD></TR><TR><TD>3 µl
</TD><TD>10X Fast AP buffer
</TD></TR><TR><TD>1.3 µl
</TD><TD>FastAP Thermosensitive Alkaline
<P>Phosphatase
</P></TD></TR><TR><TD>=34.3 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> the reaction batch was spun briefly and incubated at 37 °C for 10 minutes
</LI><LI> the reaction was stopped by heating to 75 °C for 5 minutes
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P200_.28gene_synthesis_1.2C_fragments_named_F70.2BF71.29_with_XbaI.2FAgeI_and_P198_.28gene_synthesis_2.2C_fragments_named_F72-F74.29_with_XbaI.2FAgeI"> Preparative digestion and gelelectrophoresis of P200 (gene synthesis 1, fragments named F70+F71) with XbaI/AgeI and P198 (gene synthesis 2, fragments named F72-F74) with XbaI/AgeI </SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P200 (gene synthesis 1, fragments named F70+F71) with XbaI/AgeI and P198 (gene synthesis 2, fragments named F72-F74) with XbaI/AgeI
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P200 (gene synthesis 1, fragments named F70+F71) with XbaI/AgeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid P7
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P198 (gene synthesis 2, fragments named F72-F74) with XbaI/AgeII
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid P7
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 3 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches after digestion and were loaded on a 2% agarose gel for preparative gelelectrophoresis
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P200 (gene synthesis 1, fragments named F70+F71) with XbaI/AgeI
</TD><TD>digestion of P198 (gene synthesis 2, fragments named F72-F74) with XbaI/AgeII
</TD></TR><TR><TD>Fragment lengths as expected; the two lower bands were cut out (upper:F70, lower: F71)
</TD><TD>Fragment lengths as expected; all bands were cut out (upper: F72, middle: F73, lower: F74)
</TD></TR></TBODY></TABLE><UL><LI> Bands were extracted by QIAquick Gel Extraction Kit, QIAGEN.
</LI><LI>Concentrations measured with Nano-Drop 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Fragment</B></TD><TD><B>Name</B></TD><TD><B>Length</B></TD><TD><B>Concentration</B></TD></TR><TR><TD>F70
</TD><TD>Nanoluciferase
</TD><TD>510 bp
</TD><TD>9.2 ng/µl
</TD></TR><TR><TD>F71
</TD><TD>IgKappa-SigP
</TD><TD>67 bp
</TD><TD>3.1 ng/µl
</TD></TR><TR><TD>F72
</TD><TD>SpyCatcher
</TD><TD>339 bp
</TD><TD>10.8 ng/µl
</TD></TR><TR><TD>F73
</TD><TD>SERK-TMD
</TD><TD>186 bp
</TD><TD>8.6 ng/µl
</TD></TR><TR><TD>F74
</TD><TD>SERK-SigP
</TD><TD>100 bp
</TD><TD>4.8 ng/µl
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P214-219_.28Miniprep_of_Laccase_QC_IV.29.2C_P179_.28control_before_QC_IV.29"> Analytical digestion and gelelectrophoresis  of P214-219 (Miniprep of Laccase QC IV), P179 (control before QC IV) </SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P214-219 (Miniprep of Laccase QC IV), P179 (control before QC IV)
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P214-219 (Miniprep of Laccase QC IV), P179 (control before QC IV)
</LI></UL><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P214-219, P179)
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> digestion for 4 x 10 seconds in the microwave (600 watt), with two minute breaks in between the microwaving
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P214 (Miniprep of Laccase QC IV clone 1) with NgoMVI/AgeI
</TD><TD>digestion of P215 (Miniprep of Laccase QC IV clone 2) with NgoMVI/AgeI
</TD><TD>digestion of P216 (Miniprep of Laccase QC IV clone 3) with NgoMVI/AgeI
</TD><TD>digestion of P217 (Miniprep of Laccase QC IV clone 4) with NgoMVI/AgeI
</TD><TD>digestion of P218 (Miniprep of Laccase QC IV clone 5) with NgoMVI/AgeI
</TD><TD>digestion of P219 (Miniprep of Laccase QC IV clone 6) with NgoMVI/AgeI
</TD><TD>digestion of P179 (Miniprep of ligation product Laccase QCIII clone 2) with NgoMVI/AgeI
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F75.2BF69_.28Linker.2BGFP_into_pSB1C3.29_and_F67.2BF69_.28Linker.2BGFP_into_TMD-pSB1C3-Suffix.29"> Ligation of F75+F69 (Linker+GFP into pSB1C3) and F67+F69 (Linker+GFP into TMD-pSB1C3-Suffix)</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F75+F69 (Linker+GFP into pSB1C3) and F67+F69 (Linker+GFP into TMD-pSB1C3-Suffix).
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.54 µl
</TD><TD>F75 (39.4 ng/µl, ? bp)
</TD></TR><TR><TD>8.30 µl
</TD><TD>F69 (13.0 ng/µl, ? bp)
</TD></TR><TR><TD>6.16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7.00 µl
</TD><TD>F67 (14.3 ng/µl, ? bp)
</TD></TR><TR><TD>7.62 µl
</TD><TD>F69 (13.0 ng/µl, ? bp)
</TD></TR><TR><TD>2.38 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Incubation: 1 h 22°C
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1_blue_with_F75.2BF69_.28Linker.2BGFP_into_pSB1C3.29_and_F67.2BF69_.28Linker.2BGFP_into_pSB1C3-TMD-Suffix.29"> Transformation of <I>E. coli</I> XL1 blue with F75+F69 (Linker+GFP into pSB1C3) and F67+F69 (Linker+GFP into pSB1C3-TMD-Suffix) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F75+F69(Linker+GFP into pSB1C3) and F67+F69 (Linker+GFP into pSB1C3-TMD-Suffix)
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on plates containing antibiotics (CamR).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_9th"><B>Sunday, June 9th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_of_F59.2BF65_.28Nuclease.29.2C_F59.2BF66_.28PIF6.29.2C_F59.2BF54_.28Ig-K.29.2C_F59.2BF55_.28Luciferase.29.2C_F59.2BF56_.28SERK-SigP.29.2C_F59.2BF57_.28SERK-TMD.29.2C_F59.2BF58_.28SpyCatcher.29.2C_F42.2BF51_.28pActin.29.2C_F63.2BF64">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  of F59+F65 (Nuclease), F59+F66 (PIF6), F59+F54 (Ig-K), F59+F55 (Luciferase), F59+F56 (SERK-SigP), F59+F57 (SERK-TMD), F59+F58 (SpyCatcher), F42+F51 (pActin), F63+F64 </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue of F59+F65 (Nuclease), F59+F66 (PIF6), F59+F54 (Ig-K), F59+F55 (Luciferase), F59+F56 (SERK-SigP), F59+F57 (SERK-TMD), F59+F58 (SpyCatcher), F42+F51 (pActin), F63+F64
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> F59+F65 clone 1</TD><TD>P220
</TD></TR><TR><TD> F59+F65 clone 2</TD><TD>P221
</TD></TR><TR><TD> F59+F66 clone 1</TD><TD>P222
</TD></TR><TR><TD> F59+F66 clone 2</TD><TD>P223
</TD></TR><TR><TD> F59+F56 clone 1</TD><TD>P224
</TD></TR><TR><TD> F59+F56 clone 2</TD><TD>P225
</TD></TR><TR><TD> F59+F55 clone 1</TD><TD>P226
</TD></TR><TR><TD> F59+F55 clone 2</TD><TD>P227
</TD></TR><TR><TD> F59+F57 clone 1</TD><TD>P228
</TD></TR><TR><TD> F59+F57 clone 2</TD><TD>P229
</TD></TR><TR><TD> F59+F58 clone 1</TD><TD>P230
</TD></TR><TR><TD> F59+F58 clone 2</TD><TD>P231
</TD></TR><TR><TD> F63+F64 </TD><TD>P232
</TD></TR><TR><TD> F42+F51 </TD><TD>P233
</TD></TR><TR><TD> F59+F54 clone 1</TD><TD>P234
</TD></TR><TR><TD> F59+F54 clone 2</TD><TD>P235
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P220-P235_.28.28P220-231.2C_P233-P235_with_XbaI.2FSpeI.2C_P232_with_MfeI.2FPst1.29"> Analytical digestion and gelelectrophoresis  of P220-P235 ((P220-231, P233-P235 with XbaI/SpeI, P232 with MfeI/Pst1) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P220-P235 (Minipreps)
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P220-231, P233-P235 with XbaI, SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>34 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>4.25 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>4.25 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>255 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=266.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA
</LI></UL><UL><LI> reaction batch for analytical digestion of P232 with MfeI/PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P232
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>MfeI(20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF(20 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI><LI> for P224, P225, P228, P229, P230, P231, P234, P235,a 2% agarose gel was used, gelelectrophoresis was performed at 90 V for 45 min
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>100 bp DNA ladder
</TD><TD>digestion of P224 (SERK-SigP, clone 1) with XbaI, SpeI
</TD><TD>digestion of P225 (SERK-SigP, clone 2) with XbaI, SpeI
</TD><TD>digestion of P228 (SERK TMD, clone 1) with XbaI, SpeI
</TD><TD>digestion of P229 (SERK TMD, clone 2) with XbaI, SpeI
</TD><TD>digestion of P230 (SpyCatcher, clone 1) with XbaI, SpeI
</TD><TD>digestion of P231 (SpyCatcher, clone 2) with XbaI, SpeI
</TD><TD>digestion of P234 (Ig-K-SigP, clone 1) with XbaI, SpeI
</TD><TD>digestion of P235 (Ig-K-SigP, clone 2) with XbaI, SpeI
</TD></TR><TR><TD>not as expected
</TD><TD>not as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>fragment not visible due to small size
</TD><TD> not as expected
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P220 (Thermonuclease, clone 1) with XbaI, SpeI
</TD><TD>digestion of P221 (Thermonuclease, clone 2) with XbaI, SpeI
</TD><TD>digestion of P222 (PIF6, clone 1) with XbaI, SpeI
</TD><TD>digestion of P223 (PIF6, clone 2) with XbaI, SpeI
</TD><TD>digestion of P226 (Luciferase, clone 1) with XbaI, SpeI
</TD><TD>digestion of P227 (Luciferase, clone 2) with XbaI, SpeI
</TD><TD>digestion of P232 (miniMCS) with MfeI/PstI
</TD><TD>digestion of P233 (pActin) with XbaI, SpeI
</TD></TR><TR><TD>not as expected
</TD><TD>not as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>not as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_F67.2BF69_.28SERK-TMD.2Blinker_GFP.29"> Picking of F67+F69 (SERK-TMD+linker GFP)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the study:</B> Picking of F67+F69 (SERK-TMD+linker GFP)
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (5 µl CamR  and 5 ml LB-medium)
</LI></UL><UL><LI> 3 colonies were picked for every Transformation product.
</LI></UL><UL><LI>There were no colonies on the plates for ligations F68+F75 (pActin, pSB1C3), 
</LI></UL><P>(probably due to dephosphorylation)
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P220_.28thermonuclease.2C_fragment_corrupted.29_with_NgoMIV.2FSpeI.2C_P160_.28NLS.2C_fragment_named_F77.29_with_AgeI.2FSpeI.2C_P40_.28GSAT_linker.2C_fragment_named_F79.29_with_NgoMIV.2FSpeI.2C_P7_.28PhyB.2C_fragment_named_F78.29_with_AgeI.2FSpeI"> Preparative digestion and gelelectrophoresis of P220 (thermonuclease, fragment corrupted) with NgoMIV/SpeI,  P160 (NLS, fragment named F77) with AgeI/SpeI, P40 (GSAT linker, fragment named F79) with NgoMIV/SpeI, P7 (PhyB, fragment named F78) with AgeI/SpeI </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P220 (thermonuclease, fragment corrupted) with NgoMIV/SpeI,  P160 (NLS, fragment named F77) with AgeI/SpeI, P40 (GSAT linker, fragment named F79) with NgoMIV/SpeI, P7 (PhyB, fragment named F78) with AgeI/SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P220 (thermonuclease, fragment corrupted) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P220
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P160 (NLS, fragment named F77) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>plasmid P160
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P40 (GSAT linker, fragment named F79) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>plasmid P40
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P7 (PhyB, fragment named F78) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>plasmid P40
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches containing P220 and P40 after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis
</LI></UL><UL><LI> the vector backbones where only a few base pairs were cut out were purified via PCR-purification
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>digestion of P220 (thermonuclease, fragment corrupted) with NgoMIV/SpeI
</TD><TD>1 kbp DNA ladder
</TD><TD>digestion of P40 (GSAT linker, fragment named F79) with NgoMIV/SpeI
</TD></TR><TR><TD>not as expected, band was discarded
</TD><TD>as expected, lower band (108 bp) was cut out
</TD></TR></TBODY></TABLE><UL><LI> the band was extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_Purification_of_vector_backbones_P7_.28PhyB_in_pSB1C3.2C_fragment_F78.29.2C_P160_.28NLS_in_pSB1C3.2C_fragment_F77.29"> PCR Purification of vector backbones P7 (PhyB in pSB1C3, fragment F78), P160 (NLS in pSB1C3, fragment F77)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the study:</B> PCR Purification of vector backbones P7 (PhyB in pSB1C3, fragment F78), P160 (NLS in pSB1C3, fragment F77)
</P><UL><LI> in order to get rid of the few base pairs that were cut out during digestion, the digestion batches were purified 
</LI></UL><PRE>with QIAquick PCR Purification Kit, Qiagen
</PRE><UL><LI> the yiedls were 56.2 ng/µl for F77 and 277,6 ng/µl for F78
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F78.2BF79.28PhyB.2Blinker.29_and_F59.2BF71_.28pSB1C3.2BIg-K.29"> Ligation of F78+F79(PhyB+linker) and F59+F71 (pSB1C3+Ig-K)</SPAN></H3><P><B>Investigator:</B> Katrin
<B>Aim of the experiment:</B> Ligation of F78+F79(PhyB+linker) and F59+F71 (pSB1C3+Ig-K)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F78+F79(PhyB+linker)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.36 µl
</TD><TD>F78 (277.6 ng/µl, 4807 bp)
</TD></TR><TR><TD>1.77 µl
</TD><TD>P79 (3.8 ng/µl, 108 bp)
</TD></TR><TR><TD>14.87 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F59+F71 (pSB1C3+Ig-K)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2086 bp)
</TD></TR><TR><TD>3.12 µl
</TD><TD>P71 (3.1 ng/µl, 67 bp)
</TD></TR><TR><TD>12.38 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1_blue_with_F78.2BF79.28PhyB.2Blinker.29_and_F59.2BF71_.28pSB1C3.2BIg-K.29"> Transformation of <I>E. coli</I> XL1 blue with F78+F79(PhyB+linker) and F59+F71 (pSB1C3+Ig-K) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F78+F79(PhyB+linker) and F59+F71 (pSB1C3+Ig-K) and negative controls F78 and F59
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on plates containing antibiotics (CamR).
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_P192_.28Thermonuclease.2C_O64_.26_O65.29"> PCR of P192 (Thermonuclease, O64 &amp; O65)  </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>  PCR of P192 (Thermonuclease, O64 &amp; O65)
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer  O64 (NucA_fw)
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O65 (NucA_rv)
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>1:1000 dilution of plasmid DNA (P192)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The gradient PCR program was performed after following scheme with following conditions:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>30 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_8">Week 8</SPAN></H1><DIV class="week" id="WWeek_8"><H2><SPAN class="mw-headline" id="Monday.2C_June_10rd"><B>Monday, June 10rd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_Gel_of_PCR_of_Thermonuclease_.28P192.29"> Analytical Gel of  PCR of Thermonuclease (P192) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the study:</B> Analysis of PCR of Thermonuclease (P192); supposed letgth: 447 bp
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA ladder 1kb
</TD><TD>P192 PCR product
</TD><TD>DNA ladder 1kb
</TD></TR><TR><TD><B>Result:</B></TD><TD> -
</TD><TD> as expected
</TD><TD> -
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_Purification_of_Thermonuclease_.28P192.29"> PCR Purification of Thermonuclease (P192) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the study:</B> PCR Purification of Thermonuclease (P192)
</P><UL><LI> After the PCR the batch was purified with QIAquick PCR Purification Kit, Qiagen
</LI><LI> Eluation in 35 µl EB buffer -&gt; F80
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_F67.2BF69_Ligation_.28Linker.2BGFP_in_Suffix_of_TMD.29"> Miniprep of F67+F69 Ligation (Linker+GFP in Suffix of TMD) </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the study:</B> Preparation of DNA of the F67+F69 Ligation from 09.06.13 (Linker+GFP in Suffix of TMD)
</P><P><B>Procedure:</B> according to MiniPrep Kit QIAGEN; measurement of concentration (NanoDrop)
</P><UL><LI> Eluation in 35 µl EB buffer each
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plamsid</B></TD><TD><B>Concentration</B></TD></TR><TR><TD>P236
</TD><TD>108.0 ng/µl
</TD></TR><TR><TD>P237
</TD><TD>116.0 ng/µl
</TD></TR><TR><TD>P238
</TD><TD>165.4 ng/µl
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_F78_.2B_F79_.28PhyB_.2B_Linker.29_and_F59_.2B_F71_.28pSB1C3_.2B_lgkappa.29"> Picking of F78 + F79 (PhyB + Linker) and F59 + F71 (pSB1C3 + lgkappa) </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the study:</B> Picking of F78 + F79 (PhyB + Linker) and F59 + F71 (pSB1C3 + lgkappa)
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl CamR  and 4 ml LB-medium)
</LI></UL><UL><LI> 3 colonies were picked for every Transformation product.
</LI></UL></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Sequencing_of_P150_.28FluA.29.2C_P233_.28pActin.29.2C_P213_.28Spycatcher.29_and_F67.2BF69_Ligation_.28Linker.2BGFP_.2B_TMD.29"> Sequencing of P150 (FluA), P233 (pActin), P213 (Spycatcher) and F67+F69 Ligation (Linker+GFP + TMD)</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the study:</B> Sequencing of P150 (FluA), P233 (pActin), P213 (Spycatcher) and F67+F69 Ligation (Linker+GFP + TMD)
</P><P><B>Procedure:</B>Eurofins SmartSeq Kit protocol (15 µl plasmid DNA (50 - 100 ng), 2 µl sequencing primer).
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD>P150
</TD><TD>P233
</TD><TD>P213
</TD><TD>P236
</TD><TD>P237
</TD></TR><TR><TD><B>Name</B></TD><TD>FluA
</TD><TD>pActin
</TD><TD>Spycatcher
</TD><TD>Linker+GFP+TMD clone1
</TD><TD>Linker+GFP+TMD clone2
</TD></TR><TR><TD><B>O3 (forward)</B></TD><TD>FR01002308
</TD><TD>FR01002307
</TD><TD>FR01002305
</TD><TD>FR01002299
</TD><TD>FR01002298
</TD></TR><TR><TD><B>O4 (reverse)</B></TD><TD>n.a.
</TD><TD>FR01002306
</TD><TD>n.a.
</TD><TD>FR01002297
</TD><TD>FR01002311
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR_of_BPul_Laccase_P219_.2B_Analytical_Gelelectrophoresis"> PCR of BPul Laccase P219 + Analytical Gelelectrophoresis </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B>  PCR of Bpul Laccase P219.
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P219:O24 (Bpul_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P143:O25 (Bpul_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P219)- 1:1000 dilution
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>90 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI><LI> The resulting fragment was called F82
</LI><LI> Further on an analytif Gelelectrophoresis was performed to be sure that the expected fragment was amplified.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>DNA ladder 1kb</B></TD><TD><B>P219 PCR product</B></TD></TR></TBODY></TABLE><UL><LI> As we can see, the fragment has the expected length of about 1600 BP.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_F80_.28PCR_purification_of_Thermonuclease.29_with_XbaI.2FAgeI"> Preparative digestion of F80 (PCR purification of Thermonuclease) with XbaI/AgeI </SPAN></H3><P><B>Investigator:</B> Florian, Johanna
</P><P><B>Aim of the experiment:</B> Preparative digestion of F80 (PCR purification of Thermonuclease) with XbaI/AgeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of F80 (PCR purification of Thermonuclease) with XbaI/AgeI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>F80
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> The digested fragment was purified via PCR-purification (Qiagen-kit)
</LI></UL><UL><LI> A yield of 26.2 ng/µl could be measured for F81
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F59_.2B_F81_.28Thermonuclease_in_pSB1C3.29"> Ligation of F59 + F81 (Thermonuclease in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Florian, Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F59 + F81 (Thermonuclease in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F59 + F81 (Thermonuclease in pSB1C3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2086 bp)
</TD></TR><TR><TD>2.45 µl
</TD><TD>P81 (26.2 ng/µl, 447 bp)
</TD></TR><TR><TD>13.55 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_XL1_blue_with_F59.2BF81.28Thermonuclease_in_pSB1C3.29"> Transformation of <I>E. coli</I> XL1 blue with F59+F81(Thermonuclease in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F59+F81(Thermonuclease in pSB1C3) and the negative control F59.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Addition of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspensions were plated on plates containing antibiotics (CamR).
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on plates containing antibiotics (CamR).
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_purification_of_F82_.28PCR_product_of_Bpul_Laccase.29_with_XbaI.2FAgeI-HF_and_PCR_purification_kit_of_Qiagen"> Preparative digestion and purification of F82 (PCR product of Bpul Laccase) with XbaI/AgeI-HF and PCR purification kit of Qiagen </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Preparative digestion and purification of F82 (Bpul Laccase) with XbaI+AgeI and PCR purification kit of Qiagen
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of F82 with XbaI/AgeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>F82
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> the digested fragment was purified via PCR-purification
</LI></UL><UL><LI> the resulting fragment was called F83 (F82 digested with XbaI+AgeI)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F83.28PCR_product_of_Bpul_Laccase_digested_with_AgeI-HF_.2B_XbaI.29and_F59_.28psB1C3_digested_with_AgeI-HF_.2B_XbaI.29"> Ligation of F83(PCR product of Bpul Laccase digested with AgeI-HF + XbaI)and F59 (psB1C3 digested with AgeI-HF + XbaI)</SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F83(PCR product of Bpul Laccase digested with AgeI-HF + XbaI)and F59 (psB1C3 digested with AgeI-HF + XbaI)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F83+F59 (psB1C3+Bpul Laccase)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15.5 µl
</TD><TD>F83 (9.2 ng/µl, 1600 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI><LI> The ligation was performed over night at 16°C.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_11th"><B>Tuesday, June 11th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_ligation_products_F59.2BF81_and_F59.2BF83">  Transformation of <I>E. coli</I> XL1 blue with ligation products F59+F81 and F59+F83 </SPAN></H3><P><B>Investigator:</B> Louise, Andi
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with ligation products F59+F81 and F59+F83.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_F78.2BF79_.28PhytochromB_.2BLinker.29_and_F59.2BF71_.28IgKappa.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with F78+F79 (PhytochromB +Linker) and F59+F71 (IgKappa) </SPAN></H3><P><B>Investigator:</B> Louise, Johanna
</P><P><B>Aim of the experiment:</B>Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with F78+F79 (PhytochromB +Linker) and F59+F71 (IgKappa).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> F78+F79 (PhytochromB +Linker) clone 1</TD><TD>P239
</TD></TR><TR><TD> F78+F79 (PhytochromB +Linker) clone 2</TD><TD>P240
</TD></TR><TR><TD> F78+F79 (PhytochromB +Linker) clone 3</TD><TD>P241
</TD></TR><TR><TD> F59+F71 (IgKappa) clone 4</TD><TD>P242
</TD></TR><TR><TD> F59+F71 (IgKappa) clone 5</TD><TD>P243
</TD></TR><TR><TD> F59+F71 (IgKappa) clone 6</TD><TD>P244
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P239-P241_.28PhytochromB.2BLinker.29_and_P242-P244_.28IgKappa.29_with_XbaI_and_AgeI"> Analytical digestion and gelelectrophoresis  of P239-P241 (PhytochromB+Linker) and P242-P244 (IgKappa) with XbaI and AgeI </SPAN></H3><P><B>Investigator:</B> Louise, Rosario, Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P239-P241 (PhytochromB+Linker) and P242-P244 (IgKappa) with XbaI and AgeI
</P><P><B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P293-P244)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI><LI> For P242-P244 a 2% agarose gel was used and for P293-P241 a 0.5% agarose gel was used. 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>Digestion of P242 (IgK) with XbaI &amp; AgeI, clone 1
</TD><TD>Digestion of P243 (IgK) with XbaI &amp; AgeI, clone 2
</TD><TD>Digestion of P244 (IgK) with XbaI &amp; AgeI, clone 3
</TD><TD>DNA ladder 100bp
</TD></TR><TR><TD><B>Result:</B></TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> -
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>Digestion of P239 (PhyB+Linker) with XbaI &amp; AgeI, clone 1
</TD><TD>Digestion of P240 (PhyB+Linker) with XbaI &amp; AgeI, clone 2
</TD><TD>Digestion of P241 (PhyB+Linker) with XbaI &amp; AgeI, clone 3
</TD><TD>DNA ladder 1kb
</TD></TR><TR><TD><B>Result:</B></TD><TD> failed
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> -
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P204_.28IgKappa-SigP.2C_fragment_named_F90.29_with_AgeI.2FSpeI.2C_P157_.28EreB.2C_fragment_named_F86.29_with_NgoMIV.2FSpeI.2C_P208_.28NanoLuc.2C_fragment_named_F87.29_with_NgoMIV.2FSpeI.2C_P213_.28Spycatcher.2C_fragment_named_F88.29_with_AgeI.2FSpeI_and_P170_.28Spytag.2C_fragment_named_F89.29_with_NgoMIV.2FSpeI"> Preparative digestion and gelelectrophoresis of P204 (IgKappa-SigP, fragment named F90) with AgeI/SpeI,  P157 (EreB, fragment named F86) with NgoMIV/SpeI, P208 (NanoLuc, fragment named F87) with NgoMIV/SpeI, P213 (Spycatcher, fragment named F88) with AgeI/SpeI and P170 (Spytag, fragment named F89) with NgoMIV/SpeI</SPAN></H3><P><B>Investigator:</B> Johanna, Rosario, Flo
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P204 (IgKappa-SigP, fragment named F90) with AgeI/SpeI,  P157 (EreB, fragment named F86) with NgoMIV/SpeI, P208 (NanoLuc, fragment named F87) with NgoMIV/SpeI, P213 (Spycatcher, fragment named F88) with AgeI/SpeI and P170 (Spytag, fragment named F89) with NgoMIV/SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P204 (IgKappa-SigP, fragment named F90) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P204
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P157 (EreB, fragment named F86) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P157
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P208 (NanoLuc, fragment named F87) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P208
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P213 (Spycatcher, fragment named F88) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P213
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P170 (Spytag, frament named F89) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P170
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) were added to the 50 µl reaction batches after digestion and were loaded on a 1% agarose gel (P204, P157, P208, P213) and 2 % agarose gel (P170) for preparative gelelectrophoresis
</LI><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P204 (IgKappa-SigP) with AgeI/SpeI
</TD><TD>100 bp DNA ladder (Generuler)
</TD><TD>digestion of P157 (EreB) with NgoMIV/SpeI
</TD></TR><TR><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P208 (Nanoluc) with NgoMIV/SpeI
</TD><TD>100 bp DNA ladder (Generuler)
</TD><TD>digestion of P213 (Spycatcher) with NgoMIV/SpeI
</TD></TR><TR><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder (Generuler)
</TD><TD>digestion of P170 (Spytag) with NgoMIV/SpeI
</TD></TR><TR><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> the band was extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F90.2BF86_.28IgKappa-SigP_.2B_EreB.29.2C_F90.2BF87_.28IgKappa-SigP_.2B_Nanoluciferase.29.2C_F90.2BF88_.28IgKappa-SigP_.2B_SpyCatcher.29.2C_F90.2BF89_.28IgKappa-SigP_.2B_SpyTag.29_and_negativ_control_of_F90_.28IgKappa-SigP.29"> Ligation of F90+F86 (IgKappa-SigP + EreB), F90+F87 (IgKappa-SigP + Nanoluciferase), F90+F88 (IgKappa-SigP + SpyCatcher), F90+F89 (IgKappa-SigP + SpyTag) and negativ control of F90 (IgKappa-SigP)</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F90+F86 (IgKappa-SigP + EreB), F90+F87 (IgKappa-SigP + Nanoluciferase), F90+F88 (IgKappa-SigP + SpyCatcher), F90+F89 (IgKappa-SigP + SpyTag) and negativ control of F90 (IgKappa-SigP)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F90+F86 (IgKappa-SigP + EreB)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.7 µl
</TD><TD>F86 (31.0 ng/µl, 1254 bp)
</TD></TR><TR><TD>6.5 µl
</TD><TD>F90 (15.3 ng/µl, 2144 bp)
</TD></TR><TR><TD>4.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F90+F87 (IgKappa-SigP + Nanoluciferase)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7.0 µl
</TD><TD>F87 (10.2 ng/µl, 510 bp)
</TD></TR><TR><TD>6.5 µl
</TD><TD>F90 (15.3 ng/µl, 2144 bp)
</TD></TR><TR><TD>3.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F90+F88 (IgKappa-SigP + SpyCatcher)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6.3 µl
</TD><TD>F88 (8.7 ng/µl, 339 bp)
</TD></TR><TR><TD>6.5 µl
</TD><TD>F90 (15.3 ng/µl, 2144 bp)
</TD></TR><TR><TD>4.2 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F90+F89 (IgKappa-SigP + SpyTag)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10.5 µl
</TD><TD>F89 (8.7 ng/µl, 39 bp)
</TD></TR><TR><TD>6.5 µl
</TD><TD>F90 (15.3 ng/µl, 2144 bp)
</TD></TR><TR><TD>0.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control of F 90 (IgKappa-SigP) was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_pActin_.28P143.29_and_Thermonuclease_.28P192.29_to_find_the_best_conditions_for_PCR_.2B_Measurement_via_Nanodrop_.2B_Analytical_Gelelectrophoresis_to_get_to_know_about_it.21"> PCR of pActin (P143) and Thermonuclease (P192) to find the best conditions for PCR + Measurement via Nanodrop + Analytical Gelelectrophoresis to get to know about it! </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>  PCR of pActin (P143) and Thermonuclease (P192) to find the best conditions for PCR + Measurement via Nanodrop + Analytical Gelelectrophoresis to get to know about it! 
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> The PCR for P143 (pActin) was performed three times under different conditions
</LI></UL><UL><LI> First batch - PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P143:O62 (pAct_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P143:O63 (pAct_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)- 1:1000 dilution
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Second batch - PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq GC Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P143:O62 (pAct_for))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P143:O63 (pAct_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)- 1:1000 dilution
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Third batch - PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x Herculase Fusion Polymerase reaction buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1.25 µl
</TD><TD>10 µM Forward Primer (P143:O62 (Bpul_for))
</TD></TR><TR><TD>1.25 µl
</TD><TD>10 µM Reverse Primer (P143:O63 (Bpul_rev))
</TD></TR><TR><TD>1 µL
</TD><TD>Herculase II Fusion Polymerase
</TD></TR><TR><TD>2 µl
</TD><TD>Plasmid DNA (P143)- 1:1000 dilution
</TD></TR><TR><TD>0.5 µl
</TD><TD>DMSO
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content of all batches has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>51 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>90 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> The PCR for P192(Thermonuclease) was performed three times under different conditions
</LI></UL><UL><LI> First batch - PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Reaction Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P192:O64 (NucA_fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P192:O65(NucA_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P192)- 1:1000 dilution
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Second batch - PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq GC Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (P192:O64 (NucA_fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (P192:O65(NucA_rev))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P192)- 1:1000 dilution
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Third batch - PCR reaction mixture
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x Herculase II DNA Fusion Polymerase reaction Buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1.25 µl
</TD><TD>10 µM Forward Primer (P192:O64 (NucA_fw))
</TD></TR><TR><TD>1.25 µl
</TD><TD>10 µM Reverse Primer (P192:O65(NucA_rev))
</TD></TR><TR><TD>1 µL
</TD><TD>Herculase II Fusion DNA Polymerase
</TD></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA (P192)- 1:1000 dilution
</TD></TR><TR><TD>0.5 µL
</TD><TD>DMSO
</TD></TR><TR><TD>30.5 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program of the Thermonuclease (P192) was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>54 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>40 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> Mix the content of the batches with pipette
</LI></UL><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI><LI> The concentrations of all PCR products have been measured
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>P143 - Batch #1 - PCR product - 5.9 ng/µl  </B></TD><TD><B>P143 - Batch #2 - PCR product - 40.5 ng/µl</B></TD><TD><B>P143 - Batch #3 - PCR product - 12.4 ng/µl</B></TD><TD><B>P192 - Batch #1 - PCR product - 39.7 ng/µl</B></TD><TD><B>P192 - Batch #2 - PCR product - 23.1 ng/µl</B></TD><TD><B>P192 - Batch #3 - PCR product - 82.5 ng/µl</B></TD></TR></TBODY></TABLE><UL><LI> As we can see, the highest concentration of PCR product seems to be in P143 - Batch #2 and P192 - Batch #3. 
</LI></UL><UL><LI> Further on an analytic Gelelectrophoresis was performed to be sure that the expected fragment was amplified.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>DNA ladder 1kb</B></TD><TD><B>P143 - Batch #1 - PCR product</B></TD><TD><B>P143 - Batch #2 - PCR product</B></TD><TD><B>P143 - Batch #3 - PCR product</B></TD><TD><B>P192 - Batch #1 - PCR product</B></TD><TD><B>P192 - Batch #2 - PCR product</B></TD><TD><B>P192 - Batch #3 - PCR product</B></TD></TR></TBODY></TABLE><UL><LI> As we can see, the fragment has the expected length of about 1600 BP referring to the PCR product of P143 and about XXX BP referring to the product of P192.
</LI><LI> Further on the analytical gel confirms the output of the nano drop. 
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_PCR_reaction_batch_.232_of_P143_.28pActin.29_XbaI.2FPstI_and_PCR_reaction_batch_.233_of_P192_with_XbaI.2FAgeI"> Preparative digestion of PCR reaction batch #2 of P143 (pActin) XbaI/PstI and PCR reaction batch #3 of P192 with XbaI/AgeI </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparative digestion of PCR reaction batch #2 of P143 (pActin) XbaI/PstI and PCR reaction batch #3 of P192 with XbaI/AgeI 
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of PCR reaction batch #2 of P143 (pActin) with XbaI/PstI.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>24 µl
</TD><TD>PCR reaction batch #2 of P143
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>PstI #1 (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI #2 (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> I used 1 µl of both PstI batches because one seems to be broken
</LI><LI> Incubation for 2.5 h at 37 °C.
</LI><LI> The digested fragment was purified via PCR-purification (Qiagen-kit)
</LI><LI> A yield of 16.9 ng/µl could be measured for the resulting F84
</LI></UL><UL><LI> Batch for preparative digestion of PCR reaction batch #3 of P192 (Thermonuclease) with XbaI/AgeI-HF.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>24 µl
</TD><TD>PCR reaction batch #2 of P143
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI><LI> The digested fragment was purified via PCR-purification (Qiagen-kit)
</LI><LI> A yield of 46.2 ng/µl could be measured for the resulting F85
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F84.28Digestion_of_PCR_product_.28GC_rich_buffer.29_of_P143_.28pActin.29_with_PstI.2FXbaI.29_and_F42_.28psB1C3_digested_with_PstI.2FXbaI.29_and_Ligation_of_F85.28Digestion_of_PCR_product_.28Herculase_II.29_of_P192_.28Thermonuclease.29_with_XbaI.2FAgeI_and_F59_.28psB1C3_digested_with_AgeI-HF_.2B_XbaI.29"> Ligation of F84(Digestion of PCR product (GC rich buffer) of P143 (pActin) with PstI/XbaI) and F42 (psB1C3 digested with PstI/XbaI) and Ligation of F85(Digestion of PCR product (Herculase II) of P192 (Thermonuclease) with XbaI/AgeI and F59 (psB1C3 digested with AgeI-HF + XbaI) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Ligation of F84(Digestion of PCR product (GC rich buffer) of P143 (pActin) with PstI/XbaI) with F42 (psB1C3 digested with PstI/XbaI) and Ligation of F85(Digestion of PCR product (Herculase II) of P192 (Thermonuclease) with XbaI/AgeI-HF) with F59 (psB1C3 digested with AgeI-HF/XbaI)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F84+F42 (psB1C3+pActin)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12.6 µl
</TD><TD>F84 (16.9 ng/µl, 1500 bp)
</TD></TR><TR><TD>1.4 µl
</TD><TD>F42 (73.5 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>4 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F85+F59 (psB1C3+Thermonuclease)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.6 µl
</TD><TD>F85 (46.2 ng/µl, 500 bp)
</TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2100 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared for both batches. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI><LI> The ligation was performed over night at 16°C.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_plasmids_P157.2C_P208_and_P213">  Transformation of <I>E. coli</I> XL1 blue with plasmids P157, P208 and P213 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with plasmids P157, P208 and P213.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_12th"><B>Wednesday, June 12th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F42.2BF84.2C_F90.2BF86.2C_F90.2BF87.2C_F90.2BF88.2C_F90.2BF89.2C_F59.2BF85">  Transformation of <I>E. coli</I> XL1 blue with F42+F84, F90+F86, F90+F87, F90+F88, F90+F89, F59+F85 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with F42+F84, F90+F86, F90+F87, F90+F88, F90+F89, F59+F85
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> Centrifugation for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated on new chlorampenicol plates.
</LI></UL></DIV></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P240_.28PhyB_.2B_Linker.29_with_AgeI.2FSpeI.2C_P244_.28IgKappa.29_with_AgeI.2FSpeI.2C_P197_.28XylE.29_with_NgoMIV.2FSpeI.2C_P150_.28FluA.29_with_NgoMIV.2FSpeI.2C_P126_.28N-TEV.29_with_AgeI.2FSpeI.2C_P127_.28C-TEV.29_with_NgoMIV.2FSpeI.2C_P168_.28PIF3.29_with_NgoMIV.2FSpeI_and_P222_.28PIF6.29_with_NgoMIV.2FSpeI"> Preparative digestion and gelelectrophoresis of P240 (PhyB + Linker) with AgeI/SpeI, P244 (IgKappa) with AgeI/SpeI, P197 (XylE) with NgoMIV/SpeI, P150 (FluA) with NgoMIV/SpeI, P126 (N-TEV) with AgeI/SpeI, P127 (C-TEV) with NgoMIV/SpeI, P168 (PIF3) with NgoMIV/SpeI and P222 (PIF6) with NgoMIV/SpeI </SPAN></H3><P><B>Investigator:</B> Louise, Florian
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P240 (PhyB + Linker) with AgeI/SpeI, P244 (IgKappa) with AgeI/SpeI, P197 (XylE) with NgoMIV/SpeI, P150 (FluA) with NgoMIV/SpeI, P126 (N-TEV) with AgeI/SpeI, P127 (C-TEV) with NgoMIV/SpeI, P168 (PIF3) with NgoMIV/SpeI and P222 (PIF6) with NgoMIV/SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P240 (PhyB + Linker) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P240
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P244 (IgKappa) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P244
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P197 (XylE) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P197
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P150 (FluA) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P150
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P126 (N-TEV) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P126
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P127 (C-TEV) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P127
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P168 (PIF3) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P168
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P222 (PIF6) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P222
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P126 (N-TEV) with AgeI/SpeI
</TD><TD>digestion of P127 (C-TEV) with NgoMIV/SpeI
</TD><TD>digestion of P150 (FluA) with NgoMIV/SpeI
</TD></TR><TR><TD>as expected, upper band was cut out
</TD><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>digestion of P168 (PIF3) with NgoMIV/SpeI
</TD><TD>digestion of P197 (XylE) with NgoMIV/SpeI
</TD><TD>digestion of P222 (PIF6) with NgoMIV/SpeI
</TD></TR><TR><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder (Generuler)
</TD><TD>digestion of P244 (IgKappa) with AgeI/SpeI
</TD><TD>digestion of P240 (PhyB + Linker) with AgeI/SpeI
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>as expected, upper band was cut out
</TD><TD>as expected, upper band was cut out
</TD></TR></TBODY></TABLE><UL><LI> the DNA was extracted from the gel with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL><UL><LI> the fragments received the following names:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P126 (N-TEV)
</TD><TD>F93
</TD></TR><TR><TD>P127 (C-TEV)
</TD><TD>F94
</TD></TR><TR><TD>P150 (FluA)
</TD><TD>F95
</TD></TR><TR><TD>P168 (PIF3)
</TD><TD>F96
</TD></TR><TR><TD>P197 (XylE)
</TD><TD>F97
</TD></TR><TR><TD>P222 (PIF6)
</TD><TD>F98
</TD></TR><TR><TD>P240 (PhyB + Linker)
</TD><TD>F91
</TD></TR><TR><TD>P244 (IgKappa)
</TD><TD>F92
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_plasmids_P126_.28N-TEV.29.2C_P127_.28C-TEV.29.2C_P222_.28PIF6.29.2C_P240_.28PhyB.2BLinker.29_and_P244_.28IgKappa.29">  Transformation of <I>E. coli</I> XL1 blue with plasmids P126 (N-TEV), P127 (C-TEV), P222 (PIF6), P240 (PhyB+Linker) and P244 (IgKappa) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with plasmids P126 (N-TEV), P127 (C-TEV), P222 (PIF6), P240 (PhyB+Linker) and P244 (IgKappa).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated again on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_P157.2C_P208.2C_P213.2C_F59_.2B_F81_.28Thermonuclease_in_pSB1C3.29_and_F59_.2B_F83_.28Laccase_in_pSB1C3.29"> Picking of P157, P208, P213, F59 + F81 (Thermonuclease in pSB1C3) and F59 + F83 (Laccase in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the study:</B> Picking of P157, P208, P213, F59 + F81 (Thermonuclease in pSB1C3) and F59 + F83 (Laccase in pSB1C3).
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl CamR  and 4 ml LB-medium)
</LI></UL><UL><LI> 2 colonies were picked for every ligation product and 1 colony was picked for every transformation product.
</LI></UL></DIV></DIV></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_E..C2.A0coli_XL1_blue_with_F92.2BF89.2C_F92.2BF87.2C_F92.2BF86.2C_F92.2BF88.2C_F92.2BF97.2C_F90.2BF97.2C_F91.2BF94.2C_F93.2BF79_and_their_negative_controls"> Ligation of <I>E. coli</I> XL1 blue with F92+F89, F92+F87, F92+F86, F92+F88, F92+F97, F90+F97, F91+F94, F93+F79 and their negative controls</SPAN></H3><P><B>Investigator:</B> Jeff, Flo, Louise, Christopher
</P><P><B>Aim of the experiment:</B> Ligation of <I>E. coli</I> XL1 blue with F92+F89, F92+F87, F92+F86, F92+F88, F92+F97, F90+F97, F91+F94, F93+F79 and their negative controls
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F92+F89 :
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>F92 (116.7 ng/µl, 2153 bp)
</TD></TR><TR><TD>16.14 µl
</TD><TD>F89 (6.8 ng/µl, 39 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>0.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F92+F87 :
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>F92 (116.7 ng/µl, 2153 bp)
</TD></TR><TR><TD>7.0 µl
</TD><TD>F87 (10.2 ng/µl, 510 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F92+F86:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>F92 (116.7 ng/µl, 2153 bp)
</TD></TR><TR><TD>5.7 µl
</TD><TD>F86 (31.0 ng/µl, 1260 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>10.44 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F92+F88 :
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>F92 (116.7 ng/µl, 2153 bp)
</TD></TR><TR><TD>5.4 µl
</TD><TD>F88 (8.7 ng/µl, 339 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>10.74 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F92+F97:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>F92 (116.7 ng/µl, 2153 bp)
</TD></TR><TR><TD>6.6 µl
</TD><TD>F97 (19.4 ng/µl, 918 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F90+F97:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6.5 µl
</TD><TD>F90 (15.3 ng/µl, 2144 bp)
</TD></TR><TR><TD>6.6 µl
</TD><TD>F97 (19.4 ng/µl, 918 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>3.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F91+F94:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.63 µl
</TD><TD>F91 (159 ng/µl, 4129 bp)
</TD></TR><TR><TD>4.6 µl
</TD><TD>F94 (5.6 ng/µl, 354 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11.77 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F93+F79:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.6 µl
</TD><TD>F93 (27.4 ng/µl, 2440 bp)
</TD></TR><TR><TD>3.5 µl
</TD><TD>F79 (3.8 ng/µl, 108 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> The ligation was performed at room temperature for 1 hour
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F92.2BF89.2C_F92.2BF87.2C_F92.2BF86.2C_F92.2BF88.2C_F92.2BF97.2C_F90.2BF97.2C_F91.2BF94.2C_F93.2BF79_and_their_negative_controls">  Transformation of <I>E. coli</I> XL1 blue with F92+F89, F92+F87, F92+F86, F92+F88, F92+F97, F90+F97, F91+F94, F93+F79 and their negative controls </SPAN></H3><P><B>Investigator:</B> Jeff, Florian
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with F92+F89, F92+F87, F92+F86, F92+F88, F92+F97, F90+F97, F91+F94, F93+F79 and their negative controls.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> Centrifugation for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated on new chloramphenicol plates.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_13th"><B>Thursday, June 13th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="QuikChange_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIV_-_of_P250.2C_P251.2C_P219_.28Laccase.29.2C_P163_.28NPT-Cassette.29">  QuikChange mutagenesis to remove forbidden restriction sites - SDMIV - of P250, P251, P219 (Laccase), P163 (NPT-Cassette) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment: </B> Removal of forbidden restriction site from P251  (Laccase) and P163 (NPT-Cassette).
</P><P><B>Procedure: Quickchange - PCR</B></P><P><B>Reaction batch for P219 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P251 - 1:20 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O74 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O75 (10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P163 (NPT-Cassette)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P163 - 1:60 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O70 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O71 (10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P250 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P250 - 1:20 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O74 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O75 (10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P217 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P217 - 1:100 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O74 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O75 (10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P251 (Laccase) and P163 (NPT-Cassette)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><UL><LI> After the PCR, the Quickchange product was stored in the fridge.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR_of_AlcR_.28P61.2C_pAutoRex8.29_and_IRES_.28P61.2C_pAutoRex8.29_and_analytical_gel_electrophoresis"> PCR of AlcR (P61, pAutoRex8) and IRES (P61, pAutoRex8) and analytical gel electrophoresis </SPAN></H3><P><B>Investigator: Louise, Johanna, Christopher</B></P><P><B>Aim of the experiment:</B> PCR of AlcR (P61, pAutoRex8) and IRES (P61, pAutoRex8).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture for AlcR
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (O78 (AlcR_fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (O79 (AlcR_rv))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P61)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> PCR reaction mixture for IRES
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (O68 (IRES_fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (O69 (IRES_rv))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P61)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><UL><LI> The gradient PCR program was performed after following scheme with following conditions (T<SUB>m</SUB>=56 °C; ΔG=4 °C; AlcR in row 11(=60 °C); IRES in row 1 (=52.0 °C)):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>T<SUB>m</SUB>=56 °C; ΔG=4 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>160 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> Resulting products was labeled as F99 (AlcR) and F100 (IRES).
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F99_.28AlcR.29_and_F100_.28IRES.29"> Analytical gelelectrophoresis of F99 (AlcR) and F100 (IRES) </SPAN></H3><P><B>Investigator:</B> Louise, Christopher
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F99 (AlcR) and F100 (IRES).
</P><P><B>Procedure:</B></P><UL><LI> 4 µl of F99 and F100 were mixed with 0.44  µl DNA loading buffer (10x)
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>F100
</TD><TD>F99
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_P157.2C_P208.2C_P213.2C_F59.2BF81.28Thermonuclease.29.2C_F59.2BF83_.28Laccase.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P157, P208, P213, F59+F81(Thermonuclease), F59+F83 (Laccase) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P157, P208, P213, F59+F81(Thermonuclease), F59+F83 (Laccase)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> P157</TD><TD>P245
</TD></TR><TR><TD> P208</TD><TD>P246
</TD></TR><TR><TD> P213</TD><TD>P247
</TD></TR><TR><TD> F59+F81 I</TD><TD>P248
</TD></TR><TR><TD> F59+F81 II</TD><TD>P249
</TD></TR><TR><TD> F59+F83 I</TD><TD>P250
</TD></TR><TR><TD> F59+F83 II</TD><TD>P251
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P248-P249_.28Thermonuclease.29_and_P250-251_.28Laccase.29_with_EcoRI_and_SpeI"> Analytical digestion and gelelectrophoresis  of P248-P249 (Thermonuclease) and P250-251 (Laccase) with EcoRI and SpeI </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P248-P249 (Thermonuclease) and P250-251 (LAccase) with EcoRI and SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P248-P251)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI><LI> For P248-P251 a 1% agarose gel was used.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA marker 1kb
</TD><TD>Digestion of P248 with EcoRI &amp; SpeI
</TD><TD>Digestion of P249 with EcoRI &amp; SpeI
</TD><TD>Digestion of P250 with EcoRI &amp; SpeI
</TD><TD>empty
</TD><TD>Digestion of P251 with EcoRI &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD> -
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> no insert (Laccase)
</TD><TD> -
</TD><TD> no insert (Laccase)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P204">  Transformation of <I>E. coli</I> XL1 blue with P204 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with P204
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> Centrifugation for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated on new chloramphenicol plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P61_.28pSH21.2FpAutoRex8.29"> Sequencing of P61 (pSH21/pAutoRex8)</SPAN></H3><P><B>Investigator:</B> Rosario, Katrin
</P><P><B>Aim of the study:</B> Sequencing of P61 (pSH21/pAutoRex8)
</P><P><B>Procedure:</B></P><P>Eurofins SmartSeq Kit protocol (15 µl plasmid DNA (50 - 100 ng), 2 µl sequencing primer).
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>#</B></TD><TD>P61
</TD></TR><TR><TD><B>Name</B></TD><TD>pSH 21 (AlcR/IRES)
</TD></TR><TR><TD><B>O88 (primer 1)</B></TD><TD>FR02305828
</TD></TR><TR><TD><B>O89 (primer 2)</B></TD><TD>FR02305829
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_F92.2BF89.2C_F92.2BF87.2C_F92.2BF86.2C_F92.2BF88.2C_F92.2BF97.2C_F90.2BF97.2C_F91.2BF94.2C_F93.2BF79.2C_P126_.28N-TEV.29.2C_P127_.28C-TEV.29.2C_P222_.28PIF6.29.2C_P240_.28PhyB.2BLinker.29_and_P244_.28IgKappa.29.2C_F42.2BF84.2C_F90.2BF86.2C_F90.2BF87.2C_F90.2BF88.2C_F90.2BF89.2C_F59.2BF85"> Picking of F92+F89, F92+F87, F92+F86, F92+F88, F92+F97, F90+F97, F91+F94, F93+F79,  P126 (N-TEV), P127 (C-TEV), P222 (PIF6), P240 (PhyB+Linker) and P244 (IgKappa),  F42+F84, F90+F86, F90+F87, F90+F88, F90+F89, F59+F85  </SPAN></H3><P><B>Investigator:</B> Rosario, Katrin
</P><P><B>Aim of the study:</B> Picking of F92+F89, F92+F87, F92+F86, F92+F88, F92+F97, F90+F97, F91+F94, F93+F79,  P126 (N-TEV), P127 (C-TEV), P222 (PIF6), P240 (PhyB+Linker) and P244 (IgKappa),  F42+F84, F90+F86, F90+F87, F90+F88, F90+F89, F59+F85  
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl CamR  and 4 ml LB-medium)
</LI><LI> There were no colonies on plates F42+F84 and xy
</LI><LI> There were a lot of colonies on NK 90 but there were significantly more colonies on the plates with ligation products
</LI><LI> 2 colonies were picked for every ligation product.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_14th"><B>Friday, June 14th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_P204.2C_F92.2BF86_and_F42.2BF84"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P204, F92+F86 and F42+F84  </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the study:</B> Picking of P204, F92+F86 and F42+F84 
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl CamR, 4 ml LB-medium)
</LI></UL><UL><LI> 3 colonies were picked for every transformation product.
</LI></UL></DIV><DIV class="effectors"><DIV class="safety_mechanism"><DIV class="general"><H3><SPAN class="mw-headline" id="Mass_of_Minipreps_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue">  Mass of Minipreps of overnight cultures of transformated <I>E. coli</I> XL1 blue </SPAN></H3><P><B>Investigator:</B> Rosario, Johanna, Andi
</P><P><B>Aim of the experiment:</B>Mass of Minipreps of overnight cultures of transformated <I>E. coli</I> XL1 blue.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI> The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> F59+F85</TD><TD>P252
</TD></TR><TR><TD> F59+F85</TD><TD>P253
</TD></TR><TR><TD> F90+F86</TD><TD>P254
</TD></TR><TR><TD> F90+F86</TD><TD>P255
</TD></TR><TR><TD> F90+F87</TD><TD>P256
</TD></TR><TR><TD> F90+F87</TD><TD>P257
</TD></TR><TR><TD> F90+F88</TD><TD>P258
</TD></TR><TR><TD> F90+F88</TD><TD>P259
</TD></TR><TR><TD> F90+F89</TD><TD>P260
</TD></TR><TR><TD> F90+F89</TD><TD>P261
</TD></TR><TR><TD> F90+F97</TD><TD>P262
</TD></TR><TR><TD> F90+F97</TD><TD>P263
</TD></TR><TR><TD> F91+F94</TD><TD>P264
</TD></TR><TR><TD> F91+F94</TD><TD>P265
</TD></TR><TR><TD> F92+F87</TD><TD>P266
</TD></TR><TR><TD> F92+F87</TD><TD>P267
</TD></TR><TR><TD> F92+F97</TD><TD>P268
</TD></TR><TR><TD> F92+F97</TD><TD>P269
</TD></TR><TR><TD> F92+F88</TD><TD>P270
</TD></TR><TR><TD> F92+F88</TD><TD>P271
</TD></TR><TR><TD> F92+F89</TD><TD>P272
</TD></TR><TR><TD> F92+F89</TD><TD>P273
</TD></TR><TR><TD> F93+F79</TD><TD>P274
</TD></TR><TR><TD> F93+F79</TD><TD>P275
</TD></TR><TR><TD> P222</TD><TD>P276
</TD></TR><TR><TD> P222</TD><TD>P277
</TD></TR><TR><TD> P240</TD><TD>P278
</TD></TR><TR><TD> P240</TD><TD>P279
</TD></TR><TR><TD> P126</TD><TD>P280
</TD></TR><TR><TD> P126</TD><TD>P281
</TD></TR><TR><TD> P244</TD><TD>P282
</TD></TR><TR><TD> P244</TD><TD>P283
</TD></TR><TR><TD> P127</TD><TD>P284
</TD></TR><TR><TD> P127</TD><TD>P285
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P250.2C_P251.2C_P219_.28Laccase.29.2C_P163_.28NPT-Cassette.29">  Transformation of <I>E. coli</I> XL1 blue with P250, P251, P219 (Laccase), P163 (NPT-Cassette)  </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1 blue with P250, P251, P219 (Laccase), P163 (NPT-Cassette) 
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> Centrifugation for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and these concentrated cell suspensions were plated on new chloramphenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P276_.28Retrafo_of_P222.2C_fw.29.2C_P282_.28Retrafo_of_P244.2C_fw.29.2C_P278_.28Retrafo_of_P240.2C_fw.2Brv.29_and_P217_.28fw.2Brv.29">  Sequencing of P276 (Retrafo of P222, fw), P282 (Retrafo of P244, fw), P278 (Retrafo of P240, fw+rv) and P217 (fw+rv) </SPAN></H3><P><B>Investigators:</B> Louise
</P><P><B>Aim of the experiment:</B> Sequencing of P276(Retrafo of P222, fw), P282(Retrafo of P244, fw), P278 (Retrafo of P240, fw+rv) and P217 (fw+rv).
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM).
</P><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P276
</TD><TD> PIF6
</TD><TD> FR02305830 (O3)
</TD><TD> -
</TD></TR><TR><TD> P282
</TD><TD> IgKappa-SigP in pSB1C3
</TD><TD> FR02305831 (O3)
</TD></TR><TR><TD> P278
</TD><TD> PhyB-36AALinker in pSB1C3
</TD><TD> FR02305832 (O3)
</TD><TD> FR02305833 (O4)
</TD></TR><TR><TD> P217
</TD><TD> Laccase QCIV
</TD><TD> FR02305826 (O3)
</TD><TD> FR02305827 (O4)
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P252-P275_with_EcoRI_and_SpeI"> Analytical digestion and gelelectrophoresis  of P252-P275 with EcoRI and SpeI </SPAN></H3><P><B>Investigator:</B> Rosario, Johanna, Louise
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P252-P275 with EcoRI and SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batches for every Plasmid of P252-P275 contained 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P252-P275)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>The batches for the Quickchangeproduct were prepared with 6 µl     of Quickchangeproduct and 2 µl DNA loading buffer. 
</LI></UL><UL><LI> Incubation for 60 min at 37 °C.
</LI><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI><LI> For P252-P275 a 1% agarose gel was used.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA marker 1kb
</TD><TD>Digestion of P252 with EcoRI &amp; SpeI
</TD><TD>Digestion of P253 with EcoRI &amp; SpeI
</TD><TD>Digestion of P254 with EcoRI &amp; SpeI
</TD><TD>Digestion of P255 with EcoRI &amp; SpeI
</TD><TD>Digestion of P256 with EcoRI &amp; SpeI
</TD><TD>Digestion of P257 with EcoRI &amp; SpeI
</TD><TD>Digestion of P258 with EcoRI &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD> -
</TD><TD> corrupt
</TD><TD> as expected
</TD><TD> corrupt
</TD><TD> corrupt
</TD><TD> corrupt
</TD><TD> as expected
</TD><TD> corrupt
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA marker 1kb
</TD><TD>Digestion of P259 with EcoRI &amp; SpeI
</TD><TD>Digestion of P260 with EcoRI &amp; SpeI
</TD><TD>Digestion of P261 with EcoRI &amp; SpeI
</TD><TD>Digestion of P262 with EcoRI &amp; SpeI
</TD><TD>Digestion of P263 with EcoRI &amp; SpeI
</TD><TD>Digestion of P264 with EcoRI &amp; SpeI
</TD><TD>Digestion of P265 with EcoRI &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD> -
</TD><TD> as expected
</TD><TD> as expected (sequencing necessary)
</TD><TD> as expected (sequencing necessary)
</TD><TD> bad miniprep
</TD><TD>as expected
</TD><TD> not sure (has to be repeated with control P278)
</TD><TD> not sure (has to be repeated with control P278)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA marker 1kb
</TD><TD>Digestion of P266 with EcoRI &amp; SpeI
</TD><TD>Digestion of P267 with EcoRI &amp; SpeI
</TD><TD>Digestion of P268 with EcoRI &amp; SpeI
</TD><TD>Digestion of P269 with EcoRI &amp; SpeI
</TD><TD>Digestion of P270 with EcoRI &amp; SpeI
</TD><TD>Digestion of P271 with EcoRI &amp; SpeI
</TD><TD>Digestion of P272 with EcoRI &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD> -
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected (has to be sequenced)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA marker 1kb
</TD><TD>Digestion of P273 with EcoRI &amp; SpeI
</TD><TD>Digestion of P274 with EcoRI &amp; SpeI
</TD><TD>Digestion of P275 with EcoRI &amp; SpeI
</TD><TD>Digestion of QC I product of P251 with DpnI
</TD><TD>Digestion of QC I product of P163I with DpnI
</TD><TD>Digestion of QC V product of P163II with DpnI
</TD><TD>Digestion of QC V product of P217 with DpnI
</TD></TR><TR><TD><B>Result:</B></TD><TD> -
</TD><TD> as expected (has to be sequenced)
</TD><TD> as expected (has to be sequnced)
</TD><TD> as expected (has to be sequnced)
</TD><TD> corrupt, no QC product
</TD><TD> corrupt, no QC product
</TD><TD> corrupt, no QC product
</TD><TD> as expected, QC product
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_Digestion_of_IRES_.28F100.29_and_AlcR_.28F99.29_with_EcoRI_and_SpeI"> Preparative Digestion of IRES (F100) and AlcR (F99) with EcoRI and SpeI</SPAN></H3><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>25 µl
</TD><TD>Fragment DNA (F99 or F100)
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 150 min at 37 °C
</LI><LI> Purification by QIAGEN PCR Purification Kit
</LI><LI> new Names: F99 -&gt; F101 and F100 -&gt; F102
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Oligohybridization_of_Strep-TEV_.28O90_.2B_O91.29">  Oligohybridization of Strep-TEV (O90 + O91) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment: </B>Oligohybridization of Strep-TEV (O90 + O91)
</P><P><B>Procedure:</B></P><UL><LI>25 µL of 100 pmol/µl of  Strep-TEV_fw  and 25 µL of 100 pmol/µl  Strep-TEV_rev
</LI></UL><UL><LI>Heating up to 95 °C for 5 min 
</LI></UL><UL><LI>Cooling at RT in a styropor box overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_15th"><B>Saturday, June 15th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_P204_.28IgKappa-SigP_in_pSB1C3.29.2C_F92.2BF82_.28IgKappa-SigP_EreB_in_pSB1C3.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  with P204 (IgKappa-SigP in pSB1C3), F92+F82 (IgKappa-SigP_EreB in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  with P204 (IgKappa-SigP in pSB1C3), F92+F82 (IgKappa-SigP_EreB in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI> The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD><TD align="center" style="background:#f0f0f0;"><B>Concentration in ng/µl</B></TD></TR><TR><TD> ReTraFo of P204 (IgKappa-SigP in pSB1C3)</TD><TD>P286</TD><TD>105.7
</TD></TR><TR><TD> ReTraFo of P204 (IgKappa-SigP in pSB1C3)</TD><TD>P287</TD><TD>303.6
</TD></TR><TR><TD> ReTraFo of P204 (IgKappa-SigP in pSB1C3)</TD><TD>P288</TD><TD>268.1
</TD></TR><TR><TD> F92+F86 (IgKappa-SigP_EreB in pSB1C3)</TD><TD>P289</TD><TD>121.2
</TD></TR><TR><TD> F92+F86 (IgKappa-SigP_EreB in pSB1C3)</TD><TD>P290</TD><TD>402.3
</TD></TR><TR><TD> F92+F86 (IgKappa-SigP_EreB in pSB1C3)</TD><TD>P291</TD><TD>405.2
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F41.2BF101_.28AlcR.29_and_F41.2BF102_.28IRES_of_poliovirus.29"> Ligation of F41+F101 (AlcR) and F41+F102 (IRES of poliovirus) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F41+F101 (AlcR) and F41+F102 (IRES of poliovirus).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F41+F101 (AlcR), 50 ng vector used, 1:1:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.56 µl
</TD><TD>F41 (89.9 ng/µl, 2086 bp)
</TD></TR><TR><TD>5.23 µl
</TD><TD>F101 (11.3 ng/µl, 2466 bp)
</TD></TR><TR><TD>11.21 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F41+F102 (IRES of poliovirus), 100 ng vector used, 3:1:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.11 µl
</TD><TD>F41 (89.9 ng/µl, 2086 bp)
</TD></TR><TR><TD>7.92 µl
</TD><TD>F102 (11.4 ng/µl, 628 bp)
</TD></TR><TR><TD>7.97 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared. Instead of the insert, the same volume of ddH<SUB>2</SUB>O was added to the reaction batch.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P289-P291_.28IgKappa-SigP_EreB_in_pSB1C3.29_with_EcoRI_.26_SpeI"> Analytical digestion and gelelectrophoresis  of P289-P291 (IgKappa-SigP_EreB in pSB1C3) with EcoRI &amp; SpeI </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P289-P291 (IgKappa-SigP_EreB in pSB1C3) with EcoRI &amp; SpeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P289-P291, P245)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>DNA ladder 100bp
</TD><TD>Digestion of P289 (IgKappa-SigP_EreB in pSB1C3) with EcoRI &amp; SpeI, clone 1
</TD><TD>Digestion of P290 (IgKappa-SigP_EreB in pSB1C3) with EcoRI &amp; SpeI, clone 2
</TD><TD>Digestion of P291 (IgKappa-SigP_EreB in pSB1C3) with EcoRI &amp; SpeI, clone 3
</TD><TD>Digestion of P245 (EreB in pSB1C3) with EcoRI &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected (control)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F41.2BF101_.28AlcR_in_pSB1C3.29_and_F41.2BF102_.28IRES_in_pSB1C3.29">  Transformation of <I>E. coli</I> XL1 blue with F41+F101 (AlcR in pSB1C3) and F41+F102 (IRES in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with F41+F101 (AlcR in pSB1C3) and F41+F102 (IRES in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_16th"><B>Sunday, June 16th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="QuikChange_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMIV_-_of_P217_.28Laccase.29.2C_P163_.28NPT-Cassette.29">  QuikChange mutagenesis to remove forbidden restriction sites - SDMIV - of P217 (Laccase), P163 (NPT-Cassette) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment: </B> Removal of forbidden restriction site from P217  (Laccase) and P163 (NPT-Cassette).
</P><P><B>Procedure: Quikchange - PCR</B></P><P><B>Reaction batch for P217 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P217 - 1:100 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O74 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O75 (10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P163 (NPT-Cassette)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P163 - 1:50 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O70 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O71 (10 µM)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P251 (Laccase) and P163 (NPT-Cassette)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><UL><LI> After the PCR, the Quickchange product was stored in the fridge.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_QC_of_P217.2C_P163.2C_F41.2BF101_and_F41.2BF102"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with QC of P217, P163, F41+F101 and F41+F102  </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the study:</B> Picking of QC of P217, P163, F41+F101 and F41+F102
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl Cam, 4 ml LB-medium)
</LI></UL><UL><LI> 1 colonies were picked for every transformation product.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F59.2BF74_.28SERK-SigP.29_and_F59.2BF103_.28Strep-TEV.29"> Ligation of F59+F74 (SERK-SigP) and F59+F103 (Strep-TEV) </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F59+F74 (SERK-SigP) and F59+F103 (Strep-TEV).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F59+F74 (SERK-SigP), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2086 bp)
</TD></TR><TR><TD>3 µl
</TD><TD>F74 (4.8 ng/µl, 100 bp)
</TD></TR><TR><TD>12.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F59+F103 (Strep-TEV), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>F59 (66.7 ng/µl, 2086 bp)
</TD></TR><TR><TD>15.5 µl
</TD><TD>F103 (504.9 ng/µl, 54 bp)
</TD></TR><TR><TD>7.97 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P217_.28QC_of_16.05.2013.29.2C_P163_.28QC_of_16.05.2013.29.2C_P7_.28Retrafo_of_PhyB.29_and_ligation_products_F59.2BF74_.28SERK-SigP_in_pSB1C3.29_and_F59.2BF103_.28Strep-TEV_in_pSB1C3.29">  Transformation of <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013), P163 (QC of 16.05.2013), P7 (Retrafo of PhyB) and ligation products F59+F74 (SERK-SigP in pSB1C3) and F59+F103 (Strep-TEV in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Andi, Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013) and P163 (QC of 16.05.2013), P7 (Retrafo of PhyB) and ligation products F59+F74 (SERK-SigP in pSB1C3) and F59+F103 (Strep-TEV in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product/ 2 µl of plasmid (P7) was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H1><DIV class="week" id="WWeek_9"><H2><SPAN class="mw-headline" id="Monday.2C_June_17th"><B>Monday, June 17th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_QC_I_of_P163_.28NPT-casette.29.2C_QC_V_of_P217_.28laccase.29.2C_ligation_products_F41.2BF101_.28AlcR.29_and_F41.2BF102_.28IRES.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  with QC I of P163 (NPT-casette), QC V of P217 (laccase), ligation products F41+F101 (AlcR) and F41+F102 (IRES) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue  with QC I of P163 (NPT-casette), QC V of P217 (laccase), ligation products F41+F101 (AlcR) and F41+F102 (IRES)
<B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI> The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> QC I of P163</TD><TD>P292
</TD></TR><TR><TD> QC V of P217</TD><TD>P293
</TD></TR><TR><TD> F41+F101 I</TD><TD>P294
</TD></TR><TR><TD> F41+F101 II</TD><TD>P295
</TD></TR><TR><TD> F41+F101 III</TD><TD>P296
</TD></TR><TR><TD> F41+F102 I</TD><TD>P297
</TD></TR><TR><TD> F41+F102 II</TD><TD>P298
</TD></TR><TR><TD> F41+F102 III</TD><TD>P299
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_P217_.28QC_of_16.05.2013.29.2C_P163_.28QC_of_16.05.2013.29.2C_P7_.28Retrafo_of_PhyB.29_and_ligation_products_F59.2BF74_.28SERK-SigP_in_pSB1C3.29_and_F59.2BF103_.28Strep-TEV_in_pSB1C3.29"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013), P163 (QC of 16.05.2013), P7 (Retrafo of PhyB) and ligation products F59+F74 (SERK-SigP in pSB1C3) and F59+F103 (Strep-TEV in pSB1C3)  </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the study:</B> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013), P163 (QC of 16.05.2013), P7 (Retrafo of PhyB) and ligation products F59+F74 (SERK-SigP in pSB1C3) and F59+F103 (Strep-TEV in pSB1C3)
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl Cam, 4 ml LB-medium)
</LI></UL><UL><LI> 1 colony was picked for every transformation product.
</LI></UL></DIV></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P264_.28PhyBlinker.2BcTev.29_with_AgeI.2FSpeI.2C_P299_.28IRES.29_with_NgoMIV.2FSpeI.2C_P274_.28nTevlinker.29with_AgeI.2FSpeI.2C_P111_.28TEV.29_with_AgeI.2FSpeI_and_P248_.28Nuclease.29_with_NgoMIV.2FSpeI"> Preparative digestion and gelelectrophoresis of P264 (PhyBlinker+cTev) with AgeI/SpeI, P299 (IRES) with NgoMIV/SpeI, P274 (nTevlinker)with AgeI/SpeI, P111 (TEV) with AgeI/SpeI and P248 (Nuclease) with NgoMIV/SpeI </SPAN></H3><P><B>Investigator:</B> Johanna, Flo
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P264 (PhyBlinker+cTev) with AgeI/SpeI, P299 (IRES) with NgoMIV/SpeI, P274 (nTevlinker)with AgeI/SpeI, P111 (TEV) with AgeI/SpeI and P248 (Nuclease) with NgoMIV/SpeI
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P264 (PhyBlinker+cTev) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P264
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P299 (IRES) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P299
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P274 (nTevlinker)with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P274
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P111 (TEV) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P111
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P248 (Nuclease) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P248
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and were loaded on a 1% agarose gel for preparative gelelectrophoresis.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>Digestion of P111 with AgeI &amp; SpeI
</TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P248 with NgoMIV &amp; SpeI
</TD><TD>Digestion of P264 with AgeI &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>Digestion of P274 with AgeI &amp; SpeI
</TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P299 with NgoMIV &amp; SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD>low concentration, <B>discarded</B></TD><TD>not digested, <B>discarded</B></TD></TR></TBODY></TABLE><UL><LI> The DNA was extracted from the gel with QIAquick Gel Extraction Kit, QIAGEN
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P293-P299_with_EcoRI_.26_SpeI_and_P292_with_EcoRI_.26_PstI"> Analytical digestion and gelelectrophoresis  of P293-P299 with EcoRI &amp; SpeI and P292 with EcoRI &amp; PstI </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P293-P299 with EcoRI &amp; SpeI and P292 with EcoRI &amp; PstI
<B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF (PstI-HF for P292)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P292-P299)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P292 (QC I P163 (NPT-casette)) with EcoRI &amp; PstI
</TD><TD>Digestion of P293 (QC V P217 (Laccase)) with EcoRI &amp; SpeI
</TD><TD>Digestion of P294 (F41+F101 (AlcR)) with EcoRI &amp; SpeI, clone 1
</TD><TD>Digestion of P295 (F41+F101 (AlcR)) with EcoRI &amp; SpeI, clone 2
</TD><TD>Digestion of P296 (F41+F101 (AlcR)) with EcoRI &amp; SpeI, clone 3
</TD><TD>Digestion of P297 (F41+F102 (IRES)) with EcoRI &amp; SpeI, clone 1
</TD><TD>Digestion of P298 (F41+F102 (IRES)) with EcoRI &amp; SpeI, clone 2
</TD><TD>Digestion of P299 (F41+F102 (IRES)) with EcoRI &amp; SpeI, clone 3
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD><B>Result:</B></TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="DpnI_digestion_of_P163QCI_and_P217QCV"> DpnI digestion of P163QCI and P217QCV </SPAN></H3><P><B>Investigator:</B> Flo, Jeff
</P><P><B>Aim of the experiment:</B> DpnI digestion of P163QCI and P217QCV.
</P><P><B>Procedure:</B></P><UL><LI> 1 µl of DpnI was added to the QuikChange products of P163QCI and P217QCV.
</LI></UL><UL><LI> The mixtures were incubated for 1 h at 37 °C; afterwards they were ready for transformation.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_DpnI_digestion_of_P163QCI_and_P217QCV"> Analytical gelelectrophoresis of DpnI digestion of P163QCI and P217QCV </SPAN></H3><P><B>Investigator:</B> Jeff, Flo
</P><P><B>Aim of the experiment:</B> Analysis of the DpnI digestion of P163QCI and P217QCV
</P><P><B>Procedure:</B></P><UL><LI> 4.5 µl of the QuikChange products of P163 and P217, named P163QCI and P217QCV were mixed together with 0.5 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 30 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kbp DNA ladder
</TD><TD>DpnI digestion of P163QCI.1
</TD><TD>DpnI digestion of P163QCI.2
</TD><TD>DpnI digestion of P217QCV.1
</TD><TD>DpnI digestion of P217QCV.2
</TD></TR><TR><TD><B>Result:</B></TD><TD>empty
</TD><TD>empty
</TD><TD>empty
</TD><TD>empty
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F77.2BF105_.28NLS_in_pSB1C3_.2B_mature_thermonuclease.29"> Ligation of F77+F105 (NLS in pSB1C3 + mature thermonuclease) </SPAN></H3><P><B>Investigator:</B> Jeff, Flo
</P><P><B>Aim of the experiment:</B> Ligation of F77+F105 (NLS in pSB1C3 + mature thermonuclease).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F77+F105 (NLS in pSB1C3 + mature thermonuclease), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.78 µl
</TD><TD>F77 (56.2 ng/µl, 2143 bp)
</TD></TR><TR><TD>5.22 µl
</TD><TD>F105 (12 ng/µl, 447 bp)
</TD></TR><TR><TD>10 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed overnight at 16 °C.
</LI></UL></DIV><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_P217_.28QC_of_16.05.2013.29.2C_P163_.28QC_of_16.05.2013.29_and_P160.2C_P264.2C_P274_and_P275_for_retransformations">  Transformation of <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013), P163 (QC of 16.05.2013) and P160, P264, P274 and P275 for retransformations </SPAN></H3><P><B>Investigator:</B> Flo
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013), P163 (QC of 16.05.2013) and P160, P264, P274 and P275 for retransformations.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange product/2 µl of plasmid DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Sequencing_of_P248_.28Nuc_A.2C_fw.29.2C_P275_.28N-TEV_36AAlinker.2C_fw.29.2C_P299_.28IRES.2C_fw.29.2C_P292_.28QC_I_of_P163_NPT-casette.2Cfw.2Brv.29.2C_P293_.28QC_V_of_P217_Laccase.2Cfw.2Brv.29_and_P290_.28IgKappa_EreB.2Cfw.2Brv.29">  Sequencing of P248 (Nuc A, fw), P275 (N-TEV_36AAlinker, fw), P299 (IRES, fw), P292 (QC I of P163 NPT-casette,fw+rv), P293 (QC V of P217 Laccase,fw+rv) and P290 (IgKappa_EreB,fw+rv) </SPAN></H3><P><B>Investigators:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P248 (Nuc A, fw), P275 (N-TEV_36AAlinker, fw), P299 (IRES, fw), P292 (QC I of P163 NPT-casette,fw+rv), P293 (QC V of P217 Laccase,fw+rv) and P290 (IgKappa_EreB,fw+rv).
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM).
</P><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P248
</TD><TD> NucA
</TD><TD> FR02305834 (O3)
</TD><TD> -
</TD></TR><TR><TD> P275
</TD><TD> N-TEV_36AAlinker
</TD><TD> FR02305835 (O3)
</TD></TR><TR><TD> P299
</TD><TD> IRES
</TD><TD> FR02305825 (O3)
</TD><TD> -
</TD></TR><TR><TD> P292
</TD><TD> QC I NPT-casette
</TD><TD> FR02305824 (O3)
</TD><TD> FR02305823 (O4)
</TD></TR><TR><TD> P293
</TD><TD> QC V Laccase
</TD><TD> FR02305822 (O3)
</TD><TD> FR02305821 (O4)
</TD></TR><TR><TD> P290
</TD><TD> IgKappa_EreB
</TD><TD> FR02305820 (O3)
</TD><TD> FR02305819 (O4)
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_18th"><B>Tuesday, June 18th</B></SPAN></H2><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1_blue_transformated_with_P160.2C_P264.2C_P274_and_P275"> Picking of <I>E. coli</I> XL1 blue transformated with  P160, P264, P274 and P275  </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the study:</B> Picking of <I>E. coli</I> XL1 blue transformated with  P160, P264, P274 and P275.
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl Cam, 4 ml LB-medium)
</LI></UL><UL><LI> 2 colonies were picked for every transformation product.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1_blue_with_F77.2BF105_.28NLS_in_pSB1C3_.2B_mature_thermonuclease.29_and_P299_Retrafo">  Transformation of <I>E. coli</I> XL1 blue with F77+F105 (NLS in pSB1C3 + mature thermonuclease) and P299 Retrafo </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1 blue with P217 (QC of 16.05.2013), P163 (QC of 16.05.2013) and P160, P264, P274 and P275 for retransformations.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of plasmid DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_F59.2BF103_.28Strep-TEV-Linker_in_PsB1C3.29_and_F59.2BF74_.28Serk-Sigp_in_PsB1C3.29_and_P7_.28PhyB.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with F59+F103 (Strep-TEV-Linker in PsB1C3) and F59+F74 (Serk-Sigp in PsB1C3) and P7 (PhyB)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with F59+F103 (Strep-TEV-Linker in PsB1C3) and F59+F74 (Serk-Sigp in PsB1C3) and P7 (PhyB).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> P7 (PhyB) clone 1</TD><TD>P300
</TD></TR><TR><TD> P7 (PhyB) clone 2</TD><TD>P301
</TD></TR><TR><TD> F59+F74 (Serk-Sigp in PsB1C3) clone 1</TD><TD>P302
</TD></TR><TR><TD> F59+F74 (Serk-Sigp in PsB1C3) clone 2</TD><TD>P303
</TD></TR><TR><TD> F59+F103 (Strep-TEV-Linker in PsB1C3) clone 1</TD><TD>P304
</TD></TR><TR><TD> F59+F103 (Strep-TEV-Linker in PsB1C3) clone 2</TD><TD>P305
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P300-P301_.28PhyB.29.2C_P302-P303_.28Serk-Sigp_in_PsB1C3.29_and_P304-P305_.28Strep-TEV-Linker_in_PsB1C3.29_with_XbaI_and_AgeI"> Analytical digestion and gelelectrophoresis  of P300-P301 (PhyB), P302-P303 (Serk-Sigp in PsB1C3) and P304-P305 (Strep-TEV-Linker in PsB1C3) with XbaI and AgeI </SPAN></H3><P><B>Investigator:</B> Louise, Rosario, Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P300-P301 (PhyB), P302-P303 (Serk-Sigp in PsB1C3) and P304-P305 (Strep-TEV-Linker in PsB1C3) with XbaI and AgeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P300-P305)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>Digestion of P300 with XbaI/AgeI
</TD><TD>Digestion of P301 with XbaI/AgeI
</TD><TD>Digestion of P302 with XbaI/AgeI
</TD><TD>Digestion of P303 with XbaI/AgeI
</TD><TD>Digestion of P304 with XbaI/AgeI
</TD><TD>Digestion of P305 with XbaI/AgeI
</TD><TD>1 kb DNA Ladder
</TD><TD>100 bp DNA Ladder
</TD></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P212_.28SERK-TMD.29_with_NgoMIV.2FSpeI.2C_P238_.28SERK-TMD_Linker_GFP.29_with_NgoMIV.2FSpeI.2C_P303_.28SERK-SigP.29_with_AgeI.2FSpeI_and_P304_.28StrepTEV-Linker.29_with_AgeI.2FSpeI"> Preparative digestion and gelelectrophoresis of P212 (SERK-TMD) with NgoMIV/SpeI, P238 (SERK-TMD_Linker_GFP) with NgoMIV/SpeI, P303 (SERK-SigP) with AgeI/SpeI and P304 (StrepTEV-Linker) with AgeI/SpeI </SPAN></H3><P><B>Investigator:</B> Louise, Flo, Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P212 (SERK-TMD) with NgoMIV/SpeI, P238 (SERK-TMD_Linker_GFP) with NgoMIV/SpeI, P303 (SERK-SigP) with AgeI/SpeI and P304 (StrepTEV-Linker) with AgeI/SpeI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for preparative digestion of P212 (SERK-TMD) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P212
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P238 (SERK-TMD_Linker_GFP) with NgoMIV/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P238
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P303 (SERK-SigP) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P303
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P304 (StrepTEV-Linker) with AgeI/SpeI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P304
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and P238, P303 and P304 were loaded on a 1% agarose gel, while P212 was loaded on a 2% agarose gel for preparative gelelectrophoresis.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>Digestion of P238 (SERK-TMD_Linker_GFP) with NgoMIV/SpeI
</TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P303 (SERK-SigP) with AgeI/SpeI
</TD><TD>Digestion of P304 (StrepTEV-Linker) with AgeI/SpeI
</TD></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P212 (SERK-TMD) with NgoMIV/SpeI
</TD><TD>100 bp DNA Ladder
</TD></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD></TR></TBODY></TABLE><UL><LI> The DNA was extracted from the gel with QIAquick Gel Extraction Kit, QIAGEN
</LI></UL></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F109.2BF86_.28SERK-SigP_.2B_EreB.29.2C_F109.2BF87_.28SERK-SigP_.2B_NanoLuc.29.2C_F109.2BF88_.28SERK-SigP_.2B_SpyCatcher.29.2C_F109.2BF89_.28SERK-SigP_.2B_SpyTag.29.2C_F109.2BF97_.28SERK-SigP_.2B_XylE.29.2C_F109.2BF107_.28SERK-SigP_.2B_SERK-TMD.29_and_F110.2BF108_.28Strep-tag-TEV-site-linker_.2B_SERK-TMD_8AAlinker_GFPmut1.29"> Ligation of F109+F86 (SERK-SigP + EreB), F109+F87 (SERK-SigP + NanoLuc), F109+F88 (SERK-SigP + SpyCatcher), F109+F89 (SERK-SigP + SpyTag), F109+F97 (SERK-SigP + XylE), F109+F107 (SERK-SigP + SERK-TMD) and F110+F108 (Strep-tag-TEV-site-linker + SERK-TMD_8AAlinker_GFPmut1) </SPAN></H3><P><B>Investigator:</B> Jeff, Flo
</P><P><B>Aim of the experiment:</B> Ligation of F109+F86 (SERK-SigP + EreB), F109+F87 (SERK-SigP + NanoLuc), F109+F88 (SERK-SigP + SpyCatcher), F109+F89 (SERK-SigP + SpyTag), F109+F97 (SERK-SigP + XylE), F109+F107 (SERK-SigP + SERK-TMD) and F110+F108 (Strep-tag-TEV-site-linker + SERK-TMD_8AAlinker_GFPmut1).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F109+F86 (SERK-SigP + EreB), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>5.57 µl
</TD><TD>F86 (31.0 ng/µl, 1254 bp)
</TD></TR><TR><TD>10.51 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F109+F87 (SERK-SigP + NanoLuc), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>6.89 µl
</TD><TD>F87 (10.2 ng/µl, 510 bp)
</TD></TR><TR><TD>9.19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F109+F88 (SERK-SigP + SpyCatcher), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>5.37 µl
</TD><TD>F88 (8.7 ng/µl, 339 bp)
</TD></TR><TR><TD>10.71 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F109+F89 (SERK-SigP + SpyTag), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>12.0 µl
</TD><TD>F89 (6.8 ng/µl, 39 bp)
</TD></TR><TR><TD>4.08 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F109+F97 (SERK-SigP + XylE), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>6.52 µl
</TD><TD>F97 (19.4 ng/µl, 918 bp)
</TD></TR><TR><TD>9.56 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F109+F107 (SERK-SigP + SERK-TMD), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>5.38 µl
</TD><TD>F107 (4.8 ng/µl, 186 bp)
</TD></TR><TR><TD>10.70 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F110+F108 (Strep-tag-TEV-site-linker + SERK-TMD_8AAlinker_GFPmut1), 100 ng vector used, insert:vector = 3:1
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.38 µl
</TD><TD>F110 (72.4 ng/µl, 2143 bp)
</TD></TR><TR><TD>5.53 µl
</TD><TD>F108 (23.6 ng/µl, 933 bp)
</TD></TR><TR><TD>10.09 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="QuikChange_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMV_of_P217_.28Laccase_RFC10.29_and_SDMI_of_P162_.28NPT-Cassette_RFC10.29">  QuikChange mutagenesis to remove forbidden restriction sites - SDMV of P217 (Laccase RFC10) and SDMI of P162 (NPT-Cassette RFC10) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment: </B> Removal of the last existing forbidden restriction site from P217 (Laccase) and the first one from P162 (NPT-Cassette).
</P><P><B>Procedure: Quikchange - PCR</B></P><P><B>Reaction batch for P217 (Laccase)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid template (P217 - 1:100 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O74 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O75 (=10 µM)
</TD></TR><TR><TD>18.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P163 (NPT-Cassette)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid template (P162 - 1:50 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O70 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O71 (10 µM)
</TD></TR><TR><TD>18.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P251 (Laccase) and P163 (NPT-Cassette)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><UL><LI> After the PCR, the Quickchange products were mixed with 1 µl of DpnI (10 U/µl) and incubated at 37 °C for 1 h.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_the_DpnI_digested_products_of_the_performed_QuikChange_of_SDMV_of_P217_.28Laccase_RFC10.29_and_SDMI_of_P162_.28NPT-Cassette_RFC10.29"> Analytical gelelectrophoresis of the DpnI digested products of the performed QuikChange of SDMV of P217 (Laccase RFC10) and SDMI of P162 (NPT-Cassette RFC10) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> 4.5 µl of each of the DpnI digested QuikChange products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 40 min at an 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kbp DNA ladder
</TD><TD>DpnI digestion of P163QCI
</TD><TD>DpnI digestion of P217QCV
</TD></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_products_F109.2BF86_.28SERK-SigP_.2B_EreB.29.2C_F109.2BF87_.28SERK-SigP_.2B_NanoLuc.29.2C_F109.2BF88_.28SERK-SigP_.2B_SpyCatcher.29.2C_F109.2BF89_.28SERK-SigP_.2B_SpyTag.29.2C_F109.2BF97_.28SERK-SigP_.2B_XylE.29.2C_F109.2BF107_.28SERK-SigP_.2B_SERK-TMD.29_and_F110.2BF108_.28Strep-tag-TEV-site-linker_.2B_SERK-TMD_8AAlinker_GFPmut1.29.2C_with_DpnI_digested_QuikChange_products_P162QCI_and_P217QCV.2C_and_P212_.28ReTraFo.29.2C_P238_.28ReTraFo.29"> Transformation of <I>E. coli</I> XL1-blue with ligation products F109+F86 (SERK-SigP + EreB), F109+F87 (SERK-SigP + NanoLuc), F109+F88 (SERK-SigP + SpyCatcher), F109+F89 (SERK-SigP + SpyTag), F109+F97 (SERK-SigP + XylE), F109+F107 (SERK-SigP + SERK-TMD) and F110+F108 (Strep-tag-TEV-site-linker + SERK-TMD_8AAlinker_GFPmut1), with DpnI digested QuikChange products P162QCI and P217QCV, and P212 (ReTraFo), P238 (ReTraFo) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation products F109+F86 (SERK-SigP + EreB), F109+F87 (SERK-SigP + NanoLuc), F109+F88 (SERK-SigP + SpyCatcher), F109+F89 (SERK-SigP + SpyTag), F109+F97 (SERK-SigP + XylE), F109+F107 (SERK-SigP + SERK-TMD) and F110+F108 (Strep-tag-TEV-site-linker + SERK-TMD_8AAlinker_GFPmut1), with DpnI digested QuikChange products P162QCI and P217QCV, and P212 (ReTraFo), P238 (ReTraFo).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products/10 µl of ligation products/2 µl of plasmid DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P294_.28AlcR.29">  Sequencing of P294 (AlcR) </SPAN></H3><P><B>Investigators:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P294 (AlcR).
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM).
</P><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P294
</TD><TD> AlcR
</TD><TD> FR02305817 (O3)
</TD><TD> FR02305818(O4)
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_19th"><B>Wednesday, June 19th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="QuikChange_mutagenesis_to_remove_forbidden_restriction_sites_-_SDMI_of_P294_.28AlcR.29">  QuikChange mutagenesis to remove forbidden restriction sites - SDMI of P294 (AlcR) </SPAN></H3><P><B>Investigator:</B> Florian - Master of QuikChange 2.0
</P><P><B>Aim of the experiment: </B> Removal of the first forbidden restriction site from P294 (AlcR).
</P><P><B>Procedure: Quikchange - PCR</B></P><P><B>Reaction batch for P294 (AlcR)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid template (P294 - 1:50 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O80 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O81 (=10 µM)
</TD></TR><TR><TD>18.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P294 (AlcR)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>19
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><UL><LI> After the PCR, the Quickchange products were mixed with 1 µl of DpnI (10 U/µl) and incubated at 37 °C for 1 h.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_the_DpnI_digested_products_of_the_performed_QuikChange_of_SDMI_of_P294_.28AlcR.29"> Analytical gelelectrophoresis of the DpnI digested products of the performed QuikChange of SDMI of P294 (AlcR) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of the DpnI digested products of the performed QuikChange of SDMI of P294 (AlcR).
</P><P><B>Procedure:</B></P><UL><LI> 4 µl of each of the DpnI digested QuikChange products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 50 min at an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TH>1 kbp DNA ladder
</TH><TH>DpnI digestion of P294QCI
</TH></TR><TR><TD><B>Result:</B></TD><TD>only Primers
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_DpnI_digested_QuikChange_product_P294QCI_and_P294_.28ReTrafo_of_FluA.29"> Transformation of <I>E. coli</I> XL1-blue with DpnI digested QuikChange product P294QCI and P294 (ReTrafo of FluA) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with DpnI digested QuikChange product P294QCI and P294 (ReTrafo of FluA).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products/2 µl of plasmid DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol/ampicillin plate.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P308_.28ReTrafo_of_P264.2C_PhyB_36AALinker_C-TEV_in_pSB1C3.29.2C_P310_.28ReTrafo_of_P274.2C_N-TEV_36AALinker_in_pSB1C3.29.2C_P303_.28SERK-SigP_in_pSB1C3.29.2C_P304_.28Strep-TEV-Linker_in_pSB1C3.29.2C_P305_.28Strep-TEV-Linker_in_pSB1C3.29_and_P301_.28ReTrafo_of_P7.2C_Phytotchrome_B.29">  Sequencing of P308 (ReTrafo of P264, PhyB_36AALinker_C-TEV in pSB1C3), P310 (ReTrafo of P274, N-TEV_36AALinker in pSB1C3), P303 (SERK-SigP in pSB1C3), P304 (Strep-TEV-Linker in pSB1C3), P305 (Strep-TEV-Linker in pSB1C3) and P301 (ReTrafo of P7, Phytotchrome B) </SPAN></H3><P><B>Investigators:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P308 (ReTrafo of P264, PhyB_36AALinker_C-TEV in pSB1C3), P310 (ReTrafo of P274, N-TEV_36AALinker in pSB1C3), P303 (SERK-SigP in pSB1C3), P304 (Strep-TEV-Linker in pSB1C3), P305 (Strep-TEV-Linker in pSB1C3) and P301 (ReTrafo of P7, Phytotchrome B).
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM).
</P><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P308
</TD><TD> PhyB_36AALinker_C-TEV in pSB1C3
</TD><TD> FR02305816 (O3)
</TD><TD> FR02305815 (O4)
</TD></TR><TR><TD> P310
</TD><TD> N-TEV_36AALinker in pSB1C3
</TD><TD> FR02305814 (O3)
</TD></TR><TR><TD> P303
</TD><TD> SERK-SigP in pSB1C3
</TD><TD> FR02305813 (O3)
</TD><TD> -
</TD></TR><TR><TD> P304
</TD><TD> Strep-TEV-Linker in pSB1C3
</TD><TD> FR02305812 (O3)
</TD><TD> -
</TD></TR><TR><TD> P305
</TD><TD> Strep-TEV-Linker in pSB1C3
</TD><TD> FR02305811 (O3)
</TD><TD> -
</TD></TR><TR><TD> P301
</TD><TD> Phytotchrome B
</TD><TD> FR02305810 (O3)
</TD><TD> FR02305809 (O4)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_ReTrafo_of_P160_.28SV40_NLS.29.2C_ReTrafo_of_P264_.28PhyB_36AALinker_C-TEV_in_pSB1C3.29.2C_ReTrafo_of_P274_.28N-TEV_36AALinker_in_pSB1C3.29_and_ReTrafo_of_P275_.28N-TEV_36AALinker_in_pSB1C3.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with ReTrafo of P160 (SV40 NLS), ReTrafo of P264 (PhyB_36AALinker_C-TEV in pSB1C3), ReTrafo of P274 (N-TEV_36AALinker in pSB1C3) and ReTrafo of P275 (N-TEV_36AALinker in pSB1C3)</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with ReTrafo of P160 (SV40 NLS), ReTrafo of P264 (PhyB_36AALinker_C-TEV in pSB1C3), ReTrafo of P274 (N-TEV_36AALinker in pSB1C3) and ReTrafo of P275 (N-TEV_36AALinker in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>New Tube</B></TD></TR><TR><TD> P160 I</TD><TD>P306
</TD></TR><TR><TD> P160 II</TD><TD>P307
</TD></TR><TR><TD> P264 I</TD><TD>P308
</TD></TR><TR><TD> P264 II</TD><TD>P309
</TD></TR><TR><TD> P274 I</TD><TD>P310
</TD></TR><TR><TD> P274 II</TD><TD>P311
</TD></TR><TR><TD> P275 I</TD><TD>P312
</TD></TR><TR><TD> P275 II</TD><TD>P313
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Pouring_of_CamR_gel_plates"> Pouring of CamR gel plates </SPAN></H3><P><B>Investigator:</B> Louise, Volker
</P><P><B>Aim of the experiment:</B> Pouring of CamR gel plates
</P><P><B>Procedure:</B> 
Preparation of 5 l of LB-Medium with agar was performed after standard recipe
</P></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_QC_I_of_P162_.28npt-casette_in_pSB1C3.29.2C_QC_V_of_P217_.28Laccase.29.2C_ReTrafo_of_P212.2C_ReTrafo_of_P238.2C_ReTrafo_of_P299._F77.2BF105_.28NucA.2BNLS.29.2C_F110.2BF108_.28Strep-tag-TEV-site-linker.2BSERK-TMD_8AAlinker_GFPmut1.29.2C_F109.2BF86_.28SERK-SigP.2BEreB.29.2C_F109.2BF87_.28SERK-SigP.2BNanoLuc.29.2C_F109.2BF88_.28SERK-SigP.2BSpyCatcher.29.2C_F109.2BF89_.28SERK-SigP.2BSpyTag.29.2C_F109.2BF97_.28SERK-SigP.2BxylE.29.2C_F109.2BF107_.28SERK-SigP.2BSERK-TMD.29"> Picking of QC I of P162 (npt-casette in pSB1C3), QC V of P217 (Laccase), ReTrafo of P212, ReTrafo of P238, ReTrafo of P299. F77+F105 (NucA+NLS), F110+F108 (Strep-tag-TEV-site-linker+SERK-TMD_8AAlinker_GFPmut1), F109+F86 (SERK-SigP+EreB), F109+F87 (SERK-SigP+NanoLuc), F109+F88 (SERK-SigP+SpyCatcher), F109+F89 (SERK-SigP+SpyTag), F109+F97 (SERK-SigP+xylE), F109+F107 (SERK-SigP+SERK-TMD) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Picking of QC I of P162 (npt-casette in pSB1C3), QC V of P217 (Laccase), ReTrafo of P212, ReTrafo of P238, ReTrafo of P299. F77+F105 (NucA+NLS), F110+F108 (Strep-tag-TEV-site-linker+SERK-TMD_8AAlinker_GFPmut1), F109+F86 (SERK-SigP+EreB), F109+F87 (SERK-SigP+NanoLuc), F109+F88 (SERK-SigP+SpyCatcher), F109+F89 (SERK-SigP+SpyTag), F109+F97 (SERK-SigP+xylE), F109+F107 (SERK-SigP+SERK-TMD)
</P><P><B>Procedure:</B></P><UL><LI> Retransformations (P212, P238, P299) 1 clone each
</LI><LI> Quickchanges (QCI P162, QCV P217) 2 clones each
</LI><LI> Ligation F77+F105 4 clones because of the high number of colonies at F77 negative ctrl
</LI><LI> All other Ligations (F110+F108, F109+F86, F109+F87, F109+F88, F109+F89, F109+F97, F109+F107) 2 clones each
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_20th"><B>Thursday, June 20th</B></SPAN></H2><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_QC_I_of_P162_.28npt-casette_in_pSB1C3.29.2C_QC_V_of_P217_.28Laccase.29.2C_ReTrafo_of_P212.2C_ReTrafo_of_P238.2C_ReTrafo_of_P299._F77.2BF105_.28NucA.2BNLS.29.2C_F110.2BF108_.28Strep-tag-TEV-site-linker.2BSERK-TMD_8AAlinker_GFPmut1.29.2C_F109.2BF86_.28SERK-SigP.2BEreB.29.2C_F109.2BF87_.28SERK-SigP.2BNanoLuc.29.2C_F109.2BF88_.28SERK-SigP.2BSpyCatcher.29.2C_F109.2BF89_.28SERK-SigP.2BSpyTag.29.2C_F109.2BF97_.28SERK-SigP.2BxylE.29.2C_F109.2BF107_.28SERK-SigP.2BSERK-TMD.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with QC I of P162 (npt-casette in pSB1C3), QC V of P217 (Laccase), ReTrafo of P212, ReTrafo of P238, ReTrafo of P299. F77+F105 (NucA+NLS), F110+F108 (Strep-tag-TEV-site-linker+SERK-TMD_8AAlinker_GFPmut1), F109+F86 (SERK-SigP+EreB), F109+F87 (SERK-SigP+NanoLuc), F109+F88 (SERK-SigP+SpyCatcher), F109+F89 (SERK-SigP+SpyTag), F109+F97 (SERK-SigP+xylE), F109+F107 (SERK-SigP+SERK-TMD)</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with QC I of P162 (npt-casette in pSB1C3), QC V of P217 (Laccase), ReTrafo of P212, ReTrafo of P238, ReTrafo of P299. F77+F105 (NucA+NLS), F110+F108 (Strep-tag-TEV-site-linker+SERK-TMD_8AAlinker_GFPmut1), F109+F86 (SERK-SigP+EreB), F109+F87 (SERK-SigP+NanoLuc), F109+F88 (SERK-SigP+SpyCatcher), F109+F89 (SERK-SigP+SpyTag), F109+F97 (SERK-SigP+xylE), F109+F107 (SERK-SigP+SERK-TMD).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Content</B></TH><TH><B>New Tube</B></TH></TR><TR><TD> QC I P162 I</TD><TD>P315
</TD></TR><TR><TD> QC I P162 II</TD><TD>P316
</TD></TR><TR><TD> QC V P217 I</TD><TD>P317
</TD></TR><TR><TD> QC V P217 II</TD><TD>P318
</TD></TR><TR><TD> P212 ReTrafo</TD><TD>P319
</TD></TR><TR><TD> P238 ReTrafo</TD><TD>P320
</TD></TR><TR><TD> P299 ReTrafo</TD><TD>P321
</TD></TR><TR><TD> F77+F105 I</TD><TD>P322
</TD></TR><TR><TD> F77+F105 II</TD><TD>P323
</TD></TR><TR><TD> F77+F105 III</TD><TD>P324
</TD></TR><TR><TD> F77+F105 IV</TD><TD>P325
</TD></TR><TR><TD> F110+F108 I</TD><TD>P326
</TD></TR><TR><TD> F110+F108 II</TD><TD>P327
</TD></TR><TR><TD> F109+F86 I</TD><TD>P328
</TD></TR><TR><TD> F109+F86 II</TD><TD>P329
</TD></TR><TR><TD> F109+F87 I</TD><TD>P330
</TD></TR><TR><TD> F109+F87 II</TD><TD>P331
</TD></TR><TR><TD> F109+F88 I</TD><TD>P332
</TD></TR><TR><TD> F109+F88 II</TD><TD>P333
</TD></TR><TR><TD> F109+F89 I</TD><TD>P334
</TD></TR><TR><TD> F109+F89 II</TD><TD>P335
</TD></TR><TR><TD> F109+F97 I</TD><TD>P336
</TD></TR><TR><TD> F109+F97 II</TD><TD>P337
</TD></TR><TR><TD> F109+F107 I</TD><TD>P338
</TD></TR><TR><TD> F109+F107 II</TD><TD>P339
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Retransformation_of_E..C2.A0coli_XL1_blue_with_P170.2C_P111.2C_P314.2C_P160.2C_P275.2C_P244.2C_P240.2C_P212.2C_P208.2C_P114.2C_P217.2C_P204.2C_P213_and_P222">  Retransformation of <I>E. coli</I> XL1 blue with P170, P111, P314, P160, P275, P244, P240, P212, P208, P114, P217, P204, P213 and P222 </SPAN></H3><P><B>Investigator: Johanna, Andi</B></P><P><B>Aim of the experiment:</B> Retransformation of <I>E. coli</I> XL1 blue with P170, P111, P314, P160, P275, P244, P240, P212, P208, P114, P217, P204, P213 and P222.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of ddH<SUB>2</SUB>O was added to empty tubes
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The transformated cells were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on chlorampenicol (P170, P111, P160, P275, P244, P240, P212, P208, P217, P204, P213, P222) and ampicillin plates (P314, P114).
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P320_.28ReTrafo_of_P238.2CSERK-TMD_8AAlinker_GFPmut1.29">  Sequencing of P320 (ReTrafo of P238,SERK-TMD_8AAlinker_GFPmut1)  </SPAN></H3><P><B>Investigators:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P320 (ReTrafo of P238,SERK-TMD_8AAlinker_GFPmut1).
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM).
</P><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P308
</TD><TD> SERK-TMD_8AAlinker_GFPmut1
</TD><TD> FR02305808 (O3)
</TD><TD> FR02305807 (O4)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_media_for_competent_cells"> Preparation of media for competent cells </SPAN></H3><P><B> Investigator : </B> Johanna, Louise
</P><P><B>Aim of the experiment:</B>Preparation of media for competent cells
</P><P><B>Procedure:</B></P><UL><LI> Picking of one colony of overnight stem culture of non competent XL1-Blue E.Coli in 50 ml LB-Medium without antibiotics (under the sterile bench)
</LI><LI> this was incubate overnight at 37 °C
</LI></UL><UL><LI> Batch for 1l of a 0.1 M MgCl<SUB>2</SUB>-solution
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>amount</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20.33 g
</TD><TD>MgCl<SUB>2</SUB> (M=203.3 g/mol)
</TD></TR><TR><TD>1 l
</TD><TD>ELGA H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><UL><LI> Batch for 500ml of a 50 mM CaCl<SUB>2</SUB>-solution
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>amount</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.67 g
</TD><TD>CaCl<SUB>2</SUB> (M=147.02 g/mol)
</TD></TR><TR><TD>500 ml
</TD><TD>ELGA H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><UL><LI> Batch for 50ml of a 50 mM CaCl<SUB>2</SUB>-solution, 15% v/v Glycerin
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>amount</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.368 g
</TD><TD>CaCl<SUB>2</SUB> (M=147.02 g/mol)
</TD></TR><TR><TD>0.458 g
</TD><TD>Glycerin (density=1.26)
</TD></TR><TR><TD>50 ml
</TD><TD>ELGA H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><UL><LI> Autoclave each bottle of solution and leave each in a 4°C room.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="QuikChanges_QC_pActin_and_QC1_AlcR"> QuikChanges QC pActin and QC1 AlcR </SPAN></H3><P><B> Investigator : </B> Andi
</P><P><B>Aim of the experiment:</B> QuikChanges QC pActin (P114) and QC1 AlcR (P294)
</P><P><B>Procedure:</B> Quickchange - PCR
</P><P><B>Reaction batch 1 for P114 (pActin) with Pfu Ultra II</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P114 undiluted)
</TD><TD>4.5 ng in 25 µl batch
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O76 (=10 µM)
</TD><TD>0.5 µl Primer + 4.5 µl ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O77 (=10 µM)
</TD><TD>0.5 µl Primer + 4.5 µl ddH2O
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P114 (pActin)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>4
</TD><TD>68 °C
</TD><TD>6.5 min
</TD></TR><TR><TD>5
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2 for P114 (pActin) with Q5 Polymerase Mastermix</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>12.5 µl
</TD><TD>2x QS Polymerase Mastermix
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P114 - 1:10 dilution)
</TD><TD>0.45 ng in 25 µl batch
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O76 (=10 µM)
</TD><TD>0.5 µl Primer + 4.5 µl ddH2O
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O77 (=10 µM)
</TD><TD>0.5 µl Primer + 4.5 µl ddH2O
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P114 (pActin) with Q5 Polymerase</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>55 °C
</TD><TD>1 min
</TD></TR><TR><TD>4
</TD><TD>72 °C
</TD><TD>3.25 min
</TD></TR><TR><TD>5
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P294 (AlcR) with Pfu Ultra II</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD><TD><B>Additional information</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid template (P294 dilutsion 1:100)
</TD><TD>2.79 ng in 25 µl batch
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O76 (=10 µM)
</TD><TD>0.5 µl Primer + 4.5 µl ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O77 (=10 µM)
</TD><TD>0.5 µl Primer + 4.5 µl ddH2O
</TD></TR><TR><TD>18.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P294 (AlcR)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>3
</TD><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>4
</TD><TD>68 °C
</TD><TD>6.5 min
</TD></TR><TR><TD>5
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><UL><LI> After the PCR, the Quickchange products of P114 (pActin) with Q5 Mastermix and P294 (AlcR)were mixed with 1 µl of DpnI (10 U/µl) and incubated at 37 °C for 1 h.
</LI></UL><UL><LI>After the PCR, the Quickchange products of P114 (pActin) with PfU II Polymerase was stored in the Cycler at 4 °C over night
</LI></UL></DIV></DIV><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_DpnI_digested_QuikChange_product_P114_.28pActin.29_and_P294_.28AlcR.29"> Transformation of <I>E. coli</I> XL1-blue with DpnI digested QuikChange product P114 (pActin) and P294 (AlcR) </SPAN></H3><P><B>Investigator:</B> Master of Quickchange, Gel-Penetrator
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with DpnI digested QuikChange product P294 (AlcR) QCI and P114 (pActin).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products/2 µl of plasmid DNA was added to 200 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol/ampicillin plate.
</LI></UL></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR_of_P317_.28Laccase.29_with_Q5_Mastermix_and_Herculase_Polymerase"> PCR of P317 (Laccase) with Q5 Mastermix and Herculase Polymerase </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B>  PCR of P317 (Laccase) with Q5 Mastermix and Herculase Polymerase
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI>Reaction batch with Herculase Polymerase
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x Herculase Fusion Polymerase reaction buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1.25 µl
</TD><TD>10 µM Forward Primer O24 (Bpul_for)
</TD></TR><TR><TD>1.25 µl
</TD><TD>10 µM Reverse Primer O25 (Bpul_rev)
</TD></TR><TR><TD>1 µL
</TD><TD>Herculase II Fusion Polymerase
</TD></TR><TR><TD>1.5 µl
</TD><TD>Plasmid DNA (P317)- 1:500 dilution
</TD></TR><TR><TD>0.5 µl
</TD><TD>DMSO
</TD></TR><TR><TD>32 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Reaction batch with Q5 Mastermix
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O24 (Bpul_for)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O25 (Bpul_rev)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P317)- 1:500 dilution
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>51 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>90 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After the PCR has finished, the product was hold at 4 °C in the Cycler 
</LI></UL></DIV><DIV class="safety mechanism"><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_the_DpnI_digested_products_of_the_performed_QuickChange_of_SDMI_of_P294_.28AlcR.29and_P114_.28pActin.29"> Analytical gelelectrophoresis of the DpnI digested products of the performed QuickChange of SDMI of P294 (AlcR)and P114 (pActin) </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of the DpnI digested products of the performed QuikChange of SDMI of P294 (AlcR)and P114 (pActin.)
</P><P><B>Procedure:</B></P><UL><LI> 9 µl of each of the DpnI digested QuikChange products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 50 min at an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TH>100 bp DNA ladder
</TH><TH>DpnI digestion of P294QCI
</TH><TH>DpnI digestion of P114QCI
</TH><TH>100 bp DNA ladder
</TH></TR><TR><TD><B>Result:</B></TD><TD>expected result
</TD><TD>only Primers
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P315_and_P316_.28QC_I_P162_NPT-casette.29.2C_P162_.28npt-casette_in_pSB1C3.29.2C_P317_and_P318_.28QC_V_P217_Laccase.29.2C_P217_.28Laccase.29.2C_P322_.28F77.2BF105_NucA.2BNLS.29.2C_P248_.28Thermonuc_in_pSB1C3.29.2C_P320_.28SERK-TMD_8AAlinker_GFPmut1.29.2C_P326.2BP327_.28Strep-tag-TEV-site-linker.2BSERK-TMD_8AAlinker_GFPmut1.29.2C_P328.2BP329_.28SERK-SigP.2BEreB.29.2C_P157_.28EreB.29.2C_P330.2BP331_.28SERK-SigP.2BNanoLuc.29.2C_P208_.28NanoLuc.29.2C_P332.2BP333_.28SERK-SigP.2BSpyCatcher.29.2C_P213_.28SpyCatcher.29.2C_P334.2B335_.28SERK-SigP.2BSpyTag.29.2C_P170_.28SpyTag.29.2C_P336.2BP337_.28SERK-SigP.2BxylE.29.2C_P197_.28xylE.29.2C_P338.2BP339_.28SERK-SigP.2BSERK-TMD.29.2C_P212.28SERK-TMD.29"> Analytical digestion and gelelectrophoresis  of P315 and P316 (QC I P162 NPT-casette), P162 (npt-casette in pSB1C3), P317 and P318 (QC V P217 Laccase), P217 (Laccase), P322 (F77+F105 NucA+NLS), P248 (Thermonuc in pSB1C3), P320 (SERK-TMD_8AAlinker_GFPmut1), P326+P327 (Strep-tag-TEV-site-linker+SERK-TMD_8AAlinker_GFPmut1), P328+P329 (SERK-SigP+EreB), P157 (EreB), P330+P331 (SERK-SigP+NanoLuc), P208 (NanoLuc), P332+P333 (SERK-SigP+SpyCatcher), P213 (SpyCatcher), P334+335 (SERK-SigP+SpyTag), P170 (SpyTag), P336+P337 (SERK-SigP+xylE), P197 (xylE), P338+P339 (SERK-SigP+SERK-TMD), P212(SERK-TMD) </SPAN></H3><P><B>Investigator:</B> Rosario, Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of P315 and P316 (QC I P162 NPT-casette), P162 (npt-casette in pSB1C3), P317 and P318 (QC V P217 Laccase), P217 (Laccase), P322 (F77+F105 NucA+NLS), P248 (Thermonuc in pSB1C3), P320 (SERK-TMD_8AAlinker_GFPmut1), P326+P327 (Strep-tag-TEV-site-linker+SERK-TMD_8AAlinker_GFPmut1), P328+P329 (SERK-SigP+EreB), P157 (EreB), P330+P331 (SERK-SigP+NanoLuc), P208 (NanoLuc), P332+P333 (SERK-SigP+SpyCatcher), P213 (SpyCatcher), P334+335 (SERK-SigP+SpyTag), P170 (SpyTag), P336+P337 (SERK-SigP+xylE), P197 (xylE), P338+P339 (SERK-SigP+SERK-TMD), P212(SERK-TMD)
</P><P><B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE border="1"><TBODY><TR><TH>1 kb DNA Ladder
</TH><TH>P315
</TH><TH>P316
</TH><TH>P162 (Control)
</TH><TH>P317
</TH><TH>P318
</TH><TH>P217 (Control)
</TH><TH>P322
</TH><TH>P323
</TH><TH>P324
</TH><TH>P325
</TH><TH>P248 (Control)
</TH></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>1 kb DNA Ladder
</TH><TH>P334
</TH><TH>P335
</TH><TH>P170 (Control)
</TH><TH>P336
</TH><TH>P337
</TH><TH>P197 (Control)
</TH><TH>P338
</TH><TH>P339
</TH><TH>P212 (Control)
</TH><TH>P320
</TH></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>is repeated on other gel
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>1 kb DNA Ladder
</TH><TH>P320 (Control)
</TH><TH>P326
</TH><TH>P327
</TH></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH>1 kb DNA Ladder
</TH><TH>P328
</TH><TH>P329
</TH><TH>P157 (Control)
</TH><TH>P330
</TH><TH>P331
</TH><TH>P208 (Control)
</TH><TH>P332
</TH><TH>P333
</TH><TH>P213 (Control)
</TH><TH>P326
</TH><TH>P327
</TH></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>is repeated on other gel
</TD><TD>is repeated on other gel
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_21st"><B>Friday, June 21st</B></SPAN></H2><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_the_PCR_product_of_P317_.28Laccse.29_with_PfU_II_Polymerase.2C_Herculase_Polymerase_and_the_DpnI_digested_products_of_the_performed_QuickChange_of_P114_.28pActin.29"> Analytical gelelectrophoresis of the PCR product of P317 (Laccse) with PfU II Polymerase, Herculase Polymerase and the DpnI digested products of the performed QuickChange of P114 (pActin) </SPAN></H3><P><B>Investigator:</B> Master of QuikChange
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of the PCR product of P317 (Laccse) with PfU II Polymerase, Herculase Polymerase and the DpnI digested products of the performed QuickChange of P114 (pActin)
</P><P><B>Procedure:</B></P><UL><LI> 9 µl of each of the PCR products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> 9 µl of each of the DpnI digested QuikChange product was mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 50 min at an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TH>1000 bp DNA ladder
</TH><TH>PCR product of P317 with Q5 Master Mix
</TH><TH>PCR product of P317 with Herculase Polymerase
</TH><TH>DpnI digestion of P114
</TH></TR><TR><TD><B>Result:</B></TD><TD>expected result
</TD><TD>expected result
</TD><TD>expected result
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_competent_cells_according_to_the_CaCl_2-method_.28Cohen_et_al..2C1972.29"> Preparation of competent cells according to the CaCl<SUB> 2</SUB>-method (Cohen et al.,1972)</SPAN></H3><P><B>Investigator:</B> Johanna, Louise
</P><P><B>Aim of the experiment:</B> Preparation of competent cells according to the CaCl<SUB> 2</SUB>-method (Cohen et al.,1972)
</P><P><B>Procedure:</B></P><UL><LI>Add Tetracyclin (ratio: 1:1000) to the 2YT-medium 
</LI><LI>Measure the OD<SUB>550</SUB> of it until OD<SUB>550</SUB>=0.5
</LI><LI>Put the culture in Falcon tubes (leaving on ice)
</LI><LI>Centrifuge the falcon tubes by 5000 rpm, 4°C, 10 min
</LI><LI>Resuspend the falcon tubes each with 40 ml 0,1 M MgCl<SUB>2</SUB> solution after removing the supernatants
</LI><LI>Centrifuge the falcon tubes by 5000 rpm, 4°C, 10 min
</LI><LI>Resuspend the falcon tubes each with 20 ml 50mM CaCl<SUB>2</SUB> solution after removing the supernatants
</LI><LI>Incubate the falcon tubes on ice for 30 min
</LI><LI>Centrifuge the falcon tubes by 5000 rpm, 4°C, 10 min
</LI><LI>Resuspend the falcon tubes each with 2 ml 50mM CaCl<SUB>2</SUB>, 15 % v/v Glycerin solution after removing the supernatants. From this step everything needs to be carry out in a 4°C room.
</LI><LI>Mix the solutions of each falcon tube and transfer the content in eppis (each with 100 and accordingly 150 µl)
</LI><LI>Store the boxes of eppis at -80 °C
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_Digestion_of_P308_.28PhyB-Linker-C-TEV.29.2C_P299_.28IRES.29.2C_P111_.28TEV.29.2C_P326_.28TEV-Linker-TM-GFP.29.2C_P328_.28SERKSigP_EreB.29.2C_P330_.28SERK-SigP_NLuc.29.2C_P332_.28SERK-SigP_SpyCatcher.29.2C_P334_.28SERK-SigP_SpyTag.29.2C_P336_.28SERK-SigP_XylE.29.2C_P338_.28SERK-SigP_SERK-TMD.29.2C_P8_.28pSB1C3.29_and_PCR_product_of_Laccase_.28F112.29"> Preparative Digestion of P308 (PhyB-Linker-C-TEV), P299 (IRES), P111 (TEV), P326 (TEV-Linker-TM-GFP), P328 (SERKSigP_EreB), P330 (SERK-SigP_NLuc), P332 (SERK-SigP_SpyCatcher), P334 (SERK-SigP_SpyTag), P336 (SERK-SigP_XylE), P338 (SERK-SigP_SERK-TMD), P8 (pSB1C3) and PCR product of Laccase (F112)  </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Peparative Digestion of P308 (PhyB-Linker-C-TEV), P299 (IRES), P111 (TEV), P326 (TEV-Linker-TM-GFP), P328 (SERKSigP_EreB), P330 (SERK-SigP_NLuc), P332 (SERK-SigP_SpyCatcher), P334 (SERK-SigP_SpyTag), P336 (SERK-SigP_XylE), P338 (SERK-SigP_SERK-TMD) and P8 (PhyB)
</P><P><B>Procedure:</B></P><P><B>Batch for preparative digestion of effectors (P328, P330, P332, P334, P336, P338) with AgeI and PstI)</B>
(7x Mastermix)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>28 µl
</TD><TD>Buffer 4 (10x)
</TD></TR><TR><TD>7 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>98 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=140 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 20 µl Mastermix + 20 µl Plasmid each
</LI></UL><P><B>Batches for preparative digestions:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>4 µl
</TD><TD>Buffer 4 (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 1
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 2
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>Name</B></TD><TD><B>Enzyme 1</B></TD><TD><B>Enzyme 2</B></TD></TR><TR><TD>P308
</TD><TD>PhyB-Linker-C-TEV
</TD><TD>SpeI
</TD><TD>PstI
</TD></TR><TR><TD>P299
</TD><TD>IRES
</TD><TD>XbaI
</TD><TD>PstI
</TD></TR><TR><TD>P111
</TD><TD>TEV
</TD><TD>SpeI
</TD><TD>PstI
</TD></TR><TR><TD>P326
</TD><TD>TEV-Linker-TM-GFP
</TD><TD>NgoMIV
</TD><TD>PstI
</TD></TR><TR><TD>P8
</TD><TD>pSB1C3
</TD><TD>XbaI
</TD><TD>AgeI
</TD></TR><TR><TD>F112
</TD><TD>Laccase
</TD><TD>XbaI
</TD><TD>AgeI
</TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and they were loaded on a 1% agarose gel
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Name:</B></TD><TD>P308 dig. SpeI + PstI
</TD><TD>1kb DNA ladder
</TD><TD>P299 dig. XbaI + PstI
</TD><TD>P111 dig. SpeI + PstI
</TD></TR><TR><TD><B>cut out:</B></TD><TD>Backbone
</TD><TD>Insert (lower band)
</TD><TD>Backbone
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Name:</B></TD><TD>P326 dig. NgoMIV + PstI
</TD><TD>1kb DNA ladder
</TD><TD>P328 dig. AgeI + Pst
</TD><TD>P330 dig. AgeI + PstI
</TD></TR><TR><TD><B>cut out:</B></TD><TD>Insert (lower band)
</TD><TD>Backbone
</TD><TD>Backbone
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Name:</B></TD><TD>P332 dig. AgeI + PstI
</TD><TD>1kb DNA ladder
</TD><TD>P334 dig. AgeI + PstI
</TD><TD>P336 dig. AgeI + PstI
</TD></TR><TR><TD><B>cut out:</B></TD><TD>Backbone
</TD><TD>Backbone
</TD><TD>Backbone
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Name:</B></TD><TD>P338 dig. AgeI + PstI
</TD><TD>1kb DNA ladder
</TD><TD>P8 dig. AgeI + XbaI
</TD></TR><TR><TD><B>cut out:</B></TD><TD>Backbone
</TD><TD>Backbone (<B>lower</B> band)
</TD></TR></TBODY></TABLE><UL><LI> Extracted via QIAGEN gel extraction kit, digested PCR product purified via PCR Purification Kit
</LI></UL></DIV></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F116.2BF117_.28TEV-Linker-TM-GFP.2BSERKSigP_EreB.29.2C_F116.2BF118_.28TEV-Linker-TM-GFP.2BSERK-SigP_NLuc.29.2C_F116.2BF119_.28TEV-Linker-TM-GFP.2BSERK-SigP_SpyCatcher.29.2C_F116.2BF120_.28TEV-Linker-TM-GFP.2BSERK-SigP_SpyTag.29.2C_F116.2BF121_.28TEV-Linker-TM-GFP.2BSERK-SigP_XylE.29.2C_F123.2BF124_.28Laccase.2BpSB1C3.29"> Ligation of F116+F117 (TEV-Linker-TM-GFP+SERKSigP_EreB), F116+F118 (TEV-Linker-TM-GFP+SERK-SigP_NLuc),  F116+F119 (TEV-Linker-TM-GFP+SERK-SigP_SpyCatcher), F116+F120 (TEV-Linker-TM-GFP+SERK-SigP_SpyTag), F116+F121 (TEV-Linker-TM-GFP+SERK-SigP_XylE), F123+F124 (Laccase+pSB1C3) </SPAN></H3><P><B>Investigator:</B>  Katrin
</P><P><B>Aim of the experiment:</B> Ligation of F116+F117 (TEV-Linker-TM-GFP+SERKSigP_EreB), F116+F118 (TEV-Linker-TM-GFP+SERK-SigP_NLuc),  F116+F119 (TEV-Linker-TM-GFP+SERK-SigP_SpyCatcher), F116+F120 (TEV-Linker-TM-GFP+SERK-SigP_SpyTag), F116+F121 (TEV-Linker-TM-GFP+SERK-SigP_XylE), F123+F124 (Laccase+pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F116+F117 (TEV-Linker-TM-GFP+SERKSigP_EreB) 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>3.52 µl
</TD><TD>F117 (28.4 ng/µl, 3413 bp)
</TD></TR><TR><TD>4.81 µl
</TD><TD>F116 (18.2 ng/µl, 996 bp)
</TD></TR><TR><TD>8.67 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F116+F118 (TEV-Linker-TM-GFP+SERK-SigP_NLuc)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.89 µl
</TD><TD>F118 (52.8 ng/µl, 2663 bp)
</TD></TR><TR><TD>6.16 µl
</TD><TD>F116 (18.2 ng/µl, 996 bp)
</TD></TR><TR><TD>8.95nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for  F116+F119 (TEV-Linker-TM-GFP+SERK-SigP_SpyCatcher)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.57 µl
</TD><TD>F119 (38.0 ng/µl, 2492 bp)
</TD></TR><TR><TD>6.59 µl
</TD><TD>F116 (18.2 ng/µl, 996 bp)
</TD></TR><TR><TD>7.84nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F116+F120 (TEV-Linker-TM-GFP+SERK-SigP_SpyTag)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.8 µl
</TD><TD>F120 (62.5 ng/µl, 2192 bp)
</TD></TR><TR><TD>3.75 µl
</TD><TD>F116 (18.2 ng/µl, 996 bp)
</TD></TR><TR><TD>3.96nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F116+F121 (TEV-Linker-TM-GFP+SERK-SigP_XylE)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.73 µl
</TD><TD>F121 (68.3 ng/µl, 3071 bp)
</TD></TR><TR><TD>2.56 µl
</TD><TD>F116 (18.2 ng/µl, 996 bp)
</TD></TR><TR><TD>5.95nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for  F123+F124 (Laccase+pSB1C3)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.03 µl
</TD><TD>F123 (49.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>3.80 µl
</TD><TD>F124 (58.9 ng/µl, 1555 bp)
</TD></TR><TR><TD>11.17nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_products_F116.2BF117_.28TEV-Linker-TM-GFP.2BSERKSigP_EreB.29.2C_F116.2BF118_.28TEV-Linker-TM-GFP.2BSERK-SigP_NLuc.29.2C_F116.2BF119_.28TEV-Linker-TM-GFP.2BSERK-SigP_SpyCatcher.29.2C_F116.2BF120_.28TEV-Linker-TM-GFP.2BSERK-SigP_SpyTag.29.2C_F116.2BF121_.28TEV-Linker-TM-GFP.2BSERK-SigP_XylE.29.2C_F123.2BF124_.28Laccase.2BpSB1C3.29_and_negative_controls"> Transformation of <I>E. coli</I> XL1-blue with ligation products F116+F117 (TEV-Linker-TM-GFP+SERKSigP_EreB), F116+F118 (TEV-Linker-TM-GFP+SERK-SigP_NLuc),  F116+F119 (TEV-Linker-TM-GFP+SERK-SigP_SpyCatcher), F116+F120 (TEV-Linker-TM-GFP+SERK-SigP_SpyTag), F116+F121 (TEV-Linker-TM-GFP+SERK-SigP_XylE), F123+F124 (Laccase+pSB1C3) and negative controls  </SPAN></H3><P>Investigator: Jeff, Katrin
</P><P>Aim of the experiment: Transformation of E. coli XL1-blue with ligation products F116+F117 (TEV-Linker-TM-GFP+SERKSigP_EreB), F116+F118 (TEV-Linker-TM-GFP+SERK-SigP_NLuc),  F116+F119 (TEV-Linker-TM-GFP+SERK-SigP_SpyCatcher), F116+F120 (TEV-Linker-TM-GFP+SERK-SigP_SpyTag), F116+F121 (TEV-Linker-TM-GFP+SERK-SigP_XylE), F123+F124 (Laccase+pSB1C3), QCI(Insertion) of pActin and negative controls
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products/2 µl of plasmid DNA was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol/ampicillin plate.
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P303_.28SERK-SigP_in_PsB1C3.2C_fw.29.2C_P310_.28N-TEV_36AALinker_in_pSB1C3.2C_rev.29.2C_P248_.28Thermonuc_in_pSB1C3.2C_fw.29.2C_P312_.28N-TEV_36AALinker_in_pSB1C3.2C_fw.29.2C_P316_.28npt-casette_after_QCI.2C_fw.2Crev.29.2C_P318_.28Laccase_after_QCV.2C_still_RFC10.2C_fw.2C_rev.29.2C_P320_.28SERK-TMD_8AAlinker_GFPmut1.2C_fw.2C_rev.29.2C_P326_.28Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.2C_fw.2C_rev.29.2C_P328_.28SERK-SigP_EreB.2C_fw.2C_rev.29.2C_P330_.28SERK-SigP_NanoLuc.2C_fw.29.2C_P332_.28SERK-SigP_SpyCatcher.2C_fw.29.2C_P334_.28SERK-SigP_SpyTag.2C_fw.29.2C_P336_.28SERK-SigP_XylE.2C_fw.2C_rev.29.2C_P338_.28SERK-SigP_SERK-TMD.2C_fw.29">  Sequencing of P303 (SERK-SigP in PsB1C3, fw), P310 (N-TEV_36AALinker in pSB1C3, rev), P248 (Thermonuc in pSB1C3, fw), P312 (N-TEV_36AALinker in pSB1C3, fw), P316 (npt-casette after QCI, fw,rev), P318 (Laccase after QCV, still RFC10, fw, rev), P320 (SERK-TMD_8AAlinker_GFPmut1, fw, rev), P326 (Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1, fw, rev), P328 (SERK-SigP_EreB, fw, rev), P330 (SERK-SigP_NanoLuc, fw), P332 (SERK-SigP_SpyCatcher, fw), P334 (SERK-SigP_SpyTag, fw), P336 (SERK-SigP_XylE, fw, rev), P338 (SERK-SigP_SERK-TMD, fw) </SPAN></H3><P><B>Investigators:</B> Louise, Katrin
</P><P><B>Aim of the experiment:</B> Sequencing of P303 (SERK-SigP in PsB1C3, fw), P310 (N-TEV_36AALinker in pSB1C3, rev), P248 (Thermonuc in pSB1C3, fw), P312 (N-TEV_36AALinker in pSB1C3, fw), P316 (npt-casette after QCI, fw,rev), P318 (Laccase after QCV, still RFC10, fw, rev), P320 (SERK-TMD_8AAlinker_GFPmut1, fw, rev), P326 (Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1, fw, rev), P328 (SERK-SigP_EreB, fw, rev), P330 (SERK-SigP_NanoLuc, fw), P332 (SERK-SigP_SpyCatcher, fw), P334 (SERK-SigP_SpyTag, fw), P336 (SERK-SigP_XylE, fw, rev), P338 (SERK-SigP_SERK-TMD, fw).
</P><P><B>Procedure:</B> Peparation of DNA  for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng/µl) and 2 µl sequencing primer 10 µM).
</P><P>
The different genes received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Name
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P303
</TD><TD> SERK-SigP
</TD><TD> FR02305805 (O3)
</TD></TR><TR><TD> P310
</TD><TD> N-TEV_36AAlinker
</TD><TD>  FR02305806 (O4)
</TD></TR><TR><TD> P248
</TD><TD> Thermonuc.
</TD><TD> FR02305804 (O3)
</TD></TR><TR><TD> P312
</TD><TD> NTEV_36aaLinker
</TD><TD> FR02305803 (O3)
</TD></TR><TR><TD> P316
</TD><TD> npt-casette QCI
</TD><TD> FR02305802 (O3)
</TD><TD> FR02305801 (O4)
</TD></TR><TR><TD> P318
</TD><TD> Laccase QCV
</TD><TD> FR02305800 (O3)
</TD><TD> FR02305799 (O4)
</TD></TR><TR><TD> P320
</TD><TD> SERK-TM_8aaLinker_GFPmut1
</TD><TD> FR02305798 (O3)
</TD><TD> FR02305797 (O4)
</TD></TR><TR><TD> P326
</TD><TD> Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1
</TD><TD> FR02305796 (O3)
</TD><TD> FR02305795 (O4)
</TD></TR><TR><TD> P328
</TD><TD> SERK-SigP_EreB
</TD><TD> FR02305794 (O3)
</TD><TD> FR02305793 (O4)
</TD></TR><TR><TD> P330
</TD><TD> SERK-SigP_Nanoluc
</TD><TD> FR02305792 (O3)
</TD></TR><TR><TD> P332
</TD><TD> SERK-SigP_Spycatcher
</TD><TD> FR02305791 (O3)
</TD></TR><TR><TD> P334
</TD><TD> SERK-SigP_Spy-tag
</TD><TD> FR02305790 (O3)
</TD></TR><TR><TD> P336
</TD><TD> SERK-SigP_XylE
</TD><TD> FR02305789 (O3)
</TD><TD> FR02305788 (O4)
</TD></TR><TR><TD> P338
</TD><TD> SERK-SigP_SERK-TM
</TD><TD> FR02305787 (O3)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_P61_to_get_the_IRES_of_poliovirus_1_Mahoney_strain"> PCR of P61 to get the IRES of poliovirus 1 Mahoney strain </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  PCR of P61 to get the IRES of poliovirus 1 Mahoney strain.
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> Reaction batch with Q5 Mastermix:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O68 (IRES_polio_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O69 (IRES_polio_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P61)- 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>63 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>20 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_Re-Transformations_P111_.28TEV_in_pSB1C3.2C_Re-Trafo.29.2C_P160_.28Sv40_NLS_in_pSB1C3.29.2C_P114_.28pActin.29.2C_P314_.28FluA.29.2C_Glutathione_S-transferase_.28GST.2C_BBa_K620000.29.2C_QCI_AlcR"> Picking of Re-Transformations P111 (TEV in pSB1C3, Re-Trafo), P160 (Sv40 NLS in pSB1C3), P114 (pActin), P314 (FluA), Glutathione S-transferase (GST, BBa_K620000), QCI AlcR </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Picking of Re-Transformations P111 (TEV in pSB1C3, Re-Trafo), P160 (Sv40 NLS in pSB1C3), P114 (pActin), P314 (FluA), Glutathione S-transferase (GST, BBa_K620000), QCI AlcR.
</P><P><B>Procedure:</B></P><UL><LI> Retransformations  1 clone each
</LI><LI> Resuspendsion in 4 ml LB-medium with  4 µl chloramphenicol (P111, P160) or ampicillin (P114, P314)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_22nd"><B>Saturday, June 22nd</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="QuikChange_of_npt-casette_.28P316.2C_QCII.29_and_of_AlcR_.28P342.2C_QCII.29"> QuikChange of npt-casette (P316, QCII) and of AlcR (P342, QCII) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> QuikChange of npt-casette (P316, QCII) and of AlcR (P342, QCII).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P316 (npt-casette after QCI)</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH><TH><B>Additional information</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P316 - 1:100 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O72 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O73 (=10 µM)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch for P342 (AlcR after QCI):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH><TH><B>Additional information</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P342 - 1:100 dilution)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O82 (10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O83 (10 µM)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters - P316 (npt-casette after QCI) and P342 (AlcR after QCI):</B></P><TABLE border="1"><TBODY><TR><TH><B>Segment</B></TH><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>1</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>3
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_P111.28Retrafo.29.2C_QCI_AlcR.2C_P114.28Retrafo.29.2C_P314_.28Retrafo.29.2C_P160_.28Retrafo.29.2C_Glutathione_S-transferase_.28GST.2C_BBa_K620000.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P111(Retrafo), QCI AlcR, P114(Retrafo), P314 (Retrafo), P160 (Retrafo), Glutathione S-transferase (GST, BBa_K620000)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P111(Retrafo), QCI AlcR, P114(Retrafo), P314 (Retrafo), P160 (Retrafo), Glutathione S-transferase (GST, BBa_K620000).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_QCII_AlcR_.28P341.2C_P342.29_and_control_P294_with_EcoRI_and_PstI.2C_gelelectrophoresis_of_PCR_of_IRES"> Analytical digestion and gelelectrophoresis  of QCII AlcR (P341, P342) and control P294 with EcoRI and PstI, gelelectrophoresis of PCR of IRES</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis  of QCII AlcR (P341, P342) and control P294 with EcoRI and PstI
</P><P><B>Procedure:</B></P><UL><LI> Batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P341, P342; P249)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kb DNA Ladder
</TD><TD>PCR of IRES
</TD><TD>Digestion of P249 with EcoRI and PstI (control)
</TD><TD>Digestion of P341 with EcoRI and PstI
</TD><TD>Digestion of P342 with EcoRI and PstI
</TD></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_Digestion_of_P310_.28N-Tev-linker.29.2C_P249_.28Thermonuclease.29.2C_P160_.28SV40_NLS.29.2C_PCR_of_IRES"> Preparative Digestion of P310 (N-Tev-linker), P249 (Thermonuclease), P160 (SV40 NLS), PCR of IRES  </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Preparative Digestion of P310 (N-Tev-linker), P249 (Thermonuclease), P160 (SV40 NLS),  PCR of IRES
</P><P><B>Procedure:</B></P><P><B>Preparative digestion of P310 (N-Tev-linker)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P310
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Preparative digestion of P160 (SV40 NLS)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P160
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Preparative digestion of P249 (Thermonuclease)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P249
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Preparative digestion of PCR of IRES)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR product
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and they were loaded on a 1% agarose gel
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Name:</B></TD><TD>P249 dig. NgoMIV + PstI
</TD><TD>1kb DNA ladder
</TD><TD>P160 dig. AgeI + PstI
</TD><TD>P111 dig. SpeI + AgeI
</TD></TR><TR><TD><B>cut out:</B></TD><TD>Insert(lower band)
</TD><TD>Backbone
</TD><TD>Backbone
</TD></TR></TBODY></TABLE><UL><LI> Extracted via QIAGEN gel extraction kit, digested PCR product purified via PCR Purification Kit
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F126.2BF127_.28NLS_in_pSB1C3.2BThermonuclease.29.2C_F125.2BF96_.28N-TEV-linker_in_pSB1C3.2BPIF3.29.2C_F125.2BF98_.28N-TEV-linker_in_pSB1C3.2BPIF6.29.2C_F124.2BF41_.28IRES.2BpSB1C3.29"> Ligation of F126+F127 (NLS in pSB1C3+Thermonuclease), F125+F96 (N-TEV-linker in pSB1C3+PIF3), F125+F98 (N-TEV-linker in pSB1C3+PIF6), F124+F41 (IRES+pSB1C3) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Ligation of F126+F127 (NLS in pSB1C3+Thermonuclease), F125+F96 (N-TEV-linker in pSB1C3+PIF3), F125+F98 (N-TEV-linker in pSB1C3+PIF6), F124+F41 (IREs+pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F126+F127 (NLS in pSB1C3+Thermonuclease 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>7.2 µl
</TD><TD>F126 (13.9 ng/µl, 2143 bp)
</TD></TR><TR><TD>4.1 µl
</TD><TD>F127 (15.2 ng/µl, 447 bp)
</TD></TR><TR><TD>5.7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F125+F96 (N-TEV-linker in pSB1C3+PIF3)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>6.9 µl
</TD><TD>F125 (14.5 ng/µl, 2554 bp)
</TD></TR><TR><TD>2.3 µl
</TD><TD>F96 (15.1 ng/µl, 297 bp)
</TD></TR><TR><TD>7.8nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F124+F41 (IREs+pSB1C3)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.3 µl
</TD><TD>F124 (70.8 ng/µl, 628 bp)
</TD></TR><TR><TD>1.2 µl
</TD><TD>F41 (81.1 ng/µl, 2086 bp)
</TD></TR><TR><TD>14.5nbsp;µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_products_F126.2BF127_.28NLS_in_pSB1C3.2BThermonuclease.29.2C_F125.2BF96_.28N-TEV-linker_in_pSB1C3.2BPIF3.29.2C_F125.2BF98_.28N-TEV-linker_in_pSB1C3.2BPIF6.29.2C_F124.2BF41_.28IRES.2BpSB1C3.29_and_negative_controls"> Transformation of <I>E. coli</I> XL1-blue with ligation products F126+F127 (NLS in pSB1C3+Thermonuclease), F125+F96 (N-TEV-linker in pSB1C3+PIF3), F125+F98 (N-TEV-linker in pSB1C3+PIF6), F124+F41 (IRES+pSB1C3) and negative controls</SPAN></H3><P><B>Investigator:</B> Volker
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation products F126+F127 (NLS in pSB1C3+Thermonuclease), F125+F96 (N-TEV-linker in pSB1C3+PIF3), F125+F98 (N-TEV-linker in pSB1C3+PIF6), F124+F41 (IRES+pSB1C3) and negative controls
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products/2 µl of plasmid DNA was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_F116.2BF121.2C_F116.2BF117.2C_F123.2BF124.2C_F116.2BF119.2C_F116.2BF120.2C_F116.2BF118.2C_QCI_.28insertion_of_P114_.28pActin.29"> Picking of F116+F121, F116+F117, F123+F124, F116+F119, F116+F120, F116+F118, QCI (insertion of P114 (pActin) </SPAN></H3><P><B>Investigator:</B> Katrin
<B>Aim of the experiment:</B> Picking of F116+F121, F116+F117, F123+F124, F116+F119, F116+F120, F116+F118, QCI (insertion of P114 (pActin)
</P><P><B>Procedure:</B></P><UL><LI> 2 clones were picked from each plate
</LI><LI> Resuspendsion in 4 ml LB-medium with  4 µl chloramphenicol or ampicillin (QCI (insertion of P114 (pActin))
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_23rd"><B>Sunday, June 23rd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="QuikChange_II_of_pActin_.28P359.2C_P360.29"> QuikChange II of pActin (P359, P360) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> QuikChangeII of pActin (P359, P360)
<B>Procedure:</B></P><P><B>Reaction batch</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH><TH><B>Additional information</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P359, P360)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O76 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O77 (=10 µM)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Segment</B></TH><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>6 min
</TD></TR><TR><TD>3
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_F123.2BF124.2C_F116.2BF117.2C_F116.2BF119.2C_F116.2BF118.2C_F116.2BF121.2C_F116.2BF120.2C_QCI_pActin_.28P114.29">  Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue F123+F124, F116+F117, F116+F119, F116+F118, F116+F121, F116+F120, QCI pActin (P114)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue F123+F124, F116+F117, F116+F119, F116+F118, F116+F121, F116+F120, QCI pActin (P114)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P347-P358"> Analytical digestion and gelelectrophoresis of P347-P358</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P347-P358
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P347-P358)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kb DNA Ladder
</TD><TD>Digestion of P347 with EcoRI and PstI
</TD><TD>Digestion of P348 with EcoRI and PstI
</TD><TD>Digestion of P349 with EcoRI and PstI
</TD><TD>Digestion of P350 with EcoRI and PstI
</TD><TD>Digestion of P351 with EcoRI and PstI
</TD><TD>Digestion of P352 with EcoRI and PstI
</TD><TD>Digestion of P353 with EcoRI and PstI
</TD><TD>Digestion of P354 with EcoRI and PstI
</TD></TR><TR><TD><B>Result:</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Lane:</B></TD><TD>1 kb DNA Ladder
</TD><TD>Digestion of P355 with EcoRI and PstI
</TD><TD>Digestion of P356 with EcoRI and PstI
</TD><TD>Digestion of P357 with EcoRI and PstI
</TD><TD>Digestion of P358 with EcoRI and PstI
</TD></TR><TR><TD><B>Result:</B></TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Dpn1_Digestion_and_transformation_of_E..C2.A0coli_XL1-blue_with_QCII_of_AlcR_and_QCII_of_npt-cassette"> Dpn1 Digestion and transformation of <I>E. coli</I> XL1-blue with QCII of AlcR and QCII of npt-cassette </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Dpn1 Digestion and transformation of <I>E. coli</I> XL1-blue with QCII of AlcR and QCII of npt-cassette
</P><P><B>Procedure:</B></P><UL><LI> 1 µl of Dpn1 was added to the reaction, the batch was incubated for one hour at 37°C
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_F125.2BF98.2C_F125.2BF96.2C_F128.2BF41"> Picking of F125+F98, F125+F96, F128+F41 </SPAN></H3><P><B>Investigator:</B> Katrin
<B>Aim of the experiment:</B>Picking of F125+F98, F125+F96, F128+F41
</P><P><B>Procedure:</B></P><UL><LI> 2 clones were picked from each plate
</LI><LI> Resuspendsion in 4 ml LB-medium with  4 µl chloramphenicol
</LI><LI> F126+F127 was put back into the incubator because there were no colonies visible
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P35.2C_P143.2C_P359.2C_P360.2C_P249">  Sequencing of P35, P143, P359, P360, P249 </SPAN></H3><P><B>Investigators:</B> Katrin
</P><P><B>Aim of the experiment:</B> Sequencing of P35, P143, P359, P360, P249
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P35
</TD><TD> FR02305786 (O4)
</TD></TR><TR><TD> P143
</TD><TD> FR02305785 (O92)
</TD><TD> FR02305784 (O93)
</TD></TR><TR><TD> P359
</TD><TD> FR02305783 (O62)
</TD><TD> FR02305782 (O63)
</TD></TR><TR><TD> P360
</TD><TD> FR02305781 (O62)
</TD><TD> FR02305780 (O63)
</TD></TR><TR><TD> P249
</TD><TD> FR02305779 (O3)
</TD><TD> FR02305778 (O4)
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_10"> Week 10 </SPAN></H1><DIV class="week" id="WWeek_10"><H2><SPAN class="mw-headline" id="Monday.2C_June_24th"><B>Monday, June 24th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_ligations_F125.2BF98_.28N-TEV-linker_in_pSB1Cs_.2B_PIF6.29.2C_F125.2BF96_.28N-TEV-linker_in_pSB1Cs_.2B_PIF3.29_and_F128.2BF41_.28PCR_of_IRES_.2B_pSB1C3_backbone.29_.2B_analytical_gel">  Miniprep of ligations F125+F98 (N-TEV-linker in pSB1Cs + PIF6), F125+F96 (N-TEV-linker in pSB1Cs + PIF3) and F128+F41 (PCR of IRES + pSB1C3 backbone) + analytical gel </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Preparation of ligation products F125+F98, F125+F96 and F128+F41
</P><P><B>Procedure:</B></P><UL><LI> Miniprep according to QIAGEN QIAprep Kit 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>label
</TD><TD>content
</TD><TD>conc. in ng/µl
</TD></TR><TR><TD>P361
</TD><TD>N-TEV-linker (in pSB1C3) + PIF6
</TD><TD>163.2
</TD></TR><TR><TD>P362
</TD><TD>N-TEV-linker in (in pSB1C3) + PIF6
</TD><TD>183.5
</TD></TR><TR><TD>P363
</TD><TD>N-TEV-linker in (in pSB1C3) + PIF3
</TD><TD>159.6
</TD></TR><TR><TD>P364
</TD><TD>N-TEV-linker in (in pSB1C3) + PIF3
</TD><TD>200.1
</TD></TR><TR><TD>P365
</TD><TD>PCR of IRES + (in pSB1C3) backbone
</TD><TD>136.0
</TD></TR><TR><TD>P366
</TD><TD>PCR of IRES + (in pSB1C3) backbone
</TD><TD>153.7
</TD></TR></TBODY></TABLE><UL><LI> Analytical digestion with EcoRI and PstI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume</B></TD></TR><TR><TD> CutSmart buffer
</TD><TD> 2 µl
</TD></TR><TR><TD> EcoRI
</TD><TD> 0.25 µl
</TD></TR><TR><TD> PstI
</TD><TD> 0.25 µl
</TD></TR><TR><TD> ddH2O
</TD><TD> 15 µl
</TD></TR><TR><TD> plasmid DNA
</TD><TD> 2.5 µl
</TD></TR><TR><TD><B>Total</B></TD><TD> 20 µl
</TD></TR></TBODY></TABLE><UL><LI> 60 min incubation, 37 °C
</LI></UL><UL><LI> Gel elektrophoresis on 1% agarose gel
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD> 1 kbp DNA ladder
</TD><TD> P361
</TD><TD> P362
</TD><TD> P363
</TD><TD> P364
</TD><TD> P365
</TD><TD> P366
</TD></TR><TR><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Dpn1_digestion_and_transformation_of_QCII_of_pActin_.28insertion.29_.2B_analytical_Gel"> Dpn1 digestion and transformation of QCII of pActin (insertion) + analytical Gel </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Dpn1 digestion and transformation of QCII of pActin (insertion) 
</P><P><B>Procedure:</B></P><UL><LI> 1 µl of Dpn1 (10 U/µl) was added to the reaction, the batch was incubated for one hour at 37°C
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of QuikChange products was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new ampicillin plate.
</LI></UL><UL><LI> 0.5 µl of loading buffer were added to 4.5 µl of Dpn1 digested plasmid
</LI></UL><UL><LI> Electrophoresis on 1% agarose gel
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kb dna ladder
</TD><TD>P359
</TD><TD>P360
</TD></TR><TR><TD>successful
</TD><TD>successful
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P266_.28IgKappa-SigP_Nanoluc.29.2C_P268_.28IgKappa-SigP_XylE.29.2C_P270_.28IgKappa-SigP_SpyCatcher.29.2C_P272_.28IgKappa-SigP_SpyTag.29.2C_P361_.28N-TEV-linker.2BPIF6.29.2C_P364_.28N-TEV-36AAlinker-PIF3.29_and_P366_.28IRES.29"> Sequencing of P266 (IgKappa-SigP_Nanoluc), P268 (IgKappa-SigP_XylE), P270 (IgKappa-SigP_SpyCatcher), P272 (IgKappa-SigP_SpyTag), P361 (N-TEV-linker+PIF6), P364 (N-TEV-36AAlinker-PIF3) and P366 (IRES) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Sequencing of P266 (IgKappa-SigP_Nanoluc), P268 (IgKappa-SigP_XylE), P270 (IgKappa-SigP_SpyCatcher), P272 (IgKappa-SigP_SpyTag), P361 (N-TEV-linker+PIF6), P364 (N-TEV-36AAlinker-PIF3) and P366 (IRES)
</P><P><B>Procedure:</B> DNA samples were prepared according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The samples received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P266
</TD><TD> FR02305766 fw
</TD><TD> FR02305767 rv
</TD></TR><TR><TD> P268
</TD><TD> FR02305768 fw
</TD><TD> FR02305769 rv
</TD></TR><TR><TD> P270
</TD><TD> FR02305770 fw
</TD></TR><TR><TD> P272
</TD><TD> FR02305771 fw
</TD></TR><TR><TD> P361
</TD><TD> FR02305772 fw
</TD><TD> FR02305773 rv
</TD></TR><TR><TD> P364
</TD><TD> FR02305774 fw
</TD><TD> FR02305775 rv
</TD></TR><TR><TD> P366
</TD><TD> FR02305776 fw
</TD><TD> FR02305777 rv
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Retransformation_of_P270_.28IgKappa-SigP_SpyCatcher.29.2C_P272_.28IgKappa-SigP_SpyTag.29.2C_P308_.28PhyB_36AALinker_C-TEV.29_and_P366_.28IRES.29"> Retransformation of P270 (IgKappa-SigP_SpyCatcher), P272 (IgKappa-SigP_SpyTag), P308 (PhyB_36AALinker_C-TEV) and P366 (IRES) </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Retransformation of P270 (IgKappa-SigP_SpyCatcher in pSB1C3), P272 (IgKappa-SigP_SpyTag in pSB1C3), P308 (PhyB_36AALinker_C-TEV in pSB1C3) and P366 (IRES in pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1.5 µl of plasmid DNA was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of cell suspension were plated on chlorampenicol agar
</LI></UL><UL><LI> The rest of the cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium; the concentrated cell suspension was plated on another chlorampenicol agar plate.
</LI></UL></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F92_.2B_F130_.28IgKappa-SigP_.2B_Laccase.29.2C_F109_.2B_F130_.28SERK-SigP_.2B_Laccase.29.2C_F113_.2B_F131_.28PhyB_36AA-Linker_C-TEV_.2B_IRES.29.2C_F115_.2B_F131_.28TEV_.2B_IRES.29"> Ligation of F92 + F130 (IgKappa-SigP + Laccase), F109 + F130 (SERK-SigP + Laccase), F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES), F115 + F131 (TEV + IRES) </SPAN></H3><P><B>Investigator:</B> Florian, Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F92 + F130 (IgKappa-SigP + Laccase), F109 + F130 (SERK-SigP + Laccase), F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES), F115 + F131 (TEV + IRES).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F92 + F130 (IgKappa-SigP + Laccase)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.86 µl
</TD><TD>F92 (116.7 ng/µl, 2144 bp)
</TD></TR><TR><TD>7.55 µl
</TD><TD>F130 (28.3 ng/µl, 1527 bp)
</TD></TR><TR><TD>8.59 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F109 + F130 (SERK-SigP + Laccase)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2177 bp)
</TD></TR><TR><TD>7.44 µl
</TD><TD>F130 (28.3 ng/µl, 1527 bp)
</TD></TR><TR><TD>8.64 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.82 µl
</TD><TD>F113 (122.2 ng/µl, 5281 bp)
</TD></TR><TR><TD>2.17 µl
</TD><TD>F131 (16.4 ng/µl, 627 bp)
</TD></TR><TR><TD>14.01 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F115 + F131 (TEV + IRES)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.76 µl
</TD><TD>F115 (56.9 ng/µl, 2794 bp)
</TD></TR><TR><TD>4.1 µl
</TD><TD>F131 (16.4 ng/µl, 627 bp)
</TD></TR><TR><TD>11.14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_products_F92_.2B_F130_.28IgKappa-SigP_.2B_Laccase.29.2C_F109_.2B_F130_.28SERK-SigP_.2B_Laccase.29.2C_F113_.2B_F131_.28PhyB_36AA-Linker_C-TEV_.2B_IRES.29.2C_F115_.2B_F131_.28TEV_.2B_IRES.29_and_negative_controls_and_BioBrick_BBa_K325909_.28lucABCDEG_under_pBAD.29"> Transformation of <I>E. coli</I> XL1-blue with ligation products F92 + F130 (IgKappa-SigP + Laccase), F109 + F130 (SERK-SigP + Laccase), F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES), F115 + F131 (TEV + IRES) and negative controls and BioBrick BBa_K325909 (lucABCDEG under pBAD) </SPAN></H3><P><B>Investigator:</B> Florian, Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation products F92 + F130 (IgKappa-SigP + Laccase), F109 + F130 (SERK-SigP + Laccase), F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES), F115 + F131 (TEV + IRES) and negative controls and BioBrick BBa_K325909 (lucABCDEG under pBAD).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of QuikChange products/2 µl of plasmid DNA was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P308_.28PhyB_36AAlinker_C-TEV.29.2C_P348_.28Laccase_RFC25.29_and_P366_.28polioviral_IRES.29"> Preparative digestion of P308 (PhyB_36AAlinker_C-TEV), P348 (Laccase RFC25) and P366 (polioviral IRES) </SPAN></H3><P><B>Investigator:</B> Flo, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P308 (PhyB_36AAlinker_C-TEV), P348 (Laccase RFC25) and P366 (polioviral IRES).
</P><P><B>Procedure:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>14 µl
</TD><TD>P308
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>20 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>14 µl
</TD><TD>P348
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>20 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P366
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl DNA loading buffer (10x) were added to the reaction batches after the incubation time and were loaded on a 1% agaorose gel.
</LI></UL><UL><LI> Preparative geleletrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P308 with SpeI &amp; PstI
</TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P348 with NgoMIV &amp; SpeI
</TD><TD>Digestion of P366 with XbaI &amp; PstI
</TD></TR><TR><TH><B>result</B></TH><TD>2 further unexpected bands; seems that PstI is contaminated with a prefix restriction enzyme; used PstI enzyme batch is discareded; upper band was cut out
</TD><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_F126.2BF127_.28NLS.2BThermoNuc.29.2C_AlcR_QCII_and_npt-cassette_QCII"> Picking of F126+F127 (NLS+ThermoNuc), AlcR QCII and npt-cassette QCII </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of F126+F127 (NLS+ThermoNuc), AlcR QCII and npt-cassette QCII for plasmid preparation
</P><P><B>Procedure:</B></P><UL><LI> F126+F127 (NLS+ThermoNuc) 7 clones due to high number of colonies on negative ctrl
</LI><LI> AlcR QCII and npt-cassette QCII 2 clones each
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_25th"><B>Tuesday, June 25th</B></SPAN></H2><DIV class="vitality"><H3><SPAN class="mw-headline" id="Sequencing_of_P344_pASK-Flua.28Trippelmutante.29"> Sequencing of P344 pASK-Flua(Trippelmutante) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Sequencing of P344. Concentration was determined by Nanodop to 132 ng/µl
</P><P><B>Procedure:</B></P><P>DNA samples were prepared according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
The samples received the following barcodes: 
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH></TR><TR><TD> P344 with F83 (primer from Skerra lab! not fragment 83 from iGEM!)
</TD><TD> FR02305765
</TD></TR></TBODY></TABLE></DIV><DIV class="vitality"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Retransformation_of_P35_.28Viability_sensor_with_RFP.29_and_P249_.28Thermonuc_in_pSB1C3.29"> Retransformation of P35 (Viability sensor with RFP) and P249 (Thermonuc in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Retransformation of P35 (Viability sensor with RFP) and P249 (Thermonuc in pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI><LI> empty tube was washed with 2 µl water and the volume was added to 150 µl of competent cells and gently mixed.
</LI><LI> 30 min incubation on ice
</LI><LI> 5 min. heat shock at 37 °C
</LI><LI> Adding of 1 ml LB-medium to each tube.
</LI><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI><LI> 100 µl of cell suspension were plated on chlorampenicol agar
</LI><LI> The rest of the cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI><LI> The pellet was resuspended in 100 µl of LB-medium; the concentrated cell suspension was plated on another chlorampenicol agar plate.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_QCII_of_P316_.28npt-casette_in_pSB1C3.29.2C_QCII_of_P342_.28AlcR_in_pSB1C3.29_and_F126.2BF127_.28NLS_in_pSB1C3_.2B_Thermonuclease.29"> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with QCII of P316 (npt-casette in pSB1C3), QCII of P342 (AlcR in pSB1C3) and F126+F127 (NLS in pSB1C3 + Thermonuclease)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with QCII of P316 (npt-casette in pSB1C3), QCII of P342 (AlcR in pSB1C3) and F126+F127 (NLS in pSB1C3 + Thermonuclease).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Content</B></TH><TH><B>New Tube</B></TH></TR><TR><TD> F126+F127 clone 1 </TD><TD>P367
</TD></TR><TR><TD> F126+F127 clone 2 </TD><TD>P368
</TD></TR><TR><TD> F126+F127 clone 3 </TD><TD>P369
</TD></TR><TR><TD> F126+F127 clone 4 </TD><TD>P370
</TD></TR><TR><TD> F126+F127 clone 5 </TD><TD>P371
</TD></TR><TR><TD> F126+F127 clone 6 </TD><TD>P372
</TD></TR><TR><TD> F126+F127 clone 7 </TD><TD>P373
</TD></TR><TR><TD> QCII npt clone 1 </TD><TD>P374
</TD></TR><TR><TD> QCII npt clone 2 </TD><TD>P375
</TD></TR><TR><TD> QCII AlcR clone 1 </TD><TD>P376
</TD></TR><TR><TD> QCII AlcR clone 2 </TD><TD>P377
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P367-P373_.28NLS_in_pSB1C3_.2B_Thermonuclease.29_with_EcoRI_and_PstI.2C_P374-P375_.28QCII_of_npt_casette.29_with_XbaI_and_PstI.2C_P376-P377_.28QCII_of_AlcR.29_with_XbaI_and_SpeI"> Analytical digestion and gelelectrophoresis  of P367-P373 (NLS in pSB1C3 + Thermonuclease) with EcoRI and PstI, P374-P375 (QCII of npt casette) with XbaI and PstI, P376-P377 (QCII of AlcR) with XbaI and SpeI</SPAN></H3><P><B>Investigator:</B> Louise, Rosario, Florian
</P><P><B>Aim of the experiment:</B>Analytical digestion and gelelectrophoresis  of P367-P373 (NLS in pSB1C3 + Thermonuclease) with EcoRI and PstI, P374-P375 (QCII of npt casette) with XbaI and PstI, P376-P377 (QCII of AlcR) with XbaI and SpeI.
</P><P><B>Procedure:</B></P><UL><LI> Master-Mix for P367-P373 and control P249
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>20 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2.5 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>2.5 µl
</TD><TD>PstI
</TD></TR><TR><TD>150 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=175 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Master-Mix for P374-P375 and control P316
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>8 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI
</TD></TR><TR><TD>60 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=70 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Master-Mix for P376-P377 and control P342
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>8 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>60 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=70 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for analytical digestion of P367-P377 and controls
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>17.5 µl
</TD><TD>Master-Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1 kbp DNA ladder
</TD><TD>Digestion of P367 with EcoRI and PstI
</TD><TD>Digestion of P368 with EcoRI and PstI
</TD><TD>Digestion of P369 with EcoRI and PstI
</TD><TD>Digestion of P370 with EcoRI and PstI
</TD><TD>Digestion of P371 with EcoRI and PstI
</TD><TD>Digestion of P372 with EcoRI and PstI
</TD><TD>Digestion of P373 with EcoRI and PstI
</TD><TD>Digestion of P249 with EcoRI and PstI
</TD></TR><TR><TH><B>result</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>not as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1 kbp DNA ladder
</TD><TD>Digestion of P316 with XbaI and PstI
</TD><TD>Digestion of P374 with XbaI and PstI/SpeI
</TD><TD>Digestion of P374 with XbaI and PstI/SpeI
</TD><TD>Digestion of P375 with XbaI and PstI/SpeI
</TD><TD>Digestion of P375 with XbaI and PstI/SpeI
</TD></TR><TR><TD>Digestion of P376 with XbaI and SpeI
</TD><TD>Digestion of P377 with XbaI and PstI/SpeI
</TD><TD>Digestion of P377 with XbaI and PstI/SpeI
</TD><TD>Digestion of P342 with XbaI and SpeI
</TD></TR><TR><TH><B>result</B></TH><TD>not as expected, maybe digestion with EcoRI instead of XbaI, has to be repeated
</TD><TD>not as expected, maybe digestion with EcoRI instead of XbaI, has to be repeated
</TD><TD>not as expected, maybe digestion with EcoRI instead of XbaI, has to be repeated
</TD><TD>not as expected, maybe digestion with EcoRI instead of XbaI, has to be repeated
</TD><TD>not as expected, maybe digestion with EcoRI instead of XbaI, has to be repeated
</TD></TR><TR><TD>as expected
</TD><TD>not fully digested
</TD><TD>not fully digested
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P26_.28Dnb1c_Promotor.29_and_P35_.28RD29_Promotor.29_with_EcoRI_and_KpnI"> Preparative digestion and gelelectrophoresis of P26 (Dnb1c Promotor) and P35 (RD29 Promotor) with EcoRI and KpnI </SPAN></H3><P><B>Investigator:</B> Louise, Florian
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P26 (Dnb1c Promotor) and P35 (RD29 Promotor) with EcoRI and KpnI .
</P><P><B>Procedure:</B></P><UL><LI> Batch for P26
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>1 µl
</TD><TD>KpnI
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for P35
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>1 µl
</TD><TD>KpnI
</TD></TR><TR><TD>27 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and the samples were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P26 with EcoRI and KpnI
</TD><TD>100 bp DNA ladder
</TD><TD>Digestion of P26 with EcoRI and KpnI
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TH><B>result</B></TH><TD>not as expected, no band was cut out
</TD><TD>as expected; upper band was cut out, F132
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_ligation_products_F92_.2B_F130_.28IgKappa-SigP_.2B_Laccase.29.2C_F109_.2B_F130_.28SERK-SigP_.2B_Laccase.29.2C_F113_.2B_F131_.28PhyB_36AA-Linker_C-TEV_.2B_IRES.29.2C_F115_.2B_F131_.28TEV_.2B_IRES.29_and_BioBrick_BBa_K325909_.28lucABCDEG_under_pBAD.29_and_Retransformation_of_P270_.28IgKappa-SigP_SpyCatcher.29.2C_P272_.28IgKappa-SigP_SpyTag.29.2C_P308_.28PhyB_36AALinker_C-TEV.29_and_P366_.28IRES.29"> Picking of ligation products F92 + F130 (IgKappa-SigP + Laccase), F109 + F130 (SERK-SigP + Laccase), F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES), F115 + F131 (TEV + IRES) and BioBrick BBa_K325909 (lucABCDEG under pBAD) and Retransformation of P270 (IgKappa-SigP_SpyCatcher), P272 (IgKappa-SigP_SpyTag), P308 (PhyB_36AALinker_C-TEV) and P366 (IRES)</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Picking of ligation products F92 + F130 (IgKappa-SigP + Laccase), F109 + F130 (SERK-SigP + Laccase), F113 + F131 (PhyB_36AA-Linker_C-TEV + IRES), F115 + F131 (TEV + IRES) and BioBrick BBa_K325909 (lucABCDEG under pBAD) and Retransformation of P270 (IgKappa-SigP_SpyCatcher), P272 (IgKappa-SigP_SpyTag), P308 (PhyB_36AALinker_C-TEV) and P366 (IRES).
</P><P><B>Procedure:</B></P><UL><LI>2 clones were picked for each ligation product and for the biobrick
</LI><LI>1 clone was picked for each ReTrafo
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_26th"><B>Wednesday, June 26th</B></SPAN></H2><DIV class="safety"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_transformated_E..C2.A0coli_XL1_blue_with_P270.2C_F113.2BF131.2C_F109.2BF130.2C_P308.2C_F115.2BF131.2C_F92.2BF130.2C_F115.2BF131.2C_P272.2C_P366.2C_lucBB"> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P270, F113+F131, F109+F130, P308, F115+F131, F92+F130, F115+F131, P272, P366, lucBB</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformated <I>E. coli</I> XL1 blue with P270, F113+F131, F109+F130, P308, F115+F131, F92+F130, F115+F131, P272, P366 and lucBB.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN)
</LI></UL><UL><LI>The resulting DNA were eluted in tubes labeled as follows:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Content</B></TH><TH><B>New Tube</B></TH></TR><TR><TD> P270 Retrafo </TD><TD>P379
</TD></TR><TR><TD> F113+F131</TD><TD>P380
</TD></TR><TR><TD> F109+F130 </TD><TD>P381
</TD></TR><TR><TD> P308 Retrafo </TD><TD>P382
</TD></TR><TR><TD> F115+F131 </TD><TD>P383
</TD></TR><TR><TD> F92+F130 </TD><TD>P384
</TD></TR><TR><TD> F113+F131 </TD><TD>P385
</TD></TR><TR><TD> F115+F131 </TD><TD>P386
</TD></TR><TR><TD> F92+F130 </TD><TD>P387
</TD></TR><TR><TD> P272 Retrafo </TD><TD>P388
</TD></TR><TR><TD> P366 </TD><TD>P389
</TD></TR><TR><TD> luc BB </TD><TD>P390
</TD></TR><TR><TD> F109+F130 </TD><TD>P391
</TD></TR><TR><TD> luc BB </TD><TD>P392
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P374-P375_.28QC_II_of_npt-casette.29_and_P316_.28QC_I_of_npt-casette.29_as_a_reference_with_XbaI_and_PstI"> Analytical digestion and gelelectrophoresis of P374-P375 (QC II of npt-casette) and P316 (QC I of npt-casette) as a reference with XbaI and PstI</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P374-P375 (QC II of npt-casette) and P316 (QC I of npt-casette) as a reference with XbaI and PstI
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>volume
</TD><TD>reagent
</TD></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P316,P374,P375)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TH>1000 bp DNA ladder
</TH><TH>Digestion of P316 with XbaI and PstI
</TH><TH>Digestion of P374 with XbaI and PstI
</TH><TH>Digestion of P375 with XbaI and PstI
</TH></TR><TR><TD><B>Result:</B></TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD></TR></TBODY></TABLE></DIV><DIV class="vitality"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_Retransformation_of_P35_.28Viability_sensor_with_RFP.29_and_P249_.28Thermonuc_in_pSB1C3.29"> Picking of Retransformation of P35 (Viability sensor with RFP) and P249 (Thermonuc in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Picking of Retransformation of P35 (Viability sensor with RFP) and P249 (Thermonuc in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI>1 clone was picked for each ReTrafo
</LI></UL></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P367_.28NLS-Thermonuclease.29.2C_P376_.28AlcR_QC_II.29.2C_P374_.28npt-casette_QC_II.29.2C_P348_.28Laccase.29.2C_P350_.28SERK-SigP_EreB_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P352_.28SERK-SigP_SpyCatcher_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P354_.28SERK-SigP_NanoLuc_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P356_.28SERK-SigP_XylE_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P358_.28SERK-SigP_SpyTag_Strep-tag-TEV-site-linker_SERK-TMD-8AAlinker_GFPmut1.29.2C_P26_.28DREB1C_promoter.29">  Sequencing of P367 (NLS-Thermonuclease), P376 (AlcR QC II), P374 (npt-casette QC II), P348 (Laccase), P350 (SERK-SigP_EreB_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P352 (SERK-SigP_SpyCatcher_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P354 (SERK-SigP_NanoLuc_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P356 (SERK-SigP_XylE_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P358 (SERK-SigP_SpyTag_Strep-tag-TEV-site-linker_SERK-TMD-8AAlinker_GFPmut1), P26 (DREB1C promoter) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P367 (NLS-Thermonuclease), P376 (AlcR QC II), P374 (npt-casette QC II), P348 (Laccase), P350 (SERK-SigP_EreB_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P352 (SERK-SigP_SpyCatcher_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P354 (SERK-SigP_NanoLuc_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P356 (SERK-SigP_XylE_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P358 (SERK-SigP_SpyTag_Strep-tag-TEV-site-linker_SERK-TMD-8AAlinker_GFPmut1), P26 (DREB1C promoter).
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P367
</TD><TD> FR02305740 (O3)
</TD></TR><TR><TD> P376
</TD><TD> FR02305741 (O3)
</TD><TD> FR02305742 (O4)
</TD></TR><TR><TD> P374
</TD><TD> FR02305743 (O3)
</TD><TD> FR02305744 (O4)
</TD></TR><TR><TD> P348
</TD><TD> FR02305745 (O3)
</TD><TD> FR02305746 (O4)
</TD></TR><TR><TD> P350
</TD><TD> FR02305747 (O3)
</TD><TD> FR02305748 (O4)
</TD></TR><TR><TD> P352
</TD><TD> FR02305749 (O3)
</TD><TD> FR02305750 (O4)
</TD></TR><TR><TD> P354
</TD><TD> FR02305751 (O3)
</TD><TD> FR02305752 (O4)
</TD></TR><TR><TD> P356
</TD><TD> FR02305753 (O3)
</TD><TD> FR02305754 (O4)
</TD></TR><TR><TD> P358
</TD><TD> FR02305755 (O3)
</TD><TD> FR02305756 (O4)
</TD></TR><TR><TD> P26
</TD><TD> FR02305757 (O4)
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_27th"><B>Thursday, June 27th</B></SPAN></H2><DIV class="safety"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P387_.28IgkKappa-SigP_Laccase_.28QCV.2C_RFC25.29.29.2C_P391_.28SERK-SigP_Laccase_.28QCV.2C_RFC25.29.29and_P355_.28SERK-SigP_XylE_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_clone_2.29">  Sequencing of P387 (IgkKappa-SigP_Laccase (QCV, RFC25)), P391 (SERK-SigP_Laccase (QCV, RFC25))and P355 (SERK-SigP_XylE_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 clone 2) </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Sequencing of P387 (IgkKappa-SigP_Laccase (QCV, RFC25)), P391 (SERK-SigP_Laccase (QCV, RFC25))and P355 (SERK-SigP_XylE_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 clone 2).
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P387
</TD><TD> FR02305758 (O3)
</TD><TD> FR02305759 (O4)
</TD></TR><TR><TD> P391
</TD><TD> FR02305760 (O3)
</TD></TR><TR><TD> P355
</TD><TD> FR02305761 (O3)
</TD><TD> FR02305762 (O4)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety_mechanism"><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Re-Transformation_of_E..C2.A0coli_XL1-blue_with_P170.28SpyTag.29.2C_P165.28t35s.29.2C_P308.28PhyB_36AALinker_C-TEV.29.2C_P348.28Laccase_QCV_RFC25.29.2C_P326.28Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P361.28N-TEV-36AAlinker-PIF6.29.2C_P364.28N-TEV-36AAlinker-PIF3.29.2C_P266.28IgKappa-SigP_Nanoluc.29"> Re-Transformation of <I>E. coli</I> XL1-blue with P170(SpyTag), P165(t35s), P308(PhyB_36AALinker_C-TEV), P348(Laccase QCV RFC25), P326(Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P361(N-TEV-36AAlinker-PIF6), P364(N-TEV-36AAlinker-PIF3), P266(IgKappa-SigP_Nanoluc) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Re-Transformation of <I>E. coli</I> XL1-blue with P170(SpyTag), P165(t35s), P308(PhyB_36AALinker_C-TEV), P348(Laccase QCV RFC25), P326(Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P361(N-TEV-36AAlinker-PIF6), P364(N-TEV-36AAlinker-PIF3), P266(IgKappa-SigP_Nanoluc)
<B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 150 µl of competent cells were added to the tubes containing about 1 µl of plasmid DNA and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new chlorampenicol plate.
</LI></UL></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_P143_.28pLuc-Actin.2C_Amplification_of_pActin.29"> PCR of P143 (pLuc-Actin, Amplification of pActin) </SPAN></H3><P><B>Investigator:</B> Rosario, Flo
</P><P><B>Aim of the experiment:</B> PCR of P143 (pLuc-Actin, Amplification of pActin).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> Reaction batch with Q5 Mastermix:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O96 (pActin_Oryza_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O97 (pActin_Oryza_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)- 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>62 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>45 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F133"> Analytical gelelectrophoresis of F133 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B> Analytical gelelectrophoresis of F133 (PCR product of F143 with O96 &amp; O97) to check the quality of the performed PCR.
</P><P><B>Procedure:</B></P><UL><LI> 4.5 µl of F133 were mixed together 0.5 µl of DNA loading buffer (10x)
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed on a 1% agarose gel at 90 V for 40 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>F133
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected; but byproduct visible
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P381.2C_P391.2C_P347.2C_P384.2C_P387.2C_P380.2C_P385.2C_P309.2C_P383.2C_P386.2C_P390.2C_P392"> Analytical digestion and gelelectrophoresis of P381, P391, P347, P384, P387, P380, P385, P309, P383, P386, P390, P392</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P381, P391, P347, P384, P387, P380, P385, P309, P383, P386, P390, P392 with EcoRI and PstI
</P><TABLE border="1"><TBODY><TR><TH><B>Volume</B></TH><TH><B>Reagent</B></TH></TR><TR><TD>2 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 60 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TH>1000 bp DNA ladder
</TH><TH>Digestion of P381 with EcoRI and PstI
</TH><TH>Digestion of P391 with EcoRI and PstI
</TH><TH>Digestion of P347 with EcoRI and PstI
</TH><TH>Digestion of P384 with EcoRI and PstI
</TH><TH>Digestion of P387 with EcoRI and PstI
</TH></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD><TD>not fully digested
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TH>1000 bp DNA ladder
</TH><TH>Digestion of P380 with EcoRI and PstI
</TH><TH>Digestion of P385 with EcoRI and PstI
</TH><TH>Digestion of P309 with EcoRI and PstI
</TH><TH>Digestion of P383 with EcoRI and PstI
</TH><TH>Digestion of P386 with EcoRI and PstI
</TH><TH>Digestion of P390 with EcoRI and PstI
</TH><TH>Digestion of P392 with EcoRI and PstI
</TH></TR><TR><TD><B>Result:</B></TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P391_.28SERK-SigP_Laccase.29.2C_P326_.28Strep-tag-TEV-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P385_.28PhyB_36AAlinker_C-TEV_IRES.29.2C_P361_.28N-TEV_36AAlinker_PIF6.29.2C_P364_.28N-TEV_36AAlinker_PIF3.29.2C_P266_.28IgKappa-SigP_NanoLuc.29.2C_P388_.28IgKappa-SigP_SpyTag.29.2C_P247_.28SpyCatcher.29.2C_P379_.28IgKappa-SigP_SpyCatcher.29.2C_P374_.28npt-casette.29_and_P165_.28t35S.29"> Preparative digestion and gelelectrophoresis of P391 (SERK-SigP_Laccase), P326 (Strep-tag-TEV-linker_SERK-TMD_8AAlinker_GFPmut1), P385 (PhyB_36AAlinker_C-TEV_IRES), P361 (N-TEV_36AAlinker_PIF6), P364 (N-TEV_36AAlinker_PIF3), P266 (IgKappa-SigP_NanoLuc), P388 (IgKappa-SigP_SpyTag), P247 (SpyCatcher), P379 (IgKappa-SigP_SpyCatcher), P374 (npt-casette) and P165 (t35S)  </SPAN></H3><P><B>Investigator:</B> Flo, Rosario
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P391 (SERK-SigP_Laccase), P326 (Strep-tag-TEV-linker_SERK-TMD_8AAlinker_GFPmut1), P385 (PhyB_36AAlinker_C-TEV_IRES), P361 (N-TEV_36AAlinker_PIF6), P364 (N-TEV_36AAlinker_PIF3), P266 (IgKappa-SigP_NanoLuc), P388 (IgKappa-SigP_SpyTag), P247 (SpyCatcher), P379 (IgKappa-SigP_SpyCatcher), P374 (npt-casette) and P165 (t35S).
</P><P><B>Procedure:</B></P><P><B>Batch for preparative digestion of effectors (P328, P330, P332, P334, P336, P338) with AgeI and PstI)</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 1 (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 2 (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Plasmid</B></TH><TH><B>Name</B></TH><TH><B>Enzyme 1</B></TH><TH><B>Enzyme 2</B></TH></TR><TR><TD>P391
</TD><TD>SERK-SigP_Laccase
</TD><TD>AgeI
</TD><TD>PstI
</TD></TR><TR><TD>P326
</TD><TD>Strep-tag-TEV-linker_SERK-TMD_8AAlinker_GFPmut1
</TD><TD>NgoMIV
</TD><TD>PstI
</TD></TR><TR><TD>P385
</TD><TD>PhyB_36AAlinker_C-TEV_IRES
</TD><TD>SpeI
</TD><TD>PstI
</TD></TR><TR><TD>P361
</TD><TD>N-TEV_36AAlinker_PIF6
</TD><TD>XbaI
</TD><TD>PstI
</TD></TR><TR><TD>P364
</TD><TD>N-TEV_36AAlinker_PIF3
</TD><TD>XbaI
</TD><TD>PstI
</TD></TR><TR><TD>P266
</TD><TD>IgKappa-SigP_NanoLuc
</TD><TD>EcoRI
</TD><TD>AgeI
</TD></TR><TR><TD>P388
</TD><TD>IgKappa-SigP_SpyTag
</TD><TD>AgeI
</TD><TD>PstI
</TD></TR><TR><TD>P247
</TD><TD>SpyCatcher
</TD><TD>EcoRI
</TD><TD>NgoMIV
</TD></TR><TR><TD>P379
</TD><TD>IgKappa-SigP_SpyCatcher
</TD><TD>AgeI
</TD><TD>PstI
</TD></TR><TR><TD>P374
</TD><TD>npt-casette
</TD><TD>EcoRI
</TD><TD>XbaI
</TD></TR><TR><TD>P165
</TD><TD>t35S
</TD><TD>EcoRI
</TD><TD>SpeI
</TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and they were loaded on a 1% agarose gel
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P391 with AgeI &amp; PstI
</TD><TD>1kb DNA ladder
</TD><TD>Digestion of P326 with NgoMIV &amp; PstI
</TD><TD>Digestion of P385 with SpeI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>sideproducts?!; upper band was cut out
</TD><TD>as expected, lower band was cut out
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P361 with XbaI &amp; PstI
</TD><TD>1kb DNA ladder
</TD><TD>Digestion of P364 with XbaI &amp; PstI
</TD><TD>Digestion of P266 with EcoRI &amp; AgeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD><TD>as expected, lower band was cut out
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P388 with AgeI &amp; PstI
</TD><TD>1kb DNA ladder
</TD><TD>Digestion of P382 with NgoMIV &amp; PstI
</TD><TD>Digestion of P247 with EcoRI &amp; NgoMIV
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>not expected, wrong plasmid
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P379 with AgeI &amp; PstI
</TD><TD>1kb DNA ladder
</TD><TD>Digestion of P374 with EcoRI &amp; XbaI
</TD><TD>Digestion of P165 with EcoRI &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected, but only little DNA
</TD></TR></TBODY></TABLE><UL><LI> Extracted via QIAGEN gel extraction kit, digested PCR product purified via PCR Purification Kit
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F134_.2B_F135_.28SERK-SigP-Laccase_.2B_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_F136_.2B_F137_.28PhyB_36AAlinker_C-TEV_IRES_.2B_N-TEV_36AAlinker_PIF6.29.2C_F136_.2B_F138_.28PhyB_36AAlinker_C-TEV_IRES_.2B_N-TEV_36AAlinker_PIF3.29_and_F143_.2B_F144_.28npt-csette_.2B_t35S.29"> Ligation of F134 + F135 (SERK-SigP-Laccase + Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F136 + F137 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF6), F136 + F138 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF3) and F143 + F144 (npt-csette + t35S) </SPAN></H3><P><B>Investigator:</B> Flo, Rosario
</P><P><B>Aim of the experiment:</B> Ligation of F134 + F135 (SERK-SigP-Laccase + Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F136 + F137 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF6), F136 + F138 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF3) and F143 + F144 (npt-csette + t35S).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F134 + F135 (SERK-SigP-Laccase + Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) (vector:insert = 1:3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.83 µl
</TD><TD>F134 (120.6 ng/µl, 1630 bp)
</TD></TR><TR><TD>10.29 µl
</TD><TD>F135 (17.8 ng/µl, 996 bp)
</TD></TR><TR><TD>5.88 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F136 + F137 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF6)  (vector:insert = 1:3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.5 µl
</TD><TD>F136 (201.2 ng/µl, 5916 bp)
</TD></TR><TR><TD>4.6 µl
</TD><TD>F137 (8.5 ng/µl, 771 bp)
</TD></TR><TR><TD>11.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F136 + F138 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF3)  (vector:insert = 1:3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.5 µl
</TD><TD>F136 (201.2 ng/µl, 5916 bp)
</TD></TR><TR><TD>4.07 µl
</TD><TD>F138 (9.6 ng/µl, 771 bp)
</TD></TR><TR><TD>12.43 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F143 + F144 (npt-csette + t35S) (vector:insert = 1:3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.53 µl
</TD><TD>F143 (188.8 ng/µl, 3587 bp)
</TD></TR><TR><TD>6.05 µl
</TD><TD>F144 (3.0 ng/µl, 217 bp)
</TD></TR><TR><TD>10.42 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed overnight at 16 °C.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_28th"><B>Friday, June 28th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_P338.28Ig-Kappa-SigP_SpyTag.2C_Retrafo.29.2C_F135.2BF134.28SERK-SigP_Laccase.2BStrep-TEV_linker_SERK-TMD_linker_GFP.29.2C_F136.2BF138.28PhyB_linker_C-TEV_IRES.2BN-TEV_linker_PIF3.29.2C_F136.2BF137.28PhyB_linker_C-TEV_IRES.2BN-TEV_linker_PIF6.29.2C_F143.2BF144.28npt-Cassette.2BTerminator.29_and_negative_controls_.28F135.2C_F136.2C_F143.29"> Transformation of <I>E. coli</I> XL1-blue with P338(Ig-Kappa-SigP_SpyTag, Retrafo), F135+F134(SERK-SigP_Laccase+Strep-TEV_linker_SERK-TMD_linker_GFP), F136+F138(PhyB_linker_C-TEV_IRES+N-TEV_linker_PIF3), F136+F137(PhyB_linker_C-TEV_IRES+N-TEV_linker_PIF6), F143+F144(npt-Cassette+Terminator) and negative controls (F135, F136, F143) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with P338(Ig-Kappa-SigP_SpyTag, Retrafo), F135+F134(SERK-SigP_Laccase+Strep-TEV_linker_SERK-TMD_linker_GFP), F136+F138(PhyB_linker_C-TEV_IRES+N-TEV_linker_PIF3), F136+F137(PhyB_linker_C-TEV_IRES+N-TEV_linker_PIF6), F143+F144(npt-Cassette+Terminator) and negative controls (F135, F136, F143)
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed, for re-transformation, 1 µl of P388 was added to 150 µl of competent cells
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="QuikChange_III_of_AlcR_.28P376.29"> QuikChange III of AlcR (P376) </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> QuikChange III of AlcR (P376) 
</P><P><B>Reaction batch</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH><TH><B>Additional information</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P376), (1:120 diluted)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O84 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O85 (=10 µM)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Segment</B></TH><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>3
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P388_.28Ig-Kappa-SigP_SpyTag.29.2C_P246_.28Nanoluc.29.2C_F133_.28PCR_of_pActin.29"> Preparative digestion and gelelectrophoresis of P388 (Ig-Kappa-SigP_SpyTag), P246 (Nanoluc), F133 (PCR of pActin) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P388 (Ig-Kappa-SigP_SpyTag), P246 (Nanoluc), F133 (PCR of pActin)
</P><P><B>Procedure:</B></P><P>reaction batch for P388(Ig-Kappa-SigP_SpyTag)
</P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P388
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>reaction batch for P246(Nanoluc)
</P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P246
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>
reaction batch for F133(PCR of pActin)
</P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>F133
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated for 3 h at 37 °C.
</LI></UL><UL><LI> 4 µl DNA loading buffer (10x) were added to the reaction batches after the incubation time and were loaded on a 1% agaorose gel.
</LI></UL><UL><LI> Preparative geleletrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of F133 with XbaI &amp; PstI
</TD><TD>1 kbp DNA ladder
</TD><TD>Digestion of P246 with NgoMIV,PstI
</TD><TD>Digestion of P388 with AgeI, PstI
</TD></TR><TR><TH><B>result</B></TH><TD>as expected (but smaller fragment visible), upper band was cut out
</TD><TD>as expected, lower band was cut out
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F146.2BF147_.28Ig-Kappa-SigP_SpyTag.2BNanoluc.29_and_F145.2BF42_.28pActin.2BpSB1C3.29"> Ligation of F146+F147 (Ig-Kappa-SigP_SpyTag+Nanoluc) and F145+F42 (pActin+pSB1C3) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Ligation of F146+F147 (Ig-Kappa-SigP_SpyTag+Nanoluc) and F145+F42 (pActin+pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F146+F147 (Ig-Kappa-SigP_SpyTag+Nanoluc)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.90 µl
</TD><TD>F146 (111.2 ng/µl, 2189 bp)
</TD></TR><TR><TD>3.99 µl
</TD><TD>F147 (17.5 ng/µl, 510 bp)
</TD></TR><TR><TD>12.11 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F145+F42 (pActin + pSB1C3) (batch: 30 ng vector!)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>23.35 µl
</TD><TD>F145 (2.5 ng/µl, 1353 bp)
</TD></TR><TR><TD>0.41 µl
</TD><TD>F42 (73.5 ng/µl, 2086 bp)
</TD></TR><TR><TD>2.7 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=27 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature.
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_ReTrafos_of_E..C2.A0coli_XL1-blue_with_P170.28SpyTag.29.2C_P165.28t35s.29.2C_P308.28PhyB_36AALinker_C-TEV.29.2C_P348.28Laccase_QCV_RFC25.29.2C_P326.28Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P361.28N-TEV-36AAlinker-PIF6.29.2C_P364.28N-TEV-36AAlinker-PIF3.29.2C_P266.28IgKappa-SigP_Nanoluc.29"> Picking of ReTrafos of <I>E. coli</I> XL1-blue with P170(SpyTag), P165(t35s), P308(PhyB_36AALinker_C-TEV), P348(Laccase QCV RFC25), P326(Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P361(N-TEV-36AAlinker-PIF6), P364(N-TEV-36AAlinker-PIF3), P266(IgKappa-SigP_Nanoluc)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Picking of ReTrafos of <I>E. coli</I> XL1-blue with P170(SpyTag), P165(t35s), P308(PhyB_36AALinker_C-TEV), P348(Laccase QCV RFC25), P326(Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P361(N-TEV-36AAlinker-PIF6), P364(N-TEV-36AAlinker-PIF3), P266(IgKappa-SigP_Nanoluc
</P><P><B>Procedure:</B></P><UL><LI> one clone per plate was picked
</LI><LI> 4 ml LB-medium, 4 µl chloramphenicol
</LI></UL></DIV></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Dpn1_digestion_and_transformation_of_QCIII_of_AlcR_.2B_analytical_Gel"> Dpn1 digestion and transformation of QCIII of AlcR + analytical Gel </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Dpn1 digestion and transformation of QCIII of AlcR + analytical Gel
</P><P><B>Procedure:</B></P><UL><LI> 1 µl of Dpn1 (10 U/µl) was added to the reaction, the batch was incubated for one hour at 37°C
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of QuikChange products was added to 150 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellchloramphenicol plate.
</LI></UL><UL><LI> 0.5 µl of loading buffer were added to 4.5 µl of Dpn1 digested plasmid
</LI></UL><UL><LI> Electrophoresis on 1% agarose gel
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1 kbp DNA ladder
</TD><TD>QCIII product of AlcR
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_products_F146.2BF147_.28Ig-Kappa-SigP_SpyTag.2BNanoluc.29_and_F145.2BF42_.28pActin.2BpSB1C3.29_and_negative_controls_F146_and_F42"> Transformation of <I>E. coli</I> XL1-blue with ligation products F146+F147 (Ig-Kappa-SigP_SpyTag+Nanoluc) and F145+F42 (pActin+pSB1C3) and negative controls F146 and F42 </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation products F146+F147 (Ig-Kappa-SigP_SpyTag+Nanoluc) and F145+F42 (pActin+pSB1C3) and negative controls F146 and F42
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Oligohybridization_of_mMCSII_.28mMCSII_fw_.28O98.29_.2B_mMCSII_rv_.28O99.29.29_and_.28GGGGS.29x5_TEV_.28GGGGSx5_TEV_fw_.28O102.29_.2B_GGGGSx5_TEV_rv_.28O103.29.29">  Oligohybridization of mMCSII (mMCSII_fw (O98) + mMCSII_rv (O99)) and (GGGGS)x5_TEV (GGGGSx5_TEV_fw (O102) + GGGGSx5_TEV_rv (O103)) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment: </B> Oligohybridization of mMCSII (mMCSII_fw (O98) + mMCSII_rv (O99)) and (GGGGS)x5_TEV (GGGGSx5_TEV_fw (O102) + GGGGSx5_TEV_rv (O103)).
</P><P><B>Procedure:</B></P><UL><LI> 25 µL of 100 pmol/µl of  mMCSII_fw (O98) and mMCSII_rv (O99) were pooled and mixed together/25 µL of 100 pmol/µl of  GGGGSx5_TEV_fw (O102) and GGGGSx5_TEV_rv (O103) were pooled and mixed together/
</LI></UL><UL><LI> Heating up to 95 °C for 5 min 
</LI></UL><UL><LI> Cooling at RT in a styropor box overnight
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_29th"><B>Saturday, June 29th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_Retrafos_of_P170.28SpyTag.29.2C_P165.28t35s.29.2C_P308.28PhyB_36AALinker_C-TEV.29.2C_P348.28Laccase_QCV_RFC25.29.2C_P326.28Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P361.28N-TEV-36AAlinker-PIF6.29.2C_P364.28N-TEV-36AAlinker-PIF3.29.2C_P266.28IgKappa-SigP_Nanoluc.29">  Miniprep of Retrafos of P170(SpyTag), P165(t35s), P308(PhyB_36AALinker_C-TEV), P348(Laccase QCV RFC25), P326(Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P361(N-TEV-36AAlinker-PIF6), P364(N-TEV-36AAlinker-PIF3), P266(IgKappa-SigP_Nanoluc) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of Retrafos of P170(SpyTag), P165(t35s), P308(PhyB_36AALinker_C-TEV), P348(Laccase QCV RFC25), P326(Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P361(N-TEV-36AAlinker-PIF6), P364(N-TEV-36AAlinker-PIF3), P266(IgKappa-SigP_Nanoluc)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep according to QIAGEN QIAprep Kit 
</LI></UL></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F145.2BF42_.28pActin.2BpSB1C3.2C_ligation_overnight_at_16.C2.B0C.29_and_negative_control"> Transformation of <I>E. coli</I> XL1-blue with ligation product F145+F42 (pActin+pSB1C3, ligation overnight at 16°C) and negative control </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation products F145+F42 (pActin+pSB1C3, ligation overnight at 16°C) and negative control
<B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P402_.28Ig-Kappa-SigP_Nanoluc.29"> Preparative digestion of P402 (Ig-Kappa-SigP_Nanoluc) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion of P402 (Ig-Kappa-SigP_Nanoluc)
</P><P><B>Procedure:</B></P><P>reaction batch
</P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P402
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated for 3 h at 37 °C.
</LI></UL></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_F148.2C_F149_and_digestion_product_of_P402"> Preparative gelelectrophoresis of F148, F149 and digestion product of P402 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative gelelectrophoresis of F148, F149 and digestion product of P402.
</P><P><B>Procedure:</B></P><UL><LI> Preparative gelelectrophoresis was performed on a 1% agarose gel at 90 V for 45 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1 kbp DNA ladder
</TD><TD>F148
</TD><TD>F149
</TD><TD>Digestion of P402 with AgeI &amp; SpeI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected (lower band was cut out)
</TD></TR></TBODY></TABLE><UL><LI> Gelpurified products were extracted via QIAquick gel extraction kit, QIAGEN.
</LI></UL></DIV></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR_of_P143_.28pLuc-Actin.2C_Amplification_of_pActin.29_and_P344_.28FluA_R95KA45IS114T.2C_triple_mutant_of_FluA.29"> PCR of P143 (pLuc-Actin, Amplification of pActin) and P344 (FluA R95KA45IS114T, triple mutant of FluA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR of P143 (pLuc-Actin, Amplification of pActin) and P344 (FluA R95KA45IS114T, triple mutant of FluA).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> Reaction batch with Q5 Mastermix for P143 (pActin amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O96 (pActin_Oryza_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O97 (pActin_Oryza_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)- 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch with Q5 Mastermix for P344 (FluA triple mutant amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O104 (FluA_tripmut_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O105 (FluA_tripmut_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P344)- 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme for P143 (pActin amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>62 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>45 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme for P344 (FluA triple mutant amplification)
</LI></UL><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>69 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>16 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_30th"><B>Sunday, June 30th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_F134_.2B_F135_.28SERK-SigP-Laccase_.2B_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_F136_.2B_F137_.28PhyB_36AAlinker_C-TEV_IRES_.2B_N-TEV_36AAlinker_PIF6.29.2C_F136_.2B_F138_.28PhyB_36AAlinker_C-TEV_IRES_.2B_N-TEV_36AAlinker_PIF3.29.2C_F143_.2B_F144_.28npt-csette_.2B_t35S.29_and_ReTraFo_of_P388_.28IgKappa-SigP_SpyTag.29"> Miniprep of F134 + F135 (SERK-SigP-Laccase + Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F136 + F137 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF6), F136 + F138 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF3), F143 + F144 (npt-csette + t35S) and ReTraFo of P388 (IgKappa-SigP_SpyTag) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Miniprep of F134 + F135 (SERK-SigP-Laccase + Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F136 + F137 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF6), F136 + F138 (PhyB_36AAlinker_C-TEV_IRES + N-TEV_36AAlinker_PIF3), F143 + F144 (npt-csette + t35S) and ReTraFo of P388 (IgKappa-SigP_SpyTag).
</P><P><B>Procedure:</B></P><P>Miniprep according to QIAGEN QIAprep Kit 
</P></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P403.C2.A0.E2.80.93.C2.A0P418"> Analytical digestion and gelelectrophoresis of P403 – P418 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P403 – P418.
</P><P><B>Procedure:</B></P><UL><LI> 23x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>46 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>5.75 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>5.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>345 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=402.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 23x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>17.5 µl
</TD><TD>Mastermix for EcoRI &amp; PstI
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P403 – P418)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2.22 µl of DNA loading buffer (10x) was added to the reaction batches.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P403 with EcoRI &amp; PstI
</TD><TD>Digestion of P404 with EcoRI &amp; PstI
</TD><TD>Digestion of P405 with EcoRI &amp; PstI
</TD><TD>Digestion of P406 with EcoRI &amp; PstI
</TD><TD>Digestion of P407 with EcoRI &amp; PstI
</TD><TD>Digestion of P408 with EcoRI &amp; PstI
</TD><TD>Digestion of P409 with EcoRI &amp; PstI
</TD><TD>Digestion of P410 with EcoRI &amp; PstI
</TD><TD>Digestion of P374 with EcoRI &amp; PstI
</TD><TD>F153
</TD><TD>F154
</TD></TR><TR><TH><B>Result:</B></TH><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected (control)
</TD><TD>no PCR product; corrupt
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P411 with EcoRI &amp; PstI
</TD><TD>Digestion of P412 with EcoRI &amp; PstI
</TD><TD>Digestion of P246 with EcoRI &amp; PstI (Control)
</TD><TD>Digestion of P413 with EcoRI &amp; PstI
</TD><TD>Digestion of P414 with EcoRI &amp; PstI
</TD><TD>Digestion of P391 with EcoRI &amp; PstI (Control)
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P415 with EcoRI &amp; PstI
</TD><TD>Digestion of P416 with EcoRI &amp; PstI
</TD><TD>Digestion of P417 with EcoRI &amp; PstI
</TD><TD>Digestion of P418 with EcoRI &amp; PstI
</TD><TD>Digestion of P385 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected (Control)
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_P143_.28pLuc-Actin.2C_Amplification_of_pActin.29_2"> PCR of P143 (pLuc-Actin, Amplification of pActin) </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> PCR of P143 (pLuc-Actin, Amplification of pActin).
<B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> Reaction batch with Q5 Mastermix for P143 (pActin amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O96 (pActin_Oryza_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O97 (pActin_Oryza_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)- 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme for P143 (pActin amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>62 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>45 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI>The PCR was performed in two different cyclers in parallel to elucidate wheather this might have an influence on the product. Furthermore one sample contained a 100 fold increased template concentration.
</LI><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI><LI> Analytical gelelectrophoresis was performed on a 1% agarose gel at 90 V for 60 min using NEB 2-Log DNA ladder.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1 kbp DNA ladder
</TD><TD>1:1000 P143
</TD><TD>1:1000 P143
</TD><TD>1:10 P143
</TD></TR><TR><TH><B>Result:</B></TH><TD>length as expected, but low concentration, small amount of side products
</TD><TD>length as expected, but low concentration, small amount of side products
</TD><TD>length as expected; higher concentration but increased amount of side products, too. Fragment was labelled F155.
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P367_.28NLS.2Bthermonuclease.29.2C_P416_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6.29.2C_P418_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_.29.2C_P404_.28t35S_npt-casette.29.2C_P395_.28SpyTag.29and_F154_.28FluA_triple_mutant.29"> Preparative digestion and gelelectrophoresis of P367 (NLS+thermonuclease), P416 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6), P418 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6 ), P404 (t35S_npt-casette), P395 (SpyTag)and F154 (FluA triple mutant) </SPAN></H3><P><B>Investigator:</B>  Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P367 (NLS+thermonuclease), P416 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6), P418 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6 ), P404 (t35S_npt-casette), P395 (SpyTag)and F154 (FluA triple mutant).
</P><P><B>Procedure:</B></P><P><B>Batch for preparative digestion of P404 with EcoRI and XbaI, P395 with NgoMIV and SpeI, F154 with XbaI and AgeI, P367 with XbaI and NgoMIV, P416 with SpeI and Pst I and P418 with SpeI and PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 1 (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 2 (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>PCR product
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 1 (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Enzyme 2 (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Plasmid</B></TH><TH><B>Name</B></TH><TH><B>Enzyme 1</B></TH><TH><B>Enzyme 2</B></TH></TR><TR><TD>P404
</TD><TD>t35S_npt-casette
</TD><TD>EcoRI
</TD><TD>XbaI
</TD></TR><TR><TD>P395
</TD><TD>SpyTag
</TD><TD>NgoMIV
</TD><TD>SpeI
</TD></TR><TR><TD>P418
</TD><TD>PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6
</TD><TD>SpeI
</TD><TD>PstI
</TD></TR><TR><TD>P367
</TD><TD>NLS+thermonuclease
</TD><TD>XbaI
</TD><TD>NgoMIV
</TD></TR><TR><TD>P416
</TD><TD>PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6
</TD><TD>SpeI
</TD><TD>PstI
</TD></TR><TR><TD>F154
</TD><TD>FluA triple mutant
</TD><TD>XbaI
</TD><TD>AgeI
</TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and they were loaded on a 1% agarose gel
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P367 with XbaI &amp; NgoMIV
</TD><TD>1kb DNA ladder
</TD><TD>Digestion of P416 with SpeI &amp; PstI
</TD><TD>Digestion of P418 with SpeI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P404 with EcoRI &amp; XbaI
</TD><TD>1kb DNA ladder
</TD><TD>Digestion of P395 with NgoMIV &amp; SpeI
</TD><TD>Digestion of F154 with XbaI &amp; AgeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected; lower band was extracted
</TD><TD>as expected
</TD></TR></TBODY></TABLE><UL><LI> Extracted via QIAGEN gel extraction kit, digested PCR product purified via PCR Purification Kit
</LI></UL></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F123_.2B_F152_.28.28GGGGS.29x5-TEV-site-linker_in_pSB1C3.29.2C_F156_.2B_F152_.28.28GGGGS.29x5-TEV-site-linker_NLS_Thermonuclease_in_pSB1C3.29.2C_F157_.2B_F131_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_IRES_in_pSB1C3.29.2C_F158_.2B_F131_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_IRES_in_pSB1C3.29.2C_F159_.2B_F151_.28mMCSII_t35S_npt-casette_in_pSB1C3.29.2C_F150_.2B_F160_.28SERK-SigP_Nanoluc_SpyTag.29_and_F123_.2B_F161_.28FluA_triple_mutant_in_pSB1C3.29"> Ligation of F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F156 + F152 ((GGGGS)x5-TEV-site-linker_NLS_Thermonuclease in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3), F150 + F160 (SERK-SigP_Nanoluc_SpyTag) and F123 + F161 (FluA triple mutant in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Ligation of F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F156 + F152 ((GGGGS)x5-TEV-site-linker_NLS_Thermonuclease in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3), F150 + F160 (SERK-SigP_Nanoluc_SpyTag) and F123 + F161 (FluA triple mutant in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.03 µl
</TD><TD>F123 (49.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>14.97 µl
</TD><TD>F152 (488.2 ng/µl, 102 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> F156 + F152 ((GGGGS)x5-TEV-site-linker_NLS_Thermonuclease in pSB1C3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.63 µl
</TD><TD>F156 (158.3 ng/µl, 2596 bp)
</TD></TR><TR><TD>16.37 µl
</TD><TD>F152 (488.2 ng/µl, 102 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.67 µl
</TD><TD>F157 (148.7 ng/µl, 6711 bp)
</TD></TR><TR><TD>1.71 µl
</TD><TD>F131 (16.4 ng/µl, 627 bp)
</TD></TR><TR><TD>14.62 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.76 µl
</TD><TD>F158 (132.4 ng/µl, 6711 bp)
</TD></TR><TR><TD>1.71 µl
</TD><TD>F131 (16.4 ng/µl, 627 bp)
</TD></TR><TR><TD>14.59 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.78 µl
</TD><TD>F159 (56.1 ng/µl, 3812 bp)
</TD></TR><TR><TD>15.22 µl
</TD><TD>F151 (229.4 ng/µl, 37 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> F150 + F160 (SERK-SigP_Nanoluc_SpyTag):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.59 µl
</TD><TD>F150 (169.3 ng/µl, 2660 bp)
</TD></TR><TR><TD>16.41 µl
</TD><TD>F160 (109 ng/µl, 39 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> F123 + F161 (FluA triple mutant in pSB1C3):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.03 µl
</TD><TD>F123 (49.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>2.58 µl
</TD><TD>F161 (29.1 ng/µl, 522 bp)
</TD></TR><TR><TD>12.39 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H1><DIV class="week" id="WWeek_11"><H2><SPAN class="mw-headline" id="Monday.2C_July_1st"><B>Monday, July 1st</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Everyday_I.27m_swaffeling"> Everyday I'm swaffeling </SPAN></H3><P><B>Investigator:</B> Le swaffleur passive
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_F155_.28pActin_PCR_product.29_and_P393_.28Stress_inducible_Promoter.29"> Preparative digestion and gelelectrophoresis of F155 (pActin PCR product) and P393 (Stress inducible Promoter)</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of F155 (pActin PCR product) and P393 (Stress inducible Promoter).
</P><P><B>Procedure:</B></P><P><B>Batch for preparative digestion of F155 (pActin PCR product) with XbaI and PstI </B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>28.5 µl
</TD><TD>F155
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI
</TD></TR><TR><TD>5.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Batch for preparative digestion of P393 (Stress inducible Promoter)with EcoRI and SpeI&quot;</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P393
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4.5 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches after digestion and they were loaded on a 1% agarose gel
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kb DNA ladder
</TD><TD>Digestion of F155 with XbaI &amp; PstI
</TD><TD>Digestion of P393 with EcoRI &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected, the upper band was cut out
</TD><TD>as expected, the lowest band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Extracted via QIAGEN gel extraction kit, digested PCR product purified via PCR Purification Kit
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F162_.28pActin.29_.2B_F42.28pSB1C3_backbone.29"> Ligation of F162 (pActin) + F42(pSB1C3 backbone) </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Ligation of F162 (pActin) + F42(pSB1C3 backbone) 
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for ligation of F162 (pActin) + F42(pSB1C3 backbone) 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.36 µl
</TD><TD>F42 (100 ng)
</TD></TR><TR><TD>6.64 µl
</TD><TD>F162
</TD></TR><TR><TD>91 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (NEB)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> The ligation was performed for 1 hour at room temperature; the ligation batch was transferred to 16 °C afterwads over night.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F162.2BF42_.28pActin.2BpSB1C3.2C_ligation_for_1_h_at_room_temperature"> Transformation of <I>E. coli</I> XL1-blue with ligation product F162+F42 (pActin+pSB1C3, ligation for 1 h at room temperature  </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation product F162+F42 (pActin+pSB1C3, ligation for 1 h at room temperature
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F123_.2B_F152_.28.28GGGGS.29x5-TEV-site-linker_in_pSB1C3.29.2C_F157_.2B_F131_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_IRES_in_pSB1C3.29.2C_F158_.2B_F131_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_IRES_in_pSB1C3.29.2C_F159_.2B_F151_.28mMCSII_t35S_npt-casette_in_pSB1C3.29_and_F123_.2B_F161_.28FluA_triple_mutant_in_pSB1C3.29"> Transformation of <I>E. coli</I> XL1-blue with ligation product F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3) and F123 + F161 (FluA triple mutant in pSB1C3)  </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation product F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3) and F123 + F161 (FluA triple mutant in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_2nd"><B>Tuesday, July 2nd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F162.2BF42_.28pActin.2BpSB1C3.2C_ligation_over_night_at_16_.C2.B0C"> Transformation of <I>E. coli</I> XL1-blue with ligation product F162+F42 (pActin+pSB1C3, ligation over night at 16 °C  </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation product F162+F42 (pActin+pSB1C3, ligation over night at 16 °C.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_F123_.2B_F152.2C_F157_.2B_F131.2C_F158_.2B_F131.2C_F159_.2B_F151_and_F123_.2B_F161"> Picking of F123 + F152, F157 + F131, F158 + F131, F159 + F151 and F123 + F161 </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Picking of F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3) and F123 + F161 (FluA triple mutant in pSB1C3).
</P><P><B>Procedure:</B></P><P>Picked colonies were transfered into 5 ml LB-medium with 5 µl of Chloramphenicol and incubated for at 37 °C in the 180 rpm cell-culture shaker.
</P></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F162_.28pActin.29_.2B_F42.28pSB1C3_backbone.29_and_of_F163_.28stress_inducible_Promoter_.2B_RFP_.29_.2B_F159_.28t35S-NPT-pSB1C3.29"> Ligation of F162 (pActin) + F42(pSB1C3 backbone) and of F163 (stress inducible Promoter + RFP ) +  F159 (t35S-NPT-pSB1C3)</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> As the ligation of the prmoter done on Monday failed, it is repeated with an decreased overall volume resulting in an increased DNA concentration: Ligation of F162 (pActin) + F42(pSB1C3 backbone). Furthermore the stress inducible Promoter cloned to  RFP (F163) should be ligated into a pSB1C3 in front og t35S and NPT.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for ligation of F162 (pActin) + F42(pSB1C3 backbone) 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.36 µl
</TD><TD>F42 (100 ng)
</TD></TR><TR><TD>6.64 µl
</TD><TD>F162
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The reaction was prepared twice: one time using a ligase provided by NEB and one time using a ligase providied by Thermo Scientific. Negative controls were prepared by replacing the volume of the insert by ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> Ligation batch for ligation of F163 (stress inducible promoter + RFP) + F159(t35S-NPT-pSB1C3 backbone) 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.79 µl
</TD><TD>F159 (100 ng)
</TD></TR><TR><TD>8.61 µl
</TD><TD>F163
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase
</TD></TR><TR><TD>6,6 µl
</TD><TD>H2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The ligation was performed for 2 hours at room temperature; the ligation batch was transferred to 16 °C afterwads over night.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F162.2BF42_.28pActin.2BpSB1C3.2C_ligation_batch_volume_10_.C2.B5l.2C_reaction_performed_at_room_temperature.29_and_with_F159.2BF163_.28stress_inducible_promoter.2BRFP.2Bt35S.2BNPT_in_pSB1C3.29"> Transformation of <I>E. coli</I> XL1-blue with ligation product F162+F42 (pActin+pSB1C3, ligation batch volume 10 µl, reaction performed at room temperature) and with F159+F163 (stress inducible promoter+RFP+t35S+NPT in pSB1C3)  </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation product F162+F42 (pActin+pSB1C3, ligation batch volume 10 µl, reaction performed at room temperature) and with F159+F163 (stress inducible promoter+RFP+t35S+NPT in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_P143_.28pLuc-Actin.2C_Amplification_of_pActin.29_3"> PCR of P143 (pLuc-Actin, Amplification of pActin) </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> PCR of P143 (pLuc-Actin, Amplification of pActin). The PCR product was phosphorylated afterwards to test blunt end cloning into a vector.
<B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> Reaction batch for PCR with Q5 Mastermix for P143 (pActin amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O96 (pActin_Oryza_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O97 (pActin_Oryza_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P143)- 1:10 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme for P143 (pActin amplification):
</LI></UL><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>62 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>45 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI>The PCR was performed in two different cyclers in parallel to elucidate wheather this might have an influence on the product. Furthermore one sample contained a 100 fold increased template concentration.
</LI><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI><LI> Reaction batch for phosphorylation of thePCR product:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>15 µl
</TD><TD>PCR product
</TD></TR><TR><TD>2 µl
</TD><TD>10 x forward buffer A
</TD></TR><TR><TD>2 µl
</TD><TD>10 mM ATP
</TD></TR><TR><TD>1 µl
</TD><TD>Polynucleotide Kinase (Thermo Scientific)
</TD></TR><TR><TD>=20 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>The reaction was performed for 1 h at 37 °C. Afterwards the mixture was inactivated by heat for 10 min at 75 °C.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Quickchange_mutagenesis_of_P143_.28Actin_promoter.29">  Quickchange mutagenesis of P143 (Actin promoter)  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment:</B> Deliting the forbidden EcoRI restriction site in the promoter. 
</P><P><B>Procedure:</B><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid P143 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O106
</TD></TR><TR><TD>19. 5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid P143 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>O107
</TD></TR><TR><TD>19.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Segment</B></TH><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>15
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6,5 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied. Furthermore 1 µl  Pfu Ultra II DNA polymerase was added.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P352_.28SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P358_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29_and_P395_.28SpyTag.29"> Preparative digestion and gelelectrophoresis of P352 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P358 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) and P395 (SpyTag)  </SPAN></H3><P><B>Investigator:</B>  Louise, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P352 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P358 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) and P395 (SpyTag).
</P><UL><LI> Batch for preparative digestion of P352 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P352
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P358 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P358
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P395 (SpyTag):
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P395
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C; 2.5 h
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) were added to the 40 µl reaction batches, digested with NgoMIV &amp; SpeI (P395), after digestion and they were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><UL><LI> Reaction batches which were digested with SpeI &amp; PstI were purified with QIAquick PCR purification kit, QIAGEN.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kb DNA ladder
</TD><TD>Digestion of P395 with NgoMIV &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Extracted via QIAquick gel extraction kit, QIAGEN.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F164_.2B_F131_.28SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_in_pSB1C3.2B_IRES.29_and_F165_.2B_F131_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_in_pSB1C3_.2B_IRES.29"> Ligation of F164 + F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3+ IRES) and F165 + F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  Ligation of F164 + F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3+ IRES), F165 + F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F164 + F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3+ IRES), vector:insert = 1:3:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.22 µl
</TD><TD>F164 (81.7 ng/µl, 3524 bp)
</TD></TR><TR><TD>3.25 µl
</TD><TD>F131 (16.4 ng/µl, 627 bp)
</TD></TR><TR><TD>12.53 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F165 + F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3+ IRES), vector:insert = 1:3:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.65 µl
</TD><TD>F165 (60.7 ng/µl, 3224 bp)
</TD></TR><TR><TD>3.56 µl
</TD><TD>F131 (16.4 ng/µl, 627 bp)
</TD></TR><TR><TD>11.79 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at RT for 1 h.
</LI></UL></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F164_.2B_F131_.28SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_in_pSB1C3.2B_IRES.29.2C_F165_.2B_F131_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_in_pSB1C3_.2B_IRES.29_and_retransformations_of_P352.2C_P358_and_P395"> Transformation of <I>E. coli</I> XL1-blue with ligation product F164 + F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3+ IRES), F165 + F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES) and retransformations of P352, P358 and P395  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation product F164 + F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3+ IRES), F165 + F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES) and retransformations of P352, P358 and P395.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F150_.2B_F166_.28IgKappa-SigP_NanoLuc_in_pSB1C3_.2B_SpyTag.29"> Ligation of F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  Ligation of F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag), vector:insert = 1:6:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.59 µl
</TD><TD>F150 (169.3 ng/µl, 2660 bp)
</TD></TR><TR><TD>2.67 µl
</TD><TD>F166 (3.3 ng/µl, 39 bp)
</TD></TR><TR><TD>13.79 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1_blue_transformed_with_F123_.2B_F152_.28.28GGGGS.29x5-TEV-site-linker_in_pSB1C3.29.2C_F157_.2B_F131_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_IRES_in_pSB1C3.29.2C_F158_.2B_F131_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3.2F6_IRES_in_pSB1C3.29.2C_F159_.2B_F151_.28mMCSII_t35S_npt-casette_in_pSB1C3.29_and_F123_.2B_F161_.28FluA_triple_mutant_in_pSB1C3.29"> Miniprep of <I>E. coli</I> XL1 blue transformed with F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3) and F123 + F161 (FluA triple mutant in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Ingmar, Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1 blue transformed with F123 + F152 ((GGGGS)x5-TEV-site-linker in pSB1C3), F157 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F158 + F131 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3/6_IRES in pSB1C3), F159 + F151 (mMCSII_t35S_npt-casette in pSB1C3) and F123 + F161 (FluA triple mutant in pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P420–P432.
</LI></UL></DIV></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_F152_.2B_F156_.28.28GGGGS.29x5-TEV-site-linker_.2B_NLS_Thermonuclease_in_pSB1C3.29"> Picking of of <I>E. coli</I> XL1 blue transformed with F152 + F156 ((GGGGS)x5-TEV-site-linker + NLS_Thermonuclease in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with F152 + F156 ((GGGGS)x5-TEV-site-linker + NLS_Thermonuclease in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_3rd"><B>Wednesday, July 3rd</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_F162_.2B_F42_.28pActin_.2B_pSB1C3.29.2C_F163_.2B_F159_.28Stress_inducible_promotor_RFP_.2B_t35S_NPT_in_pSB1C3.2C_P395.2C_P352.2C_P358.2C_F164.2BF131_.28SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_in_pSB1C3_.2B_IRES.29.2C_F165.2BF131_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_in_pSB1C3_.2B_IRES.29"> Picking of of <I>E. coli</I> XL1 blue transformed with F162 + F42 (pActin + pSB1C3), F163 + F159 (Stress inducible promotor_RFP + t35S_NPT in pSB1C3, P395, P352, P358, F164+F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES), F165+F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES) </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with F162 + F42 (pActin + pSB1C3), F163 + F159 (Stress inducible promotor_RFP + t35S_NPT in pSB1C3, P395, P352, P358, F164+F131 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES), F165+F131 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in pSB1C3 + IRES).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 5 mL of LB-medium + 5 µL chloramphenicol(1000x).
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overday.
</LI></UL></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1_blue_transformed_with_F152_.2B_F156_.28.28GGGGS.29x5-TEV-site-linker_.2B_NLS_Thermonuclease_in_pSB1C3.29"> Miniprep of <I>E. coli</I> XL1 blue transformed with F152 + F156 ((GGGGS)x5-TEV-site-linker + NLS_Thermonuclease in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1 blue transformed with F152 + F156 ((GGGGS)x5-TEV-site-linker + NLS_Thermonuclease in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN). 
</LI></UL><UL><LI> The resulting tubes were labeled as P433 – P435. 
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F150_.2B_F166_.28IgKappa-SigP_NanoLuc_in_pSB1C3_.2B_SpyTag.29"> Transformation of <I>E. coli</I> XL1-blue with ligation product F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag)  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-blue with ligation product F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P420.C2.A0.E2.80.93.C2.A0P435"> Analytical digestion and gelelectrophoresis of P420 – P435 </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P420 – P435.
</P><P><B>Procedure:</B></P><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P420 with EcoRI &amp; PstI
</TD><TD>Digestion of P421 with EcoRI &amp; PstI
</TD><TD>Digestion of F123 with EcoRI &amp; PstI
</TD><TD>Digestion of P422 with EcoRI &amp; PstI
</TD><TD>Digestion of P429 with EcoRI &amp; PstI
</TD><TD>Digestion of P430 with EcoRI &amp; PstI
</TD><TD>Digestion of P431 with EcoRI &amp; PstI
</TD><TD>Digestion of P432 with EcoRI &amp; PstI
</TD><TD>1000 bp DNA ladder
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P424 with EcoRI &amp; PstI
</TD><TD>Digestion of P425 with EcoRI &amp; PstI
</TD><TD>Digestion of P426 with EcoRI &amp; PstI
</TD><TD>Digestion of P427 with EcoRI &amp; PstI
</TD><TD>Digestion of P428 with EcoRI &amp; PstI
</TD><TD>Digestion of P404 with EcoRI &amp; PstI
</TD><TD>Digestion of P433 with EcoRI &amp; PstI
</TD><TD>Digestion of P434 with EcoRI &amp; PstI
</TD><TD>Digestion of P435 with EcoRI &amp; PstI
</TD><TD>Digestion of P367 with EcoRI &amp; PstI
</TD><TD>1000 bp DNA ladder
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected (control)
</TD><TD>no PCR product; corrupt
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_4th"><B>Thursday, July 4th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Quickchange_mutagenesis_I_of_P444_.28Actin_promoter.29">  Quickchange mutagenesis I of P444 (Actin promoter)  </SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Deleting the forbidden EcoRI restriction site in the promoter. 
</P><P><B>Procedure:</B><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid P444 template
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O106
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O107
</TD></TR><TR><TD>18.0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>12.5 µl
</TD><TD>Q5 Polymerase Mastermix
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid P444 template
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O106
</TD></TR><TR><TD>1.25 µl
</TD><TD>1:10 dilution of O107
</TD></TR><TR><TD>8.0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters for batch 1</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>52°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 5 min
</TD></TR><TR><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters for batch 2</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>98 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>98°C
</TD><TD> 10 sec
</TD></TR><TR><TD>55°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 4 min
</TD></TR><TR><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL><P>The successful amplification was verified on an agerose gel and the PCR-product was transformed (by Jeff)
</P></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F150_.2B_F166_.28IgKappa-SigP_NanoLuc_in_pSB1C3_.2B_SpyTag.29_and_negative_control_F150"> Ligation of F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag) and negative control F150 </SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B>  Ligation of F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag) and negative control F150
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F150 + F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag), vector:insert = 1:6:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.59 µl
</TD><TD>F150 (169.3 ng/µl, 2660 bp)
</TD></TR><TR><TD>2.67 µl
</TD><TD>F166 (3.3 ng/µl, 39 bp)
</TD></TR><TR><TD>13.79 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 22 °C for 1h
</LI><LI> negative control was prepaed with water instead of insert F166
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Test_digestion_of_pAct_in_pSB1C3_.28p435_to_p448.29"> Test digestion of pAct in pSB1C3 (p435 to p448) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><UL><LI>App. 500 ng of DNA from 13 different clones were digeted with EcoRI and PstI
</LI><LI>Mastermix consisting of 3 µl of each enzyme 30 µl Cutsmart buffer and 120 µl water
</LI></UL><UL><LI>p444 and p447 were sent for sequencing
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F150.2BF166_.28IgKappa-SigP_NanoLuc_in_pSB1C3_.2B_SpyTag.29.2C_F142.2BF147_.28Ig-Kappa-SigP_SpyCatcher_.2B_Nanoluc.29.2C_F139.2BF141_.28Ig-Kappa-SigP_Nanoluc_.2B_SpyCatcher.29_and_negative_controls"> Transformation of <I>E. coli</I> XL1-blue with ligation product F150+F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag), F142+F147 (Ig-Kappa-SigP_SpyCatcher + Nanoluc), F139+F141 (Ig-Kappa-SigP_Nanoluc + SpyCatcher) and negative controls</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1-blue with ligation product F150+F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag), F142+F147 (Ig-Kappa-SigP_SpyCatcher + Nanoluc), F139+F141 (Ig-Kappa-SigP_Nanoluc + SpyCatcher) and negative controls
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P434.2C_P412.2C_P462.2C_P453"> Preparative digestion of P434, P412, P462, P453 </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Preparative digestion of P434, P412, P462, P453
</P><UL><LI> Batch for preparative digestion of P434 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P434
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P412
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P412
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P462
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P462
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for preparative digestion of P453
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P453
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C; 3 h, then put into freezer
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F142.2BF147_.28IgK-SigP_Spycatcher_.2B_NanoLuc.29_and_F141.2BF139_.28SpyCatcher_.2B_IgK-SigP_NanoLuc.29"> Ligation of F142+F147 (IgK-SigP_Spycatcher + NanoLuc) and F141+F139 (SpyCatcher + IgK-SigP_NanoLuc)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B>  Ligation of F142+F147 and F139+F141
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F142+F147:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.81 µl
</TD><TD>F142 (123.4 ng/µl, 2489 bp)
</TD></TR><TR><TD>3.51 µl
</TD><TD>F147 (17.5 ng/µl, 510 bp)
</TD></TR><TR><TD>12.68 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F141+F139
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.83 µl
</TD><TD>F141 (54.6 ng/µl, 2425 bp)
</TD></TR><TR><TD>3.43 µl
</TD><TD>F139 (21.0 ng/µl, 583 bp)
</TD></TR><TR><TD>11.74 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 22 °C for 1h
</LI><LI> negative control was prepaed with water instead of inserts F147 and F139
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_5th"><B>Friday, July 5th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F122.2BF169_.28SERK-SigP_SERK-TMD_.2B_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29.2C_F170.2BF168_.28SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_.2B_Ig-Kappa-SigP_SpyTag_Nanoluc.29_and_negative_controls"> Ligation of F122+F169 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F170+F168 (SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_Nanoluc) and negative controls</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>  Ligation of F122+F169 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F170+F168 (SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_Nanoluc) and negative controls
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F122+F169
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.99 µl
</TD><TD>F122 (50.3 ng/µl, 2369 bp)
</TD></TR><TR><TD>4.32 µl
</TD><TD>F169 (18.1 ng/µl, 618 bp)
</TD></TR><TR><TD>10.69 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F170+F168
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.01 µl
</TD><TD>F170 (49.7 ng/µl, 4168 bp)
</TD></TR><TR><TD>4.11 µl
</TD><TD>F168 (11.0 ng/µl, 628 bp)
</TD></TR><TR><TD>10.88 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 22 °C for 1h
</LI><LI> negative control was prepaed with water instead of inserts F169 and F168
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_product_F122.2BF169_.28SERK-SigP_SERK-TMD_.2B_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29.2C_F170.2BF168_.28SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_.2B_Ig-Kappa-SigP_SpyTag_Nanoluc.29_and_negative_controls"> Transformation of <I>E. coli</I> XL1-blue with ligation product F122+F169 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F170+F168 (SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_Nanoluc) and negative controls</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>  TTransformation of <I>E. coli</I> XL1-blue with ligation product F122+F169 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F170+F168 (SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_Nanoluc) and negative controls
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1_blue_transformed_with_ligation_product_F150.2BF166_.28IgKappa-SigP_NanoLuc_in_pSB1C3_.2B_SpyTag.29.2C_F142.2BF147_.28Ig-Kappa-SigP_SpyCatcher_.2B_Nanoluc.29.2C_F139.2BF141_.28Ig-Kappa-SigP_Nanoluc_.2B_SpyCatcher.29"> Picking of <I>E. coli</I> XL1 blue transformed with ligation product F150+F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag), F142+F147 (Ig-Kappa-SigP_SpyCatcher + Nanoluc), F139+F141 (Ig-Kappa-SigP_Nanoluc + SpyCatcher)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1 blue transformed with ligation product F150+F166 (IgKappa-SigP_NanoLuc in pSB1C3 + SpyTag), F142+F147 (Ig-Kappa-SigP_SpyCatcher + Nanoluc), F139+F141 (Ig-Kappa-SigP_Nanoluc + SpyCatcher)
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI><LI> 2 colonies were picked for F150+F166 and F139+F141, 8 colonies were picked for F142+F147 (many colonies on negatibe controls)
</LI><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overday.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P3_.28pTEF2_in_pTUM100.29.2C_P341_.28QCI_of_AlcR.29.2C_P453_.28SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_.2B_IRES_in_pSB1C3.29.2C_P462_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_.2B_IRES_in_pSB1C3.29.2C_P300_.28Phytotchrome_B.29">  Sequencing of P3 (pTEF2 in pTUM100), P341 (QCI of AlcR), P453 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 +  IRES in pSB1C3), P462 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 + IRES in pSB1C3), P300 (Phytotchrome B) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P3 (pTEF2 in pTUM100), P341 (QCI of AlcR), P453 (SERK-SigP_SpyCatcher_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 +  IRES in pSB1C3), P462 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 + IRES in pSB1C3), P300 (Phytotchrome B).
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH><TH> Barcode3
</TH><TH> Barcode4
</TH></TR><TR><TD> P3
</TD><TD> FR02233147 (O5)
</TD></TR><TR><TD> P341
</TD><TD> FR02233146 (O3)
</TD><TD> FR02233145 (O4)
</TD><TD> FR02233144 (O100)
</TD><TD> FR02233143 (O101)
</TD></TR><TR><TD> P453
</TD><TD> FR02233142 (O4)
</TD></TR><TR><TD> P462
</TD><TD> FR02233141 (O4)
</TD></TR><TR><TD> P300
</TD><TD> FR02233140 (O3)
</TD><TD> FR02233139 (O4)
</TD><TD> FR02233138 (O110)
</TD><TD> FR02233137 (O111)
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Quikchange_mutagenesis_of_P447_.28Actin_promoter.29_to_remove_forbidden_EcoRI_restriction_site">  Quikchange mutagenesis of P447 (Actin promoter) to remove forbidden EcoRI restriction site  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Quikchange mutagenesis of P447 (Actin promoter) to remove forbidden EcoRI restriction site.
</P><P><B>Procedure:</B></P><P><B>Reaction batch 1:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid P447 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O106 (=10 µM)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O107 (=10 µM)
</TD></TR><TR><TD>18.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>Q5 Polymerase Mastermix
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid P447 template (1:100)
</TD></TR><TR><TD>2.5 µl
</TD><TD>1:10 dilution of O106 (=10 µM)
</TD></TR><TR><TD>2.5 µl
</TD><TD>1:10 dilution of O107 (=10 µM)
</TD></TR><TR><TD>8.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters for batch 1</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>52°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 4 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters for batch 2</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>98 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>98°C
</TD><TD> 10 sec
</TD></TR><TR><TD>55°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 2 min
</TD></TR><TR><TD>3
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="QuikChange_of_pYes_.28P3.29"> QuikChange of pYes (P3) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> QuikChange of pYes (P3) 
</P><P><B>Reaction batch</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH><TH><B>Additional information</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid template (P3), (1:10 diluted)
</TD><TD>need to get between 2.5 ng and 25 ng for a reaction batch of 25 µl
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O5 (=10 µM)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Segment</B></TH><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>2
</TD><TD>20
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>52 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>4 min
</TD></TR><TR><TD>3
</TD><TD>4 °C
</TD><TD>hold
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_the_DpnI-digested_QuikChange_products_of_P447_.28pActin_in_pSB1C3.29_and_P3_.28pTEF2_in_pTUM100.29"> Analytical gelelectrophoresis of the DpnI-digested QuikChange products of P447 (pActin in pSB1C3) and P3 (pTEF2 in pTUM100) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B> Analytical gelelectrophoresis of the DpnI-digested QuikChange products of P447 (pActin in pSB1C3)  and P3 (pTEF2 in pTUM100) to check whether QuikChange might be worked.
</P><P><B>Procedure:</B></P><UL><LI> 4.5 µl of the QuikChange products of P447 (PfuUltraII Polymerase batch and Q5 Polymerase batch (see above)) and P3 (pTEF2 in pTUM100) were mixed together 0.5 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed on a 1% agarose gel at 90 V for 40 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>QC of P3 with O44 &amp; O45 (PfuUltraII)
</TD><TD>QC of P447 with O106 &amp; O107 (PfuUltraII)
</TD><TD>QC of P447 with O106 &amp; O107 (Q5)
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>no product visible
</TD><TD>no product visible
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_QuikChange_products_of_P447_.28PfuUltraII_and_Q5.29_and_of_P3_.28pTEF2_in_pTUM100.29"> Transformation of <I>E. coli</I> XL1-blue with QuikChange products of P447 (PfuUltraII and Q5) and of P3 (pTEF2 in pTUM100) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1-blue with QuikChange products of P447 (PfuUltraII and Q5) and of P3 (pTEF2 in pTUM100) (see above).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_July_6th"><B>Saturday, July 6th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Quikchange_mutagenesis_of_P447_.28Actin_promoter.29_to_remove_forbidden_EcoRI_restriction_site_2">  Quikchange mutagenesis of P447 (Actin promoter) to remove forbidden EcoRI restriction site  </SPAN></H3><P><B>Investigator:</B> Le swaffeleur
</P><P><B>Aim of the experiment:</B> Quikchange mutagenesis of P447 (Actin promoter) to remove forbidden EcoRI restriction site.
</P><P><B>Procedure:</B></P><P><B>Reaction batch 1:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P447 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O106 (=10 µM)
</TD></TR><TR><TD>16.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD><B>=20 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch 2:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P447 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O107 (=10 µM)
</TD></TR><TR><TD>16.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD><B>=20 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The two batches were cycled seperately for the first 10 rounds.
</LI></UL><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>54°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 4 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> After the ten cycles the two reaction batches were pooled together and were additionally cycled for 20 rounds with the following conditions:
</LI></UL><P><B>PCR cycling parameters after poooling the reaction batches together:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>98 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>98°C
</TD><TD> 10 sec
</TD></TR><TR><TD>54°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 2 min
</TD></TR><TR><TD>3
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_QuikChange_products_of_P447_.28pActin_in_pSB1C3.29"> Analytical gelelectrophoresis of QuikChange products of P447 (pActin in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of QuikChange products of P447 (pActin in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> 9 µl of the QuikChange products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 30 min on a 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>QC of P447 with O106 &amp; O107 (PfuUltraII, without DMSO)
</TD><TD>QC of P447 with O106 &amp; O107 (PfuUltraII, with DMSO)
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_QuikChange_products_of_P447_.28pActin_in_pSB1C3.29"> Transformation of <I>E. coli</I> XL1-blue with QuikChange products of P447 (pActin in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff, Le Swaffeleur
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1-blue with QuikChange products of P447 (pActin in pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 8 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-blue_transformed_with_ligation_product_F122.2BF169_.28SERK-SigP_SERK-TMD_.2B_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_F170.2BF168_.28SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_.2B_Ig-Kappa-SigP_SpyTag_NanoLuc.29"> Picking of <I>E. coli</I> XL1-blue transformed with ligation product F122+F169 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA) and F170+F168 (SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_NanoLuc) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-blue transformed with ligation product F122+F169 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA) and F170+F168 (SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_NanoLuc).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 2 colonies were picked for F122+F169 and F170+F168.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_July_7th"><B>Sunday, July 7th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1_blue_transformed_with_F170.2BF168.28SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_.2B_Ig-Kappa-SigP_SpyTag_NanoLuc.29"> Miniprep of <I>E. coli</I> XL1 blue transformed with F170+F168(SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_NanoLuc) </SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1 blue transformed with F170+F168(SERK-SigP_SpyCatcher_Strep-TagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES + Ig-Kappa-SigP_SpyTag_NanoLuc).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P475 and P476.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P464.C2.A0.E2.80.93.C2.A0P476_and_P402_.28control.29"> Analytical digestion and gelelectrophoresis of P464 – P476 and P402 (control) </SPAN></H3><P><B>Investigator:</B> Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P464 – P476 and P402 (control)
</P><UL><LI> 16x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>17.5 µl
</TD><TD>Mastermix for EcoRI &amp; PstI
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P464 – P476, P402)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P464 with EcoRI &amp; PstI
</TD><TD>Digestion of P465 with EcoRI &amp; PstI
</TD><TD>Digestion of P466 with EcoRI &amp; PstI
</TD><TD>Digestion of P467 with EcoRI &amp; PstI
</TD><TD>Digestion of P402 with EcoRI &amp; PstI (control)
</TD><TD>Digestion of P475 with EcoRI &amp; PstI
</TD><TD>Digestion of P476 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P468 with EcoRI &amp; PstI
</TD><TD>Digestion of P469 with EcoRI &amp; PstI
</TD><TD>Digestion of P470 with EcoRI &amp; PstI
</TD><TD>Digestion of P471 with EcoRI &amp; PstI
</TD><TD>Digestion of P472 with EcoRI &amp; PstI
</TD><TD>Digestion of P473 with EcoRI &amp; PstI
</TD><TD>Digestion of P474 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="PCR_of_P378_.28Proteinphosphotase_1.29"> PCR of P378 (Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Katrin, Jeff
</P><P><B>Aim of the experiment:</B> PCR of P378 (Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for PCR with Q5 Mastermix for P378 (Proteinphosphotase 1):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix (normally not necessary!)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O112 (PP1 for)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O113 (PP1 rev)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P378) - 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>18 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme for P378(PP1 amplification):
</LI></UL><P><B>Cycling parameters:</B></P><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>62 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>30 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> The resulting product was labeled as F174.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F174"> Analytical gelelectrophoresis of F174 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F174.
</P><P><B>Procedure:</B></P><UL><LI> 9 µl of the PCR product F174 (=PCR of P378 with O112/O113) products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>F174
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P422_.28FluA_triple_mutant.29.2C_P464_.28IgKappa-SigP_NanoLuc_SpyCatcher.29_and_P466_.28IgKappa-SigP_NanoLuc_SpyTag.29"> Preparative digestion and gelelectrophoresis of P422 (FluA triple mutant), P464 (IgKappa-SigP_NanoLuc_SpyCatcher) and P466 (IgKappa-SigP_NanoLuc_SpyTag) </SPAN></H3><P><B>Investigator:</B> Katrin, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P422 (FluA triple mutant), P464 (IgKappa-SigP_NanoLuc_SpyCatcher) and P466 (IgKappa-SigP_NanoLuc_SpyTag).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P422 (FluA triple mutant):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P422
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P464 (IgKappa-SigP_NanoLuc_SpyCatcher):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P464
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P466 (IgKappa-SigP_NanoLuc_SpyTag):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P466
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P422 with NgoMIV &amp; SpeI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P464 with XbaI &amp; PstI
</TD><TD>Digestion of P466 with XbaI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P379_.28IgKappa-SigP_SpyCatcher.29_and_F174_.28Proteinphosphotase_1.29"> Preparative digestion of P379 (IgKappa-SigP_SpyCatcher) and F174 (Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Katrin, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P379 (IgKappa-SigP_SpyCatcher) and F174 (Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P379 (IgKappa-SigP_SpyCatcher):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P379
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for F174 (Proteinphosphotase 1):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>F174
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 4 h.
</LI></UL><UL><LI> After digestion, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F170.2BF179_.28SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyTag.29.2C_F171.2BF178_.28SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyCatcher.29.2C_F176.2BF147_.28IgKappa-SigP_SpyCatcher_in_pSB1C3_.2B_NanoLuc.29.2C_F109.2BF177_.28SERK-SigP_in_pSB1C3_.2B_FluA_triple_mutant.29_and_F123.2BF175_.28pSB1C3_only_.2B_PP1.29"> Ligation of F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) and F123+F175 (pSB1C3 only + PP1)  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  Ligation of F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) and F123+F175 (pSB1C3 only + PP1).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES</B></P><PRE>in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), vector:insert = 1:3:
</PRE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.01 µl
</TD><TD>F170 (49.7 ng/µl, 4168 bp)
</TD></TR><TR><TD>3.50 µl
</TD><TD>F179 (12.9 ng/µl, 628 bp)
</TD></TR><TR><TD>11.49 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.65 µl
</TD><TD>F171 (60.05 ng/µl, 3868 bp)
</TD></TR><TR><TD>5.80 µl
</TD><TD>F178 (12.4 ng/µl, 928 bp)
</TD></TR><TR><TD>9.55 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.61 µl
</TD><TD>F176 (163.9 ng/µl, 2489 bp)
</TD></TR><TR><TD>3.51 µl
</TD><TD>F147 (17.5 ng/µl, 510 bp)
</TD></TR><TR><TD>12.88 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2117 bp)
</TD></TR><TR><TD>2.90 µl
</TD><TD>F177 (25.5 ng/µl, 522 bp)
</TD></TR><TR><TD>13.18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F123+F175 (pSB1C3 only + PP1), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F123 (49.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>2.90 µl
</TD><TD>F175 (12.3 ng/µl, 972 bp)
</TD></TR><TR><TD>3.59 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 22 °C for 1 h
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-blue_with_ligation_products_F170.2BF179_.28SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyTag.29.2C_F171.2BF178_.28SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyCatcher.29.2C_F176.2BF147_.28IgKappa-SigP_SpyCatcher_in_pSB1C3_.2B_NanoLuc.29.2C_F109.2BF177_.28SERK-SigP_in_pSB1C3_.2B_FluA_triple_mutant.29_and_F123.2BF175_.28pSB1C3_only_.2B_PP1.29"> Transformation of <I>E. coli</I> XL1-blue with ligation products F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) and F123+F175 (pSB1C3 only + PP1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  Transformation of <I>E. coli</I> XL1-blue with ligation products F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) and F123+F175 (pSB1C3 only + PP1).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1_blue_transformed_with_QuikChange_products_of_P447_.28pActin.29_and_ligation_product_F123.2BF152_.28.28GGGGS.29x5-TEV-cleavage-site-linker.29"> Picking of <I>E. coli</I> XL1 blue transformed with QuikChange products of P447 (pActin) and ligation product F123+F152 ((GGGGS)x5-TEV-cleavage-site-linker) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1 blue transformed with QuikChange products of P447 (pActin) and ligation product F123+F152 ((GGGGS)x5-TEV-cleavage-site-linker).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol (1000x).
</LI></UL><UL><LI> 8 colonies were picked for QuikChange products of P447 (pActin)and F139+F141 and 4 colonies were picked for ligation product F123+F152 ((GGGGS)x5-TEV-cleavage-site-linker).
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight.
</LI></UL></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_12">Week 12</SPAN></H1><DIV class="week" id="WWeek_12"><H2><SPAN class="mw-headline" id="Monday.2C_July_8th"><B>Monday, July 8th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P477-P484_.28pActin.29"> Analytical digestion and gelelectrophoresis of P477-P484 (pActin) </SPAN></H3><P><B>Investigator:</B> Rosario, Ingmar
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P477-P484 (pActin in pSB1C3) after QC of forbidden EcoRI Restriction site. 
</P><UL><LI> 10x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>15 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>2 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>2 µl
</TD><TD>PstI
</TD></TR><TR><TD>30  µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=49 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The volume of plasmid DNA added was calculated in order to have equal amounts of DNA of approximately 550 ng
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1000 bp DNA ladder
</TD><TD>Digestion of P477 with EcoRI &amp; PstI
</TD><TD>Digestion of P478 with EcoRI &amp; PstI
</TD><TD>Digestion of P479 with EcoRI &amp; PstI
</TD><TD>Digestion of P480 with EcoRI &amp; PstI
</TD><TD>Digestion of P481 with EcoRI &amp; PstI (control)
</TD><TD>Digestion of P482 with EcoRI &amp; PstI
</TD><TD>Digestion of P483 with EcoRI &amp; PstI
</TD><TD>Digestion of P484 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><UL><LI>all quickchanges were successfull.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Quikchange_mutagenesis_of_P477_.28Actin_promoter.29_to_insert_50_bp_deletion">  Quikchange mutagenesis of P477 (Actin promoter) to insert 50 bp deletion  </SPAN></H3><P><B>Investigator:</B> Rosario,  Ingmar
</P><P><B>Aim of the experiment:</B> Quikchange mutagenesis of P477 (Actin promoter) to insert 50 bp deletion.
</P><P><B>Procedure:</B></P><P><B>Reaction batch 1:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2,67 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O108 (=10 µM)
</TD></TR><TR><TD>17.330 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch 2:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2.67 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O109 (=10 µM)
</TD></TR><TR><TD>17.33 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD><B>=20 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The two batches were cycled seperately for the first 10 rounds.
</LI></UL><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>54°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 4.5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> After the ten cycles the two reaction batches were pooled together and were additionally cycled for 20 rounds with the following conditions:
</LI></UL><P><B>PCR cycling parameters after poooling the reaction batches together:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>98 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>98°C
</TD><TD> 10 sec
</TD></TR><TR><TD>54°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 2 min
</TD></TR><TR><TD>3
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P477.2C_P486.2C_P487.2C_P464.2C_P466.2C_P476.2C_P435">  Sequencing of P477, P486, P487, P464, P466, P476, P435 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Sequencing of P477 (Actin promoter), P486, P487, P464, P466, P476, P435.
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD> P477
</TD><TD> FR02233136 (O3)
</TD><TD> FR02233135 (O4)
</TD></TR><TR><TD> P486
</TD><TD> FR02233134 (O3)
</TD></TR><TR><TD> P487
</TD><TD> FR02233133 (O3)
</TD></TR><TR><TD> P464
</TD><TD> FR02233132 (O4)
</TD></TR><TR><TD> P466
</TD><TD> FR02233131 (O4)
</TD></TR><TR><TD> P476
</TD><TD> FR02233130 (O3)
</TD><TD> FR02233129 (O4)
</TD></TR><TR><TD> P435
</TD><TD> FR02233128 (O3)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P187_.28LMU_GFP.29"> Preparative digestion and gelelectrophoresis of P187 (LMU GFP) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P187 (LMU GFP).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P187 :</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P187
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P187 with NgoMIV &amp; SpeI
</TD><TD>2 log DNA ladder
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F109.2BF180_.28SERK-SigP_in_pSB1C3_.2B_LMU_GFP.29_and_F92.2BF180_.28IgKappa_in_pSB1C3_.2B_LMU_GFP.29"> Ligation of F109+F180 (SERK-SigP in pSB1C3 + LMU GFP) and F92+F180 (IgKappa in pSB1C3 + LMU GFP)  </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>  Ligation of F109+F180 (SERK-SigP in pSB1C3 + LMU GFP) and F92+F180 (IgKappa in pSB1C3 + LMU GFP).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F109+F180 (SERK-SigP in pSB1C3 + LMU GFP), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.1 µl
</TD><TD>F180 (726 bp)
</TD></TR><TR><TD>0.9 µl
</TD><TD>F179 (2177 bp)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>F92+F180 (IgKappa in pSB1C3 + LMU GFP), vector:insert = 1:3:
</P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.15 µl
</TD><TD>F180 (726 bp)
</TD></TR><TR><TD>0.85 µl
</TD><TD>F92 (2144 bp)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 22 °C for 1 h
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-blue_transformed_with_ligation_products_F170.2BF179_.28SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyTag.29.2C_F171.2BF178_.28SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyCatcher.29.2C_F176.2BF147_.28IgKappa-SigP_SpyCatcher_in_pSB1C3_.2B_NanoLuc.29.2C_F109.2BF177_.28SERK-SigP_in_pSB1C3_.2B_FluA_triple_mutant.29_and_F123.2BF175_.28pSB1C3_only_.2B_PP1.29.29"> Picking of <I>E. coli</I> XL1-blue transformed with ligation products F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) and F123+F175 (pSB1C3 only + PP1)) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-blue transformed with ligation products F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher), F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) and F123+F175 (pSB1C3 only + PP1)).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 2 colonies were picked for all ligation products except F176+F147 for which 4 colonies were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_9th"><B>Tuesday, July 9th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1_blue_transformed_with_F176.2BF147_.28IgKappa-SigP_SpyCatcher_in_pSB1C3_.2B_NanoLuc.29.2C_F123.2BF175_.28pSB1C3_only_.2B_PP1.29.2C_F109.2BF177_.28SERK-SigP_in_pSB1C3_.2B_FluA_triple_mutant.29.2C_F171.2BF178_.28SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyCatcher.29_and_F170.2BF179_.28SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_NanoLuc_SpyTag.29"> Miniprep of <I>E. coli</I> XL1 blue transformed with F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F123+F175 (pSB1C3 only + PP1), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher) and F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag) </SPAN></H3><P><B>Investigator:</B> Jeff, Florian
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1 blue transformed with F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc), F123+F175 (pSB1C3 only + PP1), F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant), F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher) and F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P488 to P499.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P488-P499_.28several_ligations.29"> Analytical digestion and gelelectrophoresis of P488-P499 (several ligations) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P488-P499 (several ligations). 
</P><UL><LI> 15x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>30 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>3.75 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>3.75 µl
</TD><TD>PstI
</TD></TR><TR><TD>225  µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=262,5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl master mix and 2.5 µl plasmid DNA were mixed.
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1 kb DNA ladder
</TD><TD>Digestion of P488 F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc) with EcoRI &amp; PstI
</TD><TD>Digestion of P489 F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc) with EcoRI &amp; PstI
</TD><TD>Digestion of P490 F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc) with EcoRI &amp; PstI
</TD><TD>Digestion of P491 F176+F147 (IgKappa-SigP_SpyCatcher in pSB1C3 + NanoLuc) with EcoRI &amp; PstI
</TD><TD>Digestion of P492 F123+F175 (pSB1C3 only + PP1) with EcoRI &amp; PstI
</TD><TD>Digestion of P493 F123+F175 (pSB1C3 only + PP1) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected (950 bp)
</TD><TD>as expected (950 bp)
</TD><TD>as expected (950 bp)
</TD><TD>as expected (950 bp)
</TD><TD>as expected (972 bp)
</TD><TD>as expected (972 bp)
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1 kb DNA ladder
</TD><TD>Digestion of P422 F123+F161 (FluA triple mutant + pSB1C3) with EcoRI &amp; PstI (control)
</TD><TD>Digestion of P494 F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) with EcoRI &amp; PstI
</TD><TD>Digestion of P495 F109+F177 (SERK-SigP in pSB1C3 + FluA triple mutant) with EcoRI &amp; PstI
</TD><TD>Digestion of P496 F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher) with EcoRI &amp; PstI
</TD><TD>Digestion of P497 F171+F178 (SERK-SigP_SpyTag_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyCatcher) with EcoRI &amp; PstI
</TD><TD>Digestion of P498 F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag) with EcoRI &amp; PstI
</TD><TD>Digestion of P499 F170+F179 (SERK-SigP_SpyCatcher_StrepTagII-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_NanoLuc_SpyTag) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected (522 bp)
</TD><TD>strange
</TD><TD>as expected (622 bp)
</TD><TD>roughly as expected (2768 bp)
</TD><TD>roughly as expected (2768 bp)
</TD><TD>roughly as expected (2768 bp)
</TD><TD>roughly as expected (2768 bp)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_QuikChange_products_of_P477_.28Actin_promoter.29_to_insert_50_bp_deletion"> Analytical gelelectrophoresis of QuikChange products of P477 (Actin promoter) to insert 50 bp deletion </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of QuikChange products of P477 (Actin promoter) to insert 50 bp deletion.
</P><P><B>Procedure:</B></P><UL><LI> 9 µl of the QuikChange products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1 kb DNA ladder
</TD><TD>QC of P477
</TD></TR><TR><TH><B>Result:</B></TH><TD>no product
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Quikchange_mutagenesis_of_P477_.28Actin_promoter.29_to_insert_60.C2.A0bp_deletion.2C_second_attempt">  Quikchange mutagenesis of P477 (Actin promoter) to insert 60 bp deletion, second attempt </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Quikchange mutagenesis of P477 (Actin promoter) to insert 60 bp deletion.
</P><P><B>Procedure:</B></P><P><B>Reaction batch 1:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>3 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O108 (=10 µM)
</TD></TR><TR><TD>17.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch 2:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>3.0 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O109 (=10 µM)
</TD></TR><TR><TD>17.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD><B>=20 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The two batches were cycled seperately for the first 25 rounds.
</LI></UL><P><B>PCR cycling parameters</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>52°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 4.5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> After the ten cycles the two reaction batches were pooled together, 0.5 µl of polymerase and 1 µl of dNTP-mix was added and the reaction was cycled for 25 rounds with the following conditions:
</LI></UL><P><B>PCR cycling parameters after poooling the reaction batches together:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>98 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>98°C
</TD><TD> 10 sec
</TD></TR><TR><TD>52°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 2 min
</TD></TR><TR><TD>3
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C. 
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_10th"><B>Wednesday, July 10th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P495_.28SERK-SigP_FluA.29"> Preparative digestion of P495 (SERK-SigP_FluA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P495 (SERK-SigP_FluA).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P495 (SERK-SigP_FluA):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P495
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> After digestion, linearized DNA was purified with QIAquick PCR purification kit, QIAGEN.
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P486_.28.28GGGGS.29x5-TEV-site-linker.29.2C_P488_.28IgKappa-SigP_SpyCatcher_NanoLuc.29_and_P492_.28Proteinphosphotase_1.29"> Preparative digestion and gelelectrophoresis of P486 ((GGGGS)x5-TEV-site-linker), P488 (IgKappa-SigP_SpyCatcher_NanoLuc) and P492 (Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P486 ((GGGGS)x5-TEV-site-linker), P488 (IgKappa-SigP_SpyCatcher_NanoLuc) and P492 (Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P486 ((GGGGS)x5-TEV-site-linker):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P486
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P488 (IgKappa-SigP_SpyCatcher_NanoLuc):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P488
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P492 (Proteinphosphotase 1)):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P492
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (10 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P486 with XbaI &amp; AgeI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P488 with XbaI &amp; PstI
</TD><TD>Digestion of P492 with NgoMIV &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F156.2BF182_.28NLS_NucA_in_pSB1C3_.2B_.28GGGGS.29x5-TEV-site-linker.29.2C_F181.2BF135_.28SERK-SigP_FluA_in_pSB1C3_.2B_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_F171.2BF183_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_SpyCatcher_NanoLuc.29.2C_F109.2BF184_.28SERK-SigP_in_pSB1C3_.2B_Proteinphosphotase_1.29"> Ligation of F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  Ligation of F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.63 µl
</TD><TD>F156 (158.3 ng/µl, 2596 bp)
</TD></TR><TR><TD>2.88 µl
</TD><TD>F182 (4.1 ng/µl, 102 bp)
</TD></TR><TR><TD>13.49 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.54 µl
</TD><TD>F181 (185.7 ng/µl, 2705 bp)
</TD></TR><TR><TD>6.21 µl
</TD><TD>F135 (17.8 ng/µl, 996 bp)
</TD></TR><TR><TD>10.25 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.65 µl
</TD><TD>F171 (60.5 ng/µl, 3859 bp)
</TD></TR><TR><TD>1.22 µl
</TD><TD>F183 (59 ng/µl, 928 bp)
</TD></TR><TR><TD>14.13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.92 µl
</TD><TD>F109 (108.2 ng/µl, 2117 bp)
</TD></TR><TR><TD>2.44 µl
</TD><TD>F184 (54.9 ng/µl, 972 bp)
</TD></TR><TR><TD>13.64 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_11th"><B>Thursday, July 11th</B></SPAN></H2><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_F156.2BF182_.28NLS_NucA_in_pSB1C3_.2B_.28GGGGS.29x5-TEV-site-linker.29.2C_F181.2BF135_.28SERK-SigP_FluA_in_pSB1C3_.2B_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_F171.2BF183_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_SpyCatcher_NanoLuc.29.2C_F109.2BF184_.28SERK-SigP_in_pSB1C3_.2B_Proteinphosphotase_1.29"> Transformation of <I>E. coli</I> XL1-Blue with ligation products F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA  were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Sequencing_of_P294.2C_P296.2C_P342.2C_P488.2C_P492.2C_P495.2C_P496_and_P498">  Sequencing of P294, P296, P342, P488, P492, P495, P496 and P498 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Sequencing of P294, P296, P342, P488, P492, P495, P496 and P498.
</P><P><B>Procedure:</B></P><UL><LI> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</LI></UL><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH><TH> Barcode3
</TH></TR><TR><TD> P294
</TD><TD> FR02233072 (O100)
</TD></TR><TR><TD> P296
</TD><TD> FR02233073 (O3)
</TD><TD> FR02233074 (O4)
</TD><TD> FR02233075 (O100)
</TD></TR><TR><TD> P342
</TD><TD> FR02233076 (O3)
</TD><TD> FR02233077 (O4)
</TD><TD> FR02233078 (O100)
</TD></TR><TR><TD> P488
</TD><TD> FR02233079 (O4)
</TD></TR><TR><TD> P492
</TD><TD> FR02233080 (O3)
</TD><TD> FR02233081 (O4)
</TD></TR><TR><TD> P495
</TD><TD> FR02233082 (O3)
</TD></TR><TR><TD> P496
</TD><TD> FR02233083 (O3)
</TD><TD> FR02233084 (O4)
</TD></TR><TR><TD> P498
</TD><TD> FR02233085 (O3)
</TD><TD> FR02233086 (O4)
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_12th"><B>Friday, July 12th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="QuikChange_of_P477_.28pActin.29_to_insert_missing_60.C2.A0bp"> QuikChange of P477 (pActin) to insert missing 60 bp </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> QuikChange of P477 (pActin) to insert missing 60 bp.
</P><P><B>Procedure:</B></P><P><B>Reaction batch #1, with 5% DMSO and O108 as oligonucleotide:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O108 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>1.25 µl
</TD><TD>DMSO
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD>14.75 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch #2, with 5% DMSO and O109 as oligonucleotide:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O109 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>1.25 µl
</TD><TD>DMSO
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD>14.75 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch #3, O108 as oligonucleotide:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O108 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch #4, O109 as oligonucleotide:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P477 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O109 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Every reaction batch type was created twice; one batch of each type was used for the program with 55 °C as annealing temperature and one was used with 65 °C as annealing temperature.
</LI></UL><UL><LI> The batches were cycled seperately for the first 10 rounds
</LI></UL><P><B>PCR cycling parameters with 55 °C annealing temperature:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 4.5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters with 65 °C annealing temperature:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>65°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 4.5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> After the ten cycles the reaction batches (#1 with #2 and #3 with #4) were pooled together and the reaction was cycled for further 20 rounds with the following conditions:
</LI></UL><P><B>PCR cycling parameters with 55 °C annealing temperature:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 4.5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters with 65 °C annealing temperature:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>20
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>65°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 4.5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the pooled reaction batches and incubate for 1 h at 37 °C.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_QuikChange_products_of_P477_.28Actin_promoter.29_to_insert_60.C2.A0bp_deletion"> Analytical gelelectrophoresis of QuikChange products of P477 (Actin promoter) to insert 60 bp deletion </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of QuikChange products of P477 (Actin promoter) to insert 60 bp deletion.
</P><P><B>Procedure:</B></P><UL><LI> 9 µl of the QuikChange products were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 100 V for 40 min on a 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>QC of P477 with DMSO and annealing temperature at 55 °C
</TD><TD>QC of P477 with DMSO and annealing temperature at 65 °C
</TD><TD>QC of P477 with annealing temperature at 55 °C
</TD><TD>QC of P477 with annealing temperature at 65 °C
</TD></TR><TR><TH><B>Result:</B></TH><TD>no product
</TD><TD>no product
</TD><TD>no product
</TD><TD>no product
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_ligation_products_F156.2BF182_.28NLS_NucA_in_pSB1C3_.2B_.28GGGGS.29x5-TEV-site-linker.29.2C_F181.2BF135_.28SERK-SigP_FluA_in_pSB1C3_.2B_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_F171.2BF183_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_SpyCatcher_NanoLuc.29.2C_F109.2BF184_.28SERK-SigP_in_pSB1C3_.2B_Proteinphosphotase_1.29"> Picking of <I>E. coli</I> XL1-Blue transformed with ligation products F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed with ligation products F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol (1000x).
</LI></UL><UL><LI> 2 colonies were picked for F156+F182, F171+F182, F109+F184 and 6 colonies were picked for ligation product F181+F135.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_July_13th"><B>Saturday, July 13th</B></SPAN></H2><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1-Blue_transformed_with_ligation_products_F156.2BF182_.28NLS_NucA_in_pSB1C3_.2B_.28GGGGS.29x5-TEV-site-linker.29.2C_F181.2BF135_.28SERK-SigP_FluA_in_pSB1C3_.2B_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_F171.2BF183_.28SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_in_pSB1C3_.2B_IgKappa-SigP_SpyCatcher_NanoLuc.29.2C_F109.2BF184_.28SERK-SigP_in_pSB1C3_.2B_Proteinphosphotase_1.29"> Miniprep of <I>E. coli</I> XL1-Blue transformed with ligation products F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1-Blue transformed with ligation products F156+F182 (NLS_NucA in pSB1C3 + (GGGGS)x5-TEV-site-linker), F181+F135 (SERK-SigP_FluA in pSB1C3 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), F171+F183 (SERK-SigP_SpyTag_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES in pSB1C3 + IgKappa-SigP_SpyCatcher_NanoLuc), F109+F184 (SERK-SigP in pSB1C3 + Proteinphosphotase 1).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P508 to P519.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P508.C2.A0.E2.80.93.C2.A0P519"> Analytical digestion and gelelectrophoresis of P508 – P519 </SPAN></H3><P><B>Investigator:</B> Le Swaffeleur
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P508 – P519.
</P><P><B>Procedure:</B></P><UL><LI> 15x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>26 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>3.25 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>3.25 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>195 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=227.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl master mix and 2.5 µl plasmid DNA (P508 – P519) were mixed.
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P508, digested with EcoRI &amp; PstI
</TD><TD>P509, digested with EcoRI &amp; PstI
</TD><TD>P510, digested with EcoRI &amp; PstI
</TD><TD>P511, digested with EcoRI &amp; PstI
</TD><TD>P512, digested with EcoRI &amp; PstI
</TD><TD>P513, digested with EcoRI &amp; PstI
</TD><TD>P514, digested with EcoRI &amp; PstI
</TD><TD>P515, digested with EcoRI &amp; PstI
</TD><TD>P516, digested with EcoRI &amp; PstI
</TD><TD>P517, digested with EcoRI &amp; PstI
</TD><TD>P518, digested with EcoRI &amp; PstI
</TD><TD>P519, digested with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_July_14th"><B>Sunday, July 14th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P428_.28mMCSII_t35S_npt-casette.29_and_P477_.28pAct.29"> Preparative digestion and gelelectrophoresis of P428 (mMCSII_t35S_npt-casette) and P477 (pAct) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P428 (mMCSII_t35S_npt-casette) and P477 (pAct).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P428 (MCS-t35S-npt):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>15 µl
</TD><TD>P428
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>19 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P477 (pAct):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>15 µl
</TD><TD>P477
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (10 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 4 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 2 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P486 with XbaI &amp; AgeI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P488 with XbaI &amp; PstI
</TD><TD>2 log DNA ladder
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; intense band was cut out
</TD><TD>as expected; lowest band was cut out
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F185.2BF186_.28pAct_.2B_MCS_t35S_npt.29"> Ligation of F185+F186 (pAct + MCS_t35S_npt)</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F185+F186 (pAct + MCS_t35S_npt), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4 µl
</TD><TD>F185 (5000 bp)
</TD></TR><TR><TD>4 µl
</TD><TD>F186 (1200 bp)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 22 °C for 1 h
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P430_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES.29.2C_P431_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES.29.2C_P509_.28.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_P510_.28SERK-SigP_PP1.29"> Preparative digestion and gelelectrophoresis of P430 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES), P431 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES), P509 ((GGGGS)x5-TEV-site-linker_NLS_NucA) and P510 (SERK-SigP_PP1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P430 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES), P431 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES), P509 ((GGGGS)x5-TEV-site-linker_NLS_NucA) and P510 (SERK-SigP_PP1).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P430 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P430
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P431 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P431
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P509 ((GGGGS)x5-TEV-site-linker_NLS_NucA):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P509
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P510 (SERK-SigP_PP1)):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P510
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P430 with XbaI &amp; AgeI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P431 with XbaI &amp; PstI
</TD><TD>Digestion of P509 with NgoMIV &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P510 with AgeI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F122.2BF189_.28SERK-SigP_SERK-TMD_.2B_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29.2C_F190.2BF135_.28SERK-SigP_PP1_.2B_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29"> Ligation of F122+F189 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F190+F135 (SERK-SigP_PP1 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>  Ligation of F122+F189 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F190+F135 (SERK-SigP_PP1 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F122+F189 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.99 µl
</TD><TD>F122 (50.3 ng/µl, 2369 bp)
</TD></TR><TR><TD>0.98 µl
</TD><TD>F189 (79.9 ng/µl, 618 bp)
</TD></TR><TR><TD>14.03 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F190+F135 (SERK-SigP_PP1 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.45 µl
</TD><TD>F190 (224.7 ng/µl, 3155 bp)
</TD></TR><TR><TD>5.32 µl
</TD><TD>F135 (17.8 ng/µl, 996 bp)
</TD></TR><TR><TD>11.23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_13">Week 13</SPAN></H1><DIV class="week" id="WWeek_13"><H2><SPAN class="mw-headline" id="Monday.2C_July_15th"><B>Monday, July 15th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_the_genes_to_be_expressend_in_moss_.28retransformation.29_and_F122.2BF189_.28SERK-SigP_SERK-TMD_.2B_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29.2C_F190.2BF135_.28SERK-SigP_PP1_.2B_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29"> Transformation of the genes to be expressend in moss (retransformation) and F122+F189 (SERK-SigP_SERK-TMD + (GGGGS)x5-TEV-site-linker_NLS_NucA), F190+F135 (SERK-SigP_PP1 + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure for the retransformations:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 0.5 µl of DNA  were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> 50 µl of this  cell suspension was plated on chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_16th"><B>Tuesday, July 16th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_all_effectors"> Preparative digestion and gelelectrophoresis of all effectors </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><P><B>Mastermix:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>224
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>64 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>16 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR></TBODY></TABLE><UL><LI> 20 µl of DNA were mixed with 20 µl of Mastermix and were incubated at 37 °C for 4 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P20 (GFP) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P196 (XylE)  with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P245 (EreB)  with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P246 (nLuc) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P330 (SERK-SigP_NLuc) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P346 (Dehydrochlorinase) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><P>
The digestion of P350 was separated for an additional hour because there might be contaminations of the vector that is smaller than the protein-coding BioBrick we want to express. The separated bands looked like that: </P><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P350 (EreB-Receptor) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P354 (nLuc-Receptor) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P402 (IgK-nLuc) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of  P476  (SpyCatcher-Receptor_IRES_IgK_SpyTag-nLuc) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of  P496 (SpyTag-Receptor_IRES_nLuc_SpyCatcher with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P498 (SpyCatcher-Receptor_IRES_IgK_nLuc-SpyTAG) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P502 (IgK-GFP) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P512 (SpyTAG-Receptor-IRES_SpyCatcher-nLuc) with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P516 (FluA-Recetpro) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_P520_to_P525:_P520_.28Bba_KK509000.2CCauliflower_Mosaic_Virus_35S_promoter.29_P521-525_.28pAct.28with_del.29-MCS-T35S-npt"> Miniprep of P520 to P525: P520 (Bba_KK509000,Cauliflower Mosaic Virus 35S promoter) P521-525 (pAct(with_del)-MCS-T35S-npt </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P520 to P525.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P520_-_P525"> Analytical digestion and gelelectrophoresis of P520 - P525 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI> 7x Mastermix (EcoRI-HF &amp; PstI-HF) for analytical digestion:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>7 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>1.75 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>38.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=49 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 7 µl master mix and 3 µl plasmid DNA  were mixed.
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1.5bsp;h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P428, dsted with EcoRI &amp; PstI
</TD><TD>P522, digested with EcoRI &amp; PstI
</TD><TD>P522, digested with EcoRI &amp; PstI
</TD><TD>P523, digested with EcoRI &amp; PstI
</TD><TD>P524, digested with EcoRI &amp; PstI
</TD><TD>P525, digested with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>control
</TD><TD>no insert ;(
</TD><TD>no insert ;(
</TD><TD>no insert ;(
</TD><TD>no insert ;(
</TD><TD>no insert ;(
</TD><TD>no insert ;(
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_17th"><B>Wednesday, July 17th</B></SPAN></H2><H2><SPAN class="mw-headline" id="Thursday.2C_July_18th"><B>Thursday, July 18th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P537.C2.A0.E2.80.93.C2.A0P540_and_analytical_gelelectrophoresis_of_F206_and_F207"> Analytical digestion and gelelectrophoresis of P537 – P540 and analytical gelelectrophoresis of F206 and F207 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P537 – P540 and analytical gelelectrophoresis of F206 and F207.
</P><P><B>Procedure:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P537 – P540)
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P537 F122+F189 (SERK-SigP_SERK-TMD in pSB1C3 + (GGGGS)x5-TEV-site-linker_NLS_NucA) with EcoRI &amp; PstI
</TD><TD>Digestion of P538 F122+F189 (SERK-SigP_SERK-TMD in pSB1C3 + (GGGGS)x5-TEV-site-linker_NLS_NucA) with EcoRI &amp; PstI
</TD><TD>Digestion of P539 F190+F135 (SERK-SigP_PP1 in pSB1C3 only + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) with EcoRI &amp; PstI
</TD><TD>Digestion of P540 F190+F135 (SERK-SigP_PP1 in pSB1C3 only + Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1) with EcoRI &amp; PstI
</TD><TD>F206
</TD><TD>F207
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected (916 bp)
</TD><TD>as expected (916 bp)
</TD><TD>as expected (2080 bp)
</TD><TD>as expected (2080 bp)
</TD><TD>as expected (3484 bp)
</TD><TD>as expected (3484 bp)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_to_insert_60_missing_base-pairs_into_pActin_.28new_method.29"> PCR to insert 60 missing base-pairs into pActin (new method) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR to insert 60 missing base-pairs into pActin (new method)
</P><P><B>Procedure:</B></P><P><B>Reaction batch for PCR with Q5 Mastermixes for P477 (pActin):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O114 (pAct_ins_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O115 (pAct_ins_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P477) - 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O114 (pAct_ins_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O115 (pAct_ins_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P477) - 1:100 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><P><B>Cycling parameters:</B></P><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>72 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> The resulting product was labeled as F206 for the 1:1000 batch and F207 for the 1:100 batch.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_19th"><B>Friday, July 19th</B></SPAN></H2><H2><SPAN class="mw-headline" id="Saturday.2C_July_20th"><B>Saturday, July 20th</B></SPAN></H2><H2><SPAN class="mw-headline" id="Sunday.2C_July_21st"><B>Sunday, July 21st</B></SPAN></H2></DIV><H1><SPAN class="mw-headline" id="Week_14">Week 14</SPAN></H1><DIV class="week" id="WWeek_14"><H2><SPAN class="mw-headline" id="Monday.2C_July_22nd"><B>Monday, July 22nd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Arrival_of_the_correct_pActin_5_from_Physcomitrella_patens_from_Reski_lab">Arrival of the correct pActin 5 from Physcomitrella patens from Reski lab</SPAN></H3><P>This plasmid was named p541
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_23rd"><B>Tuesday, July 23rd</B></SPAN></H2><H2><SPAN class="mw-headline" id="Wednesday.2C_July_24th"><B>Wednesday, July 24th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="PCR_of_P541_.28PppActin5.29"> PCR of P541 (PppActin5) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR of P541 (PppActin5).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for PCR with Q5 Mastermix for P541 (PppActin5):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix (normally not necessary!)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O116 (PppAct_f)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O117 (PppAct_r)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P541) - 1:1000 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>18 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been mixed with a pipette
</LI></UL><UL><LI> The PCR program was performed after following scheme:
</LI></UL><P><B>Cycling parameters:</B></P><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>53 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>30 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> The resulting product was labeled as F208.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F208"> Analytical gelelectrophoresis of F208 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F208.
</P><P><B>Procedure:</B></P><UL><LI> 5 µl of the PCR product F208 (=PCR of P541 with O116/O117) products were mixed with 2 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>F208
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P208_.28PCR_of_PppActin5.29"> Preparative digestion of P208 (PCR of PppActin5) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><P><B>Reaction batch for F208 (PCR of PppActin5):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>F208
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=45 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C for 2 h.
</LI></UL><UL><LI> After digestion, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_and_transformation_of_F209_.2B_F41_.28PppActin5_in_pSB1C3.29"> Ligation  and transformation of F209 + F41 (PppActin5 in pSB1C3)</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F209 (PppActin5), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>7 µl
</TD><TD>F209 (36.9 ng/µl, ??? bp)
</TD></TR><TR><TD>1 µl
</TD><TD>F189 (89.9 ng/µl, ??? bp)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at RT for 1 hour.
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL><P>The ligation products were transformed into <I>E. coli</I> XL-1 Blue.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_25th"><B>Thursday, July 25th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1_blue_transformed_with_ligation_product_of_F208_and_F41_.28PppActin5_in_pSB1C3.29"> Picking of <I>E. coli</I> XL1 blue transformed with ligation product of F208 and F41 (PppActin5 in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol (1000x).
</LI></UL><UL><LI> 6 colonies were picked for ligation of F208 and F41 (PppActin5 in pSB1C3).
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_26th"><B>Friday, July 26th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_P542_to_P547_.28pSB1C2_PppActin5.29"> Miniprep of P542 to P547 (pSB1C2_PppActin5) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P542 to P547.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P542.C2.A0.E2.80.93.C2.A0P547"> Analytical digestion and gelelectrophoresis of P542 – P547 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA (P542 – P547)
</TD></TR><TR><TD>1 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>5.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P542 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P543 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P544 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P545 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P546 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P547 (PppActin5) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>result unclear because insert and backbone have the same length!
</TD><TD>result unclear because insert and backbone have the same length!
</TD><TD>result unclear because insert and backbone have the same length!
</TD><TD>result unclear because insert and backbone have the same length!
</TD><TD>result unclear because insert and backbone have the same length!
</TD><TD>result unclear because insert and backbone have the same length!
</TD></TR></TBODY></TABLE><P>Unfortunately the correctness of the clones could not be seen in this experiment.
Thus a second digestion was performed:
</P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA (P542 – P547)
</TD></TR><TR><TD>1 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>NcoI (form Fermentas
</TD></TR><TR><TD>5.25 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI>Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P542 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P543 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P544 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P545 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P546 (PppActin5) with EcoRI &amp; PstI
</TD><TD>Digestion of P547 (PppActin5) with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>Clone correct: backbone and PppActin5
</TD><TD>Clone wrong: only backbone, no insert
</TD><TD>Clone correct: backbone and PppActin5
</TD><TD>Clone correct: backbone and PppActin5
</TD><TD>Clone wrong: only backbone, no insert
</TD><TD>Clone correct: backbone and PppActin5
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P544_and_P545">  Sequencing of P544 and P545</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1
</TH><TH> Barcode2
</TH></TR><TR><TD>P544
</TD><TD>FR02233095
</TD><TD>FR02233087
</TD></TR><TR><TD>P545
</TD><TD>FR02233093
</TD><TD>FR02233094
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_July_27th"><B>Saturday, July 27th</B></SPAN></H2><H2><SPAN class="mw-headline" id="Sunday.2C_July_28st"><B>Sunday, July 28st</B></SPAN></H2></DIV><H1><SPAN class="mw-headline" id="Week_15">Week 15</SPAN></H1><DIV class="week" id="WWeek_15"><H2><SPAN class="mw-headline" id="Monday.2C_July_29th"><B>Monday, July 29th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="QuikChange_of_P544_.28PppActin5.29"> QuikChange of P544 (PppActin5)  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P544 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O118 (=10 µM)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O119 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P544 template (1:100)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O120 (=10 µM)
</TD></TR><TR><TD>1.0 µl
</TD><TD>1:10 dilution of O121 (=10 µM)
</TD></TR><TR><TD>1 µl
</TD><TD>dNTP mix 50 mM
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U/µl)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD><B>=25 µl</B></TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>PCR cycling parameters with 55 °C annealing temperature:</B></P><TABLE border="1"><TBODY><TR><TH><B>Cycles</B></TH><TH><B>Temperature</B></TH><TH><B>Time</B></TH></TR><TR><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 5 min
</TD></TR><TR><TD>1
</TD><TD>4°C
</TD><TD> HOLD
</TD></TR></TBODY></TABLE><UL><LI> Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the pooled reaction batches and incubate for 1 h at 37 °C.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_30th"><B>Tuesday, July 30th</B></SPAN></H2><H2><SPAN class="mw-headline" id="Wednesday.2C_July_31st"><B>Wednesday, July 31st</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P537_.28SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29"> Preparative digestion and gelelectrophoresis of P537 (SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P537 (SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P537 (SERK-SigP_SERK-TMD_(GGGGS)-TEV-site-linker_NLS_NucA):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P537
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P537 with XbaI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL><UL><LI> Resulting fragment was named as F210.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F187.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_F188.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29"> Ligation of F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA, vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.53 µl
</TD><TD>F187 (189.9 ng/µl, 7346 bp)
</TD></TR><TR><TD>0.68 µl
</TD><TD>F210 (55.3 ng/µl, 916 bp)
</TD></TR><TR><TD>15.79 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA, vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.73 µl
</TD><TD>F188 (136.7 ng/µl, 7346 bp)
</TD></TR><TR><TD>0.68 µl
</TD><TD>F210 (55.3 ng/µl, 916 bp)
</TD></TR><TR><TD>15.59 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL><UL><LI> Negative controls were also prepared with water instead of insert.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_1st"><B>Thursday, August 1st</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_BBa_K864100_.28SYFP2.29.2C_BBa_E0020_.28eCFP.29_and_ligation_products_F187.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_F188.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29"> Transformation of <I>E. coli</I> XL1-Blue with BBa_K864100 (SYFP2), BBa_E0020 (eCFP) and ligation products F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with BBa_K864100 (SYFP2), BBa_E0020 (eCFP) and ligation products F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_2nd"><B>Friday, August 2nd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_BBa_K864100_.28SYFP2.29.2C_BBa_E0020_.28eCFP.29_and_ligation_products_F187.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_F188.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29"> Picking of <I>E. coli</I> XL1-Blue transformed with BBa_K864100 (SYFP2), BBa_E0020 (eCFP) and ligation products F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed with BBa_K864100 (SYFP2), BBa_E0020 (eCFP) and ligation products F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA).
</P><P><B>Procedure:</B></P><UL><LI> Picked colonies were transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contains 4 mL of LB-medium + 4 µL chloramphenicol (1000x).
</LI></UL><UL><LI> 2 colonies were picked for BBa_K864100, BBa_E0020, F187+F210 and F188+F210 each.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated at 180 rpm overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_3rd"><B>Saturday, August 3rd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_Xl1-Blue_transformed_with_BBa_K864100_.28SYFP2.29.2C_BBa_E0020_.28eCFP.29_and_ligation_products_F187.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_F188.2BF210_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_in_pSB1C3_.2B_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29"> Miniprep of <I>E. coli</I> Xl1-Blue transformed with BBa_K864100 (SYFP2), BBa_E0020 (eCFP) and ligation products F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> Xl1-Blue transformed with BBa_K864100 (SYFP2), BBa_E0020 (eCFP) and ligation products F187+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and F188+F210 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P558 to P565.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P558.C2.A0.E2.80.93.C2.A0P565"> Analytical digestion and gelelectrophoresis of P558 – P565 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P558 – P565.
</P><P><B>Procedure:</B></P><P><B>Mastermix for analytical digestion with EcoRI and PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>2.5 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>150 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=175 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were mixed together with 2.5 µl of Plasmid DNA (P558 – P565)
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>Digestion of P558 (BBa_K864100)
</TD><TD>Digestion of P559 (BBa_K864100)
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P560 (BBa_E0020)
</TD><TD>Digestion of P561 (BBa_E0020)
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected (723 bp)
</TD><TD>as expected (723 bp)
</TD><TD>as expected (723 bp)
</TD><TD>as expected (723 bp)
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>Digestion of P562 (F187+F210)
</TD><TD>Digestion of P563 (F187+F210)
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P564 (F188+F210)
</TD><TD>Digestion of P565 (F188+F210)
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected (6191 bp)
</TD><TD>as expected (6191 bp)
</TD><TD>not as expected; corrupt (6191 bp)
</TD><TD>as expected (6191 bp)
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_4th"><B>Sunday, August 4th</B></SPAN></H2></DIV><H1><SPAN class="mw-headline" id="Week_16">Week 16</SPAN></H1><DIV class="week" id="WWeek_16"><H2><SPAN class="mw-headline" id="Monday.2C_August_5th"><B>Monday, August 5th</B></SPAN></H2><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P540_.28SERK-SigP_PP1_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1.29.2C_P558_.28SYFP2.29.2C_P560_.28eCFP.29.2C_P562_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29_and_P565_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_SERK-SigP_SERK-TMD_.28GGGGS.29x5-TEV-site-linker_NLS_NucA.29"> Preparative digestion and gelelectrophoresis of P540 (SERK-SigP_PP1_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P558 (SYFP2), P560 (eCFP), P562 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and P565 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P540 (SERK-SigP_PP1_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P558 (SYFP2), P560 (eCFP), P562 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA) and P565 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_SERK-SigP_SERK-TMD_(GGGGS)x5-TEV-site-linker_NLS_NucA).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P540:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P540
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>4 µl
</TD><TD>XhoI (10 U/µl)
</TD></TR><TR><TD>8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P558, P560, P562, P565:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P558, P560, P562, P565
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P540 with EcoRI &amp; PstI &amp; XhoI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P558 with EcoRI &amp; PstI
</TD><TD>Digestion of P560 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; upper band was cut out
</TD><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P562 with EcoRI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>Digestion of P565 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; upper band was cut out
</TD><TD>as expected; upper band was cut out
</TD></TR></TBODY></TABLE><UL><LI> DNA bands were extracted with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL><UL><LI> Resulting fragment was named as F214 – F218.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="PCR_of_P558_.28SYFP2.29_and_P560_.28eCFP.29_to_introduce_RFC25_pre-_and_suffixes"> PCR of P558 (SYFP2) and P560 (eCFP) to introduce RFC25 pre- and suffixes </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR of P558 (SYFP2) and P560 (eCFP) to introduce RFC25 pre- and suffixes.
<B>Procedure:</B></P><P><B>Reaction batch for PCR with Q5 Mastermixes for P558 (SYFP2):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O122 (SYFP2_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O123 (SYFP2_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P558) - 1:100 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for PCR with Q5 Mastermixes for P560 (eCFP):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O124 (eCFP_fw)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O125 (eCFP_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P560) - 1:100 dilution (desired c between 1 pg and 1 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The content has been created on ice and mixed with a pipette
</LI></UL><UL><LI> The gradient PCR program was performed after following scheme with following conditions (T<SUB>m</SUB>=67 °C; ΔG=2 °C; P558 in row 1(=65.0 °C); P560 in row 12 (=69.2 °C)):
</LI></UL><P><B>Cycling parameters:</B></P><TABLE border="1"><TBODY><TR><TH>Initial denaturation
</TH><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TH>30 cycles
</TH><TD>98 °C
</TD><TD>10 s
</TD></TR><TR><TD>T<SUB>m</SUB>=67 °C; ΔG=2 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>22 s
</TD></TR><TR><TH>Final extension
</TH><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TH>Hold
</TH><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> The resulting product was labeled as F212 for the the PCR products of P558 and F213 for the PCR product of P560.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F212_.28SYFP2_RFC25.29_and_F213_.28eCFP_RFC25.29"> Analytical gelelectrophoresis of F212 (SYFP2 RFC25) and F213 (eCFP RFC25) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F212 (SYFP2 RFC25) and F213 (eCFP RFC25).
</P><P><B>Procedure:</B></P><UL><LI> 4.5 µl of the PCR product F212 and F213 products were mixed with 0.5 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>F212
</TD><TD>2 log DNA ladder
</TD><TD>F213
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_6th"><B>Tuesday, August 6th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Oligohybridization_of_O128.28G-TEV-site-G_fw.29_and_O129_.28G-TEV-site-G_rv.29">  Oligohybridization of O128(G-TEV-site-G_fw) and O129 (G-TEV-site-G_rv) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment: </B> Oligohybridization of O128(G-TEV-site-G_fw) and O129 (G-TEV-site-G_rv).
</P><P><B>Procedure:</B></P><UL><LI> 25 µL of  O128 (100 µM) and 25 µL of O129 (100 µM) were pooled together.
</LI></UL><UL><LI> Heating up to 95 °C for 5 min.
</LI></UL><UL><LI> Cooling at RT in a styropor box overnight.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_F212_.28SYFP2_RFC25.29_and_F213_.28eCFP_RFC25.29"> Preparative digestion of F212 (SYFP2 RFC25) and F213 (eCFP RFC25) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of F212 (SYFP2 RFC25) and F213 (eCFP RFC25).
</P><P><B>Procedure:</B></P><P><B>Reaction batch for F212:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>F212
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for F213:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>F213
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2 h.
</LI></UL><UL><LI> Digested products were purified with QIAquick PCR Purification Kit, QIAGEN.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F123.2BF219_.28pSB1C3_.2B_SYFP2.29_and_F123.2BF220_.28eCFP.29"> Ligation of F123+F219 (pSB1C3 + SYFP2) and F123+F220 (eCFP) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F123+F219 (pSB1C3 + SYFP2) and F123+F220 (eCFP).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F123+F219 (pSB1C3 + SYFP2), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1 µl
</TD><TD>F123 (49.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>0.4 µl
</TD><TD>F219 (114.9 ng/µl, 723 bp)
</TD></TR><TR><TD>15.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F123+F220 (pSB1C3 + eCFP), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1 µl
</TD><TD>F123 (49.3 ng/µl, 2086 bp)
</TD></TR><TR><TD>1.2 µl
</TD><TD>F220 (41.7 ng/µl, 723 bp)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_further_clones"> Analytical digestion and gelelectrophoresis of further clones</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B>
The plasmids were cut with 0.2 µl of EcoRI and SpeI in a volume of 10 µl.
</P><UL><LI> Analytical digestion was performed at 37 °C for 2 h.
</LI></UL><UL><LI> After incubation 3  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>clone 1
</TD><TD>clone 2
</TD><TD>clone 3
</TD><TD>clone 4
</TD><TD>clone 5
</TD><TD>clone 6
</TD><TD>clone 7
</TD><TD>clone 8
</TD></TR><TR><TH><B>Result:</B></TH><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P428_.28mMCS-T35S-npt.29_and_P554_.28PppAct5-QCII.29"> Preparative digestion of P428 (mMCS-T35S-npt) and P554 (PppAct5-QCII) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P428:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P428
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P554:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>F213
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NcoI (10 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P428 with EcoRI &amp; XbaI
</TD><TD>Digestion of P554 with EcoRI &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; highest band was cut out
</TD><TD>as expected; highest band was cut out
</TD></TR></TBODY></TABLE><P>The bands were extracted from the gel and termed F221 and F222. The concentrations were determined to 42 and 37 ng/µl.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F221.2BF222_.28PppAct5_.2B_pSB1C3-mMCS-T35S-npt.29"> Ligation of F221+F222 (PppAct5 + pSB1C3-mMCS-T35S-npt) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F221+F222 (PppAct5 + pSB1C3-mMCS-T35S-npt), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>5 µl
</TD><TD>F221 (42 ng/µl, 3700 bp)
</TD></TR><TR><TD>11 µl
</TD><TD>F222 (37 ng/µl, 2000 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_F221.2BF222"> Transformation of <I>E. coli</I> XL1-Blue with ligation F221+F222 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI>After 1 h of ligation 10 µl were used for transformation.
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_7th"><B>Wednesday, August 7th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_F221.2BF222_2"> Transformation of <I>E. coli</I> XL1-Blue with ligation F221+F222 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI>After 1 h of ligation 10 µl were used for transformation.
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_with_PppAct5-mMCS-T35S-npt_from_the_ligation_performed_after_1_h"> Picking of of <I>E. coli</I> XL1 blue with PppAct5-mMCS-T35S-npt from the ligation performed after 1 h </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Procedure:</B></P><UL><LI> PppAct5-mMCS-T35S-npt: Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 12 colonies were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_8th"><B>Thursday, August 8th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_further_clones_2"> Analytical digestion and gelelectrophoresis of further clones</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B>
The plasmids were cut with 0.2 µl of EcoRI and SpeI in a volume of 10 µl.
</P><UL><LI> Analytical digestion was performed at 37 °C for 2 h.
</LI></UL><UL><LI> After incubation 3  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>clone 1
</TD><TD>clone 2
</TD><TD>clone 3
</TD><TD>clone 4
</TD><TD>clone 5
</TD><TD>clone 6
</TD><TD>clone 7
</TD><TD>clone 8
</TD><TD>clone 9
</TD><TD>clone 10
</TD></TR><TR><TH><B>Result:</B></TH><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD></TR></TBODY></TABLE><P>All these clones were discarded.

Additionally the fragments for the ligation were controlled.
</P><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>F185
</TD><TD>F211
</TD><TD>F221
</TD><TD>F222
</TD></TR><TR><TH><B>Result:</B></TH><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_with_PppAct5-mMCS-T35S-npt_from_the_ligation_performed_over_night"> Picking of of <I>E. coli</I> XL1 blue with PppAct5-mMCS-T35S-npt from the ligation performed over night </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Procedure:</B></P><UL><LI> PppAct5-mMCS-T35S-npt: Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> Picked pipette tips was transferred into cell-culture tubes with air-permeable, sterile cover. Each tube contain 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> 40 colonies were picked.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_9th"><B>Friday, August 9th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_Xl1-Blue_transformed_with_ligation_of_PppAct5_into_mMCS-T35S-npt"> Miniprep of <I>E. coli</I> Xl1-Blue transformed with ligation of PppAct5 into mMCS-T35S-npt </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P574 to P613.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P574.C2.A0.E2.80.93.C2.A0P613"> Analytical digestion and gelelectrophoresis of P574 – P613 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P574 – P613.
</P><P><B>Procedure:</B></P><P><B>Mastermix for analytical digestion with EcoRI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>45 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>8 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>400 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=453 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 9 µl of the mastermix were mixed together with 1 µl of Plasmid DNA 
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 2 h.
</LI></UL><UL><LI> After incubation 3  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>clone 1
</TD><TD>clone 2
</TD><TD>clone 3
</TD><TD>clone 4
</TD><TD>clone 5
</TD><TD>clone 6
</TD><TD>clone 7
</TD><TD>clone 8
</TD><TD>clone 9
</TD><TD>clone 10
</TD></TR><TR><TH><B>Result:</B></TH><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> good clone =&gt; only faint additional band
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>clone 1
</TD><TD>clone 2
</TD><TD>clone 3
</TD><TD>clone 4
</TD><TD>clone 5
</TD><TD>clone 6
</TD><TD>clone 7
</TD><TD>clone 8
</TD><TD>clone 9
</TD><TD>clone 10
</TD></TR><TR><TH><B>Result:</B></TH><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> best clone =&gt; no additional band (best clone)
</TD><TD> wrong result
</TD><TD> wrong result
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>clone 1
</TD><TD>clone 2
</TD><TD>clone 3
</TD><TD>clone 4
</TD><TD>clone 5
</TD><TD>clone 6
</TD><TD>clone 7
</TD><TD>clone 8
</TD><TD>clone 9
</TD><TD>clone 10
</TD></TR><TR><TH><B>Result:</B></TH><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> correct size, but additional bands
</TD><TD> correct size, but additional bands
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>clone 1
</TD><TD>clone 2
</TD><TD>clone 3
</TD><TD>clone 4
</TD><TD>clone 5
</TD><TD>clone 6
</TD><TD>clone 7
</TD><TD>clone 8
</TD><TD>clone 9
</TD><TD>clone 10
</TD></TR><TR><TH><B>Result:</B></TH><TD> wrong result
</TD><TD> wrong result
</TD><TD> correct size, but additional bands
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> correct size, but additional bands
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD><TD> wrong result
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_10th"><B>Saturday, August 10th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_Xl1-Blue_transformed_with_ligation_products_F123.2BF219_.28SYFP_RFC25.29_and_F123.2BF220_.28eCFP_RFC25.29"> Miniprep of <I>E. coli</I> Xl1-Blue transformed with ligation products F123+F219 (SYFP RFC25) and F123+F220 (eCFP RFC25) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> Xl1-Blue transformed with ligation products F123+F219 (SYFP RFC25) and F123+F220 (eCFP RFC25).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P614 to P617.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P614.C2.A0.E2.80.93.C2.A0P617"> Analytical digestion and gelelectrophoresis of P614 – P617 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P614 – P617.
</P><P><B>Procedure:</B></P><P><B>Mastermix for analytical digestion with EcoRI and PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P614 – P617)
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25nbsp;µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 120 V for 30 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P614 (SYFP2 RFC25)
</TD><TD>Digestion of P615 (SYFP2 RFC25)
</TD><TD>Digestion of P616 (eCFP RFC25)
</TD><TD>Digestion of P617 (eCFP RFC25)
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected (723 bp)
</TD><TD>as expected (723 bp)
</TD><TD>as expected (723 bp)
</TD><TD>as expected (723 bp)
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_11th"><B>Sunday, August 11th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P615_.28SYFP2_RFC25.29"> Preparative digestion of P615 (SYFP2 RFC25) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P615 (SYFP2 RFC25)
</P><P><B>Procedure:</B></P></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P617_.28eCFP_RFC25.29"> Preparative digestion and gelelectrophoresis of P617 (eCFP RFC25) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P617 (eCFP RFC25)
</P><P><B>Procedure:</B></P></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F224.2BF223_.28creation_of_Gly-TEV-site-Gly_SYFP2.29"> Ligation of F224+F223 (creation of Gly-TEV-site-Gly_SYFP2) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F224+F223 (creation of Gly-TEV-site-Gly_SYFP2).
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F224+F223 (SYFP2 + Gly-TEV-site-Gly), vector:insert = 1:3:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.66 µl
</TD><TD>F224 (152.1 ng/µl, 2086 bp)
</TD></TR><TR><TD>0.34 µl
</TD><TD>F223 (2506.9 ng/µl, 35 bp)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_Retrafos_of_P516.2C_P330.2C_P245.2C_P496.2C_P346.2C_P196.2C_P20.2C_P476.2C_P554.2C_P498.2C_P246.2C_P354.2C_P428.2C_P512.2C_P402.2C_P197.2C_P350.2C_P502"> Miniprep of Retrafos of P516, P330, P245, P496, P346, P196, P20, P476, P554, P498, P246, P354, P428, P512, P402, P197, P350, P502 </SPAN></H3><P><B>Investigator:</B> Louise, Christopher
</P><P><B>Aim of the experiment:</B> Miniprep of Retrafos of P516, P330, P245, P496, P346, P196, P20, P476, P554, P498, P246, P354, P428, P512, P402, P197, P350, P502.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI><LI> The new plasmids were numbered P618 - P635 (see Inventory)
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_17">Week 17</SPAN></H1><DIV class="week" id="WWeek_17"><H2><SPAN class="mw-headline" id="Monday.2C_August_12th"><B>Monday, August 12th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P509.2C_P512.2C_P516.2C_P537.2C_P540.2C_P562.2C_P565.2C_P583.2C_P591"> Sequencing of P509, P512, P516, P537, P540, P562, P565, P583, P591</SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Sequencing of P509, P512, P516, P537, P540, P562, P565, P583, P591
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode1 (fw)
</TH><TH> Barcode2 (rev)
</TH></TR><TR><TD>P509
</TD><TD>FR02233103
</TD></TR><TR><TD>P632
</TD><TD>FR02233102
</TD><TD>FR02233101
</TD></TR><TR><TD>P633
</TD><TD>FR02233100
</TD><TD>FR02233099
</TD></TR><TR><TD>P537
</TD><TD>FR02233098
</TD><TD>FR02233097
</TD></TR><TR><TD>P540
</TD><TD>FR02233096
</TD><TD>FR02233104
</TD></TR><TR><TD>P562
</TD><TD>FR02233105
</TD><TD>FR02233106
</TD></TR><TR><TD>P565
</TD><TD>FR02233107
</TD><TD>FR02233108
</TD></TR><TR><TD>P583
</TD><TD>FR02233109
</TD><TD>FR02233110 (Primer O131)
</TD></TR><TR><TD>P591
</TD><TD>FR02233111
</TD><TD>FR02233088 (Primer O131)
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P337.2CP540.2C_P583_and_P591"> Transformation of <I>E. coli</I> XL1-Blue with P337,P540, P583 and P591</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Retransformation of <I>E. coli</I> XL1-Blue with P337,P540, P583 and P591
</P><P><B>Procedure:</B></P><UL><LI> 1,5 µl of dd H2O were added to the empty tubes
</LI></UL><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1,5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F224.2BF223_.28SYFP2_.2B_G-TEV-site-G.29_and_negative_control_of_F224"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F224+F223 (SYFP2 + G-TEV-site-G) and negative control of F224</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F224+F223 (SYFP2 + G-TEV-site-G) and negative control of F224
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1,5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_13th"><B>Tuesday, August 13th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Gelextraction_and_Ligation_of_digested_P583_.28PppActin5-MCS-T35S-npt.29_and_P591_.28PppActin5-MCS-T35S-npt.29"> Gelextraction and Ligation of digested P583 (PppActin5-MCS-T35S-npt) and P591 (PppActin5-MCS-T35S-npt)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B>  Gelextraction and selfligation to verify correct cut of digested P583 (PppActin5-MCS-T35S-npt) and P591 (PppActin5-MCS-T35S-npt)
</P><P><B>Procedure:</B></P><UL><LI> After preparative digestion of P583 and P591 with SpfI and MfeI the gel extraction was performed after manufacturer`s protocol (QIAquick Gel Extraction Kit, QIAGEN)
</LI></UL><UL><LI> the extracted fragments were labelled F226 (digested P583) and F227 (digested P591)
</LI></UL><UL><LI> ligation was performed without extra inserts with just 100 ng of cut plasmid to test if it was cut as expected
</LI></UL><UL><LI> Ligation batch for self-ligation of F226
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.6 µl
</TD><TD>F226 (164.1 ng/µl)
</TD></TR><TR><TD>16.4 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for self-ligation of F227 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F227 (243.4 ng/µl)
</TD></TR><TR><TD>16.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_of_self-ligated_F226_and_F227_and_with_Biobrick_BBa_J45014"> Transformation of <I>E. coli</I> XL1-Blue with ligation product of self-ligated F226 and F227 and with Biobrick BBa_J45014</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product of self-ligated F226 and F227 and with Biobrick BBa_245014
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1,5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_Retrafos_of_P337.2C_P540.2C_P583_and_P591_and_of_Ligation_F224_.2B_F223"> Picking of Retrafos of P337, P540, P583 and P591 and of Ligation F224 + F223 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of Retrafos of P337 (SERK-SigP_XylE), P540 (SERK-SigP_PP1_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P583 (PppActin5-MCS-T35S-npt) and P591 (PppActin5-MCS-T35S-npt) and of Ligation F224 + F223 (SYFP2 RFC25 in pSB1C3 + Oligohybridization of G-TEV-site-G)
</P><P><B>Procedure:</B></P><UL><LI> Retrafos 1 clone each, Ligation 4 clones due to approx. 50 % positive neg. ctrl
</LI></UL><UL><LI> 4 ml LB medium w/ 4 µl chloramphenicol
</LI></UL><UL><LI> Incubation overnight at 37 °C in the 180 rpm
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_14th"><B>Wednesday, August 14th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_BBa_J45014_.28banana_odour.29"> Picking of of <I>E. coli</I> XL1 blue transformed with BBa_J45014 (banana odour)</SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with BBa_J45014 (banana odour)
</P><P><B>Procedure:</B></P><UL><LI> BBa_J45014 (banana odour): 4 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> Each colony was transferred into a tube containing 4 mL of LB-medium + 4 µL chloramphenicol(1000x).
</LI></UL><UL><LI> These tubes were transferred into a cell culture shaker at 37 °C and were incubated overnight
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P640-P643_.28G-TEV-site-G_SYFP2.29_and_P615_with_EcoRI_and_PstI_and_P637.2C_P639_with_EcoRI_and_MfeI_.28old_and_new.29"> Analytical digestion and gelelectrophoresis of P640-P643 (G-TEV-site-G_SYFP2) and P615 with EcoRI and PstI and P637, P639 with EcoRI and MfeI (old and new) </SPAN></H3><P><B>Investigator:</B> Louise, Johanna
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P640-P643 (G-TEV-site-G_SYFP2) and P615 (SYFP2, RFC25) with EcoRI and PstI and P637 (PppActin5-MCS-T35S-npt), P639 (PppActin5-MCS-T35S-npt) with EcoRI and MfeI (old and new)
</P><P><B>Procedure:</B></P><UL><LI> batch for digestion of P640-P643 
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>8.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> batch for digestion of P615
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> batch for digestion of P637 and P639
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>8.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>MfeI (old or new)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>DNA ladder
</TD><TD>Digestion of P615 (SYFP2 RFC25-reference)
</TD><TD>Digestion of P637 (with old MfeI)
</TD><TD>Digestion of P637 (with new MfeI)
</TD><TD>Digestion of P639 (with old MfeI)
</TD><TD>Digestion of P639 (with new MfeI)
</TD><TD>Digestion of P640
</TD><TD>Digestion of P641
</TD><TD>Digestion of P642
</TD><TD>Digestion of P643
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>corrupt MfeI
</TD><TD>as expected
</TD><TD>corrupt MfeI
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Sequencing_of_P614.2C_P617.2C_P637.2C_P639_and_P641">  Sequencing of P614, P617, P637, P639 and P641</SPAN></H3><P><B>Investigator: Flo, Johanna</B></P><P><B>Aim of the experiment:</B> Sequencing of P614, P617, P637, P639 and P641
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode
</TH><TH> Primer
</TH></TR><TR><TD>P614
</TD><TD>FR02233112
</TD><TD>03
</TD></TR><TR><TD>P617
</TD><TD>FR02233113
</TD><TD>03
</TD></TR><TR><TD>P641
</TD><TD>FR02233114
</TD><TD>03
</TD></TR><TR><TD>P637
</TD><TD>FR02233119
</TD><TD>061
</TD></TR><TR><TD>P639
</TD><TD>FR02233115
</TD><TD>061
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_Retrafos_of_P337.2C_P583.2C_P540.2C_P591_and_of_ligation_product_F224.2BF223"> Miniprep of Retrafos of P337, P583, P540, P591 and of ligation product F224+F223 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Miniprep of Retrafos of P337, P583, P540, P591 and of ligation product F224+F223
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI><LI> The new plasmids were numbered as followed:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Plasmid
</TH><TH> Miniprep-Plasmid
</TH></TR><TR><TD>P337
</TD><TD>P636
</TD></TR><TR><TD>P583
</TD><TD>P637
</TD></TR><TR><TD>P540
</TD><TD>P638
</TD></TR><TR><TD>P591
</TD><TD>P639
</TD></TR><TR><TD>F224+223 (version 1)
</TD><TD>P640
</TD></TR><TR><TD>F224+223 (version 2)
</TD><TD>P641
</TD></TR><TR><TD>F224+223 (version 3)
</TD><TD>P642
</TD></TR><TR><TD>F224+223 (version 4)
</TD><TD>P643
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_Retrafos_of_banana_odour.2C_P583_and_P591"> Miniprep of Retrafos of banana odour, P583 and P591 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Miniprep of Retrafos of banana odour, P583 and P591.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI><LI> The new plasmids were numbered as followed:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Plasmid
</TH><TH> Miniprep-Plasmid
</TH></TR><TR><TD>Banana 1
</TD><TD>P644
</TD></TR><TR><TD>Banana 2
</TD><TD>P645
</TD></TR><TR><TD>Banana 3
</TD><TD>P646
</TD></TR><TR><TD>Banana 4
</TD><TD>P647
</TD></TR><TR><TD>P583
</TD><TD>P648
</TD></TR><TR><TD>P591
</TD><TD>P649
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P644_.28banana_odour.29.2C_P648_.26_P649_.28PppAct5-mMCS-T35S-npt.29"> Preparative digestion of P644 (banana odour), P648 &amp; P649 (PppAct5-mMCS-T35S-npt) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P644:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P644
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P648:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>40 µl
</TD><TD>P648
</TD></TR><TR><TD>10 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>MfeI (20 U/µl)
</TD></TR><TR><TD>0 µl was empty :'(
</TD><TD>SbfI (20 U/µl)
</TD></TR><TR><TD>46 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P649:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>40 µl
</TD><TD>P649
</TD></TR><TR><TD>10 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>MfeI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>SbfI (20 U/µl)
</TD></TR><TR><TD>46 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 3 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P428 with EcoRI &amp; XbaI
</TD><TD>Digestion of P554 with EcoRI &amp; SpeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; highest band was cut out
</TD><TD>as expected; highest band was cut out
</TD></TR></TBODY></TABLE><P>The bands were extracted from the gel and termed F221 and F222. The concentrations were determined to 42 and 37 ng/µl.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_15th"><B>Thursday, August 15th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P414_and_P640_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P414 and P640 + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P414:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P414
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P640:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P640
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P414 with EcoRI &amp; PstI
</TD><TD>1kB DNA ladder
</TD><TD>Digestion of P640 with EcoRI &amp; NgoMIV
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; upper band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI><LI> The resulting fragments were named F231 (Laccase) + F232
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F230_with_F191.2C_F192.2C_F193.2C_F194.2C_F195.2C_F196.2C_F197.2C_F198.2C_F199.2C_F200.2C_F201.2C_F202.2C_F203.2C_F204.2C_F205.2C_F214.2C_F217_and_F218"> Ligation of F230 with F191, F192, F193, F194, F195, F196, F197, F198, F199, F200, F201, F202, F203, F204, F205, F214, F217 and F218 </SPAN></H3><P><B>Investigator:</B> Louise, Florian
</P><P><B>Aim of the experiment:</B> Ligation of F230 with F191, F192, F193, F194, F195, F196, F197, F198, F199, F200, F201, F202, F203, F204, F205, F214, F217 and F218
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F230+F191 (plasmid with mini MCS + GFP cytoplasmatic), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>18.7 µl
</TD><TD>F191 (2 ng/µl, 723 bp)
</TD></TR><TR><TD>0.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F192 (plasmid with mini MCS + catecholdioxygenase cytoplasmatic), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>5.94 µl
</TD><TD>F192 (8 ng/µl, 918 bp)
</TD></TR><TR><TD>10.65 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F193 (plasmid with mini MCS + EreB cytoplasmatic), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>1.85 µl
</TD><TD>F193 (35 ng/µl, 1254 bp)
</TD></TR><TR><TD>14.74 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F194 (plasmid with mini MCS + Nanoluciferase cytoplasmatic), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>8.80 µl
</TD><TD>F194 (3 ng/µl, 510 bp)
</TD></TR><TR><TD>7.79 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F195 (plasmid with mini MCS + nanoluciferase secretory_SERK), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>3.99 µl
</TD><TD>F195 (8 ng/µl, 616 bp)
</TD></TR><TR><TD>12.60 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F196 (plasmid with mini MCS + gutathiontransferase cytoplasmatic), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>1.17 µl
</TD><TD>F196 (28 ng/µl, 633 bp)
</TD></TR><TR><TD>15.42 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F197 (plasmid with mini MCS + EreB membrane associated), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>11.12 µl
</TD><TD>F197 (11 ng/µl, 2362 bp)
</TD></TR><TR><TD>5.47 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F198 (plasmid with mini MCS + Nanoluc membrane associated), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>6.44 µl
</TD><TD>F198 (13 ng/µl, 1618 bp)
</TD></TR><TR><TD>10.15 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F199 (plasmid with mini MCS + nanoluc secretory_Ig Kappa), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>1.16 µl
</TD><TD>F199 (26 ng/µl, 583 bp)
</TD></TR><TR><TD>15.43 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F200 (plasmid with mini MCS + Nanoluc+IRES+Spytag_Nanoluc), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>3.00 µl
</TD><TD>F200 (47 ng/µl, 2725 bp)
</TD></TR><TR><TD>13.59 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F201 (plasmid with mini MCS + Nanoluc+IRES+Nanoluc_Spycatcher), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>3.81 µl
</TD><TD>F201 (37 ng/µl, 2725 bp)
</TD></TR><TR><TD>12.78 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F202 (plasmid with mini MCS + Nanoluc+IRES+Nanoluc_Spytag), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>2.66 µl
</TD><TD>F202 (53 ng/µl, 2725 bp)
</TD></TR><TR><TD>13.93 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F203 (plasmid with mini MCS + IgKappa_GFP secretory), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>1.09 µl
</TD><TD>F203 (38 ng/µl, 800 bp)
</TD></TR><TR><TD>15.50 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F204 (plasmid with mini MCS + Nanoluc+IRES+Spycatcher_Nanoluc), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>2.17 µl
</TD><TD>F204 (65 ng/µl, 2725 bp)
</TD></TR><TR><TD>14.42 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F205 (plasmid with mini MCS + FluA membrane associated), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>2.11 µl
</TD><TD>F205 (40 ng/µl, 1630 bp)
</TD></TR><TR><TD>14.48 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F214 (plasmid with mini MCS + PP1 membrane associated), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>0.95 µl
</TD><TD>F214 (113 ng/µl, 2080 bp)
</TD></TR><TR><TD>15.64 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F217 (plasmid with mini MCS + PIF3 safety), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>2.76 µl
</TD><TD>F217 (116 ng/µl, 6191 bp)
</TD></TR><TR><TD>13.83 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F217 (plasmid with mini MCS + PIF3 safety), vector:insert = 3:1</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.15 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>0.86 µl
</TD><TD>F217 (116 ng/µl, 6191 bp)
</TD></TR><TR><TD>14.99 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F218 (plasmid with mini MCS + PIF6 safety), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>2.50 µl
</TD><TD>F218 (128 ng/µl, 6191 bp)
</TD></TR><TR><TD>14.09 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F218 (plasmid with mini MCS + PIF3 safety), vector:insert = 3:1</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.15 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>0.78 µl
</TD><TD>F218 (116 ng/µl, 6191 bp)
</TD></TR><TR><TD>15.07 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at roomtemperature, 1 h
</LI></UL></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F230_with_F231"> Ligation of F230 with F231 </SPAN></H3><P><B>Investigator:</B> Louise, Christopher
</P><P><B>Aim of the experiment:</B> Ligation of F230 with F231
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F230+F231 (plasmid with mini MCS + Laccase membrane-associated), vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230 (243.7 ng/µl, 5796 bp)
</TD></TR><TR><TD>14.33 µl
</TD><TD>F231 (9.5 ng/µl, ~2630 bp)
</TD></TR><TR><TD>2.26 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature, 1 h
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_F230.2BF191.2C_F230.2BF192.2C_F230.2BF193.2C_F230.2BF194.2C_F230.2BF195.2C_F230.2BF196.2C_F230.2BF197.2C_F230.2BF198.2C_F230.2BF199.2C_F230.2BF200.2C_F230.2BF201.2C_F230.2BF202.2C_F230.2BF203.2C_F230.2BF204.2C_F230.2BF205.2C_F230.2BF214.2C_F230.2BF217.2C_F230.2BF218.2C_F217.2BF230.2C_F218.2BF230_and_negative_control_of_F230"> Transformation of <I>E. coli</I> XL1-Blue with ligation products F230+F191, F230+F192, F230+F193, F230+F194, F230+F195, F230+F196, F230+F197, F230+F198, F230+F199, F230+F200, F230+F201, F230+F202, F230+F203, F230+F204, F230+F205, F230+F214, F230+F217, F230+F218, F217+F230, F218+F230 and negative control of F230</SPAN></H3><P><B>Investigator:</B> Louise, Christopher
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products F230+F191, F230+F192, F230+F193, F230+F194, F230+F195, F230+F196, F230+F197, F230+F198, F230+F199, F230+F200, F230+F201, F230+F202, F230+F203, F230+F204, F230+F205, F230+F214, F230+F217, F230+F218, F217+F230, F218+F230 and negative control of F230
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I>XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1,5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_16th"><B>Friday, August 16th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P640_.28SYFP2_RFC25_in_pSB1C3_.2B_Oligohybridization_of_G-TEV-site-G.29">  Sequencing of P640 (SYFP2 RFC25 in pSB1C3 + Oligohybridization of G-TEV-site-G)</SPAN></H3><P><B>Investigator: Rosario</B></P><P><B>Aim of the experiment:</B> Sequencing of P640.
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode
</TH><TH> Primer
</TH></TR><TR><TD>P640
</TD><TD>FR02233116
</TD><TD>03
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P640_.28SYFP2_RFC25_in_pSB1C3_.2B_Oligohybridization_of_G-TEV-site-G.29_with_EcoRI.2BNgoMIV">Preparative digestion of P640 (SYFP2 RFC25 in pSB1C3 + Oligohybridization of G-TEV-site-G) with EcoRI+NgoMIV</SPAN></H3><P><B>Investigators:</B> Louise, Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P640 (SYFP2 RFC25 in pSB1C3 + Oligohybridization of G-TEV-site-G) with EcoRI+NgoMIV
</P><P><B>Procedure:</B></P><TABLE><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>33 µl
</TD><TD>P640
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI Hf (20 U/µl)
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 3 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2 log DNA ladder
</TD><TD>Digestion of P640 with EcoRI &amp; NgoMIV
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; band was cut out
</TD></TR></TBODY></TABLE><P>The band was extracted from the gel via QIAGEN Gel Extraction Kit and termed F233. The concentration was 31.1 ng/µl.
</P></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF231.28plasmid_with_mMCS.2Bmembrane-bound_Laccase.29_and_negative_control_of_F230.2C_Retrafo_of_P541_.28PppActin5.29_and_P449.28stress_inducible_Promoter_RFP_.2Bt35S-NPT-pSB1C3.29"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F231(plasmid with mMCS+membrane-bound Laccase) and negative control of F230, Retrafo of P541 (PppActin5) and P449(stress inducible Promoter_RFP +t35S-NPT-pSB1C3)</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F231(plasmid with mMCS+membrane-bound Laccase) and negative control of F230, Retrafo of P541 (PppActin5) and P449(stress inducible Promoter_RFP +t35S-NPT-pSB1C3).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1,5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F233.2BF225_.28creation_of_eCFP_Gly-TEV-site-Gly_SYFP2.29"> Ligation of F233+F225 (creation of eCFP_Gly-TEV-site-Gly_SYFP2) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Creation of eCFP_Gly-TEV-site-Gly_SYFP2
</P><P><B>Procedure:</B></P><P><B>Ligation batch, vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>3.22 µl
</TD><TD>F233 (31.1 ng/µl, 2834 bp)
</TD></TR><TR><TD>1.93 µl
</TD><TD>F225 (39.6 ng/µl, 723 bp)
</TD></TR><TR><TD>11.85 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. w/ water instead of insert
</LI></UL><UL><LI> Ligation was performed at room temperature, 1 h
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1-Blue_transformed_with_ligation_products_F230.2BF191.2C_F230.2BF192.2C_F230.2BF193.2C_F230.2BF194.2C_F230.2BF195.2C_F230.2BF196.2C_F230.2BF197.2C_F230.2BF198.2C_F230.2BF199.2C_F230.2BF200.2C_F230.2BF201.2C_F230.2BF202.2C_F230.2BF203.2C_F230.2BF204.2C_F230.2BF205.2C_F230.2BF214.2C_F230.2BF217.2C_F217.2BF230.2C_F230.2BF218.2C_F218.2BF230.2C_F230.2BF228"> Miniprep of <I>E. coli</I> XL1-Blue transformed with ligation products F230+F191, F230+F192, F230+F193, F230+F194, F230+F195, F230+F196, F230+F197, F230+F198, F230+F199, F230+F200, F230+F201, F230+F202, F230+F203, F230+F204, F230+F205, F230+F214, F230+F217, F217+F230, F230+F218, F218+F230, F230+F228 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario, Louise, Christopher
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1-Blue transformed with ligation products F230+F191, F230+F192, F230+F193, F230+F194, F230+F195, F230+F196, F230+F197, F230+F198, F230+F199, F230+F200, F230+F201, F230+F202, F230+F203, F230+F204, F230+F205, F230+F214, F230+F217, F217+F230, F230+F218, F218+F230, F230+F228 
</P><P><B>Procedure:</B></P><UL><LI> Ligation products were labelled B to V (in the same order as in Aim of the experiment, above)
</LI></UL><UL><LI> Two clones for every ligation product except for H, L and O
</LI></UL><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P650 to P688.
</LI></UL></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P650-P688_with_XbaI_and_PstI-HF"> Analytical digestion and gelelectrophoresis of P650-P688 with XbaI and PstI-HF</SPAN></H3><P><B>Investigator:</B> Jeff, Rosario, Louise, Christopher
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P650-P688 with XbaI and PstI-HF
</P><P><B>Procedure:</B></P><UL><LI> batch for digestion of P650-P688
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.2 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.2 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>3.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_17th"><B>Saturday, August 17th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_P449.2C_P541.2C_F230.2BF231.2C_2x_K-Platte.2C_1x_L-Platte.2C_2x_M-Platte.2C_1x_O-Platte"> Picking of of <I>E. coli</I> XL1 blue transformed with P449, P541, F230+F231, 2x K-Platte, 1x L-Platte, 2x M-Platte, 1x O-Platte</SPAN></H3><P><B>Investigator: Andi, Johanna</B></P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with P449, P541, F230+F231, 2x K-Platte, 1x L-Platte, 2x M-Platte, 1x O-Platte
</P><P><B>Procedure:</B></P><UL><LI> F230+F231: 8 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> P449: 4 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> P541: 4 Colonies were picked from ampicilin plates.
</LI></UL><UL><LI> K-Plate: 4 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> L-Plate: 4 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> M-Plate: 4 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> O-Plate: 4 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> Each colony was transferred into a tube containing a relation of 1:1000 of LB-medium : antibiotic
</LI></UL><UL><LI> These tubes were transferred into a cell culture shaker at 37 °C and were incubated 16 h
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P625.2C_P516.2C_P644_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P625, P516, P644 + Preparative Gelelectrophoresis + Gelextraction</SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of Experiment:</B> Obtaining digested fragments of P625 (SERK-SigP_EreB_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1), P516  (SERK-SigP_FluA_Strep-tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1)and P644 (Banana odour BBa_J45014)
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P625, P516 and P644:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>PXXX
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5  h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI><LI> The resulting fragments were named  F234 (digestion of P516), F235 (digestion of P625), F236 (digestion of P644).
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F230.2B_F234.2C_F230.2B_F235.2C_F230.2BF236">Ligation of F230+ F234, F230+ F235, F230+F236</SPAN></H3><P><B>Investigator:</B> Johanna, Andi
</P><P><B>Aim of the experiment:</B> Ligation of F230+ F234, F230+ F235, F230+F236
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F230+F234, vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>6.3 µl
</TD><TD>F234(228 ng/µl, 1600 bp)
</TD></TR><TR><TD>0.4 µl
</TD><TD>F230 (100 ng/µl,  bp)
</TD></TR><TR><TD>10.3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F235, vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>9.0 µl
</TD><TD>F235(214 ng/µl, 2300 bp)
</TD></TR><TR><TD>0.4 µl
</TD><TD>F230 (100 ng/µl,  bp)
</TD></TR><TR><TD>7.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Ligation batch for F230+F236, vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>16.6 µl
</TD><TD>F236(330 ng/µl, 1500 bp)
</TD></TR><TR><TD>0.4 µl
</TD><TD>F230 (100 ng/µl,  bp)
</TD></TR><TR><TD>0.0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. w/ water instead of insert
</LI></UL><UL><LI> Ligation was performed at room temperature, 1 h
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_F230.2BF231_.28version_1-4.29.2C_P541_.28version_1-2.29.2C_K3.2C_K4.2C_L2.2C_L3.2C_M3.2C_M4.2C_O2.2C_O3.2C_P3-P10_and_P449_.28version_1-2.29"> Miniprep of F230+F231 (version 1-4), P541 (version 1-2), K3, K4, L2, L3, M3, M4, O2, O3, P3-P10 and P449 (version 1-2)</SPAN></H3><P><B>Investigator:</B> Johanna, Andi
</P><P><B>Aim of the experiment:</B> Miniprep of F230+F231 (version 1-4), P541 (version 1-2), K3, K4, L2, L3, M3, M4, O2, O3, P3-P10 and P449 (version 1-2)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI><LI> The new plasmids were numbered as followed:
</LI></UL><TABLE border="1"><TBODY><TR><TH> Plasmid
</TH><TH> Miniprep-Plasmid
</TH></TR><TR><TD>F230+F231 (version 1)
</TD><TD>P689
</TD></TR><TR><TD>F230+F231 (version 2)
</TD><TD>P690
</TD></TR><TR><TD>F230+F231 (version 3)
</TD><TD>P691
</TD></TR><TR><TD>F230+231 (version 4)
</TD><TD>P692
</TD></TR><TR><TD>P541 (version 1)
</TD><TD>P693
</TD></TR><TR><TD>P541 (version 2)
</TD><TD>P694
</TD></TR><TR><TD>K3
</TD><TD>P695
</TD></TR><TR><TD>K4
</TD><TD>P696
</TD></TR><TR><TD>L2
</TD><TD>P697
</TD></TR><TR><TD>L3
</TD><TD>P698
</TD></TR><TR><TD>M3
</TD><TD>P699
</TD></TR><TR><TD>M4
</TD><TD>P700
</TD></TR><TR><TD>O2
</TD><TD>P701
</TD></TR><TR><TD>O3
</TD><TD>P702
</TD></TR><TR><TD>P3
</TD><TD>P703
</TD></TR><TR><TD>P4
</TD><TD>P704
</TD></TR><TR><TD>P5
</TD><TD>P705
</TD></TR><TR><TD>P6
</TD><TD>P706
</TD></TR><TR><TD>P7
</TD><TD>P707
</TD></TR><TR><TD>P8
</TD><TD>P708
</TD></TR><TR><TD>P9
</TD><TD>P709
</TD></TR><TR><TD>P10
</TD><TD>P710
</TD></TR><TR><TD>P449 (version 1)
</TD><TD>P711
</TD></TR><TR><TD>P449 (version 2)
</TD><TD>P712
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P689-P712_with_XbaI_and_PstI"> Analytical digestion and gelelectrophoresis of P689-P712 with XbaI and PstI </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P689-P712 with XbaI and PstI
</P><P><B>Procedure:</B></P><UL><LI> batch for digestion of P689-712
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.2 µl
</TD><TD>XbaI-HF
</TD></TR><TR><TD>0.2 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>3.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>Digestion of P689
</TD><TD>Digestion of P690
</TD><TD>Digestion of P691
</TD><TD>Digestion of P692
</TD><TD>Digestion of P693
</TD><TD>Digestion of P694
</TD><TD>Digestion of P695
</TD><TD>Digestion of P696
</TD><TD>Digestion of P697
</TD><TD>Digestion of P698
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>weird
</TD><TD>weird
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>weird
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>Digestion of P699
</TD><TD>Digestion of P700
</TD><TD>Digestion of P701
</TD><TD>Digestion of P702
</TD><TD>Digestion of P703
</TD><TD>Digestion of P704
</TD><TD>Digestion of P705
</TD><TD>Digestion of P706
</TD><TD>Digestion of P707
</TD><TD>Digestion of P708
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>weird
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>Digestion of P709
</TD><TD>Digestion of P710
</TD><TD>Digestion of P711
</TD><TD>Digestion of P712
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF234.2C_F230.2BF235.2C_F230.2BF236"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F234, F230+F235, F230+F236 </SPAN></H3><P><B>Investigator:</B> Johanna, Andi
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F234, F230+F235, F230+F236.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1,5 µl of DNA  were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspension were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_18th"><B>Sunday, August 18th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="BITTE_VERVOLLST.C3.84NDIGEN_.28P-Nummern.29_Midiprep_of_Safety_PIF3.2C_PP1_membrane_bound.2C_Safety_PIF6.2C_Safety_PIF3.2C_Positve_control_mit_PppActin5.2C_EreB_cyt..2C_NanoLuc_cyt..2C_XylE_cyt..2C_NanoLuc_sek._.28IgKappa.29.2C_NanoLuc_sek._.28SERK.29.2C_GFP_cyt..2C_GFP_sek._.28IgKappa.29.2C_FluA_membrane_bound.2C_stress-inducible_promoter.2C_GST.2C_NanoLuc_membrane_bound"> BITTE VERVOLLSTÄNDIGEN (P-Nummern) Midiprep of Safety_PIF3, PP1 membrane bound, Safety_PIF6, Safety_PIF3, Positve control mit PppActin5, EreB cyt., NanoLuc cyt., XylE cyt., NanoLuc sek. (IgKappa), NanoLuc sek. (SERK), GFP cyt., GFP sek. (IgKappa), FluA membrane bound, stress-inducible promoter, GST, NanoLuc membrane bound </SPAN></H3><P><B>Investigator: Andreas, Rosario</B></P><P><B>Aim of the experiment:</B> Midiprep of RFC25 compatible RFP generator in (pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Midiprep was performed after manufacturer's protocol (QIAprep Midiprep, QIAGEN)
</LI></UL></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Digestion_of_BITTE_VERVOLLST.C3.84NDIGEN"> Digestion of BITTE VERVOLLSTÄNDIGEN </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_18">Week 18</SPAN></H1><DIV class="week" id="WWeek_18"><H2><SPAN class="mw-headline" id="Monday.2C_August_19th"><B>Monday, August 19th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_Gelelectrophoresis_of_digested_Midi-Preps_F243_-_F260"> Analytical Gelelectrophoresis of digested Midi-Preps F243 - F260</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Verification of linearisation of digest fragments F234-F260
</P><P><B>Procedure:</B></P><UL><LI> Concentrations of digestions were measured (blanked against EB-buffer), see Inventory List
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) mixed with 1 µl Midiprep-DNA, on 1% agarose gel.
</LI></UL><UL><LI> 90 V 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>F234
</TD><TD>F235
</TD><TD>F236
</TD><TD>F237
</TD><TD>F238
</TD><TD>F239
</TD><TD>F240
</TD><TD>F241
</TD><TD>F242
</TD><TD>F243
</TD><TD>F244
</TD></TR><TR><TH><B>Name</B></TH><TD> -
</TD><TD>Nanoluc secretory
</TD><TD>GFP cytoplasmatic
</TD><TD>Nanoluc membrane bound
</TD><TD>Nanoluc cytoplasmatic
</TD><TD>Ere B cytoplasm
</TD><TD>PIF6
</TD><TD>PIF3 clone1
</TD><TD>PIF3 clone2
</TD><TD>GFP_IgKappa
</TD><TD>PP1
</TD><TD>C1
</TD></TR><TR><TH><B>Results</B></TH></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>F245
</TD><TD>F246
</TD><TD>F247
</TD><TD>F248
</TD><TD>F249
</TD><TD>F250
</TD><TD>F251
</TD><TD>F252
</TD></TR><TR><TH><B>Name</B></TH><TD> -
</TD><TD>Safety_PIF3
</TD><TD>PP1 mebrane bound
</TD><TD>Safety_PIF6
</TD><TD>Safety_PIF?
</TD><TD>pos ctrl mit Ppp Actin5
</TD><TD>Ere B cytoplasm
</TD><TD>Nanoluc cytoplasm
</TD><TD>XylE cytoplasm
</TD></TR><TR><TH><B>Results</B></TH></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>F253
</TD><TD>F254
</TD><TD>F255
</TD><TD>F256
</TD><TD>F257
</TD><TD>F258
</TD><TD>F259
</TD><TD>F260
</TD></TR><TR><TH><B>Name</B></TH><TD> -
</TD><TD>Nanoluc secretory IgKappa
</TD><TD>Nanoluc secretory SERK
</TD><TD>GFP cytoplasm
</TD><TD>GFP secretory IgKappa
</TD><TD>FluA membrane bound
</TD><TD>stress inducible promotor
</TD><TD>GST
</TD><TD>Nanoluc membrane bound
</TD></TR><TR><TH><B>Results</B></TH><TD> -
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparation_of_the_linearized_DNA_.28F234-F260.29_for_the_Moss_Transformation"> Preparation of the linearized DNA (F234-F260) for the Moss Transformation </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Preparation of the linearized DNA (F234-F260) for the Moss Transfection
</P><P><B>Procedure:</B></P><UL><LI>The DNA pellets were solved in 500 µl ddH2O
</LI></UL><UL><LI>Linerization of the DNA with EcoRI: 56,6 µl CutSmart buffer and 10 µl EcoRI were added to the 500 µl DNA. Incubation over night at 37 °C.
</LI></UL><UL><LI>For purification the DNA was again precipitated with pure isopropanol: Addition of 500 µl isopropanol, mixing by inverting, centrifugation at 16,200 g for 40 min; supernatant was discarded
</LI></UL><UL><LI>Wash pellet with 200 µl 70 % ethanol, centrifugation at 16,200 g for 40 min, supernatant discarded. The pellet was air dried at 50 °C for 8 min.
</LI></UL><UL><LI>The DNA pellets were dissolved in the respectively calculated volumes of 0,1 M Ca(NO3)2 soultion to obtain final concentrations of 250 ng/µl.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>fragment</B></TH><TH><B>name</B></TH><TH><B>concentration</B></TH><TH><B>concentration w/ loss</B></TH><TH><B>volume 0,1 M Ca(NO3)2</B></TH></TR><TR><TD>F234
</TD><TD>Nanoluc SERK-SigP
</TD><TD>440,8
</TD><TD>396,72
</TD><TD>872,78
</TD></TR><TR><TD>F235
</TD><TD>GFP cytoplasm
</TD><TD>241,8
</TD><TD>217,62
</TD><TD>478,76
</TD></TR><TR><TD>F236
</TD><TD>Nanoluc membrane
</TD><TD>339,9
</TD><TD>305,91
</TD><TD>673,00
</TD></TR><TR><TD>F237
</TD><TD>Nanoluc cytoplasm
</TD><TD>219,1
</TD><TD>197,19
</TD><TD>433,82
</TD></TR><TR><TD>F238
</TD><TD>EreB cytoplasm
</TD><TD>355,9
</TD><TD>320,31
</TD><TD>704,68
</TD></TR><TR><TD>F239
</TD><TD>PIF6
</TD><TD>669,9
</TD><TD>602,91
</TD><TD>1326,40
</TD></TR><TR><TD>F240
</TD><TD>PIF3 (1)
</TD><TD>438,1
</TD><TD>394,29
</TD><TD>867,438
</TD></TR><TR><TD>F241
</TD><TD>PIF3 (2)
</TD><TD>642,9
</TD><TD>578,61
</TD><TD>1272,942
</TD></TR><TR><TD>F242
</TD><TD>GFP IgKappa
</TD><TD>339,4
</TD><TD>305,46
</TD><TD>672,012
</TD></TR><TR><TD>F243
</TD><TD>PP1 receptor
</TD><TD>333,5
</TD><TD>300,15
</TD><TD>660,33
</TD></TR><TR><TD>F244
</TD><TD>Catecholdioxygenase cytoplasm
</TD><TD>342,4
</TD><TD>308,16
</TD><TD>677,952
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_20th"><B>Tuesday, August 20th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_.2B_Gelelectrophoreses_.2B_Gelextraction_of_P643.2FF261"> Preparative digestion + Gelelectrophoreses + Gelextraction of P643/F261 </SPAN></H3><P><B>Investigator:</B> Andi, Rosario, Johanna
</P><P><B>Aim of the experiment:</B> Preparative digestion + Gelelectrophoreses + Gelextraction of P643/F261.
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P643:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P643
</TD></TR><TR><TD>5 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Result of the Gelelectrophoresis:
</P><UL><LI> Reaction batch was incubated at 37 °C for 2 h.
</LI></UL><UL><LI> Digested product was purified with QIAquick PCR Purification Kit, QIAGEN.
</LI></UL><UL><LI> Gelelectrophoresis was performed for 1 h and 90V.
</LI></UL><UL><LI> Gelextraction was performed after manufacturers protocol and the resulting Fragment was called F261.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P643_.28SYFP2.2B_G-TEV-site-G.29">  Sequencing of P643 (SYFP2+ G-TEV-site-G) </SPAN></H3><P><B>Investigator: Andi</B></P><P><B>Aim of the experiment:</B> Sequencing of P643 (SYFP2+ G-TEV-site-G)
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The different genes we sequenced received the following barcodes:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode
</TH><TH> Primer
</TH></TR><TR><TD>P643
</TD><TD>FR0223317
</TD><TD>03
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F261.2BF225_.28G-TEV-site-G_SYFP2_.2B_eCFP.29">Ligation of F261+F225 (G-TEV-site-G_SYFP2 + eCFP)</SPAN></H3><P><B>Investigator:</B> Rosario, Andi, Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F261+F225 (G-TEV-site-G_SYFP2 + eCFP)
</P><P><B>Procedure:</B></P><P><B>Ligation batch for F261+F225, vector:insert = 1:3</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1,5 µl
</TD><TD>F225
</TD></TR><TR><TD>1,2 µl
</TD><TD>F261
</TD></TR><TR><TD>14,3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. w/ water instead of insert
</LI></UL><UL><LI> Ligation was performed at room temperature, 1 h
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F225.2BF261_and_Retrafo_of_P643"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F225+F261 and Retrafo of P643 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F225+F261 and Retrafo of P643.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA (Ligation) and 1 µl of DNA (Retrafo) were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_Physcomitrella_patens_with_F234-F260"> Transformation of <I>Physcomitrella patens</I> with F234-F260 </SPAN></H3><P><B>Investigator:</B> Ingmar, Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>Physcomitrella patens</I> with F234-F260
</P><P><B>Procedure:</B></P><P>According to the Reski protocol <A href="http://www.plant-biotech.net/" class="external autonumber" rel="nofollow">[1]</A></P><UL><LI> Prepare 4&amp;nbsp% driselase solution in 0.5&amp;nbspM mannitol, vortex, keep on rotating platform for 45&amp;nbspmin (protected from light) and centrifuge at 3500 rpm, 10 min; sterile filter
</LI></UL><UL><LI> Extraction of moss from bioreactor culture by filtering through protoplast sieve (mesh size 100 µm)
</LI></UL><UL><LI> Addition of 12 ml 0.5 M mannitol solution and 4 ml of driselase stock solution (final concentration: 1 % w/v) and Incubation for 2 h at room temperature on rotating platform (protected from light)
</LI></UL><UL><LI> Gently filter moss material through protoplast sieve (mesh size 100 µm), then filtrate through 50 µm sieve
</LI></UL><UL><LI> Centrifugation at 500 rpm for 10 min with slow acceleration rates in a glass tube and discard supernatant; wash protoplasts with mannitol solution, repeat.
</LI></UL><UL><LI> Resuspend pellet in total 10 ml mannitol
</LI></UL><UL><LI> Determine concentration of protoplasts/ ml
</LI></UL><UL><LI> 100&amp;nbspµl of 250 ng/µl DNA solution with 350 µl PEG 4000 solution (8 g PEG4000 ad 20 g 3M medium, sterile filtered), incubate 30 min while gently mixing every 5 min
</LI></UL><UL><LI> Dilution with 3M medium every 5 min: 1 ml, then 2 ml, then 3, then 4
</LI></UL><UL><LI> Centrifugation at 500 rpm for 10 min, discard supernatant, resuspend in 4 ml regeneration medium and divide into 2 6wells
</LI></UL><UL><LI> Incubation in the dark over night, then light/dark regime of 16/8 h
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_21st"><B>Wednesday, August 21st</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_of_E..C2.A0coli_XL1_blue_transformed_with_ligation_product_F225.2BF261_and_Retrafo_of_P643"> Picking of of <I>E. coli</I> XL1 blue transformed with ligation product F225+F261 and Retrafo of P643 </SPAN></H3><P><B>Investigator: Rosario</B></P><P><B>Aim of the experiment:</B> Picking of of <I>E. coli</I> XL1 blue transformed with ligation product F225+F261 and Retrafo of P643.
</P><P><B>Procedure:</B></P><UL><LI> F225+F261: 3 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> P643: 1 Colony was picked from a chloramphenicol plate.
</LI></UL><UL><LI> Each colony was transferred into a tube containing a relation of 1000:1 of LB-medium : antibiotic
</LI></UL><UL><LI> These tubes were transferred into a cell culture shaker at 37 °C and were incubated 12 h
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F225.2BF261"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F225+F261 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F225+F261.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_22nd"><B>Thursday, August 22nd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_Xl1-Blue_transformed_with_ligation_product_F261_.2B_F225_and_Retrafo_of_P643"> Miniprep of <I>E. coli</I> Xl1-Blue transformed with ligation product F261 + F225 and Retrafo of P643 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> Xl1-Blue transformed with ligation product F261 + F225 and Retrafo of P643.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P714 to P716.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P714_.26_P715"> Analytical digestion and gelelectrophoresis of P714 &amp; P715 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P714 &amp; P715.
</P><P><B>Procedure:</B></P><P><B>Batch for digestion with EcoRI &amp; PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>P714
</TD><TD>P715
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD><TD> as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="ReTrafo_of_E..C2.A0coli_XL1-Blue_with_P650.2C_P652.2CP654.2C_P656.2C_P658.2C_P661.2C_P663.2C_P666.2C_P672.2C_P678.2C_P679.2C_P682.2C_P685.2C_P706.2C_P709.2C_P625.2C_P414.2C_P628.2C_P629.2C_P630.2C_P632.2C_P713"> ReTrafo of <I>E. coli</I> XL1-Blue with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> ReTrafo of <I>E. coli</I> XL1-Blue with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA (Ligation) and 1 µl of DNA (Retrafo) were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P625.2C_P414_with_EcoRI.2C_PstI_and_MlyI.2C_P628.2C_P629.2C_P630.2C_P632_with_EcoRI.2C_PstI_and_SspI_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction">Preparative digestion of P625, P414 with EcoRI, PstI and MlyI, P628, P629, P630, P632 with EcoRI, PstI and SspI + Preparative Gelelectrophoresis + Gelextraction</SPAN></H3><P><B>Investigator:</B> Johanna, Louise
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P625, P414:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>MlyI (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P628,P629,P630, P632:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>SspI (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5  h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 0.5% agarose gel.
</LI></UL><UL><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI></UL><UL><LI> The resulting fragments were named  F262 (digestion of P625), F264 (digestion of P628), F263 (digestion of P629, F265 (digestion of P630), F266 (digestion of P632) and F267 (digestion of P414).
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267"> Ligation of F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 </SPAN></H3><P><B>Investigator:</B> Flo, Rosario, Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F230+F262 (SERK-SigP_EreB_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F263 (SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyCatcher in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F264 (SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyTag_NanoLuc in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F265 (SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyTag in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F266 (SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyCatcher_NanoLuc in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F267 (SERK-SigP_Laccase_Strep-tag-II-TEV-site-Linker_SERK-TMD_8AALinker_GFPmut1 in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F230+F262, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>15.96 µl
</TD><TD>F268
</TD></TR><TR><TD>0.7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F263
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>16.6 µl
</TD><TD>F263
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F264, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>5.9 µl
</TD><TD>F264
</TD></TR><TR><TD>10.7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F265, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>3.6 µl
</TD><TD>F265
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F266, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>2.5 µl
</TD><TD>F266
</TD></TR><TR><TD>14.1 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F267, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>9.8 µl
</TD><TD>F267
</TD></TR><TR><TD>6.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. water instead of insert.
</LI></UL><UL><LI> Ligation was performed at room temperature for 1 h.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_23rd"><B>Friday, August 23rd</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P715_with_XbaI_and_PstI_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction">Preparative digestion of P715 with XbaI and PstI + Preparative Gelelectrophoresis + Gelextraction</SPAN></H3><P><B>Investigator:</B> Louise, Flo
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P715:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5  h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 0,5% agarose gel.
</LI></UL><UL><LI> The lower band was cut out
</LI></UL><UL><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P6_.28pGAL_in_pTUM_100.29"> Transformation of <I>E. coli</I> XL1-Blue with P6 (pGAL in pTUM 100) </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with P6 (pGAL in pTUM 100).
<B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new ampicillin plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Sequencing_of_P715_.28sYFP_G_TEV-site_G_cYFP.29">  Sequencing of P715 (sYFP_G_TEV-site_G_cYFP) </SPAN></H3><P><B>Investigator: Louise</B></P><P><B>Aim of the experiment:</B> Sequencing of P715 (sYFP_G_TEV-site_G_cYFP)
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The gene we sequenced received the following barcode:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode
</TH><TH> Primer
</TH></TR><TR><TD>P715
</TD><TD>FR02233118
</TD><TD>03
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Ligation_of_F187.2BF268_and_F188.2BF268">Ligation of F187+F268 and F188+F268</SPAN></H3><P><B>Investigator:</B> Flo
</P><P><B>Aim of the experiment:</B> Ligation of F187+F268 and F188+F268
</P><P><B>Procedure:</B></P><UL><LI>Ligation batch for F187+F268, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.527 µl
</TD><TD>F187
</TD></TR><TR><TD>1.406 µl
</TD><TD>F268
</TD></TR><TR><TD>15.067 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Ligation batch for F188+F268, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.732 µl
</TD><TD>F188
</TD></TR><TR><TD>1.406 µl
</TD><TD>F268
</TD></TR><TR><TD>14.862 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. water instead of insert
</LI></UL><UL><LI> Ligation was performed at room temperature for 1 h
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_P650.2C_P652.2CP654.2C_P656.2C_P658.2C_P661.2C_P663.2C_P666.2C_P672.2C_P678.2C_P679.2C_P682.2C_P685.2C_P706.2C_P709.2C_P625.2C_P414.2C_P628.2C_P629.2C_P630.2C_P632.2C_P713"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B>  Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (6 µl Cam, 6 ml LB-medium)
</LI></UL><UL><LI> 1 colony was picked for every transformation product.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Inoculation_of_50_ml_LB_meida_wirh_E..C2.A0coli_XL1_blue_transformed_with_P650.2C_P652.2CP654.2C_P656.2C_P658.2C_P661.2C_P663.2C_P666.2C_P672.2C_P678.2C_P679.2C_P682.2C_P685.2C_P706.2C_P709.2C_P625.2C_P414.2C_P628.2C_P629.2C_P630.2C_P632.2C_P713"> Inoculation of 50 ml LB meida wirh  <I>E. coli</I> XL1 blue transformed with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B>  Inoculation of Midiprepcultures of  <I>E. coli</I> XL1 blue transformed with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 
</P><P><B>Procedure:</B></P><UL><LI> 1 ml of each of the 6 ml LB Miniprepcultures prepared by Louise was used to inoculate 50 ml Lb media with 50 µl Cam. Incubation overnight in a Minitron shaker at 37 °C and 160 RPM
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_24th"><B>Saturday, August 24th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1-Blue_transformed_with_ligation_products_P650.2C_P652.2CP654.2C_P656.2C_P658.2C_P661.2C_P663.2C_P666.2C_P672.2C_P678.2C_P679.2C_P682.2C_P685.2C_P706.2C_P709.2C_P625.2C_P414.2C_P628.2C_P629.2C_P630.2C_P632.2C_P713"> Miniprep of <I>E. coli</I> XL1-Blue transformed with ligation products P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 </SPAN></H3><P><B>Investigator:</B> Jeff, Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1-Blue transformed with ligation products P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 
</P><P><B>Procedure:</B></P><UL><LI> Ligation products were labelled B to V (in the same order as in Aim of the experiment, above)
</LI></UL><UL><LI> Two clones for every ligation product except for H, L and O
</LI></UL><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P650 to P688.
</LI></UL></DIV></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_media_for_competent_cells_2"> Preparation of media for competent cells </SPAN></H3><P><B> Investigator : </B> Johanna, Louise
</P><P><B>Aim of the experiment:</B>Preparation of media for competent cells
</P><P><B>Procedure:</B></P><UL><LI> Picking of one colony of overnight stem culture of non competent XL1-Blue E.Coli in 50 ml LB-Medium without antibiotics (under the sterile bench)
</LI><LI> this was incubate overnight at 37 °C
</LI></UL><UL><LI> Batch for 1l of a 0.1 M MgCl<SUB>2</SUB>-solution
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>amount</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20.33 g
</TD><TD>MgCl<SUB>2</SUB> (M=203.3 g/mol)
</TD></TR><TR><TD>1 l
</TD><TD>ELGA H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><UL><LI> Batch for 500ml of a 50 mM CaCl<SUB>2</SUB>-solution
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>amount</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.67 g
</TD><TD>CaCl<SUB>2</SUB> (M=147.02 g/mol)
</TD></TR><TR><TD>500 ml
</TD><TD>ELGA H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><UL><LI> Batch for 50ml of a 50 mM CaCl<SUB>2</SUB>-solution, 15% v/v Glycerin
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>amount</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.368 g
</TD><TD>CaCl<SUB>2</SUB> (M=147.02 g/mol)
</TD></TR><TR><TD>0.458 g
</TD><TD>Glycerin (density=1.26)
</TD></TR><TR><TD>50 ml
</TD><TD>ELGA H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><UL><LI> Autoclave each bottle of solution and leave each in a 4°C room.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P496.2C_P661.2C_P629"> Transformation of <I>E. coli</I> XL1-Blue with P496, P661, P629 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with P496, P661, P629.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_ligation_product_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267.2C_F187.2BF268_and_F188.2BF268"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267, F187+F268 and F188+F268 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>  Picking of transformed Plasmid <I>E. coli</I> XL1 blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267, F187+F268 and F188+F268
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl Cam, 4 ml LB-medium)
</LI></UL><UL><LI> 3 colones were picked for every transformation product, 2 for F188+F268 and F187+F268.
</LI></UL></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267_2"> Ligation of F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F230+F262 (SERK-SigP_EreB_Strep-tag-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1 in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F263 (SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyCatcher in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F264 (SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyTag_NanoLuc in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F265 (SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyTag in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F266 (SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyCatcher_NanoLuc in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3), F230+F267 (SERK-SigP_Laccase_Strep-tag-II-TEV-site-Linker_SERK-TMD_8AALinker_GFPmut1 in PppActin_mMCS(insert location)_t35S_npt-cassette_pSB1C3)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F230+F262, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>15.96 µl
</TD><TD>F268
</TD></TR><TR><TD>0.7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F263
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>16.6 µl
</TD><TD>F263
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F264, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>5.9 µl
</TD><TD>F264
</TD></TR><TR><TD>10.7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F265, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>3.6 µl
</TD><TD>F265
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F266, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>2.5 µl
</TD><TD>F266
</TD></TR><TR><TD>14.1 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F267, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.4 µl
</TD><TD>F230
</TD></TR><TR><TD>9.8 µl
</TD><TD>F267
</TD></TR><TR><TD>6.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. water instead of insert.
</LI></UL><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Midiprep_and_preparative_digestion_of_cultures_with_E..C2.A0coli_XL1_blue_transformed_transformed_with_P650.2C_P652.2CP654.2C_P656.2C_P658.2C_P661.2C_P663.2C_P666.2C_P672.2C_P678.2C_P679.2C_P682.2C_P685.2C_P706.2C_P709.2C_P625.2C_P414.2C_P628.2C_P629.2C_P630.2C_P632.2C_P713"> Midiprep and preparative digestion of cultures with  <I>E. coli</I> XL1 blue transformed transformed with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 </SPAN></H3><P><B>Investigator:</B> Ingmar &amp; Louise
</P><P><B>Aim of the experiment:</B>  Production of DNA for moss transformations with P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 
</P><P><B>Procedure:</B></P><UL><LI> The preparation of plasmid DNA was executed according to the user manual provided by Qiagen. The final DNA pellet was dissolved in 500 µl EB buffer.
</LI><LI> The plasmid DNA was linearized with EcoRI HF at 37 °C overnight in batches with the following composition:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Component</B></TD><TD align="center" style="background:#f0f0f0;"><B>Volume [µl]</B></TD></TR><TR><TD> EcoRI HF</TD><TD>5
</TD></TR><TR><TD> DNA</TD><TD>500
</TD></TR><TR><TD> CutSmart</TD><TD>56
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_25th"><B>Sunday, August 25th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_competent_cells_according_to_the_CaCl_2-method_.28Cohen_et_al..2C1972.29_2"> Preparation of competent cells according to the CaCl<SUB> 2</SUB>-method (Cohen et al.,1972)</SPAN></H3><P><B>Investigator:</B> Johanna, Christopher, Louise
</P><P><B>Aim of the experiment:</B> Preparation of competent cells according to the CaCl<SUB> 2</SUB>-method (Cohen et al.,1972)
</P><P><B>Procedure:</B></P><UL><LI>Add Tetracyclin (ratio: 1:1000) to the 2YT-medium 
</LI><LI>Measure the OD<SUB>550</SUB> of it until OD<SUB>550</SUB>=0.5
</LI><LI>Put the culture in Falcon tubes (leaving on ice)
</LI><LI>Centrifuge the falcon tubes by 5000 rpm, 4°C, 10 min
</LI><LI>Resuspend the falcon tubes each with 40 ml 0,1 M MgCl<SUB>2</SUB> solution after removing the supernatants
</LI><LI>Centrifuge the falcon tubes by 5000 rpm, 4°C, 10 min
</LI><LI>Resuspend the falcon tubes each with 20 ml 50mM CaCl<SUB>2</SUB> solution after removing the supernatants
</LI><LI>Incubate the falcon tubes on ice for 30 min
</LI><LI>Centrifuge the falcon tubes by 5000 rpm, 4°C, 10 min
</LI><LI>Resuspend the falcon tubes each with 2 ml 50mM CaCl<SUB>2</SUB>, 15 % v/v Glycerin solution after removing the supernatants. From this step everything needs to be carry out in a 4°C room.
</LI><LI>Mix the solutions of each falcon tube and transfer the content in eppis (each with 100 and accordingly 150 µl)
</LI><LI>Store the boxes of eppis at -80 °C
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F230.2BF228"> Ligation of F230+F228 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F230+F228
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F230+F228, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.41 µl
</TD><TD>F230(243.7 ng/µl; 5796 bp)
</TD></TR><TR><TD>3.21 µl
</TD><TD>F228(25.5 ng/µl; 1581 bp)
</TD></TR><TR><TD>13.38 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. water instead of insert.
</LI></UL><UL><LI> Ligation was performed at room temperature for 1 h.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_P742-P757.2CP759.2C_P760">Analytical gelelectrophoresis of P742-P757,P759, P760</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of P742-P757,P759, P760
</P><P><B>Procedure:</B></P><UL><LI> Procedure of a mixture: 300 µl H<SUB>2</SUB>O + 50 µl DNA loading buffer (10x)
</LI></UL><UL><LI>  8  µl of DNA loading buffer (10x) were added to 1 µl Plasmid and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>P748
</TD><TD>P749
</TD><TD>P750
</TD><TD>P751
</TD><TD>P752
</TD><TD>P753
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>P754
</TD><TD>P755
</TD><TD>P756
</TD><TD>P757
</TD><TD>P759
</TD><TD>P760
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD><TD> as expected
</TD><TD> as not expected
</TD><TD> as not expected
</TD><TD> as not expected
</TD><TD> as not expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>P742
</TD><TD>P743
</TD><TD>P744
</TD><TD>P745
</TD><TD>P746
</TD><TD>P747
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF228">Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F228</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F228.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_ligations_F187.2BF268_.28PIF3_.2B_fret_system.29_and_F188.2BF268_.28PIF6_.2B_fret_system.29_and_analytical_digest_with_EcoRI_.26_PstI"> Miniprep of ligations F187+F268 (PIF3 + fret system) and F188+F268 (PIF6 + fret system) and analytical digest with EcoRI &amp; PstI </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Obtaining ligated plasmids F187+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES + sYFP_G_TEV-site_G_cYFP) (= P761 and P762) and F188+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES + sYFP_G_TEV-site_G_cYFP) (= P763 and P764)
</P><P><B>Procedure:</B></P><UL><LI> Preparation of the plasmids according to the QIAGEN Miniprep Kit, elution in 50 µl
</LI></UL><P><B>Batch for digestion with EcoRI &amp; PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> 1 % agarose gel with ethidium bromide, 90 V, 30 min
</LI></UL><UL><LI> Ctrl: F187 and F188 respectively
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>P761 dig. EcoRI&amp;PstI
</TD><TD>P762 dig. EcoRI&amp;PstI
</TD><TD>F187
</TD><TD>P763 dig. EcoRI&amp;PstI
</TD><TD>P764 dig. EcoRI&amp;PstI
</TD><TD>F188
</TD></TR><TR><TH><B>Result:</B></TH><TD> -
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>ctrl
</TD><TD>not as expected
</TD><TD>not as expected
</TD><TD>ctrl
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P761-P764_with_EcoRI_and_PstI_.2B_preparative_gelelectrophoresis_.2B_gelextraction"> Preparative digestion of P761-P764 with EcoRI and PstI + preparative gelelectrophoresis + gelextraction</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Obtaining fragments of PIF3/PIF6 with fret system for ligation into expression vector
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P761-P764:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5  h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 0,5% agarose gel.
</LI></UL><UL><LI><B>Analytical digest shows failed ligaton, preparative digest was cancelled</B></LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_new_competent_cells_with_P649_.28Retrafo.29"> Transformation of <I>E. coli</I> XL1-Blue new competent cells with P649 (Retrafo) </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Test-Transformation with new batch of competent cells
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on a new chlorampenicol plate.
</LI></UL><UL><LI> The negative control of competent cells which were not transfomed with Plasmid DNA but with water, were plated on all kind of plates: Chloramphenicol, Kanamycin, Ampicillin and tetracyclin
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Troubleshooting:_Repeat_of_ligation_of_F187.2BF268_and_F188.2BF268_and_re-picking_of_the_08.2F23_ligation"> Troubleshooting: Repeat of ligation of F187+F268 and F188+F268 and re-picking of the 08/23 ligation </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F187+F268 and F188+F268
</P><P><B>Procedure:</B></P><UL><LI>Ligation batch for F187+F268, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.53 µl
</TD><TD>F187
</TD></TR><TR><TD>1.41 µl
</TD><TD>F268
</TD></TR><TR><TD>15.067 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Ligation batch for F188+F268, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.73 µl
</TD><TD>F188
</TD></TR><TR><TD>1.41 µl
</TD><TD>F268
</TD></TR><TR><TD>14.862 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> neg. ctrl. water instead of insert
</LI></UL><UL><LI> Ligation was performed at 16 °C over night
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_Retrafo_of_P661_.2824.8..29.2C_P661_.2822.8..29.2C_P629_and_P496"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with Retrafo of P661 (24.8.), P661 (22.8.), P629 and P496 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B>  Picking of transformed Plasmid <I>E. coli</I> XL1 blue with Retrafo of P661 (24.8.), P661 (22.8.), P629 and P496.
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (6 µl Cam, 6 ml LB-medium)
</LI></UL><UL><LI> 2 colonies were picked for every transformation product.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Alcohol_precipitation_and_purification_of_linearized_Mididpreps_of_P650.2C_P652.2CP654.2C_P656.2C_P658.2C_P661.2C_P663.2C_P666.2C_P672.2C_P678.2C_P679.2C_P682.2C_P685.2C_P706.2C_P709.2C_P625.2C_P414.2C_P628.2C_P629.2C_P630.2C_P632.2C_P713"> Alcohol precipitation and purification of linearized Mididpreps of P650, P652,P654, P656, P658, P661, P663, P666, P672, P678, P679, P682, P685, P706, P709, P625, P414, P628, P629, P630, P632, P713 </SPAN></H3><P><B>Investigator:</B> Ingmar &amp; Louise
</P><P><B>Aim of the experiment:</B>  Purification of linearized DNA
</P><P><B>Procedure:</B></P><UL><LI> 350 µl of 1M Sodiumactetat solution (pH adjusted to 5,2 with concentrated acetic acid) were added to each digestion sample. 
</LI><LI> The DNA was precipitated with 595 µl isopropanol analytical grade. Afterwards centrifugation at 15 000 xg for 30 min at 4°C.
</LI><LI>After having removed the supernatant the DNA pellet was washed with 1 ml 70 % EtOH. Afterwards centrifugation at 15 000 xg for 30 min at 4°C.
</LI><LI>The supernatant was removed, the samples once again centrifuged at 16 200 xg for 30 sec. and the accumulated supernatant again removed. 
</LI><LI>The pellets were air-dryed for 2 min under the maniar flow hood. 
</LI><LI>The DNA was solved in 100 µl sterile ddH2O and stored at -20 °C.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_F228">Analytical gelelectrophoresis of F228</SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of F228.
</P><P><B>Procedure:</B></P><UL><LI>  1  µl of DNA loading buffer (10x) were added to 4 µl Fragment and were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>F228
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267_and_P711.2C_P712.2C_P713"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 and P711, P712, P713 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 and P711, P712, P713.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol (for P713 kanamycin) plates.
</LI></UL></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_19">Week 19</SPAN></H1><DIV class="week" id="WWeek_19"><H2><SPAN class="mw-headline" id="Monday.2C_August_26th"><B>Monday, August 26th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1-Blue_transformed_with_Retrafo_of_P661_.2824.8..29.2C_P661_.2822.8..29.2C_P629_and_P496"> Miniprep of <I>E. coli</I> XL1-Blue transformed with Retrafo of P661 (24.8.), P661 (22.8.), P629 and P496 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1-Blue transformed with Retrafo of P661 (24.8.), P661 (22.8.), P629 and P496. 
</P><P><B>Procedure:</B></P><UL><LI> Two clones for every ReTrafo
</LI></UL><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P765 to P772.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P769-P776_with_XbaI_and_PstI"> Analytical digestion and gelelectrophoresis of P769-P776 with XbaI and PstI </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P769-P776 with XbaI and PstI
</P><P><B>Procedure:</B></P><UL><LI> batch for digestion of P769-P776
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15.0 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and were loaded on a 0.5% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD>P769
</TD><TD>P770
</TD><TD>P771
</TD><TD>P772
</TD><TD>P773
</TD><TD>P774
</TD><TD>P775
</TD><TD>P776
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1-Blue_transformed_with_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267_and_P711.2C_P712.2C_P713"> Miniprep of <I>E. coli</I> XL1-Blue transformed with F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 and P711, P712, P713 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1-Blue transformed with F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267 and P711, P712, P713. 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P780 to P799.
</LI></UL></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P780.C2.A0.E2.80.93.C2.A0P797"> Analytical digestion and gelelectrophoresis of P780 – P797 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P780 – P797.
</P><P><B>Procedure:</B></P><P><B>Mastermix for analytical digestion with EcoRI and PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>40 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>5 µl
</TD><TD>PstI
</TD></TR><TR><TD>300 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=350 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were mixedd together with the respective plasmids (P780 – P797).
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 10 µl were loaded on a 0.5% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P780 digested with XbaI &amp; PstI
</TD><TD>P781 digested with XbaI &amp; PstI
</TD><TD>P782 digested with XbaI &amp; PstI
</TD><TD>P783 digested with XbaI &amp; PstI
</TD><TD>P784 digested with XbaI &amp; PstI
</TD><TD>P785 digested with XbaI &amp; PstI
</TD><TD>P786 digested with XbaI &amp; PstI
</TD><TD>P787 digested with XbaI &amp; PstI
</TD><TD>P788 digested with XbaI &amp; PstI
</TD><TD>P789 digested with XbaI &amp; PstI
</TD><TD>P790 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P791 digested with XbaI &amp; PstI
</TD><TD>P792 digested with XbaI &amp; PstI
</TD><TD>P793 digested with XbaI &amp; PstI
</TD><TD>P794 digested with XbaI &amp; PstI
</TD><TD>P795 digested with XbaI &amp; PstI
</TD><TD>P796 digested with XbaI &amp; PstI
</TD><TD>P797 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_ligation_product_F230.2BF262.2C_F230.2BF263.2C_F230.2BF264.2C_F230.2BF265.2C_F230.2BF266.2C_F230.2BF267.2C_F187.2BF268.2C_F188.2BF268.2C_F230.2BF228_and_Retrafos_of_P711.2C_P712.2C_P713.2C_P649"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267, F187+F268, F188+F268, F230+F228 and Retrafos of P711, P712, P713, P649 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B>  Picking of transformed Plasmid <I>E. coli</I> XL1 blue with ligation product F230+F262, F230+F263, F230+F264, F230+F265, F230+F266, F230+F267, F187+F268,F188+F268, F230+F228 and Retrafos of P711, P712, P713, P649
</P><P><B>Procedure:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (4 µl Cam, 4 ml LB-medium)
</LI></UL><UL><LI> 3 colones were picked for every ligation product (5 for F187+F268,F188+F268) , 2 for the Retrafos.
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_27th"><B>Tuesday, August 27th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_F187.2BF268_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_eCFP_G_TEV-site_G_SYFP2.29_and_F188.2BF268_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2.29"> Transformation of <I>E. coli</I> XL1-Blue with ligation products F187+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_eCFP_G_TEV-site_G_SYFP2) and F188+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products F187+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_eCFP_G_TEV-site_G_SYFP2) and F188+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_transformed_Plasmid_E..C2.A0coli_XL1_blue_with_P6.2CP773-P776"> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P6,P773-P776 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Picking of transformed Plasmid <I>E. coli</I> XL1 blue with P6,P773-P776
</P><P><B>Procedure:</B></P><UL><LI> P773-P776: 2 Colonies were picked from chloramphenicol plates.
</LI></UL><UL><LI> P6: 2 Colonies were picked from ampicilin plates.
</LI></UL><UL><LI> P773-P776: Each colony was transferred into a tube containing 6 mL of LB-medium + 6 µL chloramphenicol(1000x).
</LI></UL><UL><LI> P6: Each colony was transferred into a tube containing 6 mL of LB-medium + 6 µL ampicilin(1000x).
</LI></UL><UL><LI> These tubes were transferred into a cell culture shaker at 37 °C and were incubated 12 h
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL1-Blue_transformed_with_P649.2C_F230.2BF196.2C_F230.2B262.2C_F230.2B263.2C_F230.2B264.2C_F230.2B266.2C_F230.2B267"> Miniprep of <I>E. coli</I> XL1-Blue transformed with P649, F230+F196, F230+262, F230+263, F230+264, F230+266, F230+267 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL1-Blue transformed with P649, F230+F196, F230+262, F230+263, F230+264, F230+266, F230+267. 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Spin Miniprep Kit, QIAGEN).
</LI></UL><UL><LI> The resulting tubes were labeled as P800 to P809.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P794"> Transformation of <I>E. coli</I> XL1-Blue with P794 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with P794.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P290.2C_P387.2C_P787_with_EcoRI_and_PstI.2C_and_preparative_digestion_of_P799_with_XbaI_and_PstI_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction">Preparative digestion of P290, P387, P787 with EcoRI and PstI, and preparative digestion of P799 with XbaI and PstI + Preparative Gelelectrophoresis + Gelextraction</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P290/P387/P787:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>NEB 2.1 Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P799:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>NEB 3.1 Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 3  h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 0,5% (P787) and 1% agarose gel.
</LI></UL><UL><LI> The results were corrupt so the experiment was repeated
</LI></UL><UL><LI> The results were corrupt so the experiment was repeated
</LI></UL></DIV><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P800.C2.A0.E2.80.93.C2.A0P809"> Analytical digestion and gelelectrophoresis of P800 – P809 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P800 – P809.
</P><P><B>Procedure:</B></P><P><B>Mastermix for analytical digestion with EcoRI and PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>24 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI
</TD></TR><TR><TD>3 µl
</TD><TD>PstI
</TD></TR><TR><TD>180 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=210 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were mixedd together with the respective plasmids (P780 – P797).
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1.5 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 10 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P800 digested with XbaI &amp; PstI
</TD><TD>P801 digested with XbaI &amp; PstI
</TD><TD>P802 digested with XbaI &amp; PstI
</TD><TD>P803 digested with XbaI &amp; PstI
</TD><TD>P804 digested with XbaI &amp; PstI
</TD><TD>P805 digested with XbaI &amp; PstI
</TD><TD>P806 digested with XbaI &amp; PstI
</TD><TD>P807 digested with XbaI &amp; PstI
</TD><TD>P808 digested with XbaI &amp; PstI
</TD><TD>P809 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Retransformation_of_E..C2.A0coli_XL1-Blue_with_P290.2C_P387.2C_P787.2C_P799"> Retransformation of <I>E. coli</I> XL1-Blue with P290, P387, P787, P799 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Retransformation of <I>E. coli</I> XL1-Blue with P290, P387, P787, P799.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates (Kanamycin for P799).
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_Ligation_F228.2BF230_.28banana_cytoplasm._.2B_plasmid_w.2F_Mini_MCS.29"> Picking of Ligation F228+F230 (banana cytoplasm. + plasmid w/ Mini MCS) </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B></P><UL><LI> Picking of Ligation F228+F230 (banana cytoplasm. + plasmid w/ Mini MCS)
</LI></UL><P><B>Procedure:</B></P><UL><LI> 6 clones picked; 4 ml LB medium w/ 4 µl Cam each
</LI><LI> Incubation at 37 °C over night
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_Ligations_F188.2BF268_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2.29.2C_F230.2BF265_.28PppActin_SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyTag_t35S_npt-cassette.29.2C_F230.2BF266_.28PppActin_SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyCatcher_NanoLuc_t35S_npt-cassette.29_and_F230.2BF267_.28PppActin_SERK-SigP_Laccase_Strep-tag-II-TEV-site-Linker_SERK-TMD_8AALinker_GFPmut1_t35S_npt-cassette.29"> Picking of Ligations F188+F268 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2), F230+F265 (PppActin_SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyTag_t35S_npt-cassette), F230+F266 (PppActin_SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyCatcher_NanoLuc_t35S_npt-cassette) and F230+F267 (PppActin_SERK-SigP_Laccase_Strep-tag-II-TEV-site-Linker_SERK-TMD_8AALinker_GFPmut1_t35S_npt-cassette) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of Ligations F188+F268, F230+F265 (from 08/25 and from 08/26), F230+F266 (from 08/25 and from 08/26) and F230+F267 (from 08/25 and from 08/26) 
</P><P><B>Procedure:</B></P><UL><LI> 6 clones F188+F268
</LI><LI> 3 clones each of F230+F265, F230+F266 and F230+F267 from 08/25
</LI><LI> 3 clones each of F230+F265, F230+F266 and F230+F267 from 08/26
</LI><LI> 4 ml LB medium w/ 4 µl Cam each
</LI><LI> Incubation at 37 °C over night
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_P6_.28pGAL_in_pTUM100.29"> Miniprep of P6 (pGAL in pTUM100) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparation of P6 plasmid DNA
</P><P><B>Procedure:</B></P><P>Preparation according to QIAGEN Miniprep Kit, elution in 50 µl EB buffer each
</P><TABLE border="1"><TBODY><TR><TH><B>plasmid</B></TH><TH><B>c (ng/µl)</B></TH></TR><TR><TD>P810
</TD><TD>384.2
</TD></TR><TR><TD>P811
</TD><TD>134.1
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P810_and_P811_.28pGAL_in_pTUM100.29_with_with_XbaI_and_PstI_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P810 and P811 (pGAL in pTUM100) with with XbaI and PstI + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Jeff, Ingmar
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P810 and P811</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>5 µl
</TD><TD>NEB 3.1 Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis at 90 V for 30 min on 1% agarose gel.
</LI></UL><UL><LI> Lowest band from digested P810 and P811 was cut out.
</LI></UL><UL><LI> Gelextraction was performed with QIAquick Gel Extraction Kit, QIAGEN.
</LI></UL></DIV><DIV class="general"><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F230.2BF273.2C_F230.2BF274.2C_F230.2BF275.2C_F276.2BF277"> Ligation of F230+F273, F230+F274, F230+F275, F276+F277 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F230+F273, F230+F274, F230+F275, F276+F277.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F230+F273, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.40 µl
</TD><TD>F230 (243.7 ng/µl; 5796 bp)
</TD></TR><TR><TD>2.2 µl
</TD><TD>F273 (31.7 ng/µl; 1327 bp)
</TD></TR><TR><TD>14.4 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F274, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.40 µl
</TD><TD>F230 (243.7 ng/µl; 5796 bp)
</TD></TR><TR><TD>1.8 µl
</TD><TD>F274 (46.5 ng/µl; 1597 bp)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F275, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.40 µl
</TD><TD>F230 (243.7 ng/µl; 5796 bp)
</TD></TR><TR><TD>2.1 µl
</TD><TD>F275 (166.9 ng/µl; 6751 bp)
</TD></TR><TR><TD>14.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F276+F277
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.3 µl
</TD><TD>F276 (243.7 ng/µl; 3978 bp)
</TD></TR><TR><TD>12.7 µl
</TD><TD>F277 (166.9 ng/µl; 72 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> For neg. ctrl. water was used instead of insert.
</LI></UL><UL><LI> Ligation was performed at room temperature for 1 h and then put on 16 °C for the next shift to transform the ligation products.
</LI></UL></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_28th"><B>Wednesday, August 28th</B></SPAN></H2><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_F230.2BF273_.28Ig-Kappa-SigP_EreB_.2B_Plasmid_with_Mini-MCS.29.2C_F230.2BF274_.28IgKappa-SigP_Laccase_.2B_Plasmid_with_Mini-MCS.29.2C_F230.2BF275_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_eCFP_G_TEV-site_G_SYFP2_.2B_Plasmid_with_Mini-MCS.29_and_F276.2BF277_.28pBad_.2B_digested_pGAL.29"> Transformation of <I>E. coli</I> XL1-Blue with ligation products F230+F273 (Ig-Kappa-SigP_EreB + Plasmid with Mini-MCS), F230+F274 (IgKappa-SigP_Laccase + Plasmid with Mini-MCS), F230+F275 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_eCFP_G_TEV-site_G_SYFP2 + Plasmid with Mini-MCS) and F276+F277 (pBad + digested pGAL)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> XL1-Blue with ligation products F230+F273 (Ig-Kappa-SigP_EreB + Plasmid with Mini-MCS), F230+F274 (IgKappa-SigP_Laccase + Plasmid with Mini-MCS), F230+F275 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_eCFP_G_TEV-site_G_SYFP2 + Plasmid with Mini-MCS) and F276+F277 (pBad + digested pGAL).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates (having F230 as backbone) and kanamycin plates (having pBad as backbone).
</LI></UL></DIV></DIV><DIV class="general"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_F265.2BF230.2C_F228.2BF230.2C_F267.2BF230.2C_F266.2BF230_and_F268.2BF188"> Miniprep of ligation products F265+F230, F228+F230, F267+F230, F266+F230 and F268+F188 </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI>Preparation according to QIAGEN Miniprep Kit, elution in 50 µl EB buffer each
</LI></UL><UL><LI>The Plasmids were labelled as following:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>ligation product</B></TH><TH><B>label</B></TH></TR><TR><TD>F265+F230 clones 1a-3b
</TD><TD>P812-P817
</TD></TR><TR><TD>F268+F188 clones 1-6
</TD><TD>P818-P823
</TD></TR><TR><TD>F228+F230 clones 1-5
</TD><TD>P824-P828
</TD></TR><TR><TD>F267+F230 clones 1a-3b
</TD><TD>P829-P834
</TD></TR><TR><TD>F266+F230 clones 1a-3b
</TD><TD>P835-P840
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P812-P840"> Analytical digestion and gelelectrophoresis of P812-P840 </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B>Analytical digestion and gelelectrophoresis of P812-P840 .
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI and PstI
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=210 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 10 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P812 digested with XbaI &amp; PstI
</TD><TD>P813 digested with XbaI &amp; PstI
</TD><TD>P814 digested with XbaI &amp; PstI
</TD><TD>P815 digested with XbaI &amp; PstI
</TD><TD>P816 digested with XbaI &amp; PstI
</TD><TD>P817 digested with XbaI &amp; PstI
</TD><TD>P818 digested with XbaI &amp; PstI
</TD><TD>P819 digested with XbaI &amp; PstI
</TD><TD>P820 digested with XbaI &amp; PstI
</TD><TD>P821 digested with XbaI &amp; PstI
</TD><TD>P822 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>P823 digested with XbaI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>P824 digested with XbaI &amp; PstI
</TD><TD>P825 digested with XbaI &amp; PstI
</TD><TD>P826 digested with XbaI &amp; PstI
</TD><TD>P827 digested with XbaI &amp; PstI
</TD><TD>P828 digested with XbaI &amp; PstI
</TD><TD>P829 digested with XbaI &amp; PstI
</TD><TD>P830 digested with XbaI &amp; PstI
</TD><TD>P831 digested with XbaI &amp; PstI
</TD><TD>P832 digested with XbaI &amp; PstI
</TD><TD>P833 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>P834 digested with XbaI &amp; PstI
</TD><TD>2 log DNA ladder
</TD><TD>P835 digested with XbaI &amp; PstI
</TD><TD>P836 digested with XbaI &amp; PstI
</TD><TD>P837 digested with XbaI &amp; PstI
</TD><TD>P838 digested with XbaI &amp; PstI
</TD><TD>P839 digested with XbaI &amp; PstI
</TD><TD>P840 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD><TD>TBD
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_29th"><B>Thursday, August 29th</B></SPAN></H2><DIV class="effectors"><DIV class="general"><H3><SPAN class="mw-headline" id="Midiprep_cultures_from_Miniprep_cultures_P815_and_P835_as_well_as_P823-P828"> Midiprep cultures from Miniprep cultures P815 and P835 as well as P823-P828</SPAN></H3><P><B>Investigator:</B> Jeff, Ingmar
</P><P><B>Aim of the experiment:</B> Midiprep culture from Miniprep cultures P815 (SERK-SigP_SpyCatcher_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_NanoLuc_SpyTag + Plasmid mit Mini MCS), P835 (SERK-SigP_SpyTag_Strep-Tag-II-TEV-site-linker_SERK-TMD_8AAlinker_GFPmut1_IRES_IgKappa-SigP_SpyCatcher_NanoLuc + Plasmid mit Mini MCS) and P823-P828 (Banana odor generator in Mini MCS)
</P><P><B>Procedure:</B></P><UL><LI> 50 ml LB medium w/ 50 µl chloramphenicol stock (1:1000)
</LI><LI> 1 ml preculture each
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_Ligation_F276.2BF277_.28pBad_.2B_MCS_w.2F_StrepTag.29"> Picking of Ligation F276+F277 (pBad + MCS w/ StrepTag) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Procedure:</B></P><UL><LI> 2 clones picked; 4 ml LB medium w/ 8 µl Kanamycin
</LI><LI> Incubation at 37 °C
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P823_with_EcoRI_.26_PstI_and_gelextraction"> Preparative digestion of P823 with EcoRI &amp; PstI and gelextraction </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P823 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + eCFP_G_TEV-site_G_SYFP2)
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P823</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>NEB 2.1 Buffer (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>EcoRI-HF(20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C for 1.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis at 90 V for 30 min on 1% agarose gel.
</LI></UL><UL><LI>Result as expected; the upper band was cut out, processed with QIAGEN gel extraction kit and labeled F279
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_Retrafos_of_P290.2C_P387.2C_P787.2C_P794.2C_P799"> Picking of Retrafos of P290, P387, P787, P794, P799 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B></P><P>Picking of Retrafos of P290, P387, P787, P794, P799
</P><P><B>Procedure:</B></P><UL><LI> 2 clones picked; 4 ml LB medium w/ 4 µl Cam each (Kan for P799)
</LI><LI> Incubation at 37 °C over day
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_Ligation_products_F230.2BF273.2C_F230.2BF274.2C_F230.2BF275_and_F276.2BF277"> Picking of Ligation products F230+F273, F230+F274, F230+F275 and F276+F277 </SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> 
Picking of Ligation products F230+F273, F230+F274, F230+F275 and F276+F277
</P><P><B>Procedure:</B></P><UL><LI> The picked clones were each transferred into 4 ml LB medium with 4 µl Chloramphenicol (Kan for F276+F277)
</LI><LI> Incubation at 37 </LI></UL><TABLE border="1"><TBODY><TR><TH><B>Ligation</B></TH><TH><B>amount of clones picked</B></TH></TR><TR><TD>F230+F273
</TD><TD>4
</TD></TR><TR><TD>F230+F274
</TD><TD>4
</TD></TR><TR><TD>F230+F275
</TD><TD>2
</TD></TR><TR><TD>F276+F277
</TD><TD>2
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF279"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F279</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F279.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol (for P713 kanamycin) plates.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_F268.2BF187_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES_in_pSB1C3_.2B_eCFP_G_TEV-site_G_SYFP2.29_and_F268.2BF188_.28PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_in_pSB1C3_.2B_eCFP_G_TEV-site_G_SYFP2.29"> Miniprep of ligation products F268+F187 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF3_IRES in pSB1C3 + eCFP_G_TEV-site_G_SYFP2) and F268+F188 (PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES in pSB1C3 + eCFP_G_TEV-site_G_SYFP2)</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI>Preparation according to QIAGEN Miniprep Kit, elution in 50 µl EB buffer each
</LI></UL><UL><LI>The Plasmids were labelled as following:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>ligation product</B></TH><TH><B>label</B></TH></TR><TR><TD>F268+F187 clones 1-3
</TD><TD>P841-P843
</TD></TR><TR><TD>F268+F188 clones 1-3
</TD><TD>P844-P846
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P841-P846"> Analytical digestion and gelelectrophoresis of P841-P846 </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B>Analytical digestion and gelelectrophoresis of P841-846 .
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI and PstI
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 10 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P841 digested with XbaI &amp; PstI
</TD><TD>P842 digested with XbaI &amp; PstI
</TD><TD>P843 digested with XbaI &amp; PstI
</TD><TD>P844 digested with XbaI &amp; PstI
</TD><TD>P845 digested with XbaI &amp; PstI
</TD><TD>P846 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>false
</TD><TD>false
</TD><TD>false
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_F230.2BF279_.28PppActin_PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2_t35S_npt-cassette.29"> Picking of <I>E. coli</I> XL1-Blue transformed F230+F279 (PppActin_PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2_t35S_npt-cassette) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed F230+F279 (PppActin_PhyB_36AAlinker_C-TEV_IRES_N-TEV_36AAlinker_PIF6_IRES_eCFP_G_TEV-site_G_SYFP2_t35S_npt-cassette).
</P><P><B>Procedure:</B></P><UL><LI>6 colonies were picked from the plates with <I>E. coli</I>s transformed with the ligation product F230+F279.
</LI></UL><UL><LI> Picked colonies were transferred in sterile air-permeable cell culture tubes with 4 µl LB medium with chloramhpenicol.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_F230.2BF273.2C_F230.2BF274.2C_F230.2BF275.2C_F230.2BF277_and_of_P290.2C_P387.2C_P787.2C_P794.2C_P799"> Miniprep of ligation products F230+F273, F230+F274, F230+F275, F230+F277 and of P290, P387, P787, P794, P799  </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Miniprep of ligation products F230+F273, F230+F274, F230+F275, F230+F277 and of P290, P387, P787, P794, P799.
</P><P><B>Procedure:</B></P><P>Miniprep according to QIAGEN QIAprep Kit 
Tubes were labelled P847 - P858.
</P></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_30th"><B>Friday, August 30th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transition_of_F234-F260_transformed_moss_onto_full_Knop_medium_agar_plates"> Transition of F234-F260 transformed moss onto full Knop medium agar plates </SPAN></H3><P><B>Investigator:</B> Johanna, Andi
</P><P><B>Aim of the experiment:</B> Transition of F234-F260 transformed moss onto full Knop medium agar plates
</P><P>'<I>Procedure:</I></P><P>Pipet 2 ml from wells onto cellophane layer over the solid medium
</P></DIV><DIV class="effectors"><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Miniprep_of_F276_.2B_F277_.28pBAD_.2B_MCS.29_and_F230_.2B_F279_.28FRET_in_pActin_npt-cassette.29"> Miniprep of F276 + F277 (pBAD + MCS) and F230 + F279 (FRET in pActin_npt-cassette) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Miniprep of F276 + F277 (pBAD + MCS) and F230 + F279 (FRET in pActin_npt-cassette).
</P><P><B>Procedure:</B></P><P>Miniprep according to QIAGEN QIAprep Kit 
</P></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P857_and_P858"> Analytical digestion and gelelectrophoresis of P857 and P858 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P857 and P858.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>NEBuffer CutSmart
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15.25 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for ?? min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>F276
</TD><TD>P857 digested with XbaI
</TD><TD>P858 digested with XbaI
</TD></TR><TR><TH><B>Result:</B></TH><TD>inconclusive
</TD><TD>inconclusive
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P868_and_P869"> Analytical digestion and gelelectrophoresis of P868 and P869 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P868 and P869.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>NEBuffer CutSmart
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>15.25 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for ?? min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>F276
</TD><TD>P868 digested with XbaI
</TD><TD>P869 digested with XbaI
</TD></TR><TR><TH><B>Result:</B></TH><TD>inconclusive
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P870_-_P875"> Analytical digestion and gelelectrophoresis of P870 - P875 </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P870 - P875.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 0.5% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2 log DNA ladder
</TD><TD>P870 digested with XbaI &amp; PstI
</TD><TD>P871 digested with XbaI &amp; PstI
</TD><TD>P872 digested with XbaI &amp; PstI
</TD><TD>P873 digested with XbaI &amp; PstI
</TD><TD>P874 digested with XbaI &amp; PstI
</TD><TD>P875 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>bad
</TD><TD>bad
</TD><TD>as expected
</TD><TD>bad
</TD><TD>bad
</TD></TR></TBODY></TABLE></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P717_.28Laccase-rec..29_and_P860_.28EreB-sec..29_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P717 (Laccase-rec.) and P860 (EreB-sec.) + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Preparative digestion of P717 (Laccase-rec.) and P860 (EreB-sec.) + Preparative Gelelectrophoresis + Gelextraction.
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P717:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P717
</TD></TR><TR><TD>4 µl
</TD><TD>2.1 Buffer (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>3 µl
</TD><TD>MlyI (10 U/µl)
</TD></TR><TR><TD>9.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P860:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P860
</TD></TR><TR><TD>4 µl
</TD><TD>2.1 Buffer (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>12.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 3 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 0.5% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>2log DNA ladder
</TD><TD>Digestion of P717 with EcoRI &amp; PstI &amp; MlyI
</TD><TD>Digestion of P860 with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; upper band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI><LI> The resulting fragments were named F280 (Laccase-rec.) + F281 (EreB-sec.)
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F230.2BF280_.28Laccase-rec..29_and_F230.2BF281_.28EreB-sec..29"> Ligation of F230+F280 (Laccase-rec.) and F230+F281 (EreB-sec.) </SPAN></H3><P><B>Investigator:</B> Jeff, Florian
</P><P><B>Aim of the experiment:</B> Ligation of F230+F280 (Laccase-rec.) and F230+F281 (EreB-sec.).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F230+F280, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.416 µl
</TD><TD>F230 (243.7 ng/µl; 5796 bp)
</TD></TR><TR><TD>3.314 µl
</TD><TD>F280 (41.1 ng/µl; 2632 bp)
</TD></TR><TR><TD>13.3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F230+F281, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.416 µl
</TD><TD>F230 (243.7 ng/µl; 5796 bp)
</TD></TR><TR><TD>1.659 µl
</TD><TD>F281 (41.4 ng/µl; 1327 bp)
</TD></TR><TR><TD>14.93 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> For neg. ctrl. water was used instead of insert.
</LI></UL><UL><LI> Ligation was performed at room temperature for 1 h and then transformed.
</LI></UL></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F230.2BF280_.28Laccase-rec..29_and_F230.2BF281_.28EreB-sec..29"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F280 (Laccase-rec.) and F230+F281 (EreB-sec.) </SPAN></H3><P><B>Investigator:</B> Flo
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> XL1-Blue with ligation product F230+F280 (Laccase-rec.) and F230+F281 (EreB-sec.).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new chlorampenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_1st"><B>Sunday, September 1st</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P624_.28GST.29_and_P645_.28Banana.29_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P624 (GST) and P645 (Banana) + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparative digestion of P624 (GST) and P645 (Banana) + Preparative Gelelectrophoresis + Gelextraction.
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P624:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P624
</TD></TR><TR><TD>4 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P645:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P645
</TD></TR><TR><TD>4 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>1kB DNA ladder
</TD><TD>Digestion of P624 with XbaI and PstI
</TD><TD>Digestion of P645 with XbaI and PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD><TD>as expected; lower band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI><LI> The resulting fragments were named F289 (GST) and F290 (BANANE)
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="PCR.2C_purification_and_analytical_geleletrophoresis_of_P634_.28pActin.29_and_P800_.28pAct_MCS.29"> PCR, purification and analytical geleletrophoresis of P634 (pActin) and P800 (pAct_MCS) </SPAN></H3><P><B>Investigator: Rosario</B></P><P><B>Aim of the experiment:</B>  PCR, purification and analytical geleletrophoresis of P554 (pActin) and P800 (pAct_MCS).
</P><P><B>Procedure:</B></P><P>Operational sequence:
</P><UL><LI> PCR reaction mixture
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O126 (pAct5_fw_cor)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O117 (pAct5_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P634) - (desired c between 1 ng and 5 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>25 µl
</TD><TD>2x Q5 Master Mix
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Forward Primer O126 (pAct5_fw_cor)
</TD></TR><TR><TD>2.5 µl
</TD><TD>10 µM Reverse Primer O117 (pAct5_rv)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P800) - (desired c between 1 ng and 5 ng)
</TD></TR><TR><TD>19 µL
</TD><TD>ddH<SUB>2</SUB>O Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mix with pipette
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>98 °C
</TD><TD>30 s
</TD></TR><TR><TD>54 °C;
</TD><TD>60 s
</TD></TR><TR><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>2 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, the product was purified with QIAquick PCR Purification Kit, Qiagen.
</LI></UL><UL><LI> 9 µl of the purified PCR products were mixed with 1 µl DNA loading buffer (10x) for analytical gelelectrophoresis
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min in 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>Fragment of digested P634
</TD><TD>Fragment of digested P800
</TD></TR><TR><TD>Fragment-size like expected
</TD><TD>also pppActin5 without MCS
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_20">Week 20</SPAN></H1><DIV class="week" id="WWeek_20"><H2><SPAN class="mw-headline" id="Monday.2C_September_2nd"><B>Monday, September 2nd</B></SPAN></H2><H2><SPAN class="mw-headline" id="Tuesday.2C_September_3rd"><B>Tuesday, September 3rd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F276.2BF289_and_F276.2BF290"> Ligation of F276+F289 and F276+F290 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Ligation of F276+F289 and F276+F290.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F276+F289, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.3 µl
</TD><TD>F276
</TD></TR><TR><TD>6.5 µl
</TD><TD>F280
</TD></TR><TR><TD>6.2 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F276+F290, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.3 µl
</TD><TD>F276
</TD></TR><TR><TD>12.7 µl
</TD><TD>F290
</TD></TR><TR><TD>0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature for 1 h and then transformed.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_F289.2BF276_and_F290.2BF276"> Transformation of <I>E. coli</I> XL1-Blue with ligation product F289+F276 and F290+F276 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> XL1-Blue with ligation product F289+F276 and F290+F276
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 150 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new Kanamycin plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_F288.2BF282.2C_F288.2BF283.2C_F288.2BF284.2C_F288.2BF285.2C_F288.2BF286_and_F288.2BF287_.28Effectors_in_pBAD_fro_protein_expression.29"> Miniprep of F288+F282, F288+F283, F288+F284, F288+F285, F288+F286 and F288+F287 (Effectors in pBAD fro protein expression)</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Miniprep of F288+F282, F288+F283, F288+F284, F288+F285, F288+F286 and F288+F287 (Effectors in pBAD fro protein expression).
</P><P><B>Procedure:</B></P><UL><LI>Miniprep according to QIAGEN QIAprep Kit 
</LI><LI>The resulting tubes were labelled P909-P920
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P909-P920_with_XbaI_and_PstI"> Analytical digestion and gelelectrophoresis of P909-P920 with XbaI and PstI</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P909-P920 with XbaI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P909 digested with XbaI &amp; PstI
</TD><TD>P910 digested with XbaI &amp; PstI
</TD><TD>P911 digested with XbaI &amp; PstI
</TD><TD>P912 digested with XbaI &amp; PstI
</TD><TD>P913 digested with XbaI &amp; PstI
</TD><TD>P914 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P915 digested with XbaI &amp; PstI
</TD><TD>P916 digested with XbaI &amp; PstI
</TD><TD>P917 digested with XbaI &amp; PstI
</TD><TD>P918 digested with XbaI &amp; PstI
</TD><TD>P919 digested with XbaI &amp; PstI
</TD><TD>P920 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>not fully digested
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>corrupt
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_Knop_full-medium"> Preparation of Knop full-medium</SPAN></H3><P><B>Investigator:</B> Louise
</P><P><B>Aim of the experiment:</B> Preparation of 500 ml stock solution of 100x Knop full-medium.
</P><P><B>Procedure:</B></P><UL><LI>The composition of the solution is based on the Knop-Vollmedium in the Reski dissertation: <A href="http://www.freidok.uni-freiburg.de/volltexte/1684/pdf/DissHorstmann02-05.pdf" class="external free" rel="nofollow">http://www.freidok.uni-freiburg.de/volltexte/1684/pdf/DissHorstmann02-05.pdf</A>, which in turn is based on the publication: Schween, G., Fleig, S., and Reski, R. (2002) High-throughput-PCR screen of 15,000 transgenic Physcomitrella plants. Plant Molecular Biology Reporter 20, 43-47.
</LI><LI> The compounds were prepared in separate stock solutions and then added to the Knop-medium with agar by sterile filtration. With this full-medium we prepared agar plates on which the transformed moss was transferred
</LI><LI>batch for stock solutions, slightly different than from the dissertation:
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_4th"><B>Wednesday, September 4th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_F276.2BF289_and_F276.2BF290"> Picking of <I>E. coli</I> XL1-Blue transformed F276+F289 and F276+F290 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed F276+F289 and F276+F290.
</P><P><B>Procedure:</B></P><UL><LI> 2 colonies were picked from each plate with <I>E. coli</I>s transformed with the ligation product F276+F289 and F276+F290.
</LI></UL><UL><LI> Picked colonies were transferred in sterile air-permeable cell culture tubes with 4 µl LB medium with Kannamycin.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparation_of_Knop-Medium_for_Moss_3">Preparation of Knop-Medium for Moss</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparation of Knop-Medium for Moss
</P><P><B>Procedure:</B></P><UL><LI> All four prepared stocks have been: 25g/L KH2PO4; 25g/L KCl; 25g/L MgSO4 x 7 H2O; 100g/L Ca(NO3)2  
</LI><LI> Volume of prepared Medium: 5L
</LI><LI> 50 ml of each stock were converted into a 5L Erlenmeyer flask and filled up to 5L with destilled water (ELGA); 62,5mg FeSO4 x 7 H2O were added; the pH was set to 5.8 with KOH
</LI><LI> The 5L flask has been autoclaved
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P921.2BP923_with_XbaI_and_PstI"> Analytical digestion and gelelectrophoresis of P921+P923 with XbaI and PstI</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P921+P923 with XbaI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>Cut Smart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P921 digested with XbaI &amp; PstI
</TD><TD>P923 digested with XbaI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_5th"><B>Thursday, September 5th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_PL21_with_P921_and_P923"> Transformation of <I>E. coli</I> PL21 with P921 and P923 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> PL21 with P921 and P923. 
<B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> PL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new Kanamycin plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_6th"><B>Friday, September 6th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transfer_of_the_moss_from_first_transformation_onto_G418_agar_plates"> Transfer of the moss from first transformation onto G418 agar plates </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Transfer of the moss from first transformation onto G418 agar plates
</P><P><B>Procedure:</B></P><P>Transfer moss on top of the cellophane layer onto the G418 full Knop agar plates to start selection.
</P></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Retransformation_of_E..C2.A0coli_XL1blue_with_P168.2C_P247.2C_P268.2C_P270.2C_P276.2C_P303.2C_P328.2C_P334.2C_P336.2C_P358.2C_P367.2C_P374.2C_P391.2C_P412.2C_P416.2C_P418.2C_P422.2C_P464.2C_P466.2C_P486.2C_P488.2C_P492.2C_P509.2C_P565.2C_P715.2C_P823_.28Retransformation_of_Biobricks.29"> Retransformation of <I>E. coli</I> XL1blue with P168, P247, P268, P270, P276, P303, P328, P334, P336, P358, P367, P374, P391, P412, P416, P418, P422, P464, P466, P486, P488, P492, P509, P565, P715, P823 (Retransformation of Biobricks)  </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>Retransformation of <I>E. coli</I> XL1blue with P168, P247, P268, P270, P276, P303, P328, P334, P336, P358, P367, P374, P391, P412, P416, P418, P422, P464, P466, P486, P488, P492, P509, P565, P715, P823 (Retransformation of Biobricks)  
<B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of cell suspension was plated on new chloramphenicol plates
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_7th"><B>Saturday, September 7th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Expression_of_EreB.2C_FluA.2C_nLuc.2C_XylE.2C_Laccase.2C_PP1.2C_GST_and_banana_odour"> Expression of EreB, FluA, nLuc, XylE, Laccase, PP1, GST and banana odour </SPAN></H3><P><B>Investigator:</B> Andi, Rosario, Volker
</P><P><B>Aim of the experiment:</B> Expression of EreB, FluA, nLuc, XylE, Laccase, PP1, GST and banana odour.
</P><P><B>Procedure:</B></P><UL><LI> 4 ml 1:500 Kanamycin mix were added to each of eight 2 l flasks of LB medium, then they were inoculated with 50 ml overnight cultures.
</LI></UL><UL><LI> After the culture reached an OD of 0.7, expression was induced by adding 20 ml of 500 mM Arabinose solution to each flask. Protein expression was continued for 4 hours.
</LI></UL><UL><LI> Then the cells were pelleted by centrifuging for 10 min at 5000 rpm.
</LI></UL><UL><LI> The pellets were resuspended in 20 ml SA buffer containing 0.5 mg/ml Lysozyme.
</LI></UL><UL><LI> After short incubation the cells were disrupted by the application of ultrasound.
</LI></UL><UL><LI> The disrupted cells were centrifuged at 20000 rpm for 1 hour to remove cell pieces and aggregate.
</LI></UL><UL><LI> The supernatant was put in dialysis tubes and suspended in SA buffer overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Setup_of_toxicity_assay"> Setup of toxicity assay</SPAN></H3><P><B>Investigator:Louise</B></P><P><B>Aim of the experiment:</B> Testing the reaction of wild type moss to different noxious substances
</P><P><B>Procedure:</B></P><UL><LI>In 96-well plates we prepared a serial dilution of 11 substances in dilution steps of 1:4 (1/5th) over 12 wells
</LI><LI>As a negative control we used distilled water
</LI><LI> To the prepared 80 µl solutions 100 µl of homogeneous moss culture in Knop medium were added
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_biobricks_.28retrafo.29"> Picking of <I>E. coli</I> XL1-Blue transformed with biobricks (retrafo) </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed with biobricks (retrafo)
<B>Procedure:</B></P><UL><LI> 2 colonies were picked from each plate
</LI></UL><UL><LI> Picked colonies were transferred in sterile air-permeable cell culture tubes with 5 µl LB medium with Chloramphenicol.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_8th"><B>Sunday, September 8th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_biobrick-retransformations_.28P168.2C_P247.2C_P268.2C_P270.2C_P276.2C_P303.2C_P328.2C_P334.2C_P336.2C_P358.2C_P367.2C_P374.2C_P391.2C_P412.2C_P416.2C_P418.2C_P422.2C_P464.2C_P466.2C_P486.2C_P488.2C_P492.2C_P509.2C_P565.2C_P715.2C_P823.29"> Miniprep of biobrick-retransformations (P168, P247, P268, P270, P276, P303, P328, P334, P336, P358, P367, P374, P391, P412, P416, P418, P422, P464, P466, P486, P488, P492, P509, P565, P715, P823)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of biobrick retransformations
</P><P><B>Procedure:</B></P><UL><LI>Miniprep according to QIAGEN QIAprep Kit 
</LI><LI>The resulting tubes were labelled P925-P976
</LI></UL></DIV><P>!--- PLEASE DO NOT TOUCH !!!! --&gt;</P></DIV><H1><SPAN class="mw-headline" id="Week_21">Week 21</SPAN></H1><DIV class="week" id="WWeek_21"><H2><SPAN class="mw-headline" id="Monday.2C_September_9th"><B>Monday, September 9th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transition_of_F234-F260_transformed_moss_onto_G418_full_Knop_medium_agar_plates"> Transition of F234-F260 transformed moss onto G418 full Knop medium agar plates </SPAN></H3><P><B>Investigator:</B> Ingmar, Florian, Jeff
</P><P><B>Aim of the experiment:</B> Transition of the photosensitive transformed moss onto G418 full Knop medium agar plates
</P><P>'<I>Procedure:</I></P><P>Pipet 2 ml from wells onto cellophane layer over the solid medium, working under blue light only
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_10th"><B>Tuesday, September 10th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Expression_of_EreB"> Expression of EreB </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Expression of EreB.
</P><P><B>Procedure:</B></P><UL><LI> 4 ml 1:500 Kanamycin mix were added to each of six 2 l flasks of LB medium, then they were inoculated with 50 ml EreB overnight cultures.
</LI></UL><UL><LI> The absorption was measured every 30 minutes:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>EreB</B></TH></TR><TR><TD>0
</TD><TD>0.091
</TD></TR><TR><TD>30
</TD><TD>0.180
</TD></TR><TR><TD>60
</TD><TD>0.314
</TD></TR><TR><TD>90
</TD><TD>0.482
</TD></TR><TR><TD>120
</TD><TD>0.748
</TD></TR></TBODY></TABLE><UL><LI> After the culture reached an OD of 0.7, expression was induced by adding 20 ml of 500 mM Arabinose solution to each flask. Protein expression was continued for 5.5 hours and every 30 min the absorption was measured and a sample was collected for a SDS-PAGE.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>EreB</B></TH></TR><TR><TD>0
</TD><TD>0.786
</TD></TR><TR><TD>30
</TD><TD>0.960
</TD></TR><TR><TD>75
</TD><TD>1.315
</TD></TR><TR><TD>90
</TD><TD>1.408
</TD></TR><TR><TD>120
</TD><TD>1.565
</TD></TR><TR><TD>150
</TD><TD>1.624
</TD></TR><TR><TD>180
</TD><TD>1.698
</TD></TR><TR><TD>210
</TD><TD>1.738
</TD></TR><TR><TD>240
</TD><TD>1.776
</TD></TR><TR><TD>300
</TD><TD>1.839
</TD></TR><TR><TD>330
</TD><TD>1.903
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">SDS-PAGE of samples of the expression of EreB</DIV></DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TH><B>Sample:</B></TH><TD>Unstained Protein Marker
</TD><TD>0 min
</TD><TD>90 min
</TD><TD>120 min
</TD><TD>150 min
</TD><TD>180 min
</TD><TD>210 min
</TD><TD><B>330 min</B></TD><TD>240 min
</TD><TD>300 min
</TD></TR></TBODY></TABLE><UL><LI> After 5.5 hours the cells were pelleted by centrifuging for 10 min at 5000 rpm.
</LI></UL><UL><LI> The pellets were resuspended in 20 ml SA buffer containing 0.5 mg/ml Lysozyme.
</LI></UL><UL><LI> After short incubation the cells were disrupted by the application of ultrasound.
</LI></UL><UL><LI> The disrupted cells were centrifuged at 20000 rpm for 1 hour to remove cell pieces and aggregate.
</LI></UL><UL><LI> The supernatant was put in dialysis tubes and suspended in SA buffer overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P340.2C_P281_.2C_P285.2C_P621.2C_P396.2C_P926.2C_P456.2C_P288.2C_P622.2C_P319.2C_P928.2C_P253.2C_P930.2C_P932.2C_P419.2C_P934.2C_P860.2C_P303.2C_P305.2C_P320.2C_P399.2C_P938.2C_P623.2C_P332.2C_P940.2C_P942.2C_P338.2C_P398.2C_P718.2C_P458.2C_P626.2C_P355.2C_P358.2C_P400.2C_P401.2C_P389.2C_P946.2C_P948.2C_P862.2C_P950.2C_P952.2C_P717.2C_P954.2C_P956.2C_P958.2C_P430.2C_P431.2C_P797.2C_P960.2C_P962.2C_P628.2C_P964.2C_P966.2C_P968.2C_P772.2C_P630.2C_P970.2C_P510.2C_P632.2C_P633.2C_P538.2C_P638.2C_P562.2C_P972.2C_P614.2C_P617.2C_P972.2C_P864_and_P976_with_EcoRI_and_PstI"> Analytical digestion and gelelectrophoresis of P340, P281 , P285, P621, P396, P926, P456, P288, P622, P319, P928, P253, P930, P932, P419, P934, P860, P303, P305, P320, P399, P938, P623, P332, P940, P942, P338, P398, P718, P458, P626, P355, P358, P400, P401, P389, P946, P948, P862, P950, P952, P717, P954, P956, P958, P430, P431, P797, P960, P962, P628, P964, P966, P968, P772, P630, P970, P510, P632, P633, P538, P638, P562, P972, P614, P617, P972, P864 and  P976 with EcoRI and PstI </SPAN></H3><P><B>Investigator:</B> Johanna,Andi
</P><P><B>Aim of the experiment:</B>
Analytical digestion and gelelectrophoresis of  P340(1), P281(2), P285(3), P621(4), P396(5), P926(6), P456(7), P288(8), P622(9), P319(10), P928(11), P253(12), P930(13), P932(14), P419(15), P934(16), P860(17), P303(18), P305(19), P320(20), P399(21), P938(22), P623(23), P332(24), P940(25), P942(26), P338(27), P398(29), P718(30), P458(31), P626(32), P355(33), P358(34), P400(35), P401(36), P389(37), P946(38), P948(39), P862(40), P950(41), P952(42), P717(43), P954(44), P956(45), P958(46), P430(47), P431(48), P797(49), P960(50), P962(51), P628(52), P964(53), P966(54), P968(55), P772(56), P630(57), P970(58), P510(59), P632(60), P633(61), P538(62), P638(63), P562(64), P972(65), P614(66), P617(67), P972(68), P864(69) and  P976(70) with EcoRI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with EcoRI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>17.5 µl
</TD><TD>Mastermix
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for Mastermix(80x)
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>160 µl
</TD><TD>CutSmart(10x)
</TD></TR><TR><TD>20 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>20 µl
</TD><TD>PstI
</TD></TR><TR><TD>1200 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=1400 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 2 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD><TD>11
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>12
</TD><TD>13
</TD><TD>14
</TD><TD>15
</TD><TD>16
</TD><TD>17
</TD><TD>18
</TD><TD>19
</TD><TD>20
</TD><TD>21
</TD><TD>22
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>23
</TD><TD>24
</TD><TD>25
</TD><TD>26
</TD><TD>27
</TD><TD>28
</TD><TD>29
</TD><TD>30
</TD><TD>31
</TD><TD>32
</TD><TD>33
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>34
</TD><TD>35
</TD><TD>36
</TD><TD>37
</TD><TD>38
</TD><TD>39
</TD><TD>40
</TD><TD>41
</TD><TD>42
</TD><TD>43
</TD><TD>44
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>lost :(
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>45
</TD><TD>46
</TD><TD>47
</TD><TD>48
</TD><TD>49
</TD><TD>50
</TD><TD>51
</TD><TD>52
</TD><TD>53
</TD><TD>54
</TD><TD>55
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>56
</TD><TD>57
</TD><TD>58
</TD><TD>59
</TD><TD>60
</TD><TD>61
</TD><TD>62
</TD><TD>63
</TD><TD>64
</TD><TD>65
</TD><TD>66
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>67
</TD><TD>68
</TD><TD>69
</TD><TD>70
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as not expected
</TD><TD>as not expected
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P614.2CP456.2CP400.2CP401.2CP973.2CP868_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P614,P456,P400,P401,P973,P868 + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Andi,Johanna
</P><P><B>Aim of the experiment:</B> Preparative digestion of P614,P456,P400,P401,P973,P868 + Preparative Gelelectrophoresis + Gelextraction.
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P614(digested with EcoRI-HF and AgeI-HF):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>P614
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>AgeI-HF(20 U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P614(digested with NgoMIV and PstI):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>P614
</TD></TR><TR><TD>2 µl
</TD><TD>NE-Buffer 1.0
</TD></TR><TR><TD>0.5 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI(20 U/µl)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA(20 U/µl)
</TD></TR><TR><TD>6.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P456(digested with Age-HF and PstI):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>P456
</TD></TR><TR><TD>2 µl
</TD><TD>NE-Buffer 1.0
</TD></TR><TR><TD>0.5 µl
</TD><TD>Age-HF(20 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI(20 U/µl)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA(20 U/µl)
</TD></TR><TR><TD>6.8 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P456(digested with EcoRI-HF and NgoMIV):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>P456
</TD></TR><TR><TD>2 µl
</TD><TD>Cutsmart
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI-HF(20 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P400,P401,P973:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>2 µl
</TD><TD>Cutsmart
</TD></TR><TR><TD>0.5 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>0.5 µl
</TD><TD>Age-HF(20 U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P868:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P868
</TD></TR><TR><TD>4 µl
</TD><TD>Cutsmart
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><UL><LI> Picture of the first Gel was not taken, but every single digestion showed the expected bands
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P614 with EcoRI and AgeI
</TD><TD>P614 with NgoMIV and PstI
</TD><TD>P456 with AgeI and PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>lower band was cut out
</TD><TD>lower band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P456 with EcoRI and NgoMIV
</TD><TD>P401 with XbaI and Age
</TD><TD>P400 with XbaI and AgeI
</TD></TR><TR><TH><B>Result:</B></TH><TD>1kb Ladder
</TD><TD>band was cut out
</TD><TD>lower band was cut out
</TD><TD>lower band was cut out
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P973 with XbaI and AgeI
</TD><TD>P868 with AgeI and NgoMIV
</TD></TR><TR><TH><B>Result:</B></TH><TD>lower band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Gelextraction was performed after preparative Gelelectrophoresis
</LI><LI> The resulting fragments were named as you see in the inventary list.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_11th"><B>Wednesday, September 11th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P946_.28BB38.29.2C_P864_.28BB69.29.2C_P975.2BP976_.28BB70.29.2C_P382.2BP387_.28BB72.29_with_EcoRI_and_PstI"> Analytical digestion and gelelectrophoresis of P946 (BB38), P864 (BB69), P975+P976 (BB70), P382+P387 (BB72) with EcoRI and PstI</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P946 (BB38), P864 (BB69), P975+P976 (BB70), P382+P387 (BB72) with EcoRI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with EcoRI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>Cut Smart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P397 digested with EcoRI &amp; PstI
</TD><TD>P382 digested with EcoRI &amp; PstI
</TD><TD>P976 digested with EcoRI &amp; PstI
</TD><TD>P975 digested with EcoRI &amp; PstI
</TD><TD>P864 digested with EcoRI &amp; PstI
</TD><TD>P946 digested with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>ZONK
</TD></TR></TBODY></TABLE></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F296.2BF293.2C_F295.2BF294.2C_F300.2BF298.2C_F300.2BF297.2C_F300.2BF299"> Ligation of F296+F293, F295+F294, F300+F298, F300+F297, F300+F299 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Ligation of F276+F289 and F276+F290.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F296+F293, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.8 µl
</TD><TD>F296
</TD></TR><TR><TD>5.6 µl
</TD><TD>F293
</TD></TR><TR><TD>6.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F295+F294, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>8 µl
</TD><TD>F295
</TD></TR><TR><TD>4.7 µl
</TD><TD>F294
</TD></TR><TR><TD>0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F300+F298, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>F300
</TD></TR><TR><TD>2.6 µl
</TD><TD>F298
</TD></TR><TR><TD>11.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F300+F297, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>F300
</TD></TR><TR><TD>2.6 µl
</TD><TD>F297
</TD></TR><TR><TD>11.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F300+F299, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>F300
</TD></TR><TR><TD>8 µl
</TD><TD>F299
</TD></TR><TR><TD>6.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature for 1 h and then transformed.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_F296.2BF293.2C_F295.2BF294.2C_F300.2BF298.2C_F300.2BF297.2C_F300.2BF299"> Transformation of <I>E. coli</I> F296+F293, F295+F294, F300+F298, F300+F297, F300+F299 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> F296+F293, F295+F294, F300+F298, F300+F297, F300+F299. 
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> PL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new Kanamycin/Chloramphenicol plates.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_12th"><B>Thursday, September 12th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_F296.2BF293.2C_F295.2BF294.2C_P382_and_P397"> Transformation of <I>E. coli</I> F296+F293, F295+F294, P382 and P397 </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> F296+F293, F295+F294, P382 and P397. 
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> PL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new Kanamycin/Chloramphenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F296.2BF293.2C_F295.2BF294"> Ligation of F296+F293, F295+F294 </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F296+F293, F295+F294.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F296+F293, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.8 µl
</TD><TD>F296
</TD></TR><TR><TD>5.6 µl
</TD><TD>F293
</TD></TR><TR><TD>6.6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F295+F294, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>8 µl
</TD><TD>F295
</TD></TR><TR><TD>4.7 µl
</TD><TD>F294
</TD></TR><TR><TD>0 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature for 1 h and then transformed.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_13th"><B>Friday, September 13th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Expression_of_nLuc_and_XylE"> Expression of nLuc and XylE </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Expression of nLuc and XylE.
</P><P><B>Procedure:</B></P><UL><LI> 4 ml 1:500 Kanamycin mix were added to each of six 2 l flasks of LB medium, then five of them were inoculated with 50 ml nLuc overnight cultures and one of them was inoculated with 50 ml XylE overnight culture.
</LI></UL><UL><LI> The absorption was measured every 30 minutes:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>nLuc</B></TH><TH><B>XylE</B></TH></TR><TR><TD>0
</TD><TD>0.103
</TD><TD>0.083
</TD></TR><TR><TD>30
</TD><TD>0.128
</TD><TD>0.110
</TD></TR><TR><TD>60
</TD><TD>0.154
</TD><TD>0.133
</TD></TR><TR><TD>90
</TD><TD>0.224
</TD><TD>0.194
</TD></TR><TR><TD>120
</TD><TD>0.282
</TD><TD>0.246
</TD></TR><TR><TD>150
</TD><TD>0.396
</TD><TD>0.345
</TD></TR><TR><TD>180
</TD><TD>0.564
</TD><TD>0.506
</TD></TR><TR><TD>210
</TD><TD>0.765
</TD><TD>0.722
</TD></TR></TBODY></TABLE><UL><LI> After the cultures reached an OD of 0.7, expression was induced by adding 20 ml of 500 mM Arabinose solution to each flask. Protein expression was continued for 5 hours and every 30 min the absorption was measured and a sample was collected for a SDS-PAGE.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>nLuc</B></TH><TH><B>XylE</B></TH></TR><TR><TD>0
</TD><TD>0.765
</TD><TD>0.722
</TD></TR><TR><TD>30
</TD><TD>1.003
</TD><TD>0.939
</TD></TR><TR><TD>60
</TD><TD>1.228
</TD><TD>1.159
</TD></TR><TR><TD>90
</TD><TD>1.393
</TD><TD>1.353
</TD></TR><TR><TD>120
</TD><TD>1.559
</TD><TD>1.550
</TD></TR><TR><TD>150
</TD><TD>1.656
</TD><TD>1.646</TD></TR><TR><TD>180
</TD><TD>1.746
</TD><TD>1.729
</TD></TR><TR><TD>210
</TD><TD>1.760
</TD><TD>1.760
</TD></TR><TR><TD>240
</TD><TD>1.807
</TD><TD>1.816
</TD></TR><TR><TD>300
</TD><TD>1.878
</TD><TD>1.878
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">SDS-PAGE of samples of the expression of nLuc</DIV></DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TH><B>Sample:</B></TH><TD>Unstained Protein Marker
</TD><TD>0 min
</TD><TD>30 min
</TD><TD>60 min
</TD><TD>90 min
</TD><TD>120 min
</TD><TD>150 min
</TD><TD>180 min
</TD><TD>240 min
</TD><TD>300 min
</TD></TR></TBODY></TABLE><UL><LI> After 5 hours the cells were pelleted by centrifuging for 10 min at 5000 rpm.
</LI></UL><UL><LI> The pellets were resuspended in 20 ml SA buffer containing 0.5 mg/ml Lysozyme.
</LI></UL><UL><LI> After short incubation the cells were disrupted by the application of ultrasound.
</LI></UL><UL><LI> The disrupted cells were centrifuged at 20000 rpm for 1 hour to remove cell pieces and aggregate.
</LI></UL><UL><LI> The supernatant was put in dialysis tubes and suspended in SA buffer overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_P382.2C_P397.2C_pppActin_after_QCI.2C_F296.2BF293"> Picking of <I>E. coli</I> XL1-Blue transformed P382, P397, pppActin after QCI, F296+F293 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed P382, P397, pppActin after QCI, F296+F293.
</P><P><B>Procedure:</B></P><UL><LI> 3 colonies were picked from each plate with <I>E. coli</I>s transformed with P382, P397, pppActin after QCI, F296+F293.
</LI></UL><UL><LI> Picked colonies were transferred in sterile air-permeable cell culture tubes with 5 µl LB medium with Chloramphenicol.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P614_and_P456_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P614 and P456 + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparative digestion of P614,P456 + Preparative Gelelectrophoresis + Gelextraction.
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P614(digested with NgoMIV and PstI):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>P614
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI(20 U/µl)
</TD></TR><TR><TD>6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P456(digested with Age-HF and PstI):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>10 µl
</TD><TD>P456
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>Age-HF(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI(20 U/µl)
</TD></TR><TR><TD>6 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P456 with AgeI and PstI
</TD><TD>P614 with NgoMIV and PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD>lower band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Fragments were named F301(digestion of P456) and F302 (digestion of P614).
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_F297.2BF300.2C_F298.2BF300.2C_F299.2BF300_.28Effectors_in_pBAD.29"> Miniprep of F297+F300, F298+F300, F299+F300 (Effectors in pBAD)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Miniprep of F297+F300, F298+F300, F299+F300
</P><P><B>Procedure:</B></P><UL><LI>Miniprep according to QIAGEN QIAprep Kit 
</LI><LI>The resulting tubes were labelled P977-P986
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_14th"><B>Saturday, September 14th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_P4.2C_P923_for_Midiprep"> Picking of <I>E. coli</I> XL1-Blue transformed P4, P923 for Midiprep </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed P4, P923 for Midiprep.
</P><P><B>Procedure:</B></P><UL><LI> 3 colonies were picked from each plate with <I>E. coli</I>s transformed with P4, P923.
</LI></UL><UL><LI> Picked colonies were transferred in sterile air-permeable cell culture tubes with 5 µl LB medium with Chloramphenicol.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F301.2BF302"> Ligation of F301+F302</SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F301+F302.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F301+F302, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>6.9 µl
</TD><TD>F301
</TD></TR><TR><TD>6.0 µl
</TD><TD>F302
</TD></TR><TR><TD>4.3 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature for 1 h and then transformed.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_F300.2BF301"> Transformation of <I>E. coli</I> F300+F301 </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> F300+F301. 
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> PL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new Chloramphenicol plates.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_P382.2C_P397.2C_pppActin_after_QCI.2C_F296.2BF293"> Miniprep of P382, P397, pppActin after QCI, F296+F293 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Miniprep of P382,P397,pppActin after QCI P634, F296+F293.
</P><P><B>Procedure:</B></P><UL><LI>Miniprep according to QIAGEN QIAprep Kit 
</LI><LI>The resulting tubes were labelled P987-P989 (Miniprep of Retrafo P382), P990-P992( Miniprep of Retrafo P397),P993-P995 (Miniprep of F296+F293) and P996-P998 (Miniprep of pppActin QC P634.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Oligohybridization_of_O301.2BO302_.28Histag.29_and_O303.2BO304_.28Streptag.29">  Oligohybridization of O301+O302 (Histag) and O303+O304 (Streptag) </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment: </B>Oligohybridization of O301+O302 (Histag) and O303+O304 (Streptag)
<B>Procedure:</B></P><UL><LI>25 µL of 100 pmol/µl of  Oligo fw  and 25 µL of 100 pmol/µl  Oligo rev 
</LI></UL><UL><LI>Heating up to 95 °C for 5 min 
</LI></UL><UL><LI>Cooling at RT in a styropor box overnight
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_Gelelectrophoresis_.2B_Gelextraction_of_Oligohybridization_products_.28Histag_.2B_Streptag.29."> Preparative Gelelectrophoresis + Gelextraction of Oligohybridization products (Histag + Streptag). </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparative Gelelectrophoresis + Gelextraction of Oligohybridization products (Histag + Streptag).
</P><P><B>Procedure:</B></P><UL><LI> Preparative gelelectrophoresis of Oligohybridization products were performed at 90 V for 1 h in an 2% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>Oligohybridization of O301+O302
</TD><TD>Oligohybridization of O303+F304
</TD></TR><TR><TH><B>Result:</B></TH><TD>band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><UL><LI> Fragments were named F303 (Histag) and F304 (Streptag).
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P977-P986.2CP993-P998_with_XbaI_and_PstI"> Analytical digestion of P977-P986,P993-P998 with XbaI and PstI </SPAN></H3><P><B>Investigator:</B> Andi, Johanna
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P977-P986,P993-P998 with XbaI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with XbaI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>17.5 µl
</TD><TD>Mastermix
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Batch for Mastermix
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>4.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>4.5 µl
</TD><TD>PstI
</TD></TR><TR><TD>36.0 µl
</TD><TD>NEBuffer 3.1
</TD></TR><TR><TD>270 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=315 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P977
</TD><TD>P978
</TD><TD>P979
</TD><TD>P980
</TD><TD>P981
</TD><TD>P982
</TD><TD>P983
</TD><TD>P984
</TD><TD>P985
</TD><TD>P986
</TD><TD>P993
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD> =(
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P994
</TD><TD>P995
</TD><TD>P996
</TD><TD>P997
</TD><TD>P998
</TD></TR><TR><TH><B>Result:</B></TH><TD>as expected
</TD><TD>as expected
</TD><TD> =(
</TD><TD> =(
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_15th"><B>Sunday, September 15th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Expression_of_XylE_and_Laccase"> Expression of XylE and Laccase </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Expression of XylE and Laccase.
</P><P><B>Procedure:</B></P><UL><LI> 4 ml 1:500 Kanamycin mix were added to each of seven 2 l flasks of LB medium, then six of them were inoculated with 50 ml Laccase overnight cultures and one of them was inoculated with 50 ml XylE overnight culture.
</LI></UL><UL><LI> The absorption was measured every 30 minutes:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>Laccase</B></TH><TH><B>XylE</B></TH></TR><TR><TD>0
</TD><TD>0.084
</TD><TD>0.085
</TD></TR><TR><TD>30
</TD><TD>0.130
</TD><TD>0.132
</TD></TR><TR><TD>60
</TD><TD>0.195
</TD><TD>0.203
</TD></TR><TR><TD>90
</TD><TD>0.258
</TD><TD>0.265
</TD></TR><TR><TD>120
</TD><TD>0.352
</TD><TD>0.377
</TD></TR><TR><TD>150
</TD><TD>0.528
</TD><TD>0.564
</TD></TR><TR><TD>180
</TD><TD>0.730
</TD><TD>0.790
</TD></TR></TBODY></TABLE><UL><LI> After the cultures reached an OD of 0.7, expression was induced by adding 20 ml of 500 mM Arabinose solution to each flask. Protein expression was continued for 5 hours and every 30 min the absorption was measured and a sample was collected for a SDS-PAGE.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>Laccase</B></TH><TH><B>XylE</B></TH></TR><TR><TD>0
</TD><TD>0.730
</TD><TD>0.790
</TD></TR><TR><TD>30
</TD><TD>0.974
</TD><TD>1.032
</TD></TR><TR><TD>60
</TD><TD>1.428
</TD><TD>1.470
</TD></TR><TR><TD>90
</TD><TD>1.798
</TD><TD>1.712
</TD></TR><TR><TD>120
</TD><TD>2.098
</TD><TD>2.052
</TD></TR><TR><TD>150
</TD><TD>2.852
</TD><TD>2.572
</TD></TR><TR><TD>180
</TD><TD>3.060
</TD><TD>2.808
</TD></TR><TR><TD>210
</TD><TD>3.208
</TD><TD>3.000
</TD></TR><TR><TD>240
</TD><TD>3.372
</TD><TD>3.184
</TD></TR><TR><TD>270
</TD><TD>3.476
</TD><TD>3.340
</TD></TR><TR><TD>300
</TD><TD>3.640
</TD><TD>3.404
</TD></TR></TBODY></TABLE><UL><LI> After 5 hours the cells were pelleted by centrifuging for 10 min at 5000 rpm.
</LI></UL><UL><LI> The pellets were resuspended in 20 ml SA buffer containing 0.5 mg/ml Lysozyme.
</LI></UL><UL><LI> After short incubation the cells were disrupted by the application of ultrasound.
</LI></UL><UL><LI> The disrupted cells were centrifuged at 20000 rpm for 1 hour to remove cell pieces and aggregate.
</LI></UL><UL><LI> The supernatant was put in dialysis tubes and suspended in SA buffer overnight.
</LI></UL></DIV><DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Midiprep_of_P4_and_P923">  Midiprep of P4 and P923 </SPAN></H3><P><B>Investigator: Andi</B></P><P><B>Aim of the experiment:</B> Midiprep of P4 and P923.
</P><P><B>Procedure:</B></P><UL><LI> Midiprep was performed after manufacturer's protocol (QIAprep Midiprep, QIAGEN)
</LI></UL></DIV></DIV><DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Miniprep_of_P999_.2B_P1000"> Miniprep of P999 + P1000 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Miniprep of P999 + P1000.
</P><P><B>Procedure:</B></P><UL><LI>Miniprep according to QIAGEN QIAprep Kit 
</LI></UL></DIV></DIV><DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P990.2C_P994.2C_P999.2C_P1000_.2B_Preparative_Gelelectrophoresis_.2B_Gelextraction"> Preparative digestion of P990, P994, P999, P1000 + Preparative Gelelectrophoresis + Gelextraction </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Preparative digestion of P990, P994, P999, P1000 + Preparative Gelelectrophoresis + Gelextraction
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P990):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P990
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P994(digested with EcoRI and NgoMIV):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P994
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P999(digested with EcoRI and NgoMIV):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P998
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P1000(digested with EcoRI and NgoMIV):</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P1000
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>NgoMIV(20 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P990 with EcoRI and NgoMIV
</TD><TD>P994 with EcoRI and NgoMIV
</TD></TR><TR><TH><B>Result:</B></TH><TD>band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><P><A href="/File:HRSN.png" title="File:HRSN.png">500px</A></P><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P999 with EcoRI and NgoMIV
</TD><TD>P1000 with EcoRI and NgoMIV
</TD></TR><TR><TH><B>Result:</B></TH><TD>band was cut out
</TD><TD>band was cut out
</TD></TR></TBODY></TABLE><P><A href="/File:ASDFDAFDS.png" title="File:ASDFDAFDS.png">500px</A></P><UL><LI> Fragments were named Flululu(digestion of P990), Fgddf (digestion of P994), Fgddf (digestion of P999), Fgddf (digestion of P1000).
</LI></UL></DIV></DIV><DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P999_and_P1000_with_EcoRI_and_PstI"> Analytical digestion and gelelectrophoresis of P999 and P1000 with EcoRI and PstI</SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B>Analytical digestion and gelelectrophoresis of P999 and P1000 with EcoRI and PstI.
</P><P><B>Procedure:</B></P><UL><LI> Batch for digestion with EcoRI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2.0 µl
</TD><TD>Cut Smart Buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI
</TD></TR><TR><TD>14.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 20 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1kb DNA ladder
</TD><TD>P999 digested with EcoRI &amp; PstI
</TD><TD>P1000 digested with EcoRI &amp; PstI
</TD></TR><TR><TH><B>Result:</B></TH><TD> =(
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Ligation_of_F305.2BF303.2C_F306.2BF303.2C_F307.2BF304"> Ligation of F305+F303, F306+F303, F307+F304 </SPAN></H3><P><B>Investigator:</B> Andi
</P><P><B>Aim of the experiment:</B> Ligation of F305+F303, F306+F303, F307+F304.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F305+F303, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.86 µl
</TD><TD>F305
</TD></TR><TR><TD>1 µl
</TD><TD>F303
</TD></TR><TR><TD>15.14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F306+F303, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.9 µl
</TD><TD>F306
</TD></TR><TR><TD>1 µl
</TD><TD>F303
</TD></TR><TR><TD>15.1 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F307+F304, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.1 µl
</TD><TD>F307
</TD></TR><TR><TD>1 µl
</TD><TD>F304
</TD></TR><TR><TD>14.9 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed at room temperature for 1 h.
</LI></UL></DIV><P>!--- PLEASE DO NOT TOUCH !!!! --&gt;</P></DIV><H1><SPAN class="mw-headline" id="Week_22">Week 22</SPAN></H1><DIV class="week" id="WWeek_22"><H2><SPAN class="mw-headline" id="Monday.2C_September_16th"><B>Monday, September 16th</B></SPAN></H2><DIV><DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_F305.2BF303.2CF306.2BF303.2CF307.2BF304"> Transformation of <I>E. coli</I> F305+F303,F306+F303,F307+F304 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B>Transformation of <I>E. coli</I> F305+F303,F306+F303,F307+F304. 
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> PL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of DNA were added to the tubes containing 100 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new Chloramphenicol plates.
</LI></UL></DIV></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Analytical_digest_of_fragments_used_for_the_second_moss_transformation"> Analytical digest of fragments used for the second moss transformation </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Analytical digest of the fragments used for the second moss transformation
</P><P><B>Procedure:</B></P><UL><LI> The linearized fragments were originally obtained by digestion of the plasmids with EcoRI
</LI></UL><UL><LI> Batch for digestion with XbaI and PstI.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.1 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>1.0 µl
</TD><TD>3.1 Buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.5 µl
</TD><TD>PstI
</TD></TR><TR><TD>7.9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=10 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> 1 µl of DNA loading buffer (10x) added &amp; loaded on 1% agarose gel.
</LI></UL><UL><LI> Gel electrophoresis at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2-log DNA ladder
</TD><TD>P745
</TD><TD>P896
</TD><TD>empty
</TD><TD>P903
</TD><TD>P893
</TD><TD>P746
</TD><TD>P895
</TD><TD>P742
</TD><TD>P743
</TD><TD>P898
</TD></TR><TR><TH><B>Result:</B></TH><TD> -
</TD><TD>correct
</TD><TD>correct
</TD><TD> -
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>XylE!
</TD><TD>correct
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2-log DNA ladder
</TD><TD>P885
</TD><TD>P886
</TD><TD>P899
</TD><TD>P876
</TD><TD>P892
</TD><TD>P753
</TD><TD>P883
</TD><TD>P894
</TD><TD>P900
</TD></TR><TR><TH><B>Result:</B></TH><TD> -
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2-log DNA ladder
</TD><TD>P877
</TD><TD>P887
</TD><TD>P888
</TD><TD>P878
</TD><TD>P879
</TD><TD>P880
</TD><TD>P882
</TD><TD>P884
</TD><TD>P752
</TD></TR><TR><TH><B>Result:</B></TH><TD> -
</TD><TD>correct
</TD><TD>undigested band?
</TD><TD>empty vector
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>wrong
</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2-log DNA ladder
</TD><TD>P901
</TD><TD>P904
</TD><TD>P751
</TD><TD>P897
</TD><TD>P902
</TD><TD>P881
</TD><TD>P889
</TD><TD>P890
</TD><TD>P891
</TD></TR><TR><TH><B>Result:</B></TH><TD> -
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>correct
</TD><TD>Nanoluc membr.
</TD><TD>GFP secr.
</TD><TD>EreB cyt.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_17th"><B>Tuesday, September 17th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Expression_of_nLuc"> Expression of nLuc </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Expression of nLuc.
</P><P><B>Procedure:</B></P><UL><LI> 4 ml 1:500 Kanamycin mix were added to each of four 2 l flasks of LB medium, then they were inoculated with 50 ml nLuc overnight cultures.
</LI></UL><UL><LI> The absorption was measured every 30 minutes:
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>nLuc</B></TH></TR><TR><TD>0
</TD><TD>0.107
</TD></TR><TR><TD>30
</TD><TD>0.145
</TD></TR><TR><TD>60
</TD><TD>0.199
</TD></TR><TR><TD>85
</TD><TD>0.260
</TD></TR><TR><TD>130
</TD><TD>0.436
</TD></TR><TR><TD>150
</TD><TD>0.565
</TD></TR><TR><TD>180
</TD><TD>0.788
</TD></TR></TBODY></TABLE><UL><LI> After the culture reached an OD of 0.7, expression was induced by adding 20 ml of 500 mM Arabinose solution to each flask. Protein expression was continued for 5 hours and every 30 min the absorption was measured and a sample was collected for a SDS-PAGE.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Time</B></TH><TH><B>nLuc</B></TH></TR><TR><TD>0
</TD><TD>0.788
</TD></TR><TR><TD>30
</TD><TD>0.936
</TD></TR><TR><TD>60
</TD><TD>1.360
</TD></TR><TR><TD>90
</TD><TD>1.806
</TD></TR><TR><TD>120
</TD><TD>2.364
</TD></TR><TR><TD>160
</TD><TD>2.780
</TD></TR><TR><TD>180
</TD><TD>2.976
</TD></TR><TR><TD>210
</TD><TD>3.156
</TD></TR><TR><TD>240
</TD></TR><TR><TD>300
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:TUM13_20130918_SDS_920-nLuc_Expression_scan.png" title="File:TUM13 20130918 SDS 920-nLuc Expression scan.png">File:TUM13 20130918 SDS 920-nLuc Expression scan.png</A><DIV class="thumbcaption">SDS-PAGE of samples of the expression of nLuc</DIV></DIV></DIV></DIV><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>1
</TD><TD>2
</TD><TD>3
</TD><TD>4
</TD><TD>5
</TD><TD>6
</TD><TD>7
</TD><TD>8
</TD><TD>9
</TD><TD>10
</TD></TR><TR><TH><B>Sample:</B></TH><TD>Unstained Protein Marker
</TD><TD>0 min
</TD><TD>30 min
</TD><TD>60 min
</TD><TD>90 min
</TD><TD>120 min
</TD><TD>160 min
</TD><TD>180 min
</TD><TD>240 min
</TD><TD>300 min
</TD></TR></TBODY></TABLE><UL><LI> After 5 hours the cells were pelleted by centrifuging for 10 min at 5000 rpm.
</LI></UL><UL><LI> The pellets were resuspended in 20 ml SA buffer containing 0.5 mg/ml Lysozyme.
</LI></UL><UL><LI> After short incubation the cells were disrupted by the application of ultrasound.
</LI></UL><UL><LI> The disrupted cells were centrifuged at 20000 rpm for 1 hour to remove cell pieces and aggregate.
</LI></UL><UL><LI> The supernatant was put in dialysis tubes and suspended in SA buffer overnight.
</LI></UL></DIV><DIV class="general"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_F305.2BF303.2CF306.2BF303.2CF307.2BF304"> Picking of <I>E. coli</I> XL1-Blue transformed with F305+F303,F306+F303,F307+F304 </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed with F305+F303,F306+F303,F307+F304 
</P><P><B>Procedure:</B></P><UL><LI> 3 colonies were picked from each plate
</LI></UL><UL><LI> Picked colonies were transferred in sterile air-permeable cell culture tubes with 6 µl LB medium with Chloramphenicol.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_BL21_with_P980_.28N-TEV-Protease_36AAlinker_PIF3_Strep-tag-II.29.2C_P983_.28N-TEV-Protease_36AAlinker_PIF6_Strep-tag-II.29_and_P985_.28eCFP_G-TEV-site-G_SYFP2_Strep-tag-II.29"> Transformation of <I>E. coli</I> BL21 with P980 (N-TEV-Protease_36AAlinker_PIF3_Strep-tag-II), P983 (N-TEV-Protease_36AAlinker_PIF6_Strep-tag-II) and P985 (eCFP_G-TEV-site-G_SYFP2_Strep-tag-II) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> BL21 with P980 (N-TEV-Protease_36AAlinker_PIF3_Strep-tag-II), P983 (N-TEV-Protease_36AAlinker_PIF6_Strep-tag-II) and P985 (eCFP_G-TEV-site-G_SYFP2_Strep-tag-II) to express the proteins encoded in the plasmids for further characterizations.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> BL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1 µl of DNA was added to 66 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension was plated on one chloramphenicol plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on a new kanamycin plate.
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E..C2.A0coli_XL1_blue_with_ligation_products_of_F305.2BF303.2C_F307.2BF304_and_F306.2BF303">  Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with ligation products of F305+F303, F307+F304 and F306+F303  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E. coli</I> XL1 blue with ligation products of F305+F303, F307+F304 and F306+F303.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAprep Miniprep, QIAGEN.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P1002_.E2.80.93_P1004_.28His-tag_PhyB_36AAlinker_C-TEV-Protease.29"> Analytical digestion and gelelectrophoresis of P1002 – P1004 (His-tag_PhyB_36AAlinker_C-TEV-Protease)  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P1002 – P1004 (His-tag_PhyB_36AAlinker_C-TEV-Protease).
</P><P><B>Procedure:</B></P><P><B>Batch for digestion of P1002 – P1004 with EcoRI-NheI &amp; PstI</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>0.25 µl
</TD><TD>NheI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2  µl of DNA loading buffer (10x) were added to the reaction batches and 10 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane:</B></TH><TD>2log DNA ladder
</TD><TD> P1002
</TD><TD> P1003
</TD><TD> P1004
</TD></TR><TR><TH><B>Result:</B></TH><TD> as expected
</TD><TD> as expected
</TD><TD> as expected
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_18th"><B>Wednesday, September 18th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_BL21_transformed_with_P980.2C_P983_and_P985_for_Pre-preculture_for_expression"> Picking of <I>E. coli</I> BL21 transformed with P980, P983 and P985 for Pre-preculture for expression </SPAN></H3><P><B>Investigator:</B> Flo, Rosario
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> BL21 transformed with P980, P983 and P985 for Preculture for expression.
</P><P><B>Procedure:</B></P><UL><LI> 1 Colony per construct (P980, P983, P985) were picked with a pipette tip and were transferred in a sterile air-permeable cell-culture tube with 2 µl of LB medium + 4 µl of kanamycin (500x).
</LI></UL><UL><LI> Incubation at 37 °C and 800 rpm overnight.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_19th"><B>Thursday, September 19th</B></SPAN></H2><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Inoculating_preculture_of_E..C2.A0coli_BL21_transformed_with_P980.2C_P983.2C_P985"> Inoculating preculture of <I>E. coli</I> BL21 transformed with P980, P983, P985 </SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Inoculating preculture of <I>E. coli</I> BL21 transformed with P980, P983, P985.
</P><P><B>Procedure:</B></P><UL><LI> Two flasks with 50 ml + 100 µl kanamycin were incoluated each with 1 ml of the pre-preculture of each construct (P980, P983, P985).
</LI></UL></DIV><DIV class="safety_mechanism"><H3><SPAN class="mw-headline" id="Expression_of_P980.2C_P983_and_P985"> Expression of P980, P983 and P985 </SPAN></H3><P><B>Investigator:</B> Florian, Rosario
</P><P><B>Aim of the experiment:</B> Expression of P980, P983 and P985.
</P><P><B>Procedure:</B></P><UL><LI> 4 ml 1:500 Kanamycin mix were added to each of six 2 l flasks of LB medium, then they were inoculated with 50 ml overnight cultures (two flask for each construct).
</LI></UL><UL><LI> After the culture reached an OD of 0.7, expression was induced by adding 20 ml of 500 mM Arabinose solution to each flask. Protein expression was continued for 5 hours.
</LI></UL><UL><LI> Then the cells were pelleted by centrifuging for 10 min at 5000 rpm.
</LI></UL><UL><LI> The pellets were resuspended in 20 ml SA buffer containing 0.5 mg/ml Lysozyme.
</LI></UL><UL><LI> After short incubation the cells were disrupted by the application of ultrasound.
</LI></UL><UL><LI> The disrupted cells were centrifuged at 20000 rpm for 1 hour to remove cell pieces and aggregate.
</LI></UL><UL><LI> The supernatant was put in dialysis tubes and suspended in SA buffer overnight.
</LI></UL></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P928.2C_P1000_and_P1002"> Preparative digestion and gelelectrophoresis of P928, P1000 and P1002 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P928, P1000 and P1002.
</P><P><B>Procedure:</B></P><P><B>Reaction batch for P928:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P928
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P1000:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P1000
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Reaction batch for P1002:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>P1002
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 3 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1.5 h in an 0.5% agarose gel.
</LI></UL><P><A href="/File:TUM13_20130919_prepgel_P928_EcoRI.NgoMIV_P1000_XbaI.AgeI_P1002_XbaI.PstI.png" title="File:TUM13 20130919 prepgel P928 EcoRI.NgoMIV P1000 XbaI.AgeI P1002 XbaI.PstI.png">500px</A></P><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P928 with EcoRI &amp; NgoMIV
</TD><TD>2-log DNA ladder
</TD><TD>Digestion of P1000 with XbaI &amp; AgeI
</TD><TD>Digestion of P1002 with XbaI &amp; PstI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; band was cut out
</TD><TD>as expected; lower band was cut out
</TD><TD>as expected; upper band was cut out
</TD></TR></TBODY></TABLE><UL><LI>Gel extraction was performed after preparative gelelectrophoresis with QIQquick Gel Extration Kit, QIAGEN.
</LI></UL><UL><LI> The resulting fragments were named F308 for digested P928, F309 for digested P1000 and F310 for digested P1002.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F306.2BF304.2C_F308.2BF303.2C_F300.2BF309.2C_F276.2BF310"> Ligation of F306+F304, F308+F303, F300+F309, F276+F310 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F306+F304, F308+F303, F300+F309, F276+F310.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F306+F304, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>0.89 µl
</TD><TD>F306
</TD></TR><TR><TD>1 µl
</TD><TD>F304
</TD></TR><TR><TD>15.11 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F308+F303, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>3.48 µl
</TD><TD>F308
</TD></TR><TR><TD>1 µl
</TD><TD>F303
</TD></TR><TR><TD>12.52 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F300+F309, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.48 µl
</TD><TD>F300
</TD></TR><TR><TD>4.29 µl
</TD><TD>F309
</TD></TR><TR><TD>10.23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for F276+F310, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>1.2 µl
</TD><TD>F276
</TD></TR><TR><TD>0.51 µl
</TD><TD>F310
</TD></TR><TR><TD>15.29 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were also prepared by substitute linearized insert DNA with ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> Ligation was performed at room temperature for 1 h and then were transformed in <I>E. coli</I> XL1-Blue.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_20th"><B>Friday, September 20th</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P799_and_P995_.28Katrin.2C_bitte_vervollst.C3.A4ndigen.29"> Preparative digestion and gelelectrophoresis of P799 and P995 (Katrin, bitte vervollständigen) </SPAN></H3><P><B>Investgator:</B> Katrin
</P></DIV></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_21th"><B>Saturday, September 21th</B></SPAN></H2><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_F308.2BF303.2C_F306.2BF304.2C_F300.2BF309_.28.21.29"> Picking of <I>E. coli</I> XL1-Blue transformed with F308+F303, F306+F304, F300+F309 (!) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P>'<I>Aim of the experiment</I>: Picking of <I>E. coli</I> XL1-Blue transformed with F308+F303, F306+F304, F300+F309 (!)
</P></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F312.2BF304.2C_F311.2BF310"> Ligation of F312+F304, F311+F310 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of F312+F304, F311+F310
</P></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_F312.2BF304.2C_F311.2BF310"> Transformation of <I>E. coli</I> XL1-Blue with F312+F304, F311+F310 </SPAN></H3><P><B>Investigator:</B> Jeff
</P></DIV></DIV><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F295.2BF294"> Ligation of F295+F294 </SPAN></H3><P><B>Investigator:</B> Jeff
</P></DIV><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_F295.2BF294.2C_P614.2C_P995.2C_P455.2FP456"> Transformation of <I>E. coli</I> XL1-Blue with F295+F294, P614, P995, P455/P456 </SPAN></H3><P><B>Investigator:</B> Jeff
</P></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Week_23">Week 23</SPAN></H1><DIV class="week" id="WWeek_23"><H2><SPAN class="mw-headline" id="Monday.2C_September_23rd"><B>Monday, September 23rd</B></SPAN></H2><DIV class="general"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Analytical_digest_of_P1023-P1028_.28SpyTag_SYFP2.29_with_NgoMVI_.26_PstI-HF"> Analytical digest of P1023-P1028 (SpyTag_SYFP2) with NgoMVI &amp; PstI-HF </SPAN></H3><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P102 - P1028 (SpyTag_SYFP2)
</P><P><B>Procedure:</B></P><P><B>8x Mastermix for digestion with NgoMVI &amp; PstI-HF</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>16 µl
</TD><TD>Cut Smart buffer
</TD></TR><TR><TD>4 µl
</TD><TD>NgoMVI
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>118 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> 2.5 µl Plasmid DNA added to 17.5 µl mastermix each (<B>total volume:</B> 20 µl)
</LI></UL><UL><LI> Analytical digestion was performed at 37 °C for 1 h.
</LI></UL><UL><LI> After incubation 2 µl of DNA loading buffer (10x) were added to the reaction batches and 10 µl were loaded on a 1% agarose gel.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE border="1"><TBODY><TR><TD>2-log DNA ladder
</TD><TD>F294
</TD><TD>P1023
</TD><TD>P1024
</TD><TD>P1025
</TD><TD>P1026
</TD><TD>P1027
</TD><TD>P1028
</TD></TR><TR><TD> -
</TD><TD> -
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>empty
</TD><TD>as expected
</TD><TD>as expected
</TD><TD>as expected
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P1026.2C_P1027_and_P1028_.28all_SpyTag_SYFP2.29"> Preparative digestion and gelelectrophoresis of P1026, P1027 and P1028 (all SpyTag_SYFP2)</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelextraction of P1026, P1027 and P1028 (all SpyTag_SYFP2)
</P><P><B>Procedure:</B></P><P><B>Reaction batches for P1026, P1027 and P1028:</B></P><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>20 µl
</TD><TD>Plasmid DNA
</TD></TR><TR><TD>4 µl
</TD><TD>CutSmart Buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>15.5 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batches were incubated at 37 °C for 3 h.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 90 V for 1.5 h in an 1% agarose gel.
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>Lane</B></TH><TD>Digestion of P1026 with XbaI &amp; AgeI
</TD><TD>2-log DNA ladder
</TD><TD>Digestion of P1027 with XbaI &amp; AgeI
</TD><TD>Digestion of P1028 with XbaI &amp; AgeI
</TD></TR><TR><TH><B>Results</B></TH><TD>as expected; lower band was cut out
</TD><TD> -
</TD><TD>as expected; lower band was cut out
</TD><TD>somewhat off
</TD></TR></TBODY></TABLE><UL><LI> Gel extraction was performed after preparative gelelectrophoresis with QIAquick Gel Extration Kit, QIAGEN.
</LI></UL><P><B>Labeling:</B></P><TABLE border="1"><TBODY><TR><TD>P1026 digest
</TD><TD>F313
</TD></TR><TR><TD>P1027 digest
</TD><TD>F314
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Ligation_of_F300_.28Strep-tag-II_in_pBAD-Kan.29.2B_F314_.28SpyTag_SYFP2.29"> Ligation of F300 (Strep-tag-II in pBAD-Kan)+ F314 (SpyTag_SYFP2)</SPAN></H3><P><B>Investigator:</B> Rosario
</P><P><B>Aim of the experiment:</B> Ligation of F300 (Strep-tag-II in pBAD-Kan)+ F314 (SpyTag_SYFP2)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F300+F314, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.48 µl
</TD><TD>F300
</TD></TR><TR><TD>2.29 µl
</TD><TD>F314
</TD></TR><TR><TD>12.23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared by substituting linearized insert DNA (F314) with ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> Ligation was performed at 16 °C over night and then were transformed in <I>E. coli</I> XL1-Blue and <I>E. coli</I> BL21, respectively.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_24th"><B>Tuesday, September 24th</B></SPAN></H2><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Picking_of_Ligations_F310.2BF311_.28His-tag_PhyB_36AAlinker_C-TEV-Protease_.2B_pBAD-KanR_Backbone.29_and_F312.2BF304_.28SYFP2_SpyTag_in_pSB1C3.2C_prefix_opened_.2B_Streptag.29"> Picking of Ligations F310+F311 (His-tag_PhyB_36AAlinker_C-TEV-Protease + pBAD-KanR Backbone) and F312+F304 (SYFP2_SpyTag in pSB1C3, prefix opened + Streptag) </SPAN></H3><P><B>Investigator:</B> Florian
</P><P><B>Aim of the experiment:</B> Picking of Ligations F310+F311 and F312+F304
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies of F310+F311, each into 4 ml LB medium with 8 µl kanamycin stock solution
</LI></UL><UL><LI> 4 colonies of F312+F304, each into 4 ml LB medium with 4 µl chloramphenicol stock solution
</LI></UL><UL><LI> incubation over night at 180 rpm at 37 °C
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Transformation_of_Ligation_F300_.28Strep-tag-II_in_pBAD-Kan.29_.2B_F314_.28SpyTag_SYFP2.29"> Transformation of Ligation F300 (Strep-tag-II in pBAD-Kan) + F314 (SpyTag_SYFP2) </SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue and <I>E. coli</I> BL21 cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of DNA were added to the tubes containing 200 µl of competent cells and gently mixed
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37 °C <B>(WRONG; heatblock was not at the right temperature)</B></LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated on new <B>kanamycin</B> plates.
</LI></UL></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Sequencing_of_P1013_.28Strep-tag_SYFP2_SpyTag.29_and_P1027_.28SpyTag_SYFP2.29"> Sequencing of P1013 (Strep-tag_SYFP2_SpyTag) and P1027 (SpyTag_SYFP2)</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Sequencing of P1013 (Strep-tag_SYFP2_SpyTag) reverse and P1027 (SpyTag_SYFP2) forward
</P><P><B>Procedure:</B> The DNA was prepared for sequencing according to the Eurofins SmartSeq Kit protocol (15 µl of plasmid DNA (50 - 100 ng) and 2 µl sequencing primer).
</P><P>The gene we sequenced received the following barcode:
</P><TABLE border="1"><TBODY><TR><TH> Label
</TH><TH> Barcode
</TH><TH> Primer
</TH></TR><TR><TD>P1013
</TD><TD>FR02303610
</TD><TD>04
</TD></TR><TR><TD>P1027
</TD><TD>FR02303609
</TD><TD>03
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="safety_mechanism"><DIV class="effectors"><H3><SPAN class="mw-headline" id="Repetition_of_Ligation_of_F300_.28Strep-tag-II_in_pBAD-Kan.29.2B_F314_.28SpyTag_SYFP2.29"> Repetition of Ligation of F300 (Strep-tag-II in pBAD-Kan)+ F314 (SpyTag_SYFP2)</SPAN></H3><P><B>Investigator:</B> Johanna
</P><P><B>Aim of the experiment:</B> Ligation of F300 (Strep-tag-II in pBAD-Kan)+ F314 (SpyTag_SYFP2)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for F300+F314, vector:insert = 1:3
</LI></UL><TABLE border="1"><TBODY><TR><TH><B>volume</B></TH><TH><B>reagent</B></TH></TR><TR><TD>2.48 µl
</TD><TD>F300
</TD></TR><TR><TD>2.29 µl
</TD><TD>F314
</TD></TR><TR><TD>12.23 µl
</TD><TD>ddH<SUB>2</SUB>O
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was also prepared by substituting linearized insert DNA (F314) with ddH<SUB>2</SUB>O.
</LI></UL><UL><LI> Ligation was performed at 16 °C over night and then were transformed in <I>E. coli</I> XL1-Blue and <I>E. coli</I> BL21, respectively.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_25th"><B>Wednesday, September 25th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_26th"><B>Thursday, September 26th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_2">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_27th"><B>Friday, September 27th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_3">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_28th"><B>Saturday, September 28th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_4">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_29th"><B>Sunday, September 29th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_5">Experiments (see result section)</SPAN></H3></DIV><H1><SPAN class="mw-headline" id="Week_24">Week 24</SPAN></H1><DIV class="week" id="WWeek_24"><H2><SPAN class="mw-headline" id="Monday.2C_September_30th"><B>Monday, September 30th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_6">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_1st"><B>Tuesday, September 1st</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_7">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_2nd"><B>Wednesday, September 2nd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_8">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_3rd"><B>Thursday, September 3rd</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="Experiments_.28see_result_section.29_9">Experiments (see result section)</SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_4th"><B>Friday, September 4th</B></SPAN></H2><DIV class="general"><H3><SPAN class="mw-headline" id="FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE_FREEZE">FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE FREEZE</SPAN></H3></DIV></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">http://2013.igem.org/Team:TU-Munich/Notebook/Labjournal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Notebook/Labjournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Notebook/Labjournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Labjournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Labjournal&amp;oldid=324977" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>