"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Munich/Team/Attributions
From 2013.igem.org
Home
Project
Phytoremediation
Physcomitrella
Localization
BioDegradation
BioAccumulation
Kill Switch
Safety
Results
BioBricks
Effector Studies
Moss Methods
Localization
PhyscoFilter
Kill Switch
Implementation
AutoAnnotator
Entrepreneurship
Summary
Modeling
Protein Predictions
Kill Switch
Enzyme Kinetics
Filter Model
Team
Members
Sponsors
Collaborations
Pictures
Timeline
Attributions
Judging
Notebook
Labjournal
Methods
Outreach
Expert Counsel
GMO release
Publicity
Educational Kit
Tutorials
iGEM Team TU-Munich 2013
Experimental measurements
All experiments and measurements were conducted by student members of the iGEM Team TU-Munich 2013 . The team could build on the knowledge of Jeffery Truong, Ingmar Polte and Katrin Fischer who had already participated in last year's competition and already knew the most important techniques and the laboratory. They implemented the lab management from last year from the first moment on, such as our extensive Labjournal .  For the first usage of instruments we obtained an introduction into the lab techniques by an instructor. Measurement itself and the evaluation of the obtained data were done by us.
LC-MS Measurements
The LC-MS measurements were performed to confirm the successful degradation of pollutants by effector proteins which were produced recombinantly or which were expressed by transgenic moss plants. Here we contacted Prof. Dr. Thomas Hofmann, who already helped us in our last year's iGEM project with the detection of caffeine and xanthohumole. The responsible team member for the LC-MS was in both years Ingmar Polte who performed the degradation experiments, established the contact to Prof. Hofmann and brought the samples to the Chair for Molecular Sensory . At the chair, the student members Ingmar Polte or Andreas-David Brunner contacted the operator of the mass spectrometer (Mr. Friederich or Dr. Stark) with whose they did the sample preparation, the actual measurements and the data evaluation.
ESI-TOF Measurements
In order to confirm the correctness of recombinant proteins they were analyzed in the ESI-TOF mass spectrometer, which is present at our hosting laboratory. For this purpose a student team member did the sample preparation, calculated the theoretical mass using the AutoAnnotator and brought the samples to Andreas Reichert who is a doctoral student at the chair and is responsible for the ESI-TOF mass spectrometer. He measured the samples together with a student team member and showed us the deconvolution of the primary data.
Fluorescense Microscopy
All fluorescense microscopy experiments were performed at the Chair for Plant Developmental Biology (Prof. Dr. Schneitz) as plants (and especially P. patens) exhibit a strong autofluorescence caused by the photosystem. Therefore an advanced microscope with appropriate filters is absolutely necessary which we could use at the Chair for Plant Developmental Biology. The experiments were performed by our student member Dong-Jiunn Jeffery Truong who has worked during his bachelor thesis with fluorescense microscopy over several weeks and was therefore an essential experimentator for this part of the project. An introduction to the microscopes was given by Dr. Prasad Vaddepalli.
Transformation of Physcomitrella patens
The transformation of Physcomitrella patens was one of the main concerns when we thought about working with Physcomitrella as it might cause some trouble which can only be handled by using the appropriate equipment and the advises of an experienced researcher. Therefore we contacted Prof. Dr. Reski (Freiburg University, 350 km distance) in May and could win him as an advisor for our team. After traveling several times to Freiburg we could perform the transformations in his lab. For this purpose we brought all buffers and the sterile linearized DNA from Munich and performed the transformation under the instructions of Dr. Gertrud Wiedemann. All steps of the transformation were conducted by student team members as it can be seen in our Transformation Results section.
Outreach
The introduction video for our team created during this competition, was designed, produced and cut by the student team member Katrin Fischer.
The group photos of our team were created by the press office of the TUM.
Laboratory of Prof. Dr. Skerra
The research group at the Chair of Biological Chemistry at TUM works in the field of biochemical protein engineering and design with a focus on therapeutic proteins and their application. The three main fields of research are (1) the development of anticalins which is an alternative binding scaffold and which are a promising alternative for conventional antibodies (2) the extension of the plasma half-life of therapeutic proteins using a poly-peptide polymer and (3) the site-specific conjugation of therapeutic proteins. Thus the focus of our hosting laboratory is biomedical engineering whereas our iGEM topic shows no intersection with this topic.
The single negligible exception is that we used a higher engineered anticalin (<partinfo>K1159003</partinfo>) to bind fluorescein in our BioAccumulation subproject, compared to the conventional BioBrick (<partinfo>BBa_K157004</partinfo>). This higher engineered version has three additional amino acid exchanges and exhibits a 75-fold higher affinity for its binding partner (fluorescein). It was not essential to have this higher engineered version for our project but we wanted to supply the registry with this improved part.
Beside this plasmid, we also obtained a plasmid for the TEV protease from the chair which we have used to generate the Split-TEV protease ( BBa_K1159100 , BBa_K1159101 and BBa_K1159102 ).
Prof. Skerra kindly provided us with space in his laboratory, and generously advanced us money to pay for team registration, travel expenses and laboratory resources. Moreover he participated approximately one a month in our team meetings and advised us on our project.
Technical University Munich
As we had to cover several different scientific aspects during our project we contacted several professors from our university to get their opinion on our plans, obtain reagents or access to measuring instruments such as mass spectrometers or microscopes.
Laboratory of Prof. Dr. Helmreich
Sanitary Environmental Engineering is a horizontal discipline comprised of civil engineering, process engineering and chemistry/biology. Research and teaching include the fields of water supply, sewage and rain water treatment, water quality and the modeling of aquatic systems.
In our interview with Prof. Helmreich, she explained us several characterization techniques to assess water quality. Furthermore she kindly provided us with water samples from sewage treatment plants.
Laboratory of Prof. Dr. Thomas Hofmann
The chair of Food Chemistry and Molecular Sensory Science at the Technische Universität München (TUM) comprises an international research team with a focus on chemosensory active biomolecules (sensomics), metabolomics & nutritional biomarker discovery, as well as stable isotope labeling studies.
The chair gave us great support in form of the detection of degraded pollutans via LCMS.
Laboratory of Prof. Dr. Langosch
The Chair for Chemistry of Biopolymers focuses on the structural biochemistry of integral membrane proteins. Core themes are molecular interactions between membrane proteins, structural dynamics of membrane bound protein helices, membrane protein/lipid interactions and structure/function relationships of integral membrane protein complexes.
During the planning phase of our project we asked Prof. Langosch for advice concerning the design of the transmembrane domain of our constructs.
Laboratory of Prof. Dr. Rost
Prof. Rost works in the field of bioinformatics and computer-based biology, focusing on the prediction of structure and function of proteins and genes. His team's specialty is the use of artificial intelligence and machine learning algorithms to predict structure and function of proteins.
After developing an early version of our AutoAnnotator , we presented our work to Prof. Rost and his group. He gave us some very helpful advice on features we could add and provided us with access to his PredictProtein server. A special "thank you" goes to Manfred Roos for tailoring the access point of the server to our needs.
Laboratory of Prof. Dr. Schneitz
The Chair for Plant Developmental Biology is interested in the genetic and molecular basis of the regulatory pathways controlling organ development and tissue morphogenesis in plants.
The chair gave us great access to their microscopes and excellent support in how to use them. Many thanks for that, especially to Prasad Vaddepalli for his introduction into fluorescence microscopy.
Laboratory of Prof. Dr. Schwechheimer
The Chair for Systems Biology of Plants investigates a range of queries concerning the ubiquitin proteasome system of plants applying a combination of genetics, molecular biology and cell biology with both genomic and proteomic approaches.
Together with Prof. Schwechheimer we discussed several different signal transduction pathways that could be utilised for our kill-switch . Moreover he kindly granted us access to a fluorescence microscope.
Laboratory of Prof. Dr. Scherer
The Chair for microbial ecology conducts basic research in the field of molecular genetics and ecology of pathogenic microorganisms. Projects include the microbial characterization of food  and microbial diagnostics and consulting in the food industry for prevention and clearance of contaminations.
The team kindly provided us with a culture of Micrococcus luteus for the Kirby-Bauer assay and also helped out with a tube of restriction enzyme during an unexpected shortage.
Other Universities
The team also obtained help from other universities in order to complete the project. As nobody at our university works with Physcomitrella patens which the team chose to work with we had to find an advisor who could support us with the transformation procedure which is not really simple for plants. Additionally researches from other universities helped us by providing plasmids.
Laboratory of Prof. Dr. Reski at Freiburg University (Germany)
The team at the Chair for Plant Biotechnology is working on gene expression in the bryophyte model plant Physcomitrella patens (Hedw.) B.S.G. at different levels in correlation with phenotype analysis and additionally employs comparative genomic approaches.
The Reski team kindly supported us by supervising our moss transformations at their lab and by allowing us to use some of their equipment and materials for the process. Our special thanks go to Dr. Gertrud Wiedemann .
Laboratory of Prof. Dr. Fussenegger at ETH Zurich (Switzerland)
The research group at the Chair for Biotechnology and Bioengineering is implementing a progress in basic research to achieve generic and prototypic advances in human therapy by focusing on mammalian cells and capitalizing on an integrated interdisciplinary systems approach. Their current research initiatives include several programs interfacing with biopharmaceutical manufacturing, gene therapy and tissue engineering.
They kindly provided us with the pSH21 plasmid which was used as template for the Polioviral Internal Ribosome Entry Site ( BBa_K1159300 ).
Laboratory of Dr. G.D. Wright  at McMaster University (Canada)
The Wright Lab is trying to understand fundamental aspects of how antibiotics work, their sources and how bacteria become resistant to them.
They kindly provided us with the plasmids pDEST14_ereA and pDEST14_ereB which was used as a template for the Erythromycin Esterase Type II (<partinfo>BBa_K1159000</partinfo>).
Laboratory of Prof. Dr. Arndt  at Potsdam University (Germany)
The team at the Chair for Molecular Biotechnology investigates the factors that mediate interactions in coiled-coil proteins in order to target coiled-coil domains of proteins e.g. involved in tumorigenesis, tumor proliferation and metastasis.
Sven Hagen who had participated several times in iGEM in the past kindly provided us with the pBad-mVenus (RFC 25) expression plasmid. This plasmid was used for the production of recombinant protein in E. coli.
Webdesign
The webdesign was entirely done by Florian Albrecht who has programmed different websites before. He also wrote the webdesign tutorial we put online to explain other teams how to use the code, developed by our team. The graphic elements used on the website were created by several different student members (such as Ingmar Polte, Katrin Fischer, Jeffery Truong, Rosario Ciccone and others) using the open source vector software Inkscape. The only component that was bought are the two direction signs used in the header (obtained from http://vector-images.de/clipart/clp217632 ).
normalize.css
Normalize.css was used to reset the wiki style (version used: 2.1.3).
jQuery
jQuery is a powerful JavaScript extension used by many Webdesigners (version used: 1.10.2).
jQuery UI
jQuery UI is a GUI extension of jQuery and was used for the datepicker on the Arduino data page (version used: 1.10.3).
History.js
History.js was used to ensure compatibility with HTML4 browsers, that do not support history.pushState().
Slimbox 2
Slimbox 2 was used as a picture viewer (version used: 2.05). The code had to be adapted to work with the wiki and bxSlider and we added a feature that fits the viewer to the browser size.
bxSlider
bxSlider is a JavaScript content slider and powers the slideshows and picture galleries on our wiki (version used: 4.1.1). The code was modified so all the images are scaled to the same height in gallery mode.
NVD3
NVD3 is a JavaScript chart library and was used for the interactive chart of the Arduino data (version used: 1.1.13).
Software - The AutoAnnotator
The software tool we have written to facilitate and improve the annotation of protein coding BioBricks (the AutoAnnotator ) was developed and programmed by the student member Christopher Wolf . He developed this idea himself to import, translate sequences and to compute important parameters. After this was completed he contacted the bioinformatics group of Prof. Rost and presented his idea and the program in a group seminar. In the seminar, ideas for further improvements were generated. The group advised Chrostopher to implement a bioinformatic module that does alignments to several databases and bioinformatic servers. The research group of Prof. Rost has a server that concentrates bioinformatic information and they enabled Christopher to use this tool. He implemented these information into the AutoAnnotator himself. It was also Chrostopher Wolf who has written the RFC 96 to describe the usage of the AutoAnnotator and who created the introduction video .
For further information please see our Software page .
James Padolsey
We used James Padolsey´s jQuery extension for Cross-domain AJAX requests in the AutoAnnotator.
Flot.js
Flot.js (see flotchart.org ) was used to draw the graphs in the AutoAnnotator and in the Filter Calculator.
Flot-AxisLabels
Flot-AxisLabels is an extension of Flot.js, adding axis labels to Flot.js.
Excanvas
Excanvas.js extends the <canvas>-tag to IE 8.0 and earlier, allowing the use of AutoAnnotator (in particular the plots) in these browsers.
Paul Johnston
We used the MD5 generator written by Paul Johnston to translate the amino acid sequences into MD5-hashs, which were used to obtain the predictions from the database of PredictProtein.org
AutoAnnotator:
or go to the AutoAnnotator page.
Follow us:
Address:
iGEM Team TU-Munich
Emil-Erlenmeyer-Forum 5
85354 Freising, Germany
Email: igem@wzw.tum.de
Phone: +49 8161 71-4351
Retrieved from " http://2013.igem.org/Team:TU-Munich/Team/Attributions "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
