<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Results_Software"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Results/Software">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Results/Software&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Results/Software&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Results/Software&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Results/Software" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Results/Software</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="The_AutoAnnotator">The AutoAnnotator</SPAN></H2><H3><SPAN class="mw-headline" id="Introduction_to_the_Idea_behind_our_AutoAnnotator">Introduction to the Idea behind our AutoAnnotator</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1:</B> Work flow of the AutoAnnotator</DIV></DIV></DIV><P>The Parts Registry contains a wide range of interesting protein coding BioBricks, but there is no standardized way of presenting basic information about them. This is a real pity, because after the identification of the open reading frame a multitude of parameters of the protein can be computed automatically, e.g. its molecular mass, theoretical pI or codon quality for different organisms. We have developed a tool which is able to identify the open reading frame of a BioBrick, analyze the sequence and the encoded protein and export the results in a format that can easily be integrated into the part description (and the team wikis) as a single table.
</P><P>This enables users to see basic information about the BioBrick at a quick glance in a standardized table, saving time, facilitating the comparison of BioBricks and improving the annotation of the parts. The AutoAnnotator can also be used for planning new Bricks, by quickly computing the relevant parameters in a single place rather than having to go to several different websites and gather the information together.
</P><P>Try it out: <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" class="external text" rel="nofollow"><B>The AutoAnnotator</B></A>!
</P><H3><SPAN class="mw-headline" id="How_to_use_the_AutoAnnotator">How to use the AutoAnnotator</SPAN></H3><OL><LI>Enter the BioBrick number or the DNA sequence into the input window! For new parts the Registry database may have not been updated yet, so the AutoAnnotator can not get the sequence that way. In this case please enter the DNA sequence manually.
</LI><LI>Click the button!
</LI><LI>Check, that the automatically determined open reading frame (underlined in the DNA sequence) shown at the top of the table is correct!
</LI><LI>Enjoy the beauty of the table and the information in it!
</LI><LI>Copy the text in the box below the table into your part description or wiki to add the table to your site!
</LI></OL><P>Please note, that there will be updates to the AutoAnnotator and that the predictions and alignments will improve as the data bases and the number of known structures grow, so it may be worth re-applying the AutoAnnotator, when looking at a BioBrick with an older version of the Annotator. Should the contents have changed, please replace the table.
</P><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><P>The AutoAnnotator is a web-based tool compiling information about encoded proteins from the DNA sequence. It performs the following steps:
</P><OL><LI><A href="#Import_of_BioBrick_Sequences"><B>Input</B></A>: When entering a BioBrick number, the AutoAnnotator imports the nucleotide sequence directly from the Registry data base.Alternatively a nucleotide sequence can be entered directly. This has to be used for new BioBricks, which aren't in the Registry yet, but can also be helpful for planning new BioBricks.
</LI><LI><A href="#Determination_of_the_Open_Reading_Frame"><B>Finding the Open Reading Frame</B></A>: In order to determine the Open Reading Frame (ORF) the algorithm first tries to determine what BioBrick assembly standard the BioBrick is in. If necessary (e.g. for an <A href="http://parts.igem.org/Assembly_standard_25" class="external text" rel="nofollow">RFC 25</A> Brick), nucleotides are added to the sequence. Then the ORF is determined by taking the first start codon and the first matching in-frame stop codon.
</LI><LI><A href="#Computation_of_Parameters"><B>Computation of Parameters</B></A>: From the nucleotide sequence the codon usage for different organisms, i.e. whether the preferred codons are used or not (which contributes to the level of gene expression), is computed directly. Then after translating the DNA sequence into its amino acid sequence, several parameters of the encoded protein are determined, namely: the amino acid composition, the number of charged amino acids, the atomic composition, the molecular mass, the isoelectric point (pI) and the extinction coefficient of the protein. Also moving averages of the hydrophobicity and charge of the residues are calculated. For more information on each of these see <A href="#Computation_of_Parameters">below</A>. Additionally the sequence is also compared to a <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList" class="external text" rel="nofollow">list of sequence features</A>, such as binding sites or cleavage sites.
</LI><LI><A href="#Alignments_and_Predictions"><B>Alignments and Predictions</B></A>: The amino acid sequence is also sent to the servers of <A href="http://predictprotein.org" class="external text" rel="nofollow">PredictProtein.org</A>, a site created and operated by the <A href="http://rostlab.org" class="external text" rel="nofollow">Rost group</A>, who kindly granted us access to their online resources. The servers provide alignments for the entered protein as well as predictions of the secondary structure, the solvent accessibility, the sub-cellular localization, the gene ontology and of transmembrane regions and disulfide bridges. Proteins, which are not among the over 30 million stored there, will be calculated and added, and should be ready in a matter of hours.
</LI><LI><A href="#Export_of_the_Computed_Parameters"><B>Presentation of the Computed Data</B></A>: The data is then put together into a concise, structured HTML table and displayed to the user. Sequence features and predictions are additionally shown in an optional plot as part of the table. Furthermore the code producing the table is displayed underneath it and so by a single copy&amp;paste the table can be integrated into any wiki, part description or other website.
</LI></OL><H3><SPAN class="mw-headline" id="Import_of_BioBrick_Sequences">Import of BioBrick Sequences</SPAN></H3><P>Upon entering a BioBrick number the AutoAnnotator uses the <A href="http://parts.igem.org/DAS_-_Distributed_Annotation_System" class="external text" rel="nofollow">Registry DAS</A> interface to load the nucleotide sequence from the data base of the Registry. To allow this cross-domain information request, which is blocked by most browsers for security reasons, an <A href="http://james.padolsey.com/javascript/cross-domain-requests-with-jquery/" class="external text" rel="nofollow">extension</A> to the .ajax() method in jQuery written by <A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions#James_Padolsey" class="external text" rel="nofollow">James Padolsey</A> was used. This uses the <A href="http://developer.yahoo.com/yql/" class="external text" rel="nofollow">YQL</A> (Yahoo! Query Language), which is a service by Yahoo!, to redirect the request via their servers, in this way solving the security issues and allowing the Annotator to read the information from the Registry.
</P><P>For newer BioBricks the Registry database (i.e. the DAS interface) may not have updated the sequence yet, and so returns an empty sequence. If this is the case the sequence of the BioBrick must be fed manually to the AutoAnnotator by pasting the DNA sequence into its input window. In the resulting output two positions in the code will be clearly marked by </P><PRE>&lt;!------------------------Enter BioBrick number here------------------------&gt;</PRE><P> where the user should enter the BioBrick number (e.g. K801060 or E0040) in the copied table to activate the link to the part description.</P><H3><SPAN class="mw-headline" id="Determination_of_the_Open_Reading_Frame">Determination of the Open Reading Frame</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption"><B>Figure 2:</B> The logic of the algorithm finding the Open Reading Frame and Assembly Standard</DIV></DIV></DIV><P>The first step is to work out the Assembly Standard of the BioBrick, since parts of the coding sequence may be in the pre- or suffix. As of version 1.0 the most common standards <A href="http://parts.igem.org/Assembly_standard_10" class="external text" rel="nofollow">RFC 10</A> and <A href="http://parts.igem.org/Assembly_standard_25" class="external text" rel="nofollow">RFC 25</A> are supported. Then the first start codon ATG is used and the first corresponding in-frame stop codon determined. These are taken to be the open reading frame.
</P><P>The complete algorithm to find the Assembly Standard and the Open Reading Frame is illustrated in figure 2 on the right. After finding the Open Reading Frame (ORF) its length is compared to the length of the DNA sequence. If the ORF is less than three quarters (75%) of the DNA sequence, the user is warned that something may be wrong, because usually BioBricks should not have so many non-encoding nucleotides in them.
</P><P>Should for some reason the automatically determined ORF be incorrect (which should always be checked by the user), the complete protein encoding sequence - starting with the <I>ATG</I> start codon and ending with a stop codon - should be entered manually. In this case the AutoAnnotator will always pick the correct frame.
</P><H3><SPAN class="mw-headline" id="Recognizing_Sequence_Features">Recognizing Sequence Features</SPAN></H3><P>There are several useful building blocks, which are frequently integrated into BioBricks, such as different tags for analytical purposes or cleavage and docking sites for protein interaction. We have put together a list of such common sequence features and the AutoAnnotator automatically looks for these, lists the appearing features and highlights them in the amino sequence. For the currently supported features please see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList" class="external text" rel="nofollow">Feature List</A>. If you have any suggestions for other interesting features, please get in touch and we will add them.
</P><H3><SPAN class="mw-headline" id="Computation_of_Parameters">Computation of Parameters</SPAN></H3><H4><SPAN class="mw-headline" id="Amino_acid_counting.2C_atomic_composition_and_molecular_weight">Amino acid counting, atomic composition and molecular weight</SPAN></H4><P>The amino acid counting section is straight forward from the amino acid sequence. Then with the amino acid composition the atomic composition can easily be calculated by using the atomic composition of each amino acid (e.g. given <A href="http://www.matrixscience.com/help/aa_help.html" class="external text" rel="nofollow">here</A>) and adding a water molecule (for the ends). Similarly the molecular weight is obtained by adding the individual weights (using the <A href="http://web.expasy.org/findmod/findmod_masses.html#AA" class="external text" rel="nofollow">average isotopic masses</A>) and again adding the molecular weight of a water molecule.
</P><H4><SPAN class="mw-headline" id="Theoretical_pI">Theoretical pI</SPAN></H4><P>The theoretical pI is the isoelectric point of the protein ignoring effects due to folding, which can't be computed properly. By definition the isoelectric point of a protein is the pH-value where the overall charge of the protein is zero, so we need to relate the pH value to the charges of the amino acids. For acid groups HA this is done by the Henderson-Hasselbalch equation, where pK<SUB>a</SUB> is the negative logarithm of the acid dissociation constant:
</P><P>We can rearrange this to get the fraction of molecules, which are deprotonised and so negatively charged:
</P><P>Analogously by regarding HB<SUP>+</SUP> as an acid, where B is a base, we can obtain the fraction of positively charged molecules:
</P><P>These fractions can also be regarded as a &quot;fractional charge&quot;, because they give the average charge over all molecules of this type. So by adding up the fractional charge of each amino acid (those with non-basic and non-acidic residues contribute no charge) and those for the N- and C-terminal groups we can determine the charge of the protein at a specific pH. The dissociation constants were taken from [<A href="http://www.ncbi.nlm.nih.gov/pubmed/8125050" class="external text" rel="nofollow">Bjellqvist et al., 1993</A>] &amp; [<A href="http://www.ncbi.nlm.nih.gov/pubmed/8055880" class="external text" rel="nofollow">Bjellqvist et al., 1994</A>], which are also those used by the <A href="http://web.expasy.org/protparam/" class="external text" rel="nofollow">ExPASy ProtParam Tool</A> and are also shown in the tables 1 and 2 below:
</P><TABLE class="left"><CAPTION><B>Table 1:</B> pK<SUB>a</SUB> values for positily charged residues
</CAPTION><TBODY><TR><TH colspan="2">Positively charged groups
</TH></TR><TR><TH>Group
</TH><TH>pK<SUB>a</SUB></TH></TR><TR><TD>Lysine residue
</TD><TD align="right">10.00
</TD></TR><TR><TD>Arginine residue
</TD><TD align="right">12.00
</TD></TR><TR><TD>Histidine
</TD><TD align="right">5.98
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> (unless specified otherwise)
</TD><TD align="right">7.50
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Alanine
</TD><TD align="right">7.59
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Methionine
</TD><TD align="right">7.00
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Serine
</TD><TD align="right">6.93
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Proline
</TD><TD align="right">8.36
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Threonine
</TD><TD align="right">6.82
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Valine
</TD><TD align="right">7.44
</TD></TR><TR><TD>N-terminal -NH<SUB>2</SUB> on Glutamic acid
</TD><TD align="right">7.70
</TD></TR></TBODY></TABLE><TABLE class="right"><CAPTION><B>Table 2:</B> pK<SUB>a</SUB> values for negatively charged residues
</CAPTION><TBODY><TR><TH colspan="2">Negatively charged groups
</TH></TR><TR><TH>Group
</TH><TH>pK<SUB>a</SUB></TH></TR><TR><TD>C-terminal -COOH
</TD><TD align="right">3.55
</TD></TR><TR><TD>Aspartic acid residue
</TD><TD align="right">4.05
</TD></TR><TR><TD>Glutamic acid residue
</TD><TD align="right">4.45
</TD></TR><TR><TD>Cysteine residue
</TD><TD align="right">9.00
</TD></TR><TR><TD>Tyrosine residue
</TD><TD align="right">10.00
</TD></TR></TBODY></TABLE><P>Now all that remains to be done is to find the pH such that the total charge is zero. This is most easily done by the bisection method: Start with pH=7.0 and determine the charge there. If it is positive, we know the pI must be greater than 7.0 and so we only consider that interval. If it is negative, we continue with the lower half of the pH range. In the subinterval we again evaluate the charge at its middle and choose a subinterval accordingly. By repeating this algorithm we halve the remaining range of pH values on every recursion and can determine the theoretical isoelectric point upto our required precision by continuing until the remaining range is smaller than that precision. However it has to be noted, that the pK<SUB>a</SUB> values are only estimations, which depend on the experimental procedure (so you will find many different values in the literature), and that modifications to the protein and the formation of disulfide bridges affect the isoelectric point significantly, so it doesn't make sense to choose a precision of less than 0.01.
</P><H4><SPAN class="mw-headline" id="Extinction_coefficient_at_280_nm">Extinction coefficient at 280 nm</SPAN></H4><P>The calculation of the extinction coefficient of a protein at 280 nm from its amino acid composition is straight-forward [<A href="http://www.ncbi.nlm.nih.gov/pubmed/2610349" class="external text" rel="nofollow">Gill and von Hippel, 1989</A>]. The only residues absorbing at this wavelength are those of Tyrosine, Tryptophan and Cystine (which consists of two Cysteines forming a disulfide bridge).
Then the extinction coefficient is given by
</P><P>where Numb(amino acid) is the number of appearances of that amino acid in the protein. Since the number of formed disulfide bridges is impossible to calculate, two values are calculated: One under the assumption that all Cysteines are reduced, i.e. that there are no disulfide bridges, the other assuming that every Cysteine is oxidized and hence part of a disulfide bridge.
</P><H4><SPAN class="mw-headline" id="Codon_Adaptation_Index_.28CAI.29">Codon Adaptation Index (CAI)</SPAN></H4><P>The Codon Adaptation Index (CAI) measures the &quot;distance&quot; in codon usage between the gene in question and a set of highly expressed reference genes, giving an indicator of the expression rate of the gene in question. The CAI was first introduced by [<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC340524/" class="external text" rel="nofollow">Sharp and Li, 1987</A>] and is the most widespread measure of codon usage bias.
</P><DIV>For its calculation we first need to compute a &quot;weight&quot; w for each codon. This weight is defined as , where f<SUB>j</SUB> is the frequency (in the reference set) with which synonymous codon j represents the corresponding amino acid and i runs through all the codons representing this amino acid. The codon adaptation index is then found by taking the geometric mean of the weights of the codons in the sequence, i.e.  where N is the total number of codons in the sequence excluding non-synonymous codons (i.e. the codons where the corresponding amino acid is only encoded by this codon).</DIV><P>We chose to include the bacteria <I>E. coli</I> and <I>B. subtilis</I>, the yeast <I>S. cerevisiae</I>, because these are the organisms used most frequently in iGEM. Further we added the codon usage of Mammals (in particular of <I>Mus musculus</I>), of the model plant <I>A. thaliana</I> and of the moss <A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella" class="external text" rel="nofollow"><I>P. patens</I></A>, which we introduced to iGEM as a chassis this year, because these offer very interesting applications and more options. The codon usages for the reference sets of these organisms were obtained from the <A href="http://www.kazusa.or.jp/codon/" class="external text" rel="nofollow">Codon Usage Database</A> [<A href="http://www.ncbi.nlm.nih.gov/pubmed/10592250" class="external text" rel="nofollow">Nakamura et al., 2000</A>].
</P><H4><SPAN class="mw-headline" id="Average_hydrophobicity_and_charge">Average hydrophobicity and charge</SPAN></H4><P>For each stretch of five amino acids in the sequence the average hydrophobicity (based on the Kyte-Doolittle hydrophobicity index [<A href="http://www.sciencedirect.com/science/article/pii/0022283682905150" class="external text" rel="nofollow">Kyte and Doolittle, 1982</A>]) and the average charge (following <A href="http://emboss.sourceforge.net/apps/release/6.5/emboss/apps/charge.html" class="external text" rel="nofollow">EMBOSS: charge</A>) of the residues are calculated and plotted in the graph. These two characteristics of the sequence can be very useful to determine transmembrane regions (~20 hydrophobic residues embedded into the membrane followed by a stop-transfer sequence of a few positively charged acids) and other properties, such as the before mentioned stop-transfer sequences.
</P><H3><SPAN class="mw-headline" id="Alignments_and_Predictions">Alignments and Predictions</SPAN></H3><P>This information is provided to us by the <A href="http://rostlab.org" class="external text" rel="nofollow">Rost group</A>, who kindly granted us access to the servers and data bases of their prediction site <A href="http://predictprotein.org" class="external text" rel="nofollow">PredictProtein.org</A> (see <A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions#Laboratory_of_Prof._Dr._Rost" class="external text" rel="nofollow">Attributions</A>). The sequence is aligned [<A href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="external text" rel="nofollow">Altschul et al., 1997</A>] against the <A href="http://web.expasy.org/docs/swiss-prot_guideline.html" class="external text" rel="nofollow">Swiss-Prot</A> (manual annotation and review) and <A href="http://web.expasy.org/docs/userman.html#what_is_trembl" class="external text" rel="nofollow">TrEMBL</A> (automatic annotation, extensive) protein databases, as well as against the <A href="http://www.rcsb.org/pdb/static.do?p=general_information/about_pdb/index.html" class="external text" rel="nofollow">Protein Data Bank (PDB)</A> for 3D structures. For each of these the two alignments with the highest identity and those with an identity above 97% are given together with the number of amino acids which were aligned. So for example for a fusion protein it is perfectly possible to get two completely different alignments with 100% identity on different subparts of the amino acid sequence.
</P><P>The server also produces various predictions. For more information about how these are obtained, please see the corresponding papers by the <A href="http://rostlab.org" class="external text" rel="nofollow">Rost group</A>:
</P><UL><LI>for secondary structure and solvent accessibility: [<A href="http://www.ncbi.nlm.nih.gov/pubmed/15215403" class="external text" rel="nofollow">Rost et al., 2004</A>]
</LI><LI>for transmembrane helices: [<A href="http://www.ncbi.nlm.nih.gov/pubmed/8844859" class="external text" rel="nofollow">Rost et al., 1996</A>]
</LI><LI>for disulfide bridges: [<A href="http://nar.oxfordjournals.org/content/34/suppl_2/W177.full" class="external text" rel="nofollow">Frasconi et al., 2006</A>]
</LI><LI>for sub-cellular localization: [<A href="http://www.ncbi.nlm.nih.gov/pubmed/22962467" class="external text" rel="nofollow">Rost et al., 2012</A>]
</LI><LI>for gene ontology: [<A href="http://www.biomedcentral.com/1471-2105/14/S3/S7" class="external text" rel="nofollow">Rost et al., 2013</A>]
</LI></UL><P>These results are stored on the server for more than 30 million amino acid sequences and are instantly available for these. If however a sequence is entered, which has not been calculated yet, the computations are initialized and should be ready in a few hours. In this case the user will be informed and told to return and rerun the AutoAnnotator later.
</P><H3><SPAN class="mw-headline" id="Export_of_the_Computed_Parameters">Export of the Computed Parameters</SPAN></H3><P>The results are then combined in a standardized HTML-Table, which is presented to the user. Additionally the HTML code of the table is given, allowing for a quick and easy copy&amp;paste into any wiki page or part description. Here is an example of the produced table with the graphical plot section hidden, but you can open it by pressing the &quot;Show&quot; button in the table. The appearing plot contains the average hydrophobicity and charge, as well as the structural predictions. These can be toggle on and off by using the check-boxes.
</P><P><TABLE id="AutoAnnotator"><TBODY><TR><TH id="AutoAnnotatorHeader" colspan="2">Protein data table for BioBrick <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K801060">BBa_K801060</A> automatically created by the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">BioBrick-AutoAnnotator</A> version 1.0</TH></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Nucleotide sequence</STRONG> in <STRONG>RFC 10</STRONG>: (underlined part encodes the protein)<SPAN class="AutoAnnotatorSequence"> GTACACA<U>ATGCGTCGT ... TTCGAAAAA</U>TAA</SPAN> <STRONG>ORF</STRONG> from nucleotide position 8 to 1705 (excluding stop-codon)</TD></TR><TR><TD class="AutoAnnotator1col" colspan="2"><STRONG>Amino acid sequence:</STRONG> (RFC 25 scars in shown in bold, other sequence features underlined; both given below)</TD></TR><TR><TD class="AutoAnnotatorSeqNum">1 101 201 301 401 501 </TD><TD class="AutoAnnotatorSeqSeq">MRRSANYQPSIWDHDFLQSLNSNYTDEAYKRRAEELRGKVKIAIKDVIEPLDQLDLIDNLQRLGLAHRFETEIRNILNNIYNNNKDYNWRKENLYATSLEFRLLRQHGYPVSQEVFNGFKDDQGGFICDDFKGILSLHEASYYSLEGESIMEEAWQFTSKHLKEVMISKNMEEDVFVAEQAKRALELPLHWKVPMLEARWFIHIYERREDKNHLLLELAKMEFNTLQAIYQEELKEISGWWKDTGLGEKLSFARNRLVASFLWSMGIAFEPQFAYCRRVLTISIALITVIDDIYDVYGTLDELEIFTDAVERWDINYALKHLPGYMKMCFLALYNFVNEFAYYVLKQQDFDLLLSIKNAWLGLIQAYLVEAKWYHSKYTPKLEEYLENGLVSITGPLIITISYLSGTNPIIKKELEFLESNPDIVHWSSKIFRLQDDLGTSSDEIQRGDVPKSIQCYMHETGASEEVARQHIKDMMRQMWKKVNAYTADKDSPLTGTTTEFLLNLVRMSHFMYLHGDGHGVQNQETIDVGFTLLFQPIPLEDKHMAFTASPGTKG<B>TG</B>A<U>WSHPQFEK</U>*</TD></TR></TBODY></TABLE><STRONG>Sequence features:</STRONG> (with their position in the amino acid sequence, see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software/FeatureList">list of supported features</A>)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorSeqFeat2a">RFC25 scar (shown in bold): </TD><TD class="AutoAnnotatorSeqFeat3">556 to 557</TD></TR><TR><TD class="AutoAnnotatorSeqFeat2a"><I>Strep</I>-tag II: </TD><TD class="AutoAnnotatorSeqFeat3">559 to 566</TD></TR></TBODY></TABLE><STRONG>Amino acid composition:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Ala (A)</TD><TD class="AutoAnnotatorInnerAmino">33 (5.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Arg (R)</TD><TD class="AutoAnnotatorInnerAmino">25 (4.4%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asn (N)</TD><TD class="AutoAnnotatorInnerAmino">27 (4.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Asp (D)</TD><TD class="AutoAnnotatorInnerAmino">34 (6.0%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Cys (C)</TD><TD class="AutoAnnotatorInnerAmino">4 (0.7%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gln (Q)</TD><TD class="AutoAnnotatorInnerAmino">24 (4.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Glu (E)</TD><TD class="AutoAnnotatorInnerAmino">48 (8.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Gly (G)</TD><TD class="AutoAnnotatorInnerAmino">29 (5.1%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">His (H)</TD><TD class="AutoAnnotatorInnerAmino">18 (3.2%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ile (I)</TD><TD class="AutoAnnotatorInnerAmino">39 (6.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Leu (L)</TD><TD class="AutoAnnotatorInnerAmino">64 (11.3%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Lys (K)</TD><TD class="AutoAnnotatorInnerAmino">36 (6.4%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Met (M)</TD><TD class="AutoAnnotatorInnerAmino">16 (2.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Phe (F)</TD><TD class="AutoAnnotatorInnerAmino">28 (4.9%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Pro (P)</TD><TD class="AutoAnnotatorInnerAmino">17 (3.0%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Ser (S)</TD><TD class="AutoAnnotatorInnerAmino">33 (5.8%)</TD></TR></TBODY></TABLE><TABLE class="AutoAnnotatorWithBorder"><TBODY><TR><TD class="AutoAnnotatorInnerAmino">Thr (T)</TD><TD class="AutoAnnotatorInnerAmino">26 (4.6%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Trp (W)</TD><TD class="AutoAnnotatorInnerAmino">14 (2.5%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Tyr (Y)</TD><TD class="AutoAnnotatorInnerAmino">27 (4.8%)</TD></TR><TR><TD class="AutoAnnotatorInnerAmino">Val (V)</TD><TD class="AutoAnnotatorInnerAmino">24 (4.2%)</TD></TR></TBODY></TABLE><STRONG>Amino acid counting</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAminoCountingInner2">Total number:</TD><TD class="AutoAnnotatorAminoCountingInner3">566</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Positively charged (Arg+Lys):</TD><TD class="AutoAnnotatorAminoCountingInner3">61 (10.8%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Negatively charged (Asp+Glu):</TD><TD class="AutoAnnotatorAminoCountingInner3">82 (14.5%)</TD></TR><TR><TD class="AutoAnnotatorAminoCountingInner2">Aromatic (Phe+His+Try+Tyr):</TD><TD class="AutoAnnotatorAminoCountingInner3">87 (15.4%)</TD></TR></TBODY></TABLE><STRONG>Biochemical parameters</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorBiochemParInner2">Atomic composition:</TD><TD class="AutoAnnotatorBiochemParInner3">C<SUB>3002</SUB>H<SUB>4586</SUB>N<SUB>778</SUB>O<SUB>868</SUB>S<SUB>20</SUB></TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Molecular mass [Da]:</TD><TD class="AutoAnnotatorBiochemParInner3">66105.3</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Theoretical pI:</TD><TD class="AutoAnnotatorBiochemParInner3">5.38</TD></TR><TR><TD class="AutoAnnotatorBiochemParInner2">Extinction coefficient at 280 nm [M<SUP>-1</SUP> cm<SUP>-1</SUP>]:</TD><TD class="AutoAnnotatorBiochemParInner3">117230 / 117480 (all Cys red/ox)</TD></TR></TBODY></TABLE><STRONG>Plot for hydrophobicity, charge, predicted secondary structure, solvent accessability, transmembrane helices and disulfid bridges</STRONG> </P><STRONG>Codon usage</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorCodonUsage2">Organism:</TD><TD class="AutoAnnotatorCodonUsage3"><I>E. coli</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>B. subtilis</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>S. cerevisiae</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>A. thaliana</I></TD><TD class="AutoAnnotatorCodonUsage3"><I>P. patens</I></TD><TD class="AutoAnnotatorCodonUsage3">Mammals</TD></TR><TR><TD class="AutoAnnotatorCodonUsage2">Codon quality (<A href="http://en.wikipedia.org/wiki/Codon_Adaptation_Index">CAI</A>):</TD><TD class="AutoAnnotatorCodonUsage3">good (0.71)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.75)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.69)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.78)</TD><TD class="AutoAnnotatorCodonUsage3">excellent (0.80)</TD><TD class="AutoAnnotatorCodonUsage3">good (0.68)</TD></TR></TBODY></TABLE><STRONG>Alignments</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorAlignment2">SwissProt:</TD><TD class="AutoAnnotatorAlignment3"><A href="http://www.uniprot.org/uniprot/Q8L5K3">Q8L5K3</A> (99% identity on 554 AAs), <A href="http://www.uniprot.org/uniprot/Q93X23">Q93X23</A> (54% identity on 536 AAs)</TD></TR><TR><TD class="AutoAnnotatorAlignment2">TrEML:</TD><TD class="AutoAnnotatorAlignment3"><A href="http://www.uniprot.org/uniprot/A7BG59">A7BG59</A> (99% identity on 554 AAs), <A href="http://www.uniprot.org/uniprot/Q8L5K1">Q8L5K1</A> (96% identity on 554 AAs)</TD></TR><TR><TD class="AutoAnnotatorAlignment2">PDB:</TD><TD class="AutoAnnotatorAlignment3"><A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1n1z">1n1z</A> (44% identity on 530 AAs), <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1n20">1n20</A> (44% identity on 525 AAs)</TD></TR></TBODY></TABLE><STRONG>Predictions</STRONG> (obtained from <A href="http://predictprotein.org">PredictProtein.org</A>)<STRONG>Subcellular Localization</STRONG> (reliability in brackets)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorLocalization2">Archaea:</TD><TD class="AutoAnnotatorLocalization3">cytosol (100%)</TD></TR><TR><TD class="AutoAnnotatorLocalization2">Bacteria:</TD><TD class="AutoAnnotatorLocalization3">cytosol (95%)</TD></TR><TR><TD class="AutoAnnotatorLocalization2">Eukarya:</TD><TD class="AutoAnnotatorLocalization3">nucleus (15%)</TD></TR></TBODY></TABLE><STRONG>Gene Ontology</STRONG> (reliability in brackets)<TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorGO2">Molecular Function Ontology:</TD><TD class="AutoAnnotatorGO3"><A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0050551">GO:0050551</A> (24%), <A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0034768">GO:0034768</A> (24%)</TD></TR><TR><TD class="AutoAnnotatorGO2">Biological Process Ontology:</TD><TD class="AutoAnnotatorGO3"><A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0071485">GO:0071485</A> (19%), <A href="http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0009611">GO:0009611</A> (18%)</TD></TR><TR><TD class="AutoAnnotatorGO3"> </TD></TR></TBODY></TABLE><STRONG>Predicted features:</STRONG><TABLE class="AutoAnnotatorNoBorder"><TBODY><TR><TD class="AutoAnnotatorPredFeat2a">Disulfid bridges:</TD><TD class="AutoAnnotatorPredFeat3"> - </TD></TR><TR><TD class="AutoAnnotatorPredFeat2a">Transmembrane helices:</TD><TD class="AutoAnnotatorPredFeat3"> - </TD></TR></TBODY></TABLE> The BioBrick-AutoAnnotator was created by <A href="https://2013.igem.org/Team:TU-Munich">TU-Munich 2013</A> iGEM team. For more information please see the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software">documentation</A>.If you have any questions, comments or suggestions, please leave us a <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">comment</A>.<H3><SPAN class="mw-headline" id="Programming_of_the_BioBrick-Autoannotator">Programming of the BioBrick-Autoannotator</SPAN></H3><P>The Annotator is a JavaScript program using the <A href="http://jquery.com" class="external text" rel="nofollow">jQuery</A> library (version 1.10.0) and an <A href="http://james.padolsey.com/javascript/cross-domain-requests-with-jquery/" class="external text" rel="nofollow">extension</A> to the .ajax() method in jQuery written by <A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions#James_Padolsey" class="external text" rel="nofollow">James Padolsey</A> (also see <A href="#Import_of_BioBrick_Sequences">Import of BioBrick Sequences</A> above). The plot was created using the jQuery library <A href="http://www.flotcharts.org" class="external text" rel="nofollow">Flot</A>. This uses the &quot;&lt;canvas&gt;&quot;-tag, which is not supported for Internet Explorer before version 9.0, so for these browsers a work-around using <A href="http://code.google.com/p/explorercanvas/" class="external text" rel="nofollow">excanvas.js</A> is employed. To convert the sequence to its MD5-sum the <A href="http://pajhome.org.uk/crypt/md5/md5.html" class="external text" rel="nofollow">md5.js</A> script written by <A href="http://pajhome.org.uk/index.html" class="external text" rel="nofollow">Paul Johnston</A> was used. The output is HTML-code including style markup and some JavaScript to enable hiding the plot and the features in it.
</P><H5><SPAN class="mw-headline" id="The_main_source_code_of_the_BioBrick-Autoannotator_version_1.0"><A href="https://2013.igem.org/Team:TU-Munich/AnnotatorCode.js?action=raw&amp;ctype=text/js" class="external text" rel="nofollow">The main source code of the BioBrick-Autoannotator version 1.0</A></SPAN></H5><P>For simplicity and speed <A href="https://2013.igem.org/Team:TU-Munich/TUM13_jquery-1.10.2.js?action=raw&amp;ctype=text/javascript" class="external text" rel="nofollow">jQuery.js</A>, <A href="https://2013.igem.org/Team:TU-Munich/AjaxExtension.js?action=raw&amp;ctype=text/js" class="external text" rel="nofollow">AjaxExtension.js</A>, <A href="https://2013.igem.org/Team:TU-Munich/Flot.js?action=raw&amp;ctype=text/js" class="external text" rel="nofollow">Flot.js</A>, <A href="https://2013.igem.org/Team:TU-Munich/md5_pajhome.js?action=raw&amp;ctype=text/js" class="external text" rel="nofollow">md5.js</A>, <A href="https://2013.igem.org/Team:TU-Munich/events.js?action=raw&amp;ctype=text/js" class="external text" rel="nofollow">events.js</A> and <A href="https://2013.igem.org/Team:TU-Munich/excanvas.js?action=raw&amp;ctype=text/js" class="external text" rel="nofollow">excanvas.js</A> were copied to the iGEM server.
</P><H3><SPAN class="mw-headline" id="Standardized_Annotation_of_Protein-encoding_BioBricks"> Standardized Annotation of Protein-encoding BioBricks </SPAN></H3><P>A central idea of iGEM and BioBricks is standardization. However, the part descriptions in the Registry are not really standardized and every team provides different information in different places (if at all). This makes finding and assessing BioBricks for a new project very time-consuming and frustrating, since one may have to search through the description or turn to external sources. Especially for protein-encoding Bricks there is a lot of information, which can be calculated directly from the DNA sequence. So in the <A href="http://openwetware.org/wiki/The_BioBricks_Foundation:RFC#BBF_RFC_96:_The_AutoAnnotator_-_Standardized_annotation_of_protein-encoding_BioBricks" class="external text" rel="nofollow">RFC 96</A> we propose a standard table containing basic parameters, characteristics and predictions for protein-encoding BioBricks. Furthermore we developed a tool, the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" class="external text" rel="nofollow"><B>AutoAnnotator</B></A>, with which this table can easily be created, so it only needs to be pasted into the part description. For more information on how the AutoAnnotator works, please see our <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">Software page</A>.
</P><P>You can view the full BBF <A href="http://tools.ietf.org/html/rfc96" class="external mw-magiclink-rfc">RFC 96</A> as a <A href="https://static.igem.org/mediawiki/2013/c/ca/TUM13_RFC96.pdf" class="external text" rel="nofollow">PDF document</A> or in the gallery below.
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="https://predictprotein.org/" class="external text" rel="nofollow">PredictProtein</A>] Website
</P><P>[<A href="https://rostlab.org/" class="external text" rel="nofollow">Prof. Rost</A>] Lab Homepage
</P><P>[<A href="http://james.padolsey.com/" class="external text" rel="nofollow">James Padolsey</A>] Homepage 
</P><P>[<A href="http://developer.yahoo.com/yql/" class="external text" rel="nofollow">web page</A>] about YQL
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/8125050" class="external text" rel="nofollow">Bjellqvist et al., 1993</A>] Bjellqvist, B., Hughes, G.J., Pasquali, Ch., Paquet, N., Ravier, F., Sanchez, J.-Ch., Frutiger, S. and Hochstrasser, D.F. (1993). The focusing positions of polypeptides in immobilized pH gradients can be predicted from their amino acid sequences. <I>Electrophoresis</I>, 14:1023-1031.
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/8055880" class="external text" rel="nofollow">Bjellqvist et al., 1994</A>] Bjellqvist, B., Basse, B., Olsen, E. and Celis, J.E. (1994). Reference points for comparisons of two-dimensional maps of proteins from different human cell types defined in a pH scale where isoelectric points correlate with polypeptide compositions. <I>Electrophoresis</I>, 15:529-539.
</P><P>[<A href="http://web.expasy.org/protparam/" class="external text" rel="nofollow">web page about ProtParam tool</A>]
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/2610349" class="external text" rel="nofollow">Gill and von Hippel, 1989</A>] Gill, S.C. and von Hippel, P.H. (1989). Calculation of protein extinction coefficients from amino acid sequence data. <I>Anal. Biochem.</I>, 182:319-326.
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC340524/" class="external text" rel="nofollow">Sharp and Li, 1987</A>] Sharp, P.M. and Li, W.H. (1987). The Codon Adaptation Index-a measure of directional synonymous codon usage bias, and its potential applications. <I>Nucleic Acids Res.</I> 15(3):1281–95.
</P><P>[<A href="http://www.kazusa.or.jp/codon/" class="external text" rel="nofollow">Codon Usage</A>] Database
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/10592250" class="external text" rel="nofollow">Nakamura et al.</A>] Nakamura, Y., Gojobori, T. and Ikemura, T. (2000). Codon usage tabulated from the international DNA sequence databases: status for the year 2000. <I>Nucl. Acids Res.</I>, 28:292.
</P><P>[<A href="http://www.sciencedirect.com/science/article/pii/0022283682905150" class="external text" rel="nofollow">Kyte and Doolittle, 1982</A>] Kyte, J. and Doolittle, R.F. (1982). A simple method for displaying the hydropathic character of a protein. <I>Journal of Molecular Biology</I>, 157(1):105-132.
</P><P>[<A href="http://emboss.sourceforge.net/apps/release/6.5/emboss/apps/charge.html" class="external text" rel="nofollow">web page about emboss charge</A>]
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="external text" rel="nofollow">Altschul et al., 1997</A>] Altschul, S.F., Madden, T.L., Schäffer, A.A., Zhang, J., Zhang, Z., Miller, W., Lipman, D.J. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. <I>Nucleic Acids Res.</I> 25:3389-3402.
</P><P>[<A href="http://www.rcsb.org/pdb/static.do?p=general_information/about_pdb/index.html" class="external text" rel="nofollow">PDB</A>] Data Bank
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/15215403" class="external text" rel="nofollow">Rost et al., 2004</A>] Rost, B., Yachdav, G. and Liu, J. (2004). The PredictProtein server.  <I>Nucleic Acid Res.</I> 32:321-326.
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/8844859" class="external text" rel="nofollow">Rost et al., 1996</A>] Rost, B., Fariselli, P. and Casadio, R. (1996). Topology prediction for helical transmembrane proteins at 86% accuracy. <I>Protein Sci.</I> 5:1704-1718.
</P><P>[<A href="http://nar.oxfordjournals.org/content/34/suppl_2/W177.full" class="external text" rel="nofollow">Frasconi et al., 2006</A>] Ceroni, A., Passerini, A., Vullo, A. and Frasconi, P. (2006). DISULFIND: a disulfide bonding state and cysteine connectivity prediction server. <I>Nucleic Acids Res.</I> 34:177-181.
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/22962467" class="external text" rel="nofollow">Rost et al., 2012</A>] Goldberg, T., Hamp, T. and Rost, B. (2012). LocTree2 predicts localization for all domains of life. <I>Bioinformatics</I> 28:i458-i465.
</P><P>[<A href="http://www.biomedcentral.com/1471-2105/14/S3/S7" class="external text" rel="nofollow">Rost et al., 2013</A>] Hamp, T., Kassner, R., Seemayer, S., Vicedo, E., Schaefer, C., Achten, D., Auer, F., Boehm, A., Braun, T., Hecht, M., Heron, M., Honigschmid, P., Hopf, T.A., Kaufmann, S., Kiening, M., Krompass, D., Landerer, C., Mahlich, Y., Roos, M. and Rost B (2013). Homology-based inference sets the bar high for protein function prediction. <I>BMC Bioinformatics</I>. 14(Suppl.3):S7.
</P><P>[<A href="http://jquery.com/" class="external text" rel="nofollow">web page</A>] about jquery
</P><P>[<A href="http://www.flotcharts.org/" class="external text" rel="nofollow">Flot:</A>] JavaScript plotting library for jQuery
</P><P>[<A href="http://code.google.com/p/explorercanvas/" class="external text" rel="nofollow">web page</A>] about explorercanvas
</P><P>[<A href="http://pajhome.org.uk/crypt/md5/md5.html" class="external text" rel="nofollow">Paul Johnston</A>] md5.js script
</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Results/Software">http://2013.igem.org/Team:TU-Munich/Results/Software</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Results/Software" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Results/Software" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Results/Software&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Results/Software&amp;oldid=357037" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>