<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Project_Physcomitrella"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Project/Physcomitrella">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Project/Physcomitrella&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Project/Physcomitrella&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Project/Physcomitrella&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Project/Physcomitrella" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Project/Physcomitrella</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Physcomitrella_-_A_new_chassis_for_iGEM"><I>Physcomitrella</I> - A new chassis for iGEM</SPAN></H2><H3><SPAN class="mw-headline" id="General_description">General description</SPAN></H3><P>The moss <I>Physcomitrella patens</I> belongs to the land plant division Bryophyta, which are one of the earliest representatives of the land plants (Embryophyta) having evolved from green algae about 470 million years ago during the early Paleozoic. Hence mosses have a much simpler anatomy than higher land plants such as trees and flowering plants, which in particular means that they have not yet developed a vascular system, i.e an internal transport system for water and nutrients. Since they also lack a complex waterproofing system to prevent absorbed water from evaporating they need a moist environment to grow. Their main habitats are therefore shady and damp places such as woods and edges of streams but they are also found to be resistant to periods of drought and therefore can be found widely spread around the world, from the tropics to tundra regions, from coastal sand dunes up to high mountains.
</P><P>The general organization of plant tissue into roots, stem and leaves is found in a much more basic version in mosses. They show a differentiated stem with simple leaves, usually only a single layer of cells thick and lacking veins, that are used to absorb water and nutrients. Instead of roots they have similar threadlike rhizoids [<A href="http://www.plant-biotech.net/paper/Reski_1998_BotActa-111_1_scan.pdf" class="external text" rel="nofollow">Reski, 1998</A>]. These have a primary function as mechanical attachment rather than extraction of soil nutrients. Due to not having a vascular system bryophytes are doomed to stay small throughout their life-cycle typically stretching about 1-10 cm.
</P><P>However different mosses and vascular plants are because of the early diverge of the evolutionary lineages, they share fundamental genetic and physiological processes. Hence a good approach to studying the complexity of higher land plants is to look at the bryophytes with their much simpler phenotype. Here researchers chose <I>Physcomitrella patens</I> as a model organism with a genome size of about 450 Mb along 27 chromosomes that is highly similar to other land plants in both exon-intron-structure and codon usage. [<A href="http://www.plantphysiol.org/content/127/4/1430" class="external text" rel="nofollow">Schaefer and Zryd, 2001</A>]
</P><H3><SPAN class="mw-headline" id="Life_cycle">Life cycle</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B><I>Physcomitrella</I> life cycle</DIV></DIV></DIV><P>Generally land plants show an alternation of generations, the haploid (1n) gametophyte produces sperm and eggs which fuse and transform into the diploid (2n) sporophyte. This then forms haploid spores which become new gametophytes. Besides having no vascular system, bryophytes also differ from higher land plants in the fact that the gametophyte is the dominant phase of their life cycle, whereas in vascular plants the principal generation is the sporophyte.
</P><P>The life cycle of <I>P. patens</I> [<A href="http://www.plant-biotech.net/paper/Reski_1998_BotActa-111_1_scan.pdf" class="external text" rel="nofollow">Reski, 1998</A>] only takes about 3 months and starts with the spore developing into a filamentous structure, the juvenile, transitory stage of the gametophyte, called protonema, which is composed of two types of cells. The chloronema cells with large and numerous chloroplasts mostly perform photosynthesis and thus supply the photoautotrophic plant with energy while the task of the caulonema cells is fast growth. The adult stage of the gametophyte, called gametophore (&quot;gamete-bearer&quot;) has a more complex structure bearing leafs, stem and rhizoids. The transition from juvenile to adult gametophyte is started by initial cells in the protonema filament that differentiate into buds. The budding is therefore a single-cell-event, greatly stimulated by the plant hormone cytokinin, which promotes cell division.
</P><P>The sex organs of the moss develop from the tip of the gametophore. <I>P. patens</I> is monoicous, meaning that male and female organs are produced in one plant. When liquid water is surrounding the tip, flagellate sperm cells can swim from the male sex organ to the female organ and fertilize the egg within. A zygote then develops into the sporophyte, which in turn produces thousands of haploid spores by meiosis. Sporophytes are typically physically attached to and dependent on supply from the dominating gametophyte.
</P><H2><SPAN class="mw-headline" id="Advantages_of_Physcomitrella_as_a_model_organism">Advantages of <I>Physcomitrella</I> as a model organism</SPAN></H2><H3><SPAN class="mw-headline" id="General_advantages">General advantages</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2:</B> Cultivation of moss in a Bioreactor</DIV></DIV></DIV><UL><LI><I>P. patens</I> stands out among the whole plant kingdom as the sole exception where gene targeting is feasible as an easy and fast routine procedure, even with an efficiency similar to <I>S. cerevisiae</I>, due to highly efficient homologous recombination. [<A href="http://www.ncbi.nlm.nih.gov/pubmed/14586556" class="external text" rel="nofollow">Reski et al., 2004</A>] For that reason it is very easy to create knockout mosses by precise mutagenesis following the approach of reverse genetics in order to study the function of genes. Performing functional genomics in higher organisms is very important to understand biological functions of proteins in a multicellular context, e.g. in the context of cell-cell-contacts.
</LI><LI>As mosses mainly are in a haploid stage during their life cycle they are very straight-forward objects for genetics because complex backcrosses to determine changes in the genotype are not necessary
</LI><LI>Moss development starts with a filamentous tissue, the protonema, which is growing by apical cell division and, therefore is perfectly suitable for cell-lineage analysis since development of the plant can be pinpointed to the differentiation of a single cell. [<A href="http://www.plant-biotech.net/paper/Reski_1998_BotActa-111_1_scan.pdf" class="external text" rel="nofollow">Reski, 1998</A>] Also the simple life cycle makes <I>P. patens</I> a very useful item for developmental biology
</LI><LI><I>P. patens</I> is increasingly used in biotechnology as a study object with implications for crop improvement or human health. Moss bioreactors (see Figure 2) can be used as an alternative to animal cell cultures (e.g. CHO cells) for the easy, inexpensive and safe production of complex biopharmaceuticals [<A href="http://www.sciencedirect.com/science/article/pii/S0958166907000948" class="external text" rel="nofollow">Reski and Decker, 2007</A>] . For example it is a successful tool to produce asialo-EPO, a specific variant of Erythropoetin, which can perform its protective role by inhibiting apoptosis but has lost the potential doping activity. This safe drug is hard to produce in animal cell culture but easy to produce in the moss without impacting its growth or general performance [<A href="http://www.ncbi.nlm.nih.gov/pubmed/22621344" class="external text" rel="nofollow">Decker et al., 2012</A>].
</LI></UL><H3><SPAN class="mw-headline" id="Expertbox:_Prof._Reski">Expertbox: Prof. Reski</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 3:</B> Expert interview with Professor Reski.</DIV></DIV></DIV><P>When we chose Phytoremediation as our project for this yearÂ´s competition, it soon became clear that <I>Physcomitrella patens</I> is a great chassis that could bring iGEM closer to the real world applications we were looking for. Therefore we contacted Prof. Dr. Reski, who is a recognized expert on <I>Physcomitrella patens</I> and its biotechnological applications.
</P><P>Prof. Dr. Reski liked our idea of introducing his <I>Physco</I> to iGEM from the very beginning and offered his help wherever we would need it. Members of our team traveled to his lab in Freiburg (350 km) five times in total <A href="https://2013.igem.org/Team:TU-Munich/Results/Moss#3._Transformation_of_Physcomitrella_patens" class="external autonumber" rel="nofollow">[1]</A> to fetch plasmids known to be functional in <I>Physcomitrealla patens</I> or for discussions with him or his co-workers. We could win him over as an advisor for our team, which shows how iGEM brings together universities and scientific groups from many different places.
</P><H3><SPAN class="mw-headline" id="Advantages_as_a_new_chassis_for_iGEM">Advantages as a new chassis for iGEM</SPAN></H3><UL><LI>As a plant, <I>P. patens</I> offers interesting opportunities for application as it is self sustaining, renewable and a natural part of our environment. Therefore it is much easier to implement it into real world scenarios than bacteria or yeast. And although there is the disadvantage of having to wait about 4-6 weeks after transformation until experiments can be done, this is still very short considering the high complexity of the organism. Working with <I>P. patens</I> can easily be done in the timeframe of the competition by preparing the DNA constructs in bacteria. 
</LI><LI><I>P. patens</I> is a well studied model organism which means that besides having its full genome sequenced in 2006 there are well equipped <A href="http://www.cosmoss.org" class="external text" rel="nofollow">databases</A>. Furthermore, there exists an <A href="http://www.moss-stock-center.org" class="external text" rel="nofollow">International Moss Stock Center (IMSC)</A> in which many ecotypes, mutants and transgenic strains of <I>P. patens</I> are stored and accessible to the scientific community. So there is enough knowledge and material to work on for synthetic biologists. 
</LI><LI>At the same time the moss offers access to very exciting new physiological processes since it is a much more complex multicellular eukaryotic organism than the chassis already established in iGEM.
</LI><LI><I>P. patens</I> is an easy plant to work with and requires neither expensive maintenance facilities nor large laboratory space. Most of the basic tools for high precision mutagenesis have been tested on this plant, were found to work and are easily available (see Moss methods below).
</LI></UL><H2><SPAN class="mw-headline" id="Moss_methods_-_working_with_Physcomitrella">Moss methods - working with <I>Physcomitrella</I></SPAN></H2><P>The techniques used for the cultivation and manipulation of Physcomitrella patens are based on the knowledge of the chair for Plantbiotechnology of Prof. Dr. Reski at Freiburg University, which are available at <A href="http://www.plant-biotech.net" class="external text" rel="nofollow">Plant-Biotech.net</A>. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 4:</B> Working with cultivated moss on our laboratory bench</DIV></DIV></DIV><UL><LI><B>Cell culture</B>: <I>Physcomitrella patens</I> plants can be cultivated either on solidified medium or in liquid medium. Liquid cultures can be kept in Erlenmeyer flasks under constant rotation and light exposure or in bioreactors for large scale production. By regularly (ideally weekly) disrupting the plants mechanically with a mixer (e.g. Ultra-Turrax) a homogenized culture can be achieved.
</LI><LI><B>Storage</B>: Long-term storage of <I>Physcomitrella</I> strains can be achieved by cryo-preservation. This procedure ensures a maximum survival rate of the plant by preconditioning and controlled freezing. [<A href="http://www.plant-biotech.net/paper/PlantBiol_2004_Schulte_pageproof.pdf" class="external text" rel="nofollow">Schulte and Reski, 2004</A>]
</LI><LI><B>Targeted knockout</B>: <I>P. patens</I> is unique among plants in its high efficiency of gene targeting by homologous recombination. To knockout a specific gene a disruption construct has to be generated. This consists of the gene to be silenced with a selection cassette (usually the nptII gene) inserted into its center by suitable restriction sites. The moss is then transformed with this DNA construct which is integrated into the genome by homologous recombination. [<A href="http://www.ncbi.nlm.nih.gov/pubmed/14586556" class="external text" rel="nofollow">Reski et al., 2004</A>]
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 5:</B> Transformation procedure</DIV></DIV></DIV><UL><LI><B>Transformation</B>: Transformation of <I>P. patens</I> requires protoplasts of the plant, which can be obtained by cell wall digesting enzymes. The protoplasts can either be transformed by particle bombardment or via polyethylene glycol (PEG), which is the easiest and most commonly used method. The DNA constructs should be linearized for optimal transformation and contain a selection marker for subsequent screening. After transformation, the protoplasts are cultivated in the dark for 12-16 h followed by 9 days under normal growth conditions during which they regenerate their cell wall. Afterwards they are plated on solidified medium in normal petri dishes and first experiments can be executed. [<A href="http://link.springer.com/article/10.1007/BF00260654" class="external text" rel="nofollow">Schaefer et al., 1991</A>]
</LI><LI><B>Analysis of transformants</B>: First, the transformed protoplasts are plated on solidified medium containing the antibiotic G418 (Geneticin), to which successfully transformed plants are resistant due to the selection marker nptII which encodes the enzyme neomycin phosphotransferase. Secondly, the transformants are analyzed by a PCR screen using primers derived the selection marker cassetteÂ´s sequences [<A href="http://www.plant-biotech.net/paper/PlantMolecularBiologyReporter_2002_Schween.pdf" class="external text" rel="nofollow">Schween et al., 2002</A>].
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:912px;"><DIV class="thumbcaption"><B>Figure 6:</B> Comparing work flow for <I>E. coli</I>, <I>S. cerevisiae</I> and <I>P. patens</I></DIV></DIV></DIV></DIV><P>In <B>conclusion</B>, it naturally takes a bit more time to work with a plant in comparison to bacteria or yeast (see figure 6), but the advantages definitely outweigh the time factor and if teams start early enough, it is perfectly possible to work with <I>P. patens</I> as a chassis in an iGEM project!
</P><P>This is a brief summary of the most important techniques in working with <I>P. patens</I>. For a full description of these methods please see <A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods#Physcomitrella_Methods" class="external text" rel="nofollow">our methods page</A>.
</P><H2><SPAN class="mw-headline" id="BioBricks_for_protein_expression_in_Physcomitrella">BioBricks for protein expression in <I>Physcomitrella</I></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 7:</B> Our transformation vector</DIV></DIV></DIV><P>To transform the moss, we created an all-purpose backbone into which all our constructs could be integrated easily. Therefore we modified the common iGEM pSB1C3 plasmid with the following plant-specific components:
</P><UL><LI><B>pAct5 (Actin 5 promoter from <I>Physcomitrella patens</I>):</B> for strong protein expression.
</LI></UL><UL><LI><B>miniMCS:</B> This miniature multiple cloning site contains restriction sites for the enzymes MfeI and SbfI which allows you to insert BioBrick between two other Biobricks (in our case: pAct5 and t35S_npt-cassette) as a result of the compability of MfeI with EcoRI and SbfI with PstI.
</LI></UL><UL><LI><B>t35S (35S terminator):</B> This terminator originates from the 35S transcript of the cauliflower mosaic virus.
</LI></UL><UL><LI><B>nptII-cassette:</B> This selection cassette contains the neomycon phosphotransferase II gene, encoding the enzyme aminoglycoside-3'-phosphotransferase (NPTII), which inactivates a range of aminoglycoside antibiotics such as Kanamycin, Neomycin and Geneticin (G418) by phosphorylation. The latter antibiotic is used as selective agent for transformed <I>P. patens</I> plants, inhibiting the growth of untransformed plants very effectively. The neomycon phosphotransferase II gene is under control of the NOS promoter and terminator.
</LI></UL><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="http://www.plant-biotech.net/paper/Reski_1998_BotActa-111_1_scan.pdf" class="external text" rel="nofollow">Reski, 1998</A>] Reski, R. (1998). Development, Genetics and Molecular Biology of Mosses. <I>Bot. Acta</I>, 111:1-15.</P><P>[<A href="http://www.plantphysiol.org/content/127/4/1430" class="external text" rel="nofollow">Schaefer and Zryd, 2001</A>] Schaefer, D.G. and ZrÃ¿d, J. (2001). The Moss <I>Physcomitrella patens</I>, Now and Then. <I>Plant Physiology</I>, 127(4):1430-1438.</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/14586556" class="external text" rel="nofollow">Reski et al., 2004</A>] Hohe, A., Egener, T., Lucht, J.M., Holtorf, H., Reinhard, C., Schween, G. and Reski, R. (2004). An improved and highly standardised transformation procedure allows efficient production of single and multiple targeted gene-knockouts in a moss, Physcomitrella patens. <I>Curr Genet.</I>, 44(6):339-47.</P><P>[<A href="http://www.sciencedirect.com/science/article/pii/S0958166907000948" class="external text" rel="nofollow">Reski and Decker, 2007</A>] Decker, E.L. and Reski, R. (2007). Moss bioreactors producing improved biopharmaceuticals. <I>Current Opinion in Biotechnology</I>, 18(5):393-398.</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/22621344" class="external text" rel="nofollow">Reski et al., 2012</A>] Parsons, J., Altmann, F., Arrenberg, C. K., Koprivova, A., Beike, A. K., Stemmer, C., Gorr, G., Reski, R. and Decker, E. L. (2012). Moss-based production of asialo-erythropoietin devoid of Lewis A and other plant-typical carbohydrate determinants. <I>Plant Biotechnology Journal</I>, 10:851â861.</P><P>[<A href="http://www.plant-biotech.net/paper/PlantBiol_2004_Schulte_pageproof.pdf" class="external text" rel="nofollow">Schulte and Reski, 2004</A>] Schulte, J. and Reski, R. (2004). High throughput Cryopreservation of 140 000 Physcomitrella patens Mutants. <I>Plant Biology</I>, 6:119-127.</P><P>[<A href="http://link.springer.com/article/10.1007/BF00260654" class="external text" rel="nofollow">Schaefer et al., 1991</A>] Schaefer, D., Zryd, J.-P., Knight, C., Cove, D. (1991). Stable transformation of the moss <I>Physcomitrella patens</I>. <I>Molecular and General Genetics</I>, 226:418-424.</P><P>[<A href="http://www.plant-biotech.net/paper/PlantMolecularBiologyReporter_2002_Schween.pdf" class="external text" rel="nofollow">Schween et al., 2002</A>] Schween, G., Fleig, S., Reski, R. (2002). High-throughput-PCR screen of 15,000 transgenic Physcomitrella plants. <I>Plant Molecular Biology Reporter</I>, 20:43â47.</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Project/Physcomitrella">http://2013.igem.org/Team:TU-Munich/Project/Physcomitrella</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Project/Physcomitrella" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Project/Physcomitrella" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Project/Physcomitrella&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Project/Physcomitrella&amp;oldid=359706" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>