<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Project_Safety"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Project/Safety">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Project/Safety&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Project/Safety&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Project/Safety&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Project/Safety" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Project/Safety</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Biosafety">Biosafety</SPAN></H2><P><I>Biosafety deals with the prevention of unintented exposure to pathogens and toxins, or their accidental release</I> [<A href="http://www.who.int/csr/resources/publications/biosafety/Biosafety7.pdf" class="external text" rel="nofollow">WHO</A>].  
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1</B>: The three main parts of our safety system</DIV></DIV></DIV><P>Biosafety to us means minimizing the risks concering the team working in the lab, the general public and the environment. <I>Physcomitrella</I> itself does not pose any risks to the health of the researcher or the general public, since it is endemic to many parts of the world. In the lab, we are able to cultivate our moss in bioreactors in which the flow of substances is tightly controlled.  They can even be used for the production of therapeutic proteins, as done by the <A href="http://www.greenovation.com/" class="external text" rel="nofollow">greenovation</A> company. For usage in a sewage plant, such reactors could be upscaled and special filter systems could be applied to ensure that no moss can escape into the environment. However, our long-term goal is to use moss filters <B>in</B> polluted environments but at the same time ensuring the highest level of biological safety. 
</P><P>We believe that the main biosafety-issues that have to be addressed regarding our project are
</P><UL><LI> toxicity of recombinant proteins
</LI><LI> toxicity of degradation products
</LI><LI> transgene pollution (vertical and horizontal gene transfer)
</LI></UL><P>In order to minimize the aforementioned risks, we have taken several  measures, including a light-triggered killswitch, non-sporulating <I>Physcomitrella</I> strains and the evaluation of the genetic parts that we are using in terms of safety.
</P><H3><SPAN class="mw-headline" id="Toxicity_of_recombinant_proteins">Toxicity of recombinant proteins</SPAN></H3><P>The most obvious safety threat is that the recombinant proteins might be toxic and thus have a negative effect on the environment. This is the foundation of our safety consideration - that the products from all parts and genetic circuits must not cause harm by themselves. All our effector proteins (laccase, catechol-2,3-dioxygenase, erythromycin esterase B) are all from S1 organisms and are not toxic. For example, laccases are already used in the food industry [<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2963825/" class="external text" rel="nofollow">Osma et al., 2010</A>]. For the establishment of a eukaryotic expression system, we are also using parts of mammalian origin and from S2-organisms (see safety forms below). We have made the decisions to use parts from S2 organism carefully and only after consulting our safety officer. He could ensure us that the parts and circuits we are using do not pose any threat. When possible, we have tried to substitute parts from S2 organisms with parts from S1 organisms. All of the parts we are using (such as a polioviral internal ribosome entry site (IRES) and the Ig-Kappa secretion signal from mouse) are widely used in molecular biology laboratories all around the world and are considered safe.
</P><H3><SPAN class="mw-headline" id="Safety_evaluation_for_used_BioBricks_and_composite_parts">Safety evaluation for used BioBricks and composite parts</SPAN></H3><P><SMALL><B>Table 1</B>: Extended safety evaluation for biobricks of mammalian origin or from S2 organisms
</SMALL></P><TABLE class="safety"><TBODY><TR><TH width="21%"> Part
</TH><TH width="27%"> BBa_K1159004
</TH><TH width="27%"> BBa_K1159304
</TH><TH width="27%"> BBa_K729004
</TH></TR><TR><TH>Organism name and strain name or number
</TH><TD><I>Homo sapiens</I></TD><TD><I>Mus musculus</I> (mammal)
</TD><TD><I>Staphylococcus aureus</I></TD></TR><TR><TH>Risk Group
</TH><TD>1
</TD><TD>1
</TD><TD>2
</TD></TR><TR><TH>name of the part and brief description
</TH><TD>Proteinphosphatase 1 is able to bind the cyanotoxin microcystin and is used in our bioaccumulation project. Our moss could thus filter the toxin from the water
</TD><TD>Ig-Kappa is used as a signal peptide for our eucaryotic chassis <I>Physcomitrella patens</I> in order to secrete effector proteins
</TD><TD>part BBa_K729004 (nuclease from <I>Staphylococcus aureus</I>, but we have performed a PCR to remove bacterial signal peptides) It causes genomic degradation and we are using it in our kill switch to kill the moss and destroy all genetic material
</TD></TR><TR><TH>How did you physically acquire the organism or part?
</TH><TD>Cooperation with iGEM Team Dundee 2013
</TD><TD>gene synthesis
</TD><TD>parts registry
</TD></TR><TR><TH>What potential safety/health risks to team members, other people at your institution, or the general public could arise from your use of this organism/part?
</TH><TD>No safety risks
</TD><TD>No potential safety/health risks of the part itself. Composite parts containing this signal sequence could be possibly dangerous when the signal sequence is fused to a dangerous protein. However, we are not using the part in combination with possibly dangerous other parts.
</TD><TD>The protein produced by the part is not dangerous [<A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/en0181-usa-msds.pdf" class="external text" rel="nofollow">Thermo Scientific</A>]. However, <I>S. aureus</I> nuclease production could contribute to disease pathogenesis in vivo
</TD></TR><TR><TH>What measures do you intend to take to ensure that your project is safe for team members, other people at your institution, and the general public?
</TH><TD colspan="3"> We have all completed a biosafety course and we handle everything in the lab with the necessary precaution. We will not distribute any parts or organisms outside the lab, we are using moss that is not able to sporulate and have implemented a kill switch (see basic safety form)
</TD></TR><TR><TH>Explain why you believe the part iself is not dangerous
</TH><TD>The part is human-derived and should thus not be dangerous
</TD><TD>It is a eukaryotic secretion signal which enables secretion of proteins that carry this signal sequence in mammals and other eukaryotes. Therefore the part itself is not dangerous
</TD><TD><I>S. aureus</I> nuclease production contributes to disease pathogenesis in vivo so it could possibly enhance pathogenesis in other microorganisms as well [<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2982853/" class="external text" rel="nofollow">Berends et al., 2010</A>]. It is therefore important to ensure that no genetic material of our moss gets into the environment
</TD></TR><TR><TH>Why do you need to use this organism/part? Is there an organism/part from a less dangerous Risk Group that would accomplish the same purpose?
</TH><TD>Part is needed to filter the dangerous cyanotoxin microcystin from the water. Since proteinphosphatase binds microcystin with a very high affinity it is the best choice.
</TD><TD>Part is needed to be able to secrete our effector proteins. Since we are using Physcomitrella patens as a chassis, we have to use a eukaryotic secretion signal. The use of the Ig-kappa secretion signal from mouse (BSL 1) is widely established and does not pose any safety risks
</TD><TD>Part is needed in our kill switch to ensure that the moss completely killed and that all genetic material is destroyed.
</TD></TR><TR><TH>Is the organism/part listed under the Australia Group guidelines, or otherwise restricted for transport?
</TH><TD colspan="3"> No
</TD></TR><TR><TH>Please describe the BioSafety Level of the lab in which the team works
</TH><TD colspan="3"> The lab we work in has BioSafety Level 1 and we are not using organisms with a BSL level greater than 1
</TD></TR></TBODY></TABLE><P><SMALL><B>Table 2</B>: Extended safety evaluation for biobricks of mammalian origin or from S2 organisms
</SMALL></P><TABLE class="safety"><TBODY><TR><TH width="21%"> Part
</TH><TH width="27%"> BBa_K1159300
</TH><TH width="27%"> BBa_K1159201 / BBa_K1159200
</TH><TH width="27%"> BBa_K801030
</TH></TR><TR><TH>Organism name and strain name or number
</TH><TD>Human poliovirus 1 strain Mahoney
</TD><TD><I>Streptococcus pyogenes</I></TD><TD>Simean virus 40
</TD></TR><TR><TH>Risk Group
</TH><TD>2
</TD><TD>2
</TD><TD>2
</TD></TR><TR><TH>name of the part and brief description
</TH><TD>Internal ribosome entry site (IRES) from poliovirus 1 allows for translation initiation in the middle of a messenger RNA (mRNA) sequence and thus enables to build constructs consisting of several parts controlled by one promoter
</TD><TD>Spytag/Spycatcher part is used for posttranslational protein fusions (fusion of our effectors to membrane receptors).The two parts consists of a domain of streptococcus pyogenes fibronectin-binding protein FbaB. This domain was split and the fragments were rationally engineered [<A href="http://www.ncbi.nlm.nih.gov/pubmed/22366317" class="external text" rel="nofollow">Zakeri et al., 2012</A>]
</TD><TD>part BBa_K801030 is a nuclear localization sequence (NLS) from Simian virus 40, it enables targeting of proteins into the nucleus
</TD></TR><TR><TH>How did you physically acquire the organism or part?
</TH><TD>Cooperation with Prof. Dr. Fussenegger from ETH Zurich [<A href="http://www.ncbi.nlm.nih.gov/pubmed/16026881" class="external text" rel="nofollow">Hartenbach &amp; Fussenegger, 2005</A>]
</TD><TD>gene synthesis
</TD><TD>parts registry
</TD></TR><TR><TH>What potential safety/health risks to team members, other people at your institution, or the general public could arise from your use of this organism/part?
</TH><TD>Human poliovirus 1 is a human pathogen but the IRES part we are using is itself not harmful
</TD><TD>We believe that the part itself is not dangerous. Since the binding of tag/catcher is very specific, the catcher cannot bind to other proteins and only binds proteins that are fused to the tag
</TD><TD>the part itself consists of 7 amino acids and does not pose any safety or health risks
</TD></TR><TR><TH>What measures do you intend to take to ensure that your project is safe for team members, other people at your institution, and the general public?
</TH><TD colspan="3"> We have all completed a biosafety course and we handle everything in the lab with the necessary precaution. We will not distribute any parts or organisms outside the lab, we are using moss that is not able to sporulate and have implemented a kill switch (see basic safety form)
</TD></TR><TR><TH>Explain why you believe the part iself is not dangerous
</TH><TD>Internal ribosome entry sites are widely used as a means to accomplish polycistronic expression in eukaryotes
</TD><TD>Since it was partly engineered, the parts itself is not dangerous in a way that it is connected to <I>Streptococcus pyogenes</I>. The fusion product tag-catcher itself is not dangerous but this depends on the proteins tag/catcher are fused to
</TD><TD>the part is based on the plasmid <I>pGADT7 AD</I> from Clontech, so the NLS is a commercially available and widely used. We therefore consider it not dangerous
</TD></TR><TR><TH>Why do you need to use this organism/part? Is there an organism/part from a less dangerous Risk Group that would accomplish the same purpose?
</TH><TD>Without IRES we would have to put a promotor in front of every protein we want to express which would make cloning of large constructs much more difficult. There are IRES elements found in cellular mRNAs (e.g. in Fibroblast growth factor) however, they are not as established in expression systems
</TD><TD>Part is needed for posttranslational fusion of our effectors to membrane receptors. There is no part from a less dangerous risk group that would accomplish this purpose this well
</TD><TD>Part is needed to target the nuclease of our killswitch into the nucleus of our moss. Without a localization signal, the killswitch would not work. The NLS from SV40 is used in almost all commercially available vector systems and has been in use for a long time which is why we chose it
</TD></TR><TR><TH>Is the organism/part listed under the Australia Group guidelines, or otherwise restricted for transport?
</TH><TD colspan="3"> No
</TD></TR><TR><TH>Please describe the BioSafety Level of the lab in which the team works
</TH><TD colspan="3"> The lab we work in has BioSafety Level 1 and we are not using organisms with a BSL level greater than 1
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Toxicity_of_degradation_products">Toxicity of degradation products</SPAN></H3><P>None of the transgenic plants that we have created is able to produce a toxic substance. However, degradation products of hormones, antibiotics and other pollutants could be potentially toxic. Therefore, the reaction products will be monitored using LC/MS analysis.
For example for EreB, the transformation products of erythromycin possess no endocrine and mutagenic properties [<A href="http://www.endetech.eu/sites/default/files/field/image/2013.06.07-Oral%20presentation%20ICWRER%202013.pdf" class="external text" rel="nofollow">endetech</A>]. In general, it is very difficult to find data on the toxicity of these degradation products.  
</P><P>Additionally, the substances used in bioaccumulation (Glutathion-S-transferase, Proteinphosphatase 1) are not harmful themselves, but moss that has bound toxic substances (e.g. microcystin) in high concentrations is potentially toxic and has to be disposed of separately and with caution.
</P><H3><SPAN class="mw-headline" id="Transgene_pollution">Transgene pollution</SPAN></H3><P><B>Transgene pollution</B> describes the spread of transgenes beyond the genetically modified species by natural gene flow mechanisms [<A href="http://www.cabi.org/cabreviews/FullTextPDF/2003/20033177371.pdf" class="external text" rel="nofollow">Commandeur et al., 2003</A>]. Transgenes can spread from transgenic to non-transgenic populations of the same species or related wild species (<B>vertical gene transfer</B>) or to entirely different species such as bacteria (<B>horizontal gene transfer</B>). Because the impact of novel genes on other species is unpredictable, transgene pollution is undesirable and could pose a risk to the ecosystem. When talking about transgenes, one also has to differentiate between <B>primary transgenes</B> such as our effectors and <B>superfluous DNA sequences</B> such as antibiotic resistance markers. 
</P><H4><SPAN class="mw-headline" id="Preventing_vertical_gene_transfer_with_non-sporulating_Physcomitrella_strains">Preventing vertical gene transfer with non-sporulating <I>Physcomitrella</I> strains  </SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2</B>: Sporophytes of P. patens WT (A) and DSiR1 (B–D) lines 8 weeks after induction of sporulation. The spores of WT plants develop until they are mature in the spore capsules which then tear open and set free the spores. In the DSiR1 lines the spore capsules crack  open when the spores inside are still immature, Wiedemann et al., 2010</DIV></DIV></DIV><P>In order to prevent vertical gene transfer and to limit the spread of the moss in the environment, we looked for strains which are not able to form mature spores. <I>P. patens</I> is monoicous, meaning that male and female organs are produced in one plant. Normally, at the tips of adult gametophores the sexual organs, antheridia (male) and archegonia (female), are produced under inducing conditions. After fertilisation of the egg inside the archegonium a sporophyte develops which contains approx. 5000 spores. A possible way to inhibit successful germination of these spores is to knock out the enzyme sulfite reductase 1 (SiR1). This protein reduces sulfite to sulfide and is involved in sulfur metabolism. [<A href="http://www.sciencedirect.com/science/article/pii/S0014579310002462" class="external text" rel="nofollow">Wiedemann et al., 2010</A>] disrupted PpSiR1 by homologous recombination and found that ΔSiR1 plants showed strong developmental alterations and are unable to produce mature spores. In the ΔSiR1 lines, only one third of the number of sporophytes was formed, the spore capsules cracked open when the spores inside were still immature and these mutant spores did not germinate. Since it takes more time to establish a mature culture with the knock-out mutants and considering the limited time in iGEM competitions, we did not actually use this mutant in our experiments for iGEM. However, the mutant can easily be ordered from the <A href="http://www.moss-stock-center.org/|" class="external text" rel="nofollow">International Moss Stock Center</A>. A consideration for the future would be to design the integration vector in a way that targeted gene knockouts in the disulfite reductase gene are possible, thus inhibiting sporulation and having a targeted integration of constructs.
</P><H4><SPAN class="mw-headline" id="Horizontal_gene_transfer">Horizontal gene transfer</SPAN></H4><P>Concerning horizontal gene transfer, the main safety threat would be the spread of transgenes from our moss to bacteria. The transfer of primary transgenes or marker genes could lead to new bacteria strains that express these genes and could pass them on to pathogenic strains. But while this process is very common in bacteria, apart from <I>agrobacterium tumefaciens</I>, there are few examples of natural gene transfer between bacteria and higher eukaryotes [<A href="http://www.cabi.org/cabreviews/FullTextPDF/2003/20033177371.pdf" class="external text" rel="nofollow">Commandeur et al., 2003</A>]. 
</P><P>For the transgene to remain in the bacterial population and thus in the environment, there has to be a strong selective pressure. This could be the case in highly contaminated environments, where the ability to break down a toxic pollutant is beneficial. We are aware that our moss carries transgenes that could be advantageous to bacteria living in these polluted environments. But since <B>all</B> of the effectors that we are testing actually come from bacteria, we do no think that this is the main problem. The genes for these effectors are already present in the bacteria population. Nevertheless, we need to have a closer look at antibiotic resistance genes such as ereB and the selection marker nptII. Antibiotic resistance genes are the focus of attention because of their potentially strong and general selective advantage in human pathogens. But in the case of ereB, which comes from <I>E.coli</I>, the resistance gene is already present in the bacterial population on plasmids while in the moss it is controlled by a moss promotor and integrated in the moss genome. Therefore, it is much more likely that in case of ereB, the horizontal gene transfer occurs between bacteria and not between bacteria and moss. 
</P><P>A possible way to eliminate superfluous DNA sequences such as the nptII marker after a successful transformation is the use of a two-component site-specific recombination system such as Cre-loxP [<A href="http://www.ncbi.nlm.nih.gov/pubmed/1660141" class="external text" rel="nofollow">Dale and Ow, 1991</A>]. The recombinase Cre recognizes short sequences known (loxP) and is then able to excise DNA betweeen two loxP sites that are in the same orientation. Therefore, marker genes flanked by loxP sites can be efficiently excised from transgenic plants if Cre is present.
</P><H4><SPAN class="mw-headline" id="Light-triggered_Killswitch">Light-triggered Killswitch</SPAN></H4><P>In order to prevent our moss from escaping into the environment, we have included a light-triggered killswitch. When exposed to red-light, a nuclease system is activated which destroys all genetic material, killing the moss and preventing the spread of genetic material of the moss. With this genetic circuit it would be possible to create an niche for the transgenic moss by just covering an area with a blue  filter foil. For more information, look at the <A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch" class="external text" rel="nofollow">killswitch page</A>.
</P><H2><SPAN class="mw-headline" id="Biosecurity">Biosecurity</SPAN></H2><P>Biosecurity is the prevention of loss, theft, misuse, diversion or intentional release of pathogens and toxins. <I>Physcomitrella</I> itself is in no way pathogenic and is endemic to many parts of the world. However, just like with every transgenic organism, there is the theoretical possibility to use <I>Physcomitrella</I> to cause harm, e.g. when it is used to secrete toxic substances. Nevertheless, other organisms which have for example shorter generation times or are pathogenic by nature seem to be more appropriate for such dual-use applications. Additionally, none of the <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss#Transgenic_Physcomitrella_patnes_plants|" class="external text" rel="nofollow">transgenic plants we created</A> could be used to cause harm.
</P><H2><SPAN class="mw-headline" id="Labsafety">Labsafety</SPAN></H2><P>The lab we work in is classified as BSL 1 (biosafety level 1), according to the <A href="http://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=CELEX:32000L0054:en:NOT|European" class="external text" rel="nofollow">Union Directive 2000/54/EG</A> and the German &quot;Gesetz zur Regelung der Gentechnik (<A href="http://www.gesetze-im-internet.de/gentg/|" class="external text" rel="nofollow">GenTG</A>)&quot; (law for the regulation of genetic engineering). There is a total of four Biosafety levels, with BSL 1 being the lowest and BSL 4 being the highest. This classification of the respective Biosafety levels is very similar to the one given in the <A href="http://www.who.int/csr/resources/publications/biosafety/en/Biosafety7.pdf|" class="external text" rel="nofollow">World Health Organization (WHO) Laboratory Biosafety Manual</A>. Work inside a BSL 1 lab, such as ours, involves no devices that are potentially harmful to the researchers if they act according to the general precautionary measures. Especially, no pathogenic organisms are used. 
</P><P>A regular safety briefing and a lecture about the legal basics concerning biotechnology and genetic engineering are basic elements of our education at TU Munich. In this context, the handling of biological material, dangerous aspects of chemicals and the circumstances and protocols at the lab we work in are explained. Additionally a special safety briefing was held for all iGEM students by Dr. Martin Schlapschy, who is responsible for safety in Prof. Skerras lab.
</P><P>All of us have worked in laboratories before and have experience with biological parts and chemicals. When we are unsure about the safety measures that have to be taken when handling certain chemicals and devices, we always have the support of our instructors and the researchers working at Prof. Skerras lab. 
</P><H4><SPAN class="mw-headline" id="Safety_precautions_during_molecular_biology_experiments">Safety precautions during molecular biology experiments</SPAN></H4><P>Just like in every other biochemical laboratory, there are substances and devices in our lab which are potentially dangerous. Here are three examples of how these situations are handled in our lab.
</P><OL><LI>In every laboratory of molecular biology, specific chemicals are required for the staining of DNA, in order to make it visible on agarose gels. Most of them directly intercalate into double-stranded DNA, making them carcinogenic. The substance we use is ethidium bromide. To prevent skin contact, we wear protective gloves made from nitrile rubber and change them frequently to prevent contamination. All gels and materials that come into contact with ethidium bromide are disposed of separately. This is done in order to prevent their unintended leakage into the environment with subsequent harm to humans, animals and plants. 
</LI><LI>Methods of molecular biology often require strong acids or bases, like hydrochloric acid, or toxic substances such as methanol. We handle them with extreme caution under a fume hood and dispose of them separately.
</LI><LI>Many devices in the lab can be potentially dangerous towards researchers if they are used carelessly or in the wrong way. One example for this are lamps emitting ultraviolet radiation, which can cause damage to the eyes. When dealing with UV radiation, we always wear safety helmets made out of plexiglas. We are aware of the potential harm caused by devices that we are using and thus can protect ourselves appropriately.
</LI></OL><H2><SPAN class="mw-headline" id="Safety_Forms">Safety Forms</SPAN></H2><P><A href="/wiki/images/0/0b/TUM13_Basic_Safety_Form.pdf" class="internal" title="TUM13 Basic Safety Form.pdf">Basic_Safety_Form.pdf</A></P><P><A href="/wiki/images/1/1a/TUM13_Safety_Form2.pdf" class="internal" title="TUM13 Safety Form2.pdf">Safety_Form2.pdf</A></P><P><B>Safety forms were approved on October 3, 2013 by Evan Appleton.</B></P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2982853/" class="external text" rel="nofollow">Berends et al., 2010</A>] Nuclease Expression by Staphylococcus aureus Facilitates Escape from Neutrophil Extracellular Traps. J Innate Immun.  2(6): 576–586
</P><P>[<A href="http://www.cabi.org/cabreviews/FullTextPDF/2003/20033177371.pdf" class="external text" rel="nofollow">Commandeur et al., 2003</A>] The biosafety of molecular farming in plants. AgBiotechNet 2003, Vol. 5 April, ABN 110
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/1660141" class="external text" rel="nofollow">Dale and Ow, 1991</A>] Gene transfer with subsequent removal of the selection gene from the host genome. Proc Natl Acad Sci USA. 88(23):10558-62
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/16026881" class="external text" rel="nofollow">Hartenbach &amp; Fussenegger, 2005</A>] Autoregulated, bidirectional and multicistronic gas-inducible mammalian as well as lentiviral expression vectors. J Biotechnol. 120(1): 83-98
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2963825/" class="external text" rel="nofollow">Osma et al., 2010</A>] Uses of Laccases in the Food Industry. Enzyme Research, 2010: 918761
</P><P>[<A href="http://www.sciencedirect.com/science/article/pii/S0014579310002462" class="external text" rel="nofollow">Wiedemann et al, 2010</A>] Targeted knock-out of a gene encoding sulfite reductase in the moss Physcomitrella patens affects gametophytic and sporophytic development. FEBS Letters. 548(11): 2271–2278
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/22366317" class="external text" rel="nofollow">Zakeri et al., 2012</A>] Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. Proc Natl Acad Sci U S A. 109(12): E690-7
</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Project/Safety">http://2013.igem.org/Team:TU-Munich/Project/Safety</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Project/Safety" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Project/Safety" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Project/Safety&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Project/Safety&amp;oldid=350402" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>