<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Results_Summary"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Results/Summary">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Results/Summary&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Results/Summary&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Results/Summary&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Results/Summary" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Results/Summary</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Our_Project_for_this_summer:_Remediation.">Our Project for this summer: Remediation.</SPAN></H2><P>During this summer we wanted to work on an iGEM project, which has the potential to become a real world application, since we believe that it is an important step for Synthetic Biology to provide alternative solutions for global problems. For this reason we focused on <A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation" class="external text" rel="nofollow">Bioremediation:</A> The use of organisms to remove emissions caused by humans and to bring the environment back to its natural state. As water is a resource which is absolutely essential for all living organisms, we decided to focus on the pollution of aquatic ecosystems.</P><H2><SPAN class="mw-headline" id="Choice_of_the_appropriate_chassis_for_a_water_filter">Choice of the appropriate chassis for a water filter</SPAN></H2><P>Remediation is not new to iGEM, in fact it is a topic the iGEM community has worked on for nearly 10 years now. Therefore a set of promising BioBricks was already available in the Parts Registry, which we wanted to use in order to increase the knowledge on these effector proteins. Having found suitable effector proteins, we discussed about the most suitable chassis for our application. Most of the previous projects on Bioremediation were based on <I>E. coli</I>, whereas we decided to use a plant instead. Photosynthesis carried out by the plants will allow the water filter to maintain and renew itself without the addition of any nutrients. We considered algae such as <I>Chlamydomonas reinhardtii</I>, Bryophytes such as <I>Physcomitrella patens</I> and higher plants like <I>Arabidopsis thaliana</I>. In the end <A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella" class="external text" rel="nofollow"><I>Physcomitrella patens</I></A> was the chassis of choice as it already grows in a filter-like structure and can be cultivated in terrestric as well as in aquatic conditions. Additionally it is a well established organism in biotechnology. Working with <I>Physcomitrella patens</I> is not easy considering the 1-2 months it takes from the transformation process to the experiments with stable transfected plants and the doubling times of 3-6 days. As nobody at the TU Munich works with the moss <I>Physcomitrella patens</I>, we looked for an expert and found Prof. Reski who occupies a professorship at Freiburg University. We were very happy to gain him as an advisor during our project. For the use of <I>Physcomitrella patens</I> in iGEM we created a strong constitutive promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159306" class="external text" rel="nofollow">BBa_K1159306</A>), a plant terminator (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159307" class="external text" rel="nofollow">BBa_K1159307</A>) and an antibiotic selection marker (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159308" class="external text" rel="nofollow">BBa_K1159308</A>), which were all used to transform and select 21 different transgenic moss lines.
</P><H2><SPAN class="mw-headline" id="Localization_of_effector_proteins">Localization of effector proteins</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1:</B> Cytosolic protein expression and our modularized receptor for <I>Physcomitrella</I> work as expected.</DIV></DIV></DIV><P>The actual remediation of pollutants is accomplished by effector proteins which function with significantly different mechanisms. Thus it was important to enable the <A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation" class="external text" rel="nofollow">localization</A> of effector proteins at different cellular sites. Cytosolic effector proteins are easily expressed, whereas for secretion a signal peptide BioBrick is cloned ahead of the effector protein. Several receptor signal peptides from <I>Physcomitrella patens</I> and other organisms were analyzed by using <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions" class="external text" rel="nofollow">bioinformatics tools</A>. The signal peptide of the Somatic Embryogenesis Receptor-like Kinase (SERK) from <I>Physcomitrella patens</I> and the IgK signal peptide from <I>Mus musculus</I>, which is described in literature to function in <I>Physcomitrella</I>, were chosen. The secretion of a newly introduced luciferase with both of these signal peptides was investigated for 8 clones each. No detectable secretion for the IgKappa, but a high secretion rate for the SERK signal peptide was shown. <A href="https://2013.igem.org/Team:TU-Munich/Results/Localization#Secretion_via_SERK_signal_peptide_.28SERK-SigP.29" class="external text" rel="nofollow">Successful secretion could be achieved using the SERK signal peptide</A>. Because the secreted effector proteins are not attached to the moss cell, they diffuse into the water, which could be suboptimal, for example if you want to remove pollutants by simply binding them. Therefore we designed <A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation#Membrane-bound_Expression" class="external text" rel="nofollow">a modular receptor</A> for <I>Physcomitrella</I>, which can carry effector proteins at the outer side of the cell membrane. For this purpose <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions" class="external text" rel="nofollow">bioinformatics methods</A> were used again and the SERK transmembrane domain was chosen to be the best one. A receptor composed of (1) the SERK signal peptide, (2) an extracellularly located effector protein, (3) a linker with a Strep-tag II and a TEV protease cleavage site, (4) the SERK transmembrane domain, (5) a short linker domain and (6) a green fluorescent protein were assembled using the RFC[25] standard. This highly modular receptor was successfully transformed into <I>Physcomitrella patens</I> and stable cell lines were selected. These stable cell lines were used for experiments. The localization of the membrane-bound GFP could be detected clearly on the surface of the moss cells (see figure 1), whereas cells expressing GFP cytosolically in the moss, showed a uniform fluorescence all over. Further we incubated the moss cells with recombinant TEV protease, which diffused through the cell wall, cleaved the TEV site within the extracellular domain of the receptor and liberated the NanoLuc luciferase. The luciferase assay of the supernatant at the beginning of this incubation and after 16 hours showed a dramatic increase in luminescence, which is further evidence that our modular receptor is located in the membrane and - even better - in the right orientation, exposed to the extracellular space. Beyond the possibility to locate an effector protein in the extracellular space, we thought about further applications and found the <A href="https://2013.igem.org/Team:TU-Munich/Results/Localization#SpyCatcher_.26_SpyTag" class="external text" rel="nofollow">SypCatcher-SpyTag System</A> to be a perfect tool for our needs [<A href="http://www.ncbi.nlm.nih.gov/pubmed/22366317" class="external text" rel="nofollow">Zakeri et al., 2012</A>]. In this system a peptide bond is formed between the side chains of two protein domains in an efficient manner. With this system it is not necessary to fuse effector proteins to a specific terminus of the receptor any more, it even becomes possible to immobilize effector proteins which are only active as multimeric proteins and it would also be feasible to express a single receptor carrying a SpyCatcher domain at the outer side of the membrane which subsequently binds a set of different effector proteins which are secreted and are immobilized afterwards. Constructs were created with a His-tagged SypCatcher, a SypTag with an N-terminal or C-terminal SpyTag or with protein domains on both termini of the SpyTag. These constructs were produced recombinantly in <I>E. coli</I>. Afterwards the proteins were purified. Protein coupling experiments were performed and the formation of isopeptide bonds was confirmed by pull-down experiments and reducing SDS-PAGE. Summarizing these results, all our intended localizations in <I>Physcomitrella patens</I> worked, empowering the iGEM community to work creatively with <I>Phycomitrella patens</I> as a new chassis.
</P><H2><SPAN class="mw-headline" id="BioDegradation">BioDegradation</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2</B>: Degradation of erythromycin by recombinant protein and our PhyscoFilter <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss#The_PhyscoFilter_for_Erythromycin" class="external text" rel="nofollow">read more</A>.</DIV></DIV></DIV><P>Under the headline  <A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation" class="external text" rel="nofollow">BioDegradation</A> we investigated effector proteins which degrade pollutants by enzymatic catalysis. For this purpose we introduced the new BioBrick <B>Erythromycin Esterase (EreB)</B> which degrades macrolide antibiotics. Additionally we used well established BioBricks for BioDegradation. We improved the Laccase from <I>Bacillus pumilus</I>, established by the iGEM Team of Bielefeld in 2012, by converting it to RFC[25] in order to be able to fuse it to the extracellular domain of our receptor. Both enzymes were produced as recombinant proteins and purified. Enzymatic characterization was carried out concerning salt tolerance, substrate and pH dependence. For the laccase temperature dependence and the half-life in river water was additionally estimated. All data was fitted in our enzyme kinetic modeling. The aim was to analyze this data and to provide a solid base for our filter calculator. This calculator uses all the data produced in our enzyme characterization to extrapolate from them for the large scale implementation of transgenic PhyscoFilters in waste water treatment plants or on rivers. We assumed the secretory production of laccase by our moss as the laccase degrades a wide variety of important pollutants such as the pain killer diclofenac, the oral contraceptive ethinylestradiol or iodined x-ray contrast media. All these substances are present in nature and are hardly degradable by conventional methods. While using <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter" class="external text" rel="nofollow">this calculator for modeling</A>, we learned that factors such as the degree of pollution in rivers, the average temperature in a specific country, the enzyme half-life time, as well as the actual amount of secreted protein play an important role for the efficacy of our PhyscoFilter. Generally the results show that approximately an area of 20 football fields would be required to produce enough laccase to reduce the contamination of the large river Rhein with the mentioned pharmaceutical compounds, while half a field would suffice for a large waste water treatment plant. Beside this result we also produced several different stable transgenic moss lines for our BioDegradation module and were able to show that our cytoplasmically expressed Erythromycin Esterase B enables our moss to successfully degrade the macrolide antibiotic Erythromycin. This experiment was analyzed by using liquid chromatography coupled to mass spectrometry (LC-MS). It worked for the recombinant protein as well as for the transgenic plant giving the <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss" class="external text" rel="nofollow"><B>proof of principle</B> for <I>Physcomitrella patens</I> as a bioremediation organism</A>.
</P><H2><SPAN class="mw-headline" id="BioAccumulation">BioAccumulation</SPAN></H2><P>Beside the enzymatic degradation of pollutants we found different methods to bind them on our moss filter. We called this system  <A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation" class="external text" rel="nofollow">BioAccumulation</A>. The most obvious idea was to use binding proteins which were developed for human therapy, such as antibodies, for example. To investigate this idea we used an alternative binding protein, the Anticalin, which was engineered to bind fluorescein as it has a very high affinity, a small size and a robust fold [<A href="http://www.ncbi.nlm.nih.gov/pubmed/16307475" class="external text" rel="nofollow">Vopel et al., 2005</A>]. Beside this engineered binding protein, we also recognized the idea of <A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations#Collaboration%20with%20Dundee%20iGEM%20team%202013" class="external text" rel="nofollow">Dundee iGEM 2013</A> which sounded quite interesting. This team used a protein which is found in the toxicity mechanism of microcysteins. Dundee wanted to use it as a binding partner to absorb the pollutant from the water. Thus we contacted Dundee iGEM, discussed our idea for a <A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations#Collaboration%20with%20Dundee%20iGEM%20team%202013" class="external text" rel="nofollow">collaboration</A> with them, got their BioBrick, converted it to RFC[25], assembled it into our modular receptor and finally transformed and selected stable transfected moss lines which we finally characterized. Basically the limitation of BioAccumulation applications is that binding proteins are only able to bind one pollutant per binding protein. Due to that fact an extremely high number of binding  proteins is required to achieve a recognizable reduction of environmental pollutants. We transformed and selected transgenic moss lines with all three effector proteins and checked the cellular localization of these proteins using fluorescence microscopy. We were able to <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss#The_PhyscoFilter_for_Fluorescein" class="external text" rel="nofollow">confirm the activity</A> of Moss lines, containing receptor bound Anticalins, which are able to bind fluorescein, whereas moss lines with a receptor, carrying the protein phosphatase 1 (PP1) from our collaboration partner Dundee showed the  <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss#The_PhyscoFilter_for_Microcystin" class="external text" rel="nofollow">localization of our Gene-construct in cytosolic vesicles</A> and not on the membrane, as expected. This observation is in good agreement with the results presented by Dundee, that the SEC-pathway secretion is not working for this BioBrick. This observation may be explained due to surface exposed cystein residues which tend to aggregate in the oxidizing milieu. Therefore it would be necessary to perform protein engineering to exchange these cystein residues for other amino acid residues in order to increase the stability of this protein.
</P><H2><SPAN class="mw-headline" id="Kill-Switch">Kill-Switch</SPAN></H2><P>Safety is one of the most important issues in synthetic biology. Therefore we implemented a kill switch into our project. For us, it was important to use a trigger, which is ubiquitously present in the environment and absolutely autonomous. We therefore developed a <A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch" class="external text" rel="nofollow">light-triggered kill switch</A>. With this system, the transgenic moss could be cultivated under blue filter foil. As long as the moss grows under this blue foil no red light can be absorbed by the moss, so it stays alive. As soon as the moss escapes from this protected environment the red light is present and the kill switch is activated. The system was modularized into a sensor module and a suicide module. The sensor domain consists of a split TEV protease which is attached to either the proteins PhyB or PIF. These two proteins dimerize when red light is present and therefore lead to the dimerization of both split TEV subunits, so the reconstitution of the TEV protease is initiated. The suicide module consists of  a nuclease which is anchored to the intracellular membrane by a linker which contains a nuclear localization signal (NLS) and a TEV cleavage site. As soon as the sensor module is reconstituted by red light, the TEV protease cleaves the cleavage site at the membrane localized nuclease. Thereby the nuclease is liberated, is transported to the nucleus due to the nuclear localization signal and, in the end, fragments the genome. The choice to use a nuclease instead of siRNA was for example driven by our <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch" class="external text" rel="nofollow">modeling approach</A> in which we found the siRNA suicide module to be less effective as there is a negative feedback loop which impedes the efficient killing of moss cells. 
We have transformed moss cells with this kill switch and have protected the resulting growing cells, as mentioned before, by blue foil. However when we opened the blue foil after the selection process, all moss cells were dead. This can be explained by a drastically reduced transformation efficiency as the kill switch DNA was &gt;10 kDa or by the fact that the kill switch is reliably killing the cells even without a trigger. In order to test the sensor module in vitro, we have produced the two fusion proteins in vitro as recombinant proteins and have attempted to purify them, which was not successful as the proteins are probably not stable in vitro. Although we only had a single shot to test our kill switch in <I>Physcomitrella patens</I>, we have discussed this system far more than the other parts of our project that worked very well in first experiments. During these discussions on our kill switch we learnt a lot about this system and we described these findings in order to help subsequent iGEM teams who aim to design a reliable and safe kill switch.
</P><H2><SPAN class="mw-headline" id="Implementation">Implementation</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3</B>: Remediation rafts in front of the MIT</DIV></DIV></DIV><P>Projects in iGEM must not stop at the lab door and therefore it is immensely important to think about technical solutions to implement the transgenic organisms in order to show highest efficacy and safety. For this reason we convinced experts like Prof. Dr. Posten to join our <A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews#Expert_Counsel:_An_overview" class="external text" rel="nofollow">advisory board</A> and have evaluated <A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation" class="external text" rel="nofollow">different cultivation methods</A> for moss such as closed tube reactors, open pond reactors and floating remediation rafts. We came to the conclusion that in the case of immobilized effector proteins an open pond or closed tube reactor will be the superior technology as the degradation requires a large area of contact between the moss and the pollutant to be degraded. As a second possibility we evaluated the secretion of effector proteins such as the laccase, which would then be implemented best on floating <A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation#Our_swimming_remediation_raft" class="external text" rel="nofollow">remediation rafts</A> which are cheap to produce, mobile and could also be applied in third world countries with highly contaminated waters. PhyscoFilter moss could be grown on these rafts and would secrete recombinant protein into the water to degrade pollutants in the environment. For all these cultivation methods we built model reactors, tried the cultivation of moss within them and tested the flow characteristics of the systems. For the triangular remediation raft we constructed a life-size prototype which costed only US$ 50. Additionally we developed a measurement device based on an <A href="https://2013.igem.org/Team:TU-Munich/Results/How_To#Setting_up_a_basic_Arduino_measuring_device" class="external text" rel="nofollow">Arduino microcontroller</A> which measures environmental parameters, sends the data via WiFi to a webserver from where the actual data can be monitored with any smart phone or computer at any place in the world. To get an idea how such remediation rafts could look like on our rivers, we <A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations#Collaboration_with_Op.N" class="external text" rel="nofollow">talked to architects</A> and also rendered an 3D-CAD-model in front of the MIT (see figureÂ 3).
</P><H2><SPAN class="mw-headline" id="Supporting_the_iGEM_Community">Supporting the iGEM Community</SPAN></H2><P>It is really amazing to see how the iGEM community advanced over the last years. We can proudly say that we invested efforts to take the iGEM community to another level. We created a software tool which translates protein coding BioBricks in the registry to amino acid sequences, calculates various parameters and compiles alignments with data from various data banks. In the end all collected information for a BioBrick is exported into a standardized table which can easily be integrated into the part descriptions of BioBricks. We submitted a RFC for the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">AutoAnnotator</A> which obtained the number RFC[96]. Besides this software toll we have written tutorials on wiki programming, creating animated gifs of protein structures and the usage of Arduino microcontrollers for future iGEM projects.
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/19021876" class="external text" rel="nofollow">Gitzinger et al., 2009</A>] Functional cross-kingdom conservation of mammalian and moss (Physcomitrella patens) transcription, translation and secretion machineries, Plant Biotechnol J. 2009 Jan;7(1):73-86
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/16307475" class="external text" rel="nofollow">Vopel et al., 2005</A>] Vopel S, MÃ¼hlbach H, Skerra A. (2005) Rational engineering of a fluorescein-binding anticalin for improved ligand affinity. <I>Biol. Chem.</I>, 386(11):1097-104.
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/22366317" class="external text" rel="nofollow">Zakeri et al., 2012</A>] Zakeri B, Fierer JO, Celik E, Chittock EC, Schwarz-Linek U, Moy VT, Howarth M. (2012). Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. <I>Proc Natl Acad Sci U S A</I>. 20;109(12)</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Results/Summary">http://2013.igem.org/Team:TU-Munich/Results/Summary</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Results/Summary" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Results/Summary" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Results/Summary&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Results/Summary&amp;oldid=361741" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>