<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Project_Killswitch"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Project/Killswitch">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Project/Killswitch&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Project/Killswitch&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Project/Killswitch&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Project/Killswitch" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Project/Killswitch</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="A_novel_mechanism_preventing_uncontrolled_spread_of_transgenic_plants"> A novel mechanism preventing uncontrolled spread of transgenic plants </SPAN></H2><P>Working on plants is uncomplicated since as photoautotrophic organisms they can provide their own energy. So creating a photosensitive plant might seem silly at first glance. &quot;Crazy, stupid Germans!&quot;, you might think, but wait, there's  more! Green biotechnology does not have an easy stand in Germany due to the &quot;German Angst&quot; of uncontrolled spreading of transgenic plants. We therefore see it as our task and duty to meet the required high safety standards to minimize these risks for a maximum of biosafety. We created a plant that can only survive in a well-defined environment. Plants do not neccessarily need the whole spectrum of light to supply themselves with energy, so reassigning part of the spectrum to other purposes is possible. Shielded from red light by filters, the moss survives without compromising vitality or growth. Unintended release of our protected environment leads to the activation of a lethal process of no return which kills the moss.
</P><H2><SPAN class="mw-headline" id="How_our_red_light_triggered_nuclease_system_for_self-destruction_works"> How our red light triggered nuclease system for self-destruction works </SPAN></H2><H3><SPAN class="mw-headline" id="Choosing_the_effector_that_triggers_cell-death_quickly_and_safely"> Choosing the effector that triggers cell-death quickly and safely </SPAN></H3><DIV class="box-definition"><P><B>Programmed Cell-Death</B> (<B>PCD</B>) is a endogenous mechanism of the cell itself for controlled suicide. It is normally carried out e. g. upon tissue differentiation or due to a infection with a pathogen to prevent its propagiation.
</P><P><B>Micococcal nuclease</B> (also called: <B>Thermonuclease, MNase</B>) is a non-specific endo-exonuclease from <I>Staphylococcus aureus</I> which shows activity in digesting single-stranded DNA/RNA and also double-stranded DNA/RNA.
</P><P><B>Nuclear Localization Signal/Sequence</B> (<B>NLS</B>) is a short amino acid sequence of a nuclear targeted protein that is recognized by nuclear transporters, the so-called importins, and then is actively transported in the nucleus. NLS from SV40 large T antigen is mostly used to mark synthetic proteins for nuclear localization.
</P></DIV><P>In plants, the regulation and mechanism of <B>programmed cell-death</B> (<B>PCD</B>) is not yet fully understood, but analoguely to all other eukaryotic organisms it is known that the fragmentation of the genomic DNA inevitably leads to <B>PCD</B>.
</P><P>We chose the <B>Micrococcal nuclease</B> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159105" class="external text" rel="nofollow">BBa_K1159105</A>) as effector to destroy the integrity of the genomic DNA. Since the genome is protected by the nuclear envelope, it was necessary to tag the nuclease with a <B>Nuclear Localization Signal (NLS)</B> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159109" class="external text" rel="nofollow">BBa_K1159109</A>). Nucleases show their death-bringing activity already at low concentrations. It was therefore essential to put the kill-switch under tight spatio-temporal control and induce activity under specific conditions only.
</P><H3><SPAN class="mw-headline" id="Preventing_the_nuclease_from_developing_its_deadly_potential_under_well-defined_conditions"> Preventing the nuclease from developing its deadly potential under well-defined conditions </SPAN></H3><DIV class="box-definition"><P><B>Transmembrane Domain</B> (<B>TMD</B>) is the region of a membrane protein which traverses the plasma membrane.
</P><P><B>TEV Protease</B> is a site-specific cystein protease from the C4 peptidase family from Tabacco Etch Virus (TEV). Its typical recognition site is ENLYFQ(G/S).
</P></DIV><P>We adress the problem by anchoring the nuclease in the cytoplasmatic membrane under basal conditions. In its membrane-associated form (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159111" class="external text" rel="nofollow">BBa_K1159111</A>), the nuclease cannot evolve its deadly potential as it is spatially seperated from its site of action: In eukaryotic organisms the genomic DNA is subcellular located in the nucleus, sheltered by the nuclear envelope.
</P><P>To make the nuclease releasable, we designed a long linker with a TEV recognition/cleavage site between the <B>transmemmbrane domain (TMD)</B> and the NLS-tagged nuclease. In the presence of <B>TEV protease</B> activity, the nuclease is liberated from the membrane and translocates into the nucleus due to its <B>NLS</B> tag. There, it can fulfill its task as the executor of <B>PCD</B> by degrading the DNA with its endo-exonuclease activity.
</P><P>The question is how to enable the lethal activity of the nuclease upon red light illumination? The first component consists a fusion protein composed of a <B>signal peptide plus transmembrane region</B> of <I>Physcomitrella patens</I> (see <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions#Prediction_of_Signal_Peptides" class="external text" rel="nofollow">here</A> and <A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions#Prediction_of_Transmembrane_Regions" class="external text" rel="nofollow">here</A>) for <B>membrane localization</B>, a <B>TEV cleavage site</B> for cleaving off the nuclease upon red light exposition and nuclear localizaton signal plus the nuclease itself as the death bringing part.
</P><H3><SPAN class="mw-headline" id="Red_light_triggered_reconstitution_of_split_TEV_Protease"> Red light triggered reconstitution of split TEV Protease </SPAN></H3><DIV class="box-definition"><P><B>Split TEV Protease</B> is a split version of the TEV Protease which N- and C-terminal split parts shows no enzymatic activity. Bringing them to physical proximity leads to reconstitution of the cleavage activity.
</P><P><B>Phytochrome B</B> (<B>PhyB</B>) is a plant photoreceptor that exists in two interconvertible forms P<SUB>r</SUB> and P<SUB>fr</SUB>. In its P<SUB>fr</SUB> (far red light sensititve) form upon red light illumination, PhyB interacts with Phytochrome Interacting Factors, e.g. PIF3 or PIF6.
</P><P><B>Phytochrome Interacting Factor 3/6</B> (<B>PIF3/PIF6</B>) are both transcription factors with basic helix-loop-helix (bHLH) motifs that that bind to <B>PhyB</B> upon red light exposition
</P></DIV><P>As mentioned, the nuclease is normally localized at the cytoplasmatic membrane and by cleavage through a TEV protease, the NLS-tagged nuclease is released from its cage and goes directly in the nucleus.
</P><P>To liberate the nuclease after exposition to red light, we designed two fusion proteins; each of them contains either the N-terminal or the C-terminal splitted part of the TEV protease. The splitted TEV protease is not active as long as the catalytic residues are seperated. The first fusion protein has PhyB as its fusion partner which heterodimerizes with PIF3/PIF6, the fusion partner of the second fusion protein. Red light induces heterodimerization of PhyB with PIF3/PIF6 and thus reconstitutes the N- and C-terminal half of the TEV Protease resulting in a proteolytic activity for a specific TEV recognition site.
</P><P>The reconstituted TEV Protease libarates the nuclease from its membrane anchor by cleaving it off from the transmembrane domain. The nuclear localization signal ensures nuclear translocation of the nuclease.
</P><P>Reaching the nucleus, the nuclease can unfold its full deadly impact by digest the genomic DNA in fragments through its endo-exonuclease activity. Genomic fragmentation automatically triggeres endogenous programmed cell-death.
</P><H3><SPAN class="mw-headline" id="Key_features_of_the_system"> Key features of the system </SPAN></H3><P>During the design process of the kill-switch we developed a set of key features that we thought to be essential for the efficient activation of programmed cell death.
</P><UL><LI><B>Signal transduction solely on protein level:</B> By excluding transcriptional/translational transduction from the system we created a system that can assure PCD with tight temporal control. This design eliminates the risk of a negative feedback loop that would appear when transcriptional/translational efficiency is hampered due to lowered cell vitality. See: Modeling.
</LI></UL><UL><LI><B>Strong lethal activity</B>: By destroying the genetic material of a cell, the nuclease ensures a quick, non-reversible, inevitable programmed cell death.
</LI></UL><UL><LI><B>Cheap, non-toxic, automatic induction</B>: Activation by light eradicates the need for expensive, possibly toxic inductors and even of any human interaction to prevent the release of genetically modified moss.
</LI></UL><H2><SPAN class="mw-headline" id="Design_Notes"> Design Notes </SPAN></H2><H3><SPAN class="mw-headline" id="Releasable_Membrane-associated_Endo-exonuclease"> Releasable Membrane-associated Endo-exonuclease </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1</B>: From N- to C-terminus: (SERK signal peptide)_(SERK transmembrane region)_(long flexible linker with TEV-site)_(SV40 NLS)_(micrococcal nuclease)</DIV></DIV></DIV><P>For our membrane bound nuclease we have to choose a nuclease with following characteristics:
</P><UL><LI> Our nuclease should digest any DNA wihtout any preferences for recognitions sites
</LI></UL><UL><LI> Furthermore, that nuclease should not have disulfide bonds as they cannot be formed outside in the cytoplasmatic compartment.
</LI></UL><P>Micrococcal nuclease (<A href="http://parts.igem.org/Part:BBa_K1159105" class="external text" rel="nofollow">BBa_K1159105</A>) (alias S7 Nuclease, MNase, Thermonuclease) from <I>Staphylococcus aureus</I> was chosen as our death bringing executor as it fulfills our requirements (see above). It shows endo-exonuclease activity for DNA and does not contain any disulfid bonds.
</P><P>To anchor the nuclease in the cytoplasmatic membrane, we use the TMD (<A href="http://parts.igem.org/Part:BBa_K1159315" class="external text" rel="nofollow">BBa_K1159315</A>) and the N-terminal signal peptide (<A href="http://parts.igem.org/Part:BBa_K1159303" class="external text" rel="nofollow">BBa_K1159303</A>) from the chassis <I>Physcomitrella patens</I> itself.
</P><P>To make the nuclease releasable from the membrane in the presence of TEV protease activity, we designed a long linker with a TEV recognition and cleavage site. We designed a long linker (GlyGlyGlyGlySer)x5 with TEV cleavage site at the C-terminal end of the linker (<A href="http://parts.igem.org/Part:BBa_K1159310" class="external text" rel="nofollow">BBa_K1159310</A>).
</P><P>After beeing released from the membrane cage through the cleavage by TEV proteolysis, the nuclease has be guided to its site of action, the nucleus. For this purpose, we are using the NLS from SV40 (<A href="http://parts.igem.org/Part:BBa_K801030" class="external text" rel="nofollow">BBa_K801030</A>).
</P><H3><SPAN class="mw-headline" id="Red_light_triggered_C-terminal_Split-TEV_Protease"> Red light triggered C-terminal Split-TEV Protease </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2</B>: From N- to C-terminus: (First 908 AA of PhyB)_(36 AA long flexible linker)_(C-terminal half of TEV Protease)</DIV></DIV></DIV><P>For the first part of the light-induced reassembling of split TEV protease, we use the first 908 N-terminal amino acids (<A href="http://parts.igem.org/Part:BBa_K801031" class="external text" rel="nofollow">BBa_K801031</A>) for the reversible heterodimerization with PIF3/PIF6. It has been reported that the first 650 N-terminal amino acids indeed are sufficient for photoconversion of P<SUB>r</SUB> (inactive form) to P<SUB>fr</SUB> but are irreversible, so we take the first 908 N-terminal amino acids, which has been proven to  be reversible by illumination with far red light. For its function as a light-switchable protein, PhyB needs a essential chromophore, phycocyanobilin (PCB) or phytochromobilin (PPhiB), which is autocatycally incorporated. The latter chromophore phytochromobilin is endogenous avaible in <I>Physcomitrella patens</I>, so we do not have to implement the biosynthesis pathway for one of the chromophores.
</P><P>We have also first to consider how the fusion parts can be fused together and which parts work as N- or C-parts. It has been reported that PhyB should be used as N-part. The first 100 amino acids of PIF3 (<A href="http://parts.igem.org/Part:BBa_K1159103" class="external text" rel="nofollow">BBa_K1159103</A>)/PIF6 (<A href="http://parts.igem.org/Part:BBa_K1159104" class="external text" rel="nofollow">BBa_K1159104</A>) on the other side can either be used at the N- and C-terminus. The N-terminal half of TEV protease also can be used as N- or C-terminal fusion part. On the other side the C-terminal half of the TEV protease should be used only as C-terminal fusion protein because the C-terminus is in the proximity to the catalytic site.
</P><P>Another important question is to choose the length that links the light-associated proteins with the split-TEV protease halfs together. After looking at the predicted structure of PhyB (I-TASSER and HHpred) and the crystal structure of the TEV protease we decided to use a 36 amino acids long flexible linker (GSAT linker, <A href="http://parts.igem.org/Part:BBa_K243029" class="external text" rel="nofollow">BBa_K243029</A>) to allow the splitted halfs of the TEV protease to reassemble with each other.
</P><H3><SPAN class="mw-headline" id="Red_light_triggered_N-terminal_Split-TEV_Protease"> Red light triggered N-terminal Split-TEV Protease </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 3</B>: From N- to C-terminus: (C-terminal half of TEV Protease)_(36 AA long flexible linker)_(First 100 AA of PIF3 or PIF6)</DIV></DIV></DIV><P>For the second part of the light triggered reassembling of the splitted TEV Protease, we use PIF3 (<A href="http://parts.igem.org/Part:BBa_K1159103" class="external text" rel="nofollow">BBa_K1159103</A>) and PIF6 (<A href="http://parts.igem.org/Part:BBa_K1159104" class="external text" rel="nofollow">BBa_K1159104</A>) as the protein part that is recognized and bound by PhyB upon red light irradiation. Literature shows that PIF6 is bound with a higher affnity than PIF3 by PhyB.
</P><P>To link the N-terminal half of the TEV Protease (<A href="http://parts.igem.org/Part:BBa_K1159101" class="external text" rel="nofollow">BBa_K1159101</A>) to PIF3/PIF6, we also choose the same linker (GSAT linker, <A href="http://parts.igem.org/Part:BBa_K243029" class="external text" rel="nofollow">BBa_K243029</A>) as we used it for the linkage of PhyB with the C-terminal half of TEV Protease.
</P><H3><SPAN class="mw-headline" id="FRET_Reporter_for_TEV_Protease_activity"> FRET Reporter for TEV Protease activity </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4</B>: FRET between eCFP and SYFP2 as a result of excitation of eCFP. After cleavage during TEV Protease activity, eCFP is not anymore quenched by SYFP2 and can fluoresce at its maximum intensity.</DIV></DIV></DIV><P>To prove that our light-switchable TEV Protease is actually working, we designed a FRET reporter (<A href="http://parts.igem.org/Part:BBa_K1159112" class="external text" rel="nofollow">BBa_K1159112</A>). The FRET reporter is composed of two chromoproteins, namely eCFP (<A href="http://parts.igem.org/Part:BBa_K1159302" class="external text" rel="nofollow">BBa_K1159302</A>) and SYFP2 (<A href="http://parts.igem.org/Part:BBa_K1159301" class="external text" rel="nofollow">BBa_K1159301</A>), which are linked together through a short linker with an internal TEV cleavage site (<A href="http://parts.igem.org/Part:BBa_K1159318" class="external text" rel="nofollow">BBa_K1159318</A>).
</P><P>As a functional FRET pair, eCFP emits its energy after irradiation through Förster Resonance Energy Transfer (FRET) to SYFP2 due to the overlap of the emitting spectra of eCFP and the excitation spectra of SYFP2.
</P><P>In the presence of TEV Protease activity, the TEV cleavage site is recognized and processed, resulting in the seperation of the FRET pair eCFP and SYFP2. Thus, we will observe a increase of the emission spectra of eCFP and a decrease of the emission spectra of SYFP2.
</P><H3><SPAN class="mw-headline" id="Polycistronic_expression_in_Eukaryotes"> Polycistronic expression in Eukaryotes </SPAN></H3><P>For creating the red light sensitive kill switch, we have to constitutively express 3 fusion proteins: 
2 proteins for the <B>light switchable TEV Protease</B> and 1 for the <B>membrane-anchored nuclease</B>. 
</P><P>The classical approach would be to flank the parts with an upstream promoter and a downstream terminator. To avoid this inconvenient complication, we decided to use the 5'UTR of Poliovirus 1 Mahoney strain containing a <B>cap-independent Internal Ribosome Entry Site</B> (<A href="http://parts.igem.org/Part:BBa_K1159300" class="external text" rel="nofollow">BBa_K1159300</A>) for <B>polycistronic expression</B> of our proteins. This strategy enables us to put multiple CDSs under the control of only one promoter and one terminator, by simply inserting the IRES in between.
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/1896431" class="external text" rel="nofollow">Hynes et al., 1991</A>] Hynes TR, Fox RO (1991). The crystal structure of staphylococcal nuclease refined at 1.7 A resolution. <I>Proteins</I>. 10(2):92-105
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/6020427" class="external text" rel="nofollow">Heins et al., 1967</A>] Heins JN, Suriano JR, Taniuchi H, Anfinsen CB. (1967). Characterization of a nuclease produced by Staphylococcus aureus. <I>J Biol Chem.</I>, 242(5):1016-20
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/19749742" class="external text" rel="nofollow">Levskaya et al., 2009</A>] Levskaya, A., Weiner, O. D., Lim, W. A., and Voigt, C. A. (2009). Spatiotemporal control of cell signalling using a light-switchable protein interaction. <I>Nature</I>, 461(7266):997–1001.
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/12355112" class="external text" rel="nofollow">Mendelsohn, 2002</A>] Mendelsohn, A. R. (2002). An enlightened genetic switch. <I>Nat Biotechnol</I>, 20(10):985–7
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/12219076" class="external text" rel="nofollow">Shimizu-Sato et al., 2002</A>] Shimizu-Sato, S., Huq, E., Tepperman, J. M., and Quail, P. H. (2002). A light-switchable gene promoter system. <I>Nat Biotechnol</I>, 20(10):1041–4
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/17072307" class="external text" rel="nofollow">Wehr MC et al., 2006</A>] Wehr MC, Laage R, Bolz U, Fischer TM, Grünewald S, Scheek S, Bach A, Nave KA, Rossner MJ (2006). Monitoring regulated protein-protein interactions using split TEV. <I>Nat Methods</I>, 3(12):985-93
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/19021876" class="external text" rel="nofollow">Gitzinger et al., 2009</A>] Gitzinger M, Parsons J, Reski R, Fussenegger M (2009). Functional cross-kingdom conservation of mammalian and moss (Physcomitrella patens) transcription, translation and secretion machineries. <I>Plant Biotechnol J</I>. 7(1):73-86.
</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Project/Killswitch">http://2013.igem.org/Team:TU-Munich/Project/Killswitch</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Project/Killswitch" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Project/Killswitch" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Project/Killswitch&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Project/Killswitch&amp;oldid=356119" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>