<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Team_Attributions"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Team/Attributions">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Team/Attributions&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Team/Attributions&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Team/Attributions&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Team/Attributions" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Team/Attributions</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="iGEM_Team_TU-Munich_2013"> iGEM Team TU-Munich 2013 </SPAN></H2><H3><SPAN class="mw-headline" id="Experimental_measurements">Experimental measurements</SPAN></H3><P>All experiments and measurements were conducted by student members of the <A href="https://2013.igem.org/Team:TU-Munich/Team/Members" class="external text" rel="nofollow">iGEM Team TU-Munich 2013</A>. The team could build on the knowledge of Jeffery Truong, Ingmar Polte and Katrin Fischer who had already participated in last year's competition and already knew the most important techniques and the laboratory. They implemented the lab management from last year from the first moment on, such as our extensive <A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal" class="external text" rel="nofollow">Labjournal</A>.  For the first usage of instruments we obtained an introduction into the lab techniques by an instructor. Measurement itself and the evaluation of the obtained data were done by us.
</P><H4><SPAN class="mw-headline" id="LC-MS_Measurements">LC-MS Measurements</SPAN></H4><P>The LC-MS measurements were performed to confirm the successful degradation of pollutants by effector proteins which were produced recombinantly or which were expressed by transgenic moss plants. Here we contacted Prof. Dr. Thomas Hofmann, who already helped us in our last year's iGEM project with the detection of caffeine and xanthohumole. The responsible team member for the LC-MS was in both years Ingmar Polte who performed the degradation experiments, established the contact to Prof. Hofmann and brought the samples to the <A href="http://www.molekulare-sensorik.de" class="external text" rel="nofollow">Chair for Molecular Sensory</A>. At the chair, the student members Ingmar Polte or Andreas-David Brunner contacted the operator of the mass spectrometer (Mr. Friederich or Dr. Stark) with whose they did the sample preparation, the actual measurements and the data evaluation.
</P><H4><SPAN class="mw-headline" id="ESI-TOF_Measurements">ESI-TOF Measurements</SPAN></H4><P>In order to confirm the correctness of recombinant proteins they were analyzed in the ESI-TOF mass spectrometer, which is present at our hosting laboratory. For this purpose a student team member did the sample preparation, calculated the theoretical mass using the <B><A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">AutoAnnotator</A></B> and brought the samples to Andreas Reichert who is a doctoral student at the chair and is responsible for the ESI-TOF mass spectrometer. He measured the samples together with a student team member and showed us the deconvolution of the primary data.
</P><H4><SPAN class="mw-headline" id="Fluorescense_Microscopy">Fluorescense Microscopy</SPAN></H4><P>All fluorescense microscopy experiments were performed at the <A href="http://plantdev.bio.wzw.tum.de/index.php?id=36" class="external text" rel="nofollow">Chair for Plant Developmental Biology</A> (Prof. Dr. Schneitz) as plants (and especially <I>P. patens</I>) exhibit a strong autofluorescence caused by the photosystem. Therefore an advanced microscope with appropriate filters is absolutely necessary which we could use at the Chair for Plant Developmental Biology. The experiments were performed by our student member <A href="https://2013.igem.org/Team:TU-Munich/Team/Members" class="external text" rel="nofollow">Dong-Jiunn Jeffery Truong</A> who has worked during his bachelor thesis with fluorescense microscopy over several weeks and was therefore an essential experimentator for this part of the project. An introduction to the microscopes was given by Dr. Prasad Vaddepalli.
</P><H4><SPAN class="mw-headline" id="Transformation_of_Physcomitrella_patens">Transformation of <I>Physcomitrella patens</I></SPAN></H4><P>The transformation of <I>Physcomitrella patens</I> was one of the main concerns when we thought about working with <I>Physcomitrella</I> as it might cause some trouble which can only be handled by using the appropriate equipment and the advises of an experienced researcher. Therefore we contacted <B>Prof. Dr. Reski</B> (Freiburg University, 350 km distance) in May and could win him as an advisor for our team. After traveling several times to Freiburg we could perform the <B>transformations</B> in his lab. For this purpose we brought all buffers and the sterile linearized DNA from Munich and performed the transformation under the instructions of Dr. Gertrud Wiedemann. All steps of the transformation were conducted by student team members as it can be seen in our <A href="https://2013.igem.org/Team:TU-Munich/Results/Moss#3._Transformation_of_Physcomitrella_patens" class="external text" rel="nofollow">Transformation Results</A> section.
</P><H4><SPAN class="mw-headline" id="Outreach">Outreach</SPAN></H4><P>The <A href="http://vimeo.com/76195786" class="external text" rel="nofollow">introduction video for our team</A> created during this competition, was designed, produced and cut by the student team member Katrin Fischer.
The group photos of our team were created by the press office of the TUM.
</P><H2><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Skerra"> Laboratory of Prof. Dr. Skerra </SPAN></H2><P>The research group at the <A href="http://biologische-chemie.userweb.mwn.de/index.html" class="external text" rel="nofollow">Chair of Biological Chemistry</A> at TUM works in the field of biochemical protein engineering and design with a focus on therapeutic proteins and their application. The three main fields of research are (1) the development of <B>anticalins</B> which is an alternative binding scaffold and which are a promising alternative for conventional antibodies (2) the <B>extension</B> of the <B>plasma half-life</B> of therapeutic proteins using a poly-peptide polymer and (3) the <B>site-specific conjugation</B> of therapeutic proteins. Thus the focus of our hosting laboratory is biomedical engineering whereas our iGEM topic shows no intersection with this topic. 
The single negligible exception is that we used a <B>higher engineered anticalin</B> (&lt;partinfo&gt;K1159003&lt;/partinfo&gt;) to bind fluorescein in our <A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation" class="external text" rel="nofollow">BioAccumulation</A> subproject, compared to the conventional BioBrick (&lt;partinfo&gt;BBa_K157004&lt;/partinfo&gt;). This higher engineered version has three additional amino acid exchanges and exhibits a 75-fold higher affinity for its binding partner (fluorescein). It was not essential to have this higher engineered version for our project but we wanted to supply the registry with this improved part.
Beside this plasmid, we also obtained a plasmid for the TEV protease from the chair which we have used to generate the Split-TEV protease (<A href="http://parts.igem.org/Part:BBa_K1159100" class="external text" rel="nofollow">BBa_K1159100</A>, <A href="http://parts.igem.org/Part:BBa_K1159101" class="external text" rel="nofollow">BBa_K1159101</A> and <A href="http://parts.igem.org/Part:BBa_K1159102" class="external text" rel="nofollow">BBa_K1159102</A>).
</P><P><I>Prof. Skerra kindly provided us with <B>space in his laboratory</B>, and <B>generously advanced us money</B> to pay for team registration, travel expenses and laboratory resources. Moreover he <B>participated approximately one a month in our team meetings</B> and advised us on our project.</I></P><H2><SPAN class="mw-headline" id="Technical_University_Munich"> Technical University Munich </SPAN></H2><P>As we had to cover several different scientific aspects during our project we contacted several professors from our university to get their opinion on our plans, obtain reagents or access to measuring instruments such as mass spectrometers or microscopes.
</P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Helmreich"> Laboratory of Prof. Dr. Helmreich </SPAN></H4><P><A href="http://www.sww.bv.tum.de" class="external text" rel="nofollow">Sanitary Environmental Engineering</A> is a horizontal discipline comprised of civil engineering, process engineering and chemistry/biology. Research and teaching include the fields of water supply, sewage and rain water treatment, water quality and the modeling of aquatic systems.
</P><P><I>In our interview with Prof. Helmreich, she explained us several <B>characterization techniques</B> to assess water quality. Furthermore she kindly provided us with <B>water samples</B> from sewage treatment plants.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Thomas_Hofmann"> Laboratory of Prof. Dr. Thomas Hofmann </SPAN></H4><P>The <A href="http://www.molekulare-sensorik.de/index.php?id=2&amp;L=1" class="external text" rel="nofollow">chair of Food Chemistry and Molecular Sensory Science</A> at the Technische UniversitÃ¤t MÃ¼nchen (TUM) comprises an international research team with a focus on chemosensory active biomolecules (sensomics), metabolomics &amp; nutritional biomarker discovery, as well as stable isotope labeling studies.
</P><P><I>The chair gave us great support in form of the detection of degraded pollutans via LCMS.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Langosch"> Laboratory of Prof. Dr. Langosch </SPAN></H4><P>The <A href="http://www.wzw.tum.de/biopolymere" class="external text" rel="nofollow">Chair for Chemistry of Biopolymers</A> focuses on the <B>structural biochemistry</B> of <B>integral membrane proteins</B>. Core themes are molecular interactions between membrane proteins, structural dynamics of membrane bound protein helices, membrane protein/lipid interactions and structure/function relationships of integral membrane protein complexes.
</P><P><I>During the planning phase of our project we asked Prof. Langosch for advice concerning the design of the transmembrane domain of our constructs.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Rost"> Laboratory of Prof. Dr. Rost </SPAN></H4><P>Prof. Rost works in the field of <B>bioinformatics</B> and computer-based biology, focusing on the <B>prediction of structure and function of proteins</B> and genes. His team's specialty is the use of artificial intelligence and machine learning algorithms to predict structure and function of proteins.
</P><P><I>After developing an early version of our <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" class="external text" rel="nofollow">AutoAnnotator</A>, we presented our work to Prof. Rost and his group. He gave us some very helpful advice on features we could add and provided us with access to his <A href="http://www.predictprotein.org/" class="external text" rel="nofollow">PredictProtein</A> server. A special &quot;thank you&quot; goes to Manfred Roos for tailoring the access point of the server to our needs.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Schneitz"> Laboratory of Prof. Dr. Schneitz </SPAN></H4><P>The <A href="http://plantdev.bio.wzw.tum.de/index.php?id=36" class="external text" rel="nofollow">Chair for Plant Developmental Biology</A> is interested in the genetic and molecular basis of the <B>regulatory pathways controlling organ development and tissue morphogenesis</B> in plants. 
</P><P><I>The chair gave us great access to their <B>microscopes</B> and excellent support in how to use them. Many thanks for that, especially to Prasad Vaddepalli for his introduction into fluorescence microscopy.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Schwechheimer"> Laboratory of Prof. Dr. Schwechheimer </SPAN></H4><P>The <A href="http://www.sysbiol.wzw.tum.de" class="external text" rel="nofollow">Chair for Systems Biology of Plants</A> investigates a range of <B>queries</B> concerning the <B>ubiquitin proteasome</B> system of plants applying a combination of genetics, molecular biology and cell biology with both <B>genomic</B> and <B>proteomic</B> approaches. 
</P><P><I>Together with Prof. Schwechheimer we discussed several different signal transduction pathways that could be utilised for our <B><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch" class="external text" rel="nofollow">kill-switch</A></B>. Moreover he kindly granted us access to a <B>fluorescence microscope</B>.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Scherer"> Laboratory of Prof. Dr. Scherer </SPAN></H4><P>The <A href="http://www.micbio.wzw.tum.de/index.php" class="external text" rel="nofollow">Chair for microbial ecology</A> conducts basic research in the field of <B>molecular genetics and ecology of pathogenic microorganisms</B>. Projects include the microbial characterization of food  and microbial diagnostics and consulting in the food industry for prevention and clearance of contaminations.
</P><P><I>The team kindly provided us with a culture of </I>Micrococcus luteus<I> for the <B><A href="https://2013.igem.org/wiki/index.php?title=Team:TU-Munich/Results/Recombinant#Kirby-Bauer_Assay:_Measuring_remaining_erythromycin_on_a_pertri_dish" class="external text" rel="nofollow">Kirby-Bauer assay</A></B> and also helped out with a tube of restriction enzyme during an unexpected shortage.</I></P><H2><SPAN class="mw-headline" id="Other_Universities"> Other Universities </SPAN></H2><P>The team also obtained help from other universities in order to complete the project. As nobody at our university works with <I>Physcomitrella patens</I> which the team chose to work with we had to find an advisor who could support us with the <B>transformation procedure</B> which is not really simple for plants. Additionally researches from other universities helped us by <B>providing plasmids</B>. 
</P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Reski_at_Freiburg_University_.28Germany.29"> Laboratory of <A href="https://2013.igem.org/Team:TU-Munich/Team/Members#Prof._Dr._Ralf_Reski" class="external text" rel="nofollow">Prof. Dr. Reski</A> at Freiburg University (Germany) </SPAN></H4><P>The team at the <A href="http://www.plant-biotech.net/" class="external text" rel="nofollow">Chair for Plant Biotechnology</A> is working on <B>gene expression</B> in the bryophyte model plant <I>Physcomitrella patens</I> (Hedw.) B.S.G. at different levels in correlation with phenotype analysis and additionally employs comparative genomic approaches.
</P><P><I>The Reski team kindly supported us by supervising <A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss" class="external text" rel="nofollow">our moss transformations at their lab</A> and by allowing us to use some of their equipment and materials for the process. Our special thanks go to <A href="https://2013.igem.org/Team:TU-Munich/Team/Members#Dr._Gertrud_Wiedemann" class="external text" rel="nofollow">Dr. Gertrud Wiedemann</A>.</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Fussenegger_at_ETH_Zurich_.28Switzerland.29"> Laboratory of Prof. Dr. Fussenegger at ETH Zurich (Switzerland)</SPAN></H4><P>The research group at the <A href="http://www.bsse.ethz.ch/groups/group_fussenegger/index/" class="external text" rel="nofollow">Chair for Biotechnology and Bioengineering</A> is implementing a progress in basic research to achieve generic and prototypic advances in <B>human therapy</B> by focusing on <B>mammalian cells</B> and capitalizing on an integrated interdisciplinary systems approach. Their current research initiatives include several programs interfacing with <B>biopharmaceutical manufacturing</B>, <B>gene therapy</B> and <B>tissue engineering</B>.
</P><P><I>They kindly provided us with the pSH21 plasmid which was used as template for the Polioviral Internal Ribosome Entry Site (<A href="http://parts.igem.org/Part:BBa_K1159300" class="external text" rel="nofollow">BBa_K1159300</A>).</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Dr._G.D._Wright_at_McMaster_University_.28Canada.29"> Laboratory of Dr. G.D. Wright  at McMaster University (Canada) </SPAN></H4><P>The <A href="http://www.thewrightlab.com" class="external text" rel="nofollow">Wright Lab</A> is trying to understand fundamental aspects of <B>how antibiotics work</B>, their sources and <B>how bacteria become resistant</B> to them.
</P><P><I>They kindly provided us with the plasmids pDEST14_ereA and pDEST14_ereB which was used as a template for the Erythromycin Esterase Type II (&lt;partinfo&gt;BBa_K1159000&lt;/partinfo&gt;).</I></P><H4><SPAN class="mw-headline" id="Laboratory_of_Prof._Dr._Arndt_at_Potsdam_University_.28Germany.29"> Laboratory of Prof. Dr. Arndt  at Potsdam University (Germany)</SPAN></H4><P>The team at the <A href="http://www.uni-potsdam.de/index.php?id=13895" class="external text" rel="nofollow">Chair for Molecular Biotechnology</A> investigates the <B>factors that mediate interactions in coiled-coil proteins</B> in order to target coiled-coil domains of proteins e.g. involved in tumorigenesis, tumor proliferation and metastasis.
</P><P><I>Sven Hagen who had participated several times in iGEM in the past kindly provided us with the pBad-mVenus (RFC 25) expression plasmid. This plasmid was used for the production of recombinant protein in <B>E. coli</B>. </I></P><H2><SPAN class="mw-headline" id="Webdesign"> Webdesign </SPAN></H2><P>The webdesign was entirely done by Florian Albrecht who has programmed different websites before. He also wrote the <A href="https://2013.igem.org/Team:TU-Munich/Results/How_To" class="external text" rel="nofollow">webdesign tutorial</A> we put online to explain other teams how to use the code, developed by our team. The graphic elements used on the website were created by several different student members (such as Ingmar Polte, Katrin Fischer, Jeffery Truong, Rosario Ciccone and others) using the open source vector software Inkscape. The only component that was bought are the two direction signs used in the header (obtained from <A href="http://vector-images.de/clipart/clp217632/" class="external text" rel="nofollow">http://vector-images.de/clipart/clp217632</A>).
</P><H4><SPAN class="mw-headline" id="normalize.css"> normalize.css </SPAN></H4><P><A href="http://necolas.github.io/normalize.css/" class="external text" rel="nofollow">Normalize.css</A> was used to reset the wiki style (version used: 2.1.3).
</P><H4><SPAN class="mw-headline" id="jQuery"> jQuery </SPAN></H4><P><A href="http://jquery.com/" class="external text" rel="nofollow">jQuery</A> is a powerful JavaScript extension used by many Webdesigners (version used: 1.10.2).
</P><H4><SPAN class="mw-headline" id="jQuery_UI"> jQuery UI </SPAN></H4><P><A href="http://jqueryui.com/" class="external text" rel="nofollow">jQuery UI</A> is a GUI extension of jQuery and was used for the datepicker on the Arduino data page (version used: 1.10.3).
</P><H4><SPAN class="mw-headline" id="History.js"> History.js </SPAN></H4><P><A href="https://github.com/browserstate/history.js/" class="external text" rel="nofollow">History.js</A> was used to ensure compatibility with HTML4 browsers, that do not support history.pushState().
</P><H4><SPAN class="mw-headline" id="Slimbox_2"> Slimbox 2 </SPAN></H4><P><A href="http://code.google.com/p/slimbox/" class="external text" rel="nofollow">Slimbox 2</A> was used as a picture viewer (version used: 2.05). The code had to be adapted to work with the wiki and bxSlider and we added a feature that fits the viewer to the browser size.
</P><H4><SPAN class="mw-headline" id="bxSlider"> bxSlider </SPAN></H4><P><A href="http://bxslider.com/" class="external text" rel="nofollow">bxSlider</A> is a JavaScript content slider and powers the slideshows and picture galleries on our wiki (version used: 4.1.1). The code was modified so all the images are scaled to the same height in gallery mode.
</P><H4><SPAN class="mw-headline" id="NVD3"> NVD3 </SPAN></H4><P><A href="http://nvd3.org/" class="external text" rel="nofollow">NVD3</A> is a JavaScript chart library and was used for the interactive chart of the Arduino data (version used: 1.1.13).
</P><H2><SPAN class="mw-headline" id="Software_-_The_AutoAnnotator"> Software - The AutoAnnotator </SPAN></H2><P>The software tool we have written to <B>facilitate and improve the annotation</B> of protein coding BioBricks (the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">AutoAnnotator</A>) was developed and programmed by the student member <A href="https://2013.igem.org/Team:TU-Munich/Team/Members" class="external text" rel="nofollow">Christopher Wolf</A>. He developed this idea himself to import, translate sequences and to compute important parameters. After this was completed he contacted the bioinformatics group of Prof. Rost and presented his idea and the program in a group seminar. In the seminar, ideas for further improvements were generated. The group advised Chrostopher to implement a bioinformatic module that does alignments to several databases and bioinformatic servers. The research group of Prof. Rost has a server that concentrates bioinformatic information and they enabled Christopher to use this tool. He implemented these information into the AutoAnnotator himself. It was also Chrostopher Wolf who has written the <A href="http://dspace.mit.edu/handle/1721.1/81330" class="external text" rel="nofollow">RFC 96</A> to describe the usage of the AutoAnnotator and who created the <A href="http://vimeo.com/75965599" class="external text" rel="nofollow">introduction video</A>.
For further information please see our <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">Software page</A>.
</P><H4><SPAN class="mw-headline" id="James_Padolsey">James Padolsey</SPAN></H4><P><I>We used <A href="http://james.padolsey.com" class="external text" rel="nofollow">James PadolseyÂ´s</A> jQuery extension for <A href="http://james.padolsey.com/javascript/cross-domain-requests-with-jquery/" class="external text" rel="nofollow">Cross-domain AJAX requests</A> in the AutoAnnotator.</I></P><H4><SPAN class="mw-headline" id="Flot.js">Flot.js</SPAN></H4><P><I>Flot.js (see <A href="http://flotchart.org" class="external text" rel="nofollow">flotchart.org</A>) was used to draw the graphs in the AutoAnnotator and in the Filter Calculator.</I></P><H4><SPAN class="mw-headline" id="Flot-AxisLabels">Flot-AxisLabels</SPAN></H4><P><A href="https://github.com/markrcote/flot-axislabels" class="external text" rel="nofollow">Flot-AxisLabels</A> is an extension of Flot.js, adding axis labels to Flot.js.
</P><H4><SPAN class="mw-headline" id="Excanvas">Excanvas</SPAN></H4><P><A href="http://excanvas.sourceforge.net" class="external text" rel="nofollow">Excanvas.js</A> extends the &lt;canvas&gt;-tag to IE 8.0 and earlier, allowing the use of AutoAnnotator (in particular the plots) in these browsers.
</P><H4><SPAN class="mw-headline" id="Paul_Johnston">Paul Johnston</SPAN></H4><P><I>We used the <A href="http://pajhome.org.uk/crypt/md5/md5.html" class="external text" rel="nofollow">MD5 generator</A> written by Paul Johnston to translate the amino acid sequences into MD5-hashs, which were used to obtain the predictions from the database of <A href="http://PredictProtein.org" class="external text" rel="nofollow">PredictProtein.org</A></I></P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Team/Attributions">http://2013.igem.org/Team:TU-Munich/Team/Attributions</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Team/Attributions" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Team/Attributions" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Team/Attributions&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Team/Attributions&amp;oldid=359279" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>