<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Modeling_Protein_Predictions"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Modeling/Protein_Predictions">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Modeling/Protein_Predictions&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Modeling/Protein_Predictions&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Modeling/Protein_Predictions&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Modeling/Protein_Predictions" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Modeling/Protein Predictions</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Prediction_of_Protein_Structures_and_Functions">Prediction of Protein Structures and Functions</SPAN></H2><P><B>Structural properties</B> of <B>effector proteins</B> are often essential for their function, so it is advantageous to know about them. It is for example necessary to know whether <B>termini are accessible</B> for protein fusion or whether the protein is <B>only functional in a multimeric fold</B>. For this reason a structure based search was performed in the <A href="http://www.rcsb.org/pdb/home/home.do" class="external text" rel="nofollow">protein database</A>. As the number of identified structures is still limited, it is a promising attempt to look for homologous proteins whose crystal structures have been determined.
</P><H2><SPAN class="mw-headline" id="Analysis_of_Receptor_Sequences_.E2.80.93_Choosing_the_right_template">Analysis of Receptor Sequences – Choosing the right template </SPAN></H2><P>For several purposes of our project, we needed a synthetic receptor enabling us to integrate proteins into the membrane in the desired orientation, i.e. to express protein-domains on the intracellular or extracellular side of the cell membrane. We investigated several different plant-receptors from the well characterized dicotyledon <I>Arabidopsis thaliana</I> and the moss <I>Physcomitrella patens</I>, our chassis. The receptors from <I>Arabidopsis thaliana</I> have the advantage that their transgenic expression has successfully been demonstrated [<A href="http://www.pnas.org/content/88/23/10806.full.pdf" class="external text" rel="nofollow">Quail et al., 1991</A>] whereas the native receptors from <I>Physcomitrella patens</I> bear only a small risk of failing [<A href="http://www.plant-biotech.net/paper/Reski_1998_BotActa-111_1_scan.pdf" class="external text" rel="nofollow">Reski, 1998</A>].
Due to the fact that there were many different available receptors, which we could have used as template for our synthetic receptor, we used bioinformatical methods to evaluate the suitability of these receptors. The following three examples ERF, FLS2 and SERK shown in table 1 resulted from this equation.
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 1:</B> Examined Receptors
</CAPTION><TBODY><TR><TH>Receptor
</TH><TH>Organism
</TH><TH>Length (aa)
</TH><TH>Sequence reference
</TH><TH>Literature reference
</TH></TR><TR><TD>ERF
</TD><TD><I>A. thaliana</I></TD><TD>1031
</TD><TD><A href="http://www.ncbi.nlm.nih.gov/protein/NP_197548.1" class="external text" rel="nofollow">NP_197548.1</A></TD></TR><TR><TD>FLS2
</TD><TD><I>A. thaliana</I></TD><TD>1173
</TD><TD><A href="http://www.ncbi.nlm.nih.gov/protein/NP_199445.1" class="external text" rel="nofollow">NP_199445.1</A></TD></TR><TR><TD>SERK
</TD><TD><I>P. patens</I></TD><TD>625
</TD><TD><A href="http://www.ncbi.nlm.nih.gov/protein/XP_001759122.1" class="external text" rel="nofollow">XP_001759122.1</A></TD><TD><A href="http://www.freidok.uni-freiburg.de/volltexte/5390/pdf/Lienhart_Dissertation_2008.pdf" class="external text" rel="nofollow">Lienhart, 2007</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Prediction_of_Signal_Peptides">Prediction of Signal Peptides</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 1:</B> Prediction and analysis of signal peptides</DIV></DIV></DIV><P><B>Introduction</B>
A first analysis was performed to identify signal-peptides, which are bound by the cellular signal recognition particle and lead to the translocation of the bound polypeptide into the endoplasmic reticulum. Afterwards the signal peptide is cleaved by a signal peptide peptidase at a specific site. The analysis for signal peptides was done by using the <A href="http://www.cbs.dtu.dk/services/SignalP" class="external text" rel="nofollow">SignalP 4.1 Server</A>. <B>Results</B>
The program was run for different receptors and will be illustrated for the three examples mentioned above (see fig. 1). 
The figure shows the N-terminal sequence of the receptors, together with three scores: 
(1) The C-Score (raw cleavage site score) in red. 
(2) The S-Score (signal peptide score) in green. 
(3) The Y-Score (combined cleavage site score) in blue.
The C-Score shows the most probable cleavage site identified by the peptidase. It was possible to identify the most probable cleavage site for all shown receptors with ambiguous cleavage sites for the SERK-receptor. The amino acid with the highest C-score is predicted to be the first amino acid of the primary structure of the cleaved receptor. 
The S-Score was developed to identify amino acid sequences which appear in a signal peptide and others that belong to the matured receptor. The course of this parameter is high for the first 23-28 amino acids of all receptors, identifying these residues as signal peptides. The amino acid residue, which lies at the greatest decrease of the S-Score, is the predicted border between the N-terminal signal peptide and the receptor. 
The Y-Score results from the geometrical structure of the protein and the previously determined scoring parameters. It illustrates that the two first parameters show a good fit for the identification of the signal peptide in all three indicated receptors.<B>Discussion</B>
Summarizing these parameters, it can be concluded that all three pictured receptors seem to contain a sequence acting as a signal peptide. For many of the predicted receptors in the genome of <I>Physcomitrella patens</I> the prediction did not yield a positive result. With respect to the signal peptide, all mentioned receptors would be suitable as a template for our synthetic receptor. The predicted data show that the SERK-Receptor is favorable for our application, because its signal peptide is statistically the best recognized one and bears the smallest risk of failure.
</P><H3><SPAN class="mw-headline" id="Prediction_of_Transmembrane_Regions">Prediction of Transmembrane Regions</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2:</B> Prediction and analysis of transmembrane regions</DIV></DIV></DIV><P><B>Introduction</B>
Additional to the identification of the signal peptide, it was very important to identify transmembrane regions within the receptors, because we wanted to use a type I receptor as a template that contains a N-terminal extracellular domain, a transmembrane domain and a C-terminal intracellular domain (see <A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation" class="external text" rel="nofollow">our localization page</A>). To analyze this issue, the prediction tool <A href="http://www.cbs.dtu.dk/services/TMHMM" class="external text" rel="nofollow">TMHMM</A> was used for several different receptors. Again the most suitable receptors were ERK, FLS2 and SERK.<B>Results</B>
The analysis yields a signal peptide and a single transmembrane domain for all three depicted receptors (see fig. 2). The estimated reliability of the predictions was equally good for all examined receptors, whereas the signal peptide was most reliably predicted for the SERK receptor. <B>Discussion</B>
Focussing on the membrane topology point of view, all the investigated receptors would be suitable blue prints for our synthetic receptor. As the SERK-Receptor yields the best prediction, it was chosen as the favorable template. Another reason to choose the SERK-Receptor was that it is derived from <I>Physcomitrella patens</I>. The only problem, concerning this prediction, is that the N-terminus of this receptor is predicted to be extracellular. The falsification of this prediction was simple, because the SERK receptor contains a C-terminal kinase-domain, which is known to be involved in signal transduction.
</P><H3><SPAN class="mw-headline" id="Choice_of_the_SERK_Receptor">Choice of the SERK Receptor</SPAN></H3><P>Finally we decided to use the SERK receptor as a template to generate our synthetic receptor. The final receptor was designed in RFC[25] standard, which allows in frame protein fusions. The final constructs were designed containing the SERK signal peptide (<A href="http://parts.igem.org/Part:BBa_K1159303" class="external text" rel="nofollow">BBa_K1159303</A>), an extracellularly located effector protein, the transmembrane domain of the SERK receptor (<A href="http://parts.igem.org/Part:BBa_K1159305" class="external text" rel="nofollow">BBa_K1159305</A>), a short linker and a GFP, to investigate the cellular localization of our receptor with the aid of fluorescence microscopy.
</P><H2><SPAN class="mw-headline" id="Searching_for_Homologous_Structures_using_HHpred">Searching for Homologous Structures using HHpred</SPAN></H2><P>The search for homologous structures was performed by using the freely accessible web server <A href="http://toolkit.tuebingen.mpg.de/hhpred" class="external text" rel="nofollow">HHpred</A> [<A href="http://www.ncbi.nlm.nih.gov/pubmed/15980461" class="external text" rel="nofollow">Söding et al., 2005</A>]. The amino acid sequences for the BioBricks were translated into amino acid sequences using the <A href="https://2013.igem.org/Team:TU-Munich/Results/Software" class="external text" rel="nofollow">AutoAnnotator</A> and was then inserted into the the search field. The results for all proteins investigated in our project are shown in table 2. 
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 2:</B> Predicted Structures
</CAPTION><TBODY><TR><TH>Protein
</TH><TH>BioBrick
</TH><TH>PDB-code
</TH><TH>Identity
</TH><TH>Similarity
</TH><TH>Structure
</TH></TR><TR><TD>XylE
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K147002" class="external text" rel="nofollow">BBa_K147002</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=3hpy" class="external text" rel="nofollow">3hpy_A</A></TD><TD>50%
</TD><TD>0.939
</TD></TR><TR><TD>Laccase
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159002" class="external text" rel="nofollow">BBa_K1159002</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=2wsd" class="external text" rel="nofollow">2wsd_A</A></TD><TD>68%
</TD><TD>1.223
</TD></TR><TR><TD>NanoLuc
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159001" class="external text" rel="nofollow">BBa_K1159001</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=3ppt" class="external text" rel="nofollow">3ppt_A</A></TD><TD>21%
</TD><TD>0.359
</TD></TR><TR><TD>EreB
</TD><TD><A href="http://parts.igem.org/Part:BBa_K1159000" class="external text" rel="nofollow">BBa_K1159000</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=3b55" class="external text" rel="nofollow">3b55_A</A></TD><TD>19%
</TD><TD>0.318
</TD></TR><TR><TD>Spycatcher
</TD><TD><A href="http://parts.igem.org/Part:BBa_K1159200" class="external text" rel="nofollow">BBa_K1159200</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=2x5p" class="external text" rel="nofollow">2x5p_A</A></TD><TD>97%
</TD><TD>1.298
</TD></TR><TR><TD>PP1
</TD><TD><A href="http://parts.igem.org/Part:BBa_K1159004" class="external text" rel="nofollow">Part:BBa_K1159004</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=3e7a" class="external text" rel="nofollow">3e7a_A</A></TD><TD>96%
</TD><TD>1.593
</TD></TR><TR><TD>GFP
</TD><TD><A href="http://parts.igem.org/Part:BBa_K1159311" class="external text" rel="nofollow">BBa_K1159311</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=2WUR" class="external text" rel="nofollow">2WUR</A></TD><TD>98%
</TD><TD>1.477
</TD></TR><TR><TD>Glutathiontransferase / DDT Dehydrochlorinase
</TD><TD>&lt;partinfo&gt;BBa_K620000&lt;/partinfo&gt;</TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=3F6D" class="external text" rel="nofollow">3F6D</A></TD><TD>68%
</TD><TD>1.155
</TD></TR><TR><TD>SERK-TM
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1159305" class="external text" rel="nofollow">BBa_K1159305</A></TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=2ks1" class="external text" rel="nofollow">2ks1_B</A></TD><TD>24%
</TD><TD>0.233
</TD></TR><TR><TD>TEV Protease
</TD><TD>Commercial reagent
</TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=1Q31" class="external text" rel="nofollow">1Q31</A></TD><TD>n.d.
</TD><TD>n.d.
</TD></TR><TR><TD>Streptavidin
</TD><TD>Commercial reagent
</TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=3RY2" class="external text" rel="nofollow">3RY2</A></TD><TD>n.d.
</TD><TD>n.d.
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results">Results</SPAN></H4><P>The homology search showed that some of our effector proteins have very closely related proteins with a known structure. For example there are very similar protein structures available for the SypCatcher, PP1 and GFP each with identities of over 90%. Some other effector proteins such as XylE, Laccase or the DDT Dehydrochlorinase have less homologous proteins, whose structures still give good hints on structural questions. However there are also effectors where only badly matched structures are known, which can only be used as a very rough indication of the fold. The NanoLuc luciferase, which is a highly engineered protein derived from shrimps and was only published this year, is an example of a protein with no known structural homologue.
The structures obtained here were used to design our experiments. A homology modeling for the Laccase was performed to determine whether it contains disulphide bridges. The resulting homologous structures were used as illustrations, as explained in one of our <A href="https://2013.igem.org/Team:TU-Munich/Results/How_To" class="external text" rel="nofollow">How-Tos</A> about animated GIFs.
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="http://www.pnas.org/content/88/23/10806.full.pdf" class="external text" rel="nofollow">Quail et al., 1991</A>] MARGARET T. Boylan, M.T. and Quail, P.H. (1991). PhytochromeA overexpression inhibits hypocotyl elongation in transgenic <I>Arabidopsis</I>. <I>Proc. Natl. Acad. Sci.</I> 88:10806-10810.</P><P>[<A href="http://www.plant-biotech.net/paper/Reski_1998_BotActa-111_1_scan.pdf" class="external text" rel="nofollow">Reski, 1998</A>] Reski, R. (1998). Development, Genetics and Molecular Biology of Mosses. <I>Bot. Acta</I>, 111:1-15.</P><P>[<A href="http://www.freidok.uni-freiburg.de/volltexte/5390/pdf/Lienhart_Dissertation_2008.pdf" class="external text" rel="nofollow">Lienhart, 2007</A>] Lienhart, O. (2007). Untersuchungen zu einem Somatic-Embryogenesis-Receptor-like-Kinase-Homolog in <I>Physcomitrella patens</I> (Hedw.) B.S.G. PhD-thesis at Freiburg University
</P><P><A href="http://www.cbs.dtu.dk/services/SignalP/" class="external text" rel="nofollow">SignalP41</A> Server
</P><P>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/15980461" class="external text" rel="nofollow">Söding et al., 2005</A>] Söding J, Biegert A, Lupas AN. (2005). The HHpred interactive server for protein homology detection and structure prediction. <I>Nucleic Acids Res.</I> 2005 Jul 1;33(Web Server issue):W244-8.</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">http://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Modeling/Protein_Predictions" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Modeling/Protein_Predictions" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Modeling/Protein_Predictions&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Modeling/Protein_Predictions&amp;oldid=360790" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>