<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Munich_Notebook_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Munich/Notebook/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Munich/Notebook/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Methods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Munich/Notebook/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Munich/Notebook/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header-container"><DIV id="menu"><UL><LI><A id="home" href="https://2013.igem.org/Team:TU-Munich">Home</A></LI><LI><A id="project" href="https://2013.igem.org/Team:TU-Munich/Project/Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Phytoremediation">Phytoremediation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Physcomitrella"><I>Physcomitrella</I></A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Localisation">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Biodegradation">BioDegradation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Bioaccumulation">BioAccumulation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Killswitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Project/Safety">Safety</A></LI></UL><LI><A id="results" href="https://2013.igem.org/Team:TU-Munich/Results/Overview">Results</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/BioBricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Recombinant">Effector Studies</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Moss">Moss Methods</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Localization">Localization</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/GM-Moss">PhyscoFilter</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/KillSwitch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Implementation">Implementation</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Software">AutoAnnotator</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Economics">Entrepreneurship</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/Summary">Summary</A></LI><LI><A id="modeling" href="https://2013.igem.org/Team:TU-Munich/Modeling/Overview">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions">Protein Predictions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Kill_Switch">Kill Switch</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Enzyme">Enzyme Kinetics</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Modeling/Filter">Filter Model</A></LI><LI><A id="team" href="https://2013.igem.org/Team:TU-Munich/Team/Members">Team</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Members">Members</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Gallery">Pictures</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Statistics">Timeline</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Team/Judging">Judging</A></LI><LI><A id="notebook" href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Notebook/Methods">Methods</A></LI><LI><A id="human_practice" href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Overview">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Interviews">Expert Counsel</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/GMO">GMO release</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Media">Publicity</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/HumanPractice/Education">Educational Kit</A></LI><LI><A href="https://2013.igem.org/Team:TU-Munich/Results/How_To">Tutorials</A></LI></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H2><SPAN class="mw-headline" id="Materials">Materials</SPAN></H2><P>The following section lists and describes the used material in the presented iGEM project.
</P><H3><SPAN class="mw-headline" id="1._Cell_lines">1. Cell lines</SPAN></H3><UL><LI><I>E. coli</I> XL-1 Blue 
</LI><LI>Obtained from the chair for Biological Chemistry (TUM)
</LI><LI>Cell line was used for molecular cloning
</LI></UL><LI><I>E. coli</I> BL21 
</LI><LI>Obtained from the chair for Biological Chemistry (TUM)
</LI><LI>Cell line was used for cytosolic production of recombinant proteins
</LI><LI><I>P. patens</I> clone XI 
</LI><LI>Was used for all moss experiments and for the generation of genetically modified <I>P. patens</I> plants
</LI><LI>Kindly provided by the <A href="http://www.plant-biotech.net/" class="external text" rel="nofollow">Chair for Plant Biotechnology</A> (Freiburg University)
</LI><LI><I>Micrococcus luteus</I></LI><LI>Kindly provided by the <A href="http://www.micbio.wzw.tum.de/index.php" class="external text" rel="nofollow">Chair for Microbial Ecology</A> (TUM)
</LI><LI>Was used for the Kirby-Bauer assay to detect the degradation of erythromycin
</LI><H3><SPAN class="mw-headline" id="2._Reagents">2. Reagents</SPAN></H3><TABLE><TBODY><TR><TH> Reagent
</TH><TH> Supplier
</TH><TH> Location
</TH></TR><TR><TD> 2-Mercaptoethanol
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> 4-Nitrophenyl butyrate
</TD><TD> Sigma-Aldrich
</TD><TD> Taufkirchen, Germany
</TD></TR><TR><TD>ABTS tablet
</TD><TD>Roche
</TD></TR><TR><TD>ABTS buffer
</TD><TD>Roche
</TD></TR><TR><TD> Acetonitrile (LC-MS grade)
</TD><TD> Sigma-Aldrich
</TD><TD> Taufkirchen, Germany
</TD></TR><TR><TD> 30% Acrylamid with 0,8% Bisacrylamid
</TD><TD> Serva Electrophoresis GmbH
</TD><TD> Heidelberg, Germany
</TD></TR><TR><TD> Agarose
</TD><TD> peqlab Biotechnologie GmbH
</TD><TD> Erlangen, Germany
</TD></TR><TR><TD> Acetic acid (LC-MS grade)
</TD><TD> Sigma-Aldrich Chemie GmbH
</TD><TD> Taufkirchen
</TD></TR><TR><TD> Ammonium acetate
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Bacto Agar Becton
</TD><TD> BD, DIFCO Laboratories GmbH
</TD><TD> Augsburg, Germany
</TD></TR><TR><TD> Bacto Tryptone Becton
</TD><TD> BD, DIFCO Laboratories GmbH
</TD><TD> Augsburg, Germany
</TD></TR><TR><TD> Bacto Yeast Extract Becton
</TD><TD> BD, DIFCO Laboratories GmbH
</TD><TD> Augsburg, Germany
</TD></TR><TR><TD>Biotin
</TD></TR><TR><TD> Boric acid
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Bromphenol blue
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Coomassie brilliant blue R-250
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Chloramphenicol
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> D(+)-Glukose, monohydrate
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Diclofenac
</TD></TR><TR><TD> dNTPs
</TD><TD> Invitrogen AG
</TD><TD> Basel, Switzerland
</TD></TR><TR><TD> Ethanol p.p., 99,9%
</TD><TD> Honeywell Deutschland AG
</TD><TD> Neunkirchen, Germany
</TD></TR><TR><TD> Glycerol
</TD><TD> Carl Roth GmbH &amp; Co. KG
</TD><TD> Karlsruhe, Germany
</TD></TR><TR><TD> Kaliumchloride (KCl)
</TD><TD> Merck KG
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Kaliumdihydrogenphosphate (KH<SUB>2</SUB>PO<SUB>4</SUB>)
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Methanol
</TD><TD> Honeywell Deutschland AG
</TD><TD> Neunkirchen, Germany
</TD></TR><TR><TD> N,N,N’,N’-Tetramethylethylendiamin (TEMED)
</TD><TD> AppliChem GmbH
</TD><TD> Darmstadt, Germany
</TD></TR><TR><TD> Sodiumchloride (NaCl)
</TD><TD> Carl Roth GmbH &amp; Co. KG
</TD><TD> Karlsruhe, Germany
</TD></TR><TR><TD> Sodiumdihydrogenphosphate (NaH2PO4)
</TD><TD> Carl Roth GmbH &amp; Co. KG
</TD><TD> Karlsruhe, Germany
</TD></TR><TR><TD> Disodiumhydrogenphosphate (Na2HPO4)
</TD><TD> VWR International GmbH
</TD><TD> Ismaning, Germany
</TD></TR><TR><TD> Sodiumdodecylsulphate (SDS)
</TD><TD> Carl Roth GmbH &amp; Co. KG
</TD><TD> Karlsruhe, Germany
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="3._Kits_and_Standards">3. Kits and Standards</SPAN></H3><UL><LI>Kits for DNA manipulation (Qiagen)
</LI><LI>QIAprep Spin Miniprep Kit (Qiagen) 
</LI><LI>QIAquick Gel Extraction Kit (Qiagen)
</LI><LI>QIAquick PCR Purification Kit (Qiagen)
</LI><LI>QIAGEN Plasmid Midi Kit (Qiagen)
</LI></UL><UL><LI>DNA ladder
</LI><LI>GeneRuler 1 kb DNA Ladder (Fermentas)
</LI><LI>GeneRuler 100 bp DNA Ladder (Fermentas)
</LI><LI>2-Log DNA Ladder (New England Biolabs)
</LI></UL><UL><LI>Protein ladder
</LI><LI>Unstained Protein Ladder (Fermentas)
</LI></UL><UL><LI>Kit for Luciferase assays
</LI><LI>Nano-Glo Luciferase Assay
</LI></UL><H3><SPAN class="mw-headline" id="4._Consumables">4. Consumables</SPAN></H3><UL><LI>Plastics
</LI><LI>96-well plates(NUNC, Langenselbold, Germany)
</LI><LI>96-well plates for luminescense(NUNC, Langenselbold, Germany)
</LI><LI>96-well plats MaxiSorp ELISA (NUNC, Langenselbold, Germany)
</LI></UL><UL><LI> Affinity chromatography column material
</LI><LI> IMAC Sepharose High Performance (GE Healthcare Europe GmbH, Penzberg, Germany)
</LI></UL><H3><SPAN class="mw-headline" id="5._DNA_Synthesis">5. DNA Synthesis</SPAN></H3><H4><SPAN class="mw-headline" id="Oligonucleotides">Oligonucleotides</SPAN></H4><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Name
</TD><TD>Sequence
</TD></TR><TR><TD>EreA_for
</TD><TD><CODE>5’- GCGCGTCTAGATGGCCGGCACGTGGAGAACGACCAGA -3‘</CODE></TD></TR><TR><TD>EreA-rev
</TD><TD><CODE>5‘- TTCGAAACCGGTTAGGGCAACTAGGCTGTCC -3‘</CODE></TD></TR><TR><TD>EreA_SDMI_for
</TD><TD><CODE>5’- CGAATCAGGAAGAAAACTACAGTTAGTCGGAATCGAC -3‘</CODE></TD></TR><TR><TD>EreA_SDMI_rev
</TD><TD><CODE>5'- GTCGATTCCGACTAACTGTAGTTTTCTTCCTGATTCG -3'</CODE></TD></TR><TR><TD>EreA_SDMII_for
</TD><TD><CODE>5'- GACGCCTGTGGTACGGAAAATTCATGTCTGACATT - 3'</CODE></TD></TR><TR><TD>EreA_SDMII_rev
</TD><TD><CODE>5'- AATGTCAGACATGAATTTTCCGTACCACAGGCGTC - 3'</CODE></TD></TR><TR><TD>EreA_SDMIII_for
</TD><TD><CODE>5'- GTGTTCCAAGCGCGGGCAAGGACAGCC - 3'</CODE></TD></TR><TR><TD>EreA_SDMIII_rev
</TD><TD><CODE>5'- GGCTGTCCTTGCCCGCGCTTGGAACAC - 3'</CODE></TD></TR><TR><TD> EreB_for
</TD><TD><CODE>5'- GCGCGTCTAGATGGCCGGCAGGTTCGAAGAATGGGTC - 3'</CODE></TD></TR><TR><TD> EreB-rev
</TD><TD><CODE>5'- TTCGAAACCGGTTTCATAAACGACCTCAGATACAG - 3'</CODE></TD></TR><TR><TD>EreB_SDMI_for
</TD><TD><CODE>5'- GGTGAAAATTCTCATTTCATAAAAGAGTTCTTTTTGTTACGACATACGCTTTT - 3'</CODE></TD></TR><TR><TD>EreB_SDMI_rev
</TD><TD><CODE>5'- AAAAGCGTATGTCGTAACAAAAAGAACTCTTTTATGAAATGAGAATTTTCACC - 3'</CODE></TD></TR><TR><TD>EreB_SDMII_for
</TD><TD><CODE>5'- CTTATAGTGGGCATACAGCAGCCCTCTATCCGG - 3'</CODE></TD></TR><TR><TD>EreB_SDMII_rev
</TD><TD><CODE>5'- CCGGATAGAGGGCTGCTGTATGCCCACTATAAG - 3'</CODE></TD></TR><TR><TD>EreB_SDMIII_for
</TD><TD><CODE>5'- CGAGTTGATAACTTCCAACTACAGGAACCAAATGAAGGTTC - 3'</CODE></TD></TR><TR><TD>EreB_SDMIII_rev
</TD><TD><CODE>5'- GAACCTTCATTTGGTTCCTGTAGTTGGAAGTTATCAACTCG - 3'</CODE></TD></TR><TR><TD>TEV_fw
</TD><TD><CODE>5'- GCGCGTCTAGATGGCCGGCGGAGAAAGCTTGTTTAAGGGGC - 3'</CODE></TD></TR><TR><TD>TEV_rv
</TD><TD><CODE>5'- ATATAACCGGTATTCATGAGTTGAGTCGCTTCC - 3'</CODE></TD></TR><TR><TD>TEV_t387c_fw
</TD><TD><CODE>5'- ATGGTGTCAGACACTAGCTGCACATTCCCTTCATC - 3'</CODE></TD></TR><TR><TD>TEV_t387c_rv
</TD><TD><CODE>5'- GATGAAGGGAATGTGCAGCTAGTGTCTGACACCAT - 3'</CODE></TD></TR><TR><TD>N_split_TEV_fw
</TD><TD><CODE>5'- ATTCGCGGCCGCTTCTAG - 3'</CODE></TD></TR><TR><TD>N_split_TEV_rv
</TD><TD><CODE>5'- TTCGAAACCGGTAGTTTGGAAGTTGGTTGTCAC - 3'</CODE></TD></TR><TR><TD>C_split_TEV_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCAAGAGCATGTCTAGCATGGTG - 3'</CODE></TD></TR><TR><TD>C_split_TEV_rv
</TD><TD><CODE>5'- CCGCTACTAGTATTAACCGG - 3'</CODE></TD></TR><TR><TD>PIF6_2-100_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCATGTTCTTACCAACCGATTATTGTTGC - 3'</CODE></TD></TR><TR><TD>PIF6_2-100_rv
</TD><TD><CODE>5'- TTCGAAACCGGTGTCAACATGTTTATTGCTTTC - 3'</CODE></TD></TR><TR><TD>MinMCS_fo
</TD><TD><CODE>5'- CTAGACCTAGGAAGCTTCCTGCAGGACTAGTAAGCTTCTGCA - 3'</CODE></TD></TR><TR><TD>MinMCS_rev
</TD><TD><CODE>5'- GAAGCTTACTAGTCCTGCAGGAAGCTTCCTAGGT - 3'</CODE></TD></TR><TR><TD>Bpu_for
</TD><TD><CODE>5'- GCGCGTCTAGATGGCCGGCAACCTAGAAAAATTTGTTGACGAGCTG - 3'</CODE></TD></TR><TR><TD>Bpu_rev
</TD><TD><CODE>5'- TTCGAAACCGGTGATGATATCCATCGGCCGCAT - 3'</CODE></TD></TR><TR><TD>Bpu_SDMI_for
</TD><TD><CODE>5'- CACTTAAGCATTTTCTACGGGTCGATCACACCATTCA - 3'</CODE></TD></TR><TR><TD>Bpu_SDMI_rev
</TD><TD><CODE>5'- TGAATGGTGTGATCGACCCGTAGAAAATGCTTAAGTG - 3'</CODE></TD></TR><TR><TD>Bpu_SDMII_fo
</TD><TD><CODE>5'- GACACGATTAAATGTGTATGCGGGCTTAGCTGGATTTTATTTG - 3'</CODE></TD></TR><TR><TD>Bpu_SDMII_re
</TD><TD><CODE>5'- CAAATAAAATCCAGCTAAGCCCGCATACACATTTAATCGTGTC - 3'</CODE></TD></TR><TR><TD>Bpu_SDMIII_fo
</TD><TD><CODE>5'- CACCCTTAAAAATAAAGCTGGCTGCGGACAGGAAG - 3'</CODE></TD></TR><TR><TD>Bpu_SDMIII_re
</TD><TD><CODE>5'- TTCCTGTCCGCAGCCAGCTTTATTTTTAAGGGTG - 3'</CODE></TD></TR><TR><TD>Bpu_SDMIV_fo
</TD><TD><CODE>5'- CGTACGCTCACTCTTACGGGTACACAGGATAAATA - 3'</CODE></TD></TR><TR><TD>Bpu_SDMIV_re
</TD><TD><CODE>5'- TATTTATCCTGTGTACCCGTAAGAGTGAGCGTACG - 3'</CODE></TD></TR><TR><TD>xylE_for
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCAACAAAGGTGTAATGCGACCG  - 3'</CODE></TD></TR><TR><TD>xylE_rev
</TD><TD><CODE>5'- TTCGAAACCGGTGGTCAGCACGGTCATGAATC  - 3'</CODE></TD></TR><TR><TD>xylE_c312a_for
</TD><TD><CODE>5'- GAACAGTTGTGGACGGCGCGTGCGC  - 3'</CODE></TD></TR><TR><TD>xylE_c312a_rev
</TD><TD><CODE>5'- GCGCACGCGCCGTCCACAACTGTTC - 3'</CODE></TD></TR><TR><TD>xylE_g483a_for
</TD><TD><CODE>5'- CCTCATGTATGGCGACGAATTACCGGCGACCTA - 3'</CODE></TD></TR><TR><TD>xylE_g483a_rev
</TD><TD><CODE>5'- TAGGTCGCCGGTAATTCGTCGCCATACATGAGG - 3'</CODE></TD></TR><TR><TD>xylE_a834g_for
</TD><TD><CODE>5'- CAACTACCCGGACCACAAGCCGGTGACCT - 3'</CODE></TD></TR><TR><TD>xylE_a834_rev
</TD><TD><CODE>5'- AGGTCACCGGCTTGTGGTCCGGGTAGTTG - 3'</CODE></TD></TR><TR><TD>p Act_for
</TD><TD><CODE>5'- AAGCTTTCTAGAGTCGACGAGGTCATTCATATG - 3'</CODE></TD></TR><TR><TD>pAct_rev
</TD><TD><CODE>5'- CTCGAGCTGCAGCGGCCGCTACTAGTAGCTTCTACCTACAAAAAAGCTC - 3'</CODE></TD></TR><TR><TD>t35S_for
</TD><TD><CODE>5'- AAGCTTTCTAGAGCTAGAGTCCGCAAAAATCACC - 3'</CODE></TD></TR><TR><TD>t35S_rev
</TD><TD><CODE>5'- CTCGAGCTGCAGCGGCCGCTACTAGTAGGTCACTGGATTTTGGTTTTAGG - 3'</CODE></TD></TR><TR><TD>Npt_for
</TD><TD><CODE>5'- AAGCTTGAATTCGCGGCCGCTTCTAGAGATCGGATCCTGTCAAACAC - 3'</CODE></TD></TR><TR><TD>Npt_rev
</TD><TD><CODE>5'- CGGCCGCTACTAGTAATCAAGCTTGACAGGAGGC - 3'</CODE></TD></TR><TR><TD>miniMCSII_fw
</TD><TD><CODE>5'- CTAGACAATTGAAGCTTCCTGCAGGACTAGTAAGCTTCTGCA - 3'</CODE></TD></TR><TR><TD>miniMCSII_rv
</TD><TD><CODE>5'- GAAGCTTACTAGTCCTGCAGGAAGCTTCAATTGT - 3'</CODE></TD></TR><TR><TD>SpyTag_fw
</TD><TD><CODE>5'- CTAGATGGCCGGCGCTCATATTGTCATGGTTGATGCTTACAAGCCAACTAAGA - 3'</CODE></TD></TR><TR><TD>SpyTag_rv
</TD><TD><CODE>5'- CCGGTCTTAGTTGGCTTGTAAGCATCAACCATGACAATATGAGCGCCGGCCAT - 3'</CODE></TD></TR><TR><TD>PIF3(2-100)_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCCCTCTGTTTGAACTTTTCAGG - 3'</CODE></TD></TR><TR><TD>PIF3(2-100)_rv
</TD><TD><CODE>5'- CCGCTACTAGTATTAACCGG - 3'</CODE></TD></TR><TR><TD>NucA_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCGCAACTTCAACTAAAAAATTACATAAAGAACCTG - 3'</CODE></TD></TR><TR><TD>NucA_rv
</TD><TD><CODE>5'- TTCGAAACCGGTTTGACCTGAATCAGCGTTGTC - 3'</CODE></TD></TR><TR><TD>AlcR_fw
</TD><TD><CODE>5'- AAGCTTGAATTCGCGGCCGCTTCTAGATGGCAGATACGCGCCGAC - 3'</CODE></TD></TR><TR><TD>AlcR_rv
</TD><TD><CODE>5'- TTCGAAACTAGTACTACAAAAAGCTGTCAACTTTCCCATTCAAACC - 3'</CODE></TD></TR><TR><TD>SQ_pAutoRex8_1
</TD><TD><CODE>5'- CGGGGCATCACAGCGTCG - 3'</CODE></TD></TR><TR><TD>SQ_pAutoRex8_2
</TD><TD><CODE>5'- TATCGGTAACCGAGTCCG - 3'</CODE></TD></TR><TR><TD>AlcR_QCI_fw
</TD><TD><CODE>5'- GGCTAAATGTTCCTGCTGACATAGCCGCCGATG - 3'</CODE></TD></TR><TR><TD>AlcR_QCI_rv
</TD><TD><CODE>5'- CATCGGCGGCTATGTCAGCAGGAACATTTAGCC - 3'</CODE></TD></TR><TR><TD>AlcR_QCII_fw
</TD><TD><CODE>5'- GCACGGTATGGGTGCACGTTTGGACAAGCTACTCGAAAATG - 3'</CODE></TD></TR><TR><TD>AlcR_QCII_rv
</TD><TD><CODE>5'- CATTTTCGAGTAGCTTGTCCAAACGTGCACCCATACCGTGC - 3'</CODE></TD></TR><TR><TD>AlcR_QCIII_fw
</TD><TD><CODE>5'- CATCCAGTCTTGGTACGTCATTTTGGACGGTCACTGGCAT - 3'</CODE></TD></TR><TR><TD>AlcR_QCIII_rv
</TD><TD><CODE>5'- ATGCCAGTGACCGTCCAAAATGACGTACCAAGACTGGATG - 3'</CODE></TD></TR><TR><TD>AlcR_QCIV_fw
</TD><TD><CODE>5'- GCCGCAGCTGCAAGTACCCCGCTAG - 3'</CODE></TD></TR><TR><TD>AlcR_QCIV_rv
</TD><TD><CODE>5'- ATTTTTGAGAAAAATATATATTCAGGCGAATTGCACAATGAACAATAATAAGATTAAAATAG - 3'</CODE></TD></TR><TR><TD>pActin_QC_fw
</TD><TD><CODE>5'- ATTTTTGAGAAAAATATATATTCAGGCGAATTGCACAATGAACAATAATAAGATTAAAATAG - 3'</CODE></TD></TR><TR><TD>pActin_QC_rv
</TD><TD><CODE>5'- CTATTTTAATCTTATTATTGTTCATTGTGCAATTCGCCTGAATATATATTTTTCTCAAAAAT - 3'</CODE></TD></TR><TR><TD>NPT_QCI_fw
</TD><TD><CODE>5'- GTCCGGTGCCCTGAATGAATTGCAGGACGAG - 3'</CODE></TD></TR><TR><TD>NPT_QCI_rv
</TD><TD><CODE>5'- CTCGTCCTGCAATTCATTCAGGGCACCGGAC - 3'</CODE></TD></TR><TR><TD>NPT_QCII_fw
</TD><TD><CODE>5'- GCGGGACCCAAGCTTTAGATCTTGCTGCG - 3'</CODE></TD></TR><TR><TD>NPT_QCII_rv
</TD><TD><CODE>5'- CGCAGCAAGATCTAAAGCTTGGGTCCCGC - 3'</CODE></TD></TR><TR><TD>IRES_polio_fw
</TD><TD><CODE>5'- AAGCTTGAATTCGCGGCCGCTTCTAGAGTTAAAACAGCTCTGGGGTTG - 3'</CODE></TD></TR><TR><TD>IRES_polio_rv
</TD><TD><CODE>5'- TTGCAACTGCAGCGGCCGCTACTAGTAAATCCAATTCGCTTTATGATAACAATC - 3'</CODE></TD></TR><TR><TD>BPU_SDMI_r_for
</TD><TD><CODE>5'- CCACTTAAGCATTTTCTACCAGTCGATCACACCATTCACG - 3'</CODE></TD></TR><TR><TD>BPU_SDMI_r_rev
</TD><TD><CODE>5'- CGTGAATGGTGTGATCGACTGGTAGAAAATGCTTAAGTGG - 3'</CODE></TD></TR><TR><TD>Strep-TEV_for
</TD><TD><CODE>5'- CTAGATGGCCGGCAGCGCTTGGTCTCATCCTCAATTTGAAAAGGGTGAAAATCTTTATTTTCAAAGCGGAA - 3'</CODE></TD></TR><TR><TD>Strep-TEV_rev
</TD><TD><CODE>5'- CCGGTTCCGCTTTGAAAATAAAGATTTTCACCCTTTTCAAATTGAGGATGAGACCAAGCGCTGCCGGCCAT - 3'</CODE></TD></TR><TR><TD>SQ_pluc-Actin_1
</TD><TD><CODE>5'- CCGACTCAAATACAGATATGC - 3'</CODE></TD></TR><TR><TD>SQ_pluc-Actin_2
</TD><TD><CODE>5'- CTCGGATGTAGATCTGCG - 3'</CODE></TD></TR><TR><TD>pActin_QC_ins_fw
</TD><TD><CODE>5'- ACCCCCCCCTCTCCTCCCATCCCCCCAACCCTACCACCACCACCACCACCACCTCCACCTCCTCCCCCCTCGCTGCCGGACG - 3'</CODE></TD></TR><TR><TD>pActin_QC_ins_rv
</TD><TD><CODE>5'- CGTCCGGCAGCGAGGGGGGAGGAGGTGGAGGTGGTGGTGGTGGTGGTGGTAGGGTTGGGGGGATGGGAGGAGAGGGGGGGGT - 3'</CODE></TD></TR><TR><TD>pActin_Oryza_fw
</TD><TD><CODE>5'- AAGCTTTCTAGAGTAGCTAGCATACTCGAGGTCATTCATATGCTTG - 3'</CODE></TD></TR><TR><TD>pActin_Oryza_rv
</TD><TD><CODE>5'- TTCGAACTGCAGCGGCCGCTACTAGTACTTCTACCTACAAAAAAGCTCCG - 3'</CODE></TD></TR><TR><TD>GGGGSx5_TEV_fw
</TD><TD><CODE>5'- CTAGATGGCCGGCGGTGGAGGCGGTTCAGGCGGAGGTGGCTCTGGCGGTGGCGGATCGGGAGGCGGTGGCTCCGGAGGTGGAGGCTCAGGAGAGAATTTGTATTTTCAGTCAGGAA - 3'</CODE></TD></TR><TR><TD>GGGGSx5_TEV_rv
</TD><TD><CODE>5'- CCGGTTCCTGACTGAAAATACAAATTCTCTCCTGAGCCTCCACCTCCGGAGCCACCGCCTCCCGATCCGCCACCGCCAGAGCCACCTCCGCCTGAACCGCCTCCACCGCCGGCCAT - 3'</CODE></TD></TR><TR><TD>mMCSII_fw
</TD><TD><CODE>5'- AATTCGCGGCCGCTTCTAGAGCAATTGAAGCTTCCTGCAGGTA - 3'</CODE></TD></TR><TR><TD>mMCSII_rv
</TD><TD><CODE>5'- CTAGTACCTGCAGGAAGCTTCAATTGCTCTAGAAGCGGCCGCG - 3'</CODE></TD></TR><TR><TD>FluA_tripmut_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCGACGTGTACCACGACGGTG - 3'</CODE></TD></TR><TR><TD>FluA_tripmut_rv
</TD><TD><CODE>5'- TTCGAAACCGGTATTGTTGACCTTGCAGGCGG - 3'</CODE></TD></TR><TR><TD>pActin_QCI_fw
</TD><TD><CODE>5'- CATAATTTTTGAGAAAAATATATATTCAGGCGAATTGTCACAATGAACAATAATAAGATTAAAATA - 3'</CODE></TD></TR><TR><TD>pActin_QCI_rv
</TD><TD><CODE>5'- TATTTTAATCTTATTATTGTTCATTGTGACAATTCGCCTGAATATATATTTTTCTCAAAAATTATG - 3'</CODE></TD></TR><TR><TD>Alc R fw Seq.
</TD><TD><CODE>5'- CAGAGCACAATTGTCCATAC - 3'</CODE></TD></TR><TR><TD>Alc R rv Seq.
</TD><TD><CODE>5'- TCGCGGTCGATGCTCTCC - 3'</CODE></TD></TR><TR><TD>pActin_QCins_fw
</TD><TD><CODE>5'- CATCGCCACTATATACATACCCCCCCCTCTCCTCCCATCCCCCCAACCCTACCACCACCACCACCACCACCTCCACCTCCTCCCCCCTCGCTGCCGGACGACGAGCTCCTC - 3'</CODE></TD></TR><TR><TD>pActin_QCins_rv
</TD><TD><CODE>5'- GAGGAGCTCGTCGTCCGGCAGCGAGGGGGGAGGAGGTGGAGGTGGTGGTGGTGGTGGTGGTAGGGTTGGGGGGATGGGAGGAGAGGGGGGGGTATGTATATAGTGGCGATG - 3'</CODE></TD></TR><TR><TD>PP1_for
</TD><TD><CODE>5'- GCCGCTTCTAGATGGCCGGCGGATCCGCGGATTTAGATA - 3'</CODE></TD></TR><TR><TD>PP1_rev
</TD><TD><CODE>5'- GTATTAACCGGTTTTCTTTGCTTGCTTTGTGATC - 3'</CODE></TD></TR><TR><TD>PhyB fw Seq
</TD><TD><CODE>5'- GCGATTTCTCAGTTACAGGC - 3'</CODE></TD></TR><TR><TD>PhyB rv Seq.
</TD><TD><CODE>5'- CTCAAAGCACGAGAAAGAAGC - 3'</CODE></TD></TR><TR><TD>pAct_ins_fw
</TD><TD><CODE>5'- ACCACCACCACCTCCACCTCCTCCCCCCTCGCTGCCGGACGACGAGCTC - 3'</CODE></TD></TR><TR><TD>pAct_ins_rv
</TD><TD><CODE>5'- GGTGGTGGTAGGGTTGGGGGGATGGGAGGAGAGGGGGGGGTATGTATATAGTGGCGATGG - 3'</CODE></TD></TR><TR><TD>PppAct_fw
</TD><TD><CODE>5'- AAGCTTGAATTCGCGGCCGCTTCTAGATCGAGTCCTTGTGACTTTTGTGCAT - 3'</CODE></TD></TR><TR><TD>PppAct_rv
</TD><TD><CODE>5'- TTCGAAACTAGTATATTTAATACGGACCTGCACAACAACCACCAAT - 3'</CODE></TD></TR><TR><TD>PppActin_QC1_fw
</TD><TD><CODE>5'- TTATTGAGGATATCCATCTATCTTGATTTTGGACAATGTTTTACTGC - 3'</CODE></TD></TR><TR><TD>PppActin_QC1_rv
</TD><TD><CODE>5'- GCAGTAAAACATTGTCCAAAATCAAGATAGATGGATATCCTCAATAA - 3'</CODE></TD></TR><TR><TD>PppActin_QC2_fw
</TD><TD><CODE>5'- ACACTCTACATTCATGTCAAGAGTATAGGGACTTGGG - 3'</CODE></TD></TR><TR><TD>PppActin_QC2_rv
</TD><TD><CODE>5'- CCCAAGTCCCTATACTCTTGACATGAATGTAGAGTGT - 3'</CODE></TD></TR><TR><TD>PppActin_fw_corr
</TD><TD><CODE>5'- AAGCTTGAATTCGCGGCCGCTTCTAGAGCGAGTCCTTGTGACTTTTGTGCAT - 3'</CODE></TD></TR><TR><TD>SYFP2_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCGTTAGCAAGGGCGAAGAAC - 3'</CODE></TD></TR><TR><TD>SYFP2_rv
</TD><TD><CODE>5'- TTCGAAACCGGTTTTATACAGCTCATCCATACCCAG - 3'</CODE></TD></TR><TR><TD>eCFP_fw
</TD><TD><CODE>5'- AAGCTTTCTAGATGGCCGGCGTGAGCAAGGGCGAGGAG - 3'</CODE></TD></TR><TR><TD>eCFP_rv
</TD><TD><CODE>5'- TTCGAAACCGGTCTTGTACAGCTCGTCCATGCC - 3'</CODE></TD></TR><TR><TD>G-TEV-site-G_fw
</TD><TD><CODE>5'- CTAGATGGCCGGCGGAGAGAATTTGTATTTTCAGTCAGGAA - 3'</CODE></TD></TR><TR><TD>G-TEV-site-G_fw
</TD><TD><CODE>5'- CCGGTTCCTGACTGAAAATACAAATTCTCTCCGCCGGCCAT  - 3'</CODE></TD></TR><TR><TD>PppActin_seq_fw
</TD><TD><CODE>5'- CATGTCCCAGGGCCCTAG - 3'</CODE></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="6._Instruments">6. Instruments</SPAN></H3><UL><LI>Autoclaves
</LI><LI>Varioklav Typ 500 (H &amp; P Labortechnik GmbH)
</LI></UL><UL><LI>Balances
</LI><LI>XB2200C (Precisa Gravimetrics AG, Dietikon, Switzerland)
</LI><LI>125A (Precisa Gravimetrics AG, Dietikon, Switzerland
</LI></UL><UL><LI>Chromatography: Affinity chromatography
</LI><LI>Flowphotometer
</LI><LI>2138 Uvicord SII (Amersham Pharmacia Biotech Europe, Freiburg, Germany)
</LI></UL><LI>Peristaltic pump
</LI><LI>Pump-P1 (Amersham Pharmacia Biotech Europe, Freiburg, Germany)
</LI><LI>Fraction collector
</LI><LI>GradiFrac (Amersham Pharmacia Biotech Europe, Freiburg, Germany)
</LI><UL><LI>Chromatography: Size exclusion chromatography
</LI><LI>ÄKTApurifiers (GE Healthcare Europe, Germany)
</LI><LI>Tricorn 10/30 Superdex 75 
</LI><LI>Tricorn 10/30 Superdex 20
</LI><LI>HiLoad 16/60 Superdex 75
</LI><LI>HiLoad 16/60 Superdex 200
</LI></UL><UL><LI>Electrophoresis
</LI><LI>Agarose gel electrophoresis chamber (Biometra GmbH)
</LI><LI>Polyacrylamide gel electrophoresis chamber (Biometra GmbH)
</LI></UL><UL><LI>Geldocumentation
</LI><LI> Scanner
</LI><LI> Epson Perfection V700 Photo	(EPSON Deutschland GmbH, Germany)
</LI></UL><UL><LI>Mass spectrometer (ESI-TOF)
</LI><LI> maXisTM Q-TOF mass spectrometer (Bruker Daltonik GmbH, Bremen, Germany)
</LI></UL><UL><LI>pH-Meter
</LI><LI>inoLab Series pH 720 
</LI></UL><UL><LI>Spectrometers
</LI><LI> Nanodrop 2000 (Thermo Fisher Scientific Inc., St. Leon-Rot)
</LI><LI> CD	J810 Spektropolarimeter  (Jasco Research, Victoria, Kanada)
</LI><LI> UV-Vis Ultrospec 2100 pro (Amersham Bioscience Europe, Freiburg)
</LI><LI> ELISA SpectraMax 250 ELISA Reader (Molecular Devices GmbH, Ismaning)
</LI><LI> Synergy 2 ELISA plate reader (BioTek Instruments GmbH, Bad Friedrichshall)
</LI></UL><UL><LI>Thermocycler
</LI><LI>Mastercycler Gradient (Eppendorf AG, Hamburg)	
</LI></UL><UL><LI>Zentrifuges
</LI><LI> Eppendorf
</LI><LI> Sigma 4K10 			(Sigma GmbH, Osterode)
</LI><LI> Sigma 4K15 			(Sigma GmbH, Osterode)
</LI></UL><H3><SPAN class="mw-headline" id="7._Computer_programs">7. Computer programs</SPAN></H3><UL><LI>Biocapt MW; version 10.04 (Software for gel documentation)
</LI><LI>ChemDraw Ultra; version 12.0 (Creation of digital chemical structures)
</LI><LI>Chimera; version 1.7 (Visualization of PDB-files)
</LI><LI>Geneious; version 4.8.5 (Organization of DNA sequences)
</LI><LI>Gimp; version 2.8.2 (Photo 
</LI><LI>GraphPad Prism 6; 
</LI><LI>Inkscape; Version 0.48.3.1 
</LI><LI>Compass Data Analysis Software; Version 4.0	
</LI><LI>OligoCalc	
</LI><LI>ProtParam
</LI><LI>SignalP 4.1 Server (<A href="http://www.cbs.dtu.dk/services/SignalP/" class="external free" rel="nofollow">http://www.cbs.dtu.dk/services/SignalP/</A>)
</LI><LI>TMHMM Server v. 2.0 (<A href="http://www.cbs.dtu.dk/services/TMHMM/" class="external free" rel="nofollow">http://www.cbs.dtu.dk/services/TMHMM/</A>) 	
</LI><LI>UNICORN; Version 5.20
</LI></UL><H2><SPAN class="mw-headline" id="Moss-cultivation">Moss-cultivation</SPAN></H2><UL><LI>Cellophane was used to grow the plants on agar plates in order to be able to pick them easily for subsequent experiments.
</LI><LI>Sterile filters 
</LI><LI>Vacuum manifold
</LI></UL><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Molecular_Biology_Methods">Molecular Biology Methods</SPAN></H3><H4><SPAN class="mw-headline" id="Isolation_of_Plasmid_DNA_from_E._coli_.28miniprep.29">Isolation of Plasmid DNA from <I>E. coli</I> (miniprep)</SPAN></H4><P>Plasmid DNA from <I>E. coli</I> was isolated from overnight cultures using the DNA extraction mini-prep kit (Qiagen). The principle of this method is alkaline lysis of bacterial cells followed by a selective immobilization of the plasmid DNA on a column, subsequent washing steps to remove impurities and the elution of plasmid DNA.
</P><H4><SPAN class="mw-headline" id="Determination_of_DNA_Concentration">Determination of DNA Concentration</SPAN></H4><P>DNA concentration was measured using a NanoDrop Spectrophotometer (Thermo Scientific). The concentration was calculated after determination of DNA specific absorbance at 260 nm. Furthermore, the ratio of sample absorbance at 260 and 280 nm as well as at 260 and 230 nm were measured to specify the purity of the samples. A ratio of 260/280 of ~1.8 is generally accepted as “pure” for DNA. If the ratio is appreciably lower, it may indicate contamination with proteins. The 260/230 nm ratio indicates contamination with thiocyanates and phenolate ions since these absorb at 230 nm. The value is expected to be in the range of 1.8-2.2 in case DNA is relatively pure.
</P><H4><SPAN class="mw-headline" id="Agarose_Gel-Electrophoresis">Agarose Gel-Electrophoresis</SPAN></H4><P>Agarose gel-electrophoresis was used to separate double-stranded DNA fragments by length. Ethidium bromide was applied as a nucleic acid stain (Sambrook et al., 1989). This method was used for the restriction analysis of plasmids (analytical gel-electrophoresis) as well as for the isolation of DNA fragments (preparative gel-electrophoresis). After preparative gel-electrophoresis, the bands were cut out and purified using a Qiagen Gel extraction kit.
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 1:</B>TAE buffer (pH 8.0)
</CAPTION><TBODY><TR><TD>Tris/H
</TD><TD>40 mM
</TD></TR><TR><TD>Acetic acid
</TD><TD>20 mM
</TD></TR><TR><TD>EDTA
</TD><TD>1 mM
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Polymerase_Chain_Reaction_.28PCR.29">Polymerase Chain Reaction (PCR)</SPAN></H4><P>Polymerase chain reaction (PCR) was used for the selective amplification of desired DNA fragments (for example from a plasmid). Primers were designed for the desired target sequences. 
The PCR reaction was divided in to three steps which were repeated up to 30 times. Firstly, the DNA template strand was heat-denatured at 95 °C to produce single-stranded DNA. Secondly, the temperature of the reaction batch was lowered to 55 – 60 °C to allow the primers to bind. Thirdly, the temperature was raised to 72 °C. This enabled the DNA polymerase to synthesize the other DNA strand. Special PCR methods that were used include colony and genomic PCR.
</P><H5><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H5><P>A method to allow for higher throughput of clone screenings. Colonies were picked with a sterile toothpick or pipet tip. Some of the cells were smeared onto the wall of the PCR tube. Subsequently the toothpick was put into a cell culture tube with LB-medium and suitable antibiotic. Colony PCR was performed using OneTaq Hot Start DNA Polymerase (Qiagen) following this temperature scheme:
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 2:</B> PCR protocol
</CAPTION><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>10 min
</TD></TR><TR><TD>30 cycles
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>59 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1 min 55 sec
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Purification_of_PCR_products">Purification of PCR products</SPAN></H5><P>PCR products were purified using the PCR purification kit by Qiagen.
</P><H4><SPAN class="mw-headline" id="Dephosphorylation_of_DNA">Dephosphorylation of DNA</SPAN></H4><P>Dephosphorylation of digested PCR products via Shrimp Alkaline Phosphatase (Fermaentas) was done to avoid religation of the insert and enhance ligation rate.
Before dephosphorylation was performed, digest solution of restriction enzymes and buffer were purified with PCR Purification Kit (Qiagen). Afterwards 1 µg sample DNA was mixed with 10% of 10x SAP Buffer and 1 unit SAP. The mixture was incubated at 37 °C for 30 min. Inactivation occurred at 65 °C for 15 min.
</P><H4><SPAN class="mw-headline" id="DNA_Restriction_Enzyme_Digest">DNA Restriction Enzyme Digest</SPAN></H4><P>For the preparation of DNA fragments and the restriction analysis of plasmid DNA, DNA was cut using restriction endonucleases. Buffers and DNA concentrations were used according to the manufacturer's suggestions.
</P><H4><SPAN class="mw-headline" id="Ligation_.2F_Cycled_Ligation">Ligation / Cycled Ligation</SPAN></H4><P>After digestion with an restriction enzyme, plasmid fragments were inserted into vectors (which were cut with matching restriction enzymes) by ligation. The enzyme T4 ligase connected complementary overhangs of fragments by catalyzing the formation of the bond between the 5'phosphoryl group and the 3' hydroxyl group.
</P><H4><SPAN class="mw-headline" id="Oligohybridization_of_Single-Stranded_DNA">Oligohybridization of Single-Stranded DNA</SPAN></H4><P>Oligohybridization of oligo-nucleotides was used to create a mini MCS for cloning RFC10 compatible parts between promoter and terminator. This was achieved by using complementary oligo-nucleotides that contained the desired sequence with specific overhangs for cloning. For oligohybridization, 25 ml of 100 mM of forward and reverse oligos were put together in one tube and heated to 90 °C for 5 min. The samples were slowly cooled to room temperature in a styrofoam box overnight.
</P><H4><SPAN class="mw-headline" id="Site-Directed_Mutagenesis">Site-Directed Mutagenesis</SPAN></H4><P>Site-Directed Mutagenesis was used to mutate specific bases of DNA sequences. Therefore, specific primers, which bind at the same site and contain a mismatch at the specific base, were required. The original base pair that had to be replaced was replaced by the mismatch. The method works just as PCR by amplifying the desired product that contains the mismatch. Afterwards, the product was digested with the restriction enzyme DpnI to destroy the plasmids strands which do not contain the desired base pair exchange. The QuikChange Site-Directed Mutagenesis Kit by Agilent Technologies was used.
</P><H4><SPAN class="mw-headline" id="Sequencing_of_Plasmid_DNA">Sequencing of Plasmid DNA</SPAN></H4><P>DNA constructs were sequenced by <A href="http://eurofinsdna.com/" class="external text" rel="nofollow">Eurofins mwg operon</A> using our own sequencing primers.
</P><H3><SPAN class="mw-headline" id="Protein_Biochemical_Methods">Protein Biochemical Methods</SPAN></H3><H4><SPAN class="mw-headline" id="Protein_production_in_E._coli">Protein production in <I>E. coli</I></SPAN></H4><P>In order to produce our desired proteins we used the <I>E.coli</I> strain BL21 and the expression system pBAD. Transformed cells were put in a 4 ml culture tube with LB and 1:500 kanamycin to create a homogenous monoclonal bacteria solution. After a few hours 50 ml flasks were inoculated as a preparatory culture and incubated at 30°C overnight. Each 50 ml flask was then transferred in a 2l flask with LB and 1:500 kanamycin and incubated at 25°C till an OD of 0.8 is reached. Subsequently a concentration of 5mM arabinose was used to induce the production of our protein. After another 5 hours at 25°C our cells were ready to be harvested.
</P><H4><SPAN class="mw-headline" id="Cell_disruption">Cell disruption</SPAN></H4><P>To extract our protein we centrifuged our cell cultures and discarded the supernatant. The pellet of 1 l suspension was resuspended in 20 ml 0,5 mg/ml lysozyme solution and rolled for 3 hours at 4°C. For cell disruption we used a sonifier keeping the solution at under 12°C at all times.
</P><H4><SPAN class="mw-headline" id="Streptavidin-Affinity_chromatography_.28SAC.29">Streptavidin-Affinity chromatography (SAC)</SPAN></H4><P>The isolation of recombinant proteins from crude cell extracts were performed by using streptavidin-affinity chromatography (SAC). In this method, recombinant proteins that are fused to a strep tag can be purified with a streptavidin sepharose column. The crude cell extract was centrifuged at 47000 g at 4° C for 1 hour. Afterwards, the supernatant was dialysed two times against 5 l  1 x SA-buffer for about 3 hours each to remove bacterial biotin.
</P><H4><SPAN class="mw-headline" id="Concentrating_proteins_using_a_centrifugal_filter_unit">Concentrating proteins using a centrifugal filter unit</SPAN></H4><P>Proteins were concentrated  by centrifugation in filter units by Millipore with varying cutoffs. After each round of centrifugation (3 min, 3000g, 4°C) the protein solution was pipeted up and down to avoid plugging of the filter and resulting aggregation of the protein.
</P><H4><SPAN class="mw-headline" id="SDS_Polyacrylamide_Gelelectrophoresis_.28SDS-PAGE.29">SDS Polyacrylamide Gelelectrophoresis (SDS-PAGE)</SPAN></H4><P>Firstly, glass plates were wiped with 70% ethanol and then assembled onto a setting rig. No rubber spacers were added as they were already fixed to the glass plates. 
A 15% resolving gel was made as follows:
</P><UL><LI> 5 ml protogel
</LI><LI> 2.5 ml 4 x Lower Tris (pH 8.8)
</LI><LI> 2,5 ml H<SUB>2</SUB>O
</LI><LI> 50 μl ammonium persulphate (APS) (10%)
</LI><LI> 2,5 μl N,N,N’,N’-tetramethylethylenediamine (TEMED) 
</LI></UL><P>The tube was mixed thoroughly and added to the setting rig between the glass plates, covered with water and left to set for about 30 minutes. Then the water was poured off and a stacking gel was prepared as follows:
</P><UL><LI> 1 ml protogel
</LI><LI> 1.5 ml 4 x Upper Tris
</LI><LI> 3,5 ml H<SUB>2</SUB>O
</LI><LI> 36 μl ammonium persulphate (APS) (10%) 
</LI><LI> 3 μl N,N,N’,N’-tetramethylethylenediamine (TEMED) 
</LI></UL><P>All substances were mixed by inversion. 1 ml was applied on top of the already set resolving gel and a comb was put in. Once the gel was set, the comb was taken out and the wells were cleaned out with sterile water. The set gels were removed from the setting rig and placed in the running rig. 1x running buffer was poured into the rig ensuring the plates were covered.
For protein preparation, 30 µg protein in 10 µl H<SUB>2</SUB>0 and 2,5 µl 5 x Laemmli buffer were denatured for 5 minutes at 95 °C. Then 6 µl marker (unstained marker for coomassie-staining or prestained marker for Western Blot) was put into a well. The remaining wells were filled with 12,5 µl of the protein that was to be analyzed. The SDS-PAGE was performed at 120 V for about 1,5 h. For coomassie staining, the gel was incubated in coomassie staining dye for 20 minutes, then put into the first decolorizing solution for 20 minutes. Finally. the gel was put into a second decolorizing solution until the background color was gone. Another method that was used for protein staining was silver staining.
</P><H4><SPAN class="mw-headline" id="Silver_staining_of_polyacrylamide_gels">Silver staining of polyacrylamide gels</SPAN></H4><P><I>Reagents</I></P><UL><LI> Fixation solution: (50:5:45 v/v/v methanol : acetic acid : water) -&gt; 50 ml
</LI><LI> Sensitising solution: (0.02 % sodium thiosulfate) -&gt; 50 ml
</LI><LI> Coloring solution: 0.01 % AgNO3 -&gt; 50 ml
</LI><LI> Delevoping solution: (0.04 % formaldehyde in 2 % sodium carbonate) -&gt; 200 ml
</LI><LI> Safe-keeping solution: 1 % acetic acid -&gt; 200 ml
</LI></UL><P><I>Methods</I><B>1.</B> After the gel has been run, fix the protein by incubating the gel slab in fixation solution 20-30 minutes. <B>2.</B> Rinse the gel slab with water (2 changes, two minutes per change) and then leave it further in water for one hour on a shaking platform.  -&gt; !!Extended washing time helps to eliminate yellowish background usually observed after long developing of the gel!!<B>3.</B> Sensitise the gel with sensitising solution for 1-2 minutes. Discard solution and quickly rinse the gel slab with two changes of water (10 seconds each).  -&gt; !!Agitate gently to make sure that the gel slab is covered evenly!!<B>4.</B> Incubate the gel in chilled 0.1 % AgNO3 for 30 minutes at 4 °C (fridge). <B>5.</B> Discard silver nitrate solution and quickly rinse the gel with two changes of water (30 seconds per each change). <B>6.</B> Develop the gel with developing solution. Discard the developing solution as soon as it turns yellow and replace it with a fresh portion. <B>7.</B> When a sufficient degree of staining has been obtained, quench staining by discarding the developing solution and replacement with 1 % acetic acid. Was the gel with 1 % acetic      acid several times and store in the same solution.  -&gt; !!Silver stained gels can be stored at 4 °C in 1 % acetic acid for months. In some cases the colour of the stained protein bands might slightly change in time. However these changes do not affect the results of mass spectrometric sequencing!!</P><H4><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H4><P>After SDS-PAGE, gels were transferred to a nitrocellulose membrane in transfer buffer (20 ml 5x SDS buffer, 20 ml methanol, 60 ml H<SUB>2</SUB>O) at 500 mA for 1h. Membranes were washed 3x 15 min in PBS-T0.1 (PBS + 0,1 % v/v Tween 20) and subsequently blocked with 3 % BSA for one hour. For antibody detection, membranes were washed 3x 15 min in PBS-T0.1 and then incubated in detection solution containing the antibody straptavidin-AP (1:4000 in PBS-T0.1) for one hour. The Western blot was washed 2x 10 min in PBS-T0.1 and then 2x 10 min in PBS. Afterwards, the developing solution (15 ml alkaline phosphatase buffer, 45 µl BCIP (50 mg/ml in DMF) and 7,5 µl NBT (75 mg/ml in 70 % DMF) was added. After the appearance of bands, the blot was washed and stored in water.
</P><H3><SPAN class="mw-headline" id="Microbiological_Methods">Microbiological Methods</SPAN></H3><H4><SPAN class="mw-headline" id="Cultivation_of_E.coli">Cultivation of <I>E.coli</I></SPAN></H4><P><I>E.coli</I> strain XL1-Blue was cultivated in LB-medium (lysogeny broth) and LB plates. 
for the preparation of 1 liter of LB, dissolve the following and autoclave:
</P><UL><LI> 10 g tryptone
</LI><LI> 5 g yeast extract
</LI><LI> 10 g NaCl
</LI></UL><P>For making plates, add 15 g bacto-agar before autoclaving.
</P><H4><SPAN class="mw-headline" id="Heat_Shock_Transformation_of_E.coli_with_Plasmid_DNA">Heat Shock Transformation of <I>E.coli</I> with Plasmid DNA</SPAN></H4><P>Before transformation, CaCl<SUB>2</SUB> competent cells were produced after Cohen et al., 1972. For the production of competent cells, 50 ml LB medium were inoculated with an overnight culture of the used ‘’E.coli’’ strain and incubated at 37 °C, 180 rpm. After an OD <SUB>550</SUB> of 0,5 was reached, the culture was centrifuged for 4 minutes at 5000 g for 10 minutes. The pellet was then resuspended in 40 ml pre-chilled in 0,1 M MgCl<SUB>2</SUB> solution, centrifuged again and resuspended in 20 ml of pre-chilled 0,05 M CaCl<SUB>2</SUB> solution. After 30 minutes of incubation on ice, the cells were centrifuged and resuspended in 2 ml 0,05 M CaCl<SUB>2</SUB> solution, 15 % v/v glycerol. The competent cells were aliquoted and stored an – 80 °C.
For the transformation, 100 µl competent cells and 1 ng plasmid or 5 µg of a ligation product were mixed and incubated for 30 minutes on ice. Afterwards, the cells were heat shocked at 37 °C for 5 minutes, then mixed with 2 ml LB medium and incubated at 180 rpm and 37 °C for 30-45 minutes. The transformed cells were then plated on LB medium containing an antibiotic.
</P><H3><SPAN class="mw-headline" id="Physcomitrella_Methods">Physcomitrella Methods</SPAN></H3><P>The techniques used for the cultivation and manipulation of <I>Physcomitrella patens</I> are based on the knowledge of the chair for Plantbiotechnology of Prof. Dr. Reski at Freiburg University which are availible at <A href="http://www.plant-biotech.net" class="external text" rel="nofollow">Plant-Biotech.net</A>.
</P><H4><SPAN class="mw-headline" id="Preparation_of_Medium_and_Solutions">Preparation of Medium and Solutions</SPAN></H4><P>According to Reski and Abel, 1985:
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 3:</B> Stock solutions 100x
</CAPTION><TBODY><TR><TD>25 g/l
</TD><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD></TR><TR><TD>25 g/l
</TD><TD>KCl
</TD></TR><TR><TD>25 g/l
</TD><TD>MgSO<SUB>4</SUB> x 7 H2O
</TD></TR><TR><TD>100 g/l
</TD><TD>Ca(NO<SUB>3</SUB>)<SUB>2</SUB></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><CAPTION><B>Table 4:</B> Knopp medium
</CAPTION><TBODY><TR><TD>10 ml/l
</TD><TD>stock solutions
</TD></TR><TR><TD>12.5 mg/l
</TD><TD>FeSO<SUB>4</SUB>x 7 H2O
</TD></TR></TBODY></TABLE><P>The pH is adjusted to pH 5.8 with KOH or HCl, then the medium is sterilized by autoclaving.
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 5:</B> Knop media agar plates
</CAPTION><TBODY><TR><TD>1x
</TD><TD>Knop medium
</TD></TR><TR><TD>1.2% (w/v)
</TD><TD>agar (Oxoid Ltd.)
</TD></TR></TBODY></TABLE><P>The pH is again adjusted to pH 5.8 with KOH or HCl, the agar medium is sterilized by autoclaving and poured into dishes while still warm.
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 6:</B> Knopp regeneration medium
</CAPTION><TBODY><TR><TD>stock solutions
</TD></TR><TR><TD>12.5 mg/l
</TD><TD>FeSO<SUB>4</SUB> x 7 H<SUB>2</SUB>O
</TD></TR><TR><TD>50 g/l
</TD><TD>glucose
</TD></TR><TR><TD>30 g/l
</TD><TD>mannitol
</TD></TR></TBODY></TABLE><P>The pH is adjusted with KOH to pH 5.8 and the medium is sterile filtered. The osmolarity is approximately 540 mOs. Sterilize by filtering (0.22 µm). 
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 7:</B> 3M medium for the transformation
</CAPTION><TBODY><TR><TD>3.05 g
</TD><TD>MgCl<SUB>2</SUB></TD></TR><TR><TD>1 g
</TD><TD>MES (2-(N-morpholino)-ethanesulfonic acid)
</TD></TR><TR><TD>87.4 g
</TD><TD>mannitol
</TD></TR><TR><TD>800 ml
</TD><TD>H20
</TD></TR></TBODY></TABLE><P>The pH is adjusted to 5.6 with KOH. The osmolarity is approximately 580 mOs. Sterilize by autoclaving. 
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 8:</B> 0.5 M Mannitol solution for the transformation
</CAPTION><TBODY><TR><TD>91.1 g
</TD><TD>mannitol
</TD></TR><TR><TD>800 ml
</TD><TD>H<SUB>2</SUB>0
</TD></TR></TBODY></TABLE><P>The pH is adjusted to pH 5.8 with KOH or HCl and H<SUB>2</SUB>0 is added to 1 l. The osmolarity is approximately 560 mOs. Sterilize by autoclaving. 
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 9:</B> 4 % (w/v) Driselase solution for the transformation
</CAPTION><TBODY><TR><TD>0.4 g
</TD><TD>Driselase (Sigma)
</TD></TR><TR><TD>10 ml
</TD><TD>0.5 M mannitol
</TD></TR></TBODY></TABLE><P>Mix by vortexing and cover the tube with aluminum foil for light protection. Incubate for 1 h at room temperature while mixing, e.g. on a rotating table. Centrifuge at 2300 x g for 10 min and filter sterilize the supernatant (0.22 μm filter).
</P><P><B>Knopp full medium with G418 for agar plates:</B></P><P>Prepare the following stock solutions:
</P><TABLE cellspacing="0" border="1"><CAPTION><B>Table 10:</B> Stock solutions for Knopp full medium.
</CAPTION><TBODY><TR><TD>reagent
</TD><TD>stock
</TD><TD>composition
</TD></TR><TR><TD>H<SUB>3</SUB>BO<SUB>3</SUB></TD><TD>1000x
</TD><TD>154.6 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>MnSO<SUB>4</SUB></TD><TD>1000x
</TD><TD>377.5 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>ZnSO<SUB>4</SUB>*7H<SUB>2</SUB>O
</TD><TD>1000x
</TD><TD>215.61 mg/50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>(NH)<SUB>6</SUB>Mo<SUB>7</SUB>O<SUB>24</SUB>*4H<SUB>2</SUB>O 
</TD><TD>1000x
</TD><TD>30.9 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>KI
</TD><TD>1000x
</TD><TD>20.8 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>CuSO<SUB>4</SUB>*5H<SUB>2</SUB>O
</TD><TD>1000x
</TD><TD>0.6 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>CoCl<SUB>2</SUB>*6H<SUB>2</SUB>O
</TD><TD>1000x
</TD><TD>0.6 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Myo-Inositol
</TD><TD>1000x
</TD><TD>21.8 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Nikotinamid
</TD><TD>1000x
</TD><TD>48.8 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Thiamin-HCl
</TD><TD>1000x
</TD><TD>25.3 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Pyridoxin-HCl
</TD><TD>1000x
</TD><TD>12.3 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Biotin
</TD><TD>1000x
</TD><TD>0.5 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>D-Pantothen
</TD><TD>1000x
</TD><TD>43.6 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Riboflavin
</TD><TD>1000x
</TD><TD>0.8 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD><B>palmitic acid</B></TD><TD>1000x
</TD><TD>176.9 mg / 50 ml <B>warm ethanol</B></TD></TR><TR><TD>3-aminobenzoic acid
</TD><TD>500x
</TD><TD>12.3 mg / 100 ml H<SUB>2</SUB>O
</TD></TR><TR><TD>Adenin
</TD><TD>100x
</TD><TD>337.8 mg / 500 ml H<SUB>2</SUB>O
</TD></TR><TR><TD><B>Peptone</B></TD><TD>100x
</TD><TD>12.5 mg / 50 ml H<SUB>2</SUB>O
</TD></TR><TR><TD><B>Glucose</B></TD><TD>10x
</TD><TD>25 g / 500 ml H<SUB>2</SUB>O
</TD></TR></TBODY></TABLE><P>The stock solutions can be stored at -20°C for further use. The peptone stock should be prepared from scratch every time, though. Sterile filter and add the stock solutions to autoclaved, still warm (not hot!) Knop medium agar, working on a laminar flow bench. The glucose solution can be prepared and autoclaved seperately, since sterile filtering it through syringes is no fun due to the amounts needed. Add the plamictic acid last or it will lead to precipitation. You can re-use the filter tip if you can handle it in a sterile way. Last, add the G418 antibiotic for a final concentration of 25 µg/ml. Pour agar into plates while still warm.
</P><H4><SPAN class="mw-headline" id="Cultivation_of_P._patens">Cultivation of <I>P. patens</I></SPAN></H4><P>The standard cultivation conditions for <I>P. patens</I> are 25°C ±1°C under a 16/8 h light/dark photoperiod with a lightintensity of 55 μmol/(m<SUB>2</SUB>*s).
</P><P>In our lab, we cultivated <I>P. patens</I> in Erlenmeyer flasks equipped with a sterile filter in the lid made of aluminum foil. To provide sufficient CO<SUB>2</SUB>, we used compressed air which went through the sterile filter and through an attached Pasteur pipette into the liquid medium. The resulting bubbles helped to continuously stir the culture. The liquid cultures were regularly homogenized using a Ultra Turrax T25 device at 19 000 RPM and the culture was subsequently provided with fresh medium. We also experimented with liquid cultures in the incubator shaker as well as in unstirred medium and with cultivation on solid medium and on medium soaked felt, as well as some other surfaces. We tried to keep the temperature constant by air conditioning, but it was of course subject to some fluctuations. The appropriate light/dark photoperiods were produced by hooking striplights up to a time switch. The whole set-up can be seen on the pictures below and we have installed all this equipment specially for our iGEM project as the laboratory is usually used for protein biochmistry of therapeutic proteins. 
</P><P>After Transformation, the regenerating protoplasts were incubated in regeneration medium in 6-well-plates sealed with parafilm for 10 days and then cultivated on a layer of autoclaved cellophane (seperated with Whatman paper during autoclavation, so they dont stick together), on top of Knop medium agar plates for three days under standard conditions. Then we transferred the cellophane layers with the regenerated moss onto Knop medium agar plates containing 25 µg/ml G418 antibiotic for two weeks. The official protocol schedules a two week release period followed by another selection period to ensure stabile transormation, but we didn´t have that much time and went with a single round of selection. We plated only half of our transformed protoplasts. The other half was left in the 6-well-plates where 2 ml of selection medium were added to the 2 ml of regeneration medium from the transormation, with G418 diluted 1:8000.
</P><H4><SPAN class="mw-headline" id="Homogenization">Homogenization</SPAN></H4><P>The moss in liquid culture in Erlenmeyer flasks has to be homogenized weekly to disintegrate the protonema filaments. The process is carried out on a laminar flow bench to prevent contamination. Before homogenization, we let the moss settle and decanted the supernatant medium. We used an Ultra Turrax at 19.000 rpm. The duration depends on the density of the culture. The homgenized moss is distributed into several previously autoclaved flasks to achieve a lower density. Fresh medium is added to the new flasks and autoclaved air-filters are integrated into their lids.
</P><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>According to the <A href="http://www.plant-biotech.net/methods/Transformation.pdf" class="external text" rel="nofollow">Reski protocol</A></P><UL><LI>Prepare the DNA including the desired gene and the nptII selection casette by linearizing it with an appropriate enzyme, purifying it by ethanol precipitation and suspending it in Ca(NO<SUB>3</SUB>)<SUB>2</SUB> for a concentration of 0.25 µg/µl
</LI><LI>Prepare 4% driselase solution in 0.5 M mannitol, vortex, keep on rotating platform for 45 min (protected from light) and centrifuge at 3500 rpm, 10 min; sterile filter 
</LI><LI>Extraction of moss from bioreactor culture by filtering through protoplast sieve (mesh size 100 µm) 
</LI><LI>Addition of 12 ml 0.5 M mannitol solution and 4 ml of driselase stock solution (final concentration: 1% w/v) and Incubation for 2 h at room temperature on rotating platform (protected from light) 
</LI><LI>Gently filter moss material through protoplast sieve (mesh size 100 µm), then filtrate through 50 µm sieve 
</LI><LI>Centrifugation at 500 rpm for 10 min with slow acceleration rates in a glass tube and discard supernatant; wash protoplasts with mannitol solution, repeat. 
</LI><LI>Resuspend pellet in a total of 10 ml 0.5 M  mannitol 
</LI><LI>Determine concentration of protoplasts/ ml with a hemocytometer, centrifuge the material at 45 g for 10 min and resuspend in the right amount of 3M medium to adjust the concentration to 1.2*10<SUP>6</SUP> live protoplasts per ml.
</LI><LI>Mix 100 µl of 250 ng/µl linearized DNA solution with 350 µl PEG 4000 solution (8 g PEG4000 and 20 g 3M medium, sterile filtered) and 250 µl of the protoplast suspension and incubate 30 min while gently mixing every 5 min 
</LI><LI>Then dilute with 3M medium every 5 min: add 1 ml the first time, then additional 2 ml, then 3, then 4 
</LI><LI>Centrifugation at 500 rpm for 10 min, discard supernatant, resuspend in 4 ml regeneration medium and divide into 2 wells of a 6-well plate 
</LI><LI>Incubation in the dark over night, then a regeneration phase of 10 days at a light/dark regime of 16/8 h 
</LI><LI>Transfer the regenerated protoplasts on solidified Knop medium covered with a sterile cellophane sheet and grow them at standard conditions for 3 days
</LI><LI>Subsequently start selection by placing the cellophane sheet with the moss protoplasts on top of it onto selection plates (if you used the nptII selection cassette, use 25 µg/ml G418) for two weeks
</LI><LI>To ensure stable integration, give the moss a release phase of two weeks without antibiotics followed by another two weeks on selection plates
</LI></UL><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><P>[<A href="http://www.plant-biotech.net/paper/Hohe_and_Reski.pdf" class="external text" rel="nofollow">Hohe A., R. Reski, 2002</A>] Hohe A., R. Reski (2002): Optimisation of a bioreactor culture of the moss Physcomitrella patens for mass production of protoplasts. <I>Plant Sci.</I> 163, 69-74.
</P><P>[<A href="http://www.plant-biotech.net/paper/CurrGenet_2003_hohe.pdf" class="external text" rel="nofollow">Hohe et al., 2004</A>] Hohe, A., T. Egener, J. Lucht, H. Holtorf, C. Reinhard, G. Schween, R. Reski (2004): An improved and highly standardised transformation procedure allows efficient production of single and multiple targeted gene knockouts in a moss, Physcomitrella patens. <I>Current Genet.</I> 44, 339-347.
</P></DIV></DIV><DIV id="social-footer"><DIV id="annotator"><H2>AutoAnnotator:</H2><FORM action="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator" method="get"> or go to the <A href="https://2013.igem.org/Team:TU-Munich/Results/AutoAnnotator">AutoAnnotator</A> page.
  </FORM></DIV><DIV id="follow"><H2>Follow us:</H2></DIV><DIV id="address"><H2>Address:</H2><P>iGEM Team TU-Munich
     Emil-Erlenmeyer-Forum 5
     85354 Freising, Germany</P><P>Email: <A id="mail" href="mailto:igem@wzw.tum.de">igem@wzw.tum.de</A>
     Phone: +49 8161 71-4351</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:TU-Munich/Notebook/Methods">http://2013.igem.org/Team:TU-Munich/Notebook/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Munich/Notebook/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Munich/Notebook/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Munich/Notebook/Methods&amp;oldid=357149" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>