"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Munich/Modeling/Protein Predictions
From 2013.igem.org
Home
Project
Phytoremediation
Physcomitrella
Localization
BioDegradation
BioAccumulation
Kill Switch
Safety
Results
BioBricks
Effector Studies
Moss Methods
Localization
PhyscoFilter
Kill Switch
Implementation
AutoAnnotator
Entrepreneurship
Summary
Modeling
Protein Predictions
Kill Switch
Enzyme Kinetics
Filter Model
Team
Members
Sponsors
Collaborations
Pictures
Timeline
Attributions
Judging
Notebook
Labjournal
Methods
Outreach
Expert Counsel
GMO release
Publicity
Educational Kit
Tutorials
Prediction of Protein Structures and Functions
Structural properties of effector proteins are often essential for their function, so it is advantageous to know about them. It is for example necessary to know whether termini are accessible for protein fusion or whether the protein is only functional in a multimeric fold. For this reason a structure based search was performed in the protein database . As the number of identified structures is still limited, it is a promising attempt to look for homologous proteins whose crystal structures have been determined.
Analysis of Receptor Sequences – Choosing the right template
For several purposes of our project, we needed a synthetic receptor enabling us to integrate proteins into the membrane in the desired orientation, i.e. to express protein-domains on the intracellular or extracellular side of the cell membrane. We investigated several different plant-receptors from the well characterized dicotyledon Arabidopsis thaliana and the moss Physcomitrella patens, our chassis. The receptors from Arabidopsis thaliana have the advantage that their transgenic expression has successfully been demonstrated [ Quail et al., 1991 ] whereas the native receptors from Physcomitrella patens bear only a small risk of failing [ Reski, 1998 ].
Due to the fact that there were many different available receptors, which we could have used as template for our synthetic receptor, we used bioinformatical methods to evaluate the suitability of these receptors. The following three examples ERF, FLS2 and SERK shown in table 1 resulted from this equation.
Table 1: Examined Receptors
Receptor
Organism
Length (aa)
Sequence reference
Literature reference
ERF
A. thaliana
1031
NP_197548.1
FLS2
A. thaliana
1173
NP_199445.1
SERK
P. patens
625
XP_001759122.1
Lienhart, 2007
Prediction of Signal Peptides
Figure 1: Prediction and analysis of signal peptides
Introduction
A first analysis was performed to identify signal-peptides, which are bound by the cellular signal recognition particle and lead to the translocation of the bound polypeptide into the endoplasmic reticulum. Afterwards the signal peptide is cleaved by a signal peptide peptidase at a specific site. The analysis for signal peptides was done by using the SignalP 4.1 Server .
Results
The program was run for different receptors and will be illustrated for the three examples mentioned above (see fig. 1).
The figure shows the N-terminal sequence of the receptors, together with three scores:
(1) The C-Score (raw cleavage site score) in red.
(2) The S-Score (signal peptide score) in green.
(3) The Y-Score (combined cleavage site score) in blue.
The C-Score shows the most probable cleavage site identified by the peptidase. It was possible to identify the most probable cleavage site for all shown receptors with ambiguous cleavage sites for the SERK-receptor. The amino acid with the highest C-score is predicted to be the first amino acid of the primary structure of the cleaved receptor.
The S-Score was developed to identify amino acid sequences which appear in a signal peptide and others that belong to the matured receptor. The course of this parameter is high for the first 23-28 amino acids of all receptors, identifying these residues as signal peptides. The amino acid residue, which lies at the greatest decrease of the S-Score, is the predicted border between the N-terminal signal peptide and the receptor.
The Y-Score results from the geometrical structure of the protein and the previously determined scoring parameters. It illustrates that the two first parameters show a good fit for the identification of the signal peptide in all three indicated receptors.
Discussion
Summarizing these parameters, it can be concluded that all three pictured receptors seem to contain a sequence acting as a signal peptide. For many of the predicted receptors in the genome of Physcomitrella patens the prediction did not yield a positive result. With respect to the signal peptide, all mentioned receptors would be suitable as a template for our synthetic receptor. The predicted data show that the SERK-Receptor is favorable for our application, because its signal peptide is statistically the best recognized one and bears the smallest risk of failure.
Prediction of Transmembrane Regions
Figure 2: Prediction and analysis of transmembrane regions
Introduction
Additional to the identification of the signal peptide, it was very important to identify transmembrane regions within the receptors, because we wanted to use a type I receptor as a template that contains a N-terminal extracellular domain, a transmembrane domain and a C-terminal intracellular domain (see our localization page ). To analyze this issue, the prediction tool TMHMM was used for several different receptors. Again the most suitable receptors were ERK, FLS2 and SERK.
Results
The analysis yields a signal peptide and a single transmembrane domain for all three depicted receptors (see fig. 2). The estimated reliability of the predictions was equally good for all examined receptors, whereas the signal peptide was most reliably predicted for the SERK receptor.
Discussion
Focussing on the membrane topology point of view, all the investigated receptors would be suitable blue prints for our synthetic receptor. As the SERK-Receptor yields the best prediction, it was chosen as the favorable template. Another reason to choose the SERK-Receptor was that it is derived from Physcomitrella patens. The only problem, concerning this prediction, is that the N-terminus of this receptor is predicted to be extracellular. The falsification of this prediction was simple, because the SERK receptor contains a C-terminal kinase-domain, which is known to be involved in signal transduction.
Choice of the SERK Receptor
Finally we decided to use the SERK receptor as a template to generate our synthetic receptor. The final receptor was designed in RFC[25] standard, which allows in frame protein fusions. The final constructs were designed containing the SERK signal peptide ( BBa_K1159303 ), an extracellularly located effector protein, the transmembrane domain of the SERK receptor ( BBa_K1159305 ), a short linker and a GFP, to investigate the cellular localization of our receptor with the aid of fluorescence microscopy.
Searching for Homologous Structures using HHpred
The search for homologous structures was performed by using the freely accessible web server HHpred [ Söding et al., 2005 ]. The amino acid sequences for the BioBricks were translated into amino acid sequences using the AutoAnnotator and was then inserted into the the search field. The results for all proteins investigated in our project are shown in table 2.
Table 2: Predicted Structures
Protein
BioBrick
PDB-code
Identity
Similarity
Structure
XylE
BBa_K147002
3hpy_A
50%
0.939
Laccase
BBa_K1159002
2wsd_A
68%
1.223
NanoLuc
BBa_K1159001
3ppt_A
21%
0.359
EreB
BBa_K1159000
3b55_A
19%
0.318
Spycatcher
BBa_K1159200
2x5p_A
97%
1.298
PP1
Part:BBa_K1159004
3e7a_A
96%
1.593
GFP
BBa_K1159311
2WUR
98%
1.477
Glutathiontransferase / DDT Dehydrochlorinase
<partinfo>BBa_K620000</partinfo>
3F6D
68%
1.155
SERK-TM
BBa_K1159305
2ks1_B
24%
0.233
TEV Protease
Commercial reagent
1Q31
n.d.
n.d.
Streptavidin
Commercial reagent
3RY2
n.d.
n.d.
Results
The homology search showed that some of our effector proteins have very closely related proteins with a known structure. For example there are very similar protein structures available for the SypCatcher, PP1 and GFP each with identities of over 90%. Some other effector proteins such as XylE, Laccase or the DDT Dehydrochlorinase have less homologous proteins, whose structures still give good hints on structural questions. However there are also effectors where only badly matched structures are known, which can only be used as a very rough indication of the fold. The NanoLuc luciferase, which is a highly engineered protein derived from shrimps and was only published this year, is an example of a protein with no known structural homologue.
The structures obtained here were used to design our experiments. A homology modeling for the Laccase was performed to determine whether it contains disulphide bridges. The resulting homologous structures were used as illustrations, as explained in one of our How-Tos about animated GIFs.
References:
[ Quail et al., 1991 ] MARGARET T. Boylan, M.T. and Quail, P.H. (1991). PhytochromeA overexpression inhibits hypocotyl elongation in transgenic Arabidopsis. Proc. Natl. Acad. Sci. 88:10806-10810.
[ Reski, 1998 ] Reski, R. (1998). Development, Genetics and Molecular Biology of Mosses. Bot. Acta, 111:1-15.
[ Lienhart, 2007 ] Lienhart, O. (2007). Untersuchungen zu einem Somatic-Embryogenesis-Receptor-like-Kinase-Homolog in Physcomitrella patens (Hedw.) B.S.G. PhD-thesis at Freiburg University
SignalP41 Server
[ Söding et al., 2005 ] Söding J, Biegert A, Lupas AN. (2005). The HHpred interactive server for protein homology detection and structure prediction. Nucleic Acids Res. 2005 Jul 1;33(Web Server issue):W244-8.
AutoAnnotator:
or go to the AutoAnnotator page.
Follow us:
Address:
iGEM Team TU-Munich
Emil-Erlenmeyer-Forum 5
85354 Freising, Germany
Email: igem@wzw.tum.de
Phone: +49 8161 71-4351
Retrieved from " http://2013.igem.org/Team:TU-Munich/Modeling/Protein_Predictions "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
