<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UC_San_Diego_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UC San Diego/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbar"><DIV class="navlink">People
      <DIV class="navdrop"><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Team">Our Team</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Collaborations">Collaborations</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Attributions">Attributions and Sponsors</A></DIV></DIV></DIV><DIV class="navlink">Application
      <DIV class="navdrop"><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Design">Product Design</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Entrepreneurship">Entrepreneurship</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Database">Database</A></DIV></DIV></DIV><DIV class="navlink">Human Practices
      <DIV class="navdrop"><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Human_Practices_Silver">Human Practices Silver</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Human_Practices">Integrated Human Practices</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Public_Engagement">Education and Public Engagement</A></DIV></DIV></DIV><DIV class="navlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Model">Modeling</A></DIV><DIV class="navlink firstnav">Our Project
      <DIV class="navdrop"><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Design_Philosophy">Our Design Philosophy</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Description">Description</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Applied_Design">Design</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/HC">Hepatocellular Carcinoma</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Notebook">Protocols and Notebook</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Results">Parts and Results</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Medal">Medal Criteria</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/InterLab">InterLab</A></DIV><DIV class="subnavlink"><A href="https://2018.igem.org/Team:UC_San_Diego/Safety">Safety</A></DIV></DIV></DIV></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV id="wrapper"><DIV class="section"><H2>Project Description</H2><H3>Understanding the Current Methods of Cancer Diagnosis and their Limitations </H3><P>Currently, there are several techniques used for the detection of specific cancers, including <B> imaging techniques </B> such as computed tomography (CT), mammography for breast cancer diagnoses, or positron emission tomography (PET) scans. Although imaging techniques are often capable of detecting cancer with high accuracy, they suffer from being unable to distinguish benign from malignant tumors. </P><P>The gold standard for cancer diagnosis is <B>  tissue histopathology </B>, but often has significant drawbacks. For example, techniques such as fine needle aspiration (FNA) and core biopsy are<B> highly invasive </B> and are required for the extraction of suspected tumor tissue and subsequent histological evaluation. The method has also proven to be ineffective. First, comprehensive characterization of multiple tumor specimens obtained from the same patient illustrated spatial heterogeneity as well as recurrences between the primary and local tumor in the same patient. This heterogeneity poses a pivotal challenge to guide clinical decision-making in oncology, as <B> biopsies may be inaccurate in capturing the complete tumor genome </B> for an individual patient. In addition, a significant barrier to biomarker testing is the availability of an adequate amount of tissue due to increasing diagnostic demands and declining amounts of tissue per patient. Finally, tissue biopsies also increase the cost of patient care and the turnaround time for getting results, thus impacting the decision-making process. </P><H5> Figure 1. Common methods for cancer diagnosis involve imaging techniques (such as the PET and CT scans pictured) as well as physically invasive methods such as tissue biopsy. </H5><H3>The Economic and Social Relevance of the Issue</H3><P>In short,<B> biopsies are invasive, expensive procedures that are limited to certain locations and not always feasible in clinical practice. </B> Imaging techniques cannot give enough information on tumor characteristics to direct further treatment. As a result, many cases go untreated until it is too late, leading to increased costs with little results. In 2017 alone, estimated national expenditures for cancer care in the United States topped <B> $145 billion.</B> In future years, costs are likely to increase as the population ages and cancer prevalence increases. However, of these projected cancer patients, nearly a third of them can be treated effectively if found early. For this reason, we believe that one of the biggest problems facing researchers is the lack of an effective early cancer detection technique. </P><H3>Liquid Biopsy: Overview and Bottlenecks</H3><P>To circumvent some of the negative aspects of traditional tissue biopsies, researchers have turned to a new approach, termed <B> liquid biopsy </B> . This approach relies on <B> analyzing bits of tumor material that are found in blood, urine, and saliva. </B> Scientists have unearthed a mechanism that demonstrates how tumors shed some DNA out of the cell, which then begins to circulate throughout the bloodstream and can be more easily analyzed than traditional tissue samples. In recent years, much attention has been focused on utilizing this circulating tumor DNA (termed ctDNA) as a biomarker of disease status. This ctDNA is essentially short nucleic fragments (less than 200 bp)  found in plasma after being released through cell apoptosis. Scientists believe that<B> ctDNA carries the same genetic information that a traditional tissue biopsy would provide </B> , but has the inherent advantage that the information is attained in a minimally invasive manner. In addition, it avoids the issue of tumor heterogeneity because it is able to provide a snapshot of the entire tumor genome. </P><H5> Figure 2. Common targets of analysis for liquid biopsy test include circulating tumor DNA(ctDNA), messenger RNA , and proteins, all part of the central dogma. </H5><P>Although our project focuses on applications in ctDNA analysis, other common targets in liquid biopsy include cell-free RNA (cfRNA) and circulating tumor cells (CTCs). CTCs represent intact, viable cells with malignant features that can be isolated from blood that are released into the bloodstream during the spread of cancer.</P><P>Because the fragments that are crucial to the success of liquid biopsy techniques are so small (only 166 bp or so), diagnostic assays must be optimized for high sensitivity because the concentration of DNA can be significantly lower than those of DNA directly isolated from cancer tissues. In addition, purification and handling of samples still remains a concern. Scientists and clinicians have primarily focused on harnessing the power of next-generation sequencing (NGS) to identify genetic alterations and use that to make guided decisions about therapy, to mixed results so far. The biggest bottleneck can be traced to the rare nature of ctDNA, which makes it very difficult for existing assays on the market to distinguish between the actual signal and “background noise.” </P><H3>A Shift in Cancer Detection Paradigms: Epigenetic Determinants </H3><P>For the most part, cancer is understood as an alteration to the genetic code that, via perpetuation in the central dogma, leads to numerous problems in cellular function over time. As a result, there has been a particular focus on using highly powerful sequencing techniques to identify the slightest mutations in the genetic code. For scientists, ctDNA analysis is seen as a convenient method of acquiring the genomic DNA for subsequent analysis. </P><P><B>But what if there’s another way to look at cancer pathways and regulation in the cell? </B></P><P>Instead of focusing on somatic alterations, our team decided to focus on using <B> epigenetic determinants </B> . The most widely studied epigenetic alteration to date is the methylation ( 5-methylcytosines ) of DNA at the CpG dinucleotides which are highly concentrated in the CpG islands within the promoter region of the gene. Various degrees of methylation within a gene’s CpG islands leads to associated levels of gene silencing, and in cancer, promoter hypermethylation has been linked to the silencing of tumor suppressor genes and tumorigenesis. Screening for gene mutations is a common strategy, but this cannot reflect the current status or activity of the disease. Additionally, promoter methylation is often easier to evaluate due to its defined location within the promoter region of specific genes. This is in contrast to locating gene mutations which can occur at unsuspected sites that may be more challenging to pinpoint. </P><H5> Figure 3. Our team plans on looking at promoter methylation as a key indicator for cancer screening.  </H5><P>Although hypermethylation seems to be a viable alternative to current cancer detection strategies, traditional methylation analyses have their own problems. Sodium bisulfite conversion is the most widely used method to distinguish unmethylated cytosines from methylated cytosines and can be coupled with various downstream detection technologies including next-generation sequencing (NGS) and PCR-based assays. Sodium bisulfite rapidly reaminates unmethylated cytosines to uracils, whereas methylated cytosines are only slowly converted. However, bisulfite treatment can induce random DNA breaks, resulting in short single-stranded DNA fragments, especially for cfDNA that is sparse and highly fragmented.<B> Bisulfite treatment also leads to a reduction in sequence complexity and cannot distinguish from other methylated bases, both resulting in compromising efficiency.</B></P><H3> Using Synthetic Biology and Machine Learning to Advance Liquid Biopsy</H3><P>Taking into account all the information regarding cancer diagnostics, our team decided to focus on developing an alternative pipeline for cancer detection that would be more effective than histopathological assessment and help resolve some of the outstanding concerns with liquid biopsy. </P><P>In order to develop a low-cost, specific, and sensitive diagnostic test for cancer detection, <B> our team established an algorithm that would could help identify cancer-specific markers </B>. Numerous papers and scientific literature suggested that searching for aberrant DNA methylation in specific genes could be correlated to specific types of cancer. To explore this in depth, our team analyzed massive datasets from The Cancer Genome Atlas (TCGA) and used unsupervised machine learning to find cancer-specific markers for hepatocellular carcinoma. Read more about our methodology for our machine learning model here. </P><P> Another core component of our vision was to use synthetic biology at the core of our diagnostic assay. Discussions with multiple experts and corroboration with literature helped us choose <B> methyl-binding domain (MBD) proteins </B>. These MBD proteins bind to symmetrically methylated sites but lacked the specificity as they search for methylation along the entire body of the gene. Using synthetic biology and advances in materials science and microfluidics, our team designed a diagnostic assay that could detect methylated DNA at expected levels. Read more about the wetlab workflow in the Design section of our wiki. </P><H5> Figure 4. The Epinoma workflow consists of a biomarker discovery tool that uses unsupervised machine learning, engineered proteins for detection, and a digital health platform at the end of the patient care journey </H5><P>Finally, our team wanted to provide some external context and relevance to the problem that we were trying to solve. In order to accomplish this, our team realized that patient-doctor communication is also a critical component of the diagnostic pathway. In order to address some of the issues that we found through our Integrated Human Practices, <B> we assembled a digital health platform that functions in early screening and can also be used in post-therapeutic response.</B></P><H3>Future Applications and Implications</H3><P>We wanted our platform to be modular: applicable across a number of different cancers and other diseases as well. Although our work up to this point primarily focuses on detection and diagnosis of hepatocellular carcinoma, we believe that using this algorithm and aberrant methylation detection strategy could be <B> applicable for other diseases </B> as well. </P><H5> Figure 5. Scientists believe that promoter methylation could be used as an early indicator of cancer, as well as HIV and Alzheimer's disease. </H5><H3>References</H3><P> Ilié, Marius, and Paul Hofman. “Pros: Can Tissue Biopsy Be Replaced by Liquid Biopsy?” Translational Lung Cancer Research 5.4 (2016): 420–423. PMC. Web. 17 Oct. 2018. </P><P> Alberto Bardelli, Klaus Pantel. Liquid Biopsies, What We Do Not Know (Yet). Cancer Cell,Volume 31, Issue 2. 2017. </P><P> “DNA Methylation.” What Is Epigenetics?. www.whatisepigenetics.com/dna-methylation/ </P><P> https://www.asco.org/about-asco/press-center/news-releases/liquid-biopsy-tests-people-cancer-expert-review </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>