Team:UC San Diego/Results
People
Our Team
Collaborations
Attributions and Sponsors
Application
Product Design
Entrepreneurship
Database
Human Practices
Human Practices Silver
Integrated Human Practices
Education and Public Engagement
Modeling
Our Project
Our Design Philosophy
Description
Design
Hepatocellular Carcinoma
Protocols and Notebook
Parts and Results
Medal Criteria
InterLab
Safety
Results
Plasmid construction
Two basic constructs and two improved constructs were assembled in this project. Each basic construct contains a single MBD binding domain coupled with eGFP and avi-tag for signal detection. Each improved construct contains a human MBD domain and a mouse MBD domain, aiming to improve DNA affinity while maintaining good expression at the same time. More information about the constructs could be referred to the design page.
A. Basic constructs: mMBD-eGFP and hMBD-eGFP  Pre-digested pET28A vector (NcoI and XhoI) was obtained from Bottini’s lab in UCSD. The vector was analyzed with agarose gel (Figure 1) with 2-log ladder (same below unless specified). The digested vector was then extracted with Qiagen Gel Extraction Kit and eluted. All DNA concentrations were quantified by Nanodrop 2000. Concentrations of the vector were summarized in Table 1. To increase the concentration of pET28A, SpeedVac was applied to the sample to reach a final concentration of 15ng/ul.
Gblocks containing mouse MBD (mMBD), human MBD (hMBD), and enhanced green fluorescent protein (eGFP) sequences with Avi-tag and thrombin site (the whole gblock is called “eGFP” for simplicity) were purchased separately. Expected sizes for fragments were indicated in Table 2. PCR reactions were carried out with gblocks so that: 5’end of mMBD and hMBD have 20nt overlap with the linearized vector and 3’end have 20nt overlap with eGFP; 3’end of eGFP has 20nt overlap with the linearized vector. (all primer sequences: supplementary table). PCR overlap products were subsequently analyzed with agarose gel and extracted (Table 3, Figure2). A PCR extension was then carried out to combine mMBD, hMBD with eGFP, respectively. (Figure 3)
Gibson assembly was subsequently carried out with linearized pET28A and hMBD-eGFP; Restriction cloning was carried out with linearized pET28A and mMBD-eGFP using NcoI and XhoI. Clones were sub-cultured overnight and miniprepped with Qiagen kit. Plasmids were sequenced to confirm correct assembly (supplementary figure).
Figure 1. gel image of pre-digested pET28A.
Vector
pET28 (S1)
pET28 (S2)
Expected size
About 5.2kb
Concentration (ng/ul)
2.2
2.0
Table 1. Concentrations of vector samples extracted from agarose gel.
Figure 2. Gel image of overlap PCR products. Sizes of gblocks are: mMBD, 216bp; hMBD, 228bp; eGFP, 792bp.
construct
mMBD
hMBD
eGFP
mMBD-eGFP
hMBD-eGFP
Size (bp)
216
228
792
About 1100
About 1100
Table 2. summary of sizes of gblocks.
PCR extension substrate
hMBD-eGFP overlap
mMBD-eGFP overlap
eGFP-MBD overlap
eGFP-mMBD overlap
Concentration (ng/ul)
21.7
24.4
17.1
19.8
A260/A280
1.79
1.9
1.84
1.82
Table 3. summary of extracted overlap PCR products.
Figure 3. gel image of PCR extension product. Two replicates were combined into one extraction process and the concentrations of extracted product are: hMBD-eGFP, 10ng/ul; mMBD-eGFP, 33.7ng/ul.
A. Improved construct: hm-avi and hm-HRP
gblock fragments of hm-avi and hm-HRP (see design page for construct details) were purchased from IDT DNA. Primers were designed to amplify the constructs so that they contain NcoI and XhoI overhang. Amplified constructs and pET28A vectors were subsequently digested (Figure 4), ligated, and cloned. Sequencing results were provided in the supplementary section.
Figure 4. gel image of digested pET28A and hm-HRP.
Recombinant protein expression and quantification
All constructs were induced with IPTG in E.coli BL-21 and initially purified with Ni-NTA column. Variants except mMBD-eGFP required further purification for higher purity. In this project, hm-avi and hm-HRP proteins were purified by SEC column. The purified proteins were then quantified by one of the two methods: two basic constructs were quantified by OD 280 via Nanodrop 2000; two improved constructs were quantified by Fast Protein Liquid chromatography (FPLC).
The constructed plasmids with their sequences confirmed was transformed into E.coli strain BL-21 (DE3) for protein expression. A starter culture containing 6mL autoclaved LB broth and 50ug/mL Kanamycin was grown for each construct. The incubation at 37 degree generally took around 3 hours for the culture to become cloudy. Then, 1.5 mL starter culture was used to grow a 750mL large culture (1:500 dilution). The incubation at 37 degree was stopped when OD600 was around 0.8. The culture was then cooled on ice and 100uL of it was taken as the reference before IPTG induction. The protein expression was induced with 0.5mM IPTG at 18 degree for 16 hours with shaking. 100uL of culture was taken after the induction to assess the expression of the protein. The E.coli cells were harvested by centrifuging the culture at 8000rpm, 4 degree for 10min each time. Each pellet was resuspended in 12mL resuspension buffer (300mM NaCl + 20mM Tris (pH =8)). The E.coli cells were then lysed by adding ~5mg lysozyme and fast freezing in liquid nitrogen for 15min.
5uL DNAseI, 15uL 1M MgCl2 and 3uL 1M CaCl2 were added into thawed E.coli cell lysate to digest the DNA. The lysate was left in room-temperature water bath for 1 hour until it became less viscous. 5uL lysate was taken as the reference before centrifuging for separation. The soluble and insoluble fractions of the lysate was separated by centrifuging at 14500rpm, 4 degree for 1h, and 5uL supernatant was taken to assess the solubility of the protein.
Purification of the recombinant protein generally took use of the His-tag at the end. 1mL Ni-NTA agarose equilibrated with resuspension buffer (300mM NaCl + 20mM Tris (pH = 8)) was used for each construct. The supernatant after centrifugation was loaded, the column was then washed with 3 column volumes of resuspension buffer. For hMBD-eGFP and mMBD-eGFP, 5 4mL elutions containing imidazole gradient was performed (50mM, 100mM, 150mM, 200mM and  250mM). For hm-Avi and hm-HRP, 3 6mL elutions containing wider imidazole gradient was performed (10mM, 100mM, 250mM).
TGX stain-free gel was run to check the expression, solubility, and Ni column purification of each construct. Unstained protein ladder from Bio-Rad was used for all the stain-free gels. For hMBD-eGFP, the expression of the protein is not significant according to the gel (Figure 5). Neither could the corresponding band be seen in the lysate, supernatant and Ni column elutions. However, the fluorescence of eGFP was clearly observed in at least two of the Ni column elutions. Thus, the gel was transferred onto a nitrocellulose membrane and blotted with anti-His antibody labeled with HRP. The band of expected size was observed in Ni elution 2 and 3 after HRP substrate was applied. However, further purification was required to remove the impurity (Figure 6). For mMBD-eGFP, the expression of the protein is not significant according to the gel and its solubility is also hard to determine, but the band of expected size was observed in Ni elution 2 and 3 (Figure 7). From the gel, Ni elution 3 is purer, so it is used for EMSA later. To get rid of the possible effect of imidazole on a later experiment, the mMBD-eGFP Ni elution 3 was dialyzed in 3000 MWCO with 500mL 150mM NaCl + 20mM Tris (7.3) overnight. The concentration of the sample after dialysis was determined by Nanodrop to be 7.88uM. For hm-Avi, the difference between before and after induction clearly shows the expression of the protein, but it seems not all the protein in the supernatant bound to the Ni column, and Ni elution 2 contains the protein bound to the column (Figure 8). Concerned about the purity of the protein from the gel, the Ni elution 2 was concentrated using 3000 MWCO ultracentrifuge from Satorius to around 300uL and loaded onto Size Exclusion Column 650 from GE. The buffer used for the column was 150mM NaCl + 20mM Tris (pH = 7.3). 4 peaks were observed on the chromatography (Figure 9), and the fractions were loaded onto a TGX stain-free gel with the load to see their content. From the gel, fraction 6 of gel filtration is shown to contain the band of hm-Avi (Figure 10), and Western Blot with anti-His labeled with HRP confirms that (Figure 11). Th concentration of the fraction was determined directly from the chromatography to be 3.21uM, and Nanodrop gave pretty similar results. The fraction was directly used for later experiments. For hm-HRP, the expression of the protein is not significant according to the gel, and it’s also hard to determine its solubility and presence in Ni column elutions (Figure 12). But from Western Blot of anti-His labelled with HRP, it can be observed that some protein even though not a lot is in Ni elution 2 (Figure 13). Thus, the Ni elution 2 was concentrated using 5000 MWCO ultracentrifuge from Satorius to around 300uL and loaded onto size exclusion column 650. The same buffer was used. 3 peaks were observed on the chromatography (Figure 14), and the fractions were loaded onto an SDS-PAGE gel with a load to see their content. The gel was then blotted with anti-His labeled with HRP. The Blot shows that fraction 3 contains hm-HRP (Figure 15). The chromatography calculates the concentration of fraction 3 to be only 1.57uM, so it was concentrated using 10000 MWCO ultracentrifuge from Satorius to around 300uL. The concentration was once again determined by Nanodrop to be 3.29uM. The fraction after concentration was used for later experiments.
Figure 5. TGX stain-free gel of hMBD-eGFP with Unstained Protein Ladder from Bio-Rad. The expected size of hMBD-eGFP is 39 kDa.
Figure 6. Anti-His labelled with HRP of hMBD-eGFP. The expected size of hMBD-eGFP is 39 kDa.
Figure 7. TGX stain-free gel of mMBD-eGFP. The expected size of mMBD-eGFP is 39 kDa.
Figure 8. TGX stain-free gel of hmAvi. The expected size of hmAvi is 21 kDa.
Figure 9. Size Exclusion Chromatography of hmAvi Ni elution 2. Blue line is the absorbance at 280nm. Fraction 3, 5, 6 and 7 were later loaded onto a TGX stain-free gel with the load.
Figure 10. TGX stain-free gel of hmAvi gel filtration load and fraction 3, 5, 6, 7. The expected size of hmAvi is 21 kDa.
Figure 11. Anti-His labelled with HRP of hmAvi gel filtration load and fraction 3, 5, 6, 7. The expected size of hmAvi is 21 kDa.
Figure 12. TGX stain-free gel of hmHRP. The expected size of hmHRP is 54 kDa.
Figure 13. Anti-His labelled with HRP of hmHRP Ni elution samples. The expected size of hmHRP is 54 kDa.
Figure 14. Size Exclusion Chromatography of hmHRP Ni elution 2. The blue line is the absorbance at 280nm. Fraction 3, 4, and 5 were later loaded onto an SDS-PAGE gel with the load.
Figure 15. Anti-His labelled with HRP of hmHRP gel filtration load and fraction 3, 4, 5. The expected size of hmHRP is 54 kDa.
MBD variants validation
MBD constructs were characterized by electrophoretic mobility shift assay (EMSA). In this method, protein and DNA will be pre-incubated and run on a native polyacrylamide gel. If protein binds to DNA, DNA will be slowed while running; at the same time, a “retardation band” will form when the gel is visualized with DNA-specific dye. DNA not incubated with the proteins was run along the same gel as standards
Three constructs including mMBD-eGFP, hm-avi, and hm-HRP were characterized in this project. 24 bp primers containing two methylation sites in close proximity were chosen as the target to test protein-DNA binding, in order to characterize the specificity of different protein constructs. To be more clinically relevant, one of the methylation sites on the primer was cg10428836, which was shown to have significant methylation level change in hepatocellular carcinoma (HCC). The other methylation site, by contrast, has little known clinal relevance. Forward and reverse primers were pre-hybridized to dsDNA before the assay; in the figures, primers were specially labeled as like “F 0_0”, which indicates that it was a forward primer with no methylation on both sites “0”. Another example will be “R M_0”, which stands for the reverse primer with a site methylated near 5’ end “M” (this methylation site is cg10428836).
Pre-hybridization was performed for: no methylation (both 0_0), hemi-methylation (0_0 + M_0), symmetrical methylation (M_0, M_0), and unsymmetrically methylation (0_M + M_0). A good protein construct will only bind to symmetrically methylated DNA but not hemi-methylated nor unsymmetrically methylated ones; latter will be the noises for our device.
Among the tested construct, mMBD-eGFP showed hints of protein-DNA interaction while it was not decisively conclusive for a confirmed binding (supplementary figure). hm-Avi (Figure 16) was shown to bind to not symmetrically methylated DNA. One possible reason was that the concentration of hm-Avi was too high. Lowering the protein concentration in the future could potentially increase the specificity for this protein
By contrast, Hm-HRP protein demonstrated a good specificity toward symmetrically methylated DNA (Figure 17). To confirm this result, Hm-HRP on the same gel was transferred to nitrocellulose membrane and subsequently probed with anti-His labeled HRP. The membrane was then visualized with Western Femto substrate (Figure 18) to show that Hm-HRP was present in the retardation band
In conclusion, mMBD-eGFP and hm-avi protein require further validation to confirm their binding to methylated DNA. Hm-HRP is fully validated and suitable as the methylated DNA detector for our device.
Figure 16. retardation bands were present in both negative controls and experimental condition (lane 3).
Figure 17. retardation bands were present only symmetrically methylated DNA probes (lane 3), suggesting its specificity.
Figure 18. Hm-HRP was transferred and visualized with anti-His antibody. Hm-HRP/DNA complex was confirmed by the presence of protein signal at lane 3.
Characterization of Graphene oxide (GO) platform
Two steps of graphene oxide method were characterized in this project: 1. GO quenched fluorescent dye labeled DNA by hydrophobic interactions and fluorescent resonance energy transfer (FRET). 2. Fluorescence was recovered when complementary target DNA was introduced, forming duplex with fluorescent probe. Exonuclease III amplification strategy was also characterized in this section (See design pages for more details).
To provide proof-of-concept evidence of GO platform, we set up multiple small-scale reactions in 96 well plates and monitored fluorescence with Tecan Plate Reader. NEB II buffer (pH=7.9) was the solvent environment for all experiments and measurements. This was because FAM (fluorescent dye) used in the DNA probe would lose activity when pH was under 7; Exonuclease III had optimal activity at pH =7, but retained 75% activity in NEB II.
Serial dilutions of fluorescent DNA probe in NEB II (Sequences provided in supplementary) were first performed to achieve a concentration of 100nM. 0.02mg/mL GO solution was diluted from 2mg/mL stock (Sigma-Aldrich). A master mix was prepared with equal amount of DNA probe and GO solution, mixed, and subsequently transferred to 96 well plates. Fluorescent quenching was monitored and presented in Figure 19. Quenching efficiency was calculated and presented in Figure 20, which reached above 97% in 20 minutes.
In a separate trial, procedures mentioned above was repeated and the quenching was allowed to proceed for 20 minutes. 50pM DNA probe (R 0_0, see section 3) diluted in NEB II buffer was then added, followed by addition of 20U exonuclease III. Figure 21 demonstrated that addition of both target DNA and exonuclease III recovered more fluorescence compared to negative controls.
Role of exonuclease III in signal amplification was also briefly investigated by comparing samples with and without the enzyme. In low concentration of DNA (5pM), samples without adding exonuclease failed to recover fluorescent signal (Figure 22); samples with exonuclease III exhibited a similar kinetic curve compared to Figure 21, demonstrating its importance for our device to detect low concentrations of ctDNA.
Graphene oxide is able to quench fluorescent DNA probe in 20min
Figure 19. n=3. 50ul 100nM fluorescent DNA probe was mixed with 50ul 0.02mg/ml GO.
Figure 20.  Quenching efficiency was calculated by dividing each fluorescent signal with control fluorescence (108891). Quenching did not start at 0% probably because the kinetics were too fast to capture at the beginning.
Addition of target DNA can recover fluorescence in 60min
Figure 21.  similar as below, full kinetics were probably too fast to capture. Although background existed when only exonuclease was present, its signal was much lower than the one with target DNA
Addition of exonuclease enhance fluorescent recovery
Supplementary information
A. Supplementary figures and tables mentioned in lab result pages
Figure 1. pET28A vector digested with NcoI and XhoI.
Figure 2. Sequences of basic constructs: mMBD-eGFP and hMBD-eGFP; please refer to parts BBa_K2881003 and BBa_K2881004 to access sequences.
Figure 3. Sequencing results of basic constructs in pET28a: mMBD-eGFP and hMBD-eGFP
Figure 4. sequences of improved constructs: hm-avi and hm-HRP
Figure 5. Sequencing results of improved constructs in pET28a: hm-avi and hm-HRP
Figure 6. EMSA result for mMBD-eGFP. Unlike hm-avi and hm-HRP, a different ladder was used with the smallest size of 100bp. Although no retardation band was observed, DNA-MBD binding was likely happen because: 1. DNA band became thinner in symmetrically methylated substrate. 2. More “streaks” were present in the same lane. A more well-crafted gel may be necessary to visualize mMBD-eGFP/DNA binding events; internal DNA control could also be used in the future to confirm that the DNA band becomes thinner.
Table 1. sequences of primers used in this project
BirA biotin ligase was cloned and expressed for building a versatile MBD detection platform
One important consideration we had in our BioBrick design was to make constructs as versatile as possible for different contexts. For example, detecting DNA methylation in different settings may prefer different detection methods such as colorimetric and fluorescent assays. Biotin has been applied to fulfill this need as it could be recognized by streptavidin in extremely high affinity. Many established methods have been developed to attach biotin to proteins. Here we chose to introduce avi-Tag in our construct, which could be specifically biotinylated by BirA biotin ligase; the enzymatic reaction happens in mild condition, enabling maximum preservation of MBD binding activity compared to traditional chemical treatment.
In addition to adding avi-Tag in most of our constructs, the major step we committed to building a versatile biotin detection platform for our construct is to efficiently express BirA in E.Coli BL-21. In brief, BirA gblock fragment was purchased from IDT DNA and cloned into pCDF-dueT via restriction cloning with BamH1 and SaI1. In Figure 8 we demonstrated that BirA could be expressed in high yield and purified with Ni-NTA column. Key parameters of protein expression/purification were shown in Table 2. Through this  data, we aim to provide support for future IGEM projects with avi-Tag built-in constructs.
Figure 7. vector map for pCDF-Duet.
Figure 8. gel image for BirA purification
Table 2. Key parameters for BirA expression/Nickel column purification.
Parts
Check out our parts here.
