<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Rheda_Bielefeld_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Rheda Bielefeld/Methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><A href="https://2018.igem.org/Team:Rheda_Bielefeld">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Team">The Team</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/School">Our School</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/University">Bielefeld University</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Rheda">Rheda</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Description">Description</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Notebook">Notebook</A></DIV></DIV><DIV><A href="https://2018.igem.org/Team:Rheda_Bielefeld/InterLab">InterLab Study</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Experiments</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/OverviewExperiment">Overview</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Methods">Methods</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Pollen">Pollen</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/PCR">PCR</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Assays">Assays</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Cloning_Strategy">Cloning Strategy</A></DIV></DIV><DIV><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Parts">Parts</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Practice</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Marta_Event">Marta Event</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Safety</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Safety">Safety Form</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Biosafety">Biosafety</A></DIV></DIV><DIV><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Collaborations">Collaborations</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Attributions</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Attributions">Attributions</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Partners">Partners</A></DIV></DIV><DIV><A href="https://igem.org/2018_Judging_Form?team=Rheda_Bielefeld">Judging Form ⇗</A></DIV></DIV><DIV class="row"><DIV class="column left"><H2> Short Overview </H2><ARTICLE><B>DNA Extraction from plant leaves: </B>

DNA extraction kit Macherey &amp; Nagel. Extraction was performed according to protocol. The leaf pieces were either frozen with liquid nitrogen and afterwards added to the mortar and pestle or water was added. <B>Open the Pollen:</B>

Ribolyser: 
6500 rpms* two times performed *30 seconds duration of the rotation *30 seconds pause<B>Media (cultivating bacteria):</B>
LB = 20g LB powder filled up to 1L
Competent cells KRX from promega were used for transformation. Heat shock transformation were tested with the promega provided protocol and with the protocol from the iGEM Bielefeld team 2014:<I>Transformation via heat shock</I>
Thaw 200 μL Chemo competent E.coli cells on ice
Add 0.5-5μL plasmid to 200 μL chemocompetent cells
Store cells on ice for 10-30 minutes
Heatshock for (60-)90 seconds at 42°C
Transfer transformation reaction to 1 mL SOC or LB Medium and incubate 1 hour at 37°C
Centrifuge 30 seconds at 11000 rpm and plate on selective LB Medium
Incubate overnight at 37°C

Plasmid isolation was performed with the analytic Jena plasmid isolation kit. The enclosed protocol was used. It was changed a bit as the elution step was done with water instead of elution buffer.<B>PCR:</B>
Plant PCR were performed with Solis Biodyne Master Mix with the following protocol:<UL><LI> 1µl Forward Primer </LI><LI> 1µl Reverse Primer </LI><LI> 4µl Master Mix (nucleotides, Taq polymerase) (Solis Biodyne)</LI><LI> 12µl H2O </LI><LI> 2µl Template DNA </LI></UL>
Phusion PCR for cloning was performed with fermentas Phusion enzyme and the following protocol:<UL><LI>5x Phusion HF buffer (4µl) </LI><LI> 10mM dNTPs (0,4µl) </LI><LI> Forward primer (1µl) </LI><LI> Reverse Primer (1µl) </LI><LI> Template DNA (1µl) </LI><LI> Phusion DNA polymerase (0,2µl) </LI><LI> H2O (add to 20µl) </LI></UL><B> DNA Extraction from Xanthomonas campestris B100: </B><UL><LI> Pick bacterial spots from the agar plate </LI><LI> Dissolve cell with 0,9% NaCL solution (1mL) </LI><LI> Centrifuge 5 min and 8000rpm </LI><LI> Re-dissolve pellet in 1 mL steile Milli Q </LI><LI> Heat 5 minutes at 98°C  </LI><LI> Centrifuge 5 minutes at  13500 rpm </LI><LI> Collect supernatant in a new tube</LI><LI> Store at -20°C, Measure DNA by Nanodrop</LI></UL>

DNA extraction from Bacillus subtilis was performed by the protocol of the LMU Munich Team 2012:
Isolation of chromosomal DNA from Bacillus subtilis for transformation (Cutting and Van der Horn, 1990) <UL><LI>Mix 2.5 ml of an overnight culture with 2.5 ml SC buffer   </LI><LI>Harvest cells by centrifugation (5 min, 8000 rpm, RT)   </LI><LI>Resuspend cell pellet in 1 ml SC buffer   </LI><LI>Add 100 μl lysozyme and mix well   </LI><LI>Incubate for 15 min at 37°C   </LI><LI>Add 1 ml NaCl and mix well   </LI><LI>Filtrate the DNA solution with a 0.45 μm filter   </LI><LI>Use 100 μl for B. subtilis transformation   </LI></UL>

SC buffer (pH 7.0) NaCl  
Sodium citrate  
Lysozyme 2 mg/ml in SC buffer  
NaCl 4 M <B>Gel electrophoresis:</B>
TAE, 50x stock solution (provided by the supervisors); working solution 1x TAE 
0,8%-2% agarose dissolved in 1x TAE
50x TAE:
242 g Tris Base
57.1 mL Glacial Acetic Acid
100 mL 0.5 M EDTA
pH:7.8

Nanodrop ND2000 Spectrophotometer from PeqLab was used for DNA measurements. 
After the separation of the plasmids and another insert was not possible only by using specific primers, we used restriction enzymes to cut out the unnecessary insert. Herefore we used the restriction enzymes EcoR I, Pst I, Xba I and Not I. These were used by adding 4 nl DNA to 2µl Buffer, 1µl restriction enzymes and 12µl of water. After incubating this mix at 37°C we inactivated everything at 80°C for 20 minutes. Hereby we hoped to separate the psb1C3 (2070 bp) from the mCherry-insert (711 bp). You can find the exact procedure <A href="https://2018.igem.org/Team:Rheda_Bielefeld/Notebook" style="color:yellow"> here on the 21.08.2018 </A>
. 

Primers used for bet detection: 
Bet1_1 
ATGGGTGTTTTCAATTACGAAACTGAG 
Bet1_2 
TTAGTTGTAGGCATCGGAGTGTGCCAAGAGG 

Pectin C, Pectin A and Amylopectin were purchased by Carl Roth. 
Pectinase from Aspergillus Niger, which was used in the pectin assay as a positive control, was purchased by Sigma 

The experiments with liquid nitrogen and trypsin as well as the primer design strategy were especially supported by the supervisors. No antibiotics were handled by team members.
</ARTICLE></DIV><DIV class="column right"><H2> Methods </H2><ARTICLE>
We have used various different methods for our experiments. Here is an overview of the methods we have used. <B><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Pollen" style="color:yellow;font-size:25px;"> Pollen </A></B>

For our experiments with pollen, we have used two methods. The first method was using trypsin. Trypsin is an enzyme mostly found in the digestive system of many animals and can hydrolyze proteins. We have used it here in the hope that the trypsin would hydrolyze the proteins in the pollen´s outer wall and therefore making the pollen break. The Ribolyser consists of many little balls of ceramics and is used to homogenize biological samples. It works simple: you put a sample into the ribolyser and shake it well, in the best case with a mechanical shaker. By shaking it, the little ceramic balls fly around in the container and by crashing into the sample, a lot of damage can be done to your biological sample. A negative side effect of using the ribolyser is that a lot of heat is produced when the balls fly around. We used the ribolyser to break the pollens wall with both heat and the mechanical damage done. Another way we tried to break open the wall of a pollen was by using liquid nitrogen. When you put liquid nitrogen onto cells, they break because all liquids touching that extremely cold substance freeze and eventually break. We enhanced this process by using a mortar.<B><A href="https://2018.igem.org/Team:Rheda_Bielefeld/PCR" style="color:yellow;font-size:25px;"> PCR </A></B>

When we extracted the DNA from the leaves, we used a mortar, water, and some ethanol to gap their cellular wall. After crushing them we used filters from the &quot;Nucleospin DNA Extraction Kit&quot; to extract the DNA from the remains. When using this kit, you need two filters, one container for the flow-through, liquids included in the kit and a centrifuge. The kit basically binds the DNA to the filter, washes out everything else through the filter and in the end, you use an elution buffer to elute the DNA from the filter. For the PCR, you have to prepare your sample by adding the nucleotides, the primer, the polymerase, and some water. Afterward, you put the samples into the PCR machine. It goes through many cycles of changing temperatures. In that process, the DNA parts into two strings and the primers bind onto their complementary sequence and &quot;shows&quot; the polymerase where it has to start synthesizing on. Next, the polymerase goes along the string and copies the sequences with the opposite nucleotides. By that process, the DNA was multiplied. When the PCR was finished, we performed a gel electrophoresis. The gel electrophoresis consists of an agarose gel, TAE-buffer, an anode and a cathode, and a connection to electricity. Since the DNA has a negative charge, it moves towards the anode. The gel electrophoresis can be evaluated by putting it into a bath of a coloring chemical. The gel is afterward put into a dark chamber and gets lit by UV-Light. Hereby, the gel electrophoresis pictures resolve. <B><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Assays" style="color:yellow;font-size:25px;"> Assays </A></B>

The cracking of pollen is related with the destruction of pectin and cellulose. Therefore we need enzymes like pectinase and cellulase.
At first, it is important to prove the content of cellulose and pectin in specific substances. Therefore we used both natural and pure alcohol. If there were small bubbles in the substances to see, then we successfully proved pectin and cellulose in the samples. This is important for proving the appearance of pollen. After a while, pectinase and cellulase should have destructed the pectin and cellulose. The experiment is successfully performed if there aren't any bubbles in the samples. <B><A href="https://www.ncbi.nlm.nih.gov/" style="color:yellow;font-size:25px;"> NCBI </A></B>
We visited the website of the National Center for Biotechnology Information (NCBI) to get some information from their data base. With them we got to know the sequence of the amino acids in the DNA of the tree leaves we used. Afterwards we were able to check whether our results were correct.
</ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>