Team:Rheda Bielefeld/Parts
Home
Team
The Team Our School Bielefeld University Rheda
Project
Description Notebook
InterLab Study
Experiments
Overview Methods Pollen PCR Assays Cloning Strategy
Parts
Practice
Human Practices Marta Event
Safety
Safety Form Biosafety
Collaborations
Attributions
Attributions Partners
Judging Form ⇗
Plasmid Designs
Click the image to enlarge it
Fig. 1: pelB-gene from the B.sub. in psb1C3-vector
Click to enlarge
Fig. 2: yesZ-gene from the B.sub. in psb1C3-vector
Click to enlarge
Fig. 3: pelB-gene from the Xan. in psb1C3-vector
Parts
The main idea of our project is to detect pollen by a DNA-based method, but another plan has been also to differ between the different types of pollen. Concerning this, we mainly focused on the pollen most commonly causing allergic reactions in our area. To analyze and to extract the DNA, we had to break open the outer layer of the pollen. Some attempts to mechanically or chemicaly open pollen are described here: Pollen
Pollen have an outer (exine) and inner (intine) layer, parts of which consist of pectine and cellulose (Assays) . Therefore,we wanted to use Escherichia coli to produce enzymes which are able to dismantle these substances and hereby break open the pollen.
For this purpose we chose the following enzymes:
Out of the Bacillus subtilis the genes pelB and yesZ and out of Xanthomonas campestris the gene pelB. The genes pelB out of B.sub. and Xan. Code for pectinases or pectin lyases, yesZ being a beta-glucosidase, which could have enhanced the further dismanteling of the shell.
The BioBricks are supposed to be assembled performing a Gibson Assembly. For this we isolated the DNA of Bacillus subtilis and Xanthomonas campestris B100 and amplified the genes with specific primers. We wanted to clone the genes into psb1C3 and send them in. Using the E.coli vector pz9 availible to us in our lab we planned to characterize the genes. All in all we were trying to perform 6 clonings (3 clonings into psb1C3 and 3 into pz9). The psb1C3 constructs can be seen on the left. The experimentdesign was created with snapgene.
We also thought about using other, already in iGEM existing, enzymes:
Team
Part
Comments
WLC Milwaukee
2014
BBa_K1175007
yesZ frim Bacillus Subtilis
characterized in vitro assy
Statuts: It`s complicated
Edinburgh 2008
BBa_K118023
Codon optimized Caulobacter crescentus
Caulobacter crescentus codon usage
British Columbia 2016
BBa_K2139003
Codon optimized Caulobacter crescentus
sample not in stock
Endo-beta-1,4-glucanase E1
Stanford-Brown-Spelman 2014
BBa_K1499501
Endo-1,4-&-beta-glucanase (EG1) / Neisseria sicca
Not characterized, but Neisseria sicca is an S2 organism
The experiments with liquid nitrogen and trypsin as well as the primer design strategy were especially supported by the supervisors. No antibiotics were handled by team members.
What went wrong?
Why didn’t the colonings work? Possible reasons for our failure are that we tried to perform a PCR using the wrong primers or that we committed an error somewhere else in the procedure. Another possibility is that we had the wrong stem of Bacillus subtilis from the beginning, because only the stem 168 has the required genes.
We also tried to order the parts for our clonings from IDT, but they were not able to send them to us in time for us to still work with them.
