<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Rheda_Bielefeld_PCR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Rheda Bielefeld/PCR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar"><A href="https://2018.igem.org/Team:Rheda_Bielefeld">Home</A><DIV class="dropdown"><BUTTON class="dropbtn">Team</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Team">The Team</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/School">Our School</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/University">Bielefeld University</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Rheda">Rheda</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Project</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Description">Description</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Notebook">Notebook</A></DIV></DIV><DIV><A href="https://2018.igem.org/Team:Rheda_Bielefeld/InterLab">InterLab Study</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Experiments</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/OverviewExperiment">Overview</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Methods">Methods</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Pollen">Pollen</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/PCR">PCR</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Assays">Assays</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Cloning_Strategy">Cloning Strategy</A></DIV></DIV><DIV><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Parts">Parts</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Practice</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Human_Practices">Human Practices</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Marta_Event">Marta Event</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Safety</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Safety">Safety Form</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Biosafety">Biosafety</A></DIV></DIV><DIV><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Collaborations">Collaborations</A></DIV><DIV class="dropdown"><BUTTON class="dropbtn">Attributions</BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Attributions">Attributions</A><A href="https://2018.igem.org/Team:Rheda_Bielefeld/Partners">Partners</A></DIV></DIV><DIV><A href="https://igem.org/2018_Judging_Form?team=Rheda_Bielefeld">Judging Form ⇗</A></DIV></DIV><DIV class="row"><DIV class="column left"><H2> Results Gel Electrophoresis </H2><ARTICLE> These are the results from our gel electrophoresis we, have performed. The labels on the pictures translate to:<B>+</B> positive control <B>-</B> negative control  <B>B</B> birch DNA <B>O</B> oak DNA <B>H</B> hazelnut DNA<B>A</B> American amber DNA<B>M</B> maple DNA<B>Marker</B> Length standard</ARTICLE><ARTICLE> This is our first attempt of the PCR. By the fact that there are no bands, you can assume that the PCR didn´t work. <ARTICLE> Here you can see that the positive control worked, so did the PCR. For the birch and the oak DNA, you can see one band at the same height. This tells us that both the birch and oak samples contain the allergen Bet1.  </ARTICLE><ARTICLE> In this picture you can see that the positive control worked well. For the birch sample you can see two bands aligned above each other. For hazelnut and maple samples, there are two streaks. </ARTICLE></ARTICLE></DIV><DIV class="column middle"><H2> Plant PCR </H2><ARTICLE>
In Germany, about 14,8% of German people are reacting allergically to <A href="https://2018.igem.org/Team:Rheda_Bielefeld/Pollen" style="color:yellow"> pollen </A> [3] - and 80% of those react to birch pollen[1]. They react with aching eyes, they start sneezing and their nose is running. The birch pollen are very critical for them because their pollen are the most aggressive ones of them all and they grow everywhere [1] [2]. For example, the pollen of the white birch (Betula verrucosa) are the main cause for type I allergic reactions in middle and northern Europe, North America and the former Soviet Union [4]. The allergic people are suffering a lot from their allergy because one of every two years is a mast year what means that there are extremely many pollen [2].


In the process of differing pollen, we have to consider that there are many different pollen in the air. In order to differentiate them by their DNA, we need to know from which plant they came from and how they originate and distinguish from others. This is fundamental for building our biosensor because not everybody is reacting allergically to the same pollen or allergen. In order to trial the DNA sequence of the plant, we have collected and used the leaves of birches, oaks, hazelnut trees, American Amber trees and maple trees. We have extracted the DNA from these leaves and multiplied it via Polymerase Chain Reaction, short <A href="https://2018.igem.org/Team:Rheda_Bielefeld/PCR" style="color:yellow"> PCR </A>.


At first, we extracted the DNA from the leaves by crushing them with a mortar. With that technique, we were able to crush the plant cells. When crushing the plant cells you are breaking the outer sphere of a cell, causing the inner compartments such as the cytoplasm, the vacuole, mitochondria and the nucleus, containing the DNA. To get the DNA from the cell remains we used fine filters filtering out the DNA. Those filters are contained in the Nucleospin DNA extraction kit. After extracting the DNA from the leaves,  we were able to perform PCRs with the DNA. We used primers which are specified for the birch allergen Bet1. We used it on every DNA sample we put into the PCR. We specified the PCR program by testing different temperature profiles. The temperature of the binding process of the primers was set higher or lower. When the PCR was finished, we performed a Gel electrophoresis onto the multiplied DNA. The Gel electrophoresis was always performed under the same conditions. We used a 2% Agarose gel in 1% TAE Buffer and used a voltage of 120 Volt. After the Gel electrophoresis, we were able to differ most of the plants and decided afterwards whether we would perform another PCR under different conditions or not.


We were able to differ the oak and amber DNA from the birch, hazelnut, and maple DNA at first attempt, but we weren´t able to discriminate between the birch, hazelnut, and maple DNA because their bands resolving from the gel electrophoresis were laying next to each other on the same height. We thought of how we could improve the gel picture from the gel electrophoresis and came up with the idea that we could change the temperature of the PCR program. The original temperature for the PCR is 53°C. A PCR at 60°C was performed by Viviane, while another PCR at 54°C was performed by Leon. Afterwards, we tried it again at 56°C and 65°C. Still, the results remained the same.


To get better results, we could use different primers than the one specified for the allergen Bet1 for the PCR or perform a sequencing on the DNA samples so we could design our own primer for the PCR. The problem with the first idea is that there is a very little amount of prepared primers for tree allergens so we would have to design our own primers. A major issue with sequencing our DNA samples is that a sequencing takes too long and we are unable to perform a sequencing at the University in Bielefeld. Concluding from that problem, we can only continue with our experiment if we had more time. We can start a new experiment using Pollen DNA instead of leaf DNA so we can differentiate pollen from one another.
</ARTICLE></DIV><DIV class="column right"><H2> Sources </H2><ARTICLE><A href="https://www.pollenallergien.de" style="color:yellow;font-size:0.7em;">[1]www.pollenallergien.de</A><A href="https://www.sueddeutsche.de/gesundheit/bissige-birken-darum-ist-die-pollenplage-dieses-jahr-besonders-schlimm-1.3961283" style="color:yellow;font-size:0.7em;">[2]https://www.sueddeutsche.de/gesundheit/bissige-birken-darum-ist-die-pollenplage-dieses-jahr-besonders-schlimm-1.3961283 </A><A href="https://edoc.rki.de/handle/176904/1479" style="color:yellow;font-size:0.7em;">[3]https://edoc.rki.de/handle/176904/1479 (Häufigkeit allergischer Erkrankungen in Deutschland by &quot;Robert-Koch-Institut&quot;)</A><A href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/j.1460-2075.1989.tb03597.x" style="color:yellow;font-size:0.7em;">[4]https://onlinelibrary.wiley.com/doi/epdf/10.1002/j.1460-2075.1989.tb03597.x (The gene coding for the major birch pollen allergen Betv1, is highly homologous to a pea disease resistance response gene by &quot;The EMBO Journal&quot;)</A></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>