Team:XJTU-China/Circuits
XJTU-China 2018
Home
Project
Overview
Circuits
Demonstrate
Parts
Basic
Composite
Collection
Improve
Safety
Wetlab
InterLab
Protocols
Notebook
Drylab
Overview
Models
Hardware
HP
Silver
Integrated and Gold
Public Engagement
Entrepreneurship
Team
Acknowledgement
Attribution
Collaboration
Background
Sensing Circuit
Selecting Circuit
Coupling Circuit
Evolving Circuit
Circuits
Background
Artificial design of enzymes is an important theme in synthetic biology, usually in order to fundamentally increase enzyme efficiency or to achieve de novo enzyme activities. However, due to our limited knowledge of how the coding sequence determines the 3D structure, and in turn decides the function of a protein, rational design of these molecular machines is basically not possible in current stages.
Enzymes are designed by the nature, so it would be feasible to utilize the power of natural selection: evolution to design our desired enzymes. Directed Evolution (DE) thus emerges. It is now a part of the industrial standard for applying an enzyme found in nature to mass production or other specific purposes, and the method itself has won the Nobel Prize of chemistry in 2018, which reveals its significant impact to the society. However, directed evolution always suffers from low throughput in screenin. It is usually troublesome for desired property of enzyme, such its productivity of small molecules, to be converted into the growth pressure of chassis cells. Thus, the screening step in DE depends heavily depends on manual assays such as fluorescence reporters or HPLC. The throughput of these manual screening techniques is limited to around thousands, which is far smaller than the theoretical library space of an enzyme.
In the light of current techniques, the XJTU iGEM team has proposed a new framework for directed evolution. We have applied this framework to promote the productivity of D-psicose, a valuable rare sugar. In this case, the productivity of final product, D-psicose, undergoes a series of conversion and finally yields a distinguishable growth pressure to individual chassis cell. Four genetic devices have been constructed and unit tests been performed on each to ensure that the final device will work. The four devices are Sensing Circuit, Selecting Circuit, Coupling Circuit and Evolving Circuit.
This page emphasizes the mechanisms of the whole system. Go to Demostration page to know how we characterize our devices in detail!
Go to Demonstration
Sensing Circuit
Sensing Circuit converts the productivity (cellular concentration) of D-psicose to the expression level of downstream genes. This circuit is based on the repressor and inducible promoter contributed by Evry_Paris-Saclay team, which are annotated as PsiR and pPsi in the circuit schematics. Various regulatory elements are designed in order to achieve robust biosensing. This circuit (composite part) is meant to be a universal one, which the repressor, inducible promoter and the downstream part are flanked by different enzyme recognition sites The repressor and the inducible promoter adopts two different Golden Gate recognition sites to achieve seamless ligation required in a regulatory system, and the latter one uses standard BioBrick assembly, in order to adopt a spectrum of iGEM standard parts readily.
It is our design philosophy to build reusable biological devices. Our implementation of this circuit to evolve DTE, D-tagatose 3-epimerase can be considered as a proof-of-work. The operons in this circuit is alternatively forward and reverse, in order to minimize read through of terminators and to prevent homologous recombination of the plasmid itself. As this circuit will be present in chassis cells under severe growth pressure, cells could use extreme methods to maximize its survival. This design is meant to prevent such things from happening.
Selecting Circuit
We have explored the potential to use a series of antibiotic resistance genes as quantitative reporter, which their expression levels are converted into different growth rates of chassis organisms. In order to make the characterization data reusable, we have substituted pPsi promoter to pLac. pLac is a more characterized promoter than D-psicose induced one. We hope later iGEM teams could be inspired or refer to our data when they are designing their own directed evolution system or implementing their own growth pressure. Three antibiotic resistance protein for kanamycin, streptomycin and ampicillin are characterized. In summary, kanamycin achieves overall best performance owing to a wider dynamic range. Streptomycin has similar mechanisms as kanamycin, which the antibiotics bind to bacterial ribosome, but SmrR, the corresponding resistance gene, is less efficient in fighting against streptomycin, thus its linear range is limited. For ampicillin, its mechanism differs from the above two by blocking cell wall (peptidoglycan, in specific) formation. In opposite to SmrR, the resistance protein, AmpR, is too strong so that very low level of expression leads to a total survival of all bacterial, which is also not a good candidate for our directed evolution system.
It is not enough to find a suitable range of growth pressure, but it is also important for this pressure range to match the range in Sensing Circuit. During the first rounds of evolution, the enzyme suffers from poor conversion rate, which may lead to a total elimination of the library. Even though enzyme activity is expected to increase in the next few rounds, intercellular D-psicose concentration could still be no match for the extracellular IPTG level. In all, a tuning mechanism is necessary.
Coupling Circuit
The Tl-Cp cassette is the solution. The Tl-Cp cassette includes a series of 4 basic parts which are homologous but differs quantitatively. Questions that this cassette tackles with are the most crucial ones underlying directed evolution and as well as synthetic biology itself: How to match the input and output of cascades of regulatory elements and how to determine the level of final yield, in this case, of the antibiotic resistance proteins.
The design of Tl-Cp cassette takes advantage of the helicase activity of ribosome. When ribosome approaches the end of upstream CDS, it will unwind the hairpin structure between two CDS and make the downstream RBS available. The length of this hairpin structure corresponds to the optimized distance that a ribosome can unwind.
From the mechanisms above and previous researches (Mendez-Perez, 2012), we can presume with more confidence that through such hairpin structure, the downstream translational level is quantitatively associated with the upstream (which is actually driven by its own RBS).
Evolving Circuit
Evolving Circuit summarizes all regulatory mechanisms used above and acts to quantitatively convert different productivities of DTE to a spectrum of growth rates. Utilizing the characterization from Selecting and Coupling circuits, we achieved the suitable antibiotic concentration to exert growth pressure in evolution.
[1] Mendez-Perez, D, et al. "A translation-coupling DNA cassette for monitoring protein translation in Escherichia coli. " Metabolic Engineering 14.4(2012):298-305.
[2] Rennig, M., et al. "TARSyn: Tunable Antibiotic Resistance Devices Enabling Bacterial Synthetic Evolution and Protein Production. " Acs Synthetic Biology 7.2(2017).
[3] Takyar, Seyedtaghi, R. P. Hickerson, and H. F. Noller. "mRNA Helicase Activity of the Ribosome." Cell 120.1(2005):49-58.
[4] Chen, Hongyun, et al. "Chen, H. Bjerknes, M. Kumar, R. & Jay, E. Determination of the optimal aligned spacing between the Shine-Dalgarno sequence and the translation initiation codon of Escherichia coli mRNAs. Nucleic Acids Res. 22, 4953-4957." Nucleic Acids Research 22.23(1994):4953-4957.
Copyright Â© 2018 XJTU-China iGEM Team. All rights reserved.
