Team:Manchester-Graz/Project/Results
iGEM Manchester Header
iGEM Manchester-Graz - Results
Tweets by @iGEMMancGraz
Results
Validation of pCERI
via SDS-Page
To show how E. coli BL21 and E. coli Nissle 1917 deal with our synthetic vector pCERI we started a fermentation in 300 ml cultures. For each E. coli strain seven shaking flasks were inoculated with an ONC of E. coli BL21 and E. coli Nissle 1917, both transformed with pCERI , to start the fermentation with an OD600 of 0.05. In addition, negative (wild type E. coli BL21 and E. coli Nissle 1917) and positive controls were set up. The positive controls comprise recombinant E. coli BL 21 pSB3C5_J04421 and pSB3C5_J04450 . The transformed plasmids contain a lac promoter, leading to the expression of mRFP and CFP after the induction with IPTG. All samples were cultivated under the same conditions (37°C, 100 rpm). Every hour one fermentation of E. coli BL21_pCERI and E. coli Nissle 1917_pCERI was stopped on ice. One culture was fermented over night to reach 16 h. The positive, as well as the negative controls were stopped after 6h. After fermentation, all samples were centrifuged and prepared for SDS analysis . Regarding to protein concentrations measured by spectrophotometry (Nanodrop™) we loaded 10µg of soluble protein from the E. coli BL21_pCERI samples (1h-6h) as well as from the negative and positve controls on a SDS gel (Fig 1). For the E. coli Nissle 1917_pCERI and the E. coli Nissle 1917 negative control 20 µg of soluble protein were used for SDS-PAGE (Fig 2).
The SDS-PAGE of the soluble protein fractions of E. coli BL21 samples shows well separated but quite weak bands. In comparison to the negative control, the E. coli BL21_pCERI samples only show one further band at about 22 kDa. None of the proteins of our regulatory system refer to a band of this size. However, one of our proteins could be expressed in a truncated form. The band at about 24 kDa that is visible in the samples from hour 1 to 6 on the gel picture is also slightly observable in the negative control on the gel itself.
The positive controls show, next to some E. coli BL21 specific bands, typical bands for CFP (26.89 kDa) and mRFP (25.42 kDa). For the mRFP positive control two further bands at 40 kDa and 20 kDa are conspicuous. Those again, are observable on the gel itself in all other E. coli BL21 samples as well.
Figure 1 SDS-PAGE of soluble protein fractions of recombinant E. coli BL21 fermentations. Std = Standard, 1h-6h = E. coli BL21_pCERI
fermentations that were stopped on ice after the different time points, - = negative control: wild type E. coli BL21, +CFP/+mRFP =
positive controls: E. coli BL21 cultures with IPTG inducible gen expression of CFP and mRFP.
The SDS-PAGE for the E. coli Nissle 1917 as well showed quite separated but even weaker bands as the E. coli BL21 samples. For this SDS gel we tried to load as much soluble protein as possible into 10µl of SDS-PAGE sample volume. Still, not more than 20 µg, regarding spectrophotometric measurements, were possible to be prepared for the SDS analysis gel. It has to be considered, that protein concentration measurements at 280 nm by Nanodrop™ can differ from the actual yields. As E. coli Nissle 1917 is not an engineered protein expression stain, like E. coli BL21 is, the protein concentration stays lower due to protease activity.
Figure 2 SDS-PAGE of soluble protein fractions of recombinant E. coli Nissle 1917 fermentations. Std = Standard, 1h-6h = E. coli Nissle 1917_pCERI fermentations that were stopped on ice after the different time points, - = negative control: wild type E. coli Nissle 1917.
This second try of SDS-PAGE of all E. coli BL21 samples showed much weaker bands as the first try even though much more protein was loaded. That might indicate that our soluble protein fractions are not stable in 20 mM sodium phosphate buffer. The samples from the E. coli Nissle 1917 cultures were prepared for SDS-PAGE straight after fermentation. Still the final gel only showed very weak bands. A possible reason for that is that the E. coli Nissle 1917 strain still expresses several proteases, which would degrade our proteins and hence lower the total protein yield.
Both SDS-PAGE results are thus inconclusive. Hence, for more accurate characterization further experiments are performed.
via Fluorescence Assay
For the characterization of the finished pCERI a fluorescence-based assay was conducted. From all measured emission values we subtracted the measured values of non transformed BL21 or Nissle 1917 samples at the respective time to account for possible autofluorescence of the cells.  As a blank for the OD600 measurements we used sterile controls. As the positive control for mRFP we used E.coli BL21 pSB3C5_J04450 and for CFP E.coli BL21 pSB3C5_J04421 . Both were induced with a final concentration of 0.1 mM IPTG.
The measurement for mRFP whose expression should be induced by octanoyl-homoserine lactone (C8-HSL) yielded no substantial fluorescence emission at 607 nm, neither in E. coli BL21 (Fig. 3) nor in E. coli Nissle 1917 (Fig. 4). While the positive control emitted nearly 15000 RFU (relative fluorescence units) after 11 hours, none of the pCERI samples was measured with more than 100 RFU. A reason for these results could be that no or at least not enough CepR is produced. CepR works as an activator of PaidA and enables the recruitment of the RNA polymerase at the promoter and thus allows for the transcription of mRFP. This assumption is further supported by the fact, that BBa_K1670003 showed to be functional and highly active, when characterized individually.
Figure 3 Results of the emission measurement at 607 nm of E. coli BL21 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for mRFP.
Figure 4 Results of the emission measurement at 607 nm of E. coli Nissle 1917 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for mRFP.
In E. coli BL21 the emission measurement at 476 nm (Fig. 5) showed the most CFP fluorescence in the sample induced with 0.1 nM C8-HSL at nearly 8000 RFU, followed by the CFP positive control and the 100 nM C8-HSL sample at 4500 RFU. The samples with 0.1 nM and 100 nM 3-oxo-hexanoyl-homoserine lactone (3OC6-HSL) reached 3300 RFU and 2500 RFU respectively. The samples containing 0.1 nM and 100 nM of both HSLs showed the same fluorescence at 2000 RFU. The lowest result came from the samples without any HSL, which showed no increase in fluorescence at all and 600 RFU after 11 hours.
These results indicate that apparently EsaR which controls CFP expression not only binds 3OC6-HSL but is also sensitive to C8-HSL. C8-HSL (0.1 nM with 8000 RFU) even appears to have a stronger effect than 3OC6-HSL (0.1 nM with 3300 RFU). For both it appears that they seem to work better at very low concentrations, but inhibit gene expression at higher concentrations. The reason for the behavior of the samples with the combination of C8- and 3OC6-HSL has yet to be found. The fact that the sample with no added HSL showed no increase in fluorescence leads us to believe that EsaI is not expressed strong enough due to  PesaS being quite a weak promoter. This would also be consistent with the thesis, that mRFP expression is weak due to a lack of CepR  which is also under the control of PesaS.
Figure 5 Results of the emission measurement at 476 nm of E. coli BL21 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for CFP.
The OD600 measurement (Fig. 6) shows that the 0.1 nM C8-HSL sample did not just simply outgrow the others. All of the samples grew in a similar manner. Only the positive control for CFP expression showed slightly weaker growth rates.
Figure 6 Results of the OD600 measurements of E. coli BL21 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined , 0 nM HSL and the positive controls for mRFP and CFP.
In E. coli Nissle 1917 the results (Fig. 7) were very different. At the start of the measurement the fluorescence was similar to BL21 around 1000 RFU and 2000 RFU. In the course of the measurement however the emission decreased substantially in all samples. The 0 nM sample had the most fluorescence left after 11 h. It showed 1500 RFU, which is still very low compared to the results of the BL21 samples.
Figure 7 Results of the emission measurement at 476 nm of E. coli Nissle 1917 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for CFP in BL21.
These results for the Nissle 1917 pCERI transformants can be explained by the presence of proteases in E. coli Nissle 1917. It is much closer to a wild type E. coli than laboratory strains like BL21 which are optimized for labwork by the knockout of several proteases and nucleases. Apparently at first there was CFP produced but then the cells reacted to the presence of the foreign protein, putatively increased their protease expression, degraded the already formed CFP and kept CFP from reaching a measurable amount again. Comparing the different speed of CFP degradation in different samples while looking at the BL21 results, we see a possible correlation. The sample induced 0.1 nM C8-HSL in BL 21 rises fast and high to nearly 8000 RFU while in Nissle 1917 it starts at around 2000 RFU and is rapidly degraded until no emission is left after 6 h. The 100 nM 3OC6- and C8-HSL in BL21 rises slowly to about 2000 RFU while in Nissle 1917 the emission also slowly decreases over the course of time. So one could suspect that the stronger the expression of CFP occurs the stronger Nissle 1917’s protease response will be.
While this data could result from insufficient growth of the cells, this thought can be dismissed when we look at the OD600 curves (Fig. 8). They all grew perfectly well and also quite uniformly.
Figure 8 Results of the OD600 measurements of E. coli Nissle 1917 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive controls for mRFP and CFP in BL21.
via Chromobacterium violaceum CV026
E.coli BL21 pCERI and E.coli Nissle 1917 pCERI colonies are streaked along Chromobacterium violaceum CV026 and incubated over night at 30° C. C.violaceum,streaked along transformants with functional homoserine lactone synthases produces the purple pigment violacein. As positive controls 0.5 μl of pure homoserine lactones in concentrations ranging from 10 mM to 1 nM were spotted next to C. violaceum. As negative controls wildtypeBL21 and Nissle 1917 as well as 0.5 μl ddH2O were used. In the positive controls (Fig. 9, 10) one can see, that quite high homoserine lactone concentrations are needed to induce violacein expression. Octanoyl homoserine lactone induces reporter expression at concentrations of 100 µM and more (Fig.9), while 1 mM and more 3-oxohexanoyl-homoserine lactone is needed for violacein induction in C. violaceum (Fig.10).As expected in the negative controls no violacein expression can be observed in our reporter strain (Fig.11).
Figure 9 C. violaceum induced with different concentrations of octanoyl-homoserine lactone and grown over night at 30°C. A minimum of 100 µM C8-HSL is needed for violacein induction. C.v.: Chromobacterium violaceum CV026
Figure 10 C. violaceum induced with different concentrations of 3-oxohexanoyl-homoserine lactone and grown over night at 30°C. A minimum of 1 mM 3OC6-HSL is needed for violacein induction. C.v.: Chromobacterium violaceum CV026
The experiment shows functional expression of homoserine lactones in E.coli BL21 in high amounts (Fig. 12). The violacein expression in C. violaceum appears to be even stronger than in the 10 mM positive control (Fig. 9,10) of both homoserine lactones. Due to the individual characterization results LINK of the homoserine lactone synthases used in pCERI, we can assume that the reporter expression in C. violaceum is mainly due to 3-oxohexanoyl homoserine lactone synthesis by EsaI as it is highly unlikely, that the expression of the transcript under control of PesaRC is much stronger than BBa_J23100 .
Figure 11 C. violaceumstreaked along E.coli BL21 E.coli Nissle 1917 as well as 0.5 µl ddH2O spotted on the LB-plate and and grown over night at 30°C. Violacein cannot be observed due to a lack of homoserine lactones. C.v.: Chromobacterium violaceum CV026
Figure 12 E.coli BL21 pCERI and E.coli Nissle 1917 pCERI colonies streaked along Chromobacterium violaceum CV026 and incubated over night at 30° C. Homoserine lactone expression by E.coli BL21 pCERI induces violacein synthesis in C. violaceum . E.coli Nissle 1917 pCERI cannot produce enough homoserine lactones to induce violacein expression. C.v.: Chromobacterium violaceum CV026
Even though EsaI seems to be functional in E.coli Nissle 1917 when characterized individually , we cannot proof homoserine lactone synthesis of pCERI in E. coli Nissle 1917 with C. violaceum (Fig 12). No violacein expression can be observed after 16 h of incubation at 30° C.
BioBrick Characterization
CepR PaidA mRFP
The characterization of BBa_K1670002 along with BBa_K1670003 was conducted with a fluorescence-based assay as described in the experiments section .  As a positive control for mRFP expression E.coli BL21 pSB3C5_J04450 and as a positive control for CFP expression E.coli BL21 pSB3C5_J04421 diluted to an OD600 of 0.02 and induced with a final concentration of 0.1 mM IPTG are used.  From all measured emission values we subtracted the measured values of non transformed BL21 or Nissle 1917 samples at the respective time to account for possible autofluorescence of the cells.  As a blank for the OD600 measurements we used sterile controls.
The constitutively expressed cepR is an activator that will bind to its corresponding promoter PaidA when a threshold C8-HSL concentration is reached, starting the expression of mRFP.
Figure 13 Results of the mRFP emission measurement (607 nm) of E.coli BL 21 J61002_BBa_J23100_BBa_ K1670002 pSB3C5_BBa_K1670003 induced with at C8-HSL concentrations from 0.01 nM to 100 nM, 0 nM and the positive control for mRFP.
As seen in figure  13 the positive control delivered the strongest signal with up to 14900 RFU, followed by the 0.01 nM C8-HSL just above 10000 RFU. The other samples showed a very similar behavior with their emission peaking at around 7000 RFU.
Closer comparing the samples, induced with 0.01 nM, 100 nM, 0 nM respectively and the positive control (Fig. 14) we see that the 0.01 nM sample shows a higher fluorescence emission than the sample without any C8-HSL. However the 100 nM sample had a notably lower emission than the one with 0.01 nM. When cepR binds C8-HSL it should work as an activator for PaidA. Apparently this is only the case at rather low levels of C8-HSL, like 0.01 nM. At higher concentrations like 100 nM the promoter activity seems to be the same as in the sample without any C8-HSL. It looks like cepR is inhibited in its role as an activator at those C8-HSL concentrations. Considering the fluorescence of the 0 nM sample we can say that PaidA appears to have a high basal expression without induction by cepR or that cepR is also working as an inducer even when no C8-HSL is present.
Figure 14 Results of the mRFP emission measurement (607 nm) of E.coli BL 21 J61002_BBa_J23100_BBa_ K1670002 pSB3C5_ BBa_K1670003 induced with C8-HSL concentrations of 0.01 nM, 100 nM, 0 nM and the positive control for mRFP.  Error bars show the average deviation from the mean of the three measurements.
Another possible explanation for those results could be that at 100 nM C8-HSL is somehow toxic for the cells. Looking at the measured OD600 values (Fig. 15) that assumption can be dismissed. All samples carrying the genes for CepR and mRFP grow in a similar manner. After about 10 hours the OD600 of the 0.01 nM sample decreases, probably due to nutrient depletion.
Figure 15 Results of the OD600 measurement of E.coli BL 21 J61002_BBa_J23100_BBa_ K1670002 pSB3C5_ BBa_K1670003 induced with C8-HSL concentrations of 0.01 nM, 100 nM, 0 nM and the positive control for mRFP.  Error bars show the average deviation from the mean of the three measurements.
EsaR PesaRC CFP
BBa_K1670005 is characterized along with BBa_K1670001 with a fluorescence-based assay .
The constitutively expressed EsaR is expected to act as a repressor on PesaRC. In the presence of 3OC6-HSL it should dissociate from the DNA and allow expression of the PesaRC controlled CFP.
As seen in figure 16 the sample with 0.01 nM 3OC6-HSL had the highest emission at 476 nm of nearly 7000 RFU, followed by the 0 nM sample at 5000 RFU, the positive control at 4500 RFU sample, induced with 0.1 nM 3OC6-HSL, at around 3000 RFU. The other samples showed a very low emission at or under 1500 RFU.
Figure 16 Results of the CFP emission measurement (476 nm) of E.coli BL 21 J61002_BBa_J23100_BBa_ K1670005 pSB3C5_BBa_K1670001 induced with 3OC6-HSL concentrations from 0.01 nM to 100 nM, 0 nM and the positive control for CFP.
For a better overview we compare 0.01 nM, 10 nM, 0 nM and the positive control more closely (Fig. 17). Here we can observe that the fluorescence of the 0.01 nM sample is the highest. The sample without any 3OC6-HSL and the positive control are at about the same level and the sample with 10 nM showed no increase in fluorescence at all. Those results suggest that EsaR ceases its repressor function at very low 3OC6-HSL concentrations of 0.01 nM but inhibit the expression at 3OC6-HSL concentrations higher than 10 nM. The relatively high level of basal CFP expression in the 0 nM sample suggests that PesaR is not a very tightly regulated promoter.
Figure 17 Results of fluorescent emission measurement at 476 nm of E.coli BL 21 J61002_BBa_J23100_BBa_ K1670005 pSB3C5_BBa_K1670001 induced with 3OC6-HSL at concentrations of 0.01 nM, 10 nM, 0 nM and the positive control for CFP.  Error bars show the average deviation from the mean of the three measurements.
As a reason for the low fluorescence at high 3OC6-HSL concentrations you could also expect a possible toxicity for the cells, as low cell density would also mean low fluorescence. After looking at the OD600 measurements (figure 18) this thought can be dismissed. The cells inoculated with 10 nM even grew a little better than the ones with 0.01 nM or 0 nM but still showed a less fluorescence.
Figure 18 Results of the OD600 measurement of E.coli BL 21 J61002_BBa_J23100_BBa_ K1670005 pSB3C5_BBa_K1670001 induced with 3OC6-HSL concentrations of 0.01 nM, 10 nM, 0 nM and the positive control for CFP.  Error bars show the average deviation from the mean of the three measurements.
Homoserine Lactones Synthases characterization
BBa_K1670004 and BBa_K1670000 were cloned into J61002_BBa_J23100 for constitutive protein expression and transformed into E. coli BL21 and E.coli Nissle 1917. The transformed colonies are streaked along Chromobacterium violaceum CV026 and incubated over night at 30° C. C.violaceum,,streaked along transformants with functional homoserine lactone synthases produces the purple pigment violacein. As positive controls 0.5 µl of pure homoserine lactones in concentrations ranging from 10 mM to 1 nM were spotted next to C. violaceum (Fig. 9,10). As negative controls wildtype BL21 and Nissle 1917 as well as 0.5 µl ddH2O were used (Fig. 11). BBa_K1670004 seems to be functional in both E.coli strains (Fig.19). As expected the expression strain E.coli BL21 shows a better result, due to knocked out protease activity. Still E.coli Nissle 1917 produces enough BBa_K1670004 to induce weak violacein expression in C. violaceum . This confirm functionality of BBa_K1670004 .
Figure 19 BBa_K1670004 is constitutively expressed in E.coli BL 21 and E.coli Nissle 1917. BBa_K1670004 synthesizes 3-oxohexanoyl homoserine lactone that induces violacein expression in C.violaceum . C.v.: Chromobacterium violaceum CV026
Figure 20 BBa_K16700040 is constitutively expressed in E.coli BL 21 and E.coli Nissle 1917. BBa_K1670000 cannot synthesize octanoyl-homoserine lactone in sufficient concentrations to induces violacein expression in C.violaceum . C.v.: Chromobacterium violaceum CV026
BBa_K1670000 expression in both strains is putatively not functional in both used E.coli strains (Fig. 20) or the produced C8-HSL concentration is too low to induce violacein expression in C.violaceum as no violacein synthesis in C. violaceum can be observed.
Enzyme Expression & Analysis
To characterise aromatic amino acid decarboxylase BBa_K1670006 we performed biotransformations to analyse enzyme function via HPLC.
Our HPLC analysis showed little evidence that the biotransformation had been successful, as we found no significant peak for the presence of phenylethylamine (PEA), which should have been converted from phenylalanine. To confirm the presence or absence of the AADC, we ran an SDS-PAGE and subsequent Western Blot which showed that the AADC had not been expressed. After reviewing the literature [1], we decided that a higher IPTG concentration (1mM rather than 0.4mM) should be used for induction.
After repeating the cloning and induction of the AADC (with the aforementioned IPTG concentration), we performed SDS-PAGE (Fig. 21) and found a band at 71.7 kDa, suggesting to us that the AADC had been expressed.
Figure 21: SDS-PAGE gel of Nissle 1917, showing band at 71.7 kDa, suggesting presence of AADC.
To characterise transaminase ( BBa_K1670006 ) we used the same protocol as for AADC only with an IPTG concentration of 0.2 mM. This was apparently successful in inducing expression in both BL21 and Nissle, as shown by a band of 100 kDa (Fig. 22)
Figure 22 SDS-PAGE gel of BL21 and Nissle 1917, showing band at 100 kDa, suggesting presence of CvATA.
As it could be seen from the SDS-PAGE results, we were able to see bands at the size of the CvATA and AADC. This suggest that the enzymes were expressed in both BL21 and Nissle respectively, however further characterisation such as western blots are required to confirm the bands as the aforementioned enzymes. Prior to the expression, we were able to remove the PstI restriction site in the AADC gene via PCR mutagenesis such that the gene is not spliced before cloning into the submission vector.
Further work includes biotransformation of the expressed enzyme in Nissle 1917 and BL21 wherein the substrate is provided in order to view the activity of the enzymes and confirm production of the desired substance via HPLC.
[1.] Liu, F., Xu, W., Du, L., Wang, D., Zhu, Y., Geng, Z., Zhang, M. and Xu, W. (2014). Heterologous Expression and Characterization of Tyrosine Decarboxylase from Enterococcus faecalis R612Z1 and Enterococcus faecium R615Z1. Journal of Food Protection, [online] 77(4), pp.592-598. Available at: http://www.ncbi.nlm.nih.gov/pubmed/24680070 [Accessed 29 Jul. 2015].
Links
Home
Project
Team
Human Practices
Modeling
Parts
Notebook
Attributions
Safety
iGEM
Contact us
iGEM Subteam Graz
TU Graz, Institute of Molecular Biotechnology
Petersgasse 14
8010 Graz
iGEM Subteam Manchester
University of Manchester
Manchester Institute of Biotechnology
131 Princess Street
Manchester M17DN
Connect
Boring Stuff
About us
Legal Information
© iGEM Manchester-Graz
