<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester-Graz_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester-Graz/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Manchester Header</TITLE></P><P><TITLE>iGEM Manchester Notebook</TITLE></P><DIV id="inhalte"><H1>Notebook</H1><DIV class="notebookweek"><DIV class="notebookheadline2">Week 1 (29.6.2015 - 5.7.2015)</DIV><DIV class="notebooktextman"><UL><LI>This week we transformed two of our future BioBricks which we were kindly given by members of our lab. The decarboxylase (AADC) as well as the aminotransferase (CvATA) were provided inserted into pET-28a</LI><LI>Both AADC and CvATA were transformed into <I>E. coli</I> Bl21 using a standard transformation protocol and plated on LB agar Kan.</LI><LI>Colonies were observed after overnight growth (Fig. 1), plates were then stored in the fridge over the weekend for starter cultures to be set up on Monday.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 1</B> LB agar Kan plates showing colonies for AADC (labelled DC) and CvATA (labelled CV) after overnight growth at 37°C. 
</DIV></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 2 (6.7.2015 - 12.7.2015)</DIV><DIV class="notebooktextgraz"><UL style="margin-top:130px;"><LI>This week we blunt end ligated eight of our gBlocks (esaR, esaR v2, cepR, cepR v2, esaI, CFP, mRFP, bla) that are needed for the assmebly of two different variants of <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Vectordesign"> pCERI</A> into pJET 2.1 respectively, transformed them into <I> E. coli </I> TOP 10 and plated some of the obtained transfomants on LB - Amp for plasmid preps. </LI><LI>Those plasmid preps were cut with the corresponding restriction enzymes and an analyzing gel (Fig. 1-3) was run to check for correct size of the inserts.</LI><LI>Correct plasmid constructs (marked with a green arrow) were sent for sequencing.</LI><LI>The remaining plasmid DNA of the 11 correct vectors was cut and purified to be used for future OE-PCR</LI><DIV style="height:950px; width:780px;"><DIV id="pictureleft" style="margin:5px;"><B>Figure 1</B> Different gBlocks were cloned into pJET and transformed into <I>E. coli</I> TOP10. To verify correct cloning the plasmids were digested to analyse proper insert size. EsaRv2 = pJET_EsaRv2 cut with HindIII/SacI, expected insert size 863 bp; cepRv2 = pJET_cepRv2 cut with EcoRI/EcoRV, expected insert size 944 bp; EsaR = pJET_EsaR cut EcoRI/EcoRV, expected insert size 960 bp. The marked samples (green arrows) were sent for sequencing. Std: NEB 2-Log DNA Ladder
</DIV><DIV id="pictureleft" style="margin:5px;"><B>Figure 2</B> Different gBlocks were cloned into pJET and transformed into <I>E. coli</I> TOP10. To verify correct cloning the plasmids were digested to analyse proper insert size. EsaI = pJET_EsaI cut with BamHI/PstI, expected insert size 765 bp; CFP = pJET_CFP cut with BamHI/HindIII, expected insert size 934 bp; RFP = pJET_RFP cut SacI/BamHI, expected insert size 1104 bp. EsaRv2 = pJET_EsaRv2 cut with HindIII/SacI, expected insert size 863 bp The marked samples (green arrows) were sent for sequencing. Std: NEB 2-Log DNA Ladder
</DIV><DIV id="pictureleft" style="margin:5px;"><B>Figure 3</B> Different gBlocks were cloned into pJET and transformed into <I>E. coli</I> TOP10. To verify correct cloning the plasmids were digested to analyse proper insert size. bla = pJET_bla cut with EcoRV/SpeI, expected insert size 1083 bp; cepR = pJET_cepR cut with SacI/HindIII, expected insert size 843 bp; EsaI = pJET_EsaI cut BamHI/PstI, expected insert size 765 bp. The marked samples (green arrows) were sent for sequencing. Std: NEB 2-Log DNA Ladder
</DIV></DIV><DIV class="notebooktextman"><UL style="margin-top:130px;"><LI>To prepare starter cultures for AADC and CvATA colonies were picked for inoculation into LB Kan and grown overnight. 2 mL of overnight starter culture was added to 200 mL LB broth and incubated until OD600 of 0.4 was reached for AADC and 0.6 for CvATA.
	</LI><LI>Upon reaching target optical density AADC and CvATA expression cultures were induced with IPTG achieving a final concentration in the flask of 0.5 and 0.2 mM respectively.
	</LI><LI>Following induction the AADC culture was grown for a further 4 hours at 37°C, 200 rpm. After incubation cells were spun down at 4000 rpm for 20 minutes to obtain pellets which were stored at -20°C. (Fig. 4)
	</LI><LI>The CvATA expression culture was incubated overnight at 20°C following induction, cell pellets were obtained using the same methods as AADC and stored under the same conditions.
	</LI><LI>This week we started developing methods for HPLC analysis of all compounds in our pathway. We prepared standards for L-DOPA, Dopamine, L-Tyrosine, L-Tyramine, L-Phenylalanine and Phenylethylamine as a 0.1 M solution in water (Fig. 5). We first tried analysing our standards using the HPLC with 30% Acetonitrile and 70% H2O which unfortunately did not show distinct peaks for most of our compounds.


<DIV id="pictureleft" style="margin-top:55px;"><B>Figure 4</B> Tubes after centrifugation with cell pellets visible in the base of the tube.
</DIV><DIV id="pictureleft" style="margin-top:37px;"><B>Figure 5</B> Prepared HPLC standards as 0.1 M solutions in water of L-Tyrosine, Dopamine, L-DOPA, L-Tyramine, L-Phenylalanine and PEA respectively.
</DIV></LI></UL></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 3 (13.7.2015 - 19.7.2015)</DIV><DIV class="notebooktextgraz"><UL><LI>Sequencing results of week 1 were evaluated. For all gBlocks, except for <A href=" https://2015.igem.org/Team:Manchester-Graz/Project/Vectordesign"> esaRv2</A> correct constructs were found. </LI><LI>Plasmid preparations of the correct gBlocks were digested and inserts of the correct size were cut out of the analyzing gel and purified. </LI><LI>New ligations of esaR and esaRv2 with pJET 1.2 were performed and transformed into <I>E.coli </I>TOP 10. Transformants were streaked out again. After plasmid isolation they were again digested to check for the correct size of the inserts. Plasmid preparations containing the correct size were sent for sequencing. All of the obtained sequences showed mutations or deletions (Fig. 1). </LI></UL><DIV id="picturemiddle" style="margin-left:11px; margin-bottom:20px;"><B>Figure 1</B> Part of a sequencing result of EsaRv2. The sequence contains a deletion at position 618. As the mutation is in the coding region of the gene the corresponding sequence could not be used for further work.
</DIV><UL><LI>Thus the experiment was repeated.</LI><LI>pPIC9 was digested with BglII to cut out the <I> E.coli </I> origin of replication as well as the Amp-resistance marker. The linearized vector was further dephosphorylated and purified. </LI><LI>OE-PCR of bla_p15A was performed. As the first attempts resulted in some unspecific bands on the analyzing gel (Fig. 2), the PCR was repeated with more restrictive conditions. Even though an unspecific band remained, a band of the correct size (~2 kb) was obtained and purified. The fragment was digested with BglII to allow cloning into pPIC9.</LI><LI>bla_p15A was cloned into pPIC9. The resulting circular vector was transformed into <I>E.coli </I>TOP 10 and plated on LB-Amp. However no transformants were obtained, putatively due to the very low concentration of the ligation product. Transformation was repeated with a higher amount of the ligation.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 2</B> Overlap extension PCR of bla_p15A resulting in unspecific bands;Std: NEB 2-Log DNA Ladder
	</DIV></UL></DIV><DIV class="notebooktextman"><UL><LI>This week we performed whole cell biotransformation using the cell pellets obtained from our expression cultures of AADC and CvATA. </LI><LI>L-Phenylalanine and Dopamine were used as substrates for AADC and CvATA respectively. Substrates were prepared as a 5 mM solution in 1 mL 0.1 M phosphate buffer (pH 8). Whole cells were weighed and 250 uL of substrate was added to 20 mg of whole cells.</LI><LI>We also aimed to improve our HPLC methods this week by adjusting to 20/80 (ACN/H2O). This resulted in a separation of PDA from L-Phenylalanine allowing analysis of AADC biotransformation. However L-Tyrosine and L-DOPA were still too close on the spectrum.</LI><LI>A further method was trialled using a 90/10 ratio of ACN to H20 which showed good separation for L-DOPA and L-Tyrosine although L-Tyrosine and L-Tyramine aswell as L-DOPA and Dopamine did not show clearly separated spectra</LI></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 4 (20.7.2015 - 26.7.2015)</DIV><DIV class="notebooktextgraz"><UL><LI>After sequencing multiple clones of pJET_esaR and pJET_esaRv2 that all contained different mutations we decided to take one clone of each construct that only contained a single deletion, go on with the experiment and cut out the sequenced gBlocks with the corresponding restriction enzymes.</LI><LI>OE-PCR with the digested and purified fragments of <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Vectordesign"> esaR</A>, cepR and esaI as well as with esaRv2, cepRv2 and esaI were performed to obtain the two fragments esaR_cepR_esaI and cepR_EsaR_EsaI_v2, that basically contain the same genetic information, however in a shuffled order. The fused gene fragments was afterwards blunt-end cloned into pJET 1.2. Correct cloning was verified by a restriction digest and an analyzing gel (Fig. 1).</LI><LI>As both esaR_cepR_esaI as well as cepR_esaR_esaI_v2 still contained esaR-sequences with deletions site, directed mutagenesis with a primer-pair for each construct was performed to correct the sequence at the respective positions. After the PCR the methylated template plasmids were digested with DpnI to minimize the background still containing the deletions. After digestions, the corrected vectors were transformed into <I> E. coli</I> Top10.</LI><LI>Last weeks OE-PCR of bla_p15A was repeated to obtain more of the fragment. The PCR product was purified and digested with BglII. The gene fragment with the sticky ends was ligated again into pPIC9 that had been prepared last week. After ligation the circular vector was transformed into <I>E. coli </I>Top10.</LI><LI>This week pPIC9_p15A_bla transformation resulted in way more transformants. Plasmid isolation of several constructs was performed. To check for the correct insert the vector was digested with SacI and BamHI (Fig.1) . Correct vectors resulted in ~5600 bp and a ~2000 bp band. Clone 9 was found and used for further experiments.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 1</B> Analyzing gel of digested vector constructs; Overlap-extension PCR products were blunt-end cloned into pJET 1.2 and transformed into E.coli Top 10. To verify correct cloning, the plasmids were digested to analyse proper size. Std.: Quick-Load Purple 2-Log DNA Ladder; v1: pJET_EsaR_CepR_EsaI_v1 cut EcoRI/SacI, expected insert size ~ 1800 bp; v2: pJET_CepR_EsaR_EsaI_v2 cut EcoRI/SacI, expected insert size ~ 1800 bp; pPIC9: pPIC9_p15A_bla cut PstI, expected insert size ~ 2000bp; 1-11: plasmid preps 1-11of different vector constructs; X: empty slot
			</DIV></UL></DIV><DIV class="notebooktextman"><UL><LI>This week we aimed to improve our HPLC methods further in order to separate peaks of L-DOPA and Dopamine as well as L-Tyrosine and L-Tyramine. To achieve this we used 95% H2O and 5% ACN which unfortunately did now allow for more refined separation of our compounds</LI><LI>We developed a method using methanol as a solvent instead of ACN with a 95/5 ratio which lead to clear separation of L-DOPA from Dopamine as well as L-Tyrosine from L-Tyramine</LI><LI>Analysis of AADC cell pellets was conducted using SDS-PAGE to confirm the presence of the enzyme in the sample. Cells were spun down at 13.2 rpm for 15 minutes and denatured with Laemmli buffer and then boiled for 5 minutes. Samples were diluted in Laemmli buffer (1:10, 1:8, 1:6, 1:4, 1:2 respectively) to allow for improved visualisation. The SDS-PAGE was run at 250 volts for 30 minutes.</LI><LI>As the band could not be clearly identified it was decided to perform a western blot to confirm the presence of AADC in the sample (Fig. 2).</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 2</B> SDS-PAGE of cells expressing AADC (70.3 kDa in size) with a band visible at approximately 70 kDa.
	</DIV></UL></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 5 (27.7.2015 - 2.8.2015)</DIV><DIV class="notebooktextgraz"><DIV id="pictureright" style="margin-right:30px; margin-top:120px;"><B>Figure 1</B> pJET_CepI (M1 and M2) cut with PstI and NotI on the control gel.
			</DIV><UL><LI>Due to some delay in the gene fragment synthesis, finally our last gBlock <A href=" https://2015.igem.org/Team:Manchester-Graz/Project/Vectordesign"> cepI</A> arrived. We cloned it into the pJET vector with blunt-end ligation. It was then transformed into <I>E. coli </I> TOP10 for plasmid amplification.</LI><LI>The two resulting clones (M1 and M2) were plated for plasmid preparations. The obtained DNA was cut for an analyzing gel using PstI and NotI. The bands fit the expected size of around 900 bp (Fig. 1) and were sent for sequencing.</LI><LI>The sequencing results showed that one of the clones (M1) contained no mutations.</LI><LI>The remaining plasmid preparation of M1 was cut with NotI and SpeI for overlap extension PCR (OE-PCR) and purified on a preparative gel.</LI><LI>That cepI was then used for OE-PCR with cfp. The PCR product was run on a gel. It showed the expected bands at 2000 bp. The DNA was extracted from the gel slices and ligated into pJET. The resulting plasmid was used for transformation into <I> E. coli </I> TOP10.</LI><LI>This week we set up our controls as well. We transformed <A href="http://parts.igem.org/Part:BBa_J04450"> pSB3C5_BBa_J04450 (mRFP)</A> and <A href="http://parts.igem.org/Part:BBa_J04421"> pSB1C3_BBa_J04421 (cfp) </A> from the distribution kit 2015 into <I> E.coli </I> Top10 and isolated the plasmids. To get the same vector background with the same origin of replication (p15A) as our pCERI-vector, we cut both constructs with EcoRI/PstI. After digestion pSB3C5 was also dephosphorylated. BBa_J04421 was cloned into pSB3C5 afterwards.</LI><LI>The size verified pPIC9_p15A_bla clone was streaked out again to isolate more plasmid. After plasmid isolation the vector was cut with EcoRV and XbaI to get the p15A_bla fragment for the Gibson assembly of pCERI.</LI></UL></DIV><DIV class="notebooktextman"><UL><LI>This week we repeated the SDS-PAGE analysis of AADC samples in order to visualize the presence of the His-tagged enzyme via a Western Blot (as this could not be confirmed with SDS-PAGE alone). Following a run at 250 volts for 30 minutes the SDS-PAGE gel was soaked in 1x Towbin buffer while nitrocellulose membranes were soaked in methanol. We then constructed the gel ‘sandwich’ and ran the electroblotting apparatus for 60 mins. Each membrane was then soaked in 30ml PBS + 0.5% BSA for 1hr. This was then repeated, with the addition of antibody, and left for 1hr. The membrane was then washed with PBS + 0.1% TWEEN. Once completed, the membrane was soaked in BCIP/NBT purple substrate in order to visualize bands.</LI></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 6 (3.8.2015 - 9.8.2015)</DIV><DIV class="notebooktextgraz"><DIV id="pictureright" style="margin-right:30px; margin-top:120px;"><B>Figure 1</B> Analyzing gel of pJET_cfp_cepI cut PstI/SacI. Correct plasmids should yield bands at 2669 bp, 1733 bp and 416 bp.Std: NEB 2-Log DNA Ladder; 14-17: plasmid preps of different clones; 
			</DIV><UL><LI>Plasmid isolation of pJET_cfp_cepI was performed and correct cloning was verified by restriction digestion with SacI and PstI.</LI><LI>On the analyzing gel (Fig. 1) we saw the three bands we expected: One just below 3000 bp (vector backbone), one around 1700 bp (cfp_cepI) and one at 400 bp (smaller part of the backbone). </LI><LI>The clones 14 and 17 were sent for sequencing. Clone 17 showed no mutation. The remaining plasmid of clone 17 was cut with HindIII and SpeI for a preparative agarose gel. The resulting band for cfp_cepI was purified to be further used for the Gibson assembly of pCERI.</LI><LI>We conducted plasmid isolation of transformants containing <A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_BBa J04421 (cfp) </A> and size verified correct cloning by restriction digestion. Correct clones are stored on 4°C to be used as positive controls for the analysis of pCERI.</LI><LI>As the previous site directed mutagenesis attempts to correct the mutation in esaR did not get us the results we hoped for and only showed the same deletion as the template DNA we repeated the site directed mutagenesis-experiment with altered experimental parameters, prolonging the DpnI digest from 1h to 12h (overnight) to make sure to get rid of all template DNA of the PCR. This resulted in significantly less clones on the LB-Amp plates after transformation into <I> E.coli</I>  Top 10. These clones will get sequenced next week.</LI><LI>This week we also made our first Biobricks. PCRs to attach the Biobrick suffix and prefix to the sequence verified gBlocks of P<SUB>aidA</SUB>_mRFP, P<SUB>esaRC</SUB>_cfp, cepI, esaI and cepR were performed. PCR-products were digested with EcoRI and PstI and purified. All products were ligated into linearized and dephosphorylated shipping vector pSB1C3. <A href="http://parts.igem.org/Part:BBa_K1670004"> pSB1C3_BBa_K1670004</A> showed no transformants on the plates and thus transformation was repeated. Clones of all other constructs were streaked out again for plasmid preps and correct cloning was verified by restriction digests (Fig. 2-4). Clones containing size verified inserts were sent for sequencing.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 2</B><A href="http://parts.igem.org/Part:BBa_K1670003"> pSB1C3_BBa_K1670003</A>(encoding mRFP)cut EcoRI/PstI; Std: NEB 2-Log DNA Ladder; 1-4: plasmid preps of different clones; 
		</DIV><DIV id="pictureleft" style="margin:5px;"><B>Figure 3</B><A href="http://parts.igem.org/Part:BBa_K1670000"> pSB1C3_BBa_K1670000</A> (encoding cepI) cut EcoRI/PstI; Std: NEB 2-Log DNA Ladder; 5-7: plasmid preps of different clones
		</DIV><DIV id="pictureleft" style="margin:5px;"><B>Figure 4</B><A href="http://parts.igem.org/Part:BBa_K1670001"> pSB1C3_BBa_K1670001</A> (encoding cfp) and <A href="http://parts.igem.org/Part:BBa_K1670002"> pSB1C3_BBa_K1670002</A> (encoding cepR) cut EcoRI/PstI; Std: NEB 2-Log DNA Ladder; 5-7: plasmid preps of different clones
		</DIV></UL></DIV><DIV class="notebooktextman"><UL><LI>This week we transformed pCDF-1b without insert into E. coli 5-alpha for amplification. pCDF-1b will act as the expression vector for three of our gBlocks; Tyrosine Hydroxylase (TH), CYP2D6 as well as Tyrosinase. Cultures obtained on plates from overnight incubation at 37°C were inoculated into LB Kan and grown overnight. Overnight cultures were used for plamid preparation to obtain pCDF-1b.</LI><LI>pCDF-1b was further transformed into E. coli BL21 to act as an empty vector control. A pCDF-1b empty vector expression culture was grown and induced with 0.5 uM IPTG and then spun down and frozen for future use as empty vector control. </LI><LI>Empty pTrcHis2 B was transformed into E. coli 5-alpha for amplification. pTrcHis2 B will act as expression vector for one of our gBlocks; Tyrosinase Chaperone. Transformed cells were grown overnight on LB agar Kan, cultures were inoculated into LB Kan for overnight growth. Overnight cultures were used for plasmid preparation to obtain pure pTrcHis2 B.</LI></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 7 (10.8.2015 - 16.8.2015)</DIV><DIV class="notebooktextgraz"><DIV id="pictureright" style="margin-right:30px; margin-top:120px;"><B>Figure 1</B><A href="http://parts.igem.org/Part:BBa_K1670004"> pSB1C3_BBa_K1670004</A> (encoding esaI) cut EcoRI/PstI with an expected insert size of ~700 bp; Std: NEB 2-Log DNA Ladder; 1-2: plasmid preps of different clones
			</DIV><UL><LI>Sequencing reports for the size verified biobrick parts of last week showed no mutations in all sent samples, meaning we have already four biobricks ready. Transformation of <A href="http://parts.igem.org/Part:BBa_K1670004">pSB1C3_BBa_K1670004</A> was repeated. Obtained transformants were streaked out again and their plasmids isolated. Correct cloning was verified by restriction digests with EcoRI /PstI and run on an analyzing gel (Fig.1). Sequencing of both clones showed no mutations in clone 1, verifying our fifth biobrick. </LI><LI>Sequencing results of the repeated attempt of site directed mutagenesis of esaR showed that we were able to correct the deletion. Additionally, we also sequenced the other genes of the fragment esaR_cepR_esaI that also verified the sequence of the OE-PCR. Thus this fragment was excised from the pJET backbone and purified. </LI><LI>Furthermore, the sequence verified pJET_esaR_cepR_esaI clone was used as a template for PCR to attach the biobrick pre- and suffix to esaR to generate our last biobrick <A href=" http://parts.igem.org/Part:BBa_K1670005"> pSB1C3_BBa_K1670005</A>. </LI><LI>After the successful site directed mutagenesis experiment all fragments for the Gibson assembly of pCERI (P<SUB>aidA</SUB>_mRFP, P<SUB>esaRC</SUB>_cfp_cepI, P<SUB>esaS</SUB>_esaR_cepR_esaI and bla_p15A) were finally available. Gibson assembly was performed according to the protocol: 0.05 nmol of each fragment were used for the reaction. 2µl of the reaction mix were transformed into <I>E.coli </I>TOP10. Plated transformants were grown over the weekend.</LI><LI>Additionally this week we also started to set up the vectors needed for the characterization of our biobricks. <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A>and <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> each were cloned into pSB3C5 to receive a vector backbone with a lower copy number of 10-15. Further <A href="http://parts.igem.org/Part:BBa_ J23100"> J61002_J23100 </A> from the distribution kit was transformed into <I>E.coli </I>TOP 10. After plasmid isolation the vector was cut with SpeI and PstI and dephosphorylated to be able to clone the regulatory genes of our quorum sensing systems <A href="http://parts.igem.org/Part:BBa_K1670005"> BBa_K1670005</A> and <A href="http://parts.igem.org/Part:BBa_K1670002"> BBa_K1670002</A> right after the constitutive promoter BBa_J23100 next week.</LI><LI>New electrocompetent cells of <I>E.coli </I>BL21 and <I>E.coli </I>Nissle 1917 were made according to the same protocol used for E.coli Top 10 and a test transformation with both strains was performed. Transformation efficiencies of 2,3*10<SUP>8</SUP> cfu/ml for <I>E.coli </I>BL21 and 3,7*10<SUP>8</SUP>cfu/ml for <I>E.coli </I>Nissle were reached.</LI></UL></DIV><DIV class="notebooktextman"><UL><LI>After receiving all of our gBlocks we chose Tyrosine Hydroxylase (TH) to perform a digestion and attempt to insert it into the pSB1C3 iGEM submission vector. Digestion with EcoRI-HF and PstI was performed followed by ligation into the vector. The ligation reaction of TH and pSB1C2 was transformed into TOP10 but no cultures were seen the next day</LI><LI>Simultaneously PCR was performed on TH and the iGEM vector to construct a multimer. Following PCR the reaction was analysed on a gel to identify the presence of a multimer which would be expected to remain in the well. Unfortunately no DNA was identified in the well. </LI><LI>In order to clone our gBlock of TH into a vector we attempted A tailing with Taq polymerase to insert it into a TOPO vector and achieve higher cutting efficiency. A tailing was performed followed by ligation into TOPO and transformation into OneShot competent cells. Colonies were observed the following day and overnight liquid cultures were incolulated. Miniprep was performed on overnight cultures.</LI><LI>The plasmid preparation of TH TOPO was digested with EcoRI-HF and PstI to test whether insertion had occurred. After analysis via gel electrophoresis no insert was found and we decided to repeat TA cloning using TH as well as another gBlock Tyrosinase (TYR). TA tailing of gBlocks was performed followed by ligation into TOPO and transformation into One Shot competent cells. Transformed cells were plated on LB agar Kan, colonies were seen after one day on TYR TOPO and after two days on empty vector control, no colonies were seen on TH TOPO.</LI></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 8 (17.8.2015 - 23.8.2015)</DIV><DIV class="notebooktextgraz"><DIV id="pictureright" style="margin-right:30px; margin-top:120px;"><B>Figure 1</B>Analyzing gel of putatively assembled pCERI plasmid prep cut with multiple restriction enzyme combinations. 1: pCERI cut BamHI with an expected bands at 3579 bp, 1989 bp and 1617 bp; 2: pCERI cut EcoRI/PstI with expected bands at 3493 bpm 2499 bp and 1193 bp; 3: pCERI cut PstI/HindIII with expected bands at 2102 bp, 1790 bp, 1703 bp and 1590 bp Std.: NEB 2-Log DNA Ladder</DIV><UL><LI><A href="http://parts.igem.org/Part:BBa_K1670005"> BBa_K1670005</A> and <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> were ligated into <A href="http://parts.igem.org/Part:BBa_ J23100"> J61002_J23100 </A>and transformed into <I>E.coli </I>Top 10. Transformants were streaked again for plasmid isolation. Correct cloning was verified by digestions with EcoRI and PstI. Correct plasmids were cotransformed into <I>E.coli </I> BL21 and single colonies were streaked on LB-Amp.</LI><LI>Plasmid isolation was performed with colonies from last week’s Gibson assembly. The gained plasmids were cut in different combinations (Fig.1) to check for the correct assembly of pCERI. One clone showed correct bands for all different approaches and was sent for sequencing. </LI><LI>The putatively correct pCERI was transformed into <I> E.coli </I> BL21 and <I> E.coli </I> Nissle 1917 and single colonies were streaked on LB-Amp.</LI><LI><A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_J04421 </A> and <A href="http://parts.igem.org/Part:BBa_J04450"> pSB3C5_J04450 </A> were also transformed into <I> E.coli </I> BL21 to be used as positive controls later on. </LI></UL></DIV><DIV class="notebooktextman"><UL><LI>This week colonies from TOPO transformation were inoculated over the weekend and grown overnight for analysis on Monday. Colonies after one day were only observed in TYR TOPO, empty vector control plates showed colonies after two days whereas there was no growth in overnight cultures of TH TOPO.</LI><LI>Both TOPO TYR and empty TOPO cultures were miniprepped and digested with EcoRI + PstI and XbaI + SpeI for analysis. Gel analysis of digestions revealed only a single band the size of the vector in the EP digestion and a band the size of the vector plus insert in XS digestion (Fig. 2). The decision was made to repeat digestion overnight and to perform single as well as double digestions.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 2</B> Gel analysis of TYR inserted into TOPO vector cut with multiple restriction enzymes.1: TYR TOPO cut with EcoRI-HF and PstI with bands expected at 3.9 kb and 1.5 kb; 2: TYR TOPO cut with XbaI and SpeI with bands expected at 3.9 kb and 1.5 kb; 3: TYR TOPO undigested control with vector + insert size of 5.5 kb, supercoiling expected; 4: empty TOPO digested with EcoRI-HF/PstI single band expected at 3.9 kb; 5: empty TOPO undigested single band expected at 3.9 kb. L: NEB 2-Log DNA Ladder.
		</DIV></UL><UL><LI>A blunt end cloning reaction was set up for TH as well as TYR using the Zero Blunt TOPO PCR Cloning Kit by life technologies. Cloned TH TOPO, TYR TOPO as well as empty TOPO were transformed into One Shot competent cells and plated for overnight growth on LB agar Kan. A small number of colonies was seen after overnight growth and inoculation for overnight growth was performed. Overnight cultures resulted in growth of only TYR TOPO and empty TOPO leading us to believe that colonies on TYR TOPO also resulted from TOPO ligation with itself.</LI><LI>Overnight cultures of TOPO blunt end cloned TYR TOPO blunt end (BE) and empty TOPO were miniprepped and digested with various restriction enzymes to be analysed on a gel and determine whether insertion had occurred (Fig. 3), unfortunately all bands were the size of the vector without insert.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 3</B> Gel analysis of TYR inserted into TOPO vector as well as empty TOPO cut with multiple restriction enzymes. 1: TYR TOPO cut with EcoRI-HF/SpeI with bands expected at 3.5 kb and 1.5 kb; 2: TYR TOPO cut with XbaI/SpeI with bands expected at 3.5 kb and 1.5 kb; 3: TYR TOPO cut with EcoRI-HF with band expected at 5 kb; 4: TYR TOPO cut with PstI with band expected at 5 kb; 5: TYR TOPO cut with XbaI with band expected at 5 kb; 6: TYR TOPO cut with SpeI with band expected at 5 kb; 7: TYR TOPO undigested expected size of 5 kb (supercoiling expected); 8: empty TOPO cut with EcoRI-HF/PstI with band expected at 3.5 kb; 9: empty TOPO cut with XbaI/SpeI with band expected at 3.5 kb; 10: empty TOPO undigested expected size of 3.5 (supercoiling expected). L: NEB 2-Log DNA Ladder.
		</DIV></UL><UL><LI>Due to the small number of colonies resulting from blunt end TOPO cloning the decision was made to repeat the transformation of TYR TOPO, TH TOPO and empty TOPO with the hope of achieving a higher number of colonies (a successful TOPO cloning reaction should yield hundreds of colonies). Repeated TOPO cloning of TYR and TH resulted in colonies on TYR TOPO as well as TH TOPO, both were inoculated and grown overnight. Overnight cultures only showed growth for TH TOPO which was miniprepped and analysed via digestion and gel electrophoresis. Gel analysis showed bands the size of the vector only for all digestion reactions meaning that no insertion had occurred.</LI><LI>We performed PCR mutagenesis on AADC in order to remove a PstI restriction site within the gene by replacing a G with an A in the sequence, conserving the original amino acid sequence (Fig. 4). The four PCR reactions were analysed on a gel and reaction 4 was selected for further work as it yielded the brightest bands.</LI></UL><DIV id="pictureleft" style="margin:5px;"><B>Figure 4</B> Part of AADC sequencing including mutagenesis primers to remove PstI restriction site.
		</DIV><UL><LI>Phosphorylation-ligation of AADC was performed to yield a complete AADC pET-28a construct following PCR mutagenesis. The ligation reaction was further transformed into NEB DH5-alpha competent cells and grown overnight on LB agar Kan. Colonies were seen after overnight growth and inoculated for overnight growth in liquid medium.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 5</B> : Gel analysis of AADC mutagenesis PCR reactions. Wells 1-4 represent PCR reactions 1-4 which were set up under different conditions. Expected size 8 kb (AADC plus pET-28a). L: NEB 2-Log DNA Ladder.
		</DIV></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 9 (24.8.2015 - 30.8.2015)</DIV><DIV class="notebooktextgraz"><DIV id="pictureright" style="margin-right:30px; margin-top:120px;"><B>Table 1</B> OD<SUB>600</SUB> after harvesting of the cells after different incubation intervals and protein concentration, measured with Nanodrop at 280 nm, after cell disruption; PC_mRFP: positive control mRFP; PC_CFP: positive control CFP </DIV><UL><LI>This week we started <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments"> characterization</A> of pCERI. To proof expression of the regulatory proteins as well as our genes of interest, fermentations were conducted. Overnight cultures were grown with single cell colonies of<I> E.coli </I> BL21 pCERI (due to a limited amount of space in our incubator, fermentation is only performed with <I> E.coli </I> BL21 pCERI, while <I> E.coli </I> Nissle 1917 will be cultivated next week). 300 ml of LB – Amp were inoculated to a starting OD<SUB>600</SUB> of 0.05 and incubated at 37°C and 100 rpm. As positive controls recombinant <I> E.coli </I> BL 21 <A href="http://parts.igem.org/Part:BBa_J04421">pSB3C5_J04421</A> and <A href="http://parts.igem.org/Part:BBa_J04450">pSB3C5_J04450</A> are cultivated under the same conditions. As negative controls wildtyp <I>E.coli</I> BL21 as well as a sterile control containing only LB-Amp without any inoculum, are used. Cultivation was stopped after 1, 2, 3, 4, 5 and 6 hours and cultures stored on ice till the cells were harvested by centrifugation at 4000 rpm for 10 minutes. </LI><LI> 
				Cell disruption was performed by <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments">sonication</A>. An insoluble protein fraction was separated by centrifugation. Protein concentration of each sample was estimated by Nanodrop spectrometry (Tab. 1). Proteins of each sample, normalized to an OD<SUB>600</SUB> of 0.7 were separated by SDS-poly-acrylamid gel electrophoresis. Proteins were stained with Coomassie Brilliant Blue(Fig. 1). As the negative control (<I> E.coli </I> BL 21 (-))showed very poor growth during fermentation, this sample is still missing on the gel and thus the SDS-PAGE should be repeated next week. Furthermore, three SDS-gels of other research groups were conducted simultaneously to ours and resulted in a poor quality. Thus, the SDS-PAGE should be repeated anyways. However, we already see that our CFP positive control (<I> E.coli </I> BL 21 <A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_J04421 </A>) seems to show no expression of CFP at all, as no band can be seen at 26.8 kDa as opposed to the strong band of mRFP at 25.4 kDa. This confirms an old user review of <A href="http://parts.igem.org/Part:BBa_J04421"> BBa_J04421 </A>) , that this part might not work properly.
				<DIV id="pictureright"><B>Figure 1</B> Stained SDS-poly-acrylamid-gel of cell lysat samples of <I>E.coli </I> BL21 pCERI after different incubation intervals (1h – 6h) all normalized to an OD<SUB>600</SUB> of 0.7; CFP:<I> E.coli </I> BL 21 <A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_J04421 </A> induced with 0.1 mM IPTG, however the part seems to show no expression of CFP; mRFP: <I> E.coli </I> BL 21<A href="http://parts.igem.org/Part:BBa_J04450"> pSB3C5_J04450 </A> induced with 0.1 mM IPTG</DIV></LI><LI>This week also the first try of the fluorescence assay for the characterization biobrick parts <A href="http://parts.igem.org/Part:BBa_K1670002">BBa_K1670002</A> (cepR) along with <A href="http://parts.igem.org/Part:BBa_K1670003">BBa_K1670003</A> (P<SUB>aidA</SUB>_mRFP) and <A href="http://parts.igem.org/Part:BBa_K1670005">BBa_K1670005</A> (esaR) along with <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A> (P<SUB>esaRC</SUB>_cfp) was performed according to our <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments">protocol</A> with HSL concentrations ranging from 0.01 mM to 100 nM. Already at very low C8-HSL concentrations all samples containing <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> (P<SUB>aidA</SUB>_mRFP) and at low concentrations of 3OC6-HSL samples containing <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A> (P<SUB>esaRC</SUB>_cfp) showed high fluorescence emission. That might indicate, that the added HSL were diluted in wrong way and thus the experiment will be repeated next week. Once again <I> E.coli </I> BL 21 <A href="http://parts.igem.org/Part:BBa_J04421">pSB3C5_J04421</A> induced with 0.1 mM IPTG showed hardly any fluorescence at 476 nm confirming the result of the SDS-PAGE.</LI></UL></DIV><DIV class="notebooktextman"><UL><LI>This week we miniprepped six overnight cultures of mutated AADC in DH5-alpha competent cells. Miniprepped plasmids were then digested with BamHI-HF and NotI-HF (in order to excise the gene from pET-28a) as well as PstI (to determine whether the mutation had worked and the restriction site had been removed). Unmutated AADC was also digested as a control. Digestions were analysed via gel electrophoresis (Fig. 2) and sample 1 was chosen to be sent off for sequencing to confirm mutation.</LI><DIV id="pictureright"><B>Figure 2</B> Gel analysis of AADC PCR mutagenesis cut with BamHI-HF, NotI-HF and PstI. 1: mutated AADC undigested with band expected at 7.2 kb; 2: mutated AADC colony 1 (mutAADC1) with band expected at 7.2; 3: mutAADC2 with band expected at 7.2; 4: mutAADC3 with band expected at 7.2; 5: mutAADC4 with band expected at 7.2; 6: mutAADC5 with band expected at 7.2; 7: mutAADC6 with band expected at 7.2; 8: undigested unmutated AADC with band expected at 7.2 kb; 9: digested unmutated AADC with bands expected at 7 kb and 0.2 kb. L: NEB 1kb DNA Ladder.
</DIV><LI>We also started working with our Nissle 1917 cells this week by streaking out the stab culture provided by Ardeypharm on LB agar plates. Colonies were picked the next day and grown overnight in LB. Chemically competent cells were then produced from overnight cultures following the protocol by <A href="https://2012.igem.org/Team:Penn/Nissle ">Penn iGEM 2012</A></LI><LI>Furthermore we also performed PCR on our gBlocks in order to add additional nucleotides to the ends of our gene to allow for more efficient cutting. gBlocks PCR was then analysed via gel electrophoresis (Fig. 3) to confirm the presence of our genes. However, bands were only visible for TYR and CHAP and the decision was made to repeat the PCR reaction for TH and CYP2D6 using different conditions. Repeated PCR did not yield any bands for TH and CYP2D6.</LI><LI>Another PCR reaction was performed on mutAADC and CvATA in order to remove them from the pET-28a vector as well as adding the BioBrick prefix and suffix. Four reactions were set up for both genes under different conditions and analysed on a gel following PCR (Fig. 4).</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 3</B> Gel analysis of PCR performed on gBlocks for TH, TYR, CHAP and CYP2D6. 1: TH with band expected at 1.6 kb; 2: TYR with band expected at 1.6; 3: CHAP with band expected at 0.7 kb; 4: CYP2D6 with band expected at 1.6 kb. L: NEB 1 kb DNA Ladder.
		</DIV><DIV id="pictureleft" style="margin:5px;"><B>Figure 4</B> Gel analysis of mutAADC and CvATA PCR to add BioBrick prefix and suffix.
		</DIV></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 10 (31.8.2015 - 6.9.2015)</DIV><DIV class="notebooktextgraz"><DIV id="pictureright" style="margin-right:30px; margin-top:120px;"><B>Figure 1</B> Analyzing gel of J61002_BBa_J23100_BBa_K1670000 (1) and of J61002_BBa_J23100_BBa_K1670004 (2) cut EcoRI/PstI. Both constructs contain the desired insert,evident from bands at 748 bp (1) and 772 bp (2); Std.: NEB 2-Log DNA Ladder 
			</DIV><UL><LI><A href="http://parts.igem.org/Part:BBa_K1670000"> BBa_K1670000</A> and <A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> were ligated into <A href="http://parts.igem.org/Part:BBa_ J23100"> J61002_J23100 </A>and transformed into <I>E.coli </I>Top 10. Transformants were streaked again for plasmid isolation. Correct cloning was verified by digestions with EcoRI and PstI (Fig.1) that should yield bands at 2039 bp (J61002-backbone) and 748 bp for BBa_K1670000 and 772 bp for BBa_K1670004.  Correct plasmids were transformed into <I>E.coli </I> BL21 and <I>E.coli </I> Nissle 1917. Single colonies were streaked on LB-Amp to be further used for characterization purposes next week.</LI><LI>Second <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments">fermentations</A> were conducted with <I>E. coli</I> Nissle 1917 pCERI Cells were harvested after 1h, 2h, 3h, 4h, 5h, 6h, and 16h. Total protein was isolated from the cells. A SDS-PAGE was run with this week’s samples as well as with last week’s samples. Even though more protein was theoretically loaded on the gel, all bands of the BL21 samples appeared to be weaker compared to last week’s gel, which might indicate protein degradation in the 20 mM sodium-phosphate buffer (Fig 2 and 3). </LI><DIV id="pictureleft" style="margin-right:100px; margin-bottom:20px; margin-left:5px;"><B>Figure 2</B> SDS-PAGE of soluble protein fractions of recombinant <I>E. coli</I> BL21 fermentations. Std = Standard, 1h-6h = <I>E. coli</I> BL21_pCERI fermentations that were stopped on ice after the different time points, - = negative control: wild type <I>E. coli</I> BL21, +CFP/+mRFP = positive controls: <I>E. coli</I> BL21 cultures with IPTG inducible gen expression of CFP and mRFP.</DIV><DIV id="pictureleft" style="margin-bottom:40px; margin-left:5px;"><B>Figure 3</B> SDS-PAGE of soluble protein fractions of recombinant <I>E. coli</I> Nissle 1917 fermentations. Std = Standard, 1h-6h = <I>E. coli</I> Nissle 1917_pCERI fermentations that were stopped on ice after the different time points, - = negative control: wild type <I>E. coli</I> Nissle 1917.</DIV></UL><UL style="margin-top:1320px;"><LI>This week we optimized our fluorescence-based assay for the characterization of our quorum sensing regulated promoters <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A> and <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> by adjusting the measurement intervals and slightly adapting the measured emission wavelength. The <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Results">results </A> proofed the functionality of <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A> and <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> and further characterized <A href="http://parts.igem.org/Part:BBa_K1670002"> BBa_K1670002</A> and <A href="http://parts.igem.org/Part:BBa_K1670005"> BBa_K1670005</A>. The fluorescent assay was further used to characterize pCERI in both used chassis, <I>E.coli</I> BL21 and <I>E.coli</I> Nissle 1917. The <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Results">results </A> showed us that we are probably having some problems with the expression of EsaI and CepR, leading to a lack of 3OC6-HSL and no expression of mRFP and CFP. When we supplied the cells with external C8- and 3OC6-HSL however we got a good CFP emission in BL21, however hardly any mRFP expression, which might indicate, that the CepR expression under the control of the esaS-promoter is not sufficient to produce enough activator. Due to a lack of the CepR-activator, no mRFP gets expressed. This assumption is further supported by the fact, that <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> showed to be functional, when characterized individually. <I>E. coli</I> Nissle 1917 on the other hand seems to degrade the formed CFP quite fast, as the measured fluorescence seems to rise in the beginning, but starts to decrease after a few hours.</LI></UL></DIV><DIV class="notebooktextman"><UL><LI>This week mutAADC, CvATA, TYR as well as CHAP with the added BioBrick prefix and suffix were first purified and then further digested with EcoRI-HF and PstI for insertion into linearised pSB1C3 which was also digested with EcoRI-HF and PstI. </LI><DIV id="pictureright"><B>Figure 4</B> Gel analysis of digested genes coding for mutAADC, CvATA, TYR as well as CHAP to be inserted into pSB1C3 as well as expression vectors pCDF-1b and pTrcHis1b (for TYR and CHAP respectively). 1: TYR cut with EcoRI-HF and PstI with band expected at 1.6 kb; 2: CHAP cut with EcoRI-HF and PstI with band expected at 0.9 kb; 3: AADC cut with EcoRI-HF and PstI with band expected at 2 kb; 4: CvATA cut with EcoRI-HF and PstI with band expected at 1.4 kb; 5: pSB1C3 cut with EcoRI-HF and PstI with band expected at 2.1 kb; 6: TYR cut with NcoI-HF and AvrII with band expected at 1.6 kb; 7: pCDF-1b cut with NcoI-HF and AvrII with band expected at 3.6 kb; 8: CHAP cut with NcoI-HF and HindIII-HF with band expected at 0.9 kb; 9: pTrcHis2b cut with NcoI-HF and HindIII-HF with band expected at 4.4 kb. L: NEB 2-Log DNA Ladder.
</DIV><LI>TYR and CHAP were also digested with NcoI-HF and AvrII and NcoI-HF and HindIII-HF respectively for insertion into pCDF-1b and pTrcHis2b expression vectors which were also digested with the same sets of enzymes.</LI><LI>All digestions were analysed via gel electrophoresis to confirm the presence of fragments of the right size (Fig. 4). Fragments of the right size were seen for AADC, CvATA, pSB1C3, pCDF-1b as well as pTrcHis2b which were further purified and AADC and CvATA were ligated into pSB1C3 and transformed into 5-alpha. 
Colonies from overnight growth were picked and grown in LB overnight and miniprepped. Miniprepped AADC and CvATA in pSB1C3 were digested with EcoRI-HF and PstI to confirm insertion into the vector via gel analysis. The decision was made to send a sample of each gene off for sequencing to confirm insert.
</LI><LI>As no bands were seen for both TYR and CHAP after digestion the PCR reaction to add prefix and suffix was repeated yielding visible bands for both genes. Half of each chosen reaction (reaction 2 for TYR and 1 for CHAP) was purified before digestion, the other half was digested unpurified. Both TYR and CHAP were cut with EcoRI-HF and PstI. Digestions were analysed via gel electrophoresis and the presence of both TYR and CHAP was confirmed (Fig. 2) before both were ligated into pSB1C3 and transformed. Colonies were seen on all plates with more colonies on samples that were cleaned up prior to ligation.</LI><DIV id="pictureleft" style="margin:5px;"><B>Figure 5</B> Gel analysis of TYR and CHAP cut with EcoRI-HF and PstI for insertion into pSB1C3. TYR1: TYR purified before digestion with band expected at 1.6 kb; TYR2: TYR not purified before digestion with band expected at 1.6 kb; CHAP1: CHAP purified before digestion with band expected at 0.9 kb; CHAP2: CHAP not purified before digestion with band expected at 0.9 kb. Ladder: NEB 2-Log DNA Ladder.
</DIV></UL></DIV></DIV><DIV class="notebookweek"><DIV class="notebookheadline2">Week 11 (7.9.2015 - 13.9.2015)</DIV><DIV class="notebooktextgraz"><UL><LI>Our last week was used for the characterization of the synthases of pCERI after receiving the reporter strain <I> Chromobacterium violaceum </I>CV026 that is used to detect homoserine lactones in its surrounding. Due to a mutated native homoserine lactone synthase the HSL regulated violacein expression gets only induced by external synthesis of <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments">HSLs </A>. To proof functionality of our synthases EsaI and CepI, we cloned <A href="http://parts.igem.org/Part:BBa_K1670000">BBa_K1670000</A> and <A href="http://parts.igem.org/Part:BBa_K1670004">BBa_K1670004</A> downstream of a constitutive promoter and transformed the vector into <I>E.coli</I> BL21 and Nissle 1917. Transformants were streaked along <I>C. violaceum</I>. Additionally control groups were streaked along our reporter strain too. On the one hand the HSL-synthases were characterized individually but on the other hand HSL synthesis was also determined with pCERI in both <I>E.coli</I> BL21 and Nissle 1917. In a first try none of the tested samples showed to be able to induce violacein expression in <I>C. violaceum</I>, not even our positive controls, where we spotted purified HSLs alongside the reporter strain. As the experiment did not seem to work properly with the current parameters, we tried to lower the incubation temperature from 37°C to 30°C. In this setup our positive controls were able to induce violacein expression (Fig.1, 2).</LI><DIV id="pictureleft" style="width:350px; margin-top:20px;"><B>Figure 1</B><I> C. violaceum</I> induced with different concentrations of octanoyl-homoserine lactone and grown over night at 30°C. A minimum of 100 µM C8-HSL is needed for violacein induction. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV><DIV id="pictureleft" style="width:350px; margin-top:20px;"><B>Figure 2</B><I> C. violaceum</I> induced with different concentrations of 3-oxohexanoyl-homoserine lactone and grown over night at 30°C. A minimum of 1 mM 3OC6-HSL is needed for violacein induction. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV><LI>The <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Results">results </A> of this experiment showed, that only <A href="http://parts.igem.org/Part:BBa_K1670004">BBa_K1670004</A> seems to be functional in both, <I>E.coli </I> BL21 and <I>E.coli </I> Nissle 1917, however activity in Nissle 1917 is much weaker compared to BL21, which can probably explained by protease activities in Nissle 1917 (Fig.3). </LI><LI><A href="http://parts.igem.org/Part:BBa_K1670000">BBa_K1670000</A> showed no activity in both <I>E.coli </I> strains. This result might indicate, that either BBa_K1670000 is not functional or the synthesized octanoyl-homoserine lactone concentration is too low to induce violacein expression in <I>C. violaceum </I>. </LI><LI><I>E.coli</I> BL21 pCERI, which encodes for <A href="http://parts.igem.org/Part:BBa_K1670000">BBa_K1670000</A> and <A href="http://parts.igem.org/Part:BBa_K1670004">BBa_K1670004</A>, induced violacein expression strongly (Fig. 4). Interestingly even though <A href="http://parts.igem.org/Part:BBa_K1670004">BBa_K1670004</A> was able to induce violacein expression when transformed in <I>E.coli </I> Nissle 1917 individually, <I> E.coli </I> Nissle 1917 pCERI did not show any activity (Fig.4).  </LI><DIV id="pictureleft" style="width:350px; margin-top:20px;"><B>Figure 3</B><A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> is constitutively expressed in <I>E.coli </I> BL 21 and <I>E.coli </I> Nissle 1917. BBa_K1670004 synthesizes 3-oxohexanoyl homoserine lactone that induces violacein expression in <I>C.violaceum </I>. C.v.: <I>Chromobacterium violaceum </I> CV026
</DIV><DIV id="pictureleft" style="width:350px; margin-top:20px;"><B>Figure y</B><I>E.coli </I> BL21 pCERI and <I>E.coli</I> Nissle 1917 pCERI colonies streaked along <I>Chromobacterium violaceum </I> CV026 and incubated over night at 30° C. Homoserine lactone expression by  <I>E.coli </I> BL21 pCERI induces violacein synthesis in <I> C. violaceum </I>.   <I>E.coli</I> Nissle 1917 pCERI cannot produce enough homoserine lactones to induce violacein expression. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV></UL></DIV><DIV class="notebooktextman"><UL><LI>This week we inoculated colonies from TYR and CHAP in pSB1C3 which were grown over the weekend. Overnight colonies were miniprepped the next day and digested to confirm presence of insert. Gel analysis of digestion with EcoRI-HF and PstI (double and single digest) did not confirm the presence of the insert. </LI><LI>We also digested TYR and CHAP for insertion into their expression vectors pCDF-1b and pTrcHis2b respectively. Both vectors had already been digested and purified after digestion the previous week and digestions were confirmed via gel analysis with bands of the right sizes and linear vector fragments. TYR and CHAP were ligated into their corresponding vectors and ligations were transformed separately into 5-alpha. TYR was also transformed into BL21 and Nissle 1917 for expression trials.
Colonies resulting from separate transformations were grown in LB overnight and miniprepped for double transformation into BL21 and Nissle 1917.</LI><LI>CvATA and AADC in pET-28a were also transformed and expressed in both BL21 and Nissle 1917. Our chemically competent Nissle 1917 cells were used for transformation following a different <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Protocolsl">protocol</A> to BL21. Transformations for both enzymes into Nissle 1917 were successful (Fig. 1)</LI><DIV id="pictureleft"><B>Figure 5</B> Plates showing the growth of Nissle 1917 transformants of both CvATA (right) and AADC (left).
	</DIV><LI>As digestions and ligations of linearised pSB1C3 had not worked previously, we decided to perform a digestion of <A href="http://parts.igem.org/Part:BBa_J04450">BBa_J04450</A> with EcoRI-HF and PstI. Digested DNA was run on a gel (Fig. 2) and purified via gel extraction before ligation with TYR, CHAP, AADC and CvATA which had previously been digested. Ligations were transformed into 5-alpha resulting in white colonies after overnight growth indicating a successful cloning procedure. Different colonies were picked and inoculated for overnight growth and miniprep.</LI><DIV id="pictureleft" style="width:300px;"><B>Figure 6</B> : Gel analysis of  BBa_ J04450 digested with EcoRI-HF and PstI. Two digestions were run on the same gel both showing a band of 2 kb indicating the successful excision of mRFP.
	</DIV></UL></DIV></DIV></DIV><DIV id="twitter"><DIV id="twitterbox"><A class="twitter-timeline" href="https://twitter.com/iGEMMancGraz" data-widget-id="619465263411499008" height="500" width="520">Tweets by @iGEMMancGraz</A></DIV></DIV><DIV id="footer"><DIV id="footerlinks"><DIV class="footerheadline">
			Links
		</DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz" style="text-decoration:none;">Home</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Project" style="text-decoration:none">Project</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Team" style="text-decoration:none">Team</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Practices" style="text-decoration:none">Human Practices</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Modeling" style="text-decoration:none">Modeling</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Parts" style="text-decoration:none">Parts</A></DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Notebook" style="text-decoration:none">Notebook</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Attributions" style="text-decoration:none">Attributions</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Safety" style="text-decoration:none">Safety</A><A class="footerlinks" href="https://2015.igem.org" style="text-decoration:none">iGEM</A></DIV></DIV><DIV id="footercontact"><DIV class="footerheadline">
			Contact us
		</DIV>
		iGEM Subteam Graz
		TU Graz, Institute of Molecular Biotechnology
		Petersgasse 14
		8010 Graz
		
		iGEM Subteam Manchester 
		University of Manchester
		Manchester Institute of Biotechnology
		131 Princess Street
		Manchester M17DN
	</DIV><DIV id="footerconnect"><DIV class="footerheadline">
			Connect
		</DIV></DIV><DIV id="footerboringstuff"><DIV class="footerheadline">
			Boring Stuff
		</DIV><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Aboutus" style="text-decoration:none;">About us</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Legal" style="text-decoration:none">Legal Information</A></DIV><DIV id="footercopyright">© iGEM Manchester-Graz 
	</DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>