<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester-Graz_Project_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester-Graz/Project/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Manchester Header</TITLE></P><P><TITLE>iGEM Manchester-Graz - Experiments</TITLE></P><DIV id="fixednavi"><DIV id="twitterboxsmall"><A class="twitter-timeline" href="https://twitter.com/iGEMMancGraz" data-widget-id="619465263411499008" height="900" width="180">Tweets by @iGEMMancGraz</A></DIV></DIV><DIV id="inhalte"><H1>Experiments</H1><DIV class="inhalte_graz"><H2>Vector assembly</H2><DIV id="pictureright"><B>Figure 1 </B> Vector assembly strategy. [A] all gBlocks are cloned in pJet. [B] After sequence verifying the gBlocks are cut out of pJet using restriction sites in the flanking region of each gBlock, [C] With overlap extension PCR two orthree gBlocks are assembled to a bigger fragment. [D] The resulting product is cloned in pJet, sequence-verified and cut out again using restriction sites. [E] Using Gibson assembly the fragments are assembled to a full vector [F].</DIV><P>Our vector pCERI is assembled from scratch, by ordering synthesized genes on gBlocks that are needed for its functionality. All the gBlocks we ordered from IDT already contain overlapping sequences to the following gBlock. Before we actually want to assemble our vector, however, we want to make sure the used gBlocks have the right sequence. Thus all gene fragments are blunt-end cloned into the commercial vector pJET 1.2 for sequencing purpose (Fig.1 A). Sequence-verified gBlocks are afterwards cut out with flanking restriction sites on each gBlock, so no additional bases from pJET 1.2 that would disturb the following overlap-extension PCR (OE-PCR) remain (Fig.1 B). 
After sequence verification two to three gBlocks are fused to larger DNA fragments by OE-PCR (Fig.1 C). These fragments are also cloned into pJET 1.2 and cut out again after sequence verification (Fig.1 D). The fragment bla_p15A, containing the p15A origin of replication, is cloned into another commercial vector pPIC9 via two flanking BglII restriction sites to circumvent problems with two bacterial origins of replications on one plasmid (not shown). The four resulting fragments, P<SUB>aidA</SUB>_mRFP, P<SUB>esaRC</SUB>_cfp_cepI, esaI_cepR_esaR_P<SUB>esaS</SUB> and p15A_bla are fused to a functional circular vector by Gibson assembly (Fig.1 E).</P><H2>BioBrick Characterization</H2><P><A href="http://parts.igem.org/Part:BBa_K1670002"> BBa_K1670002</A> (cepR) is characterized along with <A href="http://parts.igem.org/Part:BBa_K1670003"> BBa_K1670003</A> (P<SUB>aidA</SUB>_mRFP) in a fluorescence-based assay. BBa_K1670002 is cloned into <A href="http://parts.igem.org/Part:BBa_ J23100"> J61002_J23100 </A> (Fig. 2) and is afterwards cotransformed together with <A href="http://parts.igem.org/Part:BBa_K1670003"> pSB3C5_BBa_K1670003</A> (Fig. 3) into <I> E.coli </I> BL21. </P><DIV id="pictureleft" style="margin-left:150px;"><B>Figure 2</B> J61002_BBa_J23100_BBa_ K1670002 (cepR)
</DIV><DIV id="pictureleft" style="margin-left:150px;"><B>Figure 3</B> pSB3C5_BBa_ BBa_K1670003 (P<SUB>aidA</SUB>_mRFP)
</DIV><P>Overnight cultures are inoculated and diluted to an OD of 0.02 and spotted on microtiter plates. Synthesis of mRFP is induced with different concentrations of octanoyl-homoserine lactone (C8-HSL) ranging from a final concentration of 0.01nM – 100 nM. Cultures are incubated at 37°C in a plate reader (Biotek Synergy MX) and fluorescence (Excitation 584 nm, Emission 607 nm) and OD<SUB>600</SUB>is measured every 30 minutes. 10 seconds before measuring the plate is shaken (orbital, medium). As a positive control <I>E.coli </I> BL21 <A href="http://parts.igem.org/Part:BBa_J04450"> pSB3C5_J04450 </A> diluted to an OD<SUB>600</SUB> of 0.02 and induced with a final concentration of 0.1 mM IPTG are used. As negative controls and to account for possible autofluorescence of the cells wild type <I>E.coli</I> BL21 is used. Also sterile controls containing only LB-media without any cells are pipetted onto the microtiter plates. </P><P><A href="http://parts.igem.org/Part:BBa_K1670005"> BBa_K1670005</A> (esaR) is characterized along with <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A> (P<SUB>esaRC</SUB>_cfp) in a fluorescence-based assay. BBa_K1670005 is cloned into <A href="http://parts.igem.org/Part:BBa_ J23100"> J61002_J23100 </A> (Fig. 4) and is afterwards cotransformed together with <A href="http://parts.igem.org/Part:BBa_K1670001"> pSB3C5_BBa_K1670001</A> (Fig. 5) into <I> E.coli </I> BL21. </P><DIV id="pictureleft" style="margin-left:150px;"><B>Figure 4</B> J61002_BBa_J23100_BBa_ K1670005 (esaR)
</DIV><DIV id="pictureleft" style="margin-left:150px;"><B>Figure 5</B> pSB3C5_BBa_K1670001 (P<SUB>esaRC</SUB>_cfp)
</DIV><P>Overnight cultures are inoculated and diluted to an OD of 0.02 and spotted on microtiter plates. Synthesis of mRFP is induced with different concentrations of 3-oxo-hexanoyl-homoserine lactone (3OC6-HSL) ranging from a final concentration of 0.01nM – 100 nM. Cultures are incubated at 37°C in a platereader (Biotek Synergy MX) and fluorescence (Excitation 439 nm, Emission 476 nm) and OD<SUB>600</SUB>is measured every 30 minutes. 10 seconds before measuring the plate is shook (orbital, medium). As a positive control <I>E.coli </I> BL21 <A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_J04421 </A> diluted to an OD<SUB>600</SUB>of 0.02 and induced with a final concentration of 0.1 mM IPTG are used. As negative controls and to account for possible auto fluorescence of the cells wild type <I>E.coli</I> BL21 is used. Also sterile controls containing only LB-media without any cells are pipetted on the microtiter plates.</P><P><A href="http://parts.igem.org/Part:BBa_K1670000"> BBa_K1670000</A> and <A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> are cloned into J61002_BBa_J23100 for constitutive expression of the homoserine lactone synthases CepI and EsaI (Fig. 6, 7). To verify functional expression of the synthases, single cell colonies are streaked on LB-Amp in parallel with <I> Chromobacterium violaceum </I> CV026.  Native<I> Chromobacterium violaceum </I> produces the characteristic purple pigment violacein.  In CV026 violacein production is inducible by multiple homoserine lactones with different sensitivities [1]. In consequence, violacein is only produced, if <A href="http://parts.igem.org/Part:BBa_K1670000"> BBa_K1670000</A> and <A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> turn out to be functional.  As a positive control, 3-oxohexanoyl-homoserine lactone and octanoyl-homoserine lactone dissolved in acetonitrile and diluted with ddH<SUB>2</SUB>O  to a concentration of 100 nm are spotted next to <I> C. violaceum </I> CV026 respectively. As a negative control, ddH<SUB>2</SUB>O is spotted. </P><DIV id="pictureleft" style="margin-left:150px;"><B>Figure 6</B> J61002_BBa_J23100_BBa_K1670000 (CepI)
</DIV><DIV id="pictureleft" style="margin-left:150px;"><B>Figure 7</B> J61002_BBa_J23100_BBa_K1670004 (EsaI)
</DIV><H2>pCERI characterization</H2><P>The fully sequenced pCERI is transformed into <I>E.coli</I> BL21 and <I>E.coli </I> Nissle 1917 that are used for the characterization of our regulatory system.  </P><P>The expression efficiency of the genes of interest is characterized by a fluorescence-based assay (<A href="https://2015.igem.org/Team:Manchester-Graz/Project/Results">Results</A>) Overnight cultures of both recombinant strains are inoculated with single colonies. The overnight cultures are diluted with LB-Amp to an OD of 0.02. 200 μ of the dilutions are pipetted in microtiter plates and incubated at 37 °C in the plate reader. Every 15 minutes mRFP fluorescence (Ex. 584 nm, Em. 607 nm), CFP fluorescence (Ex. 439 nm, Em. 476) and OD<SUB>600</SUB> is measured over a period of 13 h. The measured RFU (relative fluorescence unit) is either blotted against time or against OD<SUB>600</SUB> to evaluate the expression behavior of pCERI. Additionally several samples are either induced with 0.1 -50 nM of 3OC6-HSL and 0.1-100 nM of C8-HSL to explore putative advantages of higher HSL-synthesis, that might be possible by positioning each HSL-synthase, <A href="http://parts.igem.org/Part:BBa_K1670000"> BBa_K1670000</A> and <A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> at the beginning of the respective transcript. As positive controls <I>E.coli </I> BL21 <A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_J04421 </A> and <A href="http://parts.igem.org/Part:BBa_J04450"> pSB3C5_J04450 </A> diluted to an OD<SUB>600</SUB>of 0.02 and induced with a final concentration of 0.1 mM IPTG are used. As negative controls and to account for possible autofluorescence of the cells wild type <I>E.coli</I> BL21 and  E.coli Nissle 1917 are used. Also sterile controls containing only LB-media without any cells are pipetted on the microtiter plates.</P><DIV id="pictureleft" style="margin-right:10px;"><B>Table 1</B> Calculated molecular weight of expressed proteins by pCERI</DIV><P>Actual expression of our genes of interest as well as the regulatory proteins of pCERI is confirmed by fermentation of recombinant <I>E.coli</I> BL21 and <I>E.coli</I> Nissle 1917 for different intervals in the range of 1 h to 14 h. 300mL LB-Amp are inoculated with ONCs of each strain to a final OD<SUB>600</SUB> of 0.05 and incubated at 37°C and 100 rpm. As positive controls recombinant <I> E.coli </I> BL 21 <A href="http://parts.igem.org/Part:BBa_J04421"> pSB3C5_J04421 </A> and <A href="http://parts.igem.org/Part:BBa_J04450"> pSB3C5_J04450 </A> are cultivated under the same conditions. As negative controls wildtyp <I>E.coli</I> BL21 and  E.coli Nissle 1917, as well as a sterile control containing only LB-Amp without any inoculum, are used. Fermentations are stopped by putting the cultures on ice. Cells are harvested by centrifugation (4000 rpm, 10 min, 4°C). Pellets are afterwards resuspended in 20mM sodium phosphate buffer (pH=8; 10 ml buffer per gram wet cells). The resuspended cells are disrupted by sonication (6 min, 80% duty cycle). The lysate is centrifuged at 5000 rpm for 45 min to separate all insoluble protein from soluble proteins. Soluble proteins of all different fractions are separated by SDS-PAGE (12% polyacrylamide, V) and stained with Coomassie Brilliant Blue. Expected sizes of all recombinant proteins are priorly calculated with <A href="http://web.expasy.org/protparam/">ExPASy’s Protparam tool</A> (Table 1) [2].
</P><H2>L-DOPA and Dopamine Pathway Construction</H2><P>Our L-DOPA and dopamine biosynthesis pathway is made up of five enzymes, two of which were kindly donated to us by members of the Manchester Institute of Biotechnology. The remaining gene sequences were ordered as gBlocks from IDT containing the BioBrick prefix and suffix as well as subcloning restriction sites for the insertion into the respective expression vectors.</P><DIV id="pictureleft" style="width:770px;"><B>Figure 8</B>: Experimental flow for the introduction of L-DOPA and Dopamine synthesis pathway into <I>E. coli</I> BL21 and Nissle 1917</DIV><P>We took the approach of expressing each enzyme in individual vectors as this would allow for better analysis of their function. In the future, the enzymes for L-DOPA biosynthesis would be inserted into our regulation vector pCERI for regulated expression.</P><P>In order to enable subcloning into the expression vectors pCDF-1b and pTrcHis2b we used PCR to add 10 additional bases to the 3’ and 5’ ends of our gBlocks to increase restriction enzyme cutting efficiency. Unfortunately, after several attempts, only the gBlocks for tyrosinase and tyrosinase chaperone showed amplification, leading to the conclusion that there must be inconsistencies with the ordered sequences of both tyrosine hydroxylase and CYP2D6.
We decided to move forward with tyrosinase and chaperone, which were subcloned into their pCDF-1b and pTrcHis2b, respectively (Fig. 9 &amp; 10)</P><DIV id="pictureleft" style="margin-left:125px;"><B>Figure 9</B> Tyrosinase (TYR) subcloned into pCDF-1b for expression.
</DIV><DIV id="pictureleft" style="margin-left:125px;"><B>Figure 10</B> Tyrosinase Chaperone (TyrCHAP) subcloned into pTrcHis2b for expression.
</DIV><H3>BioBrick Construction &amp; PCR Site-Directed Mutagenesis</H3><P>Three of our enzymes were cut and subcloned into pSB1C3 following PCR amplification and addition of extra nucleotides. However, the AADC provided to us was found to have an internal PstI restriction site which we had to remove prior to creating <A href="http://parts.igem.org/Part:BBa_K1670006l">BBa_K1670006</A>.</P><P>We designed primers around the restriction site to incorporate the change from a CAG to a CAA codon whilst maintaining the Glutamine residue at position 563 of the enzyme (Fig. 11).
A detailed protocol for the PCR reaction can be found in our <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Protocols">Protocols</A> section.
The introduced mutation was confirmed via sequencing before subcloning into pSB1C3.</P><DIV id="pictureleft" style="margin-left:20px;"><B>Figure 11</B> Primers for site-directed PCR mutagenesis to remove internal PstI restriction site.
</DIV><H2>HPLC Method Development</H2><P>The characterisation of enzymes in our pathway involved the detection of reactants and products via HPLC (High Performance Liquid Chromatography). Separation of compounds via HPLC involves using 2 buffers in the stationary phase, one of which is usually water, while the other is methanol or acetonitrile (ACN). Polar compounds like dopamine and L-DOPA elute quicker, i.e. have shorter retention times, as they don’t interact with the column [3]. Although water enables polar compounds to travel faster through the stationary phase, presence of a hydrophobic compound in the mobile phase is also necessary for elution.</P><P>The aim was to develop an optimal mobile phase that gave different retention times to separate standards for the following substrates and products in our enzyme pathway (enzyme names are shown in brackets). These are:</P><UL style="font-size:16px;"><LI>L-Tyrosine and L-DOPA (tyrosinase or tyrosine hydroxylase)</LI><LI>L-Tyrosine and L-Tyramine (AADC)</LI><LI>L-DOPA and Dopamine (AADC)</LI><LI>L-Tyramine and Dopamine (CYP2D6)</LI><LI>Phenylalanine and phenylethylamine (AADC)    </LI></UL><P>1mM concentration of each sample was used, and elutants were detected at wavelengths of 230 nm, 254 nm (for aromatic compounds), 280 nm and 210 nm (for amino acids) [4].  We started with a mobile phase composition of 70% water, 30% ACN and 0.1% formic acid (all buffer compositions have 0.1% formic acid except when stated no acid used) as this is a generally accepted combination to separate polar/non-polar components . However, no separation of our compounds was achieved. As polar compounds elute faster, decreasing the proportion of polar ACN in the elution buffer would delay their elution. 90% water and 10% ACN gave different elution times and allowed separation for the following compounds:</P><UL style="font-size:16px;"><LI>Phenylalanine, tyrosine and L-DOPA</LI><LI>Tyramine and dopamine</LI><LI>PEA and phenylalanine</LI></UL><DIV id="pictureleft" style="margin-left:20px;"><B>Figure 12</B>: Analysis of pathway standards with peaks at 254 nm showing separated compounds: 1) phenylalanine, tyrosine and L-DOPA. 2) Tyramine and dopamine. 3) Phenylalanine and phenylethylamine 4) Tyrosine and tyramine. 5) L-DOPA and dopamine. Buffer compositions were 95% water and 5% methanol without formic acid.
</DIV><P>However, using these method we were not able to separate either Tyrosine and tyramine, or L-DOPA and dopamine. </P><P>To overcome this challenge, we further decreased ACN levels to 5% (95% hplc water), which again gave similar results. To address the issue of separation, the organic and less polar solvent methanol was later used instead of ACN. Using 90 % water / 10% methanol we separated L-tyrosine and tyramine by 0.3 min but not L-DOPA and dopamine. Given the chemical properties of dopamine we decided to run HPLC without presence of formic acid in the buffer, such that it would  give dopamine a negative charge [5]. This allows the dopamine to further interact with the column as it loses polarity and thus L-DOPA elutes faster. This was confirmed by running this method, as L-DOPA and dopamine were separated by more than a minute as seen in Fig. 12.</P><P>In conclusion, our HPLC method for enzyme characterisation, using a buffer composition of 95% water and 5% methanol without formic acid, successfully separated all combinations of our substrate and products of interest. </P><DIV style="background-color:#373737; width: 745px; height:170px; color: white; padding:10px;">
		[1] McClean, K., Winson, M., Fish, L., Taylor, A., Chhabra, S., Camara, M., Daykin, M., Lamb, J., Swift, S., Bycroft, B., Stewart, G. and Williams, P. (1997) Quorum sensing and Chromobacterium violaceum: exploitation of violacein production and inhibition for the detection of N-acylhomoserine lactones. Microbiology. 143 (12), pp. 3703-3711. 
		[2] Gasteiger, E., Hoogland, C., Gattiker, A., Duvaud, S., Wilkins, M., Appel, R. and Bairoch, A. (2005) <I>Protein Identification and Analysis Tools on the ExPASy Server</I>, John M. Walker (ed): The Proteomics Protocols Handbook, Humana Press pp. 571-607. 
[3] WHO, (2015). WHO Pharmacopoeia Library. [online] Available at: http://apps.who.int/phint/en/p/docf/ [Accessed 7 Sep. 2015].
[4] Corradini, D. (2010) Handbook of HPLC, Second Edition pg. 211-213
[5] Kurt, A. G., Aytan, E., Ozer, U., Ates, B. and Geckil, H. (2009) ‘Production of L-DOPA and dopamine in recombinant bacteria bearing the Vitreoscilla hemoglobin gene’, Biotechnology Journal, 4(7), pp. 1077–1088. doi: 10.1002/biot.200900130.</DIV></DIV></DIV><DIV id="footer"><DIV id="footerlinks"><DIV class="footerheadline">
			Links
		</DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz" style="text-decoration:none;">Home</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Project" style="text-decoration:none">Project</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Team" style="text-decoration:none">Team</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Practices" style="text-decoration:none">Human Practices</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Modeling" style="text-decoration:none">Modeling</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Parts" style="text-decoration:none">Parts</A></DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Notebook" style="text-decoration:none">Notebook</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Attributions" style="text-decoration:none">Attributions</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Safety" style="text-decoration:none">Safety</A><A class="footerlinks" href="https://2015.igem.org" style="text-decoration:none">iGEM</A></DIV></DIV><DIV id="footercontact"><DIV class="footerheadline">
			Contact us
		</DIV>
		iGEM Subteam Graz
		TU Graz, Institute of Molecular Biotechnology
		Petersgasse 14
		8010 Graz
		
		iGEM Subteam Manchester 
		University of Manchester
		Manchester Institute of Biotechnology
		131 Princess Street
		Manchester M17DN
	</DIV><DIV id="footerconnect"><DIV class="footerheadline">
			Connect
		</DIV></DIV><DIV id="footerboringstuff"><DIV class="footerheadline">
			Boring Stuff
		</DIV><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Aboutus" style="text-decoration:none;">About us</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Legal" style="text-decoration:none">Legal Information</A></DIV><DIV id="footercopyright">© iGEM Manchester-Graz 
	</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>