<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester-Graz_Project_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester-Graz/Project/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Manchester Header</TITLE></P><P><TITLE>iGEM Manchester-Graz - Results</TITLE></P><DIV id="fixednavi"><DIV id="twitterboxsmall"><A class="twitter-timeline" href="https://twitter.com/iGEMMancGraz" data-widget-id="619465263411499008" height="900" width="180">Tweets by @iGEMMancGraz</A></DIV></DIV><DIV id="inhalte" style="width:750px;"><H1>Results</H1><H2>Validation of pCERI</H2><H3>via SDS-Page</H3><P>To show how <I>E. coli</I> BL21 and <I>E. coli</I> Nissle 1917 deal with our synthetic vector pCERI we started a fermentation in 300 ml cultures. For each <I>E. coli</I> strain seven shaking flasks were inoculated with an ONC of <I>E. coli</I> BL21 and <I>E. coli</I> Nissle 1917, both transformed with <A href=" https://2015.igem.org/Team:Manchester-Graz/Project/Vectordesign">pCERI</A>, to start the fermentation with an OD<SUB>600</SUB> of 0.05. In addition, negative (wild type <I>E. coli</I> BL21 and <I>E. coli</I> Nissle 1917) and positive controls were set up. The positive controls comprise recombinant <I>E. coli</I> BL 21 <A href="http://parts.igem.org/Part:BBa_J04421">pSB3C5_J04421</A> and <A href="http://parts.igem.org/Part:BBa_J04450">pSB3C5_J04450</A>. The transformed plasmids contain a lac promoter, leading to the expression of mRFP and CFP after the induction with IPTG. All samples were cultivated under the same conditions (37°C, 100 rpm). Every hour one fermentation of <I>E. coli</I> BL21_pCERI and <I>E. coli</I> Nissle 1917_pCERI was stopped on ice. One culture was fermented over night to reach 16 h. The positive, as well as the negative controls were stopped after 6h. After fermentation, all samples were centrifuged and prepared for  <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Protocols">SDS analysis</A>. Regarding to protein concentrations measured by spectrophotometry (Nanodrop™) we loaded 10µg of soluble protein from the <I>E. coli</I> BL21_pCERI samples (1h-6h) as well as from the negative and positve controls on a SDS gel (Fig 1). For the <I>E. coli</I> Nissle 1917_pCERI and the <I>E. coli</I> Nissle 1917 negative control 20 µg of soluble protein were used for SDS-PAGE (Fig 2).</P><P>The SDS-PAGE of the soluble protein fractions of <I>E. coli</I> BL21 samples shows well separated but quite weak bands. In comparison to the negative control, the <I>E. coli</I> BL21_pCERI samples only show one further band at about 22 kDa. None of the proteins of our regulatory system refer to a band of this size. However, one of our proteins could be expressed in a truncated form. The band at about 24 kDa that is visible in the samples from hour 1 to 6 on the gel picture is also slightly observable in the negative control on the gel itself.
The positive controls show, next to some <I>E. coli</I> BL21 specific bands, typical bands for CFP (26.89 kDa) and mRFP (25.42 kDa). For the mRFP positive control two further bands at 40 kDa and 20 kDa are conspicuous. Those again, are observable on the gel itself in all other <I>E. coli</I> BL21 samples as well. </P><DIV id="pictureleft" style="margin-right:100px; margin-bottom:40px;"><B>Figure 1</B> SDS-PAGE of soluble protein fractions of recombinant <I>E. coli</I> BL21 fermentations. Std = Standard, 1h-6h = <I>E. coli</I> BL21_pCERI fermentations that were stopped on ice after the different time points, - = negative control: wild type <I>E. coli</I> BL21, +CFP/+mRFP = positive controls: <I>E. coli</I> BL21 cultures with IPTG inducible gen expression of CFP and mRFP.
</DIV><P>The SDS-PAGE for the <I>E. coli</I> Nissle 1917 as well showed quite separated but even weaker bands as the <I>E. coli</I> BL21 samples. For this SDS gel we tried to load as much soluble protein as possible into 10µl of SDS-PAGE sample volume. Still, not more than 20 µg, regarding spectrophotometric measurements, were possible to be prepared for the SDS analysis gel. It has to be considered, that protein concentration measurements at 280 nm by Nanodrop™ can differ from the actual yields. As <I>E. coli</I> Nissle 1917 is not an engineered protein expression stain, like <I>E. coli</I> BL21 is, the protein concentration stays lower due to protease activity.</P><DIV id="pictureleft" style="margin-bottom:40px;"><B>Figure 2</B> SDS-PAGE of soluble protein fractions of recombinant <I>E. coli</I> Nissle 1917 fermentations. Std = Standard, 1h-6h = <I>E. coli</I> Nissle 1917_pCERI fermentations that were stopped on ice after the different time points, - = negative control: wild type <I>E. coli</I> Nissle 1917.
</DIV><P>This second try of SDS-PAGE of all <I>E. coli</I> BL21 samples showed much weaker bands as the <A href="https://2015.igem.org/Team:Manchester-Graz/Notebook#Week9">first try</A> even though much more protein was loaded. That might indicate that our soluble protein fractions are not stable in 20 mM sodium phosphate buffer. The samples from the <I>E. coli</I> Nissle 1917 cultures were prepared for SDS-PAGE straight after fermentation. Still the final gel only showed very weak bands. A possible reason for that is that the <I>E. coli</I> Nissle 1917 strain still expresses several proteases, which would degrade our proteins and hence lower the total protein yield.
Both SDS-PAGE results are thus inconclusive. Hence, for more accurate characterization further experiments are performed.</P><H3>via Fluorescence Assay</H3><P>For the characterization of the finished pCERI a <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments">fluorescence-based assay</A>  was conducted. From all measured emission values we subtracted the measured values of non transformed BL21 or Nissle 1917 samples at the respective time to account for possible autofluorescence of the cells.  As a blank for the OD<SUB>600</SUB>  measurements we used sterile controls. As the positive control for mRFP we used E.coli BL21 <A href="http://parts.igem.org/Part:BBa_J04450">pSB3C5_J04450</A> and for CFP E.coli BL21 <A href="http://parts.igem.org/Part:BBa_J04421">pSB3C5_J04421</A>. Both were induced with a final concentration of 0.1 mM IPTG.</P><P>The measurement for mRFP whose expression should be induced by octanoyl-homoserine lactone (C8-HSL) yielded no substantial fluorescence emission at 607 nm, neither in <I>E. coli</I> BL21 (Fig. 3) nor in <I>E. coli</I> Nissle 1917 (Fig. 4). While the positive control emitted nearly 15000 RFU (relative fluorescence units) after 11 hours, none of the pCERI samples was measured with more than 100 RFU. A reason for these results could be that no or at least not enough CepR is produced. CepR works as an activator of P<SUB>aidA</SUB> and enables the recruitment of the RNA polymerase at the promoter and thus allows for the transcription of mRFP. This assumption is further supported by the fact, that <A href="http://parts.igem.org/Part:BBa_K1670003">BBa_K1670003</A> showed to be functional and highly active, when characterized individually.</P><DIV id="pictureleft"><B>Figure 3</B> Results of the emission measurement at 607 nm of <I>E. coli</I> BL21 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for mRFP.
</DIV><DIV id="pictureleft"><B>Figure 4</B> Results of the emission measurement at 607 nm of <I>E. coli</I> Nissle 1917 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for mRFP.
</DIV><P> In <I>E. coli</I> BL21 the emission measurement at 476 nm (Fig. 5) showed the most CFP fluorescence in the sample induced with 0.1 nM C8-HSL at nearly 8000 RFU, followed by the CFP positive control and the 100 nM C8-HSL sample at 4500 RFU. The samples with 0.1 nM and 100 nM 3-oxo-hexanoyl-homoserine lactone (3OC6-HSL) reached 3300 RFU and 2500 RFU respectively. The samples containing 0.1 nM and 100 nM of both HSLs showed the same fluorescence at 2000 RFU. The lowest result came from the samples without any HSL, which showed no increase in fluorescence at all and 600 RFU after 11 hours. 
These results indicate that apparently EsaR which controls CFP expression not only binds 3OC6-HSL but is also sensitive to C8-HSL. C8-HSL (0.1 nM with 8000 RFU) even appears to have a stronger effect than 3OC6-HSL (0.1 nM with 3300 RFU). For both it appears that they seem to work better at very low concentrations, but inhibit gene expression at higher concentrations. The reason for the behavior of the samples with the combination of C8- and 3OC6-HSL has yet to be found. The fact that the sample with no added HSL showed no increase in fluorescence leads us to believe that EsaI is not expressed strong enough due to  P<SUB>esaS</SUB> being quite a weak promoter. This would also be consistent with the thesis, that mRFP expression is weak due to a lack of CepR  which is also under the control of P<SUB>esaS</SUB>. </P><DIV id="pictureleft"><B>Figure 5</B> Results of the emission measurement at 476 nm of <I>E. coli</I> BL21 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for CFP.
</DIV><P>The OD<SUB>600</SUB> measurement (Fig. 6) shows that the 0.1 nM C8-HSL sample did not just simply outgrow the others. All of the samples grew in a similar manner. Only the positive control for CFP expression showed slightly weaker growth rates.</P><DIV id="pictureleft"><B>Figure 6</B> Results of the OD<SUB>600</SUB> measurements of <I>E. coli</I> BL21 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined , 0 nM HSL and the positive controls for mRFP and CFP.
</DIV><P>In <I>E. coli</I> Nissle 1917 the results (Fig. 7) were very different. At the start of the measurement the fluorescence was similar to BL21 around 1000 RFU and 2000 RFU. In the course of the measurement however the emission decreased substantially in all samples. The 0 nM sample had the most fluorescence left after 11 h. It showed 1500 RFU, which is still very low compared to the results of the BL21 samples.</P><DIV id="pictureleft"><B>Figure 7</B> Results of the emission measurement at 476 nm of <I>E. coli</I> Nissle 1917 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive control for CFP in BL21.
</DIV><P>These results for the Nissle 1917 pCERI transformants can be explained by the presence of proteases in <I>E. coli</I> Nissle 1917. It is much closer to a wild type <I>E. coli</I> than laboratory strains like BL21 which are optimized for labwork by the knockout of several proteases and nucleases. Apparently at first there was CFP produced but then the cells reacted to the presence of the foreign protein, putatively increased their protease expression, degraded the already formed CFP and kept CFP from reaching a measurable amount again. Comparing the different speed of CFP degradation in different samples while looking at the BL21 results, we see a possible correlation. The sample induced 0.1 nM C8-HSL in BL 21 rises fast and high to nearly 8000 RFU while in Nissle 1917 it starts at around 2000 RFU and is rapidly degraded until no emission is left after 6 h. The 100 nM 3OC6- and C8-HSL in BL21 rises slowly to about 2000 RFU while in Nissle 1917 the emission also slowly decreases over the course of time. So one could suspect that the stronger the expression of CFP occurs the stronger Nissle 1917’s protease response will be. </P><P>While this data could result from insufficient growth of the cells, this thought can be dismissed when we look at the OD<SUB>600</SUB> curves (Fig. 8). They all grew perfectly well and also quite uniformly. </P><DIV id="pictureleft"><B>Figure 8</B> Results of the OD<SUB>600</SUB> measurements of <I>E. coli</I> Nissle 1917 pCERI samples treated with 0.1 nM and 100 nM of C8-HSL, C6-HSL, both combined, 0 nM HSL and the positive controls for mRFP and CFP in BL21.
</DIV><H3>via <I>Chromobacterium violaceum</I> CV026</H3><P><I>E.coli </I> BL21 pCERI and <I>E.coli</I> Nissle 1917 pCERI colonies are streaked along <I>Chromobacterium violaceum </I> CV026 and incubated over night at 30° C. <I>C.violaceum</I>,streaked along transformants with functional homoserine lactone synthases produces the purple pigment violacein. As positive controls 0.5 μl of pure homoserine lactones in concentrations ranging from 10 mM to 1 nM were spotted next to <I> C. violaceum</I>. As negative controls wildtypeBL21 and Nissle 1917 as well as 0.5 μl ddH<SUB>2</SUB>O were used. In the positive controls (Fig. 9, 10) one can see, that quite high homoserine lactone concentrations are needed to induce violacein expression. Octanoyl homoserine lactone induces reporter expression at concentrations of 100 µM and more (Fig.9), while 1 mM and more 3-oxohexanoyl-homoserine lactone is needed for violacein induction in <I> C. violaceum</I> (Fig.10).As expected in the negative controls no violacein expression can be observed in our reporter strain (Fig.11).</P><DIV id="pictureleft" style="width:350px;"><B>Figure 9</B><I> C. violaceum</I> induced with different concentrations of octanoyl-homoserine lactone and grown over night at 30°C. A minimum of 100 µM C8-HSL is needed for violacein induction. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV><DIV id="pictureleft" style="width:350px;"><B>Figure 10</B><I> C. violaceum</I> induced with different concentrations of 3-oxohexanoyl-homoserine lactone and grown over night at 30°C. A minimum of 1 mM 3OC6-HSL is needed for violacein induction. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV><P>The experiment shows functional expression of homoserine lactones in <I>E.coli</I> BL21 in high amounts (Fig. 12). The violacein expression in <I>C. violaceum </I> appears to be even stronger than in the 10 mM positive control (Fig. 9,10) of both homoserine lactones. Due to the individual characterization results LINK of the homoserine lactone synthases used in pCERI, we can assume that the reporter expression in <I>C. violaceum </I> is mainly due to 3-oxohexanoyl homoserine lactone synthesis by EsaI as it is highly unlikely, that the expression of the transcript under control of P<SUB>esaRC</SUB> is much stronger than <A href="
http://parts.igem.org/Part:BBa_J230100"> BBa_J23100</A>.</P><DIV id="pictureleft" style="width:350px;"><B>Figure 11</B><I> C. violaceum</I>streaked along <I>E.coli</I> BL21 <I>E.coli</I> Nissle 1917 as well as 0.5 µl ddH<SUB>2</SUB>O spotted on the LB-plate and and grown over night at 30°C. Violacein cannot be observed due to a lack of homoserine lactones. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV><DIV id="pictureleft" style="width:350px;"><B>Figure 12</B><I>E.coli </I> BL21 pCERI and <I>E.coli</I> Nissle 1917 pCERI colonies streaked along <I>Chromobacterium violaceum </I> CV026 and incubated over night at 30° C. Homoserine lactone expression by  <I>E.coli </I> BL21 pCERI induces violacein synthesis in <I> C. violaceum </I>.   <I>E.coli</I> Nissle 1917 pCERI cannot produce enough homoserine lactones to induce violacein expression. C.v.: <I>Chromobacterium violaceum </I> CV026 
</DIV><P>
	Even though EsaI seems to be functional in <I>E.coli</I> Nissle 1917 when characterized <A href="http://parts.igem.org/Part:BBa_K1670004"> individually</A> , we cannot proof homoserine lactone synthesis of pCERI in <I> E. coli</I> Nissle 1917 with <I>C. violaceum </I> (Fig 12). No violacein expression can be observed after 16 h of incubation at 30° C. 
</P><H2>BioBrick Characterization</H2><H3>CepR P<SUB>aidA</SUB> mRFP</H3><P>The characterization of <A href="http://parts.igem.org/Part:BBa_K1670002">BBa_K1670002</A> along with <A href="http://parts.igem.org/Part:BBa_K1670003">BBa_K1670003</A>  was conducted with a fluorescence-based assay as described in the <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments">experiments section</A>.  As a positive control for mRFP expression <I>E.coli</I>BL21 <A href="http://parts.igem.org/Part:BBa_J04450">pSB3C5_J04450</A> and as a positive control for CFP expression  <I>E.coli</I> BL21<A href="http://parts.igem.org/Part:BBa_J04421">pSB3C5_J04421</A> diluted to an OD<SUB>600</SUB> of 0.02 and induced with a final concentration of 0.1 mM IPTG are used.  From all measured emission values we subtracted the measured values of non transformed BL21 or Nissle 1917 samples at the respective time to account for possible autofluorescence of the cells.  As a blank for the OD<SUB>600</SUB>  measurements we used sterile controls.</P><P> The constitutively expressed cepR is an activator that will bind to its corresponding promoter P<SUB>aidA</SUB> when a threshold C8-HSL concentration is reached, starting the expression of mRFP. </P><DIV id="pictureleft"><B>Figure 13</B> Results of the mRFP emission measurement (607 nm) of <I> E.coli </I> BL 21 J61002_BBa_J23100_BBa_ K1670002 pSB3C5_BBa_K1670003 induced with at C8-HSL concentrations from 0.01 nM to 100 nM, 0 nM and the positive control for mRFP.
</DIV><P> As seen in figure  13 the positive control delivered the strongest signal with up to 14900 RFU, followed by the 0.01 nM C8-HSL just above 10000 RFU. The other samples showed a very similar behavior with their emission peaking at around 7000 RFU.
Closer comparing the samples, induced with 0.01 nM, 100 nM, 0 nM respectively and the positive control (Fig. 14) we see that the 0.01 nM sample shows a higher fluorescence emission than the sample without any C8-HSL. However the 100 nM sample had a notably lower emission than the one with 0.01 nM. When cepR binds C8-HSL it should work as an activator for P<SUB>aidA</SUB>. Apparently this is only the case at rather low levels of C8-HSL, like 0.01 nM. At higher concentrations like 100 nM the promoter activity seems to be the same as in the sample without any C8-HSL. It looks like cepR is inhibited in its role as an activator at those C8-HSL concentrations. Considering the fluorescence of the 0 nM sample we can say that P<SUB>aidA</SUB> appears to have a high basal expression without induction by cepR or that cepR is also working as an inducer even when no C8-HSL is present. </P><DIV id="pictureleft"><B>Figure 14</B> Results of the mRFP emission measurement (607 nm) of <I> E.coli </I> BL 21 J61002_BBa_J23100_BBa_ K1670002 pSB3C5_ BBa_K1670003 induced with C8-HSL concentrations of 0.01 nM, 100 nM, 0 nM and the positive control for mRFP.  Error bars show the average deviation from the mean of the three measurements.
</DIV><P>Another possible explanation for those results could be that at 100 nM C8-HSL is somehow toxic for the cells. Looking at the measured OD<SUB>600</SUB> values (Fig. 15) that assumption can be dismissed. All samples carrying the genes for CepR and mRFP grow in a similar manner. After about 10 hours the OD<SUB>600</SUB> of the 0.01 nM sample decreases, probably due to nutrient depletion.</P><DIV id="pictureleft"><B>Figure 15</B> Results of the OD<SUB>600</SUB> measurement of <I>E.coli</I> BL 21 J61002_BBa_J23100_BBa_ K1670002 pSB3C5_ BBa_K1670003 induced with C8-HSL concentrations of 0.01 nM, 100 nM, 0 nM and the positive control for mRFP.  Error bars show the average deviation from the mean of the three measurements.
</DIV><H3>EsaR P<SUB>esaRC</SUB> CFP</H3><P><A href="http://parts.igem.org/Part:BBa_K1670005">BBa_K1670005</A> is characterized along with <A href="http://parts.igem.org/Part:BBa_K1670001"> BBa_K1670001</A> with a <A href="https://2015.igem.org/Team:Manchester-Graz/Project/Experiments"> fluorescence-based assay</A>.</P><P>The constitutively expressed EsaR is expected to act as a repressor on P<SUB>esaRC</SUB>. In the presence of 3OC6-HSL it should dissociate from the DNA and allow expression of the P<SUB>esaRC</SUB> controlled CFP.</P><P>As seen in figure 16 the sample with 0.01 nM 3OC6-HSL had the highest emission at 476 nm of nearly 7000 RFU, followed by the 0 nM sample at 5000 RFU, the positive control at 4500 RFU sample, induced with 0.1 nM 3OC6-HSL, at around 3000 RFU. The other samples showed a very low emission at or under 1500 RFU.</P><DIV id="pictureleft"><B>Figure 16</B> Results of the CFP emission measurement (476 nm) of <I> E.coli </I> BL 21 J61002_BBa_J23100_BBa_ K1670005 pSB3C5_BBa_K1670001 induced with 3OC6-HSL concentrations from 0.01 nM to 100 nM, 0 nM and the positive control for CFP.
</DIV><P>For a better overview we compare 0.01 nM, 10 nM, 0 nM and the positive control more closely (Fig. 17). Here we can observe that the fluorescence of the 0.01 nM sample is the highest. The sample without any 3OC6-HSL and the positive control are at about the same level and the sample with 10 nM showed no increase in fluorescence at all. Those results suggest that EsaR ceases its repressor function at very low 3OC6-HSL concentrations of 0.01 nM but inhibit the expression at 3OC6-HSL concentrations higher than 10 nM. The relatively high level of basal CFP expression in the 0 nM sample suggests that P<SUB>esaR</SUB> is not a very tightly regulated promoter. </P><DIV id="pictureleft"><B>Figure 17</B> Results of fluorescent emission measurement at 476 nm of <I> E.coli </I> BL 21 J61002_BBa_J23100_BBa_ K1670005 pSB3C5_BBa_K1670001 induced with 3OC6-HSL at concentrations of 0.01 nM, 10 nM, 0 nM and the positive control for CFP.  Error bars show the average deviation from the mean of the three measurements.
</DIV><P>As a reason for the low fluorescence at high 3OC6-HSL concentrations you could also expect a possible toxicity for the cells, as low cell density would also mean low fluorescence. After looking at the OD<SUB>600</SUB> measurements (figure 18) this thought can be dismissed. The cells inoculated with 10 nM even grew a little better than the ones with 0.01 nM or 0 nM but still showed a less fluorescence.</P><DIV id="pictureleft"><B>Figure 18</B> Results of the OD<SUB>600</SUB> measurement of <I>E.coli</I> BL 21 J61002_BBa_J23100_BBa_ K1670005 pSB3C5_BBa_K1670001 induced with 3OC6-HSL concentrations of 0.01 nM, 10 nM, 0 nM and the positive control for CFP.  Error bars show the average deviation from the mean of the three measurements.
</DIV><H3>Homoserine Lactones Synthases characterization</H3><P><A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A>  and <A href="http://parts.igem.org/Part:BBa_K1670000"> BBa_K1670000</A>  were cloned into J61002_BBa_J23100 for constitutive protein expression and transformed into <I>E. coli </I> BL21 and <I>E.coli</I> Nissle 1917. The transformed colonies are streaked along <I>Chromobacterium violaceum </I> CV026 and incubated over night at 30° C. <I>C.violaceum</I>,,streaked along transformants with functional homoserine lactone synthases produces the purple pigment violacein. As positive controls 0.5 µl of pure homoserine lactones in concentrations ranging from 10 mM to 1 nM were spotted next to <I> C. violaceum</I> (Fig. 9,10). As negative controls wildtype BL21 and Nissle 1917 as well as 0.5 µl ddH<SUB>2</SUB>O were used (Fig. 11). 
<A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> seems to be functional in both <I>E.coli</I> strains (Fig.19). As expected the expression strain <I> E.coli </I> BL21 shows a better result, due to knocked out protease activity. Still <I> E.coli </I> Nissle 1917 produces enough BBa_K1670004 to induce weak violacein expression in <I>C. violaceum </I>. This confirm functionality of <A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A>.
</P><DIV id="pictureleft" style="width:350px;"><B>Figure 19</B><A href="http://parts.igem.org/Part:BBa_K1670004"> BBa_K1670004</A> is constitutively expressed in <I>E.coli </I> BL 21 and <I>E.coli </I> Nissle 1917. BBa_K1670004 synthesizes 3-oxohexanoyl homoserine lactone that induces violacein expression in <I>C.violaceum </I>. C.v.: <I>Chromobacterium violaceum </I> CV026
</DIV><DIV id="pictureleft" style="width:350px;"><B>Figure 20</B><A href="http://parts.igem.org/Part:BBa_K16700040"> BBa_K16700040</A> is constitutively expressed in <I>E.coli </I> BL 21 and <I>E.coli </I> Nissle 1917. BBa_K1670000 cannot synthesize octanoyl-homoserine lactone in sufficient concentrations to induces violacein expression in <I>C.violaceum </I>. C.v.: <I>Chromobacterium violaceum </I> CV026
</DIV><P><A href="http://parts.igem.org/Part:BBa_K1670000"> BBa_K1670000</A> expression in both strains is putatively not functional in both used <I>E.coli</I> strains (Fig. 20) or the produced C8-HSL concentration is too low to induce violacein expression in <I>C.violaceum </I> as no violacein synthesis in <I> C. violaceum</I> can be observed.</P><H2>Enzyme Expression &amp; Analysis</H2><P>To characterise aromatic amino acid decarboxylase <A href="http://parts.igem.org/Part:BBa_K1670006l">BBa_K1670006</A> we performed biotransformations to analyse enzyme function via HPLC.</P><P>Our HPLC analysis showed little evidence that the biotransformation had been successful, as we found no significant peak for the presence of phenylethylamine (PEA), which should have been converted from phenylalanine. To confirm the presence or absence of the AADC, we ran an SDS-PAGE and subsequent Western Blot which showed that the AADC had not been expressed. After reviewing the literature [1], we decided that a higher IPTG concentration (1mM rather than 0.4mM) should be used for induction.</P><P>After repeating the cloning and induction of the AADC (with the aforementioned IPTG concentration), we performed SDS-PAGE (Fig. 21) and found a band at 71.7 kDa, suggesting to us that the AADC had been expressed.</P><DIV id="pictureleft"><B>Figure 21</B>: SDS-PAGE gel of Nissle 1917, showing band at 71.7 kDa, suggesting presence of AADC.

</DIV><P>To characterise transaminase (<A href="http://parts.igem.org/Part:BBa_K1670007">BBa_K1670006</A>) we used the same protocol as for AADC only with an IPTG concentration of 0.2 mM. This was apparently successful in inducing expression in both BL21 and Nissle, as shown by a band of 100 kDa (Fig. 22)</P><DIV id="pictureleft"><B>Figure 22</B> SDS-PAGE gel of BL21 and Nissle 1917, showing band at 100 kDa, suggesting presence of CvATA.

</DIV><P>As it could be seen from the SDS-PAGE results, we were able to see bands at the size of the CvATA and AADC. This suggest that the enzymes were expressed in both BL21 and Nissle respectively, however further characterisation such as western blots are required to confirm the bands as the aforementioned enzymes. Prior to the expression, we were able to remove the PstI restriction site in the AADC gene via PCR mutagenesis such that the gene is not spliced before cloning into the submission vector.</P><P>Further work includes biotransformation of the expressed enzyme in Nissle 1917 and BL21 wherein the substrate is provided in order to view the activity of the enzymes and confirm production of the desired substance via HPLC.</P><DIV style="background-color:#373737; width: 765px; height:50px; color: white; padding:10px;">
		[1.] Liu, F., Xu, W., Du, L., Wang, D., Zhu, Y., Geng, Z., Zhang, M. and Xu, W. (2014). Heterologous Expression and Characterization of Tyrosine Decarboxylase from Enterococcus faecalis R612Z1 and Enterococcus faecium R615Z1. Journal of Food Protection, [online] 77(4), pp.592-598. Available at: http://www.ncbi.nlm.nih.gov/pubmed/24680070 [Accessed 29 Jul. 2015].

	</DIV></DIV></DIV><DIV id="footer"><DIV id="footerlinks"><DIV class="footerheadline">
			Links
		</DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz" style="text-decoration:none;">Home</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Project" style="text-decoration:none">Project</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Team" style="text-decoration:none">Team</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Practices" style="text-decoration:none">Human Practices</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Modeling" style="text-decoration:none">Modeling</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Parts" style="text-decoration:none">Parts</A></DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Notebook" style="text-decoration:none">Notebook</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Attributions" style="text-decoration:none">Attributions</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Safety" style="text-decoration:none">Safety</A><A class="footerlinks" href="https://2015.igem.org" style="text-decoration:none">iGEM</A></DIV></DIV><DIV id="footercontact"><DIV class="footerheadline">
			Contact us
		</DIV>
		iGEM Subteam Graz
		TU Graz, Institute of Molecular Biotechnology
		Petersgasse 14
		8010 Graz
		
		iGEM Subteam Manchester 
		University of Manchester
		Manchester Institute of Biotechnology
		131 Princess Street
		Manchester M17DN
	</DIV><DIV id="footerconnect"><DIV class="footerheadline">
			Connect
		</DIV></DIV><DIV id="footerboringstuff"><DIV class="footerheadline">
			Boring Stuff
		</DIV><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Aboutus" style="text-decoration:none;">About us</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Legal" style="text-decoration:none">Legal Information</A></DIV><DIV id="footercopyright">© iGEM Manchester-Graz 
	</DIV></DIV></DIV></DIV></DIV></BODY></HTML>