<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester-Graz_Project_Vectordesign skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester-Graz/Project/Vectordesign</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Manchester Header</TITLE></P><P><TITLE>iGEM Manchester-Graz - Vector Design</TITLE></P><DIV id="fixednavi"><DIV id="twitterboxsmall"><A class="twitter-timeline" href="https://twitter.com/iGEMMancGraz" data-widget-id="619465263411499008" height="900" width="180">Tweets by @iGEMMancGraz</A></DIV><DIV id="inhalte"><H1>Vector Design</H1><DIV class="inhalte_graz"><DIV id="pictureright"><B>Figure 1</B> Schematic representation of our quorum sensing based regulatory system.</DIV><P>Our vector is based on two quorum sensing (QS) systems. The EsaR/I system belongs to the plant pathogen <I>Pantoea stewartii</I>, formerly known as <I>Erwinia stewartii</I>, the causative agent of Stewart’s Wilt [1]. Contrary to common QS-systems EsaR/I uses a repressor based rather than an activator-based system. EsaR binds to its corresponding binding sites on the P<SUB>esaRC</SUB> promoter and represses the expression of the genes under the promoter’s control [2][3]. When a certain concentration of 3-oxohexanoyl-homoserinelactone (3OC6-HSL) that is produced by the EsaI-synthase, is reached, it leads to an allosteric conformation change in EsaR’s structure that inhibits its repressor function. We use an engineered variant of EsaR (D91G) that showed higher sensitivity towards 3OC6-HSL [3]. When positioned in the -60 region of the P<SUB>esaS</SUB>-promoter EsaR can work as an activator too, by facilitating RNA-polymerase recruitment [2]. </P><P>The second QS-System we use, CepR/I, belongs to the opportunistic pathogen <I>Burkholderia cenocepacia</I>. Similar to the LuxR/I system, CepR acts as an activator of its corresponding promoter, P<SUB>aidA</SUB>, when a certain level of octanoyl-homoserinelactone (C8-HSL) is reached [4]. C8-HSL is produced by CepI. CepR also binds 3OC6-HSL, however will not work as an activator, as the additional two carbon-atoms are mandatory, for CepR’s RNA-Polymerase-recruiting ability [4]. This way CepR works as a competitive binding site for 3OC6-HSL, that putatively allows us to reach higher cell densities and thus higher 3OC6-HSL concentration before the EsaR/I expression system gets activated. </P><P>Our vector was designed in a way that EsaR, EsaI and CepR are constitutively expressed by the P<SUB>esaS</SUB>-promoter. As long as the 3OC6-HSL concentration is low enough, EsaR will additionally increase its own transcription, creating a positive feedback loop. </P><DIV id="pictureleft" style="height:160px; margin-right:10px;"><B>Figure 2</B> Homoserinelactone synthesis by EsaI and CepI.</DIV><P>When the 3OC6-HSL threshold is reached, transcription of the PesaRC initiates, while the P<SUB>esaS</SUB>-feedback loop is turned off. The activation of the promoter is shown and measured on the expression of cyan fluorescent protein (CFP). Additionally to the reporter gene CepI also gets expressed, resulting in the time-shifted activation of our second QS-system. When the C8-HSL threshold is reached, CepR can work as an activator of the P<SUB>aidA</SUB> promoter that transcribes monomeric red fluorescent protein (mRFP) as a second reporter gene. 
To avoid any leaky read through of transcription terminators, the constitutively expressed transcripts of the regulatory proteins of the two QS-systems as well as the beta-lactamase resistance marker, were positioned in the opposite direction of the auto-induced P<SUB>aidA</SUB> – and P<SUB>esaRC</SUB> –promoter (Figure 3). Additionally P<SUB>aidA</SUB> was placed upfront of P<SUB>esaRC</SUB>. All reporter genes can easily be replaced by any other genes by standard cloning techniques. </P><P>We wanted to assemble two variants of pCERI that only slightly differ in the positioning of the regulatory proteins EsaR and CepR on the transcript of P<SUB>esaS</SUB>. The first variant was assembled according to Figure 3, while in an alternative version (v2) the genes encoding esaR and cepR were switched. Usually, genes further downstream on the transcript get translated less efficiently. Thus, the switch in the positioning of the regulatory proteins on the transcript might influence the expression behavior of our regulatory system </P><DIV style="margin-left:115px; border:1px solid black; font-size:11px; font-family:sans-serif; height:auto; width:541px; background-color:white;"><B>Figure 3</B> Simplified map of our pCERI Vector.
</DIV><DIV style="background-color:#373737; width: 745px; height:200px; color: white; padding:10px;">
		[1] Iowa State University: Plant and Insect Diagnostic Centre (2010). [online] Available at:  http://www.ipm.iastate.edu/ipm/info/plant-diseases/stewarts-wilt [Accessed: 31 Aug. 2015] 
[2] Shong, J. and Collins, C. (2013) Engineering the esaR promoter for tunable quorum sensing-dependent gene expression. American Chemical Society. 2 (10), pp. 568–575. 
[3] Shong, J., Huang, Y-M., Bystroff, C. and Collins, C. (2013) Directed evolution of the quorum-sensing regulator EsaR for increased signal sensitivity. American Chemical Society. 8 (4), pp 789–795. 
[4] Weingart, C., White, C., Liu, S., Chai, Y., Cho, H., Tsai, C., Wei, Y., Delay, N., Gronquist, M., Eberhard, A. and Winans, S. (2005) Direct binding of the quorum sensing regulator CepR of <I>Burkholderia cenocepacia</I> to two target promoters in vitro. Molecular Microbiology. 7 (2), pp 452-467. </DIV></DIV></DIV></DIV><DIV id="footer"><DIV id="footerlinks"><DIV class="footerheadline">
			Links
		</DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz" style="text-decoration:none;">Home</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Project" style="text-decoration:none">Project</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Team" style="text-decoration:none">Team</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Practices" style="text-decoration:none">Human Practices</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Modeling" style="text-decoration:none">Modeling</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Parts" style="text-decoration:none">Parts</A></DIV><DIV class="footerlinksbox"><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Notebook" style="text-decoration:none">Notebook</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Attributions" style="text-decoration:none">Attributions</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Safety" style="text-decoration:none">Safety</A><A class="footerlinks" href="https://2015.igem.org" style="text-decoration:none">iGEM</A></DIV></DIV><DIV id="footercontact"><DIV class="footerheadline">
			Contact us
		</DIV>
		iGEM Subteam Graz
		TU Graz, Institute of Molecular Biotechnology
		Petersgasse 14
		8010 Graz
		
		iGEM Subteam Manchester 
		University of Manchester
		Manchester Institute of Biotechnology
		131 Princess Street
		Manchester M17DN
	</DIV><DIV id="footerconnect"><DIV class="footerheadline">
			Connect
		</DIV></DIV><DIV id="footerboringstuff"><DIV class="footerheadline">
			Boring Stuff
		</DIV><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Aboutus" style="text-decoration:none;">About us</A><A class="footerlinks" href="https://2015.igem.org/Team:Manchester-Graz/Legal" style="text-decoration:none">Legal Information</A></DIV><DIV id="footercopyright">© iGEM Manchester-Graz 
	</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>