<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Baltimore_BioCrew_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Baltimore BioCrew/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="jumbotron row"><H1>Saving the Baltimore Inner Harbor</H1><H3 class="text-xs-center">Is E. coli the answer?</H3></DIV><NAV class="navbar navbar-default"><DIV class="container2"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="#">Baltimore Bio Crew</A></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse-1"><UL class="nav navbar-nav"><LI id="nav-home"><A href="/Team:Baltimore_BioCrew">Home</A></LI><LI id="nav-project" class="dropdown btn-group"><A id="nav-project" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI><A href="/Team:Baltimore_BioCrew/Project">Description</A></LI><LI><A href="/Team:Baltimore_BioCrew/Design">Design</A></LI></UL><LI id="nav-composite_parts"><A href="/Team:Baltimore_BioCrew/Composite_Part">Composite Parts</A></LI><LI id="nav-notebook"><A href="/Team:Baltimore_BioCrew/Notebook">Notebook</A></LI><LI id="nav-experiments"><A href="/Team:Baltimore_BioCrew/Experiments">Experiments</A></LI><LI id="nav-safety"><A href="/Team:Baltimore_BioCrew/Safety">Safety</A></LI><LI id="nav-attributions"><A href="/Team:Baltimore_BioCrew/Attributions">Attributions</A></LI><LI id="nav-human_practices" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI><A href="/Team:Baltimore_BioCrew/Silver">Silver</A></LI><LI><A href="/Team:Baltimore_BioCrew/Gold">Gold</A></LI><LI><A href="/Team:Baltimore_BioCrew/Integrated_Practices">Integrated Practices</A></LI><LI id="nav-team"><A href="/Team:Baltimore_BioCrew/Team">Team<SPAN class="sr-only">(current)</SPAN></A></LI></DIV></DIV></NAV></DIV><DIV class="container"><H3>8/6/16</H3><P>The vector plasmids pSB1A3 and pSB1C3 were miniprepped, and a verification gel was run. The miniprep of the pSB1C3 plasmid was successful, while the one for pSB1A3 was not. The genes to be inserted were also designed using the BioBrick standard assembly and ordered from IDT. </P><P>[gel not accounted for]</P><H3>8/11/16</H3><P>The pSB1A3 plasmids were miniprepped again, and a gel was run on them using 5uL of each sample. This time the miniprep was successful.</P><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Lanes (left to right with lanes at the top)</CAPTION><TR><TD>
				Lane 1
			</TD><TD>
				Lane 2
			</TD><TD>
				Lane 3
			</TD><TD>
				Lane 4
			</TD></TR><TR><TD>
				ladder
			</TD><TD>
				tube 1
			</TD><TD>
				tube 2
			</TD><TD>
				tube 3
			</TD></TR></TBODY></TABLE><H3>8/13/16</H3><P>The forward and reverse primers were re-suspended:</P><UL><LI>The tubes with the dried primers in them were spun down</LI><LI>786 uL of dI water was added to to the forward primer </LI><LI>860 uL of dI water was added to the reverse primer</LI><LI>The primers were vortexed for 3-4 seconds</LI><LI>Both tubes were centrifuged for 1 min at full speed</LI></UL><P>and both vectors were PCR amplified using the following amounts:</P><UL><LI>67.6 uL water</LI><LI>10 uL 5x Taq buffer</LI><LI>7.5 uL DNTPs (2.3 uM)</LI><LI>6.7 uL forward primer (20 uM)</LI><LI>6.7 uL reverse primer (20 uM)</LI><LI>5 uL Taq enzyme</LI><LI>1 uL DNA (10 ng/uL)</LI></UL><H3>8/18/16</H3><P>Both vectors were PCR purified and a gel was run afterwards. Three samples of each vector were run on the gel: the vector plasmids after miniprep (1); the vectors after PCR (2); and the vectors after being PCR purified (3) </P><P>The six samples were prepared as follows</P><UL><LI>Samples A3 (1) and C3 (1):</LI><UL><LI>1 uL DNA</LI><LI>2 uL loading buffer</LI><LI>7 uL dIH2O</LI></UL><LI>Samples A3 (2) and C3 (2):</LI><UL><LI>2 uL DNA</LI><LI>2 uL loading buffer</LI><LI>6 uL dIH2O</LI></UL><LI>Samples A3 (3) and C3 (3):</LI><UL><LI>5 uL DNA</LI><LI>2 uL loading buffer</LI><LI>3 uL dIH2O</LI></UL></UL><P>10 uL of each sample was loaded into the gel</P><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Lanes (left to right with lanes at top)</CAPTION><TR><TD><P>Lane 1</P></TD><TD><P>Lane 2</P></TD><TD><P>Lane 3</P></TD><TD><P>Lane 4</P></TD><TD><P>Lane 5</P></TD><TD><P>Lane 6</P></TD><TD><P>Lane 7</P></TD></TR><TR><TD><P>ladder </P></TD><TD><P>A3 (1)</P></TD><TD><P>C3 (1)</P></TD><TD><P>A3 (2)</P></TD><TD><P>C3 (2)</P></TD><TD><P>A3 (3)</P></TD><TD><P>C3 (3)</P></TD></TR></TBODY></TABLE><P>[gel not accounted for]</P><P>The results indicated that the PCR was unsuccessful.</P><P>Meanwhile, an amplification PCR was set up for the Chlorogenate Esterase gene. The gene was resuspended in 100 uL of dIH2O for a final concentration of 10 ng/uL. The solution was incubated at 50ºC for 20 min and then centrifuged. The forward and reverse primers were also resuspended. The forward primer was resuspended in 1500 uL of dIH2O, and the reverse primer was resuspended in 1600 uL of dIH2O. The gene was PCR amplified using the following amounts:</P><UL><LI>50 uL DNA (10 ng/uL)</LI><LI>20 uL 5x Taq buffer</LI><LI>12 uL dIH2O</LI><LI>7.5 uL dNTPs (2.3 uM)</LI><LI>5 uL forward primer (20 uM)</LI><LI>5 uL reverse primer (20 uM)</LI><LI>0.5 uL Taq enzyme</LI></UL><H3>8/19/16</H3><P>The vectors were PCR amplified a second time. This time some changes were made to the basic PCR settings. The annealing temperatures (cycle 2) were changed as follows: </P><P>Both of the vectors were divided into three samples: Sample 1 and a positive control (with VF2 and VR primers) were put at 54ºC; sample 2 was put at 56ºC; and sample 3 was put at 58ºC. All samples were run for 20 sec.</P><P>A gel was run for the results of the PCR:</P><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Key</CAPTION><TR><TD><P>Lane </P></TD><TD><P>Sample </P></TD><TD><P>PCR Temperature (℃)</P></TD><TD><P>Amount Loaded (uL)</P></TD></TR><TR><TD><P>1</P></TD><TD><P>ladder</P></TD><TD><P>X</P></TD><TD><P>5</P></TD></TR><TR><TD><P>2</P></TD><TD><P>pSB1A3</P></TD><TD><P>54</P></TD><TD><P>1</P></TD></TR><TR><TD><P>3</P></TD><TD><P>pSB1A3</P></TD><TD><P>56</P></TD><TD><P>5</P></TD></TR><TR><TD><P>4</P></TD><TD><P>pSB1A3</P></TD><TD><P>58</P></TD><TD><P>5</P></TD></TR><TR><TD><P>5</P></TD><TD><P>pSB1A3 (Control)</P></TD><TD><P>54</P></TD><TD><P>5</P></TD></TR><TR><TD><P>6</P></TD><TD><P>X</P></TD><TD><P>X</P></TD><TD><P>X</P></TD></TR><TR><TD><P>7</P></TD><TD><P>pSB1C3</P></TD><TD><P>54</P></TD><TD><P>1</P></TD></TR><TR><TD><P>8</P></TD><TD><P>pSB1C3</P></TD><TD><P>56</P></TD><TD><P>5</P></TD></TR><TR><TD><P>9</P></TD><TD><P>pSB1C3</P></TD><TD><P>58</P></TD><TD><P>5</P></TD></TR><TR><TD><P>10</P></TD><TD><P>pSB1C3 (Control)</P></TD><TD><P>54</P></TD><TD><P>5</P></TD></TR><TR><TD><P>11</P></TD><TD><P>X</P></TD><TD><P>X</P></TD><TD><P>X</P></TD></TR><TR><TD><P>12</P></TD><TD><P>Chlorogenate Esterase after resuspension</P></TD><TD><P>X</P></TD><TD><P>1</P></TD></TR><TR><TD><P>13</P></TD><TD><P>Chlorogenate Esterase amplified product</P></TD><TD><P>X</P></TD><TD><P>5</P></TD></TR></TBODY></TABLE><H3>8/20/16</H3><P>Both vectors were PCR purified, and restriction digest was done on both the vectors and the Chlologenate Esterase gene. The amounts of reactants used are as follows.</P><H4>For each vector:</H4><UL><LI>33 uL dIH2O</LI><LI>10 uL DNA</LI><LI>5 uL CutSmart</LI><LI>1 uL EcoR1</LI><LI>1 uL Pst1</LI></UL><H4>For the Esterase gene:</H4><UL><LI>28 uL dIH2O</LI><LI>15 uL DNA</LI><LI>5 uL CutSmart</LI><LI>1 uL EcoR1</LI><LI>1 uL Pst1</LI></UL><P>The Chlorogenate Esterase construct was ligated into each of the vectors, and the new A3+ChEst and C3+ChEst plasmids were transformed into E.coli. There were six transformations made, each using 50 uL of competent cells mixed with the specified amount of the specified vector, with the addition of a positive control and a negative control:</P><OL><LI>3 uL A3</LI><LI>5 uL A3</LI><LI>3 uL C3</LI><LI>5 uL C3</LI><LI>+ control</LI><LI>- control</LI></OL><P>The cells were plated and allowed to grow overnight.</P><H3>8/25/16</H3><P>The Lipase gene fragment arrived by mail and was resuspended in 100 uL dIH2O for a final concentration of 10 ng/uL. Restriction digest was done on the Lipase gene before it was ligated with each vector and transformed into E.coli. The cells were plated and allowed to grow overnight.</P><H3>8/27/16</H3><P>All of the plates with the transformed cells were screened with colony PCR.</P><P>There were too many samples to be run on one gel, so two gels were used (a large one and a smaller one).</P><P>The results of the colony PCR were run on a gel. The samples were prepared as follows:</P><UL><LI>2 uL DNA</LI><LI>1 uL loading dye (7.5 uL of extra dye was accidentally added to tubes CL1 - CL2, and 7 uL extra dye was added to tubes CL3 - CL5)</LI><LI>7 uL dIH2O </LI><LI>The tubes were mixed and spun down</LI><LI>10 uL of each sample and 5 uL of each ladder were loaded</LI></UL><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Key for 1st Gel</CAPTION><TR><TD><P>Lane</P></TD><TD><P>Sample</P></TD></TR><TR><TD><P>1</P></TD><TD><P>ladder</P></TD></TR><TR><TD><P>2</P></TD><TD><P>ACE 1</P></TD></TR><TR><TD><P>3</P></TD><TD><P>ACE 2</P></TD></TR><TR><TD><P>4</P></TD><TD><P>ACE 3</P></TD></TR><TR><TD><P>5</P></TD><TD><P>ACE 4</P></TD></TR><TR><TD><P>6</P></TD><TD><P>ACE 5</P></TD></TR><TR><TD><P>7</P></TD><TD><P>CCE 1</P></TD></TR><TR><TD><P>8</P></TD><TD><P>CCE 2</P></TD></TR><TR><TD><P>9</P></TD><TD><P>CCE 3</P></TD></TR><TR><TD><P>10</P></TD><TD><P>CCE 4</P></TD></TR><TR><TD><P>11</P></TD><TD><P>CCE 5</P></TD></TR><TR><TD><P>12</P></TD><TD><P>AL 1</P></TD></TR><TR><TD><P>13</P></TD><TD><P>AL 2</P></TD></TR><TR><TD><P>14</P></TD><TD><P>AL 3</P></TD></TR><TR><TD><P>15</P></TD><TD><P>AL 4</P></TD></TR></TBODY></TABLE><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Key for 2nd Gel</CAPTION><TR><TD><P>Lane</P></TD><TD><P>Sample</P></TD></TR><TR><TD><P>1</P></TD><TD><P>ladder</P></TD></TR><TR><TD><P>2</P></TD><TD><P>AL 5</P></TD></TR><TR><TD><P>3</P></TD><TD><P>CL 1</P></TD></TR><TR><TD><P>4</P></TD><TD><P>CL 2</P></TD></TR><TR><TD><P>5</P></TD><TD><P>CL 3</P></TD></TR><TR><TD><P>6</P></TD><TD><P>CL 4</P></TD></TR><TR><TD><P>7</P></TD><TD><P>CL 5</P></TD></TR></TBODY></TABLE><P>The gels indicated that only the AL 1 and CL 1 colonies contained the gene. The Chlorogenate Esterase gene was not present in any of the screened colonies, so the PCR had to be partially re-done.</P><H3>9/01/16</H3><P>The colony PCR for the Chlorogenate Esterase transformations was done over.</P><P><STRONG>PCR Settings:</STRONG></P><P>Cycle 1 (repeat 1 time)</P><UL><LI>98ºC - 30 sec</LI></UL><P>Cycle 2 (repeat 30 times)</P><UL><LI>98ºC - 10 sec</LI><LI>55ºC - 30 sec</LI><LI>72ºC - 40 sec</LI></UL><P>Cycle 3 (repeat 1 time)</P><UL><LI>72ºC -  5 min</LI><LI>4ºC - infinity</LI></UL><P>This gel also indicated that the cells did not have the Chlorogenate Esterase gene.</P><H3>9/3/16</H3><P>A restriction digest was done on the Chlorogenate Esterase gene fragment directly from the IDT tube. A gel was run on this digested Chlorogenate Esterase gene, as well as the two digested vectors. The samples were prepared as follows:</P><UL><LI>7 uL vector/insert</LI><LI>2 uL loading dye</LI><LI>1 uL dIH2O</LI></UL><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Key</CAPTION><TR><TD><P>Lane </P></TD><TD><P>Sample </P></TD><TD><P>Amount Loaded (uL)</P></TD></TR><TR><TD><P>1</P></TD><TD><P>ladder</P></TD><TD><P>3 uL</P></TD></TR><TR><TD><P>2</P></TD><TD><P>pSB1A3</P></TD><TD><P>10 uL</P></TD></TR><TR><TD><P>3</P></TD><TD><P>pSB1C3</P></TD><TD><P>10 uL</P></TD></TR><TR><TD><P>4</P></TD><TD><P>ChEst gene</P></TD><TD><P>10 uL</P></TD></TR></TBODY></TABLE><P>The gel results indicated that the restriction digests for both the insert and the vectors were successful. </P><P>The Chlorogenate Esterase gene was ligated with both vectors. The following ingredients were mixed in a clean microcentrifuge tube for each vector:</P><UL><LI>9 uL dIH2O</LI><LI>2 uL vector</LI><LI>6 uL insert</LI><LI>2 uL Ligase Buffer</LI><LI>1 uL T4 Ligase enzyme</LI></UL><P>The ligation was incubated in a PCR machine.</P><P><STRONG>PCR Settings:</STRONG></P><P>Cycle 1</P><UL><LI>20ºC - 1h</LI><LI>55ºC - 10 min</LI><LI>4ºC - infinity</LI></UL><H3>9/08/16</H3><P>The new Chlorogenate Esterase gene in both separate vectors was transformed into E.coli.</P><P>After realizing that our samples used in the verification gel for the first colony PCR were too concentrated to show results we decided to run the gel again, this time with the samples diluted. Forty samples were used: samples of every transformation from that first colony PCR, each one diluted to both 1/10 and 1/100. This gel also indicated that the PCR was not successful. </P><H3>9/15/16</H3><P>
The colony PCR on the first colonies was done again for each combination of vector and insert. This time the samples were diluted before the PCR was run. The 64 microcentrifuge tubes were labeled as follows (A = pSB1A3 vector; C = pSB1C3 vector; CE = Chlorogenate Esterase; L = Lipase):</P><UL><LI>ACE 1/100 (1-8)</LI><LI>ACE 1/1000 (1-8)</LI><LI>CCE 1/100 (1-8)</LI><LI>CCE 1/1000 (1-8)</LI><LI>AL 1/100 (1-8)</LI><LI>AL 1/1000 (1-8)</LI><LI>CL 1/100 (1-8)</LI><LI>CL1/1000 (1-8)</LI></UL><P>
The dilutions were prepared: 99 uL dIH2O was added to each tube labeled with 1/100, and 999 uL dIH2O was added to each tube labeled 1/1000. The 64 PCR wells were prepared with 20 uL of PCR master mix each. A toothpick was used to pick a colony from a plate, to streak it on a section of a clean plate (Ampicillin plates for the A3 vector cells and Chloramphenicol plates for the C3 vector cells), and to swirl the remaining cells around inside the corresponding microcentrifuge tube marked 1/100 until the water turned cloudy. This was repeated for all PCR samples. Afterwards, 1 uL from each 1/100 microcentrifuge tube was taken and added to its matching 1/1000 microcentrifuge tube. 5 uL from each microcentrifuge tube was added into the PCR wells. The PCR well tray was put into the PCR machine.</P><P><STRONG>PCR Settings</STRONG></P>

Cycle 1 (Repeat 1 time)
<UL><LI>98ºC - 30 sec</LI></UL>

Cycle 2 (Repeat 30 times)
<UL><LI>98ºC - 10 sec</LI><LI>55ºC - 30 sec</LI><LI>72ºC - 40 sec</LI></UL>

Cycle 3 (repeat 1 time)
<UL><LI>72ºC - 5 min</LI><LI>4ºC - infinity</LI></UL><P>
A gel was run for the colony PCR. This gel indicated 13 possibly successful transformations: ACE 1, ACE 2, ACE 4, ACE 5, ACE 7, CCE 1, CCE 2, AL 5, AL 6, AL 7, AL  8, CL 1, and CL 7.</P><H3>9/22/16</H3><P>All successful transformations were plasmid purified so that the DNA could be sent off to be sequenced. </P><H3>10/8/16</H3><P>
The genetic sequencing revealed that none of the cells contained our constructs. We decided to transform all of our remaining ligation after adding Dpn1. Before the transformation, 0.5 uL of Dpn1 was added to each ligation. The tubes were mixed well and put in a 34ºC incubator. 4 transformations total were done using the Chlorogenate Esterase ligations done on 9/3/16  and the Lipase ligations from 8/25/16. 
</P><H3>10/14/16</H3><P>
Only the C3+lipase, the C3+ChEst and the A3+lipase colonies had growth. Since the C3+lipase transformation produced many colonies, 13 of them were screened with colony PCR. The 2 C3+ChEst colonies and the 3 A3+lipase colonies were screened as well.
</P><P><STRONG>PCR Settings</STRONG></P>

Cycle 1 (Repeat 1 time)
<UL><LI>98ºC - 3 min</LI></UL>

Cycle 2 (Repeat 35 times)
<UL><LI>98ºC - 30 sec</LI><LI>54ºC - 30 sec</LI><LI>72ºC - 2 min</LI></UL>

Cycle 3 (repeat 1 time)
<UL><LI>4ºC - infinity</LI></UL><H3>10/15/16</H3><P>
A gel was run of the last PCR. 2 uL of loading dye was added to each sample and 10 uL of each sample was loaded onto the gel. 5 uL of the ladder was loaded. 
</P><P>
(A = pSB1A3 vector; C = pSB1C3 vector; CE = Chlorogenate Esterase; L = Lipase).
</P><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Key</CAPTION><TR><TD><P>Lane(s)</P></TD><TD><P>Sample(s)</P></TD></TR><TR><TD><P>1 - 13</P></TD><TD><P>CL 1 - 13</P></TD></TR><TR><TD><P>14 - 15</P></TD><TD><P>CCE 1 - 2</P></TD></TR><TR><TD><P>16 - 18</P></TD><TD><P>AL 1 - 3</P></TD></TR><TR><TD><P>19</P></TD><TD><P>ladder</P></TD></TR></TBODY></TABLE><P>The gel indicated that CL 7 contained the right gene. The cells from this colony were sent off for sequencing.</P><H3>10/18/16</H3><P>12 more C3+lipase and 12 more C3+ChEst colonies were screened with colony PCR.</P><H3>10/19/16</H3><P>A gel was run of the 10/18 PCR. 2 uL of loading dye was added to each ~20 uL sample. 5 uL of the ladder and ~10 uL of each sample were loaded into the gel.</P><TABLE class="table table-bordered"><TBODY><CAPTION>Gel Key</CAPTION><TR><TD><P>Lane(s)</P></TD><TD><P>Sample</P></TD></TR><TR><TD><P>1</P></TD><TD><P>ladder</P></TD></TR><TR><TD><P>2 - 13</P></TD><TD><P>CL 1 - 12</P></TD></TR><TR><TD><P>14 - 24</P></TD><TD><P>CCE 1 - 11</P></TD></TR></TBODY></TABLE><P>The gel indicated that the following samples had the right gene:</P><UL><LI>CL 8</LI><LI>CL 11</LI><LI>CCE 7</LI></UL><P>Samples of DNA from these colonies were sent off for gene sequencing.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>