<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="container"><NAV class="navbar navbar-default navbar-fixed-top" style="margin-top:16px;font-size:14px"><DIV class="container-fluid"><DIV class="containernav-bar"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Team:Pasteur_Paris"><SPAN style="color: #383838;">Plasti</SPAN><SPAN style="color: #00a4ca;">Cure</SPAN></A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI class="dropdown"><A href="https://2015.igem.org/Team:Pasteur_Paris/Description">Project</A></LI><LI class="dropdown"><A data-toggle="dropdown" href="#">Results</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Measurement">Measurement</A></LI></UL><LI class="dropdown"><A data-toggle="dropdown" href="#">Tools</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Experiments">Methods</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Pasteur_Paris/Interlab_study">InterLab Study</A></LI><LI class="dropdown"><A data-toggle="dropdown" href="https://2015.igem.org/Team:Pasteur_Paris/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Practices">Human Practice</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Collaborations">Collaborations </A></LI><LI class="dropdown"><A data-toggle="dropdown" href="#">Team</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/About_us">About us</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Team">Members</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Attributions">Attribution</A></LI><LI><A href="https://2015.igem.org/Team:Pasteur_Paris/Acknowledgements">Acknowledgements</A></LI><LI class="dropdown"><A href="https://2015.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></DIV></DIV></DIV></NAV><DIV id="mainContainer"><DIV id="container"><DIV class="carregris"><P style="text-indent:3em;" align="justify"> Pollution of our ecosystem by plastic waste is undoubtedly one of the major issues of the century. The increasing production and use of plastic materials combined with the lack of interesting financial returns have just worsened the situation. Recent drop in oil prices made it more advantageous to use virgin plastic instead of recycled plastic thus jeopardizing countries' efforts to protect the environment.<SUP>1</SUP>
What if we could make  post-consumer plastic the <B>next black gold?</B>
To do so we want to combine into one system an optimised polyethylene glycol (PET) degradation pathway with a biosynthetic pathway: we aim to transform plastic waste into erythromycin A.
</P><P style="text-indent:3em;" align="justify">To express the degradation pathway in <EM>Escherichia coli</EM> we decided to assemble the different genes into one operon. One question remained. What is the gene combination that will give the strain the most efficient metabolism? The location of a gene and the gene combination are critical points for the efficiency of the system. Our system is composed of <B>15 genes</B>, 13 of which are recombined, yielding <B>6 billion combinations</B>.
We assumed that the genes coding for enzymes that are close in the metabolic pathway shouldn't be too much separated. Therefore we identified <B>4 clusters</B>. To automate the mixing of the genes we used site-specific recombination by means of the Cre/lox system, and we went a step further by identifying <B>5 self-recombining lox sequences</B> (Figure 1).</P><CENTER><P><B>Fig.1</B> - Spacer interaction network. In red are the the sequence we used.<SUP><B>2</B></SUP></P></CENTER><P style="text-indent:3em;" align="justify"> By assigning 1 specific lox sequence to one cluster and placing the last self-recombining one between each cluster we are able to change the position of each gene inside it's cluster and shuffle the cluster within the operon. The promoter as well as the terminator was put outside this combinatorial system to avoid the change of their position.
  This organisation allowed us to <STRONG>reduce the number of combination down to 3552</STRONG>.
  We thought to assemble the whole operon in two phases. First we would assemble each cluster individually using the <B>Gibson assembly method</B>. This method allows assembling gene fragments carrying overlaps of about 25 bp lengths at each end.
  We ordered as &quot;gene fragment&quot; (Eurofins-MWG) the cluster-specific lox sequence extended with the overlapping sequences. We added at the beginning and at the end of the cluster a non-coding nucleotide sequence for the purpose of PCR amplification of the correctly-assembled construct. 
  After this first step we expected products ranging from 2 up to 6 kb (Figure 2).</P><CENTER><P><B>Fig.2</B> - Organisation of the second cluster.</P></CENTER><P style="text-indent:3em;" align="justify"> At this stage we had to assemble the <B>4 clusters into a plasmid</B>. We decided to switch from the Gibson assembly method to the <STRONG>Yeast assembly method</STRONG>, which is more accurate during replication. To perform yeast assembly the fragments that have to be assembled are required to display a minimum of 250 bp overlap. Those fragments are then co-transformed in <EM>Saccharomyces cerevisae</EM> with an linearized plasmid, in our case an ARS/CEN-ori carrying shuttle plasmid.
  In presence of these fragments the yeast recognises them as broken DNA and triggers its repair mechanisms. Homologous recombination is performed between the overlapping regions resulting after the resolution of the Holliday junction in an uncut plasmid harbouring our degradation operon. The 250 bp overlaps were ordered as &quot;gene fragments&quot;. These fragments contain the overlap of the previous cluster followed by the fifth self-recombining lox sequence and terminated by the overlapping region of the following cluster.
The final assembly is framed with the BioBrick prefix and suffix, which will be used to insert the optimal operon in <B>pSB1C3</B> (Figure 3).</P><CENTER><P><B>Fig.3</B> - Organisation of the operon.</P></CENTER><P>
To screen the best strain, we will grow the candidates on a media that will have as sole source of carbon terephthalic acid and ethylene glycol. Those molecules are the results of the cleavage of PET. The decision to use the two molecules instead of PET was made because of the low kinetic oh PET hydrolyse. The strain harboring the best operon organisation will have the fastest growth and will therefore be sequenced.</P></DIV><FONT size="2"><P><B>References</B></P><P><B>1</B> - <A href="http://www.wsj.com/articles/recycling-becomes-a-tougher-sell-as-plastic-prices-drop-1428279575" style="text-decoration:none; color:#000000;" target="_blank">Georgi KANTCHEV and Serena NG, <B>Recycling Becomes a Tougher Sell as Oil Prices Drop</B>, <I> The Wall Street Journal</I><I>BMC Genomics</I> 2006, 7:73-85.
</A></P><P><B>2</B> - <A href="http://www.biomedcentral.com/1471-2164/7/73/about#citations" style="text-decoration:none; color:#000000;" target="_blank">Missirlis PI, Smailus DE, Holt RA: <B>A high-throughput screen identifying sequence and promiscuity characteristics of the lox P spacer region in Cre-mediated       recombination.</B><I>BMC Genomics</I> 2006, 7:73-85.

</A></P><P><SPAN style="font-size: 60px; font-family: Courier;">^</SPAN><SPAN style="font-size: 20px;">Page up</SPAN></P></FONT></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>