<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UIUC-Illinois_Meeting_Minutes"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UIUC-Illinois/Meeting_Minutes">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UIUC-Illinois/Meeting_Minutes&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UIUC-Illinois/Meeting_Minutes&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UIUC-Illinois/Meeting_Minutes&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UIUC-Illinois/Meeting_Minutes" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UIUC-Illinois/Meeting Minutes</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="hmenu" style="top:-7px"><UL><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois">Home</A></LI></UL><UL><LI><A class="orange">Team</A></LI><LI><A class="orange" href="https://igem.org/Team.cgi">Official Team Profile</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Team">Team Members</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Team/Collaboration">Collaboration and Contributions</A></LI></UL><UL><LI><A class="orange">Project</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Project/Protocols">Protocols</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Project">Project Description</A></LI></UL><UL><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Parts">Parts Submitted to the Registry</A></LI></UL><UL><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Modeling">Modeling</A></LI></UL><UL><LI><A class="orange">Documentation</A></LI><LI><A class="orange">Meeting Minutes</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Meeting_Minutes">Page Format</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Meeting_minutes">Calendar Format</A></LI></UL><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Newsletter">Newsletters</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Bootcamp">Bootcamp</A></LI><LI><A class="orange">Lab Notebooks</A></LI><LI><A class="orange" href="http://openwetware.org/wiki/IGEM:University_of_Illinois_Urbana_Champaign/2009/Notebook/Bioware_2010_RNA_Decoder">RNA Decoder</A></LI><LI><A class="orange" href="http://openwetware.org/wiki/IGEM:University_of_Illinois_Urbana_Champaign/2009/Notebook/Bioware_2010_Lock_and_Key_Decoder">Lock and Key Decoder</A></LI><LI><A class="orange" href="http://openwetware.org/wiki/IGEM:University_of_Illinois_Urbana_Champaign/2009/Notebook/Bioware_2010_Arsenic_Bioremediation">Arsenic Bioremediation</A></LI><LI><A class="orange" href="http://openwetware.org/wiki/IGEM:University_of_Illinois_Urbana_Champaign/2009/Notebook/Bioware_2010_Gold_Bioremediation">Gold Bioremediation</A></LI><UL><LI><A class="orange">Human  Practices </A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Bioethics">Survey</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Outreach">Outreach</A></LI><LI><A class="orange" href="https://2010.igem.org/Team:UIUC-Illinois/Safety">Safety</A></LI></UL></DIV><P><A href="https://2010.igem.org/Team:UIUC-Illinois/Meeting_minutes" class="external text" rel="nofollow"><B>Meeting minutes in calender format</B></A></P><P>PDF files and Powerpoints are located at the bottom of the page.
</P><P>Most recent at top of page.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#10.17.10"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">10.17.10</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#10.10.10"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">10.10.10</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#10.3.10"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">10.3.10</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#9.19.10"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">9.19.10</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#9.12.10"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">9.12.10</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#9.1.10"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">9.1.10</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#8.2F29.2F10"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">8/29/10</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#8.2F10.2F10"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">8/10/10</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#8.2F6.2F10"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">8/6/10</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#7.2F26.2F10"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">7/26/10</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#7.2F23.2F10"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">7/23/10</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#7.2F21.2F10"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">7/21/10</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#7.2F20.2F10"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">7/20/10</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#7.2F16.2F10"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">7/16/10</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#7.2F14.2F10"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">7/14/10</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#7.2F13.2F10"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">7/13/10</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#7.2F12.2F10"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">7/12/10</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#7.2F9.2F10"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">7/9/10</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#7.2F6.2F10"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">7/6/10</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#7.2F2.2F10"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">7/2/10</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#7.2F1.2F10"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">7/1/10</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#6.2F30.2F10"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">6/30/10</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#6.2F29.2F10"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">6/29/10</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#6.2F28.2F10"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">6/28/10</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#6.25.10"><SPAN class="tocnumber">25</SPAN><SPAN class="toctext">6.25.10</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#6.2F24.2F10"><SPAN class="tocnumber">26</SPAN><SPAN class="toctext">6/24/10</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#6.2F15.2F10"><SPAN class="tocnumber">27</SPAN><SPAN class="toctext">6/15/10</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#6.2F14.2F10"><SPAN class="tocnumber">28</SPAN><SPAN class="toctext">6/14/10</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#6.2F11.2F10"><SPAN class="tocnumber">29</SPAN><SPAN class="toctext">6/11/10</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#6.2F4.2F10"><SPAN class="tocnumber">30</SPAN><SPAN class="toctext">6/4/10</SPAN></A></LI><LI class="toclevel-1 tocsection-31"><A href="#5.2F28.2F10"><SPAN class="tocnumber">31</SPAN><SPAN class="toctext">5/28/10</SPAN></A></LI><LI class="toclevel-1 tocsection-32"><A href="#5.2F27.2F10"><SPAN class="tocnumber">32</SPAN><SPAN class="toctext">5/27/10</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#5.2F26.2F10"><SPAN class="tocnumber">33</SPAN><SPAN class="toctext">5/26/10</SPAN></A></LI><LI class="toclevel-1 tocsection-34"><A href="#5.2F25.2F10"><SPAN class="tocnumber">34</SPAN><SPAN class="toctext">5/25/10</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#5.2F24.2F10"><SPAN class="tocnumber">35</SPAN><SPAN class="toctext">5/24/10</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#4.2F28.2F10"><SPAN class="tocnumber">36</SPAN><SPAN class="toctext">4/28/10</SPAN></A></LI><LI class="toclevel-1 tocsection-37"><A href="#4.2F25.2F10"><SPAN class="tocnumber">37</SPAN><SPAN class="toctext">4/25/10</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#4.2F23.2F10"><SPAN class="tocnumber">38</SPAN><SPAN class="toctext">4/23/10</SPAN></A></LI><LI class="toclevel-1 tocsection-39"><A href="#4.2F21.2F10"><SPAN class="tocnumber">39</SPAN><SPAN class="toctext">4/21/10</SPAN></A></LI><LI class="toclevel-1 tocsection-40"><A href="#4.2F16.2F10"><SPAN class="tocnumber">40</SPAN><SPAN class="toctext">4/16/10</SPAN></A></LI><LI class="toclevel-1 tocsection-41"><A href="#4.2F11.2F10"><SPAN class="tocnumber">41</SPAN><SPAN class="toctext">4/11/10</SPAN></A></LI><LI class="toclevel-1 tocsection-42"><A href="#4.2F10.2F10"><SPAN class="tocnumber">42</SPAN><SPAN class="toctext">4/10/10</SPAN></A></LI><LI class="toclevel-1 tocsection-43"><A href="#3.2F19.2F10"><SPAN class="tocnumber">43</SPAN><SPAN class="toctext">3/19/10</SPAN></A></LI><LI class="toclevel-1 tocsection-44"><A href="#PDFs:"><SPAN class="tocnumber">44</SPAN><SPAN class="toctext">PDFs:</SPAN></A></LI><LI class="toclevel-1 tocsection-45"><A href="#Powerpoints:"><SPAN class="tocnumber">45</SPAN><SPAN class="toctext">Powerpoints:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="10.17.10"><B>10.17.10</B></SPAN></H2><P>Practice Presentation:  October 25th 11:00 room 612 â include some background info with the jamboree presentation.
</P><P>Meagan, Anna, and Francis will be putting together some modeling this week. 
</P><P>Barbeque: steve turned in deposit slip hopefully, so weâll be grilling out this thrusday. 
</P><P>Advertising: Amanda made a flyer/slide. Send info to newsletters/facebook/lectures 
</P><P>MCB 332âMeagan
</P><P>MCB 314âAmanda
</P><P>ACESâeveryone find a friend of a friend whoâs in ACES!
</P><P>MCB 354âTom/Erin?
</P><P>MCB 301âErin 
</P><P>Another general meeting:  Tuesday November 30th 7-8 PM 162 noyes
</P><P>Amanda, Francis, Meagan, Erin, whoever wants to help meet Wednesday at 4 PM to work on the presentation.
</P><P>Poster: find meeting time to work on it. Tentatively at 4 on Wednesday. 
</P><P>Powerpoint outline:
</P><P>Donât really need side bar. Just include it in the title of the slide.
Lots of pictures!
</P><P>1. Team: 	
</P><P>UIUC, Illinois:Combine team picture and UofI pictures and a map of location team picture, U of I project, map, bench picture.
</P><P>Just say weâre undergraduates in mcb and bioengineering
</P><P>Advisors: looks good
</P><P>Lab: put a picture of the IGB at the end with acknowledgments. 
</P><P>2. Goal
</P><P>Donât need title slides at the beginning of each section. 
</P><P>Statement: Amanda will write up a couple sentences to put into slide. 
</P><P>UIUC Bioware 2009: brief description of the progress made last year, we want to make it sound like we didnât get very far last year so we can make this year sound better. Last year was preliminary research, establishment of lab skills. 
</P><P>Bioware 2010: goal was to expand the 2009 project idea and improve logic gates with the potential application of a decoder. 
</P><P>In order to something about gold and arsenic and bioremediation. 
</P><P>3. Project
</P><P>sRNA: explain what they are, why we want to use them
</P><P>Basic sRNA Logic Gate: introductoryâinstead of all four pathways for the diagram, show, explain one. 
</P><P>Decoder:one slide introducing the idea of the decoder, one slide with out MCB version with all the names written our and stuffâ¦ just show a diagram or something and explain
</P><P>Small RNA Data: graphs and stuff
</P><P>Gold/ArsenicÂ : have one slide introducing the idea. Important components: efflux pump, metal binding displayed surface protein, gas vesicles, arsenic/gold sensing system (TFs).  The picture can cover all these parts, just show them one at a time.
</P><P>Final Construct: overall idea was to combine the two. After considering the systems in depth we found the significant part was speed of the RNA to produce a quick response to the presence of ars or gold in the environment. Donât worry too much about details yet.
</P><P>What Weâve Accomplished: general (more detail under achievements), data?
</P><P>Continuing workÂ : complete the decoder, test the metal system
</P><P>4. Modeling: Meagan by Friday evening
</P><P>Arsenic
</P><P>Gold
</P><P>Small RNA
</P><P>5. Bioethics
</P><P>Keep as is, focused on outreach, surrounded by science
</P><P>6. Achievements:
</P><P>Bronze:
</P><P>Silver:
</P><P>Gold:
</P><P>Biobricks: these are what we submitted, these are our favorites.
</P><P>Collaboration:
</P><P>Summary slideâput this under project instead. 
</P><P>Acknowledgements:
</P><P>Meagan will work on this. Sponsors, IGB, etc. 
</P><P>Fun slide at the end with random pictures at the end that says âquestionsâ and âthanks for listening or somethingâ 
</P><H2><SPAN class="mw-headline" id="10.10.10"><B>10.10.10</B></SPAN></H2><P>Grill-Out:  See if we can ask the RSO somewhere to keep the grill for the grill outs. We should try one more barbeque to make our fundraising goal. We could probably raise prices as well. So plan for next Thursday. Amanda will set up time sheet for sign up again. We can try drinks as well. If thereâs left over stuff we can have our own barbeque next Sunday!Â :) 
</P><P>Application:  Meagan has it set up, will embed in the webpage.  Francis will be at all interviews and will set those up. Will be due December 3rd.  decisions will be out on reading day. Email bhalaro and jin about talking to their undergrad classes. Amanda will make advertising slide for next Sunday.
</P><P>Get parts sent by the 22nd. 
</P><P>On Wednesday Jin and Rao wanting to see our presentation rough draft. 20 min presentation, 5 min for question. 
</P><P>Erin, Francis, Amanda are presenting at MIT.
</P><P>Drivers: Bob, Steve, Meagan.
</P><H2><SPAN class="mw-headline" id="10.3.10"><B>10.3.10</B></SPAN></H2><P>Practice presentation on the 13th during the regular meeting time.
</P><P>Meeting with Dr. Rao tomorrow at 4:00
</P><P>BioE120: keep it relatable
</P><P>Worked on Presentation âmeagan will send it out outline.
</P><H2><SPAN class="mw-headline" id="9.19.10"><B>9.19.10</B></SPAN></H2><P>We should advertise for the general meeting in ACES
</P><P>Fundraising: weâve deposited $93â¦
</P><P>Hopefully barbeque next week or the week after to barbeque. Or selling grilled cheese or something. Amanda will check 
to see if itâs profitable, set up googledoc time sheet. Amanda and steve will put this together.  Steve will get us 
space for the 30th. 
</P><P>Weâve currently made $150, we would like to aim for $200 before the jamboree. 
</P><P>Administrative stuff:
</P><P>Meagan is going to make sure everyone is registered for the competition.
</P><P>Francis will work on getting the summary in.  
</P><P>Matt will look into Engineering Council funding. 
</P><P>Francis is working on the Abbot grant. The plan is to get it out next week. Working on abstract/project description.
</P><P>Tom will call liquid nitrogen guy
</P><P>Lab Stuff:
</P><P>Meagan wants to make the fusion work this week
</P><P>Tom will be in charge of making the poster
</P><P>Budget:
</P><P>Weâre at 14,800 dollars. We started around 28,000. Weâve spent a lot on lab supplies. Hopefully weâll get better numbers on where weâre at now by next week. 
</P><P>Track: information processing. We want to combine promoters with logic gates. Then use the promoters and logic gates to express efflux proteins or gas vesicles. Then get everything together as a decoder.
</P><P>BIOE120: Oct. 4th 2:00. Francis, erin, tom, colin, palak, steve, Jeremy.  Erin, Palak, Colin can present. 
</P><P>Next Year: 
</P><P>recruiting goals: we could try to pick someone up from ACES, the broader we are, weâd have more chances we have for funding.  nothing smaller than 8 members. ACES if possible, MCB, senior MCB students who can do procedures, senior engineers to work with matlab, then make it as broad as possible from there. 
</P><P>Application Process: CV/resume, essay (motivations), online googledoc application ie: major, year etc. interview likely candidates with 2 past team members conducting each interview with basically the same questions as last year. 
</P><P>Timeline:
	application: posted right after competition	
</P><P>	application deadline: 2 days after thanksgiving break (1hr research  credit offered spring 2011)
</P><P>	Interviews: 2-3 weeks before finals
</P><P>	Team decided: Finals week
</P><P>	Project decided: before spring break
</P><P>	transition period: spring 2011
</P><P>Board Positions:
</P><P>more specific positions with more defined responsibilities. 
</P><P>Transition for transfer of responsibilities and contacts.
</P><P>Fundraising position
</P><H2><SPAN class="mw-headline" id="9.12.10"><B>9.12.10</B></SPAN></H2><P>Letâs get all parts done by the end of September. Then we can characterize in October and send it in.
</P><P>General meetings September 29th, 30th. 7:30-8:30. 
</P><P>Fundraising:
</P><P>No more dippinâ dots since weâre out of liquid nitrogen.
</P><P>Pysch studies: if you have the time, do it. 
</P><P>Barbeque on the quad?
</P><P>Volleyball tournament on the 25th, Meagan will send flier again. ADVERTISE! Sonam might be a good contact for prizes. 
</P><P>Weâre a part of engineering council! We would like to apply for funding from EC, matt will be in charge of this. 
</P><P>Hopefully bob can help with a sample budget
</P><P>BioE 120: Weâll talk to them on Oct. 4th. Similar powerpoint to the general meeting, Meagan, Colin and palak will put it together. Anyone interested in talking should plan on staying an extra half hour after the team meeting that weekend to practice. 
</P><P>MCB290 research workshop: we can submit a slide and weâll have a table during the break to talk to people. 
</P><P>Meeting with advisors this Wednesday 4:00. 
</P><P>Project:
</P><P>Project AbstractâFrancis
</P><P>PosterâFrancis, Amanda.
</P><P>Track Selectionâdecoder is more logic, ars/gold more bio-remediation. We need to pick one! 
</P><P>Start putting parts in sandbox
</P><H2><SPAN class="mw-headline" id="9.1.10"><B>9.1.10</B></SPAN></H2><P>Wiki: We should try to change the year in the web address to 2010 instead of 2009. 
</P><P>Safety: we should scan in all our MSDS forms for everything weâve used. Courtney can help us find them or we can find them online.  Basic lab safety stuffâlab coats, etc. disposal of materials, arsenic disposal
</P><P>For verifying that we constructed ArsR/lamB fusion use different primers than those we used to construct the fusion.
</P><P>When we autoclave, make sure we donât just leave the biobags in the room, make sure they get in the autoclave.
</P><H2><SPAN class="mw-headline" id="8.2F29.2F10"><B>8/29/10</B></SPAN></H2><P>Meetings every Sunday at 2:00PM
</P><P><B>Engineering Council meeting:</B> weâre going to present a couple slides at the general meeting on Wednesday. Weâll need 1 representative if we end up getting in to go to meetings once a month. They do a lot of events  and we could put events in their listserv.  As a part of engineering council, weâll just have to send someone to meetings and weâll be able to participate in engineering week. 
</P><P><B>Dippinâ Dots:</B> whenever Bob and Meagan finish RSO quizzes we can reserve a spot on the quad to sell. Soooo hopefully weâll be selling this Wednesday, and then Wed/Thurs next week. Francis PICK UP SIGN. Sign up for slots on the spreadsheet. Weâll bug the tools team for filling left over slots. Tom and Francis can pick up ice cream, Styrofoam containers, Tuesday night. Post fliers!
</P><P><B>Volleyball:</B> we only have 1 court reserved at the moment. No oneâs responded yet, WE NEED TO DO A GOOD JOB ADVERTISING!!!  Now weâre moving the tournament to the 25th. (change date on fliers)
</P><P><B>General Meeting:</B>  try to get everyone at the meetings, we can see if we can get food. 
</P><P><B>Course Credit:</B> Meagan will leave the forms on the desk in lab, come back to talk to Dr. Jin. Donât fill in project description space, Meagan will turn it in with our sheets. Dr. Jin wants a summary of our iGEM work to give credit. Meagan will email us the CRN number to register. Give the form to Meagan once itâs been signed and sheâll take it up to the BIOE office. Have the form back to Meagan no later than Sunday
</P><P><B>BIOE 120:</B> Colin, Palak and Meagan were going to put together some slides, whoever can go should go even if we all canât talk. Joan Huber wants us to do something similar to freshman seminar for MCB.  Amanda can be the contact. We want the presentation to look professional. The BIOE presentation will be at 2:00 on Monday Oct. 4th. Steve, Amanda, Tom, Erin, maybe Francis. 
</P><P><B>Advisors Meeting:</B> Wednesday, summary of our summer work.  Needs to be put together beforehand!! Not last minute! Put together 5-10 slides, keep it to 10 min or less per group. Cover what weâve accomplished and plan for the semester up until the competition.
</P><P><B>Lab Updates:</B>  buy more chemically competent cells. We want to spend less than 1000 dollars on whatever lab supplies weâll need till the end of the semester. Run an email past Bob if weâre buying anything more than $50 so we know weâre ok with the money.
</P><H2><SPAN class="mw-headline" id="8.2F10.2F10"><B>8/10/10</B></SPAN></H2><P>Meeting for games camp tomorrow at 10
</P><P>Amanda/TomâGolB on plasmid!
</P><P>We need to update project page on wiki-everyone just describe their own section of the project
</P><P>Steve put protocols on wiki!
</P><P>Lab meetings Wednesdays and fridays at 4.
</P><P>Meetings during school year: Sunday afternoons. 
</P><P>Presentation for BIOE120 freshman class on Monday from 2-3. Steve, Tom, Amanda, Bob, and Erin can make it. Recruiting 
opportunity!
</P><P>Volleyball: we have 1 court reserved at the moment costing 48 dollars. We need to get more courts hopefully. We seriously need to do a good job advertising!!
</P><P>Psych Surveys: everyone needs to do at least one, probably easiest before school starts. 
</P><P>Testing out chemical transformations today, much cheaper!!!
</P><H2><SPAN class="mw-headline" id="8.2F6.2F10"><B>8/6/10</B></SPAN></H2><P>Dr. Rao will be out of town next week.
</P><P>ClustalwâUse to compare two or more sequences. We should be expecting sequencing to be pretty dead on.  
</P><P>Maybe try some pre-mixed mastermix to troubleshoot stuff. Invitrogen used to sell taq supermixes.  Can look up components and make our own stock if we need to. It probably wonât be too much more expensive to just by the mix. 
</P><P>The first 100 bases and last 200 bases of sequencing are usually useless.  
</P><P>Colony PCRâ20 ul supermix, use a pipette tip to take a little bit of the colony (leave a little to streak later), stick in reaction, then just stick in thermocycler. 
</P><P>Restreak things out to get cleaner minipreps.
</P><P>For GES, just cut out top band from initial PCR and clone, donât re-PCR and gel extract again.
</P><P>PCR, PCR clean up, digest, run gel, extract, ligate. Use the intensities of the bands to judge ligation ratios. 
</P><P>For gel extractions, do 8 ul spin first, then can do a second 4 ul spin. 
</P><P>Make sure weâre mixing all the reactions
</P><P>Try re-transforming left over ligations the next day after letting it sit overnight.
</P><P>T-4 Ligase inhibits electroporation, heat inactivate it. Can also run it through a column as well. 
</P><P>Hit âmeasurementâ button, tells you time, then hit electroporate, write down the number that flashes, should be between 4.5-5.5. If you get something like 2, then the cells arenât good. Itâs often arcing or something. Lets you know if things are going right. Pick EC1 or EC2 based on cuvette size. 
</P><P>Bleach and weak base to clean out cuvettes. Probability of arcing goes up exponentially with each use. Once it arcs, throw it out.  
</P><P>Try chemical transformation if we can.
</P><P>Miniprep our competent cells to see if thereâs a plasmid. Test transformed cells on different antibiotics. 
</P><P>Dr. Rao wants to come to the Jamboree. Would have to fly out Friday night, can work on his own tickets, will talk to us in about a week. 
</P><P>We all need to fill out intent to travel forms so we can start buying plane tickets soon. 
</P><H2><SPAN class="mw-headline" id="7.2F26.2F10"><B>7/26/10</B></SPAN></H2><P>Games camp presentation practice Wednesday at 7âpresentation run through.
</P><P>Project overview:
</P><P>Arsenic: ARsR ArsB pArs everything cloned out except ArsB. Going to do gradient with DMSO tomorrow. ArsR on plasmid. Waiting on primers with ecoRI site for pArsR before working it into plasmid.
</P><P>Gold: GolB GolS GolT GesABC everything cloned out except GesABC. GolS on plasmid
</P><P>Collection: LamB pRpoS both cloned out, not on plasmids yet. waiting on primers with ecoRI site for pRpoS before working it into plasmid. should probably order primers for LamB with EcoRI. 
</P><P>Small RNA: micF ompF micA ompA eGFP GadY GadX Hfq all cloned out and on plasmids.
</P><P>Dippinâ Dots: Wednesday, finish off whatâs left on Thursday. 
</P><P>We should start looking into plane tickets etc. 
</P><H2><SPAN class="mw-headline" id="7.2F23.2F10"><B>7/23/10</B></SPAN></H2><P>Genomic prepâcut DNA by a factor of ten. 
</P><P>Donât have to purify before SAP
</P><P>40 micrograms/ mL for Kan plates. 
</P><P>Add 1 ul DMSO to PCR. 
</P><P>Plate 15 ul, otherwise, should probably get a lawn.
</P><P>Competent cellsâmake sure we wash enough. Final vol. should be around 1 mL. 
</P><P>Chemically competent cells may be easier. 
</P><P>There are some specialized enzymes that may be good for amplifying large fragments (ie: GesABC). Can also add DMSO to reduce melting. 
</P><P>We can be in lab as long as other people are working in the lab as well, ie: grad students. We can try calling Dr. 
Jinâs grad students to see if theyâll be in lab when weâd like to be in lab. 
</P><P>GolT try 50 ul PCR, Look at touchdown PCR, get high specificity. Maybe increase the number of cycles. Maybe a standard kit as well will help. Test it out with taq supermix where you just add primers and template. 
</P><P>Possible genomic DNA contamination in pSB1A2 for Tom and Amandaâs gel. Be crazy careful with PCR and contamination.  
</P><P>Change tips whenever you think you may have touched it to something, donât breathe on it etc. 
</P><P>Send out small RNA biobricks for sequencing.
</P><P>We can move our stuff into Dr. Jinâs -80 while Courtney looks for a -80 for us. 
</P><H2><SPAN class="mw-headline" id="7.2F21.2F10"><B>7/21/10</B></SPAN></H2><P>Weâll be able to use the plate reader by the end of the week, weâll be able to do assays and have pictures for lab meeting by the end of the week. 
</P><P>Amanda/tomâhave golT golB and golS. Need to start getting terminators and RBS on plasmids
</P><P>Everyone should try to make a schedule of what needs to be done to complete the project. 
</P><P>For Fridayâs lab meetingâoverall schematic with our design, in combination with the decoder.
</P><P>New competent cells from NEB work really well. Buy some more.
</P><H2><SPAN class="mw-headline" id="7.2F20.2F10"><B>7/20/10</B></SPAN></H2><P>Meeting with Courtney
</P><P>We can detect stuff but itâll be hard to quantify. 
</P><P>We should go to the analytical lab and ask if they can help us out
</P><P>Gold: fluorescent assay maybe? Do some reading.  Lots of info on fluorescent bound gold in the literature. 
</P><P>Spin down cells, test supernatant, lyse cells, test, whatâs in the cell. 
</P><P>Courtney has some papersâwill email or print off.
</P><P>Arsenicâchemical detection is pretty much either flaming or ash--which would tell us how much is in the cell) 
</P><P>Look up mass spec. see if we can look up how to prepare the cells for that, Courtney has mass spec. 
</P><P>To see cell surface, wash metal off by changing pH and measure after. 
</P><P>Try AA. 
</P><P>ICP and AA are what the EPA uses. 
</P><P>We need to find people on campus who have this stuff, see if theyâll help us out, run our samples. 
</P><P>See if we can contact Groenigan so we donât repeat their mistakes.
</P><P>John KaiâPCR help. 
</P><P>Go to chem library to see if we can find any info on this. Check older books to find info on how to assay by chemical 
means rather than instrumentation. 
</P><P>Look on UIUC website to see if any labs are doing this kind of research. 	
</P><P>2005 paper gold fluorescence assay. 
</P><P>Sweetcorn Festival Insuranceâsee if we can go through a local company, also need to get approved by the health 
department. 
</P><P>The missing pipetteâwas actually sitting at mechanical engineering, itâs gonna get calibrated soonâ¦
</P><P>Dip nâ dots:
</P><P>for tomorrow, just use one size. For future dates, try different sizes, large, small, shot. 
</P><H2><SPAN class="mw-headline" id="7.2F16.2F10"><B>7/16/10</B></SPAN></H2><P><U>Advisors Meeting</U></P><P>Courtney will be gone from Aug. 11th or 12th until the 21st.
</P><P>Ask liquid nitrogen guy what kind of container itâs in, if we can get it out. Keep tank in 3200 IGB. We can try making dip nâ dots in the teaching lab or courtneyâs tissue culture room since itâs not really being used for anything now. Use Courtneyâs colander, coolers. 
</P><P>Could ask blood bank for some dry ice.
</P><P>GAMES camp meeting: Monday 10:30 AM 3110 DCL. West side of building. Pick up cooler then too. 
</P><P>Meeting with Courtney: Tuesday 11:00 AM  
</P><P>Figure out wells and spacers for protein gels. 
</P><P>John Kai 3:30âreally good at PCR/troubleshooting
</P><P>Try courtneyâs transilluminator to visualize and cut out gel. Or just be really quick.  Donât want to expose it to UV for too long. Or use Courtneyâs pipette tips that cut gel for us. Try using Dark reader, used visible light but need to use cybergreen gold instead of ethidium bromide but itâs a bit more expensive. Costs 200-300 dollars, using visible light wont damage the gel. Look up the specific dye we need. 
</P><P>Try to use pure genomic DNA template instead of colony boil.  1 prep will give us a ton of PCR reactions.   
We can do reverse PCR of the backbone to amplify, no change of insert being amplified, only get backbone. Amplify linearized plasmid ,see if we can get it to work.
</P><P>Talk to Courtney to see if she has the DNA prep. Kit. 
</P><P>Water may be contaminated by DNAse or some problems with competent cells. Buy competent cells to use as positive control, compare to our cells. Try each of us having our own water so we know exactly which water was used in the experiment and if itâs correlated to success/failure/contamination.
</P><P>Dr. Jin used 2-4 ul ligation per transformation. 
</P><P>Use promoter regulated by lacI and tetR and see how regulation of the promoter compares. 
</P><P>For mercury vapor lampâbill flesher in electronics shop for school of life shop.  Basement of burrill maybe.
</P><P>DH5a has lacI deletion.
</P><P>PCR clean up should be 60-70% recovery, check buffers.
</P><P>260/280âneeds to be around 1.8, up to 2 is ok.  This ratio is more important than the 260/230. 
</P><P>Try drying column more completely with extra centrifugation step
</P><P>Use quiagen columns!!
</P><P><U>Tools Team Meeting:</U></P><P>Jeremyâs computer crashed kinda killing productivity. About a week behind right now. 
</P><P>Working on a logo
</P><P>Buying server today 
</P><P>Dip nâ dots:
</P><P>Tom and Colin going to pick up 5-6 gallons ice cream, spoons, napkins from Sams Monday evening. Store in Mattâs 
freezer. 
</P><P>everyone bring large bowls. 
</P><P>Chalk Quad
</P><P>Bob will grab table from RSO.
</P><P>poster
</P><P>Anna will make signs for Dip nâ Dots
</P><H2><SPAN class="mw-headline" id="7.2F14.2F10"><B>7/14/10</B></SPAN></H2><P>Francis/Bob: putting stuff into RFC10 format.
</P><P>Meagan/Erin: got bands for RpoS.
</P><P>Matt/Steve: Mattâs pulling a part from kit/plate. Hopefully start digests tomorrow.
</P><P>Tom/Amanda: still doing PCR to get stuff out of genome.
</P><P>Email advisors to remind them to come to the meeting.
</P><P>Collaborate with northwestern/visit?
</P><P>Chalk the quad for dip nâ dots on Monday night. 
</P><P>Trip to Chicago aug. 7th??
</P><P>How to train your dragon on the quad Thursday night at 9 PM. 
</P><H2><SPAN class="mw-headline" id="7.2F13.2F10"><B>7/13/10</B></SPAN></H2><P>Erin/Meagan: got 6 bands on the gel today!!!! If anyone has a chance, look over the lamB primers since this is the 
only one not working
</P><P>Amanda/Tom: maybe reordering primers since they might be contaminated.  Try to aliquot and dilute to 10 micromolar. 
</P><P>Matt/Steve: Mattâs gonna start doing some stuff on his own. Starting stuff over since nothingâs worked.
</P><P>Bob/Francis: most stuff PCR-ed out. Waiting on a certain strain. 
</P><P>Fundraising: We need to chalk the quad, and make sure we have all the supplies for dip nâ dots for next Wednesday. We should figure out how much liquid nitrogen we need for a pint of ice cream.  Maybe 2 pints liquid nitrogen per pint of ice cream. We should use a whisk or big spoon for stirring. Make sure we keep track of how much we use of everything so we can calculate stuff for the sweet corn festival. Soooo 30 gallons of liquid nitrogen maybe?? 
</P><P>Meaganâs gonna make a flyer. With any luck Erin will remember to make a facebook event.
</P><P>Matt will send out abstract to everyone, check it over, make suggestions by Thursday. 
</P><P>Friday at noon, meeting with Courtney about assaying metals. 
</P><P>Bob will talk to people about calibrating pipettes
</P><P>GAMES camp meeting Monday morning with Joanne. 
</P><H2><SPAN class="mw-headline" id="7.2F12.2F10"><B>7/12/10</B></SPAN></H2><P>Meeting with courtney at 11:00 tomorrow!
</P><P>Dip nâ Dots next Wednesday from 11-2
</P><P>Meagan will be out of town next week and the week after, contact by email the first week, not so much second week
</P><P>Tom/Amandaâretrying ligations, and PCR with less template.
</P><P>Francisâtrying to get RBS in front of a part 
</P><P>Erin/Meaganâgot bands today! New PCR reaction mixture
</P><H2><SPAN class="mw-headline" id="7.2F9.2F10"><B>7/9/10</B></SPAN></H2><P><B>Meeting with advisors</B></P><P>Powerpoint attached. 
</P><P>See if we can get a ladder that covers more sizes. Check stockroom and NEB website. We should use the 200bp ladder 
that we got donated.
</P><P>PCR troubleshooting: Way too much primer probably. Try cutting primers in half. Try cutting template in half. 
</P><P>50 ul: .5-1 uM template, 1 ulprimer 1 ul primer, .5-1polymerase 
</P><P>Try adding  pol individually to each tube, mix by pipetting/
</P><P>Try dropping annealing to 50
</P><P>For the gold collection system side with PCR protocol written out, use 1 ul less of template.
</P><P>Email dr rao for protocol for chromosomal prep.
</P><P><B>Meeting with Tools Team</B></P><P>Volleyball tournamentâwe have to pay for the courts ~12 dollars an hour.  Free agencyâindividuals who want to play but donât have a full team.  Saturday September 4th 2010. 
</P><P>Keep Jessie updated for reffing the tournament. 
</P><P>Dip nâ dots:  buy stuff from samâs club!! Get big tub of ice cream instead of making a mix, it will save us time and probably money.  Plan to do this ASAP. On the 15th  from 11-2.
</P><P>Tools teamâworking on openwetware,  buying server.
</P><H2><SPAN class="mw-headline" id="7.2F6.2F10"><B>7/6/10</B></SPAN></H2><P>Francis working on coloroid. Making small RNAs RFC10 compatible. Also ordered some site directed mutagenesis primers. 
</P><P>Deadline: have constructs by the end of the month. Run a ton of ligations at once, donât rush. 
</P><P>Online Notebook on openwetware, be sure to update regularly!!
</P><H2><SPAN class="mw-headline" id="7.2F2.2F10"><B>7/2/10</B></SPAN></H2><P>Donât expose our gels so much, keep intensity a bit lower.
</P><P>Do gel extractions, grab the insert from the gel and use that to ligate so we donât have the binding competition from the plasmid that originally contained the insert. Zymo gel extraction kit works best. But we can grab qiagen as well.  
Want about 4 times as much insert as backbone, at least 1:1 insert:backbone. We can use the kit from the south side. 
Try to use bigger combs, low melt agarose, be careful about taking the comb out to not tear the gel, stick it in the fridge for a little while to make it easier to pull it out without tearing.
</P><P>Maybe contaminated primers with template. Be really careful about contaminating things, change tips is anything touches it. Try just running primers and see what we get. Test the pipette by just sucking up and pipetting out, run it. Weâll see about getting a guy out here to calibrate them. 
</P><P>For synthesizing promoter/RBS: use oligos, 2 rounds of primer extension, purify on acrylamide gel. Design 250 bp primer and use that to synthesize the promoter/RBS.
</P><P>60 base pairs is pretty much the max we can use without purifying for primers. 
</P><P>cut our plasmids before running them on the gel to be able to compare them the ladders since everything is standardized against linear DNA. 
</P><P>We need to start asking companies for next yearâs funding around this time. 
</P><P>GAMES campâmeeting on July 19th in the morning. Want more images, more slides so one concept per slide. 
</P><P>RpoS promoter for the RpoS gene to use as stationary phase promoter for collection system.
</P><P>We can do a membrane prep to see if the protein fusions are in the membrane. 
</P><P>Dr. Jin has a kit for the protein gels, we can buy pre-casted gels. We can talk to Lon? About outer membrane protein gels. 
</P><P>We can talk to microanalytic people in chemistry about metal assays. Check on school of chemâs website.
</P><H2><SPAN class="mw-headline" id="7.2F1.2F10"><B>7/1/10</B></SPAN></H2><P>run controls on digestions, check emails for email from francis about what controls to run
</P><P>Run digestions at 37 degrees for 15 min, one for an hour, one for overnight to see which work best. Most protocols say 1 hour for digestions and ligations. Reaction mixes are fine. 
</P><P>One with no enzyme, one with no DNA, â¦steve is sending an email with all this info
</P><P>We should aliquot out all our buffers, dNTPs. 
</P><P>Whenever we get ladders, make sure we dilute, aliquot . can use 1 ul of ladder for each gel for clear bands. 
</P><P>Start saving electroporation cuvettes, we can re-use them. 
</P><P>Can register computer to printer on north side. Go to IGB website. The IP address you need is on the printer behind the secretaryâs desk. 
</P><H2><SPAN class="mw-headline" id="6.2F30.2F10"><B>6/30/10</B></SPAN></H2><P>Francisâconnected colloroid biobrick. Ran colony PCR. Started another digestion/ligation this morning. Hopefully will have it all done tomorrow. 20 microliter reaction, 1 for 10 min at rt, 1 for an hour at rt, 1 for an hour at 4 degrees C.  digest for 20 min. 
</P><P>PCR purification before SAP. Donât need to do it after SAP.
</P><P>Steve emailed environmental engineering profs. We might get to meet with one next Wednesday or Thursday. 
</P><P>Going to talk to dr. rao again tomorrow about fusions. Look up ArsR gene as well. 
</P><P>Courtney will talk to us July 8th at 11am about assaying metals.
</P><P>Get ready to sell Dip nâ Dots on the quadâlook into making an ice cream mix with stuff from Samâs Club. Francis will 
make a list of stuff we need. Get different flavors as well.  Steve will talk to Courtney about getting more liquid 
nitrogen. Weâll need some sort of liquid nitrogen containment. 
</P><P>Meagan has fliers designed for the volleyball tournament. 
</P><P>Meagan designed primers for the ArsR promoter. 
</P><H2><SPAN class="mw-headline" id="6.2F29.2F10"><B>6/29/10</B></SPAN></H2><P>Before sticking stuff in the negative 80, stick it in liquid nitrogen or dry ice.
</P><P>Dip nâ dots worked!! 
</P><P><B>Meaghan/Erin</B>âhopefully tomorrow weâll put promoters with gas vesicles and efflux system. 
</P><P><B>Steve/matt</B>âdigestion today, xba and spe might not be working. Theyâre testing them today. The only ones we know for sure work are ecoRI and pstI.
</P><P><B>Tom/Amanda</B>âpretty much starting over. 
</P><P>Tomorrow meeting with Dr. Rao at 2:00 at his office.  People who worked on fusion and ars primers should go. 
</P><P>Backboneâmaybe the annealing temp is too low or the primers are contaminated. 
</P><P>Order <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> primers.  Buy more plates. Buy more small pipette tips. Fill tips on weekend. 
</P><P>We need more e. Coli colony boil.
</P><H2><SPAN class="mw-headline" id="6.2F28.2F10"><B>6/28/10</B></SPAN></H2><P>BackboneâPrimerâs are working according to Francisâ work on Wednesday. Asked steve to get the ccdb strain, submit the order. Using this for cloning would be the easiest way to do this. Plasmid encodes ccdb toxin which kills the cell. 
</P><P>Need the resistant strain to grow up the plasmids, which are in the registry. Only the cells that have digested plasmid survive?
</P><P>Amanda did PCR of backbones with negative controls, one without polymerase and one without template. There were bands in each laneâ¦
</P><P>Stationary phase promoter
</P><P>Talk to professors about applications of our project, metal assays
</P><P>Get grad students to check over fusion primers and arsR promoter primers.
</P><P>Sandbox of parts registryâmeans theyâre not submitted to the registry yet but allows documentation.
</P><P>Digest PAL, ligate into plasmid. 
</P><P>SAP 20 min incubation at 37 to keep plasmid from re-ligating on itself.
Same with Gol proteins.
</P><P>After transformations, let it grow for 18 hours before we pick colonies. Donât pick red colonies.
</P><P>Backbones are ready!!!!  15 min digests, keep reaction volumes large. 
</P><P>PCR purification when changing buffers.
</P><H2><SPAN class="mw-headline" id="6.25.10"><B>6.25.10</B></SPAN></H2><P><B>Advisors meeting</B></P><P>See if any of the grad students on the south side will look over our primers since Dr. Rao is out of the office. 
</P><P>When we label gels, label the wells at the top of the picture
</P><P>CCDB resistant strain?
</P><P>Mondayâduring lunch time, bring Courtney some melted ice cream, see if we can try to make a test batch.  Also bring a 
colander or screen. 
</P><P>Metal assayingâCourtney is still looking into stuff
</P><P>July 8thâCourtney will talk to us about metal assays. 
</P><P>Protein gelsâJoanne has a biorad casting system for the gels and possibly some other supplies. 
</P><P>We might be able to make all the solutions for mini-preps as well and just buy the columns instead. 
</P><P><B>Meeting with Tools Team</B></P><P>double check that we donât need to do site directed mutagenesis before we start digesting and ligating.
</P><P>This week the tools team went over modeling. Anna has a powerpoint outlining it all. 
</P><P>ApEâplasmid designer
</P><P>Volleyball tournament july 17th.
</P><P>Meaghan will send out fliers for us to put up around campus. 
</P><P>Volleyball courts at stadium and oak. 5-9 pm. Weâll have a cookout. Either cash prizes. Anna, Erin talk to people on green street to see if we can get gift cards, bring fliers to put in the windows.
</P><P>We could try to get a cooler jug thing for water or Gatorade. 
</P><P>Advertisingâfacebook, chalk the quad, post up fliers. Fliers at CRCE, ARC, grainger, Kenney Gym, the undergrad, that cylinder thing by the union, IGB. Send out message from that facebook group on the survey, and the UIUC summer one.  
</P><P>Create event on facebook. Put it in newsletter. 
</P><P>Rec Volleyball ListservâMeagan
</P><P>StarbucksâAmanda 
</P><P>GraingerâErin
</P><P>QuadâEveryone
</P><P>Quad pillarsâAmanda 
</P><P>ArcâMatt
</P><P>CRCEâColin
</P><P>Union--Matt
</P><P>DormsâAnna
</P><P>UGLâTools
</P><P>IGBâMeaghan
</P><P>KennyâMeagan 
</P><P>County MarketâAnna
</P><P>FacebookâErin/everyone
</P><P>TwitterâSteve?
</P><P>WikiâMatt 
</P><P>NewsletterâErin
</P><P>Expresso RoyaleâAnna 
</P><P>Engineering HallâMeagan 
</P><P>Talk to people at sand courtsâWednesday, Meagan
</P><P>Barbeque with the volleyball tournamentâColin can handle it, just need supplies, maybe from Sam Club, keep receipts. 
Need grill, lighter fluid, spatula (Matt?).  Anna might be able to use her roommateâs truck to transport Mattâs grill. Vegan friendly food. Buns, Condiments, Cheese. Pricing. 
</P><P>Give money to bob or Meagan from psych studies
</P><H2><SPAN class="mw-headline" id="6.2F24.2F10"><B>6/24/10</B></SPAN></H2><P>Volleyball tournament possibly on august 7th, though summer school will have just ended. Thinking of Saturday the 31st or the 17th. Ooor did we pick a day?? Ok yes. July 17th. From 5-10
</P><P>Erin/Meaghan going do a digestion/ligation to put a promoter in front of the gas vesicle to see if it works. 
</P><P>Should have backbone by 10:00 tonight. 
</P><P>Psych studiesâsense camâsee amandaâs email.
</P><P>FRANCIS EMAIL MEAGAN THE SUMMARY OF YOUR WORK FROM LAST SEMESTER!!!!!
</P><P>Tomorrow put together 2 or 3 slides about what weâve done in the last two weeks for the advisors meeting.
</P><P>Steve we need a list of stuff to restock/do over the weekend.
</P><P>Courtney is going to come talk to us about assaying for metals sometime soon.
</P><H2><SPAN class="mw-headline" id="6.2F15.2F10"><B>6/15/10</B></SPAN></H2><P>Metals planning meeting
</P><P>Parts: efflux, gold TF, Ars repressor, stationary phase promoter, collection, inducible promoter
</P><P>1.	Get each part in a biobrick plasmid RFC10 (no promoter at this point). Bold are already in registry.
</P><P>â¢	ArsR
</P><P>â¢	ArsB
</P><P>â¢	Gold Efflux
</P><P>â¢	Gold TF
</P><P>â¢	<B>Vesicle biobrick</B></P><P>â¢	<B>ArsR/Vesicle biobrick</B></P><P>â¢	Ars Promoter
</P><P>â¢	<B>Stationary Phase Promoterâââ</B></P><P>â¢	â<I>lamBâââ</I></P><P>â¢	PAL
</P><P>â¢	â<I>AIDAâââ</I></P><P>â¢	ArsMT
</P><P>â¢	GoldMT
</P><P>â¢	â<I>Promoters J23100, J23119âââ</I></P><P>2.	PCR tricks to make metallothionine fused to membrane protein
</P><P>3.	But membrane protein/MT into biobrick format
</P><P>4.	Place efflux system under regular strength constitutive promoterâ
</P><P>5.	Combine our parts with promoters
</P><P>â¢	Try our ArsR promoter with gas vesicles
</P><P>â¢	Stationary phase promoter with vesicles part
</P><P>â¢	MT/MP under J23119
</P><P>â¢	Efflux under J23100
</P><P>â¢	ArsR under J23100
</P><P>6.	Combine systems into 2-3 compatible plasmids
</P><P>7.	Characterize System
</P><P>â¢	Effects at different concentrations of metal
</P><P>â¢	Pond water
</P><P>â¢	Protein gels to see how much MT/MB is at cell surface and to check the size of the protein.
</P><P>8.	Modeling
</P><P>â¢	Floating of cell
</P><P>â¢	Amount of metal binding surface relative to entering cell
</P><P>9.	Steal Francisâ stuff
</P><P>10.	Cure cancer, maybe AIDS
</P><P>11.	Done! iGEM WIN!
</P><H2><SPAN class="mw-headline" id="6.2F14.2F10"><B>6/14/10</B></SPAN></H2><P><B>Meeting with Dr. Rao</B></P><P>try metal sensitive promoter to regulate efflux pump
c
ollection system under a stationary phase promoter instead of a low affinity promoter. Takes longer to turn on. Or heat shock, temperature sensitive promoters. Look into those used for control of capsule formation for stationary phase promoters. 
</P><P>No to chemotaxis and thermotaxis as collection system
</P><P>How can we expect efflux system to affect binding metal to TFâshouldnât be a big deal, reach steady state. Affinity of TF higher than affinity of pump.
</P><P>Try a bunch of membrane proteins
</P><P>send Dr. Rao a copy of the paper
</P><P>use the designs described in the papers for fusing metallothioneine to membrane protein, not RFC25.  
</P><P>using pore protein shouldnât matter.
</P><P>Protein half lifeâdepends on the protein. TF tend to be more like minutes. GFP tends to be about 24 hours. 
</P><P>Agglutinationâfimbrae and add powdered yeast and theyâll clump together
</P><P>Flocculationâdivision mutants. 
</P><P>Site directed mutagenesis with DNA fragment instead of on a plasmidâneeds to be on a plasmid. 
</P><P>mega primer PCR might work. 
</P><P>take berkeleyâs biobricks for membrane proteins. 
</P><P>clone out all the membrane proteins, try them out.  Put them in high copy number plasmid. 
</P><P>with initial cloning, put into something without a promoter?? 
</P><P>compare cells with MT to cells without. 
</P><P>Assays/diagnostic toolsâouter membrane gel. -- boil cells, outer membrane prep, run on gel to visualize protein.
</P><P>GFP isnât functional in the periplasm. mCherry might be functional in the periplasm, donât know about outside the cell. 
</P><P>How much can we fit on one plasmidâup to maybe 10-12 Kb, not including the backbone. Clone operon out, make into biobrick. ABC transporter will be high affinity, other would be lower affinity. 
</P><P>clone out gold binder, strong promoter (Tac, T7âneed special cell for T7 to work)âsee if it works. Could possibly secrete it, recover it with a his tag (bind on nickel column).
</P><P>see if gas vesicles work  
</P><P>Clone out outer membrane proteins, 
</P><P>Find biobrick with GFP without promoter. Send Dr. Rao sequence of the gene it regulates and the first 500 bp upstream of gene it regulates. 
</P><P><B>Next Steps:</B></P><P>Clone out each componentâouter membrane proteins
</P><P>See if gas vesicles work
</P><P>Express gold binder on strong promoter to see if it works: Tacâstrong promoter, T7âsuper strong promoter, need special cell
</P><P>Find sequence for promoterâsend Dr. Rao plasmid (used for reporter) and sequence we want to use for promoter, and 500 bp upstream of e. coli gene regulated by Arsenic. put in front of GFP to see if we can get it to work
</P><H2><SPAN class="mw-headline" id="6.2F11.2F10"><B>6/11/10</B></SPAN></H2><P><B>Tools team update:</B></P><P>imptools can now use compound names, not just numbers. 
</P><P>worked on the wiki, openwetware.
</P><P>working on a logo
</P><P>Buying stuffâkeep all receipts, give them to Bob. 
</P><P><B>Fundraising:</B></P><P>pysch experiments--amanda
</P><P>puppychow during the semesterâamanda
</P><P>volleyball/soccer tournamentâmeagan, erin
</P><P>sweetcorn festivalâsteve, tom, Jeremy, bob
</P><P>Dip n Dotsâfrancis, matt
</P><P><B>Advisors Meeting</B></P><P>Powerpoint file attached to email.
</P><P>Orpheum childrenâs science museumâsometime in august possibly. Come through and see the lab. Maybe do something like extracting DNA from strawberries. 
</P><P>Havenât heard anything else from parkland yet. Theyâd come film us in the lab and weâd give a seminar at parkland. 
</P><P>Maybe something at uni high or other private schools during the beginning of the semester.
</P><P><B>Points to address for games camp:</B></P><P>Difference between synthetic biology and genetic engineering. 
</P><P>Whatâs the point of synthetic biology?
</P><H2><SPAN class="mw-headline" id="6.2F4.2F10"><B>6/4/10</B></SPAN></H2><P>went over powerpointâattached to email.
</P><P>Which metals would be feasible for us to work with?:
</P><P>we can definitely work with these (except for mercury) in the lab. The concern with mercury is vapor. Courtney will call DRS to see if we can work with mercury or not. 
</P><P>Gold has many usesânanoparticles. 
</P><P>Magnetiteâitâs a lot to clone, how would we know the transporters are there.
</P><P>difficult to remove metal from metallothionine once itâs bound.---could denature the protein
</P><P>Testing metal concentrationâatomic absorption. Flame, look at the color.  Courtney may have protocols somewhere. May be just able to use a magnet. Color change from reduction?
</P><P>Water filters used now donât get all the metal out of the water. Our project would improve on this.
</P><P>Francis---did you put your primers on the order list??
</P><P>when we do primers, email Courtney and let her know what you named them
</P><P>Dr. Rao likes metallothionines, goal just to grab metal, act like a filter. Use whatever metal is easiest to measure. 
</P><P>Agglutinationâfimbri, add yeast. Bacteria will bind the yeast, precipitate. Flocculate?? âmutate septation, grows as long filament, then get so big they settle to the bottom of the liquid. Put MT cytoplasmic side, periplasm, cytoplasm. Bind in different places. Could also secrete proteins attached to MT. 
</P><P>Make decision between bio-precipitation or cell surface engineering. 
</P><P><B>Meeting with Purdue:</B></P><P>Project descriptions Purdue: Photorespirationâwant to get rid of oxygen. Want to use this idea to characterize cancer stem cells in the brain by using luciferase and infrared fluorescent protein. Characterize brain cancer stem cells. Oxygen sequestration to inhibit photorespiration and application for neuroimaging in vivo. Photorespirationâoxygen has an inhibitory effect, lowers rates of photosynthesis. Kind of a backwards photosynthesis. Rubiscoâreacts with CO2 to make glucose eventually. If get too much oxygen, binds instead of CO2. Inefficient. 
</P><P>objectivesâuse oxidase or oxygen binding protein to lower oxygen level. Construct biological parts to carry out its function in vivo, use reporters to extrapolate to brain imaging for tumors.
</P><P>donât know the specific details yet. In hypoxic condition, repressor inhibits production of oxygen binding protein. Without repressor,  transcription of oxygen binding gene occurs. Sequester oxygen so it doesnât slow rates of photosynthesis. Havenât figured out which promoter to use yet. Still planning. Starting in e.coli, moving to C3 plants.
</P><P>Want to grow temperate plants in the desert.
</P><P>How can we work together: maybe run ideas past each other. Work together with bioethics. Skype every other week. Send each other powerpoints before each meeting.  
</P><P><B>Meeting with Tools Team:</B></P><P>BioMortarâbuild off imptools. Builds a plasmid for you. Sent abstract to department of defense for funding to apply biomortar to construct E. coli to metabolize toxins. Trying to link it to the parts registry. 
</P><P><B>Outreach:</B></P><P>GAMES camp
</P><P>Surrounded by Scienceâtheyâll do a TV show about us if we give a seminar about parkland about synthetic biology. 
</P><P>Quad Day
</P><P><B>Fundraising:</B></P><P>contacting companies that helped us out last year. 
</P><P>Grantsâsending one in to Abbot. Bic grant. 
</P><P>talk to advisors in your colleges for funding.
</P><P>When asking for funds, looks good if we can say that every person on the team is dedicated to fundraising efforts for the team. So it would be good if everyone could help out with fundraising. 
</P><P><B>Brainstorming:</B></P><P>everyone can volunteer at two of them, and help organize one of them.
</P><P>Dipân DotsâMatt, Francis
</P><P>Donate Plasma
</P><P>Bake Sale
</P><P>Car Wash
</P><P>Psych ExperimentsâAmanda 
</P><P>Band
</P><P>Sell T-shirts
</P><P>Printer cartridges
</P><P>Buttons
</P><P>Pizza
</P><P>Grilled Cheese
</P><P>cookie dough
</P><P>Chili
</P><P>PancakesâJeremy
</P><P>IfoundryâJeremy 
</P><P>host soccer/volleyball tournamentâErin/Meagan
</P><P>BBQ/Potluck
</P><P>fundraising dinner
</P><P>Murphyâs
</P><P>Corn FestivalâSteve 
</P><P>Lemonade 
</P><P>Heistâ¦
</P><P>During the semesterâpuppy chow, cookie dough, grilled cheeseâColin, Meagan, Tom, Bob
</P><P>Advertise food stuff to people in IGB
</P><H2><SPAN class="mw-headline" id="5.2F28.2F10"><B>5/28/10</B></SPAN></H2><P>Presentation of what we did in lab this week:
</P><P><B>Things to note for later:</B></P><P>take pictures of plates on a piece of white paper.
</P><P>Dr. Jin has a dark reader which may be useful for viewing fluorescent colonies.
</P><P>change contrast on gel picture to see bands better. 
</P><P>have Courtney show us how to manipulate pictures of our gel on the gel reader. 
</P><P>label columns in kinetics results.
</P><P>Use different dilutions for kinetics readings too.
</P><P>open wetware for colony PCR procedure.
</P><P><B>tricks to clean up the fluorescence:</B></P><P>use cells without fluorescence as a control
</P><P>spin down, re-suspend in PBS if not getting strong signal.
</P><P><B>Research:</B></P><P>Possible directions:
</P><P>1. Super metal-resistant cell
</P><P>2. MagnetosomesâDr. Rao not a fan. Canât just stick a plasmid in to get it to work.
</P><P>3. Metal Immobilization
</P><P>4. Metal mobilization
</P><P>Working on metal resistant cell for now since itâs a prereq for everything else we would want to do with metals. 
</P><P>challenges with metal systems: Dr. Rao tried to activate fer with iron but never able to do so. LB isnât gonna work, going to have to use minimal media, chelate treat to pull out ions. Have to make sure there arenât any metals around. Most not active at high concentrations of metal. Chelating agent in media to keep it from the cell. 
</P><P>try to find salmonella counterparts of the genes in e.coli 
</P><P>ZntA expressed by one of Dr. Jinâs grad student and it works to export zinc. Exports lead and copper too. Just express a few efflux pumps and it should be pretty resistant to most metals. 
</P><P><B>People to talk to:</B></P><P>Dianne Newmanâlooking for a mechanism of 3-4 genes to clone
</P><P>mutagenize e.coli, grow on metal until something that survives. Chop up DNA, put it on plasmids, transform, see which lives, thatâs the one with the metal resistant gene. 
</P><P>Metals research: design primers
</P><P>Megan: cadmium or mercury
</P><P>Tom: gold
</P><P>Matt: nickel
</P><P>Steve: zinc
</P><P>Bob: Copper
</P><P>Amanda: Lead
</P><P>Erin: Arsenic
</P><P>Francis: Silver
</P><P><B>Meeting with tools team:</B></P><P>group to build strain designer, group to improve imptools. 
</P><P>simvector4âplasmid design 
</P><P>Modeling with tools team: 9-12 Thursday morning 
</P><P>Abstract due june 18th for chemical and biological defense science and technology conference if we want to do it. 
</P><P>wetlab: meetings with advisors Fridays at 11:00 am. 
</P><P>tools: tentative, meetings with advisors Friday at 9:30 am.
</P><P>both teams: Meetings Friday 2:30 pm
</P><P>12:00 Monday barbeque
</P><H2><SPAN class="mw-headline" id="5.2F27.2F10"><B>5/27/10</B></SPAN></H2><P><B>Bootcamp Day 4</B></P><P><B>Modeling:</B></P><P>BCCS Bristol won best modelingâmostly a tools team. Modeled interactions between cells and stuff going on inside of cells. Delivery of particles, interactions, etc. also included a user manual
</P><P><A href="https://2009.igem.org/Team:BCCS-Bristol" class="external free" rel="nofollow">https://2009.igem.org/Team:BCCS-Bristol</A></P><P>atomic dipole transition: <A href="http://www.falstad.com/qmatomrad/" class="external free" rel="nofollow">http://www.falstad.com/qmatomrad/</A></P><P>michaelis menten kineticsâmodels the rate of the reaction. Velocity and substrate concentration.  Type of inhibition.
</P><P>Henderson Hasselbach equation: <A href="http://en.wikipedia.org/wiki/Henderson%E2%80%93Hasselbalch_equation" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/Henderson%E2%80%93Hasselbalch_equation</A></P><P>Last yearâs Modeling:
</P><P>Looking at the construct and gene expression/repression  vs time or concentration of IPTG.  Levels of mRNA, protein etc. 
</P><P>Matlab Simbiology
</P><P>An Introduction to Systems Biology by Yuri Alon. Book to find some modeling stuff.
</P><P><B>Bioethics:</B></P><P>handouts on Bioethics. Phrases, wording to use with the general public in presentations. 
</P><P>one concern is bioterror
</P><P>Obama wrote a letter asking for six month analysis of synthetic biology.
</P><P>during public outreach highlight differences between synthetic biology and genetic engineering
</P><P><B>discussion:</B></P><P>bookâhow to defeat your own clone. By kurpinski and Johnson. 
</P><P>how to regulate synthetic biology
</P><P>nanotechâpicked up by the industry really quickly so the general public is pretty much just trusting the industry in 
this area. We donât really understand how it reacts with out biology.
</P><P>what kind of effects, how dangerous is a synthetically created organism? 
</P><P>bioterror, bioerror
</P><P>microcosmâby Zimmer. book on all the advances weâve gained using e.coli and a few other. Also mentions iGEM. 
</P><P>Concernsâgetting in environment, bioterror, god/religion. 
</P><P><B>Other Info:</B></P><P>GAMES camp. Engineering girls camp. High school week, middle school week. July 30th, (Friday) 9:45-10:30. August 13th 9:45-10:30. Talk for about a half hour, have a hands on activity, something they can take away with them.  
</P><P>Orpheum childrens science Museusmâreally need a hands on thing if we do this. 
</P><P>Surrounded by Science--David Leake, tv show from parkland college. Person in charge wants to make sure that thereâs 
something in it for them too. Film us doing stuff, but we would also do some kind of outreach thing at parkland. could we have some kind of bioethics discussion if itâs a seminar of some sort?
</P><P>Memorial Dayâbarbeque, sand volleyball, waterballoon fight? Illini grove?
</P><P>ThermoFischer might get us some funding!
</P><P>For the meeting tomorrow: who would be able to be the public relations team to present at GAMES camp and things like that.    Anna, Amanda, Erin.
</P><H2><SPAN class="mw-headline" id="5.2F26.2F10"><B>5/26/10</B></SPAN></H2><P><B>Bootcamp Day 3</B></P><P><B>Synthetic biology</B></P><P>âengineering microbial âpolka dotsâ toward synthetic biologyâ:
</P><P>Synthetic biology uses only sequence info in order to implement an objective biological system. Biological systems engineering employs and modifies naturally existing biological systems in order to make them behave as intended.  iGEM tends to fall into the biological systems engineering category since we take existing parts and put them together rather than coming up with a sequence.
</P><P><B>System design:</B></P><P>Parts vs device vs system
</P><P>ie: radio vs radio+speakers. 
</P><P>protein vs multiple proteins (ie: repressors) to create an inverter vs multiple interacting inverters.
</P><P><B>Principles in biological systems engineering:</B></P><P>1. Abstractionâprovide a big picture without detailed characteristics of the intended system (part, device and system)
</P><P>2. Standardizationâenable interfacing among parts, devices, and systems. Predictable/reproducible performance. (we donât have to worry about this so much since most of what we use is in biobrick format)
</P><P>3. Decouplingâefficient way to deal with complexity. Getting rid of interference.  Considering other effects of utilization of the metals
</P><P>4. What devices are needed for the system. Think about what existing devices to couple with the devices we want to make.
</P><P>5. Connect the devices needed. 
</P><P>6. Implementation, detailed design of parts needed. What promoter, RBS, terminator do we want to use? What kind of regulation?
</P><P>7. Part level system configurationâhow all the parts interact in the system.
</P><P>8.  Synthesis, compilation, DNA layout, sequence information. 
</P><P>9. Debugging the system. Looking at each part, testing if it works in the system, where something might have gone wrong if itâs not working. Individually test each part in the device by putting GFP downstream to receive output from each element and manually introduce input signal.
</P><P>10. Final output and sequence information. 
</P><P>Intial simple modeling: Think about where the system will fail. Metal toxicity, low transcription/translation activity. 
</P><P>Make sure to provide good documentation of our device. Donât necessarily have to get everything working. 
</P><P>Plan out objectives for the summer, steps for achieving the goal.
</P><P><B>Research topics:</B></P><P>1. Metal resistance
</P><P>2. Magnetosomes
</P><P>3.  Metal mobilization
</P><P>4.  Metal immobilization
</P><P><B>Wiki Editing:</B></P><P>upload picture to profile, update profile.
</P><H2><SPAN class="mw-headline" id="5.2F25.2F10"><B>5/25/10</B></SPAN></H2><P><B>Bootcamp Day 2</B></P><P><B>Primer Design</B></P><P>PCR: amplifies double stranded DNA.  Usually given positive strand (5â-3â) by databases. PCR consists of heating, cooling. Heating causes the strands to separate, allowing primers to bind. Primers are specific to a specific sequence (oligoâs of DNA).  Primers bind during annealing step, then extend around 72 C. during extension, polymerase extends 5â-3â. 
</P><P><B>Basic Design:</B>
1.	Primer aka oligo. Single stranded piece of DNA, order it from a company. Want the primer to be about 20 bp long (length of complementarity). The shorter, the cheaper.
</P><P>2.	3â GC clamp. GC has 3 H bonds holding it together, AT has 2 H bonds (so AT is a weaker binding than GC). Want the tightest interaction of the primer binding the DNA to be at the 3â end because this is where elongation is going to proceed from. 
</P><P>5â-ACTG-3â
</P><P>3â-TGAC-5â 
</P><P>3.	Melting temp (Tm) greater than or equal to 60 degrees C. The higher the better. 
</P><P>CG 4 degrees C
</P><P>AT 2 degrees C
</P><P>so if a 20-mer, 20*4 means melting temp around 80 degrees for an entirely GC 20-mer. 
</P><P>Microbesonline.orgâhas every microbial genome sequences.  Can find a gene in an organism. After inputting the organism and searching a gene, click gene for gene information. click the sequences tab for the amino acid sequence and coding sequence in FASTA format.
</P><P>ie: CheV gene from Bacillus Subtilis. 
TTGTCGTTACAACAATACGAAATTTTATTGGATTCTGGTACAAATGAATTAGAAATTGTG
AAGTTTGGCGTGGGTGAAAATGCTTTCGGAATTAACGTCATGAAGGTAAGAGAAATTATT
CAGCCTGTCGAGGTGACATCAGTGCCTCACTCCCATCAGCATGTAGAAGGAATGATTAAA
CTCAGAGGAGAAATCCTCCCTGTGATCAGTCTCTTCTCATTTTTTGGAGTAGAGCCTGAA
GGATCAAAAGATGAGAAATATATCGTGACTGAATTTAATAAACGGAAAATTGTTTTTCAT
GTCGGCTCTGTTTCTCAAATTCACAGAGTATCCTGGGAAGCGATTGAAAAGCCGACATCG
TTAAATCAAGGAATGGAGCGGCACCTTACCGGTATTATTAAGCTCGAAGACCTGATGATC
TTTTTGCCTGACTATGAAAAAATTATTTATGACATTGAATCAGATTCAGGTGTTGACACG
TATAATATGCATACCGAGGGCTTCGATGAAAGAAGAACTGATAAAAAGCTTATCATTGTA
GAGGACTCACCGCTTTTGATGCGCCTCTTGCAGGATGAATTAAAAGAAGCAGGGTACAAC
AATATCGCTTCGTTTGAAAATGGAAAAGAGGCATATGAATACATTATGAACCTTGCTGAA
AACGAAACTGATTTATCAAAACAGATTGATATGATCATCACTGATATTGAAATGCCAAAA
ATGGACGGACACAGGCTCACAAAGCTGCTGAAGGAAAATCCGAAAAGCTCAGATGTGCCG
GTTATGATTTTCTCATCGTTAATTACGGATGATCTGCGTCACCGCGGCGAAGTTGTAGGC
GCAGATGAGCAAATCAGCAAGCCTGAGATCAGTGATTTGATTAAAAAAGTGGATACGTAT
GTTATTGAATAA
</P><P>Forward Primer: 
</P><P>anneals to the first 18-25 bp of the sequence on the complementary strand (3â-5â, minus strand), matches the first 18-25 of the positive strand. 
</P><P>positions for GC clamp
</P><P>TTGTCGTTACAACAATACG
</P><P>Tm: 7 GC (7*4=28)
</P><PRE>      12 AT (12*2=24)  
</PRE><PRE>      28+24=52 degrees C
</PRE><P>Reverse Primer:
</P><P>anneals to positive strand, reverse compliment of the strand itâs binding to. 
</P><P>AATTCCGG ï  CCGGAATT
</P><P>(+ 5â-3â)GGATACGTATGTTATTGAATAA ï  TTATTCAATAACATACGTATCC
</P><P>Tm:  6 GC (6*4=24)
</P><PRE>       16 AT (16*2=32)
</PRE><PRE>       32+24=56 degrees C
</PRE><P>Go to IDT website, DNA synthesis, custom DNA oligos. Order the smallest amount of DNA. Standard desalting purification. Copy in sequence into sequence box, give it a name. 
</P><P>enter primers into googledoc 
</P><P>NB: Using courier font makes the spacing of all the characters equal.
</P><P>Primer Extension
</P><P>Restriction Enzymes for biobricks: PstI, SpeI, EcoRI, XbaI
</P><P>make sure the gene doesnât have the cut sites in that weâre usingâNEB cutter, <A href="http://tools.neb.com/NEBcutter2/" class="external free" rel="nofollow">http://tools.neb.com/NEBcutter2/</A> enter in FASTA format. Tells you all the cut sites in the gene. Click link for 0 cutters to see which enzymes wonât cut in your gene. These are the enzymes you can use for biobrick assembly. 
</P><P>adding restriction enzyme sites to our primers. Adds restriction sites to our PCR product. Donât want to go above 50bp total for primer length (20 bp for binding, 30 for restriction site). Always add to the 5â end. 
</P><P>EcoRIâGATTC
</P><P>SpeIâACTAGT
</P><PRE>    Forward: GAATTC+primer sequence+some extra bases to help the site work.  ATCGGAATTCTTGTCGTTACAACAATACG
</PRE><PRE>    Reverse: ACTGACTAGTTTATTCAATAACATACGTATCC
</PRE><P><B>Designing a biobrick</B></P><P>need ribosome binding siteâgenerally AGGAGG, 8-12 bases upstream of the ATG (start codon). Take 20 bp from upstream of a well expressed gene to use as RBS. Ex: CheY is well expressed.
</P><P>20 bp upstream of CheY is atttaaaataacgaaacacaaggagagagatagatt
</P><P>
atttaaaataacgaaacacaaggagagagatagattâATGâCheVâTAAâTAA   
(2 stop codons just in case)
</P><P>Forward:
</P><P>ATCGGAATTCatttaaaataacgaaacacaaggagagagatagattATGTCGTTACAACAATACG
(changed TTG to ATG at beginning of coding sequence for better RBS)
</P><P>Reverse: added additional stop codon
</P><P>ACTGACTAGTTTATTATTCAATAACATACGTATCC
</P><P><B>Point Mutations:</B></P><P>Quickchangeâstart with a methylated plasmid. 2 primers that mind over the base pair you want to change with the only difference being at the bp to change. PCR , treat with DpnI (4 bp cutter for methylated DNA). Leaves us with only unmethylated, altered DNA. Canât use Taq pol, have to use purified primers.
</P><P>Enzyme inverse PCR
</P><P>Primer3: does calculations for you, designs primers for you.   <A href="http://frodo.wi.mit.edu/primer3/" class="external free" rel="nofollow">http://frodo.wi.mit.edu/primer3/</A></P><P>
NCBIâcan search for entire genomes:
</P><P>Ptt fileâgives you ever gene in a genome in standard form.
</P><P>fna fileâsequence of genome.
</P><P>gbk fileâreferences, updates, amino acid sequence, genome sequence in block format. 
</P><P>Colibri: can type in gene name, get info, sequence separated by codon and user defined upstream, downstream regions.
</P><H2><SPAN class="mw-headline" id="5.2F24.2F10"><B>5/24/10</B></SPAN></H2><P><B>Bootcamp Day 1</B></P><P><B>Weekly fundraiser:</B></P><P>ideas: T-shirts, Puppy chow, recycling printer cartridges, donate plasmaâ¦organsâ¦, car wash, make a band,  pizza sales, grilled cheese.
</P><P><B>Positions:</B></P><P>keeping track of lab hours: spreadsheet on google docs, input number of hours and at the end of the week, a couple of sentences about what you did that week. Also a sheet of paper in the lab to sign in and sign out. 
</P><P><B>goals, structure:</B></P><P>weekly meetings with the professors on Friday
</P><P>be done with decoder by second week of June, have it tested, modeled by end of June. 
</P><P>submit project description by June
</P><P>submit abstract by August
</P><P>spend next 2.5 weeks figuring everything out, research, design project. Sequences weâll need for each plasmid in the metals project. 
</P><P>for full timeline, see googledoc. 
</P><P><B>lab notebooksârecord keeping, wiki editing</B></P><P>keeping the notebook:  keeping notes in triplicatesâin-lab notebook in as much detail as possible. Lab notebook needs EVERYTHING. As much info as possible. Title, procedure, materials, date, time, initial at bottom. Kit used, concentration, etc. Label every lane in the gels. Always take a picture of the gel, put it in the in lab notebook. Save a copy on the computer.   In your own notebook, format however you want, general notes. Notes will also be kept on the wiki, daily log. Not very extensive, but general info. Ie: I did a transformation today, it worked, or it didnât work. 
</P><P><B>wiki:</B>
Matt types the info from the in-lab notebook into the wiki.
</P><P><B>Past iGEM projects:</B></P><P><B>Metals:</B> Denmark 2009, cornell 2009âcadmium sensor in bacillus, Groningen 2009âcell that accumulated arsenic, floated when arsenic present,Newcastle 2009âbacillus cadmium sensing, and gate, metallothionein binds cationic metal ions, Seoul 2009âpromoters for different metals, Tokyo Tech 2009âiron oxidizing bacteria, takes iron ore, brings into cell,
UQ Australia 2009âtake a look at this one, Used e. coli to take up mercury, thave the pathway mapped out, good documentation, also did something with bioprecipitation, Virginia 2009âArsenic accumulation, Imperial 2008, LCG UNAM Mexico 2008,METU Turkey 2008,Prarieview 2008,St. Petersberg 2007, Brown 2007, MIT 2007, Prarieview 2007, Turkey 2007, Edinburgh 2006, Latin America 2006
</P><P><B>RNA:</B> British Columbia 2009, Bologna 2009, Victoria BC 2009, Alberta 2008, KLeuven 2008, TUDelft 2008, Virginia 2008, Berkeley 2006
</P><P><B>groups:</B> half on decoder, half on metals. Matt, Steve, Francis, Bob on small RNAs. Amanda, Tom, Meagan, Erin on metals. 
</P><P>use transcription factors and small RNAs to make the decoder work. 
</P><P><B>Other notes:</B></P><P>everyone gets 2 weeks vacation.
</P><P>synthesizing vs. cloning genes out:
</P><P>build a schematic of what we want to do. 
</P><P>need transporter protein, metal chelator/sequesterer. 
</P><P>gold and silverânot finding a lot of info. May have to make the pathway ourselves. Expensive
</P><P>Mercuryâpast iGEM project
</P><P>Ironâeasy
</P><P>Nickle
</P><P><B>In lab procedure:</B></P><P>miniprepâbiobrick number on top of tube, date, concentration on the side of the tube. Initials if room.
digestion with restriction enzymes. See online instructions from gingobioworks website. Make sure to add enzymes last.
</P><P>incubate
</P><P>ligationâadd ligase last. 
</P><P>incubate
</P><P>transformation (electroporation)
</P><P>Selection
</P><P>Assays
</P><P>Miniprep and colony PCR
</P><P>Put parts in registry as practice. 
</P><P>Find something to reduce or oxidize a species
</P><H2><SPAN class="mw-headline" id="4.2F28.2F10"><B>4/28/10</B></SPAN></H2><P>Decoder to detect concentration rangeâabove/below limits assign 1 or 0. Can use decoder to tell what range of concentration youâre in. if in normal range, no response. If upper range, produce insulin or something. If below, promote gluconeogenisis. 
</P><P><B>Biomineralization:</B>  Silver-metal alloys not from e.coli but from organism physiologically similar to e. coli. Relatively characterized. Should work since theyâre similar organisms.  Works for decoder if we can get alloys to work.  Doesnât exploit RNA part of decoder. RNA produces quick response. Bring metal in, chelate it, reduce it, complex it, determine extracellular location. Probably need 1 porin for each metal. 
</P><P><B>Biosurfactants:</B> part of crude oil refinery. Changes the hydrophobicity of the area of the cell. Used for heavy metal crystallization, gold. bioemulsifiers. Work for a bunch of stuff. Surfactant: absorb at surfaces by modifying chemical interactions. Basically detergents, ie: soap. Emulsifier: makes oil and water mix. Important in mining and oil. Lots of uses in industry. Lots of material, information. two systemsâbiosurfactant, bioemulsifier. Make cell that produces these. All cells have these properties to some degree. Heavy metal removal from sediments with biosurfactants.   Project would be using biosurfactants and bioemulsifiers for oil refining, possible biomineralization. Used to purify components in oil spills. Surfactants enhance solubilization. Inhibits corrosion. 
Quantum dots: organisms produce specific crystallized metals, put to membrane, âgreenâ way of producing crystals for nanotechnology. Nothing with any characterization. 
</P><P><B>Decision:</B> Biomineralization applied to decoder
</P><H2><SPAN class="mw-headline" id="4.2F25.2F10"><B>4/25/10</B></SPAN></H2><P>Need to define project!!
</P><P><B>Oil:</B></P><P>Benzo[a]pyreneâone bacteria can degrade it if they have a second source of energy.
</P><P>Emulsificationâincrease surface area for oil degredation. Lots of current focus in this area. 
</P><P>Also trying to develop fertilizers for bioremediation of oilâcontain nitrogen and phosphate, emulsifiers to help the bacteria degrade oil.  
</P><P>Usually quorum dependent secretion of emulsifiers
</P><P>Keywords: oil bioremediation, oil bioaugmentation,Aromatics, aliphatic hydrocarbon chains. 
</P><P>Most bacteria can only degrade a select few, not a variety of compounds. Aromatic rings are carcinogenic. 
</P><P><B>Metals:</B></P><P>One of the problems with metals is that itâs difficult to clone out the necessary componentsâlots of things needed to make it work.  Sulfur iron clusters in many of the proteins, many enzymes going into making the actual cluster which is the catalytic core of the enzyme.  Metals usually as terminal electron acceptorâinvolved in electron transport chain, use lots of cytochromes which are different in all organisms. Youâd need to get all the cytochromes too to make the pathway work.
</P><P>One pathway, reduces selenium, uses a few enzymes, only three to construct the reductase. Pathway completely mapped out. 
</P><P>A lot of secretion, biominerilization occurs outside the cell. Some organisms can do this inside the cellârequires less enzymes, but hard to amass them into large quantities. But could sonicate cells to release the metals. 
</P><P>Quantum Dots: related to semi-conductors. Nanocrystals of different metals, conductive properties based on the molecular arrangement of the crystals.  Used for nano-technology. Grouping electrons to use for âquantum computingâ. 
</P><P>E.coli can reduce gold.
</P><P><B>Other stuff:</B></P><P>Send in some grants, submitting project proposal sometime this week. Weâre going to stay with the decoder but find some application if possible.
</P><P>Meeting Wednesday night, 20 min, officially have a project. 9:30, RSO. 
</P><H2><SPAN class="mw-headline" id="4.2F23.2F10"><B>4/23/10</B></SPAN></H2><P>Summer meetings, bootcamp overview (already covered in previous team meeting)
</P><P>Metals: May not be as easy of just transplanting the pathwayâsecrete cytochromes to grab the metal, shuttling system. Check pathways, could be difficult.  
</P><P>Maybe looking at oilâoil had different compositions, increase efficiency to degrade only whatâs in the specific spill.  Oil classified by sulfur and viscosityâfind something thatâs Sulfur sensitive.
</P><P>Documentary possiblyâI think I may have missed some info on this? 
</P><P>Biosafety: which strains (DH5a), safety precautions, emergency contacts (mckinley). Basically listing what youâre working with. finish hopefully before we get into bootcamp week. 
</P><P>Competition: nov. 5th-7th MIT, planning on 2 advisors coming as of now, Courtney, Dr. Rao. 
</P><P>IGB access: 9-5 can get in no problem, no need for prox card. Has to be someone here if weâre here at night, we canât be alone in the lab. Not likely thatâd weâd get afterhours access. Weekend, keep it within normal hours, 12-6ish.  
</P><P>Oil has bigger implicationsâindustries looking for ways to take care of oil. Look into aromatics, alkanes, alkenes, paraffin, olefin, biodegradation. 
</P><P>Get the decoder to work!
</P><P>Build each detector separately, put into one system at the end. 
</P><H2><SPAN class="mw-headline" id="4.2F21.2F10"><B>4/21/10</B></SPAN></H2><P>Biomineralization metal sensing, instead of a promoter, precipitate the metal. Short pathways. Pathways that detoxify. 4-5 enzymes per metal pathway. Sensing stuff in registry. Inputs are soluble metals.
</P><P>Nanofabrication.
</P><P>How could it help people: precipitated form is non-toxic, while soluble is toxic.
</P><P>Input: soluble metal, pick out clump of dirt on the  quad, has soluble metal.
</P><P>Really marketable, direct application, no human testing.
</P><P>Thereâs metal everywhere, donât need a lot to do this.
</P><P>Decoder: if gold, pathway to precipitate gold. If silver inputs, precipitate both. Etc.
</P><P>meeting with advisors friday, 12:00 third floor, north side conference room IGB.
</P><H2><SPAN class="mw-headline" id="4.2F16.2F10"><B>4/16/10</B></SPAN></H2><P>Project proposals:
</P><P><B>1.	Matt, Steve, Bob:</B></P><P>Bacterial Thermo-regulator:
</P><P>	Get bacteria to maintain homeostasis in fluctuating external temperature
</P><P>	Introduce genes coding for temperature sensitive genes or RNA to act as promoters to start a process to release a large amount of heat.
</P><P>	Applications: for high temp. microbes, applications to thermophiles in industrial applications, central heating, anywhere heat is used.
</P><P>	Problems: enzyme stabilityâreduction of Nitrous oxide. Does it diffuse into cells? How do we get it into cells? Metabolic rate? 
</P><P>	cspA mRNA changes conf. at low temp. (temperature sensitive RNA). Responds to cold shock. Cis regulator. Changes secondary structure depending on the temp. of the environment. Check Kyoto projectâwanted bacteria to raise temp. of mars so life could be maintained. 
</P><P>	N2O reductionâreleased 82.05 kJ/mol, only need one enzyme (nitrous oxide reductase). 
</P><P>	Feasibility: strength of the promoter would have to coincide well with metabolic rate, trial and error to find right balance.
</P><P>	Cooling: to maintain homeostasis, would need to find way to cool the bacteria. Maybe reverse the process weâd be using to generate heat. Require another set of genes, promoter. Heat sensitive lambda promoter in registry.  Exocytosis to cool the cell, similar to how mammals maintain homeostasis, sweating. 
</P><P>	Insulatorâcapsule?
</P><P>	Comments: e.coli has advanced heat shock system. Many proteins to stabilize, involves mRNA. Cold just slowing down rate of reactions. Hard to get bacterial density up in a solution to generate enough heat. Possible spin to try to evolve a thermophilic version of e.coli, then sequence genomes. But there wouldnât really be a biobrick.
</P><P>	Insulator/capsule under temperature dependent promoter is feasible. 
</P><P>	Evolution at specific temperatures.
</P><P><B>2.	Amanda, Meagan:</B></P><P>Multiple contaminant whole cell biosensor in e.coli
</P><P>Builds off bacterial decoder from last year
</P><P>Water contamination techniques. Detection techniques today are not always reliable. 
</P><P>whole cell biosensor: sensing component (usually resistance gene) reporter gene. 
</P><P>biosensors have been done in the past but only single input and single output.  Ours would take multiple inputs to give an output based on the input.
</P><P>arsenic, lead.
</P><P>ars operonâresistance gene, pump to pump out arsenic. Couple it to a reporter gene
</P><P>pbrrâprotein that binds selectively to lead ions. Clones into e.coli before so feasible. 
</P><P>theoretically simple, practical application of last yearâs project. 
</P><P>problems: need to make it more sensitive than it is in nature for the ars operon. Not entirely sure what the products of the ars operon are. Donât know much about lead sensing component. Need appropriate inputs, research what compounds are in water, whatâs feasible to detect.
</P><P>past projects: the traffic light projectâaltered promoter such that itâs stronger or weaker than wild type. Possible way to sense different levels of contaminants.  We could alter ars promoter in similar way.  
</P><P>increasing sensitivity.
</P><P>Comments: not exactly a true application of last yearâs decoder, maybe have it detect something beneficial vs something dangerous like ars. Or make a degrader instead for endocrine disrupters or estrogen. Pathogen contamination, parasites.
</P><P>Didnât get anything for synthetic RNA project b/c I was presenting
</P><P><B>Post-presentation comments:</B></P><P>Synthetic RNAâeasy to manipulate but not necessarily an effective regulator. No real definite application.
Continuing the decoder:  weâd have to find a direct application, something that needs to be decoded. Or maybe even just get it to work since we have the parts. New inputs/outputs. Degrade contaminants. 
</P><P>We chose the decoder!
</P><P>Think of what we can add to it, what we can decode, degradation of estrogen and endocrine disrupters. Still gonna use small RNA, 2 to 4 decoder
</P><H2><SPAN class="mw-headline" id="4.2F11.2F10"><B>4/11/10</B></SPAN></H2><P>Meeting with advisors to present project proposals Friday 5:00-6:30. Possibly a professional picture 4:45 for the brochure.
</P><P>Francis went over project proposal research. See the googledoc for comments.
</P><P>Top three projects to present:
Thermoregulator
</P><P>Arsenic/lead detector
</P><P>Engineering Posttranscriptional regulation in prokaryotes
</P><P>Additional research throughout the week on these projects.
</P><P>
What's Going on this week:
</P><P>Thursday: Undergraduate Research Symposium 1:30-2:45 Pine Lounge in the Union
</P><P>Friday: Meeting with Advisors 12:00-1:00, Igb fellows symposium 3:30-4:45, Professional Picture 4:45, Project Meeting 5:00-6:30. 
</P><P>see the google calendar for more info
</P><H2><SPAN class="mw-headline" id="4.2F10.2F10"><B>4/10/10</B></SPAN></H2><P>General:
</P><P>SparkâOld article and contacts, but still reachable
</P><P>MCB Open HouseâNext week
</P><P>Undergraduate Research SymposiumâNext week
</P><P>Lab:
</P><P>Moved into 3500 IGB
</P><P>Weâll just buy glasswareâwe can get what we need as we need it
</P><P>Project Overviews with advisors
</P><P>Meeting next FridayâTeam picture at 4:45, meeting 5-6:30
</P><P>Grants
</P><P>Company Contact Listâadding non-profit organizations
</P><H2><SPAN class="mw-headline" id="3.2F19.2F10"><B>3/19/10</B></SPAN></H2><P>Meeting with advisors
</P><P>One advisor said something about iGEM for regional advisors meeting for proposal. Courtney had meeting with ihotel for planningârooms, dates. We may have to host dinner, possibly at IGB. We have to come up with a portion of the funds. Needs list of team memberâs school affiliation. Tentative schedule: First night cocktail party, dinner. Conference at iHotel. We pretty much just pay for food.  First day box lunch, possible dinner. Sunday, offer breakfast, people leave. Estimating 100 people attending.  Maybe poster session highlighting IGB work, vendor show (vendors pay to participate, location source of funding for iGEM!).
</P><P>Maybe the union as an option, visitors can walk around campus. Hampton out by Beckman. 
</P><P>We were given 500 dollars to come up with cooperate brochure. Who we are, description of research, donation possibilities.  
</P><P>We may need new space if our current space gets taken. Possible use of training lab. If we go with training lab, weâll have to find equipment list.  Ie: pipette sets, power supply, thermocycler, tabletop centrifuge, hot plate, glassware, gel boxes. Maybe borrow stuff from senior design lab from bioengineering.
</P><P>Put together list of supplies that we need donated. We can ask around to see what we can get donated.
</P><P>Lab training sessions early may or late june with prof. Rao. Courtney may be able to do some lab training in a couple weeks as well. 
</P><P>Advisor Availability this summer:
Daycare on campus closes for 2 weeks in summer, so Courtney wonât be here then.
Prof. Rao: june will be on and off
Prof. Jin: Second and third week of june out of town.
Prof. Manaster has to look stuff up, sheâll get back to us.
prof. Rao wants some of our smelly e. coli. 
Credit Hours possibilityâpossibly getting fall credit for summer work so tuition isnât an issue.  Probably 4 hours of credit.
</P><H2><SPAN class="mw-headline" id="PDFs:"><B>PDFs:</B></SPAN></H2><P><A href="https://static.igem.org/mediawiki/2010/f/f8/UIUC-Bioware_4-9-2010_Profs_Meeting.ppt" class="external text" rel="nofollow">4.9.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/1/1d/3.14.10_UIUC_bioware_meeting_minutes.pdf" class="external text" rel="nofollow">3.14.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/2/22/UIUC-Bioware_Meeting_Minutes_2-16-2010.pdf" class="external text" rel="nofollow">2.16.10</A></P><H2><SPAN class="mw-headline" id="Powerpoints:"><B>Powerpoints:</B></SPAN></H2><P><A href="https://static.igem.org/mediawiki/2010/4/46/UIUC_8.6.10week_11_.ppt" class="external text" rel="nofollow">8.6.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/a/a9/UIUC_IGEM_Bioware_Advisors_week_9.ppt" class="external text" rel="nofollow">7.23.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/6/68/UIUC_IGEM_Bioware_Advisors_week_8.ppt" class="external text" rel="nofollow">7.16.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/0/01/UIUC_7.9.10_meeting_minutes.ppt" class="external text" rel="nofollow">7.9.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/b/ba/UIUC_IGEM_Summer_Week_6.ppt" class="external text" rel="nofollow">7.2.10</A></P><P><A href="https://2010.igem.org/Image:UIUC-Bioware_6.25.10_meeting_powerpoint.ppt" class="external text" rel="nofollow">6.25.10</A></P><P><A href="https://2010.igem.org/Image:UIUC-Bioware_6.4.10_meeting_powerpoint.ppt" class="external text" rel="nofollow">6.4.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/7/75/UIUC_Summer_Week_1_Professors-1.ppt" class="external text" rel="nofollow">5.28.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/a/a6/UIUC_IGEM_Advisors_Meeting_4-23-2010.ppt" class="external text" rel="nofollow">4.23.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/2/21/4.18.10_UIUC_Bioware.ppt" class="external text" rel="nofollow">4.18.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/2/2a/UIUC-Bioware_IGEM_4-2-2010_Meeting%21.ppt" class="external text" rel="nofollow">4.2.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/c/ca/Team_Meeting_3-28-2010_UIUC_Bioware.ppt" class="external text" rel="nofollow">3.28.10</A></P><P><A href="https://2010.igem.org/Image:UIUC-Bioware_IGEM_Meeting_3-19.ppt" class="external text" rel="nofollow">3.19.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/4/4d/UIUC-Bioware_Advisor%27s_update_from_3-2-2010.ppt" class="external text" rel="nofollow">3.2.10</A></P><P><A href="https://static.igem.org/mediawiki/2010/5/5f/UIUC-Bioware_IGEM_2010_Project_Ideas_for_Profs._on_2-2-10.ppt" class="external text" rel="nofollow">2.2.10</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UIUC-Illinois/Meeting_Minutes">http://2010.igem.org/Team:UIUC-Illinois/Meeting_Minutes</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UIUC-Illinois/Meeting_Minutes" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UIUC-Illinois/Meeting_Minutes" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UIUC-Illinois/Meeting_Minutes&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UIUC-Illinois/Meeting_Minutes&amp;oldid=109794" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>