<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Paris-Saclay_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry Paris-Saclay/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM Evry Paris-Saclay</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN> Menu </BUTTON><A class="navbar-brand page-scroll" href="#page-top">
iGEM Evry Paris-Saclay </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay">Home</A></LI><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="#">Project </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Applied_Design">Project Description</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Chemistry">Chemistry</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Measurement">Biosensor</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Demonstrate">Screening</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Bioproduction">Bioproduction</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Model">Modeling</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Part">Parts</A></LI></UL><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Notebook">Notebook </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Lab">Lab</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Notebook">Labbook</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Experiments">Protocols</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Medals">Medal Criterias</A></LI><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="#">Human Practice </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Human_Practices">Human Practices: Front Page</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/HP/Silver">Human Practices: Silver</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/HP/Gold_Integrated">Integrated Human Practices: Gold</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Engagement">Education &amp; Public Engagement</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Collaborations">Collaborations</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/InterLab">Interlab</A></LI><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="#">About Us </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Team">Team</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Attributions">Attributions</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Sponsors">Sponsors</A></LI></DIV></DIV><SECTION class="fullSize" id="about"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 text-center"><H1 class="section-heading content-title">Screening</H1><P class="text-left">Having finished building the biosensor, we were able to efficiently determine psicose concentration. In order to demonstrate the feasibility of our project, we then needed to improve our sugar production by using our biosensor to select the best enzyme from a bank of D-psicose 3-epimerase (Dpe) mutants in order to maximize production. </P><P class="text-left">Enzyme engineering currently relies on computation followed by directed mutagenesis on specific area of the protein. This maximizes the probability of improving activity for a defined number of mutants. But this approach restricts possible random conformational changes, with the potential to improve catalytic sites. But random mutagenesis requires the researcher to screen a much larger number of mutants, hence the need to use an efficient screening system, like the one we designed. </P><P class="text-left">To achieve this, we first had to engineer our biosensor to allow insertion of the mutants into the vector, then construct the bank, and finally screen all the clones.</P><H2 class="section-heading content-title">Mutant Drop Zone</H2><P class="text-left">Relying on the expertise we developed on Golden Gate assembly during the course of our project, we chose to use this strategy to insert our mutants into the plasmid. For this, we engineered the Mutant Drop Zone (MDZ). This short sequence (320 bp) works following the same principles as the <STRONG>Universal Biosensing chassis</STRONG> (<A href="http://parts.igem.org/Part:BBa_K2448023" target="__blank">BBa_K2448023</A> or <A href="http://parts.igem.org/Part:BBa_K2448024" target="__blank">BBa_K2448024</A>): standardized fusion sites for Golden Gate that allow quick and efficient insertion. The mutant drop zone can also be used with traditional restriction-ligation but loses its high throughput ability. </P><P class="text-left">The mutant drop zone is designed to fit between the second terminator of our biosensors and the iGEM suffix (Figure 1). </P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 1.</STRONG> Schematic design of the psicose biosensor pPsiA-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) with downstream the Mutant Drop Zone.</FIGCAPTION></FIGURE><H2 class="section-heading content-title">Construction of the Enzyme Mutants Bank</H2><P class="text-left">There are many methods available to build a bank of enzymatic mutants, depending on what you are looking for. We chose to use Error Prone PCR because it favors mutations during the elongation phase, thanks to a mutagenic buffer (for example imbalance in dNTPs concentrations) and low fidelity polymerases. This technique remains more efficient than chemical methods which rely on reagents to modify the sequence and is also safer for the user, as chemical mutagens are highly toxic. Moreover, it is an <EM>a priori</EM> free method compared to saturating mutagenesis like NNK.</P><P class="text-left">To build the bank of D-psicose 3-epimerase from <EM>C. cellulolyticum</EM> (<A href="http://parts.igem.org/Part:BBa_K2448021" target="__blank">BBa_K2448021</A>) mutants, we used the protocol described by Wilson &amp; Keefe [1]. This PCR has been performed on <EM>C. cellulolyticum</EM> Dpe gBlocks provided by IDT. Using custom primers, we were able to obtain variants with a theoretical average of 8 amino acids mutated according to [1], number of cycles and length of our sequence. We also performed a high fidelity PCR on the gBlocks in order to get our positive control, the non-mutated Dpe.

</P><P class="text-left">We then inserted the sequences in the mutant drop zone of the Psicose biosensor pPsiA-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) with downstream the Mutant Drop Zone using Golden Gate assembly since BsmBI sites were present in the overhangs of the primers (Figure 2). We transformed the Golden Gate product into <EM>E. coli</EM> K12 DH5α, even though it is not a strain designed for protein expression, it’s a well-known strain and it has been used to characterize the biosensors.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 2.</STRONG> Schematic design of the psicose biosensor pPsiA-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) with downstream D-Psicose 3-epimerase from <EM>C. cellulolyticum</EM> (<A href="http://parts.igem.org/Part:BBa_K2448021" target="__blank">BBa_K2448021</A>).</FIGCAPTION></FIGURE><H2 class="section-heading content-title">Screening</H2><P class="text-left">The screening of the bank has been performed following the protocol described in the ‘Protocol’ session of our wiki. Essentially, we cultivated all the clones present on the plates after the transformation and used a 96 well plate along with a plate reader to measure the fluorescence in each well. All tests were performed in technical duplicates. Fluorescence measurements (mCherry) have been normalized on cell density (OD<SUB>600nm</SUB>). In order to best evaluate enzyme activity, we have determined the optimal concentration of inducer, here the fructose (<A target="__blank" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Bioproduction">See details on the Bioproduction Part</A>). Therefore, all screening process have been conducted with 50 g/L of fructose per well for 9 to 10 hours as this is the optimal measurement time according to our biosensor characterization (<A target="__blank" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Measurement">See details on the Biosensors Part</A>).</P><P class="text-left">According to raw data, fluorescence of the positive control containing the wt Dpe (<A href="http://parts.igem.org/Part:BBa_K2448035" target="__blank">BBa_K2448035</A>) is around 4500 arbitrary units. Biosensor characterization allowed us to correlate fluorescence and psicose concentration. Then we can estimate that psicose produced by the wt Dpe enzyme to be around 10 g/L which is coherent with measurements perform in bioproduction (<A target="__blank" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Bioproduction">See details on the Bioproduction Part</A>).</P><P class="text-left">Figure 3 presents the outcome of our screening process. The most important observation is that 6 mutants showed an activity superior to the positive control, with 2 of them displaying more then 30% increase of psicose production. This represents 1,5% of the bank population, which is a very promising rate. As expected, the majority of the clones carry mutants with relatively low activity compared to wt Dpe, most of them showing an activity between 10 to 25% of wt and 46 clones exhibiting at least 75% of wt activity.
					</P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 3.</STRONG> In vivo characterization of D-psicose 3-epimerase mutants using pPsiA-PsiR biosensor from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) in <EM>E. coli</EM> DH5α. The total number of mutants screened were 367. All the data points are Fluorescence/ OD of each mutant normalized by the same value from controls (wt). The graphs show the repartition of all the mutants function to their relative activity compared to wt (A) and the repartition of the mutant in range of activity (B).</FIGCAPTION></FIGURE><P class="text-left">The results in Figure 3 show that our screening process works under realistic conditions. Indeed, the biosensor we developed is able to give accurate assessment of the psicose production on positive control clones carrying the wild-type enzyme, but also on bacteria carrying mutant versions of Dpe. This screening process also allowed us to screen around 400 mutants and discover at least 2 enzymes with potentially improved activity according to the fluorescence data.</P><H2 class="section-heading content-title">Perspectives</H2><H3 class="section-heading content-title">Short Term Perspectives</H3><P class="text-left">From this point, we should recover the plasmid in order to sequence the mutant enzymes, and extract the sequence by PCR to put it in an expression vector. This would allow us to thoroughly assess its activity (<A target="__blank" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Bioproduction">See details on the Bioproduction Part</A>).</P><H3 class="section-heading content-title">Longer Term Perspectives</H3><P class="text-left">In order to bring this process to its full potential a much larger number of clones would be required to be screened. There are also other possibilities to improve on the process.
First, this process could be strongly improved with the use of a flow-cytometer in order to quickly and efficiently sort the cells depending on their fluorescence. The use of such a device would empower this protein engineering method and make it high throughput.
Then, running through more cycles of screening and bioproduction, to improve the D-Psicose 3-epimerase, would allow us to select mutants with higher activity, reaching unprecedented biocatalysis yields.
</P><P class="text-left">Finally, the whole method could be transferred and applied to improve any enzyme harnessing the power of the UBC. Thus enabling the quick assembly of a biosensor able to detect, directly or indirectly, the compound of interest, whether it’s a reaction product or a substrate.</P><H2 class="section-heading content-title">References</H2><UL class="list-unstyled text-left"><LI> [1] Wilson DS, Keefe AD. Random Mutagenesis by PCR. Current Protocols in Molecular Biology (2001) 51, 8.3.</LI><DIV class="row"><DIV class="col-lg-6 text-center"><H3><A class="content-title" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Measurement"> Biosensors</A></H3></DIV><DIV class="col-lg-6 text-center"><H3><A class="content-title" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Bioproduction">Bioproduction </A></H3></DIV></DIV></UL></DIV></DIV></SECTION><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 text-center"><H2 class="section-heading">Let's Get In Touch!</H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>