<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Paris-Saclay_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry Paris-Saclay/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>IGEM Evry Paris-Saclay</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN> Menu </BUTTON><A class="navbar-brand page-scroll" href="#page-top">
iGEM Evry Paris-Saclay </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay">Home</A></LI><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="#">Project </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Applied_Design">Project Description</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Chemistry">Chemistry</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Measurement">Biosensor</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Demonstrate">Screening</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Bioproduction">Bioproduction</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Model">Modeling</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Part">Parts</A></LI></UL><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Notebook">Notebook </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Lab">Lab</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Notebook">Labbook</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Experiments">Protocols</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Medals">Medal Criterias</A></LI><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="#">Human Practice </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Human_Practices">Human Practices: Front Page</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/HP/Silver">Human Practices: Silver</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/HP/Gold_Integrated">Integrated Human Practices: Gold</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Engagement">Education &amp; Public Engagement</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Collaborations">Collaborations</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/InterLab">Interlab</A></LI><LI><A class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" href="#">About Us </A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Team">Team</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Attributions">Attributions</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Sponsors">Sponsors</A></LI></DIV></DIV><SECTION class="fullSize" id="about"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 text-center"><H1 class="section-heading content-title">Biosensors</H1><P class="text-left">Nowadays, protein engineering relies on direct or indirect screening methods to sort potentially improved enzymes from all the mutants of a bank.  Among these methods, the most accurate performs direct sensing and quantification of the compound of interest, but they remain expensive and require large devices such as High Performance Liquid Chromatography (HPLC), immunoassays or Mass Spectrometry. Devices that not every lab can afford since you need not only the resources to buy the devices, but also highly-skilled collaborators.</P><P class="text-left">To improve the screening process of mutant banks, we chose to use biosensors for their cost-effectiveness, and efficiency in assessing a chemical’s concentration, therefore relative enzyme activity. </P><P class="text-left">But first, what are biosensors?</P><P class="text-left">Biosensors can be thought of as any biological device capable of sensing: molecules, ions or physical parameters such as pH, temperature, magnetic field or pressure. Even though sniffer dogs from a k9 unit or carrier pigeons with their magnetite beak could be considered as biosensors, this term generally refers to particular genetic circuits in the field of synthetic biology. </P><P class="text-left">Biosensors are used for a wide range of applications, from medical diagnosis [1] to metabolic engineering and have recently begun being used in cell free systems. But they usually operate in a similar manner: enabling expression of colorimetric, fluorescent or selection molecules in response to the presence of the desired compound. This is the behavior that allows their use in quantification or screening methods [2].</P><P class="text-left">We can find many types of biosensor architectures, but the ones most used in biotechnology are the transcription factor based biosensors (TFB). TFB natively exist in all organisms, bacterial two-component signaling systems being a perfect example. These types of biosensors are particularly useful for screening processes since they provide appropriate dynamic range and sensitivity [3]. Transcription factor based biosensors are usually built with well characterized transcription factors (like OxyR for hydrogen peroxide) but you can dig for ‘your favorite’ transcription factor in databases such as <A href="http://www.transcriptionfactor.org" target="__blank">DBD</A> or <A href="http://regprecise.lbl.gov" target="__blank">RegPrecise</A> where there is a huge amount of predicted transcription factors that can be used to develop biosensors for ‘your favorite’ compound.</P><H2 class="content-title">Design &amp; Construction</H2><H3 class="content-title">Transcription Factors and Promoters</H3><P class="text-left">In order to build a functional transcription factor based biosensor, able to assess ‘our favorite’ compound D-Psicose, we needed a transcription factor and an inducible promoter. Searching databases like RegPrecise, we found the PsiR of Rhizobiales. PsiR is a predicted LacI family transcription factor with high affinity for D-Psicose. This implies that PsiR is potentially capable of binding a consensus sequence in the promoter region and preventing transcription of the regulated promoters in the absence of D-Psicose, in a similar manner to the way LacI does in the absence of allolactose (or the synthetic IPTG). </P><P class="text-left">PsiR occurs naturally in four different Rhizobiales species (<EM>Agrobacterium tumefaciens</EM>, <EM>Rhizobium leguminosarum</EM>, <EM>Sinorhizobium fredii</EM>, <EM>Sinorhizobium meliloti</EM>) where it regulates an operon while also self-regulating its expression. In all these species, the genetic context is very similar: the PsiR gene precedes the operon (that starts with the PsiA gene) and is in the opposite direction (as an example, see this <A href="https://goo.gl/E2PGxU" target="__blank">region in the <EM>Agrobacterium tumefaciens</EM> genome</A>). Thus, the promoter regions of PsiA and PsiR genes are overlapping and face the opposite direction.</P><P class="text-left">Using the <A href="http://linux1.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb" target="__blank">BPROM webserver</A>, we identified the -10 and -35 boxes that are in very close proximity to the predicted PsiR binding sites (the full annotations can be found in Parts Registry pages of these promoters (<A href="http://parts.igem.org/Part:BBa_K2448010" target="__blank">BBa_K2448010</A>, <A href="http://parts.igem.org/Part:BBa_K2448011" target="__blank">BBa_K2448011</A>, <A href="http://parts.igem.org/Part:BBa_K2448012" target="__blank">BBa_K2448012</A>, <A href="http://parts.igem.org/Part:BBa_K2448013" target="__blank">BBa_K2448013</A>, <A href="http://parts.igem.org/Part:BBa_K2448014" target="__blank">BBa_K2448014</A>, <A href="http://parts.igem.org/Part:BBa_K2448015" target="__blank">BBa_K2448015</A>). Knowing this, we extracted the 400 bp sequence upstream of PsiA and PsiR genes in each species in order to get 8 different biosensors theoretically psicose sensitive.</P><P class="text-left">Since PsiR is a LacI family transcription factor, we chose to engineer a well-known LacI regulated promoter, pTacI, in order to make a minimal but strong psicose inducible promoter [4]. With the help of databases and literature, we identified a 20 bp consensus sequence on which PsiR binds and replaced the LacO site of pTacI by the consensus sequence, hence creating a new promoter: pPsiTacI (<A href="http://parts.igem.org/Part:BBa_K2448016" target="__blank">BBa_K2448016</A>). </P><P class="text-left">We also designed pPsiTac2 (<A href="http://parts.igem.org/Part:BBa_K2448050" target="__blank">BBa_K2448050</A>) which differs from pPsiTacI by having also a second binding site for PsiR upstream the -35 box. This year, Team Newcastle developed an innovative biosensor screening platform, Sensynova. Their system is aimed at characterizing biosensor-based screening systems and generating variants of reporter genes, promoters and sensitivity tuning. To test their platform, they needed to be provided with a functional biosensor based screening system. Therefore, we sent them this promoter along with all the data on our biosensor (<A target="__blank" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Collaborations">See details on our Collaboration page</A>).</P><H3 class="content-title">Universal Biosensing Chassis</H3><P class="text-left">Since these Helix-Turn-Helix transcription factors and promoters were only predicted, we had to build the construct to test their behavior.  We talked about our plans with Dr. Fournier, from Sanofi-Aventis, during an interview. He advised us to standardize our building process and screening system with Golden Gate assembly [5]. This particular technique allows fast, efficient and reliable gene assembly, perfect for a high throughput process, to handle multiple cloning of multiple parts in a few steps.</P><P class="text-left">Using Dr Fournier’s suggestions, and in order to speed up the assembly of all our biosensors, we designed the Universal Biosensing Chassis (UBC) [6] (<A href="http://parts.igem.org/Part:BBa_K2448023" target="__blank">BBa_K2448023</A> or <A href="http://parts.igem.org/Part:BBa_K2448024" target="__blank">BBa_K2448024</A>). This construct  aims to provide an answer to the lack of rapid and reliable building methods for transcription-factor based biosensors. Using this chassis, one will only need a suitable transcription factor, able to bind to the molecule of interest, and its related promoter. By its design (Figure 1), the UBC allows fast cloning: depending on what is to be inserted into the UBC and one’s mastery of the Golden Gate Assembly, a functional biosensor can be obtained in less than a week (for more information, see <A href="https://2017.igem.org/Team:Evry_Paris-Saclay/Best_Part" target="__blank">Our Best Composite Part page</A>).</P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 1.</STRONG> Schematic design of the UBC.</FIGCAPTION></FIGURE><P class="text-left">Built on the UBC by replacing mEmerald with codon optimized PsiR and LacZ with pPsi. The biosensors we created in a pSB1C3 plasmid work in the following way: when pTacI is induced by IPTG, it drives the transcription of the PsiR gene coding for the PsiR protein which is predicted to be a transcription factor able to bind D-Psicose. If D-Psicose is present in the cell, the transcription factor will bind preferentially to it and thus it becomes inactivated. The repression of the related promoter pPsi will be released which will enable the transcription of a fluorescent protein, mCherry. If D-Psicose isn’t present in the cell, PsiR will bind to pPsi, preventing any transcription of mCherry.</P><H2 class="content-title">Biosensors Characterization</H2><P class="text-left">The characterizations of all the biosensors have been performed following the protocol described in the ‘Protocol’ session of our wiki and presented in the Parts Registry pages of each biosensor (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>, <A href="http://parts.igem.org/Part:BBa_K2448026" target="__blank">BBa_K2448026</A>, <A href="http://parts.igem.org/Part:BBa_K2448027" target="__blank">BBa_K2448027</A>, <A href="http://parts.igem.org/Part:BBa_K2448028" target="__blank">BBa_K2448028</A>, <A href="http://parts.igem.org/Part:BBa_K2448029" target="__blank">BBa_K2448029</A>, <A href="http://parts.igem.org/Part:BBa_K2448030" target="__blank">BBa_K2448030</A>, <A href="http://parts.igem.org/Part:BBa_K2448031" target="__blank">BBa_K2448031</A> and <A href="http://parts.igem.org/Part:BBa_K2448032" target="__blank">BBa_K2448032</A>). All tests were performed in technical duplicates and biological triplicates (Figure 2). Fluorescence measurements (mCherry) have been normalized on cell density (OD<SUB>600nm</SUB>). Raw data are for all biosensors are available here <A href="https://static.igem.org/mediawiki/2017/0/0c/T--Evry_Paris-Saclay--biosensor--archive--1.zip" download="">(Download)</A></P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 2.</STRONG> Picture of a 96 well plate for the characterization of <A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A> after 14h incubation.</FIGCAPTION></FIGURE><P class="text-left">The main goal of the characterization is to determine which biosensor is the most suitable for our screening process. In order to be able to efficiently use our biosensors for real applications, it’s mandatory to evaluate certain parameters:</P><H3 class="content-title">Optimal Measurement Time</H3><P class="text-left">The characterization of the biosensors allowed us to determine many important parameters. For instance, running the experiment for a long period (almost 18 hours) gave us an estimation of the optimal measurement time.</P><P class="text-left">To estimate this duration, we looked at the raw data of all the biosensors and observed that it takes on average around 9 hours to get an observable signal for the lowest concentration of inducer. It means that sensitivity threshold and consequently maximum accuracy is reached 9 hours after induction. </P><P class="text-left">Since we want to detect and measure D-Psicose concentration between 1 mM and 300 mM, we need a biosensor able to get maximum accuracy in this range of concentration in a minimal time. Taking into account the raw data, we can estimate for the majority of our biosensors that if D-Psicose concentration is above 10 mM, a 6 hours incubation after induction would give relevant results.</P><H3 class="content-title">Basal Expression</H3><P class="text-left">All the biosensors show a basal expression between 200 and 1800 arbitrary units of fluorescence at 9 hours post-induction. This basal activity even without psicose in the media is due to the imbalance between the amount of PsiR transcription factor available and the pPsi promoter strength. Even when PsiR is produced, the transcription factor can’t totally prevent the transcription from happening. Biosensors with low basal activity are also the ones with the worst sensing abilities. Since we care more about fold change than absolute value of fluorescence, basal activity won’t be a criterion of choice in selecting the best biosensor.</P><H3 class="content-title">Dynamic Range</H3><P class="text-left">Determining the dynamic range of our biosensors will give us an estimation of their sensitivity, their maximum and their potential use. What we can observe in figure 3 is a rising curve from 1 mM to 300 mM. This gives us two important pieces of information:</P><UL class="text-left list-unstyled"><LI> First, the results show that all PsiR seems to interact with psicose behaving as predicted for all species. Similarly for the pPsi promoters, even for pPsiTacI, that seems tightly regulated by PsiR under psicose induction, all behaving as predicted. </LI><LI> Second, the dynamic range of the biosensors is very similar for every one of them, and appear to go from 1 mM to 300 mM psicose. Given that, we could use any of them in real applications for our bioscreening protocol to assess the production of psicose ranging from 1 mM to maximum 300 mM.</LI></UL><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 3.</STRONG> In vivo characterization of D-psicose biosensors in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the mCherry measured florescence over psicose concentration in the media. Each data point is the mean of two of technical duplicates and of three biological triplicates.</FIGCAPTION></FIGURE><P class="text-left">Each biosensor show a particular fold change and linearity profile :
						<UL class="text-left list-unstyled"><LI> pPsiA-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>): Perfect foldchange and perfect linearity in range of concentrations corresponding to bioproduction range (1 mM to 300 mM psicose) (Figure 4)</LI><LI> pPsiR-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448026" target="__blank">BBa_K2448026</A>): Saturation at high concentration therefore cannot show enzyme improvements, the foldchange is very low (Figure 5). </LI><LI> pPsiTacI-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448027" target="__blank">BBa_K2448027</A>): Perfect foldchange and satisfactory linearity in range of concentrations which correspond to bioproduction range (1 mM to 300 mM psicose) (Figure 6). </LI><LI> pPsiA-PsiR from <EM>Sinorhizobium fredii</EM> (<A href="http://parts.igem.org/Part:BBa_K2448028" target="__blank">BBa_K2448028</A>): Early saturation upon increasing the concentration. The foldchange is very low. Although it shows more sensitivity (responses in lower concentrations), the extremely weak foldchange of the signal makes this biosensor a bad candidate for screening (Figure 7)</LI><LI> pPsiR-PsiR from <EM>Sinorhizobium fredii</EM> (<A href="http://parts.igem.org/Part:BBa_K2448029" target="__blank">BBa_K2448029</A>): Perfect foldchange in the signal but it tends to saturate at high concentrations. This biosensor is still suitable for screening (Figure 8).</LI><LI> pPsiA-PsiR from <EM>Sinorhizobium meliloti</EM> (<A href="http://parts.igem.org/Part:BBa_K2448030" target="__blank">BBa_K2448030</A>): Early saturation with increasing concentration combined with a very low foldchange (Figure 9). </LI><LI> pPsiR-PsiR from <EM>Sinorhizobium meliloti</EM> (<A href="http://parts.igem.org/Part:BBa_K2448031" target="__blank">BBa_K2448031</A>): Early saturation upon increasing the concentration with very low foldchange. This biosensor shows the best sensitivity but the very weak foldchange of the signal makes it a bad candidate (Figure 10).</LI></UL></P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 4.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiA-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the mCherry measured florescence of over psicose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates. </FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 5.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiR-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448026" target="__blank">BBa_K2448026</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with measured florescence of mCherry over psicose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates. </FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 6.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiTacI-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448027" target="__blank">BBa_K2448027</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the measured florescence of mCherry over psicose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 7.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiA-PsiR from <EM>Sinorhizobium fredii</EM> (<A href="http://parts.igem.org/Part:BBa_K2448028" target="__blank">BBa_K2448028</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the measured florescence of mCherry over psicose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 8.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiR-PsiR from <EM>Sinorhizobium fredii</EM> (<A href="http://parts.igem.org/Part:BBa_K2448029" target="__blank">BBa_K2448029</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the mCherry measured florescence of over psicose concentration in the media. Each data point is the mean of two of technical duplicates and of three of biological triplicates.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 9.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiA-PsiR from <EM>Sinorhizobium meliloti</EM> (<A href="http://parts.igem.org/Part:BBa_K2448030" target="__blank">BBa_K2448030</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the measured florescence of mCherry over psicose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates.</FIGCAPTION></FIGURE><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 10.</STRONG><EM>In vivo</EM> characterization of D-psicose biosensor pPsiR-PsiR from <EM>Sinorhizobium meliloti</EM> (<A href="http://parts.igem.org/Part:BBa_K2448031" target="__blank">BBa_K2448031</A>) in <EM>E. coli</EM> DH5α. The graph shows all engineered biosensors in one plot with the measured florescence of mCherry over psicose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates.</FIGCAPTION></FIGURE><P class="text-left">Three biosensors would be adequate for screening with regard to their sensitivity, fold change and linearity: First pPsiA-PsiR A.t (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>),  then pPsiTacI-PsiR A.t (<A href="http://parts.igem.org/Part:BBa_K2448027" target="__blank">BBa_K2448027</A>) and finally pPsiR-PsiR S.f (<A href="http://parts.igem.org/Part:BBa_K2448029" target="__blank">BBa_K2448029</A>). Among those three pPsiA-PsiR A.t (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) demonstrates the best linearity and fold change simultaneously making it the best candidate for production/enzyme screening.</P><H3 class="content-title">Influence of Fructose</H3><P class="text-left">Testing our psicose biosensor under real application conditions is vital. We aim to measure the concentration of psicose in high fructose level media to observe the conversion of fructose into its epimer, D-Psicose by the D-Psicose 3-epimerase from <EM>C. cellulolyticum</EM>. But to achieve this, our biosensor pPsiA-PsiR A.t (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>), and especially our transcription factor, has to be highly specific to psicose, in order to produce mCherry in proportion only to the concentration of D-Psicose.</P><P class="text-left">We can see on the graph presented in Figure 11 that fluorescence intensity and fructose concentration are not correlated since increasing concentration of fructose doesn’t influence the fluorescence. According to figure 11, fructose doesn’t influence the biosensor behavior since mCherry production isn’t a function of fructose concentration in the media. This finding implies that our transcription factor doesn’t bind to fructose and that our biosensor can be used in high fructose level media to measure psicose concentration. Therefore, the pPsiA-PsiR A.t biosensor is suitable for assessing the activity of D-Psicose-3-Epimerase converting fructose into psicose.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 11.</STRONG><EM>In vivo</EM> characterization of the Psicose Biosensor pPsiA-PsiR from <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) in <EM>E. coli</EM> DH5α. The graph shows the mCherry measured florescence over fructose concentration in the media. Each data point is the mean of two technical duplicates and of three biological triplicates, and error bars represent standard deviations.</FIGCAPTION></FIGURE><H2 class="content-title">Improvement Prospects</H2><P class="text-left">In order to finely characterize the pPsiA-PsiR A.t biosensor (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>), we assessed the effect of IPTG on the fluorescence for a fixed concentration in psicose (Figure 12). The outcome is that IPTG, regardless of its concentration, doesn’t significantly influence the production of mCherry. It means that the PsiR amount in the cell is constant. Our hypothesis is that this phenomenon is due to the pTacI promoter leakiness, leading to the steady production of PsiR. Since our biosensor is carried by a high copy plasmid (<A href="http://parts.igem.org/Part:pSB1C3" target="__blank">pSB1C3</A>), the number of lacO sites of pTacI is much bigger than the number of LacI molecules available in the cell. Even if this finding isn’t an issue for our experiments, this has to be considered.</P><P class="text-left">This issue can be addressed by performing the characterization of this biosensor in our improved backbone, the pSB1C3_LacIq plasmid (<A href="http://parts.igem.org/Part:BBa_K2448038" target="__blank">BBa_K2448038</A>), that we designed for exactly this type of situation. pSB1C3_LacIq has a built-in LacI coding sequence under the control of a mutated version of its own natural promoter known as LacIq which leads to a 10-fold increase in LacI expression compared to the natural promoter.</P><FIGURE class="figure"><FIGCAPTION class="figure-caption"><STRONG>Figure 12.</STRONG> Graph of the measure of florescence over fixed psicose concentration (1 mM) in the media with different concentrations of IPTG.</FIGCAPTION></FIGURE><P class="text-left">To summarize, at the end of this step, we have 7 different psicose biosensors well characterized. We know that the best of them is pPsiA <EM>Agrobacterium tumefaciens</EM> – PsiR <EM>Agrobacterium tumefaciens</EM> (<A href="http://parts.igem.org/Part:BBa_K2448025" target="__blank">BBa_K2448025</A>) for its dynamic range, perfect fold change and linearity. Fructose is proven to not influence the biosensor behavior. With 9h incubation post induction, this biosensor is ready to be used in a screening process to efficiently detect and quantify psicose in order to look for the best enzyme activity.</P><H2 class="content-title">References</H2><UL class="list-unstyled text-left"><LI> [1] Courbet A, Endy D, Renard E, Molina F, Bonnet J. Detection of pathological biomarkers in human clinical samples via amplifying genetic switches and logic gates. Sci Transl Med (2015) 7, 289ra83.</LI><LI> [2] Rogers JK, Taylor ND, Church GM. Biosensor-based engineering of biosynthetic pathways. Curr Opin Biotechnol (2016) 42, 84-91.</LI><LI> [3] Mahr R, Frunzke J. Transcription factor-based biosensors in biotechnology: current state and future prospects. Appl Microbiol Biotechnol (2016) 100, 79-90.</LI><LI> [4] de Boer HA, Comstock LJ, Vasser M. The tac promoter: a functional hybrid derived from the trp and lac promoters. Proc Natl Acad Sci U S A (1983) 80, 21-25.</LI><LI> [5] Engler C, Marillonnet S. Combinatorial DNA assembly using Golden Gate cloning. Methods Mol Biol (2013) 1073, 141-56.</LI><LI> [6] Brophy JA, Voigt CA. Principles of genetic circuit design. Nat Methods (2014) 11, 508-520.</LI></UL></DIV><DIV class="row"><DIV class="col-lg-6 text-center"><H2><A class="content-title" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Chemistry"> Chemistry</A></H2></DIV><DIV class="col-lg-6 text-center"><H2><A class="content-title" href="https://2017.igem.org/Team:Evry_Paris-Saclay/Demonstrate">Screening </A></H2></DIV></DIV></DIV></SECTION><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 text-center"><H2 class="section-heading">Let's Get In Touch!</H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>