<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tec-Chihuahua_Applied_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tec-Chihuahua/Applied Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Erwinions</TITLE><NAV><UL><LI><A href="//2017.igem.org/Team:Tec-Chihuahua">HOME</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/team">TEAM</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/team">MEMBERS</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Collaborations">COLLABORATIONS</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Attributions">ATTRIBUTIONS</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/description">DESCRIPTION</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Applied_Design">DESIGN</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Parts">PARTS</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Results">RESULTS</A></LI><LI><A href="#">NOTEBOOK</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/brainstorm">BRAINSTORM</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/lab_book">LAB BOOK</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/protocols">PROTOCOLS</A></LI><LI><A href="#">HUMAN PRACTICES</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/HP">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:Tec-Chihuahua/HP/Gold_Integrated">INTEGRATED PRACTICES</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Engagement">PUBLIC ENGAGEMENT</A></LI><LI><A href="#">MODEL</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/overview">OVERVIEW</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/wildcell">WILD CELL</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/ModifiedCell">MODIFIED CELL</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Model">RESULTS</A></LI><LI><A href="#">ENTREPRENEURSHIP</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/EntOverview">OVERVIEW</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Entrepreneurship">ERWINIONS</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/process_lean_method">LEAN METHOD</A></LI></NAV><LI><A href="#">JUDGING</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/awards">AWARDS</A></LI><LI><A href="//2017.igem.org/Team:Tec-Chihuahua/Safety">SAFETY</A></LI></P><DIV class="content"><DIV class="design-bg"><H1>Design</H1></DIV><DIV class="container"><H3>Overview</H3><P align="justify">As you already may know, Tec-Chihuahua’s primary purpose is to inhibit <I>Erwinia amylovora</I>´s main virulence factors. The purpose is to do it by using synthetic biology techniques to synthesize three different enzymes; the aiiA enzyme or N-Acyl homoserine lactonase, the Cyclic-di-GMP phosphodiesterase from the yhjH gene and the Putative Glycosyltransferase encoded by epsE. Nevertheless, our project can be divided into two parts: the proof of concept regarding Fire blight (characterizing and proving how the proteins work and how efficiently, as well as proving if it is a good option to design a genetic circuit with one or more of them) and the process beyond (the whole process that goes afterwards, necessary to transform Erwinions into a commercial product).  

</P><H3>Proof of Concept</H3><H4><B>Genetic System</B></H4><P align="justify">In order to inhibit <I>Erwinia</I>’s virulence factors, we knew we first had to design four genetic systems, one for each enzyme (Figure 1.1) plus a polycistronic gene that includes all of them (Figure 1.2), for both <I>Erwinia</I> and BL21(DE3). Therefore we established, as our proof of concept, the molecular and microbiological characterization of each of them. For this, we knew that it was necessary to prove that either one enzyme or the combination of them could act forcefully against the progression of the disease. Nevertheless, we ran into one crucial question: Do we design our Biobrick as a constitutive or regulated gene? 

 <CENTER><SUB><B>Figure 1. </B>Genetic design for the proof of concept stage.</SUB></CENTER></P><P align="justify">One of the most important things to do when starting a research is to acknowledge your possibilities (economic, technical, etc.). Even though we were transforming <I>E. coli</I> and <I>E. amylovora</I>, the fact that we were a lacking some reagents was vital for us to decide to construct a constitutive circuit. At last, what we were trying to prove at this point of the project was that both bacterias were able to express those exogenic proteins. For this purpose, a constitutive gene was perfect because, we believed this way we'd able to characterize an exogenous gene easily. The only problem could be the creation of inclusion bodies. If you continue reading, you may notice that this is not our last genetic system. We decided to use a T7 promoter because we knew that it would work at the BL21(DE3) for sure, while on <I>Erwininia</I> we did not know which promoter would work.</P><H5><B>Bronze</B></H5><P align="justify"> When we did our research and found out of those three enzymes, we did not know that they were already at the iGEM registry. Nevertheless, this turned out to be good news as they were registered but lacked complete characterization. It is important to highlight the fact that we practically needed to develop a new Biobrick. Because the coding sequences (BBa_C0060, BBa_K861090 and BBa_K143032) did not have a complete circuit and we lacked an expression vector, we had to develop our vector by digesting and ligating a promoter (BBa_K525998) and a terminator (BBa_B0010) to each protein (Figure 2). When we were able to complete it, we started characterizing both molecular and microbiologically. We did some protein kinetics with the help of SDS Page technique, colorimetry, and motility protocols. You can see the results either at our <A href="https://2017.igem.org/Team:Tec-Chihuahua/Results">wiki</A> or in the <A href="http://parts.igem.org/Part:BBa_C0060">iGEM Registry</A><CENTER><SUB><B>Figure 2. </B>Biobrick assembly sketch.</SUB></CENTER></P><H5><B>Silver</B></H5><P align="justify"> For our silver construct, we realized, with the help of BLAST, that most of the sequences we were using were reported as putative proteins, while there were some different sequences at the NCBI database better characterized. Therefore, we decided to construct two new Biobricks with better experimental backgrounded proteins that would help us compare the results from the existing at the iGEM registry against those in the NCBI. 

  </P><P align="justify"> The epsE gene was the first we decided to design as it was reported that it had never been characterized in any other microorganism than<I> Bacillus</I>. We thought that if we had a negative expression in BL21(DE3), it could be because iGEM`s epsE was not the optimal. If at least we had two different sequences that might code for the same protein we could make better assumptions and conclusions.

  </P><P align="justify"> On the other hand because iGEM`s aiiA was reported as putative at the NCBI and the characterization at the registry was just a substrate-enzyme kinetic (that might be influenced by an isoform), we decided to construct a new aiiA gene. Plus if we were unable to characterize epsE because of an adverse expression, therefore this might also be our back up plan. 

  </P><P align="justify"> If you want to look at the complete sequence of our construct, you can click <A href="http://parts.igem.org/Part:BBa_K2471004">here</A>.

  </P><P align="justify"> We would not have achieved this without IDT's support. Thank you so much!

</P><H3>Final Product</H3><P align="justify"> For this phase of the project, the first complication that we encountered was: If we prove that our solution works, then how am I going to apply it? As mentioned on the project description tab, there were many ideas that we thought were useful applications for our product. Nevertheless,  it was until we started gathering information for our Human Practices that one agricultural engineer, Rafael Quevedo owner of Optihumus, told us about what ended being the best option (you can read more about this Integrated Human Practices here): a biocontrol with our modified <I>Erwinia amylovora</I> as main composite. He recommended us developing a biocontrol under the same principle of the modified Mosquito for the Malaria.

</P><H4><B>NEW Genetic System</B></H4><P align="justify">We knew that for us to develop this product until the commercialization stage, we needed to create a new genetic system that ensured our transformation and, therefore, our GMO as biosecure and efficient. Unfortunately, the regulations in Mexico regarding this matter are inefficient and/or inexistent. This way, as part of our <A href="https://2017.igem.org/Team:Tec-Chihuahua/HP/Gold_Integrated">Human Practices</A> and Integrated HP, we developed Official Mexican Norms (NOMs) that were discussed and validated by the Ministers of agriculture and environment. Plus, we internationally validated it according to The Cartagena Protocol on Biosafety. These stakeholders did give us some feedback that we took into account in the creation of this new genetic system. 
  </P><P align="justify">According to what we developed for and with the governmental entities, the whole process of creating a GMO should ensure security and should be ethically correct. According to the NOM and SAGARPA, one of the higher risks when developing a GMO is the fact that researchers, in the laboratory, confer them antibiotic resistanc. Taking this into account, we proofread the protocols and good practices (rules) that we have established in the NOMs.
 </P><P align="justify">It was necessary to propose a new genetic system that met this requirement. The image below is a general sketch of the plasmid we propose. To avoid this problem, the creation of a GMO whose final intention is the release to the environment could only be through the insertion of a vector of cloning or expression with positive selection. The basis of
these vectors is the presence of a lethal or suicidal gene that can substitute an antibiotic resistance gene supplemented by a reporter gene as fluorescence (Figure 3). The coding sequence of this alternate contains multiple cloning sites so that only if the gene
is interrupted by the ligation of the desired insert, the host cell will be able to grow. Otherwise, as the insertion of the gene of interest does not happen, the suicide gene will inhibit the growth of the bacteria.<CENTER><SUB><B>Figure 3. </B>Killer gene digestion &amp; ligation. </SUB></CENTER></P><P align="justify"> The physical interpretation of the results is quite easy. According to this design, we can only have two kinds of colonies: a fluorescent one and a nonfluorescent. The first, theoretically, has our desired DNA ligated to the vector, while the nonfluorescent never acquired any kind of plasmid. Nevertheless, one big problem in our design is that, without antibiotic resistance, the colony isolation would be complicated. For this, we had come with a rough solution that is described in the protocol inside the NOM (Figure 4): 

</P><P align="justify"> &quot;12.1Perform a drop size test of the transformation mixture to be inoculated on the LB agar. 12.1.1 Extend 5, 15, 25 and 50 μL of the transformation mixture on an individual LB agar
plate. The technique is specified in NOM-092-SSA1-1994. Method for aerobic bacteria
counts in a plate.&quot;<CENTER><SUB><B>Figure 4. </B>Drop size test for colony isolation.</SUB></CENTER></P><H4><B>CRISPR Cas9</B></H4><P align="justify">The use of CRISPR in <I>E. amylovora</I> is mainly focused on the evaluation of a permanent and more stable DNA modification. Analyzes performed on CRISPR spacer sequences revealed a considerably greater genetic diversity than previously known. The use of this new technology in a more applied way is contemplated in the Official Mexican Norm regarding CRISPR. Allowing us to implement the insertion of the target genes to the bacterial chromosome; which will increase the efficiency of the biocontrol in case <I>E. Amylovora</I> degrades the plasmid. This technology seeks to achieve a more efficient, safer and highly specific product unlike other methods currently proposed. 

A Python script was used to find protospacer sequences upstream from the amsL gene (we looked at the 230 bases before the gene). The script looked for the NGG motif (PAM site) which the Cas9 nuclease requires to recognize and cleave a sequence. In these 230 bases, 7 possible 23bp targets were found and BLASTed against the <I>Erwinia amylovora</I> genome to ensure no off-target cleaving activity will occur. The BLAST results were parsed with the BioPython package in another Python script to rank the target sequences according to the number of matches and their positions. Even though all 7 targets showed matches and therefore it is probable that some off-target activity will occur regardless of which sequence is chosen, protospacer 1 (CAGTTATCCTGTTTGCCTGAAGG) seem to be the best option due to its location respect to the gene, experimental assays would help determine better which sequence is the finest.<OL><UL>Protospacer 1 CAGTTATCCTGTTTGCCTGAAGG</UL><UL>Protospacer 2 AGTTATCCTGTTTGCCTGAAGGG</UL><UL>Protospacer 3 AAGCTCCAGCCGCTTTGCGATGG</UL><UL>Protospacer 4 ATGGAACGCCCTCTTTTCTCAGG</UL><UL>Protospacer 5 CCTTTATCAGTCGCCTGTTTAGG</UL><UL>Protospacer 6 CTTTATCAGTCGCCTGTTTAGGG</UL><UL>Protospacer 7 CGCCTGTTTAGGGTCCTTGTAGG</UL></OL></P><P align="justify">The guide RNA described above is made to carry out a break in the operon of DNA (<I>ams</I> operon) from the exopolysaccharides, therefore it will make a cut after the amsL gene. This operon has the expression of twelve genes from <I>amsA</I> to <I>amsL</I>. It is believed that <I>amsC, amsH</I> and <I>amsL</I> are involved in the transport and assembly of oligosaccharides, while <I>amsA</I> possesses a tyrosine kinase activity. <I>amsB, amsD, amsE, amsG, amsJ</I> and <I>amsK</I> proteins seem to play a role in the annealing of different subunits of galactose, glucuronic acid, and pyruvate to the lipid carrier in order to form an amylovoran unit. <I>amsl</I> seems to perform a different function in the recycling of the diphosphorylated lipid carrier after the release of the synthesized repeating unit.

</P><H3>Prototype design</H3><P align="justify">Our prototype consists of a biocontrol capable of fighting fire blight caused by <I>Erwinia amylovora</I>, through the genetic transformation of this bacteria, with genes that eliminate or counteract its virulence factors that make it pathogenic. Since they belong to the same genera, they are expected to compete for nutrients and space with other pathogenic bacteria, acting as their antagonists. As a bacterium that spreads easily in optimal conditions, such climate, temperature, and humidity,  just the fact of having enough microbial density will do to make it spread. At controlled conditions in a bioreactor, its survival will become easier when applying it to the plant.
  </P><P align="justify">As a result of interviewing various potential customers and subject matter experts, it was concluded that acceptance for our product would be better if its application were foliar. Therefore, the bacteria genetically modified by chromosomal insertion using CRISPR, would ensure the transformation to the daughter cell, which would be immobilized and would inactivate the metabolic mechanism momentarily increasing the speed of the catalytic processes inside the cell. This is also a method used to enable a controlled or sustained release system, that protects the product against oxidation, photosensitivity, and volatility. This would prolong the inhibitory effect and the application of other products.
  </P><P align="justify">However, in order to confer a longer shelf life, it is necessary to undergo an industrial process of indirect spray drying which involves the transformation of liquid into powder, where an atomizer is used to spray the mixture on hot gas which acts as a dryer and provokes the encapsulated material to be wrapped by a polymeric matrix. Our final product will be hermetically packed so the content will not be affected and it can be distributed. The whole process can be visualized in Figure 5.
<CENTER><SUB><B>Figure 5. </B>Prototype process.</SUB></CENTER></P></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><DIV class="center"><H3>Follow us on social media:</H3><DIV class="row"><DIV class="col-sm-8 col-centered"><P align="left">Facebook: <A href="https://www.facebook.com/igemcuu/">facebook.com/igemcuu/</A></P></DIV><DIV class="col-sm-8 col-centered"><P align="left">Twitter: <A href="https://twitter.com/iGEM_TecChih">@iGEM_TecChih</A></P></DIV><DIV class="col-sm-8 col-centered"><P align="left">Email: <A href="igemtecchihuahua@gmail.com">igemtecchihuahua@gmail.com</A></P></DIV><CENTER><H3> We thank our sponsors:</H3></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>