Team:Carnegie Mellon/Notebook
Home
Team
Students
Advisors
Gallery
Attributions
Sponsors
Project
Notebook
Protocols
Parts
Device
Modeling
Measurement
Improved Part
Collaborations
Medal Requirements
Interlab
Community
Maker Movement
Interviews
Notebook.
We tracked everything in our handy-dandy notebook.
Bioluminescence Cloning
Proteins
Interlab Study
Estrogen Sensor
Modeling
Project Specs
Week 4
6.12.15
Goal: To amplify the Gaussia Luciferase DNA using PCR
1a. Conduct a Gaussia Luciferase experiment where we resuspended the DNA using this protocol:
20µl of water into the 200ng tubes
Use 5µl in PCR
1b. Primers
10µl of stock with 90µl H20 → 10µM
2. Then we did a Phusion PCR using the following measurements:
6.15.15
Goal:To PCR amplify the PelB Gaussia DNA and test the competence of competent cells.
Checked to see if the gel sizes were correct.
We split PelB PCR into two tubes of 25µl each for digestion because we eluted with 50µl of H20 during PCR purification instead of 30µl. That’s why for digest we added 3.2µl of buffer and 2 of each enzyme.
If so, then did digestion for PCR product
2µl of each enzyme: Xba1 & PST (Can also use EcoRI instead of Xbal)
3.2µl of buffer
No water because volume is too large (from PCR)
Put digestion pieces into new gel to extract out separated DNA
1:1 binding buffer on gel extraction (PCR purification)
Digested the backbone (101A) using the same enzymes
Digestion
14µl of 101A backbone
2µl of buffer
2µl of each enzyme (xba1 & pst1)
Gel Extraction (Cut out 2000 bp band). However because we were very confused, we cut out the incorrect band
Gel purification
Ligation
4:4 ratio
1µl buffer
1µl ligase
However we were only suppose to do a 2µl : 2µl  ratio of digestion product : insert to backbone
Transformation
50µl  competent cells (our competent cells)
5µl of ligation but it should have been the entire 10µl of ligation (screwed up which would explain why our yield was so low
500µl of LB
Plated 400µl of transformants in 37C (our yield was low because we messed up on the concentration of ligation and the digestion product to insert ratio)
6.16.15
Goal:To PCR amplify the rest of the DNAs.
Checked the plates to see of PelB Gaussia worked. It did not because we still saw fluorescence in colony which shouldn’t have happened if we extracted just the backbone so redid cloning.
PCR Phusion on all new samples not the old PelB because that was already amplified.
Firefly Luciferase CO
CD-celmRFP
PelB-RFP
CD-celGaussia
Renilla Luciferase CO
J23100-E0024-RFP (?)
PCR purification eluted in 30µl
Set up digestion for PCR product
2µl of each enzyme: Xba1 & PST (Can also use EcoRI instead of Xbal)
3.7µl of buffer
30µl sample (PCR product)
Put digestion pieces into new gel to extract out separated DNA
1:1 binding buffer on gel extraction (PCR purification)
Digested the backbone (101A) using the same enzymes
Digestion
14µl of 101A backbone
2µl of buffer
2µl of each enzyme (xba1 & pst1)
Gel Extraction (Cut out 2000 bp band) low intensity and potentially could have cut the wrong band (cut out middle band, which we thought was around 2000bp).
6.17.15
Goal:To Ligate and Transform J23100, PelBGaussia, CelmGaussia with 101A
Ligation
2:2 ratio of J23100,PelBGaussia,CelmGaussia: 101A (backbone)
1µl buffer
1µl ligase
Transformation
50µl  competent cells (our competent cells)
10µl of ligation
500µl of LB
Plated 400µl of transformants in 37C
6.18.15
Goal:Pick out to see if Transformants worked and then send them out to sequencing.
Observed the three colonies below:
J23100-101A
CelmGaussia-101A
PelBGaussia-101A
The J23100 did not fluoresce red therefore we didn’t extract it through the PCR. We only did colonies found in CelmGaussia-101A (2 colonies) and PelBGaussia-101A (6 colonies). It might be worrisome that we did not have much colonies after ligation but something is there.
For each colony, we suspended in 50µl H20 and added 1µl of the suspension into PCR tube with 9µl MM.
The MM protocol can be found in the other spreadsheet in the folder. We did have 8 samples but since the volume is 9 each, we actually had the volume for 8 PCRs so 4µl * 20 is equal to 8µl*10.
Used the Program: CherylTAQ30s and then put it through the gel extraction.
After gel extraction, will do miniprep (Gel extraction photo in folder).
Prepared minipreps on samples where the backbone was incorporated into Gaussia DNA samples. (CelmGaussia Samples did not work)
PelBGaussia1
PelBGaussia2
PelBGaussia3
PelBGaussia4
PelBGaussia5
5 of each sample, 5 Chlorophenocol, 500 LB
Overnight at 37 room
Will miniprep tomorrow.
Prepared other DNA samples (only had backbone after we transformed it)
J23100-backbone
Renilla Luciferase-backbone
CD-CelmRFP-backbone
PelBRFP-backbone
Ligated
4:4 Each sample colony with 101A backbone
1 of Ligate
1 of Ligation T4 Buffer
Transformed
1 of DNA
50 TOP10 cells
5min Ice
2min heatshock
5min Ice
6.24.15
Goal:To re-transform the CelGaussia with competent cells to see if we can get colonies as well as grow mach/competent cells for the Coelenterzine insert.
Examined both the Firefly and Gaussia plates and saw that the Firefly fluorescence but the Gaussia did not.
Re-transformed the CDCel Gaussia and grew competent cells in 5 µl for the Coelenterazine insert.
Left the samples overnight at 37°C.
6.25.15
Goal:To check if the transformations put into 37°C.
Unfortunately we realized that CelGaussia was GFP positive meaning the clone did not work properly. In addition, the Firefly was not RFP positive meaning it also did not work properly
We re-transformed the Phusion PCR using celGaussia.
Did the Colony PCR for the Firefly.
We picked 4 single colonies to do the PCR colony.
Used 50 µl of water + 1 colony in each tube.
6.26.15
Goal: To continue the bioluminescence Project
Continue purifying and isolating the DNA for CD-Gauss
Continue with setting up transformations for the remaining luminescence DNA. Remaining samples left to re-transform into backbon
PelB RFP
CD-cel mRFP
Cel-Gaussia
We digested the PCR Product of CD-Gaussia with restriction sites: Xba & Ps
We only did 25 µl and saved 5 µl in case the digestion did not work appropriately. That way, there will be more DNA to amplify aga
We also sent out 133-135 for sequencing using the similar labeling system:
Renilla 1 (iGEM 133): VR
Renilla 2 (iGEM 134): VR
Firefly 2 (iGEM 135): V
6.30.15
Goal: To colony PCR a few of the samples collected. Then using DNA IDT, send the PelB, RFP, and CelCD RFP to sequenc
Check to see if samples fluorescent
Set up colony PCR for different vector:insert dilutions of CDcelGaussia
Then I did a Tecan analysis of CD Cel RFP and CD Gaussia
Samples of CD Cel RFP and PelB RFP (4 samples each) were miniprepped and placed into overnight cultures.
After miniprep, the samples were physically examined to make sure the red is present
The samples were then sent to sequencing and labelled as such:
IGEM 136: CDCelRFP 1 Forward
IGEM 137: CDCelRFP 1 Reverse
IGEM 138: CDCelRFP 2 Forward
IGEM 139: CDCelRFP 2 Reverse
IGEM 140: CDCelRFP 3 Forward
IGEM 141: CDCelRFP 3 Reverse
IGEM 142: CDCelRFP 4 Forward
IGEM 143: CDCelRFP 4 Reverse
IGEM 146: PelBRFP 3 Forward
IGEM 147: PelBRFP 4 Forward
iGEM 148: Firefly 2 Reverse
7.01.15
Goal: To set up new gblock on the collected sample
Set up and reamplify new gblock through phusion gel
Set up cultures in order to do colony PCR the next day.
7.10.15
Goal: To set up and send out sequences for sequencing.
DNA samples were sent out for sequencing. These samples were:
161: CD Gaussia Prefix
162: CD Gaussia Suffix
163: CD-cel RFP (CD-cel Forward)
164: CD-cel RFP (CD-cel Reverse)
165: CD-cel RFP (CD-cel Forward)
166: CD-cel RFP (CD-cel Reverse)
167: CD-cel RFP (CD-cel Suffix)
168: 115 RFP1
169: 115 RFP2
7.13.15
Goal: To retransform T7 with estrogen sensor, RNA polymerase, YFP with the sites, and YFP without the sites.
What to do:
Retransform T7 with YFP with the sites, YFP without the sites, and RNA polymerase.
Plate them and set up cultures with J23100 RFP and T7 YFP without sites whose plate was lacking in colonies
7.15.15
Goal: To clone the T7 RNA Polymerase, T7 YFP with sites and T7 YFP without the sit
The colonies on the plate with the T7 YFP without the sites was growing extremely slowly so it was left in the 37C incubator for another night. For the other coplasmid transformations (T7 YFP with the sites and the T7 RNA polymerase), the colonies were yellow under UV light. Overnight cultures were set up so that the luminescence could be measured in the morning. Minipreps were done on the T7 RNA Polymerase and J2310
7.16.15
Goal: To compare the luminescence of the Estrogen Sensor with its controls
Measured the luminescence of the estrogen sensor (with and without estrogen) along with its controls (T7 RNA polymerase, T7 YFP with sites, T7 YFP without sites) by adding 10 ul of 2uM coelenterazine.
7.17.15
Goal: To compare the luminescence of the Estrogen Sensor with its controls.
Measured the luminescence of the estrogen sensor (with and without estrogen) along with its controls (T7 RNA polymerase, T7 YFP with sites, T7 YFP without sites) by adding 10 ul of 2uM coelenterazine. Two biological replicates and three technical trials were done for each sample.
7.20.15
Goal: To redo T7 Tecan Result
Took cultures out of 37C which was put in on Sunday. Did the Tecan Results for that is found in this folder.
Minipreped 101B backbon
Phusion PCR of new iDT order (fluorescent Proteins)
Set up cultures
7.21.15
Goal: To do TECAN results and PCR the new fluorescent proteins
Made PCR Phusion for new fluorescent proteins
measured TECAN results which can be seen in the excel file
7.22.15
Goal: To clone the co-optimized fluorescent genes into bacterial MACH cells
Phusion PCR was done to amplify the DNA for the fluorescent proteins from the gBlocks. PCR products were run on a gel to verify the sizes.
Lane 1: ladder
Lane 2: BFP HCO
Lane 3: eGFP HCO
Lane 4: GFP HCO
Lane 5: OFP HCO
Lane 6: YFP HCO
Lane 7: E0040
7.23.15
Goal: To clone the co-optimized genes for the fluorescent proteins into MACH bacterial cells.
Ran a digest on the purified Phusion PCR products using xba1 and spe1 restriction digest enzymes. Gel extraction was done to purify the DNA and to verify the sizes. The DNA with the appropriate sizes were transformed into bacterial MACH cells with freshly cut vector.
Lane 1: ladder
Lane 2: BFP
Lane 3: eGFP
Lane 4: GFP
Lane 5: OFP
Lane 6: YFP
Lane 7: E0040
Starter cultures were made for the estrogen sensor and controls.
7.24.15
Goal: Cultures that were made on 7.23 for the estrogen sensor and its controls were diluted 1:100 were grown overnight in the 37C shaker and will be measured using the luminometer tomorrow.
7.25.15
Goal: To analyze the effects of estrogen sensor with and without estrogen along with its controls. What we did: The luminescence of the estrogen sensor (with and without estrogen) and its controls were measured.
7.28.15
Goal: Measure the luminescence and response from the estrogen sensor. Compare the fluorescence from J23100 RFP and J23115 RFP. To analyze the sequence for CD-cel Gaussia and to determine if there is luciferase activity. What we did:
Made dilutions (1:100) from the starter cultures for the estrogen sensors and controls. Made cultures for the J23100 RFP and 115 2 RFP. Left them overnight in the 37 dry shaker
Set up transformations for CD-cel Gaussia (on plates and in culture) for testing of luciferase activity in response to addition of coelenterazine.
Checked the sequence data for CD-cel Gaussia. It works so cultures will be set up for luciferase activity testing. The CD-cel domain and gaussia domain didn’t work.
Week 3
6.8.15
Goal: To begin quantifying fluorescent proteins produced in MACH cells: mRFP, OFP, YFP, eGFP, BFP. Also to create a strongly expressing culture of YFP
Created 5 mL cultures from previous cultures, with the exception of the YFP transformed bacteria, which was taken from a plate created 5/27/2015.
Streaked a new multicolor plate with all his tagged fluorescent proteins
Calculated OD of 1:10 dilution of each culture using online calculator
OD was 0.369, thus the non dilute concentration of bacteria we used was 2.95 * 10^9, and the amount of bacteria in each sample was 4.425*10^9
His-tagged soluble protein extraction outlined in our procedure using a washing solution with dilute imidazole
Used 50 uL of beads for each
extracted proteins from 1.5 mL of each culture
OD was 0.369, thus the non dilute concentration of bacteria we used was 2.95 * 10^9, and the amount of bacteria in each sample was 4.425*10^9
Read fluorescence in a microplate reader
YFP is still not expressing, and we did indeed create a culture from a weak colony
For next time: We will test our new cultures, which should be expressing well, as we marked strong colonies on the initial YFP plate. We will try a pH elution after extraction on Ni beads, as adding too much imidazole will interfere with fluorescence readings
6.9.15
Goal: To test the yield and purity of our protein extraction methods using a representative transformed E. coli culture. We will use a pH 4.5 or lower buffer to elute his tagged fluorescent protein from the Ni NTA beads to quantify in the Nano Drop spectrometer.
Created 5 mL cultures from previous cultures, with the exception of the YFP transformed bacteria, which was taken from a plate created 5/27/2015.
Streaked a new multicolor plate with all his tagged fluorescent proteins
Calculated OD of 1:10 dilution of each culture using online calculator
His-tagged soluble protein extraction (protocol in the Protocol Page ) using a washing solution with dilute imidazole
Used 50 uL of beads for each
extracted proteins from 1.5 mL of each culture
Performed elution from beads
Used 100 mM EDTA / 10 mM Tris solution  to elute proteins from beads
incubated for 5 minutes
removed supernatant and repeat for a second elution
Quantified solution with NanoDrop
Results:
OD of cultures was 0.356 in a 1:10 dilution, thus 4.275 * 10^9 bacteria in each 1.5mL sample.
Concentration of Eluted mRFP
First Elution
Second Elution
6.15 mg/mL
1.6 mg/mL
6.353 mg/mL
1.777 mg/mL
6.352 mg/mL
1.523 mg.mL
Mean: 6.285 mg/mL
Mean: 1.63333 mg/mL
0.11691 mg/mL
0.13024 mg/mL
Total amount of protein: 395.92 µg
The elution used before the EDTA elution did not work -- had a pH of 5.0 & needed a pH of 4.5.
YFP had trouple growing & flourescing.
Linked is a picture of the trouble with the YFP plates we've made so far Link of the three YFP plates & description.
6.10.15
Goal: To improve yield of fluorescent proteins eluted from Ni NTA beads during His-tagged soluble protein extraction.
Carried out His-tagged soluble protein extraction as normal and then proceeded to elute 2 samples (50 uL of bead solution with protein from 1.5 mL of culture).
One bead sample eluted three times with 5 min. incubation period in 250 µL of 100 mM EDTA / 10 mM Tris solution for each elution
Second sample eluted twice with 10 min. incubation period in  250 µL  of 100 mM EDTA / 10 mM Tris solution for each elution
For five minute elutions, first elution had protein concentration of 0.95mg/mL, second elution had 0.355mg/mL, and the third had 1.004 mg/mL.
For ten minute elutions, the first elution had 0.615 mg/mL concentration and the second elution had 0.328 mg/mL concentration.
In total, the 5 minute elution had 384.83 µg and the 10 minute elution had 235.75 µg per 1.5 mL culture.
Ran gel from yesterday's elutions. Link to results here.
6.10.15
Goal: Refine procedure for quantifying concentration of fluorescent proteins in 1 L of culture. Focus on how higher volume of EDTA buffer affects the amount of protein eluted from Ni NTA beads.
Carried out His-tagged soluble protein extraction and eluted 2 samples (50 uL of bead solution with protein from 1.5 mL of culture, 2 cultures of OFP transformed E. coli, and another 2 of mRFP transformed E. coli).
One bead sample was eluted three times with 5 min. incubation period in 25 µL of 100 mM EDTA / 10 mM Tris solution for each elution
Second bead sample was eluted twice with a 10 min. incubation period in  50 µL  of 100 mM EDTA / 10 mM Tris solution for each elution
Eluted proteins quantified using a NanoDrop spectrophotometer.
Total amount of protein: 25 µL elution mRFP: 56.3235 µg, 50 µL elution mRFP: 81.275 µg, 25 µL elution OFP: 53.6605 µg, 50 µL elution OFP: 142.4135 µg
6.12.15
Goal: Continue work on the elution of the proteins from Ni beads.
Performed his-tagged soluble protein extraction on 4 1.5 mL cultures transformed to produce mRFP, and 4 transformed to produce OFP.
Elution with 100 mM EDTA / 10 mM Tris solution for 50, 70, 85, and 100 µL volumes for each elution, 3 rounds of elution for each sample.
Finally have a plate with expression of all 5 fluorescent proteins.
Week 4
6.16.15
Goal: Work on technique for using bio rad protein assay that works with our sample concentration and our spectrophotometer.
Standard His-tag soluble protein extraction onto 50 μL of Ni-NTA bead solution
3 elutions using 5 min incubation time in 50 μL of EDTA buffer
The concentration of 1 μg/mL that we used for the BSA standard curve did not reveal any relevant information. In the future we will use standards ranging from 3 ug/mL to 20 ug/mL to encompass our data.
6.17.15
Goal: Move fluorescent proteins from MACH competent cells, a K-strain to TOP10 competent cells, which are a W-strain (required for iGEM).
The procedure followed can be found in the Protocols page
6.18.15
Goal:Run another Bio-Rad Protein Assay test with a BSA standard curve to ensure we have the technique down and can replicate it once we have our TOP10 cells.
See 6/16/2015 for the protocol on BSA standards and testing protein concentration with the Bio-Rad Protein Assay. Today we used the range 3 μg/mL to 20 μg/mL as opposed to 1-10 which we used on 6/16.
Calculated that we have 2.543 mg of mRFP protein in 1.5mL culture.
Transformed cells from before did not work, performed transformations again.
6.19.15
Goal: Move fluorescent proteins from MACH competent cells to TOP10 competent cells. There were some problems with the overnight cultures and plates, which we have to correct.
Retransform the plasmids into new plates
Add IPTG to new cultures to induce gene expression
Restart cultures from yesterday in new media (did not add IPTG) & started cultures from transformation (added IPTG)
Measure growth of TOP10 cells relative to growth of MACH cells (using OD measurements). The following is the OD in Top10 cells
mRFP: 0.102, OFP: 0.056, YGP: 0.053, GFP: 0.054, BFP: 0.213
Week 5
6.22.15
Goal: Test the effect of IPTG on the growth of the culture.
Used UV light to check relative expression levels of each fluorescent protein and marked colonies
Took colonies from plates and put in 5 mL LB, adding 5 μL LPTG and 5uL chloramphenicol
IPTG cultures that were started from the transformation on Friday did not grow, were not expressing the protein, and were all the same color.
The cultures started from culture grew fairly well. Will compare them tomorrow with the other two cultures (from culture, from transformation compared to from plate).
Weak expression of IPTG culture was most likely due to starting it from a transformation rather than a colony. These cultures were thrown out.
6.23.15
Goal: To extract the proteins from our new cultures that were grown with IPTG.
Followed standard protein extraction protocol.
6.24.15
Goal: To quantify an array of fluorescent proteins produced in Top 10 E. coli cells and subsequently characterize their relative fluorescence in order to connect the amount of protein to fluorescence.
Normal his tagged soluble protein extraction on 50uL of Ni-NTA bead solution for mRFP, OFP, YFP, eGFP, and BFP
3 rounds of elution in 50 uL of EDTA buffer for each protein
Bio-rad Protein Assay microassay protocol using BSA standards of 3, 9, 15, and 20 ug/mL
Fluorescence measurements taken on TECAN Safire2 with 100uL of culture and 100uL of extracted protein for each culture transformed to express fluorescent proteins
Week 6
6.29.15
Goal: Start work with the Gaussia Luciferase and continue work with fluorescent proteins in TOP10 cells.
Started five 2 mL cultures of the Luciferase for experiments tomorrow.
Restreaked fluorescent proteins that were transformed last week into TOP10 cells.
Start 5 mL cultures of the fluorescent proteins for purification tomorrow. Started these from the plates that were grown on 6/19.
`
6.30.15
Goal: Run the TECAN and the BCA protein assay test on the proteins we purified last week
Conduct standard BSA protein assay protocol. Use same standards as last week & same dilutions of culture for sample readings (1:10, 1:100)
Run TECAN to try & find correlation between concentration of protein in culture with fluorescence data.
Made new BSA standards for use in BCA protein assay experiments. Made four standards; 3 μg/mL, 9 μg/mL, 15 μg/mL, and 20 μg/mL.
For 3 µg/mL of BSA, the absorbance @ 595 nm was 0.047; for 9 µg/mL, 0.707 absorbance; 15 µg/mL had 0.665; 20 µg/mL had 0.792.
7.01.15
Goal: Test the Arduino-based luminometer with the Luciferase cultures we started yesterday.
Used TECAN plate and a tube to test Light Dependent Resistor and make sure it is sensitive enough to make the measurements we need.
Added 100 μL cells to a TECAN well. Kept lights off and the Arduino recording data until it read no signal. Added coelenterazine into cells and observed what happens. Expected cells to emit light.
Did not have much luck with these tests (the LDR could not pick up such a faint signal)
Tried to purify luciferase to get a better response.
There was likely no response because of the cheap diode used.
Started an overnight from the No IPTG cultures. Added 500 mL of the No IPTG culture to 5 mL LB, 5 μL chloramphenicol and 5 μL IPTG.
Started overnights from the Luciferase cultures; three 2 mL & two 1 mL. Added 1% chloramphenicol and 10% culture to the media.
Will begin using the Arduino Mega instead of the Arduino Uno in order to have access to more SRAM (Static Random Access Memory).
Week 7
7.06.15
Goal: Test LDR with fresh dilution of coelenterazine & new Arduino Mega, determine if purification is necessary to elicit signal, and test fluorescence of the beads that were introduced into the supernatant of the spun down Luciferase (from 7/1).
Tested luminescence in the well of a TECAN plate.
Added coelenterazine in various concentrations to estimate approximate concentration needed.
Compared readings from purified proteins and culture by adding coelenterazine and measuring in TECAN machine.
The LDR was still not sensitive enough to read the luminescence from the culture.
Could see with the naked eye that the culture was giving off light, the LDR just couldn’t detect it.
Couldn’t make conclusions about concentration of coelenterazine or necessity of purification because we were not getting enough signal to quantify it.
7.08.15
Goal: Restreak TOP10 cells that contain fluorescent proteins
Restreaked cells with flourescent proteins.
Did an IPTG restart designed to induce the fluorescence of the cell in TOP10
7.09.15
Goal: Complete IPTG restart & quantify fluorescence of the cultures and purified proteins.
Did not complete the IPTG restart because the cultures did not grow properly overnight. Had an OD of 0.304, needed an OD of at least 0.6.
Yesterday's plate also did not grow properly. Only GFP had strong growth.
Started six new 5 mL Luciferase overnight cultures.
Seen below is the protocol we standardized and distributed to labs like the Citizen Science Center in an effort to gain more uniformity between data observed in different labs around the world. The back of the card has details on how to elute the protein off on the washed Ni-NTa beads. A download link for the JPEG can be found below the images.
FRONT
BACK
PROTEIN PURIFICATION CARD JPEG
Week 1
5.28.15
Goal: To do Miniprep to purify the DNA. It is a small scale isolation of plasmid DNA from bacteria. Then do restriction enzyme digest on plasmid DNA and gel electrophoresis. This technique can be used for cleaving DNA molecules at specific sites, ensuring that all DNA fragments that contain a particular sequence have the same size; furthermore, each fragment that contains the desired sequence has the sequence located at exactly the same position within the fragment. The cleavage method makes use of enzymes isolated from bacteria called restriction enzymes. They are able to cleave DNA molecules at positions at which particular short sequences of bases are present.
Did MiniPreps of MACH cells expressing fluorescence to isolate plasmid DNA from MACH cells (protocol in the Protocol Page )
Performed restriction enzyme digestion on J23108, J23109, J23111 constructs
For promoters, PST1 & Spe1 were used. For insert, xba1 and Pst1 were used.
Ran an Agarose Gel Electrophoresis
Expected size of insert (from E0040) - 720 bp
Expected size of vector backbone (J23108, J23109, J23111) - 2105 bp
Cut digests from gels (protocol in the Protocol Page ).
For next time: Will prepare Colony PCR to determine if we got the correct restriction sites and sequences from the given interlab devices.
Week 2
6.01.15
Goal: To submit minipreps for sequencing, will get results in two days and compare our constructions.
Did minipreps on samples where the devices were incorporated in (J promoters & I3504)
Sent minipreps off for sequencing.
5 µL water, 5 µL sample, 1 µL of each primer
6.02.15
Goal: Make competent MACH cells.
Made competent MACH cells (protocol in the Protocol Page )
measured fluorescence of GFP for 6 samples using a TECAN SaphireII Microplate Reader
108A, 108C, 109A, 109C, 111A, 111B
Parameter
Description
Microplate Reader Model
TECAN Sapphire II
Software
XFlour4
Measurement Mode
Flourescence Bottom
Excitation Wavelength
500 nm
Emission Wavelength
515 nm
Excitation Bandwidth
5 nm
Emission Bandwidth
5 nm
Gain (Manual)
150
Number of Reads
100
Flashmode
High Sensisitivity
Integration Time
40 µs
Temperature
37°C
Sample
108A
108C
109A
109C
111A
111B
Reading
5932
6139
6050
5789
5589
6109
For next time: Since we did it without MACH cells, will redo tomorrow with MACH cells and biosensor to compare results.
6.03.15
Goal: Observe Sequence Data and compare to our own plasmids. Redo TECAN results with blank and biosensor to see differences in fluorescent data.
Compared sequence data to registry/our own plasmids. Figured out that no promoters got inserted in. Decided to send in Minipreps of the parts of the original devices for sequencing to see if the given parts were what we expected.
Redid fluorescence reading
Very evident difference, shows the presence of GFP
May redo in the future in order to do the trianalysis.
Used same TECAN parameters as 6.02.15
Sample
Control
108A
108C
109A
109C
111A
111B
Last Year
Reading
3379
6914
6097
6381
7696
5949
6814
OVER
6.04.15
Goal: Streaked out plates, continued competent cell protocol.
Streaked out MACH cells, Top10, and Biosensor Estrogen - on to LB plates. The biosensor estrogen negative plates had kanamyacin antibiotic on it (10 ul + 50 ul water).
Began TOP10 Competent cell protocol (protocol in the Protocol Page ).
Diluted TOP10 overnight culture 1:100 for a 2ml volume in LB. Grew it from 10a-5p. Then added 300 ul to each 500 ml of LB in sterile 2L flasks. Placed in 18C shaker overnight after removing the magnetic stir rod.
6.05.15
Goal: Get back plasmid sequences to determine what went wrong with what we did. Restreaked the plates and competent cells growth is slow therefore will continue procedure Monday 6/8/15.
Placed growing TOP10 cells in 2 L flasks in cold room because it was taking too long to grow in 18C shaker. Will place back in shaker Sunday night so that it can be ready on Monday to use (need OD600 of 0.4-0.6.)
Further sequencing results on original devices sent out on 6/3/15 were obtained on 6/5/15. It showed that the promoters have 2 additional Spe1 cut sites which meant that they were cut out, which is why none of the transformants had the promoter in them.
We notified iGEM, and they will thank us on their website for finding their mistake.
Started Transformations (protocol in the Protocol Page ) for New Set Today.
Week 3
6.07.15
Goal: Set up culture preps for minipreps
Set up culture preps for minipreps (5ml) with 5ul of chlorophenical.
Samples: 106A, 101A, 101B, 101C, 101D, 117A, 117B, 117C, 117D
Left overnight in 37C shaker.
6.08.15
Goal: Redo the interlab study with the new devices given.
Forgot to do ligation and transformation, therefore we will redo this tomorrow.
6.09.15
Goal: Redo digestion of the new transformations and determine if they fluoresce.
Redid the Digestion because we forgot to do transformations (protocol in the Protocol Page ).
Did 3504 digestion with 101B, 106A, 117C
For promoter: 10 µl of MiniPrep, 4 µl of H20, 2 µl of green buffer, 2 µl of EACH enzyme (PST and SPE1)
For insert: 12 µl of MiniPrep, 2 µl of H20, 2 µl of green buffer, 2 µl of EACH enzyme (PST and Xba1)
Put digests into a gel & washed gel with 1:1 buffer and put it into 42C heat shock for ~30 minutes
Did PCR purification
Ligation done using 4 µl of promoter, 4 µl of insert, 1 µl of buffer (t4 ligase buffer) LIG, 1 µl of T4 ligase (protocol in the Protocol Page ).
Did transformation for lacYFP, FFluc, PRE, POST (protocol in the Protocol Page ).
For lacYFP (can plate): 35 µl of cells, 1 µl YFP
For FFLuc, PRE, POST (5mL LB AMP cultures): 35 µl of competent cells, 1 µl of such samples
For our samples: 50 µl of competent cells, 10 µl of our mix (3504-101B, 106A, 117C)
Also redid competent cells (autoclaved the 500mL LB, store TOP10 into 37C put in around 1microliter ← slow growth so put in more in order to grow faster)
For next time: See if our devices are green fluorescing tomorrow and to fixate our cells to the W strain in the competent cells being made.
6.10.15
Goal: Screen the transformants for ones that uptook the plasmid and to check if competent cell culture is ready.
Screened the plates with transformants using UV light.
Screened the plates with transformants using UV light.
Set up 5ml culture using one colony from each (106A, 101B, 117C) for miniprep. Grew overnight in 37C shaker. Picked colony based on fluorescence using the UV light.
Competent cells still did not reach the appropriate OD (currently 0.1 at 9:30a). Allowing it to shake overnight again.
For next time: Purify and isolate the transformant DNA using miniprep on the cultures that were set up today.
6.11.15
Goal: Purify and isolate the transformant DNA using miniprep on the cultures and then send it out for sequencing. Begin the competent cell protocol.
Did miniprep (protocol in the Protocol Page ) on the 5ml cultures containing colonies of 101B (J23101 + I31504), 106A (J23106 + I31504), and 117C (J23117 + I31504)
Sent out the DNA for sequencing.
Competent cell culture was still did not have an OD of 0.4-0.6. Will check again tomorrow morning.
For next time: Begin PCR amplification on Gaussia Luciferase using Phusion high fidelity DNA polymerase.
6.12.15
Goal: Finish competent cell protocol and start the Gaussia Luciferase experimentation.
Competent cell culture was within the OD. One of the competent cell cultures were within the appropriate range (at OD of 0.48). The other one wasn’t (at OD of ~0.2). We just started the competent cell protocol on both cultures but kept them separate so that we could compare their efficiency. The “Making competent MACH T1 cells” protocol was used for making competent TOP10 cells (protocol in the Protocol Page ).
Week 4
6.15.15
Goal: Verifying the sequences obtained to the constructs that were made. Check the competence of our competent cells.
Verified by comparing the sequence of our constructs to the sequence got from the samples we sent out.
They seemed to be accurate with a few changes.
Checked competence of competent cells using the iGEM competent transformation efficiency kit.
Tubes labeled 1 w/ X conc are from the culture with OD within the range of 0.4-0.6. Tubes labeled 2 w/ conc are from culture that did not have an OD within the range of 0.4-0.6.
Tube 1 with concentration 0.5, 5, 10, 20, and 50 picogram (5 samples in total)
Tube 2 with concentrations 0.5, 5, 10, 20, and 50 picogram (5 samples in total)
Tube with MACH cells.
For next time: Will count the number of colonies and determine the transformation efficiency of the cells.
6.16.15
Goal: Verifying the sequences obtained to the constructs that were made. Check the competence of our competent cells.
Found nothing in both our competent cell plates and MACH plates from yesterday, using the DNA iGEM transformation efficiency kit.
Redid Competent cells and MACH cells with plasmid DNA
Plated both 400 µl and 40 µl for each competent cell transformation (total: 6 plates) & put in 37 °C incubator.
For next time: Check if our competent cells worked.
6.17.15
Goal: Verifying if our competent cells worked since it didn’t work using the DNA from iGEM.
Checked the plates and found cells in both MACH and competent cells.
6.18.15
Goal: Transform colonies into TOP10 Competent Cells.
Transformed colonies (101B, 106A, 117A) into TOP10 Competent Cells and then plated them.
1 µL of DNA, 50 µL of cells, 5 min ice, 2 min heat, 5 min ice, 500 µL LB, 1 hr at 37 °C, plate
Week 5
6.23.15
Goal: Make cultures and plate cells.
Made 2 mL overnight cultures of cells that will later be used in the microplate reader to measure fluorescence.
MACH empty, Top 10 empty, 3 devices + top 10, 3 devices + mach
Plated both the mach and competent cells overnight on LB plates to see if they would grow to use for the biological and technical replicate part of the interlab.
Technical is more how many times we repeated something.
Biological is doing new sets of colonies.
6.24.15
Goal: Measure fluorescence for MACH and competent cells.
Took out Mach and Top10 Competent Cells to do measurements of flouresence with on a TECAN microplate reader.
Used same TECAN parameters as 6.02.15 except used a Gain of 100 instead of 150.
Estrogen Sensor:
7.1.15
Tried newly transformed cells, all taken from the same plate
Tried to get similar-sized colonies
Tested the following amounts of estrogen:
100 uM, 20 uM, 10 uM, 1 uM, 100 nM, 10 nM, 1 nM, 0 nM
Got very varied and inconsistent results
No pattern between amount of estrogen added & reading
High standard deviation
Think that there is a problem with the transformation
Will try another transformation
7.2.15
Tried the restreaked  cells with 1:100,000, 1:1,000,000, and 1:10,000,000 estrogen dilutions
High standard deviation amongst duplicates, no pattern between amount of estrogen added & reading
Determined that restreaked cells do not work
7.7.15
Transformation 4 of Mach cells did not work
Gave unexpected results with high standard deviation
Did not seem to be a difference between the different amounts of estrogen
For tomorrow:
Do another transformation -- possibly get Donna & Michelle to do it
Try again with the colonies we have now
7.29.15
Made dilutions (1:100) from the starter cultures for the estrogen sensors and controls.
Set up transformations for CD-cel Gaussia (on plates and in culture) for testing of luciferase activity in response to addition of coelenterazine.
The isolation of CD-cel domain and gaussia domain didn’t work.
8.03.15
Started overnight cultures of the Sensor 6 cells from the restreaked plate
Did not add estrogen
Each overnight tube from a different colony
For the next overnight: will add 100 µL of the overnights into 2mL of fresh media & add different dilutions of estrogen then
Goal: want to see if different colonies gave consistent results
Want to see if lower dilutions of estrogen give lower readings
Made another set of starter cultures for estrogen sensor (with and without estrogen) and controls by adding a colony of each sample into 2 ml of fresh LB media. Cultures shook overnight in 37C shaker.
8.04.15
Added 20 ul of starter culture to 2 ml of fresh LB containing 2 ul of chlorophenical and 1.2 ul of kanamyacin. This was done three times for each sample. For the J23100 RFP and PelB Gaussia, however, no kanamyacin was added
7.29.15
2ml cultures consisting of 2 ul of chloramphenicol were made for J23100 RFP and J23115 RFP and left overnight in the 37C shaker.
7.30.15
Goal: To compare the fluorescence of J23100 RFP with J23115 RFP.
J23100 RFP did not show consistent results from previous data, mostly likely due to reason mentioned above. New cultures of J23100 RFP and J23115 RFP from newly streaked plates will be used to set up new cultures for measuring.
Re-tested the fluorescence of the J23115 RFP and J23100 since the first set of data from the previous day did not consist of biological replicates. However, the J23100 culture did not seem to grow well since the culture was not red. One reason could be that the plates from which the colonies were taken from were from 6.29.15 and not from the newer plate. This would explain why the fluorescence for J23100 RFP is much lower than expected, when it should be much higher than that for J23115 RFP.
8.3.15
Started three 2 ml overnight cultures, each consisting of 2 ul of chlorophenical, for  J23100 RFP and J23115 RFP
One 2ml overnight culture without chlorophenical was also made for empty MACH cells as a control.
8.4.15
Measured fluorescence for each overnight culture that was set up the day before (J23100 RFP, J23115 RFP, and empty MACH cells). These values are considered biological replicates.
Measured each sample three times (technical replicates)
Conclusion: The fluorescence of the J23100 RFP was 5 times larger than that of the J23115 RFP.
History about the Arduino
The Arduino is a piece of programmable hardware created by several graduate students in Italy in the early 2000’s. The name itself comes from a bar they used to hang out in, which in turn was named after an old Italian king, Arduin of Ivrea. It was created because the hardware they had previously been using was $100 and they knew some people couldn’t afford that, so they designed a cheaper alternative. This is the embodiment of the Maker’s Movement Era!
And with that, the Arduino was born. The model we use, the “Arduino Uno,” contains a programmable chip that we can code via USB, and it has fourteen digital pins along with eight analog pins that can be used to communicate with a circuit based on voltage levels. With this, the possibilities are endless! People have made remote control lawnmowers and quadcopters, they’ve built robots and video games, and they’ve even made sensors and alarms to be used for home security and ease of access! This is what the Arduino can do: bring easy circuit making to anyone’s door step to help improve quality of life.
