<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Carnegie_Mellon_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Carnegie Mellon/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV id="header"><UL id="outside_table"><LI><A class="head" href="https://2015.igem.org/Team:Carnegie_Mellon"> Home </A></LI><LI><A class="head" href="https://2015.igem.org/Team:Carnegie_Mellon/Team"> Team </A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Team#students">Students</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Team#advisors">Advisors</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Sponsors">Sponsors</A></LI></UL><LI><A class="head" href="https://2015.igem.org/Team:Carnegie_Mellon/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols"> Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Device">Device</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Measurement">Measurement</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/improvedpart">Improved Part</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/medal">Medal Requirements</A></LI><LI><A class="head" href="https://2015.igem.org/Team:Carnegie_Mellon/Interlab">Interlab</A></LI><LI><A class="head" href="https://2015.igem.org/Team:Carnegie_Mellon/Practices">Community</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/MakerMovement">Maker Movement</A></LI><LI><A href="https://2015.igem.org/Team:Carnegie_Mellon/Interviews">Interviews</A></LI></NAV><DIV class="jumbotron"><DIV class="container"><DIV class="title">Notebook.</DIV><P>We tracked everything in our handy-dandy notebook.</P></DIV></DIV><DIV class="mtabs_wrapper"><DIV id="mtabs"><UL><LI class="active"><A href="#tab1" rel="tab1">Bioluminescence Cloning</A></LI><LI><A href="#tab2" rel="tab2">Proteins</A></LI><LI><A href="#tab3" rel="tab3">Interlab Study</A></LI><LI><A href="#tab4" rel="tab4">Estrogen Sensor</A></LI><LI><A href="#tab5" rel="tab5">Modeling</A></LI><LI><A href="#tab6" rel="tab6">Project Specs</A></LI></UL></DIV><DIV id="mtabs_content_container"><DIV id="tab1" class="mtab_content" style="display: block;"><DIV class="weekheader"> Week 4 </DIV><DIV class="dateheader"> 6.12.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To amplify the Gaussia Luciferase DNA using PCR 

			<UL><LI> 1a. Conduct a Gaussia Luciferase experiment where we resuspended the DNA using this protocol:</LI><DIV class="indentlist"><UL>20µl of water into the 200ng tubes</UL></DIV><DIV class="indentlist"><UL>Use 5µl in PCR</UL></DIV><LI> 1b. Primers </LI><DIV class="indentlist"><UL>10µl of stock with 90µl H20 → 10µM</UL></DIV><LI> 2. Then we did a Phusion PCR using the following measurements: </LI></UL></DIV><DIV class="dateheader"> 6.15.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>To PCR amplify the PelB Gaussia DNA and test the competence of competent cells. 
			<UL><LI>Checked to see if the gel sizes were correct.</LI><DIV class="indentlist"><UL>We split PelB PCR into two tubes of 25µl each for digestion because we eluted with 50µl of H20 during PCR purification instead of 30µl. That’s why for digest we added 3.2µl of buffer and 2 of each enzyme.</UL></DIV><LI>If so, then did digestion for PCR product</LI><DIV class="indentlist"><UL>2µl of each enzyme: Xba1 &amp; PST (Can also use EcoRI instead of Xbal) </UL></DIV><DIV class="indentlist"><UL>3.2µl of buffer </UL></DIV><DIV class="indentlist"><UL>No water because volume is too large (from PCR) </UL></DIV><LI>Put digestion pieces into new gel to extract out separated DNA </LI><LI>1:1 binding buffer on gel extraction (PCR purification)</LI><LI>Digested the backbone (101A) using the same enzymes</LI><LI>Digestion</LI><DIV class="indentlist"><UL>14µl of 101A backbone </UL></DIV><DIV class="indentlist"><UL>2µl of buffer </UL></DIV><DIV class="indentlist"><UL>2µl of each enzyme (xba1 &amp; pst1) </UL></DIV><LI>Gel Extraction (Cut out 2000 bp band). However because we were very confused, we cut out the incorrect band</LI><LI>Gel purification</LI><LI>Ligation</LI><DIV class="indentlist"><UL>4:4 ratio</UL></DIV><DIV class="indentlist"><UL>1µl buffer</UL></DIV><DIV class="indentlist"><UL>1µl ligase</UL></DIV><DIV class="indentlist"><UL>However we were only suppose to do a 2µl : 2µl  ratio of digestion product : insert to backbone</UL></DIV><LI>Transformation</LI><DIV class="indentlist"><UL>50µl  competent cells (our competent cells)</UL></DIV><DIV class="indentlist"><UL>5µl of ligation but it should have been the entire 10µl of ligation (screwed up which would explain why our yield was so low </UL></DIV><DIV class="indentlist"><UL>500µl of LB</UL></DIV><LI>Plated 400µl of transformants in 37C (our yield was low because we messed up on the concentration of ligation and the digestion product to insert ratio) </LI></UL></DIV><DIV class="dateheader"> 6.16.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>To PCR amplify the rest of the DNAs. 
			<UL><LI>Checked the plates to see of PelB Gaussia worked. It did not because we still saw fluorescence in colony which shouldn’t have happened if we extracted just the backbone so redid cloning.</LI><LI>PCR Phusion on all new samples not the old PelB because that was already amplified.</LI><DIV class="indentlist"><UL>Firefly Luciferase CO</UL></DIV><DIV class="indentlist"><UL>CD-celmRFP</UL></DIV><DIV class="indentlist"><UL>PelB-RFP</UL></DIV><DIV class="indentlist"><UL>CD-celGaussia</UL></DIV><DIV class="indentlist"><UL>Renilla Luciferase CO</UL></DIV><DIV class="indentlist"><UL>J23100-E0024-RFP (?)</UL></DIV><LI>PCR purification eluted in 30µl</LI><LI>Set up digestion for PCR product</LI><DIV class="indentlist"><UL>2µl of each enzyme: Xba1 &amp; PST (Can also use EcoRI instead of Xbal)</UL></DIV><DIV class="indentlist"><UL>3.7µl of buffer </UL></DIV><DIV class="indentlist"><UL>30µl sample (PCR product)</UL></DIV><LI>Put digestion pieces into new gel to extract out separated DNA</LI><LI>1:1 binding buffer on gel extraction (PCR purification)</LI><LI>Digested the backbone (101A) using the same enzymes</LI><LI>Digestion</LI><DIV class="indentlist"><UL>14µl of 101A backbone</UL></DIV><DIV class="indentlist"><UL>2µl of buffer</UL></DIV><DIV class="indentlist"><UL>2µl of each enzyme (xba1 &amp; pst1) </UL></DIV><LI>Gel Extraction (Cut out 2000 bp band) low intensity and potentially could have cut the wrong band (cut out middle band, which we thought was around 2000bp).</LI></UL></DIV><DIV class="dateheader"> 6.17.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>To Ligate and Transform J23100, PelBGaussia, CelmGaussia with 101A
			<UL><LI>Ligation</LI><DIV class="indentlist"><UL>2:2 ratio of J23100,PelBGaussia,CelmGaussia: 101A (backbone)</UL></DIV><DIV class="indentlist"><UL>1µl buffer</UL></DIV><DIV class="indentlist"><UL>1µl ligase</UL></DIV><LI>Transformation</LI><DIV class="indentlist"><UL>50µl  competent cells (our competent cells)</UL></DIV><DIV class="indentlist"><UL>10µl of ligation</UL></DIV><DIV class="indentlist"><UL>500µl of LB</UL></DIV><LI>Plated 400µl of transformants in 37C</LI></UL></DIV><DIV class="dateheader"> 6.18.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>Pick out to see if Transformants worked and then send them out to sequencing.
			<UL><LI>Observed the three colonies below:</LI><DIV class="indentlist"><UL>J23100-101A</UL></DIV><DIV class="indentlist"><UL>CelmGaussia-101A</UL></DIV><DIV class="indentlist"><UL>PelBGaussia-101A</UL></DIV><LI>The J23100 did not fluoresce red therefore we didn’t extract it through the PCR. We only did colonies found in CelmGaussia-101A (2 colonies) and PelBGaussia-101A (6 colonies). It might be worrisome that we did not have much colonies after ligation but something is there. 
</LI><LI>For each colony, we suspended in 50µl H20 and added 1µl of the suspension into PCR tube with 9µl MM.</LI><LI>The MM protocol can be found in the other spreadsheet in the folder. We did have 8 samples but since the volume is 9 each, we actually had the volume for 8 PCRs so 4µl * 20 is equal to 8µl*10.</LI><LI>Used the Program: CherylTAQ30s and then put it through the gel extraction.</LI><LI>After gel extraction, will do miniprep (Gel extraction photo in folder).</LI><LI>Prepared minipreps on samples where the backbone was incorporated into Gaussia DNA samples. (CelmGaussia Samples did not work)</LI><DIV class="indentlist"><UL>PelBGaussia1</UL></DIV><DIV class="indentlist"><UL>PelBGaussia2</UL></DIV><DIV class="indentlist"><UL>PelBGaussia3</UL></DIV><DIV class="indentlist"><UL>PelBGaussia4</UL></DIV><DIV class="indentlist"><UL>PelBGaussia5</UL></DIV><LI>5 of each sample, 5 Chlorophenocol, 500 LB</LI><LI>Overnight at 37 room</LI><LI>Will miniprep tomorrow.</LI><LI>Prepared other DNA samples (only had backbone after we transformed it)</LI><DIV class="indentlist"><UL>J23100-backbone</UL></DIV><DIV class="indentlist"><UL>Renilla Luciferase-backbone</UL></DIV><DIV class="indentlist"><UL>CD-CelmRFP-backbone</UL></DIV><DIV class="indentlist"><UL>PelBRFP-backbone</UL></DIV><LI>Ligated </LI><DIV class="indentlist"><UL>4:4 Each sample colony with 101A backbone</UL></DIV><DIV class="indentlist"><UL>1 of Ligate</UL></DIV><DIV class="indentlist"><UL>1 of Ligation T4 Buffer</UL></DIV><LI>Transformed </LI><DIV class="indentlist"><UL>1 of DNA</UL></DIV><DIV class="indentlist"><UL>50 TOP10 cells</UL></DIV><DIV class="indentlist"><UL>5min Ice</UL></DIV><DIV class="indentlist"><UL>2min heatshock</UL></DIV><DIV class="indentlist"><UL>5min Ice</UL></DIV></UL></DIV><DIV class="dateheader"> 6.24.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>To re-transform the CelGaussia with competent cells to see if we can get colonies as well as grow mach/competent cells for the Coelenterzine insert.
			<UL><LI>Examined both the Firefly and Gaussia plates and saw that the Firefly fluorescence but the Gaussia did not.</LI><LI>Re-transformed the CDCel Gaussia and grew competent cells in 5 µl for the Coelenterazine insert.</LI><LI>Left the samples overnight at 37°C. </LI></UL></DIV><DIV class="dateheader"> 6.25.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>To check if the transformations put into 37°C.
			<UL><LI>Unfortunately we realized that CelGaussia was GFP positive meaning the clone did not work properly. In addition, the Firefly was not RFP positive meaning it also did not work properly</LI><LI>We re-transformed the Phusion PCR using celGaussia. </LI><LI>Did the Colony PCR for the Firefly.</LI><DIV class="indentlist"><UL>We picked 4 single colonies to do the PCR colony. </UL></DIV><DIV class="indentlist"><UL>Used 50 µl of water + 1 colony in each tube.</UL></DIV></UL></DIV><DIV class="dateheader"> 6.26.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To continue the bioluminescence Project
			<UL><LI>Continue purifying and isolating the DNA for CD-Gauss</LI><LI>Continue with setting up transformations for the remaining luminescence DNA. Remaining samples left to re-transform into backbon</LI><DIV class="indentlist"><UL>PelB RFP</UL></DIV><DIV class="indentlist"><UL>CD-cel mRFP</UL></DIV><DIV class="indentlist"><UL>Cel-Gaussia</UL></DIV><LI>We digested the PCR Product of CD-Gaussia with restriction sites: Xba &amp; Ps</LI><DIV class="indentlist"><UL>We only did 25 µl and saved 5 µl in case the digestion did not work appropriately. That way, there will be more DNA to amplify aga</UL></DIV><LI>We also sent out 133-135 for sequencing using the similar labeling system:</LI><DIV class="indentlist"><UL>Renilla 1 (iGEM 133): VR</UL></DIV><DIV class="indentlist"><UL>Renilla 2 (iGEM 134): VR</UL></DIV><DIV class="indentlist"><UL>Firefly 2 (iGEM 135): V</UL></DIV></UL></DIV><DIV class="dateheader"> 6.30.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To colony PCR a few of the samples collected. Then using DNA IDT, send the PelB, RFP, and CelCD RFP to sequenc
			<UL><LI>Check to see if samples fluorescent</LI><LI>Set up colony PCR for different vector:insert dilutions of CDcelGaussia</LI><LI>Then I did a Tecan analysis of CD Cel RFP and CD Gaussia</LI><LI>Samples of CD Cel RFP and PelB RFP (4 samples each) were miniprepped and placed into overnight cultures.</LI><LI>After miniprep, the samples were physically examined to make sure the red is present</LI><LI>The samples were then sent to sequencing and labelled as such: </LI><DIV class="indentlist"><UL>IGEM 136: CDCelRFP 1 Forward</UL></DIV><DIV class="indentlist"><UL>IGEM 137: CDCelRFP 1 Reverse</UL></DIV><DIV class="indentlist"><UL>IGEM 138: CDCelRFP 2 Forward</UL></DIV><DIV class="indentlist"><UL>IGEM 139: CDCelRFP 2 Reverse</UL></DIV><DIV class="indentlist"><UL>IGEM 140: CDCelRFP 3 Forward</UL></DIV><DIV class="indentlist"><UL>IGEM 141: CDCelRFP 3 Reverse</UL></DIV><DIV class="indentlist"><UL>IGEM 142: CDCelRFP 4 Forward</UL></DIV><DIV class="indentlist"><UL>IGEM 143: CDCelRFP 4 Reverse</UL></DIV><DIV class="indentlist"><UL>IGEM 146: PelBRFP 3 Forward</UL></DIV><DIV class="indentlist"><UL>IGEM 147: PelBRFP 4 Forward</UL></DIV><DIV class="indentlist"><UL>iGEM 148: Firefly 2 Reverse</UL></DIV></UL></DIV><DIV class="dateheader"> 7.01.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To set up new gblock on the collected sample
			<UL><LI>Set up and reamplify new gblock through phusion gel</LI><LI>Set up cultures in order to do colony PCR the next day. </LI></UL></DIV><DIV class="dateheader"> 7.10.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To set up and send out sequences for sequencing. 

			<UL><LI>DNA samples were sent out for sequencing. These samples were:</LI><DIV class="indentlist"><UL>161: CD Gaussia Prefix </UL></DIV><DIV class="indentlist"><UL>162: CD Gaussia Suffix</UL></DIV><DIV class="indentlist"><UL>163: CD-cel RFP (CD-cel Forward)</UL></DIV><DIV class="indentlist"><UL>164: CD-cel RFP (CD-cel Reverse)</UL></DIV><DIV class="indentlist"><UL>165: CD-cel RFP (CD-cel Forward)</UL></DIV><DIV class="indentlist"><UL>166: CD-cel RFP (CD-cel Reverse)</UL></DIV><DIV class="indentlist"><UL>167: CD-cel RFP (CD-cel Suffix)</UL></DIV><DIV class="indentlist"><UL>168: 115 RFP1</UL></DIV><DIV class="indentlist"><UL>169: 115 RFP2</UL></DIV></UL></DIV><DIV class="dateheader"> 7.13.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To retransform T7 with estrogen sensor, RNA polymerase, YFP with the sites, and YFP without the sites. 
			<UL>

			What to do: 
                                        <DIV class="indentlist"><UL>Retransform T7 with YFP with the sites, YFP without the sites, and RNA polymerase. </UL></DIV><DIV class="indentlist"><UL>Plate them and set up cultures with J23100 RFP and T7 YFP without sites whose plate was lacking in colonies</UL></DIV></UL></DIV><DIV class="dateheader"> 7.15.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To clone the T7 RNA Polymerase, T7 YFP with sites and T7 YFP without the sit 

			<UL>
			The colonies on the plate with the T7 YFP without the sites was growing extremely slowly so it was left in the 37C incubator for another night. For the other coplasmid transformations (T7 YFP with the sites and the T7 RNA polymerase), the colonies were yellow under UV light. Overnight cultures were set up so that the luminescence could be measured in the morning. Minipreps were done on the T7 RNA Polymerase and J2310
                                </UL></DIV><DIV class="dateheader"> 7.16.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To compare the luminescence of the Estrogen Sensor with its controls

			<UL>
			Measured the luminescence of the estrogen sensor (with and without estrogen) along with its controls (T7 RNA polymerase, T7 YFP with sites, T7 YFP without sites) by adding 10 ul of 2uM coelenterazine.  

                                </UL></DIV><DIV class="dateheader"> 7.17.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To compare the luminescence of the Estrogen Sensor with its controls.

			<UL>
			Measured the luminescence of the estrogen sensor (with and without estrogen) along with its controls (T7 RNA polymerase, T7 YFP with sites, T7 YFP without sites) by adding 10 ul of 2uM coelenterazine. Two biological replicates and three technical trials were done for each sample. 
                                </UL></DIV><DIV class="dateheader"> 7.20.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To redo T7 Tecan Result
			<UL><LI>Took cultures out of 37C which was put in on Sunday. Did the Tecan Results for that is found in this folder. </LI><LI>Minipreped 101B backbon </LI><LI>Phusion PCR of new iDT order (fluorescent Proteins) </LI><LI>Set up cultures</LI></UL></DIV><DIV class="dateheader"> 7.21.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To do TECAN results and PCR the new fluorescent proteins
			<UL><LI>Made PCR Phusion for new fluorescent proteins</LI><LI>measured TECAN results which can be seen in the excel file</LI></UL></DIV><DIV class="dateheader"> 7.22.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To clone the co-optimized fluorescent genes into bacterial MACH cells
			<UL><LI>Phusion PCR was done to amplify the DNA for the fluorescent proteins from the gBlocks. PCR products were run on a gel to verify the sizes.</LI><DIV class="indentlist"><UL>Lane 1: ladder </UL></DIV><DIV class="indentlist"><UL>Lane 2: BFP HCO</UL></DIV><DIV class="indentlist"><UL>Lane 3: eGFP HCO</UL></DIV><DIV class="indentlist"><UL>Lane 4: GFP HCO</UL></DIV><DIV class="indentlist"><UL>Lane 5: OFP HCO </UL></DIV><DIV class="indentlist"><UL>Lane 6: YFP HCO</UL></DIV><DIV class="indentlist"><UL>Lane 7: E0040</UL></DIV></UL></DIV><DIV class="dateheader"> 7.23.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To clone the co-optimized genes for the fluorescent proteins into MACH bacterial cells. 

			<UL><LI>Ran a digest on the purified Phusion PCR products using xba1 and spe1 restriction digest enzymes. Gel extraction was done to purify the DNA and to verify the sizes. The DNA with the appropriate sizes were transformed into bacterial MACH cells with freshly cut vector. </LI><DIV class="indentlist"><UL>Lane 1: ladder </UL></DIV><DIV class="indentlist"><UL>Lane 2: BFP </UL></DIV><DIV class="indentlist"><UL>Lane 3: eGFP </UL></DIV><DIV class="indentlist"><UL>Lane 4: GFP </UL></DIV><DIV class="indentlist"><UL>Lane 5: OFP</UL></DIV><DIV class="indentlist"><UL>Lane 6: YFP </UL></DIV><DIV class="indentlist"><UL>Lane 7: E0040</UL></DIV>
                                Starter cultures were made for the estrogen sensor and controls. 
				
                                </UL></DIV><DIV class="dateheader"> 7.24.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  Cultures that were made on 7.23 for the estrogen sensor and its controls were diluted 1:100 were grown overnight in the 37C shaker and will be measured using the luminometer tomorrow.
                                </DIV><DIV class="dateheader"> 7.25.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To analyze the effects of estrogen sensor with and without estrogen along with its controls. 
			<FONT color="orange"><B><U>What we did:</U></B></FONT>  The luminescence of the estrogen sensor (with and without estrogen) and its controls were measured.
                                </DIV><DIV class="dateheader"> 7.28.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  Measure the luminescence and response from the estrogen sensor. Compare the fluorescence from J23100 RFP and J23115 RFP. To analyze the sequence for CD-cel Gaussia and to determine if there is luciferase activity. 

			<FONT color="orange"><B><U>What we did:</U></B></FONT><UL><LI>Made dilutions (1:100) from the starter cultures for the estrogen sensors and controls. Made cultures for the J23100 RFP and 115 2 RFP. Left them overnight in the 37 dry shaker</LI><LI>Set up transformations for CD-cel Gaussia (on plates and in culture) for testing of luciferase activity in response to addition of coelenterazine. </LI><LI>Checked the sequence data for CD-cel Gaussia. It works so cultures will be set up for luciferase activity testing. The CD-cel domain and gaussia domain didn’t work. </LI></UL></DIV></DIV><DIV id="tab2" class="mtab_content" style="display: block;"><DIV class="weekheader"> Week 3 </DIV><DIV class="dateheader"> 6.8.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To begin quantifying fluorescent proteins produced in MACH cells: mRFP, OFP, YFP, eGFP, BFP. Also to create a strongly expressing culture of YFP
			<UL><LI>Created 5 mL cultures from previous cultures, with the exception of the YFP transformed bacteria, which was taken from a plate created 5/27/2015.</LI><LI>Streaked a new multicolor plate with all his tagged fluorescent proteins </LI><LI>Calculated OD of 1:10 dilution of each culture using online calculator
					<DIV class="indentlist"><UL>OD was 0.369, thus the non dilute concentration of bacteria we used was 2.95 * 10^9, and the amount of bacteria in each sample was 4.425*10^9</UL></DIV></LI><LI>His-tagged soluble protein extraction outlined in our procedure using a washing solution with dilute imidazole
					<DIV class="indentlist"><UL>Used 50 uL of beads for each</UL><UL>extracted proteins from 1.5 mL of each culture</UL></DIV></LI><LI>OD was 0.369, thus the non dilute concentration of bacteria we used was 2.95 * 10^9, and the amount of bacteria in each sample was 4.425*10^9</LI><LI>Read fluorescence in a microplate reader</LI><LI>YFP is still not expressing, and we did indeed create a culture from a weak colony</LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> We will test our new cultures, which should be expressing well, as we marked strong colonies on the initial YFP plate. We will try a pH elution after extraction on Ni beads, as adding too much imidazole will interfere with fluorescence readings 
			</DIV><DIV class="dateheader"> 6.9.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To test the yield and purity of our protein extraction methods using a representative transformed E. coli culture. We will use a pH 4.5 or lower buffer to elute his tagged fluorescent protein from the Ni NTA beads to quantify in the Nano Drop spectrometer.
			<UL><LI>Created 5 mL cultures from previous cultures, with the exception of the YFP transformed bacteria, which was taken from a plate created 5/27/2015.</LI><LI>Streaked a new multicolor plate with all his tagged fluorescent proteins </LI><LI>Calculated OD of 1:10 dilution of each culture using online calculator</LI><LI>His-tagged soluble protein extraction (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>) using a washing solution with dilute imidazole
					<DIV class="indentlist"><UL>Used 50 uL of beads for each</UL><UL>extracted proteins from 1.5 mL of each culture</UL></DIV></LI><LI>Performed elution from beads
					<DIV class="identlist"><UL>Used 100 mM EDTA / 10 mM Tris solution  to elute proteins from beads</UL><UL>incubated for 5 minutes</UL><UL>removed supernatant and repeat for a second elution</UL><UL>Quantified solution with NanoDrop</UL></DIV></LI></UL><FONT color="orange"><B><U>Results:</U></B></FONT><UL><LI>OD of cultures was 0.356 in a 1:10 dilution, thus 4.275 * 10^9 bacteria in each 1.5mL sample.</LI><TABLE><TBODY><TR><TD><B>Concentration of Eluted mRFP</B></TD></TR><TR><TD><B><FONT color="orange">First Elution</FONT></B></TD><TD><B><FONT color="orange">Second Elution</FONT></B></TD></TR><TR><TD>6.15 mg/mL</TD><TD>1.6 mg/mL</TD></TR><TR><TD>6.353 mg/mL</TD><TD>1.777 mg/mL</TD></TR><TR><TD>6.352 mg/mL</TD><TD>1.523 mg.mL</TD></TR><TR><TD>Mean: 6.285 mg/mL</TD><TD>Mean: 1.63333 mg/mL</TD></TR><TR><TD>0.11691 mg/mL</TD><TD>0.13024 mg/mL</TD></TR></TBODY></TABLE><LI>Total amount of protein: 395.92 µg</LI><LI>The elution used before the EDTA elution did not work -- had a pH of 5.0 &amp; needed a pH of 4.5.</LI><LI>YFP had trouple growing &amp; flourescing.</LI><LI>Linked is a picture of the trouble with the YFP plates we've made so far<A href="#">Link of the three YFP plates &amp; description.</A></LI></UL></DIV><DIV class="dateheader"> 6.10.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To improve yield of fluorescent proteins eluted from Ni NTA beads during His-tagged soluble protein extraction. 
			<UL><LI>Carried out His-tagged soluble protein extraction as normal and then proceeded to elute 2 samples (50 uL of bead solution with protein from 1.5 mL of culture). </LI><LI>One bead sample eluted three times with 5 min. incubation period in 250 µL of 100 mM EDTA / 10 mM Tris solution for each elution</LI><LI>Second sample eluted twice with 10 min. incubation period in  250 µL  of 100 mM EDTA / 10 mM Tris solution for each elution
					<DIV class="indentlist"><UL>For five minute elutions, first elution had protein concentration of 0.95mg/mL, second elution had 0.355mg/mL, and the third had 1.004 mg/mL.</UL><UL>For ten minute elutions, the first elution had 0.615 mg/mL concentration and the second elution had 0.328 mg/mL concentration.</UL><UL>In total, the 5 minute elution had 384.83 µg and the 10 minute elution had 235.75 µg per 1.5 mL culture.</UL></DIV></LI><LI>Ran gel from yesterday's elutions. <A href="#">Link to results here.</A></LI></UL></DIV><DIV class="dateheader"> 6.10.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Refine procedure for quantifying concentration of fluorescent proteins in 1 L of culture. Focus on how higher volume of EDTA buffer affects the amount of protein eluted from Ni NTA beads.
			<UL><LI>Carried out His-tagged soluble protein extraction and eluted 2 samples (50 uL of bead solution with protein from 1.5 mL of culture, 2 cultures of OFP transformed E. coli, and another 2 of mRFP transformed E. coli). 
					<DIV class="indentlist"><UL>One bead sample was eluted three times with 5 min. incubation period in 25 µL of 100 mM EDTA / 10 mM Tris solution for each elution</UL><UL>Second bead sample was eluted twice with a 10 min. incubation period in  50 µL  of 100 mM EDTA / 10 mM Tris solution for each elution</UL><UL>Eluted proteins quantified using a NanoDrop spectrophotometer.</UL><UL>Total amount of protein: 25 µL elution mRFP: 56.3235 µg, 50 µL elution mRFP: 81.275 µg, 25 µL elution OFP: 53.6605 µg, 50 µL elution OFP: 142.4135 µg</UL></DIV></LI></UL></DIV><DIV class="dateheader"> 6.12.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Continue work on the elution of the proteins from Ni beads.
			<UL><LI>Performed his-tagged soluble protein extraction on 4 1.5 mL cultures transformed to produce mRFP, and 4 transformed to produce OFP.</LI><LI>Elution with 100 mM EDTA / 10 mM Tris solution for 50, 70, 85, and 100 µL volumes for each elution, 3 rounds of elution for each sample.</LI><LI>Finally have a plate with expression of all 5 fluorescent proteins. </LI></UL></DIV><DIV class="weekheader"> Week 4 </DIV><DIV class="dateheader"> 6.16.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Work on technique for using bio rad protein assay  that works with our sample concentration and our spectrophotometer.
			<UL><LI>Standard His-tag soluble protein extraction onto 50 μL of Ni-NTA bead solution</LI><LI>3 elutions using 5 min incubation time in 50 μL of EDTA buffer</LI><LI>The concentration of 1 μg/mL that we used for the BSA standard curve did not reveal any relevant information. In the future we will use standards ranging from 3 ug/mL to 20 ug/mL to encompass our data.</LI></UL></DIV><DIV class="dateheader"> 6.17.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Move fluorescent proteins from MACH competent cells, a K-strain to TOP10 competent cells, which are a W-strain (required for iGEM).
			<UL><LI>The procedure followed can be found in the Protocols page </LI></UL></DIV><DIV class="dateheader"> 6.18.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>Run another Bio-Rad Protein Assay test with a BSA standard curve to ensure we have the technique down and can replicate it once we have our TOP10 cells.
			<UL><LI>See 6/16/2015 for the protocol on BSA standards and testing protein concentration with the Bio-Rad Protein Assay. Today we used the range 3 μg/mL to 20 μg/mL as opposed to 1-10 which we used on 6/16.
					<DIV class="indentlist"><UL>Calculated that we have 2.543 mg of mRFP protein in 1.5mL culture.</UL></DIV></LI><LI>Transformed cells from before did not work, performed transformations again.</LI></UL></DIV><DIV class="dateheader"> 6.19.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Move fluorescent proteins from MACH competent cells to TOP10 competent cells. There were some problems with the overnight cultures and plates, which we have to correct.
			<UL><LI>Retransform the plasmids into new plates</LI><LI>Add IPTG to new cultures to induce gene expression</LI><LI>Restart cultures from yesterday in new media (did not add IPTG) &amp; started cultures from transformation (added IPTG)</LI><LI>Measure growth of TOP10 cells relative to growth of MACH cells (using OD measurements). The following is the OD in Top10 cells
					<DIV class="indentlist"><UL>mRFP: 0.102, OFP: 0.056, YGP: 0.053, GFP: 0.054, BFP: 0.213</UL></DIV></LI></UL></DIV><DIV class="weekheader"> Week 5 </DIV><DIV class="dateheader"> 6.22.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Test the effect of IPTG on the growth of the culture.
			<UL><LI>Used UV light to check relative expression levels of each fluorescent protein and marked colonies</LI><LI>Took colonies from plates and put in 5 mL LB, adding 5 μL LPTG and 5uL chloramphenicol</LI><LI>IPTG cultures that were started from the transformation on Friday did not grow, were not expressing the protein, and were all the same color.</LI><LI>The cultures started from culture grew fairly well. Will compare them tomorrow with the other two cultures (from culture, from transformation compared to from plate).</LI><LI>Weak expression of IPTG culture was most likely due to starting it from a transformation rather than a colony. These cultures were thrown out.</LI></UL></DIV><DIV class="dateheader"> 6.23.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To extract the proteins from our new cultures that were grown with IPTG.
			<UL><LI>Followed standard protein extraction protocol.</LI></UL></DIV><DIV class="dateheader"> 6.24.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To quantify an array of fluorescent proteins produced in Top 10 E. coli cells and subsequently characterize their relative fluorescence in order to connect the amount of protein to fluorescence.
			<UL><LI>Normal his tagged soluble protein extraction on 50uL of Ni-NTA bead solution for mRFP, OFP, YFP, eGFP, and BFP</LI><LI>3 rounds of elution in 50 uL of EDTA buffer for each protein</LI><LI>Bio-rad Protein Assay microassay protocol using BSA standards of 3, 9, 15, and 20 ug/mL</LI><LI>Fluorescence measurements taken on TECAN Safire2 with 100uL of culture and 100uL of extracted protein for each culture transformed to express fluorescent proteins</LI></UL></DIV><DIV class="weekheader"> Week 6 </DIV><DIV class="dateheader"> 6.29.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Start work with the Gaussia Luciferase and continue work with fluorescent proteins in TOP10 cells.
			<UL><LI>Started five 2 mL cultures of the Luciferase for experiments tomorrow.</LI><LI>Restreaked fluorescent proteins that were transformed last week into TOP10 cells.</LI><LI>Start 5 mL cultures of the fluorescent proteins for purification tomorrow. Started these from the plates that were grown on 6/19.</LI></UL>
			`</DIV><DIV class="dateheader"> 6.30.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Run the TECAN and the BCA protein assay test on the proteins we purified last week
			<UL><LI>Conduct standard BSA protein assay protocol. Use same standards as last week &amp; same dilutions of culture for sample readings (1:10, 1:100)</LI><LI>Run TECAN to try &amp; find correlation between concentration of protein in culture with fluorescence data.</LI><LI>Made new BSA standards for use in BCA protein assay experiments. Made four standards; 3 μg/mL, 9 μg/mL, 15 μg/mL, and 20 μg/mL.</LI><LI>For 3 µg/mL of BSA, the absorbance @ 595 nm was 0.047; for 9 µg/mL, 0.707 absorbance; 15 µg/mL had 0.665; 20 µg/mL had 0.792.</LI></UL></DIV><DIV class="dateheader"> 7.01.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Test the Arduino-based luminometer with the Luciferase cultures we started yesterday.
			<UL><LI>Used TECAN plate and a tube to test Light Dependent Resistor and make sure it is sensitive enough to make the measurements we need.</LI><LI>Added 100 μL cells to a TECAN well. Kept lights off and the Arduino recording data until it read no signal. Added coelenterazine into cells and observed what happens. Expected cells to emit light.</LI><LI>Did not have much luck with these tests (the LDR could not pick up such a faint signal)</LI><LI>Tried to purify luciferase to get a better response.</LI><LI>There was likely no response because of the cheap diode used.</LI><LI>Started an overnight from the No IPTG cultures. Added 500 mL of the No IPTG culture to 5 mL LB, 5 μL chloramphenicol and 5 μL IPTG.</LI><LI>Started overnights from the Luciferase cultures; three 2 mL &amp; two 1 mL. Added 1% chloramphenicol and 10% culture to the media.</LI><LI>Will begin using the Arduino Mega instead of the Arduino Uno in order to have access to more SRAM (Static Random Access Memory).</LI></UL></DIV><DIV class="weekheader"> Week 7 </DIV><DIV class="dateheader"> 7.06.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Test LDR with fresh dilution of coelenterazine &amp; new Arduino Mega, determine if purification is necessary to elicit signal, and test fluorescence of the beads that were introduced into the supernatant of the spun down Luciferase (from 7/1).
			<UL><LI>Tested luminescence in the well of a TECAN plate.
					<DIV class="indentlist"><UL>Added coelenterazine in various concentrations to estimate approximate concentration needed.</UL></DIV></LI><LI>Compared readings from purified proteins and culture by adding coelenterazine and measuring in TECAN machine.</LI><LI>The LDR was still not sensitive enough to read the luminescence from the culture.
					<DIV class="indentlist"><UL>Could see with the naked eye that the culture was giving off light, the LDR just couldn’t detect it.</UL><UL>Couldn’t make conclusions about concentration of coelenterazine or necessity of purification because we were not getting enough signal to quantify it.</UL></DIV></LI></UL></DIV><DIV class="dateheader"> 7.08.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Restreak TOP10 cells that contain fluorescent proteins
			<UL><LI>Restreaked cells with flourescent proteins.</LI><LI>Did an IPTG restart designed to induce the fluorescence of the cell in TOP10</LI></UL></DIV><DIV class="dateheader"> 7.09.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Complete IPTG restart &amp; quantify fluorescence of the cultures and purified proteins.
			<UL><LI>Did not complete the IPTG restart because the cultures did not grow properly overnight. Had an OD of 0.304, needed an OD of at least 0.6.</LI><LI>Yesterday's plate also did not grow properly. Only GFP had strong growth.</LI><LI>Started six new 5 mL Luciferase overnight cultures.</LI></UL><FONT size="4"><UL>Seen below is the protocol we standardized and distributed to labs like the Citizen Science Center in an effort to gain more uniformity between data observed in different labs around the world. The back of the card has details on how to elute the protein off on the washed Ni-NTa beads. A download link for the JPEG can be found below the images.</UL></FONT><CENTER><FONT size="4"><B>FRONT</B></FONT><IMAGE src="https://static.igem.org/mediawiki/2015/thumb/0/00/Protein_purification_1.jpg/800px-Protein_purification_1.jpg"><FONT size="4"><B>BACK</B></FONT><IMAGE src="https://static.igem.org/mediawiki/2015/thumb/d/d4/Protein_purification_2.jpg/800px-Protein_purification_2.jpg"><A href="https://static.igem.org/mediawiki/2015/0/0b/Protein_pur_card_FILES.zip"> PROTEIN PURIFICATION CARD JPEG </A></IMAGE></IMAGE></CENTER></DIV></DIV></DIV><DIV id="tab3" class="mtab_content"><DIV class="weekheader"> Week 1 </DIV><DIV class="dateheader"> 5.28.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To do Miniprep to purify the DNA. It is a small scale isolation of plasmid DNA from bacteria. Then do restriction enzyme digest on plasmid DNA and gel electrophoresis. This technique can be used for cleaving DNA molecules at specific sites, ensuring that all DNA fragments that contain a particular sequence have the same size; furthermore, each fragment that contains the desired sequence has the sequence located at exactly the same position within the fragment. The cleavage method makes use of enzymes isolated from bacteria called restriction enzymes. They are able to cleave DNA molecules at positions at which particular short sequences of bases are present. 

			<UL><LI>Did MiniPreps of MACH cells expressing fluorescence to isolate plasmid DNA from MACH cells (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>)</LI><LI>Performed restriction enzyme digestion on J23108, J23109, J23111 constructs
					<DIV class="indentlist"><UL>For promoters, PST1 &amp; Spe1 were used. For insert, xba1 and Pst1 were used. </UL></DIV></LI><LI>Ran an Agarose Gel Electrophoresis
                                        <DIV class="indentlist"><UL>Expected size of insert (from E0040) - 720 bp</UL></DIV><DIV class="indentlist"><UL>Expected size of vector backbone (J23108, J23109, J23111) - 2105 bp</UL></DIV></LI><LI>Cut digests from gels (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>).</LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> Will prepare Colony PCR to determine if we got the correct restriction sites and sequences from the given interlab devices. </DIV><DIV class="weekheader"> Week 2 </DIV><DIV class="dateheader"> 6.01.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> To submit minipreps for sequencing, will get results in two days and compare our constructions.
				<UL><LI>Did minipreps on samples where the devices were incorporated in (J promoters &amp; I3504)</LI><LI>Sent minipreps off for sequencing.
					<DIV class="indentlist"><UL>5 µL water, 5 µL sample, 1 µL of each primer</UL></DIV></LI></UL></DIV><DIV class="dateheader"> 6.02.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Make competent MACH cells.
				 <UL><LI>Made competent MACH cells (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>)</LI><LI>measured fluorescence of GFP for 6 samples using a TECAN SaphireII Microplate Reader
				 		<DIV class="indentlist"><UL>108A, 108C, 109A, 109C, 111A, 111B</UL></DIV><TABLE><TBODY><TR><TD><B><FONT color="orange">Parameter</FONT></B></TD><TD><B><FONT color="orange">Description</FONT></B></TD></TR><TR><TD>Microplate Reader Model</TD><TD>TECAN Sapphire II</TD></TR><TR><TD>Software</TD><TD>XFlour4</TD></TR><TR><TD>Measurement Mode</TD><TD>Flourescence Bottom</TD></TR><TR><TD>Excitation Wavelength</TD><TD>500 nm</TD></TR><TR><TD>Emission Wavelength</TD><TD>515 nm</TD></TR><TR><TD>Excitation Bandwidth</TD><TD>5 nm</TD></TR><TR><TD>Emission Bandwidth</TD><TD>5 nm</TD></TR><TR><TD>Gain (Manual)</TD><TD>150</TD></TR><TR><TD>Number of Reads</TD><TD>100</TD></TR><TR><TD>Flashmode</TD><TD>High Sensisitivity</TD></TR><TR><TD>Integration Time</TD><TD>40 µs</TD></TR><TR><TD> Temperature </TD><TD> 37°C </TD></TR></TBODY></TABLE><P><TABLE><TBODY><TR><TD><B><FONT color="orange">Sample</FONT></B></TD><TD>108A</TD><TD>108C</TD><TD>109A</TD><TD>109C</TD><TD>111A</TD><TD>111B</TD></TR><TR><TD><B><FONT color="orange">Reading</FONT></B></TD><TD>5932</TD><TD>6139</TD><TD>6050</TD><TD>5789</TD><TD>5589</TD><TD>6109</TD></TR></TBODY></TABLE></P></LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> Since we did it without MACH cells, will redo tomorrow with MACH cells and biosensor to compare results. 
			</DIV><DIV class="dateheader"> 6.03.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Observe Sequence Data and compare to our own plasmids. Redo TECAN results with blank and biosensor to see differences in fluorescent data. 
			<UL><LI>Compared sequence data to registry/our own plasmids. Figured out that no promoters got inserted in. Decided to send in Minipreps of the parts of the original devices for sequencing to see if the given parts were what we expected. </LI><LI>Redid fluorescence reading
					<DIV class="indentlist"><UL>Very evident difference, shows the presence of GFP</UL><UL>May redo in the future in order to do the trianalysis.</UL><UL>Used same TECAN parameters as 6.02.15</UL></DIV><TABLE><TABLE><TBODY><TR><TD><B><FONT color="orange">Sample</FONT></B></TD><TD>Control</TD><TD>108A</TD><TD>108C</TD><TD>109A</TD><TD>109C</TD><TD>111A</TD><TD>111B</TD><TD>Last Year</TD></TR><TR><TD><B><FONT color="orange">Reading</FONT></B></TD><TD>3379</TD><TD>6914</TD><TD>6097</TD><TD>6381</TD><TD>7696</TD><TD>5949</TD><TD>6814</TD><TD>OVER</TD></TR></TBODY></TABLE></TABLE></LI></UL></DIV><DIV class="dateheader"> 6.04.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Streaked out plates, continued competent cell protocol.
			<UL><LI>Streaked out MACH cells, Top10, and Biosensor Estrogen - on to LB plates. The biosensor estrogen negative plates had kanamyacin antibiotic on it (10 ul + 50 ul water).</LI><LI>Began TOP10 Competent cell protocol (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>). 
					<DIV class="indentlist"><UL>Diluted TOP10 overnight culture 1:100 for a 2ml volume in LB. Grew it from 10a-5p. Then added 300 ul to each 500 ml of LB in sterile 2L flasks. Placed in 18C shaker overnight after removing the magnetic stir rod.</UL></DIV></LI></UL></DIV><DIV class="dateheader"> 6.05.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Get back plasmid sequences to determine what went wrong with what we did. Restreaked the plates and competent cells growth is slow therefore will continue procedure Monday 6/8/15.
			<UL><LI>Placed growing TOP10 cells in 2 L flasks in cold room because it was taking too long to grow in 18C shaker. Will place back in shaker Sunday night so that it can be ready on Monday to use (need OD600 of 0.4-0.6.)</LI><LI>Further sequencing results on original devices sent out on 6/3/15 were obtained on 6/5/15. It showed that the promoters have 2 additional Spe1 cut sites which meant that they were cut out, which is why none of the transformants had the promoter in them.
					<DIV class="indentlist"><UL>We notified iGEM, and they will thank us on their website for finding their mistake.</UL></DIV></LI><LI>Started Transformations (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>) for New Set Today.</LI></UL></DIV><DIV class="weekheader"> Week 3 </DIV><DIV class="dateheader"> 6.07.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Set up culture preps for minipreps
			<UL><LI>Set up culture preps for minipreps (5ml) with 5ul of chlorophenical.
					<DIV class="indentlist"><UL>Samples: 106A, 101A, 101B, 101C, 101D, 117A, 117B, 117C, 117D </UL><UL>Left overnight in 37C shaker. </UL></DIV></LI></UL></DIV><DIV class="dateheader"> 6.08.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Redo the interlab study with the new devices given.
			<UL><LI>Forgot to do ligation and transformation, therefore we will redo this tomorrow.</LI></UL></DIV><DIV class="dateheader"> 6.09.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Redo digestion of the new transformations and determine if they fluoresce.
			<UL><LI>Redid the Digestion because we forgot to do transformations (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>).</LI><LI>Did 3504 digestion with 101B, 106A, 117C
					<DIV class="indentlist"><UL>For promoter: 10 µl of MiniPrep, 4 µl of H20, 2 µl of green buffer, 2 µl of EACH enzyme (PST and SPE1)</UL></DIV><UL>For insert: 12 µl of MiniPrep, 2 µl of H20, 2 µl of green buffer, 2 µl of EACH enzyme (PST and Xba1)</UL></LI><LI>Put digests into a gel &amp; washed gel with 1:1 buffer and put it into 42C heat shock for ~30 minutes</LI><LI>Did PCR purification</LI><LI>Ligation done using 4 µl of promoter, 4 µl of insert, 1 µl of buffer (t4 ligase buffer) LIG, 1 µl of T4 ligase (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>).</LI><LI>Did transformation for lacYFP, FFluc, PRE, POST (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>).
					<DIV class="indentlist"><UL>For lacYFP (can plate): 35 µl of cells, 1 µl YFP</UL><UL>For FFLuc, PRE, POST (5mL LB AMP cultures): 35 µl of competent cells, 1 µl of such samples </UL><UL>For our samples: 50 µl of competent cells, 10 µl of our mix (3504-101B, 106A, 117C)</UL></DIV></LI><LI>Also redid competent cells (autoclaved the 500mL LB, store TOP10 into 37C put in around 1microliter ← slow growth so put in more in order to grow faster) </LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> See if our devices are green fluorescing tomorrow and to fixate our cells to the W strain in the competent cells being made.
			</DIV><DIV class="dateheader"> 6.10.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Screen the transformants for ones that uptook the plasmid and to check if competent cell culture is ready.
			<UL><LI>Screened the plates with transformants using UV light.
					<DIV class="indentlist"><UL>Screened the plates with transformants using UV light.</UL></DIV></LI><LI>Set up 5ml culture using one colony from each (106A, 101B, 117C) for miniprep. Grew overnight in 37C shaker. Picked colony based on fluorescence using the UV light. </LI><LI>Competent cells still did not reach the appropriate OD (currently 0.1 at 9:30a). Allowing it to shake overnight again.</LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> Purify and isolate the transformant DNA using miniprep on the cultures that were set up today.
			</DIV><DIV class="dateheader"> 6.11.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Purify and isolate the transformant DNA using miniprep on the cultures and then send it out for sequencing. Begin the competent cell protocol.
			<UL><LI>Did miniprep (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>) on the 5ml cultures containing colonies of 101B (J23101 + I31504), 106A (J23106 + I31504), and 117C (J23117 + I31504)</LI><LI>Sent out the DNA for sequencing.</LI><LI>Competent cell culture was still did not have an OD of 0.4-0.6. Will check again tomorrow morning. </LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> Begin PCR amplification on Gaussia Luciferase using Phusion high fidelity DNA polymerase. 
			</DIV><DIV class="dateheader"> 6.12.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Finish competent cell protocol and start the Gaussia Luciferase experimentation.
			<UL><LI>Competent cell culture was within the OD. One of the competent cell cultures were within the appropriate range (at OD of 0.48). The other one wasn’t (at OD of ~0.2). We just started the competent cell protocol on both cultures but kept them separate so that we could compare their efficiency. The “Making competent MACH T1 cells” protocol was used for making competent TOP10 cells (protocol in the <A target="_blank" href="https://2015.igem.org/Team:Carnegie_Mellon/Protocols">Protocol Page </A>).</LI></UL></DIV><DIV class="weekheader"> Week 4 </DIV><DIV class="dateheader"> 6.15.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Verifying the sequences obtained to the constructs that were made. Check the competence of our competent cells. 
			<UL><LI>Verified by comparing the sequence of our constructs to the sequence got from the samples we sent out.
					<DIV class="indentlist"><UL>They seemed to be accurate with a few changes.</UL></DIV></LI><LI>Checked competence of competent cells using the iGEM competent transformation efficiency kit. 
					<DIV class="indentlist"><UL>Tubes labeled 1 w/ X conc are from the culture with OD within the range of 0.4-0.6. Tubes labeled 2 w/ conc are from culture that did not have an OD within the range of 0.4-0.6.</UL><UL>Tube 1 with concentration 0.5, 5, 10, 20, and 50 picogram (5 samples in total)</UL><UL>Tube 2 with concentrations 0.5, 5, 10, 20, and 50 picogram (5 samples in total)</UL><UL>Tube with MACH cells. </UL></DIV></LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> Will count the number of colonies and determine the transformation efficiency of the cells.  
			</DIV><DIV class="dateheader"> 6.16.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Verifying the sequences obtained to the constructs that were made. Check the competence of our competent cells. 
			<UL><LI>Found nothing in both our competent cell plates and MACH plates from yesterday, using the DNA iGEM transformation efficiency kit. </LI><LI>Redid Competent cells and MACH cells with plasmid DNA</LI><LI>Plated both 400 µl and 40 µl for each competent cell transformation (total: 6 plates) &amp; put in 37 °C incubator.</LI></UL><FONT color="orange"><B><U>For next time:</U></B></FONT> Check if our competent cells worked.  
			</DIV><DIV class="dateheader"> 6.17.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Verifying if our competent cells worked since it didn’t work using the DNA from iGEM.
			<UL><LI>Checked the plates and found cells in both MACH and competent cells.</LI></UL></DIV><DIV class="dateheader"> 6.18.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Transform colonies into TOP10 Competent Cells.
			<UL><LI>Transformed colonies (101B, 106A, 117A) into TOP10 Competent Cells and then plated them.
					<DIV class="indentlist"><UL>1 µL of DNA, 50 µL of cells, 5 min ice, 2 min heat, 5 min ice, 500 µL LB, 1 hr at 37 °C, plate</UL></DIV></LI></UL></DIV><DIV class="weekheader"> Week 5 </DIV><DIV class="dateheader"> 6.23.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Make cultures and plate cells.
			<UL><LI>Made 2 mL overnight cultures of cells that will later be used in the microplate reader to measure fluorescence.
					<DIV class="indentlist"><UL>MACH empty, Top 10 empty, 3 devices + top 10, 3 devices + mach </UL></DIV></LI><LI>Plated both the mach and competent cells overnight on LB plates to see if they would grow to use for the biological and technical replicate part of the interlab. 
					<DIV class="indentlist"><UL>Technical is more how many times we repeated something.</UL><UL>Biological is doing new sets of colonies.</UL></DIV></LI></UL></DIV><DIV class="dateheader"> 6.24.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT> Measure fluorescence for MACH and competent cells.
			<UL><LI>Took out Mach and Top10 Competent Cells to do measurements of flouresence with on a TECAN microplate reader.</LI><LI>Used same TECAN parameters as 6.02.15 except used a Gain of 100 instead of 150.</LI></UL></DIV></DIV><DIV id="tab4" class="mtab_content"><P><B>Estrogen Sensor:</B></P><DIV class="dateheader"> 7.1.15 </DIV><UL><DIV class="notes"><LI>Tried newly transformed cells, all taken from the same plate</LI></DIV><DIV class="indentlist"><UL>Tried to get similar-sized colonies</UL></DIV><LI>Tested the following amounts of estrogen:</LI><DIV class="indentlist"><UL>100 uM, 20 uM, 10 uM, 1 uM, 100 nM, 10 nM, 1 nM, 0 nM </UL></DIV><LI>Got very varied and inconsistent results </LI><DIV class="indentlist"><UL>No pattern between amount of estrogen added &amp; reading </UL></DIV><DIV class="indentlist"><UL>High standard deviation </UL></DIV><LI>Think that there is a problem with the transformation </LI><LI>Will try another transformation </LI></UL><DIV class="dateheader"> 7.2.15 </DIV><UL><DIV class="notes"><LI>Tried the restreaked  cells with 1:100,000, 1:1,000,000, and 1:10,000,000 estrogen dilutions</LI><LI>High standard deviation amongst duplicates, no pattern between amount of estrogen added &amp; reading</LI><LI>Determined that restreaked cells do not work </LI></DIV></UL><DIV class="dateheader"> 7.7.15 </DIV><UL><DIV class="notes"><LI>Transformation 4 of Mach cells did not work</LI><LI>Gave unexpected results with high standard deviation </LI><LI>Did not seem to be a difference between the different amounts of estrogen </LI><LI>For tomorrow:</LI><DIV class="indentlist"><UL>Do another transformation -- possibly get Donna &amp; Michelle to do it </UL></DIV><DIV class="indentlist"><UL>Try again with the colonies we have now</UL></DIV></DIV></UL><DIV class="dateheader"> 7.29.15 </DIV><UL><DIV class="notes"><LI>Made dilutions (1:100) from the starter cultures for the estrogen sensors and controls.</LI></DIV><LI>Set up transformations for CD-cel Gaussia (on plates and in culture) for testing of luciferase activity in response to addition of coelenterazine.</LI><LI>The isolation of CD-cel domain and gaussia domain didn’t work. </LI></UL><DIV class="dateheader"> 8.03.15 </DIV><UL><DIV class="notes"><LI>Started overnight cultures of the Sensor 6 cells from the restreaked plate</LI><LI>Did not add estrogen</LI><LI>Each overnight tube from a different colony </LI><LI>For the next overnight: will add 100 µL of the overnights into 2mL of fresh media &amp; add different dilutions of estrogen then</LI><LI>Goal: want to see if different colonies gave consistent results </LI><DIV class="indentlist"><UL>Want to see if lower dilutions of estrogen give lower readings</UL></DIV><LI>Made another set of starter cultures for estrogen sensor (with and without estrogen) and controls by adding a colony of each sample into 2 ml of fresh LB media. Cultures shook overnight in 37C shaker.  </LI></DIV></UL><DIV class="dateheader"> 8.04.15 </DIV><UL><DIV class="notes"><LI>Added 20 ul of starter culture to 2 ml of fresh LB containing 2 ul of chlorophenical and 1.2 ul of kanamyacin. This was done three times for each sample. For the J23100 RFP and PelB Gaussia, however, no kanamyacin was added</LI></DIV><DIV id="tab5" class="mtab_content" style="display: block;"><DIV class="dateheader"> 7.29.15 </DIV><DIV class="notes"><UL><LI>2ml cultures consisting of 2 ul of chloramphenicol were made for J23100 RFP and J23115 RFP and left overnight in the 37C shaker.</LI><DIV class="dateheader"> 7.30.15 </DIV><DIV class="notes"><FONT color="orange"><B><U>Goal:</U></B></FONT>  To compare the fluorescence of J23100 RFP with J23115 RFP.			<UL><LI>J23100 RFP did not show consistent results from previous data, mostly likely due to reason mentioned above. New cultures of J23100 RFP and J23115 RFP from newly streaked plates will be used to set up new cultures for measuring. </LI><LI>Re-tested the fluorescence of the J23115 RFP and J23100 since the first set of data from the previous day did not consist of biological replicates. However, the J23100 culture did not seem to grow well since the culture was not red. One reason could be that the plates from which the colonies were taken from were from 6.29.15 and not from the newer plate. This would explain why the fluorescence for J23100 RFP is much lower than expected, when it should be much higher than that for J23115 RFP. </LI></UL></DIV><DIV class="dateheader"> 8.3.15 </DIV><DIV class="notes"><UL><LI>Started three 2 ml overnight cultures, each consisting of 2 ul of chlorophenical, for  J23100 RFP and J23115 RFP </LI><LI>One 2ml overnight culture without chlorophenical was also made for empty MACH cells as a control. </LI></UL></DIV><DIV class="dateheader"> 8.4.15 </DIV><DIV class="notes"><UL><LI>Measured fluorescence for each overnight culture that was set up the day before (J23100 RFP, J23115 RFP, and empty MACH cells). These values are considered biological replicates. </LI><LI>Measured each sample three times (technical replicates) </LI><FONT color="orange"><B><U>Conclusion:</U></B></FONT> The fluorescence of the J23100 RFP was 5 times larger than that of the J23115 RFP.  
			</UL></DIV><DIV id="tab6" class="mtab_content" style="display: block;"><DIV class="weekheader"> History about the Arduino </DIV><P> The Arduino is a piece of programmable hardware created by several graduate students in Italy in the early 2000’s. The name itself comes from a bar they used to hang out in, which in turn was named after an old Italian king, Arduin of Ivrea. It was created because the hardware they had previously been using was $100 and they knew some people couldn’t afford that, so they designed a cheaper alternative. This is the embodiment of the Maker’s Movement Era! </P><P>And with that, the Arduino was born. The model we use, the “Arduino Uno,” contains a programmable chip that we can code via USB, and it has fourteen digital pins along with eight analog pins that can be used to communicate with a circuit based on voltage levels. With this, the possibilities are endless! People have made remote control lawnmowers and quadcopters, they’ve built robots and video games, and they’ve even made sensors and alarms to be used for home security and ease of access! This is what the Arduino can do: bring easy circuit making to anyone’s door step to help improve quality of life.
 </P></DIV></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>