<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="row color-nav" style="margin-left: -15px; margin-right: -15px; margin-top: 10px;"><DIV id="cssmenu" class="col-md-12 col-xs-12 text-center" style="margin-left: 0px; margin-right: 0px; padding-left: 0px; madding-right: 0px"><UL class="list-unstyled list-inline" style="margin-left: 0px;"><LI><DIV><A class="nav-link-o" href="https://2015.igem.org/Team:Vilnius-Lithuania">Home</A></DIV></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Project
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Project">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Application">Application</A></LI></UL></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Parts
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team_Collection">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Part_Collection">Part Collection</A></LI></NAV></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Team
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Timeline">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Pictures">Pictures</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Publicity">Publicity</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Crowdfunding">Crowdfunding</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Notebook
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Labjournal">Labjournal</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Human Practice
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/HumanPractices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/HumanPractices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/HumanPractices/Events">Events</A></LI></DIV><LI><DIV><A class="nav-link-o" href="https://2015.igem.org/Team:Vilnius-Lithuania/Measurement">Interlab</A></DIV></LI><UL><DIV id="wikicontent-container"><DIV id="wikicontent"><SECTION class="container-fluid" style="padding: 0px;"><DIV class="row" style="margin-top: 20px;"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Results</H3><P class="text-justify">Here we present you all the results we got during the project. We differentiated them into a few main parts and documented all the information depending on the purpose of the lab goals.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Cloning biobrick parts</H3><P class="text-justify">As we are a new team on iGEM, we wanted to bring something new to the competition as well, so we constructed two novel CRISPR-Cas biobricks, belonging to the I-F type CRISPR-Cas system. We did not find any biobricks from the I-type CRISPR-Cas systems in the parts registry, so we are the first!</P><P class="text-justify">We acquired the cloned genes of the main proteins of the I-F type CRISPR-Cas system, which function in the interference stage (DNA destruction). They are the Cas3 and Cascade (CRISPR associated complex for antiviral defence) genes. These genes were cloned from a paradantosis causing bacterium <EM>Aggregatibacter actinomycetem­comitans</EM> (try to say it out loud) DS-7 strain.</P><P class="text-justify">In order to make them biobrick compatible, we had to do several quick change mutagenesis reactions, because the wild type gene consisted of several restriction sites used by Standart Assembly restriction enzymes: 3 for the Cas3 gene and 1 for Cascade complex gene.</P><P class="text-justify">The wild type plasmid of Cascade complex was in a pCDF-Duet vector and had one EcoRI restriction site inside the Cascade complex gene. After restriction analysis of mutated Cascade plasmids (Figure 1) we can see, that the wild type (WT) Cascade plasmid was linearized by EcoRI, but the six mutated plasmids have retained superspiralized form, and were not cleaved by the enzyme.</P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 1.</STRONG> Cascade restriction analysis. The wild type gene was in a pCDF vector and had one EcoRI restriction site in the gene of Cascade complex. After six mutant plasmid digestion with EcoRI, no DNA linearization was present, compared to the non-mutated Cascade control (K).</P><P class="text-justify">The Cas3 gene was in a pET-Duet vector and had EcoRI, XbaI and PstI restriction sites upstream the gene (in the vector multi-cloning site area) and in the gene itself. After restriction analysis of a non-mutated Cas3 gene, two bands are visible (Figure 2). After the mutagenesis 5 out of 6 mutants were non-digestable by the enzymes.</P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 2.</STRONG> Resriction analysis of Cas3 protein. The wild type (WT) Cas3 protein was in a pETDuet vector and had EcoRI (E), XbaI (X) and PstI (P) restriction sites upstream of the Cas3 gene, and in the middle of Cas3 gene, all in close proximity. All five mutant plasmids have mutated successfully (except mutant plasmid #1), because after restriction with each restriction enzyme there were no restriction fragments detected. Restriction was compared to the unmutated non-digested wild type Cas3 plasmid (K).</P><P class="text-justify">After a few tries we successfully made all mutagenesis reactions of both Cas3 and Cascade genes, confirming them with restriction analysis and sequencing. We also cloned them to the pSB1C3 vector and shipped them to the parts registry to provide future iGEM teams with these biobricks: 
<UL><LI>Cas3 (<A href="http://parts.igem.org/Part:BBa_K1773003" class="clink">BBa_K1773003</A>)</LI><LI>Cascade complex (<A href="http://parts.igem.org/Part:BBa_K1773005" class="clink">BBa_K1773005</A>)</LI></UL></P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Functional and regulatory unit cloning</H3><P class="text-justify">During July and August we successfully cloned the functional unit of our system, which consists of 7 biobrick parts! We wanted to count how many times the cell can divide, so we cloned our functional (killing) unit with three different expression strengths. The unit consists of a gene for the Cas3 protein, genes for the Cascade complex, and a homogenous crRNA region for the Cascade complex.</P><P class="text-justify">Using custom gene synthesis, we synthesised two homogenous CRISPR regions, which had 6 identical spacers surrounded by repeat sequences. Each CRISPR region encode crRNA, which, assembled into the Cascade complex, targets an essential gene of the E. coli cell. SP1 targets E.coli DNA polymerase III δ subunit (DEG10190064), and SP2 targets RNA polymerase α subunit (DEG10190205).</P><P class="text-justify">For tunable protein expression we constructed three biobricks for Cas3, with strong RBS site (BBa_K1773019), medium RBS site (BBa_K1773020) and weak RBS site (BBa_K1773021), and every construct had a pLux/cI right promoter for controlled expression. We did the same with Cascade genes (BBa_K1773022, BBa_K1773023, BBa_K1773024). All parts were sequenced and shipped to iGEM Headquarters for future uses!</P><P class="text-justify">In addition, we almost finished to fully assemble our regulatory unit, which would regulate and turn on the functional unit when the cells are released into the environment. We managed to make an intermediate part consisting of pLac inducible cI lambda repressor attached to a LuxI gene, under a pLux/cI right promoter (BBa_K1773026). This part was also sequenced and shipped to the parts registry.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Characterization</H3><P class="text-justify">For our project to be feasible and working, our new synthesised CRISPR-Cas biobrick parts had to be expressed in the cell. We chose to perform Western immuno-blotting with anti-His6 tag antibodies for characterization. However, our created biobrick parts does not have a His6 sequence, because we did not want to make alterations to the proteins themselves, because they might loose their function.</P><P class="text-justify">Instead, we co-expressed our biobrick parts with a plasmid, coding one of the proteins of the Cascade complex (Csy3) with a His6 sequence in E.coli BL21-DE3 strain. The proteins generated from these plasmids form the Cascade complex, which is then detected with Western blot.</P><P class="text-justify">Our experiment was a success, and we have obtained an expression of our Cascade constructs, compared to the negative controls, wich had no plasmids (Figure 3). Cell cultures were grown until OD<SUB>500</SUB>~0,6, and then incubated at two different conditions. Expression of incubated cells at 16°C for 16 hours resulted in a bit stronger protein expression than those, which were incubated at 37°C for 3 hours. We wanted to see a different protein expression between the biobrick, which had a strong RBS (S), medium RBS (M) or weak RBS (W), however, the western blot did not show any differences. This may be due to the fact, that expression was too big to see subtile differences. </P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 3.</STRONG> Western blot of the Cascade biobricks. Expression of different Cascade biobricks was analysed in two incubation conditions after reaching a cell OD<SUB>500</SUB> ~0.6. Biobricks incubated at 16 °C for 16 hours showed a bit stronger expression strength rather than incubated at 37°C for 3 hours. M – molecular mass marker (Spectra broad range protein protein ladder); K – possitive control of Csy3 protein; Cascade biobricks were expressed within strong (S), medium (M) or weak (W) RBS sites. BL – BL21-DE3 strain with no transformed plasmids.</P><P class="text-justify">Another interesting result is that Cascade with strong RBS was expressed weaker than with a medium or weak RBS site. This may have to do with experimental error or in this exact construct, the strong RBS is expressed weaker than the other two.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading"><EM>In vivo</EM> cell experiments</H3><P class="text-justify">We planned an experiment, during which we can see our system's killing profficiency. We prepared BL21-DE3 strain <EM>E. coli</EM> cells with our Cascade expression construct, along with a complement plasmid, carrying the Cas3 gene in a pCola-Duet vector.</P><P class="text-justify">The whole experiment performed in order to transform the third plasmid, harbouring the crRNA region, into these bacteria. We transformed two different homogenous crRNA regions, which targeted an essential gene of <EM>E. coli</EM> genome: DNA pol. III δ subunit (SP1) and RNA polymerase α subunit (SP2). With these three components present in a bacteria and when IPTG is present in the medium (pColaDuet vector harbouring Cas3 needs IPTG for expression), the killing mechanism turns on. The expressed Cascade proteins assemble into a complex with the synthesized crRNA molecules, and binds to either the DNA pol. (SP1) or RNA pol. (SP2) genes. Cas3 is then recruited to hydrolyse these genes and bacteria die. </P><P class="text-justify">After counting bacterial CFU’s (colony forming units) (Figure 3), we estimated, that under conditions with no IPTG present all cells showed similar bacterial count with our control (K), which had no crRNA region. This result was expected, considering that IPTG is necessary for Cas3 expression. </P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 4.</STRONG> Cell killing proficiency test. CFUs were counted of bacteria transformed with genome targeting crRNAs (SP1 or SP2), compared to no crRNA harbouring control (K). Experiment was conducted in two conditions : without IPTG and with 2.5 mM IPTG present.</P><P class="text-justify">However, when IPTG is present, we can see a dramatic cell count drop of almost 100x in bacteria, harbouring our crRNA's (SP1 and SP2), compared to the control (K). According to this data, we can say, that our Cascade complex biobrick (BBa_K1773022) is expressed and actively functions in degradation of cell DNA.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Promoter characterization</H3><P class="text-justify">We wanted to make a contribution to the ever growing iGEM parts registry by characterizing one of more frequently used promoters, that is called the standard (lambda cI regulated) promoter, biobrick code – BBa_R1051.</P><P class="text-justify">To do this we employed a complex system that relies on GFP expression as a measurable identity that lets us to quantify levels of transcription from the cI regulated promoter. In order to achieve this, we cloned the cI regulated promoter in front of the GFP gene (BBa_I13504, the biobrick already contained appropriate RBS and terminator sequences.</P><P class="text-justify">Though we saw that GFP expression from the cI regulated promoter is strong. We also wanted to characterize the ability of the promoter to be repressed by producing the cI protein in vivo. So we emplyed a regulatory unit - BBa_K077039 which is composed of pLac promoter sequence (BBa_R0010), which allows the induction of transcription by addition of IPTG, followed by cI coding gene (BBa_P0151, the biobrick also included the necessary RBS and terminator sequences).</P><P class="text-justify">For the fluorescence measurement experiments we took advantage of protocol that was provided by the iGEM headquarters for the InterLab Measurement experiment. We transformed JM109 bacteria with our new constructs that were cloned into two separate compatible pasmid vectors. In theory, our regulatory unit upon induction with IPTG should produce cI repressor, which, in turn, should downregulate the expression of the GFP gene (under the control of cI regulated promoter). We seeked to quantify this by growing cell cultures overnight with different IPTG concentrations. We also used a negative and a positive control. </P><P class="text-justify">Here are the results:</P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 1.</STRONG> Relative fluorescence data normalized to sample with highest fluorescence (0.5 mM IPTG). (-) – Negative control (only BBa_K077039); 0.1 IPTG – 10 mM IPTG – samples with different IPTG concentrations. Error bars represent standard deviations of two biological replicates for each type of IPTG concentration.</P><H3 style="border-left: 5px solid rgb(236,151,31); padding-left: 5px; border-bottom: none">Discussion</H3><P class="text-justify">We determined that this device is sensitive to cI. It can be seen that upon induction of cI expression there is a significant drop in relative fluorescence, the general tendency is that d fluorescence signal depletes with increasing concentrations of IPTG, however we see great variation in our data. Similar data was obtained during replications of this experiment. We suggest that BBa_K195613 is indeed sensitive to cI and the effects of cI on transcription levels from this promoter are somewhat titratable, however we claim that this system (BBa_K077039 + BBa_K195613) is highly unpredictable. We suggest that this unpredictability might be due to the low translation levels of cI protein (weak RBS) combined with its instability (LVA tail) and its nature of a dominant repressor. These effects combined might lead to an effect in which changes in a small population of repressor might have great effects on the whole system.</P></DIV></DIV></DIV></DIV></SECTION></DIV></DIV><FOOTER class="container-fluid"><DIV class="row" style="background-color: rgb(62,62,62);"><DIV class="col-md-12 text-center" style=""><UL class="list-inline"><LI><A href="https://2015.igem.org/Special:RecentChanges">Recent changed</A></LI><LI><A href="https://2015.igem.org/Special:WhatLinksHere/Team:Vilnius-Lithuania">What links here</A></LI><LI><A href="https://2015.igem.org/Special:SpecialPages">Special pages</A></LI></UL></DIV></DIV><DIV class="row" style="background-color: rgb(62,62,62);"><DIV class="col-md-12 text-center" style="padding-bottom: 10px;"><UL class="list-inline"><LI><A href="https://2015.igem.org/Special:Preferences">My preferences</A></LI><LI><A href="https://2015.igem.org/2015.igem.org:Privacy_policy">Privacy policy</A></LI><LI><A href="https://igem.org/General_disclaimer">Disclaimer</A></LI></UL></DIV></DIV></FOOTER></UL></BODY></HTML>