<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><NAV class="row color-nav" style="margin-left: -15px; margin-right: -15px; margin-top: 10px;"><DIV id="cssmenu" class="col-md-12 col-xs-12 text-center" style="margin-left: 0px; margin-right: 0px; padding-left: 0px; madding-right: 0px"><UL class="list-unstyled list-inline" style="margin-left: 0px;"><LI><DIV><A class="nav-link-o" href="https://2015.igem.org/Team:Vilnius-Lithuania">Home</A></DIV></LI><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Project
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Project">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Application">Application</A></LI></UL></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Parts
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team_Collection">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Part_Collection">Part Collection</A></LI></NAV></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Team
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Timeline">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Pictures">Pictures</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Team/Publicity">Publicity</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Crowdfunding">Crowdfunding</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Notebook
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Labjournal">Labjournal</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Safety">Safety</A></LI></DIV><LI><DIV class="dropdown"><A href="#" class="dropdown-toggle nav-link-o" data-toggle="dropdown">Human Practice
  </A></DIV></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Practices">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/HumanPractices/Education">Education</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/HumanPractices/Ethics">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:Vilnius-Lithuania/HumanPractices/Events">Events</A></LI></DIV><LI><DIV><A class="nav-link-o" href="https://2015.igem.org/Team:Vilnius-Lithuania/Measurement">Interlab</A></DIV></LI><UL><DIV id="wikicontent-container"><DIV id="wikicontent"><SECTION class="container-fluid" style="padding: 0px;"><DIV class="row" style="margin-top: 20px;"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">At a glance</H3><P class="text-justify"><STRONG>What?</STRONG> Coliclock is a unique bacterial security system, which has never been done before.</P><P class="text-justify"><STRONG>Why?</STRONG> We built up our project with the goal to limit the prevalence of GMOs (genetically modified organisms) in nature. Our system works like a bacterial timer – that is why it is called Coliclock.</P><P class="text-justify"><STRONG>How?</STRONG> Throughout the project we used the knowledge of cloning and the function of genes from a number of different organisms. We performed cloning, mutagenesis and many other methods to create Coliclock.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Problem overview</H3><P class="text-justify">GMOs are novel and very efficient tool to produce desirable materials, such as required proteins, sugars or any fertilizers. These materials are usually produced by chemistry factories and are mostly used in a huge excess, to play their role effectively. High temperatures and pressures, as well as some toxic waste – this is what biotechnological procedures are lack of in comparison of industries of chemistry. </P><P class="text-justify">Even the biotechnological tools are much more advantageous – they are not allowed to be used outside the laboratories. Theoretically, any bacteria can not be taken out of the lab, even when they are so efficient. To deal with contaminations, to fertilize plants or help to deal with lack of nitrogen for the periods it is necessary for crops – this is what bacteria in a coat of GMOs could be used for. But there is a huge environmental problem that bacteria are almost immortal. Moreover – genetically modified bacteria are better and usually superior among naturally existing bacteria. Therefore there is a possibility to cause an environmental disaster or at least huge problems. </P><P class="text-justify'&gt;What is already done to reach this possibility to use GMOs outside the laboratory? One way is to use kill-switch, which turn on, when there is a lack/excess of some substances. But at both conditions there is a need for a physically interaction between some molecules and bacteria. Is it statistically reliable enough? Probably no, because it will be really irresponsible to use some techniques that need to rely on eventuality. Another way is to create a modified bacteria, which uses synthetic amino acids. As synthetic amino acids are not naturally found in environment, after several divisions, concentration of synthetic amino acid will be not big enough and these bacteria are probably going to die.  Here is again a word “probably” – proteins can mutate, and the difference in one amino acid is usually not worth trusting.&lt;/p&gt;
  
        &lt;p class=" text-justify="">Our team “turned their heads” (it is an expression in Lithuanian that has a meaning of hard thinking), how to make this process more safe and let to finally bring these GM bacteria out of the lab. Our Coliclock system is the key that will unlock these laboratory doors. This system not only ensures to stop the spread of GM bacteria to environment, but it also guarantees the suicide mechanism for a bacterium. It does not depend on the environment – it only depends on how you prepare your GMOs in laboratory before using them outside the lab. There is no place for coincidence and uncertainty. The Coliclock allows new applications of GMOs and expands the locations where GM bacteria could be used safely.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 class="text-heading">Project description</H3><P class="text-justify">The main celebrity of our project is CRISPR-Cas system. CRISPR-Cas system is quite common between bacteria and archaea, where it is widely used for protection from foreign nucleic acid (DNA or RNA). We used I-F type CRISPR-Cas system, which is naturally found in <I>Aggregatibacter actinomycetemcomitans</I> bacterial cultures. <I>A. actinomycetemcomitans</I> are known for causing gum and teeth infections. </P><P class="text-justify">There are three main components in CRISPR-Cas system: Cascade complex, cas3 protein and homogenic CRISPR region that codes crRNA molecules. To decribe it shortly, Cascade complex uses crRNA molecules, in order to find and recognize target genes, while cas3 helicase-nuclease protein is responsible for cleaving them.</P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 1.</STRONG> The CRISPR-Cas system. Cascade protein complex use crRNA to recognize target DNA sequence. Cas3 protein binds to the single stranded DNA and cleaves it.</P><P class="text-justify">We use this bacterial protection CRISPR-Cas system to create a method by which we could programme the life of genetically modified bacteria. One of the main goals of our project is regulating a specific number of life cycles that genetically altered bacteria could go through, when it is released into natural wildlife conditions. This is Coliclock – an automatic timer, which starts when you set the bacteria free to do its function.</P><P class="text-justify">What is more – you do not have to worry that this newly constructed bacteria will compete with regular wildtype bacteria and therefore create ecological and/or ethical problems. Coliclock bacteria, once its function is over, will die.</P><P class="text-justify">Our method can be divided into two main parts: functional unit and regulatory unit. Regulatory unit will suppress Coliclock system while you are still working with the bacteria in the laboratory and will make a start once the cells are set free. A functional unit, on the other hand, consists of all the three CRISPR-Cas system components. Once the functional unit is switched on, the cells will die.</P><P class="text-justify">Regulatory unit consists of a few genes, which regulates the start of Coliclock system. All of the regulatory genes have specific promoter sequences. Cells are cultured in the laboratory. A special inductor – lactose analog IPTG – is added to the medium. IPTG induces the expression of cI gene, which is under the pLac promoter. pLac promoter is located within regulatory unit genes of our system. This way we can create a strong expression of cl (which is λ repressor). cl repressor targets pLux/cl promoters in the cell – and we used this kind of promoter with all the other Coliclock system genes. As mentioned before, cl is a repressor of pLux/cI promoter, that is why all the genes, which have this specific promoter, are repressed from transcription. Theoretically, any other specific promoter can be used in this system, which, in turn, will be adjusted according to the promoter properties. We chose pLac, because it is widely used and we wanted to make sure that our system works.</P><P class="text-justify">When the cells are released into their natural habitat, there is no IPTG inductor, therefore, the synthesis of cl repressor stops. The concentration of the repressor is just enough for the cell to divide a few more times while all the promoters of functional subunit of the system are still repressed. After a specific times of divisions, cl repressor concentration reaches the point where it can not target all the pLux/cl promoters anymore. </P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 2.</STRONG> System function differences, when bacteria are in the lab (above) and when in the environment (below). In the lab repressor cI is synthesized, because of presence of IPTG, and pLux/cI promoter is repressed - Cd and Cas3 are not transcribed. In the environment IPTG does not occur, so repressor cI is not synthesized - Cd and Cas are transcribed.</P><P class="text-justify">This is the point at which our system switches from laboratory mode into wildlife mode. Once the concentration of cI repressor reaches a minor treshold, the transcription of LuxR and LuxI begins. At first it is not very indicative, but through the positive loop expression increases dramatically. </P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 3.</STRONG> Positive loop. Positive loop occurs, when LuxR and LuxI interacts with each other and increase transcription of itselves.</P><P class="text-justify">The product of LuxI sequence is HL (homoserine lactone, which, together with the LuxR protein, acts as an activation complex for pLux/cl promoters. This creates a positive loop of expression – the first transcribed proteins join into a complex, which activates the further transcription of the same proteins. The genes within these promoters are intensively transcribed. We call this whole construct the &quot;switch-on&quot; of the functional Coliclock system subunit.</P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 4.</STRONG> Regulatory unit. This unit consists RBS, pLux/cI and pLac promoter, LuxR, LuxI and cI. This unit is essential for system regulation.</P><P class="text-justify">Once the functional subunit is switched on and the transcription of functional subunit genes is present, Cascade, together with the crRNA molecules, forms a big ribonucleoprotein complex, which targets DNA polymerase III and RNA polymerase genes. They are essential for the survival of bacteria. Those genes are later cleaved by the cas3 nuclease. </P><P style="border-left: 5px solid rgb(236,151,31); padding-left: 3px; border-bottom: none"><STRONG>Figure 5.</STRONG> Functional unit. Three types of functional unit are different from each other, because of RBS (weak, medium, strong). Also Cascade (Cd), Cas3 and crRNA are in this regulatory unit. </P><P class="text-justify">In our project, we aimed to regulate the amount of cell divisions that a cell can make, in other words, we wanted to control the start of a functional system unit. To achieve this, we cloned different strength RBS (ribosome binding site) sequences in front of each CRISPR-Cas gene. According to these RBS sequences, the expression of CRISPR-Cas genes either is stronger or weaker, which reflects on bacterial lifetime: the stronger RBS sequence, the bigger CRISPR gene expression, the shorter bacterial lifetime. Regulatory and functional units were cloned in parallel. Regulatory unit was cloned from Standart Assembly Biobricks, sent from iGEM Headquarters, while functional units were cloned from <I>A. actinomycetemcomitans</I> by amplifying the desired genes by using PCR.</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12"><DIV class="thumbnail" style="background-color: rgb(249,249,249); border: none;"><DIV class="caption" style="color: rgb(00,00,00); background-color: rgb(249,249,249);"><H3 style="border-left: 5px solid rgb(236,151,31); padding-left: 5px; border-bottom: none">References</H3><UL><LI>Gasiunas G, Sinkunas T, Siksnys V (2014) Molecular mechanisms of CRISPR-mediated microbial immunity. Cellular and molecular life sciences: CMLS 71: 449-465.</LI><LI>Sorek R, Lawrence CM, Wiedenheft B (2013) CRISPR-mediated adaptive immune systems in bacteria and archaea. Annual review of biochemistry 82: 237-266.</LI><LI>Sinkunas T, Gasiunas G, Fremaux C, Barrangou R, Horvath P, Siksnys V (2011). Cas3 is a single-stranded DNA nuclease and ATP-dependent helicase in the CRISPR/Cas immune system. The EMBO journal 30: 1335-42.</LI><LI>Brian J. Caliando, Christopher A. Voigt. Targeted DNA degradation using a CRISPR device stably carried in the host genome. Nature Communications 6, Article number: 6989</LI></UL></DIV></DIV></DIV></DIV></SECTION></DIV></DIV><FOOTER class="container-fluid"><DIV class="row" style="background-color: rgb(62,62,62);"><DIV class="col-md-12 text-center" style=""><UL class="list-inline"><LI><A href="https://2015.igem.org/Special:RecentChanges">Recent changed</A></LI><LI><A href="https://2015.igem.org/Special:WhatLinksHere/Team:Vilnius-Lithuania">What links here</A></LI><LI><A href="https://2015.igem.org/Special:SpecialPages">Special pages</A></LI></UL></DIV></DIV><DIV class="row" style="background-color: rgb(62,62,62);"><DIV class="col-md-12 text-center" style="padding-bottom: 10px;"><UL class="list-inline"><LI><A href="https://2015.igem.org/Special:Preferences">My preferences</A></LI><LI><A href="https://2015.igem.org/2015.igem.org:Privacy_policy">Privacy policy</A></LI><LI><A href="https://igem.org/General_disclaimer">Disclaimer</A></LI></UL></DIV></DIV></FOOTER></UL></BODY></HTML>