<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Project_BioBrickBox"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Project/BioBrickBox">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Project/BioBrickBox&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/BioBrickBox&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/BioBrickBox&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Project/BioBrickBox" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Project/BioBrickBox</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="mobile-navigation"><UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Financing">Financing</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem">Problem</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus">BaKillus</A></LI><LI id="inpage-nav-sensing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#sensing">sensing</A></LI><LI id="inpage-nav-adhesion"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#adhesion">adhesion</A></LI><LI id="inpage-nav-killing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#killing">killing</A></LI><LI id="inpage-nav-suicide"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide">suicide</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox"><SPAN class="bacillus">Bacillus</SPAN> BioBrickBox 2.0</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/B_subtilis" class="bacillus">B. subtilis</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry">Registry</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application">Application</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Diagnosis_Treatment">Diagnosis &amp; Treatment</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Pretest">Pretest</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Legal_Issues">Legal Issues</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Financing_Intellectual_Property">Financing &amp; Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Expertise">Expertise</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Meetups">Meet ups</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/iGEM_School">iGEM@school</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/SynBio">SynBio Day</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Panel_Discussion">Panel Discussion</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Rathenau">Rathenau Instituut</A></LI></DIV><DIV id="tour-toggle-m" class="hidden-md hidden-lg">TOUR</DIV><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><DIV class="nav-item" id="tour-toggle">Tour</DIV></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project" class="project nav-item active" id="project">Project</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results" class="results nav-item" id="results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="labbook nav-item" id="labbook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application" class="safety nav-item" id="application">Application</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice" class="outreach nav-item" id="human-practice">Human Practice</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="team-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team" class="sub-nav-item">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Financing" class="sub-nav-item">Financing</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="project-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem" class="sub-nav-item">Problem</A></LI><LI id="inpage-nav-bakillus"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus" class="sub-nav-item">BaKillus</A></LI><LI id="inpage-nav-sensing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#sensing" class="sub-nav-item">sensing</A></LI><LI id="inpage-nav-adhesion"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#adhesion" class="sub-nav-item">adhesion</A></LI><LI id="inpage-nav-killing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#killing" class="sub-nav-item">killing</A></LI><LI id="inpage-nav-suicide"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide" class="sub-nav-item">suicide</A></LI></UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox" class="sub-nav-item"><SPAN class="bacillus">Bacillus</SPAN> BioBrickBox 2.0</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling" class="sub-nav-item">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/B_subtilis" class="sub-nav-item bacillus">B. subtilis</A></LI><UL id="results-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results" class="sub-nav-item">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry" class="sub-nav-item">Registry</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Judging" class="sub-nav-item">Judging</A></LI></UL><UL id="application-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Diagnosis_Treatment" class="sub-nav-item">Diagnosis &amp; Treatment</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Pretest" class="sub-nav-item">Pretest</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Legal_Issues" class="sub-nav-item">Legal Issues</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Financing_Intellectual_Property" class="sub-nav-item">Financing &amp; Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Expertise" class="sub-nav-item">Expertise</A></LI></UL><UL id="human-practice-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Meetups" class="sub-nav-item">Meet ups</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/iGEM_School" class="sub-nav-item">iGEM@school</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/SynBio" class="sub-nav-item">SynBio Day</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Panel_Discussion" class="sub-nav-item">Panel Discussion</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Rathenau" class="sub-nav-item">Rathenau Instituut</A></LI></UL><UL id="labbook-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="sub-nav-item">Protocols</A></LI></UL></DIV></DIV></DIV><DIV id="lmu-content"><DIV id="project-biobrickbox-teaser" class="full-width teaser-img"> </DIV><SECTION><SECTION><DIV><H1><SPAN class="mw-headline" id="Bacillus_BioBrickBox"><I>Bacillus</I> BioBrickBox </SPAN></H1><P>In 2012, the iGEM-team LMU-Munich began the task to develop essential BioBricks, like vectors, promoters, reporters and affinity tags, especially suited for the use in<I>Bacillus subtilis</I> in order to establish a new chassis in the <I>Escherichia coli</I>-dominated world of iGEM. Even after the finals of the iGEM competition in 2012, the project was pursued further since its importance for synthetic biology and resulted in a publication in the Journal of Biological Engineering about this so called <A href="http://www.jbioleng.org/content/7/1/29" class="external text" rel="nofollow"><I>Bacillus</I> BioBrick Box</A>.
</P><P>This year, our team aims to enhance the <I>Bacillus</I> BioBrick Box  by adding new parts, like different colored fluorescent proteins, a RBS-collection, evaluating protein linkers for <I>B. subtilis</I> and creating a redesigned backbone for resistance free strain generation.
</P></DIV></SECTION><SECTION class="bg-color-2"><DIV><H2><SPAN class="mw-headline" id="Resistance_free_strain_generation"> Resistance free strain generation </SPAN></H2><P>With BaKillus as an medical application it is most likely that it will be released - though in small amounts - into the environment.
Therefore we sought after a solution in order to keep the final BaKillus free of unnecessary antibiotic resistances. In order to do so we developed a resistance free gene insertion strategy for integrative <I>Bacillus subtilis</I> vectors. 
</P><DIV><LABEL for="rfc-bground">Background</LABEL><ARTICLE class="ac-small">
              
The <I>upp</I> gene from Bacillus subtilis W168 encodes for a Uracilphosphoribosyl transferase (UPRTase). 
Its key reaction in uracil salvage is the reaction of a uracil molecule with a 5'-phosphoribosyl-a-1- pyrophosphate (PRPP) molecule, resulting in the formation of UMP.
A second locus, the <I>pyrR</I> gene, encoding a second UPRTase has been identified. However it has been shown, that the UPRTase derived from <I>pyrR</I> locus has an influence on overall UPRTase activity of &lt; 1 %. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC176586/" class="external autonumber" rel="nofollow">[1]</A>
This makes the <I>upp</I>-derived UPRTase the only physiologically relevant catalyst for UPRTase activity.
Exposure to the pyrimidine analogue 5-Fluorouracil UPRTase results in production of 5-fluoro-dUMP, a very potent inhibitor of the thymidylate synthase (Neuhard, J. (1983) Utilization of preformed pyrimidine bases and nucleosides. In Metabolism of Nucleotides, Nucleosides and Nucleobases in Microorganisms. Munch- Petersen, A. (eds). New York: Academic Press, pp. 95– 148. ). As a result 5-FU is toxic to the <I>Bacillus subtilis</I> W168 strain.
<I>B. subtilis</I> 5FU-resistant (5FUR) mutants selected on low drug concentration (10 mM 5FU) are UPRTase-defective. <A href="http://www.nature.com/nature/journal/v390/n6657/abs/390249a0.html" class="external autonumber" rel="nofollow">[2]</A>.
This has made <I>upp</I> a go to choice for negative selection in combination with an <I>B. subtilis</I> W168 <I>Δupp</I> strain. So far it has been used to make clean in-frame deletions and point mutations. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12366828" class="external autonumber" rel="nofollow">[3]</A><P>However, to our knowledge, no application using <I>upp</I> for clean insertions has been established so far.
</P><P>The I-SceI Restricition endonuclease has a highly specific recognition sequence of 18 nucleotides. No such sequence is present in the <I>B.subtilis</I> W168 strain. It creates a double strand break at targeted location, which leads to an increased rate of repair at the specific site. By this, the rate of homologous recombination is increased by a factor of 100. <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0081370" class="external autonumber" rel="nofollow">[4]</A></P></ARTICLE></DIV><DIV><LABEL for="rfc-design">Design</LABEL><ARTICLE class="ac-small">

The Plan was to restructure the BioBrick compatible, integrative <I>Bacillus subtilis</I> vectors <A href="http://parts.igem.org/Part:BBa_J179000" class="external text" rel="nofollow">pSB1C</A>, <A href="http://parts.igem.org/Part:BBa_J179001" class="external text" rel="nofollow">pSB2E</A> and <A href="http://parts.igem.org/Part:BBa_J179002" class="external text" rel="nofollow">pSB4S</A>  in a fashion that leads to deletion of the antibiotic resistance after an insertion of a gene of interest has been generated.
Integration of those basic vectors is achieved via homologous recombination between the <I>amyE</I>/<I>lacA</I>/<I>thrC</I> locus, respectively, and the corresponding up and down fragments on the vector. This leads to an insertion of the gene of interest within the RCF10 compatible multiple cloning site and the resistance for positive selection (Cm<SUP>r</SUP>/MLS<SUP>r</SUP>/Spec<SUP>r</SUP>).
<P>We wanted to add the <I>upp</I>-cassette, containing an I-SceI site, as well as an additional up/down fragment to the vector. The desired vectors are presented in Fig. 1. The <I>upp</I>-cassette will allow negative selection in 5FU media. The contained I-SceI site will be cut by the I-SceI restriction endonuclease which is encoded on a helper plasmid pEBS-cop1. <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0081370" class="external autonumber" rel="nofollow">[5]</A> (Figure)
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pEBS-cop1. Plasmid containing the <I>i-sceI</I> gene</DIV></DIV></DIV></DIV><P>Cloning Strategy
</P></ARTICLE></DIV><DIV><LABEL for="rfc-results">Results</LABEL><ARTICLE class="ac-small">

Lab work on this project only started at end of July
So far the <I>upp</I> cassete has been modified to fullfill the Freiburg Standard (RCF25) criteria and was successfully sequencend and submitted to the registry as BioBrick part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351023" class="external text" rel="nofollow">BBa_K1351023</A>.
Creation of the complete and functional backbones has not been successfull. We assume that due to the duplication of homologous sequences for the integration into <I>Bacillus subtilis</I> genomic loci a deletion has occured during phusion PCR. This is supported by the results of multiple digests of the resulting plasmid by different restriction enzymes.
<P>We plan on continuing work on this project after the iGEM Competition

            </P></ARTICLE></DIV></DIV></SECTION></SECTION></DIV><SECTION class="bg-color-3"><DIV><H2><SPAN class="mw-headline" id="Fluorescent_Proteins"> Fluorescent Proteins </SPAN></H2><P>Fluorescent proteins (FPs) are important tools in research on a cellular or molecular basis and in the past years, many different FPs with very diverse qualities related to color, brightness, duration, bleaching, maturation time and other parameters were developed.
Unfortunately, most of these FPs are optimized for use in <I>E. coli</I>, yeast or mammalian cells, and we aim to evaluate the properties of different FPs for the use in <I>B. subtilis</I> to enhance our Bacillus BioBrick Box even further.

        </P><DIV><LABEL for="fps-bground">Background</LABEL><ARTICLE class="ac-small">
              
Fluorescent Proteins are often derivates of the first described FP, GFP, which was isolated from the jellyfish <I>Aequorea victoria</I>. This protein is of typical barrel-shape, built from 11 ß-sheets and a chromophore in the middle of the barrel. This barrel structure is typical for fluorescent proteins, whether they derive from GFP or not. Other colors than green were developed by changing single amino acids in the chromophore of GFP (e.g.: Yellow: YFP and Cyan: CFP) or by analyzing fluorescent proteins from other organisms, like <I>Discosoma striata</I>, a coral, from which some of our red FPs, like dsRed derive.
FPs are an important tool in today’s research, for example in order to study gene expression or the location of specific proteins in a cell or a whole organism. This is the reason, why we tried to establish some different colors of FPs for the usilization in <I>B. subtilis</I> by evaluating them for the <I>Bacillus</I> BioBrick Box.

            </ARTICLE></DIV><DIV><LABEL for="fps-design">Design</LABEL><ARTICLE class="ac-small">

Seven different FPs (Table 1) where chosen and either obtained from the registry or from the lab group Mascher. 
The BioBrick &lt;partinfo&gt;BBa_E1010&lt;/partinfo&gt;was mutated via site directed mutagenesis by overlap extension PCR in order to delete two AgeI-Restriction sites and make the BioBrick compatible for the Freiburg standard RFC25. This improved BioBrick is called <A href="http://parts.igem.org/Part:BBa_K1351021">BBa_K1351021</A>
The BioBricks and <A href="http://parts.igem.org/Part:BBa_K592100" class="external text" rel="nofollow">BBa_K592100</A> (mTagBFP) where both provided with the necessary overhangs for the Freiburg standard, the other FPs where already provided with the proper restriction overhangs.
These seven FPs where combined C- and N-terminally with a His-Tag (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823037" class="external text" rel="nofollow">BBa_K823037</A>) and cloned into the vector pBS0K-Pspac (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351040" class="external text" rel="nofollow">BBa_K1351040</A>) and transformed into <I>B. subtilis</I>.
<TABLE cellspacing="0" border="1"><CAPTION><B>Table 1</B>: Used fluorescent proteins
</CAPTION><TBODY><TR><TH>BioBrick number
</TH><TH>Name
</TH><TH>Color
</TH><TH>Excitation Peak
</TH><TH>Emission Peak
</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1351021" class="external text" rel="nofollow">BBa_K1351021</A></TD><TD> DsRed
</TD><TD>Red
</TD><TD align="right"> 584nm
</TD><TD align="right"> 607nm [1]
</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K592100" class="external text" rel="nofollow">BBa_K592100</A></TD><TD> mTagBFP
</TD><TD> Blue
</TD><TD align="right"> 399nm
</TD><TD align="right"> 456nm
</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1159302" class="external text" rel="nofollow">BBa_K1159302</A></TD><TD> eCFP
</TD><TD> Cyan
</TD><TD align="right"> 439nm
</TD><TD align="right"> 476nm
</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1159301" class="external text" rel="nofollow">BBa_K1159301</A></TD><TD> sYFP2
</TD><TD> Yellow
</TD><TD align="right"> 515nm
</TD><TD align="right"> 527nm
</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K823039" class="external text" rel="nofollow">BBa_K823039</A></TD><TD> gfpmut1
</TD><TD>Green
</TD><TD align="right"> 395nm
</TD><TD align="right"> 509nm
</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1351042" class="external text" rel="nofollow">BBa_K1351042</A></TD><TD> mcherry
</TD><TD> Red
</TD><TD align="right"> 587nm
</TD><TD align="right"> 610nm
</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K823029" class="external text" rel="nofollow">BBa_K823029</A></TD><TD> mKate 2
</TD><TD>Red
</TD><TD align="right"> 588nm
</TD><TD align="right"> 633nm
</TD></TR></TBODY></TABLE></ARTICLE></DIV><DIV><LABEL for="fps-results">Results</LABEL><ARTICLE class="ac-small"><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Fluorescence of GFP under expression of promoter Pspac</DIV></DIV></DIV><P>The mutagenesis of the Biobrick E1010 was successful and confirmed by sequencing, the thus created Biobrick is called <A href="http://parts.igem.org/Part:BBa_K1351021" class="external text" rel="nofollow">BBa_K1351021</A>. The fusion with the His tags were also successfully conducted and confirmed by sequencing.
The ligation into the vector pBS0K-Pspac was confirmed via Colony PCR and the transformation into<I> B. subtilis</I> was conducted successfully.
</P><P>The intensity and spectrum of the fluorescence was measured by a TECAN Plate Reader, kindly provided by the AG Leonhardt from the LMU.
</P><P>The fluorescence of all of the proteins, however, proved to be not very good and often at the same level as the auto fluorescence of the <I>B. subtilis</I> wild type. The examination of the cells under a microscope revealed, that the promoter is not suited for this experimental set up, since the gene expression is very heterogeneous and rather low.
</P><P>This problem will be solved by recloning the FPs into the vector pBS1C (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823023" class="external text" rel="nofollow">BBa_K823023</A>) together with the xylose-inducible promoter Pxyl(<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351039" class="external text" rel="nofollow">BBa_K1351039</A>).

            </P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION class="bg-color-2"><DIV><H2><SPAN class="mw-headline" id="Linkers"> Linkers </SPAN></H2><P>Linkers are short peptides used to fuse two protein domains (or even whole proteins). They are designed not to interact with the fused parts. A linker of the wrong length can lead to sterically inactivated (if the linker is too short) or instable (if it is too long) constructs. Last but not least, linkers need to exhibit a certain flexibility. We evaluated a set of linkers partially already found in the registry using FRET.

        </P><DIV><LABEL for="linker-bground">Background</LABEL><ARTICLE class="ac-small">
              
Förster Resonance Energy Transfer (FRET) is a phenomenon that occurs when two fluorophores with overlapping spectra are not more than 10 nanometers apart. When the donor fluorophore gets excited, and FRET happens, it does not emit all the energy as light, it transferres a part of the energy non-radiatively to the acceptor fluorophore, which starts fluorescing. The efficiency of this energy transfer decays with the sixth power of the distance between donor and acceptor. This makes FRET an ideal tool to measure or verify very short distances, like the ones between interacting proteins <I>in vivo</I>. A common application would be tagging two supposedely interacting proteins with the FRET couple and measuring acceptor fluorescence. Obviously this technique lends itself well to the evaluation of linkers.

            </ARTICLE></DIV><DIV><LABEL for="linker-design">Design</LABEL><ARTICLE class="ac-small"><P>mTurquoise (donor) and mNeonGreen (acceptor) were used as FRET-couple for this study. Translational fusion constructs were created according to a modified RFC25 standard. From the registry we took the linkers <A href="http://parts.igem.org/Part:BBa_K243004" class="external text" rel="nofollow">BBa_K243004</A>, <A href="http://parts.igem.org/Part:BBa_K243005" class="external text" rel="nofollow">BBa_K243005</A>, <A href="http://parts.igem.org/Part:BBa_K243006" class="external text" rel="nofollow">BBa_K243006</A>, <A href="http://parts.igem.org/Part:BBa_K157009" class="external text" rel="nofollow">BBa_K157009</A>, <A href="http://parts.igem.org/Part:BBa_K157013" class="external text" rel="nofollow">BBa_K157013</A>, <A href="http://parts.igem.org/Part:BBa_K243029" class="external text" rel="nofollow">BBa_K243029</A> and <A href="http://parts.igem.org/Part:BBa_K243030" class="external text" rel="nofollow">BBa_K243030</A>. We also used this method to characterize our own linkers <A href="http://parts.igem.org/Part:BBa_K1351009" class="external text" rel="nofollow">BBa_K1351009</A> and <A href="http://parts.igem.org/Part:BBa_K1351035" class="external text" rel="nofollow">BBa_K1351035</A>. For each linker a construct with mTurquise fused to it N-terminally and mNeonGreen C-terminally was created, as well as a construct where the FPs were oriented the other way around.

            </P></ARTICLE></DIV><DIV><LABEL for="linkers-results">Results</LABEL><ARTICLE class="ac-small">

The constructs have been created, and fluorescence has been observed. Due to time concerns, the FRET-assay has not yet been performed.

            </ARTICLE></DIV></DIV></SECTION></DIV><SECTION><DIV><H2><SPAN class="mw-headline" id="Bacillus_RBS_collection"><I>Bacillus</I> RBS collection</SPAN></H2><P>Ribosome Binding Sites (RBS) are essential for gene expression, since they are required for translation initiation. Much to our dismay, there used to be no <I>Bacillus</I> RBS in the registry. So we designed a broad-range RBS library for <I>B. subtilis</I>, aiming for maximum translation initiation rate space coverage. 
</P><DIV><LABEL for="rbs-bground">Background</LABEL><ARTICLE class="ac-small">
              
RBS are short nucleotide sequences that are complemetary to the 3' end of 16S rRNA, and thus are bound by the ribosome during translation initiation. The RBS is usually found ~8 nucleotides upstream of the start codon in procaryotes. In <I>B. subtilis</I> the perfect consensus RBS is AAGGAGGGATA (<A href="http://parts.igem.org/Part:BBa_K1351028" class="external text" rel="nofollow">Bba_K1351028</A>). 

            </ARTICLE></DIV><DIV><LABEL for="rbs-design">Design</LABEL><ARTICLE class="ac-small">

Using the Salis lab's <A href="https://salis.psu.edu/software/" class="external text" rel="nofollow">RBS calculator</A><ARTICLE class="ac-small"> we calculated our RBS library with the sequence  ARRRRRRGATA. The 7 RBS from this collection that were submitted as BioBricks (<A href="http://parts.igem.org/Part:BBa_K1351028" class="external text" rel="nofollow">BBa_K1351028</A>-<A href="http://parts.igem.org/Part:BBa_K1351034" class="external text" rel="nofollow">BBa_K1351034</A>) are the ones that offer the best coverage of translation initiation rate space, number 1 being the optimal <I>Bacillus</I> RBS and number 7 being the weakest. 
            </ARTICLE></ARTICLE></DIV><DIV><LABEL for="rbs-results">Results</LABEL><ARTICLE class="ac-small">

The constructs have been created, the evaluation however is not yet finished.

            </ARTICLE></DIV></DIV></SECTION></DIV><DIV id="footer"><DIV id="footer-content"><H2>Team LMU-Munich</H2><H3>igem@bio.lmu.de </H3>

AG Thorsten Mascher 
Großhaderner Str. 2 
82152 Planegg (Martinsried) </DIV></DIV><DIV id="tour-overlay"><DIV id="tour-box"><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Hi_there.21"> Hi there! </SPAN></H2><P>Welcome to our Wiki! I'm BaKillus, the pathogen-hunting microbe, and I'll guide you on this tour through our project. If you want to learn more about a specific step, you can simply close the tour and come back to it anytime you like. So let's start!

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="What.27s_the_problem.3F"> What's the problem? </SPAN></H2><P>First of all, what am I doing here? The problem is, pathogenic bacteria all around the world are becoming more and more  resistant against antimicrobial drugs. One major reason for the trend is the inappropriate use of drugs. With my BaKillus super powers, I want to reduce this misuse and thus do my part to save global health.

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Sensing_of_pathogens"> Sensing of pathogens </SPAN></H2><P>To combat the pathogenic bacteria, I simply eavesdrop on their communication. Bacteria talk with each other via quorum sensing systems, which I use to detect them and trigger my responses. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Adhesion"> Adhesion </SPAN></H2><P>The more specific and effective I can use my powers, the lower the danger is of provoking new resistance development. So I catch pathogens whenever I get hold of them and stick to them until my work is done.

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Killing"> Killing </SPAN></H2><P>Talking about my work - killing pathogens is finally what I am made for. In response to quorum sensing molecules of the pathogens, I export a range of antimicrobial substances leading to dissipation of biofilms and the killing of the targeted bacteria. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Suicide_switch"> Suicide switch </SPAN></H2><P>When the job is done and all the bad guys are finished, you don't need a super hero anymore. So after fulfilling my work I say goodbye to the world by activating my suicide switch. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Application"> Application </SPAN></H2><P>Of course I'm not only a fictional hero, but a very real one. In two different prototypes, I could be used for diagnosis or treatment of pathogen-caused diseases. However, there is still a whole lot of regulational and economical questions that have to be answered before. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="See_you.21"> See you! </SPAN></H2><P>So now you know my short story - and it is time for me to return to my fight for a safer world. Feel free to take a closer look on my <A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus" class="external text" rel="nofollow">super powers</A>, the <A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Timeline" class="external text" rel="nofollow">process of my development</A> or the plans for a <A href="https://2014.igem.org/Team:LMU-Munich/Application" class="external text" rel="nofollow">medical application</A>.

</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox">http://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Project/BioBrickBox" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Project/BioBrickBox" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/BioBrickBox&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/BioBrickBox&amp;oldid=403681" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>