<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Project_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Project/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Project/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Project/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Project/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="mobile-navigation"><UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Financing">Financing</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem">Problem</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus">BaKillus</A></LI><LI id="inpage-nav-sensing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#sensing">sensing</A></LI><LI id="inpage-nav-adhesion"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#adhesion">adhesion</A></LI><LI id="inpage-nav-killing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#killing">killing</A></LI><LI id="inpage-nav-suicide"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide">suicide</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox"><SPAN class="bacillus">Bacillus</SPAN> BioBrickBox 2.0</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/B_subtilis" class="bacillus">B. subtilis</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry">Registry</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application">Application</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Diagnosis_Treatment">Diagnosis &amp; Treatment</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Pretest">Pretest</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Legal_Issues">Legal Issues</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Financing_Intellectual_Property">Financing &amp; Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Expertise">Expertise</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Meetups">Meet ups</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/iGEM_School">iGEM@school</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/SynBio">SynBio Day</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Panel_Discussion">Panel Discussion</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Rathenau">Rathenau Instituut</A></LI></DIV><DIV id="tour-toggle-m" class="hidden-md hidden-lg">TOUR</DIV><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><DIV class="nav-item" id="tour-toggle">Tour</DIV></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project" class="project nav-item active" id="project">Project</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results" class="results nav-item" id="results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="labbook nav-item" id="labbook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application" class="safety nav-item" id="application">Application</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice" class="outreach nav-item" id="human-practice">Human Practice</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="team-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team" class="sub-nav-item">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Financing" class="sub-nav-item">Financing</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="project-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem" class="sub-nav-item">Problem</A></LI><LI id="inpage-nav-bakillus"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus" class="sub-nav-item">BaKillus</A></LI><LI id="inpage-nav-sensing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#sensing" class="sub-nav-item">sensing</A></LI><LI id="inpage-nav-adhesion"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#adhesion" class="sub-nav-item">adhesion</A></LI><LI id="inpage-nav-killing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#killing" class="sub-nav-item">killing</A></LI><LI id="inpage-nav-suicide"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide" class="sub-nav-item">suicide</A></LI></UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox" class="sub-nav-item"><SPAN class="bacillus">Bacillus</SPAN> BioBrickBox 2.0</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling" class="sub-nav-item">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/B_subtilis" class="sub-nav-item bacillus">B. subtilis</A></LI><UL id="results-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results" class="sub-nav-item">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry" class="sub-nav-item">Registry</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Judging" class="sub-nav-item">Judging</A></LI></UL><UL id="application-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Diagnosis_Treatment" class="sub-nav-item">Diagnosis &amp; Treatment</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Pretest" class="sub-nav-item">Pretest</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Legal_Issues" class="sub-nav-item">Legal Issues</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Financing_Intellectual_Property" class="sub-nav-item">Financing &amp; Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Expertise" class="sub-nav-item">Expertise</A></LI></UL><UL id="human-practice-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Meetups" class="sub-nav-item">Meet ups</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/iGEM_School" class="sub-nav-item">iGEM@school</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/SynBio" class="sub-nav-item">SynBio Day</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Panel_Discussion" class="sub-nav-item">Panel Discussion</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Rathenau" class="sub-nav-item">Rathenau Instituut</A></LI></UL><UL id="labbook-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="sub-nav-item">Protocols</A></LI></UL></DIV></DIV></DIV><DIV id="lmu-content"><DIV id="project-modeling-teaser" class="full-width teaser-img"> </DIV><SECTION style="padding-bottom:0;"><SECTION id="model-main"><DIV><H1><SPAN class="mw-headline" id="Modeling"> Modeling </SPAN></H1><H2><SPAN class="mw-headline" id="Chemotaxis"> Chemotaxis </SPAN></H2><H3><SPAN class="mw-headline" id="Motivation"> Motivation </SPAN></H3><P>To stay true to its title 'pathogen-hunting microbe', BaKillus should ideally be able to
swim towards its prey via chemotaxis. However, the ability to react to an AIP- or CSP-gradient requires
the introduction of a chimeric receptor into <I>Bacillus subtilis</I>. Chemotaxis receptors are quite complex and can,
for example, take different methylation states to allow for attenuation of the chemotactic response.
This complexity puts the construction of a custom receptor beyond the scope of an iGEM project.
</P><P>To evaluate how a hypothetical chemotactic BaKillus would act in an experimental setting, we developed a
model to simulate the movement of chemotactic hunters in response to an attractant gradient produced
by a growing colony of prey bacteria.
</P><H3><SPAN class="mw-headline" id="Overview"> Overview </SPAN></H3><P>Our multi-scale hybrid model essentially consists of 3 components: growth of prey bacteria (algebraic model), diffusion of attractant molecules (spatial partial differential equation) and chemotactic behavior of the hunter bacteria (stochastic spatial agent model coupled to differential equations for internal state with a stochastic driving force).
All these processes take place in a defined geometric space (representing, for example, a microfluidic device). The experimental scenario we wish to capture in
our model is the following: a colony of prey bacteria is spotted onto the arena, growing in place and producing attractant molecules. A fixed population of hunter
bacteria is placed into the arena as well. Their reaction (biased movement) to the building attractant gradient is what we seek to observe.
</P><H4><SPAN class="mw-headline" id="Growth_of_prey"> Growth of prey </SPAN></H4><P>The prey bacteria will grow on a population level, according to a logistic growth curve with explicit solutions [1]:
</P><P>
\[ y = ln \frac{N}{N_0} = \frac{A}{1 + e^{\frac{4\mu_m}{A} (\lambda - t) + 2}}\]

</P><P>Here, \(N_0\) is the population size at \(t = 0\), \(A = ln \frac{N_\infty}{N_0}\) is the asymptote, \(\mu_m\) is the growth rate and \(\lambda\) the duration of the lag-phase.
</P><H4><SPAN class="mw-headline" id="Diffusion_of_attractants"> Diffusion of attractants </SPAN></H4><P>The concentration of attractant molecules (\(c\)) in space changes according to a partial differential equation comprising diffusion, degradation and production by the prey bacteria.
</P><P>
\[ \frac{\partial c}{\partial t} = D\Delta c + p c_{prey} - a c\]

</P><P>
\(D\) is the diffusion coefficient, \(a\) is the degradation rate, \(p\) the rate of production and \(c_{prey}\) the concentration of prey/producer bacteria.
</P><H4><SPAN class="mw-headline" id="Chemotaxis_2"> Chemotaxis </SPAN></H4><P>The chemotaxis of the hunter bacteria (BaKillus) is represented by an agent-based model adapted from [2]. Each bacterium has its own methylation and phosphorylation
states of the chemotactic pathway which change according to a system of ordinary differential equations in response to attractant concentrations.
In a classical 2-state model of bacterial chemotaxis, each bacterium will be in either <I>running</I> or <I>tumbling</I> state at each time point.
In a running step, bacteria will swim forward in a straight line whereas in a tumbling step, they will change their direction by a random angle picked from a gamma distribution.
</P><P>It should be noted that the model used here actually describes chemotaxis of <I>E. coli</I>. While the wiring of the chemotaxis pathway differs slightly, the same behavior is expected for <I>B. subtilis</I>. Therefore, we opted for the use of the better studied <I>E. coli</I>-model.
</P><H3><SPAN class="mw-headline" id="Implementation"> Implementation </SPAN></H3><DIV><LABEL for="model-ct-source">Source</LABEL><ARTICLE class="ac-small"><P>You can download the source code of our model implementation as we used it for the presented results <A href="https://static.igem.org/mediawiki/2014/a/ad/LMU14-chemotaxis-model.zip"> here </A>.
</P><P>If you are interested in further developments after the Wiki-freeze, you can vistit the project on GitHub [<A href="https://github.com/hoerldavid/chemotaxis" class="external text" rel="nofollow">here</A>]. 
</P></ARTICLE></DIV></DIV></SECTION><SECTION><DIV><P>We implemented the model in C++, making extensive use of the <I>Boost</I> libraries for numerical integration, among other things. In addition,
we wrote gateway routines to allow handling of the model from a MATLAB script. This way, parameters can easily be changed and results can be visualized
using the plotting functionality of MATLAB.
</P><P>Initially, the arena in which chemotaxis should take place is defined as a polygon. An equally spaced 2-dimensional grid covering the polygon is created
to store concentrations of prey bacteria and attractant molecules (figure 1). The area in which prey bacteria grow (a 'colony') is defined as a rectangle and their initial concentration
is set.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1:</B> schematic overview of our model</DIV></DIV></DIV><P>A desired number of virtual hunter bacteria are created at set spacial coordinates. Initially, these bacteria have a random orientation and their chemotactic
pathway is inactive (methylation and phosphorylation levels are zero).
</P><P>The model is then updated by defined time steps until a desired end time is reached.
</P><P>
At each time step, the number of prey bacteria in the colony is updated according to their logistic growth curve. The concentration of attractant molecules changes
according to the diffusion equation (see above). The Laplacian is discretized on the grid using a standard 5-point stencil.
</P><P>In each time step, the bacteria will update their chemotactic pathway (according to the system of differential equations)
in response to the attractant concentration at their location (the mean of the nonnegative concentrations at the neighboring 4 grid nodes).
Each bacterium will then swim forward (run) or change its orientation (tumble) dependent on the phosphorylation level of CheY.
</P><H3><SPAN class="mw-headline" id="Results_.26_Discussion"> Results &amp; Discussion </SPAN></H3><P>We used the developed tool for repeated simulations of our model. We could indeed observe behavior resembling real chemotaxis with certain (rather arbitrary) combinations of parameters for producer growth
and diffusion of the attractant molecules, parameters for the behavior of the hunter bacteria were taken from [2]. However, the results leave much to be desired. Especially, the numeric integration fails under many parameter combinations due 
to the stiff nature of the differential equations encountered. We plan to improve out tool in the future.
</P><P>As an example for the encountered results, a time-lapse of a single simulation (attractant gradient, hunter positions and growth curve of prey bacteria) over 60 min
can be viewed [<A href="https://static.igem.org/mediawiki/2014/c/cf/LMU14_Chemotaxis.mp4" class="external text" rel="nofollow">here</A>]. The following parameters were used:
</P><TABLE class="wikitable"><TBODY><TR><TH>parameter
</TH><TH>value
</TH></TR><TR><TD> growth rate \( \mu_m\)
</TD><TD> \( \frac{1}{30} \big[ min^{-1} \big] \)
</TD></TR><TR><TD> lag time \( \lambda\)
</TD><TD> \( 0 \big[ sec \big] \)
</TD></TR><TR><TD> asymptote \( A\)
</TD><TD>  \( 2\)
</TD></TR><TR><TD> Diffusion coefficient \( D\)
</TD><TD> \( 0.1 \big[ \frac{mM}{mm^2} \big] \)
</TD></TR><TR><TD> production rate \( p\)
</TD><TD> \( 2 * 10^{-6} \big[ N_{prey}^{-1} \big] \)
</TD></TR><TR><TD> degradation rate \( a\)
</TD><TD> \( 2 * 10^{-3} \big[ mM^{-1} \big] \)
</TD></TR><TR><TD>  initial producers \( N_0\)
</TD><TD>  \( 5000 \big[ grid node^{-1} \big]\)
</TD></TR></TBODY></TABLE><P><SMALL><B>[1]</B> Zwietering, M. H., et al. &quot;Modeling of the bacterial growth curve.&quot; Applied and environmental microbiology 56.6 (1990): 1875-1881. </SMALL></P><P><SMALL><B>[2]</B> Matthäus, Franziska, Marko Jagodič, and Jure Dobnikar. &quot;<I> E</I>.<I> coli</I> Superdiffusion and Chemotaxis—Search Strategy, Precision, and Motility.&quot; Biophysical journal 97.4 (2009): 946-957.</SMALL></P></DIV></SECTION></SECTION></DIV><SECTION id="modelsuicide" class="scroll-spy jumptarget bg-color-3" data-scroll="suicidemodel"><DIV><H2><SPAN class="mw-headline" id="Suicide_Switch"> Suicide Switch </SPAN></H2><H3><SPAN class="mw-headline" id="Motivation_2"> Motivation </SPAN></H3><P>For safety issues we want BaKillus to autoinduce cell lysis after successful completion of his task. This can be achieved by shutting down the production of resistance proteins. <I>(see: <A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide" class="external text" rel="nofollow">Suicide-Switch</A>)</I>
Still, BaKillus must produce sufficient amounts of toxins before inducing cell lysis. Therefore, the negative feedback, which eventually stops production of the immunity protein must be delayed to provide a sufficiently long time frame for toxin production. However, as simple as this idea might sound in theory, robust practical implementation of such a module requires precise fine tuning of the kinetics of the respective signalling cascade. In this model we tried to assess whether the inclusion of an alternative sigma factor could theoretically delay the killing mechanism.
</P><P>In the following we will compare two different models, one including only one sigma factor and one including two sigma factors. The models will be compared based on a fitness function which describes the difference between the toxin and immunity production over time. This should provide a measure of the effective number of produced toxin molecules.
</P><H3><SPAN class="mw-headline" id="Basic_Model"> Basic Model </SPAN></H3><P>The basic model only contains one sigma factor. The system of differential equations was derived following the standard paradigm of gene expression including mRNA as well as protein dynamics. Inhibitory and promoting activity were modeled via hill kinetics.
</P><DIV><LABEL for="model-param">Variables</LABEL><ARTICLE class="ac-small"><H3><SPAN class="mw-headline" id="Variables"> Variables </SPAN></H3><TABLE><TBODY><TR><TD>m<SUB>sdpABC</SUB>:      </TD><TD>  Quantity of SdpABC mRNA
</TD></TR><TR><TD>p<SUB>sdpABC</SUB>:    </TD><TD>  Quantity of SdpABC proteins
</TD></TR><TR><TD>m<SUB>sdpI</SUB>:    </TD><TD>  Quantity of SdpI mRNA
</TD></TR><TR><TD>p<SUB>sdpI</SUB>:    </TD><TD>  Quantity of SdpI proteins
</TD></TR><TR><TD>m<SUB>SiAlt1</SUB>:    </TD><TD>  Quantity of sigma factor mRNA
</TD></TR><TR><TD>p<SUB>SiAlt1</SUB>:   </TD><TD>  Quantity of sigma factor proteins
</TD></TR><TR><TD>m<SUB>fsrA</SUB>:    </TD><TD>  Quantity of FsrA
</TD></TR><TR><TD>P<SUB>qs</SUB>:    </TD><TD>  Quantity of QS proteins
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Inquired_Parameters"> Inquired Parameters </SPAN></H3><TABLE><TBODY><TR><TD>C<SUB>0,P<SUB>qs</SUB></SUB>:    </TD><TD>  Basic concentration of QS proteins
</TD></TR><TR><TD>C<SUB>0,p<SUB>sdpI</SUB></SUB>:    </TD><TD>  Basic concentration of SdpI proteins
</TD></TR><TR><TD>C<SUB>0,fsrA</SUB>:    </TD><TD>  Basic concentration of FsrA
</TD></TR><TR><TD>k<SUB>0,SiAlt1</SUB>:    </TD><TD>  Transcription Rate of sigma factor
</TD></TR><TR><TD>k<SUB>1,SiAlt1</SUB>:    </TD><TD>  Translation Rate of sigma factor
</TD></TR><TR><TD>C<SUB>0,SiAlt1</SUB>:       </TD><TD>  Basic concentration of sigma factor proteins
</TD></TR><TR><TD>k<SUB>0,fsrA</SUB>:    </TD><TD>  Transcription Rate of FsrA
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Fixed_Parameters"> Fixed Parameters </SPAN></H3><P><I>In a first evaluation, the variation of following parameters were figured out to cause only insignificant variation of the model's fitness. Therefore they were kept at these specified values for all subsequent simulations:</I></P><TABLE><TBODY><TR><TD> Degradation rate (<B>g1</B>) of mRNA:    </TD><TD> 12 molecules per hour
</TD></TR><TR><TD> Degradation rate (<B>g2</B>) of proteins:    </TD><TD> 1 molecule per hour
</TD></TR><TR><TD> Transcription rate (<B>k0</B>) of Sdp:    </TD><TD> 7.2 per hour
</TD></TR><TR><TD> Translation rate (<B>k1</B>) of SdpI &amp; SdpABC:       </TD><TD> 10 per hour
</TD></TR></TBODY></TABLE></ARTICLE></DIV><DIV><LABEL for="model-code1">MATLAB-Script</LABEL><ARTICLE class="ac-small"><PRE>% par0 = lhsdesign(10000,12); 
par0 = rand(100000,12);
par = 10.^(4*par0 - 2); % Coveres range from 0.01 to 100

par(:,3) =12; % g1
par(:,5) =1; % g2
par(:,2) =0.12*60; % k0_sdp
par(:,4) =10; % k1_sdp

tout = linspace(0,10,100);
tic 
for j = 1:size(par,1)
   [~,~,x,y] = simulate_RRE_cannibalism(tout,par(j,:));
    %figure
    %plot(tout,y)
    %legend('p_{p_sdpABC}','p_{p_sdpI}')
   
   W(j) = sum(y(:,1)-y(:,2));
end
toc

parname={'C0_Pqs','k0_sdp','g1','k1_sdp','g2','C0_p_sdpI', ...
'C0_fsrA','k0_SiAlt1','k1_SiAlt1','C0_SiAlt1','k0_fsrA','k1_fsrA'};
 
for k = [1,6,7,8,9,10,11,12]
  figure
  scatter(par(:,k),W',10,'filled')
  hold on

  [xx,ind]=sort(par(:,k));
  r = ksr(log10(xx),W(ind));
 
  plot(10.^r.x,smooth(r.f,10),'k-','LineWidth',5)
  set(gca,'Yscale','lin')
  set(gca,'Xscale','log')
  xlabel(parname{k})
  ylabel('W')
  saveas(gcf,['parameter_' parname{k}]);
  print('-depsc2','-r300',['parameter_' parname{k}])
end
</PRE></ARTICLE></DIV><DIV><LABEL for="model-output1">Output</LABEL><ARTICLE class="ac-small"><H3><SPAN class="mw-headline" id="Fitness_W"> Fitness W </SPAN></H3><P><I>For analysis of the impact of each of the parameters, we used a function to determine the fitness <B>W</B> of the model. The fitness describes the integral of the surplus of toxic cannibalism proteins over the proprietary antidote. Therefore, a successful suicide equals a high fitness of the model. As we do not know the parameters for our model we sampled the whole parameter space using a uniform sampler. Successively we used a Nadaraya-Watson kernel density estimator to approximate the expected value of W given a single parameter value. This allows us to asses the expected change in W when changing a single parameter, whithout knowledge  of all other parameters.</I></P></ARTICLE></DIV><DIV><LABEL for="model-results1">Results</LABEL><ARTICLE class="ac-small"><P>As we can see, <B>C<SUB>0,p<SUB>sdpI</SUB></SUB></B> has no effect on the dynamics of the model. While high basic concentrations of <B>P<SUB>qs</SUB></B>, <B>FsrA</B> and the <B>sigma factor</B> have reciprocal impact on the fitness of the model, the transcription and translation rates of <B>FsrA</B> and the <B>sigma factor</B> affect the model's fitness positively.  
Therefore, the best results could be achieved with <B>low basic concentrations</B> and <B>high translation and transcription rates</B>.
</P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION id="model2" class="scroll-spy jumptarget bg-color-2" data-scroll="delaymodel"><DIV><H3><SPAN class="mw-headline" id="Delayed_Model"> Delayed Model </SPAN></H3><P>This model now considers a delayed SdpI inhibition which is realised by integrating a second sigma factor. 
The mathematical model for this system contains two additional equations for the new gene:
</P><DIV><LABEL for="model-code2">adapted MATLAB-Script</LABEL><ARTICLE class="ac-small"><PRE>% par0 = lhsdesign(10000,12); 
par0 = rand(100000,15);
par = 10.^(4*par0 - 2); % Coveres range from 0.01 to 100

par(:,3) =12; % g1
par(:,5) =1; % g2
par(:,2) =0.12*60; % k0_sdp
par(:,4) =10; % k1_sdp

tout = linspace(0,10,1000);
tic 
for j = 1:size(par,1)
    [~,~,x,y] = simulate_RRE_cannibalism_twosigma(tout,par(j,:));
%      figure
%      plot(tout,y)
%      legend('p_{p_sdpABC}','p_{p_sdpI}')
%     
    W(j) = sum(y(:,1)-y(:,2));
end
toc

parname={'C0_Pqs','k0_sdp','g1','k1_sdp','g2','C0_p_sdpI','C0_fsrA', 'k0_SiAlt1', ...
 'k1_SiAlt1','k0_SiAlt2','k1_SiAlt2','C0_SiAlt2','C0_SiAlt1','k0_fsrA','k1_fsrA'};


for k = [1,6,7,8,9,10,11,12,13,14,15]
   figure
   scatter(par(:,k),W',10,'filled')
   hold on
 
   [xx,ind]=sort(par(:,k));
   r = ksr(log10(xx),W(ind));

   plot(10.^r.x,smooth(r.f,10),'k-','LineWidth',5)
   set(gca,'Yscale','lin')
   set(gca,'Xscale','log')
   xlabel(parname{k})
   ylabel('W')
   saveas(gcf,['red_parameter_' parname{k}]);
   print('-depsc2','-r300',['red_parameter_' parname{k}])
end
</PRE></ARTICLE></DIV><DIV><LABEL for="model-output2">Output of delayed model</LABEL><ARTICLE class="ac-small"><P>As above, the graphs show the statistical fitness W of the model plotted against each considered parameter. The black line resembles the average. 
</P></ARTICLE></DIV><DIV><LABEL for="model-results2">Results</LABEL><ARTICLE class="ac-small"><P>As we can see, the results qualitatively reflect the insights from the basic model: <B>C<SUB>0,p<SUB>sdpI</SUB></SUB></B> still has no significant effect on the dynamics of the model at all. 
And as already discovered before, the best results can be achieved with low basic concentrations and high translation and transcription rates. Just quantitatively this model shows slightly higher fitness due to the additional delay as expected.
</P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION id="conclusion-suicide" class="scroll-spy jumptarget bg-color-2" data-scroll="delaymodel"><DIV><H3><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H3><P>To compare both models, we plotted the mean fittness of the model with one sigma factor (black) as well as the mean fittness of the model with two sigma factors (red).
One sees a flat increase across all parameters across all values by over an order of magnitude. This suggests that the inclusion of an additional sigma factor should drastically increase the killing effectivity of our suicide-switch.
</P></DIV></SECTION><H2><SPAN class="mw-headline" id="Codon_Adaptation"> Codon Adaptation </SPAN></H2><P>Organisms differ in their usage of codons for the same amino acids. These differences can hamper the expression of foreign genes. When synthesizing genes from scratch, one can improve the chances of successful expression by performing <I>codon optimization</I>. Commercial providers of gene synthesis offer free tools to optimize the codon usage of ordered synthetic genes, usually by picking the most frequently used codon for a desired amino acid in the target organism.
</P><P>However, simply picking the most common codon might not lead to optimal results. Another strategy is to pick codons with similar usage as in the source organism. This <I>condon adaptation</I> should lead to translation closely resembling the source organism. 
</P><P>We wrote a small <I>Python</I>-script to perform codon adaptation for the synthetic genes we ordered over the course of our project. It is still in a very crude state, by we nonetheless wish to share it with you. You can get the tool and follow novel developments on GitHub [<A href="https://github.com/hoerldavid/codonoptimizer" class="external text" rel="nofollow">here</A>].
</P></DIV><DIV id="footer"><DIV id="footer-content"><H2>Team LMU-Munich</H2><H3>igem@bio.lmu.de </H3>

AG Thorsten Mascher 
Großhaderner Str. 2 
82152 Planegg (Martinsried) </DIV></DIV><DIV id="tour-overlay"><DIV id="tour-box"><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Hi_there.21"> Hi there! </SPAN></H2><P>Welcome to our Wiki! I'm BaKillus, the pathogen-hunting microbe, and I'll guide you on this tour through our project. If you want to learn more about a specific step, you can simply close the tour and come back to it anytime you like. So let's start!

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="What.27s_the_problem.3F"> What's the problem? </SPAN></H2><P>First of all, what am I doing here? The problem is, pathogenic bacteria all around the world are becoming more and more  resistant against antimicrobial drugs. One major reason for the trend is the inappropriate use of drugs. With my BaKillus super powers, I want to reduce this misuse and thus do my part to save global health.

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Sensing_of_pathogens"> Sensing of pathogens </SPAN></H2><P>To combat the pathogenic bacteria, I simply eavesdrop on their communication. Bacteria talk with each other via quorum sensing systems, which I use to detect them and trigger my responses. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Adhesion"> Adhesion </SPAN></H2><P>The more specific and effective I can use my powers, the lower the danger is of provoking new resistance development. So I catch pathogens whenever I get hold of them and stick to them until my work is done.

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Killing"> Killing </SPAN></H2><P>Talking about my work - killing pathogens is finally what I am made for. In response to quorum sensing molecules of the pathogens, I export a range of antimicrobial substances leading to dissipation of biofilms and the killing of the targeted bacteria. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Suicide_switch_2"> Suicide switch </SPAN></H2><P>When the job is done and all the bad guys are finished, you don't need a super hero anymore. So after fulfilling my work I say goodbye to the world by activating my suicide switch. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Application"> Application </SPAN></H2><P>Of course I'm not only a fictional hero, but a very real one. In two different prototypes, I could be used for diagnosis or treatment of pathogen-caused diseases. However, there is still a whole lot of regulational and economical questions that have to be answered before. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="See_you.21"> See you! </SPAN></H2><P>So now you know my short story - and it is time for me to return to my fight for a safer world. Feel free to take a closer look on my <A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus" class="external text" rel="nofollow">super powers</A>, the <A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Timeline" class="external text" rel="nofollow">process of my development</A> or the plans for a <A href="https://2014.igem.org/Team:LMU-Munich/Application" class="external text" rel="nofollow">medical application</A>.

</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:LMU-Munich/Project/Modeling">http://2014.igem.org/Team:LMU-Munich/Project/Modeling</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Project/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Project/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Modeling&amp;oldid=403668" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>