<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Project_Bakillus"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Project/Bakillus">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Project/Bakillus&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Bakillus&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Bakillus&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Project/Bakillus" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Project/Bakillus</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="mobile-navigation"><UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Financing">Financing</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem">Problem</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus">BaKillus</A></LI><LI id="inpage-nav-sensing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#sensing">sensing</A></LI><LI id="inpage-nav-adhesion"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#adhesion">adhesion</A></LI><LI id="inpage-nav-killing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#killing">killing</A></LI><LI id="inpage-nav-suicide"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide">suicide</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox"><SPAN class="bacillus">Bacillus</SPAN> BioBrickBox 2.0</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/B_subtilis" class="bacillus">B. subtilis</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry">Registry</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Judging">Judging</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application">Application</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Diagnosis_Treatment">Diagnosis &amp; Treatment</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Pretest">Pretest</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Legal_Issues">Legal Issues</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Financing_Intellectual_Property">Financing &amp; Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Expertise">Expertise</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice">Human Practice</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Meetups">Meet ups</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/iGEM_School">iGEM@school</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/SynBio">SynBio Day</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Panel_Discussion">Panel Discussion</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Rathenau">Rathenau Instituut</A></LI></DIV><DIV id="tour-toggle-m" class="hidden-md hidden-lg">TOUR</DIV><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><DIV class="nav-item" id="tour-toggle">Tour</DIV></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project" class="project nav-item active" id="project">Project</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results" class="results nav-item" id="results">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="labbook nav-item" id="labbook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application" class="safety nav-item" id="application">Application</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice" class="outreach nav-item" id="human-practice">Human Practice</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="team-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team" class="sub-nav-item">Team</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Financing" class="sub-nav-item">Financing</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Collaborations" class="sub-nav-item">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Team/Attributions" class="sub-nav-item">Attributions</A></LI></UL><UL id="project-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem" class="sub-nav-item">Problem</A></LI><LI id="inpage-nav-bakillus"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus" class="sub-nav-item">BaKillus</A></LI><LI id="inpage-nav-sensing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#sensing" class="sub-nav-item">sensing</A></LI><LI id="inpage-nav-adhesion"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#adhesion" class="sub-nav-item">adhesion</A></LI><LI id="inpage-nav-killing"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#killing" class="sub-nav-item">killing</A></LI><LI id="inpage-nav-suicide"><A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus#suicide" class="sub-nav-item">suicide</A></LI></UL><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/BioBrickBox" class="sub-nav-item"><SPAN class="bacillus">Bacillus</SPAN> BioBrickBox 2.0</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling" class="sub-nav-item">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Project/B_subtilis" class="sub-nav-item bacillus">B. subtilis</A></LI><UL id="results-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results" class="sub-nav-item">Results</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry" class="sub-nav-item">Registry</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Results/Judging" class="sub-nav-item">Judging</A></LI></UL><UL id="application-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Diagnosis_Treatment" class="sub-nav-item">Diagnosis &amp; Treatment</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Pretest" class="sub-nav-item">Pretest</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Legal_Issues" class="sub-nav-item">Legal Issues</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Financing_Intellectual_Property" class="sub-nav-item">Financing &amp; Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Application/Expertise" class="sub-nav-item">Expertise</A></LI></UL><UL id="human-practice-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Meetups" class="sub-nav-item">Meet ups</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/iGEM_School" class="sub-nav-item">iGEM@school</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/SynBio" class="sub-nav-item">SynBio Day</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Human_Practice/Panel_Discussion" class="sub-nav-item">Panel Discussion</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Rathenau" class="sub-nav-item">Rathenau Instituut</A></LI></UL><UL id="labbook-sub"><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="sub-nav-item">Protocols</A></LI></UL></DIV></DIV></DIV><DIV id="lmu-content"><DIV id="project-bakillus-teaser" class="full-width teaser-img"> </DIV><SECTION><SECTION id="bakillus" class="scroll-spy" data-scroll="bakillus"><DIV><H1><SPAN class="mw-headline" id="BaKillus_-_Engineering_a_pathogen-hunting_microbe"> BaKillus - Engineering a pathogen-hunting microbe </SPAN></H1><P>Increasing bacterial resistance to classical antibiotics remains a serious threat and urges the development of novel pathogen-killing strategies. Exploiting bacterial communication mechanisms such as quorum sensing (QS) is a promising strategy to specifically target certain pathogens. The major aim of this project is the introduction of a genetic circuit enabling <I>Bacillus subtilis</I> to actively detect, attach to, and eventually kill <I>Staphylococcus aureus</I> and <I>Streptococcus pneumoniae</I>.
Initially, we will introduce the autoinducer-sensing two-component systems of <I>S. aureus</I> and <I>S. pneumoniae</I> into <I>B. subtilis</I> to create a pathogen-detecting strain. By utilizing QS-dependent promoters, we will then trigger pathogen-killing strategies like the production of antimicrobial peptides or biofilm degradation. As a safety measure, a delayed suicide-switch guarantees non-persistence of genetically modified <I>B. subtilis</I> in the absence of pathogens. We envision the use of BaKillus as a smart, cheap and simple-to-use medical device for diagnostics and targeted treatment of multiresistant superbugs.
</P></DIV></SECTION><SECTION id="sensing" class="scroll-spy jumptarget bg-color-2" data-scroll="sensing"><DIV><H2><SPAN class="mw-headline" id="Sensing"> Sensing </SPAN></H2><P>This part of the BaKillus-concept focuses on specific pathogen detection. Here we use quorum sensing to specifically recognize the presence of certain pathogens. The major aim of this subproject is transferring the quorum sensing two-component systems AgrC-II/AgrA of <I>Staphylococcus aureus</I> and ComDE of <I>Streptococcus pneumoniae</I> into <I>Bacillus subtilis</I>, thus creating a pathogen detecting strain. With this strategy we will trigger our killing strategies only if a threshold concentration of autoinducers, produced by pathogens, is present in the environment Thereby we aim to drastically increase the specificity of our antimicrobial strategies compared to commonly used broad-spectrum antibiotic therapy.  
</P><DIV><LABEL for="sensing-bground">Background</LABEL><ARTICLE class="ac-small"><H2><SPAN class="mw-headline" id="Staphylococcus_aureus"><I>Staphylococcus aureus</I></SPAN></H2><H3><SPAN class="mw-headline" id="Virulence_of_S._aureus">Virulence of  <I>S. aureus</I></SPAN></H3><P><I>S. aureus</I> is living as a commensal in humans with a prevalence of up to 30% of the population in Europe <A href="http://www.nivel.nl/sites/default/files/ECCMID%202013%20-%20Resistance%20results_0.pdf" class="external text" rel="nofollow">(1)</A>. It is also responsible for a variety of diseases ranging from skin infections to life threatening toxinosis and is quickly adapting to the use of common antibiotics by becoming resistant.  
This dual lifestyle and the expression of a multitude of virulence factors is tightly regulated by a process termed quorum sensing <A href="http://jb.asm.org/content/193/21/6020" class="external text" rel="nofollow">(2)</A>.
</P><H3><SPAN class="mw-headline" id="Quorum_sensing_in_S._aureus">Quorum sensing in <I>S. aureus</I></SPAN></H3><P>Quorum Sensing is a mechanism of bacterial cell-cell communication, granting bacteria the ability to measure the density of their surrounding cell population, and react to this by upregulation of specific genes. 
In <I>S.  aureus</I> this process is one of the major regulator of its virulence, making sure that energy-costly virulence factors are only produced when a large enough number of pathogens is present to overcome host-defence mechanisms. 	
</P><P>In this mechanism a certain peptide termed autoinducing peptide (AIP) is produced from a pro-peptide called AgrD and further processed and exported by a membrane protein AgrB to create a cyclic AIP. When a certain threshold concentration of AIP in the environment is reached, AIPs activate the receptor-histidine kinase AgrC. 
Upon induction AgrC phosphorylates the intracellular response regulator AgrA with its cytosolic histidine kinase domain <A href="http://www.annualreviews.org/doi/abs/10.1146/annurev.genet.42.110807.091640" class="external text" rel="nofollow">(3)</A>. Phosphorylated AgrA then binds to distinct sites in the P2 and P3 promoter-region and activates these promoters. <A href="http://jb.asm.org/content/193/21/6020" class="external text" rel="nofollow">(2)</A>
The P2 promoter activates transcription of the <I>agrBDCA</I>-Operon and thereby represents a positive feedback-loop. The P3 promoter activates transcription of the RNA-III transcript, involved in virulence gene expression <A href="http://jb.asm.org/content/182/22/6517.full" class="external text" rel="nofollow">(4)</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:416px;"><DIV class="thumbcaption"><B>Fig.1</B>: Quorum sensing in <I>Staphylococcus aureus</I>. Taken from Novick and Geisinger, &quot;Quorum Sensing in Staphylococci&quot;, Annual Review of Genetics, Vol. 42: 541-564 (2008)</DIV></DIV></DIV></DIV><P>Interestingly, accumulated mutations in the hypervariable region of the <I>agrBDCA</I>-operon have led to four different pherotypes of <I>S. aureus</I>, using different AgrC-types, and producing different AIPs (AIPI, AIPII, AIPIII, AIPIV), which cross-inhibit each other and might correlate with certain diseases associated with <I>S. aureus</I><A href="http://www.sciencemag.org/content/287/5452/391.full.html" class="external text" rel="nofollow">(5)</A>.
</P><P>For this project we used genomic DNA derived from the strain <I>Staphylococcus aureus</I> N315, which is a MRSA strain, belonging to the AIP-II group.
</P><H2><SPAN class="mw-headline" id="Streptococcus_pneumoniae"><I>Streptococcus pneumoniae</I></SPAN></H2><H3><SPAN class="mw-headline" id="Virulence_of_S._pneumoniae">Virulence of <I>S. pneumoniae</I></SPAN></H3><P>All over the world, <I>Streptococcus pneumoniae</I> causes the life threating invasive diseases pneumonia, sepsis and meningitis. In early childhood the diplococcus colonizes the epithelium of the upper respiratory tract as a commensal bacterium. Under certain circumstances,  it gets a threat by tissue invasion.  Still up to date, not much is known regarding the mechanism behind the shifting from a colonizer organism to an invader or how <I>S. pneumoniae</I> is able to cross the blood-brain barrier to cause meningitis <A href="http://www.ncbi.nlm.nih.gov/pubmed/19880020" class="external text" rel="nofollow">(6)</A>.
To cross the line from an opportunist to a pathogen, <I>S. pneumoniae</I> needs to express specific virulence factors in a coordinated way.  Apart from virulence factors like the polysaccharide capsule, the pore forming toxin pneumolysin or surface proteins like PspA and PspC, also biofilm production plays an important role for persistence in the human nasopharynx and contributes to pneumonia and meningitis <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1618759" class="external text" rel="nofollow">(7)</A>.
</P><P>One aspect of its infectiousness is the ability to perform natural transformation. Thus, it can take up extracellular DNA containing the information for resistance against diverse antibiotics and permanently incorporate the DNA into its genome via recombination.  The main <A href="https://2014.igem.org/Team:LMU-Munich/Project/Problem" class="external text" rel="nofollow">problem</A> of treating <I>S. pneumoniae</I> infections is the increasing spread of ß-lactam resistance. Luckily, the molecular mechanism behind natural transformation in <I>S. pneumoniae</I> is the focus of current research.
</P><H3><SPAN class="mw-headline" id="Natural_Transformation_in_S._pneumoniae">Natural Transformation in <I>S. pneumoniae</I></SPAN></H3><P>In <I>S. pneumoniae</I>, biofilm production and competence for natural genetic transformation are triggered by a two component regulatory system, responsive to environmental stimuli. There, a peptide pheromone called competence-stimulating-peptide (CSP) binds to and consequently activates the membrane-embedded sensor kinase ComD by changing the conformation of the polytopic kinase. ComD autophosphorylates and subsequently transphosphorylates the cognate response regulator ComE. ComE acts as transcription activator via binding to the -10 promoter regions of early and late competence genes. These ComE binding sites (CEbs) consist of a 9 bp direct repeat seperated by a stretch of 12 nucleotides.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:679px;"><DIV class="thumbcaption"><B>Fig. 2</B>: Competence regulation in <I>Streptococcus pneumoniae</I>. See text for more details. (Jonsborg et al. 2009)</DIV></DIV></DIV><P>A positive feedback loop ensures the availability of the two component system by activating the expression of the <I>comCDE</I> operon, where <I>comDE</I> encodes for the two component system ComDE and <I>comC</I> for the precursor peptide pre-CSP. Pre-CSP is cleaved to the mature peptide pheromone and transported to the extraplasmic room by the ABC transporter ComAB, whose expression is also activated by ComE. <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=regulation+of+natural+genetic+transformation+and+acquisition+of+transforming+DNA" class="external text" rel="nofollow">(8)</A>. 
In addition, ComE~P is able to trigger the expression of <I>comX</I> and <I>comW </I>, encoding for the alternative σ-factor X, and its stability factor ComW. This alternative σ-factor is required for the transcription of the so called late competence genes involved in the process of DNA uptake and recombination.
</P><P>
We are working with the genomic DNA of the nonencapsulated apathogenic strain R6 of <I>S. pneumoniae</I>, whose genome is completely sequenced <A href="http://www.ncbi.nlm.nih.gov/pubmed/11544234" class="external text" rel="nofollow">(9)</A>. CSP is a 17 amino acid long linear protein  with the following amino acid sequence: Glu-Met-Arg-Leu-Ser-Lys-Phe-Phe-Arg-Asp-Phe-Ile-Leu-Gln-Arg-Lys-Lys-OH <A href="http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=134925&amp;disp=4" class="external text" rel="nofollow">(10)</A> with a charged N-terminal residue and a positively charged C-terminal tail <A href="http://www.ncbi.nlm.nih.gov/pubmed/16484185" class="external text" rel="nofollow">(11)</A>.
</P><H3><SPAN class="mw-headline" id="Sources">Sources</SPAN></H3><P><SMALL>
[1] den Heijer, C. D., van Bijnen, E. M., Paget, W. J., Pringle, M., Goossens, H., Bruggeman, C. A., . . . Team, A. S. (2013). Prevalence and resistance of commensal Staphylococcus aureus, including meticillin-resistant S aureus, in nine European countries: a cross-sectional study. Lancet Infect Dis, 13(5), 409-415. doi: 10.1016/S1473-3099(13)70036-7
</SMALL></P><P>[2] Reyes, D., Andrey, D. O., Monod, A., Kelley, W. L., Zhang, G., &amp; Cheung, A. L. (2011). Coordinated regulation by AgrA, SarA, and SarR to control agr expression in Staphylococcus aureus. J Bacteriol, 193(21), 6020-6031. doi: 10.1128/JB.05436-11
</P><P>[3] Novick, R. P., &amp; Geisinger, E. (2008). Quorum sensing in staphylococci. Annu Rev Genet, 42, 541-564. doi: 10.1146/annurev.genet.42.110807.091640
</P><P>[4] Jarraud, S., Lyon, G. J., Figueiredo, A. M., Lina, G., Vandenesch, F., Etienne, J., . . . Novick, R. P. (2000). Exfoliatin-producing strains define a fourth agr specificity group in Staphylococcus aureus. J Bacteriol, 182(22), 6517-6522. 
</P><P>[5] R. P. Novick, H.F. Ross, A. M. S. Figueiredo, G. Abramochkin (2000) Activation and Inhibition of the Staphylococcal AGR System. Science 21 Vol. 287 no. 5452 p. 391
</P><P>[6] van der Poll, T., &amp; Opal, S. M. (2009). Pathogenesis, treatment, and prevention of pneumococcal pneumonia. Lancet, 374(9700), 1543-1556. doi: 10.1016/S0140-6736(09)61114-4
</P><P>[7] Oggioni, M. R., Trappetti, C., Kadioglu, A., Cassone, M., Iannelli, F., Ricci, S., . . . Pozzi, G. (2006). Switch from planktonic to sessile life: a major event in pneumococcal pathogenesis. Mol Microbiol, 61(5), 1196-1210. doi: 10.1111/j.1365-2958.2006.05310.x
</P><P>[8] Johnsborg, O., &amp; Havarstein, L. S. (2009). Regulation of natural genetic transformation and acquisition of transforming DNA in Streptococcus pneumoniae. FEMS Microbiol Rev, 33(3), 627-642.
</P><P>[9] Hoskins, J., Alborn, W. E., Jr., Arnold, J., Blaszczak, L. C., Burgett, S., DeHoff, B. S., . . . Glass, J. I. (2001). Genome of the bacterium Streptococcus pneumoniae strain R6. J Bacteriol, 183(19), 5709-5717. doi: 10.1128/JB.183.19.5709-5717.2001
</P><P>[10] Sequence obtained from <A href="http://www.microbesonline.org" class="external free" rel="nofollow">http://www.microbesonline.org</A></P><P>[11] Johnsborg, O., Kristiansen, P. E., Blomqvist, T., &amp; Havarstein, L. S. (2006). A hydrophobic patch in the competence-stimulating Peptide, a pneumococcal competence pheromone, is essential for specificity and biological activity. J Bacteriol, 188(5), 1744-1749. doi: 10.1128/JB.188.5.1744-1749.2006
</P></ARTICLE></DIV><DIV><LABEL for="sensing-design">Design</LABEL><ARTICLE class="ac-small">

To induce the quorum sensing module of <I>S. aureus</I>  and <I>S. pneumoniae</I> into BaKillus, two different parts had to be transferred to <I>B. subtilis</I>: First, the two component system  AgrCA of <I>S. aureus</I> or ComDE of <I>S. pneumoniae</I> need to be established for <I>B. subtilis</I> to sense and respond to the quorum sensing molecules of the specific pathogen. As a second step, the QS-activated promoters (P<SUB><I>QS</I></SUB>) need to be transformed into <I>B. subtilis</I> to trigger the expression of our killing factors.
For evaluation, the QS-dependent promoters were cloned in front of the <I>lux</I>-reporter cassette of the vector <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823025" class="external text" rel="nofollow">pBS3C-<I>lux</I></A> from the <A href="https://2012.igem.org/Team:LMU-Munich/Bacillus_BioBricks/Vectors" class="external text" rel="nofollow"><I>Bacillus</I> BioBrickBox</A> of the iGEM LMU team 2012.
<H3><SPAN class="mw-headline" id="QS_Two_Component_System">QS Two Component System</SPAN></H3><P>The two component system <A href="http://parts.igem.org/Part:BBa_K1351036" class="external text" rel="nofollow"><I>agrCII-A</I></A> for <I>S. aureus</I> has been designed as a composite part derived from <I>Staphylococcus aureus</I> N315 gDNA.  
The two intermediated parts <I>agrC-II</I> and <I>agrA</I> were amplified by PCR using primers with modified RFC25 prefix and suffix. And then fused by using the restriction sites EcoRI and SpeI for <I>agrC-II</I>, as well as XbaI and PstI for <I>agrA</I>, thus creating the composite part <A href="http://parts.igem.org/Part:BBa_K1351036" class="external text" rel="nofollow"><I>agrCII-A</I></A>. 
</P><P>The operon encoding for the histidine sensor kinase ComD and the response regulator ComE <I>comDE</I> for <I>S. pneumoniae</I> sensing was designed in two different ways: On one hand, the native <I>comDE</I> operon with its native ribosome binding site (RBS) (<A href="http://parts.igem.org/Part:BBa_K1351015" class="external text" rel="nofollow">BBa_K1351015</A>), on the other hand the<I>comDE</I> operon with the <I>B. subtilis</I> adapted RBS (TAAGGAGG) in the BioBrick standard RFC25 (<A href="http://parts.igem.org/Part:BBa_K1351016" class="external text" rel="nofollow">BBa_K1351016</A>). Unfortunately, the operon contains two EcoRI and one SpeI restriction site. The attempts to mutagenize these restriction sites were not completed successfully in time as one recognition site remained. Thus, the <I>comDE</I> operon could not been sent to the registry - but this will be accomplished as soon as possible.
</P><H3><SPAN class="mw-headline" id="QS_dependent_Promoters">QS dependent Promoters</SPAN></H3><P>In <I>S. aureus</I>, the AgrA-sensitive promoters P2 <A href="http://parts.igem.org/Part:BBa_K1351037" class="external text" rel="nofollow">BBa_K1351037</A> and P3 <A href="http://parts.igem.org/Part:BBa_K1351038" class="external text" rel="nofollow">BBa_K1351038</A> with four AgrA binding sites downstream of the -35 signals were used for evaluation. The intergenic region with suspected regulator binding sites is shown in Fig. 3 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:427px;"><DIV class="thumbcaption"><B>Fig. 3</B>: AgrA dependent promoters P2 and P3 with the intergenic region containing several regulator binding sites for AgrA (arrows) and SarA/R (dashed lines) (figure taken from Reyes et al. 2000)</DIV></DIV></DIV></DIV><P>
For <I>S. pneumoniae</I> sensing, the two  ComE dependent promoters P<SUB><I>comC</I></SUB><A href="http://parts.igem.org/Part:BBa_K1351024" class="external text" rel="nofollow">BBa_K1351024</A> and P<SUB><I>comAB</I></SUB><A href="http://parts.igem.org/Part:BBa_K1351025" class="external text" rel="nofollow">BBa_K1351025</A> and additional promoters containing putative ComE binding sites (CEbs) P<SUB><I>mreA</I></SUB>,  P<SUB><I>natAB</I></SUB>) were taken for evaluation (see figure 4).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Fig. 4</B>: ComE dependent promoters containing bases matching (blue) and differing (red) from the CEbs consensus sequence. (figure modified from Martin et al. 2013)</DIV></DIV></DIV></DIV><P>To test the functionality of our model, first, the two component system was cloned into the integrative vector <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823027" class="external text" rel="nofollow">pBS2E</A> from the <A href="https://2012.igem.org/Team:LMU-Munich/Bacillus_BioBricks/Vectors" class="external text" rel="nofollow"><I>Bacillus</I> BioBrick Box</A> under the control of P<SUB><I>xyl</I></SUB>, an inducible promoter that responds to extracellular xylose (see fig. A). The corresponding QS-dependent promotor was cloned in the vector <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823025" class="external text" rel="nofollow">pBS3C-<I>lux</I></A>, thus fused to <I>luxABCDE</I>.  If QS-molecules are detected from the two component system, P<SUB><I>QS</I></SUB> will trigger the expression of <I>luxABCDE</I>, resulting in luminescence which was measured by a multi-mode plate reader (18 hrs, 37 °C, shaking).
</P><P>In the distant future, the quorum sensing two component system, under the regulation of an inducible promoter, will be combined with an quorum sensing promoter expressing specific killing factors (see fig. B).
</P></ARTICLE></DIV><DIV><LABEL for="sensing-results">Results</LABEL><ARTICLE class="ac-small"><H2><SPAN class="mw-headline" id="S._aureus_Sensing"><I>S. aureus</I> Sensing</SPAN></H2><P>We constructed the biobrick &lt;bbpart&gt;BBa_K316036&lt;/bbpart&gt; encoding for a composite part containing  the AIP-II sensing two component system AgrC-IIA
<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K316036" class="external autonumber" rel="nofollow">[1]</A> 
And the AIP-II sensing dependant promoters P2  and P3 &lt;bbpart&gt;BBa_K316037&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K316038&lt;/bbpart&gt; P2  <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K316037" class="external autonumber" rel="nofollow">[2]</A> 
and P3 <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K316038" class="external autonumber" rel="nofollow">[3]</A></P><P>As a first step we recorded the promoter backgrounds of the P2 and P3 promoters with a <A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="external text" rel="nofollow">luminescence reporterassay</A>, using the strategy outlined in design. (Fig.3)
As expected according to the strength of the -35 and -10 signals, P2 is the stronger promoter showing a higher background, compared to P3. 
</P><P>In a next step we induced the expression of <I>PxylA-agrCII-A</I> by adding 0%, 0.02%, 0.2% Xylose at an OD<SUB>600</SUB> of about 0.1. 
The result is depicted in Figure 5. This result is in accordance with our expactations since strong overexpression of the two-component system should lead to unspecific induction of the P2 and P3 promoters, and thereby can be used as a method to measure relative expression of <I>agrC-IIA</I>. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:712px;"><DIV class="thumbcaption"><B>Fig.5</B>: Luminescence Reporterassay of the strain W168 <I>lacA::PxylA-agrCA; sacA::</I>P2/P3-<I>luxABCDE</I></DIV></DIV></DIV></DIV><P>Using the strategy outlined in design, we tried to induce the AgrC-IIA two-component system, with synthetic AIP-II. The synthetic peptids have been dissolved in 100% DMSO to 1 mM concentration and further diluted in dH<SUB>2</SUB>0. 
The constructs have then been induced with a range of different concentrations of AIP-II at an OD<SUB>600</SUB> of 0.1. 
The result is shown in Figure 6. Unfortunatly we were not able to show induction of the system by addition of AIP-II. Several other approaches like the addition of a protease inhibitor cocktail to the growth media, several washing steps, using LB instead of MCSE or inducing with 50 µM AIP-II did not change this observation.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:769px;"><DIV class="thumbcaption"><B>Fig.6</B>: Luminescence Reporterassay of the strain W168 <I>lacA::PxylA-agrCA; sacA::</I>P2/P3-<I>luxABCDE</I> induced with AIP-II</DIV></DIV></DIV></DIV><P>There are several possible reasons why the induction didn't work as expected. One idea is that the extracellular proteases produced by <I>Bacillus subtilis</I> destroy the synthetic AIPs. So we tried adding protease inhibitor cocktails to the growth media and using the protease deficient strain <I>Bacillus subtilis</I> WB700. We successfully transformed the WB700 strain with the reporter construct pBS4S-P<I>xylA-agrCA</I>. However the reporter constructs pBS3C-P2/P3-<I>luxABCDE</I> refused transformation.
All in all this set of constructs needs several further protocol optimisations and testing.
</P><H2><SPAN class="mw-headline" id="S._pneumoniae_Sensing"><I>S. pneumoniae</I> Sensing</SPAN></H2><P>First, we optimized the BioBrick &lt;bbpart&gt;BBa_K316018&lt;/bbpart&gt; encoding for the <I>comC</I> promoter  <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K316018" class="external autonumber" rel="nofollow">[4]</A> by sending our BioBrick &lt;bbpart&gt;BBa_K1351024&lt;/bbpart&gt; to the registry. For the previous version, no DNA was available.
</P><P>The BIG result we obtained in the night of the wiki freeze performing a <A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="external text" rel="nofollow">luminescence assay</A> with different constructs in <I>B. subtilis</I> P<SUB><I>xyl</I></SUB> - <I>comDE</I> P<SUB><I>QS</I></SUB> - <I>lux</I> based on <A href="http://parts.igem.org/Part:BBa_K1351016" class="external text" rel="nofollow">BBa_K1351016</A>. Three different P<SUB><I>QS</I></SUB> were tested: P<SUB><I>comC</I></SUB>, P<SUB><I>comAB</I></SUB>, P<SUB><I>mreA</I></SUB>. The <I>comDE</I> operon expression was induced with 0 %, 0,02 % and 0,2 % xylose. Adding 0 mg/ml CSP, 100 mg/ml CSP and 1000 mg/ml CSP <I>luxABCDE</I> expression is only activated if <I>B. subtilis</I> was able to sense CSP due to the <I>comDE</I> operon.
For P<SUB><I>mreA</I></SUB>, no significant luminescence output could be measured concerning differing CSP concentrations. Nevertheless, at a xylose concentration of 0,02 % and induction of 1000 mg/ml CSP P<SUB><I>comC</I></SUB> and P<SUB><I>comAB</I></SUB> showed a 3 fold increased luminescence output compared with an induction of 0 mg/ml and 100 mg/ml CSP.
</P><P>
These results were obtained with cultures growing in MCSE medium two hours after induction with CSP. A concentration of 0.02 % xylose was optimal to measure CSP dependent luminescence.
</P></ARTICLE></DIV></DIV></SECTION></SECTION></DIV><SECTION id="adhesion" class="scroll-spy jumptarget bg-color-3" data-scroll="adhesion"><DIV><H2><SPAN class="mw-headline" id="Adhesion"> Adhesion </SPAN></H2><P>As part of the BaKillus strategy, we want to reduce the selective pressure on non-pathogenic bacteria to prevent resistance development. Towards this end, BaKillus is equipped with pathogen-binding peptides on its surface, thereby adhering specifically to the targeted bacteria. This approach ensures high local concentration of killing factors while minimizing harmful effects on untargeted species.
</P><P>Surface display of the pathogen-binding peptides is achieved by translational fusions to naturally occurring surface proteins of <I>B. subtilis</I>. Cell wall-binding modules of autolysins are used for non-covalent interaction with the <I>B. subtilis</I> cell wall, whereas a sortase-based system functions via covalent attachment to the cell wall. In both cases, a linker between cell wall anchor and peptide allows flexibility of the fusion protein and increases its range. Peptides binding to different receptors on the surface of pathogens mediate specific binding.
</P><DIV><LABEL for="adhesion-bground">Background</LABEL><ARTICLE class="ac-small"><H3><SPAN class="mw-headline" id="Surface_display">Surface display</SPAN></H3><P>Bacterial cell surfaces are covered with a wide range of proteins functioning as transporters, interacting with molecules and cells or fulfilling a variety of other functions. As they are naturally displayed on the cell’s surface, they can also be used for presenting heterologous proteins or peptides.
</P><P>One class of proteins that have been used for surface display are <I>B. subtilis</I> autolysins, in particular LytC and LytE [1, 2]. Autolysins hydrolase peptidoglycan and thus play a role in cell wall turnover, vegetative growth and cell division. They consist of an N-terminal cell wall binding (CWB) domain with a signal peptide directing export via the Sec pathway and of a C-terminal catalytic domain [3, 4]. Fusions of the CWB domains of LytC and LytE to heterologous proteins have been shown to successfully facilitate surface display of their passenger protein in <I>B. subtilis</I> [1, 2].
Another promising system is based on the <I>B. subtilis</I> sortase YhcS and its substrate YhcR. Sortases represent a class of membrane-anchored proteins in gram-positive bacteria, which recognize their substrates via a conserved C-terminal pentapeptide sequence. They catalyze covalent binding of their substrates to lipid II, an intermediate of peptidoglycan synthesis [5]. Two putative sortase-substrate pairs have been identified in <I>B. subtilis</I>, one of them being YhcS/YhcR, which has already been used for successful surface display of recombinant proteins [5, 6].
</P><H3><SPAN class="mw-headline" id="Pathogen-binding_peptides">Pathogen-binding peptides</SPAN></H3><P>Bacterial adhesins mediate binding of the cells to all kind of surfaces, thereby preventing removal by physical influences like sheer forces. Attachment of pathogens to their host is often crucial for infection, which is why many adhesins are listed as virulence factors [7]. Binding partners of adhesins are often components of the extracellular matrix, which would be difficult to express on the BaKillus surface. In contrast, proteins of human cells that get bound by adhesins are optimal candidates for translational fusions to surface display systems. BaKillus mimics human cells by expressing parts of the proteins on its surface, leading to binding of the pathogens to BaKillus. Barbu <I>et. al.</I> 2010 [8] and Muchnik <I>et. al.</I> 2013 [9] used phage display to identify the binding peptide interaction partners of <I>Staphylococcus aureus</I> virulence factor SdrC and <I>Streptococcus pneumoniae</I> adhesin NOX, respectively. Those peptides are the specific binding sequences of SdrC and NOX within surface proteins of human cells. Thus, expression of the peptides on the BaKillus surface should enable it to adhere to the targeted pathogens. 
</P><H3><SPAN class="mw-headline" id="Sources_2">Sources</SPAN></H3><P>1.	Chen, C.L., et al., Development of a LytE-based high-density surface display system in Bacillus subtilis. Microb Biotechnol, 2008. 1(2): p. 177-90.
</P><P>2.	Kobayashi, G., et al., Accumulation of a recombinant Aspergillus oryzae lipase artificially localized on the Bacillus subtilis cell surface. J Biosci Bioeng, 2000. 90(4): p. 422-5.
</P><P>3.	Tjalsma, H., et al., Signal peptide-dependent protein transport in Bacillus subtilis: a genome-based survey of the secretome. Microbiol Mol Biol Rev, 2000. 64(3): p. 515-47.
</P><P>4.	Smith, T.J., S.A. Blackman, and S.J. Foster, Autolysins of Bacillus subtilis: multiple enzymes with multiple functions. Microbiology, 2000. 146(2): p. 249-262.
</P><P>5.	Nguyen, H.D., T.T. Phan, and W. Schumann, Analysis and application of Bacillus subtilis sortases to anchor recombinant proteins on the cell wall. AMB Express, 2011. 1(1): p. 22.
</P><P>6.	Liew, P.X., C.L. Wang, and S.L. Wong, Functional characterization and localization of a Bacillus subtilis sortase and its substrate and use of this sortase system to covalently anchor a heterologous protein to the B. subtilis cell wall for surface display. J Bacteriol, 2012. 194(1): p. 161-75.
</P><P>7.	Coutte, L., et al., Role of Adhesin Release for Mucosal Colonization by a Bacterial Pathogen. The Journal of Experimental Medicine, 2003. 197(6): p. 735-742.
</P><P>8.	Barbu, E.M., et al., beta-Neurexin is a ligand for the Staphylococcus aureus MSCRAMM SdrC. PLoS Pathog, 2010. 6(1): p. e1000726.
</P><P>9.	Muchnik, L., et al., NADH oxidase functions as an adhesin in Streptococcus pneumoniae and elicits a protective immune response in mice. PLoS One, 2013. 8(4): p. e61128.
</P></ARTICLE></DIV><DIV><LABEL for="adhesion-design">Design</LABEL><ARTICLE class="ac-small">

For cloning and evaluation, the adhesion project was divided into its two main components ‘surface display’ and ‘pathogen binding’. BioBricks were generated for the pathogen-specific peptides as well as for the parts necessary for surface display.
<H3><SPAN class="mw-headline" id="Surface_Display_2">Surface Display</SPAN></H3><P>Constructs for non-covalent and covalent surface display were cloned similar to those described by Chen <I>et. al.</I>, 2008 [1], and Liew <I>et. al.</I>, 2012 [6], respectively.
</P><P>As depicted in figure 1A, cell-wall binding domains (CWB) of LytC or LytE (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351006" class="external text" rel="nofollow">BBa_K1351006</A> or <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351007" class="external text" rel="nofollow">BBa_K1351007</A>) should be translationally fused to a protein of interest for non-covalent anchoring of the latter to the cell wall. These CWB domains also included the native signal peptide (SP) of the respective proteins to direct export via the Sec pathway. For covalent anchoring, signal peptide (<A href="http://parts.igem.org/Part:BBa_K1351008" class="external text" rel="nofollow">BBa_K1351008</A>) and cell wall anchoring domain (CWAD, <A href="http://parts.igem.org/Part:BBa_K1351010" class="external text" rel="nofollow">BBa_K1351010</A>) of the sortase substrate YhcR and a 57 amino acid linker (<A href="http://parts.igem.org/Part:BBa_K1351009" class="external text" rel="nofollow">BBa_K1351009</A>) were assembled with the protein of interest as depicted in figure 1B to generate a YhcR-like fusion.
</P><P>To evaluate the functionality of the constructs, the latter should be cloned into the replicative <I>B. subtilis</I> vector pBS0K under the control of the inducible promoter P<SUB><I>spac</I></SUB>. Two different proteins should be for successful display: First, a superfolder GFP variant (sfGFP) was used for fluorescence microscopy to check if the constructs locate to the cell wall as shown in figure 2. This variant of GFP had been shown previously to fold in the periplasm of gram-negative bacteria [10] and could thus be expected to be fuctional also in the extracellular environment, in contrast to normal GFP that does not fold under reducing conditions. As controls for the sfGFP constructs, only the signal peptides without the respective cell wall binding or anchoring domains were fused to sfGFP, which should lead to secretion of the fusion into the growth medium.
</P><P>Secondly, fusions to alkaline phosphatase (PhoA) should be used to detect surface accessibility of the presented protein, as PhoA is only active when exported out of the cytoplasm [11]. Agar plates containing the PhoA substrate XP (5-bromo-4-chloro-3-indolyl-phosphate) indicate enzyme activity by turning blue where the substrate is hydrolyzed and further oxidized to an indigo dye [12]. PhoA was also fused to intracellular GFP as a negative control and to an extracellular domain of LiaI as a positive control. The PhoA-deficient <I>B. subtilis</I> strain MH3402 will be used for expression of all PhoA constructs.
</P><P>Unfortunately, the status of these constructs is still &quot;cloning&quot;.
</P><H3><SPAN class="mw-headline" id="Pathogen_binding">Pathogen binding</SPAN></H3><P>Binding of BaKillus to pathogens in general, and for this project to <I>S. aureus</I> and <I>S. pneumoniae</I> in particular, should be mediated by specific peptides that had been identified via phage display. Unfortunately, evaluation of the <I>S. aureus</I>-specific peptide Nrx1b (<A href="http://parts.igem.org/Part:BBa_K1351000" class="external text" rel="nofollow">BBa_K1351000</A>) was not possible due to the lack of an S2 laboratory, as the Nrx1b-binding partner SdrC is a virulence factor of <I>S. aureus</I>. 
</P><P>Evaluation of peptides binding to <I>S. pneumoniae</I> was done by a biochemical pull-down assay with their binding partner NADH oxidase (NOX). As shown in figure 3, NOX was bound to streptavidin-coupled beads via an N-terminal fused Strep-Tag II, for which the pASK-IBA expression vector system was used. <I>S. pneumoniae</I>-specific peptides C4P, CSP and L5P (<A href="http://parts.igem.org/Part:BBa_K1351001" class="external text" rel="nofollow">BBa_K1351001</A>, <A href="http://parts.igem.org/Part:BBa_K1351002" class="external text" rel="nofollow">BBa_K1351002</A> and <A href="http://parts.igem.org/Part:BBa_K1351003" class="external text" rel="nofollow">BBa_K1351003</A>, respectively) were fused C-terminally to a maltose-binding protein (MBP) and a 6xHis-tag into the pKLD66 vector. This was achieved by digestion of the BioBricks and pKLD66 with NotI and HindIII and ligation of the products. The MBP prevents degradation of the short peptides by proteases and was used for detection of the peptide fusions on an SDS gel. To increase specificity of the detection, the 6xHis-tag was used for Western Blot analysis of the pulled down proteins. Additionally, <I>S. pneumoniae</I> itself was used as cellular beads for a pull down of the same peptide-MBP-His constructs, again followed by analysis with SDS gel and Western Blot.
</P><H3><SPAN class="mw-headline" id="Sources_3">Sources</SPAN></H3><P>1.	Chen, C.L., et al., Development of a LytE-based high-density surface display system in Bacillus subtilis. Microb Biotechnol, 2008. 1(2): p. 177-90.
</P><P>6.	Liew, P.X., C.L. Wang, and S.L. Wong, Functional characterization and localization of a Bacillus subtilis sortase and its substrate and use of this sortase system to covalently anchor a heterologous protein to the B. subtilis cell wall for surface display. J Bacteriol, 2012. 194(1): p. 161-75.
</P><P>11.	Manoil, C. and J. Beckwith, A genetic approach to analyzing membrane protein topology. Science, 1986. 233(4771): p. 1403-8.
</P><P>12.	Brickman, E. and J. Beckwith, Analysis of the regulation of Escherichia coli alkaline phosphatase synthesis using deletions and φ80 transducing phages. Journal of Molecular Biology, 1975. 96(2): p. 307-316.
</P></ARTICLE></DIV><DIV><LABEL for="adhesion-results">Results</LABEL><ARTICLE class="ac-small"><H3><SPAN class="mw-headline" id="Results"> Results </SPAN></H3><P><B>Overexpression of tagged NOX and peptides</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 1:SDS-PAGE for the overexpression of NOX  compared to the empty vector.</B> Different fractions of two clones (C1, C2) carrying the NOX-insert were compared to a clone carrying the empty vector. Cultures were lysed and centrifuged to separate the supernatant (SN) from the pellet (P). Samples of the cells were taken at different time points (PI = pre-induction, AI = 2h after induction). 20 μl of fractions were loaded.</DIV></DIV></DIV></DIV><P>The protein composition of the two clones show no difference in contrast to the strain carrying the empty vector. A band at around 55 kDa would have been expected for NOX. Thus, the expression of NOX was not successful. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 2:SDS-PAGE of induced (I) and uninduced (UI) strains carrying the peptides C4P, CSP or A5P fused to MBP.</B> Strains carrying the plasmids coding for the His-MBP tagged peptides were induced, lysed and centrifuged to separate the supernatant (SN) from the pellet (P). Samples were taken at different time points (PI = pre-induction, AI= 2h after induction) and uninduced strains serve as an control for expression. The lanes were loaded with 15μl.</DIV></DIV></DIV></DIV><P>The overexpression of MBP-tagged peptides was successful. The supernatant fractions showed a thick band at approximately 40kDa, corresponding to the size of MBP, which was not present in the uninduced samples. A smaller amount of the protein was also present in the lane of the pellet, but the major amount is cytosolic. Since the overexpression of NOX did not work, the planned pull-down of NOX with beads could not be performed. Instead the binding of peptides was tested by applying the His-MBP tagged peptides to <I>S. pneumoniae</I> cells.
</P><P><B>Adhesion of peptides to S.pneumoniae – Pulldown with cells</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 3:‎SDS-PAGE of the lysates and the fractions of the incubated S. pneumoniae cells.</B> E. coli cells carrying the plasmids coding for C4P, CSP and A5P were lysed. <I>S. pneumoniae</I> cells were resuspended in the lysate and incubated for 30 minutes, followed by pelleting to separate the supernatant (SP) from the cell pellet (P).  3 μl of sample were applied per lane.</DIV></DIV></DIV></DIV><P>The gel of the SDS-PAGE (Fig. 3) shows strong bands for the lysates at 40kDa, indicating a high amount of the His-MBP tagged peptides. The empty vector shows a relatively low expression of His-MBP.  The pellet fractions of the <I>S. pneumoniae</I> cells incubated with the tagged C4P and CSP peptides show light bands at the same height indicating adhesion of the peptides to the cells (Fig. 4, white arrows). Although the cells incubated in buffer alone also show a band at the same height indicating a native protein of that size (Fig. 3, black arrows) the amount of protein is clearly less. To determine whether the band represents the His-MBP tagged peptides, a western blot was performed (Figure 4).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 4: Western-Blots of the lysates and the fractions of the incubated <I>S. pneumoniae</I> cells.</B> Duplicates of the gel from Figure 4 (A) and another gel loaded with 10 μl/ lane (B) were processed by western blot with Anti-His antibodies.</DIV></DIV></DIV></DIV><P>The blot shows a clear signal for the applied lysates and all the pellet fractions of the cells incubated with the peptides, indicating their adhesion to <I>S.pneumoniae</I>. The band for the His-MBP of empty vector is missing (or very weak), which might be due to the lower expression, but one can clearly detect the bands corresponding to the His-MBP in the SDS-PAGE (Fig. 4). Our results strongly indicate that the previously described adhesion of the peptides to <I>S. pneumoniae</I> works. Thus, our BioBricks <A href="http://parts.igem.org/Part:BBa_K1351001" class="external text" rel="nofollow">BBa_K1351001</A>, <A href="http://parts.igem.org/Part:BBa_K1351002" class="external text" rel="nofollow">BBa_K1351002</A> and <A href="http://parts.igem.org/Part:BBa_K1351003" class="external text" rel="nofollow">BBa_K1351003</A> should serve as a tool for adhesion to <I>S. pneumoniae</I>.

            </P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION id="killing" class="scroll-spy jumptarget" data-scroll="killing"><DIV><H2><SPAN class="mw-headline" id="Killing"> Killing </SPAN></H2><P>A key aspect of our project is the killing module, which enables BaKillus to dispatch pathogens. Therefore, we implement a variety of killing strategies into our chassis. 
</P><P>One of our killing factors is subtilin, a lanthionine-containing antimicrobial peptide. Just like other lantibiotics, it is active against a wide range of Gram-positive bacteria by inhibiting the cell wall synthesis and the formation of membrane pores. The latter leads to the depolarisation of the membrane and thus, to rapid cell death. We decided to use subtilin, as it is natively produced by <I>Bacillus subtilis</I> ATCC6633, a close relative to our chassis. additionally lantibiotics are considered to be an alternative to classical antibiotics (to which many bacteria already developed resistance).
</P><P>Another killing factor is lysostaphin, a metallo-endopeptidase, which lysates the cell wall of some <I>Staphylococcus</I> species. Since this enzyme is highly specific, and <I>S. aureus</I> is especially sensitive towards lysostaphin, this is an ideal killing strategy for our BaKillus. In addition, lysostaphin shows no signs of toxicity and a very low potential for allergic reactions in first tests on animals and humans.
</P><P>Since pathogens are often protected by biofilms and thus not very sensitive towards killing agents, dispersin was integrated as an auxiliary device. This glycosaminhydrolase destroys biofilms of many gram-positive and gram-negative bacteria, including <I>S. aureus</I>, making them more susceptible for antibiotics and other pathogen defeating agents.
</P><P>Both, <A href="http://parts.igem.org/Part:BBa_K802000" class="external text" rel="nofollow">lysostaphin</A> and <A href="http://parts.igem.org/Part:BBa_K802001" class="external text" rel="nofollow">dispersin</A> were developed as BioBricks and tested in <I>B. subtilis</I> by the iGEM Team <A href="https://2012.igem.org/Team:Lyon-INSA" class="external text" rel="nofollow">iGEM12_Lyon_INSA</A> in 2012. We aim to evaluate the effectiveness of these two <I>S. aureus</I> defeating agents.
</P><DIV><LABEL for="killing-bground">Background</LABEL><ARTICLE class="ac-small"><H2><SPAN class="mw-headline" id="Subtilin"> Subtilin </SPAN></H2><P>Subtilin is a small antimicrobial peptide (AMP) which belongs to the class of lanthionine-containing antibiotics (lantibiotics). These are heat-stable, ribosomally synthesized molecules with a molecular weight below 4 kDa. Their main characteristic is the high proportion of unusual amino acids, synthesized by post-translational side-chain modifications of precursor peptides. Most prominent examples are the polycyclic thioether amino acids, lanthionine and β–methyllanthionine and a number of dehydrated amino acids such as dehydroalanine (Dha) and dehydrobutyrine (Dhb).
Lantibiotics are promising candidates for future antimicrobials, as they inhibit the growth of many clinically relevant pathogens comprising even multidrug-resistant bacteria. Moreover, lantibiotics have only low tendency to generate resistance. Both features make them highly attractive for medical applications. <A href="http://www.unboundmedicine.com/medline/citation/24210177/Lantibiotics:_promising_candidates_for_future_applications_in_health_care_" class="external text" rel="nofollow">[1</A>] <A href="http://www.ncbi.nlm.nih.gov/pubmed/16205711" class="external text" rel="nofollow">[2</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:623px;"><DIV class="thumbcaption">Fig. 1. Schematic representation of the structure of the lantibiotic subtilin. Dha represents a dehydroalanine residue, Dhb represents a dehydrobutirine residue, and Abu represents a dehydrobutirine residue that has formed a thioether β-methyl-lanthionine bridge with a cysteine residue. <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Quorum+sensing+control+of+lantibiotic+production%3B+nisin+and+subtilin+autoregulate+their+own+biosynthesis" class="external text" rel="nofollow">[3</A> ]</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="The_spaBTCSIFEGRK_gene_cluster">The <I>spaBTCSIFEGRK</I> gene cluster</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:733px;"><DIV class="thumbcaption">Fig. 2. Organisation of the subtilin biosynthetic gene cluster.  <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Quorum+sensing+control+of+lantibiotic+production%3B+nisin+and+subtilin+autoregulate+their+own+biosynthesis" class="external text" rel="nofollow">[3</A> ]</DIV></DIV></DIV></DIV><P>As true for many other lantibiotics, the genes for the subtilin biosynthesis are clustered. For subtilin, there is even a reasonable arrangement within the cluster. The genes <I>spaBTC</I> play a role for the posttranslational modifications and the transport of subtilin. They are regulated by their own promotor P<SUB><I>spaB</I></SUB> and polycistronically transcribed. <I>spaS</I> encodes the propeptide, which is derived from a short monocistronic mRNA. The next transcription entity, <I>spaIFEG</I>, is coding for the immunity, whereas <I>spaRK</I> encodes a two-component-system that plays an important role for the regulation. What is missing in comparison to other lantibiotic gene clusters is a specific protease (encoded by <I>lanP</I>) that cleaves off the leader sequence. In <I>Bacillus subtilis</I>, this task is accomplished by unspecific extracellular proteases.
</P><P><I>Table 1: The role of the different proteins of the <I>spa</I>-locus</I></P><TABLE class="wikitable"><TBODY><TR><TH><B>Protein</B></TH><TH><B>Annotation</B></TH><TH><B>Function</B></TH></TR><TR><TH rowspan="3"><B>Modification</B></TH><TD>SpaB
</TD><TD>Hydratase
</TD><TD>Dehydration of selected serines and threonines
</TD></TR><TR><TD>SpaT
</TD><TD>NisT-type ABC-transporter
</TD><TD>Export of newly synthesized AMPs
</TD></TR><TR><TD>SpaC
</TD><TD>Cyclase
</TD><TD>Thioether ring formation
</TD></TR><TR><TH><B>Peptide</B></TH><TD>SpaS
</TD><TD>Antimicrobial peptide (AMP)
</TD><TD>Acts as antimicrobial agent
</TD></TR><TR><TH rowspan="2"><B>Immunity</B></TH><TD>SpaI
</TD><TD>Lipoprotein
</TD><TD>Providing self-protection by binding AMP
</TD></TR><TR><TD>SpaFEG
</TD><TD>LanFEG-type ABC-transporter
</TD><TD>Providing resistance by exporting AMPs from cytoplasmic membrane to culture supernatant
</TD></TR><TR><TH><B>Regulation</B></TH><TD>SpaRK
</TD><TD>Two-component system
</TD><TD>Histidine-kinase and response regulator (quorum sensing mechanism)
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Self-protection_from_subtilin">Self-protection from subtilin</SPAN></H3><P>Of course, the producer needs to be immune against the antimicrobial agent it produces. <I>B. subtilis</I> ATCC6633 protects itself against subtilin by two mechanisms that both act independently and confer some level of resistance, however full resistance is only achieved when both mechanisms are active. SpaI is a membrane-anchored lipoprotein. It is suggested that it binds the antimicrobial peptide and by this keeps it away from the membrane. SpaFEG is a LanFEG-like ABC-transporter that works by exporting subtilin from the cytoplasmic membrane to the culture supernatant. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15659659" class="external text" rel="nofollow">[4</A>] <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Alkhatib%2C+Z.%3B+Abts%2C+A.%3B+Mavaro%2C+A.%3B+Schmitt%2C+L.%3B+Smits%2C+S.+H.%3A+Lantibiotics%3A+how+do+producers+become+self-protected%3F+Journal+of+biotechnology+2012%2C+159%2C+145-54." class="external text" rel="nofollow">[5</A>]
</P><H3><SPAN class="mw-headline" id="Dual_regulation_of_subtilin_biosynthesis_and_immunity">Dual regulation of subtilin biosynthesis and immunity</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig. 3. Dual control of subtilin biosynthesis and immunity. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11972779" class="external text" rel="nofollow">[6</A> ]</DIV></DIV></DIV><P>Subtilin biosynthesis and immunity in <I>B. subtilis</I> ATCC6633 are subject to a dual control mechanism. 
To begin with, subtilin production is positively regulated by sigma factor H (SigH). SigH itself is repressed by the transition state regulator AbrB during exponential growth phase. Thus, subtilin production is highest at the transition to stationary phase. Moreover, when a threshold level of subtilin concentration is reached in the extracellular space, the peptide acts as a pheromone. Subtilin activates the two-component system SpaRK, consisting of a histidine kinase and a response regulator, which in turn induces expression of <I>spaBTC</I>, <I>spaS</I> and <I>spaIFEG</I>. This dual control mechanism allows the coordination of subtilin biosynthesis with the physiological state of the cell.
</P><H3><SPAN class="mw-headline" id="Mode_of_action">Mode of action</SPAN></H3><P>Lantibiotics are active against a wide range of Gram-positive bacteria including multidrug-resistant pathogens. Some lantibiotics have a dual mode of action, subtilin is proposed to be among them. Subtilin forms a complex with the cell wall precursor lipid II, whereby the pyrophosphate moiety may play a key role for target recognition. Thereby, the cell wall biosynthesis is inhibited. Furthermore, the complexes aggregate and form a pore in the bacterial membrane by using lipid II as a docking molecule. This leads to a depolarisation of the membrane potential, the efflux of cytoplasma and in turn to rapid cell death. Just like nisin, which is highly similar, subtilin is still active in the nanomolar range and thus serves excellently as alternative killing strategy against multidrug-resistant pathogens.  <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Molecular+Mechanism+of+Target+parisot" class="external text" rel="nofollow">[7</A>] <A href="http://www.researchgate.net/publication/7875075_Mode_of_action_of_lipid_II-targeting_lantibiotics" class="external text" rel="nofollow">[8</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:603px;"><DIV class="thumbcaption">Fig. 4. Proposed mechanism of pyrophosphate-mediated target engagement by the lanthionine antibiotic subtilin. Membrane breaches occur in a lipid II-mediated fashion and cell wall synthesis is impeded through engagement of pyrophosphate-containing intermediates into complexes.  <A href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Molecular+Mechanism+of+Target+parisot" class="external text" rel="nofollow">[7</A> ]</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Lysostaphin"> Lysostaphin </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:486px;"><DIV class="thumbcaption">targeted pentaglycinbridge in the cellwall of some Staphylococcus species [B1]</DIV></DIV></DIV><P>Lysostaphin is a murein-hydrolase, which is naturally produced by <I>Staphylococcus simulans</I> biovar <I>staphylolyticus</I> in order to defeat other, competing Staphylococcus species.  It cleaves specifically the pentaglycinbridge between the tetrapeptides of the peptidoglycan in the cell wall. In addition to killing the cells, lysostaphin also dissolves the biofilms of lysostaphin-sensitive bacteria. <A href="http://www.ncbi.nlm.nih.gov/pubmed/19149589" class="external text" rel="nofollow">[B1</A>] 
This targeted pentaglycinbridge is a feature of the cell wall of some Staphylococcus species, leading to the high specificity of lysostaphin against <I>S. carnosus</I>, <I>S. epidermidis</I> and <I>S. aureus</I>, which is particularly sensitive towards lysostaphin. [<A href="http://www.ncbi.nlm.nih.gov/pubmed/18607587" class="external text" rel="nofollow">B2</A>] 
</P><P>The immunity of <I>S. simulans</I> is accomplished by substitution of the glycine-residues in the cell wall by serine which is mediated by the immunity factor Lif <A href="http://www.ncbi.nlm.nih.gov/pubmed/1914958" class="external text" rel="nofollow">[B1</A>] .
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:449px;"><DIV class="thumbcaption">maturation of Lysostaphin [B1]</DIV></DIV></DIV><P>The pre-pro-lysostaphin consists of a signal peptide to mediate the export, 15 tandem repeats of 13 amino acids length and two protein domains: the peptidase-domain and the C-terminal targeting domain. The signal peptide is cleaved during the export, the tandem repeats are cleaved by an additionally secreted cysteine protease. This repeats are not necessary for correct protein folding or maturation of the protein, they just inhibit protein activity. Once cleaved off, the maturated protein is 4.5 fold more active. <A href="http://www.ncbi.nlm.nih.gov/pubmed/1914958" class="external text" rel="nofollow">[B1</A>] 
</P><P>Because of its promising attributes, the utilization of lysostaphin as a coating for implants <A href="http://www.ncbi.nlm.nih.gov/pubmed/21709102" class="external text" rel="nofollow">[B3</A> ] and treatment of <I>S. aureus</I> in the nasal area via cream <A href="http://www.ncbi.nlm.nih.gov/pubmed/12709327" class="external text" rel="nofollow">[B4</A>] is currently in the focus of medical research.
</P><H2><SPAN class="mw-headline" id="Dispersin"> Dispersin </SPAN></H2><P>DispersinB (DspB) is a biofilm-dissolving enzyme, specifically a 1,6-beta-N-acetyl-glucosaminidase. It is produced by <I>Aggregatibacter actinomycetemcomitans</I>, a pathogen which causes aggressive forms of Parodontitis. <A href="http://www.ncbi.nlm.nih.gov/pubmed/23103508" class="external text" rel="nofollow">[B5</A>] In its natural habitat,<I> A. actinomycetecombitans</I> uses this enzyme to release single cells from the biofilm in order to colonize new habitats.  <A href="http://www.ncbi.nlm.nih.gov/pubmed/15878175" class="external text" rel="nofollow">[B6</A>] DspB is specific towards its substrate, PNAG, (PGA in E. coli), which is an important component of the extracellular matrix and which can make up to 90% of the dry weight of a biofilm. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15576769" class="external text" rel="nofollow">[B7</A>] The process of dissolving biofilms is accomplished by cleaving monosaccharide-residues from the polysaccharide, beginning at the non-reducing end of the PNAG. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15878175" class="external text" rel="nofollow">[B6</A>] This glycosamin-hydrolase has a typical TIM-barrel-structure, consisting of 8α- and 8β-barrels, with a large cavity in the middle, which is considered to be the substrate-binding pocket. <A href="http://www.ncbi.nlm.nih.gov/pubmed/23103508" class="external text" rel="nofollow">[B5</A>]
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Proposed mechanism of reaction <A href="http://www.ncbi.nlm.nih.gov/pubmed/17949435" class="external text" rel="nofollow">[B8</A></DIV></DIV></DIV><P>The currently proposed mechanism of reaction suggests a substrate assisted nucleophilic attack (see chart below) <A href="http://www.ncbi.nlm.nih.gov/pubmed/17949435" class="external text" rel="nofollow">[B8</A>] 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Barrel structure of DspB <A href="http://www.ncbi.nlm.nih.gov/pubmed/15878175" class="external text" rel="nofollow">[B6</A></DIV></DIV></DIV><P>DspB is active against many bacteria, both gram-positive and gram-negative, and shows some very promising features for its use in medical devices. It can prevent the growth of biofilms in coated catheters if used in combination with triclosan, and can be used to clear medical devices of biofilms. <A href="http://www.ncbi.nlm.nih.gov/pubmed/19447791" class="external text" rel="nofollow">[B9</A>]
</P><H3><SPAN class="mw-headline" id="Sources:"> Sources: </SPAN></H3><P>[1] Dischinger, J.; Basi Chipalu, S.; Bierbaum, G.: Lantibiotics: promising candidates for future applications in health care. International journal of medical microbiology : IJMM 2014, 304, 51-62.
</P><P>[2] Cotter, P. D.; Hill, C.; Ross, R. P.: Bacteriocins: developing innate immunity for food. Nature reviews. Microbiology 2005, 3, 777-88.
</P><P>[3] Kleerebezem, M.: Quorum sensing control of lantibiotic production; nisin and subtilin autoregulate their own biosynthesis. Peptides 2004, 25, 1405-14.
</P><P>[4] Stein, T.; Heinzmann, S.; Dusterhus, S.; Borchert, S.; Entian, K. D.: Expression and functional analysis of the subtilin immunity genes spaIFEG in the subtilin-sensitive host Bacillus subtilis MO1099. Journal of bacteriology 2005, 187, 822-8.
</P><P>[5] Alkhatib, Z.; Abts, A.; Mavaro, A.; Schmitt, L.; Smits, S. H.: Lantibiotics: how do producers become self-protected? Journal of biotechnology 2012, 159, 145-54.
</P><P>[6] Stein, T.; Borchert, S.; Kiesau, P.; Heinzmann, S.; Kloss, S.; Klein, C.; Helfrich, M.; Entian, K. D.: Dual control of subtilin biosynthesis and immunity in Bacillus subtilis. Molecular microbiology 2002, 44, 403-16.
</P><P>[7] Parisot, J.; Carey, S.; Breukink, E.; Chan, W. C.; Narbad, A.; Bonev, B.: Molecular mechanism of target recognition by subtilin, a class I lanthionine antibiotic. Antimicrobial agents and chemotherapy 2008, 52, 612-8.
</P><P>[8] Bauer, R.; Dicks, L. M.: Mode of action of lipid II-targeting lantibiotics. International journal of food microbiology 2005, 101, 201-16.
</P><P>[B1] M.C.F. Bastos, H. Ceotto1, M.L.V. Coelho and J.S. Nascimento: Staphylococcal Antimicrobial Peptides: Relevant Properties and Potential Biotechnological Applications in Current Pharmaceutical Biotechnology (2009) S. 38-61 
</P><P>[B2] Jaspal K. Kumar: Lysostaphin: an antistaphylococcal agent in Appl Microbiol Biotechnol (2008) S.555-561 )
</P><P>[B3] Rohan Satishkumar, Sriram Sankar, Yuliya Yurko, Amy Lincourt, John Shipp, B. Todd Heniford and Alexey Vertegel: Evaluation of the Antimicrobial Activity of Lysostaphin-Coated Hernia Repair Meshes in Antimicrobial Agents and Chemotherapy (2011) 
</P><P>[B4] John F. Kokai-Kun, Scott M. Walsh, Tanya Chanturiya, and James J. Mond: Lysostaphin Cream Eradicates Staphylococcus aureus Nasal Colonization in a Cotton Rat Model in Antimicrobial Agents and Chemotherapy, 47 (2003)
</P><P>[B5] Erika Fazekas, Lili Kandra, Gyongyi Gyemant: Model for b-1,6-N-acetylglucosamine oligomer hydrolysis catalysed by DispersinB, a biofilm degrading enzyme, in Carbohydrate Research 363 (2012), S. 7-13 
</P><P>[B6] N. RamasubbuL. M. Thomas, C. Ragunath and J. B. Kaplan: Structural Analysis of Dispersin B, a Biofilm-releasing Glycoside Hydrolase from the Periodontopathogen Actinobacillus actinomycetemcomitans, in Journal of Molecular Biology 340 (2005) S. 475-486 
</P><P>[B7] Jeffrey B. Kaplan, Kabilan Velliyagounder, Chandran Ragunath, Holger Rohde,
Dietrich Mack, Johannes K.-M. Knobloch, and Narayanan Ramasubbu: Genes Involved in the Synthesis and Degradation of Matrix Polysaccharide in Actinobacillus actinomycetemcomitans and Actinobacillus pleuropneumoniae Biofilms, in JOURNAL OF BACTERIOLOGY, 186 (2004) S.8213-8220 
</P><P>[B8] Suba G. A. Manuel, Chandran Ragunath, Hameetha B. R. Sait, Era A. Izano, Jeffrey B. Kaplan and Narayanan Ramasubbu: Role of active-site residues of dispersin B, a biofilm-releasing b-hexosaminidase from a periodontal pathogen, in substrate hydrolysis, in The FEBS journal (2007) 
</P><P>[B9] Darouiche RO, Mansouri MD, Gawande PV, et al. Antimicrobial and antibiofilm efficacy of triclosan and DispersinB combination. J Antimicrob Chemother 2009; 64: 88-93 
</P></ARTICLE></DIV><DIV><LABEL for="killing-design">Design</LABEL><ARTICLE class="ac-small"><H2><SPAN class="mw-headline" id="Subtilin_2">Subtilin</SPAN></H2><H3><SPAN class="mw-headline" id="Analysis_of_subtilin_impact_on_pathogens">Analysis of subtilin impact on pathogens</SPAN></H3><P>To prove that subtilin has an inhibiting impact on pathogens and thus, is convenient as a BaKillus killing strategy, different growth and killing assays were performed. These experiments included spot-on-lawn-assays, growth and killing curves and luminescence assays and were conducted for <I>Streptococcus pneumoniae</I> and with the kind support of the iGEM Team Groningen also with <I>Staphylococcus aureus</I> sp., including MRSA. Protocols to the specific assays can be found <A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Protocols" class="external text" rel="nofollow">here</A>. 
</P><H3><SPAN class="mw-headline" id="Heterologous_expression">Heterologous expression</SPAN></H3><P>As our working horse <I>Bacillus subtilis</I> 168 is closely related to the subtilin producer  <I>Bacillus subtilis</I> ATCC6633, one approach for subtilin production is the heterologous expression by transferring the complete <I>spaBTCSIFEGRK</I> gene cluster into <I>B. subtilis</I>. In this case, the native regulation of subtilin biosynthesis and immunity, the two-component system <I>spaRK</I>,  is maintained. Moreover, the transition state regulation should be similar for both organisms.
</P><H3><SPAN class="mw-headline" id="The_BioBrick_approach">The BioBrick approach</SPAN></H3><P>The SynBio approach of this subproject is the standardization of the subtilin gene cluster. Therefore, the gene cluster was subdivided into BioBricks according to their function and maintaining the transcription entities. However, in contrast to the classical heterologous expression, the BioBricks are standardized according to the RFC10 or the RFC25 standard, respectively. 
Moreover, gene expression and subtilin production will no longer be controlled by the native regulation system, but are rather in dependance on promotors, activated in a quorum sensing dependent matter.
</P><H3><SPAN class="mw-headline" id="Lyostaphin"> Lyostaphin </SPAN></H3><P>The BioBrick <A href="http://parts.igem.org/Part:BBa_K802000" class="external text" rel="nofollow">K802000</A>, built by the iGEM Team <A href="https://2012.igem.org/Team:Lyon-INSA" class="external text" rel="nofollow">iGEM12_Lyon_INSA</A>, was kindly sent by the team <A href="https://2014.igem.org/Team:INSA-Lyon" class="external text" rel="nofollow">Insa-Lyon</A>. Via PCR-amplification, the actual lyosostaphin-gene was amplified, adding the restriction sites required for the RFC25.
</P><P>The coding sequence was cloned together with the promoter P<SUB><I>xyl</I></SUB> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351039" class="external text" rel="nofollow">BBa_K1351039</A>) into the plasmid pBS1C (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823023" class="external text" rel="nofollow">BBa_K823023</A>). After transformation into <I>B. subtilis</I>, the iGEM-team <A href="https://2014.igem.org/Team:Groningen" class="external text" rel="nofollow">Groningen</A> will evaluate the effectiveness of lysostaphin, produced by <I>B. subtilis</I> against <I>S. aureus</I> via spot on lawn test. This test will be conducted with living cells as well as with cell lysate in order to evaluate whether the lysostaphin is secreted into the supernatent by <I>B.subtilis</I> or not.
</P><H3><SPAN class="mw-headline" id="Dispersin_2"> Dispersin </SPAN></H3><P>The BioBrick <A href="http://parts.igem.org/Part:BBa_K802001" class="external text" rel="nofollow">K802001</A> built by the iGEM Team <A href="https://2012.igem.org/Team:Lyon-INSA" class="external text" rel="nofollow">iGEM12_Lyon_INSA</A>, was kindly sent by the Team <A href="https://2014.igem.org/Team:INSA-Lyon" class="external text" rel="nofollow">Insa-Lyon</A>. The original DspB contains an AgeI restriction site, which was deleted via site directed mutagenesis by overlaping extension PCR in order to make the gene compatible with the Freiburg standard <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A>.
</P><P>Overhangs with <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible restriction sites where added during this PCR and the PCR-product was cloned into the pCRII-Topo-vector.
</P><P>After sequence confirmation, the DspB will be cloned into pSB1C3 in order to be sent in as a BioBrick. The His-Tag will be added added C- and N-terminally, and a copy of each of the His-tagged-constructs, and one copy without His-tag will be cloned together with the promoter P<SUB><I>xyl</I></SUB> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1351039" class="external text" rel="nofollow">BBa_K1351039</A>) into the vector pBS1C (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823023" class="external text" rel="nofollow">BBa_K823023</A> ). The transformation into <I> B. subtilis</I> will follow.
</P><P>In order to test the biofilm dissolving abilities of this recombinant <I>B. subtilis</I>, biofilms of <I>E. coli Nissle 1917</I>, an excellent biofilm forming bacteria with GRAS-status, will be grown in 96 well plates. The biofilms will be dyed with crystalviolett and living <I>B. subtilis</I> cells as well as cell lysate will be transferred into the wells.
If biofilm-destruction takes places, the crystalviolett will vanish by carefully washing the plates.
Additionally, an expression control will be conducted via Western Blot.
</P></ARTICLE></DIV><DIV><LABEL for="killing-results">Results</LABEL><ARTICLE class="ac-small"><H2><SPAN class="mw-headline" id="Subtilin_3">Subtilin</SPAN></H2><H3><SPAN class="mw-headline" id="Subtilin_impact_on_Streptococcus_pneumoniae">Subtilin impact on <I>Streptococcus pneumoniae</I></SPAN></H3><P>First of all, a growth curve for <I>S. pneumoniae</I> was recorded to learn about the growth behavior of this organism. It was measured in the plate reader in 100 µl THB broth at 37 °C without shaking and measurement was taken every 10 min. The mean value of 10 measurements were determined and plotted.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 1. Growth curve of <I>S. pneumoniae</I> measured in the plate reader in 100 µl THB broth.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig. 2. The spot-on-lawn assay shows a clear halo around the <I>B. subtilis</I> ATCC6633 spot. No clearing zone is visible around <I>B. subtilis</I> W168.</DIV></DIV></DIV></DIV><P>As expected, <I>S. pneumoniae</I> shows rapid growth during its exponential growth phase. It reaches its maximal OD<SUB>600</SUB> just under 5 hours. In contrast to other bacteria, for <I>S. pneumoniae</I>, there is no plateau during stationary phase, as autolysis of the cells starts and the OD<SUB>600</SUB> decreases again. 
</P><P>
As <I>S. pneumoniae</I> is a target of the BaKillus killing strategies, a spot-on-lawn-assay should reveal the inhibitory impact of subtilin. Therefore, spots of the <I>B. subtilis</I> W168 wild type and the subtilin-producer <I>B. subtilis</I> ATCC6633, respectively, were spotted on a  <I>S. pneumoniae</I> lawn. As depicted in figure 2 and 3, the subtilin producer significantly inhibits the growth of <I>S. pneumoniae</I>. In contrast,  <I>B. subtilis</I> W168 wild type does not show inhibitory impact. The experiment was performed three times and the mean value and the standard deviation are illustrated in the graph below.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig. 3. Analysis of subtilin impact on <I>S. pneumoniae</I>.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Subtilin_impact_on_Staphylococcus_aureus">Subtilin impact on <I>Staphylococcus aureus</I></SPAN></H3>
Moreover, the impact of subtilin on <I>Staphylococcus aureus</I> was tested in collaboration with iGEM Team Groningen, as they have access to S2 laboratories, in which <I>S. aureus</I> and even multidrug-resitant species are handled. A member of Team Groningen kindly performed spot-on-lawn assays with <I>B. subtilis</I> W168 wild type and the subtilin-producer <I>B. subtilis</I> ATCC6633 on different <I>S. aureus</I> species. The tested strains were CAL, NWZ (both MRSA, obtained at the UMCG hospital) and CECT240 (<I>Staphylococcus aureus</I> subsp. <I>aureus</I>, Rosenbach 1884). Strains were plated out in soft agar and spots from overnight cultures of <I>Bacillus subtilis</I> grown in either complex medium (LB) or defined medium (SMM) were added on top. <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig. 4. Spot-on-lawn assay with the subtilin producer and on three different <I>S. aureus</I> lawns</DIV></DIV></DIV></DIV><P>After incubation for 24 h at 37 °C, halos could be detected for all spots of the subtilin producer. The exact results are shown in the chart below. However, the spots of the <I>B. subtilis</I> W168 culture did not inhibit the growth of any <I>S. aureus</I> strains (no pictures provided).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Fig. 5 Bar chart of the <I>S. aureus</I> spot-on-lawn assay.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Heterologous_expression_2">Heterologous expression</SPAN></H3><P>The aim of this approach was to transfer the complete subtilin gene cluster into our working horse <I>Bacillus subtilis</I> W168 by maintaining the native regulation mainly mediated by the <I>spaRK</I> two-component system. So far, the 12 kb-locus is assumed to be transferred into our <I>Bacillus</I> wild type but only half of the locus is sequenced. Unfortunately, no subtilin production could be ascertained. This could be due to an incorrect sequence or the fact that the native regulation of <I>B. subtilis</I> ATCC6633 does not work in our wild type. Our next steps are the full sequencing of the putative <I>spa</I>-gene cluster and an attempt to transfer it into <I>B.subtilis</I> W168 Δ<I>abrB</I>, a mutant that lacks the repressing transition state regulator.
</P><H3><SPAN class="mw-headline" id="The_BioBrick_approach_2">The BioBrick approach</SPAN></H3><P>This approach aims to &quot;BioBrick&quot; and by this standardize the subtilin gene cluster according to its transcriptional entities. In contrast to the heterologous expression, the BioBricks will be under quorum sensing dependent regulation.
</P><P><B>Modification &amp; Transport</B></P><P>The BioBrick <A href="http://parts.igem.org/Part:BBa_K1351041" class="external text" rel="nofollow">BBa K1351041</A>, encoding SpaBTC, is mainly responsible for the modification and the export of the subtilin propeptide. After the removal of 6 BioBrick restriction sites within the three genes, it is now in the RFC10 BioBrick standard.
This BioBrick was accomplished just before WikiFreeze, so that no further evaluation was possible so far.
</P><P><B>Antimicrobial peptide</B></P><P><A href="http://parts.igem.org/Part:BBa_K1351012" class="external text" rel="nofollow">BBa K1351012</A> and <A href="http://parts.igem.org/Part:BBa_K1351013" class="external text" rel="nofollow">BBa K1351013</A> are coding for SpaS, the subtilin pro-peptide. Their main difference are their overhangs, one being in RFC25, which allows the tagging or translational fusion of <A href="http://parts.igem.org/Part:BBa_K1351012" class="external text" rel="nofollow">BBa K1351012</A>, whereas the other can be used according to the RFC10 BioBrick standard. Unfortunately non of them could be evaluated so far, as the development of the immunity BioBrick as well as of the modification and export BioBrick were required beforehand.
</P><P><B>Immunity</B></P><P>The immunity against subtilin (<A href="http://parts.igem.org/Part:BBa_K1351014" class="external text" rel="nofollow">BBa K1351014</A>), was - just like the spaS BioBricks - submitted to the iGEM parts registry. Self-protection is mediated by SpaIFEG, a membrane-anchored lipoprotein and an ABC-transporter.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig. 6. Supernatant-in-lawn assay shows smaller halo diameters for <I>B. subtilis</I> with BioBRick-acquired immunity.</DIV></DIV></DIV></DIV><P>The part was evaluated with a variation of an spot-on-lawn assay. In this case, holes were cut out of agar plates on which either <I>B. subtilis</I> W168 wild type or <I>B. subtilis</I> W168 <I>amyE</I>::P<SUB><I>xyl</I></SUB>_<I>spaIFEG</I> were plated. The holes were then filled with different amounts of sterile supernatant from overnight cultures of W168 or ATCC6633, respectively. The xylose-containing plates were incubated at 37 °C for 24 h and resulting halo diameters were measured. The supernatant derived from the negative control (W168) did not cause any growth inhibition to neither the wild type, nor to the strain who carries the immunity BioBrick (Figure 6). The subtilin-containing supernatant from <I>B. subtilis</I> ATCC6633 however, was able to inhibit the growth of both bacterial lawns. For the lowest amounts (32 µl), no significant difference between the halo diameters could be determined. The larger the amount of supernatant, and presumably, the higher the amount of subtilin in there, the bigger the zones of inhibition. For the largest volume (128 µl), a two-sided T-test could determine the p-value below 0.1 (p=0,007762603) and by this prove the functionality of this BioBrick. This is a major step for this project, as the immunity is the base for any strain which produces antimicrobial peptides. Now further steps can be taken and the remaining BioBricks can be evaluated.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig. 7 Evaluation of the <I>spaIFEG</I> BioBrick</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Lysostaphin_2">Lysostaphin</SPAN></H2><P>A first sequencing, prior to the PCR-Amplification revealed an error in the DNA-sequence provided by the Registry. The actual sequence of the lysostaphin of the BioBrick <A href="http://parts.igem.org/Part:BBa_K802000" class="external text" rel="nofollow">BBa_K802000</A> is codon-optimized for <I>B. subtilis</I>.
The correct sequence was posted as a user review in the experience section of the Registry.
After cloning the lysostaphin into pBS1C with the promoter Pxyl, the construct was verified by sequencing and transformed into B. subtilis.
</P><P>A culture in DSM was incubated at 37°C and 220 rpm and 5µl of this culture where spotted on a sterile filter paper and sent to the iGEM-team Groningen.
</P><H2><SPAN class="mw-headline" id="Dispersin_3"> Dispersin </SPAN></H2><P>The sample of the BioBrick K802001, which was originally sent by the Registry contained a wrong gene.
Fortunately, the sample sent by the iGEM-Team Insa_Lyon, was the matching plasmid to the given sequence. Thank you very much for providing us with this part!
The mutated gene was cloned into the pCRII-topo-vector and transformed into E. coli DH5α.
</P></ARTICLE></DIV></DIV></SECTION></DIV><SECTION id="suicide" class="scroll-spy jumptarget bg-color-2" data-scroll="suicide"><DIV><H2><SPAN class="mw-headline" id="Suicide_Switch"> Suicide Switch </SPAN></H2><P>Thanks to the numerous killing strategies listed above, BaKillus has specifically dispatched the targeted pathogens, thus its job is almost done. However, to ensure the temporary mode of action and to avoid uncontrollable spread, we implemented the so called “suicide switch“ leading to time-delayed cell death of BaKillus. 
</P><P>During stationary phase of a <I>Bacillus subtilis</I> culture it has been observed that some cells start producing a toxin which causes cell lysis of sister cells whereas the toxin-producer cells stay unaffected. <A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A>] This social and heterogenous behavior is called cannibalism and seems to be an even more promising basis for constructing the BaKillus “suicide switch“ since it has been shown that the toxin also kills a variety of Gram-positive bacteria, including MRSA-strains. <A href="http://www.pnas.org/content/107/37/16286.full" class="external text" rel="nofollow">[2</A>]
</P><P>So how does the suicide switch work?
As soon as BaKillus detects QS-molecules of <I>S. aureus</I> or <I>S. pneumoniae</I> the cannibalism toxin SDP is produced and secreted. This leads to the co-expression and co-regulation of a transmembrane protein responsible for the immunity to SDP by binding the extracellular toxin. Simultaneously to the SDP production, the expression of an alternative σ-factor initiates a delayed suicide process by activating either one or both of two types of RNA: an antisense RNA, complementary to the mRNA of the immunity and a small RNA fsrA which has a binding site in the added 5'-UTR of the immunity coding mRNA. As a result the translation of the immunity protein is inhibited and the immunity is lost with the decay of the existing immunity protein. Finally BaKillus is now sensitve to the SDP making cell lysis inevitable.
</P><P><A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A>] Gonzalez-Pastor, J. E., et al. (2003). &quot;Cannibalism by sporulating bacteria.&quot; Science 301(5632): 510-513.
</P><P><A href="http://www.pnas.org/content/107/37/16286.full" class="external text" rel="nofollow">[2</A>] Liu, W. T., et al. (2010). &quot;Imaging mass spectrometry of intraspecies metabolic exchange revealed the cannibalistic factors of Bacillus subtilis.&quot; Proc Natl Acad Sci U S A 
</P><DIV><LABEL for="suicide-bground">Background</LABEL><ARTICLE class="ac-small"><B>The <I>sdp</I>-System of <I>B. subtilis</I></B> consists of two operons: The <I>sdpABC</I> operon, coding for the production and secretion of the cannibalism toxin SDP and the <I>sdpRI</I> operon responsible for the regulation and production of the immunity protein SdpI (Fig. 1). 
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:623px;"><DIV class="thumbcaption">Fig. 1.  Gene organization for the <I>sdpABC</I><I>sdpRI</I> operons. The hairpin symbolizes thetranscriptional terminators. <A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A> ]</DIV></DIV></DIV></DIV><P>In vegetative cells, both operons are repressed by the unstable AbrB regulator. However, during early stages of sporulation AbrB itself is repressed by the master regulator of sporulation Spo0A, making <I>sdpABC</I> and <I>sdpRI</I> accessible for RNA polymerase. <A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A>]
</P><H3><SPAN class="mw-headline" id="The_sdpABC_Operon_.E2.80.93_Production_and_Secretion_of_the_Cannibalism_Toxin_SDP"> The <I>sdpABC</I> Operon – Production and Secretion of the Cannibalism Toxin SDP </SPAN></H3><P>The production of the Cannibalism Toxin SDP is a multi-step process.  The <I>sdpC</I> sequence encodes the Pro-SdpC1-203,,which is translated by the ribosome.It is a precursor peptide which needs to be processed by a signal peptidases and the two membrane proteins SdpA and SdpB to become functional. This active form of SDP is a 42-amino-acid antimicrobial peptide (AMP) containing a disulfide bond between two cysteine residues located at the N-terminus.(Fig. 2). <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697648/" class="external text" rel="nofollow">[2</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig. 2. SDP production requires multiple steps. In the cytosol, the full length SdpC (pro-SdpC1-203) is secreted via the Sec pathway. Following secretion, the signal peptidases SipS and SipT cleave the N-terminal signal peptide sequence of SdpC. Disulfide bond formation occurs independently of SdpAB. Finally, posttranslational cleavage of SdpC occurs via SdpAB to produce a 42-amino-acid SDP that will be secreted extracellulary as the active SDP peptide. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697648/" class="external text" rel="nofollow">[2</A> ]</DIV></DIV></DIV></DIV><P>SDP has been shown to be a very effective AMP against a variety of Gram-positive bacteria in the Phylum of the Firmicutes (Fig. 3). It rapidly collapses the proton motive force (PMF), thus inducing autolysis. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3839633/" class="external text" rel="nofollow">[3</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 3. SDP inhibition curves for pathogenic microbes. Relative growth of the strains named abovewith the presence of increasing concentrations of SDP is shown in the curve. As a negative control the gram-negative bacteria <I>K. pneumoniae</I> and <I>P. aeruginosa</I> are depicted, which are unaffected by the toxin SDP, as it specifically targets gram-positive bacteria. <I>B. subtilis</I> is a gram-positive bacteria, but expresses the immunity protein SdpI and is therefore relatively resistent to the toxin SDP. the <I>Stapylococcus</I> species (also MRSA) though are quite drastically reduced in the presence of the SDP. <A href="http://www.pnas.org/content/107/37/16286.full" class="external text" rel="nofollow">[4</A> ]</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="The_sdpIR_Operon_.E2.80.93_Production_and_Regulation_of_the_Immunity_Protein_SdpI"> The <I>sdpIR</I> Operon – Production and Regulation of the Immunity Protein SdpI </SPAN></H3><P>In the absence of SDP the gene coding for the immunity protein SdpI is repressed by the autorepressor SdpR. If extracellular SDP binds the transmembrane protein SdpI it causes a conformational change. As a consequence SdpR is sequestered at the membrane, forming a complex with SDP and SdpI. Since the repression by SdpR is relieved the RNA polymerase can bind the <I>sdpRI</I> promotor region and start transcription (Fig. 4). <A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:587px;"><DIV class="thumbcaption">Fig. 4. Model of how SDP induces expression of the <I>sdpRI</I> immunity operon. The signal SDP interacts with the immunity protein SdpI to induce a conformational change that allows SdpI to bind the repressor SdpR, which is sequestered at the membrane. <A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A> ]</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="ECF41:_An_alternative_sigma_factor_for_building_an_orthogonal_Delay_Switch"> ECF41: An alternative sigma factor for building an orthogonal Delay Switch </SPAN></H3><P>The alternative σ-factor <I>ecf41</I><SUB><I>bli aa 1-204</I></SUB>, which derives from <I>B. licheniformis</I> is a truncated version and binds to the target promoter P<SUB><I>ydfG</I></SUB>. It is an orthogonal σ-factor since there are no cross-reactions with other native promoters. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3426412/" class="external text" rel="nofollow">[5</A>] The introduction of this loop leads to a translational delay.
</P><H3><SPAN class="mw-headline" id="FsrA:_a_small_RNA_Represses_Expression_of_Succinate_Dehydrogenase"> FsrA: a small RNA Represses Expression of Succinate Dehydrogenase </SPAN></H3><P>Regulation of bacterial iron homeostasis is often controlled by the iron-sensing ferric uptake repressor (Fur). The <I>Bacillus subtilis</I> Fur protein acts as an iron-dependent repressor for siderophore biosynthesis and iron transport proteins. Furthermore it also coordinates an iron-sparing response that acts to repress the expression of iron-rich proteins (e.g. succinate dehydrogenase) when iron is limited. It has been shown that the downregulation of succinate dehydrogenase (SDH) is most likely caused by complementary binding of a small RNA, named fsrA, to the leader region of the sdhCAB mRNA, namely SdhC' (Fig. 5). In this project this principle is exploited to downregulate the translation of the immunity protein SdpI in a delayed response to the detection of the QS-molecules of <I>S. aureus</I> or <I>S. pneumoniae</I>. <A href="http://www.pnas.org/content/105/33/11927.full.pdf" class="external text" rel="nofollow">[6</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:735px;"><DIV class="thumbcaption">Fig. 5. Predicted pairing between fsrA and the 5’-leader region of sdhC, the first gene of the <I>sdhCAB</I> operon. <A href="http://www.pnas.org/content/105/33/11927.full.pdf" class="external text" rel="nofollow">[6</A> ]</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Sources_4"> Sources </SPAN></H3><P><A href="http://www.sciencemag.org/content/301/5632/510.full" class="external text" rel="nofollow">[1</A>] Gonzalez-Pastor, J. E. (2011). &quot;Cannibalism: a social behavior in sporulating Bacillus subtilis.&quot; FEMS Microbiol Rev 35(3): 415-424.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697648/" class="external text" rel="nofollow">[2</A>] Perez Morales, T. G., et al. (2013). &quot;Production of the cannibalism toxin SDP is a multistep process that requires SdpA and SdpB.&quot; J Bacteriol 195(14): 3244-3251.
</P><P><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3839633/" class="external text" rel="nofollow">[3</A> Lamsa, A., et al. (2012). &quot;The Bacillus subtilis cannibalism toxin SDP collapses the proton motive force and induces autolysis.&quot; Mol Microbiol 84(3): 486-500.
</P><P><A href="http://www.pnas.org/content/107/37/16286.full" class="external text" rel="nofollow">[4</A>] Liu, W. T., et al. (2010). &quot;Imaging mass spectrometry of intraspecies metabolic exchange revealed the cannibalistic factors of Bacillus subtilis.&quot; Proc Natl Acad Sci U S A 
</P><P><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3426412/" class="external text" rel="nofollow">[5</A>] Wecke, T., et al. (2012). &quot;Extracytoplasmic function sigma factors of the widely distributed group ECF41 contain a fused regulatory domain.&quot; Microbiologyopen 1(2): 194-213.
</P><P><A href="http://www.pnas.org/content/105/33/11927.full.pdf" class="external text" rel="nofollow">[6</A>] Gaballa, A., et al. (2008). &quot;The Bacillus subtilis iron-sparing response is mediated by a Fur-regulated small RNA and three small, basic proteins.&quot; Proc Natl Acad Sci U S A 105(33): 11927-11932.
</P></ARTICLE></DIV><DIV><LABEL for="suicide-design">Design</LABEL><ARTICLE class="ac-small"><P>The suicide switch module requires a high level of combinability regarding to its components. We therefore designed every  <A href="https://2014.igem.org/Team:LMU-Munich/Results/Registry" class="external text" rel="nofollow">new part</A> in RFC10 or RFC25 standard.
</P><H3><SPAN class="mw-headline" id="Production_of_the_toxin_SDP_and_the_immunity_protein_SdpI"> Production of the toxin SDP and the immunity protein SdpI </SPAN></H3><P>By the activation of the QS-dependent promoter P<SUB><I>QS</I></SUB>,  BaKillus produces the Cannibalism Toxin SDP (Fig. 1A). For evaluation purposes we cloned the <I>sdpABC</I> operon under the control of inducable xylose promoter P<SUB><I>xyl</I></SUB> (Fig. 1B) into a <I>∆sdpAB</I> and a <I>∆sdpC</I> mutant strain of <I>B. subtilis</I> W168. 
The construct was then tested by spot-on-lawn assays on <I>B. subtilis</I> W168 and <I>B. subtilis</I> W168 <I>∆sdpI</I> mutant lawns.
</P><P>To ensure a co-regulated production of the immunity protein SdpI we used the native promoter of the <I>sdpRI</I> operon P<SUB><I>sdpRI</I></SUB> which is accesible for RNA polymerase in the presence of extracellular SDP. Between the promoter and the immunity sequence <I>sdpI</I> we inserted the binding site of fsrA namely <I>sdhC’</I> (Fig. 2A). 
By expressing the SdpI immunity protein under the control of the P<SUB>xyl</SUB> promoter (Fig. 2B) we were able to evaluate the functionality of SdpI by growth curves measurements.
</P><H3><SPAN class="mw-headline" id="Delayed_loss_of_immunity"> Delayed loss of immunity </SPAN></H3><P>A time delayed loss of the immunity protein SdpI in BaKillus can be achieved by interposing the alternative σ-factor ECF41<SUB><I>bli aa 1-204</I></SUB><A href="http://parts.igem.org/Part:BBa_K823043" class="external text" rel="nofollow">BBa_K823043</A> which is also under the control of the quorum sensing promoter P<SUB><I>QS</I></SUB> (Fig.3A). When expressed, ECF41<SUB><I>bli aa 1-204</I></SUB> binds to the target promoter P<SUB><I>ydfG</I></SUB><A href="http://parts.igem.org/Part:BBa_K823041" class="external text" rel="nofollow">BBa_K823041</A> leading either to the expression of fsrA, a small RNA, or transcription of sdpI-antisense, a RNA, a complementary RNA to the sdpI mRNA (Fig. 3B). 
To veri- and quantify the time delay due to the interposition of Ecf41<SUB><I>bli aa 1-204</I></SUB> we replaced the fsrA coding sequence with the lux-cassette <I>luxABCDE</I> allowing lumi assay measurements (Fig. 3C). 
</P><P>For the loss of immunity we implemented two strategies, both causing translational inhibition of the <I>sdpI</I> mRNA. 
The first strategy is based on an artificial synthesized RNA called sdpI-antisense RNA, which is complementary to the sdpI mRNA sequence. The sdpI-antisense sequence is regulated by P<SUB><I>ydfG</I></SUB> (Fig. 4A) which is activated by the binding of Ecf41<SUB><I>bli aa 1-204</I></SUB>. As a consequence the sdpI-antisense RNA and the sdpI mRNA form a double-stranded duplex which is inaccessible for the ribosome making the cell SDP-sensitive.
We tested the sdpI-antisense RNA in B. subtilis W168 with the xylose-inducable promoter P<SUB><I>xyl</I></SUB>. (Fig. 4B)
</P><P>The second strategy involves fsrA, a small RNA regulated by P<SUB>ydfG</SUB> which binds to its binding site sdhC' (the 5'-UTR of sdhC mRNA) which is cloned between P<SUB><I>sdpRI</I></SUB> and <I>sdpI</I> (Fig. 5A). Binding of fsrA to the sdhC' site on the transcript inhibits the ribosomal translation process of the sdpI-mRNA.
To evaluate this small RNA and its binding site in <I>B. subtilis</I> we designed two strains. One <I>∆abrB ∆sdpI</I> mutant strain where we replaced the P<SUB><I>ydfG</I></SUB> promoter with the P<SUB><I>xyl</I></SUB> promoter (Fig. 5B). The second construct containing fsrA under control of P<SUB><I>xyl</I></SUB> and replaced P<SUB><I>sdpRI</I></SUB>-<I>sdhC'sdpI</I> with the P<SUB><I>lepA</I></SUB>-<I>sdhC'gfp</I>, where GFP is consecutively expressed, (Fig. 5C) was transformed into <I>B. subtilis</I> W168.
</P><P>The suicide switch as well as the delay loop are part of our modelling work:
</P><P><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling#modelsuicide" class="external text" rel="nofollow">Mathematical model of suicide switch dynamics</A><A href="https://2014.igem.org/Team:LMU-Munich/Project/Modeling#model2" class="external text" rel="nofollow">Mathematical  model with additional sigma factor for delay</A></P></ARTICLE></DIV><DIV><LABEL for="suicide-results">Results</LABEL><ARTICLE class="ac-small">
The graph shows wildtype <I>Bacillus subtilis</I>, the ΔsdpI mutant as well as a ΔsdpI mutant carrying our generated (<A href="http://parts.igem.org/Part:BBa_K1351017" class="external text" rel="nofollow">BBa_K1351017</A>) in addition the graph shows a wildype <I>Bacillus subtilis</I> carrying an antisense RNA <A href="http://parts.igem.org/Part:BBa_K1351019" class="external text" rel="nofollow">(BBa_K1351019)</A> sequence which should bind the mRNA of the sdpI immunity. The LUMI-assay shows a normal growth curve of <I>Bacillus subtilis</I> and a ΔsdpI mutant which degrades in early to late stationary phase. The two from us constructed devices show in case of the inducible immunity in the ΔsdpI mutant an almost recovery in growth behavior when compared to wild type. The antisense carrying W168-strain showed normal wildtype growth and it seems like that an translational inhibition of the immunity was not functioning. 

For the suicide switch we figured that a time delay was necessary before the suicide switch kicks in so that BaKillus has enough time to do his job. Therefore we decided to test if an insertion of an alternative sigma factor would lead to a time delay. For this purpose we designed two constructs: one having an inducible promoter in front of a lux-readout and the other device carrying an alternative sigma (ECF41 <A href="http://parts.igem.org/Part:BBa_K823043" class="external text" rel="nofollow">BBa_K823043</A>) factor after the same inducible promoter. The sigma factor has to find its own specific promoter again (P<SUB>ydfG</SUB><A href="http://parts.igem.org/Part:BBa_K823041" class="external text" rel="nofollow">BBa_K823041</A>) which was cloned in front of the lux-operon.
The <A href="https://static.igem.org/mediawiki/2014/c/c7/LMU-Munich14_Luminescence_Assay.pdf" class="external text" rel="nofollow">LUMI-assay</A> in the graph shows that we could achieve a time delay of about ten minutes using the alternative sigma factor route. For a reasonable delay we intend to implement a series of alternative sigma factors and their target promoters to achieve a cascade and therefore increase the time delay. 
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Fig. 2. ECF41-mediated time delay</DIV></DIV></DIV></DIV></ARTICLE></DIV></DIV></SECTION></DIV><DIV id="footer"><DIV id="footer-content"><H2>Team LMU-Munich</H2><H3>igem@bio.lmu.de </H3>

AG Thorsten Mascher 
Großhaderner Str. 2 
82152 Planegg (Martinsried) </DIV></DIV><DIV id="tour-overlay"><DIV id="tour-box"><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Hi_there.21"> Hi there! </SPAN></H2><P>Welcome to our Wiki! I'm BaKillus, the pathogen-hunting microbe, and I'll guide you on this tour through our project. If you want to learn more about a specific step, you can simply close the tour and come back to it anytime you like. So let's start!

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="What.27s_the_problem.3F"> What's the problem? </SPAN></H2><P>First of all, what am I doing here? The problem is, pathogenic bacteria all around the world are becoming more and more  resistant against antimicrobial drugs. One major reason for the trend is the inappropriate use of drugs. With my BaKillus super powers, I want to reduce this misuse and thus do my part to save global health.

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Sensing_of_pathogens"> Sensing of pathogens </SPAN></H2><P>To combat the pathogenic bacteria, I simply eavesdrop on their communication. Bacteria talk with each other via quorum sensing systems, which I use to detect them and trigger my responses. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Adhesion_2"> Adhesion </SPAN></H2><P>The more specific and effective I can use my powers, the lower the danger is of provoking new resistance development. So I catch pathogens whenever I get hold of them and stick to them until my work is done.

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Killing_2"> Killing </SPAN></H2><P>Talking about my work - killing pathogens is finally what I am made for. In response to quorum sensing molecules of the pathogens, I export a range of antimicrobial substances leading to dissipation of biofilms and the killing of the targeted bacteria. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Suicide_switch_2"> Suicide switch </SPAN></H2><P>When the job is done and all the bad guys are finished, you don't need a super hero anymore. So after fulfilling my work I say goodbye to the world by activating my suicide switch. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="Application"> Application </SPAN></H2><P>Of course I'm not only a fictional hero, but a very real one. In two different prototypes, I could be used for diagnosis or treatment of pathogen-caused diseases. However, there is still a whole lot of regulational and economical questions that have to be answered before. 

</P></DIV></DIV><DIV class="tour-item"><DIV class="tour-item-content"><H2><SPAN class="mw-headline" id="See_you.21"> See you! </SPAN></H2><P>So now you know my short story - and it is time for me to return to my fight for a safer world. Feel free to take a closer look on my <A href="https://2014.igem.org/Team:LMU-Munich/Project/Bakillus" class="external text" rel="nofollow">super powers</A>, the <A href="https://2014.igem.org/Team:LMU-Munich/Notebook/Timeline" class="external text" rel="nofollow">process of my development</A> or the plans for a <A href="https://2014.igem.org/Team:LMU-Munich/Application" class="external text" rel="nofollow">medical application</A>.

</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:LMU-Munich/Project/Bakillus">http://2014.igem.org/Team:LMU-Munich/Project/Bakillus</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Project/Bakillus" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Project/Bakillus" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Bakillus&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Project/Bakillus&amp;oldid=406482" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>