"
Page
Discussion
View source
History
teams
Log in
 
Team:Warwick/Parts
From 2014.igem.org
HOME
PROJECT
TEAM
PARTS
MODELLING
NOTEBOOK
POLICY & PRACTICES
INTERLAB
ATTRIBUTIONS
APTAZYME
IRES
siRNA
NEOMYCIN
T7
RdRp
P2A
MS2
RNA PROMOTERS
TESTING MODULES
EXISTING BIOBRICK
Below is an interactive schematic of our system in RNA (for simplicity, we have omitted in the drawing a T7 promoter and its terminator to transcribe the whole operon), hover your mouse over any part to see a brief description, and click to follow through to the Registry and see a more in depth description, and information regarding sequences and results. You can find the full list of parts we've submitted here . The parts labelled 'favourite' are our best parts.
5' RNA Promoter
This is derived from the 5' UTR of the HCV virus strain 1b isolate Con1. It contains the reverse complement of the RdRp initiation sequence. The secondary structure of this sequence acts as a binding site and initiates replication of the minus strand by RdRp. It also includes the first 16 amino acids of the first gene of HCV as this has been shown to increase the efficacy of binding of the RdRp to the 5' promoter. We define an RNA promoter as a sequence located at the 3' end of the transcript, consisting of a RdRp initiation sequence and a ribozyme that cleaves after the initiation sequence without leaving any scar. We foresee that such regulatory elements will allow constructing novel post-transcriptional circuits by adding operator sites in the promoters. Such operator sites could be for instance the MS2 hairpin or any other RNA sequence known to bind to a regulator. This is the RNA equivalent to a transcription factor.
IRES
This  acts as an initiation for eukaryotic   ribosomes and begins translation of the   following protein sequence. We   compared two different IRESs: the   classical EMCV IRES used in many papers in our target cells,which has been shown to be compatible   with replicons and the NKRF IRES derived from the 3’UTR of  the mammalian NF-kappaB repressing   factor. We chose to test it also because during   investigation regarding the efficacy and   strength of the EMCV IRES, the NKRF derived   IRES was shown to be 30-fold more   efficient; however, it had never been previously used in Huh7.
MS2 Box
The MS2 box acts as a binding   site for the MS2 coat protein (formed   downstream in the replicon), which   represses translation, hence preventing   exponential growth of the replicon. This "copy number control" also allows a switching off of the RdRp once a steady state is reached, which minimizes the possible interferences between RdRp and the ribosome.
Neomycin Resistance
This was included in order to select   for the cells which had been   successfully transfected with the   replicon. This is commonly used in   human cell studies and allows survival   of eukaryotic cells in the prescence of   geneticin.
Aptazyme
This is an RNA enzyme which self-  cleaves in the presence of   theophylline. Theophylline is not   endogenous to mammalian cells,   hence acts as a selective “off-switch”   in an instance such as, Hepatitis C   infection or tumour formation which   is exacerbated by lack of DPP-IV.
siRNA
This is the functional element of our replicon and shows a  potential area for RNA experimentation. This sequence was   specifically designed using RNA fold and investigating the minimum   free energy bond formation, in order to form a secondary structure   in the positive sense that would not include a length of double   stranded RNA longer than 16 bases and hence would not be   recognised by Dicer and our replicon would remain intact for further   replication. However, in the negative sense the secondary structure   will form a hairpin recognised by Dicer which will result in cleavage   and release of the siRNA from the RNA strand allowing it to bind in a   complimentary fashion to the 3’UTR of DPP-IV mRNA and cause RNA   silencing hence reducing the amount of DPP-IV protein present in the   cell.  We used siRNA sequences from various papers to design this   hairpin.
MS2 Coat Protein
This coding sequence forms  a protein which binds to the MS2 box - a hairpin   type structure, and represses translation of the   replicon.
P2A
This is derived from the Porcine  Teschovirus-1 genome and cleaves with high   efficiency and leaves no scar following the   formation of a polyprotein MS2 coat protein,   P2A and RdRp. This is required as the HCV   genome has one open reading frame   from which one long polyprotein is produced.   RdRp is the final protein in this polygenic   mRNA and hence does not have a start codon,   adding a start codon could be disastrous for   the function of RdRp.
RdRp
This is a polymerase derived from Hepatitis C Virus Strain  1b isolate Con1 which catalyses the replication of RNA from an RNA template, reffered to as NS5B in   the contact of the HCV genome. Heterelogous expression of NS5B has been achieved in insect and   bacterial hosts, with RNA-dependent RNA synthesis initiated de novo (Behrens et al., 1996; Lohmann   et al., 1997). Structural studies indicate the hydrophobic C-terminal 21 amino acid residues cause   insertion into the membrane with other intracellular protein-protein interactions implicated (Moradpour   et al., 2004) making the final 63 bases essential for HCV RNA replication in eukaryotic cells   (Moradpour et al., 2004). In prokaryotic cells the final 21 amino acids are expendable. RdRp initiates   viral RNA synthesis with nucleotidyl transfer activity found within palm motifs A and C, with several   amino acid residues implicated in nucleotide triphosphate contact (Bressanelli et al., 2002). NS5B   activity has been demonstrated ''in vitro'', with synthesis of full length HCV RNA (Lohmann et al.,   1997; Ferrari et al., 1999). 5’ and 3’ untranslated regions (UTRs) of the HCV genome contains   ordered RNA structures, which are evolutionary conserved and contain crucial cis-acting elements for   viral RNA replication. 150 nt in the 3’ termini of HCV RNA contains elements which are essential for   RdRp binding and replication of viral RNA (Cheng et al., 1999; Yi and Lemon, 2003).
3' RNA Promoter
Each unique RNA dependent RNA polymerase (RdRP) initiates de novo replication of a RNA strand by interacting with RdRP-specific RNA sequences, henceforth called RdRP/RNA promoters. The promoter is "insulated" by using a ribozyme on its 3' end that self-cleaves without leaving a scar (and, therefore, producing the right 3' end for RdRp initiation).  The RdRP chosen for our project is taken from the Hepatitis C virus (HCV) and it recognizes a limited set of such initiation sequences. All of them possess a few common characteristics: an initiation cytidylate at the 3’ end, where the replication starts; and a stable secondary structure – single stranded tail and a stem of various length. For our project in addition to the indigenous to HCV RdRP promoters, we designed alternative RNA promoter sequences previously identified by Heinz et al. as templates for replication by the HCV RdRP.
Created by: iGEM Warwick
Contact information: igem.warwick@gmail.com
Retrieved from " http://2014.igem.org/Team:Warwick/Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
