<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tsinghua_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tsinghua/project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tsinghua/project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua/project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua/project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tsinghua/project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tsinghua/project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="topbar"><UL id="topbarul"><LI id="nav-home"><A href="/Team:Tsinghua">Home</A></LI><LI id="nav-team"><A href="/Team:Tsinghua/team">Team</A></LI><LI id="nav-project"><A href="/Team:Tsinghua/project">Project</A></LI><LI id="nav-model"><A href="/Team:Tsinghua/modeling">Model</A></LI><LI id="nav-notebook"><A href="/Team:Tsinghua/experiments">Notebook</A></LI><LI id="nav-parts"><A href="/Team:Tsinghua/parts">Parts</A></LI><LI id="nav-safety"><A href="/Team:Tsinghua/safety">Safety</A></LI><LI id="nav-human"><A href="/Team:Tsinghua/HumanPractice">HumanPractice</A></LI></UL></DIV><DIV id="right_bar"><DIV class="left_gadget"><DIV class="countdown"><DIV id="counttitle">
Until Jamboree</DIV><DIV class="countback"><P class="countstatic" id="pday"> days</P><P class="countstatic" id="phour"> hours</P><P class="countstatic" id="pmin"> minutes</P><P class="countstatic" id="psec"> seconds</P></DIV></DIV></DIV><DIV class="left_gadget"><H2>Follow us on</H2></DIV><DIV class="left_gadget"><H2>Visitor Locations</H2></DIV><DIV class="left_gadget"><H2>Join the conversation</H2></DIV></DIV><DIV id="content_box"><DIV id="main_content"><H1><SPAN class="mw-headline" id="E._Colimousine">E. Colimousine</SPAN></H1><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>Single cells can be powerful in information processing, but information processors assembled from just a few cells are limited in their capabilities. In a complex information processing system, separation of sensor and processor is necessary for eliminating unwanted cross-talk, which in turn requires the signal transduction system.
</P><P>Our project is destined to generate an <I>E. coli</I> strain which can transport signal protein along a gradient back and forth. The bacteria can bind with the target protein at one place and then, by sensing the gradient of certain amino acid, move to its destination, where it releases the protein with the help from another <I>E. coli</I> strain. When this is done, it will move back to transport more target protein. In this process, four modules are necessary: binding, releasing, and movement.
</P><H2><SPAN class="mw-headline" id="Binding">Binding</SPAN></H2><P>We first need to conjugate a binding vehicle directly sensing substrate in the medium. <SPAN id="a-tip">Outer membrane protein A (OmpA) <A class="div-tip">OmpA-like transmembrane domain is an evolutionarily conserved domain of outer membrane proteins. This domain consists of an eight-stranded beta barrel. OmpA is the predominant cell surface antigen in enterobacteria found in about 100,000 copies per cell. The expression of OmpA is tightly regulated by a variety of mechanisms.</A></SPAN> comes into our notice for its outstanding property that proteins linked to its C-terminal will be pushed out of the bacteria’s membrane.
</P><P><SPAN id="a-tip">Src-homology 3(SH3) domain<A class="div-tip">Src-homology  3 (SH3) domain has high  affinity for proline-rich peptides and together they can form a left-handed poly-Pro type II helix, with the minimal consensus Pro-X-X-Pro.</A></SPAN>  which has high affinity for proline-rich peptides, could act like tongs to grab any protein with a proline riich motif. For easy track, fluorescent protein mCherry was the one linked withthe binding motif (Kd=3.67μM).
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 285px;"><DIV class="gallerytext"><P>SH3 binding domain (shown bound to proline-rich peptide)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 285px;"><DIV class="gallerytext"><P>OmpA transmembrane segment
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>The binding module eventually comprises of two proteins, namely, OmpA-SH3 protein, which functions as the binding vehicle and proline-rich containing mCherry protein, which functions as the binding substrate.
</P><H2><SPAN class="mw-headline" id="Releasing">Releasing</SPAN></H2><P>Considering of the significance of efficient and specific release from strong binding, HIV-protease site was constructed into the binding cassette, which enables proteolytic release by <SPAN id="a-tip">HIV-protease<A class="div-tip"> HIV-protease is readily available and its high efficiency, low molecular weight and high specificity matches the fine candidate</A></SPAN>. We fused HIV-protease with OmpA and expressed it in a second <I>E. coli</I> strain which we immobilize at the destination, so substrate (together with SH3 domain) could be released once the E. colimousine arrives there.
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Protease site in binding cassete
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Proteolytic release
</P></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Movement">Movement</SPAN></H2><P>Regulatory network controlling <I>E. coli</I> chemotaxis is one of the well-established models of signaling transduction, hence chemotaxis is most frequently used in guiding the movement of <I>E. coli</I> cells. Here, receptors Tar and Tsr* are assembled into large clusters that are predominantly located at the poles of the bacterium could sense extracellular signals as aspartate and serine respectively. The CheW protein binds to the receptors and acts as a scaffold protein to recruit CheA, whose phosphorylation is stimulated by the receptor. The phosphate is transferred to the cytoplasmic protein CheY, which controls flagellar rotation and in turn, controls the movement of E.coli. Previously, a pair of mutations in CheW* (V108M) and Tsr* (E402A) was shown to produce an orthogonal pair (R.B. Bourret, A.M. Stock, <I>et al</I>., 2002), where the mutated proteins interact with each other but not with wildtype Tsr or CheW. So, this can be used to toggle between activating the serine-responsive Tsr* and aspartate-responsive Tar. When we set up two gradients (serine and aspartate) in the opposite direction, the bacteria can move forth and back.
</P><P>As leakage level expression of chemotaxis receptor is sufficient to drive chemotaxis, we resorted to pChemoK plasmid constructed in Christopher's lab (T.S. Moon, E.J. Clarke, <I>et al</I>., 2011). In this plasmid, expression of Tar and Tsr is controled by fimE invertase system. As the direction of promoter is inverted, no basal transcription exists and strict control of expression is achieved.
</P><P><I>pChemoK plasmid, from Christopher's lab</I></P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Tsinghua/project">http://2011.igem.org/Team:Tsinghua/project</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tsinghua/project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tsinghua/project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tsinghua/project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tsinghua/project&amp;oldid=253979" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>