"
Page
Discussion
View source
History
teams
Log in
 
Team:Tsinghua/project
From 2011.igem.org
Home
Team
Project
Model
Notebook
Parts
Safety
Human
Practice
Until Jamboree
days
hours
minutes
seconds
Follow us on
Visitor Locations
Join the conversation
E. Colimousine
Overview
Single cells can be powerful in information processing, but information processors assembled from just a few cells are limited in their capabilities. In a complex information processing system, separation of sensor and processor is necessary for eliminating unwanted cross-talk, which in turn requires the signal transduction system.
Our project is destined to generate an E. coli strain which can transport signal protein along a gradient back and forth. The bacteria can bind with the target protein at one place and then, by sensing the gradient of certain amino acid, move to its destination, where it releases the protein with the help from another E. coli strain. When this is done, it will move back to transport more target protein. In this process, four modules are necessary: binding, releasing, and movement.
Binding
We first need to conjugate a binding vehicle directly sensing substrate in the medium. Outer membrane protein A (OmpA) OmpA-like transmembrane domain is an evolutionarily conserved domain of outer membrane proteins. This domain consists of an eight-stranded beta barrel. OmpA is the predominant cell surface antigen in enterobacteria found in about 100,000 copies per cell. The expression of OmpA is tightly regulated by a variety of mechanisms. comes into our notice for its outstanding property that proteins linked to its C-terminal will be pushed out of the bacteria’s membrane.
Src-homology 3(SH3) domain Src-homology  3 (SH3) domain has high  affinity for proline-rich peptides and together they can form a left-handed poly-Pro type II helix, with the minimal consensus Pro-X-X-Pro. which has high affinity for proline-rich peptides, could act like tongs to grab any protein with a proline riich motif. For easy track, fluorescent protein mCherry was the one linked withthe binding motif (Kd=3.67μM).
SH3 binding domain (shown bound to proline-rich peptide)
OmpA transmembrane segment
The binding module eventually comprises of two proteins, namely, OmpA-SH3 protein, which functions as the binding vehicle and proline-rich containing mCherry protein, which functions as the binding substrate.
Releasing
Considering of the significance of efficient and specific release from strong binding, HIV-protease site was constructed into the binding cassette, which enables proteolytic release by HIV-protease HIV-protease is readily available and its high efficiency, low molecular weight and high specificity matches the fine candidate . We fused HIV-protease with OmpA and expressed it in a second E. coli strain which we immobilize at the destination, so substrate (together with SH3 domain) could be released once the E. colimousine arrives there.
Protease site in binding cassete
Proteolytic release
Movement
Regulatory network controlling E. coli chemotaxis is one of the well-established models of signaling transduction, hence chemotaxis is most frequently used in guiding the movement of E. coli cells. Here, receptors Tar and Tsr* are assembled into large clusters that are predominantly located at the poles of the bacterium could sense extracellular signals as aspartate and serine respectively. The CheW protein binds to the receptors and acts as a scaffold protein to recruit CheA, whose phosphorylation is stimulated by the receptor. The phosphate is transferred to the cytoplasmic protein CheY, which controls flagellar rotation and in turn, controls the movement of E.coli. Previously, a pair of mutations in CheW* (V108M) and Tsr* (E402A) was shown to produce an orthogonal pair (R.B. Bourret, A.M. Stock, et al., 2002), where the mutated proteins interact with each other but not with wildtype Tsr or CheW. So, this can be used to toggle between activating the serine-responsive Tsr* and aspartate-responsive Tar. When we set up two gradients (serine and aspartate) in the opposite direction, the bacteria can move forth and back.
As leakage level expression of chemotaxis receptor is sufficient to drive chemotaxis, we resorted to pChemoK plasmid constructed in Christopher's lab (T.S. Moon, E.J. Clarke, et al., 2011). In this plasmid, expression of Tar and Tsr is controled by fimE invertase system. As the direction of promoter is inverted, no basal transcription exists and strict control of expression is achieved.
pChemoK plasmid, from Christopher's lab
Retrieved from " http://2011.igem.org/Team:Tsinghua/project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
