<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCAS_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCAS/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Proof_of_Concept</TITLE></P><DIV class="container-fluid" style="background-color: black" id="top"><NAV class="homeNav" style="float: right"><UL><LI><A href="https://2016.igem.org/Team:UCAS">HOME</A></LI><LI><A>TEAM</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Collaborations">Collaborations</A></LI></UL><LI><A>PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Experiments">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Results">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Notebook">Notebook</A></LI><LI><A>Parts</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Model">Model</A></LI><LI><A>HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:UCAS/Attributions">Attributions</A></LI></NAV></DIV><DIV class="container home-content bs-docs-container"><DIV class="row clearfix"><DIV class="col-xs-9 col-sm-9 col-md-9 mid_col"><DIV class="bs-docs-section"><H1 id="js-data-part1">PART 1</H1><P style="font-size:18px;">
                        Among the four kinds of oxidases, only TetX monooxygenase and MnCcP are proved effective in degrading tetracycline. We further conduct in vivo and in vitro, qualitative and quantitative experiments, at molecular and cellular levels.
                    </P><H2 id="js-data-proof1" class="page-header">Activity analysis of TetX</H2><H3 id="js-data-proof1-1" style="color:#777777;">Experiment 1 (in vivo/qualitative):</H3><P style="font-size:18px;">
                        E. Coli having tetX sequence can survive in relatively high tetracycline environment while non-resistant E. Coli cannot.
                    </P><P style="font-size:18px;">
                        Three solid media which contained respectively 0, 2.5, 5 µg/ml tetracycline were inoculated with two kinds of E. coli (one with protein expression of tetX and the other without it), and the concentration of the E. coli solution was in a series of dilution (1x, 10x, 10^2x, 10^3x, 10^4x, 10^5x)
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 1</STRONG></DIV><H3 id="js-data-proof1-2" style="color:#777777;">Experiment 2 (in vivo/qualitative):</H3><P style="font-size:18px;">
                        After 30mins reaction in 30℃, tetracycline solution discolors when TetX is added which indicates its degradation, while color of the control group remains the same.
                    </P><P style="font-size:18px;">
                        100, 200, 500, 1000uM tetracycline, 2mM NADPH, 100 µM Tris and tetX. Control group contained all the same components of the experimental group but without tetX, and pH of both reaction systems was 8.5. After 30 mins of reaction at 30℃, the color change of the experimental group(left) was very obvious, but color of the control group(right) did not changed obviously.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 2</STRONG></DIV><H3 id="js-data-proof1-3" style="color:#777777;">Experiment 3 (in vivo/qualitative):</H3><P style="font-size:18px;">
                        Enzyme activity test of TetX
                        100uL reaction system
                        Tris pH=8.5   10mM
                        Tc                   30uM
                        tetX 10uL       2.3uM
                        NADPH          200uM
                        NADPH is added at last to initiate the reaction
                    </P><P style="font-size:18px;">
                        We use the kinetics mode in ultraviolet spectrophotometer to record the changes of absorption in 360nm with total time 200 seconds and cycle time 2 seconds. The addition of NADPH at 25 s increases the absorption at 360nm dramatically. Except for adding no tetX protein, other composition of the control group is the set as the same with experiment group.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 3</STRONG></DIV><P style="font-size:18px;">
                        The characteristic absorption peak of tetracycline at 360nm in experiment group decreases rapidly, while control remains the same, which indicates the fast reaction of TetX and tetracycline in presence of NADPH. Considering of the ratio of tetracycline and NADPH, 1:1, the interference caused by NADPH absorption was eliminated. Culculation showed the turnover number (Kcat) was 12.6/min.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 4</STRONG></DIV><DIV><STRONG style="text-align: center;color: red;">Fig. 5</STRONG></DIV><P style="font-size:18px;">
                        With the reaction time passes, the characteristic absorption peak of tetracycline at 360nm in experiment group decreases gradually compared with control group.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 6</STRONG></DIV><DIV><STRONG style="text-align: center;color: red;">Fig. 7</STRONG></DIV><P style="font-size:18px;">
                        After 200 seconds, the characteristic absorption peak of tetracycline at 360nm in experiment group drops obviously as compared to 25s when the reaction is initiated by NADPH, while the absorption at 360nm in control group still shows no difference between 25s and 200s.
                    </P></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof2" class="page-header">Kinetics analysis according to Michaelis–Menten Equation</H2><P style="font-size:18px;">
                        \[{V_0} = \frac{{{V_{\max }}\left[ S \right]}}{{{K_{\rm{m}}}{\rm{ + }}\left[ S \right]}}\]
                        With double-reciprocal plot method ( also known as <B>Lineweaver-Burk Equation</B>),
                        \[\frac{1}{{{V_0}}} = \frac{{{K_{\rm{m}}}}}{{{V_{\max }}\left[ S \right]}}{\rm{ + }}\frac{1}{{{V_{\max }}}}\]
                        plotting 1/Vo to 1/[So], and linear fitting results showed the vertical intercept 1/Vmax and the slope Km/Vmax. Through calculation, Km=48.93uM, Vmax=3.20uM/s; kcat/Km=4.293*103M-1 s-1.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 8</STRONG></DIV></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof3" class="page-header">Substrate Analysis</H2><H3 id="js-data-proof3-1" style="color:#777777;">Experiment 4 (in vivo/quantitative): E.coli degrades Tc. in M9 medium</H3><P style="font-size:18px;">
                        100mL M9 liquid medium with 100uM Tc. and 1mM Glucose
                    </P><P style="font-size:18px;">
                        E. coli expressing tet X and Ruby were respectively added into experimental and control group (double dilute of the bacterium solution).
                    </P><P style="font-size:18px;">
                        After 19h or so in shake incubator under 37℃, tetracycline in the reaction systems was extracted out by HLB solid phase extraction column, and residues of tetracycline in the media could be tested by LC-MS. The chromatograms of control (red) and experiment (green) are as follows:
                    </P><P style="font-size:18px;">
                        E. coli degraded tetracycline in M9 medium(chromatographic analysis of control and experiment group)
                    </P><P style="font-size:18px;">
                        By calculated integral area, residue of the experimental group was 1.89% of that of the control group.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 9</STRONG></DIV><P style="font-size:18px;">
                        Take the reaction system before experiment as standard and measure the amount of tetracycline. The amount of Tc.: control/standard system = 60%, experiment/standard = 1.1%
                    </P><P style="font-size:18px;">
                        (Note: Compare the peak and calculated integral area at 6.72min (retention time), and the area of other parts could be deleted. Since the liquid system is nine times concentrated, the residue of experimental group was 1.89%)
                    </P><P style="font-size:18px;">
                        Different concentration of  E.coli solution's tetracycline degradation effect in M9 medium
                        1)5mL medium with 1mM Glucose       Concentration gradient of tetracycline: 20uM, 100uM, 200uM
                        2)5mL medium with 1mM Glucose       Concentration gradient of tetracycline: 10 times dilution, 20times dilution, 50times dilution</P><P style="font-size:18px;">
                        Group 1) and 2) through a night and their changes were as followed:
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 10</STRONG></DIV><P style="font-size:18px;">
                        By solid phase extraction, tetracycline residue of each group was tested by LC-MS
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 11</STRONG></DIV></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof4" class="page-header">Activity and kinetics of MnCcP</H2><H3 id="js-data-proof4-1" style="color:#777777;">MnCcp enzyme activity assay:</H3><P style="font-size:18px;">
                        Concentration of 100uLreaction system: HAc/NaAc pH=5.2-5.4 100mM；Tc 30uM；MnCcP 1ul；Mn(II)5mM；h3O2 1mM.There was no MnCcP protein in the control group. After dispose in 37℃;in 30mins and centrifuging, the ultraviolet absorption results of supemate were as follows.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 12</STRONG></DIV><DIV><STRONG style="text-align: center;color: red;">Fig. 13</STRONG></DIV><P style="font-size:18px;">
                        The protein concentration ofMnCcP is3.7uM. Accoring to Michaelis-Menten equation with  Double inverse method. Mapping1/Vo to1/[So]，and linear fitting results showed Km= 322.8uM, Vmax= 2.506uM/s; kcat=0.556757/s; kcat/Km= 1.725*10<SUP>3</SUP>M-1 s-1. Compared to TetX monooxygenase，MnCcP kcat(turnover number)、Michaelis constant are much higher，but kcat/Km（specificity constant）is a bit low, which shows its strong catalytic rate but its affinity with the substrate, its substrate specificity and its catalytic efficiency are weaker. Since the substrate specificity of MnCcP was weak, it is likely that it can become a kind of broad-spectrum oxidase for many types of antibiotics.
                    </P></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof5" class="page-header">Suppelementary materials</H2><P style="font-size:18px;">
                        Method and test of Solid phase extraction：
                    </P><P style="font-size:18px;">
                        We use Supel-Select HLB solid phase extraction column for recovery of tetracycline from M9 medium. We repeated sampling twice, pure water wash twice, and finally using DMF repeatedly wash HLB column twice. The Recovery rate of tetracycline from the M9 medium was 87%.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 14</STRONG></DIV></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof6" class="page-header">Extension：the resistance gene or degradation enzymes for other types of antibiotics.</H2><P style="font-size:18px;">
                        In our experiment design, we just took tetracycline degradation as an example. As a matter of fact, TetX and MnCcP can also degrade other members of tetracycline antibiotics, like oxytetracycline and tigecycline, etc. Moreover, by substituting tetracycline degradation enzyme with degradation enzymes for other types of antibiotics, we can achieve our goal to apply our engineering bacteria to the degradation of as many as possible kinds of antibiotics which was theoretically feasible with different degradation enzyme and our system. The chart below demonstrates the degradation enzymes for other major types of antibiotics together with its mechanism, which serves as great inspiration for future work.
                    </P><A href="https://static.igem.org/mediawiki/2016/d/df/T--UCAS--proof_Extention.pdf">File: Degradation enzymes for other types of antibiotics.pdf</A></DIV><DIV class="bs-docs-section"><H1 id="js-data-part2">PART 2</H1><P style="font-size:18px;">
                        In this part, we intended to develop a kill-switch based on TA module, so the very first thing we needed to do is to choose several candidate toxins proteins. We then used RBS Calculator to simulate the expression level of proteins under different RBSs. This shall help us avoid RBS leakage, which could fail our experiments. As a proof-of-concept, we showed that some of the toxins effectively kills the bacteria or suppress the growth and cleavage of bacteria cells, and can grow as normal when antitoxin protein is expressed.
                    </P><H2 id="js-data-proof2-1" class="page-header">Selection of TA modules</H2><P style="font-size:18px;">
                        As mentioned before, bacteria usually harbor thousands of kinds of TA modules both on genomes and plasmids. Based on a list of more than 2000 TA modules generously provided by our Instructor, Hao Jiang, we chose TA modules according to the following criteria: (1) The GC content of the proteins should be close to that of E. coli, for partially represents codon preferences and may greatly influence the expression of proteins; (2) An evolutionary tree of pre-selected TA modules and those found in E. coli is generated by our Instructor. We did not make use of it in a quantitative way, but we avoided choosing those TA modules that are too close to TA modules in E. coli, because there might be cross-talk between TA modules of E. coli and that cloned by ourselves.
                    </P><P style="font-size:18px;">
                        Finally, we selected twelve TA modules for further tests. For each TA module, there is a distinct ID number. Since some of the TA modules has not been experimentally validated (see the ‘is_experiment’ column in Table. 1) and do not have their specific family designated (see the ‘Toxin Protein’ and ‘Antitoxin Protein’ columns below), we will use this ID number later to denote corresponding TA modules. In addition, it is worth mentioning that 134 toxin/antitoxin has been submitted to the parts registry, which is a MazF homologous. We further characterized this part, as we will show later.
                    </P><DIV><STRONG style="text-align: center;color: red;">Table. 1</STRONG></DIV></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof2-2" class="page-header">In silica: Choice of RBS</H2><P style="font-size:18px;">
                        When expressing any protein in bacteria, promoter leakage is always a major concern, for unwanted expression of specific genes could disrupt the original experiment design or even lead to failure.
                    </P><P style="font-size:18px;">
                        In our TA module experiments, we needed to evaluate the toxicity of the many different toxin genes in vivo. So when it comes to the design of the plasmid, there are two major concerns: expression strength upon induction and promoter leakage. If the translation rate is too high, the toxin may straightly kill the bacterial without induction. In contrast, if the translation rate is too low, the lethality of toxin won’t be manifested, so we will not know whether the gene actually works. Not only does this problem lead to failure of our experiments, but also disrupts the implementation of our genetic circuits. So here comes the central topic——one of the most efficient ways to regulate the translation is to choose the right ribosome binding site (RBS). In our design, we used the RBS Calculator by De Novo DNA [8], [9] to predict the translation initiation rates.
                    </P><P style="font-size:18px">
                        But first, how does this calculator work? The RBS Calculator uses a statistical thermodynamic model to predict the translation initiation rate of a protein CDS. Given a RBS and a protein CDS, the model calculates the free energy change during the assembly of the 30S complex onto the mRNA. For more information, visit: https://salislab.net/software/reverse.
                    </P><P style="font-size:18px">
                        In our project, the RBS is supposed to maintain the toxin expression level relatively low and antitoxin slightly higher. According to our teachers’ advice, we first chose the weakest RBS among iGEM parts registry, BBa_B0033, as our target toxin genes’ RBS and simulated the protein expression level in the RBS Calculator. RBS BBa_B0033 could keep expression levels of most toxins lower than 10% of its original expression level in source culture, that is to say, E. coli. While some toxins with RBS BBa_B0033 still had a slightly high expression level (toxin 1198), we altered several nucleotides in RBS BBa_B0033 found out toxin expression level degrade to an appropriate extent.
                    </P><P style="font-size:18px">
                        After we measured the growth curves of E. coli with toxin genes, toxin 134, toxin 1204 and toxin 6249 manifested noticeable toxicity. Toxin 133, toxin 136 with RBS BBa_B0033 and toxin 1198 with refitted BBa_B0033 show that their toxicity is not strong enough to kill the bacteria. We apply all the RBS provided by iGEM to these three toxins and simulate the toxin expression level in RBS Calculator. Considering these three toxins’ toxicity is extremely, we chose the strongest RBS, BBa_B0034, for toxin 1198, and BBa_B0035 for toxin 136 and toxin 6249.
                    </P><P style="font-size:18px">
                        These adjustments successfully increase the toxicity of these three toxin to an appropriate extent. For antitoxins with RBS BBa_B0033, the simulate expression level is sufficiently high, so we selected BBa_B0033 as antitoxins' RBS. (Table. 2)
                    </P><DIV><STRONG style="text-align: center;color: red;">Table. 2</STRONG></DIV><P style="font-size:10px;width: 80%;font-style:oblique;font-weight:bold;">
                        The table above shows ONLY part of our calculation results, which have great reference value. In the “T&amp;A_id” column, each ID number represents a toxin or an antitoxin. In the row of “RBS”, the “source” means the calculation is carried out without any RBS in the gene’s original bacteria. 
                    </P></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof2-3" class="page-header">Clone and Identification of toxins and antitoxins</H2><P style="font-size:18px;">
                        The gene sequence of toxin/antitoxin 133, 134, 136, 1198, 1204 and 6249 is cloned from corresponding genomes. We thank Prof. Chunbo Lou for providing us with genomes of Bacillus subtilis, Mycobacterium tuberculosis, Photorhabdus luminescens. Genes of toxin/antitoxin 5693, 5694, 5695, 5980 and 4222 is commercially synthesized and has been optimized according to codon preference in E. coli.
                    </P><P style="font-size:18px;">
                        All the toxic genes are cloned to a pSB3A5-derived vector called RGP_Ptet using the Golden Gate method [10]. The expression of toxins is induced by anhydrotetracycline (aTc). To test whether these toxins can influence the growth of E. coli, we first plated them on agar LB medium without and with 25µL of 5mg/mL aTc added, respectively. As is clearly presented in Fig. 2, among 11 toxins tested, 8 of them are more or less toxic to E. coli. In particular, toxin 134, 136, 1204, 6249 has significant toxicity to the host cell.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 15</STRONG></DIV><P style="font-size:18px">
                        It is worth noting that toxin 5694 does not seem to be so toxic compared to other toxins. In fact, when we clone the gene onto the vector using Golden Gate method, we are unable to get the colony with expected plasmid. The experiment is repeated for three times and all fail. So we speculate that toxin 5694 is so toxic that even the background expression level can kill bacteria. Thus, we deleted its RBS, putting the protein coding region directly downstream of the Tet promoter. As expected, we got the right clone with a detectable level of toxicity.
                    </P><P style="font-size:18px">
                        After qualitative test of the toxicity of toxins, we used 96-well microplate spectrophotometer to measure the growth curves of host cells harboring these toxin genes to further confirm the activities of toxins. Results are illustrated in Fig. 16 and Fig. 17.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 16</STRONG></DIV><DIV><STRONG style="text-align: center;color: red;">Fig. 17</STRONG></DIV><P style="font-size:18px">
                        All the toxins in Fig. 16 were cloned by ourselves. Among them, toxin 134, 136, 1198, 1204 and 6249 showed relatively high toxicity. Toxin 133 had smaller effect, as it did not totally impede cell growth until a concentration of 5 µg/mL. What is more, the curves of 1198 and 1204 showed exponential pattern after about 8 hours, indicating that 1204 is more susceptible to mutations. In contrast, toxin 6249 is almost as toxic as 1204 and showed no observed mutations after 10 hours. So we planned to use toxin 6249 in subsequent demonstration. This conclusion is also consistent with what we got through modeling (See: Modeling).
                    </P><P style="font-size:18px">
                        In Fig.17, commercially synthesized toxin genes are listed. Most of them were not so toxic. Instead, they only made bacteria grow more slowly, including toxin 5694.
                    </P><P style="font-size:18px">
                        Note that all toxin genes except 133 and 1198 has RBS BBa_B0033, which is the weakest one, whilst toxin 5694 does not have a RBS upstream. So for further characterization and usage, stronger RBSs may be chosen to enhance expression level.
                    </P><P style="font-size:18px">
                        However, only identifying toxins is far from enough for our project. So we further cloned antitoxin genes of 134, 136, 1204, 6249, induced by IPTG. To see whether these antitoxins can really neutralize their corresponding toxins, we plated 9µL of 5mg/mL aTc onto agar LB medium to fully induce the expression of toxins. Then, 0, 2, 3, 4 µL of 1M IPTG is plated. As demonstrated in Fig. 18, all of the four toxins are neutralized by corresponding antitoxins, except for toxin 136.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 18</STRONG></DIV><P style="font-size:18px">
                        To test whether this is due to insufficient antitoxin expression level or the property of protein per se. To answer this question, we further plated 7, 8, 9 µL of 1M IPTG together with 9µL of 5 mg/mL aTc to increase the expression level of antitoxin. Unfortunately, results are the same (Fig. 19). No colony could be seen on the agar LB medium plate even at 9µL of IPTG. This indicates that antitoxin 136 cannot neutralize its corresponding toxins.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 19</STRONG></DIV><P style="font-size:18px">
                        To further characterize the behavior of bacteria co-expressing toxins and antitoxins, we also measured growth curves of those bacteria. In Fig.20, the upper line illustrates induction of antitoxin with 8mM after 4 hours of incubation, whereas toxin is induced at the beginning of the measurement by 1.7 µg/mL and 5 µg/mL, respectively. We clearly see that 134 (EndoA) showed a sigmoidal curve almost immediately after addition of IPTG, whilst significant growth of bacteria with 1204 (N-acetyltransferase GCN5) is not observed until 8 hours (480 mins in Fig.20 (a)) of incubation (4 hours after induction). Curves of toxin 136 (yobR) and 6249 (ParD2) started to grow at 10 hours, which according to our previous measurements without antitoxin, is due to mutation under selective pressure of toxins. Notably, the error bars got bigger and bigger in both 136 and 6249 after 10 hours. Since the graph is drawn from three biological replicates, this possibly suggests that varied mutations had different effects on the growth of bacteria.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 20</STRONG></DIV><P style="font-size:18px">
                        After getting both qualitative and quantitative data of toxins and antitoxins, we analyzed the morphological changes of bacteria expressing different kinds of toxins. Since among the 11 toxins we selected, only four of them showed significant toxicity to E. coli, we examined cells expressing toxin 134 (EndoA), 136 (yobR), 1204 (N-acetyltransferase GCN5), 6249(ParD2) by a Hitachi SU8010 scanning electron microscopy (SEM) at Institute of Microbiology, Chinese Academy of Sciences (IM-CAS). Specimens fixation is carried out by ourselves, and lyophilizating and gold-coating is accomplished by technicians at IM-CAS. We by ourselves operated the SEM by ourselves and results are showed in Fig. 21.
                    </P><DIV><STRONG style="text-align: center;color: red;">Fig. 21</STRONG></DIV><P style="font-size:18px;">
                        Wild type E. coli are usually longer 2µm in length, whilst just-cleaved cells are slightly shorter. EndoA (134) induced with 500µM of IPTG makes the cell round, with only about 1µm in diameter, as the picture in the left upper corner showed. Although N-acetyltransferase GCN5 (1204) is not as toxic as EndoA, it still shows observable growth rate. The cell is comparably shorter than wild type, at a little more than 1µm. It seems that ParD2 (6249) is as toxic as N-acetyltransferase GCN5, but actually to the left of the three bacteria cells in the picture, a lysed cell can be seen. We also other lysed cells in the specimen (photos not shown), this suggests that ParD2 (6249) is the most toxic, in accordance with what we got from toxic growth curve and modelling 
                    (See <A href="https://2016.igem.org/Team:UCAS/Model">modeling</A>).
                    </P></DIV><DIV class="bs-docs-section"><H2 id="js-data-proof7" class="page-header">References</H2><P style="font-size:12px;font-weight:bold;">
                        [1] Xiao Zhang, Xiaohong Li, Yucheng Hu, Shuni Li and Quanguo Zhai. Combination of enzymatic degradation by chloroperoxidase with activated sludge treatment to remove sulfamethoxazole: performance, and eco-toxicity assessment. 4888 jctb(2016)
                    </P><P style="font-size:12px;font-weight:bold;">
                        [2] Xianghua Wen, Yannan Jia, Jiaxi Li. Enzymatic degradation of tetracycline and oxytetracycline by crude manganese peroxidase prepared from Phanerochaete chrysosporium. Journal of Hazardous Materials. 177 (2010) 924–928
                    </P><P style="font-size:12px;font-weight:bold;">
                        [3] Thomas D. Pfister, Amir Y. Mirarefi, Alan J. Gengenbach, Xuan Zhao,Connor Danstrom Nicole Conatser, Yi-Gui Gao, Howard Robinson, Charles F. Zukoski, Andrew H.-J. Wang, Yi Lu Kinetic and crystallographic studies of a redesigned manganese-binding site in cytochrome c peroxidase. J Biol Inorg Chem. 12 (2007) 126–137
                    </P><P style="font-size:12px;font-weight:bold;">
                        [4]WupengYan, Heng Song, Fuhang Song, Yisong Guo, Cheng-Hsuan Wu, Ampon Sae Her, Yi Pu, Shu Wang, Nathchar Naowarojna, Andrew Weitz, Micheal P.Hendrich, Catherine E.Costello, Lixin Zhang, Pinghua Liu&amp;Yan Jessie Zhang. Endoperoxide formation by an α-ketoglutarate-dependent mononuclear non-haem iron enzyme. Nature. 527.(2015) 539-543
                    </P><P style="font-size:12px;font-weight:bold;">
                        [5] Ian F. Moore, Donald W. Hughes, and Gerard D. Wright. Tigecycline Is Modified by the Flavin-Dependent Monooxygenase TetX. Biochemistry.44, 11829-11835 (2005)
                    </P><P style="font-size:12px;font-weight:bold;">
                        [6] Gesa Volkers, Gottfried J. Palm, Manfred S. Weiss, Gerard D. Wright, Winfried Hinrichs. Structural basis for a new tetracycline resistance mechanism relying on the TetX monooxygenase. FEBS Letters. 585, 1061-1066(2011)
                    </P><P style="font-size:12px;font-weight:bold;">
                        [7] Brenda S. Speer and Abigail A Salyers. Novel Aerobic Tetracycline Resistance Gene That Chemically Modifies Tetracycline. Journal of Bacteriology. 171.148-153 (1989)
                    </P><P style="font-size:12px;font-weight:bold;">
                        [8] Salis, H. M., Mirsky, E. A. &amp; Voigt, C. A. Automated design of synthetic ribosome binding sites to control protein expression. Nat. Biotechnol. 27, 946–50 (2009)
                    </P><P style="font-size:12px;font-weight:bold;">
                        [9] Espah Borujeni, A., Channarasappa, A. S. &amp; Salis, H. M. Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites. Nucleic Acids Res. 42, 2646–2659 (2014)
                    </P><P style="font-size:12px;font-weight:bold;">
                        [10] Engler, C., Kandzia, R. &amp; Marillonnet, S. A one pot, one step, precision cloning method with high throughput capability. PLoS One 3, (2008)
                    </P></DIV></DIV><DIV class="col-xs-3 col-sm-3 col-md-3"><DIV class="bs-docs-sidebar hidden-print hidden-xs hidden-sm" role="complementary"><UL class="nav bs-docs-sidenav" style="margin-left: 0px;"><LI><A href="#js-data-part1">PART 1</A></LI><LI><A href="#js-data-proof1">Activity analysis of TetX</A></LI><LI><A href="#js-data-proof1-1">Experiment 1 </A></LI><LI><A href="#js-data-proof1-2">Experiment 2</A></LI><LI><A href="#js-data-proof1-3">Experiment 3</A></LI></UL><LI><A href="#js-data-proof2">Kinetics analysis according to M-M equation</A></LI><LI><A href="#js-data-proof3">Substrate Analysis</A></LI><LI><A href="#js-data-proof4">Activity and kinetics of MnCcP</A></LI><LI><A href="#js-data-proof5">Suppelementary materials</A></LI><LI><A href="#js-data-proof6">Extension</A></LI><LI><A href="#js-data-part2">PART 2</A></LI><LI><A href="#js-data-proof2-1">Selection of TA modules</A></LI><LI><A href="#js-data-proof2-2">In silica: Choice of RBS</A></LI><LI><A href="#js-data-proof2-3">Clone and Identification of toxins and antitoxins</A></LI><LI><A href="#js-data-proof7">References</A></LI><A class="back-to-top" href="#top">
                        Back to top
                    </A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>