<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tokyo_Tech_Experiment_Overview_of_fim_inversion_system skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tokyo Tech/Experiment/Overview of fim inversion system</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="titlearea"><H1>Overview of <I>fim</I> inversion system</H1></DIV><DIV class="textarea"><H2 class="content2">contents</H2><H3 class="link"><A href="#Introduction">1. Introduction</A></H3><H3 class="link"><A href="#How">2. How does <I>fim</I> switch works</A></H3><H3 class="link2"><A href="#Wild">2.1. The <I>fim</I> switch(wild-type)</A></H3><H3 class="link2"><A href="#Tokyofim">2.2. The <I>fim</I> switch(Tokyo_Tech)</A></H3><H3 class="link3"><A href="#wdesign">2.2.1. Results of the FimB/FimE assay on the <I>fim</I> switch(Tokyo_Tech)</A></H3><H3 class="link"><A href="#Comparison">3. Comparison of other teams’ <I>fim</I> related parts</A></H3><H3 class="link2"><A href="#T31">3.1. FimB(2013_Toronto)</A></H3><H3 class="link2"><A href="#32">3.2. <I>fim</I> switch(2006_Caltech)-fimE(2006_Caltech)</A></H3><H3 class="link2"><A href="#33">3.3. <I>fim</I> switch(2013_Michigan)-fimE(2006_Caltech)-hbiF(2012_Michigan)</A></H3><H3 class="link"><A href="#Reference">4. Reference</A></H3></DIV><DIV class="textarea"><H2 id="Introduction" class="smalltitle">1. Introduction</H2><TABLE width="940 px" border="0px"><TBODY><TR><TD width="470px"><P class="text">FimB inverts the <I>fim</I> switch in [ON] state to [OFF] state and in [OFF] state to [ON] state with approximately equal probability. FimB enable prisoner coli to select its option (cooperation or defection) randomly (Fig.3-3-1-1.).  We showed bidirectional inversion of <I>fim</I> switch by FimB. This is the first case in iGEM to show random inversion of a promoter. </P></TD></TR><TR><TD width="940px"> </TD><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig.3-3-1-1.</STRONG>In the presence of FimB, promoter in <I>fim</I> switch can be inverted at random.</H4></TD></TR><TR><TD width="470px"><P class="text">On the other hand, FimE inverts the <I>fim</I> switch predominantly in the [ON] state to [OFF] state (McClain, 1991). For implementation of tit-for-tat strategy, the most successful strategy for Prisoner’s Dilemma, we used the FimE (Fig. 3-3-1-2.).</P></TD></TR><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig.3-3-1-2.</STRONG>Prisoner A with tit-for-tat strategy</H4></TD></TR><TR><TD width="470px"><P class="text">Our biggest achievement is that we have established a tripartite relationship between the three <I>fim</I> related parts, <I>fim</I> switch and FimB/FimE (Fig. 3-3-1-3).</P></TD></TR><TR><TD width="940px"> </TD><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig.3-3-1-3.</STRONG>Overview of the <I>fim</I> related parts in iGEM</H4></TD></TR></TBODY></TABLE><H2 id="How" class="smalltitle">2. How does <I>fim</I> switch works</H2><TABLE width="940 px" border="0px"><TBODY><TR><TD width="470px"><P class="text">In the wild-type of E.coli K-12, two fim proteins invert the <I>fim</I> switch, which is a specific 314 bp DNA sequence containing a promoter, to modulate its own expressions (Abraham, 1985).  The <I>fim</I> switch has two states, ON and OFF.  The promoter in the <I>fim</I> switch directs transcription to the right when the <I>fim</I> switch is in [ON] state.  In [OFF] state of the <I>fim</I> switch, on the other hand, the promoter directs transcription to the left.  </P></TD></TR><TR><TD width="940px"> </TD><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig.3-3-2-1.</STRONG>The relationship between the <I>fim</I> related genes.</H4></TD></TR></TBODY></TABLE><P class="text">The <I>fim</I> switch is inverted by two recombinases, FimB(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632010">BBa_K1632010</A>) and FimE(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632011">BBa_K1632011</A>). These proteins have distinct activities. FimB inverts the <I>fim</I> switch in [ON] state to [OFF] state direction and in [OFF] state to [ON] state direction with approximately equal probability.  On the other hand, FimE inverts the <I>fim</I> switch predominantly in [ON] state to [OFF] state direction (McClain, 1991). </P><P class="text">We designed two <I>fim</I> switches: the <I>fim</I> switch(wild-type) and the <I>fim</I> switch(Tokyo_Tech) which promoter can be exchanged with an arbitrary promoter.</P><P class="text">In the first <I>fim</I> switch, the <I>fim</I> switch(wild-type), the sequence is derived from that of a wild-type. The <I>fim</I> switch(wild-type) we constructed are named <I>fim</I> switch[default ON](wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632004">BBa_K1632004</A>) and <I>fim</I> switch[default OFF](wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632005">BBa_K1632005</A>). 
In the second <I>fim</I> switch, the <I>fim</I> switch(Tokyo_Tech), the promoter of the <I>fim</I> switch(wild-type) is replaced with J23119 promoter(BBa_J23119) and two restriction enzyme cut sites are added in each side of the promoter.  Due to this addition of the restriction enzyme cut sites, we were able to replace the J23119 promoter in the <I>fim</I> swtich(Tokyo_Tech).  For example, we replaced J23119 promoter with Lac promoter (<A href="http://parts.igem.org/Part:BBa_K1632006">BBa_K1632006</A>).  The <I>fim</I> switch(Tokyo_Tech) we constructed are named <I>fim</I> switch[default ON](Tokyo_Tech/J23119) (<A href="http://parts.igem.org/Part:BBa_K1632000">BBa_K1632000</A>), <I>fim</I> switch[default OFF](Tokyo_Tech/J23119) (<A href="http://parts.igem.org/Part:BBa_K1632000">BBa_K1632000</A>), and <I>fim</I> switch[default ON](Tokyo_Tech/B0010) (<A href="http://parts.igem.org/Part:BBa_K1632006">BBa_K1632006</A>).</P><H3 id="Wild" class="sub5">2.1 The <I>fim</I> switch(wild-type)</H3><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-2.</STRONG> The overview of the <I>fim</I> switch(wild-type)</H4></TD></TR></TBODY></TABLE><P class="text3">We designed the <I>fim</I> switch(wild-type). The sequence of our <I>fim</I> switch(wild-type) is derived from that of a wild-type.  The <I>fim</I> switch(wild-type) has a sigma 70 promoter which functions constitutively.  We submitted two parts, one in the [ON] state (<A href="http://parts.igem.org/Part:BBa_K1632004">BBa_K1632004</A>) and the other in [OFF] state (<A href="http://parts.igem.org/Part:BBa_K1632005">BBa_K1632005</A>).</P><P class="text3">We confirmed the inversion of our <I>fim</I> switch(wild-type) by FimB and by FimE.  Considering the experimental results, fimB(wild-type) inverts the <I>fim</I> switch(wild-type) in both [ON] state to [OFF] state and  [OFF] state to [ON] state with approximately equal probability.  Similarly FimE(wild-type) inverts the <I>fim</I> switch(wild-type) predominantly in [ON] state to [OFF] state. In order to confirm inversion more precisely, we also show the percentage of [ON] state with induction by arabinose and without induction. Also we show inversion in the level of DNA sequencing.</P><P class="text3">In summary, establishing a tripartite relationship between the <I>fim</I> switch and FimB/FimE is an unprecedented achievement in iGEM. </P><H3 id="tdesign" class="sub6">2.1.1. Result of the FimB/FimE assay on the <I>fim</I> switch(wild-type)</H3><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig.3-3-2-3.</STRONG> New plasmids we constructed to confirm the function of <I>fim</I> switch</H4></TD></TR></TBODY></TABLE><P class="text3">The <I>fim</I> switch is inverted by two recombinases, FimB(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632010">BBa_K1632010</A>) and FimE(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632011">BBa_K1632011</A>). These proteins have distinct activities. FimB inverts the <I>fim</I> switch in both [ON] state to [OFF] state and [OFF] state to [ON] state direction with approximately equal probability. On the other hand, FimE inverts the <I>fim</I> switch predominantly in [ON] state to [OFF] state (McClain, 1991). </P><P class="text3">In order to assay the function of FimB(wild-type) and FimE(wild-type) for <I>fim</I> switch(wild-type), we added a GFP coding sequence on the downstream of the <I>fim</I> switch.The <I>fim</I> switch[default ON](wild-type)_gfp (<A href="http://parts.igem.org/Part:BBa_K1632007">BBa_K1632007</A>) emitts fluorescence when expressed, while the <I>fim</I> switch [default OFF](wild-type)_gfp(<A href="http://parts.igem.org/Part:BBa_K1632008">BBa_K1632008</A>) does not emit florescence when expressed. We also added p<SUB>BAD/<I>araC</I></SUB> (<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>) on the upstream of both fimB(wild-type) and fimE(wild-type). p<SUB>BAD/<I>araC</I></SUB>_<I>fimB</I>(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632012">BBa_K1632012</A>) and p<SUB>BAD/<I>araC</I></SUB>_<I>fimE</I>(<A href="http://parts.igem.org/Part:BBa_K1632013">BBa_K1632013</A>) can induce the expression of FimB(wild-type) or FimE(wild-type) in the presence of arabinose. We co-transformed a <I>fim</I> switch_gfp and a p<SUB>BAD/<I>araC</I></SUB>_<I>fimB</I> or _<I>fimE</I> in the <I>E. coli</I> DH5alpha strain(Fig. 3-3-2-3.). We measured the fluorescence intensity of the cells induced by different concentraions of arabinose.
From the experimental results(Fig. 3-3-2-4), FimB(wild-type) inverted the <I>fim</I> switch[default ON](wild-type) from [ON] state to [OFF] state and the <I>fim</I> switch[defult OFF](wild-type) from [OFF] state to [ON] state, depending on the concentration of arabinose.  FimE(wild-type) inverted the <I>fim</I> switch[default ON](wild-type) from [ON] state to [OFF] state but did not invert the <I>fim</I> switch[default OFF](wild-type) from [OFF] state to [ON] state, depending on the concentration of arabinose.
</P><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig.3-3-2-4.</STRONG> The intensity of fluroscence in cells measured using flowcytometer.</H4></TD></TR></TBODY></TABLE><P class="text4"><STRONG>2.1.1.1 FimB dependent <I>fim</I> switch(wild-type) state assay</STRONG></P><TABLE width="800 px" border="0px"><TBODY><TR><TD width="400px"><P class="text">We newly constructed plasmid, P<SUB>BAD/<I>araC</I></SUB>_<I>fimB</I>(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632012">BBa_K1632012</A>) that produces FimB(wild-type).  We also prepared two other new plasmids, (<A href="http://parts.igem.org/Part:BBa_K1632007">BBa_K1632007</A> and (<A href="http://parts.igem.org/Part:BBa_K1632008">BBa_K1632008</A>(Fig. 3-5-1-2).  (<A href="http://parts.igem.org/Part:BBa_K1632012">BBa_K1632012</A> enables arabinose-inducible expression of FimB(wild-type).  In both [ON] state and [OFF] state, <I>fim</I> switch(wild-type) is placed upstream of GFP coding sequence. </P></TD></TR><TR><TD width="400px"> </TD><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig.3-3-2-5.</STRONG>New plasmids we constructed to confirm the function of fim switch.</H4></TD></TR><TR><TD width="400px"><P class="text">First, we measured the fluorescence intensity from the GFP expression in the presence of arabinose. From the results (Fig. 3-3-2-6.), we confirmed that the <I>fim</I> switch (wild-type) is inverted from both [ON] state to [OFF] state and [OFF] state to [ON] state. </P></TD></TR><TR><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-6.</STRONG>The intensity of fluroscence in cells measured using flowcytometer</H4></TD><TD width="400px"> </TD></TR><TR><TD width="400px" colspan="2"><P class="text4"> First, we measured the fluorescence intensity from the GFP expression in the presence of arabinose. From the results (Fig. 3-3-2-6.), we confirmed that the <I>fim</I> switch (wild-type) is inverted from both [ON] state to [OFF] state and [OFF] state to [ON] state. </P><P class="text4"> Next, in order to confirm inversion more precisely, we also show the percentage of ON state (Fig. 3-3-2-7) and inversion in the level of DNA sequencing (Fig. 3-3-2-8).  Fig.3-3-2-7 and Fig.3-3-2-8 clearly shows inversion from both [ON] state to [OFF] state and [OFF] state to [ON] state.  <I>E. coli</I> harboring the [ON] <I>fim</I> switch plasmid glows in green fluorescence while <I>E. coli</I> harboring the [OFF] <I>fim</I> switch plasmid does not glow under ultra violet light.  Cells transformed by the plasmid mixture extracted from the experiment formed two types of colonies: those with strong fluorescence and those with little background fluorescence.</P><P class="text4"> Also, sequence complementarity in the specific region of the <I>fim</I> switch(wild-type) shows intended inversion of the <I>fim</I> switch from [ON] state to [OFF] state.  Furthermore, the nearly equal number of the colonies between fluorescent and non-fluorescent suggest random inversion of the <I>fim</I> switch by FimB expression.</P></TD></TR><TR><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-7.</STRONG>  Determination of percetage of ON state and colony formation using plasmid mixture extracted cell expressing FimB.</H4></TD><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-8.</STRONG>  DNA sequencing results of <I>fim</I> switch from [ON] state to [OFF] state</H4></TD></TR></TBODY></TABLE><P class="text4">For more information, see <A href="https://2015.igem.org/Team:Tokyo_Tech/Experiment/FimB_dependent_fim_switch_state_assay">experiment page.</A></P><P class="text4"><STRONG>2.1.1.2 FimE dependent <I>fim</I> switch(wild-type) state assay</STRONG></P><TABLE width="800 px" border="0px"><TBODY><TR><TD width="400px"><P class="text">To confirm the function of <I>fim</I> switch(wild-type) in the presence of FimE(wild-type), we constructed Biobrick parts, (<A href="http://parts.igem.org/Part:BBa_K1632013">BBa_K1632013</A> as Fig. 3-3-2-9.  (<A href="http://parts.igem.org/Part:BBa_K1632013">BBa_K1632013</A> enables arabinose-inducible expression of the FimE (wild-type).  In (<A href="http://parts.igem.org/Part:BBa_K1632007">BBa_K1632007</A> and (<A href="http://parts.igem.org/Part:BBa_K1632008">BBa_K1632008</A>, either the <I>fim</I> switch [default ON] or the <I>fim</I> switch [default OFF] is placed upstream of the GFP coding sequence. </P></TD></TR><TR><TD width="400px"> </TD><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig.3-3-2-9.</STRONG>New plasmids we constructed to confirm the function of fim switch.</H4></TD></TR><TR><TD width="400px"><P class="text">Our purpose is to confirm that FimE (wild-type) inverts the <I>fim</I> switch (wild-type) from [ON] state to [OFF] state predominantly. First of all, we measured the fluorescence intensity from the GFP expression in the presence of arabinose. From the results (Fig. 3-3-2-10), we confirmed that our <I>fim</I> switch(wild-type) is inverted from [ON] state to [OFF] state predominatly.</P></TD></TR><TR><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-10.</STRONG>The intensity of fluroscence in cells measured using flowcytometer</H4></TD><TD width="400px"> </TD></TR><TR><TD width="400px" colspan="2"><P class="text4"> Next, in order to confirm inversion more precisely, we also show the percentage of ON state (Fig. 3-3-2-11) and inversion in the level of DNA sequencing (Fig. 3-3-2-12).  Fig. 3-3-2-11 shows inversion from [ON] state to [OFF] state predominantly.  <I>E. coli</I> harboring the [ON] <I>fim</I> switch(wild-type) plasmid glows in green fluorescence while <I>E. coli</I> harboring the [OFF] <I>fim</I> switch(wild-type) plasmid does not glow under ultra violet light.  Cells transformed by the plasmid mixture extracted from the experiment formed two types of colonies: those with strong fluorescence and those with little background fluorescence.  Also, sequence complementarity in the specific region of the switch shows intended inversion of the <I>fim</I> switch(wild-type) from [ON] state to [OFF] state. Considering the percentage of [ON] state, FimE(wild-type) inverts the <I>fim</I> switch predominantly in [ON] state to [OFF] state.</P><P class="text4"> </P></TD></TR><TR><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-11.</STRONG>  Determination of percetage of ON state and colony formation using plasmid mixture extracted cell expressing FimE.</H4></TD><TD width="400px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-12.</STRONG> DNA sequencing results of <I>fim</I> switch.</H4></TD></TR></TBODY></TABLE><P class="text4">For more information, see <A href="https://2015.igem.org/Team:Tokyo_Tech/Experiment/FimE_dependent_fim_switch_state_assay">experiment page.</A></P><H3 id="Tokyofim" class="sub5">2.2. The <I>fim</I> switch(Tokyo_Tech)</H3><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-13.</STRONG>  The overview of <I>fim</I> switch(Tokyo_Tech)</H4></TD></TR></TBODY></TABLE><P class="text2">We designed a <I>fim</I> switch, which promoter can be exchanged with an arbitrary promoter, the <I>fim</I>  switch(Tokyo_Tech) (Fig. 3-3-2-13). </P><P class="text2">We could not confirm the inversion of the <I>fim</I> switch(Tokyo_Tech) by FimE(wild-type).  From the experimental results, FimB(wild-type) inverts the <I>fim</I> switch in both [ON] state to [OFF] state and [OFF] state to [ON] state as expected.  However, FimE(wild-type) unexpectedly did not invert the <I>fim</I> switch(Tokyo_Tech) predominantly in [ON] state to [OFF] state.  Instead, FimE(wild-type) inverted the <I>fim</I> switch(Tokyo_Tech) in both [ON] state to [OFF] state and in [OFF] state to [ON] state. In other words, FimE(wild-type) worked in the same way as FimB(wild-type).</P><P class="text2">Basically, the design of the <I>fim</I> switch(Tokyo_Tech) is similar to the <I>fim</I> switch(wild-type). The only difference is that we inserted restriction cut enzyme sites to <I>fim</I> switch(Tokyo_Tech). In detail, in <I>fim</I> switch(Tokyo_Tech), the sigma 70 promoter is exchanged to the J23119 promoter(<A href="http://parts.igem.org/Part:BBa_J23119">BBa_J23119</A>) and there are two restriction enzyme cut sites inserted each in the front (SalI and BamHI) and in the back (BglII and MluI) of the promoter. Due to the insertion of the restriction enzyme cut sites, <I>fim</I> switch(Tokyo_Tech) has a promoter which can we exchanged with an arbitrary promoter. We actually changed J23119 promoter in the <I>fim</I> switch(Tokyo_Tech) to Lac promoter(<A href="http://parts.igem.org/Part:BBa_K1632006">BBa_K1632006</A>) as shown in the Fig. 3-3-2-14. 
</P><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-2-14.</STRONG> The flow of changing the promoter in <I>fim</I> switch(Tokyo_Tech)</H4></TD></TR></TBODY></TABLE><H3 id="tFimB/FimE" class="sub6">2.2.1. Results of the FimB/FimE assay on the <I>fim</I> switch(Tokyo_Tech)</H3><P class="text3">As with assays of <I>fim</I> swtich(wild-type), in order to assay the function of FimB(wild-type) and FimE(wild-type) for <I>fim</I> switch(Tokyo_Tech), we added a GFP coding sequence on the downstream of the <I>fim</I> switch(Tokyo_Tech). The <I>fim</I> switch[default ON](Tokyo_Tech/J23119)_gfp (<A href="http://parts.igem.org/Part:BBa_K1632003">BBa_K1632003</A>) emitts fluorescence when expressed, while the <I>fim</I> switch [default OFF](Tokyo_Tech/J23119)_gfp(<A href="http://parts.igem.org/Part:BBa_K1632004">BBa_K1632004</A>) does not emit florescence when expressed. We also added p<SUB>BAD/<I>araC</I></SUB>(<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>) on the upstream of both FimB(wild-type) and FimE(wild-type). p<SUB>BAD/<I>araC</I></SUB>_<I>fimB</I> (<A href="http://parts.igem.org/Part:BBa_K1632012">BBa_K1632012</A>) and p<SUB>BAD/<I>araC</I></SUB>_<I>fimE</I>(<A href="http://parts.igem.org/Part:BBa_K1632013">BBa_K1632013</A>) can induce the expression of FimB(wild-type) or FimE(wild-type) in the presence of arabinose. We co-transformed a <I>fim</I> switch(Tokyo_Tech)_gfp and a p<SUB>BAD/<I>araC</I></SUB>_<I>fimB</I> or _<I>fimE</I> in the <I>E. coli</I> DH5alpha strain. We measured the fluorescence intensity of the cells induced by different concentraions of arabinose.
We could not confirm the inversion of the <I>fim</I> switch(Tokyo_Tech) by FimE(wild-type). From the experimental results, FimB(wild-type) inverts the <I>fim</I> switch(Tokyo_Tech) in both [ON] state to [OFF] state  and [OFF] state to [ON] state as expected. However, FimE(wild-type) unexpectedly did not invert the <I>fim</I> switch(Tokyo_Tech) predominantly in [ON] state to [OFF] state. Instead, FimE(wild-type) inverted the <I>fim</I> switch(Tokyo_Tech) in both [ON] state to [OFF] state and [OFF] state to [ON] state. In other words, FimE(wild-type) worked in the same way as FimB(wild-type).</P><P class="text3">For more information, see the parts registry page(<A href="https://2015.igem.org/Team:Tokyo_Tech/Experiment/FimB_dependent_fim_switch_state_assay">FimB_dependent_fim_switch_state_assay</A>).</P><H2 id="Comparison" class="smalltitle">3. Comparison other team’s <I>fim</I> switch</H2><P class="text">Our biggest achievement is that we have established a tripartite relationship between the three <I>fim</I> related parts, the <I>fim</I> switch and FimB/FimE.</P><P class="text">In this chapter, the novelty of constructing and assaying all three <I>fim</I> related parts (the <I>fim</I> switch, FimB, and FimE) is shown by comparison to the past iGEM teams’ <I>fim</I> related parts.  Some teams have submitted <I>fim</I> related parts.  However the information of the <I>fim</I> related parts are so complicated that we felt that it is necessary to clarify the situation. </P><P class="text">These submitted parts which show some data, are divided into three groups as shown in the figure below (Fig. 3-3-3-1.)</P><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-1.</STRONG> All past iGEM teams’ <I>fim</I> related parts.</H4></TD></TR></TBODY></TABLE><P class="text">From the results of each group, we evaluated each group’s data as shown below. 
From these evaluations, we concluded that our establishment of a tripartite relationship between the <I>fim</I> switch and FimB/FimE, is an unprecedented achievement in iGEM.</P><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-2.</STRONG> Summary of the <I>fim</I> related parts in iGEM.</H4></TD></TR></TBODY></TABLE><H3 id="31" class="sub5">3.1. fimB (2013 Toronto) : No data about inversion</H3><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-3.</STRONG> Overview of the <I>fim</I> related parts in iGEM.</H4></TD></TR></TBODY></TABLE><P class="text2"><STRONG>FimB</STRONG></P><TABLE width="90%" border="2" style="border-collapse: collapse;"><TBODY><TR height="20px"><TH width="20%">Gene Name</TH><TH width="10%">Year</TH><TH width="10%">College</TH><TH width="20%">Parts Number</TH><TH width="30%">Sequence Confirmation</TH></TR><TR height="20px"><TD width="20%">FimB</TD><TD width="10%">2013</TD><TD width="10%">Toronto</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K1019000">BBa_K1019000</A></TD><TD width="30%">None</TD></TR></TBODY></TABLE><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-4.</STRONG> Lists of submitted <I>fim</I> parts.</H4><P class="text">2013_Toronto submitted fimB (<A href="http://parts.igem.org/Part:BBa_K1019000">BBa_K1019000</A>) and showed some data. However, their data clearly did not relate to the inversion of <I>fim</I> switch. Their experimental result is to search the difference of certain parameters like OD600, in the condition of FimB (+) and FimB (-). Therefore, it can be concluded that the results of 2013_Toronto did not show any data referring to the inversion of the <I>fim</I> switch. </P><P class="text">For detailed information, see <A href="https://2013.igem.org/Team:Toronto">Team:Toronto</A></P><H3 id="32" class="sub5">3.2. <I>fim</I> switch(2008_Caltech)-fimE(2008_Caltech) : No data about inversion</H3><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-5.</STRONG> Overview of the <I>fim</I> related parts in iGEM.</H4></TD></TR></TBODY></TABLE><P class="text2"><STRONG>FimE</STRONG></P><TABLE width="90%" border="2" style="border-collapse: collapse;"><TBODY><TR height="20px"><TH width="20%">Gene Name</TH><TH width="10%">Year</TH><TH width="10%">College</TH><TH width="20%">Parts Number</TH><TH width="30%">Sequence Confirmation</TH></TR><TR height="20px"><TD width="20%">FimE</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A></TD><TD width="30%">None</TD></TR></TBODY></TABLE><P class="text2"><STRONG><I>fim</I> switch</STRONG></P><TABLE width="90%" border="2" style="border-collapse: collapse;"><H3>Gene Name</H3><TBODY><TR><TH width="10%">Year</TH><TH width="10%">College</TH><TH width="20%">Parts Number</TH><TH width="30%">Sequence Confirmation</TH></TR><TR height="20px"><TD width="20%">FimE IRR</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137008">BBa_K137008</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">FimE IRL</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137010">BBa_K137010</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">GFP fimE switch with 150 bp tetR promoter</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137057">BBa_K137057</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">GFP fimE switch with 250 bp tetR promoter</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137058">BBa_K137058</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">GFP fimE switch with 350 bp tetR promoter</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137059">BBa_K137059</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">GFP fimE switch with 450 bp tetR promoter</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137060">BBa_K137060</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">GFP fimE switch with 550 bp tetR promoter</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137061">BBa_K137061</A></TD><TD width="30%">None</TD></TR><TR height="20px"><TD width="20%">GFP fimE switch with 650 bp tetR promoter</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137062">BBa_K137062</A></TD><TD width="30%">None</TD></TR></TBODY></TABLE><H4 align="center" class="fig"><STRONG>Fig.3-3-3-6.</STRONG>Lists of submitted <I>fim</I> parts.</H4><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-7.</STRONG>  The experimental result of 2008_Caltech</H4></TD></TR></TBODY></TABLE><P class="text2">2008_Caltech only submitted fimE (<A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A>) and showed data about inversion (Fig. 3-3-3-7.). 2008_Caltech wanted to determine the range of length in which fimE (<A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A>) will function and to determine how the length affects the activities of FimE. Thus they tested how the length of the recombination site affects FimE’s ability to detect the two flanking binding sites. Therefore, GFP fluorescence / OD600 was observed in each sample in which the length of the inversion site differs. </P><P class="text2">However their data cannot be trusted completely with respect to inversion of <I>fim</I> switch. There are several reasons. First of all, they did not set Positive controls and Negative controls. Because of the lack of controls, we cannot confirm that FimE inverts the <I>fim</I> switch in [ON] state to [OFF] state predominantly. Secondly, they didn’t write down the information of plasmids used in the experiments. 
The nucleotide sequence of 2008_Caltech’s fimE (<A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A>) and FimE(wild type) which was obtained from <A href="http://www.ecogene.org/old/geneinfo.php?eg_id=EG10312) ">Ecogene</A> did not match. Therefore, we designed FimE(wild-type) (<A href="http://parts.igem.org/Part:BBa_K1632010">BBa_K1632010</A>) ) whose nucleotide sequence completely matchs that of the wild type’s fimE. Comparing these two parts, two differences were confirmed. First, FimE of 2008_Caltech lacks the nucleotide sequence of N-terminal 15 residues. Second, FimE of 2008_Caltech differs from the nucleotide sequence of FimE(wild-type) by C-terminal 2 residues as shown in Fig. 3-3-3-8.
</P><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-8.</STRONG> The difference of nucleotide sequence between <A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A>) and <A href="http://parts.igem.org/Part:BBa_K1632011">BBa_K1632011</A>) .</H4></TD></TR></TBODY></TABLE><P class="text2">For detailed information, see <A href="https://2008.igem.org/Team:Caltech">2008_Caltech wiki</A>) </P><H3 id="33" class="sub5">3.3. <I>fim</I> switch(2013_Michigan)-fimE(2008_Caltech)-hbiF(2012_Michigan) : Not enough data about inversion</H3><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-9.</STRONG> Overview of the <I>fim</I> related parts in iGEM.</H4></TD></TR></TBODY></TABLE><P class="text2"><STRONG>FimE</STRONG></P><TABLE width="90%" border="2" style="border-collapse: collapse;"><TBODY><TR height="20px"><TH width="20%">Gene Name</TH><TH width="10%">Year</TH><TH width="10%">College</TH><TH width="20%">Parts Number</TH><TH width="30%">Sequence Confirmation</TH></TR><TR height="20px"><TD width="20%">FimE</TD><TD width="10%">2008</TD><TD width="10%">Caltech</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A></TD><TD width="30%">None</TD></TR></TBODY></TABLE><P class="text2"><STRONG>HbiF</STRONG></P><TABLE width="90%" border="2" style="border-collapse: collapse;"><TBODY><TR height="20px"><TH width="20%">Gene Name</TH><TH width="10%">Year</TH><TH width="10%">College</TH><TH width="20%">Parts Number</TH><TH width="30%">Sequence Confirmation</TH></TR><TR height="20px"><TD width="20%">HbiF</TD><TD width="10%">2012</TD><TD width="10%">Michigan</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K88000">BBa_K88000</A></TD><TD width="30%">None</TD></TR></TBODY></TABLE><P class="text2"><STRONG><I>fim</I> switch</STRONG></P><TABLE width="90%" border="2" style="border-collapse: collapse;"><TBODY><TR height="20px"><TH width="20%">Gene Name</TH><TH width="10%">Year</TH><TH width="10%">College</TH><TH width="20%">Parts Number</TH><TH width="30%">Sequence Confirmation</TH></TR><TR height="20px"><TD width="20%"><I>fim</I> switch inverted repeat left IRL natural</TD><TD width="10%">2012</TD><TD width="10%">Michigan</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K1077000">BBa_K1077000</A></TD><TD width="30%">Inconsistent</TD></TR><TR height="20px"><TD width="20%"><I>fim</I> switch inverted repeat left IRR natural</TD><TD width="10%">2012</TD><TD width="10%">Michigan</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K1077001">BBa_K1077001</A></TD><TD width="30%">Inconsistent</TD></TR><TR height="20px"><TD width="20%">J23100 <I>fim</I> switch b0034 GFP</TD><TD width="10%">2012</TD><TD width="10%">Michigan</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K1077003">BBa_K1077003</A></TD><TD width="30%">Inconsistent</TD></TR><TR height="20px"><TD width="20%">J23100 <I>fim</I> switch b0034 GFP</TD><TD width="10%">2012</TD><TD width="10%">Michigan</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K1077005">BBa_K1077005</A></TD><TD width="30%">Inconsistent</TD></TR><TR height="20px"><TD width="20%">J23100 <I>fim</I> switch b0034 amilCP ON orientation</TD><TD width="10%">2012</TD><TD width="10%">Michigan</TD><TD width="20%"><A href="http://parts.igem.org/Part:BBa_K1077007">BBa_K1077007</A></TD><TD width="30%">Inconsistent</TD></TR></TBODY></TABLE><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-10.</STRONG>Lists of submitted <I>fim</I> parts.</H4><TABLE width="940 px" border="0px"><TBODY><TR><TD width="940px"><H4 align="center" class="fig"><STRONG>Fig. 3-3-3-11.</STRONG> Overview of the <I>fim</I> related parts in iGEM.</H4></TD></TR></TBODY></TABLE><P class="text2">The characteristic of the part (<A href="http://parts.igem.org/Part:BBa_K1077002">BBa_K1077002</A>) ) constructed by Michigan, is that it can be inverted in [ON] state to [OFF] state direction or in [OFF] state to [ON] state done-sidedly with induction of aTc or AHL. The repression of the tet promoter by the tet repressor is weakened by the addition of aTc. Therefore, fimE (<A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A>) ), which is on the downstream of the tet promoter is expressed by inducing aTc. As a result, the <I>fim</I> switch is inverted from [ON] state to [OFF] state. In other words, the target gene which is on the downstream of the <I>fim</I> switch, is not expressed. </P><P class="text2">On the other hand, by the addition of AHL, lux promoter is activated. Therefore, hbiF (<A href="http://parts.igem.org/Part:BBa_K880000">BBa_K880000</A>) ), which is on the downstream of the lux promoter, is expressed. HbiF inverts the <I>fim</I> switch in [OFF] state to [ON] state. As a result, the target gene which is on the downstream of the <I>fim</I> switch, is expressed. </P><P class="text2">There are two immense issues to their system, judging from their experimental results.
First of all, we could not conclude that their <I>fim</I> switch and fimE is functioning correctly, because the results were only shown for the <I>fim</I> switch[default ON] in their wiki and parts registry. In other words, the data of <I>fim</I> switch[default OFF] was not shown. Without the data of <I>fim</I> switch[default OFF], we can’t conclude that FimE (<A href="http://parts.igem.org/Part:BBa_K137007">BBa_K137007</A>) ) inverts the <I>fim</I> switch in [ON] state to [OFF] state predominantly. </P><P class="text2">Second of all, the results of Team Michigan, lack reproducibility because they have failed to set Positive controls and Negative controls in all of their experiments.</P><P class="text2"> It was impossible to verify the reproducibility because the nucleotide sequence of this part (<A href="http://parts.igem.org/Part:BBa_K1077002">BBa_K1077002</A>) ) was completely different between the distribution from the HQ and the information shown on the parts registry.</P><P class="text2"> For detailed information, see <A href="https://2013.igem.org/Team:Michigan">2013_Michigan wiki</A>).</P><H2 id="Reference" class="smalltitle">4. Reference</H2><P class="text4">1. McClain MS <I>et al</I>. (1991) Roles of fimB and fimE in site-specific DNA inversion associated with phase variation of type 1 fimbriae in Escherichia coli. J Bacteriol 173(17):5308-14.
2. John M. Abraham et al. (1985) An invertible element of DNA controls phase variation of type 1 fimbriae of Escherichia coli. Proc Natl Acad Sci USA 82(17):5724-7
3. Ian C. Blomfield <I>et al</I>. (1997) Integration host factor stimulates both FimB- andFimE-mediated site-specific DNA inversion that controls phase variation of type 1 fimbriae expression in Escherichia coli. Molecular Microbiology 23(4), 705–717
4. Matthew P. McCusker <I>et al</I>. (2008) DNA sequence heterogeneity in Fim tyrosine-integrase recombinase-binding elements and functional motif asymmetries determine the directionality of the fim genetic switch in Escherichia coli K-12. Molecular Microbiology 67(1): 171–187
5. D L Gally <I>et al</I>. (1993) Environmental regulation of the fim switch controlling type 1 fimbrial phase variation in Escherichia coli K-12: effects of temperature and media. J Bacteriol. Oct; 175(19): 6186–6193.</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>