<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pitt_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pitt/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer" class="floatyMenu"><TABLE><TBODY><TR><TD><A href="https://2015.igem.org/Team:Pitt">Home</A></TD><TD><A href="https://2015.igem.org/Team:Pitt/Description">Project</A><UL><LI><A href="https://2015.igem.org/Team:Pitt/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Pitt/Estrogen/Project">Estrogen Sensor</A></LI><LI><A href="https://2015.igem.org/Team:Pitt/Protease/Project">Protease Sensor</A></LI><LI><A href="https://2015.igem.org/Team:Pitt/3-hybrid/Project">Three-Hybrid Versatile Sensor</A></LI><LI><A href="https://2015.igem.org/Team:Pitt/Amplification/Project">Amplification and Quenching</A></LI></UL></TD><TD><A href="https://2015.igem.org/Team:Pitt/Notebook">Notebook</A><UL><LI><A href="https://2015.igem.org/Team:Pitt/Protocols">Sensor Extract Protocol</A></LI><LI><A href="https://2015.igem.org/Team:Pitt/Notebook">Daily Notebook</A></LI></UL></TD><TD><A href="https://2015.igem.org/Team:Pitt/Parts">Parts</A></TD><TD><A href="https://2015.igem.org/Team:Pitt/Practices">Human Practices</A></TD><TD><A href="https://2015.igem.org/Team:Pitt/Team">Team</A><UL><LI><A href="https://2015.igem.org/Team:Pitt/Team">Meet us!</A></LI><LI><A href="https://2015.igem.org/Team:Pitt/Attributions">Attributions</A></LI></UL></TD><TD><A href="https://2015.igem.org/Team:Pitt/Sponsors">Sponsors</A></TD><TD><A href="https://igem.org/Main_Page">2015 iGEM Home</A></TD></TR></TBODY></TABLE></DIV><DIV id="contentContainer"><H2>Cheap biosensors based on cell-free extracts</H2><P>
Pitt's 2015 iGEM team investigated the possibility of using cell-free extracts as a method of detecting extremely small concentrations of biomolecules. Furthermore, these sensors were tested after being freeze-dried on paper, to test the feasibility of mass-producing and distributing cheap sensors worldwide. While the ideas tested in this project can be applied to sense an almost unlimited number of biomolecules, we focused on three major sensors.</P><TABLE id="ptable"><TBODY><TR><TD colspan="6" class="td100"><H4>Project Idea and Motivation</H4> 
The ability to detect small concentrations of molecules accurately without the use of lab equipment is a huge step in creating portable sensing devices. While many extremely sensitive methods have been developed, there are very few that are amenable to work in the field, or at home as a consumer product. This project focuses on creating methods that allow for simple and quick detection of biomolecules without the use of laboratory instruments. <DIV><BUTTON class="expander">Click to read more...</BUTTON></DIV><SPAN style="display: none;">Recently, it was shown that freeze-drying cell extracts on paper has little effect on the ability of the cell extract to perform <I>in vitro</I> transcription and translation.(<A href="http://www.cell.com/abstract/S0092-8674(14)01291-4">Pardee 2014</A>) Since paper strips are simple to transport and store, we chose this media to create our sensors. This method has the added advantage of utilizing the natural amplification of transcription and translation to visualize incredibly small concentrations without the use of complicated instrumentation. Furthermore, results can be available in as little as an hour (<A href="http://www.cell.com/abstract/S0092-8674(14)01291-4">Pardee 2014</A>), which is significantly quicker than most routine lab tests. Based on these ideas, out final goal was to create a sensor that would function similarly to the at-home pregnancy test. While this project focuses on a proof-of-concept, the system is designed to be extremely versatile, so three different sensors were conceptualized. <DIV><BUTTON class="minimizer">Click to learn more about each individual sensor...</BUTTON></DIV></SPAN></TD></TR><TR><TD colspan="3" class="td50"><H4>Cell-free Extracts</H4>The use of cell-free extracts for sensors allows us to solve several problems at once. First of all, by using the natural amplification of both transcription and translation <I>in vitro</I>, extremely small amounts of analyte can be detected. Furthermore, it was recently shown that these cell extracts retain their function when freeze-dried on paper, which allows for easy transport of the completed sensor.(<A href="http://www.cell.com/abstract/S0092-8674(14)01291-4">Pardee 2014</A>)<DIV><BUTTON class="expander">Click to read more...</BUTTON></DIV><SPAN style="display: none;">Cell-free extracts pose a significant advantage over cell-based assays in that synthetic gene networks can easily be used without extensive cloning, as shown above. At the most basic level, cell-free extracts are essentially lysed cells. In fact, it was in cell-free extracts that the genetic code was discovered, as it allowed for easy control of translation. Since 1953, significant advancements have been made in this technology, providing extremely fine control over both transcription and translation. In its simplest form, a cell-free extract made from <I>E. coli</I> will contain all the components of the prokaryotic cell, except for the cell wall, the membrane, and the chromosomal DNA attached to the membrane. While this extract will be able to both transcribe DNA, and translate the resulting RNA, both processes occur at very low rates. During cell growth, a cell will produce and reuse a variety of small molecules, including but not limited to nucleotide triphosphates (NTPs) and amino acids. Both of these are essential to <I>in vitro</I> transcription and translation. However, once the chromosomal DNA is removed, the extract retains very little of its ability to create small molecules and create high energy molecules from energy sources such as glucose. With this realization, a number of improvements to the cell-free extract were made, by adding the necessary components to the final reaction mixture. The amino acids and NTPs are added in great excess, which not only provides the starting materials for RNA and protein synthesis, but also increases the thermodynamic drive for the synthesis of macromolecules. In addition, creatine kinase is added along with creatine phosphate as a way of regenerating high energy molecules such as ATP from ADP. After these modifications to the cell-free extract protocol, there was about a (number) increase in the rate of production of protein from extracts. However, it was soon realized that the majority of ribosomes were still bound to mRNAs from the lysed cell, some of which stalled during translation, which diverted some of the energy from producing the protein of interest to producing short peptides and other proteins. To counteract this source of contamination, an additional incubation step was added, followed by a dialysis step to free as many of the ribosomes as possible. Th final modifications to the currently accepted cell-free extract protocol were made in 2006 (<A href="http://www.ncbi.nlm.nih.gov/pubmed/16487613">Kim 2006</A>), and our group used a similar method. However, the goal of most groups is to produce as much protein as possible, for which they require an extract as free of other proteins as possible. For the purpose of this project, proteins that regulate transcription are needed in the cell extract. One way of achieving this is by purifying the protein from a cell extract, then adding to the final reaction mixture. However, a simpler approach is to induce the protein of interest directly in the cells from which the extract is made. By choosing this approach, the number of steps required for the production of our &quot;sensor extracts&quot; (SE) was greatly reduced. Currently, the entire production process requires 3 days, and the final cost of a single sensor is less than 50 cents. To see more, click <A href="2015.igem.org/Team:Pitt/Protocols">here</A>. For sensor extracts, it is typical to include proteins that allow for the modulation of transcriptional activation in the presence of an analyte, while the final reaction mix contains a DNA construct that contains the appropriate promoter followed by a reporter gene (typically a fluorescent protein such as GFP). <DIV><BUTTON class="minimizer">Click to read about other aspects of the project...</BUTTON></DIV></SPAN></TD><TD colspan="3" class="td50"><H4>Clear Responses: Amplification and Quenching</H4>One of the key aspects of creating a reliable and useful sensor is having a clear yes/no response. A good example of this is the at-home pregnancy test, where 2 lines indicates pregnancy, and 1 line indicates no pregnancy. While it is impossible to remove <I>all</I> outliers, we have been working on a system that will amplify positive signals, while quenching noise. <DIV><BUTTON class="expander">Click to read more...</BUTTON></DIV><SPAN style="display: none;">A huge advantage of using cell extracts is the possibility to create synthetic gene circuits that can modulate the response of the sensor. For this project, we created a rather simple circuit that accounts for noise, while amplifying positive signals. All three sensors that we are building rely on transcriptional activation. Thus, provided that each circuit outputs the same protein at the end of one transcription/translation cycle, the same modular amplification and quenching circuit can be applied to all 3.  As shown above, we have chosen the output from the first system to be the T3 phage RNA polymerase. By using this polymerase (which has a different promoter than any of the systems we have designed), we can amplify the signal several times through a positive feedback construct that produces more T3 RNAP in the presence of T3 RNAP. Finally, by including a DNA construct that produces a reporter (typically GFP for simple measurement by a fluorimeter), the result of the amplification can be seen. Unfortunately, this circuit also amplifies noise extremely well, which typically occurs due to leaky expression of the sensor promoter. To counteract this, a quencher was designed that binds to T3 RNAP and blocks the polymerase from transcribing, and ultimately from amplifying the noise. The modular design of the circuit allows for careful fine-tuning for each system. By increasing the amount of amplification construct, lower levels of detection can potentially be achieved, while increasing the amount of quencher can account for a larger amount of leaky transcription. To learn more about this sub-project, click <A href="2015.igem.org/Team:Pitt/Amplification/Project">here</A>.<DIV><BUTTON class="minimizer">Click to read about other aspects of the project...</BUTTON></DIV></SPAN></TD></TR><TR><TD colspan="2" class="td33"><H4>Estrogen Sensor</H4>While it has been shown that transcription in cell-free extracts can rely on RNA polymerases sensitive to small molecules (<A href="http://www.cell.com/abstract/S0092-8674(14)01291-4">Pardee 2014</A>), our team decided to test the viability of using such polymerases in our cheap, home-made sensor extracts, rather than in expensive, commercially available extracts. In doing so, we used a part from CMU's iGEM team, the estrogen-sensitive T7 RNA polymerase.<DIV><BUTTON class="expander">Click to read more...</BUTTON></DIV><SPAN style="display: none;">This subproject was the first and simplest application of our sensor extracts. Since CMU's iGEM team had been working on an estrogen-sensitive mutant of T7 RNA Polymerase, we decided to use the construct in a cell-free context. Sensors that detect estrogen quickly could be used in a variety of contexts, including but not limited to quantifying estrogen in blood, and detecting estrogen-contaminated water. The sensor would work roughly as shown above. In the absence of estrogen, the mutant T7 RNAP would be in the incorrect conformation, which would not allow it to transcribe the reporter construct. When estrogen becomes available, the protein folds into its active conformation, which allows it to transcribe the reporter, which eventually produces a visible result. To learn more about the function of the mutant T7 RNAP, visit <A href="https://2015.igem.org/Team:Carnegie_Mellon">the CMU team's webpage</A>. Since this project relies on a modified T7 RNAP, we searched for a reporter construct based on a T7 promoter. One of the constructs we found was <A href="http://parts.igem.org/Part:BBa_K567018">PT7-GFP-TAG-RFP,</A> which we characterized, and to which we added a contribution. We also constructed a <A href="http://parts.igem.org/BBa_K1833000">pT7-eGFP part,</A> which is our best characterized part. To learn more about the estrogen sensor project, click <A href="https://2015.igem.org/Estrogen/Project">here</A>.<DIV><BUTTON class="minimizer">Click to read about other aspects of the project...</BUTTON></DIV></SPAN></TD><TD colspan="2" class="td33"><H4>Protease Sensor</H4>The second sensing system we have designed relies on transcriptional repressors. By creating a synthetic repressor that gets cleaved by a specific protease, the extract we create will be sensitive to the protease. This can be used to detect breast and colorectal cancer biomarkers such as MMP-2 and MMP-9 in patients' urine.(<A href="http://www.gynecologiconcology-online.net/article/S0090-8258(11)00584-1/abstract">Coticchia 2011</A>)<DIV><BUTTON class="expander">Click to read more...</BUTTON></DIV><SPAN style="display: none;">This project uses several concepts to sense proteases. First of all, this system relies on a two-hybrid repressor previously described in some detail. (<A href="http://mic.sgmjournals.org/content/journal/micro/10.1099/00221287-147-6-1651#tab2">Di Lallo 2001</A>) Secondly, instead of using the two-hybrid as a way of detecting the interaction between two proteins, we created fusion proteins that contain both parts of the two-hybrid repressor. This allowed us to insert a linker sensitive to specific proteases, which would then inactivate the repressor, and allow transcription of the reporter to occur, as shown in the image below. To learn for about the protease sensor project, click <A href="https://2015.igem.org/Protease/Project">here</A>.<DIV><BUTTON class="minimizer">Click to read about other aspects of the project...</BUTTON></DIV></SPAN></TD><TD colspan="2" class="td33"><H4>Three-Hybrid Versatile Sensor</H4>This project aims to develop the full versatility that paper-based sensors can have. This system uses the idea of a three-hybrid system, where one part binds the promoter, another part is a subunit of <I>E. coli</I> RNA Polymerase, and the analyte of choice provides the bridge that recruits the RNAP to the DNA. For this system, we chose two analytes: VEGF-A, which is a small dimeric protein involved in many cancers, and anti-MUC1 antibodies, which are established prognostic marker for favorable outcomes of lung, breast, pancreatic, and colon cancers (<A href="http://www.ncbi.nlm.nih.gov/pubmed/10673204">Hirasawa 2000</A>, <A href="http://www.ncbi.nlm.nih.gov/pubmed/10653872">von Mensdorff-Pouilly 2000</A>, <A href="http://www.ncbi.nlm.nih.gov/pubmed/12455059">Hamanaka 2003</A>, <A href="http://www.ncbi.nlm.nih.gov/pubmed/17450466">Kurtenkov 2007</A>). <DIV><BUTTON class="expander">Click to read more...</BUTTON></DIV><SPAN style="display: none;">Both of these sensors use the same DNA binding domains and RNA Polymerase domain, inspired by a bacterial two-hybrid system developed in 2000. (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC16554/">Joung  2000</A>) In fact, the only difference between the sensors are the proteins fused to these domains. In the VEGF-A sensor, we used a single chain variable fragment antibody, which is extremely specific to its target. (<A href="http://www.sciencedirect.com/science/article/pii/S0022283699931923">Chen 1999</A>) In the anti-MUC1 antibody sensor, the bait is a portion of the MUC1 protein, as shown in the image below.To learn more about this project, click <A href="https://2015.igem.org/3-Hybrid/Project">here</A>.<DIV><BUTTON class="minimizer">Click to read about other aspects of the project...</BUTTON></DIV></SPAN></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>