"
Page
Discussion
View source
History
teams
Log in
 
Team:CBNU-Korea/Notebook/3
From 2012.igem.org
2012 iGEM
CBNU-Korea
HOME
TEAM
PROJECT
NOTEBOOK
SAFETY
JUDGING
iGEM
NOTEBOOK
- Notebook
+ MENU
01 +        NoteBook
- March,        April
- May, June
- July
-        August
-        September
July
July. 2nd – July. 5th
Monday
■A : Found the searching method of the Bio-brick to run a       software, and reviewed how to design the software.
■B : Made the chart of 4 species.
(Used same algorithm       as the previously used). 4 species list :       Escherichia_coli_str_K-12_substr_MG1655,       Pseudomonas_aeruginosa_UCBPP-PA14,       Salmonella_enterica_substr_enterica_serovar_Typhimurium_str_LT2,       Heamphilus_influenzae_Rd_KW20
■C : Set standard based on genes which is existing in the       every species.
Tuesday
■A : Made questionnaire to be held with instructors.
Thursday
■A : Reviewed the questionnaire. Discussed about additional       GUI form.
■C : Set standard of division on genome.
July. 5th – July. 10th
Thursday
■B : Expanded the process to every 17 species. Annotated Synb_Code       to each genes & Drew a chart.
(The genes contained in       every 17 species : 48 List of 17 species :       (http://cafe.naver.com/synb2011/314)
Blast standards :       identity > 25%, |Length| < 5%, e-value > 0.05, bit-score > 200.
July. 9th – July. 15th
Monday
■A : Studied codon optimization.
Wednesday
■A : Studied Promoter profiling.
■B : Got some of opinions about setting standard of BLAST from       instructors. Found a new standard through other processes.
Blast standards
Standard 1 : identity > 25%, |Length| < 15%, e-value : None, bit-score : None.
Standard 2 :       identity > 25%, |Length| < 15%, e-value: < 0.05, bit-score < 50.
Standard 3 : identity > 25%, |Length| < 15%, e-value < 0.05,       bit-score : None.
Thursday
■A : Discussed to implement SBOL or not.
■B : Studied UID theory to annotate SynB UID.
■C : Compared the strand of genome & Analyzed strand       pattern.
Friday
■C : Began to analyze with the result of Standard 1 and       standard 2.
Saturday
■A : Studied SBOLC Library to implement the function of       saving as SBOL format .
■B : Studied what gene to use as a standard and what kind of       analysis should we use.
July. 16th - July 22th
Monday
■A : Summarized research papers. Exchanged information with       other universities(Ulsan National Institute of Science and       Technology, Korea University)
■B : It taked too much time to analyze. Set SynB UID       concept, approximately.
■C : Simulated on analysis of strand pattern. Studied how to       use analysis on strand(+/-).
Tuesday
■A : Worked on document & Tried to find sponsors.
■B : Started to specify requirements of SW to begin       implementing SW.
Wednesday - Thursday
■B : Developed new analysis algorithm. It only takes 30       minutes to analyze which took a week before.
Thursday
■A : Discussed about the method of design to adjust       requirements into SW well.
■B : Implemented SAS program to calculate confidence       interval(statistic word) by sorting genes based on newly annotated       Synb UID.
Friday
■A : Started to design DB table to use in SW.
■B : Approximate Organized the requirements of SW through       consultation with the instructors.
■C : Drew a Graphed of strand(+/-) pattern of each gene in       genome.
Saturday
■B : Due to a power outage, had to run the analysis program       again. However because of this we could improve the algorithm.
July. 23rd – July. 29th
Monday
■A : Started to study GDI+ to implement graphic part of SW.
■B : Organized primary requirements of SW.
■C : Summarized research paper. (?compared genomics of the       strand switch region of Leishmania chromosome 1 reveal a novel       genus-sprecific gene and conserved structural features and       sequence motifs)
Tuesday
■B : Organized requirement document with additional       requirements.
July. 25th - July. 26th
Thursday
■A : Produced Bio-Brick icon of each type which are going to       be used in SW.
■B : Implemented new annotation algorithm. Ran program with       it.
■C : Considered method of analyzing strand, Based on       research papers.
Friday
■B : Identified annotation.
■C : Divided genome into 100 section to analyze strand       pattern. Calculated strand percentage.
July. 30th – July. 31st
Monday
■A : Implemented function of image shifting for Bio-Brick       synthetic.
■B : Set targets of Blast analysis, Bacteria Gamma. Modified       existing algorithm of analysis. Started to analyze by using all of       the computers in lab.
Blast standard 1 : identity > 25%,       |Length| < 5%, e-value > 0.05, bit-score > 200.
Blast       standard 2 : identity > 30%, |Length| < 5%, e-value > 0.05,       bit-score > 200.
Blast standard 3 : identity > 30%,       |Length| < 15%, e-value > 0.05, bit-score > 50.
■C : Analyzed strand using statistical program of SAS and R.
Tuesday
■A : Implemented interval function of Bio-Brick synthetic.
■B : Result of Blast wasn’t good enough. Set standard of Blast       differently.
Blast standard 1 : identity > 20%, |Length| < 15%, e-value < 0.05, bit-score > 200
Blast standard 2 :       identity > 30% |Length| < 5%, e-value < 0.05, bit-score > Query       Length / 3
■C : Analyze on calculating confidence interval(statistic       word) had been failed.
Retrieved from " http://2012.igem.org/Team:CBNU-Korea/Notebook/3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
