"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Biology/Sensors
From 2014.igem.org
IGEM Evry 2014
Toggle navigation
Home
Overview
Project
Achievements & Results
Team
Members
Attributions
Collaborations
Acknowledgments
Media
Biology
Parts
Toxic compounds
Chassis & Characterization
Genome Assembly
Sensors
Transposons
RNAseq
Modeling
Sponge Model
PCB Model
Phenol Model
Notebook
Cell characterization
Transposons
Sensing
RNAseq
Protocols
Policy & Practices
Philosophy
Ethics
Safety
Interlab
Aim & Results
Used Devices
Notebook
Spon-ge-sors
IGEM Bar
Biology - Classic & RNAseq-based sensors
Phenol biosensor :
Phenol and its derivative are of major concern since their accumulation in the environment, as a result of intensive human activity, cause toxicity for both flora and fauna  at relatively low concentration [1][2].In the context of bioremediation and preservation of water  we designed a biosensor of phenol  that allows us to measure the concentration of phenol in a given marine environment.
This phenol biosensor rely on both signal transducing component, DmpR and inducible fluorescence emitting component based on the Green Fluorescent Protein (GFP). Both elements are assembled in a single plasmid and allow bacteria  to respond to the presence of phenol by emitting fluorescence.
Biosensor construction
Dmpr, a phenol dependant signal transducer
Presentation
DmpR is a member of NtrC protein family. NtrC-type regulators activate RNAP containing the alternative sigma factor 54. The s54-RNAP holoenzyme forms a stable complex with -12 and -24 promoters but is unable to start transcription without further activation NtrC protein family strongly stimulate the polymerase complex. They bind to DNA regions more than 100bp upstream from the s54-RNAP binding site (UAS). Interaction between the regulatory protein and s54-RNAP is facilitated by a bend of DNA.
Structure
DmpR is a 563 amino acid long protein. Although no direct structural information have been described (e.g protein purification), comparisons with other member of NtrC family and genetic experiments have brought some insight on its structure. Four domains are classically described for members of NtrC family :
A-domain (211 amino acid long) involved in direct interaction with effector. One inducer binding site is present per monomer, which was demonstrated for DmpR  and which could be pinpointed to a subregion between amino acid residues 107 and 186
B-domain is a linker between A and C domain.
C-domain being the most highly conserved region among the family members, is involved in ATP binding and hydrolysis and in s54-RNAP interaction
D-domain for  Dna binding with HTH motif (helix turn helix).
Effectors dependant activation
DmpR-like activators require a chemical effector and ATP as the cofactor. The effector is usually the primary substrate of the target pathway or a compound related to this. Phenol and its derivative are typical effector for DmpR
Mechanism
• 1. Binding with effector : The key control event is a direct interaction of aromatic effectors (phenol) with A-domain, which leads to the expression of its otherwise repressed C-domain mediated ATPase activity (Shingler and Pavel, 1995). B domain act as a linker between A and C domains and is necessary for C domain derepression in presence of effector.
Figure 4 : DmpR binding with  phenol and ATP
• 2. Binding with ATP trigger  hexamerisation : DmpR appears to be a dimer in solution. Following  effector captation by A- domain, binding of ATP can trigger an apparent dimer to hexamer switch in subunit conformation. [3]   3. Hexamer configuration and ATPase activity allows Transcription activation
Results :
We prepared a protocol test to evaluate our Biosensor: E.coli (DH5apha) was grown overnight in M9 medium at 37 °C and then diluted 100-fold to an OD of 0.01 in fresh M9 medium containing Chloramphenicol in 96-well plates. After 6 hours’ of culture at 37 °C, each culture (200 μL) was centrifuged at 2500 r.p.m. for 15 minutes and was suspended in 200 μL of fresh M9 medium containing phenol of different concentrations  Then the fluorescence intensity of cultures was measured by microplate reader (TECAN).
We compared our biosensor (BBa k1413001-Green) with another version of it (BBa K1413002-Blue) that have been obtain by performing a mutation of 3 nucleotides within the sequence of RBS B0032 of sfGFP.  The novel RBS sequence has been designed in accordance with the consensus Shine-Dalgarno sequence: AGGAGGUAA and allows the RNA to bind more tightly to the 16S ribosome to initiate translation.
RBS B0032 = tcacacaggaaag New RBS = tcaaggaggaaag
To achieve this experiment we achieve a site directed mutagenesis using the two following primers :
Here are our results:
OD 600 of Ecoli carrying BBa K1413001 (RBS B0032- Green graph) and BBa K1413002 (new RBS - Blue graph)
Raw data of fluorescence expressed by Ecoli carrying BBa K1413001 (RBS B0032- Green graph) and BBa K1413002 (new RBS - Blue graph). Bottom graph shows raw fluorescence obtained by purified GFP at indicated concentration.
The induction ratio was calculated by dividing the fluorescence intensity of biosensors exposed to phenol by the basal fluorescence intensity of the biosensors. Here show a significant increase of sfGFP expression in both biosensors. Note that BBa K1413002 allows significant sensing of phenol at a concentration of 1µM.
Reference List  [1] Bacterial Transcriptional Regulators for Degradation Pathways of Aromatic Compounds David Tropel 2004 [2] Aromatic ligand binding and intramolecular signalling of the phenol-responsive s54-dependent regulator DmpR Eric O’Neill 1998 [3] The Regulatory N-terminal Region of the Aromaticresponsive Transcriptional Activator DmpR Constrains Nucleotide-triggered Multimerisation Petra WikstroÈm 2001 [4] Regulation of the transcriptional activator NtrC1: structural studies of the regulatory and AAA+ ATPase domains Seok-Yong Lee 2003
PCBs biosensor :
A system of detection and degradation of polychlorinated biphenyls in Pseudomonas pseudoalcaligenes KF707
Pseudomonas pseudoalcaligenes KF707 is one of the strain which are able to degrade polychlorinated biphenyls (PCBs). This strain can grow on biphenyl and salicylate as a sole source of carbon. The bacteria contains a biphenyl-catabolic (bph) gene cluster (bphR1A1A2-(orf3)-bphA3A4BCX0X1X2X3D) which degrade compounds (figure A).
Figure A: Organization of the bph gene cluster from Pseudomonas pseudoalcaligenes which is implied in degradation of PCBs (K. Furukawa and H. Fujihara, 2008)
bphR1 is a positive regulator for bphR1 protein, which belongs to the GntR family,  and for bphX0X1X2X3D. It's  implied in the degradation of PCBs. Watanabe et al showed that there is an other regulatory protein, bphr2, which is involved in the positive regulation of bphA1A2A3A4BC genes. In the absence of biphenyl, small amounts of bphR2 protein binds to bphR2 operator to repress bphR2 transcription (autorepression) and activate bphR1 weakly. When there is biphenyl in the media, bphR2 protein binds to bphR1 and bphA1A2A3A4BC operators to activate strongly their transcription. This allows to initiate the degradation of biphenyl. (figure B).
Figure B: Cross-regulation of bph gene cluster by bphR1 and bphR2 in Pseudomonas pseudoalcaligenes KF707 (K. Furukawa and H. Fujihara, 2008)
2013 Saclays' project :
In 2013, Saclays' team wanted to construct a biosensor for PCBs , a project that failed (figure C). So our first aim was to do their construction and then to optimize it and to characterize it.
Figure C: 2013 Saclay's project: construction of a PCBs biosensor
How function our biosensor ?
Our construction is composed by a constitutive promoter (BBa_J23114) , a RBS (BBa_B0034) , bphR2 gene (BBa_K1413021) , which has been mutated because of a pstI site in its sequence,  bphR1 promoter region (BBa_K1155001) , received from Saclay’s team, RFP (BBa_E1010) and a terminator (BBa_B0015) (figure D). Instead to use a system of detection with lacZ, we have opted for a fluorescence system with RFP.
Figure D: PCBs biosensor
Figure E: Mechanism of our biosensor
In absence of PCBs : bphR2 (BBa_K1413021) is bound to bphR1 promoter which activate the transcription of RFP but in very low expression.
In presence of PCBs : When compound  diffuses into the media, it binds to bphr2 protein which undergoes a conformational change that permits to activate more stronger bphR1 promoter and increase the transcription of RFP.
Improvement
In order to improve their construction, we have made some modifications. In fact, we have used to add RBS B0034 to increase the level of transcription. After bphR2 gene, we have put a terminator to prevent leakage of constitutive promoter. In their sequence of bphR2 (BBa_K1155009) , there is a pstI site so this gene has been mutated by keeping the same codon (BBa_K1413021) .
Construction of our biosensor in E. coli DH5a  :
Every parts come from the registry except bphr2 gene. Knowing that there is a pstI site in its sequence, bphR2 gene has been synthetized without this site to allow to do the construction at biobrick format.
Golden gate :
We wanted to assemble all the parts by using golden gate so in one shot. For that, we have followed this processus:
Transformation of all the parts in DH5a by heat shock
Miniprep
PCR with primers designed especially for the golden gate. Theses primers have been designed so as to have a bsaI site.
PCR clean-up
Launching of golden gate program (cf notebook)
Transformation in DH5a
=> Several tests have been done without success.
Sequential digestions :
Each part contains biobrick prefix and biobrick suffix so we have decide to do a succesion of digestion-ligation-transformation protocol (figure F). We have successed to obtaine our biobricks BBa_K1413023 and BBa_K1413024
Figure F: biobricks BBa_K1413023 (a) and BBa_K1413024 (B)
Contacts
igemevry@gmail.com
Retrieved from " http://2014.igem.org/Team:Evry/Biology/Sensors "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
