"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Notebook/Sensing
From 2014.igem.org
IGEM Evry 2014
Toggle navigation
Home
Overview
Project
Achievements & Results
Team
Members
Attributions
Collaborations
Acknowledgments
Media
Biology
Parts
Toxic compounds
Chassis & Characterization
Genome Assembly
Sensors
Transposons
RNAseq
Modeling
Sponge Model
PCB Model
Phenol Model
Notebook
Cell characterization
Transposons
Sensing
RNAseq
Protocols
Policy & Practices
Philosophy
Ethics
Safety
Interlab
Aim & Results
Used Devices
Notebook
Spon-ge-sors
IGEM Bar
Notebook - Sensing
Mutation of the Pst1 site
Digestion of plasmid containing DmpR + GFP on three strains of E.coli (DH5a, Top10 and Bl21) by EcoR1 (wells 1), Pst1 (wells 2) and EcoR1+Pst1 (wells 3).
We have the good construction in DH5a AND tOP 10 STRAINS.
Sept 30
Assembling of the parts
Ligation of all parts
Transformation on E.coli
Purification of palsmids with NucleoSpin Plamsid Kit (Macherey Nagel) on 4 colonies
Digestion with EcoRI
All colonies contain the construction of all parts.
Sept 18
Sensor construction bphR2/PbpR1
A new tentative of golden gate has been done with modification of volumes of each part:
J23114-rbs : 0,12µL
bphR2: 1,05µl
bphR1:0,31µl
RFP : 0,64µl
Terminator: 0,67µl
pSB1C3,G3: 1µl
The mix enzyme was inchanged and the program is of 8h
Sep 08
Sensor construction bphR2/PbpR1
Results PCR clean-up:
J23114-rbs : 76ng/µl
bphR2: 76,8 ng/µl
bphR1:126,4 ng/µl
RFP : 98,3ng/µl
Terminator: 42,4ng/µl
pSB1C3,G3: 1µl
Sep 07
Sensor construction bphR2/PbpR1
Miniprep was made for each part and the nanodrop gives:
J23114-rbs: 205 ng/µl
bphR2: 227,5ng/µl
bphR1:57,2ng/µl
RFP: 74,2ng/µl
terminator:54,6ng/µl
pSB1C3,G3: 182,7ng/µl
A new PCR has been done by using Q5 DNA polymerase for all the parts expected for the terminator (one taq polymerase)
Sep 05
Sensor construction bphR2/PbpR1
As the first golden gate test failed, we retry at the beginning. Parts were cultured in 10ml LB + 10µL Cam for bphR2, bphR1, RFP, Terminator, pSB1C3 and in 10ml LB + 10µL Amp for J23114 and stored at 37°C overnight
Sep 04
Sensor construction bphR2/PbpR1
There is not colonies from the transformation of golden gate product in pSB1C3,G3.
Sep 03
Sensor construction bphR2/PbpR1
Transformation has made with golden gate product and 5µL of the product migrated :
There is'nt DNA in the mix
Aug 29
Sensor construction bphR2/PbpR1
PCR colony has been done with colonies of the transformation bphR2 in pSB1C3. The tail expected was 1242bp and we obtained it:
A nanodrop has been done on the colony 1 (48ng/µl) and colony 2 (4&,4ng/µl) and the colony has been send to sequencing
With PCR clean up of each part, golden gate has been done. Volumes for each part has been calculated with cloning bench application.
Mix DNA contains:
J23114-RBS (82,1ng/µl):0,06 µl
bphR2 (139,3 ng/µl) : 0,327µl
bphR1 (101,1ng/µl): 0,21µl
RFP (75,3 ng/µl): 0,46µl
Terminator (28,1ng/µl) : 0,37µl
pSB1C3,G3 (107,2ng/µl) : 0,93µl
Mix enzyme contains:
1,5µl 10X T4 ligase buffer
0,5µl BSAI
0,5µl T4 ligase
2,5µl
Aug 28
Sensor construction bphR2/PbpR1
Some colonies had grown after the transformation of bphR2 in pSB1C3:
Products PCR from 26th august has been migrated :
Expected for the terminator, we have bands at the good tail. So for the terminator, an other PCR has been made but by using the one taq polymerase instead of Q5 DNA polymerase and we obtained the good band:
Aug 27
Sensor construction bphR2/PbpR1
We wanted to assemble the construction by using golden gate. After to have received primers, PCR was done  to amplify J23114-RBS, bphR2, bphR1-RBS, RFP and terminator with golden gate primers. The tail of each part is:
J23114-RBS = 82bp
bphR2 mutated = 929bp
bphR1-RBS = 340bp
RFP = 732bp
Terminator = 162bp
Simultaneously, the product of digestion of bphR2 and pSB1C3 has been migrated to verify the digestion then, bphR2 and pSB1C3 has been ligated and we have done a heat shock transformation.
Aug 26
Sensor construction bphR2/PbphR1 :
Miniprep of pSB1C3 was done => 68,9ng/µl and the plasmid was digested with EcoRI-HF and pstI.
Digestion products of pSB1C3 and bphr2 has been migrated on gel 1X agarosis. Bands expected were:
for pSB1C3: one band at 2069bp (vector) and one band at 1070bp (insert)
for bphr2: one band at 946 pb (insert) and one band at 1300bp (vector)
We obtained the good bands so an extraction on gel was done to recover only pSB1C3 without its insert and only bphr2 gene.
Aug 23
Transformation plate observation:
- BBa_E1010: 50-60 colonies
- BBa_J23114: 150-200 colonies
- BBa_B0015: 40-50 colonies
- PSB1C3G3: > 1000 red colonies
A PCR was performed on 8 colonies for BBa_J23115 (K823012) following the protocol Table 3 and 4.
Samples are loaded on a 1% agarose gel, 10 µL of sample + 2 µl of loading dye 6X per well. Gel ran 45 minutes at 100 mV.
Sensor construction bphR2/PbphR1 :
One colony of the transformation of pSB1C3 was incubated in 3ml LB + 3µL Cam, at 37°C, overnight
Aug 22
Sensor construction bphR2/PbpR1
BBa_J23114 was resuspended with 10 µL sterile water in order to obtain a DNA concentration around 0.2 ng/µl (according to the registry). Solution was transferred into one 1 ml eppendorf tube and stored at -20°C.
Transformations of constitutive promoter BBa_J23114 , RFP BBa_E1010 and terminator BBa_B0015 and vector pSB1C3G3 was done on DH5alpha as followed:
Remove E. coli competent tubes from -80°C and keep it on ice
Add 3 µl of template (here solubilized plasmids from the registry distribution kit) and mix gently
Incubate 10 minutes on ice
Perform an heat shock 30 seconds at 42°C
Incubate 2 minutes on ice
Add 2 ml of LB medium and incubate 60 minutes at 37°C with an agitation at 200 rpm
Plate 200 µl of pSB1C3G3, BBa_B0015 and E1010 on a chloramphenicol LB agar plate and ampicilline LB agar plate for BBa_J23114
Incubate plate overnight at 37°C
We received primers.
Table 4: Received primers with numbers and sequences for PCB sensing constructions
pSB1C3 was digested with EcoRI and pstI and a ligation with bphR2 was done before the transformation in DH5a.
50 µL were plated on Cam Lb plate and incubated at 37°C overnight
Aug 21
Sensor construction bphR2/PbphR1
We received sequencing reads: that match perfectly to the registry sequence.
26DJ54
AATAGGCGTTATCACGAGGCAGAANTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGATGTCCCTGG GTGACATGCGTGATTTGGCCGCCACGCGGATCGCGCTCGAGAGCGAAGCGTTACGCCAAAGCGTGCTGAATGGTGACGCTGAATGGGAGGCGCGGATCGTCAGTTC GTTTCACCGACTGTCATTGATTGAAGAGCCCACGATGCGGGATCCGGCTCGCTGGTTTAATGAGTGGGAGCCAGTCAACCGCGGTTTTCACGAAGCTCTTATCTCT GCCTGTTCGTCCGTCTGGATCCGGCGGTTCCTGTCCATCCTGTATGTGCATATGGAGCGCTACCGCCGATTGACTGCTTACTAGTAGCGGCCGCTGCAGTCCGGCA AAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGAAAAGATTACTTCGCGTTATGCAGGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTC GGCTGCGGCGAGCGGTATCA GCTCACTCAGGG
26DJ55
CGAGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGC AGCGGCCGCTACTAGTAAGCAGTCAATCGGCGGTAGCGCTCCATATGCACATACAGGATGGACAGGAACCGCCGGATCCAGACGGACGAACAGGCAGAGATAAGAG CTTCGTGAAAACCGCGGTTGACTGGCTCCCACTCATTAAACCAGCGAGCCGGATCCCGCATCGTGGGCTCTTCAATCAATGACAGTCGGTGAAACGAACTGACGAT CCGCGCCTCCCATTCAGCGTCACCATTCAGCACGCTTTGGCGTAACGCTTCGCTCTCGAGCGCGATCCGCGTGGCGGCCAAATCACGCATGTCACCCAGGGACATC TCTAGAAGCGGCCGCGAATTCCAGAAATCATCCTTAGCGAAAGCTAAGGATTTTTTTTATCTGAAATTCTGCCTCGTGATACGCCTATTTTTATAGGTTAATGTCA TGATAATAATGGTTTCTTAGA
Plasmid with bphR2 gene was purified with NucleoSpin Plamsid protocol (Macherey-Nagel) => we obtained 227,5ng/µL
This plasmid was digested according this protocol:
Add:
sterilized water: qsp 20µL
template DNA : 500ng
buffer 2.1: 2µL
BSA: 0,2µL
EcoRI: 1µL
PstI: 1µL
Reverse for mix
Incubate at 37°C during 45mn
Incubate at 80°C during 20mn
Ligation was not possible because pSB1C3 plasmid was not digested so the digested plasmid (bphR2 gene) was stored at -20°C
Survival test on E.coli BL21:
Bacteria survive again for different concentrations but they were very concentrated
A dilution of the medium has been done for the positive control and the six different concentrations:
Aug 20
PCR using VF2 and VR primer
Q5 polymerase
Expected bands :
DmpR: 2038 bp
GFP B0031: 1331 bp
GFP B0032: 1330 bp
Digestion
From newly extracted DNA
DmpR: SpeI&PstI
GFP B0031/32: XbaI&PstI
Analysis
VF2/VR PCR products are still in agreement with the expected size either for DmpR and GFP B0031/32.
GFP 31/32 digestion products were in agreement with the expected size.
DmpR digestion by SP is still displaying 2 close bands profile.
Aug 20
Sensor construction bphR2/PbphR1:
A colony after the DH5a transformation was cultured in 3mL of LB + 3µL Amp, at 37°C, 200rpm, overnight
Survival test on E.coli BL21:
Bacteria continue to grow for all concentrations of 2 hydroxy-3',4'dichlorobiphenyl.
Aug 19
Dna extraction
Machery Nagel DNA purification Kit ( PROTOCOL )
PCR using VF2 and VR primer
Q5 polymerase
Expected bands :
DmpR: 2038 bp
GFP B0031: 1331 bp
GFP B0032: 1330 bp
Digestion
DmpR: SpeI&PstI
GFP B0031/32: XbaI&PstI
Analysis
VF2/VR PCR products were in agreement with the expected size either for DmpR and GFP B0031/32.
GFP 31/32 digestion products were in agreement with the expected size.
However DmpR digestion revealed an unexpected profile.
Gel extraction
Extraction of GFP B0031/32 digestion product [XbaI-PstI]
After confirmation of relevant electrophoresis bands, cut the gel all around the band and as close as possible to it.
Place the resulting piece into a 2ml microcentrifuge tube. Add 1µl of Binding buffer to 1 µg of gel. Place the tube at 55°C to 60°C during 10min or until gel turn completely liquid.
Follow classical DNA purification protocol.
Machery Nagel kit ( PROTOCOL 2 )
Aug 18
Sensor construction bphR2/PbphR1:
bphR2 gene, synthetized by Eurofins in a plasmid, was dissolved in 23,5µL of elution buffer because it was lyophilized.
Concentration obtained = 200ng/µL
DH5a chimiocompetents were transformed with this plasmid according to this protocol:
Remove DH5a from -80°C (about 200µL) and let them on the ice
Add 100ng of plasmid at DH5a and let during 30mn on ice
Make a heat shock by putting bacteria at 42°C during 30s-1mn
Let 1h at 37°C
Plate 100µL of the culture on LB-agar-Amp
Incubate at 37°C overnight, 200rpm
Aug 18
Aug 17
Survival test on E.coli BL21:
After the incubation of 15th July , bacteria had grown:
Aug 16
Cell culture of E.coli received from registry
BBa_K1031211 (Pr-DmpR )
BBa_K1031221 (P0-RBS B0031-GFP)
BBa_k1031222 (P0-RBS B0032-GFP)
Liquid : 3ml of Luria Broth 1x + 34µg/L Chloramphenicol in a 15ml falcon at 37°C shaking
On plate : 25ml LB 1x + 25ml  Agarose 1x + 34µg/L Chloramphenicol
Observation: DmpR bacteria have difficulties to grow compared with GFP bacteria
Left : BBa_K1031221 (P0-RBS B0031-GFP)
Right : BBa_k1031222 (P0-RBS B0032-GFP)
Aug 15
Survival test on E.coli BL21:
Serial dilution of compound was made from 10(-2) mol/L to 10(-8) mol/L
300µL of E.coli BL21 were added in each eppendorf tube.
We had 2 control tubes:
A negative control which contains 2,7mL of M9 medium + 300µL of compound
A positive control which contains 2,7mL of M9 medium + 300µL of E.coli BL21
Tubes were incubated at 37°C overnight, 200rpm
Aug 15
Survival test on E.coli BL21:
This test allowed to know if E.coli can survive at different concentrations of 2 hydroxy-3',4'dichlorobiphenyl
For that, BL21 culture was removed from the -80°C and culturing in 4 mL of LB
E.coli was incubated overnight at 37°C, 200rpm
Aug 14
Survival test on E.coli BL21:
10mg of 2 hydroxy,3'-4'dichlorobiphenyl have been received. The compound was lyophilized so it's has been dissolved in 17mL of DMSO in order to have a final concentration of 10(-2)M
The compound was stored at -20°C, in the dark because the DMSO is sensitive to the light.
Aug 13
Aug 12
Survivability in molecules which are going to be sensed - Results:
Here are the results of the survivability of Pseudovibrio in media with molecules which are going to be sensed:
Pseudovibrio seems to survive in presence of all concentrations of nitrite, it's affect cells' growth but don't kill Pseudovibrio.
In Lead, big concentrations lead to cells' death, but Pseudovibrio survive very well until a concentration of 5µL.
Cadmium does'nt seem to block cells' growth in concentrations which we want to use.
Jul 29
Survivability in molecules which are going to be sensed:
Ranges of concentrations of molecules that we want to sense were made as described in following tables:
1. Serial dilutions of compound were made
Lead
Cadmium
Nitrite
2. 300µL of E.coli BL21 was added in each eppendorf tube.
We had 2 control tubes:
*A negative control: 2,7mL of MB medium + 300µL of compound
*A positive control: 2,7mL of MB medium + 300µL of E.coli BL21
3. Incubation of tubes at 30°C overnight, 200rpm
Jul 29
Sensor construction bphR2/PbphR1:
Plate 20µL from the culture of 27th july on LB-agar-Cam (25 mL LB-Agar + 25µL Chloramphenicol)
Incubation of plate at 37°C overnight
A glycerol stock was made with 750ml of culture and 750ml of glycerol 50% and stored at -80°C
Jul 28
Sensor construction bphR2/PbphR1:
Culturing of one colony bphR1 promoter (BBa_K1155001) , received from Paris-Saclay's team, in 5ml LB + 5µL Chloramphenicol (Cam)
Incubation at 37°C overnight, 200rpm
Jul 27
Contacts
igemevry@gmail.com
Retrieved from " http://2014.igem.org/Team:Evry/Notebook/Sensing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
