<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Notebook_Sensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Notebook/Sensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Notebook/Sensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/Sensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/Sensing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Notebook/Sensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Notebook/Sensing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/5/51/NotebookEvry.jpg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Notebook - Sensing</H1></DIV></DIV></SECTION><SECTION id="timeline"><DIV id="cd-timeline" class="cd-container"><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U> Mutation of the Pst1 site </U>
Digestion of plasmid containing DmpR + GFP on three strains of <I>E.coli</I> (DH5a, Top10 and Bl21) by EcoR1 (wells 1), Pst1 (wells 2) and EcoR1+Pst1 (wells 3). 
We have the good construction in DH5a AND tOP 10 STRAINS. </P><SPAN class="cd-date">Sept 30</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U> Assembling of the parts </U>
Ligation of all parts 
Transformation on <I>E.coli</I>
Purification of palsmids with NucleoSpin Plamsid Kit (Macherey Nagel) on 4 colonies 
Digestion with EcoRI 

All colonies contain the construction of all parts.





</P><SPAN class="cd-date">Sept 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>A new tentative of golden gate has been done with modification of volumes of each part:
<UL><LI>J23114-rbs : 0,12µL
</LI><LI>bphR2: 1,05µl
</LI><LI>bphR1:0,31µl
</LI><LI>RFP : 0,64µl
</LI><LI> Terminator: 0,67µl
</LI><LI>pSB1C3,G3: 1µl
The mix enzyme was inchanged and the program is of 8h

<SPAN class="cd-date">Sep 08</SPAN></LI></UL></P><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>Results PCR clean-up:
<UL><LI>J23114-rbs : 76ng/µl
</LI><LI>bphR2: 76,8 ng/µl
</LI><LI>bphR1:126,4 ng/µl
</LI><LI>RFP : 98,3ng/µl
</LI><LI> Terminator: 42,4ng/µl
</LI><LI>pSB1C3,G3: 1µl
<SPAN class="cd-date">Sep 07</SPAN></LI></UL></P><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>Miniprep was made for each part and the nanodrop gives:
<UL><LI>J23114-rbs: 205 ng/µl
</LI><LI>bphR2: 227,5ng/µl
</LI><LI>bphR1:57,2ng/µl
</LI><LI>RFP: 74,2ng/µl
</LI><LI>terminator:54,6ng/µl
</LI><LI>pSB1C3,G3: 182,7ng/µl
</LI></UL>A new PCR has been done by using Q5 DNA polymerase for all the parts expected for the terminator (one taq polymerase)
<SPAN class="cd-date">Sep 05</SPAN></P></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>As the first golden gate test failed, we retry at the beginning. Parts were cultured in 10ml LB + 10µL Cam for bphR2, bphR1, RFP, Terminator, pSB1C3 and in 10ml LB + 10µL Amp for J23114 and stored at 37°C overnight
<SPAN class="cd-date">Sep 04</SPAN></P></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>There is not colonies from the transformation of golden gate product in pSB1C3,G3.
<SPAN class="cd-date">Sep 03</SPAN></P></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>Transformation has made with golden gate product and 5µL of the product migrated :
There is'nt DNA in the mix
<SPAN class="cd-date">Aug 29</SPAN></P></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>PCR colony has been done with colonies of the transformation bphR2 in pSB1C3. The tail expected was 1242bp and we obtained it:
A nanodrop has been done on the colony 1 (48ng/µl) and colony 2 (4&amp;,4ng/µl) and the colony has been send to sequencing
With PCR clean up of each part, golden gate has been done. Volumes for each part has been calculated with cloning bench application.
Mix DNA contains:
<UL><LI>J23114-RBS (82,1ng/µl):0,06 µl
</LI><LI>bphR2 (139,3 ng/µl) : 0,327µl
</LI><LI>bphR1 (101,1ng/µl): 0,21µl
</LI><LI>RFP (75,3 ng/µl): 0,46µl
</LI><LI>Terminator (28,1ng/µl) : 0,37µl
</LI><LI>pSB1C3,G3 (107,2ng/µl) : 0,93µl
</LI></UL></P></DIV></DIV></DIV></SECTION>
Mix enzyme contains: 
<UL><LI>1,5µl 10X T4 ligase buffer
</LI><LI>0,5µl BSAI
</LI><LI>0,5µl T4 ligase
</LI><LI>2,5µl
</LI></UL><SPAN class="cd-date">Aug 28</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>Some colonies had grown after the transformation of bphR2 in pSB1C3: 

Products PCR from 26th august has been migrated : 
Expected for the terminator, we have bands at the good tail. So for the terminator, an other PCR has been made but by using the one taq polymerase instead of Q5 DNA polymerase and we obtained the good band:
<SPAN class="cd-date">Aug 27</SPAN></P></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbpR1</B></U>We wanted to assemble the construction by using golden gate. After to have received primers, PCR was done  to amplify J23114-RBS, bphR2, bphR1-RBS, RFP and terminator with golden gate primers. The tail of each part is:
<UL><LI> J23114-RBS = 82bp
</LI><LI> bphR2 mutated = 929bp
</LI><LI> bphR1-RBS = 340bp
</LI><LI> RFP = 732bp
</LI><LI> Terminator = 162bp
</LI></UL>Simultaneously, the product of digestion of bphR2 and pSB1C3 has been migrated to verify the digestion then, bphR2 and pSB1C3 has been ligated and we have done a heat shock transformation.
<SPAN class="cd-date">Aug 26</SPAN></P></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Sensor construction bphR2/PbphR1 : </B></U>Miniprep of pSB1C3 was done =&gt; 68,9ng/µl and the plasmid was digested with EcoRI-HF and pstI. 
 Digestion products of pSB1C3 and bphr2 has been migrated on gel 1X agarosis. Bands expected were:
<UL><LI>for pSB1C3: one band at 2069bp (vector) and one band at 1070bp (insert)
</LI><LI>for bphr2: one band at 946 pb (insert) and one band at 1300bp (vector)</LI></UL>
We obtained the good bands so an extraction on gel was done to recover only pSB1C3 without its insert and only bphr2 gene. 
</P><SPAN class="cd-date">Aug 23</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">
Transformation plate observation: 
- BBa_E1010: 50-60 colonies
- BBa_J23114: 150-200 colonies
- BBa_B0015: 40-50 colonies
- PSB1C3G3: &gt; 1000 red colonies

 A PCR was performed on 8 colonies for BBa_J23115 (K823012) following the protocol Table 3 and 4.
 Samples are loaded on a 1% agarose gel, 10 µL of sample + 2 µl of loading dye 6X per well. Gel ran 45 minutes at 100 mV.
<U><B>Sensor construction bphR2/PbphR1 :</B></U>One colony of the transformation of pSB1C3 was incubated in 3ml LB + 3µL Cam, at 37°C, overnight

</P><SPAN class="cd-date">Aug 22</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U>Sensor construction bphR2/PbpR1</U><A href="parts.igem.org/Part:BBa_J23114">BBa_J23114</A> was resuspended with 10 µL sterile water in order to obtain a DNA concentration around 0.2 ng/µl (according to the registry). Solution was transferred into one 1 ml eppendorf tube and stored at -20°C.
Transformations of constitutive promoter <A href="parts.igem.org/Part:BBa_J23114">BBa_J23114</A>, RFP <A href="parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and terminator <A href="parts.igem.org/Part:BBa_B0015">BBa_B0015</A> and vector pSB1C3G3 was done on DH5alpha as followed: </P><OL><LI> Remove E. coli competent tubes from -80°C and keep it on ice
</LI><LI> Add 3 µl of template (here solubilized plasmids from the registry distribution kit) and mix gently
</LI><LI> Incubate 10 minutes on ice
</LI><LI> Perform an heat shock 30 seconds at 42°C
</LI><LI> Incubate 2 minutes on ice
</LI><LI> Add 2 ml of LB medium and incubate 60 minutes at 37°C with an agitation at 200 rpm
</LI><LI> Plate 200 µl of pSB1C3G3, BBa_B0015 and E1010 on a chloramphenicol LB agar plate and ampicilline LB agar plate for BBa_J23114
</LI><LI> Incubate plate overnight at 37°C
</LI></OL> We received primers.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Table 4: Received primers with numbers and sequences for PCB sensing constructions</CENTER></DIV></DIV></DIV></DIV>pSB1C3 was digested with EcoRI and pstI and a ligation with bphR2 was done before the transformation in DH5a.
 50 µL were plated on Cam Lb plate and incubated at 37°C overnight
<SPAN class="cd-date">Aug 21</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U>Sensor construction bphR2/PbphR1</U>We received sequencing reads: that match perfectly to the registry sequence.
26DJ54
AATAGGCGTTATCACGAGGCAGAANTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGATGTCCCTGG
GTGACATGCGTGATTTGGCCGCCACGCGGATCGCGCTCGAGAGCGAAGCGTTACGCCAAAGCGTGCTGAATGGTGACGCTGAATGGGAGGCGCGGATCGTCAGTTC
GTTTCACCGACTGTCATTGATTGAAGAGCCCACGATGCGGGATCCGGCTCGCTGGTTTAATGAGTGGGAGCCAGTCAACCGCGGTTTTCACGAAGCTCTTATCTCT
GCCTGTTCGTCCGTCTGGATCCGGCGGTTCCTGTCCATCCTGTATGTGCATATGGAGCGCTACCGCCGATTGACTGCTTACTAGTAGCGGCCGCTGCAGTCCGGCA
AAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGAAAAGATTACTTCGCGTTATGCAGGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTC
GGCTGCGGCGAGCGGTATCA
GCTCACTCAGGG
26DJ55
CGAGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGC
AGCGGCCGCTACTAGTAAGCAGTCAATCGGCGGTAGCGCTCCATATGCACATACAGGATGGACAGGAACCGCCGGATCCAGACGGACGAACAGGCAGAGATAAGAG
CTTCGTGAAAACCGCGGTTGACTGGCTCCCACTCATTAAACCAGCGAGCCGGATCCCGCATCGTGGGCTCTTCAATCAATGACAGTCGGTGAAACGAACTGACGAT
CCGCGCCTCCCATTCAGCGTCACCATTCAGCACGCTTTGGCGTAACGCTTCGCTCTCGAGCGCGATCCGCGTGGCGGCCAAATCACGCATGTCACCCAGGGACATC
TCTAGAAGCGGCCGCGAATTCCAGAAATCATCCTTAGCGAAAGCTAAGGATTTTTTTTATCTGAAATTCTGCCTCGTGATACGCCTATTTTTATAGGTTAATGTCA
TGATAATAATGGTTTCTTAGA
 Plasmid with bphR2 gene was purified with NucleoSpin Plamsid protocol (Macherey-Nagel) =&gt; we obtained 227,5ng/µL
This plasmid was digested according this protocol:
<OL><LI>Add:
</LI><LI> sterilized water: qsp 20µL
</LI><LI> template DNA : 500ng
</LI><LI> buffer 2.1: 2µL
</LI><LI> BSA: 0,2µL
</LI><LI> EcoRI: 1µL
</LI><LI> PstI: 1µL
</LI><LI>Reverse for mix
</LI><LI>Incubate at 37°C during 45mn
</LI><LI>Incubate at 80°C during 20mn
</LI><LI> Ligation was not possible because pSB1C3 plasmid was not digested so the digested plasmid (bphR2 gene) was stored at -20°C
</LI></OL><U>Survival test on E.coli BL21:</U>Bacteria survive again for different concentrations but they were very concentrated
 A dilution of the medium has been done for the positive control and the six different concentrations:</P><SPAN class="cd-date">Aug 20</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U>PCR using VF2 and VR primer</U>
Q5 polymerase 
Expected bands :
DmpR: 2038 bp 
GFP B0031: 1331 bp 
GFP B0032: 1330 bp <U>Digestion</U>
From newly extracted DNA
DmpR: SpeI&amp;PstI
GFP B0031/32: XbaI&amp;PstI</P><U>Analysis</U>
VF2/VR PCR products are still in agreement with the expected size either for DmpR and GFP B0031/32.
GFP 31/32 digestion products were in agreement with the expected size.
DmpR digestion by SP is still displaying 2 close bands profile.<SPAN class="cd-date">Aug 20</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Sensor construction bphR2/PbphR1:</U></B> A colony after the DH5a transformation was cultured in 3mL of LB + 3µL Amp, at 37°C, 200rpm, overnight
<B><U>Survival test on E.coli BL21:</U></B> Bacteria continue to grow for all concentrations of 2 hydroxy-3',4'dichlorobiphenyl. 
</P><SPAN class="cd-date">Aug 19</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U> Dna extraction </U>
Machery Nagel DNA purification Kit (<A href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/Plasmid%20DNA%20Purification/UM_pDNA_NS.pdf">PROTOCOL</A>)
<U>PCR using VF2 and VR primer</U>
Q5 polymerase 
Expected bands :
DmpR: 2038 bp 
GFP B0031: 1331 bp 
GFP B0032: 1330 bp <U>Digestion</U>
DmpR: SpeI&amp;PstI
GFP B0031/32: XbaI&amp;PstI<U>Analysis</U>
VF2/VR PCR products were in agreement with the expected size either for DmpR and GFP B0031/32.
GFP 31/32 digestion products were in agreement with the expected size.
However DmpR digestion revealed an unexpected profile.<U>Gel extraction</U>
Extraction of GFP B0031/32 digestion product [XbaI-PstI]
After confirmation of relevant electrophoresis bands, cut the gel all around the band and as close as possible to it.
Place the resulting piece into a 2ml microcentrifuge tube. Add 1µl of Binding buffer to 1 µg of gel. Place the tube at 55°C to 60°C during 10min or until gel turn completely liquid.
Follow classical DNA purification protocol.
Machery Nagel kit (<A href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/DNA%20clean-up/UM_PCRcleanup_Gelex_NSGelPCR.pdf">PROTOCOL 2</A>)
</P><SPAN class="cd-date">Aug 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Sensor construction bphR2/PbphR1:</U></B>bphR2 gene, synthetized by Eurofins in a plasmid, was dissolved in 23,5µL of elution buffer because it was lyophilized.
Concentration obtained = 200ng/µL
DH5a chimiocompetents were transformed with this plasmid according to this protocol:
<OL><LI>Remove DH5a from -80°C (about 200µL) and let them on the ice
</LI><LI>Add 100ng of plasmid at DH5a and let during 30mn on ice
</LI><LI>Make a heat shock by putting bacteria at 42°C during 30s-1mn
</LI><LI>Let 1h at 37°C
</LI><LI>Plate 100µL of the culture on LB-agar-Amp
</LI><LI>Incubate at 37°C overnight, 200rpm
</LI></OL></P><SPAN class="cd-date">Aug 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><SPAN class="cd-date">Aug 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Survival test on E.coli BL21:</U></B>After the incubation of <A href="https://2014.igem.org/Team:Evry/Notebook/Sensing/PCBs/08-15-2014">15th July</A>, bacteria had grown:
</P><SPAN class="cd-date">Aug 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U> Cell culture of E.coli received from registry </U>
BBa_K1031211 (Pr-DmpR ) 
BBa_K1031221 (P0-RBS B0031-GFP)
BBa_k1031222 (P0-RBS B0032-GFP)
Liquid : 3ml of Luria Broth 1x + 34µg/L Chloramphenicol in a 15ml falcon at 37°C shaking 
On plate : 25ml LB 1x + 25ml 
Agarose 1x + 34µg/L Chloramphenicol 
Observation: DmpR bacteria have difficulties to grow compared with GFP bacteria 
Left : BBa_K1031221 (P0-RBS B0031-GFP)
Right : BBa_k1031222 (P0-RBS B0032-GFP)</P><SPAN class="cd-date">Aug 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Survival test on E.coli BL21:</U></B><OL><LI>Serial dilution of compound was made from 10(-2) mol/L to 10(-8) mol/L
</LI><LI> 300µL of E.coli BL21 were added in each eppendorf tube. 
 We had 2 control tubes: 
</LI><LI> A negative control which contains 2,7mL of M9 medium + 300µL of compound 
</LI><LI> A positive control which contains 2,7mL of M9 medium + 300µL of E.coli BL21
</LI><LI> Tubes were incubated at 37°C overnight, 200rpm
</LI></OL></P><SPAN class="cd-date">Aug 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Survival test on E.coli BL21:</U></B>This test allowed to know if E.coli can survive at different concentrations of 2 hydroxy-3',4'dichlorobiphenyl
<OL><LI>For that, BL21 culture was removed from the -80°C and culturing in 4 mL of LB 
</LI><LI>E.coli was incubated overnight at 37°C, 200rpm
</LI></OL></P><SPAN class="cd-date">Aug 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Survival test on E.coli BL21:</U></B>10mg of 2 hydroxy,3'-4'dichlorobiphenyl have been received. The compound was lyophilized so it's has been dissolved in 17mL of DMSO in order to have a final concentration of 10(-2)M
The compound was stored at -20°C, in the dark because the DMSO is sensitive to the light.
</P><SPAN class="cd-date">Aug 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><SPAN class="cd-date">Aug 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Survivability in molecules which are going to be sensed - Results: </B></U>

Here are the results of the survivability of Pseudovibrio in media with molecules which are going to be sensed: 
Pseudovibrio seems to survive in presence of all concentrations of nitrite, it's affect cells' growth but don't kill Pseudovibrio.
In Lead, big concentrations lead to cells' death, but Pseudovibrio survive very well until a concentration of 5µL.
Cadmium does'nt seem to block cells' growth in concentrations which we want to use.

</P><SPAN class="cd-date">Jul 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B>Survivability in molecules which are going to be sensed: </B></U>

Ranges of concentrations of molecules that we want to sense were made as described in following tables:

1. Serial dilutions of compound were made <LI>Lead </LI><LI>Cadmium </LI><LI>Nitrite 

2. 300µL of E.coli BL21 was added in each eppendorf tube.
We had 2 control tubes: 
*A negative control: 2,7mL of MB medium + 300µL of compound
*A positive control: 2,7mL of MB medium + 300µL of E.coli BL21

3. Incubation of tubes at 30°C overnight, 200rpm 

</LI></P><SPAN class="cd-date">Jul 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Sensor construction bphR2/PbphR1:</U></B><OL><LI>Plate 20µL from the culture of  <A href="https://2014.igem.org/Team:Evry/Notebook/Sensing/PCBs/07-27-2014">27th july</A> on LB-agar-Cam (25 mL LB-Agar + 25µL Chloramphenicol)
</LI><LI>Incubation of plate at 37°C overnight
</LI><LI>A glycerol stock was made with 750ml of culture and 750ml of glycerol 50% and stored at -80°C
</LI></OL></P><SPAN class="cd-date">Jul 28</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U>Sensor construction bphR2/PbphR1:</U></B><OL><LI>Culturing of one colony bphR1 promoter <A href="http://parts.igem.org/Part:BBa_K1155001">(BBa_K1155001)</A>, received from Paris-Saclay's team, in 5ml LB + 5µL Chloramphenicol (Cam)
</LI><LI>Incubation at 37°C overnight, 200rpm
</LI></OL></P><SPAN class="cd-date">Jul 27</SPAN></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Notebook/Sensing">http://2014.igem.org/Team:Evry/Notebook/Sensing</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Notebook/Sensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Notebook/Sensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Notebook/Sensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Notebook/Sensing&amp;oldid=398050" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>