<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Biology_Chassis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Biology/Chassis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Biology/Chassis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/Chassis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/Chassis&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Biology/Chassis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Biology/Chassis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/b/b0/DNA.jpeg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Biology - From the unknown depths of the ocean to a chassis in iGEM</H1></DIV></DIV></SECTION><SECTION id="speakers"><DIV class="content-wrapper"><DIV class="col-lg-10 col-lg-offset-1"><H2> A specific chassis for specific environments: Seas &amp; Oceans</H2><P>
== <B><BIG>A duo from the sea</BIG></B> ==
</P><DIV align="justify"><P>
As seen in the <A href="https://2014.igem.org/Team:Evry/Overview"><BIG> overview</BIG></A> our biosensors should work optimally in native marine conditions (approximately 24-30 g/L NaCl). Moreover it has to attach to sponges and stick with them without disturbing the microbiome for as long as possible. Therefore it is necessary to use a bacterium naturally present on sponges, may it be an epibiont or a symbiont. 
</P><P>
Unfortunately it is a condition for which <B><FONT color="red"> no current chassis</FONT></B> in iGEM is ready for. It is still worth noticing that an interesting first effort toward a marine chassis was initiated by UCL 2012.
</P><P>
To be up to the task the bacterium should:
</P><P><UL><LI>be massively present on the sponge surface / avoiding being in an unfavourable position for food competition.</LI><LI>be found mainly in sponges / avoid spreading to species in contact with sponges.</LI><LI>be the phenotypically closest possible  to a known bacterium / avoid cell cultures difficulties.</LI></UL></P><P>
To be up to the task the sponge should:
</P><P><UL><LI> be numerously present in the ocean / avoiding putting species in danger.</LI><LI> be a natural host of the bacterium / avoid adapting its microbiome to the new epibiont.</LI><LI> be easily grown in a laboratory / avoid sponge culture difficulties.</LI></UL></P><P>
The closest combination sponge/bacterium that could fit these requirements is <I><BIG>Spongia Officinalis / Pseudovibrio denitrificans </BIG></I></P><DIV align="center"><DIV align="center"><P>
== <B><BIG>Born to filter: <I> Spongia officinalis </I></BIG></B> == 
</P><DIV align="justify"><P><I> Spongia officinalis </I> has been used for millennia and for various purposes: from the Greeks for bathing and lining their armours, to today's pharmaceutical attempts to produce anti-inflammatory and other therapeutics, not even mentioning the use by Arabic physicians as early as 932 A.D of soaked sponges with narcotic drugs to place over the patient’s nose to provide a state of anesthesia. In a nutshell everyone knows the &quot;bath sponge&quot;.
</P><P><I> Spongia officinalis </I> is a leuconoid sponge. A leuconoid sponge has a thick body wall, and the ostia open into incurrent canals that draw water into the sponge’s body. These incurrent canals open into chambers that are lined with choanocytes. Water flows from these chambers into excurrent canals that empty into a relatively small spongocoel. From there, water exits through an osculum.
</P><P>
Unlike asconoid and syconoid sponges, which are basically built around their spongocoels and oscula, a leuconoid sponge has a complex, irregularly-shaped body that may have several oscula. It allows a leuconoid sponge to grow to larger size.
</P><P>
It also provides them with powerful filtration capacity (approximately 1200 times its volume per day) which makes it a perfect host for our system. But as mentioned earlier our aim is to engineer an epibiont and not the sponge itself.
</P><P>
Following this introduction, we will both describe  what <I>Pseudovibrio denitrificans</I> exactly is and how we turned it into a transformable/selectable-ready chassis.
</P></DIV><FONT color="blue"><H2><I>Pseudovibrio denitrificans </I> the bacterium to modify</H2></FONT><P>
== <B><BIG>From a poorly known genius, a jewel emerged</BIG></B> ==
</P><DIV align="justify"><P>
At first glance, working on the microbiome of sponges appears like a hazardous task: 5734 articles for marine sponges, only 51 with microbiome of sponges, only 28 articles mentioning <I>Pseudovibrio genius</I>, only 5 mentioning <I>Pseudovibrio denitrificans</I>, and finally none about a genetic engineering system in it.
</P><P>
Moreover with only 12 species in the genius, and 2 strains sequenced (Pseudovibrio sp. FO-BEG1, Pseudovibrio sp. JE062) we knew we would have to sequence our strain of <I>Pseudovibrio denitrificans </I> with a mapping to one of the 2 strains references. A formidable drawback even considering that in the first place the species was chosen because of its natural denitrification ability (see <A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds"><B><BIG> toxic compound</BIG></B></A>) and easy to source at <A href="http://www.dsmz.de/catalogues/details/culture/DSM-17465.html"><B><BIG>DMSZ</BIG></B></A>.
</P><P>
On the other hand <I>Pseudovibrio denitrificans</I>, or strains related to the species, has been shown to be in the majority in at least six microbiomes of sponges in the Mediterranean Sea, where <I>spongia officinalis</I> resides.
</P><P>
Now the thorny question is: can it be easily used  in our lab?
</P><DIV align="center"><P>
== <B><BIG>A bacterium we  could work with</BIG></B> ==
</P><DIV align="justify"><P>
In the literature <I>Pseudovibrio denitrificans </I> is described as a Gram-negative, motile by means of one to several lateral or subpolar flagella requiring NaCl for growth. It exhibits optimal growth at about 30 C°, pH 8 and 3 % NaCl, and a  doubling time of 45 min in rich media. In a nutshell the conditions seem favourable for an iGEM project.
</P><P>
They are known to be capable of anaerobic growth by carrying out denitrifying metabolism using nitrate, nitrite or nitrous oxide as terminal electron acceptors. Being consequently a main reason for our <B><BIG>RNAseq</BIG></B> study to discover the transcripts upregulated by the presence of nitrite and clone their promoters as new sensors.
</P><P>
The following section describes protocols used to optimize (Rich media) &amp; control (Minimal media) growth conditions.
</P></DIV></DIV></DIV><H2>Cell culture</H2><DIV align="justify"><P>
With the help of the 'Molecules of defense and communication in the microbial ecosystems' team of the National museum of natural history of Paris (their web page <A href="http://mcam.mnhn.fr/MDCEM/index.htm">here</A>), we succeed to cultivate <I>Pseudovibrio denitrificans</I> and the strain <I>Pseudovibrio ascidiaceicola</I> (that they have generously given to us) in two medium, the marine broth (MB) and the minimal medium M9 added 2% of CASAmino Acid and by 3% of NaCl.</P><DIV align="center"><P><B><U>Table1:</U></B> Composition of used media, Marine Broth and M9-CASA+3% NaCl.</P></DIV><P>
Because of the opaqueness of the marine broth, it was impossible to obtain reliable data of the growth of the bacteria. We try to have a curve of growth in filtered MB but even if the bacteria grow, it's not the same condition. Thus, the growth curve has been made in M9-CASA+3%NaCl 1X, with the  <A id="igem-link" target="_blank" href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1813&amp;ID=1918&amp;Menu=1&amp;Item=21.2.10.1.1 " title="Go to iGEM site">TECAN infiniteM200</A> (cf: Figure1). </P><DIV align="center"><P><B><U>Figure1:</U></B> Curve of growth in liquid M9-CASA+3%NaCl 1X and liquid MB, and microscopic photo of <I>Pseudovibrio denitrificans</I> (left) and <I>Pseudovibrio ascidiaceicola</I> (right).</P></DIV><P>
The growth of our bacteria in 200µL of M9-CASA+NaCl is not very good, moreover the ratio culture/air is not optimal in the TECAN wells. In bigger cultures, the growth is faster.
The marine broth is more complete than the M9-CASA+3%NaCl, even if we have adjusted the salinity. As we can see in the Table 2, <I>Pseudovibrio denitrificans</I> grows better on MB medium. The number of colony is higher, and they are significantly bigger (cf: Table2). Moreover, the growth is faster. </P><DIV align="center"><P><B><U>Table2:</U></B> Number of CFU of MB 1X and M9-CASA+3%NaCl 1X plates.
The number of CFU was calculated by inoculated respectively 10µL of pre-cultures of <I>Pseudovibrio denitrificans</I> in MB 1X and in M9-CASA+3%NaCl at OD (600nm) ~ 1.</P></DIV><P>
In the aim to know if we can transfer <I>Pseudovibrio denitrificans</I> from a media, to the other one, we have made some tests and calculate a ratio of survivability. We have made two pre-cultures of <I>Pseudovibrio denitrificans</I> in MB 1X and M9-CASA+3%NaCl 1X, and after a culture overnight, we streaked them on plates of MB 1X and M9-CASA+3%NaCl 1X. The CFU on each plate were counted and ratios of survivability were calculated using the formula Sample/Control (controls being the number of CFU obtained on a plate from a pre-culture in the same medium). (cf: Figure2). </P><DIV align="center"><P><B><U>Figure2:</U></B> Tests of survivability, table of ratio and photos (on the left: M9 plates, on the right: MB plates).</P></DIV><P><I>Pseudovibrio denitrificans</I> can survive even if we transfer them from a medium to the other one. But we can see that even cells come from a pre-culture of M9-CASA+3%NaCl, they grow better and faster on MB plates. Moreover, even if they come from a MB pre-culture, they don't grow very well on M9-CASA+3%NaCl plates.</P></DIV><FONT color="blue"><H2 id="antibiotic">Sensitivity to antibiotics</H2></FONT><DIV align="left"><DIV align="justify"><P>
		After having succeeded to make our bacteria grow, we tested their resistance to different antibiotics. Knowing their resistance to antibiotics is very important because it will allow us to finalize protocols of selection after transformation. 
		We chose to test the most commonly used antibiotics. We included the three antibiotics used in iGEM (kanamycin, chloramphenicol and ampicilin), plus the erythromycin and the tetracyclin. We chose the erythromycin to test a conjugation protocol which required this antibiotic for <I>E.coli</I>, and we chose to test tetracyclin because it is quite often used for inducible systems.</P><DIV align="center"><B><U>Table3:</U></B> Concentrations tested for each antibiotic </DIV><P>
		We tested several concentrations of each antibiotics (cf: Table3), and we added a supplementary concentration for erythromycin because it is known to be ineffective on GRAM- bacteria. </P><P>
		The sensitivity tests were performed in four different conditions (cf: Figure3). We tested on Marine Broth (MB) 1X plates, MB 0.5X plates (used for conjugation) and M9-CASA 1X (+3% NaCl). We also tested in liquid M9-CASA 1X (+3% NaCl) to have a test in liquid culture, unfortunately we cannot measure the optical density with the MB medium because of its opacity. 
		The cells' sensitivity was measured for <I>Pseudovibrio denitrificans</I>, <I>E. coli</I>, and different clones of <I>E. coli</I> transformed with plasmids and carrying an acquired resistance AmpR, KanR, CamR ou ErmR.
	</P><DIV align="center"><P><B><U>Figure3:</U></B> Protocol of antibiotics' tests on <I>Pseudovibrio denitrificans</I>.</P></DIV><P>
		We let all the media incubating at 30°C (with shaking for liquid cultures), and they have been checked at day 1, day 2 and day 3. 
		In the following figures, find the results obtained after three days of cultures on plates and one day of liquid cultures.</P><DIV align="center"><P><B><U>Figure4:</U></B> Results of antibiotics' tests on MB plates.  On the left: Results on MB 0.5X plates.  On the right: Results MB 1X. 
		The number of CFU was not countable because of cellular lawn. The percentage of cell growth on plates was calculated by taking the controls of the same bacteria growing without antibiotics as reference.</P></DIV><DIV align="center"><P><B><U>Figure5:</U></B> Results of antibiotics' tests for M9-CASA+3%NaCl media.  On the left: Results on M9-CASA+3%NaCl 1X plates.  On the right: Results in liquid M9-CASA+3%NaCl 1X.
	 
		The number of CFU was not countable because of cellular lawn. The percentage of cell growth on plates was calculated by taking the controls of the same bacteria growing without antibiotics as reference. The ΔOD (600nm) was calculated by subtracting the initial OD (600nm) at Day 0 from the OD(600nm) of the considered day.</P></DIV><P>
		Unfortunately, results for tetracyclin on MB medium are not useful. Indeed, all our bacteria have grown. It can be because of a technical problem with the drug, or it can also be explained by the presence of some genes bringing a resistance to this drug in <I>Pseudovibrio denitrificans</I> (see section <A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome assembly</A>). But the death of cells on M9 plates doesn't allow us to confirm this hypothesis, even if it can be explained by a conditional expression of these genes.
		We can also see that <I>Pseudovibrio denitrificans</I> manage to grow on very high concentrations of ampicillin. By sequencing our bacteria, we found resistance gene to ampicillin too. This drug is so not really useful, except in a huge concentration.
		On a brighter side, according to these results, we can select Pseudovibrio denitrificans transformed  with a PSB1C3 on a concentration of chloramphenicol adjusted to the ratio 1/1000 (cf: Table3).
		To use other drugs, best ratio (Growth on antibiotic/Growth on medium only) allowed determining optimal concentrations to use on our different media (cf: Table4).
	</P><DIV align="center"><P><B><U>Table4:</U></B> Concentrations which can be used for selection of <I>Pseudovibrio denitrificans</I></P></DIV></DIV></DIV><H2>Transformation Protocol</H2><DIV align="justify"><P>
First, we tried to transform <I>Pseudovibrio denitrificans</I> by a heat shock. Chemically competent cells were prepared with CaCl2 solution. Transformation protocols were carried out by mixing 50 µL of cells with 1 µL of plasmid pSB1C3 solution (35ng/µL). After the standard heat shock protocol for <I>E.coli</I>, cells were spread on 1X MB medium plates. We did not obtain any transformant from this protocol.
By studying transformation of bacteria phylogenetically close of our strain <I>Pseudovibrio denitrificans</I>, we noticed that marines bacteria are often resistant to common chemical transformation approaches (Piekarski T, Buchholz I, 2009) . </P><P>
We considered testing protocols of conjugation and electroporation to transform our bacteria. Unfortunately, due to the lack of time we just tested the electroporation approach.
Two important parameters had to be finalized though; the type and the number of cell washes and also the voltage.</P><LI><U>Washing test</U>
Marine bacteria live in a salty environment. Thus, a classic water wash may to lead to an osmotic shock. Therefore, we tested washes with glycerol 10% and sorbitol 2% to try to limit the shock. <DIV align="center"><B><U>Figure6:</U></B> Protocol of washing tests</DIV>
Pre-cultures of <I>Pseudovibrio denitrificans</I> and <I>E.coli</I> cells were prepared and grew in an shaking incubator overnight at 30°C and 37°C respectively. After performing the protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>) of electrocompetents cells, 20µL of pre-cultures were spread when the DO(600nm) reached 1.5 for <I>Pseudovibrio</I> and 0.5 for <I>E.coli</I>, on plates corresponding to the media used for pre-cultures. CFU were then counted for all plates (Table5).<DIV align="center"><B><U>Table5:</U></B> Results of washing test, number of CFU for the different washes.</DIV>
Washes tests reveal that sorbitol is the best way to wash cells in MB pre-culture. Gycerol can also be used. For pre-cultures in M9, the better way to wash cells is by using glycerol.</LI><LI><U>Voltage test</U>
For this part, we had to find the voltage which allows the best efficiency, by not killing the cells but by being important enough to transform our bacteria.
To achieve this goal, we developed a similar protocol as the standard protocol described previously, by testing different voltages on our cells, without plasmids (Figure7).<DIV align="center"><B><U>Figure7:</U></B> Protocol of voltage tests</DIV>

Pre-cultures of <I>Pseudovibrio denitrificans</I> and <I>E.coli</I> were prepared and grew overnight in a shaking incubator, at 30°C and 37°C respectively. After performing the protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>) of electrocompetents cells, we electroporated our cells at either 1200 V, 1800 V, 2000 V and 2200 V. Cells were then incubated respectively at 30°C for <I>Pseudovibrio</I> and 37°C for <I>E.coli</I>. Then we plated the cells on MB or M9 plates according to the media used for the pre-culture. 
Unfortunately, some plates were convered by a cellular lawn and it was impossible to count the CFU. We chose to estimate roughly the cell growths as shown in the Table 6.<DIV align="center"><B><U>Table 6:</U></B> Results of voltage tests.</DIV>

In this table we can see that the higher voltage for which we obtained a very good rate of living cells is 2000 V for Pseudovibrio. According to these results, we chose to do a 2000 V electroporation to transform our <I>Pseudovibrio denitrificans</I></LI><LI><U>Test with pSB1C3</U>
This year, all the biobricks had to be sent in the backbone pSB1C3. For our first try to transform <I>Pseudovibrio denitrificans</I>, we thus chose to test this plasmid.
We made competent cells and then we did the developed transformation protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>), which is an electroporation at 2000 V.<DIV align="center"><B><U>Figure8:</U></B> Maps of PSB1C3.</DIV>

After a lot of tries, we did not succeed to have transformant of <I>Pseudovibrio denitrificans</I> which could grow on plates supplemented with chloramphenicol.
Because this plasmid works in <I>E.coli</I> which is a GRAM- bacteria as <I>Pseudovibrio denitrificans</I>, we expected it will work in our strain as well. Unfortunately, the plasmid was not expressed.</LI><LI><U>Other plasmids tested</U>
After trying transforming our strain with a standard plasmid of iGEM, pSB1C3, we tried other plasmids, pBBR1MCS (Katzke N, Arvani S, 2010) and pRhokHI-2 (Katzke N, Bergmann R, 2012). Those plasmids were generously given by Dr Thomas DREPPER from the Heinrich-Heine-Universität Düsseldorf, Institute of Molecular Enzyme Technology (Group of Bacterial Photobiotechnology). Dr DREPPER succeed to transform several strains of a genus very closed to our <I>Pseudovibrio</I>, the genus <I>Roseobacter</I>.

We made competent cells and then we effected the developed transformation protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>), which is an electroporation at 2000 V.<DIV align="center"><B><U>Figure9:</U></B> Maps of PBBR1MCS and pRhoKHI-2.</DIV>

Once again, after a lot of tries, we failed to have transformant of <I>Pseudovibrio denitrificans</I> growing on plates supplemented with chloramphenicol, even if pBBR1MCS is a plasmid supposed to work in almost all GRAM- bacteria and those two plasmids work in bacteria very close to <I>Pseudovibrio</I>.</LI><LI><U>Creating a new plasmid</U>
After trying to transform <I>Pseudovibrio denitrificans</I> with the pSB1C3 plasmid which did not work, we had several hypothesis:   
-The promoter of the Chloramphenicol resistance gene does not work in our bacteria
-The origin of replication does not work in our bacteria
-There is a system degrading exogene DNA in <I>Pseudovibrio</I>
To verify these hypothesis, we chose to find specific constitutive promoters and origin of replications in an other member of the genus <I>Pseudovibrio</I>, which genome has been sequenced, the FOBEG1 strain. (lien article) 
A strong constitutive promoter of FOBEG1 was found with a genome browser. Upstream sequences of vital and cell cycle independant genes were explored. We were interested by the transkelotase, a key enzyme of the pathway of pentoses phosphates.
We can assume this enzyme should have a good constitutive promoter, or at least have a reliable and strong expression.
To be sure to have the whole promoter sequence, we exported the sequence (fasta format) from the beginning of the ORF of the transkelotase, to the end of the previous ORF. This sequence was amplified with primers 5 and 6 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>).
After a PCR, we obtained a 6000pb band for <I>Pseudovibrio ascidiaceicola</I>, and no band for <I>Pseuvibrio denitrificans</I>. These PCR product was sent to sequencing, as shown in figure M.<DIV align="center"><B><U>Figure10:</U></B> Result of promoter sequencing.</DIV>

Thus, there is 94.1% of identity between the transketolase promoter of the <I>Pseudovibrio FOBEG1</I> strain and the <I>Pseudovibrio ascidiaceicola</I> strain.
For the replication origin, the aim was here to find the replication origin of the plasmid of the reference strain FOBEG1. To do so, we looked for the repA, repB and repC genes on NCBI. Those three genes follow each other in the genome.
This sequence was amplified with primers 49 and 50 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>).<DIV align="center"><B><U>Figure11:</U></B> Result of ORI amplification in <I>Pseudovibrio denitrificans</I> (Pd), <I>Pseudovribrio ascidiaceicola</I> (Pa) and <I>E.coli</I> DH5a.</DIV>
The negative control with <I>E.coli</I> DH5a confirmed that there is no amplification. For <I>Pseudovibrio</I> strains, we obtained one band for <I>Pseudovibrio denitrificans</I> and two for <I>Pseudovibrio ascidiaceicola</I>. 



Meanwhile the plasmid of the transposase was tested, and the project to create a viable plasmid for pseudovibrio was abandoned (<A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A> ).
Moreover, our third hypothesis of a system which would degrade exogen DNA was supported by BLAST data, and the presence of the enzyme EcoRI in the genome (see section <A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome assembly</A>). This enzyne is indeed known to degrade DNA without a certain pattern of methylation.<DIV align="justify"><DIV align="left"><FONT color="#0099CC"><H5><B>References:</B></H5></FONT></DIV></DIV></LI><LI>Bondarev V, Richter M, Romano S, Piel J, Schwedt A, Schulz-Vogt HN. The genus Pseudovibrio contains metabolically versatile bacteria adapted for symbiosis. Environ Microbiol. 2013 Jul</LI><LI>Katzke N, Arvani S, Bergmann R, Circolone F, Markert A, Svensson V, Jaeger KE, Heck A, Drepper T. A novel T7 RNA polymerase dependent expression system for high-level protein production in the phototrophic bacterium Rhodobacter capsulatus. Protein Expr Purif. 2010 Feb.</LI><LI>Katzke N, Bergmann R, Jaeger KE, Drepper T. Heterologous high-level gene expression in the photosynthetic bacterium Rhodobacter capsulatus. Methods Mol Biol. 2012.</LI><LI>Montalvo NF, Hill RT. Sponge-associated bacteria are strictly maintained in two closely related but geographically distant sponge hosts. Appl Environ Microbiol. 2011 Oct</LI><LI>Muscholl-Silberhorn A, Thiel V, Imhoff JF. Abundance and bioactivity of cultured sponge-associated bacteria from the Mediterranean sea. Microb Ecol. 2008 Jan</LI><LI>Piekarski T, Buchholz I, Drepper T, Schobert M, Wagner-Doebler I, Tielen P, Jahn D. Genetic tools for the investigation of Roseobacter clade bacteria. BMC Microbiol. 2009 Dec 18;9:265.</LI><LI>Shieh WY, Lin YT, Jean WD. Pseudovibrio denitrificans gen. nov., sp. nov., a marine, facultatively anaerobic, fermentative bacterium capable of denitrification. Int J Syst Evol Microbiol. 2004 Nov</LI></DIV><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Biology/Chassis">http://2014.igem.org/Team:Evry/Biology/Chassis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Biology/Chassis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Biology/Chassis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Biology/Chassis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Biology/Chassis&amp;oldid=245241" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>