<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Biology_Sensors"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Biology/Sensors">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Biology/Sensors&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/Sensors&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/Sensors&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Biology/Sensors" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Biology/Sensors</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/b/b0/DNA.jpeg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Biology - Classic &amp; RNAseq-based sensors</H1></DIV></DIV></SECTION><SECTION id="speakers"><DIV class="content-wrapper"><DIV class="col-lg-10 col-lg-offset-1"><DIV class="content-wrapper"><DIV class="col-lg-10 col-lg-offset-1"><DIV align="left"><H3><U><B>Phenol biosensor :</B></U></H3><DIV align="justify"><P><UL>

Phenol and its derivative are of major concern since their accumulation in the environment, as a result of
intensive human activity, cause toxicity for both flora and fauna  at relatively low concentration [1][2].In the context of bioremediation and preservation of water  we designed a biosensor of phenol  that allows us to measure the concentration of phenol in a given marine environment.
This phenol biosensor rely on both signal transducing component, DmpR and inducible fluorescence emitting component based on the Green Fluorescent Protein (GFP). Both elements are assembled in a single plasmid and allow bacteria  to respond to the presence of phenol by emitting fluorescence.
</UL></P><LI><U><B><I>Biosensor construction</I></B></U></LI><LI><U><B><I>Dmpr, a phenol dependant signal transducer</I></B></U><U><B>Presentation</B></U>
DmpR is a member of NtrC protein family. NtrC-type regulators activate RNAP containing the alternative sigma factor 54. The s54-RNAP holoenzyme forms a stable complex with -12 and -24 promoters but is unable to start transcription without further activation NtrC protein family strongly stimulate the polymerase complex. They bind to DNA regions more than 100bp upstream from the s54-RNAP binding site (UAS). Interaction between the regulatory protein and s54-RNAP is facilitated by a bend of DNA. <U><B>Structure </B></U>
DmpR is a 563 amino acid long protein. Although no direct structural information have been described (e.g protein purification), comparisons with other member of NtrC family and genetic experiments have brought some insight on its structure. Four domains are classically described for members of NtrC family : <U>A-domain</U> (211 amino acid long) involved in direct interaction with effector. One inducer binding site is present per monomer, which was demonstrated for DmpR  and which could be pinpointed to a subregion between amino acid residues 107 and 186<U>B-domain</U> is a linker between A and C domain. <U>C-domain </U>being the most highly conserved region among the family members, is involved in ATP binding and hydrolysis and in s54-RNAP interaction<U>D-domain</U> for  Dna binding with HTH motif (helix turn helix).
<U><B>Effectors dependant activation</B></U>

 DmpR-like activators require a chemical effector and ATP as the cofactor. The effector is usually the primary substrate of the target pathway or a compound related to this. Phenol and its derivative are typical effector for DmpR
<U><B>Mechanism</B></U>
•	<U>1. Binding with effector</U> :
The key control event is a direct interaction of aromatic effectors (phenol) with A-domain, which leads to the expression of its otherwise repressed C-domain mediated ATPase activity (Shingler and Pavel, 1995).
B domain act as a linker between A and C domains and is necessary for C domain derepression in presence of effector.
Figure 4 : DmpR binding with  phenol and ATP
  

•	<U>2. Binding with ATP trigger  hexamerisation </U> :
DmpR appears to be a dimer in solution. Following  effector captation by A- domain, binding of ATP can trigger an apparent dimer to hexamer switch in subunit conformation. [3] 

3. Hexamer configuration and ATPase activity allows Transcription activation

 </LI><LI><U><B><I>Results :</I></B></U> 
We prepared a protocol test to evaluate our Biosensor:
E.coli (DH5apha) was grown overnight in M9 medium at 37 °C and then diluted 100-fold to an OD of 0.01 in fresh M9 medium containing Chloramphenicol in 96-well plates. After 6 hours’ of culture at 37 °C, each culture (200 μL) was centrifuged at 2500 r.p.m. for 15 minutes and was suspended in 200 μL of fresh M9 medium containing phenol of different concentrations  Then the fluorescence intensity of cultures was measured by microplate reader (TECAN).

We compared our biosensor (BBa k1413001-Green) with another version of it (BBa K1413002-Blue) that have been obtain by performing a mutation of 3 nucleotides within the sequence of RBS B0032 of sfGFP.

The novel RBS sequence has been designed in accordance with the consensus Shine-Dalgarno sequence: AGGAGGUAA and allows the RNA to bind more tightly to the 16S ribosome to initiate translation.

RBS B0032 = tcacacaggaaag
New RBS = tcaaggaggaaag 

To achieve this experiment we achieve a site directed mutagenesis using the two following primers :

Here are our results: <B>OD 600 of Ecoli carrying BBa K1413001 (RBS B0032- Green graph) and BBa K1413002 (new RBS - Blue graph)</B><B> Raw data of fluorescence expressed by Ecoli carrying BBa K1413001 (RBS B0032- Green graph) and BBa K1413002 (new RBS - Blue graph). Bottom graph shows raw fluorescence obtained by purified GFP at indicated concentration.</B><B>  The induction ratio was calculated by dividing the fluorescence intensity of biosensors exposed to phenol by the basal fluorescence intensity of the biosensors.</B> 
Here show a significant increase of sfGFP expression in both biosensors.
Note that BBa K1413002 allows significant sensing of phenol at a concentration of 1µM.

Reference List

[1] Bacterial Transcriptional Regulators for Degradation Pathways of Aromatic Compounds David Tropel 2004
[2] Aromatic ligand binding and intramolecular signalling of the phenol-responsive s54-dependent regulator DmpR Eric O’Neill 1998
[3] The Regulatory N-terminal Region of the Aromaticresponsive Transcriptional Activator DmpR Constrains Nucleotide-triggered Multimerisation Petra WikstroÈm 2001
[4] Regulation of the transcriptional activator NtrC1: structural studies of the regulatory and AAA+ ATPase domains Seok-Yong Lee 2003

<H3><U><B>PCBs biosensor :</B></U></H3></LI><LI><U><B><I>A system of detection and degradation of polychlorinated biphenyls in Pseudomonas pseudoalcaligenes KF707</I></B></U>
      Pseudomonas pseudoalcaligenes KF707 is one of the strain which are able to degrade polychlorinated biphenyls (PCBs). This strain can grow on biphenyl and salicylate as a sole source of carbon. The bacteria contains a biphenyl-catabolic (bph) gene cluster (bphR1A1A2-(orf3)-bphA3A4BCX0X1X2X3D) which degrade compounds (figure A). 
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER><U><B>Figure A: Organization of the <I>bph</I> gene cluster from Pseudomonas pseudoalcaligenes which is implied in degradation of PCBs (K. Furukawa and H. Fujihara, 2008)</B></U></CENTER></DIV></DIV></DIV></DIV>bphR1 is a positive regulator for bphR1 protein, which belongs to the GntR family,  and for bphX0X1X2X3D. It's  implied in the degradation of PCBs. Watanabe <I>et al</I> showed that there is an other regulatory protein, bphr2, which is involved in the positive regulation of bphA1A2A3A4BC genes. In the absence of biphenyl, small amounts of bphR2 protein binds to bphR2 operator to repress bphR2 transcription (autorepression) and activate bphR1 weakly. When there is biphenyl in the media, bphR2 protein binds to bphR1 and bphA1A2A3A4BC operators to activate strongly their transcription. This allows to initiate the degradation of biphenyl. (figure B).
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER><U><B>Figure B: Cross-regulation of bph gene cluster by bphR1 and bphR2 in Pseudomonas pseudoalcaligenes KF707 (K. Furukawa and H. Fujihara, 2008)</B></U></CENTER></DIV></DIV></DIV></DIV></LI><LI><U><B><I>2013 Saclays' project : </I></B></U>
In 2013, Saclays' team wanted to construct a <A href="https://2013.igem.org/Team:Paris_Saclay/Project">biosensor for PCBs</A>, a project that failed (figure C). So our first aim was to do their construction and then to optimize it and to characterize it.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER><U><B>Figure C: 2013 Saclay's project: construction of a PCBs biosensor</B></U></CENTER></DIV></DIV></DIV></DIV></LI><LI><U><B><I>How function our biosensor ?</I></B></U>
Our construction is composed by a constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23114">(BBa_J23114)</A>, a RBS <A href="http://parts.igem.org/Part:BBa_B0034">(BBa_B0034)</A>, bphR2 gene  <A href="http://parts.igem.org/Part:BBa_K1413021">(BBa_K1413021)</A>, which has been mutated because of a pstI site in its sequence,  bphR1 promoter region <A href="http://parts.igem.org/Part:BBa_K1155001">(BBa_K1155001)</A>, received from Saclay’s team, RFP <A href="http://parts.igem.org/Part:BBa_E1010">(BBa_E1010)</A> and a terminator <A href="http://parts.igem.org/Part:BBa_B0015">(BBa_B0015)</A> (figure D). Instead to use a system of detection with lacZ, we have opted for a fluorescence system with RFP.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER><U><B>Figure D: PCBs biosensor</B></U></CENTER></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER><U><B>Figure E: Mechanism of our biosensor</B></U></CENTER></DIV></DIV></DIV></DIV><U> In absence of PCBs : </U>bphR2 <A href="http://parts.igem.org/Part:BBa_K1413021">(BBa_K1413021)</A> is bound to bphR1 promoter which activate the transcription of RFP but in very low expression.
 <U> In presence of PCBs : </U>When compound  diffuses into the media, it binds to bphr2 protein which undergoes a conformational change that permits to activate more stronger bphR1 promoter and increase the transcription of RFP.
</LI><LI><U><B><I>Improvement</I></B></U>
In order to improve their construction, we have made some modifications. In fact, we have used to add RBS B0034 to increase the level of transcription. After bphR2 gene, we have put a terminator to prevent leakage of constitutive promoter. In their sequence of bphR2 <A href="http://parts.igem.org/Part:BBa_K1155009">(BBa_K1155009)</A>, there is a pstI site so this gene has been mutated by keeping the same codon <A href="http://parts.igem.org/Part:BBa_K1413021">(BBa_K1413021)</A>.
</LI><LI><U><B><I>Construction of our biosensor in E. coli DH5a  : </I></B></U>
Every parts come from the registry except bphr2 gene. Knowing that there is a pstI site in its sequence, bphR2 gene has been synthetized without this site to allow to do the construction at biobrick format.
<B>Golden gate : </B> We wanted to assemble all the parts by using golden gate so in one shot. For that, we have followed this processus:
</LI><LI>Transformation of all the parts in DH5a by heat shock
</LI><LI>Miniprep
</LI><LI>PCR with primers designed especially for the golden gate. Theses primers have been designed so as to have a bsaI site.
</LI><LI>PCR clean-up
</LI><LI>Launching of golden gate program (cf notebook)
</LI><LI> Transformation in DH5a
=&gt; Several tests have been done without success.
<B>Sequential digestions : </B>
Each part contains biobrick prefix and biobrick suffix so we have decide to do a succesion of digestion-ligation-transformation protocol (figure F).
We have successed to obtaine our biobricks <A href="http://parts.igem.org/Part:BBa_K1413023">BBa_K1413023</A> and <A href="http://parts.igem.org/Part:BBa_K1413024">BBa_K1413024</A><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER><U><B>Figure F: biobricks BBa_K1413023 (a) and BBa_K1413024 (B)</B></U></CENTER></DIV></DIV></DIV></DIV></LI></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Biology/Sensors">http://2014.igem.org/Team:Evry/Biology/Sensors</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Biology/Sensors" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Biology/Sensors" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Biology/Sensors&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Biology/Sensors&amp;oldid=401724" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>