<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Biology_CellCharacterization_TransformationProtocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Biology/CellCharacterization/TransformationProtocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Biology/CellCharacterization/TransformationProtocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/CellCharacterization/TransformationProtocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/CellCharacterization/TransformationProtocol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Biology/CellCharacterization/TransformationProtocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Biology/CellCharacterization/TransformationProtocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H2>Transformation Protocol</H2><DIV align="justify"><P>
First, we tried to transform <I>Pseudovibrio denitrificans</I> by a heat shock. Chemically competent cells were prepared with CaCl2 solution. Transformation protocols were carried out by mixing 50 µL of cells with 1 µL of plasmid pSB1C3 solution (35ng/µL). After the standard heat shock protocol for <I>E.coli</I>, cells were spread on 1X MB medium plates. We did not obtain any transformant from this protocol.
By studying transformation of bacteria phylogenetically close of our strain <I>Pseudovibrio denitrificans</I>, we noticed that marines bacteria are often resistant to common chemical transformation approaches (Piekarski T, Buchholz I, 2009) . </P><P>
We considered testing protocols of conjugation and electroporation to transform our bacteria. Unfortunately, due to the lack of time we just tested the electroporation approach.
Two important parameters had to be finalized though; the type and the number of cell washes and also the voltage.</P><LI><U>Washing test</U>
Marine bacteria live in a salty environment. Thus, a classic water wash may to lead to an osmotic shock. Therefore, we tested washes with glycerol 10% and sorbitol 2% to try to limit the shock. <DIV align="center"><B><U>Figure6:</U></B> Protocol of washing tests</DIV>
Pre-cultures of <I>Pseudovibrio denitrificans</I> and <I>E.coli</I> cells were prepared and grew in an shaking incubator overnight at 30°C and 37°C respectively. After performing the protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>) of electrocompetents cells, 20µL of pre-cultures were spread when the DO(600nm) reached 1.5 for <I>Pseudovibrio</I> and 0.5 for <I>E.coli</I>, on plates corresponding to the media used for pre-cultures. CFU were then counted for all plates (Table5).<DIV align="center"><B><U>Table5:</U></B> Results of washing test, number of CFU for the different washes.</DIV>
Washes tests reveal that sorbitol is the best way to wash cells in MB pre-culture. Gycerol can also be used. For pre-cultures in M9, the better way to wash cells is by using glycerol.</LI><LI><U>Voltage test</U>
For this part, we had to find the voltage which allows the best efficiency, by not killing the cells but by being important enough to transform our bacteria.
To achieve this goal, we developed a similar protocol as the standard protocol described previously, by testing different voltages on our cells, without plasmids (Figure7).<DIV align="center"><B><U>Figure7:</U></B> Protocol of voltage tests</DIV>

Pre-cultures of <I>Pseudovibrio denitrificans</I> and <I>E.coli</I> were prepared and grew overnight in a shaking incubator, at 30°C and 37°C respectively. After performing the protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>) of electrocompetents cells, we electroporated our cells at either 1200 V, 1800 V, 2000 V and 2200 V. Cells were then incubated respectively at 30°C for <I>Pseudovibrio</I> and 37°C for <I>E.coli</I>. Then we plated the cells on MB or M9 plates according to the media used for the pre-culture. 
Unfortunately, some plates were convered by a cellular lawn and it was impossible to count the CFU. We chose to estimate roughly the cell growths as shown in the Table 6.<DIV align="center"><B><U>Table 6:</U></B> Results of voltage tests.</DIV>

In this table we can see that the higher voltage for which we obtained a very good rate of living cells is 2000 V for Pseudovibrio. According to these results, we chose to do a 2000 V electroporation to transform our <I>Pseudovibrio denitrificans</I></LI><LI><U>Test with pSB1C3</U>
This year, all the biobricks had to be sent in the backbone pSB1C3. For our first try to transform <I>Pseudovibrio denitrificans</I>, we thus chose to test this plasmid.
We made competent cells and then we did the developed transformation protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>), which is an electroporation at 2000 V.<DIV align="center"><B><U>Figure8:</U></B> Maps of PSB1C3.</DIV>

After a lot of tries, we did not succeed to have transformant of <I>Pseudovibrio denitrificans</I> which could grow on plates supplemented with chloramphenicol.
Because this plasmid works in <I>E.coli</I> which is a GRAM- bacteria as <I>Pseudovibrio denitrificans</I>, we expected it will work in our strain as well. Unfortunately, the plasmid was not expressed.</LI><LI><U>Other plasmids tested</U>
After trying transforming our strain with a standard plasmid of iGEM, pSB1C3, we tried other plasmids, pBBR1MCS (Katzke N, Arvani S, 2010) and pRhokHI-2 (Katzke N, Bergmann R, 2012). Those plasmids were generously given by Dr Thomas DREPPER from the Heinrich-Heine-Universität Düsseldorf, Institute of Molecular Enzyme Technology (Group of Bacterial Photobiotechnology). Dr DREPPER succeed to transform several strains of a genus very closed to our <I>Pseudovibrio</I>, the genus <I>Roseobacter</I>.

We made competent cells and then we effected the developed transformation protocol (see section <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>), which is an electroporation at 2000 V.<DIV align="center"><B><U>Figure9:</U></B> Maps of PBBR1MCS and pRhoKHI-2.</DIV>

Once again, after a lot of tries, we failed to have transformant of <I>Pseudovibrio denitrificans</I> growing on plates supplemented with chloramphenicol, even if pBBR1MCS is a plasmid supposed to work in almost all GRAM- bacteria and those two plasmids work in bacteria very close to <I>Pseudovibrio</I>.</LI><LI><U>Creating a new plasmid</U>
After trying to transform <I>Pseudovibrio denitrificans</I> with the pSB1C3 plasmid which did not work, we had several hypothesis:   
-The promoter of the Chloramphenicol resistance gene does not work in our bacteria
-The origin of replication does not work in our bacteria
-There is a system degrading exogene DNA in <I>Pseudovibrio</I>
To verify these hypothesis, we chose to find specific constitutive promoters and origin of replications in an other member of the genus <I>Pseudovibrio</I>, which genome has been sequenced, the FOBEG1 strain. (lien article) 
A strong constitutive promoter of FOBEG1 was found with a genome browser. Upstream sequences of vital and cell cycle independant genes were explored. We were interested by the transkelotase, a key enzyme of the pathway of pentoses phosphates.
We can assume this enzyme should have a good constitutive promoter, or at least have a reliable and strong expression.
To be sure to have the whole promoter sequence, we exported the sequence (fasta format) from the beginning of the ORF of the transkelotase, to the end of the previous ORF. This sequence was amplified with primers 5 and 6 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>).
After a PCR, we obtained a 6000pb band for <I>Pseudovibrio ascidiaceicola</I>, and no band for <I>Pseuvibrio denitrificans</I>. These PCR product was sent to sequencing, as shown in figure M.<DIV align="center"><B><U>Figure10:</U></B> Result of promoter sequencing.</DIV>

Thus, there is 94.1% of identity between the transketolase promoter of the <I>Pseudovibrio FOBEG1</I> strain and the <I>Pseudovibrio ascidiaceicola</I> strain.
For the replication origin, the aim was here to find the replication origin of the plasmid of the reference strain FOBEG1. To do so, we looked for the repA, repB and repC genes on NCBI. Those three genes follow each other in the genome.
This sequence was amplified with primers 49 and 50 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>).<DIV align="center"><B><U>Figure11:</U></B> Result of ORI amplification in <I>Pseudovibrio denitrificans</I> (Pd), <I>Pseudovribrio ascidiaceicola</I> (Pa) and <I>E.coli</I> DH5a.</DIV>
The negative control with <I>E.coli</I> DH5a confirmed that there is no amplification. For <I>Pseudovibrio</I> strains, we obtained one band for <I>Pseudovibrio denitrificans</I> and two for <I>Pseudovibrio ascidiaceicola</I>. 



Meanwhile the plasmid of the transposase was tested, and the project to create a viable plasmid for pseudovibrio was abandoned (<A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A> ).
Moreover, our third hypothesis of a system which would degrade exogen DNA was supported by BLAST data, and the presence of the enzyme EcoRI in the genome (see section <A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome assembly</A>). This enzyne is indeed known to degrade DNA without a certain pattern of methylation.<DIV align="justify"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Biology/CellCharacterization/TransformationProtocol">http://2014.igem.org/Team:Evry/Biology/CellCharacterization/TransformationProtocol</A>&quot;</DIV></DIV></LI></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Biology/CellCharacterization/TransformationProtocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Biology/CellCharacterization/TransformationProtocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Biology/CellCharacterization/TransformationProtocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Biology/CellCharacterization/TransformationProtocol&amp;oldid=379484" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>