<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Interlab_Study"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Interlab_Study">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Interlab_Study&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Interlab_Study&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Interlab_Study&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Interlab_Study" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Interlab Study</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" class="top-section-title"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Interlab study</H1></DIV></DIV></SECTION><SECTION id="timeline"><DIV id="cd-timeline" class="cd-container"><DIV id="tab-circle" class="row about-details"><DIV class="col-md-4 col-sm-4 col-xs-6"><DIV class="animated hiding" data-animation="fadeInDown" data-delay="0"><FONT color="white"><H5>Chassis</H5></FONT></DIV></DIV><DIV class="col-md-4 col-sm-4 col-xs-6"><DIV class="animated hiding" data-animation="fadeInDown" data-delay="500"><FONT colors="white"><H5>Transposons</H5></FONT></DIV></DIV><DIV class="col-md-4 col-sm-4 col-xs-6"><DIV class="animated hiding" data-animation="fadeInDown" data-delay="1000"><FONT color="white"><H5>Target</H5></FONT></DIV></DIV></DIV></DIV></SECTION><DIV class="tab-content"><DIV class="tab-pane active" id="chassis"><TITLE>IGEM Evry 2014</TITLE><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/f/f1/Lego.jpg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Interlab study - Aim &amp; Results</H1></DIV></DIV></SECTION><SECTION id="speakers"><DIV class="content-wrapper"><DIV class="col-lg-10 col-lg-offset-1"><DIV align="justify">
The goal of the interlab study is to obtain GFP fluorescence data from the iGEM teams all around the world for about twenty constructions. This will permit to observe the result repeatability across technics, strains and teams.

There were two sub-parts in this study:
<UL><LI>The mandatory one: obtain fluorescence data for three specific genetic devices expressing GFP and compare them.
</LI><LI>The Extra Credit assignments: we choose to study the entire Anderson library of constitutive prokaryotic promoters <A id="igem-link" target="_blank" href="http://parts.igem.org/Promoters/Catalog/Anderson " title="Go to iGEM site">(J23100 to J23119)</A>.

This library corresponds to 19 constitutive promoters containing some nucleotide mutations on the -35 and the -10, as shown <A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices"> Used Biobricks and plasmids </A>. 
The idea is to compare the expression strength of promoters according to mutations, from the maximum amount of fluorescence data in order to have significant results.
</LI></UL></DIV><H4><FONT color="blue">Required Devices</FONT></H4><DIV align="justify">
The three required constructions shown various fluorescence intensity at OD 600 nm = 0.5, as shown on graph and bar chart below.  As shown on corrected GFP fluorescence intensity according to OD 600 nm, the relationship between corrected GFP fluorescence and OD 600 nm is linear that corresponded to the expected profile for a constitutive promoter.

Theoritically constructions I20260 and J3101 were similar, constructions and alignments in sillico matched perfectly. The difference between these constructions lies on the assembly way. However we obtained, as some other teams (University of Oxford, Paris-Bettencourt and ITESM-CEM), significant GFP expression difference. There is probably a Biobrick standard  construction problem, which relates to the traited question on our <A href="https://2014.igem.org/Team:Evry/Policy_and_Practices/Philosophy"> Phylosophy </A> subsection.

</DIV><H4><FONT color="blue">Entire Anderson library of constitutive promoters (J23100-J23119)</FONT></H4><DIV align="justify">
Our team achieved the 19 constructions of the Anderson library. Constructions were first designed in a software(Geneious v. 6.1.6)and then relized on web lab using iGEM 2014 Distribution kit. Verifications were made by eletrophoresis gel analytic PCR amplification and sequencing. Glycerol stock of 3 colonies per constructions were conserved and used for the measurments of GFP expression.

Unfortunatly, we did not collect satisfying data for the 19 constructions,although assays were repeated several times. The data profile was repeatable under people, apparatus and strain. Strain carrying promotorless vector seemed to had higher fluorescence intensity than strain containing constructions from the Anderson library, that explains negative curve on Corrected GFP fluorescence intensity according to OD 600 nm graph.
However exploitable data were collected for 5 constructions (100, 104, 105, 107 and 118) for which  the relationship between corrected GFP fluorescence and OD 600 nm was linear that corresponded to the expected profile for a constitutive promoter (See graph Corrected GFP fluorescence intensity according to OD 600 nm). Corrected GFP fluorescence intensity at OD 600 nm = 0.45 of these 5 constructions were presented on bar chart above and compared with <A id="igem-link" target="_blank" href="http://parts.igem.org/Promoters/Catalog/Anderson " title="Go to iGEM site">registry data</A> on table below.
</DIV><DIV align="justify"><I>Ratio were calculated dividing constructions corrected GFP fluorescence at 0.45 OD 600 nm by construction 100 corrected GFP fluorescence at 0.45 OD 600 nm.</I></DIV><DIV align="justify">
Results were similar to registry results for the 100, 105 and 107 constructions contrary to constructions 104 and 118 for which data were significantly divergent.



Possible approach to overcome data acquisition problems:
<UL><LI> Using M9 medium instead of LB medium to minimize the basal medium fluorescence </LI><LI> Using another E. coli strain that has a smaller natural fluorescence</LI><LI> Changing of dilution and gain to find the optimal one</LI><LI> Choosing a TECAN with a lower detection limits to collect data for weak promoters</LI></UL></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="transposons"><TITLE>IGEM Evry 2014</TITLE><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/f/f1/Lego.jpg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Interlab study - Used devices</H1></DIV></DIV></SECTION><SECTION id="speakers"><DIV class="content-wrapper"><DIV class="col-lg-10 col-lg-offset-1"><H4><FONT color="blue">Construction way</FONT></H4><DIV class="container"><DIV class="col-md-6" align="justify"><P style="text-align:left"><I> Constructions were built using classical Biobrick restriction sites. 
 </I></P><UL style="text-align:left"><LI> Required device 1: originally this device was carried by PSB1K3 and was sud-cloned onto PSB1C3.</LI><UL style="text-align:left"><LI> Vector: digestion of PSB1C3 and PSB1K3 carrying I20260 via EcoRi and PstI</LI><LI> Insert: digestion of PSB1K3 carrying I20260 via EcoRi and PstI</LI><LI>Ligation with T4 DNA ligase (New England Biolabs)</LI></UL><LI> Required device 2, 3 and the 19 constructions for the entire Anderson library: promoters were inserted into PSB1C3 carrying E0240. Process is described on following figure 1.</LI><UL style="text-align:left"><LI>Vector: digestion of PSB1C3 carrying E0240 via EcoRI and XbaI</LI><LI>Insert: digestion of PSB1C3 carrying K823012* and J61002 carrying J23100 to J23119 via EcoRI and SpeI</LI><LI>Ligation with T4 DNA ligase (New England Biolabs)</LI></UL></UL>
* BBa_K823012 corresponds to BBa_J23115 that contains 2 missmatched basepairs.
</DIV></DIV><DIV align="justify">
Constructions were first designed in a software(Geneious v. 6.1.6), Plasmid 1 and 2 below, and then relized on web lab using iGEM 2014 Distribution kit. Verifications were made by eletrophoresis gel analytic PCR amplification and sequencing. Glycerol stock of 3 colonies per constructions were conserved and used for the measurments of GFP expression.
</DIV><DIV class="container"><DIV class="col-md-6"><CENTER>Plasmid map 1: Final construction of PSB1C3 containing E0240 and J23113( EcoRI, PstI, SpeI and XbaI). Constructions carrying J23100 to J23119 and K823012 were similar to this.</CENTER></DIV><DIV class="col-md-6"><CENTER><CENTER>Plasmid map 2: Final construction of PSB1C3 containing I20260( EcoRI, PstI, SpeI and XbaI).</CENTER></CENTER></DIV></DIV><H4><FONT color="blue">Analysis protocol</FONT></H4><DIV align="justify">
 Our team decided to test the fluorescence with a  
    <A id="igem-link" target="_blank" href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1813&amp;ID=1918&amp;Menu=1&amp;Item=21.2.10.1.1 " title="Go to iGEM site">TECAN infiniteM200</A> in 96 well plates. Each assay were performed after a 16 - 18 h preculture on 5 ml of LB medium at 37Â°C with 220 rpm agitation. Cultures were distributed 200 Âµl per well. Three strains per constructions were tested in five replicates. Two controls were added on plates: LB  with chloramphenicol and the strain carrying the promoterless vector on LB  with chloramphenicol. 

    
    Fluorescence data were analyzed via following formula that come from <A id="igem-link" target="_blank" href="http://www.biomedcentral.com/1752-0509/4/55 " title="Go to iGEM site">2010 De Jong et al </A><I> Experimental and computational validation of models of fluorescent and luminescent reporter genes in bacteria </I>. 
    </DIV><DIV align="justify"><I> Equation A: A(t) is the corrected absorbance of DH5 alpha with the functional reporter system, Au(t)the uncorrected   absorbance of DH5 alpha with the functional reporter system and Ab(t)the background absorbance corresponding to LB  with chloramphenicol medium absorbance.
   Equation B: B(t) is the absorbance of the strain carrying the promoterless vector, Iu(t) the uncorrected fluorescence intensity of DH5 alpha with the functional reporter system and Ib(t) the background fluorescence intensity of LB chloramphenicol medium with the DH5 alpha carrying the promoterless vector.</I></DIV><H4><FONT color="blue">Used Biobricks and plasmids </FONT></H4><DIV class="container"><DIV class="col-md-6"><CENTER>Plasmid map 3: BBa_J61002 plasmid map with restriction sites and J23101 promoter ( EcoRI, PstI, SpeI and XbaI). Plasmids carrying J23100 to J23119 were similar to this.</CENTER></DIV><DIV class="col-md-6"><CENTER>Plasmid map 4: PSB1C3 carrying BBa_E0240 map with restriction sites and J23101 promoter ( EcoRI, PstI, SpeI and XbaI).</CENTER></DIV></DIV></DIV></DIV><H4><FONT color="blue">Sequencing results</FONT></H4><DIV class="container"><DIV class="col-md-6"><CENTER> Device I20260 Forward</CENTER><PRE>AGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGCAGCGGCCGCTACTAGTATATAAACGCAGAAAGGCCCACCCGAAGGTGAGCCAGTGTGACTCTAGTAGAGAGCGTTCACCGACAAACAACAGATAAAACGAAAGGCCCAGTCTTTCGACTGAGCCTTTCGTTTTATTTGATGCCTGGCTCTAGTATTATTATTTGTATAGTTCATCCATGCCATGTGTAATCCCAGCAGCTGTTACAAACTCAAGAAGGACCATGTGGTCTCTCTTTTCGTTGGGATCTTTCGAAAGGGCAGATTGTGTGGACAGGTAATGGTTGTCTGGTAAAAGGACAGGGCCATCGCCAATTGGAGTATTTTGTTGATAATGGTCTGCTAGTTGAACGCTTCCATCTTCAATGTTGTGTCTAATTTTGAAGTTAACTTTGATTCCATTCTTTTGTTTGTCTGCCATGATGTATACATTGTGTGAGTTATAGTTGTATTCCAATTTGTGTCCAAGAATGTTTCCATCTTCTTTAAAATCAATACCTTTTAACTCGATTCTATTAACAAGGGTATCACCTTCAAACTTGACTTCAGCACGTGTCTTGTAGTTCCCGTCATCTTTGAAAAATATAGTTCTTTCCTGTACATAACCTTCGGGCATGGCACTCTTGAAAAAGTCATGCTGTTTCATATGATCTGGGTATCTCGCANAGCATTGAACACCATAACCGAAAGTAGTGACAAGTGTTGGCCATGGAACAGGTAGTTTTCCAGTAGTGCAAATAAATTTAAGGGTAAGTTTTCCGTATGTTGCATCACCTTCACCCTCTCCACTGACAGAAAATTTGTGCCCATTAACATCACCATCTAATTCAACAAGAATTGGGACAACTCCAGTGAAAAGTTCTTCTCCTTTACGCATCTAGTACTTTCCTGTGTGACTCTAGTAGCTAGCATAATACCTAGGACTGAGCTNNNTGTAAACTCTNNNANCGGCC </PRE><CENTER> Device J23101 Forward</CENTER><PRE> ATAAAANTNGGCGTNTCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCATCTAGAGTTTACAGCTAGCTCAGTCCTAGGTATTATGCTAGCTTCTAGAGTCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCCGAAGGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTTAAAGAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGAATGGAATCAAAGTTAACTTCAAAATTAGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTTTTACCAGACAACCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCCCAACGAANAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATGGCATGGATGAACTATACAAATAATAATACTAGAGCCAGGCATCANATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAGTCCGGCAANANAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGANAAGATTACTTCGCGTTATGCAGGCTTNCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCTGCGGCGAACGG</PRE><CENTER> Device K823012 Forward</CENTER><PRE>  
TTACTATAAANTAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTTTATAGCTAGCTCAGTCCTAGGTACAATGCTAGCTACTAGAGCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCCGAAGGTTATGTACAGGAAAGAACTATATTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGNNGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTTAAAGAAGATGGAAACATTCTTGGACACANATTGNAATACAACTATAACTCACACAATGTATACATCATGGCAGACAANCNNNAGAATGNAATCAAANTTAACTTCAAAATTANACACAACA
</PRE></DIV><DIV class="col-md-6"><CENTER> Device I20260 Reverse<PRE>TAGGCGTATNANGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTCAGTCCTAGGTATTATGCTAGCTACTAGAGTCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCCGAAGGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTTAAAGAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGAATGGAATCAAAGTTAACTTCAAAATTAGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTTTTACCAGACAACCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCCCAACGAANAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATGGCATGGATGAACTATACAAATAATAATACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGAAAANATTACTTCNCGTTATGCAGGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCTGCGGCGNACGG</PRE><CENTER> Device J23101 Reverse</CENTER><CENTER> Device K823012 Reverse</CENTER></CENTER></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="tab-pane" id="target"><TITLE>IGEM Evry 2014</TITLE><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/f/f1/Lego.jpg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Interlab study - Notebook</H1></DIV></DIV></SECTION><SECTION id="timeline"><DIV id="cd-timeline" class="cd-container"><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>The three required devices</H2><CENTER><CENTER><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I> - <I>PSB1C3+E0240+J23104</I>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (09h00-16h00)<I>PSB1C3+E0240+J23119</I>:   
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (09h00-16h00)<SPAN class="cd-date">Oct 11</SPAN></CENTER></CENTER></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23110</I> - <I>PSB1C3+E0240+J23107</I>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (09h00-16h00)<I>PSB1C3+E0240+J23106</I>:   
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (09h00-16h00)<I>PSB1C3+E0240+J23108</I> - <I>PSB1C3+E0240+J23118</I> - <I>PSB1C3+E0240+J23111</I>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (14h00-21h00)<I>PSB1C3+E0240+J23102</I>:   
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (14h00-21h00)<I>PSB1C3+E0240+J23101</I> - <I>PSB1C3+E0240+J23104</I> - <I>PSB1C3+E0240+J23119</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <SPAN class="cd-date">Oct 10</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23109</I> - <I>PSB1C3+E0240+J23117</I>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (11h00-18h00)<I>PSB1C3+E0240+J23114</I>:   
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (11h00-18h00)<I>PSB1C3+E0240+J23115</I> - <I>PSB1C3+E0240+J23116</I>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (16h00-23h00)<I>PSB1C3+E0240+J23105</I>:   
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (16h00-23h00)<I>PSB1C3+E0240+J23110</I> - <I>PSB1C3+E0240+J23107</I> - <I>PSB1C3+E0240+J23106</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <I>PSB1C3+E0240+J23108</I> - <I>PSB1C3+E0240+J23118</I> - <I>PSB1C3+E0240+J23111</I> - <I>PSB1C3+E0240+J23102</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <SPAN class="cd-date">Oct 09</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (12h00-17h00)<H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (12h00-17h00)<I>PSB1C3+E0240+J23112</I> - <I>PSB1C3+E0240+J23103</I> - <I>PSB1C3+E0240+J23113</I>:   
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (17h00-00h00)<I>PSB1C3+E0240+J23109</I> - <I>PSB1C3+E0240+J23117</I> - <I>PSB1C3+E0240+J23114</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <I>PSB1C3+E0240+J23115</I> - <I>PSB1C3+E0240+J23116</I> - <I>PSB1C3+E0240+J23105</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <SPAN class="cd-date">Oct 08</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (12h00-19h00)<H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM. (12h00-19h00)<H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <SPAN class="cd-date">Oct 07</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2>
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h at 37Â°C. <H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Oct 06</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (19h00-02h00).<I>PSB1C3+E0240+J23109</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP &lt; 80nM.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<SPAN class="cd-date">Oct 04</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 80-500nM.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (19h00-02h00).<I>PSB1C3+E0240+J23109</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP &lt; 80nM.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<SPAN class="cd-date">Oct 04</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 60-250nM.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23105</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (17h00-00h00).<I>PSB1C3+E0240+J23107</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23109</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 60-250nM.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23117</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200  (17h00-00h00).<FONT color="red">OVERFLOW</FONT><SPAN class="cd-date">Oct 03</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 60-100nM.<FONT color="red">TECAN FAILED</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23105</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 60-100nM. <FONT color="red">OVERFLOW</FONT><I>PSB1C3+E0240+J23109</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23117</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<SPAN class="cd-date">Oct 02</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT><I>PSB1C3+E0240+J23109</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">TECAN FAILED</FONT><I>PSB1C3+E0240+J23111</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (17h00-00h00).<I>PSB1C3+E0240+J23117</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (9h00-16h00).<SPAN class="cd-date">Oct 01</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT><I>PSB1C3+E0240+J23105</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 40-100nM (10h30-17h30).<I>PSB1C3+E0240+J23107</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23109</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23111</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23117</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<SPAN class="cd-date">Sep 30</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23105</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23109</I>: 
The pre-culture didn't grow.
<SPAN class="cd-date">Sep 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23109</I>:  
PCR colony of 8 colonies for each construction following protocol table 1 and 2.

Pre-culture from colony 1 for glycerol stock and TECAN in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23114</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 20-60nM (15h00-22h00).<SPAN class="cd-date">Sep 28</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT><I>PSB1C3+E0240+J23108</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 60-20nM (14h00-21h00).<SPAN class="cd-date">Sep 27</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT> 
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 40-100nM (10h30-17h30).<I>PSB1C3+E0240+J23114</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (9h30-16h30).<I>PSB1C3+E0240+J23108</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23118</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 5-40nM (18h30-1h30).<SPAN class="cd-date">Sep 26</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 REPORTED.
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23110</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 40-100nM (15h00-22h00).<I>PSB1C3+E0240+J23112</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 60-20nM (15h00-22h00).<I>PSB1C3+E0240+J23114</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23118</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<SPAN class="cd-date">Sep 25</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT>
New pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23110</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23112</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<SPAN class="cd-date">Sep 24</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>: 
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.  
<I>PSB1C3+E0240+J23101</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 40-100nM (19h30-2h30).<I>PSB1C3+E0240+J23113</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 10-40nM (10h-17h).<I>PSB1C3+E0240+J23119</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 with rank of GFP 40-80nM (20h40-3h40).<SPAN class="cd-date">Sep 23</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23107</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">OVERFLOW</FONT><I>PSB1C3+E0240+J23113</I>:  
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23118</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (Xh00-Xh00).<I>PSB1C3+E0240+J23119</I>:  
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">TECAN FAILED</FONT><SPAN class="cd-date">Sep 22</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>


Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>: <UL><LI>Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
</LI><LI>Measure of the cells' growth and fluorecence on TECAN infiniteM200 (18h30-2h30).<FONT color="red">OVERFLOW</FONT></LI></UL><I>PSB1C3+E0240+J23104</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (1h30-8h30).<I>PSB1C3+E0240+J23105</I>: <UL><LI>Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
</LI><LI>Measure of the cells' growth and fluorecence on TECAN infiniteM200 (18h30-2h30).</LI></UL><I>PSB1C3+E0240+J23107</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<I>PSB1C3+E0240+J23115</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (10h50-17h50).<I>PSB1C3+E0240+J23116</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (11h00-18h00).<I>PSB1C3+E0240+J23117</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (1h00-8h00).<I>PSB1C3+E0240+J23118</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<SPAN class="cd-date">Sep 19</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (00h-07h).<H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I>: <LI>Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
</LI><LI>Measure of the cells' growth and fluorecence on TECAN infiniteM200 (16h43-23h43).<I>PSB1C3+E0240+J23102</I>: </LI><LI>Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
</LI><LI>Measure of the cells' growth and fluorecence on TECAN infiniteM200 (16h30-23h30). <I>PSB1C3+E0240+J23104</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<I>PSB1C3+E0240+J23105</I>: 
glycerol stock preparation from LB culture of the 17th September. 
<I>PSB1C3+E0240+J23107</I>: 
glycerol stock preparation from LB culture of the 17th September. 
<I>PSB1C3+E0240+J23110</I>:  
Pre-culture does'nt grow. The TECAN measure is reported.
<I>PSB1C3+E0240+J23115</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<I>PSB1C3+E0240+J23116</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<I>PSB1C3+E0240+J23117</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<I>PSB1C3+E0240+J23114</I>: 


</LI><LI> PCR colony of 5 colonies for each construction following protocol table 1 and 2.


<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 1: 1% agarose gel. PCR product of 5 colonies of the PSB1C3+E0240+J23114 construction. </CENTER></DIV></DIV></DIV></DIV>
A pre-culture was made with one colony to make a glycerol stock.<SPAN class="cd-date">Sep 18</SPAN></LI></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (11h-16h).<FONT color="red">OVERFLOW</FONT><I>PSB1C3+E0240+J23101</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">TECAN FAILED</FONT><I>PSB1C3+E0240+J23103</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (16h30-23h30).<I>PSB1C3+E0240+J23105</I>:  
pre-culture of the colony 5 in LB+Cam (1:1000) according to the result of PCR of 13th September. Incubation overnight at 37Â°C.<I>PSB1C3+E0240+J23107</I>:  
pre-culture of the colony 5 in LB+Cam (1:1000) according to the result of PCR of 13th September. Incubation overnight at 37Â°C.<I>PSB1C3+E0240+J23110</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23115</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200 (12h-19h).

PSB1C3+E0240+J23104, PSB1C3+E0240+J23112, PSB1C3+E0240+J23113: 
PCR colony of 6 colonies for each construction following protocol table 1 and 2.
Migration of the PCR products
PCR purification of clones with the good construction <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL : 1% agarose gel. PCR product of 1 colony of the PSB1C3+E0240+J23104, PSB1C3+E0240+J23112 and PSB1C3+E0240+J23113 constructions. </CENTER></DIV></DIV></DIV></DIV>

A pre-culture was made for each colony to make a glycerol stock.<SPAN class="cd-date">Sep 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23101</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23102</I>: 
<UL><LI>glycerol stock preparation from LB culture from the 14th September. 
</LI><LI>Measure of the cells' growth and fluorecence on TECAN infiniteM200.(18h30-1h30):
</LI></UL><I>PSB1C3+E0240+J23106</I>: 
<UL><LI>glycerol stock preparation from LB culture from the 14th September. 
</LI><LI>Measure of the cells' growth and fluorecence on TECAN infiniteM200.(2h-9h):
</LI></UL><I>PSB1C3+E0240+J23115</I> - <I>PSB1C3+E0240+J23115</I>: 


<UL><LI> PCR colony of 5 colonies for each construction following protocol table 1 and 2.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 1: 1% agarose gel. PCR product of 5 colonies of the PSB1C3+E0240+J23116 and PSB1C3+E0240+J23115 constructions.</CENTER></DIV></DIV></DIV></DIV></LI></UL>
A pre-culture was made with one colony for each construction to make a glycerol stock.<I>PSB1C3+E0240+J23117</I> - <I>PSB1C3+E0240+J23119</I>: 
<UL><LI> PCR colony of 6 colonies for each construction following protocol table 1 and 2.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 2: 1% agarose gel. PCR product of 6 colonies of the PSB1C3+E0240+J23117 and PSB1C3+E0240+J23119 constructions.</CENTER></DIV></DIV></DIV></DIV></LI></UL>
A pre-culture was made with one colony for each construction to make a glycerol stock.<SPAN class="cd-date">Sep 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Pre-culture from glycerol stock in LB+Cam (1:1000). Incubation 16h-18h at 37Â°C.
<H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23100</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. 
<I>PSB1C3+E0240+J23106</I>:  
Pre-culture from glycerol in 3mL of LB+Cam (1:1000). Incubation 16h-18h at 37Â°C. <SPAN class="cd-date">Sep 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I>: 
Measure of the cells' growth and fluorecence on TECAN infiniteM200.<FONT color="red">TECAN FAILED</FONT><I>PSB1C3+E0240+J23106</I>:
pre-culture of the colony 3 in LB+Cam (1:1000) according to the result of PCR of 13th September. Incubation 16h at 37Â°C.
<SPAN class="cd-date">Sep 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I>:
pre-culture of the colony 3 in LB+Cam (1:1000). Incubation 16h at 37Â°C.
<I>PSB1C3+E0240+J23105</I>, <I>PSB1C3+E0240+J23106</I> and <I>PSB1C3+E0240+J23107</I>: 


<UL><LI> PCR colony of 7 colonies for each construction following protocol table 1 and 2.


<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 1: 1% agarose gel. PCR product of 7 colonies of the PSB1C3+E0240+J23107 construction. PCR product of 8 colonies of the PSB1C3+E0240+J23106 construction. </CENTER></DIV></DIV></DIV></DIV>
A pre-culture was made with one colony for each construction to make a glycerol stock.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 2: 1% agarose gel. PCR product of 7 colonies of the PSB1C3+E0240+J23105 construction.</CENTER></DIV></DIV></DIV></DIV></LI></UL>
A pre-culture was made with one colony for each construction to make a glycerol stock.<SPAN class="cd-date">Sep 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I> and <I>PSB1C3+E0240+J23118</I>:
glycerol stock preparation from LB culture from the 11th September.



<SPAN class="cd-date">Sep 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23101</I>, <I>PSB1C3+E0240+J23118</I>, <I>PSB1C3+E0240+J23105</I>, <I>PSB1C3+E0240+J23106</I> and <I>PSB1C3+E0240+J23107</I>: 
      <UL><LI> Transformation plate from the 10th September observation. There were 50 colonies for each constructions.</LI></UL><I>PSB1C3+E0240+J23101</I> and <I>PSB1C3+E0240+J23118</I>,<UL><LI> PCR colony of 3 colonies for each construction following protocol table 1 and 2.


<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 1: 1% agarose gel.  Lane 1 to 3: PCR product of 3 colonies of the PSB1C3+E0240+J23101 construction.  Lane 4 to 6: PCR product of 3 colonies of the PSB1C3+E0240+J23118 construction. </CENTER></DIV></DIV></DIV></DIV></LI><LI> PCR purification of sample 1 and 4 of the Gel 1 were purified with the NucleoSpin kit (Macherey Nagel). DNA was quantify by Nanodrop 2000.
        </LI><LI> Preparation of samples to sequencing. NÂ° XX
        </LI><LI> Preparation of 3 ml cultures LB Cam. Incubation overnight at 37Â°C.
      </LI></UL><I>PSB1C3+E0240+J23100</I>, <I>PSB1C3+E0240+J23102</I>, <I>PSB1C3+E0240+J23103</I>, <I>PSB1C3+E0240+J23104</I> and <I>PSB1C3+E0240+J23118</I>,<UL><LI> PCR colony of 1 colony for each construction following protocol table 1 and 2.


<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>GEL 2: 1% agarose gel.  Lane 1: PCR product of 1 colonies of the PSB1C3+E0240+J23100 construction.  Lane 2: PCR product of 1 colonies of the PSB1C3+E0240+J23102 construction. Lane 3: PCR product of 1 colonies of the PSB1C3+E0240+J23103 construction. Lane 4: PCR product of 1 colonies of the PSB1C3+E0240+J23104 construction. </CENTER></DIV></DIV></DIV></DIV>
A pre-culture was made for each colony to make a glycerol stock.<SPAN class="cd-date">Sep 11</SPAN><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2> Miniprep disgestion of <I>PSB1C3+E0240</I> plasmid.
      </DIV></DIV></LI><LI> 1 Âµl EcoRI
        </LI><LI> 1 Âµl XbaI
        </LI><LI> 2 Âµl buffer 2.1
        </LI><LI> 0.2 Âµl BSA
        </LI><LI> 5 Âµl template (miniprep)
        </LI><LI> 10.8 Âµl miliQ watter
      </LI></UL> Miniprep disgestion of <I>BBa_J61002+J23101</I>, <I>BBa_J61002+J23118</I>, <I>BBa_J61002+J23105</I>, <I>BBa_J61002+J23106</I> and <I>BBa_J61002+J23107</I>plasmids.
      <UL><LI> 1 Âµl EcoRI
        </LI><LI> 1 Âµl SpeI
        </LI><LI> 2 Âµl buffer 2.1
        </LI><LI> 0.2 Âµl BSA
        </LI><LI> 5 Âµl template (miniprep)
        </LI><LI> 10.8 Âµl miliQ watter
      </LI></UL>
Incubation 45 minutes at 37Â°C. Denaturation step 20 minutes at 80 Â°C.

 Ligation to obtain constructions <I>PSB1C3+E0240+J23101</I>, <I>PSB1C3+E0240+J23118</I>, <I>PSB1C3+E0240+J23105</I>, <I>PSB1C3+E0240+J23106</I> and <I>PSB1C3+E0240+J23107</I>.
      <UL><LI> 1 Âµl T4 DNA ligase
        </LI><LI> 6 Âµl Insert (<I>digested BBa_J61002+J23101, BBa_J61002+J23118, BBa_J61002+J23105, BBa_J61002+J23106 and BBa_J61002+J23107</I>)
        </LI><LI> 2 Âµl vector (<I>digested PSB1C3+E0240</I>)
        </LI><LI> 2 Âµl buffer
        </LI><LI> 9 Âµl miliQ watter
      </LI></UL>
Incubation 45 minutes at room temperature.
 Transformation in E. coli DH5 alpha with the protocol of the 15th of August. Bacteria were plated on Chloramphenicol agar plates and incubated at 37Â°C overnight.
<SPAN class="cd-date">Sep 10</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Sep 09</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Sep 07
</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><I>PSB1C3+E0240+J23111</I>:  
PCR colony of 7 colonies for each construction following protocol table 1 and 2.

Pre-culture of colony 2 for grycerol stock
<SPAN class="cd-date">Sep 06</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Sep 05</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Sep 04</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Sep 03</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2>
Analyse of sequencing results. The two constructions were ok although the colony 4 had a better match. So we decided to keep this colony to perform promoter activity assays.
26DJ96
GTCAGTGAGCGAGGNAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGCAGC
GGCCGCTACTAGTATATAAACGCAGAAAGGCCCACCCGAAGGTGAGCCAGTGTGACTCTAGTAGAGAGCGTTCACCGACAAACAACAGATAAAACGAAAGGCCCAGT
CTTTCGACTGAGCCTTTCGTTTTATTTGATGCCTGGCTCTAGTATTATTATTTGTATAGTTCATCCATGCCATGTGTAATCCCAGCAGCTGTTACAAACTCAAGAAG
GACCATGTGGTCTCTCTTTTCGTTGGGATCTTTCGAAAGGGCAGATTGTGTGGACAGGTAATGGTTGTCTGGTAAAAGGACAGGGCCATCGCCAATTGGAGTATTTT
GTTGATAATGGTCTGCTAGTTGAACGCTTCCATCTTCAATGTTGTGTCTAATTTTGAAGTTAACTTTGATTCCATTCTTTTGTTTGTCTGCCATGATGTATACATTG
TGTGAGTTATAGTTGTATTCCAATTTGTGTCCAAGAATGTTTCCATCTTCTTTAAAATCAATACCTTTTAACTCGATTCTATTAACAAGGGTATCACCTTCAAACTT
GACTTCAGCACGTGTCTTGTAGTTCCCGTCATCTTTGAAAAATATAGTTCTTTCCTGTACATAACCTTCGGGCATGGCACTCTTGAAAAAGTCATGCTGTTTCATAT
GATCTGGGTATCTCGCANAGCATTGAACACCATAACCGAAAGTAGTGACAAGTGTTGGCCATGGAACAGGTAGTTTTCCAGTAGTGCAAATAAATTTAAGGGTAAGT
TTTCCGTATGTTGCATCACCTTCACCCTCTCCACTGACAGAAAATTTGTGCCCATTAACATCACCATCTAATTCAACAAGAATTGGGACAACTCCAGTGAAAAGTTC
TTCTCCTTTACGCATCTAGTACTTTCCTNNGTGACTCTAGTAGCTANCCATAANNCCTAGGAACTGANCTAGCTG
26DJ97
GATTACTANNAATAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTT
TACAGCTAGCTCAGTCCTAGGTATTATGCTAGCTACTAGAGTCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGT
TGAATTAGATGGTGATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAA
AACTACCTGTTCCATGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCC
ATGCCCGAAGGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAAT
CGAGTTAAAAGGTATTGATTTTAAAGAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGA
ATGGAATCAAAGTTAACTTCAAAATTAGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTT
TTACCAGACAACCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCCCAACGAAAAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTAC
ACATGGCATGGATGAACTATACAAATAATAATACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCG
GTGAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAGTCCGGCAAANAAGGGCAAGGTGTCA
CCACCCTGCCCTTTTTCTTTAAAACCGAAAANATTACTT
26DJ98
AGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGCAGC
GGCCGCTACTAGTATATAAACGCAGAAAGGCCCACCCGAAGGTGAGCCAGTGTGACTCTAGTAGAGAGCGTTCACCGACAAACAACAGATAAAACGAAAGGCCCAGT
CTTTCGACTGAGCCTTTCGTTTTATTTGATGCCTGGCTCTAGTATTATTATTTGTATAGTTCATCCATGCCATGTGTAATCCCAGCAGCTGTTACAAACTCAAGAAG
GACCATGTGGTCTCTCTTTTCGTTGGGATCTTTCGAAAGGGCAGATTGTGTGGACAGGTAATGGTTGTCTGGTAAAAGGACAGGGCCATCGCCAATTGGAGTATTTT
GTTGATAATGGTCTGCTAGTTGAACGCTTCCATCTTCAATGTTGTGTCTAATTTTGAAGTTAACTTTGATTCCATTCTTTTGTTTGTCTGCCATGATGTATACATTG
TGTGAGTTATAGTTGTATTCCAATTTGTGTCCAAGAATGTTTCCATCTTCTTTAAAATCAATACCTTTTAACTCGATTCTATTAACAAGGGTATCACCTTCAAACTT
GACTTCAGCACGTGTCTTGTAGTTCCCGTCATCTTTGAAAAATATAGTTCTTTCCTGTACATAACCTTCGGGCATGGCACTCTTGAAAAAGTCATGCTGTTTCATAT
GATCTGGGTATCTCGCANAGCATTGAACACCATAACCGAAAGTAGTGACAAGTGTTGGCCATGGAACAGGTAGTTTTCCAGTAGTGCAAATAAATTTAAGGGTAAGT
TTTCCGTATGTTGCATCACCTTCACCCTCTCCACTGACAGAAAATTTGTGCCCATTAACATCACCATCTAATTCAACAAGAATTGGGACAACTCCAGTGAAAAGTTC
TTCTCCTTTACGCATCTAGTACTTTCCTGTGTGACTCTAGTAGCTAGCATAATACCTAGGACTGAGCTNNNTGTAAACTCTNNNANCGGCC
26DJ99
TAGGCGTATNANGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTC
AGTCCTAGGTATTATGCTAGCTACTAGAGTCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGT
GATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACCTGTTCCA
TGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCCGAAGGTTAT
GTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATT
GATTTTAAAGAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGAATGGAATCAAAGTTAAC
TTCAAAATTAGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTTTTACCAGACAACCATTAC
CTGTCCACACAATCTGCCCTTTCGAAAGATCCCAACGAANAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATGGCATGGATGAACTA
TACAAATAATAATACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGAG
TCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTA
AAACCGAAAANATTACTTCNCGTTATGCAGGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCTGCGGCGNACGG
26EA00
 Impossible to read
26EA01
 Impossible to read

<H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2><H2>Other constructions of the Anderson library of constitutive promoters</H2>
Miniprep cultures from the 2cnd of september was purified with the NucleoSpin Plasmid kit (Macherey-Nagel). DNA was quantify with the Nanodrop 2000.
 Glycerol stock were prepared started from the 16 cultures from the 2cnd of september (750 Âµl of culture + 750 ÂµL of 10% glycerol) and stored at -80Â°C.

<SPAN class="cd-date">Sep 02</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2>
PCR colony were performed on colonies from transformations of the 29th August. Used protocol was the same as Table 1 and 2 with the Q5 high fidelity enzyme.
 PCR product of first colony of each transformation was loaded on 1% agarose gel and gel run 45 minutes at 110 mV.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER> PCR product of BBa_J61002 plasmids containing promoters results. </CENTER></DIV></DIV></DIV></DIV>

3 ml LB culture were started from tested colonies which presented the right PCR product profile. Cultures were incubated overnight at 37Â°C. 


<SPAN class="cd-date">Sep 01</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2><SPAN class="cd-date">Aug 31</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Other constructions of the Anderson library of constitutive promoters</H2>
Transformation plates observation.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER> Colony number according to plasmid including promoters.</CENTER></DIV></DIV></DIV></DIV>

PCR colony were performed on 8 colonies per transformation plate. PCR products were loaded on 1% agarose gel and gel ran one 45 minutes. There were nothing on gel, reason is probably a mix and temperature problem.


<SPAN class="cd-date">Aug 30</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2>
Selected colonies PCR products from the 28/08/2014 were purified with the NucleoSpin kit (Macherey Nagel). DNA was quantify by Nanodrop 2000.
<UL><LI> Colony nÂ°2: 30.7 ng/Âµl  and A260/280 1.92
</LI><LI> Colony nÂ°4: 52.3 ng/Âµl  and A260/280 1.75
</LI><LI> Colony nÂ°7: 26.1 ng/ÂµL  and A260/280 1.72
</LI></UL> Samples were send to sequencing. NÂ° 26DJ96, 26DJ97, 26DJ98, 26DJ99, 26EA00 and 26EA01.

<H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: PSB1C3 with K823012-E1010</H2>
Selected colonies PCR products from the 28/08/2014 were purified with the NucleoSpin kit (Macherey Nagel). DNA was quantify by Nanodrop 2000.
<UL><LI> Colony nÂ°2: 41.4 ng/Âµl 
</LI><LI> Colony nÂ°3: 65.8 ng/Âµl  
</LI><LI> Colony nÂ°4: 27.4 ng/ÂµL  
</LI></UL> Colony nÂ°3 was sent to sequence with the identification 26AE07 (VF2) and 26AE08 (VR)
<H2>Other constructions of the Anderson library of constitutive promoters</H2>
Transformation in DH5 alpha chemocompetent <I>E. coli</I> as doing on 21th August, with ampicilline LB agar plates
 Incubation overnight at 37Â°C.

<SPAN class="cd-date">Aug 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contruction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Contruction nÂ°3: PSB1C3 with K823012-E1010</H2>PCR colony was done on 7 colonies:
<OL><LI>Steril H2O:34,5ÂµL
</LI><LI>5X Q5 Reaction buffer: 10ÂµL
</LI><LI>Template:2ÂµL
</LI><LI>10mM dNTP: 1ÂµL
</LI><LI>10ÂµM Primer forward VF2: 1ÂµL
</LI><LI>10ÂµM Primer reverse VR: 1ÂµL
</LI><LI> Q5 DNA Polymerase: 0,5ÂµL
</LI></OL>A migration of PCR product was done. We expect to observe in each wells a band of 1176bp:
 6 colonies have a band at 1200 bp around so we decided to choose colonies nÂ°2, 3 and 4.
<H2>Other constructions of the Anderson library of constitutive promoters</H2>
Promoters were resuspended from 2014 distribution kit plates with 10 Âµl of sterile MiliQ watter and stored at -20Â°C.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER> Table: Additional promoters of the Interlab Study. In grey, plasmids already resuspended, transformed, sequenced and stock glycerol existed. </CENTER></DIV></DIV></DIV></DIV><SPAN class="cd-date">Aug 28</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contruction nÂ°1: pSB1C3 with I20260</H2><H2>Contruction nÂ°2: pSB1C3 with J23101-E1010</H2><H2> Contruction nÂ°3: pSB1C3 with K823012-E1010</H2>At the end of the day, there were 3 colonies that have grown
<SPAN class="cd-date">Aug 27</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Construction nÂ°1: PSB1C3 with I20260</H2>
Loading of the PSB1C3 digestion by EcoRI and PstI from the 25th August, on a 1% agarose gel. The 40 Âµl were loaded with 8 Âµl of loading dye 6X. Migration was performed 1 hour at 110 mV.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 6: 1% agarose gel of colony PCR products for BBa_E0240 and BBa_I20260. Lane 2 and 3: digestion product of PSB1C3 by EcoRI and PstI, Lane 4: Purple 2-Log ladder NEB</CENTER></DIV></DIV></DIV></DIV><H2>Construction nÂ°2: PSB1C3 with J23101-E1010</H2><H2> Construction nÂ°3: pSB1C3 with K823012-E1010</H2><U>Digestion of the insert pSB1C3 (J23115) with EcoRI and XbaI:</U><UL><LI> Steril H2O: 10,8ÂµL
</LI><LI> Buffer 2.1 NEB : 2ÂµL
</LI><LI> BSA: 0,2ÂµL
</LI><LI>DNA: 5ÂµL
</LI><LI>EcoRI-HF: 1ÂµL
</LI><LI>XbaI: 1Âµl 
</LI></UL><U>Digestion of the vector pSB1C3(E0240) with EcoRI and SpeI:</U><UL><LI> Steril H2O: 10,8ÂµL
</LI><LI> Buffer 2.1 NEB : 2ÂµL
</LI><LI> BSA: 0,2ÂµL
</LI><LI>DNA: 5ÂµL
</LI><LI>EcoRI-HF: 1ÂµL
</LI><LI>SpeI: 1Âµl
</LI></UL>Mix were incubated at 37Â°C during 45mn then at 80Â°C during 20 mn
<U>Ligation:</U><UL><LI> Steril H2O: 9ÂµL
</LI><LI> Insert (J23115):6ÂµL
</LI><LI>Vector (E0240): 2ÂµL
</LI><LI> 10X T4 DNA ligase reaction buffer: 2ÂµL
</LI></UL>Mix was incubated at 10mn at room temperature then at 80Â°C during 20mn before the transformation in DH5a chimiocompetent with 3ÂµL of the ligation product with the same <A href="https://2014.igem.org/Team:Evry/Interlab_Study/08-21-2014">protocol</A><SPAN class="cd-date">Aug 26</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><SPAN class="cd-date">Aug 25</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><SPAN class="cd-date">Aug 24</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contruction nÂ°1: PSB1C3 with I20260</H2>Digestion of PSB1C3 and PSB1K3-I20260 by EcoRI and PstI
      <UL><LI> 1 Âµl EcoRI-HF
        </LI><LI> 1 Âµl PstI
        </LI><LI> 2 Âµl buffer
        </LI><LI> 0.2 Âµl BSA
        </LI><LI> 13.7 or 76 Âµl respectively for I20260 and PSB1C3
        </LI><LI> 10.8 Âµl miliQ watter
      </LI></UL>Incubation 45 minutes at 37Â°C. Denaturation step 20 minutes at 80 Â°C.

Loading on a 1% agarose gel to perform an extraction from gel

<H2>Contruction nÂ°2: PSB1C3 with J23101-E1010</H2><SPAN class="cd-date">Aug 23</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify"> Transformation plate observation:
- BBa_J23101: 11 isolated colonies
- BBa_K823012: 100-150 isolated colonies
 A PCR was performed with 8 colonies for BBa_K823012 and for BBa_J23101 following the protocol Table 3 and 4.
 To verify PCR products, 10 Âµl was loaded on a 1% agarose gel with 2 Âµl of loading dye 6X.

</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>1% agarose gel of PCR products. Lane 1 to 8: BBa_K823012 PCR products and Lane 9: Purple 2-Log. ladder NEB</CENTER></DIV></DIV></DIV></DIV> We expected to obtain a band between 300 and 400 bp. So we decided to amplify the colony nÂ°8. The colony was seed in 5 ml of LB medium and 5 Âµl of chloramphenicol. Culture was incubated overnight at 37Â°C with 200 rpm agitation.
<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>1% agarose gel of PCR products. Lane 10 to 17: BBa_J23101 PCR products and Lane 9: Purple 2-Log ladder NEB</CENTER></DIV></DIV></DIV></DIV> We expected to obtain a band around 1200 bp. So we decided to amplify the colony nÂ°2 corresponding to lane 11. The colony was seed in 5 ml of LB medium and 5 Âµl of chloramphenicol. Culture was incubated overnight at 37Â°C with 200 rpm agitation.

<SPAN class="cd-date">Aug 22</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">Transformations of registry parts (BBa_K823012 and J23101) were performed to obtain colonies with plasmid containing parts for future amplifications.  The transformation was performed on DH5 alpha <I>E. coli</I>, as followed: </P><OL><LI> Remove <I>E. coli</I> competent tubes from -80Â°C and keep it on ice
</LI><LI> Add 3 Âµl of template (here solubilized plasmids from the registry distribution kit) and mix gently
</LI><LI> Incubate 10 minutes on ice
</LI><LI> Perform an heat shock 30 seconds at 42Â°C
</LI><LI> Incubate 2 minutes on ice
</LI><LI> Add 2 ml of LB medium and incubate 60 minutes at 37Â°C with an agitation at 200 rpm
</LI><LI> Plate 200 Âµl of BBa_K823012 or BBa_J23101 on a chloramphenicol LB agar plate.
</LI><LI> Incubate plate overnight at 37Â°C
</LI></OL><SPAN class="cd-date">Aug 21</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">
Miniprep cultures from the 19th August was purified with the NucleoSpin Plasmid kit (Macherey-Nagel). DNA was quantify with the Nanodrop 2000. DNA concentrations and A260/280 were respectively for BBa_E0240 and BBa_I20260: 42 ng/Âµl and 1.91 ; 13.7 ng/Âµl and 1.71.
 A glycerol stock was prepared for each culture and stored at -80Â°C.
</P><SPAN class="cd-date">Aug 20</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">

Preparation of a 1% agarose gel: 1.01 g of Top Vision agarose (Thermo Scientific) + 100 ml of TAE 1X. Microwave 30s by 30s until agarose total dissolution Gel was cooling down until to be lukewarm, one BET drop was added. Gel was loaded with 10Âµl per sample previously added with 2 Âµl of loading dye 6X, and 5 Âµl for ladders. Gel running 45 minutes at 100 mV in TAE 1X buffer.

</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 6: 1% agarose gel of colony PCR products for BBa_E0240 and BBa_I20260. Lane 1 and 10: Purple 2-Log ladder NEB, Lane 2 to 9: BBa_E0240 PCR products, Lane 11 to 17: BBa_I20260 PCR products </CENTER></DIV></DIV></DIV></DIV>

We expected one band at 1200 bp for both BioBrick. So we decided to amplify the colony 1 for BBa_E0240 and colony 1 for  BBa_I20260. PCR products were purified with the GeneJET purification kit (Thermo Scientific) followed by DNA quantification with the NanoDrop 2000 (Thermo Scientific). BBa_E0240 purified PCR product: 61 ng/Âµl and BBa_I20260 purified PCR product: 37.8 ng/Âµl.

 Purified PCR product aliquot was send to sequencing. The sequencing number were 26DJ68, 69, 70, 71. 
 Preparation of miniprep culture of BBa_I20260 and BBa_E0240 selected colonies in 5 ml of LB with respectively 5 Âµl of Kana or chloramphenicol solution stock. Incubation overnight at 37Â°C and 300 rpm.
<SPAN class="cd-date">Aug 19</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify"> Sequencing results were received for BBa_J23101 (26DJ62 and 26DJ63) and BBa_K823012(26DJ64 and 26DJ65) PCR products.
 26DJ62 </P><P align="justify">TCAGANNAAAAAAATCCTTAGCTNNCGCTAAGGATGANNTCTGGAATTCGCGGCCGCTTCTAGAG<B><FONT color="blue">TTTACAGCTAGCTCAGTCCTAGGTATTATGCTAGC</FONT></B>TACTAGTAG CGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGAAAAGATTACTTCGCGTTATGCAGGCTTCCTCGCTCACTGACTC GCTGCGCTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCACTCAGAG</P> 26DJ63<P align="justify">GAGCGCANCGAGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTG CAGCGGCCGCTACTAGTA<B><FONT color="blue">GCTAGCATAATACCTAGGACTGAGCTAGCTGTAAA</FONT></B>CTCTAGAAGCGGCCGCGAATTCCAGAAATCATCCTTAGCGAAAGCTAAGGATTTTT TTTATCTGAAATTCTGCCTCGTGATACGCCTATTTTTATAGGTTAATGTCATGATAATAATGGTTTCTTAGACGCAAGGTGGCAAA</P> 26DJ64<P align="justify">TCAGANNAAAAAAATCCTTAGCTNNCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGG<B><FONT color="blue">TTTATAGCTAGCTCA<FONT color="red">T</FONT>CCT<FONT color="red">A</FONT>GGTACAATGCTAGC</FONT></B>TACTAG TAGCGGCCGCTGCAGTCCGGCAAAAAAGGGCAAGGTGTCACCACCCTGCCCTTTTTCTTTAAAACCGAAAAGATTACTTCGCGTTATGCAGGCTTCCTCGCTCACTGAC
TCGCTGCGCTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCACTCAGG </P> 26DJ65<P align="justify">CNGCGAGTCAGTNAGCGAGGAAGCCTGCANNACGCGAAGTNATCTTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTNGCCCTTTTTTGCCGGACTGCAGC GGCCGCTACTAGTA<B><FONT color="blue">GCTAGCATTGTACCTAGGACTGAGCTAGCTATAAA</FONT></B>CTCTAGAAGCGGCCGCGAATTCCAGAAATCATCCTTAGCGAAAGCTAAGGATTTTTTTTA TCTGAAATTCTGCCTCGTGATACGCCTATTTTTATAGGTTAATGTCATGATAATAATGGTTTCTTAAC</P>The mapping was perfect for BBa_J23101 for BBa_K823012.
Sequencing results were received for BBa_E0240 (26DJ68 and 26DJ69) and BBa_I20260 (26DJ70 and 26DJ71) PCR products.
 26DJ68 
GAGCNCAGCGAGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTG
CAGCGGCCGCTACTAGTATATAAACGCAGAAAGGCCCACCCGAAGGTGAGCCAGTGTGACTCTAGTAGAGAGCGTTCACCGACAAACAACAGATAAAACGAAAGGCCC
AGTCTTTCGACTGAGCCTTTCGTTTTATTTGATGCCTGGCTCTAGTATTATTATTTGTATAGTTCATCCATGCCATGTGTAATCCCAGCAGCTGTTACAAACTCAAGA
AGGACCATGTGGTCTCTCTTTTCGTTGGGATCTTTCGAAAGGGCAGATTGTGTGGACAGGTAATGGTTGTCTGGTAAAAGGACAGGGCCATCGCCAATTGGAGTATTT
TGTTGATAATGGTCTGCTAGTTGAACGCTTCCATCTTCAATTTGTGTCTAATTTTGAAGTTAACTTTGATTCCATTCTTTTGTTTGTCTGCCATGATGTATACATTGT
GTGAGTTATAGTTGTATTCCAATTTGTGTCCAAGAATGTTTCCATCTTCTTTAAAATCAATACCTTTTAACTCGATTCTATTAACAAGGGTATCACCTTCAAACTTGA
CTTCAGCACGTGTCTTGTAGTTCCCGTCATCTTTGAAAAATATAGTTCTTTCCTGTACATAACCTTCGGGCATGGCACTCTTGAAAAAGTCATGCTGTTTCATATGAT
CTGGGTATCTCGCAAAGCATTGAACACCATAACCGAAAGTAGTGACAAGTGTTGGCCATGGAACAGGTAGTTTTCCAGTAGTGCAAATAAATTTAAGGGTAAGTTTTC
CGTATGTTGCATCACCTTCACCCTCTCCACTGACAGAAAATTTGTGCCCATTAACATCACCATCTAATTCAACAAGAATTGGGACAACTCCAGTGAANAGTTCTTCTC
CTTTACGCATCTAGTACTTTCCTGTGTGACTCTAGTAGCTAGCATAATACCTAGGACTGAGCTAGCTGTNAACTCTANAAGCGGCCGCGAATTCCAGAAATCATCCTT
ANCNNAAGCTAAGGATTTTTTTTATCTGNAATTCTGCCTC
 26DJ69
GATTACTATNAAATAGGCGTANNANGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTTTACAG
CTAGCTCAGTCCTAGGTATTATGCTAGCTACTAGAGTCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATT
AGATGGTGATGTTAATGGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACC
TGTTCCATGGCCAACACTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCCGA
AGGTTATGTACAGGAAAGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAA
AGGTATTGATTTTAAAGAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGAATGGAATCAA
AGTTAACTTCAAAATTAGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTTTTACCAGACAA
CCATTACCTGTCCACACAATCTGCCCTTTCGAAAGATCCCAACGAANAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATGGCATGGA
TGAACTATACAAATAATAATACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCT
ACTAGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAGTCCGGCNAAAAAGGGCAAGGTGTCACCACCNTGCCCTTT
TTCTTTAAAACCGNAAA
 26DJ70 
AGCGAGTCAGTGAGCGAGGAAGCCTGCATAACGCGAAGTAATCTTTTCGGTTTTAAAGAAAAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGCAGCGG
CCGCTACTAGTATATAAACGCAGAAAGGCCCACCCGAAGGTGAGCCAGTGTGACTCTAGTAGAGAGCGTTCACCGACAAACAACAGATAAAACGAAAGGCCCAGTCTT
TCGACTGAGCCTTTCGTTTTATTTGATGCCTGGCTCTAGTATTATTATTTGTATAGTTCATCCATGCCATGTGTAATCCCAGCAGCTGTTACAAACTCAAGAAGGACC
ATGTGGTCTCTCTTTTCGTTGGGATCTTTCGAAAGGGCAGATTGTGTGGACAGGTAATGGTTGTCTGGTAAAAGGACAGGGCCATCGCCAATTGGAGTATTTTGTTGA
TAATGGTCTGCTAGTTGAACGCTTCCATCTTCAATGTTGTGTCTAATTTTGAAGTTAACTTTGATTCCATTCTTTTGTTTGTCTGCCATGATGTATACATTGTGTGAG
TTATAGTTGTATTCCAATTTGTGTCCAAGAATGTTTCCATCTTCTTTAAAATCAATACCTTTTAACTCGATTCTATTAACAAGGGTATCACCTTCAAACTTGACTTCA
GCACGTGTCTTGTAGTTCCCGTCATCTTTGAAAAATATAGTTCTTTCCTGTACATAACCTTCGGGCATGGCACTCTTGAAAAAGTCATGCTGTTTCATATGATCTGGG
TATCTCGCAAAGCATTGAACACCATAACCGAAAGTAGTGACAAGTGTTGGCCATGGAACAGGTAGTTTTCCAGTAGTGCAAATAAATTTAAGGGTAAGTTTTCCGTAT
GTTGCATCACCTTCACCCTCTCCACTGACAGAAAATTTGTGCCCATTAACATCACCATCTAATTCAACAAGAATTGGGACAACTCCAGTGAAAAGTTCTTCTCCTTTA
CGCATCTAGTACTTTCCTGTGTGACTCTAGAAGCGGCCGCGAATTCCAGAAATCATCCTTAGCGATGAATTCCAGAAATCATCCTTANCGAAAGCTNAGGATTTTTTT
TATCTGNAATTCTGNCTCG
 26DJ71 
GATTAAACTATNAATAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCATCGCTAAGGATGATTTCTGGA
ATTCGCGGCCGCTTCTAGAGTCACACAGGAAAGTACTAGATGCGTAAAGGAGAAGAACTTTTCACTGGAGTTGTCCCAATTCTTGTTGAATTAGATGGTGATGTTAAT
GGGCACAAATTTTCTGTCAGTGGAGAGGGTGAAGGTGATGCAACATACGGAAAACTTACCCTTAAATTTATTTGCACTACTGGAAAACTACCTGTTCCATGGCCAACA
CTTGTCACTACTTTCGGTTATGGTGTTCAATGCTTTGCGAGATACCCAGATCATATGAAACAGCATGACTTTTTCAAGAGTGCCATGCCCGAAGGTTATGTACAGGAA
AGAACTATATTTTTCAAAGATGACGGGAACTACAAGACACGTGCTGAAGTCAAGTTTGAAGGTGATACCCTTGTTAATAGAATCGAGTTAAAAGGTATTGATTTTAAA
GAAGATGGAAACATTCTTGGACACAAATTGGAATACAACTATAACTCACACAATGTATACATCATGGCAGACAAACAAAAGAATGGAATCAAAGTTAACTTCAAAATT
AGACACAACATTGAAGATGGAAGCGTTCAACTAGCAGACCATTATCAACAAAATACTCCAATTGGCGATGGCCCTGTCCTTTTACCAGACAACCATTACCTGTCCACA
CAATCTGCCCTTTCGAAAGATCCCAACGAANAGAGAGACCACATGGTCCTTCTTGAGTTTGTAACAGCTGCTGGGATTACACATGGCATGGATGAACTATACAAATAA
TAATACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGAGTCACACTGG
CTCACCTTCGGGTGGGCCTTTCTGCGTTTATATACTAGTAGCGGCCGCTGCAGTCCGGCAAANAAGGGCAAGGTGTCACCACCNTGCCCTTTTTCTTTAAAACCGNAA
AGATTACTTCNNNGTTATGCAGGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCT
The mapping were correct for BBa_E0240 and BBa_I20260.

 Colony PCRs were performed on 8 colonies for BBa_E0240 and BBa_I20260, form plates from the transformation of 16th August.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Table 3: PCR mix Preparation for One Taq PCR colony  </CENTER></DIV></DIV></DIV></DIV>
49 Âµl of mix were distributed in each PCR tubes. The One Taq PCR program was applied.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Table 4:IGEM One Taq PCR program termocycling conditions </CENTER></DIV></DIV></DIV></DIV><SPAN class="cd-date">Aug 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2>
      Colonies had grown on plates as followed:
<P><UL><LI> LB agar: more than 1000 colonies for the two biobricks</LI><LI> LB agar kanamycin: 10 colonies for BBa_E0240 and &gt;40 colonies for BBa_I20260</LI><LI> LB agar chloramphenicol: &gt;50 colonies for BBa_E0240 and 3 colonies for BBa_I20260</LI></UL></P><P align="justify">  Plate storage at 4Â°C. </P><SPAN class="cd-date">Aug 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">  Nothing had grown during the night on both plates. Taking into consideration different hypothesis, the transformation was repeated with a change on step 6 and 7. At step 6, only 1 ml of LB medium was added. To plate, 200 Âµl of BBa_E0240 and BBa_I20260 were plated on 3 plates: LB agar, LB agar kanamycin and LB agar chloramphenicol. </P><SPAN class="cd-date">Aug 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">  BBa_E0240 and BBa_I20260 are respectively into a PSB1C3 and a PSB1K3 vector. In order to select colonies, LB complemented with kanamycin or chloramphenicol are necessary. 200 ml of LB agar medium was prepared with 7 g of LB Agar powder(Sigma) into 200 ml of miliQ water. After complete dissolution, the solution was sterilized in the Tuttmauer 2540ML apparatus, STE 20 minutes at 121Â°C and EXT+DRY 15 minutes. 50 Âµl of kanamycin stock solution (25 mg/L) was added to 50 ml of LB agar to pour 2 plates. 150 Âµl of chloramphenicol stock solution was added in the 150 LB agar remaining ml to pour 6 plates.  
 The transformation was performed into DH5 alpha ''E. coli'', as followed: </P><OL><LI> Remove E. coli competent tubes from -80Â°C and keep it on ice
</LI><LI> Add 1 Âµl of template (here solubilized plasmids from the registry distribution kit) and mix gently
</LI><LI> Incubate 10 minutes on ice
</LI><LI> Perform an heat shock 30 seconds at 42Â°C
</LI><LI> Incubate 2 minutes on ice
</LI><LI> Add 3 ml of LB medium and incubate 60 minutes at 37Â°C with an agitation at 200 rpm
</LI><LI> Plate 200 Âµl of BBa_E0240 on a chloramphenicol LB agar plate and BBa_I20260 on a Knamycin LB agar plate
</LI><LI> Incubate plate overnight at 37Â°C
</LI></OL><SPAN class="cd-date">Aug 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">  To amplify BBa_I20260 and BBa_0240, a PCR was perform with the mix described Table 1. The program IGEM Q5 PCR was applied, see Table 2. PCR products were cleaned with the GeneJET purification kit (Thermo Scientific)followed by DNA quantification with the NanoDrop 2000 (Thermo Scientific). Purified samples of I20260 and E1010 had respectively following concentration 30.9 ng/Âµl and 43.3 ng/Âµl. </P><P align="justify">Preparation of a 1% agarose gel: 0.52 g of Top Vision agarose (Thermo Scientific) + 50 ml of TAE 1X. Microwave 30s by 30s until agarose total dissolution Gel was cooling down until to be lukewarm, one BET drop was added. Gel was loaded with 20 Âµl of BBa_E0240 PCR product and BBa_I20260 purified PCR product added with 4 Âµl of loading dye, at 10Âµl of the mix per well.
        Gel running 45 minutes at 100 mV in TAE 1X buffer. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 4: 1% agarose gel of PCR products from 08.14.2014 after PCR clean up. Lane 1 and 5: Purple 2-Log ladder NEB, Lane 2, 3 and 4: BBa_E0240 purified PCR product, Lane 6,7 and 8: BBa_I20260 purified PCR product </CENTER></DIV></DIV></DIV></DIV><P align="justify">3 pieces of gel were sampled in four tubes to perform a DNA purification from gel. The DNA purification was made with the GeneJET purification kit (Thermo Scientific).
     A verification electrophoresis was performed on a 1% agarose gel. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 5: 1% agarose gel of purified fragments from gel. Lane 1: Purple 2-Log ladder NEB, Lane 2: BBa_E0240 1200 bp purified fragment and Lane 3: BBa_I20260 1200 bp purified fragment </CENTER></DIV></DIV></DIV></DIV><P align="justify">  The major part of the DNA was lost during the purification because the amount of was to weak to be purified from gel. So we decide do transform ''E. coli'' to isolate a  colony containing the desired plasmid for BBa_E0240 and BBa_I20260. </P><SPAN class="cd-date">Aug 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">  PCR products were cleaned with the GeneJET purification kit (Thermo Scientific)followed by DNA quantification with the NanoDrop 2000 (Thermo Scientific). 
Purified PCR product of  BBa_J61002/J23101 and BBa_J61002/K823012 concentration were respectively: 53.9 ng/Âµl and 54.1 ng/Âµl.</P>Preparation of a 1% agarose gel: 0.56 g of Top Vision agarose (Thermo Scientific) + 50 ml of TAE 1X. Microwave 30s by 30s until agarose total dissolution Gel was cooling down until to be lukewarm, one BET drop was added. Gel was loaded with 10Âµl per sample previously added with 2 Âµl of loading dye 6X, and 5 Âµl for ladders. Gel running 45 minutes at 100 mV in TAE 1X buffer.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 2: 1% agarose gel of PCR products from 08.12.2014 after PCR clean up. Lane 1 and 7: Purple 2-Log ladder NEB, Lane 2: BBa_J23101 purified PCR product, Lane 3: BBa_K823012 purified PCR product, Lane 4: BBa_E0240 purified PCR product, Lane 5: BBa_I20260 purified PCR product and Lane 6: pBHR1 purified PCR product  </CENTER></DIV></DIV></DIV></DIV><P align="justify">  Profiles were same as before PCR clean up. To sequence the amplified parts, we decided to a purification of fragments from the gel as presented below. </P><P align="justify"> Preparation of a 1% agarose gel: 0.56 g of Top Vision agarose (Thermo Scientific) + 50 ml of TAE 1X. Microwave 30s by 30s until agarose total dissolution. Gel was cooling down until to be lukewarm, one BET drop was added. Gel was loaded with 30 Âµl of BBa_E0240 purified PCR product and BBa_I20260 purified PCR product added with 6 Âµl of loading dye, at 10Âµl of the mix per well. Gel running 45 minutes at 100 mV in TAE 1X buffer. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 3: 1% agarose gel of PCR products from 08.12.2014 after PCR clean up. Lane 4 and 8: Purple 2-Log ladder NEB, Lane 1, 2 and 3: BBa_E0240 purified PCR product, Lane 5, 6 and 7: BBa_I20260 purified PCR product </CENTER></DIV></DIV></DIV></DIV><P align="justify">  Four pieces of gel were sampled in four tubes to perform a DNA purification from gel. </P><SPAN class="cd-date">Aug 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H2>Contructions PSB1C3 with I20260, J23101-E1010 and K823012-E1010</H2><P align="justify">
  The 4 needed parts are: BBa_J23101, BBa_K823012, BBa_E0240 and BBa_I20260. Corresponding wells were located on 2014 Distribution kit plates and resuspended with 10 ÂµL steril water. That permits to obtain a DNA concentration around 0.2 ng/Âµl (according to the registry. Solutions were transferred into 1 ml eppendorf tubes and stored at -20Â°C.
  To amplify fragments, a PCR was performed on the 4 constructions, with the mix described on table 1.
  </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Table 1: PCR mix preparation </CENTER></DIV></DIV></DIV></DIV><P align="justify">
  Distribution of 49 Âµl of mix per PCR tube. Application of program IGEM Q5 PCR.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Table 2: IGEM Q5 PCR program thermocycling conditions </CENTER></DIV></DIV></DIV></DIV><P align="justify">  Preparation of a 1% agarose gel: 0.56 g of Top Vision agarose (Thermo Scientific) + 50 ml of TAE 1X.Microwave 30s by 30s until agarose total dissolution. Gel was cooling down until to be lukewarm, one BET drop was added. Gel was loaded with 10Âµl per sample previously added with 2 Âµl of loading dye 6X, and 5 Âµl for ladders. Gel running 45 minutes at 100 mV in TAE 1X buffer. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Figure 1: 1% agarose gel of PCR products. <P style="padding-top:1%;">Lane 1: BBa_K823012 PCR product, Lane 2: pBHR1 PCR product, Lane 3 and 6: Purple 2-Log ladder NEB, Lane 4: BBa_E0240 PCR product, Lane 5: BBa_I20260 PCR product, Lane 7: 1 Kb plus Ladder Ogene Ruller and Lane 8: BBa_J23101 PCR product </P></CENTER></DIV></DIV></DIV></DIV><P align="justify">  We expected to obtain one band per PCR sample corresponding to the interesting amplified fragment. For Lane 2, 3 and 6 it was ok. We had the expected profile. By contrast 2 bands were visible, one at the expected size (around 1200 bp) and another around 600 bp. We decided to perform a purification on gel of each bands. </P><SPAN class="cd-date">Aug 12</SPAN></DIV></DIV></DIV></SECTION><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="statistic" class="parallax" style="background-image: url(https://static.igem.org/mediawiki/2014/4/45/Under-water-wallpaper-hd-104.jpg);"><FOOTER id="footer"><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></FOOTER></SECTION></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Interlab_Study">http://2014.igem.org/Team:Evry/Interlab_Study</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Interlab_Study" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Interlab_Study" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Interlab_Study&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Interlab_Study&amp;oldid=354646" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>