<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Biology_Transposons"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Biology/Transposons">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Biology/Transposons&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/Transposons&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/Transposons&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Biology/Transposons" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Biology/Transposons</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/b/b0/DNA.jpeg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Biology - Transposons</H1></DIV></DIV></SECTION></DIV><SECTION id="speakers"><DIV id="cd-timeline" class="cd-container"><DIV class="content-wrapper"><DIV class="col-lg-10 col-lg-offset-1"><DIV align="justify"><H1><CENTER>Transposon</CENTER></H1><P>Transposons, known as mobile elements or transposable elements, are DNA sequences able to move randomly within the genome. This phenomenon is based on different transposition mechanisms. One of them, the cut-and-paste mechanism, requires an enzyme named transposase. Transposases bind to the end of transposons sequences, which consist of inverted repeats and then catalyze the excision and insertion of the transposons into the genome.



</P><B>Fig. 1</B> General transposon mechanism
1. The transposable element, surrounded by IS sequence is &quot;cut-and-paste&quot; from a locus to another thanks to the transposase (arrow).
2. Insertion of the transposon occurs randomly in the genome


<H3><CENTER>Transposase Tn10 / IS10 </CENTER></H3><P>
The complex Tn10/IS10 is involved in the non-replicative cut-and-paste mechanism. The transposable segment is excised at its ends and is then re-inserted randomly in a DNA site.

The Tn10 transposase protein is made of 402 amino-acids, which recognises inverted repeats insertion sequence; Is10-right and Is10-left. The Tn10 protein expression is strongly regulated by various positive and negative regulation mechanisms.

IS 10 is an insertion sequence composing the transposon Tn10. The two IS10 elements, IS10-Right and IS10-Left, contain all of the Tn10 encoded genetic determinants; as the coding region of the transposase protein Tn10. The two ends of IS-10 have a similar terminal inverted repeat of 23 bp, corresponding to the transposase binding site. However, some genetic drifts occurring between both of these sequences caused variation in their functionality. Indeed, IS10-right is fully functional while IS10-left is partially functional.
 
</P><H3><CENTER> Our project </CENTER></H3><P>
We tested different plasmids and methods to transform Pseudovibrio denitrificans but yet unsuccessfully. Thus, the integration in the genome was tested. Doctor Brian Jester gave us the pNK2 plasmid and DH5α pir cells. In fact, the plasmid pNK2 contains a particular origin of replication OriVR6Kgamma. This ORI VR6K gamma is controlled by the pi protein, which is encoded by the pir gene. Indeed, the pi protein allows the replication of the plasmid by binding a particular site in the ORI sequence. Hence, the oriVR6K gamma can only be replicated in a bacterial strain producing the pi protein. This ori was already used in the iGEM competition in 2009 by a french team. The gamma origin is adjacent to the pi protein binding site and other sites bounded by the host cell proteins involved in its own reproduction.
</P><B>Fig. 2</B> Plasmid map of pNK2-CRPIIh
The plasmid is 6,356 bp long and annotated. It contains the transposase Tn10 and the origin of replication oriVR6K. The sequence including the antibiotic resistance cassette kanR, and the mGFP gene is surrounded by IS sites. Restriction sites used by the registry are encircled by red rectangle.
<FONT color="#0099CC"><H4><CENTER><B> Insertion of transposon</B></CENTER></H4></FONT><OL><LI><B> Transformation Pseudovibrio <I>denitrificans</I> with pNK2</B> 
We tested pNK2 by transforming <I>Pseudovibrio denitrificans</I> bacteria with the plasmid by electroporation.
The selection of cells was performed in 1X marine broth medium supplemented with kanamycine 50µg/mL. In fact,  <I>Pseudovibrio denitrificans</I> can grow on medium with kanamycine 25µg/mL. (<A href="https://2014.igem.org/Team:Evry/Biology/CellCharacterization#antibiotic">Sensitivity to antibiotics</A>)
</LI><LI><B>Phenotypic verification </B><B>Fig. 3</B> Image of <I>Pseudovibrio denitrificans</I> colonies in a petri dish containing kanamycin. <I>Pseudovibrio denitrificans</I> were previously transformed with the pNK2-CRPIIh plasmid.<B>Fig. 4</B> Growth curve of <I>Pseudovibrio denitrificans</I> in M9 supplemented with casamino acids and 3% NaCl
</LI><LI><B> Genotypic verification</B><DIV align="center">a) Amplification of transposon</DIV> The kanamycin gene is included in the transposon sequence of pNK2 and we thus amplify this gene for our verification PCR. 
We successfully obtained an amplicon corresponding to the kanamycin gene in our transformed cells. Considering the plasmids are not able to be replicated in an other strain than pir cells, we assumed the amplification we obtained were from ADNg and not from plasmids.
<B>Fig. 5</B> Picture of an electrophorese gel samples come from the PCR which amplify the kanamycineR cassette. All clones have the insertion
of teh transposon. The Control, Pseudovibrio denitrificans, do not the insertion as expected.

As a first verification, we obtained this gel for 15 colonies of Pseudovibrio <I>denitrificans</I>.

<DIV> align=&quot;center&quot;&gt;b) Sequencing of 16S</DIV> The amplification of transformed cells' DNA with primers 16S, which amplify the sequence of the ribosome 16S, were purified and sent to sequencing. 

The sequence we obtained proves that we transformed actual Pseudovibrio bacteria with an integrated transposon.
<DIV align="center">c) Amplification of specific sequence of Pseudovibrio <I>denitrificans</I></DIV>
After the results of sequencing of Pseudovibrio <I>denitrificans</I>' genome, a sequence specific of this strain has been discovered. 

This sequence is the following (read of 1000 bp):
GACGGTGTCATTGATCTGGATAATGTCAACGAGCAGACCGGATCTTATCAGTTTGTCGGTGATGATGGGTTTGATACGGCAGGGCGCAGTATCTCTTCAGCTGGTGATG
TTGATGGTGATGGTAAGGATGATCTGCTCATCGGTGCTGCGAATGCTAATGGTAGTGGTGCCAACCAAGGATCCGCTTCAGGGGCTGCTTATCTGATGACGGCTTCTGCACTA
GCAGCCGCTGATGCCGCTGACGGCACCACTGATGGTGTTATTGATTTGGGTAATGTCAATGAGCAGACTGGATCTTATCAGTTCAATGGTACAGAAGTAATGGACCAAGCCGG
AACTCGTGTAACATCTGCAGGCGATGTGGATGGCGATGGCAAAGATGATGTCTTTATCAGCAGCATTTTTGCAGATGATGGCGGCTCCAGTTCTGGTGAAGCATATTTGCTGA
CAGCTGCTGCTATGGCTTCAGCTGATGCCGCTGACGGCACTACTGACGGCATCATTGATTTGGACAATGTCAATGAGCAAACCAACTCTTATCAGTTTGTTGGCACCCAAGCA
GATGACCTGGCCGGCATTGATATCTCAGCTGCTGGTGATGTTGATGGCGATGGCAAAAATGACTTCTTGATCGGTGCTCGGGCAGCAGATGGTGGCGGCGCTGGCTCGGGTGA
GGCCTATCTGTTGACTGCAGCAGCACTTGCTTCAGCTGATGCAGCTGATGGCACCACTGATGGGATTATCGATCTAGATAATGTCAATGAGCAGACTAACTCTTATCAGTTCG
TTGGTACGGAAGTTGGCGATGATGCGGGAATTAGCGTGTCATTTGTCGGTGATGTTGACAATGATGGTAAGGACGATCTGTTGATTGGTGCACGTAATGCTGACGGCGGTGGC
TCCAACTCTGGTGAAGCCTATCTAATGTCTATTGCTTCACTGGCGACTGCTGATGCAGCTGATGGCACCATTGATGGTGTTATCGATTTGGAT
<DIV align="center">d) Integration of transposons in Pseudovibrio <I>denitrificans</I></DIV>
Here, we tried to proved that there was actually a random integration of the transposon in the strain's genome. To achieve this, we performed a reverse PCR.

In order to verify both number and frequency of insertion per transformation we set up the following protocol:
 
Thanks to the genome assembly we were able to digest the genomic DNA in silico and find a restriction enzyme that would not cut in our insert but cut enough times in the genome to be able to recircularize pieces of DNA. 
 
HindIII cuts (demande a cécile si tu peux elle a le tableau du coupage par les enzymes) which is enough to generate small fragment ready for religation.
 
After the recircularization we looked for and enzyme that could cut in our insert but with the least number of knacks in the genome sequence in order to cut only once the circular DNAs.

XbaI exhibits the lowest number of cut in the gene and does not cut our insert.
 
With the relinearize fragment we can perform a PCR to find: 
 
How many bands appear ? A question which relates to the number of inserts in the genome.
What is the size and numbers of bands between genomic DNA samples? A question which relates to potential discovery of favourable spots of integration in the genome and the randomness of the number of insertions in the genome.
 
One clone clearly displayed numerous insertions, while the others having fewer bands tend to show band sof the same size, which could prefigure the existence of favourite spot, as can be seen in E.coli.
 
With further experiments and sequencing run we will be able to precisely locate the insertions and the better define how many times an insertion event can occur at a given concentration of plasmid electroporated.
<DIV align="center">The transformation of Pseudovibrio <I>denitrificans</I> has been verified by the study of its phenotype and its genotype. We therefore confirm the efficiency of the transposon system in our bacteria.  c</DIV><FONT color="#0099CC"><H4><B>Creation of Transposon Plasmid </B></H4></FONT>
The second part of our work consisted in introducing our construction in this plasmid. A problem we had to face though was the presence of biobrick restriction sites in our plasmid. We decided to modify our Transposon plasmid so it would no longer have those restriction sites. 

Here is the schema of our project.


Our aim was to merged pSB1C3 and pNK2 into a single plasmid, in order to have a plasmid that would replicate in non-pir cells. Also, our goal was to have a plasmid matching the requirements of the biobrick format. Here we used the pSB1C3 plasmid as the backbone. 
 To isolate the two plasmids it would be possible to digeste the merged plasmid with BglII and then extract by gel eletrophoresis our Transposon plasmid, which only contains the core elements needed for the transposition. Between the two transposable elements iS10 we will eventually integrate the biobrick prefix and suffix to be able to integrate any biobrick in a genome. 
 
As a first step, we have to amplify and assemble each element by the Golden Gate assembly method.
</LI><LI><B> Mutation of oRiVR6Kgamma</B>
The first issue we encountered was the presence of XbaI restriction sites in the OriVR6Kgamma sequence. Thus, before amplifying it with Golden Gate overhangs, we had to induce mutations in that site of the plasmid using PCR mutagenesis. 
 pNK2 plasmid was amplified with primers allowing us to modify the XbaI site : 5' TCTAGA 3' en 5' ACTAGA 3'.
 After amplification of the plasmid, we purified the PCR products obtained and transformed several colonies we got. Many colonies were inoculated in liquid cultures supplemented with kanamycin (50µg/mL) and the pNK2 plasmid was finally extracted. Those plasmids were digested by XbaI in order to verify the mutation of the XbaI site.



1) Mutation of OriVR6Kgamma

The initial Pnk2 plasmid is digested into three fragments by XbaI.
The XbaI site successfully modified we obtained was used as the template of the origin of
replication amplified for the Golden Gate and Biobrick.

2) Plasmid biobrick :
Two biobricks were sent to the Registry. The first being the origin of replication
oriVR6Kgamma and the second being the transposase Tn10. To obtain those biobricks into 
the pSB1C3 (RFC 92) plasmid, which contains the essential BsaI restriction site, we used 
the Golden Gate assembly method.

White and red colonies grew on the plate. Red colonies contain the initial pSB1C3 plasmid
bearing a mRFP gene. White cells could be either cells transformed with an empty vector 
or with our expected construction.
<B>Fig </B> Picture of electrophoresis gel, samples come from pCR which amplify Tn10 and oriVR6KgammaBBa_K1413041 orivr6kgamma
BBa_K1413043 transposas Tn10


3) Plasmid transposas :

Each element of the plasmid transposas was amplified with Golden Gate overhangs.
<B>Fig </B> Picture of electrophoresis gel of each elements of Transposon Plasmid Golden gate 
Golden Gate was performed with the pSB1C3 (RFC 92) plasmid. We then used the Golden
Gate product to transform DH5α cells.

<B>Fig </B> Picture of electrophoresis gel of Transposon plasmid isolated to pSB1C3
We have our Transposon Plasmid. We just need to add our sensing constructions inside the transposon plasmid to test these sensor into<I> P. denitrificans</I><H4>References</H4><P align="justify">

Bender, J., &amp; Kleckner, N. (1988). Genetic Evidence That TnlO Transposes by a Nonreplicative Mechanism, 45, 801–815.

Biology, C. (1996). Two Classes of TnlO Transposase Mutants That Suppress Mutations i n, (1972).

Chalmers, R. M. (1995). Identification characterization pre-cleavage complex early transposition, 14(17), 4374–4383.

Crellin, P., &amp; Chalmers, R. (2001). Protein±DNA contacts and conformational changes in the Tn 10 transpososome during assembly and activation for cleavage, 20(14).

Foster, T. J., Davis, M. A., Roberts, D. E., Takeshita, K., Kleckner, N., &amp; Laboratories, T. B. (1981). Genetic Organization of Transposon TnlO,23(January), 201–213.

Humayun, S., Wardle, S. J., Shilton, B. H., Pribil, P. a, Liburd, J., &amp; Haniford, D. B. (2005). Tn10 transposase mutants with altered transpososome unfolding properties are defective in hairpin formation. Journal of Molecular Biology, 346(3), 703–16. doi:10.1016/j.jmb.2004.12.009

Ross, J. a, Wardle, S. J., &amp; Haniford, D. B. (2010). Tn10/IS10 transposition is downregulated at the level of transposase expression by the RNA-binding protein Hfq. Molecular Microbiology, 78(3), 607–21. doi:10.1111/j.1365-2958.2010.07359.x

R. (2010). Recent advances in the Biodegradation of Phenol: A review, 1(2), 219–234.
  
J.C  Blouzard, O. Valette, C.Tardif, P. de Philipp (2010) Random Mutagenesis of Clostridium cellulolyticum by Using a Tn1545 Derivative, applied and environmental microbiology,
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Biology/Transposons">http://2014.igem.org/Team:Evry/Biology/Transposons</A>&quot;</DIV></LI><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Biology/Transposons" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Biology/Transposons" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Biology/Transposons&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Biology/Transposons&amp;oldid=401669" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></OL></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>