<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Biology_RNAseq_ExperimentalDesign"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Biology/RNAseq/ExperimentalDesign">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Biology/RNAseq/ExperimentalDesign&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/RNAseq/ExperimentalDesign&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Biology/RNAseq/ExperimentalDesign&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Biology/RNAseq/ExperimentalDesign" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Biology/RNAseq/ExperimentalDesign</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H2>Experimental design</H2><DIV align="justify"><DIV align="center"><H3>Why RNA-seq</H3>
RNA-seq is a powerful technique to determine the expression of transcripts in specific conditions. It allows to know the relative expression of each gene for an organism. In principle, the more RNA molecules are present, the more protein will be produced. Using this technology is a chance and we wanted to compare gene expression of <I>Pseudovibrio denitrificans</I> depending on different conditions. These conditions are: M9 media as a reference, or M9 media supplemented with either cadmium, nitrite, or lead. By comparing the gene expressions between conditions and the reference we should be able to find specific highly overexpressed gene. By mapping these reads to the genome (thanks to the sequencing of <I> Pseudovibrio denitrificans</I> previously done by the team), and applying an expert eye to these results, we could find promoters that can be used to build new biological sensors. These sensors would be created by taking 500 bp upstream these overexpressed genes. The promoters would be characterized adding a reporting gene as the GFP to quantify the expression depending on different concentrations of the compounds used for the RNA-seq. RNA-seq is a critical technique because of the RNA stability. Also, each sample preparation must be done in the exact same conditions than the others to be properly compared. 

Pseudovibrio cultures growth in the different conditions were measured by using a Tecan (Infinite M200).
The compounds we tested were nitrite, cadmium and lead. 

The kinetic parameters were determined during the characterization of the bacterium <A href="https://2014.igem.org/wiki/index.php?title=Team:Evry/Biology/CellCharacterization/Culture">characterization of the bacteria</A>.
We tested three compounds in marine broth.<B>

Title: Kinetic of pseudovibrio in Marine Broth (MB) with either lead, nitrite or cadmium. We diluted the bacteria cultures by 1/5 and 1/10.</B>

Because of the marine broth, which is a cloudy rich medium, we were not able to draw conclusions as it prevented to have a correct measurement of the OD.

Due to that problem with the medium, we preferred to use a M9 medium with casamino acids, because it is a clear media. 

In order to be able to observe differences in the bacterial growth, we had to cause a stress to the bacteria by adding the compounds of interest.
<B> Title: Stress of pseudovibrio denitrificans with differents concentrations of compounds. </B>

The compounds were added after two hours.
After analysis of the data, the concentrations we decided to test for the RNA-seq were:

<UL><LI> cadmium : 300uM
</LI><LI> lead : 100 ppm 
</LI><LI> nitrite : 0,9 ng/L
</LI></UL>
 

because that modified the bacterial growth but do not kil it.
 

So,  we must to be check the quality of RNA. <DIV align="center">
With the aid of Tolonen Lab, we make the extraction of DNA and RNA.

The DNA of bacteria was sent after extraction for the DNAseq and the results were analyzed and assembled. 
<A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">DNAseq</A>

For to check the quality  of RNA, we extract RNA from new culture in log phase with the protocol of RNA <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">protocol of RNA</A> extraction from Tolonen Lab.


Throughout of the experiment, electrophoresis gels are done in order to confirm which have always RNA. </DIV><B> Title : RNA controls by electrophoresis gel. </B>
Gel1 : 1.Ladder, 2. Positive control, 3. Pseudovibrio d. and 4.Other bacteria
Gel2 : 1.Ladder, 2. Pseudovibrio d. and  3.Positive control.


And we were sent samples for to drop off on Agilent chip. 

This is the results of Agilent chip.
For to know if our RNA is not degrade, we look rRNA ratio ( it have to included between 1 and 2) and the RIN ( included between  7 and 10). This is approved the method of extraction.

If the quality is confirmed, we do again extractions with the culture cells which are subjected a stress with the differents compounds. 
A control by electrophoresis gel is done.

<B> Title : Control of RNA presence </B>
Gel1 and 2 : 1.Ladder, 2.Positive Control, 3.Pseudovibrio d., 4.Pseudovibrio d. make stressed by cadmium, 5. Pseudovibrio d. make stressed by nitrite and 5. Pseudovibrio d. make stressed by lead.

The RNA of bacteria was sent and the obtained results were analysed.

After an analysis of the RNA-seq data, we found out that we actually got a contamination by a bacterium called Pseudomonas fluoresense. We assume that this contamination is due to the kluldge. We are not certain of the source of that contamination.
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Biology/RNAseq/ExperimentalDesign">http://2014.igem.org/Team:Evry/Biology/RNAseq/ExperimentalDesign</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Biology/RNAseq/ExperimentalDesign" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Biology/RNAseq/ExperimentalDesign" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Biology/RNAseq/ExperimentalDesign&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Biology/RNAseq/ExperimentalDesign&amp;oldid=401774" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>