<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Notebook_Transposons"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Notebook/Transposons">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Notebook/Transposons&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/Transposons&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/Transposons&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Notebook/Transposons" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Notebook/Transposons</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/5/51/NotebookEvry.jpg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Notebook - Transposons</H1></DIV></DIV></SECTION><SECTION id="timeline"><DIV id="cd-timeline" class="cd-container"><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Trying to show BBa_1413044 work even empty in DH5alpha </H4><P>
8 colonies of DH5alpha were transformed with BBa_1413044 and grown in 3 ml LB + 2 negative control


Preparation for PCR colony: 
Spin down 500 ul culture at 14000 g - 2 min, resuspend in 100 ul H20 MilliQ, put at 95 C° for 10 min then spin down at 14000 g - 2 min.

PCR: enzyme: Q5; template: 3 ul of supernatent for the culture; oligo: F 100/ R 101; Tm tested: 55; elongation time: 1m00s
</P>
The two last wells are the controls which surprisingly displayed a band, and we did not obtain a single band for the 8 strains.
In the end the gel revealed itself to be inconclusive with regards an empty transposon vector.
<H4> Defining where the transposase integrate </H4><P>
find enzyme with a good cut numbers in the genome and not in our insert: HindIII

Digestion of 8 <I>Pseudovibrio denitrificans</I> DNA preparation by HindIII
50ul final volume, 200 ng DNA preparation, 5 ul  Neb 2.1 buffer, 38 ul H2O, 1 ul HindIII. 1h

Then religate:
Ligation: 20 ul final volume, 2 ul 10x T4 ligase buffer, 1 ul T4

then light digestion with XbaI
50ul final volume, 20 ul DNA religated preparation, 5 ul  Neb 2.1 buffer, 38 ul H2O, 1 ul XbaI enzyme. 15 min
deactivation at 80 C° for 20 min

then PCR
PCR: enzyme: Q5; template: 3 ul of digestion by XbaI; oligo: F 105/ R 106; Tm tested: 55; elongation time: 4m00s
</P><P>One clone clearly displayed numerous insertions, while the others having fewer bands tend to show band sof the same size, which could prefigure the existence of favourite spot, as can be seen in E.coli.

With further experiments and sequencing run we will be able to precisely locate the insertions and the better define how many times an insertion event can occur at a given concentration of plasmid electroporated.</P><SPAN class="cd-date">Oct 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> 14.10.2014 ligation </H4><P>
it gave only one clone. 

Clone culture:
5 ml LB + 5 ul Kan (25 mg/ml) - 37 C°
</P><H4> Transformation <I>Pseudovibrio denitrificans</I>&amp; Dh5 alpha with mRFP in BBa_1413044 </H4><P>
electrocompetent cells:

1 mm cuvette; 1800 V
recuperation media: MB for <I>Pseudovibrio denitrificans</I> , LB for DH5alpha

plating:

MB agar for <I>Pseudovibrio denitrificans</I>LB agar for LB for DH5alpha
</P><H4> Transformation <I>Pseudovibrio denitrificans</I>&amp; Dh5 alpha with BBa_1413001 in BBa_1413044 </H4><P>
electrocompetent cells:

1 mm cuvette; 1800 V
recuperation media: MB for <I>Pseudovibrio denitrificans</I> , LB for DH5alpha
plating:

MB agar for <I>Pseudovibrio denitrificans</I>LB agar for LB for DH5alpha
</P><H4> Verification testing of pnK2 in <I>Pseudovibrio denitrificans</I></H4><P>
16  clones of <I>Pseudovibrio denitrificans</I> were grown in 3 ml MB, 30 C°, 24h

DNA extraction:

Invitrogen DNA extraction kit used with the Gram - protocol.
nanodrops gave value between 80 ng/ul &amp; 130 ng/ul

PCR for Kan gene insertion:
PCR: enzyme: Q5; template: 200 ng genomic preparation <I>Pseudovibrio denitrificans</I>; oligo: F 82/ R 83; Tm tested: 55; elongation time: 2m00s
</P><P>
PCR for <I>Pseudovibrio denitrificans</I> specific oligo:
PCR: enzyme: Q5; template: 200 ng genomic preparation <I>Pseudovibrio denitrificans</I>; oligo: F 72/ R 73; Tm tested: 55; elongation time: 2m00s
</P><P>
As a control we tested the specificity of our oligo on the strain we work with:
</P><SPAN class="cd-date">Oct 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Ligation of BBa_K1413001 </H4><P>
strategy:</P><P>
Digestion of BBa_K1413001 by E/S :
50 ul final volume, 5 ul Buffer NeB 3.1, 5 ul (35 ng/ul) BBa_1413001 in pSB1C3, 1 ul E/S, 39 ul H2O. 1h
Then Thermoscientific &quot;PCR clean up&quot; Kit in 30 ul final elution</P><P>
Digestion of Kan 82/83 by E/S &amp; X/P:
50 ul final volume, 5 ul Buffer NeB 4.1, 5 ul (30 ng/ul) Kan 82/83 in pSB1C3, 1 ul E/S or X/P, 38ul H2O. 1h
Then Thermoscientific &quot;PCR clean up&quot; Kit in 30 ul final elution

Ligation: 
I° strategy:
 20 ul final volume, 5 ul Buffer T4 fermentas, 3 ul digestion product BBa_K1413001 by E/S + 3 ul Kan 82/83 + 5 BBa_1413044 transposon plasmid. 4 ul H2O. 1 ul T4 ligase. 1h

II° Strategy:
 20 ul final volume, 5 ul Buffer T4 fermentas, 3 ul digestion product BBa_K1413001 by E/S + 5 BBa_1413044 transposon plasmid. 4 ul H2O.1 ul T4 ligase. 1h


Clean up:
II° strategy:
Ligation product by Thermoscientific &quot;PCR clean up&quot; Kit in 30 ul final elution


Digestion: II° strategy
 50 ul final volume, 5 ul Buffer NeB 4.1, 5 ul ligation product, 1 ul E/X, 39 ul H2O. 1h
Then Thermoscientific &quot;PCR clean up&quot; Kit in 30 ul final elution

Ligation:

II° Strategy:
 20 ul final volume, 5 ul Buffer T4 fermentas, 3 ul ligation product E/X + 5 ul Kan 82/83 E/S. 4 ul H2O. 1h

Transformation: electrocompetent

50ul electrocompetent cells
2 ul Ligation product of I° strategy / II° strategy in <I>E.coli DH5alpha</I>.
</P><SPAN class="cd-date">Oct 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> BBa_1413044 retrieval of the transposon plasmid </H4><P>
strategy:</P><P>
Digestion of BBa_1413044 by BglII :
50 ul final volume, 5 ul Buffer NeB 3.1, 5 ul (20 ng/ul) BBa_1413044 in pSB1C3, 1 ul BglII, 39 ul H2O.

Gel electrophoresis:
</P>
Purification sur gel:

Kit thermoscientific &quot;Gel extraction&quot;. Elution final in 30 ul.
<H4>Obtaining the Kan gene on pNK2</H4><P>
PCR: enzyme: Q5; template: 150 ng pNK2 first three well from new stock, next ones the old stock; oligo: F 82/ R 83; Tm tested: 55/60/65; elongation time: 1m30s
</P><P>
Purification sur gel:

Kit thermoscientific &quot;Gel extraction&quot;. Elution final in 30 ul.
</P><SPAN class="cd-date">Oct 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Universal Transposon Plasmid </H1>
A PCR on 63 white colonies in the plate of trasformed cells is realized. 
Two colonies of 63 have the band expected.  

 the product of these PCR is purified. 
<H1> Verification of insert and Isolation of Transposon Plasmid </H1>
The Universal Transposon Plasmid is purified. 
 After that this purification is digest with Bgl II to isolated the Transposon Plasmid of pSB1C3. 

<SPAN class="cd-date">Oct 09</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Transformation with Golden Gate product </H1>
DH5α is transformed with the product of new Golden Gate.


<SPAN class="cd-date">Oct 08</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Universal Transposon Plasmid </H1>
All the part of Golden Gate are amplified again. Each part is purified and the golden gate is realized. 



<SPAN class="cd-date">Oct 07</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Universal Transposon Plasmid </H1>The plate contains some white colonies and other are red.
20 colonies is resuspended and a PCR with primer VF and VR is realised. After migration all the bands are 500 bp. so all the colonies contains the empty plasmid. 


 

<SPAN class="cd-date">Oct 06</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Univeral Transposon Plasmid </H1>
The product of Golden Gate is conserved at -20°C. This product is reused to transformed DH5α with Universal  plasmid. 

<SPAN class="cd-date">Oct 05</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Construction of Universal Transposas Plasmid </H1>
We want mRFP digested with biobrick enzyme to put into Universal Transposon Plasmid. 
For this a plasmid pSB1C3 (RFC 92) is took and digested with X P. 

After that the product of digestion is migrated, to do an extraction on gel of mRFP. 
<H1> Universal Transposas Plasmid </H1>
There is ¾ of the plate Golden gate Universal Plasmid with white colonie. The rest is red, and tese is the colonie with mRFP. 

PCR on 11 colonies is realized with primer VF and VR. 
After migration all the samples have a band at 500 bp. It corresponds to empty plasmid. 
The Golden Gate didn't work. 

<SPAN class="cd-date">Oct 04</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Transformation Universal Transposas Plasmid </H1>
A transformation of DH5α with Golden Gate universal plasmid  is realized and sowed in LB chloramphenicol (33µg/mL).
 

<SPAN class="cd-date">Oct 03</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">


Migration on the gel of PCR products. 
Gel : 
</P><H1> Golden Gate Is10 </H1>
The PCR of Is10 kan GFP Is10 didn’t work, there is no amplification. 
<H1> Golden Gate Universal Transposon Plasmid </H1>
The PCR product to build the Universal Transposon Plasmid is purified, and a Golden Gate is realized with plasmid pSB1C3 (RFC 92). 

<SPAN class="cd-date">Oct 02</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Is10 </H1>
To do PCR to amplified :
Is10 kan GFP Is10 in the plasmid pNK2 with primer 68 and 69. 
OriVR6Kgamma muted with primer 70 and 71
To send the fragment like biobrick.
<H1> Golden Gate Universal Transposon Plasmid </H1>
To do PCR to build the universal plasmid :
Is10 (1) with primer 74 and 75
Is10 (2) with primer 76 and 77
Ori VR6K gamma with primer 78 and 79
Tn10 (with this promoter and terminator) with primer 80 and 81 

<SPAN class="cd-date">Oct 01</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">

The PCR of PseudoD with some colonie having integrated the transposon with primer 72 and 73 (specific of PseudoD) is used. 

We obtains a gel with a band at 1000bp for all Pseudovibrio transformed. 



</P><SPAN class="cd-date">Sep 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Verification of Pseudovibrio </H1>
The specific primer, 72 and 73 of PseudoD is designed, which amplified the sequence following : 
We used basic PCR program, but the Tm of these primers is 60°C.

For this we used a read of 1000 bp :

GACGGTGTCATTGATCTGGATAATGTCAACGAGCAGACCGGATCTTATCAGTTTGTCGGTGATGATGGGTTTGATACGGCAGGGCGCAGTATCTCTTCAGCTGGTGATGTTGATGGTGATGGTAAGGATGATCTGCTCATCGGTGCTGCGAATGCTAATGGTAGTGGTGCCAACCAAGGATCCGCTTCAGGGGCTGCTTATCTGATGACGGCTTCTGCACTAGCAGCCGCTGATGCCGCTGACGGCACCACTGATGGTGTTATTGATTTGGGTAATGTCAATGAGCAGACTGGATCTTATCAGTTCAATGGTACAGAAGTAATGGACCAAGCCGGAACTCGTGTAACATCTGCAGGCGATGTGGATGGCGATGGCAAAGATGATGTCTTTATCAGCAGCATTTTTGCAGATGATGGCGGCTCCAGTTCTGGTGAAGCATATTTGCTGACAGCTGCTGCTATGGCTTCAGCTGATGCCGCTGACGGCACTACTGACGGCATCATTGATTTGGACAATGTCAATGAGCAAACCAACTCTTATCAGTTTGTTGGCACCCAAGCAGATGACCTGGCCGGCATTGATATCTCAGCTGCTGGTGATGTTGATGGCGATGGCAAAAATGACTTCTTGATCGGTGCTCGGGCAGCAGATGGTGGCGGCGCTGGCTCGGGTGAGGCCTATCTGTTGACTGCAGCAGCACTTGCTTCAGCTGATGCAGCTGATGGCACCACTGATGGGATTATCGATCTAGATAATGTCAATGAGCAGACTAACTCTTATCAGTTCGTTGGTACGGAAGTTGGCGATGATGCGGGAATTAGCGTGTCATTTGTCGGTGATGTTGACAATGATGGTAAGGACGATCTGTTGATTGGTGCACGTAATGCTGACGGCGGTGGCTCCAACTCTGGTGAAGCCTATCTAATGTCTATTGCTTCACTGGCGACTGCTGATGCAGCTGATGGCACCATTGATGGTGTTATCGATTTGGAT

There is a blast with this sequence, and she is specific of PseudoD.

Test of this coupled of primers with some E.coli and Pseudovibrio ascidiacea. 

Gel 

The primers are specific of pseudo. 




<SPAN class="cd-date">Sep 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Verification of transposon </H1>
A PCR to amplified the kanamycin gene in transposas is realized on cells Pseudovibrio and DH5α transformed with pNK2. These cells have been integrated the transposon. 
The couple primer 82 and 83 is used. 

<SPAN class="cd-date">Sep 22</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate iS10 </H1>
An other PCR with primer Golden Gate Is10, which amplified the part between Is10 and Is10 in pNK2 is realized. The polymerase is modified, instead Q5 high fidelity we used phusion. 

<SPAN class="cd-date">Sep 19</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Is10 </H1>
A PCR with primer Golden Gate Is10, which amplified the part between Is10 and iS10 in pNK2 is realized. But this time the Tm is modified : Tm= 60°C instead of 55°C. 
 After migration of product PCR, there haven't bands. 


<SPAN class="cd-date">Sep 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate iS10 </H1>A PCR with primer Golden Gate Is10, which amplified the part between Is10 in pNK2 is realized. 
After migration of product PCR, there aren't bands. 



<SPAN class="cd-date">Sep 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate OriVR6gamma </H1>Some white colonies transformed with pSB1C3 (RFC92) OriVR6Kgamma muted  is resuspended. A PCR with primers VF and VR is realized. 
Gel : 

There is a band to 1000 bp. The plasmid contains so OriVR6Kgamma. 

<SPAN class="cd-date">Sep 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Tn10 </H1>
Some white colonies transformed with pSB1C3 (RFC92) Tn10 is sowed. A PCR with primers VF and VR is realized. 


gel
There is 2 bands or 300bp or 1600bp. The first band corresponding to plasmid empty and the second band corresponding to plasmid with Tn10. So it is my construction. 
The plasmid is purified and send to sequence with primer VF VR. 


<SPAN class="cd-date">Aug 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">A PCR with primer Golden Gate OriVR6Kgamma muted : 78 and 79 is realized. 
We obtain a band at 430 bp. 

A Golden Gate with this Product of PCR is realized with pSB1C3 (RFC 92). 




</P><SPAN class="cd-date">Aug 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Tn10 </H1>
DH5α is transformed with the new product of Golden Gate Tn10. 



<SPAN class="cd-date">Aug 11</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Competent cells </H1>
A new stock of DH5α and DH5α pir competent is produced. 

<H1> Golden Gate Tn10 </H1>
A PCR colonies on white cells DH5α transformed with pSB1C3 Tn10 with primer VF and VR. 
The revelation shows bands at 1500 bp, which correspond to Tn10 (1 200 bp) and every side of VF and VR. 
The golden Gate is OK and the plasmid is start to send at the registry. 






<SPAN class="cd-date">Aug 10</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Golden Gate Tn10 </H1>
The Golden Gate Tn10 is begun again and the transformation of DH5α with this new product is also begun again. 
After that the cell growth, the plasmid is purified. 

<H1> OriVR6Kgamma muted </H1>
Each plasmid pNK2 (2) purified is digested with XbaI to verified the mutation.  

Gel 

The site XbaI in Ori have been muted. There are 2 bands instead of 3






<SPAN class="cd-date">Sep 08</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Results of sequencing to Tn10  </H1>
The results of sequencing to Tn10 with primer biobrick is received. The sequence corresponding to Tn10 in the plasmid to 99,9%. The 0,1% corresponding at the base N. 
DH5α pir is transformed with the product of Golden Gate Tn10. 
<H1> Transformation with pNK2 (2) </H1>
The DH5α pir transformed with pNK2 (2) have growth. 
Some colonies are resuspended in 4mL to LB. 




<SPAN class="cd-date">Sep 04</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> GoldenGate Tn10 </H1>
Tn10 is inserted in pSB1C3 (RFC 92) with Golden Gate method. For this the “cloning bench” (software) is used to calculate the proportion of insert and plasmid in the mix. 




<SPAN class="cd-date">Sep 03</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H1> Amplification of pNK2 muted </H1>
The DH5α is transformed with this new product dialysed called pNK2 (2).
Transformation Pseudovibrio with pNK2 and sowed in selection medium kanamycin (50µg/mL) for to have a fresh transformed cells. 

<H1> Transposas Tn10 </H1>
PCR Tn10 with primer biobrick 64 and 65 is accomplished. After migration a bands at 1200 bp. This product is purified and send to sequence. 




<SPAN class="cd-date">Sep 02</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Purification of pNK2 with Ori muted </H4><P>
The product of PCR is purified with dialyse during 2h. </P><H4> Competent cells </H4><P>
To do sorbitol 0,20g/L and glycerol 10%.
Pseudovibrio competent with wash sorbitol and glycerol in M9 medium. 

</P><SPAN class="cd-date">Sep 01</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> OriVR6Kgamma mutation </H4><P>The PCR product is treated with Dpn1 during 2h. This enzyme cuts methylated DNA.
The transformation of DH5α pir is realized. But the cells arc all the time. The problem provides to PCR product. 

</P><SPAN class="cd-date">Aug 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> PCR mutageneses of OriVR6Kgamma </H4><P>
The PCR mutagenes is realized with primers 70 and 71. The template is pNK2 purified, with standard protocol of amplification except that the Tm is 60°C and the time of elongation is of most longer (size of plasmid = 6,4 kb).



</P><SPAN class="cd-date">Aug 27</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">PCR of cell Pseudovibrio with transposas, with primers: 
65F / 65R (which correspond to cassette of resistance kanamycin). 
42b / 92c
112F / 112R,
 is begun again, but the Tm is decrease 52°C instead of 55°C. It don't work.

The PCR didn't work.




</P><SPAN class="cd-date">Aug 24</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Universal Plasmid </H4><P>
We decided to create a new Universal Transposon Plasmid. For this we designed some primers to amplified OriVR6Kgamma, the transposas Tn10 in the format biobrick, to amplified OriVR6Kgamma transposas and the 2 Is10 to build the Universal plasmid with Golden Gate method.
</P><P>
First it is necessary to muted the site XbaI in the Orivr6kgamma. For this the method of PCR mutagenes is used. For this the primer is designed to amplified all the plasmid. 

The site Xba I : 5' TCTAGA 3'  is modified by 5' ACTAGA 3'</P><SPAN class="cd-date">Aug 23</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Induction of mGFP </H4><P>
The promoter front emGFP is inductible by glycerol (8%). So a test of induction has been realized. Different cells is cultivated with LB 1X and glycerol 8% add kanamycin (25µg/mL) for DH5α pir and Bl21 transformed and MB 1X and glycerol 8% add kanamycin (50µg/mL) for PseudoD.



</P><SPAN class="cd-date">Aug 22</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Measure of GFP </H4><P>A measure of fluorescence (GFP) is realised by means of TECAN.  This Tecan isn't concluding. 
</P><H4> Verification of Pseudovibrio transformed </H4><P>A result of sequencing of PseudoD with transposon is received. The sequence of 16S pseudoD is compared with the sequencing. There are 100% of identity. So it's  PseudoD , which is transformed. 




</P><SPAN class="cd-date">Aug 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">

A preculture of DH5α pir with pNK2 and PseudoD is performed.




</P><SPAN class="cd-date">Aug 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">

The map of the first plasmid pNK2 isn’t ok. There was mistake because this isn’t the RFP in the transposon but emGFP. 
To verified the insertion we used a new couple of primer 42b and 92c.
This approach did not work. There is no band revealed on the gel.
So this new primers which amplified the kanamycin in the transposas is bought.



</P><SPAN class="cd-date">Aug 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">To confirm that PseudoD is transformed, a PCR is realized with primer 112R 112F and Q5 high fidelity.
This development did not work. There is no band revealed on the gel.
Verification of insertion pNK2 in PseudoD with other primers 65F and 65R.  
This development did not work. There is no band revealed on the gel.

</P><SPAN class="cd-date">Aug 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Verification of transformed cells </H4><P>There are some colonies in the plate PseudoD transformed. 
PCR 16S of PseudoD  cell transformed with pNK2 is actualized. We used primer 1 and 2.

This PCR is purified by means of Macherey-Nagel Kit and send to sequence. 


</P><SPAN class="cd-date">Aug 14</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">

Pseudovibrio denitrificans transformed have been still growth.


</P><SPAN class="cd-date">Aug 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><H4> Transformation of Pseudovibrio denitrificans </H4><P>
Transformation of Pseudovibrio denitrificans (PseudoD) with pNK2 is realised, with Electric shock of 2000V .This voltage had been optimised in the part transformation. As Pseudovibrio lives with kanamycine at 25µg/mL, the cells is sowed in Marine Broth (MB) 1X with kanamycin 50µg/mL.

It parallel BL21 is so transformed with same plasmid but with electric shock of 1800V and sowed in LB 1X add kanamycin (25µg/mL).


</P><SPAN class="cd-date">Aug 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P>
A stock of the cells DH5α pir with pNK2 is performed.
 The plasmid pNK2 is purified from precultur of DH5α pir transformed. A kit of Macherey-Nagel of Plasmid DNA purification is used.
 DH5α pir transformed is sowed in LB kanamycin and one glycerol is making.

</P><SPAN class="cd-date">Aug 11</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">

Colonies in the plate DH5α pir with pNK2 is put in culture in 3mL of LB add kanamycin (25µg/mL) and the plasmid pNK2 is purified.

</P><SPAN class="cd-date">Aug 08</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify">
The electrocompetent DH5α pir is maked in 400mL of LB. 
The transformation of this cells with plasmid pNK2 is effected and the cells is sowed in LB kanamycine 25µg/mL.



</P><SPAN class="cd-date">Aug 07</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P>
It’s the first day where Brian Jester gave us the plasmid pNK2. This plasmid contains transposon. 
 A Precultur of DH5α pir is realized.In a 15 mL tube, add 2 mL of LB medium and inoculate cells from glycerol. These cells can replicate the plasmid because they contains pi protein. 

It parallel annotation of plasmid pNK2 with Blast is realized.</P><SPAN class="cd-date">Aug 08</SPAN></DIV></DIV></DIV></SECTION><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Notebook/Transposons">http://2014.igem.org/Team:Evry/Notebook/Transposons</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Notebook/Transposons" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Notebook/Transposons" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Notebook/Transposons&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Notebook/Transposons&amp;oldid=389602" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>