"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Biology/CellCharacterization/Antibiotics
From 2014.igem.org
Sensitivity to antibiotics
After having succeeded to make our bacteria grow, we tested their resistance to different antibiotics. Knowing their resistance to antibiotics is very important because it will allow us to finalize protocols of selection after transformation.
We chose to test the most commonly used antibiotics. We included the three antibiotics used in iGEM (kanamycin, chloramphenicol and ampicilin), plus the erythromycin and the tetracyclin. We chose the erythromycin to test a conjugation protocol which required this antibiotic for E.coli, and we chose to test tetracyclin because it is quite often used for inducible systems.
Table3: Concentrations tested for each antibiotic
We tested several concentrations of each antibiotics (cf: Table3), and we added a supplementary concentration for erythromycin because it is known to be ineffective on GRAM- bacteria.
The sensitivity tests were performed in four different conditions (cf: Figure3). We tested on Marine Broth (MB) 1X plates, MB 0.5X plates (used for conjugation) and M9-CASA 1X (+3% NaCl). We also tested in liquid M9-CASA 1X (+3% NaCl) to have a test in liquid culture, unfortunately we cannot measure the optical density with the MB medium because of its opacity.
The cells' sensitivity was measured for Pseudovibrio denitrificans, E. coli, and different clones of E. coli transformed with plasmids and carrying an acquired resistance AmpR, KanR, CamR ou ErmR.
Figure3: Protocol of antibiotics' tests on Pseudovibrio denitrificans.
We let all the media incubating at 30°C (with shaking for liquid cultures), and they have been checked at day 1, day 2 and day 3.    In the following figures, find the results obtained after three days of cultures on plates and one day of liquid cultures.
Figure4: Results of antibiotics' tests on MB plates.
On the left: Results on MB 0.5X plates.
On the right: Results MB 1X.
The number of CFU was not countable because of cellular lawn. The percentage of cell growth on plates was calculated by taking the controls of the same bacteria growing without antibiotics as reference.
Figure5: Results of antibiotics' tests for M9-CASA+3%NaCl media.
On the left: Results on M9-CASA+3%NaCl 1X plates.
On the right: Results in liquid M9-CASA+3%NaCl 1X.
The number of CFU was not countable because of cellular lawn. The percentage of cell growth on plates was calculated by taking the controls of the same bacteria growing without antibiotics as reference. The ΔOD (600nm) was calculated by subtracting the initial OD (600nm) at Day 0 from the OD(600nm) of the considered day.
Unfortunately, results for tetracyclin on MB medium are not useful. Indeed, all our bacteria have grown. It can be because of a technical problem with the drug, or it can also be explained by the presence of some genes bringing a resistance to this drug in Pseudovibrio denitrificans (see section Genome assembly ). But the death of cells on M9 plates doesn't allow us to confirm this hypothesis, even if it can be explained by a conditional expression of these genes.
We can also see that Pseudovibrio denitrificans manage to grow on very high concentrations of ampicillin. By sequencing our bacteria, we found resistance gene to ampicillin too. This drug is so not really useful, except in a huge concentration.
On a brighter side, according to these results, we can select Pseudovibrio denitrificans transformed  with a PSB1C3 on a concentration of chloramphenicol adjusted to the ratio 1/1000 (cf: Table3).
To use other drugs, best ratio (Growth on antibiotic/Growth on medium only) allowed determining optimal concentrations to use on our different media (cf: Table4).
Table4: Concentrations which can be used for selection of Pseudovibrio denitrificans
Retrieved from " http://2014.igem.org/Team:Evry/Biology/CellCharacterization/Antibiotics "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
