"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Biology/GenomeAssembly
From 2014.igem.org
IGEM Evry 2014
Toggle navigation
Home
Overview
Project
Achievements & Results
Team
Members
Attributions
Collaborations
Acknowledgments
Media
Biology
Parts
Toxic compounds
Chassis & Characterization
Genome Assembly
Sensors
Transposons
RNAseq
Modeling
Sponge Model
PCB Model
Phenol Model
Notebook
Cell characterization
Transposons
Sensing
RNAseq
Protocols
Policy & Practices
Philosophy
Ethics
Safety
Interlab
Aim & Results
Used Devices
Notebook
Spon-ge-sors
IGEM Bar
Biology - Genome Assembly
De novo Genome assembly
In order to perform the genome assembly of our  bacterium, we choose to used the velvet software. It is an algorithm base  on short read sequencing alignments. This reconstruction algorithm is based on the Brujin graphs which is in graph theory  a directed graph which represents overlaps between sequences of symbols, in or case our short contis DNA.  We choose 4 kmer size. For the first one at  31 we obtained  1493 contigs, for a kmer size of 65,  413 contigs are obtain. For the two last kmer tested, 97 and 113 we obtained 275 and 245 contigs.The prokka software is an automated tools for genome annotation. The run take less than ten minutes, as they was  proposed on their articles.In order to control the quality of our DNA seq we choose to made first a Proteome comparisons with Pseudovibrio  FOBEG1 which is our reference strain. For that purpose we decided to look at our first genome version contains 275 contigs which was provide by the Velvet de novo assembly tools. For the 5456 proteins of Pseudovibrio FO-BEG1, 5132 proteins  (which are our reference strain) are match with a prokka anotation CDS.  5002 with a unique alignment 4936, 4687 with an alignement  > 90%, 4415   > 95% and 2525 > 99%. For that purpose, we are sure that we sequenced a strain related to the Pseudovibrio genus.  After this proteome comparison, we tried to improved our genome assembly by improved the kmer size and we saw that the contigs numbers was reduced to 245. In  order to analyse the annotation and the quality of our genome annotation, we looked at four main specific annotation types such as antibiotic resitance, restriction enzyme, metals (such as cadmium, copper and mercury), and other toxic compounds (such as phenol, nitrate, nitrite).
Antibiotic resistance
Concerning the antibiotics, we look at specific antibiotics such as kanamycin, erythromycin, tetracycline, ampicillin, and chloramphenicol and other type of resistance gene. For the kanamycin and erythromycin, we found not relevant annotation. However, for the other ones, tetracycline, ampicillin, and chloramphenicol, we found specific annotations.  Regarding the tetracycline, we found that it possessed one Tetracycline repressor protein class H , two Tetracycline resistance protein, class C and nine Bacterial regulatory proteins, tetR family annotated genes. This result are coincident with our experiment where we found that Pseudovibrio is resistant to tetracycline.  About the ampicillin, one Metallo-beta-lactamase superfamily protein, Beta-lactamase, Beta-lactamase type II precursor, two Beta-lactamase precursor, Beta-lactamase hydrolase-like protein HTH-type transcriptional activator AmpR and five Putative beta-lactamase HcpC precursor were predicted by the Prokka software. Also this result is coincident with our experiment.  For the Chloramphenicol antibioctics, we predicted that the chloramphenicol phosphotransferase-like protein, Chloramphenicol 3-O phosphotransferase and Chloramphenicol acetyltransferase are present in our strain. The last one is used as a reporter gene in molecular biology. But in contrary to other antibiotics, the test does not reveal, that our strain is resistant to the Chloramphenicol.  Finally we look at other antibiotic resistance and we see that other antibiotics such as the Bleomycin are predicted. Bleomycin is used as a chemotherapy agents for the Hodgkin's lymphoma. Also it possessed the Multidrug resistance protein, MdtA, MdtA precursor, MdtB, MdtK, MdtN, MdtH, MdtG, which are known to play an important role in antibiotic resistance.
Restriction enzyme
During the transformation tests of our strain, we supposed that Pseudovibrio Denitrificans, was able to degrade DNA. Indeed all our electroporation attempts failed. They could not keep the plasmid inside them. One relevant research led us to the discovery of a Type I Restriction enzyme EcoKI. This site specific DNA methylase is not very common but can be found in a wide range of Gammaproteobacteria and so potentially into PseudoVibrio Denitrificans. We blasted the sequence of EcoRI gene with our bacteria genome, and we found that Pseudovibro have the gene.  EcoKI recognises the sequence 5’ AAC(N)6GTGC-3’ and acts depending on the methylation state of the DNA substrate. It can be a methyltransferase or an endonuclease. Psb1c3, first plasmid that we tried to transform ( E.Coli Ori) does not have the target sequence of EcoRI. Otherwise, in one plasmid sent by Dr Thomas Drepper from Heinrich-Heine-Universität Düsseldorf, Institute of Molecular Enzyme Technology, Group of Bacterial Photobiotechnology, which was from a bacterium very closed with Pseudovibro and which we tried as well to transform, there is the target sequence of the enzyme.  At this moment, Pseudovibro was not sequenced, so we could not find the localisation of the EcoKI gene. In order to integrate our constructions just in time, we opted for a transposon strategy.
Figure 1: Sequence of the pRhokHi vector and the hightlight of the EcoKI target sequence
Nitrate/nitrite
Concerninrg the nitrate and nitrite a potential gene transporter NarK Nitrate/and Nitrite was predicte. Also we discovered the protein NapE was predict in the genome of Pseudovibrio Denitrificans. This enzyme is known to reduce nitrate to nitrite. Other some other genes such as Respiratory nitrate reductase 1 alpha and game chaine and other which are involve in the nitrification process. It possessed also Required for formate-dependent nitrite reduction. Not required for the biosynthesis of any of the c-type cytochromes nor for the secretion of the periplasmic cytochromes. They are involve in the reduction of nitrite to amoniac
Cadmium
Concerning one of our compound target the cadmium, we were really surprised about it’s resistance to hight concentration. We found that the annotation reaveal two potentials genes for cadmium resistance. The putative cadmium-transporting ATPase  involved in cadmium/zinc transport and also the Cobalt-zinc-cadmium resistance protein CzcB. CzcB has been asociated with a Gene ontology terms  for Biological Process metal ion tranport and Molecular function  metal ion transmembrane transporter activity.  Other experiment need to be performed in order to be sure that the bacterium is resistant. A simple Northern Blot  of the CzcB, mRNA will give  a simple way to analyse the data and provide information of this enzyme
Copper
Copper is an essential heavy metal; chemical element. It plays a role in vital oxidation and reduction processes. Multicopper oxidase are enzymes able to oxidize substrate and are involved in copper homeostasis. They play a part in iron transfer and enhanced oxidase activity.   The role of copper homeostasis protein is still unknown, however some studies have established a link in intracellular trafficking of CuI. Copper-transporting P-type ATPase plays an essential role in copper balance as intestinal transport of copper in mammalian. Copper-sensing transcriptional repressor CsoR is involved in the cellular response in order to increase copper concentration inside bacterium.  Copper resistance protein A precursor is a soluble protein which catalyzes oxidation of Cu and interact with membrane. Periplasmic copper-binding protein (NosD) is a periplasmic protein which can insert copper into reductase apoenzyme. Copper-exporting P-type ATPase A forms a homodimer at high copper concentration and export it in co-operation with copZ.
Mercuric
Concerning the Mercure, a gene  named Mercuric reductase was found. It is known to  catalyzes the  two-electron reduction of mercuric ions   to elemental mercury using NADPH as an electron donor . That shows that potentially our bacteria can in  high mercury concentration be able to survive due to the creaction of NADPH which is an important metabolite.
Phenol
One of our project aim was to degrade phenol thank to Pseudovivrio denitrificans. However, many organisms are able to degrade phenol by their own, anaerobia or aerobe. (Khazi Mahammedilyas Basha, 2010) Before to transform our bacterium in this way we were looking for a biodegradation pathway. As Pseudovibrio denitricans is an anaerobic bacterium, we have focused on the anaerobic degradation pathway for phenol.
Figure2 : Anaerobic pathway degradation for phenol
In order to determine if our bacterium degrade phenol we watched if it owns different enzymes in the phenol pathway degradation. For the first enzyme; 4-hydroxybenzoate decarboxylase we found 4 coding sequence with similarity; they have an enzymatic promiscuity. The second enzyme; p-hydroxy benzoate 3-monooxygenase from the phenol degradation pathway match perfectly with our bacterium. For Protocatechuate 3, 4 dioxygenase , we have found two coding sequences corresponding. The cycloisomerase coding sequence match perfectly in our bacterium. As the previous enzyme, enol - lactonase have matching too For the transferase enzyme, we have found two similar sequence leading to promiscuity enzymes. For the two last enzymes; they gave two different final product. One of them; thiolase have 4 similar sequences leading to promiscuity enzymes, though the second one; acyltransferase have 17 similar sequences leading to promiscuity enzymes.     So, this analyze lead to think that Pseudovibrio denetifricans own the all anaerobic phenol degradation pathway. Hence, we have found all coding sequences correlate to enzyme from phenol degradation. And the bacterium may rather the acetyl-coA pathway than succenyle-coA because of the promiscuity enzyme quantity.
REFERENCE
Article, R. (2010). Recent advances in the Biodegradation of Phenol: A review, 1(2), 219–234.
Harwood, C. S., & Parales, R. E. (1996). THE β -KETOADIPATE PATHWAY AND THE BIOLOGY OF SELF-IDENTITY.
Powell, L. M., Dryden, D. T. F., & Murray, N. E. (1998). Sequence-specific DNA Binding by Eco KI , a Type IA DNA Restriction Enzyme.
Contacts
igemevry@gmail.com
Retrieved from " http://2014.igem.org/Team:Evry/Biology/GenomeAssembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
