<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Notebook_CellCharacterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Notebook/CellCharacterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Notebook/CellCharacterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/CellCharacterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/CellCharacterization&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Notebook/CellCharacterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Notebook/CellCharacterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>IGEM Evry 2014</TITLE></P><HEADER id="top-header"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2014.igem.org/Team:Evry">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Overview </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Overview/Achievements">Achievements &amp; Results</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team </A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam">Members</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Acknowledgments">Acknowledgments</A></LI><LI><A href="https://2014.igem.org/Team:Evry/EvryTeam/Press">Media</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Biology </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/ToxicCompounds">Toxic compounds</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Chassis">Chassis &amp; Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/GenomeAssembly">Genome Assembly</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Sensors">Sensors</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Biology/RNAseq">RNAseq</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/Sponge">Sponge Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/pcb_model">PCB Model</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Model/phenol_model">Phenol Model</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/CellCharacterization">Cell characterization</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Transposons">Transposons</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/RNAseq">RNAseq</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Policy &amp; Practices </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Philosophy">Philosophy</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Policy and Practices/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab </A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Results">Aim &amp; Results</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Used_Devices">Used Devices</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Interlab_Study/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Evry/Spon-ge-sors">Spon-ge-sors</A></LI><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">IGEM Bar </A></LI></UL></DIV></DIV></NAV></HEADER><DIV id="st-container" class="st-container"><DIV class="st-pusher"><DIV class="st-content"><SECTION id="top-section-hero" style="background-image: url('https://static.igem.org/mediawiki/2014/5/51/NotebookEvry.jpg');"><DIV class="jumbotron text-left"><DIV class="animated hiding" data-animation="fadeInLeft" data-delay="500"><H1>Notebook - Cell Characterization</H1></DIV></DIV></SECTION><SECTION id="timeline"><DIV id="cd-timeline" class="cd-container"><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">New protocols of electrocompetent <I>Pseudovibrio</I> - Results </FONT></BIG></B></U>
The transformation didn't work with <I>Pseudovibrio</I> washed 5 times or 6 times. </P><SPAN class="cd-date">Oct 09</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">New protocols of electrocompetent <I>Pseudovibrio</I></FONT></BIG></B></U>
Transformation was perfomed on the new stocks made the 10-06-14 with two types of washes.
1µL of plasmids PBBR1MCS and pRhokHI-2 (25 and 33ng/mL). 
2000 V 
(<A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocole</A>)
</P><SPAN class="cd-date">Oct 07</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">New protocols of electrocompetent <I>Pseudovibrio</I></FONT></BIG></B></U>
On a pre-culture of <I>Pseudovibrio</I> in MB 1X 3mL, we relaunch the culture in 100mL of MB 1X. We made the electrocompetent protocol (<A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocole</A>).
We tested to make 5 and 6 washes with glycerol 10% instead of three. 
Stocks of cells were stored at -80°C (3 weeks maximum) </P><SPAN class="cd-date">Oct 06</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Confirmation </FONT></BIG></B></U>
We tried to confirm the presence of our plasmid of previous transformations by extracting potential plasmids and digest them with EcoRI. After the incubation and the inactivation of the enzyme, products were migrating.
No band on the gel. The control (lab stock) didn't work either, so there is probably a problem of quantity of DNA.</P><SPAN class="cd-date">Sep 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Results </FONT></BIG></B></U>
The transformation didn't work.</P><U><B> Stock of pBBR1MCS and pRhokHI-2 - Results</B></U><SPAN class="cd-date">Sep 11</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Confirmation </FONT></BIG></B></U>
A contamination has been shown in another transformation test. We can't use our previous results because there is a chance that it's not <I>Pseudovibrio</I> but the contamination which had been transformed.
We can still try to prove that it's our bacteria (with amplification of specific sequence) ond that there is our plasmids.
New transformation was tried with another stock of competent <I>Pseudovibrio</I> (<A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocole</A>) and cells were plated on MB/MB+Cam(1:1000)/MB+Kan(1:1000).<U><B> Stock of pBBR1MCS and pRhokHI-2 </B></U>
The colony of E.Coli transformed in LB 1X + Cam (1:1000) culture of 3mL was sowed on new LB/LB+Cam(1:1000)/LB+Kan(1:1000) plates and incubated with shaking overnight at 37°C.</P><SPAN class="cd-date">Sep 10</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Confirmation </FONT></BIG></B></U>

The previous digestion could fail because of the too little time of incubation (15min). We re-try to make the digestion with an incubation of 1 hour. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Gel of digestion with XbaI (2)</CENTER></DIV></DIV></DIV></DIV><P>
There was a problem with the digestion. We are going to try with another enzyme.<U><B> Stock of pBBR1MCS and pRhokHI-2 </B></U>
To remake a stock of these plasmids, we showed a colony of E.Coli transformed in a new LB 1X + Cam (1:1000) culture of 3mL.</P><SPAN class="cd-date">Sep 09</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Confirmation</FONT></BIG></B></U></P><P>We made a restriction control of our plasmid, and we tried to prove that these plasmids are present in our Pseudovibrio. 
After made a purification of plasmids in A1 and A2 with NucleoSpin Plamsid Kit (Macherey Nagel), we obtained concentrations at 380.1 ng/µL for pRhokHI-2 and at 91.1 ng/µL for pBBR1MCS.
We disgested these sample and our stock of pRhokHI-2 and pBBR1MCS with XbaI (unique restriction site on these plasmids) and we have make migrate the product of digestion. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><CENTER>Gel of digestion with XbaI</CENTER></DIV></DIV></DIV></DIV><P>
There was a problem with the digestion. We are going to try with a longer time of incubation.
<U><B> Stock of pBBR1MCS and pRhokHI-2 </B></U>
To remake a stock of these plasmids, we put in liquid LB+Cam (1:1000) culture, a colony from a LB+Cam plate of E.Coli transformed.
</P><SPAN class="cd-date">Sep 08</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333"> New stock of electro competents Pseudovribio - Antibiotics' control</FONT></BIG></B></U></P><SPAN class="cd-date">Sep 06</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">New stock of electro competents Pseudovribio </FONT></BIG></B></U>
We choose to make a new stock of electro competent Pseudovibrio denitrificans.
We made a culture of 100mL from a pre-culture of 10mL made the 04/09. 
The new stock was test on plates MB 1X with Cam and Kan (1:1000).</P><SPAN class="cd-date">Sep 05</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333"> Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Results </FONT></BIG></B></U><LI> Positive control with E.Coli : </LI></P><DIV align="center"><LI> Resulst with Pseudovibrio : 
NB: The negative control with E.Coli does'nt work on any plates. The culture was probably too old. 
A problem occurs with stocks of electro-competent Pseudovibrio because wa can see that it growth on Kanamycine 1:1000. 
On Chloramphenicol, the contamination does'nt seem to grow. 
We have to prove that it's Pseudovibrio wich growth on Chloramphenicol.


<SPAN class="cd-date">Sep 02</SPAN></LI></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B> Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Transformation </B></U><TABLE border="1"><TBODY><TR><TD><B>pBBR1MCS </B></TD><TD><B>pRhokHI-2 </B></TD><TD><B>Ø plasmid </B></TD></TR><TR><TD>Pseudo replicate 1 </TD><TD>Pseudo replicate 1 </TD><TD>Pseudo </TD></TR><TR><TD>Pseudo replicate 2 </TD><TD>Pseudo replicate 2 </TD><TD>E.Coli Bl21 </TD></TR><TR><TD>E.Coli Bl21 </TD><TD>E.Coli Bl21 </TD></TR></TBODY></TABLE></P><P><U><B> Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Selection </B></U>
Plates are showed and organised like shown in the following picture: </P><DIV style="float:left;"><TABLE border="1"><TBODY><TR><TD><B>A1 </B></TD><TD><B>B1 </B></TD><TD><B>C1 </B></TD><TD><B>D1 </B></TD></TR><TR><TD>Pseudo+pBBR1MCS (1)</TD><TD>Pseudo+pBBR1MCS (2)</TD><TD>Pseudo </TD><TD>Pseudo Ø plasmid </TD></TR></TBODY></TABLE></DIV><DIV style="float:right;"><TABLE border="1"><TBODY><TR><TD><B>A2 </B></TD><TD><B>B2 </B></TD><TD><B>C2 </B></TD><TD><B>D2 </B></TD></TR><TR><TD>Pseudo+pRhokHI-2 (1)</TD><TD>Pseudo+pRhokHI-2 (2)</TD><TD>Pseudo </TD><TD>Pseudo Ø plasmid </TD></TR></TBODY></TABLE></DIV><DIV style="float:left;"><TABLE border="1"><TBODY><TR><TD><B>A3 </B></TD><TD><B>B3 </B></TD><TD><B>C3 </B></TD></TR><TR><TD>E.Coli Bl21 </TD><TD>Bl21 Ø plasmid </TD><TD>Bl21+pRhokHI-2 </TD></TR></TBODY></TABLE></DIV><DIV style="float:right;"><TABLE border="1"><TBODY><TR><TD><B>A4 </B></TD><TD><B>B4 </B></TD><TD><B>C4 </B></TD></TR><TR><TD>E.Coli Bl21 </TD><TD>Bl21 Ø plasmid </TD><TD>Bl21+pBBR1MCS </TD></TR></TBODY></TABLE></DIV><DIV align="center"><P>
We also made MB only plates to see the normal growth of our bacteria,which were showed and organised like shown in the following picture: </P></DIV><TABLE border="1"><TBODY><TR><TD><B>A </B></TD><TD><B>B </B></TD><TD><B>C </B></TD><TD><B>D </B></TD><TD><B>a </B></TD><TD><B>b </B></TD><TD><B>c </B></TD><TD><B>d </B></TD><TD><B>e </B></TD><TD><B>f </B></TD></TR><TR><TD>E.Coli Bl21 </TD><TD>Bl21 Ø plasmid </TD><TD>Bl21+pBBR1MCS </TD><TD>Bl21+pRhokHI-2 </TD><TD>Pseudo </TD><TD>Pseudo Ø plasmid </TD><TD>Pseudo+pRhokHI-2 (1)</TD><TD>Pseudo+pRhokHI-2 (2)</TD><TD>Pseudo+pBBR1MCS (1)</TD><TD>Pseudo+pBBR1MCS (2)</TD></TR></TBODY></TABLE><SPAN class="cd-date">Sep 01</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333"> Transformation of Pseudovrbrio with pBBR1MCS and pRhokHI-2 - Competents cells</FONT></BIG></B></U>

New stock of electro-competent Pseudovibrio was made from a pre-culture of 100mL.
(<A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocole</A>)

</P><SPAN class="cd-date">Aug 29</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><FONT color="#003333"><B> Amplification of pRhokHI-2 in E.Coli Bl21 - Results</B></FONT></BIG></U>
After incubation overnight we can see some colonies of E.Coli Bl21 transformed on LB+Cam 1:1000 and on LB+Kan 1:1000. Our Bl21 which had not been transformed doesn't growth on these media but both grow on LB only.
We had well success to tranform E.Coli and amplifly the plasmid.

</P><P>
We tranfered two colonies into liquid MB+Kan(1:1000) and let them in incubator overnight to have a pre-culture for the plasmid purification with NucleoSpin Plamsid Kit (Macherey Nagel).

</P><SPAN class="cd-date">Aug 28</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Differed launch of antibiotics' tests</FONT></B></BIG></U><LI> Tests of Ampicilin's resistance cassette: 
</LI></P><DIV align="center"><B><U>Graph and photo of Ampicilin's resistance tests on MB 1X plates-Day3</U></B></DIV><DIV align="center"><B><U>Graph and of Ampicilin's resistance tests on MB 0.5X plates-Day3</U></B></DIV><DIV align="center"><B><U>Graph and of Ampicilin's resistance tests on M9 1X plates-Day3</U></B></DIV><DIV align="center"><B><U>Photos of plates-Day3</U></B></DIV><SPAN class="cd-date">Aug 28</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Amplification of pRhokHI-2 in E.Coli Bl21</FONT></BIG></B></U>
The transformation of E.Coli with pRhokHi-2 still didn't works. We still have to amplify it so we re-tried to the tranformation (1µL of plasmid for 40µL of cells, 1800V). 
This time, after a liquid culture of only one hours cells were plated on LB 1X + Kan (1:1000), LB 1X + Cam (1:1000), and LB only and let in incubator overnight.
</P><SPAN class="cd-date">Aug 27</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Differed launch of antibiotics' tests</FONT></B></BIG></U><LI> Tests of Ampicilin's resistance cassette: 
</LI></P><DIV align="center"><B><U>Graph of Ampicilin's resistance tests on MB 1X plates-Day2</U></B></DIV><DIV align="center"><B><U>Graph of Ampicilin's resistance tests on MB 0.5X plates-Day2</U></B></DIV><DIV align="center"><B><U>Graph of Ampicilin's resistance tests on M9 1X plates-Day2</U></B></DIV><SPAN class="cd-date">Aug 27</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of Pseudovribrio with pBBR1MCS - Results</FONT></BIG></B></U>
We saw no colony on our plates. We decided to test our two replicates of transformation on MB 1X plates with other concentrations of antibiotic. We made three plates with Cam 1:5000, and 1:10 000, then Pseudovibrio were showed on them and let overnight in incubator.








</P><SPAN class="cd-date">Aug 26</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Differed launch of antibiotics' tests</FONT></B></BIG></U><LI> Tests of Ampicilin's resistance cassette: 
</LI></P><DIV align="center"><B><U>Graph of Ampicilin's resistance tests in liquid M9 1X-Day3</U></B></DIV><DIV align="center"><B><U>Graph of Ampicilin's resistance tests on MB 1X plates-Day1</U></B></DIV><DIV align="center"><B><U>Graph of Ampicilin's resistance tests on MB 0.5X plates-Day1</U></B></DIV><DIV align="center"><B><U>Graph of Ampicilin's resistance tests on M9 1X plates-Day1</U></B><B>No readable results</B></DIV><SPAN class="cd-date">Aug 26</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Amplification of pRhokHI-2 in E.Coli Bl21 </FONT></BIG></B></U>
We re-tried to transform E.Coli with pRhokHI-2 to amplify it (1µL of plasmid for 40µL of cells, 1800V). This time, after a liquid culture of only one hours cells were plated on LB 1X + Kan (1:1000) and let in incubator overnight.
<U><B><BIG><FONT color="#003333">Transformation of Pseudovribrio with pBBR1MCS</FONT></BIG></B></U>
After made a purification of pBBR1MCS with NucleoSpin Plamsid Kit (Macherey Nagel), electrocompetent Pseudovibrio cells were tranformed with the plasmid by electroporation (1µL of plasmid for 40µL of cells, 1800V), then they were incubated for three hours and showed on MB 1X+Cam (1:1000) plates. Plates are let in incubator overnight.





</P><SPAN class="cd-date">Aug 25</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Differed launch of antibiotics' tests</FONT></B></BIG></U><LI> Tests of Kanamycin's resistance cassette: 
</LI></P><DIV align="center"><B><U>Graph of Kanamycin's resistance tests in liquid M9 1X-Day3</U></B></DIV><LI> Tests of Ampicilin's resistance cassette: 
<DIV align="center"><B><U>Graph of Ampicilin's resistance tests in liquid M9 1X-Day2</U></B></DIV><SPAN class="cd-date">Aug 25</SPAN></LI></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Amplification of plasmids pBBR1MCS and pRhokHI-2 in E.Coli Bl21 - Results</FONT></BIG></B></U></P><P>
Amplification of pBBR1MCS works and we purified it with NucleoSpin Plamsid Kit (Macherey Nagel) . 
Unfortunately the transformation with pRhokHI-2 didn't works so we have to retry.


</P><SPAN class="cd-date">Aug 24</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Differed launch of antibiotics' tests</FONT></B></BIG></U><LI><B><B> Tests of Kanamycin's resistance cassette: 
</B></B></LI></P><DIV align="center"><B><U>Graph of Kanamycin's resistance tests in liquid M9 1X-Day2</U></B></DIV><LI> Tests of Ampicilin's resistance cassette: 
<DIV align="center"><B><U>Graph of Kanamycin's resistance tests in liquid M9 1X-Day1</U></B></DIV><SPAN class="cd-date">Aug 24</SPAN></LI></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Amplification of plasmids pBBR1MCS and pRhokHI-2 in E.Coli Bl21</FONT></BIG></B></U>

Two plasmids pBBR1MCS and pRhokHI-2 were generously sent by Dr Thomas DREPPER (University Duesseldorf, Dept. of Biology, Institute of Molecular Enzyme Technology)
</P><P>
Bowth were transfered into Bl21 electro-competent cells by electroporation (two replicate)(1µL of plasmid for 40µL of cells, 1800V).  After a liquid culture of three hours, cells were plated on LB 1X + Cam (1:1000) and let in incubator overnight.</P><SPAN class="cd-date">Aug 23</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Antibiotics' tests on MB 1X plates - Results Day3</FONT></B></BIG></U>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.<LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  </LI></P><U>Antibiotics' tests on MB 0.5X plates - Results Day3</U>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.<LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <U>Antibiotics' tests on M9 1X plates - Results Day3</U><P>Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.</P></LI><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <U>Differed launch of antibiotics' tests</U></LI><LI> Tests of Kanamycin's resistance cassette: 
<DIV align="center"><B><U>Graph of Kanamycine's resistance tests in liquid M9 1X-Day1</U></B></DIV></LI><LI> Tests of Ampicilin's resistance cassette: 
<P>
Antibiotics' tests in liquid M9 for E.Coli transformed with a plasmid PSB1A3 containing an ampicilin resistance cassette were launch.
In two days, these cultures will be plated on MB 1X, MB 0.5X and M9 1X plates
</P><SPAN class="cd-date">Aug 23</SPAN></LI></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><U><BIG><FONT color="#003333">Antibiotics' tests on MB 1X plates - Results Day2</FONT></BIG></U><P>Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.</P><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <U>Antibiotics' tests on MB 0.5X plates - Results Day2</U><P>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.</P></LI><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <U>Antibiotics' tests on M9 1X plates - Results Day2</U>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.</LI><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <DIV align="center">Aug 22</DIV></LI></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Antibiotics' tests in liquid M9 1X - Results Day3</FONT></B></BIG></U><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  </LI></P><U>Antibiotics' tests on MB 1X plates - Results Day1</U>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.<LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <U>Antibiotics' tests on MB 0.5X plates - Results Day1</U>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.</LI><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  <DIV align="center">&quot;</DIV><U>Antibiotics' tests on M9 1X plates - Results Day1</U>
Results of antibiotics' tests on plate are calculated as percentage of bacteria's growth by compared with the controle case.
Pseudovibrio coming from culture with antibiotics (part B on plates), are taken from antibiotics' range in M9 at Day2.
  </LI><LI>Chloramphenicol: 
<B>No readable results</B></LI><LI>Kanamycin: 
<B>No readable results</B></LI><LI>Ampicilin: 
<B>No readable results</B></LI><LI>Tetracyclin: 
<B>No readable results</B></LI><LI>Erythromycin: 
<B>No readable results</B><SPAN class="cd-date">Aug 21</SPAN></LI></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Antibiotics' tests in liquid M9 1X - Results Day2</FONT></B></BIG></U><LI>Chloramphenicol
  </LI><LI>Kanamycin
  </LI><LI>Ampicilin
  </LI><LI>Tetracyclin
  </LI><LI>Erythromycin
  </LI></P><U>Launch of antibiotics' tests on plates</U>

Plates are showed and organised like shown in the following picture, and showed with 1µL of each culture:<TABLE border="1"><TBODY><TR><TD><B>A </B></TD><TD><B>B </B></TD><TD><B>C </B></TD><TD><B>D </B></TD></TR><TR><TD>Pseudovibrio denitrificans </TD><TD>Pseudovribrio denitrificans* </TD><TD>E. Coli</TD><TD>E. Coli**</TD></TR></TBODY></TABLE>

*Bacteria cultivated in media with antibiotic
**Bacteria tranformed with a plasmid containing a resistance cassette for the antibiotic<SPAN class="cd-date">Aug 20</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><BIG><B><FONT color="#003333">Antibiotics' tests in liquid M9 1X - Results Day1</FONT></B></BIG><UL><LI>Chloramphenicol
</LI><LI>Kanamycin

Because of technical problems, the tests on E.Coli+KanR are launched with a gap of few days.

</LI><LI>Ampicilin
</LI><LI>Tetracyclin
</LI><LI>Erythromycin
</LI></UL></P><SPAN class="cd-date">Aug 19</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><BIG><B><FONT color="#003333">Tests of antibiotics' stocks - Results(2)</FONT></B></BIG></U><TABLE border="1"><TBODY><TR><TD><B>Plates </B></TD><TD>LB Agar only </TD><TD>LB+Kan </TD><TD>LB+Erm (1:100)</TD></TR><TR><TD><B>E.coli</B></TD><TD>X  </TD><TD>0  </TD><TD>0  </TD></TR></TBODY></TABLE></P><P>
New stocks are ready for antibiotics' tests on Pseudovribrio<U>Launch of antibiotics's tests in liquid M9 1X</U>

For each tested antibiotic we made the following range of concentrations:</P><TABLE border="1"><TBODY><TR><TD><B>0 </B></TD><TD><B>1:500 </B></TD><TD><B>1:1000 </B></TD><TD><B>1:50 000 </B></TD><TD><B>1:100 000 </B></TD></TR></TBODY></TABLE><P>
We made these range two time to have a replicate and we added a controle tube without antibiotic for each bacteria.

Each tube is made like in the following table in Falcon tubes (15mL):</P><TABLE border="1"><TBODY><TR><TD><B>/ </B></TD><TD><B>0 </B></TD><TD><B>1:500 </B></TD><TD><B>1:1000 </B></TD><TD><B>1:50 000 </B></TD><TD><B>1:100 000 </B></TD></TR><TR><TD><B>Medium+agar </B></TD><TD>3mL </TD><TD>3mL </TD><TD>3mL </TD><TD>3mL </TD><TD>3mL </TD></TR><TR><TD><B>Antibiotic </B></TD><TD>0 </TD><TD>6µL of stock</TD><TD>3µL of stock</TD><TD>6µL of stock diluted 1/100</TD><TD>3µL of stock diluted 1/100</TD></TR><TR><TD><B>Pre-culture </B></TD><TD>300µL</TD><TD>300µL</TD><TD>300µL</TD><TD>300µL</TD><TD>300µL</TD></TR></TBODY></TABLE><P>

For each range, we test Pseudovibrio denitrificans but also E.Coli and E.Coli tranformed with a speicific resistance cassette (NM:The E.Coli strain for the control case is the same strain that is tranformed for the cassette efficiency test).
The initial OD at 600nm of each bacteria has been mesured to after be able to calculate the ΔOD at 600nm, during three days.</P><TABLE border="1"><TBODY><TR><TD><B>/ </B></TD><TD><B>Pseudo </B></TD><TD><B>Bl21 </B></TD><TD><B>DH5a </B></TD><TD><B>Top 10 </B></TD><TD><B>DH5a pyr+KanR </B></TD><TD><B>DH5a+CamR </B></TD><TD><B>Top 10+ErmR </B></TD></TR><TR><TD><B>OD init </B></TD><TD><B>0.02 </B></TD><TD><B>0.795 </B></TD><TD><B>0.873 </B></TD><TD><B>0.693 </B></TD><TD><B>0.408 </B></TD><TD><B>0.687 </B></TD><TD><B>0.319 </B></TD></TR></TBODY></TABLE><P><U>Plates for antibiotics's tests</U>

For each tested antibiotic we made the same range of concentrations than in liquid.
We also made these range two time to have a replicate and we added a controle plate without antibiotic on which we showed all our tested bacteria. Plates will be checked during three days.
We made these same ranges of tested antibiotic's concentrations for three media:</P><UL><LI>MB agar 1X
</LI><LI>MB agar 0.5X
</LI><LI>M9 agar 1X
</LI></UL><P>
Each plate is made like in the following table:</P><TABLE border="1"><TBODY><TR><TD><B>/ </B></TD><TD><B>0 </B></TD><TD><B>1:500 </B></TD><TD><B>1:1000 </B></TD><TD><B>1:50 000 </B></TD><TD><B>1:100 000 </B></TD></TR><TR><TD><B>Medium+agar </B></TD><TD>20mL </TD><TD>20mL </TD><TD>20mL </TD><TD>20mL </TD><TD>20mL </TD></TR><TR><TD><B>Antibiotic </B></TD><TD>0 </TD><TD>40µL of stock</TD><TD>20µL of stock</TD><TD>40µL of stock diluted 1/100</TD><TD>20µL of stock diluted 1/100</TD></TR></TBODY></TABLE><SPAN class="cd-date">Aug 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><B><U><BIG><FONT color="#003333">Tests of antibiotics' stocks - Results</FONT></BIG></U></B><TABLE border="1"><TBODY><TR><TD><B>Plates </B></TD><TD>LB Agar only </TD><TD>LB+Cam </TD><TD>LB+Kan </TD><TD>LB+Amp </TD><TD>LB+Erm </TD><TD>LB+Tet </TD></TR><TR><TD><B>E.coli</B></TD><TD>X  </TD><TD>0  </TD><TD>X  </TD><TD>0  </TD><TD>X  </TD><TD>0  </TD></TR></TBODY></TABLE><P>
There was a problem with the stock of Kanamycine so we made a new one at 25mg/mL.
For the erythrompycine, we learn that E. Coli is not really sensitive to this antibiotic and that it's better to use it  with a dilution at 1:100.</P><B><U><BIG><FONT color="#003333">Survivability tests - Results</FONT></BIG></U></B><TABLE border="1"><TBODY><TR><TD><B>liquid cultures/Plates </B></TD><TD><B>MB </B></TD><TD><B>M9 </B></TD></TR><TR><TD><B>MB </B></TD><TD><B>++++ </B></TD><TD><B>++ </B></TD></TR><TR><TD><B>M9 </B></TD><TD><B>++ </B></TD><TD><B>+ </B></TD></TR></TBODY></TABLE><P>
Any liquid culture can be plated and will grow up. Of course, it works better on MB medium, wich is a rich medium than on M9.
</P><SPAN class="cd-date">Aug 17</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><B><U><BIG><FONT color="#003333">Tests of antibiotics' stocks</FONT></BIG></U></B>

Six plates of LB agar were made. 
Five of them contained one of those antibiotics in the dilution 1:1000:<LI> Chloramphénicol
</LI><LI> Kanamycin
</LI><LI> Erythromycin
</LI><LI> Ampicilin
</LI><LI> Tetracyclin
 

(The last one was the control of the growth of our bacteria without antibiotics)
We sowed 10µL of a liquid LB culture of Bl21, and we let them incubate overnight at 37°C.<B><U><BIG><FONT color="#003333">Survivability tests</FONT></BIG></U></B>
Two plates of MB 1X and M9 1X were made and divised in two parts.
Each plate was sowed with 5µL of a liquid MB 1X culture of Pseudovibrio denitrificans(dated 12th August) in a part and with 5µL of a liquid M9 1X culture Pseudovibrio denitrificans (dated 12th August) in the other part.<U><B><BIG><FONT color="#003333">Pre-cultures</FONT></BIG></B></U>
Cultures in MB 1X and M9 1X of Pseudovibrio were passed by a 1/10 dilution and let incubate overnight at 30°C.
New cultures of different E.Coli were also started from glycerol or plate and let incubated overnight at 37°C.

From glycerol stocks:</LI><LI>Bl21
</LI><LI>Top10
</LI><LI>DH5a

From plates:
</LI><LI>DH5a tranformed with pCB1C3 (CamR)
</LI><LI>Top10 transformed with pQexp (ErmR)
</LI><LI>DH5a pyr tranformed with pMK2 (KanR)

Bacteria tranformed with plasmid were cultivated with the corresponding antibiotic for the selection.
For each medium we make a negative contrôle without bacteria.


</LI></P><SPAN class="cd-date">Aug 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333"> Tests for development of the electroporation's protocol </FONT></BIG></B></U>

*Results of M9 1X plates - Day2: </P><P>
Cold sorbitol seems to be the best way to wash our Pseudovibrio cells and avoid cells' death on M9 plates too.</P>
*Results CFU: <P>
Cold sorbitol is the best way to wash cells because it induce the bigest number of colonies on MB1X and bigger colonies on M91X.</P><SPAN class="cd-date">Aug 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Tests for development of the electroporation's protocol </FONT></BIG></B></U></P><P>
*Results of M9 1X plates - Day1: </P><P>
Results on M9 plates are not very readable.</P><P>
*Results of MB 1X plates - Day1: </P><P>
Cold sorbitol seems to be the best way to wash our Pseudovibrio cells and avoid cells' death on MB plates.
</P><SPAN class="cd-date">Aug 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Tests for development of the electroporation's protocol</FONT></BIG></B></U>

*Showed of MB Pseudovribrio electro-competent (which are stocked at -80°C) on MB 1X plates.
*Showed of M9 Pseudovribrio electro-competent (which are stocked at -80°C) on M9 1X plates.</P><SPAN class="cd-date">Aug 11</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Tests for development of the electroporation's protocol </FONT></BIG></B></U>

Because of technical problems, we have to re-try with a culture of Pseudvirbrio in MB 1X.
Same protocol as 5th August.</P><SPAN class="cd-date">Aug 06</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333"> Tests for development of the electroporation's protocol</FONT></BIG></B></U><LI>Re-launch of 50mL of Pseudovribrio in MB and M9 from pre-culture of 4th August. </LI><LI>Incubation at 30°C and waiting for the exponential growth phase.
&gt; OD(600nm) in M9 = 0.15 
&gt; OD(600nm) in MB = 1.5 <B>WORK IN ICE</B></LI><LI>Centrifuge 10min at 4000rmp and 4°C
</LI><LI>Wash cells 1:1 of the initial volume with <U>cold sorbitol</U>, <U>cold MiliQ H2O</U> or <U>cold glycerol 10%</U></LI><LI>Centrifuge 10min at 4000rmp and 4°C
</LI><LI>Wash cells 1:2 of the initial volume with <U>cold sorbitol</U>, <U>cold MiliQ H2O</U> or <U>cold glycerol 10%</U></LI><LI>Centrifuge 10min at 4000rmp and 4°C
</LI><LI>Wash cells 1:10 of the initial volume with <U>cold sorbitol</U>, <U>cold MiliQ H2O</U> or<U> cold glycerol 10%</U></LI><LI>Centrifuge 10min at 4000rmp and 4°C
</LI><LI>Resuspend cells 1:100 of the initial volume with <U>cold sorbitol</U>, <U>cold MiliQ H2O</U> or <U>cold glycerol 10%</U></LI><LI>Stock cells in aliquots of 50µL at -80°C





</LI></P><SPAN class="cd-date">Aug 05</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Tests for development of the electroporation's protocol</FONT></BIG></B></U>

*Pre-culture of Pseudovibrio denitrificans in 3mL of Marine broth 1X 
*Pre-culture of Pseudovibrio denitrificans in 3mL of M9+CasAmino Acids 1X 
Incubation overnight, 30°C.


</P><SPAN class="cd-date">Aug 04</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Voltage tests</FONT></BIG></B></U></P><SPAN class="cd-date">July 31</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Curve of growth</FONT></BIG></B></U>
We previously tried to af a curve of grotwh by mesuring the DO (600nm), but because of the opaqueness of the MB these data are useless.
We measured the growth of <I>Pseudovibrio denitrificans</I> on the TECAN in M9-CASA+3%NaCl and MB filtred (because of the opaqueness of the non filtred MB)</P> 
Unfortunately, cells didn't grow very well in the TECAN due to the little volume use and the ratio air/culture which is not optimal.<SPAN class="cd-date">July 30</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Construction of a new plasmid - Choice and amplification of the ORI</FONT></BIG></B></U>

Purification of these PCR products.
Use “GenJEt” PCR Purification for it. The protocol is same that PCR purification except that the
binding buffer is add (1μL for 1mg) and boil at 55°C during 10 mn.

After the purification, a new PCR is realized with 3μL to purificated fragments.
Send to sequencing. </P><SPAN class="cd-date">July 25</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Construction of a new plasmid - Choice and amplification of the ORI</FONT></BIG></B></U>
PCR with 
<LI>primers 49 and 50 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>)
</LI><LI>Q5 high fidelity enzyme
</LI><LI>TM=60°C.

Migration of the PCR products, we have two bands per well.</LI></P><DIV align="center"><B><U>Figure:</U></B> Result of ORI amplification in <I>Pseudovibrio denitrificans</I> (Pd), <I>Pseudovribrio ascidiaceicola</I> (Pa) and <I>E.coli</I> DH5a.</DIV>
The negative control with <I>E.coli</I> DH5a confirms that there is no amplification. For <I>Pseudovibrio</I> strains, we obtain one band for <I>Pseudovibrio denitrificans</I> and two for <I>Pseudovibrio ascidiaceicola</I>. <SPAN class="cd-date">July 23</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Construction of a new plasmid - Choice and amplification of the ORI</FONT></BIG></B></U>
We chose to amplify the genes RepA, RepB and RepC. We find these three genes with the genome browser on the genome of <I>Pseudovibrio FOBEG1</I>.
PCR with 
<LI>primers 49 and 50 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>)
</LI><LI>Q5 high fidelity enzyme
</LI><LI>TM=55°C.

Migration of the PCR products, we have no band the amplification didn't work.</LI></P><SPAN class="cd-date">July 22</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Construction of a new plasmid - Verification of the promoter</FONT></BIG></B></U>
A PCR was performed with primers 5 and 6 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>). 
We purified those PCR product and sent them to be sequenced
Results :
</P><SPAN class="cd-date">July 21</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Construction of a new plasmid - Choice and amplification of the promoter</FONT></BIG></B></U>
We chose to amplify the tkt promoter of the transkelotase gene. We find this strong constitutive promoter with the genome browser on the genome of <I>Pseudovibrio FOBEG1</I>.
PCR with primers 5 and 6 (see primers table in <A href="https://2014.igem.org/Team:Evry/Notebook/Protocols">Protocols</A>), Q5 high fidelity enzyme and TM=55°C.
</P><SPAN class="cd-date">July 18</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of <I>Pseudovibrio</I> with PSB1C3 (2) - Results</FONT></BIG></B></U>
We have no colonies on the plates.</P><SPAN class="cd-date">July 16</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of <I>Pseudovibrio</I> with PSB1C3 (2)</FONT></BIG></B></U>
We made electrocompetent <I>Pseudovibrio</I> and transform them by electroporation with 
<LI>1µL of palsmid (33ng/µL)
</LI><LI>50µL of cells
</LI><LI>2000V
and sowed cells on MB 1X+Cam 1/1000 after 2hours of incubation.</LI></P><SPAN class="cd-date">July 15</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of <I>Pseudovibrio</I> with PSB1C3 - Results</FONT></BIG></B></U>
We have no colonies on the plates.</P><SPAN class="cd-date">July 13</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Transformation of <I>Pseudovibrio</I> with PSB1C3</FONT></BIG></B></U>
We made electrocompetent <I>Pseudovibrio</I> and transform them by electroporation with 
<LI>1µL of palsmid (33ng/µL)
</LI><LI>50µL of cells
</LI><LI>1800V
and sowed cells on MB 1X+Cam 1/1000 after 2hours of incubation.</LI></P><SPAN class="cd-date">July 12</SPAN></DIV></DIV><DIV class="cd-timeline-block"><DIV class="cd-timeline-content"><P align="justify"><U><B><BIG><FONT color="#003333">Curve of growth</FONT></BIG></B></U>

We try to measure a curve of grotwh by mesuring the DO (600nm) of a culture of <I>Pseuovibrio denitrificans</I> in MB 1X with a spectrometer 
</P>

Unfortunately these data looks like useless. Because of the opaqueness of the media, reliable data are not possibles to obtain.<SPAN class="cd-date">July 10</SPAN></DIV></DIV></DIV></DIV></DIV></SECTION><DIV class="row"><DIV class="col-md-4 col-sm-4 col-xs-6"><H5>Contacts</H5><UL><LI><A href="mailto:igemevry@gmail.com" target="_blank">igemevry@gmail.com</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Evry/Notebook/CellCharacterization">http://2014.igem.org/Team:Evry/Notebook/CellCharacterization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Notebook/CellCharacterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Notebook/CellCharacterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Notebook/CellCharacterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Notebook/CellCharacterization&amp;oldid=388116" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>