<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Naphthalene skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Naphthalene</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="fixed"><UL id="full_width"><LI class="seven_percent"><A href="https://2015.igem.org/Team:Uppsala">Home</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Team">Team</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Enzymes">Enzymatic Degradation</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Biosurfactants">Biosurfactants </A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Naphthalene">Naphthalene Pathway</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Design">Design</A></LI></UL><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Results">Results</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Basic_Part">Basic Parts</A></LI></NAV></P><LI><A href="https://2015.igem.org/Team:Uppsala/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Part_Collection">Part Collection</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Experiments">Experiments &amp; Protocols</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Attributions">Attributions</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI class="thirteen_percent"><A href="https://2015.igem.org/Team:Uppsala/Practices">Policies &amp; Practices</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Schools">Visiting a School</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Power_Plant">Power plant visit</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Outreach">Outreach</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Modeling">Modeling</A></LI></DIV><DIV id="main"><H1>Naphthalene pathway</H1><P>
   The Nah7-group has been working on extracting and characterizing the upper naphthalene-degrading pathway of the Nah7 plasmid from <I>Pseudomonas putida</I> G7. This will serve two purposes. First, it will break down the low-weight polycyclic aromatic hydrocarbon (PAH) naphthalene to the non-toxic product salicylate. Second, the produced salicylate will activate the PsaI/NahR promoter system, thus inducing the production of laccases and dioxygenases resulting in the degradation of heavier PAHs such as benzo(a)pyrene (BaP).
  </P><H2>Introduction</H2><P>
  Some bacteria have developed enzymatic pathways capable of degrading the toxic PAHs in order to utilize them as a carbon source. Naphthalene is one such PAH, and the PpG7 strain of <I>P. putida</I> is one such bacterium. <I>P. putida</I> is a class 1 safety level bacterium found mainly in soil. It grows optimally at room temperature and is considered non-pathogenic. The Nah7 plasmid of <I>P. putida</I> contains two pathways that together degrade naphthalene into carbon dioxide (Sota A. <I>et al</I>., 2006). The upper of these two pathways contains six enzymes that break down naphthalene to salicylate, and it is this upper pathway that we attempted to isolate and express in <I>Escherichia coli</I>. The enzymes, substrates and intermediates of the upper naphthalene degrading pathway can be found in fig.1.
  </P><FIGCAPTION><B>Figure 1</B> shows the enzymes, substrates and intermediates of the upper naphthalene degrading pathway as well as the gene giving rise to each enzyme.</FIGCAPTION><P>
  The reason for excluding the lower pathway from our construct is that we are really interested in one of the intermediates of the breakdown, salicylate, as it can be used to induce the Psal promoter (<A href="http://parts.igem.org/Part:BBa_K1031021"><SPAN class="res_link">BBa_K1031021</SPAN></A>). The NahR protein is continuously bound to the Psal promoter, and upon interaction with salicylate, transcription of the subsequent genes is induced. In our system, the gene products are a laccase, a dioxygenase and a fluorophore. In the presence of naphthalene, our pathway will produce salicylate and our laccase and dioxygenase genes will be expressed.
  </P><P>
  The sequence of interest, therefore, is an operon called the upper naphthalene pathway. The genetic sequence of the upper pathway is made up of ten genes and is approximately 8000 base pairs long and the entire Nah7 plasmid is 82,232 base pairs (Sota A. <I>et al</I>. 2006). 
  </P><H2>Methods</H2><U><P>Growing <I>Pseudomonas putida</I></P></U><P>
  The <I>P. putida</I> strain PpG7, acquired from Professor J. R. van der Meer, requires naphthalene to be supplied to the culture to ensure the bacteria does not rid itself of the plasmid. This was achieved by placing a small amount of naphthalene crystals in the lid of the agar plates of the cultures. We also constructed a contraption for growing liquid <I>P. putida</I> cultures, where naphthalene was placed in an open Eppendorf tube suspended in the E-flask above the culture. 
  </P><FIGCAPTION><B>Figure 2</B> shows the contraption used for growing <I>P.putida</I> in liquid culture. The naphthalene crystals were placed in an open eppendorf tube suspended from strips of parafilm in the E-flask.</FIGCAPTION><P>
  Naphthalene is, as mentioned, a PAH and is thus carcinogenic and teratogenic. Hence, the chemical was handled with appropriate care and safety measures. All cultures containing naphthalene were kept in fume hoods and handled with gloves. We attempted to extract the plasmids directly from the liquid cultures, as this would provide a more pure and better template for our PCRs. However, since Nah7 has a very low copy-number in <I>P. putida</I> it was not possible to achieve a high enough DNA concentration. As such, a modified version of colony PCR was used. The cells were boiled prior to PCR amplification and the denaturation and annealing times were extended.
  </P><U><P>Isolation of sequence (primer design, PCR and insertion into DH5α)</P></U><P>
  Since the pathway (<A href="http://parts.igem.org/Part:BBa_K1688016"><SPAN class="res_link">BBa_K1688016</SPAN></A>) was extracted through PCR, the primers were designed with an overhang in order to add a standardized iGEM ribosome binding site (RBS) sequence, as well as the standardized iGEM prefix and suffix (<A href="http://parts.igem.org/Part:BBa_K1688015"><SPAN class="res_link">BBa_K1688015</SPAN></A>). The prefix contains the restriction sites corresponding to the EcoRI and XbaI restriction enzymes. The suffix overhang, however, contained only one of the customary restriction sites, namely the SpeI site. This was due to the fact that the upper naphthalene pathway contains two endogenous PstI sites, preventing us from using that enzyme without digesting our sequence of interest. 
  </P><P>
  Initially we designed several sets of primers to ensure that at least one pair would prove successful. We designed one forward primer (forw. 1 in figure 3) with the iGEM prefix and RBS as described above. Moreover, we designed one forward primer (forw.2 in figure 3) with just the prefix and no RBS, with the purpose of lifting the pathway with its native promoter and RBS. The same reverse primer (rev. 1 in figure 3) was used for both of these forward primers.
  </P><P>
  Furthermore, we designed primers for lifting our sequence in two separate fragments. Thus, we also designed a reverse primer (rev. 2 in figure 3) starting in the middle of our sequence, as well as a forward primer (forw. 3 in figure 3) starting at the same point. After testing all of these primers, we could conclude that the first pair described above was the most apt.
  </P><FIGCAPTION><B>Figure 3</B> shows the upper naphthalene pathway genes and the placement of the different primers tried.</FIGCAPTION><P>
  Initially we had some trouble with the PCR amplification, as the very long primers had a tendency to form secondary structures. This was abated by adding various concentrations of DMSO and GC buffer to the mastermix.
  </P><U><P>Assembly with promoter</P></U><P>
  The PCR product was inserted into the standard iGEM plasmid backbone pSB3C5 and the ligation product was used to transform <I>E.coli</I> of the strain DH5α, which were subsequently streaked on plates containing chloramphenicol. The plasmid backbone was chosen because of its low to medium copy number, which matches the copy number of the Nah7 plasmid in <I>P.putida</I>. This is important to ensure that the level of overexpression does not interfere with important cell functions.
  </P><P>
  Successfully transformed cells were once again streaked on plates and grown overnight, followed by a plasmid extraction. The extracts were assembled with the two promoters (<A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A>)(<A href="http://parts.igem.org/Part:BBa_k1688013"><SPAN class="res_link">BBa_K1688013</SPAN></A>) and (<A href="http://parts.igem.org/Part:BBa_J23102"><SPAN class="res_link">BBa_J23101</SPAN></A>)(<A href="http://parts.igem.org/Part:BBa_K1688014"><SPAN class="res_link">BBa_K1688014</SPAN></A>) using the iGEM standard assembly protocol. 
<I>E.coli</I> of the two strains DH5α and BL21DE3 were then transformed with the ligated plasmids. 
Using BL21DE3 in addition to the standard DH5α was motivated by its prominent protein-expressing capabilities. It should hence be better at breaking down naphthalene. Throughout the process, gel electrophoresis was used to confirm the lengths of the fragments.
  </P><U><P>Mutagenesis of internal restriction sites</P></U><P>
  As mentioned previously, the upper naphthalene pathway contains two PstI-sites. These were removed through site-directed mutagenesis(<A href="http://parts.igem.org/Part:BBa_K1688018"><SPAN class="res_link">BBa_K1688018</SPAN></A>). For each respective mutation site, an overlapping and complementary primer pair containing a silent mutation was designed. Prior to the first site-directed mutagenesis PCR, the primers were phosphorylated with T4 polynucleotide kinase. All remaining methylated DNA was subsequently digested using DpnI and the plasmid was recirculated. Thereafter, the second site-directed mutagenesis PCR utilizing the second set of primers was exerted with ligated product. The PCR product was once again recirculated and transformed into competent DH5α and BL21DE3 cells.(<A href="http://parts.igem.org/Part:BBa_K1688017"><SPAN class="res_link">BBa_K1688017</SPAN></A>)
  </P><P>To view our results from this part of the project, please click <A href="https://2015.igem.org/Team:Uppsala/Results#naph"><SPAN class="res_link">here</SPAN></A>.</P><P class="letter">Dearest Michael,
A fortnight has passed since last our gazes entwined. Much has occurred but our love remains as strong as the putrid slime of the hagfish. We long to once again rest in your embrace, but before your return there is much to be unveiled.</P><P class="letter">To our great astonishment the isolation of our most desired gene-fragments has been successful, even without your most revered knowledge of the dreaded, but ever so noble, art of PCR. Furthermore these most coveted fragments have been introduced into the loving care of a more suitable, and undoubtably more competent, guardian, known to the world as “pSB1C3”.</P><P class="letter">
In order to uncover the true nature of these our most cherished fragments, we had to resort to the most cruel of methods. ‘Tis said that you hurt the ones you love the most, and in truth hurt is what we have done. We cut them… I die pronouncing it, as Shakespeare wrote, but we cut them! May the Gods forgive us this foulest of sins. These severed fragments will reveal to us the truth of their identity, as the act of cutting them has so cruelly revealed ours. Their souls were forced to wander the ever twisting valley of the shadow of Death, passing before the Crimson Fields of judgement before being sacrificed at the sacred altar of the Azure Light deep in the Everdark. 
</P><P class="letter">
The seeds that we have sown are growing strong, yet Putida does not thrive in the hostile and unloving environment of barren desolation that is the overnight flask. She has grown to maturity and has been purged of her impurities in the cleansing waters of the ceremonial preparation of the plasmids. She is pure now, perfected for our sordid purpose.  May her soul please the Dark Void videlicet our Borean box..
</P><P class="letter">
Upon your most joyous return our work shall proceed with the so dearly missed guidance of your magnificient mind. With your blessing and leadership we seek to forge the prime key that unlocks the definitive knowledge of our fragments’ nature, and to rid them of their ungodly flaws. Such affronts towards utility and beauty must not be allowed to persist in our quest for perfection, which our fellow scholars have come to name “Decyclifier.” A worthy name, for an equally worthy cause.
</P><P class="letter">
We anxiously await your arrival and count the hours until our long-awaited reunion. May your travels be swift and true and carry you safely back into our arms. Godspeed, dearest one.
</P><H2>References</H2><P>
  Masahiro Sota, Hirokazu Yano, Akira Ono, Ryo Miyazaki, Hidenori Ishii, Hiroyuki Genka,2 Eva M. Top, and Masataka Tsuda. <I>Genomic and Functional Analysis of the IncP-9 Naphthalene-Catabolic Plasmid Nah7 and Its Transposon Tn4655 Suggests Catabolic Gene Spread by a Tyrosine Recombinase. J Bacteriol.</I> 2006 Jun; 188(11): 4057–4067. doi: 10.1128/JB.00185-06
  </P></DIV></DIV></DIV></DIV></BODY></HTML>