<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Enzymes_method skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Enzymes method</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="fixed"><UL id="full_width"><LI class="seven_percent"><A href="https://2015.igem.org/Team:Uppsala">Home</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Team">Team</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Enzymes">Enzymatic Degradation</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Biosurfactants">Biosurfactants </A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Naphthalene">Naphthalene Pathway</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Design">Design</A></LI></UL><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Results">Results</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Basic_Part">Basic Parts</A></LI></NAV></P><LI><A href="https://2015.igem.org/Team:Uppsala/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Part_Collection">Part Collection</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Experiments">Experiments &amp; Protocols</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Attributions">Attributions</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI class="thirteen_percent"><A href="https://2015.igem.org/Team:Uppsala/Practices">Policies &amp; Practices</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Schools">Visiting a School</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Power_Plant">Power plant visit</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Outreach">Outreach</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Modeling">Modeling</A></LI></DIV><DIV id="main"><H1>Methods</H1><H4>RFC</H4><P>
   Restriction free cloning is a method that we used for assembling the HlyA-tag and the enzymes CueO, CotA and catechol 1,2 dioxygenase without creating a scar. Figure 1 describes the RFC-steps for fusing the HlyA-tag to the dioxygenase-gene but the principle is the same for both the CueO and CotA. We first created primers to cut the HlyA-tag gene from its plasmid by using PCR. The product from the first PCR was a linear fragment encoding the HlyA-tag-gene with overhangs on both sides. The overhang on the 5’-end was complementary to the 3’-end of the enzyme gene and the overhang on the 3’-end was complementary to the suffix on the target plasmid. With a second PCR the DNA-fragment from the first PCR  was inserted into the target plasmid between the gene and its suffix. In this secondary PCR the product from step one was used as a primer and the target plasmid with the desired enzyme was the template. The product from this PCR was a circular plasmid. We wanted to make sure that there was nothing left of the original template plasmids and therefore a digestion with DpnI FastDigest™ was done on the PCR-product. This enzyme degrades Dam methylated (GATC) DNA and therefore it degraded only the template plasmid and not the PCR product. After the DpnI had been heat inactivated the PCR-product was transformed into competent E.coli. The backbone of the target plasmid (pSB1C3) possessed chloramphenicol resistance and the transformed E.coli cells were grown on plates with chloramphenicol. Colonies on the plate would mean that the restriction free cloning was successful. Screening with colony PCR was done on all colonies to verify that the RFC products had the right length. 
For further verification that the HlyA-tag had been attached properly, the RFC products were plasmid prepared and sent for sequencing.
</P><H4>Immobilized metal affinity cromatography of ModLac</H4><P>
   An IMAC column (1 cm in diameter) was set up and connected to a pump via plastic tubes. Sephadex gel was added to the column to a sedimented height of 3 cm. The packed column was washed with approximately 8 column volumes (the volume of gel inside the column) of double-deionized water (ddH<SUB>2</SUB>O). The column was then loaded with 6 column volumes of 20 mM nickel(II)sulphate solution followed by 5 column volumes of ddH<SUB>2</SUB>O. Excess nickel ions were washed away with 5 column volumes of acidic buffer (0.02 M sodium acetate, 0.5 M NaCl; pH 4). The column was equilibrated with 5 column volumes of loading buffer.
</P><P>
The lysate solution was pumped onto the column (at approximately 1 mL/minute) and a total of three 5 mL fractions of the flow-through (denoted F1 - F3) were immediately collected until all sample had been loaded. The column was washed with 30 mL loading buffer and six 5 mL fractions (denoted F5 - F10) were collected. The protein was eluted with 40 mL elution buffer (50 mM NaH<SUB>2</SUB>PO<SUB>4</SUB>, 300 mM imidazole, 1 M NaCl; pH 7.1) and eight 5 mL fractions were collected (denoted F11 - 18). 
   </P><P>
   The protein concentration of the fractions was estimated by measuring absorbance at 280 nm. Enzyme activity measurements were performed with 2 mM ABTS (2,2'-azino-bis(3-ethylbenzothiazoline-6-sulphonic acid)) as substrate, which when oxidized turns dark green, on relevant fractions. The reaction was followed spectrophotometrically over time at 420 nm.
   </P><H4> Ion exchange column of overnight with Cathecol-1,2-dioxygenase</H4><P>
   Overnight cultures of the catechol 1,2 dioxygenase in LB-medium with chloramphenicol were used to inoculate expression cultures (4x0.5 L) in TB-buffer with chloramphenicol. The cells had grown to OD<SUB>600</SUB> around 4 when they were harvested and then they were spun down in 50 mL Falcon tubes in 4 degrees at 4000 rpm. The supernatant was discarded and the cell pellets were snap frozen in liquid nitrogen and then stored at -80°C overnight to make them fragile and easy to lyse. The cell pellets were then thawed and resuspended in a lysis buffer (50 mM K<SUB>2</SUB>HPO<SUB>4</SUB> + KH<SUB>2</SUB>PO<SUB>4</SUB>, pH 7.4) containing lysozyme, followed by lysis in sonication bath (75 W, 2 min on, 30 sec off, 4°). Lysate was centrifuged at 16800 g for 1h in 20°C. Supernatant was collected and then filtered (0.2 µM pore size) and collected in 50 mL falcon tube. The raw lysate was further purified via anion exchange chromatography with a Q-Sepharose column with the same lysis buffer (without lysozyme). Due to the catechol 1,2 dioxygenase pI of 5.21 the lysis buffer (with pH 7.4) gives the enzyme a negative net charge. The proteins were eluted with a stepwise  NaCl gradient (0-1 M, 50 mM steps) and the eluted fractions were measured for protein content with NanoDrop™ (molar extinction coefficient of 16.8, Mw=34249.12) and screened for catechol 1,2 dioxygenase with an SDS-PAGE. Raw cell lysate and protein ladder were used as references. The fractions with the highest protein concentration that had a band of the correct size on the gel were used when measuring the enzyme activity. This fraction was diluted 10 times before measuring its activity. 
   </P><H4>Protein characterization</H4><P>
   Laccases CueO and CotA as well as catechol 1,2 dioxygenase <A href="http://parts.igem.org/Part:BBa_K1092003"><SPAN class="res_link">BBa_K1092003</SPAN></A>, were assembled with a constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A> and successful transformants were restreaked on bromophenol blue agar plates (0.1 g/L) with ampicillin. The colonies were then monitored for dye degradation and halo formation to determine whether the enzymes are excreted or not.
   </P><P>
   In order to increase the amount of protein produced by the cells we transformed the constructs into competent BL21DE3 E.coli cells, instead of DH5α strain that had been used previously for gene cloning, due to the latter’s poor performance in protein expression.
   </P><P>
   Activity assays were performed on a Shimadzu UV-1800 spectrophotometer at 260 nm, where the product of catechol degradation ,cis,cis-muconic acid (molar extinction coefficient of 16.0) absorbs strongest. Serial dilutions of catechol, from 5 µM to 10 mM, were prepared. The spectrophotometer was blanked with 100 µM catechol, 100 µL of 200 µG/ml BSA stock in phosphate buffer. The samples were contained 100 µL of the fraction with the highest protein concentration and varying catechol concentrations. All tests were performed at room temperature. Data was then plotted and derivatives of the resulting curves analyzed in LoggerPro™. For Km and V<SUB>max</SUB> determination, GraphPad was used. 
   </P><P>
   Temperature activity was assayed in 100 µl reaction volumes, following the same proportions as the 1 ml reactions. Preheated thermocycler was used to control temperatures in each tube, and reaction was stopped after 90 seconds with addition of 50 µl 5M HCl. Measurement of this reaction was performed on NanoDrop™ T100 due to the small sample volume.
Activity of catechol 1,2 dioxygenase in varying pH environments was assayed in 1 ml reaction volumes, with 100 µl of enzyme and 100 µl 1 mM catechol (final concentration of 100 µM) dissolved in 800 µl of buffers of pH varying from 2 to 13. At lower pH catechol is stable, but at highly alkaline conditions it spontaneously degrades, obscuring results of the experiment. Hence no data is presented for pH higher than 10. Controls were prepared for each pH separately, analogous to previous experiments. 
   </P></DIV></DIV></DIV></DIV></BODY></HTML>