<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Enzymes skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Enzymes</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="fixed"><UL id="full_width"><LI class="seven_percent"><A href="https://2015.igem.org/Team:Uppsala">Home</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Team">Team</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Enzymes">Enzymatic Degradation</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Biosurfactants">Biosurfactants </A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Naphthalene">Naphthalene Pathway</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Design">Design</A></LI></UL><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Results">Results</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Basic_Part">Basic Parts</A></LI></NAV></P><LI><A href="https://2015.igem.org/Team:Uppsala/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Part_Collection">Part Collection</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Experiments">Experiments &amp; Protocols</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Attributions">Attributions</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI class="thirteen_percent"><A href="https://2015.igem.org/Team:Uppsala/Practices">Policies &amp; Practices</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Schools">Visiting a School</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Power_Plant">Power plant visit</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Outreach">Outreach</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Modeling">Modeling</A></LI></DIV><DIV id="main"><H1>Enzymatic degradation</H1><P>
The aim of this part of the Decyclifier system is to degrade the polycyclic aromatic hydrocarbons, PAHs, into less toxic compounds using the degrading enzymes laccase and dioxygenase. We also want to include an on and off switch to the degradation pathway so that we do not stress the cells when the enzymes are not needed. The switch is connected to a fluorescent reporter gene which will indicate when PAHs are present and hence when the construct is active. 
  </P><H2>Introduction</H2><H4>Biosensor</H4><P>
  Overexpressing enzymes is very expensive for cells when it comes to energy, as well as resources. Therefore, regulated expression is often preferred. A previous iGEM team, Peking 2013, inspired us to use the NahR construct (<A href="http://parts.igem.org/Part:BBa_J61051"><SPAN class="res_link">BBa_J61051</SPAN></A>), a system consisting of two promoters and a protein coding sequence. One of the promoters, the Pr promoter regulates the upstream sequence of the construct, which also contains the coding sequence for the NahR protein. This protein functions as a repressor at the other promoter in this construct, the Psal promoter. Contrary to the Pr promoter, the Psal promoter regulates transcription of genes downstream of the construct. However, this repression can be lifted. When salicylate binds to the NahR protein, the conformational change it undergoes makes transcription of the sequence upstream the Psal promoter possible. (<A href="https://2013.igem.org/Team:Peking"><SPAN class="res_link">iGEM Peking. 2013. Aromatic scouts.</SPAN></A>)
  </P><H4>Reporter system</H4><P>
  One of the main parts of our whole system is the visual response of PAH degradation. When these compounds are being degraded by the expressed laccase and dioxygenase we want to have an indication that our system is working; one that could easily be detected. Candidates for reporter genes were both genes expressing fluorescent proteins and chromoproteins. The choice fell on fluorescent reporter genes, since measurement and quantification of the expression would be easier and less time-consuming. 
  </P><H4>Fluorometer</H4><P>
  We faced the need to characterize a promoter system inhibited by the NahR protein, which is  induced by salicylate. We decided to test the expression of the NahR/Psal promoter system with different salicylate concentrations. Among the existing approaches for measuring the strength of a promoter, we chose to use fluorometry with the RFP dTomato as a reporter gene. A fluorometer was not available to us at the time, so we decided to create our own cheap fluorometer. You can read more about our fluorometer <A href="https://2015.igem.org/Team:Uppsala/Software"><SPAN class="res_link">here</SPAN></A>. 
  </P><H4>Degradation</H4><P>
  We chose to use two different types of enzymes for degrading polycyclic aromatic hydrocarbons; laccases and dioxygenases. Laccases (originally from Chinese lacquer tree sap) are multicopper oxidases, that are employed in various industries, where they take part in beer maturation, textile dyeing, and enzymatic biofuel cells. Due to their broad specificity and ability to oxidize aromatic compounds, their application in bioremediation is a topic under investigation. The laccases we chose were three bacterial laccases: CueO - a laccase from <I>E. coli</I>, CotA - a laccase from Bacillus pumilus, and also a modified version of CueO.
  </P><P>
  We acquired the sequence for a modified CueO from Kunishige Kataoka. The modified laccase is a multicopper oxidase with a double mutation where aspartic acid has been changed with alanine and methionine with leucine (D439A/M510L). The mutation has been proved to have more efficiency to degrade ABTS than the unmodified one. We choose to work with D439A/M510L due to its efficiency and the supposed impact it would have on our constructs degradation time (Kataoka, K et al. 2012).
  </P><P>
  Dioxygenase is a type of enzyme that uses dioxygen to catalyze the oxidation of hydrocarbons. This type of enzyme is used to catalyze cleavage of the aromatic rings in our degradation pathway. We chose to use a catechol dioxygenase which is an important part of bacterial degradation of aromatic compounds. There are two main types of catechol dioxygenase; intradiol and extradiol dioxygenase. The end product in the intradiol degradation pathway is a carboxylic acid which is unharmful to the cell and can be used in the bacteria's metabolism. The end product in the extradiol degradation pathway is an aldehyde which is much more harmful to the cell than a carboxylic acid (Bugg, 1 September 2003). Therefore an intradiol dioxygenase; catechol 1,2 dioxygenase, was chosen. This enzyme had been used by a previous iGEM team; <A href="https://2013.igem.org/Team:Hong_Kong_CUHK/backgroundPAH"><SPAN class="res_link">Hong Kong 2013</SPAN></A> but had not been characterized (<A href="http://parts.igem.org/Part:BBa_K1092003"><SPAN class="res_link">BBa_K1092003</SPAN></A>). This specific isoform of dioxygenase is suitable for extracellular export, since most known dioxygenases are dimers, stabilized by noncovalent interactions, and the current catechol 1,2 dioxygenase is a monomer. Inferring from the crystal structure of the enzyme, both its C and N termini do not participate in critical interactions within the enzyme, which makes them suitable sites for export tag fusion. 
  </P><FIGCAPTION><B>Figure 1</B>: Crystal structure of catechol-1,2-dioxygenase. </FIGCAPTION><H4>Secretion system</H4><P>
  We also incorporated an export-tag to our enzymes. This enables extracellular degradation of PAH and therefore protects the cells from the toxic PAHs and its metabolites. We are using the secretion tag HlyA (<A href="http://parts.igem.org/Part:BBa_K554002"><SPAN class="res_link">BBa_K554002</SPAN></A>) from the α-hemolysin system, originated from Staphylococcus aureus. It’s a 60 amino acid long sequence attached to the C-terminal of our enzymes. This sequence is recognized by the Type 1 secretion system (<A href="http://parts.igem.org/Part:BBa_K1166002"><SPAN class="res_link">BBa_K1166002</SPAN></A>) .
  </P><H2>System design</H2><P>
  We designed our construct in order to have a regulated expression of PAH-degrading enzymes with an on and off switch. We chose a sensor that could react to salicylate when present; the production of our degrading enzymes would start in combination with a modified red fluorescent protein, dTomato, that indicates that there are PAHs present. We found one interesting repressor system known as the NahR-Psal-promoter which is a promoter that is induced by compounds similar to salicylic acid. Salicylic acid is a compound with a single aromatic ring and is a metabolite of naphthalene degradation. The enzymes to convert naphthalene to salicylic acid is incorporated in a separate construct plasmid (you can read about this part of the project <A href="https://2015.igem.org/Team:Uppsala/Naphthalene"><SPAN class="res_link">here</SPAN></A>). Because of the copresence of naphthalene and other PAHs in the environment, this is a suitable system design.
  </P><P>
  When induced the expression of the red fluorescent protein, dTomato, starts and the degradation part of the construct gets expressed. The laccase and dioxygenase gets translated and transported to extracellular space thanks to the HlyA-tag (See secretion system). The laccase oxidizes the PAH and dioxygenase catalyzes the ring cleavage.
  </P><H4>Modification of CueO D439A/M510L</H4><P>
  The modified CueO was synthesized with iGEM prefix and suffix, RBS (<A href=" http://parts.igem.org/Part:BBa_B0034"><SPAN class="res_link">BBa_B0034</SPAN></A>) and a M-scar was added after the RBS in order to guarantee the function of the RBS. Two Nsil restriction sites were put before and after a 6xHis-tag so that the enzyme could be purified with an IMAC and so that the tag could be removed. The gene is followed by a KpnI restriction site with glycines and serines on each side, which are small amino acids which increase the flexibility of the protein. This is needed for the export tag HlyA , to make it easier for the resulting signal protein to be folded properly. A second KpnI restriction site is put after the HlyA so that the export tag can be removed easily. It has then two stop codons, and a terminator at the end.
  </P><H4>Attaching the secretion tag</H4><P>
  An issue when combining two biobricks with standard 3A Assembly is that a “scar” containing a stop codon is created between them. When attaching a signal sequence to a gene encoding an enzyme there can not be stop codons in between because the two genes need to be expressed together as one polypeptide chain. To assemble the HlyA-tag with the enzymes without getting a scar, two different methods were tried. These were overlap extension PCR and restriction free cloning (<A href="https://2015.igem.org/Team:Uppsala/Experiments"><SPAN class="res_link">RFC</SPAN></A>). The aim was to attach the HlyA-tag to the biobricks encoding the two laccases CotA and CueO as well as to the catechol-1,2-dioxygenase. 
  </P><H4>Secretion system and characterization</H4><P>
  It was uncertain whether the cells had the secretion system enzymes that would enable enzymes with a HlyA tag to be transported through the cell membranes. Therefore a cassette containing the genes encoding the secretion system enzymes was ligated to each biobrick. This construct was intended to be compared with a construct without the secretion system to see if any of them had exported proteins. The final step in our project was to get the cells to express the enzymes of interest to be able to characterize them. The enzymes were purified and measured for activity using enzyme kinetics.
  </P><H2>Method</H2><P>The methods that we used for this part of the project, we thus chose to describe them on a separate <A href="https://2015.igem.org/Team:Uppsala/Enzymes_method"><SPAN class="res_link">page</SPAN></A>.</P><P>To view our results from this part of the project, please click <A href="https://2015.igem.org/Team:Uppsala/Results#enz"><SPAN class="res_link">here</SPAN></A>.</P></DIV></DIV></DIV></DIV></BODY></HTML>