<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="fixed"><UL id="full_width"><LI class="seven_percent"><A href="https://2015.igem.org/Team:Uppsala">Home</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Team">Team</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Enzymes">Enzymatic Degradation</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Biosurfactants">Biosurfactants </A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Naphthalene">Naphthalene Pathway</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Design">Design</A></LI></UL><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Results">Results</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Basic_Part">Basic Parts</A></LI></NAV></P><LI><A href="https://2015.igem.org/Team:Uppsala/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Part_Collection">Part Collection</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Experiments">Experiments &amp; Protocols</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Attributions">Attributions</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI class="thirteen_percent"><A href="https://2015.igem.org/Team:Uppsala/Practices">Policies &amp; Practices</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Schools">Visiting a School</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Power_Plant">Power plant visit</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Outreach">Outreach</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Modeling">Modeling</A></LI></DIV><DIV id="main"><H1>Notebook</H1><H4>Week 1: (8-14 June)</H4><P>Primers were designed for PCR extraction of the naphthalene upper pathway of the Nah7 plasmid from Pseudomonas putida, and to remove T7-promoters from CueO, CotA and catechol 1,2-dioxygenase biobricks.  We also designed primers to do overlap extension PCR to remove the scar that was created when attaching the HlyA-tag with the CueO, CotA and catechol 1,2-dioxygenase genes.</P><UL><LI><P>
Received the ModLac sequence from professor Kataoka.
</P></LI></UL><UL><LI><P>
Stock solutions and agar plates with and without antibiotic resistance were made.
</P></LI></UL><H4>Week 2: (15-21 June)</H4><P>Primers were designed for site directed mutagenesis to eliminate the restrictions sites for improving the NoKoGen biobricks, and primers were also designed for restriction free cloning to assemble the HlyA-tag with the CueO, CotA and dioxygenase genes, without creating a scar. </P><UL><LI><P>
rhlA and rhlB genes from the DNA distribution kit were transformed into E.coli DH5α and plasmid preparations were made.
</P></LI></UL><H4>Week 3: (22-28 June)</H4><P>Transformations were done to insert biobricks with the enzymes CueO, CotA, catechol 1,2-dioxygenase and the HlyA-tag genes into DH5α. Frozen stock and plasmid preparations were made of these.</P><UL><LI><P>
Assembled the HlyA-tag with CueO, CotA and dioxygenase with 3A assembly. 
</P></LI></UL><UL><LI><P>
Transformed the NahR construct, dTomato and super yellow fluorescent protein 2 (SYFP2), as well as plasmid prepared the transformed DNA and evaluated with PCR followed by further evaluation by agarose gel electrophoresis.
</P></LI></UL><UL><LI><P>
Designed primers for sequencing of the Nah7 pathway, as well as primers flanking the entire pathway. </P></LI></UL><UL><LI><P>
Made an attempt to extract the Nah7 plasmid using the designed primers, including gel electrophoresis of the PCR product. The gel showed that the PCR had not worked.</P></LI></UL><UL><LI><P>
Since the NoKoGen biobricks had already been improved, transformation were done with the improved biobricks of rhlA, rhlB, and with RBS and the (<A href="http://parts.igem.org/Part:BBa_J23101"><SPAN class="res_link">BBa_J23101</SPAN></A>) promoter from the kit into DH5α. After that plasmid preparation were done. This as a preparation for building the biosurfactant construct.</P></LI></UL><H4>Week 4: (29-5 June/July)</H4><P>
A first attempt to remove the T7-promoter from the biobricks with CueO, CotA and catechol 1,2-dioxygenase was done. The products were run on a gel and this showed that the PCR was unsuccessful.</P><UL><LI><P>
A transformation on the ligations with CueO, CotA and catechol 1,2 dioxygenase with the HlyA-tag gene were transformed but with no results. The assemblies were redone, but a colony PCR indicated that the assembly was unsuccessful.
</P></LI></UL><UL><LI><P>
Assembled NahR with the reporter genes received from Erik Gullberg.
 </P></LI></UL><UL><LI><P>
Assembling rhlA, rhlB, with RBS and the (<A href="http://parts.igem.org/Part:BBa_J23101"><SPAN class="res_link">BBa_J23101</SPAN></A>) promoter.
</P></LI></UL><UL><LI><P>
Assembled plasmids with chromoproteins to plasmid backbones and then transformed them into DH5α. Made bactoart using the transformed bacteria.</P></LI></UL><H4>Week 5: (6-12 July)</H4><P>
The HlyA-tag we have been working with showed to be something entirely different, due to stabbing the kit wrong. A new transformation were done with the right HlyA-tag biobrick into DH5α cells.
</P><UL><LI><P>
Different PCR methods e.g. touchdown protocol, gradients and adding DMSO was tried to remove the T7-promoter from the CueO, CotA and catechol 1,2-dioxygenase biobricks. The results were good for CueO and CotA.
</P></LI></UL><UL><LI><P>
Made liquid overnight cultures of the strains containing NahR and dTomato with different salicylate concentrations. 
</P></LI></UL><UL><LI><P>
Continuing to assemble the rhlA, rhlB with RBS and (<A href="http://parts.igem.org/Part:BBa_J23101"><SPAN class="res_link">BBa_J23101</SPAN></A>) promoter.
 </P></LI></UL><H4>Week 6: (13-19 July)</H4><P>
The first step in the restriction free cloning was done with the genes for the enzymes CueO, CotA and catechol 1,2-dioxygenase with the HlyA-tag. The products were loaded on a 2% gel and extracted to be used in the next step.
</P><UL><LI><P>
We did a new 3A assembly with the correct HlyA-tag and our enzymes CueO, CotA and catechol 1,2-dioxygenase. We tried to remove the T7-promoter from the ligations from this correct assembly. This did not work.
</P></LI></UL><UL><LI><P>
The modified laccase arrived. It was transformed into DH5α and the cultures were used for plasmid preparation to extract the DNA.
 </P></LI></UL><UL><LI><P> 
Transformed our full construct with (<A href="http://parts.igem.org/Part:BBa_J23101"><SPAN class="res_link">BBa_J23101</SPAN></A>) promoter, RBS, rhlA and rhlB into DH5α cells. The characterisation with CTAB-plate test started. No halos were shown.
</P></LI></UL><UL><LI><P> 
Digested the naphthalene degrading pathway, ran the product on an electrophoresis gel and extracted it from the gel. Ligated the gel extract to the two different promoters and used the product to transform into DH5α. 
</P></LI></UL><H4>Week 7: (20-26 July)</H4><P>
We did the second PCR in the restriction free cloning with the CueO, CotA and catechol 1,2-dioxygenase with the HlyA-tag gene. The products from this PCR were transformed into DH5α but we only got growth on the plates with CotA+HlyA-tag. Therefore the first PCR of the RFC was redone.
</P><UL><LI><P>
The HlyA-tag was attached to the two laccases CueO and CotA that we had successfully removed the T7-promoter from with 3A assembly. The ligations were transformed into cells that did not grew. 
</P></LI></UL><UL><LI><P>
Assembled the NahR and reporter gene system with the modified laccase gene. The reporter gene sequences and the modified laccase sequence were also assembled with two different Anderson promoters, (<A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A>) and (<A href="http://parts.igem.org/Part:BBa_J23101"><SPAN class="res_link">BBa_J23101</SPAN></A>).
</P></LI></UL><UL><LI><P>
We sent the parts of the rhamnolipid construct for sanger sequencing.
 </P></LI></UL><UL><LI><P> 
Grew bacteria containing Phusion and Pfu overnight in liquid medium. Phusion and Pfu was extracted from bacteria.
</P></LI></UL><UL><LI><P> 
Removed two PstI restriction sites from the naphthalene pathway through site-directed mutagenesis.
</P></LI></UL><H4>Week 8: (27-2 July/August)</H4><P>
The restriction free cloning for CueO and catechol 1,2-dioxygenase genes with the HlyA-tag was successfully executed. Screening with colony PCR showed satisfactory results. Since we didn’t get any growth with CotA and HlyA-tag we decided to abandon the CotA enzyme.
</P><UL><LI><P>
Ligation of the biobricks CueO and catechol 1,2-dioxygenase with a standard promoter (<A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A>) were preformed.
</P></LI></UL><UL><LI><P>
Performed digestion with KpnI restriction enzyme on the modified laccase sequence to remove the export HlyA-tag sequence. The digested sequence was run on an agarose gel followed by an extraction of the gene sequence.
 </P></LI></UL><UL><LI><P> 
Preparation of bromophenol blue plates began, to check the presence of extracellular laccase.
</P></LI></UL><UL><LI><P> 
The rhamnolipid construct were tested with drop collapse test along with a CTAB test wich showed presence of rhamnolipids. 
</P></LI></UL><UL><LI><P>
Assembled dTomato construct at the end of our full rhamnolipid construct by 3A assembly.
</P></LI></UL><UL><LI><P>
Prepared BL21DE3 competent cells.
</P></LI></UL><H4>Week 9: (3-9 August)</H4><P>
The CueO, catechol 1,2-dioxygenase and the modified laccase genes was assembled with a secretion system. 
</P><UL><LI><P>
Large overnight on the dioxygenase and the HlyA-tag with the J23110 promoter was made.
</P></LI></UL><UL><LI><P>
The biobrick with dioxygenase was transformed into BL21DE3 cells.
</P></LI></UL><UL><LI><P>
We started assembling on of the laccase genes, CueO with the NahR and reporter gene system, while also continuing the assembly of the final enzymatic degradation construct by adding the catechol 1,2-dioxygenase after the modified laccase gene.
 </P></LI></UL><UL><LI><P> 
Performed a BPB plate test on the modified laccase and CueO.
</P></LI></UL><UL><LI><P> 
Transformed the modified laccase gene into BL21DE3 cells.
</P></LI></UL><UL><LI><P>
We transformed our rhamnolipid construct in BL21D3 cells due to poor expression of mono-rhamnolipids in DH5α.
</P></LI></UL><UL><LI><P>
Performed TLC test on the expressions from the rhamnolipid construct. It indicated presence of  rhamnolipids.
</P></LI></UL><UL><LI><P>
Analyzed differences in survivability between cells containing the naphthalene degrading pathway and negative control cells containing an RFP-coding gene by adding fixed amounts of naphthalene crystals in the lid of the agar plates.
</P></LI></UL><H4>Week 10: (10-16 August)</H4><P>
The RFC products with CueO containing the HlyA-tag and catechol 1,2-dioxygenase with the HlyA-tag were sent for sequencing. The sequence showed a frameshift where the HlyA-tag were inserted. 
</P><UL><LI><P>
The CueO and the modified laccase genes were transformed into BL21DE3 cells. Big overnights of BL21DE3 cells with catechol 1,2-dioxygenase, CueO and the modified laccase were made.
</P></LI></UL><UL><LI><P>
Extraction and purification of the catechol 1,2-dioxygenase, CueO and the modified laccase.
</P></LI></UL><UL><LI><P>
The rhamnolipid constructs was in pSB3K3, so they were transferred to pSB1C3 backbone as a biobrick requirement. Assembly with dTomato was tried with construct in pSB1C3 plasmid which gave positive result. </P></LI></UL><UL><LI><P> 
More drop collapse tests were performed with the expressions from the rhamnolipid construct.
</P></LI></UL><UL><LI><P>
Analyzed the presence of salicylate, both directly in culture with cells assumed to have the naphthalene pathway and in its supernatant. The tested cells and its supernatants came from the transformation cultures of DH5α and BL21DE3 with promoters (<A href="http://parts.igem.org/Part:BBa_J23101"><SPAN class="res_link">BBa_J23101</SPAN></A>) and (<A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A>) as well as a DH5α with a simple RFP insert as a negative control.
</P></LI></UL><H4>Week 11: (17-23 August)</H4><P>
An anion-exchange chromatography was done to extract catechol 1,2-dioxygenase from cell lysate. The fractions eluted from the chromatography were run on an SDS-PAGE which showed that the dioxygenase was overexpressed in the cells and the concentration was much higher in some of the fractions.
</P><UL><LI><P>
Tested the enzyme kinetics and the enzymes activity for the catechol 1,2-dioxygenase at different temperatures and pH. 
</P></LI></UL><UL><LI><P>
SDS-PAGE was performed on the IMAC fractions of the modified laccase and CueO.
</P></LI></UL><UL><LI><P>
The modified laccase gene without the export tag was also transformed into BL21DE3.
 </P></LI></UL><UL><LI><P> 
Kinetic measurements of CueO and the modified laccase.
</P></LI></UL><UL><LI><P> 
All rhamnolipid parts were sent for sequencing.
</P></LI></UL><UL><LI><P>
Analysed the growth of our cells with the naphthalene degrading pathway with no naphthalene present, naphthalene present directly in cell culture and gaseous naphthalene presence.
</P></LI></UL><UL><LI><P>
These pictures shows liquid culture containing naphthalene degrading pathway or red insert, where there is either no naphthalene present, naphthalene present directly in culture or gaseous naphthalene present.
</P></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>