<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Results_enz skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Results enz</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="fixed"><UL id="full_width"><LI class="seven_percent"><A href="https://2015.igem.org/Team:Uppsala">Home</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Team">Team</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Description">Project</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Enzymes">Enzymatic Degradation</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Biosurfactants">Biosurfactants </A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Naphthalene">Naphthalene Pathway</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Design">Design</A></LI></UL><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Results">Results</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Basic_Part">Basic Parts</A></LI></NAV></P><LI><A href="https://2015.igem.org/Team:Uppsala/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Part_Collection">Part Collection</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Experiments">Experiments &amp; Protocols</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Attributions">Attributions</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI class="thirteen_percent"><A href="https://2015.igem.org/Team:Uppsala/Practices">Policies &amp; Practices</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Schools">Visiting a School</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Power_Plant">Power plant visit</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Uppsala/Outreach">Outreach</A></LI><LI class="ten_percent"><A href="https://2015.igem.org/Team:Uppsala/Modeling">Modeling</A></LI></DIV><DIV id="main"><H2 id="enz_deg">Enzymatic degradation</H2><U><B><P>Restriction free cloning (RFC)</P></B></U><P>
  The first PCR-step in the RFC was run with different concentrations of DMSO to inquire into what concentration would be optimal. The results of this PCR is shown in Figure 1.
  </P><FIGCAPTION><B>Figure 1</B>: Agarose gel with the PCR products from the first test PCR with different DMSO concentrations. The first 4 wells in the upper left corner has 0% DMSO, the 4 wells in the upper right corner has 3% DMSO, the 4 wells in the lower left corner has 5% DMSO and the last 4 wells has 10% DMSO. The upper bands are the PCR products and the lower are primer dimers. The gel shows that the PCR products are 250 bp long, which is the correct length. The products with 5% DMSO have the sharpest bands on the gel.</FIGCAPTION><P>
  The results from the previous PCR showed that 5% DMSO was the optimal concentration for our primers and therefore a new PCR was run under the same conditions as previously: This was to yield more product that could be used in the next step of the PCR. The results from this PCR is shown in Figure 2.
  </P><FIGCAPTION><B>Figure 2</B>: 2% agarose gel with PCR product (primers) from the first PCR done with 5% DMSO.  Wells 11 to 20 contain primers for catechol 1,2 dioxygenase and CueO. These have the correct length; around 250 bp. The other wells contain negative controls. The upper bands show primer dimers.</FIGCAPTION><P>
  After the second PCR the PCR products (pSB1C3 backbones with the laccases CotA and CoeO with inserted HlyA-tag as well as catechol 1,2 dioxygenase with inserted HlyA-tag) were transformed into competent DH5α after digestion of methylated plasmids with Dpn1. After incubation of the transformed cells on agar plates with chloramphenicol overnight, colonies had appeared on the plates with dioxygenase and CueO. The RFC did however not seem to work for the CotA. The colonies were screened using colony PCR and the result from this PCR is shown in Figure 3. The results shown in this figure indicates that the RFC was successful for both the catechol 1,2 dioxygenase and the CueO.
  </P><FIGCAPTION><B>Figure 3</B>: Agarose gel with PCR product from colony PCR of RFC product from the second PCR. Well 5 contains dioxygenase without the HlyA-tag attached and well 4 contains the RFC product which should have the HlyA-tag attached. The band in well 5 has moved further than the band in well 4 which shows that the PCR product in well 4 is longer than the product in well 5. Well 2 and 3 contains CueO with and without the HlyA-tag respectively. There is a difference in length between these two bands as well and this shows that the RFC was successful also for CueO.</FIGCAPTION><P>
  Samples were sent for Sanger sequencing to SciLifeLab in Uppsala to confirm correct assembly of the HlyA-tag. Results showed that a 2 base frameshift in the coding sequence was introduced due to faulty primer design. This resulted in incorrect expression of the export tag, which made the tag useless. 
  </P><P><B><U>Purification and characterization of catechol-1,2-dioxygenase</U></B></P><P>
  Conditions of cell lysis proved effective, with 100% of cells lysed as estimated by visual inspection. After centrifugation and filtration, the crude cell extract showed 12.47±2.31 mg/ml, as measured by NanoDrop at 280 nm. The total volume of the cell extract was around 60 ml.
  </P><FIGCAPTION><B>Figure 4</B>: SDS PAGE of elution fractions. 20 µl of each fraction (and BSA stock), and 10 µl ladder were run on 10% acrylamide gel to confirm protein presence in the fraction. L: PageRuler Unstained Protein Ladder; B: BSA, 200µg/ml; C: crude cell extract; 0-1000: Different fractions as indicated of millimoles of NaCl present. Image is composite of two gel pictures and is digitally enhanced for contrast and clarity with demonstrative purpose. </FIGCAPTION><P>
  Collected fractions of each salt concentration (0 to 500 mM NaCl in 50 mM steps) were then ran on an SDS PAGE gel to confirm presence of the catechol 1,2 dioxygenase enzyme. The 
results, shown in Figure 4, confirmed that the protein is being successfully expressed in large quantities and is of the expected size. The peak of the protein was spread between the 300 mM and 350 mM step. Each fraction was then measured for protein concentration and the results for 300 and 350 fractions were 879 µg/ml, and 1161 µg/ml respectively.
  </P><P><FIGCAPTION><B>Figure 5</B>: Initial velocity versus substrate concentration plot. Vmax = 32,64 µM/min, and Km = 16,85 µM-1. Individual data points not shown.</FIGCAPTION></P><FIGCAPTION><B>Figure 6</B>: Enzymatic activity depending on temperature. Catechol exhibits highest activity at 40 °C, making 50,625 µM of product in 1 minute, and very low amounts of product at temperatures above 55 °C or below 15 °C. </FIGCAPTION><P>
  Results of the enzymatic assay measurements are shown on Figure 5. Determined values were V<SUB>max</SUB>=32.64 µM.min<SUP>-1</SUP>, and K<SUB>M</SUB>=16.85 µM. The enzyme showed highest velocity at 150 µM concentration of catechol, but for the rest of the tests, 100 µM was used instead. The enzyme exhibits strong substrate inhibition above 400 µM substrate concentration, with 10mM completely inhibiting enzyme activity (data not shown in Figure 5).
Graphical representation of the results from the measurement of activity depending on temperature is shown in Figure 6.
Results obtained from the assay of enzyme activity at different pH are presented on Figure 7. 
  </P><FIGCAPTION><B>Figure 7</B>: pH optimum of catechol-1,2-dioxygenase. The enzyme shows a pH optimum in the neutral range, with slight bias towards alkaline pH.</FIGCAPTION><P>
  Maximum velocity of 34,7875 µM/min was obtained at pH 8,16 and 22°C, with very similar results for neutral pH as well (33,9 µM/min at pH 7,17). pH lower than 5 completely inhibits enzyme activity, as well as pH&gt;9. 
  </P><U><B><P>Bromophenol blue test</P></B></U><P>
  A composite picture of 4 simultaneous bromophenol blue tests is shown on Figure 8. A slight difference in color can be seen between the negative control (1) with dioxygenase and the colonies with laccases (CotA (3) and CueO (4)). This indicates that the laccases are able to degrade the bromophenol blue which is a positive result. (When performing this experiment the frameshift in the HlyA-tag had not yet been discovered and therefore the result in section number 2 is not relevant.)
  </P><FIGCAPTION><B>Figure 8</B>: Bromophenol blue assay. This composite image contains 4 separate plates, restreaked with a single colony of dioxygenase with <A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A> promoter (1, negative control), CueO with <A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A> promoter and Hlya tag (2) CotA with <A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A> promoter (3) and CueO with <A href="http://parts.igem.org/Part:BBa_J23110"><SPAN class="res_link">BBa_J23110</SPAN></A> promoter (4)</FIGCAPTION><U><B><P>Removal of the HlyA-tag from ModLac</P></B></U><P>
  To ensure that the fused HlyA tag did not affect the enzymatic activity of the laccase, the HlyA tag was removed before characterization of the modified laccase. This was done by digesting with the restriction enzyme Kpn1, ligating it together and PCR amplifying the segments with VF2 and VR primers. This resulted in two main segments: the ModLac without tag and the ModLac re-ligated with the HlyA tag. To separate these two segments, the PCR product was run on an agarose gel, and the shorter segment was extracted from the gel. 
  </P><U><B><P>Purification of ModLac and CueO (Ecol)</P></B></U><P>
  The resulting chromatograms of the IMAC purifications of ModLac and CueO (Ecol) can be seen below in Figure 9 and 10.
  </P><FIGCAPTION><B>Figure 9</B>: Chromatograms of all IMAC fractions of ModLac. All fractions can be seen in the left chromatogram while the last 15 fractions can be seen in the right. A big peak in absorbance is visible around fraction No. 2 while only small peaks can be seen in the left picture.</FIGCAPTION><FIGCAPTION><B>Figure 10</B>: Chromatograms of all IMAC fractions of CueO(Ecol). All fractions can be seen in the left chromatogram while the last 15 fractions can be seen in the right. A big peak in absorbance is visible around fraction No. 2 while only minor peaks can be seen in the left picture.</FIGCAPTION><U><B><P>ModLac characterization</P></B></U><P>
  The breakdown of ABTS was measured at 420 nm using a spectrophotometer. The stable oxidized form of ABTS absorbs at 420 nm. ModLac laccase was concluded to have a higher enzymatic activity than the CueO laccase. 
</P><FIGCAPTION><B>Figure 11</B>: The degradation of ABTS by the lysate of ModLac and the lysate of CueO over time.</FIGCAPTION><FIGCAPTION><B>Figure 12</B>: SDS-PAGE results of the IMAC fractions of the modified laccase (left) and CueO (right)</FIGCAPTION><FIGCAPTION><B>Figure 13</B>: shows 9 fractions from Ion exchange column; one band of 20-30kDa. </FIGCAPTION><U><B><P>NahR characterization</P></B></U><P>
  The regulative capabilities of the NahR promoter system was tested under different salicylate concentrations. dTomato was attached after the NahR biobrick (<A href="http://parts.igem.org/Part:BBa_J61051"><SPAN class="res_link">BBa_J61051</SPAN></A>) as a reporter gene, and the fluorescence was measured using our <A href="https://2015.igem.org/Team:Uppsala/Software"><SPAN class="res_link">fluorometer prototype</SPAN></A>.
  </P><FIGCAPTION><B>Figure 14</B>:NahR/Psal induction test in liquid culture. Expression of dTomato under the NahR/Psal promoter system with different salicylate concentrations. </FIGCAPTION><FIGCAPTION><B>Figure 15</B>: NahR/Psal induction test on agar plate. Concentration of salicylate from left to right: 0,1 μM, 1 μM, 10 μM, 100 μM and 1 mM. </FIGCAPTION><U><B><H3>Conclusions:</H3></B></U><U><B><P>Restriction Free Cloning</P></B></U><P>
   The result of the first test PCR on dioxygenase, CotA and CueO, displayed in figure 1, showed that a 5% concentration of DMSO gave the best results for this PCR. This could be seen on the bands on the gel, which were thicker when 5% DMSO had been used. The results displayed in figure 2 showed that the part to be inserted had the correct length. This indicated that the insert contained the right gene with the overhangs, which is important for the next PCR to be successful. When the RFC products were transformed into the cells the fact that there were colonies on the plates confirmed that the RFC was successful. The cells would not have survived if they did not have a plasmid with chloramphenicol resistance because they were grown on agar plates with chloramphenicol. Because the original plasmids are methylated from being amplified in <I>E.coli</I> cells they can be degraded with Dpn1 and theoretically the only plasmids that are left should be our wanted plasmids. This means that the PCR product from the first PCR containing the HlyA-tag worked successfully as primers in the second PCR and hence since the HlyA-tag is the main part of the primer, it should have been inserted. According to the results in figure 3 the HlyA-tag was successfully attached to the dioxygenase and CueO since the RFC products were around 200 bp longer than the original genes.The RFC products did however encode defective export tags due to faulty primers. No extensive experiments to test the effect of the faulty enzyme were done.  
   </P><U><B><P>Characterization of catechol 1,2 dioxygenase</P></B></U><P>
  Catechol 1,2 dioxygenase was successfully expressed and extracted in large quantities, while preserving its activity, hence proving <I>E.coli</I> as a potent host for its production. Since a major point of the project is to export the degrading enzymes out of the cell, the primary focus was on pH and temperature stability, as well as Km to find its optimal working conditions.  The dioxygenase showed good stability in a wide temperature range, losing its activity completely only in extreme conditions, as displayed in figure 6. The enzyme had its highest residual catalytic functions at temperatures around 40 °C, which is beneficial for our purposes to create a bioreactor since the enzyme works in the same temperatures as the cells will grow. It’s pH range was narrow, showing a peak around pH 7, see figure 7. These results, showing that the enzyme will be able to work optimally in the same pH as our cells, was also seen as good results considering our bioreactor.
Dioxygenase showed very high substrate specificity and quickly approached Vmax with increase in catechol concentrations, shown in figure 5. This is favorable for extracellular conditions, where any substrate produced by laccases is diluted several orders of magnitude.
  </P><U><B><P>BPB-Test</P></B></U><P>
    The bromophenol blue plate test with the CueO and CotA showed clear regions of a brighter colour in the agar around the cells that contained PAH-degrading enzymes, thus this indicated the presence of laccases. There was no significant difference between the CueO with or without the HlyA-tag. On our negative controls, less bromophenol blue had been degraded than on the parts on the plate containing CueO and CotA. This indicated that it actually was our enzymes that were responsible for breaking down the bromophenol (see figure 8). The results of the bromophenol test with liquid cultures was however hard to assess, since we did not have any positive controls to compare with. 
   </P><U><B><P>NahR/Psal</P></B></U><P>
   We initially tested the NahR/Psal promoter system with dTomato on a pSB1C3 plasmid. The cells containing the plasmid were streaked on agar plates with different salicylate concentrations in the agar medium. Similarly, we also examined liquid cultures with different concentrations of salicylate in the solution. Both of these tests showed that the bacteria were producing the dTomato protein in presence of salicylate, independent of the salicylate being in liquid culture or in the solid growth medium. It can also be easily observed that the fluorescence increased with increasing salicylate concentrations (See figure 14 and 15). The measurements were made with  the fluorometer that our team built in order to be able to characterize this system. These results were compared to MACS measurements on the same colonies, and can be viewed on the fluorometer site. This is a further improvement of the  characterization of <A href="http://parts.igem.org/Part:BBa_J61051"><SPAN class="res_link">BBa_J61051</SPAN></A> done by the <A href="https://2013.igem.org/Team:Peking"><SPAN class="res_link">iGEM team Peking 2013</SPAN></A>.
   </P><U><B><P>IMAC</P></B></U><P>
   The chromatograms of ModLac and CueO (Ecol) showed very low protein concentrations (see Figures 9 and 10). The eluate fractions from the IMAC did not show any enzyme activity. Examination of these fractions with SDS-PAGE (see Figure 12) also showed that there were no, or very low concentrations of protein in them. This raises the question as to what went wrong in the purification method. The theory of IMAC rests upon the fact that his-tagged proteins will bind to the column and not elute until a high concentration of imidazole is pumped through the column. Enzyme activity was found in the lysate of CueO and its mutant, which means that there were functioning proteins prior to the IMAC. Possible explanations as to why no activity was measured in the eluate could be that the protein concentration was too low in the elution fraction (due to dilution) or that no protein bound to the column. This was confirmed when enzyme activity was found in the initial wash of the column. These results point toward the fact that something went wrong with the interaction between the protein and the column. It is unlikely that the nickel ions had not bound to the gel, since the columns had a clear light turquoise colour after being loaded. Also, the colour was not washed away during the IMAC procedure. There is a risk that the polyhistidine-tag, that was fused to the proteins, had folded into the protein and was not exposed on the protein surface. The tag needs to be on the surface of the protein to enable interaction with the nickel-enriched gel. This could have been checked by denaturing the protein samples prior to the IMAC, which would ensure that the his-tag was fully unfolded. This would however require renaturation of the protein to be able to measure enzyme activity.  
   </P><U><B><P>ABTS assay on ModLac and CueO</P></B></U><P>
   ABTS is a commonly used substrate when evaluating reaction kinetics of specific enzymes. Due to its reduction potential, it acts as an effective electron donor. Since we are working with laccases, which are multi copper oxidases, which oxidize substrates, ABTS is a suitable substrate. ABTS will donate electron to reduce molecular oxygen. The oxidized ABTS has a different absorption spectrum and the reaction can thus be observed in a spectrophotometer. 
   </P><P>
   The enzyme assays of the lysates with ABTS showed a significant difference in enzyme activity between CueO and the mutant CueO, also known as ModLac (Figure 11). In an article by Kataoka’s research group, the investigated CueO mutant showed to have a higher enzyme activity than the wild type (Kataoka K et al. 2012). This is confirmed by our results. The ModLac is designed so that the active site is the same as the double mutant that Kataoka and his research team investigated, but with some modifications at the C- and N-terminus. These modifications were made so that the enzyme would be exported from the cell and so that it could be easily purified. Restriction sites were added around these features so that it could be easily manipulated by our team and other iGEM teams that would like to use it for different purposes. These modifications could have affected the enzymatic activity by indirectly interfering with the active site. We tried to prevent this by adding a flexible linker sequence between the sequence coding for the laccase and the HlyA-export tag so that it would fold correctly. The conclusion that can be made after observing the results is that the designed ModLac have a better enzymatic activity than the wild type Laccase CueO (<A href="http://parts.igem.org/Part:BBa_K863006"><SPAN class="res_link">BBa_K863006</SPAN></A>). We have therefore improved the function of an already existing biobrick and supplemented further characterization of wild type CueO (<A href="http://parts.igem.org/Part:BBa_K863006"><SPAN class="res_link">BBa_K863006</SPAN></A>) when it is compared with ModLac. The new biobrick that we have designed has additional properties that makes it easier to manipulate than the wild type with simple tools in biotechnology. 
   </P><U><B><P>Ion-exchange chromatography</P></B></U><P>
  During the purification of the modified laccase, we tried using an ion-exchange column since the IMAC results was unsatisfying. However, to verify which fraction contained the CueO D439A/M510L mutant we used the Nanodrop and SDS-PAGE. The SDS-PAGE showed that the fractions didn’t contain any protein of significance (Figure 12) and the NanoDrop showed some clearly irregular numbers. Spectrophotometric measurements of the fractions with ABTS also confirmed what the SDS-PAGE showed, there seemed to be no proteins in the fractions, or they were very diluted (Data not shown).
  </P><P>
  The SDS-PAGE (Figure 13) showed that we did not manage to purify the CueO D439A/M510L mutant, though the NanoDrop results shows that there should be 11 g/ml of the protein.  The SDS-PAGE and Nanodrop relative to each other we can establish that the NanoDrop results are unreliable. 
  </P><P>
  The results clearly showed that it’s not CueO D439A/M510L mutant in the SDS-PAGE. The NanoDrop results can be discarded since 11g/mL also it not an accurate result of the protein concentration, by itself and compared to the SDS-PAGE results.
  </P><P>
  The error sources could be many but when going through the method of choice; there is an obvious error in the pH of the buffer. The theoretical isoelectric point of CueO D439A/M510L mutant is 6.5 and the buffers pH differs med +0.2. This is not a sufficient pH for making the CueO D439A/M510L mutant negatively charged.
This however may not be the only source of error. No protease inhibitor was used during the experiment, thus the proteins could also have been degraded.
  </P><P>
  The IEC is an anion exchange column and was used earlier with good results. It was also thoroughly washed and equilibrated with the mixed buffer.
  </P><P>
  The probable source of error is the buffer. The SDS-PAGE (Figure 13)results showed that it can be carbonic anhydrase (Can), that is produced during stress and starvation in the BL21DE3 strain and has a pI of 6.4. The overnight had been in the cold room for some time and had started to show floating webs of colonies.
  </P></DIV></DIV></DIV></DIV></BODY></HTML>