Team:NortheasternU-Boston/Description
NEU IGEM 2017 Expansion of Cell-Free Manufacturing with Post-Translational Modification
Home
Team
Collaborations
Attributions
+ Project
Description
Design
Experiments
Notebook
Results
Parts
Safety
+ Human Practices
Human Practices
Education and Public Engagement
Human Practices Silver
Judging Form
Project Description
Inspiration
Our team wanted to work with a synthetic biology system aimed at improving access to medicine in areas of the world lacking in strong medical and shipping infrastructure. While searching for ideas in this sphere we came across the notion of Cell-free expression reactions on a shelf stable paper substrate (Pardee, et al 2014, Pardee, et al. 2016). We were struck by the notion of circumventing the cold-chain and excited by the idea of building our own cell-free expression systems to produce medications for the developing world. As we sunk our teeth into the literature we realized that it would be difficult to find a realistic candidate for a medicine which was:
Limited by cold-chain shipping and storage
A simple enough compound to be produced in a cell free context
Would have important medical impact for the developing world
We decided to focus our attention on the problem of complexity and decided that rather than focus on the medicinal aspect of the problem we would focus our efforts instead on characterizing and improving cell-free reaction platforms in order to produce more complex biologics. Specifically we wanted to focus on post-translational modifications. Expanding cell-free reactions with post-translational modifications for important functions like glycosylation or cleavage and subsequent rearrangement to occur; offering the chance to produce complex biologics like antibodies or insulin in a cell-free context (Knorre et al. 2009).
Model System
The space of post-translational modifications is huge, and so the team needed to focus on a specific set of molecules and a set of post-translational modifications. We decided to work with antimicrobial peptides (AMPs) -- short peptides which exist as part of the immune system in many kinds of derived eukaryotes (Izadpanah et al. 2005). These were an ideal choice of model for several reasons. Groups had already worked with AMPs in a cell-free context using reaction preparations that were feasible for us to recapitulate in our lab and there is a large amount of diverse research on AMPs in general (Hancock et al. 2006, Pardee et al. 2016, Wu et al. 2014). There is a broad database of AMPs from a myriad of organisms which target broadly and specifically organisms from E. coli to P. falciparum (Wang et al. 2016).The diversity of this class of peptide made it attractive as a model because we could investigate factors like length, class of AMP, and predicted or known secondary structure. Producing AMPs across a complexity spectrum would give us some hint as to the capability of our cell-free system. AMPs also offered a key opportunity as a model in that many of them do not begin with a methionine as part of their peptide sequence. In our E. coli extract based system of cell-free expression we needed our expressed peptides to begin with a methionine start codon. For AMPs which do not have a canonical N-terminal methionine this residue would have to be removed in a post-translational reaction. We searched the literature for a way to achieve this and found work by Liao et al. (2004) working on optimizing mutants of methionine aminopeptidase isolated from E. coli. They presented findings of a GTG triple mutant methionine aminopeptidase with improved ability to cleave N-terminal methionines from peptides with bulky penultimate N-terminal residues. We now had our model system: Pick AMPs across a complexity gradient, with and without canonical N-terminal methionines and then test these for function against bacteria with and without the presence of methionine aminopeptidase.
Cell Free Reactions
Working with the limitations of our lab space and set up, and with the idea in mind to create the most economically viable cell-free reactions possible we decided to work with a system based off of crude E. coli extract. We adapted methodologies from various sources to the tools that were available to us (Kwon et al. 2015, Pardee et al. 2016, Sun et al 2013). CF reactions were made by growing up T7 Express E. coli induced to express the T7 polymerase. Homogenizing these cells and collecting a supernatant allowed to undergo a run-off reaction to digest endogenous RNA and DNA and then the subsequent combination of this crude extract with the supplementary buffers, tRNAs, and other compounds found in Sun et al. 2013, according to their protocol. The resulting expression mixture was then aliquoted onto paper strips in microcentrifuge tubes, flash frozen, and freeze dried over 24 hours and was then ready for use.
Citations
Guangshun Wang, Xia Li, Zhe Wang; APD3: the antimicrobial peptide database as a tool for research and education, Nucleic Acids Research, Volume 44, Issue D1, 4 January 2016, Pages D1087–D1093, https://doi.org/10.1093/nar/gkv1278
Hancock, R. E. W., & Sahl, H.-G. (2006). Antimicrobial and host-defense peptides as new anti-infective therapeutic strategies. 24, 1551. doi:10Wu.1038/nbt1267 http://www.nature.com/nbt/journal/v24/n12/full/nbt1267.html
Izadpanah, A., Gallo, R.L., 2005. Antimicrobial peptides. J. Am. Acad. Dermatol. 52, 381-390; quiz 391-392. doi:10.1016/j.jaad.2004.08.026 https://www.ncbi.nlm.nih.gov/pubmed/15761415
Knorre, D.G., Kudryashova, N.V., Godovikova, T.S., 2009. Chemical and Functional Aspects of Posttranslational Modification of Proteins. Acta Naturae 1, 29–51. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3347534/
Kwon, Y.-C., & Jewett, M. C. (2015). High-throughput preparation methods of crude extract for robust cell-free protein synthesis. 5, 8663. doi:10.1038/srep08663 https://www.nature.com/articles/srep08663#supplementary-information
Pardee, Keith, et al. “Paper-Based Synthetic Gene Networks.” Cell, vol. 159, no. 4, Nov. 2014, pp. 940–54. www.cell.com, doi:10.1016/j.cell.2014.10.004. http://www.cell.com/abstract/S0092-8674(14)01291-4
Pardee, K., Slomovic, S., Nguyen, P. Q., Lee, J. W., Donghia, N., Burrill, D., . . . Collins, J. J. (2016). Portable, On-Demand Biomolecular Manufacturing. Cell, 167(1), 248-259.e212. doi: https://doi.org/10.1016/j.cell.2016.09.013 http://www.sciencedirect.com.ezproxy.neu.edu/science/article/pii/S0092867416312466
Liao, Y.-D., Jeng, J.-C., Wang, C.-F., Wang, S.-C., & Chang, S.-T. (2004). Removal of N-terminal methionine from recombinant proteins by engineered E. coli methionine aminopeptidase. Protein Science : A Publication of the Protein Society, 13(7), 1802-1810. doi:10.1110/ps.04679104 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2279930/
Wu, X., Wang, Z., Li, X., Fan, Y., He, G., Wan, Y., . . . Yang, L. (2014). In Vitro and In Vivo Activities of Antimicrobial Peptides Developed Using an Amino Acid-Based Activity Prediction Method. Antimicrobial Agents and Chemotherapy, 58(9), 5342-5349. doi:10.1128/AAC.02823-14 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4135812/
Sun, Z. Z., Hayes, C. A., Shin, J., Caschera, F., Murray, R. M., Noireaux, V. (2013). Protocols for Implementing an Escherichia coli Based TX-TL Cell-Free Expression System for Synthetic Biology. J. Vis. Exp. (79), e50762, doi:10.3791/50762 https://www.jove.com/video/50762/protocols-for-implementing-an-escherichia-coli-based-tx-tl-cell-free
Our Generous Sponsors
