<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_End_of_crack_signalling_system"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/End_of_crack_%26_signalling_system">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/End_of_crack_%26_signalling_system&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/End_of_crack_%26_signalling_system&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/End_of_crack_%26_signalling_system&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/End_of_crack_%26_signalling_system" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/End of crack &amp;amp; signalling system</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="newcastle-page"><TABLE style="width:100%" cellspacing="0" cellpadding="0"><TBODY><TR><TD><A href="/Team:Newcastle" title="Team:Newcastle">Home</A></TD><TD><A href="/Team:Newcastle/the_team" title="Team:Newcastle/the team">Team</A></TD><TD><A href="/Team:Newcastle/problem" title="Team:Newcastle/problem">Problem</A></TD><TD><A href="/Team:Newcastle/solution" title="Team:Newcastle/solution">Solution</A></TD><TD><A href="/Team:Newcastle/modelling" title="Team:Newcastle/modelling">Modelling</A></TD><TD><A href="/Team:Newcastle/notebook" title="Team:Newcastle/notebook">Lab book</A></TD><TD><A href="/Team:Newcastle/meetings" title="Team:Newcastle/meetings">Meetings</A></TD><TD><A href="/Team:Newcastle/safety" title="Team:Newcastle/safety">Ethics</A></TD><TD><A href="/Team:Newcastle/Medals" title="Team:Newcastle/Medals">Judging</A></TD><TD><A href="/Team:Newcastle/other" title="Team:Newcastle/other">Other</A></TD><TD><A href="/Team:Newcastle/Sponsors" title="Team:Newcastle/Sponsors">Sponsors</A></TD></TR></TBODY></TABLE><DIV style="padding:30px"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#End_of_crack_cell-signalling_system"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">End of crack cell-signalling system</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Subtilin_Production_part"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Subtilin Production part</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Subtilin_Immunity_part"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Subtilin Immunity part</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Cloning_Strategy"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Cloning Strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Testing_and_Characterisation"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Testing and Characterisation</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Selection_for_integration"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Selection for integration</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Isolation_of_subtilin_from_B._subtilis_ATCC_6633"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Isolation of subtilin from B. subtilis ATCC 6633</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Characterisation_of_BioBrick"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Characterisation of BioBrick</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Expected_results"><SPAN class="tocnumber">1.4.3.1</SPAN><SPAN class="toctext">Expected results</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="End_of_crack_cell-signalling_system">End of crack cell-signalling system</SPAN></H1><P>Subtilin belongs to a class of peptide antibiotics, called lantibiotic that contain polycyclic thioether amino acids as well as the unsaturated amino acids dehydroalanine and 2-aminoisobutyric acid. It is produced by <I>Bacillus subtilis</I> ATCC 6633, which is effective against a wide range of Gram positive bacteria including pathogenic bacteria such as propionibacteria, staphylococci, clostridia, enterococci and streptococci.
</P><P>Subtilin also doubles up as a quorum sensing molecule, which we can use to trigger calcium carbonate precipitation and filament cell formation once the bacteria have reached a sufficient density inside a crack. We developed a subtilin immunity part and a subtilin production part to add to the already-existing <A href="http://partsregistry.org/Part:BBa_K104001" class="external text" rel="nofollow">subtilin sensing part on the parts registry</A>.
</P><P><B>Figure 1:</B> Schematic diagram of subtilin cell signalling system in <I>B. subtilis</I> 168
</P><H2><SPAN class="mw-headline" id="Subtilin_Production_part">Subtilin Production part</SPAN></H2><P>The subtilin production Biobrick consists of the <I>spaBCTS</I> gene cluster under an oxygen limitation sensitive promoter. In low oxygen conditions, similar to conditions at the end of a crack with high cell density, <I>spaS</I> will be expressed. The immature SpaS will then have to be modified and transported out of the cell by SpaBCT, an ABC transporter.
</P><P>We will place this part in front of the existing <A href="http://partsregistry.org/Part:BBa_K104001" class="external text" rel="nofollow">subtilin-sensor part</A>, allowing the subtilin signal to propagate from the end of the crack throughout the cell population.
</P><P>This is <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K302021" class="external text" rel="nofollow">BBa_K302021 on the parts registry</A>.
</P><H2><SPAN class="mw-headline" id="Subtilin_Immunity_part">Subtilin Immunity part</SPAN></H2><P>The subtilin immunity BioBrick consists of the <I>spaIFEG</I> gene cluster under a constitutive promoter, Pveg. Immunity is based on two systems. SpaI is a lipoprotein that will bind to and interacts with subtilin at the cell surface. SpaFEG forms an ABC transporter homologue that will expel subtilin from the cell membrane into the extra cellular matrix.
</P><P>This is <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K302018" class="external text" rel="nofollow">BBa_K302018 on the parts registry</A>.
</P><H2><SPAN class="mw-headline" id="Cloning_Strategy">Cloning Strategy</SPAN></H2><P><A href="/wiki/images/8/89/Newcastle_CloningStrategy_BioBrick_Subtilin_Immunity.pdf" class="internal" title="Newcastle CloningStrategy BioBrick Subtilin Immunity.pdf">Subtilin Immunity</A></P><H2><SPAN class="mw-headline" id="Testing_and_Characterisation">Testing and Characterisation</SPAN></H2><H4><SPAN class="mw-headline" id="Selection_for_integration">Selection for integration</SPAN></H4><P>To select for integration of the plasmid into the chromosome, <I>B. subtilis</I> will be tested for the ability to hydrolyse starch. Integration of the BioBrick will be done by homologous recombination at the <I>amyE</I> gene, therefore destroying the ability to express amylase. Colonies that do not contain the integrated BioBrick will be able to hydrolyse starch, therefore forming a white halo around the colony as iodine interacts with starch to form blue colour.
</P><H4><SPAN class="mw-headline" id="Isolation_of_subtilin_from_B._subtilis_ATCC_6633">Isolation of subtilin from <I>B. subtilis</I> ATCC 6633</SPAN></H4><TABLE><TBODY><TR><TD>1.	<I>B. subtilis</I> ATCC6633 were grown in 250 ml flasks containing 100 ml of BHI broth and incubate for 48 h at 32°C in shaker at 125 cycles/min.
</TD></TR><TR><TD>2.	Culture media were centrifuged at 10000g for 15 min and the supernatants were filtered through 0.22 um membrane.
</TD></TR><TR><TD>3.	The pH of the filtrates has to be between 7.0 to 8.0 pH.
</TD></TR><TR><TD>4.	Store the filtrates at -20°C for further use.
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Characterisation_of_BioBrick">Characterisation of BioBrick</SPAN></H3><TABLE><TBODY><TR><TD>1.	Make up 4 LB agar plates and inoculate with the following organism:Negative Control –<I> B. subtilis</I> 168,  Positive Control – <I>B. subtilis</I> ATCC6633 and Test (Duplicates) – <I>B. subtilis</I> 168 containing subtilin immunity BioBrick.
</TD></TR><TR><TD>2.	Apply 20 ul of the cell free filtrate containing subtilin onto cellulose disks (6 mm) and place onto the LB agar plates.
</TD></TR><TR><TD>3.	Plates were incubated for 24 h at 37 °C.
</TD></TR><TR><TD>4.	Zone of inhibition were then measured.
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Expected_results">Expected results</SPAN></H4><TABLE><TBODY><TR><TD>1.	Negative control – Zone of inhibition to be observed
</TD></TR><TR><TD>2.	Positive control – Zone of inhibition not observed
</TD></TR><TR><TD>3.	Test (Duplicates) – Zone of inhibition not observed or a smaller zone of inhibition as compared to the negative control
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>A.S. Motta and A. Brandelli. (2002) Characterization of an antibacterial peptide produced by Brevibacterium linens. Journal of Applied Microbiology 92, 63-70.
</P><P>A.S. Motta, F. Cladera-Olivera and A. Brandelli (2004) Screen for antimicrobial activity among bacteria isolated from the amazon basin. Brazilian Journal of Microbiology 35, 307-310.
</P><CENTER><TABLE style="text-align:center"><TBODY><TR><TD>    </TD></TR><TR><TD>        </TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Newcastle/End_of_crack_%26_signalling_system">http://2010.igem.org/Team:Newcastle/End_of_crack_%26_signalling_system</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/End_of_crack_%26_signalling_system" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/End_of_crack_%26_signalling_system" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/End_of_crack_%26_signalling_system&amp;_signalling_system=&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/End_of_crack_%26_signalling_system&amp;oldid=152287" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>