<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Newcastle_research"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Newcastle/research">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Newcastle/research&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/research&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Newcastle/research&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Newcastle/research" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Newcastle/research</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="newcastle-page"><TABLE style="width:100%" cellspacing="0" cellpadding="0"><TBODY><TR><TD><A href="/Team:Newcastle" title="Team:Newcastle">Home</A></TD><TD><A href="/Team:Newcastle/the_team" title="Team:Newcastle/the team">Team</A></TD><TD><A href="/Team:Newcastle/problem" title="Team:Newcastle/problem">Problem</A></TD><TD><A href="/Team:Newcastle/solution" title="Team:Newcastle/solution">Solution</A></TD><TD><A href="/Team:Newcastle/modelling" title="Team:Newcastle/modelling">Modelling</A></TD><TD><A href="/Team:Newcastle/notebook" title="Team:Newcastle/notebook">Lab book</A></TD><TD><A href="/Team:Newcastle/meetings" title="Team:Newcastle/meetings">Meetings</A></TD><TD><A href="/Team:Newcastle/safety" title="Team:Newcastle/safety">Ethics</A></TD><TD><A href="/Team:Newcastle/Medals" title="Team:Newcastle/Medals">Judging</A></TD><TD><A href="/Team:Newcastle/other" title="Team:Newcastle/other">Other</A></TD><TD><A href="/Team:Newcastle/Sponsors" title="Team:Newcastle/Sponsors">Sponsors</A></TD></TR></TBODY></TABLE><DIV style="padding:30px"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Research_teams"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Research teams</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Voigt_Paper"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Voigt Paper</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Secretion_control"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Secretion control</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Kill_switch"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Kill switch</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Caulobacter_Crescentus_Glue_vs._Calcium_Carbonate_and_Spider_Silk"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Caulobacter Crescentus Glue vs. Calcium Carbonate and Spider Silk</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Injection"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Injection</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Concrete_properties"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Concrete properties</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Size_of_pores_and_microcracks"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Size of pores and microcracks</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Creating_cracks_for_testing"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Creating cracks for testing</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Properties_of_calcium_carbonate_and_spider_silk"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Properties of calcium carbonate and spider silk</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Surface_tension"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Surface tension</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Mixing_bacteria_into_concrete"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Mixing bacteria into concrete</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Genetics"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Genetics</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Bacillus_subtilis_spores"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Bacillus subtilis spores</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Bacillus_subtilis_life_cycle"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Bacillus subtilis life cycle</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Autonomous_Linear_DNA_Clock"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Autonomous Linear DNA Clock</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Research_teams"> Research teams </SPAN></H2><P>BioBrick 1 (<A href="/Team:Newcastle/Calcium_Carbonate" title="Team:Newcastle/Calcium Carbonate">CaCO<SUB>3</SUB></A> /<A href="/Team:Newcastle/Urease" title="Team:Newcastle/Urease">Urease</A>): All
</P><P>BioBrick 2 (<A href="/Team:Newcastle/End_of_crack_%26_signalling_system" title="Team:Newcastle/End of crack &amp; signalling system">End of crack &amp; signalling system</A>): D.Y.; P.H.; Y.E.; H.S.
</P><P>BioBrick 3 (<A href="/Team:Newcastle/Non-target-environment_kill_switch" title="Team:Newcastle/Non-target-environment kill switch">Non-target-environment kill switch</A>): S.W.; A.L.; R.B.; J.S. 
</P><P>BioBrick 4 (<A href="/Team:Newcastle/Spider_silk" title="Team:Newcastle/Spider silk">Spider silk</A>&amp;<A href="/Team:Newcastle/Filamentous_Cells" title="Team:Newcastle/Filamentous Cells">Filamentous Cells</A>): J.S.; R.B.; S.W.; Y.E.   
</P><P>BioBrick 5 (<A href="/Team:Newcastle/glue" title="Team:Newcastle/glue">Glue &amp; lysis</A>): H.S.; A.L.; P.H.; D.Y.
</P><P>Secretion Systems
</P><H2><SPAN class="mw-headline" id="Voigt_Paper">Voigt Paper</SPAN></H2><UL><LI>Type III secretion system (T3SS) - exports proteins, translocates polypeptides through both the inner and outer membranes.
</LI><LI>Samonella Pathogenicity Island 1 (SP-1) - tool for export proteins of interest, can be applied to the recombinant production of spider silk proteins, which form fibrils if allowed to accumulate inside of confined volume of the cell.
</LI><LI> They've used the whole gene DNA synthesis to construct DNA sequences that matches exactly the wild type amino acid sequence for the known fragments of silk monomers.
</LI><LI>The synthetic genes were designed for 3 silk genes, from the spider named <I>Araneus diadenatas</I>.
</LI><LI>ADF-1: expressed in the minor ampullate gland, using during construction. High tensile strength, but inelastic.
</LI><LI>ADF-2: expressed in the cylindrical gland, used for egg sacks, sequence similar to human elastins.
</LI><LI>ADF-3: expressed in the major ampullate gland, forms extremely tough and elastic draglines which anchors the web.
</LI></UL><LI>Each of the genes is expressed and exported using the Salmonella SP-1 T3SS.
</LI><H2><SPAN class="mw-headline" id="Secretion_control">Secretion control </SPAN></H2><P>In Gram-positive bacteria, proteins can be sorted to at least four different destinations: the cytoplasm, the cytoplasmic membrane, the cell wall and the extracellular medium.
</P><P>By far the largest number of translocated and membrane proteins in Gram-positive bacteria are predicted to follow the general protein secretion (Sec) pathway, which involves SecA, SecY, SecE, SecG and a number of accessory proteins.
</P><P>Sec secretion system in the rod-shaped bacterium <I>Bacillus subtilis</I> have been shown to localise in spirals along the cytoplasmic membrane.
</P><P>The Tat secretion system, which is used for the transport of folded proteins, probably localises in the cytoplasmic membrane and at the cell poles of <I>B. subtilis</I>.
</P><P><I>B. subtilis</I> contains two Oxa1p (acts as a general membrane insertion machinery for proteins) homologues that involved in membrane protein biogenesis and in protein secretion, namely SpoIIIJ and YqjG . Both proteins are randomly distributed throughout the membrane and are thus not enriched in the vicinity of the Sec machinery.
</P><H2><SPAN class="mw-headline" id="Kill_switch">Kill switch</SPAN></H2><P>Please see Richard's attached notes <A href="https://2010.igem.org/Image:Bacterial_programmed_cell_death-1-.pdf" class="external autonumber" rel="nofollow">[1]</A>.
</P><H2><SPAN class="mw-headline" id="Caulobacter_Crescentus_Glue_vs._Calcium_Carbonate_and_Spider_Silk">Caulobacter Crescentus Glue vs. Calcium Carbonate and Spider Silk</SPAN></H2><P>ULB-Brussels developed BioBricks for production of <I>Caulobacter crescentus</I> glue, and won the Best New BioBrick Part, Natural award in 2009.
<I>Caulobacter crescentus</I> is a Gram-negative bacterium, and the parts are optimised for <I>E. coli</I>. Unclear whether they will work in <I>B. subtilis</I>.
</P><P>Genes: <I>hfsE, hfsF, hfsG, hfsH, (hfsC), hfsI, hfsD, (hfsA, hfsB)</I>.
</P><P>The inner membrane HfsE protein initiates glycosyltransferase by transferring N-acetylglucosamine (NAG) from UDP−NAG to a lipid carrier. HfsG, a second glycosyltransferase protein, transfers NAG subunits to the growing polysaccharide chain. HfsH deacetylates one or more NAG residues. The HfsF protein translocates the polysaccharide chain linked to the lipid carrier across the inner membrane. The polymerases HfsC and HfsI proteins link the NAG repeat units together. The holdfast polysaccharide is transferred across the outer membrane by HfsA, HfsB and HfsD proteins. Hfa proteins mediate the polysaccharide attachment to the cell.
</P><P>Only <I>hfsG</I> and <I>hfsH</I> were inserted, since the other genes are already present in <I>E. coli</I>. Does <I>B. subtilis</I> have the other genes, or do we need to insert those too? Will it work at all in a Gram-positive bacterium?
</P><P>References:
</P><P><B>ULB-Brussels 2009 iGEM team.</B> A new generation of glue.
</P><P><B>Chris S. Smith, Aaron Hinz, Diane Bodenmiller, David E. Larson, and Yves V. Brun.</B> 2002 Identification of Genes Required for Synthesis of the Adhesive Holdfast in Caulobacter crescentus
</P><H2><SPAN class="mw-headline" id="Injection">Injection</SPAN></H2><P>Steve &amp; Zoltan
</P><H2><SPAN class="mw-headline" id="Concrete_properties">Concrete properties</SPAN></H2><P>Concrete has a high compressive strength (3000 to 4000 psi) and a very low tensile strength (between 8% to 15% of the compressive strength.). Cracking is due to the low tensile strength.
</P><P>In reinforced concrete the steel reinforcement provides the tensile strength lacking in concrete. Steel has tensile strength equal to approximately 100 times that of concrete. Steel reinforcing is also capable of resisting compression forces.
</P><P>Concrete and steel work very well together in relation to temperature changes because their coefficients of thermal expansion are quite close to each other. For steel it is 0.0000065 per unit length per degree Fahrenheit, while for concrete it is about 0.0000055.
</P><P>Reference: Design of Reinforced Concrete 5th Edition, Jack C. McCormac.
</P><P><A href="/wiki/index.php?title=User:Steven_Woodhouse&amp;action=edit&amp;redlink=1" class="new" title="User:Steven Woodhouse (page does not exist)">Steven Woodhouse</A> 19:23, 25 March 2010 (UTC)
</P><H2><SPAN class="mw-headline" id="Size_of_pores_and_microcracks">Size of pores and microcracks</SPAN></H2><P>Pores are roughly between 0.01 and 100 micrometers and a microcrack is &lt;1 mm in diameter, according to <I>Application of bacteria as self-healing agent for the development of sustainable concrete</I>, Jonkers et. al
</P><H2><SPAN class="mw-headline" id="Creating_cracks_for_testing">Creating cracks for testing</SPAN></H2><P>A procedure using cylinders of concrete is presented in <I>Use of bacteria to repair cracks in concrete</I>, Tittelboom et al.
</P><H2><SPAN class="mw-headline" id="Properties_of_calcium_carbonate_and_spider_silk">Properties of calcium carbonate and spider silk</SPAN></H2><P>Calcium carbonate has a similar thermal expansion coefficient to concrete.
</P><P>Some spider silks have a similar or better tensile strength than steel.
</P><P>Glue, I haven't been able to find details of yet, but it is unlikely it has as good qualities.
</P><P>Fibre-reinforced concrete. Addition of steel fibres (usually between 1% to 2% by volume) can appreciably improve their characteristics. Strength is not significantly improved, but significantly tougher and have greater resistance to cracking. The fibres used are usually from about ¼ inch up to about 3 inches with diameters of 0.01 inch up to 0.03 inches. While a reinforcing bar provides reinforcing only in the direction of the bar, randomly distributed fibres provide additional strength in all directions.
</P><P>Recommendation at the moment: calcium carbonate with spider silk. Also, if we can get silk working it would give us some novelty vs. the other concrete projects we've found.
</P><P>However: which silks have been sequenced? Is silk alkali-resistant?
</P><P><A href="/wiki/index.php?title=User:Steven_Woodhouse&amp;action=edit&amp;redlink=1" class="new" title="User:Steven Woodhouse (page does not exist)">Steven Woodhouse</A> 19:24, 25 March 2010 (UTC)
</P><UL><LI> How porous is concrete?
</LI><LI> How fast will concrete soak up liquid? 
</LI><LI> Will a surfactant speed up this process?
</LI><LI> How to inspect for micro cracks:
</LI><LI> fluorescence
</LI><LI> ultra sound
</LI></UL><H2><SPAN class="mw-headline" id="Surface_tension">Surface tension</SPAN></H2><P>If surface tension is a problem a surfactant could be produced by the bacteria or mixed into the spray.
See, for example, <I>Which surfactants reduce surface tension faster? A scaling argument for diffusion-controlled adsorption</I>, Ferri and StebeU.
</P><H2><SPAN class="mw-headline" id="Mixing_bacteria_into_concrete">Mixing bacteria into concrete</SPAN></H2><P>In a study done by Delft University of Technology (Jonkers et al., Application of bacteria as self-healing agent for the development of sustainable concrete), alkali-resistant spores were directly added to the concrete mixture prior to casting and remained viable for a period up to 4 months.
</P><P>Incorporation of a high number of bacterial spores resulted in a decrease in compressive strength of less than 10% for 3, 7 and 28 days cured specimens. The incorporation of calcium lactate did not substantially affect strength (3 and 7 days cured) or even resulted in a slight increase (28 days cured) in compressive strength values.
</P><P>Possible solutions to the loss of viability of the bacterial spores after 4 months are suggested in the paper.
</P><P><A href="/wiki/index.php?title=User:Steven_Woodhouse&amp;action=edit&amp;redlink=1" class="new" title="User:Steven Woodhouse (page does not exist)">Steven Woodhouse</A> 20:55, 21 March 2010 (UTC)
</P><P>
In Use of bacteria to repair cracks in concrete, it was found that the bacteria had to be immobilised in silica gel to protect them from the high pH in concrete.
<A href="/wiki/index.php?title=User:Steven_Woodhouse&amp;action=edit&amp;redlink=1" class="new" title="User:Steven Woodhouse (page does not exist)">Steven Woodhouse</A> 17:15, 23 March 2010 (UTC)
</P><H2><SPAN class="mw-headline" id="Genetics">Genetics</SPAN></H2><P>Harsh and Rachel
</P><P>Nutrition and Plasmid vs genome
</P><P>Integration of stable extracellular DNA released from <I>E. coli</I> into <I>B. subtilis</I> genome vector by culture mix method.
</P><P>
Nutrition: Rather than the nutrient gradient, we could just spray nutrients on the wall and when bacterial population increases, the nutrient levels would decrease activating a promoter which would activate sporulation.
</P><H2><SPAN class="mw-headline" id="Bacillus_subtilis_spores"><I>Bacillus subtilis</I> spores</SPAN></H2><P>Alan and Da
</P><P>Sporulation in <I>Bacillus</I> is governed by SpoOA that in turns activates over 100 genes involved in sporulation.
</P><P>DNA replication have been linked to sporulation and is controlled by the AAA+ replication initiator protein DnaA.
If there is disturbance in DNA replication, then DnaA will activate the sda gene encoding the sporulation inhibitor protein Sda.DnaA was shown to act as a positive regulator of sda transcription when DNA replication is perturbed. Thus,under conditions of replication stress, the Sda checkpoint prevents cells from initiating sporulation.
</P><P>During each cell cycle the checkpoint protein Sda couples the initiation of DNA replication with the initiation of sporulation. Sda serves as a genetic timer that allows a window of opportunity toward the end of each replication cycle for Spo0A;P to reach the levels required to initiate sporulation, thus ensuring that a cell enters the sporulation pathway when
it contains the correct copy number and has completed DNA replication and repair. Disruption of the coordination of replication and sporulation leads to an increase in the frequency of spores with more than one chromosome and a reduction in spore viability.
</P><P>Therefore trageting or controlling Sda level maybe the key to obtaining high amount of spores, but at the same time running into the risks of low spores viability.
</P><P>The paper, <I>Bacillus subtilis</I> spore coats in the drop box describle the process of sporulatuion with diagrams and various porteins involved.
</P><P>Takes about 8h from the time of initiation of sporulation to complete spore formation.
</P><H2><SPAN class="mw-headline" id="Bacillus_subtilis_life_cycle"><I>Bacillus subtilis</I> life cycle</SPAN></H2><P>Alan and Da
</P><P>Low nutrient availability leads to low level of DnaA in the cells, therefore no chromosome replication. DnaA can also be regulated  by Soj in <I>Bacillus</I>.
</P><P>The bacterial cell cycle is divided into three stages: the period between division (cell ‘birth’) and the initiation of chromosome replication (known as the B period); the period required for replication (known as the C period); and the time
between the end of replication and completion of division (known as the D period.)
</P><P>Chromosome replication is governed by DnaA and is regulated by nutrient availability and also Soj. Cell division is regulated by the formation of FtsZ in mid-cell, leading to the formation the divisome.
</P><P>The paper, Metabolism, cell growth and the bacterial cell cycle, provides nice diagrams on the process.
</P><H2><SPAN class="mw-headline" id="Autonomous_Linear_DNA_Clock"> Autonomous Linear DNA Clock </SPAN></H2><A href="/Team:Newcastle/Autonomous_linear_DNA_Clock" title="Team:Newcastle/Autonomous linear DNA Clock">Richard's notes</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Newcastle/research">http://2010.igem.org/Team:Newcastle/research</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Newcastle/research" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Newcastle/research" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Newcastle/research&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Newcastle/research&amp;oldid=11495" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>