<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUBU-Wuhan_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUBU-Wuhan/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class=""><UL class="nav"><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Collaborations">Collaborations</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Results">Results</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Parts">Part Table</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Improve">Improve</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Calendar">Calendar</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Protocol</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Intergrated_Human_Practices">Intergrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Public_Engagement">Education&amp; Engagement</A></LI><LI><A href="#">Contribution</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Model">Model</A></LI></DIV><SECTION class="story section" style="padding-bottom:30px;padding-top:150px;"><DIV class="section-title"><H2><SPAN>Protocol</SPAN></H2></DIV></SECTION><SECTION class="story section" style="padding-top:20px;padding-left:15%;padding-right:15%;"><DIV class="section-title"><H1 style="text-align: left;padding:60px 0;">1.	Primer design:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    15bp homologous arm（Forward:ATCCTCAGTGTCAGC; Reverse:GCCCTTGCTCACCAT）+ DNA sequences.
		</P><H1 style="text-align: left;padding:60px 0;">2.	Amplification of target gene:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    Take the total DNA as a template, using designed primers to obtain target fragments with promoters.
		</P><H1 style="text-align: left;padding:60px 0;">3.	Construction of recombinant plasmids:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    T5 Enzyme Assisted Cloning.         1)	Prepare reactive system (5 μL).             <B>F/V</B>: molar ratio of 3:1             <B>T5 enzyme</B>: 0.5 μL             <B>Buffer</B>: 0.5 μL             <B>ddH<SUB>2</SUB>O</B>: complemented to 5 μL     All regents were mixed and reacted on the ice for 5 minutes.     2)	Add 100 μL DH5α competent cell and reacted on the ice 30 min.    3)	42℃, heat shock 45 s, standing in 2-3 min.    4)	Add 100 μL NZY to recovery, culture at 37℃ with 250 rpm for 1 h.             {NZY medium: 1% NZ-Amine, 0.5% yeast extract, 0.5% NaCl, (autoclaving), 12.5 mmol/L MgCl<SUB>2</SUB>·6H<SUB>2</SUB>O, 12.5 mmol/L MgSO<SUB>4</SUB>·7H<SUB>2</SUB>O, 20 mmol/L glucose (filtration sterilization)}.    5)	Plate cells on LB agar plates with Spe(20 mg/mL ).
		</P><H1 style="text-align: left;padding:60px 0;">4.	Screening <I>E.coli</I> recombinant strain:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    10 μL aseptic water was taken from the super-clean workbench and 8 single colonies were taken from each sample in a PCR tube. The samples were dissolved in 10 μL aseptic water as templates.
		</P><H1 style="text-align: left;padding:60px 0;">5.	Sequencing:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    Sending the sample to sequencing. Analysing the sequencing result with Snapgene. If the sequencing result of plasmid was correct, preserve the strain.
		</P><H1 style="text-align: left;padding:60px 0;">6.	Electrotransformation:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Precool the cuvettes and prepare competent cell in the ice.     2)	Mix plasmid (0.5-10 ng) and 50 μL competent cells in the cooling cuvette.    3)	Carefully dry the water on the outside of cuvette, and then insert the cuvette into the electroporator and press the start button.    4)	Add 1 mL liquid medium (RM: rich media) to recovery bacteria and incubate at 30℃for 6-12 h RM: 20.0 g/L, glucose; 10.0 g/L, yeast extract; 2.0 g/L, KH<SUB>2</SUB>PO<SUB>4</SUB>.    5)	Plate cells on LB agar plates with special antibiotic marker. LB: 10.0 g/L tryptone, 5.0 g/L yeast extract, and 10.0 g/L NaCl.

		</P><H1 style="text-align: left;padding:60px 0;">7.	FACS:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    Flow cytometer was used to detect fluorescence. Record the data.
		</P><H1 style="text-align: left;padding:60px 0;">8.	Double Digestion:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Add 5ul 10X FlyCut buffer to PCR tube.    2)	Add an amount containing 1ug of DNA sample. If DNA concentration is &lt;62.5ng/uL, add 16ul.    3)	Add 0.5ul Endonuclease each.    4)	For total volume&lt;50ul, add ddH<SUB>2</SUB>O to make up 50ul.    5)	Incubate at 37°C for 30min.

		</P><H1 style="text-align: left;padding:60px 0;">9.	Ligation:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Measure concentration of insert and vector.    2)	Add at least 20ng of vector.    3)	Add insert in 1:3 vector to insert ratio.    4)	Add 2uL of 5X T4 Ligase Buffer.    5)	Add 0.5uL of T4 Ligase.    6)	Top up till 10uL with ddH<SUB>2</SUB>O.    7)	Incubate at 4℃ temperature for 18 hours.

		</P><H1 style="text-align: left;padding:60px 0;">10.	Transformation:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Thaw Competent Cells on ice.    2)	Add 10ul ligated product or plasmid into Competent Cell and quickly place cell back on ice, incubating for 30mins.    3)	Conduct heat shock at 42°C for 45s.    4)	Chill cell on ice for 3min.    5)	Add 100uL NZY to cell.    6)	Incubate at 37°C for 1h.    7)	Spread 200uL of culture on LB antibiotic agar plate.    8)	Culture overnight at 37°C.    9)	Store plate at 4°C.


		</P><H1 style="text-align: left;padding:60px 0;">11.	Isobutanol production measurement(HPLC):</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Inoculate strains for overnight culture in 5mL RMG5 in Culture Flask at 30°C.    2)	Measure the OD600 of the overnight culture with 200uL of sample.    3)	Dilute OD600 to 0.001 in 4mL RMG5 media in Deep-well Multiwell Plate and incubate in 30°C.    4)	Take samples after 24 hours by filtration sterilization.    5)	Make HPLC measurement and sort data.


		</P><H1 style="text-align: left;padding:60px 0;">12.	Alkaline hydrolysis:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Mix 12% NaOH solution and material(the waste carton) in ratio 15:1 (mL:g).    2)	It’s placed in the constant temperature shaker ,and it’s treated under 55°and the speed of 150r/min for 1.5h.

		</P><H1 style="text-align: left;padding:60px 0;">13.	Acid hydrolysis:</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    1)	Mix 3% H<SUB>2</SUB>SO<SUB>4</SUB> solution and material(the waste carton) in ratio 15:1 (mL:g).    2)	It’s placed in the constant temperature shaker ,and it’s treated under 55°and the speed of 150r/min for 1.5h.    3)	Adjust pH to 4.8 by adding NaOH.

		</P><P style="text-align: left;line-height:24px;padding:30px 0;"><B>References:</B>    [1]Jingjing Zhang,etal.
Journal of Anhui Agri Sci,2009,37(35):17312-17314.
     [2]Duan Yuan,etal.
[J].Renewable Energy Sources,2011,29(3):30-34.
    [3]Xiao Lingping,Tang Yong,Deng Lihong.Enzymatic hydrolysis of waste corrugated cellulose and changes in fiber structure during enzymatic hydrolysis[J].Forestry Chemicals and Industry,2010,30(02):83-88.
		</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>