<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUBU-Wuhan_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUBU-Wuhan/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class=""><UL class="nav"><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Collaborations">Collaborations</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Results">Results</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Parts">Part Table</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Improve">Improve</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Calendar">Calendar</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Protocol</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Intergrated_Human_Practices">Intergrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Public_Engagement">Education&amp; Engagement</A></LI><LI><A href="#">Contribution</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Model">Model</A></LI></DIV><SECTION class="story section" style="padding-bottom:30px;padding-top:150px;"><DIV class="section-title"><H2><SPAN>Improve</SPAN></H2></DIV></SECTION><SECTION class="story section" style="padding-top:20px;padding-bottom:150px;padding-left:15%;padding-right:15%;"><DIV class="section-title"><P style="text-align: left;line-height:24px;padding:30px 0;">    Bielefeld-CeBiTec team in iGEM 2014 attempted to construct a metabolic pathway to produce isobutanol in <I>E. coli</I>. We improved upon the project with new isobutanol production module which contains different genes and optimized promoters and RBS. And our metabolic pathway of isobutanol production is constructed in non-model organism <I>Zymomonas mobilis</I>. We intend to construct isobutanol production modules which are not only suitable in <I>Zymomonas mobilis</I>, but also compatible in other microorganisms.
		</P><H1 style="text-align: left;padding:60px 0;">Part No.1</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    <B>Part name:</B>  RBS-10, RBS-1000, RBS-10000, RBS-Maximum    <B>Part number:</B>  BBa_K2800031, BBa_K2800032, BBa_K2800033, BBa_K2800034    <B>Description:</B>            Our RBS is predicted to be very suitable to <I>Zymomonas mobilis</I> by the algorithm of SALIS with <I>Zymomonas mobilis’s</I> 16sRNA. We test these RBS with our dual reporter-gene system in <I>Zymomonas mobilis</I>, and the relativity of experimental results is coincident with predicted results. And we also test these RBS in <I>E. coli</I> and the results shows that they are also suitable to <I>E. coli</I>.

		</P><H1 style="text-align: left;padding:60px 0;">Part No.2</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    <B>Part name:</B>  Pgap-3s-2, Pgap-3s-11, Pgap-3s-14, Pgap-4s-p1-18, Pgap-4S-P2-4, Pgap-4S-P2-6    <B>Part number:</B>  BBa_K2800000, BBa_K2800001, BBa_K2800002, BBa_K2800003, BBa_K2800004, BBa_K2800005    <B>Description:</B>            The strong constitutive promoter Pgap is expressed in <I>Zymomonas mobilis</I>, but not in <I>E. coli</I>. We construct Pgap mutants library by error-prone PCR and dual reporter-gene system in <I>E. coli</I> T1 strain and screen out expression enhanced cell gate through cell sorting. Then find the mutation sites through sequencing and do further verification to get Pgap mutants. Finally, we obtained these 6 mutants which have more than 1.5-fold changes compared with wild type Pgap. These mutant Pgap can be used in isobutanol production module in <I>E. coli</I>.

		</P><H1 style="text-align: left;padding:60px 0;">Part No.3</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    <B>Part name:</B>  Ethanol inducible promoter 0435, Ethanol inducible promoter 0405, Up-regulated in stationary phase promoter 0038    <B>Part number:</B>  BBa_K2800006, BBa_K2800007, BBa_K2800008    <B>Description:</B>            These promoters can give priority to strain’s growth and make it begin fermentation to produce isobutanol later. In fact, <I>Zymomonas mobilis</I> is a capable ethanologenic bacterium with high ethanol productivity and ethanol tolerance. As ethanol accumulated, promoters get higher expression in stationary phase.

		</P><H1 style="text-align: left;padding:60px 0;">Part No.4</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    <B>Part name:</B>  P0038-RBS-kdcA    <B>Part number:</B>  BBa_K2800021    <B>Description:</B>            Promoter P0038 is up-regulated in stationary phase. As a result, the strain grows and reproduces swiftly during the early exponential phase, consuming a small quantity of glucose and accumulating amino acid intermediates, and then before the glucose run out, enhance the expression of kdcA to decarboxylate 2-keto acids to aldehydes at a relatively high rate. Let the strain regulates itself, like a automatic control system, so as to improve the isobutanol yield.

		</P><H1 style="text-align: left;padding:60px 0;">Part No.5</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    <B>Part name:</B>  Ptet-RBS-BsAls2-RBS-ilvc-RBS-ilvd-Terminator-Pgap-RBS-kdcA-Terminator, Ptet-RBS-BsAls2-RBS-ilvc-RBS-ilvd-Terminator-Peno-RBS-kdcA-Terminator    <B>Part number:</B>  BBa_K2800029, BBa_K2800030    <B>Description:</B>            This part is improved upon BBa_K1465306 of Bielefeld-CeBiTec team in iGEM 2014. BBa_K1465306 consists of an operon, but this part consists of two operons and has different promoter, gene and RBS. The first operon is regulated by tetracycline induced promoter Ptet and acts on <I>Zymomonas mobilis's</I> amino acid biosynthetic pathway to divert pyruvate to the amino acid intermediates. The second operon is regulated by constitutive strong promoter Peno or Pgap to enhance the expression of kdcA to decarboxylate 2-keto acids to aldehydes and further divert it to isobutanol by <I>Zymomonas mobilis's</I> own alcohol dehydrogenase (ADH)
This part is a isobutanol production module constructed in non-model organism <I>Zymomonas mobilis</I> and expected to be suitable for other microorganisms.<B>References:</B>    [1]Metabolic Engineering of Pichia pastoris for Production of Isobutanol and Isobutyl Acetate. W Siripong et al.  Biotechnol Biofuels (2018)

		</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>