<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUBU-Wuhan_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUBU-Wuhan/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class=""><UL class="nav"><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Collaborations">Collaborations</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Results">Results</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Parts">Part Table</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Improve">Improve</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Calendar">Calendar</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Protocol</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Intergrated_Human_Practices">Intergrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Public_Engagement">Education&amp; Engagement</A></LI><LI><A href="#">Contribution</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Model">Model</A></LI></DIV><SECTION class="story section" style="padding-bottom:30px;padding-top:150px;"><DIV class="section-title"><H2><SPAN>Design</SPAN></H2></DIV></SECTION><SECTION class="story section" style="padding-top:20px;"><DIV class="section-title"><P style="text-align: left;line-height:24px;padding:30px 0;">    On the one hand, convert waster cartons into glucose, then use the glucose as carbon source for microorganisms’ growth. On the other hand, construct a metabolic pathway in <I>Zymomonas mobilis</I> to produce isobutanol and optimize this pathway to be a isobutanol-producing module which could be compatible in other microorganisms.</P><H1 style="text-align: left;padding:60px 0;">Part 1. Conversion of Paper Cartons into Sugars</H1><P style="text-align: center;line-height:24px;padding:10px 0;">(from Liu Zilu web)</P><P style="text-align: left;line-height:24px;padding:30px 0;">    The primary ingredient of paper cartons is cellulose (ca 63%). Pretreatment using  H<SUB>2</SUB>SO<SUB>4</SUB> and the subsequent enzymatic hydrolysis will break down the recalcitrant structure of cartons to release the fermentable mono-sugars.</P><H1 style="text-align: left;padding:60px 0;">Part 2.Construction of Isobutanol-production Module</H1><P style="text-align: center;line-height:24px;padding:0 0;"><B>Fig. 1. </B>Strategy to build isobutanol production module.</P><P style="text-align: center;line-height:24px;padding:0 0;"><B>Fig. 2. </B>Isobutanol production module construction and test.</P><P style="text-align: left;line-height:24px;padding:30px 0;">    Construct a metabolic pathway to make <I>Zymomonas mobilis</I> produce isobutanol, on the basis of its own metabolic pathway. Screen different candidates of the key genes of the metabolic pathway and try different assembly to construct more optimized isobutanol-producing modules. And transform these modules to other microorganisms to test their efficiency and compatibility.</P><H1 style="text-align: left;padding:60px 0;">Part 3：Promotor optimizing</H1><P style="text-align: left;line-height:24px;padding:30px 0;">    We aim to find some promoters which can give priority to strain’s growth and make it begin fermentation later to produce isobutanol. In fact, <I>Zymomonas mobilis</I> is a capable ethanologenic bacterium with high ethanol productivity and ethanol tolerance. As ethanol accumulated, promoters get higher expression in stationary phase. </P><P style="text-align: left;line-height:24px;padding:30px 0;">    As a result, the strain grows and reproduces swiftly during the early exponential phase, consuming a small quantity of glucose, and then before the glucose run out, produce isobutanol at a relatively high rate. Let the strain regulates itself, like a automatic control system, so as to improve the isobutanol yield.</P><P style="text-align: left;line-height:24px;padding:30px 0;">    Candidate promoter regulated by cellular growth and/or ethanol concentration were selected based on previous systems biology studies, which were confirmed through dual report-gene system and FACS to balance cellular growth and isobutanol production. </P><P style="text-align: center;line-height:24px;padding:10px 0;">（from advisor Yongfu Yang）</P><P style="text-align: center;line-height:24px;padding:0 0;"><B>Table 1. </B>Identification of regulable promoters.</P><P style="text-align: left;line-height:24px;padding:30px 0;">    At the same time, the strong promoter Pgap in <I>ZM</I> was weakly expressed in E.coli. We constructed a Pgap mutant library by mutation and linking with the dual reporter system. The cell population with enhanced expression is then screened by cell sorting. The mutation site was identified by sequencing and further verified to obtain a Pgap mutant.</P><P style="text-align: left;line-height:24px;padding:30px 0;padding-bottom:150px;"><B>References:</B>    [1]Yang S, Pan C, Tschaplinski TJ, Hurst GB, Engle NL, et al. (2013) Systems Biology Analysis of Zymomonas mobilis ZM4 Ethanol Stress Responses.     [2]He M, Wu B, Shui Z, Hu Q, Wang W, et al.(2012) Transcriptome profiling of Zmomonas mobilis under ethanol stress.    [3]Alper H1, Fischer C, Nevoigt E, Stephanopoulos G. (2005) Tuning genetic control through promoter engineering.
		</P></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>