<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUBU-Wuhan_Calendar skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUBU-Wuhan/Calendar</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class=""><UL class="nav"><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan">Home</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Collaborations">Collaborations</A></LI></UL><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Results">Results</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Parts">Part Table</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Improve">Improve</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Calendar">Calendar</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Experiments">Protocol</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Intergrated_Human_Practices">Intergrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Public_Engagement">Education&amp; Engagement</A></LI><LI><A href="#">Contribution</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HUBU-Wuhan/Model">Model</A></LI></DIV><SECTION class="story section" style="padding-bottom:30px;padding-top:150px;"><DIV class="section-title"><H2><SPAN>Calendar</SPAN></H2></DIV></SECTION><SECTION class="story section" style="padding-top:20px;padding-bottom:150px;padding-left:15%;padding-right:15%;"><DIV class="story-content"><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">April</H3><P><B>Identifying the project, proposing the experimental plans, and divided the project into four experimental parts.</B></P><P class="p2">    <B>Part 1.</B> Conversion of Paper Cartons into Sugars    <B>Part 2.</B> Construction of Isobutanol-production Module    <B>Part 3.</B> Promotor optimization I    <B>Part 4.</B> Promotor optimization II (mutant Pgap)</P></DIV></DIV><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">May</H3><P><B>Part 1.</B></P><P class="p2">    Using the blander to cut up the waste carton, followed by treatment with alkali ( 12% NaOH solution : the waste carton = 1 : 15(mL:g),  55 ℃ , 150rpm , hydrolyzed 1.5 h )
				</P><P><B>Part 2. </B></P><P class="p2">    <B>1)</B>	Analyzing the growth curve of wild-type <I>Zymomonas mobilis</I> at different concentrations of isobutanol and tetracycline, and the tolerance of <I>Zymomonas mobilis</I> to isobutanol.    <B>2)</B>	Integrating the gene fragment Ptet-kdcA into the genome of wild-type <I>Zymomonas mobilis</I>, then analyzing growth, glucose consumption, and the production of ethanol and isobutanol of the recombinant strain.     <B>3)</B>	Identifying the gene fragments of ALS and BDH from database, obtaining 5 ALS and 3 BDH genes.
				</P></DIV></DIV><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">June</H3><P><B>Part 1.</B></P><P class="p2">    <B>1)</B>	Using the blander to cut the waste carton into pieces, followed by treatment with acid ( 3% H2SO4  solution : the waste carton = 1：15(mL:g),  55 ℃ , 150rpm , hydrolyzed 1.5 h). After hydrolysis, the solution pH was adjusted to 4.5, which is the most suitable pH for the cellulase used in our project.    <B>2)</B>	After comparing alkaline hydrolysis with acid hydrolysis, we found that more raw materials were wasted after the alkaline hydrolysis. Therefore, the acid hydrolysis strategy was chosen to be used in our project.
				</P><P><B>Part 2. </B></P><P class="p2">    <B>1)</B>	Integrating the gene fragment Peno-BsAls2-pEZ15A, Peno-EcAls-pEZ15A and Ppdc-EcBDHopt-pEZ15A into the genome of <I>Zymomonas mobilis</I> containing the Ptet-kdcA by electro-transformation. The integration of Peno-BsAls2-pEZ15A failed.    <B>2)</B>	Analyzing the growth curve of these two kinds of <I>Zymomonas mobilis</I>.
				</P></DIV></DIV><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">July</H3><P><B>Part 1.</B></P><P class="p2">    <B>1)</B>	The waste cartons were treated with acid hydrolysis.     <B>2)</B>	Adding different amounts of cellulase to facilitate the hydrolysis. (50℃, 150rpm, hydrolyzed 36 h)    <B>3)</B>	Comparing the glucose production under different conditions.
				</P><P><B>Part 2. </B></P><P class="p2">    <B>1)</B>	Testing the glucose consumption of the recombinant <I>Zymomonas mobilis</I>, which were obtained in June. (Some data is abnormal due to mechanical failure.)electro-transformation. The integration of Peno-BsAls2-pEZ15A failed.    <B>2)</B>	Through data comparison, BsAls2 strain was selected for the subsequent experiments.    <B>3)</B>	Determined how to combine the genetic modules for isobutanol production.
				</P><P><B>Part 3. </B></P><P class="p2">    Designed the primers V1 for the new promoters.    <B>1)</B>	We found 11 promoters can be stronger or weaker due to the concentration of ethanol. And these are their code name : 0037, 0038, 0101, 0203, 0309, 0405, 0435, 0443, 1039, 1029 and 1838.    <B>2)</B>	We designed the V1 primers for these new promoters.
				</P></DIV></DIV><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">August</H3><P><B>Part 1.</B></P><P class="p2">    <B>1)</B>	Treating the waste cartons with acid hydrolysis.    <B>2)</B>	Adding different amounts of cellulase to facilitate the hydrolysis. (50℃, 150rpm, hydrolyzed 36 h)    <B>3)</B>	After the enzymatic hydrolysis, the samples were divided into two parts for different treatment. One part was filtered and the other part is not processed with any further treatments.     <B>4)</B><I>Zymomonas mobilis</I> was added into the samples in 3), and the ethanol production was compared. No difference of the ethanol production between these two samples were detected, so filter sterilization was not a necessary step after the cellulase treatemnt.    <B>5)</B>	At the same time, we found that the yield of liquid enzymes(the first kind of cellulase from WUST Doctor Gong) is lower. So we chose to use the solid enzyme(the another kind of cellulase from Doctor Yi).

				</P><P><B>Part 2. </B></P><P class="p2">    <B>1)</B>	Cloning the fragments of Peno, kdcA and Ptet-BsAls2-ilvc-ilvd into pUC57 plasmid to obtain the recombinant plasmids Peno-kdcA-puc57 and Ptet-BsAls2-ilvc-ilvd-puc57.    <B>2)</B>	Attempted to generate recombinant plasmids Peno-kdcA-pEZ15A, Ptet-BsAls2-ilvc-ilvd-pEZ15A, and Peno-kdcA-Ptet-BsAls2-ilvc-ilvd-pEZ15A, but not successfully.    <B>3)</B>	Modified the restriction sites RFC 10 in fragment Ptet-BsAls2-ilvc-ilvd and Dual-Luciferase Reporter Assay System.

				</P><P><B>Part 3. </B></P><P class="p2">    <B>1)</B>	Amplifying the promoters for the project.     <B>2)</B>	We did many times of PCR for these 11 promoters.                 Aug.7  Got promoters 405, 435, 1029                Aug.8  Got promoters 443, 1838                Aug.10  Got promoters 0037, 0038, 0101, 0203                Aug.11  Got promoter 0309                Aug.13  Got promoter 1039        a.　Connected the plasmid to the Dual-Luciferase Reporter Assay System by T5 transformation. Cultured them on the plates. Did PCR with the colonies from the plates.        b.　Extracted these 11 plasmids. Sequenced them.        c.　Made competent cells of <I>Zymomonas mobilis</I>.        d.　Got 11 kinds of <I>Zymomonas mobilis</I> by electrotransformation.(1039 had failed.)          Got 11 kinds of <I>E.coli</I> T1 by T5 transformation.        e.　We used flow cytometer to test the fluorescence expression of the strains, and judge the strength of their regulation by the intensity of fluorescence.</P><P><B>Part 4. </B></P><P class="p2">    <B>1)</B>	A mutant library of Pgap was generated using sequential error-Prone PCR. Then, this library was cloned into Dual-Luciferase Reporter Assay System, which was transformed in the <I>E.coli</I> T1 cells.    <B>2)</B>	The mutant library (in <I>E.coli</I> T1) was analyzed and sorted using flow cytometry, and the mutant with enhanced Pgap was obtained.

				</P></DIV></DIV><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">September</H3><P><B>Part 2. </B></P><P class="p2">    <B>1)</B>	1<SUP>st</SUP> week : 4 recombinant plasmids, including Peno-kdcA-pEZ15A, Peno-kdcA-pSB1C3, Pgap-kdcA-pEZ15A and Pgap-kdcA-pSB1C3, were obtained.    <B>2)</B>	2<SUP>nd</SUP> week : 3 recombinant plasmids, including Ptet-BsAls2-ilvc-ilvd-pSB1C3, Ptet-BsAls2-ilvc-ilvd-Peno-kdcA-pSB1C3, and Ptet-BsAls2-ilvc-ilvd-Pgap-kdcA-pSB1C3, were obtained.    <B>3)</B>	3<SUP>rd</SUP> week : 2 recombinant plasmids, including Ptet-BsAls2-ilvc-ilvd-Peno-kdcA-pEZ15A and Ptet-BsAls2-ilvc-ilvd-Pgap-kdcA-pEZ15A, were obtained. And the recombinant plasmids, Peno-kdcA-pEZ15A and Pgap-kdcA-pEZ15A were transformed into <I>Zymomonas mobilis</I>.    <B>3)</B>	4<SUP>th</SUP> week :  Attempted to generate the recombinant plasmid, Ptet-BsAls2-ilvc-ilvd-pEZ15A, but failed.

				</P><P><B>Part 3. </B></P><P class="p2">    <B>1)</B>	1<SUP>st</SUP> week and 2<SUP>nd</SUP> week : The fluorescence expression of the strains we got next month under different conditions, such as ethanol induction and ethanol inhibition, were evaluated by flow cytometry. And 3 promoters, 0101, 0405, 0435 with the highest fluorescence were selected.     <B>2)</B>	3<SUP>rd</SUP> week and 4<SUP>th</SUP> week : Cloned the promoters in the projects into the plasmid pSB, followed by sequencing verification. 

				</P><P><B>Part 4. </B></P><P class="p2">    <B>1)</B>	Sorted by flow cytometry to obtain individuals with enhanced effects two times more. After verification by PCR, found 4 kinds of mutants.     <B>2)</B>	Sorted again and found 7 kinds of new mutants.     <B>3)</B>	We got 11 mutants together. Transformed them into E.coli. Used flow cytometer to test the fluorescence expression of these strains.    <B>4)</B>	Got 11 kinds of <I>Zymomonas mobilis</I> by electrotransformation,and used flow cytometer to test the fluorescence expression of these strains to show strength of different Pgap in <I>Zymomonas mobilis</I>.

				</P></DIV></DIV><DIV class="single-story"><SPAN>2018</SPAN><DIV class="inner-content"><H3 style="color:#00CC00;">October</H3><P><B>Part 1. </B></P><P class="p2">    Evaluating the isobutanol production by the recombinant <I>Zymomonas mobilis</I>.

				</P><P><B>Part 2. &amp; Part 3. &amp;Part 4.</B></P><P class="p2">    <B>1)</B>	Cloned the promoters in the projects into the plasmid pSB.    <B>2)</B>	Cloned the fragments in the projects into the plasmid pSB. And we got 4 new plasmids with the parts which can product isobutanol.    <B>3)</B>	Finished the uploading.    <B>4)</B>	Cloned the fragments(from Part 2.) in the projects into the plasmid pEZ15A. And we got 5 new plasmids. Transformed them into <I>Zymomonas mobilis</I> and <I>E.coli</I>.     <B>5)</B>	The <I>E.coli</I> we got can not product isobutanol. And only two plasmids, pEZ15A-Peno-kdcA, pEZ15A-Pgap-kdcA, had succeeded. So we tested their growth and the production of isobutanol.

				</P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>