<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kent_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kent/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="nav"><LABEL for="show-menu" class="show-menu">Show Menu</LABEL><DIV class="nav_hide"><DIV class="nav_menu_item hvr-fade"><DIV class="nav_link"><A href="https://2016.igem.org/Team:Kent">Home</A></DIV></DIV><DIV class="nav_menu_item hvr-fade"><LABEL for="toggleTeam" class="toggle_nav nav_link hvr-fade">Team</LABEL><DIV class="nav_submenu" id="toggleTeam_submenu"><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Team">Meet the team</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Attributions">Attributions</A></DIV></DIV></DIV><DIV class="nav_menu_item hvr-fade"><LABEL for="toggleProject" class="toggle_nav nav_link hvr-fade">Project</LABEL><DIV class="nav_submenu" id="toggleProject_submenu"><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Description">Description</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Experiments">Experiments</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Results">Results</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Safety"> Safety </A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Achievements">Achievements</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://igem.org/2016_Judging_Form?id=1985">Judging</A></DIV></DIV></DIV><DIV class="nav_menu_item hvr-fade"><LABEL for="toggleParts" class="toggle_nav nav_link hvr-fade">Parts</LABEL><DIV class="nav_submenu" id="toggleParts_submenu"><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Parts">Overview</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Basic">Basic</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Composite_Part">Composite</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Part_Collection"> Collection</A></DIV></DIV></DIV><DIV class="nav_menu_item hvr-fade"><LABEL for="togglePractices" class="toggle_nav nav_link hvr-fade"><A href="https://2016.igem.org/Team:Kent/Human_Practices">Practices</A></LABEL><DIV class="nav_submenu" id="togglePractices_submenu"><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Human_Practices">HP</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Collaborations">  Collaborations </A></DIV></DIV></DIV><DIV class="nav_menu_item hvr-fade"><LABEL for="toggleModelling" class="toggle_nav nav_link hvr-fade">Modelling</LABEL><DIV class="nav_submenu" id="toggleModelling_submenu"><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Model">Introduction</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Algorithm">Algorithm</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/Methodology">Methodology</A></DIV><DIV class="nav_menu_subitem hvr-fade"><A href="https://2016.igem.org/Team:Kent/ModelResults">Results</A></DIV></DIV></DIV><DIV class="nav_menu_item hvr-fade"><DIV class="nav_link"><A href="https://2016.igem.org/Team:Kent/Notebook">Notebook</A></DIV></DIV></DIV></DIV><DIV class="nav_replace">
    Navigation
  </DIV><UL id="accordion" class="top_menu"><LI class="menu_item"><A href="https://2016.igem.org/Team:Kent">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Kent/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Kent/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Kent/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Kent/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Kent/Attributions"> ATTRIBUTIONS </A></LI><LI class="menu_item"> PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Kent/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Kent/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Kent/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Kent/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Kent/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><DIV class="subs"><H1 class="first_title">Notebook</H1><H2 style="color:black">This page contains the documentation for our Mag-nano-tite Project. </H2><H2 style="color:black">It will be split into 6 sections containing the work done  through the duration of our project.</H2></DIV><DIV class="seasons"><DIV style="width: 90%; margin: auto;"><DIV class="hovereffect"><DIV class="overlay"><H2>Pre - June</H2><A class="modal_link info" id="season1a" href="#season1">Click here</A></DIV></DIV></DIV><DIV id="season1"><DIV class="close-season1"><B>X</B></DIV><DIV class="modal-content"><H1>Pre-June</H1><H2>February</H2><UL><LI>Team Kent for iGEM 2016 was formed.</LI><LI>We had our first iGEM meeting where we got acquainted with some of the team members.</LI><LI>We also met with our team supervisor</LI><LI>The iGEM timeline was discussed and key dates highlighted.
     </LI><LI>October 27-31 – iGEM Jamboree</LI><LI>Submissions deadline – 21 st October 2016</LI><LI>October 19 – Wiki-freeze</LI></UL><LI>We discussed our aims and goals for our project.</LI><LI>Specific medal criteria – Bronze, Silver, <B>Gold</B></LI><LI>A team organiser and social media officer was appointed.</LI><H2>March</H2><UL><LI>Our iGEM team was complete 10 students, 2 supervisors and 3 advisors.</LI><LI>We completed our official iGEM registration individually and as a team.</LI><LI>We began establishing our plans for the summer.
    </LI><LI>We were appointed a space in the bioscience lab.</LI><LI>We would have weekly meeting from the start of our summer projects.</LI></UL><LI>We began brainstorming for our project.
    </LI><LI>Problem based approach - What problems needed to be addressed?</LI><LI>Are we inventing something novel? Are we improving something that has already been done?</LI><LI>How can we approach a problem that has already been addressed to get a different response?</LI><LI>Is this project feasible in the time available?</LI><LI>Is there a sensible approach for validation?</LI><LI>How will we explore this project?</LI><LI>We viewed the track selections for iGEM
    </LI><LI>Which track would our project lay in?</LI><LI>Tracks
    </LI><LI>Diagnostics</LI><LI>Energy</LI><LI>Environment</LI><LI>Food and nutrition</LI><LI>Information processing</LI><LI>Manufacturing</LI><LI>Therapeutics</LI><LI>Art and design</LI><LI>Hardware</LI><LI>Measurement</LI><LI>Software</LI><LI>We began exploring previous iGEM projects by looking at previous team wikis.</LI><LI>Our ideas
    </LI><LI>Bacterial Batteries</LI><LI><B>Magnetic bacteria
        </B></LI><LI>To conduct energy</LI><LI>Combat heavy metal poisoning</LI><LI>Lead</LI><LI>CRISPR – specific alteration of genomes</LI><LI>Material formation – with applications to fibre formation (continuation of nanowire project) (fibre formation an area of Dr Xue’s expertise)</LI><LI>Magnetic nanowires to conduct electricity?</LI><H2>April</H2><UL><LI>The brainstorming continued.
	</LI><LI>Bacteria that deals with free radicals
		</LI><LI>Fluorescent proteins that detect the radicals</LI></UL><LI>Fluorescent proteins
		</LI><LI>For detection</LI><LI>Bacteria which grows hair		
		</LI><LI>Clear applications to this</LI><LI>Swimming bacteria</LI><LI>Protein cages for drugs (be transported through Tat pathway)</LI><LI>Protein sculptures – visual</LI><LI>We re-visited the idea of magnetic bacteria.</LI><LI>We started researching magnetic bacteria species.</LI><H2>May</H2><UL><LI>We continued brainstorming ideas
</LI><LI>Solar panels
	</LI><LI>Harnessing photosynthetic capability of bacteria</LI><LI>Phylum cyanobacteria (slow)</LI><LI>Researchers are thinking of modifying the bacteria</LI><LI><U>Current problems:</U></LI><LI>Removal of waste material (solar panel practically inaccessible after installation)</LI></UL><LI>Efficiency of energy conversion, only 8% when solar panel gives 40%</LI><LI>Evolved to protect themselves from full sunlight, hence, low efficiency</LI><LI>Bacteria antimicrobial peptides (protein targeting)
	</LI><LI>To combat infections</LI><LI>Here, the protein is the drug</LI><LI>Bacteria preventing the growth of other bacteria</LI><LI>Exam season!</LI><LI>After months of brainstorming it was time to make a decision.</LI><LI>We revisited an idea we had been discussion from our first brainstorming session.</LI><LI><B>Magneto bacteria
</B></LI><LI>It has been ‘tricked’ to bio-mineralizing crystals is which iron was replace with other metals</LI><LI>By growing the bacteria in a solution with a high concentration of both metals</LI><LI>Shown that it can incorporate manganese into the crystal structure of magnetite</LI><LI>Some success with cobalt</LI><LI><U>Uses</U>: waste removal or drug delivery</LI><LI>We started researching relevant literature relating to this topic
</LI><LI>What are the key proteins? Can we obtain them?</LI><LI>Can it be incorporated into E.coli</LI><LI>What are the applications.</LI><LI>We created a google drive where would feed in information throughout our project.</LI></DIV></DIV><DIV style="width: 90%; margin: auto;"><DIV class="hovereffect"><DIV class="overlay"><H2>June</H2><A class="modal_link info" id="season2a" href="#season2">Click here</A></DIV></DIV></DIV><DIV id="season2"><DIV class="close-season2"><B>X</B></DIV><DIV class="modal-content"><H1>June</H1><H2> May 31- June 5</H2><UL><LI>We began to gather scientific papers (PubMed, WebofKnowledge, Science Direct) to post on to
our shared folder.
  </LI><LI>We explored the iGEM archives for similar ideas identify Bio-bricks (Mam proteins?)
  </LI><LI>Identify key words to search magnetosomes, MMS6 protein, nano-particles.
</LI></UL><H2>June 6 - June 12</H2><UL><LI>We began to plan how we would conduct our experiments.
   </LI><LI>What genes are relevant?
     </LI><LI>Can we obtain them
     </LI><LI>Do we need specific strains of E. coli?
     </LI><LI>Can we purchase these strains?
     </LI><LI>If not who do we speak to, to obtain these strains?
     </LI><LI>What safety risks do these pose?
   </LI></UL><H2>June 13 - June 19</H2><UL><LI>We began to obtain sequences we would use.
  </LI><LI>It was important here to start to identify the experiments we were going to use.
</LI></UL><H2>June 20 - June 26</H2><UL><LI>We continue reading through and understanding the relevant literature.
   </LI><LI>We began to understand our safety implications of our project.
</LI></UL></DIV></DIV><DIV style="width: 90%; margin: auto;"><DIV class="hovereffect"><DIV class="overlay"><H2>July</H2><A class="modal_link info" id="season3a" href="#season3">Click here</A></DIV></DIV></DIV><DIV id="season3"><DIV class="close-season3"><B>X</B></DIV><DIV class="modal-content"><H1>July</H1><H2>June 27 th – 3 rd July (Week 1)</H2><H2>Day 1</H2><UL><LI>We received our lab induction. We received training on how to use unfamiliar equipment.</LI><LI>Having already researched the different protocols we would be using during our project we conducted risk assessments.</LI><LI>We received training on the correct use and disposal of waste and materials.</LI><LI>We were appointed different plasmids:
</LI><LI>pET3a mamX/mamT/mamP</LI><LI>pET23B SecS mamX/ mamT/ mamP</LI><LI>pSB1A3 RFP</LI><LI>pEC86</LI><LI>pSB1c3 RFP</LI><LI>pET3a</LI><LI>pSB1C3</LI></UL><LI>We began writing our project descriptions for the upcoming deadline.</LI><LI>We set up our social media accounts: Twitter,</LI><LI>Facebook and Instagram.</LI><H2>Day 2</H2><UL><LI>We made LB plates and broth ensuring to add the relevant antibiotics (ampicillin or chloramphenicol).</LI></UL><H2>Day 3</H2><UL><LI>Parts from our distribution kits were re-suspended.</LI><LI>Our plasmids were transformed in top10 cells and left overnight for 15-18hours incubation.</LI></UL><H2>Day 4</H2><UL><LI>Re-streaks were performed on successful transformations.</LI><LI>Unsuccessful transformations were repeated.</LI><LI>The writing of our project was continued.</LI><LI>Registration for a student account for Enthought Canopy,</LI><LI>Tutorials were completed on TutorialSpoint/Python</LI><LI>We attempted to ‘scrape’ one iGEM wiki</LI></UL><H2>July 4 th – July 10 th (Week 2)</H2><H2>Day 5</H2><UL><LI>Liquid cultures were prepared frim our transformed cells.</LI><LI>Using SnapGene we carried out plasmid map analysis using Snapgene to identify compatible restriction sites on our individual plasmids.</LI></UL><H2>Day 6</H2><UL><LI>Our plasmids were purified from the overnight liquid cultures.</LI><LI>The concentration of the DNA of our individual plasmids were determined.</LI><LI>Restriction enzyme digests were conducted based on the restriction sites identified on the plasmid maps including the appropriate buffer.
</LI><LI>pET3a – Xbal, HindIII, promega E</LI><LI>pET3a mamP – Xbal, EcoRI, promega H</LI><LI>pET3a mamT – Xbal, HindIII, promega E</LI><LI>pET3a mamX – Xbal, HindIII, promega E</LI><LI>pET23b mamP – NdeI, EcoRV, promega D</LI><LI>pET23b mamT – XbaL, PstI, promega H</LI><LI>pET23b mamX – NdeI, EcoRV, promega D</LI><LI>pET23b mamP – NdeI, EcoRV, promega D</LI><LI>pEC86 – Ndel, EcoRV, promega D</LI></UL><H2>Day 7</H2><UL><LI>Agarose gels were conducted to validate our various plasmids.</LI><LI>We uploaded our project description onto our wiki</LI><LI>Gels were imaged.</LI></UL><H2>Day 8</H2><UL><LI>Troubleshooting began who electrophoresis gels which were unsuccessful.</LI></UL><H2>Day 9</H2><UL><LI>pSB1C3 RFP and pSB1A3 RFP plasmids were validated.</LI></UL><H2>July 11 th – July 17 th (Week 3)</H2><H2>Day 10</H2><UL><LI>Gel extraction protocols were used to purify validated plasmids.</LI><LI>Liquid cultures were prepared to form glycerol stocks from validated plasmids.</LI><LI>We prepared our outreach presentations for Simon</LI><LI>Langton Grammar School for Boys Canterbury.</LI></UL><H2>Day 11</H2><UL><LI>We successfully gave outreach presentations at Simon</LI><LI>Langton Grammar School for Boys Canterbury.</LI><LI>Two presentations delivered to physics and biology classes respectively.</LI><LI>Questions from both physics classes were documented.</LI></UL><H2>Day 12</H2><UL><LI>We continued with the validation of our individual plasmids with the aim to validate all our plasmids.</LI><LI>Gel electrophoresis gels were continued.</LI><LI>Plasmids pET3a and pET23b sec.S mamT/ mamP/ mamX were validated.</LI></UL><H2>Day 13</H2><UL><LI>Plasmids pET3a and pET23b sec.S mamT/ mamP awere transformed into BL21 cells.</LI><LI>Liquid cultures were prepared from the Top10 constructs of Plasmids pET3a and pET23b SecS mamT/ mamP</LI></UL><H2>Day 14</H2><UL><LI>Glycerol stocks were made for pet3aMamT and psb1c3, pSB1C3 RFP and pSB1A3 RFP.</LI><LI>Liquid cultures were made for these plasmids</LI><LI>Plasmid purification was done for the necessary plasmids to build up the DNA stocks </LI></UL><H2>July 18 th – July 24 th (Week 4)</H2><H2>Day 15</H2><UL><LI>Liquid cultures from BL21 cells containing SecS.MamX/MamT/MamP plasmids.</LI><LI>Using SnapGene we looked at primer sequences in preparation for PCR.</LI><LI>We began brainstorming for the iGEM meet up at the University of Westminster on August 17-19 th</LI></UL><H2>Day 16</H2><UL><LI>We gathered detailed the sequences of the Mam proteins using their respective SnapGene</LI><LI>We ensure that we validated the sequences UniProt data and found the correct open reading frames.</LI><LI>We began growing up liquid cultures in preparation for the purification of our proteins MamP and MamT.</LI></UL><H2>Day 17</H2><UL><LI>We finished gathering possible primer sequences for our Mam proteins.</LI><LI>We began protein preparations for protein expression and purification.</LI><LI>MamP, MamX and MamT grown in</LI></UL><H2>Day 18</H2><UL><LI>Buffers were prepared.
</LI><LI>Lower Tris buffer prepared using: 1.5M Tris, 0.4% SDS, pH 8.8</LI><LI>Upper Tris buffer prepared using: 1M Tris, 0.4% SDS, pH 6.8</LI><LI>Tris-Glycine running buffer prepared – 30g Tris base, 144g Glycine, 150ml SDS (1.5% w/v)</LI></UL><LI>Liquid cultures from BL21 cells were made from transformed MamX, MamT and MamP cells and incubated overnight.</LI><H2>Day 19</H2><UL><LI>SDS page gels was made.</LI><LI>Samples taken from the liquid cultures were boiled to denatured protein.</LI><LI>Samples obtained were run on an SDS-page gel.</LI><LI>The gel was stained with comassie blue over the duration of the weekend.</LI></UL><H2>July 25 th – July 31 st (Week 5)</H2><H2>Day 20</H2><UL><LI>We decided on the team name ‘Mag-nano-tite’</LI><LI>A collaboration was formed between Team Kent and UC Davis</LI><LI>We wrote code for a search bar on iGEMMatch.org</LI><LI>SDS PAGE gel de-stained using methanol on shaker for ~3 hours.</LI><LI>Gel was imaged and proved unsuccessful.</LI><LI>Troubleshooting began.</LI></UL><H2>Day 21</H2><UL><LI>Continued work on ideas/concepts for the upcoming Westminster presentation and poster.</LI><LI>Nickel column B and wash buffers were made and set to the correct pH.</LI><LI>overnight liquid</LI><LI>Liquid cultures were also prepared from SecS.MamX, MamP, MamT and pET3a Top10 cells.</LI></UL><H2>Day 22</H2><UL><LI>The plasmids were purified from the liquid cultures made the previous day and the concentrations determined</LI><LI>We contacted the iGEM team at the University of Westminster team regarding the meet up and continued work to the presentation.</LI></UL><H2>Day 23</H2><UL><LI>we conducted an inventory to determine which plasmids still required transformation, or repeats.</LI><LI>pEC86 was transformed into BL21 cells.</LI></UL><H2>Day 24</H2><UL><LI>We attempted co-transformation into BL21 cells using plasmids secS mamP &amp;amp; pec86) and incubated over the weekend.</LI></UL></DIV></DIV><DIV style="width: 90%; margin: auto;"><DIV class="hovereffect"><DIV class="overlay"><H2>August</H2><A class="modal_link info" id="season5a" href="#season5">Click here</A></DIV></DIV></DIV><DIV id="season5"><DIV class="close-season5"><B>X</B></DIV><DIV class="modal-content"><H1>August</H1><H2>August 1st – August 7th (Week 6)</H2><H2>Day 25 </H2><UL><LI>We began preparing for the iGEM deadline on August 12th.</LI><LI>We began compiling information for our abstract</LI><LI>Co-transformed plasmids pEC86 + SecS MamP were made into BL21 DE3 competent cells.</LI><LI>SecS MamP were transformed in BL21 in DE3 competent cells. </LI></UL><H2>Day 26</H2><UL><LI>SecS MamX were transformed in BL21 in DE3 competent cells. </LI><LI>We created a code which measures the frequency of a key word for our Topic modelling programme. </LI></UL><H2>Day 27 </H2><UL><LI>Competent cells were created for pEC86 and SecS MamX were, measured into 0.5mL aliquoted and stored in the -80°C deep freezer.</LI><LI>We received our primers,</LI></UL><H2>Day 28 </H2><UL><LI>The primers were re-suspended.</LI><LI>We carried out our first PCR using the Mam genes (SecS and insoluble) to amplify our DNA.</LI><LI>The PCR products were purified.</LI></UL><H2>Day 29</H2><UL><LI>An electrophoresis gel was conducted to validate the PCR products. This gel was unsuccessful, and the PCR was repeated adjusting the temperatures in a bid to correct the errors from the first PCR.</LI><LI>Prepared Mam proteins were analysed using an SDS-page gel. They were stained using coomasie-blue for the duration of the weekend </LI></UL><H2>August 8th – August 14th (Week 7)</H2><H2>Day 30 </H2><UL><LI>SDS PAGE gels were de-stained; resulting image shown below.</LI><LI>The team delivered a mock presentation in preparation for Westminster conference.</LI><LI>Final adjustments were made for iGEM deadline for track Selections and project abstract</LI></UL><H2>Day 31 </H2><UL><LI>Mam proteins were isolated using His-tag purification. </LI><LI>After a third failed attempt to gain the correct PCR products, PCR was repeated using different enzymes. </LI><LI>Plasmid maps were view to select new primers.  primers. </LI></UL><H2>Day 32</H2><UL><LI>pET3a plasmid validated. </LI><LI>We had a meeting with team and supervisors to discuss direction of our project and set targets to achieve Gold medal criteria</LI><LI>PCR was repeated using new primers</LI></UL><H2>Day 33</H2><UL><LI>We began researching the literature relating to the chemical synthesis of magnetic nano-particles.</LI><LI>We spoke to a chemical expert in the field of nano-particles.</LI><LI>We liaised with teams for a possible collaboration. </LI></UL><H2>Day 34</H2><UL><LI>iGEM deadline met, abstract submitted and track selection completed – New Applications. </LI><LI>We continued sourcing the materials for the chemical synthesis of chemical nano-particles.</LI><LI>Preparation for the Westminster meet-up continued. </LI></UL><H2>August 15th – August 21st (Week 8)</H2><H2>Day 35</H2><UL><LI>We began working on an outline for our wiki page. </LI><LI>A Gibson assembly was performed and fragment sup35nm was re-suspended. </LI><LI>We finalised our presentations and posters for the University of Westminster conference. </LI></UL><H2>Day 36 </H2><UL><LI>We started identifying improvements we could make of the previous years’ project.</LI><LI>Liquid culture was prepared from pEC86 + MamP and MamT</LI><LI>We observed that the induced liquid culture when spun down was red in colour likely due to the cytochrome C domain in the protein binding iron.</LI></UL><H2>Day 37 </H2><UL><LI>We developed our Topic modelling programme to read multiple text files.</LI><LI>Members of the iGEM team presented our poster at the University of Westminster iGEM conference</LI><LI>Using nickel ion chromatography, we purified out MamX, MamT, MamP proteins </LI></UL><H2>Day 38</H2><UL><LI>Members of the iGEM team gave a spoken presentation at the University of Westminster iGEM conference</LI></UL><H2>Day 39</H2><UL><LI>We began compiling our data, results in preparation for our wiki. </LI></UL><H2>August 22nd – August 28th (Week 9)</H2><H2>Day 40</H2><UL><LI>We reflected on the presentations given at the Westminster </LI><LI>Gibson assemblies were performed.</LI><LI>We double digested pSB1C3 and ran a gel to validated,</LI><LI>We transformed NEB 5-alpha competent E. coli cells with 2ul of the Gibson reaction mixture.</LI></UL><H2>Day 41</H2><UL><LI>Colonies from plates were Re-streaked some white colonies and made liquid cultures.</LI></UL><H2>Day 42</H2><UL><LI>Re-streaks showing white colonies were mini-prepped.</LI><LI>QIAprep protocol to purify the plasmids. </LI></UL><H2>Day 43</H2><UL><LI>We focused on dry lab tasks such as planning the mock questions for future presentations.</LI></UL><H2>Day 44</H2><UL><LI>We continued compiling information the wiki</LI><LI>Planned future experiments </LI></UL></DIV></DIV><DIV style="width: 90%; margin: auto;"><DIV class="hovereffect"><DIV class="overlay"><H2>September</H2><A class="modal_link info" id="season6a" href="#season6">Click here</A></DIV></DIV></DIV><DIV id="season6"><DIV class="close-season6"><B>X</B></DIV><DIV class="modal-content"><H1>September</H1><H2>August 29 th – September 4 th (Week 10)</H2><H2>Day 45</H2><UL><LI>The AFM imaging of samples commenced.</LI><LI>Double digest of PSB1C3 RFP with Sup35nm using Ecor1 and Pst1</LI><LI>PCR using usual protocol (with double digested template DNA) and original long primer on pET23B SecS genes.</LI><LI>Prepared 500ml LB in 2L Baffled flasks for protein expression</LI></UL><H2>Day 46</H2><UL><LI>Repeated agarose gels</LI><LI>New forward primers arrived therefore PCR was done using normal protocol (with annealing temp at 52 degree and double digested template DNA).</LI><LI>We set up 4ml overnight liquid cultures containing double antibiotics, 2 for each Pet23b Sec.S MamT, MamP, and MamX with pEC86 using the same colony.</LI><LI>Made fresh antibiotic samples.</LI></UL><H2>Day 47</H2><UL><LI>Made 1ml glycerol stocks using 500ul of liquid culture for Sec.S MamP, MamT and MamX with Pec86 in BL21’s.</LI><LI>Using the remaining 3.5ml to inoculate 500ml LB in the Baffled flasks.</LI><LI>We grew up the cells and induced with IPTG for protein expression.</LI><LI>Ligation and transformations carried out.</LI></UL><H2>Day 48</H2><UL><LI>We finalised our plans for collaboration with UEA over skype conference. It was agreed that we would carry out some AFM images of their S. oneidensis cells on a variety of different surfaces.</LI><LI>SDS gels carried for MamP SecS, MamX SecS, MamT SecS and stained overnight.</LI></UL><H2>Day 49</H2><UL><LI>De-strained SDS page gels and imaged them.</LI><LI>In preparation, we shipped off AFM sample discs to UEA in Norwich in preparation for collaboration the following week.</LI></UL><H2>September 5 th - September 11 th (Week 11)</H2><H2>Day 50</H2><UL><LI>We prepared liquid cultures with all Mam pET23B genes.</LI><LI>Using plasmid maps we designed new primers.</LI></UL><H2>Day 55</H2><UL><LI>Mini-prepped liquid cultures and ran undigested verification gels</LI><LI>Re-attempted PCR</LI><LI>Web design progress for our wiki.</LI><LI>We used the absorbance reading at 280nm and extinction coefficient 8.73 mM -1 cm -1 to estimate the concentration of mam P as 27.4 µM.</LI><LI>We used the absorbance reading at 280 and extinction coefficient 27.32 mM -1 cm -1 to estimate the concentration of mam P as 39.2 µM</LI></UL><H2>Day 56</H2><UL><LI>Using SnapGene plasmid map sequences we designed new SecS primers</LI><LI>We carried out spin columns for protein purification of mamP/T/X</LI><LI>Using the absorbance reading at 280 nm and extinction coefficients, we estimated the concentrations for mamP/X/T for subsequent anaerobic reaction vessels.</LI><LI>Carried out the reaction using purified proteins with iron solutions within anaerobic reaction vessel.</LI></UL><H2>Day 57</H2><UL><LI>Reactions between purified proteins and iron solutions were placed onto EM grids and prepared for imaging (in vivo).</LI></UL><H2>September 12 th – September 18 th (Week 12)</H2><UL><LI>The team logo and banner were designed for review by the team.</LI><LI>We collaborated UEA to AFM image with samples.</LI><LI>We continued making our final constructs.</LI></UL><H2>September 19 th - September 25 th (Week 13)</H2><UL><LI>The team logo and banner were finalised.</LI><LI>We continued writing</LI><LI>AFM imaging of our samples continued.</LI></UL></DIV></DIV><DIV style="width: 90%; margin: auto;"><DIV class="hovereffect"><DIV class="overlay"><H2>October</H2><A class="modal_link info" id="season4a" href="#season4">Click here</A></DIV></DIV></DIV><DIV id="season4"><DIV class="close-season4"><B>X</B></DIV><DIV class="modal-content"><H1>October</H1><H2>September 26th – October 2nd </H2><UL><LI>Semester resumes</LI><LI>We began planning our outreach presentation for the University of Kent summer student Bioscience Symposium.</LI><LI>We prepared a presentation using updated information about our project and updated results </LI><LI>We continued imaging our samples to validate our constructs, these constructs include: pSB1A3-[AraC- pBAD]-[CsgA- SS-Sup35NM- 1-61], pSB1A3-[AraC- pBAD]-[CsgA- SS-Sup35NM- 1-61- Cytb562], pSB1A3-[AraC- pBAD]-[CsgA- SS-Sup35NM]</LI></UL><H2>October 3rd – October 9th </H2><UL><LI>We continued writing up our results and compiling information for our wiki.</LI><LI>The team attended a University of Kent Bioscience open day. We spoke to the public about our iGEM project, the iGEM competition and its future applications.</LI><LI>We continued the AFM imaging of samples.  </LI><LI>The final construct of pSB1A3 mamOPXT transformed.</LI></UL><H2>October 10th – October 16th </H2><UL><LI>pSB1A3 mamOPXT</LI><LI>We continued the AFM imaging of our samples.</LI><LI>E. coli cells were induced containing the mamOPXT plasmid, we did this by growing the cells in a liquid media containing arabinose, IPTG and Ferric Citrate.</LI><LI>Electron microscopy grids prepped for imaging.</LI><LI>Writing for wiki continued.</LI></UL><H2>October 17th – October 23rd</H2><UL><LI>The team Completed content for our wiki </LI><LI>Wiki freeze.</LI><LI>Final electron microscopy images produced.</LI><LI>Prepared our posters, banner and presentation for the jamboree </LI><LI>Outreach at second University of Kent open day. </LI></UL><H2>October 24th – October 31st</H2><UL><LI>iGEM Jamboree in Boston.</LI></UL></DIV></DIV></DIV><DIV style="display:none;"><P> Document the dates you worked on your project.</P><DIV class="column half_size"><H5>What should this page have?</H5><UL><LI>Chronological notes of what your team is doing.</LI><LI> Brief descriptions of daily important events.</LI><LI>Pictures of your progress. </LI><LI>Mention who participated in what task.</LI></UL></DIV><DIV class="column half_size"><H5>Inspiration</H5><P>You can see what others teams have done to organize their notes:</P><UL><LI><A href="https://2014.igem.org/Team:ATOMS-Turkiye/Notebook">2014 ATOMS-Turkiye</A></LI><LI><A href="https://2014.igem.org/Team:Tec-Monterrey/ITESM14_project.html#tab_notebook">2014 Tec Monterrey</A></LI><LI><A href="https://2014.igem.org/Team:Kyoto/Notebook/Magnetosome_Formation#title">2014 Kyoto</A></LI><LI><A href="https://2014.igem.org/Team:Cornell/notebook">2014 Cornell</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>