Team:Kent/Notebook
Show Menu
Home
Team
Meet the team
Attributions
Project
Description
Experiments
Results
Safety
Achievements
Judging
Parts
Overview
Basic
Composite
Collection
Practices
HP
Collaborations
Modelling
Introduction
Algorithm
Methodology
Results
Notebook
Navigation
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
ATTRIBUTIONS
PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Notebook
This page contains the documentation for our Mag-nano-tite Project.
It will be split into 6 sections containing the work done  through the duration of our project.
Pre - June
Click here
X
Pre-June
February
Team Kent for iGEM 2016 was formed.
We had our first iGEM meeting where we got acquainted with some of the team members.
We also met with our team supervisor
The iGEM timeline was discussed and key dates highlighted.
October 27-31 – iGEM Jamboree
Submissions deadline – 21 st October 2016
October 19 – Wiki-freeze
We discussed our aims and goals for our project.
Specific medal criteria – Bronze, Silver, Gold
A team organiser and social media officer was appointed.
March
Our iGEM team was complete 10 students, 2 supervisors and 3 advisors.
We completed our official iGEM registration individually and as a team.
We began establishing our plans for the summer.
We were appointed a space in the bioscience lab.
We would have weekly meeting from the start of our summer projects.
We began brainstorming for our project.
Problem based approach - What problems needed to be addressed?
Are we inventing something novel? Are we improving something that has already been done?
How can we approach a problem that has already been addressed to get a different response?
Is this project feasible in the time available?
Is there a sensible approach for validation?
How will we explore this project?
We viewed the track selections for iGEM
Which track would our project lay in?
Tracks
Diagnostics
Energy
Environment
Food and nutrition
Information processing
Manufacturing
Therapeutics
Art and design
Hardware
Measurement
Software
We began exploring previous iGEM projects by looking at previous team wikis.
Our ideas
Bacterial Batteries
Magnetic bacteria
To conduct energy
Combat heavy metal poisoning
Lead
CRISPR – specific alteration of genomes
Material formation – with applications to fibre formation (continuation of nanowire project) (fibre formation an area of Dr Xue’s expertise)
Magnetic nanowires to conduct electricity?
April
The brainstorming continued.
Bacteria that deals with free radicals
Fluorescent proteins that detect the radicals
Fluorescent proteins
For detection
Bacteria which grows hair
Clear applications to this
Swimming bacteria
Protein cages for drugs (be transported through Tat pathway)
Protein sculptures – visual
We re-visited the idea of magnetic bacteria.
We started researching magnetic bacteria species.
May
We continued brainstorming ideas
Solar panels
Harnessing photosynthetic capability of bacteria
Phylum cyanobacteria (slow)
Researchers are thinking of modifying the bacteria
Current problems:
Removal of waste material (solar panel practically inaccessible after installation)
Efficiency of energy conversion, only 8% when solar panel gives 40%
Evolved to protect themselves from full sunlight, hence, low efficiency
Bacteria antimicrobial peptides (protein targeting)
To combat infections
Here, the protein is the drug
Bacteria preventing the growth of other bacteria
Exam season!
After months of brainstorming it was time to make a decision.
We revisited an idea we had been discussion from our first brainstorming session.
Magneto bacteria
It has been ‘tricked’ to bio-mineralizing crystals is which iron was replace with other metals
By growing the bacteria in a solution with a high concentration of both metals
Shown that it can incorporate manganese into the crystal structure of magnetite
Some success with cobalt
Uses: waste removal or drug delivery
We started researching relevant literature relating to this topic
What are the key proteins? Can we obtain them?
Can it be incorporated into E.coli
What are the applications.
We created a google drive where would feed in information throughout our project.
June
Click here
X
June
May 31- June 5
We began to gather scientific papers (PubMed, WebofKnowledge, Science Direct) to post on to our shared folder.
We explored the iGEM archives for similar ideas identify Bio-bricks (Mam proteins?)
Identify key words to search magnetosomes, MMS6 protein, nano-particles.
June 6 - June 12
We began to plan how we would conduct our experiments.
What genes are relevant?
Can we obtain them
Do we need specific strains of E. coli?
Can we purchase these strains?
If not who do we speak to, to obtain these strains?
What safety risks do these pose?
June 13 - June 19
We began to obtain sequences we would use.
It was important here to start to identify the experiments we were going to use.
June 20 - June 26
We continue reading through and understanding the relevant literature.
We began to understand our safety implications of our project.
July
Click here
X
July
June 27 th – 3 rd July (Week 1)
Day 1
We received our lab induction. We received training on how to use unfamiliar equipment.
Having already researched the different protocols we would be using during our project we conducted risk assessments.
We received training on the correct use and disposal of waste and materials.
We were appointed different plasmids:
pET3a mamX/mamT/mamP
pET23B SecS mamX/ mamT/ mamP
pSB1A3 RFP
pEC86
pSB1c3 RFP
pET3a
pSB1C3
We began writing our project descriptions for the upcoming deadline.
We set up our social media accounts: Twitter,
Facebook and Instagram.
Day 2
We made LB plates and broth ensuring to add the relevant antibiotics (ampicillin or chloramphenicol).
Day 3
Parts from our distribution kits were re-suspended.
Our plasmids were transformed in top10 cells and left overnight for 15-18hours incubation.
Day 4
Re-streaks were performed on successful transformations.
Unsuccessful transformations were repeated.
The writing of our project was continued.
Registration for a student account for Enthought Canopy,
Tutorials were completed on TutorialSpoint/Python
We attempted to ‘scrape’ one iGEM wiki
July 4 th – July 10 th (Week 2)
Day 5
Liquid cultures were prepared frim our transformed cells.
Using SnapGene we carried out plasmid map analysis using Snapgene to identify compatible restriction sites on our individual plasmids.
Day 6
Our plasmids were purified from the overnight liquid cultures.
The concentration of the DNA of our individual plasmids were determined.
Restriction enzyme digests were conducted based on the restriction sites identified on the plasmid maps including the appropriate buffer.
pET3a – Xbal, HindIII, promega E
pET3a mamP – Xbal, EcoRI, promega H
pET3a mamT – Xbal, HindIII, promega E
pET3a mamX – Xbal, HindIII, promega E
pET23b mamP – NdeI, EcoRV, promega D
pET23b mamT – XbaL, PstI, promega H
pET23b mamX – NdeI, EcoRV, promega D
pET23b mamP – NdeI, EcoRV, promega D
pEC86 – Ndel, EcoRV, promega D
Day 7
Agarose gels were conducted to validate our various plasmids.
We uploaded our project description onto our wiki
Gels were imaged.
Day 8
Troubleshooting began who electrophoresis gels which were unsuccessful.
Day 9
pSB1C3 RFP and pSB1A3 RFP plasmids were validated.
July 11 th – July 17 th (Week 3)
Day 10
Gel extraction protocols were used to purify validated plasmids.
Liquid cultures were prepared to form glycerol stocks from validated plasmids.
We prepared our outreach presentations for Simon
Langton Grammar School for Boys Canterbury.
Day 11
We successfully gave outreach presentations at Simon
Langton Grammar School for Boys Canterbury.
Two presentations delivered to physics and biology classes respectively.
Questions from both physics classes were documented.
Day 12
We continued with the validation of our individual plasmids with the aim to validate all our plasmids.
Gel electrophoresis gels were continued.
Plasmids pET3a and pET23b sec.S mamT/ mamP/ mamX were validated.
Day 13
Plasmids pET3a and pET23b sec.S mamT/ mamP awere transformed into BL21 cells.
Liquid cultures were prepared from the Top10 constructs of Plasmids pET3a and pET23b SecS mamT/ mamP
Day 14
Glycerol stocks were made for pet3aMamT and psb1c3, pSB1C3 RFP and pSB1A3 RFP.
Liquid cultures were made for these plasmids
Plasmid purification was done for the necessary plasmids to build up the DNA stocks
July 18 th – July 24 th (Week 4)
Day 15
Liquid cultures from BL21 cells containing SecS.MamX/MamT/MamP plasmids.
Using SnapGene we looked at primer sequences in preparation for PCR.
We began brainstorming for the iGEM meet up at the University of Westminster on August 17-19 th
Day 16
We gathered detailed the sequences of the Mam proteins using their respective SnapGene
We ensure that we validated the sequences UniProt data and found the correct open reading frames.
We began growing up liquid cultures in preparation for the purification of our proteins MamP and MamT.
Day 17
We finished gathering possible primer sequences for our Mam proteins.
We began protein preparations for protein expression and purification.
MamP, MamX and MamT grown in
Day 18
Buffers were prepared.
Lower Tris buffer prepared using: 1.5M Tris, 0.4% SDS, pH 8.8
Upper Tris buffer prepared using: 1M Tris, 0.4% SDS, pH 6.8
Tris-Glycine running buffer prepared – 30g Tris base, 144g Glycine, 150ml SDS (1.5% w/v)
Liquid cultures from BL21 cells were made from transformed MamX, MamT and MamP cells and incubated overnight.
Day 19
SDS page gels was made.
Samples taken from the liquid cultures were boiled to denatured protein.
Samples obtained were run on an SDS-page gel.
The gel was stained with comassie blue over the duration of the weekend.
July 25 th – July 31 st (Week 5)
Day 20
We decided on the team name ‘Mag-nano-tite’
A collaboration was formed between Team Kent and UC Davis
We wrote code for a search bar on iGEMMatch.org
SDS PAGE gel de-stained using methanol on shaker for ~3 hours.
Gel was imaged and proved unsuccessful.
Troubleshooting began.
Day 21
Continued work on ideas/concepts for the upcoming Westminster presentation and poster.
Nickel column B and wash buffers were made and set to the correct pH.
overnight liquid
Liquid cultures were also prepared from SecS.MamX, MamP, MamT and pET3a Top10 cells.
Day 22
The plasmids were purified from the liquid cultures made the previous day and the concentrations determined
We contacted the iGEM team at the University of Westminster team regarding the meet up and continued work to the presentation.
Day 23
we conducted an inventory to determine which plasmids still required transformation, or repeats.
pEC86 was transformed into BL21 cells.
Day 24
We attempted co-transformation into BL21 cells using plasmids secS mamP &amp; pec86) and incubated over the weekend.
August
Click here
X
August
August 1st – August 7th (Week 6)
Day 25
We began preparing for the iGEM deadline on August 12th.
We began compiling information for our abstract
Co-transformed plasmids pEC86 + SecS MamP were made into BL21 DE3 competent cells.
SecS MamP were transformed in BL21 in DE3 competent cells.
Day 26
SecS MamX were transformed in BL21 in DE3 competent cells.
We created a code which measures the frequency of a key word for our Topic modelling programme.
Day 27
Competent cells were created for pEC86 and SecS MamX were, measured into 0.5mL aliquoted and stored in the -80°C deep freezer.
We received our primers,
Day 28
The primers were re-suspended.
We carried out our first PCR using the Mam genes (SecS and insoluble) to amplify our DNA.
The PCR products were purified.
Day 29
An electrophoresis gel was conducted to validate the PCR products. This gel was unsuccessful, and the PCR was repeated adjusting the temperatures in a bid to correct the errors from the first PCR.
Prepared Mam proteins were analysed using an SDS-page gel. They were stained using coomasie-blue for the duration of the weekend
August 8th – August 14th (Week 7)
Day 30
SDS PAGE gels were de-stained; resulting image shown below.
The team delivered a mock presentation in preparation for Westminster conference.
Final adjustments were made for iGEM deadline for track Selections and project abstract
Day 31
Mam proteins were isolated using His-tag purification.
After a third failed attempt to gain the correct PCR products, PCR was repeated using different enzymes.
Plasmid maps were view to select new primers.  primers.
Day 32
pET3a plasmid validated.
We had a meeting with team and supervisors to discuss direction of our project and set targets to achieve Gold medal criteria
PCR was repeated using new primers
Day 33
We began researching the literature relating to the chemical synthesis of magnetic nano-particles.
We spoke to a chemical expert in the field of nano-particles.
We liaised with teams for a possible collaboration.
Day 34
iGEM deadline met, abstract submitted and track selection completed – New Applications.
We continued sourcing the materials for the chemical synthesis of chemical nano-particles.
Preparation for the Westminster meet-up continued.
August 15th – August 21st (Week 8)
Day 35
We began working on an outline for our wiki page.
A Gibson assembly was performed and fragment sup35nm was re-suspended.
We finalised our presentations and posters for the University of Westminster conference.
Day 36
We started identifying improvements we could make of the previous years’ project.
Liquid culture was prepared from pEC86 + MamP and MamT
We observed that the induced liquid culture when spun down was red in colour likely due to the cytochrome C domain in the protein binding iron.
Day 37
We developed our Topic modelling programme to read multiple text files.
Members of the iGEM team presented our poster at the University of Westminster iGEM conference
Using nickel ion chromatography, we purified out MamX, MamT, MamP proteins
Day 38
Members of the iGEM team gave a spoken presentation at the University of Westminster iGEM conference
Day 39
We began compiling our data, results in preparation for our wiki.
August 22nd – August 28th (Week 9)
Day 40
We reflected on the presentations given at the Westminster
Gibson assemblies were performed.
We double digested pSB1C3 and ran a gel to validated,
We transformed NEB 5-alpha competent E. coli cells with 2ul of the Gibson reaction mixture.
Day 41
Colonies from plates were Re-streaked some white colonies and made liquid cultures.
Day 42
Re-streaks showing white colonies were mini-prepped.
QIAprep protocol to purify the plasmids.
Day 43
We focused on dry lab tasks such as planning the mock questions for future presentations.
Day 44
We continued compiling information the wiki
Planned future experiments
September
Click here
X
September
August 29 th – September 4 th (Week 10)
Day 45
The AFM imaging of samples commenced.
Double digest of PSB1C3 RFP with Sup35nm using Ecor1 and Pst1
PCR using usual protocol (with double digested template DNA) and original long primer on pET23B SecS genes.
Prepared 500ml LB in 2L Baffled flasks for protein expression
Day 46
Repeated agarose gels
New forward primers arrived therefore PCR was done using normal protocol (with annealing temp at 52 degree and double digested template DNA).
We set up 4ml overnight liquid cultures containing double antibiotics, 2 for each Pet23b Sec.S MamT, MamP, and MamX with pEC86 using the same colony.
Made fresh antibiotic samples.
Day 47
Made 1ml glycerol stocks using 500ul of liquid culture for Sec.S MamP, MamT and MamX with Pec86 in BL21’s.
Using the remaining 3.5ml to inoculate 500ml LB in the Baffled flasks.
We grew up the cells and induced with IPTG for protein expression.
Ligation and transformations carried out.
Day 48
We finalised our plans for collaboration with UEA over skype conference. It was agreed that we would carry out some AFM images of their S. oneidensis cells on a variety of different surfaces.
SDS gels carried for MamP SecS, MamX SecS, MamT SecS and stained overnight.
Day 49
De-strained SDS page gels and imaged them.
In preparation, we shipped off AFM sample discs to UEA in Norwich in preparation for collaboration the following week.
September 5 th - September 11 th (Week 11)
Day 50
We prepared liquid cultures with all Mam pET23B genes.
Using plasmid maps we designed new primers.
Day 55
Mini-prepped liquid cultures and ran undigested verification gels
Re-attempted PCR
Web design progress for our wiki.
We used the absorbance reading at 280nm and extinction coefficient 8.73 mM -1 cm -1 to estimate the concentration of mam P as 27.4 µM.
We used the absorbance reading at 280 and extinction coefficient 27.32 mM -1 cm -1 to estimate the concentration of mam P as 39.2 µM
Day 56
Using SnapGene plasmid map sequences we designed new SecS primers
We carried out spin columns for protein purification of mamP/T/X
Using the absorbance reading at 280 nm and extinction coefficients, we estimated the concentrations for mamP/X/T for subsequent anaerobic reaction vessels.
Carried out the reaction using purified proteins with iron solutions within anaerobic reaction vessel.
Day 57
Reactions between purified proteins and iron solutions were placed onto EM grids and prepared for imaging (in vivo).
September 12 th – September 18 th (Week 12)
The team logo and banner were designed for review by the team.
We collaborated UEA to AFM image with samples.
We continued making our final constructs.
September 19 th - September 25 th (Week 13)
The team logo and banner were finalised.
We continued writing
AFM imaging of our samples continued.
October
Click here
X
October
September 26th – October 2nd
Semester resumes
We began planning our outreach presentation for the University of Kent summer student Bioscience Symposium.
We prepared a presentation using updated information about our project and updated results
We continued imaging our samples to validate our constructs, these constructs include: pSB1A3-[AraC- pBAD]-[CsgA- SS-Sup35NM- 1-61], pSB1A3-[AraC- pBAD]-[CsgA- SS-Sup35NM- 1-61- Cytb562], pSB1A3-[AraC- pBAD]-[CsgA- SS-Sup35NM]
October 3rd – October 9th
We continued writing up our results and compiling information for our wiki.
The team attended a University of Kent Bioscience open day. We spoke to the public about our iGEM project, the iGEM competition and its future applications.
We continued the AFM imaging of samples.
The final construct of pSB1A3 mamOPXT transformed.
October 10th – October 16th
pSB1A3 mamOPXT
We continued the AFM imaging of our samples.
E. coli cells were induced containing the mamOPXT plasmid, we did this by growing the cells in a liquid media containing arabinose, IPTG and Ferric Citrate.
Electron microscopy grids prepped for imaging.
Writing for wiki continued.
October 17th – October 23rd
The team Completed content for our wiki
Wiki freeze.
Final electron microscopy images produced.
Prepared our posters, banner and presentation for the jamboree
Outreach at second University of Kent open day.
October 24th – October 31st
iGEM Jamboree in Boston.
Document the dates you worked on your project.
What should this page have?
Chronological notes of what your team is doing.
Brief descriptions of daily important events.
Pictures of your progress.
Mention who participated in what task.
Inspiration
You can see what others teams have done to organize their notes:
2014 ATOMS-Turkiye
2014 Tec Monterrey
2014 Kyoto
2014 Cornell
