<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Exeter_Results_GvsB"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Exeter/Results/GvsB">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Exeter/Results/GvsB&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Results/GvsB&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Results/GvsB&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Exeter/Results/GvsB" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Exeter/Results/GvsB</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menu_interface"><DIV id="main_options"><UL id="main_options"><DIV id="button_project"><DIV id="main_options_project" align="center"><A href="#" class="listhead" id="project">ExiGEM</A></DIV></DIV><DIV id="button_tsci"><DIV id="main_options_tsci" align="center"><A href="#" class="listhead" id="the_science">Concept</A></DIV></DIV><DIV id="button_btsci"><DIV id="main_options_btsci" align="center"><A href="#" class="listhead" id="beyond_the_science">BeyondTheScience</A></DIV></DIV><DIV id="button_results"><DIV id="main_options_results" align="center"><A href="#" class="listhead" id="results">Results</A></DIV></DIV></UL><DIV id="menu_bkgr"><DIV id="slide_menu"><UL class="project"><LI><A href="https://2012.igem.org/Team:Exeter/Introduction" class="listmain" id="results">Introduction</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/The ExiGEM2012 Team" class="listmain" id="results">Team</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Collaborators" class="listmain" id="results">Collaborators</A></LI></UL><UL class="the_science"><LI><A href="https://2012.igem.org/Team:Exeter/Concept" class="listmain" id="results">Background &amp; Concept</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Applications" class="listmain" id="results">Future Applications</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Human_Practices" class="listmain" id="results">Human Practices</A></LI></UL><UL class="results"><LI><A href="https://2012.igem.org/Team:Exeter/Results" class="listmain" id="results">Project Conclusions</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Safety" class="listmain" id="results">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Diary" class="listmain" id="project">Diary</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Lab_Book" class="listmain" id="results">Lab Books</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Results/biobricks" class="listmain" id="results">Bio-Bricks</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="beyond_the_science"><LI><A href="https://2012.igem.org/Team:Exeter/Ideas" class="listmain" id="results">Early Ideas</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Outreach" class="listmain" id="results">Outreach</A></LI></UL></DIV></DIV></DIV><DIV id="external_links"><UL id="link_text"><DIV id="blog_link"><A href="http://exigem2012.wordpress.com/">Follow our Blog</A></DIV><DIV id="twitter_link"><A href="http://twitter.com/#!/iGEMExeter2012">Follow us on Twitter</A></DIV></UL></DIV></DIV><P><TITLE>Results: Bio-Brick Assembly vs. Gibson Assembly</TITLE></P><P>Gibson or BioBrick assembly? - Mary Beton</P><FONT color="#1d1d1b" size="2" face="Verdana"><P><B>Why this question is important:</B></P><P>One of the aims of the iGEM foundation is for advancement of synthetic biology, and if techniques available to the teams are outdated then this could seriously impede their
    progress, as well as potentially teaching the competing students redundant methods. There has been a small trend in recent years of iGEM teams starting to consider and use 
    Gibson assembly as an alternative (notably Cambridge 2010, who produced a song about it) to BioBrick assembly. Is it in the interests of the iGEM foundation to consider
    including this as an iGEM standard technique? Or does the specific nature of the Parts Registry mean teams who choose Gibson Assembly should retrofit BioBrick restriction sites
    onto their new parts and use the technique so long as, overall, they can cohere with the current iGEM requirements? We thought it was a significant question which has to be 
    asked if iGEM is to play a genuine role in synthetic biology development.</P><P><FONT color="#57B947" size="3" face="Verdana"><B>Biobrick Assembly</B></FONT></P><P><B>How it works</B></P><P>The BioBrick platform has lead to the wide exploration and use of different restriction enzymes and assembly standards, Biobrick assembly is the assembly format familiar to 
    most iGEM teams. Its reliance on four key restriction enzymes (shown in fig. 1) make it easy to cut upstream or downstream of the DNA sequence as required. Rather than merely 
    using the EcoRI and PstI sites, the additional use of the internal XbaI and SpeI sites, which cut to form identical overhangs (fig. 2) ensure not only that two biobricks can be
    ligated together at the same time as they are put into a plasmid using the external E and P sites, but also that the recombined X-S site ‘scar’ won’t be recognised by either of 
    the two enzymes again when correctly ligated (fig. 2).</P><TABLE width="980" align="center" cellspacing="20"><TBODY><TR align="center"><TD><P><CENTER><FONT color="#1d1d1b" size="2" face="Verdana">Fig. 1</FONT></CENTER></P></TD><TD><P><CENTER><FONT color="#1d1d1b" size="2" face="Verdana">Fig. 2</FONT></CENTER></P></TD></TR></TBODY></TABLE><P><B>Advantages and Disadvantages</B></P><P>Although biobrick assembly is not a new technique, the format of it and its wide use, particularly by iGEM teams, has lead to several assembly standards being developed and 
    the technique generally has been updated and progressed with time. Standard assembly, requiring inefficient gel extraction, could create difficulties with small promoters and 
    ribosome binding sites which might not show up easily on a gel. 3A assembly was introduced (fig. 1). All it required was the upstream fragment, downstream fragment and 
    destination vectors to have different antibiotic resistance (‘3A’) so only transformed cells with the correctly ligated destination vector will have resistance to the antibiotic
    of choice. A rare issue with this is that an undigested vector containing the upstream or downstream parts can be taken up by the cell along with the correctly assembled vector,
    so resistance is still conferred. This is considered unlikely and can be tested using the fragment vector antibiotics and sequencing.</P><P>A feature of the assembly method is the inclusion of a ‘scar’ where the two fragments have joined at the X-S sites. The scar sequence is 6 bp long, but was 8bp in the 
    original method before being updated (Shetty, Endy and Knight 2008). 8 base pairs will, of course, introduce a frame shift which is why the method is obsolete, but the 6bp scar 
    instead creates a stop codon . For the purposes of most synthetic biology this is not a problem, but could become a difficulty if two proteins needed to be combined. The 
    development of the BglBricks standard (Anderson et al 2010) has circumvented the issue by changing the scar sequence to code for innocuous amino acids. For BioBricks, the 
    inability of XbaI and SpeI to recognise the scar gives a permanence to ligation of two biobricks which could be a disadvantage if a mistake is made in assembly, but otherwise is
    a strong advantage: you can’t accidentally undo earlier work when forming a large construct.</P><P><B>Personal findings</B></P><P>Most of our genes were synthesised in vitro so the restriction sites were easily included that way, and the steadily falling price of gene synthesis could be argued to be in 
    the favour of maintained use of biobrick assembly - it is easy to obtain a gene with the required sites this way. A small extra requirement for us was including restriction 
    sites in the primers designed for using PCR to get the wbbC gene from the BL21 genome. For some reason, wbbC proved impossible to synthesise. This fortunately gave some insight 
    into how an iGEM team with insufficient funding to synthesise genes would struggle with BioBrick assembly when using genes not in the Registry - not at all. One argument against
    biobrick assembly is the length of time required for making a large construct. We managed to create several complete promoter-RBS-gene-terminator constructs, but our four and 
    five gene operons were never achieved in the time we had available.</P><P><B>In a Nutshell:</B></P><P><U>Pros</U></P><UL type="square"><LI>Flexible</LI><LI>Easy to use</LI><LI>Selection techniques are easy</LI><LI>Easy to share parts with Parts Registry</LI><LI>DNA can be synthesised/from PCR/from Registry</LI><LI>Easy and requires little foreknowledge</LI><LI>When parts are joined they can’t be split with the same enzymes</LI></UL><P><U>Cons</U></P><UL type="square"><LI>Can only accurately combine two parts at a time - long process</LI><LI>Creates ‘scars’</LI><LI>Can have two-plasmid transformation problems</LI><LI>Mistakes are hard to undo</LI></UL><P><FONT color="#57B947" size="3" face="Verdana"><B>Gibson Assembly</B></FONT></P><P><B>How it Works</B></P><P>Blunt-ended fragments of DNA with overlapping end sequences can be obtained through PCR, or synthesised directly. The 5’ ends are then chewed back by an exonuclease. The 
    single stranded matching regions anneal, a DNA polymerase copies the sequence from the 3’ ends and finally a DNA ligase joins the fragments permanently (fig. 3). The technique 
    was developed by Gibson et al and published in 2009.</P><TABLE width="980" align="center" cellspacing="20"><TBODY><TR align="center"><TD><P><CENTER><FONT color="#1d1d1b" size="2" face="Verdana">Fig. 3</FONT></CENTER></P></TD></TR></TBODY></TABLE><P><B>Advantages and Disadvantages</B></P><P>Some extra work is required if Gibson assembly is used by iGEM teams - to get parts into the registry they need restriction sites added which can pose difficulties due to 
    prefix and suffix homology. The Cambridge 2012 iGEM team stated that although Gibson appears to work well from positive controls, its actual results were very dependent on the 
    initial concentration of DNA fragments, a problem which may have stemmed from gel extraction inefficiency.</P><P><B>Personal findings</B></P><P>I have no idea whether Gibson assembly is as good or bad as I have been told, because we never got to try it. Nine different Gibson fragments were needed to construct the 
    three operons, and while seven were created using PCR (some immediately, some with more attempts), the remaining two (the promoter and the first gene of the operon) had three 
    weeks, three different polymerases and multiple changes to reaction conditions expended on them and still failed to appear. The primers had ribosome binding site sequences 
    included, which should have been an advantage but may have made the primers long and difficult to use. Since we were never able to test Gibson assembly itself it is hard to know
    whether a few bases added to the end of the RBS overlaps would have been sufficiently specific for construct assembly.</P><P>The primers were designed with the help of Gibthon, an online tool developed by The University of Cambridge. While it was helpful, I personally thought it didn’t fully assist
    with designing the primer pairs we wanted. The primer tool designs primers based on the left and right sequences which will overlap but would be more use specifically designing 
    primers for a given fragment and the sequences to either side of it. Having said this, I don’t think this is a failing of Gibthon (it’s a really useful tool that does a lot 
    more, such as calculating the required volume of fragments in the assembly mix) I think it’s a failing of Gibson assembly in general, because a lot more fore-thought is required
    and mistakes are very possible. There is potential for this to be improved if it becomes more widely used.</P><P>If we had had the money we could have ordered genes with overlap regions already present. This, however, would have presented more problems for our project: constructing 
    operons was only a quarter of our labwork aim. For three of the four lab miniprojects we had the same genes with different uses. Our original hope was to have the genes 
    synthesised with the overlap, but as we aimed for more things with the project this would have limited the use of each gene and thus was ruled out completely. I thought 
    designing primers took too long, especially with the rapid changes we made to the project at the start. My personal verdict is that the whole process was inflexible and, given 
    that we never made an operon, never showed itself to be better than BioBrick assembly.</P><P><B>In a Nutshell</B></P><P><U>Pros</U></P><UL type="square"><LI>Good for large constructs</LI><LI>Rapid (when you have the right fragments and ratios)</LI><LI>Simple procedure with easy mastermixes available</LI><LI>DNA can be synthesised/from PCR</LI></UL><P><U>Cons</U></P><UL type="square"><LI>PCR fragments might be difficult and time-consuming to obtain</LI><LI>Requires some foreknowledge and planning to design primers</LI><LI>Yield decreases with complexity of construct</LI><LI>Less flexibility for multiple varied constructs</LI><LI>Antibiotic selection is still required after transformation</LI><LI>Not very compatible with Parts Registry</LI><LI>Can be tricky to get optimum volumes of fragments</LI></UL><P><FONT color="#57B947" size="3" face="Verdana"><B>Summary:</B></FONT></P><P>While I think Gibson assembly has potential to be a lot more efficient for creating large constructs and the premise is very easy to understand, as an undergraduate who 
    hadn’t had much experience of planning labwork before I felt I used the technique very badly and it wasn’t surprising. It was unfamiliar to the people I talked to in the 
    Bioscience department so I had no prior advice, but equally we had no prior advice regarding BioBrick assembly and had no difficulties carrying out that from the protocols. It 
    was impossible to use Gibson more widely in our project because we needed the flexibility to use the same gene in multiple ways, which BioBrick assembly gave us. While Gibson
    assembly may save time, it didn’t get anywhere in our case. It’s far from what I expected, but of the two I preferred BioBrick assembly.</P><P>BioBrick assembly was too time consuming for us to manage the operon construction in the allotted time, but it was very successful for smaller constructs in the other 
    miniprojects. Ideally, and impossibly, BioBrick assembly would be updated to yield a rapid method which did not rely on purification of the construct through transformation, 
    brothing and miniprepping, which requires a minimum of three days. The advantages of BioBrick flexibility mean it is more suited to a broader iGEM project like ours, but for 
    teams who want a large construct Gibson may be better.</P><P>There are, of course, other assembly methods which are newer than BioBrick and Gibson assembly. While for now it is these two which need to be examined for iGEM, and 
    hopefully in future there will be data to do this satisfactorily, I think it is most likely that we are a few years away from a technique emerging which gives flexibility, speed
    and hopefully can be afforded by teams with less funding.</P><P><B><FONT color="#1d1d1b" size="2" face="Verdana">I would like to extend my thanks to members of the Cambridge 2010 iGEM Team who very kindly shared their experiences of 
    Gibson Assembly with me.</FONT></B></P><P>References:</P><P>Shetty R.P., Endy D. &amp; Knight T.F. (2008) Engineering BioBrick vectors from BioBrick parts. <I>Journal of Biological Engineering.</I> 2 (1) DOI: 10.1186/1754-1611-2-5</P><P>Anderson J.C., Dueber J.E., Leguia M., Wu G.C., Goler J.A., Arkin A.P. &amp; Keasling J.D. (2010) BglBricks: A flexible standard for biological part assembly. <I>Journal of Biological Engineering.</I><B>4</B>, 1. DOI: 10.1186/1754-1611-4-1</P><P>Gibson D.G., Young L., Chuang R.Y., Venter J.C., Hutchison C.A. &amp; Smith H.O. (2009) Enzymatic assembly of DNA molecules up to several hundred kilobases. <I>Nature Methods.</I><B>6</B> pp.343-345. doi:10.1038/nmeth.1318</P></FONT><TABLE width="980"><TBODY><TR><TD align="left"><FONT face="Verdana" color="#57b947" size="3"><A href="https://2012.igem.org/Team:Exeter/Results"><U>&lt;&lt; Return to Results</U></A></FONT></TD></TR></TBODY></TABLE><TABLE width="980" align="center" cellspacing="20"><TBODY><TR align="center"><TD><FONT color="#57B947" size="1" face="Verdana"><P><U>Website Designed and Built by: Ryan Edginton, James Lynch &amp; Alex Clowsley</U>  |  <A href="https://igem.org/Team.cgi?id=764" style="color:#57B947" target="_blank"><U>Contact Us</U></A>  |  <A href="https://2012.igem.org/Team:Exeter/site_map" style="color:#57B947"><U>Site Map</U></A></P></FONT></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Exeter/Results/GvsB">http://2012.igem.org/Team:Exeter/Results/GvsB</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Exeter/Results/GvsB" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Exeter/Results/GvsB" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Exeter/Results/GvsB&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Exeter/Results/GvsB&amp;oldid=232966" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>