<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Exeter_Concept"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Exeter/Concept">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Exeter/Concept&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Concept&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Exeter/Concept&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Exeter/Concept" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Exeter/Concept</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menu_interface"><DIV id="main_options"><UL id="main_options"><DIV id="button_project"><DIV id="main_options_project" align="center"><A href="#" class="listhead" id="project">ExiGEM</A></DIV></DIV><DIV id="button_tsci"><DIV id="main_options_tsci" align="center"><A href="#" class="listhead" id="the_science">Concept</A></DIV></DIV><DIV id="button_btsci"><DIV id="main_options_btsci" align="center"><A href="#" class="listhead" id="beyond_the_science">BeyondTheScience</A></DIV></DIV><DIV id="button_results"><DIV id="main_options_results" align="center"><A href="#" class="listhead" id="results">Results</A></DIV></DIV></UL><DIV id="menu_bkgr"><DIV id="slide_menu"><UL class="project"><LI><A href="https://2012.igem.org/Team:Exeter/Introduction" class="listmain" id="results">Introduction</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/The ExiGEM2012 Team" class="listmain" id="results">Team</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Collaborators" class="listmain" id="results">Collaborators</A></LI></UL><UL class="the_science"><LI><A href="https://2012.igem.org/Team:Exeter/Concept" class="listmain" id="results">Background &amp; Concept</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Applications" class="listmain" id="results">Future Applications</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Human_Practices" class="listmain" id="results">Human Practices</A></LI></UL><UL class="results"><LI><A href="https://2012.igem.org/Team:Exeter/Results" class="listmain" id="results">Project Conclusions</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Safety" class="listmain" id="results">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Diary" class="listmain" id="project">Diary</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Lab_Book" class="listmain" id="results">Lab Books</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Results/biobricks" class="listmain" id="results">Bio-Bricks</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Achievements" class="listmain" id="results">Achievements</A></LI></UL><UL class="beyond_the_science"><LI><A href="https://2012.igem.org/Team:Exeter/Ideas" class="listmain" id="results">Early Ideas</A></LI><LI><A href="https://2012.igem.org/Team:Exeter/Outreach" class="listmain" id="results">Outreach</A></LI></UL></DIV></DIV></DIV><DIV id="external_links"><UL id="link_text"><DIV id="blog_link"><A href="http://exigem2012.wordpress.com/">Follow our Blog</A></DIV><DIV id="twitter_link"><A href="http://twitter.com/#!/iGEMExeter2012">Follow us on Twitter</A></DIV></UL></DIV></DIV><P><TITLE>Concept</TITLE></P><P><B><U>Background to the Project</U></B></P><FONT color="#1d1d1b" size="2" face="Verdana"><P><A href="https://2012.igem.org/Team:Exeter/Applications" style="color:#57b947"><U>Polysaccharides</U></A> - polymers of sugar molecules - have a spectacular range of 
     properties and uses, from the structural (cellulose) and medicinal (bacterial vaccines, cyclodextrin) to foods (starch), glues (levan) and lubricants (hyaluronan (HA)). This 
     range of properties stem from both the chemical nature of the sugars, their physical arrangement within the polymer and the macro-scale arrangement of the polymer itself.</P><P>Currently, scientific research relies on the study of polysaccharides isolated from their native source and subsequent modification, or on expensive, time-consuming and 
     potentially <A href="https://2012.igem.org/Team:Exeter/Human_Practices/lande" style="color:#57b947"><U>environmentally-polluting</U></A> production via
     <A href="https://2012.igem.org/Team:Exeter/Human_Practices/impact" style="color:#57b947"><U>synthetic chemistry</U></A>. This project, e-candi, aims to investigate whether 
     synthetic biology can provide a platform to revolutionise polysaccharide research by allowing rapid biosynthesis of designer polysaccharides. If this were possible we could 
     implement an iterative design cycle in polysaccharide research. We posed a range of questions to test whether this might be possible. In undertaking this project we set 
     ourselves several smaller tasks, or mini-projects. </P><P><FONT face="Verdana" color="#57b947" size="+2"><B><U>The Concept of Our Miniprojects</U></B></FONT></P><P><B><U>The Wzy-Dependent System*</U></B></P><P>The Wzy-dependent system* is a method of polysaccharide biosynthesis. It involves construction of short saccharide repeat units from a small number of monosaccharides (which
     are started on und-PP-GalNAc/GlcNAc attached to the inner membrane) and connected in order by glycosyltransferase (GTase) enzymes. The repeat unit is then handled by three key 
     enzymes:</P><UL type="square"><LI>Wzx is a flippase enzyme, which takes the repeat unit through the inner membrane and into the periplasm.</LI><LI>Wzy is a polymerase enzyme which adds the repeat units together within the periplasm. The polysaccharide formed by this is length regulated by Wzz.</LI><LI>Wzz, a length regulating enzyme. The mechanism by which the enzyme works is poorly understood, but there are variants of Wzz which produce varying chain lengths. Their 
      seemingly limited specificity enables them to act on different polysaccharides.</LI></UL><P>The foundational idea of our project was based on harnessing the GTases of the Wzy-dependent system in order to synthesise polysaccharides based on the required pattern of 
     monosaccharide units. This way, <A href="https://2012.igem.org/Team:Exeter/Applications" style="color:#57b947"><U>new polysaccharides</U></A> could be created to remove the 
     bounds of the properties of existing polysaccharides, or existing bacterial coat sugars could be imitated (for vaccines).</P><P><B><U><A href="https://2012.igem.org/Team:Exeter/Results#glycos">GlycoBase/GlycoWeb</A></U></B></P><P>So why do we need a database and what should it be capable of?</P><P> We identified the endogenous Wzy-dependent polysaccharide biosynthesis machinery as a suitable target for manipulation by heterologous expression of <I>E. coli</I> GTases. 
     GTases are specific to different donor and acceptor monosaccharides. The database makes it possible to find an enzyme combination which will correctly connect monosaccharides 
     to create the polysaccharide of design. This will improve the efficiency of the polysaccharide lab work and speed up the process of polysaccharide production.</P><P>It is not unforeseeable that there may be more than one enzymatic approach to producing a particular polysaccharide; as a long term aim of the database project, it may be 
     possible that our database could return some of the advantages/disadvantages of certain construction pathways. Indeed if some repeating units are not possible, the database 
     could suggest similar or alternative products with ‘nearly’ identical properties.</P><P><B><U><A href="https://2012.igem.org/Team:Exeter/Results/showcase">Showcasing Polysaccharides</A></U></B></P><P>Before considering GTase combinations however, we asked whether we could synthesise useful polysaccharides in <I>Escherichia coli</I> with single genes. Evolution has 
     provided us with a remarkable variety of polysaccharides that have unique properties and countless uses. Their applications in medicine, industry, food, cosmetics, engineering 
     etc. have been recognised and the biological synthesis of some of these polysaccharides, to avoid issues with limited bioavailability and the tedious chemical synthesis methods,
     is a recent development.</P><P>The three polysaccharides:</P><UL type="square"><LI>Hyaluronan is a powerful lubricant found in human joints and skin which has vast medical applications from joint lubricators and surgical glues, as well as cosmetic 
      applications in skin moisturisers.</LI><LI>Levansucrase is a potential glue recognised by Newcastle iGEM 2010 (BacillaFilla) that could repair cracks in concrete. The addition of a signal peptide, <I>ompA</I>, was 
      planned to ensure expression of Levansucrase outside the cell.  Within the cell, levansucrase has toxic properties which prevent it being made without cell death and give it 
      use as a selection marker. We hoped export from the cell would give it dual use and enable the levansucrase to accumulate to explore it's sticky properties.</LI><LI>The third polysaccharide we are producing is a cyclic oligosaccharide made from starch rather than a linear polysaccharide - cyclodextrin. Cyclodextrin has important 
      medical applications as a drug delivery system, and food applications through the removal of cholesterol.</LI></UL><P>We have built biobricks coding for HA synthase (hyaluronan production), cyclodextrin glycosyltransferase (cyclodextrin production) and have modified the existing <I>sacB</I>
     biobrick to attempt extracellular levansucrase biosynthesis in <I>E. coli</I>. In so doing we have introduced an <I>ompA</I> export signal peptide coding sequence to the 
     registry, the first signal peptide to be submitted.</P><P><B><U><A href="https://2012.igem.org/Team:Exeter/Results#GvB">Operon Construction</A></U></B></P><P>Secondly, we asked whether it was possible to synthesise a novel polymer sequence in <I>E. coli</I>. We have designed three operons containing combinations of genes coding 
     for GTases, which should work together with the wzy-dependent system to synthesise a polysaccharide never before seen in nature. The three variants with similar, but not 
     identical, enzymes aimed to demonstrate the ability to construct different monosaccharide combinations within the repeat unit and the aim was to analyse this by mass 
     spectrometry.</P><P> In the course of operon construction we have conducted parallel comparisons between the <A href="https://2012.igem.org/Team:Exeter/Results/GvsB" style="color:#57b947"><U>Biobrick and Gibson Assembly Methods</U></A>. The purpose if this is for a comparison of the assembly methods intended for future use by iGEM teams.</P><P><B><U><A href="https://2012.igem.org/Team:Exeter/Results/inducible">The 3-Gene Inducible Plasmid</A></U></B></P><P>Thirdly we asked how the system already designed may be developed further.</P><P>We designed a plasmid with differentially inducible and/or repressible GTase gene expression to investigate whether it is possible to control polymer biosynthesis through 
     gene expression rather than repeated genetic transformation.</P><P>This  mini project aim was to create a three gene inducible plasmid. Each gene on the plasmid is controlled by a unique promoter, thereby giving the capability to turn each 
     gene on and off. Why is this important? The ability to turn the genes on and off will allow the creation of a monosaccharide, a disaccharide and a trisaccharide. While this in 
     itself will not create a library of polysaccharides available from a single <I>E. coli</I> colony, it is a proof of concept with the projected scope for more than 50 GTases 
     coded for within the genome of one simple organism. The result would be a small polysaccharide factory without having to redesign the genetics every time a new bespoke 
     polysaccharide is wanted.</P><P><B><U><A href="https://2012.igem.org/Team:Exeter/Results/characterise">Single Gene Plasmids and Enzyme Characterisation</A></U></B></P><P>With the database of <I>E. coli</I> GTases we wanted the ability to choose the optimal GTases where several options are available for catalysis of the same reaction. To 
     improve system understanding and performance we decided to create single gene plasmids for enzyme characterisation.</P><P> Single gene expression plasmids were constructed to determine protein expression and analyse enzyme function. GTases are poorly understood and verification of the preferred
     donor and acceptor sugars, in addition to enzyme kinetics, will be essential for the proper functioning of the database. To determine enzyme kinetics, a glycosyltransferase 
     assay was chosen based on the cleavage of inorganic phosphate from the pyrophosphate moiety of the sugar diphosphonucleotide carrier. The release of inorganic phosphate yields 
     a colour change which can be detected simply using a spectrophotometer. Because release of the diphosphonucleotide carrier is quantitative to enzyme rate, the change in colour 
     will reflect enzyme catalysis rate directly. In addition, SDS-PAGE will be used to determine the molecular weight of each GTase and compared to predicted values, as well as 
     solubility of each GTase to check functionality in vivo.</P><P>* References:</P><P> Whitfield C. (2010) Polymerases: glycan chain-length control, <I>Nature Chemical Biology</I>.<B>6</B>:403-404.</P><P>Woodward R. (2010) <I>In vitro</I> bacterial polysaccharide biosynthesis: defining the functions of Wzy and Wzz, <I>Nature Chemical Biology</I>. <B>6</B>:418-423.</P></FONT></DIV><TABLE width="980" align="center" cellspacing="10"><TBODY><TR><TD align="left"><FONT face="Verdana" color="#57b947" size="3"><A href="https://2012.igem.org/Team:Exeter/Collaborators"><U>&lt;&lt; Meet Our Collaborators</U></A></FONT></TD><TD align="right"><FONT face="Verdana" color="#57b947" size="3"><A href="https://2012.igem.org/Team:Exeter/Applications"><U>A Polysaccharide Future &gt;&gt;</U></A></FONT></TD></TR></TBODY></TABLE><TABLE width="980" align="center" cellspacing="20"><TBODY><TR align="center"><TD><FONT color="#57B947" size="1" face="Verdana"><P><U>Website Designed and Built by: Ryan Edginton, James Lynch &amp; Alex Clowsley</U>  |  <A href="https://igem.org/Team.cgi?id=764" style="color:#57B947" target="_blank"><U>Contact Us</U></A>  |  <A href="https://2012.igem.org/Team:Exeter/site_map" style="color:#57B947"><U>Site Map</U></A></P></FONT></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Exeter/Concept">http://2012.igem.org/Team:Exeter/Concept</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Exeter/Concept" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Exeter/Concept" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Exeter/Concept&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Exeter/Concept&amp;oldid=231387" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>