"
Page
Discussion
View source
History
teams
Log in
 
Team:Exeter/Results
From 2012.igem.org
ExiGEM
Concept
Beyond
The
Science
Results
Introduction
Team
Collaborators
Background & Concept
Future Applications
Human Practices
Project Conclusions
Safety
Diary
Lab Books
Bio-Bricks
Achievements
Early Ideas
Outreach
Follow our Blog
Follow us on Twitter
ExiGEM2012 Lab Book Home
GlycoBase  |  GlycoWeb  |  Bio-Brick vs. Gibson Assembly  |  Useful Polysaccharides  |  Showcasing Polysaccharide Production  |  Modelling
The 3-Gene Inducible Plasmid  |  Single Gene Plasmids and Enzyme Characterisation
Results
GlycoBase  |  Contributors: Liam Stubbington, Alex Baldwin
Result in Brief:
We created a database containing over 120 different glycosyltransferases found mainly in E. coli. The glycosyltransferases were listed according to their donor sugar, acceptor sugar, organism, structure and whether they were previously characterised or not. From the database, the Exeter iGEM team could then see the possible repeat unit chains (and therefore polysaccharides) which could be produced based on the glycosyltransferases available. GlycoBase became an important part of the iGEM project, acting as the mediator between the wet lab and dry lab teams.
GlycoWeb  |   Contributors: Liam Stubbington, James Lynch
Source Code: for GlycoWeb is available here
Result in Brief:
A URL was created for GlycoBase, which was uploaded to form the online interface GlycoWeb. In GlycoWeb we have created a piece of software which not only allows users to search a polysaccharide and see if its creation is possible, but also to add to the database interactively based on their own research. This will lead to better glycosylytransferase characterisation in the future. GlycoWeb is an open source piece of software.  GlycoApp, the android software version of GlycoBase is currently under construction. In the future we see Material Scientists, Engineers and Biologists ordering their desired polysaccharide using GlycoApp, therefore improving the efficiency and ease of desired material creation.
Bio-Brick vs. Gibson Assembly  |   Contributors: Mary Beton, Freddie Dudbridge, Ryan Edginton
Result in Brief:
Taking on the challenge of designing and constructing an operon in such a short experimental window was always expected to be ambitious, and running two different techniques side by side meant that the project ran out of time. The allotted time was insufficient to finish either BioBrick assembly or Gibson assembly of an entire operon, and consequently it was impossible to test for biosynthesis of a novel polysaccharide. Through BioBrick assembly, several RBS-gene constructs were made which, the promoter_RBS construct was made and terminators were added to the ends of the constructs. Some of these we have managed to transfer into pSB1C3 plasmids and have gone on to the registry as new BioBricks. The use of both Gibson assembly and BioBrick assembly has built a comparative picture of the two assembly methods in the hope that future iGEM teams might get quantitative data to assist decisions over the use of Gibson and BioBrick assembly in the iGEM competition. We hope that if we have a chance to continue this work, we could complete one of the operons and test for novel polysaccharides.
Useful Polysaccharides |  Contributors: Mary Beton, Freddie Dudbridge, Ryan Edginton
Result:
Due to an insufficient experimental time window, we were unable to complete the ambitious attempt to assemble an entire operon whilst comparing Bio-Brick Assembly and Gibson Assembly, as noted above. Consequently it was impossible to test for biosynthesis of a novel polysaccharide. Several BioBricks, both coding and composite parts, were created for the operon. While they were not combined to make the final construct, they will go to the registry and hopefully be used by future teams for polysaccharide production and the continuation of our work.
Showcasing Polysaccharide Production  |   Contributors: Alex Clowsley, Freddie Dudbridge, Becca Philp
Result in Brief:
The demonstration of a variety of polysaccharides from different natural sources with very vast applications was attempted in our polysaccharide showcase. We used 3A assembly to successfully submit the single enzymes cyclodextrin glycosyltransferase and Hyaluronan synthase to the registry. We also sequenced and submitted gene constructs of our three enzymes linked to terminators, and of levansucrase to a signal peptide and promoter-RBS sequences. Due to time constraints we were unable to clone the constructs into expression vectors but we hope to have inspired others to realise the potential of these polysaccharides.
Modelling  |  Contributors: Andy Corbett, Alex Baldwin
Result in Brief:
The motivation for constructing a mathematical model to represent our system consisted of two parts. Firstly, to provide additional software to aid GlycoWeb. The model that       has been made can rank out puts of database giving the user optimal enzyme combinations that facilitate the build of a given polysaccharide. This is achieved by using a        ranking system, where the rank of a combination of enzymes is dependant on the yield and rate of production of the desired polysaccharide Whilst the model is currently        uncalibrated, the software is ready to be united with GlycoWeb in this way.
The second motivation behind the modelling task may not have an instant and obvious application objective, but some would argue that it is a far more important reason to        invest in a piece of science. The model was intended to represent each reaction inside the cell mathematical and precisely. Upon alignment with data, the model could highlight       fundamental properties of the system we use. It could indicate assumptions which we held true about the system to be false and gives an abstract view of the biology which we        are working with.
The 3-Gene Inducible Plasmid  |   Contributors: Freddie Dudbridge, Alex Clowsley, Ryan Edginton, James Lynch
Result in Brief:
We set out to create a three gene inducible plasmid and nearly got there. At the end of the project we managed to add two genes to promoter, RBS and terminators and create the promoter_RBS and the gene terminator ready to put together for the third construct. A ligation of the two full constructs is in the freezer ready to be transformed. Although the project has come short and the goal has not been achieved, we were still able to use the constructs made to test for protein expression to help characterise the promoters we used. Given more time it would be great to be able to complete this side of the project in the future seeing as we managed to get so close.
Single Gene Plasmids and Enzyme Characterisation  |   Contributors: Freddie Dudbridge, Alex Clowsley,
Alex Baldwin, Alice Bond
Result in Brief:
This mini-project accomplished the significant cloning work required to generate the coding sequences required for operon construction and inducible gene expression, as well as for enzyme characterisation though it was not possible to conduct the enzyme assays on expressed genes. Importantly, this project also identified a suitable glycosyltransferase assay that could be used for characterisation of the enzymes. Principally, this would be to determine Vmax and KM values to help with the modelling of our system (see above). The assay relies on mass spectrometry to identify both oligosaccharide production and loss of diphosphonucleotide carrier. This mini-project resulted in the submission of the majority of single gene plasmids (BBa_K764000-BBa_764004, BBa_K754007, BBa_K764022 and BBa_K764023) in addition to a number of constructs (BBa_K764009, BBa_K764011, BBa_K764024, BBa_K764026, BBa_K764033 and BBa_K764034) to the Parts Registry. Descriptions are available on Bio-Bricks .
<< Return to Our Lab Books
See Our Achievements >>
Website Designed and Built by: Ryan Edginton, James Lynch & Alex Clowsley   |   Contact Us   |   Site Map
Retrieved from " http://2012.igem.org/Team:Exeter/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
