<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYMU-Taipei_experimentsandresults skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYMU-Taipei/experimentsandresults</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="container"><DIV class="nymu_menu"><DIV id="title">HAIR TO STAY</DIV><DIV class="dropdowns menu-part"><DIV class="dropdown"><DIV id="1r" onclick="closeintro(event),openintro(event)">
			TEAM
		</DIV><DIV class="dropdown-content" id="1c"><A href="https://2018.igem.org/Team:NYMU-Taipei/Team"><DIV>
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Collaborations"><DIV>
					Collaborations
				</DIV></A></DIV></DIV><DIV class="dropdown"><DIV id="2r" onclick="closeintro(event),openintro(event)">
			PROJECT
		</DIV><DIV class="dropdown-content" id="2c"><A href="https://2018.igem.org/Team:NYMU-Taipei/Description"><DIV>
					Description
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Design"><DIV>
					Design
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/experimentsandresults"><DIV>
					Experiments &amp; Results
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Notebook"><DIV>
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/InterLab"><DIV>
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Model"><DIV>
					Model
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Demonstrate"><DIV>
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Improve"><DIV>
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Attributions"><DIV>
					Attributions 
				</DIV></A></DIV></DIV><DIV class="dropdown"><DIV id="3r" onclick="closeintro(event),openintro(event)">
			PARTS
		</DIV><DIV class="dropdown-content" id="3c"><A href="https://2018.igem.org/Team:NYMU-Taipei/Parts"><DIV>
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Basic_Part"><DIV>
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Composite_Part"><DIV>
					Composite Parts
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Part_Collection"><DIV>
					Part Collection
				</DIV></A></DIV></DIV><A href="https://2018.igem.org/Team:NYMU-Taipei/Safety"><DIV class="dropdown direct_link">
				SAFETY
			</DIV></A><DIV class="dropdown"><DIV id="4r" onclick="closeintro(event),openintro(event)">
			HUMAN PRACTICES 
		</DIV><DIV class="dropdown-content" id="4c"><A href="https://2018.igem.org/Team:NYMU-Taipei/Human_Practices"><DIV>
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Public_Engagement"><DIV>
					Education &amp; Engagement
				</DIV></A></DIV></DIV><DIV class="dropdown"><DIV id="5r" onclick="closeintro(event),openintro(event)">
			AWARDS
		</DIV><DIV class="dropdown-content" id="5c"><A href="https://2018.igem.org/Team:NYMU-Taipei/Applied_Design"><DIV>
					Applied Design
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Entrepreneurship"><DIV>
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:NYMU-Taipei/Model"><DIV>
					Model
				</DIV></A></DIV></DIV><A href="https://igem.org/2018_Judging_Form?team=NYMU-Taipei"><DIV class="dropdown direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV></DIV><DIV id="page-content"><DIV class="column"><DIV class="story section"><DIV class="paragraphs"><H1 id="1" class="subtitle">Cell Model</H1><H1>Experiments and results</H1><DIV class="protocols"><DIV class="protocol"><DIV id="32t" class="trigger" onclick="detail(event)">Exp1. Constructing a reporter plasmid with DKK1 promoter</DIV><DIV id="32d" class="detail"><H4>Target</H4><P>We aim to construct reporter plasmids with human DKK1 promoter. The downstream reporter protein would be secreted upon activation of the DKK1 promoter.</P><H4>Method</H4><OL><LI><P>PCR of the DKK1 promoter:</P><P>We used PCR technique to acquire human DKK1 promoter sequence using the genomic DNA of HEK 293T cell line. The primers for the PCR are shown below.</P></LI><LI><P>We constructed the DKK1 promoter and mCherry reporter gene on pUC19 backbone.</P><P>The following figure is the cloning procedure of the DKK1 + mCherry plasmid.</P></LI></OL><H4>Results</H4><P>With high concentration of Mg2+, we got better PCR results from genomic DNA.</P></DIV></DIV><DIV class="protocol"><DIV id="33t" class="trigger" onclick="detail(event)">Exp2. Constructing a secreting reporter system</DIV><DIV id="33d" class="detail"><H4>Target</H4><P>We aim to construct a secreting reporter system so that fluorescence signal can be detected extracellularly using a plate reader instead of using a confocal microscope.</P><H4>Method</H4><OL><LI><P>Selecting a signal peptide and synthesizing appropriate primers:</P><P>After reviewing related literature[1], we decided to use albumin secretion peptide (ALB) as our secretion tag. ALB is located on serum-albumin-proproprotein-intron 1. </P></LI><LI><P>We used junction PCR to combine the ALB signal peptide with our reporter gene.</P></LI></OL><H4>Results</H4><P>We used the method of annealing oligonucleotides to get our desired segment of ALB signal peptide.</P><P>Reporters fused with ALB signal peptide were slightly longer (54bp) than our original segment.</P><H4>References</H4><OL><LI>Attallah, C., Etcheverrigaray, M., Kratje, R., &amp; Oggero, M. (2017). A highly efficient modified human serum albumin signal peptide to secrete proteins in cells derived from different mammalian species. Protein Expression and Purification, 132, 27-33. doi:10.1016/j.pep.2017.01.003
</LI></OL></DIV></DIV><DIV class="protocol"><DIV id="34t" class="trigger" onclick="detail(event)">Exp3. Constructing monomer EGFP (mEGFP)</DIV><DIV id="34d" class="detail"><H4>Target</H4><P>We aim to produce monomer EGFP (mEGFP), a modified version of EGFP. mEGFP has the same function as EGFP but tends to form monomers while EGFP forms dimers.</P><H4>Method</H4><P>We did a point mutation and changed 206a.a. from Ala into Lys.</P><H4>Results</H4><P>We did fusion PCR to get mutated EGFP.</P><P>Plasmid containing mEGFP was transfected into HEK 293T cells. Green fluorescence was seen under confocal microscope.</P></DIV></DIV><DIV class="protocol"><DIV id="35t" class="trigger" onclick="detail(event)">Exp 4. Transfecting cells with plasmids containing reporter gene</DIV><DIV id="35d" class="detail"><H4>Target</H4><P>We want to transfect cells with the plasmids that we have constructed and evaluate whether our reporter genes are functional.</P><H4>Method</H4><OL><LI>We transfected the plasmids into HEK 293T cells via calcium phosphate transfection method. </LI><LI>We tested both intracellular and extracellular expression of mCherry and mEGFP under CMV promoter. We also tested the expression of mCherry under DKK1 promoter. </LI></OL><H4>Result</H4></DIV></DIV><DIV class="protocol" id="exp5"><DIV id="36t" class="trigger" onclick="detail(event)">Exp 5. Testing on the relationship between testosterone concentration and mCherry expression</DIV><DIV id="36d" class="detail"><H4>Target</H4><P>According to literature, testosterone can be converted into dihydrotestosterone (DHT) via 5-alpha reductase, and thereby induce DKK1 production [1]. We were interested in how testosterone concentration interferes with mCherry expression under the control of a DKK1 promoter.</P><H4>Method</H4><OL><LI>We transfected HEK 293T cells with a plasmid containing DKK1 promoter and mCherry reporter.</LI><LI>We added in 0.05ug, 0.5ug, and 5ug of testosterone 6 hours post-transfection, and observed under confocal microscope after 24 hours. All photos were taken under the same exposure time. Quantification of cells expressing the mCherry protein was determined via Metamorph multi-wavelength cell scoring. Intensity 10 graylevels above local background was determined as real fluorescence emission.</LI></OL><H4>Result</H4><H4>References</H4><OL><LI>Inui, S., &amp; Itami, S. (2011). Molecular basis of androgenetic alopecia: From androgen to paracrine mediators through dermal papilla. Journal of Dermatological Science, 61(1), 1-6. doi:10.1016/j.jdermsci.2010.10.015 </LI></OL></DIV></DIV><DIV class="protocol"><DIV id="37t" class="trigger" onclick="detail(event)">Exp 6. Testing on the relationship between dihydrotestosterone (DHT) concentration and mCherry expression</DIV><DIV id="37d" class="detail"><H4>Target</H4><P>DKK1 protein has been proved to be dihydrotestosterone (DHT) inducible. Therefore, we were interested in how testosterone concentration interferes with mCherry expression under the control of a DKK1 promoter.</P><H4>Method</H4><OL><LI>We transfected HEK 293T cells with a plasmid containing DKK1 promoter and mCherry reporter.</LI><LI>We added in 0nM, 10nM, 50nM, 100nM, 150nM, 200nM of DHT 6 hours post-transfection, and observed under confocal microscope after 12 and 24 hours. </LI></OL><H4>Result</H4></DIV></DIV><DIV class="protocol"><DIV id="38t" class="trigger" onclick="detail(event)">Exp7. Optimizing assay time for our screening system </DIV><DIV id="38d" class="detail"><H4>Target</H4><P>We want to understand the peaking time of mCherry expression in order to optimize the appropriate timing for mCherry result detection.</P><H4>Method</H4><P>We transfected both CMV promoter + mCherry and CMV promoter + ALB + mCherry plasmids into HEK 293T cells. Although the CMV promoter + ALB + mCherry plasmid is in secrete form, we still expect to be able to see mCherry intracellularly. Fluorescent photos were taken 12, 24, and 36 hours post transfection. </P><H4>Result</H4><P>We decided that 24 hours after transfection is when mCherry expression peaks. Therefore, it is the best time for us to detect results.</P></DIV></DIV><DIV class="protocol"><DIV id="39t" class="trigger" onclick="detail(event)">Exp8. Co-transfection of internal control </DIV><DIV id="39d" class="detail"><H4>Target</H4><P>To account for differences in transfection conditions, we used the CMV promoter + mEGFP plasmid as an internal control.</P><H4>Method</H4><P>We co-transfected the DKK1 promoter + mCherry plasmid and the CMV promoter + mEGFP plasmid into cells. The CMV promoter + mEGFP plasmid is used as internal control to standardize different samples. </P><H4>Result</H4></DIV></DIV></DIV></DIV></DIV><DIV class="story section"><DIV class="paragraphs"><H1 id="2" class="subtitle">FRET Model</H1><H1>Experiments and results</H1><DIV class="protocols"><DIV class="protocol"><DIV id="26t" class="trigger" onclick="detail(event)">Exp1. Construction of “The Skeletons”</DIV><DIV id="26d" class="detail"><H4>Target</H4><P>We aim to replace the Multiple Cloning Site (MCS) of pET32a, which is the plasmid we use for protein expression, with the new MCS that we designed.</P><H4>Method</H4><OL><LI>pET32a’s MCS was cleaved off and replaced by inserting with FPF-Skel1designed by us. This is the plasmid pET32a FPF-skel1, which was designed to accommodate a DKK1-binding protein fused to the N-terminus of a fluorescent protein.</LI><LI>Polyhistidine tag (His tag), which was synthesized as two single-stranded DNA fragments and annealed by ourselves, was inserted into the new MCS of pET32a FPF-skel1. This is the plasmid pET32a-FPFSkel2, whose new MCS can accommodate a DKK1-binding protein fused to the C-terminus of a fluorescent protein. </LI></OL><H4>Results</H4></DIV></DIV><DIV class="protocol"><DIV id="27t" class="trigger" onclick="detail(event)">Exp2. Insertion of CyPet and YPet</DIV><DIV id="27d" class="detail"><H4>Target</H4><P>To insert CyPet and YPet, which are the fluorescent proteins we use for FRET, into the skeletons that we constructed in Exp1.</P><H4>Method</H4><P>CyPet and YPet, which are amplified by PCR, are inserted into pET32a-FPFSkel1, creating pET32a-FPFSkel1-CyPet and pET32a-FPFSkel1-YPet, respectively.</P><H4>Results</H4></DIV></DIV><DIV class="protocol"><DIV id="28t" class="trigger" onclick="detail(event)">Exp3. Insertion of DKK1-binding proteins</DIV><DIV id="28d" class="detail"><H4>Target</H4><P>To insert the nucleotide sequence of DKK1-binding proteins into pET32a-FPFSkel1-CyPet and pET32a-FPFSkel1-YPet, constructed in Exp2.</P><H4>Method</H4><P>LRP6BP1, LRP6BP1BP2, LRP6BP3, LRP6BP3BP4, G5(VHH for DKK1), H7(VHH for DKK1) are amplified by PCR and inserted into pET32a-FPFSkel1-CyPet or pET32a-FPFSkel1-YPet or both. Creating a list of new plasmids as follows: </P><OL><LI>pET32a FPF-Skel1 YPet-Ubc9 </LI><LI>pET32a FPF-Skel1 YPet-H7</LI><LI>pET32a FPF-Skel1 YPet-E1E2 </LI><LI>pET32a FPF-Skel1 YPet-E3E4 </LI><LI>pET32a FPF-Skel1 YPet-E1 </LI><LI>pET32a FPF-Skel1 YPet-E3 </LI><LI>pET32a FPF-Skel1 CyPet-SUMO1 </LI><LI>pET32a FPF-Skel1 CyPet-VHH G5 </LI><LI>pET32a FPF-Skel1 CyPet-VHH H7 </LI><LI>pET32a FPF-Skel1 CyPet-E3E4 </LI><LI>pET32a FPF-Skel1 CyPet-E3 </LI></OL><H4>Results</H4></DIV></DIV><DIV class="protocol"><DIV id="29t" class="trigger" onclick="detail(event)">Exp4. Mass Production of Fusion Proteins</DIV><DIV id="29d" class="detail"><H4>Target</H4><P>To mass produce and purify fusion proteins that are used later in our in vitro DKK1 quantification.</P><H4>Method</H4><OL><LI>Plasmids constructed in Exp4 are transformed into E. coli BL21 strain. BL21 is incubated in flask and induced with lactose or IPTG.</LI><LI>BL21 cells are disrupted and the proteins are purified through affinity column.</LI></OL><H4>Results</H4></DIV></DIV><DIV class="protocol" id="eexp5"><DIV id="30t" class="trigger" onclick="detail(event)">Exp5. FRET Positive Control</DIV><DIV id="30d" class="detail"><H4>Target</H4><P>To confirm that we correctly carry out a FRET experiment.</P><H4>Method</H4><OL><LI>SUMO1 and Ubc9 are amplified by PCR and inserted into pET32a-FPFSkel1 CyPet and pET32a-FPFSkel1 YPet respectively, resulting in pET32a-FPFSkel1-YPet-Ubc9 and pET32a-FPFSkel1 CyPet SUMO1</LI><LI>The two plasmids are transformed into BL21. BL21 is incubated in flask and induced with lactose or IPTG.</LI><LI>BL21 cells are disrupted and the proteins are purified through column, producing SUMO1-CyPet and Ubc9-YPet fusion proteins.</LI><LI>The two fusion proteins are added together with varied concentrations and ratios in a proper buffer and observed with spectrophotometer plate readers, with xx being the negative control.</LI></OL><H4>Results</H4></DIV></DIV><DIV class="protocol"><DIV id="31t" class="trigger" onclick="detail(event)">Exp6. Quantification of DKK1</DIV><DIV id="31d" class="detail"><H4>Target</H4><P>To test if the proteins we made can be used to determine the amount of DKK1.</P><H4>Method</H4><OL><LI>Two fusion proteins of different FRET pairs are added into a proper buffer with varied concentrations of DKK1 and ratio of the FRET pairs. The buffer without DKK1 is used as the control.</LI><LI>The results are measured with spectrophotometer plate readers.</LI></OL><H4>Results</H4></DIV></DIV></DIV></DIV></DIV><DIV class="story section"><DIV class="paragraphs"><H1 id="3" class="subtitle">Protocols</H1><H2>E. coli Protocols</H2><DIV class="protocols"><DIV class="protocol"><DIV id="1t" class="trigger" onclick="detail(event)">Two-day Efficient Cloning Cycle</DIV><DIV id="1d" class="detail"><P>We used an efficient two-day cloning cycle split into a &quot;Light&quot; day and a &quot;Heavy&quot; day.</P><H3>Light Day</H3><P>The light day consists of : Colony PCR and liquid culture of colonies transformed from a previous day.</P><OL><LI>
The 3-in-1
<P>First, count the number of colonies that you want to check. Then, do the following 3 things sequentially:</P></LI><LI>Liquid culture</LI><LI>2nd time plate</LI><LI>Colony PCR</LI><P>(Use the same tip to add the template to these three things)</P><LI>Make The Gel For Electrophoresis</LI><LI>Run Gel Electrophoresis To Check The Colony PCR Product</LI></OL><H3>Heavy Day</H3><P>The heavy day consists of:</P><OL><LI>Previously grown plasmid extraction</LI><LI>Plasmid PCR</LI><LI>Gel extraction</LI><LI>Digestion</LI><LI>Ligation</LI><LI>Transformation
</LI></OL></DIV></DIV><DIV class="protocol"><DIV id="2t" class="trigger" onclick="detail(event)">Colony PCR (Thermo DreamTaq)</DIV><DIV id="2d" class="detail"><OL><LI>Make the Colony PCR mix (we use Thermo' DreamTaq) with the mix amount slightly modified:
<TABLE border="1"><TBODY><TR><TH scope="col">Item</TH><TH scope="col">uL</TH></TR><TR><TD style="width:800;">Primer(Forward and reverse)</TD><TD style="width:200;">1</TD></TR><TR><TD style="width:800;">dNTP (10mM)</TD><TD style="width:200;">1</TD></TR><TR><TD style="width:800;">10x DreamTaq buffer</TD><TD style="width:200;">5</TD></TR><TR><TD style="width:800;">Taq Polymerase</TD><TD style="width:200;">0.2</TD></TR><TR><TD style="width:800;">ddH20</TD><TD style="width:200;">42.8</TD></TR><TR><TD style="width:800;">Total</TD><TD style="width:200;">50</TD></TR></TBODY></TABLE></LI><LI>Select a colony using a tip or toothpick.</LI><LI>Dip it in a PCR tube and swirl it around.
<P>PCR run protocol</P><TABLE border="1"><TBODY><TR><TD style="width:400;">Temperature</TD><TD style="width:500;">Time</TD><TD style="width:400;"> </TD></TR><TR><TD style="width:400;">94℃</TD><TD style="width:500;">60s</TD><TD style="width:400;"> </TD></TR><TR><TD style="width:400;">94℃</TD><TD style="width:500;">15s</TD><TD style="width:400;"> </TD></TR><TR><TD style="width:400;">55℃</TD><TD style="width:500;">20s</TD><TD style="width:400;">30-35 cycles</TD></TR><TR><TD style="width:400;">72℃</TD><TD style="width:500;">1kb/min + 5-10s</TD><TD style="width:400;"> </TD></TR><TR><TD style="width:400;">72℃</TD><TD style="width:500;">300s</TD><TD style="width:400;"> </TD></TR></TBODY></TABLE></LI></OL></DIV></DIV><DIV class="protocol"><DIV id="3t" class="trigger" onclick="detail(event)">Liquid culture</DIV><DIV id="3d" class="detail"><OL><LI>Aliquot 4 mL of LB medium in a centrifugal tube for each colony.</LI><LI>Use a tip and dip it in the LB culture and swirls it around.</LI></OL></DIV></DIV><DIV class="protocol"><DIV id="4t" class="trigger" onclick="detail(event)">Draw on 2nd time plate</DIV><DIV id="4d" class="detail"><P>Take out new plates, for second-time purpose, from the fridge and divide them into smaller sections. Label the date  properly. Cross out in red pen if any section is wrong after the Colony PCR is checked.</P></DIV></DIV><DIV class="protocol"><DIV id="5t" class="trigger" onclick="detail(event)">Making Gel for gel electrophoresis</DIV><DIV id="5d" class="detail"><OL><LI>Make 200mL of gel at a time and select a relevant percentage (e.g. 1%, 1.5%, or 2%)</LI><LI>Measure and calculate the relevant percentage in agarose (e.g. 1g of agarose for 100mL of 1% gel).</LI><LI>Fill it up with 1xTAE buffer.</LI><LI>Microwave in few seconds, constantly taking it out to swirl and mix it each time. Keep radiating until solution turn transparent. Make sure it is completely transparent</LI><LI>Cool down the temperature of solution but make sure there is still a warmness in the flask. If the gel gets too cold and starts to harden, reheat the solution by microwave oven. Make sure it is transparent</LI><LI>Add 5uL of Safe-seeing dye for every 100mL of gel after cooled to the fine tempearature. Mix it well by swirling</LI><LI>Pour it onto the molds quickly. Put a cover on it to block out the light.</LI><LI>Wait at least 15-20 min until using it.</LI><LI>Store in 4°C refridgerator and away from light.</LI></OL></DIV></DIV><DIV class="protocol"><DIV id="6t" class="trigger" onclick="detail(event)">Gel electrophoresis</DIV><DIV id="6d" class="detail"><OL><LI>Select a relevant percentage agarose gel based on your own experience</LI><LI>Load 5uL from each tube of Colony PCR, mix it with 1uL of 6x DNA Dye and put it in a well.</LI><LI>Load 3uL of marker into a well.</LI><LI>Run in the 13x13cm box at 60V or 70V and 400mA for the desired amount of time.</LI><LI>View in the gel viewer machine.</LI></OL></DIV></DIV><DIV class="protocol"><DIV id="7t" class="trigger" onclick="detail(event)">Plasmid Extraction</DIV></DIV><DIV class="protocol"><DIV id="8t" class="trigger" onclick="detail(event)">PCR (DreamTaq or KOD)</DIV><DIV id="8d" class="detail"><TABLE border="1" align="left"><TBODY><TR><TH style="width:800;" scope="col">Item</TH><TH style="width:200;" scope="col">uL</TH></TR><TR><TD style="width:800;">Primer(Forward and reverse)</TD><TD style="width:200;">0.2</TD></TR><TR><TD style="width:800;">Template(100ng)</TD><TD style="width:200;">?</TD></TR><TR><TD style="width:800;">dNTP (10mM)</TD><TD style="width:200;">0.2</TD></TR><TR><TD style="width:800;">10x DreamTaq buffer</TD><TD style="width:200;">1</TD></TR><TR><TD style="width:800;">Taq Polymerase</TD><TD style="width:200;">0.04</TD></TR><TR><TD style="width:800;">ddH20</TD><TD style="width:200;">to 10 uL</TD></TR><TR><TD style="width:800;">Total</TD><TD style="width:200;">10</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH style="width:800;" scope="col">Item</TH><TH style="width:200;" scope="col">uL</TH></TR><TR><TD style="width:800;">Primer(Forward and reverse)</TD><TD style="width:200;">0.2</TD></TR><TR><TD style="width:800;">Template(100ng)</TD><TD style="width:200;">?</TD></TR><TR><TD style="width:800;">dNTP (10mM)</TD><TD style="width:200;">0.2</TD></TR><TR><TD style="width:800;">10x KOD buffer</TD><TD style="width:200;">1</TD></TR><TR><TD style="width:800;">KOD Polymerase</TD><TD style="width:200;">0.04</TD></TR><TR><TD style="width:800;">Mg2+</TD></TR><TR><TD style="width:800;">ddH20</TD><TD style="width:200;">to 10 uL</TD></TR><TR><TD style="width:800;">Total</TD><TD style="width:200;">10</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="protocol"><DIV id="9t" class="trigger" onclick="detail(event)">Digestion</DIV><DIV id="9d" class="detail"><P>Insert and backbone:</P><TABLE border="1"><TBODY><TR><TH style="width:500;" scope="col">Item</TH></TR><TR><TD style="width:500;">DNA</TD><TD style="width:500;">600 or 1000ng</TD></TR><TR><TD style="width:500;">10x Buffer</TD><TD style="width:500;">2uL</TD></TR><TR><TD style="width:500;">Enzyme1</TD><TD style="width:500;">0.6 or 1 uL</TD></TR><TR><TD style="width:500;">Enzyme2</TD><TD style="width:500;">0.6 or 1 uL</TD></TR><TR><TD style="width:500;">ddH2O</TD><TD style="width:500;">to 20 uL</TD></TR><TR><TD style="width:500;">Total</TD><TD style="width:500;">20</TD></TR></TBODY></TABLE><P>Backbone check:To check if the backbone is cut.</P><TABLE border="1"><TBODY><TR><TH style="width:500;" scope="col">Item</TH></TR><TR><TD style="width:500;">DNA</TD><TD style="width:500;">100ng</TD></TR><TR><TD style="width:500;">10x Buffer</TD><TD style="width:500;">1 uL</TD></TR><TR><TD style="width:500;">Enzyme1 or 2</TD><TD style="width:500;">0.1 uL</TD></TR><TR><TD style="width:500;">ddH2O</TD><TD style="width:500;">to 10 uL</TD></TR><TR><TD style="width:500;">Total</TD><TD style="width:500;">10</TD></TR></TBODY></TABLE><P>Digest for at least 1hr;over 2hr is better. EcoRI doesn't have star activity when cut this way (even overnight)</P><P>We run insert digests and backbone digests with backbone check on a gel. Then use GeneAid's gel extraction kit. The modifications in the protocol include:</P><P>Warm EB to 60-70℃ before elution.</P><P>Always use the Gel (sequencing) protocol for gel extraction.</P></DIV></DIV><DIV class="protocol"><DIV id="10t" class="trigger" onclick="detail(event)">Ligation</DIV><DIV id="10d" class="detail"><P>We use Thermo's and NEB's T4 Ligase:</P><TABLE border="1"><TBODY><TR><TD style="width:300;">Item</TD><TD style="width:700;">amount</TD></TR><TR><TD style="width:300;">Vector</TD><TD style="width:700;">8.5uL, total of approximately 100ng of DNA.</TD></TR><TR><TD style="width:300;">Insert</TD></TR><TR><TD style="width:300;">ddH2O</TD></TR><TR><TD style="width:300;">10x Ligase Buffer</TD><TD style="width:700;">1</TD></TR><TR><TD style="width:300;">T4 Ligase</TD><TD style="width:700;">0.5</TD></TR><TR><TD style="width:300;">Total</TD><TD style="width:700;">10</TD></TR></TBODY></TABLE><P>Incubate at room temperature for 2hr then transform 1uL then put the remaining amount in a small bag and put it in 4℃ overnight in case the transformation fails and retransformation is required.
</P></DIV></DIV><DIV class="protocol"><DIV id="11t" class="trigger" onclick="detail(event)">Transformation</DIV><DIV id="11d" class="detail"><P>We majorly use commercial E. coli DH5α competent cells and BL21 competent cells we made by ourselves.</P><UL><LI>Add 1uL of plasmid or ligation mix to 20 uL of competent cells.</LI><LI>Put mixture on ice for 30 minutes.</LI><LI>Heat shock at 42℃ for 1 min.</LI><LI>Put the mixture back on ice for another 20 minutes.</LI><LI>Add 200 uL of LB broth to repair the cell wall; incubate at 37℃ for 1.5 hr.</LI><LI>Plate it on a relevant antibiotic plate.</LI><LI>Incubate plate at 37℃ overnight.</LI></UL></DIV></DIV><DIV class="protocol"><DIV id="12t" class="trigger" onclick="detail(event)">Glycerol Stock</DIV></DIV><DIV class="protocol"><DIV id="13t" class="trigger" onclick="detail(event)">Genomic DNA Extraction</DIV></DIV><DIV class="protocol"><DIV id="14t" class="trigger" onclick="detail(event)">Pouring Agar Plates</DIV></DIV><DIV class="protocol"><DIV id="15t" class="trigger" onclick="detail(event)">Preparation of Competent E.coli</DIV></DIV><DIV class="protocol"><DIV id="16t" class="trigger" onclick="detail(event)">Annealing</DIV></DIV><DIV class="protocol"><DIV id="25t" class="trigger" onclick="detail(event)">Protein Purification</DIV></DIV><DIV class="protocol"><DIV id="24t" class="trigger" onclick="detail(event)">Lac operon induction and E.coli cell disruption</DIV></DIV></DIV><H2>Mammalian Cells Protocols</H2><DIV class="protocols"><DIV class="protocol"><DIV id="17t" class="trigger" onclick="detail(event)">Transfection</DIV></DIV><DIV class="protocol"><DIV id="18t" class="trigger" onclick="detail(event)">HEK293 Culture</DIV></DIV><DIV class="protocol"><DIV id="19t" class="trigger" onclick="detail(event)">Human DKK1 ELISA</DIV></DIV><DIV class="protocol"><DIV id="20t" class="trigger" onclick="detail(event)">Freezing And Thawing Cells</DIV></DIV><DIV class="protocol"><DIV id="21t" class="trigger" onclick="detail(event)">Handling Cells Upon Arrival</DIV></DIV><DIV class="protocol"><DIV id="22t" class="trigger" onclick="detail(event)">Important Considerations For Cell Culture</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>