Team:NYMU-Taipei/experimentsandresults
HAIR TO STAY
TEAM
Team Members
Collaborations
PROJECT
Description
Design
Experiments & Results
Notebook
InterLab
Model
Demonstrate
Improve
Attributions
PARTS
Parts Overview
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Human Practices
Education & Engagement
AWARDS
Applied Design
Entrepreneurship
Model
JUDGING FORM ⇗
Cell Model
Experiments and results
Exp1. Constructing a reporter plasmid with DKK1 promoter
Target
We aim to construct reporter plasmids with human DKK1 promoter. The downstream reporter protein would be secreted upon activation of the DKK1 promoter.
Method
PCR of the DKK1 promoter:
We used PCR technique to acquire human DKK1 promoter sequence using the genomic DNA of HEK 293T cell line. The primers for the PCR are shown below.
We constructed the DKK1 promoter and mCherry reporter gene on pUC19 backbone.
The following figure is the cloning procedure of the DKK1 + mCherry plasmid.
Results
With high concentration of Mg2+, we got better PCR results from genomic DNA.
Exp2. Constructing a secreting reporter system
Target
We aim to construct a secreting reporter system so that fluorescence signal can be detected extracellularly using a plate reader instead of using a confocal microscope.
Method
Selecting a signal peptide and synthesizing appropriate primers:
After reviewing related literature[1], we decided to use albumin secretion peptide (ALB) as our secretion tag. ALB is located on serum-albumin-proproprotein-intron 1.
We used junction PCR to combine the ALB signal peptide with our reporter gene.
Results
We used the method of annealing oligonucleotides to get our desired segment of ALB signal peptide.
Reporters fused with ALB signal peptide were slightly longer (54bp) than our original segment.
References
Attallah, C., Etcheverrigaray, M., Kratje, R., & Oggero, M. (2017). A highly efficient modified human serum albumin signal peptide to secrete proteins in cells derived from different mammalian species. Protein Expression and Purification, 132, 27-33. doi:10.1016/j.pep.2017.01.003
Exp3. Constructing monomer EGFP (mEGFP)
Target
We aim to produce monomer EGFP (mEGFP), a modified version of EGFP. mEGFP has the same function as EGFP but tends to form monomers while EGFP forms dimers.
Method
We did a point mutation and changed 206a.a. from Ala into Lys.
Results
We did fusion PCR to get mutated EGFP.
Plasmid containing mEGFP was transfected into HEK 293T cells. Green fluorescence was seen under confocal microscope.
Exp 4. Transfecting cells with plasmids containing reporter gene
Target
We want to transfect cells with the plasmids that we have constructed and evaluate whether our reporter genes are functional.
Method
We transfected the plasmids into HEK 293T cells via calcium phosphate transfection method.
We tested both intracellular and extracellular expression of mCherry and mEGFP under CMV promoter. We also tested the expression of mCherry under DKK1 promoter.
Result
Exp 5. Testing on the relationship between testosterone concentration and mCherry expression
Target
According to literature, testosterone can be converted into dihydrotestosterone (DHT) via 5-alpha reductase, and thereby induce DKK1 production [1]. We were interested in how testosterone concentration interferes with mCherry expression under the control of a DKK1 promoter.
Method
We transfected HEK 293T cells with a plasmid containing DKK1 promoter and mCherry reporter.
We added in 0.05ug, 0.5ug, and 5ug of testosterone 6 hours post-transfection, and observed under confocal microscope after 24 hours. All photos were taken under the same exposure time. Quantification of cells expressing the mCherry protein was determined via Metamorph multi-wavelength cell scoring. Intensity 10 graylevels above local background was determined as real fluorescence emission.
Result
References
Inui, S., & Itami, S. (2011). Molecular basis of androgenetic alopecia: From androgen to paracrine mediators through dermal papilla. Journal of Dermatological Science, 61(1), 1-6. doi:10.1016/j.jdermsci.2010.10.015
Exp 6. Testing on the relationship between dihydrotestosterone (DHT) concentration and mCherry expression
Target
DKK1 protein has been proved to be dihydrotestosterone (DHT) inducible. Therefore, we were interested in how testosterone concentration interferes with mCherry expression under the control of a DKK1 promoter.
Method
We transfected HEK 293T cells with a plasmid containing DKK1 promoter and mCherry reporter.
We added in 0nM, 10nM, 50nM, 100nM, 150nM, 200nM of DHT 6 hours post-transfection, and observed under confocal microscope after 12 and 24 hours.
Result
Exp7. Optimizing assay time for our screening system
Target
We want to understand the peaking time of mCherry expression in order to optimize the appropriate timing for mCherry result detection.
Method
We transfected both CMV promoter + mCherry and CMV promoter + ALB + mCherry plasmids into HEK 293T cells. Although the CMV promoter + ALB + mCherry plasmid is in secrete form, we still expect to be able to see mCherry intracellularly. Fluorescent photos were taken 12, 24, and 36 hours post transfection.
Result
We decided that 24 hours after transfection is when mCherry expression peaks. Therefore, it is the best time for us to detect results.
Exp8. Co-transfection of internal control
Target
To account for differences in transfection conditions, we used the CMV promoter + mEGFP plasmid as an internal control.
Method
We co-transfected the DKK1 promoter + mCherry plasmid and the CMV promoter + mEGFP plasmid into cells. The CMV promoter + mEGFP plasmid is used as internal control to standardize different samples.
Result
FRET Model
Experiments and results
Exp1. Construction of “The Skeletons”
Target
We aim to replace the Multiple Cloning Site (MCS) of pET32a, which is the plasmid we use for protein expression, with the new MCS that we designed.
Method
pET32a’s MCS was cleaved off and replaced by inserting with FPF-Skel1designed by us. This is the plasmid pET32a FPF-skel1, which was designed to accommodate a DKK1-binding protein fused to the N-terminus of a fluorescent protein.
Polyhistidine tag (His tag), which was synthesized as two single-stranded DNA fragments and annealed by ourselves, was inserted into the new MCS of pET32a FPF-skel1. This is the plasmid pET32a-FPFSkel2, whose new MCS can accommodate a DKK1-binding protein fused to the C-terminus of a fluorescent protein.
Results
Exp2. Insertion of CyPet and YPet
Target
To insert CyPet and YPet, which are the fluorescent proteins we use for FRET, into the skeletons that we constructed in Exp1.
Method
CyPet and YPet, which are amplified by PCR, are inserted into pET32a-FPFSkel1, creating pET32a-FPFSkel1-CyPet and pET32a-FPFSkel1-YPet, respectively.
Results
Exp3. Insertion of DKK1-binding proteins
Target
To insert the nucleotide sequence of DKK1-binding proteins into pET32a-FPFSkel1-CyPet and pET32a-FPFSkel1-YPet, constructed in Exp2.
Method
LRP6BP1, LRP6BP1BP2, LRP6BP3, LRP6BP3BP4, G5(VHH for DKK1), H7(VHH for DKK1) are amplified by PCR and inserted into pET32a-FPFSkel1-CyPet or pET32a-FPFSkel1-YPet or both. Creating a list of new plasmids as follows:
pET32a FPF-Skel1 YPet-Ubc9
pET32a FPF-Skel1 YPet-H7
pET32a FPF-Skel1 YPet-E1E2
pET32a FPF-Skel1 YPet-E3E4
pET32a FPF-Skel1 YPet-E1
pET32a FPF-Skel1 YPet-E3
pET32a FPF-Skel1 CyPet-SUMO1
pET32a FPF-Skel1 CyPet-VHH G5
pET32a FPF-Skel1 CyPet-VHH H7
pET32a FPF-Skel1 CyPet-E3E4
pET32a FPF-Skel1 CyPet-E3
Results
Exp4. Mass Production of Fusion Proteins
Target
To mass produce and purify fusion proteins that are used later in our in vitro DKK1 quantification.
Method
Plasmids constructed in Exp4 are transformed into E. coli BL21 strain. BL21 is incubated in flask and induced with lactose or IPTG.
BL21 cells are disrupted and the proteins are purified through affinity column.
Results
Exp5. FRET Positive Control
Target
To confirm that we correctly carry out a FRET experiment.
Method
SUMO1 and Ubc9 are amplified by PCR and inserted into pET32a-FPFSkel1 CyPet and pET32a-FPFSkel1 YPet respectively, resulting in pET32a-FPFSkel1-YPet-Ubc9 and pET32a-FPFSkel1 CyPet SUMO1
The two plasmids are transformed into BL21. BL21 is incubated in flask and induced with lactose or IPTG.
BL21 cells are disrupted and the proteins are purified through column, producing SUMO1-CyPet and Ubc9-YPet fusion proteins.
The two fusion proteins are added together with varied concentrations and ratios in a proper buffer and observed with spectrophotometer plate readers, with xx being the negative control.
Results
Exp6. Quantification of DKK1
Target
To test if the proteins we made can be used to determine the amount of DKK1.
Method
Two fusion proteins of different FRET pairs are added into a proper buffer with varied concentrations of DKK1 and ratio of the FRET pairs. The buffer without DKK1 is used as the control.
The results are measured with spectrophotometer plate readers.
Results
Protocols
E. coli Protocols
Two-day Efficient Cloning Cycle
We used an efficient two-day cloning cycle split into a "Light" day and a "Heavy" day.
Light Day
The light day consists of : Colony PCR and liquid culture of colonies transformed from a previous day.
The 3-in-1
First, count the number of colonies that you want to check. Then, do the following 3 things sequentially:
Liquid culture
2nd time plate
Colony PCR
(Use the same tip to add the template to these three things)
Make The Gel For Electrophoresis
Run Gel Electrophoresis To Check The Colony PCR Product
Heavy Day
The heavy day consists of:
Previously grown plasmid extraction
Plasmid PCR
Gel extraction
Digestion
Ligation
Transformation
Colony PCR (Thermo DreamTaq)
Make the Colony PCR mix (we use Thermo' DreamTaq) with the mix amount slightly modified:
Item
uL
Primer(Forward and reverse)
1
dNTP (10mM)
1
10x DreamTaq buffer
5
Taq Polymerase
0.2
ddH20
42.8
Total
50
Select a colony using a tip or toothpick.
Dip it in a PCR tube and swirl it around.
PCR run protocol
Temperature
Time
 
94℃
60s
 
94℃
15s
 
55℃
20s
30-35 cycles
72℃
1kb/min + 5-10s
 
72℃
300s
 
Liquid culture
Aliquot 4 mL of LB medium in a centrifugal tube for each colony.
Use a tip and dip it in the LB culture and swirls it around.
Draw on 2nd time plate
Take out new plates, for second-time purpose, from the fridge and divide them into smaller sections. Label the date  properly. Cross out in red pen if any section is wrong after the Colony PCR is checked.
Making Gel for gel electrophoresis
Make 200mL of gel at a time and select a relevant percentage (e.g. 1%, 1.5%, or 2%)
Measure and calculate the relevant percentage in agarose (e.g. 1g of agarose for 100mL of 1% gel).
Fill it up with 1xTAE buffer.
Microwave in few seconds, constantly taking it out to swirl and mix it each time. Keep radiating until solution turn transparent. Make sure it is completely transparent
Cool down the temperature of solution but make sure there is still a warmness in the flask. If the gel gets too cold and starts to harden, reheat the solution by microwave oven. Make sure it is transparent
Add 5uL of Safe-seeing dye for every 100mL of gel after cooled to the fine tempearature. Mix it well by swirling
Pour it onto the molds quickly. Put a cover on it to block out the light.
Wait at least 15-20 min until using it.
Store in 4°C refridgerator and away from light.
Gel electrophoresis
Select a relevant percentage agarose gel based on your own experience
Load 5uL from each tube of Colony PCR, mix it with 1uL of 6x DNA Dye and put it in a well.
Load 3uL of marker into a well.
Run in the 13x13cm box at 60V or 70V and 400mA for the desired amount of time.
View in the gel viewer machine.
Plasmid Extraction
PCR (DreamTaq or KOD)
Item
uL
Primer(Forward and reverse)
0.2
Template(100ng)
?
dNTP (10mM)
0.2
10x DreamTaq buffer
1
Taq Polymerase
0.04
ddH20
to 10 uL
Total
10
Item
uL
Primer(Forward and reverse)
0.2
Template(100ng)
?
dNTP (10mM)
0.2
10x KOD buffer
1
KOD Polymerase
0.04
Mg2+
ddH20
to 10 uL
Total
10
Digestion
Insert and backbone:
Item
DNA
600 or 1000ng
10x Buffer
2uL
Enzyme1
0.6 or 1 uL
Enzyme2
0.6 or 1 uL
ddH2O
to 20 uL
Total
20
Backbone check:
To check if the backbone is cut.
Item
DNA
100ng
10x Buffer
1 uL
Enzyme1 or 2
0.1 uL
ddH2O
to 10 uL
Total
10
Digest for at least 1hr;over 2hr is better. EcoRI doesn't have star activity when cut this way (even overnight)
We run insert digests and backbone digests with backbone check on a gel. Then use GeneAid's gel extraction kit. The modifications in the protocol include:
Warm EB to 60-70℃ before elution.
Always use the Gel (sequencing) protocol for gel extraction.
Ligation
We use Thermo's and NEB's T4 Ligase:
Item
amount
Vector
8.5uL, total of approximately 100ng of DNA.
Insert
ddH2O
10x Ligase Buffer
1
T4 Ligase
0.5
Total
10
Incubate at room temperature for 2hr then transform 1uL then put the remaining amount in a small bag and put it in 4℃ overnight in case the transformation fails and retransformation is required.
Transformation
We majorly use commercial E. coli DH5α competent cells and BL21 competent cells we made by ourselves.
Add 1uL of plasmid or ligation mix to 20 uL of competent cells.
Put mixture on ice for 30 minutes.
Heat shock at 42℃ for 1 min.
Put the mixture back on ice for another 20 minutes.
Add 200 uL of LB broth to repair the cell wall; incubate at 37℃ for 1.5 hr.
Plate it on a relevant antibiotic plate.
Incubate plate at 37℃ overnight.
Glycerol Stock
Genomic DNA Extraction
Pouring Agar Plates
Preparation of Competent E.coli
Annealing
Protein Purification
Lac operon induction and E.coli cell disruption
Mammalian Cells Protocols
Transfection
HEK293 Culture
Human DKK1 ELISA
Freezing And Thawing Cells
Handling Cells Upon Arrival
Important Considerations For Cell Culture
