Team:Cambridge-JIC/Homoplasmy
Cambridge-JIC
Toggle navigation
Project
Description
Design
Motivation
Modelling
Biolistics
Growth Facility
Homoplasmy Strategy
Proof Of Concept
Demonstration
Parts
Experiment
Results
Notebook
Team
Team Members
Advisors
Collaborations
Awards
Hardware
Model
Medals
Human Practices
Human Practices
Silver
Gold
Integrated Practices
Engagement
Safety
Attributions
HOMOPLASMY STRATEGY
A video has been prepared below to help explain our homoplasmy strategy. A transcript of the video can also be found below
Transcript
The Cambridge-JIC 2016 IGEM team has developed a chloroplast transformation toolkit to allow future teams to use plastids as a new chassis to implement genetic circuits and exploit the potential of plastid engineering.
Chloroplasts hold a massive potential as an alternative protein expression system, due to their outstanding expression yields, diversity of post-translational modifications and auto/mixotrophic lifestyles of plants and microalgae. They are also the target of research, aiming to enhance crop and biofuel yields.Various proteins have already been successfully expressed in chloroplasts, including monoclonal antibodies , antigens, anti-toxins and growth factors.
Unfortunately, chloroplasts engineering has not been fully explored, due to a series of bottlenecks, which we tried to tackle with our project. One of them is that homoplasmy, the last step of chloroplast transformation  leading to the stable integration of the transgene in the genome, takes 2-3 months to achieve. We have developed a strategy which may, in principle, significantly decrease this time. This strategy was designed specifically to be used in model organisms with single chloroplasts, such as the microalgae Chlamydomonas reinhardtii, and may not be applicable to higher plants.
Homoplasmy is a term used to describe a cell whose plastid genome copies are identical, or in our case, have all been transformed. After initially transformation, only a few our of over 80 copies of the chloroplast genome take up the transfected cassette. It then usually takes months of segregation driven by antibiotic resistance selection pressure, to achieve homoplasmic strains.
Our team has designed a system in which this can be achieved much quicker with CRISPR/CAS9 technology. This video will explain our design. Additionally, you can find we have generated a parts library as well as hardware and modelling to make the necessary tools to implement this idea.
Our team has designed a system to accelerate the spread of any cassette of interest among the genome copies with the use of CRISPR/Cas9 technology. It depends on a co-transformation of the chloroplasts with two plasmids, the “driver” and the cassette-of-interest.  The device is divided into two separate plasmids to ensure the  the biological containment of the cas9 protein, as discussed later.
The first of the plasmids, the “driver”contains the cas9 protein along with an antibiotic-resistance, the guide RNA and the necessary flanking homology regions to mediate homologous recombination with the chloroplast genome.The cassette of interest contains the gene of interest, which can be virtually anything, a second antibiotic resistance gene and a different set of homology regions.  Our library of parts contain the necessary elements to construct these modular plasmids, providing flexibility for the user’s guide sequence, insertion site and gene of interest.
Due to the plasmids having two different antibiotic resistance genes, double transformants can be selected on plates containing both antibiotics.
Both the “driver” cassette and the cassette of interest would integrate into a chloroplast genome copy by homologous recombination as mediated by the two sets of homology regions.
Rather than waiting for the the transformed copy of the genome to be replicated, we want to spread the cassette of interest among the native genome copies. To achieve this, we want to use Cas9 to introduce double-strand breaks at the desired site of insertion. As non-homologous end-joining is absent in chloroplast, the only available mechanism of repair is homologous recombination. All the native genome copies would be cut repeatedly until the gRNA binding site has been disrupted by the insertion of the cassette of interest. On the other hand, homologous recombination with the native copies, although probabilistically more likely, would only lead to the recovery of the sequence gRNA is targeted to.
Unlike the cassette-of-interest, the “driver” cassette, together with Cas9 would not propagate, as it is flanked by a different set of homology regions, different from the ones next to the introduced double-strand breaks.  This means that our strategy has nothing in common with the “gene drive”.
Moreover, we want to further ensure biological containment of the genetic modifications by ensuring that the Cas9 is lost.
We suggest that, once homoplasmy is confirmed, the cells are replated to plates containing only one of the antibiotics, removing the selection pressure for the “driver cassette” to be maintained.
As Cas9 was found to be mildly toxic in cyanobacteria, the closest ancestors of chloroplasts, this may also be the case in chloroplasts, promoting its loss.
So, by the end of the procedure, all the copies of the chloroplast genome would contain the cassette of interest. We also would like to clarify that we have not had time to test our strategy in vivo. We, however, put effort in its careful design and have provided the necessary parts for its implementation by future iGEM teams.
Thank you for watching this video and please explore our wiki for a complete overview of our project!  In particular we would like to thank Nicola Patron (Earlham Institute),  Tom Knight (Ginkgo Bioworks) and Sarah Richardson (Ignition genomics)
ABOUT US
We are a team of Cambridge undergraduates, competing for the plant synthetic biology prize in iGEM 2016.
read more
FOLLOW US ON
LOCATION
Department of Plant Sciences,
University of Cambridge
Downing Street
CB2 3EA
CONTACT US
Email: cambridgeigem@hotmail.com
Tel: +447827442925
