<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cambridge-JIC_Description2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cambridge-JIC/Description2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cambridge-JIC</TITLE></P><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar3"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar3" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Motivation">Motivation</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Model">Modelling</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Biolistics">Biolistics</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/GrowthFacility">Growth Facility</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Homoplasmy">Homoplasmy Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Experiments">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Awards </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Medals">Medals</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Attributions">Attributions</A></LI></DIV><SECTION style="background-color:#fff; text-align: center"><DIV class="container" style="color:#000"><CENTER><H1 style="font-family:Montserrat ; line-height:1.295em">OVERVIEW</H1></CENTER><CENTER><P> Our team were planning multiple ways of saving the world with plants in two months we have available. In the end however we realised that plant engineering and especially chloroplast engineering poses many difficulties making its application almost impossible for iGEM teams them and putting off many scientists. The major obstacle is the length of time it takes (2-3 months) to achieve homoplasmy (when all the copies of chloroplast DNA are transformed by our cassettes and so they are stable and expressed at high levels). However there are also further challenges - for example chloroplasts can be reliably transformed almost only by biolistics and commercial biolistic devices are very expensive.</P></CENTER><CENTER><P>On the other hand chloroplast engineering has a very bright future and it is definitely worth using. Chloroplasts are cell factories for a lot of metabolic products, they can produce oils for biofuels, products can reach many-fold higher quantities and photosynthetic machinery can be modified and improved there. Plants and algae also offer huge possibilities for farming worldwide and out of the lab. Because if that they offer interesting perspectives for delivering edible vaccines and virus-like particles which can have huge impacts in developing countries where classical vaccines are unaffordable and hard to store and apply safely.</P></CENTER><CENTER><P>Because of that we haven't abandoned our idea about plant chloroplast transformation and instead decided to provide quick, cheap and accessible toolbox tackling some of the challenges researches and iGEM teams need to cope with when working with plants in synthetic biology. Our organism of choice is and alga Chlamydomonas reinhardtii which provides relatively quickly growing system for prototyping and itself is a very attractive target for biofuels, edible vaccines development and much more (read more about plastids and Chlamydomonas here (need to provide a link to there)). And we have many challenges to solve!</P></CENTER><CENTER><H2>Chlamydomonas</H2></CENTER><CENTER><H4>Model for higher plants</H4></CENTER><CENTER><UL><LI>Chlamydomonas reinhardtii is a unicellular alga which is a relatively well established model organism and many techniques can be readily transferred to higher plants.
            </LI><LI>It is registered as safe to humans so there is a lot of interest in developing edible vaccines
            </LI><LI>Fast growing
            </LI><LI>Very efficient homologous recombination in chloroplast
          </LI></UL></CENTER><CENTER><H4>Sustainable energy</H4></CENTER><CENTER><UL><LI>Algae are progressively moving in the centre of attention for development of biofuels because of their ease of farming (even vertically so that less ground is taken) and potential to produce oils and bioethanol. The production of biofuels in current use is very ineffective and its positive environmental effects are questionable. Algae could be our way to real sustainability. 
          </LI></UL></CENTER><H2>Challenges</H2><H4>Chloroplast transformation</H4><UL><LI>A widely used molecular tool for DNA editing CRISPER/Cas9 has not been successfully used in plant of algal chloroplasts so far. It has also been found toxic when expressed in Chlamydomonas nuclear genome or in cyanobacteria. Despite that we are aiming to overcome the obstacles and make this powerful tool available for chloroplast genome editing
          </LI></UL><H2>Applications</H2><H4>Chloroplast toolkit</H4><P>The very nature of our chloroplast toolkit is that it will be applicable to the whole field of synthetic biology. Chlamydomonas is only the stepping stone to the opening of this exciting field to higher plant species. This will allow fast development of genetically engineered genomes in plants to tackle global issues, for example the production of vaccines in plants and the faster growth of biofuels.</P><P>In terms of impact, our operational platform for plastid modification will lead to a revolution in the approaches taken in plant/algal based genetic engineering, thus facilitating the integration of plants sciences into the world of synthetic biology.</P><P>In additional to advancing the field, low cost hardware from this project will support our outreach goals, to make the field more accessible to smaller labs, hobby scientists and schools.</P></DIV></SECTION><SECTION style="background-color: #ddd"><DIV class="container"><H1 style="font-family:Montserrat ; text-align: center">BIOLOGY</H1><H2 style="font-family:Montserrat">Homoplasmy</H2><P>Our major goal is to achieve homoplasmy in Chlamydomonas chloroplasts much quicker than 2-3 months of repeated antibiotic selection. We are aiming to incorporate the desired genes of interest into all the copies of the chloroplast DNA in just one generation. Our current plan is to use CRISPR/Cas9 to cut the desired site of insertion so that homologous recombination (repair of the site with our cassette) would be facilitated. Non-homologous end joining does not exist in the chloroplast which means that even though at the beginning the probability of repair with a native chloroplast copy is high, the DNA strands will be cut again until they are repaired with our cassette. The cassettes we are using are shown in figure below:
        
        We are also thinking about safety (against the gene drive) in our design and so the “driver” will be expressed from just one copy of chloroplast DNA (not homoplamic and so lost when the selection pressure is relieved) as shown above or transiently. The gRNA in the cassette is targeting Cas9 to cut the site corresponding to homologies 1 and 2 (TrnI-TrnA site in particular which has been tested to give high product yields). 
        
        We are aiming to make this system very modular, assembling it with Golden Braid allowing further changes and it could be easily adapted to any site of insertion by changing just the gRNA (plus if needed homologies 1,2) and it should be easy to deliver any genes of interest. 
        </P><H2 style="font-family:Montserrat">CRISPR/Cas9 expression/delivery in chloroplasts</H2><P>We intend to use CRISPR/Cas9 system for the first time in Chlamydomonas chloroplasts. CRISPR/Cas9 is a powerful tool enabling targeted gene editing and may facilitate the insertion of constructs into the genomes. It is currently a very popular tool in synthetic biology. However Cas9 has been found toxic in Chlamydomonas cytoplasm and also in cyanobacteria so we plan to tackle this challenge and try different ways of expressing and delivering Cas9 into the chloroplast. We use transient expression of CRISPR/Cas9 delivered biolistically as well as stably integrated construct (however without aiming for homoplasmy so that it can be removed). If the planned approach does not work, we want to try delivering preassembled complex of gRNA and Cas9 into the chloroplast or regulating Cas9 expression by Nac2 system (toxicity may be likely because of too high Cas9 levels).
        </P><H2 style="font-family:Montserrat">Quick Chlamydomonas culturing and transformation</H2><P>We want to optimise the protocols for growing Chalamydomonas quickly and recovering it straight into liquid medium after the transformation by biolistics.
        </P><H2 style="font-family:Montserrat">Double transformation and double antibiotics selection</H2><P>We are aiming for double transformation and double selection on two antibiotics. We want to try this for the first time and optimise the process, assess if the two chloroplast antibiotic resistances or antibiotics themselves do not interfere and if one selection pressure is not significantly stronger to affect the results.
        </P><H2 style="font-family:Montserrat">A library of chloroplast parts in PhytoBrick standard</H2><P>Importantly as we are opening a new track and specifically pioneering chloroplast transformation we plan to deliver a lot of codon-optimised parts into the registry in the PhytoBrick standard. They should include many fluorescent proteins which have been so far only poorly expressed in chloroplasts. They would also include primers, UTRs, codon-optimised Cas9 and Cas9 fused to a fluorescent protein to be trackable, proteins with tags for Western blotting, gRNA used in our construct or antibiotics resistances to name a few. These should establish a firm base for future teams working on chloroplasts. 
        </P><H2 style="font-family:Montserrat">Transient expression</H2><P>Assess how long a transiently expressed plasmid remains in chloroplasts.
        </P><H2 style="font-family:Montserrat">Delivering a virus-like particle with the use as edible vaccine (using our homoplasmy tool)</H2><P>Provided our homoplasmy took works and there is time available, we would also like to add a gene of with potential therapeutic/industrial interest into our cassette as a proof of concept. An example of it may be a vaccine antigen. Notably the cell wall of Chlamydomonas protects its proteins from digestion in the stomach, and this opens a prospects of edible vaccines.
        </P></DIV></SECTION><SECTION style="background-color: rgba(255,255,255, 1); margin-bottom:-10px"><DIV class="container"><H1 style="font-family:Montserrat ; text-align:center">HARDWARE</H1><H2 style="font-family:Montserrat">Growth facility</H2><P>Recently there has been a strong effort within the synthetic biology community to create open-source hardware, which is easy to replicate and available at a fraction of the cost of commercially available equivalents. Despite this effort, when it comes to wanting to build one’s own lab equipment, there is little information online which is easily found and accessible to those without much electronic or mechanical experience.
          
          One of the aims of the hardware side of our iGEM project is to combat this problem by designing a low-cost growth facility for 85mm petri dishes, and documenting this comprehensively online, allowing SynBio open-source hardware to become accessible to “novice makers”. In order for the project to be worthwhile it is essential that anyone will be able to find to find this documentation, therefore as well as publishing the documentation on our wiki, tutorials will also be uploaded to Instructables, and also possibly to Youtube. In order to alleviate the frustration of searching for suitable parts, safely supplying them with power, etc., all documentation will include comprehensive part names and wiring diagrams. 
          
          Current specifications for the growth facility are that it should have temperature control loops to set a constant temperature appropriate to the sample being grown, lighting control loops which can achieve appropriate luminescence in programmable circadian rhythms, data-logging to monitor sample-growth, and imaging capabilities, which can store pictures of the sample online in real time. 
          
          It is hoped that the accompanying documentation will allow designers to easily extend the ideas presented in this project.
          </P><H2 style="font-family:Montserrat">Biolistic Device</H2><P>To add to our open-source hardware “toolkit”, we also aim to design, build and document a full protocol for a low-cost gene gun. Biolistics is a particle bombardment technique widely used  in a range of cell/tissue type transformations, capable of rapid delivery of multiple plasmids for transient or stable expression and without the use of carrier DNA. The high-velocity microparticles can penetrate even tough plant cell walls, enabling transformation of plastids such as the chloroplast, which we will use in our project.
          
          As useful as this technique is, the cost of commercially-available biolistics systems can run into the tens of thousands of pounds, making them unattainable for smaller laboratories and the hobbyist Syn Bio community. Our design will feature similar functions to these systems, allowing optimisation of firing pressure, duration and distance from the target, but for just 1% of their cost. The gene gun will be designed to use lower-cost consumable options, such as easily-replaceable CO2 cartridges, and will also incorporate safety features for the user (something which is generally neglected in existing DIY designs available online).
          
          Coupled with a detailed protocol suitable for novice makers, our DIY gene gun will open up the world of plant transformation and synthetic biology for smaller laboratories and Bio-makespaces.
          </P><H2 style="font-family:Montserrat">Modelling</H2><P>Modelling the system mathematically, with the aim of predicting the timescales in which homoplasmy is likely to occur, will serve several purposes:</P><UL><LI>A comparison of theoretical order of magnitude estimates for gene integration timescales with Cas9 and without (i.e. via homologous recombination) will hopefully validate the idea that our system is faster than current methods
              </LI><LI>An accurate estimate of the timescales in which homoplasmy could reasonably have been expected to occur in a significant portion of cells will be useful for efficiently deciding when to begin screening for homoplasmy when carrying out the transformation protocol behind the project
              </LI><LI>Observing how the above timescales change as various model parameters change will allow users to pinpoint the optimal conditions, and possible troubleshooting points, for transformation
            </LI></UL><P>Our model will be implemented using open source, thoroughly commented MATLAB code, which will output an estimated timescale for homoplasmy. Modelling will be done using law of mass action and free energy calculations. Also worth noting is that the CRISPR/Cas9 part of the model is not specific for Chlamydomonas, and thus, portions of the code can be adapted to have relevance to other researchers and teams using CRISPR/Cas9.
          </P></DIV></SECTION><DIV class="container" style="margin-top:0px"><DIV class="row  pad-bottom"><DIV class="col-md-3"><H4><STRONG>ABOUT US</STRONG></H4><P style="text-align:center">
                    We are a team of Cambridge undergraduates, competing for the plant synthetic biology prize in iGEM 2016.
                    <A href="//2016.igem.org/Team:Cambridge-JIC/Team">read more</A></P></DIV><DIV class="col-md-3"><H4><STRONG>FOLLOW US ON</STRONG></H4></DIV><DIV class="col-md-3"><H4><STRONG>LOCATION</STRONG></H4><P style="text-align:center">
                    Department of Plant Sciences, 
                    University of Cambridge  
                    Downing Street 
                    CB2 3EA
                    </P></DIV><DIV class="col-md-3"><H4><STRONG>CONTACT US</STRONG></H4><P style="text-align:center">
                    Email: <A href="mailto:cambridgeigem@hotmail..com" target="_blank">cambridgeigem@hotmail.com</A>
                    Tel: <A href="tel:+447528848745">+447827442925</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>