<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cambridge-JIC_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cambridge-JIC/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cambridge-JIC</TITLE></P><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar3"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar3" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Motivation">Motivation</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Model">Modelling</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Biolistics">Biolistics</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/GrowthFacility">Growth Facility</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Homoplasmy">Homoplasmy Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Experiments">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Awards </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Medals">Medals</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Attributions">Attributions</A></LI></DIV><SECTION style="background-color:#3d3d3d; text-align: center; padding: 15% 0%"><DIV class="container" style="color:#fff"><H1 class="darkBlue" style="font-family:Montserrat; text-align:center; font-size:180%">NOTEBOOK</H1></DIV></SECTION><SECTION style="background-color: #fff"><DIV class="container"><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">27th June</H1><H3>Entire team assembled for the first time in the Department of Plant Sciences. Had some fun in building sticky note tower (the tallest one was 2.5 meters, built in one hour without previous preparation!) Various coffees and teas are available in the department, very planty.
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> In the first week we had lectures and discussion on synthetic biology, photosynthesis, DNA assembly and genetic engineering on plants. We also practiced using Geneious software and visited the botanic garden. Lots of brainstorming creates tonnes of ideas about ways to save the world using plants in 10 weeks.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">1st July</H1><H3>By the end of the week we have a long list of ideas but nothing too concrete so that we could assess feasibility. We have made a big progress in getting the team together, communicating biology to engineers, finding the common language. By monday we have a challenge - to everybody come up with a well elaborated idea about what to do. 
               </H3></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">4th July</H1><H3>Bio-plasters
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> We got all really excited for bioplasters. Lucie has found two cool articles where they have introduced Chlamydomonas into biomaterials to produce oxygen. It worked nicely and caused no immune reaction but they haven't modifies Chlamy to do something cool. We could do that! The group has elaborated on it further but then we realised it would be hard to find supervisors for that and there might be issues with working with mammalian material. We had a presentation from everybody about our ideas - Geoff about light-producing algae, Mie about environment cleaning and biohouses, Jake about growth facility, etc.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">5th July</H1><H3>Finding our biological direction: Homoplasmic transformation
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> After a meeting with advisors we came to reconsider our idea of establishing a plant toolbox opening the Plant Track and especially chloroplasts to iGEM and science. And we found a very exciting goal - to make a tool which would propagate our constructs into all the plastid DNA copies quickly and efficiently using CRISPR/Cas9. No more tedious selection rounds for 2-3 months!
                </H3><H3> In the following days we discussed with the experts in the Plant Sciences Department and dived into the literature on chloroplast transformation.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">6th July</H1><H3>Promotion and Marketing
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3>Alex has designed our logo and first flyers! So excited! We also have all the Twitter and Instagram accounts and we are working on getting the Facebook page handed over. Or course we are not forgetting about our human practices planning, sponsorship and propagation.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">8th July</H1><H3>Visiting Cambridge Consultants
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> All members were invited to Cambridge Consultants. We discussed about the project, visited their laboratory and had a delightful lunch in the cafe. They are really keen to give us advice! In the afternoon we have established some strategic roles - who will manage the team, work on graphical design, wiki programming, content, etc.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">10th July</H1><H3>Scientific 'pool'
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Still swimming in the scientific literature and trying to get as much help from our busy advisors as possible. Last week we aimed for sending all the sequences for synthesis today. And do everything at once. But over the time we realise that eating an elephant requires cutting it into smaller pieces. We need to test everything to know what went wrong, we plan controls for everything and design experiments which will teach us even if the result is negative.
                </H3><H3>But we are still thinking far ahead and considering actually introducing some gene of interest by our finished system, if it works. We spend a lot of time researching into edible vaccines and virus-like particles which could have an impact in the field and it's really exciting!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">11th July - 13th July</H1><H3>Co-lab workshop
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Lovely French students were were coming down to Cambridge to give us a workshop about connecting synthetic biology with design. An opportunity not to be missed! We joined some keen designers as well as people with other backgrounds and together we were working in the lab, had an etnographic workshop, had interesting lectures and were looking for a common language. We got a lot of interesting inputs for our human practices and got to design a few human practices projects happening later. For example we want to make discussion workshops in between students interested in synthetic biology, philosophy and journalism to bring them together to share thoughts. Hopefully we'll also go to schools to educate about synthetic biology.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">15th July</H1><H3>Wet-lab setup
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> We started preparing our lab to actually welcome some experiments. Unpacking leftovers from last years, making lists of chemicals and equipment, laying out pipettes, etc.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">18th July</H1><H3>Protocols
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> We intensely worked on some easy protocols - tried our miniprep for amplifying our desired DNA in bacteria, culturing am bulking up our Chlamydomonas as well as preparing general and antibiotic plates for them. We have set up an experiment testing how well Chlamydomonas grows under different conditions - in the dark on TAP, in the light and in the light with CO2 concentration elevated to 5%. Thanks Dr Moritz Meyer for massive help!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">19th July</H1><H3><B>PLASMID DESIGN UPDATE</B></H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Working further on our plasmids made of DNA parts. We have so far considered many different designs. Initially we planned to have two plasmids - one with the Cas9 regulated by a regulatable promoter which would be only transiently expressed. The second plasmid with the gRNA under one promoter and a separate one promoter for our markers and the gene of interest. We considered luciferase of fluorescent proteins as our markers. We were struggling to find a sharp promoter and terminator functional in chloroplasts so that our gRNA has the desired length and sequence in the end. We even considered ribozymes which are capable of self-splicing of the gRNA and used in mammals.
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">21st July</H1><H3>Parts for synthesis?
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Today we thought our synthesis sequences were done, just needed final checks. Haha, had no idea how many major changes of the whole technique will come before we send them off. But with every change we learn more and more, nothing is a lost time!
                </H3><H3>We are also in constant contact with the lab of Prof Saul Purton at UCL who is giving us amazing advice from experience with working on this system and even some material his lab produced. He has been amazingly helpful especially when we need to decide on alternative parts - they know what usually works in Chlamydomonas chloroplasts! 
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">22th -23th July</H1><H3>iGEM + Biodesign NightScience &amp; iGAMER
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> During the Friday and Saturday we visited Paris for a cool conference about design of biological systems, DIY hardware, community labs and workshops, games teaching science, we have given a talk about our project. Were networking with many people from the whole Europe showing us also perspectives of designers. We made friends with Paris Bettencourt iGEM team and established a collaboration. 
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">25th - 27th July</H1><H3>Open Plant Forum conference in Norwich
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> We went to Norwich to participate a very inspiring conference on synthetic biology in plants. The two days were packed with lectures and panel discussions from scientists from our Plants Department, Norwich (JIC and other institutes as well) and from some commercial companies. The Open Plant initiative aims to promote sharing of knowledge, working materials - physical as well as DNA sequences which would pave the way towards bigger inventions in science and industry much the same way standardised basic parts do in engineering. 
                </H3><H3>During all the lunch and tea breaks as well as evenings we worked very hard on all our sequences, were chasing all the amazing scientists to discuss our project and get advice. It was amazingly productive and we thank a lot to all the people moving our project further, especially Dr Nicola Patron, Dr Tom Knight, Dr Steven Burgess and Ms Kher Xing Chan (Cindy)!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">28th July</H1><H3>Preparing lawns of cells for Biolistics tomorrow. Collecting protocols.
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> We hoped to send out our sequences but there are more and more problems appearing with their nature (low GC content, length, etc.) so we keep working on them. 
                </H3><H3>Got some results from our growth experiment. Our algae grow certainly the best in light and with elevated CO2 concentration in the air.
                </H3><H3>An article about us came out on Cambridge Consultants <A href="http://blog.cambridgeconsultants.com/medical-technology/engineering-a-future-for-synthetic-biology/">blog</A>!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">29th July</H1><H3><B>FINALLY SENDING OFF OUR SEQUENCES! (PLASMID DESIGN UPDATE VOL. 2)</B></H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Some had too high AT content and repeats to be synthesised (primers surprisingly :D ) but we will try to amplify them from the chloroplast genome. Our plasmid design had at least 10 major changes of direction and was evolving quicker than some of us were able to track! But planning is important, right? 
                </H3><H3>Some of us have spent the afternoon learning biolistics - transforming our cells with fluorescent proteins
                </H3><H2><B>PLASMID DESIGN UPDATE VOL.2:</B></H2><H3>MARKERS</H3><H3><UL><LI>First we have abandoned the idea of marking our genes with luciferase as that would not give us quick and easily detectable response. Fluorescent proteins should be more reliable although there are troubles with expressing them in Chlamydomonas chloroplasts - well, something for us to troubleshoot with trying our 18 possible FPs!
                  </LI></UL></H3><H3>SAFETY - CAS9 REGULATION AND WAYS OF EXPRESSION</H3><H3><UL><LI>We are very concerned with safety as our design may resemble a gene drive, we want to make sure we are definitely not making one. We were considering Lac regulatable promoters, light regulated ones and also Nac2. Nac2 is still a system we consider to try - we could switch off the expression of Cas9 when needed and regulate it in case it's toxic in higher amounts by concentration of thiamin. But it is actually in the UTR and regulated by a nuclear gene which regulates a photosynthetic gene. So we need to obtain a strain which has been designed to have these pathways independent, we'd need to optimise concentrations of thyamin and design more constructs.!
                  </LI><LI>We also considered another level of safety by introducing another gRNA which would make the Cas9 to cleave itself after homoplasmy has been achieved. However as NHEJ does not exist in chloroplasts, we would also need to supply the template. Still an option! But at the moment we plan to introduce Cas9 cassette either transiently or just into one copy of DNA, not homoplasmic (we plan to try both), let it drive the propagation of our genes of interest and then be lost when we remove the selection pressure of antibiotics. A semi-stable type of integration was decided as an alternative to transient because transient may not produce enough product and be list too quickly.                    
                  </LI><LI>We are also considering delivery of the preassembled complex of gRNA and Cas9 or its parts into chloroplast by biolistics, although it might be beyond our time-scope
                  </LI><LI>Because of safety our Cas9 and gRNA were initially separated either physically or one of them was outside the homology regions so that it won't be integrated. It was because we had only one insertion site and corresponding homologies. In the current design the Cas9 and gRNA are in one construct but gRNA is targeting another region on the chloroplast genome (so that it drives expression).
                  </LI><LI>Our polycistronic design of cassette might be simpler and produce shorter DNA fragments but maybe complicated for cell processing so we decided to put separate promoters for gRNA, antibiotic resistance and Cas9 fused with FP
                  </LI></UL></H3><H3>ACTIVITY OF CAS9</H3><H3><UL><LI>We had troubles with assessing that our Cas9 is active as one of the first steps. It would be very elegant to let it drive propagation of its own cassette with markers (or changing colours of algae already transformed with a fluorescent protein), making the strain less viable in field but the same in the lab (eg. by nutrient utilisation defficiency). The best solution would be restoring functionality of some mutant strain by the cassette. But we want to avoid a gene drive at all costs. We have also considered surveyor assay. So we are still looking for ideal solutions, probably by tracking Cas9 cuts by qPCR. (or do we have a solution?)
                  </LI><LI>Our Cas9 is now fused with the fluorescent protein. It means each molecule of the FP corresponds to a molecule of Cas9. And we can supply an interesting part into the registry
                  </LI></UL></H3><H3>TYPES OF CAS9</H3><H3><UL><LI>We were also considering different versions of Cas9 - from different bacteria, expressed from nucleus targeted to chloroplast (unluckily it is probably toxic in the cytoplasm), transiently or stably expressed, and for long we were considering using nickase (which makes just single-stranded breaks and so pair of gRNAs is used to reduce off target activity by around hundreds of folds). But then we learned the toxicity is more likely because of sucking RNAs rather than cutting off-target.
                  </LI></UL></H3><H3>WORKING SYSTEM</H3><H3><UL><LI>We considered prototyping experiments on isolated chloroplasts to speed them up but at the moment we plan to transform chloroplasts in the whole Chlamydomonas
                  </LI></UL></H3><H3>INSERTION SITES</H3><H3><UL><LI>This design also enables any researcher with a prepared cassette for insertion to work with our system just by changing the gRNA (20 nucleotides) to correspond to the desired site of insetrion. We have searched different sites of insertion (homologies) and in our default modular design we have settled for far on a using the sites tried out by Prof Purton's lab (trnE2-psbH) for the driver (see picture below) and also a site which in literature is reported as producing high yields (TrnI-TrnA) for our gene of interest. Right?
                  </LI></UL></H3><H3>MODULARITY AND COMPATIBILITY</H3><H3><UL><LI>We design the system highly modular and compatible with Golden Gate so that it can be easily used and changed, it should work with any standard strain of Chlamydomonas. We are looking into possibility of maintaining our system modular by using GoldenBraid technique.
                </LI></UL></H3><H3>Well done if you got up to here! Have a look at our current design:</H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">2nd August</H1><H3>The day of meetings
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> In the morning we have met Cambridge Consultants to report our progress and discuss the cell selection and flow cytometry use as well as some hardware tips. 
                </H3><H3>In the afternoon we had meeting with advisors discussing our long and short term goals.
                </H3><H3>Later we followed with planning of immediate experiments:</H3><H3><UL><LI>Testing many different fluorescent protein markers in Chlamydomonas chloroplasts</LI><LI>Testing toxicity of Cas9 in the chloroplast (it has been found toxic (probably RNA sponge) in the cytoplasm and cyanobacteria)</LI><LI>Testing double selection on two antibiotics and the interaction of antibiotics and their resistances + cotransfection with two constructs in general</LI><LI>Plus we plan to put our parts in iGEM backbones soon after they come</LI></UL></H3><H3>Group meetings are also very important!</H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">3th August</H1><H3>Designing and Ordering of Primers
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Ordering our primers for PCR, so that we can amplify our synthesised DNA. We will also be adding PhytoBrick standards on our parts using site-directed mutagenesis. Finally working on our wiki content. 
                </H3><H3>Hardware components for growth facility came!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">4th August</H1><H3>Experimental preperation
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Planning the immediate experiments, reagents we need to order 
                </H3><H3>Some more components coming, for biolistics as well!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">10th August</H1><H3>Another weekly Biomakespace organisation meeting
               </H3></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">11th August</H1><H3>Proper labwork, amplifying our mVenus fluorescent protein and the backbone to put it in
Constructing primers for amplification of our DNA parts and especially Cas9 for future Gibson Assembly
               </H3></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">16-17 August</H1></DIV><DIV class="col-md-12"><DIV class="description"><H3> Ligation of our mVenus construct
                </H3><H3>Amplification of the other parts.
                </H3><H3>Finally trying some biolistics! 
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">18th August</H1><H3>UK meetup of iGEM teams! 
               </H3></DIV><DIV class="col-md-12"><DIV class="description"><H3> Stressed with the presentation, got to know some nice new people, realised we are probably the only ones working on plants and that we have been put into overgraduate category despite all being undergrads. Fun times! Thanks The University of Westminster for organising!
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">19th August</H1></DIV><DIV class="col-md-12"><DIV class="description"><H3> Further amplification of our parts, planning of new design of our transformation vectors. 
                </H3><H3>Biolistics again, this time successfully shot the mVenus ligated into a vector and the original vector with a different fluorescent protein as a control. Letâ€™s wait to see if some genes got into our algae and produced any fluorescence! 
                </H3></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-9"><H1 style="font-family:Montserrat ; font-size:24px" id="test">22nd August</H1></DIV><DIV class="col-md-12"><DIV class="description"><H3> All PCRs finished or repeated to put our parts into level 0 vectors! 
                </H3></DIV></DIV></DIV></DIV></SECTION><SECTION id="footer-sec"><DIV class="container" style="margin-top:0px"><DIV class="row  pad-bottom"><DIV class="col-md-3"><H4><STRONG>ABOUT US</STRONG></H4><P style="text-align:center">
                    We are a team of Cambridge undergraduates, competing for the plant synthetic biology prize in iGEM 2016.
                    <A href="//2016.igem.org/Team:Cambridge-JIC/Team">read more</A></P></DIV><DIV class="col-md-3"><H4><STRONG>FOLLOW US ON</STRONG></H4></DIV><DIV class="col-md-3"><H4><STRONG>LOCATION</STRONG></H4><P style="text-align:center">
                    Department of Plant Sciences, 
                    University of Cambridge  
                    Downing Street 
                    CB2 3EA
                    </P></DIV><DIV class="col-md-3"><H4><STRONG>CONTACT US</STRONG></H4><P style="text-align:center">
                    Email: <A href="mailto:cambridgeigem@hotmail..com" target="_blank">cambridgeigem@hotmail.com</A>
                    Tel: <A href="tel:+447528848745">+447827442925</A></P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>