Team:Cambridge-JIC/Results
Cambridge-JIC
Toggle navigation
Project
Description
Design
Motivation
Modelling
Biolistics
Growth Facility
Homoplasmy Strategy
Proof Of Concept
Demonstration
Parts
Experiment
Results
Notebook
Team
Team Members
Advisors
Collaborations
Awards
Hardware
Model
Medals
Human Practices
Human Practices
Silver
Gold
Integrated Practices
Engagement
Safety
Attributions
RESULTS
Gold Medal Award
Best Plant Synthetic Biology (Overgrad)
It has been a summer of a lot of work, many mistakes and many corrections. Below you will find a bullet points of the broad overview of the successes and less successful aspects of our wetlab project. We then elaborate in text our experience and where we put our effort in. We have included “failures” and “mistakes” as we consider these to be useful to future iGEM teams who brave the challenge of working with plastids!
What we achieved this summer
Generated a library of parts for Chlamydomonas reinhardtii chloroplasts
Verified these parts through sequencing and/or restriction digest
Verified our aadA antibiotic cassette in E.coli
Biolistically transformed Chlamydomonas with gene gun
Library-based design of homoplasmy strategy for chloroplasts
Outside the wetlab our work this summer has lead to
DIY biolistics gene gun
DIY Growth facility
Modelling software for experimental design
Aims to be completed
We have everything required to test our homoplasmy strategy: a comprehensive, well-documented design (after numerous discussions with professionals in the field), and a library of tested and highly modular Phytobrick parts. However, we did not test our homoplasmy strategy in vivo
We have included many fluorescent proteins in our Phytobrick library, codon-optimised for Chlamydomonas chloroplasts. However, we were not able to establish a positive strain expressing a fluorescent protein
We improved upon a previous iGEM part submission for cas9 , by codon-optimising for Chlamydomonas chloroplasts and modifying to fit the Phytobrick standard, and submitted our part to the registry . However, we have yet to test its expression and toxicity in our chassis (despite planning the methods theoretically)
Future plans for the project
With available parts we provide, establish positive strains for more complex experiments
Express cas9 linked to VFP-HA to study toxicity
Apply and debug the homoplasmy acceleration strategy
Optimization of protocols for transformation with our DIY gene gun
Excited about the potential harnessed in plastid engineering to solve global challenges, we want to make this chassis accessible to future iGEM teams so that great ideas can be developed. Our wetlab results revolve around the achievement of the first C. reinhardtii chloroplast library in phytobrick standard which, among others, allows to creation of a modular transformation vector for chloroplasts (not previously available in the registry) and the necessary elements to use CRISPR/Cas9 technology.
This section only considers the biological wetlab work, please read about our library-based experimental design which we also consider a result of the summer’s work with the aim of implementing a crispr-cas9 mechanism to achieve homoplasmy in a much quicker way, as necessary for this technology there are important biological containment considerations.
Parts were mainly synthesized through iDT, however there were significant challenges.
Firstly, chloroplast genomes have a high A/T content, therefore many could not be synthesized in this manner. The two solutions were to modify bases in specific regions with high A/T content or add a string of nucleotides at the 5’ and 3’ end which would be later cleaved with a restriction enzyme. We decided on modifying some regions on the sequence (always making sure to maintain the encoded information) as well as yielded some parts from plasmids received from labs working on Chlamydomonas. In the case of CAS9, which was additionally too large, we subdivided the sequence in 4 parts and through PCR added the necessary overhangs for a golden gate assembly without affecting the sequence.
The design of parts included codon-optimization through the software programme developed at Saul Purton lab for chloroplast genes as well as the removal of illegal restriction sites whilst maintaining the integrity of the coding sequence.
Due to a misinterpretation of the novel phytobrick documentation we designed the incorrect overhangs and need to modify each part. We then proceeded to amplify each part with the appropriate overhangs for the phytobrick syntax. New challenges arose with this due to the A/T-rich genome and the high frequency of same-nucleotide tracks. This lead to many sessions of PCR troubleshooting, modulating and tweaking the variables and the PCR protocols to finally achieve all the parts in functional standard.
Yet, for the verification of parts we required a functional plasmid to introduce the gene of interest (GOI), as transgenes are inserted into the chloroplast genome through homologous recombination and there was no available backbone with this purpose in the iGEM registry. We set ourselves the task of developing this plasmid for future teams and to make it modular by providing the homology regions as parts which can be assembled. Following recommendation of Marco Larrea Alvarez from Saul Purton algal biotechnology lab in London we created a promoter and terminator directly linked to homology regions. He has shown interest in these parts for his own research, further, he gave a plasmid to restore photosynthesis in a different strain (TN72) and thus already possessed homology regions into which we could clone a aadA cassette as selection pressure for our GOI. This parallel cloning strategy was the safest way to proceed to achieve verifications.
We verified our parts through sequencing data, indicated on the parts description (for example BBa_K2148001 ) as well as through restriction digest (for example BBa_K2148005 ).
Restriction digest was important to verify the correct usage of the phytobrick syntax for cloning into the level 0 backbone. It was particularly useful in the case of cloning the party of homology region with an adjacent promoter (H3R1) which was cloned from plasmids provided to us by Saul Purton. However, due to the size of the fragment  we need to extract it in three parts to then assemble by golden gate cloning. When incorporated the part into the level 0 backbone and digested it newly and run on the gel, we saw that the first and second fragments were of the appropriate size but not the third, which was much smaller and we needed to repeat the PCR amplification.
Further, our aadA gene was verified by bacterial transformation. TOP10 competent cells were able to grow on both ampicillin and spectinomycin plates(details shown in BBa_K2148001 part page).
This is truly symbolic as it represents how chloroplast parts function in bacteria
Experimenting with synthetic biology in chloroplasts is the natural segue into higher plants, due to their bacterial ancestry. Manipulation and insertion of genetic circuits into higher autotrophic plants has a lot of promising potential.
The generation of the plasmid with our parts also served the purpose of proof of concept of the functionality and compatibility of our parts library. Once the L1 were assembled, we ensured the presence of the elements by restriction digest. Referring to the design of our CRISPR-cas9 project, we acknowledge the need to insert two genes simultaneously for experiments and so have cloned two sets of different homology regions, although only one set of them has been provided to the registry. The second set of homology regions were not linked to either the promoter or the terminator and thus we decided to clone into a L1 backbone (no appropriate section in level 0 parts) but the ligation did not work in time.
The generation of the plasmid in this manner, through golden gate assembly, allows for modularity of the user but also fits in the standard requirements of the iGEM competition.
The plasmids yielded were transformed into Chlamydomonas reinhardtii biolistically with the appropriate positive and negative controls. To read more about this please refer to the demonstration page, as this aspect of the project served the purpose to demonstrate the potential of our modular parts to modify chloroplasts, not just of algae but of plants in general.
ABOUT US
We are a team of Cambridge undergraduates, competing for the plant synthetic biology prize in iGEM 2016.
read more
FOLLOW US ON
LOCATION
Department of Plant Sciences,
University of Cambridge
Downing Street
CB2 3EA
CONTACT US
Email: cambridgeigem@hotmail.com
Tel: +447827442925
