<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cambridge-JIC_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cambridge-JIC/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Cambridge-JIC</TITLE></P><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar3"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar3" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Motivation">Motivation</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Model">Modelling</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Biolistics">Biolistics</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/GrowthFacility">Growth Facility</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Homoplasmy">Homoplasmy Strategy</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Proof">Proof Of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Experiments">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Notebook">Notebook</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Awards </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Medals">Medals</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Cambridge-JIC/Attributions">Attributions</A></LI></DIV><SECTION style="background-color:#fff; text-align: center"><CENTER><H1 style="font-family:'Montserrat'; line-height:1.295em">MODELLING</H1></CENTER><DIV class="row scroller" style="color:#000"><DIV class="col-md-4"><A href="#formulation" class="darkBlue" style="font-family: 'Pacifico'"><H2 style="text-align: center">Model Formulation</H2></A></DIV><DIV class="col-md-4"><A href="#param" class="darkBlue" style="font-family: 'Pacifico'"><H2 style="text-align: center">Paramterising the Model</H2></A></DIV><DIV class="col-md-4"><A href="#implementation" class="darkBlue" style="font-family: 'Pacifico'"><H2 style="text-align: center">Practical Implementation</H2></A></DIV><DIV class="col-md-4"><A href="#results" class="darkBlue" style="font-family: 'Pacifico'"><H2 style="text-align: center">Results &amp; Discussions</H2></A></DIV><DIV class="col-md-4"><A href="#references" class="darkBlue" style="font-family: 'Pacifico'"><H2 style="text-align: center">References</H2></A></DIV><DIV class="col-md-4"><A href="https://static.igem.org/mediawiki/2016/f/f1/T--Cambridge-JIC--model--code.zip" class="darkBlue" style="font-family: 'Pacifico'"><H2 style="text-align: center">Download</H2></A></DIV></DIV><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>INTRODUCTION</EM></H3><DIV class="container"><P style="font-family:'Roboto Condensed'; font-size:150%">The aim of the modelling section was to create an integrated, kinetic model of our Cas9-guided chloroplast transformation mechanism. This model can be used to understand the internal workings of our proposed transformation method, and to determine whether it is viable and genuinely superior to existing methods. It also has practical use, in determining the expected timescales for homoplasmy (and thus when re-plating and selection could plausibly begin). The model is fully documented, and the code is available online and thoroughly commented. It could also be adapted not just to predict homoplasmy times in the chloroplasts of other organisms, but to yield useful information about the kinetics of Cas9-driven genetic transformation in any organism, encompassing the kinetics of:</P><UL style="font-family: Roboto Condensed; font-size: 150%; "><LI>Cas9-gRNA active complex formation
      </LI><LI>Cas9 cleavage on the chloroplast genome (for both on- and off-target sites)
      </LI><LI>integration of the gene of interest, via homologous recombination
    </LI></UL><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>PREVIOUS MODELS</EM></H3><P style="font-family:'Roboto Condensed'; font-size:150%">Kinetic modelling of CRISPR/Cas9 action has been attempted by several iGEM teams in the past. However, past iGEM models have largely been geared towards using dCas9, a modified Cas9 molecule with no cleavage activity, to form genetic circuits. To the best of our knowledge, no past iGEM project has included a freely available, integrated model of Cas9-induced cleavage or homologous recombination. The Salis lab at Penn State University, however, have recently published a complete, biophysical model of CRISPR/Cas9 cleavage activity [1], which was a major point of reference for our modelling.</P></DIV></SECTION><SECTION id="formulation" style="background-color:#B7E2F0; text-align: center"><DIV class="container" style="padding:0% 0%"><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>MODEL FORMULATION</EM></H3><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>1. Gene expression</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">For active Cas9 cleavage, a gRNA template must be transcribed, the Cas9 protein transcribed and translated, and a gRNA-Cas9 complex formed. In our proposed chloroplast transformation method, these are both expressed from the same “driver” cassette, introduced into the chloroplast by a transformation method such as biolistic transformation with a gene gun, but under individual promoters (the psaA promoter for the gRNA, and atpA promoter for Cas9). The Cas9 and gRNA then diffuse randomly through the chloroplast until they encounter one another, at which point they form a Cas9:gRNA complex. A further isomerisation reaction must then take place for this complex to become capable of cleavage.
This process is modelled in the following 5 equations:</P><FIGURE><CENTER><FIGCAPTION>Eqn. 1
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Where:</P><UL style="font-family: Roboto Condensed; font-size: 150%; text-align:left"><LI>N<SUB>gRNA</SUB> is the number of gRNA strands in a single chloroplast
      </LI><LI>N<SUB>Cas9 mRNA</SUB> is the number of mRNA strands coding for Cas9 in a single chloroplast
      </LI><LI>N<SUB>Cas9</SUB> is the number of Cas9 molecules in a single chloroplast
      </LI><LI>N<SUB>intermediate</SUB> is the number of inactive, pre-isomerisation, Cas9:gRNA complexes in a single chloroplast
      </LI><LI>N<SUB>Cas9:gRNA</SUB> is the number of active Cas9:gRNA complexes in a single chloroplast
      </LI><LI>N<SUB>driver</SUB> is the number of driver cassettes introduced into the chloroplast by a given transformation method
      </LI><LI>The various k values represent first or second order rate constants
      </LI><LI>The various δ values represent degradation rates
      </LI><LI>r<SUB>c[i] represents the cleavage rate of the ith target site on the chloroplast genome
    </SUB></LI></UL><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>2. Cleavage rates</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">Once the Cas9:gRNA active complex has been formed, it is free to diffuse through the chloroplast until it finds a site on the genome. Cas9 uses no external energy source for binding, relying solely on the bound state being an energetically favourable configuration. It will only bind to the site if it “recognises” the PAM – i.e., if the binding of Cas9 to the PAM is energetically favourable (notably, this does not require a perfect match).</P><P style="font-family:'Roboto Condensed'; font-size:150%">Once the site has been “recognised”, Cas9 moves down the genome, unwinding it nucleotide by nucleotide, displacing the complementary DNA strand, and allowing the gRNA to bind in its place (by Watson-Crick base pairing). This is known as R-loop formation. Each mismatch between the gRNA and genome carries an energy penalty. Enough mismatches will make the interaction energetically unfavourable, causing Cas9 to dissociate from the genome and diffuse freely, until it finds another site. If the interaction is still energetically favourable once the R-loop has been formed along the full length of the gRNA, Cas9 will cause a double stranded break in the DNA, 3-4 base pairs from the PAM [1], and dissociate, losing its efficacy (as a single-turnover enzyme) [2].</P><P style="font-family:'Roboto Condensed'; font-size:150%">The kinetics used to model this process are very heavily influenced by the work of the Salis Lab, although similar work was done by the 2013 iGEM team from Wuhan University in China [3]. From its site of production, the Cas9:gRNA complex can be taken to undergo molecular diffusion in the form of an isotropic, 3-dimensional random walk. This leads to the following equation to describe the rate of contact of the Cas9:gRNA complex with all possible sites on the genome:</P><FIGURE><CENTER><FIGCAPTION>Eqn 2
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Where:</P><UL style="font-family: Roboto Condensed; font-size: 150%; text-align:left"><LI>r<SUB>RW</SUB> is the rate of DNA site contact
      </LI><LI>D is Cas9’s diffusivity
      </LI><LI>λ is the characteristic length between the site of Cas9:gRNA complex production and binding site
      </LI><LI>V is the chloroplast volume
    </LI></UL><P style="font-family:'Roboto Condensed'; font-size:150%">As mentioned previously, Cas9 binding is governed by the sum of the free energy changes involved. Three energy exchanges are involved in this process. The first energy exchange results from binding of the PAM, ΔG<SUB>PAM</SUB>.</P><P style="font-family:'Roboto Condensed'; font-size:150%">A second energy exchange, ΔΔG<SUB>exchange</SUB>, comes from R-loop formation. Using a nucleic acid nearest neighbour model, this can be considered a weighted sum of the energy exchanges involved in binding each gRNA duplex to its corresponding DNA duplex. 256 of these energy exchange parameters exist (for all existing RNA duplex combinations, binding to all possible DNA duplex combinations) and are expected to be near 0 for a match (e.g. rCG/dGC) and to carry a positive energy penalty for a mismatch (e.g. rCG/dAC). Mismatches occurring nearer to the PAM have been shown to carry more of an energy penalty than those further away. Thus, the equation for ΔΔG<SUB>exchange</SUB> for a 20 nucleotide gRNA is given by</P><FIGURE><CENTER><FIGCAPTION>Eqn 3
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Where d<SUB>k</SUB> is the weight of a mismatch k nucleotides from the PAM, and ΔΔG<SUB>gRNA:Cas9[k,k+1]</SUB> is the free energy change from binding of the duplex formed by nucleotides at the kth and k+1th positions.</P><P style="font-family:'Roboto Condensed'; font-size:150%">A final energy consideration proposed by the Salis lab is that which results from the supercoiling which is induced by Cas9’s binding to other, nearby sites. When Cas9 binds to a site, the uncoiling (negative supercoiling) of the genome which is necessary to form an R-loop is likely to further coil (positive supercoiling) all adjacent sites within a certain distance, as DNA’s linking number is conserved. This carries a positive energy penalty, making the nearby sites harder to uncoil. Discrepancies in the degree of supercoiling across the genome will result in a similar effect. However, due to the fact that most Cas9 binding will occur at a single site (the on-target site) in the regime we are considering, we were able to neglect the first effect. We also neglected the second, as there is only evidence of small variations in supercoiling across the C. reinhardtii genome [4].</P><P style="font-family:'Roboto Condensed'; font-size:150%">The probability of the Cas9:gRNA complex binding to a potential DNA site is governed by a Boltzmann function, taking into account all of the potential Gibbs free energy</P><FIGURE><CENTER><FIGCAPTION>Eqn 4
        </FIGCAPTION></CENTER></FIGURE><UL style="font-family: Roboto Condensed; font-size: 150%; text-align:left">Where:
      <LI>N<SUB>target[i]</SUB> is the ith potential DNA binding site
      </LI><LI>ΔG<SUB>target[i]</SUB> is the free energy change associated with Cas9 binding to the ith potential site
      </LI><LI>k<SUB>B</SUB> is the Boltzmann constant
      </LI><LI>T is the temperature (taken here as 25°C)
      </LI><LI>N is the length of the genome
    </LI></UL><P style="font-family:'Roboto Condensed'; font-size:150%">It follows that the rate of Cas9 binding to a given site is give as the rate of contact with the site (r<SUB>RW</SUB>) multiplied by the probability of binding. Once Cas9 has bound, it may cause a double stranded break in the site, usually around 3 base pairs from the PAM. However, there is still a chance that it will dissociate from the site before it has the chance to cleave, leading to the equation</P><FIGURE><CENTER><FIGCAPTION>Eqn 5
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Where k<SUB>c</SUB> is the cleavage rate constant, and k<SUB>d</SUB> the dissociation rate constant.</P><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>3. Homologous recombination</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">Once a site has been cleaved by Cas9, the site will either be repaired by homologous recombination, or its lack of stability will degrade this copy of the chloroplast genome until it is unable to be repaired. In the C. reinhardtii chloroplast, there is no evidence of non-homologous end joining, and thus repair seems to happen exclusively through homologous recombination [5]. Homology-directed repair requires another, un-cleaved copy of the broken site, to act as a template for repair of the cleaved one. We thus model it as a second order process, dependent on both the number of cleaved and uncleaved sites. The rate of degradation is considerably less than the rate of recombination [6], and so here we neglect it, leading to the following expression for a given target site:</P><FIGURE><CENTER><FIGCAPTION>Eqn 6
        </FIGCAPTION></CENTER></FIGURE><UL style="font-family: Roboto Condensed; font-size: 150%; text-align:left">Where:
      <LI>N<SUB>target[i]</SUB> is the number of un-cleaved copies of the Cas9 binding site
      </LI><LI>N<SUB>total[i]</SUB> is the total number of both cleaved and un-cleaved copies of the binding site, such that Ntotal[i]-N<SUB>target[i]</SUB> is the number of cleaved copies of the binding site
      </LI><LI>k<SUB>h</SUB> is the homologous recombination rate parameter
    </LI></UL><P style="font-family:'Roboto Condensed'; font-size:150%">As we are neglecting the possibility of degradation of the cut site, N<SUB>total[i]</SUB> will be equal to the chloroplast’s copy number, N<SUB>copy</SUB>, for all sites but the on-target site (i.e., the site which exactly matches the gRNA). However, for the on-target site, N<SUB>total[i]</SUB> will steadily decrease, as these sites are converted to containing the gene of interest, and are no longer viable Cas9 binding sites. Here, we assume perfect selection pressure – i.e. that once the on-target site has been cut, there is 100% chance of it being repaired using the gene of interest, and once it is converted to containing the gene of interest (and the antibiotic resistance which is found on the same cassette), there is 0 probability of it being converted back. The first assumption was somewhat validated by our sensitivity analysis (see below), and the second seems reasonable considering that once the site is converted, it is no longer a target for Cas9 cleavage. This leads to the expression</P><FIGURE><CENTER><FIGCAPTION>Eqn 7
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Here, N<SUB>GOI</SUB> is the number of copies of the gene of interest present in the chloroplast, with flanking homology regions. This will initially be the number of “gene of interest” cassettes inserted via the transformation method (e.g. biolistics), but will increase as more copies of the genome contain the gene of interest, following the expression</P><FIGURE><CENTER><FIGCAPTION>Eqn 8
        </FIGCAPTION></CENTER></FIGURE></DIV></SECTION><SECTION id="param" style="background-color:#fff; text-align: center"><DIV class="container" style="padding:0% 0%"><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>PARAMETERISING THE MODEL</EM></H3><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>1. Gene expression</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">As with all chloroplasts, regulation of gene expression in the C. reinhardtii chloroplast is a complex process. The majority of gene regulation happens at the post-transcriptional level, with factors changing the shape of mRNA’s 3’ and 5’ untranslated regions (UTRs) in response to a range of factors including light conditions, and nuclear signals indicating conditions such as the point in a C. reinhardtii cell’s circadian rhythm [7]. This results in extremely dynamic mRNA degradation and translation rates, making gene expression very hard to model accurately. Further complicating the matter is a reasonably noticeable lack of data on absolute gene expression levels in the C. reinhardtii chloroplast.</P><P style="font-family:'Roboto Condensed'; font-size:150%">After a considerable literature review, no absolute, dynamic data on mRNA levels in C. reinhardtii chloroplasts was found. This meant that the only way to accurately parameterise the gene expression portion of the model would be to perform wet lab experiments – however, our part library took sufficiently long to assemble to prohibit the running of these experiments in the allotted time. Thus, our parameters are likely to only give rough, order of magnitude solutions.</P><P style="font-family:'Roboto Condensed'; font-size:150%">The transcription rates, k<SUB>atpA</SUB> and <SUB>kpsaA</SUB> (for genes expressed under the atpA and psaA promoters respectively), are likely to be tied to be strongly tied to promoter strength. However, in lieu of any data on the strength of these two promoters, we used the simple order-of-magnitude transcription rate formula proposed by the NTU-Singapore iGEM team in 2008 [8]:</P><FIGURE><CENTER><FIGCAPTION>Eqn 9
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">This formula has been used by several iGEM teams since, with reasonable accuracy reflected in the results. The psaA promoter is used to express the 20 nucleotide gRNA + 455 nucleotide gRNA scaffold (for 475 nucleotides total), and the atpA promoter used to express the 4196 nucleotide Cas9 mRNA. It should also be noted that these promoters were chosen based on personal correspondence with Prof. Saul Purton, and could be changed to different promoters if experiments show them to be superior, further validating our decision to use a more general estimate of transcription rates.</P><P style="font-family:'Roboto Condensed'; font-size:150%">RNA degradation rates vary considerably throughout the C. reinhardtii cell cycle, responding to factors mentioned above. Regulation happens primarily through formation of stem loop structures, mediated by the UTRs which are attached to the promoters, altering the RNA stability [9]. However, while the mRNA for Cas9 is expressed with the 3’ and 5’ UTRs for the atpA gene cluster, gRNA is not expressed with any UTRs for proper interaction with Cas9. This leads to a large number of unknown factors in determining these degradation parameters. We decided to just use the average mRNA half-life for C. reinhardtii chloroplasts, and then convert it to the same degradation rate for each chloroplast by the formula</P><FIGURE><CENTER><FIGCAPTION>Eqn 10
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">However, no data on RNA half-lives in C. reinhardtii was found, so we used a value of t<SUB>1/2</SUB> = 5.352 hours, taken by averaging the data for half-lives of several different RNA strands in Arabidopsis thaliana chloroplasts [10], leading to an average RNA degradation rate of 0.129hr<SUP>-1</SUP>.</P><P style="font-family:'Roboto Condensed'; font-size:150%">Cas9 translation rate was again very hard to parameterise without wet lab experiments. Translation rate is a highly variable parameter in C. reinhardtii chloroplasts, acting as a hotspot for gene regulation. Translation is mediated by factors in the 5’ and 3’ UTRs of C. reinhardtii chloroplasts, however, experiment shows that steady state protein levels for the atpA gene are invariant of mRNA levels, suggesting that only a small portion of mRNA is actually transcribed at any given time [12].</P><P style="font-family:'Roboto Condensed'; font-size:150%">Data exists on steady-state levels of VFP expressed under the atpA promoter (and associated UTRs) in C. reinhardtii chloroplasts [13], and we attempted to use this data, with the previously calculated degradation and transcription rates (VFP is also a highly stable protein in C. reinhardtii chloroplasts, likely to “degrade by dilution”), to work backwards and calculate the translation rate. However, this yielded a value 0.00027 min<SUP>-1</SUP>, 1-2 orders of magnitude lower than translation rates we had seen elsewhere. This yielded unexpectedly long timescales for homoplasmy (see results section). Because of this, we instead used a value of 0.0057min<SUP>-1</SUP>, used by the 2014 Waterloo team for Cas9 production in Streptococcus aureus [14]. This is hopefully a more accurate estimation (having been calculated from peptide elongation and ribosome binding data) given that translational control in chloroplasts bears considerable similarity to its prokaryotic origins.
</P><P style="font-family:'Roboto Condensed'; font-size:150%">Based on personal correspondence with Prof. Howard Salis suggesting that Cas9 is a very stable molecule, the Cas9 degradation rate was taken to be the same as the C. reinhardtii cell division rate (“degradation by dilution”), for which we used the fastest plausible C. reinhardtii cell division rate of 12 hours [11] (as a &quot;worst case scenario&quot; situation). Cas9:gRNA complex degradation rates was taken as the quicker of the gRNA and Cas9 degradation rates. All other parameters involving Cas9 formation were taken directly from the Salis Lab’s paper, which were fitted from the results of in vitro Cas9 cleavage experiments in a cytoplasm-like buffer.</P><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>2. Cleavage rates</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">All values from this section were taken directly from the paper by the Salis Lab, except for the volume of the C. reinhardtii chloroplast, which was taken to be 25µm<SUP>3</SUP>, half of the median Chlamydomanas cell volume of 50µm<SUP>3</SUP> [15]. Cas9’s diffusivity and characteristic length were estimated from its structure, while the other values were fitted from in vitro experiments in a cytoplasm-like buffer.</P><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>3. Homologous recombination</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">Chloroplast copy number was held constant, at C. reinhardtii’s average copy number of 80 [16]. The only other relevant parameter in this section is the homologous recombination rate constant, k<SUB>h</SUB>. Here, we used a value of k<SUB>h</SUB> = 0.43 min<SUP>-1</SUP>, lifted directly from a paper which measured this directly in E. coli [6]. Two issues about this parameter must be addressed – firstly, that it is a parameter for E. coli, and not C. reinhardtii, and secondly, that homologous recombination was modelled in the paper as a first order reaction, while we model it as a second order one.</P><P style="font-family:'Roboto Condensed'; font-size:150%">The literature seems to suggest that the first issue is not a problem. Chloroplasts are hypothesised to have originated as prokaryotic organisms, and behave similarly to them. In fact, the gene encoding the recA enzyme, which plays an enormous part in mediating homologous recombination, exhibits 53% similarity in E. coli and C. reinhardtii chloroplasts [17]. The second should not be a problem either, as we can safely assume that while recombination in E. coli, as with C. reinhardtii, is in fact a second order reaction - dependent on both the number of broken sites, and the number of available repair templates - there will always only be one repair template (the sister chromosome), allowing it to appear as though it is a first order reaction.</P><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>4. Final table of parameters</EM></P><TABLE style="font-family:Arvo; font-size:130%; border: 5px;margin-left:auto; margin-right:auto"><TBODY><TR><TH>Reference</TH><TH>Parameters</TH><TH>Value</TH><TH>Calculated from</TH></TR><TR><TD>[8]</TD><TD>k<SUB>atpA</SUB></TD><TD>1.05 min<SUP>-1</SUP></TD><TD>Generalised formula</TD></TR><TR><TD>[8]</TD><TD>k<SUB>psaA</SUB></TD><TD>8.84 min<SUP>-1</SUP></TD><TD>Generalised formula</TD></TR><TR><TD>[10]</TD><TD>δ<SUB>atpA/psaA</SUB></TD><TD>1.05 min<SUP>-1</SUP></TD><TD>A. thaliana</TD></TR><TR><TD>Personal correspondence with Prof. Howard Salis</TD><TD>δ<SUB>Cas9</SUB></TD><TD>1.39e-3 min<SUP>-1</SUP></TD><TD>“Degradation by dilution”</TD></TR><TR><TD>[14]</TD><TD>k<SUB>Cas9</SUB></TD><TD>0.0057 min<SUP>-1</SUP></TD><TD>S. aureus</TD></TR><TR><TD>[6]</TD><TD>k<SUB>h</SUB></TD><TD>0.43 min<SUP>-1</SUP></TD><TD>E. Coli</TD></TR><TR><TD>[16]</TD><TD>N<SUB>copy</SUB></TD><TD>80</TD><TD>C. reinhardtii</TD></TR><TR><TD>[1]</TD><TD>k<SUB>c</SUB>/k<SUB>d</SUB></TD><TD>0.0016</TD><TD>Fitted to data taken in cytoplasm-like buffer</TD></TR><TR><TD>[1]</TD><TD>k<SUB>l</SUB></TD><TD>60 min<SUP>-1</SUP></TD><TD>Fitted to data taken in cytoplasm-like buffer</TD></TR><TR><TD>[1]</TD><TD>k<SUB>f</SUB></TD><TD>4.8 min<SUP>-1</SUP></TD><TD>Fitted to data taken in cytoplasm-like buffer</TD></TR><TR><TD>[1]</TD><TD>D</TD><TD>45 µm<SUP>3</SUP>s<SUP>-1</SUP></TD><TD>Cas9 protein structure</TD></TR><TR><TD>[1]</TD><TD>λ</TD><TD>0.015 µm</TD><TD>Cas9 protein structure</TD></TR></TBODY></TABLE></DIV></SECTION><SECTION id="implementation" style="background-color:#B7E2F0; text-align: center"><DIV class="container" style="padding:0% 0%"><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>PRACTICAL IMPLEMENTATION</EM></H3><P style="font-family:'Roboto Condensed'; font-size:150%">The model was implemented in MATLAB, due to its efficient functioning in solving systems of ordinary differential equations. The script “Cas9.m” reads in the C. reinhardtii genome from a text file, and generates a copy of the reverse strand of the genome. It then checks if all potential DNA sites on both the forward and reverse genome have a PAM which could bind to Cas9 in an energetically advantageous way (although Cas9’s PAM is NGG, several other PAMs are still capable of binding Cas9). For the circular, 203 828 base pair C. reinhardtii genome, this means scanning 407 656 different sites, yielding 88 740 potential binding sites.</P><P style="font-family:'Roboto Condensed'; font-size:150%">The script then calculates the free energy change which would be involved in Cas9’s binding to all potential target sites. However, here an issue was encountered with the ΔΔG<SUB>Cas9:gRNA</SUB> values which we were using. Theory suggests that perfect matches will have a ΔΔG<SUB>Cas9:gRNA</SUB> value of 0, while mismatches will have a positive ΔΔG<SUB>Cas9:gRNA</SUB> value. However, many perfect matches had positive ΔΔG<SUB>Cas9:gRNA</SUB> values, and some mismatches had negative ones, leading to imperfect identification of the on-target site. We solved this problem in the same way as the Salis lab – by calculating the average mismatch free energy for the C. reinhardtii genome using the given data set (yielding a value of 0.9039 kcal/mol), and simply using this value for ΔΔG<SUB>Cas9:gRNA</SUB> for all duplex mismatches, and a value of 0 for all duplex matches.</P><FIGURE><CENTER><FIGCAPTION>Graph 1
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">88 740 target sites should yield 88 747 differential equations (5 for Cas9:gRNA active complex formation, 88 740 for the cleavage/repair of each site, and 2 for the evolution of N<SUB>total[on-target site]</SUB> and N<SUB>GOI</SUB>). This can be simplified by assuming that there is 0 binding probability if ΔG<SUB>target</SUB> is greater than 0, however, this only narrows it down to 29 146 target sites – still too many for MATLAB to deal with in any reasonable amount of time.</P><FIGURE><CENTER><FIGCAPTION>Graph 2
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">The final simplification in our model stems from the Salis lab’s observation that 87% of significant Cas9 binding activity occurs at target sites with at most 1 mismatch in the first 8 nucleotides proximal to the PAM. Only 84 of the aforementioned 29 146 sites in the C. reinhardtii chloroplast genome meet this specification, and so we made the simplification of only considering these 84 sites (including the on-target site) with individual differential equations. While the other sites may not have significant binding activity by themselves, the activities of all of them together are likely to make a noticeable impact on the system. However, as their binding rates are likely to be small (particularly in early time periods such as those considered in our model, where there is still a multitude of on-target DNA sites which Cas9 is attracted to far more strongly), we were able to make the simplification of combining all of their action together into one differential equation, representing the average cleavage rate of a site with more than 1 mismatch in the first 8 PAM-proximal nucleotides.</P><FIGURE><CENTER><FIGCAPTION>Graph 3
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">This results in a system of 92 ODEs (5 for Cas9:gRNA active complex formation, 84 for the cleavage/repair of each site with 0-1 mismatches in the first 8 PAM-proximal nucleotides, 1 for the average cleavage repair of the other 29 062 sites, and 2 for the evolution of N<SUB>total[on-target site]</SUB> and N<SUB>GOI</SUB>), easily solvable by MATLAB’s ode15s function. The form of the ODEs is set up in the “CRISPRsim.m” file. Another file, “stopevent.m”, allows the simulation to be ended when all copies of the chloroplast genome have been transformed. The completed code can be found in the download link at the top of the page, including a sample file “tests.m” for running the sensitivity analyses mentioned in the next section.</P></DIV></SECTION><SECTION id="results" style="background-color:#fff; text-align: center"><DIV class="container" style="padding:0% 0%"><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>RESULTS &amp; DISCUSSIONS</EM></H3><P style="font-family:'Roboto Condensed'; font-size:150%">Initial simulations were run assuming that precisely 1 copy of each cassette (the “driver” and gene of interest) had been inserted into the chloroplast. The number of plasmids inserted into the chloroplast will vary, and future teams may want to build on this section by describing the numbers of cassettes likely to enter the cell by various transformation methods probabilistically. However, if one of the cassettes does not enter the chloroplast at all, a homoplasmic transformation will never occur, so one of each plasmid represents a “worst case scenario” situation for homoplasmy timescales in a cell which will eventually transform.</P><P style="font-family:'Roboto Condensed'; font-size:150%">As mentioned above, initial simulation used a parameter of 0.00027 min<SUP>-1</SUP> for Cas9 translation rate, yielding an estimate of 36h15m for homoplasmy. This was out of line with our estimates based on personal correspondence with several sources for Cas9 cleavage timescales, and so we updated this parameter to 0.0057min<SUP>-1</SUP>, which decreased the expected time to <B>7h5m</B>, more in line with our expectations. This is less than even C. reinhardtii’s shortest possible cell division time, showing <B>homoplasmy to be achievable in a single generation using Cas9</B>. This is a significant improvement on current chloroplast transformation methods, which can take months to reliably achieve homoplasmy.</P><FIGURE><CENTER><FIGCAPTION>Graph 4
        </FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION>Graph 5
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">The graphs of on-target sites over time show the expected. For early time periods, the cleavage of the on-target site (which has around 7X higher probability of cleavage than the next most likely site) dwarfs all other sites, and untransformed genome copies tail off rapidly. Around 4 hours in, the rate of transformation begins to tail off, as less on-target sites exist and the probability of being “found” by Cas9 is lower. In response, cleavage rates of off-target sites increase, due to decreased competition, but quickly regain steady state levels due to the speed of homologous recombination.</P><FIGURE><CENTER><FIGCAPTION>Graph 6
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Plots of gRNA and Cas9 over time show the gRNA to be the limiting factor in Cas9:gRNA active complex formation, due to its higher degradation rate. Plots of off-target sites over time show very few of them to be cleaved at any given time, due to the high probability of homology-directed repair. This is encouraging, as it shows our assumption of being able to combine Cas9 activity at several thousand other sites into a single differential equation due to very low rates of change to be valid. Only one site (shown on the graph in purple) – which has a free energy change of only around 1 kcal.mol<SUP>-1</SUP> less than the on-target site – shows any significant activity at steady state.</P><FIGURE><CENTER><FIGCAPTION>Graph 7
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%"><EM>2. Sensitivity analysis</EM></P><P style="font-family:'Roboto Condensed'; font-size:150%">The Salis lab have already done sensitivity tests for their own model parameters, however, we thought it pertinent to check our own model’s sensitivity to our own parameters. Without experimental data to check our model against, we chose instead to measure how the model’s expected time to homoplasmy varies with each parameter across one order of magnitude difference (centred around our final value), if all other parameters are held constant. This would also allow us to validate several assumptions about the model.</P><P style="font-family:'Roboto Condensed'; font-size:150%">First, we checked the effect of the assumption of “perfect selection pressure”, by varying the percentage of the time in which the on-target site is transformed by homology-directed repair, rather than being repaired back to its original state. The results were encouraging – even when the cut site used the gene of interest as its template for homology-directed repair a mere 10% of the time (with it using another genome copy the other 90% and reverting to its original form), the time to homoplasmy was only increased by 50 minutes (a 12% increase), with very little change in the homoplasmy time observed from about a 50-50 chance of template choice onwards.</P><P style="font-family:'Roboto Condensed'; font-size:150%">We next checked the effect of increasing the number of “driver” and “gene of interest” cassettes. The number of “gene of interest” cassettes inserted into the chloroplast had no effect at all on the timescales to homoplasmy, as the information on this cassette is rapidly duplicated during the transformation. However, increasing the number of “driver” cassettes greatly decreased the timescale to homoplasmy, with an order of magnitude increase (from 1 to 10) more than halving the timescale.
</P><FIGURE><CENTER><FIGCAPTION>Graph 8
        </FIGCAPTION></CENTER></FIGURE><P style="font-family:'Roboto Condensed'; font-size:150%">Finally, we varied our values for k<SUB>psaA</SUB>, k<SUB>atpA</SUB>, δ<SUB>psaA</SUB>/δ<SUB>atpA</SUB>, δ<SUB>Cas9</SUB>, k<SUB>Cas9</SUB>, and k<SUB>h</SUB>. Varying k<SUB>psaA</SUB> and k<SUB>h</SUB> had negligible effect on homoplasmy timescales and is not graphed here. The results show the model to be highly sensitive to varying Cas9’s transcription and translation rates in either direction, and to increasing RNA degradation rates, yet reasonably insensitive to all other parameter changes. Encouragingly, the maximum plausible timescale for homoplasmy which results from parameter variation across two orders of magnitude is 1345 minutes (22.45 hours), still within a single day.
</P><TABLE style="font-size:130%; border: 5px;margin-left:auto; margin-right:auto"><TBODY><TR><TH>Parameter</TH><TH colspan="2">Lower bound</TH><TH colspan="2">Upper bound</TH><TH colspan="2">Sensitivity (as percentage of final value)</TH></TR><TR><TH>Value (min<SUP>-1</SUP>)</TH><TH>Homoplasmy time (min)</TH><TH>Value (min<SUP>-1</SUP>)</TH><TH>Homoplasmy time (min)</TH></TR><TR><TD>k<SUB>atpA</SUB></TD><TD>0.105</TD><TD>1345</TD><TD>10.5</TD><TD>167</TD><TD>271.18</TD><TD>-60.52</TD></TR><TR><TD>δ<SUB>psaA</SUB>/δ<SUB>atpA</SUB></TD><TD>2.16e-4</TD><TD>379</TD><TD>2.16e-2</TD><TD>970</TD><TD>-10.40</TD><TD>129.3</TD></TR><TR><TD>δ<SUB>Cas9</SUB></TD><TD>1.39e-4</TD><TD>408</TD><TD>1.39e-2</TD><TD>431</TD><TD>-3.55</TD><TD>1.89</TD></TR><TR><TD>k<SUB>Cas9</SUB></TD><TD>0.00057</TD><TD>1345</TD><TD>0.057</TD><TD>167</TD><TD>217.18</TD><TD>-60.52</TD></TR></TBODY></TABLE><FIGURE><CENTER><FIGCAPTION>Graph 9
        </FIGCAPTION></CENTER></FIGURE></DIV></SECTION><SECTION id="references" style="background-color:#B7E2F0; text-align: center"><DIV class="container" style="padding:0% 0%"><H3 style="font-family:Roboto; font-weight:bold; text-align: center"><EM>REFERENCES</EM></H3><P style="font-family:'Roboto Condensed'; font-size:150%">[1]<A target="_blank" href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004724
"> Farasat I, Salis HM (2016). “A Biophysical Model of CRISPR/Cas9 Activity for Rational Design of Genome Editing and Gene Regulation”. PLoS Comput Biol 12(1): e1004724.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[2] <A target="_blank" href="http://www.nature.com/nature/journal/v507/n7490/full/nature13011.html"> Sternberg SH, Redding S, Jinek M, Greene EC, Doudna JA (2014). “DNA interrogation by the CRISPR RNA-guided endonuclease Cas9”. Nature 507(7490): 62-67.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[3] <A target="_blank" href=" https://2013.igem.org/Team:WHU-China/modelingCas9"> WHU China iGEM team 2013</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[4] <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/2339053"> Thompson RJ, Mosig G (1990). “Light affects the structure of Chlamydomonas chloroplast chromosomes”. Nucleic Acids Res 18(9): 2625-31</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[5]<A target="_blank" href=" https://www.ncbi.nlm.nih.gov/pubmed/18036204">  Odom Odom OW, Baek KH, Dani RN, Herrin DL (2008). “Chlamydomonas chloroplasts can use short dispersed repeats and multiple pathways to repair a double-strand break in the genome”. Plant J 53(5): 842-53.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[6] <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC392555/"> Sinden RR, Cole RS (1978). “Topography and kinetics of genetic recombination in Escherichia coli treated with psoralen and light”. Proc Natl Acad Sci USA 75(5): 2373–2377.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[7]<A target="_blank" href="http://www4.rz.ruhr-uni-bochum.de:8253/imperia/md/content/allgbotanik/jn_photores.pdf">  Herrin DL, Nickelsen J (2004). “Chloroplast RNA processing and stability”. Photosynthesis Research 82: 301–31.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[8] <A target="_blank" href="https://2008.igem.org/Team:NTU-Singapore/Modelling/Parameter">NTU Singapore iGEM team 2008</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[9] <A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/18036204"> Stern DB, Radwanski ER, Kindle KL (1991). “A 3' stem/loop structure of the Chlamydomonas chloroplast atpB gene regulates mRNA accumulation in vivo”. Plant Cell 3(3):285-97</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[10] <A target="_blank" href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r45"> Sidaway-Lee K, Costa MJ, Rand DA, Finkenstadt B, Penfield S (2014). “Direct measurement of transcription rates reveals multiple mechanisms for configuration of the Arabidopsis ambient temperature response”. Genome Biology 15(R45)</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[11] <A target="_blank" href="http://link.springer.com/article/10.1007/s00425-010-1282-y"> Vítová, M., Bišová, K., Umysová, D. et al. (2011). “Chlamydomonas reinhardtii: duration of its cell cycle and phases at growth rates affected by light intensity”. Planta 233: 75.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[12]<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC160129/">  Drapier D, Girard-Bascou J, Wollman FA (1992). “Evidence for Nuclear Control of the Expression of the atpA and atpB Chloroplast Genes in Chlamydomonas”. Plant Cell 4(3): 283–295. </A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[13] <A target="_blank" href="http://onlinelibrary.wiley.com/doi/10.1002/biot.201400566/abstract"> Braun-Galleani S, Baganz F, Purton S (2015). “Improving recombinant protein production in the Chlamydomonas reinhardtii chloroplast using vivid Verde Fluorescent Protein as a reporter”. Biotechnology Journal, 10: 1289–1297</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[14] <A target="_blank" href="https://2014.igem.org/Team:Waterloo/Math_Book/CRISPRi"> 2014 Waterloo iGEM team</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[15]<A target="_blank" href="http://jcs.biologists.org/content/joces/54/1/173.full.pdf">  Craigie RA, Cavalier-Smith T (1982). “Cell Volume and the Control of the Chlamydomonas Cell Cycle”. Journal of Cell Science 54: 173-191.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[16] <A target="_blank" href="http://link.springer.com/article/10.1007/BF01453455"> Misumi O, Suzuki L, Nishimura Y, Sakai A, Kawano S, Kuroiwa H, Kuroiwa T (1999). “Isolation and phenotypic characterization of Chlamydomonas reinhardtii mutants defective in chloroplast DNA segregation”. Protoplasma 209: 273–282.</A></P><P style="font-family:'Roboto Condensed'; font-size:150%">[17]<A target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/7760798">  Cerutti H, Johnson AM, Boynton JE, Gillham NW (1995). “Inhibition of chloroplast DNA recombination and repair by dominant negative mutants of Escherichia coli RecA”. Mol Cell Biol 15(6): 3003-11</A></P></DIV></SECTION><SECTION id="footer-sec"><DIV class="container" style="margin-top:0px"><DIV class="row  pad-bottom"><DIV class="col-md-3"><H4><STRONG>ABOUT US</STRONG></H4><P style="text-align:center">
                    We are a team of Cambridge undergraduates, competing for the plant synthetic biology prize in iGEM 2016.
                    <A href="//2016.igem.org/Team:Cambridge-JIC/Team">read more</A></P></DIV><DIV class="col-md-3"><H4><STRONG>FOLLOW US ON</STRONG></H4></DIV><DIV class="col-md-3"><H4><STRONG>LOCATION</STRONG></H4><P style="text-align:center">
                    Department of Plant Sciences, 
                    University of Cambridge  
                    Downing Street 
                    CB2 3EA
                    </P></DIV><DIV class="col-md-3"><H4><STRONG>CONTACT US</STRONG></H4><P style="text-align:center">
                    Email: <A href="mailto:cambridgeigem@hotmail..com" target="_blank">cambridgeigem@hotmail.com</A>
                    Tel: <A href="tel:+447528848745">+447827442925</A></P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>