<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-Software_Validation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-Software/Validation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-Software/Validation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/Validation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-Software/Validation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-Software/Validation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-Software/Validation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team:SYSU-Software</TITLE></P><DIV class="myContainer"><DIV class="navigation1" id="nav"><DIV class="item project"><UL style="top: 8px;"><LI><A href="https://2014.igem.org/Team:SYSU-Software/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Design">Design</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Validation">Validation</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Demo">Demo</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Download">Download</A></LI></UL></DIV><DIV class="item doc"><UL style="top: 25px;"><LI><A href="https://2014.igem.org/Team:SYSU-Software/Tutorial">Tutorial</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Model">Models</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Documentation">Documentation</A></LI></UL></DIV><DIV class="item require"><UL style="top: 35px;"><LI><A href="https://2014.igem.org/Team:SYSU-Software/Medal">Medals</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Collaboration">Collaboration</A></LI></UL></DIV><DIV class="item hp"><UL style="top: 50px;"><LI><A href="https://2014.igem.org/Team:SYSU-Software/Meetup">Meetups</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/SYSU-Software-BBO">BBO</A></LI></UL></DIV><DIV class="item team"><UL style="top: 25px;"><LI><A href="https://2014.igem.org/Team:SYSU-Software/Member">Members</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Attribution">Attribution</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Ackmt">Acknowledgement</A></LI><LI><A href="https://2014.igem.org/Team:SYSU-Software/Sponsor">Sponsors</A></LI></UL></DIV></DIV><DIV class="bigFather"><DIV id="bgLeft"><DIV class="cardS card1"><A href="#myPage1" class="mC1">Validation in Practice</A></DIV><DIV class="cardS card2"><A href="#myPage2" class="mC2">Validation in Theory</A></DIV></DIV><DIV id="bgContent"><DIV id="myArticle" name="myArticle"><H1 style="text-align: center; font-size: 40px; font-weight: bold; font-family: verdana;">Validation</H1><P>A convincing validation of our development includes tests on efficiency and accuracy. Efficiency, on one hand, is mainly about the balance of complexity and run time of our algorithms. Accuracy, on the other hand, emphasizes the importance of how well our algorithms can mimic the realistic experimental results. </P><P>In this wiki page, we offer our validation on FLAME, and provide lines of evidence on the reliability of FLAME.</P><P id="myPage1" name="myPage1" style="font-weight:bold;">Validation in Practice -- An Experimental Approach</P><P>We often feel obliged to validate our ideas with experiments, and the following are our results.</P><P>We constructed a simple circuit that can control the expression of GFP and thus the strength of fluorescence via IPTG. pET-28a plasmids contain a <SPAN style="font-style:italic;">lacI</SPAN> gene, a specific T7 promoter and a kanamycin resistance gene. The <SPAN style="font-style:italic;">gfp</SPAN> gene was cloned into the plasmids downstream of T7 promoter. The host cell for the plasmids was <SPAN style="font-style:italic;">Escherichia coli</SPAN> strain BL21 (DE3), which expresses the T7 RNA polymerase and contains the lactose promoter and operator in its genome (See Figure 1). Fluorescence intensity of GFP in the presence of 0.05 mM or 0.1 mM IPTG was measured every 30 minutes<SUP>[1][2][3]</SUP>.</P><P style="font-size:12px; text-align:center;"><SPAN style="font-weight:bold;">Figure 1</SPAN> Schematic drawing of pET-28a plasmid and <SPAN style="font-style:italic;">E. coli</SPAN> strain strain BL21 (DE3) host chromosome</P><P>Results are as follows. The background fluorescence intensity is 14.</P><P>As can be seen in the curves, the increase in fluorescence intensity versus time can be depicted by the Hill function when the Hill coefficient equals 1, and it represents a progress curve.</P><P>The strength of GFP fluorescence is related to the concentration of GFP proteins, and the strength increases in direct proportion to the concentration. So, we obtained the concentration curve of GFP proteins versus time in FLAME, and compared it with the curve of the strength of fluorescence versus time measured in experiments. The simulation results are as follows.</P><P style="font-size:12px; text-align:center;"><SPAN style="font-weight:bold;">Figure 2 </SPAN>Experimental results versus simulation results. Upper: curve of fluorescence intensity versus time when 0.1 mM IPTG is added. Middle: curve of fluorescence intensity versus time when 0.05 mM IPTG is added. Lower: simulation results of Dynamic Performance.</P><P>Through the choice of time interval, we can obtain various Dynamic Performance curves. Compare the obtained simulation curves with real experimental results, several conclusions can be drawn. First, the obtained curves are progress curve, rather than an “S” shape; Second, although the X-axis has different time intervals, the fraction of time when the curves are increasing is about the same. Third, both the curves start from the original point. We can draw an initial conclusion that our simulation is in accordance with the realistic experimental results.</P><P>The wetlab validation verified our mathematical simulation. The simulation in FLAME can depict the real biological processes and give instructive guidance to researchers.</P><P id="myPage2" name="myPage2" style="font-weight:bold;">Validation in Theory -- Why the Hill Equation is Adopted?</P><P>After a validation through wetlab experiments, we hope to validate and verify our model in a theoretical way; that is, why the utilization of the Hill equation is reasonable?</P><P>Activation or Repression can be depicted by equations below:</P><P>repression:   </P><P>activation:   </P><P>(The biological senses of all these parameters can be seen in the <A href="https://2014.igem.org/Team:SYSU-Software/Model">Models</A> page.)</P><P>It is of note that, to depict the rate of protein synthesis, we utilize the Hill equation. There are at least two reasons. </P><P>First, the rate of protein synthesis is not dependent upon the concentration of the protein that is being synthesized; rather, it depends on the concentration of activator or repressor. In the Hill equation, the concentration of activators or repressors are considered[].</P><P>Second, the curve of concentration of a protein versus time is usually a sigmoid curve (“S” shape) or a progress curve. In the Hill equation, when the Hill coefficient equals to 1, a progress curve will be obtained; when the Hill coefficient is greater than 1, we will get a sigmoid curve.</P><P style="font-size:12px; text-align:center;"><SPAN style="font-weight:bold;">Figure 3</SPAN> Curves of Hill equation. When the Hill coefficient n = 1, a progress curve can be obtained; when n &gt; 1, a sigmoid curve can be obtained. The curves of increasing slope correspond to n = 1, 2, 4, 8<SUP>[4]</SUP>.</P><P>Third, the Hill equation can be easily modified to depict increase or decrease in amount versus time, which can be used to describe the increase or decrease in the rate of protein synthesis. When adopted in the ODE, it can depict biological processes of activation or repression. And many circuits and/or systems can be described as the interaction (both positive, activation or negative, repression) between their components within. </P><P>Fourth, the relatively simple Hill equation can enhances the operation rate of FLAME.</P><P>Therefore, it is reasonable to use the Hill equation to furthermore depict the process of repression and/or activation. We began with the two basic equation mention above and wrote the algorithms for every structure. Utilization of the Hill function to represent the process of activation and/or repression is our method to maintain a balance between accuracy and efficiency in Simulation. For more details, please see <A href="https://2014.igem.org/Team:SYSU-Software/Model">Models</A> page. </P><P style="font-size:13px;">[1] JR Kelly, Tools and reference standards supporting the engineering and evolution of synthetic biological systems, Department of Biological Engineering,2006</P><P style="font-size:13px;">[2] Iadevaia S., and Mantzaris N.V., Genetic network driven control of PHBV copolymer composition, Journal of Biotechnology, 2006,122(1), 99—121.</P><P style="font-size:13px;">[3] Gardner, T.S., Cantor, C.R. &amp; Collins, J.J. Construction of a genetic toggle switch in Escherichia coli. <SPAN style="font-style:italic;">Nature</SPAN> 403, 339-342 (2000).</P><P style="font-size:13px;">[4] Frank, S.A. Input-output relations in biological systems: measurement, information and the Hill equation. <SPAN style="font-style:italic;">Biol Direct</SPAN> 8, 31 (2013).

</P></DIV></DIV><DIV id="address1"><P>Email: sysusoftware@126.com</P><P>Address: 135# Xingang Rd(W.), Sun Yat-sen University, Guangzhou, China</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SYSU-Software/Validation">http://2014.igem.org/Team:SYSU-Software/Validation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-Software/Validation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-Software/Validation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-Software/Validation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-Software/Validation&amp;oldid=368915" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>