<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL class="navigation-bar"><LI><A href="https://2017.igem.org/Team:Fudan">HOME</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Project">  Project  </A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Applied_Design">SwordS</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/HCC">HCC</A></LI></UL><LI><A href="https://2017.igem.org/Team:Fudan/Results">  Results  </A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/Demonstrate">Demonstrate</A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/Basic_Part">Basic Part</A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/Composite_Part">Composite Part</A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/Part_Collection">Part Collection</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Lab">  Lab  </A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/ModelandSoftware">  Model  </A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Software">Software</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/HP">  HP  </A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/HP/Silver">Silver HP</A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/HP/Gold_Integrated">Integrated HP</A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/Engagement">Cell War!</A></LI><LI class="long"><A href="https://2017.igem.org/Team:Fudan/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Team">  Team  </A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Teamers">Teamers</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Attributions#Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2017.igem.org/Team:Fudan/Attributions#Attributions">Attributions</A></LI></P><DIV class="container"><DIV class="block main-text sub-title  dark-blue"><H class="sup-title dark-blue">Results</H></DIV><DIV class="block main-text main-text"><H class="highlight-title dark-blue">Surface antigen density heterogeneity model </H><P>Although antigen density heterogeneity has been reported, it has been considered as an obstacle to improve cellular immunotherapy for tumor<SUP>(1)</SUP>, rather than an opportunity to employ diverse and/or combined treatment. To demonstrate the feasibility of utilizing antigen density heterogeneity, we first established a cell model that is able to stably express different antigen densities on cell surface. We chose surface-expressed EGFP (surEGFP) as representative antigen due to the convenience of detecting, quantifying and observing <A href="#figure1">(Figure 1A)</A>. </P><P>
After viral infection, K562-surEGFP cells were observed by microscopy. Image shows that the endoplasmic reticulum (ER) membrane has strong fluorescence, while the plasma membrane has relative weak signal. It consists with the process that surEGFP is expressed, translocated and matured inside ER lumen with its transmembrane embed in the ER membrane, and later surEGFP is transported and exposed to culture media when fluorescence declines due to oxidation in the extracellular environment. Thus, surEGFP was successfully located to cell membrane <A href="#figure1">(Figure 1B)</A>. We sorted stable cell lines depending on EGFP fluorescence intensities <A href="#figure1">(Figure 1C</A>, <A href="#figure1sr">S1A)</A>. The FACS gate set in FSC-SSC channel was relatively narrow to ensure we can get a group of cells with homogeneous size, to exclude the possibility that the heterogeneous expression of EGFP was due to different cell surface area <A href="#figure1sr">(Figure S1B, C)</A>. We confirm the FASC-ed cell clines with different EGFP expression by Western blotting using an anti-EGFP antibody <A href="#figure1">(Figure 1D)</A>. The bands clearly show difference between low-, medium-, high-expression groups <A href="#figure1">(Figure 1D)</A>.
We also characterized two HCC (hepatocellular carcinoma) cell lines using an anti-GPC3 antibody. We confirmed that SMMC-7721 cells express no GPC3, while Huh-7 cells highly express GPC3 <A href="#figure1">(Figure 1E)</A>. These cell lines are our physiological surface antigen density heterogeneity model. </P></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure 1. Establish and characterize two surface antigen density heterogeneity models. </H>(A) Sketch map outlines the surface-expressed EGFP density heterogeneity model. (B) Microscopy images show K562-surEGFP cells successfully express surEGFP. The <A href="https://en.wikipedia.org/wiki/Endoplasmic_reticulum" target="_blank">endoplasmic reticulum (ER)</A> membrane has strong fluorescence while the plasma membrane has relatively weak signal probably due to oxidation in the extracellular environment causing surEGFP to bleach really fast. The yellow arrowheads indicate surEGFP on the ER membrane facing ER lumen, before transporting to the plasma membrane. (C) K562 EGFP stable lines were sorted into three subtypes depending on EGFP fluorescence intensity by FACS. (D) FACS-ed cell lines have different EGFP expression. GAPDH (internal reference) shows the loading between lanes was equal. (E) Huh-7 cells highly express GPC3, while SMMC-7721 cells express no GPC3 - our HCC density heterogeneity model.
</P></DIV><DIV class="block main-text main-text"><H class="highlight-title dark-blue" id="Antigen">Engineering SynNotch with antigen-density dependent response</H><P>Engineered Notch has been used in a previous project <A href="https://2011.igem.org/Team:MIT">(iGEM 2011 MIT)</A>. They replaced the intracellular domain of Notch1 by Gal4-VP16 <A href="http://parts.igem.org/Part:BBa_K511301">(BBa_K511301)</A>. When the extracellular domain of Notch1 binding to the Delta ligand, Gal4-VP64 would be cleavage, located into nucleus, started transcription of its corresponding promoter Gal4-UAS <A href="http://parts.igem.org/Part:BBa_K511003">(BBa_K511003)</A>. 
This year, we took a step forward on their achievement. Referring to a latest publication<SUP>(2)</SUP>, we engineered a SynNotch that could recognize surEGFP and response with tTa <A href="#figure2">(Figure 2)</A>. </P></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure 2. SynNotch with LaG16 was used to recognize surEGFP. </H>HeLa cell stably expresses anti-EGFP synNotch with an activating-form transcription factor (tTa) whose corresponding promoter is pTight. The HeLa cells were transfected with a plasmid containing pTight-mCherry. When LaG16 (an anti-EGFP <A href="https://en.wikipedia.org/wiki/Single-chain_variable_fragment" target="_blank">scFv</A>) binds to surEGFP, Cleavages occur. Then, in the same cell, tTa will translocate into the nucleus and activate pTight, driving mCherry expression.
</P></DIV><DIV class="block main-text main-text"><P>
In HeLa with LaG16-SynNotch-tTa and pTight cells, the tTa, which originally linked to the intracellular domain of SynNotch, after LaG16 (scFv against GFP) binds to surEGFP, will translocate from the plasma membrane into the nucleus. tTas activate pTight promoter and mCherry starts expression. We used the cells highly expressing surEGFP as the sender. Images show that the HeLa expresses mCherry when contacts with K562-surEGFP, while mCherry won’t be seen in the condition that the HeLa don’t contact with K562-surEGFP <A href="#figure3">(Figure 3)</A>. Thus, we confirmed our SynNotch can respond to surEGFP. </P></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure 3. LaG16-SynNotch-tTa is activated by contacting with surEGFP. </H>Upper image. The rounded cell in the lower left corner is K562-surEGFP. In overexposed EGFP view (EGFP/OE), upper right corner, a group cells, HeLa-LaG16-SynNotch-tTa (we sort this stable line by cytoplasmic GFP, <I>please ask us for more details</I>), which express no mCherry (max-Z, maximum intensity projection of all optical slices we took of the field, with 0.2 micron step), because of non-contact with K562-EGFP. Lower image. HeLa-LaG16-SynNotch-tTa is expressing mCherry when contacts with K562-surEGFP. Slice 51 is very close to the middle plane of attached HeLa cells, and clearly show expressed mCherry and weak (only visible in OE) cytoplasmic GFP signal indicating the expression of LaG16-SynNotch-tTa. Max-Z shows the position of attached K562-surEGFP cells, which are floating and couple microns above attached HeLa cells.
</P></DIV><DIV class="block main-text main-text"><H class="highlight-title dark-blue" id="Antigen">Stripe modeling to study tri-response required conditions</H><P>To generate an antigen density-dependent tri-response pattern for SwordS, designing the gene transcription network in Stripe plays a central role. We model the transcription network of Stripe <A href="/Team:Fudan/Model/GTN">[Model: Network Modelling]</A> to better understand the characteristics of it. The theoretical basis of our model is explained in detail in <A href="/Team:Fudan/Model/HE">[Model: Theoretical Basis]</A>.
In silico modeling indicates that the key criterion for functional Stripe construction is the matching of orthogonal SynTF-SynPro pairs that enabled sufficient separation of the low and high thresholds. It enlightened us to create and characterize a set of SynTF-SynPro to achieve tunable intervals against tumor antigen with heterogeneity. 
To provide further insights into the modeling procedures, we have created an online software with dynamic interactions <A href="/Team:Fudan/Software/Netro_Fudan">[Software: Netro_Fudan]</A>. You will even be able to design and analyze your own gene transcription network in <A href="/Team:Fudan/Software/Netro">[Software: Netro]</A>.
</P></DIV><DIV class="block main-text main-text"><H class="highlight-title dark-blue" id="Antigen">Wiring orthogonal and tunable SynTF-SynPro set</H><P>Here, we present an approach to design customized mammalian synthetic transcription factor (SynTF) –synthetic promoter (SynPro) pairs. SynTFs enable binding to user-specified DNA sequences (response elements, REs), SynPros, and silence or activate the transcription after SynPros. The SynTFs we designed were in a unified style containing three core domains from N-terminal to C-terminal: DNA binding domain (DBD), nuclear location sequence (NLS), and transcription regulating domain. We chose (G4S) as the linker to add between DBD and SV40 NLS<SUP>(3)</SUP> to provide region flexibility<SUP>(4)</SUP>. We chose KRAB<SUP>(5)</SUP> or VP64<SUP>(6)</SUP> as transcription regulating domain to construct silencing- or activating-form of SynTFs, named SynTF(S)s or SynTF(A)s. Their corresponding silencing- or activating-form SynPros, SynPro(S)s or SynPro(A)s, were pSV40-N*RE or N*RE-minCMV <A href="#figure4">(Figure 4A)</A>.
The critical step in choosing optimal SynTF group is to find enough specific and orthogonal DBDs. We used two approaches. First, we did reference search for those commonly used DBD originating from different species. Secondly, we in-house designed based on artificial zinc-finger (ZF). For the first approach, we chose Gal4DBD<SUP>(2)</SUP>, PIP<SUP>(7)</SUP>, ZFHD1<SUP>(8)</SUP>. For the second approach, we utilized a modified 3-tandem Cys2-His2 ZF<SUP>(9)</SUP> as protein chassis. By replacing the DNA-interactional amino residues<SUP>(10)</SUP> on ZF modules<SUP>(11)</SUP>, we have generated RE-specific mammalian synthetic ZF (SynZF) <A href="#figure4">(Figure 4B)</A>. 
The SynTF-SynPro pairs we wired were listed in <A href="#Tablehere">(Table 1)</A>.</P></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure 4. Design of SynTF-SynPro. </H>(A) Wiring silencing- and activating-form SynTF-SynPro. (G4S), a protein linker to provide region flexibility. NLS, SV40 nuclear localization signal. N*RE, responding elements in N repeat. (B) Synthesis DBD via SynZF. Three zinc finger motifs were applied. SynTF, synthetic zinc finger. The 4 oligo DNA sequences illustrated are the RE sequence corresponding to ZF21-16, ZF43-8, ZF42-10, ZF54-8.
</P></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Table 1. The SynTF-SynPro pairs we wired. </H> Symbol: □, we test; △, worked as designed; ☆, our favorite parts. (Untest pairs were not included.)
</P></DIV><DIV><TABLE class="tg" width="60%" style="border-collapse:separate; border-spacing:10px 0px;"><TBODY><TR><TH class="tg-yw4l"><B>State</B></TH><TH class="tg-yw4l" colspan="2"><B>Silencing-form</B></TH><TH class="tg-yw4l" colspan="2"><B>Activating-form</B></TH></TR><TR><TD class="tg-yw4l"><B>SynTF(S)</B></TD><TD class="tg-yw4l"><B>SynPro(S)</B></TD><TD class="tg-yw4l"><B>SynTF(A)</B></TD><TD class="tg-yw4l"><B>SynPro(A)</B></TD></TR><TR><TD class="tg-yw4l"><CENTER>□△</CENTER></TD><TD class="tg-yw4l">SynTF(S)-Gal4</TD><TD class="tg-yw4l">SynPro(S)-Gal4;4*</TD><TD class="tg-yw4l">SynTF(A)-Gal4</TD><TD class="tg-yw4l">SynPro(A)-Gal4;4*</TD></TR><TR><TD rowspan="3" class="tg-yw4l">SynTF(S)-PIP</TD><TD class="tg-yw4l">SynPro(S)-PIP;2*</TD><TD class="tg-yw4l">SynTF(A)-PIP</TD><TD class="tg-yw4l">SynPro(A)-PIP;8*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□△☆</CENTER></TD><TD class="tg-yw4l">SynPro(S)-PIP;4*</TD><TD class="tg-yw4l">SynTF(A)-ZFHD1</TD><TD class="tg-yw4l">SynPro(A)-ZFHD1;4*</TD></TR><TR><TD class="tg-yw4l">SynPro(S)-PIP;8*</TD><TD class="tg-yw4l">SynTF(A)-21-16</TD><TD class="tg-yw4l">SynPro(A)-21-16;8*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□△</CENTER></TD><TD class="tg-yw4l">SynTF(S)-ZFHD1</TD><TD class="tg-yw4l">SynPro(S)-ZFHD1;4*</TD><TD class="tg-yw4l">SynTF(A)-42-10</TD><TD class="tg-yw4l">SynPro(A)-42-10;8*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□△</CENTER></TD><TD class="tg-yw4l">SynTF(S)-21-16</TD><TD class="tg-yw4l">SynPro(S)-21-16;8*</TD><TD class="tg-yw4l">SynTF(A)-43-8</TD><TD class="tg-yw4l">SynPro(A)-43-8;8*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□</CENTER></TD><TD class="tg-yw4l">SynTF(S)-42-10</TD><TD class="tg-yw4l">SynPro(S)-42-10;8*</TD><TD class="tg-yw4l">SynTF(A)-54-8</TD><TD class="tg-yw4l">SynPro(A)-54-8;8*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□△</CENTER></TD><TD rowspan="3" class="tg-yw4l">SynTF(S)-43-8</TD><TD class="tg-yw4l">SynPro(S)-43-8;2*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□△</CENTER></TD><TD class="tg-yw4l">SynPro(S)-43-8;4*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□△☆</CENTER></TD><TD class="tg-yw4l">SynPro(S)-43-8;8*</TD></TR><TR><TD class="tg-yw4l"><CENTER>□</CENTER></TD><TD class="tg-yw4l">SynTF(S)-54-8</TD><TD class="tg-yw4l">SynPro(S)-54-8;8*</TD></TR></TBODY></TABLE></DIV><DIV class="block main-text main-text"><P>
After co-transfected SynTFs, the mCherry expressions controlled by corresponding SynPros were significantly reduced. The functional SynPros’ silencing fold arranged from 6-23 (compared to cells without SynTFs) <A href="#figure5">(Figure 5A)</A>. We found that the RE repeats inserted after pSV40 3’ end can influence the basal expression of SynPros even without co-transfecting corresponding SynTFs. Above 7 SynTFs-SynPros we constructed and tested, 2 of them didn’t work. (1) It probably was due to the RE of SynPro(S)-42-10 couldn’t been recognized by SynZF-42-10 <A href="#figure1sr">(Figure S2A,B,C)</A>. Thus, the expression after co-transducing SynTF(S)-42-10 was higher than the solo-expression of some SynPros. (2) RE of SynPro(S)-54-8 might be antagonized by unknown proteins inside cells. Thus, the expression without or with SynTF were both low. We test our guess by co-transfecting different SynTFs with SynPro(S)42-10 and SynPro(S)-54-8 <A href="#figure1sr">(Figure S2B)</A>. 
</P><P>
The zinc finger is a highly structured motif. In tandem with multiple zinc finger motif, the recognition precision can be enhanced as its extensive usage in genome editing<SUP>(10)</SUP>. We applied this strategy to construct tunable SynTF-SynPro pairs. With more RE repeats, our SynTF has stronger inhibiting activity <A href="#figure5">(Figure 5B)</A>. 
</P><P>
To construct Stripe, two pairs of SynTF-SynPro are needed. The interaction between the pairs is not allowed. We did orthogonality experiments to check that. We confirmed that all of the 5 pairs were truly orthogonal <A href="#figure5">(Figure 5C)</A>, as you could see the grids on the diagonal were always the darkest. Three DBDs that are commonly used in previous works performed well in our hands. However, the expression level of the RE loaded SynPros were relatively low compared to SynPro(S)-ZF serials. As the blue rectangle in the lower right corner of the orthogonality may show the SynPro(S)-ZF has high basic expression with unpaired SynTFs, but could be silenced to the similar fold of commonly used DBDs corresponding SynPros. The SynPro(S)-ZF was likely won’t be a target by other unpaired DBD, hence the purple appeared on the bottom rows. 
</P><P>
In summary, we successfully constructed a SynTF-SynPro repertoire whose expression features could be finely tuned by either using different DBDs with REs or changing the number of RE repeats. Meanwhile, among all the orthogonal SynTF-SynPro pairs we constructed and test, our favorite ones were SynTF(S)-PIP-SynPro(S)-PIP;4* and SynTF(S)-ZF43-8-SynPro(S)-ZF43-8;8*. The combining results in pink rectangles show that they are both with low leakage, and the thresholds were far separated (PIP: -71 folds; ZF43-8: -8 folds) <A href="#figure5">(Figure 5C</A>, <A href="#figure1sr">S2D, E)</A>. So far, we have characterized and obtained two promising SynTF-SynPro pairs to use in the Stripe module. </P></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure 5. Wiring a repertoire of orthogonal and tunable silencing-form SynTF-SynPro pairs. </H>(A) SynPro(S)s’ Basic and silenced expression by cognate SynTF(S). The mCherry fluorescent intensity was normalized by basic expression of SynPro(S)-54-8;8*. Data are mean (SEM). (B) Tuning the silencing fold by adjusting the RE repeats of 2*, 4*, 8* on SynPro(S)-43-8. Values are the mean of n = 3 ± SEM. ****, p ＜ 0.0001. (C) Orthogonality of SynTF-SynPro pairs. Grids in blue rectangle showed that SynTF-SynPro pairs constructed by using SynZF as DBD with good orthogonality. Grids in pink rectangles replaced our favorite SynTF-SynPro pairs. At least 20,000 cells were analyzed for each condition in both histogram and each grid in the heat map. Data are recorded by FACS at 24h after co-transfecting.  
</P></DIV><DIV class="block main-text main-text"><H class="highlight-title dark-blue">Conclusion  </H><P>First, we have established a cell model with heterogeneous surEGFP expression and characterized the expression pattern of two HCC cell line (Huh-7, SMMC-7721) with different GPC3 expression level as our physiological model. Second, we engineered a functional SynNotch and proved they can generate response upon binding to their corresponding ligands. Third, we have mathematically modeled our transcription network in the Stripe model, and confirmed our design will generate tri-response with one heterogeneous antigen. Fourth, we used two approaches to wire SynTF-SynPro pair. We constructed, characterizes a set of pairs and obtain two optimal SynTF-SynPro pairs. We are still one step away from demonstrating the feasibility of SynNotch-Stripe experimentally, and we are doing it right now. We believe it is a promising platform for the next generation cellular immunotherapy. 
</P></DIV><DIV class="block main-text main-text"><H class="highlight-title dark-blue">Future work </H><P>
1.	Use SynTF(S)-PIP–SynPro(S)-PIP;4* and SynTF(S)-43-8–SynTF(S)-43-8;8* to construct a Stripe prototype and test its tri-response with EGFP coated beads.</P><P>
2.	Couple SynNotch module with Stripe module to construct a SwordS prototype and test its tri-response with different densities of surEGFP.</P><P>
3.	Test SwordS with HCC cells with different surface densities of GPC3. Currently, we have two cell lines, and our collaborators are actively searching for the third one.</P><P>
4.	Replace response product A and B with therapeutic factor I as IL-12, II as IL-18. Test its therapeutic effect on HCC cell lines. <FONT color="red"><BOLD>The patent is being prepared.</BOLD></FONT></P><P>
5. We have tested a set of SynTF(S)–SynPro(S) pairs and obtained good results. Similarly, there is a set of SynTF(A)–SynPro(A) pairs, in our initial design. But, we didn’t have time to test them for this year competition. Please note our Swords uses orthogonal SynTF(S)–SynPro(S) pairs.
</P></DIV><DIV class="block main-text notation " id="figure1sr"><H class="bold-text dark-blue fold-trigger3"><I>Read more: Supplementary Figures</I></H><DIV class="folded dark-grey"><DIV class="block main-text main-text"><H class="highlight-title dark-blue">Supplementary Figures 1</H></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure S1. Sorting stable cell lines depending on EGFP fluorescence. 
 </H>(A) FSC-SSC dual-detection was used to examine cell size and morphological complexity, and healthy cells with uniform size were gated. (B) FSC channel (trigger pulse width) was used to ensure cells at single cell status are gated to the next step.  (C) We set 3 gates (L, M, H) in mCherry-EGFP channel to sort cells with low-, medium-, high-expression of surEGFP. Those cells were all mCherry negative, and the mCherry signal was record only because the machine was set to record both channels.
</P></DIV><DIV class="block main-text main-text" id="figure2sr"><H class="highlight-title dark-blue">Supplementary Figures 2</H></DIV><DIV class="block main-text fig-notation silver-green"><P><H class="notation">Figure S2. Orthogonality analysis procedures of the SynTF-SynPro pairs, related to Figure 5.
</H>(A) FACS recorded EGFP and mCherry fluorescence intensities are presented in log scale. For this specific pair, SynTF(S)-42-10 cannot silence SynPro(S)-42-10;8*, because there was no difference between the distribution of mCherry fluoresence without or with SynTF. (B) Recorded fluorescence intensities are presented as in A. For this specific pair, SynTF(S)-PIP efficiently silence SynPro(S)-PIP;8*, because a great reduction of mCherry fluoresence was observed after introducing SynTF. (C) SynPro(S)-42-10;8* constantly drive high expression of mCherry no matter whatever SynTF(S) were cotransfected. Note the designed pair here is SynPro(S)-42-10;8* and SynTF(S)-42-10, and all other SynTF(S)s were set to be the negative controls for the experiments - but NOT. In another case, SynPro(S)-54-8;8* were constantly silenced by many SynTF(S)s as well. These two SynPro(S)s have very poor orthogonality. (D, E) The SynTF-SynPro pairs we test. (D) Fold unit in each row was normalized by diagonal grids to directly showed the silencing intensity, for comparison between pairs. (E) Original data of our orthogonality experiment. At least 20,000 cells were analyzed for each condition. Fluorescence intensities were recorded by FACS at 24h after transfecting cells with designed plasmids. Raw data were as shown in A and B, <I>Data Analysis see <A href="https://static.igem.org/mediawiki/2017/7/7f/T--Fudan--DataAnalysis.pdf">[here]</A></I>.
</P></DIV></DIV></DIV><DIV class="block main-text notation "><H class="bold-text dark-blue fold-trigger2"><I>Read more: Reference</I></H><DIV class="folded dark-grey"><P>
1.	K. Newick, S. O'Brien, E. Moon, S. M. Albelda, CAR T Cell Therapy for Solid Tumors. Annual Review of Medicine 68, 139--152 (2017).</P><P>
2.	L. Morsut et al., Engineering Customized Cell Sensing and Response Behaviors Using Synthetic Notch Receptors. Cell 164, 780--791 (2016).</P><P>
3.	J. Szulc, M. Wiznerowicz, M.-O. Sauvain, D. Trono, P. Aebischer, A versatile tool for conditional gene expression and knockdown. Nature methods 3, 109-116 (2006).</P><P>
4.	X. Chen, J. L. Zaro, W.-C. Shen, Fusion protein linkers: property, design and functionality. Advanced drug delivery reviews 65, 1357-1369 (2013).</P><P>
5.	R. Witzgall, E. O'Leary, A. Leaf, D. Onaldi, J. V. Bonventre, The Krüppel-associated box-A (KRAB-A) domain of zinc finger proteins mediates transcriptional repression. Proceedings of the National Academy of Sciences 91, 4514-4518 (1994).</P><P>
6.	R. R. Beerli, D. J. Segal, B. Dreier, C. F. Barbas, Toward controlling gene expression at will: specific regulation of the erbB-2/HER-2 promoter by using polydactyl zinc finger proteins constructed from modular building blocks. Proceedings of the National Academy of Sciences 95, 14628-14633 (1998).</P><P>
7.	M. Fussenegger et al., Streptogramin-based gene regulation systems for mammalian cells. Nature biotechnology 18, 1203--1208 (2000).</P><P>
8.	J. L. Pomerantz, P. A. Sharp, C. O. Pabo, Structure-based design of transcription factors. Science 267, 93 (1995).</P><P>
9.	J. J. Lohmueller, T. Z. Armel, P. A. Silver, A tunable zinc finger-based framework for Boolean logic computation in mammalian cells. Nucleic Acids Research 40, 5180--5187 (2012).</P><P>
10.	A. S. Khalil et al., A synthetic biology framework for programming eukaryotic transcription functions. Cell 150, 647--658 (2012).</P><P>
11.	A. Klug, The discovery of zinc fingers and their applications in gene regulation and genome manipulation. Annual review of biochemistry 79, 213--231 (2010).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>