<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hawaii_PCR_Amplification_of_pRL1383a"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Hawaii/PCR_Amplification_of_pRL1383a">Page               </A></LI><LI><A href="/Talk:Team:Hawaii/PCR_Amplification_of_pRL1383a">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/PCR_Amplification_of_pRL1383a&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/PCR_Amplification_of_pRL1383a&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hawaii/PCR_Amplification_of_pRL1383a" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hawaii/PCR Amplification of pRL1383a</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="background-color: white; border-spacing: 5px; text-align: center;" width="970"><TBODY><TR><TH style="width:240px; text-align:left; background-color:#86C43B; border-width:0px; padding:3px;"><DIV style="float:left;vertical-align:text-bottom;font-size:0.7em;line-height:1.4;"><A href="https://2008.igem.org/Team:Hawaii/Birds_Eye_View"> Birds Eye View</A></DIV><DIV style="position:absolute; left:110px;"><A href="/Team:Hawaii/Team" title="Team:Hawaii/Team">Team</A></DIV></TH><TH style="width:240px; text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Project" title="Team:Hawaii/Project">Projects</A></TH><TH style="width:240px; text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Event" title="Team:Hawaii/Event">Events</A></TH><TH style="width:240px; text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Resources" title="Team:Hawaii/Resources">Resources</A></TH></TR><TR><TD style="text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Sponsors" title="Team:Hawaii/Sponsors">Sponsors</A></TD><TD style="text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Experiment" title="Team:Hawaii/Experiment">Experiments</A></TD><TD style="text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Milestone" title="Team:Hawaii/Milestone">Milestones</A></TD><TD style="text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Protocols" title="Team:Hawaii/Protocols">Protocols</A></TD></TR><TR><TD style="text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Notebook" title="Team:Hawaii/Notebook">Notebook</A> (<A href="/Template:Team:Hawaii/Notebook" title="Template:Team:Hawaii/Notebook">t</A>)
</TD><TD style="text-align:center; background-color:#86C43B; border-width:0px; padding:3px;"><A href="/Team:Hawaii/Meeting" title="Team:Hawaii/Meeting">Meetings</A> (<A href="/Template:Team:Hawaii/Meeting" title="Template:Team:Hawaii/Meeting">t</A>)
</TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#PCR_Amplification_of_Parts_to_be_used_in_the_Broad-Host-Range_Vector"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">PCR Amplification of Parts to be used in the Broad-Host-Range Vector</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Methods"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Results"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Discussion"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Follow-up_Experiments"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Follow-up Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#7.2F2.2F08"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">7/2/08</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#7.2F4.2F08"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">7/4/08</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#7.2F7.2F08"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">7/7/08</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#7.2F18.2F08"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">7/18/08</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#7.2F19.2F08"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">7/19/08</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#7.2F23"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">7/23</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#7.2F29.2F08"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">7/29/08</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#8.2F4"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">8/4</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#8.2F5"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">8/5</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#8.2F6"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">8/6</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#8.2F7"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">8/7</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Completed_PCR_Products"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Completed PCR Products</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Discussion_2"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="PCR_Amplification_of_Parts_to_be_used_in_the_Broad-Host-Range_Vector"> PCR Amplification of Parts to be used in the Broad-Host-Range Vector</SPAN></H2><UL><LI> Several regions of pRL1383a will be amplified with BioBrick based primers. These components will be used later in the construction of a pRL1383a BioBrick based vector. These parts include the <I>aadA</I> region from pRL1383a and from the BioBrick Registry, the origin of vegetative replication (oriV), and the replication proteins.
</LI></UL><H3><SPAN class="mw-headline" id="Methods"> Methods </SPAN></H3><P><STRONG>Materials Per Reaction</STRONG>
Added in order:
</P><UL><LI> 3.5uL nanopure water
</LI><LI> 1uL 10uM forward/reverse primers 
</LI><LI> 1uL pRL1383a as template (1:10 dilution of <A href="/Team:Hawaii/Large-Scale_Preparation_of_Plasmid_from_E._coli" title="Team:Hawaii/Large-Scale Preparation of Plasmid from E. coli">large-scale plasmid prep</A>). 
</LI><LI> 5uL Taq polymerase: Accusure(hot start, 68°C) and Red Taq
</LI></UL><P>Note: the taq and water can be combined and aliquoted together as long as reaction is kept at 4°C.
</P><UL><LI> cold blocks
</LI><LI> pcr reaction tubes, with tops
</LI><LI> pipette and tips
</LI></UL><TABLE border="1"><CAPTION><B><STRONG>PCR Running Conditions</STRONG></B></CAPTION><TBODY><TR><TH width="150">Duration of Time
</TH><TH width="150">T° 5 cycles
</TH><TH width="150">T° 30 cycles
</TH><TH width="150">Purpose
</TH></TR><TR><TD>10 minutes
</TD><TD>95°C
</TD><TD>95°C
</TD><TD>heat activate taq
</TD></TR><TR><TD>30 seconds
</TD><TD>95°C
</TD><TD>95°C
</TD><TD>denaturation
</TD></TR><TR><TD>30 seconds
</TD><TD>48.1°C, 53.4°C, 57.9°C
</TD><TD>59°C, 60.6°C, 61.9°C
</TD><TD>annealing
</TD></TR><TR><TD>5 minutes
</TD><TD>68°C
</TD><TD>68°C
</TD><TD>extension (1.5 minutes per kb, go with longest)
</TD></TR><TR><TD>10 minutes
</TD><TD>68°C
</TD><TD>68°C
</TD><TD>finishing the extension
</TD></TR><TR><TD>infinity
</TD><TD>4°C
</TD><TD>4°C
</TD><TD>(until you remove product)
</TD></TR></TBODY></TABLE><P><STRONG> Materials for the gel </STRONG></P><UL><LI>1% agarose gel
</LI><LI>10ug/mL Ethidium Bromide
</LI><LI>running apparatus
</LI><LI>gel running conditions: 95V for ~1 hour
</LI></UL><P><STRONG> PCR Clean-up using Qiagen Gel Extraction Kit </STRONG></P><DL><DD><UL><LI>Use after running products on a gel
</LI></UL></DD></DL><OL><LI>add 3 volumes of QG to 1 volume of gel
</LI><LI>incubate 50°C for 10 minutes, until gel dissolves; spin every few minutes
</LI><LI>add 1 gel volume of isopropanol, mix by inverting
</LI><LI>place sample in minelute spin column
</LI><LI>centrifuge 1 minute at 13k rpm
</LI><LI>discard flow-through
</LI><LI>add 750uL PE buffer to wash; centrifuge 1 minute 13k rpm
</LI><LI>discard flow-through; centrifuge 1 min 13k rpm
</LI><LI>place column in clean 1.5 mL tube
</LI><LI>add 10ul TE or water to center of column, wait 1 minute, centrifuge 1 min
</LI></OL><H3><SPAN class="mw-headline" id="Results"> Results </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR Gel 1 (left) and PCR Gel 2 (right).</DIV></DIV></DIV><TABLE border="1"><CAPTION><B>PCR Gel 1</B></CAPTION><TBODY><TR><TH width="100">Lane
</TH><TH width="100">Contents
</TH><TH width="100">Description
</TH><TH width="100">Predicted annealingT°C,5 cycles
</TH><TH width="100">Predicted annealing T°C,30 cycles
</TH></TR><TR><TD>1
</TD><TD>Tri-dye 10kb ladder
</TD><TD>ok
</TD></TR><TR><TD>2
</TD><TD>omega (aadA)#1
</TD><TD>(+), ~1kb
</TD><TD>54.4,50.4
</TD><TD>60.7,69.5
</TD></TR><TR><TD>3
</TD><TD>oriV #1
</TD><TD>(+), ~0.5kb
</TD><TD>51.3, 53
</TD><TD>60.9, 71.2
</TD></TR><TR><TD>4
</TD><TD>mob #1
</TD><TD>smear
</TD><TD>53.5, 48
</TD><TD>62, 69.8
</TD></TR><TR><TD>5
</TD><TD>rep #1
</TD><TD>smear, low kb
</TD><TD>53.9, 57.2
</TD><TD>59.1, 73.5
</TD></TR><TR><TD>6
</TD><TD>(-)control
</TD><TD>all clear
</TD></TR><TR><TD>7
</TD><TD>(+) control
</TD><TD>bright band, ~0.5kb
</TD></TR><TR><TD>8
</TD><TD>omega #2
</TD><TD>(+), ~1kb
</TD><TD>54.4,50.4
</TD><TD>60.7,69.5
</TD></TR><TR><TD>9
</TD><TD>oriV #2
</TD><TD>(+) ~0.5kb
</TD><TD>51.3, 53
</TD><TD>60.9, 71.2
</TD></TR><TR><TD>10
</TD><TD>mob #2
</TD><TD>nothing, slight smear
</TD><TD>53.5, 48
</TD><TD>62, 69.8
</TD></TR></TBODY></TABLE><TABLE border="1"><CAPTION><B>PCR Gel 2</B></CAPTION><TBODY><TR><TH width="100">Lane
</TH><TH width="100">Contents
</TH><TH width="100">Description
</TH><TH width="100">Predicted annealing T°C,5 cycles
</TH><TH width="100">Predicted annealing T°C,30 cycles
</TH></TR><TR><TD>1
</TD><TD>Tri-dye 10kb ladder
</TD><TD>no ladder
</TD></TR><TR><TD>2
</TD><TD>rep#2
</TD><TD>light smear
</TD><TD>53.9, 57.2
</TD><TD>59.1, 73.5
</TD></TR><TR><TD>3
</TD><TD>(-)control
</TD><TD>all clear
</TD></TR><TR><TD>4
</TD><TD>(+)control
</TD><TD>bright band ?0.5kb
</TD></TR><TR><TD>5
</TD><TD>omega #3
</TD><TD>bright band, ?1kb
</TD><TD>54.4,50.4
</TD><TD>60.7,69.5
</TD></TR><TR><TD>6
</TD><TD>oriV #3
</TD><TD>bright band, 0.5kb?
</TD><TD>51.3, 53
</TD><TD>60.9, 71.2
</TD></TR><TR><TD>7
</TD><TD>mob
</TD><TD>light band, some smear, 3kb?
</TD><TD>53.5, 48
</TD><TD>62, 69.8
</TD></TR><TR><TD>8
</TD><TD>rep #3
</TD><TD>band in low kb
</TD><TD>53.9, 57.2
</TD><TD>59.1, 73.5
</TD></TR><TR><TD>9
</TD><TD>(-) control
</TD><TD>all clear
</TD></TR><TR><TD>10
</TD><TD>(+) control
</TD><TD>bright band, 0.5kb?
</TD></TR></TBODY></TABLE><P><STRONG>PCR Clean-up</STRONG></P><P>MinElute PCR Purification Kit Protocol from Qiagen
using a microcentrifuge
</P><P>This protocol is designed to purify double-stranded DNA fragments from PCR reactions
resulting in high end-concentrations of DNA (see page 12). Fragments ranging from
70 bp to 4 kb are purified from primers, nucleotides, polymerases, and salts using
MinElute spin columns in a microcentrifuge.
</P><OL><LI> Mix PB 5:1 PCR reaction.
</LI><LI> Place a MinElute column in a provided 2 ml collection tube.
</LI><LI> Apply ALL of sample to the MinElute column; centrifuge (13,000rpm)1 min.
</LI><LI> Discard flow-through. Place the MinElute column back into the same tube.
</LI><LI> Add 750 μl Buffer PE to the MinElute column; centrifuge (13,000rpm) 1 min.
</LI><LI>Discard flow-through and place the MinElute column back in the same tube.
</LI></OL><P>Centrifuge the column for an additional 1 min at maximum speed.
</P><OL><LI>Place the MinElute column in a clean 1.5 ml microcentrifuge tube.
</LI><LI>Add 10 μl 1x TE to the center of the membrane, let the column stand for 1 min; centrifuge 1 min.
</LI></OL><H3><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H3><UL><LI>The gels are very warped. I did not let them dry long enough before I poured the running buffer,so some products did not run correctly therefore this experiment will be repeated on Monday when the necessary reagents are delivered.
</LI><LI>I mixed up the temperature gradient!!! I need to be more careful about labeling my reactions.
</LI><LI>The rep protein did not come out at all and the mob region only came out at one temperature, so these must be PCRed again to determine the correct annealing temperature.
</LI></UL><H2><SPAN class="mw-headline" id="Follow-up_Experiments"> Follow-up Experiments </SPAN></H2><H3><SPAN class="mw-headline" id="7.2F2.2F08"> 7/2/08 </SPAN></H3><UL><LI>A second attempt was made to PCR the rep region. An annealing temperature of 53.4° was used for the first 5 cycles and annealing temperature 60.6 was used for the subsequent 30 cycles. 
</LI><LI>Results: There are 2 faint bands in the low kb region. This is probably primers. 
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1 contains the ladder, lanes 3 through 6 contain the products of a plasmid prep,  <STRONG>lane 7 contains the PCR of the rep region</STRONG>, while lanes 8 and 9 are the positive and negative control, respectively. Lane 10 contains one of NW's PCR reactions.</DIV></DIV></DIV><UL><LI>Discussion: The rep region did not amplify under these conditions. There were several <A href="/Team:Hawaii/Meeting/2008-07-_3" title="Team:Hawaii/Meeting/2008-07- 3">suggestions</A> made by the professors that I will try this weekend. 
</LI></UL><H3><SPAN class="mw-headline" id="7.2F4.2F08"> 7/4/08 </SPAN></H3><UL><LI>Materials:
</LI></UL><OL><LI> 5uL Red Taq (per reaction)
</LI><LI> 3.5uL nanopure water (per reaction)
</LI><LI> 1uL 10uM f/r <I>rep</I> region primers (per reaction)
</LI><LI> 1uL of either 1:100, 1:500, 1:1000 dilution pRL1383a as template (1:10 dilution of <A href="/Team:Hawaii/Large-Scale_Preparation_of_Plasmid_from_E._coli" title="Team:Hawaii/Large-Scale Preparation of Plasmid from E. coli">large-scale plasmid prep</A>)
</LI><LI> 5uL accusure (for one reaction to test the dilution with this enzyme)
</LI></OL><TABLE border="1"><CAPTION><B><STRONG>PCR Running Conditions</STRONG></B></CAPTION><TBODY><TR><TH width="150">Duration of Time
</TH><TH width="150">T° 5 cycles
</TH><TH width="150">T° 30 cycles
</TH><TH width="400">Purpose
</TH></TR><TR><TD>10 minutes
</TD><TD>95°C
</TD><TD>95°C
</TD><TD>heat activate taq
</TD></TR><TR><TD>30 seconds
</TD><TD>95°C
</TD><TD>95°C
</TD><TD>denaturation
</TD></TR><TR><TD>30 seconds
</TD><TD>54
</TD><TD>59
</TD><TD>annealing
</TD></TR><TR><TD>5 minutes
</TD><TD>70°C (red taq),68°C (accusure)
</TD><TD>68°C (red taq),68°C (accusure)
</TD><TD>extension (1.5 minutes per kb, go with longest)
</TD></TR><TR><TD>10 minutes
</TD><TD>70°C (red taq),68°C (accusure)
</TD><TD>68°C (red taq),68°C (accusure)
</TD><TD>finishing the extension
</TD></TR><TR><TD>infinity
</TD><TD>4°C
</TD><TD>4°C
</TD><TD>(until you remove product)
</TD></TR></TBODY></TABLE><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR Gel of <I>rep</I> region amplified with red taq.</DIV></DIV></DIV><P><STRONG> Results </STRONG></P><TABLE border="1"><CAPTION><B>PCR Gel 1</B></CAPTION><TBODY><TR><TH width="100">Lane
</TH><TH width="100">Contents
</TH><TH width="400">Description
</TH></TR><TR><TD>1
</TD><TD>Tri-dye 10kb ladder
</TD><TD>faint, not well defined
</TD></TR><TR><TD>2
</TD><TD><I>rep</I> 1:100 dilution
</TD><TD>3.3kb, brighter than ladder, very thick band, smear (too much DNA?)
</TD></TR><TR><TD>3
</TD><TD><I>rep</I> 1:500 dilution
</TD><TD>3.3kb, brighter than ladder, very thick band, smear (too much DNA?)
</TD></TR><TR><TD>4
</TD><TD><I>rep</I> 1:1000 dilution
</TD><TD>3.3kb, brighter than ladder, very thick band, smear (too much DNA?)
</TD></TR><TR><TD>5
</TD><TD>(-)control
</TD><TD>very faint band in low kb region (maybe contamination from other well
</TD></TR><TR><TD>6
</TD><TD>(+) control
</TD><TD>faint band in 0.5 kb region
</TD></TR><TR><TD>7
</TD><TD>1:500 dilution <I>rep</I> region amplified by accusure
</TD><TD>0.5kb band and band in low kb region, faint
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="7.2F7.2F08"> 7/7/08 </SPAN></H3><OL><LI><DL><DD><I>rep</I> and <I>mob</I> regions did not amplify with accusure, so they were amplified using red/green taq. Same conditions from 7/4 were used. 
</DD></DL></LI></OL><P><STRONG> results </STRONG></P><OL><LI><DL><DD><I>mob</I> and <I>rep</I> regions were amplified by both red and green taq. Use green in the future, it is cheaper.
</DD></DL></LI></OL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR amplification of <I>mob</I> and <I>rep</I> regions.</DIV></DIV></DIV><P><STRONG> Discussion </STRONG></P><P><I>rep</I> region appears to be amplified by red taq efficiently. Need to use much less DNA. Find out what dilution to use. Accusure + some reaction stabilizing reagent might be used to get this reaction going, but as of right now, this region is not being amplified by this enzyme.
This temperature seems to have worked well also.
</P><H3><SPAN class="mw-headline" id="7.2F18.2F08"> 7/18/08 </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR amplification of the mob/rep region, the omega interposon, the aada gene, oriV, a (-) control, and (+)control. All products that amplified are too bright to use the ladder scale.</DIV></DIV></DIV><DL><DD><UL><LI>PCR amplification of oriV, omega interposon, aadA gene, &amp; rep/mob all using green taq. 
</LI><LI> PCR conditions: 2min@94°C, 30sec@95°C, 30sec@50.4°C, 5min30sec@72°C(1 minute per kilobase), infinity@4°C
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="7.2F19.2F08"> 7/19/08 </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Lane 1 and 12 is the tridye 2-log ladder, 2,3,4, &amp;5 are the mob/rep region, the omega interposon, the negative and positive control, respectively amplified with Green Taq. Lanes 6 and 7 are the mob/rep and omega amplified with accuzyme. Lanes 8,9,10,&amp;11 are the mob/rep region, the omega interposon, the negative and positive control, respectively amplified with velocity taq.</DIV></DIV></DIV><DL><DD><UL><LI>PCR amplification of omega interposon and rep/mob regions using 3 different types of taq: <DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Velocity_taq.jpg" title="File:Velocity taq.jpg">File:Velocity taq.jpg</A><DIV class="thumbcaption">PCR amplification with velocity taq, green taq and accuzyme with the hopes that one will amplify these products. I also changed the annealing temperature to 48°C in order to accommodate the primer with the lowest annealing temperature.</DIV></DIV></DIV></LI></UL></DD></DL><P><STRONG>Methods</STRONG>
For all reactions, I heated up the PCR block to 94C before adding my sample. I did this by running another method which only involves 94C for infinity.
</P><DL><DD><UL><LI>Accuzyme:
</LI></UL></DD><DD><UL><LI>Reaction conditions: 3.5ul H<SUB>2</SUB>, 1ul Primers, 0.5ul template, 5ul taq
</LI><LI>Running conditions: 95C 10min, 95C 30sec, 48C 30sec, 68C 10 min, 68C 10min, 4C infinity.
</LI></UL></DD></DL><UL><LI>Green Taq: 
</LI></UL><DL><DD><UL><LI>Reaction conditions: 3.5ul H<SUB>2</SUB>, 1ul Primers, 0.5ul template, 5ul taq
</LI><LI>Running conditions: 95C 10min, 95C 30sec, 48C 30sec, 68C 10 min, 68C 10min, 4C infinity.
</LI></UL></DD></DL><UL><LI>Velocity DNA Polymerase:
</LI></UL><DL><DD><UL><LI>Reaction conditions: 2ul 5X Buffer, 0.1ul dNTPs, 5.65ul H<SUB>2</SUB>O, 1ul Primers, 0.5ul template, 0.25ul taq(0.5U)
</LI><LI>Running conditions: 98C 2min, 98C 30sec, 48C 30sec, 72C 2min34sec, 72C 10min, 4C infinity.
</LI></UL></DD></DL><P><STRONG>Results</STRONG></P><DL><DD><UL><LI>As seen in the very small picture, the only amplification product of this reaction is of the positive control amplified by green taq.
</LI><LI>The omega interposon template (pSMC121) was run on a gel alongside the products of <A href="/Team:Hawaii/Notebook/2008-07-21" title="Team:Hawaii/Notebook/2008-07-21">the colony PCR from 7_21</A>. There was no band meaning that there was no template for this PCR reaction. 
</LI></UL></DD></DL><P><STRONG>Discussion</STRONG></P><DL><DD><UL><LI> Need to get in touch with Dr. Callahan and get more template. Also, just try again with green taq only... problem solve.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="7.2F23"> 7/23 </SPAN></H3><DL><DD><UL><LI>PCR amplification of the omega interposon region of pSMC121 and the mob/rep region of pRL1383a using green taq.
</LI><LI>Reaction conditions: same as above
</LI><LI>Running conditions: same as above for green taq.
</LI></UL></DD></DL><P><STRONG> Results </STRONG></P><DL><DD><UL><LI> PCR amplification of mob/rep region and omega interposon. Lane 1: 2-log tridye ladder, lane 2:omega interposon, lane 3: mob/rep region, lane 4: negative control, lane 5: positive contol.
</LI></UL></DD></DL><DL><DD><UL><LI> The omega interposon amplification yielded only primer dimers. The mob/rep region is a smear, there is a lot of mis-priming. The positive and negative controls yield a bright band and no band respectively.
</LI></UL></DD></DL><P><STRONG> Discussion </STRONG></P><DL><DD><UL><LI>the omega interposon primers are likely to form dimers  because there are 4 overlapping nucleotides at the 5' and 3' ends with a Gibbs free energy of -6.61 kcal/mole. I can alleviate this problem by finding a higher annealing temperature than 48°C. The annealing temperature is 49.5°, so I can use a gradient to find a good temperature. I will try from 48deg&amp;;C to 55°C.
</LI></UL></DD></DL><DL><DD><UL><LI> The above stated method for solving the primer dimer problem can also be a possible solution for the mis-priming of the mob/rep region.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="7.2F29.2F08">7/29/08</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR amplification of oriV, aada, rep.</DIV></DIV></DIV><DL><DD><UL><LI>Amplification of  <A href="http://partsregistry.org/Part:BBa_J23012" class="external text" rel="nofollow">BBa_J23012</A> which is aadA, aadA from pRL1383a, and the rep region. I used accusure for all and also green taq for rep. 
</LI></UL></DD></DL><P><STRONG>Materials</STRONG></P><P>Accusure Master-Mix (for 20 50ul reactions), stored in 4°C:
</P><UL><LI>10x buffer: 100ul
</LI><LI>100uM dNTPs: 10ul
</LI><LI>Accusure enzyme: 20ul
</LI><LI>nano-pure-water: 38.5ul
</LI><LI> for each reaction add: 4ul f&amp;r primer, and 1ul template
</LI></UL><P><STRONG>Methods</STRONG></P><DL><DD><UL><LI>reaction conditions: 
</LI></UL></DD><DD><UL><LI>accusure reaction: 45ul master mix, 4ul f&amp;r primer, and 1ul template
</LI><LI>green taq reaction: 20ul water, 25ul green taq, 4ul f&amp;r primer, 1ul template
</LI></UL></DD></DL><DL><DD><UL><LI>running conditions:
</LI></UL></DD></DL><DL><DD><UL><LI>accusure reaction: annealing temp: 50.4°C, 5minutes extension time at 68°C
</LI><LI>green taq reaction: 53.9°C, 5minutes extension time at 72°C
</LI></UL></DD></DL><P><STRONG>Results</STRONG></P><TABLE border="1"><CAPTION><B>PCR Products</B></CAPTION><TBODY><TR><TH width="100">lane
</TH><TH width="100">name
</TH><TH width="100">quantity
</TH><TH width="100">size
</TH><TH width="400">comments
</TH></TR><TR><TD>2
</TD><TD>aadA from  <A href="http://partsregistry.org/Part:BBa_J23012" class="external text" rel="nofollow">BBa_J23012</A></TD><TD>3.2 ng/ul
</TD><TD>0.85kb
</TD><TD>the best aadA band
</TD></TR><TR><TD>3
</TD><TD>aadA from pRL1383a
</TD><TD>1.6ng/ul
</TD><TD>0.8 kb
</TD><TD>smaller than the BB aadA
</TD></TR><TR><TD>4
</TD><TD>rep region from pRL1383a
</TD><TD>24ng/ul
</TD><TD>3.5kb
</TD><TD>brighter than any band, hard to quantify. Trail of DNA on both sides of the band &amp; bright band @ 100bp region.
</TD></TR><TR><TD>5
</TD><TD>oriV
</TD><TD>nothing
</TD><TD>nothing
</TD><TD>I will amplify with green taq.
</TD></TR><TR><TD>6
</TD><TD>rep with accusrue
</TD><TD>nothing
</TD><TD>nothing
</TD><TD>will use the green taq version
</TD></TR><TR><TD>7
</TD><TD>negative control
</TD><TD>1ng/ul
</TD><TD>0.6kb
</TD><TD>The water used for the master mix is contaminated
</TD></TR><TR><TD>8
</TD><TD>positive control
</TD><TD>light band
</TD><TD>0.5 kb
</TD><TD>This did not amplify well
</TD></TR></TBODY></TABLE><P><STRONG> Discussion </STRONG></P><DL><DD><UL><LI>I was able to amplify some of the parts, though they are not as bright and/or distinct as I may need them for downstream work. The master-mix appears to be contaminated because UPA amplified something in the 0.6kb region when there should have been no band. 
</LI></UL></DD></DL><DL><DD><UL><LI>I will try to amplify oriV with green taq.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="8.2F4">8/4</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR amplification of rep and oriV, plus verification of oriT and P<SUB>lac</SUB></DIV></DIV></DIV><DL><DD><STRONG>Margaret</STRONG></DD></DL><DL><DD><UL><LI> Rep, oriV, verification of I14032, oriT(just to make sure)
</LI></UL></DD></DL><P><STRONG> Materials &amp; Methods </STRONG></P><DL><DD><UL><LI>Reaction Conditions: 3.5ul water, 0.5ul template, 1.0ul f&amp;r primer, 5ul econotaq
</LI><LI>Running conditions (I14032, oriT): 94°C hold, 94°C 2', (95°C 30&quot;, 62°C 30&quot;,  72°C 30&quot;) x 30cycles, 72°C 10', 4°C hold 
</LI><LI>Running conditions (oriV + controls): 94°C hold, 94°C 2', (95°C 30&quot;, 51.3°C 30&quot;,  72°C 30&quot;) x 30cycles, 72°C 10', 4°C hold 
</LI><LI>Running conditions (rep + controls): 94°C hold, 94°C 2', (95°C 30&quot;, 53.9°C 30&quot;,  72°C 3'20&quot;) x 20 cycles, (95°C 30&quot;, 53.9°C 30&quot;,  72°C 3'30&quot;) x 10 cycles72°C 10', 4°C hold 
</LI></UL></DD></DL><P><STRONG> Results </STRONG></P><P><STRONG>Lane 2:</STRONG> PCR verification of I14032: There are two bands. The top 
band does not correspond to Plac, while the bottom band is just above 
0.4kb, about 50bp more than would be expected.
<STRONG>Lane 3:</STRONG> PCR verification of OriT: The band is just above 0.4kb, and the
verification part is 0.415 kb.
<STRONG>Lane 4:</STRONG> Amplification of oriV from pRL1383a:The band is at 4.5kb, as 
expected.
<STRONG>Lane 5&amp;8:</STRONG> pSB1A7 was used as a (+) control: While the same size was amplified
in both positive control reactions, I would expect a band at 1265bp, but
the amplicon is just below 0.9kb.
<STRONG>Lane 6&amp;9:</STRONG> VF2 and VR were used to amplify water. As expected, nothing was amplified.
<STRONG>Lane 7:</STRONG>The rep region was amplified correctly. I expected a band at 3.352kb and the band 
occurred in this region.
</P><P><STRONG> Discussion </STRONG></P><DL><DD><UL><LI>I did some trouble shooting on this PCR with advice from: [<A href="http://www.med.yale.edu/genetics/ward/tavi/Trblesht.html" class="external autonumber" rel="nofollow">[1]</A>]. 
</LI><LI>In most cases, I just got as close to the recommended annealing temperature and extension time in order to decrease the amount of unspecific products. 
</LI><LI>I only ran 30 cycles as opposed to 35 in the past. I think this has made the efficiency of my taq more consistent, especially with the longer products. 
</LI><LI>I also added an additional 10 seconds to last 20 cycles of the rep region, [<A href="http://www.med.yale.edu/genetics/ward/tavi/Trblesht.html" class="external autonumber" rel="nofollow">[2]</A>], recommends this extra time because the taq supposedly loses efficiency and thus there will be multiple products of suspicious size. This seems to have worked for the rep region, which contains only one product.
</LI><LI><STRONG> Next Step </STRONG> is to amplify both rep and oriV to large quantities using their respective conditions.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="8.2F5">8/5</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">PCR amplification of P1 lytic region, aadA from pRL1383a, and aadA from BioBrick Registry. Test.</DIV></DIV></DIV><DL><DD><STRONG> Margaret</STRONG></DD></DL><DL><DD><UL><LI>testing PCR conditions for P1 lytic region and aadA (registry and pRL1383a)
</LI></UL></DD></DL><P><STRONG> Materials &amp; Methods </STRONG></P><DL><DD><UL><LI>Reaction conditions: 3.5ul water, 5ul taq, 0.5ul template (P1 lytic: pSB2K3(Plac), (+)&amp;(-):pSB2K3(Plac): , aadA: pRL1383a, (+)&amp;(-):pSB1A7, aadA: BioBrick Registry), 1ul primer (P1 lytic: p1lytic f&amp;r, (+)&amp;(-):VF2&amp;VR , aadA: aadA f&amp;r, (+)&amp;(-):VF2&amp;VR, aadA: aadA f&amp;r)
</LI></UL></DD></DL><DL><DD><UL><LI>Running Conditions:
</LI></UL></DD><DD><UL><LI>p1 lytic region: 94°C hold, 94°C 2', 95°C 30&quot;, <STRONG>54.2°C 30&quot;, 72°C 1'10&quot;</STRONG>, 72°C 10', 4°C hold.
</LI><LI>aadA (pRL1383a &amp; BB registry): 94°C hold, 94°C 2', 95°C 30&quot;, <STRONG>50°C 30&quot;, 72°C 30&quot;</STRONG>, 72°C 10', 4°C hold.
</LI></UL></DD></DL><P><STRONG>Results</STRONG></P><DL><DD><UL><LI><STRONG>Lane 1</STRONG> contains the tri-dye 2-log ladder, which is not well resolved. K&amp;G had better resolution of the ladder at lower voltage. 
</LI><LI><STRONG>Lane 2</STRONG> contains the amplification of the P1 lytic region from the I14032 prep which is on pSB2K3. This part did not amplify. The positive and negative controls in 
</LI><LI><STRONG>Lanes 3&amp;4</STRONG> are actually the same thing: pSB2K3 (containing I14032) and VF2+VR. I mistakenly put Primer and Template in the negative. I do get amplification, though, which tells me the plasmid is there. The band should be around 353bp but looks to be above 400bp. This discrepancy is found in all amplifications of this part (which I have verified twice before: <A href="/Team:Hawaii/Notebook/2008-08-_4" title="Team:Hawaii/Notebook/2008-08- 4">here</A> and <A href="/Team:Hawaii/Notebook/2008-07-30" title="Team:Hawaii/Notebook/2008-07-30">here</A>. K&amp;G also amplified it directly from the BB registry and got the same band. :* <STRONG>Lane 5</STRONG> aadA from pRL1383a was amplified and large amounts of the right size are visualized on the gel. 
</LI><LI><STRONG>Lanes 6&amp;7</STRONG> are the positive and negative control but this time VF2 and VR are amplifying pSB1A7 (proven on several occasions to amplify consistently; like <A href="/Team:Hawaii/Notebook/2008-08-_4" title="Team:Hawaii/Notebook/2008-08- 4">here</A>). The same mistake with the other controls in this experiment happended:(. 
</LI><LI><STRONG>Lane 8</STRONG> contains the amplification of aadA from the BioBrick Registry. It has the right size and large amounts.
</LI></UL></DD></DL><P><STRONG> Conclusions </STRONG></P><DL><DD><UL><LI> The aadA region amplified well under these conditons (for both the pRL1383a &amp; BB Registry amplifications). The next step is to amplify large quantities. 
</LI><LI> The P1 lytic region did not amplify. The plasmid is there, though, because the (+) control (pSB2K3(Plac)) amplified a band of the correct size. I will re-evaluate the running conditions, and also extract the pSB2K3 plasmid from the registry and use it to amplify with the P1 lytic primers. 
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="8.2F6">8/6</SPAN></H3><DL><DD><UL><LI> Large-scale PCR reactions for rep, oriV, and aadA (from pRL1383a and BBregistry)
</LI></UL></DD></DL><P><STRONG>Materials &amp; Methods</STRONG></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Large-scale PCR of rep, aadA (both), and oriV.</DIV></DIV></DIV><DL><DD><UL><LI>Used reaction and running conditions specified in the reactions from 8/5 and 8/4. 
</LI></UL></DD></DL><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Test PCR for omega and P1 lytic.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Mini-prep of pSMC121 used for PCR of omega region.</DIV></DIV></DIV><DL><DD><UL><LI> Try out conditions for P1 lytic and omega.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="8.2F7">8/7</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Large scale PCR for amplification of P1 lytic and aadA region.</DIV></DIV></DIV><P><STRONG>Margaret</STRONG></P><DL><DD><UL><LI>Large-scale PCR amplification of  aadA (it was lost in a gel cutting mishap) and P1 lytic. 
</LI><LI>Reaction conditions: 3 50ul green taq reactions for each.
</LI><LI>Running conditions: see below
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Completed_PCR_Products"> Completed PCR Products </SPAN></H2><TABLE border="1"><CAPTION><B>PCR Products</B></CAPTION><TBODY><TR><TH width="100">Name
</TH><TH width="100">description
</TH><TH width="200">reaction conditions
</TH></TR><TR><TD>OriV
</TD><TD>origin of vegetative replication
</TD><TD>94°C hold, 94°C 2', (95°C 30&quot;, 51.3°C 30&quot;, 72°C 30&quot;) 30 cycles, 72°C 10', 4°C hold
</TD></TR><TR><TD>rep region
</TD><TD>proteins necessary for autonomous replication
</TD><TD>94°C hold, 94°C 2', (95°C 30&quot;, 53.9°C 3'20&quot;, 72°C 30&quot;) 10 cycles, (95°C 30&quot;, 53.9°C 30&quot;, 72°C 3'30&quot;) 20 cycles , 72°C 10', 4°C hold
</TD></TR><TR><TD>aadA (pRL1383a &amp; BB registry)
</TD><TD>Sm, Sp resistance
</TD><TD>94°C hold, 94°C 2', (95°C 30&quot;, 50°C 30&quot;, 72°C 48&quot;) 30 cycles, 72°C 10', 4°C hold
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Discussion_2">Discussion</SPAN></H2><DL><DD><UL><LI>Using the reaction conditions in the table above and green taq as the polymerase, all products were successfully amplified first in a test run, then on a large scale. In the next steps the products will be digested with restriction enzymes and ligated to their respective vectors. The experiments for this can be found <A href="/Team:Hawaii/Ligation_of_pRL1383a_Parts" title="Team:Hawaii/Ligation of pRL1383a Parts">here</A>.
</LI></UL></DD></DL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Hawaii/PCR_Amplification_of_pRL1383a">http://2008.igem.org/Team:Hawaii/PCR_Amplification_of_pRL1383a</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hawaii/PCR_Amplification_of_pRL1383a" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hawaii/PCR_Amplification_of_pRL1383a" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hawaii/PCR_Amplification_of_pRL1383a&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hawaii/PCR_Amplification_of_pRL1383a&amp;oldid=99267" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>