"
Page
Discussion
View source
History
teams
Log in
 
Team:Hawaii/PCR Amplification of pRL1383a
From 2008.igem.org
 Birds Eye
 View
Team
Projects
Events
Resources
Sponsors
Experiments
Milestones
Protocols
Notebook ( t )
Meetings ( t )
Contents
1 PCR Amplification of Parts to be used in the Broad-Host-Range Vector
1.1 Methods
1.2 Results
1.3 Discussion
2 Follow-up Experiments
2.1 7/2/08
2.2 7/4/08
2.3 7/7/08
2.4 7/18/08
2.5 7/19/08
2.6 7/23
2.7 7/29/08
2.8 8/4
2.9 8/5
2.10 8/6
2.11 8/7
3 Completed PCR Products
4 Discussion
PCR Amplification of Parts to be used in the Broad-Host-Range Vector
Several regions of pRL1383a will be amplified with BioBrick based primers. These components will be used later in the construction of a pRL1383a BioBrick based vector. These parts include the aadA region from pRL1383a and from the BioBrick Registry, the origin of vegetative replication (oriV), and the replication proteins.
Methods
Materials Per Reaction Added in order:
3.5uL nanopure water
1uL 10uM forward/reverse primers
1uL pRL1383a as template (1:10 dilution of large-scale plasmid prep ).
5uL Taq polymerase: Accusure(hot start, 68°C) and Red Taq
Note: the taq and water can be combined and aliquoted together as long as reaction is kept at 4°C.
cold blocks
pcr reaction tubes, with tops
pipette and tips
PCR Running Conditions
Duration of Time
T° 5 cycles
T° 30 cycles
Purpose
10 minutes
95°C
95°C
heat activate taq
30 seconds
95°C
95°C
denaturation
30 seconds
48.1°C, 53.4°C, 57.9°C
59°C, 60.6°C, 61.9°C
annealing
5 minutes
68°C
68°C
extension (1.5 minutes per kb, go with longest)
10 minutes
68°C
68°C
finishing the extension
infinity
4°C
4°C
(until you remove product)
Materials for the gel
1% agarose gel
10ug/mL Ethidium Bromide
running apparatus
gel running conditions: 95V for ~1 hour
PCR Clean-up using Qiagen Gel Extraction Kit
Use after running products on a gel
add 3 volumes of QG to 1 volume of gel
incubate 50°C for 10 minutes, until gel dissolves; spin every few minutes
add 1 gel volume of isopropanol, mix by inverting
place sample in minelute spin column
centrifuge 1 minute at 13k rpm
discard flow-through
add 750uL PE buffer to wash; centrifuge 1 minute 13k rpm
discard flow-through; centrifuge 1 min 13k rpm
place column in clean 1.5 mL tube
add 10ul TE or water to center of column, wait 1 minute, centrifuge 1 min
Results
PCR Gel 1 (left) and PCR Gel 2 (right).
PCR Gel 1
Lane
Contents
Description
Predicted annealingT°C,5 cycles
Predicted annealing T°C,30 cycles
1
Tri-dye 10kb ladder
ok
2
omega (aadA)#1
(+), ~1kb
54.4,50.4
60.7,69.5
3
oriV #1
(+), ~0.5kb
51.3, 53
60.9, 71.2
4
mob #1
smear
53.5, 48
62, 69.8
5
rep #1
smear, low kb
53.9, 57.2
59.1, 73.5
6
(-)control
all clear
7
(+) control
bright band, ~0.5kb
8
omega #2
(+), ~1kb
54.4,50.4
60.7,69.5
9
oriV #2
(+) ~0.5kb
51.3, 53
60.9, 71.2
10
mob #2
nothing, slight smear
53.5, 48
62, 69.8
PCR Gel 2
Lane
Contents
Description
Predicted annealing T°C,5 cycles
Predicted annealing T°C,30 cycles
1
Tri-dye 10kb ladder
no ladder
2
rep#2
light smear
53.9, 57.2
59.1, 73.5
3
(-)control
all clear
4
(+)control
bright band ?0.5kb
5
omega #3
bright band, ?1kb
54.4,50.4
60.7,69.5
6
oriV #3
bright band, 0.5kb?
51.3, 53
60.9, 71.2
7
mob
light band, some smear, 3kb?
53.5, 48
62, 69.8
8
rep #3
band in low kb
53.9, 57.2
59.1, 73.5
9
(-) control
all clear
10
(+) control
bright band, 0.5kb?
PCR Clean-up
MinElute PCR Purification Kit Protocol from Qiagen using a microcentrifuge
This protocol is designed to purify double-stranded DNA fragments from PCR reactions resulting in high end-concentrations of DNA (see page 12). Fragments ranging from 70 bp to 4 kb are purified from primers, nucleotides, polymerases, and salts using MinElute spin columns in a microcentrifuge.
Mix PB 5:1 PCR reaction.
Place a MinElute column in a provided 2 ml collection tube.
Apply ALL of sample to the MinElute column; centrifuge (13,000rpm)1 min.
Discard flow-through. Place the MinElute column back into the same tube.
Add 750 μl Buffer PE to the MinElute column; centrifuge (13,000rpm) 1 min.
Discard flow-through and place the MinElute column back in the same tube.
Centrifuge the column for an additional 1 min at maximum speed.
Place the MinElute column in a clean 1.5 ml microcentrifuge tube.
Add 10 μl 1x TE to the center of the membrane, let the column stand for 1 min; centrifuge 1 min.
Discussion
The gels are very warped. I did not let them dry long enough before I poured the running buffer,so some products did not run correctly therefore this experiment will be repeated on Monday when the necessary reagents are delivered.
I mixed up the temperature gradient!!! I need to be more careful about labeling my reactions.
The rep protein did not come out at all and the mob region only came out at one temperature, so these must be PCRed again to determine the correct annealing temperature.
Follow-up Experiments
7/2/08
A second attempt was made to PCR the rep region. An annealing temperature of 53.4° was used for the first 5 cycles and annealing temperature 60.6 was used for the subsequent 30 cycles.
Results: There are 2 faint bands in the low kb region. This is probably primers.
Lane 1 contains the ladder, lanes 3 through 6 contain the products of a plasmid prep, lane 7 contains the PCR of the rep region, while lanes 8 and 9 are the positive and negative control, respectively. Lane 10 contains one of NW's PCR reactions.
Discussion: The rep region did not amplify under these conditions. There were several suggestions made by the professors that I will try this weekend.
7/4/08
Materials:
5uL Red Taq (per reaction)
3.5uL nanopure water (per reaction)
1uL 10uM f/r rep region primers (per reaction)
1uL of either 1:100, 1:500, 1:1000 dilution pRL1383a as template (1:10 dilution of large-scale plasmid prep )
5uL accusure (for one reaction to test the dilution with this enzyme)
PCR Running Conditions
Duration of Time
T° 5 cycles
T° 30 cycles
Purpose
10 minutes
95°C
95°C
heat activate taq
30 seconds
95°C
95°C
denaturation
30 seconds
54
59
annealing
5 minutes
70°C (red taq),68°C (accusure)
68°C (red taq),68°C (accusure)
extension (1.5 minutes per kb, go with longest)
10 minutes
70°C (red taq),68°C (accusure)
68°C (red taq),68°C (accusure)
finishing the extension
infinity
4°C
4°C
(until you remove product)
PCR Gel of rep region amplified with red taq.
Results
PCR Gel 1
Lane
Contents
Description
1
Tri-dye 10kb ladder
faint, not well defined
2
rep 1:100 dilution
3.3kb, brighter than ladder, very thick band, smear (too much DNA?)
3
rep 1:500 dilution
3.3kb, brighter than ladder, very thick band, smear (too much DNA?)
4
rep 1:1000 dilution
3.3kb, brighter than ladder, very thick band, smear (too much DNA?)
5
(-)control
very faint band in low kb region (maybe contamination from other well
6
(+) control
faint band in 0.5 kb region
7
1:500 dilution rep region amplified by accusure
0.5kb band and band in low kb region, faint
7/7/08
rep and mob regions did not amplify with accusure, so they were amplified using red/green taq. Same conditions from 7/4 were used.
results
mob and rep regions were amplified by both red and green taq. Use green in the future, it is cheaper.
PCR amplification of mob and rep regions.
Discussion
rep region appears to be amplified by red taq efficiently. Need to use much less DNA. Find out what dilution to use. Accusure + some reaction stabilizing reagent might be used to get this reaction going, but as of right now, this region is not being amplified by this enzyme. This temperature seems to have worked well also.
7/18/08
PCR amplification of the mob/rep region, the omega interposon, the aada gene, oriV, a (-) control, and (+)control. All products that amplified are too bright to use the ladder scale.
PCR amplification of oriV, omega interposon, aadA gene, & rep/mob all using green taq.
PCR conditions: 2min@94°C, 30sec@95°C, 30sec@50.4°C, 5min30sec@72°C(1 minute per kilobase), infinity@4°C
7/19/08
Lane 1 and 12 is the tridye 2-log ladder, 2,3,4, &5 are the mob/rep region, the omega interposon, the negative and positive control, respectively amplified with Green Taq. Lanes 6 and 7 are the mob/rep and omega amplified with accuzyme. Lanes 8,9,10,&11 are the mob/rep region, the omega interposon, the negative and positive control, respectively amplified with velocity taq.
PCR amplification of omega interposon and rep/mob regions using 3 different types of taq:
File:Velocity taq.jpg
PCR amplification with velocity taq, green taq and accuzyme with the hopes that one will amplify these products. I also changed the annealing temperature to 48°C in order to accommodate the primer with the lowest annealing temperature.
Methods For all reactions, I heated up the PCR block to 94C before adding my sample. I did this by running another method which only involves 94C for infinity.
Accuzyme:
Reaction conditions: 3.5ul H2, 1ul Primers, 0.5ul template, 5ul taq
Running conditions: 95C 10min, 95C 30sec, 48C 30sec, 68C 10 min, 68C 10min, 4C infinity.
Green Taq:
Reaction conditions: 3.5ul H2, 1ul Primers, 0.5ul template, 5ul taq
Running conditions: 95C 10min, 95C 30sec, 48C 30sec, 68C 10 min, 68C 10min, 4C infinity.
Velocity DNA Polymerase:
Reaction conditions: 2ul 5X Buffer, 0.1ul dNTPs, 5.65ul H2O, 1ul Primers, 0.5ul template, 0.25ul taq(0.5U)
Running conditions: 98C 2min, 98C 30sec, 48C 30sec, 72C 2min34sec, 72C 10min, 4C infinity.
Results
As seen in the very small picture, the only amplification product of this reaction is of the positive control amplified by green taq.
The omega interposon template (pSMC121) was run on a gel alongside the products of the colony PCR from 7_21 . There was no band meaning that there was no template for this PCR reaction.
Discussion
Need to get in touch with Dr. Callahan and get more template. Also, just try again with green taq only... problem solve.
7/23
PCR amplification of the omega interposon region of pSMC121 and the mob/rep region of pRL1383a using green taq.
Reaction conditions: same as above
Running conditions: same as above for green taq.
Results
PCR amplification of mob/rep region and omega interposon. Lane 1: 2-log tridye ladder, lane 2:omega interposon, lane 3: mob/rep region, lane 4: negative control, lane 5: positive contol.
The omega interposon amplification yielded only primer dimers. The mob/rep region is a smear, there is a lot of mis-priming. The positive and negative controls yield a bright band and no band respectively.
Discussion
the omega interposon primers are likely to form dimers  because there are 4 overlapping nucleotides at the 5' and 3' ends with a Gibbs free energy of -6.61 kcal/mole. I can alleviate this problem by finding a higher annealing temperature than 48°C. The annealing temperature is 49.5°, so I can use a gradient to find a good temperature. I will try from 48deg&;C to 55°C.
The above stated method for solving the primer dimer problem can also be a possible solution for the mis-priming of the mob/rep region.
7/29/08
PCR amplification of oriV, aada, rep.
Amplification of BBa_J23012 which is aadA, aadA from pRL1383a, and the rep region. I used accusure for all and also green taq for rep.
Materials
Accusure Master-Mix (for 20 50ul reactions), stored in 4°C:
10x buffer: 100ul
100uM dNTPs: 10ul
Accusure enzyme: 20ul
nano-pure-water: 38.5ul
for each reaction add: 4ul f&r primer, and 1ul template
Methods
reaction conditions:
accusure reaction: 45ul master mix, 4ul f&r primer, and 1ul template
green taq reaction: 20ul water, 25ul green taq, 4ul f&r primer, 1ul template
running conditions:
accusure reaction: annealing temp: 50.4°C, 5minutes extension time at 68°C
green taq reaction: 53.9°C, 5minutes extension time at 72°C
Results
PCR Products
lane
name
quantity
size
comments
2
aadA from BBa_J23012
3.2 ng/ul
0.85kb
the best aadA band
3
aadA from pRL1383a
1.6ng/ul
0.8 kb
smaller than the BB aadA
4
rep region from pRL1383a
24ng/ul
3.5kb
brighter than any band, hard to quantify. Trail of DNA on both sides of the band & bright band @ 100bp region.
5
oriV
nothing
nothing
I will amplify with green taq.
6
rep with accusrue
nothing
nothing
will use the green taq version
7
negative control
1ng/ul
0.6kb
The water used for the master mix is contaminated
8
positive control
light band
0.5 kb
This did not amplify well
Discussion
I was able to amplify some of the parts, though they are not as bright and/or distinct as I may need them for downstream work. The master-mix appears to be contaminated because UPA amplified something in the 0.6kb region when there should have been no band.
I will try to amplify oriV with green taq.
8/4
PCR amplification of rep and oriV, plus verification of oriT and Plac
Margaret
Rep, oriV, verification of I14032, oriT(just to make sure)
Materials & Methods
Reaction Conditions: 3.5ul water, 0.5ul template, 1.0ul f&r primer, 5ul econotaq
Running conditions (I14032, oriT): 94°C hold, 94°C 2', (95°C 30", 62°C 30",  72°C 30") x 30cycles, 72°C 10', 4°C hold
Running conditions (oriV + controls): 94°C hold, 94°C 2', (95°C 30", 51.3°C 30",  72°C 30") x 30cycles, 72°C 10', 4°C hold
Running conditions (rep + controls): 94°C hold, 94°C 2', (95°C 30", 53.9°C 30",  72°C 3'20") x 20 cycles, (95°C 30", 53.9°C 30",  72°C 3'30") x 10 cycles72°C 10', 4°C hold
Results
Lane 2: PCR verification of I14032: There are two bands. The top  band does not correspond to Plac, while the bottom band is just above  0.4kb, about 50bp more than would be expected. Lane 3: PCR verification of OriT: The band is just above 0.4kb, and the verification part is 0.415 kb. Lane 4: Amplification of oriV from pRL1383a:The band is at 4.5kb, as  expected. Lane 5&8: pSB1A7 was used as a (+) control: While the same size was amplified in both positive control reactions, I would expect a band at 1265bp, but the amplicon is just below 0.9kb. Lane 6&9: VF2 and VR were used to amplify water. As expected, nothing was amplified. Lane 7:The rep region was amplified correctly. I expected a band at 3.352kb and the band  occurred in this region.
Discussion
I did some trouble shooting on this PCR with advice from: [ [1] ].
In most cases, I just got as close to the recommended annealing temperature and extension time in order to decrease the amount of unspecific products.
I only ran 30 cycles as opposed to 35 in the past. I think this has made the efficiency of my taq more consistent, especially with the longer products.
I also added an additional 10 seconds to last 20 cycles of the rep region, [ [2] ], recommends this extra time because the taq supposedly loses efficiency and thus there will be multiple products of suspicious size. This seems to have worked for the rep region, which contains only one product.
Next Step is to amplify both rep and oriV to large quantities using their respective conditions.
8/5
PCR amplification of P1 lytic region, aadA from pRL1383a, and aadA from BioBrick Registry. Test.
Margaret
testing PCR conditions for P1 lytic region and aadA (registry and pRL1383a)
Materials & Methods
Reaction conditions: 3.5ul water, 5ul taq, 0.5ul template (P1 lytic: pSB2K3(Plac), (+)&(-):pSB2K3(Plac): , aadA: pRL1383a, (+)&(-):pSB1A7, aadA: BioBrick Registry), 1ul primer (P1 lytic: p1lytic f&r, (+)&(-):VF2&VR , aadA: aadA f&r, (+)&(-):VF2&VR, aadA: aadA f&r)
Running Conditions:
p1 lytic region: 94°C hold, 94°C 2', 95°C 30", 54.2°C 30", 72°C 1'10", 72°C 10', 4°C hold.
aadA (pRL1383a & BB registry): 94°C hold, 94°C 2', 95°C 30", 50°C 30", 72°C 30", 72°C 10', 4°C hold.
Results
Lane 1 contains the tri-dye 2-log ladder, which is not well resolved. K&G had better resolution of the ladder at lower voltage.
Lane 2 contains the amplification of the P1 lytic region from the I14032 prep which is on pSB2K3. This part did not amplify. The positive and negative controls in
Lanes 3&4 are actually the same thing: pSB2K3 (containing I14032) and VF2+VR. I mistakenly put Primer and Template in the negative. I do get amplification, though, which tells me the plasmid is there. The band should be around 353bp but looks to be above 400bp. This discrepancy is found in all amplifications of this part (which I have verified twice before: here and here . K&G also amplified it directly from the BB registry and got the same band. :* Lane 5 aadA from pRL1383a was amplified and large amounts of the right size are visualized on the gel.
Lanes 6&7 are the positive and negative control but this time VF2 and VR are amplifying pSB1A7 (proven on several occasions to amplify consistently; like here ). The same mistake with the other controls in this experiment happended:(.
Lane 8 contains the amplification of aadA from the BioBrick Registry. It has the right size and large amounts.
Conclusions
The aadA region amplified well under these conditons (for both the pRL1383a & BB Registry amplifications). The next step is to amplify large quantities.
The P1 lytic region did not amplify. The plasmid is there, though, because the (+) control (pSB2K3(Plac)) amplified a band of the correct size. I will re-evaluate the running conditions, and also extract the pSB2K3 plasmid from the registry and use it to amplify with the P1 lytic primers.
8/6
Large-scale PCR reactions for rep, oriV, and aadA (from pRL1383a and BBregistry)
Materials & Methods
Large-scale PCR of rep, aadA (both), and oriV.
Used reaction and running conditions specified in the reactions from 8/5 and 8/4.
Test PCR for omega and P1 lytic.
Mini-prep of pSMC121 used for PCR of omega region.
Try out conditions for P1 lytic and omega.
8/7
Large scale PCR for amplification of P1 lytic and aadA region.
Margaret
Large-scale PCR amplification of  aadA (it was lost in a gel cutting mishap) and P1 lytic.
Reaction conditions: 3 50ul green taq reactions for each.
Running conditions: see below
Completed PCR Products
PCR Products
Name
description
reaction conditions
OriV
origin of vegetative replication
94°C hold, 94°C 2', (95°C 30", 51.3°C 30", 72°C 30") 30 cycles, 72°C 10', 4°C hold
rep region
proteins necessary for autonomous replication
94°C hold, 94°C 2', (95°C 30", 53.9°C 3'20", 72°C 30") 10 cycles, (95°C 30", 53.9°C 30", 72°C 3'30") 20 cycles , 72°C 10', 4°C hold
aadA (pRL1383a & BB registry)
Sm, Sp resistance
94°C hold, 94°C 2', (95°C 30", 50°C 30", 72°C 48") 30 cycles, 72°C 10', 4°C hold
Discussion
Using the reaction conditions in the table above and green taq as the polymerase, all products were successfully amplified first in a test run, then on a large scale. In the next steps the products will be digested with restriction enzymes and ligated to their respective vectors. The experiments for this can be found here .
Retrieved from " http://2008.igem.org/Team:Hawaii/PCR_Amplification_of_pRL1383a "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
