<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hawaii_Initial_Synth_Oligo_Assembly"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Hawaii/Initial_Synth._Oligo_Assembly">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hawaii/Initial_Synth._Oligo_Assembly&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/Initial_Synth._Oligo_Assembly&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/Initial_Synth._Oligo_Assembly&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hawaii/Initial_Synth._Oligo_Assembly" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hawaii/Initial Synth. Oligo Assembly</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Promoter_and_Signal_Sequence_Assembly"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Promoter and Signal Sequence Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Overview"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Protocol"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Hybridization_of_Parts"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Hybridization of Parts</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#DNA_Ligation"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">DNA Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Restriction_Enzyme_Digest_of_Ligated_Oligonucleotides"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Restriction Enzyme Digest of Ligated Oligonucleotides</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Restriction_Enzyme_Digest_of_BBa_C0012"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Restriction Enzyme Digest of BBa_C0012</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Gel_extraction"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Gel extraction</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Transformation_into_DH5.CE.B1"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Transformation into DH5α</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Colony_PCR"><SPAN class="tocnumber">1.2.7</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Sequencing"><SPAN class="tocnumber">1.2.8</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-12"><A href="#Results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#First_attempt"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">First attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Second_attempt"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Second attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Third_attempt"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Third attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Fourth_attempt"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Fourth attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Fifth_attempt"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Fifth attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Plasmid_prep_and_digestion_of_BBa_C0012_harboring_vector"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">Plasmid prep and digestion of BBa_C0012 harboring vector</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Transformation_into_DH5.CE.B1_2"><SPAN class="tocnumber">1.3.7</SPAN><SPAN class="toctext">Transformation into DH5α</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Sixth_attempt"><SPAN class="tocnumber">1.3.8</SPAN><SPAN class="toctext">Sixth attempt</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Sequencing_2"><SPAN class="tocnumber">1.3.9</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#nir_promoter"><SPAN class="tocnumber">1.3.9.1</SPAN><SPAN class="toctext">nir promoter</SPAN></A></LI><LI class="toclevel-4 tocsection-23"><A href="#slr2016_signal_sequence"><SPAN class="tocnumber">1.3.9.2</SPAN><SPAN class="toctext">slr2016 signal sequence</SPAN></A></LI><LI class="toclevel-4 tocsection-24"><A href="#pilA_signal_sequence"><SPAN class="tocnumber">1.3.9.3</SPAN><SPAN class="toctext">pilA signal sequence</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Discussion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Construction_of_OriT_from_RP4_using_Overlapping_Oligonucleotides:_Attempt_1"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Construction of OriT from RP4 using Overlapping Oligonucleotides: Attempt 1</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Protocol_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Results_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Discussion_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Promoter_and_Signal_Sequence_Assembly">Promoter and Signal Sequence Assembly</SPAN></H1><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>Construct the <I>nir</I> promoter and <I>slr2016</I> and <I>pilA</I> signal sequences in Biobrick format and clone them in <I>E. coli</I> as part 1 of cyanobacterial secretion system development.
</P><P>Order of events:
</P><OL><LI> Synthesize oligonucleotides encoding the <I>nir</I> promoter and <I>slr2016</I> and <I>pilA</I> signal sequences in Biobrick format.
</LI><LI> Anneal sense and anti-sense oligonucleotides.
</LI><LI> Ligate oligonucleotides to create the full Biobrick parts.
</LI><LI> Restriction digest the ligation products.
</LI><LI> Run annealed and digested ligated products on a gel to verify annealing and ligation.
</LI><LI> Extract bands corresponding to the correct ligated products from the gel.
</LI><LI> Ligate the <I>nir</I> promoter and <I>slr2016</I> and <I>pilA</I> signal sequences to a vector.
</LI><LI> Subclone in <I>E. coli</I>.
</LI></OL><H2><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H2><H3><SPAN class="mw-headline" id="Hybridization_of_Parts">Hybridization of Parts</SPAN></H3><OL><LI>Mix: 
<DL><DD>3 μl 100 µM sense oligo 
</DD><DD>3 μl 100 µM anti-sense oligo 
</DD><DD>3 μl 10 x PNK (polynucleotide kinase) buffer 
</DD><DD>2 μl 10mM ATP 
</DD><DD>2 μl T4 polynucleotide kinase (PNK) 
</DD><DD>17 μl distilled water 
</DD><DD>Total volume = 30 μl 
</DD></DL></LI><LI>Incubate at 37C for 1.5 hours.
</LI><LI>Add 4 μl 0.5 M NaCl. 
</LI><LI>Place in boiling water bath for 2 min., then remove water bath from the heat source and allow the reaction (still in the water bath) to cool to room temperature (approx. 30 minutes)
</LI></OL><DL><DD><I>Reference: Pam Silver Lab. <A href="http://openwetware.org/wiki/Silver:_Oligonucleotide_Inserts|" class="external text" rel="nofollow">Oligonucleotide Inserts.</A></I></DD></DL><H3><SPAN class="mw-headline" id="DNA_Ligation">DNA Ligation</SPAN></H3><OL><LI>Combine 4 ul of construct one with 4 ul of construct 2.  
</LI><LI>Add 1 ul of nanopure H<SUB>2</SUB>0.
</LI><LI>Add 10 ul of 2x quick ligation reaction Buffer.  In order to avoid shearing the DNA, mix by resuspending very slowly.
</LI><LI>Add 1 ul of Quick T4 DNA ligase and mix thoroughly by resuspending very slowly.
</LI><LI>Incubate at room temperature for 5 minutes. 
</LI><LI>Cool on ice then transform, or store at -20<SUP>o</SUP>C
</LI></OL><DL><DD><I>Reference: Quick Ligation Kit from NEB.</I></DD></DL><H3><SPAN class="mw-headline" id="Restriction_Enzyme_Digest_of_Ligated_Oligonucleotides">Restriction Enzyme Digest of Ligated Oligonucleotides</SPAN></H3><UL><LI> May not be necessary
</LI></UL><OL><LI> Added 7 μl nanopure water, 1 μl NEBuffer 3, 1 μl XbaI, and 1 μl PstI to the 20μl ligation reaction.
</LI><LI> Incubated 1 hour at 37C.
</LI><LI> Stopped reaction by incubating 10 min. at 65C.
</LI></OL><P>Note:
</P><UL><LI> This reaction did not follow the reagent amounts recommended by NEB or &quot;Short Protocols&quot; because the ligation buffer contained ingredients not usually found in DNA solutions to be digested. Dr. Presting revised the recommended protocols to adjust for this factor.
</LI></UL><DL><DD><I>Reference: Short Protocols in Molecular Biology. Vol. 1. 5th edition. 3-3 to 3-4.</I></DD></DL><H3><SPAN class="mw-headline" id="Restriction_Enzyme_Digest_of_BBa_C0012">Restriction Enzyme Digest of BBa_C0012</SPAN></H3><OL><LI>  Combined 0.4 μl C0012 (from plasmid prep), 15.6 μl nanopure water, 2 μl 10X NEBuffer 2, 1 μl XbaI, and μl PstI.
</LI><LI> Incubated 2.5 hours at 37C (recommended incubation of at least 1 hour)
</LI><LI> Stopped reaction by incubation 10 min. at 65C.
</LI></OL><P>General Notes on RE Digests:
</P><UL><LI> Remember to check that REs are compatible.
</LI><LI> Reaction can also be stopped by adding 5μl 10X loading buffer.
</LI><LI>&lt;10% of digest should be RE
</LI><LI> REs are stored in glycerol. &lt;5% glycerol should be present in digest. Therefore, a minimum 10-fold dilution is necessary so the enzymes don't act funky.
</LI><LI> It's always a good idea to gently shake and spin down RE and RE buffer prior to use.
</LI></UL><DL><DD><I>References: Short Protocols in Molecular Biology. Vol. 1. 5th edition. 3-3 to 3-4. </I></DD><DD><I>NEB</I></DD></DL><H3><SPAN class="mw-headline" id="Gel_extraction"><A href="/Team:Hawaii/Protocols/gel_extraction" title="Team:Hawaii/Protocols/gel extraction">Gel extraction</A></SPAN></H3><UL><LI>Used Qiagen MiniElute Gel Extraction kit.
</LI></UL><H3><SPAN class="mw-headline" id="Transformation_into_DH5.CE.B1"><A href="/Team:Hawaii/Protocols/Transformation_E_coli" title="Team:Hawaii/Protocols/Transformation E coli">Transformation into DH5α</A></SPAN></H3><UL><LI> Incubated 8 minutes on ice instead of 30 minutes. Dr. Callahan recommended 5-10 min. incubation because longer incubations may decrease the number of transformants.
</LI></UL><H3><SPAN class="mw-headline" id="Colony_PCR"><A href="/Team:Hawaii/Protocols/Colony_PCR" title="Team:Hawaii/Protocols/Colony PCR">Colony PCR</A></SPAN></H3><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P>Five microliters of the PCR reactions were treated with two microliters of ExoSAP and diluted to a concencentration of ~20ng/100bp.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="First_attempt">First attempt</SPAN></H3><P>Nice bands observed for annealed oligos. Bands appear to be the correct size (&lt;100bp) but we can't tell if anneal worked (EtBr doesn't tell us if these are ssDNA or dsDNA bands). Smear is probably from loading way too much DNA (see discussion). Possible faint band for pnir ligation. No bands for slr2016 and pilA ligation, probably because way too much DNA was added for the ligation reaction.
</P><H3><SPAN class="mw-headline" id="Second_attempt">Second attempt</SPAN></H3><P>Did a ligation using 1 μl of a 10<SUP>-6</SUP> dilution of each annealed product. Ligated at both room temperature and at 40C (recommended by Frank, <I>etal</I>). Nothing was visible on the EtBr stained 4% agarose gel because not enough DNA was present.
</P><H3><SPAN class="mw-headline" id="Third_attempt">Third attempt</SPAN></H3><P>Did a ligation using 10<SUP>-2</SUP> dilutions of annealed product (ligated at room temperature and at 40C). Faint bands were visible for the annealed products (10<SUP>-2</SUP> dilutions) but no bands were observed for the ligated products. Not enough DNA was present. Ran 10<SUP>-1</SUP> dilutions of annealed product on SYBR Safe stained 2% agarose gel.
</P><H3><SPAN class="mw-headline" id="Fourth_attempt">Fourth attempt</SPAN></H3><P>Took 10 μl ligation reaction from first attempt and restriction digested with XbaI and PstI (1 μl each) in NEBuffer 3 (1 μl). A ladder of bands were observed for the ligation product, indicating incomplete digestion.
</P><H3><SPAN class="mw-headline" id="Fifth_attempt">Fifth attempt</SPAN></H3><P>Repeated ligation using full concentrations of annealed product. RE digested as in fourth attempt, with incubation for 2.5 hours. Loaded 20 μl of ligation into well with 2μl loading dye. Less loading dye was used so the bromophenol blue would not block out our band. Band(s) will not resolve on the gel if less than 20μl of the ligation is loaded. 
</P><P>Gel turned out funky (was it poured too late and the agarose already began solidifying?) Ladder did not run correctly but luckily, the pnirs did (we know their size). Also, from the fourth attempt gel, we knew the approximate location of ligation bands. Bands excised from the gel are circled in the figure below. For the <I>slr2016</I> signal sequence ligation, no band was observed at the expected size. Very faint bands were observed above and below the expected ~80bp mark so both were excised and purified.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">First assembly attempt. Ten microliters loaded into each well of an EtBr stained ~3% agarose gel ran at 95V for 1 hr.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Third ligation attempt. Ten microliters loaded into each well of an EtBr stained 4% agarose gel ran at 95V for 28 min.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">SYBR Safe stained 2% agarose gel ran at 95V for 30 min. Ten microliters of a 10<SUP>-1</SUP> dilution of annealed product was loaded into each well.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Fourth attempt. Ten microliters of 10<SUP>-1</SUP> dilution of annealed product or 1:1 ligation product was loaded into each well. Ran on EtBr stained 4% agarose gel at 95V for 30 min.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Fifth assembly attempt. Ten microliters of ladder, pnir1, and pnir2 loaded in wells. Twenty microliters of pnir1+2, slr1+2, and pilA1+2 loaded into wells. SYBR Safe stained 4% agarose gel ran at 95V for 50 min. Circled bands were extracted from the gel.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Plasmid_prep_and_digestion_of_BBa_C0012_harboring_vector">Plasmid prep and digestion of BBa_C0012 harboring vector</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Restriction digest of BBa_C0012. Twenty microliters of the digest were loaded onto an EtBr stained 0.8% agarose gel ran at 95V for 20 min.</DIV></DIV></DIV><P>The plasmid prep was RE digested using XbaI and PstI per NBE's instructions. Distinct bands were observed the gel at 1.2kb and 2.1kb, corresponding to the lacI gene (BBa_C0012) and the vector. Comparing the intensity of the vector band to the ladder bands, it is estimated that there is ~40ng of vector.
</P><H3><SPAN class="mw-headline" id="Transformation_into_DH5.CE.B1_2">Transformation into DH5α</SPAN></H3><P>No transformants were observed.
</P><H3><SPAN class="mw-headline" id="Sixth_attempt">Sixth attempt</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Ligation products. Ten microliters of ligation product were loaded into each well of a 3% agarose gel stained with EtBr and ran at 95V for 1 hour.</DIV></DIV></DIV><P>The ligation was repeated because the gel from the 5th attempt was of questionable quality and no transformants were observed. Three different constructs of the full <I>nir</I> promoter and <I>slr2016</I> and <I>pilA</I> signal sequences were created. The first followed the ligation and restriction digest protocols previously used, using full concentrations of annealed products. The second was a ligation and restriction digest carried out using 10<SUP>-1</SUP> dilutions of annealed product. It was suggested by SC that if too much DNA was added to the ligation mixture, &quot;DNA hairballs&quot; would form between ssDNA (unannealed) and dsDNA, resulting in smears and multiple bands in the gel. The last method of construction simply placed equal concentrations of annealed products together. In theory, DNA overhangs would anneal appropriately, yielding an unligated but full promoter or signal sequence.</P><P>Ligation at full concentrations of annealed product worked best. The strongest bands were observed for this method. Multiple bands were still observed, above and below the expected band length (80-100bp) indicating both unligated DNA fragments and incompletely digested DNA or hairball structures. However, the strongest band for all three constructs corresponded to the desired product. The 10<SUP>-1</SUP> ligations were barely visible on the gel, and multiple bands were still observed, so ligation and restriction digest efficiency was not improved by the addition of less DNA. The last method of annealing but not ligating yielded two bands in the gel corresponding to the two original DNA fragments.</P><P>A brand new, pure, agarose gel was run again using the products of the same reactions. Twenty microliters of product were loaded into each well and bands corresponding to the desired products were excised and purified. This gel ran at 60V for 2 hours for better band separation. Bands were twice as bright as before and farther apart, but otherwise identical to the gel pictured.</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Ligations of insert and vector. Fifteen microliters of the ligation reactions were loaded into each well of an EtBr stained 3% agarose gel ran at 95V for 1 hour.</DIV></DIV></DIV><P>The purified DNA was then ligated to a Biobrick vector for subcloning in <I>E. coli.</I> Ligations were carried out using 3:1 molar concentrations of insert to vector. Ligations were performed using full concentrations of insert and vector as well as 1:10 dilutions of both. A third method used full concentrations of annealed product and vector. When ran on a 3% agarose gel for verification of ligation, only products from the third method were visible. Again, a smear similar to that seen in the first ligation attempt (see above) was observed indicating &quot;DNA hairballs.&quot; The 1:1 dilutions ran were probably not visible because the DNA concentration was too low, despite the fact that 15 μl (out of 20 μl) of the ligation mixture was loaded. </P><P>Five microliters of the ligation reactions were used to transformation DH5α. None of the 1:10 dilutions used for transformation resulted in any colonies, probably because there was too little DNA. Colonies were observed for all assemblies at 1:1 dilutions. One of the negative controls, transformation using only the vector, resulted in colonies. It is suspected that the RE did not digest all of the DNA completely and that band cutting of the gel was messy. Presence of desired inserts were verified using colony PCR.</P><P>Colony PCR was carried out at three annealing temperatures: 53C, 58C, and 62C. The T<SUB>m</SUB> of the Biobrick verification primers is around 58C and Reshma Shetty recommended annealing at 62C on <A href="http://openwetware.org/wiki/Engineering_BioBrick_vectors_from_BioBrick_parts/Colony_PCR" class="external text" rel="nofollow">her protocol on OpenWetWare</A>. No visible difference was detected between the three annealing temperatures when the PCR products were run on a gel. The 58C gel picture shows wavy bands because the bands ran too close to the bottom of the gel. The 0.8% agarose used was not intended to reveal exact sizes of PCR products; rather, it was intended to verify the presence of inserts within the size range expected. PCR products have an extra ~240bp added to the size of the desired insert to account for the upstream/downstream location of the verification sites on the Biobrick plasmids. The bands for pnir, slr2016, and pilA all approximate 300bp, within the size range of the expected products (~60-100bp + ~240). The band for C0012 is ~1000bp (1128bp + ~240). The vector PCR product showed a band very similar in size to the C0012 band, supporting the suspicion that the vector was not completely digested and band excision was sloppy. This also explains why transformants were observed. All PCR products will be sent for sequencing to verify their identities.</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Colony PCR of transformants. Ten microliters of the PCR reactions were loaded into each well of an EtBr stained 0.8% agarose gel ran at 95V for 1 hour.</DIV></DIV></DIV><DIV style="text-align: center;"><B>Transformation of DH5α with assembled oligonucleotides</B></DIV><TABLE border="1"><TBODY><TR><TH> Assembly
</TH><TH> Colony Forming Units
</TH><TH> Plate
</TH></TR><TR><TD align="center">BBa_C0012 derived vector only (negative control)
</TD><TD align="center">77
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center">DH5α (negative control)
</TD><TD align="center">0
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center">BBa_C0012 insert + BBa_C0012 derived vector (positive control)
</TD><TD align="center">206
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center">BBa_C0012 insert + BBa_C0012 derived vector (1:10 dilution) (positive control)
</TD><TD align="center">0
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center"><I>nir</I> promoter + BBa_C0012 derived vector
</TD><TD align="center">40
</TD><TD>LB + amp<SUB>100</SUB></TD></TR><TR><TD align="center"><I>nir</I> promoter + BBa_C0012 derived vector (1:10 dilution)
</TD><TD align="center">0
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center"><I>slr2016</I> signal sequence (large band) + BBa_C0012 derived vector
</TD><TD align="center">22
</TD><TD>LB + amp<SUB>100</SUB></TD></TR><TR><TD align="center"><I>slr2016</I> signal sequence (large band) + BBa_C0012 derived vector (1:10 dilution)
</TD><TD align="center">0
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center"><I>slr2016</I> signal sequence (small band) + BBa_C0012 derived vector
</TD><TD align="center">45
</TD><TD>LB + amp<SUB>100</SUB></TD></TR><TR><TD align="center"><I>slr2016</I> signal sequence (small band) + BBa_C0012 derived vector (1:10 dilution)
</TD><TD align="center">0
</TD><TD>LB + amp<SUB>50</SUB></TD></TR><TR><TD align="center"><I>pilA</I> signal sequence + BBa_C0012 derived vector
</TD><TD align="center">54
</TD><TD>LB + amp<SUB>100</SUB></TD></TR><TR><TD align="center"><I>pilA</I> signal sequence + BBa_C0012 derived vector (1:10 dilution)
</TD><TD align="center">0
</TD><TD>LB + amp<SUB>50</SUB></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Sequencing_2">Sequencing</SPAN></H3><P>Sequencing by the Greenwood Molecular Biology Facility returned the following results:
</P><H4><SPAN class="mw-headline" id="nir_promoter"><I>nir</I> promoter</SPAN></H4><H4><SPAN class="mw-headline" id="slr2016_signal_sequence"><I>slr2016</I> signal sequence</SPAN></H4><H4><SPAN class="mw-headline" id="pilA_signal_sequence"><I>pilA</I> signal sequence</SPAN></H4><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>We need to dilute our annealed products before running them in a gel or attempting ligation. 20-125ng DNA (per lane) is good for running gels. One reaction using the NEB Quick Ligation Kit can ligate up to 180ng of DNA.
</P><P>After the gel extraction, we end up with very little product to transform with. In the future, we should run larger ligation reactions of annealed product and perhaps run 2-3 lanes of it in the gel.
</P><P>
Success! Our sequenced DNA matches that of the desired products.</P><H1><SPAN class="mw-headline" id="Construction_of_OriT_from_RP4_using_Overlapping_Oligonucleotides:_Attempt_1">Construction of OriT from RP4 using Overlapping Oligonucleotides: Attempt 1</SPAN></H1><H2><SPAN class="mw-headline" id="Protocol_2">Protocol</SPAN></H2><P>The protocol differs from above in that there are 6 oligonucleotides to overlap and there fore 2 separate ligations. There is also a modification of the restriction digest.
</P><P><B>RP4 Origin of Transfer Sequences for Oligonucleotide Extension</B></P><TABLE border="1"><TBODY><TR><TH> name
</TH><TH> oligonucleotide set
</TH><TH> Notes
</TH></TR><TR><TD>oriT1_ob._na.1
</TD><TD>ctagaggaataagggacagtgaagaaggaacacccgctcg
</TD><TD>oriT1 + oriT4 = construct 1
</TD></TR><TR><TD>oriT2_ob._na.1
</TD><TD>cgggtgggcctacttcacctatcctgcccggctgacgccg
</TD><TD>oriT2 + oriT5 = construct 2
</TD></TR><TR><TD>oriT3_ob._na.1
</TD><TD>ttggatacaccaaggaaagtctacatactagtagcggccgctgca
</TD><TD>oriT3 + oriT5 = construct 3
</TD></TR><TR><TD>oriT4_ob._na.1
</TD><TD>GCGGCCGCTACTAGTAtgtagactttccttggtg
</TD></TR><TR><TD>oriT5_ob._na.1
</TD><TD>tatccaacggcgtcagccgggcaggataggtgaagtaggcc
</TD></TR><TR><TD>oriT6_ob._na.1
</TD><TD>cacccgcgagcgggtgttccttcttcactgtcccttattcCT
</TD></TR></TBODY></TABLE><P><STRONG>DNA Ligation</STRONG></P><OL><LI>Combine 4 ul of construct 1 with 4 ul of construct 2.  
</LI><LI>Add 1 ul of nanopure H<SUB>2</SUB>0.
</LI><LI>Add 10 ul of 2x quick ligation reaction Buffer.  In order to avoid shearing the DNA, mix by resuspending very slowly.
</LI><LI>Add 1 ul of Quick T4 DNA ligase and mix thoroughly by resuspending very slowly.
</LI><LI>Incubate at room temperature for 5 minutes. 
</LI><LI>Cool on ice 
</LI><LI>Restriction digest with XbaI.
<DL><DD> 0.75uL dH<SUB>2</SUB>O, 2.5uL 10x Buffer, 0.25uL 100x BSA, 20uL ligation product, 1uL XbaI. Combine in tube, incubate 1 hour 37C, followed by heat inactivation 20 minutes at 65C.
</DD></DL></LI><LI>Gel purification in 4% gel. 
</LI><LI>Cut band and purify from gel with Minelute Gel Extraction kit from Qiagen.
</LI><LI>Resulting product is (construct 1 + construct 2)= ligation product 1.
</LI></OL><OL><LI>Combine 4 ul of construct 3 with 4 ul of ligation product 1.  
</LI><LI>Add 1 ul of nanopure H<SUB>2</SUB>0.
</LI><LI>Add 10 ul of 2x quick ligation reaction Buffer.  In order to avoid shearing the DNA, mix by resuspending very slowly.
</LI><LI>Add 1 ul of Quick T4 DNA ligase and mix thoroughly by resuspending very slowly.
</LI><LI>Incubate at room temperature for 5 minutes. 
</LI><LI>Cool on ice 
</LI><LI>Restriction digest with PstI and XbaI.
<DL><DD> 0.25uL dH<SUB>2</SUB>O, 2.5uL 10x NEBuffer3, 0.25uL 100x BSA, 20uL ligation product, 1uL XbaI, 1uL PstI. Combine in tube, incubate 1 hour 37C, followed by heat inactivation 20 minutes at 65C.
</DD></DL></LI><LI>Gel purification in 4% gel. 
</LI><LI>Cut band and purify from gel with Minelute Gel Extraction kit from Qiagen.
</LI><LI>Resulting product is (construct 3 + ligation product 1)= ligation product 2.
</LI></OL><DL><DD><I>Reference: Quick Ligation Kit from NEB.</I></DD></DL><OL><LI>Purify ligation product 2 from gel, store at -20C. Use in a ligation reaction with appropriate vector.
</LI></OL><H2><SPAN class="mw-headline" id="Results_2">Results</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><STRONG>Ligation Product 1:</STRONG> A tri-dye 2-log ladder was run in lane 1, 10uL of constructs 1 and 2 were run in lanes 3 and 5 respectively and in lane 7, 20uL of ligation product 1 was run.</DIV></DIV></DIV><DL><DD><OL><LI> Ligation product 1 can be seen in the following figure.  A tri-dye 2-log ladder was run in lane 1, 10uL of constructs 1 and 2 were run in lanes 3 and 5 respectively and in lane 7, 20uL of ligation product 1 was run.
</LI><LI> For the construction of Ligation product 2, the first ligation product was purified from a gel. Additionally, ligation product was assembled again in a reaction, the plasmid was precipitated (with isopropanol and cleaned with 70% ethanol)after the XbaI digestion. Both were used in separate ligation reactions. No picture of this gel was taken, but is summarized in the following table.
</LI></OL></DD></DL><P><B>Ligation 2 Results</B></P><TABLE border="1"><TBODY><TR><TH> Lane
</TH><TH> Contents
</TH><TH> Description
</TH></TR><TR><TD>1
</TD><TD>Tri-dye 2-log ladder
</TD><TD>Faint, but distinct bands.
</TD></TR><TR><TD>2
</TD><TD>Empty
</TD></TR><TR><TD>3
</TD><TD>construct 1
</TD></TR><TR><TD>4
</TD><TD>construct 2
</TD></TR><TR><TD>5
</TD><TD>construct 3
</TD></TR><TR><TD>6
</TD><TD>empty
</TD></TR><TR><TD>7
</TD><TD>ligation product 2, with ligation product 1, gel purified
</TD><TD>NO BAND!!!
</TD></TR><TR><TD>8
</TD><TD>empty
</TD></TR><TR><TD>9
</TD><TD>ligation product2, no gel
</TD><TD>faint band
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Discussion_2">Discussion</SPAN></H2><DL><DD><OL><LI> After the first ligation, the ligation product is digested with XbaI, and a single band, moderately bright band comes out. After the second ligation, using PstI concurrently with XbaI, a ladder of bands appears. Does this mean we need to use one enzyme at a time? Grace is doing this experiment today (7/14/08). 
</LI><LI> The gel purified ligation product was not visualized on the gel. Does this mean our kit is no good? Can we optimize the experiment so that gel purification is not necessary?
</LI></OL></DD></DL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Hawaii/Initial_Synth._Oligo_Assembly">http://2008.igem.org/Team:Hawaii/Initial_Synth._Oligo_Assembly</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hawaii/Initial_Synth._Oligo_Assembly" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hawaii/Initial_Synth._Oligo_Assembly" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hawaii/Initial_Synth._Oligo_Assembly&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hawaii/Initial_Synth._Oligo_Assembly&amp;oldid=21297" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>