"
Page
Discussion
View source
History
teams
Log in
 
Team:Hawaii/Norman's Primer Design Notes
From 2008.igem.org
Contents
1 Method Variation Comparisons
1.1 Alignment Of Various Methods
1.2 OpenWetware Wiki Referenced as of 6/16/2008
1.3 Notes:
2 Excerpt of Various Methods
2.1 Use this approach for promoters, ribosome binding sites, terminators and most other BioBricks parts.
2.2 Use this approach for proteins
2.3 (WRONG INSTRUCTION: suffix primer needs to be reverse complemented)
2.4 Silver Lab PCR Strategy
2.5 Silver Lab Oligo Strategy (total chemical synthesis of DNA)
Method Variation Comparisons
Alignment Of Various Methods
Forward 5’-3’                                                     EcoRI  NotI       XbaI                                      SpeI     NotI    PstI [1] defined by Knight 2003                          GAATTC GCGGCCGC T TCTAGA G [                            ] T ACTAGT A GCGGCCG CTGCAG [2] wetware RBS & TT                       gtttcttc GAATTC GCGGCCGC T TCTAGA G [                            ] [3] wetware Protein                        gtttcttc GAATTC GCGGCCGC T TCTAG    [ATG start                   ] [4] wetware ("Designing_Primers")                               cct T TCTAGA G [                            ] [5] silver PCR ends (fusion brick)                              cct T TCTAGA   [                            ] [6] silver Oligo Synthesis (fusion brick)                              CTAGA   [fwd coding sequence         ]   ACTAGT A GCGGCCG CTGCA         Reverse 5’-3’                                                     PstI   NotI      SpeI                                       XbaI     NotI     EcoRI [1] defined by Knight 2003                          CTGCAG CGGCCGC T ACTAGT A [                             ] C TCTAGA A GCGGCCGC GAATTC [2] wetware RBS & TT                       gtttcttc CTGCAG CGGCCGC T ACTAGT A [                             ]  [3] wetware Protein                        gtttcttc CTGCAG CGGCCGC T ACTAGT A [TTA TTA double stop codon    ] [4] wetware ("Designing_Primers")              aagg CTGCAG CGGCCGC T ACTAGT A [                             ]  [5] silver PCR ends (fusion brick)             aagg CTGCAG CGGCCGC T ACTAGT   [                             ]  [6] silver Oligo Synthesis (fusion brick)                G CGGCCGC T ACTAGT   [rvc coding sequence          ]   T
OpenWetware Wiki Referenced as of 6/16/2008
[1] http://dspace.mit.edu/handle/1721.1/21168 (has error in XbaI site, reverse complement should be AGATCT instead of ACATCT)
[2] http://openwetware.org/wiki/Synthetic_Biology:BioBricks/Part_fabrication#Quick_reference
[3] http://openwetware.org/wiki/Synthetic_Biology:BioBricks/Part_fabrication#Quick_reference_2
[4] http://openwetware.org/wiki/Designing_primers#BioBrick_primers
[5] http://openwetware.org/wiki/Silver:_PCR
[6] http://openwetware.org/wiki/Silver:_Oligonucleotide_Inserts
[7] http://partsregistry.org/Help:BioBrick_Prefix_and_Suffix
[8] http://openwetware.org/wiki/Cfrench:bbprimerdesign
Notes:
lowercase letters denote unknown (reason?) extensions
spacers (*) exist between: NotI * XbaI * [gene] * SpeI * NotI, fusion bricks are missing spacers flaking [gene]
wetware protein definition removed AG before the gene. last A of XbaI overlaps with ATG of start codon; G is a spacer and is removed.
NotI recognition/cut site is 8 letters; the second NotI site in BioBrick Definition is only 7 letters because its last C overlaps with first C of the following PstI
{fusion brick} It is important to note that the spacer nucleotides between the part and the XbaI and SpeI sites were originally incorporated in order to prevent DNA methylation and its consequential inhibition of the required restriction enzymes. These spacer nucleotides are not present in the assembly strategy presented here {silver fusion brick}. Hence, it is required to prescreen parts to ensure that methylation sites are not incorporated. Dam methylation of the XbaI site is the most critical. Parts beginning with the sequences TCx create a site capable of inhibiting XbaI. If a desired part naturally begins with this sequence (coding for serine), it is necessary to change the first codon to either AGT or AGC (both also encoding serine).
Excerpt of Various Methods
Use this approach for promoters, ribosome binding sites, terminators and most other BioBricks parts.
http://openwetware.org/wiki/Synthetic_Biology:BioBricks/Part_fabrication#Quick_reference
Quick reference  Once you are ready to design your primers for making a BioBrick, you can copy and paste the following sequences into your primers.  Copy and paste the following 30 bp sequence onto the 5' end of your upstream primer: 5' ---> 3' GTT TCT TCG AAT TCG CGG CCG CTT CTA GAG  Copy and paste the following 29 bp sequence onto the 5' end of your downstream primer: 5' ---> 3' GTT TCT TCC TGC AGC GGC CGC TAC TAG TA
Use this approach for proteins
http://openwetware.org/wiki/Synthetic_Biology:BioBricks/Part_fabrication#Quick_reference_2
Construction of protein coding sequences in BioBricks form requires slightly specialized BioBricks prefixes and suffixes for two reasons.     1. The prefix is slightly altered to ensure appropriate spacing between the ribsome binding site and the start codon.    2. BioBricks coding sequences standardly end with two sequential TAA stop codons.       Quick reference  Once you are ready to design your primers for making a BioBrick, you can copy and paste the following sequences into your primers.  Copy and paste the following 31 bp sequence onto the 5' end of your upstream primer for your coding sequence: includes the ATG start codon! 5' ---> 3' GTT TCT TCG AAT TCG CGG CCG CTT CTA G [ATG start]  Copy and paste the following 35 bp sequence onto the 5' end of your downstream primer for your coding sequence: includes the TAATAA double stop codon! 5' ---> 3' GTT TCT TCC TGC AGC GGC CGC TAC TAG TA [TTA TTA double stop codon]
(WRONG INSTRUCTION: suffix primer needs to be reverse complemented)
http://openwetware.org/wiki/Designing_primers
To BioBrick a part, the following tails should be added to your primers:      * PREFIX Primer cctttctagag 11 bp     * SUFFIX Primer tactagtagcggccgctgcagcctt 25 bp   The prefix primer adds an Xba I site, and the suffix adds the entire BB suffix (Spe I-Not I-Pst I)  Check the annealing temperature both without the tail (the first cycle or so) and with the tail (the later cycles).
Silver Lab PCR Strategy
http://openwetware.org/wiki/Silver:_PCR
For typical BioBricks construction, you want a PCR product which has an XbaI site upstream of your part, and SpeI, NotI, and PstI sites downstream of your part. Then, the forward primer should be of the form: 5' CCTTTCTAGA (15-20 bp of the coding strand) 3' and the reverse primer should be of the form: 5' AAGGCTGCAGCGGCCGCTACTAGT (15-20 bp reverse complement) 3'
Silver Lab Oligo Strategy (total chemical synthesis of DNA)
http://openwetware.org/wiki/Silver:_Oligonucleotide_Inserts
Design of oligonucleotide inserts      * The sense oligo should be of the form:       5' CTAGA(coding sequence)ACTAGTAGCGGCCGCTGCA 3'     * The anti-sense oligo should be of the form:       5' GCGGCCGCTACTAGT(reverse complement of coding sequence)T 3'
Retrieved from " http://2008.igem.org/Team:Hawaii/Norman%27s_Primer_Design_Notes "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
