"
Page
Discussion
View source
History
teams
Log in
 
Team:Hawaii/Project/Part B
From 2008.igem.org
Contents
1 Expression System for Synechocystis sp. PCC6803
2 Introduction
3 Background
4 Objective
5 Proposed Materials and Methods
5.1 Step 1: Synthesis and assembly of the nirA promoter and pilA and slr2016 signal sequences
5.2 Step 2: Site-directed mutagenesis of GFP (BBa_E0040) into a fusion brick
5.3 Step 3: Conversion of pRL1383a into a Biobrick plasmid
5.4 Step 4: Device construction
5.5 Step 5: Testing for protein secretion
5.6 Controls
6 References
Expression System for Synechocystis sp. PCC6803
Introduction
Production of proteins with clinical or industrial value can be performed by bacterial cell  cultures. Once synthesized, however, problems may arise in recovering these recombinant  proteins in substantial yields. Protein recovery oftentimes requires the cells to be lysed in order to recover the protein of interest, thus limiting the amount of product to a one-time yield of the total biomass that can be maintained at a given time. Production can be optimized if multiple  yields can be collected over time from a living cell culture. [1] Ideally, this can be done by transporting the protein product out of the cell and collecting it from the growth medium.
Background
Cyanobacteria are defined by their ability to photosynthesize and can be engineered to synthesize substances of biotechnological interest.  Thus cyanobacteria provide the opportunity for autotrophic production of practically any biomolecule. The ability to extract engineered biomolecules will make this bacterium a renewable, nearly self-sustaining "factory" that may revolutionize current approaches to biological engineering. The Gram-positive bacterium Bacillus subtilis has been used to produce high quality industrial enzymes and a few eukaryotic proteins via secretion into the growth medium. [2] Synechocystis sp. PCC6803 is a Gram-negative cyanobacterium that has undergone continuous study.  The genome of Synechocystis has been completely sequenced [3], making it a model organism for the exploration of secretion in Gram-negative bacteria. [4]
The major obstacle for secretion of proteins in Gram-negative bacteria is the additional outer membrane with a cell wall made of peptidoglycan murein.  As a Gram-negative  bacteria, Synechocystis sp. PCC6803 possess a functional periplasm between its inner and outer membranes [5].  Proteins destined for extracellular secretion must first be  transported through both of these membranes and the periplasmic space.
Proteins are known to be secreted by bacteria via a number of different pathways. [6] The simplest way to  exploit PCC6803 as a protein production factory is to take advantage of naturally existing protein secretion pathways in the organism.
Seven distinct proteins have been found to be secreted by Synechocystis sp. PCC6803 into its culture medium under normal growth conditions. [7] Analysis of the amino-terminal sequences of these proteins led to the identification of protein secretion signal polypeptides by Sergeyenko and Los.  Additional studies have produced recombinant cyanobacterial strains that use PCC6803 signal peptides to secrete a foreign reporter protein, lichenase, into the culture  medium. [8]
Objective
To create BioBricks encoding signal peptides that can be combined with a protein coding sequence in order to express the protein of interest extracellularly.  The ability of the signal peptide to export a protein will be tested by combining it in a BioBrick device with a fusion BioBrick part for Green Fluorescent Protein (GFP) and a BioBrick part of the nitrate-inducible nirA promoter.
Proposed Materials and Methods
Step 1: Synthesis and assembly of the nirA promoter and pilA and slr2016 signal sequences
The nir promoter and the pilA and slr2016 secretion signal sequences will be syntheized with the standard Biobrick sites. Oligonucleotide fragments of each will be hybridized with its complement and ligated together to form whole, fully functional promoters and signal sequences. Assembly of these new Biobricks will be verified by gel electrophoresis as well as sequencing.
nir promoter
Oligonucleotide
Sequence
Length
pnir1_fb.syn.1
CTAGAGCTAAATGCGTAAACTGCATATGCCTTCGCTGAGTGTAATTTACGTTACA
55
pnir2_fb.syn.1
AATTTTAACGAAACGGGAACCCTATATTGATCTCTACTACTAGTAGCGGCCGCTGCA
57
pnir2_rb.syn.1
GCGGCCGCTACTAGTAGTAGAGATCAATATAGGGTTCCCGTTTCGTTAAAATTTGTAAC
59
pnir1_rb.syn.1
GTAAATTACACTCAGCGAAGGCATATGCAGTTTACGCATTTAGCT
45
Signal sequences
Oligonucleotide
Sequence
Length
pilA1_fp.syn.1
CTAGATGGCTAGTAATTTTAAATTCAAACTCCTCTCTCAAC
41
pilA2_ff.syn.1
TCTCCAAAAAACGGGCAGAAGGTGGTACTAGTAGCGGCCGCTGCA
45
pilA2_rf.syn.1
GCGGCCGCTACTAGTACCACCTTCTGCCCGTTTTTTGGAGAGTTGAG
47
pilA1_rp.syn.1
AGAGGAGTTTGAATTTAAAATTACTAGCCAT
31
slr2016-1_fp.syn.1
CTAGATGGCAGCAAAACAACTATGGAAAATTTTCAATC
38
slr2016-2_ff.syn.1
CTAGACCGATGAAGGGTGGAACTAGTAGCGGCCGCTGCA
39
slr2016-2_rf.syn.1
GCGGCCGCTACTAGTTCCACCCTTCATCG
29
slr2016-1_rp.syn.1
GTCTAGGATTGAAAATTTTCCATAGTTGTTTTGCTGCCAT
40
Step 2: Site-directed mutagenesis of GFP (BBa_E0040) into a fusion brick
GFP, as it currently exists in the Biobrick Registry of Parts , is a protein Biobrick, meaning that it will ligate out of frame with our signal sequence Biobricks. A primer will be designed for site-directed mutagenesis of the GFP start codon to convert BBa_E0040 into a fusion Biobrick.
Primer
Sequence
Length
G/C content
Tm
GFP (BBa_E0040) fusion / foward primer
GCCGCTTCTAGAcgtaaaggag
22 bp
54.55%
60.2 C
GFP (BBa_E0040) fusion / reverse primer
cgagtcagtgagcgaggaag
20 bp
60%
59.6
Step 3: Conversion of pRL1383a into a Biobrick plasmid
Part A of our project focuses on converting the RSF1010 based plasmid, pRL1383a into a sophisticated broad-host Biobrick plasmid. While we aim to ultimately express our secretion system in this new plasmid as part of a cyanobacterial expression system, we need a workable shuttle vector between E. coli (where constructs will be made) and PCC6803 (the ultimate host). Converting pRL1383a into a much simpler Biobrick plasmid will fulfill this requirement. Verification regions, transcriptional terminators, and the Biobrick multiple cloning site (MCS) will be isolated from the plasmid containing BBa_B0030 via PCR. PCR primers will also include HindIII and BamHI restriction sites for ligation into pRL1383a. This ligation will replace the original pRL1383a MCS which includes Biobrick and Biobrick compatible restriction sites. The MCS replacement will be verified by restriction digest and plasmid sequencing.
Primer
Sequence
Length
G/C content
Tm
Notes
HindIII-VF2BB
cctAAGCTTtgccacctgacgtctaagaa
29 bp (20 bp)
48.3% (50.0%)
65.9 C (58.6 C)
Includes RE extension HindIII site and three 5' nucleotides for efficient cutting. Parentheses indicate primer information w/o RE site and 3 nucleic acids. Based on VF2 primer.
BamHI-VRBB
ccaGGATCCattaccgcctttgagtgagc
29 bp (20 bp)
55.2% (50.0%)
67.9 C (58.0 C)
Includes RE extension BamHI site and three 5' nucleotides for efficient cutting. Parentheses indicate primer information w/o RE site and 3 nucleic acids. Based on VR primer.
Step 4: Device construction
Figure 1: Construct of proposed GFP secretion device.
The synthesized signal peptides and nirA promoter BioBricks will be combined with at least three existing BioBricks to create two (or more) nitrate-regulated protein secretion devices according to the scheme detailed in Figure 1.  The resulting devices will be placed in a Synechocystis compatible BioBrick vector derived from the RSF1010 derived plasmid pRL1383a. In the proposed devices, the signal peptides will be situated so they are in-frame with GFP.  The translated polypeptide should consist of a N-terminal signal polypeptide leader sequence attached to a fluorescent protein.
Step 5: Testing for protein secretion
The BioBrick vector can be inserted into Synechocystis sp. PCC6803 by triparental conjugation with E. coli harboring a transmissible plasmid (like RP1) and another E. coli containing our engineered plasmid. Plated Synechocystis sp. PCC6803 colonies successfully transformed wwill exhibit a glowing  halo of secreted GFP.  Transformed Synechocystis sp. PCC6803 grown in liquid media will result in fluorescent culture media.  The efficacy of the signal peptides in transporting GFP into the extracellular media can be measured using a spectrofluorometer.
Controls
A number of controls will also be constructed in parallel with our device to test device assembly at each step.
Construct
Control
nirA promoter
lac promoter (BBa_I14032)
nirA promoter + rbs (BBa_B0034)
nirA or lac promoter + rbs (BBa_B0034) + GFP (BBa_E0040) + txn term. (BBa_B0024)
IPTG induced GFP device (BBa_J04430)
nirA promoter + rbs (BBa_B0034) + pilA or slr2016 signal sequence + GFP fusion brick + txn term. (BBa_B0024)
nirA or lac promoter + rbs (BBa_B0034) + GFP (BBa_E0040) + txn term. (BBa_B0024) and nirA promoter + rbs (BBa_B0034) + lichenase (duplicating Sergeyenko construct)
References
[1] Levinson, P. “Protein Purification from Microbial Cell Culture.” Protein Purification Applications: A Practical Approach. 73-82 (1990).
[2] Zweers, J et al. “Towards the development of Bacillus subtilis as a cell factory for membrane proteins and protein complexes.” Microbial Cell Factories 7 (2008).
[3] Synechocystis sp. PCC6803, complete genome. NCBI. 9 June 2008. < http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=NC_000911 >.
[4] Kaneko T et al. “Sequence analysis of the genome of the unicellular cyanobacterium Synechocysitis sp. Strain PCC 6803. II. Sequence determination of the entire genome and  assignment of potential protein-coding regions.” DNA Research 3, 109-136 (1996).
[5] Mackle, M.M. and Zilinskas, B.A. “Role of Signal Peptides in Targeting of Proteins” Journal of  Bacteriology 4: 1857-1864 (1994).
[6] Filloux A et al. “Protein secretion in gram-negative bacteria: transport across the outer  membrane involves common mechanisms in different bacteria.” Dec 9(13):4323-9 (1990).
[7] Sergeyenko, T.V. and Los, D.A. Identification of secreted proteins of the cyanobacterium Synechocystis sp. strain PCC 6803." FEMS Microbiol. Lett. 193: 213-216 (2000).
[8] Sergeyenko, T.V. and Los, D.A. "Cyanobacterial leader peptides for protein secretion." FEMS Microbiol. Lett. 218: 351-357 (2003).
[9] Ivanikova, N. V. “Lake Superior Phototrophic picoplankton: Nitrate Assimilation measure with  cyanobacterial nitrate-responsive bioreporter and genetic diversity of the natural community.” 9 June 2008. <www.ohiolink.edu/etd/send-pdf.cgi?acc_num=bgsu1142559572>
Retrieved from " http://2008.igem.org/Team:Hawaii/Project/Part_B "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
