"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Hawaii/Primer Design for BioBricking
From 2008.igem.org
Contents
1 U. Hawaii BioBrick Primer Design Procedure
1.1 1. Assess BioBrickability
1.2 2. Make Primers
1.3 3. Attach BioBrick Ends
1.3.1 Back-Insert Primers (X-SP, preferred)
1.3.1.1 Prefix [X]
1.3.1.2 Suffix [SP]
1.3.2 Front-Insert Primers (EX-S, not preferred)
1.3.2.1 Prefix [EX]
1.3.2.2 Suffix [S]
1.3.3 Back-Insert De Novo Synthesis [prefix]-[gene]-[suffix]
1.4 4. Error Checking
2 Tools
3 Design Notes
4 References
U. Hawaii BioBrick Primer Design Procedure
1. Assess BioBrickability
Assess gene BioBricking feasibility.  Analyze sequence for existing BioBrick sites to see if additional processing is needed before designing BioBricking primers. site removal reference
These_sites_MUST_be_absent [1]
EcoRI, XbaI, SpeI, PstI, NotI
Strongly prefer that these sites be absent [2]
ApoI (probably impossible because it occurs frequently), MfeI, AvrII, NheI, NsiI, SbfI
Would prefer that these sites be absent [3]
Offset cutters
AarI, AcuI (medium priority), BbsI, BbvCI, BciVI (would be nice, medium priority), BfuAI (high priority), BmrI (high priority), BsmBI, BsaI, BsgI (medium priority), BsmI (includes nicking enzyme, high priority), BspMI, BsrDI (includes nicking enzyme, high priority), BtgZI, EarI, EcoP15I (high priority), FokI (best effort), Nt.BstNBI, SapI (high priority, should already be eliminated from EarI), TspRI (probably difficult, best effort)
Nicking enzymes
Nt.BstNBI, BbvCI, Nt.AlwI (best effort to at least remove sites near each other)
Homing endonucleases
I-CeuI, I-SceI, PI-PspI, PI-SceI, I-PpoI
Others
AgeI, AscI, FseI, RsrII, SgrAI, XmnI, XcmI
Would be convenient if these sites were removed [4]
These are common, efficient cutters that people might want to use.
HindIII, BamHI, XhoI, NcoI, SacI, NdeI
Here are other low priority cut sites to remove
KasI, MssI, NgoMIV, PacI, PmeI, SalI, SfiI, SgfI, SmiI, SrfI, SwaI, XmaI, ZraI
Sites to include [5]
Having GATC sites in your part can sometimes be useful because it allows plasmid purified DNA to be cut (via DpnI) whereas PCR product DNA is not. Such a trick is useful for some site directed mutagenesis protocols.
Miscellaneous [6]
BaeI is an enzyme whose is exact cut position is unknown. It is explicitly included in some pSB plasmid replications origins so that the plasmid can optionally be destroyed via another enzyme. You may want to eliminate this site from your part if you intend to use this feature of the BioBricks plasmids.
2. Make Primers
Pick first ~20 and last ~20(reverse complement) residues of full length desired region for making primers, adjust cut length at 3' boundry according to calculated Tm. (target for 60C Tm for first-cycle annealing to your target DNA). Account for necessary adjustments such as replacing stop codon with TAATAA (where replaced region no longer match/anneal to template) . Only calculate Tm for region that is annealing to target site in the first cycle.
we would like to standardize Tm calculation algorithm used by all team members.
Please use: IDT OligoAnalyzer with the parameters: Oligo content of 0.4 uM, Na+ concentration of 50 mM, Mg++ concentration of 2mM, and a dNTP concentration of 2mM.
3. Attach BioBrick Ends
Attach biobrick ends accordingly, using the table below. Calculate the primer+extension Tm.
Back-Insert Primers (X-SP, preferred)
Prefix [X]
EcoRI  NotI       XbaI GAATTC GCGGCCGC T TCTAGA G  regular             cct T TCTAGA G [anything]  protein (A=overlaps XbaI A, T=spacer, G=n/a)             cct T TCTAG    [ATG]  fusion protein (skips spacer to be in-frame)             cct T TCTAGA   [no start ATG]  edinbricks?
Suffix [SP]
PstI   NotI      SpeI      CTGCAG CGGCCGC T ACTAGT A  regular aagg CTGCAG CGGCCGC T ACTAGT A [anything]  protein aagg CTGCAG CGGCCGC T ACTAGT A [TTA TTA double stop codon]  fusion protein (skips spacer to be in-frame) aagg CTGCAG CGGCCGC T ACTAGT   [no stop codon]  edinbricks?
Front-Insert Primers (EX-S, not preferred)
Prefix [EX]
EcoRI  NotI       XbaI at GAATTC GCGGCCGC T TCTAGA G  regular at GAATTC GCGGCCGC T TCTAGA G [anything]  protein (A=overlaps XbaI A, T=spacer, G=n/a) at GAATTC GCGGCCGC T TCTAG    [ATG]  fusion protein (skips spacer to be in-frame) at GAATTC GCGGCCGC T TCTAGA   [no start ATG]  edinbricks?
Suffix [S]
PstI   NotI      SpeI      CTGCAG CGGCCGC T ACTAGT A  regular                   c T ACTAGT A [anything]  protein                   c T ACTAGT A [TTA TTA double stop codon]  fusion protein (skips spacer to be in-frame)                   c T ACTAGT   [no stop codon]  edinbricks?
Back-Insert De Novo Synthesis [prefix]-[gene]-[suffix]
Forward Strand: 5'-3'
de novo synthesis regular                    CTAGA G [anything      ] T ACTAGT A GCGGCCG CTGCA                        - -  --------------  - ------ - ------- - de novo synthesis protein                    CTAG    [ATG-----TAATAA] T ACTAGT A GCGGCCG CTGCA                             --------------  - ------ - ------- - de novo synthesis fusionbricks                    CTAGA   [nostart-nostop]   ACTAGT A GCGGCCG CTGCA                        -    --------------    ------ - ------- - de novo synthesis endinbricks
Reverse Strand: 5'-3'
de novo synthesis regular           G CGGCCGC T ACTAGT A [anything      ] C T       ----- ------- - ------ -  --------------  - ----- de novo synthesis protein           G CGGCCGC T ACTAGT A [TTATTA-----CAT]        ----- ------- - ------ -  --------------     ---- de novo synthesis fusionbricks           G CGGCCGC T ACTAGT   [nostart-nostop]   T       ----- ------- - ------    --------------    ----- de novo synthesis endinbricks
4. Error Checking
use IDT OligoAnalyzer
hairpins
self-dimer
hetero-dimers
Tools
IDT OligoAnalyser OligoAnalyzer
Oligo content of 0.4 uM
Na+ concentration of 50 mM
Mg++ concentration of 2mM
dNTP concentration of 2mM
Alternative Tools
oligoTM (web interface to primer3 package's oligotm algorithm)
Nifty alternative tool to calculate Tm for oligos (thanks Grace!)
Design Notes
Norman's Primer Design Notes
Adam's Primer Design Notes
References
Idempotent Vector Design for Standard Assembly of BioBricks by Tom Knight
BioBrick Primer Design Notes by Chris French
RBS & Protein Coding Sequence biobricking primer design considerations
Syntheticbiology.org Biobrick Part Fabrication notes
NEB: Number of extra spacer nucleotides needed for efficient restriction digestion
Retrieved from " http://2008.igem.org/Team:Hawaii/Primer_Design_for_BioBricking "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
