<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hawaii_Primer_Design_for_BioBricking"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Hawaii/Primer_Design_for_BioBricking">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hawaii/Primer_Design_for_BioBricking&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/Primer_Design_for_BioBricking&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/Primer_Design_for_BioBricking&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hawaii/Primer_Design_for_BioBricking" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hawaii/Primer Design for BioBricking</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#U._Hawaii_BioBrick_Primer_Design_Procedure"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">U. Hawaii BioBrick Primer Design Procedure</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#1._Assess_BioBrickability"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">1. Assess BioBrickability</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#2._Make_Primers"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">2. Make Primers</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#3._Attach_BioBrick_Ends"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">3. Attach BioBrick Ends</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Back-Insert_Primers_.28X-SP.2C_preferred.29"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Back-Insert Primers (X-SP, preferred)</SPAN></A></LI><LI class="toclevel-4 tocsection-6"><A href="#Prefix_.5BX.5D"><SPAN class="tocnumber">1.3.1.1</SPAN><SPAN class="toctext">Prefix [X]</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#Suffix_.5BSP.5D"><SPAN class="tocnumber">1.3.1.2</SPAN><SPAN class="toctext">Suffix [SP]</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-8"><A href="#Front-Insert_Primers_.28EX-S.2C_not_preferred.29"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Front-Insert Primers (EX-S, not preferred)</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Prefix_.5BEX.5D"><SPAN class="tocnumber">1.3.2.1</SPAN><SPAN class="toctext">Prefix [EX]</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Suffix_.5BS.5D"><SPAN class="tocnumber">1.3.2.2</SPAN><SPAN class="toctext">Suffix [S]</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Back-Insert_De_Novo_Synthesis_.5Bprefix.5D-.5Bgene.5D-.5Bsuffix.5D"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Back-Insert De Novo Synthesis [prefix]-[gene]-[suffix]</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#4._Error_Checking"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">4. Error Checking</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Tools"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Tools</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Design_Notes"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Design Notes</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="U._Hawaii_BioBrick_Primer_Design_Procedure"> U. Hawaii BioBrick Primer Design Procedure </SPAN></H3><H4><SPAN class="mw-headline" id="1._Assess_BioBrickability"> 1. Assess BioBrickability </SPAN></H4><UL><LI> Assess gene BioBricking feasibility.  Analyze sequence for existing BioBrick sites to see if additional processing is needed before designing BioBricking primers. <A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#These_sites_MUST_be_absent" class="external text" rel="nofollow">site removal reference</A></LI></UL><DL><DD><B>These_sites_MUST_be_absent</B><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#These_sites_MUST_be_absent" class="external autonumber" rel="nofollow">[1]</A></DD><DD><PRE>EcoRI, XbaI, SpeI, PstI, NotI</PRE></DD></DL><DL><DD><B>Strongly prefer that these sites be absent</B><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#Strongly_prefer_that_these_sites_be_absent" class="external autonumber" rel="nofollow">[2]</A></DD><DD> ApoI (probably impossible because it occurs frequently), MfeI, AvrII, NheI, NsiI, SbfI 
</DD></DL><DL><DD><B>Would prefer that these sites be absent</B><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#Would_prefer_that_these_sites_be_absent" class="external autonumber" rel="nofollow">[3]</A></DD><DD> Offset cutters
</DD><DD><PRE>AarI, AcuI (medium priority), BbsI, BbvCI, BciVI (would be nice, medium priority), BfuAI (high priority), BmrI (high priority), BsmBI, BsaI, BsgI (medium priority), BsmI (includes nicking enzyme, high priority), BspMI, BsrDI (includes nicking enzyme, high priority), BtgZI, EarI, EcoP15I (high priority), FokI (best effort), Nt.BstNBI, SapI (high priority, should already be eliminated from EarI), TspRI (probably difficult, best effort)</PRE></DD><DD> Nicking enzymes
</DD><DD><PRE>Nt.BstNBI, BbvCI, Nt.AlwI (best effort to at least remove sites near each other)</PRE></DD><DD> Homing endonucleases
</DD><DD><PRE>I-CeuI, I-SceI, PI-PspI, PI-SceI, I-PpoI</PRE></DD><DD> Others
</DD><DD><PRE>AgeI, AscI, FseI, RsrII, SgrAI, XmnI, XcmI</PRE></DD></DL><DL><DD><B>Would be convenient if these sites were removed</B><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#Would_be_convenient_if_these_sites_were_removed" class="external autonumber" rel="nofollow">[4]</A></DD><DD> These are common, efficient cutters that people might want to use.
</DD><DD><PRE> HindIII, BamHI, XhoI, NcoI, SacI, NdeI </PRE></DD><DD> Here are other low priority cut sites to remove
</DD><DD><PRE>KasI, MssI, NgoMIV, PacI, PmeI, SalI, SfiI, SgfI, SmiI, SrfI, SwaI, XmaI, ZraI</PRE></DD></DL><DL><DD><B>Sites to include</B><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#Sites_to_include" class="external autonumber" rel="nofollow">[5]</A></DD><DD> Having GATC sites in your part can sometimes be useful because it allows plasmid purified DNA to be cut (via DpnI) whereas PCR product DNA is not. Such a trick is useful for some site directed mutagenesis protocols. 
</DD></DL><DL><DD><B>Miscellaneous</B><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html#Miscellaneous" class="external autonumber" rel="nofollow">[6]</A></DD><DD> BaeI is an enzyme whose is exact cut position is unknown. It is explicitly included in some pSB plasmid replications origins so that the plasmid can optionally be destroyed via another enzyme. You may want to eliminate this site from your part if you intend to use this feature of the BioBricks plasmids.
</DD></DL><H4><SPAN class="mw-headline" id="2._Make_Primers"> 2. Make Primers </SPAN></H4><UL><LI> Pick first ~20 and last ~20(reverse complement) residues of full length desired region for making primers, adjust cut length at 3' boundry according to calculated Tm. (target for 60C Tm for first-cycle annealing to your target DNA). Account for necessary adjustments such as replacing stop codon with TAATAA (where replaced region no longer match/anneal to template) . Only calculate Tm for region that is annealing to target site in the first cycle.
</LI><LI> we would like to standardize Tm calculation algorithm used by all team members. 
</LI></UL><P>Please use:<A href="http://scitools.idtdna.com/analyzer/Applications/OligoAnalyzer/" class="external text" rel="nofollow">IDT OligoAnalyzer</A>
with the parameters: Oligo content of 0.4 uM, Na+ concentration of 50 mM, Mg++ concentration of 2mM, and a dNTP concentration of 2mM.
</P><H4><SPAN class="mw-headline" id="3._Attach_BioBrick_Ends"> 3. Attach BioBrick Ends </SPAN></H4><UL><LI> Attach biobrick ends accordingly, using the table below. Calculate the primer+extension Tm.
</LI></UL><H5><SPAN class="mw-headline" id="Back-Insert_Primers_.28X-SP.2C_preferred.29"> Back-Insert Primers (X-SP, preferred) </SPAN></H5><H6><SPAN class="mw-headline" id="Prefix_.5BX.5D"> Prefix [X] </SPAN></H6><PRE>
EcoRI  NotI       XbaI
GAATTC GCGGCCGC T TCTAGA G

regular
            cct T TCTAGA G [anything]

protein (A=overlaps XbaI A, T=spacer, G=n/a)
            cct T TCTAG    [ATG]

fusion protein (skips spacer to be in-frame)
            cct T TCTAGA   [no start ATG]

edinbricks?

</PRE><H6><SPAN class="mw-headline" id="Suffix_.5BSP.5D"> Suffix [SP] </SPAN></H6><PRE>
     PstI   NotI      SpeI
     CTGCAG CGGCCGC T ACTAGT A

regular
aagg CTGCAG CGGCCGC T ACTAGT A [anything]

protein
aagg CTGCAG CGGCCGC T ACTAGT A [TTA TTA double stop codon]

fusion protein (skips spacer to be in-frame)
aagg CTGCAG CGGCCGC T ACTAGT   [no stop codon]

edinbricks?

</PRE><H5><SPAN class="mw-headline" id="Front-Insert_Primers_.28EX-S.2C_not_preferred.29"> Front-Insert Primers (EX-S, not preferred) </SPAN></H5><H6><SPAN class="mw-headline" id="Prefix_.5BEX.5D"> Prefix [EX] </SPAN></H6><PRE>
   EcoRI  NotI       XbaI
at GAATTC GCGGCCGC T TCTAGA G

regular
at GAATTC GCGGCCGC T TCTAGA G [anything]

protein (A=overlaps XbaI A, T=spacer, G=n/a)
at GAATTC GCGGCCGC T TCTAG    [ATG]

fusion protein (skips spacer to be in-frame)
at GAATTC GCGGCCGC T TCTAGA   [no start ATG]

edinbricks?

</PRE><H6><SPAN class="mw-headline" id="Suffix_.5BS.5D"> Suffix [S] </SPAN></H6><PRE>
     PstI   NotI      SpeI
     CTGCAG CGGCCGC T ACTAGT A

regular
                  c T ACTAGT A [anything]

protein
                  c T ACTAGT A [TTA TTA double stop codon]

fusion protein (skips spacer to be in-frame)
                  c T ACTAGT   [no stop codon]

edinbricks?

</PRE><H5><SPAN class="mw-headline" id="Back-Insert_De_Novo_Synthesis_.5Bprefix.5D-.5Bgene.5D-.5Bsuffix.5D"> Back-Insert De Novo Synthesis [prefix]-[gene]-[suffix] </SPAN></H5><P>Forward Strand: 5'-3'
</P><PRE>
de novo synthesis regular
                   CTAGA G [anything      ] T ACTAGT A GCGGCCG CTGCA
                       - -  --------------  - ------ - ------- -
de novo synthesis protein
                   CTAG    [ATG-----TAATAA] T ACTAGT A GCGGCCG CTGCA
                            --------------  - ------ - ------- -
de novo synthesis fusionbricks
                   CTAGA   [nostart-nostop]   ACTAGT A GCGGCCG CTGCA
                       -    --------------    ------ - ------- -
de novo synthesis endinbricks

</PRE><P>Reverse Strand: 5'-3'
</P><PRE>
de novo synthesis regular
          G CGGCCGC T ACTAGT A [anything      ] C T
      ----- ------- - ------ -  --------------  - -----
de novo synthesis protein
          G CGGCCGC T ACTAGT A [TTATTA-----CAT] 
      ----- ------- - ------ -  --------------     ----
de novo synthesis fusionbricks
          G CGGCCGC T ACTAGT   [nostart-nostop]   T
      ----- ------- - ------    --------------    -----
de novo synthesis endinbricks

</PRE><H4><SPAN class="mw-headline" id="4._Error_Checking"> 4. Error Checking </SPAN></H4><OL><LI> use <A href="http://scitools.idtdna.com/analyzer/Applications/OligoAnalyzer/" class="external text" rel="nofollow">IDT OligoAnalyzer</A></LI><LI> hairpins
</LI><LI> self-dimer
</LI><LI> hetero-dimers
</LI></OL><H3><SPAN class="mw-headline" id="Tools"> Tools </SPAN></H3><UL><LI> IDT OligoAnalyser <A href="http://scitools.idtdna.com/analyzer/Applications/OligoAnalyzer/IDT" class="external text" rel="nofollow">OligoAnalyzer</A></LI><LI> Oligo content of 0.4 uM 
</LI><LI> Na+ concentration of 50 mM
</LI><LI> Mg++ concentration of 2mM
</LI><LI> dNTP concentration of 2mM
</LI></UL><LI> Alternative Tools
</LI><LI><A href="http://bioinfo.ut.ee/oligotm/" class="external text" rel="nofollow">oligoTM</A> (web interface to primer3 package's oligotm algorithm)
</LI><LI><A href="http://www.promega.com/biomath/calc11.htm" class="external text" rel="nofollow">Nifty alternative tool</A> to calculate Tm for oligos (thanks Grace!)
</LI><H3><SPAN class="mw-headline" id="Design_Notes"> Design Notes </SPAN></H3><UL><LI><A href="/Team:Hawaii/Norman%27s_Primer_Design_Notes" title="Team:Hawaii/Norman's Primer Design Notes">Norman's Primer Design Notes</A></LI><LI><A href="/Tean:Hawaii/Adam%27s_Primer_Design_Notes" title="Tean:Hawaii/Adam's Primer Design Notes" class="mw-redirect">Adam's Primer Design Notes</A></LI></UL><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><UL><LI><A href="http://hdl.handle.net/1721.1/21168" class="external text" rel="nofollow">Idempotent Vector Design for Standard Assembly of BioBricks</A> by <A href="http://openwetware.org/wiki/Tom_Knight" class="external text" rel="nofollow">Tom Knight</A></LI><LI><A href="http://openwetware.org/wiki/Cfrench:bbprimerdesign" class="external text" rel="nofollow">BioBrick Primer Design Notes by Chris French</A></LI><LI><A href="http://partsregistry.org/cgi/htdocs/Assembly/rbs_cds.cgi" class="external text" rel="nofollow">RBS &amp; Protein Coding Sequence biobricking primer design considerations</A></LI><LI><A href="http://syntheticbiology.org/BioBricks/Part_fabrication.html" class="external text" rel="nofollow">Syntheticbiology.org Biobrick Part Fabrication notes</A></LI><LI><A href="http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/cleavage_linearized_vector.asp" class="external text" rel="nofollow">NEB: Number of extra spacer nucleotides needed for efficient restriction digestion</A></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Hawaii/Primer_Design_for_BioBricking">http://2008.igem.org/Team:Hawaii/Primer_Design_for_BioBricking</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hawaii/Primer_Design_for_BioBricking" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hawaii/Primer_Design_for_BioBricking" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hawaii/Primer_Design_for_BioBricking&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hawaii/Primer_Design_for_BioBricking&amp;oldid=76200" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>