"
Page
Discussion
View source
History
teams
Log in
 
Team:Hawaii/Construction of Omega Interposon BioBrick
From 2008.igem.org
 Birds Eye
 View
Team
Projects
Events
Resources
Sponsors
Experiments
Milestones
Protocols
Notebook ( t )
Meetings ( t )
Contents
1 Construction of Omega Interposon BioBrick
2 Methods
3 Results
4 Discussion
4.1 8-14 experiment
4.2 8-18
4.3 8-19
4.4 8-20
Construction of Omega Interposon BioBrick
The omega interposon (OI) BioBrick will be constructed using insertional mutagenesis methods. OI will be inserted into the ccdB region of pSB1A3 by restricting both pSB1A3 (containing ccdB) and pSMC121-a plasmid containing OI- with either XmaI or SmaI, depending on which enzyme is more available.
The omega interposon is an insertional mutagenesis tool containing the aadA gene from plasmid R1001.1 which confers Spectinomycin and Streptomycin resistance. Flanking the aadA gene are transcriptional termination sites of the T4 gene 32 so that transcription cannot be achieved through the omega interposon from either side. To avoid polypeptide synthesis at the position of the omega interposon, synthetic translational stop codons were also included. Flanking this feature are two polylinkers (Prentki 1983). OI can be used in experiments where disruption of a gene is needed. These experiments are often performed when discerning the function of a gene.
Methods
Cut pSMC121 with SmaI; cut pSB1A2 with the same enzyme.
50uL reaction volume: 5uL reaction buffer, 1ul SmaI, 0.5uL BSA, Xul DNA, Yul water.
Reaction conditions: 25°C for 5 hours, heat inactivated at 65°C for 20 min
Ligate the omega interposon with pSB1A2:
20ul reaction: 10ul 2X reaction buffer,
Transformation:
Transform DH5-a with the ligated product, select on Amp100.
Transform DB3.1(ccdB resistant strain) with the ligated product, select on Amp100 as a negative control. If the experimental plate has no colonies and the (-) control does, this means that the intramolecular ligation of pSB1A3 was  the favored reaction.
PCR verification of the insert with VF2 and VR primers.
If PCR verification is unclear, plate cells on Sm& Sp LB plates, if they grow, they have the interposon.
Results
Transformation
DH5-alpha competent cells were transformed with the ligation product (pSB1A2+omega interposon), 43 colonies were counted the next day.
DB3.1 cells were also transformed with the ligation product.
DH5-alpha cells were transformed with pUC18.
PCR verification:
The bands were uniformly 1kb.
PCR verification of omega interposon ligated into pSB1A2
Plating on Sm&Sp plates:
No colonies grew for the ligation product
positive control, pSMC121 was plated and a lawn of colonies grew
negative control pSB1A2 were plated and no colonies grew
Discussion
All components of this experiment will be cut with the same restriction enzyme. This means there will probably be a large background of intramolecular ligations (pSB1A3 will ligate back to itself and pSMC121(minus the omega interposon)will ligate to itself as well). The negative control should let us know the degree to which this will occur.
8-14 experiment
Transformation:
43 colonies grew for the transformation of the ligation product into DH5-alpha cells. This means the ccdB gene was disrupted, or else the death proteins would have killed the cells.
The transformed DB3.1 cells did not grow on ampicilin which is curious. If the ligation was successful it would have grown in both of these strains.
That the DH5-alpha cells transformed with pUC18 did not grow makes me think there was something wrong with the transformation.
PCR verification
The band was consistently around 1kb. This is the size expected if the ccdB region was amplified.
Sm&Sp plating
It is clear that the cells transformed with the ligation product do not contain a functional version of the omega interposon because the cells showed no resistance to the antibiotics.
Conclusions:
I need to run a gel to verify that the omega interposon was digested properly away from its vector. This digestion product can be purified from the gel and another ligation can be performed.
8-18
Restriction digest of pSMC121 and pSB1A2 with SmaI.
Ran gel of restriction digest of omega interposon and pSB1A2. The bands were cut out and purified in preparation for another ligation.
Ligation was performed over-night at 4°C using T4 ligase (not the quick ligation kit). A 2:1 insert to vector ratio was used.
8-19
50ul DH5-alpha cells were transformed with 5ul ligation product from 8-18.
50ul DH5-alpha cells were transformed with no plasmid as a negative control.
pSB1A2 was ligated back to itself  and transformed into DH5-alpha cells. In DB3.1 cells, these plasmids should have transformed and the cells would survive on amp100 plates, but being that this ligation would possibly re-create the ccdB gene, these cells should not survive***the DB3.1 cells would have given better information about the experiment.
8-20
PCR verification that omega interposon was inserted into pSB1A2.
12 colonies grew on the amp100 plate, 4 of these were selected for PCR verification.
For both controls, 2 colonies grew. For the negative control this tells me that I will have some background from contamination. That may be the message from the pSB1A2 (mis-hap) control, or it may mean that the death gene was permanently disrupted.
PCR verification: colonies labeled omega1 & omega4 show bands which correspond to the size of the omega interposon, which is about 2kb. omega2 is about 250bp, the verification size of pSB1A2 with no insert is 238bp. omega3 has a band at 2kb, but it appears that some non-specific amplification is occurring. This could be the PCR reaction itself, or contamination of satellite colonies. The controls suggest that there is DNA contamination in the (-) control from the set up of the PCR. The (+) control, from pSB1A3 is the wrong size. It should be 991bp but is about 2kb. It also happens to be about the size of the omega interposon bands which appear to be correct. There could be contamination, or there might be errors setting up the experiment.
Next:
The colonies will be streaked on Sp/Sm plates. If there is a functional omega interposon, they should grow under these selective conditions.
Sequencing is another possible way to verify the constructs.
Retrieved from " http://2008.igem.org/Team:Hawaii/Construction_of_Omega_Interposon_BioBrick "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
