<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hawaii_Project_Part_B"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Hawaii/Project/Part_B">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hawaii/Project/Part_B&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/Project/Part_B&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hawaii/Project/Part_B&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hawaii/Project/Part_B" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hawaii/Project/Part B</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Expression_System_for_Synechocystis_sp._PCC6803"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Expression System for Synechocystis sp. PCC6803</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Background"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Objective"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Objective</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Proposed_Materials_and_Methods"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Proposed Materials and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Step_1:_Synthesis_and_assembly_of_the_nirA_promoter_and_pilA_and_slr2016_signal_sequences"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Step 1: Synthesis and assembly of the nirA promoter and pilA and slr2016 signal sequences</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Step_2:_Site-directed_mutagenesis_of_GFP_.28BBa_E0040.29_into_a_fusion_brick"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Step 2: Site-directed mutagenesis of GFP (BBa_E0040) into a fusion brick</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Step_3:_Conversion_of_pRL1383a_into_a_Biobrick_plasmid"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Step 3: Conversion of pRL1383a into a Biobrick plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Step_4:_Device_construction"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Step 4: Device construction</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Step_5:_Testing_for_protein_secretion"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Step 5: Testing for protein secretion</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Controls"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Controls</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Expression_System_for_Synechocystis_sp._PCC6803"> Expression System for <I>Synechocystis</I> sp. PCC6803</SPAN></H2><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>Production of proteins with clinical or industrial value can be performed by bacterial cell  cultures. Once synthesized, however, problems may arise in recovering these recombinant  proteins in substantial yields. Protein recovery oftentimes requires the cells to be lysed in order to recover the protein of interest, thus limiting the amount of product to a one-time yield of the total biomass that can be maintained at a given time. Production can be optimized if multiple  yields can be collected over time from a living cell culture. <SUP>[1]</SUP> Ideally, this can be done by transporting the protein product out of the cell and collecting it from the growth medium.
</P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>Cyanobacteria are defined by their ability to photosynthesize and can be engineered to synthesize substances of biotechnological interest.  Thus cyanobacteria provide the opportunity for autotrophic production of practically any biomolecule. The ability to extract engineered biomolecules will make this bacterium a renewable, nearly self-sustaining &quot;factory&quot; that may revolutionize current approaches to biological engineering. The Gram-positive bacterium <I>Bacillus subtilis</I> has been used to produce high quality industrial enzymes and a few eukaryotic proteins via secretion into the growth medium. <SUP>[2]</SUP><I>Synechocystis</I> sp. PCC6803 is a Gram-negative cyanobacterium that has undergone continuous study.  The genome of <I>Synechocystis</I> has been completely sequenced <SUP>[3]</SUP>, making it a model organism for the exploration of secretion in Gram-negative bacteria. <SUP>[4]</SUP></P><P>The major obstacle for secretion of proteins in Gram-negative bacteria is the additional outer membrane with a cell wall made of peptidoglycan murein.  As a Gram-negative  bacteria, <I>Synechocystis</I> sp. PCC6803 possess a functional periplasm between its inner and outer membranes <SUP>[5]</SUP>.  Proteins destined for extracellular secretion must first be  transported through both of these membranes and the periplasmic space.   
</P><P>Proteins are known to be secreted by bacteria via a number of different pathways. <SUP>[6]</SUP> The simplest way to  exploit PCC6803 as a protein production factory is to take advantage of naturally existing protein secretion pathways in the organism.  
</P><P>Seven distinct proteins have been found to be secreted by <I>Synechocystis</I> sp. PCC6803 into its culture medium under normal growth conditions. <SUP>[7]</SUP> Analysis of the amino-terminal sequences of these proteins led to the identification of protein secretion signal polypeptides by Sergeyenko and Los.  Additional studies have produced recombinant cyanobacterial strains that use PCC6803 signal peptides to secrete a foreign reporter protein, lichenase, into the culture  medium. <SUP>[8]</SUP></P><H2><SPAN class="mw-headline" id="Objective">Objective</SPAN></H2><P>To create BioBricks encoding signal peptides that can be combined with a protein coding sequence in order to express the protein of interest extracellularly.  The ability of the signal peptide to export a protein will be tested by combining it in a BioBrick device with a fusion BioBrick part for Green Fluorescent Protein (GFP) and a BioBrick part of the nitrate-inducible <I>nirA</I> promoter.
</P><H2><SPAN class="mw-headline" id="Proposed_Materials_and_Methods">Proposed Materials and Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Step_1:_Synthesis_and_assembly_of_the_nirA_promoter_and_pilA_and_slr2016_signal_sequences">Step 1: Synthesis and assembly of the <I>nirA</I> promoter and <I>pilA</I> and <I>slr2016</I> signal sequences</SPAN></H3><P>The <I>nir</I> promoter and the <I>pilA</I> and <I>slr2016</I> secretion signal sequences will be syntheized with the standard Biobrick sites. Oligonucleotide fragments of each will be hybridized with its complement and ligated together to form whole, fully functional promoters and signal sequences. Assembly of these new Biobricks will be verified by gel electrophoresis as well as sequencing. <B><I>nir</I> promoter</B></P><TABLE class="wikitable" align="center" border="1"><TBODY><TR><TH> Oligonucleotide
</TH><TH> Sequence
</TH><TH> Length
</TH></TR><TR><TD> pnir1_fb.syn.1
</TD><TD> CTAGAGCTAAATGCGTAAACTGCATATGCCTTCGCTGAGTGTAATTTACGTTACA
</TD><TD> 55
</TD></TR><TR><TD> pnir2_fb.syn.1
</TD><TD> AATTTTAACGAAACGGGAACCCTATATTGATCTCTACTACTAGTAGCGGCCGCTGCA
</TD><TD> 57
</TD></TR><TR><TD>pnir2_rb.syn.1
</TD><TD> GCGGCCGCTACTAGTAGTAGAGATCAATATAGGGTTCCCGTTTCGTTAAAATTTGTAAC
</TD><TD> 59
</TD></TR><TR><TD>pnir1_rb.syn.1
</TD><TD> GTAAATTACACTCAGCGAAGGCATATGCAGTTTACGCATTTAGCT
</TD><TD> 45
</TD></TR></TBODY></TABLE><P><B>Signal sequences</B></P><TABLE class="wikitable" align="center" border="1"><TBODY><TR><TH> Oligonucleotide
</TH><TH> Sequence
</TH><TH> Length
</TH></TR><TR><TD>pilA1_fp.syn.1
</TD><TD> CTAGATGGCTAGTAATTTTAAATTCAAACTCCTCTCTCAAC
</TD><TD> 41
</TD></TR><TR><TD>pilA2_ff.syn.1
</TD><TD> TCTCCAAAAAACGGGCAGAAGGTGGTACTAGTAGCGGCCGCTGCA
</TD><TD> 45
</TD></TR><TR><TD>pilA2_rf.syn.1
</TD><TD> GCGGCCGCTACTAGTACCACCTTCTGCCCGTTTTTTGGAGAGTTGAG
</TD><TD> 47
</TD></TR><TR><TD>pilA1_rp.syn.1
</TD><TD> AGAGGAGTTTGAATTTAAAATTACTAGCCAT
</TD><TD> 31
</TD></TR><TR><TD>slr2016-1_fp.syn.1
</TD><TD> CTAGATGGCAGCAAAACAACTATGGAAAATTTTCAATC
</TD><TD> 38
</TD></TR><TR><TD>slr2016-2_ff.syn.1
</TD><TD> CTAGACCGATGAAGGGTGGAACTAGTAGCGGCCGCTGCA
</TD><TD> 39
</TD></TR><TR><TD>slr2016-2_rf.syn.1
</TD><TD> GCGGCCGCTACTAGTTCCACCCTTCATCG
</TD><TD> 29
</TD></TR><TR><TD>slr2016-1_rp.syn.1
</TD><TD> GTCTAGGATTGAAAATTTTCCATAGTTGTTTTGCTGCCAT
</TD><TD> 40
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Step_2:_Site-directed_mutagenesis_of_GFP_.28BBa_E0040.29_into_a_fusion_brick">Step 2: Site-directed mutagenesis of GFP (BBa_E0040) into a fusion brick</SPAN></H3><P>GFP, as it currently exists in the Biobrick <A href="http://www.partsregistry.org" class="external text" rel="nofollow">Registry of Parts</A>, is a protein Biobrick, meaning that it will ligate out of frame with our signal sequence Biobricks. A primer will be designed for site-directed mutagenesis of the GFP start codon to convert BBa_E0040 into a fusion Biobrick.
</P><TABLE class="wikitable" border="1" align="center"><TBODY><TR><TH> Primer
</TH><TH> Sequence
</TH><TH> Length
</TH><TH> G/C content
</TH><TH> T<SUB>m</SUB></TH></TR><TR><TD> GFP (BBa_E0040) fusion / foward primer
</TD><TD> GCCGCTTCTAGAcgtaaaggag
</TD><TD> 22 bp
</TD><TD> 54.55%
</TD><TD> 60.2 C
</TD></TR><TR><TD> GFP (BBa_E0040) fusion / reverse primer
</TD><TD> cgagtcagtgagcgaggaag
</TD><TD> 20 bp
</TD><TD> 60%
</TD><TD> 59.6
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Step_3:_Conversion_of_pRL1383a_into_a_Biobrick_plasmid">Step 3: Conversion of pRL1383a into a Biobrick plasmid</SPAN></H3><P><A href="/Team:Hawaii/Project/Part_A" title="Team:Hawaii/Project/Part A"> Part A</A> of our project focuses on converting the RSF1010 based plasmid, pRL1383a into a sophisticated broad-host Biobrick plasmid. While we aim to ultimately express our secretion system in this new plasmid as part of a cyanobacterial expression system, we need a workable shuttle vector between <I>E. coli</I> (where constructs will be made) and PCC6803 (the ultimate host). Converting pRL1383a into a much simpler Biobrick plasmid will fulfill this requirement. Verification regions, transcriptional terminators, and the Biobrick multiple cloning site (MCS) will be isolated from the plasmid containing BBa_B0030 via PCR. PCR primers will also include <I>Hind</I>III and <I>Bam</I>HI restriction sites for ligation into pRL1383a. This ligation will replace the original pRL1383a MCS which includes Biobrick and Biobrick compatible restriction sites. The MCS replacement will be verified by restriction digest and plasmid sequencing.
</P><TABLE class="wikitable" border="1" align="center"><TBODY><TR><TH> Primer
</TH><TH> Sequence
</TH><TH> Length
</TH><TH> G/C content
</TH><TH> T<SUB>m</SUB></TH><TH> Notes
</TH></TR><TR><TD> HindIII-VF2BB
</TD><TD> cctAAGCTTtgccacctgacgtctaagaa
</TD><TD> 29 bp (20 bp)
</TD><TD> 48.3% (50.0%)
</TD><TD> 65.9 C (58.6 C)
</TD><TD> Includes RE extension HindIII site and three 5' nucleotides for efficient cutting. Parentheses indicate primer information w/o RE site and 3 nucleic acids. Based on VF2 primer.
</TD></TR><TR><TD> BamHI-VRBB
</TD><TD> ccaGGATCCattaccgcctttgagtgagc
</TD><TD> 29 bp (20 bp)
</TD><TD> 55.2% (50.0%)
</TD><TD> 67.9 C (58.0 C)
</TD><TD> Includes RE extension BamHI site and three 5' nucleotides for efficient cutting. Parentheses indicate primer information w/o RE site and 3 nucleic acids. Based on VR primer.
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Step_4:_Device_construction">Step 4: Device construction</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Figure 1: Construct of proposed GFP secretion device.</DIV></DIV></DIV><P>The synthesized signal peptides and nirA promoter BioBricks will be combined with at least three existing BioBricks to create two (or more) nitrate-regulated protein secretion devices according to the scheme detailed in Figure 1.  The resulting devices will be placed in a <I>Synechocystis</I> compatible BioBrick vector derived from the RSF1010 derived plasmid pRL1383a. In the proposed devices, the signal peptides will be situated so they are in-frame with GFP.  The translated polypeptide should consist of a N-terminal signal polypeptide leader sequence attached to a fluorescent protein.
</P><H3><SPAN class="mw-headline" id="Step_5:_Testing_for_protein_secretion">Step 5: Testing for protein secretion</SPAN></H3><P>The BioBrick vector can be inserted into <I>Synechocystis</I> sp. PCC6803 by triparental conjugation with <I>E. coli</I> harboring a transmissible plasmid (like RP1) and another <I>E. coli</I> containing our engineered plasmid. Plated <I>Synechocystis</I> sp. PCC6803 colonies successfully transformed wwill exhibit a glowing  halo of secreted GFP.  Transformed <I>Synechocystis</I> sp. PCC6803 grown in liquid media will result in fluorescent culture media.  The efficacy of the signal peptides in transporting GFP into the extracellular media can be measured using a spectrofluorometer.
</P><H3><SPAN class="mw-headline" id="Controls">Controls</SPAN></H3><P>A number of controls will also be constructed in parallel with our device to test device assembly at each step.
</P><TABLE align="center" border="1"><TBODY><TR><TH>Construct
</TH><TH>Control
</TH></TR><TR><TD align="center"><I>nirA</I> promoter
</TD><TD align="center"> lac promoter <A href="http://partsregistry.org/Part:BBa_I14032:Design" class="external text" rel="nofollow">(BBa_I14032)</A></TD></TR><TR><TD align="center"><I>nirA</I> promoter + rbs <A href="http://partsregistry.org/Part:BBa_B0034:Design" class="external text" rel="nofollow">(BBa_B0034)</A></TD></TR><TR><TD align="center"><I>nirA</I> or lac promoter + rbs <A href="http://partsregistry.org/Part:BBa_B0034:Design" class="external text" rel="nofollow">(BBa_B0034)</A> + GFP <A href="http://partsregistry.org/Part:BBa_E0040:Design" class="external text" rel="nofollow">(BBa_E0040)</A> + txn term. <A href="http://partsregistry.org/Part:BBa_B0024:Design" class="external text" rel="nofollow">(BBa_B0024)</A></TD><TD align="center"> IPTG induced GFP device <A href="http://partsregistry.org/Part:BBa_J04430:Design" class="external text" rel="nofollow">(BBa_J04430)</A></TD></TR><TR><TD align="center"><I>nirA</I> promoter + rbs <A href="http://partsregistry.org/Part:BBa_B0034:Design" class="external text" rel="nofollow">(BBa_B0034)</A> + <I>pilA</I> or <I>slr2016</I> signal sequence + GFP fusion brick + txn term. <A href="http://partsregistry.org/Part:BBa_B0024:Design" class="external text" rel="nofollow">(BBa_B0024)</A></TD><TD align="center"><I>nirA</I> or lac promoter + rbs <A href="http://partsregistry.org/Part:BBa_B0034:Design" class="external text" rel="nofollow">(BBa_B0034)</A> + GFP <A href="http://partsregistry.org/Part:BBa_E0040:Design" class="external text" rel="nofollow">(BBa_E0040)</A> + txn term. <A href="http://partsregistry.org/Part:BBa_B0024:Design" class="external text" rel="nofollow">(BBa_B0024)</A> and <I>nirA</I> promoter + rbs <A href="http://partsregistry.org/Part:BBa_B0034:Design" class="external text" rel="nofollow">(BBa_B0034)</A> + lichenase (duplicating Sergeyenko construct)
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1] Levinson, P. “Protein Purification from Microbial Cell Culture.” <I>Protein Purification Applications: A Practical Approach</I>. 73-82 (1990). 
</P><P>[2] Zweers, J et al. “Towards the development of Bacillus subtilis as a cell factory for membrane proteins and protein complexes.” <I>Microbial Cell Factories</I> 7 (2008). 
</P><P>[3] Synechocystis sp. PCC6803, complete genome. NCBI. 9 June 2008. &lt;<A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&amp;val=NC_000911" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&amp;val=NC_000911</A>&gt;. 
</P><P>[4] Kaneko T et al. “Sequence analysis of the genome of the unicellular cyanobacterium <I>Synechocysitis</I> sp. Strain PCC 6803. II. Sequence determination of the entire genome and  assignment of potential protein-coding regions.” <I>DNA Research</I> 3, 109-136 (1996). 
</P><P>[5] Mackle, M.M. and Zilinskas, B.A. “Role of Signal Peptides in Targeting of Proteins” <I>Journal of  Bacteriology</I> 4: 1857-1864 (1994). 
</P><P>[6] Filloux A et al. “Protein secretion in gram-negative bacteria: transport across the outer  membrane involves common mechanisms in different bacteria.” Dec 9(13):4323-9 (1990). 
</P><P>[7] Sergeyenko, T.V. and Los, D.A. Identification of secreted proteins of the cyanobacterium <I>Synechocystis</I> sp. strain PCC 6803.&quot; <I>FEMS Microbiol. Lett.</I> 193: 213-216 (2000). 
</P><P>[8] Sergeyenko, T.V. and Los, D.A. &quot;Cyanobacterial leader peptides for protein secretion.&quot; <I>FEMS Microbiol. Lett.</I> 218: 351-357 (2003). 
</P><P>[9] Ivanikova, N. V. “Lake Superior Phototrophic picoplankton: Nitrate Assimilation measure with  cyanobacterial nitrate-responsive bioreporter and genetic diversity of the natural community.” 9 June 2008. &lt;www.ohiolink.edu/etd/send-pdf.cgi?acc_num=bgsu1142559572&gt;</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Hawaii/Project/Part_B">http://2008.igem.org/Team:Hawaii/Project/Part_B</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hawaii/Project/Part_B" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hawaii/Project/Part_B" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hawaii/Project/Part_B&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hawaii/Project/Part_B&amp;oldid=84645" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>