<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_GreatBay_China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:GreatBay China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>GBC-Demonstrate</TITLE></P><DIV class="container"><DIV class="top-nav"><UL><LI><DIV class="res" target="_self" id="menuhead"><H5> Meow~</H5></DIV></LI><LI><A href="#"><SPAN>Team</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Team">Team</A><A href="https://2018.igem.org/Team:GreatBay_China/Collaborations">Collaboration</A><A href="https://2018.igem.org/Team:GreatBay_China/Attributions">Attributions</A></DIV></LI><LI id="active"><A href="#"><SPAN>Project</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Description">Description</A><A href="https://2018.igem.org/Team:GreatBay_China/Design">Design</A><A href="https://2018.igem.org/Team:GreatBay_China/Demonstrate">Results</A></DIV></LI><LI><A href="#"><SPAN>Parts</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Parts">Parts</A><A href="https://2018.igem.org/Team:GreatBay_China/Basic_Part">Basic</A><A href="https://2018.igem.org/Team:GreatBay_China/Composite_Part">Composite</A><A href="https://2018.igem.org/Team:GreatBay_China/Part_Collection">Collection</A><A href="https://2018.igem.org/Team:GreatBay_China/Improve">Improve</A></DIV></LI><LI><A href="#"><SPAN>HP</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/HP">Background</A><A class="long" href="https://2018.igem.org/Team:GreatBay_China/Human_Practices">Integrated HP</A><A class="long" href="https://2018.igem.org/Team:GreatBay_China/Applied_Design">Product Design</A><A href="https://2018.igem.org/Team:GreatBay_China/Hardware">Hardware</A><A class="long" href="https://2018.igem.org/Team:GreatBay_China/Public_Engagement">Public Engagement</A></DIV></LI><LI><A href="#"><SPAN>Lab</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Experiments">Experiments</A><A href="https://2018.igem.org/Team:GreatBay_China/Notebook">Notebook</A><A href="https://2018.igem.org/Team:GreatBay_China/Interlab">Interlab</A></DIV></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China/Judging"><SPAN>Judging</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China/Safety"><SPAN>Safety</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China/Model"><SPAN>Model</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China"><SPAN>Home</SPAN></A></LI></UL></DIV><DIV class="rolldown"><DIV class="container"><DIV class="footer"><DIV class="row" style="padding-left:50px"><DIV class="contact" style="width:450px"><H5 style="color:white;">mCATNIP2018@outlook.com</H5></DIV></DIV><DIV class="row" style="padding-left:50px"><DIV class="contact" style="width:450px"><H5 style="color:white;">@Igem2018C</H5></DIV></DIV></DIV><DIV class="row "><DIV class="jumbotron" style="color:white;"><H2>Demonstration</H2><UL style="font-size: 18px"><LI> ✓ Geraniol production by <EM>E. coli</EM></LI><LI> ✓ Conversion of geraniol to nepetalactol by <EM>S. cerevisiae</EM></LI><LI> ✓ Creating a stable microbial consortium of <EM>E. coli</EM> and <EM>S. cerevisiae</EM></LI></UL></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H5 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 1 An overview of mCATNIP: distributing the biosynthesis pathway of nepetalactol between <EM>E. coli</EM> and  <EM>S. cerevisiae</EM></H5></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Geraniol Production by <EM>E. coli</EM></H1><DIV class="fig"><H5 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 2 Geraniol production analysis by gas chromatography. (A) GPPS and GES are arranged in operon regulated by pTac, placed on a high copy vector pUC20. (B) The MVA pathway is split into two clusters and placed on a low copy vector with the upper cluster containing three genes and the downstream containing four. (C) A combined plasmid with GPPS&amp;GES operon and MVA pathway with a low copy p15A origin. (D) Gas chromatography for geraniol produced by <EM>E. coli</EM> expressing pMVA only (middle trace) or pMVA-GPPS-GES (bottom trace) whose peaks coincide with a geraniol standard (top trace). (E) Geraniol yield is noticeable with only the heterologous MVA pathway. Upon introduction of GPPS and GES, the yield doubled relative to the negative control pMVA only.</H5></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">We have proposed two different strategies for engineering a geraniol-producing <EM>E. coli</EM> strain: over-expressing GPPS and GES, which relies on native <EM>E. coli</EM> MEP pathway to supply IPP and DMAPP; and introducing a heterologous yeast MVA pathway along with GPPS and GES. For realizing these designs, we constructed (Figure. 1 A) pUC20-pTac-GPPS-GES and let Genscript constructed (Figure. 1 C) pMVA-GPPS-GES, and (Figure. 1 B) negative control pMVA for us. After transforming them into <EM>E. coli</EM>, we carried out a 24h shake-flask fermentation upon 25μM IPTG induction at a point when the OD600 equaled 1. Then, an n-hexane phase is used for harvesting geraniol from the culture which was analyzed by gas-chromatography. </P><P style="font-family: Delius; font-size: 18px;">The <EM>E. coli</EM> strain containing pUC20-pTac-GPPS-GES didn’t produce any detectable geraniol titer. Expression of pMVA-GPPS-GES yielded 11.4mg/L geraniol whereas expression of pMVA alone also produced 5.65mg/L geraniol. From this phenomenon we inferred that provision of the precursors IPP and DMAPP by heterologous pathway was necessary for achieving high geraniol yield. </P><P style="font-family: Delius; font-size: 18px;">However, we have not yet explored whether this expression level of GPPS and GES was the optimum at this condition. If the expression of GPPS and GES were tuned to suit the flux of precursors, we might see even greater titer of geraniol produced. Nonetheless, although at this moment we aimed at geraniol production by <EM>E. coli</EM> alone, the <EM>E. coli</EM> would eventually be co-cultured with <EM>S. cerevisiae</EM> for the production of nepetalactol. Then, since the change in culturing conditions has agitated the internal environment of the host, the pre-set ratio might be lost. Transcription-activator-like effector stabilized promoters (TALE sp) provided a solution to this problem through a regulatory mechanism employing the incoherent feedforward loop (iFFL). </P><P style="font-family: Delius; font-size: 18px;">We decided to utilize TALE sp to uncover the most suitable expression level of GPPS and GES. But the current number of TALE sp available was too small for us to make sensible decisions, which was the reason why we expanded the TALE sp library.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H5 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 3 Fluorescence characterization of the transcription-activator-like effector stabilized promoter (TALE sp) family. (A) Sequences of core promoter variants. TALE1 and TALE2 mean the TALE used this the promoter for negative regulation. Colored sequences refer to TALE binding sites. Names of the TALE sps harboring each core promoters are placed on the left of respective sequences. (B) Illustration for genetic parts design used in characterization. TALE1sp and TALE2sp are representatives of core promoters shown in (A). Identical characterization elements which include a RiboJ, an RBS, a sfGFP, and a terminator are placed downstream of TALE sps as well as J23 constitutive promoters for reference. (C) Performance of J23119, J23101, J23105 constitutive promoters on three plasmids with different copy numbers. The x-axis shows the reported copy number of plasmids used. (D) Performance of TALE1 stabilized promoters on the same set of plasmids used for characterizing constitutive promoters. Identities of the promoters are shown as legends to the right. In comparison to the sloped graph of J23 promoters, TALE1sps generate trend line that is more horizontal. (E) Performance of TALE2 stabilized promoters on the same set of plasmids used for characterizing constitutive promoters. Identities of the promoters are shown as legends to the right. </H5></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">The expansion of the library was based on introducing a number of new core promoters — the TALE-repressible promoters (Figure. 2 A), and assembling them with the genetic circuit encoding for TALE (Figure. 2 B). After adding six more TALE sp, the library then had twelve members, whose performance was characterization by fluorescence given out by the sfGFP they regulated. Besides the TALE sp, we also characterized three constitutive promoters from the J23 consensus promoter family. (Figure. 2 C) Each promoter was placed on three backbones: the high copy pSB1C3, the medium copy pR6K, and the low copy pSC101, so that we could assess the stabilization ability. From the results measured by flow cytometry, we could see that the influence of copy number on fluorescence level has sharply decreased, signifying valid stabilization by TALE sp.</P><P style="font-family: Delius; font-size: 18px;">From the library, we chose three TALE sp to geraniol synthesis which were TALE2sp6, TALE2 sp1 and TALE1 sp1 in ascending promoter strength. They along with a pTac promoter were used for driving gene expression of GPPS and GES. Similar to the fluorescence characterization, the constructs were placed on three different vectors of high, medium and low copy. As MVA pathway was a requisite for geraniol production from our previous experience, we transferred the plasmids into the <EM>E. coli</EM> strain already harboring pMVA that we have created. Because pMVA contained chloramphenicol resistance which was the same as pSB1C3, pUC20 was chosen as the high copy vector for geraniol production assay.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 4 Geraniol production assay of TALE sp relative to pTac IPTG-inducible promoter. (A) Twelve combinations of promoters and plasmid backbones used to express a geraniol synthesis operon for characterization. pSC101 is the low copy vector, pR6K is the medium copy vector and pUC20 is the high copy vector. The combinations are transformed and tested in <EM>E. coli</EM> already harboring pMVA plasmids. (B) Geraniol yield achieved under the regulation of pTac, TALE1 sp1, TALE2 sp1, and TALE2 sp6 across different copy numbers is shown. The vertical error bars represent standard deviations of geraniol yield measured, and the horizontal error bars signify the reported range of copy number of plasmid backbones used. The portion of the graph belong to TALE2sp6 is not shown as there is no detectable geraniol production.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center;">Table. 1 Geraniol production by different promoters and vector combinations</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">With pTac promoters, geraniol yield increased with the copy number of the vector, showing positively correlated relation. However, when TALE stabilized promoter (TALE sp) was used, higher the copy number of the vector, lower the production of geraniol, being the opposite of pTac. And the stronger TALE sp promoter, TALE1sp1 gave generally reduced yield compared to its weaker counterpart TALE2sp1. TALE2 sp6 whose strength measure with fluorescence is about half of the strength of TALE2 sp1 produced the lowest titre among the four promoters tested. Interestingly, the graphs of the TALE stabilised promoters appeared to be seemingly parallel to each other.</P><P style="font-family: Delius; font-size: 18px;">We surmised that the yield of geraniol was affected by two factors: the expression level of enzymes and the cellular burden. As for enzyme expression, there exists an optimal gene expression level that produces just enough enzyme to metabolize all the substrate. The yield would be the greatest at this level. But if lower than this level, production would increase with enzyme expression since there is a surplus of substrates. And if the enzyme expression is higher, the more enzymes now becomes a cellular burden as it no longer contributes to more product.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 5 Gas chromatography analysis for geraniol produced regulated by pTac and TALE2sp1 compared with the peak of geraniol standard. (A) Geraniol production accomplished by pTac regulation across low (pSC101), medium (pR6K) and high (pUC20) copy number. The area of the peak which appears at around 4.41s concurring with that of the geraniol standard increases along the copy number. (B) Geraniol yield assay using TALE2sp1 with the same set of copy number. The largest peak of geraniol emerges on the low copy vector pSC101. The peak shrinks in size on higher copy.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">In the case of pTac, it fits into the scenario when the gene expression is lower than the optimum: higher copy produced more enzymes to catalyze geraniol synthesis reaction. As for TALE stabilized promoters, the expression level of enzymes would remain unchanged regardless of copy number due to the stabilization nature of TALE sp (<A href="https://2018.igem.org/Team:GreatBay_China/Design#tale">Go to Design</A>). So in low copy vector where the cellular burden isn’t significant, the product yield is only related to the strength of the promoter, explaining why TALE2 sp1 has better performance than both TALE1 sp1 and TALE2 sp6 as it’s closer to the optimum. But when the copy number increases, the expression level is unaffected while the cellular burden rises sharply because much more TALE would be made to stabilize expression, leaving less energy available for geraniol synthesis. And it in turns answers why a negative trend of production is shown when regulated by TALE sp. </P><P style="font-family: Delius; font-size: 18px;">It was reported that TALE stabilized promoters could maintain the level of gene expression without the need for “re-tuning” upon genomic insertion, so we used pOSIP to integrate TALE1sp1, the core promoter of TALE1sp1, TALE2sp1, and the core promoter of TALE2sp1 with the sfGFP into the attB locus in the genome individually and characterized the fluorescence again. The result shows that after genomic insertion, TALE stabilized promoters maintain a higher expression level in comparison to the core promoters which are constitutive promoters. Also, we inserted TALE2sp1-GPPS-GES into the genome to investigate if the trend of geraniol yield increasing on lower copy stayed consistent. In the genome where theoretic copy number is one, geraniol yield rose slightly from the previous level on pSC101, reaching 94.5 mg/L.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 6 Influence of genomic insertion upon the functionality of TALE stabilized promoters. (A) Fluorescence characterization of TALE1 sp1, TALE2 sp1 and their core promoters inserted to the same locus in the genome. TALE1 sp1 and TALE2 sp1 are shown in brown whereas their core promoters are in green. (B) The change in geraniol yield across different copy number when using TALE2sp1. From left to right, the first point presents the geraniol obtained then the operon is inserted to the genome. The second, the third and the fourth are values of geraniol titer on pSC101, pR6K, and pUC20 respectively.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Conversion of geraniol to nepetalactol by <EM>S. cerevisiae</EM></H1><P style="font-family: Delius; font-size: 18px;">From literature we have comprehended that fact that the conversion was unlikely unless promiscuous endogenous enzymes were knocked out, so we began expression vector construction and gene deletion at the same time. </P><P style="font-family: Delius; font-size: 18px;">We constructed pCRCT-OYE2, pCRCT-OYE3, and pCRCT-ADH7 which were three CRISPR-Cas9 plasmids targeting OYE2, OYE3, and ADH7 individually. Since an 8bp deletion is not visible using gel electrophoresis, and must be confirmed by sequencing, we made a small alteration in the design of homologous recombination disruption donors for knocking out OYE2 to make a 100bp deletion. In this way, the result of gene disruption would be easily determined using gel electrophoresis. We achieved OYE2 deletion through this method, but failed when applying it to knock out OYE3 and ADH7, so the former approach of removing 8bp was used instead. After few cycles of transforming pCRCT, confirming the deletion, and discarding pCRCT, we obtained an  <EM>S. cerevisiae</EM> strain with all three genes knocked out.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 7 Sequencing results confirmed a successful disruption of oye2, oye3, and adh7. 100bp of OYE2 coding sequence is removed. As for OYE3 and ADH7, an 8bp deletion is generated.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Conversion of geraniol to nepetalactol by <EM>S. cerevisiae</EM></H1><P style="font-family: Delius; font-size: 18px;">As for the expression vector, at the beginning we used <EM>E. coli</EM> as the cloning host for constructing the vector, but to our surprise, no matter what assembly method we used — Gibson or biobrick assembly, the <EM>E. coli</EM> colony grown always missed the G8H component. After careful debug, we found that pTDH3 contains wild-type <EM>E. coli</EM> promoters which have triggered the expression of G8H in <EM>E. coli</EM>, resulting in cytotoxicity which forced <EM>E. coli</EM> to cleave G8H. So later we changed out a method by transforming the ligated DNA directly into yeast and using colony PCR to determine if the construction was successful. </P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 8 The results of the conversion of geraniol to nepetalactol. (A) <EM>S. cerevisiae</EM> expression vector with three genetic circuits encoding for G8H, GOR, and ISY on a 2 micron backbone. (B) Colony PCR gel electrophoresis image. Bands of correct length are seen meaning the construct was successful. (C) Gas chromatography results of geraniol consumption and conversion to nepetalactol. (1) is geraniol. (2),(3),and (4) are isomers of nepetalactol. (5) is an unknown products of metabolism. (D) The standard curve plotting to quantify nepetalactol yield</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">When we have eventually acquired the expression vector, the gene oye2 was knocked out. Therefore we started the characterization of using engineered yeast to convert geraniol to nepetalatcol. Having transferred the vector into both BY4741 and △oye2 BY4741, we shared the yeast at 30℃ and fed them with 40μl 200mM geraniol every two hours for 24h, then used gas chromatography for biocatalytic activity analysis. From the GC result, BY4741 has not produced nepetalactol, but we verified that geraniol was consumed by geraniol and nepetalactol was produced by △oye2 BY4741. However, an unknown metabolite was also generated and in a larger quantity than nepetalactol. Our assumption was that this peak was a shunt product reported. And if oye3 and adh7 were deleted in addition to oye2, it would be very likely that the nepetalactol yield increased and the shunt product decreased. At this moment, the nepetalactol yield we have obtained is 1.267mg/L.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Creating a stable microbial consortium of <EM>E. coli</EM> and <EM>S. cerevisiae</EM></H1><P style="font-family: Delius; font-size: 18px;">Co-culturing <EM>E. coli</EM> and  <EM>S. cerevisiae</EM> has been a challenging task due to their different doubling time, optimal growth conditions, and ethanol excretion of  <EM>S. cerevisiae</EM> inhibiting <EM>E. coli</EM> growth. Previous studies reveal a mechanism that prevents the competition between <EM>E. coli</EM> and  <EM>S. cerevisiae</EM> to maintain stability in a co-culture system: using xylose as the carbon source. When xylose is served as the carbon source, only <EM>E. coli</EM> can directly utilize it but  <EM>S. cerevisiae</EM> can not. Hence the only way for  <EM>S. cerevisiae</EM> to survive is to use acetate, the waste product of the xylose metabolism in <EM>E. coli</EM>, as its carbon source. In addition, acetate inhibits the growth of <EM>E. coli</EM> so in this way <EM>E. coli</EM> and  <EM>S. cerevisiae</EM> are mutually dependent on each other, which is known as commensalism.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 50px； padding-right: 50px">Figure. 9 Ratio of <EM>E. coli</EM> and  <EM>S. cerevisiae</EM> in the co-culture system given different carbon sources in 24h measured by flow cytometry. Flow cytometry sorts <EM>E. coli</EM> and  <EM>S. cerevisiae</EM> into two piles. The lower region represents <EM>E. coli</EM> and the upper region represents  <EM>S. cerevisiae</EM>. (A) When xylose is used as the carbon source, the percentages of  <EM>S. cerevisiae</EM> at different time are: 2h, 34.3%; 4h, 25.5%; 6h, 27.9%; 8h, 20.83%; 10h, 8.14%; 12h, 2.66%; 14h, 2.59%; 16h, 2.43%; 18h, 1.71%; 20h, 1.80%; 22h, 1.97%; 24h, 1.88%. (B) The percentages of  <EM>S. cerevisiae</EM> in 24h when fed glucose are: 2h, 23.47%; 4h, 38.91%; 6h, 54.90%; 8h, 73.85%; 10h, 77.02%; 12h, 75.03%; 14h, 75.59%; 16h, 74.68%; 18h, 72.59%; 20h, 72.34%; 22h, 66.92%; 24h, 63.59%.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">We have examined the use of two carbon sources, glucose and xylose, to create a stable co-culture system of <EM>E. coli</EM> and  <EM>S. cerevisiae</EM>.</P><P style="font-family: Delius; font-size: 18px;">In the first trial when glucose is fed,  <EM>S. cerevisiae</EM> is the dominant party in the system during the first 24h. It started with a percentage of 23.47%, increased to 77.02% in the tenth hour, and dropped to 63.59% later at the twenty-fourth hour. However, we cannot conclude that the microbial consortium has reached a steady state as the percentage of  <EM>S. cerevisiae</EM> kept fluctuating, and has shown a decreasing trend at the end. Longer monitoring might be needed in order to draw a valid conclusion.</P><P style="font-family: Delius; font-size: 18px;">And upon the use of xylose, <EM>E. coli</EM> became the advantageous species. The percentage of  <EM>S. cerevisiae</EM> decreased right after the inoculation, but the proportion of it has remained relatively constant around 1.84% since 12h, from which we could know that the microbial consortium reached stability, proving the validity of xylose upon maintaining the balance in an <EM>E. coli</EM> and  <EM>S. cerevisiae</EM> consortium. </P><P style="font-family: Delius; font-size: 18px;">In the future, if the proportion of  <EM>S. cerevisiae</EM> is to be increased, we can enhance acetate supply through inactivation of the gene atpFH in <EM>E. coli</EM>.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><A href="https://2018.igem.org/Team:GreatBay_China/Design"><H1 class="title"> An Outlook for Product Design</H1></A></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center; padding-left: 100px； padding-right: 100px">Figure. 10 Brief demonstration of the project applied design. (A) The first step, engineering a microbial consortium for the production of microbial catnip. A geraniol-producing <EM>E. coli</EM> strain (right) and the negative control <EM>E. coli</EM> strain (left) are shown. (B) The second step, extraction of microbial catnip. An n-hexane organic phase covers the culture of  <EM>S. cerevisiae</EM> for obtaining nepetalactol. (C) The third step, incorporation with the Kitty Wonderland, a hardware we designed for attracting, capturing, and accommodating feral cats, using microbial catnip as a relaxant to let down the guard of alert feral cats.</H6></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>