<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_GreatBay_China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:GreatBay China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>GBC-Design</TITLE></P><DIV class="container"><DIV class="top-nav"><UL><LI><DIV class="res" target="_self" id="menuhead"><H5> Meow~</H5></DIV></LI><LI><A href="#"><SPAN>Team</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Team">Team</A><A href="https://2018.igem.org/Team:GreatBay_China/Collaborations">Collaboration</A><A href="https://2018.igem.org/Team:GreatBay_China/Attributions">Attributions</A></DIV></LI><LI id="active"><A href="#"><SPAN>Project</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Description">Description</A><A href="https://2018.igem.org/Team:GreatBay_China/Design">Design</A><A href="https://2018.igem.org/Team:GreatBay_China/Demonstrate">Results</A></DIV></LI><LI><A href="#"><SPAN>Parts</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Parts">Parts</A><A href="https://2018.igem.org/Team:GreatBay_China/Basic_Part">Basic</A><A href="https://2018.igem.org/Team:GreatBay_China/Composite_Part">Composite</A><A href="https://2018.igem.org/Team:GreatBay_China/Part_Collection">Collection</A><A href="https://2018.igem.org/Team:GreatBay_China/Improve">Improve</A></DIV></LI><LI><A href="#"><SPAN>HP</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/HP">Background</A><A class="long" href="https://2018.igem.org/Team:GreatBay_China/Human_Practices">Integrated HP</A><A class="long" href="https://2018.igem.org/Team:GreatBay_China/Applied_Design">Product Design</A><A href="https://2018.igem.org/Team:GreatBay_China/Hardware">Hardware</A><A class="long" href="https://2018.igem.org/Team:GreatBay_China/Public_Engagement">Public Engagement</A></DIV></LI><LI><A href="#"><SPAN>Lab</SPAN></A><DIV><A href="https://2018.igem.org/Team:GreatBay_China/Experiments">Experiments</A><A href="https://2018.igem.org/Team:GreatBay_China/Notebook">Notebook</A><A href="https://2018.igem.org/Team:GreatBay_China/Interlab">Interlab</A></DIV></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China/Judging"><SPAN>Judging</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China/Safety"><SPAN>Safety</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China/Model"><SPAN>Model</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:GreatBay_China"><SPAN>Home</SPAN></A></LI></UL></DIV><DIV class="rolldown"><DIV class="container"><DIV class="footer"><DIV class="row" style="padding-left:50px"><DIV class="contact" style="width:450px"><H5 style="color:white;">mCATNIP2018@outlook.com</H5></DIV></DIV><DIV class="row" style="padding-left:50px"><DIV class="contact" style="width:450px"><H5 style="color:white;">@Igem2018C</H5></DIV></DIV></DIV><DIV class="row "><DIV class="jumbotron" style="color:white;"><H2>Project Design</H2><UL style="font-size: 18px"><LI>Geraniol production</LI><LI>Conversion of geraniol to nepetalactol by yeast</LI><LI>Optimisation</LI><DL><DD>TALE stabilised promoters</DD><DD>Gene deletion</DD><LI>Co- culture</LI><DIV class="row" style="padding-left:50px; padding-right: 50px;"><DIV class="fig"><H6 style="text-align: center;">Figure. 1 Distributing the biosynthesis pathway of nepetalactol between <EM>E. coli</EM> and <EM>S. cerevisiae</EM></H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Geraniol Production by <EM>E. coli</EM></H1><DIV class="fig"><H6 style="text-align: center;">Figure. 2 Geraniol synthesis pathway and genetic parts design. (A) The biosynthetic route for obtaining geraniol from acetyl-coA, which requires either a MVA or MEP pathway, a GPPS, and a GES. (B) Plasmid design for strategy one. GPPS and GES are arranged in operon regulated by pTac, placed on a high copy vector pUC20. (C) Design of pMVA only. The MVA pathway is split into two clusters and placed on a low copy vector with the upper cluster containing three genes and the downstream containing four. (D) A combined plasmid with GPPS&amp;GES operon and MVA pathway with a low copy p15A origin.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">In this study, we have planned two strategies for engineering a geraniol-producing <EM>E. coli</EM> strain. </P><H4><B><EM>Strategy one: over-expressing only GPPS and GES</EM></B></H4><P style="font-family: Delius; font-size: 18px;">Because of the presence of endogenous MEP pathway in <EM>E. coli</EM>, we want to investigate whether <EM>E. coli</EM> is able to produce geraniol with the heterologous expression of only GPPS and GES. So we used a colE1 high-copy plasmid backbone for constructing the vector pUC20-GPPS-GES for over-expressing an <EM>Abies Grandis</EM> GPPS and an <EM>Ocimum basilicum</EM> GES upon the induction by IPTG. The two synthases are arranged in an operon similar to the characterised limonene synthesis operon by J. Keasling.<SUP>[1][2]</SUP> An inducible promoter pTac is chosen to prevent bringing to much burden to the host by silencing the expression until it’s needed.</P><H4><B><EM>Strategy two: supplementing a MVA pathway</EM></B></H4><P style="font-family: Delius; font-size: 18px;">From our early research, we know that introducing an upstream heterologous yeast MVA pathway beside monoterpenoid synthase is an effective approach to enhance monoterpenoid production.<SUP>[3]</SUP> So our second plasmid contains the yeast MVA pathway, the GPPS&amp;GES operon. A single low-copy p15A origin is used to avoid heavy cellular burden. </P><P style="font-family: Delius; font-size: 18px;">In addition, we need a plasmid containing only the yeast MVA pathway same as the one on pMVA-GPPS-GES as a negative control.</P><P style="font-family: Delius; font-size: 18px;">Although E. coli strain DH1 was reported to be suitable for monoterpene synthesis<SUP>[4]</SUP>, we directly use our cloning host DH5alpha for geraniol synthesis because it saves time without needing to extract plasmids and transform it into a new host.</P><P style="font-family: Delius; font-size: 18px;">To verify geraniol yield, we carry out shake-flask fermentations with LB medium. 25μM of IPTG is added when OD600 reached 1<SUP>[1]</SUP> and the fermentation continued for 24h. and analysed the result using gas chromatography (GC). A more accurate method would be using gas chromatography-mass spectrum (GC-MS) for result analysis. However, due to difficulties in accessing GC-MS, we are only able used GC and have referred to the time when peaks appeared for qualitative analysis.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Conversion of geraniol to nepetalactol by <EM>S. cerevisiae</EM></H1><DIV class="fig"><H6 style="text-align: center;">Figure. 3 Heterologous biocatalytic route in <EM>S. cerevisiae</EM> and relevant plasmids design. (A) Three cytochrome P-450, G8H, GOR, and ISY, are needed for converting geraniol to nepetalactol. (B) <EM>S. cerevisiae</EM> expression vector design. G8H expression is under the regulation of a strong constitutive promoter pTDH3 and terminator tADH1, but later we abandoned this design and changed pTDH3 to a galactose-inducible promoter pGAL. pTEF1 and tENO1 control the expression of GOR. pRP18B and tTDH3 control the expression of ISY.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">Though being the most common eukaryotic engineering chassis, yeast doesn’t have as many available, well-characterized parts compared to <EM>E. coli</EM>. In this study, we have selected most of the parts used in this subproject from an established yeast toolbox developed by J.M. Dueber<SUP>[5]</SUP>. </P><P style="font-family: Delius; font-size: 18px;">We designed three separate genetic circuits encoding for G8H, GOR and ISY from the species <EM>Catharanthus</EM> roseus in order to enhance the expression of G8H. To ensure a higher expression of G8H, we have selected the strongest constitutive promoter pTDH3 from the toolbox for G8H over-expression. Worrying that high constant expression of G8H would pose to much stress to the host, we created another version of G8H genetic circuit using galactose-inducible promoter pGAL instead of pTDH3. GOR and ISY are under the regulation of pTEF1 and pRP18B which are two consensus promoters of similar strength lower than that of pTDH3. </P><P style="font-family: Delius; font-size: 18px;">Initially, we have chosen the following terminators from the toolbox: tENO1, tENO2, and tTDH1. However, later we found that tENO1 and tENO2 had near 100% similarity, which might induce homologous recombination in the host, so tENO1 was replaced by tADH1 in the 2018 distribution kits. </P><P style="font-family: Delius; font-size: 18px;">The three genetic circuits were put together on a single yeast expression vector pYES whose copy number was 50~100 per cell. We used www.syntegron.org to generate random spacers of 25bp between circuits.</P><P style="font-family: Delius; font-size: 18px;">BY4741 is the strain we chose to for nepetalactol biosynthesis as the highest biosynthetic nepetalactol titre was achieved with this strain.</P><P style="font-family: Delius; font-size: 18px;">Above all, the design of genetic circuits for <EM>S. cerevisiae</EM> was what we found the most challenging.</P></DIV><DIV class="row" id="tale" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Designing, Building, and Testing Transcription-activator-like Effector Stabilized Promoters for Geraniol Synthesis</H1><DIV class="fig"><H6 style="text-align: center;">Figure. 4 Transcription-activator-like effector stabilized promoters design. (A) Transcription-activator-like-effector stabilised promoters (TALEsp) achieve independence of expression level to copy number using a non-cooperative repression by TALE which binds specifically to DNA sequence (B) By employing incoherent feedforward loop (iFFL), copy number accretes both gene expression and the repression to the gene expression, thus the effect of copy number on expression level is canceled out. (C) TALE1 and TALE2 are two TALEs used in TALEsps. By changing the core promoter that they repress, we can obtain new TALEsps. A sfGFP is placed downstream of the promoter for characterization. (D) Core promoter sequences used in the library. Each contains a binding site of the corresponding TALE.</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">Genetic engineering requires elaborative design and a sensible proportion of different components<SUP>[6</SUP>]. For instance, unbalanced enzyme expression in metabolic engineering would lead to not only shunted flux to the product, but also an unnecessary waste of carbon source and host growth inhibition<SUP>[7]</SUP>.  Plasmids are common engineering tools used to tune gene expression. People often assume that plasmids exist in stable copies, but this is a misleading assumption to make as plasmid copy number is actually subjected to huge variability. Altered chassis strains, new culture medium composition, different temperature, and the difference in growth phase all hugely affect plasmid copy number<SUP>[8][9]</SUP>. In our case, although E. coli and yeast are engineered separately, they eventually would be cultured together and probably using very different conditions because compromise is made to meet both requirements of both species. Subsequently, gene expression regulated by such design is easily agitated, leading to potential failure of a synthetic device. </P><P style="font-family: Delius; font-size: 18px;">Transcription-activator-like-effector (TALE) stabilised promoters are a type of promoters able to untie gene expression level from gene copy number using an incoherent feedforward loop (iFFL) in which transcription-activator-like effectors (TALEs) function as a perfectly non-cooperative negative regulation. While copy number accretes gene expression, it also elevates the repression to the gene expression, thus has canceled out the effect of copy number on expression level<SUP>[10]</SUP>. This design can also eliminate the impact of the location of genes (whether they are placed on plasmids or in the genome, and wherein the genome) on gene expression.</P><P style="font-family: Delius; font-size: 18px;">Having comprehended the incredible capability and potential of TALEsp, we were deeply inspired yet felt sorry that there were only six TALEsp available to use<SUP>[11]</SUP>. In metabolic engineering and other areas of synthetic biology, people select promoters very carefully to ensure the most suitable strength is chosen, which seems difficult given so few candidates. Therefore, we expanded the TALEsp library through mutating core promoters of existing TALEsp, and adding TALE binding sites to the classical consensus promoter J23119, we created six more TALEsp, and have added all of twelve TALEsps to the iGEM Registry.</P><DIV class="sameline" style="vertical-align: top;"><H4>TALE1:</H4><UL><LI>TALE1 sp1 (<A href="http://parts.igem.org/Parts:BBa_K2753030">http://parts.igem.org/Parts:BBa_K2753030</A>)</LI><LI>TALE1 sp2 (<A href="http://parts.igem.org/Parts:BBa_K2753024">http://parts.igem.org/Parts:BBa_K2753024</A>)</LI><LI>TALE1 sp3 (<A href="http://parts.igem.org/Parts:BBa_K2753025">http://parts.igem.org/Parts:BBa_K2753025</A>)</LI><LI>TALE1 sp4 (<A href="http://parts.igem.org/Parts:BBa_K2753026">http://parts.igem.org/Parts:BBa_K2753026</A>)</LI><LI>TALE1 sp5 (<A href="http://parts.igem.org/Parts:BBa_K2753027">http://parts.igem.org/Parts:BBa_K2753027</A>)</LI><LI>TALE1 sp6 (<A href="http://parts.igem.org/Parts:BBa_K2753029">http://parts.igem.org/Parts:BBa_K2753029</A>)</LI></UL><H4>TALE2:</H4><UL><LI>TALE2 sp1 (<A href="http://parts.igem.org/Parts:BBa_K2753018">http://parts.igem.org/Parts:BBa_K2753018</A>)</LI><LI>TALE2 sp2 (<A href="http://parts.igem.org/Parts:BBa_K2753019">http://parts.igem.org/Parts:BBa_K2753019</A>)</LI><LI>TALE2 sp3 (<A href="http://parts.igem.org/Parts:BBa_K2753020">http://parts.igem.org/Parts:BBa_K2753020</A>)</LI><LI>TALE2 sp4 (<A href="http://parts.igem.org/Parts:BBa_K2753021">http://parts.igem.org/Parts:BBa_K2753021</A>)</LI><LI>TALE2 sp5 (<A href="http://parts.igem.org/Parts:BBa_K2753022">http://parts.igem.org/Parts:BBa_K2753022</A>)</LI><LI>TALE2 sp6 (<A href="http://parts.igem.org/Parts:BBa_K2753023">http://parts.igem.org/Parts:BBa_K2753023</A>)</LI></UL></DIV><DIV class="sameline" style="padding-left: 40px；vertical-align:top;"><DIV class="fig" style="vertical-align:top;"><H6 style="text-align: center;vertical-align:top;">Table. 1 The source of all promoters in the library</H6></DIV></DIV><DIV class="fig"><H6 style="text-align: center;">Figure. 5 Twelve plasmids designs for geraniol synthesis, generated by combinations of four promoters with three vectors. They are transformed into <EM>E. coli</EM> with pMVA for geraniol synthesis characterization.</H6></DIV><P style="font-family: Delius; font-size: 18px;">We first characterized the GFP expression driven by these promoters on low (pSC101), medium (pR6K), and high (pUC20) copy vectors using flow cytometry to measure the fluorescence. Then, we used three TALEsp in the library varying in strength (TALE1 sp1, TALE2 sp1, TALE2 sp6) and an inducible-promoter pTac along with the same set of vectors to express the geraniol synthesis operon, by which we investigated the impact of using TALEsp in metabolic engineering.  In total, we tested twelve combinations of promoter and vector, and have co-expressed them with pMVA in E. coli. 25μM IPTG is added to the culture when OD600 reached 1. Geraniol yield is measured using gas chromatography 24h after induction. After we found out the promoter which drove the highest geraniol production, we integrated the entire geraniol synthesis construct to the attB locus of E. coli genome using the plasmid pOSIP provided by Yiming Dong.<SUP>[12]</SUP></P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Deletion of Endogenous Genes from <EM>S. cerevisiae</EM> to reduce Shunt Products</H1><DIV class="fig"><H6 style="text-align: center;">Figure. 6 Preventing carbon source diversion from nepetalactol synthesis by knocking out <EM>oye2</EM>, <EM>oye3</EM>, <EM>adh7</EM> using CRISPR-Cas9. (A) Carbon source flux is diverted to other pathways at geraniol, 8-hydroxygeraniol, and 8-oxogeranial respectively. OYE2 and ATF1 show unwanted biocatalytic activity upon geraniol. OYE2 and OYE3 metabolize 8-hydroxygeraniol. 8-oxogeranial is subjected by dehydrogenation by ADH6 and ADH7. (B) An ultrahigh copy number plasmid pCRCT harboring both Cas9 and sgRNA. (C) An 8bp deletion is made between two homologous recombination disruption donor upon disruption by CRISPR</H6></DIV></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><P style="font-family: Delius; font-size: 18px;">The crosstalk between the heterologous synthesis pathway, endogenous ‘ene’-reduction and alcohol dehydrogenation pathways is the main factor preventing effective biosynthesis of nepetalactol in <EM>S. cerevisiae</EM>, making the deletion of promiscuous endogenous enzymes a requisite. A number of genes including <EM>oye2, oye3, atf1, ari1, adh6, adh7</EM> were reported by negatively affect nepetalactol production. However, as time is limited we only targeted three genes, which were <EM>oye2, oye3</EM>, and <EM>adh7</EM>.</P><P style="font-family: Delius; font-size: 18px;">Homologous recombination and CRISPR-Cas9 are two common tools for gene deletion in <EM>S. cerevisiae</EM>. In addition, a Homology-Integrated CRISPR−Cas (HI-CRISPR) System was reported able to generate multiple gene disruption in yeast.</P><P style="font-family: Delius; font-size: 18px;">At first, we tried to use a selection marker to replace the target gene by homologous recombination. But after the success of the substitution, we experienced difficulties in losing the selection marker. Since recycling is necessary with only a limited number of selection markers, we eventually gave up this method and went for Hi-CRISPR which sounded very efficient. However, although Hi-CRISPR achieved deletion of <EM>oye2</EM>, it failed to knock out <EM>oye3</EM> and <EM>adh7</EM>. It was mentioned that the efficiency of gene deletion using Hi-CRISPR is influenced by the order in which sgRNAs of multiple targets are arranged, so we then use the receiver plasmid of Hi-CRISPR, pCRCT, to perform single gene disruption at each time. Once the success in gene knock-out was verified, we plated yeast culture on 5-FOA plate, forcing it to abandon pCRCT which contains URA3 and would be changed to produced toxic materials by 5-FOA. After, another pCRCT carrying the sgRNA of next gene would be transformed into yeast again. This process repeated until all three genes were knocked out.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H1 class="title"> Creating a Stable, Mutualistic Microbial Consortium using Xylose as the Carbon Source</H1><P style="font-family: Delius; font-size: 18px;">Functional reconstitution of nepetalactol biosynthesis pathway in <EM>E. coli</EM> and yeast is a promising tool for enhancing nepetalactol yield. However, maintaining an equilibrium of such a co-culture system could be a difficult task due to difference in metabolism and optimal growth conditions. To be more specific, co-culturing <EM>E. coli</EM> and yeast is challenging for the following reasons:</P><UL style="font-size: 18px"><LI>E. coli has a doubling time much shorter than <EM>S. cerevisiae</EM>, resulting in the domination of bacteria over fungus in the system<SUP>[12]</SUP></LI><LI>When fed glucose, <EM>S. cerevisiae</EM> produces toxic metabolites such as ethanol which inhibit the growth of E. coli</LI><LI>Determining the growth conditions like temperature and pH is a time-consuming drudgery</LI></UL><P style="font-family: Delius; font-size: 18px;">In this case, replacing the carbon source from glucose to xylose provides an ingenious solution<SUP>[13][14]</SUP>.  <EM>S. cerevisiae</EM> cannot utilize xylose as a carbon source. So, it can only rely on <EM>E. coli</EM> to provide it with acetate, a product of metabolizing xylose but is inhibitory to and excreted by <EM>E. coli</EM>, which is then consumed by <EM>S. cerevisiae</EM> as energy-supplying fuel. Since acetate in yeast is a downstream product of ethanol, when fed acetate yeast would not synthesize ethanol. </P><P style="font-family: Delius; font-size: 18px;">Yeast provides a better growing environment for <EM>E. coli</EM>, meanwhile <EM>E. coli</EM> supply to yeast its only available food. With these mutually beneficial interactions, we would only need to inoculate an overpopulation of yeast at the beginning to create a stable microbial consortium.</P><A href="https://2018.igem.org/Team:GreatBay_China/Model" style="color:#000;"><P style="font-family: Delius; font-size: 18px;"><EM>We have developed a model to describe and predict the change in cell numbers of E. coli and yeast over time. Click here to check our model!</EM></P></A><P style="font-family: Delius; font-size: 18px;">We set the initial inoculation ratio of <EM>E. coli</EM> and yeast to about 1:40 to offset the slower growth rate of yeast and culture them using a previously reported medium for co-culture supplemented with ammonium phosphate to improve their growth.</P></DIV><DIV class="row" style="padding-left:50px; padding-right: 50px;"><H2>References</H2><OL><LI> Alonso-gutierrez, J., Chan, R., Batth, T. S., Adams, P. D., Keasling, J. D., Petzold, C. J., &amp; Soon, T. (2013). Metabolic engineering of Escherichia coli for limonene and perillyl alcohol production. Metabolic Engineering, 19, 33–41. https://doi.org/10.1016/j.ymben.2013.05.004</LI><LI> Smanski, M. J., Bhatia, S., Zhao, D., Park, Y., Woodruff, L. B. A., Giannoukos, G., … Voigt, C. A. (2014). Functional optimization of gene clusters by combinatorial design and assembly. Nature Biotechnology, (November). https://doi.org/10.1038/nbt.3063</LI><LI> Martin, V. J. J., Pitera, D. J., Withers, S. T., Newman, J. D., &amp; Keasling, J. D. (2003). Engineering a mevalonate pathway in Escherichia coli for production of terpenoids, 21(7), 796–802.</LI><LI> Redding-Johanson, A. M., Batth, T. S., Chan, R., Krupa, R., Szmidt, H. L., Adams, P. D., … Petzold, C. J. (2011). Targeted proteomics for metabolic pathway optimization: Application to terpene production. Metabolic Engineering, 13(2), 194–203. doi:10.1016/j.ymben.2010.12.005</LI><LI> Lee, M. E., DeLoache, W. C., Cervantes, B., &amp; Dueber, J. E. (2015). A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. ACS Synthetic Biology, 4(9), 975–986. doi:10.1021/sb500366v</LI><LI> Li, G.-W., Burkhardt, D., Gross, C. &amp; Weissman, J.S. Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources. Cell 157, 624–635 (2014).</LI><LI> Pfleger, B.F., Pitera, D.J., Smolke, C.D. &amp; Keasling, J.D. Combinatorial engineering of intergenic regions in operons tunes expression of multiple genes. Nat. Biotechnol. 24, 1027–1032 (2006).</LI><LI> Lin-Chao, S. &amp; Bremer, H. Effect of the bacterial growth rate on replication control of plasmid pBR322 in. Escherichia coli. Mol. Gen. Genet. 203, 143–149.</LI><LI> Węgrzyn, G. Replication of plasmids during bacterial response to amino acid starvation. Plasmid 41, 1–16 (1999).</LI><LI> Mangan, S. &amp; Alon, U. Structure and function of the feedforward-loop network motif. Proc. Natl. Acad. Sci. USA 100, 11980–11985 (2003).</LI><LI> St-Pierre, FrançoisCui, L., Priest, D. G., Endy, D., Dodd, I. B., &amp; Shearwin, K. E. (2013). One-Step Cloning and Chromosomal Integration of DNA. ACS Synthetic Biology.</LI><LI> Davison, B. H., &amp; Stephanopoulos, G. (1986). Effect of pH oscillations on a competing mixed culture. Biotechnology and Bioengineering, 28(8), 1127–1137. doi:10.1002/bit.260280802</LI><LI> Zhou, K., Qiao, K., Edgar, S., &amp; Stephanopoulos, G. (2015). Distributing a metabolic pathway among a microbial consortium enhances production of natural products. Nature Biotechnology, (November 2014). https://doi.org/10.1038/nbt.3095</LI><LI> Wei Zhang, Hong Liu, Xia Li, Duo Liu, Xiu-Tao Dong, Fei-Fei Li, En-Xu Wang, Bing-Zhi Li, Y.-J. Y. (2017). Production of naringenin from D-xylose with co-culture of E. coli and S. cerevisiae. Engineering in Life Sciences, 2–27. https://doi.org/10.1002/elsc.201700039</LI></OL></DIV></DL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>