<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Exeter_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Exeter/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV id="menu"><DIV class="menu_column" style="top: 10px; left:14.2%;"><A><DIV class="menu_heads">
                        PROJECT
                    </DIV></A><A href="https://2018.igem.org/Team:Exeter/Description">DESCRIPTION</A><A href="https://2018.igem.org/Team:Exeter/Design">DESIGN</A><A href="https://2018.igem.org/Team:Exeter/Model">MODELLING </A><A href="https://2018.igem.org/Team:Exeter/Gallery">GALLERY </A></DIV></DIV><DIV class="menu_column" style="top: 10px; left:28.5%;"><A><DIV class="menu_heads">
                        LABORATORY
                    </DIV></A><A href="https://2018.igem.org/Team:Exeter/Protocols">PROTOCOLS</A><A href="https://2018.igem.org/Team:Exeter/Experiments">EXPERIMENTS &amp; RESULTS </A><A href="https://2018.igem.org/Team:Exeter/InterLab">INTERLAB </A><A href="https://2018.igem.org/Team:Exeter/Notebook">NOTEBOOK</A><A href="https://2018.igem.org/Team:Exeter/Safety">SAFETY</A></DIV></DIV><DIV class="menu_column" style="top: 10px; left:42.8%;"><A><DIV class="menu_heads">
                        PARTS 
                    </DIV></A><A href="https://2018.igem.org/Team:Exeter/Parts">BASIC </A><A href="https://2018.igem.org/Team:Exeter/Improve">IMPROVED PARTS </A></DIV></DIV><DIV class="menu_column" style="top: 10px; left:57.1%;"><A><DIV class="menu_heads">
                        HUMAN PRACTICES
                    </DIV></A><A href="https://2018.igem.org/Team:Exeter/Human_Practices">INTRODUCTION</A><A href="https://2018.igem.org/Team:Exeter/HP/ELSA">ELSA</A><A href="https://2018.igem.org/Team:Exeter/HP/Bioreactor">BIOREACTOR</A><A href="https://2018.igem.org/Team:Exeter/Public_Engagement">ENGAGEMENT </A></DIV></DIV><DIV class="menu_column" style="top: 10px; left:71.4%;"><A><DIV class="menu_heads">
                        TEAM
                    </DIV></A><A href="https://2018.igem.org/Team:Exeter/Team">MEMBERS</A><A href="https://2018.igem.org/Team:Exeter/Supervisors">SUPERVISORS</A><A href="https://2018.igem.org/Team:Exeter/Collaborations">COLLABORATORS </A><A href="https://2018.igem.org/Team:Exeter/Attributions">ATTRIBUTIONS </A></DIV></DIV><DIV class="menu_column" style="top: 10px; left:85.7%;"><A><DIV class="menu_heads">
                        ACHIEVEMENTS
                    </DIV></A><A href="https://2018.igem.org/Team:Exeter/Medals">MEDALS</A><A href="https://2018.igem.org/Team:Exeter/Awards">AWARDS</A></DIV></DIV><DIV id="title"><B>Loading...</B></DIV><DIV id="title2" style="mix-blend-mode: color"><B>Loading...</B></DIV><DIV id="title3"><B>Loading...</B></DIV></DIV><P><CENTER><I><A href="#Introduction" class="myclass">Introduction</A><A href="#WT" class="myclass">Bacterial Growth Curves</A><A href="#SP" class="myclass">Signal Peptides + GFP</A><A href="#cld" class="myclass">Chlorite Dismutase</A><A href="#pcr" class="myclass">Perchlorate Reductase</A><A href="#summary" class="myclass">Summary</A></I></CENTER></P><DIV class="padder"><CENTER><A name="Introduction" style="text-decoration: none;"><H1>Introduction</H1></A></CENTER>
The focus of our lab work was to clone the <I>pcr</I> and <I>cld</I> gene sequences into <I>E. coli</I> and express the enzymes in order to reduce the compound perchlorate into chloride and oxygen. 

To do this we grew <I>E. coli</I> in perchlorate and chlorite in order to determine if <I>E. coli</I> could survive and at what concentrations, as this showed what our bioreactor bacteria could cope with. This was done with <I>E. coli</I> BL21 (DE3)  that did not contain our two gene sequences.

After this, we cloned the genes separately into the plasmid pSB1C3 using the cloning strain <I>E. coli</I> DH5α. Sequences were confirmed by Sanger sequencing and then the DNA constructs were transformed into <I>E. coli</I> BL21 (DE3). This strain was chosen because we used the inducible T7 promoter to express our enzymes and this strain contains the genes which will allow expression.

The <I>pcr</I> and <I>cld</I> genes from the organisms we chose (<I>Azospira oryzae, Azospira suillum, Dechloromonas aromatica</I> and <I>Nitrospira defluvii</I>, respectively abbreviated to: AO, AS, DA and ND), contained native signal-peptides. In order to determine whether these signal peptides were functional in <I>E. coli</I> we fused these to GFP (sGFP). Again the T7 promoter was used to drive expression.  

Our lab work also looked at assaying for perchlorate in tap water by using microscopy to identify crystals produced by the compound. This was part of a key collaboration with Virginia, a university in an area with high perchlorate concentration (see <A href="https://2018.igem.org/Team:Exeter/Collaborations" style="color: black"><B>collaborations</B></A> page). 
<CENTER><H1>Methods and Results</H1></CENTER><A name="WT" style="text-decoration: none;"><H2>Growth curves of various bacterial species in perchlorate and chlorite</H2></A>
At the very beginning of our lab work, we needed to see if <I>E. coli</I> could survive in the presence of perchlorate and chlorite, and at what concentrations. This was important information, as it allowed us to determine a baseline level of robustness of the wild type bacterial strain to the environment they will face in the bioreactor on Mars.  We used the BL21 (DE3) strain for these growth curves. This meant we could tackle the following question:
<CENTER><H3><I>&quot;Can <I>E. coli</I> survive in perchlorate and chlorite conditions that are likely to be encountered?&quot;</I></H3></CENTER>
Four perchlorate concentrations were chosen. Baseline growth was determined by adding 0 mM perchlorate. Our modelling informed us that if we could maintain a constant concentration of 1 mM perchlorate in our bioreactor we could produce enough oxygen to sustain one person at rest with a 48 litre bioreactor. However, to mitigate for fluctuations we chose a 2 mM concentration. If 100 g of Martian regolith was added to 100 ml of water (i.e. 100 % w/v) this would give a 100 mM perchlorate concentration. Finally, <A href="https://2017.igem.org/Team:UrbanTundra_Edmonton" style="color: black;">Urban Tundra 2016</A> determined that 200 mM was a toxic concentration of perchlorate for <I>E. coli</I>. 
<CENTER><FIGCAPTION style="line-height: 30px;">
    Figure 1: 12 hour growth curves for <I>E. coli</I> BL21 (DE3) in 0, 2, 100, 200 mM perchlorate. 0.2ml culture volumes in a 96 well-plate grown at 37°C, 600rpm measuring OD<SUB>600</SUB> every 30 minutes. 
</FIGCAPTION></CENTER>
Figure 1 shows that <I>E. coli</I> can survive in all tested concentrations of perchlorate. Although reduced <I>E. coli</I> can still grow in 200 mM perchlorate which is contradictory to the results from Urban Tundra. 

Assuming that all perchlorate added is converted to chlorite, we had to determine whether <I>E. coli</I> could survive if only our perchlorate reductase was active i.e. the chlorite is not converted to oxygen and chloride. Therefore we repeated the growth curves in the corresponding concentrations off chlorite.
<CENTER><FIGCAPTION style="line-height: 30px;">
    Figure 2: 12 hour growth curves for <I>E. coli</I> BL21 (DE3) in 0, 2, 100, 200 mM chlorite. 0.2ml culture volumes in a 96 well-plate grown at 37°C, 600rpm measuring OD<SUB>600</SUB> every 30 minutes. 
</FIGCAPTION></CENTER>
Figure 2 shows that <I>E. coli</I> cannot survive in the higher chlorite concentrations and has significant reduced growth at a chlorite concentration of 2 mM. Hence, in our bioreactor we need to ensure that both the perchlorate reductase and the chlorite dismutase are active to prevent the build-up of chlorite, which would otherwise kill the bacteria. On our <A href="https://2018.igem.org/Team:Exeter/Model" style="color: black"><B>modelling page</B></A>, we have investigated the different reaction rates of enzymes, and which one would be most suitable to use in our bioreactor. 

One of our aims is to bioremediate perchlorate from Martian regolith to allow future colonisers to use the soil to grow crops. The Newcastle University 2018 iGEM team are working with denitrifying soil bacteria. We were interested to discover whether these bacteria would suffer in the present conditions.   
<CENTER><FIGCAPTION style="line-height: 30px;">
    Figure 3: Newcastle collaboration data for alternative soil bacteria in perchlorate
</FIGCAPTION></CENTER>
Figure 3 shows that all three of their bacterial species are unable to grow in the 100 mM conditions (100 g of Martian regolith in 100 ml water). Therefore demonstrating our goal to bioremediate perchlorate is essential. The data also shows that if we are able to reduce perchlorate contamination the three bacterial species can survive in 2 mM perchlorate concentrations, indicating we do not need to completely remove perchlorate. 
<A name="SP" style="text-decoration: none;"><H2><B>Signal peptides and GFP</B></H2></A>
Perchlorate reducing bacteria rescue the substrate perchlorate all the way to oxygen and chloride, via a toxic intermediate chlorite. Perchlorate-associated oxidative stress, the toxicity of chlorite, and the potential of reducing reactive chlorine species are likely reasons why perchlorate reduction occurs in the periplasm of these bacteria. Signal peptides are at the N-terminal of the majority of the enzymes involved in this pathway, to enable export from the cyotplasm into the periplasm. As our project involves expressing these enzymes in <I>E. coli</I> we needed to determine whether the signal peptides from perchlorate reducing bacteria are able to export the enzymes into the periplasm of <I>E. coli</I>. This was so we could address the question:
<CENTER><H3><I>&quot;Will our target enzymes be transported into the periplasm of <I>E. coli</I>?&quot;</I></H3></CENTER>
We inserted the signal peptides from: <I>A. suillum</I> PcrA, PcrC and Cld; <I>A. oryzae</I> Cld; <I>D. aromatica</I> Cld; and <I>N. defluvii</I> Cld. These were synthesised by IDT along with the T7 promoter from plasmid pET21 (Novagen) and the B0015 terminator, and a <A href="https://2018.igem.org/Team:Exeter/Protocols" style="color: black;"><B>modular cloning</B></A> strategy was used to build the complete genes in pSB1C3. As the T7 promoter contains an XbaI site the complete genes have not been submitted to the iGEM registry. Instead the coding sequences, in pSB1C3, have been <A href="https://2018.igem.org/Team:Exeter/Parts" style="color: black;"><B>submitted</B></A> for future teams to use with their own chosen promoters.

Expression was confirmed in <I>E. coli</I> BL21 (DE3). We also expressed a GFP lacking a signal peptide as a control to verify that it was the signal peptide, not GFP itself, that is responsible for transferring to the periplasm. The cell preparation for the periplasm Western blot involved cell <A href="https://2018.igem.org/Team:Exeter/Protocols" style="color: black;"><B>fractionation</B></A> in order to isolate the periplasmic fraction for the assay. The cytoplasm Western blot was performed using an iBind membrane <A href="https://2018.igem.org/Team:Exeter/Protocols" style="color: black;"><B>protocol</B></A>.
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 4: Western blot of cyotplasm and periplasm. Expected molecular weights (kDa): 
A = 0 | B = 29.90 | C = 29.90 | D = 29.71 | E = 29.03 | F = 29.92 | G = 29.46
</FIGCAPTION></DIV>
Figure 4 shows that GFP expression occurred in all of our seven cultures. GFP lacking the signal peptide is only seen in the cytoplasm as is GFP fused to the signal peptide from <I>N. defluvii</I>. All other signal peptide GFP fusions have been successfully transferred to the periplasm, as demonstrated by the periplasmic Western blot. This indicates our chosen perchlorate reductase and three of our chlorite dismutases should be transported by <I>E. coli</I>'s export system into the periplasm. 
<A name="cld" style="text-decoration: none;"><H2><B>Chlorite dismutase</B></H2></A>
Once we had demonstrated that the signal peptides were functional in <I>E. coli</I>, we needed to express the four Cld proteins. We also needed to determine if the enzymes were active and could breakdown chlorite into oxygen and chloride. This helped answer the question:
<CENTER><H3><I>&quot;Could we express the four enzymes in <I>E. coli</I>, and if so are they active?&quot;</I></H3></CENTER>
We inserted a His-tag and flanking flexible linkers between the signal peptides and enzyme amino acid sequence of <I>A. suillum</I> Cld; <I>A. oryzae</I> Cld; <I>D. aromatica</I> Cld; and <I>N. defluvii</I> Cld. These were synthesised by IDT along with the T7 promoter from plasmid pET21 (Novagen) and the B0015 terminator, and a <A href="https://2018.igem.org/Team:Exeter/Protocols" style="color: black;"><B>modular cloning</B></A> strategy was used to build the complete genes in pSB1C3. As the T7 promoter contains an XbaI site the complete genes have not been submitted to the iGEM registry. Instead the coding sequences, in pSB1C3, have been <A href="https://2018.igem.org/Team:Exeter/Parts" style="color: black;"><B>submitted</B></A> for future teams to use with their own chosen promoters.
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 5: Gel electrophoresis after a restriction enzyme digest of the chlorite dismutase constructs with EcoRI and PstI
.
</FIGCAPTION></DIV>
EcoRI and PstI digested Cld genes are all expected to give rise to bands on the gel of ~1 150 bp. The pSB1C3 backbone is expected to give a band of ~2 029 bp. Figure 5 shows the expected band sizes for all four constructs and confirms that we have been successful in cloning all of our Cld constructs with the promoter and terminator. 

The His-tags allow us to perform Western blot analysis to determine if we are able to express the Cld enzymes. 
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 6: Western blot for chlorite dismutase constructs

Expected molecular weights (kDa): A = 32.97 | B = 32.99 | C = 32.96 | D = 31.20
</FIGCAPTION></DIV>
Figure 6 indicates that we have seen expression from <I>A. oryzae</I> Cld, <I>A. suillum</I> Cld and <I>D. aromatica</I> Cld. There is no band corresponding to the correct molecular weight of <I>N. defluvii</I> Cld. 

After confirmation of expression we now needed to demonstrate activity. For this we assayed for oxygen evolution after addition of the substrate chlorite using a Clark oxygen electrode <A href="https://2018.igem.org/Team:Exeter/Protocols" style="color: black;"><B>protocol</B></A>.
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 7: Kinetic trace of O<SUB>2</SUB> production from the chlorite dismutase Clark electrode 
</FIGCAPTION></DIV>
Figure 7 demonstrates that up until the point of chlorite addition the oxygen concentration in the Clark electrode decreases. On addition of sodium chlorite (at time = 0) the oxygen concentration increases in the presence of <I>A. oryzae</I> Cld, <I>A. suillum</I> Cld and <I>D. aromatica</I> Cld. No oxygen is evolved in the <I>N. defluvii</I> experiment but this is not surprising due to the lack of expression as demonstrated in Figure 6. 
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 8: Rate of O<SUB>2</SUB> production from the chlorite dismutase Clark electrode 
</FIGCAPTION></DIV>
Figure 8 demonstrates that <I>D. aromatica</I> Cld has the fastest rate of oxygen production and that <I>A. oryzae</I> Cld and <I>A. suillum</I> Cld has similar rates. Not only have we demonstrated that the enzymes are active we have also confirmed the different activities reported in Hofbauer <I>et al</I> 2014. This data can now be used to determine which chlorite dismutase will be most suitable for use in our bioreactor in collaboration with the information provided by our modelling.

By demonstrating expression and activity of <I>D. aromatica</I> Cld we have improved <A href="https://2016.igem.org/Team:Leiden" style="color: black;"><B>Leiden's</B></A><A href="https://2018.igem.org/Team:Exeter/Parts" style="color: black;"><B>parts</B></A>
We have also added two new parts to the registry: the Cld's from <I>A. oryzae</I> and <I>A. suillum</I>.

<A name="pcr" style="text-decoration: none;"><H2><B>Perchlorate reductase</B></H2></A>
The Pcr enzyme complex: PcrAB catalyses the reduction of perchlorate to chlorite. PcrC is a cytochrome-type protein that provides electrons to PcrAB, and PcrD is an essential accessory protein thought to provide the molybdenum co-factor to PcrAB. 
The operon encoding this enzyme complex had to be synthesised in two separate parts due to it's length. Previous teams have also struggled to work with such a large operon. Our question for the <I>pcrABCD</I> operon was: 
<CENTER><H3><I>&quot;Can we clone the operon into a suitable transformation vector, and can we express the 4 enzymes?&quot;</I></H3></CENTER>
The T7 promoter from pET21A was inserted upstream of the <I>pcrA</I> coding sequence flanked by the BioBrick prefix and suffix. The B0034 RBS was inserted upstream of each of <I>pcrB</I>, <I>pcrC</I> and <I>pcrD</I>, and the B0015 was inserted downstream. This entire sequence was flanked by the BioBrick prefix and suffix. Both of these individual parts were synthesised by IDT and the BioBrick assembly method was used to construct the entire operon in pSB1C3. 
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 9: Gel electrophoresis after restriction enzyme digest of the two separate parts for assembly of the <I>pcrABCD</I> construct
</FIGCAPTION></DIV>
Figure 9 shows successful digests of the individual parts and the pSB1C3 backbone. Bands were excised from the gel and using a Promega Wizard gel extraction kit, the DNA was purified. The three parts were combined in a ligation reaction with T4 DNA ligase (ThermoFisher). Transformation into DH5α, growth of overnight cultures followed by DNA miniprep extraction yielded DNA that was sent for Sanger sequencing. This confirmed, to our great surprise and joy, that we had <B>successfully managed</B> to build the complete operon. 

Carrying on from this success we then attempted to express the operon in <I>E. coli</I> BL21 (DE3). PcrA, PcrC, PcrD had been synthesised with either a His-tag, Flag-tag or HA-tag respectively. 
<DIV style="text-align: center;"><FIGCAPTION style="line-height: 30px;">
    Figure 10: Western blot of PcrA (106 kDa), PcrC (26.8 kDa) and PcrD (26.7 kDa)
</FIGCAPTION></DIV>
Figure 10 shows strong bands corresponding to the molecular weights of PcrC and PcrD. However, there is no band for the expected molecular weight of PcrA. We know that <I>pcrA</I> must be transcribed (due to the presence of PcrC and PcrD), therefore either PcrA is not translated, is mis-folded and bound up in the cell debris (after <A href="https://2018.igem.org/Team:Exeter/Protocols" style="color: black;"><B>BugBusting</B></A> protocol), or is rapidly degraded. 
<CENTER><A name="summary" style="text-decoration: none;"><H1>Summary</H1></A></CENTER>
The following evidence demonstrates that many of our parts worked. Firstly we needed to determine if the signal peptides native to <I>A. oryzae, A. suillum, D. aromatica</I> and <I>N. defluvii</I> were functional in <I>E. coli</I>. To do this we engineered GFP to contain the signal peptides from these proteins and performed cell fractionation and a Western blot to assess whether the proteins were present in the periplasm. This Western blot showed that 3 of the 4 signal peptides were functional in <I>E. coli</I>, with the exception of <I>N. defluvii</I>. Next we determined whether to clorite dismutases were expressed in <I>E. coli</I>, again via Western blot analysis. This demonstrated that three of the four were expressed, again with the exception of <I>N. defluvii</I>. We then demonstrated the activity of the three expressed chlorite dismutases using a Clark electrode and showed differing rates of oxygen production. Finally, we have managed to clone the <I>pcrABCD</I> operon and demonstrated expression of PcrC and PcrD.
<DIV style="font-size:11px; font-family:rawline; position: absolute;">
References:

The following papers have been invaluable in finding the information behind our project:
  <OL><LI> Hofbauer <I>et al</I>, (2014) <I>Chlorite dismutases - a heme enzyme family for use in bioremediation and generation of molecular oxygen</I>. Biotechnol. J. <B>9</B>,461
    </LI><A href="#" class="Back to Top" style="position: bottom;text-decoration: none;">Back to Top</A></OL></DIV></DIV></BODY></HTML>