<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:BNU-CHINA - 2016.igem.org</TITLE>
style = &quot;font-family: Helvetica;&quot;
</P><DIV class="header-container"><NAV class="navbar navbar-fixed-top navbar-inverse nav-adjust" role="navigation" style="margin-top: 14px;"><DIV class="container-fluid nav-container scrolled-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" id="navbar-toggle"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="navbar-brand" href="https://2018.igem.org/Team:BNU-China" style="font-family: Helvetica;">BNU-China</A></DIV><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:BNU-China" class="" style="font-family: Helvetica;">Home</A></LI><LI class="dropdown "><A href="https://2018.igem.org/Team:BNU-China/Project" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Project </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Background" style="font-family: Helvetica;">Background</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Design" style="font-family: Helvetica;">Design</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Results" style="font-family: Helvetica;">Results</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Improve" style="font-family: Helvetica;">Improvement</A></LI></UL><LI class="dropdown focus"><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Modeling </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Main" style="font-family: Helvetica;">Main model</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Probability" style="font-family: Helvetica;">Probability of distribution</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#E" style="font-family: Helvetica;">E.coli growth rate</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Growth" style="font-family: Helvetica;">Growth promoting effect</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Gene" style="font-family: Helvetica;">Gene expression level</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Ribosome" style="font-family: Helvetica;">Ribosome fall-off rate</A></LI><LI class="dropdown "><A href="./achievement.html" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Measurement" style="font-family: Helvetica;">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Demonstrate" style="font-family: Helvetica;">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Judging" style="font-family: Helvetica;">Judging</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Parts" style="font-family: Helvetica;">Parts</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Overview" style="font-family: Helvetica;">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Silver" style="font-family: Helvetica;">Silver</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Gold" style="font-family: Helvetica;">Gold</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Social Activities" style="font-family: Helvetica;">Social Activities</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Public " style="font-family: Helvetica;">Public Engagement</A></LI><LI class=""><A href="https://2018.igem.org/Team:BNU-China/Safety" style="font-family: Helvetica;">Safety</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Team </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Team#" style="font-family: Helvetica;">Members</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Attributions" style="font-family: Helvetica;">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Collaborations" style="font-family: Helvetica;">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Acknowledgement" style="font-family: Helvetica;">Acknowledgement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Notebook" style="font-family: Helvetica;">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Protocol" style="font-family: Helvetica;">Protocol</A></LI><LI class="dropdown "><A href="https://2018.igem.org/Team:BNU-China/InterLab" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">InterLab </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Calibration" style="font-family: Helvetica;">Calibration</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Cell Measurement" style="font-family: Helvetica;">Cell measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#CFU Measurement" style="font-family: Helvetica;">CFU measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Epilogue" style="font-family: Helvetica;"> Epilogue</A></LI></DIV></DIV></NAV></DIV><DIV class="main-container"><DIV><DIV class="page-story"><ARTICLE id="Calibration" class="col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0 col-sm-offset-0 col-sm-12"><HEADER class="page-header"><H1 id="model-introduction">Design</H1></HEADER><P>In biological aspect, BNU-China wants to solve the questions of plasmids’ genetic and structural instability, which directly cause strain’s degeneration and then weaken the function of target genes. These instability and degeneration-related influence can be clearly valued by observing expression-related phenomenon, like the amount of production and enzyme activity.</P><H3>Design: Specific gene circuit:</H3><P>We firstly design the gene loops described in figure 2. When engineered strains produce the wanted product, the downstream growth factor will be expressed and accelerate the growth of strains. In opposite situation, the growth factor will not be activated and strains will relatively get the growth inhibition.</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I>Fig.2 Design of the product-specific loop.</I></B>It contains three composition.<I>1) Exogenous pathway transformed into strains. </I>In this scheme, pathway is consist of gene1 and gene 2. Genes’ co-work can catalyze the precursor to the product. <I>2) Repressor. </I>Repressor can specifically bind with promoter at the upstream of growth factor. It also can bind with product, then change its conformation and release from promoter, allowing the subsequent genes expressed. <I>3) Growth factor.</I> It can give engineering bacteria growth advantage, making engineered-strains a relatively dominant species compared with degenerated strains.
                    </FIGCAPTION></FIGURE><P>In order to use specific experiments to verify our first idea, we separate the loop into three modules. Module 1 is growth factor module aiming at give strains the growth advantage. Module 2 includes a valuable product, a specific allosteric protein, and a negtive promoter. General relationship in module 2 can be clarified by that in β-galactosidase, lac1 repressor and lac promoter. Module 3 is where the product can be generated.
</P><H4>Module 1</H4><P>According to literature, expression of glucose dehydrogenase (gdh) can significantly enhance the growth rate of Escherichia coli and Bacillus subtilis. In Bacillus subtilis, when consumed same amount of glucose, the dry weight of the gdh-overexpressed strain is larger. That is to say, gdh-overexpression significantly increases the metabolic efficiency of glucose1-3. Gdh works in the branch of the pentose phosphate pathway. It converts glucose to gluconate, and then gluconate is phosphorylated into 6-P-gluconate and enter the pentose phosphate pathway (Figure 3). This metabolic process generate products includes NADPH, which significantly increases the cell's reducing power and helps cells fight against oxygen free radicals, and different carbon number compounds (C3 to C7), which play an important role in cell damage repair, energy supply and so on (Figure 4). (For example, product ribose-5-phosphate is a precursor to synthetic DNA and RNA, and product glyceraldehyde triphosphate is an important intermediate in glycolysis.)</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I>Fig.3</I></B>  Part of the pentose phosphate pathway. Gdh: glucose dehydrogenase that converts glucose to gluconate. Gluconate is phosphorylated into 6-P-gluconate and enter the pentose phosphate pathway (PPP).
                    </FIGCAPTION></FIGURE><FIGURE class="text-center"><FIGCAPTION><B><I> Fig.4 Important metabolic pathway in cells</I></B></FIGCAPTION></FIGURE><H4>Module 2</H4><P>There are some allosteric proteins in E. coli. They can works together with other genes, forming an operon to perform certain function. One of the operons, emrRAB (also known as mprA or marRAB), generally has resistance to some bacteria-harmful uncoupler. emrR, the repressor, can be expressed and bind to its own promoter and inhibit the express of emrA and emrB. When emrR bind with uncoupler like DNP, it is no longer able to bind with its promoter and open or enhance the expression of emrA and emrB. Then cell membrane’s permeability to uncoupler can increase allowing the bacteria to pump uncouplers out of the cell. According to literature, salicylic acid (SA) is also one of the allosteric inhibitors of emrR.</P><FIGURE class="text-center"><FIGCAPTION><B><I> Fig.5 Principle of emrRAB operon</I></B></FIGCAPTION></FIGURE><H4>Module 3: pchBA producing salicylic acid</H4><P>Salicylic acid (SA) is an important organic product that can be used in the production of cosmetics, preservatives, medical supplies, and so on. Nowadays, commonly used industrial synthetic method to produce SA is the Kolbe-Schmitt method, which is a three-step reaction using phenol as raw material. However, this method costs a lot of energy and quite a long time, and phenol has a carcinogenic effect, which is very harmful to the human body and the environment. Therefore, it is promising to replace chemical synthesis method by biosynthesis method.</P><FIGURE class="text-center"><FIGCAPTION><B><I>  Fig.6 Kolbe-Schmitt phenol medium pressure solvent process flow chart</I></B></FIGCAPTION></FIGURE><P>In nature, only few strains can produce SA, and they are difficult to be isolated or be screened. Recent years, some laboratories have succeed in SA’s biosynthesis in <I>E.coli</I> by utilizing the shikimate synthesis pathway, with a maximum yield of up to 1 g/L. Chorismate, an intermediate
 product in the pathway, is catalyzed to iso-branched acid by isochorismate synthase (ICS), and then catalyzed to SA by iso-branched pyruvate lyase (IPL) (Fig.7).</P><FIGURE class="text-center"><FIGCAPTION><B><I>  Fig.7 Synthetic pathway of salicylic acid. ICS: isochorismate synthase; IPL: isochorismate pyruvate lyase.</I></B></FIGCAPTION></FIGURE><P>Based on the research results from literature, we have the idea of applying our anti-degeneration loop to produce SA in order to increase production and reduce costs. It will undoubtedly have a positive effect on the industrialized biosynthesis of SA.</P><P>Previous experiments and literature have shown that both Pseudomonas putida and Pseudomonas fluorescens contains natural ICS and IPL genes and both genes can express and work in <I>E.coli</I>. Meanwhile, we found pchBA (Part: BBa_J45319), an IPL-ICS genes extracted from Pseudomonas fluorescens, in iGEM2018kit. So we use this pchBA and the positive-feedback loop mentioned above to design the pathway showed in figure 8. When pchBA is correctly expressed, SA is produced in the bacteria and makes the downstream gdh expressed. The expression of gdh can give bacteria growth advantage, so that the bacteria can produce larger amount of SA. In this way, we construct a positive feedback, allowing a vigorously growth and higher production to bacteria.</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I>Fig.8</I></B> Pcons: a constitutive promoter. RBS: ribosome-binding site. T: transcriptional terminator. EmrR: repressor protein, which specifically binds to PemrR and blocks the transcription of PemrR. After binding to SA, emrR change its own conformation and is no longer able to bind with PemrR. PemrR: negative promoter, which is unable to transcript after binding to emrR. Only works when SA or uncouplers release the inhibition of emrR. gdh: glucose dehydrogenase. It works as a growth factor that can enhance the pentose phosphate pathway and give the strains growth advantage.
                    </FIGCAPTION></FIGURE><H3 id="universal">Design: universal gene circuit:</H3><P>We have also realized that our genetic loop has an unavoidable defect: If we apply this pathway to prevent degradation of an engineered bacterial exogenous metabolic pathway, then we must find a repressor protein. The repressor protein can specifically bind to the promoter sequence to prevent transcription and change in conformation with the fermentation product to leave the specific sequence. However, the proteins present in nature are limited, so it is difficult to specifically find a repressor protein for a certain product molecule. And redesigning the repressor protein and then synthesizing is very complicated and cumbersome. Although many studies have provided good methods of finding, designing or engineering repressor proteins, their practical applications are still not very convenient, in other words, the existing gene loops do not have good universality.</P><P>Based on the above discussion, the existing experimental results and HP’s research, we can get the following conclusions: (1) The yield of salicylic acid will not be significantly enhanced using previous gene loop, but the response of emrR protein to salicylic acid is very sensitive, so salicylic acid can be used as a signal molecule; (2) The existing gene loop is not universal, and the number of the existing transducers that can respond to the product – the repressor proteins is limited; (3)In industrial production, E. coli is commonly used to produce a number of protein and peptide products.</P><P>In response to the above conclusions, we have designed a new universal anti-degradation loop for use in E. coli (Principle in Figure 9).</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I>Fig.9 </I></B><I>A) </I>MCS: multiple cloning site, any sequence can be inserted; stop-start codons overlap: TGATG sequence, TGA is the stop codon, ATG is the start codon, and when the ribosome encounters TGA or TAA stop codon, a termination factor is recruited to cleave the large subunit and small subunit of the ribosome to terminate the translation, but it is also possible to directly recognize the ATG without cleavage and initiate translation of the downstream sequence. This ensures that after translation of the MCS sequence, the following signal sequence has a certain possibility to be translated; signal: It can generate signal molecules to recognize the downstream Pinducible to initiate transcription and express growth-promoting factors; growth-promoting factors: can promote the growth of bacteria after expression. <I>B)</I> Experimental group pathway by embodying the above ideas, we have designed the following possible integrated pathways.
                    </FIGCAPTION></FIGURE><P>By embodying the above ideas, we have designed the following possible integrated pathways.</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I> Fig.10 Integrated plasmid.</I></B> A) GFP is directly linked to gdh by “TGATG” or haripin.B) GFP is linked to gdh by “TGATG”, T7 polymerase and T7 promoter. C) GFP is linked to gdh by “TGATG”, cI protein and lacI/cI promoter. D) GFP is linked to gdh by “TGATG”, pchBA and emrR binding promoter. It also uses CIChE technique.
                    </FIGCAPTION></FIGURE><P>Before verifying the overall function of the intergrate plasmid, it is necessary to verify the operation of each part. So we separate them into the following modules. Module 4 includes Stop-Start Codon “TGATG”, Module 5 includes cI repressor and its promoter, Module 6 includes T7 RNA polymerase and its promoter.</P><H4>Module 4: Stop-Start Codon “TGATG”</H4><P>TGATG is one of the core components in our system. Although made up of five simple bases, it has great functional advantages.</P><P>TGATG could form a special unit named “Stop-Start Codon”, which means stop codon(UGA) is in close proximity to start codon(AUG). There are many similar combinations: UGA combines with AUG(UGAAUG/UGAUG), UAA combines with AUG(UAAAUG/UAAUG), UAG combines with AUG(UAGAUG). As for TGATG we use in SAM, the stop codon overlap with start codon.</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I>Fig.11 Concept of “Stop-Start Codon”.</I></B></FIGCAPTION></FIGURE><P>In the process of translation, ribosome ought to drop at stop codon, but it has some possibility that the ribosome could directly identify start codon and move on to translate the next gene, which means we could save a promoter.</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I>Fig.12 Ribosome movement at “Stop-Start Codon”.</I></B></FIGCAPTION></FIGURE><P>Actually, this phenomenon is not common. According to Vivian, The transcription ratio of the downstream gene and the upstream gene is about 1:5-1:10. Multiple factors impact the data, such as the strain of the bacteria, the plasmid backbone, the gene itself... and so on.</P><H4>Module 5 : cI repressor and its promoter</H4><P>One of the keys to our project is how to couple the target gene to the expression of growth-promoting factors, and how to avoid minor variations in the regulatory part bringing a huge impact on the entire system. Initially our idea was to use a large number of operon regulation systems in prokaryotes—usually a small molecule substrate corresponding to a repressor. When a small molecule substrate is present, the repressor dissociates from the operon region that is located upstream of the promoter so that downstream products can be expressed. Therefore, the gene catalyzing the synthesis of the small molecule substrate and the target gene are expressed in tandem by a specific DNA sequence, and the repressor is integrated into the genome for constant expression, so that simultaneous expression of the growth-promoting factor when the target product is expressed can be achieved. Thus, the strain carrying the target gene is given a growth advantage.</P><P>However, this approach has two important disadvantages. First, building such a system is cumbersome and difficult. This is mainly reflected in the fact that there is often more than one gene for synthesizing a small molecule, and transferring these genes into the bacteria not only brings a large metabolic burden to the recipient bacteria, but also makes the whole system complicated and hard to predict. Second, small molecular substances are often easily transported across the membrane, which may make it possible that the bacteria that cannot correctly synthesize such a signal molecule (that is, the bacteria whose target gene is not normally expressed) obtains the same growth advantage as the bacteria that normally express the target gene. This invalidates the entire system.</P><P>Therefore, we hope that the signal conversion part of the system is as simple, efficient and leak-free as possible, which makes us think of transcription activators. These proteins tend to be of low molecular weight and are not normally transported extracellularly. It can promote the expression of downstream target genes by interacting with the transcription factors originally present in the cells, without the need to transfer additional proteins to help it. After reviewing the literature, we found that there is a cI protein in λ phage, which can achieve both the promotion and inhibition of a same promoter by binding DNA sequences at different positions on a specific promoter. Thus, a DNA sequence that binds to a cI protein resulting in transcriptional activation is fused to other promoters and subjected to some artificial design, such that the cI protein significantly enhances expression of the downstream product of the promoter. According to this principle, the protein has even been applied to the construction of a prokaryotic two-hybrid system, and its effect is expected to be significant.
</P><FIGURE class="text-center"><FIGCAPTION style="font-size: 16px"><B><I> Fig.13 </I></B><I>A)</I>The gene circuit design of the experiment group, with P<SUB>lac</SUB> promotes the transcription of CI and P<SUB>lac</SUB>/CI promotes the transcription of GFP. Both the promoters are activated by IPTG and P<SUB>lac</SUB>/CI is additionally activated by CI.<I> B) </I>The gene circuit design of the control group, in which CI is absent. <I>C)</I> A general view of the structure of CI, emphasizing the functional domains.
                    </FIGCAPTION></FIGURE><H4>Module 6 : T7 RNA polymerase and its promoter</H4><P>The T7final part we constructed contains the lactose operon, GFP, T7 ploymerase, T7 promoter, GDH, and ampicillin resistance genes, of which GDH and T7 ploymerase are coupled to express by TGATG structure. Our experiments focus on how to use positive feedback regulation to ensure the expression of the target gene on the plasmid by the growth-promoting effect of gdh. The specific mechanism is to couple GFP to T7 RNA Polymerase gene by TGATG structure. That is, to express GFP, the corresponding T7RNA Polymerase gene will also be expressed. But since the ribosome will have a certain probability of detaching from the mRNA when it encounters the TGA stop codon of GFP during translation, the translation of the T7RNA Polymerase gene will be less than the translation of the GFP gene, at the same time, the T7RNA Polymerase produced specifically activates the T7promoter downstream of the plasmid, allowing the gdh gene behind it to be expressed, and gdh promotes the growth of the bacterium. The result is that in the bacteria the more GFP is produced, the more T7 RNA Polymerase is expressed, and more T7 promoters will be activated to express more gdh which enables the high-yield GFP-producing strains to gain growth advantages. Thereby the yield of the strains increases. When high-yielding bacterium gain growth advantages, in the process of continuous selection, the degraded strains will gradually lose the competitive advantage. Thereby we can achieve the purpose of controlling the degradation of strains. The advantage of our module is that it can be used to select strains without antibiotics thereby reducing the use of antibiotics, and can be applied to a wide range of industrial production such as food and medicine.</P></ARTICLE></DIV></DIV><DIV id="footer"><DIV id="footer-div"><UL id="footer-ul"><LI><A href="https://2018.igem.org/Team:BNU-China">Home</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Background">Project</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab">InterLab</A></LI></UL><SPAN>© 2018 BNU</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>