Team:BNU-China/Protocol
Team:BNU-CHINA - 2016.igem.org
style = "font-family: Helvetica;"
Toggle navigation
BNU-China
Home
Project
Background
Design
Results
Improvement
Modeling
Main model
Probability of distribution
E.coli growth rate
Growth promoting effect
Gene expression level
Ribosome fall-off rate
Achievements
Measurement
Demonstrate
Judging
Parts
Human Practices
Overview
Silver
Gold
Social Activities
Public Engagement
Safety
Team
Members
Attributions
Collaborations
Acknowledgement
Notebook
Protocol
InterLab
Calibration
Cell measurement
CFU measurement
Epilogue
Protocol
1. PCR system
1) 20μL system is used for verification
Taq:
Super mix:
2) 50μL system for PCR target genes
Pfu mix:
DNA: plasmid 50ng; genome 100~200ng
3) 20μL PCR system for microbes
Super mix:
Bacteria (after picking on the board, back up on the new board and rinse it in the system that has been prepared)
4) Fusion PCR
1. Basic PCR2. Use the PCR product of step 1 as template to do PCR3. Use the PCR product of step 2 as template to do PCR, but first five cycles don't add primer, add primer at the sixth cycle and continue PCR.
The system of step 2: H2O 21μL 2x primeSTAR 25μL R+F-Primer 2 μL Template1 1μL Template2 1μL
2. Plasmid concentration
1. Dilute collected DNA to 400μl. (If the plasmid is eluted by ddH2O, then it is diluted by ddH2O as well; If the plasmid is eluted by elution buffer, then it is diluted by elution buffer as well)
2. Add 900μl absolute ethanol to precipitate DNA, and 40μl NaAc (3M PH=5.2) to help precipitation. Be sure to mix it up.
3. Put the reactant in ice-bath for 1 hour, or put it under -20 degrees Celsius overnight.
4. Centrifuge the reactant under 12000rpm for 10 minutes, then throw away the supernatant.
5. Add 1ml 75% ethanol (the 75% ethanol is prepared by using absolute ethanol rather than ethanol for disinfection) and mix up to wash away salt ions.
6. Centrifuge the reactant under 12000rpm for 5 minutes, then throw away the supernatant.
7. Centrifuge the reactant for another 3-4 seconds and use transfer liquid gun to throw away residual supernatant.
8. Centrifuge the reactant for another 3-4 seconds and use transfer liquid gun to throw away residual supernatant.
9. Open the centrifuge tube and expose the reactant to air. Put the centrifuge tube into incubator at 37 degrees Celsius for 15 minutes until the plasmid turn into transparent.
10. Add appropriate volume of ddH2O to dilute the plasmid according to precipitation capacity.
11. Use 1μl solution and add 9μl ddH2O to dilute it, then use electrophoresis to measure its quantity.
3. Plasmid purification
1. Culture of E-coli: Select single colonies from plate culture medium then inoculate them into 1-3ml liquid nutrient medium containing antibiotics, then culture them at 37 degrees Celsius overnight. (Usually the culture time is 12-16 hours, because if the cells have been cultured over 16 hours, it is hard to lysis the cells so that plasmid production will decrease. Besides, the medium should not be excessive, because too many bacteria are difficult to be fully lysised so that plasmid purity is affected.)
2. Collect 1-4ml overnight culture of bacterial fluid, then centrifuge it at 12000rpm for 2 minutes and throw away the filtrate.
3. Use 250μl Solution | (be sure that RNase A is added to Solution |) to fully suspend the bacterial precipitation. (Pay attention not to leave over small pieces of bacteria. Vortex can be used to make the bacteria fully suspended)
4. Add 250μl Solution || and gently blend the solution by turning it up and down for 5-6 times to make the bacteria fully lysis until the solution becomes transparent. (Be sure to blend the solution gently instead of oscillating fiercely, and this step should be no more than 5 minutes.)
5. Add 350μl Solution ||| which was pre-cooled at 4 degrees Celsius, gently blend the solution by turning it up and down for 5-6 times until tight agglutination block is formed, then stand it under room temperature for 2 minutes.
6. Centrifuge the solution at 12000rpm for 10 minutes and keep the filtrate. (Centrifuge at 4 degrees Celsius is not benefit to sedimentation.)
7. Place the Spin Column on the Collection Tube.
8. Transfer the supernatant from Step (6) to the Spin Column, then centrifuge it at 12000rpm for 1 minutes and throw away the filtrate.
9. Add 500μl Buffer WA to the Spin Column then centrifuge it at 12000rpm for 30 seconds and throw away the filtrate.
10. Add 700μl Buffer WB to the Spin Column then centrifuge it at 12000rpm for 30 seconds and throw away the filtrate. (Make sure that specified volume of 100% ethanol has been added to the Buffer WB.)
11. Repeat Step (10) once.
12. Place the Spin Column on the Collection Tube, then centrifuge it at 12000rpm, under room temperature for 1 minutes and throw away the filtrate.
13. Place the Spin Column on a new 1.5ml centrifuge tube. Add 50μl sterile water or Elution Buffer to the center of the membrane of the Spin Column and stand the solution under room temperature for 1 minutes. (If the sterile water or Elution Buffer is heated to 60 degrees Celsius, then the elution efficiency can be improved.)
14. Centrifuge the solution at 12000rpm, under room temperature for 1 minutes to elute DNA.
4. Enzyme digestion
1)  Enzyme digestion test (10μl system)
X: look up the enzyme digestion table
Y: If the length of the minimal fragment is a, the length of the plasmid is b, the concentration of the plasmid is c, then: 20ng/a=(m ng)/b; y=m/c
2)  Enzyme digestion to get linear plasmid (50μl system)
X: look up the enzyme digestion table
y: 2.5μg
5. Infusion
1) Primer design
1. Each In-Fusion primer must contain two parts: the primer 5 'end must be included and will be connected to the DNA fragment (such as carrier or another fragment) 15 ends completely homologous base. Primers 3 must contain primer sequences of target genes.
2. Each primer 3' should be:
- with gene specificity
- length is 18 - 25, the content of GC is 40 - 60%
- annealing temperature (Tm) is of 58 - 65 C. Forward and reverse primers of the Tm value difference should not exceed 4 degrees, otherwise the amplification effect is not ideal. Note that the evaluation of Tm values should be based on primer 3 (gene specificity), rather than on the whole primer. If Tm is too low, we can appropriately extend the part of primer with gene specificity until Tm reaches 58 - 65 degrees C. - do not contain successive identical nucleotides. The last 5 nucleotides of each primer '3' should not contain more than 2 guanine (G) or cytosine (C).
3. The complementary sequence between the primers is avoided to prevent the primer itself from producing hairpin structure.
4. You can do BLAST search to make sure that the 3’ end of each primer is highly specific. (www.ncbi.nlm.nih.gov/BLAST/)
2) Reaction system and steps
* linear plasmid and target gene volume should be smaller than 2μL
1. mixing system
2. static in 50 degrees Celsius environment, reaction for 15min
3. convert on ice, or -20 preservation
4. 2.5μL reaction liquid was added into 50μL competent cells during transformation
6. DNA fragment purification
1. Add Buffer DC as much as 3 times the volume of the PCR reaction liquid to the PCR reaction liquid (or other enzymatic reaction fluid), then mix them up. (If the needed Buffer DC is less than 100μl, then add to 100μl.)
2. Place the Spin Column on the Collection Tube.
3. Transfer the solution in Step (1) into the Spin Column, then centrifuge it at 12000rpm, under room temperature for 1 minutes and throw away the filtrate. (If the filtrate is added back to the Spin Column to centrifuge again, the recovery rate of the DNA can be improved.)
4. Add 700μl Buffer WB to the Spin Column then centrifuge it at 12000rpm, under room temperature for 30 seconds and throw away the filtrate. (Make sure that specified volume of 100% ethanol has been added to the Buffer WB.)
5. Repeat Step (4) once.
6. Place the Spin Column on the Collection Tube, then centrifuge it at 12000rpm, under room temperature for 1 minutes.
7. Place the Spin Column on a new 1.5ml centrifuge tube. Add 25-30μl sterile water or Elution Buffer to the center of the membrane of the Spin Column and stand the solution under room temperature for 1 minutes. (If the sterile water or Elution Buffer is heated to 60 degrees Celsius, then the elution efficiency can be improved.)
8. Centrifuge the solution at 12000rpm, under room temperature for 1 minutes to elute DNA.
7. Gel electrophoresis
1) Prepare agarose gel
Experimental supplies: 1×TAE, agarose
1. Weigh agarose according to its working concentration. Usually 0.8% or 1% is suitable for running quantitative gel or testing PCR product.
2. Using 1×TAE to dissolve the agarose by careful warming. (Heat it in microwave oven and boil it two times so that the agarose dissolves completely)
3. When the temperature decreases to 55 degrees Celsius, add Gelstain and shake it up. (1μl Gelstain per 10 ml gel)
4. Pour the prepared gel into the electrophoresis bath, wait for the gel to concrete and reserve it.
2) Add sample
1. For running quantitative gel, add 1μl sample; For testing PCR product, add 3-5μl sample.
2. Mix up the sample and 10×loading buffer, add the mixture into spotting hole. Be careful not to leave any air bubbles, which will change the volume of the sample.
3. Add corresponding volume of Marker according to the manual. (Usually 5μl Marker per 10μl spotting hole)
4. Set up constant voltage 170V, switch on for about 15 minute. When the indicator line reach 1/2-2/3 of the whole gel, switch off.
5. Use Gel imager to show the imaging of the gel.
8. Agarose Gel DNA Extraction
1. Agarose gel is prepared using TAE buffer or TBE buffer, then agarose gel electrophoresis is performed on the target DNA.
2. Cut the agarose gel containing the target DNA under the UV lamp and suck the liquid on the gel surface with a paper towel. At this point, we should pay attention to the removal of the gel which does not contain the target DNA, minimize the gel volume and increase the DNA recovery rate. When glue blocks exceed 300 mg, use more than one Column Recycling, otherwise seriously affecting the yield. Note: when cutting, please be careful not to expose the DNA to UV light for a long time to prevent DNA damage.
3. Shredding the gel block. When the adhesive block is chopped up, the dissolution time of the block in the operation step 6 can be accelerated, and the DNA recovery rate can be improved.
4. Weighing the weight of the glue block and calculating the volume of the glue block. When calculating the volume of the block, 1mg=1μL is used to calculate.
5. Buffer GM are added into the gel block, and the amount of solution Buffer GM.
6. After mixing evenly at room temperature 15-25 degrees C dissolve glue block (glue concentration is more difficult to dissolve and can be heated at 37 C). At this point should be interrupted oscillation mixing, so that the block fully dissolved (about 5~10 minutes). Note: the block must be fully dissolved, otherwise it will seriously affect the recovery rate of DNA. High concentration gel can prolong the time appropriately.
7. When the gel is completely dissolved, observe the solution color, if the solution color changed from yellow to orange or pink, add 3 M sodium acetate solution to the rubber block solution (pH5.2) 10μL, mixed solution to restore yellow. When the DNA fragment is less than 400bp, the isopropanol with a final concentration of 20% should be added to the solution.
8. Place the Spin Column in the reagent kit on the Collection Tube.
9. Transfer the solution of the operation step 7 to the Spin Column with 12000 rpm centrifugation for 1 minutes, filtrate rejection. Note: if the filtrate is added to the Spin Column to be centrifuged once, it can be improved DNA recovery rate.
10. Add 700μL Buffer WB to Spin Column at room temperature 12000 rpm centrifuge for 30 seconds, discard filtrate. Note: please confirm that Buffer WB has added 100% ethanol of the specified volume.
11. Repeat step 10.
12. Place the Spin Column on the Collection Tube at room temperature 12000rpm Centrifuge for 1 minutes.
13. Place the Spin Column on the new 1.5 ml centrifuge tube at Spin. Add 30μl of sterile water or Elution Buffer at the center of the Column film and leave it for 1 minutes at room temperature. Note: heating the sterilized water or Elution Buffer to 60 degrees can improve the elution efficiency when used.
14. 12000 rpm centrifugations at room temperature, 1 minute elution DNA.
9. Saccharomyces cerevisiae genomic extraction
1. Mix the bacterial fluid into the 2ml centrifuge tube and collect the cells by centrifugation at 12000rpm 2min;
2. Add 1ml sterile water or washing buffer and wash 1 time;
3. Adding equal volume glass beads, and adding 100μL TENTS, using a cell crusher to break cells for 1min;
4. Add 600 L TENTS, mix upside down, then put in room temperature for 1min;
5. Centrifuge 8000 rpm, 1 min, pipette extract supernatant, transfer to the new 2ml EP tube;
6. Add the volume of phenol: chloroform (1:1) [pay attention to exact the lower layer], mixing evenly, place in room temperature10 min;
7. Centrifuged 10 min 120000 rpm, carefully remove the supernatant with a 200 L pipette [pay attention to not to the middle of the protein layer];
8. Add 2-3 times the volume of anhydrous ethanol, 1/10 volume of NaAc, mixing evenly, ice bath for 40 min-1h;
9. Centrifuged 10 min 12000 rpm and discard supernatant;
10. Add 1 ml 75% ethanol, mix evenly [wash precipitation], centrifuge 5 min 12000 rpm;
11. Discard the supernatant [caution not to drop the bottom white], Short, the residue is removed with a pipette, and the 37 degrees incubator is dried for 20 min;
12. Precipitation is transparent and add 100μL ddH2O dissolve DNA [this step can be 4 degrees overnight], and then add 1.5μl RNase, 37 degrees [remove RNA] reaction 1-2 h;
13. Added 100μL ddH2O per tube [dilute the DNA to reduce the loss of dilution, extraction process], with an equal volume of phenol: chloroform (1:1), upside down mixing, 12000 rpm centrifugal for 10min, carefully pipette supernatant with 200μL[note the protein in the middle layer];
14. Repeat step 13 once;
15. Add 2-3 times the volume of anhydrous ethanol, 1/10 volume of NaAc, upside down, mixing evenly, ice bath 40 min-1 h;
16. 12000 rpm, centrifuge 10 min and discard supernatant;
17. Add 1 ml 75% ethanol, mix evenly [wash precipitation], 12000 rpm, centrifuge 5 min;
18. Discard the supernatant [caution not to drop the bottom white], Short, the residue is removed with a pipette, and the 37 degrees incubator is dried for 20 min;
19. Precipitation is transparent and add 20-30 L ddH2O dissolve DNA [this step can be 4 degrees overnight].
10. Preservation in Glycerol
1) The manufacture of glycerin tubes
Configuration 50% glycerin (Glycerol: ddH2O = 1:1)
Packaged in 2ml storage tubes and 550μl per tube
Sterilization at 121 degrees for 20min
2) Preservation
In advance, the target strains were cultured in the liquid medium containing antibiotics (E. coli in LB medium, yeast in YPD) for 12-14h
Secondly, the liquid fraction of 10ml~20ml was centrifuged twice, and the bacterial precipitate was suspended by 1ml without the corresponding liquid medium of μ antibiotics.
Add the suspension to the sterilized glycerin tube, blow it several times and screw down the lid.
11. Inversion
1. Add appropriate amount of plasmid (no more than 4μL) into 50μL competence.
2. Place the reactant on ice for 30 minutes.
3. Put the reactant under 42 degrees Celsius for 45s-90s.
4. Place the reactant on ice for 2 minutes.
5. Add 950μL Luria-Bertani into the reactant inside the super-clean bench.
6. Shaking culture the reactant for 1 hour, at 37 degrees Celsius, 200rpm.
7. Centrifuge the reactant for 5 minutes at 3000-4000rpm to collect the cell. Throw away 500μL supernatant, then re-suspend the rest reactant. Use 100μL reactant to spread plate(Luria-Bertani + appropriate antibiotic), and the rest is reserved under 4 degrees Celsius.
8. Culture the cell overnight under 37 degrees Celsius. (If chloramphenicol is added, then the culture time can be longer)
12. Prepare competent cell
1. Inoculate strains in 5ml YPD medium and culture them at 200rpm, 30 degrees Celsius overnight.
2. Add 500μl mother culture into 50ml YPD and culture them at 200rpm, 30 degrees Celsius for 12-14 hours, then make sure the OD value of the fungal fluid is between 1.3-1.5.
3. Put the fungal fluid into 50ml sterilized centrifuge tube and make sure the solution is balanced before centrifuging. Centrifuge the fungal fluid at 4000rpm, 4 degrees Celsius for 5 minutes and then collect the sediment. Add 3ml icy sterile water, then use the transfer liquid gun to mix up and re-suspend the sediment. Add 25ml icy sterile water to wash the sediment and centrifuge it at 4000rpm, 4 degrees Celsius for 5 minutes for two times. (Do not touch the fungal fluid with hands to make sure it is kept under low temperature during the process.)
4. Add 10ml icy sterile water to re-suspend the sediment, then add 1ml pH=7.510×TE loading buffer and mix them up on ice on the super-clean bench. Add 10×LiAc and shaking culture it at 90rpm, 30 degrees Celsius for 45 minutes.
5. Add 0.4ml 1mol/L DTT and shaking culture it at 90rpm, 30 degrees Celsius for 15 minutes.
6. Centrifuge the fungal fluid at 4000rpm, 4 degrees Celsius for 5 minutes and then throw away the supernatant. Use 25ml icy sterile water to wash the sediment. (The same washing method as step (3).)
7. Use 10ml 1mol/L icy sorbitol to wash the sediment. Centrifuge it at 4000rpm, 4 degrees Celsius for 5 minutes and then throw away the supernatant.
8. Add 200μl sorbitol into each centrifuge tube to dissolve the sediment and transfer it into EP tube (180μl per tube). Use 2 tube of EPY100 for electroporation, and add 10μl glycerol into each of the rest tubes and preserve it under -80 degrees Celsius.
13. Electric transfer
The cleaning of the electric rotor:
Clean with distilled water, immerse with alcohol in the clean bench for 10min, add sterile water, use a syringe suction cleaning for 3 times, absorb liquid and open the cover (lid should be washed) under UV for 1H, place in ice precooling.
1. Add 5~10μg DNA (volume <10μL) to the competent cell, blow with a gun (gently); (9 L PYD1, 1- alpha 1 g/ L), transfer to the pre-cooled electric cup, and put aside for 5min;
2. Set shock parameters: 1.5Kv, 25 F, 200Ω;
3. Immediately add 1ml precooling sorbitol, transfer to the EP tube, 30 degrees and be static for 1h;
4. 4000rpm and 5min are set to centrifuge at normal temperature, and then supernatant is abandoned and add 1ml YPD, centrifuge at 30 degrees for 200rpm and 2h;
5. 4000rpm, 5min at room temperature centrifugation, suction 550μL supernatant, 150μL/ plate used for coating board [before coating board, culture board should be drying].
14. SDS-PAGE
Materials
Running Buffer
1. The SDS polyacrylamide gels are prepared in the so-called PerfectBlue™ Twin Double Gel System.
2. After ensuring that the equipment is waterproof, the 12% (or 18%) running gel is mixed and filled into the chamber. Pipetting about 1 ml of H2O on top of the running gel to seal the gel.
3. After polymerization, the remaining H2O is removed and the 12% stacking gel is filled on top. Insert a comb to create sample pockets.
4. After the stacking gel also polymerized, 1x running buffer is used to run the Double Gel System via the SDS gel.
5. After loading the generated pockets with the samples, the stacking gel is run at 100 V and then running gel at 120 V.
15. Western Blot
1) System
TBST:1000 mL(pH=7.4)
Imprint buffer：2000 mL (pH=8.3) Transfer Buffer
2) Transfer
(Prepare transfer Buffer just before glue leaking, and precool at -20℃).
1. Put the transfer Buffer and the black subface of transfer splint downward, and lay a sponge in it. Several filter paper(three pieces of filter paper), glue(except Stacking Gel).
2. Activation PVDF membrane in advance with anhydrous ethanol, and put it on the membrane.
3. Three layers of filter paper, sponge, Squeeze out of the bubbles, turn tight.
4. The black subface electric rotary groove stick to each other, put in ice.
5. 110V, 120min.
6. 5% skim milk powder (prepared by TBST), block for a night.
7. Dilute Primary antibody at the proportion of 1:2000 with 3% skim milk powder (add 0.02% sodium azide), incubate 1h at the room temperature.
8. TBST elute, wash with shocking for 5min, three times.
9. Dilute Secondary antibody at the proportion of 1:2000 with 3% skim milk powder, incubate 1h at the room temperature.
10. TBST elute, wash with shocking for 5min, three times.
11. Color development.
16. Detection of proteins
1. Remove the film, blot the water with the filter paper and wrap it in a dark room.
2. Exposure: prepare develop and fix buffer, pull the curtain, open the lamp. Take the plate and cut it into two parts. Open the blind clip, put the film in the NC film, cover it, and expose for 10 seconds to 5 minutes.
3. Take the film and place it in the develop buffer for 2 minutes.
4. Leave the develop liquid in the fixative for 1 minutes, and the ribbon appears.
17. Target protein purification
Using Ni Sepharose 6 Fast Flow
1. If the column contains 20% ethanol, wash it with 5 column volumes of distilled water. Use a flow velocity of 50 to 100 cm/h. Refer to Appendix 8 for flow rate calculations.
2. Equilibrate the column with at least 5 column volumes of binding buffer at a flow velocity of 150 to 600 cm/h.
3. Load the sample at a flow velocity of 150 to 600 cm/h.
4. Wash with 20 column volumes of wash buffer at a flow velocity of 150 cm/h.
5. Elute with elution buffer using a one-step procedure. Five column volumes of elution buffer is usually sufficient. Alternatively, a linear elution gradient (10 to 20 column volumes) may give higher purity, at the expense of lower target protein concentration in eluted fractions. Use a flow velocity of 150 cm/h.
18. Protocol of salicylic acid inducing the emrR bacterial strain
Step1: Preparation of strain
1. Remove DH5a from the -80 degrees Celsius refrigerator and let it melt on ice.
2. Mix competent cells with plasmids
3. Ice bath for 30 minutes.
4. 42 degrees Celsius heat shock 90 seconds
5. Ice bath for 2 minutes.
6. Add 1000ul of antibiotic-free LB medium，37 degree shaker recovery for one hour
7. Centrifuge at 3000rpm，5min
8. Discard 500ul of supernatant, and mix and drain the remaining bacteria
9. Pipette 100-200 ul of the mixed bacteria solution and apply it on the LB plate with ampicillin resistance.
10. After overnight incubation for about 12 hours, the plate should have colony growth.
Step2：Determination of the optimum concentration of salicylic acid
1. Pick a single colony from the top of the board and expand it with liquid LB containing Amp, and control it around 11h.
2. Prepare for salicylic acid induction verification
(1) The whole experiment can be done in 3-5 groups. In the ultra-clean platform, the cultured bacterial liquid is divided into several 20ml large centrifuge tubes, each tube is filled with 2ml. Note that when the bacteria is filled, the large centrifuge tube is placed. The lid is removed, wrapped with a sealing film, and the rubber band is tight (because the sealing film can be ventilated, if the lid is covered with oxygen, it will be insufficient)
(2) Add salicylic acid to make the salicylic acid concentration in the system 0mg/L、0.001mg/L、0.01mg/L、0.1mg/L、0.3mg/L、0.5mg/L、1mg/L.
(3) 37 ° C shaker culture induction 5h
(4) All samples and the expanded E.coli K12 96-well plate (operated in a clean bench) were used to measure the fluorescence intensity and OD of each sample well by a microplate reader. The fluorescence was mcherry red fluorescence (absorption wavelength 580 nm, Emission wavelength is 610nm
3. Looking for the optimal concentration of salicylic acid induced expression
(1) After recording all the data, calculate the fluorescence intensity per unit OD of each sample (fluorescence intensity / OD)
(2) A scatter image of induced expression is obtained by plotting the fluorescence intensity per OD - salicylic acid concentration
(3) According to the obtained image, the optimal concentration of salicylic acid-induced expression is roughly judged, that is, the minimum concentration of salicylic acid when the fluorescence intensity per unit OD reaches the maximum value.
(4) If the value of the fluorescence intensity corresponding to the concentration of 0.001 mg/L is already the maximum value of the fluorescence intensity, then our gradient concentration setting is too high, the gradient concentration should be further set at 0.001 mg/concentration, and the above experimental procedure is repeated. (important)
Step3: Find the optimal induction time at the optimum concentration
1. Do 3-5 parallel groups. In the ultra-clean platform, the expanded bacteria solution is divided into several 20ml large centrifuge tubes, each tube is filled with 2ml. Note that when the bacteria is filled, the large centrifuge tube is placed. The lid is removed, wrapped with a sealing film, and the rubber band is tight (requires similar to the above)
2. Adding salicylic acid to the system makes the salicylic acid concentration in all systems the optimum concentration
3. Immediately after the addition of salicylic acid, the fluorescence intensity and OD value were measured by a microplate reader on a 96-well plate.
4. Then, the fluorescence intensity and OD were measured every 30 min, and the experimental data was recorded until 5 h.
5. Perform a fluorescence intensity-time relationship image of the unit OD to find the optimal induction time
19.  Digestion
20.  Ligation
21. Transformation
1) Material
2) Protocol
1. Preparation of the competent cells
2. 1 μL ligation product + 50 μL cells
3. Heatshock of Trans5α(42°C, 45s)
4. Put on ice(2min)
5. Add 500 μL LB media and incubate for 1h(37°C, 150rpm)
6. Centrifuge at 4000 rpm for 1min and remove 400 μL supernatant
7. Resuspend the pellets using the left supernatant
8. Spread plates(with Kan;Chl)
9. Incubate for 12~16h(37°C)
22. Protein expression
1. Inoculated 3 mL LB media including relevant antibiotics with the monoclonal colony of expression plasmid, incubate for 12~16h(37°C, 190rpm)
2. Inoculated 100 mL TM expression media including relevant antibiotics with the 1 mL bacteria liquid, incubate for 3h(37°C, 250rpm,OD600=0.6~0.8)
3. Add IPTG into it until its final concentration is 1 mmol/L, incubate for 4~6h(37°C, 250rpm)
4. Centrifuge at 6000 rpm for 10min and remove supernatant
5.      Gather sediment, cryopreserve at -20°C
23. Electronic model protocol
1. Turn on the main switch NO.1 and the main switch NO.2.
2. Turn on the switch NO.3 but turn off the switch NO.4.
3. Push the button then the corresponding LED light will flicker with a low frequency. (There are four buttons which are corresponding to four different color LED lights)
4. Change the switch NO.3 off with the switch NO.4 on.
5. Push the button then the corresponding LED light will flicker with a high frequency.
6. Turn off all switches.
Home
Project
Modeling
Achievements
Human Practices
Safety
Team
InterLab
© 2018 BNU
