<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-China_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-China/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:BNU-CHINA - 2016.igem.org</TITLE>
style = &quot;font-family: Helvetica;&quot;
</P><DIV class="header-container"><NAV class="navbar navbar-fixed-top navbar-inverse nav-adjust" role="navigation" style="margin-top: 14px;"><DIV class="container-fluid nav-container scrolled-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" id="navbar-toggle"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="navbar-brand" href="https://2018.igem.org/Team:BNU-China" style="font-family: Helvetica;">BNU-China</A></DIV><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:BNU-China" class="" style="font-family: Helvetica;">Home</A></LI><LI class="dropdown focus"><A href="https://2018.igem.org/Team:BNU-China/Project" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Project </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Background" style="font-family: Helvetica;">Background</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Design" style="font-family: Helvetica;">Design</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Results" style="font-family: Helvetica;">Results</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Improve" style="font-family: Helvetica;">Improvement</A></LI></UL><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Modeling </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Main" style="font-family: Helvetica;">Main model</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Probability" style="font-family: Helvetica;">Probability of distribution</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#E" style="font-family: Helvetica;">E.coli growth rate</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Growth" style="font-family: Helvetica;">Growth promoting effect</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Gene" style="font-family: Helvetica;">Gene expression level</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Ribosome" style="font-family: Helvetica;">Ribosome fall-off rate</A></LI><LI class="dropdown "><A href="./achievement.html" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Measurement" style="font-family: Helvetica;">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Demonstrate" style="font-family: Helvetica;">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Judging" style="font-family: Helvetica;">Judging</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Parts" style="font-family: Helvetica;">Parts</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Overview" style="font-family: Helvetica;">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Silver" style="font-family: Helvetica;">Silver</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Gold" style="font-family: Helvetica;">Gold</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Social Activities" style="font-family: Helvetica;">Social Activities</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Public " style="font-family: Helvetica;">Public Engagement</A></LI><LI class=""><A href="https://2018.igem.org/Team:BNU-China/Safety" style="font-family: Helvetica;">Safety</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Team </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Team#" style="font-family: Helvetica;">Members</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Attributions" style="font-family: Helvetica;">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Collaborations" style="font-family: Helvetica;">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Acknowledgement" style="font-family: Helvetica;">Acknowledgement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Notebook" style="font-family: Helvetica;">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Protocol" style="font-family: Helvetica;">Protocol</A></LI><LI class="dropdown "><A href="https://2018.igem.org/Team:BNU-China/InterLab" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">InterLab </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Calibration" style="font-family: Helvetica;">Calibration</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Cell Measurement" style="font-family: Helvetica;">Cell measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#CFU Measurement" style="font-family: Helvetica;">CFU measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Epilogue" style="font-family: Helvetica;"> Epilogue</A></LI></DIV></DIV></NAV></DIV><DIV class="main-container"><DIV class="container-fluid page-heading" style="background-image: url(https://2018.igem.org/File:T--BNU-China--collaborations.jpg);"><H3 style="font-family:Helvetica;"> Collaborations </H3></DIV><DIV class="page-story"><ARTICLE id="project" class="col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0 col-sm-offset-0 col-sm-12"><H2>1.HBUT-China &amp; BNU-China</H2><P>In our collaboration with HBUT-China, we helped them develop a mathematical model of the NcrB protein operon. Their corresponding page can be accessed here:
               <A href="https://2018.igem.org/Team:HBUT-China/Collaboration">Team:HBUT-China/Collaboration</A></P><H3>BNU-China:modeling</H3><H4>Overview</H4><P>Ni2+ is one of the heavy metal pollutants in our environment. In order to detect the concentration of the Ni2+, HBUT-China team are going to use the NcrB protein operon to achieve this goal. In this system, the nikABCDE gene will express the NikABCDE protein, which belongs to the ATP-binding cassette transporter. It can transport Ni2+ from the environment into the cell consuming ATP at the same time. And Ni2+ is the signal we need to detect. At the same time the ncrB gene will express NcrB protein, which is a repressor combining with the pncrA promoter, inhibiting the expression of the downstream gene luxCDABE. However Ni2+ will bind with the NcrB protein, causing the NcrB protein drop from the pncrA promoter. In this way, the luxCDABE gene will express. And the LuxCDABE protein can give out light by consuming energy in the cell. Thus we can detect the concentration of Ni2+ by measuring the luminescence intensity.</P><H4>Hypothesis</H4><H5>1. The hypothesis on the diffusion equilibrium of Ni<SUP>2+</SUP>.</H5><P>We assume that the ions distribute uniformly inside the cell immediately after they are transported into it through NikABCDE. As long as the speed of transportation is lower than the speed of diffusion, the ions are unlikely to aggregate around the transporter protein(e.g., NikABCDE). According to our knowledge, most of the transporter proteins do not possess such a high transporting efficiency, which means this hypothesis is valid in a larger degree.</P><H5>2. The hypothesis on the relationship between NcrB concentration and the copy number of the recombinant vector.</H5><P>Since each of the recombinant vector contains only one copy of the target gene, we can assume that the vectors have the same ability of expressing NcrB. Furthermore, it’s reasonable to assume that the final concentration of NcrB is proportional to the copy number of the recombinant vector. </P><H5>3. The hypothesis on the concentration of NcrB during the process in which Ni and pncrA bind with NcrB competitively</H5><P>There are three subtypes of NcrB in the cell: the free state NcrB, the NcrB binding with pncrA and the NcrB binding with Ni2+. We assume that the concentration of total NcrB is not influenced by the expression of NcrB gene or its degradation process, because the speed of these two processes is relatively low compared with the competitively binding process. More importantly, based on this hypothesis, we can further assume that the concentration of free state NcrB remains constant during the process of competitive binding. This is because the kinetics feature of the process is similar to enzymatic reactions. </P><DIV class="notation"><TABLE><CAPTION>Notations</CAPTION><TBODY><TR><TD>Ki</TD><TD>the reaction rate constant of each reaction</TD></TR><TR><TD>Ni</TD><TD>the concentration of Ni<SUP>2+</SUP> inside the cell</TD></TR><TR><TD>Ne</TD><TD>the concentration of Ni<SUP>2+</SUP> outside the cell</TD></TR><TR><TD>E</TD><TD>the concentration of the Ni<SUP>2+</SUP> transporter protein NikABCDE</TD></TR><TR><TD>P</TD><TD>the promoter pncrA which binds with NcrB</TD></TR><TR><TD>B</TD><TD>the copy number of the recombinant plasmid in a single cell</TD></TR><TR><TD>LI</TD><TD>luminescence intensity</TD></TR></TBODY></TABLE></DIV><H4>The construction and the solution of the math model</H4><P>According to the hypotheses proposed above, the following dynamic process exists inside the engineering bacteria:</P><FIGURE class="text-center"><FIGCAPTION id="Probability">
                     dynamic proces
                 </FIGCAPTION></FIGURE><H5>1. The diffusion of Ni<SUP>2+</SUP></H5><P>We described this process through the differential equations below: </P><P>This system of equations is a set of first-order linear constant coefficient differential equations, and this system of equations has analytical solutions. </P><P>Where C1, C2, r1, r2 and D are fixed constants, </P><H5>2. The transcription of LuxCDABE conducted by NcrB</H5><P>Since the concentration of the free state NcrB remains constant, and the concentration of total NcrB is a function of the copy number of the plasmid, we can present the following equations:</P><P>Furthermore, we know that there is an equilibrium in the process of NcrB binding with Ni, which can be stated as follows:</P><P>Combine 2.1 2.2 2.3, and we can have:</P><P>As is proposed above, LI is linearly dependent on the number of free state promoter. That is to say,</P><P>Substitute 2.4 into 2.5, we can get the following equation:</P><P>Further substitute 2.6 into 1.4, now we have obtained the final expression of LI, in which time and the concentration of Ni2+ are the variables.</P><H4>discussion</H4><H5>1. Luminescence intensity versus time curve with a constant concentration of Ni<SUP>2+</SUP> ions</H5><P>According to the relevant data in the literature and obtained by experimental measurements, we roughly made a function image of the brightness signal as a function of time, and the image is as follows.</P><P>From the above figure, we can see that the intensity of luminescence increases with time. When the time is longer, the intensity of luminescence will gradually become a fixed value, that is,.</P><H5>2. Relationship between final luminescence intensity and Ni<SUP>2+</SUP> ion concentration</H5><P>From the above figure, it is not difficult to see that as the concentration of Ni2+ continues to rise, the final luminescence intensity is also increasing, and according to the formula derived above, this is a linear function relationship, which means that the linear response of luminescence intensity to Ni2+ concentration can be achieved by this system.</P><H3>HBUT-China: Experimentg</H3><H4>Overview</H4><P>Bioengineering uses stable, highly productive mutants, target strains, which contain foreign genes. However, screening these mutants costs vast time and workforce, and it is difficult to avoid using antibiotics. The  philosophy of our team is to use salicylic acid to induce glucose dehydrogenase expression to enhance the strain's pentose phosphate pathway, giving the strain a growth advantage. EmrR protein can inhibit the role of the downstream promoter P<SUB>emrR</SUB>, so that the gene downstream of P<SUB>emrR</SUB> can not be expressed, but salicylic acid (SA) can bind to emrR to eliminate this inhibition. Our team sets the gene of P<SUB>emrR</SUB> downstream to gdh, using SA to control this route. After the our team replaced gdh with mCherry, HUBT-China helped us induce the expression of mCherry by SA, measured its fluorescence intensity to determine the optimal concentration induced by SA and the optimal induction time at the optimal concentration.</P><H4>Protocol</H4><H5>I. Determination of the optimum concentration of SA for induction</H5><H6>Process:</H6><P>1. Pick a single colony from the top of the board, expand it with liquid LB containing Amp, and control the time at about 11h.</P><P>2. Prepare for SA induction verification:</P><P>(1) The whole experiment can be done in 3-5 parallel groups. In the ultra-clean platform, the expanded liquid is divided into several 20ml large centrifuge tubes, each tube is filled with 2ml. Pay attention to the cover of the large centrifuge tube when filling the bacteria solution, wrap it with the sealing film, and tighten the rubber band (because the sealing film can be ventilated, if the cover is covered with oxygen, it will be insufficient).</P><P>(2) Adding SA, so that the concentration of SA in the system is 0 mg / L (ie, without SA), 0.001 mg / L, 0.01 mg / L, 0.1 mg / L, 0.3 mg / L, 0.5 mg/L, 1 mg/L.</P><P>(3) 37-degree shake flask culture induction for 5 h.</P><P>(4) All the samples and the blank E.coli K12 of the expanded culture were placed in the upper 96-well plate (operated in a clean bench), using ELIASA to measure the fluorescence intensity and OD of each sample well, and the fluorescence was mcherry red fluorescence. (Absorption wavelength 580 nm, emission wavelength 610 nm).</P><H6>Results:</H6><P>As shown in the picture, when the SA concentration is 0.3-0.5 mg/L, the maximum fluorescence intensity per unit OD can be detected, so the optimum concentration for SA induction is 0.3-0.5 mg/L.</P><H5>II. Find the optimal induction time at the optimum concentration</H5><H6>Process:</H6><P>Find out the relationship between the time of induced expression and the fluorescence intensity at the SA concentration of 0.3 mg/L and 0.5 mg/L.</P><P>1. Expand the culture of puc19+emrr strain (note the addition of AMP resistance) for about 11 hours. </P><P>2. Set 3-5 parallel groups, in the ultra-clean platform, divided the expanded bacterial liquid into several 20ml large centrifuge tubes, each tube is filled with 2ml. Pay attention to the cover of the large centrifuge tube when filling the bacteria solution, wrap it with the sealing film, and tighten the rubber band(similar to the above).</P><P>3. Add SA to the system so that the SA concentration in all systems becomes the optimum.</P><P>4. Immediately measure the fluorescence intensity and OD value on a 96-well plate using ELIASA after the addition of SA.</P><P>5. Measure the fluorescence intensity and OD every 30 min, after 5h, record the experimental data.</P><P>6. Making the fluorescence intensity- time image of the unit OD and find the optimal induction time.</P><H6>Results:</H6><P>As shown in the picture, the fluorescence intensity per unit OD reached the highest at 3.5 h, so the optimum induction time at the optimum concentration is 3.5 h.</P><H2>2.HUST-China &amp; BNU-China</H2><P>At first, our team intent to develop a new synthetic biological method for preventing desertification and fixing sands. We found that in 2015 HUST-China design a system succeed in stabilizing concrete underwater by using viscous protein Mcfp-3 and silica binding protein Si-tag, whose parts have potential to be used in our own new design. Hence, we contacted their team member and communicated about these parts’ characteristic. They nicely provide us what they know about these parts and some tips in processing the experiments. Their corresponding page can be accessed here:
                <A href="https://2018.igem.org/Team:HUST-China/Collaborations">Team:HUST-China/Collaborations</A></P><H2>3.ZJUT-China &amp; BNU-China</H2><P>ZJUT-China had a similar topic in this year’s iGEM competition. Both of us concerned with the effects of antibiotics on people’s health, we agreed that the abuse of antibiotics is a serious hazard to public citizens. As a result, we established a collaborating program to produce a manifesto, calling for laws to regulate the use of antibiotics and biotechnology. Our project aims to enhance the whole nation’s awareness of the danger incident to the overuse of antibiotics. Our manifesto can be accessed here. Their corresponding page can be accessed here:
                 <A href="https://2018.igem.org/Team:ZJUT-China/Collaborations">Team:ZJUT-China/Collaborations</A></P><H2>4.UCAS-China &amp; BNU-China</H2><P>In our collaboration with University of Chinese Academy of Sciences (UCAS), we took up most of art-designing project. In order to make the profound knowledge more accessible to ordinary people, we decided to demonstrate synthetic biology in the form of animation and cartoon. Our team member Shujuan Jiang successfully designed a series of imaginative illustrations, greatly enriched the details of wiki pages. Moreover, we worked together to develop a calender about synthetic biology. Their corresponding page can be accessed here:
                  <A href="https://2018.igem.org/Team:UCAS-China/Collaborations">Team:UCAS-China/Collaborations</A></P><H3>UCAS-China:Experiment</H3><P>UCAS-China validated the SA-emrR system in controlling expression under our guidance. Their protocol is the same as HBUT-China’s except for the concentration of salicylic acid(SA). In addition, UCAS-China also provided us with flow cytometry, which allowed us to obtain more accurate experimental data in the start-stop codon overlap verification experiment and glucose dehydrogenase anti-degeneration experiment. More details about the result of experiment can be assesses here:
                    <A href="https://2018.igem.org/Team:BNU-China/Results#linked_to_collaborations">Results-Module 4:Stop-Star Codon “TGATG”</A></P><H2>5.OUC-China &amp; BNU-China</H2><P>How to promote synthetic biology knowledge for the public has always been an important proposition of human practice of the iGEM team. Popular science comics is undoubtedly a very popular carrier to promote synthetic biology. The science comics created by OUC-China are vivid and interesting. Our team just needs an attractive carrier to carry out the next promotion and publicize synthetic biology. So our two teams have cooperated here, OUC-China has provided us with a full set of popular science comics, and we are responsible for promoting comics, we have done the following work:</P><P>(1)Use popular science comics to popularize the propaganda;</P><P>(2)Use science comics to attract everyone's attention when holding outfields;</P><P>(3) Promote comics on the official Weibo of Beijing Normal University.</P><P>In the end, OUC-China's popular science comics were well received by everyone, and the official Weibo was read by more than 20,000 people. We believe that this approach greatly enhances the public's awareness of synthetic biology. Their corresponding page can be accessed here&quot;
                <A href="https://2018.igem.org/Team:OUC-China/Collaborations">Team:OUC-China/Collaborations</A></P><H2>6.UESTC-China &amp; BNU-China</H2><H3>BNU-China:</H3><P>This year, UESTC-China is committed to converting cellulose to butanol for the purpose of recycling waste paper. However, during the experiment, it was found that the metabolic pathway of butanol and the ethanol metabolic pathway of E. coli antagonized each other, resulting in low yield of butanol. Therefore, it is necessary to knock out the adhE producing ethanol to increase the yield of butanol. Our team happens to have a set of gene knockout crisper cas9 system. For UESTC-China adhE gene, we designed knockout vector and guided adhE gene knockout.</P><H3>UESTC-China:</H3><P>At the same time, UESTC-China also helped us to conduct salicylic acid-induced emrR system expression experiments. The protocol is the same as the above school, details are shown in HBUT-China &amp; BNU-China on the previous page.</P><H2>7. Peking &amp; BNU-China</H2><P>Peking provided us with a microplate reader to make our data collection smoother. In the meetup sponsored by Peking, 2016iGEM member Li Cheng provided us with guidance. Their corresponding page can be accessed here:
                <A href="https://2018.igem.org/Team:Peking/Collaborations">Team:Peking/Collaborations</A></P><P>We have alos collaborated with a high school team-------SBS-Sh-112144------- in the area of modeling. We discussed about how to use anneal arithmetic to build a 3D model on matlab.</P><P>This summer we participated in three meetups, hosted by Peking, UCAS-China and ShanghaiTech (CCiC). We share our ideas in meetup and actively seek for cooperation.</P></ARTICLE></DIV></DIV><DIV id="footer"><DIV id="footer-div"><UL id="footer-ul"><LI><A href="https://2018.igem.org/Team:BNU-China">Home</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Background">Project</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab">InterLab</A></LI></UL><SPAN>© 2018 BNU</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>