<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:BNU-CHINA - 2016.igem.org</TITLE>
style = &quot;font-family: Helvetica;&quot;
</P><DIV class="header-container"><NAV class="navbar navbar-fixed-top navbar-inverse nav-adjust" role="navigation" style="margin-top: 14px;"><DIV class="container-fluid nav-container scrolled-top"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" id="navbar-toggle"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" id="navbar-brand" href="https://2018.igem.org/Team:BNU-China" style="font-family: Helvetica;">BNU-China</A></DIV><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2018.igem.org/Team:BNU-China" class="" style="font-family: Helvetica;">Home</A></LI><LI class="dropdown "><A href="https://2018.igem.org/Team:BNU-China/Project" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Project </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Background" style="font-family: Helvetica;">Background</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Design" style="font-family: Helvetica;">Design</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Results" style="font-family: Helvetica;">Results</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Improve" style="font-family: Helvetica;">Improvement</A></LI></UL><LI class="dropdown focus"><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Modeling </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Main" style="font-family: Helvetica;">Main model</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Probability" style="font-family: Helvetica;">Probability of distribution</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#E" style="font-family: Helvetica;">E.coli growth rate</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Growth" style="font-family: Helvetica;">Growth promoting effect</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Gene" style="font-family: Helvetica;">Gene expression level</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model#Ribosome" style="font-family: Helvetica;">Ribosome fall-off rate</A></LI><LI class="dropdown "><A href="./achievement.html" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Measurement" style="font-family: Helvetica;">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Demonstrate" style="font-family: Helvetica;">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Judging" style="font-family: Helvetica;">Judging</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Parts" style="font-family: Helvetica;">Parts</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Overview" style="font-family: Helvetica;">Overview</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Silver" style="font-family: Helvetica;">Silver</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Gold" style="font-family: Helvetica;">Gold</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Social Activities" style="font-family: Helvetica;">Social Activities</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Human_Practices#Public " style="font-family: Helvetica;">Public Engagement</A></LI><LI class=""><A href="https://2018.igem.org/Team:BNU-China/Safety" style="font-family: Helvetica;">Safety</A></LI><LI class="dropdown "><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">Team </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Team#" style="font-family: Helvetica;">Members</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Attributions" style="font-family: Helvetica;">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Collaborations" style="font-family: Helvetica;">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Acknowledgement" style="font-family: Helvetica;">Acknowledgement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Notebook" style="font-family: Helvetica;">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Protocol" style="font-family: Helvetica;">Protocol</A></LI><LI class="dropdown "><A href="https://2018.igem.org/Team:BNU-China/InterLab" class="dropdown-toggle" data-toggle="dropdown" style="font-family: Helvetica;">InterLab </A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Calibration" style="font-family: Helvetica;">Calibration</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Cell Measurement" style="font-family: Helvetica;">Cell measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#CFU Measurement" style="font-family: Helvetica;">CFU measurement</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/InterLab#Epilogue" style="font-family: Helvetica;"> Epilogue</A></LI></DIV></DIV></NAV></DIV><DIV class="main-container"><DIV><DIV class="page-story"><ARTICLE id="Calibration" class="col-lg-10 col-lg-offset-1 col-md-12 col-md-offset-0 col-sm-offset-0 col-sm-12"><HEADER class="page-header"><H1 id="model-introduction">Results</H1></HEADER><H2>Result: Specific gene circuit</H2><H3>Module I:The growth-promoting effects of glucose dehydrogenase(GDH)</H3><P>
Several studies have proven that overexpression of GDH does have growth-promoting effects on bacteria. In our experiments, E. coli was used as the chassis species and the gene sequence of GDH came from the genome of E. coli K12 strain.</P><P>Former studies only applied SDS-PAGE Gel electrophoresis in order to confirm the existence of GDH, which merely provides the information of molecular weight. In order to validate the existence of GDH precisely, two tags (6xHis and V5) were added at the C-terminal of the protein, which enabled further experiments including purification and western blot. Eventually, we proved the expression of our recombinant plasmid (Fig. 14a) in E. coli through western blotting (Fig. 14b).</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.14 Overexpression of glucose dehydrogenaseA). Design of the experimental plasmid. B) Western-blot image of the overexpressed GDH, from left to right are: bacteria with the recombinant plasmid, bacteria with the wild-type plasmid and wild-type bacteria (no plasmid transformed). C) the growth curve with a dilution ratio of 1:10, y-axis represents OD<SUB>600</SUB> and x-axis represents time. Through all the experiments, M9 media is used. The original OD<SUB>600</SUB> is 1.60. D) The bacteria growth curve after a dilution of 1:50, original OD<SUB>600</SUB> is 0.58.
                    </FIGCAPTION></FIGURE><P>2. Experiments in labs are always carried out with antibiotics present. Therefore, to imitate the antibiotic-free condition in industrial procedures, we improved our protocol. Firstly, we used M9 media with equal IPTG added to ensure a reliable outcome. No antibiotics was added to the media. Cultures were incubated overnight in LB media (liquid) with Ampicillin, and OD<SUB>600</SUB> of all cultures were adjusted to the same value using an UV Spectrophotometer. We then diluted the cultures to 10 or 50 times using antibiotic-free media and incubated them in a shaker (37°C, 200rpm). By measuring the OD<SUB>600</SUB> value of samples taken under proper timing, growth curves of different strains could thus be drawn (Fig. 9c and Fig 9d). Conclusively, we found that the strain with GDH gene grew faster than others, while the wild-type one took the second place. The strain with an empty vector transformed grew the lowest, which behaved as predicted, for an additional metabolic burden was added. These results showed GDH does promote the growth of bacteria.</P><P>Thus far, the verification of the ability of promoting bacteria’s growth of GDH had been finished. However, our final goal was to make the strain with GDH grows faster than those without a target gene, and becomes the dominant population of the system accordingly. Combining previous works and our data, some discussions are made as follows. We can see from Fig. 9c and 9d that overexpressed GDH helped in the increase of K value, while previous articles showed that dry weight of those with GDH overexpressed accumulated more during the same amount of time. Therefore, we could conclude that overexpressed GDH can improve the metabolic efficiency of bacteria, and such a character has the ability to resist the problem of target strain’s degeneration caused by the unequal distribution of plasmids. Meanwhile, in a fermentation system with vast amount of strains, those with higher metabolic efficiency can be seen as more “advantageous”. However, to what extent can the overexpression of GDH resist the trend of degeneration of strains with target genes remains unknown. The key to this problem is to validate the ability of GDH in keeping target genes. Hence, two tasks are needed to be carried out in this module: 1. Verify if GDH has the ability to keep target genes. 2. Build a persuasive model to answer the question stated above (To what extent can the overexpression of GDH resist the trend of degeneration of strains with target genes?).</P><H3>Module 3: pchBA producing salicylic acid</H3><P>In this module, we used the part BBa_J45319 given by iGEM authority to transform E. coli strain BL21 (DE3). Salicylic acid was confirmed generated after several times of induced expression with IPTG and liquid chromatography testing of the supernatant. The harvest was 10mg/L, more or less (Fig. 15a). For the convenience of further testing, we also drew a standard curve (Fig 15b) of salicylic acid’s concentration with a series of standard salicylic acid samples of which the concentration was known.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.15 a) The liquid chromatogram of salicylic acid in the supernatant of our cultures b) The standard concentration curve of salicylic acid
                    </FIGCAPTION></FIGURE><H3>Module 2: Salicylic acid and emrR binding promoter binds emrR protein competitively</H3><P>This module is a transforming unit of the whole gene circuit, through which the signal of salicylic acid is transformed to the advantage in bacteria growth. The point of this module is to verify the inhibition effect of the repressor protein and the disinhibition effect of salicylic acid. Thus, we designed a part which expresses the repressor constantly, and an mcherry gene is placed downstream of the emrR binding promoter. Theoretically, this mcherry can only be expressed under the circumstance that salicylic acid is added to the media.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.16 A) The expression of emrR prevents the initiation of P<SUB>emrR</SUB>, but the protein changes its configuration after binding with salicylic acid and leaves the promoter region, thus enables the expression of downstream proteins (in this case, mcherry). The promoter of emrR is a constant promoter, acquired from E. coli, originally responsible for the initiation of GADPH gene. When salicylic acid exists, red florescence can be observed. B) Control strain under white light, no salicylic acid added. C) Control strain under green light excitation, no salicylic acid added. D) Experiment strain under white light, SA conc. is 5mg/L. E) Experiment strain under green light excitation, SA conc. is 5 mg/L.
                    </FIGCAPTION></FIGURE><P>The result of our experiment showed very low expression level of RFP when there was no salicylic acid in the media. When a concentration of 5~10mg/L salicylic acid was added to the media, the bacteria showed a peak level of red florescence intensity (induction time: 4h). The florescence intensity had not risen significantly when the concentration of salicylic acid was further increased. Meanwhile, as the concentration of SA increased, growth of the bacteria seemed to be repressed. We also found that cultures incubated in the 96-well plates did not show significant difference between control and experiment, but visible difference was observed when they were incubated in 10ml tubes. This probably due to the different growing environment of E. coli, indicating that the induction procedure of salicylic acid needs relatively large incubating space.</P><P>In conclusion, salicylic acid induction needs larger space, while emrR is highly sensitive when binding with SA, for that the peak level of expression is reached under very low SA concentration. However, the peak level is indeed not very large, which would possibly make the growth promoting effect not obvious, for a quite low level of SA would enable the expression of GDH keeps at a constant (peak) speed, which is quite low as well. Consequently, the harvest of SA would not obviously increase. Possible optimizations may focus on these following aspects: 1. Since the key problem is the low expression level of the growth promoting factor (GDH) and realizing that the RBS of emrR binding promoter from E. coli genome is not efficient, we could replace it with another RBS with higher ribosome binding efficiency. 2. Low expression level may ought to the fact that the circuit shown in Fig 16 is sensitive to salicylic acid. Thus, increase the level of expression of emrR could possibly decrease the sensitivity.</P><P>Salicylic acid does play the role of launching emrR binding promoter, but the expression level of downstream gene is too low after that, which cannot be applied to the whole gene loop, and optimization of the gene loop is needed. Direct optimization of the coding reading frame to optimize protein structure is the most fundamental method, but it takes long time and has a low success rate. Therefore, we intend to change the RBS of emrR binding promoter to increase the translation speed of mRNA. RBS B0034 was introduced on the original loop ultimately (Figure 17).</P><P>The expression of the optimized vector was significantly increased compared to that using the wild promoter and RBS. In order to perform repeated experiments more efficiently, we sent the plasmid dry powder to HBUT-China and UCAS-China for different experiments. The experimental results of HBUT-China showed that the expression of mcherry fluorescence was not significant under the induction of low concentration of salicylic acid, and there were peaks of red fluorescence under the induction of 0.3 and 0.5 mg/L. Further experiments showed that the optimal induction time of salicylic acid was above 4.5h. Next, UCAS-China helped us to carry out experiments using high-concentration salicylic acid. After 6 hours of induction, the amount of mcherry fluorescence in the experimental group was significantly different from that in the control group, and the higher the salicylic acid concentration is, the red fluorescence intensity is stronger.</P><P>The optimal concentration of salicylic acid was above 40 mg/L, and the protein expression of the vector after the modification was significantly increased compared to the previous vector (Fig. 17A).</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.17  A) Optimized experimental pathway, a standardized RBS is adding before mcherry: B0034. B) induction effect of low concentration salicylic acid (0 ~ 1mg / L). C) The fluorescence expression changing with time when salicylic acid concentration is 0.3mg / L and 0.5mg/L. Data from Hubei University of Technology (HBUT-China). D) After induction of high concentration of salicylic acid (0~40mg/L) for six hours the relationship between salicylic acid concentration and fluorescence/OD. The horizontal axis is salicylic acid concentration (mg/L), the vertical axis is fluorescence/OD. Excitation: 560 nm, emission: 610 nm. Data from National University of Science and Technology (UCAS-China). E) The salicylic acid concentration from 0 to 40 mg/L(from left to right) after centrifugation of the solution of the induced bacteria.
                    </FIGCAPTION></FIGURE><H2>Result: universal gene circuit</H2><H3>Module 5:λCI repressor</H3><P>We used the part BBa_C0051 (provided by iGEM authority) to acquire the CI gene and further connected it after the lac promoter. To simplify the detecting procedure, V5 tag was added to the C-terminal end of the protein. Through SDS-PAGE and western blotting, we validated that the CI protein was successfully expressed in our target strain. Meanwhile, we applied the part BBa_K1139151 (sequence uploaded by Tokyo-Tech 2013, sample synthesized by ourselves), a hybrid promoter which can be regulated by both lacR and λ CI repressor. The promoter reaches its highest intensity of expression when both IPTG and CI exists. At the downstream of the promoter, a GFP was connected to simplify further experiments.</P><P>For the reason that the expression of both CI and GFP in our circuit are at least partially controlled by the concentration of IPTG, merely comparing the florescence intensity of our target strain between IPTG’s present and absent is not abundant to describe the function of CI since there is a possibility that the induction effect may be caused by adding IPTG only. Considering this point, we designed another control vector which only has the hybrid promoter and GFP downstream but lacks CI. We separately transformed these two plasmids to E. coli BL21(DE3) to make the control strain and the experiment strain. Therefore, through comparing the difference between IPTG’s present and absent in each strain, and the difference between the differences (previously mentioned) of the two strains, we could thus describe the function of CI rigorously.</P><P>To state it in detail, we first obtained the florescence intensity (FI) and OD<SUB>600</SUB> values of different cultures (4 in total), and calculated the single cell florescence intensity (SFI):</P><P align="right">SFI = FI / OD<SUB>600</SUB>, for a certain culture</P><P>There’s no harm to define the total induction effect (IEtot) of IPTG as follows:</P><P align="right">IEtot = SFI of the culture with IPTG present / SFI of the culture with IPTG absent,
for a certain strain</P><P>According to our gene circuits, IPTG activates both GFP and CI (which further activates GFP as well) in the experiment strain. While in the control strain, IPTG only activates GFP directly. Therefore, the neat induction effect of CI (NIECI)can be simply calculated using the formula provided below:</P><P align="right">NIECI = IEtot of the experiment strain - IEtot of the control strain</P><P>We first incubated monoclones of each strain in LB media with ampicillin overnight to obtain cultures reaching the plateau state of growth. We then dilute the culture separately to a final OD<SUB>600</SUB> value of 0.1 using fresh LB media with ampicillin, and incubated them again until the OD<SUB>600</SUB> value reached 0.5. IPTG was added to half of these cultures, respectively. Timing started here and this point was marked as 0h. Samples were taken every one hour (0h included) until the value of single cell florescence intensity (SFI) stopped changing significantly (roughly 4 hours, according to our experience). OD<SUB>600</SUB> and florescence intensity were measured through a plate reader.</P><P>During a series of repetition experiments, we discovered that under the induction of IPTG, the increase of single-cell fluorescence of the experiment group is greater than the control. When CI-protein exists, the expression level of GFP gene will increase 1.5 times than the control, which indicates that CI-protein has the ability to improve the downstream gene’s expression. However, on the condition of the anti-cascading mechanism of TGATG linkage or the hairpin structure, present result can hardly satisfy the growth advantage as we expected.</P><P>The primary cause is the dosage of IPTG inadequate or too strong. The experimental group has two IPTG-inducible promoters, including the CI and GFP promoter. However, the control only has GFP promoter. As a result, the exogenous IPTG in control can be fully utilized to induce the expression of GFP but when it comes to experimental group, the amount of exogenous IPTG doesn’t equalize that for GFP expression.</P><P>As a consequence of we don’t have accurate data for the expression-level and the sensitivity to IPTG, in order to guarantee the induction dosage is adequate in two systems, we designed a gradient-experiment with the IPTG concentration of 0.5mM, 1mM, 2mM and under 1mM we observed the most conspicuous activation phenomenon, whereas the strains had already lysed under 2mM.</P><P>
All the same, we can’t fully guarantee that the concentration of 1mM IPTG is overdose, which is affected by the direct or secondhand interaction between two lac promoters. In the follow-up experiments, we will replace CI-promoter with a constant-expression promoter to avoid the concentration problem. Moreover, we can screen out separated dots to modify the deviation caused by equipment problems.</P><P>
Besides, it is reported that the co-expression of the carboxyl of CI-protein and the C-Terminal Domain of bacterial RNA polymerase can greatly improve motivation effect by two orders of magnitude, which will remain to be tested in future experiments.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.18  A) The western blot image of CI, marker on the right. The protein is between 34kD and 25kD, which is precisely the exact molecular weight of itself. B) Data of the induction test. Each row indicates one time point and the columns from left to right represent IEtot of experiment strain, IEtot of the control strain and CI effect respectively. C) Bar gram drawn using the data presented in c to give a more intuitive impression.
</FIGCAPTION></FIGURE><H3 id="linked_to_collaborations">Module 4: Stop-Star Codon “TGATG”</H3><P>We did some experiments to detect the ratio in SAM.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.19  Plasmids constructed for testing TGATG efficiency.
</FIGCAPTION></FIGURE><P>Four plasmids were constructed on the backbone named pUC-yder. “Control-1” only expresses GFP, as well as “Control 3”. “Control 2” shows the result of GFP expression overlap with m-cherry. “Experiment” shows the ratio between GFP expression and m-cherry expression. After transfected into <I>E.coli</I> stbl3, we confirmed the plasmid with DNA sequencing. Also we activated the wild type and used IPTG to induce the expression of all target genes.</P><P>The detect process was done with FCM(flow cytometry).</P><P>In experimental group, we set X-axis as FSC and Y-axis as SSC to circle the living bacteria which is in green, yellow and red color. After that, we changed the X-axis as FITC-A(represent for green fluorescent intensity)and Y-axis as Histogram(represent for bacteria density), then selecting the highest single peak around X-axis coordinate 103. We recorded the number of bacteria in this green fluorescence peak (named Group 1). Then we changed X-axis to mCherry-A (represent for pink fluorescent intensity) and used similar method to record the number of bacteria with pink fluorescence in Group 1 (named Group 2). An average of 20% of the bacteria in Group 1 were included in Group 2. The geometric mean of fluorescence of mCherry and GFP also be recorded.</P><P>In control group 1 and 3, almost the same method is used to find the wanted statistics. We didn't need to change X-axis after selecting the Group 1, neither do we need to record the number of bacteria.</P><P>In control group 2, we firstly processed the living bacteria and then we set X-axis as FITC-A and Y-axis as mCherry, using Quad to split the whole space into 4 parts. Q3 is the part without any fluorescence, Q1 and Q4 is the place in which bacteria can only express single kind of fluorescence. Q2 is representing for bacteria express two kinds of fluorescence and is what we want and we use the similar method to record The geometric mean of two kinds of fluorescence.</P><P>By comparing the relative fluorescent intensity in Group 2 with that of Con2(Part:BBa_K2717016), we found the TGATG causes a 13.2% reduction to pink fluorescent intensity. That is to say, using TGATG can succeed in reducing the expression of following gene.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.20  FCM data and processing.
</FIGCAPTION></FIGURE><H3>Integrated Plasmid.</H3><H3>A) Plasmid where GFP is directly linked to gdh by “tgatg” or haripin.</H3><P>Previous experimental results show that gdh can bring growth advantages to the strains and implement screening simultaneously. TGATG can make the two genes combined by it express in a certain proportion. In addition, we found that the hairpin structure can bring a better coupling expression effect than TGATG by consulting the literature. Therefore, we constructed the pathway as follow: We combined the TGATG with gdh and we also couple up hairpin and gdh, in order to obtain high-yield strains that can automatically screen out highly expression, which initially confirms the design of the final project.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.21  the pathway of each experimental group Control1 GFP: Part BBa_J364007.Control2 Uncoupling Expressing: Part BBa_J364007 and gdh were reverse inserted into the plasmid, which can express respectively. Experiment1 Stop-Start Codon Overlap: lac promotor +RBS B0034+gfp+TGATG+gdh+terminator. If the expression ability of the strain is strong enough, gdh will be expressed, whereby the strain gains a growth advantage. At the same time, due to the TGATG, the expression level of GFPhas been very high. Gdh promotes the metabolism of strains, which means it will express mor gdh to gain growth advantage, resulting in more GFP being expressed. Experiment2 HIRPIN: lac promotor +RBS B0034+gfp+hairpin structure+gdh+terminator.
</FIGCAPTION></FIGURE><P>We constructed 4 plasmids on the backbone named pUC-yder(Ampr). “Control-1” only expresses GFP, which can be used to verify the screening effect caused by gdh and the high yield features by TGATG. “Control 2” makes the GFP and gdh express respectively. This will confirm that the introduction of TGATG allows the plasmid to highly express the gene of interest (the upstream gene of TGATG), thereby avoiding the overall metabolic enhancement effect brought by gdh. The experimental group are as shown above.</P><P>After transforming plasmids into <I>E.coli</I> trans5α, we inoculated recombinant bacteria in M9 medium supplemented with amp and calibrated OD<SUB>600<SUB> = 1 for each group. After a while, we measured the fluorescence intensity and OD<SUB>600<SUB> of each group. Fluorescence intensity / OD<SUB>600<SUB>is used to roughly indicate the amount of fluorescence expression per cell, so that we can rule out the deviation caused by the number of cells.</SUB></SUB></SUB></SUB></SUB></SUB></P><P>The experimental results show that the average fluorescence intensity of Stop-Start Codon Overlap and HAIRPIN were significantly higher than that of control1, which indicates that the introduction of TGATG/hairpin and gdh successfully screened high-yield strains with strong expression ability. It was higher than the control2, indicating that the introduction of TGATG/hairpin brought gdh’s growth advantage for strains with high expression of GFP. The results of this experiment preliminarily prove that our final design is feasible.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.22  average fluorescence intensity of each group average fluorescence intensity= Fluorescence intensity / OD<SUB>600</SUB></FIGCAPTION></FIGURE><P>After 24h, the average expression: Experiment2 HAIRPIN&gt; Experiment1 Stop-Start Codon Overlap&gt; Control1 GFP&gt; Control2 Uncoupling Expressing</P><H3>Plasmid where GFP is linked to gdh by “TGATG”, T7 polymerase and T7 promoter.</H3><H4>Experimental design</H4><P>The experimental group's bacteria were transferred to our designed T7 final part. Our system hopes to reduce the target gene degradation rate and prolong the plasmid retention time by positively promoting growth, thus replacing negative selection with antibiotics. Our experiments are built around the purpose of verifying this. In this experiment, GFP was used as the target gene. We can determine the expression level of the target gene by measuring the fluorescence intensity of GFP. The fluorescence intensity of a certain volume of bacteria is not only affected by the amount of GFP in the bacteria, but also the total number of bacteria in the bacteria. In order to judge the expression level of GFP by fluorescence intensity more accurately and scientifically, we chose to use the fluorescence intensity/od value at the same time to finally measure the level of GFP expression of the strains transferred to different plasmids(Single cell fluorescence Value = culture fluorescence value / culture OD<SUB>600</SUB> value). The ability of the system to stably express the target gene was further judged. In practice, the GFP gene can be replaced with any target gene to produce the desired product. In the control group, the plasmid we transferred contained only the lactose operon, GFP and ampicillin resistance genes. By comparing the single cell fluorescence  ii.values of the experimental and control groups under the same conditions, we can analyze the
effect of the T7 final part. In order to prove that our system can prevent the non-expression of the target gene and the loss of the plasmid due to unstable plasmid structure without antibiotics, we set the control conditions for adding antibiotics (ampic acid) and no antibiotics. On the other hand, since the plasmid backbone we use contains the lactose operon region, the lactose operon may affect the expression of GFP to some extent. So we set the control conditions for IPTG induction and no IPTG. The two control conditions of the group were combined to form four different culture conditions: no IPTG without ampicillin, IPTG without ampicillin, no IPTG with ampicillin, and IPTG with ampicillin. We simultaneously cultured the E. coli of the experimental group and the control group under these four conditions, and measured the fluorescence intensity and OD value every hour.</P><P>Specifically, we transferred the two plasmids to the strain containing no T7 RNA Polymerase, and picked the monoclonal in 20 mL of LB liquid medium supplemented with ampicillin, and cultured at 37 °C at 200 rpm overnight. When the OD<SUB>600<SUB> reaches about 3, the bacterial solution is inoculated into the new ampicillin-resistant LB liquid medium and the LB medium without ampicillin according to the final OD<SUB>600<SUB>=0.1, and cultured at 37 °C at 200 rpm until the logarithmic growth phase (OD<SUB>600<SUB> in the range of 0.5~1.0, it takes about 1~2 hours). At this time, part of the culture solution was added IPTG to 1.5 mM , and the other was not added, so that both the experimental group and the control group contained the above four different culture conditions. This time was recorded as time 0, and the bacterial liquid in each bottle was quickly taken in 96 hole plate. In the plate, each bottle added four samples, and the OD<SUB>600<SUB> and fluorescence values were measured in a enzyme-labeled instrumen. Thereafter, the bacteria solution was added to the 96 hole plate every hour, and the OD<SUB>600<SUB> and fluorescence values were measured for 5-6 hours until the fluorescence value was substantially stable and no longer increased. And data for various treatments per time period were obtained. Finally, the fluorescence intensity corresponding to each OD at each time is calculated according to the method already stated above.</SUB></SUB></SUB></SUB></SUB></SUB></SUB></SUB></SUB></SUB></P><H4>Analysis and discussion of results</H4><FIGURE class="text-center"><FIGCAPTION>
                        Fig.23 A) No IPTG without ampicillin. B) IPTG without ampicillin. C) No IPTG with ampicillin. D) IPTG with ampicillin.
</FIGCAPTION></FIGURE><H5>Preliminary analysis of results</H5><P>We have obtained the experimental results shown in the above figure through repeated experiments. Through the experimental results, we found that the system we constructed with TGATG and T7 RNA Polymerase as the core did not exert our expected results. The single-cell fluorescence intensity of the control group in the overall experimental results was generally greater than that of the experimental group, which is contrary to our expectation of increasing yield. One possible reason is that GFP is a smaller protein meanwhile T7 RNA Polymerase and gdh are both relatively large proteins, so the T7 final part plasmid transferred into the experimental group is much larger than the plasmid transferred into the control group. Therefore, compared with the GFP-only strain, the strain containing the T7 system has a greater burden due to the production of T7 RNA Polymerase and gdh, so that the strain with the T7 system has a lower yield than the strain containing only GFP. In addition, by reviewing the literature, we find T7 promoter is a very strong promoter. It has been shown in the literature that it can be ligated into a medium copy number plasmid to produce 50% of the protein in the whole cell, not to mention the vector we use is a high copy number plasmid. Therefore, when a large amount of gdh is produced, a large amount of nutrients are consumed for the production of gdh. So that other biochemical metabolic pathways within the cell are inhibited. This is adverse to the growth of cells. For these two issues, on the one hand, we speculate that if the sequence of the target gene is relatively long and the expressed protein is more complex, that is, when the target gene is expressed, the bacteria will be subjected to a certain pressure, our system will have more significant effect. On the other hand, we should look for a promoter with the suitable amount of expression to control the expression of GDH. At the same time, we can look for smaller and more efficient growth-promoting factors to further improve our system.</P><P>Through the comparison in Figure 22, we can clearly see that the single cell fluorescence of Figure 22D is higher than that of Figure 22B, and the single cell fluorescence of Figure 22C is higher than that of Figure 1. Scilicet, the E. coli target gene in both the experimental group and the control group was higher expressed under the condition of adding ampicillin. This shows that the traditional method of antibiotic resistance selection is very effective, and it can indeed stably express the target gene by reducing the structural damage of the plasmid or the loss of the plasmid. On the other hand, our system can effectively play the above role with low concentrations of antibiotics, and our system can reduce the use of antibiotics to a certain extent.</P><P>From the experimental results, we can find out whether the addition of IPTG has little effect on the final expression of GFP. Even if IPTG is added to some extent, the fluorescence of single cells will decrease. This phenomenon is not in line with expectations, because the lactose operon should be promoted by IPTG to promote the expression of downstream genes, that is, the expression of GFP should be increased after the addition of IPTG. We initially believe that it may be that the promoter we use is strong and the response to IPTG is high. The final concentration of IPTG we calibrated was 1.5 mM/L. The IPTG highly concentrated caused a large amount of response from the promoter. We used a high copy number plasmid, which increased the load of E. coli and eventually caused some bacterium overload to death. In the follow-up experiments, we will reduce the concentration of IPTG or remove the lactose operon region from the plasmid to solve this problem.</P><H2>Further Improvement</H2><H3>Gradient antibiotic concentration experiment</H3><H4>Design</H4><P>In our anti-degradation verification experiment, we found that the anti-degeneration effect of the genetic loop we constructed against the uneven distribution of the plasmid was not obvious, but it was also found to have a growth-promoting effect, that is, the growth rate of the strains introduced the plasmid was higher than that of the control group. It proves that the gene loop can make the strains containing the gene expressing the target product have a growth advantage, thereby achieving high yield. However, since the plasmid may still be lost, we need other means to help retain the plasmid. The experiments have shown that low concentrations of antibiotics can retain plasmids, and we have learned that low concentrations of antibiotics do not have much cost and downstream separation burden in actual production, so we envisage assisting in the retention of plasmids by adding lower concentrations of antibiotics. Since the plasmid-containing strains have a large growth advantage, we can think to a certain extent that the growth rate can indirectly reflect the plasmid retention rate, and the two are positively correlated. In addition, in the presence of antibiotic selection pressure, the retention rate of the plasmid increases with increasing antibiotic concentration, but at the same time, it will also exert pressure on the growth of the strain, which will slow down the growth rate of bacteria. Therefore, in order to obtain the optimum concentration of antibiotics to maximize the growth rate, we set up experiments with different antibiotic concentration gradients to repeatedly measure growth curves. Therefore, the concentration of antibiotics with the highest plasmid retention rate can be determined indirectly by the concentration of antibiotics that can grow bacteria fastest.</P><H4>Results</H4><P>It can be seen from the experimental data that the growth rate of the strain increases with the increase of antibiotic concentration within a certain range, but after reaching an optimal concentration, the increase of antibiotic concentration will cause the growth curve to decline. This is because, when antibiotic concentration has little stress on growth, as the concentration of antibiotics increases, the growth advantage of the strain gradually appears, and the retention rate of the plasmid gradually increases. Therefore, before the concentration of the antibiotic reaches the optimum concentration, the growth curve gradually rises. But after the concentration of the antibiotic reaches the optimum concentration, the pressure of the antibiotic on the growth exceeds the positive effect of the growth advantage of the retained plasmid on the growth rate. Therefore, as the concentration of the antibiotic is increased, the growth rate declines.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.24 Results of gradient antibiotic concentration experiment
</FIGCAPTION></FIGURE><H3>Gene knockout of gdh</H3><P>The current experiments show that the plasmid retention ability of gdh does not meet our original expectations, but there is only few factors as we know that can effectively promote growth, so it is still a feasible solution to prevent the degradation of engineering bacteria by the growth promoting effect of gdh. In the program, we try to use the growth-promoting function of gdh to achieve the purpose of retaining the plasmid as much as possible, so as to have a positive effect on industrial production.</P><P>Since gdh already exists in the genome of the strain, we cannot determine the exact function of exogenous introduction of gdh. Therefore, we knocked out the gdh on the genome of the strain, making the growth of the strain after gdh knockout restricted, and importing gdh with exogenous source. The growth rate difference of the strain is larger than before. That is, the growth advantage of the strain exogenously introduced into gdh after knocking out the genome gdh is more obvious than that of the defective strain, so that the growth promoting function of gdh can be more clearly manifested. If gdh does have a certain plasmid retention capacity, under this apparent contrast, we think that the plasmid retention time should be longer when the exogenous gdh is introduced and fully expressed after knocking out gdh on the genome, while other strains, including strains that have lost the plasmid and the strains with weak expression ability, will be eliminated due to slow growth, thus achieving the function of screening the plasmid-preserving strains, which is of great significance for the industrial production of engineering bacteria.</P><FIGURE class="text-center"><FIGCAPTION>
                        Fig.25 Colony PCR results of gdh gene knockout.
</FIGCAPTION></FIGURE></ARTICLE></DIV><DIV id="footer"><DIV id="footer-div"><UL id="footer-ul"><LI><A href="https://2018.igem.org/Team:BNU-China">Home</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Project">Project</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Integrated_Practices">Practices</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Lab">Lab</A></LI><LI><A href="https://2018.igem.org/Team:BNU-China/Interlab">Interlab</A></LI></UL><SPAN>© 2018 BNU</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>