"
Page
Discussion
View source
History
teams
Log in
 
Team:SJTU-BioX-Shanghai/Part1 Connect
From 2014.igem.org
HOME
PROJECT
OVERVIEW
CONNECTION
POLYMERIZATION
TAL IMPROV.
ACHIEVEMENTS
RESULTS
MODELING
PARTS
JUDGING FORM
HUMAN PRACTICE
NOTEBOOK
NOTES & PROTOCOL
SAFETY
TEAM
TEAM MEMBERS
TEAM PROFILE
ATTRIBUTIONS
Basic Test
——One Connectee Binds Connector
At first, tests should be taken to check whether TAL can bind to plasmid DNA in prokaryotic system. Here we used E.coli. As mentioned in the overview, in order for the protein to bind to the plasmid(the connector), we have designed two kinds of delicate fusion proteins(the connectee). One is anchored to the cell membrane, the other is free in the cytoplasm.
Connectee
Connector
Test Method
Two Convenient Parts
References
Connectee:
Schematic Introduction
The connectee consists of linkers and various sections, which are shown in the schematic diagrams below.
1. TAL effector: A transcription activator-like protein
TAL effector can bind to a target sequence on DNA. The 2012 Freiburg iGEM team has offered us a whole set of 96 TAL-protein direpeat bioparts, with which we are supposed to build functional TAL proteins. Since each TAL protein is able to identify a 14-nucleotide target sequence, the first and fourteenth nucleotide being Thymine, all the 96 parts can be used to identify more than 16 million different nucleotide sequences, which makes it very convenient for us to choose a sequence for the fusion protein to bind to.
Figure 1.2.1 TAL Golden gate
2. Membrane anchor system: ssDsbA-Lgt
ssDsbA, the signal sequence of DsbA, directs the fusion protein to the periplasm. Lgt is a transmembrane protein.
The membrane anchor system has been identified by iGEM12_SJTU-BioX-Shanghai.
3. Fluorescent protein
CFP is a Cyan Fluorescent Protein which has an excitation peak at 439 nm and an emission peak at 476 nm.
YFP is a Yellow Fluorescent Protein which has an excitation peak at 514 nm and an emission peak at 527 nm.
mRFP is a Red Fluorescent Protein which has an excitation peak at 584 nm and an emission peak at 607 nm.
Connectee 1:
The first kind of connectee is designed to be tested on the membrane, which consists of three major domains: membrane anchor system(ssDsbA-Lgt), mRFP and TAL effector.
The reasons why we chose a membrane anchor are as follows.
1. TAL could bind nucleoid which may bring some negative effect on bacteria growth.
2. Membrane scaffold is a natural scaffold.
3. Exogenous proteins often form inactive inclusion bodies when expressed in the prokaryotic system.
Figure1.2.2 Connectee 1
The membrane anchor system (ssDsbA-Lgt) comes from BBa_K771000 designed by iGEM12_SJTU-BioX-Shanghai; mRFP comes from BBa_E1010 designed by Antiquity; TAL effector comes from TAL-Protein DiRepeats (BBa_K747000 to BBa_K747095) designed by iGEM12_Freiburg. We used Helical Linker to connect mRFP and Lgt, in accord with iGEM12_SJTU-BioX-Shanghai. While in consideration of any possible stereospecific blockade when TAL binds to plasmid DNA, we chose Flexible Linker to connect Lgt and TAL effector.
Connectee 2:
The second kind of connectee is designed to be tested in the cytoplasm, which only contains TAL effector.
Figure 1.2.3 Connectee 2
Connector:
As iGEM12_Freiburg designed, we can choose a TAL protein to recognize a 14-nucleotide-long sequence, TXXXXXXXXXXXXT, on the plasmid DNA.   The plasmid here is called connector.
The principle for choosing a TAL recognizing sequence:
I. It does not exist in expression vector.
II. It does not exist in the sequence of connectee.
Test Method:
1. pBluescript II KS(+) is chosen as the connector in our test for several reasons:
I. High copy number;
II.Medium length—2961bp;
III. Easy to detect whether binding a TAL may affect gene expression — through lacI & blue-white spot screening.
Figure 1.2.4 pBluescript II KS(+)
2. After checking the sequence of pBluescript II KS(+), we chose TTCGATATCAAGCT as the recognition sequence for test and designed TAL1.
Two kinds of connectee with TAL1 are shown below:
Figure 1.2.5 ssDsbA-mRFP-Lgt-TAL1-His Tag
Figure 1.2.6 TAL1-His Tag
3. Considering our multiple-enzyme system might be applied in the following experiment, we chose pRSFDuet–1(NOVAGEN) as the expression vector.
Figure 1.2.7 Expression
4. Similar to the cross-linked ChIP, we used formaldehyde to cross-link connectee and connector
5. After that we did immunoprecipitation to obtain the protein-plasmid complex and digestd protein.
6. Finally, we used PCR to check whether there was any existing plasmid DNA.
Two Convenient Parts:
In order to integrate TAL-Protein DiRepeats (BBa_K747000 to BBa_K747095) properly into our system, we also designed two corresponding parts; one is ssDsbA-mRFP-Lgt-TAL USB-His Tag (BBa_K1453000), the other is Lgt-TAL USB-His Tag (BBa_K1453006).
Figure 1.2.8 ssDsbA-mRFP-Lgt-TAL USB-His Tag
Figure 1.2.9 TAL USB-His Tag
The TAL USB of these parts all consist of T1 sequence, T14 sequence and two sites for type II restriction enzyme BsmBI. When digested with BsmBI, these parts produce two sticky-ends. One can complement with the first TAL-Protein DiRepeat (BBa_K747000 to BBa_K747015) at 5’of the sequence, while the other can complement with the sixth TAL-Protein DiRepeat (BBa_K747080 to BBa_K747095) at 3’.
With the rest of TAL-Protein DiRepeats (BBa_K747016 to BBa_K747079), users can synthesize a specific TAL protein to recognize their own connector and design their own polymerization. (Golden Gate Cloning)
For more details about membrane anchor, please view this page .
For more details about TAL and Golden Gate Cloning, please view this page .
References:
Bogdanove, Adam J., and Daniel F. Voytas. “TAL effector: customizable proteins for DNA targeting.” Science 333.6051 (2011): 1843–1846.
Deng, Dong, et al. “Structural basis for sequence-specific recognition of DNA by TAL effector.” Science 335.6069 (2012): 720–723.
Pailler, J., W. Aucher, et al. (2012). “Phosphatidylglycerol: prolipoprotein diacylglyceryl transferase (Lgt) of Escherichia coli has seven transmembrane segments, and its essential residues are embedded in the membrane.” Journal of bacteriology 194(9): 2142–2151.
Schierle, C. F., M. Berkmen, et al. (2003). “The DsbA signal sequence directs efficient, cotranslational export of passenger proteins to the Escherichia coli periplasm via the signal recognition particle pathway.” Journal of bacteriology 185(19): 5706–5713.
Scholze, H. & Boch, J. TAL effector are remote controls for gene activation. ‘‘Current Opinion in Microbiology’’ 14, 47–53 (2011).
Moscou, M. J. & Bogdanove, A. J. A Simple Cipher Governs DNA Recognition by TAL Effector. ‘‘Science’’ 326, 1501–1501 (2009).
Conrado, R. J., G. C. Wu, et al. (2012). “DNA-guided assembly of biosynthetic pathways promotes improved catalytic efficiency.” Nucleic acids research 40(4): 1879–1889.
Yang, Zhong, et al. “Highly efficient production of soluble proteins from insoluble inclusion bodies by a two-step-denaturing and refolding method.” PloS one 6.7 (2011): e22981.
Bio-X Institutes
Shanghai Jiao Tong University
800, Dongchuan Road 200240, Shanghai, China
sunyaojin2011@foxmail.com
Retrieved from " http://2014.igem.org/Team:SJTU-BioX-Shanghai/Part1_Connect "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
