<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wrap"><HEADER role="banner"><DIV class="flex"><H1><A href="#top" class="site-title ir">HOME</A></H1><NAV role="navigation"><UL id="cbp-tm-menu" class="cbp-tm-menu"><LI><A href="/Team:SJTU-BioX-Shanghai/Project">PROJECT</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project" class="cbp-tm-icon-archive">OVERVIEW</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part1_Connect" class="cbp-tm-icon-cog">CONNECTION</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part2_Extension" class="cbp-tm-icon-cog">POLYMERIZATION</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part3_TAL_Improvement" class="cbp-tm-icon-cog">TAL IMPROV.</A></LI></UL><LI><A href="/Team:SJTU-BioX-Shanghai/Achievement">ACHIEVEMENTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Results" class="cbp-tm-icon-screen">RESULTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Modeling" class="cbp-tm-icon-mail">MODELING</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts" class="cbp-tm-icon-contract">PARTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Judging_Form" class="cbp-tm-icon-pencil">JUDGING FORM</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Human_Parctice" class="cbp-tm-icon-article">HUMAN PRACTICE</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook">NOTEBOOK</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/daynotes" class="cbp-tm-icon-archive">NOTES &amp; PROTOCOL</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Safety" class="cbp-tm-icon-cog">SAFETY</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Team">TEAM</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Team_Members" class="cbp-tm-icon-archive">TEAM MEMBERS</A></LI><LI><A href="https://igem.org/Team.cgi?id=1453" class="cbp-tm-icon-archive">TEAM PROFILE</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Attributions" class="cbp-tm-icon-link">ATTRIBUTIONS</A></LI></NAV></DIV></HEADER></DIV><DIV class="content"><DIV class="jiao"><DIV class="projtile_only"><CENTER><H2>ModelingandSimulation</H2></CENTER><CENTER><P>&quot;All models are wrong, but some are useful.&quot;
		When we decided to use TAL effectors building CROWN, our project, there were three main challenges concerning the efficiency of this system.
		First, allowing some DNA mutations, can the CROWN be as efficient as before?
		Second, given that CROWN can be successfully distributed on certain area of single cell, can it make sense?
		Third, how to design the sequence of Golden Gate?
		The following three parts focus on the three questions. </P></CENTER></DIV><DIV class="projtile" id="dingweidian2"><CENTER><A href="#dingweidian2" title="Part I Single Cell"><H2>Part I Single Cell</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#dianweidian9" title="Part II Docking"><H2>Part II Docking</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#dianweidian10" title="Part III K-clique Problem"><H2>Part III K-clique Problem</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#Discussion" title="Reference"><H2>Reference</H2></A></CENTER></DIV></DIV><ARTICLE class="post__article"><H2 id="partisinglecell">PartISingleCell</H2><P>Our project is about the system involving various enzymes, mostly the series enzymes, combining into certain area. This area can be more efficient when it comes to synthesizing or degrading chemicals. So the first question is, whether this system can be so useful when distributing multiple similar areas in a single cell.</P><H3 id="fourtypesofdistribution">Four Types of Distribution</H3><P><STRONG>Type 1: Membrane &amp; Random</STRONG>
			The position of enzyme is distributed randomly on the cell membrane.</P><P><STRONG>Type 2: Membrane &amp; Polymerization</STRONG>
			Certain enzymes are polymerized on the cell membrane.</P><P><STRONG>Type 3: Matrix &amp; Random</STRONG>
			The position of enzyme is distributed randomly inside the cell.</P><P><STRONG>Type 4: Matrix &amp; Polymerization</STRONG>
			The polymerization of certain enzymes is distributed randomly inside the cell.</P><H3 id="hypothesisofsimulation">Hypothesis of Simulation</H3><H4>1. Metabolism</H4><CENTER><SMALL>Figure2.2.1 The process of the metabolism: s0, s1, s2, s3 are the substrates and E0, E1,E2 are the enzymes </SMALL></CENTER><P>Enzymes: E0, E1,E2</P><P>Substrates:S0,S1,S2,S3</P><CENTER><SMALL>Figure2.2.2 the simulation of the CROWN</SMALL></CENTER><H4>2. Initial Distribution of Substrates</H4><P>All substrates are randomly distributed OUTSIDE the cell in all four simulations.</P><H4>3. Movement of Substrates</H4><P>The motion of molecules is random, including the rate and orientation.</P><H4>4. Catalytic reaction</H4><P>The time period of reaction is neglected. When the type of chemical match the type of enzyme, distance is less than threshold, then the enzyme reaction is recognized and recorded.</P><H4>5. Other Hypothesis</H4><P>Other physical and chemical parameters are under the scaling rule. The whole modeling combined with periodic boundary condition(PBC) to show the real performance of substrates and enzyme system.</P><H3 id="results:">Results:</H3><H4>All Results</H4><CENTER><A href="http://v.youku.com/v_show/id_XODAyOTY5MzM2.html">Click to watch the video</A></CENTER><CENTER><SMALL>Figure2.2.3 All the results of the four types.</SMALL></CENTER><H4>Type 1</H4><P>Click to watch the video<A href="https://www.youtube.com/watch?v=EJQXhFBMqN4&amp;list=UUjN3REkaTC_YulQONweFpSA"> Youtube</A><A href="http://v.youku.com/v_show/id_XODAyMzQ4OTYw.html">Youku</A></P><CENTER><SMALL>Figure2.2.4 The extent of reaction of type 1.</SMALL></CENTER><H4>Type 2</H4><P>Click to watch the video<A href="https://www.youtube.com/watch?v=hjr4DZ7nJwA&amp;list=UUjN3REkaTC_YulQONweFpSA"> Youtube</A><A href="http://v.youku.com/v_show/id_XODAyMzQ3NjA4.html">Youku</A></P><CENTER><SMALL>Figure2.2.5 The extent of reaction of type 2.</SMALL></CENTER><H4>Type 3</H4><P>Click to watch the video<A href="https://www.youtube.com/watch?v=W-AkV9MUITQ&amp;list=UUjN3REkaTC_YulQONweFpSA&amp;index=2"> Youtube</A><A href="http://v.youku.com/v_show/id_XODAyMzQ1MDAw.html">Youku</A></P><CENTER><SMALL>Figure2.2.6 The extent of reaction of type 3</SMALL></CENTER><H4>Type 4</H4><P>Click to watch the video<A href="https://www.youtube.com/watch?v=eS2nZS7mFsc"> Youtube</A><A href="http://v.youku.com/v_show/id_XODAyMzQwODg4.html">Youku</A></P><CENTER><SMALL>Figure2.2.7 The extent of reaction of type 4</SMALL></CENTER><H2 id="part2">PartIIDocking</H2><H3 id="WhydoweneedDocking?">Why do we need Docking?</H3><P>Biobrick designers and users want to understand the characteristics of a particular biobrick, especially the performance and accuracy. Because they need to answer a question, that is, were there to be a certain mutation, whether a huge change would happen to the protein function. We hope to introduce evaluation methods of bioinformatics, to evaluate binding of protein and DNA.</P><H3 id="Materials">Materials</H3><P>TAL (transcription activator-like) effectors, secreted by phytopathogenic bacteria, recognize host DNA sequences through a central domain of tandem repeats. Each repeat consists of 33 to 35 conserved amino acids and targets a specific base pair by using two hypervariable residues [known as repeat variable diresidues (RVDs)] at positions 12 and 13.</P><P><STRONG>PDB:3V6T</STRONG></P><H3 id="Mutations">Mutations</H3><P>We designed fifteen sequences derived from raw sequence. These mutated sequences contain different mutations, ranging from one to five. Through a series of calculations, we obtained scores to represent the binding of TAL effectors and DNA.</P><P><LI><STRONG>The highlighted Letters represent the mutation site.</STRONG></LI></P><P><LI><STRONG>The white DNA sequences on the graph is the originated position and orange one represents the possible binding DNA.</STRONG></LI></P><P><LI><STRONG>The higher Docking scores, the better Docking</STRONG></LI></P><UL style="padding-left:5%;"><LI><STRONG>mutation-1</STRONG></LI><LI><STRONG>Score：1164.128</STRONG></LI><LI><STRONG>mutation-2</STRONG></LI><LI><STRONG>Score:1170.910</STRONG></LI><LI><STRONG>mutation-3</STRONG></LI><LI><STRONG>Score:1153.537</STRONG></LI><LI><STRONG>mutation-4</STRONG></LI><LI><STRONG>Score:1377.231</STRONG></LI><LI><STRONG>mutation-5</STRONG></LI><LI><STRONG>Score:1169.283</STRONG></LI><LI><STRONG>mutation-6</STRONG></LI><LI><STRONG>Score:1179.122</STRONG></LI><LI><STRONG>mutation-7</STRONG></LI><LI><STRONG>Score:1482.902</STRONG></LI><LI><STRONG>mutation-8</STRONG></LI><LI><STRONG>Score:1161.824</STRONG></LI><LI><STRONG>mutation-9</STRONG></LI><LI><STRONG>Score:1482.897</STRONG></LI><LI><STRONG>mutation-10</STRONG></LI><LI><STRONG>Score:1174.229</STRONG></LI><LI><STRONG>mutation-11</STRONG></LI><LI><STRONG>Score:1237.449</STRONG></LI><LI><STRONG>mutation-12</STRONG></LI><LI><STRONG>Score:1482.896</STRONG></LI><LI><STRONG>mutation-13</STRONG></LI><LI><STRONG>Score:1483.352</STRONG></LI><LI><STRONG>mutation-14</STRONG></LI><LI><STRONG>Score:1482.048</STRONG></LI><LI><STRONG>mutation-15</STRONG></LI><LI><STRONG>Score:1164.128</STRONG></LI></UL><H3 id="Analysis">Analysis</H3><CENTER><P><STRONG>Table</STRONG></P></CENTER><CENTER><P><STRONG>Scatter Diagram</STRONG></P></CENTER><CENTER><SMALL>Figure 2.2.8 The correlation between the number of mutation sites and the docking scores.The higher docking scores indicates the better combination of TAL and target sequence.</SMALL></CENTER><P>From the docking scores, we can see that in the event of single nucleotide mutation, binding of TAL effectors and DNA differs greatly from normal. However, when there are more than two mutation sites, the difference becomes less drastic.</P><P id="dianweidian10">From the PDB document, we can find that mutation at certain sites may lead to huge conformational distortions of TAL-DNA complex. With as many as five mutations, the binding site changes greatly.</P><P>In conclusion, we strongly recommend that TAL designers and users ensure the accuracy of DNA binding sequence. If not, the specificity of binding site will not be guaranteed.</P><H2>Part Ⅲ K-clique Problem</H2><H3 id="Problem Identification">Problem Identification</H3><P>Before designing the TAL sequence, what we need to do first is to design a set of sequences, which are consist of seven fragments containing four nucleotides. In order to accomplish this, we offer a new sequence alignment algorithm for these fragmental sequences comparison. By using Loose Algorithm and Strict Algorithm, two fragmental sequences can be scored, and when the score is less than or equal to a specific value (eg 2), we can accept that they are a valid combination.</P><P>As we can see, the sequence database contains a total of 4 * 4 * 4 * 4 = 256 kinds of fragmental sequences. Now, what we should do is to find out some seven sequences whose each two fragments can be a valid combination, based on a large (256*256) score table.</P><H3 id="Assumption and Model Formulation">Assumption and Model Formulation</H3><P>We try to solve this problem by making full use of graph theory. In our model, each fragmental sequence can be treated as a node, and two “node” is connected, if they are a valid combination (the score less than or equal to 2). After doing that, we just need to find such seven nodes which are consist of a complete graph from a graph with 256 nodes. (Note: not every two nodes in original graph is connected)</P><H3 id="Brief introduction to graph theory">Brief introduction to graph theory</H3><P>For an undirected graph G = (V, E)，if  U⊆V, and for any u, v ⊆ U， (u, v) ⊆ E，  U is called complete subgraph of G.</P><P>A clique in an undirected graph G = (V, E) is a subset of the vertex set C ⊆ V, such that for every two vertices in C, there exists an edge connecting the two. A maximal clique is a clique that cannot be extended by including one more adjacent vertex, that is, a clique which does not exist exclusively within the vertex set of a larger clique.</P><P>U is the maximal complete subgraph of G, if and only if U is a clique of U and U is not contained in a larger subgraph.</P><P>The k-clique problem is to find the complete graph with k nodes in a specific graph. What’s more, k-clique algorithm is defined in the paper &quot;Uncovering the overlapping community structure of complex networks in nature and society&quot; - G. Palla, I. Derényi, I. Farkas, and T. Vicsek - Nature 435, 814–818 (2005).</P><P>Although a deterministic algorithm for this problem with an O(n*2^n) algorithm time complexity, fortunately, in our experiment, the problem we should solve is just with 256 nodes, so a non-deterministic algorithm can be applied.</P><H3 id="Solution">Solution</H3><P>Early in 2005, a scientist have tried to use some kinds of Backtracking Algorithm to solve this problem, which have published in Science. Based on this excellent work, we have gone further to offer two more efficient algorithms.</P><H3 id="Algorithm 1">Algorithm 1</H3><P>In the solution space tree containing all available solutions, we search the solution space tree from the root according to the depth-first strategy. When reaching a node, we always firstly determine whether the node include any solutions or not. If not, we don't need to search the subtree whose root is such node, and then backtrack the ancestor nodes step-by-step; otherwise, we should continue our depth-first search.</P><H3 id="Algorithm 2">Algorithm 2</H3><P>1.Sort the degree of each node.</P><P>2.In the current data set, from the first degree to the last, we remain the nodes which is relative to such degree, and delete the unconnected one.</P><P>3.In the set, containing the nodes connected to the previous one, for each node, determine whether it is connected to others, and then sort the nodes based on their degree.</P><P>4.Divide the problem into some much smaller size problems, repeat above method.</P><H3 id="Result">Result</H3><P>Based on our efficient algorithm, we have found a possible solution:</P><H3 id="Discussion">Discussion</H3><P>Clique problem play an important role in graph theory as well as is quite complex. However, we have provided some valid combinations for the TAL users here.</P><H2 id="Reference">Reference</H2><OL style="font-style: italic;"><LI>Pierce, Brian G., Yuichiro Hourai, and Zhiping Weng. &quot;Accelerating protein docking in ZDOCK using an advanced 3D convolution library.&quot; PloS one 6.9 (2011): e24657.</LI><LI>Mintseris, Julian, et al. &quot;Integrating statistical pair potentials into protein complex prediction.&quot; Proteins: Structure, Function, and Bioinformatics 69.3 (2007): 511-520.</LI><LI>  G. Palla, I. Derényi, I. Farkas, and T. Vicsek. &quot;Uncovering the overlapping community structure of complex networks in nature and society&quot; Nature 435, 814–818 (2005)</LI></OL></ARTICLE></DIV><DIV class="footer"><DIV class="footer-content"><P>Bio-X Institutes</P><P>Shanghai Jiao Tong University</P><P>800, Dongchuan Road 200240, Shanghai, China</P><P>sunyaojin2011@foxmail.com</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SJTU-BioX-Shanghai/Modeling">http://2014.igem.org/Team:SJTU-BioX-Shanghai/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Modeling&amp;oldid=390040" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>