<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Part2_Extension"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/Part2_Extension">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/Part2_Extension&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Part2_Extension&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Part2_Extension&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/Part2_Extension" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/Part2 Extension</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wrap"><HEADER role="banner"><DIV class="flex"><H1><A href="#top" class="site-title ir">HOME</A></H1><NAV role="navigation"><UL id="cbp-tm-menu" class="cbp-tm-menu"><LI><A href="/Team:SJTU-BioX-Shanghai/Project">PROJECT</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project" class="cbp-tm-icon-archive">OVERVIEW</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part1_Connect" class="cbp-tm-icon-cog">CONNECTION</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part2_Extension" class="cbp-tm-icon-cog">POLYMERIZATION</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part3_TAL_Improvement" class="cbp-tm-icon-cog">TAL IMPROV.</A></LI></UL><LI><A href="/Team:SJTU-BioX-Shanghai/Achievement">ACHIEVEMENTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Results" class="cbp-tm-icon-screen">RESULTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Modeling" class="cbp-tm-icon-mail">MODELING</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts" class="cbp-tm-icon-contract">PARTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Judging_Form" class="cbp-tm-icon-pencil">JUDGING FORM</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Human_Parctice" class="cbp-tm-icon-article">HUMAN PRACTICE</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook">NOTEBOOK</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/daynotes" class="cbp-tm-icon-archive">NOTES &amp; PROTOCOL</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Safety" class="cbp-tm-icon-cog">SAFETY</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Team">TEAM</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Team_Members" class="cbp-tm-icon-archive">TEAM MEMBERS</A></LI><LI><A href="https://igem.org/Team.cgi?id=1453" class="cbp-tm-icon-archive">TEAM PROFILE</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Attributions" class="cbp-tm-icon-link">ATTRIBUTIONS</A></LI></NAV></DIV></HEADER></DIV><DIV class="content"><DIV class="jiao"><DIV class="projtile_only"><H2 id="subtitleyjn1">More than One Jewel on the Crown</H2><H2 id="subtitleyjn2">——Booster and Gearbox</H2><CENTER><P>The <STRONG><EM>Crown</EM></STRONG> will help us achieve the goal of selective enzyme polymerization. For polymerization, we introduce the <STRONG><EM>Booster</EM></STRONG>, which is an accelerator with enzymes collecting on it. For selectivity, we introduce the <STRONG><EM>Gearbox</EM></STRONG>, which offers users the right to make use of different sets of enzymes.</P></CENTER></DIV><DIV class="projtile" id="dingweidian2"><CENTER><A href="#dingweidian2" title="Booster"><H2>Booster</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#dingweidian16" title="Gearbox"><H2>Gearbox</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#dingweidian3" title="How to be a Craftsman"><H2>How to be a Craftsman</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#Testmethods" title="Application"><H2>Application</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#Application" title="What’s more ?"><H2>What’s more ?</H2></A></CENTER></DIV><DIV class="projtile"><CENTER><A href="#What'smore" title="Reference"><H2>Reference</H2></A></CENTER></DIV></DIV><ARTICLE class="post__article"><H2 id="Booster">Booster</H2><P><STRONG><EM>Booster</EM></STRONG> is the model of polymerization and maximization. Polymerized enzymes, different from normal scattered ones, have much more opportunities to contact with the substrates. Therefore, our project aims to increase as well as to control the efficiency of complex reactions. We have built a framework that makes multi-enzyme complex easy to form, which we call a <STRONG><EM>Crown</EM></STRONG>.</P><P id="dingweidian16">And based on the success of the individual fusion protein(The first jewel) expression, we tried to add more jewels on the <STRONG><EM>Crown</EM></STRONG>.</P><P>In order to build a polymerization system, we made more fusion proteins bind to the same <STRONG><EM>Connector</EM></STRONG>. Then with more enzymes added, it has the practical effect of accelerating the reactions and enhancing the production efficiency. So far, the <STRONG><EM>Crown</EM></STRONG> is built with more than one jewel shining on it.</P><H2 id="Gearbox">Gearbox</H2><P><STRONG>Gearbox</STRONG> is the model of selectivity. Our fusion protein can not only be used to polymerize enzymes but also be used to control selective combinations. We can control the direction of pathway by simply transforming different <STRONG><EM>Connector</EM></STRONG> plasmids. </P><CENTER><SMALL><STRONG>Figure 1.3.1 Principle of selectivity</STRONG></SMALL></CENTER><P>All the three enzymes involved are expressed in the bacteria. Each fusion protein contains ssDsbA, Lgt, FL3, enzyme, HL and TAL; different TAL parts can recognize different sites. <STRONG><EM>Connector</EM></STRONG> is originally designed with three recognition sequences(RS), which are combined to form two different connectors, one with RSⅠ and RSⅡ while the other one with RSⅠ and RSⅢ (shown in the figure below). <STRONG><EM>Connector</EM></STRONG> with RSⅠand RSⅡ can bind to Connectee-enzymeⅠ and Connectee-enzymeⅡ then get production A in the end, while <STRONG><EM>Connector</EM></STRONG> with RSⅠ and RSⅢ can bind to Connectee-enzymeⅠ and Connectee-enzymeⅢ then get production B.</P><CENTER><SMALL><STRONG>Figure 1.3.2 pBluescript II KS(+) ScaI/EcoRV deletion</STRONG></SMALL></CENTER><H2 id="HowtobeaCraftsman">How to be a Craftsman</H2><P>A compact machine requires bold ideas and careful practicing. Here comes how we built our <STRONG><EM>Crown</EM></STRONG>.</P><H3 id="ConstructionMethod">Construction Method</H3><P>Above all, we used 3 sets of <STRONG><EM>Connectee1</EM></STRONG> to build polymerization system. There are four main parts in a <STRONG><EM>Connectee1</EM></STRONG> for testing.</P><CENTER><SMALL><STRONG>Figure 1.3.3 ssDsbA-FP-Lgt-TAL-His Tag</STRONG></SMALL></CENTER><P><STRONG><EM>ssDsbA</EM></STRONG>: SsDsbA is the signal recognition particle (SRP)-dependent signaling sequence of DsbA. SsDsbA-tagged proteins are exported to the periplasm through the SRP pathway. With ssDsbA fused to the N-terminus, fusion proteins with Lgt are expected to be anchored onto inner membrane of <I>E.coli</I>.</P><P><STRONG><EM>FP</EM></STRONG>: To visualize the localization of fusion protein with fluorescence test , we added FP in the <STRONG><EM>Connectee1</EM></STRONG> and placed it just after the ssDsbA. We chose mRFP, CFP and YFP in our system.</P><P><STRONG><EM>Lgt</EM></STRONG>: Phosphatidylglycerol prolipoprotein diacylglyceryl transferase (Lgt) is an inner membrane protein that acts as a membrane anchor of <I>E.coli</I> with seven transmembrane segments and has been successfully overexpressed in E. coli without causing harm to cells.</P><P><STRONG><EM>TAL effectors</EM></STRONG>：As mentioned earlier, we chose three kinds of combinations to build three different TAL proteins, based on the parts that 2012 Freiburg iGEM team offered. These three TAL proteins can identify three different 14bp nucleotide sequences on a <STRONG><EM>Connector</EM></STRONG>. Note that we added a His Tag at the end of TAL protein to facilitate separation and purification.</P><P>For the three kinds of corresponding <STRONG><EM>Connectee2</EM></STRONG>, we did not introduce ssDsbA-Lgt section to keep them in a free intracellular state.</P><P>In the final production of our construction, we added an easy-to-hand interface sequence between the Lgt and TAL protein in <STRONG><EM>Connectee1</EM></STRONG> or just before the TAL protein in <STRONG><EM>Connectee2</EM></STRONG>, as sites to adding enzymes.</P><CENTER><SMALL><STRONG>Figure 1.3.4 ssDsbA-Lgt-Enzyme USB(AarI/BsmAI)-TAL USB-His Tag</STRONG></SMALL></CENTER><H3 id="Co-Transformation&amp;InducedExpression">Co-Transformation &amp; Induced Expression</H3><P>Expression vectors used in the project for membrane protein expression are modified versions of pRSFDuet-1, pETDuet-1,pACYCDuet-1(NOVAGEN), which are originally regulated by T7 promoter. These plasmids can coexist in one cell.</P><P>The <STRONG><EM>Connector</EM></STRONG> is pBluescript II KS(+) .</P><CENTER><SMALL><STRONG>Figure 1.3.5 pBluescript II KS(+)</STRONG></SMALL></CENTER><P>We used the four reformed plasmids to co-transform and induced the <STRONG><EM>Connectors</EM></STRONG> to express. Now the structure of the crown appears in the <I>E.coli</I> cell membrane or cytoplasm. Here we used the fusion proteins with two kinds of enzymes and TAL1 as examples. </P><CENTER><SMALL><STRONG>Figure 1.3.6 ssDsbA-Lgt-pykF/poxB-TAL1-His Tag</STRONG></SMALL></CENTER><H3 id="Testmethods">Test Methods</H3><P>We used formaldehyde to stabilize the connected fusion proteins and <STRONG><EM>Connector</EM></STRONG>, then separated the protein-Connector complexes and digested the protein. Finally we used PCR to detect the three sequences of the <STRONG><EM>Connector</EM></STRONG> which TAL proteins were designed to bind to.</P><P>Then, we were able to test the yield of different products to determine the efficiency of selective combination.</P><H2 id="Application">Application</H2><P>Based on our design, our project has many application prospects. In prokaryotic systems, enzyme polymerization can be used to regulate various metabolic pathways, such as the example we mentioned above. pykF and poxB are two enzymes involved in pyruvate metabolism. By binding them to the connector, we can improve the reaction efficiency to a great extent. Besides, our project can also be applied in eukaryotic systems, including gene therapy and regulation of stem cell differentiation.</P><H2 id="What'smore">What’s more ?</H2><P>We are trying to put more enzymes into the polymerization system, which makes the <STRONG><EM>Crown</EM></STRONG> more practical and brilliant. And with more enzymes on the <STRONG><EM>Crown</EM></STRONG>, there will be more possible choices for selective combination.</P><H2 id="References">References</H2><OL style="font-style: italic;"><LI>Pailler, Jérémy, et al. &quot;Phosphatidylglycerol:: prolipoprotein diacylglyceryl transferase (Lgt) of Escherichia coli has seven transmembrane segments, and its essential residues are embedded in the membrane.&quot; Journal of bacteriology 194.9 (2012): 2142-2151.</LI><LI>Schierle, Clark F., et al. &quot;The DsbA signal sequence directs efficient, cotranslational export of passenger proteins to the Escherichia coli periplasm via the signal recognition particle pathway.&quot; Journal of bacteriology 185.19 (2003): 5706-5713.</LI><LI>Katsuyama, Tomonori, et al. &quot;An efficient strategy for TALEN-mediated genome engineering in Drosophila.&quot; Nucleic acids research 41.17 (2013): e163-e163.</LI><LI>Bogdanove, Adam J., Sebastian Schornack, and Thomas Lahaye. &quot;TAL effectors: finding plant genes for disease and defense.&quot; Current opinion in plant biology 13.4 (2010): 394-401.</LI></OL></ARTICLE></DIV><DIV class="footer"><DIV class="footer-content"><P>Bio-X Institutes</P><P>Shanghai Jiao Tong University</P><P>800, Dongchuan Road 200240, Shanghai, China</P><P>sunyaojin2011@foxmail.com</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SJTU-BioX-Shanghai/Part2_Extension">http://2014.igem.org/Team:SJTU-BioX-Shanghai/Part2_Extension</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/Part2_Extension" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/Part2_Extension" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Part2_Extension&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Part2_Extension&amp;oldid=370946" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>