<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Protocol&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wrap"><HEADER role="banner"><DIV class="flex"><H1><A href="#top" class="site-title ir">HOME</A></H1><NAV role="navigation"><UL id="cbp-tm-menu" class="cbp-tm-menu"><LI><A href="/Team:SJTU-BioX-Shanghai/Project">PROJECT</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Project" class="cbp-tm-icon-archive">OVERVIEW</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part1_Connect" class="cbp-tm-icon-cog">CONNECTION</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part2_Extension" class="cbp-tm-icon-cog">POLYMERIZATION</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Part3_TAL_Improvement" class="cbp-tm-icon-cog">TAL IMPROV.</A></LI></UL><LI><A href="/Team:SJTU-BioX-Shanghai/Achievement">ACHIEVEMENTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Results" class="cbp-tm-icon-screen">RESULTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Modeling" class="cbp-tm-icon-mail">MODELING</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Parts" class="cbp-tm-icon-contract">PARTS</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Judging_Form" class="cbp-tm-icon-pencil">JUDGING FORM</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Human_Parctice" class="cbp-tm-icon-article">HUMAN PRACTICE</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Notebook">NOTEBOOK</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/daynotes" class="cbp-tm-icon-archive">NOTES &amp; PROTOCOL</A></LI><LI><A href="/Team:SJTU-BioX-Shanghai/Safety" class="cbp-tm-icon-cog">SAFETY</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Team">TEAM</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Team_Members" class="cbp-tm-icon-archive">TEAM MEMBERS</A></LI><LI><A href="https://igem.org/Team.cgi?id=1453" class="cbp-tm-icon-archive">TEAM PROFILE</A></LI><LI><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Attributions" class="cbp-tm-icon-link">ATTRIBUTIONS</A></LI></NAV></DIV></HEADER></DIV><DIV id="sidenav_container" style="font-family: Rancho,Impact, Charcoal, sans-serif; font-size: 1.1em; letter-spacing: 1.5px;"><DIV class="sidenav"><DIV class="heading">
    				Week Notes
    			</DIV><DIV class="content"><P><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/daynotes#July">July</A></P><P><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/daynotes#August">August</A></P><P><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/daynotes#September">September</A></P></DIV></DIV><DIV class="sidenav"><DIV class="heading">
    				Protocol
    			</DIV><DIV class="content"><P><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Protocol#Molecule">Molecule</A></P><P><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Protocol#Cell">Cell</A></P><P><A href="https://2014.igem.org/Team:SJTU-BioX-Shanghai/Protocol#Protein">Protein</A></P></DIV></DIV></DIV><DIV style="padding-left: 15%; background-color: #fff;"><DIV class="content"><ARTICLE class="post__article"><H2 id="Molecule">Molecule</H2><H3>Golden gate Protocol</H3><P>After purification of the PCR product, you can digest your part with EcoRI and PstI using the following Protocol:
                            </P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TH>Component</TH><TH>Amount (μl)</TH></TR><TR><TD>Purified PCR product</TD><TD>30</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>BsaI</TD><TD>0.5</TD></TR><TR><TD>NEB buffer 4 (10x)</TD><TD>4</TD></TR><TR><TD>ddH2O</TD><TD>3.5</TD></TR><TR><TD>Total Volume</TD><TD>40</TD></TR></TBODY></TABLE><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>Thermocycler programm</TD></TR><TR><TD>1.     37°C, 12 hours</TD></TR><TR><TD>2.     80°C, 20 minutes</TD></TR></TBODY></TABLE><P><STRONG>Important note:</STRONG></P><P>We very much advise you to digest the vector for 12 hours and purify the product on a gel. This significantly reduces the risk of religation of your vector. We usually had no colonies on our negative control plate after ligation with T4 ligase and transformation into DH10B cells.</P><P>Since most standard iGEM plasmids contain binding sites of the most common two type II restriction enzymes (namely BsaI/Eco31I and BsmBI/Esp3I), we propose using BbsI/BpiI. We have tested this enzyme in various reaction conditions with many different reaction additives (such as ATP or DTT). Although ligase buffer worked best with other type II restriction enzymes (in those cases, ligase activity probably was the bottleneck), we had best results with G Buffer (Fermantas) plus several additives using BbsI.</P><H3>Assembling</H3><P>For assembling parts that are in Golden Gate standard, we recommend the following protocol:</P><H3>Ligase Reaction</H3><P>With the six TAL BioBricks and the fusion enzyme in your reaction tube you now only need the type II restriction enzyme BsmB1 and a T7 ligase to put all the parts together.</P><P>Transform 5 μl of the GATE assembly product into 50 μl of transformation competent bacteria.</P><P><STRONG>Important note:</STRONG></P><P>Your cells need to be sensitive to the ccdB kill cassette in our TAL expression vectors! Otherwise also bacteria that have taken up plasmids without the six direpeats will form false positive colonies. We used the DH10B <I>E.coli</I> strain.</P><P>In case you want to express your TALE in bacteria, you need to induce the promoter of our prokaryotic expression plasmid with IPTG. </P><P>For use in a eukaryotic system, such as HEK 239 cells, perform a midiprep and directly transfect the eukaryotic TAL expression plasmid (or its derivatives pTAL-TF, pTALEN etc.) according to your transfection protocol. </P><P>We always used this 8:40 hour thermocycler program to obtain best results. However, you can also reduce the number of cycles.
</P><H3>Digestion</H3><P><STRONG>Material:</STRONG></P><P>BioLabs ： BstXⅠ R0113S/R0580L</P><P><STRONG>Procedures:</STRONG></P><P>Time-Saver Protocol:</P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TH>Components</TH><TH>Volume</TH></TR><TR><TD>Restriction Enzyme</TD><TD>1 μl</TD></TR><TR><TD>DNA</TD><TD>1 μl</TD></TR><TR><TD>10× NEBuffer </TD><TD>5μl</TD></TR><TR><TD>Water</TD><TD>43μl</TD></TR></TBODY></TABLE><P>Incubation : 55℃ 5-15min</P><P><STRONG>Material:</STRONG></P><P>Thermo ：T4 DNA ligase</P><P>Procedures:</P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>Linear vector DNA</TD><TD>20-100 ng</TD></TR><TR><TD>Insert DNA</TD><TD>1:1 to 5:1 molar ratio over vector</TD></TR><TR><TD>10× T4 DNA Ligase Buffer</TD><TD>2μl</TD></TR><TR><TD>50% PEG 4000 Solution</TD><TD>2μl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>5 U</TD></TR><TR><TD>Water</TD><TD>to 20μl </TD></TR></TBODY></TABLE><P>Incubate for 1 hour at 22℃</P><P><STRONG>Material:</STRONG></P><P>Thermo ：Esp3I(BsmBI)</P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TH>Components</TH><TH>Volume</TH></TR><TR><TD>Restriction Enzyme</TD><TD>1 μl</TD></TR><TR><TD>DNA</TD><TD>1 μl</TD></TR><TR><TD>10× NEBuffer </TD><TD>5μl</TD></TR><TR><TD>Water</TD><TD>43μl</TD></TR></TBODY></TABLE><P>Incubation : 55℃ 5-15min</P><P><STRONG>Material:</STRONG></P><P>Thermo ：T4 DNA ligase</P><P>Procedures:</P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>Esp3I</TD><TD>0.5-2μl</TD></TR><TR><TD>10×Buffer Tango</TD><TD>2μl</TD></TR><TR><TD>DNA(0.5-1 μg/μL)</TD><TD>1μl</TD></TR><TR><TD>DTT(20mM)</TD><TD>1μl</TD></TR><TR><TD>Water</TD><TD>16μl</TD></TR></TBODY></TABLE><P>Incubate at 37℃ for 1-16 hours</P><H3>Gel extraction</H3><P><STRONG>Material:</STRONG></P><P>OMEGA  :  E.Z.N.A Gel Extraction Kit(Gel Extraction Spin Protocol)</P><P><STRONG>Procedures:</STRONG></P><P>1.	Preform agarose gel/ethidium bromide electrophoresis to fractionate DNA fragments</P><P>2.	When adequate seprartion of bands has occurred, CAREFULLY excise the DNA fragment of interest using a wide, clean, sharp scalpel. Minimize the size of the gel by removing extra agarose.</P><P>3.	Determine the appropriate volume of the gel slice by weighing it in a clean 1.5ml microcentrifuge tube. Add an equal volume of Binding Buffer(XP2). Incubate the mixture at 55℃-66℃ for 7min or until the gel has completely melted. Mix by shaking or vortexing the tube in increments of 2-3 minutes.</P><P>4.	Place a DNA column in a provided 2 ml collection tube.</P><P>5.	Apple 700μl of the DNA/agarose solution to the DNA column, and centrifuge at 10,000×g for 1min at room temperature.</P><P>6.	Discord liquid and place the DNA column back into the same collection tube.</P><P>7.	Add 300ml of Binding Buffer(XP2) into the DNA column. Centrifuge at 10,000×g for 1min at room temperature to wash the column. Discard the flow-through and re-use the collection tube.</P><P>8.	Wash the column by adding 700μl of SPW Wash Buffer diluted with absoluted ethanol. Centrifuge at 10,000×g for 1min at room temperature.</P><P>9.	Discard liquid and centrifuge the empty DNA column for 2 min at maximal speed(&gt;13,000×g)to dry the column matrix.</P><P>10.	Place a DNA column into a clean 1.5ml microcentrifuge tube. Add 30μl of distilled water directly onto the column matrix and incubate at room temperature for 1 minute. Centrifuge for 1min at maximal speed to elute DNA.</P><P><STRONG>Important Notes:</STRONG></P><P>1.	If the balance liquid turns turbid, incubate it at 37℃ for several minutes.</P><P>2.	When cutting the gel, keep the UV irradiation time period as short as possible, otherwise DNA 3.would be damaged.</P><P>3.	For those DNA fragment shorter than 100bp or larger than 10kb, the sol solution volume should be increased as well as the time period for adsorption and elution.</P><P>4.	The efficiency of eluting DNA from the DNA column is dependent of pH. So make sure that the pH of water is around 8.0.</P><H3>PCR Clean-Up</H3><P><STRONG>Material:</STRONG></P><P>FavorPrep PCR Purification Kit :  FAVORGEN</P><P><STRONG>Procedures:</STRONG></P><P>1.	Transfer up to 100μl of PCR product and add 5 volumes of FADF Buffer to a microcentrifuge tube then mix by vortexing.</P><P>2.	Place a FADF column into a Collection Tube.</P><P>3.	Transfer the sample mixture to the FADF Column. Centrifuge for 30 seconds then discard the flow-through.</P><P>4.	Add 750μl of Wash Buffer(ethanol added)to the FADF Column. Centrifuge for 30 seconds then discard the flow-through.</P><P>5.	Centrifuge again for an additional 3 minutes to dry the column.</P><P>6.	Place the FADF Column to a new microcentrifuge tube.</P><P>7.	Add 40μl of Elution Buffer or ddH2O to the membrane center of the FADF Column. Stand the FADF Column for 2 min.</P><P>8.	Centrifuge for 2 min to elute the DNA.</P><H3>PCR:</H3><P><STRONG>Material:</STRONG></P><P>TOYOBO :  KOD-Plus-Neo</P><P><STRONG>Procedures:</STRONG></P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>Components</TD><TD>Volume</TD></TR><TR><TD>10×Buffer Tango</TD><TD>2μl</TD></TR><TR><TD>Autoclaved, distilled water</TD><TD>(33-x)μl</TD></TR><TR><TD>Template</TD><TD>xμl</TD></TR><TR><TD>Primers</TD><TD>0.75~1.5μl each</TD></TR><TR><TD>10× PCR Buffer for KOD-Plus-Neo</TD><TD>5μl</TD></TR><TR><TD>2mM dNTPs</TD><TD>5μl</TD></TR><TR><TD>25mM MgSO4</TD><TD>3μl</TD></TR><TR><TD>KOD-Plus-Neo</TD><TD>1μl</TD></TR></TBODY></TABLE><P>Reaction temperature:
                          Two-step method:
                          Predenature
                          94℃，2min
                          Denature and extension
                          {
                          98℃ 10sec 
                          Tm℃ 30sec/kb
                          }
                          × 25-45 cycles
                          4℃
                     </P><P><STRONG>Important Notes:</STRONG></P><P>1.KOD enzymes should be operated in ice.</P><P>2.Two-step method is preferred if primer Tm is relatively high.</P><P>3.1.2% agarose gel is used to separate 1kb-5kb DNA; 0.8% agarose gel is used to separate 3k-10k DNA; 2.5-3% agarose gel is used to separate DNA larger than 100bp.</P><P>4.PCR reaction system varies from 20uL to 100uL. Under most cases, 20uL system is preferred as long as PCR is for identification only. However, when subsequent experiments such as tailing, digestion which require certain amount of DNA are necessary, 50uL system is usually preferred.</P><P>5.KOD enzymes are preferred if the purpose of PCR is for gene amplification. At the same time, PCR for identification or tailing usually requires taq enzymes.</P><P><STRONG>Overoverlap PCR</STRONG></P><P>1.Prepare the system according to the KOD protocol without primers and template.</P><P>2.Firstly add equimolar of ssDsbA and mRFP as template at 58℃ for 50 s.</P><P>3.Then add equimolar of Lgt as template at 58℃ for 1min.</P><P>4.Do step 2 and 3 for 5 cycles.</P><P>5.At last add ssDsbA Forward primer and Lgt-Fl3 Reverse primer at 58℃ for 90 s and cycle 30 times.</P><H3>Plasmid Extraction</H3><P><STRONG>Material:</STRONG></P><P>BIOMIGA : EZgene Plasmid Miniprep Kit(Plasmid Miniprep Spin Protocol)</P><P><STRONG>Procedures:</STRONG></P><P>1.Inoculate 1-4ml LB containing appropriate antibiotic with a fresh colony from a freshly streaked selective plate. Incubate at 37℃ for 14-16 hours with vigorous shaking.</P><P>2.Harvest the bacterial culture by centrifugation for 1min at 10,000 rpm. Pour off the supernatant and blot the inverted tube on a paper towel to remove residue medium. Remove the residue medium completely.</P><P>3.Add 250μL Buffer A1 and completely resuspend bacterial pellet by vortexing or pipetting.</P><P>4.Add 250μL Buffer B1, mix gently by inverting the tube 10 times(do not vortex), and incubate at room temperature for 5 minutes.</P><P>5.Add 350μL Buffer N1, mix completely by inverting/shaking the vial for 5 times and sharp hand shaking for 2 times. </P><P>6.Centrifuge the lysate at 13,000 rpm for 10 minutes at room temperature.</P><P>7.Carefully transfer the clear lysate into a DNA column with a collection tube, avoid the precipitations, spin at 13,000 rpm for 1 minute, discard the flow-through and put the column back to the collection tube.</P><P>8.Add 500μL Buffer KB into the spin column, centrifuge at 13,000 rpm for 1 minute. Romove the spin column from the tube and discard the flow-through. Put the column back to the collection tube.</P><P>9.Add 700μL DNA Wash Buffer into the spin column, centrifuge at 13,000 rpm for 1 min at room temperature. Remove the spin column from the tube and discard the flow-through.</P><P>10.Reinsert the spin column, with the lid open, into the collection tube and centrifuge for 2 minutes at 13,000 rpm.</P><P>11.Carefully transfer the spin column into a sterile 1.5mL tube and add 50μL Sterile ddH2O into the center of the column and let it stand for 2 minutes. Elute the DNA by centrifugation at 13,000 rpm for 1 minute. Reload the eluate into the column and elute again.</P><H2 id="Cell">Cell</H2><H3>Bacteria Culture</H3><P>Strain:
                            pBluescript II KS(+)，pACYCDuet-1，pCDFDuet-1，pRSFDuet-1，pET28a(+)，pET30a(+)</P><P>LB Medium</P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>NaCl</TD><TD>10g</TD></TR><TR><TD>Tryptone</TD><TD>10g</TD></TR><TR><TD>Yeast Extract</TD><TD>5 g</TD></TR><TR><TD>Water</TD><TD>up to 1000mL</TD></TR></TBODY></TABLE><P>TB Medium(1L=900ml+100ml)</P><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>Tryptone</TD><TD>12g</TD></TR><TR><TD>Yeast Extract</TD><TD>24g</TD></TR><TR><TD>Glycerol</TD><TD>4ml</TD></TR><TR><TD>Water</TD><TD>up to 900mL</TD></TR></TBODY></TABLE><TABLE border="1" style="visibility: visible;margin: 10px;"><TBODY><TR><TD>K2HPO4•3H2O</TD><TD>16.4g</TD></TR><TR><TD>KH2PO4</TD><TD>2.31g</TD></TR><TR><TD>Water</TD><TD>up to 100mL</TD></TR></TBODY></TABLE><H3>Transformation</H3><P><STRONG>Material:</STRONG></P><P>TransGen Biotech : Trans5α Chemically Competent Cell</P><P><STRONG>Procedures:</STRONG></P><P>1.Remove competent cells from freezer and allow to thaw.</P><P>2.Add 3-5 μl of DNA to the cells.</P><P>3.Incubate on ice for 30 min.</P><P>4.Heat shock the cells at 42℃ for 90 seconds.</P><P>5. Add 200μl of LB medium(no antibiotics), and incubate in a shaker at 37°C for 1 hour.</P><P>6.Centrifuge for 3 min at 3,000 g, remove and keep 100 μl of the supernatant, and pour away the rest.</P><P>7.Re-suspend the pellet in the 100μl of supernatant that was removed earlier.</P><P>8.Spread all onto one plate with corresponding antibiotics, and then pour the rest onto another. Incubate the plates overnight at 37℃.</P><H3>Co-transformation</H3><P><STRONG>Material:</STRONG></P><P>TransGen Biotech :  Trans5α Chemically Competent Cell</P><P><STRONG>Procedures:</STRONG></P><P>1.Prepare the plate with different kinds antibiotics corresponding to the resistance of different plasmid at a ratio of 1:2000 for each.</P><P>2.Measure the concentration of plasmids. Add different kinds of plasmids in equimolar ratio.</P><P>3.Co-transform all plasmids following the protocol of normal transformation.</P><H3>Blue White Colony Selection</H3><P><STRONG>Procedures:</STRONG></P><P>1.Prepare for the LB Medium</P><P>2.Add agar 15g for every 1L Medium</P><P>3.Autoclave on liquid and cool the media to 50℃</P><P>4.Add 1ml of X-Gal</P><P>5.Add 1ml of 100mM IPTG Solution</P><P>6.Add antibiotic for selection</P><P>7.Mix well and pour prepared LB agar into plates and dry.</P><H2 id="Protein">Protein</H2><H3>Transform appropriate DNA plasmid into BL21(DE3) E. coli cells and make a starter culture for protein expression.</H3><P>1.Plate the cell culture onto LB agar plates. Incubate the plate at 37℃ for 12 to 14 hours (overnight generally).  Longer times of incubations will allow for satellite colonies to appear.</P><P>2.Pick a colony from your plate and add to 5ml LB broth/ampicillin tube. Incubate at 37℃ for 12 to 14 hours (overnight generally).</P><H3>Make a big batch of protein expression. Usually to about 50 mL of LB broth.</H3><P>1.Generally, the antibiotic is ampicillin, which is added to the LB broth at 100  g/ml final concentration. </P><P>2.Pick up 0.5ml seed broth and add to the LB broth/ampicillin flask. Incubate in a shaker (220 RPM) at 37℃ until the OD at 600 nm is 0.8 to 1.0 OD.  (Under these dense growth conditions the plasmid seems to be rejected or turned-off by BL21(DE3)s.)</P><P>3.Then induce the culture to express protein by adding 0.2 to 0.4mM IPTG.  Keep the culture shaking at 37℃.</P><P>4.Induction of protein generally takes 3 to 4 hours. Take sample of 1ml each hour. After induction, centrifuge your bacteria in 50 ml bottles in the rotor at the highest speed.  Do this in batches until all your culture is spun down. Freeze the cell paste at -80℃.</P><H3>Lysis and sonication of the bacteria. There are many ways to lyse bacteria, but this method is tried and true. </H3><P>1.Make Lysis Buffer: 25 mM TRIS-Cl, 2 mM EDTA, pH 7.6. Then add lysozyme at 100 g/ml concentration (or just a tipful from a spatula). Generally the volume lysis buffer is 1/20 to 1/50 the volume of the bacterial culture. </P><P>2.Resuspend the frozen cell paste as best you can in the Lysis Buffer using a 10 ml pipet. Let this suspension incubate for 20 minutes at room temperature, or until the suspension becomes turbid and viscous due to release of the bacteria's genomic DNA. Then centrifuge at highest speed for 30 minutes at 4℃.</P><P>3.Save the pellet and the supernatent. </P><H3>SDS-PAGE</H3><P><STRONG>Materials</STRONG></P><P>a)Sample buffer containing SDS and either sucrose or glycerol.</P><P>b)Mercaptoethanol.</P><P>c)1X Running Buffer,</P><P>25 mM Tris-HCL, 200 mM Glycine, 0.1％SDS, Approximately volume of water keeping below 1 liter that depends on electrophoresis system.</P><P>d)Sample Buffer</P><P>10％SDS, 10 mM Dithiothreitol or betamercapto-ethanol, 20％ Glycerol, 0.2 M Tris-HCL, Ph-6.8, 0.05％ Bromophenolblue
</P><P><STRONG>Procedure</STRONG></P><P>1.Gel pouring into miniprotein cassette</P><P>2.Electrophoresis </P><P>a)Take the glass frames from casting strand and keep in to electrophoresis chamber.</P><P>b)Fill the chamber with buffer and load the protein samples in to wells of gel then close the electrophoresis chamber.</P><P>c)Connect the red and black electrical leads in to the matching colored electrodes of the electrode assembly.</P><P>d)Put voltage around 120 volt and we can use 12% separating gel. Normally run the gel about 40-50 minutes till bromophenol dye has migrated to the bottom of gel.</P><P><STRONG>Protein analysis and detection</STRONG></P><P>Visualize the protein using Coomassie Brilliant Blue, silver stain, or we can use any other protein stain.</P><H3>Protein purification</H3><P>Use Ni-NTA to purificate the former protein solution, Elute with different concentration of imidazole.</P><P>1.Use ten times volume of ddwater to wash Ni-NTA.</P><P>2.Use ten times volume of binding buffer to balance Ni-NTA.</P><P>3.Add the lysate prepared twice.</P><P>4.Wash with ten times volume of binding buffer to wash off hybrid protein. </P><P>5.Wash with low concentration of washing buffer to wash off hybrid protein.</P><P>6.Wash with high concentration of washing buffer to wash off target protein. </P><P>7.Wash with ddwater and add 10ml strip buffer to rebalance Ni-NTA. </P><P>8.Add 10ml charge buffer, put Ni-NTA under 4℃ environment.</P></ARTICLE></DIV></DIV><DIV class="footer"><DIV class="footer-content"><P>Bio-X Institutes</P><P>Shanghai Jiao Tong University</P><P>800, Dongchuan Road 200240, Shanghai, China</P><P>sunyaojin2011@foxmail.com</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SJTU-BioX-Shanghai/Protocol">http://2014.igem.org/Team:SJTU-BioX-Shanghai/Protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/Protocol&amp;oldid=372855" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>