"
Page
Discussion
View source
History
teams
Log in
 
Team:Nevada/Notebook
From 2009.igem.org
Home
The Team
The Project
Parts
Modeling
Notebook
University of Nevada, Reno
Contents
1 June 8, 2009 to June 15, 2009
1.1 Janice/Leigh:
1.2 Tony/Nick:
2 June 16, 2009 to June 23, 2009
2.1 Janice/Leigh:
2.2 Chris/Joey:
2.3 Tony/Nick:
3 June 24, 2009 to June 30, 2009
3.1 Janice/Leigh:
3.2 Chris/Joey:
3.3 Tony/Nick:
3.4 Alex:
4 July 1, 2009 to July 7, 2009
4.1 Janice/Leigh:
4.2 Chris/Joey:
4.3 Tony/Nick:
4.4 Alex:
5 July 8, 2009 to July 15, 2009
5.1 Janice/Leigh:
5.2 Chris/Joey:
5.3 Tony/Nick:
5.4 Alex:
6 July 16, 2009 to July, 23 2009
6.1 Janice/Leigh:
6.2 Chris/Joey:
6.3 Tony/Nick:
6.4 Sheena:
6.5 J/Tyler:
7 July 24, 2009 to July 31, 2009
7.1 Janice/Leigh:
7.2 Chris/Joey:
7.3 Tony/Nick:
7.4 J/Tyler:
8 August 1, 2009 to August 8, 2009
8.1 Janice/Leigh:
8.2 Chris/Joey:
8.3 Tony/Nick:
8.4 Sheena:
8.5 J:
9 August 9, 2009 to August 16, 2009
9.1 Janice/Leigh:
9.2 Chris/Joey:
9.3 Tony/Nick:
9.4 Sheena:
10 August 17, 2009 to August 24, 2009
10.1 Janice/Leigh:
10.2 Chris/Joey:
10.3 Tony/Nick:
10.4 Sheena:
11 August 25, 2009 to September 1, 2009
11.1 Janice/Leigh:
11.2 Chris/Joey:
11.3 Tony/Nick:
11.4 Sheena:
12 September 2, 2009 to September 9, 2009
12.1 Janice/Leigh:
12.2 Chris/Joey:
12.3 Tony/Nick:
12.4 Sheena:
13 September 10, 2009 to September 17, 2009
13.1 Janice/Leigh:
13.2 Chris/Joey:
13.3 Tony/Nick:
13.4 Sheena:
14 September 18, 2009 to September 25, 2009
14.1 Janice/Leigh:
14.2 Chris/Joey:
14.3 Tony/Nick:
14.4 Sheena:
15 September 26, 2009 to October 3, 2009
15.1 Janice/Leigh:
15.2 Chris/Joey:
15.3 Tony/Nick:
15.4 Sheena:
15.5 J/Tyler:
16 October 4, 2009 to October 11, 2009
16.1 Janice/Leigh:
16.2 Chris/Joey:
16.3 Tony/Nick:
16.4 Sheena:
17 October 12, 2009 to October 19, 2009
17.1 Janice/Leigh:
17.2 Chris/Joey/Tony:
17.3 Nick:
17.4 Sheena:
17.5 J/Dr. Ellison:
June 8, 2009 to June 15, 2009
Janice/Leigh:
Made LB agar plates and liquid LB with tetracycline.
Made tetracycline stock solution (5mg/ml).
Tony/Nick:
RE digestion of RBS, Lac I, 2x Term, out of BioBrick Plasmids (2x Terminator digested with XbaI/SpeI, everything digested with NotI.)
Ran 0.8% Agarose gel of digestions above.
June 16, 2009 to June 23, 2009
Janice/Leigh:
Made LB liquid media and performed QIAGEN Plasmid Maxi Preps for
BBa_B0014 in pSB1AK3 (double terminator)
BBA_B0034 in pSB1A3 (ribosome binding site)
BBA_R0011 in pSB1A3 (lac I promoter)
BBa_I0500 in pSB2K3 (pBAD/Arac promoter).
Made glycerol stocks for 1) to 4).
Minipreps were done on the following Arabidopsis genes:
cinnamoyl-CoA reductase (CCR2)
cinnamoyl-CoA reductase (CCR1)
phenylalanine-ammonia lyase
4-coumarate:CoA ligase 1
4-coumarate:CoA ligase 2
Digestions:
Cinnamoyl-CoA reductase (CCR2) was digested with HindIII and SalI.
Cinnamoyl-CoA reductase (CCR1) was digested with EcorI and HindIII.
Phenylalanine-ammonia lyase was digested with EcoRI and SacI.
4-coumarate:CoA ligase 1 was digested with HindIII.
4-coumarate:CoA ligase 2 was digested with EcoRI and HindIII.
Chris/Joey:
Prepare chemically competent DB3 cells (from Invitrogen) as described here: [1]
TSS buffer was prepared as described in aforementioned article
Tested Comp cells by transformation of pGEM
Transform cells with 10ug pGEM
3 Amp plates: 10uL, 100uL, Remaining cells, Negative Control
Efficiency = 1X10^5
Transform BBa_I52001 and BBa_J04450 into DB3 with Neg. Control
BBa_I52001 transformation failed
Tony/Nick:
PCR of CCR2 (cinamoyl CoA reductase gene)
June 24, 2009 to June 30, 2009
Janice/Leigh:
BBa_J04450 in plasmid pSB1AT3 (RFP), BBa_J04450 in plasmid pSB1AT3 (ccdB), BBa_I52001 in plasmid pSB3T5 (ccdb), BBa_J04450 in plasmid pSB3T5 (RFP) were transformed into NEB10 chemically competent cells.  These are the tetracycline resistant plasmid backbones.
Cultures were grown and DNA was extracted for the plasmid backbones described above.
Chris/Joey:
Repeat BBa_I52001 transformation
Prepare liquid clutures of BBa_I52001 transformed cells
Miniprep unsing Qiagen Quickspin protocol
30.21 ng/uL BBa I52001
48.77 ng/uL BBa_I52001
Ran gel of miniprep digested with EcoRI to confirm size
Tony/Nick:
Topo clone CCR2, tranformed into One Shot, and spread on LB+Amp plates.
Isolated colonies from transformation and spread on fresh LB+Amp plates.
Minipreped transformed colonies from fresh plate.
RE digestion of Minipreped CCR2 and run on 0.8% Agarose gel.
RE digestion of CCR2, CCR1(another Cinamoyl CoA reductase), Phenyalanine Lyase (PAL1) and Ran Gel.
Alex:
Work on modeling of the genes in the pathway.
July 1, 2009 to July 7, 2009
Janice/Leigh:
The Arabidopsis genes were digested and were analyzed by gel electrophoresis.
The double terminator, ribosome binding site, lac I promoter, pBAD promoter, and two tetracycline resistant plasmid backbones were restriction digested and ran on an agarose gel.
Chris/Joey:
Restriction Digests were done with EcoR1 on Minipreps of BBa_I52001
.8% agarose gel was run to check band sizes of digests
Make NEB10 Competent Cells ; Check for Efficiency
Tony/Nick:
RE digestion of CCR2 with XbaI, EcoRI, PstI, SpeI (separate digests).
Sequenced CCR2.
Alex:
Researched the program Mathematica to write the program for the protein modeling.
July 8, 2009 to July 15, 2009
Janice/Leigh:
Performed the 3-way ligation for the lac I promoter (upstream part), ribosome binding site (RBS) (downstream part), and the destination plasmid (BBa_I52001 in pSB3C5; chloramphenicol resistance).
Digestion: lac I promoter was digested with EcoRI and SpeI; RBS was digested with XbaI and PstI; destination plasmid was digested with EcoRI and PstI.  All were digested at 37ºC for one hour and the restriction enzymes were deactivated at 80ºC for 20 minutes.
Ligation: The lac I promoter, RBS, and the destination plasmid (BBa_I52001 in pSB3C5) were ligated and incubated for one hour.
The 3-way ligation was transformed into NEB10 competent cells.
20 μl of the 3-way ligation described above were added to 100 μl of NEB10 competent cells.
Chloramphenicol was spread onto the LB plates to a final concentration of 25 μg/ml.
10, 100, and 160 μl of the transformation were added to 3 LB plates.
80 colonies of the 3-way ligation were screened on Amp and Chloramphenicol plates, resulting with 6 colonies that grew only on Chloramphenicol.
Chris/Joey:
Design Primers for PCR of U22800 for pEntr
Run PCR (Trial 1) - Check on gel - Failed
Run PCR (Trial 2) - Check on gel - Failed
Run PCR (Trial 3) - Check on gel - Positive Control Worked
Adjusting Annealing Temperatures for Primers
Tony/Nick:
PCR CCR2 and ran on gel.
Alex:
Wrote the program to make graphs of the activity of the proteins in the cinnamaldehyde producing pathway.
July 16, 2009 to July, 23 2009
Janice/Leigh:
Screening of the 3-way ligation (lac promoter/RBS/backbone) colonies
Six selected colonies were cultured in LB-Chloramphenicol (25μg/ml)
The DNA was collected using a QIAGEN miniprep kit and concentration was determined by the Nanodrop.
All six colonies were digested with EcoRI to linearize, expecting a fragment length of 2.7kb.
Digests were run on a 1% agarose gel along with uncut DNA.  Five of the six colonies resulted in the correct band length.
3-way ligation of RBS (BBa_R0011), pBAD promoter (BBa_I0500), and Chloramphenicol backbone (pSB3C5)
The upstream part, pBAD promoter, was digested with EcoRI and SpeI
The downstream part, RBS, was digested with XbaI and PstI
The Chloramphenicol bakcbone was digested with EcoRI and PstI
All digests were incubated for one hour at 37C followed by a 20 minute deactivation step at 80C
4μl of each of the three digests were used in a final 20μl ligation reaction with a one hour incubation at root temperature, followed by a 20 minute deactivation step at 80C
The ligation was transformed into NEB10 cells and volumes of 10, 50, 100μl, and the rest was spread on LB plates containing Chloramphenicol (25μg/ml)
The plates were incubated overnight at 37C
Chris/Joey:
Run PCR (Trail 4) using Gradient, change extension temperature to 68C - Check on gel - Failed
Checking primers to see if they are correct. (Turns out they were reversed - FAIL)
Redesign Primers Correctly
Tony/Nick:
Topo cloned CCR2 and transformed into NEB 10 Cells.
Plated Topo clones on Xgal/Amp plates.
Streaked selected colonies on Amp plates.
Minipreped and RE Digestion of CCR2 Topo clones.
Sheena:
Liquid Media Production
All media were produced as specified in "Modified from Callus Induction and Regeneration in Spirodela and Lemna" by J. Li, et al, and "High Expression of Transgene Protein in Spirodela" by Ron Vunsh, et al.  Some modifications were made, but overall, it's the same basic procedures.
pH for liquid media produced on July 21st was 5.77 (desired range = 5.6 - 5.8)
J/Tyler:
Produced minipreps of U22800 and candidate colonies transformed with lacI-RBS-BBa-IS2001 inserted in pSB3C5, and purified them using QIAspin filters and protocol.
July 24, 2009 to July 31, 2009
Janice/Leigh:
Screening of the 3-way ligation (pBAD promoter/RBS/backbone) colonies
7 colonies were selected and cultured in 3ml of LB-Chloramphenicol (25μg/ml)
Each colony was digested with PstI to linearize and a double digest of PstI/EcoRI to cut out the pBAD promoter(1.2kb)
The digests were run on a 1% agarose gel, none of which resulted with the expected fragments lengths.
Sequencing Results of the 3-way ligation (lac promoter/RBS/backbone)
Two colonies which resulted with the correct fragment length from the restriction digests were sent to be sequenced to the Nevada Genomics Center using the Biobrick primers VF2 and VR.  500ng of each of the colonies was used for each of the sequencing reactions
Chris/Joey:
Run PCR of U22800 for pEntr with new primers (Trial 5)
Check samples on agarose gel - Failed (Time to remake dNTPs)
Remake dNTPs and buy new Taq
Tony/Nick:
3 Way ligation of CCR2 with 2x terminator, and (BBa_52001) CPC resistant Backbone.
Transformation of 3 Way into NEB 10.
J/Tyler:
Miniprep of culture from E.coli U22800 glycerol stock to verify correct plasmid transformation.
All purifications with QIAspin columns failed
August 1, 2009 to August 8, 2009
Janice/Leigh:
Repeated the 3-way ligation for the lac I promoter (upstream part), ribosome binding site (RBS) (downstream part), and the destination plasmid (BBa_I52001 in pSB3C5; chloramphenicol resistance).
Digestion: lac I promoter was digested with EcoRI and SpeI; RBS was digested with XbaI and PstI; destination plasmid was digested with EcoRI and PstI.  All were digested at 37ºC for one hour and the restriction enzymes were deactivated at 80ºC for 20 minutes.
Ligation: The lac I promoter, RBS, and the destination plasmid (BBa_I52001 in pSB3C5) were ligated and incubated for one hour.
The 3-way ligation was transformed into NEB10 competent cells.
20 μl of the 3-way ligation described above were added to 100 μl of NEB10 competent cells.
Chloramphenicol was spread onto the LB plates to a final concentration of 25 μg/ml.
10, 100, and 160 μl of the transformation were added to 3 LB plates.
80 colonies of the 3-way ligation were screened on Amp and Chloramphenicol plates, resulting with 6 colonies that grew only on Chloramphenicol.
Sequencing results resulted that the lac I promoter had mutated and was not the complete sequence
Re-transformation of Lac I promoter (BBa_R0011) in NEB10 company competent cells
selected one colony and cultured in 3ml of LB-Amp
DNA of the lac I promoter was collected using the QIAGEN miniprep kit
Chris/Joey:
Run PCR with new primers and dNTPs (Trial 6)
Check on gel - PCR was Successful, finally
Samples degraded
Re-Run PCR same as Trial 6
Check on gel - Successful again
Tony/Nick:
Streaked colonies from CCR2 + 2x term + I52001 CPC onto LB+CPC plates.
Selected colonies and Minipreped them.
Repeat of 3-way ligation, selection, and miniprep.
Sheena:
Produced Lemna Callus Media
pH was 5.62
J:
Two minipreps of E.coli U22800 culture from liquid stock, one purified via QIAspin column protocol, the other purified via alkali lysis protocol.  Results showed a much greater and yield of plasmid via alkali lysis.
August 9, 2009 to August 16, 2009
Janice/Leigh:
Made NEB10 competent cells
Checked for competency by transforming the cells with 100ng of PUC
Sent the chosen Lac I promoter (BBa_R0011) in for sequencing to the Nevada Genomics Center. The promoter was sequenced in both directions using Biobrick primers VF2 and VR.
Sequencing results confirmed that BBa_R0011 contains the complete lac I promoter sequence
Cultures set for the promoter in LB-Amp and DNA was collected using a QIAGEN miniprep kit
3-way ligation of lac I promoter (BBa_R0011), RBS (BBa_B0034), and destination plasmid pSB3C5
The upstream part, lac I promoter, was digested with EcoRI and SpeI
The downstream part, RBS, was digested with XbaI and PstI
The Chloramphenicol bakcbone was digested with EcoRI and PstI
All digests were incubated for one hour at 37C followed by a 20 minute deactivation step at 80C
4μl of each of the three digests were used in a final 20μl ligation reaction with a one hour incubation at root temperature, followed by a 20 minute deactivation step at 80C
The ligation was transformed into NEB10 cells and volumes of 10, 50, 100μl, and the rest was spread on LB plates containing Chloramphenicol (25μg/ml)
The plates were incubated overnight at 37C
Chris/Joey:
Run PCR for gel extraction to isolate 1kb fragment of Gene 3
Used QIAGEN Gel Cleanup kit for extraction
Gel extraction was unsuccessful, lost almost all the DNA in the process
Develop biobrick plant plasmid
Tony/Nick:
Minipreps from 3 way ligation (CCR2+2xTerm+I52001)ran on 1% agarose gel.
RE Digestion of 3 way to confirm insert
3 way sequenced
Sheena:
Produced SH + Sucrose media (500mL + 500mL in seperate flasks)
pH was 5.76 and 5.64, respectively.
August 17, 2009 to August 24, 2009
Janice/Leigh:
Screening of the 3-way ligation (lac promoter/RBS/backbone) colonies
62 colonies were replica plated on LB-AMP and LB-chloramphenicol
Four selected colonies that only grew on LB-chloramphenicol were cultured in LB-Chloramphenicol (25μg/ml)
The DNA was collected using a QIAGEN miniprep kit and concentration was determined by the Nanodrop.
All six colonies were digested with EcoRI to linearize, expecting a fragment length of 2.7kb.
Digests were run on a 1% agarose gel along with uncut DNA.  Three for the four colonies resulted in the correct band length.
DNA from two of the colonies was sent for sequencing at the Nevada Genomics center.  Both colonies were sequenced in two directions using the Biobrick primers VF2 and VR.
Chris/Joey:
Make plates in prep for Transformation of gene 3 in pEntr D into E. Coli
Plates are LB - Kan
Carry out TOPO cloning Reaction for Transformation
Continue Development of Biobrick Plant Plasmid
Tony/Nick:
4-coumarate: CoA Ligase (Gene 2) minipreped.
Gene 2 RE Digested to linearlize (SalI).
Sheena:
Transfered Callus Duckweed to new media.
Liquid media (original grown) were poured into autoclaved beakers under sterile hood, and then plants were transfered into fresh media using sterile spatula.
Solid media calli were transfered via sterile protocol (tweezers/forceps used to grip and transfer lemna to new plate). Unhealthy Lemna (yellow) were left in original plate media to grow for next two weeks.
August 25, 2009 to September 1, 2009
Janice/Leigh:
Sequencing results of the 3-way ligation (lac/RBS/chloramphenicol backbone) resulted with the correct sequence
Bulk-up DNA of the 3-way ligation using the I, II, III miniprep protocol
Transformation of mutated AT4CL2 (4-Coumarate:CoA ligase 2) sent from Dr. Kombrink into DH5α
Chris/Joey:
Transformation of Gene 3 into Ecoli
Plate Control and Experimental samples
12 Colonies grew - All will be minipreped
Miniprep the 12 Colonies using 1,2,3 Miniprep
Continue Development of Plant Biobrick Plasmid
Tony/Nick:
PCR pf Gene 2.
Sheena:
SH + Sucrose Duckweed Transfer
Transfered individual species of Lemna and Wolffia to 2 new plates each. Performed under sterile conditions, an original plate and a closed (left to grow for 2 more weeks in case of contamination to new plates) plate were kept for each specimen type (Lemna gibba, Lemna minor, and 2 species of Wolffia) while two new plates were acquired for each specimen type. Closed and original plates will be disposed of upon observation of healthy plates before next transfer.
September 2, 2009 to September 9, 2009
Janice/Leigh:
Selected colonies of the AT4CL2 transformation were cultured in LB-Amp
DNA was collected using the QIAGEN miniprep kit
DNA was digested with PstI and Sal I
3-way ligation of final construct (lac I promoter/RBS/Gene 3/terminator)
500 ng of the upstream part, lac/RBS 3-way ligation, was digested with EcoRI and SpeI
500ng of the downstream part, Gene 3/terminator 3-way ligation, was digested with XbaI and PstI
500 ng of the high copy tetracylcine resistant backbone (pSB1AT3) was digested with EcoRI and PstI
All digest were incubated for an hour at 37C and deactivated for 20 minutes at 80C
4μl of each of the digests was used for the ligation reaction
the ligation reaction was incubated at room temperature for 1 hour and deactivated for 20 minutes at 80C
The ligation was transformed into NEB10 competent cells and plated on LB-Tet
Chris/Joey:
Run Gel to Check Minipreps - Samples 2,4,5, and 6 will be checked by restriction digest
Restriction Digest of 2,4,5,and 6 with Sal 1 and Not 1 (Digest by Sal1 to linearize, digest by Sal1 and Not 1 to cut out insert)
Run Gel to check Digests
Samples 4, and 5 are not what we want
Samples 2, and 6 have the expected band size but the gene may be going into the vector in the wrong orientation
Restriction Digest of samples 2,6,7,8,9,10,11, and 12 (Double Digest: Sal1/Not1)
Tony/Nick:
3 way ligation of (gene 2 + Lac/RBS + I52001 CPC BB)
3 way ligation of (gene 3 + Lac/RBS + I52001 Tet BB)
2 way ligation of (Gene 2 + I52001 CPC BB)
Sheena:
Calli and Liquid Transfer
Looked at Calli progress under microscope and took pictures in order to document. Transfered each calli (productive) to new plate.  Also transfered the liquid media Lemna. Growth is plentiful an very successful on each specimen/sample).
September 10, 2009 to September 17, 2009
Janice/Leigh:
Screening of the colonies from the 3-way final construct (promoter/RBS/Gene3/terminator)
Colonies were replica plated on LB-tetracycline and LB-chloramphenicol plates
Four of the six colonies grew only on the LB-tetracycline plates
The four colonies were digested with SalI and the digests were run on a 1% agarose gel
None of the digests resulted with the correct fragment lengths
Redo of the 3-way ligation final construct (promoter/RBS/Gene3/terminator)
Six colonies were replica plated on LB-tetracylcine and LB-chloramphenicol plates
Four colonies grew only on LB-tetracycline
DNA of the four colonies was collected using the QIAGEN miniprep kit
DNA was digested with EcoRI and run on a 1% agarose gel
DNA from four colonies was sent for sequencing to the Nevada Genomics Center using the Biobrick primers, VF2 and VR.
Chris/Joey:
Gel for Restriction Digests of pEntr Colonies 2,6,7,8,9,10,11,12
Sample 7 Cut Correctly - Displaying 2.7kb band for the plasmid and 1kb band for the gene
Grow Cultures of pK7WG2D.1 binary vector
Miniprep pK7WG2D.1 using QIAquickspin protocol
Check minipreps on agarose gel - Successful
Tony/Nick:
Transformation of 3 way ligations, and 2 way ligation above into NEB 10.
RE Digestion of 3 way ligations, and 2 way ligation and ran on 1% agarose gel
Sheena:
Produced Liquid Media
pH was 5.62
SH Media Duckweed Transfer
Transfered Lemna and Wolffia to new plates via solid media transfer protocol.
Liquid Media and Calli Transfer
Transfered liquid media specimens and calli via protocol.  One calli from last transfer was found contaminated, but only on one small area on side of plate, so I transfered that plate very last. Labeled the plate "contaminated" in an attempt to possibly salvage calli. If plate is again contaminated upon next transfer, the plate and its specimens will be disposed of.
September 18, 2009 to September 25, 2009
Janice/Leigh:
Sequencing results returned did not contain the sequence of the final construct
Chris/Joey:
LR Clonase Reaction was carried out
LB- Spec plates were made in prep for transformation of LR Clonase Reaction
LR Clonase Reaction was transformed into DH5alpha Cells and plated (pUC19 Positive Control used)
14 Colonies Grew on the plates
Tony/Nick:
Sheena:
Lemna and Wolffia Transfer
Regular transfer of duckweed to SH + Sucrose media.
September 26, 2009 to October 3, 2009
Janice/Leigh:
Re-do of the 3-way final construct: promoter/RBS/Gene 3/terminator
New Tetracycline stock and LB-tetracycline plates were prepared
Two colonies grew
Cultures were set for high-copy tetracycline backbone, low-copy tetracycline backbone, the two final construct colonies and the lac/RBS
DNA was digested and run on a 1% agarose gel
The two colonies of the final construct did not result with the correct size
Chris/Joey:
Made cultures of all 14 colonies for minipreps
Miniprep using QIAquickspin protocol
Run gel to check minipreps - Sample 7 Excluded from future Restriction Digests
Double digest of Minipreps 1-14 (excluding 7) using Spe1 and Sac2
Gel of Double Digests - Successful all colonies cut correctly.
Sending in for sequencing to further confirmation
Tony/Nick:
Minipreped 3 way ligations above, RE digested, and ran on 1% agarose Gel
Sheena:
Media Production
Produced 500mL of callus media and 1L of SH + Sucrose media. Labeled agar plates accordingly.
pH was 5.73 and 5.68, respectively.
Infection Media and Growth Media Production
Produced infection and growth mediums in preparation for wolfia transformations.
Acetocyringone was not yet added when making media and must still be added before agrobac. are added.
Calli Transfer
Contaminated plate was still contaminated and was, therefore, disposed of.
Transfered Lemna to new (fresh) media.
J/Tyler:
IPTG expression:
Transformation of E.coli BL21 strain with lacI-RBS-modified 4-coumarate:CoA ligase in an ampR plasmid vector (donated by Dr. Kombrink at the Max Planck Institute, Munchen, Germany) via heat shock protocol, sample was cultured o/n on LB-amp.
Culture was used to produce two 25mL culture flasks of LB-amp, each OD600~0.1.  One culture was grown to OD600~0.5, at which point IPTG was added to 5mM.  Sample pellet was taken at t=0hr, and again at t=7.0hr.  The other culture was also pelleted for use as a non-induced control.  Pellets were stored at 0°C.
October 4, 2009 to October 11, 2009
Janice/Leigh:
Last attempt for 3-way ligation of lacI/RBS/Gene 3/term.  Attempt was unsuccessful and the protocol will be revised, possibly using a 2-way ligation procedure.
Chris/Joey:
Nanodrop Minipreps of the 14 binary colonies (Successful around 200-400 ng/ul)
Pause to help with other constructs
Miniprep fresh Gene2/Terminator, RBS, an Gene3/Terminator samples for final 3-way attempt.
Tony/Nick:
3 way ligation of (Gene 2 + RBS + I52001 Tet BackBone.)
Minipreped Janice and Leigh's Final Construct of (LacI/RBS + Gene 3 + 2x Term + I52001 TET BB).
RE Digestion (EcoRI/PstI) of (Gene 2 + 2x term + I52001 CPC BB).
RE Digestion (EcoRI/PstI) of (Gene 2 + RBS + I52001 Tet BB).
Sheena:
SH + Sucrose Media Transfer
Transfered Lemna and Wolffia to new SH + Sucrose agar plates. All looks healthy.
Liquid Media Transfer
Transfered Liquid Media specimens. Lemna looked fairly yellow over-all, so the healthier-looking green plants were picked out via forceps and placed in new liquid media. Otherwise, specimens are growing healthily.
October 12, 2009 to October 19, 2009
Janice/Leigh:
Transformation of 2-way ligation into NEB10 competent Cells.  20 colonies were cultured for screening of the final construct.  Minipreps using the QIAGEN Miniprep Spin Kit resulted with insufficient DNA.
Chris/Joey/Tony:
Help with 2 way ligation of Gene3/term with Lac/RBS and Gene2/Term with Lac/RBS
Nick:
Worked on the Nevada Wiki
Updated the Parts Registry
Sheena:
Propagated the duckweed population by transferring the plants into new media.  Callus induction maintained of Lemna minor.
J/Dr. Ellison:
Dismembrated transgenic E. coli from last month, as well as a wild-type culture for control--pelleted 25mL cultures were resuspended in 1mL 50mM sodium phosphate 200mM sodium chloride buffer, sonicated for 1 minute, then clarified by centrifugation
Purified transgenic E.coli extracts on a nickel column ("HiTrap nickel chelating column," GE Biosciences).
Performed protein concentration assays on clarified extracts and purified fractions by measuring absorbance at 280nm.  Protein concentrations were determined to be as follows (in mg protein/mL solution):
Wild type clarified extract: 86.2
Uninduced transgenic culture clarified extract: 196.3
IPTG-induced transgenic culture clarified extract: 7.9
Purified IPTG-induced transgenic culture extract (most concentrated fraction): 0.65
Performed time-course kinetic assays of 4-coumarate:CoA ligase activity in the wild-type clarified extract, the uninduced transgenic culture clarified extract and the IPTG-induced transgenic culture clarified extract. Assay was performed as follows:
Assay mixture: 3.3mg trans-cinnamic acid dissolved in 50uL ethanol, 2.0mg coenzyme A and 6.9mg adenosine triphosphate were suspended in 10mL 50mM Tris buffer 2.5mM magnesium chloride, pH 7.5.
50uL extract was added to 500uL assay mixture at room temperature in a quartz cuvette, and change in absorbance was measured at 311nm.
Assay results indicate that 4-coumarate:CoA ligase is constitutively expressed in transgenic E. coli BL21 strain cells despite lacI site regulation, and is not at all expressed in wild-type E. coli.
Ran a 12% SDS-PAGE of wild-type E. coli extract, uninduced transgenic E. coli extract and IPTG-induced transgenic E. coli extract
Results indicate that 4-coumarate:CoA ligase is expressed in transgenic cells regardless of induction, and not at all in wild-type cells.  Unfortunately, the purified 4-coumarate:CoA ligase extract does not keep well in the freezer, so no activity was recorded for the nickel column eluent.
Retrieved from " http://2009.igem.org/Team:Nevada/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
