"
Page
Discussion
View source
History
teams
Log in
 
Team:Nevada/Modeling
From 2009.igem.org
Home
The Team
The Project
Parts
Modeling
Notebook
University of Nevada, Reno
Contents
1 Modeling
2 Figure 1
3 Figure 2
4 Figure 3
5 Figure 4
6 Figure 5
7 Table 1
8 References
Modeling
We developed a computer model for measuring enzyme kinetics of our pathway (Figure 1) . The model was developed using Mathematica based upon steady-state Michaelis–Menten kinetics (Figure 2) . It revealed that the kinetic parameters for the wild type isoform of 4-coumarate:CoA ligase were creating a bottleneck and significantly slowing down the overall reaction (Figure 3) . Our group postulated and tested two potential solutions to get around this problem:
(1)
To use a mutant enzyme (Figure 4) with more favorable kinetic parameters or
(2)
To increase the concentration of the wild type isoform of 4-coumarate:CoA ligase (Figure 5) . The mutant isoform of 4-coumarate:CoA ligase was selected for based on a 2000 paper by Stuible, et al. at the Max Planck Institute who altered three polar amino acids into hydrophobic groups and discovered the 4-coumarate:CoA ligase mutant's affinity for the nonpolar sites of cinnamic acid increased dramatically (3) .  For all of the kinetics characterized assume all of the enzymes are in equal concentrations, except for the model involving 10x the concentration of the wild type isoform of 4-coumarate:CoA ligase (Figure 5) . We chose to investigate both of these methods to insure no ill-conceived reaction kinetics were occurring.
While we have a hypothetical  model for the reverse reaction, it is purely speculative as no real world data exists characterizing the reverse reaction rates of the aforementioned enzymes. This also means our model does not account for reverse reaction kinetics due to the lack of available data. We also used kinetic values from a highly conserved variant of Cinnamoyl-CoA reductase called Feruloyl-CoA reductase. Data for the latter was obtained from the Arabidopsis thaliana isoform and is structurally similar enough to act as a proper substitute for Cinnamoyl-CoA reductase (Table 1) .
Our model yielded vital information regarding the proper kinetic parameters our team needed to consider in order to establish an efficient, and effective pathway.
Figure 1
Cinnamaldehyde Synthesis Pathway
Figure 2
Steady State Model of Cinnamaldehyde Production from an L-Phenylalanine Precursor
using the wild type isoform of 4-coumarate:CoA ligase
Figure 3
Steady State Model of Cinnamaldehyde Production from an L-Phenylalanine Precursor
using the mutant isoform of 4-coumarate:CoA ligase
Figure 4
Steady State Model of Cinnamaldehyde Production from an L-Phenylalanine Precursor
using 10 times the concentration of the wild type isoform of 4-coumarate:CoA ligase
Figure 5
Modeling Source Code
Table 1
Enzyme Modeling Kinetics and Environmental Conditions
References
Mulquiney, P.J., Kuchel, P.W. Modeling Metabolism with Mathematica. CRC Press: 2003.
Schneider, K., Hovel, K., Witzel, K., Hamberger, B., Schombur, D., Kombrink, E., Stuible, H.P. 2003. The substrate specificity-determining amino acid code of 4-coumarate:CoA ligase. PNAS. 100, 8601-8606.
Stuible, H.P., Buttner, D., Ehlting, J., Hahlbrock, K., Kombrink, E. 2000. Mutational analysis of 4-coumarate:CoA ligase identifies functionally important amino acids and verifies its close relationship to other adenylate-forming enzymes. FEBS Letters. 467, 117-122.
Retrieved from " http://2009.igem.org/Team:Nevada/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
