<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NUDT_CHINA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NUDT CHINA/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>NUDT_CHINA 2016</TITLE></P><DIV class="off-canvas-wrap" data-offcanvas=""><DIV class="inner-wrap"><DIV id="header-outer-wrap" style=""><HEADER id="header" class="row"><DIV class="medium-24 columns"><A href="/Team:NUDT_CHINA"><H1 class="site-id">Header...</H1></A><UL class="nav-utility inline-list"><LI class="nav-utility__item" style="height:16px;"> </LI></UL><NAV class="nav-global" role="navigation"><UL class="nav-global__list inline-list"><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="#">TEAM</A></LI><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="#">PROJECT</A></LI><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="#">PARTS</A></LI><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="/Team:NUDT_CHINA/Safety">SAFETY</A></LI><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="/Team:NUDT_CHINA/Attributions">ATTRIBUTIONS</A></LI><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="#">HUMANPRACTICES</A></LI><LI class="nav-global__item trigger "><A class="nav-global__link" title="" href="#">AWARDS</A></LI></UL><DIV class="content support" id="menu01"><DIV class="column-1"><NAV class="dropdown-arrow-list"><UL><LI><A href="/Team:NUDT_CHINA/Team">Team</A></LI><LI><A href="/Team:NUDT_CHINA/Collaborations">Collaborations</A></LI></UL></NAV></DIV></DIV><DIV class="content support" id="menu02"><DIV class="column-1"><NAV class="dropdown-arrow-list"><UL><LI><A href="/Team:NUDT_CHINA/Description">Description</A></LI><LI><A href="/Team:NUDT_CHINA/Design">Design</A></LI><LI><A href="/Team:NUDT_CHINA/Experiments">Experiments</A></LI><LI><A href="/Team:NUDT_CHINA/Proof">Proof of Concept</A></LI><LI><A href="/Team:NUDT_CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:NUDT_CHINA/Results">Results</A></LI><LI><A href="/Team:NUDT_CHINA/Notebook">Notebook</A></LI></UL></NAV></DIV></DIV><DIV class="content support" id="menu03"><DIV class="column-1"><NAV class="dropdown-arrow-list"><UL><LI><A href="/Team:NUDT_CHINA/Parts">Parts</A></LI><LI><A href="/Team:NUDT_CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:NUDT_CHINA/Composite_Part">Composite Parts</A></LI><LI><A href="/Team:NUDT_CHINA/Part_Collection">Part Collection</A></LI></UL></NAV></DIV></DIV><DIV class="content support" id="menu06"><DIV class="column-1"><NAV class="dropdown-arrow-list"><UL><LI><A href="/Team:NUDT_CHINA/Human_Practices">Human Practices</A></LI><LI><A href="/Team:NUDT_CHINA/HP/Silver">Silver</A></LI><LI><A href="/Team:NUDT_CHINA/HP/Gold">Gold</A></LI><LI><A href="/Team:NUDT_CHINA/Integrated_Practices">Integrated Practices</A></LI><LI><A href="/Team:NUDT_CHINA/Engagement">Engagement</A></LI></UL></NAV></DIV></DIV><DIV class="content support" id="menu07"><DIV class="column-1"><NAV class="dropdown-arrow-list"><UL><LI><A href="/Team:NUDT_CHINA/Model">Model</A></LI></UL></NAV></DIV></DIV></NAV></DIV></HEADER></DIV><DIV id="feature" class="row collapse text-light feature-footer-active" style="background-color: rgb(3,13,38);background-image:url(https://static.igem.org/mediawiki/2016/thumb/3/3f/NUDT_CHINA2016_FILES-img-banner-3.0.png/800px-NUDT_CHINA2016_FILES-img-banner-3.0.png);background-position:0px -350px;background-repeat:no-repeat;"><DIV class="inner-wrap medium-24 columns"><DIV class="carousel-container hide-pager" id="c4811"><DIV class="carousel " id="slideshow-4811" data-cycle-log="false" data-cycle-auto-height="calc" data-cycle-prev="#prev-4811" data-cycle-next="#next-4811" data-cycle-pager="#cycle-pager-4811" data-cycle-easing="easeInOutQuad" data-cycle-fx="scrollVertUp" data-cycle-center-horz="true" data-cycle-speed="1000" data-cycle-timeout="8000" data-cycle-paused="true" data-cycle-slides="&gt; div.slide"><DIV class="slide first"><DIV class="row collapse " id="c6601" data-theme-class="bh-color__theme-c6601"><DIV class="columns"><SECTION class="big-header bh__height-size-3 text-light bh__video" style="height:300px;"><DIV class="bh__content-wrap" style="height:120px;"><DIV class="bh__content "><DIV class="bh__content-inner-wrap"><H2 style="font-size:36px;" class="bh__title "><SPAN style="line-height:46px;" class="line-wrap"><SPAN style="line-height:46px;" class="line">Development of A Novel</SPAN></SPAN><SPAN style="line-height:46px;" class="line-wrap"><SPAN style="line-height:46px;" class="line">Blood-MicroRNA Handy Detection System with CRISPR</SPAN></SPAN></H2></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="margin-top:20px;" id="content-wrap-ink"><DIV class="row footer-link" style=""><DIV style="text-align:right;"><H5><A style="color:rgb(10,31,84);" href="/Team:NUDT_CHINA">HomePage</A>•PROJECT •<A style="color:rgb(10,31,84);" href="/Team:NUDT_CHINA/Description">Description</A></H5></DIV><H2><SPAN><SPAN style="color:#7f1015">Abstract</SPAN></SPAN></H2><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">MicroRNAs,
serve as critical gene expression regulators at the transcriptional and
post-transcriptional levels, have also been found as important blood-based
biomarkers for early detection of cancers. However, their current in vitro
detection methods are relatively complex, costly and low sensitive. Our project
attempts to establish a novel in vitro microRNA detection system which is
efficient, sensitive, specific, low-cost and output-visible. In this system, CRISPR-Cas9 technique
is modified to integrate with split-HRP reporting systems.
The advanced chain-replacement and rolling circle amplification technologies are also involved and optimized. This system may ideally be compatible
for the detection of various series of small non-coding RNAs. To our knowledge,
we are the first to use the CRISPR-Cas9 system as a small non-coding RNA
monitor in vitro. Its establishment and further development might provide a new
approach for rapid and low-cost cancer screening, virus detection and curative
efficacy assessment.</SPAN></P><H2><SPAN><SPAN style="color:#7f1015">Introduction</SPAN></SPAN></H2><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Nowadays,
cancers, due to their high incidence and serious mortality, are affecting
populations in all countries and all regions (Figure 1). However, in most
countries, resources for prevention and diagnosis of cancer still remain
limited due to their high cost and low cost-effectiveness</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">1</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, whereas the early
detection of cancer has been proven to result in improved survival, less
extensive treatment and less possibility to metastasis</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">2-4</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. Such situation
highlighted the undiminished importance of the development of a low-cost,
easily accessible and rapid tool for early screening and detection of cancers.</SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P><P><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Figure 1.
Global distribution of estimated age-standardized world cancer incidence rate
(ASR) per 100 000 in (A) men, (B) women, and mortality rate (ASR) per 100 000
in (C) men and (D) women. (WHO, World cancer report, 2014)</SPAN></B></P><P align="center" style="text-align:center;text-indent:22.1pt;"><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></B></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">MicroRNAs
(miRNAs), as a kind of small non-coding RNA containing approximately 22 nucleic
acids, have been proven to play important roles on post-transcriptional regulation
of the gene expression, thus involving in the regulation of many important
biological events</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">5</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. Recently, it was reported
that serum miRNAs can serve as a promising cancer biomarker because their
expression pattern can be correlated with cancer type, stage, and other
clinical variables, which then, implying that miRNA profiling can be used as a
tool for cancer diagnosis and prognosis</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">6-8</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. Moreover, circulating miRNAs
have been proven to remain stable under some extreme condition such as RNase
exposure, multiple freeze-thaw cycles, and extreme pH, thus making them strong
candidates for low-cost detection and analysis </SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">9</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. However, due to their
short length, low expression level and high homologous sequence similarity, the
quantified detection and analyzation of circulating miRNAs remain challenging
nowadays. Old-schools such as Northern Blotting, microarray and qRT-PCR
technique are still our approach to detect and analyze the quantity of miRNA</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">10</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. Notably, the expanded
application of these techniques, as well as some other new approaches such as
bioluminescence</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">11</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, Nanopore sensors</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">12</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> were severely limited
due to their relatively low sensitivity (which were mostly nM sensitivity
against the pM or even fM concentration of blood miRNA), cumbersome and complex
in operation, and relative high cost. More recently, Deng </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">et.al </SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">reported a single-molecule resolution </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">in situ</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> miRNA detection technique based on rolling circle
amplification (RCA)</SPAN><SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">13</SPAN></SUP><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. However, this approach
has been restricted only in the application of cellular </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">in situ</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> analysis. Its expandability to circulating miRNA detection
still faces a major problem that the degree of one-step signal amplification and
differentiation might not be sufficient to meet the requirements of sensitivity
and specificity. At the main time, such method still relies on equipment such
as Fluoresce microplate readers or fluorescence microscopes, which are highly
costly. </SPAN></P><P><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Figure 2.
Workflow for our CRISPR-based blood-microRNA detection system.</SPAN></B></P><P><SPAN style="line-height:2;font-family:Perpetua;font-size:16px;">Using sequence information from online databases,
probes for RCA reaction were designed in silico. Once synthesized and sealed to
form the dumbbell structure, the probe, together with other necessary materials
can be embedded into tubes and freeze-dried to remain stable in room
temperature for a relatively long time. For the detection process, serum
samples were pre-treated by boiling in 95</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">℃</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> for 15 min to expose
the miRNAs completely. The amount of the specific RNA was indicated by a color
difference in the tube from colorless to blue</SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">In
our project, we designed a novel cell-free platform built with synthetic
bio-components to achieve the low-cost, handy and visualized detection of serum
miRNAs, which can be employed in low-resource settings (Figure 2). Using miR
let-7a (a bio-marker for non-small cell lung cancer (NSCLC)) as a demo of our
scheme, we modified the RCA based DNA amplification system and introduced it
into nucleic acid detection in liquid samples such as serum, and then conducted
Sybr I mediated fluorescent assay for its validation and assessment. The
improvements of sensitivity and specificity of RCA output signal as well as the
visualization of RCA outputs were achieved through a single guide RNA (sgRNA)
mediated dCas9 binding system and a conjugated split-HRP reporting system.
Meanwhile, a mathematic model was also developed to provide theoretical
approval to our scheme and basic guideline for wet-lab experiments. Finally, we
employed a simple sample-pretreatment protocol to reliably expose miRNAs in
serum samples and demonstrated robust detection with this scheme to compare
let-7a concentrations among blood samples collected from NSCLC patients and healthy
volunteers. </SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P><H2><SPAN><SPAN style="color:#7f1015">Improvement We Made</SPAN></SPAN></H2><P><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"><STRONG>BBa_I715019 and BBa_K1789004</STRONG></SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">In
our project this year, a new protein-protein interaction (PPI) toolkit
containing several split reporting systems were modified and designed and
introduced into the registry. As a classical PPI indicator, split-GFP system,
developed previously in our project in iGEM2015 (BBa_K1789004 and
previous-existing BBa_I715019), was also included in our kit. Several improvements has been made
for this system including:</SPAN></P><P style="text-indent:-0.55pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">1.</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">      </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Improved
characterization for previous parts</SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">To
further improve the function of existing parts, we stimulate an </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">in vivo</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> PPI situation, and tried to
optimize the culture condition for a better signal-to-noise ratio (SNR). For
such matter, two devices, containing split-GFP fragments and a complete or
spited zinc finger protein, were built under control of a lac operon controlled
T7 promoter. The complete zinc finger protein was to stimulate a PPI positive
situation, while the split one was to stimulate a PPI negative situation
. Fluorescence signal was detected by a microplate reader after an
overnight culture under various conditions.</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Relative fluorescence intensity was then calculated with normalization
of OD</SPAN><SUB><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">600</SPAN></SUB><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> value. The relative fluorescence intensity of each control
group was set arbitrarily at 1.0 (data not shown), and the levels of the other
groups were adjusted correspondingly. Results shown a better SNR under 20</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">℃</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> and 0.5mM IPTG induction (Figure 3). Thus indicating that better performance
of such system could be expected under lower culturing temperature.</SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P><P><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Figure 3. Evaluation of the split-GFP system under different expression condition. </SPAN></B></P><P><SPAN style="line-height:2;font-family:Perpetua;font-size:16px;">Relative fluorescence intensity was calculated with normalization of the OD600 value. The relative fluorescence intensity of each control group was set arbitrarily at 1.0 (data not shown), and the levels of the other groups were adjusted correspondingly. Green fluorescence was measured under 488nm of excitation and 538nm of emission. This experiment was run in three parallel reactions, and the data represent results obtained from at least three independent experiments. *p&lt;0.05, **p&lt;0.01.</SPAN></P><P style="text-indent:6.5pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">2.</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">     </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">To
further improve the function of split-GFP system, another method of splitting
GFP was introduced and tested in our project. Instead of a traditional two-part
split, we split the GFP protein into three fragments namely GFP10 (residues
194-212), GFP11 (residues 213-233) and GFP 1-9 (residues 1-193)<SUP>14</SUP>. Due to their short
length, two small fragments can be easily fused onto proteins with less
affection on their folding </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">(figure 4A)</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">. </SPAN></P><P><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P><P><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Figure 4 Evaluation of two different split-GFP systems.</SPAN></B></P><P><SPAN style="line-height:2;font-family:Perpetua;font-size:16px;">Relative fluorescence intensity was calculated with normalization of the OD600 value. For Relative FI ratios, relative fluorescence intensity of each control group was set arbitrarily at 1.0, and the levels of the other groups were adjusted correspondingly. This experiment was run in three parallel reactions, and the data represent results obtained from at least three independent experiments. *p&lt;0.05, **p&lt;0.01.</SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P><P style="text-indent:22pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Comparing
with previous split-GFP system, higher SNR was reached under the same
expression condition, while the total signal intensity suffered tolerable
decrease (Figure 4B).</SPAN></P><P><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P><H2><SPAN><SPAN style="color:#7f1015">Reference</SPAN></SPAN></H2><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">1</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Jeong,
K. E. &amp; Cairns, J. A. Review of economic evidence in the prevention and
early detection of colorectal cancer. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Health
Econ Rev</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">3</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 20,
doi:10.1186/2191-1991-3-20 (2013).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">2</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Etzioni, R.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> The case for early detection. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Nat Rev Cancer</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">3</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">,
243-252, doi:10.1038/nrc1041 (2003).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">3</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Wolf, A. M.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> American Cancer Society guideline for the early detection
of prostate cancer: update 2010. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">CA
Cancer J Clin</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">60</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 70-98,
doi:10.3322/caac.20066 (2010).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">4</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">McPhail, S., Johnson, S., Greenberg,
D., Peake, M. &amp; Rous, B. Stage at diagnosis and early mortality from cancer
in England. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Br J Cancer</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">112 Suppl 1</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, S108-115,
doi:10.1038/bjc.2015.49 (2015).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">5</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Ambros, V. MicroRNA pathways in flies
and worms: growth, death, fat, stress, and timing. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Cell</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">113</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 673-676
(2003).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">6</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Lu, J.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> MicroRNA expression profiles classify human cancers. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Nature</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">435</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 834-838, doi:10.1038/nature03702 (2005).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">7</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Jansson, M. D. &amp; Lund, A. H.
MicroRNA and cancer. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Mol Oncol</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">6</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 590-610,
doi:10.1016/j.molonc.2012.09.006 (2012).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">8</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Schultz, N. A.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> MicroRNA biomarkers in whole blood for detection of
pancreatic cancer. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">JAMA</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">311</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 392-404,
doi:10.1001/jama.2013.284664 (2014).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">9</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Chen, X.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> Characterization of microRNAs in serum: a novel class of
biomarkers for diagnosis of cancer and other diseases. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Cell Res</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">18</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 997-1006,
doi:10.1038/cr.2008.282 (2008).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">10</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">       </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Hunt, E. A., Broyles, D., Head, T. &amp;
Deo, S. K. MicroRNA Detection: Current Technology and Research Strategies. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Annu Rev Anal Chem (Palo Alto Calif)</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">8</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 217-237,
doi:10.1146/annurev-anchem-071114-040343 (2015).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">11</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">       </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Cissell, K. A., Rahimi, Y., Shrestha,
S., Hunt, E. A. &amp; Deo, S. K. Bioluminescence-based detection of microRNA,
miR21 in breast cancer cells. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Anal Chem</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">80</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 2319-2325, doi:10.1021/ac702577a
(2008).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">12</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">       </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Venkatesan, B. M. &amp; Bashir, R.
Nanopore sensors for nucleic acid analysis. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Nature
nanotechnology</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">6</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 615-624,
doi:10.1038/nnano.2011.129 (2011).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">13</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">       </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Deng, R.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> Toehold-initiated rolling circle amplification for
visualizing individual microRNAs in situ in single cells. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Angew Chem Int Ed Engl</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">53</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">,
2389-2393, doi:10.1002/anie.201309388 (2014).</SPAN></P><P style="text-indent:-36pt;"><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">14</SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">         </SPAN><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Cabantous, S.</SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> et al.</SPAN></I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> A new protein-protein
interaction sensor based on tripartite split-GFP association. </SPAN><I><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">Scientific reports</SPAN></I><B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">3</SPAN></B><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;">, 2854, doi:10.1038/srep02854 (2013).</SPAN></P><P><SPAN style="line-height:2;font-family:Perpetua;font-size:18px;"> </SPAN></P></DIV></DIV><DIV style="border:solid 1px rgb(10,31,84);background-color:rgb(10,31,84);"><DIV class="row footer-link" style="border:solid 0px red;margin-top:100px;"><DIV id="c_menu9909"><DIV class="medium-6 columns" style="width:auto;"><H5 class="column-header">TEAM</H5><DIV class="menu-link-container"><A href="/Team:NUDT_CHINA/Team">Team</A><A href="/Team:NUDT_CHINA/Collaborations">Collaborations</A></DIV></DIV><DIV class="medium-6 columns" style="width:auto;"><H5 class="column-header">PROJECT</H5><DIV class="menu-link-container"><A href="/Team:NUDT_CHINA/Description">Description</A><A href="/Team:NUDT_CHINA/Design">Design</A><A href="/Team:NUDT_CHINA/Experiments">Experiments</A><A href="/Team:NUDT_CHINA/Proof">Proof of Concept</A><A href="/Team:NUDT_CHINA/Demonstrate">Demonstrate</A><A href="/Team:NUDT_CHINA/Results">Results</A><A href="/Team:NUDT_CHINA/Notebook">Notebook</A></DIV></DIV><DIV class="medium-6 columns" style="width:auto;"><H5 class="column-header">PARTS</H5><DIV class="menu-link-container"><A href="/Team:NUDT_CHINA/Parts">Parts</A><A href="/Team:NUDT_CHINA/Basic_Part">Basic Parts</A><A href="/Team:NUDT_CHINA/Composite_Part">Composite Parts</A><A href="/Team:NUDT_CHINA/Part_Collection">Part Collection</A></DIV></DIV><DIV class="medium-6 columns" style="width:auto;"><A href="/Team:NUDT_CHINA/Safety" style="text-decoration:none;"><H5 class="column-header">SAFETY</H5></A></DIV><DIV class="medium-6 columns" style="width:auto;"><A href="/Team:NUDT_CHINA/Attributions" style="text-decoration:none;"><H5 class="column-header">ATTRIBUTIONS</H5></A></DIV><DIV class="medium-6 columns" style="width:auto;"><H5 class="column-header">HUMAN
          PRACTICES</H5><DIV class="menu-link-container"><A href="/Team:NUDT_CHINA/Human_Practices">Human Practices</A><A href="/Team:NUDT_CHINA/HP/Silver">Silver</A><A href="/Team:NUDT_CHINA/HP/Gold">Gold</A><A href="/Team:NUDT_CHINA/Integrated_Practices">Integrated Practices</A><A href="/Team:NUDT_CHINA/Engagement">Engagement</A></DIV></DIV><DIV class="medium-6 columns" style="width:auto;"><H5 class="column-header">AWARDS</H5><DIV class="menu-link-container"><A href="/Team:NUDT_CHINA/Model">Model</A></DIV></DIV></DIV></DIV></DIV><FOOTER id="footer"><DIV class="row footer-link"><DIV class="medium-6 columns"><DIV class="content-open" style="width:300px;"><P class="intro" style="font-size:18px;line-height:30px;">No. 109 Deya Road, Kaifu District, 
Changsha, Hunan Province 410073
P.R.China

                </P></DIV></DIV><DIV style="position:relative;left:250px;top:10px;width:500px;"><H5 style="color:white;">SPONSORS</H5></DIV></DIV><DIV data-footer-end="" class="footer-end row"><DIV class="nav-utility inline-list privacy" style="width:100%;text-align:center;color:white;font-size:11px;"><SPAN>COPYRIGHT © 2016 NUDT_CHINA</SPAN></DIV></DIV></FOOTER></DIV><DIV onclick="top_it()" id="to_top">
TOP
</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>