Team:Alverno CA/Results
Toggle                 navigation
Main
Team
Our                                 Team
Collaborations
Project
Description
Design
Experiment
Protocols
Engagement
Interlab Study
Results
Notebook
Parts
Safety
Attributions
Human                         Practices
Alverno iGEM 2016
Results
GG28_1-5
GG28_1-5 [Plasmids that contain RFP and GFP coding devices with a certain number bp spacer between them on DVK_AE backbone (derived from pSB1K3)—one of them is likely 1000bp] *Please note that all parts notated as “GG” are called because they are assembled by “Golden Gate Assembly.” Each GG plasmid contains an RFP and GFP coding device in a unique orientation, with a specified spacer part between them on either DVK_AE (a backbone derived from pSB1K3) or pSB1C3.
This set of plasmids is assembled so that the RFP and GFP inducers are convergent (Pointing towards each other). The time course fluorescence traces from 1000-bp spacer constructs from our plate reader measured the fluorescence of RFP and GFP in the bacteria. In figure 1:
Figure 1
This graph shows the fluorescence of RFP within the plasmids. (D01, E01,F01, G01, and H01 are all clones of the same plasmid). As seen in the graph, there is a strange range of growth in bacteria, which was present in almost all of our results from TX-TL.
Figure 2
Figure 2 shows the plate reader’s measurement of the fluorescence of GFP in our bacteria. (D12, E12, F12, G12, H12 are all clones of the same plasmid)  This graph shows results that are similar to that of RFP, however, the overall fluorescence levels are higher than the fluorescence levels in RFP.
Figure 3
These strains have been transformed with both RFP and GFP (A1, A2, A3, A4, A5 are all clones of the same plasmid with both inducers. ) The fluorescence traces shown in this graph shows that overall the bacteria was successfully transformed, however, this data is not consistent with the data from the two previous graphs (those specific to RFP and GFP respectively).
Figure 4
Figure 5
Figure 4 shows another test of the bacteria transformed with both RFP and GFP. The first graph measures GFP fluorescence. (A1, A2, A3, A4, A5 are all clones of the same plasmid with both inducers. ) Figure 5 measures RFP fluorescence. These results show the significant difference in fluorescence between RFP and GFP in the same bacteria; GFP was more expressed compared to the expression of RFP.
While looking back on our experiment, we realized that condensation occurred in the plates that were used in the plate reader which made the Absorbance graph unreliable.
Figure 6
Figure 6 shows the measure of arbitrary fluorescence units of each well in a plate that had only water in its wells, but a lot of condensation on the top of the plate lid. The condensation could therefore have affected our results and resulted in abnormally high levels of absorbance.
Two-reporter plasmids with 500-bp spacer
GG95_102
Because we observed less condensation forming on the edge of the 96-well plate, in order to avoid condensation, we pipetted the cultures (GG95-102) around the edges of the plate. We then measured for RFP and GFP expression along with absorbance. Our results can be found here .
We found that there seemed to be less noise, with a relatively similar rate of increase in absorbance. For each GG plasmid, there were 3 clones that were obtained from the original plate and grown in cultures. These liquid cultures were then run in the plate reader. Here are the GFP expression results for each part. (GG95-102)
Two-reporter plasmids with dCas9 binding site spacers
GG105-108
We also measured the plasmids containing the dCas9 clamp binding site between the GFP coding device and RFP coding device. From this we compared the differences between the results without inducer and with inducer. We observed no expression of either GFP or RFP regardless of inducer concentration. This could possibly be because we used an invitro method (TX-TL) instead of an in vivo method. We are presently investigating this issue. Below we have images of our results:
GFP Fluorescence
(Note that the one that shows high expression is our GFP control, we only had a control for our TX-TL because it is necessary when comparing data)
RFP Fluorescence
Conclusions
Strong bimodality in our plasmids’ expression makes us suspect that we may have not actually cloned RFP and GFP together in our bacteria (they may have instead received RFP-only or GFP-only plasmids). When we attempted to sequence our plasmids, the results were very much inconclusive. We realized that when compared to our data of the transformed bacteria with just RFP or just GFP, the data more closely aligned with that.   From our sequencing results, we were unable to determine whether or not most of our plasmids were correctly made. One exception was GG105_2, a plasmid that contains a RFP coding device and a GFP coding device, both pointing forward relative to the backbone, with a clamp binding site between the two on a pSB1C3 backbone (See part: BBa_K2145100, for more details on orientation and parts present). For this plasmid we were able to determine from sequencing two main attributes of the plasmid: 1) it contains both GFP and RFP gene sequences. (These are parts BBa_J04450 and BBa_I13522) 2) the color of the colony for the plasmid is red, which means it contains RFP (see picture of gridded plate GG105-108_1-4). As a result, we were able to conclude that for this plasmid there is some kind of interference between the expression of RFP and GFP in the plasmid that shut down the production of GFP on that plasmid. We attributed this to supercoiling.
