<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Alverno_CA_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Alverno CA/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON aria-expanded="false" class="navbar-toggle collapsed" data-target="#bs-example-navbar-collapse-1" data-toggle="collapse" type="button"><SPAN class="sr-only">Toggle
                navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1" style="font-size: 12pt;"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:Alverno_CA">Main</A></LI><LI class="dropdown"><A aria-expanded="false" aria-haspopup="true" class="dropdown-toggle" data-toggle="dropdown" href="" role="button">Team</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Team">Our
                                Team</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Collaborations">
                                Collaborations</A></LI></UL><LI class="dropdown"><A aria-expanded="false" aria-haspopup="true" class="dropdown-toggle" data-toggle="dropdown" href="" role="button">Project</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Description">
                                Description</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Experiment">
                                Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Protocols">
                                Protocols</A></LI><LI><A href=" https://2016.igem.org/Team:Alverno_CA/Engagement">
                                Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/InterLab">
                                Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Notebook">
                                Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Basic_Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Human_Practices">Human
                        Practices</A></LI></DIV></DIV><TITLE>Alverno iGEM 2016</TITLE><H2><CENTER>Our Project</CENTER></H2><H3><CENTER>Utilizing spacers of a variety of lengths, and dCas9 clamp, to block propagation of supercoiling
generated during transcription</CENTER></H3><H4>Executive Summary:</H4><H5>    We propose using spacers (of a variety on lengths), or deactivated Cas9 (dCas9) as a DNA clamp, to block propagation of supercoiling generated during transcription, improving the modularity, predictability, and scalability of single-vector, multi-gene synthetic systems.</H5><H4>Project Description:</H4><H5>    Synthetic biology is best pursued with orthogonal, composable parts. For example, when a promoter is characterized, it should perform predictably no matter what gene or UTR it is composed with (at least under the same environmental conditions). This form of orthogonality is generally assumed to hold, and is the basis for most models of part characterization and composition.</H5><H5>    One context in which orthogonality is demonstrably broken is composition of multiple independent gene expression unit on the same piece of linear, genomic, or plasmid DNA. When two fully-characterized expression units (say, for example, a GFP coding sequence with promoter, RBS, and terminator, and an RFP coding sequence with promoter, RBS, and terminator) are placed next to each other on the same plasmid, their expression levels will be different than if they had been placed on different plasmids[1]. Worse, the relative orientation of the two genes dramatically affects expression. These effects are difficult to predict, so multi-gene systems typically must be optimized using time-consuming screens to achieve desired expression levels (or are not optimized at all).</H5><H5>    It is not known for certain why compositional context affects gene expression, but one promising hypothesis involves DNA supercoiling. The process of transcription torques DNA near the transcription site, introducing supercoils that can propagate down the DNA[2]. Supercoiling state, in turn, is known to affect transcription, with negative and positive supercoils making transcription more and less efficient, respectively[3]. Taken together, these effects suggest a mechanism by which genes on the same plasmid can affect each other’s expression levels[1].</H5><H5>    In our proposed project, we will attempt to isolate the expression of genes on the same plasmid (or other piece of DNA) by using spacers, or DNA binding proteins as “clamps”, to block propagation of supercoiling. We will experiment with a variety of lengths of spacers to &quot;absorb&quot;  supercoiling between genes.  Also, by placing a recognition sequence for one of several DNA binding proteins (i.e., tetR, cI, etc.) between two expressing genes (and possibly another on the opposite side of the plasmid), we hope to block the propagation of supercoils between those genes, thereby isolating their expression.</H5><H5>    As one of the stronger known DNA binding proteins, Cas9 is an ideal candidate for a DNA-binding clamp. Specifically, we will use dCas9, a version of Cas9 engineered to bind, but not cut, DNA. In addition to being a strongly-binding clamp, dCas9 has the advantage of not being used naturally as a transcription factor; unlike a naturally-occurring repressor such as tetR or lacI, dCas9 can be used to isolate contiguous gene constructs without otherwise affecting host gene expression.</H5><H5>    To test dCas9 as a clamp, we will clone several plasmids each containing two different fluorescent reporters, in different relative orientations, with several dCas9 binding sites between them. We will express these plasmids both in vivo and in an in vitro gene circuit prototyping system alongside a dCas expression plasmid and gRNA expression plasmids targeting zero or more clamp sites. We expect expression levels of the reporter proteins to vary by orientation when no gRNA is expressed, but for those differences to be at least partially diminished when one or more targeting gRNAs are expressed.</H5> 
​
<H4>Potential Applications and Implications:</H4><H5>    If successful, a dCas9-based isolating clamp, or supercoiling-absorbing spacers, could be used in any multi-gene circuit assembly, making multi-gene assemblies more predictable and their assembly much more efficient.</H5><H5>    This is particularly relevant in metabolic engineering, where circuits of many genes are routinely constructed. The physical layout of these circuits can unpredictably affect production of output by several orders of magnitude[4], so large engineered metabolic pathways must typically be hand-tuned or have many configurations screened for activity. By making gene expression more predictable, our results could greatly improve the predictability (and, therefore, designability) of large gene circuits for metabolic engineering.</H5><H4>References:</H4><H5> 1. Yeung, E. 2016. Reverse Engineering and Quantifying Context Effects in Synthetic Gene Networks [dissertation]. [Caltech (CA)]: Caltech.</H5><H5> 2. Tsao, Y., Wu, H., Liu, L. F. 1989. Transcription-Driven Supercoiling of DNA: Direct Biochemical Evidence from In Vitro Studies. Cell. 56:111-118.</H5><H5> 3. Hanafi, E.D., Bossi, L. 2000. Activation and silencing of leu-500 promoter by transcription-induced DNA supercoiling in the Salmonella chromosome. Mol Microbiol. 37(3):583-94.</H5><H5> 4. Smanski, M. J., Bhatia, S., Zhao, D., Park, Y., Woodruff, L. B. A., et al. 2014. Functional optimization of gene clusters by combinatorial design and assembly. Nature Biotechnology. 32:1241-1249.</H5></DIV></DIV></DIV></DIV></DIV></BODY></HTML>