<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Alverno_CA_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Alverno CA/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON aria-expanded="false" class="navbar-toggle collapsed" data-target="#bs-example-navbar-collapse-1" data-toggle="collapse" type="button"><SPAN class="sr-only">Toggle
                navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1" style="font-size: 12pt;"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:Alverno_CA">Main</A></LI><LI class="dropdown"><A aria-expanded="false" aria-haspopup="true" class="dropdown-toggle" data-toggle="dropdown" href="" role="button">Team</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Team">Our
                                Team</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Collaborations">
                                Collaborations</A></LI></UL><LI class="dropdown"><A aria-expanded="false" aria-haspopup="true" class="dropdown-toggle" data-toggle="dropdown" href="" role="button">Project</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Description">
                                Description</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Experiment">
                                Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Protocols">
                                Protocols</A></LI><LI><A href=" https://2016.igem.org/Team:Alverno_CA/Engagement">
                                Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/InterLab">
                                Interlab Study</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Notebook">
                                Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Basic_Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Alverno_CA/Human_Practices">Human
                        Practices</A></LI></DIV></DIV><TITLE>Alverno iGEM 2016</TITLE><H2><CENTER>Protocols</CENTER></H2><H5>*Note: Pipettes are needed.</H5><H3>Making the Agarose Gel</H3><H4>Ingredients/Materials: *</H4><H5>• 0.6g agarose</H5><H5>• 50mL TBE 1x</H5><H5>• 5μL SYBRsafe / 2.5μL EcoStain (light- and heat-sensitive)</H5><H5>• 250mL conical flask</H5><H5>• Scale</H5><H5>• Microwave</H5><H5>• Gel mold</H5><H4>Directions</H4><H5>1. Weigh 0.6g of agarose into flask.</H5><H5>2. Add 50mL of TBE 1x Buffer</H5><H5>3. Microwave agarose solution until dissolved for 1 min (take out halfway to swirl)</H5><H5>4. (If the liquid looks distorted, agarose solution needs to be microwaved more)</H5><H5>5. Cool slightly and add SYBRsafe or EcoStain</H5><H5>6. Pour gel into mold and put the combs into their spots</H5><H5>7. Cool until solidified.</H5><H3>Gel Electrophoresis</H3><H4>Ingredients/Materials: *</H4><H5>• TBE 1x Buffer</H5><H5>• Agarose gel (w/ right number of wells)</H5><H5>• DNA reaction (either parts=PUCTV, or PCR check for GG plasmids)</H5><H5>• Purple loading dye</H5><H5>• Gel box</H5><H4>Directions</H4><H5>1. Place the agarose gel into gel box with wells on the negative side.</H5><H5>2. Fill up gel box up with TBE 1x Buffer up to the line or at least above the gel.</H5><H5>3. Pipette 6ul of 2-log ladder on side lanes (or given wells).</H5><H5>4. Pipette 1ul of purple loading dye on parafilm. Then pipette 5ul of DNA reaction and mix with loading dye on parafilm. (*For PCR Check: Directly pipette 2ul of purple loading dye into 10ul of PCR Check Reaction in microfuge tube)</H5><H5>5. Set to 6ul and pipette 6ul of mixed reaction with loading dye into selected wells according to drawn diagram.</H5><H5>6. Set machine to 175V and for 20 minutes. Make sure bubbles are appearing!</H5><H5>7. After running place gel onto transilluminator (must be off) in the glove box. Place box over it with the hole and then place iPad above it and set to either video or time-lapse. Then turn on transilluminator within closed glove box while videotaping.</H5><H5>8. Results can now be analyzed.</H5><H3>PCR (Polymerase Chain Reaction) for Parts</H3><H4> Ingredients</H4><H5>• 2.5μL Part Forward Primer</H5><H5>• 2.5μL Part Reverse Primer</H5><H5>• 0.1μL G-block / DNA template</H5><H5>• 25μL Q5 2x High-Fidelity MasterMix</H5><H5>• 19.9μL NFW (nuclease free water)</H5><H4>Materials: *</H4><H5>• Centrifuge</H5><H5>• Thermocycler</H5><H5>• Mini microfuge PCR tube(s)</H5><H4>Directions:</H4><H5>1. Mix above ingredients listed together in microfuge PCR tube. (Notice to add MasterMix last.)</H5><H5>2. Spin in the centrifuge.</H5><H5>3. Put in the thermocycler. Process it in thermocycler as follows</H5><H5>-- Step 1: 98°C for 30 sec</H5><H5>-- Step 2: 98°C for 10 sec</H5><H5>-- Step 3: 70°C for 20 sec</H5><H5>-- Step 4: 72°C for 20-30sec/kilobase (typically)</H5><H5>-- Step 5: Enter “Go To” and then Step 2 and repeat for 25 cycles (or “times”)</H5><H5>-- Step 6: 72°C for 2 min</H5><H5>-- Step 7: 4°C for ∞ (Set to 00:00:00)</H5><H5>-- Step 8: End</H5><H3>PCR Purification of DNA</H3><H4>Ingredients/Materials: *</H4><H5>• PCR reaction(s) (DNA Part(s))</H5><H5>• Molecular Biology Kit, which includes:</H5><H5>• Buffer B3 (with pre-added isopropyl alcohol)</H5><H5>• Wash Solution (with pre-added ethanol)</H5><H5>• Elution Buffer</H5><H5>• EZ-10 column(s)</H5><H5>• 1.5mL microfuge tube(s)</H5><H4>Directions: (for each PCR reaction)</H4><H5>1. Transfer the PCR reaction mixture (usually 50ul, ranges from 35-50ul) to a 1.5mL microfuge tube and add 5 volumes (5 x amount of PCR reaction mixture) of Buffer B3 (with pre-added isopropyl alcohol).</H5><H5>2. Transfer above mixture to EZ-10 column and leave at room temperature for 2 minutes. Centrifuge at 10,000rpm for 2 minutes.</H5><H5>3. Remove/empty flow-through in bottom tube. Add 750ul of Wash Solution (with pre-added ethanol) and centrifuge at 10,000rpm for 2 minutes.</H5><H5>4. Repeat washing procedure (from Step 3, “Add 750ul of…”). Remove/empty flow-through again. Spin at 10,000rpm for an additional minute. Throw away bottom clear tube with any remaining liquid.</H5><H5>5. Place top tube with white filter into clean 1.5mL microfuge tube. Check for ethanol using pipette tip.</H5><H5>6. Add 30-50uL (usually 40uL) of Elution Buffer to the center of the tube. Incubate at room temperature for 2 minutes.</H5><H5>7. Centrifuge at 10,000rpm for 2 minutes to elute DNA.</H5><H5>8. Store at -20 degrees Celsius, or nanodrop for concentration and for dilutions (see Parts Dilutions Protocol).</H5><H3>Parts Dilutions</H3><H4>Ingredients/Materials: *</H4><H5>• NFW</H5><H5>• PCR Purified DNA Part Reaction (nanodropped with concentration)</H5><H5>• 1.5 mL microfuge tube</H5><H4>Directions</H4><H5>1. Identify number of bases and the concentration (in ng/uL, which is basically ug/mL).</H5><H5>2. Plug in numbers (bases and concentration) into Promega Biomath Calculator to convert from ug/mL (or ng/uL) to pmol/uL (http://www.promega.com/a/apps/biomath/index.html?calc=ugmlpmolul).</H5><H5>3. Multiply resulting number by 1000 and that is the concentration in nM.</H5><H5>4. Plug into dilution equation: C1*V1=(30nM)(V2), where C1 is the concentration in nM, and V2 is equal to the amount wanted (typically 10uL-20uL). Then solve for V1.</H5><H5>5. Put in the amount of V1 of selected Part in 1.5mL microfuge tube.</H5><H5>6. Subtract V1 from V2. Put this amount of NFW into the tube.</H5><H5>7. Centrifuge.</H5><H5>8. Store at -20 degrees Celsius.</H5><H3>Golden Gate Assembly Assembly Protocol</H3><H5>(Example with Golden Gate Assembly Protocol for GG37-52; multiple GG Assembly for Plasmids can be done at a time in different tubes as seen in example)</H5><H4>Ingredients (per Golden Gate Assembly)</H4><H5>• 1μL: P part (i.e. P1a, P2a, P3a, P4a)</H5><H5>•1μL: UC part (i.e. UC1a, UC2a, …, UC8a, etc.)</H5><H5>• 1μL: T part (i.e. T1a, T2a, T3a, T4a)</H5><H5>• 1μL: V part (i.e. V19d, V1a, V2a, etc.)</H5><H5>• 1μL: GFP Mut Parts: P1ab, P2ab, T3ab, T4ab (Note: for all parts with GFP, matches according to Part, see combos image for example; some do not have GFP Mut parts and so add 1ul to NFW amount instead) </H5><H5>• 1.5μL: T4 Ligase Buffer</H5><H5>• 0.15μL: 100x BSA Standard</H5><H5>• 1μL: BsaI</H5><H5>• 2μL: T4 Ligase (2M cohesive units)</H5><H5>• 5.35μL NFW</H5><H4>Materials: *</H4><H5>• Centrifuge</H5><H5>• Thermocycler</H5><H5>• Mini microfuge tube(s)</H5><H4>Directions</H4><H5>1. Write down combos of plasmid(s)</H5><H5>2. Mix above ingredients in labeled mini microfuge tube(s).</H5><H5>3. Spin down in the centrifuge.</H5><H5>4. Put in thermocycler, process it in thermocycler as follows:</H5><H5>-- Step 1: 37°C for 3 min</H5><H5>-- Step 2: 16°C for 4 min</H5><H5>-- Step 3: Go to Step 1 and repeat for 25 cycles</H5><H5>-- Step 4: 50°C for 5 min</H5><H5>-- Step 5: 80°C for 5 min</H5><H5>-- Step 6: 4°C for ∞ (Set to 00:00:00)</H5><H5>-- Step 7: End</H5><H3>PCR Check for Golden Gate Plasmids</H3><H5>(multiple can be done at a time in different tubes)</H5><H4>Ingredients</H4><H5>• 0.5μL V part forward sequencing primer</H5><H5>• 0.5μL V part reverse sequencing primer</H5><H5>• 0.1μL GG Assembly</H5><H5>• 5μL 2x MasterMix</H5><H5>• 3.9μL NFW</H5><H4>Materials *</H4><H5>• Centrifuge</H5><H5>• Thermocycler</H5><H5>• Mini microfuge tube(s)</H5><H4>Directions</H4><H5>1. Mix above ingredients in mini microfuge tube(s).</H5><H5>2. Spin in the centrifuge.</H5><H5>3. Put it in thermocycler, process it in thermocycler as follows</H5><H5>Step 1: 98°C for 30 sec</H5><H5>Step 2: 98°C for 10 sec</H5><H5>Step 3: 56°C for 20 sec</H5><H5>Step 4: 72°C for 20-30sec/kilobase (typically)</H5><H5>Step 5: Enter “Go To” and then Step 2 and repeat for 25 cycles (or “times”)</H5><H5>Step 6: 72°C for 2 min</H5><H5>Step 7: 4°C for ∞ (Set to 00:00:00)</H5><H5>Step 8: End</H5><H3>Making LB Media (&amp; Autoclaving)</H3><H4>Ingredients</H4><H5>• LB powder</H5><H5>• Distilled water</H5><H4>Materials</H4><H5>• Scale</H5><H5>• Glass bottle</H5><H5>• Autoclave</H5><H4>Directions</H4><H5>1. Measure out 5g LB powder.</H5><H5>2. Add LB powder to the bottle.</H5><H5>3. Add 250mL distilled water to bottle.</H5><H5>4. Autoclave for ~30 min.</H5><H4>To Autoclave</H4><H5>1. Fill autoclave with water until it covers metal ring.</H5><H5>2. Place the metal bucket in the autoclave. Put in the metal dish. Place bottle (or item(s) to autoclave) at center.</H5><H5>3. Put the lid on with the dangly metal cord in place.&lt;</H5><H5>4. Bolt on bolts tightly to seal the lid.</H5><H5>5. Open both manual release valves.</H5><H5>6. Plug in and turn on the switch.</H5><H5>7. Wait until steam comes out of the valve and wait for 5 more minutes.</H5><H5>8. Close both valves and wait for the temperature to rise.</H5><H5>9. Switch off the autoclave before it reaches red at 0.15.</H5><H5>10. Maintain between 0.1 and 0.5 for 30 min.</H5><H5>11. After complete, switch off and wait until pressure goes down back to 0.</H5><H5>12. Open manual valves and release steam.</H5><H3>Making LB Media w/ Antibiotic Resistance for Plates</H3><H4>Ingredients</H4><H5>• 300ml H2O</H5><H5>• 6g LB Powder</H5><H5>• 4.5g Agar Powder</H5><H5>• Antibiotic (usually use Kanamycin - 1μL per 1mL H2O)</H5><H4>Materials: *</H4><H5>• Autoclave</H5><H5>• Scale</H5><H5>• Glass bottle</H5><H4>Directions</H4><H5>1. Measure out the LB and Agar. Pour into a glass bottle and then pour in distilled water up to 300mL line.</H5><H5>2. Mix the above ingredients.</H5><H5>3. Autoclave for 30 minutes.</H5><H5>4. Cool down to around 50°C (~122°F).</H5><H5>5. Add 300μL Kanamycin (or chosen antibiotic, added accordingly).</H5><H5>6. Open the lid to each plate carefully and pour plate near the flame.</H5><H3>Bacterial Transformation of Plasmids (&amp; Growing Liquid Cultures)</H3><H4>Ingredients/Materials: *</H4><H5>• DNA GG Plasmid Mixture</H5><H5>• Competent Cells</H5><H5>• SOC Media (or LB Media if SOC is contaminated…)</H5><H5>• 2ml Microtubes</H5><H5>• Tube Rack</H5><H5>• Ice</H5><H5>• Timer</H5><H5>• 42°C Water Bath (set early on!)</H5><H5>• 37°C Incubator (set early on!)</H5><H5>• Petri Plates with LB agar and antibiotic</H5><H5>• Sterile Spreader or sterile glass beads</H5><H4>Directions</H4><H5>1. Thaw competent cells on ice. (If delicate, take out after step 3, and do step 2 after step 3)</H5><H5>2. Pipette 20μL of competent cells into 2mL tube.</H5><H5>3. Pipette 1μL of control DNA into 2mL tube.</H5><H5>4. Pipette 1μL of resuspended DNA into 2mL tube.</H5><H5>5. Close 2ml tubes and incubate on ice for 30 minutes.</H5><H5>6. Heat shock tubes at 42°C for 1 minute.</H5><H5>7. Incubate on ice for 5 minutes. </H5><H5>8. Pipette 200μL SOC media (or LB media) to each transformation.</H5><H5>9. Incubate at 37°C for 2 hours—in incubator.</H5><H5>10. Pipette each transformation on petri plates (labelled!).</H5><H5>11. Incubate transformations overnight (14-18 hours) at 37°C.</H5><H4>The Next Day</H4><H5>1. Pick single colonies.</H5><H5>2. Transfer each single colony to a gridded plate (labeled), dip pipette tip into PCR reaction to do a colony PCR to verify part size, then place tip into liquid culture to grow up liquid cell cultures.</H5><H5>3. To do Colony PCR, create mixture according to PCR Check for Golden Gate Plasmids Protocol without 0.1uL of GG Assembly (or DNA Plasmid).</H5><H5>4. Put in the MasterMix as well. After picking colony with pipette tip dip into reaction mixture and then put PCR reaction into the thermocycler and continue PCR protocol.</H5><H5>5. Incubate gridded plate and the liquid cultures at 37°C overnight. Take out the next morning and store in the refrigerator (4°C).</H5><H3>Plate Reading (for Fluorescence, Absorbance, Induction, etc.)</H3><H4>Ingredients/Materials *</H4><H5>• Liquid cultures</H5><H5>• 96 well plate (A-H by 1-12)</H5><H5>• Plate Reader (we use VICTOR X3)</H5><H5>• LB Media</H5><H4>Directions</H4><H5>1. Pipette in 100uL per well for each liquid culture.</H5><H5>2. Place plate into the reader.</H5><H5>3. Set protocol to absorbance and hit run (green arrow button).</H5><H5>4. Measure Absorbance at 600nm for all samples in all standard measurement modes in the plate reader.</H5><H5>5. Import data into Excel Sheet after the run is finished.</H5><H5>6. Use Normalization sheet tab and copy over.</H5><H5>7. Enter data accordingly.</H5><H5>8. Record the data, specifically volume of preloading culture and preloading media from the table in the notebook.</H5><H5>9. Dilute accordingly (media is LB media with antibiotic) in a new 96 well plate (if needed). Usually about 500uL per well.</H5><H5>10. Place plate back into reader and set to OD-RFP-GFP protocol (made in plate reader) and start run.</H5><H5>11. Let it run overnight (120 runs total with about 3 minute intervals) and check in the morning. </H5><H5>12. Import data results into Excel spreadsheet.</H5><H5>13. Optional: Upload to Google drive. </H5><H5>14. To analyze data using Python program—file must be in a csv format. (If you would like the code for analyzing this type of data, please contact us!)</H5><H3>Purification of Plasmid DNA</H3><H4>Ingredients/Materials *</H4><H5>• Liquid cultures</H5><H5>• Miniprep DNA kit (we used BioBasic kit)</H5><H4>Directions</H4><H5>1. Add 1.5 mL-5mL overnight culture in the tube and centrifuge at 12,000 rpm for 2 mins. Drain liquid completely. </H5><H5>2. Add 100μL of solution I to pellet, mix well, and keep for 1 minute. </H5><H5>3. Add 1μL of visullyse to the mixture above. </H5><H5>     *Note: Addition of visullyse is an optional step. </H5><H5>4. Add 200μL of solution II to the mixture, and mix gently by inverting the tube 4-6 times and then keep at room temperature for 1 min. To prevent contamination from genomic DNA, do not vortex. If visuallyse has been added the solution will do not vortex. If visuallyse has been added the solution will turn blue after addition of solution II. A homogeneously blue suspension should also be observed. If the suspension contains uneven blue color, or white/brownish cell clumps, continue mixing carefully. </H5><H5>5. Add 350μL of solution III, and mix gently. Incubate at room temperature for 1 minute. A fluffy white material forms and lysate should become less viscous. If visualyse has been added in step 3, the suspension should be mixed until all traces of blue has gone and lysate becomes colorless. </H5><H5>6. Centrifuge at 12,000 rpm for 5 minutes. </H5><H5>7. Transfer above supernatant (step 6) to the EZ-10 column: centrifuge at 10,000 for 2 minutes.</H5><H5>8. Discard flow-through in the tube. Add 750μL of wash solution to the column, and centrifuge at 10,000rpm for 2 minutes. </H5><H5>9. Repeat wash procedure in step 8. </H5><H5>10. Discard the flow-through in the collection tube. Centrifuge at 10,000rpm for an additional minute to remove any residual wash solution. </H5><H5>11. Transfer the column to a clean 1.5mL microfuge tube. Add 50μL of Elution Buffer into the center part of the column and incubate at room temperature for 2 minutes. Centrifuge at 10,000rpm for 2 minutes. </H5><H5>12. Store purified DNA at -20°C.</H5><H4>For centrifuging culture to pellets:</H4><H5>1. In a 1.5mL tube pipette 1000μL liquid culture. </H5><H5>2. Spin at max speed for 30 sec.</H5><H5>3. Pour off extra media.</H5><H5>4. Pipette 500μL same liquid culture.</H5><H5>5. Pour off rest.</H5><H5>6. Freeze.</H5><H3>Plate Reading (for Fluorescence using TX-TL for GG105-108 w/ dcas9 expression plasmids)</H3>Ingredients/Materials: *<H5>• TX-TL Buffer</H5><H5>• TX-TL Extract</H5><H5>• Setup Spreadsheet w/ entered values (for clean spreadsheet: http://www.jove.com/files/ftp_upload/50762/TXTL_JoVE.xlsx)</H5><H5>• NFW</H5><H5>• dcas9 expression plasmids</H5><H5>• gRNA plasmids</H5><H5>• GG reaction plasmids w/ clamp binding sites (in our case, GG105-108)</H5><H5>• 384-well plate</H5><H4>Directions: </H4><H5>1. Dilute gRNA plasmids accordingly by concentration to setup values in spreadsheet in PCR tube strip. </H5><H5>2. Dilute GG105-108 plasmids accordingly by concentration to setup values in spreadsheet in PCR tube strip. </H5><H5>3. Pipette diluted DNA amount (uL) and water (uL) accordingly in second strip. </H5><H5>4. Make Master Mix (MM): </H5><H5>-- a) amount of TX-TL Buffer (uL) </H5><H5>-- b) amount of TX-TL Extract (uL) </H5><H5>-- c) amount of dcas9 expression plasmid (uL) —place values in blue grid section in top left corner for calculations </H5><H5>5. Add amount given of MM (uL) to each tube with a GG plasmid and a gRNA plasmid. </H5><H5>6. Pipette 10uL of each reaction from tube to each well in the plate. </H5><H5>7. Place plate in plate reader and set to correct protocol (created protocol for us: iGEM_TX-TL_GFP_RFP) to measure 384-well plate and run plate reader. </H5><H5>8. Let it run overnight (120 runs total with about 3 minute intervals) and check in the morning. </H5><H5>9. Import data results into Excel spreadsheet. Optional: Upload to Google Drive. </H5><H5>10. To analyze data using Python program—file must be in csv format. (If you would like the code for analyzing this type of data, please contact us!) </H5><H5> For more information on TX-TL, see: http://www.jove.com/video/50762/protocols-for-implementing-an-escherichia-coli-based-tx-tl-cell-free</H5><H3>For any questions about Protocols, email: alverno.igem@gmail.com</H3></DIV></DIV></DIV></DIV></DIV></BODY></HTML>