<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV lang="en" id="main" class="container-fluid"><NAV id="navbar" class="navbar navbar-expand-lg navbar-light bg-light"><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="navbar-nav mr-auto mt-2 mt-lg-0"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Hamburg">Home<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Hamburg/Team#main-content" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Team
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Team#main-content">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Collaborations#main-content">Collaborations</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Hamburg/Description#main-content" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Project
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Description#main-content">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Design#main-content">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Experiments#main-content">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Notebook#main-content">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/InterLab#main-content">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Model#main-content">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Results#main-content">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Demonstrate#main-content">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Improve#main-content">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Chemistry#main-content">Chemistry</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Attributions#main-content">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Parts
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Parts#main-content">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Basic_Part#main-content">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Composite_Part#main-content">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Part_Collection#main-content">Part Collection</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:Hamburg/Safety#main-content">Safety</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:Hamburg/Human_Practices#main-content">
                                    Human Practices
                                </A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Awards
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Entrepreneurship#main-content">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Hardware#main-content">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Model#main-content">Model</A></DIV></LI></UL></DIV></NAV><DIV id="main-content" class="container bg-light"><H1>
                        Modeling
                    </H1><DIV class="jumbotron"><H2>Achievements</H2><UL><LI>
                                We built a new model for protein synthesis including DNA, RNA and protein species
                            </LI><LI>
                                We predicted performance of our Basic Part <A href="http://parts.igem.org/Part:BBa_K2588004">BBa_K2588004</A> in a generic circuit setup
                            </LI><LI>
                                We based our choice of a constitutive promoter for our NOT-gate of MlcRE, <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>, on insights gained from this model
                            </LI></UL></DIV><P>
                        Modelling in synthetic biology serves the purpose to predict functions and
                        behaviours that are difficult or impossible to characterize experimentally,
                        and to drive design decisions that otherwise would take much time to gather
                        information on. In our project, we implemented modelling for both aspects.
                    </P><P><I>Shigella flexneri</I> icsA is coding for an outer membrane protein and virulence factor, and is
                        highly regulated by a unique mechanism involving a short noncoding RNA
                        (ncRNA) which is complementary to icsA 5’ untranslated region (5’ UTR),
                        called RnaG120 hereafter. When binding to nascent icsA mRNA, RnaG120
                        induces stem loop formation, leading to termination of transcription before
                        icsA coding sequence is transcribed<A id="page-reference-1" href="#reference-1" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Tran, C. N., Giangrossi, M., Prosseda, G., Brandi, A., Di Martino, M. L., Colonna, B. &amp; Falconi, M. A multifactor regulatory circuit involving H-NS, VirF and an antisense RNA modulates transcription of the virulence gene icsA of Shigella flexneri. &lt;span class='journal'&gt;Nucleic Acids Res.&lt;/span&gt; &lt;span class='volume'&gt;39,&lt;/span&gt; 8122–34 (2011).">1</A>.
                    </P><P>
                        While RnaG120 was studied extensively <I>in situ</I>, to our knowledge it
                        has never been used in an engineering context. We identified the vast
                        potential of RnaG120 as a compact, extremely energy efficient NOT-gate.
                        Assuming that binding of RnaG120 to icsA 5’ UTR is sufficient to induce
                        stem loop formation and termination of transcription, expression of any
                        gene cloned downstream of icsA 5’ UTR could be inhibited by expression of
                        RnaG120. Based on this assumption, we created <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> as a proof of
                        concept of a general RnaG120-based NOT gate.
                    </P><H2><A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> – An RnaG120-based NOT gate of glucose-inhibited promoter
                        MlcRE
                    </H2><P>
                        To build iGEM registry’s first ever transcription regulator that is induced
                        by glucose, we employed RnaG120 to create a NOT-gate of glucose-inhibited
                        promoter MlcRE (<A href="http://parts.igem.org/Part:BBa_K2588000">BBa_K2588000</A>). It is comprised of multiple subparts, in 5’
                        to 3’ order: <A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> terminates transcription of RnaG120. A constitutive
                        promoter provides constant gene of interest (GOI) mRNA transcription. icsA
                        5’ UTR, which is complementary to RnaG120, acts as 5’ UTR for any GOI
                        downstream of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>. MlcRE is cloned in opposite reading direction,
                        downstream of icsA 5’ UTR, activating transcription of RnaG120. Downstream
                        of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>, any gene of interest could be placed (Fig. 1).
                    </P><FIGURE><FIGCAPTION><B>Fig. 1 Map of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>. </B>Parts in order are: terminator <A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>, a
                            constitutive promoter, icsA 5’ UTR <A href="http://parts.igem.org/Part:BBa_K2588004">BBa_K2588004</A>, MlcRE <A href="http://parts.igem.org/Part:BBa_K2588000">BBa_K2588000</A> (coded
                            backwards). Note that only one DNA molecule is depicted, but annotated in
                            two rows to accommodate for RnaG120/icsA operator, which has one function
                            in each reading direction.
                        </FIGCAPTION></FIGURE><H2>
                        The Objective
                    </H2><P>
                        We assume that the performance of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> heavily relies on the ratio
                        of transcribed mRNA and present RnaG120 depending on current induction. The
                        easiest way to optimize this ratio is to select a constitutive promoter
                        based on its strength relative to MlcRE. The Anderson collection offers a
                        wide selection of constitutive promoters which have been characterized
                        extensively. Testing all of them to select the one best suited for our
                        application would be extremely laborious. Therefore, we decided on building
                        a model that is able to predict the GOI expression based on constitutive
                        promoter strength and glucose input.
                    </P><P>
                        Our aim was to select the Anderson promoter which provides the best GOI
                        expression signal to noise ratio.
               </P><H2>
                        Building a model of RNA/RNA interactions using iBioSim
                    </H2><P>
                        iBioSim is a java-based desktop application designed to create and analyse
                        genetic circuit models<A id="page-reference-2" href="#reference-2" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Watanabe, L., Nguyen, T., Zhang, M., Zundel, Z., Zhang, Z., Madsen, C., Roehner, N. &amp; Myers, C. iBioSim 3: A Tool for Model-Based Genetic Circuit Design. &lt;span class='journal'&gt;ACS Synth. Biol.&lt;/span&gt; acssynbio.8b00078 (2018). doi:10.1021/acssynbio.8b00078">2</A>. Models built with iBioSim simulate cellular agents
                        commonly used to predict kinetics of genetic circuits. Classic genetic
                        circuits are built with regulated promoters inducing production of
                        proteins. While iBioSim computes both transcription and translation using
                        internal mRNA, it does not support manual editing regulatory functions of
                        mRNA. <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> is a genetic circuit that performs logic purely on RNA
                        level. Therefore, iBioSim does not support models of our application of the
                        shelf.
                    </P><P>
                        As a workaround, we built our model to simulate both mRNA and RnaG120 as
                        protein-level species behaving like RNA by setting stoichiometry of
                        production, which usually is used to model the number of proteins produced
                        from one mRNA molecule before its degradation, to 1. To model translation,
                        icsA mRNA was set up to activate an implicit promoter acting as ribosome
                        binding site to induce translation of a GOI. The transcription-terminating
                        function of RnaG120 was modelled as complex formation of RnaG120 with icsA
                        mRNA. The resulting complex would not activate translation of GOI,
                        accounting for transcription termination before transcription of GOI coding
                        sequence (Fig. 2).
                    </P><FIGURE><FIGCAPTION><B>Fig. 2 Setup for kinetic model of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>.</B> MlcRE was set up as
                            activateable promoter with an activated RNAP binding equilibrium smaller
                            than its basal RNAP binding equilibrium to account for the repressing
                            function of glucose. Constitutive promoter and MlcRE induce transcription
                            of GOI mRNA and RnaG120, respectively. GOI mRNA induces translation of GOI.
                            RnaG120 and GOI mRNA undergo complex formation. The resulting complex does
                            not induce translation of GOI.
                        </FIGCAPTION></FIGURE><P>
                        Despite being a workaround, this model should kinetically perform exactly
                        like the real construct, given that correct values are used for the
                        relevant parameters of both constitutive promoter and MlcRE.
                    </P><P>
                        Genetic circuit models rely on many measured parameters to be accurate. We
                        previously characterized MlcRE in <A href="http://parts.igem.org/Part:BBa_K2588039">BBa_K2588039</A>, comparing MlcRE-induced GFP
                        expression to InterLab positive control <A href="http://parts.igem.org/Part:BBa_I20270">BBa_I20270</A>, in which strong
                        Anderson promoter <A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A> drives GFP expression. Using this data,
                        relative strengths of all Anderson promoters compared to MlcRE were
                        computed and used as input values for the model. RnaG120 was extensively
                        characterized by Tran <EM>et al</EM>., including binding behaviour of
                        RnaG120 to icsA 5’ UTR. All model input values are summarized in Table 1.
                    </P><H2>Table 1: User-set values of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> kinetic model</H2><TABLE width="0" cellspacing="0" cellpadding="0" border="0"><TBODY><TR><TD width="97" nowrap=""><P align="left"><STRONG>Species</STRONG></P></TD><TD width="142" valign="bottom" nowrap=""><P align="left"><STRONG>Parameter</STRONG></P></TD><TD width="71" valign="bottom" nowrap=""><P align="left"><STRONG>Value</STRONG></P></TD><TD width="295" valign="bottom" nowrap=""><P align="left"><STRONG>Reference</STRONG></P></TD></TR><TR><TD width="97" nowrap=""><P align="left">
                                        Glucose
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Degradation rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.0075
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Natarajan <EM>et al</EM><A id="page-reference-3" href="#reference-3" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Natarajan, A. &amp; Srienc, F. Glucose uptake rates of single E. coli cells grown in glucose-limited chemostat cultures. &lt;span class='journal'&gt;J. Microbiol. Methods&lt;/span&gt; &lt;span class='volume'&gt;42,&lt;/span&gt; 87–96 (2000).">3</A></P></TD></TR><TR><TD width="97" nowrap=""><P align="left">
                                        RnaG120
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Degradation rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">      0.068
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Selinger <EM>et al</EM><A id="page-reference-4" href="#reference-4" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Selinger, D. W., Saxena, R. M., Cheung, K. J., Church, G. M. &amp; Rosenow, C. Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation. &lt;span class='journal'&gt;Genome Res.&lt;/span&gt; &lt;span class='volume'&gt;13,&lt;/span&gt; 216–23 (2003).">4</A></P></TD></TR><TR><TD width="97" nowrap=""><P align="left">
                                        GOI mRNA
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Degradation rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.068
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Selinger <EM>et al</EM><A id="page-reference-99" href="#reference-4" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Selinger, D. W., Saxena, R. M., Cheung, K. J., Church, G. M. &amp; Rosenow, C. Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation. &lt;span class='journal'&gt;Genome Res.&lt;/span&gt; &lt;span class='volume'&gt;13,&lt;/span&gt; 216–23 (2003).">4</A></P></TD></TR><TR><TD rowspan="2" width="97" nowrap=""><P align="left">
                                        RnaG120/GOI mRNA complex
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Degradation rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.068
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Selinger <EM>et al</EM><A id="page-reference-100" href="#reference-4" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Selinger, D. W., Saxena, R. M., Cheung, K. J., Church, G. M. &amp; Rosenow, C. Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation. &lt;span class='journal'&gt;Genome Res.&lt;/span&gt; &lt;span class='volume'&gt;13,&lt;/span&gt; 216–23 (2003).">4</A></P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Complex formation equilibrium
                          </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.7
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Tran <I>et al</I><A href="#reference-1" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Tran, C. N., Giangrossi, M., Prosseda, G., Brandi, A., Di Martino, M. L., Colonna, B. &amp; Falconi, M. A multifactor regulatory circuit involving H-NS, VirF and an antisense RNA modulates transcription of the virulence gene icsA of Shigella flexneri. &lt;span class='journal'&gt;Nucleic Acids Res.&lt;/span&gt; &lt;span class='volume'&gt;39,&lt;/span&gt; 8122–34 (2011).">1</A></P></TD></TR><TR><TD width="97" nowrap=""><P align="left">
                                        GOI
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Degradation rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.012
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Set arbitrarily
               </P></TD></TR><TR><TD width="97" nowrap=""><P align="left"><STRONG>Promoters</STRONG></P></TD></TR><TR><TD rowspan="3" width="97" nowrap=""><P align="left">
                                        MlcRE
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Basal production rate
               </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.81
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        MlcRE characterization
                                    </P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Activated production rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        0.19
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        MlcRE characterization
                                    </P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Stoichiometry of production
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        1
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        model-intrinsic
                                    </P></TD></TR><TR><TD rowspan="4" width="97" nowrap=""><P align="left">
                                        Constitutive Promoter
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Basal production rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        1
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left"><A href="http://parts.igem.org/Part:BBa_I20270">BBa_I20270</A> compared to MlcRE in characterization<A id="page-reference-5" href="#reference-5" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Part:BBa I20270 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_I20270. (Accessed: 15th October 2018)">5</A></P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Activated production rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        1
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left"><A href="http://parts.igem.org/Part:BBa_I20270">BBa_I20270</A> compared to MlcRE in characterization<A id="page-reference-5" href="#reference-5" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Part:BBa I20270 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_I20270. (Accessed: 15th October 2018)">5</A></P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Open complex productio rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        varying (0.2-2)
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Anderson promoter family<A id="page-reference-6" href="#reference-6" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Part:BBa J23100 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_J23100. (Accessed: 15th October 2018)">6</A></P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Stoichiometry of production
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        1
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        model-intrinsic
                                    </P></TD></TR><TR><TD rowspan="2" width="97" nowrap=""><P align="left">
                                        RBS
                                    </P></TD><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Activated production rate
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        1
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        model-intrinsic
                                    </P></TD></TR><TR><TD width="142" valign="bottom" nowrap=""><P align="left">
                                        Stoichiometry of production
                                    </P></TD><TD width="71" valign="bottom" nowrap=""><P align="right">
                                        20
                                    </P></TD><TD width="295" valign="bottom" nowrap=""><P align="left">
                                        Set arbitrarily
                                    </P></TD></TR></TBODY></TABLE><H2>
                        Results
                    </H2><P><A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> was modelled using a hierarchical Runge-Kutta-Fehlberg
                        simulation<A id="page-reference-7" href="#reference-7" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Fehlberg, E. Klassische Runge-Kutta-Formeln vierter und niedrigerer Ordnung mit Schrittweiten-Kontrolle und ihre Anwendung auf Wärmeleitungsprobleme. &lt;span class='journal'&gt;Computing&lt;/span&gt; &lt;span class='volume'&gt;6,&lt;/span&gt; 61–71 (1970).">7</A>. The simulation was run for 1000 time units, with addition of
                        glucose after 300 time units. Glucose was set up to be degraded over time
                        to simulate consumption. Levels of RnaG120, GOI, and RnaG120-mRNA complex
                        was monitored over time (Fig. 3A). The model predicts basal expression of
                        RnaG120 and GOI, as well as complex formation of RnaG120 with GOI mRNA. Due
                        to the RnaG120/GOI mRNA binding constant, RnaG120 is predicted to not fully
                        bind to GOI mRNA. Upon glucose addition, RnaG120 and RnaG120-GOI mRNA
                        levels are predicted to decline before returning to normal upon glucose
                        consumption. Consequently, GOI translation from free GOI mRNA is predicted
                        to raise GOI levels. Since GOI protein stability is set arbitrarily, it
                        declines over time.
                    </P><P>
                        Consecutively, constitutive promoter strength was varied between 0.2 times
                        measured MlcRE strength to 2 times measured MlcRE strength. Predicted
                        signal to noise ratio was calculated as the ratio of basal GOI level before
                        addition of glucose and peak GOI level after induction (Fig. 3B). A
                        constitutive promoter with roughly half the strength of MlcRE without
                        glucose was predicted to be optimal for <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>.
                    </P><FIGURE><FIGCAPTION><B>Fig. 3 Results of <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A> model. A: </B>Predicted RnaG120, GOI, and
                            RnaG120-GOI mRNA complex levels over time, simulated using a hierarchical
                            Runge-Kutta-Fehlberg simulation. After addition of glucose, RnaG120 and
                            RnaG120-GOI mRNA complex are depleted, and GOI level is raised before
                            returning to basal level after depletion of glucose. <B>B:</B> Predicted signal to
                            noise ratio of GOI expression depending on constitutive promoter strength
                            relative to basal MlcRE.
                        </FIGCAPTION></FIGURE><H2>
                        Conclusions
                    </H2><P>
                        A constitutive promoter inducing transcription half as strongly as MlcRE
                        without glucose repression was predicted to be optimal for use in
                        <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>. Consequently, we chose <A href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A> as constitutive promoter.
                    </P><P>
                        Our sponsor Integrated DNA Technologies kindly synthesized <A href="http://parts.igem.org/Part:BBa_K2588002">BBa_K2588002</A>. We
                        assembled it in pSB1C3 as basic part, however we were unable to assemble it with a GFP reporter. Therefore, we could not experimentally verify this model. We are positive that this part would have a tremendous value for future synthetic biology applications, and encourage every team to look further into it.</P><H2>
                        References
                    </H2><OL class="reference-list"><LI id="reference-1"><A href="#page-reference-1">↑</A><A target="_blank" href="https://doi.org/10.1093/nar/gkr521">Tran, C. N. et al. A multifactor regulatory circuit involving H-NS, VirF and an antisense RNA modulates transcription of the virulence gene icsA of Shigella flexneri. <I>Nucleic Acids Res.</I><B>39</B>, 8122–34 (2011).</A></LI><LI id="reference-2"><A href="#page-reference-2">↑</A><A target="_blank" href="https://doi.org/10.1021/acssynbio.8b00078">Watanabe, L. et al. iBioSim 3: A Tool for Model-Based Genetic Circuit Design. <I>ACS Synth. Biol.</I> acssynbio.8b00078 (2018). doi:10.1021/acssynbio.8b00078</A></LI><LI id="reference-3"><A href="#page-reference-3">↑</A><A target="_blank" href="https://doi.org/">Natarajan, A. &amp; Srienc, F. Glucose uptake rates of single E. coli cells grown in glucose-limited chemostat cultures. <I>J. Microbiol. Methods</I><B>42</B>, 87–96 (2000).</A></LI><LI id="reference-4"><A href="#page-reference-4">↑</A><A target="_blank" href="https://doi.org/10.1101/gr.912603">Selinger, D. W., Saxena, R. M., Cheung, K. J., Church, G. M. &amp; Rosenow, C. Global RNA half-life analysis in <I>Escherichia coli</I> reveals positional patterns of transcript degradation. <I>Genome Res.</I><B>13</B>, 216–23 (2003).</A></LI><LI id="reference-5"><A href="#page-reference-5">↑</A><A target="_blank" href="https://doi.org/">Part:BBa I20270 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_I20270. (Accessed: 15th October 2018)</A></LI><LI id="reference-6"><A href="#page-reference-6">↑</A><A target="_blank" href="https://doi.org/">Part:BBa J23100 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_J23100. (Accessed: 15th October 2018)</A></LI><LI id="reference-7"><A href="#page-reference-7">↑</A><A target="_blank" href="https://doi.org/10.1007/BF02241732">Fehlberg, E. Klassische Runge-Kutta-Formeln vierter und niedrigerer Ordnung mit Schrittweiten-Kontrolle und ihre Anwendung auf Wärmeleitungsprobleme. <I>Computing</I><B>6</B>, 61–71 (1970).</A></LI></OL><H2 class="sponsors-h">
                        Funding
                    </H2><DIV id="footer"><DIV id="license-wrapper"><P>Ⓒ 2018 iGEM Hamburg
                            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>