<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hamburg_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hamburg/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV lang="en" id="main" class="container-fluid"><NAV id="navbar" class="navbar navbar-expand-lg navbar-light bg-light"><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="navbar-nav mr-auto mt-2 mt-lg-0"><LI class="nav-item active"><A class="nav-link" href="https://2018.igem.org/Team:Hamburg">Home<SPAN class="sr-only">(current)</SPAN></A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Hamburg/Team#main-content" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Team
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Team#main-content">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Collaborations#main-content">Collaborations</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Hamburg/Description#main-content" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Project
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Description#main-content">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Design#main-content">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Experiments#main-content">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Notebook#main-content">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/InterLab#main-content">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Model#main-content">Model</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Results#main-content">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Demonstrate#main-content">Demonstrate</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Improve#main-content">Improve</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Chemistry#main-content">Chemistry</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Attributions#main-content">Attributions</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Parts
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Parts#main-content">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Basic_Part#main-content">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Composite_Part#main-content">Composite Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Part_Collection#main-content">Part Collection</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:Hamburg/Safety#main-content">Safety</A></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:Hamburg/Human_Practices#main-content">
                                    Human Practices
                                </A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                    Awards
                                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Entrepreneurship#main-content">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Hardware#main-content">Hardware</A><A class="dropdown-item" href="https://2018.igem.org/Team:Hamburg/Model#main-content">Model</A></DIV></LI></UL></DIV></NAV><DIV id="main-content" class="container bg-light"><H1>
                        Design
                    </H1><P>

                        Classic molecular biology relies on established processes and design patterns that work well in laboratory and industrial applications. Both hardware and DNA constructs are optimized for maximum yield and efficiency, always striving towards more product at less costs. In environmental synthetic biology, this approach is bound to fail.
                    </P><H2>

                        Environmental applications require a paradigm change
                    </H2><P>

                        Optimizing biologic systems for yield works under tightly controlled conditions. Cells need to be under constant surveillance, and slightest changes quickly lead to a total failure of the production chain. If we decided to use classic molecular biology approaches to solve environmental problems by releasing GMOs which serve a designed function in the lab, probably not much would happen. Over years of optimization, we disregarded the self-regulating abilities of biological systems to a point at which we call cell growth regulators “toxins”<A id="page-reference-1" href="#reference-1" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Brown, J. M. &amp; Shaw, K. J. A novel family of Escherichia coli toxin-antitoxin gene pairs. &lt;span class='journal'&gt;J. Bacteriol.&lt;/span&gt; &lt;span class='volume'&gt;185,&lt;/span&gt; 6600–8 (2003).">1</A>. 
                        We need to drastically change our view of a well-designed genetic circuit. Instead of using constitutive or strong inducible promoters, we need transcriptional regulators which respond to the environment. Instead of the strongest ribosome binding sites we need carefully selected combinations to minimize metabolic burden in energy-demanding systems. Instead of bioreactors we supply with virtually limitless energy, we need elaborate hardware that can protect and sustain environmental synthetic biology applications, even they are not watched day and night.
                        At iGEM Hamburg, we made sustainability of environmental synthetic biology an emerging reality. Here, we present design patterns we applied to the S.H.I.E.L.D., as a proof of concept we are confident many future environmental synthetic biology applications can use as a foundation towards a future in which we do not need to change our environment to meet our inabilities, but let our environment shape the abilities of our intelligent biological systems.
                    </P><H2>

                        Basic Parts as building blocks for sustainable circuits
               </H2><P>

                        We created a set of basic parts to both monitor and respond to environmental changes. Our environmental change sensors include glucose-repressed promoter MlcRE (<A href="http://parts.igem.org/Part:BBa_K2588000">BBa_K2588000</A>), improved cold-shock promoter HybB (<A href="http://parts.igem.org/Part:BBa_K2588001">BBa_K2588001</A>), and RNA-based transcription inverter icsA 5’ UTR/RnaG120 (<A href="http://parts.igem.org/Part:BBa_K2588003">BBa_K2588003</A>). Using these, as well as combinations of them and Parts of the registry, simple ways to monitor changes in nutrient availability and temperature can be monitored and converted into transcriptional signals.
                        How an appropriate respond to these signals looks like heavily depends on the application. A common response which we employ in the S.H.I.E.L.D. is repression of bacterial growth. There are many applications that already have implement to limit the number of bacteria in a system. These applications most often rely on killing surplus bacteria using a kill switch. While this certainly meets its goal, it is an unreasonable waste of energy and supplies. Instead, we propose to use regulator proteins from E. coli genome that already fulfil the task of responding to limiting factors with stalling cell growth and division. We identified key regulators of all important steps of cell growth: cspD (<A href="http://parts.igem.org/Part:BBa_K2588005">BBa_K2588005</A>) inhibits DNA replication<A id="page-reference-2" href="#reference-2" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Yamanaka, K., Zheng, W., Crooke, E., Wang, Y. H. &amp; Inouye, M. CspD, a novel DNA replication inhibitor induced during the stationary phase in Escherichia coli. &lt;span class='journal'&gt;Mol. Microbiol.&lt;/span&gt; &lt;span class='volume'&gt;39,&lt;/span&gt; 1572–84 (2001).">2</A>, mraZ (<A href="http://parts.igem.org/Part:BBa_K2588007">BBa_K2588007</A>) inhibits peptidoglycan synthesis<A id="page-reference-3" href="#reference-3" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Eraso, J. M., Markillie, L. M., Mitchell, H. D., Taylor, R. C., Orr, G. &amp; Margolin, W. The Highly Conserved MraZ Protein Is a Transcriptional Regulator in Escherichia coli. &lt;span class='journal'&gt;J. Bacteriol.&lt;/span&gt; &lt;span class='volume'&gt;196,&lt;/span&gt; 2053–2066 (2014).">3</A>, and, cbtA (<A href="http://parts.igem.org/Part:BBa_K2588006">BBa_K2588006</A>) and sulA (<A href="http://parts.igem.org/Part:BBa_K2588008">BBa_K2588008</A>) inhibit cell elongation and division<A id="page-reference-4" href="#reference-4" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Heller, D. M., Tavag, M. &amp; Hochschild, A. CbtA toxin of Escherichia coli inhibits cell division and cell elongation via direct and independent interactions with FtsZ and MreB. &lt;span class='journal'&gt;PLoS Genet.&lt;/span&gt; &lt;span class='volume'&gt;13,&lt;/span&gt; e1007007 (2017).">4</A>,<A id="page-reference-5" href="#reference-5" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Freudl, R., Braun, G., Honoré, N. &amp; Cole, S. T. Evolution of the enterobacterial sulA gene: a component of the SOS system encoding an inhibitor of cell division. &lt;span class='journal'&gt;Gene&lt;/span&gt; &lt;span class='volume'&gt;52,&lt;/span&gt; 31–40 (1987).">5</A>. Using a carefully selected combination of these, E. coli growth should be stalled or even completely inhibited without wasting resources.
                    </P><H2>

                        Assembling sustainable constructs
                    </H2><P>

                        While providing basic parts is good, the largest energy and resource savings can be made with the largest and most complex constructs that fulfil difficult tasks. For the S.H.I.E.L.D., we employ multiple highly complex partly synthetic, partly natural metabolic pathways for odour bait- and insecticide synthesis. Just overexpressing each enzyme of each pathway would certainly lead to production of each molecule, but in the S.H.I.E.L.D., E. coli needs to be able to maintain production without maintenance over extended periods of time. This requires special design considerations when assembling each pathway, which may impact efficiency and yield, but allow for a prolonged functionality.
                    </P><H2>

                        Making a bait attractive to its victim
                    </H2><P>

                        The ability to attract malaria mosquitoes is one of the most important functions of the S.H.I.E.L.D. With the goal of creating a lure that would imitate the mixture of volatiles on the human skin, attractive to malaria mosquitoes, we searched through generations of academic and industrial research. As it turns out, bacteria present on the human skin are mainly responsible for the production of these volatiles and even more intriguing: volatiles produced by skin bacteria are still more attractive to mosquitoes than the same volatiles produced synthetically<A id="page-reference-6" href="#reference-6" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Smallegange, R. C., Knols, B. G. J. &amp; Takken, W. Effectiveness of synthetic versus natural human volatiles as attractants for Anopheles gambiae (Diptera: Culicidae) sensu stricto. &lt;span class='journal'&gt;J. Med. Entomol.&lt;/span&gt; (2010). doi:10.1603/ME09015">6</A>.
                        Rather than applying one attractant alone, the combination of many attractants has been shown to lure human targeting mosquitoes the most effectively<A id="page-reference-7" href="#reference-7" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Okumu, F. O., Killeen, G. F., Ogoma, S., Biswaro, L., Smallegange, R. C., Mbeyela, E., Titus, E., Munk, C., Ngonyani, H., Takken, W., Mshinda, H., Mukabana, W. R. &amp; Moore, S. J. Development and field evaluation of a synthetic mosquito lure that is more attractive than humans. &lt;span class='journal'&gt;PLoS One&lt;/span&gt; (2010). doi:10.1371/journal.pone.0008951">7</A>,<A id="page-reference-8" href="#reference-8" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Verhulst, N. O., Beijleveld, H., Knols, B. G., Takken, W., Schraa, G., Bouwmeester, H. J. &amp; Smallegange, R. C. Cultured skin microbiota attracts malaria mosquitoes. &lt;span class='journal'&gt;Malar. J.&lt;/span&gt; (2009). doi:10.1186/1475-2875-8-302">8</A>,<A id="page-reference-9" href="#reference-9" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Verhulst, N. O., Mbadi, P. A., Kiss, G. B., Mukabana, W. R., van Loon, J. J. A., Takken, W. &amp; Smallegange, R. C. Improvement of a synthetic lure for Anopheles gambiae using compounds produced by human skin microbiota. &lt;span class='journal'&gt;Malar. J.&lt;/span&gt; &lt;span class='volume'&gt;10,&lt;/span&gt; 28 (2011).">9</A> We therefore chose lactic acid, myristic acid and 3-methyl-1-butanol as lures to implement into the S.H.I.E.L.D., a blend that has previously been shown to be a very effective lure<A id="page-reference-9" href="#reference-9" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Verhulst, N. O., Mbadi, P. A., Kiss, G. B., Mukabana, W. R., van Loon, J. J. A., Takken, W. &amp; Smallegange, R. C. Improvement of a synthetic lure for Anopheles gambiae using compounds produced by human skin microbiota. &lt;span class='journal'&gt;Malar. J.&lt;/span&gt; &lt;span class='volume'&gt;10,&lt;/span&gt; 28 (2011).">9</A>.
                        Each of these synthesis pathways has been already implemented in E.coli bacteria, for various industrial applications. As we aim for sustainability of the S.H.I.E.L.D., we designed constructs for the production of each compound. Because of the large metabolic load, constructs would be implemented only separately in designated cultures within the E.coli compartment.
                        The metabolic load is additionally reduced by the growth inhibition module, present in all lure-synthesizing colonies. This ensures, that lure synthesis is only activated at presence of nutrition and at activated, energy saving growth inhibition.
                    </P><H3>

                        Lactate
                    </H3><P>

                        Lactate dehydrogenase A (ldhA) is an enzyme catalysing the synthesis of lactate from pyruvate, as part of the fermentation metabolism activated upon anaerobic conditions. Lactate is used as part of the so called “basic blend”<A id="page-reference-9" href="#reference-9" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Verhulst, N. O., Mbadi, P. A., Kiss, G. B., Mukabana, W. R., van Loon, J. J. A., Takken, W. &amp; Smallegange, R. C. Improvement of a synthetic lure for Anopheles gambiae using compounds produced by human skin microbiota. &lt;span class='journal'&gt;Malar. J.&lt;/span&gt; &lt;span class='volume'&gt;10,&lt;/span&gt; 28 (2011).">9</A>,<A id="page-reference-10" href="#reference-10" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Mathew, N., Ayyanar, E., Shanmugavelu, S. &amp; Muthuswamy, K. Mosquito attractant blends to trap host seeking Aedes aegypti. &lt;span class='journal'&gt;Parasitol. Res.&lt;/span&gt; &lt;span class='volume'&gt;112,&lt;/span&gt; 1305–1312 (2013).">10</A> which is a combination of compounds that have been proven to be reliable attractors in the context of  research of Anopheles attractants. Therefore we introduced ldhA (<A href="http://parts.igem.org/Part:BBa_K2588009">BBa_K2588009</A>) as novel basic part and characterized it using the characterization construct pSB1C3-<A href="http://parts.igem.org/Part:BBa_K206000">BBa_K206000</A>-<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>-ldhA (<A href="http://parts.igem.org/Part:BBa_K2588041">BBa_K2588041</A>).
                    </P><H3>

                        Myristic acid
                    </H3><P>                    Like lactate myristic acid has been used as part of the anopheles mosquito attracting basic blend<A id="page-reference-9" href="#reference-9" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Verhulst, N. O., Mbadi, P. A., Kiss, G. B., Mukabana, W. R., van Loon, J. J. A., Takken, W. &amp; Smallegange, R. C. Improvement of a synthetic lure for Anopheles gambiae using compounds produced by human skin microbiota. &lt;span class='journal'&gt;Malar. J.&lt;/span&gt; &lt;span class='volume'&gt;10,&lt;/span&gt; 28 (2011).">9</A>,<A id="page-reference-10" href="#reference-10" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Mathew, N., Ayyanar, E., Shanmugavelu, S. &amp; Muthuswamy, K. Mosquito attractant blends to trap host seeking Aedes aegypti. &lt;span class='journal'&gt;Parasitol. Res.&lt;/span&gt; &lt;span class='volume'&gt;112,&lt;/span&gt; 1305–1312 (2013).">10</A>. We therefore designed a module, where we implemented a pathway that leads to a significantly higher production of C-14 fatty acids, including myristic acid. For this module we submitted 5 novel genes, which encode 4 subunits: accA (<A href="http://parts.igem.org/Part:BBa_K2588010">BBa_K2588010</A>), accB (<A href="http://parts.igem.org/Part:BBa_K2588011">BBa_K2588011</A>), accC (<A href="http://parts.igem.org/Part:BBa_K2588012">BBa_K2588012</A>), accD (<A href="http://parts.igem.org/Part:BBa_K2588013">BBa_K2588013</A>), and tesA (<A href="http://parts.igem.org/Part:BBa_K2588014">BBa_K2588014</A>). We assembled all genes into one module, to be regulated by glucose through the NOT-MlcRE promoter (<A href="http://parts.igem.org/Part:BBa_K2588022">BBa_K2588022</A>). For characterization a construct was designed with arabinose induce promoter (<A href="http://parts.igem.org/Part:BBa_K2588023">BBa_K2588023</A>), with the goal of verifying myristic acid by HPLC (high performance liquid chromatography).
                    </P><H3>

                        3-Methyl-1-butanol
                    </H3><P>

                        3-Methyl-1-butanol (3MB) is an organic compound that has been isolated as one of fourteen volatiles produced by human skin bacteria, that showed attracting effects on A. gambiae. Tested on its own 3MB was shown to be the best attractant, which was able to increase the effectiveness of the basic blend three fold in a lab and field test setup. We chose key enzymes from an engineered pathway that catalysed the reaction from pyruvate to 3MB. Those enzymes are encoded by LeuABCD Operon (<A href="http://parts.igem.org/Part:BBa_K2588015">BBa_K2588015</A>), kivD (<A href="http://parts.igem.org/Part:BBa_K2588016">BBa_K2588016</A>), ADH2 (<A href="http://parts.igem.org/Part:BBa_K2588017">BBa_K2588017</A>) and were designed to be assembled to a glucose controlled (NOT-MlcRE) module (<A href="http://parts.igem.org/Part:BBa_K2588024">BBa_K2588024</A>) to be implemented in S.H.I.E.L.D. To ensure, the expression of ADH2 and kivD, they are located upstream of the LeuABCD gene. To characterize the module, an arabinose induced promoter was chosen (<A href="http://parts.igem.org/Part:BBa_K2588025">BBa_K2588025</A>), again with the goal of verifying 3MB by HPLC.
                    </P><P>

                        However, mosquitoes are not only attracted by volatiles alone. In fact factors like heat and moisture play also a role in the attraction of mosquitoes<A id="page-reference-11" href="#reference-11" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Olanga, E. A., Okal, M. N., Mbadi, P. A., Kokwaro, E. D. &amp; Mukabana, W. R. Attraction of Anopheles gambiae to odour baits augmented with heat and moisture. &lt;span class='journal'&gt;Malar. J.&lt;/span&gt; &lt;span class='volume'&gt;9,&lt;/span&gt; 6 (2010).">11</A>. Having moisture already provided by the hydrogel, we therefore searched for ways to implement heat into our design.
                        We decided to employ <A href="http://parts.igem.org/Part:BBa_K410000">BBa_K410000</A>, a registry Part submitted by GeorgiaTech in 2010.
                        <A href="http://parts.igem.org/Part:BBa_K410000">BBa_K410000</A> is composed of cold-shock promoter HybB (<A href="http://parts.igem.org/Part:BBa_J45503">BBa_J45503</A>) and alternative oxidase a1 from Nelumbo nucifera (NnAOX1a), which we submitted as basic part as <A href="http://parts.igem.org/Part:BBa_K2588018">BBa_K2588018</A> since it was not individually submitted by GeorgiaTech themselves. NnAOX1 salvages electrons from the respiratory chain for heat generation. Combined in <A href="http://parts.igem.org/Part:BBa_K410000">BBa_K410000</A>, they have the function of inducing heat production at low temperatures.
                        S.H.I.E.L.D. is heavily focused on sustainability. We liked GeorgiaTech’s approach of coupling heat production to a negative feedback loop to ensure E. coli does not take damage from overexpression of a transmembrane protein salvaging electrons from their respiration chain. That HybB is active only at low temperatures, and only shows low expression at higher temperatures, compliments our philosophy of only using moderate expression to ensure viability of cells over long periods of time. HybB is active only at low temperatures, and only shows low expression at higher temperatures, which complements our philosophy of only using moderate expression to ensure viability of cells over long periods of time.
                        We expect the intrinsic feedback loop of <A href="http://parts.igem.org/Part:BBa_K410000">BBa_K410000</A> to work even at higher temperatures, despite its overall lower expression. <A href="http://parts.igem.org/Part:BBa_J45503">BBa_J45503</A> encodes more than just the HybB promoter
                        When we analysed <A href="http://parts.igem.org/Part:BBa_K410000">BBa_K410000</A> to confirm that it is properly designed, HybB caught our attention. Despite claiming to be a promoter sequence, an alignment against the E. coli genome revealed that <A href="http://parts.igem.org/Part:BBa_J45503">BBa_J45503</A> not only comprises of the clean promoter sequence, but also contains the Hyb0 5’ UTR and the first 100 bp of Hyb0 coding sequence. When using <A href="http://parts.igem.org/Part:BBa_J45503">BBa_J45503</A> in a standard assembly with a downstream gene of interest, we expect this to result in a conflict in translation of Hyb0 with the gene of interest encoded on the mRNA, which we expect to reduce the expression of the gene of interest for the benefit of truncated Hyb0 as unwanted side-product.
                    </P><H2>

                        Putting the E in S.H.I.E.L.D.
                    </H2><P>

                        Black Scorpion alpha Insect Toxin BjaIT (not registered as basic part) is a highly potent biological insecticide without toxicity to mammals<A id="page-reference-12" href="#reference-12" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="Arnon, T., Potikha, T., Sher, D., Elazar, M., Mao, W., Tal, T., Bosmans, F., Tytgat, J., Ben-Arie, N. &amp; Zlotkin, E. BjαIT: a novel scorpion α-toxin selective for insects—unique pharmacological tool. &lt;span class='journal'&gt;Insect Biochem. Mol. Biol.&lt;/span&gt; &lt;span class='volume'&gt;35,&lt;/span&gt; 187–195 (2005).">12</A>. Recombinant production in E. coli has been described before for industrial applications, in which BjaIT is produced intracellularly, and purified using an affinity tag after lysis<A id="page-reference-13" href="#reference-13" class="reference" data-toggle="tooltip" data-html="true" data-placement="top" title="李洪波, 夏玉先 &amp; 蒋鹰. CN106754944A - Efficient production method of nerve poison Bj alpha IT no-label active protein - Google Patents. (2016).">13</A>. In the S.H.I.E.L.D., E. coli needs to export BjaIT itself. We employ the type II secretion system composed of HlyB, HlyD, and TolC to export BjaIT from the cell. The secretion system was first registered at iGEM by UNICAMP EMSE Brazil as <A href="http://parts.igem.org/Part:BBa_K554013">BBa_K554013</A>. It can be used to export any protein with a HlyA signal peptide (<A href="http://parts.igem.org/Part:BBa_K554002">BBa_K554002</A>) at its C terminus. Since we were unsure whether BjaIT is active with the HlyA signal present, we incorporated an outer membrane protease T (OmpT, <A href="http://parts.igem.org/Part:BBa_K2588020">BBa_K2588020</A>) site into the linker between BjaIT and HlyA. We expanded <A href="http://parts.igem.org/Part:BBa_K554013">BBa_K554013</A> functionality by exchanging its promoter to more general use cases using pBAD promoter <A href="http://parts.igem.org/Part:BBa_K206000">BBa_K206000</A>, and expanded the mRNA by OmpT coding sequence. We registered the resulting composite part as <A href="http://parts.igem.org/Part:BBa_K2588030">BBa_K2588030</A>. Since all proteins expressed by this parts are transmembrane proteins and thus exert significant stress on cells expressing them, we suggest not fully inducing the pBAD promoter.
                        We registered BjaIT with linker and OmpT site as basic part <A href="http://parts.igem.org/Part:BBa_K2588019">BBa_K2588019</A>. While BjaIT is harmless to mammals, there are similar proteins which could be used for harmful causes. Note that we do not annotate or explain its sequence in detail due to biosecurity concerns.
                    </P><H2>

                        Complementing sustainable synthetic biology with sustaining hardware
                    </H2><P>

                        Using genetically engineered organisms in unmaintained, self-controlling field applications is not a prospect we greet light-heartedly. On the other hand, our bacteria would probably not survive for a long time due to the heavy duty synthetic biology they have to perform. With both problems in mind, we do not think that there is a biological solution to cater sustainability, biosafety, social acceptance, and regulation at the same time.
                        That is how the Sustainable Human-Imitating Elimination and Lure Device was born.
                        We began our hardware design process by defining all necessary functions:
                    </P><H3>

                        Producability
                    </H3><P>

                        The S.H.I.E.L.D. has a high task: Preventing malaria where all other methods have failed. Economic restrictions are a major reason for that failure. Therefore, any new solution needs to cost less for the same or a better effect. While we are positive that synthetic biology will produce better solutions, producing fancy hardware can quickly produce costs that render an application not feasible for mass production.
                        To produce the S.H.I.E.L.D.’s prototype, we employed 3D printing. However, for mass production, this method is far too time consuming and expensive. Designing the S.H.I.E.L.D., we therefore employed simple shapes that could also be produced with less elaborate and cheaper production methods like injection moulding. Injection moulding requires producing parts which do not have internal overhangs, so they can be taken out of their mould. Since the S.H.I.E.L.D. requires complex shapes, this means that it has to be made of several separate parts. Our prototype consists of six individual parts that have to be assembled prior to use. Between every two parts, there are seams. This brings up the problem of biosafety.
                    </P><H3>

                        Biosafety
                    </H3><P>
                        The S.H.I.E.L.D. needs to ensure that no GMOs can exit, and no biological agents can enter from outside. We employ durable NDR-70 sealing rings between each two separate parts to prevent release and contamination.
                        Odour bait molecules, the insecticide, oxygen and CO2 need to be able to enter and exit the S.H.I.E.L.D. Therefore, the S.H.I.E.L.D. cannot be sealed entirely. As interface between S.H.I.E.L.D. interior and environment we selected a commercially available nitrocellulose nano filter. Usually used to filter sterilize water in the laboratory, it is the most economic, most established, and most fit alternative for this task.
                    </P><H3>

                        Odour bait release
                    </H3><P>

                        Myristic acid, 3-methylbutan-1-ol, and lactic acid have low solubility in water. They can be attractive to mosquitoes due to their volatile nature, as they quickly evaporate from the human skin. To emulate this effect, we needed a surface from which odour bait molecules can evaporate evenly.
                    </P><H3>

                        Kill visiting mosquitoes
                    </H3><P>

                        Once attracted by odour baits, mosquitoes need to take up insecticides from the S.H.I.E.L.D. In the laboratory, mosquitoes are fed with human blood in a flat bowl spanned with parafilm. Mosquitoes can sting through the parafilm to take up the blood. The parafilm is destroyed in the process. For the S.H.I.E.L.D., we needed a solution that enables mosquitoes to take up insecticides without impairing successive function.
                        To emulate human skin for both odour bait release and mosquito behaviour, we employ a self-healing poly ammonium salt hydrogel. It simultaneously serves as a reservoir for the insecticide for
                    </P><H3>

                        Sustain a bacteria culture
                    </H3><P>

                        The S.H.I.E.L.D. is designed to be a long-lasting mosquito trap used in the warm malaria regions all over the world. To control the amount of bacteria we could have used a kill switch system. Instead of this, we decided to transform a growth inhibition module in our E. coli, which just interrupts cell division, but not the functionality of the bacterium. So the bacterium will live for a long period of time and doesn't grow until death phase. Usually for a sustainable bacteria culture there has to be a constant addition of medium, as the iGEM team Düsseldorf, we collaborated with, examined. By creating a co-culture of E. coli and cyanobacteria we were able to eliminate the necessity for additional medium. The cyanobacteria, placed in the transparent top of the S.H.I.E.L.D., will provide glucose by photosynthesis and other nutrients by its basic metabolic reactions. Thus the E. coli culture can last for extended periods of time.
                    </P><H3>

                        Perform without human interference
                    </H3><P>

                        With the growth inhibition module the S.H.I.E.L.D. will never be overgrown with E. coli. The device can be used by everybody, even if they are untrained with bacterial handling. Also there is no need for maintenance of the S.H.I.E.L.D. With the co-culture of E. coli and cyanobacteria the medium does not have to be refilled as the cyanobacteria provide all needed nutrients for E. coli.

                    </P><H2>
                        References
                    </H2><OL><LI id="reference-1"><A href="#page-reference-1">↑</A><A target="_blank" href="https://doi.org/">Brown, J. M. &amp; Shaw, K. J. A novel family of Escherichia coli toxin-antitoxin gene pairs. <I>J. Bacteriol.</I><B>185</B>, 6600–8 (2003).</A></LI><LI id="reference-2"><A href="#page-reference-2">↑</A><A target="_blank" href="https://doi.org/">Yamanaka, K., Zheng, W., Crooke, E., Wang, Y. H. &amp; Inouye, M. CspD, a novel DNA replication inhibitor induced during the stationary phase in Escherichia coli. <I>Mol. Microbiol.</I><B>39</B>, 1572–84 (2001).</A></LI><LI id="reference-3"><A href="#page-reference-3">↑</A><A target="_blank" href="https://doi.org/10.1128/JB.01370-13">Eraso, J. M. et al. The Highly Conserved MraZ Protein Is a Transcriptional Regulator in Escherichia coli. <I>J. Bacteriol.</I><B>196</B>, 2053–2066 (2014).</A></LI><LI id="reference-4"><A href="#page-reference-4">↑</A><A target="_blank" href="https://doi.org/10.1371/journal.pgen.1007007">Heller, D. M., Tavag, M. &amp; Hochschild, A. CbtA toxin of Escherichia coli inhibits cell division and cell elongation via direct and independent interactions with FtsZ and MreB. <I>PLoS Genet.</I><B>13</B>, e1007007 (2017).</A></LI><LI id="reference-5"><A href="#page-reference-5">↑</A><A target="_blank" href="https://doi.org/">Freudl, R., Braun, G., Honoré, N. &amp; Cole, S. T. Evolution of the enterobacterial sulA gene: a component of the SOS system encoding an inhibitor of cell division. <I>Gene</I><B>52</B>, 31–40 (1987).</A></LI><LI id="reference-6"><A href="#page-reference-6">↑</A><A target="_blank" href="https://doi.org/10.1603/ME09015">Smallegange, R. C., Knols, B. G. J. &amp; Takken, W. Effectiveness of synthetic versus natural human volatiles as attractants for Anopheles gambiae (Diptera: Culicidae) sensu stricto. <I>J. Med. Entomol.</I> (2010). doi:10.1603/ME09015</A></LI><LI id="reference-7"><A href="#page-reference-7">↑</A><A target="_blank" href="https://doi.org/10.1371/journal.pone.0008951">Okumu, F. O. et al. Development and field evaluation of a synthetic mosquito lure that is more attractive than humans. <I>PLoS One</I> (2010). doi:10.1371/journal.pone.0008951</A></LI><LI id="reference-8"><A href="#page-reference-8">↑</A><A target="_blank" href="https://doi.org/10.1186/1475-2875-8-302">Verhulst, N. O. et al. Cultured skin microbiota attracts malaria mosquitoes. <I>Malar. J.</I> (2009). doi:10.1186/1475-2875-8-302</A></LI><LI id="reference-9"><A href="#page-reference-9">↑</A><A target="_blank" href="https://doi.org/10.1186/1475-2875-10-28">Verhulst, N. O. et al. Improvement of a synthetic lure for Anopheles gambiae using compounds produced by human skin microbiota. <I>Malar. J.</I><B>10</B>, 28 (2011).</A></LI><LI id="reference-10"><A href="#page-reference-10">↑</A><A target="_blank" href="https://doi.org/10.1007/s00436-012-3266-2">Mathew, N., Ayyanar, E., Shanmugavelu, S. &amp; Muthuswamy, K. Mosquito attractant blends to trap host seeking Aedes aegypti. <I>Parasitol. Res.</I><B>112</B>, 1305–1312 (2013).</A></LI><LI id="reference-11"><A href="#page-reference-11">↑</A><A target="_blank" href="https://doi.org/10.1186/1475-2875-9-6">Olanga, E. A., Okal, M. N., Mbadi, P. A., Kokwaro, E. D. &amp; Mukabana, W. R. Attraction of Anopheles gambiae to odour baits augmented with heat and moisture. <I>Malar. J.</I><B>9</B>, 6 (2010).</A></LI><LI id="reference-12"><A href="#page-reference-12">↑</A><A target="_blank" href="https://doi.org/10.1016/j.ibmb.2004.11.005">Arnon, T. et al. BjαIT: a novel scorpion α-toxin selective for insects—unique pharmacological tool. <I>Insect Biochem. Mol. Biol.</I><B>35</B>, 187–195 (2005).</A></LI><LI id="reference-13"><A href="#page-reference-13">↑</A><A target="_blank" href="https://doi.org/">李洪波, 夏玉先 &amp; 蒋鹰. CN106754944A - Efficient production method of nerve poison Bj alpha IT no-label active protein - Google Patents. (2016).</A></LI></OL><H2 class="sponsors-h">
                        Funding
                    </H2><DIV id="footer"><DIV id="license-wrapper"><P>Ⓒ 2018 iGEM Hamburg
                            </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>