Team:Hamburg/Model
Home(current)
Team
Members Collaborations
Project
Description Design Protocols Notebook InterLab Model Results Demonstrate Improve Chemistry Attributions
Parts
Overview Basic Parts Composite Parts Part Collection
Safety
Human Practices
Awards
Entrepreneurship Hardware Model
Modeling
Achievements
We built a new model for protein synthesis including DNA, RNA and protein species
We predicted performance of our Basic Part BBa_K2588004 in a generic circuit setup
We based our choice of a constitutive promoter for our NOT-gate of MlcRE, BBa_K2588002 , on insights gained from this model
Modelling in synthetic biology serves the purpose to predict functions and                         behaviours that are difficult or impossible to characterize experimentally,                         and to drive design decisions that otherwise would take much time to gather                         information on. In our project, we implemented modelling for both aspects.
Shigella flexneri icsA is coding for an outer membrane protein and virulence factor, and is                         highly regulated by a unique mechanism involving a short noncoding RNA                         (ncRNA) which is complementary to icsA 5’ untranslated region (5’ UTR),                         called RnaG120 hereafter. When binding to nascent icsA mRNA, RnaG120                         induces stem loop formation, leading to termination of transcription before                         icsA coding sequence is transcribed 1 .
While RnaG120 was studied extensively in situ, to our knowledge it                         has never been used in an engineering context. We identified the vast                         potential of RnaG120 as a compact, extremely energy efficient NOT-gate.                         Assuming that binding of RnaG120 to icsA 5’ UTR is sufficient to induce                         stem loop formation and termination of transcription, expression of any                         gene cloned downstream of icsA 5’ UTR could be inhibited by expression of                         RnaG120. Based on this assumption, we created BBa_K2588002 as a proof of                         concept of a general RnaG120-based NOT gate.
BBa_K2588002 – An RnaG120-based NOT gate of glucose-inhibited promoter                         MlcRE
To build iGEM registry’s first ever transcription regulator that is induced                         by glucose, we employed RnaG120 to create a NOT-gate of glucose-inhibited                         promoter MlcRE ( BBa_K2588000 ). It is comprised of multiple subparts, in 5’                         to 3’ order: BBa_B0015 terminates transcription of RnaG120. A constitutive                         promoter provides constant gene of interest (GOI) mRNA transcription. icsA                         5’ UTR, which is complementary to RnaG120, acts as 5’ UTR for any GOI                         downstream of BBa_K2588002 . MlcRE is cloned in opposite reading direction,                         downstream of icsA 5’ UTR, activating transcription of RnaG120. Downstream                         of BBa_K2588002 , any gene of interest could be placed (Fig. 1).
Fig. 1 Map of BBa_K2588002 . Parts in order are: terminator BBa_B0015 , a                             constitutive promoter, icsA 5’ UTR BBa_K2588004 , MlcRE BBa_K2588000 (coded                             backwards). Note that only one DNA molecule is depicted, but annotated in                             two rows to accommodate for RnaG120/icsA operator, which has one function                             in each reading direction.
The Objective
We assume that the performance of BBa_K2588002 heavily relies on the ratio                         of transcribed mRNA and present RnaG120 depending on current induction. The                         easiest way to optimize this ratio is to select a constitutive promoter                         based on its strength relative to MlcRE. The Anderson collection offers a                         wide selection of constitutive promoters which have been characterized                         extensively. Testing all of them to select the one best suited for our                         application would be extremely laborious. Therefore, we decided on building                         a model that is able to predict the GOI expression based on constitutive                         promoter strength and glucose input.
Our aim was to select the Anderson promoter which provides the best GOI                         expression signal to noise ratio.
Building a model of RNA/RNA interactions using iBioSim
iBioSim is a java-based desktop application designed to create and analyse                         genetic circuit models 2 . Models built with iBioSim simulate cellular agents                         commonly used to predict kinetics of genetic circuits. Classic genetic                         circuits are built with regulated promoters inducing production of                         proteins. While iBioSim computes both transcription and translation using                         internal mRNA, it does not support manual editing regulatory functions of                         mRNA. BBa_K2588002 is a genetic circuit that performs logic purely on RNA                         level. Therefore, iBioSim does not support models of our application of the                         shelf.
As a workaround, we built our model to simulate both mRNA and RnaG120 as                         protein-level species behaving like RNA by setting stoichiometry of                         production, which usually is used to model the number of proteins produced                         from one mRNA molecule before its degradation, to 1. To model translation,                         icsA mRNA was set up to activate an implicit promoter acting as ribosome                         binding site to induce translation of a GOI. The transcription-terminating                         function of RnaG120 was modelled as complex formation of RnaG120 with icsA                         mRNA. The resulting complex would not activate translation of GOI,                         accounting for transcription termination before transcription of GOI coding                         sequence (Fig. 2).
Fig. 2 Setup for kinetic model of BBa_K2588002 . MlcRE was set up as                             activateable promoter with an activated RNAP binding equilibrium smaller                             than its basal RNAP binding equilibrium to account for the repressing                             function of glucose. Constitutive promoter and MlcRE induce transcription                             of GOI mRNA and RnaG120, respectively. GOI mRNA induces translation of GOI.                             RnaG120 and GOI mRNA undergo complex formation. The resulting complex does                             not induce translation of GOI.
Despite being a workaround, this model should kinetically perform exactly                         like the real construct, given that correct values are used for the                         relevant parameters of both constitutive promoter and MlcRE.
Genetic circuit models rely on many measured parameters to be accurate. We                         previously characterized MlcRE in BBa_K2588039 , comparing MlcRE-induced GFP                         expression to InterLab positive control BBa_I20270 , in which strong                         Anderson promoter BBa_J23100 drives GFP expression. Using this data,                         relative strengths of all Anderson promoters compared to MlcRE were                         computed and used as input values for the model. RnaG120 was extensively                         characterized by Tran et al., including binding behaviour of                         RnaG120 to icsA 5’ UTR. All model input values are summarized in Table 1.
Table 1: User-set values of BBa_K2588002 kinetic model
Species
Parameter
Value
Reference
Glucose
Degradation rate
0.0075
Natarajan et al 3
RnaG120
Degradation rate
0.068
Selinger et al 4
GOI mRNA
Degradation rate
0.068
Selinger et al 4
RnaG120/GOI mRNA complex
Degradation rate
0.068
Selinger et al 4
Complex formation equilibrium
0.7
Tran et al 1
GOI
Degradation rate
0.012
Set arbitrarily
Promoters
MlcRE
Basal production rate
0.81
MlcRE characterization
Activated production rate
0.19
MlcRE characterization
Stoichiometry of production
1
model-intrinsic
Constitutive Promoter
Basal production rate
1
BBa_I20270 compared to MlcRE in characterization 5
Activated production rate
1
BBa_I20270 compared to MlcRE in characterization 5
Open complex productio rate
varying (0.2-2)
Anderson promoter family 6
Stoichiometry of production
1
model-intrinsic
RBS
Activated production rate
1
model-intrinsic
Stoichiometry of production
20
Set arbitrarily
Results
BBa_K2588002 was modelled using a hierarchical Runge-Kutta-Fehlberg                         simulation 7 . The simulation was run for 1000 time units, with addition of                         glucose after 300 time units. Glucose was set up to be degraded over time                         to simulate consumption. Levels of RnaG120, GOI, and RnaG120-mRNA complex                         was monitored over time (Fig. 3A). The model predicts basal expression of                         RnaG120 and GOI, as well as complex formation of RnaG120 with GOI mRNA. Due                         to the RnaG120/GOI mRNA binding constant, RnaG120 is predicted to not fully                         bind to GOI mRNA. Upon glucose addition, RnaG120 and RnaG120-GOI mRNA                         levels are predicted to decline before returning to normal upon glucose                         consumption. Consequently, GOI translation from free GOI mRNA is predicted                         to raise GOI levels. Since GOI protein stability is set arbitrarily, it                         declines over time.
Consecutively, constitutive promoter strength was varied between 0.2 times                         measured MlcRE strength to 2 times measured MlcRE strength. Predicted                         signal to noise ratio was calculated as the ratio of basal GOI level before                         addition of glucose and peak GOI level after induction (Fig. 3B). A                         constitutive promoter with roughly half the strength of MlcRE without                         glucose was predicted to be optimal for BBa_K2588002 .
Fig. 3 Results of BBa_K2588002 model. A: Predicted RnaG120, GOI, and                             RnaG120-GOI mRNA complex levels over time, simulated using a hierarchical                             Runge-Kutta-Fehlberg simulation. After addition of glucose, RnaG120 and                             RnaG120-GOI mRNA complex are depleted, and GOI level is raised before                             returning to basal level after depletion of glucose. B: Predicted signal to                             noise ratio of GOI expression depending on constitutive promoter strength                             relative to basal MlcRE.
Conclusions
A constitutive promoter inducing transcription half as strongly as MlcRE                         without glucose repression was predicted to be optimal for use in BBa_K2588002 . Consequently, we chose BBa_J23106 as constitutive promoter.
Our sponsor Integrated DNA Technologies kindly synthesized BBa_K2588002 . We                         assembled it in pSB1C3 as basic part, however we were unable to assemble it with a GFP reporter. Therefore, we could not experimentally verify this model. We are positive that this part would have a tremendous value for future synthetic biology applications, and encourage every team to look further into it.
References
↑ Tran, C. N. et al. A multifactor regulatory circuit involving H-NS, VirF and an antisense RNA modulates transcription of the virulence gene icsA of Shigella flexneri. Nucleic Acids Res. 39, 8122–34 (2011).
↑ Watanabe, L. et al. iBioSim 3: A Tool for Model-Based Genetic Circuit Design. ACS Synth. Biol. acssynbio.8b00078 (2018). doi:10.1021/acssynbio.8b00078
↑ Natarajan, A. & Srienc, F. Glucose uptake rates of single E. coli cells grown in glucose-limited chemostat cultures. J. Microbiol. Methods 42, 87–96 (2000).
↑ Selinger, D. W., Saxena, R. M., Cheung, K. J., Church, G. M. & Rosenow, C. Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation. Genome Res. 13, 216–23 (2003).
↑ Part:BBa I20270 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_I20270. (Accessed: 15th October 2018)
↑ Part:BBa J23100 - parts.igem.org. Available at: http://parts.igem.org/Part:BBa_J23100. (Accessed: 15th October 2018)
↑ Fehlberg, E. Klassische Runge-Kutta-Formeln vierter und niedrigerer Ordnung mit Schrittweiten-Kontrolle und ihre Anwendung auf Wärmeleitungsprobleme. Computing 6, 61–71 (1970).
Funding
Ⓒ 2018 iGEM Hamburg
