<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Glasgow_Project_Mobility_Proteins"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Glasgow/Project/Mobility_Proteins">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Glasgow/Project/Mobility_Proteins&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Glasgow/Project/Mobility_Proteins&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Glasgow/Project/Mobility_Proteins&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Glasgow/Project/Mobility_Proteins" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Glasgow/Project/Mobility Proteins</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Bubble Test Page</TITLE></P><DIV id="BelowHeader"><DIV id="ddmenu"><UL id="menu"><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow" class="menulinks"> Home </A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Team" class="menulinks"> Team </A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Team#Students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Team#Supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Team#Superhero">Superhero</A></LI></UL><LI class="menuitems"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Glasgow" target="_blank" class="menulinks"> Team  Profile </A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Project" class="menulinks"> Project</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Project/Switch">The Switch</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Project/Measurements/Gas_Vesicles">The Gas Vesicle Proteins</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Project/Mobility_Proteins">The Motility Proteins</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Modeling_Intro">Dry Lab Modelling</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Project/Measurements">Measurements</A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Parts" class="menulinks"> BioBricks</A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Modeling_Intro" class="menulinks"> Modelling</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Modeling">Random Walk</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Modeling_Part2_align%3D">Bacteria and Buoyancy</A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Notebook" class="menulinks">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Weekly_Report">Weekly Report</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Notebook/Protocols">Protocols</A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Safety" class="menulinks"> Safety </A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Human_Practices" class="menulinks">Policy and Practice</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Human_Practices#Biodesalination" style="font-size:88%;">Biodesalination</A></LI><LI><A href="https://2014.igem.org/Team:Glasgow/Human_Practices#outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Wilkins">Artwork - Wilkins</A></LI><LI class="menuitems"><A href="https://2014.igem.org/Team:Glasgow/Attributions" class="menulinks"> Attributions </A></LI></DIV></DIV><DIV id="maincontent"><H2 class="pageheading">The Motility Genes</H2><P>
We wanted to place bacterial cell motility under the control of our recombinase-based switch, so that it could be switched OFF as gas vesicle production is switched ON. In order to do this we had we had to:
</P><OL><LI>Knock out motility genes.</LI><LI>Show that we can restore swimming with motility biobricks</LI><LI>Place the motility genes under the control of the switch</LI></OL><H2 class="subheading">Summary of Results</H2><P>We used recombineering to knock out the gene for the flagellar motor protein MotA, or the major flagellar protein FliC. 
Both of these knockouts abolished swimming in <EM>E. coli</EM>. Next we made biobrick versions of <EM>fliC</EM> and <EM>motA</EM> to see if they would restore swimming. 
Our composite biobrick consisting of <EM>fliC</EM> with the strong ribosome binding site BBa_B0034 in pSB1C3 did not restore swimming. However when a biobrick 
promoter was added, swimming was restored. The extent of mobility correlated well with the strength of the promoter driving <EM>fliC</EM>, and we could 
control swimming with IPTG by placing the <EM>fliC</EM> biobrick under the control of the lac promoter. We also made a biobrick version of 
<EM>motA</EM> with the BBa_B0032 ribosome binding site. However, this did not restore swimming with any of the promoters we tested.
 We reasoned that out <EM>motA</EM> knockout might be disrupting expression of the downstream motor gene <EM>motB</EM>. 
 We therefore made a composite biobrick consisting of <EM>motA</EM> and <EM>motB</EM> both with BBa_B0032 ribosome binding sites. 
 This restored swimming to our <EM>motA</EM> knockout, confirming our hypothesis that our <EM>motA</EM> insertion mutation has polar effects on <EM>motB</EM> expression.
 
 The next step would have been to engineer reversed versions of <EM>fliC</EM> or <EM>motAB</EM> and place them under the control of our switch so that they 
 were turned off by expression of phiC31 integrase. However, we ran out of time before we could do this.
<STRONG>Our results are shown in more detail in the sections below</STRONG></P><P align="center"><A href="#knockout">Knocking out MotA and FliC </A><A href="#fliC">Creating a <EM>fliC</EM> biobrick and restoring swimming to the <EM>fliC</EM> mutant</A><A href="#motA">Creating a <EM>motA</EM><EM>motB</EM> biobrick and restoring swimming to the <EM>motA</EM> mutant</A></P><A name="knockout"><H2 class="subheading">Gene Knockouts</H2></A><P>
The erythromycin resistance gene from Tn1545 together with its promoter was amplified from genomic DNA of an <EM>E. coli</EM> strain 
carrying this gene on its chromosome. The primers contained 50 nucleotides of sequences from either side of the <EM>motA</EM> gene 
or the <EM>fliC</EM> genes so that homologous recombination would precisely delete these genes and replace them with the erythromycin 
resistance gene. Two motile <EM>E. coli</EM> strains (DS941 and MG1655 Z1) were transformed with pKOBEG-C (Chaveroche et al 2000), a 
temperature sensitive plasmid that encodes the lambda red functions required for efficient homologous recombination with linear 
DNA in <EM>E. coli</EM>. Linear PCR products were transformed into electrocompetent cells and erythromycin resistant colonies were selected. 
Finally, pKOBEG-C was removed from the cells by growth at 42 degrees C.  In this way we made four different gene knockout strains: 
DS941 <EM>motA</EM>, DS941 <EM>fliC</EM>, MG1655 Z1 <EM>motA</EM>, and MG1655 Z1 <EM>fliC</EM>. 

We then used swarm assays to test motility of these knockout mutants and their parental strains. On soft agar nutrient plates 
(0.3% agar instead of the usual 1.5%), <EM>E. coli</EM> can swim across the surface. If a small spot of motile chemotactic bacteria is 
inoculated onto the centre of such a plate, the growing bacteria quickly use up the nutrients and migrate outwards towards unused 
nutrients. However, non-motile bacteria are unable to move and remain in a compact spot at the centre of the plate. The extent of 
migration over time can be used as a measure of swimming speed. Both <EM>fliC</EM> and <EM>motA</EM> knockout mutants were totally defective in swimming,
 whereas the parental DS941 and MG1655 Z1 strains could swim to the edges of a 9 cm plate in a 16 hour assay at 37 degrees C 
 <STRONG>(Figures 1A,B and 3A,B)</STRONG>. Loss of swimming behaviour was also observed in living cells by phase contrast microscopy.
</P><A name="fliC"><H2 class="subheading">FliC</H2></A><P>
To make the <EM>fliC</EM> biobrick, <EM>fliC</EM> was amplified by PCR using the proofreading Phusion polymerase with DS941 genomic DNA as template.
 The forward primer incorporated the prefix and added the BBa_B0034 ribosome binding site (RBS) and a scar sequence just upstream of <EM>fliC</EM>.
 The reverse primer incorporated the suffix and removed one undesirable PstI restriction site in <EM>fliC</EM>. <STRONG>Primers for FliC</STRONG></P><P>
 Taq polymerase was used to add overhanging A’s to the PCR product, which was then cloned into the PCR2.1 vector by TOPO-TA cloning. 
 To remove one unwanted SpeI site and two unwanted PstI sites, an NdeI-ClaI fragment of DNA was replaced by a 484 basepair synthetic 
 NdeI-ClaI g-block fragment synthesised by IDT with changes that removed these unwanted sites but did not alter the FliC protein sequence. 
 The <EM>fliC</EM> biobrick was in the correct orientation in PCR2.1 to be driven by the lac promoter in this vector. We found that this plasmid
 restored swimming in the <EM>fliC</EM> knockout of MG1655 Z1 (a strain that expresses high levels of the lac repressor LacI) in an IPTG-dependent manner. 
 In a swarm assay, MG1655 Z1 <EM>fliC</EM> / PCR2.1-<EM>fliC</EM> swam much further in the presence of IPTG than in its absence (data not shown).
 We sequenced our <EM>fliC</EM> biobrick and found that it had the expected sequence, identical to <EM>fliC</EM> of the sequenced <EM>E. coli</EM> strain MG1655 at all positions
 except for the changes we had made to remove three PstI sites and one SpeI site. However, there were two coding differences between our biobrick and a 
 previous <EM>fliC</EM> biobrick in the parts registry K777109. These differences are due to different source strains (We used MG1655 genomic DNA, they used <EM>E. coli</EM> strain K-12 substr. DH10B). It is unclear if these nonsynonymous changes alter flagella formation. The <EM>fliC</EM> biobrick K1463600 together with its BBa_B0034 RBS was cut out with EcoR1 and Pst1 and inserted into pSB1C3. This was tested to see if it restored 
 swimming using a swarm motility assay. The results showed that the <EM>fliC</EM> biobrick alone was not able to restore swimming <STRONG>(See Fig 1C)</STRONG>. 
 We hypothesised that this was due to a lack of promoter present to drive expression of <EM>fliC</EM>. 
 Therefore, we took double-stranded oligonucleotides (with EcoRI and XbaI ends) containing a promoter (either J23100, J23106 or J23116) and the B0032 
 RBS and inserted these upstream of B0034-<EM>fliC</EM> biobrick in pSB1C3. Due to an oversight, this meant our new biobricks 
 (BBa_K1463602, BBa_K1463603 and BBa_K1463604) contained both the B0032 and the B0034 RBS. Nevertheless, we tested these for restoration of swimming 
 <STRONG>(See Fig 1D- to 1H)</STRONG>. BBa_K1463604 containing the strongest J23100 promoter failed to restore swimming. However, on sequencing we found this plasmid to
 have a mutation in the promoter, explaining this result. We failed to clone a functional J23100 promoter in front of the <EM>fliC</EM> biobrick, suggesting that
 strong over expression of <EM>fliC</EM> may be toxic. Much more encouraging results were obtained with the other two promoters, J23106 and J23116.  The stronger 
 promoter J23106 restored swimming to wild-type levels <STRONG>(Figures 1G and H)</STRONG>, while the slightly weaker promoter restored swimming to a slightly lower level
 <STRONG>(Figures 1E and F)</STRONG>.
 The results of the swarm assay are also summarised in a histogram in <STRONG>figure 2</STRONG>.</P><DIV id="fig1&amp;2"><DIV id="figure2"><P class="figuretext">Figure 2: Average swarm diameter (cm) after growth at 37 degrees for 16 hours on 0.3% agar plates.
 Strains shown are DS941, DS941 <EM>fliC</EM>, and then DS941 <EM>fliC</EM> with plasmids containing J23100-(mutant)-<EM>fliC</EM>, J23116-<EM>fliC</EM> and J23106-<EM>fliC</EM>.</P></DIV><DIV id="figure1"><P class="figuretext">Figure 1:<STRONG> FliC Swarm Motility Assays.</STRONG> (A) DS941, (B) DS941 Δ<EM>fliC</EM>, (C) DS941 Δ<EM>fliC</EM> + pSB1C3 <EM>fliC</EM> (no promoter),
 (D) DS941 Δ<EM>fliC</EM> + J23100 (mutant promoter) <EM>fliC</EM>, (E) DS941 Δ<EM>fliC</EM> + J23116-<EM>fliC</EM>(1), (F) DS941 Δ<EM>fliC</EM> + J23116-<EM>fliC</EM>(2), (G) DS941 Δ<EM>fliC</EM> + J23106-<EM>fliC</EM>(1), (H) DS941 Δ<EM>fliC</EM> + J23106-<EM>fliC</EM>(2)</P></DIV></DIV><A name="motA"><H2 class="subheading">MotA</H2></A><P>
 To make the <EM>motA</EM> biobrick, <EM>motA</EM> was amplified by PCR using the proofreading Phusion polymerase using DS941 genomic DNA as template. The forward primer incorporated the prefix, 
 added the BBa_B0032 ribosome binding site (RBS) and a scar sequence just upstream of <EM>motA</EM> and changed the natural GTG start codon to ATG. 
 The reverse primer incorporated the suffix and changed the stop codon to TAA.</P><P>
 Note that an earlier <EM>motA</EM> biobrick (K777113) started at the first ATG codon within <EM>motA</EM> and therefore started at the wrong start codon 58 bp into the natural full length <EM>motA</EM>. Our <EM>motA</EM> has the 
 correct start as annotated on the <EM>E. coli</EM> genome sequence and our <EM>motA</EM> biobrick part <A href="http://parts.igem.org/Part:BBa_K1463700" target="_window">BBa_K1463700</A> is an improvement over K777113.

Our <EM>motA</EM> biobrick PCR product, complete with B0032 RBS, was then ligated into the pSB1C3 submission vector and also the plasmid J61002 containing the strong J23100 promoter (between SpeI and PstI sites,
 replacing the mRFP gene in this vector). The ligations were transformed into strains DH5α and TOP10, but colonies were only obtained with the pSB1C3 vector. A repeated ligation into the vector containing the
 strong J23100 promoter gave two colonies. However, sequencing showed that while <EM>motA</EM> clones in pSB1C3 had the correct sequence, the two inserts downstream of BBa_J23100 had mutations. One contained a mutation in
 the ribosome binding site, while the other had a 5 base deletion at the 5' end of the <EM>motA</EM> gene. This suggested that promoter J23100 was too strong and that high levels of <EM>motA</EM> expression might be toxic to the cells. 

We therefore decided to insert the BBa_B0032 RBS – <EM>motA</EM> biobrick into the BBa_J61002 vector containing a variety of different promoters from the parts distribution:</P><UL><LI>BBa_J23106 (½ the strength of J23100)</LI><LI>BBa_J23116 (¼ the strength of J23100)</LI><LI>BBa_J23103 (very weak promoter)</LI><LI>BBa_J23112 (weakest promoter we could find in the registry, barely any expression)</LI></UL><P>
(Strength measured with RFP: Part <A href="http://parts.igem.org/Part:BBa_J23100" target="_window">BBa_J23100</A>) 
These were checked by DNA sequencing and all found to have the expected sequence.

We then used swarm assays (semi-solid agar motility test) to investigate whether these plasmids would rescue swimming of a <EM>motA</EM> mutant. DS941 Δ<EM>motA</EM> (with <EM>motA</EM> deleted) was transformed with pSB1C3 <EM>motA</EM> (no promoter), <EM>motA</EM> transcribed from the four different weaker promoters in BBa_J61002, and also <EM>motA</EM> with the strong J23100 promoter with the mutated ribosome binding site. The results of the swarm assays are shown in <STRONG>Figures 3 and 4</STRONG>. DS941 and MG1655-Z1 (another positive swimming control) swam to approximately the same distance. Three different isolates of DS941 Δ<EM>motA</EM> did not swim at all, as expected for this mutant knocked out for the MotA motor protein. However, none of the plasmids containing <EM>motA</EM> restored swimming to the mutant to any significant extent, although it is possible that pSB1C3-<EM>motA</EM> (with no promoter) and BBa_J23100 – <EM>motA</EM> plasmid (with mutant RBS) gave slightly more mobility 
than no plasmid at all <STRONG>(Figures 3 and 4)</STRONG>.
</P><DIV id="fig3&amp;4"><DIV id="figure4"><P class="figuretext">Figure 4: A 5µl drop of overnight culture of the strains shown was spotted at the centre of a soft-agar 
nutrient plate and left to incubate overnight at 37°C</P></DIV><DIV id="figure3"><P class="figuretext">Figure 3: A 5µl drop of overnight culture of the strains shown was spotted at the centre of a soft-agar
 nutrient plate and left to incubate overnight at 37°C</P></DIV></DIV><P>
MotA is expressed from an operon containing two flagellar motor genes, <EM>motA</EM> and <EM>motB</EM>., and both of these genes are required for motor function, and hence swimming. Deletions in upstream genes in operons are often known to have “polar” effects, disrupting expression of downstream genes. Therefore our <EM>motA</EM> deletion might be severely reducing expression of <EM>motB</EM>. To test this, we decided to make a <EM>motA</EM>-<EM>motB</EM> biobrick and check whether it restores swimming to our delta-<EM>motA</EM> mutant.
 The <EM>motB</EM> gene was amplified from DS941 with a forward primer that incorporated a prefix, BBa_B0032 RBS and a reverse primer that incorporated a suffix and changed the stop codon from TGA to a stronger TAA, making our composite part BBa_K1463751. 
 The plasmids containing <EM>motA</EM> driven by different promoters in BBa_J61002 were digested with SpeI and PstI and the B0032 <EM>motB</EM> BBa_K1463751 PCR product was digested with XbaI and PstI. The fragments were ligated and then transformed into <EM>E. coli</EM> DS941. The J23100 promoter construct didn't give any colonies but all other ligations did, suggesting that the J23100 promoter is too strong, and over expression of motility proteins could be toxic. DS941 Δ<EM>motA</EM> was then transformed with BBa_J23103 <EM>motA</EM><EM>motB</EM>, J BBa_23106 <EM>motA</EM><EM>motB</EM>, BBa_J23112 <EM>motA</EM><EM>motB</EM> and BBa_J23116 <EM>motA</EM><EM>motB</EM> all in the plasmid vector BBa_J61002. Gene rescue was checked again by doing swarm assay <STRONG>(Figure 5)</STRONG>. This time we saw a significantly better result than just with <EM>motA</EM>, supporting our hypothesis that the <EM>motA</EM> mutation disrupts expression of <EM>motB</EM>.
 The diameter of migration on the swarm plates is shown in the histograms in <STRONG>figure 6</STRONG>.  The distance migrated when <EM>motA</EM> and <EM>motB</EM> were introduced into DS941 Δ<EM>motA</EM> correlated well with the strength of the promoters driving expression of <EM>motA</EM> and <EM>motB</EM>. The two stronger promoters BBa_J23116 and BBa_J23106 restored swimming to a greater extent than the two weaker promoters BBa_J23103 and BBa_J23112.
</P><DIV id="fig5&amp;6"><DIV id="figure6"><P class="figuretext">Figure 6: The histogram shows the diameter of growth on the swarm plates. </P></DIV><DIV id="figure5"><P class="figuretext">Figure 5: Photographs of swarm plates showing complementation of <EM>motA</EM> mutant with biobrick containing <EM>motA</EM> and <EM>motB</EM> driven by various promoters.</P></DIV></DIV></DIV><A class="editlink" href="https://2014.igem.org/wiki/index.php?title=Team:Glasgow/Project/Mobility_Proteins&amp;action=edit">Edit</A><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Glasgow/Project/Mobility_Proteins">http://2014.igem.org/Team:Glasgow/Project/Mobility_Proteins</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Glasgow/Project/Mobility_Proteins" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Glasgow/Project/Mobility_Proteins" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Glasgow/Project/Mobility_Proteins&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Glasgow/Project/Mobility_Proteins&amp;oldid=390647" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>