"
Page
Discussion
View source
History
teams
Log in
 
Team:Glasgow/Weekly Report
From 2014.igem.org
Bubble Test Page
Home
Team
Students
Supervisors
Superhero
Team
Profile
Project
The Switch
The Gas Vesicle Proteins
The Motility Proteins
Dry Lab Modelling
Measurements
BioBricks
Modelling
Random Walk
Bacteria and Buoyancy
Notebook
Weekly Report
Protocols
Safety
Policy and Practice
Biodesalination
Outreach
Artwork - Wilkins
Attributions
Weekly Report
Welcome to the weekly report pages!  Here, we have documented the week-to-week activities of the group, to help ourselves (and you!) see how the project is progressing.
Week 1
Wet Lab
Lab space was set up, involving arranging equipment and material and organizing work spaces
We ordered bits and bob we'd be needing for the lab work
We contacted some suppliers, in the hope of getting discounts or help in obtaining materials
Planning! We had to finalise the genes we required, and design the recombinase switch.  PCR primer design was also crucial, allowing us to isolate our specific genes ad insert preffixes, suffices and RBS
In the lab, we set up the E.Coli strains we would be using: DS941, DS941-Z1, Top10 and DH5 alpha.  We also generated transformed strains using plasmids from the iGEM 2014 distribution - including those containing GvpA and GvpC
Smaller sub teams were created, to spread the work out evenly and improve efficiency
Dry Lab
A dry lab was set up for the Biomedical Engineers, who are currently working on methods to model, observe and measure how buoyancy will affect the bacteria
Admin and Outreach
Facebook, Twitter and a group email account were set up, to allow us to communicate our project to the public
Plans began for our stall at the "Meet the Expert" event at Glasgow Science Center, an excellent opportunity to interact with the public and gauge their reaction to our proposed project.  This involved planning activities for the stall, and completing risk assessments.
CAKE ROTA was established, no lie.  Every Friday, a different team member would supply baked goods for our weekly meeting.
Week 2
Wet Lab
Isolated DNA from the transformed plasmids, for use in PCR later in the week.
PCR used to amplify and isolate our desired DNA from their plasmids, using the plasmids we designed in week one.
PCR products then run on a gel to confirm they were of the correct size.  Desired DNA was isolated for later use using gel extraction.
Primers designed for GvpA , GvpC, and for the purposes of isolating FliC from the E. Coli genome.  This will be used later on in the project to study the effect of the flagella in the presence of gas vesicles.
Bacteria transformed with plasmids taken from the distribution.  The parts used contained useful things such as GFP, a promoter, and vectors that will be used at a later stage.  The useful DNA from these transformations was isolated and digested using EcoR1 and Pst1, to ensure the sizes were correct.
One transformation was done using 2 different vectors containing RFP (pSB6A1 and pSB3C5), and two plasmids containing GFP, each with a different RBS. Another used instead the vector pSB1C3 and another containing the promoter J23100(CHECK) – both expressing RFP.  This later transformation will be run on a gel in week 3.
We had chosen 2 samples of our miniprepped DNA from out first transformation (one with GvpA, the other with GvpC) to be sent off for sequencing.  At the end of the week, this sequenced data was analysed, and proved that out isolated DNA contained the desired sequences.
An important step was taken this week: the ligation of our PCR-produced GvpA and GvpC - containing prefix, suffix and desired RBS – into separate pSB1C3 plasmid vectors.  In order to do this, the PCR products and the vector underwent a restriction digest using EcoR1 and Pst1.  The resultant DNA fragments were run on a gel and the fragments containing GvpA and GvpC and pSB1C3 were cut out and the DNA purified using gel extraction. GvpA was then ligated into pSB1C3.  Seperately, GvpC was ligated into pSB1C3.  The pSB1C3 was obtained by digesting the GFP-0034 BioBrick part from the distribution.
As the above was taking place, FliC was also digested with EcoR1 and Pst1.  The resultant gel showed a restriction site within FliC – a problem which must be resolved.
Dry Lab
Exploring the modelling of bacteria movement using MATLAB – with flagella and/or gas vesicles.   One of the key concepts is the Random Walk model.  Most often used to describe the random movement of gas particles in air, we modified the most general model slightly to more accurately represent the bacteria movement.
Basically, the bacteria will swim at a constant speed in a straight line, know as the “run”.  After a given amount of time (mean of 1 second), it will change direction, or “tumble”, setting off at a new angle, with a distribution described by a normal distribution.
This model was then combined with the upward movement supplied by the gas vesicles.  The calculations are not finalised yet, and more accurate representations will be worked on in week 3.
Admin and Outreach
Spoke to travel agent, getting details of flights to Boston (!) and accommodation while we're there.
Contacted some more potential sponsors who could supply us with useful products for free or at a discount.
Social media was updated with new photos.
Wiki was worked upon – the engineers are still learning HTML, so it's a work in progress!
Recombinase switch arrived – exciting stuff to come!
Baked goods supplied by Beth – lots of fudge and even more chocolate. Exceedingly yummy.  So yummy, in fact, that we never managed to take a picture!
Weeks 1&2
Weeks 3&4
Weeks 5&6
Weeks 7&8
Weeks 9&10
Edit
Retrieved from " http://2014.igem.org/Team:Glasgow/Weekly_Report "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
