"
Page
Discussion
View source
History
teams
Log in
 
Team:Glasgow/Weekly Report/Weeks 5and6
From 2014.igem.org
Bubble Test Page
Home
Team
Students
Supervisors
Superhero
Team
Profile
Project
The Switch
The Gas Vesicle Proteins
The Motility Proteins
Dry Lab Modelling
Measurements
BioBricks
Modelling
Random Walk
Bacteria and Buoyancy
Notebook
Weekly Report
Protocols
Safety
Policy and Practice
Biodesalination
Outreach
Artwork - Wilkins
Attributions
Week 5
Wet Lab
Testing and analysis of constructs
A lot of work was done this week to check that our constructs are correct.  Sequencing of PSB1C3/GvpA/GvpC revealed a 4 basepair deletion in the GvpA RBS.  We therefore decided to go back to our original glycerols of pSB1C3/GvpA and pSB1C3/GvpC.
A restriction digest using EcoR1+ Pst1 was carried out to ensure they contained the correct insert.  Another restriction digest was carried out on pSC101BB/MCS to determine whether or not the MCS was inserted correctly.
The isolated DNA was sent for sequencing as well as J23100/GvpA/GvpC and J23100/MotA. Sequencing of MotA/pSB1C3 showed no mutation.
A restriction digest revealed that the Top10 transformaiton of J23100/GvpA/GvpC contain a mix of plasmids.  This was confirmed by testing uncut DNA, single digest and a double digest.
Sample #1 which contains the desired DNA was then transformed into Top10, DH5α and DS941.  Some of the resultant transformations were miniprepped and digested (EcoR1 + Pst1) – this showed that the mixture of DNA had separated in the transformants.  Unfortunately, the single DS941 transformed did not contain the desired DNA.  Restriction digests of the transformants from the Top10 and DH5α strains revealed which transformants contains the desired plasmid.
FliC
PCR was carried out to isolate the FliC gene; this was to be used in a topisomerase reaction to remove a restriction site within the gene.  Unfortunately, the topisomerase reaction was unsuccessful – this may be due to a faulty kit.
MotA Knockouts
The swimming ability of the MotA KO strain was compared to the WT strain using “sloppy” agar, revealing that the Kos were unable to swim).  The KO was also transformed with the MotA plasmids (J23100/MotA and pSB1C3/MotA) to test the ability of the plasmid to rescue swimming.
GFP
In order to prove the presence of GFP is due to the promotor changing direction in response to integrase, colonies were short streaked.  Streaks were then selected to be miniprepped, and digested and compared on a gel.
Four colonies which fluoresced brightly, four which did not fluoresce and two which had intermediate fluorescence were selected. Restriction digest with HindIII and Pst1 revealed that the bright colonies contained a switched promoter whereas the non-fluorescing colonies do not. The intermediate colonies contained either both switched and non-switched plasmids or very little switched DNA.
After establishing that the switch is successful, oligos were designed for the reverse components of the switch: reverse RFP, reverse MotA and the different reverse RBS. Oigos were also designed to allow Phi C31 integrase and Gp3 to be made into biobricks.
Dry Lab
All the components required for the gas vesicle measurement apparatus was assembled.
A lot of work was also done on the wiki design: including the front page.
Figure 1: Mini Iced Sponge Cake - with added Haribo (baked by Amy)
Week 6
Wet Lab
PCR
PCR was carried out on reverse RBS/RFP/MotA along with φc31 integrase and Gp3.  The PCR products were then purified by gel extraction.
The PCR products were then digested with EcoR1 and Spe1 in order for them to be inserted into psB1C3/switch/RBS 34/GFP (digested with EcoR2 + Xba1).   φc31 integrase and gp3 were digested with EcoR1 and Pst1 to allow for insertion into pSB1C3 (also digested with EcoR1 + Pst1).
The ligations of the reverse components to the GFP/switch plasmid and  φc31 genes to pSB1C3 were successfully transformed into Top10 and DH5α.  The resultant transformations were then miniprepped and digested to ensure they contained the correct constructs.
FliC
A FliC topoisomerase reaction was set up to insert the FliC PCR product into PCR 2.1 plasmid.  The successful transformants were plated on Xgcel media, minipreps of both the resultant blue and white colonies were taken.  The DNA was then digested with EcoR1 and Pst1 to determine if the plasmid contains the insert, and if so, in which orientation.
RBS Mutation
In order to correct the mutation in the RBS of GvpA/GvpC/pSB1C3, PCR was used to isolate GvpA/GvpC from the plasmid.  The design of the primers altered the mutated site and allowed for the insertion of the desired sequence.  The PCR product was then purifed by gel extraction.
Promoters
Various promoters within J6100z were transformed into DH5α to eventually allow for the different expression levels of GvpA/GvpC.  The success of these transformations will be later determined by restriction digest and visualising by gel.
Oligos
Oligos were purified for later use later in mutagenic PCR; this is to allow  the switch to be inserted into a low copy number plasmid (pSC101-vector) and the integrase to be inserted into a plasmid under the control of an arabinose – inducible promoter (in pBAD33).  In order to accomplish this an EcoR1 site must be removed from pBAD33 and Spe1 site from the one on pSC101BB.
Note: upon inserting a MCS into pSC101-vector, it allows for pSC101-vector to become briobrick compatible.  Therefore, it will be now made reference to as pSC101 BB.
Swarm Experiments
Swarm experiment was carried out once again to compare the ability to swim of the WT, KO and rescure MotA strain. The rescue is able to swim slightly better than the KO strain, but swims very little when compared to the WT strain.  This suggests more may be required to rescue swimming.
Visualization of RFP and GFP
Cultures were set up in order to determine whether or not RFP and GFP can be visualised simultaneously using Typhoon.  However, it was discovered that it is difficult to measure GFP when RFP is present.
Dry Lab
Silicone beads
An experiment was set up to observe and measure the movement of silicone beads in solution; this will simulate the speed at which the bacteria sink in solution – the beads and bacteria have similar densities
Admin and Outreach
A lot of work was also done to plan how the project could be changed in response to the view of the public and other synthetic biologists.
Figure 2: Chocolate Brownies (baked by Lydia)
Weeks 1&2
Weeks 3&4
Weeks 5&6
Weeks 7&8
Weeks 9&10
Edit
Retrieved from " http://2014.igem.org/Team:Glasgow/Weekly_Report/Weeks_5and6 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
