"
Page
Discussion
View source
History
teams
Log in
 
Team:Glasgow/Project/Measurements/Gas Vesicles
From 2014.igem.org
Bubble Test Page
Home
Team
Students
Supervisors
Superhero
Team
Profile
Project
The Switch
The Gas Vesicle Proteins
The Motility Proteins
Dry Lab Modelling
Measurements
BioBricks
Modelling
Random Walk
Bacteria and Buoyancy
Notebook
Weekly Report
Protocols
Safety
Policy and Practice
Biodesalination
Outreach
Artwork - Wilkins
Attributions
What are Gas Vesicles?
Gas vesicles (GVs) have been found to be expressed in various species of bacteria as well as halophilic archaea. In these organisms, gas vesicles are employed to allow the movement of the organisms up the water column, from areas of low oxygen concentration to a more oxygen-rich environment. Photosynthetic bacteria also use gas vesicles to reach areas with optimal light intensity. The hydrophobic proteins which compose the gas vesicles are known as Gas vesicle proteins or Gvps.
Many genes have been implicated in the formation of gas vesicles. GvpA is the major structural protein of the gas vesicle. GvpA has been shown to form gas vesicles with a single layer wall with a bicone terminal structures. This structure appears to be strengthened and extended by other Gvps including GvpC, which is the second major structural component of the gas vesicles. In 2007, the Melbourne iGEM team used an 11kbp construct containing many gas vesicle genes to produce floating bacteria; however, this construct did not include gvpA suggesting that GvpA may not be essential to gas vesicle production. This system was later improved by the Groningen iGEM team in 2009, which used the system as an indicator for lead accumulation in the cells.
In 2012, the Ocean University China (OUC) team created a construct using gvpA and gvpC coupled to GFP expression. Planktothrix rubescens was used as the source of the gas vesicle genes to insert into E. coli. They found that upon transformation of the E. coli with 2 plasmids (one containing gvpA with a constitutive promoter and the other containing gvpC with a constitutive promoter) that their constructs appeared to work better than the Melbourne BioBrick. Expression of these two biobricks led to slower settling of cells in a buffered saline solution.
Our team decided to insert gvpA and gvpC genes into E. coli on a single plasmid to bring about floating under the control of our recombination switch promoter. The team decided to isolate gvpA from the OUC BioBrick K737017 and the gvpC from K737007 which were both available in the iGEM distribution. The floating assay protocol detail by Melbourne was used to determine whether or not our E. coli were able to float. How we achieved this, and the results of our efforts, are detailed below.
GvpA and GvpC
Before any work could be done with gvpA and gvpC, the genes had to be isolated from the constructs found in the distribution. In order to do this, PCR primers were designed to isolate the gvp genes as well as being designed to add the BioBrick prefix and suffix to the genes. To allow the genes to be translated, a RBS was added into the primers between the prefix and the coding sequence. The gvpA forward primer contained the sequence for RBS 0034 (a relatively strong RBS) and the gvpC forward primer contained the sequence for RBS 0032 – comparatively weak RBS, reflecting the relative amounts of GvpA and GvpC proteins in gas vesicles. The genes encoding GvpA and GvpC were then isolated and amplified by PCR; this was that confirmed by gel electrophoresis of a sample of the PCR.
The DNA was then digested with EcoRI and PstI to allow the construct to be inserted into pSB1C3, the constructs were then transformed successfully into TOP10 and DH5α. The resultant tranformant DNA was isolated by miniprep; the presence of the constructs was confirmed by running the DNA on a gel.
In an attempt to cause GV formation, gvpA and gvpC would need to be placed under the control of a promoter. The first step taken was to insert gvpA into an Ampr plasmid (BBa_J61002) containing a strong constitutive promoter (J23100); to insert gvpA downstream of the promoter, the gene had to be isolated by restriction digest using XbaI and PstI to allow the ends to be compatible with theBBa_J61002 - J23100 plasmid – which has also been digested with SpeI and PstI. The gvpC gene was then inserted downstream of gvpA using the same process. pSB1C3 containing the gvpC biobrick was disgested with XbaI and PstI and the J23100/gvpA plasmid was digested with SpeI and PstI – this caused the formation of a scar sequence between GvpA and GvpC upon ligation of the correct fragments. Again, the constructs were transformed into TOP10 and DH5α. The DNA was isolated by miniprep to be digested and run on the gel to check the presence of the inserts. The DNA was also sent for sequencing.
A similar process was carried out to place gvpC downstream of gvpA, each with their own RBS in pSB1C3. The resulting plasmid was given the biobrick number BBa_K1463340, and the construct with the J23100 promoter is BBa_K146345. The results of the sequencing showed that the construct had mutated in the RBS and would therefore be unable to function.
In order to resolve the mutant construct, new PCR primers were designed to isolate the mutated construct from the plasmid by PCR – as well as correct the mutation – and reinsert it into a new plasmid.
The construct was reinserted into the plasmid containing the J23100 promoter. The corrected construct was also inserted into BBa_J61002-based plasmids containing promoters of varying strengths (J23103, J23106, J23112 and J23116). Again the constructs were confirmed by transforming into TOP10 and DH5α, the transformants were minipreped and restricted to run on a gel. Liquid cultures of the different transformants were left for 3 days in 30C and 37C to give the cultures time to grow and settle – this would allow any floating to be observed. Unfortunately, there appeared to be no difference between the different transformants and floating was not observed.
The construct was also placed in a chloramphenicol resistant plasmid under the control of the arabinose inducible pBAD promoter (pBAD33; Guzman et al 1995). The resultant DNA from the transformants was sent for sequencing to confirm the presence of an unmutated construct. The confirmed transformants were then observed for their ability to float.
Cells were first grown in LB until mid exponential phase had been reached, and then grown a further 2 hours with either 0.2% glucose, or 0.2% arabinose to induce GvpA and GvpC expression. Cells were harvested by centrifugation and re-suspended in filter sterilised 0.15% NaCl. The samples were observed at intervals over 5 days but no difference was visible between the induced and non-induced cell samples. Both sets of cells contain the pBAD33-gvpAC plasmid, but one had been induced with arabinose and the other had not.
Figure 2: Comparison of control and arabinose induced cells after 96 hours.
Figure 1: Comparison of control and arabinose induced cells after 72 hours.
Figure 3: Comparison of control and arabinose induced cells after 120 hours.
References
Offner. S., Hofacker. A., Wanner. G., Pfeifer. F.,Eight of Fourteen gvp Genes Are Sufficient for Formation of Gas Vesicles in Halophilic Archaea, 2000, Journal of Bacteriology, Vol 182, No 15, pg 4328-4336
Pfeifer. F.,Distribution, formation and regulation of gas vesicles, 2012, Nature Reviews, Microbiology, Vol 10, pg 705-715
Ramsay. J.P. and Salmond. G.P.C.,Quorum sensing-controlled buoyancy through gas vesicles, 2011, Communicative & Integrative Biology, Vol 5, Issue 1. pg 96-98
Edit
Retrieved from " http://2014.igem.org/Team:Glasgow/Project/Measurements/Gas_Vesicles "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
