"
Page
Discussion
View source
History
teams
Log in
 
Team:Glasgow/Project/Mobility Proteins
From 2014.igem.org
Bubble Test Page
Home
Team
Students
Supervisors
Superhero
Team
Profile
Project
The Switch
The Gas Vesicle Proteins
The Motility Proteins
Dry Lab Modelling
Measurements
BioBricks
Modelling
Random Walk
Bacteria and Buoyancy
Notebook
Weekly Report
Protocols
Safety
Policy and Practice
Biodesalination
Outreach
Artwork - Wilkins
Attributions
The Motility Genes
We wanted to place bacterial cell motility under the control of our recombinase-based switch, so that it could be switched OFF as gas vesicle production is switched ON. In order to do this we had we had to:
Knock out motility genes.
Show that we can restore swimming with motility biobricks
Place the motility genes under the control of the switch
Summary of Results
We used recombineering to knock out the gene for the flagellar motor protein MotA, or the major flagellar protein FliC.  Both of these knockouts abolished swimming in E. coli. Next we made biobrick versions of fliC and motA to see if they would restore swimming.  Our composite biobrick consisting of fliC with the strong ribosome binding site BBa_B0034 in pSB1C3 did not restore swimming. However when a biobrick  promoter was added, swimming was restored. The extent of mobility correlated well with the strength of the promoter driving fliC, and we could  control swimming with IPTG by placing the fliC biobrick under the control of the lac promoter. We also made a biobrick version of motA with the BBa_B0032 ribosome binding site. However, this did not restore swimming with any of the promoters we tested.  We reasoned that out motA knockout might be disrupting expression of the downstream motor gene motB.   We therefore made a composite biobrick consisting of motA and motB both with BBa_B0032 ribosome binding sites.   This restored swimming to our motA knockout, confirming our hypothesis that our motA insertion mutation has polar effects on motB expression.
The next step would have been to engineer reversed versions of fliC or motAB and place them under the control of our switch so that they   were turned off by expression of phiC31 integrase. However, we ran out of time before we could do this.
Our results are shown in more detail in the sections below
Knocking out MotA and FliC
Creating a fliC biobrick and restoring swimming to the fliC mutant
Creating a motA motB biobrick and restoring swimming to the motA mutant
Gene Knockouts
The erythromycin resistance gene from Tn1545 together with its promoter was amplified from genomic DNA of an E. coli strain  carrying this gene on its chromosome. The primers contained 50 nucleotides of sequences from either side of the motA gene  or the fliC genes so that homologous recombination would precisely delete these genes and replace them with the erythromycin  resistance gene. Two motile E. coli strains (DS941 and MG1655 Z1) were transformed with pKOBEG-C (Chaveroche et al 2000), a  temperature sensitive plasmid that encodes the lambda red functions required for efficient homologous recombination with linear  DNA in E. coli. Linear PCR products were transformed into electrocompetent cells and erythromycin resistant colonies were selected.  Finally, pKOBEG-C was removed from the cells by growth at 42 degrees C.  In this way we made four different gene knockout strains:  DS941 motA, DS941 fliC, MG1655 Z1 motA, and MG1655 Z1 fliC.
We then used swarm assays to test motility of these knockout mutants and their parental strains. On soft agar nutrient plates  (0.3% agar instead of the usual 1.5%), E. coli can swim across the surface. If a small spot of motile chemotactic bacteria is  inoculated onto the centre of such a plate, the growing bacteria quickly use up the nutrients and migrate outwards towards unused  nutrients. However, non-motile bacteria are unable to move and remain in a compact spot at the centre of the plate. The extent of  migration over time can be used as a measure of swimming speed. Both fliC and motA knockout mutants were totally defective in swimming,  whereas the parental DS941 and MG1655 Z1 strains could swim to the edges of a 9 cm plate in a 16 hour assay at 37 degrees C (Figures 1A,B and 3A,B). Loss of swimming behaviour was also observed in living cells by phase contrast microscopy.
FliC
To make the fliC biobrick, fliC was amplified by PCR using the proofreading Phusion polymerase with DS941 genomic DNA as template.  The forward primer incorporated the prefix and added the BBa_B0034 ribosome binding site (RBS) and a scar sequence just upstream of fliC.  The reverse primer incorporated the suffix and removed one undesirable PstI restriction site in fliC.
Primers for FliC
Taq polymerase was used to add overhanging A’s to the PCR product, which was then cloned into the PCR2.1 vector by TOPO-TA cloning.   To remove one unwanted SpeI site and two unwanted PstI sites, an NdeI-ClaI fragment of DNA was replaced by a 484 basepair synthetic   NdeI-ClaI g-block fragment synthesised by IDT with changes that removed these unwanted sites but did not alter the FliC protein sequence.
The fliC biobrick was in the correct orientation in PCR2.1 to be driven by the lac promoter in this vector. We found that this plasmid  restored swimming in the fliC knockout of MG1655 Z1 (a strain that expresses high levels of the lac repressor LacI) in an IPTG-dependent manner.   In a swarm assay, MG1655 Z1 fliC / PCR2.1-fliC swam much further in the presence of IPTG than in its absence (data not shown).
We sequenced our fliC biobrick and found that it had the expected sequence, identical to fliC of the sequenced E. coli strain MG1655 at all positions  except for the changes we had made to remove three PstI sites and one SpeI site. However, there were two coding differences between our biobrick and a   previous fliC biobrick in the parts registry K777109. These differences are due to different source strains (We used MG1655 genomic DNA, they used E. coli strain K-12 substr. DH10B). It is unclear if these nonsynonymous changes alter flagella formation. The fliC biobrick K1463600 together with its BBa_B0034 RBS was cut out with EcoR1 and Pst1 and inserted into pSB1C3. This was tested to see if it restored   swimming using a swarm motility assay. The results showed that the fliC biobrick alone was not able to restore swimming (See Fig 1C).   We hypothesised that this was due to a lack of promoter present to drive expression of fliC.
Therefore, we took double-stranded oligonucleotides (with EcoRI and XbaI ends) containing a promoter (either J23100, J23106 or J23116) and the B0032   RBS and inserted these upstream of B0034-fliC biobrick in pSB1C3. Due to an oversight, this meant our new biobricks   (BBa_K1463602, BBa_K1463603 and BBa_K1463604) contained both the B0032 and the B0034 RBS. Nevertheless, we tested these for restoration of swimming (See Fig 1D- to 1H). BBa_K1463604 containing the strongest J23100 promoter failed to restore swimming. However, on sequencing we found this plasmid to  have a mutation in the promoter, explaining this result. We failed to clone a functional J23100 promoter in front of the fliC biobrick, suggesting that  strong over expression of fliC may be toxic. Much more encouraging results were obtained with the other two promoters, J23106 and J23116.  The stronger   promoter J23106 restored swimming to wild-type levels (Figures 1G and H), while the slightly weaker promoter restored swimming to a slightly lower level (Figures 1E and F).
The results of the swarm assay are also summarised in a histogram in figure 2.
Figure 2: Average swarm diameter (cm) after growth at 37 degrees for 16 hours on 0.3% agar plates.  Strains shown are DS941, DS941 fliC, and then DS941 fliC with plasmids containing J23100-(mutant)-fliC, J23116-fliC and J23106-fliC.
Figure 1: FliC Swarm Motility Assays. (A) DS941, (B) DS941 ΔfliC, (C) DS941 ΔfliC + pSB1C3 fliC (no promoter),  (D) DS941 ΔfliC + J23100 (mutant promoter) fliC, (E) DS941 ΔfliC + J23116-fliC(1), (F) DS941 ΔfliC + J23116-fliC(2), (G) DS941 ΔfliC + J23106-fliC(1), (H) DS941 ΔfliC + J23106-fliC(2)
MotA
To make the motA biobrick, motA was amplified by PCR using the proofreading Phusion polymerase using DS941 genomic DNA as template. The forward primer incorporated the prefix,   added the BBa_B0032 ribosome binding site (RBS) and a scar sequence just upstream of motA and changed the natural GTG start codon to ATG.   The reverse primer incorporated the suffix and changed the stop codon to TAA.
Note that an earlier motA biobrick (K777113) started at the first ATG codon within motA and therefore started at the wrong start codon 58 bp into the natural full length motA. Our motA has the   correct start as annotated on the E. coli genome sequence and our motA biobrick part BBa_K1463700 is an improvement over K777113.
Our motA biobrick PCR product, complete with B0032 RBS, was then ligated into the pSB1C3 submission vector and also the plasmid J61002 containing the strong J23100 promoter (between SpeI and PstI sites,  replacing the mRFP gene in this vector). The ligations were transformed into strains DH5α and TOP10, but colonies were only obtained with the pSB1C3 vector. A repeated ligation into the vector containing the  strong J23100 promoter gave two colonies. However, sequencing showed that while motA clones in pSB1C3 had the correct sequence, the two inserts downstream of BBa_J23100 had mutations. One contained a mutation in  the ribosome binding site, while the other had a 5 base deletion at the 5' end of the motA gene. This suggested that promoter J23100 was too strong and that high levels of motA expression might be toxic to the cells.
We therefore decided to insert the BBa_B0032 RBS – motA biobrick into the BBa_J61002 vector containing a variety of different promoters from the parts distribution:
BBa_J23106 (½ the strength of J23100)
BBa_J23116 (¼ the strength of J23100)
BBa_J23103 (very weak promoter)
BBa_J23112 (weakest promoter we could find in the registry, barely any expression)
(Strength measured with RFP: Part BBa_J23100 )
These were checked by DNA sequencing and all found to have the expected sequence.
We then used swarm assays (semi-solid agar motility test) to investigate whether these plasmids would rescue swimming of a motA mutant. DS941 ΔmotA (with motA deleted) was transformed with pSB1C3 motA (no promoter), motA transcribed from the four different weaker promoters in BBa_J61002, and also motA with the strong J23100 promoter with the mutated ribosome binding site. The results of the swarm assays are shown in Figures 3 and 4. DS941 and MG1655-Z1 (another positive swimming control) swam to approximately the same distance. Three different isolates of DS941 ΔmotA did not swim at all, as expected for this mutant knocked out for the MotA motor protein. However, none of the plasmids containing motA restored swimming to the mutant to any significant extent, although it is possible that pSB1C3-motA (with no promoter) and BBa_J23100 – motA plasmid (with mutant RBS) gave slightly more mobility  than no plasmid at all (Figures 3 and 4).
Figure 4: A 5µl drop of overnight culture of the strains shown was spotted at the centre of a soft-agar  nutrient plate and left to incubate overnight at 37°C
Figure 3: A 5µl drop of overnight culture of the strains shown was spotted at the centre of a soft-agar  nutrient plate and left to incubate overnight at 37°C
MotA is expressed from an operon containing two flagellar motor genes, motA and motB., and both of these genes are required for motor function, and hence swimming. Deletions in upstream genes in operons are often known to have “polar” effects, disrupting expression of downstream genes. Therefore our motA deletion might be severely reducing expression of motB. To test this, we decided to make a motA-motB biobrick and check whether it restores swimming to our delta-motA mutant.
The motB gene was amplified from DS941 with a forward primer that incorporated a prefix, BBa_B0032 RBS and a reverse primer that incorporated a suffix and changed the stop codon from TGA to a stronger TAA, making our composite part BBa_K1463751.
The plasmids containing motA driven by different promoters in BBa_J61002 were digested with SpeI and PstI and the B0032 motB BBa_K1463751 PCR product was digested with XbaI and PstI. The fragments were ligated and then transformed into E. coli DS941. The J23100 promoter construct didn't give any colonies but all other ligations did, suggesting that the J23100 promoter is too strong, and over expression of motility proteins could be toxic. DS941 ΔmotA was then transformed with BBa_J23103 motA motB, J BBa_23106 motA motB, BBa_J23112 motA motB and BBa_J23116 motA motB all in the plasmid vector BBa_J61002. Gene rescue was checked again by doing swarm assay (Figure 5). This time we saw a significantly better result than just with motA, supporting our hypothesis that the motA mutation disrupts expression of motB.
The diameter of migration on the swarm plates is shown in the histograms in figure 6.  The distance migrated when motA and motB were introduced into DS941 ΔmotA correlated well with the strength of the promoters driving expression of motA and motB. The two stronger promoters BBa_J23116 and BBa_J23106 restored swimming to a greater extent than the two weaker promoters BBa_J23103 and BBa_J23112.
Figure 6: The histogram shows the diameter of growth on the swarm plates.
Figure 5: Photographs of swarm plates showing complementation of motA mutant with biobrick containing motA and motB driven by various promoters.
Edit
Retrieved from " http://2014.igem.org/Team:Glasgow/Project/Mobility_Proteins "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
