<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Exeter_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Exeter/Project</SPAN></H1></DIV></DIV></DIV><NAV id="Top" class="navbar navbar-inverse navbar-static-top"><DIV class="container-fluid"><DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI><DIV id="links_drop" class="dropdown"><BUTTON class="dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"><SPAN style="margin-bottom:4px;">Lab</SPAN></BUTTON></DIV></LI><LI><A id="links" style="margin:10px 0 30px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Project">Lab Project</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Labbook">Lab Book</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Safety">Safety</A></LI></UL></DIV><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Parts">Parts</A></LI><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Team">Team</A></LI><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Interlab">InterLab</A></LI><LI><DIV id="links_drop" class="dropdown"><BUTTON class="dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"><SPAN style="margin-bottom:4px;">Human Practices</SPAN></BUTTON></DIV></LI><LI><A id="links" style="margin:10px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Integrated_Practices">Integrated</A></LI><LI><A id="links" style="background:none;line-height:0.7vh;margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Engagement">Public Engagement&amp; Education</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Log">Log</A></LI></DIV><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Attributions">Attributions</A></LI><LI><DIV id="links_drop" class="dropdown"><BUTTON class="dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"><SPAN style="margin-bottom:4px;">Awards</SPAN></BUTTON></DIV></LI><LI><A id="links" style="margin:10px 0 30px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Awards">Awards</A></LI><LI><SPAN style="margin:10px 0 30px 2px;padding:0;"><U>Special pages</U></SPAN></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/HP/Silver">HP Silver</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/HP/Gold">HP Gold</A></LI></DIV><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Model">Models</A></LI><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Collaborations">Collaborations</A></LI></NAV><DIV id="scrollPosition" style="display:none;">0</DIV><DIV class="wrap"><DIV class="div_vl backgroundimage"><H1 id="title">Project <I>Exe</I>pire</H1><DIV class="div_banner"><DIV class="col-xs-12 col-sm-8 subdiv_banner middle"><A href="#section_1" class="banner_link col-xs-6 col-sm-3"><SPAN class="oneline">Introduction</SPAN></A><A href="#section_2" class="banner_link col-xs-6 col-sm-3"><SPAN class="oneline">Method &amp; Results</SPAN></A><A href="#section_3" class="banner_link col-xs-6 col-sm-3"><SPAN class="oneline">Future</SPAN></A><A href="#section_4" class="banner_link col-xs-6 col-sm-3"><SPAN class="oneline">Protocols</SPAN></A></DIV></DIV></DIV><DIV class="col-xs-12 div_content"><DIV id="contentTitle">
			Introduction	
		</DIV><P id="pp">Bio-safety is an important issue in synthetic biology. Concerns surrounding synthetic organisms
 escaping into the environment have prompted the development of novel methods of bio-containment. 
 Many iGEM projects that require an organism to be released from the lab use kill switches to address 
 concerns about the effect of genetically modified organisms (GMOs) on the environment.  Unfortunately, kill switches - inducible genetic 
 devices that cause cell death - are poorly categorised in the standard registry of genetic parts. 
 There is a distinct lack of quantitative data which prevents them being used with confidence.</P><P id="pp">Our initial concerns surrounding the use of kill switches as bio-safety devices were centred 
around their efficiency, robustness over time and ability to prevent transfer of synthetic DNA into the 
wild population. We contacted <A href="http://www.synthace.com/2010/06/26/markus-gershater/">Dr Markus Gershater</A>, chief scientific officer at <I>Synthace Ltd</I>, to ask him 
what the application of kill switches might be in an industrial setting and what evidence would be satisfactory
 for their use. Dr Gershater gave the view that kill switches would not be as effective or economical as the
 physical and chemical bio-containment methods that <I>Synthace</I> currently employ. One of his concerns was that
 any leakiness in a system would provide a strong selection pressure against cells with fully functional kill 
 switches. In order for Dr Gershater to be satisfied, the kill switch would need to be tested across a wide 
 range of environmental conditions and still remain effective. He would also need to see high efficiency levels, 
 as in the context of large culture vessel, even a low proportion of cell survival would result in a large 
 population lacking an effective kill switch.</P><DIV><H5><Q style="margin:auto;display:block;"><I>Thereâ€™s no need for kill switches in the industry area that we operate. Even in the case where we might think there was a need for extra containment, the idea of putting in multiple systems to a production strain is concerning</I></Q><SPAN id="afQuote" style="padding-left:0;"> -Dr Markus Gershater, Synthace Ltd</SPAN></H5></DIV><P id="pp">We also contacted <A href="http://www.imperial.ac.uk/people/t.ellis">Dr Tom Ellis</A> of Imperial College London and 
 asked his opinion on kill switch reliability. He gave the view that combining multiple mechanisms could 
 greatly reduce organism escape rates after kill switch induction. Combining several kill switches was an
 approach we had discussed during the development phase of our project. Dr Gershater advised that the 
 different systems would need to be truly orthogonal. For example, in an industrial setting two different 
 kill switches that both rely on protein production could potentially be circumvented by the over expression
 of a useful enzyme that is being commercially produced.</P><DIV><H5><Q style="margin:auto;display:block;"><I>There is of course no such thing as completely (infinitely) efficient, so instead it is best to aim for the most efficient possible system - e.g. chance of escape less than 1 in 10<SUP>20</SUP>. This is not unrealistic because multiple mechanisms can be combined that each have less than 1 in a billion escape rates, which gets you into this territory.</I></Q><SPAN id="afQuote" style="padding-left:0;"> -Dr Tom Ellis. Imperial College London</SPAN></H5></DIV><P id="pp">We spoke to Professor Richard Titball leader of the Microbes and Disease research group at Exeter University. We asked him about potential applications of kill switches. He talked about how physical containment methods traditionally used in microbiology may have limitations when applied to vaccines, an area of his research, as they are administered to the population. He thought that if triggered by a specific environmental condition, a kill switch could be an elegant bio-containment solution as it is a system that can be finely tuned. In practice however he was skeptical that kill switches could be made reliable. Interestingly when we discussed the public perception of synthetic DNA and its potential release into the environment, Prof Titball believed that it was an issue that shouldn't be discussed by the scientific community alone, but that the public should be involved in the risk/benefit assessment of the use of genetically modified organisms. This prompted us to find ways to engage the public in order to better their understanding of synthetic biology and include them in the debate. You can see our interview with Prof Titball on the Exeter iGEM 2016 youtube channel.</P><DIV><H5><Q style="margin:auto;display:block;"><I>This is not a zero risk activity, on the other hand there are massive benefits that might be realised from GMOs. The only way I can see of moving us forward is to educate the public a bit more about the issues.</I></Q><SPAN id="afQuote" style="padding-left:0;"> -Prof Richard Titball. University of Exeter</SPAN></H5></DIV><P id="pp">After talking to individuals from industry and 
 academia about the strengths and limitations of kill switches, it was clear that there was not a consensus on their efficacy. We decided to investigate the effectiveness
 of different types of kill switch, to quantify their robustness after several generations and investigate
 the possibility for horizontal gene transfer (HGT). We aimed to test if multiple kill 
switches in a system would reduce failure rate and if integration into the genome would increase stability.</P><P id="pp">We have developed three types of kill switch to cover a broad range 
of strategies that may be employed in kill switch design:
<UL><LI>A metabolic kill switch that uses the production of reactive oxygen species to kill the cell</LI><LI>An enzymatic kill switch that uses the production of lysozyme (<I>Gallus gallus</I>)</LI><LI>And two systems that target DNA:</LI><LI style="padding-left: 20px;">A CRISPR based kill switch based on the work of Caliando and Voigt (2015) that targets specific sites for degradation</LI><LI style="padding-left: 20px;">A broad spectrum system that non-specifically degrades DNA mediated by expression of DNase 1</LI></UL></P><H3 id="culture">Continuous culture</H3><P id="pp">Before starting the project we spoke to <A href="http://biosciences.exeter.ac.uk/staff/index.php?web_id=Rob_Beardmore">Prof. Robert Beardmore</A> EPSRC Leadership Fellow in the Mathematical
 Biosciences at Exeter University. Much of his research has been into antibiotic resistance. We discussed how high
 selection pressure is applied by prolonged use of antibiotics and how kill switches may be analogous to this. 
 It is clear that cells which develop a mutation that inactivates the kill switch would be strongly selected for.
 It was estimated that functional loss of the kill switch would occur in a short amount of time as a result,
 and if this was the case, could have strong implications for kill switch longevity. To test this we decided
 to use a ministat (Fig. 4) to perform a continuous culture. The ministat was developed in the <I>Dunham Lab</I> at the 
 University of Washington (Miller <I>et al</I>, 2013). Each ministat chamber is fed from its own media container (Fig. 1)
 via a peristaltic pump (Fig. 2), with the culture volume set by the height of the effluent needle in the chamber (Fig. 3). Air is
 bubbled through flasks of water to hydrate it and then used to agitate the culture. Chambers were inoculated with
 freshly transformed <I>E. coli</I> BL21 (DE3) and samples taken to test if the kill switches were still viable.
 By simulating in miniature how a kill switch might behave in an industrial setting, the ministat provides a <A href="https://2016.igem.org/Team:Exeter/Proof">proof 
 of concept</A> for how a kill switch might be maintained in larger chemostats during a continuous culture. A protocol
 for running experiments in the ministat can be found <A href="#MinistatProt">here</A></P><DIV class="col-xs-12" style="margin:0;padding:0;"><DIV class="col-xs-12 col-sm-4 text-align: center"><SPAN class="caption">Fig. 1 Media container used to feed a single ministat chamber.</SPAN></DIV><DIV class="col-xs-12 col-sm-4 text-align: center"><SPAN class="caption">Fig. 2 Peristaltic pump</SPAN></DIV><DIV class="col-xs-12 col-sm-4 text-align: center"><SPAN class="caption">Fig. 3 Ministat chambers in heatblock and 1 litre Duran bottle used to collect effluent</SPAN></DIV></DIV><DIV class="row"><DIV class="col-xs-12 text-align: center"><DIV class="col-xs-6"><SPAN class="caption" style="padding: 5px 30% 5px 30%;">Fig. 4 Ministat running a preliminary 
		experiment to calibrate parameters such as dilution rate and temperature of the heat block. 50 ml burettes used
		here to accurately measure effluent levels. 1 litre Duran bottles were used for effluent collection in the main
		experiment due to greater volumes of effluent. </SPAN></DIV></DIV></DIV><P id="pp">Our own growth curve was performed to determine the maximum specific growth rate of <I>E. coli </I> 
BL21 (DE3) in our lab, but could not be conducted for a sufficient length of time to be accurate. A maximum specific 
growth rate value of 1.730 was used (Cox, 2004). The ministat must be run at a dilution rate less 
than maximum specific growth rate, this prevents the culture being washed out of the growth chambers. The dilution rate
 of the culture was calculated by measuring flow rate at a setting of 7.5 rpm on the peristaltic pump. For practical reasons 
 the pump could not be run faster than this due to the amount of media needed. The dilution rate was set at 0.2 which 
 produced cultures that grew at an average optical density (OD) of 3.47 for KillerRed samples, 3.64 for KillerOrange samples and 3.17 for 
 lysozyme samples. OD was measured daily with a Bug Lab OD scanner. When the same
 sample was measured in a tecan infinite 200 pro plate reader the Bug Lab showed readings approximately three times higher.
 The difference between the samples was consistent regardless of the method used to measure OD and OD measurements remained stable throughout the experiment.
 </P></DIV><DIV class="col-xs-12 div_content"><DIV id="contentTitle">
			Method and Results		
		</DIV><H3>Metabolic kill switch</H3><P id="pp">KillerRed (Fig. 5) and KillerOrange are homologues of Green fluorescence protein (GFP)
 which, when irradiated with green and blue light respectively, generate reactive oxygen species (ROS). KillerRed has been
 shown to effectively kill cells when exposed to green light (540â€“580 nm) and is much less effective under blue light
 (460â€“490 nm) (Bulina <I>et al</I>, 2006). KillerOrange effectively kills cells when exposed to 450-495nm (Sarkisyan 2015), the range that KillerRed does not. There is a Î²-barrel present in both these proteins. A water-filled channel that is in contact with a chromophore area and located at the cap of the said Î²-barrel is thought to confer these proteins with their phototoxic capabilities (Pletnev S <I>et al</I>, 2009).</P><DIV class="row"><DIV class="col-xs-12 text-align: center"><DIV class="col-xs-6"><SPAN class="caption" style="padding: 5px 30% 5px 30%;">Fig. 5 Structure of KillerRed showing the open channel which allows contact between the chromophore and the solvent.</SPAN></DIV></DIV></DIV><P id="pp">
The mechanism by which ROS kill cells isnâ€™t completely understood. However ROS have been shown to have various detrimental roles in cells such as the oxidation of thiols, ascorbate and proteins containing (Fe-S)4 clusters as well as reducing various transition metals (Farr and Kogama, 1991). Prolonged exposure and or high levels of ROS triggers apoptosis like mechanisms (Held, 2015).Our metabolic kill switches build on previous iGEM projects which have used the expression of highly phototoxic 
fluorescent proteins to kill the cells by exposing the culture to light. In 2013, the iGEM team from Carnegie Mellon developed a phage delivery system of the KillerRed gene, which was then expressed in the infected bacteria, killing it on exposure to light. Carnegie Mellon 2014 continued characterisation of KillerRed and its monomeric form Supernova by analysing their photobleaching characteristics. Neither team tested the longevity of the kill switch or provided details on the light intensity that the cultures were exposed to. We aim to quantify the length of time for which the kill switch remains viable and provide absolute values for the intensity of our light source. 

Firstly we improved KillerRed, an existing registry part, by codon optimising it for <I>E. coli</I> and 
improved its characterisation by exposing cultures expressing the protein to previously untested light intensity. We designed and submitted KillerOrange as a new part and tested it in the same way. We compared the phototoxicity of KillerRed and KillerOrange to the commonly used Red fluorescence protein (RFP). This was to determine how phototoxic KillerRed and KillerOrange are in relation to RFP and if prolonged excitation of a commonly used fluorescent protein is deleterious to bacterial growth. Once we had established the efficiency of KillerRed and KillerOrange, ministat chambers were
 inoculated with samples of each to determine the robustness of the kill switches over time.</P><H6>Method</H6><P id="pp">The following samples were tested for phototoxicity by exposing them to 1.2 mW/cm<SUP>2</SUP> of white light from a 4x8 LED array (Fig. 6) for 6 hrs. Samples were then spread plated and colony forming units (CFUs) were counted. All parts were carried on the pSB1C3 plasmid and transformed into <I>E. coli</I> BL21 (DE3). Samples that were induced were done so with Isopropyl Î²-D-1-thiogalactopyranoside (IPTG) to a final concentration of 0.2 nM.</P><P id="pp">Henceforth samples will be refered to as:</P><UL><LI>KillerOrange induced</LI><LI>KillerOrange not induced</LI><LI>KillerRed induced</LI><LI>KillerRed not induced</LI><LI>RFP</LI><LI>Control: BL21 (DE3)</LI></UL><P id="pp">
See <A href="#KRKOProt">protocol</A> for detailed method.</P><P id="pp">KillerRed is excited by green/yellow light (540-580 nm) and KillerOrange by blue light (460-490 nm). We constructed a box around the LED array to prevent ambient light entering. Access to the inside of the box was gained through an opening cut in the front. With help from <A href="http://emps.exeter.ac.uk/physics-astronomy/staff/rse204">Ryan Edginton</A>, we used a portable spectrometer (Ocean Optics USB2000+VIS-NIR-ES, connected to a CC3 cosine corrector with a 3.9 mm collection diameter attached to a 0.55 mm diameter optical fibre) to measure light spectra and absolute intensity in the visible range. </P><DIV class="row"><DIV class="col-xs-6" style="padding:5px 2% 5px 10%;margin:0;"><SPAN class="caption">Fig. 6 Light box and thermocouple</SPAN></DIV><DIV class="col-xs-6" style="padding:5px 10% 5px 2%;margin:0;"><SPAN class="caption">Fig. 7 A graph showing the relative intensities of the spectra for our light source</SPAN></DIV></DIV></DIV><H3>Metabolic Kill Switch: Results</H3><H6><U>Characterisation experiment</U></H6><P id="pp">The following graphs show the average percentage viable cells for induced and uninduced samples after 6 hrs of exposure 
to 1.2 mW/cm<SUP>2</SUP> of white light and an average temperature of 38.63 Â°C. CFU count for the control condition was treated as 100 % and viable cells calculated 
as a proportion of that value. CFUs were not counted above 300. Error bars represent
 the standard error of the mean. Little to no difference in CFU count is shown between the control (BL21 (DE3)) and kill switch samples when they are kept in the dark. There is a significant difference in the number of CFUs when the kill switch samples are exposed to light.
</P><DIV class="row"><DIV class="col-xs-6" style="padding:5px 2% 5px 10%;margin:0;"><SPAN class="caption">Fig. 8. Percentage viable cells after 6 hrs in the light box. BL21 (DE3) transformed 
			with pSB1C3 containing an RFP maker is compared to a control with no plasmid. 10 ml falcon tubes containing 
			4.5 ml of sample were covered in tin foil before being placed in the light box.</SPAN></DIV><DIV class="col-xs-6" style="padding:5px 10% 5px 2%;margin:0;"><SPAN class="caption">Fig. 9. Percentage viable cells after 6 hrs in the light box. BL21 (DE3) transformed 
			with pSB1C3 containing an RFP maker is compared to a control with no plasmid. 10 ml falcon tubes containing 
			4.5 ml of sample were placed label down in the light box to allow maximum exposure to the light.</SPAN></DIV></DIV><DIV class="row"><DIV class="col-xs-6" style="padding:5px 2% 5px 10%;margin:0;"><SPAN class="caption">Fig. 10. Percentage viable cells after 6 hrs in the light box. BL21 (DE3) transformed 
			with KillerRed (BBa_K1914003) is compared to a control with no plasmid. 10 ml falcon tubes containing 4.5 ml
			of sample were covered in tin foil before being placed in the light box.</SPAN></DIV><DIV class="col-xs-6" style="padding:5px 10% 5px 2%;margin:0;"><SPAN class="caption">Fig. 11. Percentage viable cells after 6 hrs in the light box. BL21 (DE3) transformed 
			with KillerRed (BBa_K1914003) is compared to a control with no plasmid. 10 ml falcon tubes containing 4.5 ml 
			of sample were placed label down in the light box to allow maximum exposure to the light.</SPAN></DIV></DIV><DIV class="row"><DIV class="col-xs-6" style="padding:5px 2% 5px 10%;margin:0;"><SPAN class="caption">Fig. 12. Percentage viable cells after 6 hrs in the light box. BL21 (DE3) transformed
			with KillerOrange (BBa_K1914001) is compared to a control with no plasmid. 10 ml falcon tubes containing 4.5 
			ml of sample were covered in tin foil before being placed in the light box.</SPAN></DIV><DIV class="col-xs-6" style="padding:5px 10% 5px 2%;margin:0;"><SPAN class="caption">Fig. 13.Percentage viable cells after 6 hrs in the light box. BL21 (DE3)
			transformed with KillerOrange (BBa_K1914001) is compared to a control with no plasmid. 10 ml falcon tubes 
			containing 4.5 ml of sample were placed label down in the light box to allow maximum exposure to the light.</SPAN></DIV></DIV><H6 id="MSE"><U>Ministat experiment</U></H6><P id="pp">All samples from the ministat were tested using the KillerRed, KillerOrange protocol found <A href="#KRKOProt">
here</A>. Glycerol stocks were made from the samples taken at each time interval, testing was done using these glycerol stocks.</P><P id="pp">The following graphs show the average number of colonies of samples taken at 0 h, 24 h, 120 h and 168 h of continuous culture and then tested in the light box. Values were 
averaged across three biological repeats. Colonies were not counted above 300 and so this is the maximum value given. All samples were induced to a final concentration of 0.2 nM IPTG. All samples were 
diluted 1000 times in a final volume of 4.5 ml liquid broth (LB). Error bars represent the standard error of the mean.</P><DIV class="row"><DIV class="col-xs-6" style="padding:5px 2% 5px 10%;margin:0;"><SPAN class="caption">Fig. 14. Comparison of CFUs formed by KillerRed exposed to light and kept in the dark for each sample taken from the ministat. The efficiency of the kill switch decreases over time as shown by the increasing number of CFUs.</SPAN></DIV><DIV class="col-xs-6" style="padding:5px 10% 5px 2%;margin:0;"><SPAN class="caption">Fig. 15. Comparison of CFUs formed by KillerOrange exposed to light and kept in the dark. The efficiency of the kill switch decreases over time as shown by the increasing number of CFUs. The effect is not as obvious in KillerOrange compared to KillerRed as the starting efficiency of KillerOrange is lower. </SPAN></DIV></DIV><H3>Enzymatic kill switch</H3><P id="pp">Lysozymes are a group of enzymes that are an important part of the immune response against bacteria 
(Myrnes et al, 2013). They are defined as 1,4-fl-N-acetylmuramidases that cleave the glycosidic bonds between carbon 1 of N-acetylmuramic acid  and carbon 4 of N-acetylglucosamine in the peptidoglycan that makes up a bacterial
 cell wall (JollÃ¨s and JollÃ¨s, 1984).
Lysozymes are commonly used in mass spectrometry for protein mass calibration and are also effective lysing agents
 against Gram-positive and to lesser extent, Gram-negative bacteria (Sigma aaldrich, 2016). UNICAMP-Brazil 2009 iGEM team used the Lysozyme <I>Gallus gallus</I> part BBa_K284001 previously and other lysis mechanisms have been used as kill switches by TU-Delft 2013, Newcastle 2010, Imperial College London 2011 and METU-Ankara 2011. As others team have used lysis mechanisms in their kill switches we thought Lysozyme (<I>Gallus gallus</I>) 
 would be a suitable candidate to test the effectiveness of lysis as a kill switch mechanism and investigate the
 potential for HGT if lysis is successful. We added an OmpA signal peptide to Lysozyme C which 
 targets it to the periplasm, this was to ensure that the enzyme would be translocated to the cell wall where it would be
 most effective. Our model predicted the complete degradation of the cell wall to be within the first generation of <I>E. coli</I> (details of the model can be found <A href="https://2016.igem.org/Team:Exeter/Model#section_3">here</A>). This gave us confidence that lysozyme would be effective. 

</P><H6>Method</H6><P id="pp">To show the activity of lysozyme, a molecular probes EnzCheck lysozyme assay kit (Thermo fisher 
scientific) was used. The coding sequence (CDS) contains an OmpA signal peptide targeting it to the perisplasm therfore lysozyme will
 only be detectable if the cells have lysed. The kit uses a substrate containing <I>Micrococcus lysodeikticus</I> 
 cell walls labelled with fluorescein to such as degree that fluorescence is quenched. The presence of lysozyme 
 causes a sharp increase in fluorescence by easing the quenching. The increase in fluorescence is proportional
 to lysozyme activity in the sample. The fluorescence assay was used to measure the activity of the freshly 
 transformed kill switch and that of the cultures grown in the ministat. CFUs were also used as a measure of
 efficiency by comparing the number of colonies to a control. 5 ml ovenights of <I>E. coli</I> BL21 (DE3) transformed 
 with pSB1C3 lysozyme were used to inoculate 250 ml Erlenmeyer flasks containing 50 ml of LB laced with 35 Âµg/ml 
 chloramphenicol. Once an OD of 0.23 was reached IPTG was added to a final concentration of 0.2 nM. Protein production
 was allowed to proceed for 2 hrs. The sample was serially diluted (10<SUP>-2</SUP>,10<SUP>-3</SUP>,10<SUP>-4</SUP>).
 200 Âµl of each dilution factor was spread plated and incubated at 37 Â°C overnight. CFUs were then compared to a control
 treated in the same way.</P><P id="pp">The potential for horizontal gene tranfser was tested using the lysozyme (<I>Gallus gallus</I>) 
provided in the EnzCheck lysosyme assay kit. Cells were lysed, the enzyme inactivated 
and then transformation of the resulting lysate performed. For a detailed protocol see <A href="#HGTProt">HGT protocol</A></P><H3>Enzymatic Kill switch: Results</H3><H6><U>Characterisation</U></H6><P id="pp">No difference in CFUs was observed between the control and the samples producing lysozyme. The results of the 
EnzCheck lysozyme assay were inconclusive. </P><H6><U>Horizontal Gene Transfer</U></H6><P id="pp">The HGT experiment showed that DNA present in lysate can be successfully transformed into a different 
<I>E. coli</I> strain with an average of 4 colonies per transformation (stdev=3.38). The BL21 (DE3) competent cells 
all gained the antibiotic resistance and RFP marker from the plasmid present in the lysed DH5Î±. 2 colonies from each
 plate were cultured over night and showed a fluorescence value concordant with that of the original culture. The 
 starting cultures of DH5Î± had an average starting OD of 1.11 and fluorescence value of 258 arbitrary units (A.U) before lysis. The BL21 
 (DE3) cultures transformed with the lysate had an average OD of 0.75 and average fluorescence of 306 A.U. None of the 
 spread plated lysate produced any colonies, showing that all cells were killed in the lysis reaction.</P><H3>DNase</H3><P id="pp">DNase I is a nonspecific deoxyribonuclease originally extracted from bovine pancreatic tissue. It degrades both
 double-stranded and single-stranded DNA resulting in the release of di-, tri- and oligonucleotide products with 5Â´
 -phosphorylated and 3Â´-hydroxylated ends (Vanecko, 1961). DNase I has also been shown to work on chromatin and DNA:RNA 
 hybrids (Kunitz, 1950). DNase I degrades these target polymer molecules through the hydrolytic cleavage of phosphodiester linkages in their backbone (Suck, 1986).</P><P id="pp">For a kill switch to be effective as a bio-containment device, the release of synthetic DNA must be mitigated.
 We aimed to do this in our project using the expression of DNase I. DNase I is commonly used in a laboratory 
 setting to degrade unwanted DNA. It was shown by Worrall and Connolly (1990) that expression of DNase I is 
 possible in <I>E. coli</I> as long as it is under the control of a promoter with a strong off state. We constructed a 
 part with DNase I under the control of the T7 promoter. Unfortunately no transformations were successful and all colonies
 produced contained empty plasmid backbone. Worral and Connolly reported that a promoter which is less tightly regulated
 (pKK223-3) would result in transformation failure. As was shown in our metabolic kill switch, the T7 promoter we used to 
 control expression of the CDS is very leaky. This is likely the reason why transformations were unsuccessful as immediately
 after transformation, production of DNase I would commence killing all the cells. If this is the case future work on a
 system that uses DNase I as a kill switch but under much tighter control, may prove very effective. </P><H3>Discussion</H3><H6>Metabolic Kill Switch: KillerRed and KillerOrange.</H6><P id="pp">We have shown that KillerRed and KillerOrange can effectively kill cells (Fig. 11 &amp; 13) under much lower light intensity 
than is used in the literature (Sarkisyan <I>et al</I>, 2015). On investigation into the kind of light source that was needed to produce
 the 1 W/cm<SUP>2</SUP> of previous experiments (Bulina <I>et al</I>, 2005), it became clear that 1 W/cm<SUP>2</SUP> 
 was impractically bright and would generate large amounts of heat which would kill <I>E. coli</I>. We decided to use a much less powerful LED array that produces 1.2 mW/cm<SUP>2</SUP> at the wavelengths most effective for KillerRed and KillerOrange (Fig. 7) and expose our samples to light for a greater length of time. We showed that this was still effective with an average
 survival rate in the + IPTG condition of 2.2% for KillerRed (Fig. 11) and 12.7 % for KillerOrange (Fig. 13). A wider range of exposure times
 and light intensities would greatly improve the characterisation of these parts, unfortunately time limitations prevented 
 us from testing this. The leakiness of the T7 promoter used in our kill switches was quantified by comparing protein production in the + IPTG condition and - IPTG condition. A one tail t-test assuming equal variance was performed for the mean fluorescence values of the cultures tested in the light box. Fluorescence was used as a measure of protein production. No statistically significant difference was found between the + IPTG condition and â€“ IPTG condition (a significance value of &lt; 0.05 was used. p-value for KillerRed 0.18, p-value for KillerOrange 0.16). CFU counts 
 for + IPTG conditions were within the standard error of â€“ IPTG (Fig. 11 &amp; 13). For KillerRed the induced kill switch appears to be more 
 effective whereas the uninduced switch is more effective in KillerOrange. The leakiness of the T7 promoter has likely
 lead to near equal expression in both conditions, possibly exacerbated by the length of time that the cultures were left 
 to grow in order for the protein to fully mature. The literature showed that cells had been kept in a cold room at 4 
 Â°C for 24 hrs before exposing the samples to light (Sarkisyan <I>et al</I>, 2015), the reason given for this was to &quot;increase the fraction of mature protein&quot;. We tested the validity of this as cultures were incubated at 37 Â°C 220 rpm overnight not 4 Â°C and the phototoxicity of KillerRed and KillerOrange was still evident. The light box itself had a negative effect on <I>E. 
 coli</I> growth. In our experiment each sample was first diluted to 10<SUP>-3</SUP>,10<SUP>-4</SUP> and 10<SUP>-5</SUP> before exposure 
 to light. The dark condition for the control formed a lawn of bacteria on the agar plate regardless of the starting dilution factor, however in the light condition, the 10<SUP>-3</SUP> dilution produced the same amount of colonies as the dark but in greater dilutions the number of colonies decreased. This decrease was not significant enough to have affected the results but it should be noted that exposure to 1.2 mW/cm<SUP>2</SUP> for 6 hrs slows the growth rate of <I>E. coli</I> BL21 DE3.</P><P id="pp">The continuous culture of KillerRed showed a 15 fold increase in the percentage of viable cells after 168 hrs. A similar pattern is shown for KillerOrange but with around a two fold increase. Both KillerRed and KillerOrange show greater numbers of colonies forming over time (Fig. 14 &amp; 15). This number approaches the amount produced in the dark condition by 168 hrs.
The average fluorescence reading for 0 hr KillerRed samples was 506.3 A.U (recorded at an average OD of of 0.745). After 168 
hrs the average fluorescence reading  was 436 A.U (at an average OD of 0.96). It seems unlikely due to the readings being 
similar that a mutation has occurred in the kill switch itself. As fluorescence is proportional to the amount of ROS 
being produced, up regulation of native <I>E. coli</I> enzymes that mitigate the effects of ROS may be the cause of the increase 
in cell survival. Future transcriptome analysis could provide interesting data on the mechanism of this change, this was 
unfortunately beyond the scope of this project. This shows that there may be many ways for bacteria to circumvent the effects of a kill switch given the high selection pressure they pose.</P><H6>Enzymatic Kill Switch: Lysozyme</H6><P id="pp">The Enzcheck fluorescence assay used to determine lysozyme activity produced values that were not consistent
 with the CFU count of the plated sample. The HGT experiment we conducted showed that 50 Âµl of 500 U/ml lysozyme C normally
 used in the EnzCheck assay to produce a standard curve would effectively kill all the cells in a 50 Âµl sample of DH5Î±. 
 The assay showed that 20x diluted sample produced near 500 U/ml activity readings yet this culture would still produce 
 a lawn of bacteria when 200 Âµl was spread plated. It is noted that the standard curve was of poor quality. 
 The samples of lysozyme were assayed in the same way after continuous culture and did show a decrease in lysozyme activity over time, however the original readings that were used as a comparison have an error of sufficient size that this is not conclusive. 
 The CFU count for lysozyme showed no difference from the control. Lysozyme added to a sample extra-cellularly was shown to 
 lyse all the cells in our HGT experiment, even though Gram-negative bacteria are partially protected from its action due to 
 their outer membrane (Callewaert, 2008). Yet lysozyme produced intra-cellularly and targeted to the periplasm was not 
 effective. There may have been issues with translocation of the protein to the target area, however this seems unlikely 
 due to the effectiveness of its extracellular action. Another explanation may be that lysozyme as a kill switch mechanism 
 is inherently ineffective, as any cells that are resistant will proliferate and a population that are not affected by its
 production very quickly develops. Regardless of the ability of lysozyme to kill cells effectively, we have shown that HGT
 is a concern with using this form of kill switch; antibiotic resistance markers commonly used in synthetic biology can be 
 transferred to a different strain of <I>E. coli</I> and in principle any wild type organisms outside the lab. Therefore kill switches that use lysis as a mechanism for cell death should not be considered bio-containment.</P></DIV><DIV class="col-xs-12 div_content"><DIV id="contentTitle">
			Future
		</DIV><H3>Failsafe</H3><P id="pp"> One area that we were unable to explore was the incorporation of multiple kill switches into the same system. 
Initially we aimed to construct an operon that contained KillerRed and  KillerOrange. This was unfeasible with the cloning
 strategy that we were using as the overhangs that join the ribosome binding site (RBS) to the CDS would not differentiate between KillerRed and 
 KillerOrange. Constructing KillerRed and KillerOrange on plasmid backbones with different antibiotic resistance markers
 would allow both to be transformed together. This is a simpler way to test the hypothesis and would be interesting for 
 the future.</P><H3>Genome integration</H3><P id="pp">While plasmids are widely used to carry genetic parts, integration into the host genome

could prove a more robust approach to introducing genes into an organism. Genome

integration removes the need for a selectable antibiotic resistance marker as parts

will be faithfully replicated and the variability of copy number is removed. We aimed to 

investigate whether integration into the <I>E. coli</I> genome will affect the efficiency of our

kill switches and whether they would remain functional for longer in a continuous culture. We aimed to use the lambda red 
recombination method to integrate our parts into the <I>ars</I>B locus using the pKD4 plasmid as a vector. Integrating at 
<I>ars</I>B has been shown not to affect <I>E. coli</I> growth (Sabri <I>et al</I>, 2013). However the pKD4 plasmid contained
 illegal EcoRI and XBal restriction sites.

To resolve this we decided to carry out site directed mutagenesis to change one nucleotide in each sequence 
of the restriction sites. Primers were designed for use with the Q5 site directed mutagenesis kit. The first attempt 
using this kit involved a 2 step PCR reaction, this was shown by gel electrophoresis of the product to have been 
unsuccessful. The protocol was changed to a 3 step PCR reaction and a successful product was produced. The PCR
 product underwent a KLD reaction and was transformed into <I>E. coli</I> DH5Î±. 

The transformation was unsuccessful and so mutagenesis was carried out again and re-transformed. Each time the
 transformation was unsuccessful. We attempted to use an Agilent Quick change multi site-directed mutagenesis kit to 
 remove the illegal sites. Unfortunately, this kit was also unsuccessful. We then learned that the strains of <I>E. 
 coli</I> that we had available would not be able to replicate the pKD4 plasmid and this was the reason for our failed 
 transformations.  </P><H3>DNA degradation</H3><P id="pp">We had hoped to develop a CRISPR based kill switch building on the work of Caliando and Voigt (2015).
 We designed the spacer array to target three essential genes <I>polA</I>, <I>rpoC</I> and <I>topA</I> using the 
 <I>deskgen</I> platform. We selected three protospacers within the CDS of each essential gene. The cleavage sites were 
 designed to be in the first third, the centre third and the final third of the CDS. The spacer array was designed
 to be carried on the pSB1C3 plasmid under the control of a constituitive promoter (BBa_J23100). Our aim was to 
 investigate how many essential genes would be needed for the kill switch to be effective, whether some genes 
 were more effective targets than others and whether targeting multiple protospacers simultaneously was more
 effective than a single cleavage site. We designed primers to obtain the Cas9 and tracr RNA sequence from BBa_K1218011
 provided in the distribution kit. After several attempts transformations remained unsuccessful. The spacer array could 
 also not be produced as a G-block due to the high number of repeating sequences. A CRISPR based kill switch was shown 
 by Caliando and Voigt (2015) to be stable for many months when integrated into the genome at multiple loci. Making this 
 system available to iGEM teams could greatly improve on the short comings we have shown in the stability of toxic protein 
 based switches carried on plasmids. If a system of this kind were to target just the synthetic DNA that had been introduced into the organism, the system would prevent release of synthetic DNA into the environment without the high selection pressure of death. This would potentially allow the switch to be retained for longer. </P><H3>Ministat</H3><P id="pp">The modularity of the ministat allows several environmental conditions to be tested simultaneously.
 Future studies that would build on the work started in this project should include the testing of different
 media types, growth over different temperature ranges and cultures grown at a range of dilution rates.</P><H6>References</H6><DIV class="wrap" style="font-size:60%"><UL><LI>Bulina, M.E., Chudakov, D.M., Britanova, O.V., Yanushevich, Y.G., Staroverov, D.B., Chepurnykh, T.V., Merzlyak, E.M., Shkrob, M.A., Lukyanov, S., Lukyanov, K.A. (2006).<I> A genetically encoded photosensitizer</I>. Nature Biotechnology.24(1).</LI><LI>Caliando, B.J., Voigt, C.A. (2015). <I>Targeted DNA degradation using a CRISPR device
stably carried in the host genome</I>. Nature communications, 6:6989.</LI><LI>Callewaert, L., Aertsen, A. , Deckers, D., Vanoirbeek, K.G.A, Vanderkelen, L., Herreweghe, J.V.M., , Masschalck, B. , Nakimbugwe, D. , Robben, J. ,  Michiels, C.W. (2008).<I> A New Family of Lysozyme Inhibitors Contributing to Lysozyme Tolerance in Gram-Negative Bacteria</I>. PLoS Pathogens, 4(3): e1000019. doi:10.1371/journal.ppat.1000019.</LI><LI>Carpentier, P., Violet, S., Blanchoin, L., Bourgeois, D. (2009).<I>Structural basis for the phototoxicity of the fluorescent protein KillerRed.</I>. FEBS letters, 583, pp 2839-2842.</LI><LI>Cox, R.A. (2004).<I>Quantitative relationships for speci?c growth rates and macromolecular compositions of Mycobacterium tuberculosis, Streptomyces coelicolor A3(2) and Escherichia coli B/r: an integrative theoretical approach.</I>. Microbiology, 150, pp 1413-1426.</LI><LI>Farr, S.B. and Kogama, T. (1991).<I>Oxidative Stress Responses in Escherichia coli and Salmonella typhimurium</I>. Microbiological Reviews, 55(4), pp 561-585.</LI><LI>Held, P. (2014),<I> An introduction to reactive oxygen species</I>[Online]. Available at: http://www.biotek.com/resources/articles/reactive-oxygen-species.html [Accessed: 16/8/16].</LI><LI>JollÃ¨s, P. and JollÃ¨s, J., (1984).<I> What's new in lysozyme research?</I>. Molecular and cellular biochemistry, 63(2), pp.165-189.</LI><LI>Kunitz, M.L., (1950). <I>Crystalline desoxyribonuclease I. isolation and general properties spectrophotometric method for the measurement of desoxyribonuclease activity.</I> The Journal of general physiology, 33(4), pp. 349-362.</LI><LI>Miller, A. W., Befort, C., Kerr, E. O., Dunham, M. J. (2013) <I>Design and Use of Multiplexed Chemostat Arrays.</I> Journal of Visualised Experiments. 72:e50262, doi:10.3791/50262.</LI><LI>Myrnes, B., Seppola, M., Johansen, A., Ã˜verbÃ¸, K., Callewaert, L., Vanderkelen, L., Michiels, C.W.,  Nilsen, I.W. (2013). <I>Enzyme characterisation and gene expression profiling of Atlantic salmon chicken-and goose-type lysozymes.</I> Developmental &amp; Comparative Immunology, 40(1), pp.11-19.</LI><LI>Sabri, S., Steen, J.A., Bongers, M., Nielsen, L.K., Vickers, C.E.<I>Knock-in/Knock-out (KIKO) vectors for rapid
integration of large DNA sequences, including whole metabolic pathways, onto the Escherichia coli chromosome at
 well-characterised loci</I>. Microbial Cell Factories, 12:60.</LI><LI>Sarkisyan, K.S., Zlobovskaya, O.A., Gorbachev, D.A, Bozhanova, G., Sharonov, G.V., Staroverov, D.B., Egorov, E.S., Ryabova, A.V., Solntsev, K.M., Mishin, A.S., Lukyanov, K.A. (2015).<I> KillerOrange, a Genetically Encoded Photosensitizer Activated by Blue and Green Light</I>.
 PLoS ONE,10(12).</LI><LI>Sigma-Aldrich (2016). <I>Lysozyme from chicken egg white for Molecular Biology</I>[Online]. Available at: https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Datasheet/7/l7651dat.pdf [Accessed 1/9/16].</LI><LI>Suck, D. and Oefner, C., (1986).<I> Structure of DNase I at 2.0 Ã… resolution suggests a mechanism for binding to and cutting DNA.</I>. Nature, 321, pp 620-625.</LI><LI>Vanecko, S. and Laskowski, M., (1961).<I> Studies of the Specificity of Deoxyribonuclease I III. Hydrolysis of Chains Carrying a Monoesterified Phosphate on Carbon 5'</I>. Journal of Biological Chemistry, 236(12), pp.3312-3316.</LI><LI>Worral, A.F., Connolly B.A. (1990),<I>The Chemical Synthesis of a Gene Coding for Bovine Pancreatic DNase I and Its Cloning and Expression in Escherichia coli.</I> The Journal of Biological Chemistry, 265(35), pp 21889-21895.</LI><LI>Weber, E., Engler, C., Gruetzner, R., Werner, S., &amp; Marrillonnet, S. (2011). A Modular Cloning System for Standardized Assembly of Multigene Constructs. PLoS ONE, 6(2), e16765.</LI></UL></DIV><DIV class="col-xs-12 div_content"><DIV id="contentTitle">
			Protocols
		</DIV><H6>Transformation of competent <I>E. coli</I> cells</H6><OL><LI>Defrost one 100 Âµl (or 200 Âµl if available) aliquot of <I>E. coli</I> competent cells per transformation,
			plus one extra as a control (use whole tube and do not refreeze).
</LI><LI>Add 1-5 Âµl of plasmid DNA (depending on concentration) to competent cells, mix by rolling or flicking 
			the eppendorf.
</LI><LI>Incubate on ice for 40 mins.</LI><LI>Heat shock in eppendorf at 42 Â°C for 2 mins.</LI><LI>Incubate on ice for 5 mins.</LI><LI>Add 0.7 ml of pre-warmed LB medium, incubate at 37 Â°C, 200-220 rpm for 45-60 mins. </LI><LI>Spin down cells at 8000 rpm for 5 mins.</LI><LI>Remove 0.5 ml of the supernatant.</LI><LI>Resuspend cells in remaining supernatant.</LI><LI>Plate out 200 Âµl on agar laced with 100 Âµg/ml ampicillin, 50 Âµg/ml kanamycin, 35 Âµg/ml chloramphenicol
			or 15 Âµg/ml tetracycline as appropriate.</LI><LI>Incubate at 37 Â°C overnight.</LI></OL><H6 id="KRKOProt">KillerRed KillerOrange protocol</H6><OL><LI>Prepare 5 ml overnight cultures of KillerRed, KillerOrange and RFP transformed into <I>E.coli</I>
			BL21 (DE3). This can be done from a glycerol stock or fresh transformation. Place in 37 Â°C 220 rpm 
			incubator overnight.</LI><LI>Prepare 250 ml erlenmeyer flasks with 50 ml of LB broth and add 50 Âµl of 35mg/ml chloramphenicol 
			to a final concentration of 35 Âµg/ml. Cover the flasks in tin foil.</LI><LI>Measure the OD of the overnight cultures and inoculate the 250 ml flasks to a starting OD of 0.1
			on a tecan infinite 200 pro plate reader. </LI><LI>Once an OD of 0.23 has been reached, induce + IPTG samples with 100 Âµl of 0.1 M IPTG to a final 
			concentration of 0.2 nM. Incubate at 37 Â°C 220 rpm overnight.</LI><LI>Measure the OD and fluorescence of each culture.</LI><LI>Perform a serial dilution of the samples by adding 5 Âµl of undiluted culture to 4995 Âµl of LB
			broth in a 10 ml falcon tube and invert 5-10 times, this is the 10<SUP>-3</SUP> culture. Take 500 
			Âµl of the 10<SUP>-3</SUP> and add it to 4500 Âµl of LB broth, invert the tube 5-10 times, this is t
			he 10<SUP>-4</SUP>. Finally take 500 Âµl of the 10<SUP>-4</SUP> and add it to 4500 Âµl of LB broth, 
			invert the tube 5-10 times, this is the 10<SUP>-5</SUP> culture. Remove 500 Âµl from the 10<SUP>-5</SUP>
			culture, all falcon tubes will contain 4500 Âµl of sample.  Repeat this twice for all the samples 
			(KillerRed induced, KillerRed not induced, KillerOrange induced, KillerOrange not induced, RFP and control:
			BL21 (DE3). One set of samples should be covered in tin foil, the other set left uncovered.</LI><LI>Place the falcon tubes in the light box, label down for the uncovered samples.</LI><LI>Expose for 6 hrs, take the temperature inside the box periodically using a thermocouple.</LI><LI>Spread plate 200 Âµl of each sample and incubate at 37 Â°C overnight.</LI><LI>Count the number of colonies on each plate.</LI></OL><H6 id="MinistatProt">Ministat Protocol</H6><OL><LI>Prepare the desired number of media containers and culture chambers ready to be autoclaved.</LI><LI>Add media to the containers. Add 100 Âµl of inoculum to the sterile culture chamber. Connect the
			media container to the culture chamber.</LI><LI>Arrange the ministat array in the heat block and set the effluent volume to around 35 ml by moving
			the effluent needle up to the desired height in the chamber.</LI><LI>Set the peristaltic pump to 90 rpm and allow the chambers to fill. Once at 35 ml turn off the
			peristaltic pump. Turn on the air pump to start aeration and agitation. Let the culture grow for 24 hrs</LI><LI>Turn on the peristaltic pump to 7.5 rpm this will give a flow rate of approx 4 ml/hr. Ensure
			that the effluent tubes are uncovered and inside a collecting vessel. </LI><LI>Each day take a sample in a sterile container for testing. Glycerol stocks should be taken.
			These can then be tested later.</LI></OL><H6 id="HGTProt">HGT Protocol</H6><OL><LI>Prepare a 5 ml overnight of pSB1C3 RFP transformed into <I>E. coli</I> DH5Î± in a 10 ml falcon tube.</LI><LI>Prepare working solution of lysozyme in reaction buffer from a molecular probes EnzCheck lysozyme 
			assay kit by diluting 1000 U/ml lysozyme C (Galus Galus) two fold in reaction buffer. 50 Âµl of working 
			solution is needed per reaction. </LI><LI>Add 50 Âµl of working solution to 50 Âµl of overnight culture in a PCR tube incubate at room temp 
			for 30 mins</LI><LI>Incubate overnight at 55 Â°C (this step is critical to ensure all lysozyme is inactivated)</LI><LI>Perform full transformation protocol using 3 Âµl of the lysate with BL21 (DE3) competent cells.</LI><LI>Plate the remaining lysate onto a chloramphenicol plate as a negative control.</LI><LI>Count colonies, overnight these and measure fluorescence.</LI></OL><H6 id="COMPcellsprot">Preparation of Competent Cells protocol</H6><OL><LI>Scrape a few cells from an E. coli strain, straight form the -80Â°C freezer onto a marked space on an L-agar plate. (N.B. no ampicillin!) Streak out three times with a wire and grow overnight at 37ÂºC</LI><LI>Remove one colony from the plate and re-streak onto a second plate with a wire and again grow overnight at 37ÂºC.</LI><LI>Inoculate 5 ml LB with one colony and incubate overnight at 37ËšC.</LI><LI>Inoculate 40 ml of LB in a 250 ml Monod flask with 0.4 ml of the overnight culture. Grow at 37ËšC and 200 rpm until A650 = 0.4-0.5.</LI><LI>Transfer the culture to a 40 ml Falcon sterile tube and harvest by centrifugation at 8,000 rpm for 8 minutes at 4ËšC.</LI><LI>Drain the pellet and re-suspend cells in 8 ml of Transformation buffer, TF-1.</LI><LI>Place on ice for 15 minutes and then spin as above.</LI><LI>Thoroughly drain the pellet and re-suspend in 4 ml of TF-2.</LI><LI>Competent cells can be kept on ice for a few hours or aliquoted (0.2 ml) and frozen at â€“70ËšC immediately.</LI><P>Modified from Hanahan D. (1985) in â€˜DNA cloning 1â€™ Ed D. M. Glover, pp 109-135, IRL Press (ISBN 0-947946-18-7).</P></OL></DIV></DIV></BODY></HTML>