<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Exeter_Labbook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Exeter/Labbook</SPAN></H1></DIV></DIV></DIV><NAV id="Top" class="navbar navbar-inverse navbar-static-top"><DIV class="container-fluid"><DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI><DIV id="links_drop" class="dropdown"><BUTTON class="dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"><SPAN style="margin-bottom:4px;">Lab</SPAN></BUTTON></DIV></LI><LI><A id="links" style="margin:10px 0 30px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Project">Lab Project</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Labbook">Lab Book</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Safety">Safety</A></LI></UL></DIV><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Parts">Parts</A></LI><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Team">Team</A></LI><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Interlab">InterLab</A></LI><LI><DIV id="links_drop" class="dropdown"><BUTTON class="dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"><SPAN style="margin-bottom:4px;">Human Practices</SPAN></BUTTON></DIV></LI><LI><A id="links" style="margin:10px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Integrated_Practices">Integrated</A></LI><LI><A id="links" style="background:none;line-height:0.7vh;margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Engagement">Public Engagement&amp; Education</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Log">Log</A></LI></DIV><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Attributions">Attributions</A></LI><LI><DIV id="links_drop" class="dropdown"><BUTTON class="dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"><SPAN style="margin-bottom:4px;">Awards</SPAN></BUTTON></DIV></LI><LI><A id="links" style="margin:10px 0 30px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/Awards">Awards</A></LI><LI><SPAN style="margin:10px 0 30px 2px;padding:0;"><U>Special pages</U></SPAN></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/HP/Silver">HP Silver</A></LI><LI><A id="links" style="margin:30px 0 10px 2px;padding:0;font-size:1.8vh;" href="https://2016.igem.org/Team:Exeter/HP/Gold">HP Gold</A></LI></DIV><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Model">Models</A></LI><LI><A id="links" href="https://2016.igem.org/Team:Exeter/Collaborations">Collaborations</A></LI></NAV><DIV id="scrollPosition" style="display:none;">0</DIV><DIV class="container"><DIV class="col-xs-12 div_content"><H1>Lab Book</H1><H2>13/07/16</H2><H6>Team: Dan, Eloise, Hannah, Jack, Pablo</H6><P id="pp">Our first day in the lab working on our project began by resuspending KillerRed part from the iGEM distribution kit in 10μl of MQ water and left for 5 mins.
 The T5 promoter (BBa_K592008), Double terminator (BBa_B0015), RBS (BBa_B0034), non codon optimized KillerRed (BBa_K1184000),
 Strong, Medium and Weak Constitutive Promoters (BBa_J23101,BBa_J23110,BBa_J23103) 1μl of resuspended DNA was then transformed. 
 We plated on the corresponding antibiotic and LB plates and left the plates overnight in the non-shaking 37℃ incubator.</P><H2>14/07/16</H2><H6>Team: Alice, Dan, Eloise, Pablo</H6><P id="pp"> The transformation of resuspended parts from the previous day were not completely successful. 
We did have growth of 1-2 colonies from the constitutive and T5 promoters, 
as a result we decided to overnight these colonies but also re-transform the DNA to produce more colonies 
for the remaining parts. We believe the transformation was unsuccessful due to the concentration of the resuspended
 DNA being too low so we transformed a higher volume of 5μl.</P><H2>15/07/16</H2><H6>Team: Dan, Eloise, Emily, Jack, Pablo</H6><P id="pp"> The transformations were successful with the higher volume of DNA, 
consequently made overnights of these colonies at the end of the day.
 We were able to Mini prep
 the promoters from the first transformations which had been overnighted the day before.
 The strength of the promoter produced a significant colour difference between each overnight culture; 
 the strong promoter was red, medium was orange and the weak promoter formed a yellow culture.

Today we blunted our KillerOrange G-block into pJet using an empty backbone as our control. 

Resuspension: 20μl elution buffer to give 50mg/μl (pJet).
Working with 3:1 ratio, use 3μl of insert to 1μl of backbone

Composition of KillerOrange solutions detailed in table below.</P><TABLE><TBODY><TR><TH>Solution A (μl)</TH><TH>Solution B (μl)</TH></TR><TR><TD>Reaction buffer</TD><TD>10</TD><TD>10</TD></TR><TR><TD>PCR product</TD><TD>1</TD><TD>3</TD></TR><TR><TD>pJet</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Water</TD><TD>7</TD><TD>5</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD></TR></TBODY></TABLE><P id="pp">We transformed 5μl of KO_A and KO_B. </P><H2>18/07/16</H2><H6>Team: Dan</H6><P id="pp">Glycerol stocks of the transformed <I>E. coli</I> were made using 500μl 50 % glycerol and 500μl <I>E. coli </I>
overnight culture. Mini preps were then made and stored in the freezer.</P><H2>20/07/16</H2><H6>Team: Dan, Eloise, Jack</H6><P id="pp">Q5 site-directed mutagenesis was used to remove restriction site EcoRI from the pKD4 plasmid 
available in the lab for Lambda red recombination. The primers ordered for the Q5 kit were resuspended according
 to the protocol and then diluted to achieve 10μM concentration. The template pKD4 plasmid DNA was also diluted as only 
 10ng were required for the entire reaction.
The Q5 protocol was followed as a two step PCR reaction.
 Extension was run for 105 secs (3 x 30sec per kbp as pKD4 is 3267 bp, add 15 sec for extra time)
 
Transformed Q5 PCR products into DH5α <I>E. coli</I>.</P><H2>21/07/16</H2><H6>Team: Dan, Eloise, Jack</H6><P id="pp">Our first Q5 site-directed mutagenesis attempt was unsuccessful 
yesterday and so we carried out the procedure again using a three step PCR reaction by changing round 
3 to 70&amp;#8451 for annealing.
An 0.8 % agarose gel was made to check that the PCR product was inclusive of the correct size
 pKD4 3267 bp plasmid. 
 
Also today the Qubit was used to quantify the concentration of mini prep DNA.

Jack spent today carrying out a biobrick digestion and ligation reaction to join the resuspended parts from the registry.
 We are following a multi step process in which the first digestion and ligation is used to ligate the promoter and RBS 
 as one part and the KillerRed protein coding region with the terminator as a second part. These two initial parts must 
 then be ligated to form the entire promoter, RBS, coding region and terminator sequence. Today's work involved the initial
 digestion and ligation using a tetracycline plasmid backbone. The PCR product produced was then transformed into DH5α</P><H2>22/07/16</H2><H6>Team: Jack, Eloise, Hannah, Dan, Joel, Emily</H6><P id="pp">Yesterday's digest-ligation transformation did not work.
 We repeated the digestion with RFP-tetracycline plasmid given to us by our supervisor and produced more transformation
 controls to validate our experimental procedure.
Q5- site directed Mutagenesis
The PCR and gel electrophoresis were successful yesterday. 
However, the PCR product was not successfully transformed into DH5α.
 It was later discovered that this is due to the DH5α strains inability to replicate the pKD4 plasmid. 
All of the PCR product remaining was loaded onto the gel so the reaction had to be repeated.</P><H2>25/07/16</H2><H6>Team: Dan, Eloise, Hannah</H6><P id="pp">The successful transformation of the ligation products promoter and RBS,
 KillerRed and terminator were overnighted and then today the Mini-prep protocol was followed to retrieve this DNA. Qubit was then used to find the concentrations.
MYE media was also made for future ministat experiments testing different types of media for continuous
 culture of kill switches.
Transforming the Q5 PCR product was continued today using a new KLD enzyme mix.

Cas9 protein was resuspended using too much water from the 2016 distribution kit so the correct amount
 of resuspension was added to our left over 2015 distribution kit. This was then transformed into DH5α.</P><H2>26/07/16</H2><H6>Team: Dan, Jack, Emily</H6><P id="pp">Digestion and ligation of promoter + RBS, and CDS + terminator was carried out to 
join together the next set of ligations to create the full part for non codon optimized KillerRed. 
The ligation product was then transformed.</P><H2>27/07/16</H2><H6>Team: Eloise, Emily, Hannah, Dan</H6><P id="pp">The new cloning protocol MoClo (Modular Cloning) requires the use of the ccdB gene which is
 toxic to all but one strain of <I>E.coli</I>. We are using this cloning method to create our codon optimized
 KillerRed and KillerOrange for characterisation and part for the registry.
 

An adjusted digestion and ligation protocol was followed to cleave RFP from the template backbones with each antibiotic 
resistance (Ampicillin = A, Chloramphenicol = C, Tetracycline = T and Kanamycin = K) and the ccdB gene from the pS797 
plasmid by digestion using enzymes with EcoR1 and PstI restriction sites. These digestion products were then run on a 
gel so as to separate the plasmids and RFP/ ccdB genes.
The Promega Wizard SV Gel and PCR Clean- Up system was used to extract and obtain the separated DNA from the gel
 to be used for ligation. The ccdB gene was then ligated into each plasmid. Once the 
 ligation was complete, the product was transformed into the ccdB survival strain from invitrogen.

The ligation protocol temperatures were altered for incubation at 37℃ for 15 mins
then heating at 65℃ for 20 mins.

</P><H2>28/07/16</H2><H6>Team: Dan, Jack, Pablo, Emily, Joel, Hannah</H6><P id="pp">The ligated KillerRed parts were mini prepped and the single successful Cas9 
colony was overnighted. Unsuccessful ccdB gene work was re-transformed as it was assumed that ampicillin 
and kanamycin backbones were plated onto the opposite plates as only the tetracycline and chloramphenicol 
plasmid backbones grew colonies. 
An OD growth curve was produced for positive controls of 3 backbones, A,C,K.
KillerRed ligations were sent off for sequencing to check that we made the part in the correct order sequence.</P><H2>29/07/16</H2><H6>Team:Jack, Emily, Joel, Hannah</H6><P id="pp">Glycerol stock, mini prep and qubit of Cas9 part were completed.</P><H2>01/08/16</H2><H6>Team: Jack, Pablo, Eloise, Hannah</H6><P id="pp">The ccdB plasmid backbones that were not successfully transformed were tried again. 
This required more ccdB gene to ligate with gel extracted DNA. However we were unsuccessful in
 amplifying enough PCR product of the gene because the digested pS797 plasmid was in too low concentration.
 This was determined by again qubitting the pS797 stocks, we will use the highest concentration for the next digestion.
Successfully transformed pSB1T3 ccdB and pSB1C3 ccdB plasmid backbones were overnighted to be used for cloning </P><H2>02/08/16</H2><H6>Team: Dan, Jack, Eloise, Hannah</H6><P id="pp">More mini prep and qubit was carried out.
The QuickChange (QC)Multi kit was used today instead of the Q5 kit which had not been successful. 
This kit is different in that up to 5 sites can be mutated at once, we are using it to swap three
 separate base pairs in one EcoR1 site and two Xbal sites within the pKD4 plasmid. It involves three 
 forward primers and three reverse primers being added to separate PCR reaction tubes.

Also today we began using the MoClo cloning technique to make codon optimized KillerRed and killerOrange parts. The resulting PCR product was transformed into the S171 E.coli strain.</P><H2>03/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo, Joel, Hannah</H6><P id="pp">Although production of pSB1T3 ccdB and pSB1C3 were successful, 
the digestion and ligation of the pSB1_3 ccdB plasmids was carried out again from the beginning and transformed into the 
ccdB Survival Strain from Invitrogen.
The QC multi kit was unsuccessful yesterday and so was repeated again today to remove multiple restriction 
sites in the pKD4 plasmid.
Overnights were made of the cloned KillerRed and KillerOrange in DH5α.</P><H2>04/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo, Joel, Hannah</H6><P id="pp">Glycerol stocks, mini preps and Qubit was carried out on the KillerOrange and KillerRed overnights in S171.
Overnights for successfully transformed ccdB plasmid backbones and KillerOrange and KillerRed in S171 and DH5α were made.</P><H2>05/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo, Joel, Hannah</H6><P id="pp">Glycerol stocks, mini prep and Qubit of ccdB plasmid backbones as well as KillerOrange and KillerRed from MoClo in S171 
and DH5α strains were carried out before sending these for sequencing.
The QC multi was carried out again and the controls showed it to be successful as the x-gal colonies were blue.</P><H2>08/08/16</H2><H6>Team: Dan, Jack, Pablo,</H6><P id="pp">The pJET protocol was followed for our terminator, promoter and RBS,
 these were used for our mini experiments on reverse GFP.
 Transformations were also carried out for pKD4, KO and KR.
Overnights of pSB1C3 and wild type DH5α were made to be used in the mini stat tomorrow. 
Dan and supervisor Paul investigated the setup and control of the mini stat today ready to be used tomorrow
 for initial testing.</P><H2>09/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo</H6><P id="pp">Today MoClo was used to begin cloning DNase and Lysozyme for our additional Kill switch tests.
The mini stat was set up. Starting optical densities (OD) of the overnighted pSB1C3 was
 taken before it was inoculated into the mini stat chambers. 
More overnights were made of the KillerOrange and KillerRed as well as reverse GFP pJET products in DH5α.</P><H2>10/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo, Joel</H6><P id="pp">All overnights from yesterday grew successfully. However DNase and Lysozyme did not form colonies even
 though positive and negative controls were successful. The mini stat showed significant colour differences
 in the growth of the RFP plasmid in different types of LB media, no salt seems to be fluorescing less.
Glycerol stock, mini prep and qubit of the overnighted cultures. The pJET reverse GFP products were sent for sequencing.
The MoClo and transformations of DNase and lysozyme were repeated.
Sequencing information for the KillerRed and KillerOrange protein came back to confirm which mini preps contain
 the correct DNA product.
KillerOrange was transformed into protein production strain BL21 (DE3). KillerRed will also need to be transformed.

The mini stat pump was set to 7.5 rpm and switched on at 13:30. The effluent needles were set to 20 ml.
 The air was switched off at time 0 when initial measurements were taken once the cultures reached the 
 same volume level. We need to ensure that each time the mini stat has been thoroughly autoclaved and sterilised.</P><H2>11/08/16</H2><H6>Team: Dan, Eloise, Jack</H6><P id="pp">Pablo began MoClo cloning on reverse GFP using the pJet parts.
 Qubits of these parts were carried out before MoClo was started to ascertain the exact volumes required 
 according to the DNA part concentration.
Transformations of KillerRed into BL21 (DE3), DNase and lysozyme into DH5α. 
The OD of the continuous cultures in the mini stat were taken. The burettes were emptied and the flow rate was lowered 
to reduce the amount of effluence produced overnight.
KillerOrange in BL21 (DE3) was overnighted. Four flasks containing 50 ml LB were set up in the hood and wrapped 
in tin foil ready for antibiotic and KO inoculation tomorrow.
Dan prepared overnights of the LB in the ministat against LB without antibiotic so as to see if the chloramphenicol 
was still functioning in our LB media to understand how long the antibiotic remains effective.</P><H2>12/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo</H6><P id="pp">Dan's experiment showed that the chloramphenicol was still working after 1 day in the ministat.
 However by day 2 all media was contaminated.
Again transformations of DNase and Lysozyme were not successful.
Three of the flasks were inoculated with KillerOrange and one with control pSB1C3 RFP, OD was measured every hour
 using the Tecan and Cuvette reader until optical density reached 0.4. At this OD IPTG was added to flasks B
 and C containing KO. The induced flasks were left wrapped in foil for maturation of the protein for 1 hour.
 After maturation 200μL of each culture A,B and C were spread plated.The control, induced culture B and non-induced
 culture A plates were placed in the light box for 1 hour while induced culture C's plate was left wrapped in foil 
 on the bench. After the hour all plates were placed in the cold room for testing on Monday.
The transformed KR plate was also placed in the cold room to be overnighted on Monday.</P><H2>15/08/16</H2><H6>Team: Dan, Eloise, Jack</H6><P id="pp"> The spread plates of KilllerOrange still had live colonies concluding that the kill switch had not worked. 
A sample of cultures of KO in flask that had been left over the weekend was taken and the rest was poured into
 plates and again left in the light box for the entire day. The sample was tested using the FACS machine, 
 this showed that the cells were still live and on the non induced culture A,
 KillerOrange protein had been produced because the T7 promoter is leaky.
KillerRed was overnighted.</P><H2>16/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo</H6><P id="pp">DNase and lysozyme were cloned again then transformed into DH5α.
 KillerRed overnights were glycerol stocked, mini prepped and quibitted.
 Flasks were set up to test KillerRed and KillerOrange again.</P><H2>17/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo</H6><P id="pp">The transformations of DNase and Lysozyme were successful. 
The colonies were overnighted. The flasks prepared yesterday were inoculated with KillerOrange and KillerRed, 
OD reading were taken till 0.29 on the tecan reader was reached. 
After this OD reading each was induced with 100μL of 0.1 M IPTG.
 A 1.5ml sample was taken from each culture,
 spun down and treated with bug zapper in preparation for an SDS page gel the next day.
 The cultures were then incubated in at 37&amp;#8451 and 220rpm overnight. The DNase and Lysozyme colonies were overnighted.</P><H2>18/08/16</H2><H6>Team: Dan</H6><P id="pp">Overnights of the DNase and Lysozyme were glycerol stocked mini prepped and qubitted.
 5ml samples of the overnight KillerOrange and KillerRed cultures were pipetted into 10ml falcon tubes and placed label down in the 
 light box set to 3. An SDS page gel was performed using the samples taken previously and samples taken 20 hours 
 after induction. Spread plates were made of all the samples under the light after 6hrs of exposure. </P><H2>19/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo,</H6><P id="pp">Spread plates were checked to find all had live colonies. Work will begin again on Monday.</P><H2>22/08/16</H2><H6>Team: Dan, Eloise, Jack,</H6><P id="pp">Our Enzcheck lysozyme kit arrived so we began by making stock solutions and aliquoting 
these reagents to be stored for later work. We transformed KillerOrange, KillerRed and Lysozyme 
into BL21 (DE3) and started competent cell prep of this strain. Later in the day overnights were 
made to be used for testing tomorrow for an SDS page gel, these included KillerRed BL21 (DE3) and DH5α.
 The DNase was again cloned using the MoClo method and placed in the PCR machine overnight. 
 We prepared a streak plate of DH5α KillerOrange to be sent to Glasgow iGEM team for a collaboration.</P><H2>23/08/16</H2><H6>Team: Dan, Eloise, Jack</H6><P id="pp">MoClo DNase was removed from the PCR machine and transformed into DH5α. 
In addition to this KillerRed was also transformed into DH5α. Overnights were made of all transformations from yesterday as all
 were successful. The lysozyme horizontal gene transfer experiment was performed
 and left overnight in the PCR machine at 55&amp;#8451.
5ml samples were taken from the culture flasks of KillerRed induced, KillerRed not induced and DH5α and put into the cold room. </P><H2>24/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo.</H6><P id="pp">We inoculated flasks from yesterday with 0.5ml of inoculum. 
And grew them to an OD of 0.4 for KillerRed and 0.7 for KO and induced with 0.1ml of 0.1M IPTG prepared today and left overnight 
37 degrees and 220 rpm. Dilutions of the 20 hr and 4hr induced KillerRed from 23/8/16 were prepared to 10<SUP>-1</SUP>,10<SUP>-2</SUP>,10<SUP>-3</SUP>, 
these were then exposed to white light for 7.5 hrs and then 0.2ml of each was spread plated. The incubated lysozyme was
 removed from the PCR machine, 3μL was transformed in the usual protocol, 3μL was incubated with with 200μL 
 and 0.7ml of LB. The left over lysate was plated out as a control to see if the cells survived the reaction. 
 The EnzChek standard curve was performed. Competent BL21 (DE3) cells were made and put into the -80℃ freezer.
 An SDS page gel was performed on the 4 hr and 20 hr induced KillerRed, and DH5α,
 it didn't show any real difference. Overnights of the successful DNase plates were made.  </P><H2>25/08/16</H2><H6>Team: Dan, Eloise, Jack, Pablo</H6><P id="pp">The plates prepared last night using the lysate the the HGT experiment were checked. 
Plated out lysate showed no colonies, the full transformation protocol showed colonies producing RFP, 
the incubation condition also showed colonies.
 This is an initial indication that HGT of DNA from a cell that has undergone lysis using production of lysozyme 
 can in principle happen. More repeats of the experiment will be undertaken.
 The overnight cultures of KO BL21 (DE3), KillerRed BL21 (DE3) and pSB1C3 RFP DH5α were used to make 4.5ml
 samples at a dilution factor of 10-3,10-4 and 10-5. One set of fifteen samples 
 was exposed to light in the light box. A duplicate set of samples was kept 
 in the dark outside the light box. The temperature inside the light box was also taken. It was observed that the
 temperature in the box was around 37℃, and outside was not. Repeats of the experiment will include 
 the dark samples being covered in foil and placed in the box with the exposed samples.
 Fluorescence and OD will be taken of all dilutions before and after exposure from now on.
All samples were spread plated.
</P><H2>26/08/16</H2><H6>Team: Dan, Jack, Emily, Pablo</H6><P id="pp">
CFU's were counted on the spread plates from the light exposure experiment. 
Concentrations of the DNAse miniprep was measured so this can now be sent for sequencing. 
Glycerol stocks were made of the KillerRed in DH5α and lysozyme in BL21 (DE3)
</P><H2>30/08/16</H2><H6>Team: Dan, Emily, Pablo, Joel</H6><P id="pp"> Cultures were prepared for the KillerRed and KillerOrange experiment to be run. The ministat chambers were cleaned ready t
 o be autoclaved. DNAse was transformed and the miniprep was sent for sequencing. 5ml overnights 
 were prepared for a repeat of the KillerRed, KillerOrange experiment. Overnights for the interlab were performed. 


</P><H2>31/08/16</H2><H6>Team: Dan, Pablo</H6><P id="pp"> Cultures were started from the overnights of KillerRed and KillerOrange ready to be induced, 
the cultures induced yesterday, were diluted and put in the light box. Spread plates were performed for each
 sample after 6hrs of exposure.</P><H2>1/09/16</H2><H6>Team: Dan, Emily, Joel</H6><P id="pp"> KillerRed and KillerOrange were diluted and exposed to light for 6hrs before being spread plated. 
The FACS data for interlab was completed. The overnight cultures of the constructs were grown from a starting 
OD of 0.02 for 6hrs then samples were run through the FACS machine. The data was analysed and sent to iGEM. </P><H2>2/09/16</H2><H6>Team:Emily, Pablo</H6><P id="pp"> CFU's were counted, there were two anomalies RFP 10<SUP>-3</SUP> in the dark and KillerOrange 10<SUP>-5</SUP> not induced
, this data will be discounted as it was an issue with the plates the colonies were grown on </P><H2>6/09/16</H2><H6>Team:Emily, Dan</H6><P id="pp"> The chambers and media containers for the ministat were prepared and autoclaved.
 A new configuration for the effluent was set up using 1 litre duran bottles instead of 50 ml burettes. 
 This solves the practical problem of overflow. Overnights of RFP, KillerRed,KillerOrange,BL21 (DE3) and lysozyme. </P><H2>7/09/16</H2><H6>Team:Emily, Dan</H6><P id="pp"> Set up a ministat, had issues with one of the chambers most likely because of blockage
 from using water when autoclaving the previous day. KR, RFP, WT were diluted and exposed to light for 5hrs 
 (shorter time than previous because of some errors made during dilutions e.g putting LB with CAM and WT together).
 These were then spread plated and incubated overnight. Received zstrain of <I>E.coli</I> from Glasgow </P><H2>8/09/16</H2><H6>Team:Emily, Dan</H6><P id="pp"> CFU count of KR, BL21 (DE3) and RFP. WT grew in both conditions,
 showing RFP is phototoxic, but not to the same extent as KR or KO. Will repeat with WT as well as KillerOrange and KillerRed.
 Made LB plates. Issue with effluent bubbling over, fixed problem by moving effluent tubes higher 
 to stop bubbles forming on top of each other. </P><H2>9/09/16</H2><H6>Team:Dan</H6><P id="pp">Counted CFU's for the KillerRed and KillerOrange experiment </P><H2>12/09/16</H2><H6>Team:Dan, Eloise, Emily</H6><P id="pp">Prepared overnights for KillerRed, KillerOrange and lysozyme samples.</P><H2>13/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp">KillerRed and KillerOrange and Lysozyme samples were grown up in 50 ml of media and induced. Overnights were made of the KillerRed and KillerOrange.</P><H2>14/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp"> Ministat was inoculated with samples of KillerRed, KillerOrange and lysozyme in BL21 (DE3) and the batch phase of the culture started. KillerRed and KillerOrange experiment was performed </P><H2>15/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp">Ministat pump was switched on and the continuous culture started. CFU's were counted. 
</P><H2>16/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp"> KillerRed and KillerOrange experiment was performed. Samples of each culture in the ministat were taken and glycerol stocked. Enzcheck assay was performed on the lysozyme samples at different dilution factors to determine what was optimal for the assay. Samples were taken from the ministat and glycerol stocked</P><H2>20/09/16</H2><H6>Team:Dan, Eloise, Pablo</H6><P id="pp">We took KillerRed, KillerOrange and Lysozyme samples from the ministat chambers today, and glycerol stocked the samples.</P><H2>22/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp">We took KillerRed, KillerOrange and Lysozyme samples from the ministat chambers today, and glycerol stocked the samples.</P><H2>23/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp">Today, we prepared overnights of the glycerol stocks made previously from the ministat chambers. We incubated the overnights and left them to grow.</P><H2>26/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp">To gather more data and compare with our pre-existing results, we repeated the Lysozyme Enzcheck assay today to measure the activity of Lysozyme.</P><H2>27/09/16</H2><H6>Team:Dan, Eloise</H6><P id="pp"> Today, we tested the samples we had taken previously from the ministat to test whether our kill switches - KillerRed, KillerOrange and Lysozyme - were still viable and working.</P></DIV></DIV></BODY></HTML>