Team:Exeter/Log
Lab
Lab Project
Lab Book
Safety
Parts
Team
InterLab
Human Practices
Integrated
Public Engagement
& Education
Log
Attributions
Awards
Awards
Special pages
HP Silver
HP Gold
Models
Collaborations
0
Timeline
March
April
May
June
July
August
September
October
November
Log entry
V
Vlog entry
Easter & Exams
S
M
T
W
T
F
S
1 2 3 4 5
6 7 8 9 10 11 12
13 14 15 16 17 18 19
20 21 22 23 24 25 26
27 28 29 30 31
1 2
3 4 5 6 7 8 9
10 11 12 13 14 15 16
17 18 19 20 21 22 23
24 25 26 27 28 29 30
1 2 3 4 5 6 7
8 9 10 11 12 13 14
15 16 17 18 19 20 21
22 23 24 25 26 27 28
29 30 V31
V1 V2 3 4
1 V6 V7 8 V9 V10 11
12 V13 V14 V15 V16 17 18
19 20 V21 V22 V23 24 25
26 27 V28 V29 V30
1 2
3 V4 5 V6 7 8 9
10 V11 12 V13 V14 15 16
17 V18 19 V20 V21 22 23
24 25 26 27 28 29 30
31
V1 2 V3 4 5 6
7 8 9 10 11 12 13
14 15 16 17 18 19 20
21 22 23 24 25 26 27
28 29 30 31
1 2 3
4 5 6 7 8 9 10
11 12 13 14 15 16 17
18 19 20 21 22 23 124
25 26 27 28 29 30
1
2 3 4 5 6 7 8
9 10 11 12 13 14 15
16 17 18 19 20 21 22
23 24 25 26 27 28 29
30 31
1 2 3 4 5
6 7 8 9 10 11 12
13 14 15 16 17 18 19
20 21 22 23 24 25 26
27 28 29 30
March 2nd
Log entry
We had our first meet up today. To get to know eachother better in small teams we were set the task to construct a spaghetti tower to hold a marshmallow as high as possible, only a bit of tape and some string.
We met the 2015 Exeter team who gave us advice concerning the iGEM competition.
March 9th
Log entry
Today Professor John Love gave us an introductory synthetic biology lecture which helpful to those of us who are not studying biosciences.
March 16th
Log entry
We had our team and individual photos taken today by Aaron Yeung of Exeter Photo Society. This was followed up by our first group brainstorm session.
The preliminary ideas include:
Finding traces of peanut
Degrading plastic
Creating rocket fuel
Soft bioluminescent lighting
Making a screen using bioluminescence
Making plastic
Detecting STD's
We then split up into groups to research broad categories, including food security, diseases and space, and plan to feedback to each other in the next meeting.
March 18th
Log entry
You can see below the team photo taken the other day.
March 22nd
Log entry
Today we were given a talk by Markus Gershater, the cofounder of Synthace
We discussed the foundations of synthetic biology itself, the core problems in the synthetic biology field and potential routes to go down with our project. Our following discussion focused on taking a measurements based approach to iGEM instead, testing components to identify their optimum operating conditions. The idea was to attempt to identify how environmentally dependant the biobricks are.
March 23rd
Log entry
Today, we had a short discussion about some of the points discussed on 16.03.16 with our iGEM advisors to decide which were viable routes to pursue, and which would likely not work or take far too long.
We also had a tour around the main biology lab, and were introduced to the equipment and machinery we will be using.
May 23rd
Log entry
Today we discussed the medal criteria and outreach ideas.
These ideas include talking with experts in the field, holding introductory workshops in schools to introduce children to synthetic biology and running stalls at science fairs.
Potential ideas:
Synthesizing a microportha virus
Kill switches evolutionary stability, potential to oscillate a kill switch (genetic clocks)
Synthesizing PAF with E.Coli and combating fungal diseases in plants
May 24th
Log entry
We mostly focused on how to develop a good grounding for our human practises. We want to educate people about synthetic biology. We also set up a YouTube channel to allow us to start uploading videos.
May 25th
Log entry
We have decided to do daily vlog to recap our progress throughout the day.
We discussed the kill switch idea more in depth, looking at the possibility of using a lab grown amino acid to trigger the kill switch.
Above an idea relating PAF and repressilators can be seen.
We identified resources that we could offer for our collaborations. Our main specialties include: modelling and data processing, some specialised equipment and mentoring new teams.
We also began to identify key academics and companies we can establish connections with to better inform our project, and develop a deeper understanding of the synthetic biology involved in kill switches, antifungal proteins and repressilators.
At this point in time, we have been in contact with teams based in: Leiden, Newcastle, Cardiff and Costa Rica. We are hoping to set up Skype meeting with them in the near future.
May 26th
Log entry
Today we looked at Astro Synbio, an educational web series about synthetic biology and space, and developed the idea of creating an episode of our own that could include developing rocket fuel, terraforming, soil purification (allowing a collaboration with Leiden), water purification, crop growth and food in space. For the rest of the day we started creating the iGEM wiki and learning basic HTML.
May 27th
Log entry
We discussed the idea of a killswitch in the pathogen responsible for Rice Blast disease and realised that we would need to spray an entire rice field for it to be effective. To overcome this issue we came up with the idea of using a genetic clock to construct a “time bomb kill switch” in which the killswitch was triggered by the build up of a specific protein.
We are very excited about the idea of creating a board game targeted at secondary school children. Some potential concepts included a set cards that have basic questions, a build-a-cell game where the first to obtain all necessary parts wins or a combination of the two.
May 30th
Log entry
Today was mostly focused on writing up a Penicillium antifungal protein (PAF) proposal for the Synenergene grant and expanding an idea of creating a CRISPR/Cas9 killswitch.       Unfortunately there much more we can do at the moment beyond fleshing out ideas and emailing academics.
May 31st
Your browser does not support the audio element.
Log entry
It was a very exciting day as today we recorded our very first blog entry! Vlogs are a great way for both the public and any absent iGEM members to keep up with our project and see what progress we have made in bite sized chunks. At the moment we have decided to publish them on YouTube two weeks after filming to allow time for any editing.
We opened dialogue with teams based in Newcastle and Costa Rica today and have scheduled a Skype meeting with Leiden for tomorrow.
A few of the team members met with Professor Rob Beardmore to discuss our current ideas. He gave us some great feedback on potential flaws which we are now working on fixing.
June 1st
Your browser does not support the audio element.
Log entry
To start off the day today we had a Skype call with the Leiden iGEM team where we discussed potential ideas and what we’re currently looking at doing for human practises.
There was a suggestion to combine the PAF and the killswitch idea to create a system with a reliable and well characterised killswitch that could be used outside the laboratory in the environment.
June 2nd
Your browser does not support the audio element.
Log entry
We met with Dr Paul James to discuss the project and the human practises ideas.
Some members of the team met with Professor Peter Winlove who raised the following questions in regards for our PAF idea:
How much protein do you need to kill rice blast?
Could we model how much protein is required to kill rice blast?
Could we have a timer system as a killswitch?
Is there a heat change associated with fungal disease?
Could we use this heat change to turn on PAF production?
Could we engineer a bug to manufacture particles of different magnetic properties?
Can we change the structure of flagella on E. coli to change their swimming patterns/motions?
What is our end point/scenario? How can we work back from there?
June 3rd
Log entry
Today we had a meeting with Dr Nick Harmer who suggested we look at secretion systems in regards to the PAF idea. This would allow PAF to pass through the outer membrane of E.coli. He also put forward the idea of using a gram positive bacteria to get around the problem of the double membrane of the gram-negative E. coli.
There was a new idea created involving a novel method of spreading a killswitch throughout a population of fungi: a virus would act as a vector for a killswitch which would pass throughout a population of fungi via its natural sexual reproduction.. The virus could have the genes that codes for the production of PAF so when the viral genetic information is inserted into the fungus PAF would be expressed, killing the fungus. However our main concern thus far is how the public would perceive an idea like this.
June 6th
Your browser does not support the audio element.
Log entry
There was a new idea created involving a novel method of spreading a killswitch throughout a population of fungi: a virus would act as a vector for a killswitch which would pass throughout a population of fungi via its natural sexual reproduction.. The virus could have the genes that codes for the production of PAF so when the viral genetic information is inserted into the fungus PAF would be expressed, killing the fungus. However our main concern thus far is how the public would perceive an idea like this.
Several emails were sent out today to various other iGEM teams, companies and people, including Adam Zivanic from the City of London school for potential human practises collaboration with his iGEM team, London Biohackspace; EMW Bookshop Community Lab, Street Bio, Genspace and William and Mary.
June 7th
Your browser does not support the audio element.
Log entry
We established contact today with Adam Zivanic today, head of Biology at City of London School. They’re entering a high school team into iGEM this year and the team seemed very interested in collaborating with us on the board game front. We intend to meet with them when we travel to London in August.
Following this we carried out research into current education on synthetic biology and unfortunately found there was very little education prior to postgraduate level, despite there being a fair amount of funding going into the field. In 2015, £40M was invested into synthetic biology research in the UK (see link) 1. This has encouraged our team to want to look at potential ways to close that gap between funding and education, and introduce the synthetic biology earlier on. One way of doing this is developing a new module for the university.
A few members of the team worked through papers on the killswitch KillerRed to gauge the current level of characterisation for it.
http://www.bbsrc.ac.uk/news/policy/2015/150129-pr-business-secretary-40m-for-synbio/
June 8th
Log entry
The work experience student worked through iGEM Labster and his feedback was that he thinks it needs better prompts to be more accessible. We spoke with him about his current GCSE syllabus, and found that synthetic biology could be taught alongside genetic modification and cloning. However, some students are confused between the differences of cloning and genetic modification. He also gave us his thoughts on the board game and overall he said it looks fun.
Peter Reader, a past Exeter iGEM team member, spoke with us about potential problems to be careful of. He advised us to be wary of parts which haven’t been used much, as it tends to mean they haven’t been characterised well meaning more room for error. He also advised meeting multiple lecturers to have them pull our ideas apart, point out potential flaws and make suggestions to amend these problems.
We also spoke with Ryan Edginton, a former iGEM member, about the wiki and the presentation. He gave us basic facts about the wiki, for example what happens on the day of the wiki freeze and required content.
He also gave us some more detailed advice about developing a brand for the project, including a logo and colour scheme, establishing a team leader for each section, fully defining the meaning of all equations used and explain the symbols and constants involved, and constantly keep testing on all internet browsers to ensure the wiki works on all browsers.
June 9th
Your browser does not support the audio element.
Log entry
A lot of today was spent working through past team’s projects to see what could be relevant to our project ideas, what couldn’t be completed in the past, what will be difficult, and what we could expand on for our own idea. There was a suggestion of using a delivery system, or developing a way to stick E. coli to the fungal cell wall to ensure delivery of PAF is direct.
We got our first card prototypes for the board game printed off today which we were all excited about. As shown below, there are four card types: promoters, ribosome binding sites, protein coding regions and terminators. We intend to make some rare and super rare cards.
In addition to this, we started developing an instruction manual to accompany the game.
We are working on how to “brand” the project by looking at different colour schemes so that when we start coding the wiki we have a general idea of what we want it to look like. Alan has put together a powerpoint introducing basic HTML concepts, so that the team members who haven’t coded before can learn.
June 10th
Your browser does not support the audio element.
Log entry
We made some slight changes to the board game cards today so we can incorporate the concept of scenarios, where the cards played by a team have to be justified. This makes the game a lot more interactive and encourages students to actually think about what cards they play, as opposed to simply playing cards to earn points.
There was a lot of looking at past wikis so we could start forming an idea of what we wanted the layout of our wiki to look like.
We discussed the PAF further. We did some research into the use of a secretion system to get PAF out of E. coli and noted how the mode of action of PAF is currently not well understood. We looked at how our project can tackle this problem, for example attatching GFP to the signal peptide known on PAF would allow us to “follow” it.
It was the work experience student’s last day with us today so we him what he had learned about synthetic biology during the week, how his perceptions towards the topic had changed and if he had enjoyed the week.
June 13th
Your browser does not support the audio element.
Log entry
Today was the first day with the majority of the team back together, so the key focus today was sitting down and re-brainstorming ideas before we settle on one. As can be seen in the mind map below, the PAF idea was fleshed out and developed in more detail.
We wrote an improved proposal for both of the measurement projects about killswitches and genome integration.
Some of the team spoke to Dr Mark Ramsdale, a fungal cell biologist. He gave feedback on the PAF dea, raising the question as to why we wanted to express PAF in E. coli as opposed to a fungus. Dr Ramsdale pointed out that due to how small PAF was as a protein, using GFP as a tag could affect how it is transported around the cell, and therefore skew the results. He told us about fungi specific promoters in Neurospora crassa which acts as a Circadian Clock using light and heat as inducers.
This prompted a discussion about using a viral vector that contains a killswitch that. The viral vector will insert its genetic information in the host, a strain of Magnaportha oryzae. The  Magnaportha oryzae would breed and the population would be infected with the virus containing the killswitch. The possibility of this being a heat activated killswitch was considered, so during the warmer months when plants grow the killswitch fungi is active to produce PAF.
Some of the team also met with Dr Nicky King, who is going to get us a stand at the Big Bang Science Fair for the South West (29.06.16) and the Britain Needs Scientists fair (06.07.16). She also discussed with us the possibility of writing an undergraduate Synthetic Biology module to be taken up by either Natural Sciences or Biosciences, and seemed very keen on the possibility of this.
We also met with Professor Rob Beardmore to discuss the genome integration idea. He said it sounds interesting but raised concerns about if it would even be possible to mathematically model it. He also liked the killswitch idea and said testing KillerRed and KillerOrange under a gradient of slowly increasing light intensity could be the strongest selector for resistance.
We also experimented with the vlog format, testing out the dynamics of using three people instead of two which we felt it worked a lot better.
June 14th
Your browser does not support the audio element.
Log entry
Some of the team skyped with the Newcastle iGEM team. We discussed potential for collaboration – they offered the skills of their computer scientists to help us with coding, and we offered the skills of our physicists to help with their understanding of electronics. We decided to Skype again in future and keep in touch with each other, as it seems there’s a lot of potential for collaborating.
We also spoke with Alice Mills, a Physics outreach officer, who is helping us develop ideas for the Physics/Maths/Computing side of our Synthetic Biology module. She is also going to put us in touch with a few different schools to allow us to test our board game with a range of classes.
June 15th
Your browser does not support the audio element.
Log entry
We made major progress with the board game today. Everyone put forward ideas to help improve it. The preliminary designs have been made and have been test printed. The ATCG counters used to keep score have been 3D printed in plastic.
We also had several meeting today. Dr Harmer spoke with us about the potential pitfalls of the PAF idea and how we can get around/solve them. Professor Smirnoff spoke with us about several of our ideas, giving his opinion of what he thought will work best.
We started talking with the iGEM teams based in Westminster, Purdue and UNL with the intention of setting up collaborations with them.
June 16th
Your browser does not support the audio element.
Log entry
Today was a day focused heavily on developing our human practises. We started constructing a booklet of rules and brainstormed ideas for the scenario cards. The role of scenario cards is to make the players justify their use of Promoter cards and how well it fits the scenario dictates how many points the card scores.
We met with two of our instructors today. Jamie Gilman discussed the benefits of using design of experiments with us. Dr Chloe Milner discussed our current potential ideas with us and ordered a gBlock for us so we could test whether or not PAF could be expressed in E. coli, which is one of the main concerns we have before we commit to a project.
June 17th
Log entry
Everyone in the team had a lab induction today with Dagmara Kolak to familiarise ourselves with the the Mezzanine lab.
We split into three teams to research the three main ideas and prepare to present them to academics from various disciplines later on:
PAF (Pablo, Alice, Andy and Hannah)
Viral Vector Killswitch (Jack, Eloise and Leanne)
Measurement- kill switches and genome integration (Dan, Emily, Joel and Alan)
We hope to get expert feedback on flaws in our projects and what’s actually possible to complete within the summer.
June 18th
Log entry
The board game was the focus of today again as we are trying to finish the first draft in time for the Judd school visit and continuing working on ideas and presentations in anticipation for the meeting with academics.
June 19th
Log entry
We met with Dr Burton today who helped us a lot with ideas on the human practises side. She gave us contacts to help with trying to get synthetic biology on the GCSE syllabus, told us about a microbiology grant we were eligible for and gave us the idea of doing a “walkthrough science” video in collaboration with Soapbox Science.
We also started talking with Grace Fleet over email, who is a public engagement officer, focusing on scientific research. We intend to set up a meeting with her in the coming days.
June 20th
Log entry
Part of the team went to the lab today to go through the Interlab study materials that we had been sent, but we couldn’t start as our kit was missing parts. We have emailed iGEM headquarters in the hopes they can be quickly replaced.
We also began working on the synthetic biology module. We currently have 7 potential textbooks which we’re taking notes on to inform our decision on what could and should go into the module, and which textbooks will best supplement this. We ensured that the textbooks were available online through the university, making them accessible to all who wanted to take the module.
The potential textbooks include:
The Analysis of Biological Data by Whitlock and Schulter
Understanding Bioinformatics by Zvelebil and Baum
Core Maths for the Biosciences by Reed
Next Generation DNA Sequencing Informatics by Brown
Molecular Biology of the Gene by Watson, Baker, Bell, Gann, Levine and Losick
Choosing and Using Statistics: A Biologist’s Guide by Dytham
Biostatistical Analysis by Zar
June 21st
Your browser does not support the audio element.
Log entry
Today was the first day of the Judd School trip. Alice, Andy, Alan and Jack left early this morning to travel to the school, and will be focusing on running board game sessions with the students, filming the game being played and filming interviews with the teachers and students.
The rest of the team remained at the university and continued to work through textbooks for the synthetic biology module.
We officially signed up for the interlab study today as and today’s lab team for the day attempted to test what we had with some competent cells prepared by Jamie Gilman to see what we could accomplish with the current kit.
Curves on the plate reader in the lab to prepare for measuring fluorescence levels, and incubated some flask cultures overnight in the 37 shaking incubator.
We also began to discuss possible activities we could include on our stand at the upcoming Big Bang Science Fair. As the demographic present was 9-18 year old students, teachers and parents, we were aware that we would have to have different activities and resources focused at the different ages.
June 22nd
Your browser does not support the audio element.
Log entry
For the second day of the Judd School trip the team led a synthetic biology talk for the students there and recorded a few more interviews. We received very positive feedback about the board game and the school asked for their own copy.
The morning lab team tested the cells that they cultured overnight, but the OD600 was lower than expected, and transformed the competent cells.
We also released part 1 of our ‘Meet the Team’ videos, signalling the start of our YouTube channel.
June 23rd
Your browser does not support the audio element.
Log entry
We checked on the competent cells transformed the previous day and all the colonies expressed strong fluorescence however the interlab parts had not worked.
During researching for data to put into our powerpoints for the Big Bang Science Fair we found that all the data on public perception of synthetic biology was rather out of date.
We decided to set up a synthetic biology survey which we intend to leave it running for the duration of our project and then analyse the data we have collected.
We Skyped Westminster and Purdue and now we’re setting up follow up talks with both teams to try and sort out potential collaborations.
June 24th
Log entry
Most of the day was spent preparing presentations of our project ideas for an upcoming meeting with academics. We focused on the PAF idea, the genome integration idea and the kill switch idea. The viral-vector fungal killswitch idea was scrapped as there were too many potential flaws for us to iron out in the ten week process.
We met with last year’s Exeter iGEM team and discussed our ideas, what we’re doing for human practises and just general advice they wanted to give us.
June 27th
Log entry
Professor Winlove and Dr Porter came to the academic meeting.They liked all three ideas and gave us feedback and advice. After discussing them the possible issues of each idea, we decided on a slightly expanded version of the kill switch idea as our project.
The rest of the day was split between researching more into kill switches and working on ideas for our stand at the Big Bang Science Fair, such as leaflets, posters, presentations, printed surveys and interactive activities to engage visitors.
June 28th
Your browser does not support the audio element.
Log entry
Today we have been preparing for the Big Bang Science Fair. We finished and printed off a poster showing how biology, chemistry, engineering and maths are all important in synthetic biology, a Synthetic Biology leaflet and an iGEM leaflet, the front and back of which are pictured below.
Also, we completed two presentations on Synthetic Biology and iGEM which we intend to run on a loop on two iPads for the duration of the fair.
The abstract for our project was written up and a basic form was uploaded to the Wiki. It focuses on how we can improve killswitches, as opposed to the initial killswitch idea of characterising the parts extensively. We intend to submit KillerOrange as our new part, and compare its functionality against that of KillerRed with a continuous culture. We also want to test if it is beneficial to have two killswitches in a system as opposed to one and look at the difference in stability of the killswitch if we integrate into into the chromosome.
June 29th
Your browser does not support the audio element.
Log entry
Today was the Big Bang Science Fair where the board game got a lot of interest and seemed very popular with the students who tried it out.
On top of that, we got the contact details of a home school teacher and a vice principle who are interested in potentially helping us set up focus groups for the board game. We also got a lot of new survey responses to add into our data collection on public perception on synthetic biology.
get img working
We started the interlab study today as our new parts arrived. We completed the transformation section of the study and left our prepared petri dishes in the incubator overnight.
June 30th
Your browser does not support the audio element.
Log entry
The agar plates we incubated overnight had colonies and they expressed GFP, as you can be seen in the photo below. The agar plates shown below are the competent cells containing plasmid 1, 2 and 3.
We released the video about the Big Bang Science Fair today, featuring interviews with several of the attendants there. We also started to write up the CRISPR proposal for the “CRISPR Design and DNA support” offered by Twist Bioscience and Desktop Genetics.
We skyped with Purdue for the second time and arranged a collaboration together. We have a Google Drive folder set up so it’s easier for us to communicate and request help from each other. Purdue will be running a continuous culture of a killswitch developed by Purdue’s previous iGEM team and sending us the data, and we’ll be helping them write up summaries of past team projects for their project database.
July 1st
Log entry
Today was the third day of Interlab and our data so far is coming out as we expected.
We continued with the CRISPR proposal. We also skyped with UCL today, but due to how different our projects are, we had difficulty finding potential areas for collaboration.
July 4th
Your browser does not support the audio element.
Log entry
We discussed our ideas with Dr Paul James and told him that we had chosen to go with the kill switch idea and outlined the experiments that we wanted to do and began looking at parts for the constructs.
July 5th
Log entry
Today, we started the day meeting with Dr Paul James and Dr Chloe Singleton to recap our project idea, and the receive feedback from them on the approach we should be taking towards it, and how to better structure and word our CRISPR proposal. It was quite reassuring to have them both quite positive about our idea.
Leanne and Emily met with Dr Nicky King again to discuss what we have so far for the synthetic biology module we are writing. It was incredibly useful, as Dr King gave us advice on several different module structures we could look at and several other academics we could talk to. She seems very enthusiastic about our idea, offering to propose it to the Natural Sciences and the Biosciences committee at the start of first term to find out if it’s something we can implement.
We also redid headshots and our team photo, as everyone was present today and the previous image sizes were not appropriate for the wiki. We definitely like these ones, and it’s great to have the whole student side of the team together!
July 6th
Your browser does not support the audio element.
Log entry
Today Dan and Pablo spoke to supervisors Dr Paul James and Dr Chloe Singleton about TRACR and selective advantages with methylation, Dr Singleton suggested we research into catabolic repression.
Andy made a human practices infographic and Joel made a GIF of Killer Red. Alice continued to work on the video documenting the visit to Judd High School.
We attended the ‘Britain Needs Scientists’ fair in the afternoon. We we able to interview students and teachers about their knowledge on Synthetic Biology to further our human practices work. We had our board game and sweet DNA display which enabled us to engage with more teachers about how to educate students on synbio.
After this, the lab team designed constructs for our Killer Red experiment choosing suitable parts from the iGEM registry.
July 7th
Log entry
We recorded the very first episode of ‘Desert Island… Science?’ with Dr Nicky King today. We also had a meeting with Dr Sara Burton to discuss potential module structures and possible areas for funding.
We also skyped with UNL in the afternoon and looked at what experiments we are going to run.
July 8th
Log entry
We have improved our board game, BioMech, for  the upcoming school visit to Colyton Grammar School to trial the game.
We released the very first episode of ‘Desert Island… Science?’ today, with Dr Nicky King of the University of Exeter
July 11th
Your browser does not support the audio element.
Log entry
Today, we set up a collaboration with the Newcastle iGEM team. We hope to start running experiments to determine the thermal conductivity and convection effects in LB and M9 broth for them.
July 12th
Log entry
Alice recorded and released the second episode of ‘Desert Island… Science?’ today. This time, she interviewed Dr Sara Burton, a senior Biosciences lecturer at the University of Exeter. Dr Mark Ramsdale also agreed to be interviewed for our series, and we have begun to reach out to people outside of the university so we can expand our series.
We also had a Skype session with the UCC iGEM team today, who were hilarious! They’ve said they’ll show us around Boston at the Jamboree as some of their current members have been on past teams, and we also sorted a collaboration with them for later down the line of testing out each others parts.
July 13th
Your browser does not support the audio element.
Log entry
Today we got a lot of the Purdue collaboration done (Purdue wanted us to write summaries for their past project database), but we intend to get far more done over the coming weeks. Their whole idea of a database is really cool though, as it means it’ll be so much easier for future teams to look at relevant past projects, to find out what has previously been done and what can be built upon.
It was the first session back in the lab after doing the Interlab study too. Most of the team - Dan, Pablo, Eloise, Hannah, Andy and Jack - were in the lab at some point today. They resuspended Killer Red DNA from the iGEM parts we had been sent, then attempted to transform it into competent cells. They made 9 plates for incubation - positive and negative controls, 3 chloramphenicol resistance plates and 4 ampicillin resistance plates.
Beyond that, the day was mostly focused on designing parts and designing experiments.
July 14th
Your browser does not support the audio element.
Log entry
Dr John Love, one of our lead supervisors, came back from a conference trip today. We met with him this morning (at 8am!!!) in Costa (so at least we could get some coffee) to discuss what we’ve done so far on lab work, human practises, wiki and modelling, and get some feedback.
He gave some incredibly useful advice on how we could expand our lab work more, looking at different ways to characterise the kill switches. In regards to human practises, he loves the board game (and gave us funding for 16 more physical copies), and suggested using surveys to quantify the impact of the board game on the students using it. He also agreed to take part in ‘Desert Island… Science?’ which is fantastic!
We’ll be meeting with Dr Love and Dr James, our two lead supervisors, in sub teams next Tuesday, so it’s time to start finalising who fits into what roles so we can organise those teams.
The KillerRed transformations from yesterday were unsuccessful - there was either no growth of 1-2 colonies on all the plates. However, we believe we know the error - we attempted to transform 1μl of DNA, which may be too low of a DNA concentration, so the transformations will be reattempted with 5μl.
July 15th
Log entry
Exciting news for ‘Desert Island… Science?’ today! We will officially be expanding outside of just academics based at the University of Exeter - Dr Jon Marles-Wright and Dr Thomas Howard, both academics at the University of Newcastle, have agreed to be interviewed for our series! We’ve sent out a few more messages, but haven’t had any more responses yet - fingers crossed for over the weekend!
Today was incredibly exciting in terms of our board game too - we received the 16 printed copies of BioMech, and they look incredible! We released a photo of the finalised board game on Twitter earlier, with this being the first time we’ve publically mentioned our board game, and the response has been fantastic so far! It really has been so wonderful how well it has been received!
We have also been in touch with The Sutton Trust in regards to the game, and they seem quite interested in the idea. Hopefully we will be able to sort something out with them in regards to widening participation.
The Killer Red transformations were successful this time, which is great news and means it was only a concentration issue and not a procedure error. The lab team today - Dan, Eloise, Emily, Jack and Pablo - worked on transforming Killer Orange into plasmids using a 3:1 ratio for insert to backbone. They made both ampicillin and chloramphenicol plates again for incubating the transformations. They also prepared more chloramphenicol stocks for future lab sessions.
July 18th
Your browser does not support the audio element.
Log entry
Today, four of our team members (Alice, Joel, Andy and Jack) went on our second school visit to Colyton Grammar School based in Colyford, East Devon. They conducted a board game session with 26 students - our first board game session since having the final design printed - and gave out surveys so we could gauge the impact of the board game on the students.
We had a new work experience student join us today called Juliet - she’s a year 12 at the moment with a strong interest in Biology, and will be helping us out in and out of labs (but mostly in). Juliet will be joining the team for the next two weeks.
They also left behind three board games so the school could get extended use out of them, and continue to send us surveys on board game reception - we’re hoping to receive the first batch back within the next few days so we can get an idea of how effective the game is.
We also had two very exciting confirmations for ‘Desert Island… Science?’ today - Professor Paul Freemont of Imperial College London, and Professor Jim Al-Khalili of the University of Surrey (and presenter/guest presenter on several shows, including ‘Horizon’, ‘Light and Dark’ and ‘Shock and Awe’).
Due to some key members of the lab team being on the Colyton Grammer School visit, today was a slow lab day - we prepared some glycerol stocks using 500μl glycerol and 500μl E. coli culture and stored them.
July 19th
Log entry
We split into sub-teams to meet with Dr John Love and Dr Paul James separately and discuss progress in our respective areas.
July 20th
Log entry
More exciting news for ‘Desert Island… Science?’ today - Professor Richard Kitney of Imperial College London agreed to partake! Professor Kitney has been a visiting Professor at MIT since 1991, holds an OBE for his work, and is globally recognised as one of the leading researchers in the field of Synthetic Biology.
We also received the board game survey data back from Colyton Grammar School and have done a basic analysis on the results - overall, 77.8% of students reacted positively, saying they had enjoyed the board game (“It was a fun way for learning about synthetic biology.”) and/or had found the board game session educational (“It helped me remember most of the information because it was fun and engaging.”).
The lab team attempted Q5 site-directed mutagenesis today using a two-step PCR and a 1 in 10 dilution for the primers (10μl primer : 90μl water). The products have been transformed into competent E. coli DH5α cells and left in the 37oC incubator overnight. They also made a table to document the contents of the freezer box (a mix of DNA and PCR products).
July 21st
Your browser does not support the audio element.
Log entry
Today was predominantly focused around talking with other iGEM teams and organising collaborations - we had a Skype session with Cardiff and talked about our projects and current progress. We also spoke with Westminster over Twitter about the possibility of collaborating.
Yesterday’s attempt at Q5 site-directed mutagenesis turned out to be unsuccessful, so we will be re-attempting that today. We also measured the concentrations of DNA we had for various promoters, terminators, minipreps, etc, and made a table for future reference so we can track how much we have left.
5μl of the digestion and ligation products were transformed into competent E. coli cells and left to incubate overnight. We also did a PCR and gel electrophoresis, and left the transformed products to incubate overnight.
It was mostly just a day of general progress on all areas today though, with people working in the lab, on the wiki, pushing through modelling and making progress on human practises.
July 22nd
Log entry
The digestion-ligation transformation from yesterday did not work, so we will be repeating the digestion stage with RFP-tetracycline plasmids from Dr Chloe Singleton. For ligations, we made: promoter + RBS + destination, CDS + terminator + destination, and a control.
Yesterday’s PCR and gel electrophoresis were successful, but the products were not successfully transformed. We repeated the gel electrophoresis, which was successful, and transformed the products into E. coli DH5α cells which were left to incubate at 37°C overnight.
July 25nd
Log entry
Professor Paul Freemont replied to our email request in regards to featuring on our interview series “Desert Island… Science?”. He said he would be keen to do so after his leave, so we will speak with him again in late August.
Transformations of ligations were successful. We made transformations of PCR products with a new KLD enzyme mix and plated all the transformations, and made glycerol stocks before doing minipreps.
July 26th
Log entry
Today, we skyped with Dr Jon Marles-Wright for ‘Desert Island… Science?’, making this our first interview with an academic outside of the university. Dr Marles-Wright is one of the lead supervisors for the Edinburgh iGEM team.
We also decided to shift our project aim slightly. Instead of using Cas9, which we could not get to work, we will be looking genome integration instead, and comparing Killer Red and Killer Orange to ‘natural kill switches’ like hen egg Lysozyme and bovine DNAse1.
It was also a lab heavy day - we did digestion and ligation of promoter + RBS and CDS + terminator combinations in plasmids, and did transformations of ligation (1) (Promoter-RBS-CDS-Terminator in CAM), ligation (2) (Promoter-RBS-CDS-Terminator in CAM), positive control (RFP + chloramphenicol plasmid), negative control (WT competent cells) and destination (CAM destination plasmid).
July 27th
Log entry
Andy and Alan worked in the Physics labs today, measuring the intensity of our light box, and calculating the W/cm^2 that the plates placed inside the light box will be receiving. We need to know this so we can look at how intensity impacts upon the rate at which Killer Red and Killer Orange kill cells.
Today’s lab team - Eloise, Emily, Hannah and Dan - started using a new method of cloning called Modular Cloning, which we intend to use codon optimised Killer Red and Killer Orange to submit to the registry. They also used an adjusted digestion and ligation method to cut RFP from template backbones, ran on a gel, then transformed for overnight incubation.
July 28th
Log entry
The ligated Killer Red parts from yesterday were mini prepped, and the successful Cas9 colony was put in overnight incubation. We re-transformed the unsuccessful transformations from yesterday, as only the plasmids with chloramphenicol and tetracycline backbones were successful.
The successful Killer Red ligations were sent off for sequencing today to check that we made the part in the correct order sequence.
July 29th
Log entry
Today marked a few additions to the modelling team due to the shift in project aims: Joel is continuing working on the KillerRed model, Andy will begin working on a hen egg Lysozyme model and Leanne will begin working on a bovine DNAse1 model. The current program of choice is SimBiology, an extension app on MATLAB, but that may change over the coming weeks depending on how compatible it is with our model aims.
We also released a new ‘Desert Island… Science?’ interview today, featuring Dr John Love. Dr Love is an associate professor of Synthetic Biology at the University of Exeter, and is our primary supervisor for iGEM.
The lab team for today - Jack, Emily, Joel and Hannah - did glycerol stocks, minipreps and qubits of the Cas9 part.
August 1st
Your browser does not support the audio element.
Log entry
Over the weekend, we released a short video on our interview with Richard Dawkins, who had come to Exeter as part of the International Society for Behavioral Ecology conference. The event, hosted by the university, brought academics from all across the globe to campus.
We had a new work experience student join us today called Jorge. He, like our previous work experience student Juliet, will be here with us for two weeks, helping out in the lab and generally operating as a standard team member.
Jorge was introduced to the full team during today’s team meeting, where we discussed the goals for this week inside and outside of the lab, and generally just checked up on the progress of each area of the project.
The lab team - Jack, Eloise, Pablo and Hannah - attempted to redo the unsuccessful plasmid backbone transformations from yesterday, but were not successful in amplifying the PCR product because the concentration was too low. However, some of the plasmid backbone transformations done today were successful, and have been overnighted so we can use them for cloning.
August 2nd
Log entry
We swapped to using the QuickChange Multi kit instead of the Q5 kit for site-directed mutagenesis today as the Q5 kit had so far been unsuccessful, and the QuickChange kit could be used to mutate 5 sites at a time.
Today’s lab team also began using MoClo (Modular Cloning, mentioned in an earlier entry) to make codon optimised Killer Red and Killer Orange parts that we can submit to the registry. The products were transformed into S171 E. coli cells and overnighted. They also did some standard miniprepping and carried out qubits.
August 3rd
Your browser does not support the audio element.
Log entry
We put the order for our team t-shirts and hoodies/sweatshirts so they arrive in time for the Westminster meet-up. They have the iGEM logo on the front, our social media accounts on the back, and once we have a logo designed we’ll be sending them back to get the logo put on the front too!
Yesterday’s production of plasmid backbones was successful but the digestion and ligation of said backbones was not, so we repeated the digestion-ligation process today. The QuickChange multi kit also turned out to be unsuccessful, so we repeated that today to try remove multiple restriction sites in a pkD4 plasmid.
We transformed cloned Killer Red and Killer Orange in E. coli DH5α and overnighted them.
August 4th
Log entry
Today we started working on the Newcastle collaboration - we were determining the thermal conductivity of LB and M9 broth for them. The primary focus was setting up the equipment and calibrating it with water to ensure we got the expected value.
The installment of ‘Desert Island… Science?’ featuring Dr Jon Marles-Wright was released today. The delay between recording and releasing was for Alice to deal with sound issues caused by recording over Skype, which were all dealt with.
Lab work today predominantly focused on the overnighted Killer Red and Killer Orange in S171 E. coli. The lab team - Dan, Eloise, Jack, Pablo, Joel, Hannah - made glycerol stocks of, miniprepped and carried out qubits on the overnighted samples.
They also made new overnights of Killer Red, Killer Orange and the successfully transformed plasmid backbones in both DH5α and S171 E. coli strains.
August 5th
Your browser does not support the audio element.
Log entry
We now have LB thermal conductivity data from the Newcastle collaboration experiments, and will be analysing them and sending them to the Newcastle iGEM team once we have completed the same for M9.
Joel began producing some fantastic logo ideas today, drawing from a moodboard produced by us a while back. we plan to stick with the colour scheme, but may alter the design depending on further feedback.
Today in the lab, we made glycerol stocks of, miniprepped and carried out qubits of the overnights of Killer Red, Killer Orange and the plasmid backbones in DH5α and S171 strains that were made yesterday, and sent samples off for sequencing.
We also tried using the QuickChange multi kit again for site-directed mutagenesis, and the controls made indicated that we had been successful this time as they turned blue when tested.
August 8th
Log entry
Rather excitingly, all the hotel rooms and transport has been booked for Westminster today! The bulk of the team - Andy, Emily, Pablo, Joel, Hannah, Eloise, Jack and Alan - are going to be going to the meet-up, and are beginning to prepare the poster and presentation from today onwards.
Our work experience student, Jorge, is joining us for his second week of work experience - he was incredibly helpful in the lab last week and works well with the whole team, so we’re grateful for his presence.
Alice recorded another ‘Desert Island… Science?’ interview today, this time with Dr Mark Ramsdale, an academic here at the University of Exeter. The interview will likely be posted to our YouTube channel shortly.
Whilst lab work has slowed down a little, the lab team decided to conduct some side experiments on reverse GFP, motivated purely by curiosity. Jack and Pablo followed the pJET protocol for the mini experiment. They also carried out some transformations for Killer Red, Killer Orange and pkD4 plasmids.
Meanwhile, Dan and Dr Paul James investigated the control and setup of the mini stat in preparation for initial tests tomorrow, and made overnights of wild type DH5α and plasmid backbones to be used in the mini stat.
August 9th
Log entry
As of today, our project name will be ‘Exepire’. We settled on using the word expire in the end as we wanted a short, snappy title which related to what we are doing - part of the project is looking at deterioration of kill switch function over time. The spelling Exepire is a play on the word to incorporate a link to our university, the University of Exeter. There were quite a few other ideas, but after a vote from team members and supervisors, Exepire was the winner.
The Westminster team focused on writing the presentation today, with the various team members doing write-ups on their areas - human practises, modelling, wiki design, etc - to incorporate into the talk.
The lab team for today - Dan, Eloise, Jack and Pablo - started using Modular Cloning to clone Lysozyme and DNase1 for our comparison kill switch tests. They also set the ministat up today and started taking ODs (optical density measurements) of the plasmid backbone before inoculating it in the minstat chambers.
They also made more overnights of Killer Red, Killer Orange and the pJET reverse GFP in DH5α E. coli cells.
August 10th
Your browser does not support the audio element.
Log entry
We had a full team meeting today, with all our supervisors and instructors - Dr John Love, Dr Paul James, Dr Chloe Singleton, Jamie Gilman and Ryan Edginton - to discuss what we’re all currently working on, what recent setbacks we’ve had and what we need help with. We collectively decided that the logo needed a little revising to dispel possible negative connotations (e.g. dangerous), but overall, they were happy with our progress which was quite the motivational and emotional boost.
Alice recorded an episode of ‘Desert Island… Science?’ with Professor Chris Lintott over Skype. Professor Lintott is the second academic outside of the university to participate in our YouTube series, and is best known for his work as a presenter on the BBC show ‘The Sky At Night’.
In preparation for the upcoming deadline on 12.08.16, the biologists wrote and uploaded our project title and abstract to the iGEM team page (as they have a better technical understanding), and we settled on our track being ‘Measurement’.
Today’s lab team was Dan, Eloise, Jack, Pablo and Joel. All the overnights from yesterday grew successfully, so they made glycerol stocks of, miniprepped and carried out qubits of the cultures. The pJET reverse GFP was sent off for sequencing.
The DNase1 and Lysozyme, however, did not grow any colonies despite the positive and negative controls both being successful. The lab team redid Modular Cloning and transformations of DNase1 and Lysozyme.
The sequencing results came back for Killer Red and Killer Orange, confirming the mini stat contained the right DNA products. Killer Orange was transformed into an inducible strain, and they intend to do the same with Killer Red. The mini stat showed significant colour differences in the chambers, indicating different rates of growth of the RFP plasmid in types of LB media (no salt had the least fluorescence).
August 11th
Log entry
Joel and Jack began working on the poster design and Westminster presentation today - Leanne and Joel wrote up the section on Modelling, Emily wrote up the section on Human Practises, Dan wrote up a section about the Mini Stat, and Dan and Eloise wrote up the introduction section and Lab section.
Our team t-shirts and hoodies/sweatshirts arrived from Supply Line Solutions, and they look fantastic! It’ll definitely make the team look more like a team at Westminster, acting as a type of informal uniform.
The lab team for today consisted of Dan, Eloise, Jack and Pablo. Pablo carried out qubits on reverse GFP to determine DNA part concentration, then began Modular Cloning on reverse GFP using the pJET parts. They transformed the Killer Red 3 strain into BL21DE3 (a strain of competent E. coli cells), Lysozyme and DNase1 into DH5α E. coli cells.
Killer Orange was transformed into BL21DE3 cells and overnighted. Four flasks containing 50ml of LB media were prepared under the hood, ready for Killer Orange and antibiotic inoculation tomorrow.
ODs were taken of the mini stat continuous cultures. The flow rate was reduced to lower effluence produced overnight. Dan prepared overnights of LB in the mini stat to compare to LB with chloramphenicol to see how long the antibiotic acts for.
August 12th
Log entry
Joel finished the logo redesign today - this time, he based the logo idea on a power button, recreating that with DNA and an E. coli cell, as you can see below. We continued with the teal based colour scheme, as it’s a gentle colour which we all liked, and the grey-teal colour scheme on the wiki looks really good.
The LB experiment Dan set up yesterday showed us that the chloramphenicol still functioned after one day, but all media was contaminated after two days. The DNase1 and Lysozyme transformations were unsuccessful again.
Of the LB flasks prepared yesterday, three were inoculated with Killer Orange and one was used as a control. The OD was measured every hour, and the proteins were left to mature for an hour. The lab team spread plated the cultures and tested two in the light box, then placed the spread plates in the cold room so they can be tested on Monday. The transformed Killer Red plate was put in the cold room with them so it can be overnighted on Monday.
August 15th
Log entry
The poster for the Westminster meetup was sent to the printing office today, and will be back with us tomorrow morning, ready for our presentation. The bulk of today focused on the team members visiting Westminster preparing, with Eloise and Jack (the speakers for our presentation) practising and perfecting the talk.
The lab team for today was Dan, Eloise and Jack. They found the Killer Orange spread plates had not worked, as there were still live colonies present. They took a sample of the Killer Orange cultures from the flasks and left them in the light box for the full day this time. The samples were tested in the FACS machine, which showed Killer Orange proteins only being produced in the non-induced culture due to a leaky T7 promoter.
August 16th
Log entry
Today was spent finalising anything we needed for the Westminster meet-up. Jack and Eloise practised the presentation some more, and we picked up the poster from Printing Services to check it over before we left.
As most of the team were doing last minute preparations for Westminster today, it was a slower lab day. The lab team - Dan, Eloise, Jack and Pablo - used Modular Cloning the clone DNAse1 and Lysozyme again, and transformed the products into DH5α E. coli cells. The Killer Red overnight cultures were glycerol stocked, miniprepped and had qubits carried out on them. The team prepared flasks of Killer Orange and Killer Red again, so Dan could test them in the light box whilst the others were away.
The Westminster team - Andy, Emily, Pablo, Joel, Hannah, Eloise, Jack and Alan - are currently on their way down to London by train, where they’ll stop overnight in preparation for the meet-up starting tomorrow!
August 17th
Log entry
Today was the first day of team presentations at Westminster - we had our talk at 2:30, and Jack and Eloise did an amazing job of presenting our project to the other teams. Our human practises aspect of the project was received very well by the others teams, as indicated by speaking with them later and the tweets we got about it.
It was great watching the other teams present and seeing how far they’ve come, and the guest speakers - Professor Jane Lewis (introduction), Dr Anatoliy Markiv (introduction), Edward Perello (CRISPR) and Dr Robert Smith (social and ethical dimensions of synthetic biology) - were all very interesting to listen to. We’re very much looking forward to tomorrow!
Yesterday’s DNAse1 and Lysozyme transformations turned out to be successful, so they were overnighted. Dan inoculated the flasks prepared yesterday with Killer Red and Killer Orange and took OD readings on them - the cultures were then incubated at 37℃ and 220rpm overnight. The DNAse1 and Lysozyme colonies were also overnighted in the 37℃, non-shaking incubator.
August 18th
Log entry
Today was our turn to sit back and observe - the team presentations were all fantastic, and guest speakers Dr Tom Ellis (balancing biology and engineering) and Dr Vitor Bernardes Pinheiro (directed evolution) both delivered fascinating talks.
We have organised a collaboration with the Glasgow iGEM team as a result of the Westminster meetup - we’ll be sending them Killer Red and Killer Orange transformed in E. coli DH5α cells to run some proof of concept tests for us, and they’ll be sending us Z1 lac-repressor E. coli strains to use in our kill switch experiments.
Dan made glycerol stocks of, miniprepped and carried out qubits on the DNAse1 and Lysozyme overnights prepared yesterday. He also took samples of the overnighted Killer Red and Killer Orange to test again in the light box, and made spread plates of all the samples after 6 hours of exposure.
August 19th
Log entry
Andy, Alan and Joel met with Dr John Rowe of Exeter’s Physics department today to discuss modelling problems with him. After the meeting, they decided to move forth with modelling in the coding language C, as opposed to on a MATLAB application. This is because Simulink and SimBiology were not user friendly, and often did not have functions that we needed in constructing the models.
We checked the spread plates to find they all had live colonies, meaning the Killer Red and Killer Orange kill switches hadn’t worked again.
August 22th
Log entry
Joel began to further develop his Killer Red model today, moving away from a basic structure and looking a possible limiting factors like maturation rates of proteins, degradation due to photobleaching, maximum mRNA in an E. coli cell, etc.
Hannah received confirmation today that Professor Jim Al-Khalili would be happy to take part in ‘Desert Island… Science?’ - the interview has been scheduled for 23.09.16.
Andy has been designing a survey for 1st/2nd/3rd year Biosciences students to look at their perception and understanding of synthetic biology, and more specifically, kill switches. Dr Burton has offered to help us distribute the survey to new students during Welcome Week.
Today’s lab team was Dan, Eloise and Jack. Our Enzcheck Lysozyme kit arrived today, so they made stock solutions to prepare to future experiments. They transformed Killer Red, Killer Orange and Lysozyme into BL21DE3 E. coli cells, and made overnights of Killer Red in BL21DE3 and DH5α wild type cells.
They also cloned DNAse1 again using Modular Cloning, and placed the product in the PCR machine overnight. The DH5α KillerOrange streak plate was prepared to posting to the Glasgow iGEM team (for one of our collaborations).
August 23rd
Your browser does not support the audio element.
Log entry
We transformed both the Modular Cloning DNAse1 (overnighted in the PCR) and Killer Red 3 strain into DH5α E. coli cells, and made overnights of all the transformations from yesterday as they were all successful.
We attempted the Lysozyme horizontal gene transfer experiment, and left it overnight in the PCR machine at 55°C. We also took 5ml samples from induced Killer Red, non-induced Killer Red and wild type DH5α culture flasks and stored them in the cold room.
August 24th
Log entry
Today the module team (led by Emily) met with Dr John Love to discuss what currently exists of the module structure, look at where it would be best to propose module implementation (our aim is to have it established as a second year Biosciences module) and receive general feedback on how to progress. He is keen on the idea, which is definitely encouraging.
The lab team today - Dan, Eloise, Jack and Pablo - inoculated the Killer Red and Killer Orange flasks prepared yesterday and grew them to OD 0.4 and 0.7 respectively, then overnighted them at 37℃ and 220 rpm. They exposed dilutions of induced Killer Red were exposed to white light for 7.5 hours then spread plated them.
They removed the incubated Lysozyme from the PCR machine - some was transformed, some was incubated and the remainder was used as a control to see if cells could survive the reaction. They also made overnights of the successful DNAse1 plates.
August 25th
Log entry
Jack and Alice spoke with Dr Chloe Singleton, one of our instructors, about some more ideas they had had about the integrated side of our Human Practises. Andy finished developing a basic model for the mechanism of action of Lysozyme today, modelling percentage of remaining peptidoglycan in the cell wall against time.
We checked the plates prepared yesterday with lysate (from the HGT (horizontal gene transfer) experiment) - the plated out lysate showed no colonies, but fully transformed and the incubation condition both showed colonies producing RFP. This initial test shows the HGT experiment with lysozyme is successful, and we will be repeating it.
We made samples from the Killer Red BL21DE3, Killer Orange BL21DE3 and plasmid backbone RFP DH5α cultures at dilution factors of 10-3,10-4 and 10-5 and exposed to light in the light box. A duplicate set was kept in the dark room. We measured the temperature inside the light box to be 37oC. After exposure, we spread plate all the samples.
August 26th
Log entry
Today, we had a meeting with Ryan Edginton, one of our instructors, in regards to the wiki. He gave us feedback on what had been done so far - credit to Andy, Alan and Joel for coding - and talked with us about what we still needed to do to meet medal requirements for the wiki. It’s definitely coming along though, and has improved significantly in the last few weeks compared to the initial design.
Andy has been making progress on a ‘Log’ page for the wiki, on which these day to day entries will be featured. The current design has an interactive calendar which uses a colour code to show which days have log entries and vlog entries, and allows visitors to click through to each day with ease.
Due to the upcoming bank holiday weekend, very little lab work could be done today. CFUs were measured for spread plates that were exposed to light yesterday, and concentrations of the DNAse1 miniprep were taken so it can now be sent off for sequencing. Glycerol stocks were made of Killer Red 3 in DH5α and Lysozyme in BL21DE3.
August 30th
Log entry
We mailed the Killer Orange spread plate to the Glasgow iGEM team for the collaboration, and they’ll be posting the Z1 lac-repressor E. coli strains to us tomorrow morning. Dr Tom Ellis responded to an email from Jack, giving his definition of a kill switch and bringing up potential ways to build upon the project if more time was available (like constructing an almost 100% efficient operon system).
Joel designed a title banner for the wiki over the weekend which incorporated our colour scheme, logo and title (pictured just below).
Today’s lab team consisted of Dan, Emily, Pablo and Joel. They prepared cultures of Killer Red and Killer Orange for an upcoming experiment, and cleaned the mini stat chambers so they can be autoclaved and then used again.
They transformed DNAse1 and sent samples of the miniprep off for sequencing. They also prepared overnights of Killer Red and Killer Orange for repeat experiments, and performed overnights for the Interlab Study.
August 31st
Log entry
The Glasgow iGEM team posted to Z1 lac-repressor E. coli strains to us this morning, so we should be receiving them sometime in the next few days.
Today’s lab team - Dan and Pablo - induced cultures from yesterday’s overnights of Killer Red and Killer Orange, and diluted the cultures induced yesterday for light box testing. After 6 hours of exposure, they made spread plates of said cultures.
September 1st
Log entry
Emily has been working on further developing the module idea, as the suggestion from Dr John Love is to develop both a second and third year synthetic biology module. Whilst it will be a lot of work to develop a whole second module in such short time, it will be incredible if they are actually implemented. Pablo and Leanne began working on infographic designs to go up on the wiki, looking predominantly on visually explaining the mechanism of action of Killer Red/Orange, Lysozyme and DNAse1.
We sent off our Interlab Study data today to iGEM, in anticipation for the deadline tomorrow. We had decided to do the Interlab from the start as it would be a good way to introduce non-bio team members to the lab procedures, but when we chose Measurement track it also became a compulsory requirement.
Dan, Emily and Joel worked in the lab today. Killer Red and Killer Orange samples were diluted, exposed to light for 6 hours and spread plated. The construct overnights were grown from a starting OD of 0.02 for 6 hours then put through the FACS machine.
September 2nd
Log entry
Pablo continued to work on infographic designs, sketching up neater drawings of the step by step process involved in the mechanism of action for Killer Red/Orange, Lysozyme and DNAse1. Leanne began to look at potential schools we can contact for distributing our board game, BioMech, as most schools as starting back within a week.
Andy worked on writing up what he had done so far for modelling, so that was ready to be put up on the wiki. Emily continued to work on developing ideas for the prospective third year module and trying to find ways that the third year module could build upon the second year one.
Emily and Pablo worked in the lab today - they did CFU counts on plates. Two of the spread plates - RFP 10-3 in the dark and KO 10-5 not induced - were found to be anomalous due to an issue with the spread plates used.
September 5th
Log entry
Today was a day that focused on writing up wiki content. Dan uploaded and started to analyse the Killer Red and Killer Orange data from the last few weeks, and typed up the latest few lab book entries.
Emily had a meeting with Dr John Love and Dr Paul James about the second and third year modules we are developing - the meeting was very positive, with them giving advice on how to improve on the current module ideas. She also worked on the wiki home page, improving on what is currently written.
Leanne worked on writing up sections for the Human Practises page on the wiki - she typed up about the science fairs (Big Bang Science Fair and Britain Needs Scientists Fair), the work experience students who have joined us through the summer (Barnaby, Juliet and Jorge) and the data analysis on the BioMech survey we conducted at Colyton Grammar School.
Today’s lab team - Emily and Dan - prepared and autoclaved the ministat chambers so they are now ready for use, and set up a new configuration using 1L Duran bottles to deal with previous effluent problems. They also prepared overnights of Killer Red, Killer Orange, Lysozyme, RFP and the wild type.
