Team:Exeter/Parts
Lab
Lab Project
Lab Book
Safety
Parts
Team
InterLab
Human Practices
Integrated
Public Engagement
& Education
Log
Attributions
Awards
Awards
Special pages
HP Silver
HP Gold
Models
Collaborations
0
Parts
KillerRed KillerOrange Lysozyme DNase
KillerRed
Name
pT7- E. coli optimised - KillerRed (EOKR)
Description
KillerRed is a red fluorescent protein that generates reactive oxygen species after illumination with light between the wavelengths of 540-580 nm[1].
We further characterised this kill switch by illuminating induced cultures 24 hours after induction with IPTG as well as uninduced cultures. After 6 hours in the light box CFU’s were counted to determine if the kill switch was successful. This was also reproduced on cultures grown in a ministat for 120 and 168 hours to test how long the kill switch remains functional. The full protocol can be found here and the results can be found here .
The mechanism by which KillerRed kills cells isn’t fully understood yet
Here we are submitting KillerRed as a composite part under a T7 promoter (BBa_I712074), an Elowitz ribosome binding site (BBa_B0034) and a double terminator (BBa_B0015).
The sequence for KillerRed protein coding region can be found here (BBa_K1141002)
Biobrick Code
BBa_K1914003
KillerOrange
Name:
pT7- E. coli optimised - KillerOrange (EOKO)
Description:
KillerOrange is a mutant of the fluorescent protein KillerRed (BBa_K1141002, BBa_K1491015) activated by blue and green light. It carries a tryptophan-based chromophore that is novel for photosensitizers [2].
KillerOrange has an excitation maximum of 512 nm and emission maximum at 555 nm and its absorbance spectrum has two peaks, at 455 and 514 nm.
The mechanism by which KillerOrange kills cells isn’t fully understood yet. However, it is believed that KillerOrange's ability to generate reactive oxygen species depends on a water-filler channel reaching the chromophore area from the end cap of the ß-barrel [3].
We characterised this part in the same way as KillerRed ( protocol ), and the results of which can be found here
Here we are submitting KillerOrange as a composite part under a T7 promoter (BBa_I712074), an Elowitz ribosome binding site (BBa_B0034) and a double terminator (BBa_B0015).
The sequence for KillerOrange protein coding region can be found here (BBa_K1914000)
Biobrick Code
BBa_K1914001
Lysozyme
Name:
pT7 Lysozyme E. coli codon optimised, signal peptide, flag tag
Description:
Lysozyme  from chicken egg white, attacks the cell wall in bacteria by hydrolysing the ß-1,4 linkages between N-acetylmuramic acid and N-acetylglucosamine of peptidoglycan [4].
Characterisation of this part involved using the Enzcheck assay kit which detects lysozyme activity as a measure of fluorescence as well as using the same protocol we used to characterise KillerRed ( protocol )and the results of which can be found here
Here we are submitting lysozyme as a composite part under a T7 promoter (BBa_I712074), an Elowitz ribosome binding site (BBa_B0034) and a double terminator (BBa_B0015).
We have codon optimised the protein coding region, added a FLAG tag and exchanged the native signal peptide for one which is E. coli specific and directs Lysozyme to the periplasm [5]. The sequence for Lysozyme protein coding region can be found here (BBa_K1914004)
Biobrick Code
BBa_K1914005
DNase
Name:
DNase
Description:
DNase was designed [6] as a DNA degrading kill switch aiming to prevent cross contamination of DNA
A composite part containing DNase could not be created. This could be due to the need for a tightly regulating promoter to prevent DNase protein being produced before induction and destroying DNA.
References
Takemoto, K., Matsuda, T., Sakai, N., Fu, D., Noda, M., Uchiyama, S., Kotera, I., Arai, Y., Horiuchi, M., Fukui, K. and Ayabe, T., 2013. SuperNova, a monomeric photosensitizing fluorescent protein for chromophore-assisted light inactivation. Scientific reports, 3.
Sarkisyan, K.S., Zlobovskaya, O.A., Gorbachev, D.A., Bozhanova, N.G., Sharonov, G.V., Staroverov, D.B., Egorov, E.S., Ryabova, A.V., Solntsev, K.M., Mishin, A.S. and Lukyanov, K.A., 2015. KillerOrange, a Genetically Encoded Photosensitizer Activated by Blue and Green Light. PloS one,10(12), p.e0145287.
Pletnev, S., Gurskaya, N.G., Pletneva, N.V., Lukyanov, K.A., Chudakov, D.M., Martynov, V.I., Popov, V.O., Kovalchuk, M.V., Wlodawer, A., Dauter, Z. and Pletnev, V., 2009. Structural basis for phototoxicity of the genetically encoded photosensitizer KillerRed. Journal of Biological Chemistry, 284(46), pp.32028-32039.
Blake, C.C.F., Koenig, D.F., Mair, G.A., North, A.C.T., Phillips, D.C. and Sarma, V.R., 1965. Structure of hen egg-white lysozyme: a three-dimensional Fourier synthesis at 2 Å resolution. Nature, 206(4986), pp.757-761.
Fischer, B., Perry, B., Phillips, G., Sumner, I. and Goodenough, P., 1993. Physiological consequence of expression of soluble and active hen egg white lysozyme in Escherichia coli. Applied Microbiology and Biotechnology, 39, pp.537–540.
Chen, C.Y., Lu, S.C. and Liao, T.H.,1998. Cloning, sequencing and expression of a cDNA encoding bovine pancreatic deoxyribonuclease I in Escherichia coli: purification and characterization of the recombinant enzyme. Gene, 206, pp.181–184.
