<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU-Trondheim_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU-Trondheim/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU-Trondheim/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU-Trondheim/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU-Trondheim/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Trondheim iGEM 2013 </TITLE></P><DIV class="layout-1000"><DIV class="row"><DIV id="cssmenu"><UL><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim"><SPAN>Home</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Project</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Project"><SPAN>Project description</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/novelapproach"><SPAN>A novel approach</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Model"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results"><SPAN>Experiments and Results</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Parts"><SPAN>BioBrick Parts</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Acknowledgements"><SPAN>Acknowledgements</SPAN></A></LI></UL><LI class="has-sub"><A href="#"><SPAN>Technical Stuff</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Notebook"><SPAN>Notebooks</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Primers"><SPAN>Primers</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Team</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Team"><SPAN>Meet the team</SPAN></A></LI><LI class="last"><A href="https://igem.org/Team.cgi?id=1082"><SPAN>Official Team Profile</SPAN></A></LI><LI class="has-sub"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Outreach"><SPAN>Outreach</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Judging</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Achievements"><SPAN>Achievements</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Medalcriteria"><SPAN>Medal criteria</SPAN></A></LI><LI class="last"><A href="https://2012.igem.org/Team:NTNU_Trondheim/Matchmaker"><SPAN>Matchmaker</SPAN></A></LI></DIV></DIV><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B> Project </B></DIV><P>All gram negative bacteria produce outer membrane vesicles (OMVs) by bulging of from their outer membrane. These OMVs contain different proteins and carry out varies functions such as quorum sensing, involvement in pathogenesis and transporting enzymes to distal locations. In our project we wish to prove that OMVs can be manipulated and to thereby implicate OMVs potential as a drug delivery vehicle. This is an innovative and <A href="https://2013.igem.org/Team:NTNU-Trondheim/novelapproach"> novel approach </A>to create a safe way to deliver drugs in the body. 

We are going to prove that the OMVs can be manipulated by introducing two different proteins. The first one is a fusion protein of GFP and RFP. This will make it possible to visualize the vesicles as well as investigating the new functionalities of fusing two fluorescent proteins together. The second protein we wish to introduce to the vesicles is the transmembrane protein G derived from <I> Streptococcus dysgalactiae ssp. equisimilisi</I>. Protein G is known to bind to Human Serum Albumin (HSA) which helps <I>S.dysgalactiae subsp. equisimilis</I> hide from the immune system. Protein G will therefore be a potential important piece in a drug carrier by masking it from immunological destruction and making it stable in the blood stream. 

In order to direct the proteins into the periplasm and vesicles they need to have a tat signal peptide at the N-terminal. This will transport the proteins through twin-arginine translocation pathway (Tat pathway. Studies show that a twin-arginine signal peptide is able to direct the export of active green fluorescent protein (GFP) in <I> Escherichia coli </I>. We also wish to be able to regulate the production and thereby the export of protein to OMVs. In order to accomplish this task we will set the gene constructs under regulation of the Pm/Xyls promotor system. This is a positive regulation system that gets activated by the inducer m-toluic acid

Introducing these proteins into vesicles also demonstrate that it is indeed possible to manipulate the content and therefore the properties of OMV's. We will prove the concept that OMVs can be engineered to meet the criteria of a drug delivery vehicle. </P></DIV></DIV><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B> Bacterial Outer Membrane Vesicles </B></DIV><P>Gram negative (and some gram positive) bacteria produce outer membrane vesicles (OMV) by outward bulging of the outer plasma membrane. These OMV has several properties such as quorum sensing, biofilm development, involvement in pathogenesis (some for pathogenic bacteria), DNA transfer and transport of enzymes to distal locations <A href="http://www.ncbi.nlm.nih.gov/pubmed/20825345">[1]</A>. The OMVs contain different proteins on their membrane surface and within their membrane with the most abundant proteins being OmpF, OmpC and OmpA <A href="http://www.ncbi.nlm.nih.gov/pubmed/16855227">[2]</A>. When running a SDS-PAGE on a vesicle sample a protein distinct pattern will arise as illustrated in the figure below:</P></DIV><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 1:</B> A typical SDS-PAGE of a purified vesicle sample.</P></DIV></DIV>



Research indicates that the proteins that end up in the vesicles and what lipids the vesicles consist of are not random. The composition of proteins and lipids in the vesicles are exactly the same as in the outer membrane. This suggests there are some sort of mechanism that determines what goes in the vesicles. The excact machanism is unkown <A href="http://www.ncbi.nlm.nih.gov/pubmed/20825345">[1]</A>. 

The ability of some pathogenic bacteria to use their OMVs in pathogenesis is particularly interesting. <I>Pseudomonas Aeruginosa</I> and pathogenic <I>E.coli</I> has been studied to some extent for their OMVs role in pathogenesis.  These vesicles contains virulence factors such as proteases, pro-inflammatory proteins and toxins, and attack eukaryotic cells by binding on the surface and then internalize (see figure below) <A href="http:http://www.ncbi.nlm.nih.gov/pubmed/15549136">[3]</A>. <DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 2:</B> Gold labeled vesicles from enterogenic  <I>E.coli</I> that binds and internalizes in HT29 cancer cells <A href="http:http://www.ncbi.nlm.nih.gov/pubmed/15549136">[3]</A>.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>


The vesicles will display a lot of the same proteins as the bacteria as they bulge of from the outer membrane. This trait, and the fact that the vesicles will not be able to multiply on their own, has qualified them as a possible candidate for vaccines. Engineered vesicles has been shown to be immunogenic in mice, but more research remain before OMV as vaccines can be tested on humans <A href="http://www.pnas.org/content/early/2010/01/14/0805532107">[4]</A>].</P><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B> Potential as a drug delivery vehicle</B></DIV><P>Whereas OMVs function and contents has been studied for decades, their potential as a drug carrier has not previously been investigated. As it is still many obstacles for an effective production method of different drug carriers within the field of nano-medicine, exploiting gram negative bacteria OMV production could be a possibility for drug delivery in the future. Making drug carriers synthetically in small sizes is difficult and the vesicles measure only 20-200 nm in size which makes them attractive for this purpose. Especially vesicles from pathogenic bacteria could be further manipulated as these vesicles naturally can attack human eukaryotic cells. The issue is that they contain PAMP's (Pathogen Associated Molecular Patterns) that are recognized by the immune system. Gram negative bacteria contain LPS (Lipopolysaccharide) in their outer membrane that is recognized as foreign by immune cells. To stop the immune cells from attacking the vesicles we propose to introduce the outer membrane protein,Protein G, into the vesicles. This protein binds Human Serum Albumin (HSA) which tricks the immune system into thinking the vesicles are harmless blood cells. This will make the vesicles more suitable as a drug delivery vehicle.  </P><DIV class="row"><DIV align="justify"><DIV class="col6-2"><P style="text-align:center; color:black; "><B>Figure 3:</B> Immune cells recognize and attack vesicles.</P></DIV><DIV class="col6-2"><P style="text-align:center; color:black; "><B>Figure 4:</B> Protein G binds Human Serum Albumin and masks the vesicles from the immune system.</P></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B> Fluorescent Proteins</B></DIV><P>The first fluorescent protein (FP) to be discovered was the green fluorescent protein (GFP) isolated from <I>Aequorea victoria</I>. Many other different fluorescent protein (FP) with different colours (red, cyan, yellow and other version of these) has since been created by mutating GFP. The FPs has been crucial in terms of visualising cells and determining intracellular proteins by fusing the FPs to proteins of interest. Having a variety of FPs gives scientists the opportunity to study different proteins at the same time.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" http:=""><P style="text-align:center; color:black; "><B> Figure 5:</B> Fluorescent Proteins.</P></DIV></DIV></DIV><P>The discovery of green fluorescent protein in the early 1960s eventually created a new era in cell biology by assisting investigators to apply molecular cloning methods, fusing the fluorophore moiety to a wide variety of protein and enzyme targets, in order to monitor cellular processes in living systems using optical microscopy and related methodology<A href="http://www.ncbi.nlm.nih.gov/pubmed/20664080">[5]</A>.</P><P>Fluorescent proteins are structurally homologous proteins that share the unique property of being self-sufficient to form a visible wavelength chromophore from a sequence of 3 amino acids within their own polypeptide sequence. Biologists introduce a gene (or a gene chimera) encoding an engineered fluorescent protein into living cells and subsequently visualize the location and dynamics of the gene product using fluorescence microscopy.</P><P>The presence of a fluorescent component in the bioluminescent organs of Aequorea victoria jellyfish was introduced by Davenport and Nicol in 1955, but it was Osamu Shimomura who was the first to recognize that this fluorophore is actually a protein. He mentioned in one of his papers <I>A protein giving solutions that look slightly greenish in sunlight though only yellowish under tungsten lights, and exhibiting a very bright, greenish fluorescence in the ultraviolet of a Mineralite that has been isolated from squeezate</I>. <I>Squeezate</I> describing a solution that resulted from the squeezing of the excised bioluminescent tissues of the jellyfish through a cotton bag. After that Shimomura reported the fluorescence emission spectrum of this protein and also he suggested that energy transfer from aequorin to this green fluorescent protein could explain why the in vivo luminescence of Aequorea is greenish and not blue like the luminescence of purified aequorin<A href="http://onlinelibrary.wiley.com/doi/10.1002/jcp.1030590302/abstract ">[6]</A>.</P><P>The gene for green fluorescent protein was first cloned in 1992, but the important possibility to be used as a molecular probe was not achieved until several years later when fusion products were used to track gene expression in bacteria and nematodes. Green fluorescent protein has been engineered to produce numbers of various colored mutants, fusion proteins, and biosensors which are broadly referred to as fluorescent proteins. Recently, fluorescent proteins from other species are also identified and isolated, resulting in further expansion of the color palette. With the rapid evolution of fluorescent protein technology, the utility of this genetically encoded fluorophore for a wide spectrum of applications beyond the simple tracking of tagged biomolecules in living cells is now becoming fully appreciated.</P><P>A broad range of fluorescent protein genetic variants have been developed that feature fluorescence emission spectral profiles spanning almost the entire visible light spectrum. Mutagenesis efforts in the original Aequorea victoria jellyfish green fluorescent protein have resulted in new fluorescent probes that range in color from blue to yellow, and are some of the most widely used in vivo reporter molecules in biological research. Longer wavelength fluorescent proteins, emitting in the orange and red spectral regions, have been developed from the marine anemone, Discosoma striata, and reef corals belonging to the class Anthozoa. Still other species have been mined to produce similar proteins having cyan, green, yellow, orange, and deep red fluorescence emission. Developmental research efforts are ongoing to improve the brightness and stability of fluorescent proteins, thus improving their overall usefulness<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1181180/ ">[7]</A>.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" http:=""><P style="text-align:center; color:black; "><B>Figure 6:</B> Fluorescent Protein Excitation and Emission Spectra.</P></DIV></DIV><DIV class="row"><DIV class="col12-2"><P>
Fluorescence is basically a color-resolved technique, to choose a right FP is important to consider its spectral profile, that is, the color of its fluorescence. A broad range of FP variants that span nearly the entire visible spectrum has been developed and optimized. For comparing the brightness of various FPs, each normalized excitation spectrum is multiplied by its peak molar extinction coefficient, and then divided by the peak molar extinction coefficient of EGFP. Likewise, each normalized emission spectrum is multiplied by its molecular brightness (molar extinction coefficient × quantum yield) and then divided by the brightness of EGFP. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1181180/">[8]</A>.

Choosing of the best-performing FPs in each color class are based on a number of critical factors, including maturation efficiency, spectral properties, photostability, monomeric character, brightness, fidelity in fusions and potential efficiency as a Förster resonance energy transfer (FRET) donor or acceptor.<A href="http://www.nature.com/nchembio/journal/v5/n2/abs/nchembio.138.html">[9]</A>.

The blue, green, and yellow FPs appear to have reached their potential, but improvements are likely to continue for orange, red and combinations of different FPs. Each FP has its own defined emission and excitation spectrum. </P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" http:=""><P style="text-align:center; color:black; "><B>Table 1:</B> Fluorescent Protein Excitation and Emission Wavelength.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>By using multi-color fluorescence microscopy, FPs are often used in combination to examine interactions between their fusion partners. 
In the second part of our project, we want to create new combinations of FPs. We intend to make this happen by fusing different FPs together into a new construct, namely GFP, red fluorescent protein (RFG), cyan fluorescent protein (CFP) and yellow fluorescent protein (YFP). This will give us new combinations of FPs with new absorption spectrum. The goal of this work was to expand the color palette of FPs with variants exhibiting improved brightness and contrast relative to single FP.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 7:</B> Our Tat_GFP_RFP construct.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B>Protein G</B></DIV><P> Protein G is a transmembrane protein (se figure below) found on the surface of the opportunistic pathogen  <I>Streptococcus dysgalactiae ssp. equisimilis)</I> group C and G <A href="”http://www.ncbi.nlm.nih.gov/pubmed/21098039”">[10]</A>. The fully sequenced protein G gene comes from <I>S. dysgalactiae ssp. equisimilis)</I><A href="”http://www.ncbi.nlm.nih.gov/nuccore/X06173”"> strain 148</A> and has a gene length of 1679 bp and a protein size of 593 amino acid and a mass of 63.292 kDa <A href="”http://www.ncbi.nlm.nih.gov/nuccore/X06173”"> [11]</A>. Protein G is usually not a fixed size and can vary from strain to strain <A href="">[12]</A>. The main reason that Protein G is attractive for our project is its ability to bind to human serum albumin (HSA), the most abundant protein in the blood. This binding has proven to help masking the <I>S. dysgalactiae ssp. equisimilis)</I> cells from the immune system  by creating a outer layer of HSA <A href="”http://www.ncbi.nlm.nih.gov/pubmed/21098039”">[10]</A>. Protein G is therefore considerd to be involved in, but not the determinator of pathogenesis.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B> Figure 8:</B> Protein G binds to human serum albumin (HSA) and masks <I>S. dysgalactiae ssp. equisimilis)</I> from the immune system.</P></DIV></DIV></DIV><DIV class="row"><DIV class="row"><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B> Twin-Arginine Translocation System </B></DIV><P>Transport of proteins is an essential part of cellular life. Proteins destined for transmembrane transportation
are usually synthesized with amino-terminal signal sequences, signal peptides, that direct the proteins to 
specific membrane transporter complexes. During the transport/export the signal peptide is cleaved from the 
protein.


Gram negative bacteria export numerous proteins into the periplasm. The twin-arginine translocation pathway (Tat pathway) is a protein export, or secretion pathway found in plants, bacteria, and archaea. This transport across membranes is via distinct signal peptides with twin-arginine motifs. In contrast to the Sec pathway which transports proteins in an unfolded manner, the Tat pathway serves to actively translocate folded 
proteins across a lipid membrane bilayer. <I>In vitro</I> studies (Clark and Theg 1997; Hyndis et al. 1998) confirms that the Tat system is able to transport fully folded proteins in bacteria. Several tat genes
have been identified, and in <I>E. coli</I> the export pathway requires four integral membrane proteins, TatA, TatB, 
TatC and TatE. Other componenets cannot be ruled out. Studies show that a twin-arginine signal peptide 
is able to direct the export of active green fluorescent protein (GFP) in <I>E. coli</I> and that translocation 
almost exclusively occur by the Tat-pathway. As GFP is unable to fold in the periplasm it is assumed that its 
translocation must occur in folded conformation <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2001.02253.x/pdf">[13]</A>.

Taking this information into account we discussed the possibility of transporting different proteins into the 
periplasm of bacteria, and what this may be useful for. As all gram negative bacteria produce outer membrane 
vesicles (OMV's) we looked into the content of these vesicles. Was the sorting of proteins random? Could we 
direct certain proteins toward them? And what function would that give? Can we use this to our advantage?

Tat targeting signals exhibit a structure, like Sec signal peptides, comprising of three regions. A polar 
'n-region', a hydrophobic 'h-region', and a polar 'c-region'. The twin-arginine motif is conserved and always
located between the 'n-', and 'h-region'(Berks, 1996). The 'c-region' often contain proline residues, a 
positive charge and an AA cleavage site. It is believed that the positive charge prevents mis-targeting 
of Tat signal peptides to the Sec pathway (Bogsch et al., 1997)<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2005.00049.x/pdf">[14]</A>.

A commonly used signal peptide is trimethylamine N-oxide reductase (TorA) from <I>E. coli</I>. The TorA signal 
peptide is 39 aminoacids long giving this base sequence of 117 nucletides: 

5' ATGGCCAATAACGATCTCTTTCAGGCATCACGTCGGCGTTTTCTGGCACAACTCGGCGGC
TTAACCGTCGCCGGGATGCTGGGGCCGTCATTGTTAACGCCGCGACGTGCGACTGCGGCG
CAAGCG 3'
<TABLE style="border:1px solid black;"><TBODY><TR><TH>Forward Primer</TH><TH>Reverse Primer </TH><TH>T<SUB>m</SUB></TH></TR><TR><TD>5'AGAGAAAGAGGAGAAATACTAGATGGCCAATAACGATCTCTTTCAGGCATCACG 3'(AAC)</TD><TD>5'GAATTCGCGGCCGCTTCTAGATGGCCAATAACGATCTCTTTCAGGCATCACG 3'(AAC)</TD><TD>72°C</TD></TR></TBODY></TABLE>

The first 22 bases of the forward primer and the first 20 bases of the reverse primer are linker sequences. The forward primer was designed to make an overlap with the backbone plasmid and the reverse overlaps with the linker on GFP.


TorA has successfully been fused with the coding region of GFP's in previous studies. The construct was 
further cloned into a vector plasmid and introduced in <I>E. coli</I>. The construct, when expressed will then 
have the GFP attatched to TorA signal peptide which will direct the protein through the Tat pathway into
the periplasm of the cell. In the periplasm the signal peptide is cleaved of. The GFP is then mainly found 
in the cell's periplasm, not in the cytoplasm <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2001.02253.x/pdf">[13]</A>.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 9:</B> Visualization of GFP in the periplasm of the cell using fluorescence microscopi.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>When in the periplasm the GFP may also be included in the vesicles as they budd of. The contents of the 
vesicles vary, but is believed to be regulated in some way. The proteins that reach the periplasm is 
transported there by signal peptides for a reason. Knowing this it would be possible to transport other 
proteins to the periplasm and direct them toward the vesicles. As the vesicles are not a replicative 
unit like bacteria or viruses it may be a perfect transport vessel for drugs, vaccines or other medical 
applications. This is the quality that we wish to examine further. Can we introduce surface-proteins on 
the vesicles that mask them from the immune system? What about a cancer cell recognition type of construct? 
Program the vesicles to release content only when in contact with harmful cells?</P></DIV><DIV class="row"><DIV class="col12-2"><DIV class="col8" align="justify" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B>The Pm/Xyls Promotor system </B></DIV><P>The Pm promoter originate from ''Pseudonomas putida''TOL-plasmid pWW0 <A href="http://www.ncbi.nlm.nih.gov/pubmed/4418209"> [15]</A>. The Pm/XylS expression system is positively regulated by m-toluic acid. The m-toluic acid binds to the XylS protein which is constitutively expressed by the Pm/XylS promoter system. The XylS-m-toluic acid complex binds to the promoter, and activates it. 

</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 10:</B> Overview of how the Pm/Xyls Promotor system funtions. Production of recombinant protein is dependent on access to the indicer m-toluic acid.</P></DIV></DIV><DIV class="row"><DIV class="col12-2"><P>There is a positive correlation between the inducer concentration and the protein production. By increasing the inducer concentration, the expression system will be stimulated and produce more product. This opportunity to control the amount of protein produced, makes it an expression system which is often used, especially here at NTNU. It is  also shown to work in different bacterias(<I>Pseudomonas subspecies</I>, <I>Aeromonas hydrophila</I>, <I>Aerobacter aerogenes</I>, <I>Serratia marcescens</I>, <I>Erwinia carotovora</I> and importantly, <I>Escherichia coli</I> <A hraf=" http://www.ncbi.nlm.nih.gov/pubmed/3525513">[16]</A> .
The properties of the Pm/XylS promoter system, was an important part in making this in to a BioBrick. It could then be used together with our GFP-RFP construct and regulate how much of the dimer got produced. </P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NTNU-Trondheim/Project">http://2013.igem.org/Team:NTNU-Trondheim/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU-Trondheim/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU-Trondheim/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Project&amp;oldid=305128" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>