"
Page
Discussion
View source
History
teams
Log in
Â 
Team:NTNU-Trondheim/Achievements
From 2013.igem.org
Trondheim iGEM 2013
Home
Project
Project description
A novel approach
Modelling
Experiments and Results
BioBrick Parts
Acknowledgements
Technical Stuff
Notebooks
Protocols
Primers
Safety
Team
Meet the team
Official Team Profile
Outreach
Judging
Achievements
Medal criteria
Matchmaker
Achievements
Gibson Assembly and transformation
Our overlapping DNA fragments; tat, GFP, RFP and plasmid backbone was cloned together by Gibson Assembly and transformed into E.col strain ER2566 cells. The photograph below shows two of the resulting colonies (named ER1 and ER2) from this transformation.
Figure: Two of the transformed colonies with the tat_GFP_l_RFP construct. Hereby named ER1 and ER2.
Sequencing and characterization
The plasmid from the ER1 samples was isolated by the Promega Wizard Plus SV Minipreps DNA Purification System A1460 and sequenced. Figure below shows the alignment of the sequencing results with the reference DNA sequence.
Figure: Aligment of tat_GFP_l_RFP (ER1) with reference DNA.
The sequence align almost perfectly. There seems to be some sort of extra insert at the linker region, but this insert is dividable by 3, so the reading frame is maintained. This is supported by the fact that the colonies with this construct is red (see figure above). Sequencing confirms that we have succesfully made our tat_GFP_RFP construct.
Red ER1-cells in liquid media was immobilized in agar and then viewed in a confocal microscope for seeing if RFP was localized in the periplasm. The results can be seen in the two figures below:
Figure:Red ER1-cells viewed in confocal microscope in 2D
Figure:Red ER1-cells viewed in confocal microscope in 3D
Pm/xylS promoter
The Pm/xylS promoter was modified (removing an XbaI site) and turned into a BioBrick by adding the prefix and suffix. With 3A assembly, the promoter was attached to a GFP generator (BBa_E0240), and a backbone ( pSB3K3).  The results show that the GFP are expressed in the samples with the Pm/xylS promoter, indicating that the promoter works. The figures below show the result from the testing. As expected, the reference promoter has a high rate of GFP synthesis.
Figure X: The ratio of GFP synthesis as a function of inducer concentration
Figure X: The ratio of GFP synthesis as a function of time
Protein G
We have introduced Protein G from Streptococcus dysgalactiae ssp equisimilis into E.coli where it is succesfully expressed.
Figure 15:Alignment of the full tat_ProteinG construct.
The Protein G gene from the Streptococcus dysgalactiae ssp equisimilis sample we collected from the hospital is clearly missing two bigger segments compared to the known gene sequence . We conclude that this is Protein G as the rest of the sequence aligns well.
To check weather Protein G is even produced in the S3-2B cells we did a SDS-PAGE with these cells along with a wildtype ER2566 cell sample (see figure below):
Figure 17:SDS-PAGE of wildtype (WT) ER2566 cells and ER2566 S3-2B cells with tat_ProteinG construct. Ladder applied is Precision Plus ProteinTM Unstained Standards.
There is a very clear additional band on the S3-2B sample of about 60 kDa in protein mass. Highly indicating that tat_Protein G is produced in the cells.
The conclusion is that Protein G is obviously produced in the E.coli strain ER2566 cells, but the tat signal peptide seems to fail to direct the Protein G into the periplasm and OMVs.
Retrieved from " http://2013.igem.org/Team:NTNU-Trondheim/Achievements "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
