<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU-Trondheim_Experiments_and_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU-Trondheim/Experiments_and_Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU-Trondheim/Experiments_and_Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Experiments_and_Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Experiments_and_Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU-Trondheim/Experiments_and_Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU-Trondheim/Experiments and Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Trondheim iGEM 2013 </TITLE></P><DIV class="layout-1000"><DIV class="row"><DIV id="cssmenu"><UL><LI class="active"><A href="https://2013.igem.org/Team:NTNU-Trondheim"><SPAN>Home</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Project</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Project"><SPAN>Project description</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/novelapproach"><SPAN>A novel approach</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Model"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results"><SPAN>Experiments and Results</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Parts"><SPAN>BioBrick Parts</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Acknowledgements"><SPAN>Acknowledgements</SPAN></A></LI></UL><LI class="has-sub"><A href="#"><SPAN>Technical Stuff</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Notebook"><SPAN>Notebooks</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Primers"><SPAN>Primers</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Team</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Team"><SPAN>Meet the team</SPAN></A></LI><LI class="last"><A href="https://igem.org/Team.cgi?id=1082"><SPAN>Official Team Profile</SPAN></A></LI><LI class="has-sub"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Outreach"><SPAN>Outreach</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Judging</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Achievements"><SPAN>Achievements</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Medalcriteria"><SPAN>Medal criteria</SPAN></A></LI><LI class="last"><A href="https://2012.igem.org/Team:NTNU_Trondheim/Matchmaker"><SPAN>Matchmaker</SPAN></A></LI></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B>Creating the tat_GFP_l_RFP construct</B></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B> PCR for amplification of tat, plasmid backbone, GFP and RFP</B></DIV><P>
There were essentially four DNA pieces that we wanted to combine together to make a construct: The tat signal sequence followed by GFP, a small linker region and RFP put into a plasmid backbone. As our cloning techniques rely on overlapping DNA fragments we used mostly primers with overhengs in the PCR reactions. As templates we used the biobricks <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> for RFP, <A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A> for GFP, <A href="http://parts.igem.org/Part:BBa_J01101">BBa_J01101</A> for the plasmid backbone and genomic DNA from <I>Escherichia coli</I> strain ER2566 for the tat signal sequence.<B>Table 1:</B> Primers applied in creating the tat_GFP_l_RFP construct. Lowercase letters indicate DNA that anneal to the template whereas the uppercase letters indicate DNA which serves as an overhang.
<TABLE style="border:1px solid black;"><TBODY><TR><TH>Amplifing</TH><TH>Primer</TH><TH>Sequence</TH></TR><TR><TD>tat</TD><TD>F_pl.b_tat</TD><TD>AGAGAAAGAGGAGAAATACTAGatggccaataacgatctctttcaggcatcacg</TD></TR><TR><TD>tat</TD><TD>R_tat</TD><TD>cgcttgcgccgcagtcgcacgtcg</TD></TR><TR><TD>GFP</TD><TD>F_tat_GFP</TD><TD>CGACGTGCGACTGCGGCGCAAGCGatgcgtaaaggagaagaac</TD></TR><TR><TD>GFP</TD><TD>R_l_GFP</TD><TD>ACTACCACCGGATCCACCTGATCCACCGGATCCACCtttgtatagttcatccatgcc</TD></TR><TR><TD>RFP</TD><TD>F_l_RFP</TD><TD>GGTGGATCCGGTGGATCAGGTGGATCCGGTGGTAGTatggcttcctccgaagacg</TD></TR><TR><TD>RFP</TD><TD>R_pl.b_RFP</TD><TD>GCCTTTCGTTTTATTTGATGCCTGGgcgatctacactagcactatcagcg</TD></TR><TR><TD>Plasmid backbone</TD><TD>F_pl.b</TD><TD>ccaggcatcaaataaaacgaaagg</TD></TR><TR><TD>Plasmid backbone</TD><TD>R_pl.b</TD><TD>ctagtatttctcctctttctctagtagtgc</TD></TR></TBODY></TABLE>

The linker region is going to be 36 bp long and will be created by overlapping overhangs on the reverse primer of GFP and forward primer of RFP. The primers for the plasmid backbone is designed to include the TetR repressible promoter (<A href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A>), RBS (<A href="http://parts.igem.org/Part:BBa_R0034">BBa_R0034</A>) and two terminators (<A href="http://parts.igem.org/Part:BBa_R0010">BBa_R0010</A>) and (<A href="http://parts.igem.org/Part:BBa_R0012">BBa_R0012</A>) in the PCR product.
All of the PCR products were treated with the enzyme DpnI that digests methylated DNA and purified by the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#PCR_Purification"> QIAquick PCR Purification kit</A>.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col8" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>Gibson Assembly and transformation</B></DIV><P>
Our overlapping DNA fragments; tat, GFP, RFP and plasmid backbone was cloned together by <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Gibson_Assembly"> Gibson Assembly</A> and <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Transformation"> transformed</A> into <I>E.col</I> strain ER2566 cells. The photograph below shows two of the resulting colonies (named ER1 and ER2) from this transformation.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 1:</B> Two of the transformed colonies with the tat_GFP_l_RFP construct. Hereby named ER1 and ER2.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col8" align="justify" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>Sequencing and characterization</B></DIV><P>The plasmid from the ER1 samples was isolated by the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Transformed_cells"> Promega Wizard Plus SV Minipreps DNA Purification System A1460</A> and sequenced. Figure below shows the alignment of the sequencing results with the reference DNA sequence.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 2:</B> Aligment of tat_GFP_l_RFP (ER1) with reference DNA.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>The sequence align almost perfectly. There seems to be some sort of extra insert at the linker region, but this insert is dividable by 3, so the reading frame is maintained. This is supported by the fact that the colonies with this construct is red (see figure above).

Red ER1-cells in liquid media was immobilized in agar and then viewed in a confocal microscope for seeing if RFP was localized in the periplasm. The results can be seen in the two figures below:</P><DIV class="col4" style="background-color:white;&gt; &lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 3:</B>Red ER1-cells viewed in confocal microscope in 2D </P></DIV><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 4:</B>Red ER1-cells viewed in confocal microscope in 3D </P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There is no indication that the red fluorescence is more concentrated in the periplasm, as we should expect due to the transport through the tat transport pathway.

We performed a excitation and emission scan of the ER1 bacterias together with <I>E-coli</I> strain ER2566 cells that were transformed to produce single RFP. The samples were centrifuged and the pallets were resuspended in DPBSS buffer. These solutions were den studied in a fluorometer(see figures below):</P><DIV class="col6-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 5:</B>Excitation scan of ER1, ER2 and the referance sample with RFP </P></DIV><DIV class="col6-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 6:</B>Emission scan of ER1, ER2 and the referance sample with RFP </P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There are no significant difference between the referance sample and the ER samples. It seems that GFP is not present or doesn't fold properly. All of the samples had an excitation/emission at 584/607 which is the same as for singel RFP (<A href="http://parts.igem.org/Part:BBa_R1010">BBa_1010</A>). The test weather a GFP-RFP dimer is produced or only RFP is produced we ran a SDS-PAGE with the ER1 <I>E.coli</I> strain ER2566 cells along with wild type ER2566 cells (see figure below).</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 7:</B> SDS-PAGE with wildtype (WT) ER2566 cells and ER1 ER2566 cells that has the tat_GFP_l_RFP construct </P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>The mass size of the additional band on the ER1 sample is about 28-30 kDa, close to the size of a single fluorescent protein of 26-30 kDa. We therefor conclude that only single RFP is produced in the ER1 cells as there are no green fluorescence. 


Vesicles were isolated from ER1 cells and wildtype <I>E.coli</I> strain ER2566 cells by the <AHREF> vesicle isolation protocol. 
SDS-PAGE was done on both samples with one diluted sample (1:2) and one undiluted sample (1:1) for both.</AHREF></P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 8:</B>Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards. WT stands for wildtype and is the unstransformed ER2566 samples.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There are no additional bands in the ER1 samples compared to the unstransformed ER2566 sample, indicating that there is no GFP-RFP in the vesicles.
There was run an excitation scan of the undiluted vesicle ER1 sample that was compared to a same scan on vesicles from wildtype bacteria. The results were as shown in the two figures below.</P><DIV class="col6-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 9:</B>Fluorescence excitation scan of vesicles from the ER1 cells and wildtype ER2566 (right).</P></DIV><DIV class="col6-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 10:</B>Fluorescence excitation scan of vesicles from the wildtype ER2566 cells.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There is no real difference between the samples other than strength of the signals. Both samples have peaks at 504, 540 and 582 nm. It is likely that there is no detectable GFP-RFP dimer in the vesicles.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B>Protein G</B></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>PCR for amplification of tat Protein G</B></DIV><P>
Our template for this PCR reaction was genomic DNA from <I>Streptococcus dysgalactiae ssp equisimilis</I> collected from St. Olavs Hospital. We had 5 different samples of <I>Streptococcus dysgalactiae ssp equisimilis</I> and 6 different combinations of primers giving 30 different PCR reactions. Standard conditions for Phusion PCR was used. As indicated in the figure below, only one of these reaction gave a PCR product.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 11:</B>PCR results of Protein G from 30 different PCR reactions.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>
A very clear band with the size of around 1200-1300 bp was found in one of the samples. This is a smaller DNA fragment compared to the <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> known gene sequence</A> of about 1700 bp. Since the size of Protein G is known to vary among the different strains, we can still conclude that our PCR-product is Protein G. The PCR product with the genomic DNA sample 1 as template and F_tat_PrG and R_pl.b_PrGstop (see table below) as primers is most likely Protein G.<B>Table 2:</B> Primers applied in creating the tat_ProteinG construct. Lowercase letters indicate DNA that anneal to the template whereas the uppercase letters indicate DNA that serves as an overhang.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Amplifing</TH><TH>Primer</TH><TH>Sequence</TH></TR><TR><TD>Protein G</TD><TD>F_tat_PrG</TD><TD>CGACGTGCGACTGCGGCGCAAGCGgttgactcaccaatcgaagatacccc</TD></TR><TR><TD>Protein G</TD><TD>R_pl.b_PrGstop</TD><TD>GCCTTTCGTTTTATTTGATGCCTGG ttagtcttctttacgttttgaagcgac</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>Attaching tat signal sequence by PEC</B></DIV><P>For attachment of the tat DNA fragment to the Protein G PCR fragment Polymerase Extension Cloning (<A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#PEC"> PEC</A>) was applied. After the PEC PCR reaction the product was run on a agarose gel together with the PCR product of Protein G without the tat attached.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 12:</B>From the left; Protein G with tat attached and Protein G without tat.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>The PEC product is clearly a little bit longer then the Protein G PCR product without tat attched. This indicates that PEC was successful.
</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>Cloning by Direct Transformation and SLIC</B></DIV><P>The tat_ProteinG fragment was mixed with the same plasmid backbone PCR product that was produced for the tat_GFP_l_RFP construct in varies different cloning reactions: <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> CPEC</A>, <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> SLIC</A> and <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> direct transformation</A>. 18 different reactions was run. The cloning products was transformed into <I>E.coli</I> DH5α cells. The plasmids were then <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> isolated</A> and a conformation PCR was executed with F_pl.b_tat and R_pl.b_PrGstop as primers. The conformation results can be viewed in the figure below:</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 13:</B>First row contains tat_ProteinG samples 1-9, second row contains tat_ProteinG samples 10-18.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>All of the samples that had a PCR product of the expected size of 1200-1300 bp (10 samples) was sent for sequencing. The sequencing results showed that two of the constructs had approximately the DNA sequence they should have. However, these constructs, named Dt8 (created by direct transformation) and S3 (created by SLIC) had an addition and a deletion of a guanine residue in the beginning of the tat sequence, respectively (see figures below). These errors would create a frame shift that would produce a dysfunctional Protein G.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 14:</B>Aligment of tat_ProteinG DT8 (left) and S3 (right). There is one segment missing in our ProteinG, as is expected as the ProteinG variant that we are dealing with is a shorter than the <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> registered gene sequence</A>. There is however, an addition of a guanine residue in the beginning of the tat-sequence in the DT8 sample and a deletion at the same site in the S3 sample, which will cause a frame shift resulting in a non-functional protein product.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>Because it is only possible to sequence approximately 1000 bp, we don't have all of the sequence of our tat_ProteinG construct.
</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>Fixing deletion by PCR</B></DIV><P>In order to correct the Guanin deletion/addition a PCR was run on the DT8 and S3 sample with the primers F_pl.B_tat and R_pl.b_PrGstop. The PCR products was cloned together with the same plasmid backbone in a direct transformation. Plasmids from the transformed <I>E.coli</I> DH5α cells were <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> isolated</A> and sequenced. One of the new construct originated from PCR with the S3 sample has the desired DNA sequence.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><B>Sequencing and Characterization</B></DIV><P>The rest of the tat_ProteinG was sequenced (see figure below)</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 15:</B>Alignment of the full tat_ProteinG construct. </P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>The Protein G gene from the <I>Streptococcus dysgalactiae ssp equisimilis</I> sample we collected from the hospital is clearly missing two bigger segments compared to the <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> known gene sequence</A>. We conclude that this is Protein G as the rest of the sequence aligns well.

Vesicles were isolated from <I>E.coli</I> ER2566 cells by the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Small-Scale_Vesicle_Preparation"> vesicle isolation protocol</A>. 

SDS-PAGE was done on both samples with one diluted sample (1:2) and one undiluted sample (1:1) for both.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B>Figure 16:</B>SDS-PAGE , with the ladder Precision Plus Protein<SUP>TM</SUP> Unstained Standards. First sample is a diluted (1:2) sample of vesicles from cell with tat_ProteinG construct, second sample is the same but undiluted (1:1). The bands are not as visible as with the naked eye.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There are no detectable additional bands in the tat_ProteinG samples compared to the wild type ER2566 (see SDS-PAGE figure above) sample, indicating that there is no Protein G in the vesicles.
</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>To check weather Protein G is even produced in the S3-2B cells we did a SDS-PAGE with these cells along with a wildtype ER2566 cell sample (see figure below):</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B> Figure 17:</B>SDS-PAGE of wildtype (WT) ER2566 cells and ER2566 S3-2B cells with tat_ProteinG construct. Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>
There is a very clear additional band on the S3-2B sample of about 60 kDa in protein mass. Highly indicating that tat_Protein G is produced in the cells. 

The <B>conclusion</B> is that Protein G is obviously produced in the <I>E.coli</I> strain ER2566 cells, but the tat signal peptide seems to fail to direct the Protein G into the periplasm and OMVs. </P><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B>Pm/XylS promoter system</B></DIV><P>The Pm/xylS promoter was  modified (removing an XbaI site) and turned into a BioBrick by adding the prefix and suffix. With 3A assembly, the promoter was attached to a GFP generator (<A href="http://parts.igem.org/Part:BBa_E0240">BBa_E0240</A>), and a backbone (<A href="http://parts.igem.org/Part:pSB3K3"> pSB3K3</A>). We tested the promoter and the reference promoter (<A href="http://parts.igem.org/Part:BBa_I20260"> BBa_I20260</A>) as done in <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=((Kelly%2C%20Jason%20R%5BAuthor%5D)%20AND%20Rubin%2C%20Adam%20J%5BAuthor%5D)%20AND%20Davis%2C%20Joseph%5BAuthor%5D"> this article</A>.
The reference promoter is BBa_J23101, which is a part of a constitutive promoter family. This promoter is one of the strongest in the family and will therefore produce a lot of GFP. The absorbance was measured at 600 nm, excitation at 485 nm and emission at 525 nm. Background fluorescence was also determined by measuring only media and this measurement was substracted from the measurements of the samples. For the reference promoter, we measured every 30 min for five hours (see figure X). The Pm/xylS is positively regulated by m-toluic acid, so the samples were added different concentrations (0nM, 100nM, 1µM, 10µM, 100µM, 1mM and 2mM) to see the effect. (figure X). The fluorescence and absorbance measurements were used to calculate the ratio of GFP synthesis.  The results show that the GFP are expressed in the samples with the Pm/xylS promoter, indicating that the promoter works. The figures below show the result from the testing. As expected, the reference promoter has a high rate of GFP synthesis. Even though the rate of GFP production in the Pm/xyls promoter is not consistent with the increasing inducer concentration, it still is producing GFP at a high rate. </P><DIV class="col6" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B> Figure 18:</B> The ratio of GFP synthesis as a function of inducer concentration</P></DIV></DIV><DIV class="col6" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "><B> Figure 19:</B> The ratio of GFP synthesis as a function of time</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results">http://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU-Trondheim/Experiments_and_Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU-Trondheim/Experiments_and_Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Experiments_and_Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Experiments_and_Results&amp;oldid=306396" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>