<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU-Trondheim_Notebook_September"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU-Trondheim/Notebook/September">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU-Trondheim/Notebook/September&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/September&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/September&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU-Trondheim/Notebook/September" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU-Trondheim/Notebook/September</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Trondheim iGEM 2013 </TITLE></P><DIV class="layout-1000"><DIV class="row"><DIV id="cssmenu"><UL><LI class="active"><A href="https://2013.igem.org/Team:NTNU-Trondheim"><SPAN>Home</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Project</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Project"><SPAN>Project description</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/novelapproach"><SPAN>A novel approach</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Model"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results"><SPAN>Experiments and Results</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Parts"><SPAN>BioBrick Parts</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Acknowledgements"><SPAN>Acknowledgements</SPAN></A></LI></UL><LI class="has-sub"><A href="#"><SPAN>Technical Stuff</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Notebook"><SPAN>Notebooks</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Primers"><SPAN>Primers</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Team</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Team"><SPAN>Meet the team</SPAN></A></LI><LI class="last"><A href="https://igem.org/Team.cgi?id=1082"><SPAN>Official Team Profile</SPAN></A></LI><LI class="has-sub"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Outreach"><SPAN>Outreach</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Judging</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Achievements"><SPAN>Achievements</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Medalcriteria"><SPAN>Medal criteria</SPAN></A></LI><LI class="last"><A href="https://2012.igem.org/Team:NTNU_Trondheim/Matchmaker"><SPAN>Matchmaker</SPAN></A></LI></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""><B> September </B></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Tuesday 03.09.2013</DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Analysis of sequencing results</DIV><P>Today we got back the sequencing results from last week (see Thursday 29.08.2013). The figures below show the alignments of the samples (always first line) and the expected sequences (always second line). Red indicates proper alignment, blue indicates mismatches.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure:  Alignment of tat_ProteinG S2. As the last time there seems to be a deletion on a guanine residue that shifting the reading frame. In addition the last part of the sequence does not seem to match like S3 and D8 did.</P></DIV></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> C4 </P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">C5</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">DT2 </P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">DT7</P></DIV></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> DT10</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">DT5</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> DT6</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> DT9</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>Figures: Alignment of tat_ProteinG C4, tat_ProteinG C5, tat_ProteinG DT2, tat_ProteinG DT7 and tat_ProteinG  DT10. None of these sequences makes any sense. It is safe to conclude that ProteinG is not in these sequences.</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> TCY1</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">TCY2</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">TCY3</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">TCY4 </P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>Figures: Alignment of Gibson tCY 1, Gibson tCY 2, Gibson tCY 3 and SLIC tCY. tat and YFP is present, but CFP is absent in all of these sequences.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Fluorescence of ER1 and ER2</DIV><P>After the positive sequence results we had (see tuesday 27.08.2013), we decided to check weather there where any GFP in the ER1 and ER2 bacteria (see figure 8).
LB media samples with ER1, ER2 (see figure 8) and <I>E.coli</I> cells transformed with standard RFP, which served as a referance, were centrifuged. The pellets were resuspended in DPBSS buffer. These solutions were then studied in a fluorometer by doing a excitation and emission scan (see figures below):</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Excitation</P></DIV><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Emission.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>Figures: Excitation and emission scan of ER1, ER2 and the referance sample with RFP.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There are no significant differance between the referance sample and the ER samples. It seems that GFP is not present or doesent fold properly.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Wednesday 04.09.13</DIV><P>Transformed the parts needed for testing the promoter: GFP generator (<A href="http://parts.igem.org/BBa_E0240">BBa_E0240</A>), backbone (<A href="http://parts.igem.org/pSB3K3"> pSB3K3</A> and the reference promoter (<A href="http://parts.igem.org/BBa_I20260"> BBa_I20260</A>). 
</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Thursday 05.09.13</DIV><P>We wanted to try to fix the addition and deletion in tat-ProteinG DT and S3 (respectively) by using the forward primer for tat (F_pl.b_tat) that has the right sequence. PCR-reactions for amplifying the backbones (BB) and the tat_ProteinG of both DT8 and S3 were run. These 4 samples were then run on a gel to confirm the results (figure below).</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: PCR products of tat_ProteinG and BB from DT8 (well 1 and 2) and S3 (well 3 and 4). Ladder applied is <A href="http://66.155.211.155/nebecomm/products_intl/productN3232.asp"> 1 kb DNA ladder</A>. The fragments nicely matches the expected sizes of 2800 bp for backbone and 1300-1400 for tat_proteinG.
</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>The PCR products were also incubated at 37 °C with Dpn1 to remove all old plasmids. The PCR products where then used in direct transformation of ''E.coli'' DH5α. The fragments that originated from the same plasmid where used together (BB from DT8 with tat_PrG from DT8 and so on). There was also performed a direct transformation tat_GFPmut3 with BB from S3. There were made two parallels of all combinations yielding 6 direct transformation. Controls with all the 5 fragments used was also prepared.
The transformed cells from yesterday was transfered to liquid media and incubated at 37°C overnight.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Friday 06.09.13</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> PCR</DIV><P>Turned the Pm XylS promoter system into a biobrick using PCR. The following recipe was used:
* 31.5 μl dH<SUB>2</SUB>O
* 10 μl 10x phusion buffer
* 1 μl dNTPs
* 2.5 μl fwd primer diluted 1:10 compared to stock solution (primer sequence: GAATTCGCGGCCGCTTCTAGTCAAGCCACTTCCTTTTTGC)
* 2.5 μl rev primer diluted 1:10 compared to stock solution (primer sequence: TACTAGTAGCGGCCGCTGCAGTGCATAAAGCCTAAGGGGTAGG)
* 1 μl template DNA
* 1 μl DMSO
* 0.5 μl Phusion DNA polymerase
Two different templates were used, and two samples were made for each template, giving in total four PCR batches. The first template was the Pm XylS plasmid, while the second was a PCR product of the Pm XylS promoter, modified in order to eliminate an XbaI restriction site.
The following PCR program was used:<TABLE style="border:1px solid black;"><TBODY><TR><TH>Program step</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD>Heated lid</TD><TD>103°C</TD><TD> - </TD></TR><TR><TD>Initial denaturation</TD><TD>98°C</TD><TD>30 s</TD></TR><TR><TD>Cycle 30x</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>15 s</TD></TR><TR><TD>Annealing</TD><TD>64°C</TD><TD>15 s</TD></TR><TR><TD>Elongation</TD><TD>72°C</TD><TD>53 s</TD></TR><TR><TD>End of cycle</TD></TR><TR><TD>Final elongation</TD><TD>72°C</TD><TD>10 min</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD></TR></TBODY></TABLE>

Gel picture of the PCR products:</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>The PCR products were rinsed using the QIAgen PCR Purification kit. The two parallel samples using the same template were eluted in the same eppendorf tube. The following concentrations were measured:<TABLE style="border:1px solid black;"><TBODY><TR><TH>BioBrick</TH><TH>Concentration</TH></TR><T><TR><TD>Pm XylS from BB template</TD><TD>98.3 ng/μl</TD></TR><TR><TD>Pm XylS with XbaI site</TD><TD>171.9 ng/μl</TD></TR></T></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Miniprep</DIV><P><A href="http://parts.igem.org/BBa_E0240">BBa_E0240</A> and <A href="http://parts.igem.org/pSB3K3">pSB3K3</A> were isolated using the Wizard <I>Plus</I> SV Minipreps DNA Purification System kit. The following concentrations were measured after the isolation:

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Part</TH><TH>Concentration</TH></TR><TR><TD><A href="http://parts.igem.org/BBa_E0240">BBa_E0240</A></TD><TD>79.8 ng/μl</TD></TR><TR><TD><A href="http://parts.igem.org/pSB3K3">pSB3K3</A></TD><TD>42.8 ng/μl</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Checking plates from Thursday 05.09.2013</DIV><P>All of the plates that should have colonies had colonies. The S3-1 plate had to be disregarded because the agar had detatched from the plastic plate. 4 of the 5 controls also had colonies, indicating that there where still colonies in the sample.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Sunday 08.09.2013</DIV><P>Two colonies from each of the usable plates (D8-1, D8-2, S3-1, tatGFPmut3-1 and tatGFPmut3-2) that where checked on friday 06.09 where transfered to LB media with ampenicillin.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Monday 09.09.2013</DIV><P>The bacterial culture from the day before where minipreped and the reminding plasmid samples where prepared for DNA-sequencing with the <A href="http://parts.igem.org/Primers/Catalog"> standard iGEM forward sequencing primer VF2</A>. The samples where labled tatPrG D8-1A, tatPrG D8-1B, tatPrG D8-2A, tatPrG D8-2B, tatPrG S3-2A, tatPrG S3-2B, tatGFPmut3-1A, tatGFPmut3-1B, tatGFPmut3-2A and tatGFPmut3-2B. The samples where sent for DNA-sequencing the next day.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">3A Assembly</DIV><P>3A Assembly was done on the Pm/Xyls promoter to attach it to the GFP and backbone. This was done according to the official iGEM <A href="http://parts.igem.org/Help:Assembly/3A_Assembly"> protocol</A>.
</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Tuesday 10.09.13</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Restriction digest</DIV><P>Linearized pSB1C3 and the two versions of Pm XylS (modified and unmodified, one of them containing an XbaI restriction site) were cut with EcoRI and PstI in order to put the promoter system in the shipping plasmid. The following volumes of DNA and water were used in the digestions:<TABLE style="border:1px solid black;"><TBODY><TR><TH>DNA part</TH><TH>Concentration of DNA</TH><TH>Calculated volume of DNA used in digestion</TH><TH>Volume of dH<SUB>2</SUB>O used in digestion</TH></TR><TR><TD>Linearized pSB1C3</TD><TD>25 ng/μl</TD><TD>10 μl</TD><TD>6 μl</TD></TR><TR><TD>Pm XylS BB</TD><TD>98.3 ng/μl</TD><TD>2.5 μl</TD><TD>13.5 μl</TD></TR><TR><TD>Pm XylS XbaI</TD><TD>171.9 ng/μl</TD><TD>1.5 μl</TD><TD>14.5 μl</TD></TR></TBODY></TABLE>

Buffer 3 were used in all restriction digests. In the table above, Pm XylS BB refers to the biobrick that has been modified compared to the original Pm XylS promoter, which contains an illegal XbaI restriction site. Pm XylS XbaI is the original promoter, that contains the XbaI site.
After the restriction digest, the products were purified using the QIAquick PCR Purification Kit, which has a filter size that allows the short ends of the prefix and suffix of the linearized plasmids and biobricks that should be present after the restriction digest, to pass through the filter. The following concentrations were measured after the purification:<TABLE style="border:1px solid black;"><TBODY><TR><TH>DNA part</TH><TH>Concentration</TH></TR><TR><TD>pSB1C3</TD><TD>4.2 ng/μl</TD></TR><TR><TD>Pm XylS BB</TD><TD>6.8 ng/μl</TD></TR><TR><TD>Pm XylS XbaI</TD><TD>7.6 ng/μl</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Gel electrophoresis</DIV><P>Gel electrophoresis were performed to ensure that the right DNA fragments are present after the restriction digest.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify">
In this picture, the first band is pSB1C3, which linearized should be 2070 bp long. The Pm XylS promoter system has a length of 1759 bp, and the modified and unmodified version of the promoter system corresponds to band 2 and 3.</DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Ligation</DIV><P>pSB1C3 was ligated to Pm XylS BB and Pm XylS XbaI, respectively. The following ligation mix was set up. The same mix were used for both promoter systems, since the concentrations were almost the same. The volume of backbone used corresponds to 25 ng of DNA, which is the recommended amount of backbone suggested on the partsregistry website.

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Component</TH><TH>Volume</TH></TR><TR><TD>pSB1C3</TD><TD>6 μl</TD></TR><TR><TD>Pm XylS BB / Pm XylS XbaI</TD><TD>7 μl</TD></TR><TR><TD>T4 DNA ligase buffer</TD><TD>1.5 μl</TD></TR><TR><TD>T4 DNA ligase</TD><TD>0.5 μl</TD></TR></TBODY></TABLE>

The ligation mix was put in the fridge for overnight ligation.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Wednesday 11.09.13</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Transformation</DIV><P>The Pm XylS BB in pSB1C3 and PmXylS XbaI in pSB1C3 systems that were ligated overnight were transformed to competent ''E. coli'' DH5α cells and plated out on agar plates having chloramphenicol resistance.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Thursday 12.09.13</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Transfer of cells from agar plates to liquid medium</DIV><P>The cells transformed yesterday were transferred to liquid medium containing 1 μg/ml chloramphenicol.</P></DIV><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Another round of small-scale vesicle preparation</DIV><P>Step 1 and 2 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with ER2566 transformed with the ER1 construct from the second Gibson assembly attempt that had a postive sequencing results. One colony were selected from the recently transformed bacteria plate and incubated for 16 hours in step 2 in a total of 1L LB. The cultures were incubated aerobically in LB media with ampenicillin (100 µL/mL) in step 1 and areobically in plain LB in step 2.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Testing of BioBrick promoter</DIV><P>We tested the promoter as done in <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=((Kelly%2C%20Jason%20R%5BAuthor%5D)%20AND%20Rubin%2C%20Adam%20J%5BAuthor%5D)%20AND%20Davis%2C%20Joseph%5BAuthor%5D"> this article</A>. Measured the absorbance at 600 nm, excitation at 485 nm and emission at 525 nm. Backgroud fluorescence was determined by measuring only media. For the reference promoter, we measured every 30 min for five hours. The Pm/XylS was added inducer in different concentrations (0nM, 10nM, 100nM, 1µM, 10µM, 100µM, 1mM and 2mM) to see the effect. </P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Friday 13.09.13 </DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Miniprep</DIV><P>Pm XylS BB in pSB1C3 and Pm XylS XbaI in pSB1C3 was miniprepped using the Promega Wizard Plus SV Minipreps DNA Purification kit. The following concentrations were measured:

<TABLE style="border:1px solid black;"><TBODY><TR><TH>BioBrick</TH><TH>Concentration</TH></TR><TR><TD>Pm XylS BB + pSB1C3</TD><TD>44.3 ng/μl</TD></TR><TR><TD>Pm XylS XbaI + pSB1C3</TD><TD>67.0 ng/μl</TD></TR></TBODY></TABLE></P></DIV><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Small-scale vesicle preparation</DIV><P>Step 4-12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with ER1. In addition a 10<SUP>-6</SUP> dilution of the sample were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plate had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4. The samples were diluted 1:10 with LB media and had a value of 0.436.
The cell cultures and cell pallets were not red or pink when they were taken out of the incubator and centrifuge. The pallet did, however, start to get a red color after a while.
Samples for SDS-PAGE (one undiluted and one diluted 1:2) was prepared in step 12 and stored in the fridge.
RFU was measured as described in step 12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> protcol. The blank sample had a value of 153 while the vesicle containing sample had the value of 2128. 
There was also conducted a fluorecense scan of a undiluted sample without FM4-64. This will be desribed on &quot;Sunday 15.09.2013&quot;.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Saturday 14.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Another round of small-scale vesicle preparation</DIV><P>A new round of vesicle isolation was started to create a referance to the ER1 from friday. Step 1 and 2 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with unstransformed ER2566. In step 2 the bacteria was incubated for 16 hours in a total of one liter LB.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Transformation</DIV><P>All of the tat_ProteinG plasmids that where sent for sequencing (see monday 09.09.2013) where applied in a transformation of ''E.coli'' strain ER2566 by the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Transformation"> transformation protocol</A>.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Sunday 15.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Small-scale vesicle preparation</DIV><P>Step 4-12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with unstransformed ER2566.
Optical denisty (OD) at 600 nm was mesured in step 4. The samples were diluted 1:10 with LB media and had a value of 0.351.
Samples for SDS-PAGE (one undiluted and one diluted 1:2) was prepared in step 12 and was run with the SDS-PAGE from friday 13.09.2013. See figure below.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards. WT stands for wildtype and is the unstransformed ER2566 samples.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>There are noe additional bands in the ER1 samples compared to the unstransformed ER2566 sample, indicating that there are no GFP-RFP in the vesicles.
RFU was measured as decribed in step 12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> protocol. The blank sample had a value of 150 while the vesicle containing sample had the value of 8708.
There was run a excitation scan of the undiluted unstransformed ER2566 sample that was compared to the similar scan done on friday 13.09.2013. The results were as shown in the two figures below.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>Figures: Fluorescence excitation scan of vesicles from the ER1 cells (left) and wildtype ER2566 (right).
There are no real differance between the samples other then strength of the signals. Both samples have peaks at 504, 540 and 582 nm. It is likely that there are no tGR construct in the vesicles.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PCR and digestion for putting on a prefix on the tat-GFP-RFP construct</DIV><P>A PCR were run on the plasmid containing tat_GFP_RFP construct with the F_prefix_tat and R_pl.b primers. This would in theory give us a PCR product that is the same size as the original plasmid. 15 µL of the PCR-product and 4 µL of the linerized pSB1C3 backbone where digested with the <A href="http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones"> Enzyme Master Mix for Plasmid Backbone</A> according to the <A href="http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones"> recomended protocole</A>.
The PCR product and the digested PCR product were run on a agarose gel, giving the result as indicated in the figure below.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Ladder applied is GeneRuler<SUP>TM</SUP> 100 bp plus DNA ladder. The ladder is for some reason not completly functional.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify">
As there were not just one band and the ladder was unreliable, no fragment was cut out of the gel.</DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Sunday 22.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Analysing DNA sequencing results</DIV><P>The results from the last round of sequencing (see monday 09.09) was analysed with the prefered sequence. The alignments were as follows:</P><DIV class="col2-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> .</P></DIV><DIV class="col2-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> .</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>Figures: Alignment of tat_ProteinG D8-1A, tat_ProteinG D8-1B, tat_ProteinG D8-2A, tat_ProteinG D-2B and tat_ProteinG  S3-2A. These sequences either makes no sense or does not have the exact sequence we are looking for. These constructs will not be applied for later.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of tat_ProteinG S3-2B. This sequence is just how its supposed to be and will be used later in a vesicle isolation.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>The alignments of the tat_GFPmut3 samples did not give any sense at all in will therefore not be included here as figures.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Monday 23.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Sending Biobricks</DIV><P>The biobricks tat_GFP_RFP (BBa_K1082001) and the Pm/XylS promoter system (BBa_K1082002) was sent in to the registery.</P><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">DNA samples sendt in for sequencing</DIV><P>The rest of the biobrick tat_GFP_RFP (BBa_K1082001) was sent in for sequencing with the F_seq and R_seq primers. The tat_ProteinG S3-2B (see sunday 22.09.2013) was also sent in for sequencing with the R_seq primer</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Wednesday 25.09.2013</DIV><P>Step 1 and 2 of the [https://2013.igem.org/Team:NTNU-Trondheim/Protocols small-scale vesicle preparation] were performed with ER2566 transformed with the tat_ProteinG S3-2B (see sequencing results for sunday 22.09.2013) that had a postive sequencing result. One colony were selected from the recently transformed bacteria plate and incubated for 16 hours in step 2 in a total of 1L LB. The cultures were incubated aerobically in LB media with ampenicillin (100 µL/mL) in step 1 and aerobically in plain LB in step 2.</P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Testing of BioBrick promoter</DIV><P>Because non-satisfying result from the last test, we did another test. This was similar to the one done on thursday 12.09.13 with the same time-series and inducer concentrations. </P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Thursday 26.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> PCR and digestion for creating a briobrick of tat_GFP_RF</DIV><P>PCR was run on the tat_GFP_RFP PCR-product from sunday 16.09. This time the F_prefix_tat primer and a reverse primer that anneals to the suffix in plasmid was applied. The PCR-prduct was digested with EcoRI and PstI. The digested PCR-product was run on a gel and one gelband was cut out. The fragment was purified with the <A href="http://www.qiagen.com/Products/Catalog/Sample-Technologies/DNA-Sample-Technologies/DNA-Cleanup/QIAquick-PCR-Purification-Kit#productdetails"> QIAquick PCR Purification kit</A>.</P></DIV><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Ligation</DIV><P>The purified and digested (see above) tat_GFP_RFP with prefix was added into a ligation mix with the digested pSB1C3 backbone according to the <A href="http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones"> iGEM protocole</A>. The ligation product was transformed into DH5α cells.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Thursday 26.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Small-scale vesicle preparation</DIV><P>Step 4-12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with ER1. In addition a 10<SUP>-6</SUP> dilution of the sample were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plate had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4. The samples were diluted 1:10 with LB media and had a value of 0.165.
Samples for SDS-PAGE (one undiluted and one diluted 1:2) was prepared in step 12 and stored in the fridge.
RFU was measured as decribed in step 12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> protcol. The blank sample had a value of 5531 while the vesicle containing sample had the value of 44 255.</P></DIV><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Friday 27.09.2013</DIV><P>SDS-PAGE were performed with the samples from the day before (see figure below):

</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards. The bands are not as visible as with the naked eye.</P></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><P>There are no additional bands compared with the wildtype ER2566 vesicles (see sunday 15.09) and therefore no indication that Protein G are in the vesicles.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Sunday 29.09.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Analysis of sequencing results</DIV><P>Results of DNA samples that were sent in for sequencing on monday 23.09 ware analysed.
</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 1.</P></DIV><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 1.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>Figures: Sequencing results of the tat_GFP_RFP construct with the F_seq (left) and R_seq (right) primer. </P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>The new sequencing results from the ER1 sample that is supposed to have the tat_GFP_RFP construct does not make sense. It is indeed very unclear why our construct suddenly is lost.

</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Sequencing results of the tat_ProteinG S3_2B construct with the R_seq primer..</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>Putting this tat_ProteinG results together with the results from sunday 22.09 with the forward (F_seq) primer gives the alignment as follows:</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of the full tat_ProteinG construct.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>The Protein G gene from the ''Streptococcus dysgalactiae ssp equisimilis'' sample we collected from the hospital are clearly missing two bigger segments compared to the <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> known gene sequence</A>. We conclude that this is Protein G as the rest of the sequence aligns well.</P><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> PCR on fluorescent proteins</DIV><P>PCR was run on YFP (<A href="http://parts.igem.org/BBa_E0030">BBa_E0030</A>), CFP (<A href="http://parts.igem.org/BBa_E0020">BBa_E0020</A>), GFP (<A href="http://parts.igem.org/BBa_E0040">BBa_E0040</A>), BFP (<A href="http://parts.igem.org/BBa_K592100">BBa_K5921000</A>) and SYFP (<A href="http://parts.igem.org/BBa_K864100">BBa_K864100</A>) by using F_seq and R_seq as primers. These primers anneal to VF2 (<A href="http://parts.igem.org/BBa_G00100">BBa_G00100</A>) and VR (<A href="http://parts.igem.org/BBa_G00101">BBa_G00101</A>). The products were run on a gel and all samples had the expected size of about 1000 bp (see figure below).</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Ladder applied is GeneRuler<SUP>TM</SUP> 1 kb DNA ladder..</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Analysis of sequencing results</DIV><P>Another 3A assembly was done with the PmXyls promoter, GFP and backbone. The result from the test showed no fluorescence, so the GFP was not expressed as it should. </P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NTNU-Trondheim/Notebook/September">http://2013.igem.org/Team:NTNU-Trondheim/Notebook/September</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU-Trondheim/Notebook/September" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU-Trondheim/Notebook/September" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/September&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/September&amp;oldid=303486" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>