"
Page
Discussion
View source
History
teams
Log in
 
Team:NTNU-Trondheim/Experiments and Results
From 2013.igem.org
Trondheim iGEM 2013
Home
Project
Project description
A novel approach
Modelling
Experiments and Results
BioBrick Parts
Acknowledgements
Technical Stuff
Notebooks
Protocols
Primers
Safety
Team
Meet the team
Official Team Profile
Outreach
Judging
Achievements
Medal criteria
Matchmaker
Creating the tat_GFP_l_RFP construct
PCR for amplification of tat, plasmid backbone, GFP and RFP
There were essentially four DNA pieces that we wanted to combine together to make a construct: The tat signal sequence followed by GFP, a small linker region and RFP put into a plasmid backbone. As our cloning techniques rely on overlapping DNA fragments we used mostly primers with overhengs in the PCR reactions. As templates we used the biobricks BBa_E1010 for RFP, BBa_R0040 for GFP, BBa_J01101 for the plasmid backbone and genomic DNA from Escherichia coli strain ER2566 for the tat signal sequence.
Table 1: Primers applied in creating the tat_GFP_l_RFP construct. Lowercase letters indicate DNA that anneal to the template whereas the uppercase letters indicate DNA which serves as an overhang.
Amplifing
Primer
Sequence
tat
F_pl.b_tat
AGAGAAAGAGGAGAAATACTAGatggccaataacgatctctttcaggcatcacg
tat
R_tat
cgcttgcgccgcagtcgcacgtcg
GFP
F_tat_GFP
CGACGTGCGACTGCGGCGCAAGCGatgcgtaaaggagaagaac
GFP
R_l_GFP
ACTACCACCGGATCCACCTGATCCACCGGATCCACCtttgtatagttcatccatgcc
RFP
F_l_RFP
GGTGGATCCGGTGGATCAGGTGGATCCGGTGGTAGTatggcttcctccgaagacg
RFP
R_pl.b_RFP
GCCTTTCGTTTTATTTGATGCCTGGgcgatctacactagcactatcagcg
Plasmid backbone
F_pl.b
ccaggcatcaaataaaacgaaagg
Plasmid backbone
R_pl.b
ctagtatttctcctctttctctagtagtgc
The linker region is going to be 36 bp long and will be created by overlapping overhangs on the reverse primer of GFP and forward primer of RFP. The primers for the plasmid backbone is designed to include the TetR repressible promoter ( BBa_R0040 ), RBS ( BBa_R0034 ) and two terminators ( BBa_R0010 ) and ( BBa_R0012 ) in the PCR product. All of the PCR products were treated with the enzyme DpnI that digests methylated DNA and purified by the QIAquick PCR Purification kit .
Gibson Assembly and transformation
Our overlapping DNA fragments; tat, GFP, RFP and plasmid backbone was cloned together by Gibson Assembly and transformed into E.col strain ER2566 cells. The photograph below shows two of the resulting colonies (named ER1 and ER2) from this transformation.
Figure 1: Two of the transformed colonies with the tat_GFP_l_RFP construct. Hereby named ER1 and ER2.
Sequencing and characterization
The plasmid from the ER1 samples was isolated by the Promega Wizard Plus SV Minipreps DNA Purification System A1460 and sequenced. Figure below shows the alignment of the sequencing results with the reference DNA sequence.
Figure 2: Aligment of tat_GFP_l_RFP (ER1) with reference DNA.
The sequence align almost perfectly. There seems to be some sort of extra insert at the linker region, but this insert is dividable by 3, so the reading frame is maintained. This is supported by the fact that the colonies with this construct is red (see figure above).
Red ER1-cells in liquid media was immobilized in agar and then viewed in a confocal microscope for seeing if RFP was localized in the periplasm. The results can be seen in the two figures below:
Figure 3:Red ER1-cells viewed in confocal microscope in 2D
Figure 4:Red ER1-cells viewed in confocal microscope in 3D
There is no indication that the red fluorescence is more concentrated in the periplasm, as we should expect due to the transport through the tat transport pathway.  We performed a excitation and emission scan of the ER1 bacterias together with E-coli strain ER2566 cells that were transformed to produce single RFP. The samples were centrifuged and the pallets were resuspended in DPBSS buffer. These solutions were den studied in a fluorometer(see figures below):
Figure 5:Excitation scan of ER1, ER2 and the referance sample with RFP
Figure 6:Emission scan of ER1, ER2 and the referance sample with RFP
There are no significant difference between the referance sample and the ER samples. It seems that GFP is not present or doesn't fold properly. All of the samples had an excitation/emission at 584/607 which is the same as for singel RFP ( BBa_1010 ). The test weather a GFP-RFP dimer is produced or only RFP is produced we ran a SDS-PAGE with the ER1 E.coli strain ER2566 cells along with wild type ER2566 cells (see figure below).
Figure 7: SDS-PAGE with wildtype (WT) ER2566 cells and ER1 ER2566 cells that has the tat_GFP_l_RFP construct
The mass size of the additional band on the ER1 sample is about 28-30 kDa, close to the size of a single fluorescent protein of 26-30 kDa. We therefor conclude that only single RFP is produced in the ER1 cells as there are no green fluorescence.
Vesicles were isolated from ER1 cells and wildtype E.coli strain ER2566 cells by the
vesicle isolation protocol.
SDS-PAGE was done on both samples with one diluted sample (1:2) and one undiluted sample (1:1) for both.
Figure 8:Ladder applied is Precision Plus ProteinTM Unstained Standards. WT stands for wildtype and is the unstransformed ER2566 samples.
There are no additional bands in the ER1 samples compared to the unstransformed ER2566 sample, indicating that there is no GFP-RFP in the vesicles.
There was run an excitation scan of the undiluted vesicle ER1 sample that was compared to a same scan on vesicles from wildtype bacteria. The results were as shown in the two figures below.
Figure 9:Fluorescence excitation scan of vesicles from the ER1 cells and wildtype ER2566 (right).
Figure 10:Fluorescence excitation scan of vesicles from the wildtype ER2566 cells.
There is no real difference between the samples other than strength of the signals. Both samples have peaks at 504, 540 and 582 nm. It is likely that there is no detectable GFP-RFP dimer in the vesicles.
Protein G
PCR for amplification of tat Protein G
Our template for this PCR reaction was genomic DNA from Streptococcus dysgalactiae ssp equisimilis collected from St. Olavs Hospital. We had 5 different samples of Streptococcus dysgalactiae ssp equisimilis and 6 different combinations of primers giving 30 different PCR reactions. Standard conditions for Phusion PCR was used. As indicated in the figure below, only one of these reaction gave a PCR product.
Figure 11:PCR results of Protein G from 30 different PCR reactions.
A very clear band with the size of around 1200-1300 bp was found in one of the samples. This is a smaller DNA fragment compared to the known gene sequence of about 1700 bp. Since the size of Protein G is known to vary among the different strains, we can still conclude that our PCR-product is Protein G. The PCR product with the genomic DNA sample 1 as template and F_tat_PrG and R_pl.b_PrGstop (see table below) as primers is most likely Protein G.
Table 2: Primers applied in creating the tat_ProteinG construct. Lowercase letters indicate DNA that anneal to the template whereas the uppercase letters indicate DNA that serves as an overhang.
Amplifing
Primer
Sequence
Protein G
F_tat_PrG
CGACGTGCGACTGCGGCGCAAGCGgttgactcaccaatcgaagatacccc
Protein G
R_pl.b_PrGstop
GCCTTTCGTTTTATTTGATGCCTGG ttagtcttctttacgttttgaagcgac
Attaching tat signal sequence by PEC
For attachment of the tat DNA fragment to the Protein G PCR fragment Polymerase Extension Cloning ( PEC ) was applied. After the PEC PCR reaction the product was run on a agarose gel together with the PCR product of Protein G without the tat attached.
Figure 12:From the left; Protein G with tat attached and Protein G without tat.
The PEC product is clearly a little bit longer then the Protein G PCR product without tat attched. This indicates that PEC was successful.
Cloning by Direct Transformation and SLIC
The tat_ProteinG fragment was mixed with the same plasmid backbone PCR product that was produced for the tat_GFP_l_RFP construct in varies different cloning reactions: CPEC , SLIC and direct transformation . 18 different reactions was run. The cloning products was transformed into E.coli DH5α cells. The plasmids were then isolated and a conformation PCR was executed with F_pl.b_tat and R_pl.b_PrGstop as primers. The conformation results can be viewed in the figure below:
Figure 13:First row contains tat_ProteinG samples 1-9, second row contains tat_ProteinG samples 10-18.
All of the samples that had a PCR product of the expected size of 1200-1300 bp (10 samples) was sent for sequencing. The sequencing results showed that two of the constructs had approximately the DNA sequence they should have. However, these constructs, named Dt8 (created by direct transformation) and S3 (created by SLIC) had an addition and a deletion of a guanine residue in the beginning of the tat sequence, respectively (see figures below). These errors would create a frame shift that would produce a dysfunctional Protein G.
Figure 14:Aligment of tat_ProteinG DT8 (left) and S3 (right). There is one segment missing in our ProteinG, as is expected as the ProteinG variant that we are dealing with is a shorter than the registered gene sequence . There is however, an addition of a guanine residue in the beginning of the tat-sequence in the DT8 sample and a deletion at the same site in the S3 sample, which will cause a frame shift resulting in a non-functional protein product.
Because it is only possible to sequence approximately 1000 bp, we don't have all of the sequence of our tat_ProteinG construct.
Fixing deletion by PCR
In order to correct the Guanin deletion/addition a PCR was run on the DT8 and S3 sample with the primers F_pl.B_tat and R_pl.b_PrGstop. The PCR products was cloned together with the same plasmid backbone in a direct transformation. Plasmids from the transformed E.coli DH5α cells were isolated and sequenced. One of the new construct originated from PCR with the S3 sample has the desired DNA sequence.
Sequencing and Characterization
The rest of the tat_ProteinG was sequenced (see figure below)
Figure 15:Alignment of the full tat_ProteinG construct.
The Protein G gene from the Streptococcus dysgalactiae ssp equisimilis sample we collected from the hospital is clearly missing two bigger segments compared to the known gene sequence . We conclude that this is Protein G as the rest of the sequence aligns well.
Vesicles were isolated from E.coli ER2566 cells by the vesicle isolation protocol .
SDS-PAGE was done on both samples with one diluted sample (1:2) and one undiluted sample (1:1) for both.
Figure 16:SDS-PAGE , with the ladder Precision Plus ProteinTM Unstained Standards. First sample is a diluted (1:2) sample of vesicles from cell with tat_ProteinG construct, second sample is the same but undiluted (1:1). The bands are not as visible as with the naked eye.
There are no detectable additional bands in the tat_ProteinG samples compared to the wild type ER2566 (see SDS-PAGE figure above) sample, indicating that there is no Protein G in the vesicles.
To check weather Protein G is even produced in the S3-2B cells we did a SDS-PAGE with these cells along with a wildtype ER2566 cell sample (see figure below):
Figure 17:SDS-PAGE of wildtype (WT) ER2566 cells and ER2566 S3-2B cells with tat_ProteinG construct. Ladder applied is Precision Plus ProteinTM Unstained Standards.
There is a very clear additional band on the S3-2B sample of about 60 kDa in protein mass. Highly indicating that tat_Protein G is produced in the cells.
The conclusion is that Protein G is obviously produced in the E.coli strain ER2566 cells, but the tat signal peptide seems to fail to direct the Protein G into the periplasm and OMVs.
Pm/XylS promoter system
The Pm/xylS promoter was  modified (removing an XbaI site) and turned into a BioBrick by adding the prefix and suffix. With 3A assembly, the promoter was attached to a GFP generator ( BBa_E0240 ), and a backbone ( pSB3K3 ). We tested the promoter and the reference promoter ( BBa_I20260 ) as done in this article .
The reference promoter is BBa_J23101, which is a part of a constitutive promoter family. This promoter is one of the strongest in the family and will therefore produce a lot of GFP. The absorbance was measured at 600 nm, excitation at 485 nm and emission at 525 nm. Background fluorescence was also determined by measuring only media and this measurement was substracted from the measurements of the samples. For the reference promoter, we measured every 30 min for five hours (see figure X). The Pm/xylS is positively regulated by m-toluic acid, so the samples were added different concentrations (0nM, 100nM, 1µM, 10µM, 100µM, 1mM and 2mM) to see the effect. (figure X). The fluorescence and absorbance measurements were used to calculate the ratio of GFP synthesis.  The results show that the GFP are expressed in the samples with the Pm/xylS promoter, indicating that the promoter works. The figures below show the result from the testing. As expected, the reference promoter has a high rate of GFP synthesis. Even though the rate of GFP production in the Pm/xyls promoter is not consistent with the increasing inducer concentration, it still is producing GFP at a high rate.
Figure 18: The ratio of GFP synthesis as a function of inducer concentration
Figure 19: The ratio of GFP synthesis as a function of time
Retrieved from " http://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
