<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU-Trondheim_test"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU-Trondheim/test">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU-Trondheim/test&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/test&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/test&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU-Trondheim/test" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU-Trondheim/test</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Trondheim iGEM 2013 </TITLE></P><DIV class="layout-1000"><DIV class="row"><DIV class="col12-2"><UL class="topnav"><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim">Home</A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Berkeley/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Berkeley/Judging">Judging</A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Safety">Safety</A></LI></UL></DIV><DIV class="row"><DIV class="col12-2"><DIV align="justify">

Gram negative (and some gram positive) bacteria produce outer membrane vesicles (OMV) by outward bulging of the outer plasma membrane. These OMV has varies function such as quorum sensing, biofilm development, involvement in pathogensis (some for pathogenic bacteria) and transport of enzymes to distal locations [http://www.ncbi.nlm.nih.gov/pubmed/20825345 (1)].<DIV class="row"><DIV class="col12-2"><DIV align="justify">
====Potential as a drug delivery vehicle====
Whereas OMVs function and contents has been studied for decades, their potential as a drug carrier has not been investigated before. As it is still many obstacles for an effective production method of different drug carriers within the field of nano-medicine, exploiting gram negative bacteria OMV production could be a possibility for drug delivery in the future. Especially vesicles from pathogenic bacteria could be further manipulated as these vesicles naturally can attack human eucaryotic cells (see figure below)

===Fluorescent proteins===
The first fluorescent protein to be discovered was the green fluorescent protein (GFP) isolated from ''Aequorea victoria''. Many other different fluorescent protein (FP) with different colours (red, cyan, yellow and other version of these) has since been created by mutating GFP. The FPs has been crucial in terms of visualising cells and determining intracellular proteins by fusing the FPs to the proteins of interest. Having a variety of FPs gives scientists the opportunity to study different proteins at the same time.

The discovery of green fluorescent protein in the early 1960s eventually published a new era in cell biology by assisting investigators to apply molecular cloning methods, fusing the fluorophore moiety to a wide variety of protein and enzyme targets, in order to monitor cellular processes in living systems using optical microscopy and related methodology. 

Fluorescent proteins are in the group of structurally homologous class of proteins that they share the unique property of being self-sufficient to form a visible wavelength chromophore from a sequence of 3 amino acids within their own polypeptide sequence. Biologists introduce a gene (or a gene chimera) encoding an engineered fluorescent protein into living cells and subsequently visualize the location and dynamics of the gene product using fluorescence microscopy.

The presence of a fluorescent component in the bioluminescent organs of Aequorea victoria jellyfish was introduced by Davenport and Nicol in 1955, but it was Osamu Shimomura who was the first to recognize that this fluorophore is actually a protein. He mentioned in one of his papers “A protein giving solutions that look slightly greenish in sunlight though only yellowish under tungsten lights, and exhibiting a very bright, greenish fluorescence in the ultraviolet of a Mineralite has also been isolated from squeezates&quot; (Shimomura et al. 1962). &quot;Squeezate&quot; describing a solution that resulted from the squeezing of the excised bioluminescent tissues of the jellyfish through a cotton bag. After that Shimomura reported the fluorescence emission spectrum of this protein and also he suggested that energy transfer from aequorin to this green fluorescent protein could explain why the in vivo luminescence of Aequorea is greenish and not blue like the luminescence of purified aequorin.

The gene for green fluorescent protein was first cloned in 1992, but the important possibility to be used as a molecular probe was not achieved until several years later when fusion products were used to track gene expression in bacteria and nematodes. Green fluorescent protein has been engineered to produce numbers of various colored mutants, fusion proteins, and biosensors which are broadly referred to as fluorescent proteins. Recently, fluorescent proteins from other species are also identified and isolated, resulting in further expansion of the color palette. With the rapid evolution of fluorescent protein technology, the utility of this genetically encoded fluorophore for a wide spectrum of applications beyond the simple tracking of tagged biomolecules in living cells is now becoming fully appreciated.

A broad range of fluorescent protein genetic variants have been developed that feature fluorescence emission spectral profiles spanning almost the entire visible light spectrum. Mutagenesis efforts in the original Aequorea victoria jellyfish green fluorescent protein have resulted in new fluorescent probes that range in color from blue to yellow, and are some of the most widely used in vivo reporter molecules in biological research. Longer wavelength fluorescent proteins, emitting in the orange and red spectral regions, have been developed from the marine anemone, Discosoma striata, and reef corals belonging to the class Anthozoa. Still other species have been mined to produce similar proteins having cyan, green, yellow, orange, and deep red fluorescence emission. Developmental research efforts are ongoing to improve the brightness and stability of fluorescent proteins, thus improving their overall usefulness.

==<CENTER>Our mission</CENTER>==
===Bacterial outer membrane vesicles===
We want, as a part of our project, to introduce protein G from ''S.dysgalactiae subsp. equisimilis'' into ''Escherichia coli'' OMV's. Protein G is known to bind to human serum albumin (has) which helps ''S.dysgalactiae subsp. equisimilis'' hiding from the immune system. Protein G therefore could be a potential important piece in a drug carrier by masking it from immunological destruction. Introducing protein G into vesicles also demonstrate that it is indeed possible to manipulate the content and therefore the properties of OMV's.

===Fluorescent proteins===
In the second part of our project, we want to create new combinations of FP's. We intend to make this happen by fusing different FPs together into a new construct, namely GFP, red fluorescent protein (RFG), cyan fluorescent protein (CFP) and yellow fluorescent protein (YFP). This will give us new combinations of FPs with new absorption spectrum.


</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NTNU-Trondheim/test">http://2013.igem.org/Team:NTNU-Trondheim/test</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU-Trondheim/test" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU-Trondheim/test" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/test&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/test&amp;oldid=265077" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>