"
Page
Discussion
View source
History
teams
Log in
 
Team:NTNU-Trondheim/Notebook/September
From 2013.igem.org
Trondheim iGEM 2013
Home
Project
Project description
A novel approach
Modelling
Experiments and Results
BioBrick Parts
Acknowledgements
Technical Stuff
Notebooks
Protocols
Primers
Safety
Team
Meet the team
Official Team Profile
Outreach
Judging
Achievements
Medal criteria
Matchmaker
September
Tuesday 03.09.2013
Analysis of sequencing results
Today we got back the sequencing results from last week (see Thursday 29.08.2013). The figures below show the alignments of the samples (always first line) and the expected sequences (always second line). Red indicates proper alignment, blue indicates mismatches.
Figure:  Alignment of tat_ProteinG S2. As the last time there seems to be a deletion on a guanine residue that shifting the reading frame. In addition the last part of the sequence does not seem to match like S3 and D8 did.
C4
C5
DT2
DT7
DT10
DT5
DT6
DT9
Figures: Alignment of tat_ProteinG C4, tat_ProteinG C5, tat_ProteinG DT2, tat_ProteinG DT7 and tat_ProteinG  DT10. None of these sequences makes any sense. It is safe to conclude that ProteinG is not in these sequences.
TCY1
TCY2
TCY3
TCY4
Figures: Alignment of Gibson tCY 1, Gibson tCY 2, Gibson tCY 3 and SLIC tCY. tat and YFP is present, but CFP is absent in all of these sequences.
Fluorescence of ER1 and ER2
After the positive sequence results we had (see tuesday 27.08.2013), we decided to check weather there where any GFP in the ER1 and ER2 bacteria (see figure 8).
LB media samples with ER1, ER2 (see figure 8) and E.coli cells transformed with standard RFP, which served as a referance, were centrifuged. The pellets were resuspended in DPBSS buffer. These solutions were then studied in a fluorometer by doing a excitation and emission scan (see figures below):
Excitation
Emission.
Figures: Excitation and emission scan of ER1, ER2 and the referance sample with RFP.
There are no significant differance between the referance sample and the ER samples. It seems that GFP is not present or doesent fold properly.
Wednesday 04.09.13
Transformed the parts needed for testing the promoter: GFP generator ( BBa_E0240 ), backbone ( pSB3K3 and the reference promoter ( BBa_I20260 ).
Thursday 05.09.13
We wanted to try to fix the addition and deletion in tat-ProteinG DT and S3 (respectively) by using the forward primer for tat (F_pl.b_tat) that has the right sequence. PCR-reactions for amplifying the backbones (BB) and the tat_ProteinG of both DT8 and S3 were run. These 4 samples were then run on a gel to confirm the results (figure below).
Figure: PCR products of tat_ProteinG and BB from DT8 (well 1 and 2) and S3 (well 3 and 4). Ladder applied is 1 kb DNA ladder . The fragments nicely matches the expected sizes of 2800 bp for backbone and 1300-1400 for tat_proteinG.
The PCR products were also incubated at 37 °C with Dpn1 to remove all old plasmids. The PCR products where then used in direct transformation of ''E.coli'' DH5α. The fragments that originated from the same plasmid where used together (BB from DT8 with tat_PrG from DT8 and so on). There was also performed a direct transformation tat_GFPmut3 with BB from S3. There were made two parallels of all combinations yielding 6 direct transformation. Controls with all the 5 fragments used was also prepared.
The transformed cells from yesterday was transfered to liquid media and incubated at 37°C overnight.
Friday 06.09.13
PCR
Turned the Pm XylS promoter system into a biobrick using PCR. The following recipe was used:
* 31.5 μl dH2O
* 10 μl 10x phusion buffer
* 1 μl dNTPs
* 2.5 μl fwd primer diluted 1:10 compared to stock solution (primer sequence: GAATTCGCGGCCGCTTCTAGTCAAGCCACTTCCTTTTTGC)
* 2.5 μl rev primer diluted 1:10 compared to stock solution (primer sequence: TACTAGTAGCGGCCGCTGCAGTGCATAAAGCCTAAGGGGTAGG)
* 1 μl template DNA
* 1 μl DMSO
* 0.5 μl Phusion DNA polymerase
Two different templates were used, and two samples were made for each template, giving in total four PCR batches. The first template was the Pm XylS plasmid, while the second was a PCR product of the Pm XylS promoter, modified in order to eliminate an XbaI restriction site.
The following PCR program was used:
Program step
Temperature
Duration
Heated lid
103°C
-
Initial denaturation
98°C
30 s
Cycle 30x
Denaturation
98°C
15 s
Annealing
64°C
15 s
Elongation
72°C
53 s
End of cycle
Final elongation
72°C
10 min
Hold
4°C
∞
Gel picture of the PCR products:
The PCR products were rinsed using the QIAgen PCR Purification kit. The two parallel samples using the same template were eluted in the same eppendorf tube. The following concentrations were measured:
BioBrick
Concentration
Pm XylS from BB template
98.3 ng/μl
Pm XylS with XbaI site
171.9 ng/μl
Miniprep
BBa_E0240 and pSB3K3 were isolated using the Wizard Plus SV Minipreps DNA Purification System kit. The following concentrations were measured after the isolation:
Part
Concentration
BBa_E0240
79.8 ng/μl
pSB3K3
42.8 ng/μl
Checking plates from Thursday 05.09.2013
All of the plates that should have colonies had colonies. The S3-1 plate had to be disregarded because the agar had detatched from the plastic plate. 4 of the 5 controls also had colonies, indicating that there where still colonies in the sample.
Sunday 08.09.2013
Two colonies from each of the usable plates (D8-1, D8-2, S3-1, tatGFPmut3-1 and tatGFPmut3-2) that where checked on friday 06.09 where transfered to LB media with ampenicillin.
Monday 09.09.2013
The bacterial culture from the day before where minipreped and the reminding plasmid samples where prepared for DNA-sequencing with the standard iGEM forward sequencing primer VF2 . The samples where labled tatPrG D8-1A, tatPrG D8-1B, tatPrG D8-2A, tatPrG D8-2B, tatPrG S3-2A, tatPrG S3-2B, tatGFPmut3-1A, tatGFPmut3-1B, tatGFPmut3-2A and tatGFPmut3-2B. The samples where sent for DNA-sequencing the next day.
3A Assembly
3A Assembly was done on the Pm/Xyls promoter to attach it to the GFP and backbone. This was done according to the official iGEM protocol .
Tuesday 10.09.13
Restriction digest
Linearized pSB1C3 and the two versions of Pm XylS (modified and unmodified, one of them containing an XbaI restriction site) were cut with EcoRI and PstI in order to put the promoter system in the shipping plasmid. The following volumes of DNA and water were used in the digestions:
DNA part
Concentration of DNA
Calculated volume of DNA used in digestion
Volume of dH2O used in digestion
Linearized pSB1C3
25 ng/μl
10 μl
6 μl
Pm XylS BB
98.3 ng/μl
2.5 μl
13.5 μl
Pm XylS XbaI
171.9 ng/μl
1.5 μl
14.5 μl
Buffer 3 were used in all restriction digests. In the table above, Pm XylS BB refers to the biobrick that has been modified compared to the original Pm XylS promoter, which contains an illegal XbaI restriction site. Pm XylS XbaI is the original promoter, that contains the XbaI site.
After the restriction digest, the products were purified using the QIAquick PCR Purification Kit, which has a filter size that allows the short ends of the prefix and suffix of the linearized plasmids and biobricks that should be present after the restriction digest, to pass through the filter. The following concentrations were measured after the purification:
DNA part
Concentration
pSB1C3
4.2 ng/μl
Pm XylS BB
6.8 ng/μl
Pm XylS XbaI
7.6 ng/μl
Gel electrophoresis
Gel electrophoresis were performed to ensure that the right DNA fragments are present after the restriction digest.
In this picture, the first band is pSB1C3, which linearized should be 2070 bp long. The Pm XylS promoter system has a length of 1759 bp, and the modified and unmodified version of the promoter system corresponds to band 2 and 3.
Ligation
pSB1C3 was ligated to Pm XylS BB and Pm XylS XbaI, respectively. The following ligation mix was set up. The same mix were used for both promoter systems, since the concentrations were almost the same. The volume of backbone used corresponds to 25 ng of DNA, which is the recommended amount of backbone suggested on the partsregistry website.
Component
Volume
pSB1C3
6 μl
Pm XylS BB / Pm XylS XbaI
7 μl
T4 DNA ligase buffer
1.5 μl
T4 DNA ligase
0.5 μl
The ligation mix was put in the fridge for overnight ligation.
Wednesday 11.09.13
Transformation
The Pm XylS BB in pSB1C3 and PmXylS XbaI in pSB1C3 systems that were ligated overnight were transformed to competent ''E. coli'' DH5α cells and plated out on agar plates having chloramphenicol resistance.
Thursday 12.09.13
Transfer of cells from agar plates to liquid medium
The cells transformed yesterday were transferred to liquid medium containing 1 μg/ml chloramphenicol.
Another round of small-scale vesicle preparation
Step 1 and 2 of the small-scale vesicle preparation were performed with ER2566 transformed with the ER1 construct from the second Gibson assembly attempt that had a postive sequencing results. One colony were selected from the recently transformed bacteria plate and incubated for 16 hours in step 2 in a total of 1L LB. The cultures were incubated aerobically in LB media with ampenicillin (100 µL/mL) in step 1 and areobically in plain LB in step 2.
Testing of BioBrick promoter
We tested the promoter as done in this article . Measured the absorbance at 600 nm, excitation at 485 nm and emission at 525 nm. Backgroud fluorescence was determined by measuring only media. For the reference promoter, we measured every 30 min for five hours. The Pm/XylS was added inducer in different concentrations (0nM, 10nM, 100nM, 1µM, 10µM, 100µM, 1mM and 2mM) to see the effect.
Friday 13.09.13
Miniprep
Pm XylS BB in pSB1C3 and Pm XylS XbaI in pSB1C3 was miniprepped using the Promega Wizard Plus SV Minipreps DNA Purification kit. The following concentrations were measured:
BioBrick
Concentration
Pm XylS BB + pSB1C3
44.3 ng/μl
Pm XylS XbaI + pSB1C3
67.0 ng/μl
Small-scale vesicle preparation
Step 4-12 of the small-scale vesicle preparation were performed with ER1. In addition a 10-6 dilution of the sample were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plate had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4. The samples were diluted 1:10 with LB media and had a value of 0.436.
The cell cultures and cell pallets were not red or pink when they were taken out of the incubator and centrifuge. The pallet did, however, start to get a red color after a while.
Samples for SDS-PAGE (one undiluted and one diluted 1:2) was prepared in step 12 and stored in the fridge.
RFU was measured as described in step 12 of the small-scale vesicle preparation protcol. The blank sample had a value of 153 while the vesicle containing sample had the value of 2128.
There was also conducted a fluorecense scan of a undiluted sample without FM4-64. This will be desribed on "Sunday 15.09.2013".
Saturday 14.09.2013
Another round of small-scale vesicle preparation
A new round of vesicle isolation was started to create a referance to the ER1 from friday. Step 1 and 2 of the small-scale vesicle preparation were performed with unstransformed ER2566. In step 2 the bacteria was incubated for 16 hours in a total of one liter LB.
Transformation
All of the tat_ProteinG plasmids that where sent for sequencing (see monday 09.09.2013) where applied in a transformation of ''E.coli'' strain ER2566 by the transformation protocol .
Sunday 15.09.2013
Small-scale vesicle preparation
Step 4-12 of the small-scale vesicle preparation were performed with unstransformed ER2566.
Optical denisty (OD) at 600 nm was mesured in step 4. The samples were diluted 1:10 with LB media and had a value of 0.351.
Samples for SDS-PAGE (one undiluted and one diluted 1:2) was prepared in step 12 and was run with the SDS-PAGE from friday 13.09.2013. See figure below.
Figure: Ladder applied is Precision Plus ProteinTM Unstained Standards. WT stands for wildtype and is the unstransformed ER2566 samples.
There are noe additional bands in the ER1 samples compared to the unstransformed ER2566 sample, indicating that there are no GFP-RFP in the vesicles.
RFU was measured as decribed in step 12 of the small-scale vesicle preparation protocol. The blank sample had a value of 150 while the vesicle containing sample had the value of 8708.
There was run a excitation scan of the undiluted unstransformed ER2566 sample that was compared to the similar scan done on friday 13.09.2013. The results were as shown in the two figures below.
Figures: Fluorescence excitation scan of vesicles from the ER1 cells (left) and wildtype ER2566 (right).
There are no real differance between the samples other then strength of the signals. Both samples have peaks at 504, 540 and 582 nm. It is likely that there are no tGR construct in the vesicles.
PCR and digestion for putting on a prefix on the tat-GFP-RFP construct
A PCR were run on the plasmid containing tat_GFP_RFP construct with the F_prefix_tat and R_pl.b primers. This would in theory give us a PCR product that is the same size as the original plasmid. 15 µL of the PCR-product and 4 µL of the linerized pSB1C3 backbone where digested with the Enzyme Master Mix for Plasmid Backbone according to the recomended protocole .
The PCR product and the digested PCR product were run on a agarose gel, giving the result as indicated in the figure below.
Ladder applied is GeneRulerTM 100 bp plus DNA ladder. The ladder is for some reason not completly functional.
As there were not just one band and the ladder was unreliable, no fragment was cut out of the gel.
Sunday 22.09.2013
Analysing DNA sequencing results
The results from the last round of sequencing (see monday 09.09) was analysed with the prefered sequence. The alignments were as follows:
.
.
Figures: Alignment of tat_ProteinG D8-1A, tat_ProteinG D8-1B, tat_ProteinG D8-2A, tat_ProteinG D-2B and tat_ProteinG  S3-2A. These sequences either makes no sense or does not have the exact sequence we are looking for. These constructs will not be applied for later.
Figure: Alignment of tat_ProteinG S3-2B. This sequence is just how its supposed to be and will be used later in a vesicle isolation.
The alignments of the tat_GFPmut3 samples did not give any sense at all in will therefore not be included here as figures.
Monday 23.09.2013
Sending Biobricks
The biobricks tat_GFP_RFP (BBa_K1082001) and the Pm/XylS promoter system (BBa_K1082002) was sent in to the registery.
DNA samples sendt in for sequencing
The rest of the biobrick tat_GFP_RFP (BBa_K1082001) was sent in for sequencing with the F_seq and R_seq primers. The tat_ProteinG S3-2B (see sunday 22.09.2013) was also sent in for sequencing with the R_seq primer
Wednesday 25.09.2013
Step 1 and 2 of the [https://2013.igem.org/Team:NTNU-Trondheim/Protocols small-scale vesicle preparation] were performed with ER2566 transformed with the tat_ProteinG S3-2B (see sequencing results for sunday 22.09.2013) that had a postive sequencing result. One colony were selected from the recently transformed bacteria plate and incubated for 16 hours in step 2 in a total of 1L LB. The cultures were incubated aerobically in LB media with ampenicillin (100 µL/mL) in step 1 and aerobically in plain LB in step 2.
Testing of BioBrick promoter
Because non-satisfying result from the last test, we did another test. This was similar to the one done on thursday 12.09.13 with the same time-series and inducer concentrations.
Thursday 26.09.2013
PCR and digestion for creating a briobrick of tat_GFP_RF
PCR was run on the tat_GFP_RFP PCR-product from sunday 16.09. This time the F_prefix_tat primer and a reverse primer that anneals to the suffix in plasmid was applied. The PCR-prduct was digested with EcoRI and PstI. The digested PCR-product was run on a gel and one gelband was cut out. The fragment was purified with the QIAquick PCR Purification kit .
Ligation
The purified and digested (see above) tat_GFP_RFP with prefix was added into a ligation mix with the digested pSB1C3 backbone according to the iGEM protocole . The ligation product was transformed into DH5α cells.
Thursday 26.09.2013
Small-scale vesicle preparation
Step 4-12 of the small-scale vesicle preparation were performed with ER1. In addition a 10-6 dilution of the sample were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plate had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4. The samples were diluted 1:10 with LB media and had a value of 0.165.
Samples for SDS-PAGE (one undiluted and one diluted 1:2) was prepared in step 12 and stored in the fridge.
RFU was measured as decribed in step 12 of the small-scale vesicle preparation protcol. The blank sample had a value of 5531 while the vesicle containing sample had the value of 44 255.
Friday 27.09.2013
SDS-PAGE were performed with the samples from the day before (see figure below):
Figure: Ladder applied is Precision Plus ProteinTM Unstained Standards. The bands are not as visible as with the naked eye.
There are no additional bands compared with the wildtype ER2566 vesicles (see sunday 15.09) and therefore no indication that Protein G are in the vesicles.
Sunday 29.09.2013
Analysis of sequencing results
Results of DNA samples that were sent in for sequencing on monday 23.09 ware analysed.
Figure 1.
Figure 1.
Figures: Sequencing results of the tat_GFP_RFP construct with the F_seq (left) and R_seq (right) primer.
The new sequencing results from the ER1 sample that is supposed to have the tat_GFP_RFP construct does not make sense. It is indeed very unclear why our construct suddenly is lost.
Figure: Sequencing results of the tat_ProteinG S3_2B construct with the R_seq primer..
Putting this tat_ProteinG results together with the results from sunday 22.09 with the forward (F_seq) primer gives the alignment as follows:
Figure: Alignment of the full tat_ProteinG construct.
The Protein G gene from the ''Streptococcus dysgalactiae ssp equisimilis'' sample we collected from the hospital are clearly missing two bigger segments compared to the known gene sequence . We conclude that this is Protein G as the rest of the sequence aligns well.
PCR on fluorescent proteins
PCR was run on YFP ( BBa_E0030 ), CFP ( BBa_E0020 ), GFP ( BBa_E0040 ), BFP ( BBa_K5921000 ) and SYFP ( BBa_K864100 ) by using F_seq and R_seq as primers. These primers anneal to VF2 ( BBa_G00100 ) and VR ( BBa_G00101 ). The products were run on a gel and all samples had the expected size of about 1000 bp (see figure below).
Ladder applied is GeneRulerTM 1 kb DNA ladder..
Analysis of sequencing results
Another 3A assembly was done with the PmXyls promoter, GFP and backbone. The result from the test showed no fluorescence, so the GFP was not expressed as it should.
Retrieved from " http://2013.igem.org/Team:NTNU-Trondheim/Notebook/September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
