<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU-Trondheim_Notebook_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU-Trondheim/Notebook/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU-Trondheim/Notebook/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/August&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU-Trondheim/Notebook/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU-Trondheim/Notebook/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Trondheim iGEM 2013 </TITLE></P><DIV class="layout-1000"><DIV class="row"><DIV id="cssmenu"><UL><LI class="active"><A href="https://2013.igem.org/Team:NTNU-Trondheim"><SPAN>Home</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Project</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Project"><SPAN>Project description</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/novelapproach"><SPAN>A novel approach</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Model"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Experiments_and_Results"><SPAN>Experiments and Results</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Parts"><SPAN>BioBrick Parts</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Acknowledgements"><SPAN>Acknowledgements</SPAN></A></LI></UL><LI class="has-sub"><A href="#"><SPAN>Technical Stuff</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Notebook"><SPAN>Notebooks</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Primers"><SPAN>Primers</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Team</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Team"><SPAN>Meet the team</SPAN></A></LI><LI class="last"><A href="https://igem.org/Team.cgi?id=1082"><SPAN>Official Team Profile</SPAN></A></LI><LI class="has-sub"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Outreach"><SPAN>Outreach</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Judging</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:NTNU-Trondheim/Achievements"><SPAN>Achievements</SPAN></A></LI><LI class="last"><A href="https://2013.igem.org/Team:NTNU-Trondheim/Medalcriteria"><SPAN>Medal criteria</SPAN></A></LI><LI class="last"><A href="https://2012.igem.org/Team:NTNU_Trondheim/Matchmaker"><SPAN>Matchmaker</SPAN></A></LI></DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:#C3A7F3;&gt;
&lt;p style=" text-align:center=""> Notebook</DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Thursday 01.08.13</DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Small-scale vesicle preparation</DIV><P>Step 4-7 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed. In addition a 10<SUP>-6</SUP> dilution of the ER2566 samples were made and plating of 100µL of 1 mL of the dilution on Chl-LA was performed in step 4. The plates were left in the incubator overnight. The plate had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH></TR><TR><TD>Transformed ER2566 (1)</TD><TD>0.34 </TD></TR><TR><TD>Transformed ER2566 (2)</TD><TD> 0.32 </TD></TR><TR><TD>Unstransformed ER2566</TD><TD> 0.37</TD></TR></TBODY></TABLE>

Due to complications in step 7, this vesicle isolation attempt was terminated.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Confirmation PCR of Gibson Construct</DIV><P>To confirm that our result from yesterday was in fact our construct with tat-sequense, GFP and RFP we performed several PCR reactions with different targets. We used 5 templates, 2 samples from the 2µl plats and 3 from the 5µl plates. All run with 4 different primersets. T<SUB>m</SUB> was calculated using <A href="https://www.neb.com/tools-and-resources/interactive-tools/tm-calculator"> T<SUB>m</SUB> Caculator</A>. Only the actual primer, not the flankingsequence was included in the calculations. The <A href="https://www.neb.com/protocols/2012/09/10/protocol-for-a-routine-pcr-reaction-e0553"> Phusion PCR Protocol</A> with 20µl sample was used.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Target</TH><TH>F_Primer</TH><TH>R_Primer</TH><TH>T<SUB>m</SUB></TH></TR><TR><TD>GFP_RFP</TD><TD>F_tat_GFP </TD><TD>R_RFP</TD><TD>58°C</TD></TR><TR><TD>Tat_GFP_RFP)</TD><TD>F_pl.b_tat</TD><TD>R_RFP </TD><TD>64/72°C</TD></TR><TR><TD>Tat_GFP</TD><TD>F_pl.b_tat</TD><TD>R_GFP </TD><TD>58°C</TD></TR><TR><TD>Tat</TD><TD>F_pl.b_tat</TD><TD>R_tat </TD><TD>72°C</TD></TR></TBODY></TABLE>

All were run on GelGreen 0.8% Agarose. Bands indicate no GFP, tat or tat_GFP_RFP construct present, and tat_RFP-construct(which should not be possible) present. </P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Friday 02.08.13</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Transformation</DIV><P>Biobrick <A href="http://parts.igem.org/Part:BBa_J04450"> BBa_J04450</A> (in plasmid with kanamycin resistance) was transformed into ''E.coli'' strain ER2566 according to our <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Transformation"> transformation protocole</A>.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Saturday 03.08.13</DIV><P>Step 1 and 2 of the were performed <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> with ''E.coli'' strains ER2566. Untransformed ER2566 was incubated for 14 hours, transformed ER2566 (with biobrick <A href="http://parts.igem.org/Part:BBa_J04450"> BBa_J04450</A>) for 14 and 16 hours in step 2. The untransformed sample was incubated in plain LB whereas the transformed samples were incubated in LB with kanamycin (50µL/mL)</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Sunday 04.08.13</DIV><P>Step 4-12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed. In addition a 10<SUP>-6</SUP> dilution of the transformed ER2566 samples (both 14 and 16 hour incubation time) were made and plating of 100µL of 1 mL of the dilution on Kan-LA was performed in step 4. The plates were left in the incubator overnight. ''The plates had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH></TR><TR><TD>Unstransformed ER2566 14h</TD><TD>0.324 </TD></TR><TR><TD>Transformed ER2566 14h</TD><TD> 0.449</TD></TR><TR><TD>Unstransformed ER2566 16h</TD><TD> 0.493</TD></TR></TBODY></TABLE>

Samples for SDS-PAGE was prepared in step 12.
RFU was measured as decribed in step 12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> protcol. Results were as indicated in the table below:<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>RFU (exitation/emission at 515/640 nm)</TH></TR><TR><TD>Empty</TD><TD>100</TD></TR><TR><TD>Blank</TD><TD> 4 656</TD></TR><TR><TD>Untransformed ER2566 14h</TD><TD> 24 549</TD></TR><TR><TD>Transformed ER2566 14h</TD><TD> 45 732</TD></TR><TR><TD>Transformed ER2566 16h</TD><TD> 30 655</TD></TR></TBODY></TABLE></P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Another round of Small-scale vesicle preparation</DIV><P>Step 1 and 2 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with untransformed ER2566 that was incubated for 14 hours in plain LB in step 2, transformed ER2566 ( with biobrick <A href="http://parts.igem.org/Part:BBa_K530015"> BBa_K530015</A>) that were incubated in LB with chloramphenicole (25 µL/mL) for 16 hours and the same transformed ER2566 in plain LB for 14 hours (incubated WITH chloramphenicole (25 µL/mL) in step 1).</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>04.08.13</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Small-scale vesicle preparation</DIV><P>Step 4-6 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed. In addition a 10<SUP>-6</SUP> dilution of the transformed ER2566 samples (both 14 and 16 hour incubation time) were made and plating of 100µL of 1 mL of the dilution on Kan-LA was performed in step 4. The plates were left in the incubator overnight. The plates had growth, indicating that the bacteria still had their plasmid. This result also means that transformed ER2566 with Chl-resistance can hold on to their plasmid even without a selective marker in the media for at least 14 hours of incubation.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH></TR><TR><TD>Unstransformed ER2566 14h</TD><TD>0.379 </TD></TR><TR><TD>Transformed ER2566 14h -Chl</TD><TD> 0.369</TD></TR><TR><TD>Unstransformed ER2566 16h +Chl</TD><TD> 0.373</TD></TR></TBODY></TABLE>

The experiment was terminated after step 6 because we will not be needing the results.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">SDS-PAGE of vesicle samples</DIV><P>SDS-PAGE was run on the samples from the day before. As figure 7 shows the unstransformed sample and the transformed ER2566 14h had a clear indication of vesicles. According to the SDS-PAGE the untransformed sample had a greater content of protein compared to ransformed ER2566 14h, whereas the transformed samples had more vesicles than the unstransformed sample judging from the RFU measurements (table, sunday 04.08.13). It seems that transformation with antibiotics in the media increases vesicle production, but decreases production of vesicle assosiated proteins.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure  7: Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Yet another round of small-scale vesicle preparation</DIV><P>Step 1 and 2 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with ER2566 transformed with construct from the second Gibson assembly attempt. Two colonies (see picture 8) were selected and incubated for 14 and 16 hours in step 2. All of the cultures were incubated in LB media with ampenicillin (100 µL/mL).</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 8</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Different PCR with Gibson construct</DIV><P>The results from the last confirmation PCR gave strange results, so it was decided to run it again. Changes where; the whole primer including the flanking sequence to calculate the annealing temperature, included the RFP only, and we used 2-step PCR. The results were the same. Confirmed RFP and tat_RFP, but no tat_GFP_RFP construct. It is likely that the GFP is not present in our product. A new transformation was performed with the GFP to use in next Gibson Assembly.
The tat_RFP construct was purified and frozen at -20°C.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Tuesday 06.08.13</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Small-scale vesicle preparation</DIV><P>Step 4-12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed. In addition a 10<SUP>-6</SUP> dilution of all the samples  were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plates had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH></TR><TR><TD>14-1</TD><TD>0.432 </TD></TR><TR><TD>14-2</TD><TD>0.391</TD></TR><TR><TD>16-1</TD><TD> 0.425</TD></TR><TR><TD>16-2</TD><TD>0.420</TD></TR></TBODY></TABLE>

The cell cultures and cell pallets were not red or pink when they were taken out of the incubator and centrifuge. The pallet did, however, start to get a red color after a while. This indicate that the cells reqiure a more aerobic enviroment for production of RFP.
Samples for SDS-PAGE was prepared in step 12.
RFU was measured as decribed in step 12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> protcol. As the vesicles were expected to contain RFP, RFU was also measured without FM64-4 at exitation/emission at 605/670 (the known exitation/emission for RFP) Results were as indicated in the table below:<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>RFU (exitation/emission at 515/640 nm) with FM64-4</TH><TH>RFU (exitation/emission at 605/670 nm) without FM64-4</TH></TR><TR><TD>Empty</TD><TD>36</TD><TD>3258 </TD></TR><TR><TD>Blank</TD><TD>2148</TD><TD>2614 </TD></TR><TR><TD>14-1</TD><TD>3571 </TD><TD>4104</TD></TR><TR><TD>14-2</TD><TD> 46480</TD><TD>6181 </TD></TR><TR><TD>16-1</TD><TD>3475 </TD><TD>3629 </TD></TR><TR><TD>16-2</TD><TD>17430 </TD><TD>4565</TD></TR></TBODY></TABLE>

NB! Later it was discovered that the RFP that we have in the vesicles (from biobrick <A href="http://parts.igem.org/wiki/index.php/Part:BBa_E1010"> Part:BBa_E1010</A> has an exitation/emission peak at 584/607nm.
</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Colony PCR of GFP</DIV><P>We transferred around 15 colonies to 20µl dH<SUB>2</SUB>O and incubated in the PCR-machine at 95°C for 15 min. Sentrifuged for 2 min 14.000rpm. 1µl was used as template for PCR. Gel electroforeses confirm a band around 750bp which is the expected size of the GFP. DNA purification was performed using a putification kit and the concentration was measured using Nanodrop.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Gibson Assembly 3</DIV><P>A new gibson with the newly transformed GFP was performed. Insert and vector concentrations of 200ng, 10µl DNA 10µl Gibson Mastermix. 50°C, 15 min. A sample of 2µl was removed after 5 min. Product was incubated in -20°C immidiately to stop the reaction. 2µl transformed in ER2566 and plated. Control with all DNA was plated on Amp and Chl plates.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Plate</TH><TH>Colonies</TH><TH>Concentration(ng/µl)</TH></TR><TR><TD>1</TD><TD>57 </TD><TD>108.1 </TD></TR><TR><TD>130 </TD><TD>109.3 </TD></TR><TR><TD>3</TD><TD>52 </TD><TD>29.3 </TD></TR><TR><TD>4</TD><TD> 12</TD><TD>19.6 </TD></TR><TR><TD>5</TD><TD>38 </TD><TD>99.3 </TD></TR><TR><TD>6</TD><TD>31 </TD><TD>25.4 </TD></TR><TR><TD>7</TD><TD>85 </TD><TD>23.3 </TD></TR><TR><TD>8</TD><TD>42 </TD><TD>24 </TD></TR><TR><TD>Control Amp</TD><TD>95 </TD></TR><TR><TD>Control Chl</TD><TD>34 </TD></TR></TBODY></TABLE>

The Concentration was measured after miniprep using nanodrop. The colonies on both controlplates indicate contamination at some stage. 
A new transformation was started with RFP, BB and Ptet GFP <A href="http://parts.igem.org/BBa_I13522">BBa_I13522</A>, to check if the different GFP will work. Tat was isolated from ER2566 cells using colony PCR and product used as template in PCR with tat-primers. They were plated and incubated at 37°C overnight.
</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Wednesday 07.08.13</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">SDS-PAGE of vesicle samples</DIV><P>SDS-PAGE was run on the samples from the day before. As showed in figure 9 all of the samples had vesicles, but sample 14-2 had a higher density.</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 9: Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">New transformation</DIV><P>BB, RFP, GFP and tat was transformed again to try and eliminate contamination at any possible step. The standard iGEM <A href="http://parts.igem.org/Help:Protocols/Transformation"> Transformation Protocol</A> was used.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Another round of small-scale vesicle preparation</DIV><P>Step 1 and 2 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed with ER2566 transformed with construct from the second Gibson assembly attempt. Two colonies (see picture 8) were selected and incubated for 14 and 16 hours in step 2. The cultures were incubated aerobically in LB media with ampenicillin (100 µL/mL) in step 1 and aerobically in plain LB in step 2. The samples are again labeled as 'hours incubation in step 2 - colony number'</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Confirmation PCR of Gibson Construct with Taq Polymerase</DIV><P>We got several bands on the gel from gibson, it is unsure if they are the right size so to confirm what product was made in the gibson assembly we ran a confirmation PCR using Taq Polymerase <A href="https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273"> Taq Pol Protocol</A>.
Different combinations of primers to detect different combinations.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>Target</TH><TH>F-primer</TH><TH>R_primer</TH><TH>T<SUB>m</SUB></TH></TR><TR><TD>A</TD><TD>RFP</TD><TD>F_l_RFP</TD><TD>R_RPF</TD><TD>66</TD></TR><TR><TD>B</TD><TD>GFP+RFP</TD><TD>F_tat_GFP</TD><TD>R_RFP</TD><TD>66</TD></TR><TR><TD>C</TD><TD>Tat+GFP+RFP</TD><TD>F_pl.b_Tat</TD><TD>R_RFP</TD><TD>63</TD></TR><TR><TD>D</TD><TD>GFP</TD><TD>F_tat_GFP</TD><TD>R_l_GFP</TD><TD>68</TD></TR><TR><TD>E</TD><TD>Tat</TD><TD>F_pl.b_Tat</TD><TD>R_Tat</TD><TD>63</TD></TR><TR><TD>F</TD><TD>Tat+GFP</TD><TD>F_pl.b_Tat</TD><TD>R_GFP</TD><TD>63</TD></TR></TBODY></TABLE>

Band was visible for RFP around 750bp, so that is confirmed. Strong band for the Tat_GFP_RFP construct, but the size is only around 850bp. For it to contain tat and the 2 FPs it should be around 1600bp. It is probable that tat has attached to the RFP instead of the GFP even though the primers are not designed so that this is possible. More tests will be run.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Thursday 08.08.13</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Small-scale vesicle preparation</DIV><P>Step 4-12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> were performed. In addition a 10<SUP>-6</SUP> dilution of all the samples  were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plates with bacteria from colony 1 had growth (approximately 250-300 colonies) indicating that the bacteria still had their plasmid. The control plate with bacteria from colony 2 had 14 colonies (14-2) and 2 colonies (16-2).
The cellmedia with bacteria from colony 1 (14-1 and 16-1) has a pink color, while the other samples did not.
Optical density (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media. The true OD of 14-1 and 16-1 might be disturbed by the production of RFP, giving these samples a higher OD.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>OD<SUB>600</SUB></TH></TR><TR><TD>14-1</TD><TD>0.566</TD></TR><TR><TD>14-2</TD><TD>0.400</TD></TR><TR><TD>16-1</TD><TD>0.555</TD></TR><TR><TD>16-2</TD><TD>0.504</TD></TR></TBODY></TABLE>

Sample 16-1 was disregarded for RFU-measurements in step 12 due to an experimental mistake.
Samples for SDS-PAGE was prepared in step 12.
RFU was measured as decribed in step 12 of the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> small-scale vesicle preparation</A> protcol. As the vesicles were expected to contain RFP if the tat sequence is present, RFU was also measured without FM4-64 at exitation/emission at 605/670 (the known exitation/emission for RFP). Results were as indicated in the table below:<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>RFU (exitation/emission at 515/640 nm) with FM4-64</TH><TH>RFU (exitation/emission at 584/607 nm) with FM4-64</TH></TR><TR><TD>Empty</TD><TD>37</TD><TD>18147</TD></TR><TR><TD>Blank</TD><TD>1736</TD><TD>45 927</TD></TR><TR><TD>14-1</TD><TD>9030</TD><TD>40 818</TD></TR><TR><TD>14-2</TD><TD>20 745</TD><TD>31 325</TD></TR><TR><TD>16-1</TD></TR><TR><TD>16-2</TD><TD>46 126</TD><TD>37 304</TD></TR></TBODY></TABLE>

As the highest 584/607-measurement is the blank sample, we conclude that there are no RFP present in the vesicles.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PCR</DIV><P>Primers for CFP (<A href="http://parts.igem.or/BBa_E0020">BBa_E0030</A>) and YFP (<A href="http://parts.igem.org/BBA_0030">BBa_E0030</A>) arrived today, so PCR was run with these primers. TO avoid any contamination, we added 1 µl of Dpn1 to the amplicon and put it in a 37 °C waterbath for an hour. We used the QIAGEN PCR purification kit to isolate the DNA and measured the concentration by  <A href="http://www.nanodrop.com/library/nd-1000-v3.7-users-manual-8.5x11.pdf"> NanoDrop ND-1000 Spectrophotometer</A>.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>Concentration (ng/µl)</TH></TR><TR><TD>YFP</TD><TD>64</TD></TR><TR><TD>CFP</TD><TD>92.8</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Restriction cutting of construct</DIV><P>2 different samples of the gibson assembly product was used in restriction analysis with 3 enzymes. Two of the enzymes cut on either side of the construct and the third cuts the GFP at 323 bp, roughly in the middle of the gene. 4 different combinations were performed. We used the restriction digest <A href="http://parts.igem.org/Help:Protocols/Restriction_Digest"> protocol</A> from iGEM webpage.

<TABLE style="border:1px solid black;"><TBODY><TR><TH>#</TH><TH>Enzyme<SUB>1</SUB></TH><TH>Enzyme<SUB>2</SUB></TH></TR><TR><TD>1</TD><TD>EcoRI</TD><TD>SpeI</TD></TR><TR><TD>2</TD><TD>EcoRI</TD><TD>PmlI</TD></TR><TR><TD>3</TD><TD>PmlI</TD><TD>SpeI</TD></TR><TR><TD>4</TD><TD>BsaAI</TD><TD>SpeI</TD></TR></TBODY></TABLE>

FIND PIC OF GEL!
</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Friday 09.08.13</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">SDS-PAGE of vesicle samples</DIV><P>SDS-PAGE was run on the samples from the day before. As showed in figure 10 the 16-2 sample seems to have the highest protein content followed by 16-1. It seems that the 16 hour incubation is to prefer when considering the SDS-PAGE (figure 9) and the RFU-measurements (table, thursday 08.08.3013)</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">Figure 10:Ladder applied is Precision Plus Protein<SUP>TM</SUP> Unstained Standards.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PCR</DIV><P>New PCR on GFP, RFP, BB and tat. Very weak band on BB. Ran new Expand high fidelity DNA polymerase kit with BB-PCRproduct as template. Still low concentration.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Gibson Assembly 4</DIV><P>In addition to the Tat_GFP_RFP construct(TGR) it was made a Tat_CFP_YFP construct(TCY). Tranformed(X5) and incubated overnight.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Saturday 10.08.2013</DIV><P>The colonies on the 10 plates were counted.

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Plate</TH><TH>TCY</TH><TH>Colonies TGR</TH></TR><TR><TD>1</TD><TD>59</TD><TD>45</TD></TR><TR><TD>2</TD><TD>32</TD><TD>36</TD></TR><TR><TD>3</TD><TD>62</TD><TD>39</TD></TR><TR><TD>4</TD><TD>49</TD><TD>14</TD></TR><TR><TD>5</TD><TD>100</TD><TD>20</TD></TR></TBODY></TABLE>

The plates with the most colonies from both constructs(TCY:plate 5, TGR: plate 1) were used for colony PCR, appr. 20 colonies from each plate was transferred to 20 µl of dH<SUB>2</SUB>O, and incubated in the PCR for 15 min, at 95 °C. One colony from the same plates was incubated in liquid media and left overnight for miniprep the next day. 
The results were disappointing again and confirmation PCR did not give the desired sizes.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Monday 12.08.2013</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Gibson assembly with different construct</DIV><P>The experiments using tat_2xFPs_BB have been unsuccessful at this point according to our confirmation PCRs. We are therefor trying to make a 2 piece construct instead; Tat_GFPmut3* and Tat_RFP. Gibson Assembly was performed as described in the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> protocol page</A>.

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Template</TH><TH>Concentration(ng/µl)</TH><TH>Volume in GA(µl)</TH></TR><TR><TD>Tat</TD><TD>36</TD><TD>3</TD></TR><TR><TD>GFPmut3*</TD><TD>58.1</TD><TD>2</TD></TR><TR><TD>BB</TD><TD>39</TD><TD>5</TD></TR></TBODY></TABLE>

Add 10 µl of gibson mastermix. Incubate for 15 min at 95 °C.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Template</TH><TH>Concentration(ng/µl)</TH><TH>Volume in GA(µl)</TH></TR><TR><TD>Tat</TD><TD>36</TD><TD>3</TD></TR><TR><TD>RFP</TD><TD>140</TD><TD>1</TD></TR><TR><TD>BB</TD><TD>39</TD><TD>5</TD></TR></TBODY></TABLE>

Add 10 µl dH<SUB>2</SUB>O and 10 µl of gibson mastermix. Incubate for 15 min at 95 °C. Then both were transformed in DH5α cells.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Improving concentration of BB</DIV><P>The BB was run with Expand high fidelity polymerase <A href="http://francois.schweisguth.free.fr/protocols/High_fildelity_roche.pdf"> Protocol</A> to get a higher concentration, so tht it will be better for future gibson assemblies. The annealing temperatur was set to 65 °C. 

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Reactant</TH><TH>µl</TH></TR><TR><TD>Buffer 2</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>0.5</TD></TR><TR><TD>Primers</TD><TD>0.5</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>Polymerase</TD><TD>0.25</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>15.25</TD></TR></TBODY></TABLE>

This gave an even weaker result, which is unexpected. Our next move is to use colonies of the backbone and incubate in liquid media for new miniprep and hopefully a better yield. As we are running short on purified tat that will also be incubated and run on the PCR again.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PCR with Protein G</DIV><P>Our Primers for protein G has arrived, as has the 5 different ''Streptococcus dysgalactiae ssp equisimilis'' samples collected from St. Olavs hospital. The strains of the bacteria samples are unknown as they are collected from patients. There are 3 different  forward primers, one has the tat sequence overhang (F_tat_PrG), one has the natural sequence that the protein has in ''Streptococcus dysgalactiae ssp. equisimilis'' (F_pl.b_PrG) and one has the pelB sequence overhang (R_pl.b_pelB_PrG). There are two different reverse primers, one that will anneal to the end of the DNA sequence for the gene (R_pl.b_PrG) and another that will anneal to the end of the protein coding sequence (see <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> registered gene sequence</A>). They will be run in 6 different combinations:

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>F_Primer</TH><TH>R_Primer</TH><TH>T<SUB>m</SUB></TH></TR><TR><TD>A</TD><TD>F_pl.b_PrG</TD><TD>R_pl.b_PrG</TD><TD>61</TD></TR><TR><TD>B</TD><TD>F_pl.b_PrG</TD><TD>R_pl.b_PrGstop</TD><TD>64</TD></TR><TR><TD>C</TD><TD>F_tat_PrG</TD><TD>R_pl.b_PrG</TD><TD>63</TD></TR><TR><TD>D</TD><TD>F_tat_PrG</TD><TD>R_pl.b_PrGstop</TD><TD>68</TD></TR><TR><TD>E</TD><TD>F_pl.b_pelB_PrG</TD><TD>R_pl.b_PrG</TD><TD>63</TD></TR><TR><TD>F</TD><TD>F_pl.b_pelB_PrG</TD><TD>R_pl.b_PrGstop</TD></TR></TBODY></TABLE>
 
As we have 5 different ''Streptococcus dysgalactiae ssp. equisimilis'' samples, 30 different PCR-reactions were run.
Standard conditions for Phusion PCR was used. The products was run on gel for appr. 1h. See figure below</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 11.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify">
A very clear band with the  size of around 1200-1300 bp was found in sample D1. This is a smaller DNA fragment compared to the <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> known gene sequence</A> of about 1700 bp. Since the size of Protein G is known to vary among the different strains, we can still conclude that our PCR-product is Protein G. The PCR product with the genomic DNA sample 1 as template and F_tat_PrG and R_pl.b_PrGstop as primers is most likely Protein G.</DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Tuesday 13.08.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Restriction digest of BB</DIV><P>To remove any contaminants in our BB sample we are running a restriction digest with only one enzyme, HindIII, and cutting the correct band from the gel. The band was purified with the <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols"> QIAquick PCR purification kit</A>.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PEC</DIV><P>To link two PCR products together we used a tecnique called PEC-Polymerase extension cloning. This method uses the established overhangs of the fragments to link them toghether in the PCR using only the polymerase(phusion). And then after some 10 cycles the program is paused and primers for the whole construct is added to amplify that. 

The first experiment using SLIC: To link toghether tat_GFP and tat_GFPmut3*. A two-step PCR program was setup with 10 cycles in the first step and 20 cycles in the second step. For the first step the flanking(overlapping) sequences were used to determine annealing temperature = 72 °C, and for the second step only the primers without the flanking sequence were used to calculate anealingtemperature = 62 °C. Other than that standard PCR conditions using phusion polymerase was used. Se figure below for results</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; ">Figure 12: Agarose Electrophoresis gel with (from the left) tat_GFP, tat_GFPmut, tat_GFP_l_RFP in plasmid (SLIC) and tat_CFP_l_YFP in plasmid (SLIC).</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>CPEC-circular polymerase extension cloning is the same principle as PEC, only circular. Du to this there is no need for primers, only that the overhans all fit to each other. We attempted this method with our to FP constructs. Tat_GFP_RFP and Tat_CFP_YFP.<TABLE style="border:1px solid black;"><TBODY><TR><TH>Reactant</TH><TH>Volume(µl)</TH><TH>Concentration(ng/µl)</TH></TR><TR><TD>HF-buffer</TD><TD>4</TD></TR><TR><TD>dNTP</TD><TD>1.6</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.2</TD></TR><TR><TD>Vector BB</TD><TD>4</TD><TD>50</TD></TR><TR><TD>GFP</TD><TD>1</TD><TD>88.4</TD></TR><TR><TD>RFP</TD><TD>0.5</TD><TD>140</TD></TR><TR><TD>Tat</TD><TD>1.7</TD><TD>36.5</TD></TR><TR><TD>CFP</TD><TD>1</TD><TD>92.8</TD></TR><TR><TD>YFP</TD><TD>1</TD><TD>64</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>up tp 20</TD></TR></TBODY></TABLE>

One reaction with Tat_GFP_RFP_BB and one with Tat_CFP_YFP_BB. (I dont have the PCR conditions used here!!ANYONE??).
</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Wednesday 14.08.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PCR with TCY construct</DIV><P>Standars phusion polymerase protocol with a 20 µl reaction. (FIND PIC)


</P><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PCR to link tat and GFP</DIV><P>As the PEC reaction did not confirm a link, were trying the same using PCR. Phusion polymerase 20 µl reaction. Annealing temperature 72/62 in 2-step PCR.(FIND PIC).</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Restriction digest of tGR and tCY product from CPEC(13.08.13</DIV><P>The standard iGEM [http://parts.igem.org/Help:Protocols/Restriction_Digest Restriction Digest Protocol] was used applied with SpeI enzyme on the tGR and tCY products from yesterday (13.08.13). The digested samples were then run on a gel together with a previous PCR-product of BB (see figure below) that served as a referance.

</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 13:From the left; tGR, tCY and BB (PCR product)</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>As indicated in the picture, there does not seem to be any tGR or tCY construct as the restricted products are the same size as the BB when they should be longer. There also seems to be a bit of contamination in the BB sample.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PEC to connect tat to Protein G</DIV><P>We applied <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#PEC">PEC</A> to attach tat to Protein G. The product was then purified by <A hef="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#PCR_Purification"> Gibson Assembly<SUP>TM</SUP> Cloning Kit</A> and then run on a agaorse gel together with a PCR product of Protein G without the tat attached (see figure below)</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 14:From the left; Protein G with tat attached and Protein G without tat.&lt;</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>The PEC product is clearly a little bit longer then the Protein G PCR product without tat attched. This indicates that PEC was successful.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Thursday 15.08.2013</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Purify BB by running it on gel</DIV><P>To remove any contaminants from the BB sample we ran 50 µl og BB sample on a gel, cut it and purified it using <A href="https://2013.igem.org/Team:NTNU-Trondheim/Protocols#Purification_of_DNA_fragments_from_an_agarose_gel"> QIAquick PCR Purification kit</A>.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PEC to connect tat to Protein G</DIV><P>The same conditions as last time were used. BB: 4 µl and tat: 3µl. Phusion 20 µl reaction.

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Step</TH><TH>Temperature(°C)</TH><TH>Time(sec)</TH></TR><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD></TR><TR><TD>Annealing</TD><TD>52</TD><TD>30</TD></TR><TR><TD>Elongation</TD><TD>72</TD><TD>110</TD></TR><TR><TD>Final Elongation</TD><TD>72</TD><TD>5 min</TD></TR></TBODY></TABLE>

The product was transformed into DH5α cells and ER2566 cells.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Direct transformation(DT) of BB and Tat_PrG</DIV><P>1:1 Ratio, BB: 2 µl, Tat_PrG: 1.5 µl. Directly transferred into DH5α cell and ER2566 cells</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">SLIC with BB and Tat_PrG</DIV><P><TABLE style="border:1px solid black;"><TBODY><TR><TH>Reactant</TH><TH>Volume</TH></TR><TR><TD>NEB Buffer</TD><TD>1</TD></TR><TR><TD>BSA</TD><TD>1</TD></TR><TR><TD>T4 polymerase</TD><TD>0.2</TD></TR><TR><TD>BB vector</TD><TD>2.1 </TD></TR><TR><TD>Tat_PrG</TD><TD> 3.5</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>2.2</TD></TR></TBODY></TABLE>
Incubated in room tempreature for 2.5 minutes, then 10 min on ice and transformed into DH5α cell and ER2566 cells.
These are the transformations done today:<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>Method</TH><TH>Cells</TH></TR><TR><TD>1</TD><TD>CPEC</TD><TD>DH5α</TD></TR><TR><TD>2</TD><TD>CPEC</TD><TD>ER2566</TD></TR><TR><TD>3</TD><TD>DT</TD><TD>DH5α</TD></TR><TR><TD>4</TD><TD>DT</TD><TD>ER2566</TD></TR><TR><TD>5</TD><TD>SLIC</TD><TD>DH5α</TD></TR><TR><TD>6</TD><TD>SLIC</TD><TD>ER2566</TD></TR><TR><TD>7</TD><TD>Control BB</TD><TD>DH5α</TD></TR><TR><TD>8</TD><TD>Control Tat_PrG</TD><TD>DH5α</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">New primers and PCR</DIV><P>New primers arrived for BB, tat and YFP. The new primers (except YFP primer) has overhengs where the equivalent primers had not. A new reverse primer for YFP was designed because the old one lacked the terminator (TAATAA) sequence New PCR was run with standard phusion protocol 50 µl reaction.(FIND PIC)


<TABLE style="border:1px solid black;"><TBODY><TR><TH>Primer name</TH><TH>Target</TH></TR><TR><TD>R_GFP_tat</TD><TD>tat</TD></TR><TR><TD>R_CFP_tat</TD><TD>tat</TD></TR><TR><TD>F_GFP_pl.b</TD><TD>BB or pl.b</TD></TR><TR><TD>R_CFP_pl.b</TD><TD>BB or pl.b</TD></TR><TR><TD>R_pl.b_YFPnew</TD><TD>YFP</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Friday 16.08.2013</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Redoing PCR form yesterday</DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Gibson with new TGR overhangs</DIV><P><TABLE style="border:1px solid black;"><TBODY><TR><TH>Reactant</TH><TH>volume</TH></TR><TR><TD>BB</TD><TD>2.5</TD></TR><TR><TD>Tat</TD><TD>0.3</TD></TR><TR><TD>GFP</TD><TD>0.7</TD></TR><TR><TD>RFP</TD><TD>0.45</TD></TR><TR><TD>Gibson mastermix</TD><TD>15</TD></TR></TBODY></TABLE>
The products were transformed and plated. A control for tat and BB was also plated. </P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">SLIC with new overhangs</DIV><P><TABLE style="border:1px solid black;"><TBODY><TR><TH>Reactant</TH><TH>volume</TH></TR><TR><TD>BB vector</TD><TD>4.3</TD></TR><TR><TD>Tat</TD><TD>0.3</TD></TR><TR><TD>GFP</TD><TD>1.4</TD></TR><TR><TD>RFP</TD><TD>0.9</TD></TR></TBODY></TABLE>

Transformed and plated.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PEC with new overhangs</DIV><P><TABLE style="border:1px solid black;"><TBODY><TR><TH>Reactant</TH><TH>Volume(µl)</TH><TH>Concentration(ng/µl)</TH></TR><TR><TD>HF-buffer</TD><TD>4</TD></TR><TR><TD>dNTP</TD><TD>1.6</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.2</TD></TR><TR><TD>Tat</TD><TD>1</TD></TR><TR><TD>GFP</TD><TD>1.5</TD></TR><TR><TD>RFP</TD><TD>1</TD></TR><TR><TD>dH<SUB>2</SUB>O</TD><TD>9.3</TD></TR></TBODY></TABLE>

(this one gave no construct)</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Confirmation PCR on gibson</DIV><P>
It was done a confirmation PCR on the remaining sample from gibson(before transformation) using F_pl.b_tat and R-pl.b_RFP with T<SUB>m</SUB>=72 °. (FIND PIC)</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Saturday 17.08.2013</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Minipreping of Protein G samples</DIV><PA total="" of="" 18="" liquid="" media="" samples="" containing="" tat_proteing="" plasmids="" were="" minipreped.="" in="" the="" table="" below="" there="" is="" an="" overview="" of="" the="" samples:="" table:="" c="CPEC," s="SLIC" and="" dt="direct" transformation.="">SampleMarkingSampleMarkingProteinG 1C (1)Protein 10DT (3)ProteinG 2S (1)Protein 11C (4)ProteinG 3DT (1)ProteinG 12DT (4)ProteinG 4C (2)ProteinG 13C (5)ProteinG 5S (2)ProteinG 14DT (6)ProteinG 6DT (2)ProteinG 15DT (7)ProteinG 7C (3)ProteinG 16DT (8)ProteinG 8S (3)ProteinG 17DT (9)ProteinG 9DT (3)ProteinG 18DT (10)<DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Confirmation PCR</DIV><P>INGRID&amp;ELLEN: I think this is where you guys did a mistake when you though you were doing conf. PCR on tCY/tGR but infact it was ProteinG. The correct proteinG pcr was run on sunday===

On Gibson TGR 1+2 and TCY, tat_PrG CPEC Samples 1-5, tat_PrG SLIC Samples 1-3, tat_PrG DT Samples 1-10. Ran on gel (FIND PIC)

Colonies from plates with TGR, TCY Gibson and SLIC were transferred to liquid media.

EVERYTHING THAT WAS DONE DURING THIS WEEKEND IS IN SOMEONE ELSES LABJOURNAL :) FILL IN PLEASE.. Im pretty sure we did some things!</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Sunday 18.08.2013</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Miniprep</DIV><P>The Gibson and SLIC with tGR and tCY that were transerred to liquid media yesterday were minipreped.</P></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Conformation PCR on tat_ProteinG</DIV><P>We did a confirmation PCR of all the 18 tat_ProteinG plasmids that were minipreped the day before (17.08.2013). Them we ran the PCR product on a gel (see figure below).</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 15: First row contains tat_ProteinG samples 1-9, second row contains tat_ProteinG samples 10-18 (see table from saturday 17.08.2013).</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Thursday 22.08.2013</DIV><P>Samples (see table below) were prepared and sent for sequencing. The DNA fragments (tat_GFP and tat_GFPmut3) were sent with their respectful reverse primer and the plasmids (the rest of the samples) were sent with the standard
iGEM forward sequencing primer. The samples were made with the concentration between 80-100 ng/µl. The samples with to low concentration was evaporated to get higher concentration. 

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample#</TH><TH>Sample name</TH></TR><TR><TD>1</TD><TD>tat_GFP
</TD></TR><TR><TD>2</TD><TD>tat_GFPmut3</TD></TR><TR><TD>3</TD><TD>SYFP</TD></TR><TR><TD>4</TD><TD>BFP</TD></TR><TR><TD>5</TD><TD>ProteinG (DT8)</TD></TR><TR><TD>6</TD><TD>ProteinG (S3)</TD></TR><TR><TD>7</TD><TD>tGR 1 (minipreped 18.08.13)</TD></TR><TR><TD>8</TD><TD>tGR 2 (minipreped 18.08.13)</TD></TR><TR><TD>9</TD><TD>ER1 (see figure 8)</TD></TR><TR><TD>10</TD><TD>ER2 (see figure 8)</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Friday 23.08.2013</DIV><P>Red ER1-cells (see figure 8) was first immobilized in agar and them viewed in a confocal microscope for seeing if the RFP was localized in the periplasm. The results can be seen in the two figures below:

</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>There are no indication that the RFP is more concentrated in the periplasm.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Saturday 24.08.2013</CENTER></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">PEC</DIV><P>We did a PEC reaction with:
1. Tat_GFP + RFP, T<SUB>m</SUB>= 72/72 °C (F_pl.b_tat + R_pl.b_RFP)
2. Tat + CFP, T<SUB>m</SUB>= 58/72 °C (F_pl._tat + R_l_CFP)
Same procedure as last time, 20 µl reaction, 1 µl primers added after 10 cycles. (FIND PIC)</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Confirmation PCR</DIV><P><TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>Target</TH></TR><TR><TD>1</TD><TD>Gibson TGR 1</TD></TR><TR><TD>2</TD><TD>Gibson TGR 2</TD></TR><TR><TD>3</TD><TD>SLIC TGR 1</TD></TR><TR><TD>4</TD><TD>SLIC TGR 2</TD></TR><TR><TD>5</TD><TD>PEC TGR</TD></TR><TR><TD>6</TD><TD>PEC Tat_CFP</TD></TR><TR><TD>7</TD>Control CFP</TR></TBODY></TABLE>

(FIND PIC)</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Tuesday 27.08.2013</DIV><P>Today we got the sequencing results on the plasmids/DNA fragments we sent last week. The figures below show the alignments of the samples (always first line) and the expected sequences (always second line). Red indicates proper alignment, blue indicates mismatches.</P><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure 1: Alignment of the tat_GFP DNA fragment. The GFP part of the sequence is correct, but we do not know is the tat sequence is correct as this was not included in the sequencing results.</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure:  Alignment of the tat_GFPmut3 DNA fragment. The GFPmut3 part of the sequence is correct, but we do not know is the tat sequence is correct as this was not included in the sequencing results.</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure:Alignment of SYFP. The SYFP biobrick has the correct sequence and it is highly unknown why our primers did not work in PCR amplification of the gene..</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure:Alignment of SYFP. The BFP biobrick has the correct sequence and it is highly unknown why our primers did not work in PCR amplification of the gene.</P></DIV></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of tat_ProteinG DT8. There is one segment missing in our ProteinG, as is expected as the ProteinG variant that we are dealing with is a shorter than the [http://www.ncbi.nlm.nih.gov/nuccore/X06173 registered gene sequence]. There is how ever an addition of a guanin residue in the beginning of the tat-sequence, which will cause a frameshift resulting in a non-functional protein product.</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of tat_ProteinG DT8. There is one segment missing in our ProteinG, as is expected as the ProteinG variant that we are dealing with is a shorter than the <A href="http://www.ncbi.nlm.nih.gov/nuccore/X06173"> registered gene sequence</A>. There is how ever an deletion of a guanin residue in the beginning of the tat-sequence, which will cause a frameshift resulting in a non-functional protein product.</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of tat_GFP_l_RFP (tGR 1). The aligment is just nonsense. This plasmid does not have the intended sequence.</P></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alingment of tat_GFP_l_RFP (tGR 2). The aligment is just nonsense. This plasmid does not have the intended sequence.</P></DIV>&gt;

<DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of tat_GFP_l_RFP (ER1). The sequence align almost perfectly. There seems to be some sort of extra insert at the linker region, but this insert is dividable by 3, so the reading frame is maintained. This is supported by the fact that the colonies with this construct is red (see figure 8).</P></DIV></DIV><DIV class="col3-2" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure: Alignment of tat_GFP_l_RFP (ER2). The aligment is just nonsense. This plasmid does not have the intended sequence.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""><CENTER>Thursday 27.08.2013</CENTER></DIV></DIV><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Preparing Protein G and tCY construct for sequencing</DIV><P>All the rest of the samples with tat_protein G that had a positive result on the PCR confrmation (see figure on sunday 18.08.2013) was prepared and sent for sequencing. The samples were made with the concentration between 80-100 ng/µl. The samples with to low concentration was evaporated to get higher concentration. 

<TABLE style="border:1px solid black;"><TBODY><TR><TH>Sample</TH><TH>Name</TH>Seq.ID</TR><TR><TD>1</TD><TD>ProteinG (S2)</TD><TD>17|J83</TD></TR><TR><TD>2</TD><TD>ProteinG (C4)</TD><TD>17|J84</TD></TR><TR><TD>3</TD><TD>ProteinG (C5)</TD><TD>17|J85</TD></TR><TR><TD>4</TD><TD>ProteinG (DT2)</TD><TD>17|J86</TD></TR><TR><TD>5</TD><TD>ProteinG (DT3)</TD><TD>17|J87</TD></TR><TR><TD>6</TD><TD>ProteinG (DT5)</TD><TD>17|J89</TD></TR><TR><TD>7</TD><TD>ProteinG (DT6)</TD><TD>17|J88</TD></TR><TR><TD>8</TD><TD>ProteinG (DT7)</TD><TD>17|J90</TD></TR><TR><TD>9</TD><TD>ProteinG (DT9)</TD><TD>17|J91</TD></TR><TR><TD>10</TD><TD>ProteinG (DT10)</TD><TD>17|J92</TD></TR><TR><TD>11</TD><TD>GIBSON TCY1</TD><TD>17|J93</TD></TR><TR><TD>12</TD><TD>GIBSON TCY2</TD><TD>17|J94</TD></TR><TR><TD>13</TD><TD>GIBSON TCY3</TD><TD>17|J95</TD></TR><TR><TD>14</TD><TD>SLIC TCY</TD><TD>17|J96</TD></TR></TBODY></TABLE></P></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Thursday 29.08.2013</DIV></DIV><DIV class="row"><DIV class="col12-2" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center=""> Confirmation PCR on sequenced construct</DIV><P>The sequencing confirmed our construct: Tat_GFP_RFP. It was confirmed in one of our previous Gobson assemblies by sequencing. We therefore wanted to run a PCR to confirm the results since we know that the construct is there.  The PCR was run with F_pl.b_tat and R_pl.b_RFP and the PCR products were run on a gel (see figure below):

</P><DIV class="col4" style="background-color:white;&gt;&lt;a href=" https:=""><P style="text-align:center; color:black; "> Figure:Gel of conformation PCR on ER1 (tGR) construct.</P></DIV></DIV><DIV class="row"><DIV class="col12-3" align="justify"><P>There is a distinct band below 1000 bp that we are not certain what is. There is also a visible band at around 1700 bp, which is the expected size of the tGR construct.</P></DIV><DIV class="row"><DIV class="col12-3" align="justify"><DIV class="col12-3" align="center" style="background-color:white;&gt;
&lt;p style=" text-align:center="">Freezing of tat_GFP_RFP construct</DIV><P>Bacteria with the tat_GFP_RFP construct that had a positive sequencing result (the ER1 plasmid) was prepared for freezing.

5 cryotubes: 800 µl of cell media(overnight incubated colonies of construct)
             400 µl of 60% glycerol

Stored in -80 freezer(Rack 6, Shelf 7: pink box marked iGEM).</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NTNU-Trondheim/Notebook/August">http://2013.igem.org/Team:NTNU-Trondheim/Notebook/August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU-Trondheim/Notebook/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU-Trondheim/Notebook/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU-Trondheim/Notebook/August&amp;oldid=303460" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></PA></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>