"
Page
Discussion
View source
History
teams
Log in
 
Team:NTNU-Trondheim/Notebook/August
From 2013.igem.org
Trondheim iGEM 2013
Home
Project
Project description
A novel approach
Modelling
Experiments and Results
BioBrick Parts
Acknowledgements
Technical Stuff
Notebooks
Protocols
Primers
Safety
Team
Meet the team
Official Team Profile
Outreach
Judging
Achievements
Medal criteria
Matchmaker
Notebook
Thursday 01.08.13
Small-scale vesicle preparation
Step 4-7 of the small-scale vesicle preparation were performed. In addition a 10-6 dilution of the ER2566 samples were made and plating of 100µL of 1 mL of the dilution on Chl-LA was performed in step 4. The plates were left in the incubator overnight. The plate had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.
Sample
OD600
Transformed ER2566 (1)
0.34
Transformed ER2566 (2)
0.32
Unstransformed ER2566
0.37
Due to complications in step 7, this vesicle isolation attempt was terminated.
Confirmation PCR of Gibson Construct
To confirm that our result from yesterday was in fact our construct with tat-sequense, GFP and RFP we performed several PCR reactions with different targets. We used 5 templates, 2 samples from the 2µl plats and 3 from the 5µl plates. All run with 4 different primersets. Tm was calculated using Tm Caculator . Only the actual primer, not the flankingsequence was included in the calculations. The Phusion PCR Protocol with 20µl sample was used.
Target
F_Primer
R_Primer
Tm
GFP_RFP
F_tat_GFP
R_RFP
58°C
Tat_GFP_RFP)
F_pl.b_tat
R_RFP
64/72°C
Tat_GFP
F_pl.b_tat
R_GFP
58°C
Tat
F_pl.b_tat
R_tat
72°C
All were run on GelGreen 0.8% Agarose. Bands indicate no GFP, tat or tat_GFP_RFP construct present, and tat_RFP-construct(which should not be possible) present.
Friday 02.08.13
Transformation
Biobrick BBa_J04450 (in plasmid with kanamycin resistance) was transformed into ''E.coli'' strain ER2566 according to our transformation protocole .
Saturday 03.08.13
Step 1 and 2 of the were performed small-scale vesicle preparation with ''E.coli'' strains ER2566. Untransformed ER2566 was incubated for 14 hours, transformed ER2566 (with biobrick BBa_J04450 ) for 14 and 16 hours in step 2. The untransformed sample was incubated in plain LB whereas the transformed samples were incubated in LB with kanamycin (50µL/mL)
Sunday 04.08.13
Step 4-12 of the small-scale vesicle preparation were performed. In addition a 10-6 dilution of the transformed ER2566 samples (both 14 and 16 hour incubation time) were made and plating of 100µL of 1 mL of the dilution on Kan-LA was performed in step 4. The plates were left in the incubator overnight. ''The plates had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.
Sample
OD600
Unstransformed ER2566 14h
0.324
Transformed ER2566 14h
0.449
Unstransformed ER2566 16h
0.493
Samples for SDS-PAGE was prepared in step 12.
RFU was measured as decribed in step 12 of the small-scale vesicle preparation protcol. Results were as indicated in the table below:
Sample
RFU (exitation/emission at 515/640 nm)
Empty
100
Blank
4 656
Untransformed ER2566 14h
24 549
Transformed ER2566 14h
45 732
Transformed ER2566 16h
30 655
Another round of Small-scale vesicle preparation
Step 1 and 2 of the small-scale vesicle preparation were performed with untransformed ER2566 that was incubated for 14 hours in plain LB in step 2, transformed ER2566 ( with biobrick BBa_K530015 ) that were incubated in LB with chloramphenicole (25 µL/mL) for 16 hours and the same transformed ER2566 in plain LB for 14 hours (incubated WITH chloramphenicole (25 µL/mL) in step 1).
04.08.13
Small-scale vesicle preparation
Step 4-6 of the small-scale vesicle preparation were performed. In addition a 10-6 dilution of the transformed ER2566 samples (both 14 and 16 hour incubation time) were made and plating of 100µL of 1 mL of the dilution on Kan-LA was performed in step 4. The plates were left in the incubator overnight. The plates had growth, indicating that the bacteria still had their plasmid. This result also means that transformed ER2566 with Chl-resistance can hold on to their plasmid even without a selective marker in the media for at least 14 hours of incubation.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.
Sample
OD600
Unstransformed ER2566 14h
0.379
Transformed ER2566 14h -Chl
0.369
Unstransformed ER2566 16h +Chl
0.373
The experiment was terminated after step 6 because we will not be needing the results.
SDS-PAGE of vesicle samples
SDS-PAGE was run on the samples from the day before. As figure 7 shows the unstransformed sample and the transformed ER2566 14h had a clear indication of vesicles. According to the SDS-PAGE the untransformed sample had a greater content of protein compared to ransformed ER2566 14h, whereas the transformed samples had more vesicles than the unstransformed sample judging from the RFU measurements (table, sunday 04.08.13). It seems that transformation with antibiotics in the media increases vesicle production, but decreases production of vesicle assosiated proteins.
Figure  7: Ladder applied is Precision Plus ProteinTM Unstained Standards.
Yet another round of small-scale vesicle preparation
Step 1 and 2 of the small-scale vesicle preparation were performed with ER2566 transformed with construct from the second Gibson assembly attempt. Two colonies (see picture 8) were selected and incubated for 14 and 16 hours in step 2. All of the cultures were incubated in LB media with ampenicillin (100 µL/mL).
Figure 8
Different PCR with Gibson construct
The results from the last confirmation PCR gave strange results, so it was decided to run it again. Changes where; the whole primer including the flanking sequence to calculate the annealing temperature, included the RFP only, and we used 2-step PCR. The results were the same. Confirmed RFP and tat_RFP, but no tat_GFP_RFP construct. It is likely that the GFP is not present in our product. A new transformation was performed with the GFP to use in next Gibson Assembly. The tat_RFP construct was purified and frozen at -20°C.
Tuesday 06.08.13
Small-scale vesicle preparation
Step 4-12 of the small-scale vesicle preparation were performed. In addition a 10-6 dilution of all the samples  were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plates had growth, indicating that the bacteria still had their plasmid.
Optical denisty (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media.
Sample
OD600
14-1
0.432
14-2
0.391
16-1
0.425
16-2
0.420
The cell cultures and cell pallets were not red or pink when they were taken out of the incubator and centrifuge. The pallet did, however, start to get a red color after a while. This indicate that the cells reqiure a more aerobic enviroment for production of RFP.
Samples for SDS-PAGE was prepared in step 12.
RFU was measured as decribed in step 12 of the small-scale vesicle preparation protcol. As the vesicles were expected to contain RFP, RFU was also measured without FM64-4 at exitation/emission at 605/670 (the known exitation/emission for RFP) Results were as indicated in the table below:
Sample
RFU (exitation/emission at 515/640 nm) with FM64-4
RFU (exitation/emission at 605/670 nm) without FM64-4
Empty
36
3258
Blank
2148
2614
14-1
3571
4104
14-2
46480
6181
16-1
3475
3629
16-2
17430
4565
NB! Later it was discovered that the RFP that we have in the vesicles (from biobrick Part:BBa_E1010 has an exitation/emission peak at 584/607nm.
Colony PCR of GFP
We transferred around 15 colonies to 20µl dH2O and incubated in the PCR-machine at 95°C for 15 min. Sentrifuged for 2 min 14.000rpm. 1µl was used as template for PCR. Gel electroforeses confirm a band around 750bp which is the expected size of the GFP. DNA purification was performed using a putification kit and the concentration was measured using Nanodrop.
Gibson Assembly 3
A new gibson with the newly transformed GFP was performed. Insert and vector concentrations of 200ng, 10µl DNA 10µl Gibson Mastermix. 50°C, 15 min. A sample of 2µl was removed after 5 min. Product was incubated in -20°C immidiately to stop the reaction. 2µl transformed in ER2566 and plated. Control with all DNA was plated on Amp and Chl plates.
Plate
Colonies
Concentration(ng/µl)
1
57
108.1
130
109.3
3
52
29.3
4
12
19.6
5
38
99.3
6
31
25.4
7
85
23.3
8
42
24
Control Amp
95
Control Chl
34
The Concentration was measured after miniprep using nanodrop. The colonies on both controlplates indicate contamination at some stage.
A new transformation was started with RFP, BB and Ptet GFP BBa_I13522 , to check if the different GFP will work. Tat was isolated from ER2566 cells using colony PCR and product used as template in PCR with tat-primers. They were plated and incubated at 37°C overnight.
Wednesday 07.08.13
SDS-PAGE of vesicle samples
SDS-PAGE was run on the samples from the day before. As showed in figure 9 all of the samples had vesicles, but sample 14-2 had a higher density.
Figure 9: Ladder applied is Precision Plus ProteinTM Unstained Standards.
New transformation
BB, RFP, GFP and tat was transformed again to try and eliminate contamination at any possible step. The standard iGEM Transformation Protocol was used.
Another round of small-scale vesicle preparation
Step 1 and 2 of the small-scale vesicle preparation were performed with ER2566 transformed with construct from the second Gibson assembly attempt. Two colonies (see picture 8) were selected and incubated for 14 and 16 hours in step 2. The cultures were incubated aerobically in LB media with ampenicillin (100 µL/mL) in step 1 and aerobically in plain LB in step 2. The samples are again labeled as 'hours incubation in step 2 - colony number'
Confirmation PCR of Gibson Construct with Taq Polymerase
We got several bands on the gel from gibson, it is unsure if they are the right size so to confirm what product was made in the gibson assembly we ran a confirmation PCR using Taq Polymerase Taq Pol Protocol .
Different combinations of primers to detect different combinations.
Sample
Target
F-primer
R_primer
Tm
A
RFP
F_l_RFP
R_RPF
66
B
GFP+RFP
F_tat_GFP
R_RFP
66
C
Tat+GFP+RFP
F_pl.b_Tat
R_RFP
63
D
GFP
F_tat_GFP
R_l_GFP
68
E
Tat
F_pl.b_Tat
R_Tat
63
F
Tat+GFP
F_pl.b_Tat
R_GFP
63
Band was visible for RFP around 750bp, so that is confirmed. Strong band for the Tat_GFP_RFP construct, but the size is only around 850bp. For it to contain tat and the 2 FPs it should be around 1600bp. It is probable that tat has attached to the RFP instead of the GFP even though the primers are not designed so that this is possible. More tests will be run.
Thursday 08.08.13
Small-scale vesicle preparation
Step 4-12 of the small-scale vesicle preparation were performed. In addition a 10-6 dilution of all the samples  were made and plating of 100µL of 1 mL of the dilution on Amp-LA was performed in step 4. The plates were left in the incubator overnight. The plates with bacteria from colony 1 had growth (approximately 250-300 colonies) indicating that the bacteria still had their plasmid. The control plate with bacteria from colony 2 had 14 colonies (14-2) and 2 colonies (16-2).
The cellmedia with bacteria from colony 1 (14-1 and 16-1) has a pink color, while the other samples did not.
Optical density (OD) at 600 nm was mesured in step 4 as indicated in the table below. The samples were diluted 1:10 with LB media. The true OD of 14-1 and 16-1 might be disturbed by the production of RFP, giving these samples a higher OD.
Sample
OD600
14-1
0.566
14-2
0.400
16-1
0.555
16-2
0.504
Sample 16-1 was disregarded for RFU-measurements in step 12 due to an experimental mistake.
Samples for SDS-PAGE was prepared in step 12.
RFU was measured as decribed in step 12 of the small-scale vesicle preparation protcol. As the vesicles were expected to contain RFP if the tat sequence is present, RFU was also measured without FM4-64 at exitation/emission at 605/670 (the known exitation/emission for RFP). Results were as indicated in the table below:
Sample
RFU (exitation/emission at 515/640 nm) with FM4-64
RFU (exitation/emission at 584/607 nm) with FM4-64
Empty
37
18147
Blank
1736
45 927
14-1
9030
40 818
14-2
20 745
31 325
16-1
16-2
46 126
37 304
As the highest 584/607-measurement is the blank sample, we conclude that there are no RFP present in the vesicles.
PCR
Primers for CFP ( BBa_E0030 ) and YFP ( BBa_E0030 ) arrived today, so PCR was run with these primers. TO avoid any contamination, we added 1 µl of Dpn1 to the amplicon and put it in a 37 °C waterbath for an hour. We used the QIAGEN PCR purification kit to isolate the DNA and measured the concentration by NanoDrop ND-1000 Spectrophotometer .
Sample
Concentration (ng/µl)
YFP
64
CFP
92.8
Restriction cutting of construct
2 different samples of the gibson assembly product was used in restriction analysis with 3 enzymes. Two of the enzymes cut on either side of the construct and the third cuts the GFP at 323 bp, roughly in the middle of the gene. 4 different combinations were performed. We used the restriction digest protocol from iGEM webpage.
#
Enzyme1
Enzyme2
1
EcoRI
SpeI
2
EcoRI
PmlI
3
PmlI
SpeI
4
BsaAI
SpeI
FIND PIC OF GEL!
Friday 09.08.13
SDS-PAGE of vesicle samples
SDS-PAGE was run on the samples from the day before. As showed in figure 10 the 16-2 sample seems to have the highest protein content followed by 16-1. It seems that the 16 hour incubation is to prefer when considering the SDS-PAGE (figure 9) and the RFU-measurements (table, thursday 08.08.3013)
Figure 10:Ladder applied is Precision Plus ProteinTM Unstained Standards.
PCR
New PCR on GFP, RFP, BB and tat. Very weak band on BB. Ran new Expand high fidelity DNA polymerase kit with BB-PCRproduct as template. Still low concentration.
Gibson Assembly 4
In addition to the Tat_GFP_RFP construct(TGR) it was made a Tat_CFP_YFP construct(TCY). Tranformed(X5) and incubated overnight.
Saturday 10.08.2013
The colonies on the 10 plates were counted.
Plate
TCY
Colonies TGR
1
59
45
2
32
36
3
62
39
4
49
14
5
100
20
The plates with the most colonies from both constructs(TCY:plate 5, TGR: plate 1) were used for colony PCR, appr. 20 colonies from each plate was transferred to 20 µl of dH2O, and incubated in the PCR for 15 min, at 95 °C. One colony from the same plates was incubated in liquid media and left overnight for miniprep the next day.
The results were disappointing again and confirmation PCR did not give the desired sizes.
Monday 12.08.2013
Gibson assembly with different construct
The experiments using tat_2xFPs_BB have been unsuccessful at this point according to our confirmation PCRs. We are therefor trying to make a 2 piece construct instead; Tat_GFPmut3* and Tat_RFP. Gibson Assembly was performed as described in the protocol page .
Template
Concentration(ng/µl)
Volume in GA(µl)
Tat
36
3
GFPmut3*
58.1
2
BB
39
5
Add 10 µl of gibson mastermix. Incubate for 15 min at 95 °C.
Template
Concentration(ng/µl)
Volume in GA(µl)
Tat
36
3
RFP
140
1
BB
39
5
Add 10 µl dH2O and 10 µl of gibson mastermix. Incubate for 15 min at 95 °C. Then both were transformed in DH5α cells.
Improving concentration of BB
The BB was run with Expand high fidelity polymerase Protocol to get a higher concentration, so tht it will be better for future gibson assemblies. The annealing temperatur was set to 65 °C.
Reactant
µl
Buffer 2
2
dNTP
0.5
Primers
0.5
Template
1
Polymerase
0.25
dH2O
15.25
This gave an even weaker result, which is unexpected. Our next move is to use colonies of the backbone and incubate in liquid media for new miniprep and hopefully a better yield. As we are running short on purified tat that will also be incubated and run on the PCR again.
PCR with Protein G
Our Primers for protein G has arrived, as has the 5 different ''Streptococcus dysgalactiae ssp equisimilis'' samples collected from St. Olavs hospital. The strains of the bacteria samples are unknown as they are collected from patients. There are 3 different  forward primers, one has the tat sequence overhang (F_tat_PrG), one has the natural sequence that the protein has in ''Streptococcus dysgalactiae ssp. equisimilis'' (F_pl.b_PrG) and one has the pelB sequence overhang (R_pl.b_pelB_PrG). There are two different reverse primers, one that will anneal to the end of the DNA sequence for the gene (R_pl.b_PrG) and another that will anneal to the end of the protein coding sequence (see registered gene sequence ). They will be run in 6 different combinations:
Sample
F_Primer
R_Primer
Tm
A
F_pl.b_PrG
R_pl.b_PrG
61
B
F_pl.b_PrG
R_pl.b_PrGstop
64
C
F_tat_PrG
R_pl.b_PrG
63
D
F_tat_PrG
R_pl.b_PrGstop
68
E
F_pl.b_pelB_PrG
R_pl.b_PrG
63
F
F_pl.b_pelB_PrG
R_pl.b_PrGstop
As we have 5 different ''Streptococcus dysgalactiae ssp. equisimilis'' samples, 30 different PCR-reactions were run.
Standard conditions for Phusion PCR was used. The products was run on gel for appr. 1h. See figure below
Figure 11.
A very clear band with the  size of around 1200-1300 bp was found in sample D1. This is a smaller DNA fragment compared to the known gene sequence of about 1700 bp. Since the size of Protein G is known to vary among the different strains, we can still conclude that our PCR-product is Protein G. The PCR product with the genomic DNA sample 1 as template and F_tat_PrG and R_pl.b_PrGstop as primers is most likely Protein G.
Tuesday 13.08.2013
Restriction digest of BB
To remove any contaminants in our BB sample we are running a restriction digest with only one enzyme, HindIII, and cutting the correct band from the gel. The band was purified with the QIAquick PCR purification kit .
PEC
To link two PCR products together we used a tecnique called PEC-Polymerase extension cloning. This method uses the established overhangs of the fragments to link them toghether in the PCR using only the polymerase(phusion). And then after some 10 cycles the program is paused and primers for the whole construct is added to amplify that.   The first experiment using SLIC: To link toghether tat_GFP and tat_GFPmut3*. A two-step PCR program was setup with 10 cycles in the first step and 20 cycles in the second step. For the first step the flanking(overlapping) sequences were used to determine annealing temperature = 72 °C, and for the second step only the primers without the flanking sequence were used to calculate anealingtemperature = 62 °C. Other than that standard PCR conditions using phusion polymerase was used. Se figure below for results
Figure 12: Agarose Electrophoresis gel with (from the left) tat_GFP, tat_GFPmut, tat_GFP_l_RFP in plasmid (SLIC) and tat_CFP_l_YFP in plasmid (SLIC).
CPEC-circular polymerase extension cloning is the same principle as PEC, only circular. Du to this there is no need for primers, only that the overhans all fit to each other. We attempted this method with our to FP constructs. Tat_GFP_RFP and Tat_CFP_YFP.
Reactant
Volume(µl)
Concentration(ng/µl)
HF-buffer
4
dNTP
1.6
Phusion polymerase
0.2
Vector BB
4
50
GFP
1
88.4
RFP
0.5
140
Tat
1.7
36.5
CFP
1
92.8
YFP
1
64
dH2O
up tp 20
One reaction with Tat_GFP_RFP_BB and one with Tat_CFP_YFP_BB. (I dont have the PCR conditions used here!!ANYONE??).
Wednesday 14.08.2013
PCR with TCY construct
Standars phusion polymerase protocol with a 20 µl reaction. (FIND PIC)
PCR to link tat and GFP
As the PEC reaction did not confirm a link, were trying the same using PCR. Phusion polymerase 20 µl reaction. Annealing temperature 72/62 in 2-step PCR.(FIND PIC).
Restriction digest of tGR and tCY product from CPEC(13.08.13
The standard iGEM [http://parts.igem.org/Help:Protocols/Restriction_Digest Restriction Digest Protocol] was used applied with SpeI enzyme on the tGR and tCY products from yesterday (13.08.13). The digested samples were then run on a gel together with a previous PCR-product of BB (see figure below) that served as a referance.
Figure 13:From the left; tGR, tCY and BB (PCR product)
As indicated in the picture, there does not seem to be any tGR or tCY construct as the restricted products are the same size as the BB when they should be longer. There also seems to be a bit of contamination in the BB sample.
PEC to connect tat to Protein G
We applied PEC to attach tat to Protein G. The product was then purified by Gibson AssemblyTM Cloning Kit and then run on a agaorse gel together with a PCR product of Protein G without the tat attached (see figure below)
Figure 14:From the left; Protein G with tat attached and Protein G without tat.<
The PEC product is clearly a little bit longer then the Protein G PCR product without tat attched. This indicates that PEC was successful.
Thursday 15.08.2013
Purify BB by running it on gel
To remove any contaminants from the BB sample we ran 50 µl og BB sample on a gel, cut it and purified it using QIAquick PCR Purification kit .
PEC to connect tat to Protein G
The same conditions as last time were used. BB: 4 µl and tat: 3µl. Phusion 20 µl reaction.
Step
Temperature(°C)
Time(sec)
Initial denaturation
98
30
Denaturation
98
15
Annealing
52
30
Elongation
72
110
Final Elongation
72
5 min
The product was transformed into DH5α cells and ER2566 cells.
Direct transformation(DT) of BB and Tat_PrG
1:1 Ratio, BB: 2 µl, Tat_PrG: 1.5 µl. Directly transferred into DH5α cell and ER2566 cells
SLIC with BB and Tat_PrG
Reactant
Volume
NEB Buffer
1
BSA
1
T4 polymerase
0.2
BB vector
2.1
Tat_PrG
3.5
dH2O
2.2
Incubated in room tempreature for 2.5 minutes, then 10 min on ice and transformed into DH5α cell and ER2566 cells.
These are the transformations done today:
Sample
Method
Cells
1
CPEC
DH5α
2
CPEC
ER2566
3
DT
DH5α
4
DT
ER2566
5
SLIC
DH5α
6
SLIC
ER2566
7
Control BB
DH5α
8
Control Tat_PrG
DH5α
New primers and PCR
New primers arrived for BB, tat and YFP. The new primers (except YFP primer) has overhengs where the equivalent primers had not. A new reverse primer for YFP was designed because the old one lacked the terminator (TAATAA) sequence New PCR was run with standard phusion protocol 50 µl reaction.(FIND PIC)
Primer name
Target
R_GFP_tat
tat
R_CFP_tat
tat
F_GFP_pl.b
BB or pl.b
R_CFP_pl.b
BB or pl.b
R_pl.b_YFPnew
YFP
Friday 16.08.2013
Redoing PCR form yesterday
Gibson with new TGR overhangs
Reactant
volume
BB
2.5
Tat
0.3
GFP
0.7
RFP
0.45
Gibson mastermix
15
The products were transformed and plated. A control for tat and BB was also plated.
SLIC with new overhangs
Reactant
volume
BB vector
4.3
Tat
0.3
GFP
1.4
RFP
0.9
Transformed and plated.
PEC with new overhangs
Reactant
Volume(µl)
Concentration(ng/µl)
HF-buffer
4
dNTP
1.6
Phusion polymerase
0.2
Tat
1
GFP
1.5
RFP
1
dH2O
9.3
(this one gave no construct)
Confirmation PCR on gibson
It was done a confirmation PCR on the remaining sample from gibson(before transformation) using F_pl.b_tat and R-pl.b_RFP with Tm=72 °. (FIND PIC)
Saturday 17.08.2013
Minipreping of Protein G samples
Sample Marking Sample Marking ProteinG 1 C (1) Protein 10 DT (3) ProteinG 2 S (1) Protein 11 C (4) ProteinG 3 DT (1) ProteinG 12 DT (4) ProteinG 4 C (2) ProteinG 13 C (5) ProteinG 5 S (2) ProteinG 14 DT (6) ProteinG 6 DT (2) ProteinG 15 DT (7) ProteinG 7 C (3) ProteinG 16 DT (8) ProteinG 8 S (3) ProteinG 17 DT (9) ProteinG 9 DT (3) ProteinG 18 DT (10)
Confirmation PCR
INGRID&ELLEN: I think this is where you guys did a mistake when you though you were doing conf. PCR on tCY/tGR but infact it was ProteinG. The correct proteinG pcr was run on sunday===  On Gibson TGR 1+2 and TCY, tat_PrG CPEC Samples 1-5, tat_PrG SLIC Samples 1-3, tat_PrG DT Samples 1-10. Ran on gel (FIND PIC)  Colonies from plates with TGR, TCY Gibson and SLIC were transferred to liquid media.  EVERYTHING THAT WAS DONE DURING THIS WEEKEND IS IN SOMEONE ELSES LABJOURNAL :) FILL IN PLEASE.. Im pretty sure we did some things!
Sunday 18.08.2013
Miniprep
The Gibson and SLIC with tGR and tCY that were transerred to liquid media yesterday were minipreped.
Conformation PCR on tat_ProteinG
We did a confirmation PCR of all the 18 tat_ProteinG plasmids that were minipreped the day before (17.08.2013). Them we ran the PCR product on a gel (see figure below).
Figure 15: First row contains tat_ProteinG samples 1-9, second row contains tat_ProteinG samples 10-18 (see table from saturday 17.08.2013).
Thursday 22.08.2013
Samples (see table below) were prepared and sent for sequencing. The DNA fragments (tat_GFP and tat_GFPmut3) were sent with their respectful reverse primer and the plasmids (the rest of the samples) were sent with the standard iGEM forward sequencing primer. The samples were made with the concentration between 80-100 ng/µl. The samples with to low concentration was evaporated to get higher concentration.
Sample#
Sample name
1
tat_GFP
2
tat_GFPmut3
3
SYFP
4
BFP
5
ProteinG (DT8)
6
ProteinG (S3)
7
tGR 1 (minipreped 18.08.13)
8
tGR 2 (minipreped 18.08.13)
9
ER1 (see figure 8)
10
ER2 (see figure 8)
Friday 23.08.2013
Red ER1-cells (see figure 8) was first immobilized in agar and them viewed in a confocal microscope for seeing if the RFP was localized in the periplasm. The results can be seen in the two figures below:
There are no indication that the RFP is more concentrated in the periplasm.
Saturday 24.08.2013
PEC
We did a PEC reaction with:
1. Tat_GFP + RFP, Tm= 72/72 °C (F_pl.b_tat + R_pl.b_RFP)
2. Tat + CFP, Tm= 58/72 °C (F_pl._tat + R_l_CFP)
Same procedure as last time, 20 µl reaction, 1 µl primers added after 10 cycles. (FIND PIC)
Confirmation PCR
Sample
Target
1
Gibson TGR 1
2
Gibson TGR 2
3
SLIC TGR 1
4
SLIC TGR 2
5
PEC TGR
6
PEC Tat_CFP
7
Control CFP
(FIND PIC)
Tuesday 27.08.2013
Today we got the sequencing results on the plasmids/DNA fragments we sent last week. The figures below show the alignments of the samples (always first line) and the expected sequences (always second line). Red indicates proper alignment, blue indicates mismatches.
Figure 1: Alignment of the tat_GFP DNA fragment. The GFP part of the sequence is correct, but we do not know is the tat sequence is correct as this was not included in the sequencing results.
Figure:  Alignment of the tat_GFPmut3 DNA fragment. The GFPmut3 part of the sequence is correct, but we do not know is the tat sequence is correct as this was not included in the sequencing results.
Figure:Alignment of SYFP. The SYFP biobrick has the correct sequence and it is highly unknown why our primers did not work in PCR amplification of the gene..
Figure:Alignment of SYFP. The BFP biobrick has the correct sequence and it is highly unknown why our primers did not work in PCR amplification of the gene.
Figure: Alignment of tat_ProteinG DT8. There is one segment missing in our ProteinG, as is expected as the ProteinG variant that we are dealing with is a shorter than the [http://www.ncbi.nlm.nih.gov/nuccore/X06173 registered gene sequence]. There is how ever an addition of a guanin residue in the beginning of the tat-sequence, which will cause a frameshift resulting in a non-functional protein product.
Figure: Alignment of tat_ProteinG DT8. There is one segment missing in our ProteinG, as is expected as the ProteinG variant that we are dealing with is a shorter than the registered gene sequence . There is how ever an deletion of a guanin residue in the beginning of the tat-sequence, which will cause a frameshift resulting in a non-functional protein product.
Figure: Alignment of tat_GFP_l_RFP (tGR 1). The aligment is just nonsense. This plasmid does not have the intended sequence.
Figure: Alingment of tat_GFP_l_RFP (tGR 2). The aligment is just nonsense. This plasmid does not have the intended sequence.
>
Figure: Alignment of tat_GFP_l_RFP (ER1). The sequence align almost perfectly. There seems to be some sort of extra insert at the linker region, but this insert is dividable by 3, so the reading frame is maintained. This is supported by the fact that the colonies with this construct is red (see figure 8).
Figure: Alignment of tat_GFP_l_RFP (ER2). The aligment is just nonsense. This plasmid does not have the intended sequence.
Thursday 27.08.2013
Preparing Protein G and tCY construct for sequencing
All the rest of the samples with tat_protein G that had a positive result on the PCR confrmation (see figure on sunday 18.08.2013) was prepared and sent for sequencing. The samples were made with the concentration between 80-100 ng/µl. The samples with to low concentration was evaporated to get higher concentration.
Sample
Name
Seq.ID
1
ProteinG (S2)
17|J83
2
ProteinG (C4)
17|J84
3
ProteinG (C5)
17|J85
4
ProteinG (DT2)
17|J86
5
ProteinG (DT3)
17|J87
6
ProteinG (DT5)
17|J89
7
ProteinG (DT6)
17|J88
8
ProteinG (DT7)
17|J90
9
ProteinG (DT9)
17|J91
10
ProteinG (DT10)
17|J92
11
GIBSON TCY1
17|J93
12
GIBSON TCY2
17|J94
13
GIBSON TCY3
17|J95
14
SLIC TCY
17|J96
Thursday 29.08.2013
Confirmation PCR on sequenced construct
The sequencing confirmed our construct: Tat_GFP_RFP. It was confirmed in one of our previous Gobson assemblies by sequencing. We therefore wanted to run a PCR to confirm the results since we know that the construct is there.  The PCR was run with F_pl.b_tat and R_pl.b_RFP and the PCR products were run on a gel (see figure below):
Figure:Gel of conformation PCR on ER1 (tGR) construct.
There is a distinct band below 1000 bp that we are not certain what is. There is also a visible band at around 1700 bp, which is the expected size of the tGR construct.
Freezing of tat_GFP_RFP construct
Bacteria with the tat_GFP_RFP construct that had a positive sequencing result (the ER1 plasmid) was prepared for freezing.  5 cryotubes: 800 µl of cell media(overnight incubated colonies of construct)              400 µl of 60% glycerol  Stored in -80 freezer(Rack 6, Shelf 7: pink box marked iGEM).
Retrieved from " http://2013.igem.org/Team:NTNU-Trondheim/Notebook/August "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
