<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kyoto_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Kyoto/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kyoto/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kyoto/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kyoto/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="kyoto-sub"><DIV id="kyoto-main"><DIV id="kyoto-content"><DIV id="kyoto-tab-contents"><DIV id="kyoto-tab-Florigen" class="displayOn"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#"><SPAN class="tocnumber">1</SPAN></A></LI><LI class="toclevel-2"><A href="#Realizing_Flower_Fairy_in_real_world"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Realizing Flower Fairy in real world</SPAN></A></LI><LI class="toclevel-2"><A href="#Our_Goal_is_to_induce_flower_formation_just_by_putting_Flower_Fairy_E.coli_on_leaves.21"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Our Goal is to induce flower formation just by putting Flower Fairy E.coli on leaves!</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#_2"><SPAN class="tocnumber">2</SPAN></A></LI><LI class="toclevel-2"><A href="#1.EXPRESSION"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">1.EXPRESSION</SPAN></A></LI><LI class="toclevel-3"><A href="#Modifying_FT_gene_for_Biobrick"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Modifying FT gene for Biobrick</SPAN></A></LI><LI class="toclevel-3"><A href="#Confirming_expression_of_FT"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Confirming expression of FT</SPAN></A></LI><LI class="toclevel-2"><A href="#2.SECRETION"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">2.SECRETION</SPAN></A></LI><LI class="toclevel-3"><A href="#Proteins_are_required_to_go_through_two_membranes"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Proteins are required to go through two membranes</SPAN></A></LI><LI class="toclevel-3"><A href="#TorA_signal_enables_proteins_to_go_through_the_inner_membrane_via_Tat_pathway"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">TorA signal enables proteins to go through the inner membrane via Tat pathway</SPAN></A></LI><LI class="toclevel-3"><A href="#We_improved_usability_of_TorA_signal_BioBrick"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">We improved usability of TorA signal BioBrick</SPAN></A></LI><LI class="toclevel-3"><A href="#Kil_protein_enables_proteins_to_go_through_outer_membrane"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">Kil protein enables proteins to go through outer membrane</SPAN></A></LI><LI class="toclevel-3"><A href="#Kil_protein_had_no_significant_effect_on_E.coli.27s_growth"><SPAN class="tocnumber">2.2.5</SPAN><SPAN class="toctext">Kil protein had no significant effect on E.coli's growth</SPAN></A></LI><LI class="toclevel-3"><A href="#Improvement_of_secretion_system"><SPAN class="tocnumber">2.2.6</SPAN><SPAN class="toctext">Improvement of secretion system</SPAN></A></LI><LI class="toclevel-2"><A href="#3.PENETRATION"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">3.PENETRATION</SPAN></A></LI><LI class="toclevel-3"><A href="#R9_peptide_enables_FT_protein_penetrate_membranes_by_endocytosis"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">R9 peptide enables FT protein penetrate membranes by endocytosis</SPAN></A></LI><LI class="toclevel-3"><A href="#Given_the_R9_system.2C_it_induces_endocytosis_whether_R9_peptides_and_target_proteins_are_fused_or_not."><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Given the R9 system, it induces endocytosis whether R9 peptides and target proteins are fused or not.</SPAN></A></LI><LI class="toclevel-3"><A href="#We_used_R9_and_GFP_separately_and_to_check_the_R9_peptides_function"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">We used R9 and GFP separately and to check the R9 peptides function</SPAN></A></LI><LI class="toclevel-2"><A href="#4._Activation"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">4. Activation</SPAN></A></LI><LI class="toclevel-3"><A href="#The_detail_function_of_FT"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">The detail function of FT</SPAN></A></LI><LI class="toclevel-3"><A href="#Injecting_FT_and_verifying_its_function_by_RT-PCR"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Injecting FT and verifying its function by RT-PCR</SPAN></A></LI><LI class="toclevel-3"><A href="#Establishing_an_effective_way_of_RNA_purification"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">Establishing an effective way of RNA purification</SPAN></A></LI><LI class="toclevel-3"><A href="#Conducting_RT-PCR_and_qPCR_again_by_using_new_RNA_purification"><SPAN class="tocnumber">2.4.4</SPAN><SPAN class="toctext">Conducting RT-PCR and qPCR again by using new RNA purification</SPAN></A></LI><LI class="toclevel-3"><A href="#Checking_FT_protein_purification_by_Western_Blotting"><SPAN class="tocnumber">2.4.5</SPAN><SPAN class="toctext">Checking FT protein purification by Western Blotting</SPAN></A></LI><LI class="toclevel-2"><A href="#Achievement"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Achievement</SPAN></A></LI><LI class="toclevel-1"><A href="#_3"><SPAN class="tocnumber">3</SPAN></A></LI><LI class="toclevel-2"><A href="#Future_Works"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Future Works</SPAN></A></LI><LI class="toclevel-2"><A href="#Biosafety"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI><LI class="toclevel-1"><A href="#_4"><SPAN class="tocnumber">4</SPAN></A></LI><LI class="toclevel-1"><A href="#_5"><SPAN class="tocnumber">5</SPAN></A></LI><LI class="toclevel-2"><A href="#What.27s_Golden_Gate_Assembly"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">What's Golden Gate Assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#_6"><SPAN class="tocnumber">6</SPAN></A></LI><LI class="toclevel-2"><A href="#Plasmid_backbone_BBa_K797013"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Plasmid backbone BBa_K797013</SPAN></A></LI><LI class="toclevel-2"><A href="#Golden_Pass"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Golden Pass</SPAN></A></LI><LI class="toclevel-3"><A href="#How_to_use"><SPAN class="tocnumber">6.2.1</SPAN><SPAN class="toctext">How to use</SPAN></A></LI><LI class="toclevel-1"><A href="#_7"><SPAN class="tocnumber">7</SPAN></A></LI><LI class="toclevel-1"><A href="#_8"><SPAN class="tocnumber">8</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Realizing_Flower_Fairy_in_real_world">Realizing Flower Fairy in real world</SPAN></H3><P>Have you ever seen flower fairies? Probably the answer is &quot;NO&quot;, (though some of you might have come across them in your childhood,) because they are imaginary creatures which exist only in fairy tales. Don’t you think it would be wonderful if you could live with flower fairies? In addition, their lovely power to make flowers bloom would be profitable for us in many ways, such as application to agriculture. This is why we have set our project for realizing Flower Fairy E.coli with synthetic biology!!
In order to realize our dream, we focused on <B>FT protein</B>, known as Florigen.This protein is a kind of plant hormones. First of all, FT protein is produced in leaves, and then move to the shoot apex and bloom flowers. Therefore, FT protein is the key to our project.
</P><H3><SPAN class="mw-headline" id="Our_Goal_is_to_induce_flower_formation_just_by_putting_Flower_Fairy_E.coli_on_leaves.21">Our Goal is to induce flower formation just by putting Flower Fairy E.coli on leaves!</SPAN></H3><P>When you want to use our Flower Fairy E.coli, all you have to do is just put them on plant leaves! When you spread Flower Fairy E.coli, FT protein is secreted and penetrate the cell membrane of a plant, and the plant starts blooming.</P><P>We had to go through four steps in order to achieve our goal――Flower Fairy E.coli.
These four steps are <A href="#Expression">“<B>EXPRESSION</B>”</A>,<A href="#Secretion">”<B>SECRETION</B>”</A>,
<A href="#Penetration">”<B>PENETRATION</B>”</A>, and <A href="#Activation">”<B>ACTIVATION</B>”</A></P><P>
In each step, we have some problems to be attacked.<A href="#Expression">“<B>EXPRESSION</B>”</A>; It is unclear whether <I>E.coli</I> can express FT protein, because FT protein is derived from plant cells .<A href="#Secretion">”<B>SECRETION</B>”</A>; After produced, ''E.coli'' have to secrete FT protein outside of themselves.<A href="#Penetration">”<B>PENETRATION</B>”</A>; FT protein has to penetrate into plant cells.<A href="#Activation">”<B>ACTIVATION</B>”</A>; Even if FT protein could get into the cells, it is unclear whether FT protein produced by <I>E.coli</I> can activate genes in shoot apex cells and induce flower formation.
</P><H2><SPAN class="mw-headline" id="1.EXPRESSION">1.EXPRESSION</SPAN></H2><P>The first step is <B>EXPRESSION</B>. We had to make <I>E.coli</I> produce FT protein.
</P><P>However, in the natural environment, wild<I>E.coli</I> don't have <I>FT</I> gene. Therefore we tried to make a new BioBrick part including <I>FT</I> gene and introduce it into <I>E.coli</I>.
</P><H3><SPAN class="mw-headline" id="Modifying_FT_gene_for_Biobrick"> Modifying <I>FT</I> gene for Biobrick </SPAN></H3><P><I>FT</I> gene is derived from <I>Arabidopsis thaliana</I>, a model plant. Professor Araki in Kyoto University kindly gave us <I>FT</I> cDNA in TOPO blunt end 2(Invitrogen). <I>FT</I> cDNA has two cleavage sites of iGEM restriction enzymes, EcoR1 and Pst1 (Fig.1-1 A), therefore we modified <I>FT</I> gene sequence by Inverse PCR with primers containing two base mismatches between primer and cDNA(Fig.1-1 B). As a result, we could get mutated FT cDNA, which are not cleaved by iGEM restriction enzymes, and then we added prefix and suffix to <I>FT</I>. Finally, we could make new BioBrick parts of <I>FT</I> (Fig.1-1 C). 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Fig.1-1A : Necessity for mutation of <I>FT</I> cDNA. <I>FT</I> sequence had two cleavage sites of iGEM restriction enzymes.B : Inverse PCR Method Inverse PCR is a measure of mutating <I>FT</I> gene segment. Inverse direction of primers and designing those including mismatch residue leads to mutation of the original plasmid.C : Standardization of <I>FT</I> cDNA. We had mutated and added prefix and suffix to <I>FT</I>.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Confirming_expression_of_FT"> Confirming expression of FT </SPAN></H3><P>We constructed the plasmid shown in the Fig.1-2. One was <I>FT</I> gene with T7 promoter
(<A href="http://partsregistry.org/Part:BBa_I719005" class="external text" rel="nofollow">BBa_I719005</A>) and 6His tag, and the other was <I>FT</I> gene only with T7 promoter. T7 promoter is a strong promotor regulated by T7 polymerase. In our strain of <I>E.coli</I>, the expression of T7 polymerase can be induced by IPTG. 6His tag, which is used in later steps, enabled us to purify FT protein from <I>E.coli</I> using affinity chromatography.
</P><P>We needed to confirm that E. coli can express FT protein, because there was a possibility that <I>E.coli</I> can not translate FT or FT is unstable or toxic in <I>E.coli</I>. 
In order to confirm the expression of FT protein, we performed Western blotting using anti-FT antibody.
As a result,　we observed FT and 6 His:FT bands　at the expected molecular weight region(Fig.1-2).
So we successfuly confirmed the FT expression!
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Fig.1-2 Construction of <I>FT</I> generator and the results of Western blotting against FT and 6 His:FT protein Expression of T7 polymerase is regulated by IPTG. In both of constructions, left lane shows the band NOT induced by IPTG and right lane shows the band induced by IPTG.</DIV></DIV></DIV><P><B> We succeeded in  confirming the mutation and the expression of <I>FT</I> and 6 His:FT protein in <I>E.coli!</I></B></P><H2><SPAN class="mw-headline" id="2.SECRETION">2.SECRETION</SPAN></H2><P>The second step is <B>SECRETION</B><I>. </I>We want<I> E.coli </I>to secrete FT protein to outside of the cell.
Now our <I>E.coli</I> can produce FT protein, but a big issue still remains: 
how they can transport proteins to the outside of the cells? Many people might think cell lysis is the best way. 
But in our project lysis is not very good, because it would possibly cause all fairies' deaths. We hoped to see the continuous effect of Flower Fairy E.coli, not just temporary. So, we adopted a method other than lysis which enables <I>E.coli</I> to transport FT protein continuously. </P><H4><SPAN class="mw-headline" id="Proteins_are_required_to_go_through_two_membranes">Proteins are required to go through two membranes</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fig.2-1 FT protein have two hurdles in order to get out of the cell</DIV></DIV></DIV><P><I>E.coli</I> have two membranes: inner membrane and outer membrane. To transport FT protein to outside of <I>E.coli</I>, FT protein has to pass through the two membranes. So we searched for secretion systems to make FT protein go through these membranes. 
</P><H4><SPAN class="mw-headline" id="TorA_signal_enables_proteins_to_go_through_the_inner_membrane_via_Tat_pathway">TorA signal enables proteins to go through the inner membrane via Tat pathway</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig.2-2 FT protein with TorA signal go through the inner membrane to the periplasm.</DIV></DIV></DIV><P>Wild <I>E.coli</I> have many secretion systems. In order to enable proteins to go through inner membrane, we decided to use one of these systems, called Twin Arginate Translocation (Tat) pathway. Tat pathway is more suitable than other secretion systems for our project, because by Tat pathway <I>E.coli</I> secrete proteins into the periplasm keeping proteins’ conformation and function. The following is  the mechanism of Tat pathway; a Tat transporter recognizes TorA signal, and proteins having a TorA signal at their N terminals only can get into the periplasm. </P><H4><SPAN class="mw-headline" id="We_improved_usability_of_TorA_signal_BioBrick">We improved usability of TorA signal BioBrick</SPAN></H4><P>We tried to combine TorA signal with FT protein. Searching previous iGEM projects, we actually found TorA signals as iGEM parts (such as <A href="http://partsregistry.org/Part:BBa_K638402" class="external text" rel="nofollow">BBa_K638402</A>) submitted by other teams. These parts, however, have two big problems. One problem is that these parts do not have RBS. So iGEMers who use the parts have to spend additional processing time. The other problem is that stop codon appear between signal region and target coding sequence when these parts are combined with some other parts by standard or 3A assembly. In short, when iGEMers use these parts, TorA-fusion protein would not be expressed at all, or only TorA would be expressed. </P><P>To solve these problems, we produced new applicable TorA signal <A href="http://partsregistry.org/Part:BBa_K797002" class="external text" rel="nofollow">BBa_K797002</A> (Fig.2-2). Our part has two improvements. <A href="http://partsregistry.org/Part:BBa_K797002" class="external text" rel="nofollow">BBa_K797002</A> contains RBS and indels to prevent the emergence of stop codon between signal region and target cording sequence. We sequenced <A href="http://partsregistry.org/Part:BBa_K797002" class="external text" rel="nofollow">BBa_K797002</A> and confirmed that stop codon did not appear when we used it in Standard and 3A assembly. Using green fluorescent protein (GFP) as a target protein, we constructed plasmid of plac-RBS-TorA-GFP-DT and introduced it into E.coli. After that, we observed the TorA-GFP-fusion-expressing cells (Fig.2-3) suggesting that the TorA-GFP fusion was successfully expressed. This meant RBS in our TorA signal worked well and stop codon didn't appear after assembly. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.2-3 Previous TorA signal BioBricks (such as BBa_K638402) and our modified TorA signal (BBa_K797002)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.2-4 <I>E.coli</I> (left) sample stained with Hoechst and observed with 352nm wavelength. Hoechst can stain DNA. <I>E.coli</I> (right) sample having plac-TorA-GFP-DT and observed with 489nm wavelength.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Kil_protein_enables_proteins_to_go_through_outer_membrane">Kil protein enables proteins to go through outer membrane</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig.2-5 Kil protein makes holes on outer membrane.</DIV></DIV></DIV><P>With Tat secretion pathway, FT protein can be transported into the periplasm. Next, FT protein needs to move to the outside of E.coli. For secretion, we used kil protein, which is derived from λ phage. Kil protein makes holes in the outer membrane of <I>E.coli</I>. So in our project, we introduced <I>kil</I> gene into <I>E.coli</I>. But the function of outer membrane is essential for <I>E.coli</I> to survive. Overexpression of <I>kil</I> gene, therefore, causes cell death. For this reason, we must check whether <I>kil</I> gene is harmful or not under our condition. </P><H4><SPAN class="mw-headline" id="Kil_protein_had_no_significant_effect_on_E.coli.27s_growth">Kil protein had no significant effect on <I>E.coli'</I>s growth</SPAN></H4><P>We made the construction plac-RBS-kil-double terminator, whose backbone is pSB3C5. After culturing for 18hr at 37℃, we removed the supernatant, and diluted it to OD600=0.1. Then we resuspend it. And then, we added 0/0.001/0.01/0.1/1mM IPTG to each. While culturing again at 37℃, we measured OD600, which indicate the density of <I>E.coli</I>. The figure below shows the results. These results indicated that the expression of plac-RBS-kil-DT (pSB3C5) makes no effect on the survival of <I>E.coli</I>. 
</P><P><B>We established the new biobrick for the useful secretion system. When we apply these systems, torA signal and kil protein, to Flower Fairy E.coli, we can make <I>E.coli</I> secrete FT protein without cell lysis!!(Fig.2-7)</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:523px;"><DIV class="thumbcaption">Fig.2-6 The result of evaluation of kil protein.  Vertical axis means the value of OD 600. And horizontal axis means incubation time. Time caused changes of cell growth after IPTG induction.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.2-7 Overall vision of Tat secretion system and kil protein. We apply this system for Flower Fairy E.coli.   1.Tat transporters are piercing in the inner membrane. They recognize TorA signal, and transport into the periplasm only proteins that have TorA signal at their N terminals. 2. Kil protein makes holes in the outer membrane of E.coli and the proteins go through the outer membrane.</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Improvement_of_secretion_system">Improvement of secretion system</SPAN></H4><P>Now we can let<I> E.coli</I> to secrete FT protein to the outside by using Tat pathway and kil protein. But <I>E.coli</I> secrete only a small amount of FT protein when they use Tat transporters which they inherently have. 
To make E.coli secrete enough amount of FT protein, we needed to improve the efficiency of their secretion systems. To reach this goal, we used two genes. One is composed of TatA, TatB, and TatC, which composes Tat transporter. Another is phage-shock protein A (pspA), which wild <I>E.coli</I> have. When their inner membrane is damaged, pspA gene is expressed ,and pspA maintains membrane potential and H<SUP>+</SUP> concentration gradient between the periplasm and cytoplasm. It is known that pspA promote trasport of proteins to the periplasm through the detail mechanisms are unknown. As the next step of secretion, by the extra induction of these genes, we tried to increase the amount of secresion.
We constructed Tat secretion cassette with constitutive promoter (BBa_K797004).(Fig.2-8) 
This part includes TatA, B and C proteins coding region and pspA. By using this part, the amount of Tat transporter is and pspA can be increased . In short, we can make <I>E.coli</I> secrete more proteins with TorA.(Fig.2-9) 
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Fig.2-8 Tat secretion cassette; Tat ABC, pspA and constitutive promoter(BBa_K797004)
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Fig.2-9 pspA helps proteins to go through Tat secretion pathway.
</P></DIV></DIV></TD></TR></TBODY></TABLE><P>
Kyoto 2012 suggests this new way of secretion, Tat pathway and kil protein, and provides iGEMers with these genes regulated by constitutive promoter. We checked the sequence of TatABC <A href="http://partsregistry.org/Part:BBa_K797000" class="external text" rel="nofollow">(BBa_K797000)</A> and the sequence of pspA <A href="http://partsregistry.org/Part:BBa_K797001" class="external text" rel="nofollow">(BBa_K797001)</A> individually, and then, we made Tat construction composed of constitutive promoter <A href="http://partsregistry.org/Part:BBa_J23107" class="external text" rel="nofollow">(BBa_J23107)</A>, TatABC <A href="http://partsregistry.org/Part:BBa_K797000" class="external text" rel="nofollow">(BBa_K797000)</A>, pspA <A href="http://partsregistry.org/Part:BBa_K797001" class="external text" rel="nofollow">(BBa_K797001)</A> and double terminator <A href="http://partsregistry.org/Part:BBa_B0015" class="external text" rel="nofollow">(BBa_B0015)</A>. We performed electrophoresis of this cassette confirmed the length of our parts and sequenced them partially.
</P><H2><SPAN class="mw-headline" id="3.PENETRATION">3.PENETRATION</SPAN></H2><P>The third step is <B>PENETRATION</B>.
In order to induce flower formation, FT protein from <I>E.coli</I> must enter into plant cells.　This is because FT protein upregulates other proteins which lead to flower formation in plant cells.
</P><P>However, normally proteins cannot penetrate a cell membrane of a plant. Therefore, we needed a method to send FT protein into plant cells.
Thanks to the advice from Doctor Washida, we found a method for the penetration of cell membrane. In this method, we use polyarginines called R9 peptides.</P><H5><SPAN class="mw-headline" id="R9_peptide_enables_FT_protein_penetrate_membranes_by_endocytosis">R9 peptide enables FT protein penetrate membranes by endocytosis</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fig.3-1 Image of R9 peptide : R9 peptide consists of 9 arginine residues.</DIV></DIV></DIV><P>R9 peptide consists of nine arginine residues(Fig.3-1). It is known as a kind of CPP (Cell Penetrating Peptide). Arginine-rich peptides induce macropinocytos, a type of endocytosis. From this, we judged that R9 peptides were suitable for cell membrane penetration.
</P><P>This is the mechanism of how R9 peptides work(Fig3-2).
</P><P>Firstly, R9 peptides adhere to a cell membrane of a plant because of their hydrophobic character.
Secondly, the cell responds to this stimulus and starts to endocytose.

Finally, proteins around the endocytosing region of the cell are taken into the cell.</P><P>
R9 peptides seem to work effectively whether or not R9 peptides are fused with target proteins. 
However, there are few examples about endocytosis of plants, so <B>we needed to check the function of R9 when used on plant cells</B>.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Fig.3-2  Mechanism of endocytosis by R9 peptide  1.R9 peptides adhere to a cell membrane of a plant because of their hydrophobic character. 2.The cell responds to this stimulus and starts to endocytose. 3.FT protein around an invaginating region of the cell are taken into the cell.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Given_the_R9_system.2C_it_induces_endocytosis_whether_R9_peptides_and_target_proteins_are_fused_or_not."> Given the R9 system, it induces endocytosis whether R9 peptides and target proteins are fused or not. </SPAN></H4><P>Considering this system, target proteins near the R9 peptides are taken into cells by endocytosis. 
So, we assumed that the system would induce endocytosis in both cases where R9 peptides and target proteins were fused and not fused.

However, as we wanted to introduce proteins as efficiently as possible, we deliberated whether we should fuse R9 and target proteins. 
We thought the shorter the distance between R9 peptides and target proteins is, the more easily they are taken into a plant cell. <B>Therefore, we expected that fusing R9 peptide and a target protein would lead to higher efficiency.</B></P><P><B>So, we tried to fuse R9 peptides and target proteins to increase penetration efficiency</B></P><P>We transformed <I>E.coli</I>, fusing R9 pepetides and target protein's gene in line. By doing this, we tried to make <I>E.coli</I> express R9::GFP fusion protein.
</P><P>In order to visualize the function of R9, we tried to prepare R9::GFP fusion protein. However, <I>E.coli</I> expressing the R9::GFP fusion protein was not effectively cultivated.(Fig3-3). Then, we checked whether R9 peptides had a bad effect on the expression of R9::GFP fusion protein by Western blotting and RT-PCR. Although we succeeded in confirming the existence of the mRNA　(Fig3-4), we did not find the proteins (Fig3-5). These results are probably caused by poor translation or quick breakdown of the protein. 
</P><P>We used the existing GFP generator part, <A href="http://partsregistry.org/Part:BBa_I746915" class="external text" rel="nofollow">BBa_I746915</A>. 
</P><TABLE class="gallery" cellspacing="0" cellpadding="0"><TBODY><TR><TD><DIV class="gallerybox" style="width: 225px;"><DIV class="gallerytext"><P>Fig.3-3 Poor growth of <I>E.coli</I> expressing R9::GFP. IPTG induction for the 4hours.Culture fluid of <I>E.coli</I> with R9::GFP fusion protein on the right hand is cloudier than that on the left.
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 225px;"><DIV class="gallerytext"><P>Fig.3-4 Western blotting for checking the expression. No band of R9::GFP fusion protein.  Lane1: Molecular marker Lane2: GFP([Part:BBa_I746915]) Cell lysate 10µL, not induced Lane3: GFP([Part:BBa_I746915]) Cell lysate 10µL, IPTG induced Lane4: R9::GFP Cell lysate 10µL, not induced Lane5: R9::GFP Cell lysate 10µL, IPTG induced
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 225px;"><DIV class="gallerytext"><P>Fig.3-5 RT-PCR of R9::GFP Lane1:100bp ladder Lane2:GAPDH (Negative control) Lane3:GAPDH(R9::GFP) Lane4:GFP (Negative control) Lane5:GFP (R9::GFP)</P></DIV></DIV></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="We_used_R9_and_GFP_separately_and_to_check_the_R9_peptides_function">We used R9 and GFP separately and to check the R9 peptides function</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption">Fig.3-6 Preparation for samples verifying R9 function 1.Cutting leaves from <I>Arabidopsis thaliana</I> 2.Removing cuticle by scratching 3.Soaking them with solutions. (Left side, control group):a solution of only GFP (Right side, experimental group):that of GFP and R9 peptide</DIV></DIV></DIV><P>It was a question whether R9 peptides work properly and introduce FT protein into plant cells.　Although endocytosis is often observed in animal cells, there are few examples of plant cell’s endocytosis. We, therefore, needed to verify the function of R9 peptides. For this reason, we performed the following experiment(Fig3-6). 
</P><P>
First, we removed the cuticles from <I>Arabidopsis thaliana'</I>s leaves by scratching them. Second, we devided the leaves into two groups, and soaked one into a solution of only GFP, and the other into that of GFP and R9. After 5 minutes we washed cells by PBS in order to wash away GFP and R9 peptides from around the leaves. After that, we contrasted leaves having soaked into the only GFP solution with ones having soaked into the solution of R9 and GFP mixture. Then we succeeded in getting the figure of GFP fluorescence (Fig3-7).
</P><P>
The control groups on the left were soaked in only GFP, and the experimental group on the right were soaked in GFP and R9. These fluorescence indicated that R9 peptides properly kept GFP in or on the plant cells. This figure strongly suggests that R9 peptides work successfully and GFP penetrate cell membrane, because this was taken by a confocal microscopy and seen as cross sections.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.3-7 Verification of the R9 peptide function with use of GFP.Cells of <I>Arabidopsis thaliana</I> leaves soaked in a solution for five minutes, and Hoechst dyeing. Left side samples are soaked in only GFP, the right side samples in GFP and R9 peptide. From top to bottom, nuclei by Hoechst , GFP fluorescence (low magnification), nuclei by Hoechst , GFP fluorescence (high) (With confocal microscope Fluoview FU10i OLYMPUS)</DIV></DIV></DIV></DIV><P><B>We can make FT protein penetrate cell membrane of plants by R9 peptide function!</B>. 
</P><H2><SPAN class="mw-headline" id="4._Activation">4. Activation</SPAN></H2><P>The final step is; <B>Activation</B>. We verified whether FT protein made by <I> E.coli</I> work normally in plant cells. 
Actually, FT protein is a transcriptional factor and we can check the ability of FT protein by observing the transcription levels of genes up-regulated by FT protein.  </P><H3><SPAN class="mw-headline" id="The_detail_function_of_FT">The detail function of FT</SPAN></H3><P>FT protein increases transcript activities of flowering factors which lead to flower formation in a plant cell.  
Let us introduce the function mechanism of FT protein. Actually, FT function is to up-regulate genes relating to bloom flowers. at the stage of flowering of plants, the expressions are incresed in these genes, FUL, SEP3, and so on. These genes work on shoot apex and change the form of shoot apex in order to start flowering. We focused on the amount of FUL and SEP3 because they are the most popular genes among them.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:612px;"><DIV class="thumbcaption">Fig.4-1 FT protein triggers some flowering genes and they induce flower formation. FT protein up-regulates some flowering genes and they induce plants to bloom.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Injecting_FT_and_verifying_its_function_by_RT-PCR">Injecting FT and verifying its function by RT-PCR</SPAN></H3><P>To evaluate the function of FT produced by our <I>E.coli</I>, we tried to check the amount of flowering factors' mRNA upregulated by FT. Experimental procedure is as follows.
1. We prepared solutions. One is composed of R9 peptides and GFP, and the other is composed of R9 peptides and FT. GFP protein is used as a control group. 2. We injected these solutions into each leaf by a syringe. 3.After incubation, we tried to perform RT-PCR in order to check up-regulation of flowering factors.</P><P>We used leaf cells of <I>Arabidopsis　thaliana</I>, insted of cells of a shoot apex. This is because shoot apex' cells are too small to observe. FT protein upregulates FUL and SEP3 genes in leaf cells, too. GFP is suitable for control experiments because  GFP's molecular weight(27kDa) is relatively similar to that of FT(20kDa.)
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:612px;"><DIV class="thumbcaption">Fig.4-2 injecting FT protein into plant leaves in order to verify the function of FT protein.</DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.4-2 6 His tag fusion FT protein was purified with Ni-NTA agarose column. 30mg  of leaves of <I>Arabidopsis thaliana</I> before bolting were used for one sample. FT or GFP protein and R9 peptide were diluted in PBS (pH7.4), 50ug/L and 500ug/uL each. Leaves were soaked into FT-R9 or GFP-R9 solution for 5min. and incubated for 16hr. in PBS(pH7.4.) After incubation, leaves were freezed with liquid nitrogen and glinded immediately. Total RNA was extracted by phenol-chloroform extraction.  cDNA was synthesized by reverse transcription and used as templates of RT-PCR.  TUBULIN was used for internal control of mRNA expression. Lane1: TUBULIN (GFP-R9 treated) amplicon 61bp Lane2: TUBULIN (FT-R9 treated)  Lane3: FUL (GFP-R9 treated) amplicon 132bp Lane4: FUL (FT-R9 treated)  Lane5: SEP3 (GFP-R9 treated) amplicon 87bp Lane6: SEP3 (FT-R9 treated)  Lane7: AP1 (GFP-R9 treated) amplicon 958bp</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:293px;"><DIV class="thumbcaption">Fig.4-3 Electrophoresis of RNA. RNA concentration is adjusted. Lane1: RNA of 16h incubated leaves(GFP-R9 treated) Lane2: RNA of 16h incubated leaves(FT-R9 treated) Lane3: RNA of fresh leaves</DIV></DIV></DIV><P>As shown above, we got no correct ampicon band, even from <I>tubulin</I>, which is a high expressed gene we used as internal control. One possible reason of this failure was the poor quality of extracted RNA in this experiment. To check this, we compared the total RNA by electrophoresis, shown in fig.4-3.
</P><P>As shown in the Fig.4-3, this time, RNA samples were degradated.
To add to this, the waveforms of them had a law peak at 260nm(Fig.4-4.)</P><H3><SPAN class="mw-headline" id="Establishing_an_effective_way_of_RNA_purification">Establishing an effective way of RNA purification</SPAN></H3><P>We found that RNA purification method needed to be improved in ordrer to perform RT-PCR successfully .
We improved following things;
1. Total leaf volume was increased.
2. Samples ware freezed with liquid nitrogen and suspended in ISOGEN more rapidly.
3. We centrifuged samples and collected supernatant twice after adding ISOGEN.
After the improvement, 260nm peak became higher and the degradation is minimized(Fig.4-4A.) 
Then we used better quality of RNA for reverse transcription and retried RT-PCR.
Fig.4-4C shows the result of this RT-PCR.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig.4-4A: the waveform of RNA purified in the previous way. it indicated that RNA purity is low B: Improved RNA waveform C: Electrophoresis of RNA. RNA concentration is adjusted. These band showed all sample were purified correctly  D: Lane1: 100bp ladder Lane2: TUBULIN Lane3: FUL Lane4: SEP3 Lane5: AP1</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Conducting_RT-PCR_and_qPCR_again_by_using_new_RNA_purification">Conducting RT-PCR and qPCR again by using new RNA purification</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">fig4-5 The result of RT-PCRLane1: TUBULIN (GFP-R9 treated) amplicon 61bp Lane2: TUBULIN (FT-R9 treated) Lane3: FUL (GFP-R9 treated) amplicon 132bp Lane4: FUL (FT-R9 treated) Lane5: SEP3 (GFP-R9 treated) amplicon 87bp Lane6: SEP3 (FT-R9 treated) Lane7: AP1 (GFP-R9 treated) amplicon 958bp</DIV></DIV></DIV>After we　purified RT-PCR via the new method, we performed RT-PCR again for SEP3 and FUL. The result is shown in fig. We were able to get sufficiently purified RNA, and amplify each gene.
<P>To confirm the exact differnece between experimental group(FT+) and negative control(FT-), we conducted qPCR and compared the relative RNA expression of each gene. However, there was not significant difference between experimental group and negative control.
</P><H3><SPAN class="mw-headline" id="Checking_FT_protein_purification_by_Western_Blotting">Checking FT protein purification by Western Blotting</SPAN></H3><P>We could not confirm FT function though enough quality of RNA was extracted. Therefore, we questioned about FT protein quality, and performed western blotting to confirm FT protein is successfully purified.
Unfortunately, the band of FT was not detected. Because of the time shortage, we could not conduct further experiment. We have to reconsider about FT purification and now we are trying.</P><H2><SPAN class="mw-headline" id="Achievement">Achievement</SPAN></H2><P>1.Expression</P><UL><LI> Mutate FT sequence
</LI><LI> Standardize FT as an iGEM part
</LI><LI> Confirm expression of FT protein in <I>E.coli</I></LI></UL><P>2.Secretion</P><UL><LI>Modify TorA signal to be easy to use the signal more
</LI><LI>Construct Tat secretion cassette that contains Constitutive promoter, RBS, TatABC, pspA, double terminator
</LI><LI>Standardize kil gene
</LI><LI>Multiply TatABC in order to strengthen Tat secretion system--<B>not yet</B></LI></UL><P>3.Penetration</P><UL><LI>Keep GFP in or around plant cells using R9 peptide
</LI><LI>Introduce FT in plant cells using R9 peptide--<B>not yet</B></LI></UL><P>4.Activation</P><UL><LI>Get high quality of RNA
</LI><LI>Amplify genes successfully
</LI><LI>Check the function of FT--<B>not yet</B></LI></UL><P>At <B>PENETRATION</B> step, GFP::R9 fusion protein connected at GFP’s N-terminal was not expressed. However, if GFP is tagged with R9 at C-terminal, that fusion protein may be successfully expressed.
This is because there is an example where proteins tagged with arginine at C- terminals were correctly expressed whereas proteins tagged at N-terminals were not(<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K249005" class="external text" rel="nofollow">BBa_K249005</A> ).
By using GFP::R9 fusion protein connected at GFP's C-terminal, we might improve the efficiency of macropinocytos, a type of endocytosis.
</P><P>
At<B> Activation</B>, in order to see whether FT is usable or not, we measured the relative RNA expression of FUL and SEP3 by qPCR. However, there was not any significant difference between experimental group and negative control.
We found that FT was not successfully purified and there is the room for improvement in that.
In addition of this, several reasons can be considered. 
The cause of failure may be that FT protein was not active or decomposed in the cell or even that FT protein didn't enter plant cells.
If we perform Western blotting of FT in plant cell after putting FT into it with R9, we will be able to understand whether the problem was FT protein or R9 peptides.
</P><P>If the cause was that our FT protein was not active though FT had entered the cell, FT will be confirmed. 
In that case, we have to consider the possibilities that FT needs Post-translational modification or Arabidopsis thaliana that we used was old.
</P><P>When FT needs Post-translational modification, we have to do more research about difference of Post-translational modification between E.coli and Arabidopsis thaliana.
</P><P>When Arabidopsis thaliana that we used was old, it might have originally expressed enough FT, so our FT might not be necessary to induce FUL and SEP3.
</P><P>If the cause was that FT didn’t enter plant cell or FT was decomposed in the cell, FT will not be confirmed. 
When FT didn’t enter plant cell, we have to reconstruct experimental system, for example, by using other kinds of R9 since there exist more effective R9s.
Wnen FT was decomposed in the cell, we have to investigate the mechanism of decomposing protein. 
</P><H3><SPAN class="mw-headline" id="Future_Works">Future Works</SPAN></H3><P>We noticed only flowering and florigen in this time but there are many other plant hormones. We made translocation pathway from <I>E.coli</I> into plant cells, so we will be able to introduce plant hormones into plant cells if <I>E.coli</I> can make them. It means we can control plant growth in any stage through genetically engineered <I>E.coli</I>. In the future that is not so far, we will be able to meddle in plants' growth――germinating, elongation, flowering, and fructification. We human will finally accomplish a technology that control plants perfectly.
</P><P>Moreover, R9 peptide functions not only plant cell. R9 peptide works on animal cell similarly. It means that we found a pathway into any kinds of cells. R9 peptide tag enables us to introduce proteins into any cells, so we will be able to control all living cells using this technology.
</P><H3><SPAN class="mw-headline" id="Biosafety">Biosafety</SPAN></H3><P>We cooperated with KAIT-Japan and the mark on the left indicates Biosafety Level of our parts.</P><P>[1]<A href="http://www.ncbi.nlm.nih.gov/pubmed/15695452" class="external text" rel="nofollow">Microsugar Chang et al.(2005) &quot;Cellular internalization of fluorescent proteins via arginine-rich intracellular delivery peptide in plant cells&quot; Plant Cell Physiol, 46(3), 482–488</A>
[2]<A href="http://www.ncbi.nlm.nih.gov/pubmed/16155177" class="external text" rel="nofollow">Paula Teper-Bamnolker and Alon Samach1.(2005) &quot;The flowering integrator FT regulates SEPALLATA3 and FRUITFULL accumulation in Arabidopsis leaves&quot; The Plant Cell, 17, 2661–2675</A>
[3]<A href="http://www.ncbi.nlm.nih.gov/pubmed/16099980" class="external text" rel="nofollow">Philip A. Wigge et al.(2005) &quot;Integration of spatial and temporal information during floral induction in Arabidopsis&quot; Science, 309(5737), 1056-1059</A>
[4]<A href="http://www.mdpi.com/1424-8247/3/4/961/htm" class="external text" rel="nofollow">Sara Trabulo et al.(2010) &quot;Cell-penetrating peptides—mechanisms of cellular uptake and generation of delivery systems&quot; Pharmaceuticals, 3, 961-993</A>
[5]<A href="http://www.ncbi.nlm.nih.gov/pubmed/15147914" class="external text" rel="nofollow">Unnamalai N, Kang BG, Lee.(2004) &quot;Cationic oligopeptide-mediated delivery of dsRNA for post-transcriptional gene silencing in plant cells&quot; FEBS Lett 21, 566(1-3), 307-10</A>
[6]<A href="http://www.ncbi.nlm.nih.gov/pubmed/22683878" class="external text" rel="nofollow">Tracy Palmer and Ben C. Berks.(2012) &quot;The twin-arginine translocation (Tat) protein export pathway&quot; Nat Rev Microbiol, 10(7), 483-96</A>
[7]<A href="http://www.ncbi.nlm.nih.gov/pubmed/14966662" class="external text" rel="nofollow">Choi JH, Lee SY.(2004) &quot;Secretory and extracellular production of recombinant proteins using Escherichia coli&quot; Appl Microbiol Biotechnol, 64(5), 625-35</A>
[8]<A href="http://www.ncbi.nlm.nih.gov/pubmed/9042754" class="external text" rel="nofollow">Miksch G, Fiedler E, Dobrowolski P, Friehs K.(1997) &quot;The kil gene of the ColE1 plasmid of Escherichia coli controlled by a growth-phase-dependent promoter mediates the secretion of a heterologous periplasmic protein during the stationary phase&quot; Arch Microbiol, 167(2-3), 143-50</A>
[9]<A href="http://www.ncbi.nlm.nih.gov/pubmed/11854367" class="external text" rel="nofollow">Seibel BA, Walsh PJ.(2002) &quot;Trimethylamine oxide accumulation in marine animals: relationship to acylglycerol storage&quot; J Exp Biol, 205(Pt 3), 297-306</A>
[10]<A href="http://www.ncbi.nlm.nih.gov/pubmed/11123687" class="external text" rel="nofollow">Thomas JD, Daniel RA, Errington J, Robinson C.(2001) &quot;Export of active green fluorescent protein to the periplasm by the twin-arginine translocase (Tat) pathway in Escherichia coli&quot; Mol Microbiol, 39(1), 47-53</A>
[11]<A href="http://www.ncbi.nlm.nih.gov/pubmed/3139642" class="external text" rel="nofollow">Suit JL, Luria SE.(1988) &quot;Expression of the kil gene of the ColE1 plasmid in Escherichia coli Kilr mutants causes release of periplasmic enzymes and of colicin without cell death&quot; J Bacteriol, 170(10), 4963-4966</A>
[12]<A href="http://www.ncbi.nlm.nih.gov/pubmed/14702305" class="external text" rel="nofollow">DeLisa MP, Lee P, Palmer T, Georgiou G.(2004) &quot;Phage shock protein PspA of Escherichia coli relieves saturation of protein export via the Tat pathway&quot; J Bacteriol, 186(2), 366-373</A>
[13]<A href="http://www.ncbi.nlm.nih.gov/pubmed/16099979" class="external text" rel="nofollow">Araki, T et al.(2005) “FD, a bZIP protein mediating signals from the floral pathway integrator FT at the shoot apex” Science 309(5737), 1052–1056</A></P></DIV><DIV id="kyoto-tab-GoldenGate"><P>BioBricks are useful for us because we can look for required BioBrick parts from their registry and recombine genes easily. When we want to introduce many parts into one plasmid, however, we have to repeat the process; restrict enzyme digestion and ligation. It takes us too much time and sometimes we lose time for other experiments.
</P><P>We want to reduce the time required for the recombination of genes and get time for verification of the expression and the effect of genes.
</P><P>Golden Gate assembly is one of the ways to make it possible.
</P><P>Some teams like 2011 WHU-China have used this assembly. But they didn't seem to spread Golden Gate Assembly through other iGEM teams. So, we created plasmid backbone parts <A href="http://partsregistry.org/Part:BBa_K797013" class="external text" rel="nofollow">&quot;BBa_K797013&quot;</A> to make it easier to use Golden Gate assembly.
</P><P>We also created software which designs primers for Golden Gate assembly.
</P><H2><SPAN class="mw-headline" id="What.27s_Golden_Gate_Assembly">What's Golden Gate Assembly</SPAN></H2><P>Golden Gate Assembly is developed by Carola Engler, Ramona Gruetzner, Romy Kandzia and Sylvestre Marillonnet.[2]
This method enables us to introduce plural gene segments into one plasmid all at once.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.1 BsaI recognition sites.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.2 irreversibe ligation</DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.3 conventional method.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig.4 Golden Gate Assembly.</DIV></DIV></DIV></DIV><P>Golden Gate Assembly takes advantage of the characteristic restrict enzyme &quot;BsaI&quot;.
Most restrict enzyme cuts its own recognition sites.
But BsaI recognizes the sequence &quot;GGTCTC&quot;(Figure 1) and cuts downstream of the recognition site as shown in the Figure 2.
And BsaI activity is independent of the sequences of the downstream of the
recognition site.
</P><P>Restrict enzyme digestion and ligation are completed by just one PCR because once DNA is cut and ligated irreversibly, the recognition site of BsaI disappears.
We create the segments which have complementary ligation sites so that we can introduce plural DNA segments into one plasmid at the same time and arrange them in the way we like. (Figure 4)
Learn more about <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0005553" class="external text" rel="nofollow">Golden Gate Assembly</A>.
</P><H2><SPAN class="mw-headline" id="Plasmid_backbone_BBa_K797013">Plasmid backbone <A href="http://partsregistry.org/Part:BBa_K797013" class="external text" rel="nofollow">BBa_K797013</A></SPAN></H2><P>Before we started　assembly, we had to make proper gene segments for Golden Gate assembly.
The DNA segments we created had four bp for ligation, one base pair spacer,
BsaI recognition sites and four base pair at the both ends.
</P><P>
We amplified plasmid backbone(psB1K3).
</P><P>After amplification, we assembled these DNA segments referring to <A href="https://2012.igem.org/Team:Kyoto/MethodAndMaterial" class="external text" rel="nofollow">the protocol</A>.(See the part of Golden Gate Assembly)
</P><P>After we confirmed this part has the restriction cites and restriction enzyme cutting sites of BsaI. After this, we mixed DpnI with enzyme-treated plasmid and conducted ligation.
Other iGEM teams can use this backbone plasmid for their Golden Gate assembly.
</P><H2><SPAN class="mw-headline" id="Golden_Pass">Golden Pass</SPAN></H2><P>We created software which design primers to create DNA segments for Golden Gate assembly.
This software gives you the sequences of primers when you input the sequences of the parts and melting temperature.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Screen Shot</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="How_to_use">How to use</SPAN></H4><OL><LI>input the sequences of the parts starting from 5' terminal
</LI><LI>set melting temperature
</LI><LI>click the &quot;calculate&quot;
</LI></OL><P><A href="http://igemkyotogoldenpass.appspot.com/" class="external text" rel="nofollow">Use the web version of GoldenPass...</A></P><P>We created the plasmid backbone <A href="http://partsregistry.org/Part:BBa_K797013" class="external text" rel="nofollow">BBa_K797013</A> and software to design primers for Golden Gate assembly.
They make it easier to use Golden Gate assembly.
We want other teams to use this parts and software to use precious time efficiently.
</P><P>[1]<A href="http://www.ncbi.nlm.nih.gov/pubmed/18985154" class="external text" rel="nofollow">Carola Engler, Romy Kandzia, Sylvestre Marillonnet.(2008) &quot;A One Pot, One Step, Precision Cloning Method with High Throughput Capability&quot; PLoS ONE 3(11), e3647.</A>
[2]<A href="http://www.ncbi.nlm.nih.gov/pubmed/19436741" class="external text" rel="nofollow">Carola Engler, Ramona Gruetzner, Romy Kandzia, Sylvestre Marillonnet.(2009) &quot;Golden Gate Shuffling: A One-Pot DNA Shuffling Method Based on Type IIs Restriction Enzymes&quot; PLoS ONE 4(5), e5553. doi:10.1371/journal.pone.0005553 doi:10.1371/journal.pone.0003647</A></P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Kyoto/Project">http://2012.igem.org/Team:Kyoto/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kyoto/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kyoto/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kyoto/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kyoto/Project&amp;oldid=268497" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>