<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Naringenin_Pathway skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Naringenin Pathway</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots/Notebook</TITLE><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/1/15/T--NewCASTLE--PATHWAYoverviewcover.png" data-natural-width="3000" data-natural-height="2200" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Naringenin Synthesis Pathway Model
                    </H1><DIV class="home-content__buttons"><A href="#Rationale" class="smoothscroll btn btn--stroke">
                        Rationale
                    </A><A href="#Background" class="smoothscroll btn btn--stroke">
                        Background
                    </A><A href="#Model_Design" class="smoothscroll btn btn--stroke">
                        Model Design
                    </A><A href="#notebook1" class="smoothscroll btn btn--stroke">
                        Preliminary Scans
                    </A><A href="#SensitivityScans" class="smoothscroll btn btn--stroke">
                      Sensitivity Scans
                    </A><A href="#RegulatoryChanges" class="smoothscroll btn btn--stroke">
                      Regulatory Changes
                    </A><A href="#RedesignedOperon" class="smoothscroll btn btn--stroke">
                      Redesigned Operon
                    </A></DIV></DIV></DIV></SECTION></DIV><SECTION id="Rationale" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Rationale &amp; Aim</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">We have chosen to use <I>Pseudomonas</I> sp. as a chassis organism due to its rapid and abundant colonisation abilities within the rhizosphere. To ensure <I>Pseudomonas</I> sp. can colonise effectively, the metabolic load and resource drain of our system on the cell must be kept to a minimum in order to conserve natural homeostasis and minimise waste. As our system utilises enzymes found in flavonoid production, our naringenin synthesis pathway will be most taxing on resources used to maintain the natural homeostasis of flavonoid pathways. Some of these resources (ATP, CoA and Malonyl CoA, for example) are included but not restricted to flavonoid production. If we are to program a cell to produce an amount of naringenin per unit time, we need to consider what range of output works best in terms of maintaining the cells homeostasis.</FONT></P><P><FONT size="3">On top of maintaining the cells homeostasis, sufficient naringenin needs to be produced by the pathway to attract the desired nitrogen fixing bacteria. The results of our <A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis" class="black">chemotaxis experiments</A> and previous research have shown higher concentrations of naringenin to inhibit growth of multiple soil microbes (1). It is therefore important that production by our system is stable in variable environmental conditions such that there are no detrimental effects on the rhizosphere community. By creating an enzymatic model of the pathway, we aimed to alter the operon design in order to minimise resource drain and stabilise naringenin production as a means of increasing system optimisation and robustness. </FONT></P><P><FONT size="3">The chemotaxis modelling and wetware element to our project characterise the production rate of naringenin required for chemotaxis to occur. When the production rates of naringenin by our endophytic chassis have been characterised, the pathway model will give us the tools to alter the operon in order to optimise production.</FONT></P></DIV></DIV><SECTION id="Background" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Background</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">The core of the model uses enzymatic turnover rates (K<SUB>cat</SUB>) to simulate the relative rate (molecules per second) of conversion from substrate to product for each enzymatic reaction in the pathway. Each reaction is split into two parts, formation of the enzyme substrate complex (Substrate binding) and conversion of the complex to the reaction products (Catalysis). The formation of the substrate enzyme complex is reversible whilst the conversion of the complex to the reaction products is not. In the case of 4CL and CHS, for which more than one substrate is required for catalysis of the product, additional steps are coded as separate reversible reactions to emulate these circumstances. </FONT></P><DIV class="row services-list block-1 block-tab-full"><DIV class="service-item" data-aos="fade-up"><DIV class="service-text"><P><FONT size="2"><CENTER>Figure 1. Naringenin Synthesis Pathway</CENTER></FONT></P></DIV></DIV></DIV></DIV></DIV></SECTION></SECTION><SECTION id="Model_Design" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Model Design</H1><H3 class="subhead">Core Enzyme Kinetics</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">Naringenin, our chemoattract of choice, is synthesised from L-tyrosine via the enzymatic action of tyrosine ammonia lyase (TAL), 4-coumarayl ligase (4CL), chalcone synthase (CHS) and chalcone isomerase (CHI) (Figure 1). These four enzymes are contained within the iGEM part <A href="http://parts.igem.org/Part:BBa_K1497017" class="black">BBa_K1497017</A>. Within this part all four enzymes are under the control of a single strong T7 promoter <A href="http://parts.igem.org/Part:BBa_I712074" class="black">(BBa_I712074)</A>, with each enzyme having the same strong RBS <A href="http://parts.igem.org/Part:BBa_B0034" class="black">(BBa_B0034)</A>. This system gives equal expression of each of the four enzymes. Previous research has identified that changing relative expression of each enzyme in the pathway improved naringenin production (2). We intended to optimise our system by using a model to explore how we could redesign the operon in order to maximise flux through the pathway whilst minimising resource consumption and waste. </FONT></P><P><FONT size="3">The model was built on Matlab using differential equations which employ mass action kinetics to describe the behaviour of reactants and products for each reaction step. Mass action kinetics describes the behaviour between a substrate and reactant as an equation where the velocity or rate of a chemical reaction is directly proportional to the concentration of the reactants. These kinetics are used for substrate binding in which substrate and enzyme diffuse towards each other to dimerise via the formation of a covalent bond, and catalysis in which the covalent bond breaks to produce two or more products (excluding the reverse reaction of substrate binding which represents failed substrate binding).</FONT></P><P><FONT size="3">Mass Action kinetics are used due to the fact that we have not fully characterised all aspects of molecular biology with respect to enzyme mechanics. Mass action kinetics are also deterministic in contrast to the stochastic nature of biochemical systems. Deterministic models don’t account for the variety of outcomes that can occur when a low molecular count is used as the probability of collision changes under low molecular counts due to the fact that conversion become event dependant and not rate dependant (5).</FONT></P><P><FONT size="3">Enzymatic rates were obtained from literature via the BRENDA enzyme database (6). Enzymatic rates from the origin species of each gene in the operon were used and are displayed in Table 1. Where more than one figure was available, the most cited value was used. The CHI enzyme in the operon originates from <I>Gerbera</I> sp. No rates were identified from this species so the only available rate from <I>Eubacterium rumulus</I> was used. </FONT></P><P><FONT size="2">Table 1. K<SUB>M</SUB> constant for each substrate of the four enzymes coded for by the naringenin synthesis operon <A href="http://parts.igem.org/Part:BBa_K1497017" class="black">BBa_K1497017</A> and the source literature used to identify each value. </FONT></P><DIV class="row services-list block-1 block-tab-full"><DIV class="service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="subhead">Transcription and Translation</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">It was critical to include the effects of changing both promoters and ribosome binding sites as naringenin production and steady state metabolite concentrations are dependent on enzyme abundance. This was done by modelling both transcription and translation as regulatory mechanisms such that the expression of genes could be tuned relative to one another. Transcription was spilt into two reactions for each enzyme, the first reaction being transcription initiation by recruitment of the RNA polymerases to the promoter site and the second reaction synthesis of mRNA, which was set to 40 nucleotides a second (13).</FONT></P><P><FONT size="3">Translation was also split into two separate reactions, recruitment of the ribosome and translation, which was 16.5 AA per second, based on <I>Escherichia coli</I> poly-peptide elongation (14). Changing the promoter and RBS would change the rate of recruitment of RNA polymerase and ribosomes and so affect rate of change of mRNA and Protein with respect to time. </FONT></P><P><FONT size="3">Since the RNA polymerase and ribosome concentration would remain constant and non-limiting, the rate of transcription and translation had to be defined for each of the four enzymes. Transcription rates were based on the average prokaryotic transcription rate of
approximately 40 nucleotides per second (15). The number of nucleotides for each enzyme was calculated by multiplying the number of amino acids by three. This was then divided by 40 to give the transcription time. The transcription rate was calculated by dividing one by the transcription time. Therefore, as TAL has the longest amino acid sequence its transcription time is the longest. Translation times were calculated using a rate of 16.5 amino acids per second (14).</FONT></P><P><FONT size="3">Degradation rates were included in the model to consider natural degradation of both mRNA and proteins over time. mRNA degradation rates were based on mean mRNA half-lives (16). Protein degradation was calculated assuming a protein half-life time of 20 hours (17).</FONT></P></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="notebook1" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Preliminary Scans</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">The basic deterministic design of our model means that complexity is removed at the cost of some accuracy. By manipulating the regulatory structure of the pathway through the design of our model, bottlenecks and stoichiometric imbalances could be investigated. Figure 1A shows that the model meets the most basic requirement in converting L-tyrosine to naringenin. </FONT></P><P><FONT size="3">The extent of active site saturation for each of the four enzymes is displayed in Figure 1A. Both CHI and 4CL remain unsaturated throughout the run time suggesting that these two reactions in the pathway are not rate limiting. TAL is initially highly saturated when L-tyrosine is in greatest excess but as the system reaches a steady state TAL concentration has a less significant effect on the flux of the pathway. As the system moves towards a steady state CHS becomes the most rate limiting enzyme. P-coumaroyl CoA produced by 4CL starts to build up and binds to the active site of CHS. Insufficient malonyl CoA in the system means that naringenin chalcone cannot be produced quickly enough and CHS becomes increasingly saturated with P-coumaroyl CoA.</FONT></P><P><FONT size="2">Figure 1A. Change in relative number of free enzymes for each of the four enzymes in the system over a time period of 50 seconds.</FONT></P><P><FONT size="3">Figure 1B shows how the three intermediate resources in the pathway change over time in comparison to naringenin production. Malonyl CoA is utilised more rapidly than ATP in the system as three molecules of malonyl CoA and only one ATP are required for each molecule of naringenin. In the pathway, one free CoA is utilised to produce P-coumaroyl CoA and four CoA’s are produced as a by-product in naringenin chalcone production. Due to higher flux through the pathway up until P-coumaroyl CoA, after initial fluctuation CoA concentration remains steady. </FONT></P><P><FONT size="2">Figure 1B. Consumption of input resources by the system over a time period of 50 seconds.</FONT></P><P><FONT size="3">The rate of accumulation of each waste product from the pathway is shown in Figure 1C. As CoA is both a substrate and waste product it is included in both Figure 1B and 1C. One molecule of ammonia and a proton as well as AMP and PPi are produced by TAL and 4CL respectively. Production of these pairs are therefore equal, and each pair are represented by one line in Figure 1C. With the exception of CoA, CO<SUB>2</SUB> is the only by-product produced after P-coumaroyl CoA. Three waste molecules of CO<SUB>2</SUB> are produced for every Ammonia, H<SUP>+</SUP>, AMP and PPi. Despite the similar amount of each waste product are produced. This again suggests an imbalance of flux in the pathway.</FONT></P><P><FONT size="2">Figure 1C. Accumulation of waste resources in the system over a time period of 50 seconds.</FONT></P><P><FONT size="3">In Figure 2A, 2B and 2C the bottleneck is removed by increasing the input amount of malonyl CoA by 3-fold compared to the other required resources. Figure 2B shows that the change has removed the bottleneck as CHS no longer becomes saturated. P-coumaroyl CoA no longer remains bound to CHS as sufficient malonyl CoA is available for production of naringenin chalcone.</FONT></P><P><FONT size="2">Figure 2A. Change in relative number of free enzymes for each of the four enzymes in the system over a time period of 50 seconds when input of Malonyl CoA is 3x other resources.</FONT></P><P><FONT size="3">Figures 2C and 2D show that after an initial decrease CoA increases over time. This is evidence that flux through the system is more balanced as more CoA is produced than utilised by the system so this would be expected to increase over time.
</FONT></P><P><FONT size="2">Figure 2B. Consumption of input resources by the system over a time period of 50 seconds when input of Malonyl CoA is 3x other resources.</FONT></P><P><FONT size="2">Figure 2C. Accumulation of waste resources in the system over a time period of 50 seconds when input of Malonyl CoA is 3x other resources.</FONT></P><P><FONT size="3">The results of the initial characterisation of the system show that CHS has the greatest control of flux through the pathway due to the high demand on malonyl CoA which is a limiting resource. This bottleneck could be removed by two approaches. Additional parts could be added to the operon to up regulate malonyl CoA production. This would increase the metabolic load and resource drain on the chassis organism potentially impacting on its ability to colonise the plant. Another approach would be to downregulate p-coumaroyl CoA production by altering expression of 4CL and TAL. Only trace amounts of naringenin are required for chemotaxis this option is more suitable for our system.</FONT></P></DIV></DIV></SECTION><SECTION id="SensitivityScans" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Sensitivity Scans</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">To characterise performance and identify regulatory elements within the design, scans of the model were performed where 100 simulations were run for each species; each time increasing an individual species count by 5 molecules from 0 to 500. Other species amounts were kept at a constant of 100 molecules. The effect of varying each species on conversion of naringenin to L-tyrosine (d[Naringenin] / d[L-tyrosine]) was plotted to outline performance sensitivity for each species on overall output. Through these scans parts of the system most likely to cause stability issues in a dynamic environment can be identified. Further evidence of bottlenecks within the system is also provided. With this information conclusions can be drawn on how the design may be improved by changing parts.</FONT></P><P><FONT size="3">TAL, CHS and 4CL do not affect d[Naringenin] / d[L-tyrosine] in a linear manner at low concentrations, behaviour may therefore not be consistent and predictable in the dynamic environment for which this system is being built (Figures 3A, 3B and 3C).  TALs sensitivity fluctuates as its amount increases from 0 to 40. Similar behaviour is seen when molecule counts are greater than 10 for 4CL. Production plateaued at around 50 molecules for 4CL and CHI and at 150 molecules for TAL and CHS. TAL and CHS therefore have a greater regulatory effect on naringenin production.</FONT></P><P><FONT size="2">Figure 3A. Change in d[Naringenin]/d[L-tyrosine] as the number of 4CL molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 3B. Change in d[Naringenin]/d[L-tyrosine] as the number of TAL molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 3C. Change in d[Naringenin]/d[L-tyrosine] as the number of CHS molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 3D. Change in d[Naringenin]/d[L-tyrosine] as the number of CHI molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="3">The demand of the system on ATP, CoA and malonyl CoA varies across all three resources. CoA is only rate limiting up to about 45 molecules whereas ATP and malonyl CoA are limiting up to 100 and 375 respectively (Figure 3E). This is because the metabolite produced using CoA produces three CoAs as waste upon catalysis with CHS. The net change is therefore +2 so as long as there is sufficient CoA to begin with, the systems waste will satisfy its demand. ATPs sensitivity fluctuates till its plateau at approximately 100 molecules (Figure 3F). The fluctuations may be the result of ATP increasing the rate of conversion of p-coumaric acid to p-coumaroyl CoA such that CoA is used quicker than initially produced so waste does not satisfy demand. Malonyl CoA is the most rate limiting resource as the demand per unit time is three times than any other resources (Figure 3G). This can be seen with its plateau being approximately three times that of ATP. Malonyl CoA also has by far the greatest effect on production, suggesting it is the most rate limiting resource.</FONT></P><P><FONT size="2">Figure 3E. Change in d[Naringenin]/d[L-tyrosine] as the number of CoA molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 3F. Change in d[Naringenin]/d[L-tyrosine] as the number of ATP molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 3G. Change in d[Naringenin]/d[L-tyrosine] as the number of Malonyl CoA molecules is increased from 0 to 500 in 100 steps when all other input molecules are set to 100.</FONT></P><P><FONT size="3">The sensitivity scans found that malonyl CoA had the largest overall effect on d[Naringenin] / d[L-tyrosine], with there being approximately a 25-fold difference on overall d[Naringenin] / d[L-tyrosine] between minimal and maximal starting concentrations of malonyl CoA (Figure 3G).</FONT></P><P><FONT size="3">Malonyl CoA was removed as a rate limiting resource to determine how this would change the behaviour of the system. It was hoped that this would stabilise phase transient fluctuations to make the system more predictable and robust at low molecular counts seeing as our system requirements are to have a steady and continuous output of Naringenin, potentially at low concentrations.</FONT></P><P><FONT size="3">Increasing malonyl CoA relative to the enzymes meant that transient phase fluctuations for the enzymes were no longer present (Figures 2.B). Moreover, TAL becomes the most rate limiting enzyme under these conditions and could be adjusted to alter naringenin output without comprising stability. This could be done by adjusting regulatory parts preceding the TAL coding sequence. Removing the malonyl CoA bottle neck changed the range of effect on d[Naringenin] / d[L-tyrosine] such that minimal and maximal sensitivity values were 0 to 20-24 for all instead of 0 to 1-2.5 (Figures 3 and 4).</FONT></P><P><FONT size="2">Figure 4A. Change in d[Naringenin]/d[L-tyrosine] as the number of TAL molecules is increased from 0 to 500 in 100 steps when number Malonyl CoA molecules is set to 300 and all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 4B. Change in d[Naringenin]/d[L-tyrosine] as the number of 4CL molecules is increased from 0 to 500 in 100 steps when number Malonyl CoA molecules is set to 300 and all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 4C. Change in d[Naringenin]/d[L-tyrosine] as the number of CHS molecules is increased from 0 to 500 in 100 steps when number Malonyl CoA molecules is set to 300 and all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 4D. Change in d[Naringenin]/d[L-tyrosine] as the number of CHI molecules is increased from 0 to 500 in 100 steps when number Malonyl CoA molecules is set to 300 and all other input molecules are set to 100.</FONT></P><P><FONT size="3">ATP fluctuations were removed such that increasing the supply of ATP gradually increased d[Naringenin] / d[L-tyrosine] till the plateau. Alterations in should affect d[Naringenin] / d[L-tyrosine] in a more linear fashion making the behavior of the system more predictable and robust.</FONT></P><P><FONT size="2">Figure 4E. Change in d[Naringenin]/d[L-tyrosine] as the number of CoA molecules is increased from 0 to 500 in 100 steps when number Malonyl CoA molecules is set to 300 and all other input molecules are set to 100.</FONT></P><P><FONT size="2">Figure 4F. Change in d[Naringenin]/d[L-tyrosine] as the number of ATP molecules is increased from 0 to 500 in 100 steps when number Malonyl CoA molecules is set to 300 and all other input molecules are set to 100.</FONT></P><P><FONT size="3">For the system to be robust and predictable, the most sensitive regulatory elements of the system need be stable in a dynamic environment. Increasing the supply of malonyl CoA to three times that of other species removed transient phase perturbations on d[Naringenin] / d[L-tyrosine] making system behaviour more robust and predictable. Removing the malonyl CoA bottle neck increased the range of effect of individual enzyme expression on d[Naringenin] / d[L-tyrosine] allowing more intricate tuning with respect to making the system stable or broadening its range of function. To implement this, we plan to under-express TAL and 4CL in turn removing malonyl CoA limitations.</FONT></P></DIV></DIV></SECTION><SECTION id="RegulatoryChanges" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Regulatory Changes</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">To model the effects of changing device design (i.e. promoter and RBS components), a transcription and translation system was built. This system was used to model the effects of decreasing promoter strength for TAL and 4CL. The transcription rate of TAL and 4CL was reduced by a factor of ten compared to CHS and CHI and the effects on Malonyl CoA demand and system stability were analysed. </FONT></P><P><FONT size="3">Figures 5A and 5B show that decreasing the transcription rate of TAL and 4CL relative to CHS and CHI resulted in a decrease in abundance of these enzymes. Figures 5C and 5D show how a reduction in enzyme abundance due to lowering polymerase recruitment via a weaker promoter affects Malonyl CoA demand. When TAL and 4CL expression is reduced, Malonyl CoA supply is used up more gradually (Figure 5D) compared to when all enzymes have the same promoter strength (Figure 5C).</FONT></P><P><FONT size="2">Figure 5A. Accumulation of enzymes in system over time when all enzymes are expressed equally. </FONT></P><P><FONT size="2">Figure 5B. Accumulation of enzymes in the system over time when expression of TAL and 4CL is reduced by a factor of 10 compared to CHS and CHI.</FONT></P><P><FONT size="2">Figure 5C. Rate of consumption of Malonyl CoA compared to accumulation of soil naringenin when all enzymes are expressed equally.</FONT></P><P><FONT size="2">Figure 5D. Rate of consumption of Malonyl CoA compared to accumulation of soil naringenin when expression of TAL and 4CL is reduced by a factor of 10 compared to CHS and CHI.</FONT></P><P><FONT size="3">The drawn-out demand that results from reducing the expression of TAL and 4CL, relative to CHI and CHS, also reduces transient phase perturbations for CHI and CHS (Figure 5E). In a dynamic environment, this will increase how stable and robust the defined output range is.</FONT></P><P><FONT size="2">Figure 5E. Accumulation of enzymes in the system over time when expression of TAL and 4CL is reduced by a factor of 10 compared to CHS and CHI (i) and when all enzymes are expressed equally (ii).</FONT></P></DIV></DIV></SECTION><SECTION id="RedesignedOperon" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Redesigned Operon</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">The original objective when creating a synthesis pathway model was to improve on the design by team Darmstadt 2014 in order to increase flux through the pathway. The Darmstadt design consisted of all four enzyme coding sequences under the control of a single constitutive promoter (Figure 6).</FONT></P><P><FONT size="2">Figure 6. Naringenin synthesis operon <A href="http://parts.igem.org/Part:BBa_K1497016" class="black"> BBa_K1497016 </A> designed by team Darmstadt 2014.</FONT></P><P><FONT size="3">The new design is based on the results of the model that identified conversion of malonyl CoA and p-coumaric acid to p-coumaroyl CoA by CHS as the major rate limiting step in the pathway. Rather than introduce new coding sequences to up-regulate malonyl CoA production the design down-regulates p-coumaric acid production. The promoters in the new operon design were sourced from literature examining promoter strength in <I>Pseudomonas putida</I> (18). The promoter selected for TAL and 4CL induces expression weaker than the promoter for CHS and CHI by approximately ten-fold. The team has examined measurement of promoter strength as part of our project <A href="https://2018.igem.org/Team:Newcastle/Measurement" class="black">(Measurement Page)</A>. Information from these new methods of measurement and automated transformation can be deployed when characterising promoters in our endophyte chassis, which can be taken into account when finalising the model and construct design for deployment in our <I>Pseudomonas</I> sp. chassis.</FONT></P><P><FONT size="2">Figure 7. Novel naringenin synthesis operon designed for optimum pathway flux.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References &amp; Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="3"><B>Attributions: Frank Eardley</B></FONT></P><P><FONT size="3">1.Kumar S &amp; Pandey AK (2013) Chemistry and Biological Activities of Flavonoids: An Overview. <I>The Scientific World Journal</I> 2013.</FONT></P><P class="about-para"><FONT size="3">2.Santos CNS, Koffas M, &amp; Stephanopoulos G (2011) Optimization of a heterologous pathway for the production of flavonoids from glucose. <I>Metabolic Engineering</I> 13(4):392-400. </FONT></P><P class="about-para"><FONT size="3">3.Vannelli T, Wei Qi W, Sweigard J, Gatenby AA, &amp; Sariaslani FS (2007) Production of p-hydroxycinnamic acid from glucose in <I>Saccharomyces cerevisiae</I> and <I>Escherichia coli</I> by expression of heterologous genes from plants and fungi. <I>Metabolic Engineering</I> 9(2):142-151. </FONT></P><P class="about-para"><FONT size="3">4.Fowler Z &amp; Koffas M (2009) Biosynthesis and biotechnological production of flavanones: current state and perspectives. <I>Applied Microbiology and Biotechnology</I> 83(5):799-808. </FONT></P><P class="about-para"><FONT size="3">5.Sayikli C &amp; Bagci EZ (2011) <I>Limitations of Using Mass Action Kinetics Method in Modeling Biochemical Systems: Illustration for a Second Order Reaction </I>(Berlin, Heidelberg: Springer Berlin Heidelberg, Berlin, Heidelberg) pp 521-526. </FONT></P><P class="about-para"><FONT size="3">6.Placzek S, et al. (2017) BRENDA in 2017: new perspectives and new tools in BRENDA. <I>Nucleic acids research </I>45(D1):D380. </FONT></P><P class="about-para"><FONT size="3">7.Xue Z, McCluskey M, Cantera K, Sariaslani F, &amp; Huang L (2007) Identification, characterization and functional expression of a tyrosine ammonia-lyase and its mutants from the photosynthetic bacterium <I>Rhodobacter sphaeroides</I>. <I>Official Journal of the Society for Industrial Microbiology</I> 34(9):599-604. </FONT></P><P class="about-para"><FONT size="3">8.Ehlting J, et al. (1999) Three 4-coumarate:coenzyme A ligases in <I>Arabidopsis thaliana</I> represent two evolutionarily divergent classes in angiosperms. <I>Plant Journal</I> 19(1):9-20. </FONT></P><P class="about-para"><FONT size="3">9.Stuible HP, Büttner D, Ehlting J, Hahlbrock K, &amp; Kombrink E (2000) Mutational analysis of 4‐coumarate:CoA ligase identifies functionally important amino acids and verifies its close relationship to other adenylate‐forming enzymes. <I>FEBS Letters</I> 467(1):117-122. </FONT></P><P class="about-para"><FONT size="3">10.Hess V, Vitt S, &amp; Muller V (2011) A Caffeyl-Coenzyme A Synthetase Initiates Caffeate Activation prior to Caffeate Reduction in the Acetogenic Bacterium <I>Acetobacterium woodii</I>. <I>The Journal of Bacteriology</I> 193(4):971. </FONT></P><P class="about-para"><FONT size="3">11.Ozeki Y, et al. (1985) Purification and some properties of chalcone synthase from a carrot suspension culture induced for anthocyanin synthesis and preparation of its specific antiserum. <I>Journal of biochemistry</I> 98(1):9. </FONT></P><P class="about-para"><FONT size="3">12.Herles C, Braune A, &amp; Blaut M (2004) First bacterial chalcone isomerase isolated from <I>Eubacterium ramulus</I>. <I>Archives of Microbiology</I> 181(6):428-434. </FONT></P><P class="about-para"><FONT size="3">13.Clancy S (2008) DNA Transcription. <I>Nature Education</I> 1(1):41. </FONT></P><P class="about-para"><FONT size="3">14.Young R &amp; Bremer H (1976) Polypeptide-chain-elongation rate in <I>Escherichia coli</I> B/r as a function of growth rate. <I>The Biochemical journal</I> 160(2):185. </FONT></P><P class="about-para"><FONT size="3">15.Lodish HF (2016)<I> Molecular cell biology</I> (New York : W.H. Freeman Macmillan Learning) Eighth edition..; Global edition.. Ed. </FONT></P><P class="about-para"><FONT size="3">16.Bernstein JA, Khodursky AB, Lin P-H, Lin-Chao S, &amp; Cohen SN (2002) Global Analysis of mRNA Decay and Abundance in <I>Escherichia coli</I> at Single-Gene Resolution Using Two-Color Fluorescent DNA Microarrays. <I>Proceedings of the National Academy of Sciences of the United States of America </I>99(15):9697-9702. </FONT></P><P class="about-para"><FONT size="3">17.Borek E, Ponticorvo L, &amp; Rittenberg D (1958) Protein turnover in microorganisms. <I>Proceedings of the National Academy of Sciences</I> 44(1):369-374.</FONT></P><P><FONT size="3">18. Zobel S, et al. (2015) Tn7-Based Device for Calibrated Heterologous Gene Expression in <I>Pseudomonas putida</I>. <I>ACS synthetic biology</I> 4(12):1341.



                    
   
</FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>