<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/5/5a/T--Newcastle--q4534rtw.png" data-natural-width="3000" data-natural-height="2000" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                   Measurement, Automation and Reproducibility
                    </H1><DIV class="home-content__buttons"><A href="#Introduction" class="smoothscroll btn btn btn--stroke">
         Introduction
        </A><A href="#Improved Measurement Plasmids" class="smoothscroll  btn btn btn--stroke">
         Improved Plasmids
        </A><A href="#Automation" class="smoothscroll btn btn btn--stroke">
         Automation 
        </A><A href="#Media" class="smoothscroll btn btn btn--stroke">
         Media composition
        </A><A href="#Conclusions" class="smoothscroll btn btn btn--stroke">
         Conclusion
        </A></DIV></DIV></DIV></SECTION><SECTION id="Introduction" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Introduction</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">The Interlab study <A href="https://2018.igem.org/Team:Newcastle/InterLab" class="black"> Newcastle iGEM InterLab </A> has provided a platform to develop robust measurement procedures for assessing promoter strength and examining cross laboratory variation in genetic devices and measurement equipment [1,2]. Reproducibility however, is taken as a defining feature of science, but the reproducibility of experimental research is an assumption not a fact [1,2]. To aid in the continual improvement of the InterLab methodologies we investigated biodesign automation, statistical experimental design and standardised reporting of materials and methods. Our goal was to contribute to iGEM’s ongoing efforts to reduce experimental variability, increase measurement reliability and broaden the range of open source protocols. An additional goal was to road-test the foundation skills and technologies that we would be needed in the ‘domestication’ of our endophytic <I>Pseudomonas</I> chassis.  </FONT></P><P><FONT size="3"> All methods are documented in the <A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods" class="black">Materials and Methods section</A>.

</FONT></P></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Bio-design Automation</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">Bio-design automation (BDA) is defined as robots + software + biology [3]. BDA therefore co-develops automated hardware solutions (i.e. the integration of robotics into wet-lab procedures) with automated software that drives the workflow, the hardware and in silico modelling, design and optimisation [3]. Automated hardware - such as liquid handling robots and microfluidic systems - allow for improvements in the reproducibility of standardised workflows, while removing human error as a source of variation [1]. Integrating software with the hardware further allows for <I>in silico</I> modelling including extracting and sharing information from many databases, as well as providing computer aided construct design and laboratory and hardware management. Within the BDA framework, complex genetic circuits can be built, designed and tested rapidly and the data retained to allow future biodesign. Recent developments in affordable technologies for lab equipment, notably in the release of the Opentrons OT-2 liquid handling robot, and the ever-growing library of software, means that interest in the BDA framework is high and likely to rise further.</FONT></P></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Design of Experiments</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> Design of Experiments (DoE) is statistical method that provides a systematic means to explore large, multifactorial design spaces. This is an efficient approach to optimising complex bio-processes, and also allows for interacting experimental factors to be identified.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Minimal Information Standards</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">Minimum information standards (MIEO) provide explicit information in advance on what information needs to be recorded and reported with each experiment. This cataloguing and reporting of experimental metadata provides a record of the various factors that could influence the reproducibility of the result [4]. Such experimental factors relating to growth and productivity of engineered organisms have been described previously by Hecht et al. [5].</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">This Study</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> The Alternative Roots team investigated ways to improve measurement, reproducibility and automation aspects of future InterLab studies. The team employed automated protocols, liquid handling robotics and the DoE methodology to aid in optimisation of <I>Escherichia coli</I> (<I>E. coli</I>) transformation protocols and chemically defined media. We also developed new InterLab study plasmids featuring an internal standard fluorescent signal and improved GFP signal. The results from this work can be seen in the three sections below:  </FONT></P></DIV></DIV><SECTION id="Improved Measurement Plasmids" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Improved Interlab Measurement Plasmids</H1><H3 class="subhead">Inclusion of an RFP internal standard into the 2018 Interlab test plasmids </H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">One goal of the 2018 iGEM InterLab study is to overcome the weakness inherent in the commonly-used measurement of promoter strength, that of reporting the fluorescence value relative to the optical density at 600 nm (OD<SUB>600</SUB> nm) of the culture media. There is clearly the potential for a discrepancy to occur between OD<SUB>600</SUB> nm and actual cell concentration which can result in an erroneous conclusion being drawn about the effectiveness of the promoter. This year the 2018 iGEM study sought to reduce lab-to-lab variability by measuring GFP fluorescence relative to absolute cell counts or colony forming units (cfu). Normalisation of fluorescence against cfu ensures measurement of fluorescence relative only to viable cells. </FONT></P><P><FONT size="3">Our RFP internal standard (IS) was developed to address the issue that copy number of the same plasmid is not consistent across cells. The inclusion of an IS RFP signal (Figure 1 <A href="http://parts.igem.org/Part:BBa_K2797013" class="black">Part:Bba_K2797013</A>) is therefore designed to allow measurement of variation in gene expression between cultures and transformant lines. The underpinning assumption is that because the IS devices are identical across each of the plasmids, the GFP fluorescence values of the test devices of interest can be reported relative to their internal RFP signal.</FONT></P><P><CENTER><B><FONT size="2"> Figure 1. The K2797013 plasmid following simulated Gibson Assembly of the RFP internal standard sequence into the pSB1C3 vector backbone. </FONT></B><FONT>As shown, the RFP internal standard inserts between the chloramphenicol resistance gene and the ORI. The RFP gene is shown in pink.</FONT></CENTER></P><P><FONT size="3">The new plasmids were tested. Analysis of the internal standards involved comparing the original InterLab test device plasmid performance against the new internal standard plasmid performance. MTP wells A-D represented the <I>E. coli</I> containin the RFP internal standard vector and wells E-H represented <I>E. coli</I> containing the original test devices. Column 9 wells A-H contained an LB+cam blank. The microtiter plate was incubated for 24 hours in the plate reader with Abs600, fluorescence (GFP): Excitation 485 nm, Emission 420 nm and fluorescence; (RFP): Excitation 588 nm, emission 635 nm measured every 15 minutes following a short shake at 420 rpm at a low shake diameter.</FONT></P><P><FONT size="3">Although there was no calibration standard available for our RFP, there was a clear indication that there was variation in RFP gene expression between each device (Figure 2). It is important to remember that the only genetic difference between these devices is in the promoter used to drive GFP expression - not RFP expression. Moreover, the inclusion of the RFP impacted on the response of the GFP signals across the test devices.  GFP fluorescence values of the modified InterLab plasmids were seen to be lower than standard InterLab devices. Devices 1 &amp; 4 – the devices with the strongest promoters – exhibited lower GFP fluorescence than unmodified InterLab plasmids and did not show any RFP fluorescence (sequencing confirmed the presence of the RFP part). Taken together these observations suggest that there is competition for cellular resources (i.e.  Transcriptional/translational machinery) [6].This has been observed previously in <I>E. coli</I>, where genes are activated and deplete the pool of free RNAP and ribosomes, reducing their availability to other cassettes [7,8]. In this case, it may be that gene expression of GFP in devices 1 &amp; 4 is so powerful that there is no available cellular machinery to enable protein synthesis whilst the variation in RFP signals across all plasmids is likely to result from similar, but less pronounced competition. </FONT></P><P><CENTER><B><FONT size="2">Figure 2. A) Scatter plots showing the relationship between Relative AFUs (GFPmut3b fluorescein/OD) against time (x-axis) for the original test device and the internal standard test devices &amp; B)Scatter plots showing the relationship between Relative AFUs (RFP fluorescence/OD) (y-axis) and time (x-axis) for the RFP internal standard test devices.</FONT></B><FONT> A)The legend for each device is shown on the left of each graph. Over 24 hours, the original test devices reach peak fluorescence at around the 6-7 hour mark, with devices 4 &amp; 1 being the strongest and 3 &amp; 6 being the weakest. After the 6 hour mark there is a decrease in fluorescence to the 22 hour mark where fluorescence beings to increase once again. The internal standard devices all showed a consistent increase with no decrease over 24 hours. Device 2 was the most fluorescent with devices 1 and 4 being significantly lower than the original group. Fluorescence was lower for each device initially than the original but eventually surpassed the original study as the fluorescence decreased over time. 
B&gt; Over the 24 period, the control groups and devices 1 &amp; 4 exhibited a fluorescence/OD of 0. The remaining other devices - 2, 3 &amp; 6 – all showed a consistent positive increase in fluorescence/OD. Device 2 however has the highest fluorescence/OD throughout the study, followed by devices 6, 3 and 2 with fluorescence/OD values of 0.064, 0.048 and 0.038 respectively. These groups began fluorescing between 200 and 440 min. </FONT></CENTER></P><P><FONT size="3">A further interesting observation is the difference seen in GFP signal longevity. The fluorescence/OD<SUB>600</SUB> of the original test devices exhibited a significant decrease after six hours while in the IS containing devices, although the signal was lower, it remained more consistent over 24 hours. Genes have been known to act as transcriptional modulators for one another in the same operon [10]. However, genes on the same plasmid but at different locations can act as transcriptional modulators for one another [6]. It may be that a coupling of these genes due to competition for transcriptional machinery has led to the modulation of GFP gene expression, causing it to be less productive but more consistent. There is room for improvement in this study. RFP is a slow maturing protein. Fluorescent proteins like mCherry, which possess brighter fluorescence and fold faster [11] are a better option for use as an IS. Furthermore, a calibration with a red fluorescent compound (e.g. rhodamine red) would be beneficial. This would allow molecules of equivalent rhodamine red’ values so estimate RFP or other internal standard concentrations in cells. </FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Comparisons between the existing GFPmut3b test devices and a new series of mNeon green test devices</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> From the literature, we gathered information on the potential problems associated with the GFPmut3b reporter – notably its low photostability upon exposure to natural light [9]. However, since GFPmut3b is the brightest GFP variant, it would make sense to utilise a suitably divergent fluorescent protein with similar fluorescence characteristics to GFPmut3b but higher photostability – mNeonGreen fits this mould.</FONT></P><P><FONT size="3">InterLab test devices 1 – 4 were successfully converted from GFPmut3b to mNeon green (Figure 3). Devices 5 and 6 were not successfully converted. </FONT></P><P><CENTER><B><FONT size="2"> Figure 3. A) UV imaging of LB+cam agar streak plates containing <I>E. coli</I> transformed with respective pSB1C3 mNeonGreen vectors. B)Box and whisker plot of the fluorescein/OD (y-axis) values against time (x-axis) for the original and mNeonGreen test devices in the InterLab study at 0 hours and 6 hours.</FONT></B><FONT> A) Each vector held the mNeonGreen gene, inserted via Gibson Assembly. The figure shows, in order, colonies from test device 1 (Top Left) through to test device 4 (bottom right). Devices 1, 2, and 4 yield the brightest colonies, with device 4 beginning to show a yellow/green colour in comparison to the others which are predominantly green. Device 3 is the weakest promoter of the group and its colonies show little fluorescence. B)The legend for each device is shown at the bottom of the graph. Throughout the tests, for both replicate colonies the sample groups deviated from normality. Kruskal-Wallis testing was performed and revealed a significant difference in median between the device groups (p &lt; 0.001). Post-hoc Dunn testing for each of the colonies and time points revealed that there were no significant differences in median fluorescein/OD of the original vectors and their mNeonGreen counterparts of the same device group At hour 0, throughout the GFP and mNeonGreen devices there are large spreads of data. At hour 6, mNeonGreen devices can be seen to have a smaller spread of data when compared to GFP in both colonies 1 &amp; 2. Also, in most cases, the fluorescence is much higher in the mNeonGreen, with mNeonGreen TD4 having the highest median fluorescein/OD value (1.57) of the whole study. The pattern of fluorescence regarding the most productive devices is the same as that of the original InterLab study.</FONT></CENTER></P><P><FONT size="3"> The data indicate that mNeonGreen was not significantly brighter than mut3GFP as was proposed. Figure 3 shows that the fluorescence patterns in mNeonGreen are similar to those of the original test devices, with 4 &amp; 1 being the strongest promoters and 3 being the weakest. However, the values for both replicate colonies show that the spread of fluorescence/OD600nm values for the mNeonGreen reporter is lower in each test device group.</FONT></P><P><FONT size="3"> The results of this experiment therefore do not support the original goal of the study. The mNeonGreen reporter, despite reports in the literature, was not seen to be significantly different to mut3GFP in its fluorescence/OD<SUB>600</SUB> nm. This is consistent with previous reports demonstrating a lower fluorescence in vivo, compared to a higher in vitro fluorescence [12,13]. The mNeonGreen did, however, show a smaller range of fluorescein/OD data in comparison to GFPmut3b. Since there were no differences in fluorescence values, the smaller range of values from the mNeonGreen reporter may be due to its better photostability, or a difference in maturation time in vivo than the literature suggests. Regardless, the switch to mNeonGreen in InterLab test devices may further improve measurement reliability. </FONT></P></DIV></DIV></SECTION><SECTION id="Automation" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Automated protocols for <I>Escherichia coli</I> transformation</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> Bacterial transformations are a core part of much of iGEM and the InterLab study. We therefore developed an automated protocol for small-scale, high-throughput transformation of <I>E. coli</I> using the OpenTrons OT-2 (See <A href="https://2018.igem.org/Team:Newcastle/Software/OT" class="black">software</A> for more details). We used this protocol to improve the transformation efficiency of this procedure.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Preliminary Research</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> A literature search, examining protocols for the preparation of chemically competent <I>E. coli</I> cells and transformation procedures, identified factors reported to influence transformation efficiency. This included factors such as <I>E. coli</I> strain, growth medium, growth volume, cell density, wash steps, wash buffer composition, DNA concentration, recovery medium and recovery time. It was noted that transformation buffer (TB) composition, used in cell preparation, not only varied between protocols, but was also deemed the primary variable for inducing competency. Therefore, TB composition was selected for optimisation and other factors were set at a constant level.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Minimising and automating the protocol (including new OT2 hardware)</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> Laboratory automation, liquid handling and reaction miniaturisation enable high-throughput experimentation. Protocols for competent cell preparation however typically use 50-250 mL of culture volumes. We set out to downsize competent cell preparation so that optimisation could be performed using automated workflows.</FONT></P><P><FONT size="3"> No significant difference in transformation efficiency was found between cells prepared in 50 mL or 2 mL tubes, but transformation efficiency was found to be significantly lower in cells prepared in 96-well plate format (Figure 4). Due to this result, all non-automated workflows were carried out using 2 mL microcentrifuge tubes to ensure transformation success. Once automation workflows began, the reduced transformation efficiency would be considered when analysing the results. </FONT></P><P><FONT size="2"><CENTER><B>Figure 4. Impact of downscaling competent cell production from 50 mL falcon tube to 400 µL 96 plate well volumes. Black circles = 50 mL falcon tube, grey squares = 2 mL microcentrifuge tubes, light grey triangle = 400 µL 96 well plate.</B> Competent cells were produced using MgCl<SUB>2</SUB>+CaCl<SUB>2</SUB> protocol and transformed using standard heat shock. 100 µL of transformed cells were then plated out on SOB+CAM and incubated overnight at 37 degrees. Colonies were counted and transformation efficiency (TrE) calculated. A significant difference in transformation efficiency depending on reaction vessel was shown (ANOVA, F2,15 = 8.24, P = 0.004). Post hoc Tukey test indicated that both 50 mL Falcon tubes and 2 mL microcentrifuge tube volumes had a statistically insignificant difference in TrE (T = 0.06, p = 0.998). 96 well plate TrE was statistically lower than both 50 mL and 2 mL volumes (p = 0.009 and p = 0.008 respectively). Plasmid concentration had a significant effect on TrE, with the 100 pg/µL TrE a power of 10 greater on average (t= -2.81, d.f = 16, p = 0.013).  </CENTER></FONT></P><P><FONT size="3">  To reduce protocol complexity and length, wash number and wash combination were evaluated, with the results suggesting no significant impact on transformation efficiency (Figure 5). Wash steps were excluded moving forward to streamline and decrease protocol complexity, without significant loss of transformation efficiency.</FONT></P><P><FONT size="2"><CENTER><B>Figure 5. Effect of different wash steps on overall transformation efficiency (TrE). All competent cell preparation followed the standard MgCl<SUB>2</SUB>-CaCl<SUB>2</SUB> protocol, with only the wash steps altered.</B> 0 Wash - initial culture pellet followed by immediate aliquot of 100 µL storage/transformation buffer. 1 wash – a combined 100 mM MgCl<SUB>2</SUB> and 100 mM CaCl<SUB>2</SUB> buffer with 1 wash step. MgCl<SUB>2</SUB> + CaCl<SUB>2</SUB>  – a combined 100 mM MgCl<SUB>2</SUB> and 100 mM CaCl<SUB>2</SUB> buffer with original two wash steps. MgCl<SUB>2</SUB>/CaCl<SUB>2</SUB> – a 100 mM MgCl<SUB>2</SUB> wash step, followed by a separate 100 mM CaCl<SUB>2</SUB> wash step as per standard protocol.  No significant impact on TrE (Kruskal-Wallis, H = 1.34, d.f. = 3, p = 0.720) was shown. Removing the wash step was the most effective (mean TrE = 2.30 x 106), with the more time consuming MgCl<SUB>2</SUB>-CaCl<SUB>2</SUB> protocol being the second most effective (mean = 2.18 x 106). The least effective were the combined MgCl<SUB>2</SUB>/CaCl<SUB>2</SUB> two wash (mean TrE = 1.79 x 106) and one wash (mean TrE = 1.66 x 106). </CENTER></FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">DoE scoping</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  To determine suitable ranges for each experimental factor, preliminary scoping experiments were performed. Several different areas were investigated: concentration ranges and choice of cryoprotectant, buffering agents, and PEG8000 inclusion.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Low, Medium and High Concentration Transformation Buffers with Glycerol or DMSO as a Cryoprotectant</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  TB composition was investigated at three levels, and two commonly used cryoprotectants, DMSO and glycerol, were also tested.</FONT></P><P><FONT size="3">  Data indicated that transformation efficiency was optimal at a moderate TB concentration. Choice of cryoprotectant alone did not affect transformation efficiency, however, there was evidence of an interaction between buffer complexity and cryoprotectant choice. Specifically, at a moderate concentration of TB with the inclusion of DMSO, transformation efficiency was significantly higher (Figure 6).</FONT></P><P><FONT size="2"><CENTER><B>Figure 6. Initial DoE scoping test of low, medium and high transformation buffer concentrations with different cryoprotectants.</B> Black – 7.5% DMSO, grey  – 18% glycerol. Low Buffer concentration consisted of 15 mM CaCl<SUB>2</SUB>.2H<SUB>2</SUB>O solution. High buffer concentration consisted of a 100 mM MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O, 100 mM CaCl<SUB>2</SUB>.6H<SUB>2</SUB>O. 10 mM kOAc, 100 mM MnCl<SUB>2</SUB>.4H<SUB>2</SUB>O, 100 mM RbCl, 100 mM NiCl<SUB>2</SUB>, 3 mM [Co(NH<SUB>3</SUB>)<SUB>6</SUB>]Cl<SUB>3</SUB> and 100 mM KCl solution. Two-way ANOVA determined a significant difference in TrE dependent on Buffer Concentration (ANOVA, Buffer Concentration: F<SUB>2,16</SUB> = 4.593, p = 0.0265) whilst there was no significant difference between cryoprotectants (ANOVA, Cryoprotectant: F<SUB>1,16</SUB> = 3.469, p = 0.0810). There was a significant interaction between the two (ANOVA: Interaction: <SUB>F2,16</SUB> = 6.548, p = 0.0084). Post hoc Tukey test confirmed that the medium wash concentration with DMSO resulted in significantly greater TrE (mean TrE = 9.29 x 106) whilst all other TB compositions were insignificantly different.</CENTER></FONT></P><P><FONT size="3"> Glycerol is commonly used in TB and no negative effects were observed when it was included. DMSO, however, has been seen to inhibit transformation efficiency in low complexity TB while significantly increasing transformation efficiency in the more complex buffers. Going forward, DMSO was selected as the cryoprotectant, as complex TB composition was the target of the main investigation.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">pH Buffering Agent</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  Initial research highlighted pH as an important factor relating to transformation efficiency. Therefore, several buffering agents were investigated: HEPES, PIPES, MES and MOPS. HEPES, PIPES and MES all had a positive effect on transformation efficiency, whereas MOPS showed no significant difference compared to the control (Figure 7).</FONT></P><P><FONT size="3"> HEPES was selected for use in subsequent investigations as the highest transformation efficiency was observed with this buffer. </FONT></P><P><FONT size="2"><CENTER><B>Figure 7. Assessment of pH buffer effect on overall transformation efficiency (TrE). </B> Transformation buffer used was the medium scoping buffer (MSB) with 7.5 % DMSO. Control – MSB without pH buffer, HEPES – MSB + 10 mM HEPES, PIPES – MSB + 10 mM PIPES, MES – MSB + 10 mM MES, MOPS – MSB + 10 mM MOPS. All buffers were adjusted to 6.8 pH for comparison and to prevent manganese dioxide from precipitating out of the MSB. Inclusion of pH buffering agent significantly affected  TrE (ANOVA, F4,10 = 6.45, p = 0.008) (Figure 9). Post hoc Tukey test clarified that HEPES, PIPES and MES had a significant increase in TrE when compared to the control (p = 0.006, p = 0.032, p = 0.035 respectively). MOPS had minimal effect on TrE when compared with the control with no significant difference being shown (p = 0.234), yet  mean TrE was still 4.80 x 105 greater than control. </CENTER></FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">PEG 8000 Inclusion</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> Previous studies evidence the benefit of including PEG 8000 [14,15,16]. Our data (mean TrE = 4.80 x 106 when compared to a TB control mean TrE = 2.26 x 105) (t = -3.75, d.f. = 2, t = 0.064)). agreed with these findings, leading to increased transformation efficiency, therefore, PEG 8000 was included in the DoE optimisation </FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Workflow optimisation</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  Optimisation of the automated competent cell and transformation workflow was successful and showed an increase in transformation efficiency. The initial protocol was semi-automated, containing multiple manual steps and a mean transformation efficiency of 1.89 x 10<SUP>4</SUP>. Next, the code and OT-2 deck environment were optimised to achieve a fully automated workflow [https://2018.igem.org/Team:Newcastle/Software/OT]. Following optimisation, the code was refactored to improve efficiency. This led to a significant increase in transformation efficiency, with a mean transformation efficiency of 1.20 x 10<SUP>5</SUP> (Figure 8). This optimised workflow was combined with a DoE complex TB buffer construction protocol to make the ‘Automated Transformation Buffer Optimisation and Transformation efficiency analysis protocol’ (ATBOT), to be used for the definitive screening of TB buffer components.</FONT></P><P><FONT size="2"><CENTER><B>Figure 8. Comparison between initial unoptimised and further optimised automated competent cell and transformation protocols.</B> Circles indicate raw data values, with central line indicating mean with SD error bars.  Both protocols followed the same 0 wash method. A significant increase in TrE was shown, with a mean TrE of 1.20 x 10<SUP>5</SUP> (Mann-Whitney, U = 333.0, n. = 21,30, p &lt; 0.001). </CENTER></FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Hardware optimisation</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> The opensource and adaptable nature of the OT-2 allowed us to design bespoke hardware specific to our needs. This included a tube rack to contain 8 x 20 mL universal tubes which could be inserted into a multipurpose ice box designed to keep reagents at 0-4 °C. This rack, along with 100 mL and 250 mL Duran bottles for cultures or microbial waste were defined and uploaded to the robot API. This provided all the necessary components to carry out the protocol. </FONT></P><P><FONT size="3"> The original protocol required manual interactions to chill, heat shock and incubate the cells. In the ATBOT protocol, the TempDeck was employed to maintain cells at 4 °C, heat shock cells at 42 °C and provide recovery incubation at 37 °C. However, it was noted that the heat shock step did not provide the same shock as would occur during the manual protocol. As such, the heat shock step was modified in the code.</FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Software optimisation</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> The following sections describe the notable steps made during optimisation and refactoring of the ATBOT protocol (Figure 9) and is described for non-python users. For full python script and details, see <A href="https://2018.igem.org/Team:Newcastle/Software/OT" class="black">here</A> (for individual scripts and download see: https://github.com/jbird1223/Newcastle-iGEM/tree/master/OT-2%20Protocol). This protocol allowed ~775 individual pipetting steps to be accurately automated and with a total active run time of approximately 70 minutes.</FONT></P><P><FONT size="2"><CENTER><B>Figure 9. Optimised workflow for the automated transformation buffer optimisation and transformation efficiency (TrE) analysis protocol (ATBOT).</B> Light grey boxes indicate manual steps whereas dark grey boxes indicate automated steps. Y/N show a logic step in python script. All liquid handling was carried out by the OT-2 robot, with TempDeck module allowing for temperature control and heatshock steps without manual interaction. Box (*) describes the logic steps that are undertaken during the for and if/else/elif loops required for the OT-2 to carry out complex P10/P300 pipetting steps. Outcomes of ATBOT are highlighted as 1, 2 or 3. Outcome 1 allows for the assessment of accurate CFU and calculation of TrE which can be inputted into JMP Pro and used to further model the DoE design space. Outcome 2 allows for successfully transformed colonies to be isolated for further testing or use. Outcome 3 transfers post-recovery transformants into a selection broth, allowing for either overnight incubation or plate reader assessment. Plate reader assessment can be used to determine growth rates and may potentially be used as a means to accurately calculate TrE.</CENTER></FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">DoE TB buffer Construction Protocol</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> DoE was used to design a set of complex TBs encompassing multiple factors at difference concentrations. The experimental design was generated in JMP Pro 13 software and complied as a [list] within [list] block which enabled the conversion of CSV format reagent concentration and volume to python script. (note that any text after # denotes a comment, not code): 

Stock1 = [
     [1,2,3,4,5]
     [1,2,3,4,5]]
Stock_pos = [
‘A1’, ‘A3’] 
# 20 mL Universal labware containing stocks
# Number equates to uL of stock transferred
# Positions of stocks in Stock1 labware
# Stock position in labware
This list within a list provided both the means to rapidly import multiple factors into the code, and for non-python users to easily modify factors. </FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Competent Cell and Transformation Protocol</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  The automated competent cell and transformation protocol used only basic transfer and mixing steps. A modifiable command block was included allowing non-python users to easily modify the protocol. Notable commands added during the optimisation and refactoring process were the inclusion of the TempDeck and a single transfer loop that removed supernatant, added TB buffer and fully resuspended the pellet.</FONT></P><P><FONT size="3"> The TempDeck maintained an environment of 0-4 °C, widely regarded as optimal for inducing competency in <I>E. coli</I> DH5α. For use in heatshock, the standard 45 seconds at 42 °C was modified to account for time taken for the TempDeck to reach 42 °C. The TempDeck °T from 4→42 °C took approximately 40 seconds, so a 100 second delay was scripted to allow a prolonged heating period. </FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Deploying ATBOT</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  Based on scoping data, concentration ranges were adjusted to achieve a more targeted design. A definitive screening design (DSD) was used to generate 25 separate buffer recipes for testing. These copositions covered 11 factors at a range of concentrations, maximising experimental coverage and reducing the number of runs.</FONT></P><P><FONT size="3"> The ATBOT protocol was used for experimental set-up and execution, with each run performed in triplicate. Transformation efficiency data was analysed in JMP Pro 13 and used to construct a standard least squares model.</FONT></P><P><FONT size="3"> As expected, TB buffers with highest average transformation efficiency of 2.21 x 10<SUP>6</SUP> contained minimal DMSO and lacked NiCl<SUB>2</SUB>, with the highest concentration of MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O (80 mM) that was factored into the design. Reduces transformation efficiency due to 96-well plate format was also factored in, giving a theoretical transformation efficiency of 2.21 x 10<SUP>8</SUP>. Four out of five TB buffers that produced no transformants, contained NiCl<SUB>2</SUB> at its highest concentration. The fifth contained no NiCl<SUB>2</SUB>, but had the lowest cationic compound composition and included 15 % DMSO. This strongly suggests that NiCl<SUB>2</SUB> is a major inhibitor of competency. </FONT></P><P><FONT size="3"> JMP analysis was also used to identify interacting factors, using the principle of effect sparsity. Five of 11 factor were identified as exhibiting interactions (Figure 12). A prediction profile for individual reagents at desirable concentrations was generated (Figure 12A) as well as the visualisation of interactions between reagents at differing concentrations (Figure 11B). NiCl<SUB>2</SUB> and DMSO were found to have inhibitory effects, reinforcing earlier modelled predictions (Figure 10). The interaction profile (Figure 11B) highlighted two significant interactions, NiCl<SUB>2</SUB>*DMSO (P = 0.0054) and MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O*RbCl (P = 0.0095). In both cases, high concentrations of both components resulted in a large significant increase in transformation efficiency, however when either concentration were lowered, transformation efficiency was reduced. MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O was found to have a significantly positive (P = 0.0105) effect on transformation efficiency, while KOAc was found to be significantly inhibitory (P = 0.0172). </FONT></P><P><FONT size="2"><CENTER><B>Figure 10. Prediction profiler modelling the effect that varying reagent concentration has on overall transformation efficiency (TrE). </B> Overall prediction profile set to most desirable transformation buffer composition. Black line indicates concentration relative to predictive TrE. Vertical dashed red line indicates the concentration of reagent at most desirable composition while horizontal dashed red line indicates predicted average TrE at desirable composition. Top panels show calculated effects of reagents after TrE was calculated after a 37 ℃ overnight 16 hour incubation post transformation recovery step. Bottom panels show calculated effects of reagents after TrE was recalculated after 96 hour incubation at room temperature (22-25 ℃) post transformation recovery step. Highlighted green panel indicate that a higher MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O concentration had a statistically significant (p=0.0096) positive increase in overall TrE after a 96 hour incubation. Its effect after 16 hours incubation was shown to be positive, however not statistically significant. Highlighted blue panels indicate a statistically significant decrease in overall TrE when concentration is increased. Blue panel (A) indicates that the presence NiCl<SUB>2</SUB> is significantly inhibitory to overall TrE (p = 0.0058) after 16 hours post recovery. Blue panel (B) suggests that CaCl<SUB>2</SUB>.6H<SUB>2</SUB>O has a negative impact with increasing concentration (p = 0.0490) and blue panel (C) indicates that kOAc has a significant inhibitory effect (p = 0.0153) on overall TrE after 96 hours post incubation. [Co(NH<SUB>3</SUB>)<SUB>6</SUB>]Cl<SUB>3</SUB> at both post transformation time points is shown to have negligible effect on TrE. </CENTER></FONT></P><P><FONT size="2"><CENTER><B>Figure 11. Prediction profile modelling the interactions of transformation buffer (TB) constituents deemed to significantly affect overall transformation efficiency (TrE) after 16 hours of post-recovery incubation. </B>Reagents were selected based on their significance, with only reagents with individual P values &lt; 0.1 being selected for modelling. (A) Prediction profile set to most desirable TB composition. Black line indicates concentration relative to predictive TrE. Vertical dashed red line indicates the concentration of reagent at most desirable composition while horizontal dashed red line indicates predicted average TrE at desirable composition. Highlighted blue panels indicate a significant inhibitory effect on TrE. Highlighted green panels indicate significantly positive increase in TrE. Both MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O and RbCl have a positive interaction with increasing concentration (P = 0.0095). NiCl<SUB>2</SUB> and DMSO have a significant inhibitory interaction with individual increasing concentration (P = 0.0002 and P = 0.0479), however when both concentrations are increased they have a positive interaction with each other (P = 0.0054). (B) Interaction profile describing notable interactions between all significant reagents. Each panel represents an interaction between two reagents, with one reagent set at maximum concentration and the other at either lowest concentration (red line) or highest concentration (blue line). Panels positioned above or below the reagents in the graph are the reagents at maximum concentration, whereas panels positioned left or right are the reagents at either low or high concentrations. All other reagents included in the model but are not being examined in the panel are set to maximum.  </CENTER></FONT></P></DIV></DIV></SECTION><SECTION id="Media" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Variation in Media Composition</H1><H3 class="subhead">Variation in expression levels within and between batches of undefined media</H3></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  The InterLab study is an opportunity for crowdsourcing a large amount of data on the inherent variability of a given biological system. Here we examined how expression of GFP in the test devices was affected across different batches of Lysogeny Broth (LB) (A-C). </FONT></P><P><FONT size="3"> Growth curves were performed using LB A-C (Figures 12A and B) to evaluate variability between batches and within each individual batch. Interestingly, the variation within each batch was greater than the variation between batches, suggesting that the differences in composition of yeast extract and tryptone can be accounted for by having a large enough sample size. However, for comparing expression between studies, total standard deviation is considerably higher. As such, differences in expression may be sufficient to affect the accuracy of comparisons of part performance across the literature. </FONT></P><P><FONT size="2"><CENTER><B>Figure 12. A) fluorescence per OD<SUB>600</SUB> for recombinant <I>E. coli</I> DH5α </B> expressing the iGEM InterLab positive control device GFP gene E0040 in three batches (designated A, B and C) of LB broth made with tryptone and yeast extracts from different batches and different manufacturers with standard deviations (shaded areas); B: standard deviation of fluorescence per OD between each batch and total standard deviation across the whole dataset. Dotted vertical line represents the end of exponential growth phase and start of stationary phase. </CENTER></FONT></P></DIV></DIV><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Defined rich media</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">  Development of a defined media in which variability may be reduced would benefit efforts to attain greater reproducibility and standardisation. Towards this end, we modelled how components of a defined media affected <I>E. coli</I> growth. Growth in defined rich media was assessed and the experimental execution was performed using the OT-2 pipetting robot (Opentrons) (python code for OT-2 protocol available <A href="https://github.com/jbird1223/Newcastle-iGEM/tree/master/OT-2%20Protocol" class="black">here</A>). After 10 hours, growth was recorded in several runs, with growth in ten runs reaching stationary phase after 24 hours (Figure 13). </FONT></P><P><FONT size="2"><CENTER><B>Figure 13. Growth curves for <I>E. coli</I> DH5α in rich defined media DoE runs over 22 hours.</B></CENTER></FONT></P><P><FONT size="3"> The OD<SUB>600</SUB> values at 24 hours were used to build a Projection to Latent Structures (or Partial Least Squares (PLS)) model using JMP software. This allowed visualisation of factor effects on <I>E. coli</I> growth (Figure 15). It is observed that there are positive effects on growth of increasing citrate, thiamine and tricine concentration, while there are negative effects of increasing K2HPO4. Additionally, four interactions between components in the media were shown to be important predictors of DH5α growth. These are interactions between citrate and tricine (VIP = 1.45), thiamine and EDTA (VIP = 1.27), thiamine and tricine (VIP = 1.20) and citrate and thiamine (VIP = 1.11). The media composition predicted by the model to maximise DH5α growth is shown in Figure 14. </FONT></P><P><FONT size="2"><CENTER><B>Figure 14. Prediction profiler for the model of factors affecting growth of <I>E. coli</I> DH5α in a defined rich medium after 24 hours.</B> The vertical red dashed line represents the factor concentration or level in the medium, changing which alters the horizontal red dashed line, representing culture OD<SUB>600</SUB>. Statistically significant factors highlighted in green were citrate, thiamine, tricine and KH<SUB>2</SUB>HPO<SUB>4</SUB> (VIP = 2.39, VIP = 2.14 and VIP = 1.99, VIP = 1.04 respectively). Factor levels are set to max desirability, shown in red text beneath each x-axis (i.e. levels producing the maximum OD<SUB>600</SUB>). The model was constructed using JMP Pro v.12 statistical software (SAS Institute Inc.), based on experimental results of a definitive screening based experimental design. </CENTER></FONT></P><P><FONT size="3"> From the results here, and from the previously shown membrane permeability altering characteristics of EDTA, it is likely that interactions would be observed with other media components which require active transport into the cell. As such, for the further development of a rich defined media for recombinant expression in DH5α, and with the aim of reducing variability, it would be recommended that EDTA and citrate be omitted - the model produced here predicts good growth is possible without these components. Tricine on the other hand appears to be an excellent buffer for resisting the decreasing media pH as a result of metabolism and inclusion of which is recommended in further development of this media. </FONT></P></DIV></DIV></SECTION><SECTION id="Conclusions" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Conclusions </H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3"> The improvement of measurements not only requires the optimisation of the device but also of the ability to standardise the process and reduce variation in test environment. We focused on the optimisation of several InterLab protocols as well as the addition of an internal standard. By modelling <I>E. coli</I> Dh5α growth and transformation we have identified several important factors to optimise reliability and standardise the protocols. By the addition of an internal standard to each test device we can further standardise the results and allow a more accurate comparisons between constructs.</FONT></P></DIV></DIV></SECTION></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">InterLab</H3><H1 class="display-2">References and Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References and Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="2">1. Beal J, Haddock-Angelli T, Gershater M, de Mora K, Liarazo M, Hollenhorst J, Rettberg R, I GEMISC. (2016) Reproducbility of Fluorescent Expression from Engineered Biological Constructs in E. coli. PLOS one. DOI:10.1371/journal.pone.0150182</FONT></P><P class="about-para"><FONT size="2">2. Beal J, Haddock-Angelli T,Baldwin G, Gershater M, Dwijayanti A, Storch M, de Mora K, Lizarazo M, Rettberg R, iGEM interlab study contributers. (2018) Quantification of bacterial Fluorescence using independent calibrants. PLOS one. DOI:10.1371/journal.pone.0199432</FONT></P><P class="about-para"><FONT size="2">3. Densmore DM, Bhatia S (2014) Bio-design automation: software + biology + robots. Trends in Biotechnology. 32:111-113</FONT></P><P class="about-para"><FONT size="2">4. Decoene T, De Paepe B, Maertens J, coussement P, Peters G, De Maeseneire SL, De Mey M (2018) Standardisation in synthetic biology: an engineering discipline coming of ages. Critical Reviews in Biotechnology. 38:647-656</FONT></P><P class="about-para"><FONT size="2">5. Hecht A, Filiben J, Munro SA, Salit M (2018) Reproducing bench-scale cell growth and productivity. bioRxiv. DOI:10.1101/302786</FONT></P><P class="about-para"><FONT size="2">6.  Carbonell-Ballestero M, García-Ramallo E, Montañez R, Rodríguez-Caso C, Macía J (2015) Dealing with the genetic load in bacterial synthetic biology circuits: convergences with the Ohm's law. Nucleic acids research 44:496-507 </FONT></P><P class="about-para"><FONT size="2">7. Vind J, Sørensen MA, Rasmussen MD, Pedersen S (1993) Synthesis of proteins in Escherichia coli is limited by the concentration of free ribosomes: expression from reporter genes does not always reflect functional mRNA levels. Journal of molecular biology 231:678-688</FONT></P><P class="about-para"><FONT size="2">8. Qian Y, Huang H-H, Jiménez JI, Del Vecchio D (2017) Resource competition shapes the response of genetic circuits. ACS synthetic biology 6:1263-127</FONT></P><P class="about-para"><FONT size="2">15. Barbier M, Damron HF (2016) Rainbow Vectors for Broad-Range Bacterial Fluorescence Labeling. PLoS One 11:(3)</FONT></P><P class="about-para"><FONT size="2">10. Levin-Karp A, Barenholz U, Bareia T, Dayagi M, Zelcbuch L, Antonovsky N, Noor E, Milo R (2013) Quantifying translational coupling in E. coli synthetic operons using RBS modulation and fluorescent reporters. ACS synthetic biology 2:327-336 </FONT></P><P class="about-para"><FONT size="2">11. Beilharz K, van Raaphorst R, Kjos M, Veening J-W (2015) Red fluorescent proteins for gene expression and protein localization studies in Streptococcus pneumoniae and efficient transformation with Gibson assembled DNA. Applied and environmental microbiology:AEM. 02033-02015 </FONT></P><P class="about-para"><FONT size="2">12. Hostettler L, Grundy L, Käser-Pébernard S, Wicky C, Schafer WR, Glauser DA (2017) The bright fluorescent protein mNeonGreen facilitates protein expression analysis in vivo. G3: Genes, Genomes, Genetics:g3. 116.038133 </FONT></P><P class="about-para"><FONT size="2">13. Heppert JK, Dickinson DJ, Pani AM, Higgins CD, Steward A, Ahringer J, Kuhn JR, Goldstein B (2016) Comparative assessment of fluorescent proteins for in vivo imaging in an animal model system. Molecular biology of the cell 27:3385-3394 </FONT></P><P class="about-para"><FONT size="2">14. Chung CT, Niemela SL, Miller TH (1989) One-step preparation of competent Escherichia coli transformation and storage of bacterial cells in the same solution. Proceedings of the national academy of sciences. 86:2172-2175</FONT></P><P class="about-para"><FONT size="2">15. Klebe RJ, Harris JV, Sharp ZD, Douglas MG (1983) A general method for polyethyleneglycol-induced genetic transformation of bacteria and yeast. Gene. 25:333-341</FONT></P><P class="about-para"><FONT size="2">16. Zhiming T, Guangyuan H, Kexiu XL, Mingjie JC, Junli C, Ling C, Qing Y, Dongping PL, Huan Y, Jiantao S et al. (2005) An improved system for competent cell preparation and high efficiency plasmid transformation using different Escherichia coli strains. Electronic Journal of Biotechnology. 8:114-120</FONT></P><P class="about-para"><FONT size="2"><STRONG>Attributions: Kyle Stanforth, Sam Went &amp; Matthew Burridge
</STRONG></FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>