<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Notebook_Operon skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Notebook/Operon</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots/Notebook</TITLE><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/9/93/T--Newcastle--OperonNotebookCover.png" data-natural-width="3000" data-natural-height="2200" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Naringenin Operon
                    </H1><DIV class="home-content__buttons"><A href="#notebook" class="smoothscroll btn btn--stroke">
                        Read Notebook
                    </A></DIV></DIV><DIV class="home-content__scroll"><A href="#notebook" class="scroll-link smoothscroll"><SPAN>Scroll Down</SPAN></A></DIV></DIV></SECTION><SECTION id="notebook" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">NOTEBOOK</H3><H1 class="display-2">Naringenin Operon</H1></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing - 06/08/18</H3><P> A four-gene operon coding for naringenin biosynthesis was designed using Benchling, this was based on the parts used by the 2014 TU Darmstadt iGEM team. For synthesis by IDT, manual codon optimisation was required for the TAL gene to lower the G/C content. Following this, 4 gBlocks were ordered from IDT. These gBlocks and the primers designed for detection of the assembled operon arrived, therefore lab work could commence. The pSB1C3 backbone for naringenin biosynthesis was amplified using PCR. As the plasmid came at a stock concentration of 25 ng/µl and the PCR reaction requires between 1 pg – 1 ng the plasmid was diluted by X50. A negative control with 1 µl H<SUB>2</SUB>O instead of plasmid DNA was made up. An annealing temperature of 71 °C was used, determined by the NEB TM Calculator. Conditions for PCR are shown in the table below. Agarose gel electrophoresis was used for detection of the 2 kb amplified band, this was then purified using a QIAquick PCR Purification Kit, however, it resulted in a very low concentration of DNA around 3.58 µg/ml. Therefore, it was planned to conduct this PCR again to increase the concentration.
An overnight culture of DH5α cells was inoculated to make more competent cells for subsequent transformations. These cells were grown in a 37 °C shaking incubator until the OD<SUB> reaches 0.2 – 0.3, followed by additions of 75 mM MgCl<SUB>, 75 mM CaCl<SUB> and 15 % glycerol to induce competency.

                    </SUB></SUB></SUB></P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing - 13/08/18</H3><P>A positive control for Gibson assembly was undertaken by assembling 2 overlapping dsDNA fragments and the backbone pUC19 control DNA provided by NEB, using the NEBuilder HiFi DNA Assembly Cloning Kit. This was used to ensure that Gibson assembly using the master mix worked, and that the cells had been made competent. This positive control reaction was then transformed into the competent cells made earlier in the week using NEBuilder HiFi DNA Assembly Transformation Protocol. These cells underwent heat shock before being spread onto LB plates containing ampicillin as the positive control plasmid carried resistance to ampicillin. All 3 of the positive control plates had growth of DH5α cells, the negative control plate had no growth of DH5α cells, as they contained no plasmid and were not ampicillin resistant. This showed that the Gibson master mix was effective and the competent cells had been successfully made competent.
				    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing - 20/08/18</H3><P>gBlocks and the primers for the detection of the gBlocks were resuspended following instructions by IDT. Gibson assembly of the gBlocks and the pSB1C3 backbone was conducted using the volumes calculated through NEB: the vector should be 0.05 pmol and each insert between 0.1 – 0.2 pmol so that the total does not exceed 0.5 pmol. Competent cells were then transformed using the reaction mix and plated onto LB agar containing chloramphenicol instead of previously used ampicillin, as pSB1C3 has chloramphenicol resistance. Following overnight incubation, no colonies were observed on either the test plates of the negative control plates.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing- 27/08/18</H3><P>A second attempt at Gibson assembly was conducted and the transformation protocol was performed the same as previously. No colony growth resulted from this. It was considered that the backbone concentration was too low, which could be limiting transformation. More of the backbone was amplified and purified to a stock concentration of 26.2 µg/ ml.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing- 3/09/18</H3><P>A third attempt of Gibson assembly was done using the increased backbone concentration resulting in a total reaction volume of 36.3 µl. This was then transformed into DH5α cells resulting in 1 colony on each plate where  200 µl of transformation mix had been plated. However, these colonies only showed that the pSB1C3 backbone had been take up by the cells due to the observed chloramphenicol resistance. In order to test for the correct naringenin operon assembly, colony PCR was undertaken. This utilised primers 4CLF and CHSR to amplify from the start and end of the operon with 66 °C annealing temperature. This reaction should amplify the full operon if it has been inserted into the backbone correctly, this would result in a PCR product of ~5 kb. A 3 minute extension time was required for amplification the large fragment. Following analysis of PCR products by agarose gel electrophoresis, no 5 kb band could be seen on the gel. It was considered that no band was observed due to inherent difficulties with amplifying large PCR products. As an alternative means of analysis, overnight cultures of selected colonies were prepared to undergo plasmid extraction by miniprep. Plasmids were extracted from overnight cultures of two colonies using Qiagen QIAprep Plasmid Miniprep Kit. Extracted plasmids were analysed by agarose gel electrophoresis where a band of ~7 kb was expected if the Gibson assembly had been successful.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing- 10/09/18</H3><P>Following agarose gel electrophoresis, the only bands present for multiple samples of the two colonies were at 2 kb showing only the uptake of the plasmid from which the pSB1C3 had been amplified. Next, Gibson assembly was conducted again, however, instead of having an 18 µl reaction, the mixture was spun down in a SpeedVac 120 for 5 minutes to increase the DNA concentration. This was performed so that the Gibson assembly reaction volume could be reduced to the recommended 20 µl. 5 µl of the Gibson assembly was used to transform DH5α cells, this was increased from the 2 µl used previously. However, following overnight incubation, there was no colony growth on any of the plates. Primers were designed for the amplification of the gBlocks and ordered from IDT so that the parts could be amplified for use in additional Gibson assemblies.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing- 17/09/18</H3><P>The new primers for gBlock amplification arrived and were used to amplify the gBlocks by PCR. A gradient PCR was attempted to try a range of annealing temperatures and a 1 minute extension time was used. Analysis by agarose gel electrophoresis showed that gBlocks 3 and 4 and the pSB1C3 backbone amplified well resulting in bright bands at the expected sizes. However, the amplification of gBlock 1 resulted in two bands on the agarose gel and this raised concerns over contamination. gBlock 2 had not successfully amplified, and no band was observed on the agarose gel. To troubleshoot this, a gradient PCR was attempted across a range of different annealing temperatures, this resulted in 8 reactions (Table 4). Reaction 3 of the gBlock 2 amplification showed the clearest band, therefore this was purified and the DNA concentration quantified which determined a concentration of 51.8 µg/ml. Gel extraction was performed on the other bands amplified from gBlock 2, as there was uncertainty as to which band was correct. Two bands were obtained for gBlock 1, so both of these were gel extracted to recover the PCR product. The stock concentrations of all purified fragments are shown in Table 5. A series of Gibson assembly reactions were performed using different combinations of the amplified products. Reaction mixes were used to transform commercial TOP10 chemically competent cells (Thermo Fisher Scientific) and between 30-60 colonies grew on each plate following overnight incubation on selective medium.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing- 24/09/18</H3><P>Overnight cultures for 3 colonies from each of the 4 plates were grown and plasmids were extracted from the resultant cultures. Plasmid extractions were analysed by agarose gel electrophoresis (Figure 4). Only the backbone had been taken up by the cells. An additional 10 more colonies were analysed in the same way. An additional Gibson assembly was also attempted and two transformations of commercial TOP10 <I>E. coli</I> were carried out, one using 2 µl of reaction mix and one with 5 µl, each reaction used 50 µl of cells. Unfortunately, these attempts did not yield any colonies. The plasmids extracted from the additional 10 colonies were ran on agarose gel, however only 2 kb bands were observed. An additional 10 colonies were analysed in the same way. Agarose gel electrophoresis showed a promising band at 7 kb for the miniprep of colony 19 (Figure 5), which was the correct size of the plasmid and the operon. In order to reduce the observed level of background resulting from the plasmid backbone, the PCR amplification of the pSB1C3 backbone was treated with DpnI (NEB) and purified to a stock concentration of 9.84 µg/ml. A subsequent attempt at Gibson assembly was then attempted using 6.53 µl of the DpnI treated backbone. To check the results of the miniprep of colony 19, PCR was performed using the primers 4CLF and CHSR in an attempt to amplify the 5 kb operon. Additional PCR was performed using primers for the detection of each gBlock (Table 6).
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing- 1/10/18</H3><P>All PCR products were analysed by agarose gel electrophoresis, however, no bands were observed on the gel. Despite this, colony 19 was sent for sequencing as the plasmid was of the expected size (Eurofins Genomics). This plasmid was also used to transform <I>E. coli</I> BL21(DE3) cells for potential future expression analysis. In concordance with this, the Gibson assembly with the DpnI treated backbone was used to transform TOP10 and DH5α commercial <I>E. coli</I> cells. This resulted in lots of colonies on the DH5α cell plates and less than 5 on each TOP10 plate. Colonies 21-25 from the Gibson assemblies using variants of gel extracted and amplified gBlocks were transferred to overnight cultures, alongside colonies 27-34 transformations with the DpnI treated Gibson assembly mix. Minipreps were performed on overnight cultures of colonies 21-34, and one band was observed at 7 kb for colony 23 when analysed by agarose gel electrophoresis. This was subsequently sent for sequencing. Plasmids extracted from a further 10 colonies were analysed, however, all were found to contain the template plasmid from the pSB1C3 amplification. Sequencing data for the two plasmids sequenced did not provide sufficient coverage to confirm the correct assembly of the naringenin operon into the pSB1C3 backbone.
                    </P></DIV></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References &amp; Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="2"><STRONG>Attributions: Heather Bottomley and Patricya Ubysz
</STRONG></FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>