<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Results_Chemotaxis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Results/Chemotaxis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/c/c3/T--Newcastle--ChemotaxisCoverPhoto.png" data-natural-width="3000" data-natural-height="2000" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Exploring Bacterial Chemotaxis
                    </H1><DIV class="home-content__buttons"><A href="#Intro" class="smoothscroll btn btn--stroke">
         Introduction
        </A><A href="#GrowthOnAgar" class="smoothscroll  btn btn--stroke">
         Colony Morphology
        </A><A href="#GrowthinLiquidMedia" class="smoothscroll btn btn--stroke">
         Growth in LB
        </A><A href="#NaringeninMIC" class="smoothscroll btn btn--stroke">
         Naringenin &amp; Growth
        </A><A href="#CapillaryMethod" class="smoothscroll btn btn--stroke">
         Quantification
        </A><A href="#Microscopy" class="smoothscroll btn btn--stroke">
         Microscopy
        </A><A href="#AgarChemotaxis" class="smoothscroll btn btn--stroke">
         Chemotaxis on Agar
        </A><A href="#Conclusions" class="smoothscroll btn btn--stroke">
         Conclusions
        </A></DIV></DIV></DIV></SECTION><SECTION id="Intro" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Naringenin Chemotaxis</H1></DIV></DIV><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">An Introduction</H3><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">We examined how three species of free-living nitrogen-fixing bacteria respond to the presence of the flavonoid naringenin. The three species,  <I>Azorhizobium caulinodans</I>  strain ORS571,  <I>Azospirillum brasilense</I>  strain SP245, and  <I>Herbaspirillum seropedicae</I>  strain Z67, were selected because they all have potential to form different types of interactions with plant roots. <I>A. caulinodans</I>  has been shown to fix nitrogen both as a free-living microbe and when in symbiosis with the semi-aquatic leguminous tree <I>Sesbania rostrata</I> (1).  <I>H. seropedicae</I>  is a root endophyte and has shown potential to colonise popular crops such as wheat and maize (2).   </P></DIV></DIV><SECTION id="team" class="s-services"><SECTION id="GrowthOnAgar" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Bacteria Characterisation</H1></DIV></DIV><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Colony Morphology</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">Before commencing chemotaxis studies, we needed to understand the growth characteristics of the three free-living nitrogen-fixing bacteria to be used in our project. We first examined the colony morphology of these three species in the absence of any chemoattractants. Familiarisation with the bacteria allows identification of abnormal behaviour and contamination. For colony morphology, the size after a minimum of 24 hours and morphology (shape and pigmentation) were recorded (Table 1, Figure 1).  </P><FONT size="2">Table 1. Qualitative analysis of <I>Azorhizobium caulinodans</I>, <I>Azospirillum brasilense</I>, and <I>Herbaspirillum seropedicae</I> colonies grown on solid media.</FONT><TABLE id="protocols"><THEAD><TR><TH>Species (Strain)</TH><TH>Colony Pigmentation</TH><TH>Colony Morphology</TH><TH>Points of Interest</TH></TR></THEAD><TBODY><TR><TD><I>Azorhizobium caulinodans</I> (ORS571)</TD><TD>White</TD><TD>Regular form, Typically raised, Entire margin</TD><TD>Colonies rarely grow to a measurable size when grown at 30 ˚C on Yeast Extract Broth agar after 24 hours</TD></TR><TR><TD><I>Azospirillum brasilense</I> (SP245)</TD><TD>Orange/Pink</TD><TD>Non-slimy, Regular and round form, Entire margins</TD><TD>Both immature and dead colonies lack the orange/pink pigment, Colonies wrinkle with age</TD></TR><TR><TD><I>Herbaspirillum seropedicae</I> (Z67)</TD><TD>Cream/Light Green</TD><TD>Circular or Irregular form (occasionally rhizoid), Raised elevation, Shiny</TD><TD>Colonies took on a different morphology depending on how the media was inoculated; stab-inoculation lead to rhizoid form while spreading leads to circular/irregular form</TD></TR></TBODY></TABLE><P style="font-size:medium"><I>A. caulinodans</I> (Figure 1a): Colonies do not grow to a measurable size within 24 hours at 30 ˚C on Yeast Extract Broth agar. Colonies contain white pigmentation and are raised in elevation with an entire margin – a continuous, uninterrupted border of the colony. Colonies rarely grow larger than 2 mm whilst smaller colonies, which are much more numerous, could not be accurately measured.  </P><P style="font-size:medium"><I>A. brasilense</I> (Figure 1b): Colonies are distinguishable by their distinctive orange/pink pigmentation, though both immature and dead colonies lack this pigmentation. Older colonies became ingrained into the agar, making them hard to remove without damaging the agar. Older colonies also began to wrinkle with time. The average diameter for a colony of this species after 24 hours incubation at 37 ˚C on LB agar was 3 mm, making <I>A. brasilense</I> the fastest growing of our nitrogen-fixing bacteria. Young <I>A. brasilense</I> colonies were shiny, round and with entire margins. These young colonies may have some pigmentation near the centre as the colony matures. This is in contrast to older colonies which maintain a different phenotype; losing their shine and gaining the odd wrinkle. Wrinkling often leads to the loss of the round shape. </P><P style="font-size:medium"><I>H. seropedicae</I> (Figure 1c): the colonies take different forms depending on how the plate is inoculated. If the plate is stab-inoculated, the colony takes a rhizoid appearance (Figure 1d). If the culture is spread across the plate, then it typically takes a circular or irregular form (Figure 1c). Colonies possess a green-cream pigmentation and are raised from the surface. Most colonies were shiny and typically 1.5 mm in diameter after 24 hours at 30 ˚C. </P><FONT size="2">Figure 1. Observations of bacterial preservation plates. a) <I>A. caulinodans</I> colonies grown on 1 % Yeast Extract Broth agar after incubation at 30 °C for 56 hours. Plates were inoculated via streaking. b) <I>A. brasilense</I> colonies grown on 1 % LB after incubation at 37 °C for 16 hours. Plate inoculated via streaking. c) <I>H. seropedicae</I> colonies showing circular growth on 1 % LB agar after incubation at 30 °C for 24 hours. Plates were inoculated via streaking. d) <I>H. seropedicae</I> colonies showing rhizoid growth on 1 % LB agar after incubation at 30 °C for 24 hours. Plates were stab-inoculated.  </FONT></DIV></DIV><SECTION id="GrowthinLiquidMedia" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Growth Rates in Liquid Media</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">From initial iterations of our <A href="https://2018.igem.org/Team:Newcastle/Modelling/Community" class="black">community model</A>, it became apparent that quantitative data on the growth rates of the bacteria were required in order to inform the model. For this, we observed changes in absorbance at 600 nm over 72 hours of the three nitrogen-fixing bacteria and <I>E. coli</I> in liquid culture at 30 °C using a ThermoFisher Scientific Varioskan LUX Microplate Reader (Figure 2).</P><P style="font-size:medium">The data showed that <I>A. brasilense</I> grew at a slow, steady rate before sharply dying off after approximately 60 hours. The slow growth rate is likely to be because its optimal growth temperature is 37 °C rather than 30 °C. <I>H. seropedicae</I> and <I>A. caulinodans</I> showed very similar growth curves when grown at 30 °C: initial growth rate was very fast and then growth became very slow or static after 20 hours. <I>E. coli</I> grew at a medium pace to begin with and steadily slowed down with time. </P><FONT size="2">Figure 2. Growth curves showing changes in absorbance at 600 nm of <I>E. coli</I>, <I>A. caulinodans</I>, <I>H. seropedicae,</I> and <I> A. brasilense</I> in LB at 30 °C for 70 hours. n=4 replicates, error bars indicate standard error of the mean.</FONT></DIV></DIV></DIV></SECTION><SECTION id="NaringeninMIC" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Effect of Naringenin on Growth Rate in Liquid Culture</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">Initial research for the Alternative Roots project noted that naringenin possesses antimicrobial properties, particularly towards <I>E. coli</I> (3). As <I>E. coli</I> DH5α was to be used as both a control in our chemotaxis assays and as the organism in which our <A href="https://2018.igem.org/Team:Newcastle/Results/Operon" class="black">naringenin biosynthesis operon</A> would first be assembled, it was deemed important to characterise the effect of increasing naringenin concentrations on growth rates of both our free-living nitrogen-fixing bacteria, and <I>E. coli</I> in LB medium. This was essential to guide the chemotaxis assays enabling an understanding of naringenin concentrations which would not have detrimental impacts upon the cell. If cell health is impaired, then there is potential that cell death may lead to results similar to chemorepulsion. This is particularly problematic when applying the response index as a semi-quantitative measure of chemotactic response as the method utilises ratios between colony edges to determine the significance of chemotaxis (4).</P><FONT size="2">Figure 3. Optical density at 600 nm wavelength of 4 bacterial species (<I>A. brasilense</I>, <I>A.  caulinodans</I>, <I>H. seropedicae</I>, and <I>E. coli</I>) after 24 hours of growth when grown in liquid media (LB) containing different concentrations of naringenin. n=3 replicates, error bars indicate standard error of the mean. </FONT><P style="font-size:medium">All species successfully grew in the presence of 0-150 μM naringenin (Figure 3). However, it was noted that when the concentration of naringenin exceeded 100 μM, the amount of error also increases. This suggests that naringenin begins to have a greater impact on some, but not all, bacteria in the solution. As such, naringenin concentrations of &lt;100 μM were used as part of subsequent chemotaxis assays to avoid negatively impacting bacterial growth.  </P></DIV></DIV></DIV></SECTION><SECTION id="CapillaryMethod" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Characterising Chemotactic Behaviour</H1></DIV></DIV><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Quantification Utilising Capillaries</H3></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">To characterise chemotactic behaviour in response to naringenin, a quantitative approach is desirable. This allows for direct comparison of the strength of the response between different species. Results from a quantitative assay would also be better suited for our <A href="https://2018.igem.org/Team:Newcastle/Modelling/Community" class="black">community model</A> as it allows a ranking of bacterial responses to naringenin.  </P><FONT size="2">Table 2: Colony forming units of four bacterial species from capillaries containing 1 µl 100 µM naringenin or chemotaxis buffer solution (10 mM potassium phosphate, 0.1 mM EDTA, 10 mM glucose, pH 7.0) after 45 minutes open-end submersion in sterile conditions at room temperature/pressure. Values are mean cfu.μl<SUP>-1</SUP>. Difference between colony counts from capillaries containing naringenin or chemotaxis buffer was non-significant for all species (P &gt; 0.05). </FONT><TABLE id="protocols"><THEAD><TR><TH>Species (Strain)</TH><TH>Colony Count (Naringenin)</TH><TH>± Standard Error</TH><TH>Colony Count (Control)</TH><TH>± Standard Error</TH><TH>Significant Difference</TH></TR></THEAD><TBODY><TR><TD><I>A. caulinodans</I> (ORS571)</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>No</TD></TR><TR><TD><I>A. brasilense</I> (SP245)</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>No</TD></TR><TR><TD><I>H. seropedicae</I> (Z67)</TD><TD>458</TD><TD>86</TD><TD>376</TD><TD>236</TD><TD>No</TD></TR><TR><TD><I>E. coli</I> (DH5α)</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>No</TD></TR></TBODY></TABLE><P style="font-size:medium">After 24 hours incubation at either 30 °C (<I>A. caulinodans</I> and <I>H. seropedicae</I>) or 37 °C (<I>A. brasilense</I> and <I>E. coli</I>), the number of colonies which grew on the LB agar plate was counted (Table 2). The results showed that of the four test bacterial species, only one was able to move into the capillary. This species was <I>H. seropedicae</I> which was able to move successfully into capillaries containing either the control (buffer solution) or the chemoattractant. This was demonstrated by the growth of colonies on LB agar from the contents of each capillary (Figure 4). Both methods of agar inoculation (spreading and pipetteing) lead to colony growth.</P><P style="font-size:medium">After counting colonies from the contents of both the control and naringenin capillaries, no significant difference between mean colony count of the two conditions was observed. The results therefore show no evidence for positive chemotaxis using this method. It should be considered, however, that <I>H. seropedicae</I> was the only species that demonstrated growth on agar, and therefore the only one able to enter the capillaries. We concluded that this methodology is not yet sufficiently optimised for our application and may be having a confounding effect upon chemotactic response. </P><FONT size="2">Figure 4. a) Growth of <I>H. seropedicae</I> on Typtone and Yeast Extract agar inoculated with contents of a 1 µl capillary containing 100 µM naringenin after 45 minutes open-end submersion in bacterial solution. Plate was incubated for 24 hours at 30 °C. b) Growth of <I>H. seropedicae</I> on 1 % LB agar inoculated with contents of a 1 µl capillary containing chemotaxis buffer after 45 minutes open-end submersion in bacterial solution. Plates were inoculated via streaking technique and  incubated for 24 hours at 30 °C.
    </FONT></DIV></DIV></SECTION></DIV></SECTION><SECTION id="Microscopy" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Microscopy Observations</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium"> An alternative method of observing chemotactic responses is through the use of microscopy. Brightfield microscopy allows direct observations of bacterial responses. This will allow comparisons of motility and morphology from our experimental data to that of the published literature that was used to underpin our first iteration of the <A href="https://2018.igem.org/Team:Newcastle/Modelling/Community" class="black">community model</A>. Using microscopy enables the development of a cell density:optical density index (CD:OD index), a method of converting the two values. This index was also used in the community model to adapt the growth curve data collected during bacterial characterisation in standard laboratory conditions.</P><P style="font-size:medium">The CD:OD index was produced utilising data collected from a haemocytometer. A haemocytometer is a specialised microscopy slide of a known volume, it also contains a grid at the centre. By counting the number of cells in 16 squares at the top left  and performing a series of <A href="https://2018.igem.org/Team:Newcastle/Protocols" class="black">mathematical calculations</A>, we were able to determine cell density. By utilising a spectrophotometer, we were also able to take a reading of the optical density (600 nm) and thus link the two together (Table 3).</P><P style="font-size:medium"><I>A. brasilense</I> was unable to be counted accurately. This was as the bacteria was difficult to view under the microscope due to human limitations, in addition to highly variable optical density readings but consistently low cell densities despite more than sufficient incubation times. This happened throughout all replicates and as such results for this species have been omitted. Exploration of why this occurred shall be work for the future with the potential alternative approach of utilising flow cytometry. </P><FONT size="2">Table 3: Cell density (cells.ml<SUP>-1</SUP>) of  <I>A.  caulinodans</I>, <I>H. seropedicae</I> and <I>E. coli</I> at different optical densities</FONT><TABLE id="protocols"><THEAD><TR><TH>Species (Strain)</TH><TH>Optical Density</TH><TH>Cell Density</TH></TR></THEAD><TBODY><TR><TD><I>A. caulinodans</I> (ORS571)</TD><TD>1.00</TD><TD>3.14x10<SUP>5</SUP></TD></TR><TR><TD><I>H. seropedicae</I> (Z67)</TD><TD>0.05</TD><TD>1.05x10<SUP>6</SUP></TD></TR><TR><TD><I>E. coli</I> (DH5α)</TD><TD>1.00</TD><TD>7.4x10<SUP>8</SUP></TD></TR></TBODY></TABLE><P style="font-size:medium">Utilising a haemocytometer to count cells also allows observations of cell morphology and behaviour. This was used to an advantage as it allowed us to explore whether our bacteria’s morphology aligns with the literature that was followed to produce the community model. It also enabled observations of motility which was important after theories that the species were no longer motile which is why only <I>H. seropedicae</I> showed movement into the capillary during the attempt to quantify chemotaxis. </P><FONT size="2">Table 4: Microscopy observations of diameter (µm), cell length (µm), and motility in <I>A. brasilense</I>, <I>A.  caulinodans</I>, and <I>H. seropedicae</I> on a haemocytometer at 40x objective compared to information in utilised literature.</FONT><TABLE id="protocols"><THEAD><TR><TH>Species (Strain)</TH><TH>Length in Literature</TH><TH>Mean Diameter</TH><TH>Cell Length in Literature</TH><TH>Mean Cell Length</TH><TH>Motile (Y/N)</TH></TR></THEAD><TBODY><TR><TD><I>A. caulinodans</I> (ORS571)</TD><TD>1.5-2.5(5)</TD><TD>1.9</TD><TD>0.5-0.6 (5)</TD><TD>0.5</TD><TD>No</TD></TR><TR><TD><I>A. brasilense</I> (SP245)</TD><TD>2.1-3.8 (6)</TD><TD>2.0</TD><TD>1.0 (6)</TD><TD>0.7</TD><TD>Yes</TD></TR><TR><TD><I>H. seropedicae</I> (Z67)</TD><TD>1.5-5 (7)</TD><TD>2.4</TD><TD>0.7 (7)</TD><TD>0.7</TD><TD>Yes (Highly)</TD></TR></TBODY></TABLE><P style="font-size:medium">While completing microscopy evaluations, we utilised the ibidi µ-Slide III 3-in-1 Chemotaxis Microscopy Slide provided by our sponsors at ibidi. This specialised microscopy slide was designed to allow real time observations of chemotactic behaviour in response to an adjustable chemical gradient. Through working with experts at ibidi, we were able to successfully seed all 4 species of our bacteria onto the uncoated, hydrophobic variant of the slide. This would allow us to begin quantifying naringenin chemotaxis in a modern and easily repeatable manner. Continued work with ibidi in the future demonstrates the exciting potential to quantify naringenin to further improve the design of our community model in the future. More information on ibidi and the µ-Slide III 3-in-1 Chemotaxis Microscopy Slide can be found <A href="https://ibidi.com/channel-slides/55--slide-iii-3in1.html" class="black">here.</A></P></DIV></DIV></DIV></SECTION><SECTION id="AgarChemotaxis" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Chemotaxis on Agar</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">Our third approach to understanding bacterial chemotaxis involved both qualitative and semi-quantitative analysis. Growth of the nitrogen-fixing bacteria on solid media was observed to understand chemotactic behaviour in response to naringenin. </P><P style="font-size:medium">Multiple different variants of agar assays were conducted to optimise the methodology.</P><P style="font-size:medium"><B><U>Method 1:</U></B></P><P style="font-size:medium">The original method utilised 0.75 % LB agar plates with 10 μl of 200 μM naringenin applied to one side of the plate and a sterile water control to the other. Bacteria were inoculated into the centre of the plate and it was hypothesised that colony growth would be distorted towards the side that contained naringenin, in the case of positive chemotactic behaviour. This assay was conducted with <I>A. brasilense</I> and <I>E. coli</I>. Neither bacterium showed a growth response favouring either the side with naringenin or the control. </P><P style="font-size:medium">From the results in this iteration, several key elements were identified that were incorporated moving forward. For example, as bacterial growth exhibits inherent variability, a qualitative assay may not be sufficient to identify differences in behaviour in response to different treatments. As such, a more quantitative approach was adopted for future assays. Another issue noted was that there is the potential that the agar percentage was too high resulting in poor diffusion through the medium. This may also have impacted the bacteria’s ability to move and grow towards the naringenin source.</P><P style="font-size:medium"><B><U>Method 2:</U></B></P><P style="font-size:medium">The second iteration of agar assays reduced the agar concentration to 0.5 % and the naringenin concentration to 100 μM to align with the findings of the impact of naringenin on growth rate. The plate was also laid out in a more quantifiable manner. This followed concerns of the chemoattractant diffusing onto the side of the control when on the same plate. In this method, the distance of bacterial growth towards the naringenin/control source was measured (Table 5). </P><FONT size="2">Table 5: Mean distance of colony growth towards either naringenin or control source of <I>A. brasilense</I>, <I>A.  caulinodans</I>, <I>H. seropedicae</I> and <I>E. coli</I> measured from the point of inoculation after 24 hours incubation. Distance is given in mm.</FONT></DIV><TABLE id="protocols"><THEAD><TR><TH>Species (Strain)</TH><TH>Growth Distance Towards Naringenin</TH><TH>Growth Distance Towards Control</TH></TR></THEAD><TBODY><TR><TD><I>A. caulinodans</I> (ORS571)</TD><TD>3.70</TD><TD>4.01</TD></TR><TR><TD><I>A. brasilense</I> (SP245)</TD><TD>7.32</TD><TD>7.10</TD></TR><TR><TD><I>H. seropedicae</I> (Z67)</TD><TD>5.87</TD><TD>5.66</TD></TR><TR><TD><I>E. coli</I> (DH5α)</TD><TD>8.33</TD><TD>7.67</TD></TR></TBODY></TABLE><P style="font-size:medium">Once again, no evidence of chemotaxis towards naringenin was observed in any species. It was noted that the ‘halo’ around <I>H. seropedicae</I> colonies on plates containing naringenin were constricted and more closely situated to the colony margin. Through consulting the literature, it was revealed that high concentrations of naringenin can repress genes involved in chemotactic behaviour in this species (8). This may provide an explanation as to why chemotaxis was not observed in this species.</P><P style="font-size:medium"><B><U>Method 3:</U></B></P><P style="font-size:medium">The third and final iteration of agar assays was based on the gradient plate experiment used by Reyes-Darias et al. (2016) (9). In this variant, 0.25 % Minimal A Salt agar was utilised and the naringenin concentration was further reduced to 50 μM. The concentration gradients were also left for 16 hours at 4 ˚C in order to form instead of 12 hours at room temperature. Initially, bacterial species were inoculated at different distances from the centre line where the naringenin or control was added; this interval increased by 5 mm until 40 mm. After analysing initial results, the inoculation distance was changed to reflect that which gave the best response index. The control was also altered to 1.5 % (v/v) ethanol as the method of dissolving naringenin was changed to be within the same percentage.</P><P style="font-size:medium">The response index, developed by Pham and Parkinson (4), accounts for a ratio between the edge of the colony nearest the chemoattractant source and the edge furthest from the same source. This ratio is then used to determine if there has been positive chemotaxis (RI &gt; 0.52), no effect (RI = 0.48-0.52) or negative chemotaxis (RI &lt; 0.48).</P><P style="font-size:medium">Results (Table 6) indicated that both <I>A. brasilense</I> and <I>H. seropedicae</I> experienced positive chemotaxis towards 50 μM between distances of 5-25 mm and 5-10 mm respectively. As such, further investigation utilised the distance that corresponded with the greatest RI value (15 mm and 10 mm respectively). For <I>H. seropedicae</I>, the colonies nearer the centre line again showed more constricted halos which may indicate that the naringenin concentration may still be too high. The response index of the control for all species at 5 mm was &lt; 0.48, suggesting chemorepulsion. This was anticipated as the control contains ethanol which possesses known antimicrobial properties and is commonly used to disinfect lab equipment.</P><FONT size="2">Table 6: Average Response Index and standard error of <I>A.  caulinodans</I>, <I>A. brasilense</I>, <I>H. seropedicae</I> and <I>E. coli</I> colonies grown on 0.25 % Minimal A Salt agar containing a gradient of either 50 µM naringenin or 1.5 % ethanol (control). RI = D1/(D1+D2) in which D1 represents distance between colony edge nearest chemical source to site of inoculation whilst D2 represents distance between colony edge furthest from chemical source to site of inoculation (4). Bacteria were inoculated 15 mm (<I>A. brasilense</I> and <I>E. coli</I>) or 10 mm (<I>A. caulinodans</I> and <I>H. seropedicae</I>) from naringenin source and incubated at 30 ˚C.</FONT><TABLE id="protocols"><THEAD><TR><TH>Species (Strain)</TH><TH>Naringenin Response Index</TH><TH>Control Response Index</TH><TH>Chemotactic Response</TH></TR></THEAD><TBODY><TR><TD><I>A. caulinodans</I> (ORS571)</TD><TD>0.478</TD><TD>0.473</TD><TD>Negative</TD></TR><TR><TD><I>A. brasilense</I> (SP245)</TD><TD>0.552</TD><TD>0.515</TD><TD>Positive</TD></TR><TR><TD><I>H. seropedicae</I> (Z67)</TD><TD>0.625</TD><TD>0.5</TD><TD>Positive</TD></TR><TR><TD><I>E. coli</I> (DH5α)</TD><TD>0.472</TD><TD>0.493</TD><TD>Negative</TD></TR></TBODY></TABLE><P style="font-size:medium">The response index for <I>E. coli</I> and <I>A. caulinodans</I> indicated negative chemotaxis in response to naringenin. This may be due to the fact that the naringenin was dissolved in 1.5 % ethanol which is commonly used to to sterilise due to ethanol's antimicrobial properties. As the other two species of nitrogen-fixers were demonstrated to show chemoattraction and both of which were motile, unlike <I>A. caulinodans</I>, it may be possible that the loss of motility combined with the antimicrobial properties of ethanol are triggering this result. This would mean the experimental set-up was not appropriate and thus requires further work. This will be work for the future.</P></DIV></DIV></SECTION><SECTION id="Conclusions" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Conclusions</H1></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P style="font-size:medium">After successfully characterising how <I>A. brasilense</I>, <I>A.  caulinodans</I>, <I>H. seropedicae</I> and <I>E. coli</I> behave in a laboratory environment through means of understanding colonies and growth rates, we began to explore bacterial chemotaxis toward naringenin. While it may be possible to observe this behaviour in a quantitative fashion via microscopy or microfluidic methods. These methods, from the data gathered from these series of experiments, require a higher level of optimisation than semi-quantitative based methods. </P><P style="font-size:medium">Importantly, we were able to successfully demonstrate chemotaxis of <I>A. brasilense</I> and <I>H. seropedicae</I> toward 50 µM naringenin, these results came from semi-quantitative agar-based assays. While no evidence for chemotaxis was demonstrated in <I>A. caulinodans</I>, it may be possible to do so in the future with aforementioned optimisation.</P></DIV></DIV></DIV></SECTION></SECTION></DIV></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="2"><STRONG>Attributions: Connor Trotter and Sadiya Quazi</STRONG></FONT></P><P class="about-para"><FONT size="2">1.	Liu W, et al. (2017) Azorhizobium caulinodans Transmembrane Chemoreceptor TlpA1 Involved in Host Colonization and Nodulation on Roots and Stems. Frontiers in Microbiology 8:1327.</FONT></P><P class="about-para"><FONT size="2">2.	Pedrosa FO, et al. (2011) Genome of Herbaspirillum seropedicae Strain SmR1, a Specialized Diazotrophic Endophyte of Tropical Grasses. PLoS Genetics 7(5):e1002064. 261.</FONT></P><P class="about-para"><FONT size="2">3.	Lee K-A, Moon SH, Kim K-T, Mendonca AF, &amp; Paik H-D (2010) Antimicrobial Effects of Various Flavonoids on Escherichia coli O157:H7 Cell Growth and Lipopolysaccharide Production. Food Science and Biotechnology 19(1):257.</FONT></P><P class="about-para"><FONT size="2">4.	Pham HT &amp; Parkinson JS (2011) Phenol Sensing by Escherichia coli Chemoreceptors: a Nonclassical Mechanism. Journal of Bacteriology 193(23):6597-6604.</FONT></P><P class="about-para"><FONT size="2">5.	Dreyfus BG, JL; Gillis, M (1988) Characterization of Azorhizobium caulinodans gen. nov., sp. nov., a Stem-Nodulating Nitrogen-Fixing Bacterium Isolated from Sesbania rostrata. International Journal of Systematic Bacteriology 38:89-98.</FONT></P><P class="about-para"><FONT size="2">6.	Tarrand JJ, Kried NR, Doebereiner J (1978) A Taxonomic Study of the Spirillum lipoferum Group, with Descriptions of a New Genus, Azospirillum gen. nov. and Two Species, Azospirillum lipoferum (Beijerinck) comb. nov. and Azospirillum brasilense sp. nov. Canadian Journal of Microbiology 24: 967-980.</FONT></P><P class="about-para"><FONT size="2">7.	Baldani JI, Baldani VLD, Seldin L, Doebereiner J (1986) Characterization of Herbaspirillum seropedicae gen. nov., sp. nov., a Root-Associated Nitrogen-Fixing Bacterium International Journal of Systematic and Evolutionary Microbiology 36: 86-93, doi: 10.1099/00207713-36-1-86.</FONT></P><P class="about-para"><FONT size="2">8.	Tadra-Sfeir MZ, et al. (2015) Genome Wide Transcriptional Profiling of Herbaspirillum seropedicae SmR1 Grown in the Presence of Naringenin. Frontiers in Microbiology 6:491.</FONT></P><P class="about-para"><FONT size="2">9. Reyes-Darias JA, García V, Rico M, Corral-Lugo A, &amp; Krell T (2016) Identification and Characterization of Bacterial Chemoreceptors Using Quantitative Capillary and Gradient Plate Chemotaxis Assays. Bio-protocol 6(8):e1789.</FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>