<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Notebook_Endophytic_chassis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Notebook/Endophytic chassis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots/Notebook</TITLE><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/5/51/T--Newcastle--EndophyticChassisDNotebookCover.png" data-natural-width="3000" data-natural-height="2200" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Endophytic Chassis
                    </H1><DIV class="home-content__buttons"><A href="#notebook" class="smoothscroll btn btn--stroke">
                        Read Notebook
                    </A></DIV></DIV><DIV class="home-content__scroll"><A href="#notebook" class="scroll-link smoothscroll"><SPAN>Scroll Down</SPAN></A></DIV></DIV></SECTION><SECTION id="notebook" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">NOTEBOOK</H3><H1 class="display-2">Endophytic Chassis</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><H3 class="h2">Week Beginning 30/07</H3><P><FONT size="3">Development of our new endophytic chassis began on the 1st of August with the arrival of root colonising <I>Pseudomonas</I> sp. CT 364 (DSM25356 from DSMZ in Germany). The strain arrived in a glass ampoule and was revived onto tryptone soy agar (TSA) plates using methods outlined by DSMZ. The plates were incubated at 28 ℃ for 24 hours after which emergent colonies were used to inoculate liquid cultures grown in tryptone soy broth (TSB) for initial growth characterisation and antibiotic testing.</FONT></P><H3 class="h2">Week Beginning 06/08</H3><P><FONT size="3">In order to select transformants when introducing foreign DNA, we first needed to determine antibiotics active against <I>Pseudomonas</I> sp. We initially planned to transform <I>Pseudomonas</I> sp. with one of the InterLab test devices that had been modified to contain mNeonGreen instead of GFP. Creating a mutant <I>Pseudomonas</I> sp. with strong fluorescence would be useful for identification and visualisation when carrying out microscopy on roots inoculated with <I>Pseudomonas</I> sp. As the test devices are in a pSB1C3 backbone, activity of chloramphenicol against <I>Pseudomonas</I> sp. had to be characterised. To test activity, 200 µl of <I>Pseudomonas</I> sp. TSB overnight culture was spread onto TSA containing 50 and 100 µg/ml of chloramphenicol and plates were incubated at 28 ℃ for 24 hours. Following incubation, growth was observed on all plates indicating resistance to chloramphenicol at the concentrations tested.</FONT></P><H3 class="h2">Week Beginning 13/08</H3><P><FONT size="3">As <I>Pseudomonas</I> sp. was found to be resistant to chloramphenicol, a selection of four further antibiotics (kanamycin, gentamicin, streptomycin and carbenicillin) were tested for activity against <I>Pseudomonas</I> sp. 20 ml of stock solution was prepared for each of the antibiotics at concentrations show in Table 1 and water or an ethanol solution was used as the solvent. Stocks were distributed into 1 ml aliquots and frozen at -20 °C.</FONT></P><P><FONT size="3"><CENTER>Table 1. Antibiotic concentration and solvent of stock solutions prepared.</CENTER></FONT></P><P><FONT size="3">TSA plates were prepared containing 50 µg/ml and 100 µg/ml of each antibiotic for screening of activity against <I>Pseudomonas</I> sp. These two concentrations were either equal to or above working concentrations suggested by Addgene. Two replicates were prepared at both concentrations for each of the four antibiotics. 200 µl of overnight culture was spread onto each plate which were subsequently incubated overnight at 28 °C. Following incubation, growth inhibition was only observed on plates supplemented with streptomycin and gentamicin.</FONT></P><H3 class="h2">Week Beginning 27/08</H3><P><FONT size="3">After identifying streptomycin and gentamicin as active against <I>Pseudomonas</I> sp. the next step was to characterise the susceptibility to each antibiotic by identifying the minimum inhibitory concentration (MIC) for each antibiotic. To do this, a range of concentrations were tested in 96-well plate format to produce a ‘kill curve’. To improve the accuracy of these experiments, the Opentrons OT-2 liquid handling robot (won by the Newcastle iGEM 2018 team) was used for experimental set-up. To do this, bespoke python code for the robot had to be designed specific to these experiments. Much of this week was spent outside of the lab working on the code. The code was designed in order to distribute liquid from 4 different sources into a 96-well plate. These four sources were sterile tryptone soy broth, <I>Pseudomonas</I> sp. liquid culture, antibiotic solution and sterile distilled water. The code used a list function to define the volume of each of the four solutions distributed into each well. These experiments also utilised the universal racks designed, 3D printed and assembled by the team originally for carrying out heat-shock transformations using the OT-2.</FONT></P><H3 class="h2">Week Beginning 03/09</H3><P><FONT size="3">This week started with the first testing of our python code for the MIC experiments. An initial problem encountered with the code was that when using the 10 µl tip the robot was not completely dispensing all the liquid out of the tip and subsequently dripping into other wells causing contamination. This problem was fixed by adding a blow-out function to the code, this pushed air through the tip after the initial distribution to remove any remaining fluid. A second problem encountered was that the working volume of the 20 ml universals used to hold each of the four solutions was not sufficient for the volume of sterile broth required. As the OT-2 pipette returns to the same position for each aspiration rather aspirating from a lower position as the solution is used up the working volume of the universal is limited by the height of the pipette tip. This equated to a working volume of approximately 8 ml, we therefore added two more universal containers to the labware to hold sterile broth. The perimeter wells were not used for the assay and were each filled with 200 µl of sterile broth due to reduced accuracy when measuring absorbance in these wells. Experiments were performed in triplicate for each antibiotic at a range of three concentrations. The assays used an antibiotic stock concentration of 1.0 mg/ml, 100 µg/ml and 10 µg/ml. Suitable working concentrations in liquid culture were identified for each antibiotic. </FONT></P><H3 class="h2">Week Beginning 10/09</H3><P><FONT size="3">Following on from successful characterisation of the MICs in both liquid and solid culture for each antibiotic, transformations were attempted. Addgene plasmid number 79813 was selected for introduction into <I>Pseudomonas</I> sp. This plasmid was selected as it was the only plasmid found that was isolated from a <I>Pseudomonas</I> species and contained either a gentamicin or a streptomycin resistance gene. The plasmid arrived from Addgene in <I>E. coli</I> and was streaked onto LB agar containing streptomycin (50 µg/ml). Liquid cultures were incubated overnight at 37 °C and two samples of the plasmid were isolated from <I>E. coli</I> using a Qiagen QIAprep Spin Miniprep Kit. The DNA concentration of each sample was then quantified using a Qubit fluorometer. Each sample was then split into two aliquots and one aliquot from each sample was concentrated using a DNA Speedvac resulting in four samples with a range of DNA concentrations.</FONT></P><P><FONT size="3">Electroporation was chosen as the first method of transformation to attempt as it was the most documented method for <I>Pseudomonas</I> sp. Electrocompetent cells were made by  washing an overnight culture of <I>Pseudomonas</I> sp. cells with 300 mM sucrose solution. Electroporation was performed with three different DNA concentrations along with a control containing only sterile water. Electroporated cells were resuspended in TSB and incubated at 28 °C for 3 hours before spreading onto TSA containing gentamicin (10 µg/ml).</FONT></P><P><FONT size="3">During this week the team also designed a streptomycin resistance gene that could be used in <I>Pseudomonas</I> sp. The design consisted of an aadA gene under the control of a strong Anderson promoter and strong RBS, a native terminator new to the iGEM registry was also included to prevent any readthrough.</FONT></P><H3 class="h2">Week Beginning 17/09</H3><P><FONT size="3">The streptomycin resistance gBlock (IDT) was assembled into a pSB1C3 backbone by Gibson assembly. The reaction mix contained 8.09 µl of DNA solution and an equal volume of Gibson assembly master mix. A positive control reaction mix was also prepared containing 10 µl of DNA and 10 µl of Gibson assembly master mix. Each sample was incubated at 50 °C for 60 minutes on a heat block and stored at room temperature until transformation.</FONT></P><P><FONT size="3">Prior to transformation, <I>E. coli</I> DH5α cells were tested for susceptibility to streptomycin. LB agar plates containing streptomycin at concentrations of 10, 50 and 100 µg/ml of streptomycin and a control plate containing no antibiotic were prepared. Each plate was inoculated with overnight culture and incubated at 37 °C for 20 hours.</FONT></P><P><FONT size="3">Heat shock transformations were performed using commercial chemically competent <I>E. coli</I> DH5α cells. Three separate heat shock reactions were carried out, the positive control (consisting of a standard NEBuilder reaction mix containing a backbone and and two gBlocks (IDT)), the streptomycin resistance gene assembly, and a negative control containing only molecular grade water and competent cells. </FONT></P><H3 class="h2">Week Beginning 24/09</H3><P><FONT size="3">The assembled plasmid containing the streptomycin gene was isolated from <I>E. coli</I> DH5α cells again using a Qiagen QIAprep Spin Miniprep Kit. The resultant plasmid was sent for sequencing (Eurofins Genomics) using primers verification forward (VF2) and verification reverse (VR). Sequence data confirmed that the assembly had been successful.</FONT></P><P><FONT size="3">The streptomycin resistant part was characterised by comparing growth rates of the transformant and wild-type <I>E. coli</I> at a range of streptomycin concentrations on a 96-well plate. Serial dilutions were performed from wells containing streptomycin (64 µg/ml) and inoculated with transformant and wild-type overnight culture. The plate was subsequently incubated at 37 °C for 24 hours with a slow double orbital shake during which absorbance was measured at 600 nm. The transformant was found to be resistant to streptomycin at the highest concentration tested (64 µg/m).</FONT></P><H3 class="h2">Week Beginning 01/10</H3><P><FONT size="3">The gentamicin resistant transformant <I>Pseudomonas</I> sp. was characterised to quantify the strength of resistance. Transformant growth rates were compared to wild-type <I>Pseudomonas</I> sp. at gentamicin concentrations of 0.0 µg/ml, 5.0 µg/ml and 50 µg/ml on a 96-well plate. The plate was subsequently incubated at 28 °C for 24 hours with a slow double orbital shake during which absorbance was measured at 600 nm.</FONT></P></DIV></DIV><SECTION id="references" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References &amp; Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="3"><B>Attributions: Frank Eardley and Lewis Tomlinson</B></FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>