<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots</TITLE><SECTION id="world" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/2/29/T--Newcastle--ImprovePage.png" data-natural-width="3000" data-natural-height="2000" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Improve </H1><DIV class="home-content__buttons"><A href="#overview" class="smoothscroll btn btn--stroke">
                        Overview
                    </A></DIV></DIV></DIV></SECTION><SECTION id="overview" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Improved InterLab measurement plasmids</H1></DIV></DIV><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Inclusion of an RFP internal standard into the 2018 InterLab test device pSB1C3 vectors.</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">Our RFP internal standard (IS) was developed to address the issue that copy number of the same plasmid is not consistent across cells. The inclusion of an IS RFP signal is therefore designed to allow measurement of variation in gene expression between cultures and transformant lines. The underpinning assumption is that because the IS devices are identical across each of the plasmids, the GFP fluorescence values of the test devices of interest can be reported relative to their internal RFP signal. RFP was cloned into the non-coding region between the chloramphenicol resistance gene and the ORI. The new plasmids were tested and analysis of the internal standards involved comparing the original InterLab test device plasmid performance against the new internal standard plasmid performance.</FONT></P><P><FONT size="3">Results show that inclusion of the IS altered the behaviour of the test devices (Figure 1). The IS repressed expression of GFP from all devices and affected expression rates - particularly in devices with strong promoters. Further, the IS signal varied across plasmids though the only difference was the strength of the test device. The data generated using this approach indicates competition for cellular resources between the IS and test device. As a result, measurements and conclusions regarding the strength of strong reporters should be treated with caution.</FONT></P><P><CENTER><B>Figure 1. A) Scatter plots showing the relationship between Relative AFUs (GFPmut3b fluorescein/OD) against time (x-axis) for the original test device and the internal standard test devices &amp; B)Scatter plots showing the relationship between Relative AFUs (RFP fluorescence/OD) (y-axis) and time (x-axis) for the RFP internal standard test devices.</B> A)The legend for each device is shown on the left of each graph. Over 24 hours, the original test devices reach peak fluorescence at around the 6-7 hour mark, with devices 4 &amp; 1 being the strongest and 3 &amp; 6 being the weakest. After the 6 hour mark there is a decrease in fluorescence to the 22 hour mark where fluorescence beings to increase once again. The internal standard devices all showed a consistent increase with no decrease over 24 hours. Device 2 was the most fluorescent with devices 1 and 4 being significantly lower than the original group. Fluorescence was lower for each device initially than the original but eventually surpassed the original study as the fluorescence decreased over time. 
B&gt; Over the 24 period, the control groups and devices 1 &amp; 4 exhibited a fluorescence/OD of 0. The remaining other devices - 2, 3 &amp; 6 – all showed a consistent positive increase in fluorescence/OD. Device 2 however has the highest fluorescence/OD throughout the study, followed by devices 6, 3 and 2 with fluorescence/OD values of 0.064, 0.048 and 0.038 respectively. These groups began fluorescing between 200 and 440 min.</CENTER></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797013" class="black">BBa_K2797013</A></FONT></P><P style="text-align:center"><FONT size="3"><A href="https://2018.igem.org/Team:Newcastle/Measurement" class="black">Measurement</A></FONT></P></DIV></DIV></DIV><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Comparisons between the existing GFPmut3b test devices and a new series of mNeon green test devices</H3></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P><FONT size="3">From the literature, we gathered information on the potential problems associated with the GFPmut3b reporter – notably its low photostability upon exposure to natural light [1]. Replacing the reporter to prevent such problems became an option. However, since GFPmut3b is the brightest GFP variant, it would make sense to utilise a suitably divergent fluorescent protein with similar fluorescence characteristics to GFPmut3b but a higher photostability – mNeonGreen met these requirements. The data indicated that mNeonGreen was not significantly brighter than mut3GFP as was proposed. However, the values for both replicate colonies show that the spread of fluorescence/OD<SUB>600</SUB> values for the mNeonGreen reporter is lower in each test device group.</FONT></P><P><FONT size="3">The results of this experiment therefore do not support the original goal of the study. The mNeonGreen reporter, despite reports in the literature, was not seen to be significantly different to mut3GFP in its fluorescence/OD<SUB>600</SUB>. The mNeonGreen did, however, show a smaller range of fluorescein/OD data in comparison to GFPmut3b. Since there were no differences in fluorescence values, the smaller range of values from the mNeonGreen reporter may be due to its better photostability, or a difference in maturation time in vivo than the literature suggests. Regardless, the switch to mNeonGreen in InterLab test devices may further improve measurement reliability.</FONT></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797003" class="black">mNeonGreen coding sequence</A></FONT></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797004" class="black">mNeonGreen Test Device 1 </A></FONT></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797005" class="black">mNeonGreen Test Device 2 </A></FONT></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797006" class="black">mNeonGreen Test Device 3 </A></FONT></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797007" class="black">mNeonGreen Test Device 4 </A></FONT></P><P style="text-align:center"><FONT size="3"><A href="http://parts.igem.org/Part:BBa_K2797008" class="black">mNeonGreen Positive Control</A></FONT></P><P style="text-align:center"><FONT size="3"><A href="https://2018.igem.org/Team:Newcastle/Measurement" class="black">Measurement</A></FONT></P></DIV></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References &amp; Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="2">1. Levin-Karp A, Barenholz U, Bareia T, Dayagi M, Zelcbuch L, Antonovsky N, Noor E, Milo R (2013) Quantifying translational coupling in E. coli synthetic operons using RBS modulation and fluorescent reporters. ACS synthetic biology 2:327-336</FONT></P><P class="about-para"><FONT size="2"><STRONG>Attributions: Kyle Stanforth
</STRONG></FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>