<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots/Notebook</TITLE><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/5/5e/T--Newcastle--ComputerUSBPic.jpeg" data-natural-width="3000" data-natural-height="2200" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Notebook
                    </H1></DIV></DIV></SECTION><SECTION id="notebook" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Follow the Newcastle iGEM team on their journey</H1></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 1 - 07/06/18</H3><P>We started the day with introductory talks from iGEM alumni explaining iGEM and the wiki, we then went onto look at previous 
                    competition entries. Jem Stach and Phil Wright spoke about their experience in the field, research interests 
                    and what makes a good competition entry.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 2 - 08/06/18</H3><P>The day began with a talk from Jon Marles-Wright about the importance of human practices and the impacts of 
                    our projects in the real world. This was followed by Dana Ofiteru who taught us about mathematical modelling of 
                    biological systems. In the afternoon, plant biologist Max Kapralov discussed using synthetic biology to improve 
                    the efficiency of photosynthesis. Finally, Rachel Armstrong (professor of experimental architecture) discussed 
                    the potential of microorganism in waste management systems and energy production in the built environment. Rachel 
                    also outlined political and economic issues surrounding synthetic biology.
				    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 3 - 11/06/18</H3><P>We started the day with a talk from Angel Goni Moreno who discussed the principles of synthetic biology and provided 
                    guidance on the implementation of logic in Biology. We then had some time to discuss our own ideas and come up with new ideas as a team. Following this, some Synthetic Biology Masters students came to talk to us about the importance of modelling lab work. Each member of the team gave a presentation on a previous competition 
                    entry that has won an award so we could familarise ourselves with the components of a successful iGEM project and explore some exciting science!
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 4 - 12/06/18</H3><P>We started the day with a talk from Alice banks and Colette Whitfield. Followed by ideas discussion with Tom, to get some feedback on 
                    our original ideas and what expertise/equipment may be available to us. Then Nick Aldred, a marine biologist/engineer, talked to us about boat anti-fouling coatings. Finally we had a
                     talk from Vas Andriotis who focused on plant biology (corn yield). Finishing with another ideas session to refine our ideas.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 5 - 13/06/18</H3><P>The day began with a talk from Martyn Dade-Robertson who discussed the use of biotechnology in architecture and his research
                     into pressure sensing <I>E.coli</I>. In the afternoon Harold Freemann led a discussion around the use of mathematical modelling and its
                      uses in synthetic biology. Finally Alice Banks and Colette Whitfield covered current topics in synthetic biology and the 
                      equipment available to us in the laboratory.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 6 - 14/06/18</H3><P>This was our first day in which we were given freedom to explore our ideas and gain a necessary background surrounding 
                    the topics in order to refine into 6 ideas that will be pitched to research supervisors and the guest speakers who have 
                    given presentations since the project began.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 7 - 15/06/18</H3><P>We continued to explore our ideas for the majority of the day, narrowing our proposals down from ten ideas to six. 
                    We worked more as individuals, researching specific aspects of the proposals.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 8 - 18/06/18</H3><P>Today was more orientated around the ethical implications of Synthetic Biology. We started with a discussion with 
                    Ilke Turkendag, a lecturer in Law Innovation &amp; Society. We discussed our six proposals and outlined possible ethical/legal 
                    issues which could arise. This was followed by an interesting talk from Simon Woods and Ken Taylor, they spoke about potential 
                    stakeholders and how we could go about talking/communicating with them. They also took each of our ideas and looked at the 
                    potential impacts our projects could have on the wider public. We then finished the day preparing for our six pitches for Tuesday.
					The Opentrons OT-2 robot, which the Newcastle iGEM team won, was also delivered today<SUP>1</SUP>.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 9 - 19/06/18</H3><P>We pitched the 6 ideas that we selected to the instructors and masters students. After receiving feedback and suggestions
                     from the advisory team we narrowed it down to 2 ideas to be taken forward. Further, we constructed a template for both idea 
                     presentations and allocated team members to each idea. 
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 10/11 - 21/06/18</H3><P>Following on from the 2 ideas that we chose to take forward, we spent these next 2 days researching further into these proposals. 
                    During our research, we took a holistic approach when evaluating our ideas and attempted to account for topics such as human practices, 
                    ethical implications and possible art and design ideas. We incorporated this into our 2 ideas which we then pitched to the instructors 
                  and Master's students. We also managed to successfully setup the OT-2 robot. The smiles speak for themselves...
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 12 - 22/06/18</H3><P>This day was used to carefully consider both ideas that were pitched the previous day before voting on one to take forward. 
                    We started by summarising each idea in 140 characters which highlighted some discrepancies in individual team members understanding 
                    of the ideas. After these discrepancies were rectified, the team spent some time considering each idea before voting on one to take 
                    up as our project. In the end, it was decided that the project involving engineering microbes to colonise plant roots and attract 
                    nitrogen-fixing bacteria was the best to take forward. We spent some time thinking about the modelling/human practices/lab work that 
                    may be involved in our project.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Day 13 - 25/06/18</H3><P>On our first official day of project-specific work, various tasks were allocated to team members for them to research. 
                    Some background reading was conducted on the topic, the team name was confirmed and some ideas for the team logo were suggested. 
                    Also, some ideas for a hydroponics system were discussed. With a rare bit of sunshine, we decided to work outdoors.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 25/06/18</H3><P>On the Monday, members of the team were provided an overview of the plant growth facilities at Newcastle University by 
                    Vas Andriotis who has shown great interest in our project since its pitch and has given many resources to enhance 
                    and develop our ideas. The remainder of the group began on the research for the several experiments that will be ran in 
                    parallel over the Summer. 

					Tuesday was dedicated to further fleshing out our research by identifying species which would be suitable for our iGEM project. 
					Some members moved in an artistic direction and produced a handful of potential logos before the entire team voted on our two 
					favourites. Once these were decided, merchandising began development. 

					Development on our hydroponics system moved into full swing on the Wednesday, with our 2 engineers beginning work on the general 
					structure of the system which has been influenced by conversations with plant biologists within the university; working once again 
					with Vas in order to understand his needs from the system. 

					Thursday presented our team with the incredible opportunity to speak with Steven Burgess who is currently performing plant research 
					at the University of Illinois in order to discuss our project and identify potential for sharing our work among the wider 
					scientific community. 

					Members of the Biology and Engineering teams went to meet with Dr Montero-Calasanz; a Microbiologist whom specialises in plant-microbe 
					interactions. This gave a valuable opportunity to identify potential protocols which would relate to our chassis organism. The week was 
					rounded off by talks from Alice Banks and Collette Whitfield concerning sponsorship and outreach.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 02/07/18</H3><P>The team focused on preparation for the wetware projects by ordering the <I>Pseudomonas</I> chassis, in addition to various free-living
					nitrogen-fixing bacteria. A variety of flavonoid chemoattractants were also ordered in order to reduce the wait time before lab work begins.
					The human practices and outreach team continued getting their work by in touch with potential sponsors. Some of the team members with prior 
					experience in programming began working with the OT-2 to design a heat-shock protocol for <I>E. coli</I> using the python-based language within the
					Opentrons API.
					The whole team received inductions for the Cat2 lab so that we can begin the wetware side of the project and follow all the safety protocols
					and rules and regulations set out by the University for safe lab use.
					After receiving the lab inductions, we prepared the lab equipment for testing <I>Arabidopsis</I> germination so that we can begin testing the following week.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 09/07/18</H3><P>The ball started to roll this week, we managed to get access to the labs and plant our first set of seeds in agar as well as 
                    preparing chemically competent <I> E. coli</I> cells. We began the process of styling our wiki using bootstraps and designing the poster 
                    for the European Meetup. Towards the end of the week we attended a modelling workshop, ran by Synthetic Biology Masters students,
                    this gave us an insight into modelling and tips we can apply to our modelling work. The team was also lucky enough to receive goodie bags
                    from Eppendorf!
We had previously decided that the naringenin flavonoid would most likely be the most successful, therefore looked into utilising parts from the naringenin operon previously used by TU Darmstadt 2014 iGEM team. We also Skyped Daniel Sachs, a member of the TU Darmstadt iGEM team to hear his advice on what worked well with the operon and with which promoters. Finally, we codon optimised one of the gblocks for the operon (TAL) and ordered all the segments with their corresponding RBS’s from IDT.<SUP>2</SUP>.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 16/07/2018</H3><P>This week was full of action. The architects in the team 3D printed a cold deck for the Opentrons OT-2 and laser cut accompanying trays to hold test tubes. Now that we have the cold deck we can manufacture trays to hold lots of different apparatus increasing the number and complexity of experiments we can carry out on the Opentrons. In addition, we had a meeting with Martyn Dade-Robertson from the School of Architecture, Planning &amp; Landscape; he spoke to us about the importance of finding a context in which to situate our human practices, once we have this it will provide the design parameters necessary to progress. Finally, stencils were made for the 'Alternative Roots' logo enabling us to tag any hardware we build. 

Some of the team also travelled to Munich this week for the European iGEM meet up. Those who attended throughly enjoyed
liasing with other iGEM teams as well as the event organisers and speakers. The team's poster was on display at the
meetup which gathered some interest around our project as well as many questions and suggestions for improvement which
have been duly noted by the team for further consideration. Potential collaborations were also sought after in Munich. 

Lab work for the chemotaxis assays commenced with trial runs using the DH5α <I>E. coli</I> control in order to test 
the protocol. It was noted from these attempt that the concentration of naringenin was too high as the plates 
did not show signs of growth for 2 days; after which, signs of growth were minimal. 
The concentration was diluted to 100µM and 50µM for future trials to see if this would overcome the issue. We also Designed forward and reverse primers for each of the four segments, for detection that the operon had been transformed.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 23/07/2018</H3><P>The long-awaited <I>Pseudomonas</I> sp. arrived this week, having come all the way from Germany, 
meaning our iGEM team could start preparation for the transformation process. The team worked with 
Dr Montero-Calasanz to make initial agar and liquid cultures in both LB and TS mediums and now 
have enough bacteria to start the initial transformation experiments.

This week the some of the parts for the hardware arrived meaning we could get underway with the 
construction of the hydroponics. First the basic circuit for the LED lights was created, the code 
we used to control these lights was an adaptation of a freely available library (FastLED) which can 
be found online. We made minor adjustments to suit our requirements, for example the 
light intensity was increased. Our micro-controller (Arduino), was then loaded with 
this edited code. This device will be responsible for monitoring and maintaining our entire 
system. We faced issues with timing the lights for a 16 hour day and 8 hour night cycle. 
We decided to use a ‘count’ within a loop to control when the LED’s need to be turned off. 
This meant we had to calibrate for the time a single loop took and multiply that up for 
16 hours. We are currently waiting for some parts to arrive so that we can begin construction of an
automated switching circuit.



The week was rounded off with a presentation to the stakeholders in the project, such as the PI's 
and instructors, to provide an update on how the project has progressed thus far. The presentation
also offered a valuable opportunity to highlight areas of the project that needed to be strengthened
and was a good opportunity to develop our presentation skills prior to the Giant Jamboree in Boston.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 30/07/2018</H3><P>This week we managed to get off campus and talk to potential stakeholders. We were lucky enough to arrange an interview with GrowModule 365. As a company, they produce shipping containers that contain hydroponics systems. There concept adopts a new approach to farming, allowing crop production in unconventional places. The visit allowed us to see how traditional farming is being challenged and gain an insight into hydroponics systems. We gained lots of useful information from the meeting; we learned which wavelengths are most effective for increased yields, the alternative growing mediums available, the markets perception to innovative farming and which control parameters are most important. Paul Brown (Director) gave us lots of useful contacts within the industry and seemed very passionate about our project, however he did mention some areas of the market may be sceptical when it comes to eating produce that has been grown using genetically engineered microbes.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 06/08/2018</H3><P>This week we started to work on a video for the opening of the presentation in Boston. We wanted to continue with the theme portrayed in a previous 'teaser' we produced. The hydroponics switching circuit was finalised by the engineers, this controlled the day and night cycle to simulate specific environments for the seeds. The Hydroponics system aesthetics were also worked on, we applied spray paint and stencils to make it more visually appealing. At the end of the week Matt Burridge (One of our team members) conducted a workshop to teach people how to write and run protocols on the Opentrons OT-2 robot.

In the lab we: amplified the backbone of pSBS1C3 through thermocycling using primers for the 2kb sequence required for assembly and this allowed the plasmid to be linearised and prepare for later insertion of the gblocks for the naringenin operon,
ran this PCR product using agarose gel electrophoresis to check for separation of the backbone required from the rest of the plasmid, purified the PCR product and quantified the DNA concentration thus making the backbone ready for Gibson Assembly.
Towards the end of the week amplified the positive control for Gibson assembly practice using the NEBuilder HiFi DNA Assembly Cloning Kit.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 13/08/2018</H3><P>  We were in the final stages of building the hydroponics system this week, we have determined optimal wavelengths to run the system at. Chris 3D printed a series of trays for seeds to fit in. We were also lucky enough to have a meeting with Chris Tapsell From KWS (the 4th largest seed company in the world). We outlined our project and discussed the issues surrounding GM crops and how the industry and wider public respond to GM use. Chris seemed to be passionate about the project and we intend to stay in contact with each other.

In the lab we made chemically competent DH5α <I>E. coli</I> cells for Gibson transformation through the use of repeated centrifugations and the reagents MgCl2, CaCl2 and Glycerol.
We also transformed these chemically competent cells with the positive control from Gibson assembly and found that the Gibson Assembly method was successful, as the positive control grew on ampicillin plates due to the DH5α cells taking up the positive control overlapping dsDNA fragments containing ampicillin resistance. No growth was observed on the negative control plates with un- transformed DH5α cells, as they are not naturally resistant to ampicillin.

                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 20/08/2018</H3><P>The week started with a visit to Scotland to meet the Edinburgh iGEM team. The visit was extremely useful as it allowed us to share our progress with Edinburgh and vice versa and we also gave each other feedback on our respective projects. Additionally, we discussed the potential for collaborations, ate lots of pizza and even got to see some of the shows in the Edinburgh Fringe festival. We thoroughly enjoyed our time in Edinburgh and meeting the Edinburgh team, who were very hospitable and we hope to maintain a close relationship with Edinburgh iGEM moving forward.

Two members of the team, Will and Umar, also travelled to London to meet with Richard Ballard, co-founder of Growing Underground. We were shown around their tunnels of hydroponics which grow approximately 33 metres below the busy streets of Clapham. Growing Underground sustainably grow fresh micro greens and salad leaves, which are unaffected by weather and seasonal changes, before providing these to wholesalers and local restaurants such as M&amp;S and Ocado. The meeting was very productive, we discussed how our project may be beneficial to the company and, also, how the company can help our project develop. We discussed the project at length and considered how the project might be refined to better meet the needs and concerns of a company such as Growing Underground. We hope to continue working with Richard and the Growing Underground team as we continue to develop the project.

This week in the lab, we resuspended the gblocks and primers from IDT. We also calculated the volumes of pSB1C3 and gblock inserts required for specific concentrations in the Gibson assembly and conducted a Gibson Assembly using the NEBuilder HiFi DNA Assembly Reaction Protocol. We then transformed the Gibson Assembly using the NEBuilder HiFi DNA Assembly Protocol, and then spread the reaction mixture onto chloramphenicol selection plates for overnight incubation. There was no colony growth on any of the plates therefore transformation was unsuccessful.

                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 27/08/2018</H3><P>On the chemotaxis side of things, we spent the week producing kill curves for each of our bacteria as to determine whether our concentration of naringenin was having an adverse effect on the growth of our bacteria. This was as we knew naringenin possessed antimicrobial properties thus it was important to work out if this was the reason that we found chemorepulsion for <I>E. coli</I> instead of no effect. During this week we also secured a sponsorship from ibidi who we are happy to say provided the team with their specialised μ-slideIII 3-in-1 chemotaxis slide which allowed us to visualise cell movement in response to naringenin via timelapse videos. It was also during this week that we reinvented our chemotaxis assay to be more in line with the work of Reyes-Darias et al. (2014) by reducing our agar percentage even further and utilising large, square plates in order to run multiple replicates and test distances on a single plate.

We also conducted a second attempt of Gibson with re – calculated values for the inserts. This was transformed as before and there was no growth on the plates. As this was thought to be due to the backbone concentration being far too low after purification, another PCR to linearise the backbone was conducted. This was ran on a gel and quantified to have a much higher concentration.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 03/09/2018</H3><P>This week we carried out our reinvented chemotaxis assays that involved using minimal agar on square plates, and setting up a concentration gradient of naringenin, and obtained results which showed positive chemotaxis of <I>Azospirillum brasilens</I> towards 100uM narinegenin at distances between 0.5cm and 2.5cm! The most successful result was when the bacteria was at a distance of 1.5cm, so we will carry out repeats of this distance.

Another Gibson attempt with the new backbone and the gblocks was done. Following transformation this reaction yielded 2 colonies showing that the plasmid with chloramphenicol resistance had been taken up. In order to see whether the rest of the operon had been integrated into the plasmid, colony PCR and plasmid miniprep was conducted on both colonies. However only the plasmid had been taken up without the operon.
                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 10/09/2018</H3><P>This week we were able to successfully transform <I>Pseudomonas sp.</I> with a gentamicin resistance gene by electroporation.

We also tested methods of sterilising plant tissue, such as using different concentrations of ethanol and bleach. Seeds and seedlings were inoculated with wild type <I>Pseudomonas</I> sp.

Performed square plate assays with <I>Herbaspirillum seropedicae</I> and <I>Azorhizobium caulinodans</I>, but results were inconclusive as colonies grew too large and overlapped. Repeats will be performed with bacteria inoculated at greater distances from each other.

As wiki freeze draws closer, we are trying to develop more pages of our wiki and keep on top of it to hopefully lighten the load later when the deadline becomes imminent.

                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 17/09/2018</H3><P>In the lab this week we carried out repeats of the square plate assays, and processed the results. <I>H. seropedicae</I> showed positive chemotaxis, whereas <I>A. caulidolens</I> showed no chemotaxis towards naringenin. We also learned to use the microscope to look at our nitrogen fixers in preparation for using the u-slides gifted by our sponsor ibidi.</P><P>We assembled a streptomycin resistance gblock into a pSB1C3 backbone by Gibson assembly, and successfully transformed <I>E. coli</I> DH5α with the streptomycin resistance gene via heat shock.</P><P>For the naringenin operon, 4 Gibson Assemblies were set up in parallel with different volumes of DNA. These were then transformed into commercial DH5 alpha cells. Between 30 – 60 colonies grew on all 4 plates.

                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 24/09/2018</H3><P>The streptomycin resistant part made last week was characterised by comparing growth rates of the transformant and wild-type <I>E. coli </I>DH5α at a range of streptomycin concentrations on a 96-well plate. </P><P>We were lucky enough to set up a stand at the European Researchers Night held at The Great North Museum this week. We explained the basic premise of our project as well as our Victoria Tunnel vision to members of the public. This was a chance to see how the public felt about the scheme, we also took the opportunity to ask how people felt about GMOs and if they felt our project was tackling the stigma surrounding them. </P><P>We also gave a talk to the new students in the School of Engineering. The talk was very useful in helping us create a narrative for the presentation in Boston. There was substantial interest in the hydroponics and the application of engineering in Synthetic Biology. </P><P>We conducted a miniprep of 20 colonies from overnight cultures from the cells transformed with the naringenin operon, however bands at 2kb were observed for all except colony 19 with a 7kb band matching the size of the operon and the plasmid. Carried out a PCR using primers for the start and end of the operon and reactions for each segment, to check if the operon has been taken up in colony 19. However the primers did not bind, but sent colony 19 off for sequencing. We treated the backbone with DPN1 to clear un-circularised backbone, and repeated Gibson assembly.


                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 1/10/2018</H3><P>This week we set up growth curves for Exeter iGEM team in order to help them understand how perchlorate affects soil bacteria. We incubated our 3 nitrogen fixers at 30°C in a 96 well plate with varying concentrations of perchlorate salt and recorded their growth using a plate reader.</P><P>The gentamycin resistant transformant <I>Pseudomonas</I> sp. was characterised to quantify the strength of resistance. </P><P>Mini-preps of a further 25 colonies showed a single colony with a 7kb band matching the combined operon and plasmid size, so we conducted a PCR of the genes within colony 23, which resulted in the primers not binding to the sequence. Result of the sequencing came back but were inconclusive for colony 19 and colony 23 was incorrect.



                    </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H3 class="h2">Week commencing: 08/10/2018</H3><P>As the Giant Jamboree draws close, we cut key-rings to begin preparation for the Giant Jamboree. </P><P>We have also been storyboarding our presentation for the final in Boston. To advertise our presentation / project we have produced a flyer to hand out at the jamboree. </P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><H5>Acknowledgements</H5><P>1. Credit: <A href="https://twitter.com/alice_banks/status/1008710700485799941" target="_blank" class="black">Dr. Alice Banks</A>
                       2. Credit: <A href="https://twitter.com/JoshuaLoh6/status/1009456142320328705" target="_blank" class="black">Joshua Loh</A></P></DIV></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References &amp; Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="2"><STRONG>Attributions: Everyone</STRONG></FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>