<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Results_Operon skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Results/Operon</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots/Design</TITLE><SECTION id="home" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/3/3b/T--Newcastle--heatherheader.png" data-natural-height="700px" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Naringenin Operon Assembly Results
                    </H1><DIV class="home-content__buttons"><A href="#introduction" class="smoothscroll btn btn--stroke">
                        Introduction
                    </A><A href="#experiment" class="smoothscroll btn btn--stroke">
                   Operon Design
                    </A><A href="#experiments" class="smoothscroll btn btn--stroke">
                   Experiments 
                    </A><A href="#conclusions" class="smoothscroll btn btn--stroke">
               Conclusions
                    </A></DIV></DIV></DIV></SECTION><SECTION id="introduction" class="s-about"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead subhead--dark">Results</H3><H1 class="display-1 display-1--light">Introduction</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para">Guided by the successful <A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis#Conclusions" class="white">chemotaxis results</A>, proving that 50 µM naringenin attracts the nitrogen-fixing bacteria <I>Azospirillum brasilense</I> and <I>Herbospirillum seropedicae</I>, we aimed to engineer a naturally colonising endophyte <I>Pseudomonas </I> sp. (CT 364) to produce naringenin. For proof of concept the production of naringenin would first need to be demonstrated in <I>E. coli</I> before being tested in <I>Pseudomonas </I>sp., our final chassis organism.</P><P class="about-para">Naringenin biosynthesis is achieved through the expression of an operon containing four genes encoding the enzymes that constitute the naringenin biosynthetic pathway (Figure 1). This operon was previously assembled and submitted to the iGEM Registry by the TU Darmstadt 2014 iGEM team <A href="http://parts.igem.org/Part:BBa_K1497016" class="white">BBa_K1497016</A> and is a composite of the following four genes, each with the strong ribosome binding site (RBS) <A href="http://parts.igem.org/Part:BBa_B0034" class="white">BBa_B0034</A>: 
<UL>4-Coumaryl ligase - 4CL <A href="http://parts.igem.org/Part:BBa_K1033001" class="white">BBa_K1033001</A></UL><UL>Tyrosine ammonia lyase - TAL <A href="http://parts.igem.org/Part:BBa_K1033000" class="white">BBa_K1033000</A></UL><UL>Chalcone isomerase - CHI <A href="http://parts.igem.org/Part:BBa_K1497000" class="white">BBa_K1497000</A></UL><UL>Chalcone synthase - CHS <A href="http://parts.igem.org/Part:BBa_K1497001" class="white">BBa_K1497001</A></UL></P><P style="text-align:center"> Figure 1. The naringenin synthesis pathway from L-tyrosine.</P><P class="about-para">We were required to optimise the sequence for the enzyme tyrosine ammonia lyase by reducing the G/C content so that it could be synthesised by IDT <A href="http://parts.igem.org/Part:BBa_K2797014" class="white">BBa_K2797014</A>. The entire operon was synthesised in four separate parts referred to as gBlocks that were subsequently used for Gibson assembly, as this was how TU Darmstadt assembled their operon successfully. Moreover this assembly does not leave restriction site scars due to the overlapping fragments. Instead of using <I>Pseudomonas </I> sp. it was deemed logical to use <I>E. coli</I> DH5α during the cloning experiments as we had a greater understanding of its transformation ability. Using this as a proof of concept would allow us to carry out a preliminary screening for naringenin production before attempting to transform our endophyte.</P><P class="about-para">Alongside this, in an attempt to optimise naringenin production, a new design of the naringenin operon was made in Benchling. This was guided by the <A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway" class="white">pathway modelling results</A> and was constructed to show how  BG28 and BG51 dual <I>E. coli-Pseudomonas</I> promoters with a 10-fold difference in strength could increase naringenin production (1).</P></DIV></DIV></SECTION><SECTION id="experiment" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Results</H3><H1 class="display-1">Operon and Plasmid Design</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><H3 class="subhead--dark">Plasmid Design</H3><DIV class="col-full"><P class="about-para"> The initial plasmid design for naringenin biosynthesis was based on TU Darmstadt’s design but with a codon optimised  tyrosine ammonia lyase. We used pSB1C3 as a backbone, into which we would clone  each of the four necessary genes downstream  of  a strong RBS (BBa_B0034). This construct  was  under the control of a  strong  Anderson  promoter (J23100) to  allow for constitutive expression of the operon (Figure 2 and Figure 4). Once biosynthesis under the control of J23100 is achieved,  future experiments will test this under the strong inducible T7 promoter in  <I>E. coli</I> (Figure 3 and Figure 5). Parts for biosynthesis in root-colonising  <I>Pseudomona</I>  sp.,  will  be implemented into a plasmid backbone more suitable for its uptake.</P></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 2. The naringenin biosynthetic operon under control of a J23100 promoter created in Benchling.</P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 3. The naringenin biosynthetic operon under control of a T7 promoter created in Benchling.</P></DIV></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 4. The naringenin biosynthetic operon construct under control of a J23100 promoter created in SBOL.</P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 5. The naringenin biosynthetic operon contruct under control of a T7 promoter created in SBOL.</P></DIV></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="service-text"><DIV class="col-full"><H3 class="h2">Naringenin pathway modelling influenced design</H3><P class="about-para"> Results of the naringenin pathway modelling demonstrated that weaker expression of the first two genes and 10-fold stronger expression of the last two genes would reduce the build-up of malonyl CoA, a pathway intermediate, and optimise naringenin synthesis. As a result of this, two synthetic promoters (BG28 and BG51) (Figure 7) were selected and an additional <I>E. coli</I> his operon terminator was placed after the first two genes (Figure 6 and Figure 8). These changes to the operon design could allow enhanced naringenin production in future experiments. More information about the pathway can be found <A href="https://2018.igem.org/Team:Newcastle/Model/Pathway/Main" class="black">here.</A></P></DIV></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 6. The naringenin biosynthetic operon under control of synthetic promoters BG28 and BG51 created in Benchling.</P></DIV></DIV><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 7. A close up of the synthetic promoters BG28 and BG51 placement in the operon, created in Benchling.</P></DIV></DIV></DIV><DIV class="row services-list block-1-2 block-tab-full"><DIV class="col-block service-item" data-aos="fade-up"><DIV class="service-text"><P style="text-align:center"> Figure 8. The naringenin biosynthetic operon contruct under control of a BG28 and BG51 promoters and an additional his operon terminator created in SBOL.</P></DIV></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para">Primers were designed for amplification of the backbone and the 4 gBlocks (Table 1). These were designed in Benchling. The primers were used to both amplify the gBlock templates (synthesised by IDT) for use in Gibson assemblies, and to detect the specific genes from the operon that could be present in transformed cells.</P><CENTER><P class="about-para"> Table 1. Primers designed in Benchling for amplification of the 4 gBlocks and the pSB1C3 backbone.</P><TABLE style="width:85%"><TBODY><TR><TH>Primer name</TH><TH>Sequence</TH><TH>Tm</TH><TH>Ta Q5</TH><TH>Amplified product(bp)</TH><TH>Shown to work</TH><TH>Description</TH></TR><TR><TD>pSB1C3F</TD><TD>tactagtagcggccgctgc</TD><TD>70</TD><TD>71</TD><TD>2070</TD><TD>6/8/18</TD><TD>To amplify pSB1C3 backbone</TD></TR><TR><TD>pSB1C3R</TD><TD>ctctagaagcggccgcga</TD><TD>70</TD><TD>71</TD><TD>2070</TD><TD>6/8/18</TD><TD>To amplify pSB1C3 backbone</TD></TR><TR><TD>4CLF</TD><TD>ccaaatcgccgccaattttc</TD><TD>59</TD><TD>56</TD><TD>1686</TD><TD>6/9/18</TD><TD>To amplify 4CL part</TD></TR><TR><TD>4CLR</TD><TD>cgtcgtcgttttgaagtggt</TD><TD>59.07</TD><TD>56</TD><TD>1686</TD><TD>28/9/18</TD><TD>To amplify 4CL part</TD></TR><TR><TD>TALF</TD><TD>gaatgtccgaacgctacagg</TD><TD>58.72</TD><TD>55</TD><TD>1649</TD><TD>28/9/18</TD><TD>To amplify TAL part</TD></TR><TR><TD>TALR</TD><TD>tcggaattgagcaggtcgat</TD><TD>59.18</TD><TD>56</TD><TD>1649</TD><TD>28/9/18</TD><TD>To amplify TAL part</TD></TR><TR><TD>CHIF</TD><TD>ctgggcatagaggtctggag</TD><TD>58.95</TD><TD>56</TD><TD>726</TD><TD>28/9/18</TD><TD>To amplify CHI part</TD></TR><TR><TD>CHIR</TD><TD>caccttctccgagtactgct</TD><TD>58.82</TD><TD>56</TD><TD>726</TD><TD>28/9/18</TD><TD>To amplify CHI part</TD></TR><TR><TD>CHSF</TD><TD>aagacgtgcctgggttgata</TD><TD>59.02</TD><TD>56</TD><TD>1197</TD><TD>28/9/18</TD><TD>To amplify CHS part</TD></TR><TR><TD>CHSR</TD><TD>gcttctcctccttcaaccct</TD><TD>59.01</TD><TD>56</TD><TD>1197</TD><TD>6/9/18</TD><TD>To amplify CHS part</TD></TR><TR><TD>gb1F</TD><TD>ctggaattcgcggccgct</TD><TD>72</TD><TD>54</TD><TD>1686</TD><TD>20/9/18</TD><TD>To amplify 4CL part</TD></TR><TR><TD>gb1R</TD><TD>ttacaatccatttgctag</TD><TD>53</TD><TD>54</TD><TD>1686</TD><TD>20/9/18</TD><TD>To amplify 4CL part</TD></TR><TR><TD>gb2F</TD><TD>ggcaaaactagcaaatgg</TD><TD>59</TD><TD>59</TD><TD>1649</TD><TD>20/9/18</TD><TD>To amplify TAL part</TD></TR><TR><TD>gb2R</TD><TD>ttatcagacgggagattg</TD><TD>58</TD><TD>59</TD><TD>1649</TD><TD>20/9/18</TD><TD>To amplify TAL part</TD></TR><TR><TD>gb3F</TD><TD>cttgcagcaatctcccgt</TD><TD>65</TD><TD>59</TD><TD>726</TD><TD>20/9/18</TD><TD>To amplify CHI part</TD></TR><TR><TD>gb3R</TD><TD>ctagactccaatcactgg</TD><TD>58</TD><TD>59</TD><TD>726</TD><TD>20/9/18</TD><TD>To amplify CHI part</TD></TR><TR><TD>gb4F</TD><TD>tactattccagtgattgg</TD><TD>54</TD><TD>55</TD><TD>1197</TD><TD>20/9/18</TD><TD>To amplify CHS part</TD></TR><TR><TD>gb4R</TD><TD>cggactgcagcggccgct</TD><TD>78</TD><TD>55</TD><TD>1197</TD><TD>20/9/18</TD><TD>To amplify CHS part</TD></TR></TBODY></TABLE></CENTER></DIV></DIV></SECTION><SECTION id="experiments" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Results</H3><H1 class="display-1">Experimental Work</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="service-text"><DIV class="col-full"><H3 class="h2">Backbone amplification</H3><P class="about-para"> The  pSB1C3  backbone was amplified, purified and quantified in preparation for Gibson  assembly  of the naringenin operon.  Amplification was performed by PCR using  Q5® High-Fidelity  DNA  Polymerase and primers pSB1C3F and pSB1C3R. The resultant PCR product  was  analysed by agarose gel electrophoresis and showed a band at 2 kb that corresponded to the size of the plasmid backbone. The backbone was then purified using  Qiagen  QIAquick  PCR Purification Kit and the  DNA concentration quantified using a Qubit fluorometer, this was determined to be 3.58 µg/ml. As this concentration was far too low for Gibson assembly, the backbone was amplified and purified again using the same techniques resulting in a brighter band at 2 kb than before (Figure 9) and a stock concentration of 26.2 µg/ml.</P><A href="https://2018.igem.org/Team:Newcastle/Protocols" class="black">Protocol Details found here</A></DIV></DIV><CENTER><P style="text-align:center">Figure 9. Agarose gel showing the 2 kb band representing the amplified plasmid backbone (pSB1C3).</P></CENTER><DIV class="service-text"><DIV class="col-full"><H3 class="h2">Positive control - Gibson assemblies</H3><P class="about-para"> A positive control of the Gibson  assembly was conducted. The NEBuilder® HiFi DNA Assembly Cloning Kit was used. A positive control reagent  supplied with the kit contained two  overlapping dsDNA fragments and the pUC19 plasmid. The positive control was conducted to check that the assembly mix was working, the protocols for transformation were correct and that the cells  prepared  previously were  competent.</P><P class="about-para">The results of the positive control showed that the <I>E. coli</I> DH5α  cells had successfully been made competent as they were able to take up the  three-part assembly of the two overlapping fragments and the pUC19 backbone (Figure 10A).  Furthermore, the reagents and protocols used in the Gibson assemblies were shown to have worked proving their viability for use in future experiments.  No colonies grew on the negative control plates (Figure 10B), verifying that <I>E. coli</I> DH5α cells are not ordinarily resistant to ampicillin and that no contamination had occurred.</P><P style="text-align:center">Figure 10. A) Successful <I>E. coli</I> DH5α transformation of the positive control provided in the NEBuilder® HiFi DNA Assembly Cloning Kit in LB ampicillin plates. B) Negative control plate of un-transformed <I>E. coli</I> DH5α cells. </P></DIV></DIV><DIV class="service-text"><DIV class="col-full"><H3 class="h2">Initial Gibson Assemblies of the naringenin operon</H3><P class="about-para">All Gibson assemblies were carried out using the 4 gBlocks, the pSB1C3 backbone and the competent cells produced on the 10/8/18. Gibson assemblies were conducted using the NEBuilder HiFi Assembly mix. For the assembly of 4 – 6 fragments the total concentration of DNA for the reaction had to be between 0.2 – 0.5 pmol with equimolar concentrations between all the inserts and the vector (0.05 pmol each) (Table 2). This reaction was incubated for 60 minutes at 50°C. The DH5α cells were then heat-shock transformed with the Gibson assembly product. The cells were spread onto LB plates containing chloramphenicol. Negative control plates containing un-transformed DH5α cells with no DNA were produced.</P><P style="text-align:center">Figure 11. A) <I>E. coli</I> DH5α colony from cells transformed with the second Gibson assembly. B) <I>E. coli</I> DH5α colony from cells transformed with the third Gibson assembly. C) Negative control plate of un- transformed <I>E. coli</I> DH5α cells showing no colonies.</P><P class="about-para">For the first Gibson assembly (Table 2), the backbone had a very low concentration, resulting in a low transformation efficiency.There was no colony growth on the transformed plates indicating the plasmid had not been taken up. Additionally there was no colony growth on the negative control.</P><P class="about-para">Following the lack of colonies, the amplified backbone with the higher concentration of 26.2µg/ml was used coupled with reducing the total reaction volume in Gibson assembly from 36µl to 20µl (Table 3). This method yielded two colonies from cells that had been transformed (Figure 11A and 11B) and a corresponding negative control plate (Figure 11C). It was discovered through agarose gel electrophoresis of the miniprep of these colonies that they had only taken up the 2 kb plasmid without the operon, as the only bright band was at 2 kb when compared to the ladder (Figure 12). The absence of colony growth indicated a low transformation efficiency, so commercial cells were used in future experiments instead of homemade competent cells.</P><P style="text-align:center">Figure 12. Agarose gel electrophoresis of the two miniprepped colonies from the second and third Gibson assemblies. 6 columns as split the 2 colonies into 3 separate tubes for the miniprep.</P></DIV></DIV><DIV class="service-text"><DIV class="col-full"><H3 class="h2">Gibson assembly transformations into commercial cells</H3><P class="about-para">Following multiple unsuccessful Gibson assembly attempts, the gBlocks and backbone were amplified by PCR using Q5® High-Fidelity DNA Polymerase. Gel extraction was performed using the QIAquick Gel Extraction Kit. Four Gibson assemblies using combinations of these amplified, gel extracted and original un- amplified gBlocks were set up (Table 4, Table 5, Table 6 and Table 7). These were then used to transform commercial DH5α cells competent cells. As a result, 30 to 60 colonies per transformation plate grew (Figure 13).</P><P style="text-align:center">Figure 13. Colonies from transformed cells with Gibson assemblies 1-4 using different volumes of gBlocks.</P><P class="about-para">As primers had been designed for detection of the operon from transformants, colony PCR was conducted on 5 of the colonies from the plates 1-4 using Q5® High-Fidelity DNA Polymerase to amplify the 5 kb band corresponding to the full length of the operon. No bands were observed following agarose gel electrophoresis of the PCR products (Figure 14).</P><P style="text-align:center">Figure 14. Colony PCR of colonies from transformed cells using the primers for the beginning and end of the operon.</P><P class="about-para">No bands were observed that corresponded to the 5 kb operon, possibly due to inherent difficulties commonly associated with the amplification of large PCR products. Based on this, colony PCR was not utilised in the future for the detection of the operon. As an alternative, plasmids would be extracted from the colonies obtained in future transformations. These would be analysed by agarose gel electrophoresis performed to visualise the plasmid. 45 colonies of DH5α transformants were inoculated in LB medium containing chloramphenicol and those cultures were used to carry out plasmid extractions that were later run on an agarose gel (Figure 15 and Figure 16). Two colonies from plate 4 (Figure 13) showed a promising 7 kb band corresponding to the expected size of the operon and the plasmid.</P><P style="text-align:center">Figure 15. Agarose gel electrophoresis plasmid extractions from transformants which were using Gibson assemblies 1-4 with the amplified and un-amplified gBlocks. Bands are visible at 2 kb corresponding to the plasmid backbone without the operon.</P><P style="text-align:center">Figure 16. Agarose gel electrophoresis of plasmid extractions showing two 7 kb bands at position 9 on the first gel and 4 on the second. These were both from plate 4 of the 1-4 Gibson assemblies with amplified and un-amplified gBlocks.</P><P class="about-para">Plasmids extracted from the two colonies displaying a 7 kb band were sent for sequencing using primers Gb1F, Gb1R, Gb2F, Gb2R, Gb3F and Gb2R (Eurofins Genomics). These primers were selected as the combined reads should provide sufficient sequence data coverage for the complete operon. Unfortunately, the sequence reads obtained did not provide sufficient data to confirm the correct assembly. Although some reactions provided sequence reads &gt; 1 kb, others provided little or poor quality data which resulted in insufficient coverage. One of the plasmids sequenced appeared to contain regions that aligned to the naringenin operon design, however due to gaps in the sequence reads this was not conclusive evidence of successful assembly (Figure 17). This may be due to the sensitive nature of the primers when annealing to the template during the sequencing reaction. Overall, it is likely that the relatively large size of the naringenin operon inserted into the pSB1C3 backbone (7 kb) led to reduced efficiency in transformation.</P><P style="text-align:center">Figure 17. Alignment of the sequences amplified by Gb1F and Gb1R to the gblock1 from the naringenin operon. Alignment constructed in Benchling.</P><P class="about-para">The Gibson assembly was attempted a final time, using DpnI to remove all the plasmid template from the reaction mixture (Table 8).</P><P style="text-align:center">Figure 18. Colonies from cells transformed with Gibson assembly treated with Dpn1. 5-6 were DH5α cells and 7-8 were TOP10 commercial cells.</P><P class="about-para">In plates 5-6, DH5α cells were transformed and in plates 7-8 TOP10 cells were transformed (Figure 18). Dpn1 digestion had been implemented. This resulted in over 40 colonies on each plate for DH5α and 2-3 colonies of TOP10 transformed cells on each plate. Following the same procedure as before; plasmids were extracted from 8 colonies, however only the 2 kb backbone was present following analysis by agarose gel electrophoresis (Figure 19).</P><P style="text-align:center">Figure 19. Agarose gel electrophoresis of plasmids extracted from transformants where Dpn1 had been used in the preparation of the Gibson assembly mix, subsequently used for transformation. 2 kb bands can be observed corresponding to the pSB1C3 plasmid backbone.</P></DIV></DIV></DIV></SECTION><SECTION id="conclusions" class="s-about"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead subhead--dark">Results</H3><H1 class="display-1 display-1--light">Conclusions</H1></DIV></DIV><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para">Two colonies obtained through transformation using a Gibson assembly of the naringenin operon, were found to contain a 7 kb plasmid. This corresponded to the size of the operon and pSB1C3 plasmid backbone. However, sequencing of these two plasmids did not provide sufficient evidence to confirm their correct assembly. Future experimentation should attempt to assemble the gBlocks one by one into the plasmid backbone and gain sequencing results that show full alignment. Following this, BL21 expression cells should be transformed with the 7 kb plasmid to produce naringenin. This would be extracted using ethyl acetate and analysed using HPLC. HPLC should be conducted using a series of known naringenin concentrations to create a standard curve. Following successful assembly of the operon under constitutive expression, Q5 site directed mutagenesis could be used to replace the J23100 promoter with the inducible T7 promoter. The same method could also be used to introduce the BG51 and BG28 promoters, suggested by the naringenin biosynthesis pathway modelling. This construct could be introduced into the chassis organism <I>Pseudomonas</I> sp.</P></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References &amp; Attributions</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size="2"><STRONG>Attributions: Heather Bottomley and Patrycja Ubysz</STRONG></FONT></P><P class="about-para"><FONT size="2">References: 1.	Zobel S, et al. (2015) Tn7-Based Device for Calibrated Heterologous Gene Expression in Pseudomonas putida. Acs Synth Biol 4(12):1341-1351.</FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>