<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Newcastle_Measurement_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Newcastle/Measurement/Methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position: fixed; top: 16px; left:0px; width:100%; height:50px; z-index:100"><UL id="navbar" class="navbar_ul" style="margin:0px"><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle">Home</A></LI><LI class="navbar_li"><A href="#">Project</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Description">Description</A><A href="https://2018.igem.org/Team:Newcastle/Safety">Safety</A><A href="https://2018.igem.org/Team:Newcastle/Parts">Parts Used</A><A href="https://2018.igem.org/Team:Newcastle/Part_Collection">New Parts Collection</A></DIV></LI><LI class="navbar_li"><A href="#">Human Practices</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Human_Practices">Integrated HP</A><A href="https://2018.igem.org/Team:Newcastle/Public_Engagement">Outreach and Public Engagement</A></DIV></LI><LI class="navbar_li"><A href="#">Hardware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Hardware">Hardware Overview</A><A href="https://2018.igem.org/Team:Newcastle/Circuitry">Circuitry</A><A href="https://2018.igem.org/Team:Newcastle/Results">Results</A><A href="https://2018.igem.org/Team:Newcastle/Developments">Future Developments</A></DIV></LI><LI class="navbar_li"><A href="#">Software</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Software">Software Overview</A><A href="https://2018.igem.org/Team:Newcastle/Software/NH1">NH-1</A><A href="https://2018.igem.org/Team:Newcastle/Software/OT">Opentrons</A></DIV></LI><LI class="navbar_li"><A href="#">Wetware</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte1">Root Colonisation</A><A href="https://2018.igem.org/Team:Newcastle/Results/Endophyte">Endophytic Chassis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Chemotaxis">Chemotaxis</A><A href="https://2018.igem.org/Team:Newcastle/Results/Operon">Naringenin Operon</A><A href="https://2018.igem.org/Team:Newcastle/Protocols">Protocols</A><A href="https://2018.igem.org/Team:Newcastle/Experiments">Lab Book</A></DIV></LI><LI class="navbar_li"><A href="#">Modelling</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Model">Modelling Overview</A><A href="https://2018.igem.org/Team:Newcastle/Modelling/Community">Microbial Community Modelling</A><A href="https://2018.igem.org/Team:Newcastle/Naringenin_Pathway">Naringenin Pathway</A></DIV></LI><LI class="navbar_li"><A href="#">Measurement</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/InterLab">InterLab Study</A><A href="https://2018.igem.org/Team:Newcastle/Measurement">Measurement, Automation &amp; Reproducibility</A><A href="https://2018.igem.org/Team:Newcastle/Measurement/Methods">Materials and Methods</A></DIV></LI><LI class="navbar_li"><A href="#">Team</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Team">Members</A><A href="https://2018.igem.org/Team:Newcastle/Collaborations">Collaborations</A><A href="https://2018.igem.org/Team:Newcastle/Attributions">Attributions</A></DIV></LI><LI class="navbar_li"><A href="https://2018.igem.org/Team:Newcastle/Notebook">Notebook</A></LI><LI class="navbar_li"><A href="#">Judging</A><DIV class="navbar_dropdown_content"><A href="https://2018.igem.org/Team:Newcastle/Medals">Medals</A><A href="https://2018.igem.org/Team:Newcastle/Improve">Improve</A><A href="https://2018.igem.org/Team:Newcastle/Demonstrate">Demonstrate</A></DIV></LI></UL><DIV id="expand_btn"><BUTTON class="toggle" onclick="myFunction()">Hide/Show</BUTTON></DIV></DIV><TITLE>Alternative Roots</TITLE><SECTION id="world" class="s-home target-section" data-parallax="scroll" data-image-src="https://static.igem.org/mediawiki/2018/2/26/T--Newcastle--3432485.jpg" data-natural-width="3000" data-natural-height="2000" data-position-y="center"><DIV class="home-content"><DIV class="row home-content__main"><H3>Alternative Roots</H3><H1>
                    Materials and Methods </H1><DIV class="home-content__buttons"><A href="#RFP" class="smoothscroll btn btn--stroke" style="width:300px">
         Improved Plasmids 
        </A><A href="#Biodesign" class="smoothscroll  btn btn--stroke">
         Automation
        </A><A href="#Variation" class="smoothscroll btn btn--stroke">
         Media Variation  
        </A></DIV></DIV></DIV></SECTION><SECTION id="RFP" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Implementing the New Devices</H1><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Bacterial Strains.</H3></DIV></DIV><P><FONT size="3">Transformations with, and expression of, iGEM test devices and controls were carried out using chemically competent <I>Escherichia coli</I> DH5α. Competency was conferred using the MgCl<SUB>2</SUB>-CaCl<SUB>2</SUB> method [1]. Briefly, a single colony of DH5α was incubated in Leuria Bertoni (LB) broth overnight at 37 °C with shaking at 220 rpm. Overnight culture was diluted 1:100, further incubated until an optical density (OD<SUB>600</SUB>nm) of 0.3 – 0.6 was reached and then placed on ice for 30 minutes. Cells were centrifuged at 4000 g for 5 minutes at 4 °C, resuspended in 0.1 M MgCl<SUB>2</SUB> and incubated on ice for 30 minutes. The suspension was centrifuged again as before, resuspended in 0.1 M CaCl<SUB>2</SUB> and placed on ice for 30 minutes. Cells were spun down again, resuspended in 0.1 M CaCl<SUB>2</SUB> with 15 % glycerol and frozen at -80 °C. </FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Plate Reader Set-up</H3></DIV></DIV><P><FONT size="3">Culture absorbance and fluorescence were measured in 96 well plates using a Thermofisher Varioskan Lux plate reader (Thermofisher scientific) unless stated otherwise. Absorbance was measured at 600 nm. GFP Fluorescence was measured at 525 nm with excitation at 485 nm. RFP fluorescence was measured at 635 nm with excitation at 588 nm. All readings took place at 25 °C after a 5 second 300 rpm shake step to homogenise the culture. Readings used a 12 nm bandpass width and pathlength correction was disabled, as per the iGEM Interlab study guidelines. </FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Internal Standard &amp; mNeonGreen Design</H3></DIV></DIV><P><FONT size="3"> An RFP construct was designed for use as an internal standard for each test device. The RFP construct was designed using Benchling. The parts used for building the RFP construct were Anderson promoter BBa_J23108, RBS BBa_0032, the RFP gene - gained from SnapGene - and double terminator BBa_B0015. Gibson ends were also designed for cloning into pSB1C3 using the NEBuilder DNA assembly tool and the gBlock was synthesised by IDT. The promoter has a measured strength of 0.51 relative to BBa_J23100.</FONT></P><P><FONT size="3">The mNeonGreen construct was designed for use as an alternate fluorescent reporter for each test device - replacing GFP. The mNeonGreen sequence was codon optimised using Benchling and the Gibson ends were designed using NEBuilder for cloning into pSB1C3. The subsequent sequence was synthesised by IDT.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Cloning of New Devices into pSB1C3</H3></DIV></DIV><P><FONT size="3">Plasmid vectors were purified from <I>E. coli</I> via miniprep (Qiagen) and the concentration for each mini-prepped test device was determined using a Qubit fluorometer and diluted to 0.5 ng/µl. The diluted pSB1C3 vectors were linearised using a 2 step PCR system following a Q5 Polymerase protocol (NEB). This protocol utilised forward and reverse primers with T<SUB>m</SUB> values of 72 °C. The internal standard and mNeonGreen primers were designed by using the NEB Tm calculator and Benchling. The Internal Standard bind in a non-coding region of the pSB1C3 vector – a region between the chloramphenicol resistance gene and the ORI. The mNeonGreen primers consisted of 6 reverse primers, one complimentary to each test device promoter, and a single forward primer over the terminator. The amplified DNA was then digested with DpnI, heat treated to inactivate the enzyme and assembled via Gibson Assembly using the NEBuilder HiFi DNA Assembly Kit. Following their protocol, a 2-fragment reaction with 0.5 pmol of DNA in a 2:1 insert to vector ratio was done and transformants were plated onto agar plates with the appropriate antibiotic (LB+cam for each test device and LB+amp for the controls). Following growth of colonies, plasmid DNA was miniprepped from DH5α transformed with both the internal standard and the mNeonGreen vector and sequenced to verify presence of the genes.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Internal Standard &amp; mNeonGreen Analysis </H3></DIV></DIV><P><FONT size="3">Analysis of the internal standards involved comparing the original InterLab test device plasmids against the new internal standard plasmids. Wells A-D represented the RFP containing <I>E. coli</I> and wells E-H represented the original test device containing <I>E. coli</I>. Column 9 wells A-H contained an LB+CAM blank. The microtiter plate was incubated for 24 hours in the plate reader with Abs<SUB>600</SUB>, fluorescence (GFP): Excitation 485 nm, Emission 420 nm and fluorescence (RFP): Excitation 588 nm, emission 635 nm measured every 15 minutes following a short shake at 420 rpm at a low shake diameter. </FONT></P><P>Three further Interlab studies were carried out for mNeonGreen expressing <I>E. coli</I> and those containing the original test devices, using the same conditions as the original study. The results fluorescence/OD, MEFL/particle and mean standard error of the mNeonGreen study was compared to the original Interlab. </P></DIV></DIV></SECTION><SECTION id="Biodesign" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Bio-Design Automation</H1><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Bacterial Strains.</H3></DIV></DIV><P><FONT size="3">Primary transformation buffers (TB) before optimisation were CCMB80 protocol and a CaCl<SUB>2</SUB>-MgCl<SUB>2</SUB> protocol. CCMB80 buffer was made from the following: 10 mM KOAc, 80 mM CaCl<SUB>2</SUB>, 20 mM MnCl<SUB>2</SUB>, 10 mM MgCl<SUB>2</SUB>, 10 % glycerol and pH was adjusted to 6.4 with 0.1 N HCl. For the CaCl<SUB>2</SUB>-MgCl<SUB>2</SUB> protocol, a 100 mM CaCl<SUB>2</SUB> and a 100 mM MgCl<SUB>2</SUB> solution were made. For preliminary DoE scoping experiments, concentrated stock solutions were made up of each individual reagent, buffer or compound. These stocks were used for the low, medium and high scoping (Table 1), buffer (Table 2), wash step (no wash, 1 wash or 2 wash) and cryoprotectant experiments (Medium TB with either DMSO 7.5 % or glycerol 18 %). All buffers were filter sterilised using Soft-Ject® syringes with Minisart 0.2 µm filter (bar DMSO which was filtered with a DMSO-Safe Acrodisc® filter) and stored in 30 mL sterile universal tubes at 4 ℃. Fresh stocks were made when solutions ran out, however concentrations and storage remained the same.</FONT></P></DIV></DIV></SECTION><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Plasmid Preparation</H3></DIV></DIV><P><FONT size="3">RFP test and GFP (TD4) plasmids were purified using a QIAprep Spin Miniprep Kit (Qiagen). One colony of transformed E. coli was grown at 37 ℃ at 220 rpm O/N (16 hours) in 5 mL SOB media in a 50 mL falcon tube. 1 mL was then aliquot into four different 2 mL microcentrifuge tubes. The official protocol was followed, including the addition of RNase A, LyseBlue reagent and 100% EtOH to the appropriate buffers. Instead of using Buffer EB to elute the DNA in step 10, sterile ddH2O was used to remove any potential downstream interactions that the buffer could have during analysis and transformation. The elution was run through the QIAgen 2.0 spin column a second time to maximise the yield. To determine plasmid concentration, a Qubit dsDNA BR Assay Kit (Invitrogen) was used following its standard protocol. 100 pg/µL stocks of both RFP and GFP were made with ddH<SUB>2</SUB>O and frozen at -20 ℃. </FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Manual Competent Cell and Transformation Workflow </H3></DIV></DIV><P><FONT size="3">Competent cells for the iGEM Interlab were made using the CCMB80 buffer (iGEM 2018) and CaCl<SUB>2</SUB>-MgCl<SUB>2</SUB> buffer protocol. For downscale experimentation from 250 mL tube to 96 well plate, as well as wash step experiments, CaCl<SUB>2</SUB>-MgCl2 buffer was used. For all other experiments, the competent cells were made using a modified 0 wash protocol with varying TBs. The 0 wash protocol was an adapted version of the CaCl<SUB>2</SUB>-MgCl<SUB>2</SUB> protocol. 1 colony was used to inoculate 10 mL of SOB in a 50 mL falcon tube for an O/N culture. 1 mL of the O/N culture was then used to inoculate 50 mL SOB in a 250 mL baffled conical flask the following morning. Once OD<SUB>600</SUB> of 0.4-0.6 reached, 1.5 mL of culture was aliquoted into 2 mL microcentrifuge tubes and chilled on ice for 15 minutes, before being pelleted by a 5417R microcentrifuge (Eppendorf) at 3500 RCF for 5 minutes at 4 ℃. Pellets were then resuspended as 100 µL aliquots, made with TB, and with cryoprotectants glycerol or DMSO added if TB did not already contain one. These aliquots could then be frozen or used after a 40 minutes ice incubation. </FONT></P><P><FONT size="3">For the transformations, there was no variance in protocol, with the following standard transformation workflow being used for all transformations. Competent <I>E. coli</I> DH5α aliquots were thawed on ice for 15 minutes before 1 µL plasmid DNA added (plasmid concentration range from 10 pg/µl – 50 ng/µl). <I>E. coli</I> was then incubated on ice for 40 minutes before a heat shock at 42 ℃ for 45 seconds was carried out. After heat shock, cells were returned to ice for at least 5 minutes. For 2 mL Eppendorf’s, 900 µL preheated SOB (37℃) was then added to aliquots. For 96 well microtiter plates, 150 µL preheated SOB was added. For 2 mL microcentrifuge transformations, if ampicillin was the plasmids selection agent, incubation recovery times were reduced to 40 minutes at 37 ℃ at 220 rpm. For chloramphenicol, incubation recovery times were extended to 2 hours. For 96 well plate transformations not following automated protocol, incubation recovery times were kept the same for the respective antibiotics, however plates were incubated in plate reader at 37 ℃ with 600 rpm shake and low force. Using automation, 96 wells plates were incubated at 37 ℃ static with TempDeck (Opentrons, United States).</FONT></P><P><FONT size="3">To assess transformation efficiency, 50 µL of transformed culture was spread on SOB+CAM agar plates and incubated O/N at 37℃. Colonies were counted and TrE was determined by CFU. The calculations are shown below: </FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Miniprep and Colony PCR</H3></DIV></DIV><P><FONT size="3">Miniprep plasmid and TD4 amplicon were analysed using gel electrophoresis. Plasmid was generated via the previously described QIAprep Spin Miniprep Kit (Qiagen). TD4 amplicon was generated using VF2 forward and VR reverse primers using Q5 High-Fidelity DNA polymerase (New England BioLabs) and supplied protocol. Hyperladder 1kb+ was used for ladder (Bioline). Amplification times followed recommended protocol provided by NEB, with Tm set at 60 ℃. 1 % agarose gel in 0.5x TBE with 2.5 µL GelRed was prepared to a volume of 50 mL. Gel electrophoresis was carried out at 100 volts for 1 hour and gel analysed using GelDocEZ (Bio-Rad).</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Liquid Handling</H3></DIV></DIV><P><FONT size="3">To automate experimentation and ascertain a high-throughput BDA workflow, all automated operations were carried out by OT-2 liquid handling robot (Opentrons, United States). The OT-2 was fitted with two single channel pipettes, a 1-10 µL (P10) and a 30-300 µL (P300). Specific design and specification data for these pipettes can be found here: https://opentrons.com/pipettes. Sterile Opentrons 300 µL, TipOne 200 µL and Pipetman diamond 10 µL, tips were the only tips used. ThermoScientific Sterilin single use plastics were the only 96 well plates used (unless specified otherwise in the Interlab). To cool 96 well plates during competent cell and transformation protocols, a TempDeck (Opentrons, United States) was used allowing cooling to 4 ℃ and to heat shock cells to 42 ℃ during transformation. All automated workflows were run through the free OT-2 App.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Python and Module Design</H3></DIV></DIV><P><FONT size="3">Python 3.7.0 (package downloaded from <A href="https://www.python.org/" class="black">Python</A>was the programming language used for development of liquid handling protocols. Python script was written and edited using TextWrangler (Bare Bones Software, United States). TextWrangler.py files could then be read by the OT-2 App (http://opentrons.com/ot-app). Once protocols were developed and showed preliminary success, python scripts were uploaded to the GitHub development platform (GitHub, United States) for open-source sharing and further collaborative development (see <A href="https://github.com/jbird1223/Newcastle-iGEM" class="black">GitHub Newcastle iGEM</A> for file downloads and access).</FONT></P><P><FONT size="3">Custom made plastic ice box containers with customisable tube rack inserts were drawn up using EazyDraw 3.10.9 software (Dekorra Optics LLC, United States) and 3D printed. Cold boxes were 3D modelled in SketchUp (Trimble Inc.) and constructed out of PLA using an Ultimaker 3 Extended desktop printer (Ultimaker). Customisable tube rack inserts were redrawn accurately using Adobe illustrator and exported as a PDF. A FB700 laser cutter was then used to cut the inserts. Models were uploaded to GitHub to allow open access.  A 200 µL TipOne rack adapter downloaded from open-source module page was also constructed using an Ultimaker.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Automated Competent Cell and Transformation Workflow  </H3></DIV></DIV><P><FONT size="3">This followed the same principle as the manual 0 wash protocol and standard transformation, however it had various adaptions to allow for a more optimised workflow. Full deck configuration for DoE, competent cell and transformation protocols can be seen in Figure 3. O/N cultures were grown as per manual method with an O/N followed by a morning culture to an OD<SUB>600</SUB> of 0.4-0.6. If the previous day was a testing day, transformation efficiencies were calculated and results inputted into JMP software. New design space was calculated, exported as a CSV file and inputted into DoE protocol (Appendix B2). Complex TB’s were then constructed during morning incubation, using a 1 mL 96 deepwell plate as a storage container. TB’s were sealed using ThermoScientific AB-0558 adhesive PCR film and put on ice to chill.</FONT></P><P><FONT size="3">Once OD<SUB>600</SUB> reached between 0.4-0.6, culture was put on ice and competent cell protocol initiated to allow time for the TempDeck to reach 4 ℃. When target temperature was reached, culture was added to relevant deck slot and competent cell protocol carried out. This followed the manual protocol, however volume was reduced to a max of 200 µL and plates were centrifuged in a 5090 R table-top centrifuge (Eppendorf) at 2000 RCF for 20 minutes at 4 ℃.  Once competent cell protocol finished, plasmids were taken out of the freezer to thaw and placed into microcentrifuge rack. The automated transformation protocol was then loaded and carried out using the recently made aliquots. This did not deviate from the standard transformation protocol, bar the incubation and heat shock being carried out by the TempDeck.</FONT></P><P><FONT size="2"><B> Figure 1. Fully configured OT-2 deck that allows for minimal modification when cycling through Design of Experiments (DoE) buffer protocol, competent cell protocol and transformation protocol. </B> A = DoE buffer protocol, B = Competent cell protocol, C = Transformation protocol and D = Robot deck slot diagram (robot deck diagram adapted from: <A href="https://docs.opentrons.com/labware.html" class="black">Opentrons Labware Library</A>). White highlights indicate labware used for each protocol. The labware from slots are as follows: 1 = None, 2 = 30 mL Universal tube cold deck, 3 = 96 Deep well plate, 4 = Fresh culture/ microbial waste, 5 = 30 mL Universal tube cold deck, 6 = TempDeck with 96 well plate, 7 = SOB, 8 = 300 µL Opentrons pipette tips, 9 = TipOne 200 µL pipette tips, 10 = 2 mL Microcentrifuge rack, 11 = TipOne 10 µL pipette tips and trash = Built in trash box.</FONT></P><SECTION id="Variation" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">Variation in Media Composition</H1><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Comparison of batch to batch expression variation in complex media</H3></DIV></DIV><P><FONT size="3"> Three separate batches of Luria Bertani broth were made using tryptone and yeast extract from different manufacturers: LB A consisted of LB Broth EZ Mix Powder (Sigma, batch 066K8206); LB B consisted of EZ Mix tryptone (Sigma, batch 32K8201) and ForMedium yeast extract (batch 16/MFM/1835); LB C consisted of For Medium tryptone (batch 18/MFM/2127) and for medium yeast extract (batch 17/MFM/2004). Twenty 200 µl volumes of each media were loaded onto a 96 well plate and inoculated with 5µl of exponential growth phase DH5α transformed with the positive control test device using an epMotion 5073 pipetting robot to minimise variation by operator error. Plates were then transferred to the plate reader and incubated at 37 °C with shaking at 600 rpm for 24 hours, with absorbance and fluorescence readings every 30 minutes.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Rich Media </H3></DIV></DIV><P><FONT size="3">For rich defined media optimisation components of each of four commonly used rich defined media were used as factors for optimisation of a rich defined media (Table 3). MDAG-11, MDA-5052 and EZ contained trace metals of various concentrations. As trace metals were at such low concentrations, it was not always practically possible to accurately prepare individual concentrations of each range. Therefore, a master mix of trace metals was included as a categorical variable (i.e. present or not present) for DoE. As with minimal media, in the literature various carbon sources were included in each of the rich media. To simplify DoE, glycerol was retained as the sole carbon source as a constant in DoE design.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">DoE and Experimental Runs </H3></DIV></DIV><P><FONT size="3">DoE for optimal experimental design was performed using JMP Pro 12 statistical software (SAS Institute Inc., USA). The number of runs and media composition of each was determined by the custom design platform to provide the optimum number of runs to allow collection of sufficient data to allow estimation of the main effects and two factor interactions. For minimal and rich media DoE respectively, an experimental design consisting of 20 runs and 30 runs of varying media compositions was derived and allowed the implementation of experimental runs in triplicate and in duplicate respectively on a single 96 well plate. This compared to 8192 and 4.19 x 10<SUP>6</SUP> experimental runs respectively for full factorial analysis. </FONT></P><P><FONT size="3">Twenty millilitres of LB broth was inoculated with 1ml of <I>E. coli</I> DH5α and cultured overnight at 37 °C, 220 rpm. Fresh media was then inoculated with overnight cultured, grown to exponential growth phase (OD = 0.3) and washed in triplicate by pelleting at 8700g and re-suspending in phosophate buffered saline (PBS) to remove LB broth. Total well volumes for each run were made up to 200 μl by an epMotion 5073 pipetting robot and inoculated with 5μl of exponential growth phase DH5α. Plates were then transferred to the plate reader and incubated at 37 °C with shaking at 600 rpm for 24 hours, with absorbance and fluorescence readings every 30 minutes.</FONT></P><P><FONT size="3">Incubation in the plate reader in 200 μl volumes produced poor growth, therefore the rich media DoE runs were also performed in 10 ml volumes in a shaking incubator, incubated at 37 °C at 220 rpm. Media for each run was assembled using the OT-2 pipetting robot. Each hour, 200 μl of culture for each run was transferred to a 96-well plate and fluorescence and absorbance reading taken as previously. After 10 hours of initial growth in 10 mL volumes, all samples were transferred to a 96-well plate and incubated for a further 12 hours, with fluorescence and absorbance reading every hour.</FONT></P><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H3 class="subhead">Modelling DH5α growth in rich defined media </H3></DIV></DIV><P><FONT size="3">Data modelling and visualisation was performed using JMP Pro v.12 (SAS Institute Inc., USA). A partial least squares (PLS) model was fitted to the data, with model validation performed using KFold (10) cross validation with application of the SIMPLS (statistically inspired modification of the partial least squares method) algorithm. Significant factors were indicated by a variable importance for the projection (VIP) score of greater than 1. The prediction profiler utility allows an interactive visualisation of the effects of factors on the OD<SUB>600</SUB>, allowing the user to alter factor levels or concentrations to show the change in OD<SUB>600</SUB> predicted by the model. The model fit was assessed by the root mean PRESS value for the coefficient estimates for X and Y for different models, with lower root mean PRESS values indicating a lower prediction error and a better model fit. </FONT></P></DIV></DIV></SECTION><SECTION id="team" class="s-services"><DIV class="row section-header has-bottom-sep" data-aos="fade-up"><DIV class="col-full"><H1 class="display-2">References &amp; Attributions</H1></DIV></DIV><BUTTON class="collapsible">Click for References</BUTTON><DIV class="content"><DIV class="row about-desc" data-aos="fade-up"><DIV class="col-full"><P class="about-para"><FONT size=""><STRONG>Attributions: Matthew Burridge, Kyle Stanforth and Sam Went
</STRONG></FONT></P><P class="about-para"><FONT size="2">1. Sambrook J, Russel DW. (2001) Molecular cloning: A laboratory manual. Cold Spring Harbour. </FONT></P></DIV></DIV></DIV></SECTION><DIV class="row footer-main"><DIV class="col-six tab-full right footer-subscribe"><DIV class="row footer-bottom"><DIV class="col-twelve"><DIV class="copyright"><SPAN><A class="link" style="color: #39b54a;" href="https://experiment.com/projects/genetically-engineered-microbes-for-sustainable-agriculture" target="_blank">Help fund our project</A></SPAN><SPAN>Site Template by <A class="link" style="color: #39b54a;" href="https://www.colorlib.com/" target="_blank">Colorlib</A></SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>