<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Lab_Labjournal_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Lab/Labjournal/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Lab/Labjournal/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/July&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Lab/Labjournal/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Lab/Labjournal/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="nav_menu gradient_golden"><DIV id="menu_center"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware">Home</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">Project</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance#Main_Results">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">Mutation Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody">Antibody Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus">Selection Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam Standard">Potsdam Standard</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Collaboration">Collaboration</A></LI></UL><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Sponsoring">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Attribution">Attribution</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Lab</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Group_Meetings">Group Meetings</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">All Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">SocialBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_Politics">Science Meets Politics</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_People">Science Meets People</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Software">Software</A></LI></DIV></DIV><DIV class="box_round white_bg"><P><A href="/wiki/index.php?title=UP12_Labjournal&amp;action=edit&amp;redlink=1" class="new" title="UP12 Labjournal (page does not exist)">back to UP12_Lab journal</A></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#AID"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">AID</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#2012-07-02"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">2012-07-02</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#2012-07-03"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">2012-07-03</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#2012-07-04"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">2012-07-04</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#2012-07-05"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">2012-07-05</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#2012-07-06"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">2012-07-06</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#2012-07-07"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">2012-07-07</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#2012-07-08"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">2012-07-08</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#2012-07-09"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">2012-07-09</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#2012-07-10"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">2012-07-10</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#2012-07-11"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">2012-07-11</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#2012-07-12"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">2012-07-12</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#2012-07-13"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">2012-07-13</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#2012-07-16"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">2012-07-16</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#2012-07-23"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">2012-07-23</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#2012-07-25"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">2012-07-25</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#2012-07-26"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">2012-07-26</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#2012-07-27"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">2012-07-27</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#2012-07-28"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">2012-07-28</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#2012-07-29"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">2012-07-29</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#2012-07-30"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">2012-07-30</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#2012-07-31"><SPAN class="tocnumber">1.21</SPAN><SPAN class="toctext">2012-07-31</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-23"><A href="#Antik.C3.B6rper"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Antikörper</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#2012-07-06_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">2012-07-06</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#2012-07-07_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">2012-07-07</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#2012-07-10_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">2012-07-10</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#2012-07-12_2"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">2012-07-12</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#2012-07-15"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">2012-07-15</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#2012-07-16_2"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">2012-07-16</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#2012-07-17"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">2012-07-17</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#2012-07-20"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">2012-07-20</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#2012-07-26_2"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">2012-07-26</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#2012-07-28_2"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">2012-07-28</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#2012-07-31_2"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">2012-07-31</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#2012-07-31_3"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">2012-07-31</SPAN></A></LI><LI class="toclevel-1 tocsection-36"><A href="#Virus"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Virus</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#02.07.2012"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">02.07.2012</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#04.07.2012"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">04.07.2012</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#04.07.2012_2"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">04.07.2012</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#11.07.2012"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">11.07.2012</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#12.07.2012"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">12.07.2012</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#13.07.2012"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">13.07.2012</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#2012-17-07"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">2012-17-07</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#2012-18-07"><SPAN class="tocnumber">3.8</SPAN><SPAN class="toctext">2012-18-07</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#2012-27-07"><SPAN class="tocnumber">3.9</SPAN><SPAN class="toctext">2012-27-07</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#2012-01-08"><SPAN class="tocnumber">3.10</SPAN><SPAN class="toctext">2012-01-08</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="AID">AID</SPAN></H2><H3><SPAN class="mw-headline" id="2012-07-02"><P style="background-color: rgb(240, 20, 70);">2012-07-02</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Overnight culture of CMV and polyA carrying cells</P><P><B>Investigators:</B> Mario, Tom S. </P><P><B>Time:</B> 2012-07-02</P><P><B>AIM:</B> Preparation of wild type AID</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Chloramphenicol 25 mg/mL stock solution in 70 % EtOH</LI></UL><UL><LI> Plasmids: pSB1C3 with CMV; pSB1C3 with Poly-A
</LI></UL><P><B>Method:</B></P><P>Inoculation of cell sample each in 5 ml LB medium </P><P>shaking over night at 37 °C, 300 rpm, approx. 16 hours </P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><H3><SPAN class="mw-headline" id="2012-07-03"><P style="background-color: rgb(240, 20, 70);">2012-07-03</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Glycerolstocks, Miniprep and preparative digestion</P><P><B>Investigators:</B> Basia, Tom S., Chris, Mario </P><P><B>Time:</B> 2012-07-03</P><P><B>Aim:</B> Preparation of wildtype AID</P><P><B>Materials:</B></P><UL><LI> Glycerol </LI></UL><UL><LI> Miniprep Kit</LI></UL><UL><LI> overnight culture (pSB1C3 with CMV); overnight culture (pSB1C3 with Poly-A)
</LI></UL><UL><LI> CMV: Restriction enzymes (SpeI, PstI); NEB buffer 2
</LI></UL><UL><LI> Poly-A: Restriction enzymes (Poly A: PstI, XbaI); NEB buffer 3
</LI></UL><P><B>Method:</B></P><P>Glycerol stock: 500 µL Glycerol 99,8 % + 500 µL overnight cultures --&gt; put in -80 °C freezer
</P><P>Miniprep (both over night culture (pSB1C3 with CMV) and over night culture (pSB1C3 with Poly-A)
</P><P>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL NEB (2 or 3) buffer
</P><P><B>Results:</B></P><P>DNA - concentrations via nanodrop:</P><P>pcDNA5 (AG) = 642,9 ng/µL </P><P>pcDNA5 (good) = 729,1 ng/µL </P><P>pcDNA5 (bad) = 705,4 ng/µL </P><P>pSB1C3 with CMV = 311,9 ng/µL </P><P>pSB1C3 with Poly-A = 360,3 ng/µL </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis</P><P><B>Investigators:</B> Chris, Mario </P><P><B>Time:</B> 2012-07-03</P><P><B>Aim</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P>cut samples:
</P><UL><LI> CMV: Restriction enzymes (SpeI, PstI); NEB buffer 2
</LI></UL><UL><LI> Poly-A: Restriction enzymes (PstI, XbaI); NEB buffer 3
</LI></UL><P><B>Method:</B></P><P>samples:</P><P>- 30 µL CMV cut with SpeI and PstI + 7,5 µL loading dye</P><P>- 30 µL polyA + 7,5 µL loading dye</P><P>gel electrophoresis conditions:</P><P>30 µL of each sample into one big slot</P><P>V = 120 V</P><P>duration roughly 50 minutes</P><P><B>Results:</B></P><P>Marked fragments were cut out of the gel and transferred into 1,5 mL Eppendorf tubes</P><P><B>Further Tasks:</B></P><P>Gel Extraction
</P><H3><SPAN class="mw-headline" id="2012-07-04"><P style="background-color: rgb(240, 20, 70);"> 2012-07-04</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel Extraction of CMV and polyA</P><P><B>Investigators:</B></P><P>Mario, Tom S. </P><P><B>Aim:</B></P><P>Gel Extraction of CMV and polyA
</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up - Kit, thermo block, nanodrop
</P><P>Molecular weight calculator online resource: <A href="http://www.encorbio.com/protocols/Nuc-MW.htm" class="external free" rel="nofollow">http://www.encorbio.com/protocols/Nuc-MW.htm</A></P><P>multiplicate with factor 2 when DNA is double stranded</P><P><B>Method:</B></P><P>extract DNA: according to the manual
</P><P><B>Results:</B></P><P>DNA-concentrations via nanodrop:</P><P>CMV = 106,8 ng/µL -&gt; 63,7 nM (with mass conc. of 1676525,6 Da)</P><P>polyA = 15,1 ng/µL -&gt; 48,5 nM (with mass conc. of 311550 Da)</P><P>location: -20 °C freezer, topmost drawer
</P><P>ready DNA for Ligation
</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Overnight culture of AID carrying cells</P><P><B>Investigators:</B> Sascha </P><P><B>Time:</B> 2012-07-04</P><P><B>Materials:</B></P><P>LB medium </P><P>ampicillin 100 mg/ ml stock solution</P><P>glycerol stocks <I>E. coli</I> XL1 blue with plasmids: pSB1C3 with AID
</P><P><B>Method:</B></P><P>Inoculation of cell sample in 5 ml LB medium </P><P>shaking overnight at 37°C, 300 rpm, approx. 16 hours </P><P><B>Further tasks:</B></P><P>Miniprep </P><H3><SPAN class="mw-headline" id="2012-07-05"><P style="background-color: rgb(240, 20, 70);">2012-07-05</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Miniprep and preparative digestion</P><P><B>Investigators:</B> Chris </P><P><B>Time:</B> 2012-07-05</P><P><B>Materials:</B></P><P>Miniprep Kit</P><P>over night culture (pSB1A3 with AID)
</P><P>AID: Restriction enzymes (XbaI, PstI); NEB buffer 3
</P><P><B>Method:</B></P><P>Miniprep according to the manual
</P><P>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL NEB 3 buffer
</P><P><B>Results:</B></P><P>DNA - concentrations via nanodrop:</P><P>pSB1A3 with AID = 85,5 ng/µL </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis</P><P><B>Investigators:</B> Mario, Tom S. </P><P><B>Time:</B> 2012-07-05</P><P><B>Aim</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P>cut samples:</P><P>AID: Restriction enzymes (XbaI, PstI); NEB buffer 3
</P><P><B>Method:</B></P><P>samples:</P><P>- 30 µL AID cut with XbaI and PstI + 7,5 µL loading dye</P><P>gel electrophoresis conditions:</P><P>30 µL of each samples into one big slot converted</P><P>V = 120 V</P><P>duration roughly 65 minutes</P><P><B>Results:</B></P><P>Marked fragment was cut out of the gel and transferred into 1,5 mL Eppendorf tube</P><P><B>Further Tasks:</B></P><P>Gel extraction
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction of AID</P><P><B>Investigators:</B></P><P>Mario, Tom S. </P><P><B>Aim:</B></P><P>Gel extraction of AID
</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up - Kit, thermo heater, nanodrop
</P><P><B>Method:</B></P><P>Gel extraction according to the manual
</P><P><B>Results:</B></P><P>DNA-concentrations via nanodrop:</P><P>AID = 9,2 ng/µL -&gt; 23,8 nM (with mass conc. of 386864,4 Da)</P><P>location: -20 °C freezer, topmost drawer
</P><P>ready DNA for Ligation
</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of CMV (Insert + backbone) and AID (insert)</P><P><B>Investigators:</B></P><P>Mario, Tom S. </P><P><B>Aim:</B></P><P>Ligation of CMV (Insert + backbone) and AID (insert)
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, samples(CMV + AID)
</P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation:
</P><UL><LI> 1 µL (CMV Fragment) c=106,8 ng/µL(63,9 nM) -&gt; 6,4 nmol
</LI></UL><UL><LI> 6 µL (AID Fragment) c=9,2 ng/µL(23,8 nM) -&gt; 14,3 nmol
</LI></UL><UL><LI> 1 µL (T4 DNA-Ligase)
</LI></UL><UL><LI> 2 µL (DNase free water)
</LI></UL><P>incubation of sample 1,5 h at 22 °C</P><P><B>Results:</B></P><P>location: -20 °C freezer, topmost drawer
</P><P>ready DNA Transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><H3><SPAN class="mw-headline" id="2012-07-06"><P style="background-color: rgb(240, 20, 70);">2012-07-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated sample</P><P><B>Investigators:</B> Mario, Tom S. </P><P><B>Time:</B> 2012-07-06</P><P><B>Materials:</B></P><UL><LI> Bunsen burner, Agar Plate with Chloramphenicol, 37 °C heat block, centrifuge</LI></UL><UL><LI> ligated sample (compare last step 07-05-2012)
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual
</P><P>Plate incubation start: 1:30 pm
</P><P><B>Results:</B></P><P>grown colonies
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-07-07"><P style="background-color: rgb(240, 20, 70);">2012-07-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Overnight culture of pSB1C3+CMV+AID carrying cells</P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-07-07 6pm </P><P><B>Materials:</B></P><P>LB medium, chloramphenicol 25 mg/ ml stock solution, plates with <I>E. coli</I> XL1 blue with plasmids: pSB1C3+CMV+AID </P><P><B>Method:</B> picking clones (2 per plate-&gt;Nr.1-6) and inoculation in 5 ml LB medium + 5µl chloramphenicol stock shaking over night at 37°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>glycerolstocks &amp; Miniprep
</P><H3><SPAN class="mw-headline" id="2012-07-08"><P style="background-color: rgb(240, 20, 70);">2012-07-08</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Glycerol stocks, Miniprep</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-07-08 11:00am</P><P><B>Materials:</B></P><P>Glycerol </P><P>Miniprep Kit</P><P>6x overnight culture (pSB1C3 with CMV+AID);
</P><P><B>Method:</B></P><P>Glycerol stock: 500 µL Glycerol 99,8 % + 500 µL overnight cultures --&gt; put in -80 °C freezer
</P><P>Miniprep (all 6 over night cultures (pSB1C3 with CMV+AID)
</P><P><B>Results:</B></P><P>6 Cryostocks are stored in the igem box in the -80°C freezer and Plasmids are stored in -20°C in the 4th drawer on styrofoam rack
</P><H3><SPAN class="mw-headline" id="2012-07-09"><P style="background-color: rgb(240, 20, 70);">2012-07-09</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Measuring DNA-concentration of plasmids from 2012-07-08</P><P><B>Investigators:</B> Mario, Tom S. </P><P><B>Time:</B> 2012-07-09</P><P><B>Materials:</B></P><UL><LI> Plasmids: pSB1C3 with CMV+AID (samples: 1, 2, 3, 4, 5, 6)</LI></UL><UL><LI> Nanodrop</LI></UL><UL><LI> NE-buffer</LI></UL><P><B>Method:</B></P><P>2 µL of each DNA-sample onto nanodrop (Ne-buffer blank) </P><P><B>Results:</B></P><P>DNA-concentrations:</P><P>1 = 290 ng/µL</P><P>2 = 361,2 ng/µL</P><P>3 = 316,8 ng/µL</P><P>4 = 360,5 ng/µL</P><P>5 = 392,5 ng/µL</P><P>6 = 390 ng/µL</P><P><B>Further tasks:</B></P><UL><LI> restriction enzyme digestion with XbaI und PstI</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B> Mario, Tom S. </P><P><B>Time:</B> 2012-07-09</P><P><B>Materials:</B></P><UL><LI> Plasmids: pSB1C3 with CMV+AID (samples: 1, 2, 3, 4, 5, 6)</LI></UL><UL><LI> Restriction enzymes (XbaI and PstI)</LI></UL><UL><LI> NE3-buffer</LI></UL><P><B>Method:</B></P><P>heat block (37 °C) </P><P>sample preparation: each DNA 25 µL + 3 µL NE3-buffer + 1 µL XbaI + 1 µL PstI</P><P>incubation of samples for 4 h at 37 °C</P><P><B>Results:</B></P><P>none
</P><P><B>Further tasks:</B></P><UL><LI> gel electrophoresis</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis</P><P><B>Investigators:</B> Mario, Tom S. </P><P><B>Time:</B> 2012-07-09</P><P><B>Aim</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P>cut samples:
</P><P>AID: Restriction enzymes (XbaI, PstI); NEB buffer 3
</P><P>CMV: Restriction enzymes (XbaI, PstI); NEB buffer 3
</P><P>AID+CMV: Restriction enzymes (XbaI, PstI); NEB buffer 3
</P><P><B>Method:</B></P><P>samples:</P><P>- 10 µL AID cut with XbaI and PstI + 2,5 µL loading dye</P><P>gel electrophoresis conditions:</P><P>10 µL of each sample into one big well</P><P>V = 120 V</P><P>duration roughly 95 minutes</P><P><B>Results:</B></P><P><B>Further Tasks:</B></P><P>overnight culture with AID+CMV sample 1, 2 and 3
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Overnight culture of AID+CMV carrying cells</P><P><B>Investigators:</B> Basia, Tom S. </P><P><B>Time:</B> 2012-07-09, 17:30</P><P><B>Materials:</B></P><P>LB medium </P><P>chloramphenicol 25 mg/ ml stock solution</P><P>glycerol stocks <I>E. coli</I> XL1 blue with Plasmids: pSB1C3 with AID+CMV of sample 1,2 and 3,
</P><P><B>Method:</B></P><P>Inoculation of cell samples in 3 ml LB medium </P><P>shaking over night at 37°C, 300 rpm </P><P><B>Further tasks:</B></P><P>Miniprep, preparative digestion, Plasmid ligation with polyA </P><H3><SPAN class="mw-headline" id="2012-07-10"><P style="background-color: rgb(240, 20, 70);">2012-07-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Miniprep of CMV+AID carrying plasmids</P><P><B>Investigators:</B> Tom S., Mario </P><P><B>Time:</B> 2012-07-10</P><P><B>Materials:</B></P><UL><LI> samples(CMV + AID: 1, 2, 3) - for detailed info check lab day 2012-07-09</LI></UL><UL><LI> Miniprep Kit</LI></UL><UL><LI> overnight culture (pSB1C3 with AID + CMV: samples 1, 2, 3,)
</LI></UL><P><B>Method:</B></P><P>Plasmid isolation via Kit (check manual)
</P><P>concentration measurement via nanodrop (2 µL sample)
</P><P><B>Results:</B></P><P>DNA - concentrations via nanodrop:</P><P>AID + CMV (1) = 303,2 ng/µL </P><P>AID + CMV (2) = 366,6 ng/µL </P><P>AID + CMV (3) = 378,5 ng/µL </P><P><B>Further Tasks:</B></P><P>preparative digestion (<B>use sample #3;</B> samples 1 and 2 for back up in -20 °C freezer topmost drawer)</P><P>fragment cut</P><H3><SPAN class="mw-headline" id="2012-07-11"><P style="background-color: rgb(240, 20, 70);">2012-07-11</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B> Tom S.</P><P><B>Time:</B> 2012-07-11 09:00</P><P><B>Materials:</B></P><UL><LI> pSB1C3 Vector with CMV+AID
</LI></UL><UL><LI> Restriction enzymes (SpeI, PstI); Fast Digest Green Buffer
</LI></UL><P><B>Method:</B></P><P>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL Fast Digest Green Buffer (incubation for 2 h)
</P><P><B>further tasks:</B></P><P>Gel electrophoresis</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Gel electrophoresis of cut pSB1C3 (CMV + AID) fragments</P><P><B>Investigators:</B> Tom S.</P><P><B>Time:</B> 2012-07-11 11:00</P><P><B>Materials:</B></P><UL><LI> cut sample (CMV + AID, PstI + SpeI)
</LI></UL><UL><LI> Gel electrophoresis material
</LI></UL><P><B>Method:</B></P><P>sample preparation: none
</P><P>loading into wll: 30 µL</P><P>duration: 70 minutes
</P><P><B>Results:</B></P><P>one band
</P><P><B>Further Tasks:</B></P><P>Gel extraction
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Gel extraction and measurement of DNA concentration</P><P><B>Investigators:</B> Chris, Mario</P><P><B>Time:</B> 2012-07-11 13:00 - 14:00</P><P><B>Materials:</B></P><UL><LI> Analytic Jena gel extraction kit
</LI></UL><UL><LI> measurement of DNA concentration via nanodrop
</LI></UL><P><B>Method:</B></P><P>Gel extraction via manual</P><P><B>Results:</B></P><P>DNA concentration via nanodrop: 98.7 ng/µL --&gt;2051640 Da (2051,64 kDa)--&gt; c=48.1 nM
</P><P><B>Further Tasks:</B></P><P>Ligation of fragment with Poly-A
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Ligation CMV+AID in pSB1C3 (cut:Spe1 and Pst1) with hGH-polyA (cut:Xba1 and Pst1)</P><P><B>Investigators:</B> Chris, Mario</P><P><B>Time:</B> 2012-07-11 16:45 - 17:30</P><P><B>Materials:</B></P><P>digested fragments: CMV+AID in pSB1C3 (cut:Spe1 and Pst1) c=48.1 nM , hGH-polyA (cut:Xba1 and Pst1) c= 15,1 ng/µL -&gt; 48,5 nM
</P><P><B>Method:</B></P><P>mix 1µL CMV+AID in pSB1C3 (cut:Spe1 Pst1) c=48.1 nM, 3µL hGH-polyA (cut:Xba1 Pst1) c=48.5 nM, 1µl T4 Ligase, 1µ 10x Buffer,4 µL H20</P><P>incubate 1.5 h
</P><P><B>Results:</B> not visible
</P><P><B>Further Tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of XL1 Blue with CMV+AID+hGH-polyA in pSB1C3 </P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-07-11 finished:18 Uhr</P><P><B>Materials:</B></P><P>LB medium, <I>E. coli</I> XL1 Blue, ligation product (CMV+AID+hGH-polyA), agar-LB-paltes with 1:1000 chloramphenicol
</P><P><B>Method:</B></P><P>transformation - standard operating procedures</P><P><B>Results:</B> two plates with transformed <I>E. coli</I> (CMV+AID+polyA)</P><P><B>Further Tasks:</B></P><P>picking colonies &amp; inoculate 5 ml overnight culture
</P><H3><SPAN class="mw-headline" id="2012-07-12"><P style="background-color: rgb(240, 20, 70);">2012-07-12</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Overnight culture of pSB1C3 CMV+AID+hGH-polyA, CMV+AID, CMV, hGH-polyA and pSB1A3 AID carrying cells</P><P><B>Investigators:</B> Tom S.</P><P><B>Time:</B> 2012-07-12 6pm</P><P><B>Materials:</B></P><P>LB medium, chloramphenicol 25 mg/ ml stock solution, plates with <I>E. coli</I> XL1 blue with plasmids: pSB1C3+CMV+AID, glycerol stocks: pSB1C3 with AID+CMV, CMV, hGH-polyA and pSB1A3 with AID</P><P><B>Method:</B> picking clones(3 per plate-&gt;Nr.1-6) and inoculation in 5 ml LB medium + 5µl chloramphenicol stock shaking over night at 37°C, 300 rpm, approx. 16 hours, samples from glycerolstocks in 3 ml LB + 3 µL chloramphenicol or 3µL Amp</P><P><B>Further tasks:</B></P><P>glycerol stocks &amp; Miniprep
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Planing BBa_K929001</P><P><B>Investigators:</B> Tom S., Chris, Basia, Rico, Mario, Kevin </P><P><B>Aim:</B> planing how to digest and ligate the vectors for BBa_K929001 </P><P><B>Material:</B> Genious </P><P><B>Results:</B></P><P>pSB1C3 with CMV -&gt; (cut with SpeI and XbaI) 2072 bp (pSB1C3 backbone) + 662 bp (rest)
</P><P>PCR-amplificate -&gt; (cut with SpeI and XbaI) 597 bp (modified AID insert)
</P><P><B>Further tasks:</B></P><P>design and ordering of primers, practical part
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Primer design and ordering for BBa_K929001</P><P><B>Investigators:</B> Tom S., Rico</P><P><B>Time:</B> 2012-07-12 7pm</P><P>Primer (forward) with XbaI recognition site, kozak consensus sequence, NLS:
</P><P>ATCTAGAGCCGCCACCATGGGACCCAAGAAGAGGAAGGTGATGGACAGCCTCTTGATGAACCGGAGG
</P><P>Primer (reverse, complement)with AgeI and SpeI recognition site:
</P><P>CCACTAGTATTAACCGGTGGGCAAAAGGATGCGCCGAAGC
</P><H3><SPAN class="mw-headline" id="2012-07-13"><P style="background-color: rgb(240, 20, 70);">2012-07-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Mini Prep of WT Plasmids, nanodrop</P><P><B>Investigators:</B> Tom S., Mario</P><P><B>Time:</B> 2012-07-13 10am</P><P><B>Materials:</B></P><P>Miniprep Kit</P><P>Overnight culture of AID-WT tranfected <I>E. coli</I> strains</P><P><B>Method:</B> Kit via manual</P><P><B>Results:</B></P><P>DNA-concentrations via nanodrop:</P><P>WT-AID 1: 387,2 ng/µL</P><P>WT-AID 2: 453,0 ng/µL</P><P>WT-AID 3: 415,8 ng/µL</P><P>WT-AID 4: 445,5 ng/µL</P><P>WT-AID 5: 474,1 ng/µL</P><P>WT-AID 6: 645,1 ng/µL</P><P>AID: 393,5 ng/µL</P><P>CMV: 188,0 ng/µL</P><P>CMV+AID 221,9 ng/µL</P><P>hGH 318,2 ng/µL</P><P><B>Further tasks:</B></P><P>digestion and gelelectrophoresis
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B> Tom S.</P><P><B>Time:</B> 2012-07-11 09:00</P><P><B>Materials:</B></P><UL><LI> pSB1C3 Vectors with CMV+AID+hGH-polyA, AID, CMV, hGH, CMV+AID
</LI></UL><UL><LI> Restriction enzymes (SpeI, PstI); Fast Digest Green Buffer
</LI></UL><P><B>Method:</B></P><P>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL Fast Digest Green Buffer (incubation for 2 h)
</P><P><B>further tasks:</B></P><P>Gel electrophoresis</P><H3><SPAN class="mw-headline" id="2012-07-16"><P style="background-color: rgb(240, 20, 70);">2012-07-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Gel electrophoresis of cut ligation samples (WT AID - CMV+AID+hGH-polyA)</P><P><B>Investigators:</B> Tom S.</P><P><B>Time:</B> 2012-07-16; </P><P><B>Materials:</B></P><P>gel electrophoresis equipment</P><P>samples</P><P><B>Method:</B></P><P>loading wells with 10 µL of each cut sample (ca. 600-800 ng DNA per sample) </P><P>gel ectrophoresis standard operating procedure</P><P><B>Results:</B></P><P><B>Further tasks:</B></P><P>sequencing
</P><H3><SPAN class="mw-headline" id="2012-07-23"><P style="background-color: rgb(240, 20, 70);">2012-07-23</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR of AID+NLS+Kozak sequence</P><P><B>Investigators:</B></P><P>Basia, Tom S. </P><P><B>Aim:</B></P><UL><LI> amplification of the AID with inserted Kozak sequence and NLS sequence via PCR
</LI></UL><P><B>Materials:</B></P><UL><LI> Phusion, template (AID insert), Primers designed by Tom S. and Rico on 12.07.2012, dNTPs, Polymerase)
</LI></UL><UL><LI> PCR clean-up kit
</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [µL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing + elongation </TD><TD> 72 </TD><TD> 45 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 25 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> ∞ </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Results:</B> 125ng/µl - 1st sample, 135ng/µl 2nd sample
</P><P><B>Further tasks:</B></P><UL><LI> digestion + agarose gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Primer design and ordering for sequencing BBa_K929001 and BBa_K929003</P><P><B>Investigators:</B> Tom S., Rico</P><P><B>Time:</B> 2012-07-23 </P><P>Primer bind on pSB1C3-vector (left next to backbone-prefix): </P><P>GGCGTATCACGAGGCAG</P><P>Primer (reverse, complement) bind on pSB1C3-vector (right next to backbone-suffix):</P><P>CGAGTCAGTGAGCGAGG</P><H3><SPAN class="mw-headline" id="2012-07-25"><P style="background-color: rgb(240, 20, 70);">2012-07-25</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B> Tom S.</P><P><B>Time:</B> 2012-07-25 08:30</P><P><B>Materials:</B></P><P>pSB1C3 Vector with CMV
</P><P>2 PCR-products - AID without NES, with NLS+Kozak Sequence (theoretically the same) (SpeI, XbaI; Fast Digest); Fast Digest Green Buffer
</P><P><B>Method:</B></P><P>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL Fast Digest Green Buffer (incubation for 2 h)-&gt; for the pSB1C3 backbone </P><P>preparative digestion: 18 µL DNA + 1 µL of each enzyme + 2 µL Fast Digest Green Buffer (incubation for 2 h)-&gt; for the PCR-products
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis</P><P><B>Investigators:</B> Tom S. </P><P><B>Time:</B> 2012-07-25</P><P><B>Aim</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P><B>Method:</B></P><P>cut samples:
</P><UL><LI> pSB1C3 backbone: Restriction enzymes (XbaI, SpeI; Fast Digest); Fast Digest buffer
</LI></UL><UL><LI> PCR-products (AID without NES, with NLS+Kozak Sequence): Restriction enzymes (XbaI, SpeI; Fast Digest); Fast Digest buffer
</LI></UL><P>wells loaded with 30 or 22 µL of digested samples via gel electrophoresis - standard operating procedure</P><P>gel electrophoresis conditions:</P><P>V = 120 V</P><P>duration roughly 50 minutes</P><P><B>Results:</B></P><P>Marked fragment was cut out of the gel and transferred into 1,5 mL Eppendorf tube</P><P><B>Further Tasks:</B></P><P>Gel extraction
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction of pSB1C3 backbone and modified AID insert (AID without NES, with NLS+Kozak Sequence)</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Gel extraction of pSB1C3 backbone and modified AID insert
</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up - Kit, thermo heater, nanodrop
</P><P><B>Method:</B></P><P>DNA extraction: according to the manual
</P><P><B>Results:</B></P><P>DNA-concentrations via nanodrop:</P><P>pSB1C3 backbone = 83,6 ng/µL -&gt; 65,7 nM (with mass conc. of 1273,3 kDa)</P><P>PCR-product 1 = 74,0 ng/µL -&gt; 201,5 nM (with mass conc. of 367,18 kDa)</P><P>PCR-product 1 = 77,5 ng/µL -&gt; 211,1 nM (with mass conc. of 367,18 kDa)</P><P>location: -20 °C freezer, topmost drawer
</P><P>ready DNA for ligation
</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of PCR-product (AID without NES, with NLS+Kozak Sequence) and pSB1C3 backbone</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Ligation of PCR-product and pSB1C3 backbone
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, samples(PCR-product 1 and 2 + pSB1C3 backbone)-&gt; PCR products 1 and 2 are theoretically the same
</P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation:
</P><UL><LI> 2 µL (PCR-product) c=75,8 ng/µL(206,4 nM)-&gt; 41,3 nM
</LI></UL><UL><LI> 2 µL (pSB1C3 backbone) c=83,6 ng/µL(65,7 nM) -&gt; 13,2 nM
</LI></UL><UL><LI> 1 µL (T4 DNA-Ligase)
</LI></UL><UL><LI> 1 µL 10x T4 DNA Ligase Buffer
</LI></UL><UL><LI> 4 µL (DNase free water)
</LI></UL><P>incubation of sample 1,5 h at 22°C</P><P><B>Results:</B></P><P>samples ligated</P><P>location: -20 °C freezer, topmost drawer
</P><P>ready DNA for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated sample</P><P><B>Investigators:</B> Tom S. </P><P><B>Time:</B> 2012-07-25</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Chloramphenicol, 37 °C heater, centrifuge</LI></UL><UL><LI> ligated sample (compare last step 25-07-2012)
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual
</P><P>Plate incubation start: 5:00 pm
</P><P><B>Results:</B></P><P>ready for growing mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-07-26"><P style="background-color: rgb(240, 20, 70);">2012-07-26</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">picking clones &amp; inoculation</P><P><B>Investigators:</B> Sascha </P><P><B>Time:</B> 2012-07-27 6 pm </P><P><B>Materials:</B></P><P>LB medium, chloramphenicol 25 mg/ ml stock solution, plates with <I>E. coli</I> XL1 blue with plasmids: pSB1C3+pcr-products(AID with NLS,without NES+Kozak sequence), glycerol stocks: pSB1C3 with CMV</P><P><B>Method:</B> picking clones(3 per plate) and inoculation in 5 ml LB medium + 5µl chloramphenicol stock shaking over night at 37°C, 300 rpm, approx. 16 hours, samples of glycerol stocks in 5 ml LB + 5 µL chloramphenicol</P><P><B>Further tasks:</B></P><P>glycerolstocks &amp; Miniprep
</P><H3><SPAN class="mw-headline" id="2012-07-27"><P style="background-color: rgb(240, 20, 70);">2012-07-27</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Miniprep</P><P><B>Investigators:</B> Tom S. </P><P>Time:
</P><P>2012-07-03 8 am
</P><P>Materials:
</P><P>Glycerol, Miniprep Kit, over night culture (pSB1C3 with CMV); overnight culture (pSB1C3 with modified AID)
</P><P>Method:
</P><P>Glycerol stock: 300 µL Glycerol 99,8 % + 700 µL over night cultures --&gt; put in -80 °C freezer
</P><P>Miniprep (both overnight culture (pSB1C3 with CMV) and overnight cultures (pSB1C3 with PCR 1 colony 1-3 and PCR 2 colony 1-3)
</P><P>Results:
</P><P>DNA - concentrations via nanodrop:
</P><P>PCR1 C1= 163.7 ng/µL </P><P>PCR1 C2= 154.7 ng/µL </P><P>PCR1 C3= 165.5 ng/µL </P><P>PCR2 C1= 117.1 ng/µL </P><P>PCR2 C2= 164.7 ng/µL </P><P>PCR2 C3= 94,4 ng/µL </P><P>CMV= 144.2 ng/µL
</P><P><B>Further tasks:</B></P><P>preparative digestion
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B>Chris</P><P><B>Time:</B> 2012-07-27 11:30</P><P><B>Materials:</B></P><P>pSB1C3 Vector with CMV (SpeI, PstI; Fast Digest); Fast Digest Green Buffer</P><P>PCR 1 C 3 and PCR 2 C 2 in pSB1C3 vector(PstI, XbaI; Fast Digest); Fast Digest Green Buffer
</P><P><B>Method:</B></P><UL><LI>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL Fast Digest Green Buffer (incubation for 2,5 h)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis</P><P><B>Investigators:</B> Tom S. </P><P><B>Time:</B> 2012-07-27</P><P><B>Aim:</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P>cut samples:
</P><UL><LI> digested CMV: Restriction enzymes (SpeI, PstI; Fast Digest); Fast Digest buffer
</LI></UL><UL><LI> digested PCR-products: Restriction enzymes (PstI, XbaI; Fast Digest); Fast Digest buffer
</LI></UL><P><B>Method:</B></P><P>samples:</P><P>loading wells with 30µl of digested samples via gel electrophoresis, standard operating procedure</P><P>gel electrophoresis conditions:</P><P>30 µL of each samples into one big well</P><P>V = 120 V</P><P>duration 72 minutes</P><P><B>Results:</B></P><P>Marked fragments were cut out of the gel and transferred into 1,5 mL Eppendorf tube for gel extraction</P><P><B>Further Tasks:</B></P><P>Gel extraction
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction of digested CMV+backbone and PCR-products (AID without NES, with NLS+Kozak sequence) insert</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Gel extraction of digested CMV+backbone and PCR-products insert
</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up - Kit, heat block, nanodrop
</P><P><B>Method:</B></P><P>extraction of DNA: according to the manual
</P><P><B>Results:</B></P><P>DNA-concentrations via nanodrop:</P><P>CMV+backbone = 85,6 ng/µL -&gt; 50,9 nM (with mass conc. of 1681,3 kDa)</P><P>PCR 1 C 3 = 33,7 ng/µL -&gt; 89,1 nM (with mass conc. of 378,3 kDa)</P><P>PCR 2 C 2 = 29,4 ng/µL -&gt; 77,7 nM (with mass conc. of 378,3 kDa)</P><P>location: -20 °C freezer, topmost drawer
</P><P>ready DNA for Ligation
</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><H3><SPAN class="mw-headline" id="2012-07-28"><P style="background-color: rgb(240, 20, 70);">2012-07-28</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of PCR-products(AID without NES, with NLS+Kozak sequence) and CMV+backbone</P><P><B>Investigators:</B></P><P>Basia </P><P><B>Aim:</B></P><P>Ligation of PCR-products and CMV+backbone
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, samples(PCR 1 C 3 or PCR 2 C 2 and CMV+backbone)
</P><P><B>Method:</B></P><P>DNA fragment ligation: according to the manual</P><P>sample preparation:
</P><UL><LI> 4 µL PCR 1 C3 c=33,7 ng/µL(89,1 nM)-&gt; 35,6 nM; (for the other sample 4µL PCR 2 C 2 c=29,4 ng/µL(77,7 nM)-&gt; 31,1 nM
</LI></UL><UL><LI> 2 µL (CMV+backbone) c=85,6 ng/µL(50,9 nM) -&gt; 10,2 nM
</LI></UL><UL><LI> 1 µL (T4 DNA-Ligase)
</LI></UL><UL><LI> 1 µL 10x T4 DNA Ligase Buffer
</LI></UL><UL><LI> 2 µL (DNase free water)
</LI></UL><P>incubation of sample 1,5 h at 22°C</P><P><B>Results:</B></P><P>location: -20°C freezer, topmost drawer
</P><P>ready DNA Transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated sample</P><P><B>Investigators:</B> Basia </P><P><B>Time:</B> 2012-07-28 12:30</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Chloramphenicol, 37°C heat block, centrifuge</LI></UL><UL><LI> ligase sample (from last step 28.07.2012)
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation according to the manual
</P><P>Plate incubation start: 14:30 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-07-29"><P style="background-color: rgb(240, 20, 70);">2012-07-29</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Overnight culture of <I>E. coli</I> containing plasmids with PCR products (AID without NES, with NLS+Kozak sequence) and CMV promoter</P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-07-29 7 pm</P><P><B>Materials:</B></P><P>LB medium, chloramphenicol 25 mg/ ml stock solution, plates with <I>E. coli</I> XL1 blue with plasmids: CMV+PCR1C3 and CMV+PCR2C2</P><P><B>Method:</B> picking clones(2 per plate) and inoculation in 5 ml LB medium + 5µl chloramphenicol stock shaking over night at 37°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>glycerol stocks &amp; Miniprep
</P><H3><SPAN class="mw-headline" id="2012-07-30"><P style="background-color: rgb(240, 20, 70);">2012-07-30</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of BBa_E0040 (wild-type GFP) from Distribution Plate 1 Kit 2012</P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-07-30 10:30</P><P><B>Materials:</B></P><P>Bunsen Burner, Agar Plate with ampicillin, icebox, competent <I>E. coli</I> cells (XL 1 Blue)
</P><P><B>Method:</B></P><P>Transformation according to the manual
</P><P>Plate incubation start: 13:30 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Glycerol stocks &amp; miniprep of CMV+AID without NES, with NLS+Kozak sequence containing plasmids</P><P><B>Investigators:</B> Tom S. </P><P>Time:
</P><P>2012-07-03 8:30 am
</P><P>Materials:
</P><P>Glycerol, Miniprep Kit, overnight cultures (CMV+PCR1C3C1-2; CMV+PCR2C2C1-2)
</P><P>Method:
</P><P>Glycerol stock: 300 µL Glycerol 99,8 % + 700 µL overnight cultures --&gt; put in -80 °C freezer
</P><P>Miniprep overnight cultures (CMV+PCR1C3C1-2; CMV+PCR2C2C1-2) procedure according to the manual
</P><P>Results:
</P><P>DNA concentrations via nanodrop:</P><P>PCR1C3 C1= 389,5 ng/µL </P><P>PCR1C3 C2= 394,3 ng/µL </P><P>PCR2C2 C1= 409,8 ng/µL </P><P>PCR2C2 C2= 383,4 ng/µL </P><P><B>Further tasks:</B></P><P>preparative digestion with PCR1C3 C2 and PCR2C2 C1
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B>Chris</P><P><B>Time:</B> 2012-07-30 11:00</P><P><B>Materials:</B></P><P>pSB1C3 Vector with hGH-PolyA (XbaI, PstI; Fast Digest); Fast Digest Green Buffer</P><P>PCR1C3 C2 and PCR2C2 C1 in pSB1C3 vector - AID without NES, with NLS+Kozak sequence+CMV(PstI, SpeI; Fast Digest); Fast Digest Green Buffer
</P><P><B>Method:</B></P><P>preparative digestion: 25 µL DNA + 1 µL of each enzyme + 3 µL Fast Digest Green Buffer (incubation for 2 h)
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis &amp; Gel extraction</P><P><B>Investigators:</B> Chris </P><P><B>Time:</B> 2012-07-30</P><P><B>Aim:</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P>cut samples:
</P><P>digested hGH-PolyA: Restriction enzymes (XbaI, PstI; Fast Digest); Fast Digest buffer</P><P>digested PCR-products: Restriction enzymes (PstI, SpeI; Fast Digest); Fast Digest buffer
</P><P><B>Method:</B></P><P>gel ectrophoresis - standard operating procedure</P><P>gel electrophoresis conditions:</P><P>30 µL of each samples into one big slot</P><P>V = 120 V</P><P>duration roughly 60 minutes</P><P><B>Results:</B></P><P>Marked fragments were cut out of the gel and transferred into 1,5 mL Eppendorf tubes</P><P>Gel Extraction: </P><P>final concentrations: </P><P>CMV+PCR1C3C2(3318 nt, MW=2048.48 kDa) : 188.1 ng/µl (91.8 nM)</P><P>CMV+PCR2C2C1 : 182.2 ng/µl (88.9 nM)</P><P>hGH-polyA (495 nt, MW=305.4 kDa): 26.1 ng/µl (85 nM)</P><P><B>Further Tasks:</B></P><P>Ligation, transformation
</P><H3><SPAN class="mw-headline" id="2012-07-31"><P style="background-color: rgb(240, 20, 70);">2012-07-31</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Planing BBa_K929003</P><P><B>Investigators:</B> Tom S., Chris, Basia, Rico, Mario, Kevin </P><P><B>Aim:</B> planing how to digest and ligate the vectors for BBa_K929003 </P><P><B>Material:</B> Genious </P><P><B>Results:</B></P><UL><LI> pSB1C3 with CMV+ mod. AID (without NES, with NLS) -&gt; cut with AgeI and SpeI, 3322 bp - pSB1C3 backbone with CMV + mod. AID, 14 bp - rest
</LI></UL><UL><LI> eGFP -&gt; cut with SpeI and NgoMIV, 731 bp - eGFP insert, 2074 bp - pSB1C3 backbone
</LI></UL><UL><LI> pSB1C3 with CMV+ mod. AID + eGFP -&gt; cut with Spe1 and Pst1, 4035 bp - pSB1C3 backbone with CMV + mod. AID+eGFP, 18 bp - rest
</LI></UL><UL><LI> hGH-polyA -&gt; cut with Xba1 and Pst1, 505 bp - hGH-polyA insert, 2053 - pSB1C3 backbone
</LI></UL><P><B>Further tasks:</B></P><UL><LI> practical part
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">inoculation of GFP in pSB1A2 (BBa_E0040) &amp; eGFP (from Sven eGFP in pSB1C3 in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A>-like <A href="http://partsregistry.org/Part:BBa_K404316" class="external text" rel="nofollow">BBa_K404316</A></P><P><B>Investigators:</B> Tom S., Chris </P><P><B>Time:</B> 2012-07-31 3pm</P><P><B>Materials:</B></P><P>LB medium, chloramphenicol 25 mg/ ml stock solution, ampicillin stock solution, plates with <I>E. coli</I> XL1 blue with plasmid: GFP in pSB1A2 (BBa_E0040), glycerol stock from Sven: eGFP in pSB1C3 in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A></P><P><B>Method:</B> picking clones (1 per plate from 2 plates) GFP in pSB1A2 (BBa_E0040) and inoculation in 5 ml LB medium + 5µl ampicillin stock, 2 times inoculation of glycerol stock (eGFP in pSB1C3 in <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A>) in 5mL LB medium + 5µL chloramphenicol stock, shaking over night at 37°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>glycerol stocks &amp; Miniprep
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">preparation of WT-AID &amp; modified AID (without NES, with NLS) in pSB1C3 for sequencing &amp; send to GATC</P><P><B>Investigators:</B> Tom S., Chris </P><P><B>Time:</B> 2012-07-31 7pm</P><P><B>Materials:</B></P><P>20 µl primer c=10µM for each sequencing, 20 µl sample with concentration around 70 ng/µl </P><P><B>Method:</B> mark the samples with bar code stickers &amp; order the sequencing from GATC, put the cups into the orange boxes</P><P>samples: </P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>barcode</B></TD><TD><B>primer</B></TD><TD><B>sample</B></TD><TD><B>barcode</B></TD><TD><B>primer</B></TD></TR><TR><TD>WT AID clone 1 (Cmv+WtAID+PolyA in pSB1C3)</TD><TD>CC0697</TD><TD>forward pSB1C3</TD><TD>WT AID clone 1 </TD><TD>CC0709</TD><TD>reverse pSB1C3</TD></TR><TR><TD>WT AID clone 2</TD><TD>CC0698</TD><TD>forward pSB1C3</TD><TD>WT AID clone 2</TD><TD>CC0710</TD><TD>reverse pSB1C3</TD></TR><TR><TD>WT AID clone 3 </TD><TD>CC0699</TD><TD>forward pSB1C3</TD><TD>WT AID clone 3 </TD><TD>CC0711</TD><TD>reverse pSB1C3</TD></TR><TR><TD>WT AID clone 4 </TD><TD>CC0700</TD><TD>forward pSB1C3</TD><TD>WT AID clone 4 </TD><TD>CC0712</TD><TD>reverse pSB1C3</TD></TR><TR><TD>WT AID clone 5 </TD><TD>CC0701</TD><TD>forward pSB1C3</TD><TD>WT AID clone 5 </TD><TD>CC0713</TD><TD>reverse pSB1C3</TD></TR><TR><TD>WT AID clone 6 </TD><TD>CC0702</TD><TD>forward pSB1C3</TD><TD>WT AID clone 6 </TD><TD>CC0714</TD><TD>reverse pSB1C3</TD></TR><TR><TD>PCR1 C1 (modified AID- Kozak sequence+NLS+AID without NES+Age1-restriction site in pSB1C3) </TD><TD>CC0703</TD><TD>forward pSB1C3</TD><TD>PCR1 C1 </TD><TD>CC0715</TD><TD>reverse pSB1C3</TD></TR><TR><TD>PCR1 C2 </TD><TD>CC0704</TD><TD>forward pSB1C3</TD><TD>PCR1 C2 </TD><TD>CC0716</TD><TD>reverse pSB1C3</TD></TR><TR><TD>PCR1 C3 </TD><TD>CC0705</TD><TD>forward pSB1C3</TD><TD>PCR1 C3 </TD><TD>CC0717</TD><TD>reverse pSB1C3</TD></TR><TR><TD>PCR2 C1 </TD><TD>CC0706</TD><TD>forward pSB1C3</TD><TD>PCR2 C1 </TD><TD>CC0718</TD><TD>reverse pSB1C3</TD></TR><TR><TD>PCR2 C2 </TD><TD>CC0707</TD><TD>forward pSB1C3</TD><TD>PCR2 C2 </TD><TD>CC0719</TD><TD>reverse pSB1C3</TD></TR><TR><TD>PCR2 C3 </TD><TD>CC0708</TD><TD>forward pSB1C3</TD><TD>PCR2 C3 </TD><TD>CC0720</TD><TD>reverse pSB1C3</TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><P>alignment
</P><H2><SPAN class="mw-headline" id="Antik.C3.B6rper">Antikörper</SPAN></H2><H3><SPAN class="mw-headline" id="2012-07-06_2"><P style="background-color: rgb(240, 20, 70);">2012-07-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Sequencing of pcDNA5-FRT and pOG44 by GATC</P><P><B>Investigators:</B> Sascha </P><P><B>Aim:</B> Sequencing of invitrogen vectors pcDNA5-FRT and pOG44</P><P><B>Materials:</B></P><UL><LI> Geneious
</LI></UL><UL><LI> Lablife Sequences of pcDNA5-FRT and pOG44
</LI></UL><UL><LI> GATC</LI></UL><P><B>Method:</B> forward sequencing using GATC-CMV-forward-primer </P><UL><LI> pcdna5frt_cmv_forward-CMV-F</LI></UL><UL><LI> pog44_cmv_forward-CMV-F </LI></UL><P><B>Further tasks:</B> checking sequences </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Primer-design for assembly-PCR, generating geneconstruct: scFv-TEV-TMD-EYFP </P><P><B>Investigators:</B> Sascha </P><P><B>Aim:</B> Design of primer for assembly-pcr of geneconstruct scFv-TEV-TMD-EYFP </P><P><B>Materials:</B></P><UL><LI> Lablife Sequences of pcDNA5-FRT and pOG44
</LI></UL><UL><LI> Databases
</LI></UL><UL><LI> Oligocalc
</LI></UL><UL><LI> Wordpad
</LI></UL><P><B>Method:</B></P><UL><LI> P1_Signalp_N-term</LI></UL><UL><LI> P2_Signalp_C-term</LI></UL><UL><LI> P3_TMD-N-term</LI></UL><UL><LI> P4_TMD-C-term/N-YFP</LI></UL><UL><LI> P5_YFP-N-term/TMD-C-term</LI></UL><UL><LI> P6_YFP-C-term</LI></UL><UL><LI> P7_Gesamtanfang</LI></UL><UL><LI> P8_Gesamtende</LI></UL><P><B>Further tasks:</B></P><UL><LI> checking primer with Geneious
</LI></UL><UL><LI> adapt primer to NEB-Phusion-polymerase conditions </LI></UL><H3><SPAN class="mw-headline" id="2012-07-07_2"><P style="background-color: rgb(240, 20, 70);">2012-07-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Primer-design for assembly-PCR, generating geneconstruct: scFv-TEV-TMD-EYFP </P><P><B>Investigators:</B> Sascha </P><P><B>Aim:</B> Design of primer for assembly-pcr of geneconstruct scFv-TEV-TMD-EYFP </P><P><B>Materials:</B></P><UL><LI> Lablife Sequences of pcDNA5-FRT and pOG44
</LI></UL><UL><LI> Databases
</LI></UL><UL><LI> Oligocalc
</LI></UL><UL><LI> Wordpad
</LI></UL><P><B>Method:</B> forward sequencing using GATC-CMV-forward-primer </P><UL><LI> P1_Signalp_N-term</LI></UL><UL><LI> P2_Signalp_C-term</LI></UL><UL><LI> P3_TMD-N-term</LI></UL><UL><LI> P4_TMD-C-term/N-YFP</LI></UL><UL><LI> P5_YFP-N-term/TMD-C-term</LI></UL><UL><LI> P6_YFP-C-term</LI></UL><UL><LI> P7_Gesamtanfang</LI></UL><UL><LI> P8_Gesamtende</LI></UL><P><B>Further tasks:</B></P><UL><LI> checking primer with Geneious
</LI></UL><UL><LI> adapt primer to NEB-Phusion-polymerase conditions </LI></UL><H3><SPAN class="mw-headline" id="2012-07-10_2"><P style="background-color: rgb(240, 20, 70);">2012-07-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Checking of Sequencing of GATC-results of pcDNA5-FRT and pOG44 </P><P><B>Investigators:</B> Sascha </P><P><B>Aim:</B> Control of pcDNA5-FRT- and pOG44-sequences</P><P><B>Materials:</B></P><UL><LI> Geneious
</LI></UL><UL><LI> Lablife Sequences of pcDNA5-FRT and pOG44
</LI></UL><UL><LI> GATC-viewer
</LI></UL><UL><LI> NCBI BLASTn</LI></UL><P><B>Results:</B></P><UL><LI> sequences are correct
</LI></UL><H3><SPAN class="mw-headline" id="2012-07-12_2"><P style="background-color: rgb(240, 20, 70);">2012-07-12</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> ordering of primer for assembly-PCR, generating geneconstruct: scFv-TEV-TMD-EYFP </P><P><B>Investigators:</B> Sascha, Maria </P><P><B>Aim:</B> ordering of primer for assembly-pcr of geneconstruct scFv-TEV-TMD-EYFP </P><P><B>Materials:</B></P><UL><LI> Lablife Sequences of pcDNA5-FRT and pOG44
</LI></UL><UL><LI> Databases
</LI></UL><UL><LI> Oligocalc
</LI></UL><UL><LI> Wordpad
</LI></UL><UL><LI> Geneious
</LI></UL><UL><LI> NEB-Calculator for high-fidelity Phusion-polymerase
</LI></UL><P><B>Method:</B> forward sequencing using GATC-CMV-forward-primer </P><UL><LI> P1_Signalp_N-term</LI></UL><UL><LI> P2_Signalp_C-term</LI></UL><UL><LI> P3_TMD-N-term</LI></UL><UL><LI> P4_TMD-C-term/N-YFP</LI></UL><UL><LI> P5_YFP-N-term/TMD-C-term</LI></UL><UL><LI> P6_YFP-C-term</LI></UL><UL><LI> P7_Gesamtanfang</LI></UL><UL><LI> P8_Gesamtende</LI></UL><P><B>Further tasks:</B></P><UL><LI> extension-assembly-pcr
</LI></UL><H3><SPAN class="mw-headline" id="2012-07-15"><P style="background-color: rgb(240, 20, 70);">2012-07-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Retransformation with scFv, transmembrane domain and YFP</P><P><B>Investigators:</B> Maria </P><P><B>Aim:</B> Retransformation with scFv anti-EGFR, transmembrane domain BBa_KI157010, YFP BBa_E0030 </P><P><B>Date/Time:</B> 15th July 2012, 2:30 – 4:30 pm </P><P><B>Materials:</B> competent <I>E. coli</I> cells XL1 Blue, Biobricks BBa_K157010 and BBa_E0030, scFv anti-EGFR, 42°/37° C shaker, centrifuge</P><P><B>Method:</B> see transformation protocol </P><P><B>Results:</B> colonies on Amp plates with scFv-, transmembrane domain- and YFP- plasmid </P><P><B>Further tasks:</B> picking clones and overnight culture (16th July)</P><H3><SPAN class="mw-headline" id="2012-07-16_2"><P style="background-color: rgb(240, 20, 70);">2012-07-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Overnight culture of scFv, YFP and transmembrane domain carrying cells</P><P><B>Investigators:</B> Maria</P><P><B>Time:</B> 2012-07-16; 6:30 - 7 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Amp. 25 mg/ ml stock solution</LI></UL><UL><LI> Plasmids: scFv; BBa-KI57000 with transmembrane domain; pSB1AK3 with YFP
</LI></UL><P><B>Method:</B></P><P>Inoculation of cell sample each in 5 ml LB medium with Amp</P><P>shaking over night at 37°C, 300 rpm, approx. 17 hours </P><P><B>Further tasks:</B></P><UL><LI> Miniprep </LI></UL><H3><SPAN class="mw-headline" id="2012-07-17"><P style="background-color: rgb(240, 20, 70);">2012-07-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Mini Prep of YFP and transmembrane domain

</P><P><B>Investigators:</B> Stefan, Tarek, Kerstin</P><P><B>Time:</B> 2012-07-17; 2 - 3.30 pm</P><P><B>Materials:</B></P><UL><LI> overnight cultures from 2012-07-16 </LI></UL><UL><LI> GeneJET Plasmid Miniprep Kit (Thermo Scientific)</LI></UL><UL><LI> Plasmids: BBa-KI57000 with transmembrane domain; pSB1AK3 with YFP
</LI></UL><P><B>Method:</B></P><P>Miniprep according to Kit</P><P><B>Note:</B></P><P>overnight culture of retransformation of scFv did not grow --&gt; Resistance was not known in scFv vector, screening was done with ampicillin
</P><P><B>Further Tasks:</B></P><P>PCR to produce sp-scFv-tmd-yfp construct
</P><H3><SPAN class="mw-headline" id="2012-07-20"><P style="background-color: rgb(240, 20, 70);">2012-07-20</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: extension-assembly-pcr of scFv-TEV-TMD-EYFP geneconstruct

</P><P><B>Investigators:</B> Stefan, Sascha</P><P><B>Materials:</B></P><P><B>Materials:</B>scFv anti-EGFR, BBa-KI57000 with transmembrane domain; BBa_E0030 with EYFP</P><UL><LI> P1_Signalp_N-term</LI></UL><UL><LI> P2_Signalp_C-term</LI></UL><UL><LI> P3_TMD-N-term</LI></UL><UL><LI> P4_TMD-C-term/N-YFP</LI></UL><UL><LI> P5_YFP-N-term/TMD-C-term</LI></UL><UL><LI> P6_YFP-C-term</LI></UL><UL><LI> P7_Gesamtanfang</LI></UL><UL><LI> P8_Gesamtende</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 50µl PCR mix of each plasmid= 10µl HF-Buffer; 1µl dNTPs(10mM); forward and reverse primer each 2,5µl (10µM); template DNA: 1µl of 200ng/µl scFv --&gt; 200ng, 5µl of BBa_E0030 with EYFP (46,3ng/µl--&gt; ca. 230ng),5µl of BBa-KI57000 with transmembrane domain (43,9ng/µl --&gt; ca. 230ng); Phusion DNA polymerase 0,5µl; nclease-free water ad to 50µl </LI></UL><UL><LI> PCR-program: initial Denat. 40´´ at 98°C; Denat 12´´ at 98°C; Annealing 15´´ at 55°C; Elongation 11´´ at 72°C; final-Elongation 10´at 72°C; 30 cycles
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><UL><LI> assembly-pcr
</LI></UL><H3><SPAN class="mw-headline" id="2012-07-26_2"><P style="background-color: rgb(240, 20, 70);">2012-07-26</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Planning the antibody construct for genesynthesis</P><P><B>Investigators:</B> Maria </P><P><B>Aim:</B> construct with scFv 425bla, LoxP and TEV recognition site, E-YFP</P><P><B>Date/Time:</B> 26th July 2012, 2 - 5 pm </P><P><B>Materials:</B> Geneious </P><P><B>Results:</B> antibody construct RCF25 </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: extension-pcr of scFv-TEV-TMD-EYFP geneconstruct

</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B>scFv anti-EGFR, BBa-KI57000 with transmembrane domain; BBa_E0030 with EYFP</P><UL><LI> P1_Signalp_N-term</LI></UL><UL><LI> P2_Signalp_C-term</LI></UL><UL><LI> P3_TMD-N-term</LI></UL><UL><LI> P4_TMD-C-term/N-YFP</LI></UL><UL><LI> P5_YFP-N-term/TMD-C-term</LI></UL><UL><LI> P6_YFP-C-term</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 50µl PCR mix of each plasmid= 10µl HF-Buffer; 1µl dNTPs(10mM); forward and reverse primer each 2,5µl (10µM); template DNA: 1µl of 200ng/µl scFv--&gt; 200ng, 5µl of BBa_E0030 with EYFP (46,3ng/µl--&gt; ca. 230ng),5µl of BBa-KI57000 with transmembrane domain (43,9ng/µl --&gt; ca. 230ng); Phusion DNA polymerase 0,5µl; nclease-free water ad to 50µl </LI></UL><UL><LI> PCR-program: initial Denat. 40´´ at 98°C; Denat 12´´ at 98°C; Annealing 15´´ at 55°C; Elongation 11´´ at 72°C; final-Elongation 10´at 72°C; 30 cycles
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gelelectrophoresis of extended genes after extension-pcr</P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> separation of extended genes scFv-TEV, TEV-TMD, EYFP in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><UL><LI> extended genes
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel
</LI></UL><UL><LI> 70 min at 105V
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gelextraction of extended genes after extension-pcr

</P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> gelextraction of extended genes(scFv-TEV, TEV-TMD, EYFP) out of 1% agarosegel </P><P><B>Method:</B></P><UL><LI> DNA extracted according to the manual
</LI></UL><P><B>Results: concentration measuremnt using nanodrop</B></P><UL><LI> concentrations of extended scFv:
</LI></UL><UL><LI> concentrations of extended TMD:
</LI></UL><UL><LI> concentrations of extended EYFP:
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> assembly-pcr
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: assembly-pcr with extended genes to scFv-TEV-TMD-EYFP geneconstruct

</P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> assembly-pcr with extended genes to scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Materials:</B>scFv anti-EGFR, BBa-KI57000 with transmembrane domain; BBa_E0030 with EYFP</P><UL><LI> P7_Gesamtanfang (forward-primer)</LI></UL><UL><LI> P8_Gesamtende (reverse-primer)</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> first assembly-pcr-mix--&gt; 150:50: 10µl HF-Buffer; 1µl dNTPs(10mM); 5µl of extended TMD (30ng/µl --&gt; 150ng), 5µl of extended EYFP (10ng/µl --&gt; 50ng), 2µl of extended scFv (25ng/µl --&gt; 50ng); nuclease-free water ad to 45µl; 0,5µl Phusion-polymerase</LI></UL><UL><LI> second assembly-pcr-mix--&gt; 90:30: 10µl HF-Buffer; 1µl dNTPs(10mM); 2,5µl of extended TMD (30ng/µl --&gt; 90ng), 3µl of extended EYFP (10ng/µl --&gt; 30ng), 1µl of extended scFv (25ng/µl --&gt; 25ng); nuclease-free water ad to 45µl; 0,5µl Phusion-polymerase</LI></UL><UL><LI> PCR-program to let the extended genes prime each other: initial Denat. 40´´ at 98°C; Denat 12´´ at 98°C; Annealing 15´´ at 55°C; Elongation 27´´ at 72°C; final-Elongation 10´at 72°C; 15 cycles
</LI></UL><UL><LI> after annealing of extended genes to each other 2,5µl of primer forw_P7_Gesamtanfang and primer rev_P8_Gesamtende were added
</LI></UL><UL><LI> PCR-prgram with end-primers: initial Denat. 40´´ at 98°C; Denat 12´´ at 98°C; Annealing 15´´ at 57°C; Elongation 27´´ at 72°C; final-Elongation 10´at 72°C; 27 cycles
</LI></UL><UL><LI> Gelelectrophoresis at 105V for 75`
</LI></UL><P><B>Results:</B></P><UL><LI> extraction of wrong 1100bp DNA-bond
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> assembly-pcr
</LI></UL><UL><LI> extraction of correct DNA (1701bp)
</LI></UL><H3><SPAN class="mw-headline" id="2012-07-28_2"><P style="background-color: rgb(240, 20, 70);">2012-07-28</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: assembly-pcr with extended genes to scFv-TEV-TMD-EYFP geneconstruct

</P><P><B>Investigators:</B> Sascha, Tarek</P><P><B>Aim:</B> assembly-pcr with extended genes to scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Materials:</B>scFv anti-EGFR, BBa-KI57000 with transmembrane domain; BBa_E0030 with EYFP</P><UL><LI> P7_Gesamtanfang (forward-primer)</LI></UL><UL><LI> P8_Gesamtende (reverse-primer)</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> first assembly-pcr-mix--&gt; 90:30: 10µl HF-Buffer; 1µl dNTPs(10mM); 2,5µl of extended TMD (30ng/µl --&gt; 90ng), 3µl of extended EYFP (10ng/µl --&gt; 30ng), 1µl of extended scFv (25ng/µl --&gt; 25ng); nuclease-free water ad to 45µl; 0,5µl Phusion-polymerase</LI></UL><UL><LI> second assembly-pcr-mix--&gt; 140:20: 10µl HF-Buffer; 1µl dNTPs(10mM); 1µl of extended TMD (140ng/µl --&gt; 140ng), 12,5µl of extended EYFP (1,6ng/µl --&gt; 20ng), 4,8µl of extended scFv (4,2ng/µl --&gt; ca. 20ng); nuclease-free water ad to 45µl; 0,5µl Phusion-polymerase</LI></UL><UL><LI> PCR-program to let the extended genes prime each other: initial Denat. 40´´ at 98°C; Denat 12´´ at 98°C; Annealing 15´´ at 55°C; Elongation 27´´ at 72°C; final-Elongation 10´at 72°C; 15 cycles
</LI></UL><UL><LI> after annealing of extended genes to each other 2,5µl of primer forw_P7_Gesamtanfang and primer rev_P8_Gesamtende were added
</LI></UL><UL><LI> PCR-prgram with end-primers: initial Denat. 40´´ at 98°C; Denat 12´´ at 98°C; Annealing 15´´ at 57°C; Elongation 27´´ at 72°C; final-Elongation 10´at 72°C; 27 cycles
</LI></UL><UL><LI> Gelelectrophoresis at 105V for 85`
</LI></UL><P><B>Results:</B></P><UL><LI> extraction of correct DNA (1701bp)
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> digestion of pcDNA5-FRT and geneconstruct scFv-TEV-TMD-EYFP
</LI></UL><UL><LI> ligation of plasmid and insert
</LI></UL><H3><SPAN class="mw-headline" id="2012-07-31_2"><P style="background-color: rgb(240, 20, 70);">2012-07-31</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Planning and reviewing the antibody construct for genesynthesis</P><P><B>Investigators:</B> Maria </P><P><B>Aim:</B> construct with scFv 425bla, LoxP and TEV recognition site, mVenus</P><P><B>Date/Time:</B> 27th July 2012, 3 - 5 pm </P><P><B>Materials:</B> Geneious </P><P><B>Results:</B> antibody construct RCF25 </P><H3><SPAN class="mw-headline" id="2012-07-31_3"><P style="background-color: rgb(240, 20, 70);">2012-07-31</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Construct for genesynthesis with human scFv 425-72000</P><P><B>Investigators:</B> Maria </P><P><B>Aim:</B> construct with human scFv 425-72000, LoxP and TEV recognition site, mVenus</P><P><B>Date/Time:</B> 27th July 2012, 3 - 5 pm </P><P><B>Materials:</B> Geneious </P><P><B>Results:</B> antibody construct RCF25 </P><H2><SPAN class="mw-headline" id="Virus">Virus</SPAN></H2><H3><SPAN class="mw-headline" id="02.07.2012"><P style="background-color: rgb(240, 20, 70);"> 02.07.2012</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR</P><P><B>Investigators:</B> Xenia and Kathi </P><P><B>Aim:</B></P><UL><LI> to test the primer
</LI></UL><UL><LI> amplification of the cap and VP-region, insertion of kozak-, Sortasemotive-, mycTag- and restiction sites Sequence </LI></UL><P><B>Materials:</B></P><UL><LI> new primer combination (prr_VP2_pstI_Temp68 and prf_XbaI_kozak_So rtlN_myc_VP2 [c= 10 µM])</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagenz</B></TD><TD><B>volumen [µL]</B></TD></TR><TR><TD>HE buffer</TD><TD>5</TD></TR><TR><TD>dNTPs (NEB)</TD><TD>1.25</TD></TR><TR><TD>Primer (prr_VP2_pstI_Temp68)</TD><TD>1.0</TD></TR><TR><TD>Primer (prf_XbaI_kozak_So rtlN_myc_VP2)</TD><TD>1.0</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1.0 </TD></TR><TR><TD> Phusion polymerase </TD><TD> 1.0 </TD></TR><TR><TD> water </TD><TD> 33.75 </TD></TR></TBODY></TABLE><P><B>program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>temperature [°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD>120 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 95 </TD><TD> 30 </TD><TD> 30 </TD></TR><TR><TD> annealing </TD><TD> 68 </TD><TD> 60 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 60 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 60 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> ∞ </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> agarose gel electrophoresis
</LI></UL><H3><SPAN class="mw-headline" id="04.07.2012"><P style="background-color: rgb(240, 20, 70);"> 04.07.2012</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: gel electrophoresis </P><P><B>Investigator:</B> Laura </P><P><B>Aim:</B></P><UL><LI> measurement of DNA-concentration of pcr product and DARPin+cmv plasmid
</LI></UL><UL><LI> test pcr product and DRAPin+cmv plasmid</LI></UL><P><B>Materials:</B></P><UL><LI> pcr product (02.07.) and DARPin+cmv Plasmid (Sven)
</LI></UL><P><B>Method:</B></P><UL><LI> agarose gel electrophoresis</LI></UL><UL><LI> nanodrop
</LI></UL><P><B>Results:</B></P><UL><LI> DNA-concentrations:
</LI></UL><UL><LI> pcr-prduct: 266.1 ng/µL
</LI></UL><UL><LI> plasmid: 441.4 ng/µL
</LI></UL><UL><LI> gel electrophoresis
</LI></UL><UL><LI> pcr product present but concentration too high --&gt; next time better: serial dilution of pcr product!</LI></UL><P>IMPORTANT!: Our primer is wrong! we have put the C-terminal sortase-motiv into our primer, but we just need the N-terminal sortase-tag (glycin-rests)
</P><P><B>further tasks:</B></P><P>design new primer
</P><H3><SPAN class="mw-headline" id="04.07.2012_2"><P style="background-color: rgb(240, 20, 70);"> 04.07.2012</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Primer design - Cloning: Change the C-terminal Sortase-Tag against the N-terminal </P><P><B>Investigators:</B>Tobias and Xenia </P><P><B>Aim:</B></P><UL><LI> Change the C-terminal sortase-tag against the N-terminal </LI></UL><P><B>Materials:</B></P><UL><LI> prf_XbaI_kozak_SortaseMotivN_myc_VP2
</LI></UL><P><B>Method:</B> Geneious</P><P><B>Results:</B></P><UL><LI> prf_Xba1 koz_GGGGG_VP2 (forward primer)
</LI></UL><UL><LI> prf_XbaI_koz_GGGGG_myc_VP2 (forward primer)
</LI></UL><P>the sortase-tag was changed in both primers and prf_Xba1 koz_GGGGG_VP2 contain no myc-tag</P><P><B>Further tasks:</B></P><P>PCR
</P><H3><SPAN class="mw-headline" id="11.07.2012"><P style="background-color: rgb(240, 20, 70);"> 11.07.2012</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR - Cloning: PCR using the new forward primers</P><P><B>Investigators:</B> Kathi and Laura </P><P><B>Aim:</B></P><UL><LI> PCR using the both new forward primer (prf_Xba1 koz_GGGGG_VP2 and prf_XbaI_koz_GGGGG_myc_VP2)
</LI></UL><UL><LI> gel electrophoresis </LI></UL><P><B>Materials:</B></P><UL><LI> prf_Xba1 koz_GGGGG_VP2 (FP 1)--&gt; myc-
</LI></UL><UL><LI> prf_XbaI_koz_GGGGG_myc_VP2(FP2)--&gt; myc+
</LI></UL><UL><LI> prr_VP2_pstI_Temp68 (RP) </LI></UL><P><B>Method:</B></P><UL><LI> PCR (PCR protocol: 02.07.2012)
</LI></UL><UL><LI> gel electropohoresis: : 2.1 and 0.5 µL pcr product of primer pair myc+ and myc- loaded on gel</LI></UL><P><B>Results:</B></P><UL><LI> gel electrophoresis: pcr product at 2000 bp present, primer dimer obvious</LI></UL><P><B>Further tasks:</B></P><UL><LI> ligation
</LI></UL><H3><SPAN class="mw-headline" id="12.07.2012"><P style="background-color: rgb(240, 20, 70);"> 12.07.2012</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR </P><P><B>Investigators:</B> Xenia and Laura </P><P><B>Aim:</B></P><UL><LI> test the primer using different annealing temperatures to get the right sequence of 2000 bp</LI></UL><P><B>Materials:</B></P><P>primer combination:
</P><UL><LI> prr_VP2_pstI_Temp68 and prf_XbaI_koz_GGGGG_VP2 (FP1)-&gt; myc-
</LI></UL><UL><LI> prr_VP2_pstI_Temp68 (RP) and prf_XbaI_koz_GGGGG_myc_VP2(FP2)--&gt; myc+
</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagenz</B></TD><TD><B>volumen [µL]</B></TD></TR><TR><TD>HF buffer</TD><TD>5</TD></TR><TR><TD>dNTPs (NEB)</TD><TD>1.25</TD></TR><TR><TD>Primer (prr_VP2_pstI_Temp68)</TD><TD>1.0</TD></TR><TR><TD>Primer (prf_XbaI_kozak_So rtlN_myc_VP2)</TD><TD>1.0</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1.0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 1.0 </TD></TR><TR><TD> water </TD><TD> 33.75 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="2"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 10 </TD><TD> 30 </TD></TR><TR><TD> annealing </TD><TD> 64; 68; 70; 72 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> ∞ </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Results:</B></P><P><B>Further tasks:</B></P><UL><LI> agarose gel electrophoresis
</LI></UL><H3><SPAN class="mw-headline" id="13.07.2012"><P style="background-color: rgb(240, 20, 70);">13.07.2012</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gel electrophoresis of the pcr product </P><P><B>Investigators:</B>Xenia and Mario </P><P><B>Aim:</B>gel electrophoresis of the pcr product (12.07.2012)</P><P><B>material and method:</B></P><UL><LI> gel electrophoresis
</LI></UL><UL><LI>samples:
</LI></UL><UL><LI> 4 µL dest. Wasser + 1 µL pcr sample + 1 µL loading dye</LI></UL><P><B>Results:</B></P><P>The fragment should have the size of 2000 bp, but the band is by 3000 bp</P><P><B>Further Tasks:</B></P><UL><LI> test digestion
</LI></UL><H3><SPAN class="mw-headline" id="2012-17-07"><P style="background-color: rgb(240, 20, 70);">2012-17-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: test digestion</P><P><B>Investigators:</B> Tobias </P><P><B>Materials:</B></P><UL><LI> pcr products of 02-07 and 12-07-2012 (Ta= 70°C)
</LI></UL><UL><LI> restriction enzymes (FastDigest XbaI, PstI)</LI></UL><P><B>Method:</B></P><P>digestion: 10 µL DNA + 1 µL of each enzyme + 2 µL green-buffer + 17 µL water </P><P><B>Results:</B></P><UL><LI> caused on the gel qualtity: repetition of the digestion
</LI></UL><P><B> Further Tasks: </B></P><UL><LI> Digestion
</LI></UL><H3><SPAN class="mw-headline" id="2012-18-07"><P style="background-color: rgb(240, 20, 70);">2012-18-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preperative digestion an gel extraction of pcr fragments</P><P><B>Investigators:</B> Tobias and Laura </P><P><B>Materials:</B></P><UL><LI> pcr-products of 02-07-2012 and 12-07-2012 (Ta= 64 °C and 68 °C)
</LI></UL><UL><LI> restriction enzymes (FastDigest (FD) XbaI, PstI)</LI></UL><P><B>Method:</B></P><P>digestion: 10 µL DNA + 1 µL of each enzyme + 2 µL FD green-buffer + 17 µL water</P><P><B>Results:</B></P><H3><SPAN class="mw-headline" id="2012-27-07"><P style="background-color: rgb(240, 20, 70);">2012-27-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: digestion of pcr fragments and plasmid</P><P><B>Investigators:</B> Tobias </P><P><B>Materials:</B></P><UL><LI> pcr-products of 12-07-2012
</LI></UL><UL><LI> plasmid (P10_pSB1C3_001_CMV_DARPin_ML_VP2/3_587KO_6xHis)
</LI></UL><UL><LI> restriction enzymes (FastDigest (FD) XbaI, PstI, SpeI)</LI></UL><P><B>Method:</B></P><P>digestion: 10 µL DNA + 1 µL of each enzyme + 2 µL FD green-buffer + 17 µL water</P><P><B>Results:</B></P><H3><SPAN class="mw-headline" id="2012-01-08"><P style="background-color: rgb(240, 20, 70);">2012-01-08</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Sequencing of pcr-pruducts</P><P><B>Investigator:</B> Tobias </P><P><B>Materials:</B></P><UL><LI> PCR-products of 12-07-2012 (+myc and-myc)
</LI></UL><UL><LI> Primer: prr_VP2_pstI_Temp68 </LI></UL><P><B>Method:</B></P>
GATC</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal/July">http://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal/July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Lab/Labjournal/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Lab/Labjournal/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/July&amp;oldid=179081" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>