<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Lab_Labjournal_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Lab/Labjournal/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Lab/Labjournal/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/August&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Lab/Labjournal/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Lab/Labjournal/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="nav_menu gradient_golden"><DIV id="menu_center"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware">Home</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">Project</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance#Main_Results">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">Mutation Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody">Antibody Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus">Selection Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam Standard">Potsdam Standard</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Collaboration">Collaboration</A></LI></UL><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Sponsoring">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Attribution">Attribution</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Lab</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Group_Meetings">Group Meetings</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">All Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">SocialBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_Politics">Science Meets Politics</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_People">Science Meets People</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Software">Software</A></LI></DIV></DIV><DIV class="box_round white_bg"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#AID"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">AID</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#2012-08-01"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">2012-08-01</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#2012-08-02"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">2012-08-02</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#2012-08-03"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">2012-08-03</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#2012-08-04"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">2012-08-04</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#2012-08-06"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">2012-08-06</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#2012-08-07"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">2012-08-07</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#2012-08-08"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">2012-08-08</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#2012-08-09"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">2012-08-09</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#2012-08-10"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">2012-08-10</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#2012-08-11"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">2012-08-11</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#2012-08-13"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">2012-08-13</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#2012-08-14"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">2012-08-14</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#2012-08-15"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">2012-08-15</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#2012-08-16"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">2012-08-16</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#2012-08-17"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">2012-08-17</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#2012-08-20"><SPAN class="tocnumber">1.16</SPAN><SPAN class="toctext">2012-08-20</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#2012-08-21"><SPAN class="tocnumber">1.17</SPAN><SPAN class="toctext">2012-08-21</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#2012-08-22"><SPAN class="tocnumber">1.18</SPAN><SPAN class="toctext">2012-08-22</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#2012-08-23"><SPAN class="tocnumber">1.19</SPAN><SPAN class="toctext">2012-08-23</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#2012-08-24"><SPAN class="tocnumber">1.20</SPAN><SPAN class="toctext">2012-08-24</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#2012-08-27"><SPAN class="tocnumber">1.21</SPAN><SPAN class="toctext">2012-08-27</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#2012-08-28"><SPAN class="tocnumber">1.22</SPAN><SPAN class="toctext">2012-08-28</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#2012-08-29"><SPAN class="tocnumber">1.23</SPAN><SPAN class="toctext">2012-08-29</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#2012-08-30"><SPAN class="tocnumber">1.24</SPAN><SPAN class="toctext">2012-08-30</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#2012-08-31"><SPAN class="tocnumber">1.25</SPAN><SPAN class="toctext">2012-08-31</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-27"><A href="#Antibody"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Antibody</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#2012-08-02_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">2012-08-02</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#2012-08-03_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">2012-08-03</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#2012-08-05"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">2012-08-05</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#2012-08-06_2"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">2012-08-06</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#2012-08-07_2"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">2012-08-07</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#2012-08-08_2"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">2012-08-08</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#2012-08-10_2"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">2012-08-10</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#2012-08-13_2"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">2012-08-13</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#2012-08-14_2"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">2012-08-14</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#2012-08-15_2"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">2012-08-15</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#2012-08-16_2"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">2012-08-16</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#2012-08-17_2"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">2012-08-17</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#2012-08-18"><SPAN class="tocnumber">2.13</SPAN><SPAN class="toctext">2012-08-18</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#2012-08-19"><SPAN class="tocnumber">2.14</SPAN><SPAN class="toctext">2012-08-19</SPAN></A></LI><LI class="toclevel-2 tocsection-42"><A href="#2012-08-20_2"><SPAN class="tocnumber">2.15</SPAN><SPAN class="toctext">2012-08-20</SPAN></A></LI><LI class="toclevel-2 tocsection-43"><A href="#2012-08-21_2"><SPAN class="tocnumber">2.16</SPAN><SPAN class="toctext">2012-08-21</SPAN></A></LI><LI class="toclevel-2 tocsection-44"><A href="#2012-08-25"><SPAN class="tocnumber">2.17</SPAN><SPAN class="toctext">2012-08-25</SPAN></A></LI><LI class="toclevel-2 tocsection-45"><A href="#2012-08-27_2"><SPAN class="tocnumber">2.18</SPAN><SPAN class="toctext">2012-08-27</SPAN></A></LI><LI class="toclevel-2 tocsection-46"><A href="#2012-08-28_2"><SPAN class="tocnumber">2.19</SPAN><SPAN class="toctext">2012-08-28</SPAN></A></LI><LI class="toclevel-2 tocsection-47"><A href="#2012-08-27_3"><SPAN class="tocnumber">2.20</SPAN><SPAN class="toctext">2012-08-27</SPAN></A></LI><LI class="toclevel-2 tocsection-48"><A href="#2012-08-29_2"><SPAN class="tocnumber">2.21</SPAN><SPAN class="toctext">2012-08-29</SPAN></A></LI><LI class="toclevel-2 tocsection-49"><A href="#2012-08-31_2"><SPAN class="tocnumber">2.22</SPAN><SPAN class="toctext">2012-08-31</SPAN></A></LI><LI class="toclevel-2 tocsection-50"><A href="#2012-08-30_2"><SPAN class="tocnumber">2.23</SPAN><SPAN class="toctext">2012-08-30</SPAN></A></LI><LI class="toclevel-2 tocsection-51"><A href="#2012-08-29_3"><SPAN class="tocnumber">2.24</SPAN><SPAN class="toctext">2012-08-29</SPAN></A></LI><LI class="toclevel-2 tocsection-52"><A href="#2012-08-30_3"><SPAN class="tocnumber">2.25</SPAN><SPAN class="toctext">2012-08-30</SPAN></A></LI><LI class="toclevel-2 tocsection-53"><A href="#2012-08-31_3"><SPAN class="tocnumber">2.26</SPAN><SPAN class="toctext">2012-08-31</SPAN></A></LI><LI class="toclevel-1 tocsection-54"><A href="#Virus"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Virus</SPAN></A></LI><LI class="toclevel-2 tocsection-55"><A href="#2012-08-13_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">2012-08-13</SPAN></A></LI><LI class="toclevel-2 tocsection-56"><A href="#2012-08-14_3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">2012-08-14</SPAN></A></LI><LI class="toclevel-2 tocsection-57"><A href="#2012-08-15_3"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">2012-08-15</SPAN></A></LI><LI class="toclevel-2 tocsection-58"><A href="#2012-08-17_3"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">2012-08-17</SPAN></A></LI><LI class="toclevel-2 tocsection-59"><A href="#2012-08-28_3"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">2012-08-28</SPAN></A></LI><LI class="toclevel-2 tocsection-60"><A href="#2012-08-29_4"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">2012-08-29</SPAN></A></LI><LI class="toclevel-2 tocsection-61"><A href="#2012-08-30_4"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">2012-08-30</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="AID">AID</SPAN></H2><H3><SPAN class="mw-headline" id="2012-08-01"><P style="background-color: rgb(240, 20, 70);">2012-08-01</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">glycerol stocks &amp; miniprep of eGFP and GFP</P><P><B>Investigators:</B> Tom S. </P><P><B>Time:</B></P><P>2012-07-03 7:30 am
</P><P><B>Materials:</B></P><P>Glycerol, Miniprep Kit, overnight cultures (eGFP; GFP)
</P><P><B>Method:</B></P><P>Glycerol stock: 300 ÂµL Glycerol 99,8Â % + 700 ÂµL over night cultures --&gt; put in -80 Â°C freezer
</P><P>Miniprep of overnight cultures (eGFP in pSB1C3 (BBa_K404316); 2 clones of GFP in pSB1A2 (BBa_E0040))
</P><P><B>Results:</B></P><P>DNA concentrations via nanodrop:</P><P>eGFP = 335,6 ng/ÂµL </P><P>GFP 1 = 291,5 ng/ÂµL </P><P>GFP 2 = 261,7 ng/ÂµL </P><P><B>Further tasks:</B></P><P>preparative digestion with eGFP and CMV+mod. AID (without NES, with NLS)
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Ligation CMV &amp; modified AID (without NES, with NLS) in pSB1C3 with hGH-PolyA</P><P><B>Investigators:</B></P><P>Chris </P><P><B>Aim:</B></P><P>Ligation of CMV &amp; modified AID in pSB1C3(backbone) and hGH-polyA (insert)
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, T4-Ligase-buffer, samples:</P><P>CMV+PCR1C3C2(3318 nt, MW=2048.48 kDa)Â : 188.1 ng/Âµl (91.8 nM)</P><P>CMV+PCR2C2C1Â : 182.2 ng/Âµl (88.9 nM)</P><P>hGH-polyA (495 nt, MW=305.4 kDa): 26.1 ng/Âµl (85 nM)</P><P><B>Method:</B></P><P>DNA fragment ligation: according to the manual</P><P>sample preparation 1:
</P><UL><LI> 2 ÂµL CMV+PCR1C3C2 c=188.1 ng/Âµl (91.8 nM) -&gt;</LI></UL><UL><LI> 6 ÂµL hGH-polyA c=26.1 ng/Âµl (85 nM) -&gt;</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase
</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase buffer
</LI></UL><P>sample preparation 2:
</P><UL><LI> 2 ÂµL CMV+PCR2C2C1 c=182.2 ng/Âµl (88.9 nM) -&gt;</LI></UL><UL><LI> 6 ÂµL hGH-polyA c=26.1 ng/Âµl (85 nM) -&gt;</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase
</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase buffer
</LI></UL><P>incubation of samples for 1,5 h at room temperature</P><P><B>Results:</B></P><P>ligated plasmids(final construct mod.-AID (CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA, without eGFP) for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: transformation of ligated samples (CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA, without eGFP)</P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-08-01 3 pm</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of ligation samples were used
</P><P>Plate incubation start: 13:30 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B> Chris </P><P><B>Time:</B> 2012-08-01</P><P><B>Materials:</B></P><UL><LI> Plasmids: pSB1C3 with CMV+PCR1C3C2 and CMV+PCR2C2C1 and eGFP</LI></UL><UL><LI> Restriction enzymes (SpeI, NgoMIV and AgeI)</LI></UL><UL><LI> NE4-buffer</LI></UL><P><B>Method:</B></P><P>sample preparation: each DNA 25 ÂµL + 3 ÂµL NE4-buffer + 1 ÂµL SpeI + 1 ÂµL AgeI (for CMV+AID+Kozak sequence) or 1ÂµL NgoMIV (for eGFP)</P><P>incubation of samples for 3,5 h at 37 Â°C</P><P><B>Further tasks:</B></P><P>gel electrophoresis</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Separation of cut DNA fragments via gel electrophoresis</P><P><B>Investigators:</B> Tom S. </P><P><B>Time:</B> 2012-08-01</P><P><B>Aim:</B> Separation of cut DNA fragments via gel electrophoresis</P><P><B>Materials:</B></P><P>gel electrophoresis material</P><P>cut samples:
</P><UL><LI> CMV+PCR1C3C2: Restriction enzymes (AgeI, SpeI); NEB buffer 4
</LI></UL><UL><LI> CMV+PCR2C2C1: Restriction enzymes (AgeI, SpeI); NEB buffer 4
</LI></UL><UL><LI> eGFP: Restriction enzymes (NgoMIV, SpeI); NEB buffer 4
</LI></UL><P><B>Method:</B></P><P>samples:</P><P>- 30 ÂµL AID cut with XbaI and PstI + 7,5 ÂµL loading dye</P><P>gelelectrophoresis conditions:</P><P>30 ÂµL of each samples into one big well</P><P>V = 90 V</P><P>duration roughly 75 minutes</P><P><B>Results:</B></P><P>Marked fragments were cut out of the gel and transferred into 1,5 mL Eppendorf tubes</P><P><B>Further Tasks:</B></P><P>Gel extraction
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Gel extraction of CMV + mod. AID + backbone and eGFP-insert
</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up - Kit, heat block, nanodrop
</P><P><B>Method:</B></P><P>DNA extraction: according to the manual
</P><P><B>Results:</B></P><P>DNA concentrations via nanodrop:</P><P>CMV+PCR1C3C2+backbone = 158,1 ng/ÂµL -&gt; 77,1 nM (with mass conc. of 2050,94 kDa)</P><P>CMV+PCR2C2C1+backbone = 232,0 ng/ÂµL -&gt; 113,1 nM (with mass conc. of 2050,94 kDa)</P><P>eGFP = 55,8 ng/ÂµL -&gt; 123,4 nM (with mass conc. of 452,3 kDa)</P><P>location: -20 Â°C freezer, topmost drawer
</P><P>ready DNA for ligation
</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><H3><SPAN class="mw-headline" id="2012-08-02"><P style="background-color: rgb(240, 20, 70);">2012-08-02</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of CMV+AID without NES, with NLS+Kozak sequence+backbone and eGFP</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Ligation of CMV+AID without NES, with NLS+Kozak sequence+backbone and eGFP
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, samples(CMV+PCR1C3C2+backbone, CMV+PCR2C2C1+backbone,eGFP)
</P><P><B>Method:</B></P><P>DNA fragment ligation: according to the manual</P><P>1st sample preparation:
</P><UL><LI> 4 ÂµL (eGFP) c=55,8 ng/ÂµL(123,4 nM)-&gt; 49,4 nM
</LI></UL><UL><LI> 2 ÂµL (CMV+PCR1C3C2+backbone) c=158,1 ng/ÂµL(77,1 nM) -&gt; 15,4 nM
</LI></UL><UL><LI> 1 ÂµL (T4 DNA-Ligase)
</LI></UL><UL><LI> 1 ÂµL 10x T4 DNA Ligase Buffer
</LI></UL><UL><LI> 2 ÂµL (DNase free water)
</LI></UL><P>2nd sample preparation:
</P><UL><LI> 4 ÂµL (eGFP) c=55,8 ng/ÂµL(123,4 nM)-&gt; 49,4 nM
</LI></UL><UL><LI> 2 ÂµL (CMV+PCR2C2C1+backbone) c=232,0 ng/ÂµL(113,1 nM) -&gt; 22,6 nM
</LI></UL><UL><LI> 1 ÂµL (T4 DNA-Ligase)
</LI></UL><UL><LI> 1 ÂµL 10x T4 DNA Ligase Buffer
</LI></UL><UL><LI> 2 ÂµL (DNase free water)
</LI></UL><P>incubation of sample for 1,5 h at 22Â°C</P><P><B>Results:</B></P><P>ready DNA for transformation</P><P>location: -20 Â°C freezer, topmost drawer
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated samples - CMV+AID without NES, with NLS+Kozak sequence+eGFP</P><P><B>Investigators:</B> Tom S. , Chris</P><P><B>Time:</B> 2012-08-02 3 pm</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><UL><LI> ligated samples (CMV+PCR1C3C2+eGFP in pSB1C3, same consruct with PCR2C2C1)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of ligation samples were used
</P><P>Plate incubation start: 14:00 pm
</P><P><B>Results:</B></P><P>ready for growing mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: picking clones &amp; inoculation in 5 mL LB of CMV+AID without NES, with NLS+hGH-PolyA in pSB1C3 </P><P><B>Investigators:</B> Chris</P><P><B>Method:</B> picking clones (1 per plate --&gt;4) CMV + modified AID(PCR1C3C2&amp;PCR2C2C1) +hGH(PolyA)in pSB1C3 and inoculation in 5 ml LB medium + 5Âµl chloramphenicol stock, shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>glycerol stocks &amp; Miniprep
</P><H3><SPAN class="mw-headline" id="2012-08-03"><P style="background-color: rgb(240, 20, 70);">2012-08-03</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">glycerol stocks &amp; miniprep of CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA, without eGFP plasmids</P><P><B>Investigators:</B> Chris </P><P>Time:
</P><P>2012-08-03 8:00 am
</P><P>Materials:
</P><P>Glycerol, Miniprep Kit, (CMV+PCR1C3C2/PCR2C2C1+polyA)
</P><P>Method:
</P><P>Glycerol stock: 300 ÂµL Glycerol 99,8Â % + 700 ÂµL over night cultures --&gt; put in -80 Â°C freezer
</P><P>Mini Prep over night cultures (CMV+PCR1C3C2-Nr1+polyA in pSB1C3,same construct with PCR1C3C2-Nr2, PCR2C2C1-Nr1,PCR2C2C1-Nr2)
</P><P>Results:
</P><P>transfection ready biobricks: CMW+ modified AID+hGH-polyA
</P><P>DNA - concentrations via nanodrop:</P><P>CMV+PCR1C3C2-Nr1+polyA in pSB1C3= 628 ng/ÂµL </P><P>CMV+PCR1C3C2-Nr2+polyA in pSB1C3= 577.5 ng/ÂµL </P><P>CMV+PCR2C2C1-Nr1+polyA in pSB1C3= 590.8 ng/ÂµL </P><P>CMV+PCR2C2C1-Nr2+polyA in pSB1C3= 571.4 ng/ÂµL </P><P><B>Further tasks:</B></P><P>sequencing
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Overnight culture of CMV+AID without NES, with NLS+Kozak sequence+eGFP</P><P><B>Investigators:</B> Chris, Tom S.</P><P><B>Time:</B> 2012-08-03 5 pm</P><P><B>Materials:</B></P><P>LB medium, chloramphenicol 25 mg/ ml stock solution, plates with <I>E. coli</I> XL1 blue with plasmids: CMV+PCR1C3C2C1-2+eGFP and CMV+PCR2C2C1C1-2+eGFP</P><P><B>Method:</B> picking clones(1 per plate) and inoculation in 5 ml LB medium + 5Âµl chloramphenicol stock shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>glycerol stocks &amp; Miniprep
</P><H3><SPAN class="mw-headline" id="2012-08-04"><P style="background-color: rgb(240, 20, 70);">2012-08-04</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">glycerol stocks &amp; miniprep</P><P><B>Investigators:</B> Basia </P><P>Time:
</P><P>2012-08-04 10:00 am
</P><P>Materials:
</P><P>Glycerol, Miniprep Kit, overnight culture
</P><P>Method:
</P><P>Glycerol stock: 300 ÂµL Glycerol 99,8Â % + 700 ÂµL overnight cultures --&gt; put in -80 Â°C freezer
</P><P>Miniprep overnight cultures (CMV+PCR1C3C2-Nr1+eGFP in pSB1C3,same construct with PCR1C3C2-Nr2, PCR2C2C1-Nr1,PCR2C2C1-Nr2)
</P><P>Results:
</P><P>transfection ready biobricks: CMV+ AID without NES, with NLS+Kozak sequence+ eGFP
</P><P>DNA - concentrations via nanodrop:
</P><P>CMV+ AID+eGFP 1 in pSB1C3= 386,8 ng/ÂµL </P><P>CMV+ AID+eGFP 2 in pSB1C3= 404,9 ng/ÂµL </P><P>CMV+ AID+eGFP 3 in pSB1C3= 442,6 ng/ÂµL </P><P>CMV+ AID+eGFP 4 in pSB1C3= 377,4 ng/ÂµL </P><P><B>Further tasks:</B></P><P>digestion
</P><H3><SPAN class="mw-headline" id="2012-08-06"><P style="background-color: rgb(240, 20, 70);">2012-08-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: preparative digestion</P><P><B>Investigators:</B>Tom S.</P><P><B>Time:</B> 2012-08-06 </P><P><B>Materials:</B></P><UL><LI> pSB1C3 Vector with CMV+AID without NES, with NLS+Kozak sequence+eGFP (SpeI, PstI; Fast Digest); Fast Digest Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI>preparative digestion: 25 ÂµL DNA + 1 ÂµL of each enzyme + 3 ÂµL Fast Digest Green Buffer (incubation for 2,5 h)
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Gel electrophoresis &amp; Gel extraction</P><P><B>Investigators:</B> Rico, Tom S., Chris </P><P>Gel extraction: final concentrations: CMV+PCR???1+eGFP-cutS+P(4035 nt, MW=2492 kDa)Â : 246 ng/Âµl (98.7 nM) </P><P>CMV+PCR???2+eGFP-cutSCMV-cutS+P: 255 ng/Âµl (102.3 nM) </P><P>CMV+PCR???3+eGFP-cutSCMV-cutS+P: 187.6 ng/Âµl (75.3 nM) </P><P>CMV+PCR???4+eGFP-cutSCMV-cutS+P: 178.6 ng/Âµl (71.7 nM)</P><P><B>Further Tasks</B>: Ligation, Transformation
</P><H3><SPAN class="mw-headline" id="2012-08-07"><P style="background-color: rgb(240, 20, 70);">2012-08-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Ligation</P><P><B>Investigators:</B> Chris </P><P><B>Aim:</B></P><P>Ligation of CMV + AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3(backbone) and hGH-polyA (insert)
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, T4-Ligase-buffer, samples:</P><P>CMV+PCR???1+eGFP-cut S+P(4035 nt, MW=2492 kDa)Â : 246 ng/Âµl (98.7 nM</P><P>CMV+PCR???2+eGFP-cut S+P (4035 nt, MW=2492 kDa)Â : 182.2 ng/Âµl (88.9 nM)</P><P>CMV+PCR???3+eGFP-cut S+P: 187.6 ng/Âµl (75.3 nM) </P><P>CMV+PCR???4+eGFP-cut S+P: 178.6 ng/Âµl (71.7 nM)</P><P>hGH-polyA (495 nt, MW=305.4 kDa): 26.1 ng/Âµl (85 nM)</P><P>hGH-polyA (cut:Xba1 and Pst1) c=48.5 nM
</P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation 1-4:
</P><UL><LI> 1 ÂµL CMV+PCR1C3C2 c=188.1 ng/Âµl (91.8 nM) -&gt;</LI></UL><UL><LI> 3 ÂµL hGH(polyA) c=26.1 ng/Âµl (85 nM) -&gt;</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase
</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase buffer
</LI></UL><UL><LI> 4 ÂµL H<SUB>2</SUB>0
</LI></UL><P>incubation of samples for 1,5 h at room temperature</P><P><B>Results:</B></P><P>ligated plasmids(CMV + AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA) for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">alignment of the complete WT AID and Sequencing</P><P><B>Investigators:</B> Rico, Tom S.</P><P><B>Aim:</B></P><P>alignment of the complete WT AID and Sequencing (checking the quality of the clones)
</P><P><B>Materials:</B></P><P>6 forward sequences of WT AID from GATC and 6 reverse sequences of the same fragment,
</P><P>draft sequences, Geneious
</P><P><B>Results:</B></P><UL><LI>1., 3., 5. and 6. clone: mutation of the 3rd nucleotide in front of the AID startcodon (is good);
</LI></UL><UL><LI>2. and 4. clone: frameshift (loss of the 155th nucleotide (2nd clone) or loss of the 239th nucleotide (4th clone))
</LI></UL><P><B>Further tasks:</B></P><P>discard the mutated clones and preparation for transfection of the final BioBrick BBa_K929000
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">alignment of the mod. AID PCR-product and Sequencing</P><P><B>Investigators:</B> Rico, Tom S.</P><P><B>Aim:</B></P><P>alignment of the mod. AID PCR-product and Sequencing (checking the quality of the clones)
</P><P><B>Material:</B></P><P>6 forward sequences of mod. AID PCR-product from GATC and 6 reverse sequences of the same fragment,draft sequences, Geneious
</P><P><B>Results:</B></P><UL><LI>all vectors have no PCR-insert in the backbone (instead CMV)
</LI></UL><P><B>Further tasks:</B></P><P>discard all samples and clones, made a eGFP and PCR-insert fusion with XbaI, NgeMIV and AgeI digestion, design new primer with PstI recognition site
</P><H3><SPAN class="mw-headline" id="2012-08-08"><P style="background-color: rgb(240, 20, 70);">2012-08-08</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Planing cooperation -&gt; Fusion of AID without NES with TAL-Protein, primer design </P><P><B>Investigators:</B> Chris, Rico, </P><P>AID will be fused to the TAL-protein (N--TAL-AID--C) with BpiI.We will get the fusion protein in an eukaryotic expression-vektor (with CMV promoter or hopefully a weaker one, depending on Team-Freiburg). </P><P>The 14 bp sequence for the TAL-protein has to have the following properties: (done by Antibody team) </P><P><B> 5' positon 0- must be T,  positon 1- no T , Position 2- no A,  Position 13- no G, position 14- must be G </B></P><P>There are two of these sequences in the VH-region of the sc-FV-anti EGFR 425 but not in the direct neighborhood of CDR3.
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Planing BBa_K929001 and BBa_K929003</P><P><B>Investigators:</B> Tom S., Chris, Basia, Rico, Mario, Kevin </P><P><B>Aim:</B> planing how to digest and ligate the vectors for BBa_K929001 and BBa_K929003 </P><P><B>Material:</B> Geneious </P><P><B>Results 1:</B></P><UL><LI> pSB1C3 with eGFP -&gt; cut with NgoMIV and XbaI, 2793 bp - pSB1C3 backbone+eGFP, 12 bp - rest
</LI></UL><UL><LI> PCR-amplificate (without PstI restriction site) -&gt; cut with AgeI and XbaI, 589 bp - modified AID insert (AID without NES, with NLS+Kozak sequence)
</LI></UL><P><B>Results 2:</B></P><UL><LI> pSB1C3 with CMV -&gt; cut with PstI and XbaI, 2061 bp - pSB1C3 backbone, 680 bp - rest
</LI></UL><UL><LI> PCR-amplificate (with PstI restriction site) -&gt; cut with PstI and XbaI, 613 bp - modified AID insert (AID without NES, with NLS+Kozak sequence)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> design and ordering of primers, practical part
</LI></UL><P><A href="/File:UP12_BBa_K929001_08.082.JPG" title="File:UP12 BBa K929001 08.082.JPG">500px</A></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Primer design and ordering for BBa_K929001</P><P><B>Investigators:</B> Tom S., Rico</P><P><B>Time:</B> 2012-08-08 </P><P>Primer (reverse, complement)with AgeI, SpeI and PstI recognition site:
</P><P>GCCTGCAGCGGCCGCTACTAGTATTAACCGGTGGGCAAAAGGATGCGCCGAAGC
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Primer design and ordering for sequencing BBa_K929003</P><P><B>Investigators:</B> Tom S., Rico</P><P><B>Time:</B> 2012-08-08 </P><P>Primer bind on mod. AID Sequence:
</P><P>TTTCAAAGCCTGGGAAGG
</P><P>Primer (reverse, complement)bind on eGFP sequence:
</P><P>GTGCCCATTAACATCACC
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> PCR of AID without NES, with NLS+ Kozak sequence</P><P><B>Investigators:</B></P><P>Rico </P><P><B>Aim:</B></P><UL><LI> amplification of the AID and insertion of Kozak sequence and NLS sequence
</LI></UL><P><B>Materials:</B></P><UL><LI> Phusion, template (AID insert), Primers designed by Tom S. and Rico on 12.07.2012 , dNTPs, Polymerase)
</LI></UL><UL><LI> PCR clean-up kit
</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing + elongation </TD><TD> 72 </TD><TD> 45 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 25 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Results:</B> 112 ng/Âµl - 1st sample, 113,8ng/Âµl 2nd sample, 22,1 negative control
</P><P><B>Further tasks:</B></P><UL><LI> digestion + agarose gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Digestion of amplified AID without NES, with NLS+Kozak sequence and GFP and separation with gel electrophoresis </P><P><B>Investigators:</B>Rico </P><P><B>Aim:</B> digestion of mod.-AID with XbaI and AgeI and GFP with NgoMIV and XbaI and separation via gel electrophoresis</P><P><B>Results:</B></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction of eGFP + pSB1C3 backbone and AID without NES, with NLS+Kozak sequence insert</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Gel extraction of eGFP + pSB1C3 backbone and AID without NES, with NLS+Kozak sequence insert
</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up - Kit, heat block, nanodrop
</P><P><B>Method:</B></P><P>extract DNA: according to the manual
</P><P><B>Results:</B></P><P>DNA-concentrations via nanodrop:</P><P>eGFP + pSB1C3 backbone = 59,1 ng/ÂµL -&gt; 34 nM</P><P>AID without NES, with NLS+Kozak sequence 1 = 22,8 ng/ÂµL -&gt; 62 nM</P><P>AID without NES, with NLS+Kozak sequence 2 = 25,2 ng/ÂµL -&gt; 69.3 nM</P><P>location: -20 Â°C freezer, topmost drawer
</P><P>ready DNA for ligation
</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><H3><SPAN class="mw-headline" id="2012-08-09"><P style="background-color: rgb(240, 20, 70);">2012-08-09</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of insert: PCR-Product (AID without NES, with NLS+Kozak sequence cut: XbaI, AgeI) &amp; backbone: eGFP in pSB1C3 (cut: NgoMIV, Xba1) </P><P><B>Investigators:</B></P><P>Chris </P><P><B>Aim:</B></P><P>Ligation of insert: PCR-Product (AID without NES, with NLS+Kozak sequence cut: XbaI, AgeI) &amp; backbone: eGFP in pSB1C3 (cut: NgoMIV, Xba1)
</P><P><B>Materials:</B></P><P>T4 DNA-Ligase, T4-Ligase-buffer, samples:</P><P>eGFP + pSB1C3 backbone(2793bp, MW=1725.2 kDA) = 59,1 ng/ÂµL -&gt; 34 nM </P><P>PCR-product 1 (598 bp, MW=363.4 kDA)= 22,8 ng/ÂµL -&gt; 62 nM </P><P>PCR-product 2 (598 bp, MW=363.4 kDA)= 25,2 ng/ÂµL -&gt; 69.3 nM </P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation (same for both samples):
</P><UL><LI> 1 ÂµL eGFP + pSB1C3 backbone 59,1 ng/ÂµL -&gt; 34 nM
</LI></UL><UL><LI> 2 ÂµL PCR-product 22,8 ng/ÂµL -&gt; 62 nM
</LI></UL><UL><LI> 5 Âµl wather
</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase
</LI></UL><UL><LI> 1 ÂµL T4 DNA-ligase buffer
</LI></UL><P>incubation of samples for 1,5 h at room temperature</P><P><B>Results:</B></P><P>ligated plasmids(final construct AID without NES, with NLS+Kozak sequence+eGFP) for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: transformation of ligated samples (AID without NES, with NLS+Kozak sequence+eGFP)</P><P><B>Investigators:</B> Chris</P><P><B>Time:</B> 2012-08-09 5 pm</P><P><B>Materials:</B></P><UL><LI> samples: ligated AID without NES, with NLS+Kozak sequence 1+eGFP in pSB1C3, ligated AID without NES, with NLS+Kozak sequence 2+eGFP in pSB1C3</LI></UL><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of ligated samples were used
</P><P>Plate incubation start: 5:00 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-08-10"><P style="background-color: rgb(240, 20, 70);">2012-08-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> PCR of AID to create AID without NES, with NLS+Kozak sequence </P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Time:</B> 2012-08-10 8 am</P><P><B>Aim:</B></P><P>PCR of AID to create AID without NES, with NLS+Kozak sequence </P><P><B>Method:</B></P><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing + elongation </TD><TD> 72 </TD><TD> 45 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 25 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P> PCR-clean up, final concentrations: spektra inconsistent, A260/280 very low(maybe contamination, problem with clean up):</P><P>negativ control 1: 97.4 ng/ÂµL (spektrum inconsistent) </P><P>negativ control 2: 108.3 ng/ÂµL (spektrum inconsistent) </P><P>PCR product 1-1: <B>73.1 ng/ÂµL</B> (spektrum inconsistent but somehow looked like a DNA curve) </P><P>PCR product 1-2: 141.5 ng/ÂµL (spektrum inconsistent) </P><P>PCR product 2-1: 222 ng/ÂµL (spektrum inconsistent) </P><P>PCR product 2-2: 162.2 ng/ÂµL (spektrum inconsistent) </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> test digestion of PCR product(AID without NES, with NLS+Kozak sequence) with PmlI </P><P><B>Investigators:</B> Rico, Chris </P><P><B>Materials:</B></P><UL><LI> samples: negative control and PCR1-1 (AID without NES, with NLS+Kozak sequence)
</LI></UL><UL><LI> FastDigest PmlI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> sterile water
</LI></UL><P><B>Method:</B></P><UL><LI> 30 Âµl mix: 1 ÂµL negative control, 1ÂµL PmlI, 3ÂµL 10x FD Green Buffer, 25 ÂµL sterile water
</LI></UL><UL><LI> 30 Âµl mix: 2 ÂµL negative control, 1ÂµL PmlI, 3ÂµL 10x FD Green Buffer, 24 ÂµL sterile water
</LI></UL><UL><LI> 30 Âµl mix: 2 ÂµL PCR product 1, 1ÂµL PmlI, 3ÂµL 10x FD Green Buffer, 24 ÂµL sterile water
</LI></UL><UL><LI> 30 Âµl mix: 3 ÂµL PCR product 1, 1ÂµL PmlI, 3ÂµL 10x FD Green Buffer, 23 ÂµL sterile water
</LI></UL><UL><LI> digestion incubated at 37Â°C for 2 h 30 min
</LI></UL><P><B>further tasks:</B></P><UL><LI> gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel electrophoresis to control the test digestion of the PCR-product (AID without NES, with NLS+Kozak sequence) </P><P><B>Investigators:</B> Rico</P><P><B>Time:</B> 2012-08-11; </P><P><B>Materials:</B></P><P>gel electrophoresis equipment</P><P>digested samples</P><P><B>Method:</B></P><P>loading slots with 30 ÂµL digested sample (digested samples and 7,5 ÂµL Loading dye) </P><P>standard gel electrophoresis procedure</P><P><B>Results:</B></P><P>(sizes were calculated with ImageJ)
</P><P><B>Further tasks:</B></P><P>sequencing
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> planing the AID phage display project </P><P><B>Investigators:</B> Tom S.,Rico , Chris </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Inoculation of AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3, pBAD-mYFP Venus (phage display: arabinose promoter-vector), CREB-PYP UT156- sSD-pAK100 (phage display: vector for antibody-P3-fusion) </P><P><B>Investigators:</B> Chris
</P><P><B>Time:</B> 2012-08-10</P><P><B>Materials:</B></P><P>LB medium </P><P>Chloramphenicol 25 mg/ ml stock solution</P><P>Plasmids: pSB1C3 with AID without NES, with NLS+Kozak sequence(PCR1&amp;2)+eGFP-chloramphenicol resistance, pBAD-mYFP Venus-ampicillin resistance, CREB-PYP UT156- sSD-pAK100-chloramphenicol resistance
</P><P><B>Method:</B></P><P>Inoculation of:
</P><P>1 culture pSB1C3 with AID without NES, with NLS+Kozak sequence(PCR1)+eGFP from plate in 5 ml LB medium + 5ÂµL chloramphenicol
</P><P>1 culture pSB1C3 with AID without NES, with NLS+Kozak sequence(PCR2)+eGFP from plate in 5 ml LB medium + 5ÂµL chloramphenicol
</P><P>2 cultures pBAD-mYFP Venus from cryostock in 5 ml LB medium + 5ÂµL ampicillin
</P><P>2 CREB-PYP UT156- sSD-pAK100 from cryostock in 5 ml LB medium + 5ÂµL chloramphenicol</P><P>shaking over night at 37Â°C, 300 rpm, approx. 16 hours </P><P><B>Further tasks:</B></P><P>Miniprep </P><H3><SPAN class="mw-headline" id="2012-08-11"><P style="background-color: rgb(240, 20, 70);">2012-08-11</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">glycerolstocks &amp; miniprep of AID without NES, with NLS+Kozak sequence+eGFP</P><P><B>Investigators:</B> Basia </P><P>Time:
</P><P>2012-08-11 10:30 am
</P><P>Materials:
</P><P>Glycerol, Miniprep Kit, overnight culture
</P><P>Method:
</P><P>Glycerol stock: 300 ÂµL Glycerol 80Â % + 700 ÂµL overnight cultures --&gt; put in -80 Â°C freezer </P><P>Miniprep of overnight cultures (pAK 100, PCR1+eGFP im PSB1C3, PCR2+eGFP im PSB1C3, Arabinose promoter in pBad)
</P><P>Results:
</P><P>DNA concentrations via nanodrop:</P><P>pak 100 = 187,7ng/Âµl</P><P>PCR1+eGFP im PSB1C3 = <FONT color="red">12,5ng/Âµl</FONT></P><P>PCR2+eGFP im PSB1C3 = 227,2 ng/Âµl</P><P>Arabinose promoter im pbad = 341,2 ng/Âµl</P><P><B>Further tasks:</B></P><P>test digestion
</P><H3><SPAN class="mw-headline" id="2012-08-13"><P style="background-color: rgb(240, 20, 70);">2012-08-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> PCR of AID to create AID without NES, with NLS+Kozak sequence with PstI</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Time:</B> 2012-08-13 11 am</P><P><B>Aim:</B></P><P>PCR of AID to create AID without NES, with NLS+Kozak sequence with PstI (in the next step the PCR-product will be digested to find out whether the PCR was successful or not)</P><P><B>Method:</B></P><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (Plasmid) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing + elongation </TD><TD> 72 </TD><TD> 45 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 25 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P> PCR-clean up, final concentrations:</P><P>PCR 1: 20,9 ng/ÂµL </P><P>PCR 2: 145,1 ng/ÂµL </P><P>PCR 3: 71,6 ng/ÂµL </P><P>PCR 4: 32,4 ng/ÂµL</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> preparative digestion, test digestion and gel electrophoresis </P><P><B>Investigators:</B>Rico, Tom S., Chris </P><P><B>Materials:</B></P><UL><LI> samples: AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3, AID in pSB1C3, pBAD-mYFP Venus, CMV in pSB1C3, AID without NES, with NLS+Kozak sequence
</LI></UL><UL><LI> FastDigest PmlI, XbaI, PstI, SpeI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> sterile water
</LI></UL><P><B>Method:</B></P><UL><LI> 30 Âµl mix: 2ÂµL test digestion sample, 1ÂµL PmlI, 3ÂµL 10x FD Green Buffer, 24 ÂµL sterile water
</LI></UL><UL><LI> 30 Âµl mix: 25 ÂµL sample, 1ÂµL each Enzyme, 3ÂµL 10x FD Green Buffer
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>Enzyme 1</B></TD><TD><B>Enzyme 2</B></TD></TR><TR><TD> pBAD-YFP Venus</TD><TD> XbaI</TD><TD>PstI</TD></TR><TR><TD> AID</TD><TD> XbaI</TD><TD> PstI</TD></TR><TR><TD>AID+eGFP</TD><TD> PmlI</TD><TD> -</TD></TR><TR><TD> AID-PCR-product AID without NES, with NLS+Kozak sequence</TD><TD> PmlI</TD><TD>-</TD></TR><TR><TD> AID-PCR-product</TD><TD> XbaI</TD><TD> PstI </TD></TR><TR><TD> CMV </TD><TD> XbaI </TD><TD> PstI </TD></TR><TR><TD> AID+eGFP</TD><TD> XbaI </TD><TD> PstI </TD></TR><TR><TD> CMV </TD><TD> SpeI</TD><TD> PstI </TD></TR></TBODY></TABLE><UL><LI> digestion incubated at 37Â°C for 2,5-5 h
</LI></UL><P><B>results:</B></P><P><B>further tasks:</B></P><UL><LI> gel extraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> gel extraction</P><P><B>Investigators:</B>Chris, Tom S.</P><P><B>Materials:</B></P><P>centrifuge, Nucleo Spin and PCR clean up Kit, heat block, nanodrop
</P><P><B>Method:</B></P><P>extract DNA: according to the manual
</P><P><B>Results:</B></P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>mass concentration</B></TD><TD><B>molecular weight </B></TD><TD><B>molar concentration </B></TD></TR><TR><TD> pBAD cut X+P</TD><TD> 50.4 ng/ÂµL</TD><TD> 2484.7 kDA</TD><TD> 20.28 nM </TD></TR><TR><TD> CMV cut: S+P</TD><TD> 18.9 ng/ÂµL</TD><TD> 1681.32 kDA</TD><TD> 11.24 nM </TD></TR><TR><TD> mod. AID1(with P) cut: X+P</TD><TD> 65.4 ng/ÂµL</TD><TD> 378.3 kDA</TD><TD> 172.9 nM </TD></TR><TR><TD> mod. AID4(with P) cut: X+P</TD><TD> 14.8 ng/ÂµL</TD><TD> 378.3 kDA</TD><TD> 39.1 nM </TD></TR><TR><TD> wt AID cut: X+P</TD><TD> 7.0 ng/ÂµL</TD><TD> 386.28 kDA</TD><TD> 18 nM </TD></TR><TR><TD> CMV cut: X+P</TD><TD> 52.1 ng/ÂµL</TD><TD> 1272.52 kDA</TD><TD> 40.9 nM </TD></TR></TBODY></TABLE><P>the samples could be confounded? we`ll have to control it via gel electrophoresis</P><P><B>Further tasks:</B></P><P>ligation of fragments
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of 4 further cultures of AID without NES, with NLS+Kozak sequence+ eGFP in pSB1C3</P><P><B>Investigators:</B> Chris
</P><P><B>Time:</B> 2012-08-13 5 pm</P><P><B>Materials:</B></P><P>LB medium </P><P>Chloramphenicol 25 mg/ ml stock solution</P><P>plates with cultures: AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 (from 2012.08.09)
</P><P><B>Method:</B></P><P>Inoculation of:
</P><P>2 cultures pSB1C3 with AID without NES, with NLS+Kozak sequence(PCR1)+eGFP per plate in 5 ml LB medium + 5ÂµL chloramphenicol
</P><P>(--&gt; 4 cultures)
</P><P><B>Further tasks:</B></P><P>Miniprep </P><H3><SPAN class="mw-headline" id="2012-08-14"><P style="background-color: rgb(240, 20, 70);">2012-08-14</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Plasmid isolation of AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3</P><P><B>Investigators:</B></P><P>Rico </P><P><B>Time:</B> 2012-08-14 8 am</P><P><B>Aim:</B></P><P>Isolation of AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3</P><P><B>Method:</B></P><P>Miniprep: according to the manual
</P><P><B>Results:</B></P><UL><LI> AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 1: 183.3 ng/ÂµL </LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 3: 257.5 ng/ÂµL </LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 4: 180.6 ng/ÂµL </LI></UL><P><B>Further Tasks:</B></P><UL><LI>Digestion with PstI, XbaI (preparative digestion) and PmlI (test digestion)
</LI></UL><UL><LI>Gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Preparative digestion of isolated AID without NES, with NLS+Kozak sequence+eGFP with XbaI and PstI and test digestion of AID without NES, with NLS+Kozak sequence+eGFP with PmlI and gel electrophoresis</P><P><B>Investigators:</B></P><P>Chris, Rico </P><P><B>Time:</B> 2012-08-14 8 am</P><P><B>Aim:</B></P><P>preparative and test digestion of AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3</P><P><B>Method:</B></P><P>Digestion with XbaI, PstI and PmlI, gel electrophoresis</P><P><B>Results:</B></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction of digested AID without NES, with NLS+Kozak sequence+eGFP</P><P><B>Investigators:</B></P><P>Rico </P><P><B>Method:</B></P><P>Gel extraction kit</P><P><B>Results:</B></P><UL><LI> AID without NES, with NLS+Kozak sequence+eGFP (digested with X and P): 135.2 ng/ÂµL
</LI></UL><P><B>Further Tasks:</B></P><UL><LI>Ligation of AID without NES, with NLS+Kozak sequence+eGFP with CMV-Promotor
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of CMV and AID without NES, with NLS+Kozak sequence(+P)+eGFP1&amp;4, pSB1C3 + AID without NES, with NLS+Kozak sequence(+P)+eGFP1&amp;4 ,pBAD +wtAID</P><P><B>Investigators:</B></P><P>Chris </P><P><B>Materials:</B></P><P>T4 DNA-Ligase, T4-Ligase buffer, water, samples</P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation:
</P><TABLE border="1"><TBODY><TR><TD><B>Fragment 1(BB)</B></TD><TD><B>Fragment 2(insert)</B></TD><TD><B>T4 DNA-Ligase</B></TD><TD><B>T4 DNA-Ligase buffer</B></TD><TD><B>water</B></TD></TR><TR><TD> pBAD cut X+P (20.28 nM)</TD><TD>wt AID cut: X+P (18 nM)</TD></TR><TR><TD>2 ÂµL</TD><TD>6 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>0 ÂµL</TD></TR><TR><TD>CMV cut: S+P (11.24 nM)</TD><TD>mod. AID1(with P)+eGFP cut: X+P (172.9 nM) </TD></TR><TR><TD>5 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>2 ÂµL</TD></TR><TR><TD>CMV cut: S+P (11.24 nM)</TD><TD>mod. AID4(with P)+eGFP cut: X+P (39.1 nM) </TD></TR><TR><TD>4 ÂµL</TD><TD>4 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>0 ÂµL</TD></TR><TR><TD>pSB1C3 cut: X+P (40.9 nM)</TD><TD>mod. AID1(with P)+eGFP cut: X+P (172.9 nM) </TD></TR><TR><TD>3 ÂµL</TD><TD>2 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>3 ÂµL</TD></TR><TR><TD>pSB1C3 cut: X+P (40.9 nM)</TD><TD>mod. AID4(with P)+eGFP cut: X+P (39.1 nM) </TD></TR><TR><TD>2 ÂµL</TD><TD>6 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>0 ÂµL</TD></TR></TBODY></TABLE><P>incubation of samples for 1,5 h at room temperature</P><P><B>Results:</B></P><P>ligated plasmids ready for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated samples AID without NES, with NLS+Kozak sequence(+P)+eGFP+CMV and pBad + wtAID</P><P><B>Investigators:</B> Chris</P><P><B>Materials:</B></P><UL><LI> samples (see above): ligated pBAD +wtAID (ampR), BB+CMV+modAID(+P) from PCR 1&amp;4 (chlR), pSB1C+ modAID(+P) from PCR 1&amp;4 (chlR)</LI></UL><P>-&gt; 2 plates per ligated sample +2 extra plates pBAD +wtAID (condensed water from the lid dripped on the plated cultures in the first two plates), all in all 12 plates
</P><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol &amp; ampicillin (for pBAD +wtAID)
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of ligated samples were used
</P><P>Plate incubation start: 4 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><H3><SPAN class="mw-headline" id="2012-08-15"><P style="background-color: rgb(240, 20, 70);">2012-08-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation and transformation of CMV (Insert + backbone) and AID without NES, with NLS+Kozak sequence+eGFP (insert)</P><P><B>Investigators:</B></P><P>Rico, Tom S. </P><P><B>Aim:</B></P><P>Ligation of CMV (Insert + backbone) and AID without NES, with NLS+Kozak sequence+eGFP (insert), later transformation in XL-1
</P><P><B>Materials:</B></P><UL><LI> T4 DNA-Ligase, samples(CMV and AID)</LI></UL><UL><LI> Bunsen Burner, Agar Plate with Chloramphenicol, 37 Â°C heater, centrifuge, spoon for weight</LI></UL><UL><LI> ligated sample (compare last step 14.08-2012)
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation:
</P><UL><LI> 5 ÂµL (CMV Fragment)(MW: 1681,3 kDa) c=18,9 ng/ÂµL(11,2 nM) -&gt; 5,6 nM
</LI></UL><UL><LI> 1 ÂµL (mod. AID+eGFP Fragment) (MW: 821,92 kDa) c=135,2 ng/ÂµL(164,5 nM) -&gt; 16,5 nM
</LI></UL><UL><LI> 1 ÂµL (T4 DNA-Ligase)
</LI></UL><UL><LI> 1 ÂµL (T4 DNA-Ligase Buffer)
</LI></UL><UL><LI> 2 ÂµL (DNase free water)
</LI></UL><P>Transformation via manual
</P><P>Plate incubation start: 5 pm
</P><P>incubation of sample for 1,5 h at 22 Â°C</P><P><B>Results:</B></P><P>location: -20 Â°C freezer, topmost drawer
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of AID without NES, with NLS+Kozak sequence in pSB1C3, AID without NES, with NLS+Kozak sequence+CMV and pBAD+WT AID</P><P><B>Investigators:</B> Tom S.
</P><P><B>Time:</B> 2012-08-15 5 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Chloramphenicol 25 mg/ ml stock solution</LI></UL><UL><LI> Ampicillin 100 mg/ ml stock solution</LI></UL><UL><LI>plates with cultures: AID without NES, with NLS+Kozak sequence+CMV in pSB1C3 (from 2012-08-14), AID without NES, with NLS+Kozak sequence in pSB1C3 and pBAD+WT AID
</LI></UL><P><B>Method:</B></P><P>Inoculation of:
</P><P>2 clones of every plate
</P><P><B>Further tasks:</B></P><P>Miniprep and cryo stocks </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of pKMEF425bla</P><P><B>Investigators:</B> Chris</P><P><B>Materials:</B></P><UL><LI> 1 ÂµL pKMEF425bla Plasmid
</LI></UL><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 1 Âµl of pKMEF425bla Plasmid was used
</P><P>Plate incubation start: 3:30 pm
</P><P><B>Results:</B></P><P>2 plates (Chloramphenicol) with XL1 blue <I>E. coli</I> colonies carrying pKMEF425bla
</P><P><B>Further tasks:</B></P><P>picking clones &amp; inoculation (for our group and the antibody group)
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">PCR of AID to create AID with BpiI-restriction sides for TAL-AID fusion, cooperation with team Freiburg</P><P><B>Investigators:</B></P><P>Chris, Rico </P><P><B>Aim:</B></P><P>PCR of AID to create AID with BpiI-restriction sides for TAL-AID fusion, cooperation with team Freiburg</P><P><B>Method:</B></P><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume [ÂµL]</B></TD></TR><TR><TD>HF Phusion buffer 5x</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>1</TD></TR><TR><TD>Primer (Forward)</TD><TD>1,25</TD></TR><TR><TD>Primer (Reverse)</TD><TD>1,25</TD></TR><TR><TD> DNA (BBa_K103001 AID in pSB1A2 10 ng/Âµl) </TD><TD> 1,0 </TD></TR><TR><TD> Phusion Polymerase </TD><TD> 0,5 </TD></TR><TR><TD> water </TD><TD> 35,0 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="1"><TBODY><TR><TD><B>step</B></TD><TD><B>Temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD>30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD> annealing</TD><TD> 50 </TD><TD> 20 </TD><TD> 17 </TD></TR><TR><TD>elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 17 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 5 </TD><TD> 17 </TD></TR><TR><TD>annealing+elongation </TD><TD> 72 </TD><TD> 18 </TD><TD> 17 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 600 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P> PCR-clean up, final concentrations:</P><P>PCR TAL-AID1: 80.7 ng/ÂµL, inconsistent curve- no DNA (also see gels below), can be discarded</P><P>PCR TAL-AID2: 202.6 ng/ÂµL </P><P>PCR TAL-AID3: 123.3 ng/ÂµL </P><P>PCR TAL-AID4: 124 ng/ÂµL, inconsistent curve- no DNA (also see gels below), can be discarded</P><UL><LI>PCR TAL-AID ist just <B>5` XbaI BpiI AID (without NES) BpiI(reversed) PstI 3`</B>(but will be fused with TAL-therefore this PCR-product is called PCR TAL-AID)
</LI></UL><P>Position: -20Â°C freezer-extrabox &quot;phagedisplay, cooperation&quot;
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> test digestion of PCR TAL-AID 1-4 with PmlI and uncut PCR TAL-AID 1-4, Gelelectrophoresis</P><P><B>Investigators:</B></P><P>Chris, Tom S., Rico </P><P><B>Aim:</B></P><P>check whether the PCR worked</P><P><B>Method:</B></P><P>Gel electrophoresis of PCR TAL-AID 1-4 uncut &amp; PCR TAL-AID 1-4 cut with PmlI</P><P><B>Results:</B></P><P>the lowest lane could be primer dimers, therefore, PCR TAL-AID 2 &amp; 3 will be used for further cloning. 1 and 4 can be discarded. The Location is the box phagedisplay and cooperation (-20 Â°C freezer).
</P><H3><SPAN class="mw-headline" id="2012-08-16"><P style="background-color: rgb(240, 20, 70);">2012-08-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures of AID without NES, with NLS+Kozak sequence in pSB1C3, pBAD-wt AID and CMV-AID without NES, with NLS+Kozak sequence</P><P><B>Investigators:</B></P><P>Rico, Tom S. </P><P><B>Aim:</B></P><P>Miniprep of AID without NES, with NLS+Kozak sequence in pSB1C3, pBAD-wt AID and CMV-AID without NES, with NLS+Kozak sequence
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence in pSB1C3 4-1-1</TD><TD>224.3</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence in pSB1C3 1-2-2</TD><TD>297.0</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence in pSB1C3 4-1-2</TD><TD>210.4</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence in pSB1C3 1-2-1</TD><TD>246.7</TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence in pSB1C3 4-2-2</TD><TD> 271.4 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence in pSB1C3 4-2-1</TD><TD> 229.1 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence in pSB1C3 1-1-2</TD><TD> 193.4 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence in pSB1C3 1-1-1</TD><TD> 233.5 </TD></TR><TR><TD> pBAD wt AID 2-1</TD><TD> 143.7 </TD></TR><TR><TD> pBAD wt AID 3-1</TD><TD> 140.9</TD></TR><TR><TD> pBAD wt AID 4-2</TD><TD> 211.6 </TD></TR><TR><TD> pBAD wt AID 1-1</TD><TD> 164.9 </TD></TR><TR><TD> pBAD wt AID 3-2</TD><TD> 156.6 </TD></TR><TR><TD> pBAD wt AID 4-1</TD><TD> 145.3</TD></TR><TR><TD> pBAD wt AID 2-2</TD><TD> 114.6 </TD></TR><TR><TD> pBAD wt AID 1-2</TD><TD> 114.6</TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 4-1-2</TD><TD> 254.3 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 1-1-2</TD><TD> 334.8 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 1-2-1</TD><TD> 228.7 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 1-2-2</TD><TD> 311.9 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 1-1-1</TD><TD> 352.9 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 4-2-2</TD><TD> 241.2 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 4-1-1</TD><TD> 346.5 </TD></TR><TR><TD> CMV + AID without NES, with NLS+Kozak sequence 4-2-1</TD><TD> 318.2 </TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><P>Test digestion of AID without NES, with NLS+Kozak sequence in pSB1C3 with PmlI, XpaI and PstI, and test digestion of CMV-AID without NES, with NLS+Kozak sequence with XbaI and PstI
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> test digestion of CMV+AID without NES, with NLS+Kozak sequence with XbaI and PstI and test digestion of pBAD+WT AID and AID without NES, with NLS+Kozak sequence in pSB1C3 with XbaI, PstI and PmlI, Gel electrophoresis</P><P><B>Investigators:</B></P><P>Tom S.</P><P><B>Aim:</B></P><P>check which clone is the right one</P><P><B>Method:</B></P><P>Gel electrophoresis of all cut samples</P><P><B>Results:</B></P><P>We chose 1-1-2, 4-2-1 and 4-2-2 of AID without NES, with NLS+Kozak sequence in pSB1C3; all pBAD+WT AID and nothing of CMV + AID without NES, with NLS+Kozak sequence for further preparation.</P><P><B>Further Tasks:</B></P><P>sequencing and further cloning of the complete constructs
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of CMV+AID without NES, with NLS+Kozak sequence+eGFP</P><P><B>Investigators:</B> Tom S.
</P><P><B>Time:</B> 2012-08-16 5 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Chloramphenicol 25 mg/ ml stock solution</LI></UL><UL><LI>plates with cultures: CMV+AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 (from 2012.08.15.)
</LI></UL><P><B>Method:</B></P><P>Inoculation of:
</P><P>2 clones of every plate
</P><P>(--&gt; 4 cultures)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep and cryo stocks </LI></UL><H3><SPAN class="mw-headline" id="2012-08-17"><P style="background-color: rgb(240, 20, 70);">2012-08-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures of CMV+AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures of CMV+AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific: according to the manual</P><P><B>Results:</B></P><P>1-1 = 391,3 ng/ÂµL</P><P>1-2 = 372,1 ng/ÂµL</P><P>2-1 = 409,7 ng/ÂµL</P><P>2-2 = 422,7 ng/ÂµL</P><P><B>Further Tasks:</B></P><P>Test digestion of these samples PmlI, XpaI and PstI
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> test digestion of CMV+AID without NES, with NLS+Kozak sequence+eGFP with XbaI, PmlI and PstI, Gel electrophoresis</P><P><B>Investigators:</B></P><P>Tom S.</P><P><B>Aim:</B></P><P>check which clone is the right one</P><P><B>Method:</B></P><P>digestion for 4h</P><P>Gel electrophoresis of all cut samples</P><P><B>Results:</B></P><P>We chose 2-1 and 2-2 of CMV+AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 for further preparation.</P><P><B>Further Tasks:</B></P><P>sequencing and further cloning of the complete constructs
</P><H3><SPAN class="mw-headline" id="2012-08-20"><P style="background-color: rgb(240, 20, 70);">2012-08-20</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Preparative digestion of isolated CMV+AID without NES, with NLS+Kozak sequence+eGFP and AID without NES, with NLS+Kozak sequence with SpeI and PstI and TAL-Dom. with XbaI and PstI and Gelelectrophoresis</P><P><B>Investigators:</B></P><P>Tom S. </P><P><B>Time:</B> 2012-08-20 8 am</P><P><B>Aim:</B></P><P>preparative digestion </P><P><B>Method:</B></P><P>Digestion with XbaI, PstI and SpeI, Gel electrophoresis</P><P><B>Results:</B></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Gel extraction of digested samples from 20.08.2012</P><P><B>Investigators:</B></P><P>Chris</P><P><B>Method:</B></P><P>Gel extraction kit</P><P><B>Results:</B></P><UL><LI> CMV+AID without NES, with NLS+Kozak sequence+eGFP 2-1 (in pSB1C3)(digested with S and P): 236.2 ng/ÂµL (96.6 nM)
</LI></UL><UL><LI> CMV+AID without NES, with NLS+Kozak sequence+eGFP 2-2 (in pSB1C3)(digested with S and P): 37.0 ng/ÂµL (15.1 nM)
</LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence 1-1-2 (in pSB1C3)(digested with S and P): 135.1 ng/ÂµL (82.7)
</LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence 4-2-1 (in pSB1C3)(digested with S and P): 171.2 ng/ÂµL (104.8 nM)
</LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence 4-2-2 (in pSB1C3)(digested with S and P): 96 ng/ÂµL (58.7 nM)
</LI></UL><UL><LI> AID for Fusion with TAL 2 (digested with X and P):51.1 ng/ÂµL (136.6 nM)
</LI></UL><UL><LI> AID for Fusion with TAL 3 (digested with X and P):35.8 ng/ÂµL (95.5 nM)
</LI></UL><P><B>Further Tasks:</B></P><P>Ligation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of CMV+AID without NES, with NLS+Kozak sequence+eGFP in pSB1C3 (S+P) &amp; hGH poly A (X+P), AID without NES, with NLS+Kozak sequence in pSB1C3 (S+P) &amp; hGH poly A (X+P), AID for Fusion with TAL (X+P) &amp; pSB1C3 (X+P)</P><P><B>Investigators:</B></P><P>Chris </P><P><B>Materials:</B></P><P>T4 DNA-Ligase, T4-Ligase-buffer, water, samples</P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation:
</P><TABLE border="0"><TBODY><TR><TD><B>Fragment 1(BB)</B></TD><TD><B>Fragment 2(insert)</B></TD><TD><B>T4 DNA-Ligase</B></TD><TD><B>T4 DNA-Ligase Buffer</B></TD><TD><B>water</B></TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence +eGFP <B>2-1</B> in pSB1C3 cut:S+P (96.6 nM)</TD><TD> hGH poly A cut: X+P (48.5 nM)</TD></TR><TR><TD>0.5 ÂµL</TD><TD>2 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>5.5 ÂµL</TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence +eGFP <B>2-2</B> in pSB1C3 cut:S+P (15.1 nM)</TD><TD> hGH poly A cut: X+P (48.5 nM) </TD></TR><TR><TD>7 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>0 ÂµL</TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence <B>4-2-1</B> in pSB1C3 cut S+P (82.7 nM) </TD><TD> hGH poly A cut: X+P (48.5 nM) </TD></TR><TR><TD>0.5 ÂµL</TD><TD>2 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>5.5 ÂµL</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence <B>4-2-2</B> in pSB1C3 cut S+P (104.8 nM)</TD><TD> hGH poly A cut: X+P (48.5 nM) </TD></TR><TR><TD>0.5 ÂµL</TD><TD>2 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>5.5 ÂµL</TD></TR><TR><TD>AID without NES, with NLS+Kozak sequence <B>1-1-2</B> in pSB1C3 cut S+P (58.7 nM)</TD><TD>AID without NES, with NLS+Kozak sequence 4(with P) cut: X+P (39.1 nM) </TD></TR><TR><TD>0.5 ÂµL</TD><TD>2 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>5.5 ÂµL</TD></TR><TR><TD>AID for Fusion with TAL <B>2</B> cut: X+P (136.6 nM)</TD><TD>pSB1C3 cut: X+P (40.9 nM) </TD></TR><TR><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>6 ÂµL</TD></TR><TR><TD>AID for Fusion with TAL <B>3</B> cut: X+P (95.5 nM)</TD><TD>pSB1C3 cut: X+P (40.9 nM) </TD></TR><TR><TD>2 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>5 ÂµL</TD></TR></TBODY></TABLE><P>incubation of samples for 1,5 h at room temperature</P><P><B>Results:</B></P><P>ligated plasmids ready for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated samples</P><P><B>Investigators:</B> Chris</P><P><B>Materials:</B></P><UL><LI> samples (see above): ligated CMV+AID without NES, with NLS+Kozak sequence +eGFP +hGH poly A in pSB1C3, AID without NES, with NLS+Kozak sequence + hGH-polyA pSB1C3 , AID for Fusion with TAL in pSB1C3 (all chlR)</LI></UL><P>-&gt;1 plate per ligated sample, all in all 7 plates
</P><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of ligation samples were used
</P><P>Plate incubation start: 9.20 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of pKMEF425bla, CMV in pSB1C3, CMV+wtAID+hGH-polyA in pSB1C3</P><P><B>Investigators:</B> Chris
</P><P><B>Time:</B> 2012-08-16 7 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Chloramphenicol 25 mg/ ml stock solution</LI></UL><UL><LI>plates with cultures of pKMEF425bla (from 2012.08.16.)
</LI></UL><UL><LI>cryo sotocks of CMV in pSB1C3 &amp; CMV+wtAID+hGH-polyA in pSB1C3 Nr. 3
</LI></UL><P><B>Method:</B></P><P>Inoculation of:
</P><P>2 clones of pKMEF425bla in 5 ml LB,
</P><P>1 tip of cryo stock from CMV in pSB1C3 (in 5 ml LB + Chloramph) and of CMV+wtAID+hGH-polyA in pSB1C3 Nr. 3 (in 50 ml LB + Chloramph)
</P><P>(--&gt; 4 cultures)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep(pKMEF425bla, CMV in pSB1C3) and endotoxinfree mediprep (CMV+wtAID+hGH-polyA in pSB1C3) and cryo stocks </LI></UL><H3><SPAN class="mw-headline" id="2012-08-21"><P style="background-color: rgb(240, 20, 70);">2012-08-21</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of pKMEF425bla &amp; CMV in pSB1C3 CMV+wtAID+hGH-polyA in pSB1C3</P><P><B>Investigators:</B> Chris </P><P><B>Materials:</B></P><P>Miniprep Kit
</P><P>overnight cultures (5 mL) of pKMEF425bla and CMV in pSB1C3 </P><P><B>Method:</B></P><P>according to manual
</P><P><B>Results:</B> concentrations:
</P><UL><LI> CMV in pSB1C3 = 247.5 ng/ÂµL
</LI></UL><UL><LI> pKMEF425bla= 100 ng/ÂµL
</LI></UL><P><B>Further tasks:</B></P><P>preperative digestion of CMV with SpeI and PstI </P><P>test digestion of pKEF425bla with XbaI and AgeI </P><P>preperative digestion of pKEF425bla AscI and SfiI </P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> endotoxin free mediprep CMV+wtAID+hGH-polyA in pSB1C3 for transfection in CHO </P><P><B>Investigators:</B> Chris </P><P><B>Materials:</B></P><UL><LI> endotoxin free Mediprep Kit
</LI></UL><UL><LI> overnight cultures (50 mL) of CMV+wtAID+hGH-polyA in pSB1C3 </LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B> concentrations:
</P><UL><LI> CMV+wtAID+hGH-polyA in pSB1C3(200 ÂµL)=513.8 ng/ÂµL
</LI></UL><P><B>Further tasks:</B></P><P>Transfection into CHO cells
</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">analytical digestion pKEF425bla (XbaI&amp; AgeI) and preparative digestion of CMV-SpeI &amp; PstI (with high and low DNA amount)</P><P><B>Investigators:</B> Chris</P><P><B>Materials:</B></P><P>preparative CMV High:
</P><UL><LI>25 ÂµL CMV in pSB1C3 (247,5 ng/ÂµL)
</LI></UL><UL><LI>3 ÂµL FD-Green Buffer
</LI></UL><UL><LI>1 ÂµL SpeI
</LI></UL><UL><LI>1 ÂµL PstI</LI></UL><P>preparative CMV Low:</P><UL><LI>6 ÂµL CMV in pSB1C3 (247,5 ng/ÂµL)
</LI></UL><UL><LI>19 ÂµL water
</LI></UL><UL><LI>3 ÂµL FD-Green Buffer
</LI></UL><UL><LI>1 ÂµL SpeI
</LI></UL><UL><LI>1 ÂµL PstI</LI></UL><P>test digestion pKEF425bla (NEB-Method)
</P><UL><LI>2 ÂµL pKMEF425bla (100 ng/ÂµL)
</LI></UL><UL><LI>23 ÂµL water
</LI></UL><UL><LI>3 ÂµL NEB4 Buffer
</LI></UL><UL><LI>1 ÂµL XbaI
</LI></UL><UL><LI>1ÂµL AgeI</LI></UL><UL><LI> digestion incubated at 37Â°C for 2h 30 min
</LI></UL><P><B>further tasks:</B></P><UL><LI> gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">gel electrophoresis (test digestion pKMEF425bla - XbaI &amp; AgeI, preparative digestion CMV in pSB1C3- SpeI &amp; PstI) and gel extraction of CMV in pSB1C3</P><P>concentrations after gel extraction:</P><P>pSB1C3+CMV high= 191.8 ng/ÂµL</P><P>pSB1C3+CMV low= 55.4 ng/ÂµL
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of CMV+AID without NES, with NLS+Kozak sequence +eGFP+hGH, AID without NES, with NLS+Kozak sequence+hGH and TAL-AID in pSB1C3</P><P><B>Investigators:</B> Chris, Tom S.
</P><P><B>Time:</B> 2012-08-16 7 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Chloramphenicol 25 mg/ml stock solution</LI></UL><UL><LI>plates with cultures of TAL-AID in pSB1C3 (from 2012.08.20.)
</LI></UL><UL><LI>plates with cultures of CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH (from 2012.08.20.)
</LI></UL><UL><LI>plates with cultures of AID without NES, with NLS+Kozak sequence+hGH (from 2012.08.20.)
</LI></UL><P><B>Method:</B></P><P>Inoculation of:
</P><P>2 clones per plate
</P><P>(--&gt; 14 cultures)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep and cryo stocks </LI></UL><H3><SPAN class="mw-headline" id="2012-08-22"><P style="background-color: rgb(240, 20, 70);">2012-08-22</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Miniprep of overnight cultures of CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH, AID without NES, with NLS+Kozak sequence+hGH and TAL-AID in pSB1C3</P><P><B>Investigators:</B></P><P>Chris, Tom S. </P><P><B>Aim:</B></P><P>Miniprep of overnight cultures of CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH, AID without NES, with NLS+Kozak sequence+hGH and TAL-AID in pSB1C3
</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific: according to the manual</P><P><B>Results:</B></P><P><B>Concentrations</B></P><TABLE border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>Concentration [ng/ÂµL]</B></TD></TR><TR><TD>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH 2-1-1</TD><TD>397.6</TD></TR><TR><TD>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH 2-1-2</TD><TD>446.1</TD></TR><TR><TD>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH 2-2-1</TD><TD>418.9</TD></TR><TR><TD>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH 2-2-2</TD><TD>266.6</TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+hGH 1-2-2-1</TD><TD> 319.3 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+hGH 1-2-2-2</TD><TD> 377.8 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+hGH 4-2-1-1</TD><TD> 415.0 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+hGH 4-2-1-2</TD><TD> 317.0 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+hGH 4-2-2-1</TD><TD> 307.0 </TD></TR><TR><TD> AID without NES, with NLS+Kozak sequence+hGH 4-2-2-2</TD><TD> 317.0</TD></TR><TR><TD> pBAD wt AID 4-2</TD><TD> 211.6 </TD></TR><TR><TD> TAL-AID in pSB1C3 2-1</TD><TD> 291.1 </TD></TR><TR><TD> TAL-AID in pSB1C3 2-2</TD><TD> 305.4 </TD></TR><TR><TD> TAL-AID in pSB1C3 3-1</TD><TD> 460.5 </TD></TR><TR><TD> TAL-AID in pSB1C3 3-2</TD><TD> 292.2 </TD></TR></TBODY></TABLE><P><B>Further Tasks:</B></P><P>Test digestion of all with PmlI, XbaI and PstI, and preparative digestion of AID without NES, with NLS+Kozak sequence+hGH with XbaI and PstI
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> test digestion of AID without NES, with NLS+Kozak sequence+hGH, CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH and TAL-Dom. in pSB1C3 with XbaI, PstI and PmlI, Gelelectrophoresis</P><P><B>Investigators:</B></P><P>Chris, Tom S.</P><P><B>Aim:</B></P><P>check which clone is the right one</P><P><B>Method:</B></P><P>Gel electrophoresis of all cut samples</P><P><B>Results:</B></P><P>we need to repeat the test digestion of CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH and AID without NES, with NLS+Kozak sequence+hGH;</P><P><B>Further Tasks:</B></P><P>sequencing and further cloning of the complete constructs; repeat the digestion
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Preparative digestion of isolated AID without NES, with NLS+Kozak sequence+hGH-polyA with XbaI and PstI and gel extraction after gel electrophoresis</P><P><B>Investigators:</B></P><P>Chris, Tom S. </P><P><B>Time:</B> 2012-08-22</P><P><B>Aim:</B></P><P>preparative digestion</P><P><B>Method:</B></P><P>Digestion with XbaI, PstI, SfiI and AscI, gel electrophoresis and gel extraction</P><P><B>Results:</B></P><UL><LI> AID without NES, with NLS+Kozak sequence+hGH-polyA 1-2-2-1 (in pSB1C3)(digested with X and P): 25.9 ng/ÂµL -&gt; 38.2 nM (MW: 678,7 kDa)
</LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence+hGH-polyA 1-2-2-2 (in pSB1C3)(digested with X and P): 18.2 ng/ÂµL -&gt; 26.8 nM
</LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-1 (in pSB1C3)(digested with X and P): 39.4 ng/ÂµL -&gt; 58.1 nM
</LI></UL><UL><LI> AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-2 (in pSB1C3)(digested with X and P): 38.7 ng/ÂµL -&gt; 57.0 nM
</LI></UL><P><B>Further Tasks:</B></P><P>Ligation
</P><H3><SPAN class="mw-headline" id="2012-08-23"><P style="background-color: rgb(240, 20, 70);">2012-08-23</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">repetition of test digestion of AID without NES, with NLS+Kozak sequence+hGH-polyA and CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA+eGFP with XbaI, PstI and PmlI, gel electrophoresis</P><P><B>Investigators:</B></P><P>Chris</P><P><B>Aim:</B></P><P>check which clone is the right one</P><P><B>Method:</B></P><P>Gel electrophoresis of all digested samples</P><P><B>Results:</B></P><P><B>Further Tasks:</B></P><P>sequencing and further cloning of the complete constructs
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Ligation of AID without NES, with NLS+Kozak sequence+hGH-polyA (X+P) with pSB1C3+CMV (S+P) </P><P><B>Investigators:</B></P><P>Chris </P><P><B>Materials:</B></P><P>T4 DNA-Ligase, T4-Ligase-buffer, water, samples</P><P><B>Method:</B></P><P>DNA Fragment ligation: according to the manual</P><P>sample preparation:
</P><TABLE border="1"><TBODY><TR><TD><B>Fragment 1(BB)</B></TD><TD><B>Fragment 2(insert)</B></TD><TD><B>T4 DNA-Ligase</B></TD><TD><B>T4 DNA-Ligase Buffer</B></TD><TD><B>water</B></TD></TR><TR><TD> pSB1C3+CMV S+P low(32 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 1-2-2-1 cut: X+P (38.2 nM)</TD></TR><TR><TD>1 ÂµL</TD><TD>3 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>4 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P high(191 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 1-2-2-1 cut: X+P (38.2 nM)</TD></TR><TR><TD>0.5 ÂµL</TD><TD>7.5 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>0 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P low(32 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 1-2-2-2 cut: X+P (26.8 nM)</TD></TR><TR><TD>1 ÂµL</TD><TD>4 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>3 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P high(191 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 1-2-2-2 cut: X+P (26.8 nM)</TD></TR><TR><TD>0.5 ÂµL</TD><TD>7.5 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>0 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P low(32 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-1 cut: X+P (58.1 nM)</TD></TR><TR><TD>2 ÂµL</TD><TD>4 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>2 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P high(191 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-1 cut: X+P (58.1 nM)</TD></TR><TR><TD>0.5 ÂµL</TD><TD>6 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1.5 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P low(32 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-2 cut: X+P (57 nM)</TD></TR><TR><TD>2 ÂµL</TD><TD>4 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>2 ÂµL</TD></TR><TR><TD> pSB1C3+CMV S+P high(191 nM)</TD><TD> AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-2 cut: X+P (57 nM)</TD></TR><TR><TD>0.5 ÂµL</TD><TD>6 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>1.5 ÂµL</TD></TR></TBODY></TABLE><P>incubation of samples for 1,5 h at room temperature</P><P><B>Results:</B></P><P>ligated plasmids for transformation
</P><P><B>Further tasks:</B></P><P>Transformation
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: Transformation of ligated samples</P><P><B>Investigators:</B> Chris</P><P><B>Materials:</B></P><UL><LI> samples (see above): AID without NES, with NLS+Kozak sequence+hGH-polyA (X+P) with pSB1C3+CMV (S+P) (chlR)</LI></UL><P>-&gt; 1 plates per ligated sample, 8 plates
</P><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 10 Âµl of ligation samples were used
</P><P>Plate incubation start: 6 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparing ligated pcdna5frt-scfv-tev-tmd-eyfp-construct from 2012-08-21 for ordering sequencing by GATC</P><P><B>Investigators:</B> Kerstin/Stefan/Sascha</P><P><B>Materials/Methods:</B></P><UL><LI>dilute DNA (clons 4,8) according to GATC requirements
</LI><LI>Geneious</LI><LI>Using GATC standard-primer pcdna3.1FP.1 and pcdna3.1RP.1</LI></UL><P><B>Further Tasks:</B></P><UL><LI> analysis of sequencing results
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-24"><P style="background-color: rgb(240, 20, 70);">2012-08-24</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B></P><P>Tom S.</P><P>mod.AID = AID without NES, with NLS+Kozak sequence
</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD><TD><B>GATC number</B></TD></TR><TR><TD> mod. AID 1-1-2 Forward </TD><TD> II3282 </TD><TD> pSB1C3 Forward </TD><TD> 919656 </TD></TR><TR><TD> mod. AID 1-1-2 Reverse </TD><TD> II3294 </TD><TD> pSB1C3 Reverse </TD><TD> 919657 </TD></TR><TR><TD> mod. AID 4-2-1 Forward </TD><TD> II3283 </TD><TD> pSB1C3 Forward </TD><TD> 919656 </TD></TR><TR><TD> mod. AID 4-2-1 Reverse </TD><TD> II3295 </TD><TD> pSB1C3 Reverse </TD><TD> 919657 </TD></TR><TR><TD> mod. AID 4-2-2 Forward </TD><TD> II3284 </TD><TD> pSB1C3 Forward </TD><TD> 919656 </TD></TR><TR><TD> mod. AID 4-2-2 Reverse </TD><TD> II3296 </TD><TD> pSB1C3 Reverse </TD><TD> 919657 </TD></TR><TR><TD> mod. AID+eGFP 3 Forward</TD><TD> II3285 </TD><TD> pSB1C3 Forward </TD><TD> 919656 </TD></TR><TR><TD> mod. AID+eGFP 3 Reverse </TD><TD> II3297 </TD><TD> pSB1C3 Reverse </TD><TD> 919657 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-1 Forward </TD><TD> II3286 </TD><TD> pSB1C3 Forward </TD><TD> 919658 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-1 Reverse </TD><TD> II3298 </TD><TD> pSB1C3 Reverse </TD><TD> 919659 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-1 AID </TD><TD> II3299 </TD><TD> AID-C-Term Forward </TD><TD> 919660 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-1 eGFP </TD><TD> II3300 </TD><TD> eGFP-N-Term Reverse </TD><TD> 919661 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-2 Forward </TD><TD> II3287 </TD><TD> pSB1C3 Forward </TD><TD> 919658 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-2 Reverse </TD><TD> II3301 </TD><TD> pSB1C3 Reverse </TD><TD> 919659 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-2 AID </TD><TD> II3302 </TD><TD> AID-C-Term Forward </TD><TD> 919660 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-1-2 eGFP </TD><TD> II3303 </TD><TD> eGFP-N-Term Reverse </TD><TD> 919661 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-1 Forward </TD><TD> II3288 </TD><TD> pSB1C3 Forward </TD><TD> 919658 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-1 Reverse </TD><TD> II3304 </TD><TD> pSB1C3 Reverse </TD><TD> 919659 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-1 AID </TD><TD> II3305 </TD><TD> AID-C-Term Forward </TD><TD> 919660 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-1 eGFP </TD><TD> II3306 </TD><TD> eGFP-N-Term Reverse </TD><TD> 919661 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-2 Forward </TD><TD> II3289 </TD><TD> pSB1C3 Forward </TD><TD> 919658 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-2 Reverse </TD><TD> II3307 </TD><TD> pSB1C3 Reverse </TD><TD> 919659 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-2 AID </TD><TD> II3308 </TD><TD> AID-C-Term Forward </TD><TD> 919660 </TD></TR><TR><TD> CMV+mod. AID+eGFP+hGH 2-2-2 eGFP </TD><TD> II3309 </TD><TD> eGFP-N-Term Reverse </TD><TD> 919661 </TD></TR><TR><TD> TAL-Dom. 2-1 Forward </TD><TD> II3290 </TD><TD> pSB1C3 Forward </TD><TD> 919662 </TD></TR><TR><TD> TAL-Dom. 2-1 Reverse </TD><TD> II3310 </TD><TD> pSB1C3 Reverse </TD><TD> 919663 </TD></TR><TR><TD> TAL-Dom. 2-2 Forward </TD><TD> II3291 </TD><TD> pSB1C3 Forward </TD><TD> 919662 </TD></TR><TR><TD> TAL-Dom. 2-2 Reverse </TD><TD> II3311 </TD><TD> pSB1C3 Reverse </TD><TD> 919663 </TD></TR><TR><TD> TAL-Dom. 3-1 Forward </TD><TD> II3292 </TD><TD> pSB1C3 Forward </TD><TD> 919662 </TD></TR><TR><TD> TAL-Dom. 3-1 Reverse </TD><TD> II3312 </TD><TD> pSB1C3 Reverse </TD><TD> 919663 </TD></TR><TR><TD> TAL-Dom. 3-2 Forward </TD><TD> II3293 </TD><TD> pSB1C3 Forward </TD><TD> 919662 </TD></TR><TR><TD> TAL-Dom. 3-2 Reverse </TD><TD> II3313 </TD><TD> pSB1C3 Reverse </TD><TD> 919663 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparative digestion of pAK 100 and scFV with AscI and SfiI</P><P><B>Investigators:</B></P><P>Rico</P><P><B>Aim:</B></P><P>digest pAK 100 and scFV into fragments</P><P><B>Method:</B></P><UL><LI> 3 ÂµL NEB4 buffer, 1 ÂµL AscI, 25 ÂµL DNA, water
</LI></UL><UL><LI> 4 h -&gt; 37 Â°C
</LI></UL><UL><LI> ad 1 ÂµL SfiI, 4 h -&gt; 50 Â°C
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-27"><P style="background-color: rgb(240, 20, 70);">2012-08-27</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">gel electrophoresis of pAK100 (SfiI &amp; AscI) &amp; pKMEF425bla (AscI+SfiI) and gel extraction of pKMEF425bla (AscI+SfiI) </P><P><B>Investigators:</B></P><P>Basia </P><P><B>Results:</B></P><P>final concentration pKMEF425bla (AscI+SfiI):3.4 ng/ÂµL
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B></P><P>Rico, Chris </P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD><TD><B>GATC number</B></TD></TR><TR><TD> pBad+WtAID 4-2 </TD><TD> II3321 </TD><TD>pBAD-FP</TD><TD>10199242 </TD></TR><TR><TD>pBad+WtAID 4-2 </TD><TD> II3322 </TD><TD>pTrcHis-RP </TD><TD>10199243 </TD></TR><TR><TD> pBad+WtAID 4-1 </TD><TD> II3323 </TD><TD>pBAD-FP</TD><TD>10199244 </TD></TR><TR><TD> pBad+WtAID 4-1 </TD><TD> II3324 </TD><TD> pTrcHis-RP </TD><TD> 10199245 </TD></TR><TR><TD> pBad+WtAID 1-1 </TD><TD> II3325 </TD><TD>pBAD-FP</TD><TD>10199246 </TD><P>&lt;/tr&gt;</P></TR><TR><TD> pBad+WtAID 1-1 </TD><TD> II3326 </TD><TD> pTrcHis-RP </TD><TD> 10199247 </TD></TR></TBODY></TABLE><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> inoculation of cryo stocks of pAK100 and pKMEF425bla; CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA </P><P><B>Investigators:</B> Chris, Rico
</P><P><B>Time:</B> 2012-08-16 7 pm</P><P><B>Materials:</B></P><UL><LI> LB medium </LI></UL><UL><LI> Chloramphenicol 25 mg/ ml stock solution</LI></UL><UL><LI>plates with cultures CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA: 4-2-2-1 L, 4-2-2-1 H, 4-2-2-2 L, 4-2-2-2 H, 1-2-2-1 L, 1-2-2-1 H, 1-2-2-2 L, 1-2-2-2 H
</LI></UL><UL><LI>cryostocks of pAK100 and pKMEF425bla
</LI></UL><P><B>Method:</B></P><P>Inoculation of:
</P><P>1 clones per plate
</P><P>(--&gt; 8 cultures)
</P><P><B>Further tasks:</B></P><UL><LI> Miniprep and cryo stocks </LI></UL><H3><SPAN class="mw-headline" id="2012-08-28"><P style="background-color: rgb(240, 20, 70);">2012-08-28</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Miniprep and cryostocks of overnight cultures </P><P><B>Investigators:</B> Rico
</P><P><B>Aim:</B> purification of the CMV+AID without NES, with NLS+Kozak sequence+hGH in pSB1C3 construct</P><P><B>Method:</B></P><P>Miniprep Thermo Scientific</P><P><B>Results:</B></P><P>pAK 100 = 414.8 ng/ÂµL
</P><P>1-2-2-1-H = 311,3 ng/ÂµL</P><P>1-2-2-1-L = 249,4 ng/ÂµL</P><P>1-2-2-2-H = 303,5 ng/ÂµL</P><P>1-2-2-2-L = 230,9 ng/ÂµL</P><P>4-2-2-1-H = 259,3 ng/ÂµL</P><P>4-2-2-1-L = 288,3 ng/ÂµL</P><P>4-2-2-2-H = 513,8 ng/ÂµL</P><P>4-2-2-2-L = 325,7 ng/ÂµL</P><P><B>Further Tasks:</B></P><P>Test digestion of these samples with PmlI, XpaI and PstI
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Test digestion of CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA with PmlI, XpaI and PstI </P><P><B>Investigators:</B> Basia, Rico
</P><P><B>Aim:</B> Test digestion of CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA construct with PmlI, XpaI and PstI</P><P><B>Method:</B></P><UL><LI>1 ÂµL DNA, 1 ÂµL PmlI, 1 ÂµL PstI, 1 ÂµL XbaI, 3 ÂµL FD Buffer, 23 ÂµL water</LI></UL><UL><LI>incubation for 1,5 h at 37 Â°C</LI></UL><P><B>Results:</B></P><P><A href="/File:UP_12AID-28-08_2.jpg" title="File:UP 12AID-28-08 2.jpg">300px</A></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Checking the sequencing data</P><P><B>Investigators:</B> Tom S.
</P><P><B>Aim:</B> choose the right constructs</P><P><B>Results:</B></P><P>AID without NES, with NLS+Kozak sequence in pSB1C3 1-1-2 -&gt; construct not the right one</P><P>AID without NES, with NLS+Kozak sequence in pSB1C3 4-2-1 -&gt; mutation of the 4th nt in the Kozak sequence (impact can't estimate)</P><P>AID without NES, with NLS+Kozak sequence in pSB1C3 4-2-2 -&gt; is good</P><P>TAL-Dom. in pSB1C -&gt; all ok</P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH -&gt; the flanking sequences are good, but the middle primers were wrong and must be repeated
</P><H3><SPAN class="mw-headline" id="2012-08-29"><P style="background-color: rgb(240, 20, 70);">2012-08-29</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparative digestion of pAK 100 with AscI and SfiI</P><P><B>Investigators:</B></P><P>Rico</P><P><B>Aim:</B></P><P>Preparative digestion of pAK 100 with AscI and SfiI for Phage Display</P><P><B>Method:</B></P><UL><LI>10 ÂµL DNA, 15 ÂµL water, 3 ÂµL 10x FD Buffer, 1 ÂµL AscI 6 h @ 37 Â°C
</LI></UL><UL><LI>add 1 ÂµL SfiI 5 h @ 50 Â°C</LI></UL><P><B>Further Tasks:</B></P><P>gel electrophoresis</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Test digestion of CMV+AID without NES, with NLS+Kozak sequence+hGH construct with PmlI, PstI and XbaI</P><P><B>Investigators:</B></P><P>Basia, Rico</P><P><B>Aim:</B></P><P>Test digestion of CMV+AID without NES, with NLS+Kozak sequence+hGH 4-2-2-2 L, 4-2-2-2 H, 4-2-2-1 L, 4-2-2-1 H</P><P><B>Method:</B></P><UL><LI>0.5 ÂµL DNA, 0.5 ÂµL PmlI, 0.5 ÂµL PstI, 0.5 ÂµL XbaI, 1 ÂµL 10x FD Buffer, 7 ÂµL water</LI></UL><P><B>Results:</B></P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">send the DNA to sequencing</P><P><B>Investigators:</B></P><P>Tom S.</P><TABLE border="1"><TBODY><TR><TD><B>sample</B></TD><TD><B>GATC number</B></TD><TD><B>Seq. Primer</B></TD><TD><B>GATC number</B></TD></TR><TR><TD>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-1</TD><TD> II3331 </TD><TD> AID-C-Term. Forward </TD><TD> 919664 </TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence+eGFP-hGH-polyA 2-1-1 </TD><TD> II3332 </TD><TD> eGFP-N-Term. Reverse </TD><TD> 919665 </TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-1-L </TD><TD> II3333 </TD><TD> pSB1C3 Forward </TD><TD> 919666 </TD></TR><TR><TD> CMV+AID without NES, with NLS+Kozak sequence+hGH 4-2-2-1-L </TD><TD> II3334 </TD><TD> pSB1C3 Reverse </TD><TD> 919667 </TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2012-08-30"><P style="background-color: rgb(240, 20, 70);">2012-08-30</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Endotoxin-free mediprep of CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA construct</P><P><B>Investigators:</B></P><P>Rico, Tom,
</P><P><B>Method:</B></P><P>Midiprep
</P><P><B>Result:</B></P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-2 = 1546.7 ng/ÂµL</P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-2-1 = 1115.0 ng/ÂµL</P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-1 = 1358.4 ng/ÂµL</P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-2-2 = 1518.9 ng/ÂµL</P><P>POG 4 = 1518.9 ng/ÂµL
</P><P>Clone 4 = 1239.0 ng/ÂµL
</P><P><B>Further Tasks:</B></P><P>construct is ready for transfection
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Miniprep of YFP-construct from transfected CHO-Cells</P><P><B>Investigators:</B></P><P>Rico, Tom
</P><P><B>Aim:</B></P><UL><LI>Miniprep of YFP-construct from transfected CHO cells to proof whether it is possible to transform <I>E. coli</I> with the purified construct for sequencing
</LI></UL><P><B>Method:</B></P><UL><LI>harvesting of the CHO-cells with 0.3 mL Trypsin/EDTA and 0.5 mL medium Ham's F-12 (2.5 * 10^5 cells were seeded 3 day before)
</LI></UL><UL><LI>Miniprep of cells
</LI></UL><UL><LI>Elution with 20 ÂµL
</LI></UL><P><B>Results:</B></P><P>1 Âµg DNA with 1 ÂµL PEI = 33.3 ng/ÂµL</P><P>1 Âµg DNA with 1.5 ÂµL PEI = 55.2 ng/ÂµL</P><P>1 Âµg DNA with 2 ÂµL PEI = 30.9 ng/ÂµL</P><P>1 Âµg DNA with 2.5 ÂµL PEI = 29.4 ng/ÂµL</P><P>1 Âµg DNA with 3 ÂµL PEI = 40.2 ng/ÂµL</P><P>1 Âµg DNA with 3.5 ÂµL PEI = 63.4 ng/ÂµL</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: transformation of Venus vector </P><P><B>Investigators:</B> Basia</P><P><B>Time:</B> 2012-08-30</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plates with chloramphenicol
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1 Blue)
</LI></UL><UL><LI> Venus plasmid - sample 1
</LI></UL><P><B>Method:</B></P><P>Transformation via manual, 2 Âµl of Venus plasmid was used
</P><P>Plate incubation start: 7:30 pm
</P><P><B>Results:</B></P><P>ready mutants to pick clones
</P><P><B>Further tasks:</B></P><P>picking clones
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Preparative digestion of pAK 100 with AscI and SfiI and gel extraction</P><P><B>Investigators:</B></P><P>Chris, Rico
</P><P><B>Results:</B></P><P>c(pAK 100 cut:AscI and SfiI) from left lane:9 ng/ÂµL</P><P>c(pAK 100 cut:AscI and SfiI) from right lane: 19 ng/ÂµL</P><P>because the fragments where not separated well we repeated the digestion (simultanously going on with ligation &amp; transformation)
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Ligation &amp; transformation of pAK 100(AscI and SfiI) &amp; scFV425bla(AscI and SfiI)</P><P>Ligation: </P><TABLE border="1"><TBODY><TR><TD><B>Fragment 1(BB)</B></TD><TD><B>Fragment 2(insert)</B></TD><TD><B>T4 DNA-Ligase</B></TD><TD><B>T4 DNA-Ligase Buffer</B></TD><TD><B>water</B></TD></TR><TR><TD> pAK 100-left lane(9 ng/ÂµL)</TD><TD> 425bla (3 ng/ÂµL)</TD></TR><TR><TD>1/3 ÂµL</TD><TD>3 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>4,6 ÂµL</TD></TR><TR><TD> pAK 100-right lane(50% diluted-&gt;9.5 ng/ÂµL)</TD><TD> 425bla (3 ng/ÂµL)</TD></TR><TR><TD>1 ÂµL</TD><TD>3 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>6 ÂµL</TD></TR><TR><TD> pAK 100-right lane(50% diluted-&gt;9.5 ng/ÂµL)</TD><TD> 425bla (3 ng/ÂµL)</TD></TR><TR><TD>0.1 ÂµL</TD><TD>4.5 ÂµL</TD><TD>1 ÂµL</TD><TD>1 ÂµL</TD><TD>3 ÂµL</TD></TR></TBODY></TABLE><P>Transformation:</P><P>Transformation via protocol, 10 ÂµL of ligated samples were used</P><P><B>Results:</B> plates with transformed <I>E. coli</I></P><P><B>Further Tasks:</B> picking clones</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">repetition of preparative digestion of pAK 100 with AscI and SfiI</P><P>sample: </P><P>2.4 ÂµL(~1000 ng) pAK 100</P><P>3 ÂµL 10x NEB 4 Buffer</P><P>2 ÂµL AscI</P><P>22.6 ÂµL water</P><P>incubation at 37 Â°C for 5 h and heat inactivation for 20 min at 65Â°C, freeze at - 20Â°C </P><P><B>Further Tasks:</B> digestion with SfiI
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Checking the sequencing data</P><P><B>Investigators:</B> Chris, Tom S.
</P><P><B>Aim:</B> choose the right constructs</P><P><B>Results:</B></P><P>CMV+AID without NES, with NLS+Kozak sequence+eGFP+hGH-polyA 2-1-1 -&gt; is good</P><P>CMV+AID without NES, with NLS+Kozak sequence+hGH-polyA 4-2-2-1-L -&gt; is good</P><P>pBAD+WT AID 1-1 -&gt; 3 bp in front of start codon and direct after the stop codon are mutations</P><P>pBAD+WT AID 4-1 -&gt; 3 bp in front of start codon and direct after the stop codon are mutations</P><P>pBAD+WT AID 4-2 -&gt; 3 bp in front of start codon and direct after the stop codon are mutations</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: Preparation of overnight culture of <I>E. coli</I> strain XL-1 Blue</P><P><B>Investigators:</B></P><P>Basia </P><P><B>Aim:</B></P><P>preparation of competent cells of <I>E. coli</I> strain XL-1 Blue
</P><P><B>Materials:</B></P><P>LB Medium, Tetracycline, XL-1 Blue stock
</P><P><B>Method:</B></P><P>Competent <I>E. coli</I> -&gt; Standard protocols
</P><P><B>Results:</B></P><P>culture grew
</P><P><B>Further tasks:</B></P><P>further preparation of competent cells with CaCl2.
</P><H3><SPAN class="mw-headline" id="2012-08-31"><P style="background-color: rgb(240, 20, 70);"> 2012-08-31</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: preparation of competent XL1 Blue <I>E. coli</I></P><P><B>Investigators:</B>Basia</P><P><B>Aim:</B></P><P>get competent <I>E. coli</I> Xl1 Blue
</P><P><B>Materials:</B></P><P>CaCl2, overnight culture of <I>E. coli</I> XL1 Blue, Eppendorf tubes
</P><P><B>Method:</B></P><P>Competent <I>E. coli</I> -&gt; Standard protocols
</P><P><B>Results:</B></P><UL><LI> ca. 80 (100 Âµl) Stocks frozen competent <I>E. coli</I> XL1 Blue</LI></UL><UL><LI> location: 2 paper boxes with competent <I>E. coli</I> Xl1 Blue in -80Â°C Freezer (middle shelf, most right)
</LI></UL><P><B>Further tasks:</B></P><UL><LI> testing ability for transformation of competent cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: picking clones &amp;inoculation in 5 mL LB of retransformed Venus out of CHO cells </P><P><B>Investigators:</B> Tom S.</P><P><B>Method:</B> picking clone (and inoculation in 5 ml LB medium + 5Âµl chloramphenicol stock, shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>Miniprep
</P><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: picking clones &amp;inoculation in 5 mL LB of pAK 100(AscI and SfiI) &amp; scFV425bla(AscI and SfiI) </P><P><B>Investigators:</B> Chris </P><P><B>Method:</B> picking clones (and inoculation in 5 ml LB medium + 5Âµl chloramphenicol stock, shaking over night at 37Â°C, 300 rpm, approx. 16 hours</P><P><B>Further tasks:</B></P><P>Miniprep
</P><H2><SPAN class="mw-headline" id="Antibody">Antibody</SPAN></H2><H3><SPAN class="mw-headline" id="2012-08-02_2"><P style="background-color: rgb(240, 20, 70);">2012-08-02</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Planning the antibody construct with new antibody (GCN4) and optimizing the old ones</P><P><B>Investigators:</B> Maria</P><P><B>Time:</B> 2012-08-02 9- 15 pm</P><P><B>Materials:</B> Geneious</P><P><B>Results:</B> optimized constructs with scFv 425bla, scFv425-72000, scFv GCN4 </P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">miniprep of ligated pcDNA5-FRT-scFv-TEV-TMD-EYFP genecostruct from 2012-07-31</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI> Miniprep Kit
</LI></UL><UL><LI> o.n. cultures of 2 clones from each ligation (1:2, 1:3, 1:4)</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results:</B> concentration measurement of preped ligated geneconstrcuts</P><UL><LI> clone 1 from 1:2 ligation=
</LI></UL><UL><LI> clone 2 from 1:2 ligation=
</LI></UL><UL><LI> clone 1 from 1:3 ligation=
</LI></UL><UL><LI> clone 2 from 1:3 ligation=
</LI></UL><UL><LI> clone 1 from 1:4 ligation=
</LI></UL><UL><LI> clone 2 from 1:4 ligation=
</LI></UL><P><B>further tasks:</B></P><UL><LI> analytical digestion
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;">analytical digestion of ligated pcDNA5-FRT-scFv-TEV-TMD-EYFP genecostruct from 2012-07-31</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI> all 6 preped ligated pcDNA5-FRT-scFv-TEV-TMD-EYFP genecostruct from 2012-07-31
</LI></UL><UL><LI> FastDigest NheI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> steril water
</LI></UL><P><B>Method:</B></P><UL><LI> 10Âµl mix: 1Âµl NheI, 1Âµl 10x FD Green Buffer, 1Âµl of each clone (approximately 850ng DNA), steril water ad to 10Âµl
</LI></UL><UL><LI> digestion incubated at 37Â°C for 30Â´
</LI></UL><P><B>further tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;">Topic: gelelectrophoresis of analytical digested pcDNA5-FRT-scFv-TEV-TMD-EYFP genecostruct from 2012-07-31 </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> checking plasmid-size after ligation of pcDNA5-FRT-scFv-TEV-TMD-EYFP in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel, 90ml
</LI></UL><UL><LI> 70 min at 115V
</LI></UL><P><B>Results:</B></P><UL><LI> ligation failed </LI></UL><P><B>Further Tasks:</B></P><UL><LI> amplifying new scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><UL><LI> preparative digestion of pcDNA5-FRT and scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><UL><LI>ligation of pcDNA5-FRT and scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-03_2"><P style="background-color: rgb(240, 20, 70);">2012-08-03</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Reviewing old antibody constructs and planning construct with new anti-GFP nanobody</P><P><B>Investigators:</B> Maria</P><P><B>Time:</B> 2012-08-03 10 - 17 pm</P><P><B>Materials:</B> Geneious</P><P><B>Results:</B> optimized constructs with anti-GFP nanobody and mcherry </P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> amplifying of scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> amplifying of scFv-TEV-TMD-EYFP geneconstruct</P><P><B>Materials:</B></P><UL><LI> final assembeld scFv-TEV-TMD-EYFP geneconstruct from 2012-07-28
</LI></UL><UL><LI> P7_Gesamtanfang (forward-primer)</LI></UL><UL><LI> P8_Gesamtende (reverse-primer)</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><P>pcr-mix: 10Âµl HF-Buffer; 1Âµl dNTPs(10mM); 2Âµl of final assembeled scFv-TEV-TMD-EYFP geneconstruct from 2012-07-28;2,5Âµl of P7_Gesamtanfang (forward-primer); 2,5Âµl of P8_Gesamtende (reverse-primer); nuclease-free water ad to 50Âµl; 0,5Âµl Phusion-polymerase</P><UL><LI> PCR-prgram with end-primers: initial Denat. 40Â´Â´ at 98Â°C; Denat 12Â´Â´ at 98Â°C; Annealing 15Â´Â´ at 57Â°C; Elongation 27Â´Â´ at 72Â°C; final-Elongation 10Â´at 72Â°C; 32 cycles
</LI></UL><P><B>further Tasks:</B></P><UL><LI> PCR-Clean-Up
</LI></UL><UL><LI> analytical gelelectrophoresis of cleaned and amplifyied scFv-TEV-TMD-EYFP genecostruct
</LI></UL><UL><LI> digestion of cleaned and amplifyied scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean-Up of scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> cleaning of scFv-TEV-TMD-EYFP genecostruct</P><P><B>Materials:</B></P><UL><LI> PCR-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results: concentration measurement via nanodrop</B></P><UL><LI> concentration of cleaned scFv-TEV-TMD-EYFP genecostruct=
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of cleaned and amplifyied scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> analytical gelelectrophoresis to check size of amplifyied scFv-TEV-TMD-EYFP geneconstruct in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1Âµl 10x FD Green, 1Âµl of amplifyied scFv-TEV-TMD-EYFP geneconstruct, water add to 10Âµl
</LI></UL><UL><LI> 1% agarosegel, 90ml
</LI></UL><UL><LI> 70 min at 120V
</LI></UL><P><B>Results:</B></P><UL><LI> amplification worked --&gt; DNA-bond at 1701bp </LI></UL><P><B>Further Tasks:</B></P><UL><LI> preparative digestion of pcDNA5-FRT and scFv-TEV-TMD-EYFPv geneconstruct
</LI></UL><UL><LI>ligation of pcDNA5-FRT and scFv-TEV-TMD-EYFPv geneconstruct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of pcDNA5-FRT from </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> digestion of pcDNA5-FRT with NheI and ApaI to generate sticky ends for ligation with scFv-TEV-TMD-EYFPv geneconstruct </P><P><B>Materials:</B></P><UL><LI> Fast Digest NheI
</LI></UL><UL><LI> Fast Digest ApaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> pcDNA5-FRT(gut) from
</LI></UL><UL><LI> steril water
</LI></UL><P><B>Method:</B></P><UL><LI> 2,5Âµl pcDNA5-FRT/gut (729,ng/Âµl) --&gt; 1822ng
</LI></UL><UL><LI> 2Âµl NheI
</LI></UL><UL><LI> 2Âµl ApaI
</LI></UL><UL><LI> 3Âµl 10x FD Green Buffer
</LI></UL><UL><LI> water add to 30Âµl
</LI></UL><UL><LI> digestion for 2,5h at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> dephosphorylation
</LI></UL><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> dephosphorylation of pcDNA5FRT digested with NheI and ApaI</P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B>dephosphorylation of digestd pcDNA5-FRT to prevent religation </P><P><B>Materials:</B></P><UL><LI> Antarctic Phosphatase
</LI></UL><UL><LI> 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> digested pcDNA5-FRT
</LI></UL><P><B>Method:</B></P><UL><LI> 3,3Âµl 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> 1Âµl Antarctic Phosphatase
</LI></UL><UL><LI> incubation for 30min at 37Â°C to dephosphorylate 5Â´-ends of pcDNA5-FRT
</LI></UL><UL><LI> heat inactivation of Phosphatase for 5min at 65Â°C
</LI></UL><UL><LI> store at -20Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><UL><LI> digestion of scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-05"><P style="background-color: rgb(240, 20, 70);">2012-08-05</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of scFv-TEV-TMD-EYFPv geneconstruct </P><P><B>Investigators:</B> Kerstin</P><P><B>Aim:</B> digestion of scFv-TEV-TMD-EYFPv geneconstructwith NheI and ApaI to generate sticky ends for ligation with digested pcDNA5-FRT </P><P><B>Materials:</B></P><UL><LI> Fast Digest NheI
</LI></UL><UL><LI> Fast Digest ApaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> scFv-TEV-TMD-EYFPv geneconstruct
</LI></UL><UL><LI> steril water
</LI></UL><P><B>Method:</B></P><UL><LI> 26Âµl final assembled scFv-TEV-TMD-EYFP geneconstruct(76,4Âµng/Âµl) --&gt; 1986,4ng
</LI></UL><UL><LI> 1Âµl NheI
</LI></UL><UL><LI> 1Âµl ApaI
</LI></UL><UL><LI> 3,1Âµl 10x FD Green Buffer
</LI></UL><UL><LI> water add to 31Âµl
</LI></UL><UL><LI> digestion for 2,5h at 37Â°C
</LI></UL><UL><LI> heat inactivation of NheI and ApaI --&gt; 5min at 65Â°C
</LI></UL><UL><LI> store at -20Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><UL><LI> ligation with digested pcDNA5-FRT
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-06_2"><P style="background-color: rgb(240, 20, 70);">2012-08-06</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Reviewing new construct with anti-GFP nanobody and contacting IDT and GeneArt</P><P><B>Investigators:</B> Maria</P><P><B>Time:</B> 2012-08-06 12 - 14 pm</P><P><B>Materials:</B> Geneious</P><P><B>Results:</B> final construct with anti-GFP nanobody and mcherry </P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative gelelectrophoresis of digested scFv-TEV-TMD-EYFP geneconstruct and digested + dephosporylated pcDNA5-FRT </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> preparative gelelectrophoresis to separate digested scFv-TEV-TMD-EYFP geneconstruct and pcDNA5-FRT in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> digested DNA
</LI></UL><P><B>Method:</B></P><UL><LI> 30Âµl mix of digested and dephosphorylated pcDNA5-FRT
</LI></UL><UL><LI> 31Âµl mix of digested scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><UL><LI> 1% agarosegel, 100ml
</LI></UL><UL><LI> 90 min at 95V
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><UL><LI> ligation
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gelextracton of digested scFv-TEV-TMD-EYFP geneconstruct and digested + dephosporylated pcDNA5-FRT out of 1% agarosegel </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> extract of digested scFv-TEV-TMD-EYFP geneconstruct and pcDNA5-FRT</P><P><B>Materials:</B></P><UL><LI> Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><UL><LI> failure at elution step
</LI></UL><P><B>Further Tasks:</B></P><P>pcr-Clean-up of digested scFv-TEV-TMD-EYFP geneconstruct and pcDNA5-FRT
</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean-Up of digested scFv-TEV-TMD-EYFP geneconstruct and digested + dephosporylated pcDNA5-FRT after gelextraction </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> cleaning and eluting of digested scFv-TEV-TMD-EYFP geneconstruct and pcDNA5-FRT</P><P><B>Materials:</B></P><UL><LI> PCR/Gel-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results: concentration measurement using nanodrop</B></P><UL><LI> concentration of cleaned-up pcDNA5-FRT= 8,4ng/Âµl </LI></UL><UL><LI> concentration of celaned-up scFv-TEV-TMD-EYFP geneconstruct= 10,4ng/Âµl</LI></UL><P><B>Further Tasks:</B></P><UL><LI> ligation of pcDNA5-FRT and scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> ligation of digested scFv-TEV-TMD-EYFP geneconstruct and digested + dephosporylated pcDNA5-FRT </P><P><B>Investigators:</B> Sascha</P><P><B>Aim:</B> ligation of digested scFv-TEV-TMD-EYFP geneconstruct and pcDNA5-FRT</P><P><B>Materials:</B></P><UL><LI> ligation calculator: <A href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html" class="external free" rel="nofollow">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A></LI></UL><UL><LI> T4 DNA ligae
</LI></UL><UL><LI> ligase buffer
</LI></UL><UL><LI> digested scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><UL><LI> digested + dephosporylated pcDNA5-FRT
</LI></UL><P><B>Method: ligation-ratio--&gt; 1:3</B></P><UL><LI> 1Âµl T4 DNA ligase
</LI></UL><UL><LI> 2,5Âµl T4 DNA-ligase buffer
</LI></UL><UL><LI> 11,9Âµl digested + dephosporylated pcDNA5-FRT (8,4ng/Âµl) --&gt; 100ng
</LI></UL><UL><LI> 9,67Âµl digested scFv-TEV-TMD-EYFP geneconstruct (10,4ng/Âµl) --&gt; 100,66ng
</LI></UL><UL><LI> no religation-copntrol
</LI></UL><UL><LI> incubation for 1h at RT
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of ligation mix into XL1-blue competente <I>E. coli</I> cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP into XL1-blue competente <I>E. coli</I> cells</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicilin, 37 Â°C heater, centrifuge,</LI></UL><UL><LI> ligase sample
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> Transformation via manual
</LI></UL><UL><LI> 50Âµl of resuspended cell-suspension were plated on a LB-Amp-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-07_2"><P style="background-color: rgb(240, 20, 70);">2012-08-07</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Preparing GeneArt order of nanobody construct </P><P><B>Investigators:</B> Maria</P><P><B>Time:</B> 2012-08-07 09 - 11 am</P><P><B>Materials:</B> Geneious</P><P><B>Results:</B> Analysis and quotation by GeneArt (24h processing time)</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> colony-pcr of ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Investigators:</B> Sascha</P><P><B>AIM:</B> identifying correct ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Materials:</B></P><UL><LI> Taq DNA-polymerase 1kb/1min
</LI></UL><UL><LI> 10x Taq DNA-polymerase buffer</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer in C-terminus of CMV of pcDNA5-FRT-backbone: fp_qRT-CMV_rev (Tm=55Â°C)
</LI></UL><UL><LI> reverse primer in C-terminus in EYFP: ps_mcherry_rev_BamHI (Tm= 59,31Â°C)
</LI></UL><UL><LI> Thermocycler
</LI></UL><P><B>Methods:</B></P><UL><LI> 20Âµl colony-pcr mix for one clone: 2Âµl 10x Standard Taq-DNA-Polymerase Buffer; 0,4Âµl dNTPs (10mM), 0,4Âµl fp_qRT-CMV_rev(10ÂµM), 0,4Âµl ps_mcherry_rev_BamHI (10ÂµM); 0,1Âµl Taq-DNA-Polymerase; nuclease-free water add to 20Âµl
</LI></UL><UL><LI> colony-pcr mix were adapted to 26 clones
</LI></UL><UL><LI> after transferation of clone into 20Âµl colony-pcr mix, the rest of clone at the tip was plated on a LB-AMP-plate </LI></UL><UL><LI> LB-AMP-plate were incubated o.n. at 37Â°C</LI></UL><UL><LI> negative control: clon #24 from relagtion-control (ligation-ratio 1:0) from 2012-07-31
</LI></UL><UL><LI> program of colony-pcr: initial Denaturation:5Â´at 95Â°C; Denaturation: 25Â´Â´ at 95Â°C, Annealing: 25Â´Â´ at 51Â°C; Elongation: 114Â´Â´ (1845bp) at 68Â°C; final Elongation: 5Â´at 86Â°C; 30 cycles; store at 8Â°C
</LI></UL><P><B>further Taks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> picking positiv clones for o.n.-culture
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gelelectrophoresis of colony-pcr </P><P><B>Investigators:</B> Sascha</P><P><B>AIM:</B> identifying correct ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Materials:</B></P><UL><LI> 1 100 ml 1%agarosegel with 20 slots
</LI></UL><UL><LI> 1 90 ml 1% agarosegel with 9 slots
</LI></UL><UL><LI> 1x TAE-buffer
</LI></UL><UL><LI> agarose
</LI></UL><P><B>Method:</B></P><UL><LI> 90 ml 1% agarosegel for 80 min at 110V
</LI></UL><UL><LI> 100 ml 1% agarosegel for 100 min at 110V </LI></UL><P><B>Results:</B></P><UL><LI> (positive) clones 3, 4, 5, 8, 21, 23 were chosen for o.n.-culture because of desired DNA-bond at appropriate 1845bp
</LI></UL><P><B>further Tasks:</B></P><UL><LI> o.n.-culture of positive clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> o.n.-culture of positive ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Investigators:</B> Tom S.</P><P><B>Materials:</B></P><UL><LI> 8x 15 ml Falcons
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Ampicilin
</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>Method:</B></P><UL><LI> 2x o.n.-culture of clones 8 and 23
</LI></UL><UL><LI> 1x o.n.-culture of clones 3,4,5,21
</LI></UL><UL><LI> each o.n.-culture 5ml LB-Medium + 5Âµl Ampicilin
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-08_2"><P style="background-color: rgb(240, 20, 70);">2012-08-08</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Ordering nanobody construct from GeneArt</P><P><B>Investigators:</B> Maria</P><P><B>Time:</B> 2012-08-08 10 - 11 am</P><P><B>Materials:</B> Geneious</P><P><B>Results:</B> Final nanobody construct, estimated producing time aprox. 16 days</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> miniprep of o.n.-culture of positive ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Investigators:</B> Stefan</P><P><B>Materials:</B></P><UL><LI> Miniprep-Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results: concentration measurement via nanodrop</B></P><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone3=
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone4=
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone5=
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone8=
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone21=
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone23=
</LI></UL><P><B>further Tasks:</B></P><UL><LI> looking for primer for GATC-sequencing of positive pcDNA5-FRT_scFv-TEV-TMD-EYFP_clones
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-10_2"><P style="background-color: rgb(240, 20, 70);">2012-08-10</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparing ligated pcdna5frt-scfv-tev-tmd-eyfp-construct for ordering sequencing by GATC</P><P><B>Investigators:</B> Sascha</P><P><B>Materials:</B></P><UL><LI>dilute DNA (clons 3,8,21) according to GATC requirements
</LI><LI>Geneious</LI><LI>Using GATC standard-primer pcdna3.1FP.1 and pcdna3.1RP.1</LI></UL><P><B>Further Tasks:</B></P><UL><LI> analysis of sequencing results
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-13_2"><P style="background-color: rgb(240, 20, 70);">2012-08-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Cell culture </P><P><B>Investigators:</B> Kerstin </P><P><B>Time:</B> 2012-08-13 3 pm</P><P><B>Materials:</B> culture flasks (75cmÂ²), 6-Well Plates, Trypsin/EDTA, PBS, Ham's F12 Medium </P><P><B>Method:</B></P><UL><LI>passage of CHO Flp-In cells, 1:10 dilution </LI></UL><UL><LI>Preparation for Transfection: seeding of CHO Flp-In cells in 6-Well plates (1*10^5, 1,5*10^5, 2*10^5,..., 3,5*10^5 cells/well)
</LI></UL><UL><LI>Incubation for 2 days
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of ligated pcdna5frt-scfv-tev-tmd-eyfp-construct </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from pcdna5frt-scfv-tev-temd-eyfp-construct (clons 3,8,21)
</LI><LI> Blastn
</LI><LI> Wordpad</LI></UL><P><B>Results:</B></P><UL><LI> all clones lacked approximately 500bp into the eyfp-sequence</LI></UL><P><B>Further Tasks:</B></P><UL><LI> new assembly PCR</LI></UL><H3><SPAN class="mw-headline" id="2012-08-14_2"><P style="background-color: rgb(240, 20, 70);">2012-08-14</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of ligated pcdna5frt-scfv-tev-tmd-eyfp-construct </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from pcdna5frt-scfv-tev-tmd-eyfp-construct (clons 3,8,21)
</LI><LI> GeneiousBlastn</LI></UL><P><B>Results:</B></P><UL><LI> all clones lacked approximately 500bp into the eyfp-sequence</LI></UL><P><B>Further Tasks:</B></P><UL><LI> new assembly PCR</LI></UL><H3><SPAN class="mw-headline" id="2012-08-15_2"><P style="background-color: rgb(240, 20, 70);">2012-08-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transfection of CHO cells with WildTyp AID and CMV_mVenus_construct </P><P><B>Investigators:</B> Kerstin, Stefan </P><P><B>Time:</B> 2012-08-15 11 am</P><P><B>Materials:</B> cells in 6-Well Plates, DNA WildTyp AID, CMV_mVenus_construct, Opti-MEM, PEI Transfection Reagent </P><P><B>Method:</B></P><UL><LI>wells with 80% confluence were chosen for transfection (2,5*10^5cells/well seeded on 13.08.12) </LI></UL><P>protocol 1:
</P><UL><LI> 4,8fold PEI transfection reagent according to DNA amount
</LI></UL><UL><LI> 2,5Âµg CMV_mVenus_construct DNA + 2,5Âµg WildTYp AID DNA, add to 200Âµl with Opti-MEM
</LI></UL><UL><LI> 24Âµl PEI + 176Âµl Opti-MEM
</LI></UL><UL><LI> vortex 3 times for 10 sec
</LI></UL><UL><LI> incubate for 2 min
</LI></UL><UL><LI> mix solutions <B>(always add PEI mix to DNA mix)</B></LI></UL><UL><LI> incubation for 15 min
</LI></UL><UL><LI> spread on cells and mix gently
</LI></UL><P>protocol 2:
</P><UL><LI> 2,5fold PEI transfection reagent according to DNA amount
</LI></UL><UL><LI> 1Âµg CMV_mVenus_construct DNA + 1Âµg WildTYp AID DNA, add to 200Âµl with Opti-MEM
</LI></UL><UL><LI> 5Âµl PEI + 195Âµl Opti-MEM
</LI></UL><UL><LI> vortex 3 times for 10 sec
</LI></UL><UL><LI> incubate for 2 min
</LI></UL><UL><LI> mix solutions <B>(always add PEI mix to DNA mix)</B></LI></UL><UL><LI> incubation for 15 min
</LI></UL><UL><LI> spread on cells and mix gently
</LI></UL><P><B>Results:</B></P><P>fluorescence image of protocol 1 treated cells
</P><P><A href="/File:UP_12_Transfection_16.08_prot1.tiff" title="File:UP 12 Transfection 16.08 prot1.tiff">File:UP 12 Transfection 16.08 prot1.tiff</A></P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of ligated pcdna5frt-scfv-tev-tmd-eyfp-construct </P><P><B>Investigators:</B> Sascha<B>Methods/Materials:</B></P><UL><LI> calling GATC and asking to repeat sequencing reaction of clon3 pcdna5frt-scfv-tev-tmd-eyfp-construct with pcdna3.1-RP.1 primer</LI></UL><P><B>Results:</B><B>Further Tasks:</B></P><UL><LI> analyze repeated sequencing</LI></UL><H3><SPAN class="mw-headline" id="2012-08-16_2"><P style="background-color: rgb(240, 20, 70);">2012-08-16</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Retransformation with scFv, transmembrane domain and YFP</P><P>&lt;b&gt;Investigators: Kerstin/Stefan </P><P><B>Aim:</B> Retransformation with scFv anti-EGFR </P><P><B>Date/Time:</B> 15th August 2012, 2:30 â 4:30 pm </P><P><B>Materials:</B> competent <I>E. coli</I> cells XL1 Blue, scFv anti-EGFR, 42Â°/37Â° C shaker, centrifuge</P><P><B>Method:</B> see transformation protocol </P><P><B>Results:</B> colonies on Chloramphenicol plates with scFv </P><P><B>Further tasks:</B> picking clones and overnight culture (16th August)</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of clon3 pcdna5frt-scfv-tev-tmd-eyfp-construct with pcdna3.1-RP.1 primer </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from pcdna5frt-scfv-tev-temd-eyfp-construct clons 3
</LI><LI> Blastn
</LI><LI> Wordpad</LI></UL><P><B>Results:</B></P><UL><LI> clone 3 lacks approximately 500bp into the eyfp-sequence</LI></UL><P><B>Further Tasks:</B></P><UL><LI> troubleshooting
</LI><LI> new assembly PCR</LI></UL><H3><SPAN class="mw-headline" id="2012-08-17_2"><P style="background-color: rgb(240, 20, 70);">2012-08-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> extension-pcr of scFv-TEV-TMD-EYFP geneconstruct

</P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Materials:</B>scFv anti-EGFR, BBa-KI57000 with transmembrane domain; BBa_E0030 with EYFP</P><UL><LI> P1_Signalp_N-term</LI></UL><UL><LI> P2_Signalp_C-term</LI></UL><UL><LI> P3_TMD-N-term</LI></UL><UL><LI> P4_TMD-C-term/N-YFP</LI></UL><UL><LI> P5_YFP-N-term/TMD-C-term</LI></UL><UL><LI> P6_YFP-C-term</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 50Âµl PCR mix of each plasmid= 10Âµl HF-Buffer; 1Âµl dNTPs(10mM); forward and reverse primer each 2,5Âµl (10ÂµM); template DNA: 1Âµl of 200ng/Âµl scFv--&gt; 200ng, 5Âµl of BBa_E0030 with EYFP (46,3ng/Âµl--&gt; ca. 230ng),5Âµl of BBa-KI57000 with transmembrane domain (43,9ng/Âµl --&gt; ca. 230ng); Phusion DNA polymerase 0,5Âµl; nclease-free water ad to 50Âµl </LI></UL><UL><LI> PCR-program: initial Denat. 40Â´Â´ at 98Â°C; Denat 12Â´Â´ at 98Â°C; Annealing 15Â´Â´ at 55Â°C; Elongation 11Â´Â´ at 72Â°C; final-Elongation 10Â´at 72Â°C; 30 cycles
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> gelelectrophoresis of extended genes after extension-pcr</P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> separation of extended genes scFv-TEV, TEV-TMD, EYFP in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><UL><LI> extended genes
</LI></UL><P><B>Method:</B></P><UL><LI> 1% agarosegel
</LI></UL><UL><LI> 70 min at 105V
</LI></UL><P><B>Results:</B></P><P><B>Further Tasks:</B></P><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> gelextraction of extended genes after extension-pcr

</P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> gelextraction of extended genes(scFv-TEV, TEV-TMD, EYFP) out of 1% agarosegel </P><P><B>Method:</B></P><UL><LI> DNA extracted according to the manual
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> assembly-pcr
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> assembly-pcr with extended genes to scFv-TEV-TMD-EYFP geneconstruct

</P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> assembly-pcr with extended genes to scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Materials:</B>scFv anti-EGFR, BBa-KI57000 with transmembrane domain; BBa_E0030 with EYFP</P><UL><LI> P7_Gesamtanfang (forward-primer)</LI></UL><UL><LI> P8_Gesamtende (reverse-primer)</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> assembly-pcr-mix--&gt; 3:1:3 10Âµl HF-Buffer; 1Âµl dNTPs(10mM); extended TMD (1ng), extended EYFP (2,75ng), extended scFv (2,75ng); nuclease-free water ad to 45Âµl; 0,5Âµl Phusion-polymerase</LI></UL><UL><LI> PCR-program to let the extended genes prime each other: initial Denat. 40Â´Â´ at 95Â°C; Denat 12Â´Â´ at 95Â°C; Annealing 15Â´Â´ at 68Â°C; Elongation 27Â´Â´ at 72Â°C; final-Elongation 10Â´at 72Â°C; 35 cycles
</LI></UL><UL><LI> after annealing of extended genes to each other 2,5Âµl of primer forw_P7_Gesamtanfang and primer rev_P8_Gesamtende were added
</LI></UL><UL><LI> PCR-prgram with end-primers: initial Denat. 40Â´Â´ at 95Â°C; Denat 12Â´Â´ at 95Â°C; Annealing 15Â´Â´ at 55Â°C; Elongation 27Â´Â´ at 72Â°C; final-Elongation 10Â´at 72Â°C; 30 cycles
</LI></UL><UL><LI> Gelelectrophoresis at 105V for 75`
</LI></UL><P><B>Results:</B></P><UL><LI> extraction of DNA (1701bp)
</LI></UL><P><B>Further Tasks:</B></P><UL><LI>Amplification of assembly pcr-Product
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of clon3 pcdna5frt-scfv-tev-tmd-eyfp-construct with pcdna3.1-RP.1 primer </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from pcdna5frt-scfv-tev-tmd-eyfp-construct clons 3
</LI><LI> Geneious</LI></UL><P><B>Results:</B></P><UL><LI> clone 3 lacks approximately 500bp into the eyfp-sequence</LI></UL><P><B>Further Tasks:</B></P><UL><LI> troubleshooting
</LI><LI> new assembly PCR</LI></UL><H3><SPAN class="mw-headline" id="2012-08-18"><P style="background-color: rgb(240, 20, 70);">2012-08-18</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> amplifying of scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> amplifying of scFv-TEV-TMD-EYFP geneconstruct</P><P><B>Materials:</B></P><UL><LI> final assembeld scFv-TEV-TMD-EYFP geneconstruct from 2012-07-28
</LI></UL><UL><LI> P7_Gesamtanfang (forward-primer)</LI></UL><UL><LI> P8_Gesamtende (reverse-primer)</LI></UL><UL><LI> Phusion -polymerase, dNTPs (10mM), Phusion HF-Buffer
</LI></UL><P><B>Method:</B></P><P>pcr-mix: 10Âµl HF-Buffer; 1Âµl dNTPs(10mM); 2Âµl of final assembeled scFv-TEV-TMD-EYFP geneconstruct from 2012-07-28;2,5Âµl of P7_Gesamtanfang (forward-primer); 2,5Âµl of P8_Gesamtende (reverse-primer); nuclease-free water ad to 50Âµl; 0,5Âµl Phusion-polymerase</P><UL><LI> PCR-program with end-primers: initial Denat. 40Â´Â´ at 98Â°C; Denat 12Â´Â´ at 98Â°C; Annealing 15Â´Â´ at 57Â°C; Elongation 27Â´Â´ at 72Â°C; final-Elongation 10Â´at 72Â°C; 32 cycles
</LI></UL><P><B>further Tasks:</B></P><UL><LI> PCR-Clean-Up
</LI></UL><UL><LI> analytical gelelectrophoresis of cleaned and amplifyied scFv-TEV-TMD-EYFP genecostruct
</LI></UL><UL><LI> digestion of cleaned and amplifyied scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> PCR-Clean-Up of scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> cleaning of scFv-TEV-TMD-EYFP genecostruct</P><P><B>Materials:</B></P><UL><LI> PCR-Clean-Up Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analytical gelelectrophoresis of cleaned and amplifyied scFv-TEV-TMD-EYFP geneconstruct </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> analytical gelelectrophoresis to check size of amplifyied scFv-TEV-TMD-EYFP geneconstruct in 1% agarosegel</P><P><B>Materials:</B></P><UL><LI> agarose
</LI></UL><UL><LI> 1xTAE-buffer
</LI></UL><UL><LI> 10xFD Green Buffer
</LI></UL><P><B>Method:</B></P><UL><LI> 1Âµl 10x FD Green, 1Âµl of amplifyied scFv-TEV-TMD-EYFP geneconstruct, water add to 10Âµl
</LI></UL><UL><LI> 1% agarosegel, 90ml
</LI></UL><UL><LI> 70 min at 120V
</LI></UL><P><B>Results:</B></P><UL><LI> amplification worked --&gt; DNA-bond at 1701bp </LI></UL><P><B>Further Tasks:</B></P><UL><LI> preparative digestion of pcDNA5-FRT and scFv-TEV-TMD-EYFPv geneconstruct
</LI></UL><UL><LI>ligation of pcDNA5-FRT and scFv-TEV-TMD-EYFPv geneconstruct
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> preparative digestion of pcDNA5-FRT from </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> digestion of pcDNA5-FRT with NheI and ApaI to generate sticky ends for ligation with scFv-TEV-TMD-EYFPv geneconstruct </P><P><B>Materials:</B></P><UL><LI> Fast Digest NheI
</LI></UL><UL><LI> Fast Digest ApaI
</LI></UL><UL><LI> 10x FD Green Buffer
</LI></UL><UL><LI> pcDNA5-FRT(gut) from
</LI></UL><UL><LI> steril water
</LI></UL><P><B>Method:</B></P><UL><LI> 2,5Âµl pcDNA5-FRT/gut (729,ng/Âµl) --&gt; 1822ng
</LI></UL><UL><LI> 2Âµl NheI
</LI></UL><UL><LI> 2Âµl ApaI
</LI></UL><UL><LI> 3Âµl 10x FD Green Buffer
</LI></UL><UL><LI> water add to 30Âµl
</LI></UL><UL><LI> digestion for 2,5h at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> dephosphorylation
</LI></UL><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> dephosphorylation of pcDNA5FRT digested with NheI and ApaI</P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B>dephosphorylation of digestd pcDNA5-FRT to prevent religation </P><P><B>Materials:</B></P><UL><LI> Antarctic Phosphatase
</LI></UL><UL><LI> 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> digested pcDNA5-FRT
</LI></UL><P><B>Method:</B></P><UL><LI> 3,3Âµl 10x Antarctic Phosphatase Reaction Buffer
</LI></UL><UL><LI> 1Âµl Antarctic Phosphatase
</LI></UL><UL><LI> incubation for 30min at 37Â°C to dephosphorylate 5Â´-ends of pcDNA5-FRT
</LI></UL><UL><LI> heat inactivation of Phosphatase for 5min at 65Â°C
</LI></UL><UL><LI> store at -20Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> gelextraction
</LI></UL><UL><LI> digestion of scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-19"><P style="background-color: rgb(240, 20, 70);">2012-08-19</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> ligation of digested scFv-TEV-TMD-EYFP geneconstruct and digested + dephosporylated pcDNA5-FRT </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Aim:</B> ligation of digested scFv-TEV-TMD-EYFP geneconstruct and pcDNA5-FRT</P><P><B>Materials:</B></P><UL><LI> ligation calculator: <A href="http://www.insilico.uni-duesseldorf.de/Lig_Input.html" class="external free" rel="nofollow">http://www.insilico.uni-duesseldorf.de/Lig_Input.html</A></LI></UL><UL><LI> T4 DNA ligae
</LI></UL><UL><LI> ligase buffer
</LI></UL><UL><LI> digested scFv-TEV-TMD-EYFP geneconstruct
</LI></UL><UL><LI> digested + dephosporylated pcDNA5-FRT
</LI></UL><P><B>Method: ligation-ratio--&gt; 1:3</B></P><UL><LI> 1Âµl T4 DNA ligase
</LI></UL><UL><LI> 2,5Âµl T4 DNA-ligase buffer
</LI></UL><UL><LI> 11,9Âµl digested + dephosporylated pcDNA5-FRT (8,4ng/Âµl) --&gt; 100ng
</LI></UL><UL><LI> 9,67Âµl digested scFv-TEV-TMD-EYFP geneconstruct (10,4ng/Âµl) --&gt; 100,66ng
</LI></UL><UL><LI> religation-control
</LI></UL><UL><LI> incubation for 1h at RT
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of ligation mix into XL1-blue competente <I>E. coli</I> cells
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> Transformation of ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP into XL1-blue competente <I>E. coli</I> cells</P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Materials:</B></P><UL><LI> Bunsen Burner, Agar Plate with Ampicilin, 37 Â°C heater, centrifuge,</LI></UL><UL><LI> ligase sample
</LI></UL><UL><LI> icebox
</LI></UL><UL><LI> competent <I>E. coli</I> cells (XL 1)
</LI></UL><P><B>Method:</B></P><UL><LI> Transformation via manual
</LI></UL><UL><LI> 50Âµl of resuspended cell-suspension were plated on a LB-Amp-plate
</LI></UL><UL><LI> incubation o.n. at 37Â°C
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> picking clones
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-20_2"><P style="background-color: rgb(240, 20, 70);">2012-08-20</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> colony-pcr of ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>AIM:</B> identifying correct ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Materials:</B></P><UL><LI> Taq DNA-polymerase 1kb/1min
</LI></UL><UL><LI> 10x Taq DNA-polymerase buffer</LI></UL><UL><LI> dNTPs (10mM)
</LI></UL><UL><LI> forward primer in C-terminus of CMV of pcDNA5-FRT-backbone: fp_qRT-CMV_rev (Tm=55Â°C)
</LI></UL><UL><LI> reverse primer in C-terminus in EYFP: ps_mcherry_rev_BamHI (Tm= 59,31Â°C)
</LI></UL><UL><LI> Thermocycler
</LI></UL><P><B>Methods:</B></P><UL><LI> 20Âµl colony-pcr mix for one clone: 2Âµl 10x Standard Taq-DNA-Polymerase Buffer; 0,4Âµl dNTPs (10mM), 0,4Âµl fp_qRT-CMV_rev(10ÂµM), 0,4Âµl ps_mcherry_rev_BamHI (10ÂµM); 0,1Âµl Taq-DNA-Polymerase; nuclease-free water add to 20Âµl
</LI></UL><UL><LI> colony-pcr mix were adapted to 26 clones
</LI></UL><UL><LI> after transferation of clone into 20Âµl colony-pcr mix, the rest of clone at the tip was plated on a LB-AMP-plate </LI></UL><UL><LI> LB-AMP-plate were incubated o.n. at 37Â°C</LI></UL><UL><LI> negative control: clon #24 from relagtion-control (ligation-ratio 1:0) from 2012-07-31
</LI></UL><UL><LI> program of colony-pcr: initial Denaturation:5Â´at 95Â°C; Denaturation: 25Â´Â´ at 95Â°C, Annealing: 25Â´Â´ at 51Â°C; Elongation: 114Â´Â´ (1845bp) at 68Â°C; final Elongation: 5Â´at 86Â°C; 30 cycles; store at 8Â°C
</LI></UL><P><B>further Taks:</B></P><UL><LI> gelelectrophoresis
</LI></UL><UL><LI> picking positiv clones for o.n.-culture
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> gelelectrophoresis of colony-pcr </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>AIM:</B> identifying correct ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Materials:</B></P><UL><LI> 1 100 ml 1%agarosegel with 20 slots
</LI></UL><UL><LI> 1 90 ml 1% agarosegel with 9 slots
</LI></UL><UL><LI> 1x TAE-buffer
</LI></UL><UL><LI> agarose
</LI></UL><P><B>Method:</B></P><UL><LI> 90 ml 1% agarosegel for 80 min at 110V
</LI></UL><UL><LI> 100 ml 1% agarosegel for 100 min at 110V </LI></UL><P><B>Results:</B></P><UL><LI> (positive) clones 4, 8, 12, 16 were chosen for o.n.-culture because of desired DNA-bond at appropriate 1845bp
</LI></UL><P><B>further Tasks:</B></P><UL><LI> o.n.-culture of positive clones
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> o.n.-culture of positive ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Investigators:</B> Kerstin/Stefan.</P><P><B>Materials:</B></P><UL><LI> 4x 15 ml Falcons
</LI></UL><UL><LI> LB-Medium
</LI></UL><UL><LI> Ampicilin
</LI></UL><UL><LI> bunsen burner
</LI></UL><P><B>further Tasks:</B></P><UL><LI> miniprep of o.n.-cultures
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-21_2"><P style="background-color: rgb(240, 20, 70);">2012-08-21</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> miniprep of o.n.-culture of positive ligated pcDNA5-FRT_scFv-TEV-TMD-EYFP clones </P><P><B>Investigators:</B> Kerstin/Stefan</P><P><B>Materials:</B></P><UL><LI> Miniprep-Kit
</LI></UL><P><B>Method:</B></P><UL><LI> according to manual
</LI></UL><P><B>Results: concentration measurement via nanodrop</B></P><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone4= 540 ng/Âµl
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone8= 663 ng/Âµl
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone12=572ng/Âµl
</LI></UL><UL><LI> pcDNA5-FRT_scFv-TEV-TMD-EYFP_clone16=580ng/Âµl
</LI></UL><P><B>further Tasks:</B></P><UL><LI> sequencing of the clones 4 and 8
</LI></UL><UL><LI> if they are correct ===&gt; stable transfection
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-25"><P style="background-color: rgb(240, 20, 70);">2012-08-25</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of clon3 pcdna5frt-scfv-tev-tmd-eyfp-construct with pcdna3.1-RP.1 primer </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from pcdna5frt-scfv-tev-temd-eyfp-construct clons 4,8
</LI><LI> Geneious</LI></UL><P><B>Results:</B></P><UL><LI>  clon4 with pcdna3.1-FP.1 shows accordance with theoretical pcdna5frt-scfv-tev-tmd-eyfp-construct </LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of clon4 for endotoxin free preparation </LI></UL><H3><SPAN class="mw-headline" id="2012-08-27_2"><P style="background-color: rgb(240, 20, 70);">2012-08-27</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Transformation of Clon4 (scFv425+TEV+TMD+YFP in pcDNA5/FRT) and Pog44 (Flp recombinase vector)</P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-27 5:00pm </P><P><B>Materials: </B>competent <I>E. coli</I> cells (XL1 blue), icebox, Bunsen burner, agar plate with ampicillin</P><P><B>Methods: </B>Transformation according to manual, 2Âµl of Clon4 and Pog44 were used, plate incubation started 6:30pm </P><P><B>Results: </B>colonies ready for picking and overnight culture </P><P><B>Further tasks: </B>Picking colonies</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Overnight culture of CMV-Venus</P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-27 5:00pm </P><P><B>Materials: </B>glycerol stock CMV-Venus, 50ml LB media with chloramphenicol</P><P><B>Methods: </B>inoculation of LB-chloramphenicol</P><P><B>Results: </B>overnight culture for endotoxin free DNA preparation</P><P><B>Further tasks: </B>endotoxin free preparation</P><H3><SPAN class="mw-headline" id="2012-08-28_2"><P style="background-color: rgb(240, 20, 70);">2012-08-28</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Endotoxin free preparation of CMV-venus </P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-28 11:00 am </P><P><B>Materials: </B>endotoxin free Mediprep kit, overnight culture CMV-Venus</P><P><B>Methods: </B>according to manual </P><P><B>Results: </B>CMV-Venus with --- ng/Âµl </P><P><B>Further tasks: </B>transient transfection of CHO cells</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Overnight culture Clon4 and Pog44 </P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-28 5:00 am </P><P><B>Materials: </B>colonies of Clon4 and Pog44, 5ml LB media with ampicillin</P><P><B>Methods: </B>inoculation of LB-ampicillin</P><P><B>Results: </B>overnight culture for endotoxin free DNA preparation</P><P><B>Further tasks: </B>endotoxin free preparation </P><H3><SPAN class="mw-headline" id="2012-08-27_3"><P style="background-color: rgb(240, 20, 70);">2012-08-27</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-08-29 11:00 am </P><P><B>Topic: </B> cell culture </P><P><B>Materials: </B> 6-well plates; </P><UL><LI> transfection of 6-well plate with different PEI amounts
</LI></UL><UL><LI><B>well</B><B>1)</B> 1 fold PEI; <B>2)</B> 1,5 fold PEI; <B>3)</B> 2 fold PEI; <B>4)</B> 2,5 fold PEI; <B>5)</B> 3 fold PEI; <B>6)</B> 3,5 fold PEI of DNA amount
</LI></UL><UL><LI> NOTE: due to not conclusive experiments, it was decided to keep the original protocol with 2,5 fold PEI
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight:bold;"> analyzing sequencing results of clon3 pcdna5frt-scfv-tev-tmd-eyfp-construct with pcdna3.1-RP.1 primer </P><P><B>Investigators:</B> Sascha<B>Materials:</B></P><UL><LI> sequencing results of GATC from pcdna5frt-scfv-tev-tmd-eyfp-construct clons 4,8
</LI><LI> Geneious</LI></UL><P><B>Results:</B></P><UL><LI>  clon4 with pcdna3.1-RP.1 shows accordance with theoretical pcdna5frt-scfv-tev-tmd-eyfp-construct </LI></UL><P><B>Further Tasks:</B></P><UL><LI> transformation of clon4 for endotoxin free preparation </LI></UL><H3><SPAN class="mw-headline" id="2012-08-29_2"><P style="background-color: rgb(240, 20, 70);">2012-08-29</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-08-29 11:00 am </P><P><B>Materials: </B>cell-culture flask (75cm^2); 6-well plates; ibidi-dishes</P><P><B>Topic: </B> cell culture; </P><UL><LI> thowing CHO cells, cultured w/o zeocin
</LI></UL><UL><LI> passaging (1:10) of CHOâs one flask w/ and one w/o zeocin
</LI></UL><UL><LI> seeding CHOâs in 6-well 2*10^5 cells/well
</LI></UL><UL><LI> seeding CHOâs in ibidi dishes 5*10^4 cells/well
</LI></UL><UL><LI> transfection of clon 4 (540ng/Âµl) (small construct) of CHOâs in ibidi dishes, 1Âµg(DNA):2,5Âµl(PEI) = 0,25Âµg DNA:0,625Âµl PEI
</LI></UL><UL><LI> transfection of 6-well plate with different DNA amounts
</LI></UL><UL><LI><B>well</B><B>1)</B> 1Âµg(0.5ÂµgmVenus+0,5Âµg AID) DNA; <B>2)</B> 1,5Âµg DNA; <B>3)</B> 2Âµg DNA; <B>4)</B> 3Âµg DNA; <B>5)</B> 4Âµg DNA; <B>6)</B> 5Âµg DNA with 2,5 fold PEI
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-31_2"><P style="background-color: rgb(240, 20, 70);">2012-08-31</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-08-31 09:00 am </P><P><B>Topic: </B>cell-culture </P><P><B>Methods: </B></P><UL><LI> passaging of thawed CHOâs w/ and w/o zeocin (of 29.08.2012 seeded cells)
</LI></UL><UL><LI> imaging (longtime imaging over 48 hours) of the transfected cells (30.08.2012) CHOâs in ibidi dishes with m-Venus+wild Type AID
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-30_2"><P style="background-color: rgb(240, 20, 70);">2012-08-30</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">cell culture; </P><P><B>Investigator:</B>Kerstin/Stefan</P><P><B>Time: </B>2012-08-30 09:00 am </P><P><B>Topic: </B>cell-culture </P><UL><LI> change of medium of thawed CHOâs w/ zeocin
</LI></UL><UL><LI> transfection of seeded (29.08.2012) CHOâs in ibidi dishes with m-Venus (250ng) and wild type AID (250ng)
</LI></UL><UL><LI> z-stack of clon 4(small construct)
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-29_3"><P style="background-color: rgb(240, 20, 70);">2012-08-29</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Endotoxin free preparation of Clon4 and Pog44</P><P><B>Investigator:</B>Rico, Tom, Chris</P><P><B>Time: </B>2012-08-29</P><P><B>Materials: </B>endotoxin free Mediprep kit, overnight culture Clon4, Pog44</P><P><B>Methods: </B>according to manual </P><P><B>Results: </B>Clon 4 with --- ng/Âµl, Pog44 with ----ng/Âµl </P><P><B>Further tasks: </B>stable transfection of CHO cells</P><H3><SPAN class="mw-headline" id="2012-08-30_3"><P style="background-color: rgb(240, 20, 70);">2012-08-30</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Primer design for Biobricks with RCF 25 standard</P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-30 6:30 am</P><P><B>Materials: </B>Geneious</P><P><B>Results: </B>Primer big construct
</P><UL><LI> Primer 1.1 and 1.2: Signal peptide-Nanobody-Fc
</LI></UL><UL><LI> Primer 2.1 and 2.2: TEV-LoxP-TMD-mcherry-LoxP
</LI></UL><UL><LI> Primer 3.1 and 3.2: Nanobody
</LI></UL><UL><LI> Primer 4.1 and 4.2(=1.2): Fc
</LI></UL><UL><LI> Primer5.1 and 5.2: TMD-mcherry
</LI></UL><P>small construct, mutagenese PCR
</P><UL><LI> Primer 1.RCF25 and 2.mut: RCF25 prefix and 1st mutation PstI in scFV
</LI></UL><UL><LI> Primer 3.mut and 4.mut: 1st and 2nd mutation PstI in scFV
</LI></UL><UL><LI> Primer 5.mut and &amp;.RCF25: 2nd mutation and RCF25 suffix
</LI></UL><P>Primer scFv: Primer 7.Rcf25 prefix and 8.RCF25 suffix</P><P><B>Further tasks: </B>PCR and seuqencing</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Sequence alignment of Geneart construct </P><P><B>Time: </B>2012-08-30 12 pm</P><P><B>Materials: </B>Geneious, sequences delivered by Geneart</P><P><B>Results: </B>sequence correct, no mutations, no undesired restriction sites</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Transformation of Geneart construct Nanobody-Fc-LoxP-TEV-TMD-mcherry </P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-30 17 pm</P><P><B>Materials: </B>competent <I>E. coli</I> cells (XL1 blue), icebox, Bunsen burner, agar plate with kanamycin </P><P><B>Method: </B>Transformation according to manual, 2Âµl of Geneart construct</P><P><B>Results: </B>colonies ready for picking and overnight culture </P><P><B>Further tasks: </B>Picking colonies </P><H3><SPAN class="mw-headline" id="2012-08-31_3"><P style="background-color: rgb(240, 20, 70);">2012-08-31</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Control of Primer and Ordering Primers </P><P><B>Investigator:</B>Sascha, Maria</P><P><B>Time: </B>2012-08-31 11 am</P><P><B>Materials: </B>Geneious</P><P><B>Results: </B>17 primer for Biobrick assembly with RCF25 standard</P><P><B>Further tasks: </B>PCR</P><P style="background-color: rgb(238, 221, 130); font-weight:bold;">Overnight culture Geneart construct</P><P><B>Investigator:</B>Maria</P><P><B>Time: </B>2012-08-31 5:00pm </P><P><B>Materials: </B>colonies Geneart construct, 5ml LB media with kanaymycin</P><P><B>Methods: </B>inoculation of LB-kanaymycin</P><P><B>Results: </B>overnight culture for mini prep</P><P><B>Further tasks: </B>Mini prep</P><H2><SPAN class="mw-headline" id="Virus">Virus</SPAN></H2><H3><SPAN class="mw-headline" id="2012-08-13_3"><P style="background-color: rgb(240, 20, 70);"> 2012-08-13</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR </P><P><B>Investigator:</B> Xenia </P><P><B>Aim:</B>test primer prr_VP2_pstI_2 (reverse primer)</P><P><B>Materials:</B></P><P>primer combination:
</P><UL><LI> prr_VP2_pstI_2 (reverse primer) and prf_XbaI_koz_GGGGG_VP2 (forward primer)-&gt; myc-
</LI></UL><UL><LI> prr_VP2_pstI_2 (reverse primer) and prf_XbaI_koz_GGGGG_myc_VP2(forward primer)--&gt; myc+</LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagenz</B></TD><TD><B>volumen [ÂµL]</B></TD></TR><TR><TD>HF buffer</TD><TD>5</TD></TR><TR><TD>dNTPs (NEB)</TD><TD>1.25</TD></TR><TR><TD>primer (prr_VP2_pstI_Temp68)</TD><TD>1.0</TD></TR><TR><TD>Primer (prf_XbaI_koz_GGGGG_VP2/prf_XbaI_koz_GGGGG_myc_VP2)</TD><TD>1.0</TD></TR><TR><TD> DNA (plasmid) </TD><TD> 1.0 </TD></TR><TR><TD> Phusion polymerase </TD><TD> 1.0 </TD></TR><TR><TD> water </TD><TD> 33.75 </TD></TR></TBODY></TABLE><P><B>program</B></P><TABLE border="2"><TBODY><TR><TD><B>step</B></TD><TD><B>temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 10 </TD><TD> 30 </TD></TR><TR><TD> annealing </TD><TD> 70 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> agarose gel electrophoresis
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-14_3"><P style="background-color: rgb(240, 20, 70);"> 2012-08-14</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR von VP2 </P><P><B>Investigators:</B> Tobias </P><P><B>Materials:</B></P><UL><LI> Phusion polymerase
</LI></UL><UL><LI> Templat: Plasmid (P10_pSB1C3_001_CMV_DARPin_ML_VP2/3_587KO_6xHis)
</LI></UL><UL><LI> primer:
</LI></UL><UL><LI> prr_VP2_pstI_2 (reversed) and prf_XbaI_koz_GGGGG_VP2 (FP1)-&gt; myc-
</LI></UL><UL><LI> prr_VP2_pstI_2 (reversed) and prf_XbaI_koz_GGGGG_myc_VP2(FP2)--&gt; myc+ </LI></UL><P><B>Method:</B></P><UL><LI> polymerase chain reaction </LI></UL><P><B>Mastermix</B></P><TABLE border="1"><TBODY><TR><TD><B>reagenz</B></TD><TD><B>volumen [ÂµL]</B></TD></TR><TR><TD>HF buffer</TD><TD>10</TD></TR><TR><TD>dNTPs (NEB)</TD><TD>1</TD></TR><TR><TD>primer (prr_VP2_pstI_2)</TD><TD>2.0</TD></TR><TR><TD>Primer (prf_XbaI_koz_GGGGG_VP2/prf_XbaI_koz_GGGGG_myc_VP2)</TD><TD>2.0</TD></TR><TR><TD> DNA (plasmid) </TD><TD> 1.0 </TD></TR><TR><TD> Phusion polymerase </TD><TD> 0.5 </TD></TR><TR><TD> water </TD><TD> 33.5 </TD></TR></TBODY></TABLE><P><B>Program</B></P><TABLE border="2"><TBODY><TR><TD><B>step</B></TD><TD><B>temperature [Â°C]</B></TD><TD><B>duration [s]</B></TD><TD><B>cycles</B></TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 30 </TD><TD>1 </TD></TR><TR><TD> denaturation </TD><TD> 98 </TD><TD> 10 </TD><TD> 30 </TD></TR><TR><TD> annealing </TD><TD> 65 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> elongation </TD><TD> 72 </TD><TD> 40 </TD><TD> 30 </TD></TR><TR><TD> final elongation </TD><TD> 72 </TD><TD> 300 </TD><TD> 1 </TD></TR><TR><TD> cooling </TD><TD> 4 </TD><TD> â </TD><TD> 1 </TD></TR></TBODY></TABLE><P><B>Further tasks:</B></P><UL><LI> PCR-clean up
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-15_3"><P style="background-color: rgb(240, 20, 70);"> 2012-08-15</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: PCR-clean up</P><P><B>Investigator:</B> Tobias </P><P><B>Materials:</B> NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel) </P><P><B>Methods:</B></P><UL><LI>CleanUp-protocol
</LI></UL><P><B>Results:</B></P><P>-myc-product: 18 ng/ÂµL --&gt; 14 nM
</P><P>+myc-product: 7 ng/ÂµL --&gt; 5 nM
</P><P><B>Further tasks:</B></P><UL><LI> digestion
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: ligation, transformation</P><P><B>Investigator:</B> Tobias </P><P><B>Materials and Methods: </B></P><UL><LI> the ligation and the transformation were done using the standard protocol
</LI></UL><P><B>Further tasks:</B></P><UL><LI> picking clones
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-17_3"><P style="background-color: rgb(240, 20, 70);"> 2012-08-17</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: pick clones, DNA preparation, gel electrophoresis</P><P><B>Investigators:</B> Kathi </P><P><B>Materials:</B></P><UL><LI> LB-medium
</LI></UL><UL><LI> GeneJET Plasmid Miniprep Kit
</LI></UL><UL><LI> gel electrophoresis
</LI></UL><P><B>Methods:</B></P><UL><LI> Pick clones, eight hours incubation (LB-medium, 37 Â°C)
</LI></UL><UL><LI> perform the DNA preparation
</LI></UL><UL><LI>control (agarose gel electrophorese)
</LI></UL><P><B>results:</B></P><UL><LI> p10_PsB1c3_CMV_kozak_sortase: c= 185 ng/ÂµL v= 50 ÂµL
</LI></UL><UL><LI> p10_PsB1c3_CMV_kozak_sortase_myc: c= 202 ng/ÂµL v= 50 ÂµL
</LI></UL><P><B>further tasks:</B> test digestion
</P><H3><SPAN class="mw-headline" id="2012-08-28_3"><P style="background-color: rgb(240, 20, 70);"> 2012-08-28</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> purification of plasmid DNA</P><P><B>Investigators:</B> Xenia and Kathi </P><P><B>Materials methods </B></P><UL><LI> endotoxin free preparation - Plasmid Plus Midi Kit (QIAGEN)- elution volume 200 ÂµL
</LI></UL><P><B>Results:</B></P><P>concentration of plasmids
</P><UL><LI> P01 (mVENUS linked on VP1): 1776.7 ng/ÂµL
</LI></UL><UL><LI> P05 (VP2 knock out)Â : 2336.2 ng/ÂµL
</LI></UL><UL><LI> P06 (VP1 knock out): 1256.7 ng/ÂµL
</LI></UL><UL><LI> phelper: 1628 ng/ÂµL
</LI></UL><UL><LI> psB1c (GOU=CFP): 493 ng/ÂµL
</LI></UL><P><B>Further tasks:</B></P><UL><LI> test digestion
</LI></UL><UL><LI> gel electrophoresis
</LI></UL><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> test digestion, Gelelectrophoresis</P><P><B>InvestigatorÂ :</B> Xenia </P><P><B>Materials and Method:</B></P><UL><LI> test digestion
</LI></UL><UL><LI> 15 ÂµL water + 2 ÂµL 10x FD green buffer + 2 ÂµL DNA (300 ng/ÂµL) + 1 ÂµL of each enzyme
</LI></UL><UL><LI> psB1c (GOI=CFP): SpeI, XbaI und ApaI
</LI></UL><UL><LI> phelper: NheI, SpeI, ApaI
</LI></UL><UL><LI> P06 (VP1 knock out): XbaI, ApaI
</LI></UL><UL><LI> P05 (VP2 knock out): XbaI, ApaI
</LI></UL><UL><LI> P01 (YFP linked on VP1): XbaI, ApaI
</LI></UL><UL><LI> 1 hour at 37 Â°C
</LI></UL><UL><LI> gel electrophoresis
</LI></UL><P><B>Results:</B></P><P>fragments calculated in Genious:
</P><UL><LI> psB1c (GOI=CFP): 4069 bp, 284 bp, 219 bp
</LI></UL><UL><LI> phelper: 6242 bp, 4011 bp, 906 bp, 492 bp
</LI></UL><UL><LI> P06 (VP1 knock out): 2119 bp, 290 bp, 631 bp, 2630 bp, 821 bp
</LI></UL><UL><LI> P05(VP2 knock out):Â : 2119 bp, 290 bp, 631 bp, 2630 bp, 821 bp
</LI></UL><UL><LI> P01(YFP linked on VP1): 2566 bp, 2169 bp, 821 bp, 323 bp
</LI></UL><P>fragments - gel electrophoresis
</P><UL><LI> phelper: 6242 bp, 4011 bp, 906 bp, 492 bp
</LI></UL><UL><LI> P06 (VP1 knock out): 2119 bp, 290 bp, 631 bp, 2630 bp, 821 bp + ~ 3000 bp
</LI></UL><UL><LI> P05(VP2 knock out):Â : 2119 bp, 290 bp, 631 bp, 2630 bp, 821 bp + ~ 3000 bp
</LI></UL><UL><LI> P01(YFP linked on VP1): 2566 bp, 2169 bp, 821 bp, 323 bp
</LI></UL><UL><LI> psb1c3 (GOI=CFF): other fragments in the gel electrophoresis than calculated in Genious
</LI></UL><P><B>Further Tasks:</B></P><UL><LI> sequences of the psb1c3 (GOI=CFP)-plasmid
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-29_4"><P style="background-color: rgb(240, 20, 70);"> 2012-08-29</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: transfection of plasmids</P><P><B>Investigators:</B> Kerstin and Stefan </P><P><B>Aim:</B></P><UL><LI>The plasmids were transfected in HEK-cells with the following plasmids:
</LI></UL><UL><LI> pSB1C: GOI is CFP
</LI></UL><UL><LI> phelper
</LI></UL><UL><LI> P01: YFP linked on VP1
</LI></UL><UL><LI> p06: VP1 knock out</LI></UL><P><B> Materials:</B></P><UL><LI> Medium: opti Mem
</LI></UL><UL><LI> transfection reagenz: PEI
</LI></UL><UL><LI> 15 Âµg DNA per each transfection
</LI></UL><P><B> Methods:</B></P><UL><LI> transfection preparation
</LI></UL><UL><LI> 1875 ng per plasmid; add 200 ÂµL opti MEM-medium per each transfection
</LI></UL><UL><LI> 4.8 * DNA --&gt; 72 ÂµL PEI add 200 ÂµL per each transfection
</LI></UL><UL><LI> three times 10 s mixing
</LI></UL><UL><LI> 2 min RT
</LI></UL><UL><LI> centrifagation
</LI></UL><UL><LI> give the PEI preparation to the DNA preparation
</LI></UL><UL><LI> mixing
</LI></UL><UL><LI> 15 min RT
</LI></UL><UL><LI> give the transfection preparation to the HEK AAV 293 cells
</LI></UL><P><B>Further tasks</B></P><UL><LI> change medium (DMEM) after 16 hours
</LI></UL><UL><LI> after three days virus maturation, preparation of the primary virus stocks
</LI></UL><H3><SPAN class="mw-headline" id="2012-08-30_4"><P style="background-color: rgb(240, 20, 70);"> 2012-08-30</P></SPAN></H3><P style="background-color: rgb(238, 221, 130); font-weight: bold;"> Topic: change medium</P><P><B>Investigators:</B> Kerstin and Stefan </P><P><B>Aim: </B> change the medium of the transfected cells </P><P><B> Materials: </B></P><UL><LI> PBS
</LI></UL><UL><LI> DMEM
</LI></UL><P><B> Methods: </B></P><UL><LI> aspirate the old medium
</LI></UL><UL><LI> washed with PBS
</LI></UL><UL><LI> give new DMEM medium to the cells
</LI></UL><P><B>Further tasks</B></P><UL><LI> after three days virus maturation - virus preparation on 02.09.2012
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal/August">http://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal/August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Lab/Labjournal/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Lab/Labjournal/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Lab/Labjournal/August&amp;oldid=221755" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>