<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Biobricks_All_Biobricks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Biobricks/All_Biobricks">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Biobricks/All_Biobricks&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Biobricks/All_Biobricks&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Biobricks/All_Biobricks&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Biobricks/All_Biobricks" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Biobricks/All Biobricks</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="nav_menu gradient_golden"><DIV id="menu_center"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware">Home</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">Project</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance#Main_Results">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">Mutation Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody">Antibody Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus">Selection Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam Standard">Potsdam Standard</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Collaboration">Collaboration</A></LI></UL><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Sponsoring">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Attribution">Attribution</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Lab</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Group_Meetings">Group Meetings</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">All Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">SocialBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_Politics">Science Meets Politics</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_People">Science Meets People</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Software">Software</A></LI></DIV></DIV><DIV class="box_round gradient_grey"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#All_BioBricks"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">All BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Antibody_BioBricks"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Antibody BioBricks</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Characterization:"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Characterization:</SPAN></A></LI><LI class="toclevel-4 tocsection-4"><A href="#Expression_of_Smaller_Construct_in_CHO_Cells"><SPAN class="tocnumber">1.1.1.1</SPAN><SPAN class="toctext">Expression of Smaller Construct in CHO Cells</SPAN></A></LI><LI class="toclevel-4 tocsection-5"><A href="#Expression_of_Advanced_Construct_in_CHO_Cells"><SPAN class="tocnumber">1.1.1.2</SPAN><SPAN class="toctext">Expression of Advanced Construct in CHO Cells</SPAN></A></LI><LI class="toclevel-4 tocsection-6"><A href="#Functionality_of_Applied_Cre.2FLoxP_System"><SPAN class="tocnumber">1.1.1.3</SPAN><SPAN class="toctext">Functionality of Applied Cre/LoxP System</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Mutation_BioBricks"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mutation BioBricks</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Characterization:_2"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Characterization:</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Selection_BioBricks"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Selection BioBricks</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Characterization:_3"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Characterization:</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Potsdam_Standard_BioBricks"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Potsdam Standard BioBricks</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Characterization:_4"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Characterization:</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="All_BioBricks">All BioBricks</SPAN></H2><P>As the project is divided into three  subproject, the resulting biobricks are listet &amp; characterized in the three categories. Furthermore, the Biobricks for the new RFC &quot;Potsdam Standard&quot; are listed.<B>For detailed information on parts design, sequence, composites click the BioBrick numbers</B></P><H3><SPAN class="mw-headline" id="Antibody_BioBricks">Antibody BioBricks</SPAN></H3><P>All antibody module BioBricks for the smaller construct were designed out of already existing parts and BioBricks and were expanded with further functional units by assembly PCR. The advanced antibody construct was derived by de novo synthesis based on sequence information provided by GenBank and UniProt and by one already existing Biobrick part. The gene synthesis construct was optimized for expression in CHO cell and used as template for all subsequently derived BioBricks by PCR.
</P><TABLE border="1"><TBODY><TR><TD><A href="http://partsregistry.org/Part:BBa_K929100" class="external text" rel="nofollow">BBa_K929100</A> Anti-EGFR scFv425 </TD><TD><P>The single chain fragment (scFV425) specifically binds to the epidermal growth factor receptor (EGFR) domain 3. The part has a shortened N-terminal FLAG-tag sequence of five amino acids (DYKDE) for purification and detection.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929101" class="external text" rel="nofollow">BBa_K929101</A> Anti-EGFR scFv425 with TEV site, transmembrane region an eYFP </TD><TD><P>The BioBrick BBa_K929101 is an extended version of the BioBrick BBa_K929100 and is composed of the already existing human EGFR (ErbB-1) signal peptide (BBa_K157001), the scFv anti-EGFR-domain-3, a TEV recognition site (according to life technologies AcTEV TM Protease manual), the transmembrane domain: transmembrane domain of B-cell receptor with glycine-serine linker (BBa_K157010) and the enhanced YFP ( BBa_E0030) reporter. The composite part was derived by PCR assembly.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929102" class="external text" rel="nofollow">BBa_K929102</A> Anti-GFP Nanobody with IgG kappa signal peptide and IgG1 Fc domain </TD><TD><P>This part represents an antibody unit composed of a IgG kappa signal peptide (UniProt: P01601), an anti-GFP nanobody (PDB: 3OGO) and an IgG1 Fc domain (UniProt: P01857).
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929103" class="external text" rel="nofollow">BBa_K929103</A> B-cell transmembrane region with mCherry and TEV site, framed by LoxP</TD><TD><P>This part represents the switchable membrane anchoring region of or antibody construct (BBa_K929107) and consists of the TEV recognition site, the modified transmembrane domain of B-cell receptor with a shortened linker (modified BBa_K157010), the mCherry reporter and two framing LoxP sites.  
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929104" class="external text" rel="nofollow">BBa_K929104</A> Anti-GFP Nanobody </TD><TD><P>The green fluorescent protein (GFP)-nanobody is a single-chain VHH antibody domain which shows specific binding activity against GFP (PDB: 3OGO). 
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929105" class="external text" rel="nofollow">BBa_K929105</A> IgG1 Fc domain </TD><TD><P>A human immunoglobulin gamma-1 heavy chain constant region (IGHG1) was added to the construct and enables the direct interaction of the antibody unit with the Fc receptor and complement proteins (UniProt: P01857). 
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929106" class="external text" rel="nofollow">BBa_K929106</A> B-cell transmembrane region with mCherry</TD><TD><P>This part is a composed part of the transmembrane domain of a B-cell receptor (modified BBa_K157010)fused to the mCherry reporter for detection of cellular localization.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929107" class="external text" rel="nofollow">BBa_K929107</A> Anti-GFP, IgG1 Fc with TEV site, TMR, mCherry, framed by LoxP</TD><TD><P>The antibody construct consists of two major building blocks represented by the actual antibody unit ( BBa_K929102) and the switchable membrane anchoring region (BBa_K929103). The antibody unit is represented by the human Ig kappa chain V-I region signal peptide (UniProt: P01601), the anti-GFP Nanobody (PDB: 3OGO) and the Fc region (UniProt: P01857). TEV protease recognition site, 2 LoxP sites, the B-cell receptor transmembrane domain (modified BBa_K157010) and the mCherry reporter display the switchable membrane anchoring region. 
</P></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Characterization:">Characterization:</SPAN></H4><H5><SPAN class="mw-headline" id="Expression_of_Smaller_Construct_in_CHO_Cells">Expression of Smaller Construct in CHO Cells</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 1:</B> eYFP fluorescence of the small construct in stably transfected CHO cells</DIV></DIV></DIV><P>The small antibody construct was designed with a eYFP reporter to check the a) expression level of the construct, its b) cellular localization and to enable the selection of the transfected cells via FACS.
</P><P>The fluorescence microscopy image (Figure 1) shows the transfected CHO cells with the smaller antibody construct containing the scFV425 (BBa_K929101). It can be proven that the construct is expressed in CHO cells and also shows a high fluorescent signal. Its accumulation in the vesicular compartment of the golgi apparatus implements a membranous localization. FACS selection was successfully realized.</P><H5><SPAN class="mw-headline" id="Expression_of_Advanced_Construct_in_CHO_Cells">Expression of Advanced Construct in CHO Cells</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 2:</B> mCherry fluorescence of advanced antibody construct showing membrane localisation on CHO cells</DIV></DIV></DIV> The advanced antibody construct was designed with a mCherry reporter to check the a) expression level of the construct, its b) cellular localization and to enable the selection of the transfected cells via FACS. 
<P>The fluorescence microscopy image (Figure 2) shows the transfected CHO cells with the advanced antibody construct containing the anti-GFP nanobody (PDB: 3OGO). The images indicate that the construct is expressed in CHO cells and shows a high fluorescent signal sufficient for detection. With fluorescence microscopy, confocal microscopy, immunfluorescence and FACS we were able to show the expression of the transfected parts. Under certain conditions we have seen a membrane localization of the advanced construct (Figure 2) yet we have not been able to generate cells that transport enough molecules to their surface for successful detection.</P><H5><SPAN class="mw-headline" id="Functionality_of_Applied_Cre.2FLoxP_System">Functionality of Applied Cre/LoxP System</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 3:</B> Western Blot of purified Nanobody-Fc from supernatant of cells co-transfected with Cre recombinase and advanced construct</DIV></DIV></DIV><P>As an additional functional unit the advanced construct contains two LoxP sites which flank the membrane anchoring region, thus allowing the switch from surface presentation to secretion on genetically level.
</P><P>By co-transfecting CHO cells with our advanced antibody construct and the cre recombinase (pBS185) we were able to proof the functionality of the applied cre/LoxP system. The immune blot image (Figure 3) shows purified Nanobody-Fc from supernatant obtained from co-transfected CHO cells colored with anti-human Fc coupled to streptavidin. Purification was carried out with magnetic beads. A correct band can be seen at 41kDa.</P><H3><SPAN class="mw-headline" id="Mutation_BioBricks">Mutation BioBricks</SPAN></H3><P>All the Mutation BioBricks are improved/modified versions of the wildtype AID that was available as a BioBrick (<A href="http://partsregistry.org/Part:BBa_K103001" class="external text" rel="nofollow">BBa_K103001</A>). AID is known to be responsible for somatic hypermutation and the class-switch recombination of immunoglobulin in B cells. This enzyme of 28 kDa originally occurs in B cells but does also show activity after transfection into CHO cells. AID induces the deamination of cytidine to uridine at actively transcribed single strand DNA. The replacement of cytidine by uridine leads to a mismatch during DNA replication and integrates a single base substitution predominantly in the immunoglobulin genes.
Therefore this enzymes was used in our project for diversification of antibodies.With PCR and cloning we developed AID`s for the differnt requirements. 
</P><TABLE border="1"><TBODY><TR><TD><A href="http://partsregistry.org/Part:BBa_K929000" class="external text" rel="nofollow">BBa_K929000</A> AID with CMV promoter and hGH-polyadenylation signal sequence</TD><TD><P>CMV-promoter &amp; hGH polyadenylationsequence were added to the wildtype AID for strong expression.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929001" class="external text" rel="nofollow">BBa_K929001</A> modified_AID without NES, with NLS and Kozak sequence</TD><TD><P>To improve the mutation rate the AID was located into the nucleus, because there it mutates transcribed DNA. Therefor an aditional NLS(Nuclear Localization Sequence)was added and the naturally occurring NES(Nuclear Exprot Sequence) was removed.For stronger Expression an Kozak Sequence was added. To fuse modified AID((C-terminal of mod. AID) with <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> parts, we added an AgeI restriction site.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929002" class="external text" rel="nofollow">BBa_K929002</A> modified_AID with CMV and hGH-polyA</TD><TD><P>For strong expression of the previously described part an CMV-promoter and hGH polyadenylation sequence were added.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929004" class="external text" rel="nofollow">BBa_K929004</A> modified_AID+eGFP</TD><TD><P>The modified AID was fused with eGFP to check 1.)the tranfection success 2.)the cellular lokalization of the improved AID and 3.) to select transfected cells via FACS.
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929003" class="external text" rel="nofollow">BBa_K929003</A>modified AID with CMV, hGH-polyA and eGFP</TD><TD><P>For strong expression of the previously described part an CMV-promoter and hGH polyadenylation sequence were added.
</P></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Characterization:_2">Characterization:</SPAN></H4><P><B>Mutation rate</B> in CHO-cells:</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4:</B> Comparison of the mutation rates between the wildtype AID, modified AID and modified AID-eGFP</DIV></DIV></DIV><P>We checked the mutation rate of wildtype AID(<A href="http://partsregistry.org/part:BBa_K929000" class="external text" rel="nofollow">BBa_K929000</A>), modified AID(<A href="http://partsregistry.org/part:BBa_K929002" class="external text" rel="nofollow">BBa_K929002</A>) and modified AID+eGFP(<A href="http://partsregistry.org/part:BBa_K929003" class="external text" rel="nofollow">BBa_K929003</A>) (all expressed with CMV-promoter and hGHpolyA). Therefor we cotransfected CHO cells with a single chain construct and one of the AID versions. After certain expression time we purified the the single chain plasmids and transformed them into <I>E. coli</I> to enrich the mutated plasmids. After overnight culture and purification of the transformed plasmids, samples where sequenced. 
There is a significant higher mutation rate when wildtype AID, modified AID or modified AID-eGFP is added,  compared to the same experiment without AID. Surprisingly, the mutation rate of wt AID is two times higher than the mutation rate of the modified AID or modified AID-eGFP (Figure 4). This observation is contrary to our expectation. The modified AID+eGFP is located in the nucleus (Figure 5), that means our construct works and should have a higher mutation rate. One possible explanation is that the modified AID has a very high mutation rate and therefore the transfected cells die or inactivate the plasmid like it was observed for the wildtype AID (Martin and Scharff (2002)).</P><P><B>Green fluorescence reporter</B>:
The modified AID was fused with eGFP to check 1.)the tranfection success 2.)the cellular lokalization of the improved AID and 3.) to select transfected cells via FACS. <B> Transfection success and cellular localization: </B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 5:</B>  CHO cells were transfected  A)with &quot;modified AID+eGFP&quot; (BBa_K929003)and B)with &quot;modified AID+eGFP&quot; (BBa_K929003) and a mCherry labeled nanobody construct (BBa_K929107)</DIV></DIV></DIV>.
<P>The fluorescence microscope images(Figure 5) show transfected cells A)with &quot;modified AID+eGFP&quot; (BBa_K929003)and B)with &quot;modified AID+eGFP&quot; (BBa_K929003) and a mCherry labeled nanobody construct (BBa_K929107)]]. &quot;Modified AID+eGFP&quot; (BBa_K929003)proved to be located in the nucleus. This shows that the deletion of the NES(Nuclear Export Sequence) and the addition of an NLS(Nuclear Localization Sequence) leads to an accumulation in the nucleus as it was planed. Therefore we would expect an increased mutation rate with &quot;modified AID+eGFP&quot; but as described above the experimental setup possible can not reflect this. It is also shown that transfected cells(green nucleus, Figure 5 A&amp;B) can be discriminated from not transfected (grey, Figure 5). This can be used for a fast selection of transfected cells via FACS. Figure 5 B shows that &quot;modified AID+eGFP&quot; can be used in combination with an other fluorescence labeld construct (mCherry labeled nanobody construct ,BBa_K929107) and co transfected cells can be easily found or selected via FACS. </P><H3><SPAN class="mw-headline" id="Selection_BioBricks">Selection BioBricks</SPAN></H3><TABLE border="1"><TBODY><TR><TD><A href="http://partsregistry.org/Part:BBa_K929200" class="external text" rel="nofollow">BBa_K929200</A> N-terminal sortase motif linked to AAV2-VP2 with Myc-tag, kozak and CMV </TD><TD><P>N-terminal sortase motif is required for being ligate to several Proteins with the C-terminal sortase motif. The sortase motif was fused to the N-terminus of VP2 cap-gene, which present the fused proteins on the surface of the virus and is not essential for viral infectivity. For stronger Expression an Kozak Sequence was added. The CMV promotor is used for a high expression of the transgene. Myc-tag is used to recognize that the sortase motif is on the surface of the virus
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929201" class="external text" rel="nofollow">BBa_K929201</A> N-terminal sortase motif linked to AAV2-VP2 with kozak and CMV</TD><TD><P>The part exists of the same parts like BBa_K929200, but it lacks the myc-tag
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929202" class="external text" rel="nofollow">BBa_K929202</A> EGF-Receptor_3rd domain with PelB leadersequence, His-tag and C-terminal Sortase motif</TD><TD><P>The C-terminal sortase motif is recognized by the sortase and is cleaved between Gly and Thr. The 3rd domain of EGF-Receptor was used to target the adeno-associated virus to a high affine anti-EGFR antibody.PelB leader directs the expressed EGFR to the <I>E. coli</I> periplasm and the his-tag is used to purify the expressed Protein EGFR in <I>E. coli</I>. 
</P></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Characterization:_3">Characterization:</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption"><B>Figure 6:</B> expression of the 3rd EGFR domain in <I>E. coli</I> detection was performed using SDS-PAGE and Western-blot</DIV></DIV></DIV><P>Using the Biobricks with the Sortase motif fused to the VP2/3 region <A href="http://partsregistry.org/Part:BBa_K929200" class="external text" rel="nofollow">BBa_K929200</A>; <A href="http://partsregistry.org/Part:BBa_K929200" class="external text" rel="nofollow">BBa_K929200</A> it is easily to ligate any proteins containing the C-terminal sortase motif with the enzyme sortase. We were interested to bring EGFR on the surface of the virus. Therefore we expressed the 3rd EGFR domain with the C-terminal sortase motif <A href="http://partsregistry.org/Part:BBa_K929202" class="external text" rel="nofollow">BBa_K929202</A> in <I>E. coli</I>. We detect successful expression of EGFR on western blot (Figure 6)
</P><H3><SPAN class="mw-headline" id="Potsdam_Standard_BioBricks">Potsdam Standard BioBricks</SPAN></H3><P>The Potsdam Assembly Standard is a modified PLICing (Phosphorothioate-based ligase-independent gene cloning) method (Blanusa <I>et al.</I> (2010), Liu XP <I>et al.</I> (2010)). The original PLICing is based on the amplification of insert and backbone to ligate them together by knocking out thiophosphates at the 3' end. For the Potsdam Standard, we designed a new cloning vector with an RFP expression cassette as insert and two new restriction enzyme recognition sites in the suffix and prefix in pSB1C3. In the prefix we added the Apa I recognition site and in the suffix the Sph I recognition site. Both enzymes causing a 3’ overhang with 4 nucleotides. For the cloning process we cut the vector with Apa I and Sph I. In this assembly standard that’s the only step where we have to use restriction enzymes.
The insert was amplified with primers that contain 4 phosphothioate nucleotides and the recognition sites for Apa I and Sph I at the 5’ end. After incubation in an iodine/ethanol solution the thiophosphates were cut out resulting in a 3’ overhang which is suitable to the overhangs that was created by cutting the new assembly vector.
After that the digested vector and the PLICed insert were mixed and transformed into <I>E. coli</I>. By using the RFP expression cassette we created a ligation control system. Due to the fact that red fluorescent colonies have a failed vector ligation we can tell which colonies are correctly ligated. The colonies that do not show red fluorescence are the positive clones.
</P><TABLE border="1"><TBODY><TR><TD><A href="http://partsregistry.org/Part:BBa_K929301" class="external text" rel="nofollow">BBa_K929301</A> Backbone of the Potsdam Standard </TD><TD><P>backbone contains two new restriction sites: Apa I and Sph I and a RFP expression cassette as ligation control
</P></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_K929302" class="external text" rel="nofollow">BBa_K929302</A> AID in Potsdam Standard</TD><TD><P>AID in new Potsdam Standard vector with inactivated Apa I and Sph I, but active <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A> restriction sites 
</P></TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Characterization:_4">Characterization:</SPAN></H4><P>
We tested different combination to prove the most efficient experimental design. Therefore, we tried four different conditions: the ligation of the pliced GOI with the unpurified digested backbone with and without ligase and the ligation of the pliced GOI with the purified digested backbone with and without ligase. After transformation, we picked only clones which have lost their red fluorescence indicating that the ligation was successful (Figure 9).
To quantify the results, we counted the red colonies and the colonies without red fluorescence (Figure 10). It is obvious, that the ligation with the electrophoretic purified digested backbone is more efficient than without any purification. Nevertheless, the use of ligase is not necessary for the ligation process under these conditions.</P><TABLE class="table center" border="0"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7:</B> Overnight culture of transformed <I>E. coli</I></DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 8:</B> After centrifugation, the cell pellets are coloured like cherries</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 9:</B> Results of different ligation conditions</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 10:</B> Results of different ligation conditions</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">http://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Biobricks/All_Biobricks" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Biobricks/All_Biobricks" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Biobricks/All_Biobricks&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Biobricks/All_Biobricks&amp;oldid=286815" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>