<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Project_Part_AID"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Project/Part_AID">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Project/Part_AID&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Part_AID&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Part_AID&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Project/Part_AID" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Project/Part AID</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="nav_menu gradient_golden"><DIV id="menu_center"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware">Home</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">Project</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance#Main_Results">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">Mutation Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody">Antibody Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus">Selection Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam Standard">Potsdam Standard</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Collaboration">Collaboration</A></LI></UL><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Sponsoring">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Attribution">Attribution</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Lab</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Group_Meetings">Group Meetings</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">All Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">SocialBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_Politics">Science Meets Politics</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_People">Science Meets People</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Software">Software</A></LI></DIV></DIV><DIV class="box_round gradient_grey"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Mutation_Module"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Mutation Module</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Abstract"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Introduction"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Structure_of_the_AID"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Structure of the AID</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Substrate_specificity_of_the_AID"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Substrate specificity of the AID</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#AID_as_a_key_component_of_the_Antibody_generating_system"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">AID as a key component of the Antibody generating system</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Testing_AID_mutation_rate_in_E.coli_with_phage_display"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Testing AID mutation rate in E.coli with phage display</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Cloning_of_the_wildtype.2C_modified_AID_and_modified_AID_fused_with_eGFP"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Cloning of the wildtype, modified AID and modified AID fused with eGFP</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Intracellular_localization_of_the_modified_AID"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Intracellular localization of the modified AID</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Co-transfection_of_modified_AID_and_antibody_construct"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Co-transfection of modified AID and antibody construct</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Comparison_of_mutation_rates_of_modified_and_wildtype_AID"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Comparison of mutation rates of modified and wildtype AID</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Mutation_rate_in_E._coli_measured_via_Phage_display"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Mutation rate in E. coli measured via Phage display</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Discussion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Cloning_of_wt_AID.2C_modified_AID_and_modified_AID_with_eGFP"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Cloning of wt AID, modified AID and modified AID with eGFP</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Comparison_of_the_mutation_rates_of_wt_AID_and_modified_AID_in_connection_with_the_intracellular_localization"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Comparison of the mutation rates of wt AID and modified AID in connection with the intracellular localization</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Mutation_Module">Mutation Module</SPAN></H3><H4><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H4><P>Activation Induced Cytidine Deaminase (AID) is one of the key enzymes of antibody maturation in immune system of mammalian organisms. We used the AID to mutate the antibody sequences in CHO cells and in <I>E. coli</I> during Phage Display. In prior publications, it was shown that the AID is active in CHO cells and <I>E. coli</I> by using transfection or transformation, respectively.
It was shown, that the AID mutates single stranded DNA. That is the reason why there is a correlation between mutation rate and transcription rate, when DNA is single stranded. Therefore, the AID has to be localized in the nucleus to mutate single stranded DNA. In the sequence of the AID, a N-terminal, non-functional NLS and a C-terminal NES was found restraining the ability to diffuse into the nucleus and, therefore, to mutate. We suggested, that a modified form of the AID, with NLS and without NES, would mutate more efficiently.
Therefore, we constructed both, the wildtype AID (wt AID) with a CMV promoter and a hGH polyA tail for expression in eukaryotic cells and the modified AID with NLS, without NES fused with eGFP.
After transfection in CHO cells, we observed a high concentration of the fusion protein in the nucleus indicating that the NLS is functional and that the modified AID is localized in the nucleus.
For determining the mutation rate, we co-transfected CHO cells with wildtype AID, modified AID and antibody construct, lysed them after 1, 2 and 3 days of incubation and purified the antibody constructs. After that, we transformed the antibody constructs into <I>E. coli</I>, purified and sequenced them.
The results show that the mutation rate of wt AID is three times higher than the control group without AID. Surprisingly, the mutation rate of modified AID is only two times higher than the control group. We suppose, that the mutation rate of the modified AID is much higher than of wt AID. Meaning that the transfected cells die or inactivate the modified AID more efficiently and decrease the mutation rate.
</P><H4><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H4><P>The activation-induced cytidine deaminase (AID) is one of the key enzymes of mammalian immune system and a central controlling point for the somatic hypermutation (SHM) (Muramatsu <I>et al.</I> (2000), Revy <I>et al.</I> (2000)), class switch recombination (Muramatsu <I>et al.</I> (2000)) and gene conversion (Arakawa <I>et al.</I> (2002), Harris <I>et al.</I> (2002)) (figure 1). The AID is expressed in B-lymphocytes and active during B-lymphocyte activation. Because of sequence similarities to APOBEC (apolipoprotein B messenger RNA-editing enzyme catalytic polypeptide) enzymes many functional and structural aspects of the AID were postulated (Prochnow <I>et al.</I> 2007). Today we know that the AID catalyzes the deamination of cytidine to uracil on the DNA level. A deficiency of AID or mutations within the coding sequence causes a defect of antibody maturation, class switch recombination and gene conversion.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1:</B> Role of the AID in immunoglobulin gene diversification (Vallur <I>et al.</I> (2007))</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Structure_of_the_AID"> Structure of the AID </SPAN></H5><P>The AID is a 28 kDa small protein with a highly conserved structure. The three-dimensional structure is not clear up until today. But because of structural homologies to the APOBEC enzymes some structural aspects were suggested (Prochnow <I>et al.</I> (2007)) (figure 2). However, it is known that the AID has a non-functional N-terminal nuclear location sequence (NLS) and a C-terminal nuclear export sequence (NES). In the middle of the primary sequence the cytidine motif and the APOBEC-like motif is localized (figure 3). Because of simulations it was suggested that the AID is a dimer and every subunit binds to the DNA.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:182px;"><A href="/File:AID_homology.gif" title="File:AID homology.gif">File:AID homology.gif</A><DIV class="thumbcaption"><B>Figure 2:</B> AID homologue structure (computed with <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Http://toolkit.tuebingen.mpg.de/hhpred" class="new" title="Http://toolkit.tuebingen.mpg.de/hhpred">HHPred</A>)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3:</B> primary structure of AID (Vallur <I>et al.</I> (2007))</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Substrate_specificity_of_the_AID"> Substrate specificity of the AID </SPAN></H5><P>It was shown that the AID mutates DNA instead of RNA in case of APOBEC enzymes. The substrate of the AID is the single stranded DNA. Therefore, a correlation between the mutation rate and the transcription rate existed. This means that the greater the transcription activity, the higher the mutation rate. That is the reason why the AID has to be localized in the nucleus of eukaryotic cells to mutate the DNA sequence. Because of the NES the enzyme is actively exported. Nevertheless, the small size of the AID allows it to diffuse through the nuclear pore complexes into the nucleus.
The somatic hypermutation (SHM) should take place on the variable regions (V-regions) of the antibodies; the binding sites to the antigen. Therefore, the AID should bind specific to the V-regions and catalysis the mutation. However, the SHM was also observed in the proto-oncogenes <I>bcl-6</I> and <I>FasL</I> albeit with a much lower mutation rate (Pasqualucci <I>et al.</I> (1998, 2001), Shen <I>et al.</I> (1998), Muschen <I>et al.</I> (2000)). Consequently, it was suggested that the AID has no direct substrate specificity and may interact with some cis acting factors to ensure a significantly greater mutation rate in the V-regions than in other genes. After sequence analysis of the mutated target sequences it was found that the AID mutates predominantly small sequence motifs, so called hotspots.</P><H5><SPAN class="mw-headline" id="AID_as_a_key_component_of_the_Antibody_generating_system">  AID as a key component of the Antibody generating system </SPAN></H5><P>In a prior publication (Bowers <I>et al.</I>, 2001) it was shown that the AID can be used to maturate antibody sequences to produce high affine binding molecules in CHO cells. In our project we also use the AID to mutate the antibody sequences in CHO cells (figure 4). Therefore, we cloned the AID BioBrick (BBa_K103001) with the CMV promoter and the hGH-polyA tail together to transiently transfect the CHO cells. We expected that the mutation rate is depending on the predominant location of this enzyme in the cell. Therefore, we concluded that the mutation rate increases with the increasing resting time of the AID in the nucleus. In order to test it, we knocked out the C-terminal NES and added a functional NLS to the N-terminus to ensure a greater resting time in the nucleus. To investigate whether the modifications are functional we fused the modified AID with the eGFP (BBa_K929003).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:922px;"><DIV class="thumbcaption"><B>Figure 4:</B> Mutation module: The enzyme AID mutates the antibody gene -&gt; maturation</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Testing_AID_mutation_rate_in_E.coli_with_phage_display">  Testing AID mutation rate in <I>E.coli</I> with phage display  </SPAN></H5><P>In order to characterize the AID and its potential to mutate antibody fragments, phage display is used. This powerful selection tool tests for proteins or peptides from a large recombinant library with specific binding abilities for a defined target. A gene of interest is introduced into the bacteriophage genome and is presented on the surface of the phage allowing the selection for the phenotype closely coupled to the genotype. 
We transformed simultaneously <I>E.coli</I> cells (strain ER2738) with an arabinose inducible AID enzyme and a single chain antibody fragment EGFR-C on the phagemid vector pAK100. By allowing the enzyme to mutate the antibody fragment over a certain time we are generating a diverse library of antibody sequences. After the infection of <I>E.coli</I> cells with the helper phage, bacteriophages are produced and the mutated antibody sequence is randomly incorporated into their genome. Subsequently, <I>E.coli</I> cells were infected with the phages containing mutated antibody fragments. To observe the mutation rate of the antibody fragment, after three cycles of infection and phage production in presence of AID, multiple phagemids were sequenced.</P><H4><SPAN class="mw-headline" id="Results"> Results </SPAN></H4><H5><SPAN class="mw-headline" id="Cloning_of_the_wildtype.2C_modified_AID_and_modified_AID_fused_with_eGFP"> Cloning of the wildtype, modified AID and modified AID fused with eGFP </SPAN></H5><P>For cloning the composite part wildtype AID (wt AID) we used the BioBrick BBa_K103001. To ensure the expression in eukaryotic cells we cloned the wt AID together with the CMV promoter and the polyA tail from the hGH gene.
The modified AID was amplified with primers which contain the Kozak sequence and the NLS, and which delete the present NES in the sequence. The Kozak sequence combined with the CMV promoter ensures a greater expression than the CMV promoter alone. The NLS directs the protein into the nucleus. This is supported by the knock out of the NES. For detecting the expression of the modified AID we fused this protein with eGFP.
To detect the expression we measured the time dependent fluorescence in order to compute the expression rates.
</P><H5><SPAN class="mw-headline" id="Intracellular_localization_of_the_modified_AID"> Intracellular localization of the modified AID </SPAN></H5><P>With fluorescence microscopy we detected the eGFP fluorescence of the modified AID-eGFP fusion protein. In the following picture, we can see that the modified AID is actively transported into the nucleus where a great eGFP fluorescence occurs (figure 5). That means the NLS is functional and able to recruit the factors necessary for import into the nucleus.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5:</B> Intracellular location of modified AID with eGFP</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Co-transfection_of_modified_AID_and_antibody_construct"> Co-transfection of modified AID and antibody construct </SPAN></H5><P>To determine the mutation rate, we transfected the modified AID-eGFP with the antibody construct-mCherry in CHO cells. We observed the red and green fluorescence indication that the co-transfection was successful (figure 6)
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6:</B> Co-transfection of modified AID-eGFP with Nanobody-mCherry construct in CHO cells (green and red fluorescence indicating that the co-transfection was successful)</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Comparison_of_mutation_rates_of_modified_and_wildtype_AID"> Comparison of mutation rates of modified and wildtype AID </SPAN></H5><P>For detecting the mutation sites and rates of the wt AID and the modified AID, we developed two approaches to detect mutation in a certain query sequence. First of all, we transfected the CHO cells with the single chain construct and the AID variant. After certain expression time we purified the cotransfected plasmids for the single chain construct for the first approach and transformed them into <I>E. coli</I> to enrich the mutations. After overnight culture and purification of the transformed plasmids we sent the samples for sequencing. For the second approach, we sorted the cells transfected with the modified AID-eGFP construct and the single chain YFP construct <I>via</I> FACS. These cells were lysed and the plasmids with the single chain purified. After that, the purified single chain constructs were transformed into <I>E. coli</I> to enrich the mutations.
There is a significant higher mutation rate in the cells transfected with the wildtype AID, modified AID and modified AID-eGFP in comparison to the ones without AID (figure 7). Surprisingly, the mutation rate of wt AID is two times higher than the mutation rate of the modified AID or modified AID with eGFP. This observation is opposite to our original expectations. One possible explanation is that the modified AID has a very high mutation rate and therefore the transfected cells die or inactivate the plasmid like it was observed for the wildtype AID (Martin and Scharff (2002)).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7:</B> Comparison of the mutation rates between the wildtype AID, modified AID and modified AID-eGFP. For each sample 10 clones were sequenced with an approximate read length of 1000 nt</DIV></DIV></DIV></DIV><H5><SPAN class="mw-headline" id="Mutation_rate_in_E._coli_measured_via_Phage_display"> Mutation rate in <I>E. coli</I> measured <I>via</I> Phage display </SPAN></H5><P>The expected mutations did not occur, so we started wondering what could be the reason for that. We decided to check if the expression of AID occurs after addition of arabinose. We took samples after 3 and 5 hours. There were no bands on the gel so we had no AID expression even though theoretically AID is placed behind the arabinose inducible promoter. We think the reason could be the 11bp long distance between RBS and start codon instead of the regular 7bp in the designed pBAD vector with AID. The experiments should therefore be repeated, starting from vector design. </P><P>How did we do that?
Figure 8 describes the subsequent steps: we first created two vectors: pAK100 with scFV - phagemid via PCR (Point 1) and pBAD with AID (Point 2). Then, we transformed the <I>E.coli</I> ER2738 strain with both of them (Point 3 and 4). Afterwards, we added the helper phages (VSCM13) and let them infect the cells. The culture was then streaked on an agar plate with proper antibiotica and the positive clones were sent for sequencing. Simultaneously, the phages were purified. For the subsequent rounds of phage display we collected all of the colonies grown on the plate and used them for overnight culture landing again in point three. Also instead of helper phages, the purified phages from previous round were used for infection.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 8:</B> Phage display - experiment design</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H4><H5><SPAN class="mw-headline" id="Cloning_of_wt_AID.2C_modified_AID_and_modified_AID_with_eGFP">Cloning of wt AID, modified AID and modified AID with eGFP</SPAN></H5><P>The sequencing results showed that the cloning of wt AID, modified AID and modified AID with eGFP was successful without any mutation in the coding sequence.
</P><H5><SPAN class="mw-headline" id="Comparison_of_the_mutation_rates_of_wt_AID_and_modified_AID_in_connection_with_the_intracellular_localization">Comparison of the mutation rates of wt AID and modified AID in connection with the intracellular localization</SPAN></H5><P>Like it was mentioned in the results section of the mutation module, there is a difference between the mutation rates of wt AID and modified AID. The wt AID has a two times higher mutation rate than the modified AID. We expected that this observation would be other way round. There are three possible explanations for this behavior which are based on the observation that there is a very high concentration of modified AID in the nucleus because of the engineered NLS and the knocked out NES, as we can see in the figure 4. This means that the mutagenic enzyme which mutates unspecifically single stranded DNA is highly expressed in the nucleus. In contrast, the wt AID is actively transported out of the nucleus and is located in the cytosol.
First possible explanation is that the cells that contain a functional, modified AID died and cannot be observed anymore.
The second possibility is that because of the high mutation rate the modified AID is inactivated by transfected cells similarly to what was observed for wt AID transfected CHO cells.(Martin and Scharff (2002))
And the last justification can be that the cells that lost the DNA coding for the modified AID have a selection advantage and therefore grow predominantly. 
We did not expect that the modified AID is disfunctional because of mutation or any folding interfering sequences like the NLS or NES because, firstly, the sequencing results show no mutation in the coding, promoter or terminator region of the modified AID. And secondly, we even observed a two times higher mutation rate in comparison to the control cells which are not transfected with AID or modified AID even after one day incubation. In prior publications, the incubation of the AID with selection marker for eukaryotic cells takes much longer to see any mutations.
</P><H4><SPAN class="mw-headline" id="References"> References </SPAN></H4><UL><LI> Muramatsu M, Kinoshita K, Fagarasan S, Yamada S, Shinkai Y, Honjo T. Class switch recombination and hypermutation require activation-induced cytidine deaminase (AID), a potential RNA editing enzyme. Cell. 2000 Sep 1;102(5):553-63. <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Muramatsu%2C%20M.%2C%20Kinoshita%2C%20K.%2C%20Fagarasan%2C%20S.%2C%20Yamada%2C%20S.%2C%20Shinkai%2C%20Y.%26Honjo%2C" class="external text" rel="nofollow">PubMed PMID: 11007474.</A></LI><LI> Revy P, Muto T, Levy Y, Geissmann F, Plebani A, Sanal O, Catalan N, Forveille M, Dufourcq-Labelouse R, Gennery A, Tezcan I, Ersoy F, Kayserili H, Ugazio AG, Brousse N, Muramatsu M, Notarangelo LD, Kinoshita K, Honjo T, Fischer A, Durandy A. Activation-induced cytidine deaminase (AID) deficiency causes the autosomal recessive form of the Hyper-IgM syndrome (HIGM2). Cell. 2000 Sep 1;102(5):565-75. <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Revy%2C%20P.%2C%20Muto%2C%20T.%2C%20Levy%2C%20Y.%2C%20Geissmann%2C%20F.%2C%20Plebani%2C%20A.%2C%20Sanal%2C%20O.%2C%20Catalan%2C" class="external text" rel="nofollow">PubMed PMID: 11007475.</A></LI><LI> Arakawa H, Hauschild J, Buerstedde JM. Requirement of the activation-induced deaminase (AID) gene for immunoglobulin gene conversion. Science. 2002 Feb 15;295(5558):1301-6. <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Arakawa%2C%20H.%2C%20Hauschild%2C%20J.%20%26%20Buerstedde%2C%20J.%20M.%20(2002)%20Science%20295%2C%201301%E2%80%931306." class="external text" rel="nofollow">PubMed PMID: 11847344.</A></LI><LI> Harris RS, Sale JE, Petersen-Mahrt SK, Neuberger MS. AID is essential for immunoglobulin V gene conversion in a cultured B cell line. Curr Biol. 2002 Mar 5;12(5):435-8.  <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Harris%2C%20R.%20S.%2C%20Sale%2C%20J.%20E.%2C%20Petersen-Mahrt%2C%20S.%20K.%20%26%20Neuberger%2C%20M.%20S.%20(2002)%20Curr." class="external text" rel="nofollow">PubMed PMID: 11882297.</A></LI><LI> Prochnow C, Bransteitter R, Klein MG, Goodman MF, Chen XS. The APOBEC-2 crystal structure and functional implications for the deaminase AID. Nature. 2007 Jan 25;445(7126):447-51. Epub 2006 Dec 24. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17187054" class="external text" rel="nofollow">PubMed PMID: 17187054.</A></LI><LI> Vallur AC, Yabuki M, Larson ED, Maizels N. AID in antibody perfection. Cell Mol Life Sci. 2007 Mar;64(5):555-65. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17262167" class="external text" rel="nofollow">Pubmed PMID: 17262167</A></LI><LI> Pasqualucci L, Migliazza A, Fracchiolla N, William C, Neri A, Baldini L, Chaganti RS, Klein U, Küppers R, Rajewsky K, Dalla-Favera R. BCL-6 mutations in normal germinal center B cells: evidence of somatic hypermutation acting outside Ig loci. Proc Natl Acad Sci U S A. 1998 Sep 29;95(20):11816-21. <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Pasqualucci%2C%20L.%2C%20Migliazza%2C%20A.%2C%20Fracchiolla%2C%20N.%2C%20William%2C%20C.%2C%20Neri%2C%20A.%2C%20Baldini%2C" class="external text" rel="nofollow">PubMed PMID: 9751748;</A></LI><LI> Pasqualucci L, Neumeister P, Goossens T, Nanjangud G, Chaganti RS, Küppers R, Dalla-Favera R. Hypermutation of multiple proto-oncogenes in B-cell diffuse large-cell lymphomas. Nature. 2001 Jul 19;412(6844):341-6 <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Pasqualucci%2C%20L.%2C%20Neumeister%2C%20P.%2C%20Goossens%2C%20T.%2C%20Nanjangud%2C%20G.%2C%20Chaganti%2C%20R.%20S.%2C" class="external text" rel="nofollow">PubMed PMID: 11460166.</A></LI><LI> Shen HM, Peters A, Baron B, Zhu X, Storb U. Mutation of BCL-6 gene in normal B cells by the process of somatic hypermutation of Ig genes. Science. 1998 Jun 12;280(5370):1750-2.  <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Shen%2C%20H.%20M.%2C%20Peters%2C%20A.%2C%20Baron%2C%20B.%2C%20Zhu%2C%20X.%20%26%20Storb%2C%20U.%20(1998)%20Science%20280%2C" class="external text" rel="nofollow">PubMed PMID: 9624052.</A></LI><LI>  Müschen M, Re D, Jungnickel B, Diehl V, Rajewsky K, Küppers R. Somatic mutation of the CD95 gene in human B cells as a side-effect of the germinal center reaction. J Exp Med. 2000 Dec 18;192(12):1833-40. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11120779" class="external text" rel="nofollow">PubMed PMID: 11120779.</A></LI><LI> Bowers PM, Horlick RA, Neben TY, Toobian RM, Tomlinson GL, Dalton JL, Jones HA, Chen A, Altobell L 3rd, Zhang X, Macomber JL, Krapf IP, Wu BF, McConnell A, Chau B, Holland T, Berkebile AD, Neben SS, Boyle WJ, King DJ. Coupling mammalian cell surface display with somatic hypermutation for the discovery and maturation of human antibodies. Proc Natl Acad Sci U S A. 2011 Dec 20;108(51):20455-60. <A href="http://www.ncbi.nlm.nih.gov/pubmed/22158898?dopt=Abstract" class="external text" rel="nofollow">Pubmed PMID: 22158898</A></LI><LI> Martin A, Scharff MD. Somatic hypermutation of the AID transgene in B and non-B cells. Proc Natl Acad Sci U S A. 2002 Sep 17;99(19):12304-8. Epub 2002 Aug 29.  <A href="http://www.ncbi.nlm.nih.gov/pubmed?term=Somatic%20hypermutation%20of%20the%20AID%20transgene%20in%20B%20and" class="external text" rel="nofollow">PubMed PMID: 12202747.</A></LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">http://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Project/Part_AID" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Project/Part_AID" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Part_AID&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/Part_AID&amp;oldid=290290" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>