<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Potsdam_Bioware_Project_At_a_Glance"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Potsdam_Bioware/Project/At_a_Glance">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Potsdam_Bioware/Project/At_a_Glance&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/At_a_Glance&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/At_a_Glance&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Potsdam_Bioware/Project/At_a_Glance" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Potsdam Bioware/Project/At a Glance</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="nav_menu gradient_golden"><DIV id="menu_center"><UL id="menu"><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware">Home</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">Project</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">At a Glance</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance#Main_Results">Main Results</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID">Mutation Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody">Antibody Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus">Selection Module</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam Standard">Potsdam Standard</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Collaboration">Collaboration</A></LI></UL><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Team</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Sponsoring">Sponsoring</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Team/Attribution">Attribution</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Lab</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Group_Meetings">Group Meetings</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Lab/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">BioBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Biobricks/All_Biobricks">All Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">SocialBricks</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_Politics">Science Meets Politics</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick/Science_Meets_People">Science Meets People</A></LI><LI><A href="https://2012.igem.org/Team:Potsdam_Bioware/Software">Software</A></LI></DIV></DIV><DIV class="box_round white_bg"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#At_a_Glance:_Antibody_Generation_System"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">At a Glance: Antibody Generation System</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Antibodies_in_a_Nutshell"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Antibodies in a Nutshell</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#State_of_the_Art"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">State of the Art</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Our_Approach"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Our Approach</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#The_Antibody_Module"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">The Antibody Module</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#The_Mutation_Module"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">The Mutation Module</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#The_Selection_Module"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">The Selection Module</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Main_Results"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Main Results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Antibody_Module:_Antibody_Expression"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Antibody Module: Antibody Expression</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Mutation_Module:_Antibody_Maturation_by_Mutation_with_the_AID_Enzyme"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Mutation Module: Antibody Maturation by Mutation with the AID Enzyme</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Selection_Module:_Selection_or_Screening_for_the_Desired_Clone"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Selection Module: Selection or Screening for the Desired Clone</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Modeling:_Analyzing_and_Predicting_the_Viral_Antibody_Selection"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Modeling: Analyzing and Predicting the Viral Antibody Selection</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#SocialBricks"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">SocialBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Potsdam_Standard_BBF_RFC_91:_a_hybrid_approach_to_assemble_your_gene_of_interest"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Potsdam Standard BBF RFC 91: a hybrid approach to assemble your gene of interest</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="At_a_Glance:_Antibody_Generation_System">At a Glance: Antibody Generation System</SPAN></H2><H3><SPAN class="mw-headline" id="Antibodies_in_a_Nutshell">Antibodies in a Nutshell</SPAN></H3><P>The goal of our &quot;Antibody Generation System&quot; is to streamline the generation and production of antibodies by integrating all steps in one cell line. Antibodies are indispensable tools for research and diagnostics and they are used in diverse applications such as ELISA, western blot, affinity purification, and imuno-histology. Antibodies also represent the most important class of biopharmaceuticals with US sales of over 18 billion US$ in 2010 (Aggarwal, 2011) and various therapeutic areas such as cancer, arthritis, and macular degeneration. Depending on the use of the antibodies specific demands must be met. In research, the murine IgG is very typical. For therapy, antibodies need to be 'humanized' to avoid immunological reactions. For some applications downsized or modified versions of antibodies can be used: such as single chain Fv fragments (scFv), Fab fragments, or heavy chain only (camelid) antibodies. However most of the time, the Fc part of the antibody provides crucial functionality like secondary antibody binding in research or cell dependent cytotoxicity in therapy. Most antibodies produced in larger scale are recombinantly expressed in CHO (chinese hamster ovary) cells, due to the fact that CHO cells mimic human glycosylation and can produce large amounts of antibodies. In general, almost 70% of all biopharmaceuticals are produced in CHO cells (CHO consortium, 2012).
</P><H3><SPAN class="mw-headline" id="State_of_the_Art">State of the Art</SPAN></H3><P>So far, antibodies are typically produced by immunizing mice, sacrificing mice, generating hybridoma by cell fusion and selecting the desired hybridoma clones. This process is very time consuming and the expression quality varies widely. In addition, only natural murine antibodies can be obtained. For modifications or large scale production, the antibodies genes are mostly identified by phage display and then recloned in a defined expression cell line such as CHO. As this is very time consuming, <I>E. coli</I> based antibody fragment libraries and phage display emerged as a second route. However, in here only scFv or Fab fragments are obtained in the first place, and most applications benefit from the divalent character of the full antibody and need the Fc part. Since the Fc part requires glycosylation for stability a switch from <I>E. coli</I> to eukaryotic cells is necessary, which again is a laborious step.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:922px;"><DIV class="thumbcaption"><B>Figure 1:</B> schematic illustration: Overview of the Antibody Generation System</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Our_Approach">Our Approach</SPAN></H3><P>We aspire to implement all steps for antibody production the in the well established CHO cells. Our method avoids animal immunization or phage display and the additional recloning and thus bears the potential to be very fast and easy to automatize from start to finish. In addition, we can implement all desired antibody formats large or small and with or without glycosilation. To accomplish our goal, we conceptually divided the steps in modules, which can be tested independently. 
</P><H4><SPAN class="mw-headline" id="The_Antibody_Module">The Antibody Module</SPAN></H4><P>The design specifications for the antibody module are manifold. The antibody needs to be expressed in CHO cells in a manner that is amenable to selection and maturation as well as soluble/secreted expression. For selection we use cell surface display, which couples phenotype and genotype. Maturation is outlined in the mutations module (see below). And finally for production, we need to implement a genetic switch from cell surface expression to soluble expression. All steps require stable expression over longer periods necessitating stable transfection (i.e. chromosomal integration) of the antibody construct. 
</P><P>We achieved stable integration in CHO cells using the recombination based Flp-In system. As a start we used two different antibody constructs. Our &quot;small antibody construct&quot; gene comprises a signal peptide, a scFv directed against the EGFR (epidermal growth factor receptor), a protease cleavage site, a transmembrane region and as an intracellular reporter a YFP (yellow fluorescent protein). Our advanced antibody construct is a fully synthetic gene comprising a signal peptide, a heavy chain only (nanobody) fragment directed against GFP (green fluorescent protein), an human IgG Fc part, a protease cleavage site a loxP site (which gets translated), a transmembrane region, an mCherry as intracellular reporter and a second loxP site. Both antibody constructs are expressed under the CMV promoter. These two constructs serve to test the system and only at a later stage we plan to generate diversified antibody libraries. The loxP sites in the advanced construct enabled the genetic switch to soluble expression by deleting the transmembrane region and the mCherry reporter upon transfection and expression of the Cre-Recombinase.
</P><H4><SPAN class="mw-headline" id="The_Mutation_Module">The Mutation Module</SPAN></H4><P>We expect that stable transfection of antibody libraries is a limiting step in our systems since the up-scale of this process is cumbersome. We estimate that a diversity in the range 10<SUP>6</SUP> can be achieved, which is a good starting point to identify lead compounds for binding. However, to generate high affinity antibodies, further maturation might be required. To do so, we utilize the enzyme Activation Induced Cytidine Deaminase (AID), which is one of the key enzymes of antibody maturation in the mammalian immune system. We generated several variants of the AID with deleted nuclear export and added nuclear import sequences and a fused GFP reporter. The resulting mutation rate on the small antibody construct upon co-transfection in CHO-cells was tested to identify the best candidate for mutation. The new constructs localized as expected, and co-expression increased the mutation rate. In comparison, the wt-AID maintained the highest mutation rate. In the next step we will combine maturation with selection.
</P><H4><SPAN class="mw-headline" id="The_Selection_Module">The Selection Module</SPAN></H4><P>Selection is key to the identification of the best binders. Our &quot;selection module&quot; comprises a set of methods to select the desired cell clones from the collection of cells expressing variants of an antibody construct. Depending on the needs, different selection methods might have specific advantages. We set up three methods, whereby two are well established but discontinuous: i) fluorescent activated cell sorting (FACS), which yields many data but requires expensive instrumentation and ii) magnetic beads, which are less expensive and easy to automatize; and last but not least, we develop iii) a novel continuously acting viral selection method. The latter method is based on the assumption that antibody presentation and affinity can be used to modulate viral infection. Virus particles displaying the desired antigen can then either deliver a growth promoting or inhibiting signal. We achieve this by manipulating virus-like particles based on the recombinant Adeno-Associated Virus (rAAV) to present the antigen on the viral surface by either genetic fusion to the capsid protein or by a general enzymatic coupling step using sortase ligation technology. So far we have tested cell sorting (FACS) based on antibody expression, we have coupled YFP, which is the antigen for our test antibody system, to magentic beads, and we have produced rAAV fused at the genetic level to the antigens YFP, CFP and the sortase ligation motif. As a growth inhibiting signal our virus particles deliver a thymidine kinase gene, which enables killing by ganciclovir, and as growth promoting signal a neomycin/geneticin resistance cassette. We are in the process to determine which selection method works the best. As we expect that the viral uptake increases with antibody affinity this system was used in our modeling approaches.
</P><P>
Aggarwal S. What's fueling the biotech engine--2010 to 2011. Nat Biotechnol. 2011 Dec 8;29(12):1083-9. doi: 10.1038/nbt.2060. <A href="http://www.ncbi.nlm.nih.gov/pubmed/22158359" class="external text" rel="nofollow">PubMed PMID: 22158359.</A>
CHO consortium <A href="http://hugroup.cems.umn.edu/CHO" class="external free" rel="nofollow">http://hugroup.cems.umn.edu/CHO</A></P><H2><SPAN class="mw-headline" id="Main_Results">Main Results</SPAN></H2><H3><SPAN class="mw-headline" id="Antibody_Module:_Antibody_Expression"><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody" class="external text" rel="nofollow">Antibody Module:</A> Antibody Expression</SPAN></H3><P>The aim of the antibody module was to design and assemble antibody constructs that would demonstrate the principle of our generation system. We designed two exemplary antibody constructs: The smaller construct contains a single chain fragment variable domain (scFv) targeting the human EGF-Receptor, a transmembrane domain, a TEV protease cleavage site and an eYFP. The advanced antibody construct consists of a nanobody binding GFP/YFP, a transmembrane domain, a TEV protease cleavage site, mCherry and two LoxP sites. 
</P><P>We successfully stably and transiently transfected both antibody constructs in CHO Flp-in cells. With fluorescence microscopy, confocal microscopy, immunfluorescence and FACS we were able to show the expression of the transfected parts. Under certain conditions we have seen a membrane localization of the advanced construct yet we have not been able to generate cells that transport enough molecules to their surface for successful detection. However we have shown that the switch from presenting antibodies to secreting antibodies via the Cre recombinase was successful. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody" class="external text" rel="nofollow">read more</A></P><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>transiently transfected CHO cell lines with two different antibody constructs (left picture: advanced antibody construct anti-GFP <A href="http://partsregistry.org/Part:BBa_K929107" class="external text" rel="nofollow">BBa_K929107</A>and smaller antibody construct anti-EGFR scFv425 <A href="http://partsregistry.org/Part:BBa_K929101" class="external text" rel="nofollow">BBa_K929101</A> (not shown)) </B></TD></TR><TR><TD><B>stably transfected CHO cell lines with two different antibody constructs (advanced antibody construct anti-GFP nanobody <A href="http://partsregistry.org/Part:BBa_K929107" class="external text" rel="nofollow">BBa_K929107</A> (not shown), right picture: smaller antibody construct with anti-EGFR scFv425 <A href="http://partsregistry.org/Part:BBa_K929101" class="external text" rel="nofollow">BBa_K929101</A>) </B></TD></TR><TR><TD><B>membrane localization of advanced construct (<A href="http://partsregistry.org/Part:BBa_K929107" class="external text" rel="nofollow">BBa_K929107</A>)</B></TD></TR></TBODY></TABLE><TABLE style="border-spacing:0px"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:397px;"><DIV class="thumbcaption"><B>Figure 2:</B> Fluorescence image of mCherry -  advanced antibody construct showing membrane localisation on CHO cells</DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:462px;"><DIV class="thumbcaption"><B>Figure 3:</B> Fluorescence image of eYFP of the small construct in stably transfected CHO cells</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>proof of functionality of Cre recombinase</B></TD></TR></TBODY></TABLE><P>The switch from antibody presentation to antibody secretion is induced by the Cre Recombinase. Under the influence of Cre Recombinase the transmembrane domain and mCherry of the advanced antibody construct are eliminated and the antibody is secreted into the supernatant. We demonstrated the functionality of the Cre recombinase via Western Blot. The supernatant was collected and incubated with magnetic beads coupled to GFP via biotin-streptavidin. We detected a band at 41kDa which correlates with the size of the nanobody-Fc construct. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Antibody" class="external text" rel="nofollow">read more</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:178px;"><DIV class="thumbcaption"><B>Figure 4:</B> Western Blot of purified Nanobody-Fc from supernatant of cells co-transfected with Cre recombinase and advanced construct. Purification was carried out with magnetic beads. Correct band can be seen at 41kDa</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Mutation_Module:_Antibody_Maturation_by_Mutation_with_the_AID_Enzyme"><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID" class="external text" rel="nofollow">Mutation Module:</A> Antibody Maturation by Mutation with the AID Enzyme</SPAN></H3><P><B>A</B>ctivation <B>I</B>nduced Cytidine <B>D</B>eaminase (AID) is one of the key enzymes of antibody maturation in immune system of mammalian organisms. We used the AID to mutate the antibody sequences in CHO cells and in <I>E. coli</I> during Phage Display. In prior publications, it was shown that the AID is active in CHO cells and <I>E. coli</I> by using transfection or transformation, respectively. 
The goals of the mutation module were to express the wildtype and modified AID in CHO cells, to prove the nuclear localization of the modified AID, to determine the mutation rate of wildtype AID and modified AID and to co-transfect the AID with antibody construct. The goal to maturate antibodies was not achieved. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID" class="external text" rel="nofollow">read more</A></P><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>expression of wildtype <A href="http://partsregistry.org/Part:BBa_K929000" class="external text" rel="nofollow">BBa_K929000</A> and modified AID (<A href="http://partsregistry.org/Part:BBa_K929003" class="external text" rel="nofollow">BBa_K929003</A>&amp;<A href="http://partsregistry.org/Part:BBa_K929002" class="external text" rel="nofollow">BBa_K929002</A>) in CHO cells</B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>nuclear localisation of modified AID+eGFP <A href="http://partsregistry.org/Part:BBa_K929003" class="external text" rel="nofollow">BBa_K929003</A></B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>co-transfection of AID and antibody construct </B></TD></TR></TBODY></TABLE><TABLE cellpadding="0"><TBODY><TR><TD valign="top"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:507px;"><DIV class="thumbcaption"><B>Figure 5:</B> Overlay image of intracellular localization of modified AID with eGFP</DIV></DIV></DIV></DIV></TD><TD valign="top"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 6:</B> Overlay image of cotransfection of modified AID with eGFP and antibody-mCherry construct</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>determination of mutation rates of wildtype and modified AID</B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>recloning of purified antibody plasmids from CHO cells in <I>E. coli</I></B></TD></TR></TBODY></TABLE><TABLE><TABLE class="table center" border="0"><TBODY><TR><TD><P>To find out the mutation rates of the different AID variant, we tried an unusal setup: recloning of purified antibody plasmids from CHO cells in <I>E. coli</I>. That means that CHO cells were co-transfected with one of the AID constructs (wildtype, modified AID, modified AID+eGFP) and cultured over several days. Afterwards CHO-cells were lysed and plasmid DNA was prepped. To multiply the mutated (and non mutated)plasmids, they were recloned into <I>E.coli</I> and single clones were sequenced. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_AID" class="external text" rel="nofollow">read more</A></P></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7:</B> Comparison of the mutation rates between the wildtype AID, modified AID and modified AID-eGFP in CHO cells (approximately 5000 nucleotides were sequenced)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Selection_Module:_Selection_or_Screening_for_the_Desired_Clone"><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus" class="external text" rel="nofollow">Selection Module:</A> Selection or Screening for the Desired Clone</SPAN></H3><P>To select the desired cells from the ones expressing the antibody construct, targeted vital particles are required. Therefore, recombinant adeno-associated virus (rAAV) with yellow fluorescence protein (YFP) on the surface and cyan fluorescence protein (CFP) as Gene of interest was constructed.
HT1080 were infected with the rAAV. The rAAV with the YFP  on the surface and CFP as Gene of interest was used. As the rAAV can infect the HT1080 cells, the HT1080 cells show cyan fluorescence.
CHO-cells infected with rAAV with the YFP on the surface and CFP  as Gene of interest. The CHO-cells can not be infected by the rAAV, which is shown by the yellow fluorescence in the unbleached picture. Bleached CHO-cells dont show YFP fluorescence. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus" class="external text" rel="nofollow">read more</A></P><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>proof of concept: infection by rAAV leads to CFP expression as a selection marker, YFP labeled rAAV is attached to CHO cells</B></TD></TR></TBODY></TABLE><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 8:</B> Fluorescence image of the positive control – infected HT1080 cells with the recombinant adeno-associated virus with YFP on the surface and CFP as the Gene of interest. Cyan fluorescence is the indicator for a successful infection</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 9:</B> Fluorescence image of the negative control - non transfeced CHO-cells with the recombinant adeno-associated virus rAAV). The rAAV are not able to infect the CHO-cells (without nanobody). A) unbleached B) bleached</DIV></DIV></DIV></DIV><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>expression of sortase motiv+EGFR domain 3 wokrs as a first step to universal virus surface modification</B></TD></TR></TBODY></TABLE><CENTER><P>Sortase is an enzyme which catalyzes specific ligation of two proteins to each other. The expressed external ligand-binding 3rd domain of EGFR contains the C-terminal Sortase motif to be recognized by the enzyme Sortase. The coupled protein consisting of 3rd domain of EGFR and the Sortase motif has the size of 24.8 kDa. The N-terminal Sortase motif was fused to the N-terminus of VP2/3 gene of adeno-associated virus to allow the ligation to EGFR by the Sortase. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Part_Virus" class="external text" rel="nofollow">read more</A></P></CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 10:</B> expression of the 3rd EGFR domain in <I>E. coli</I> detection was performed using SDS-PAGE and Western-blot</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Modeling:_Analyzing_and_Predicting_the_Viral_Antibody_Selection"><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling" class="external text" rel="nofollow">Modeling:</A> Analyzing and Predicting the Viral Antibody Selection</SPAN></H3><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>The theoretical model supports our assumptions </B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><P>Modeling is a powerful tool to understand complex interactions or processes. We created a model to illustrate our selection system and to get a better understanding of its behavior under different conditions. Thereby, we wanted to answer two main questions:
</P><OL><LI>At which time point selection is finished? 
</LI><LI>How many antibody presenting cells and how many virus particles do we need at the beginning? 
</LI></OL><P>To answer these questions we did both deterministic and stochastic modeling using MATLAB. Furthermore, we did a huge parameter analysis for each model to check what kind of influence each parameter has on our selection system. 
</P><P>Stochastic results showed that the selection is finished on random time points after 100 h or later. Furthermore the analysis of initial concentrations of wt cells and virus revealed that there is an optimum for initial concentrations of wt cells and virus (Figure 11). Both to less and to excessive concentrations prevent the success of the selection system. This optimum is influenced by the other parameters, but if we can determine some of these parameters we can calculate the optimal concentrations to start our selection. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Modeling" class="external text" rel="nofollow">read more</A></P></TD></TR><TR><TD><P><B>Figure 11:</B> Dependency of success form initial virus and wt cell concentrations. There are optimal initial concentrations for virus and wt cell
</P></TD></TR></TBODY></TABLE></TABLE><H3><SPAN class="mw-headline" id="SocialBricks"><A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick" class="external text" rel="nofollow">SocialBricks</A></SPAN></H3><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>Science Meets People &amp; Science Meets Politics </B></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD valign="top"><P>The idea of SocialBricks is to divide all human practice activities into different parts: the SocialBricks. Here, the SocialBricks stand for every activity which aims to inform people about the Synthetic Biology. We hope that the term SocialBricks will be accepted like the term BioBrick and will be integrated in a registry to show the society what every team has done for more elucidation.
For this year we focused on two SocialBricks: “Science meets Politics” and “Science meets People”. For the first one we interviewed politicians from the German parliament: the Bundestag and discussed about Synthetic Biology. For the second part we organized a survey to ask about people's knowledge of Synthetic Biology and for their opinion on this scientific field. We also organized the day of synthetic biology in Potsdam on the main street.The main result of the human practice this year is that the citizens and the politicians see the great potential of Synthetic Biology but also the challenges of this new scientific field. <A href="https://2012.igem.org/Team:Potsdam_Bioware/SocialBrick" class="external text" rel="nofollow">read more</A></P></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 12:</B> Meeting with the Minister of Education and Research: Prof. Dr. Schavan</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Potsdam_Standard_BBF_RFC_91:_a_hybrid_approach_to_assemble_your_gene_of_interest"><A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam_Standard" class="external text" rel="nofollow">Potsdam Standard BBF RFC 91:</A> a hybrid approach to assemble your gene of interest</SPAN></H3><TABLE style="border-spacing:0px"><TBODY><TR><TD><B>Potsdam Standard works, new Biobrick was made with Potsdam Standard </B></TD></TR><TR><TD><B>the correct clones can be seen with naked eye </B></TD></TR></TBODY></TABLE><P>The main problems in using the classical approach to assemble different parts with restriction enzymes are on one hand ineffective enzymes with different optimum conditions and on the other hand illegal restriction site in the sequence which restricts the use of standard assemblies.
That is the reason why we tried to establish a new RFC with a reduced use of restriction enzymes. This cloning standard is based on the use of thiophosphate primer at the 5’ end for PCR to amplifying the insert. The insert is incubated in iodine/ethanol solution to knock out the 5’ thiophosphates. After that, the new standard cloning vector, developed by us, with a RFP expression cassette as a ligation control is digested with the enzymes Apa I and Sph I. These enzymes generate 3’ overhangs which correspond to the 3’overhang generated by knocking out the thiophosphates. The digested backbone and the pliced insert is mixed, ligated and transformed into <I>E.coli</I>.
To proof the new assembly standard, we insert the AID into the new standard cloning vector using the Potsdam Standard. After sequencing, we saw that the cloning was successful without any mutation in the AID sequence. <A href="https://2012.igem.org/Team:Potsdam_Bioware/Project/Potsdam_Standard" class="external text" rel="nofollow">read more</A></P><CENTER><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 13:</B> Overnight culture of transformed <I>E.coli</I> using potsdam standard</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 14:</B> After centrifugation - red coloured cell pellets</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance">http://2012.igem.org/Team:Potsdam_Bioware/Project/At_a_Glance</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Potsdam_Bioware/Project/At_a_Glance" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Potsdam_Bioware/Project/At_a_Glance" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/At_a_Glance&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Potsdam_Bioware/Project/At_a_Glance&amp;oldid=298347" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>